<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mlc1p Light Chain Unconventional Myosin <br /> Myo2p Saccharomyces cerevisiae <br /> Richard C. Stevens Trisha N. Davis <br /> Department Biochemistry, University Washington, Seattle, Washington 98195-7350 <br />  <br />  <br />  <br />  <br /> Abstract. Saccharomyces cerevisiae, unconven-                         haploinsufficiency. Reduced levels MYO2 overcome <br /> tional myosin Myo2p fundamental importance                          haploinsufficiency MLC1. mutant MYO2- <br /> polarized growth. explore role neck                         D6IQ able suppress haploinsufficiency  <br /> gion associated light chains regulating Myo2p                     letion MLC1. used modified gel overlay assay <br /> function. Surprisingly, precise deletion                     demonstrate direct interaction Mlc1p IQ sites neck region results myosin,                     neck Myo2p. Overexpression MYO2 toxic, <br /> Myo2-D6IQp, support growth yeast                           causing severe decrease growth rate. MYO2 <br /> strain 90% rate wild-type isogenic strain.                    overexpressed, Myo2p fourfold stable exploit mutant characterization light                       wild-type strain. High copies MLC1 completely  <br /> chains Myo2p.  demonstrate local-                       come growth defects increase stability ization calmodulin sites polarized growth                           Myo2p. results suggest Mlc1p responsible largely depends IQ sites neck Myo2p.                        stabilizing myosin binding neck region. <br /> Second, demonstrate previously uncharacter- <br /> ized protein, Mlc1p, myosin light chain Myo2p.                       Key words: myosin &#226;&#8364;&#162; polarized &#226;&#8364;&#162; stability &#226;&#8364;&#162; Myo4 <br /> MLC1 (YGL106w) essential gene exhibits                            &#226;&#8364;&#162; cytokinesis <br />  <br />  <br />  <br />                                                                              Mooseker, 1992; Rayment et al., 1993). IQ sites z25&#226;&#8364;&#8220; <br />  <br /> T <br />        Saccharomyces cerevisiae unconventional myo- <br />        sin Myo2p member class V myosins                      <span id='am-37' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-38' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> residue motifs bind calmodulin myosin <br />        implicated vesicle movement polar-                    light chains. cases, binding small <br /> ized growth (Johnston et al., 1991; Govindan et al., 1995).                  EF-hand proteins activates Mg21ATPase activity Homologues Myo2p include mouse dilute, plays                      myosins. example, calmodulin required role melanosome transport (Provance et al., 1996;                         Mg21ATPase activity chicken myosin V (Espindola et <br /> Nascimento et al., 1997), chicken myosin V,                    al., 1992). Furthermore, light chains perform structural role neuron filopodial extension (Wang et al., 1996).                   roles affecting myosin head orientation orien- <br /> Myo2p essential growth S. cerevisiae, local-                tation light chains  example, izes bud tip bud formation bud                      light chains scallop myosin II required stabilize <br /> neck cytokinesis (Brockerhoff et al., 1994; Lillie                    Ca21 binding myosin head domain alter Brown, 1994). temperature-sensitive mutation                        myosin head orientation (Fromherz Szent-Gyorgyi, <br /> MYO2 confers defects polarized growth vacuole                      1995). Myo2p tandem IQ sites (Johnston et al., <br /> inheritance general secretion (Johnston et al.,                   1991). <br /> 1991; Govindan et al., 1995; Hill et al., 1996).                                previously shown calmodulin binds   myosins light chain binds                    Myo2p perform essential function polarized myosin heavy chain light chain binding motif                       growth (Brockerhoff et al., 1994). calmodulin <br /> called IQ site (Cheney Mooseker, 1992; Xie et al.,                    present sites polarized growth (Brockerhoff 1994). IQ sites tandem repeats                    Davis, 1992), calmodulin Myo2p direct phys- head-motor domain tail domain (Cheney                        ical contact IQ sites neck Myo2p <br />                                                                              (Brockerhoff et al., 1994). CMD1 mutants allele-spe- <br />                                                                              cific synthetic lethality mutant myo2-66, sug- <br />                                                                              gesting calmodulin Myo2p share essential <br /> Address correspondence Trisha N. Davis, Department Biochemis- <br />                                                                              function. <br /> try, Box 357350, University Washington, Seattle, WA 98195-7350. Tel.:        Interestingly, class V myosins S. cerevisiae, <br /> (206) 543-5345. Fax: (206) 685-1792. E-mail: tdavis@u.washington.edu         Myo2p Myo4p, confer deleterious effects cell <br />  <br />  <br />  <br /> &#239;&#163;&#169; Rockefeller University Press, 0021-9525/98/08/711/12 $2.00 Journal Cell Biology, Volume 142, Number 3, August 10, 1998 711&#226;&#8364;&#8220;722 <br /> http://www.jcb.org                                                           711 <br />  Table  Plasmids Used Study <br /> Plasmid                                  Parent vector                            Relevant markers construction*                              Reference source <br />  <br /> pBluescriptII KS (1)                                                        r <br />                                                                        <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span> f1 origin                                                        Stratagene <br /> pGEX-2T                                                                Ptac gene encoding GST                                          Pharmacia <br /> pQE30                                                                  Ptac 6XHis polylinker                                           Qiagen <br /> pRS306                                                                 URA3 f1 origin                                                       (Sikorski Hieter, 1989) <br /> pRS315                                                                 CEN6 ARSH4 LEU2 f1 origin                                            (Sikorski Hieter, 1989) <br /> pRS316                                                                 CEN6 ARSH4 URA3 f1 origin                                            (Sikorski Hieter, 1989) <br /> pGF27                              pRS304                              2-mm origin (YEp24 fragment) inserted AatII site                  G. Zhu <br /> pGF29                              pRS306                              2-mm origin (YEp24 fragment) inserted AatII site                  G. Zhu <br /> pJG28                              pSB5                                ampr trc promoter, CMD1                                              J. Geiser <br /> pJP10-2B                           YCp50                               URA3 MYO2 CEN4 ARS1                                                  (Johnston et al., 1991) <br /> pLI831                             pBluescriptII SK (1)                ADE3                                                                 E. Muller <br /> myo4D::URA3                        pBluescriptII KS (1)                myo4D::URA3                                                          S. Brown <br /> pRS23                              pBluescriptII SK (1)                5.6-kb ClaI fragment MYO2                                         study <br /> pRS28                              pRS27                               myo2D::TRP1                                                          study <br /> pRS29                              pRS25                               5.6-kb ClaI fragment MYO2                                         study <br /> pRS31                              pGF29                               5.6-kb ClaI fragment MYO2                                         study <br /> pRS43                              pRS29                               NheI site removed                                                    study <br /> pRS50                              pRS37                               2-mm origin (YEp24 fragment) inserted NotI site                   study <br /> pRS72                              pRS43                               NcoI site added codon 1 MYO2                                   study <br /> pRS172                             pRS43                               Internal deletion MYO2 (S787G                                     study <br />                                                                           Daa788&#226;&#8364;&#8220;927) known MYO2-D6IQ <br /> pRS174                             pRS306                              5.4-kb ClaI fragment MYO2-D6IQ                                    study <br /> pRS221                             pGF29                               5.4-kb ClaI fragment MYO2-D6IQ                                    study <br /> pRS276                             pBluescriptII KS (1)                3.3-kb BamHI fragment carrying MLC1                                  study <br /> pRS286                             pRS285                              mlc1D::TRP1                                                          study <br /> pRS289                             pRS316                              1.4-kb BamHI-SacI fragment MLC1                                   study <br /> pRS290                             pGF27                               1.4-kb BamHI-SacI fragment MLC1                                   study <br /> pRS296                             pQE30                               ampr promoter 6XHis-MLC1                                             study <br /> pRS321                             pRS315                              1.4-kb BamHI-SacI fragment MLC1                                   study <br /> pSB6                               pACYC177                            kanr lysis genes lambda                                           (Brockerhoff et al., 1994) <br /> pSB20                              pGEX-3X                             MYO2 (aa740&#226;&#8364;&#8220;1457) fused GST                                       (Brockerhoff et al., 1994) <br /> pSB21                              pGEX-3X                             MYO2 (aa740&#226;&#8364;&#8220;1116) fused GST                                       (Brockerhoff et al., 1994) <br /> pSB24                              pGEX-3X                             MYO2 (aa790&#226;&#8364;&#8220;924) fused GST                                        (Brockerhoff et al., 1994) <br /> pSB25                              pGEX-3X                             MYO2 (aa740&#226;&#8364;&#8220;833) fused GST                                        (Brockerhoff et al., 1994) <br /> pSB27                              pGEX-3X                             MYO2 (aa247&#226;&#8364;&#8220;740) fused GST                                        (Brockerhoff et al., 1994) <br /> pTD28                              pTD17                               2-mm origin LYS2 CMD1                                                (Davis Thorner, 1989) <br /> pTD29                              YEp24                               2-mm origin LYS2                                                     (Geiser et al., 1993) <br /> *Unless stated, markers parent plasmid present new construct. MYO2-D6IQ encodes internal deletion Myo2p mutation S787G missing amino acids 787&#226;&#8364;&#8220;927. MLC1 identical Saccharomyces cerevisiae open reading frame YGL106w. <br />  <br />  <br />  <br />  <br /> growth overexpressed (Haarer et al., 1994). Myo4p                                  cloned ClaI site pBluescriptII KS1 (Stratagene, La Jolla, CA) <br /> localizes bud essential polarized distri-                         make pRS23. pRS24, 4.1-kb NdeI-AflII fragment MYO2                                                                                         removed. Plasmid pRS24 digested EcoRI, ends filled bution asymmetry determinant, Ash1p (Bobola et al.,                              Klenow fragment DNA polymerase destroy EcoRI site, <br /> 1996; Jansen et al., 1996). MYO4 essential growth                            ends ligated T4 DNA ligase make pRS26.  <br /> (Haarer et al., 1994). cause growth defects ei-                             maining coding sequence MYO2 pRS26 replaced EcoRI <br /> ther MYO2 MYO4 overexpressed unknown.                                          site oligonucleotide-directed mutagenesis (Kunkel et al., 1987) using <br />                                                                                         primer MYO2D (59-CTCGCGCCATCAGTTGAATTCTATACG- <br />    investigate role light chains function                         CTGTAATATCTGCTG-39), creating plasmid pRS27. plasmid Myo2p. Localization calmodulin sites cell growth                            pRS28, 0.8-kb EcoRI fragment containing TRP1 (Davis et al., 1986) depends neck region Myo2p. S. cerevisiae ge-                              cloned EcoRI site pRS27. <br /> nome sequencing project revealed small protein simi-                                plasmid shuffle used study required plasmid encoding <br /> larities calmodulin myosin light chains.                            ADE3 MYO2. 5.6-kb ClaI fragment MYO2 cloned                                                                                         SmaI site pLI831 (Geiser et al., 1993) make pRS37. shuttle vec- protein, Mlc1p (myosin light chain), binds                           tor pRS50 contains 2-mm origin 2.1-kb EcoRI fragment YEp24 myosin neck regulates stability Myo2p.                                ligated NotI site pRS37. plasmids, 59 overhangs <br />                                                                                         filled treatment Klenow fragment DNA poly- <br />                                                                                         merase. <br /> Materials Methods                                                                      Additional plasmids containing MYO2 carrying URA3 <br />                                                                                         selectable marker. Plasmid pRS25 derivative pRS316 (Sikorski Plasmids                                                                                Hieter, 1989) HindIII-NotI fragment polylinker                                                                                         removed, 59 overhangs filled Klenow, plasmid plasmids used study listed Table                                   recircularized T4 DNA ligase. plasmid pRS29 contains    Plasmid pRS28, used precisely delete MYO2,                    5.6-kb ClaI fragment MYO2 plasmid pJP10-2B cloned steps. 5.6-kb ClaI fragment pJP10-2B, containing MYO2,                       ClaI site pRS25. pRS29, coding sequence MYO2 runs  <br />  <br />  <br />  Journal Cell Biology, Volume 142, 1998                                           712 <br />  direction f1 origin. later studies, NheI site           MMY28 carrying integrated form GFP&#226;&#8364;&#8220;CMD1 (Moser et al., 1997) <br /> pRS29 removed digestion NheI, filling Klenow,              make diploid strain RSY33. myo4D::URA3 construct  gift reclosing T4 DNA ligase create pRS43. high-copy number                 Susan Brown, University Michigan, Ann Arbor, MI) transformed <br /> plasmid pRS31 created cloning 5.6-kb ClaI fragment MYO2              RSY33 create strain RSY38, heterozygous GFP&#226;&#8364;&#8220;CMD1, ClaI site pGF29, 2-mm plasmid.                                       MYO2-D6IQ, myo4D. <br />    Deletion region encoding neck MYO2 oligo-                Strain RSY105 carrying deletion MLC1 difficult create  <br /> <span id='am-29' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-30' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>-directed mutagenesis (Kunkel et al., 1987).              cause MLC1 displays haploinsufficiency. create disruption, large size MYO2, cassette created containing middle               transformed JGY46 plasmid pRS289, carrying MLC1 gene. gene. 2.6-kb BamHI-EcoRI fragment MYO2                     resulting strain functioning copies MLC1. copy cloned BamHI EcoRI sites pBluescriptII KS1 make                 MLC1 removed step gene replacement, resulting <br /> pRS78. step deleting IQ sites introduce           strain copies gene. 1.7-kb BamHI-NdeI <br /> NarI restriction endonuclease cleavage sites coding sequence           fragment plasmid pRS286 encoding mlc1D::TRP1 transformed MYO2 end region encoding IQ sites. oligonucle-            JGY46(pRS289). 42 colonies obtained, unable lose <br /> otide NAR1 (59-GCAATAAGATGCATAATGGCGCCATTGTTATGA-                                  URA3-marked MLC1 plasmid pRS289 streaked minimal <br /> TCCAGAAG-39) results addition <span id='am-44' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-45' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> amino acid               medium containing 59-fluoro <span id='am-24' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orotic acid</span> (59-FOA).1 colonies <br /> residues 786 787 mutation S787A. oligonucleotide            likely candidates heterozygous mlc1D strains. test hy- <br /> NAR4       (59-GAAATTGAAACAATTGAAGGGCGCCGACGCTAA-                                  pothesis, strains transformed LEU2-marked MLC1 <br /> ATCAGTTAATC-39) results addition <span id='am-56' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> amino                 plasmid pRS321 streaked minimal medium containing 59-FOA. <br /> acids 927 928. mutating pRS78 using NAR1 NAR4,                strains able lose pRS289 given alterna- <br /> sulting plasmid digested NarI reclosed using T4 DNA ligase.           tive source MLC1 pRS321. RSY105 shown heterozy- mutant cassette ligated BamHI- EcoRI-cut pRS43             gous MLC1/mlc1D::TRP1 Southern blot analysis. RSY105 carry create plasmid pRS172, encodes Myo2p S787G muta-               plasmid source MLC1 survive. <br /> tion lacks amino acids 788&#226;&#8364;&#8220;927. mutant named MYO2-                  study null phenotype mlc1D, strains RSY105(pRS289) D6IQ. Plasmid pRS221 5.4-kb Asp718-SacI fragment containing                JGY46 sporulated SpoIII medium, 50 tetrads dissected <br /> MYO2-D6IQ cloned pGF29.                                                       strain. 24 h, spores examined germination    plasmid pRS174 created integrate MYO2-D6IQ ge-              colony growth. Images spores colonies obtained described <br /> nome S. cerevisiae. pRS174 ligating 5.4-kb ClaI frag-           using fluorescent microscope (model Axioplan; Carl Zeiss, , <br /> ment pRS172 containing MYO2-D6IQ ClaI site pRS306.                  Thornwood, NY) 0.3-s, low-light exposures. Germinating spores <br />    gene YGL106w encodes 149&#226;&#8364;&#8220;amino acid protein similarities             transferred 20 ml minimal medium containing 3.7% formalde- calmodulin. Based work described paper,               hyde. 60 min room temperature, 4,69-diamidino-2-phenylindole <br /> named YGL106w MLC1. 3.3-kb BamHI fragment lambda PM-                     (DAPI) added final concentration 50 mg/ml, nuclei 3353 (American Type Culture Collection, Rockville, MD) cloned             examined described  BamHI site pBluescriptII KS1, creating plasmid pRS276. Plasmid <br /> pRS287 derivative pRS276, 2.0-kb NdeI-NotI fragment removed, 59 overhangs filled Klenow, plasmid              Immunoblot Analysis <br /> recircularized T4 DNA ligase. plasmid pRS287, MLC1 open reading frame remaining pBluescript polylinker.                        SDS-PAGE immunoblot analysis performed described (Gei- <br />    plasmid delete MLC1 precisely similar way           ser et al., 1991) following exceptions. Strains analyzed myo2D construct. Plasmid pRS284 digesting pRS276                  grown 50 Klett units appropriate medium. centrifugation, <br /> EcoRI, filling 59 overhangs Klenow, recircularizing            cells washed lysis buffer (50 mM Tris, pH 8.0, 13 proteinase inhib- <br /> plasmid T4 DNA ligase. coding sequence MLC1 replaced               itor cocktail [Drubin et al., 1988]). Cells lysed mixing equal vol- EcoRI site oligonucleotide-directed mutagenesis (Kunkel et al.,         umes cells, glass beads (Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> , St. Louis, MO), lysis <br /> 1987) using primer MLC1D (59-CTAAATTTGCAGTTCCGCACTCT-                              buffer vortexing times 1 min alternating 1-min incuba- <br /> CAGAATTCTGTTATTCTATGTATGTGCG-39), creating pRS285.                               tions ice. centrifugation 15 min microcentrifuge 48C, <br /> 0.8-kb EcoRI fragment containing TRP1 (Davis et al., 1986) cloned              pellets washed lysis buffer solubilized using cracking EcoRI site pRS285 make pRS286. pRS286, TRP1                  buffer (10 mM sodium phosphate, pH 7.2, 1% b-mercaptoethanol, 1% <br /> gene orientation original MLC1.                              SDS, 6 M <span id='am-83' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-84' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span>). detection Myo2p, 30 mg protein loaded <br />    Plasmid pRS296, designed express 6XHis-tagged MLC1 Escheri-               lane 6% SDS&#226;&#8364;&#8220;polyacrylamide gel. Proteins transferred <br /> chia coli, steps. NcoI site placed         nitrocellulose membranes using wet-transfer apparatus (Bio-Rad <br /> codon MLC1 oligonucleotide-directed mutagenesis using primer                 Laboratories, Hercules, CA) following instructions provided MLC1-NCOI            (59-GGCTCTGGTGGCTGCCATGGTTATTCTAT-                            manufacturer. membranes incubated 1:400 dilution af- <br /> GTATGTGC-39) (Kunkel et al., 1987) make pRS291. mutagenesis                finity-purified anti-Myo2p antibody (Lillie Brown, 1994). Secondary created mutation S2A. MLC1 fragment digest-               antibody blotting grade goat anti&#226;&#8364;&#8220;rabbit IgG horseradish peroxidase <br /> ing pRS291 NcoI, filling using Klenow fragment DNA poly-            conjugate (Bio-Rad Laboratories) 1:5,000 dilution. Signal  <br /> merase, digesting SacI, purifying 0.8-kb fragment. Plasmid            tected using Renaissance luminol reagent (DuPont/NEN, Boston, MA) <br /> pRS296 digesting pQE30 (Qiagen, , Chatsworth,                 Hyperfilm-MP (Amersham Corp., Arlington Heights, . <br /> CA) BamHI, filling using Klenow fragment DNA poly- <br /> merase, digesting SacI, ligating 0.8-kb MLC1 fragment. <br />                                                                                    Gel Overlays <br />                                                                                    Production 35S-labeled 6XHis-Mlc1p follows. E. coli strain <br /> Media Strains                                                                  GM1 (Coulondre Miller, 1977) containing plasmids pRS296                                                                                    pSB6 grown 30 ml M9 medium (Miller, 1972) 1.0 mM <br /> Media growth S. cerevisiae E. coli described previously <br />                                                                                    MgCl2 (instead MgSO4), 0.4% glucose, 1 mg/ml thiamine, 50 mg/ml <br /> (Zhu et al., 1993). yeast strains used study listed Table II. <br />                                                                                    ampicillin, 15 mg/ml kanamycin, 8 mCi carrier-free Na35SO4 (DuPont/ <br /> Genetic manipulations transformations performed essentially                                                                                    NEN) 20 Klett units induced adding isopropyl-b-d-thioga- <br /> described previously (Sherman et al., 1986). RSY2 carrying deletion                                                                                    lactopyranoside final concentration 2 mM. 5 h 378C, MYO2 created step gene replacement (Rothstein, 1991) using <br />                                                                                    cells collected centrifugation resuspended 400 ml lysis plasmid pRS28 digested BglII ClaI transformed RSY1. <br />                                                                                    buffer (50 mM Tris, pH 7.5, 1 mM MgCl2, 1 mM phenylmethylsulfonyl strains containing ade3D-100 histidine auxotrophs, ADE3                                                                                    fluoride). sample frozen thawed times, alternating plasmid selected histidine prototrophy (Jones Fink, 1982). <br /> RSY2 transformed pRS50 carrying ADE3 MYO2. strain <br /> RSY2-60B MYO2 deleted genome carries pRS50. <br />    mutant MYO2-D6IQ integrated yeast step gene <br /> replacement (Rothstein, 1991) using plasmid pRS174 digested                   1. Abbreviations used paper: 59-FOA, 59-fluoro orotic acid; DAPI, <br /> BamHI. Strains RSY21 RSY22 shown contain MYO2-D6IQ                     49,6-diamidino-2-phenylindole; GFP, green fluorescent protein; GST, glu- PCR Southern blot analysis. RSY22 mated strain                    tathione-S-transferase. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Myosin Light Chain Myo2p                          713 <br />  Table II. Yeast Strains <br /> Strain                                                           Genotype*                                                            Reference <br />  <br /> CRY1                MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 trp1-1 ura3-1                                           R. Fuller (Stanford University, CA) <br /> JGY46               MATa/MATa ade2-1oc/ade2-1oc can1-100/can1-100 his3-                                                  (Geiser et al., 1991) <br />                      11,15/his3-11,15 leu2-3,112/leu2-3,112 trp1-1/trp1-1 ura3-1/ura3-1 <br /> EMY55-5D            MATa ade2-1oc ade3D-100 can1-100 cyh2r his3-11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1               E. Muller (University                                                                                                                             Washington, Seattle, WA) <br /> EMY55-6B            MATa ade2-1oc ade3D-100 can1-100 cyh2S his3-11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1               E. Muller (University                                                                                                                          Washington) <br /> MMY28               MATa ade2-1oc ade3D-100 can1-100 (S65T-GFP)&#226;&#8364;&#8220;CMD1 his3-                                               (Moser et al., 1997) <br />                       11,15 leu2-3,112 lys2D::HIS3 trp1-1 ura3-1 <br /> RSY1                EMY55-5D X EMY55-6B                                                                                  study <br /> RSY2                MATa/MATa ade2-1oc/ade2-1oc ade3D-100/ade3D-100 can1-100/can1-100 cyh2r/cyh2S                        study <br />                        his3-11,15/his3-11,15 leu2-3,112/leu2-3,112 lys2D::HIS3/lys2D::HIS3 MYO2/myo2D::TRP1 <br />                       trp1-1/-trp1-1 ura3-1/ura3-1 <br /> RSY2-60B            MATa ade2-1oc ade3 D-100 can1-100 his3-11,15 leu2-3,112                                              study <br />                       lys2D::HIS3 myo2D::TRP1 trp1-1 ura3-1 carrying plasmid pRS50 <br /> RSY21               MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 MYO2-D6IQ trp1-1 ura3-1                                 study <br /> RSY22               MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 MYO2-D6IQ trp1-1 ura3-1                                 study <br /> RSY38               MATa/MATa ade2-1oc/ade2-1oc ADE3/ade3D-100 can1-100/can1-100 CMD1/GFP&#226;&#8364;&#8220;CMD1                           study <br />                        his3-11,15/his3-11,15 leu2-3,112/leu2-3,112 MYO2/MYO2-D6IQ MYO4/myo4D::URA3 <br />                       trp1-1/trp1-1 ura3-1/ura3-1 <br /> RSY38-9D            MATa ade2-1oc ade3D-100 can1-100 GFP&#226;&#8364;&#8220;CMD1 his3-11,15                                                 study <br />                       leu2-3112 lys2D::HIS3 MYO2-D6IQ trp1-1 ura3-1 <br /> RSY38-16A           MATa ade2-1oc ade3D-100 can1-100 GFP&#226;&#8364;&#8220;CMD1 his3-11,15                                                 study <br />                       leu2-3,112 lys2D::HIS3 myo4D::URA3 trp1-1 ura3-1 <br /> RSY38-16C           MATa ade2-1oc ade3D-100 can1-100 his3-11,15 leu2-3,112                                               study <br />                       lys2D::HIS3 MYO2-D6IQ myo4D::URA3 trp1-1 ura3-1 <br /> RSY38-17C           MATa ade2-1oc ade3D-100 can1-100 GFP&#226;&#8364;&#8220;CMD1 his3-11,15                                                 study <br />                       leu2-3,112 lys2D::HIS3 trp1-1 ura3-1 <br /> RSY105              MATa/MATa ade2-1 oc/ade2-1 oc can1-100/can1-100 his3-                                                study <br />                       11,15/his3-11,15 leu2-3,112/leu2-3,112 MLC1/mcl1D::TRP1 <br />                       trp1-1/trp1-1 ura3-1/ura3-1 carrying plasmid pRS289 <br /> RSY105-6A           MATa ade2-1oc can1-100 his3-11,15 leu2-3,112 mcl1D::TRP1                                             study <br />                       trp1-1 ura3-1 carrying plasmid pRS289 <br /> RSY107              RSY21 X RSY105-6A carrying plasmid pRS289                                                            study <br /> RSY112              MATa/MATa ade2-1oc/ade2-1oc ade3D-100/ade3D-100 can1-100/can1-100 his3-11,15/his3-11,15              study <br />                        leu2-3,112/leu2-3,112 lys2D::HIS3/lys2D::HIS3 MLC1/mcl1D::TRP1 <br />                       MYO2/myo2D::TRP1 trp1-1/trp1-1 ura3-1/ura3-1 carrying plasmid pRS289 <br /> *GFP&#226;&#8364;&#8220;CMD1 encodes fusion protein green fluorescent protein calmodulin. <br />  <br />  <br />  <br />  220 208C lyse cells, centrifuged 10,000 g          branes probed 16 h 218C 140 mM 35S-labeled 30 min 48C microcentrifuge. supernatant fraction batch           6XHis-Mlc1p 156 mM 35S-labeled calmodulin buffer C (buffer mixed 10 min 700 ml preequilibrated Ni-NTA beads (Qiagen,                5 mM CaCl2 0.05% Tween-20). membranes ). beads poured form column washed 10 ml ly-           washed 40 min  changes) buffer C.  membranes sis buffer containing 25 mM imidazole. Purified 6XHis-Mlc1p eluted            air dried, dipped 7% 2,5-diphenyloxazole acetone, air dried, ex- column using lysis buffer containing 500 mM imidazole. imi-          posed Hyperfilm-MP (Amersham Corp.) 5&#226;&#8364;&#8220;8 d. <br /> dazole buffer exchanged 100 mM ammonium bicarbonate PD-10 G25 column (Pharmacia Biotech, Piscataway, NJ). Protein                 Northern Analysis MYO2 Transcript <br /> stored 2808C needed. Protein concentration determined using <br /> bicinchoninic acid (Sigma Chemical ). purified 35S-labeled 6XHis-          Total mRNA purified strains CRY1 carrying pRS43, <br /> Mlc1p specific activity 27 dpm/fmol.                                     pRS172, pRS31, pRS221 using previously described procedure (Wise, <br />     Purified 35S-labeled calmodulin previously described (Brock-      1991). 500-ml cultures, z7.0 mg RNA obtained. Total mRNA <br /> erhoff et al., 1992), 8 mCi Na35SO4 used labeling.    purified samples using Promega PolyA Tract mRNA purified 35S-labeled calmodulin specific activity 39 dpm/fmol.       Isolation (Madison, WI) MagneSphere particles. Approxi- <br />     gel overlay assays calmodulin Mlc1p binding              mately 50 mg poly containing mRNA purified culture. <br /> formed previously described (Brockerhoff et al., 1994) slight mod-           Northern blot analysis performed using standard procedures (Sam- <br /> ifications. brief, extracts E. coli strain GM1 (Cou-        brook et al., 1989) following modifications. 10 mg mRNA londre Miller, 1977) containing plasmid pGEX-2T, pSB20,                loaded lane prerun <span id='am-68' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> gel. removing pSB21, pSB24, pSB25, pSB27. Approximately 30 mg total protein              formaldehyde washing 23 DEPC dH2O, mRNA transferred loaded lane SDS-PAGE. Proteins transferred Immo-            nitrocellulose membrane using capillary transfer (Sambrook et al., <br /> bilon membranes (Millipore Corp., Bedford, MA) using wet-transfer               1989). membrane baked 658C 1 h. Probes using <br /> apparatus (Bio-Rad Laboratories) following instructions provided           3.4-kb NcoI-EcoRI fragment MYO2 pRS72 PCR prod- manufacturer. transfer, proteins renatured washing          uct containing 0.5-kb fragment TRX2. incubating mem- Immobilon membranes total 40 min  changes) buffer          brane z1 3 107 cpm probe (pooled) 48 h, membrane <br /> (20 mM Hepes, pH 7.2, 100 mM NaCl) blocked 6 h buffer                washed analyzed using Molecular Dynamics PhosphorImager <br /> containing 3% bovine serum albumin 0.05% Tween-20. mem-                   (model 400S; Sunnyvale, CA) using 176-micron pixel size. <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 142, 1998                                     714 <br />  Indirect Immunofluorescence Green Fluorescent                               Myo2-D6IQp stably produced wild-type <br /> Protein Fusions                                                                 Myo2p present strain (Fig. 1 C). mutant cells immunolocalization Myo2p strains CRY1 RSY21  <br />                                                                                 normal budding index average 10% <br /> formed previously described (Brockerhoff et al., 1994; Lillie            larger size. distribution bud sizes similar Brown, 1994).                                                                   seen wild-type cultures. mutant cultures  <br />    localization GFP&#226;&#8364;&#8220;Cmd1p strains RSY38-9D, RSY38-16A,                tained 5% large unbudded cells RSY38-16C, RSY38-17C performed using fluorescent micro- <br />                                                                                 wild-type cultures. strain carrying MYO2-D6IQ does <br /> scope (model Axioplan; Carl Zeiss, ) (Moser et al., 1997). 200 cells strain scored presence polarized calmodulin. Calmod-       grow 38.58C, isogenic wild-type strain <br /> ulin judged polarized fluorescent signal stronger   grows poorly. strain carrying MYO2-D6IQ grows bud mother portion cell. Images captured using         medium 1 M sorbitol (data shown). Imagepoint-cooled CCD video camera (Photometrics, Tucson, AZ) fit-             neckless mutant protein localizes indistinguishably <br /> ted microscope conjunction IP Lab software (Signal Analyt- <br /> ics, Vienna, VA). Images GFP&#226;&#8364;&#8220;calmodulin 5-s exposures. Previous <br />                                                                                 wild-type Myo2p (Fig. 2). cases,  <br /> studies shown expression GFP yeast resulted          budded cells localization small patch uniform distribution fluorescence cell, excluding         Myo2p. small- medium-budded cells, Myo2p vacuole (Niedenthal et al., 1996).                                              present concentrated region bud tip. large- <br />                                                                                 budded cells, Myo2p bud tip Results                                                                         bud neck (Fig. 2). results consistent previ- <br />                                                                                 ous analysis Myo2p localization (Brockerhoff et al., <br /> Myo2p Functions Neck Domain                                           1994; Lillie Brown, 1994). neck Myo2p contains IQ sites tandem array <br /> spanning amino acids 790&#226;&#8364;&#8220;921 (Johnston et al., 1991). <br />                                                                                 Neckless MYO2-D6IQ Affects Calmodulin Localization <br /> Myo2p essential growth, IQ sites                     previous results strongly suggest calmodulin light mutant MYO2-D6IQ, lacking sequence encoding                             chain Myo2p. Calmodulin localizes sites cell <br /> amino acids 787&#226;&#8364;&#8220;927, supports nearly normal growth source Myo2p, causing 10% reduction growth rate (Fig. 1). immunoblot confirmed neck- <br />  <br />  <br />  <br />  <br /> Figure 1. mutant MYO2-D6IQ allows growth. Strains carry- <br /> ing wild-type MYO2 (CRY1) MYO2-D6IQ (RSY21) plated YPD medium incubated 3 d 30 378C  . <br /> Log-phase cultures diluted 10,000 cells/ml YPD  <br /> dium, growth monitored 4 h. Cell number  <br /> termined Coulter counter (Coulter Corporation, Hialeah, <br /> FL). (B) j, wild-type MYO2 strain, CRY1; d, MYO2-D6IQ                           Figure 2. Comparison localization Myo2p Myo2- <br /> strain, RSY21. immunoblot extracts log-phase                    D6IQp. Cells stained affinity-purified anti-Myo2p anti- <br /> cultures strains CRY1 RSY21 (C). antibody anti-                  body described Materials Methods. Cells simulta- <br /> Myo2p (Lillie Brown, 1994).                                                 neously stained DAPI stain DNA. Bar, 5 mm. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Myosin Light Chain Myo2p                       715 <br />  Figure 3. Localization GFP&#226;&#8364;&#8220;calmodulin myo4D strains. Cultures RSY38-17c (WT), RSY38-16A (myo4D MYO2), RSY38-9D <br /> (MYO4 MYO2-D6IQ), RSY38-16C (myo4D MYO2-D6IQ) grown log phase. Samples collected, mixed equal vol- <br /> umes 1% agarose dissolved SDC, mounted microscope slides image collection described Materials Meth- <br /> ods. Bar, 5 mm. <br />  <br />  <br /> growth, calmodulin mutants display defects polar-           MYO2 localization GFP&#226;&#8364;&#8220;calmodulin ized growth (Brockerhoff Davis, 1992; Davis, 1992;             wild-type strain containing MYO4 MYO2. <br /> Ohya Botstein, 1994). Mutations CMD1 exacerbate              strain carrying MYO2-D6IQ myo4D, <br /> defects caused myo2-66 allele-specific manner.            GFP&#226;&#8364;&#8220;calmodulin polarized 4% cells (Fig. 3 viability MYO2-D6IQ strain allows test         Table III).  calmodulin localization sites localization calmodulin sites cell growth          polarized growth largely dependent IQ sites pends IQ sites Myo2p. Localization fu-          Myo2p, Myo4p contributes small extent. <br /> sion GFP calmodulin examined wild-type neckless mutant strains. GFP&#226;&#8364;&#8220;calmodulin functionally            MLC1, New Light Chain <br /> places calmodulin localizes properly (Moser et al.,            newly completed genome sequence S. cerevisiae <br /> 1997). strain carrying wild-type MYO2, GFP&#226;&#8364;&#8220;calmod-            identified potential myosin light chain. YGL106w  <br /> ulin sites cell growth 84% cells (Fig.    codes 149&#226;&#8364;&#8220;amino acid polypeptide significant ho- <br /> 3 Table III). number equivalent values ob-          mology calmodulin myosin light chains. Based tained immunolocalization (Brockerhoff Davis,               results described  renamed gene <br /> 1992). cells polarized calmodulin           MLC1. alignment vertebrate calmodulin large-budded cells process moving calmodulin             similar light chain reveals homology bud tip bud neck unbudded cells started bud formation. strain carrying <br /> MYO2-D6IQ, 15% cells polarized cal-               Table III. Calmodulin Localization Myosin Mutants <br /> modulin greatly reduced diffuse signal                                                                    Genotype                                    Polarized calmodulin (%) cells. Myo4p localizes diffusely yeast bud <br /> (Jansen et al., 1996). tested remaining             Wild type                                             84 <br /> GFP&#226;&#8364;&#8220;calmodulin bud MYO2-D6IQ strain              myo2-D6IQ                                             15 <br /> dependent Myo4p, essential growth.             myo4D                                                 80 <br /> Strains deleted MYO4 containing wild-type                  myo4D myo2-D6IQ                                        4 <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 142, 1998                      716 <br />                                                                                                  Figure 4. Alignment                                                                                                 predicted amino acid se- <br />                                                                                                 quence MLC1                                                                                                 small EF-hand proteins. <br />                                                                                                 Vertebrate calmodulin (Homo <br />                                                                                                 sapiens) amino acids 7&#226;&#8364;&#8220;149,                                                                                                 chicken light chain (Gallus <br />                                                                                                 gallus, MLE1) amino acids <br />                                                                                                 45&#226;&#8364;&#8220;190, Acanthamoeba <br />                                                                                                 myosin IC light chain   cas- <br />                                                                                                 tellanii, MICLC) amino acids <br />                                                                                                 7&#226;&#8364;&#8220;149 compared                                                                                                 complete amino acid sequence <br />                                                                                                 Mlc1p. Amino acid residues <br />                                                                                                 sharing identity                                                                                                 Mlc1p                                                                                                 sequences shaded. <br />                                                                                                 Amino acids potential <br />                                                                                                 Ca21-binding loops la- <br />                                                                                                 beled solid line. <br />  <br />  <br />  <br />  <br /> sequences (Fig. 4). Mlc1p shares 35% identity verte-       Haploid spores contain mlc1D gene disruption  <br /> brate calmodulin, second yeast calmodulin         quire MLC1 plasmid (data shown).  <br /> yeast proteins (Davis Thorner, 1986). Mlc1p 35%          MLC1 haploinsufficient essential. <br /> identical chicken light chain (MLE1) (Matsuda et al.,         begin characterizing phenotype mlc1D cells, <br /> 1981). Mlc1p 31% identical Acanthamoeba myosin 1C         examined spores hemizygous diploid strain. light chain, unconventional myosin light chain          diploid requires plasmid copy MLC1 survive. (Wang et al., 1997).                                            spores receiving plasmid grow colonies <br />                                                                 carried genomic copy mlc1D. mlc1D <br /> MLC1 Essential Displays Haploinsufficiency               spores did inherit plasmid identified Initial attempts make diploid hemizygous MLC1          20% spores germinated did form unsuccessful. Southern blot analysis revealed         colonies. mlc1D cells grew elongated buds (Fig. 6). step gene replacement (Rothstein, 1991) resulted         Cytokinesis complete removal cell <br /> strains containing wild-type copies MLC1        wall treatment zymolyase did result separa- copy deletion construct. possible      tion buds mother cells. DAPI staining  <br /> explanation effect copy MLC1        vealed elongated cells multiple nuclei (Fig. 6 <br /> diploid genome insufficient viability. phenome-     C). 1% spores control strain <br /> non called haploinsufficiency (Wilkie, 1994). tested      formed elongated cells. MLC1 haploinsufficient repeating gene disrup- <br /> tion strain carrying copies MLC1 genome <br />                                                                 Mlc1p Interacts Neck Myo2p <br /> plus copy MLC1 plasmid. case,       lines evidence suggest Mlc1p light chain able delete copy MLC1 genome           Myo2p.  haploinsufficiency exhibited MLC1 scribed Materials Methods. hemizygous dip-       suppressed reduced copies MYO2 (Fig. 5). diploid <br /> loid requires plasmid carrying MLC1 viability (Fig. 5).   strain hemizygous MYO2 MLC1 viable. <br />  <br />  <br />  <br />  <br />                                                                                                 Figure 5. MLC1 displays <br />                                                                                                 haploinsufficiency.      Strains <br />                                                                                                 JGY46, RSY105, RSY107, <br />                                                                                                 RSY112, carrying <br />                                                                                                 MLC1 plasmid pRS289, <br />                                                                                                 streaked min- <br />                                                                                                 imal medium lacking <span id='am-78' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-79' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> <br />                                                                                                 minimal medium contain- <br />                                                                                                 ing 59-FOA  kill <br />                                                                                                 cells require plas- <br />                                                                                                 mid pRS289). plates                                                                                                 incubated 308C 3 d. <br />  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Myosin Light Chain Myo2p       717 <br />                                                                       Figure 7. <span id='am-12' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-13' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Glutathione</span>-S-transferase (GST)&#226;&#8364;&#8220;Myo2p fusion pro- <br />                                                                      teins. <br />  <br />  <br />  <br />                                                                         Second, Mlc1p binds directly IQ sites Myo2p                                                                      assayed gel overlay protocol. constructs used                                                                      experiment presented Fig. 7. 6XHis-Mlc1p <br />                                                                      binds specifically fusion proteins contain IQ <br />                                                                      sites (Fig. 8 B). control, calmodulin                                                                      binds specifically fusion proteins contain IQ sites <br />                                                                      (Fig. 8 C). calmodulin 6XHis-Mlc1p bind equally <br />                                                                      presence Ca21 EGTA (data shown). <br />                                                                      data demonstrate Mlc1p direct interac- <br />                                                                      tion IQ sites Myo2p. <br />  <br />                                                                      Toxicity Overexpressed MYO2 Suppressed                                                                      MLC1 CMD1 <br />                                                                      looked functional interaction Mlc1p                                                                      Myo2p. yeast strain carrying high-copy number plas- <br />                                                                      mid MYO2 grows 40% slower strains normal <br />                                                                      levels MYO2 (data shown). Microscopy  <br />                                                                      vealed 5% cells MYO2 overexpression strain <br /> Figure 6. mlc1D spores defects cell separation. Strains      abnormal bud necks multiple buds (data JGY46 RSY105(pRS289) sporulated tetrads            shown). similar defect high-copy number MYO4 <br /> dissected. 24 h, spores characterized classes: (1)   strains caused neck region Myo4p, germinated formed colonies, (2) germinated did           overexpression calmodulin does suppress effect <br /> form colonies, (3) did germinate. Examples class      (Haarer et al., 1994). possibility high copies shown  . 200 spores strain classified (B).      myosin light chains overcome defects associated Germinated spores RSY105(pRS289) stained              MYO2 overexpression. <br /> DAPI. Elongated spores DAPI staining shown (C). <br />                                                                         strain overexpressing MYO2 shows noticeable  <br /> Bars, 10 mm. <br />                                                                      duction growth rate (Fig. 9 . strain  <br />                                                                      expressing MYO2-D6IQ grows normally. Northern blot <br /> Suppression haploinsufficiency hemizygous                  analysis reveals MYO2 MYO2-D6IQ MYO2 demonstrates Mlc1p ordinarily                      transcribed similar steady-state levels present minimum level necessary cell growth.              strains fivefold normal levels (Fig. 9 B). <br /> MYO2-D6IQ suppress haploinsufficiency                Surprisingly, immunoblot strain overexpressing <br /> MLC1 (Fig. 5).  MYO2-D6IQ bypass                  MYO2 does higher levels Myo2p (Fig. 9, C <br /> requirement Mlc1p haploid strain (data                  D). fact, Myo2p actually decrease shown), essential func-          strain overexpressing MYO2. Overexpressing MYO2- <br /> tion Mlc1p yeast.                                              D6IQ results high levels Myo2-D6IQp (Fig. 9, C D). <br />  <br />  <br />  Journal Cell Biology, Volume 142, 1998                        718 <br />  Figure 8. Gel-overlay assays vitro binding Mlc1p calmodulin IQ sites Myo2p. Myo2p fusion proteins ex- <br /> pressed subjected SDS-PAGE described Materials Methods. Polyacrylamide gel stained Coomassie blue.  <br /> length fusion protein extract marked arrow  . Autoradiographs membranes incubated 35S-labeled 6XHis- <br /> Mlc1p (B) 35S-labeled calmodulin (C). gel overlay assays performed described Materials Methods. Lanes 1&#226;&#8364;&#8220;6 <br /> contain extracts IPTG-induced E. coli strain GM1 containing plasmids pGEX-2T, pSB20, pSB27, pSB21, pSB24, pSB25, respec- <br /> tively. Fig. 7 Table description fusion proteins. <br />  <br />  <br />  <br />   hypothesized high levels calmodulin                mal end head 20-nm neck likely corre- <br /> Mlc1p counteract toxicity overexpressed               sponds IQ sites (Cheney et al., 1993). region <br /> MYO2. Overexpressing CMD1 unable overcome                predicted helix neck class II myo- <br /> decreased growth rate (Fig. 9  low levels              sins. class II myosins, neck region stabilized Myo2p (Fig. 9 D) caused overexpressed MYO2.              binding myosin light chains (Rayment et al., 1993; <br /> trast, high-copy number plasmid carrying MLC1 able            Houdusse Cohen, 1996). Calmodulin previously fully suppress overexpression MYO2 defects           identified light chain chicken yeast <br /> sults strain high levels Myo2p (Fig. 9,     myosin V (Cheney et al., 1993; Brockerhoff et al., 1994). <br /> C). results indicate neck region MYO2          calmodulin does stabilize yeast myosin. responsible overexpression growth defect              present evidence MLC1 (YGL106w)  toxicity overcome high levels Mlc1p.        codes new light chain yeast class V myosin <br />                                                                    Myo2p. results argue Mlc1p binds neck  <br /> Mlc1p Stabilizes Myo2p                                             gion stabilizes Myo2p. high copies MLC1 able overcome                   overexpressed, Myo2p decreased half-life <br /> fects caused overexpression MYO2, tested              toxic yeast cells. Overproduction calmodulin <br /> hypothesis Mlc1p stabilizes Myo2p. Yeast cultures             does affect stability Myo2p, overproduction analyzed stability Myo2p following              Mlc1p stabilizes Myo2p ameliorates toxicity. <br /> Myo2p breakdown immunoblot analysis (Fig. 10).             requirement Mlc1p stabilize Myo2p bypassed <br /> wild-type strain, Myo2p half-life greater 8 h.       deleting IQ sites Myo2p.  presence strain carrying MYO2-D6IQ, Myo2-D6IQp               IQ sites destabilizes Myo2p unless sufficient Mlc1p <br /> half-life greater 8 h. neckless Myo2-D6IQp          present. stable wild-type Myo2p. overex-                   Removal IQ sites Myo2p causes slightly <br /> pressed, half-life Myo2p 2&#226;&#8364;&#8220;3 h. data          slower growth rate slightly larger cells. mutant Dic- <br /> explain Myo2p levels overexpression strain          tyostelium myosin II heavy chain DBLCBS, lacks increased. strain overexpressing MYO2             IQ sites, maintains 20% wild-type motor activity <br /> carrying high copies MLC1, half-life Myo2p 4 h.       measured sliding filament assay (Uyeda et al., <br /> High copies MLC1 able stabilize Myo2p,              1996). DBLCBS myosin allows cell division vivo suggesting Mlc1p light chain responsible stabilizing      demonstrates slightly slowed furrow formation cy- neck region Myo2p.                                          tokinesis (Zang et al., 1997).  Myo2-D6IQp                                                                    DBLCBS myosin confer relatively minor defects. <br />                                                                    results indicate efficiency myosin <br />                                                                    essential myosin function vivo. <br /> Discussion                                                            MLC1 essential haploinsufficient. Haploinsuffi- <br /> Electron micrographs chicken myosin V molecule              ciency rarely observed yeast genes. heads connected 30-nm stalk. proxi-           reasons reduction number gene copies  <br />  <br />  <br />  <br /> Stevens Davis M1c1p Myosin Light Chain Myo2p          719 <br />                                                                                                   Figure 9. High copies                                                                                                  MLC1, high copies                                                                                                  CMD1, overcome toxic <br />                                                                                                  effects overexpressed <br />                                                                                                  MYO2. Plasmids used                                                                                                  experiments listed <br />                                                                                                  Table  EMY55-5D                                                                                                  transformed plasmids <br />                                                                                                  encoding high copies                                                                                                  MYO2, MYO2-D6IQ, CMD1, <br />                                                                                                  MLC1. Plates incu- <br />                                                                                                  bated 3 d 308C  . <br />                                                                                                  CRY1 transformed                                                                                                  pRS43, pRS172, pRS31,                                                                                                  pRS221, mRNA col- <br />                                                                                                  lected log-phase cul- <br />                                                                                                  tures. 10 mg polyA  <br />                                                                                                  taining RNA samples                                                                                                  analyzed described Ma- <br />                                                                                                  terials Methods. Blots <br />                                                                                                  simultaneously hybrid- <br />                                                                                                  ized MYO2 TRX2 <br />                                                                                                  32 <br />                                                                                                    P-labeled probes (B). CRY1 <br />                                                                                                  transformed  <br />                                                                                                  dicated constructs study <br />                                                                                                  effects high copies                                                                                                  MLC1 (pRS290) strains <br />                                                                                                  extra copy                                                                                                  MYO2 (pRS43) ex- <br />                                                                                                  tra copy MYO2-D6IQ <br />                                                                                                  (pRS172), overexpressing <br />                                                                                                  MYO2 (pRS31) MYO2- <br />                                                                                                  D6IQ (pRS221) (C). CRY1 <br />                                                                                                  transformed  <br />                                                                                                  dicated constructs study <br />                                                                                                  effects high copies                                                                                                  CMD1 (pTD28) strains <br />                                                                                                  overexpressing MYO2 (pRS31) <br />                                                                                                  MYO2-D6IQ (pRS221) <br />                                                                                                  (D). Log-phase extracts                                                                                                  described Materi- <br />                                                                                                  als Methods. Samples <br />                                                                                                  assayed immunoblot <br />                                                                                                  analysis using anti-Myo2p <br />                                                                                                  antibody (Lillie Brown, <br />                                                                                                  1994) detect steady-state <br />                                                                                                  levels Myo2p. <br />  <br />  <br />  <br /> deleterious effects. cases, reduced gene copy            eliminated adding extra copies MLC1. Finally,  <br /> number affects regulatory genes working threshold           moval IQ binding sites Myo2p rescues hap- <br /> level (Wilkie, 1994). example form regulation      loinsufficiency exhibited MLC1. dosage-dependent sex determination Drosophila. Al-            Myo2-D6IQp does overcome requirement ternately, proteins produced minimum          Mlc1p cell viability. observation does address <br /> level proper function. ACT1 gene displays            essential nature interaction Mlc1p temperature-sensitive growth defects increased osmo-         Myo2p. merely demonstrates Mlc1p sensitivity present low levels (Shortle et al., 1984).   essential function does involve binding <br /> Finally, stoichiometry various protein components         neck Myo2p. myosins important. lines evidence suggest       yeast essential growth (Brown, 1997), suggesting <br /> fixed ratio Mlc1p Myo2p required           Mlc1p essential function unrelated myosins. <br /> fer normal growth.  diploid yeast strain carrying        Inviable mlc1D cells striking defect cytokinesis, copy MLC1 grow          resulting enlarged, multinucleate cells. phenotype copy wild-type MYO2. Second, growth defects           identical defects detected cells deleted IQG1, <br /> associated yeast strain overexpressing MYO2           encodes newly discovered IQGAP family member <br />  <br />  <br />  <br />  Journal Cell Biology, Volume 142, 1998                    720 <br />  Figure 10. Degradation Myo2p. Cycloheximide added final concentration 100 mg/ml log-phase cultures. Cultures allowed continue shaking, samples collected immunoblot analysis described Materials Methods. CRY1  <br /> taining plasmids (m), plasmids pRS31 (high copies MYO2) pRS290 (high copies MLC1) (j), pRS31 (high copies MYO2) pGF27 (vector  (d) examined. Bands immunoblots probed anti-Myo2p antibody quantified, values normalized loading using Coomassie blue&#226;&#8364;&#8220;stained band demonstrated high stability  . error bars rep- <br /> resent standard deviation (wild type),  high copies MYO2 high copies MLC1),  high cop- <br /> ies MYO2) independent experiments. immunoblots probed anti-Myo2p antibody demonstrate Myo2p stability CRY1 <br /> carrying pRS31 pRS290 (B) pRS31 pGF27 (C), CRY1 (MYO2 strain) (D) RSY21 (MYO2-D6IQ <br /> strain) (E). Numbers represent hours cycloheximide addition. <br />  <br />  <br />  <br />  yeast (Epp Chant, 1997; Lippincott Li, 1998).              conclusion, presented lines evidence <br /> Iqg1p contains IQ sites act binding sites        Myo2p light chains, calmodulin Mlc1p. propose essential function                newly characterized protein, Mlc1p. Mlc1p regulates Mlc1p act light chain IQGAP.                         stability Myo2p binding neck region.    Calmodulin additional light chain Myo2p.            characterization Mlc1p aid identification previously shown calmodulin Myo2p coim-               additional light chains chicken class V myosins. <br /> munoprecipitate yeast cell extracts interact            thank Susan Brown plasmid myo4D::URA3. <br /> vitro. mutation actin-binding site Myo2p           work supported National Institutes Health grant interferes Myo2p function dramatically worsens             GM40506 (T.N. Davis). R.C. Stevens supported Public Health Ser- phenotype conferred mutations calmodulin                  vices National Research Service Award T32 GM07270. <br /> (Brockerhoff et al., 1994). localiza-         Received publication 18 September 1997 revised form 15 June <br /> tion calmodulin sites cell growth predominately         1998. <br /> dependent IQ sites Myo2p small  <br /> tribution yeast class V myosin Myo4p.                                                                     References <br /> MYO2-D6IQ strain lacking MYO4, calmodulin localizes spindle pole body fast moving patches             Bobola, N., R.P. Jansen, T.H. Shin, K. Nasmyth. 1996. Asymmetric accu- <br />                                                                       mulation Ash1p postanaphase nuclei depends myosin restricts <br /> cell surface. protein target binds calmodulin             yeast mating-type switching mother cells. Cell. 84:699&#226;&#8364;&#8220;709. fast moving patches identified.             Brockerhoff, S.E., T.N. Davis. 1992. Calmodulin concentrates regions IQGAP protein yeast localizes actin ring             cell growth Saccharomyces cerevisiae. J. Cell Biol. 118:619&#226;&#8364;&#8220;629. <br />                                                                     Brockerhoff, S.E., C.G. Edmonds, T.N. Davis. 1992. Structural analysis bud neck cytokinesis (Epp Chant, 1997; Lip-                wild-type mutant yeast calmodulins limited proteolysis electro- <br /> pincott Li, 1998). calmodulin localization             spray ionization mass spectrometry. Protein Sci. 1:504&#226;&#8364;&#8220;516. <br />                                                                     Brockerhoff, S.E., R.C. Stevens, T.N. Davis. 1994. unconventional  bud neck cytokinesis depends Myo2p,              osin, Myo2p, calmodulin target sites cell growth Saccharomyces <br /> calmodulin does bind yeast IQGAP calmodulin                 cerevisiae. J. Cell Biol. 124:315&#226;&#8364;&#8220;323. <br /> binds IQGAP levels detection.                              Brown, S.S. 1997. Myosins yeast. Curr. Opin. Cell Biol. 9:44&#226;&#8364;&#8220;48. <br />                                                                     Cheney, R.E., M.S. Mooseker. 1992. Unconventional myosins. Curr. Opin. <br />    noncalmodulin unconventional myosin                 Cell Biol. 4:27&#226;&#8364;&#8220;35. <br /> light chain characterized. Biochemical studies             Cheney, R.E., M.K. O&#226;&#8364;&#8482;Shea, J.E. Heuser, M.V. Coelho, J.S. Wolenski, E.M. <br /> identified MICLC light chain Acanthamoeba myo-               Espreafico, P. Forscher, R.E. Larson, M.S. Mooseker. 1993. Brain myo- <br />                                                                       sin-V headed unconventional myosin motor activity. Cell. 75: <br /> sin IC (Wang et al., 1997). Mlc1p MICLC share 31%                 13&#226;&#8364;&#8220;23. <br /> sequence identity, proteins share significant              Coulondre, C., J.H. Miller. 1977. Genetic studies lac repressor: IV. <br />                                                                       Mutagenic specificity lacI gene Escherichia coli. J. Mol. Biol. 117: <br /> identity calmodulin/EF-hand superfamily mem-               577&#226;&#8364;&#8220;606. <br /> bers. chicken class V myosin light           Davis, T.N. 1992. temperature-sensitive calmodulin mutant loses viability <br /> chains, calmodulin additional uncharacterized                 mitosis. J. Cell Biol. 118:607&#226;&#8364;&#8220;617. <br />                                                                     Davis, T.N., J. Thorner. 1986. Calmodulin calcium-binding pro- <br /> small proteins copurify myosin (Cheney et al.,          teins yeast. Yeast Cell Biology. Vol. 33. J.B. Hicks, e <br /> </body></html>