<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Product Saccharomyces cerevisiae RSS1 <br /> Gene, Identified High-Copy Suppressor rat7-1 Temperature-sensitive Allele RAT7INUP159 Nucleoporin, Required Efficient <br /> mRNA Export <br />  <br />  <br />         RAT7/NUP159 identified previously screen genes products         important nucleocytoplasmic export poly + RNA encodes essential <br />         nucleoporin. report identification RSS1 (Rat Seven Suppressor)         high-copy extragenic suppressor rat7-1 temperature-sensitive allele. Rsslp encodes <br />         novel essential protein 538 amino acids, contains extended predicted <br />         coiled-coil domain located nuclear pore complexes (NPCs)         cytoplasm. RSS1 reported high-copy extragenic suppressor mutant <br />         nucleoporin. Overexpression Rsslp partially suppresses defects nucleocytoplas- <br />         mic export poly + RNA, rRNA synthesis processing, nucleolar morphology <br />         seen rat7-1 cells shifted nonpermissive temperature 37&#194;&#176;C   restores <br />         processes levels adequate growth rate approximately half         wild-type cells. shift 37&#194;&#176;C, mutant Rat7-lp/Nupl59-lp lost         nuclear rim rat7-1 cells NPCs, clustered cells grown <br />         permissive conditions substantially clustered. Overexpression         Rsslp did result retention mutant Rat7-lp/Nupl59-lp NPCs, did <br />         result partial maintenance NPC-clustering phenotype seen mutant cells. <br />         Depletion Rsslp placing RSS1 open reading frame (ORF) control         GALl promoter led cessation growth nuclear accumulation poly + RNA <br />         affecting nuclear protein import nuclear pore complex distribution, suggest- <br />         ing RSS1 directly involved mRNA export. rat7-1 cells cells <br />         depleted Rsslp defective mRNA export, data consistent gene <br />         products playing essential roles process mRNA export suggest Rsslp <br />         overexpression suppresses growth defect rat7-1 cells 37&#194;&#176;C acting maintain <br />         mRNA export. <br />  <br />  <br /> INTRODUCTION                                              cells. transcription post-transcriptional RNA exchange macromolecules nucleus        processing, mRNAs transported cytoplasm cytoplasm essential process eukaryotic   nuclear pore complexes function <br />                                                           protein synthesis. Initial approaches investigate <br />                                                           RNA export involved microinjection RNA-coated <br />     ~Corresponding author.                                gold particles nucleus Xenopus oocytes, <br />  1996 American Society Cell Biology                                                              1601 <br />  V. Del Priore et al. <br />  showed poly + RNA, tRNA, 5S                 basis primary sequence, fall rRNA exported efficiently saturable          broad classes. class consists proteins manner (Dworetzky Feldherr, 1988). Nucleocyto-           multiple repeats short motifs include GLFG, <br /> plasmic export Balbiani ring granules, abundant           XFXFG, XXFG. repeats interact specific premessenger ribonucleoprotein particles        multiprotein complex contains karyophilic <br /> (RNP) produced salivary glands Chironomus          protein imported nucleus het- <br /> tentans, shown electron microscopy            erodimeric receptor recognizes nuclear localiza- <br /> mography proceed central channel           tion sequences (NLSs). receptor consists Srplp NPC (Mehlin et al., 1992). RNP particles           Kap95p (Gorlich et al., 1995; lovine et al., 1995; <br /> change globular linear conformation         Moroianu et al., 1995; Rexach Blobel, 1995). begin translocate NPCs, 5&apos;       second class nucleoporins consists end RNP particle moving NPC         transmembrane domains thought anchor <br /> (Mehlin et al., 1992). proteins mediate         pore nuclear envelope (Wozniak et al., 1989, <br /> RNA transport identified yeast.          1994; Wozniak Blobel, 1992; Greber et al., 1990). proteins, called Npl3p, closely related mam-     class consists proteins lack <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> <br /> malian hnRNP proteins known shuttle             repeats transmembrane proteins. tween nucleus cytoplasm (Flach et al.,           NPC proteins contain potential coiled-coil domains <br /> 1994). Despite findings, lack vitro   provide basis interactions NPC study RNA export difficult          polypeptides. <br /> gain complete understanding process            screen temperature-sensitive mutants <br /> identify gene products process requires.     S. cerevisiae accumulate poly + RNA   Nuclear pore complexes (NPCs) form chan-          nuclei shift nonpermissive temperature, <br /> nels nucleus cytoplasm embed-            isolated mutant alleles RAT2/NUP120, RAT3/ <br /> ded nuclear envelope places inner        NUP133, RAT7/NUP159, RAT9/NUP85, NUP145 outer nuclear membranes fused. com-            (Gorsch et al., 1995; Heath et al., 1995; Li et al., 1995; <br /> plex structures, mass -125 mDa metazoan         Goldstein et al., 1996; Dockendorff Cole, unpub- <br /> cells 65 mDa Saccharomyces cerevisiae,            lished results). cells carrying rat7-1 allele thought contain 50 different polypeptide         RAT7/NUP159, poly + RNA accumulates rap- <br /> species  review, Rout Wente, 1994; Davis,            idly nuclei shift nonpermissive <br /> 1995; Doye Hurt, 1995). high-resolution             temperature, apparent defect nuclear <br /> electron microscopy image reconstruction, NPCs           protein import. rat7-1 mutant cells clustering thought consist central spoke/ring assembly          NPCs 23&#194;&#176;C defects nucleolar func- anchored nuclear envelope transmem-        tion structure 37&#194;&#176;C (Gorsch et al., 1995; Heath et <br /> brane protein components NPC, central trans-        al., 1995; Kraemer et al., 1995). rat7-1 cells porter substrates             shifted 37&#194;&#176;C, Rat7-lp lost NPCs, actively translocated nucleus,          pores rapidly clustered (Gorsch et al., <br /> basket-like structure located nucleoplasmic face      1995). immunoelectron microscopy, Rat7p/Nupl59p NPC, fibrils emanate cyto-          localized position cytoplasmic <br /> plasmic face NPC (Maul, 1977; Unwin               periphery NPC, component Milligan, 1982; Richardson et al., 1988; Allen           cytoplasmic fibrils (Kraemer et al., 1995). <br /> Douglas, 1989; Akey, 1990, 1991, 1995; Akey Ra-             conducted screen high-copy suppressors dermacher, 1993; Reichelt et al., 1990; Jamik Aebi,      ts growth defect rat7-1 strain. RAT7/ <br /> 1991; Goldberg Allen, 1992; Hinshaw et al., 1992;        NUP159 gene plasmids encoding carboxyl <br /> Ris Malecki, 1993). NPCs possess 9-nm channels           portion Rat7p/Nupl59p isolated repeatedly. solutes, ions, metabolites, proteins       frequently isolated extragenic suppressor <br /> smaller 50 kDa passive diffusion.           named RSS1 (Rat Seven Suppressor). Rsslp <br /> transport larger macromolecules, including pro-           encodes novel protein 538 amino acids, teins ribonucleoproteins, thought use          contains extended predicted coiled-coil domain, <br /> transporter central channel NPC            located NPCs cytoplasm. requires energy specific signals         presence high-copy RSS1 permitted rat7-1 strain <br /> transported macromolecule  review, Elliot et al.,        grow 37&#194;&#176;C does absence 1994; Fabre Hurt, 1994; Davis, 1995; Izaurralde          suppressor 23&#194;&#176;C, did restore wild-type Mattaj, 1995).                                           growth properties rat7-1 cells. RSS1 partially sup- <br />    Approximately 20 yeast nucleoporins smaller         pressed nucleolar defects RNA export <br /> number metazoan cells identified           block rat7-1 cells, mutant Rat7-lp/ <br /> biochemical, immunochemical, genetic ap-                 Nupl59-lp lost nuclear pores cells <br /> proaches  review, Doye Hurt, 1995).           shifted nonpermissive temperature. Inter- <br />  <br /> 1602                                                                                         Molecular Biology Cell <br />                                                                                RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> estingly, presence high-copy RSS1, NPCs                     duplicate incubated 23&#194;&#176;C 37&#194;&#176;C. Duplicate <br /> remained partially clustered rat7-1 cells shifted               samples removed culture various times                                                                       30 h, cell number determined counting. nonpermissive temperature. RSS1 open <br /> reading frame (ORF) placed control GALl promoter, cells able grow                  Isolation Suppressors Identification RSS1 <br /> GALl promoter active. Rsslp depleted transferring cells glucose-containing                  isolate high-copy suppressors rat7-1 allele, grew cells <br />                                                                       (LGY101) optical density (OD) 1.4 (mid-log phase) dia, cells developed defect                transformed electroporation using yeast genomic <br /> export poly + RNA 9 h did                    library present high-copy plasmid YEpl3 (Nasmyth defect nuclear protein import, suggesting                   Tatchell, 1980). Transformants plated SC-Leu plates Rsslp plays primary role RNA export.                             incubated restrictive temperature 37&#194;&#176;C. Colonies ap- <br />                                                                       peared contained putative high-copy suppressors. confirm                                                                       suppression gene present library plasmid <br /> MATERIALS METHODS                                                 secondary mutation strain genome, retrieved <br />                                                                       plasmid individual strains grew 37&apos;C tested Yeast Strains, Cell Culture, Genetic Methods                      ability restore growth 37&#194;&#176;C rat7-1 cells (LGY101). <br /> Yeast strains used study listed Table 1,       Plasmids digested various restriction endonucleases plasmids used listed Table 2. Strains cultured stan-   identify contained overlapping DNA fragments. dard methods (Sherman, 1991), using rich (YPD) media defined       majority plasmids identified contained RAT7/NUP159, <br /> media, synthetic complete (SC), lacking appropriate amino acids   fragments RAT7/NUP159, region chromosome IV  nucleotides. Wild-type strain FY86 obtained Fred Win-     taining RSS1 gene, subject report. <br /> ston (Harvard Medical School, Boston, MA). Genetic techniques,           RSS1 cloned retrieving plasmid containing library <br /> including matings, sporulations, plasmid transformations     fragment suppressor gene rat7-1 mutant strain. <br /> electroporation, performed standard methods (Rose et al.,     ends library fragment sequenced ABI373 <br /> 1989; Guthrie Fink, 1991).                                        automated sequencer. identified region yeast chromo- <br />   temperature-shift experiments, cells cultured liquid media   IV sequenced yeast genome room temperature shifted 37&#194;&#176;C water bath, incu-      sequencing project contained multiple open reading frames. <br /> bation continued indicated figure legends. prepare   <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> sequence portion chromosome IV growth curves, single colonies wild-type, mutant, suppressed    determined Applied Biosystems 373 (Foster City, CA) auto- <br /> yeast cells inoculated 5 ml SC-Leu allowed grow   mated sequencer used fluorescent dye terminators cycle <br /> overnight room temperature. Cell density determined     sequencing. sequence DNA strands determined. <br /> hemocytometer. Duplicate cultures strain diluted      Results set sequencing reactions used design 1 2 x 106 cells/ml SC-Leu recounted.           primers permit extension sequence. <br />  <br />  <br />  <br /> Table 1. Yeast strains <br /> Strain         Genotype                                                             Comments <br /> FY86           MATa ura3-52 his3A200 leu2Al                                         Wild-type; derived S288C; (Winston et al., <br />                                                                                       1995) <br /> LGY101         MATa ura3-52 his3A200 leu2A1 rat7-1ts                                Segregant 3rd backcross (Gorsch et al., 1995) <br /> VDPY102        MATa ura3-52 his3A200 leu2Al rat7-1ts (pVDP2: LEU2 RSS1 2,4          pVDP2 isolated high-copy suppressor rat7-1 <br /> VDPY107        MATa ura3-52 his3A200 leu2AI rat7-lts (pVDP7: LEU2 RSS1 24*          pVDP7: RSS1 ORF subcloned YEpLacl81 <br /> ACY1           MATa/MATa ura3-52/ura3-52 leu2A/leu2Al his3A200/his3A200             provided Dr. Anita Corbett, Harvard Medical <br />                  trpIA63ITRP1                                                         School, Boston, MA <br /> VDPY108        MATa/MATa ura3-52/ura3-52 leu2A1/leu2Al his3A200/his3A200            heterozygous diploid rssl null strain <br />                  trplA63ITRP1 RSS1/RSS1::HIS3 <br /> VDPY109        MATa/MATa ura3-52/ura3-52 leu2A/1leu2A1 his3A200/his3A200            heterozygous diploid rssl null strain harboring <br />                  trplA63/TRP1 RSSI/RSS1::HIS3 lpVDP9: URA3 RSS1 CEN)                  RSS1 plasmid <br /> VDPY111        MATa ura3-52 leu2Al his3A200 trplA63 RSS1::HIS3 IpVDP9:              haploid rssl null resulting VDPY109 <br />                  URA3 RSSI CENI                                                       sporulation <br /> VDPY112        MATa ura3-52 leu2Al his3A200 trplA63 RSS1::HIS3 IpVDP12:             haploid rssl null strain harboring RSSlmyc- <br />                  LEU2 RSSImyc CEN)                                                    containing plasmid <br /> VDPY113        MATa ura3-52 leu2Al his3A200 trplA63 RSS1::HIS3 lpVDP13:             haploid rssl null strain harboring RSSlmyc 2At <br />                  LEU2 RSSlmyc 2p1                                                     plasmid <br /> VDPY114        MATa ura3-52 leu2Al his3A200 trplA63 RSS1::HIS3 IpVDP14:             haploid rssl null strain harboring GAL1::RSS1- <br />                  LEU2 GAL::RSS1 CEN)                                                  containing plasmid <br /> VDPY115        MATa ura3-52 leu2Al his3A200 trplA63 RSS1::HIS3 IpVDP15:             haploid rssl null strain harboring GALI::Rsslmyc- <br />                  LEU2 GAL::RSS1M CEN)                                                 containing plasmid <br /> OLY101         MATa his3A200 leu2,1 ura3-52 rat3-1/nupl33-1                         Li et al., 1995 <br /> CCY282         MATa trplA63 ura3-52 leu2Al rat2-1/nupl20-1                          Heath et al., 1995 <br /> AGY401         MATa ura3-52 leu2A<span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>;l trplA63 rat9-1/nup85-1                          Goldstein et al., 1996 <br /> AGY48          MATa his3A&amp;200 leu2A1 ura3-52 nupl45-10                              Dockendorff Cole, unpublished data <br />  <br />  <br />  <br /> Vol. 7, October 1996                                                                                                                 1603 <br />  V. Del Priore et al. <br />  <br />                                                                          plasmid-bome wild-type copy RSS1 eliminated plating <br /> Table 2. Plasmids                                                        cells 5-FOA. obtain GALI::RSS1myc construct, fragment <br />                                                                          RSS1 ORF containing &quot;myc&quot; epitope carboxy ter- <br /> Plasmids       Markers                 Comments                          minus subcloned similar procedures. <br /> pVDP2          LEU2    2t Ampr         YEp13 containing library <br />                                                                          Situ Hybridization Immunofluorescence <br />                                          fragment RSS1 <br /> pVDP7          LEU2 2y Ampr            RSS1 subcloned               Assays <br />                                          YEplacl8l                       situ hybridization assays poly + RNA localization pVDP8          LEU2 CEN4 Ampr          RSS1 subcloned               indirect immunofluorescence   techniques protein localization <br />                                          YCpLaclll                       described previously (Amberg et al., 1992; Copeland pVDP9          URA3 CEN4 Ampr          RSS1 subcloned               Snyder, 1993). Antibody Rat7p/Nupl59p raised guinea <br />                                          YCpLac33                        pigs (Gorsch et al., 1995) used 1:3000 dilution. FITC-conju- <br /> pVDP12         LEU2 CEN4 Ampr          myc-tagged RSS1 subcloned         gated goat anti-guinea pig immunoglobulin G (IgG; Vector Labora- <br />                                          YCpLaclll                  tories, Burlingame, CA) used 1:250 dilution. RL-1 monoclo- <br /> pVDP13         LEU2 2t Ampr            myc-tagged RSS1 subcloned         nal antibody  gift Larry Gerace, Scripps Research Institute, <br />                                          YEpLacl81                  La Jolla, CA; Snow et al., 1987) used dilution 1:200. <br /> pVDP14         LEU2 CEN4 Ampr          RSS1 ORF subcloned           FITC-conjugated goat anti-mouse IgM (Vector Laboratories)                                          YCpGALl                         used dilution 1:600. Monoclonal antibody 2.3B, recog- <br /> pVDP15         LEU2 CEN4 Ampr          myc-tagged RSS1 ORF               nizes nucleolar Nsrlp yeast  gift M. Snyder, Yale Univer- <br />                                          subcloned YCpGALl          sity, New Haven, CT), used dilution 1:10. FITC-conju- <br />                                                                          gated goat anti-mouse IgG (Vector Laboratories) used                                                                          dilution 1:600. <br />                                                                             fluorescence microscopy, images obtained using   ORF responsible suppression rat7-1 mutation         cooled CCD (charge-coupled device) camera. Identical exposure <br /> identified deleting various ORFs contained library plas-       conditions used comparable images figure; <br /> mid, transforming derived plasmids rat7-1 cells,        fluorescein isothiocyanate (FITC) images single <br /> plating 37&#194;&#176;C. RSS1 ORF flanking regulatory regions            figure obtained using exposure conditions, sufficient suppression rat7-1 mutation subcloned    4&apos;,6-diamidino-2-phenylindole (DAPI) images, YepLacl81 (Gietz Sugino, 1988).                                      exposure conditions FITC images DAPI images                                                                           Composites prepared Adobe Photoshop <br />                                                                          altering images. <br /> Disruption RSS1 polymerase chain reaction (PCR)-based gene deletion approach           Analysis Pre-rRNA Processing Pulse Labeling used delete RSS1 (Baudin et al., 1993). PCR amplification, cassette generated contained yeast HIS3 gene flanked      RNA extracted method Tollervey (Snow et al., 1987), ends 45 nucleotides identical sequences just upstream       incorporation [3HI-<span id='am-33' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uridine</span> rRNA precursors downstream RSS1 coding region. oligonucleo-          monitored described previously (Heath et al., 1995). <br /> tides used PCR, 17 nucleotides identical sequences ends HIS3 selectable gene. oligo sequences following: 5&apos; end, 5&apos;-CGAAGAATGAGATTTGTGTTCGAT-                  Epitope Tagging Immunolocalization <br /> GAGGTTTTCAATTCAGATACTGGCCTCCTCTAGTACACTC-3&apos;;                             Site-directed mutagenesis used create unique restriction 3&apos; end, 5&apos;-GTGCATATATAAGTTCAGAATTTTCTA-                      endonuclease site near C terminus RSS1 gene AGGAGACATTTCCGGAAAGCGCGCCTCGTTCAGAATG-3&apos;.                             sequence encoding copies epitope recognized used plasmid pBM2815 (obtained P. Silver, Harvard Medical           9E10 anti-myc monoclonal antibody inserted frame School, Boston, MA) containing HIS3 gene template        RSS1 coding region. triple myc tag inserted PCR reaction. PCR conditions 30 cycles denaturation        BamHI site created <span id='am-21' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> 529 (total 538 amino acids) 94&#194;&#176;C 1.5 min, annealing 50&#194;&#176;C 2.0 min, extension     using site-directed mutagenesis kit (Clonetech Laboratories, Palo <br /> 72&#194;&#176;C 2.0 min. PCR product transformed wild-           Alto, CA). plasmid containing myc-tagged RSS1 (pVDP12 <br /> type diploid strain (ACY1), homozygous his3A200. flanking        pVDP13) transformed strain VDPY111 carrying upstream downstream sequences targeted deletion              disruption RSS1 plasmid containing wild-type copy struct RSS1 locus RSSI ORF replaced        RSS1 (pVDP9). selection SC-Leu, plasmid containing <br /> HIS3 homologous recombination. Recombinants selected             wild-type RSS1 gene eliminated plating 5-FOA. plating transformed cells SC plates. Homologous            Cells carrying myc-tagged RSS1 allele (VDPY112, VDPY113) <br /> recombination locus verified PCR Southern          subjected indirect immunofluorescence. primary anti- <br /> analysis.                                                                body used 9E10 mouse IgG monoclonal anti-myc tissue <br />    determine RSS1 essential vegetative growth,      culture hybridoma cells  gift Drs. J. Zhu J.M. Bishop, <br /> transformed strain heterozygous RSS1 (VDPY108)              University California, San Francisco, CA) dilution 1:1. pVDP9, contains WT RSS1 YCplac33 (ura+). transfor-          secondary antibody used FITC-conjugated goat anti-mouse IgG <br /> mants (VDPY109) sporulated, tetrads dissected,         (Vector Laboratories) dilution 1:100. spore tetrads scored auxotrophic markers          examine colocalization Rsslpmy,C NPC antigens, ability grow 5-FOA plates.                                     stained cells simultaneously RL-1 anti-nucleoporin anti- <br />                                                                          body (1:200 dilution) 9E10 anti-myc monoclonal antibody <br />                                                                          (1:1 dilution). RL-1 IgM, 9E10 IgG. used FITC- <br /> GAL1:.RSS1 GAL1::RSSlmyc Constructs                                  conjugated goat anti-mouse IgG (1:100 dilution) visualize PstI-HindIII fragment containing RSS1 ORF subcloned            sites Rsslpmyc located used Texas Red-conjugated BamHI-HindIII sites YCpGAL1 vector (provided          goat anti-mouse IgM (1:600 dilution) visualize RL-1 anti- <br /> Anita Corbett, Harvard Medical School, Boston, MA). resulting        gens located. FITC-conjugated secondary antibodies <br /> plasmid (pVDP14) transformed VDPY111 strain,            left  signal Texas Red secondary <br />  <br />  <br />   1604                                                                                                          Molecular Biology Cell <br />                                                                                     RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> antibody FITC channel; similarly, Texas Red  <br /> jugated secondary antibodies left  signal     Table 3. Suppressor screening FITC-conjugated secondary antibody Texas Red channel. <br />                                                                                                                                 Number <br />                                                                                                                                 times <br /> Western Analysis                                                         High-copy suppressors                                  isolated <br /> Cells grown total OD600 0.75. Cells pelleted centrifugation 2500 rpm 2.5 min washed water.       Extragenic <br /> Acid-washed glass beads (200 mg) 200 ,ul hot sample buffer          Suppressor 9/RSS1a                                       4 <br /> (62.5 mM Tris-Cl, pH 6.8, 2% SDS, 10% glycerol, 8 M <span id='am-38' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span>, 0.72 M           Suppressor33 <br /> 2-mercaptoethanol, 0.05% bromophenol blue) added                 Suppressor4                                            1 cell pellets. brief vortexing, cells transferred         Suppressor8                                            1 <br /> 100&#194;&#176;C water bath 3 min. Cells lysed follows: 10 s    Intragenic <br /> vortexing 50 s incubation 100&#194;&#176;C water bath, repeated      RAT7/NUP159                                             22 <br /> times 5 min. supernatant transferred clean        C-terminal domain RAT7/NUP159                         6 <br /> Eppendorf tube, equal volumes cell lysate loaded        Revertants                                                 3 10% SDS-polyacrylamide gel. gel run 200 V -50           Total                                                     38 <br /> min. Proteins gel transferred polyvinylidene di- <br /> fluoride (PVDF) membrane electroblotting overnight 100 mA <br />                                                                                                                                                       Rat Seven Suppressor. cold room. membrane briefly washed 1 x PBS, <br /> 0.1% Tween 20 solution (solution  blocked 1 h 1 x PBS, 0.1% Tween 20, 5% nonfat milk solution (solution B). membrane incubated 9E10 anti-myc monoclonal                suggests domain probably plays critical role <br /> antibody 1:10 dilution solution B 2 h. washes 10     function Rat7p/Nupl59p, studies <br /> min solution  membrane                possibility progress. addition incubated anti-mouse antibody coupled enzyme              intragenic suppressors, isolated number extra- <br /> ersham, Arlington Heights,  diluted 1:3000 solution B 1 h. <br /> Washes performed  . Signal devel-           genic suppressors. independently <br /> oped treating membrane enhanced chemilumines-            identified times, gene RSS1 (Rat <br /> cence (ECL) kit (Amersham). control loading, membranes          Seven Suppressor 1). frequency incubated polyclonal rabbit anti-Secl3p antibody (kindly       RSS1 isolated, selected provided Dr. Charles Barlowe, Dartmouth Medical School, <br /> Hanover, NH) 1:2000 dilution anti-rabbit antibody coupled       study. horseradish peroxidase (Amersham) 1:10,000 dilution.               identify RSS1 gene, plasmid containing <br />                                                                          extragenic suppressor retrieved Electron Microscopy                                                      rat7-1 strain, DNA sequence obtained                                                                          ends library fragment. sequencing <br /> S. cerevisiae examined electron microscopy follow-      data positioned RSS1 chromosome IV ing previously described protocols: Byers Goetsch (1975), <br /> Wright Rine (1989), Goldstein et al. (1996). Sections cut   NHP2 HEM3. suppressing plasmid contained Sorvall MT5000 (RMC, Tucson, AZ) ultramicrotome              new ORFs addition fragments NHP2 <br /> section thickness 90 nm examined JEOL 100CX (Pea-            HEM3 (Figure 1A). determine body, MA) electron microscope 80 kV.                                  ORFs corresponded RSS1, deleted putative <br />                                                                          ORF individually, religated library vector RESULTS                                                                  transformed rat7-1 cells, cells plated                                                                          37&#194;&#176;C. ORFI deleted, growth defect Identification RSS1 Screen High-Copy                         37&#194;&#176;C rat7-1 cells suppressed, indicating <br /> Extragenic Suppressors rat7-1 ts Allele                           ORF likely RSS1. confirm, sub- identify proteins interact Rat7p/                     cloned ORFi flanking regulatory signals Nupl59p, performed high-copy extragenic sup-                        YepLacl81  produce pVDP7), pressor screen rat7-1 ts allele. rat7-1 strain                suppress growth defect rat7-1 cells 37&#194;&#176;C. <br /> (LGY101) grown mid-log phase trans-                             RSS1 encodes novel protein 538 amino acids <br /> formed electroporation yeast genomic li-                       predicted molecular weight 62 kDa pI <br /> brary present 2,u-based, high-copy vector.                      9.03. sequence protein shown Figure <br /> transformants plated 37&#194;&#176;C select                    1B. extended segment Rsslp (amino acids 124- acquired ability grow nonpermissive                   255) predicted (Lupas et al., 1991) temperature presence library plas-                   high potential (p &gt; 99.9%) form coiled-coil struc- <br /> mid. 30,000 colonies screened 38                      ture (Figure 1C). obvious structural features <br /> putative suppressors isolated. results summa-                    motifs  Rsslp shows homology rized Table 3. isolated plasmids containing                    proteins outside potential coiled-coil  <br /> complete RAT7/NUP159 gene times,                      gion. <br /> DNA fragments encoding C-terminal portion                           determine RSS1 essential gene, Rat7p/Nupl59p. fact C-terminal end                      replaced RSS1 coding region HIS3 gene. Rat7p/Nupl59p suppress rat7-1 ts allele                       Haploid cells containing disrupted copy RSS1 <br />  <br /> Vol. 7, October 1996                                                                                                                1605 <br />  V. Del Priore et al. <br />                     x     <br />  <br />         -. <br />                             Nri IrI                                                        1               I1 <br />  <br /> NHP2                          ORF1                                   ORF2               ORF3 <br />                               RSS1                                                           HEM3 <br />  <br /> B <br />    1         MRFVFDEVFN            SDTDSPEFEE   TCSTTSSTSS QCPTPEPSPA            IKLPSFTKVG                 TKKLVNESVV <br />   61         ILDPALENAL            RDLNLQSKLI   PINEPIVAAS SIIVPHSTNM            PLPRASHSSL                 LDNAKNSNAT <br />  121         APLLEAIEES            FQRKMQNLVL   ANQKEIQSIR ENKRRVEEQR            KRKEEEERKR                 KEAEEKAKRE <br />  181         QELLRQKKDE            EERKRKEAEA   KLAQQKQEEE RKKIEEQNEK            ERQLKKEHEA                 KLLQQKDKLG <br />  241         KAVTNFDKIS            KMFWHYKDKI   AQIKQDIVLP IKKADVNVRN            LLSRHKRKIN                 PKFGQLTNSN <br />  301         QQLFKIQNEL            TQLINDTKGD   SLAYHWILNF IAKAVVHQAE            TEVRVKPESA                 LPLGKLTLYL <br />  361         LVQFPELQEL            FMARLVKKCP   FVIGFTCEID TEKGRQNMGW            KRNNENKWED                 NTSYDERMGG <br />  421         ILSLFAIITR            LQLPQEFITT   TSHPFPIALS WHILARICNT            PLNLITNTHF                 VILGSWWDAA <br />  481         AVQFLQAYGN            QASKLLILIG   EELTSRMAEK KYVGAARLRI            LLEAWQNNNM                 ESFPEMSP <br />  <br />  <br />  <br /> C <br />  <br />  <br />  <br />                                Coiled-coil probability Rss1 p <br />               1     - <br />  <br />  <br />  <br />  <br />              0.9    - <br />  <br />  <br />              0.8    - <br />  <br />  <br />              0.7    - <br />  <br />  <br />  <br />         ,&quot;   0.6.- <br />  <br />  <br />     - <br />              0.5 <br />                                                                                                                          Figure 1. S. cerevisiae RSS1 gene.   Sche- <br />     , 0.4                                                                                                                matic representation region yeast chro- <br />                                                                                                                          mosome IV RSS1 located. Important <br />              0.3 <br />                                                                                                                          restriction endonudease digestion sites              0.2                                                                                                         shown. arrows putative ORFs                                                                                                                          region. Hindm sites indicated              0.1                                                                                                         used subclone RSS1. (B) Predicted amino acid <br />                0* <br />                                                                                                                          sequence Rsslp, contains 538 amino <br />                                                                                                                          acids. (C) Coiled-coil probability Rsslp;                             N Lf   M            0   N <br />                                                     N <br />                                                         u <br />                                                             N_ <br />                                                                                                                                   N <br />                                                                      0 <br />                                                                          iaao   r-..v   0 <br />  <br />                                                                                                                                                                                                        oun                PEPCOIL program used search Rsslp                                                     Amino Acid #                                                         potential coiled-coil regions (Lupas et al., 1991). <br />  <br />   able grow absence plasmid                                                      determine suppression rat7-1 containing RSS1 gene, indicating RSS1                                                     high-copy RSSI specific rat7-1 mutant <br /> essential growth. transformed pVDP7                                                       allele, introduced pVDP7 strains carrying <br />  Yeplacl81 vector containing RSS1) strain                                                  rat2-1/nup120-1, rat3-1/nupI33-1, rat9-1/nup85-1, <br /> containing disruption RAT7/NUP159                                                        nupl45-10 ts mutant alleles. transformants <br /> taining wild-type RAT7/NUP159 gene                                                           plated 23 37&#194;&#176;C. 23&#194;&#176;C, URA3/CEN plasmid. cells unable grow 5-FOA, indicating overexpression                                                                                                       growth of strains affected RSS1 bypass need Rat7p/Nupl59p.                                                        RSS1 suppressor.  growth detected RSS1 gene cloned CEN plas-                                                        strains restrictive temperature mid, longer able suppress rat7-1                                                     presence high-copy RSS1. data indicate <br /> growth defect 37&#194;&#176;C, indicating overexpres-                                                    RSS1 general suppressor ts alleles sion RSS1 required suppression.                                                             yeast nucleoporins. <br />  <br /> 1606                                                                                                                                        Molecular Biology Cell <br />                                                                               RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> Partial Suppression rat7-1 Defects                                rat7-1 cells accumulate poly + RNA nuclei <br /> High-Copy RSS1                                                        rapidly shift restrictive temperature. <br /> rat7-1 cells rapidly stop growing shifted                 15 min 37&#194;&#176;C, 100% cells dramat- <br /> nonpermissive temperature 37&#194;&#176;C (Gorsch et al.,                     ically enhanced nuclear accumulation poly + <br /> 1995). determine RSS1 high copy sup-                   RNA (Gorsch et al., 1995). examined effect pressed growth defect rat7-1 cells 37&#194;&#176;C,                 high-copy RSS1 mRNA export rat7-1 cells monitored growth rates wild-type rat7-1                    using fluorescence situ hybridization. Figure 3 <br /> cells presence absence RSS1 (Figure 2).              shows subcellular location poly + RNA permissive temperature (23&#194;&#176;C), growth behav-                  wild-type cells, rat7-1 cells, rat7-1 cells ior rat7-1 presence                     presence high-copy RSS1, permissive <br /> absence high-copy RSS1, cells did multiply                 restrictive temperatures. strains grown rapidly wild-type cells. 37&#194;&#176;C, presence               23&#194;&#176;C, poly + RNA distributed high-copy RSS1 permitted rat7-1 cells grow 370C                 cell, nuclear accumulation rate essentially identical rat7-1 cells            poly + RNA seen rat7-1 cells (Figure 3E), <br /> 23&#194;&#176;C.  growth rate               presence high-copy RSS1 (Figure 31). <br /> wild-type cells (FY86).  high-copy RSS1 partially                2-h shift 37&#194;&#176;C, nuclear accumulation suppressed growth defect rat7-1 cells 370C,                 poly + RNA seen 100% rat7-1 cells overexpression deleterious effect               cytoplasmic signal (Figure 3G). Poly + <br /> growth rat7-1 wild-type cells 230C.                          RNA pattern multiple punc- <br />  <br />  <br />  <br />  <br />                                                           Cellstml <br />                                                            (X 106) <br />  <br />  <br />  <br />  <br />                                                                                               Time (hrs) <br />  <br />  <br />  <br />                                                                            RSSI                            RSSI <br /> Figure 2. Comparison growth characteristics strains FY86 (WT), LGY101 (rat7-1), VDPY <br /> (rat7-1 containing RSS1 2,L plasmid) 23 370C. Cells allowed reach stationary phase diluted; growth monitored 23 37&#194;&#176;C. Duplicate samples taken 2, 3,            WT <br /> 6 h growth, cells counted hemacy-                                           WT <br /> tometer. point average cell counts duplicate samples. Strains grown SC-Leu <br /> plates 4 d 23 37&#194;&#176;C.                                              230C                            37&#194;&#176;C <br />  <br />  <br /> Vol. 7, October 1996                                                                                                         1607 <br />  V. Del Priore et al. <br />  <br />  <br />                          230C                                                                               370C <br />           FITC                         DAPT                                                 FITC                           DAPI <br />  <br />  <br />  <br />                                                                   WT <br />  <br />  <br />  <br />                                                               rat7-J <br />  <br />  <br />  <br />  <br />                                                                 RSSJ <br />  <br />  <br />  <br />  <br /> Figure 3. High-copy RSS1 partially suppresses mRNA export defect rat7-1 cells.  D) WT (FY86); (E-H) rat7-1 cells;  L) RSS1 <br /> (rat7-1 cells transformed pVDP7). Cells grown mid-log phase 23&apos;C left 23&#194;&#176;C   B, E, F,  J) shifted 370C 2 h (C, D, G, H, K, L). Cells fixed <span id='am-28' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> subjected situ hybridization using digoxigenin-tagged oligo (dT50) probe detect poly + RNA. Hybridizing probe visualized FITC-conjugated anti-digoxigenin <br /> antibody   C, E, G,  K). fields cells stained DAPI (B, D, F, H, J, L). Images taken using cooled <br /> CCD camera.   WT, 23&#194;&#176;C, FITC; (B) WT, 23&apos;C, DAPI; (C) WT, 370C, FITC; (D) WT, 370C, DAPI; (E) rat7-1, 23&#194;&#176;C, FITC; (F) rat7-1, 23&#194;&#176;C, DAPI; <br /> (G) rat7-1, 37&#194;&#176;C, FITC; (H) rat7-1, 370C, DAPI;   high-copy RSS1, 230C, FITC; (J) high-copy RSS1, 230C, DAPI; (K) high-copy RSS1, 370C, <br /> FITC; (L) high-copy RSS1, 370C, DAPI. <br />  <br /> tate spots, colocalize nucleolar antigens                         Fragmentation nucleolus shift (Del Priore Cole, unpublished results).                           37&#194;&#176;C observed nonpermissive tem- <br /> rat7-1 cells containing high-copy RSS1 shifted                     perature yeast strains defective mRNA <br /> 37&#194;&#176;C 2 h, poly + RNA signal                          export (Kadowaki et al., 1994, 1995; Heath et al., <br /> present nucleus, clear                         1995; Goldstein et al., 1996), including strains carry- <br /> cytoplasmic signal present rat7-1 cells                    ing rat7-1 allele (Heath et al., 1995). examine <br /> lacking high-copy RSS1 (Figure 3, compare K G).                       nucleolar fragmentation rat7-1 cells grown  cytoplasmic signal weaker                           23&#194;&#176;C shifted 37&#194;&#176;C 2 h, performed indi- cytoplasmic signal seen wild-type cells                        rect immunofluorescence analysis monoclo- <br /> shifted 37&#194;&#176;C (Figure 3C). changes signal                          nal antibody nucleolar protein, Nsrlp <br /> intensities seen cultures shifted                     (monoclonal antibody 2.3B). results shown 37&#194;&#176;C 4 h (Del Priore Cole, unpublished                          Figure 4. grown permissive temper- <br /> results). conclude high-copy RSS1 partially                        ature, rat7-1 cells distribution nucle- <br /> suppressed block mRNA export caused                          olar antigen wild type (Figure 4, <br /> rat7-1 mutation.                                                           compare E). signal present cres- <br />  <br /> 1608                                                                                                              Molecular Biology Cell <br />                                                                                    RSSI High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br />                          230C                                                                            370C <br />          FITC                           DAPI                                             FITC                            DAPI <br />  <br />  <br />                                                                   WT <br />                                                                                                                 <br />                                                                 rat7-1 <br />  <br />  <br />  <br />                                                                 RSSJ <br />  <br /> Figure 4. Indirect immunofluorescence anti-fibrillarin antibody test nucleolar fragmentation.  D) WT (FY86); (E-H) rat7-1; <br />  L) high-copy RSS1 (rat7-1 cells transformed pVDP7). Cells grown mid-log phase 23&#194;&#176;C maintained temperature   B, E, F,  J) shifted 37&#194;&#176;C 2 h (C, D, G, H, K, L). Fixation immunofluorescence techniques performed following protocol described MATERIALS METHODS. DAPI images fields cells FITC images left.   WT, 23&#194;&#176;C, FITC; (B) WT, 23&#194;&#176;C, DAPI; (C) WT, 37&#194;&#176;C, FITC; (D) WT, 37&#194;&#176;C, DAPI; (E) rat7-1, 23&#194;&#176;C, FITC; (F) rat7-1, 23&#194;&#176;C, DAPI; <br /> (G) rat7-1, 37&#194;&#176;C, FITC; (H) rat7-1, 37&#194;&#176;C, DAPI;   high-copy RSS1, 23&apos;C, FITC; (J) high-copy RSS1, 23&#194;&#176;C, DAPI; (K) high-copy RSS1, 37&#194;&#176;C, <br /> FITC; (L) high-copy RSS1, 37&#194;&#176;C, DAPI. <br />  <br /> cent-shaped pattern, DAPI signal,                          pattern nucleolar antigen distribution represents DNA staining, excluded                            wild-type rat7-1 cells (Figure 4, compare location intensity nucle-                      E). 2-h shift nonpermissive <br /> olar signal reaches maximum. shifted                         temperature 37&#194;&#176;C, dramatic nucleolar frag- <br /> 37&#194;&#176;C, wild-type cells displayed distribu-                       mentation seen presence high-copy <br /> tion nucleolar antigen seen 23&#194;&#176;C                      RSS1 similarly treated rat7-1 cells (Figure 4, <br /> (Figure 4C). contrast, 2-h shift 37&#194;&#176;C,                     compare K G). cells showed frag- dramatic fragmentation nucleoli                       mentation nucleolus, cells rat7-1 cells, agreement previous                     crescent-shaped morphology seen wild-type <br /> ings (Heath et al., 1995). cells,                   cells.  RSS1 present high-copy nucleolar fragments seen (Figure 4G).                       rat7-1 cells, nucleolar fragmentation phenotype <br /> Fragmentation detected soon 20 min                       observed absence RSS1 partially sup- <br /> shifting nonpermissive temperature (Gorsch                        pressed. Cole, unpublished results).                                             temperature-sensitive yeast strains   grown permissive temperature,                           nucleolar disruption seen shift non- <br /> rat7-1 cells containing high-copy RSS1 displayed                      permissive temperature, defects rRNA synthesis <br />  <br /> Vol. 7, October 1996                                                                                                                      1609 <br />  V. Del Priore et al. <br />  processing reported (Traglia et                   WTi         raf 7-1          RSSI                    rIt/l al., 1989; Aebi et al., 1990). fact rat7-1 <br /> strain showed severe nucleolar disruption pheno-                                mem_A <br />                                                                                      ..                            32DS <br /> type shifted nonpermissive temperature                                                                 35S <br />                                                                                                                                  I_ <br /> prompted determine showed                            40a.~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ <br />                                                                                                         ...                 IV <br /> defects rRNA processing. performed [3H]- <br /> uridine incorporation assay wild-type rat7-1                  ~~~~~~~~~~~G <br />                                                                          1                                5; <br />                                                                                                         ? : <br />                                                                                                                    25S <br /> cells permissive nonpermissive tem-              1 <br />                                                                        3&apos;t&apos;2 <br />                                                                      l&apos; <br />                                                                        2 <br />                                                                           C 37( <br />                                                                              3 <br />                                                                                            7C <br />                                                                                           56 <br />                                                                                                 -,,37c .7( <br />  <br /> peratures. rat7-1 cells shifted 37&#194;&#176;C,                          *             : incorporation [3H1-uridine precipitable <br /> counts decreased 20% level incorpora-                                                                        23&apos;(&apos; 37&apos;C~ <br />                                                                                                                            9     10 <br /> tion seen 23&#194;&#176;C 15 min shift 37&#194;&#176;C 10% 30-min shift. labeled rRNA         Figure 5. Analysis rRNA processing. Cells labeled produced analyzed electrophoresis flu-         [5-3H]-uridine, total RNA isolated WT (FY86), rat7-1, <br /> orography (Figure 5). autoradiograph shows            high-copy RSSI (rat7-1 transformed pVDP7) cells grown <br />                                                           23&#194;&#176;C shifted 37&#194;&#176;C 1 2 h. Labeled RNA was significant difference rRNA          analyzed electrophoresis approximately equal numbers processing wild-type rat7-1 cells        counts labeled RNA 1.2% agarose-formaldehyde gel. grown permissive temperature             control rRNA processing defects, strain carrying rnal-1 <br /> (compare Figure 5, lanes 1 3). contrast,     allele included. (1) WT grown 23&#194;&#176;C; (2) WT grown 37&#194;&#176;C                                                           2 h; (3) rat7-1 grown 23&#194;&#176;C; (4) rat7-1 grown 37&#194;&#176;C 1 h; (5) <br /> 1 h 37&#194;&#176;C, rat7-1 cells accumulated 35S precur-     rat7-1 grown 37&#194;&#176;C 2 h; (6) high-copy RSS1 grown 23&#194;&#176;C; (7) <br /> sor, primary transcript yeast rDNA        high-copy RSS1 grown 37&#194;&#176;C 1 h; (8) high-copy RSS1 grown genes, aberrant 23S RNA (Figure 5, lane 4).        37&#194;&#176;C 2 h; (9) rnal grown 23&#194;&#176;C; (10) rnal grown 37&#194;&#176;C Accumulation 35S 23S rRNAs seen           40 min. Asterisk indicates 23S intermediate product seen                                                           strains rRNA processing defects. control strain carrying rnal-1 allele shifted 37&#194;&#176;C 40 min (Figure 5, lane 10), reported previously (Traglia et al., 1989).   Rat7p Lost NPCs 37&#194;&#176;C defects seen soon 15 min       Presence Absence High-Copy RSS1, temperature shift rat7-1 cells (Del Priore     NPC Clustering Partially Maintained Cole, unpublished results) severe 2 h       Rsslp Overexpression shift (Figure 5, lane 5).                       rat7-1 cells shifted nonpermissive <br />    determine presence high-copy         temperature, mutant Rat7p rapidly lost RSS1 rat7-1 cells effect level       NPCs, determined immunofluorescence rRNA synthesis fidelity rRNA processing        anti-Rat7p antibody (Gorsch et al., 1995). strain shifted 37&#194;&#176;C, measured         overexpression Rsslp sufficient incorporation [3H]-uridine precipitable       maintain Rat7p nuclear rim, examined counts. 1-h shift 37&#194;&#176;C, incorporation   location Rat7p/Nupl59p indirect immunofluo- <br /> presence high-copy RSS1 20% level        rescence wild-type cells rat7-1 cells incorporation seen 23&#194;&#176;C subsequently            lacking containing high-copy RSS1. results creased 30% 23&#194;&#176;C incorporation 2 h     shown Figure 6. 23&#194;&#176;C, wild-type cells showed 37&#194;&#176;C. analyzed labeled rRNA elec-          punctuate staining pattern surrounding nucleus <br /> trophoresis. data  presence       (Figure 6A), localized staining                                                           DAPI (Figure 6B). reported previously (Gorsch et <br /> high-copy RSS1, partial suppression        al., 1995), NPCs clustered rat7-1 cells grown rRNA processing defects seen rat7-1 cells       23&#194;&#176;C (Figure 6F). identical pattern clustered restrictive temperature (Figure 5, compare lanes      NPCs seen 23&#194;&#176;C rat7-1 cells containing high- <br /> 7 8 lanes 4 5). Accumulation 35S          copy RSS1 (Figure 6K). <br /> rRNA detected lesser degree          cells shifted 37&#194;&#176;C 2 h, anti- rat7-1 cells, significant accumu-     Rat7p/Nupl59p antibody continued stain wild- <br /> lation 23S rRNA.  rat7-1 cells shifted      type cells punctate rim staining pattern <br /> 37&#194;&#176;C, defects levels rRNA syn-       seen cells grown 23&#194;&#176;C (Figure 6C). contrast, thesis kinetics fidelity processing     signal Rat7p/Nupl59p detectable rat7-1 35S rRNA precursor. presence             cells, absence (Figure 6H) presence <br /> high-copy RSS1, observed partial suppression        (Figure 6M) high-copy RSS1.  presence defects levels kinetics rRNA pro-       high-copy RSS1 unable prevent loss cessing longer saw defects fidelity      NPCs mutant Rat7p/Nupl59p mutant cells <br /> processing.                                               shifted 37&#194;&#176;C. <br />  <br /> 1610                                                                                                           Molecular Biology Cell <br />                                                                                  RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br />                        230C                                                                      370C <br />         FTTC                    DAPT                                   FITC                     DAPT                       DIC <br />  <br />  <br />                                                     WT <br />  <br />  <br />  <br />                                                   rat7-1 <br />  <br />  <br />  <br />                                                   RSSI <br />  <br /> Figure 6. Indirect immunofluorescence anti-Rat7p antibody.  E) WT (FY86); (F-J) rat7-1; (K-O) RSS1 (rat7-1 cells transformed pVDP7). Cells grown mid-log phase 23&apos;C maintained temperature   B, F, G, K, L) shifted 37&apos;C 2 h (C-E, H-J, M-O). Cells fixed processed indirect immunofluorescence polyclonal anti-Rat7p antibody, described MATERIALS METHODS. DAPI DIC images fields cells FITC images left.   <br /> WT, 23&#194;&#176;C, FITC; (B) WT, 23&apos;C, DAPI; (C) WT, 37&#194;&#176;C, FITC; (D) WT, 37&apos;C, DAPI; (E) WT, 37&apos;C, DIC; (F) rat7-1, 23&apos;C, FITC; (G) rat7-1, 23&apos;C, <br /> DAPI; (H) rat7-1, 37&#194;&#176;C, FITC;   rat7-1, 37&#194;&#176;C, DAPI; (J) rat7-1, 37&apos;C, DIC; (K) high-copy RSS1, 23&apos;C, FITC; (L) high-copy RSS1, 23&apos;C, DAPI; <br /> (M) high-copy RSS1, 37&apos;C, FITC; (N) high-copy RSS1, 37&apos;C, DAPI; (0) high-copy RSS1, 37&apos;C, DIC. <br />  <br />  <br />    Clustering NPCs common phenotype seen                     Figure 7,  F, K, respectively). shifted yeast cells carrying mutant alleles different                37&#194;&#176;C 2 h, high percentage rat7-1 cells showed <br /> nucleoporins (Wente et al., 1992; Wente Blobel,                     redistribution NPC antigens pattern 1993, 1994; Bogerd et al., 1994; Doye et al., 1994;                     similar wild-type distribution (Figure 7, <br /> Aitchison et al., 1995; Gorsch et al., 1995; Heath et al.,              compare H C), agreement observations <br /> 1995; Li et al., 1995; Pemberton et al., 1995; Goldstein et             reported previously (Gorsch et al., 1995). contrast, al., 1996). rat7-1 strain novel NPC clustering                presence high-copy RSS1, NPCs rat7-1 cells <br /> phenotype pores considerably                        showed tendency retain clustered distribu- <br /> clustered shift 37&#194;&#176;C (Gorsch et al., 1995).             tion (Figure 7M). <br /> altered distribution NPCs rat7-1 cells occurs rap-                  examined rat7-1 cells presence idly, 1 h shift 37&#194;&#176;C. determine                   absence high-copy RSS1 section electron <br /> high-copy RSS1 affected phenotype,                          microscopy. different cells, NPCs seen formed indirect immunofluorescence RL-1                        evenly distributed, grouped, clustered. consid- <br /> monoclonal antibody, recognizes NPC antigens                      ered NPCs clustered metazoan yeast cells (Snow et al., 1987;                    NPCs section clustered  form- <br /> Copeland Snyder, 1993). 23&#194;&#176;C wild-type                        ing single &quot;grape-like&quot; structure. Pores consid- <br /> cells rat7-1 cells presence absence                 ered grouped multiple NPCs adja- <br /> high-copy RSS1, pattern staining RL-1                   cent plane nuclear envelope.  <br /> antibody seen anti-                       multiple groups seen single section, Rat7p antiserum (compare Figure 6,  F, K,                    additional isolated NPCs present. Cluster- <br />  <br /> Vol. 7, October 1996                                                                                                                   1611 <br />  V. Del Priore et al. <br />  <br />                        230C                                                                       370C <br />        FITC                     DAPI                                     FITC                     DAPI                        DIC <br />  <br />  <br />                                                       WT <br />  <br />  <br />  <br />                                                     rat7-1 <br />  <br />  <br />  <br />                                                      RSSI <br />  <br /> Figure 7. Indirect immunofluorescence examine distribution NPCs.  E) WT (FY86); (F-J) rat7-1; (K-O) high-copy RSS1 (rat7-1 <br /> cells transformed pVDP7). Cells grown log phase permissive temperature maintained 23&#194;&#176;C   B, F, <br /> G, K, L) shifted nonpermissive temperature 37&#194;&#176;C (C-E, H-J, M-O). fixation processing indirect <br /> immunofluorescence, cells incubated RL-1 monoclonal antibody, recognizes NPC proteins organisms, <br /> including S. cerevisiae. DAPI DIC images fields cells FITC images left.   WT, 23&#194;&#176;C, FITC; (B) WT, <br /> 23&#194;&#176;C, DAPI; (C) WT, 37&#194;&#176;C, FITC; (D) WT, 37&#194;&#176;C, DAPI; (E) WT, 37&#194;&#176;C, DIC; (F) rat7-1, 23&apos;C, FITC; (G) rat7-1, 23&apos;C, DAPI; (H) rat7-1, 37&#194;&#176;C, FITC; <br />   rat7-1, 37&#194;&#176;C, DAPI; (J) rat7-1, 37&#194;&#176;C, DIC; (K) high-copy RSS1, 23&#194;&#176;C, FITC; (L) high-copy RSS1, 23&#194;&#176;C, DAPI; (M) high-copy RSS1, 37&#194;&#176;C, FITC; <br /> (N) high-copy RSS1, 37&#194;&#176;C, DAPI; (0) high-copy RSS1, 37&#194;&#176;C, DIC. <br />  <br />  <br /> ing extreme distribution pattern group-                   distributed nucleus (E) grouped <br /> ing. examined sections 50 separate cells                    (F). RSS1 overexpressed rat7-1 cells strain, data summarized Figure                    grown 23&#194;&#176;C, little change NPC <br /> 8. Nuclei showing different phenotypes                         distribution patterns observed rat7-1 <br /> shown Figure 9. 80% sections                    cells, clustering grouping observed &gt;70% wild-type cells examined showed pores                          cells. Figure 9, G H, shows sections rat7-1 <br /> evenly distributed, small groups NPCs                         cells overexpressing Rsslp grown 23&#194;&#176;C, seen -20% sections, clusters                     NPCs clustered (G) grouped (H). observed. Figure 9, B, shows wild-type                         indirect immunofluorescence suggested little change <br /> nuclei NPCs evenly distributed                      NPC distribution cells shifted nucleus   grouped (B). rat7-1 cells                   37&#194;&#176;C, section electron microscopy revealed grown 23&#194;&#176;C, NPCs clustered grouped                     modest increase fraction sections  &gt;80% sections examined; grouping                          ing evenly distributed NPCs slight decrease common clustering. Figure 9, C D,                          fraction sections showing clustered grouped <br /> shows nuclei representative rat7-1 cells grown                    NPCs. Figure 9, J, shows nuclei rat7-1 cells 23&#194;&#176;C, NPCs clustered (C) grouped                       overexpressing RSS1 shifted 37&#194;&#176;C, (D). NPCs substantially clustered                           pores grouped   evenly distrib- <br /> grouped rat7-1 cells shifted 37&#194;&#176;C, NPCs                       uted (J). data indicate NPCs remained distributed evenly 66% sections                       clustered grouped rat7-1 cells shifted 37&#194;&#176;C <br /> NPCs grouped clustered 34%. Figure 9, E                     RSS1 overexpressed absence F, shows rat7-1 cells grown 37&#194;&#176;C, NPCs                       RSS1 overexpression. suggests retention  <br /> 1612                                                                                                           Molecular Biology Cell <br />                                                                       RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> clustering grouping important partial           conclude Rsslp located nuclear pores <br /> suppression mRNA export defect rat7-1 cells          cytoplasm. RSS1.                                                         Figure 10C shows Western blot identical <br />                                                                amounts protein loaded lane                                                                9E10 anti-myc monoclonal antibody used  <br /> Immunolocalization Rsslp                                    tect Rsslpmyc, migrated apparent mo- determine subcellular location Rsslp,             lecular weight -66 kDa, close expected <br /> tagged protein triple &quot;myc&quot; epitope.            position tagged protein. Comparison lanes 2 <br /> epitope introduced C-terminal end            3 indicate Rsslpmyc expressed level <br /> protein (aa 529), BamHI site created            approximately eightfold higher expressed site-directed mutagenesis. myc-tagged RSS1 gene            high-copy plasmid (Figure 10C, lane 3) able complement fully RSS1 null strain               expressed CEN plasmid (Figure 10C, lane 2), present low-copy (CEN) high-copy             consistent stronger fluorescent signal seen (2,) plasmids;  unable suppress          Figure 10B comparison 10A. 9E10 anti- <br /> rat7-1 mutation. Indirect immunofluorescence               body detected band wild-type cells (Figure <br /> performed 9E10 anti-myc antibody.                10C, lane 1) migrated closely position <br /> observed conventional fluorescence microscopy,            Rsslpmyc.  immunofluorescent signal cells showed strong cytoplasmic signal             detected wild-type cells antibody (Del <br /> weak rim staining nucleus  unpub-               Priore Cole, unpublished results). <br /> lished results). intense cytoplasmic signal partially masking nuclear rim pat- <br /> tern, samples examined confocal fluorescence           Depletion Rsslp Causes mRNA Export Defect <br /> microscopy, showed clearly Rsslp            RSS1 essential gene, wished located cytoplasm nuclear           establish effect yeast cells caused loss rim (Figure 10). portion Rsslpmyc located       Rsslp function.  Rsslp Rsslpmyc <br /> nuclear rim colocalized nucleoporins recognized           ORFs placed control GALl pro- RL-1 monoclonal antibody (Figure 10A)               moter CEN plasmid transformed Rat7p/Nup159p (Del Priore Cole, unpub-                RSS1 null strain, GAL depletion experi- <br /> lished results). tagged protein readily           ments performed. Figure 1 shows detected cytoplasm yeast cells,           cells lacking genomic copy RSS1 primarily cytoplasmic signal increased          Rsslp Rsslpmyc expression driven protein overexpressed placing gene              GALl promoter, growth occurred cells encoding Rsslpmyc high-copy (2,u) vector (Figure          grown galactose grown glucose. <br /> 10, compare B). control, indirect immuno-           Figure 1 B shows growth curves cells  <br /> fluorescence performed localize plasmid-           taining Rsslp Rsslpmyc control GALl <br /> borne myc-tagged nucleoporins, cytoplasmic              promoter grown galactose transferred signal observed Rat7p/Nupl59pmyc             glucose-containing media 37&#194;&#176;C. rate Rat2p/Nupl20pmyc  unpublished results).                 growth decreased 10 12 h transfer <br />  <br />  <br />  <br />  <br />                                                                                                          ungrouped <br />  <br />                                                                                                          grouped/clustered <br /> Figure 8. Nuclear pore clustering partially <br /> maintained rat7-1 cells overexpressing RSS1 37&#194;&#176;C. Electron microscopy performed wild-type cells grown 37&#194;&#176;C rat7-1 rat7-1 cells containing high-copy <br /> RSS1 plasmid, grown 23&#194;&#176;C shifted 37&#194;&#176;C. sections 50 different cells exam- <br /> ined strain, percentage cells <br /> showing ungrouped (filled squares) grouped/clustered NPCs (open squares)           WT     rat7-1    rat7-1    RSS1       RSS1 <br /> determined.                                                230C      370C      230C       370C <br />  <br />  <br /> Vol. 7, October 1996                                                                                                  1613 <br />  V. Del Priore et al. <br />  <br />                                    cells expressing wild-type Rsslp glucose                                    6 8 h cells expressing Rsslpmyc.                                    graph suggests depletion Rsslpmyc                                    achieved earlier depletion wild-type <br />                                    Rsslp. result indicates myc-tagged Rsslp <br />                                    stable wild-type Rsslp. monitor                                    decrease levels Rsslpmyc cells                                    transferred glucose, performed Western blot <br />                                    extracts cells grown repressing (glu- <br />                                    cose) medium different periods time (Figure <br />                                    liC). figure shows  2.5 h growth <br />                                    glucose, level protein reduced                                     -10% level cells grown galactose.                                    5 h growth glucose, Rsslpmyc barely  <br />                                    tectable, indicating protein readily  <br />                                    pleted. Secl3p used control equal load- <br />                                    ing. <br />                                      determine depletion Rsslp resulted                                    mRNA export defect, performed situ <br />                                    hybridization assay cells expressing Rsslp                                    GALI promoter. Cells grown galactose                                    transferred glucose 3, 6, 9, 12 h (Figure 12A). <br />                                    cells grown galactose, poly + RNA <br />                                    evenly distributed cell (Figure <br />                                    12A, Oh), case wild-type cells. cells <br />                                    grown 3 h glucose, change                                    distribution pattern seen, 6 h glucose, <br />                                    nuclear accumulation poly + RNA detect- <br />                                    able -20-30% cells. 9 h, percentage <br />                                    cells accumulating mRNA nuclei increased <br />                                    -80%. 12 h glucose, essentially cells <br />                                    showed nuclear accumulation poly + RNA                                    substantially reduced cytoplasmic signal. result <br />                                    provides strong evidence Rsslp involved                                    mRNA export. <br />                                      strain bearing GALl-driven myc- <br />                                    tagged RSS1 plasmid ceased growth rapidly <br />                                    transfer glucose-containing media cells <br />                                    bearing GALl-driven wild-type RSS1 plasmid, <br />                                    tested reflected rapid <br />                                    onset mRNA export defect performing                                    situ hybridization assay GALl-driven <br />                                    myc-tagged RSS1-containing strain. Cells grown                                    galactose maintained galactose                                    shifted glucose 5 h (Figure 12B).                                    grown galactose, cells showed nu- <br />                                    clear accumulation poly + RNA. 5 h  <br />  <br />                                    Figure 9 (cont). Representative electron micrographs nuclei <br />                                    showing different types NPC distributions seen wild-type <br />                                    cells incubated 37C  B), rat7-1 cells incubated 23&#194;&#176;C (C <br />                                    D) shifted 37&#194;&#176;C (E F), rat7-1 cells overexpressing <br />                                    RSS1 incubated 23&#194;&#176;C (G H) 37&#194;&#176;C  J). figure <br />                                    shows examples evenly distributed (ungrouped), grouped,                                    clustered NPCs seen strains. large black arrows                                    examples grouped clustered NPCs, small black arrows <br />                        Figure 9.   individual NPCs;  nucleolus. Bar, 0.5 ,um. <br />  <br /> 1614                                                                    Molecular Biology Cell <br />                                                                   RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> glucose, 90% cells showed nuclear          DISCUSSION <br /> mRNA export defect, 6 h glucose,          paper, identification strain bearing wild-type allele RSS1            RSS1 gene using screen high-copy extra- <br /> showed nuclear accumulation 20-30%              genic suppressors rat7-1 allele. RSS1 encodes cells. data suggest myc-           novel 538-amino-acid protein located tagged version Rsslp stable wild-type      NPCs, colocalizes nucleoporins, <br /> Rsslp, depletion protein occurred        cytoplasm. expressing Rsslp earlier, resulting rapid development        GALI promoter, Rsslp depleted poly + RNA export defect. lower level Rsslpmyc (compared wild-type Rsslp)           cells transferring galactose media likely explains inability Rsslpmyc sup-     glucose media. defect export poly + RNA <br /> press rat7-1 mutation.                                 observed cells 9-12 h <br />   examined nuclear protein import, nuclear         transfer glucose. kinetics pore distribution, nucleolar integrity strain   growth ceased transfer glucose similar <br /> bearing GALl-driven wild-type RSS1. did         kinetics cells displayed nuclear <br /> detect defects nuclear protein import, nuclear      accumulation poly + RNA. defects nu- <br /> pore distribution, nucleolar fragmentation,        clear protein import, nuclear pore distribution, 12 h growth glucose  unpublished           nucleolar integrity seen Rsslp  <br /> results). findings suggest defect        pleted.  results suggest Rsslp <br /> mRNA export primary defect resulting           normally plays role nucleocytoplasmic export depletion Rsslp.                                        poly + RNA. <br />  <br /> Figure 10. Rsslp localizes nu- <br /> clear pore complexes cy- <br /> toplasm. Immunolocalization                                                    FITC                                  Texas Red <br /> myc-tagged Rsslp performed.        Cells carrying disruption RSS1 transformed CEN plas- <br /> mid encoding myc-tagged Rsslp. <br /> Transformed cells grown log phase, fixed, processed indirect immunofluorescence. Im- <br /> ages taken confocal mi- <br /> croscope.   FITC shows signal <br /> obtained anti-myc antibody <br /> 9E10 used, Texas Red <br /> shows signal distribution cells costained RL-1 anti-nucleoporin anti- <br /> body. (B) Indirect immunofluores- <br /> cence performed cells carrying disruption RSS1 carrying 2,L plasmid encoding myc-tagged <br /> Rsslp. (C) Western analysis myc-                  FITC <br /> tagged Rsslp. SDS polyacryl- <br /> amide gel electrophoresis, myc-        B                                        C       v <br />                                                                                             :; tagged Rsslp migrates 66 kDa                                                         ;.s, <br />                                                                                                             -   116 overexpressed pro- <br /> duced 21L vector RSS1 <br />                                                                                        , <br />                                                                                        *w*- <br />  <br />  <br />                                                                                        *;m.S <br />                                                                                                             - 97 <br />  <br />  <br /> null strain. anti-myc 9E10 anti-                                                   .,s,-_. _..          - 66 <br /> body used detect myc-                                                           .qXJ <br />                                                                                        .-;- * <br />                                                                                        :::. <br /> tagged Rsslp. (Lane 1) WT (FY86);                                                        .;. <br />  <br />                                                                                                    9_ <br />                                                                                                     ; <br /> (lane 2) rsslA cells transformed CEN plasmid encoding myc-                                                                            - 45 <br /> tagged Rsslp (VDPY112); (lane 3) <br /> rsslA cells transformed 2,u <br /> plasmid encoding myc-tagged <br /> Rsslp (VDPY113). Secl3p used control equal loading,                                                                        Secl3p anti-Secl3p antibody obtained C. Barlowe (Dartmouth Med- <br /> ical School, Hanover, NH).                                                                     1    2   3 <br />  <br />  <br />                                             time (hs) <br /> Figure 11. Growth 37&#194;&#176;C wild-type yeast culture yeast strains disrupted chromosomal RSS1 locus harboring <br /> plasmid-borne GAL1::RSS1 GALl ::RSSl myc alleles, galactose shifted glucose. (B) Growth curves yeast strains shown  <br /> Cells grown overnight, diluted early log phase, incubated 37&#194;&#176;C galactose-containing media shifted glucose-containing media indicated time. Cell numbers determined hemocytometer. (C) Western analysis cell extracts <br /> prepared strain containing GALl::RSS1myc allele RSS1 various times transfer glucose-containing medium. Secl3p used control equal loading. <br />  <br />  <br />  <br />  <br /> 1616                                                                                                          Molecular Biology Cell <br />  <br />  <br />  <br /> Figure 12. Depletion Rsslp results defect mRNA export.   situ hybridization assay performed cells harboring GALI::RSSI. Cells grown galactose-containing medium shifted glucose-         DA P1 <br /> containing medium 3, 6, 9, 12 h. location nuclei determined staining DNA DAPI. <br /> (B) situ hybridization assay yeast cells harboring <br /> GALI::RSS1 myc. Cells grown galactose shifted glucose 5 h. <br />  <br />    High-copy RSS1 suppressed growth defect                     Western blot analysis (Figure 8), expression rat7-1 cells permitted mutant cells grow               high-copy plasmid led Rsslp levels ap- 37&#194;&#176;C did 23&#194;&#176;C absence                     proximately eightfold greater CEN plas- <br /> suppressor.  suppression partial,                    mid.  CEN plasmids maintained yeast rat7-1 cells grew considerably slowly                 cells -1-4 copies/cell, result 23&#194;&#176;C did wild-type cells (Figure 2),               modest overexpression, compared levels considerable accumulation poly + RNA                    Rsslp produced genomic copy. estimate <br /> nuclei. Overexpression RSS1 deleterious                     Rsslp overexpressed &gt;10-fold ex- <br /> effects growth wild-type rat7-1 cells             pressed high-copy plasmid. epitope- 23&#194;&#176;C. Suppression observed RSS1                       tagged version Rsslp lower stability expressed CEN plasmid. basis                     wild-type form, likely explains <br />  <br /> Vol. 7, October 1996                                                                                                         1617 <br />  V. Del Priore et al. <br />  inability epitope-tagged form suppress         Rsslp directly involved RNA export.  deple- rat7-1 mutation.                                         tion Rsslp causes defect nucleocytoplasmic <br />    addition cessation growth, rat7-1 cells          export poly + RNA defects nuclear <br /> shifted 37&#194;&#176;C rapid inhibition mRNA ex-            protein import NPC distribution. Second, Rsslpmyc <br /> port, rapid fragmentation nucleolus, dra-       present cytoplasm NPCs, matic decrease rRNA production. believe           distribution similar seen factors  defects nucleolar morphology rRNA bio-            volved nucleocytoplasmic transport.  synthesis caused directly mutation             RSS1 overexpressed rat7-1 ts strain, Rat7p/Nupl59p likely indirect conse-            able suppress partially mRNA export defect quences defect RNA ribosome export.          strain. <br /> expression RSS1 partially suppressed phe- <br /> notypes. rat7-1 cells shifted 37&#194;&#176;C 2 h <br /> display virtually cytoplasmic signal poly +         Rsslp Act Suppress RNA <br /> RNA (Figure 3; Gorsch et al., 1995), presence         Export Defect? <br /> RSS1 high copy led maintenance cytoplasmic        important observation studies <br /> signal poly + RNA. signal reflect           high-copy RSS1 bypass suppressor dramatically reduced turnover cytoplasmic          null allele RAT7/NUP159.  detectable <br /> poly + RNA continued mRNA export.              rat7-lp lost nuclear rim shift explanation consistent ability         rat7-1 cells 37&#194;&#176;C, presence absence rat7-1 cells form colonies 37&#194;&#176;C RSS1           high-copy RSS1 (Figure 6). rat7-1 cells  <br /> present high copy.  conclude            tinue grow 37&#194;&#176;C presence high-copy <br /> mRNA export maintained rat7-1 cells shifted        RSS1, mutant rat7-lp perform essential <br /> 37&#194;&#176;C presence high-copy RSS1 level            function growth conditions. Possibly adequate support cell growth division. Al-            low level rat7-1p remains associated cytoplasmic mRNA present             NPCs temperate shift, rat7-1 cells suppressed high-copy RSS1 quite          detected indirect immunofluorescence. High- <br /> low, noteworthy reduced level suf-       copy RSS1 act directly facilitate RNA export <br /> ficient support moderate rate growth.        NPCs lacking detectable rat7-lp, examined nuclear protein import rat7-1 cells         act facilitate limited retention mutant rat7-lp multiple assays, cases, karyophilic       NPCs. know direct <br /> proteins imported efficiently nuclei (Gorsch       interaction Rsslp Rat7p/Nupl59p. <br /> et al., 1995; Del Priore Cole, unpublished results).     Rat7p/Nupl59p readily extracted NPCs fact rat7-1 cells defects mRNA export       100 mM KCl 0.2% Triton X-100. nuclear protein import suggests Rat7p/       unable coprecipitate Rat7p/Nupl59p Rsslp <br /> Nupl59p directly involved RNA export.                  using extraction conditions anti-Rat7p/ <br />    rat7-1 cells, Rat7p lost nuclear rim       Nupl59p polyclonal antibodies (Del Priore Cole, mutant cells shifted 37&#194;&#176;C (Gorsch et al.,        unpublished results). Rsslp does interact <br /> 1995). RAT7/NUP159 essential gene,             Rat7p/Nupl59p, interaction high-copy RSS1 acting maintain Rat7p          weak survive mild extraction conditions. Al- <br /> NPCs.  presence high-copy RSS1,            ternatively, Rsslp interact directly compo- mutant protein lost NPCs shift       nents NPC Rat7p/Nupl59p, 37&#194;&#176;C rapidly absence high-copy        interact non-NPC factors involved mRNA <br /> RSS1 (Figure 6). novel phenotypes         export. <br /> rat7-1 cells loss Rat7p NPCs rat7-1      different models hypothesized ex- <br /> cells shifted 37&#194;&#176;C associated substantial       plain mechanism suppression. model, <br /> reversal clustering NPCs seen cells       Rsslp play dynamic enzymatic role grown 23&#194;&#176;C (Gorsch et al., 1995). Interestingly,   nucleocytoplasmic transport. conventional fluores- <br /> presence high-copy RSS1, clustering           cence microscopy, observed nuclear rim grouping NPCs seen 23&#194;&#176;C retained            staining Rsslpmyc substantially intense <br /> cells shifted 37&#194;&#176;C, suggesting maintenance        cytoplasmic signal. distribution simi- clustering important maintaining RNA         lar soluble factors involved export.                                                      nuclear protein import. example, importin <br />    data presented paper suggest RSS1,       ,3/karyopherin 13/Kap95p, subunits high copy, permits rat7-1 cells grow non-       heterodimeric receptor recognizes nuclear local- <br /> permissive temperature 37&#194;&#176;C acting directly         ization signals, cytoplasm, binds partially suppress RNA export defect rat7-1           substrate NPCs, does accumulate cells. lines evidence support idea         nucleus (Gorlich et al., 1995; Moroianu et al., 1995).  <br /> 1618                                                                                         Molecular Biology Cell <br />                                                                        RSS1 High-Copy Suppressor ts Allele RAT7/NUP159 <br />  <br /> small GTPase Gsplp plays central role nuclear            ting rat7-1 NPCs retain ability export mRNP <br /> protein import, GTPase activating protein             substrate. <br /> Rnalp cytoplasmic higher concentration near nucleus (Hopper et al., 1990), pattern somewhat         ACKNOWLEDGMENTS <br /> similar Rsslp. Gsplp yeast homologue metazoan Ran protein. yeast Gsplp-          thank members Cole laboratory helpful discussions <br /> binding proteins Yrblp distribution similar        Dr. Erwin Schmidt support encouragement                                                               DNA sequence analysis portion chromosome IV containing Rsslp (Schlenstedt et al., 1995).          RSS1 gene. thank C. Barlowe, D. Compton, C. Heath NPCs rat7-1 cells absolutely defective         critical reading manuscript. thank C. Barlowe, L. <br /> RNA export, rate export low          Gerace, M. Snyder, J. Zhu, J.M. Bishop antibodies F. <br /> support growth mutant cells. Overexpressed Rsslp           Winston  Corbett yeast strains. Finally, thank Ken <br />                                                               Orndorff Ann Lavanway assistance microscopy. act directly increase rate export             nucleotide sequence analysis described supported Eu- mutant NPCs, restoring growth.                ropean Union Biotech II Program/Yeast Genome Sequencing Net- <br />    second model, RSS1 overexpression af-           work. studies supported grant National <br /> fect structure interactions NPCs mutant         Institutes Health (GM-33998) C.N.C. core grant <br />                                                               (CA-16038) Norris Cotton Cancer Center, Dartmouth Medical <br /> cells, stabilizing nuclear pore subcomplex        School, Hanover, NH. normally contains Rat7p/Nupl59p main- <br /> taining interactions mutant NPCs ele-              <br /> ments cytoskeleton nucleoskeleton critical RNA export. absence Rat7p/            <br /> Nupl59p, subcomplex NPC contains             <br /> Rat7p/Nupl59p assemble            assembled NPCs. Rsslp acts         <br /> ensure components putative Rat7p          <br /> subcomplex retained pore mutant              <br /> rat7-lp lost shift 37&#194;&#176;C. NPCs known     attached nuclear envelope lattice metazoan          <br /> cells (Maul, 1977; Goldberg Allen, 1992; Ris    <br /> Malecki, 1993), evidence similar          <br /> attachment occurs yeast cells (Allen Douglas,         <br /> 1989). association NPCs cytoskeletal ele-           <br /> ments observed metazoan cells (Franke,           <br /> 1971; Jones et al., 1982), evidence    <br /> similar associations yeast cells. Rat7p/          <br /> Nupl59p located cytoplasmic face NPCs,        distance consistent cyto-      <br /> plasmic fibrils NPC, Rat7p/Nupl59p              likely involved interactions           <br /> NPCs cytoskeletal elements nucleoskel-      <br /> etal elements. interactions essential    proper transport substrates NPCs.          <br /> critical connections NPCs cytoplasmic              <br /> structures disrupted mutation Rat7p/          <br /> Nupl59p nucleoporin, export substrates              unable exit NPCs associate       <br /> translation machinery. significant fraction polyri-       <br /> bosomes, mRNA, factors required protein syn-         <br /> thesis associated actin cytoskeleton (Lenk      <br /> et al., 1977; Cervera et al., 1981; Yang et al., 1990;         <br /> Zambetti et al., 1990a,b; Hesketh Pryme, 1991;         <br /> Hansen Ingber, 1992).  connections cytoskeleton NPCs,       <br /> mRNP exiting pore actively transferred          actin cytoskeleton cytoskeleton-bound ribo-          <br /> somes. high-copy Rsslp acts maintain                NPC-cytoskeletal interactions, permit-          <br />  <br /> </body></html>