<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 2 October 2008                                              Nucleic Acids Research, 2008, Vol. 36,  19 6269&#226;&#8364;&#8220;6283 <br />                                                                                                                 doi:10.1093/nar/gkn636 <br />  <br />  <br /> Evolutionary origins human apoptosis genome-stability gene networks <br /> Mauro   Castro1,2,*, Rodrigo J. S. Dalmolin1, Jose&#194;&#180; C. F. Moreira1, <br /> Jose&#194;&#180; C. M. Mombach3 Rita M. C. Almeida4 <br /> 1 <br />  Bioinformatics Unit, Department Biochemistry, Federal University Rio Grande Sul (UFRGS), Rua Ramiro <br /> Barcelos 2600-anexo, Porto Alegre 90035-003, 2Department Biological Sciences, Lutheran University Brazil, <br /> Gravata&#196;&#177;&#194;&#180; 94170-240, 3Department Physics, Federal University Santa Maria (UFSM), Santa Maria 97105-900 4Institute Physics, Federal University Rio Grande Sul (UFRGS), Avenida Bento Gonc&#194;&#184;alves 9500, Porto <br /> Alegre 91501-970, Caixa Postal 15051, Brazil <br />  <br /> Received 23, 2008; Revised September 14, 2008; Accepted September 15, 2008 <br />  <br />  <br />  <br /> ABSTRACT                                                                        INTRODUCTION <br /> Apoptosis essential complex multicellular                                concept apoptosis associated mainte- <br /> organisms failure associated                                    nance tissue homeostasis (1). programmed cell <br /> genome instability cancer. Interactions                             death (PCD) perspective multicellular organisms <br /> apoptosis genome-maintenance mechanisms                                     guarantees substitution old dysfunctional <br />                                                                                 cells, impaired accumulation cellular extensively documented include <br />                                                                                 damages environmental insults, partici- <br /> transactivation-independent -dependent func-                                pates directly tissue development (2). According tions, tumor-suppressor protein p53                                KEGG (3), reference pathway database, works &#226;&#8364;&#732;molecular node&#226;&#8364;&#8482; DNA-damage                                   100 genes coordinately working apoptosis. Removing <br /> response. apoptosis genome stability                               components a&#239;&#172;&#8364;ects identified ancient pathways eukary-                             impair pathway. complex metazoan <br /> ote phylogeny, biological evolution underlying                              organisms, defective apoptosis associated orga- emergence integrated remains lar-                              nogenesis disorders uncontrolled cell growth, <br /> gely unknown.  using computational methods,                                typically neoplastic diseases (4). reconstruct evolutionary scenario linked                            perspective cancer cell, suppressed apoptosis apoptosis genome stability pathways func-                             requirement order enhance cell &#239;&#172;?tness (5).                                                                                 extent, thought apoptosis related tional human gene/protein association network.                                                                                 genome instability sense mutation prone entanglement DNA repair, chromo-                              clones, containing aberrant genetic content  e. high stability apoptosis gene networks                                      number chromosome aberrations DNA point- <br /> appears caspase gene family anti-                              mutations), need defective apoptosis escape cell <br /> apoptotic gene BCL2.  critical nodes                               death (6&#226;&#8364;&#8220;8). entangle apoptosis genome stability                                   Genome-maintenance mechanisms intimately linked <br /> cancer genes (e.g. ATM, BRCA1, BRCA2, MLH1,                                     apoptotic components, indicates high number <br /> MSH2, MSH6 TP53), orthologs                                  proteins interact tumor-suppressor arisen different points evolution.                               protein p53. fact, protein interacts results demonstrate genome stability                                    major DNA repair mechanisms: <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> excision repair <br /> apoptosis opted evolution recruiting                             (NER), base excision repair (BER), mismatch repair <br />                                                                                 (MMR) recombinational repair (RER)&#226;&#8364;&#8221;homologous <br /> genes merge systems. provide <br />                                                                                 recombinational repair (HRR) nonhomologous end- examples exploit evolutionary plat- <br />                                                                                 joining (NHEJ). Concerning NER MMR, p53 form, judiciously extended informa-                               act transactivation-independent -dependent <br /> tion gene essentiality inferred model                                   manner (9). Furthermore, DNA repair proteins <br /> organisms human.                                                             stimulate apoptosis response DNA lesions, <br />  <br />  correspondence addressed. Tel: +55 51 3308 5577; Fax: +55 51 3308 5540; Email: mauro@ufrgs.br <br /> Correspondence addressed Rita M.C. Almeida. Tel: +55 51 3308 6521; Fax: +55 51 3308 7286; Email: rita ufrgs.br <br />  authors wish known  opinion, authors regarded joint Authors. <br />  <br /> &#195;&#376; 2008 Author(s) Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  6270 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  example BER-associated protein poly(<span id='am-10' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>-            human gene network. result obtained gene <br /> ribose) polymerase-1 (PARP1) (10) MMR                  network possible discriminate ancient, proteins MSH2, MSH6 MLH1 (11).             plastic essential regions earlier, plas- <br /> lapping apoptosis DNA repair genes renders           tic essential ones. Furthermore, cancer <br /> di&#239;&#172;&#402;cult precise de&#239;&#172;?nition functional boundaries           genes identi&#239;&#172;?ed gene network located systems, characteristic complex        earlier, plastic essential region. antici- <br /> biological networks (12).                                      pate analyses applied study origins <br />    hand, apoptosis genome-stability net-      broad range neoplastic diseases. <br /> works di&#239;&#172;&#8364;erent evolutionary roots. instance, core machinery eukaryotic repair systems conserved domains life,         MATERIALS METHODS <br /> expressive number eukaryotic proteins counter- archaea bacteria (13). Likewise, metazoan apo-      Human gene/protein-association network <br /> ptosis contains components identi&#239;&#172;?ed       protein&#226;&#8364;&#8220;protein interaction network associating 180 ancient organisms prokaryotes unicellular       human genes apoptosis genome-stability pathways <br /> eukaryotes.  molecular sources             extensively described Ref. (19). Brie&#239;&#172;&#8218;y, eukaryotic apoptosis network inherited         network generated using database <span id='am-16' about='xsd:string' typeof='owl:Thing'>STRING</span> (24) prokaryotes horizontal gene transfer (HGT)          input options &#226;&#8364;&#732;databases&#226;&#8364;&#8482;,&#226;&#8364;&#8482;experiments&#226;&#8364;&#8482; 0.700 <br /> di&#239;&#172;&#8364;erent events, exapted new functions form        con&#239;&#172;?dence level. STRING integrates di&#239;&#172;&#8364;erent curated, <br /> apoptosis network (14).                                        public databases containing information direct    Notwithstanding components networks        indirect functional protein&#226;&#8364;&#8220;protein associations. having extensively identi&#239;&#172;?ed eukaryote phylogeny       protein identi&#239;&#172;?ed according gene HUGO <span id='am-25' about='oboInOwl:id' typeof='owl:Thing'>ID</span> <br /> (15,16), data available evolutionary         (25) Ensembl <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>Peptide</span> ID (26) (Supplementary <br /> scenario functionally linked apoptosis genome-         Table S1). results search saved data <br /> stability gene network (5,17,18). approach assess       &#239;&#172;?les describing links genes handled role component given interacting network      Medusa software (27). comparative genomics. Using studied <br /> models, yeast mouse, comparative genomics pro-          Parsimony analysis: inferring evolutionary roots human <br /> vides powerful tools draw evolutionary inferences       apoptosis genome-stability genes <br /> poorly studied organisms (16). <br />    previous paper characterized entanglement       parsimony analysis divided major steps apoptosis genome-stability pathways             order construct parsimonious scenarios individual <br /> human protein&#226;&#8364;&#8220;protein-association network (19).           sets orthologous, given species tree. &#239;&#172;?rst built extend characterization build platform         consensus phylogeny eukaryotes listed STRING <br /> transfer functional information organisms         database (22). eukaryote phylogeny based human. idea based consensus         manual integration variety phylogenies (28&#226;&#8364;&#8220;33). <br /> component gene/protein interaction network         determined presence homologs present living organisms origin point   organisms species tree 180 genes apoptosis <br /> evolution. scenario gives rise present    genome-stability networks. Likely homology network tracked searching root           inferred using orthology information eukary- component given species tree.                        otic clusters orthologous groups proteins (KOGs) <br />    goal create orthology map        (21), retrieved orthology assign- <br /> species tree human apoptosis genome-stability      ments STRING server; STRING augmented <br /> gene/protein-association network order transfer       KOG orthology information adding additional <br /> humans information described eukaryotes.         species (currently 35 eukaryotes) creating searched orthologs  e. homologous genes derived       groups (NOGs, nonsupervised orthologous groups) single ancestral gene common ances-         giving direct association domain <br /> tor (LCA) compared species (20)] 35 fully             phylogeny. total, 142 eukaryotic orthologous groups <br /> sequenced eukaryotic genomes. Likely orthology             identi&#239;&#172;?ed (Supplementary Table S1). benchmark <br /> inferred orthologous groups using STRING data-            analysis, retrieved orthologous groups <br /> base (21,22), set orthologs       set genes using Inparanoid database, parsimonious scenario eukaryote phylogeny          discussed later. <br /> (23). verify orthology data, reconstructed         second major step reconstruction entire analysis using Inparanoid database di&#239;&#172;&#8364;erent        evolutionary scenario individual set ortho- <br /> data source, essentially obtained results         logous genes. problem previously formu- <br />  Supplementary materials). network char-        lated follows (23): given species tree set acterizations, estimated gene plasticity measuring       orthologs particular phyletic pattern, &#239;&#172;?nd gene abundance distribution orthologous            parsimonious mapping set orthologs <br /> groups extant species, considered essenti-       tree. Precisely, concerning problem, ality data available yeast mouse orthologs.       question restated  orthologous <br /> plasticity essentiality information transferred    group associated human apoptosis                                                                          Nucleic Acids Research, 2008, Vol. 36,  19 6271 <br />  <br />                                                                              P <br /> genome-stability genes, &#239;&#172;?nd earliest ortholog          p&#195;&#176;i;  &#195;&#382; &#194;&#188; 1. normalized Shannon infor- <br /> eukaryote phylogeny.                                              mation function H  de&#239;&#172;?ned   incongruence evolutionary scenario resolved according gain/penalty approach (23),                       1 X <br />                                                                   H  &#194;&#188;            p&#195;&#176;i;  &#195;&#382; ln p&#195;&#176;i;  &#195;&#382;;                  3 parsimonious scenario presence/absence                  ln M genes ancestral nodes tree obtained using inconsistency function de&#239;&#172;?ned                      divided ln(M) order normalize                                                                   quantities, guaranteeing 0   H    1. Observe S &#194;&#188; l &#195;&#190; g ;                                                  1    gene organism, N  = M, p  ) = 1/M, <br />                                                                   H  = 1. fact, H  re&#239;&#172;&#8218;ects spread distribu- l number gene losses,   number          tion s  ), e. measures diversity exists gene gains g gain penalty. di&#239;&#172;&#8364;erent           ath OG. H  near 0 indicates poor diversity, H  <br /> scenario function S calculated parsimo-         close 1 suggests high diversity. complementary <br /> nious scenario chosen yields mini-         quantity, estimate abundance D  ath <br /> mum value S. relative costs evolutionary            OG simply obtaining ratio number events consider cost units gene birth gene             items (orthologous genes) number organisms. <br /> acquisition  e. g = 2), cost unit gene loss. ratio proposed Mirkin coworkers (23).              Transference functional information yeast Subsequently, works validate 2:1 ratio pro-          mouse human gene/protein-association network <br /> karyotes (34,35) used similar <br /> analysis eukaryotes prokaryotes (36&#226;&#8364;&#8220;38).           predict developmental essentiality human gene, details corresponding evolutionary scenario       used mammalian phenotype information corre- <br /> orthologous groups presented Supplementary                 sponding mouse orthologs. analysis, gene Figures S14&#226;&#8364;&#8220;S49 provided spreadsheet                  de&#239;&#172;?ned &#226;&#8364;&#732;essential&#226;&#8364;&#8482; organism development format (Supplementary Table S3).                                  knock mouse ortholog confers embryonic   verify robustness orthology analysis           perinatal lethality (44). obtained mouse pheno- <br /> compared gene evolutionary scenario corre-            type data curated knock collection available <br /> sponding obtained using di&#239;&#172;&#8364;erent data source.            Mouse Genome Database (MGD) (http://www. case, reconstructed entire evolutionary analysis      informatics.jax.org) (45). predict cellular essentiality <br /> considering Inparanoid database (39). contrast          human gene, used phenotype information KOG algorithm, Inparanoid designed &#239;&#172;?nd orthologs            corresponding yeast orthologs. analysis, paralogs species separate            human gene de&#239;&#172;?ned &#226;&#8364;&#732;essential&#226;&#8364;&#8482; cellular level paralogs paralogs. KOG Inparanoid orthol-            knock ortholog confers lethality yeast. <br /> ogy analysis lead roughly conclusions.             yeast knock data obtained present discuss results Supplementary                Saccharomyces SGD project &#226;&#8364;&#732;Saccharomyces Genome <br /> Material Online (Supplementary Figures S3&#226;&#8364;&#8220;S6, S50&#226;&#8364;&#8220;S94             Database&#226;&#8364;&#8482; (http://www.yeastgenome.org/) (46). Human Table S4).                                                    yeast orthology veri&#239;&#172;?ed using data source <br />                                                                   Inparanoid database (47) provided                                                                   Supplementary Table S1. analysis, essential <br /> Diversity analysis orthologous groups                          genes, 32, listed orthologs using <br />                                                                   Inparanoid  genes presented Figure 5A orthologous group (OG) corresponds set                 asterisk names). <br /> genes di&#239;&#172;&#8364;erent extant species common <br /> gene ancestor. obtain quantitative expression orthologous distribution  e. distribution            Human gene mutation statistics <br /> items orthologous group), measured                  data analysis genes obtained information content di&#239;&#172;&#8364;erent databases                 Cancer Gene Census (48). germline-mutated (STRING Inparanoid) using Shannon Information                 somatic-mutated genes retrieved Theory (7,40&#226;&#8364;&#8220;43) de&#239;&#172;?ned follows. Consider n             crossed list 180 genes study. identi- <br /> number selected OGs, representing ortho-           &#239;&#172;?ed 25 genes placed network-based model logous groups. OG labeled  = 1, . . ., n)      apoptosis genome stability (Supplementary Table S1). N  items (orthologous genes), distributed M                Genotype statistics germline genes located r <br /> possible organisms. Consequently, given OG           module analyzed XP mutation database <br /> de&#239;&#172;?ne s  number items given organ-       (http://www.xpmutations.org). representativeness ism   = 1, . . ., M), sum given adds      sample tested second database [Human <br /> N  probability p   N  items         gene Mutation Database&#226;&#8364;&#8221;HGMD (49)] regarded OG, randomly chosen belongs organism          reference mutation database (published) gene <br /> written                                                        lesions responsible human inherited diseases. Table 1 <br />                                                                   shows equivalent samples obtained             s&#195;&#176;i;  &#195;&#382;                                               HGMD XP database.  contains <br /> p&#195;&#176;i;  &#195;&#382; &#194;&#188;           ;                                        2 <br />               N                                                   limited gene information comparing (50). <br />  6272 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br /> Table 1. Allelic distribution genes placed r module according XP mutation database (Panel . Sample representativeness compared second databases (Panel B) <br />  <br /> Panel             Number Genotypes (%                                            Total genotypes          (Panel B) Entriesb <br />  gene            null/non-null         non-null/non-null          null/null                                   XP database            HGMD <br />  <br /> ERCC2               20   (43.5)            26   (56.5)               0   (0.0)           46                       76c                    48 <br /> ERCC3                3   (60.0)             2   (40.0)               0   (0.0)            5                        8                     11 <br /> ERCC4                0   (0.0)              7   (100.0)              0   (0.0)            7                       18d                    17 <br /> ERCC5                0   (0.0)              5   (100.0)              0   (0.0)            5                       10                     12 <br /> XPA                  6   (6.0)             94   (94.0)               0   (0.0)          100                      128e                    25 <br /> XPC                  0   (0.0)             13   (100.0)              0   (0.0)           13                       28f                    42 <br /> DDB2                 0   (0.0)              5   (100.0)              0   (0.0)            5                        8g                     8 <br />                     29   (16.0)           153   (84.1)               0   (0.0)          182                      276                    163   Data obtained XP mutations database (http://www.xpmutations.org) compiled according absence (null) presence (non-null) gene alleles. Null/non-null genotypes heterozygous, non-null/non-null genotypes include heterozygous homozygous. <br /> b <br />   number allelic records present XP mutations database compared second human inherited mutation database [Human gene <br /> Mutation Database &#226;&#8364;&#8221; HGMD (49)] order attest sample representativeness. <br /> c <br />   allele duplicated database  XP1BR entry). <br /> d <br />   alleles mutation data (XP80TO, XP81TO XP89TO entries). <br /> e <br />   allele zygosity information (XP10OS entry). <br /> f <br />  alleles zygosity information (XP6BR, XP4BR, XP3BE XP22BE entries). Polymorphisms considered analyses. <br /> g <br />   allele duplicated (XP25PV entry). <br />  <br />  <br />   successfully retrieve zygosity infor-                  clusters orthologous groups proteins (KOGs) (21), <br /> mation accessing XP database.                                      using orthology assignments STRING server <br />                                                                             (22). total, apoptosis genome-stability genes                                                                             distributed 142 KOGs ortho- <br /> RESULTS <br />                                                                             logous groups parsimonious mapping <br /> Apoptosis genome-stability gene set                                     eukaryote phylogeny. Figure 2A present analysis begins list 180 genes participating               topology species tree used analysis <br /> human apoptosis genome-stability functions pre-                      (28&#226;&#8364;&#8220;33), arranged 17 subdivisions (monophy- <br /> viously de&#239;&#172;?ned (19) provided supplementary mate-                     letic groups) based phylogenetic relationships. rial online (Supplementary Table S1). de&#239;&#172;?ne gene                    species-tree node (STN) labeled according ascend- <br /> set characterized genome-maintenance                        ing subtree, referred LCA subset. <br /> mechanisms interactions com-                        quantitative view evolutionary roots <br /> ponents. Figure 1A reproduce interactions                    inferred 180 human genes studied  plotted <br /> illustrate links apoptosis genome-stability                 number human apoptotic genome-stability <br /> gene networks, collectively referred                    orthologs STN (Figure 2B). Accordingly, genome-maintenance gene network. node corre-                           distribution suggests sequential enlargement net- <br /> sponds gene-network node (GNN), lines                        work, progressive increase apoptosis.  <br /> represent direct (physical) indirect (functional)                    trast, genome-stability orthologs mainly rooted associations according STRING database human.                        STN-P  base eukaryote species tree), suggesting derived high-quality systematic protein&#226;&#8364;&#8220;                      orthologs involved apoptosis recent. <br /> protein interaction mapping (22). Note position                      Furthermore, order assess robustness TP53 gene network topology connecting apoptosis                      orthology analysis reconstructed entire evolution- 18 genome-stability components (Figure 1A, arrow,                        ary scenarios using Inparanoid database di&#239;&#172;&#8364;erent Figure 1B). functional overlap empha-                   data source, essentially obtained results. sized Figure 1C complete network, shows                    contrast KOG algorithm, Inparanoid designed &#239;&#172;?nd number links distributed gene set.                          orthologs paralogs species apoptosis genome stability equivalent                     separate paralogs paralogs (39). used <br /> number components network  e. 86:100),                     second approach construct evolutionary incon- <br /> connectivity 2-fold, indicated                sistence score (R) estimates divergence Venn diagram. di&#239;&#172;&#8364;erence arises mainly                   scenarios  e.  STN). present discuss <br /> large number associations NER, MMR                             results Supplementary Material Online (Supple- <br /> chromosome stability components, yielding highly                      mentary Figures S3&#226;&#8364;&#8220;S6, S50&#226;&#8364;&#8220;S94 Supplementary <br /> nected gene module (Figure 1A, r).                                          Table S4). Brie&#239;&#172;&#8218;y, apoptosis genes, R = 1.709 STNs <br />                                                                              0.224 (SE) genome-stability genes R = 0.807 <br /> Construction parsimonious evolutionary scenarios                         STNs  0.202 (SE) (Figure 2C). means order infer ancestral states human apoptosis                   root inferred analyses, estimated error apo- genome-stability genes considered eukaryotic                         ptosis approximately STNs                                                                                                                        Nucleic Acids Research, 2008, Vol. 36,  19 6273 <br />  <br />  <br />                                                                                                       TNFRSF10D <br />                                                                                                                                         B <br />                                                                                                               TNFRSF10C                                            CHUK <br />                                                   DCAMKL1 <br />                                                                                                                                                                            HMGB1 <br />                 PIK3CG                                                     TNFRSF10A <br />  CAPN2   NGFB                                                     PAK7                                                                                                               BRCA2 <br />                                                                                FAS                                                                                   ATR <br />                                                                                            TNFSF10                                         NFKBIA                           XRCC6 <br />                              CAST                                                                                                                                       ATM <br />                                                                    FASLG                                                                                  IKBKB <br />                                        IRAK4 <br />                                                IRAK2    MYD88 <br />                                                             TRADD                                                                         BCL2L1                        PARP1 BRCA1 PRKDC    APEX1 <br />                          NTRK1                                           RIPK1                 DFFB <br />             CAPNS1                                         IRAK1CASP10       BIRC4 TNFRSF10B                 SHFM1                                 BCL2                          CHEK2 <br />                    PIK3R5         PIK3CB <br />                                             PIK3C3 <br />                                                    PIK3R1 <br />                                                          TNF            BIRC2 <br />                                                                                                                     PNKP <br />                                                                                                                                                                         TP53 <br />                                      IL1RAP                                                                                                                 CHEK1                    RPA1 RAD51 <br />                                                              IL1A CASP8        FADD CASP6                              PMS2 <br />                  PIK3R3        CAPN1                                  BIRC3                                                                         BAX <br />                                         PIK3CA           IKBKG                                                              LIG3 <br />                                                MAP3K14                                                  LIG4 <br />                                                                                                                                  PMS2L3 <br />                                                                                                                                                                   ERCC6 <br />                               PIK3C2A                  TRAF2 NFKB2NFKB1 CHUK             PARP2 RAD1         DFFA <br />                     PIK3CD                   IL1B                                 CASP7             MBD4              DCLRE1C <br />                                                              IL1R1                         HMGB1               BRAP                                                     MNAT1 ERCC3 <br />                                PIK3R4                                   CFLAR <br />                                                                                   CASP3                                       OGG1 EME1 <br />                                             AKT1 AKT2 BID                                             RAD50 <br />                                                                                                                      XRCC4                                         GTF2H1 <br />                     PIK3R2                                      TNFRSF1A                    NBN                                                                                ERCC2 <br />                                                  AKT3                      CASP9                     BRCA2                         MLH3 <br />                                                            APAF1                  ATR     XRCC6            XRCC1           EXO1 <br />                  PIK3C2G      PIK3C2B                                RELA <br />                                             IL3RA                                  ATM            PRKDC          MLH1                                                     CCNH <br />                  CSF2RB                               NFKBIA   IKBKB                       BRCA1         MSH2                 RAD54L <br />                           PRKAR2B                                                                                  TOPBP1 <br />                                                                BCL2                            RAD17                              SMUG1 <br />                                                       BCL2L1                CYCS PARP1               LIG1 XRCC5 APEX1 MUS81 <br />                            PRKAR1B <br />                                        IL3 BAD                                        MRE11A               MSH6                    MPG    C <br />                  PRKACB                                            CHEK1                          CHEK2          RAD51 <br />                             PRKACA                                       PPP3CA                                         MSH3 <br />                                                               BAX                                          UNG                MSH4 <br />                     PRKACG                                                                   RFC1                         FEN1 RAD54B <br />                               PRKAR2A <br />                                                                                                 RPA2 RPA1 POLE                                                          Genomestability <br />                                                                    PDCD8                RFC3                      MUTYH ORC1L <br />                                                                                                    PCNA                           TDG         Apoptosis <br />                                            PPP3CC                                                           POLD3 <br />                                                             PPP3R1              ERCC6 RFC5                                 APEX2 <br />                                                     PPP3CB           TP53                    RFC4 RPA3 <br />                                                                                                         POLD2 <br />                                                                                                                 POLD4 <br />                                                                                                                        NTHL1 <br />  <br />                                                                                                  &#207;?               RAD52 DMC1 MSH5 <br />                                                                                 ERCC8 <br />                                                                                       MNAT1ERCC3          POLD1 <br />                 Apoptosis                                                                                     CDC6 <br />                                                                                   GTF2H1             ERCC5            RAD23B <br />                                                                                           CCNHERCC2      XPA <br />                 Chromosome stability                                                                              XPC <br />                                                                                     GTF2H3                  XAB2 <br />                                                                                        CDK7      GTF2H2          DDB2 <br />                 Recombinational repair                                                     ERCC4 <br />                                                         Genome                                      GTF2H4 DDB1                                    355            112             708 <br />                 Mismatch repair                                                               ERCC1                               CETN2 <br />                                                          stability <br />                 Base-excision repair <br />                                                                                  GTF2H5 <br />                 Nucleotide-excision repair                                                                                 RAD23A <br />  <br />  <br />  <br />  <br /> Figure 1. Human apoptosis genome-stability gene network.   Graph interactions genes involved apoptosis DNA repair <br /> pathways, previously characterized Castro et al. (19). Di&#239;&#172;&#8364;erent pathways represented di&#239;&#172;&#8364;erent colors. Network nodes color represent genes participating pathway. Gene IDs pathway provided Supplementary Table S1. (B) Magni&#239;&#172;?cation TP53 gene position network topology. highlights functional overlap TP53, linking apoptosis genome-stability <br /> components. (C) Venn diagram showing distribution links apoptosis genome-stability pathways. overlapped area corresponds links connecting systems. large number associations NER, MMR chromosome-stability components designed r module. <br />  <br />  <br /> rooting point species tree, genome stability                                              nodes according roots inferred species tree error approximately STN                                                             (Figure 3D). Note gene-network node <br />    order test phylogeny Caenorhabditis ele-                                                  (GNN) represents ortholog gene human <br /> gans root metazoa included                                                         apoptosis genome-stability gene network. <br /> Nematostella vectensis, changes base                                                        orthology information regarding STNs metazoa (Supplementary Figure S9). chose organ-                                                    provided Supplementary Table S1. quantitatively <br /> ism   Nematostella cnidarian; (ii) idea                                                 showed Figures 2B&#226;&#8364;&#8220;D, recent STNs concen- cnidarians base metazoa                                                 trate apoptosis roots (round GNNs Figure 3A B). <br /> troversial nematodes; (iii) switching taxon                                                  qualitative di&#239;&#172;&#8364;erence: observe like goes way testing e&#239;&#172;&#8364;ect phylo-                                                 pooled origins inferred components apo- <br /> geny used. result process roots                                              ptosis extrinsic (Figure 3A) intrinsic (Figure 3B) human genes remain  complete                                                       pathways. <br /> analysis available Supplementary Table S5)                                                          analyze result important consider discussed supplements (section 1.4: deep                                                    biochemical signature apoptosis,  caspase <br /> root metazoans).                                                                                        activation, triggered intrinsic                                                                                                            extrinsic apoptosis pathways. intrinsic pathway species-tree nodes gene-network nodes                                                              associated mitochondrial outer membrane permeabi- assess details evolutionary scenario described                                               lization cytochrome c (CYCS) release response pri- <br /> earlier context known predicted gene func-                                                   marily developmental cues cellular damage. tions, used network-based model presented                                                        triggers apoptosis Bcl-2 gene family Figure 1A (19).                                                                                            initiator protease caspase-9. contrast, extrinsic <br />    Starting complete network graph generated                                                   pathway characterized ligation cell surface relevant orthology projections characterize                                                   receptors speci&#239;&#172;?c death ligands, TNF gene prod- <br /> functional di&#239;&#172;&#8364;erences apoptosis genome sta-                                                    uct, generate catalytically active caspase-8 (51,52). bility (Figure 3A&#226;&#8364;&#8220;C). graphs highlighted                                                   protein encoded TNF gene multifunctional <br />  6274 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br />                                                                                                                    a1 Homo sapiens <br />                                                                                                              B         a2 Pan troglodytes <br />                                                 STN          LCA                                         C             b Macaca mulatta <br />                                                                                                         D <br />                                                 B            Primate                                E                  c1 Rattus norvegicus <br />                                                                                              F <br />                                                 F            Mammalia                       G                          c2 Mus musculus <br />                                                                                            H <br />                                                            Vertebrata                                                d Canis familiaris <br />                                                 J            Chordata                                                e   Bos taurus <br />                                                                                    J <br />                                                 L            Metazoa                                                   f   Monodelphis domestica <br />                                                 M            Opisthokonta LK                                           g Gallus gallus <br />                                                 P-Q          Eukarya                                                   h Xenopus tropicalis <br />                                                                                                                        i1 Takifugu rubripes <br />                                                                                                                        i2 Tetraodon nigroviridis <br />                                                                                                                        i3 Danio rerio <br />                                                               M                                                        j   Ciona intestinalis <br />                                                              N                                                         k1 Drosophila melanogaster <br />                                                         O                                                              k2 Anopheles gambiae <br />                                                   P                                                                    k3 Apis mellifera <br />                                                                                                                        l   Caenorhabditis elegans <br />                                                 Q                                                                      m1 Kluyveromyces lactis <br />                                                                                                                        m2 Saccharomyces cerevisiae <br />                                                                                                                        m3 Candida glabrata <br />                                                                                                                        m4 Eremothecium gossypii <br />                                                                                                                        m5 Debaryomyces hansenii <br />                                                                                                                        m6 Yarrowia lipolytica <br />                                                                                                                        m7 Aspergillus fumigatus <br />                                                                                                                        m8 Schizosaccharomyces pombe <br />                                                                                                                        m9 Filobasidiella neoformans <br />                                                                                                                        m10 Encephalitozoon cuniculi <br />                                                                                                                        n Dictyostelium discoideum <br />                                                                                                                        o1 Arabidopsis thaliana <br />                                                                                                                        <span id='am-26' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="O2" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>o2</span> Cyanidioschyzon merolae <br />                                                                                                                        p1 Plasmodium falciparum <br />                                                                                                                        p2 Cryptosporidium hominis <br />                                                                                                                        p3 Thalassiosira pseudonana <br />                                                                                                                        q Giardia lamblia <br />  <br />                                        B 125                                                                                    C <br />                                                                                                                                 inconsistency score (R) <br />                                                                                                                                                           2.0 <br />                                                               D E-G        H           J KL <br />                                                        100                                               MN                                               1.6 <br />                                        Orthologs (n) <br />  <br />  <br />  <br />  <br />                                                                                                                                     Evolutionary <br />                                                                                                                   O <br />                                                                                                                    P <br />                                                         75 D E-G          H                                                                           1.2 <br />                                                                                            JK <br />                                                         50                                      L                        Q                                0.8 <br />                                                                     Apoptosis                            MN <br />                                                         25                                                 OP                                             0.4 <br />                                                                Genome stability                                          Q <br />                                                          0                                                                                                0.0 <br />                                                                                                                                                                          b. <br />                                                                                                                                                                  <br />  <br />  <br />                                                          0.00      0.05    0.10    0.15   0.20   0.25 <br />                                                                                                                                                                os <br />  <br />                                                                                                                                                                         ta <br />                                                                                                                                                               pt <br />  <br />                                                                                                                                                                     .S <br />  <br />  <br />  <br />  <br />                                                                 Evolutionary distance STNs <br />                                                                                                                                                                                                                                                                                                                              po <br />                                                                                                                                                                 G <br />                                                                                                                                                            <br />  <br />  <br />  <br /> Figure 2. Inferring evolutionary roots human apoptosis genome-stability genes.   Eukaryote species tree topology used parsimony <br /> analysis. phylogenic relationship 35 eukaryotes based manual integration variety phylogenies (28&#226;&#8364;&#8220;33). STNs corresponding LCA indicated. (B) Distribution apoptotic genome-stability orthologs according roots inferred species tree plotted function divergence STNs (based branch-length estimates). Supplementary Material Online exempli&#239;&#172;?ed parsimony analysis. evolutionary distances computed using protein families regarded conserved distant taxa described able reconstruct domain phylogeny: 40S ribosomal proteins, translation initiation factor 5A proteins Flap structure- <br /> speci&#239;&#172;?c endonuclease 1 proteins (73). proteins used analysis aligned Supplementary Figures S10&#226;&#8364;&#8220;S12. distances expressed fraction sites di&#239;&#172;&#8364;er branches multiple alignment, approximation branch-length separates STNs. <br /> (C) Divergence KOG Inparanoid-derived scenarios. apoptosis genes, R = 1.709 STNs  0.224 (SE) genome-stability genes <br /> R = 0.807 STNs  0.202 (SE). means root inferred analyses, estimated error apoptosis approximately STNs rooting point species tree, genome stability error approximately STN  <br />  <br />  <br />  <br /> proin&#239;&#172;&#8218;ammatory cytokine belongs tumor                                                                extrinsic pathway, ligands receptors necrosis factor (TNF) superfamily, includes                                                           rooted STN projection, later (e.g. IL1A, IL3RA, ligands FAS (FASLG) TRAIL (TNFSF10).                                                                     IL3 TNFRSF10D genes observed mam- ligands bind members TNF-                                                                    mals,  STN-F later, evinced comparing <br /> receptor superfamily (e.g. TNFRSF1A, TNFRSF10A,                                                                  STN projection versus complete human network; details <br /> TNFRSF10B FAS receptors) involved                                                             orthologs presented explicit parsimony <br /> regulation wide spectrum biological processes,                                                      analysis&#226;&#8364;&#8221;Supplementary Figures S46, S48 S49). immune surveillance, innate immunity, haematopoiesis                                                             STN-P projection (Figure 3C),  small tumor regression  review, (53)].                                                                     fraction genes belongs apoptosis. Instead, graph <br />    Accordingly, noticeable components                                                          remarkable large presence genome-stability <br /> intrinsic pathway rooted mainly STN-L earlier                                                          components (triangular GNNs), quantitatively <br /> (e.g. CYCS deeply rooted eukaryote species tree&#226;&#8364;&#8221;                                                           addressed Figure 2B. <br /> Figure 3C). contrast, subsequent enlargement                                                          Taking results  evolutionary scenario network graph provided mainly orthologs                                                             genome-maintenance mechanisms marked                                                                                                           Nucleic Acids Research, 2008, Vol. 36,  19 6275 <br />  <br />  <br />                                                                                                       B <br />                                                    TNFRSF10A <br />       NGFB                                           FAS <br />                                                                TNFSF10 <br />                                            FASLG <br />                                         TRADD                                                                  NTRK1 <br />                                                       TNFRSF10B                                                                    PIK3R1CASP10                          SHFM1 <br />                                    TNF <br />                                                                                                         PIK3R3       IL1RAP               CASP8 <br />                                                      FADD                                                                                                  CASP6 <br />                                                                                                                                                    CHUK    CASP7 <br />                                                                          DFFA <br />                            IL1B       IL1R1                                                                                                       CFLAR <br />                                           TNFRSF1A <br />                                                                                 XRCC4                       PIK3R2 <br />                                                                                                                                                   CASP9 CASP3 <br />                                                                                                                                              IKBKB <br />                                                                                                                                     BCL2L1 <br />  <br />  <br />                             BAD                                                             MPG                                          BCL2 <br />                                                      TP53 <br />  <br />  <br />  <br />  <br />                                                                  &#207;?                                                                                                 &#207;? <br />                   GS APO                                                                                      GS APO <br />                            GNNs rooted STN                                                                          GNNs rooted STN-L <br />                            GNNs present STN                                                                         GNNs present STN-L <br />  <br />  <br />  <br />  <br />   C                                                                                                     D <br />   C APN2 <br />                                                                                                                                  Species-treenodes <br />                                                                                                                                  (STNs) <br />                                                                                                                                                                    a1 <br />                                                                                                                                                                    a2 <br />                                                                                                                                                                    b <br />                                                                                                                                                                    c1 <br />                                                                                    PNKP                                                     STN                  c2 <br />                      C APN1                                                                                                                 (Vertebrata)           d <br />                                                                          LIG4                                                          STN-L                       e <br />                                                                                                                                                                    f <br />                                                              HMGB1                 DCLRE1C                                             (Metazoa)                   g <br />                                                                             BRAP         OGG1 <br />                             AKT1 AKT2                                                                                                                              h <br />                                                                         BRCA2                                                                                      i1 <br />                                                                XRCC6                          MLH3                                                                 i2 <br />                                AKT3                                                                                                                                i3 <br />                                                                                           RAD54L                                                                   j <br />                                                     CYCS            RAD17                                                                                          k1 <br />                                                                                         MUS81                                                                      k2 <br />                                                                  CHEK2                                                   STN-P                                     k3 <br />                                                                              MSH3                                                                                  l <br />                                                                       UNG       MSH4 <br />                                                                                     RAD54B                             (Eukarya)                                   m1 <br />                                                                  RPA2    MUTYH                                                                                     m2 <br />                                                                     RPA1        ORC1L                                                                              m3 <br />  <br />                                                                     &#207;? <br />                                                                                                                                                                    m4 <br />                                                                         POLD2                 MSH5                                                                 m5 <br />                                                 ERCC8       MNAT1                                                                                                  m6 <br />                                                                     ERCC5                                                                                          m7 <br />                   GS APO                           GTF2H1    CCNH                    RAD23B                                                                        m8 <br />                                                                          XAB2                                                                                      m9 <br />                            GNNs rooted STN-P                            XPC                                                                                     m10 <br />                                                     GTF2H3                DDB2                                                                                     n <br />                            GNNs present STN-P                        DDB1                                                                                       o1 <br />                                                                                                                                                                    o2 <br />                                                             ERCC1 GTF2H4                                                                                           p1 <br />                                                                                           RAD23A                                                                   p2 <br />                                                                                                                                                                    p3 <br />                                                                                                                                                                    q <br />  <br />  <br /> Figure 3. STNs gene-network nodes (GNNs). Orthology projection genes rooted STN  , STN-L (B) STN-P(C). Roots ortholog: color nodes; presence ortholog: white nodes. location STNs species tree indicated (D). Supplementary <br /> Material Online provide examples orthology projection approach (Supplementary Figure S2) compared Inparanoid <br /> evolutionary scenarios di&#239;&#172;&#8364;erent orthology data source (Supplementary Figures S3&#226;&#8364;&#8220;S6). <br />  <br />  <br />  <br /> major functional increments: &#239;&#172;?rst evolution                                          GNNs placed periphery networks <br /> genome-stability gene network, components origi-                                             represent recent evolutionary innovations.  <br /> nate basal position species tree [STN-P,                                            fore, striking feature graphs increasing <br /> inconsistency datasets R = 0.63 STNs  0.22                                                 association apoptosis genome-stability func- <br /> (SE)]; second appearance apoptotic                                           tions emergence entangled gene network, <br /> intrinsic components, rooted near metazoan divergence                                              fully consistent evolutionary strategy <br /> [STN-L, inconsistency datasets R = 1.23 STNs                                               used eukarya adding complexity existing core <br />  0.23 (SE)]; consists network enrichment                                          systems (54,55). (Inparanoid database essentially produces apoptotic extrinsic members happens                                               evolutionary scenario; Supplementary <br /> near chordate-vertebrata root [STN  inconsistency                                                Figure S6.) datasets R = 0.35 STNs  0.16 (SE)]. net-                                                   additional evidence ancestral roots work core apoptosis genome-stability systems                                            genome stability inferred considering likely <br /> rooted tree divergence metazoans,                                            origin ancestral eukaryotic KOGs identifying <br />  6276 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  closest prokaryotic orthologous groups (COGs).           abundance (Figure 4C). &#239;&#172;?rst class   refers genes KOG COG correspondence presented                  placed orthologous groups low diversity low <br /> Supplementary Figure S8, shows 77.0%           abundance (Figure 4A B, white GNNs; Figure 4C, <br /> genome-stability orthologs identi&#239;&#172;?able prokaryotic        white diamonds). means organisms present <br /> orthologous groups, 39.5% apoptotic ortholo-       orthologs, associated orthologous groups <br /> gous genes.                                                    components. implies recent origin <br />    Despite organisms consid-        GNNs,   present humans, ered, construction gene network directed      end species tree; (ii) present human.  interpretation evolutionary       extant species. example, TP53 FAS scenarios ultimately linked characterization   origins STN  shown Figures S38 S40 human gene network. means infer         Supplementary Material Online. class genes gene network actual organism root     located region network plastic <br /> eukaryotes smaller. stated introduc-     accept new genes. second (b) refers tion section, goal create orthology map    genes placed orthologous groups high diversity species tree order transfer human informa-    low abundance (Figure 4A B, black GNNs; <br /> tion described eukaryotes. way         Figure 4C, black diamonds), indicating small number <br /> strategy, explored subsequent sections.        genes organism, present di&#239;&#172;&#8364;erent spe- <br />                                                                cies. genes located ancient region Plasticity analysis                                            network. implies poorly plastic genes, highly  <br />                                                                served species. class (c) refer Genetic plasticity understood ability       genes placed orthologous groups high diversity <br /> functional gene network tolerate changes compo-      high abundance (Figure 4A B, red GNNs; <br /> nents. di&#239;&#172;&#8364;erent sources changes (gene       Figure 4C, red diamonds), clearly requires high <br /> duplication, gene loss, mutations horizontal gene          plasticity. Note red white GNNs (plastic <br /> transfers), di&#239;&#172;&#8364;erent causes e&#239;&#172;&#8364;ects. changes     GNNs) segregated black GNNs (poorly plas- genome naturally selected,            tic) network. segregation expected <br /> depending e&#239;&#172;&#8364;ect cell &#239;&#172;?tness      plasticity characteristic set inter- <br /> organism viability, case complex organisms.      acting genes characteristic individual <br /> result evolutionary dynamics genetic varia-      gene. Figure 4D supports &#239;&#172;?nding showing bility organisms species  ultimately,     relative presence classes genes speciation. Gene networks equally plastic          STNs: recent genes network emerge equally respond variation pressures:     highly plastic regions network, depending gene, function, in&#239;&#172;&#8218;uence         ancient ones located poorly plastic regions. <br /> genes, relevance, changes likely          Observe inhomogeneous distribution white, tolerated selected                        red black GNNs network graph re&#239;&#172;&#8218;ects    Focusing networks general, expect        function performed genes. white red <br /> gene networks tolerant variation         GNNs clearly populating apoptosis network, black <br /> present larger variability inside species        GNNs placed mainly genome stability. result <br /> species. Focusing individual genes, organisms           suggests high evolutionary conservation genome- tolerant drastic changes (e.g. gene          stability orthologs  e. class b, orthologs present knock  change performed genes located       organisms variants), contrasting apo- plastic network. characteristics,      ptosis GNNs concentrate plasticity network <br /> gene variability di&#239;&#172;&#8364;erent genomes organ-          e. class c orthologs variants organisms). <br /> ism response knock single genes, allow inde- <br /> pendent measures estimate gene network plasticity.      Essentiality Saccharomyces cerevisiae. second, inde- <br /> possible plasticity measure estimating number       pendent plasticity measure obtained assessing cell distribution orthologs di&#239;&#172;&#8364;erent organisms.        lethality data. considered eukaryotic model second, independent plasticity measure obtained       Saccharomyces cerevisiae available Saccharomyces assessing cell lethality data. follows present   Genome Database (SGD) (46). transferred infor- discuss plasticity measures.                     mation STN representing LCA yeast                                                                human  e. STN-M), projected cor- <br /> Diversity abundance analysis. evaluated diver-      responding human network topology. yeast results <br /> sity abundance orthologous groups estimate       showed Figure 5A. Observe essential genes plasticity gene human apoptosis         concentrated speci&#239;&#172;?c portion network <br /> genome-stability gene network (precise de&#239;&#172;?nition        (blue GNNs) corresponding lower plasticity area <br /> Materials methods section exempli&#239;&#172;?ed           showed Figure 4 (black GNNs . Furthermore, Supplementary Material Online).                             likely orthology inferred LCA yeast human <br />    network graph presented Figure 4A B              indicates yeast lost genes course <br /> incorporates diversity abundance statistics, allow-        evolution, mainly apoptotic genes (white GNNs <br /> ing discrimination distinct classes genes      Figure 5A). loss, presence <br /> based distribution diversity function        essential genes overlaid genome-stability area <br />                                                                                                                                          Nucleic Acids Research, 2008, Vol. 36,  19 6277 <br />  <br />  <br />  <br />                                                                                                                                                                        TNFRSF10D <br />                                                       Apoptosis <br />                                                                                                                                                                                 TNFRSF10C <br />                                                                                                                DCAMKL1 <br />  <br />                                             PIK3CG                                                                                     TNFRSF10A <br />                      CAPN2           NGFB                                                                                    PAK7 <br />                                                                                                                                            FAS <br />                                                              CAST                                                                                        TNFSF10 <br />                                                                                                                              FASLG <br />                                                                                                IRAK4 <br />                                                                                                        IRAK2         MYD88 <br />                                                                                                                          TRADD <br />                                                         NTRK1                                                                       RIPK1                    DFFB <br />                                          CAPNS1 <br />                                                                                                IRAK1 CASP10        BIRC4 TNFRSF10B                    SHFM1 <br />                                               PIK3R5                PIK3CB            PIK3R1                BIRC2 <br />                                                                               PIK3C3         TNF                                                            PNKP <br />                                                                        IL1RAP                         IL1A                                                                    B <br />                                                                                                        CASP8          FADD                                     PMS2 <br />                                               PIK3R3                                            IKBKG                         CASP6 <br />                                                                  CAPN1 <br />                                                                             PIK3CA                           BIRC3                                                  LIG3 <br />                                                                                    MAP3K14                                  CASP7              LIG4 <br />                                                                PIK3C2A                                NFKB2       CHUK           PARP2 RAD1        DFFA                   PMS2L3 <br />                                                                                           TRAF2 <br />                                                                                IL1B                       NFKB1                            MBD4 <br />                                                  PIK3CD                                           IL1R1                                                       DCLRE1C <br />                                                                                                                                   HMGB1                BRAP <br />                                                                 PIK3R4                               TNFRSF1A CFLAR CASP3                                             OGG1 EME1 <br />                                                                                 AKT1 AKT2 BID                     CASP9               NBN RAD50 <br />                                                      PIK3R2                                     APAF1                                                        XRCC4 <br />                                                                                     AKT3                                                     BRCA2                          MLH3 <br />                                                                                          NFKBIA                         ATR        XRCC6          XRCC1           EXO1 <br />                                                  PIK3C2G       PIK3C2B                                     RELA <br />                                                                               IL3RA                                                        PRKDC        MLH1 <br />                                                                                                                           ATM <br />                                                  CSF2RB                                             IKBKB                          BRCA1        MSH2                     RAD54L <br />                                                           PRKAR2B                          BCL2L1                                                         TOPBP1 <br />                                                                                                    BCL2                                RAD17                                 SMUG1 <br />                                                                                                                    CYCS PARP1                LIG1XRCC5 <br />                                                             PRKAR1B                                                                                       APEX1 MUS81 <br />                                                                          IL3 BAD                                              MRE11A              MSH6                       MPG <br />                                                  PRKACB                                                 CHEK1                                           RAD51 <br />                                                              PRKACA                                                                       CHEK2                 MSH3 <br />                                                                                                               PPP3CA <br />                                                                                                                         TP53                                           MSH4 <br />                                                      PRKACG                                       BAX                               RFC1          UNG <br />                                                                PRKAR2A                                                                                           FEN1       RAD54B <br />                                                                                                                                         RPA2 RPA1 <br />                                                                                                                                 RFC3               POLE            ORC1L <br />                                                                                                        PDCD8                                               MUTYH <br />                                                                                                                                            PCNA                            TDG <br />                                                                               PPP3CC                                   ERCC6 RFC5                  POLD3             APEX2 <br />                                                                                                  PPP3R1                             RFC4 <br />                                                                                        PPP3CB                                            RPA3 POLD2 POLD4 NTHL1 <br />                                                                                                                       ERCC8                                                  MSH5 <br />                                                                                                                                                                &#207;? <br />                                                                                                                              MNAT1 ERCC3                 RAD52 DMC1 <br />                                                                                                                                                  POLD1 <br />  <br />                                                                                                                                              GTF2H1              ERCC5 CDC6 RAD23B <br />                                                                                                                                                        CCNH ERCC2   XPA <br />  C                                                               D                                                                                                          XPC <br />                                                                                                                                                GTF2H3                 XAB2 <br />                    1.2                                                                                                                            CDK7       GTF2H2 <br />                              (b)                                                                                                                                           DDB2 <br />                                                                  Orthologs (fraction) <br />  <br />  <br />  <br />  <br />                                                                                         1.00 <br />                                                                                                                                                       ERCC4 <br />   Diversity (H&#206;&#177;) <br />  <br />  <br />  <br />  <br />                                             (c)                                                                                                                GTF2H4 DDB1 <br />                    0.8                                                                  0.75                                                             ERCC1                                  CETN2 <br />                                                                                         0.50 <br />                    0.4                                                                                                                      GTF2H5 <br />                                                                                         0.25                                                                                                RAD23A <br />  <br />                    0.0                                                                  0.00 <br />                                                                                                B C D E F G H J K L MN O P Q                                    Genome stability <br />                       1.0               10.0            100.0 <br />                                    Abundance (D&#206;&#177;)                                                        Species-treenodes <br />  <br />  <br /> Figure 4. Plasticity analysis orthologous groups.  B) Diversity H  abundance D  orthologous groups overlaid apoptosis genome-stability gene network according categories de&#239;&#172;?ned C. (C) Distribution H  function D :   orthologous groups low <br /> diversity low abundance (white); (b) orthologous groups high diversity low abundance (black); (c) orthologous groups high <br /> diversity high abundance (red). (D) Fraction orthologous groups present STNs: orthologous groups low diversity low <br /> abundance (white dashed line); orthologous groups high diversity low abundance (black solid line); orthologous groups high <br /> diversity high abundance (red solid line). Supplementary Material Online provide examples diversity analysis. <br />  <br />  <br />  <br /> (blue triangular GNNs), indicates evolutionary                                                                       day 2;  knock alleles indicate &#226;&#8364;&#732;developmental <br /> scenario consistent plasticity measures shown                                                                     lethality&#226;&#8364;&#8482; &#226;&#8364;&#732;essentiality&#226;&#8364;&#8482; embryonic stem cells. Evidence Figure 4: lost genes represented plastic GNNs                                                                   mouse lethality obtained according frequency <br /> (red white symbols Figure 4).                                                                                          expected Mendelian genetics  e. zygosity allelic <br />                                                                                                                               distribution observed o&#239;&#172;&#8364;spring): signi&#239;&#172; Lethality Mus musculus. order complement                                                                        deviation expected frequency knock lethality measure complex multicellular eukaryotic                                                                     allele indicates lethality.  putative 178 <br /> model, assessed Mus musculus lethality data Mouse                                                                       Mus musculus orthologs identi&#239;&#172;?ed analysis, &#239;&#172;?nd <br /> Genome Database&#226;&#8364;&#8221;MGD (45). phenotypic statistics                                                                           124 genes knock data available (Supple- MGD database consider lethal allele causes                                                                        mentary Table S1). majority produced viable <br /> death anytime fertilization postnatal                                                                    phenotypes, 39 knock alleles associated <br />  6278 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />                                                                                                                                                              B                 CAPN2 <br />                                CAST* <br />  <br />  <br />                                                                                                                                                                                            CAPNS1 <br />                                                                                                                                                                                                                    PIK3CB <br />                                                                     IKBKG*                                                                                                                                                                              CASP8FADD <br />                                                                                                                                                                                                                        PIK3CA                                              LIG4      LIG3 <br />                                                     MAP3K14* <br /> Projection functional                                                                                                                                      Projection functional                                                  TRAF2              CHUK CASP7 <br /> data LCA human                                                                                                                                          data LCA human                                                                                HMGB1 <br />                                                                                                                                                                                                                                                         CFLAR         NBN RAD50 yeast                                                                                                                                                     mouse                                                                                                           XRCC4 <br />                                                                                              ATR                                                                          STN-C                                                                           CASP9 CASP3 <br />                                                                                                                                                                                                                                                                            BRCA2 <br />                                                                                                         PRKDC*                                                                                                                                    APAF1         ATR           XRCC1 <br />                                                                                                                                                                                                                                                     IKBKB <br />                                                                                                                              TOPBP1*                                                     Mouse <br />                                                                                                            LIG1                                                                          data                                                                       CYCS   BRCA1 <br />     STN-M                                                                                                                                                                                input                                      BCL2L1 <br />                                                                                                                                                                                                                                                                               RAD17 <br />                                                                                                       CHEK2                                                                                          PRKACA                                                 CHEK1 <br />                                                                                                   RFC1                                                                                                                                                                                  RAD51 <br />                       Yeast                                                                           RPA2 RPA1 <br />                                                                                                RFC3            POLE                                                                                                                                                                          FEN1 <br />                       data                                                                                                           ORC1L <br />                       input                                                                   RFC5 RFC4 PCNA                                                                                                                                      PPP3R1 <br />                                                                                                             POLD2 <br />                                                                                                  ERCC3       POLD1                                                                                                                                                     MNAT1 <br />                                                                                            MNAT1 <br />                                                                                                                CDC6 <br />                            GS APO                                                       GTF2H1 CCNH ERCC2                                                                                   GS APO                                                                                    XPA RAD23B <br />                                                                                                               XAB2                                                                                                                                                                      XAB2 <br />                                                                      Essentiality <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                          Essentiality <br />                                                                                         GTF2H3 <br />                                                 +               +                   +                                                                                                                              +                +                   + <br />                                                     Knock  <br />  <br />  <br />  <br />                                                                                                                                                                                                                         Knock                                                                                                    GTF2H2                                                                                                                                                                  ERCC2 <br />                                     Orthology <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                        Orthology <br />                                                 +               +                   &#226;&#710;&#8217;       CDK7                                      CETN2*                                                                       +                +                   &#226;&#710;&#8217; <br />                                                                                                         GTF2H4                                                                                                                                                                        DDB1 <br />                                                 &#226;&#710;&#8217;               &#226;&#710;&#8217;                   &#226;&#710;&#8217;                                                                                                                              +                &#226;&#710;&#8217;                   ?                    ERCC1 <br />  <br />  <br />  <br />  <br />                                                                           C <br />                                                                                                                                                                           FAS <br />  <br />  <br />  <br />                                                                                                              NTRK1 <br />  <br />                                                                                                                                                                   BIRC3 <br />                                                                                                                                                                                                       PMS2 <br />                                                                                                                                      PIK3CA <br />                                                                           Projection human                                                                 NFKB2 <br />                                                                           functional data                                                                                           NBN <br />                                                                                                                                                       AKT2 <br />                                                                                                 Human                                                                                      BRCA2 <br />                                                                                                 data <br />                                                                                                 input                                                                       ATM                 MLH1 <br />                                                                                                                                                                                    BRCA1    MSH2 <br />                                                                                                                                                              BCL2 <br />  <br />                                                                                                                                                                                         CHEK2 MSH6 <br />                                                                                                                                                                            TP53 <br />  <br />                                                                                                                                                                                                      MUTYH <br />  <br />  <br />                                                                                                                                                                                   ERCC3 <br />                                                                                                         GS APO <br />                                                                                                                                                                                   ERCC2     ERCC5 <br />                                                                                                                                  +                    -                                              XPC <br />                                                                                                                                       Germline mut. <br />                                                                                                                   Somatic mut. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                            XPA <br />                                                                                                                                  -                    +                           ERCC4             DDB2 <br />  <br />                                                                                                                                  +                    + <br />                                                                                                                                  ?                    ? <br /> Figure 5. Integrating evolutionary functional data.   Projection yeast lethality data human apoptosis genome-stability gene <br /> network: essential (blue GNNs) nonessential yeast orthologs (grey GNNs) according SGD database (46). graph presents orthologs <br /> inferred LCA yeast human  e. rooted present STN-M). White GNNs correspond genes present branch absent yeast, predicted parsimony analysis  &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section). Asterisks identify GNNs orthology predicted orthologous groups con&#239;&#172;?rmed Inparanoid database (47). (B) Projection mouse lethality data human apoptosis genome- <br /> stability gene network: essential (red GNNs) nonessential (grey GNNs) mouse orthologs according MGD database (45). graph presents GNNs orthologs inferred LCA mouse human  e. rooted present STN-C). GNNs lack knock data MGD database indicated white GNNs (mainly r module). (C) Projection genes causally implicated human cancer genes&#226;&#8364;&#8221; <br /> according Cancer Gene Census (48). Colors indicate gene somatically mutated cancer (red GNNs) mutated germline <br /> predisposing cancer (blue GNNs)  White GNNs indicate genes mentioned Cancer Gene Census. <br />  <br />  <br />  embryonic-perinatal lethality. data trans-                                                                                                   cellular expansion viable. reading complements <br /> ferred STN representing LCA mouse                                                                                                            results yeast, nonessential human  e. STN-C) projected correspond-                                                                                                       yeast nonessential mouse cellular level. pic- <br /> ing human network topology (Figure 5B). data pro-                                                                                                         torial consequence complementarity results jection shows homogeneous distribution lethal alleles                                                                                                     yeast mice set blue symbols Figure 5A nonlethal ones (red grey GNNs, respectively),                                                                                                        overlap red symbols Figure 5B. concentration genomic stability network genes lacking knock data (white GNNs). Figure 5B <br />                                                                                                                                                                Correlating plasticity cancer statistics <br /> highlights essentiality apoptosis genome- <br /> stability gene network organism development.                                                                                                        systematical, available data functional  genes knock information                                                                                                     impairment human genome-maintenance mechanisms <br /> (mainly placed r module), mouse statistics indicate                                                                                                    comes cancer statistics. According global vast majority knock alleles nonessential                                                                                                     human disease network described Goh et al. (44), <br /> cellular level, given gene disruption                                                                                                      180 genes listed genome-maintenance <br />                                                                         Nucleic Acids Research, 2008, Vol. 36,  19 6279 <br />  <br />  <br /> gene network, 51 associated human disor-           gene loci. attempt obtain proportion null <br /> der.  &gt;50% implicated cancer.            non-null alleles human following strategy <br /> application plasticity estimates presented        used mouse infer lethality according expected <br /> previous sections, consider cancer statistics data.       frequency Mendelian distribution. focus    Genes causally implicated cancer collectively          analysis set genes placed r module, <br /> identi&#239;&#172;?ed cancer genes genes (48), share            collectively represented locus-speci&#239;&#172;?c muta- <br /> common feature: potentially lethal             tion database&#226;&#8364;&#8221;XP mutation database (http://www. <br /> organism disruption tissue architecture, muta-         xpmutations.org). genes associated <br /> tions genes lead cancer lethal   DNA repair function (nucleotide-excision <br /> cell. mutations types: somatic germ-         repair) related rare autosomal recessive <br /> line. &#239;&#172;?rst arise organism development        human clinical disorders (Xeroderma pigmentosum, cells, second inherited&#226;&#8364;&#8221;present        Cockayne Syndrome Trichothiodystrophy), ception continue cell.           turns reliable obtaining representative <br /> fact, germline mutations genes cause cancer pre-          sample (XP database repository XP mutations iden- <br /> disposition, cancer se, contrasting somatic         ti&#239;&#172;?ed patients worldwide). retrieved 182 mutated <br /> mutations large extent primary cause        genotypes available database, cancers (56).                                                    pooled according zygosity presence    Mutations lead cancer increase cell &#239;&#172;?tness (5,57),    null nonnull alleles (Table 1, Panel . Sample <br /> implying gene network tolerate  cell        number compared second database order bene&#239;&#172;?t  genetic change (58).                  attest representativeness database (Table 1, <br /> Consequently reasonable expect genes           Panel B)  Supplementary Material Online located plastic gene networks.                            details). Given data, case null/null patients exist    assessed cancer statistics available Cancer     extent human population, strong <br /> Gene Census Cancer Genome Project&#226;&#8364;&#8221;CGP                     argument essentiality genes located (http://www.sanger.ac.uk/genetics/CGP). graph             poorly plastic region network. pointed Figure 5C shows projection mutations causally             Table 1, case. total absence implicated human cancer retrieved census.           null alleles homozygous.  considering equiva- <br /> Observe genes polarized distribution          lent criteria human, mouse yeast infer leth- network topology. presenting exclusively               ality, data consistent lethality germline somatic mutations associated apoptotic functions        genes network projection, allowing plastic <br /> (red GNNs), plastic portion network,       area regarded essential human. presenting exclusively germline ones asso- <br /> ciated genome stability (blue GNNs), poorly <br /> plastic region. Conversely, genes             DISCUSSION <br /> mutation types overlap apoptosis genome-stability networks.                                   presented orthology map order locate    location germline mutations poses chal-          eukaryotic genes human apoptosis genome- <br /> lenge evolutionary scenario. explain           stability gene/protein-association network. According <br /> germline mutations human genes, given         scenario, apoptosis genome stability located poorly plastic region?  care        di&#239;&#172;&#8364;erent origins evolution, spite taken order consider results        complex interaction systems observed yeast mouse di&#239;&#172;&#8364;erences statistical data.        human gene network  Figure 6 summary). instance, gene statistics comes mainly epi-         genome-stability network emerged ear- <br /> demiological data shows exclusively genes           lier eukaryotic evolution. <br /> mutations causally implicated oncogenesis               results consistent scenarios described independent reports,         described di&#239;&#172;&#8364;erent authors. instance, position <br /> showing mutations primary patient material (48).              genome stability base eukaryotic species tree According CGP census, underlying rationale            highly consistence DNA repair functions <br /> interpreting mutated gene causal cancer develop-         described prokaryotes [DNA repair Escherichia coli <br /> ment number pattern mutations          extensively recognized served paradigm gene likely selected confer      investigation organisms: NER (59), BER (60), <br /> growth advantage cell population           MMR (61) RER (62)].  root BCL2 cancer developed (48).  contrast mouse        base LCA metazoans consistent identi&#239;&#172;?ed <br /> yeast knock alleles, gene range            pro-survival functioning Bcl-2 protein family members <br /> mutations, single nucleotide substitution com-       C. elegans (63,64). Likewise, position caspases <br /> plete transcript disruption  e. null alleles       base LCA metazoans previously <br /> severe situation, equivalent mouse yeast knock-           described (14), consistent origins data).                                                       intrinsic pathway components predate TNF-like cyto- <br />    order circumvent data limitations              kines (65). TNF extrinsic pathway core components <br /> improve analysis investigated human           described vertebrates (47) corroborate <br /> statistics assessing genotypic pro&#239;&#172;?le         scenario, line mammalian-like functioning <br />  6280 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                             Inferred changes genetic systems <br />                                                                                                                                                                                                                                                                                                                                          Root 76% genome stability components mapped network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                             STN-P/Q <br />                                                                                                                                                                                                                                                                                                                                          (46% BER; 81% CS; 87% MMR; 89% NER; 68% RER) <br />     Distribution orthologs                                                                                                                                                                                                                                                                                                            Root 39% apoptotic components mapped network; <br />                                                                                                                                                                                                                                                                                                                                          earliest apoptotic core genes (e.g. Cytochrome c) <br />                                   Apoptosis <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                   Q <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                       P <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                               STN-L <br />                                                                                                                                                                                                                                                                                                                                                                                                                             Network enrichment apoptotic core <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                  O <br />                                   Genome stability                                                                                                                                                                                                                                                                                                                                                                          components intrinsic pathway <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                         N <br />                                                                                                                                                                                                                                                                                                                                                                                                                             (e.g. BCL2 initiator caspase-9) <br />                                                                                                                                                                                                                             Eukarya <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                         M <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       Network enrichment  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       STN                                                                                                                                                                                                                                                                                                                                                                                                              L <br />     Distribution plasticity data                                                                                                                                                                                                                                                                                                                                                                                                                                                   apoptotic core components extrinsic <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                             K <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       pathway (e.g. TNF superfamily members) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                           J <br />                                                      Class orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                        <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     STN-F <br />                                                                                                                                                                                                                                                Opisthokonta                                                                                                                                                                                                                                                                      Presence 100% genome <br />                                                      Class-b orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  H <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  stability components mapped <br />                                                      Class-c orthologs <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                G <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     F <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  E <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             Presence 99%  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     STN-B <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      D <br />                                                                                                                                                                                                                                                                                                                                                  Metazoa                                                                                                                                                                                                    apoptotic components <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               C <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             mapped network <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    B <br />                                                                                                                                                                                                                                                                                                                                                                              Chordata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                           Vertebrata <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              18 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               1. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                 0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               0 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                         15 <br />                                                                                                                                                                                                                                                                                                                                                                                                                Mammalia <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    12 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      0. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               ) <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                             (n <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       0 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          5 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                90 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                        gs <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                              60 <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          0. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     lo <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                  0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   ho <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                Primate <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                          30 <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               rt <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      0 <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                               O <br />      G. lamblia <br />                   T. pseudonana <br />                                   C. hominis <br />                                                P. falciparum <br />                                                                C. merolae <br />                                                                              thaliana <br />                                                                                           D. discoideum <br />                                                                                                           E. cuniculi <br />                                                                                                                         F. neoformans <br />                                                                                                                                         S. pombe <br />                                                                                                                                                     fumigatus <br />                                                                                                                                                                   Y. lipolytica <br />                                                                                                                                                                                   D. hansenii <br />                                                                                                                                                                                                 E. gossypii <br />                                                                                                                                                                                                               C. glabrata <br />                                                                                                                                                                                                                              S. cerevisiae <br />                                                                                                                                                                                                                                              K. lactis <br />                                                                                                                                                                                                                                                          C. elegans <br />                                                                                                                                                                                                                                                                        mellifera <br />                                                                                                                                                                                                                                                                                       gambiae <br />                                                                                                                                                                                                                                                                                                   D. melanogaster <br />                                                                                                                                                                                                                                                                                                                     C. intestinalis <br />                                                                                                                                                                                                                                                                                                                                       D. rerio <br />                                                                                                                                                                                                                                                                                                                                                  T. nigroviridis <br />                                                                                                                                                                                                                                                                                                                                                                    T. rubripes <br />                                                                                                                                                                                                                                                                                                                                                                                  X. tropicalis <br />                                                                                                                                                                                                                                                                                                                                                                                                  G. gallus <br />                                                                                                                                                                                                                                                                                                                                                                                                              M. domestica <br />                                                                                                                                                                                                                                                                                                                                                                                                                             B. taurus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                         C. familiaris <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                         M. musculus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       R. norvegicus <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                       M. mulatta <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                    P. troglodytes <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                     H. sapiens <br /> Figure 6. Summary inferred changes genetic systems. histograms distribution 180 human orthologs according roots <br /> inferred eukaryote species tree  details, Figures 2 3). STNs corresponding LCA indicated. Inset graph shows presence fraction orthologs STNs  details, Figure 4D). Diverse important events related roots sets genes pointed STNs. Chromosome stability (CS). <br />  <br />  extrinsic apoptosis pathway described Danio rerio                                                                                                                                                                                                                                                                                                            contrasting early origin genome-stability absence TNF TNF receptor superfamily                                                                                                                                                                                                                                                                                                                genes. identi&#239;&#172;?ed expansion apoptotic compo- <br /> members C. elegans (52).                                                                                                                                                                                                                                                                                                                                        nents KOG Inparanoid-derived data. <br />     novelty results                                                                                                                                                                                                                                                                                                             numerical expansion apoptotic components genome-maintenance mechanisms  net-                                                                                                                                                                                                                                                                                                            related origin cell functions. <br /> work-based model, produce unique evolutionary sce-                                                                                                                                                                                                                                                                                                            assumption illustrated TP53 appear- <br /> nario. point view allows investigating                                                                                                                                                                                                                                                                                                                 ance transition later evolutionary scenarios: p53 <br /> sequential events led entanglement apoptosis                                                                                                                                                                                                                                                                                                        protein regulates apoptosis, key genome-stability gene networks.                                                                                                                                                                                                                                                                                                                                regulator cellular senescence, de&#239;&#172;?ned permanent <br />    course human genome-maintenance network                                                                                                                                                                                                                                                                                                               cell cycle arrest (66). Senescence alternative tumor <br /> evolution, major functional increments remark-                                                                                                                                                                                                                                                                                                           suppressor mechanism, damaged cells prevented <br /> able summarized Figure 6. &#239;&#172;?rst associated                                                                                                                                                                                                                                                                                                       dividing (67). senescence functionally base species tree comprises genome-stability                                                                                                                                                                                                                                                                                                        emerged TP53 gene, second tumor-suppressor <br /> genes. second evolves gradually, especially near                                                                                                                                                                                                                                                                                                           mechanism relaxed selective pressure metazoan origin, gene components added                                                                                                                                                                                                                                                                                                                apoptosis, increasing tolerance nonadaptive <br /> apoptosis intrinsic pathway, BCL2 cas-                                                                                                                                                                                                                                                                                                             processes (e.g. genetic drift, mutation recombination) <br /> pase gene family members. continues apo-                                                                                                                                                                                                                                                                                                             favoring evolution. results consistent ptosis enrichment addition extrinsic                                                                                                                                                                                                                                                                                                           emergence major mechanisms tumor  <br /> components, TNF superfamily members.                                                                                                                                                                                                                                                                                                                       trol metazoan evolution, regards <br />    Furthermore, macroevolutionary perspective                                                                                                                                                                                                                                                                                                            senescence genes taken account conclusions considered                                                                                                                                                                                                                                                                                                             draw safe conclusion. <br /> estimated evolutionary error  e. species tree nodes                                                                                                                                                                                                                                                                                                             Likewise, TP53 exemplify evolution genome- rooting point species tree),                                                                                                                                                                                                                                                                                                        stability gene network. Acting transcription factor, conceivable genes actually recent                                                                                                                                                                                                                                                                                                       p53 protein able modulate DNA-repair processes think.  conclusions point                                                                                                                                                                                                                                                                                                            (9,68). DNA-repair gene response p53 protein course human genome-maintenance gene                                                                                                                                                                                                                                                                                                                line evidences showing conserved gene <br /> network evolution dramatic                                                                                                                                                                                                                                                                                                                  functions subject substantial evolution regu- <br /> increase number apoptotic components,                                                                                                                                                                                                                                                                                                                    latory level (69). <br />                                                                           Nucleic Acids Research, 2008, Vol. 36,  19 6281 <br />  <br />  <br />    plasticity analysis pointed genes           plasticity emerges follows trends evolution su&#239;&#172;&#8364;ered duplication,             unicellular organisms. <br /> poorly abundant widely distributed extant                   test putative evolutionary scenario species. results locate conserved genes             human genome-maintenance network given loca- <br /> mainly genome-stability network,              tion human genes. complex organ- ancient portion network. contrast, <span id='am-31' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span>cer</span>-         isms natural selection acts di&#239;&#172;&#8364;erent levels (organism <br /> tain pairs genes known function human            &#239;&#172;?tness cell &#239;&#172;?tness), stem con&#239;&#172;&#8218;icting selective placed di&#239;&#172;&#8364;erent distribution abundance (e.g.            pressures: fast proliferating cell clone naturally <br /> ATM BRCA1, MUS81 EME1, PCNA                            selected unicellular organism, fast proliferating cell <br /> RPA1, RPA1 RPA2&#226;&#8364;&#8221;Figure 4). Analyzing                  clone complex organism represent tumor indicate enlargement network            end killing organism. complex organ- occur addition new nodes             isms, apoptosis genome-stability networks work tually evolve work ancient ones.                  tissue-maintenance mechanisms, favoring natural selec- <br />    Lethality measures performed complemen-             tion acting organism level. disruption tary ways: assessing knock data yeast genes             mechanism favor natural selection acting cell level, second regarding essentiality mice.           stands reason genes located <br /> measures complementary following reasons:              plastic, cell-essential region genome- <br /> yeast unicellular organism lethality concerns             maintenance network. cell viability, mouse multicellular animal,           Speci&#239;&#172;?cally, concerning human functional data, complex ontogeny. later case, viable              questions emerge evolutionary analysis embryo implies survival egg implantation rele-         cancer statistics:   distribution genes vant cell expansion. consequence, organism            polarized major segments described labeled viable, certainly cell viable   evolutionary scenario? (ii) genes implicated <br /> organism.  organism viable,            types cancers (somatic germline) overlap <br /> experimental procedure does discriminate                apoptosis genome-stability networks? addi- problem occurred cell organism                tional work needed fully characterize rele- <br /> level. summary, lethality data unicellular organisms         vance results, clear yeast sound information genes essen-           evolutionary perspective bring insights tial cell viability, multicellular organisms       understanding cancer origins. <br /> mice sound information genes essential cell level. Transferring cell essentiality infor- <br /> mation mice yeast human apoptosis              SUPPLEMENTARY DATA <br /> genome-stability gene network revealed essential              Supplementary Data available NAR Online. <br /> genes cell level located ancestral <br /> region network. <br />    integration information ancestrality, plas-       ACKNOWLEDGEMENTS <br /> ticity essentiality poses challenging questions. ancient, plastic essen-          thank D. Jung technical assistance. acknowl- <br /> tial genes located genome stability,          edge STRING, Inparanoid, MGD, SGD, CGP XP <br /> apoptosis network comprises recent, plastic          databases providing public access data. essential genes. Genome stability required guarantee information transference parental <br /> genome o&#239;&#172;&#8364;spring provides essen-         FUNDING <br /> tial ingredients natural selection act: memory.       Brazilian Agencies FAPERGS, CAPES CNPq (grant surprising genome-stability network rooted          140947/2006-0, partially). Funding open access charge: <br /> early possible species tree. reasonable       grant 40947/2006-0. crucial function performed highly  <br /> served genes, gene duplication favored         Con&#239;&#172;&#8218;ict statement. declared. high possibility disrupting essential pathway, <br /> yielding poorly plastic network. Ancestrality, plasticity <br />                                                                    REFERENCES essentiality pointed correlated features typical prokaryotes (70). hand, multicel-          1. Danial,N.N. Korsmeyer,S.J. (2004) Cell death: critical control <br /> lular organisms complex ontogeny,                 points. Cell, 116, 205&#226;&#8364;&#8220;219. <br />                                                                    2. Lettre,G. Hengartner,M.O. (2006) Developmental apoptosis Mus musculus, available literature reports having             C. elegans: complex CEDnario. Nat. Rev. Mol. Cell Biol., correlations (71,72). &#239;&#172;?nd cell gene               7, 97&#226;&#8364;&#8220;108. <br /> essentiality correlated ancestrality plasticity     3. Kanehisa,M., Goto,S., Hattori,M., Aoki-Kinoshita,K.F., Itoh,M., unicellular complex multicellular organisms.              Kawashima,S., Katayama,T., Araki,M. Hirakawa,M. (2006) point discriminate cell lethality            genomics <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> genomics: new developments KEGG. <br />                                                                       Nucleic Acids Res., 34, D354&#226;&#8364;&#8220;D357. <br /> organism lethality: isolating data essential genes         4. Hipfner,D.R. Cohen,S.M. (2004) Connecting proliferation cell survival essential genes organism viabil-           apoptosis development disease. Nat. Rev. Mol. Cell Biol., <br /> ity, correlation cell essentiality, ancestrality          5, 805&#226;&#8364;&#8220;815. <br />  6282 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br />  5. Crespi,B. Summers,K. (2005) Evolutionary biology cancer.        28. Ciccarelli,F.D., Doerks,T., von Mering,C., Creevey,C.J., Snel,B. <br />     Trends Ecol. Evol., 20, 545&#226;&#8364;&#8220;552.                                           Bork,P. (2006) automatic reconstruction highly <br />  6. Yan,B., Wang,H., Peng,Y., Hu,Y., Wang,H., Zhang,X., Chen,Q.,               resolved tree life. Science, 311, 1283&#226;&#8364;&#8220;1287. <br />     Bedford,J.S., Dewhirst,M.W. Li,C.Y. (2006) unique role        29. Letunic  Bork,P. (2007) Interactive Tree Life (iTOL):     DNA fragmentation factor maintaining genomic stability.             online tool phylogenetic tree display annotation. <br />     Proc. Natl Acad. Sci. USA, 103, 1504&#226;&#8364;&#8220;1509.                                 Bioinformatics, 23, 127&#226;&#8364;&#8220;128. <br />  7. Castro,M , Onsten,T.G.H., Moreira,J.C.F.                    30. Pennisi,E. (2003) Drafting tree. Science, 300, 1694. <br />     Almeida,R.M.C. (2006) Chromosome aberrations solid tumors           31. Baldauf,S.L. (2003) deep roots eukaryotes. Science, <br />     stochastic nature. Mutat. Res., 600, 150&#226;&#8364;&#8220;164.                       300, 1703&#226;&#8364;&#8220;1706. <br />  8. Zhivotovsky,B. Kroemer,G. (2004) Apoptosis genomic             32. Katinka,M.D., Duprat,S., Cornillot,E., Metenier,G., Thomarat,F., <br />     instability. Nat. Rev. Mol. Cell Biol., 5, 752&#226;&#8364;&#8220;762.                        Prensier,G., Barbe,V., Peyretaillade,E., Brottier,P., Wincker,P. <br />  9. Sengupta,S. Harris,C.C. (2005) p53: Tra&#239;&#172;&#402;c cop cross-            et al. (2001) Genome sequence gene compaction     roads DNA repair recombination. Nat. Rev. Mol. Cell Biol.,          eukaryote parasite Encephalitozoon cuniculi. Nature, 414, 450&#226;&#8364;&#8220;453. <br />     6, 44&#226;&#8364;&#8220;55.                                                              33. Delsuc,F., Brinkmann,H. Philippe,H. (2005) Phylogenomics <br /> 10. Alano,C.C., Ying,W. Swanson,R  (2004) Poly(ADP-ribose)               reconstruction tree life. Nat. Rev. Genet., <br />     polymerase-1-mediated cell death astrocytes requires NAD+               6, 361&#226;&#8364;&#8220;375. <br />     depletion mitochondrial permeability transition. J. Biol. Chem.,   34. Snel,B., Bork,P. Huynen,M  (2002) Genomes &#239;&#172;&#8218;ux:     279, 18895&#226;&#8364;&#8220;18902.                                                          evolution archaeal proteobacterial gene content. Genome <br /> 11. Duckett,D.R., Bronstein,S.M., Taya,Y. Modrich,P. (1999)                Res., 12, 17&#226;&#8364;&#8220;25. <br />     hMutSalpha- hMutLalpha-dependent phosphorylation p53            35. Kunin,V. Ouzounis,C  (2003) balance driving forces <br />     response DNA methylator damage. Proc. Natl Acad. Sci.                genome evolution prokaryotes. Genome Res., <br />     USA, 96, 12384&#226;&#8364;&#8220;12388.                                                      13, 1589&#226;&#8364;&#8220;1594. <br /> 12. Barabasi L. Oltvai N. (2004) Network biology:            36. Campillos,M., von Mering,C., Jensen,L.J. Bork,P. (2006) <br />     standing cell&#226;&#8364;&#8482;s functional organization. Nat. Rev. Genet.,             Identi&#239;&#172;?cation analysis evolutionarily cohesive functional <br />     5, 101&#226;&#8364;&#8220;113.                                                                modules protein networks. Genome Res., 16, 374&#226;&#8364;&#8220;382. <br /> 13. Aravind,L., Walker,D.R. Koonin,E.V. (1999) Conserved               37. Itoh,M., Nacher,J., Kuma,K  Goto,S. Kanehisa,M. (2007) <br />     domains DNA repair proteins evolution repair systems.            Evolutionary history functional implications protein <br />     Nucleic Acids Res., 27, 1223&#226;&#8364;&#8220;1242.                                         domains combinations eukaryotes. Genome Biol., <br /> 14. Koonin,E.V. Aravind,L. (2002) Origin evolution                  8, R121. <br />     eukaryotic apoptosis: bacterial connection. Cell Death Di&#239;&#172;&#8364;er.,     38. Pal,C., Papp,B. Lercher,M.J. (2005) Adaptive evolution <br />     9, 394&#226;&#8364;&#8220;404.                                                                bacterial metabolic networks horizontal gene transfer. <br /> 15. Lin , Kong,H., Nei,M. Ma,H. (2006) Origins evolution             Nat. Genet., 37, 1372&#226;&#8364;&#8220;1375. <br />     recA/RAD51 gene family: evidence ancient gene dupli-        39. Remm,M., Storm,C.E.V. Sonnhammer,E.L.L. (2001) <br />     cation endosymbiotic gene transfer. Proc. Natl Acad. Sci. USA,         Automatic clustering orthologs paralogs pairwise <br />     103, 10328&#226;&#8364;&#8220;10333.                                                          species comparisons. J. Mol. Biol., 314, 1041&#226;&#8364;&#8220;1052. <br /> 16. Aravind,L., Dixit,V.M. Koonin,E.V. (2001) Apoptotic molec-         40. Shannon,C.E. (1948) mathematical theory communication. <br />     ular machinery: vastly increased complexity vertebrates revealed        Bell Syst. Tech. J., 27, 379&#226;&#8364;&#8220;423. <br />     genome comparisons. Science, 291, 1279&#226;&#8364;&#8220;1284.                        41. Kendal,W.S. (1990) use information theory analyze <br /> 17. Merlo,L.M.F., Pepper,J.W., Reid,B.J. Maley,C.C. (2006)                 genomic changes neoplasia. Math. Biosci., 100, 143&#226;&#8364;&#8220;159. <br />     Cancer evolutionary ecological process. Nat. Rev. Cancer,    42. Castro,M , Onsten,T.T.G., Almeida,R.M.C.     6, 924&#226;&#8364;&#8220;935.                                                                Moreira,J.C.F. (2005) Pro&#239;&#172;?ling cytogenetic diversity 18. Greaves,M. (2007) Darwinian medicine: case cancer. Nat. Rev.         entropy-based karyotypic analysis. J. Theor. Biol., 234, 487&#226;&#8364;&#8220;495. <br />     Cancer, 7, 213&#226;&#8364;&#8220;221.                                                    43. Gatenby,R  Frieden,B.R. (2004) Information dynamics 19. Castro,M , Mombach,J.C.M., Almeida,R.M.C.                       carcinogenesis tumor growth. Mutat. Res., 568, 259&#226;&#8364;&#8220;273. <br />     Moreira,J.C.F. (2007) Impaired expression NER gene network       44. Goh,K , Cusick,M.E., Valle,D., Childs,B., Vidal,M.     sporadic solid tumors. Nucleic Acids Res., 35, 1859&#226;&#8364;&#8220;1867.                  Barabasi L. (2007) human disease network. Proc. Natl Acad. <br /> 20. Koonin,E.V. (2005) Orthologs, paralogs, evolutionary                   Sci. USA, 104, 8685&#226;&#8364;&#8220;8690. <br />     genomics. Annu. Rev. Genet., 39, 309&#226;&#8364;&#8220;338.                              45. Eppig,J.T., Blake,J , Bult,C.J., Kadin,J , Richardson,J.E. 21. Tatusov,R., Fedorova,N., Jackson,J., Jacobs , Kiryutin,B.,               Mouse Genome Database Group (2007) mouse genome <br />     Koonin,E., Krylov,D., Mazumder,R., Mekhedov,S.,                            database (MGD): new features facilitating model  Nucleic <br />     Nikolskaya  et al. (2003) COG database: updated version           Acids Res., 35, D630&#226;&#8364;&#8220;D637. <br />     includes eukaryotes. BMC Bioinformatics, 4, 41.                        46. Hirschman,J.E., Balakrishnan,R., Christie,K.R., Costanzo,M.C., <br /> 22. von Mering,C., Jensen,L.J., Kuhn,M., Cha&#239;&#172;&#8364;ron,S., Doerks,T.,                Dwight,S.S., Engel,S.R., Fisk,D.G., Hong,E.L., Livstone,M.S., <br />     Kruger,B., Snel,B. Bork,P. (2007) STRING 7&#226;&#8364;&#8220;recent develop-             Nash,R. et al. (2006) Genome Snapshot: new resource     ments integration prediction protein interactions.           Saccharomyces Genome Database (SGD) presenting overview <br />     Nucleic Acids Res., 35, D358&#226;&#8364;&#8220;D362.                                         Saccharomyces cerevisiae genome. Nucleic Acids Res., <br /> 23. Mirkin,B.G., Fenner,T , Galperin,M.Y. Koonin,E.V. (2003)             34, D442&#226;&#8364;&#8220;D445. <br />     Algorithms computing parsimonious evolutionary scenarios       47. O&#226;&#8364;&#8482;Brien,K.P., Remm,M. Sonnhammer,E.L.L. (2005) <br />     genome evolution, universal common ancestor domi-             Inparanoid: comprehensive database eukaryotic orthologs. <br />     nance horizontal gene transfer evolution prokaryotes.         Nucleic Acids Res., 33, D476&#226;&#8364;&#8220;D480. <br />     BMC Evol. Biol., 3, 2.                                                 48. Futreal,P , Coin,L., Marshall,M., T., Hubbard,T., <br /> 24. von Mering,C., Jensen,L.J., Snel,B., Hooper,S.D., Krupp,M.,                Wooster,R., Rahman,N. Stratton,M.R. (2004) census     Foglierini,M., Jou&#239;&#172;&#8364;re,N., Huynen,M  Bork,P. (2005)                   human cancer genes. Nat. Rev. Cancer, 4, 177&#226;&#8364;&#8220;183. <br />     STRING: known predicted protein-protein associations,              49. Stenson,P.D., Ball,E.V., Mort,M., Phillips D., Shiel,J , <br />     integrated transferred organisms. Nucleic Acids Res.,           Thomas,N.S.T., Abeysinghe,S., Krawczak,M. Cooper,D.N. <br />     33, D433&#226;&#8364;&#8220;D437.                                                             (2003) Human gene mutation database (HGMD (R)): 2003 update. <br /> 25. Wain,H.M., Lush,M.J., Ducluzeau,F., Khodiyar,V.K. Povey,S.             Hum. Mutat., 21, 577&#226;&#8364;&#8220;581. <br />     (2004) Genew: Human Gene Nomenclat <br /> </body></html>