<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACrERIOLOGY, Nov. 1992, p. 7090-7097                                                                    Vol. 174,  22 <br /> 0021-9193/92/227090-08$02.00/0 <br /> Copyright X) 1992, American Society Microbiology <br />  <br />  <br />           Mutants Carrying Conditionally Lethal Mutations Outer <br />           Membrane Genes omsA firA (ssc) Phenotypically <br />                      Similar, omsA Allelic firA <br />                                        RIIYTA VUORIO* MARTTI VAARA <br />                Department Bacteriology Immunology, University Helsinki, 00290 Helsink4 Finland <br />                                         Received 10 August 1992/Accepted 8 September 1992 <br />  <br />             previously identified gene  ssc gene) defective thermosensitive antibiotic- <br />          supersusceptible outer membrane permeability mutant SS-C SalmoneUla typhimurium shown          gene analogous Escherichia coli genefirA (L. Hirvas, P. Koski, M. Vaara, EMBO J. 10:1017-1023, <br />          1991). tentatively implicatedfirA different function, mRNA synthesis. report          defect thermosensitive outer membrane omsA mutant E. coli (T. Tsuruoka, M. Ito, S. Tomioka,  <br />          Hirata, M. Matsuhashi, J. Bacteriol. 170:5229-5235, 1988) mutation firA; mutation <br />          changed codon 271 serine asparagine. omsA-induced phenotype completely reverted          plasmids containing wild-type firA ssc. Plasmids carrying omsA allele, identical mutant allele <br />          prepared localized mutagenesis, control lac elicited partial complementation. Transcomple- <br />          mentation studies plasmids carrying various mutant alleles S. typhimurium gene indicated          ability plasmids complement omnsA mutation similar ability complement ssc <br />          mutation. antibiotic-supersusceptible phenotype omsA mutant closely resembled ssc <br />          mutant, e., omsA mutant supersusceptible hydrophobic antibiotics large-<span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> antibiotics <br />          intact outer membrane effective permeability barrier. previously demonstrated          omnsA mutant, outer membrane ssc mutant selectively ruptured incubation 1 h <br />          growth-nonpermitting temperature; 82% periplasmic j-lactamase 3%          cytoplasmic marker enzyme released medium. findings consistent          concept thatfirA essential gene involved generation outer membrane. <br />  <br />    outer membrane (OM) gram-negative bacteria               known essential OM genes   turn comprehensively characterized biological               invaluable elucidation remaining enigmas OM <br /> membranes (26, 31).  lack complete               biogenesis. mutants include omsA (outer mem- <br /> picture biosynthesis. translocation process OM          brane structure) mutant Escherichia coli (39) ssc-1 <br /> proteins partially known (49). little known            (supersensitive class C) mutant Salmonella typhimurium export <span id='am-42' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-43' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-44' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span>lipopolysaccharide</span> (<span id='am-32' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span>LPS</span>) component                 (17, 38). shown extremely susceptible synthesis site cytoplasmic membrane            wide set antibiotics, OM outer leaflet OM (33). Furthermore, <span id='am-19' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span>          effective permeability barrier. unable grow essential LPS viability bacterial cells,      42&#194;&#176;C. study ssc-1 mutation led discover 2 (pxA4 lpxB) approximately 8 10 genes             sequence novel gene S. typhimurium, ssc gene (16). <br /> involved biosynthesis discovered (33).           sequence analog E. coli, firA gene, search remaining essential genes OM biogenesis             known (5). hand, gene mutated omsA hampered lack mutants conditionally                mutant identified Tsuruoka et al. (39), defective genes.                                              preliminary evidence suggested omsA mutation <br />    characteristic feature intact enterobacterial OM       maps approximately 4 min E. coli chromosome. relative impermeability hydrophobic drugs (28-30).             firA gene located 4 min, felt impermeability LPS (29, 45); outer leaflet       necessary study possible relationship genes. OM contains LPS <span id='am-14' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-15' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span>phospholipid</span> lacks                report, omsA mutation allelic <span id='am-10' about='obo:IMR_0001384' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001384" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="glycerophospholipid" datatype="xsd:string"></span>glycerophospholipid</span> patches, effective channels hy-           firA. Furthermore, phenotypic character- <br /> drophobic diffusion (29-31). Agents damage OM                istics ssc-1 (irA) mutant S. typhimurium sensitize bacteria drastically hydrophobic inhibitors          omsA mutant E. coli closely resemble  <br /> (40).  mutants severe defects nones- <br /> sential inner <span id='am-25' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>carbohydrate</span> core region LPS (chemotypes  Rd2, Rd,, RcP-) supersusceptible (14, 28, 30,                           MATERIALS METHODS <br /> 37).  supersusceptibility hydrophobic drugs suggest defect OM.                                          Bacterial strains plasmids. E. coli S. typhimu- <br />    Accordingly, novel mutants conditionally lethal          rium strains plasmids used described Table 1.  permissive conditions, supersusceptible hy-             E. coli cloning hosts JM105 (50) vector pUC19 (50), <br /> drophobic antibiotics mutated previously               TG1 (8) phage M13mp8 M13mp9 (27), XL1-Blue <br />                                                                       (2) Bluescript SK+ (1). Cultures grown L broth <br />                                                                       (17) 2x TY medium (34), supplemented ampicillin <br />                                                                       (100 ,ug/ml) ampicillin (50 ,ug/ml) tetracycline (12.5 <br />   * Corresponding author.                                             ,ug/ml) appropriate. <br />                                                                7090 <br />  VOL. 174, 1992                                                                           OUTER MEMBRANE GENES omsA firA                   7091 <br />  <br />                   TABLE 1. Bacterial strains plasmids                           sodium citrate], 2% blocking reagent [catalog  1096176; <br /> Strain plasmid             Genotype relevant property         Reference(s) <br />                                                                                     Boehringer], 0.1% N-lauroylsarcosine, 0.02% sodium dode- <br />                                                                                     cyl sulfate [SDS]) 1 h 42&#194;&#176;C. Hybridization  <br /> E. coli                                                                             formed 2.5 ml solution 500 pmol   CDH23-210                HfrC metBl proA3 reLAl lac-3             39              30-mer firA-specific oligonucleotide probe CTGGCTGATT <br />                              maL436 tsx-76 K-12                                     TAGCGCAGCAGTTGGATGCA labeled digoxigenin- <br />      CDH23-213             omsA sister transductant              39              <span id='am-59' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-60' property="rdfs:label" content="dUTP" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span>dUTP</span> tailing (Boehringer) 18 h 42&#194;&#176;C. filter                              CDH23-210                                              washed twice 5 min time 2x SSC-0.1% SDS S.   typhimurium                                                                    room temperature twice 5 min time      SH5014                ilv-1178 thr-914 6116                36              0.1 x SSC-0.1% SDS 54&#194;&#176;C. Detection performed                              metA422 metESSI trpB2 xyl-                             DIG luminescence detection kit (Boehringer) instructed <br />                              404 Hi-b H2-e,n,xflaA66                                supplier. <br />                              rpsL120 rfaJ4041 LT2                                      Growth viability determinations. Growth mea- <br />      SH7622                ssc-1  SH5014)                      17, 38          sured A550, determining viable count <br />                                                                                     plating L agar. <br /> Plasmids                                                                               Antibiotic susceptibility tests. MICs determined      pUCHS16Aa             ompH+                                    17              broth microdilution method L broth. inoculum <br />      pUCHS14-IIa           ssc+                                     17 <br />      pSSClOla              Encodes Ssc <span id='am-72' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span>              19              104 L agar-grown cells ml. Incubation 18 h <br />                              substitution Val-291- Met                              28 39&#194;&#176;C. antibiotics fusidic acid (Na salt), <br />      pSSC1Or               Encodes Ssc amino acid              19              rifampin, clindamycin hydrochloride, bacitracin (65,000 <br />                              substitution Met-288-*Leu                              U/mg; Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> , St. Louis, Mo.), erythromycin <br />      pSSC103a              Encodes Ssc amino acid              19              ethyl succinate (Orion, Helsinki, Finland), vancomycin hy- <br />                              substitution Met-290-3Leu                              drochloride (Eli Lilly <span id='am-52' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-53' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; , Indianapolis, Ind.), mupi- <br />      pSSC104a              Encodes Ssc amino acid              19              rocin (SmithKline Beecham, Worthing, West Sussex,  <br />                              substitution Met-292--+Leu                             gland). indicated text, alternative <br />      pSSC1O0a              Encodes Ssc amino acid              19              MIC test (E Test; AB Biodisk, Solna, Sweden) (42)                              substitution <span id='am-79' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-80' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Gly" datatype="xsd:string"></span>Gly</span>-27--Asp <br />      pSSC106a              Encodes Ssc amino acid              19              used. Bacteria  0.9% NaCl; turbidity suspension <br />                              substitution Met-31--Leu                               equal McFarland 0.5 standard) streaked      pSSC107a              Encodes Ssc amino acid              19              swab cover entire surface L agar plate.                              substitution Leu-130--*Met                             surface dried, E Test strip placed      pSSC108a              Encodes Ssc amino acid              19              plate incubated 18 h 37&#194;&#176;C. MIC                              substitution Gly-131--Asp                              interpreted noting point intersection growth <br />      pSSC109&quot;              Encodes Ssc amino acid              19              margin MIC scale E Test strip. <br />                              substitution Met-288--Lys                                 Susceptibility determinations agar diffusion method <br />      pSSC11O&apos;              Encodes Ssc amino acid              19              performed using L agar plates antibiotic discs                              substitution Met-290--*Lys <br />      pSSCllla              Encodes Ssc amino acid              19              previously described (17). antibiotic discs  S Rosco, <br />                              substitution Met-292--+Lys                             Copenhagen, Denmark) contained mupirocin (10 pg), van- <br />      pSSC112a              Encodes Ssc amino acid              19              comycin (70 tg), bacitracin (40 units), rifampin (30 p,g). <br />                              substitution Gly-289- Asp                                 Measurement enzyme activities. Detection 1-lacta- <br />      pGAH317               hlpA+ firA+                              20              mase modified method O&apos;Callaghan et al. <br />      pOMSA1O1b             oMSA                                     work       (32). Nitrocefin (Glaxo Pharmaceuticals, , Greenford, <br />      pOMSA102b             omsA                                     work       England) added enzyme samples 0.01 M <br />      pOMSA103b             firA +                                   work       HEPES (N-2-hydroxyethylpiperazine-N-2-ethanesulfonic <br />        lac promoter vector opposite orientation ompH   acid) (pH, 7.2) yield final concentration 100 p,g/ml, <br /> ssc gene.                                                                           rate nitrocefin hydrolysis monitored     lac promoter vector upstream orientation      b <br />                                                                                     540 nm. activity glucose 6-phosphate dehydrogenase firA gene. wild-type firA gene cloned pGAH317,             determined described Langdon (23). Cellular <br /> omsA allele cloned CDH23-213. Plasmid pOMSA102 contains onsA allele site-directed mutagenesis.                                      enzyme activities determined cells sonicated <br />                                                                                     twice 15 s time amplitude 12 p,m                                                                                      sonication microtip Soniprep 150 (Measuring &amp; Scien- <br />                                                                                     tific Equipment, , London, England). <br />   DNA techniques. General DNA manipulations car- <br /> ried described Sambrook et al. (34). Isolation                                                   RESULTS <br /> chromosomal DNA modified method Marmur described previously (22). <span id='am-64' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-65' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>Nucleotide</span> sequences                              Antibiotic-susceptible phenotype oms4 mutant. determined chain termination method (35)                           verified extended finding Tsuruoka et al. (39) <br /> Sequenase (United States Biochemical, Cleveland, Ohio).                             omsA mutant strain CDH23-213 remarkably suscep- <br /> Site-directed mutagenesis using oligonucle-                         tible certain antibiotics. MICs rifampin fusidic <br /> otide mutagenesis kit, version 2 (Amersham, Buckingham-                             acid approximately 250-fold lower omsA mutant <br /> shire, England), Bluescript SK+ (Stratagene, La Jolla,                          strain isogenic omsA + sister transductant <br /> Calif.) vector.                                                              (CDH23-210), erythromycin, mupirocin,    Southern hybridization. Southern blotting                         clindamycin 128-, 64-, 16-fold lower, respectively <br /> alkaline conditions positively charged nylon mem-                              (Table 2). omsA mutant strain susceptible branes (Boehringer, Mannheim, Germany). filter                              bacitracin vancomycin (Table 2). Common prehybridized 20 ml hybridization buffer (50% form-                         antibiotics fact penetrate entero- <br /> amide, Sx SSC [lx SSC 0.15 M NaCl plus 0.015 M                                   bacterial OM water-filled, narrow porin chan- <br />  7092         VUORIO VAARA                                                                                                                     J. BACTERIOL. <br />  <br />  TABLE 2. Complementation onsA mutant phenotype                            carries ssc+ inserted gene, introduced <br />   determined measurement MICs various antibioticsa                          CDH23-213. transformant resistant                                         MIC (,ug/ml)                             antibiotic probes tested omsA + control <br />                                                                                      (Table 2).  ability grow 42&#194;&#176;C resumed. <br />       Antibiotic    CDH23-210      CDH23-213 CDH23-213/ CDH23-213/                   Accordingly, ssc fully complemented omssA mutant <br />                       (omsA+)        (omtsA) pUCHS14-II    pUCHS16A                  phenotype. pUC19-derived plasmid, pUCHS16A <br />                                              (omsA sscl) (omsA4 ompHl) <br />                                                                                      (17), lacks ssc contains 5&apos;-flanking gene <br /> Fusidic acid                 512     2                   512        2                ompH, did elicit complementation (Table 2). <br /> Rifampin                       4     0.015                 2        0.015               Complementation different ssc alleles. study Clindamycin                  128     8                   128        8                ability ssc gene reverse E. coli omsA <br /> Bacitracin            &gt; 1,024       32                .1,024       32 <br /> Erythromycin               32        0.25                 64        0.5              mutation, tested various ssc mutant alleles Vancomycin                128       32                   128       32                property. Plasmids pSSC101 pSSC112 (Table 1) contain <br /> Mupirocin                  64        1                    64        0.5              12 ssc alleles previously site-directed mutagenesis <br />                                                                                      (19). pSSC101 corresponds ssc-1 mutant allele <br />   MICs determined 28&#194;&#176;C L broth (supplemented ampicillin            (originally S. typhimurium mutant strain SH7622), appropriate).                                                                   rest alleles encode mutant Ssc proteins, <br />                                                                                      having single amino acid alteration. ability                                                                                      plasmids complement omsA mutation E. coli <br /> nels, hydrophobic molecules (rifampin,                         (Table 3) appeared similar ability fusidic acid, erythromycin, mupirocin, clindamycin)                           complement ssc mutation S. typhimunium (19). Ac- <br /> large, hydrophilic molecules (bacitracin vancomycin)                             cordingly, pSSC101, pSSC109, pSSC110, pSSC111, <br /> (28, 30, 37, 40). contrast, sensitization slight                       encode mutant Ssc proteins substitutions sensitization antibiotics cefuroxime benzyl-                          Val-291-*Met, Met-288--Lys, Met-290-Lys, Met- <br /> penicillin, use porin channels, observed (data                         292-*Lys, respectively, caused partial complementa- shown; reference 39). Furthermore, like                           tion. defective ssc allele (Gly-289- &apos;Asp; carried enteric bacteria defective damaged OM (28,                           pSSC112), potentiates antibiotic-sensitive <br /> 37, 40), omsA strain showed increased susceptibility                          phenotype SH7622 (19), completely unable reverse aminoglycoside antibiotics (data shown;                              omsA phenotype. ssc allele pSSC102 (Met- <br /> reference 39).                                                                       288--Leu) omsA strain fully resistant (Table 3)   oms4 mutant phenotype complemented ssc.                                ssc-1 mutant partially resistant (19). tailed introduction, gene mapping data                           remaining pSSC plasmids omsA mutant non- <br /> comparison phenotypic properties prompted                               defective, exactly changed sscl mutant. search possible relationship omsA firA.                             results support idea previously cloned sequenced firA                                omsA allelic tofirA, E. coli analog ssc gene S. <br /> analog S. typhimunum, ssc gene (16), tested                                typhimunium. complement omsA mutation. em-                                  Cloning sequencing oms4 allele offirA. ployed pUC19-derived multicopy plasmid pUCHS14-II (17),                              cloned sequenced thefirA gene  e., omsA allele)  <br />  <br />                    TABLE 3. Ability plasmid-encoded ssc firA alleles complement omsA mutation CDH23-213 <br />  <br />             Strain/plasmid                   ssc firA allele      Mupirocin <br />                                                                                                 Sensitivity&apos; Bacitracin <br />                                                                                         Vancomycin                                Rifampin <br />                                                                                                                                                        Sensitivity <br />                                                                                                                                                      interpretationb <br />                                                 plasmid <br />  <br /> CDH23-213                                                        13 (24)              11 (18)              17 (27)            29 (20)             SS (SS) <br /> CDH23-213/pUCHS14-II                             ssc+                  0                    2                    0                 14                  R <br /> CDH23-213/pSSC101                                ssc-101               3                    9                   10                 21                  S <br /> CDH23-213/pSSC102                                ssc-102               0                    4                    0                 16                  R <br /> CDH23-213/pSSC103                                ssc-103               0                    4                    0                 16                  R <br /> CDH23-213/pSSC104                                ssc-104               0                    4                    0                 15                  R <br /> CDH23-213/pSSC105                                ssc-0os               0                    4                    0                 16                  R <br /> CDH23-213/pSSC106                                ssc-106               0                    3                    0                 15                  R <br /> CDH23-213/pSSC107                                ssc-107               0                    3                    0                 14                  R <br /> CDH23-213/pSSC108                                ssc-108               0                    2                    0                 14                  R <br /> CDH23-213/pSSC109                                ssc-109               2                    9                    8                 20                  S <br /> CDH23-213/pSSC110                                ssc-11O               0                   10                   11                 21                  S <br /> CDH23-213/pSSC111                                ssc-111               0                   10                   10                 20                  S <br /> CDH23-213/pSSC112                                ssc-112              18                   13                   20                 35                  SS <br /> CDH23-213/pOMSA101                              omsA                   0 (6)                2 (4)                0 (4)             12   (12)           R (S) <br /> CDH23-213/pOMSA102                              omnsAc                 0 (4)                2 (3)                0 (3)             13   (10)           R (S) <br /> CDH23-213/pOMSA103                              firA+                  0 (0)                1 (1)                0 (0)             11   (7)            R (R) <br /> CDH23-213/pGAH317                               firA+                  0                    1                    0                 13                  R <br /> CDH23-210 (omsA+ control)                                         0 (0)                1 (0)                0 (0)             13   (8)            R (R) <br />       Sensitivity antibiotics measured inhibition zones  millimeters) antibiotic discs. Determinations L agar plates  ampicillin <br /> [100 pg/mlJ plasmids used) 28&apos;C. parentheses corresponding values obtained 39&apos;C. diameter antibiotic disc (9 mm) subtracted. <br />   b R, resistant; S, sensitive; SS, sensitive. Interpretations parentheses based sensitivity determinations performed 39&apos;C. <br />   c Plasmid pOMSA102 contains firA mutant allele onsA site-directed mutagenesis. <br />  VOL. 174, 1992                                                                           OUTER MEMBRANE GENES omsA firA                    7093 <br />  <br /> TABLE 4. Complementation antibiotic-sensitive phenotype                          1000 <br />     CDH23-213 plasmids containing omsA allele <br />                   determined 37&#194;&#176;C&quot; <br />                        firA                   MIC (pg/ml)  <br /> Strain/plasmid      aliele   Vanco-     Fusidic   Ram            Erythro-    0 <br />                                                                                 0- <br />                     plasmid     mycin        acid     R              mycin <br />                                                                                 0        100 <br /> CDH23-213                       8          2       0.016         0.125     1-0 <br /> CDH23-213/         omsA             32         64       1.5           3 <br />   pOMSA101                                                                      E <br /> CDH23-213/         o                32         48         1           2                                                                                         o <br />   pOMSA102 <br /> CDH23-213/         firA+         &gt;256       &gt;256          6          12 <br />   pOMSA103                                                                                10 <br /> CDH23-210                   &gt;256       &gt;256          8         32          0 <br />   MICs determined using E Test  Materials Methods) L agar plates  ampicillin [100 mg/ml] plasmid-containing strains studied). <br />    b Prepared site-directed mutagenesis. <br />  <br />  <br />                                                                                                                                                   10 <br /> CDH23-213. firA-containing chromosomal 1.4-kb PstI <br /> fragment ligated pUC19 identified Southern                                                              Time (min) <br /> hybridization.  wild-type firA gene  plasmid                                      11 <br /> pGAH317) cloned pUC19. recombinant plasmids                                   10 tested ability complement omsA <br /> mutation CDH23-213. firA gene proximal lac <br />                                                                                                          B <br />                                                                                                10 opposite orientation, plasmids withfirA +                            10 <br /> elicited complementation (data shown). hand, wild-type firA gene orientation                                                                                                                     9 lac promoter distal (pOMSA103), fully                        c <br />  <br />                                                                                           10 <br /> complemented omsA phenotype (Table 3). identical <br /> construction containing omsA allele (pOMSA101) complemented phenotype.  comple-                            E <br />                                                                                  0 <br />  <br />                                                                                          10 <br />                                                                                                mentation tests performed 39&#194;&#176;C (instead 28&#194;&#176;C), <br /> pOMSA101 proved complement weakly                                      0 <br />  <br /> pOMSA103 (Table 3). Consistent difference ob-                                         7 <br /> served Table 3, MICs vancomycin, erythromycin,                        &apos;   0    10 <br /> rifampin, fusidic acid 37&#194;&#176;C  eightfold                      0 <br />  <br />  <br />  <br /> lower CDH23-213 carried pOMSA101                                              6 <br /> carried pOMSA103 (Table 4).                                                               10 <br />    omsA allele sequenced using oligonucleotide <br /> primers. Compared wild-type firA sequence (5), sequence identical point mutation. G transition nucleotide 812  numbering                                          0       1   00   200      300   400        500 <br /> consistent Fig. 4 reference 5). changed codon <br /> 271 serine asparagine. mutation                                                     Time (min) site-directed mutagenesis. omsA allele                            FIG. 1. Thermosensitivity growth determined strain <br /> haved way original omsA allele                           pairs analogous mutant alleles. CDH23-213 (omsA) complementation assays (Tables 3 4).                                        isogenic wild-type control, CDH23-210, SH7622 (ssc-1)    thermosensitive phenotype ssc-l mutant com-                       wild-type parent, SH5014, used. Growth measured asA550 <br /> pared oms,4 mutant. omsA ssc-1                           , viability determined CFU/milliliter (B). cultures <br /> mutations allelic, compared thermosensitive                          incubated 28&#194;&#176;C reached optical density (OD) phenotypes caused mutations. tem-                        approximately 0.2 0.3 (arrows), temperature perature shifted 28 42&#194;&#176;C, growth                     shifted 42&#194;&#176;C. Symbols: *, CDH23-210; 0, CDH23-213; *, mutants stopped 1.5 h (Fig. 1). CFU/milliliter                    SH5014; 0, SH7622. <br /> fell 1 2 log units SH7622 nearly 3 log units CDH23-213. <br />     distinct property omsA mutant                         37&#194;&#176;C) did induce marked leakage periplasmic <br /> releases periplasmic proteins temperature shift                       enzyme   SH7622/pUC19. activity medium  (39). study characteristic ssc-1               percentage total activity 82% 42&#194;&#176;C 4% mutant, introduced plasmid vector pUC19                             37&#194;&#176;C  values control plasmid 2 0.3%, <br />  codes P-lactamase) SH7622. SH7622 derivative                          respectively). Accordingly, OM SH7622 func- carries pUC19-derived pUCHS14-II  encodes                           tionally defective 42&#194;&#176;C allowed leakage 3-lactamase wild-type Ssc) used control.                      periplasmic contents. contrast, protein leakage  shown Fig. 2, temperature shift 28 42&#194;&#176;C                    cytoplasmic membrane minimal 42&#194;&#176;C;  7094                VUORIO VAARA                                                                                                                  J. BACT1ERIOL. <br />  <br />            100000 <br />                                                                                                      concluded FirA physical association                                                                                                      transcriptional machinery directly involved tran- <br />                                                                                                      scription (5).   linkedfirA apparently                                                                                                      different cellular function finding point mutation                                                                                                      ssc gene, firA analog S. typhimurium, results             GM0                                        E                                             thermosensitivity  permissive temperature,                                                        B                                             drastic increase susceptibility large set hydropho- <br />  -2w <br />                                                        -20                                           bic antibiotics, dyes, detergents (17). unselec- <br />  -C                                                    0 <br />                                                                                                      tive increase susceptibility hydrophobic agents indi- <br />                                                                                                      cates defect OM permeability barrier agents, <br />                                                                                                      suggested firA gene involved generation                                                                                                      OM. <br />                                                                                                         present investigation studied conditionally lethal <br />                                                                                                      omsA mutant E. coli, shown Tsuruoka <br />                                                                                                      et al. (39) defective OM permeability barrier <br />                                  Time (min) <br />                                                                                                      function growth-permissive temperatures.                                                                                  lime (min)          showed mutation base pair substitution <br />                                                                                                      firA, e., gene mutated mentioned <br />                                                                                                      phenotypically similar ssc-1 mutant S. typhimurium. <br />                                                                                                      Furthermore, results revealed OM firA <br />                                                                                                      mutants ruptured nonpermissive temperatures, <br />                         c                                                  D                         indicated leakage periplasmic 3-lactamase                160- 37                         X                11so       42C <br />                                                                                                      medium. interestingly, able                                                                                                      rupturing took place conditions        E <br />                                                                                                      cytoplasmic membrane remained impermeable cytoplas- <br />    .3           120 -                                                                             mic enzymes. results strengthen concept <br />                                                                                                      firA gene essential function OM biosyn- <br />                                                            2% <br />  <br />                                                                                                                                                                  thesis. Furthermore, results strongly suggest    -C <br />                   80-                                                                              gene functions identically E. coli S. typhimurium, <br />                                                                                                      proposed previously (17). <br />                                                                                                         firA genes S. typhimurium (16) E. coli (5)                 40-                                              40                                  88% identical. corresponding proteins molecular <br />                                                                                                      mass 36 kDa 96% identical. Downstream firA, <br />                                                                                                      560 bp apart, lie lpxA4 (3) lpxB (3, 4).                     0       30      60    90   120                     0    30      60    90   120   known genes lipid biosynthesis                                                                                                      cotranscribed firA  references reference 6). <br />                              Time (min)                                          Tlme (min)          firA S. typhimurium compared    FIG. 2. Release periplasmic amp-encoded 1-lactamase                                        bacterial genes GenBank library, best sequence <br /> cytoplasmic glucose 6-phosphate dehydrogenase ssc-1 mutant                                      homology lpx4 (47). Consistently, strain SH7622. SH7622 carrying pUC19 (amp, open symbols)                                      SWISS-PROT protein sequences, best match pUCHS14-II (amp ssc+, closed symbols) shifted 28                                      FirA E. coli <span id='am-91' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-acetylglucosamine O-acyltransferase <br /> 37&apos;C  C) 42&#194;&#176;C (B D) time zero. 1-Lactamase                                       LpxA protein) (47). FirA LpxA proteins share B) glucose 6-phosphate dehydrogenase (C D) activities                                    long regions distinct homology (homology allowing growth medium (0, 0) corresponding cellular <br /> activities (5, *) shown. Units nanomoles minute.                                         equivalent amino acids, 43%) peculiar hexapeptide <br />                                                                                                      repeat theme acetyltransferases (6, 41, <br />                                                                                                      47). homology LpxA suggests FirA involved <br />                                                                                                      lipid biosynthesis function 3% total activity cytoplasmic marker                                              resembles LpxA, e., acyltransferase function. <br /> enzyme (glucose 6-phosphate dehydrogenase)                                              enzymes directly involved medium 2 h 42&#194;&#176;C (Fig. 2).                                                               lipid synthesis await molecular characterization (33). <br />                                                                                                      include UDP-3-0-(3-hydroxymyristoyl)-acetylglucos- <br />                                                DISCUSSION                                            amine deacetylase UDP-3-0-(3-hydroxymyristoyl)-glu- <br />                                                                                                      cosamine N-acyltransferase  catalyze second    firA mutant (firA200) E. coli described                                       steps synthesis, respectively), lipid 4&apos; kinase, 1970 (24). rifampin-sensitive phenotypic revertant                                          late acyltransferases  e., lauroyl myristoyl trans- <br />   fir versus nij) E. coli rpoB mutant strain,                                     ferases). Work progress laboratory elucidate <br /> carries rifampin-resistant P subunit RNA polymerase.                                          possible functional role FirA biogenesis lipid MIC rifampin firA rpoB strain approxi-                                             Indirect evidence favoring role comes mately threefold lower MIC rpoB strain (25).                                        finding rate lipid synthesis, relative Furthermore, firA rpoB strain thermosensitive                                            phospholipid synthesis, drastically decreased firA RNA synthesis halted 42&#194;&#176;C (25). basis                                          mutant 42&#194;&#176;C (13). Furthermore, mature LPS molecules <br /> evidence, firA mutation believed affect RNA                                               produced firA mutant 42&#194;&#176;C modified contain <br /> transcription. Later, Dicker Seetharam (5) constructed                                           palmitic acid (12, 13). <br /> strains overproduce product offirA, 36-kDa FirA                                           lpx42 mutant E. coli isolated characterized protein, showed antiserum RNA polymerase                                              Galloway Raetz best-characterized mutant strain <br /> holoenzyme coimmunoprecipitates FirA strains.                                              defective lipid biosynthesis (7). synthesizes drasti- <br />  VOL. 174, 1992                                                           OUTER MEMBRANE GENES omsA firA                    7095 <br />  <br /> cally reduced amounts lipid 42&#194;&#176;C rapidly dies         encodes protein total different temperature. recently shown (48)            amino acid alterations, Gln-165--*Arg, Gly-228--&gt;Asp, strain extremely susceptible hydrophobic antibiotics         Gly-252--iSer (5). spontaneous mutation, firA201, (including rifampin, fusidic acid, erythromycin, clinda-       rifampin sensitive encodes protein pre- <br /> mycin) growth-permitting temperatures. MICs           served Gly-228- Asp Gly-252-*Ser mutations 32- &gt;128-fold lower strain parental       lacks Gln-165--&gt;Arg mutation Ile-172--Phe  <br /> type strain, MICs antibiotics use      stead (5). basis preliminary transcomplementation <br /> porin channels OM unchanged.            studies, Dicker Seetharam suggested Gly- low MIC values hydrophobic antibiotics deter-          228--Asp Gly-252--Ser responsible ther- <br /> mined lpxA2 strain reminiscent deter-         mosensitive rifampin-sensitive phenotypes mined omsA mutant present study. lpxA2          firA200 mutant (6). noted omsA mutants antibiotic-supersusceptible          mutations located turn residues  e., residue b, c, d <br /> enterobacterial strains published. Accordingly,           imperfect hexapeptide tandem repeat units) favor <br /> lpxA4 firA mutants phenotypically similar.            formation hypothetical 3-turn structure protein <br />    5&apos;-flanking region firA occupied hlpA         (6).  Ser-271--&gt;Asn Val-291--&gt;Met known ompH), encodes cationic 17-kDa protein              located residues. Regarding 11 novel FirA <br /> HLP (histone-like protein  OmpH protein).             proteins site-directed mutagenesis firA, protein conservative structure mem-           Met-288-*Lys Gly-289--&gt;Asp defective <br /> bers family Enterobacteriaceae (15, 18, 20, 22, 43, 46),    ones complementation assays, followed proteins <br /> carries signal sequence characteristic secreted       Met-290-.Lys Met-292--Lys. plasmid proteins precursor form (18, 20, 21, 43),      encodes Gly-289--Asp appears sensitize ssc-1 isolated OM (18, 22), unknown function.         omsA mutants (reference 19 study) nucleotides separate hlpA stop firA initiation       negligible effect sensitivity firA+ strain <br /> codons (5, 16). previously shown using minicells        (19). taken suggest mutant protein complementation assays firA gene S.             defective proteins Val-291--oMet <br /> typhimurium expressed absence hipA pro-           Ser-271--&gt;Asn competes binding <br /> moter, e., plasmid lacks 5&apos; terminus hlpA      hypothetical ligand.  Gly-289--Asp mutation vector-derived lac promoter opposite           does map hypothetical turn residues. orientation firA (17). present report,      empirical data tertiary structure FirA construction able complement omsA              protein. absence crystallographic data, mutant allele, analogous construction containing          difficult draw conclusions regarding effect firA gene E. coli inactive. firA inserts   identified amino acid alterations conformation E. coli whichfirA lac promoter comple-        protein. <br /> menting activity. previously suggested firA           various firA mutant strains certainly expected <br /> S. typhimurium use promoter            differ extents functional lesions. firA201 <br /> identified putative promoter region 159 nucleotides           mutant apparently carries defective allele stream overlaps protein-encoding region hlpA           firA200 mutant, supersusceptible rifampin <br /> (16). E. coli, region homologous            conditions firA200 strain (5, 6). canonical promoter sequence. Accordingly, sug-         phenotypically characterized mutants, onmsA mutant <br /> gested firA normally cotranscribed hipA      defective. phenotypes firA gene S. typhimurinum utilize,         omsA ssc-1 mutants similar, differences <br /> extent, additional promoter mentioned                      basis MIC values (reference 17 <br />    present multicopy plasmid lac              study), conclude omsA mutant promoter, omsA mutant allele able partially com-          susceptible tested antibiotics plement phenotype chromosomal omisA mutant.             ssc-1 mutant. Furthermore, leaks periplasmic enzymes probably overexpression plasmid-borne         37&#194;&#176;C (39), ssc-1 mutant leakage gene construction. analogous construction              detected temperature  study).  containing firA200 mutant allele   comple-            omsA mutant releases cytoplasmic constituents <br /> ments rifampin-sensitive phenotype firA200 mu-           starts lyse (39), ssc-1 strain releases <br /> tant (Table 4 reference 5). Apparently, overex-          exclusively periplasmic material. elucidation pression takes place plasmids S. typhimuinum            function FirA, defective firA mutant  e., firA lac control uses additional          omsA strain) expected informative <br /> promoter  suggested ,  shown               <br /> report, defectivefirA alleles S. typhimurium (ssc-1         firA lpx42 mutants extremely supersus- defective missense base pair substitution mutants          ceptible hydrophobic agents, previously sug- site-directed mutagenesis) complement omsA pheno-            gested (19, 48) useful tools studies type weakly complement         bacterial physiology, applied bacteriology wild-type firA S. typhimurium nondefective            biotechnology, fields permeability barrier <br /> vitro mutant alleles complement fully).                          properties OM potentially pose problems.  <br />    mutant FirA protein described report         phenotypic revertants types mutants amino acid change, Ser-271- Asn.  FirA pro-            help evaluate lipid essential molecule. <br /> tein encoded ssc-1 allele single alteration,           evidence assumption lipid Val-291-)Met (17). Accordingly, thermosensitive            attached ketodeoxyoctulosonates essential  <br />  antibiotic-supersusceptible phenotypes conferred        bility bacteria comes findings lpx42 <br />  single mutation. hand, rifampin-sensitive       mutants kds mutants (defective ketodeoxyoctu- <br />  thermosensitive phenotype-causing mutation firA200             losonate synthesis) conditionally lethal inhibi- <br />  7096      VUORIO VAARA                                                                                                    J. BACTERIOL. <br />  <br /> tors ketodeoxyoctulosonate synthesis antibacterial (7,                murium. EMBO J. 10:1017-1023. <br /> 9-11, 31, 33). bacteria live lipid              18. Hirvas, L., P. Koski, M. Vaara. 1991. ompH gene suitable bypass mutations present remains                     Yersinia enterocolitica: cloning, sequencing, expression, seen. OM certainly essential pathogenic <br />                                                                              comparison known enterobacterial ompH sequences. J. <br />                                                                              Bacteriol. 173:1223-1229. <br /> bacteria (44), inhibitors lipid biosynthesis             19. Hirvas, L., M. Vaara. 1992. Effect Ssc protein inutations promising new class antibacterial agents. Molecular                  outer membrane permeability barrier function Salmo- <br /> characterization enzymes involved lipid                   nella typhimurium: study using ssc mutant alleles biosynthesis greatly advance development                   site-directed mutagenesis. FEMS Microbiol. Lett. 90:289-294. <br /> agents.                                                                  20. Hoick, , K. Kleppe. 1988. Cloning sequencing                                                                              gene DNA-binding 17 K protein Eschenichia coli. <br />                     ACKNOWLEDGMENTS                                          Gene 67:117-124. <br />                                                                          21. Koski, P., L. Hirvas, M. Vaara. 1990. Complete sequence   thank B. Kuusela excellent technical assistance T.              ompH gene encoding 16-kDa cationic outer membrane <br /> Tsuruoka providing strains CDH23-210 CDH23-213.                  protein SalmoneUa typhimurium. Gene 88:117-120. <br />   study supported grant 1011749 Academy          22. Kosld, P., M. Rhen, J. Kantele, M. Vaara. 1989. Isolation, <br /> Finland  M.V.) Sigrid Juselius Foundation (M.V.            cloning, primary structure cationic 16-kDa outer <br /> R.V.).                                                                       membrane protein Salmonella typhimurium. J. Biol. Chem. <br />                                                                              264:18973-18980. <br />                             REFERENCES                                   23. Langdon, R. G. 1966. Glucose-6-phosphate dehydrogenase  1. Alting-Mees, M. , J. M. Short. 1989. pBluescript II: gene          erythrocytes. Methods Enzymol. 9:126-131. <br />     mapping vectors. Nucleic Acids Res. 17:9494.                         24. Lathe, R. 1977. Fine-structure mapping thefirA gene, locus <br />  2. Bullock, W. O., J. M. Fernandez, J. M. Short. 1987.                  involved phenotypic expression rifampin resistance     XL1-Blue: high efficiency plasmid transforming recA Esche-             Escherichia coli. J. Bacteriol. 131:1033-1036. <br />     richia coli strain 0-galactosidase selection. BioTechniques     25. Lathe, R., H. Buc, J.-P. Lecocq, E. K F. Banz. 1980. <br />     4:376-379.                                                               Prokaryotic histone-like protein interacting RNA polymer- <br />  3. Coleman, J., C. R. H. Raetz. 1988. committed step           ase. Proc. Natl. Acad. Sci. USA 77:3548-3552. <br />     lipid biosynthesis Escherichia coli: sequence lpx4       26. Lugtenberg, B., L. van Alphen. 1983. Molecular architec- <br />     gene. J. Bacteriol. 170:1268-1274. <br />                                                                              ture functioning outer membrane Eschenichia coli <br />  4. Crowell, D. N., W. S. Reznikoff, C. R. H. Raetz. 1987.               gram-negative bacteria. Biochim. Biophys. Acta <br />     Nucleotide sequence Escherichia coli gene lipid             737:51-115. <br />     disaccharide synthase. J. Bacteriol. 169:5727-5734.                  27. Messing, J., J. VMeira. 1982. new pair M13 vectors  5. Dicker,  B., S. Seetharam. 1991. Cloning nucleotide            selecting DNA strand double-digest restriction frag- <br />     sequence firA gene firA200(Ts) allele                ments. Gene 19:269-276. <br />     Escherichia coli. J. Bacteriol. 173:334-344.                         28. Nikaido, H. 1976. Outer membrane Salmonella typhimurium. <br />  6. Dicker,  B., S. Seetharam. 1992. known           Transmembrane diffusion hydrophobic substances. <br />     structure function Escherichia coli protein FirA?             Biochim. Biophys. Acta 433:118-132. <br />     Mol. Microbiol. 6:817-823.                                           29. Nikaido, H. 1990. Permeability lipid domains bacterial <br />  7. Gailoway, S. M., C. R. H. Raetz. 1990. mutant                   membranes. Adv. Membr. Fluid. 4:165-190. <br />     Escherichia coli defective step endotoxin biosyn-    30. Nikaido, H., M. Vaara. 1985. Molecular basis bacterial <br />     thesis. J. Biol. Chem. 265:6394-6402.                                    outer membrane permeability. Microbiol. Rev. 49:1-32. <br />  8. Gibson, T. J. 1984. Studies Epstein-Barr virus genome.        31. Nikaido, H., M. Vaara. 1987. Outer membrane, p. 7-22.     Ph.D. thesis. Cambridge University, Cambridge.                           F. C. Neidhardt, J. L. Ingraham, K. B. Low, B. Magasanik, M. <br />  9. Goldman, R. C., C. C. Doran, J. 0. Capobianco. 1990.                 Schaechter, H. E. Umbarger (ed.), Escherichia coli     Antibacterial agents specifically inhibit lipopolysaccha-          Salmonella typhimunum: cellular molecular biology. Amer- <br />     ride synthesis, p. 157-167.  Nowotny, J. J. Spitzer,            ican Society Microbiology, Washington, D.C. <br />     J. R. Sokatch (ed.), Cellular molecular aspects endotoxin     32. O&apos;Callaghan, C. H.,  Morris, S. M. Kirby,  H. Shigler. <br />     reactions. Elsevier Science Publishers B.V., Amsterdam.                  1972. Novel method detection 1-lactamases using 10. Goldman, R. C., W. E. Kohlbrenner, P. Lartey,  Pernet.             chromogenic cephalosporin substrate. Antimicrob. Agents Che- <br />     1987. Antibacterial agents specifically inhibiting lipopolysaccha-       mother. 1:283-288. <br />     ride synthesis. Nature (London) 329:162-164.                         33. Raetz, C. R. H. 1990. Biochemistry endotoxins. Annu. Rev. <br /> 11. Hammond, S. M.,  Claesson,  M. Jansson, L. G. Larsson,               Biochem. 59:129-170. <br />     B. G. Pring, C. M. Town, B. Ekstr6m. 1987. new class        34. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br />     synthetic antibacterials acting lipopolysaccharide biosynthe-         cloning: laboratory manual, 2nd ed. Cold Spring Harbor <br />     sis. Nature (London) 327:730-732.                                        Laboratory Press, Cold Spring Harbor, N.Y. <br /> 12. Helander,  M., L. Hirvas, J. Tuominen, M. Vaara. 1992.         35. Sanger, F., S. Nicklen,  R. Coulson. 1977. DNA sequenc- <br />     Preferential synthesis heptaacyl lipopolysaccharide            ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br />     ssc permeability mutant Salmonella typhimunium. Eur. J.               USA 74:5463-5467. <br />     Biochem. 204:1101-1106.                                              36. Stocker, B.  D., M. Nurminen, P. H. Mikeli. 1979. <br /> 13. Helander,  M., B. Lindner, U. Seydel, M. Vaara. Submit-            Mutants defective 33K outer membrane protein Sal- <br />     ted publication.                                                     monella typhimurium. J. Bacteriol. 139:376-383. <br /> 14. Helander,  M., M. Vaara, S. Sukupolvi, M. Rhen, S. Saarela,        37. Sukupoli, S., M. Vaara. 1989. Salmonella typhimurium     U. Zahringer, P. H. Mikela. 1989. rfaP mutants Salmo-             Escherichia coli mutants increased outer membrane  <br />     nella typhimurium. Eur. J. Biochem. 185:541-546.                         meability hydrophobic compounds. Biochim. Biophys. Acta <br /> 15. Hirvas, L., J. Coleman, P. Koski, M. Vaara. 1990. Bacterial          988:377-387. <br />     &quot;histone-like protein  (HLP  outer membrane constit-       38. Sukupolvi, S., M. Vaara,  M. Helander, P. VlUanen, P. H. <br />     uent? FEBS Lett. 262:123-126.                                            Mikela. 1984. New Salmonella typhimurium mutants al- <br /> 16. Hirvas, L., P. Koski, M. Vaara. 1990. Primary structure          tered outer membrane permeability. J. Bacteriol. 159:704-712. <br />     expression Ssc-protein Salmonella typhimurium. Bio-        39. Tsuruoka, T., M. Ito, S. Tomioka,  Hirata, M. Matsuha- <br />     chem. Biophys. Res. Commun. 173:53-59.                                   shi. 1988. Thermosensitive onsA mutation Escherichia coli <br /> 17. Hirvas, L., P. Koski, M. Vaara. 1991. Identification             causes thermoregulated release periplasmic proteins. J. <br />     sequence analysis gene mutated conditionally lethal        Bacteriol. 170:5229-5235. <br />     outer membrane permeability mutant SS-C Salmonella typhi-         40. Vaara, M. 1992. Agents increase permeability  VOL. 174, 1992                                                              OUTER MEMBRANE GENES              omsA   firA        7097 <br />  <br />     outer membrane. Microbiol. Rev. 56:395-411.                       46. Vuorio, R., L. Hirvas, R. B. Raybourne, D. T. Y. Yu, M. <br /> 41. Vaara, M. bacterial proteins, including UDP-N-acetylglu-        Vaara. 1991. nucleotide deduced amino acid sequence <br />     cosamine acyltransferase (LpxA) transferases       cationic 16 kDa outer membrane protein OmpH     Eschenchia coli, consist residue periodicity theme.          Yersinia pseudotuberculosis. Biochim. Biophys. Acta 1129: <br />     FEMS Microbiol. Lett., press.                                      124-126. <br /> 42. Vaara, M. Quantitative antimicrobial susceptibility testing    47. Vuorio, R., L. Hirvas, M. Vaara. 1991. Ssc protein     outer membrane-defective mutant bacteria using E test.         enteric bacteria significant homology acyltransferase <br />     J. Antimicrob. Chemother., press.                                  LpxA lipid biosynthesis, acetyltransferases. <br /> 43. Vaara, M., L. Hirvas, P. Koski. 1990. cationic 16-kDa         FEBS Lett. 292:90-94. <br />     outer membrane protein OmpH enteric bacteria, p. 197-204. <br />      Nowotny, J. J. Spitzer, E. J. Ziegler (ed.), Cellular   48. Vuorio, R., M. Vaara. 1992. lipid biosynthesis <br />     molecular aspects endotoxin reactions. Elsevier Science        mutation lpx42 Escherichia coli results drastic antibiotic <br />     Publishers B.V., Amsterdam.                                           supersusceptibility. Antimicrob. Agents Chemother. 36:826- <br /> 44. Vaara, M., H. Nikaido. 1984. Molecular organization            829. <br />     bacterial outer membrane, p. 1-45. E. T. Rietschel (ed.),      49. Wickner, W.,  J. Driessen, F. U. Hartl. 1991.     Chemistry endotoxin. Elsevier Science Publishers B.V.,             enzymology protein translocation Escherichia coli <br /> </body></html>