<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Yeast Nop15p RNA-binding protein required pre-rRNA processing cytokinesis <br />  <br />  <br />  <br />                              NIFK localized regions condensed mitotic <br />                                                                    chromosomes, interact forkhead- <br />  <br />                                               associated (FHA) domain cell proliferation marker <br />  <br />                                                                    pKI-67, proposed function mitosis (Sueishi <br />  <br />                                                                    et al., 2000; Takagi et al., 2001). database search  <br />                                                                    sequence clearly identi&#194;&#174;ed YNL110c probable <br />                                                                    yeast homologue (32% identity, 55% similarity 174 <br /> Nop15p essential protein contains RNA <br /> recognition motif (RRM) localizes nucleo-               amino acids). YNL110c/NOP15 reported essen- <br />                                                                    tial viability systematic analyses  Antoni et al., <br /> plasm nucleolus. Cells depleted Nop15p failed <br />                                                                    1997; Capozzo et al., 2000). Spores carrying ynl110D synthesize 25S 5.8S rRNA components                                                                    mutation failed bud arrested &#194;&#174;rst 60S ribosomal subunit, exonucleolytic 5&#194;&#162; pro- <br />                                                                    cycle, producing abnormal bud smaller larger <br /> cessing 5.8S rRNA strongly inhibited. Pre- <br />                                                                    wild-type. led suggestion Ynl110p/ <br /> rRNAs precipitated tagged Nop15p rmed <br />                                                                    Nop15p functions cell cycle (Capozzo et al., association early pre-60S particles                                                                    2000). <br /> Nop15p bound pre-rRNA transcript vitro. <br />                                                                       Recent papers described factors function <br /> Nop15p-depleted cells unusually abrupt <br />                                                                    ribosome synthesis activities required cell <br /> growth arrest prior substantial depletion riboso- <br /> mal subunits. Following cell synchronization mito-              cycle progression. Net1p/C&#194;&#174;1p regulates release                                                                    activation Cdc14p phosphatase, triggering <br /> sis, Nop15p-depleted cells undergo nuclear division <br />                                                                    mitotic exit (Shou et al., 1999; Straight et al., 1999; wild-type kinetics, activate mitotic exit net- <br /> work disassemble mitotic spindle.                Visintin et al., 1999), promotes transcription                                                                    pre-rRNA RNA polymerase (Shou et al., 2001). uniformly arrest cytokinesis fail assem- <br />                                                                    Yph1p/Nop7p Noc3p associated origin ble contractile actin ring bud neck. dividing <br />                                                                    replication complex (ORC) play key roles wild-type cells, segregation nucleolar proteins                                                                    initiation DNA replication (Du Stillman, 2002; <br /> daughter nuclei occurs separation nucleo- <br />                                                                    Zhang et al., 2002).  Nop7p Noc3p plasm. late mitotic cells, Nop15p partially <br />                                                                    components pre-60S particles required <br /> delocalized nucleolus nucleoplasm, <br />                                                                    pre-rRNA processing, 60S subunit synthesis export <br /> consistent speci&#194;&#174;c function cell division                                                                    cytoplasm (Harnpicharnchai et al., 2001; Milkereit <br /> addition role ribosome synthesis. <br />                                                                    et al., 2001; Adams et al., 2002; Oef&#194;&#174;nger et al., 2002). <br /> Keywords: cell division/nucleolus/pre-rRNA/ribosome <br /> synthesis/Saccharomyces cerevisiae                                 60S synthesis factor Sda1p (Ba&#195;&#187;ler et al., 2001; <br />                                                                    Harnpicharnchai et al., 2001; Ihmels et al., 2002)                                                                    required pass `Start&apos; checkpoint G1&#194;&#177;S <br />                                                                    boundary (Buscemi et al., 2000; Zimmerman Introduction                                                       Kellogg, 2001) allele reported lead steps ribosome synthesis place             defects actin cytoskeleton.  high- <br /> nucleolus, specialized subnuclear structure.              throughput screen implicated ribosome <br /> ribosome synthesis, complex processing pathway               synthesis factors ribosomal proteins determining <br /> verts large pre-rRNA mature 18S, 5.8S 25S/            timing Start, size cell 28S rRNAs. addition, mature rRNA sequences                  division (Jorgensen et al., 2002). pre-RNA undergo extensive covalent <span id='am-18' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span>              work progress, Nop15p identi&#194;&#174;ed <br /> modi&#194;&#174;cation assemble ~80 ribosomal proteins.              component pre-60S particle, strain 140 non-ribosomal proteins required         expressing haemaglutinin (HA)-tagged form ribosome synthesis identi&#194;&#174;ed genetic              Nop15p showed reduction 60S subunit accumu- <br /> biochemical approaches Saccharomyces cerevisiae             lation (Harnpicharnchai et al., 2001). Kressler et al., 1999; Venema Tollervey, 1999;                 Nop15p required pre-rRNA processing, Warner, 2001; Fatica Tollervey, 2002).                         speci&#194;&#174;c function cell cycle progression. <br />    Far fewer human ribosome synthesis factors characterized, product open reading frame <br /> (ORF) NNP18/NOPP34/hNIFK shown associ- <br />                                                                    Results <br /> ated pre-ribosomal particles (Fujiyama et al., 2002).         Cells depleted Nop15p defective 60S <br /> NIFK reported localize nucleolus (Takagi             subunit synthesis pre-rRNA processing <br /> et al., 2001) identi&#194;&#174;ed proteomic analysis         predicted ORF YNL110c/NOP15 contains 220 <br /> puri&#194;&#174;ed human nucleoli (Andersen et al., 2002).            amino acids (25.4 kDa), analysis PSORT (Nakai <br />  <br /> &#195;&#163; European Molecular Biology Organization                                                                                 6573 <br />  M.Oef&#194;&#174;nger D.Tollervey <br />  <br />  <br />                                                                                Horton, 1999) identi&#194;&#174;ed canonical RNA recog- <br />                                                                                nition motif (RRM; residues 92&#194;&#177;168) putative <br />                                                                                nuclear localization signals (NLS), consistent role <br />                                                                                nuclear RNA processing. <br />                                                                                   examine function Nop15p, expression                                                                                placed control repressible galactose <br />                                                                                promoter using step PCR technique strain <br />                                                                                YDL401  Materials methods). Growth                                                                                GAL::nop15 strain (YMO11) clearly different <br />                                                                                isogenic wild-type strain (YDL401)                                                                                permissive, galactose medium, progressively <br />                                                                                slowed following transfer repressive, glucose medium <br />                                                                                (Figure 1). Mild growth impairment initially seen, <br />                                                                                commencing ~6 h transfer, stringent <br />                                                                                growth inhibition 20 h. sharp ection point <br />                                                                                growth curve complete growth inhibition Fig. 1. Depletion Nop15p leads abrupt growth arrest. Growth              unusual features strains ribosome synthesis <br /> curves GAL::nop15 strains following transfer glucose medium,             factors depleted GAL repression, expression Rpl25p-eGFP. Strains pre-grown             complete  . RGS medium transferred glucose medium times                     determine depletion Nop15p inhibits <br /> indicated. Strains maintained exponential growth dilution pre-warmed medium. Cell densities measured OD600 shown             rRNA synthesis, metabolic labelling [H3]<span id='am-36' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> corrected dilution. Wild-type (open squares); GAL::nop15; (&#194;&#174;led            performed 16 h transfer glucose minimal medium <br /> diamonds); GAL::nop15; rpl25-GFP (&#194;&#174;lled squares).                              (Figure 2). GAL::nop15 (YMO14) strain, synthesis <br />  <br />  <br />  <br />  <br /> Fig. 2. Pulse&#194;&#177;chase analysis rRNA synthesis.   Structure processing sites 35S pre-rRNA. precursor contains sequences mature 18S, 5.8S 25S, separated internal transcribed spacers ITS1 ITS2 &#194;&#175;anked external transcribed <br /> spacers 5&#194;&#162;ETS 3&#194;&#162;ETS. positions oligonucleotide probes utilized northern hybridization primer extension analyses indicated. <br /> (B) Pre-rRNA processing pathway. discussion processing pathway enzymes, Kressler et al. (1999), Venema Tollervey <br /> (1999), Warner (2001) Fatica Tollervey (2002). (C D) Pre-rRNA pulse-labelled [3H]uracil 2 min 30&#194;&#176;C chased large excess unlabelled uracil times indicated. Labelling performed GAL::nop15 strain wild-type strain 16 h transfer glucose medium. (C) High molecular weight RNA separated 1.2% agarose/<span id='am-26' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> gel. (D) Low molecular weight RNA separated 8% polyacrylamide/<span id='am-41' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span> gel. <br />  <br />  <br /> 6574 <br />                                                                                           Ribosome synthesis cell division <br />  <br />  mature 25S 5.8S rRNA completely                depletion cytoplasmic ribosomes defects inhibited (Figure 2C D). 27SA pre-rRNA               assembly export 60S subunits. <br /> accumulated, 27SB 7S pre-rRNAs lost. contrast, synthesis 20S pre-rRNA 18S rRNA          Nop15p associated precursors 25S <br /> remained robust. Synthesis 5S rRNA tRNA           5.8S rRNAs little affected (Figure 2D).                                RNAs precipitated TAP-tagged Nop15p <br />     effects Nop15p depletion pre-rRNA process-         (Figure 4A&#194;&#177;C, Nop15p-TAP lanes) compared ing assessed northern hybridization       mock precipitation non-tagged parental strain primer extension (Figure 3). Depletion Nop15p             YDL401 (Figure 4A&#194;&#177;C, Mock lanes). Northern hybridiza- <br /> resulted accumulation 35S primary                 tion (Figure 4A) showed 7SL 7SS pre-rRNA <br /> transcript little impact levels 27SA2       components pre-60S ribosomes precipitated 20S pre-rRNAs (Figure 3A). indicates early         Nop15p-TAP, detectably recovered mock <br /> cleavages sites A0, A1 A2 kinetically delayed     precipitation YDL401. U3 snoRNA, occur good ef&#194;&#174;ciency. contrast, levels      component 90S pre-ribosomes, precipitated 27SB pre-rRNAs mildly reduced 8 h             lower ef&#194;&#174;ciency. 6S pre-rRNA mature <br /> strongly reduced 16 h transfer glucose medium.       rRNAs precipitated detectably. High mol- <br /> Levels 27SA2 + A3 pre-rRNAs,            ecular weight pre-rRNAs analysed primer exten- <br /> detected oligo 020 (Figure 3A, b), mildly                sion (Figure 4B C). 27SA2, 27SBL, 27SBS increased following Nop15p depletion. contrast        26S pre-rRNA components pre-60S ribosomes  depletion 27SA2 (Figure 3A, , suggesting        precipitated Nop15p-TAP. contrast, pre- 27SA3 pre-rRNA accumulated.                 cipitation 35S pre-rRNA observed. <br /> &#194;&#174;rmed primer extension (Figure 3C), showed                 results Nop15p associated little alteration level 27SA2, shown         early pre-60S ribosomes. precipitation U3 <br /> primer extension stop site A2, strong accumulation        snoRNA reveals association earlier pre- 27SA3, shown stop site A3. level 27SBS       ribosomes.  lack precipitated 35S clearly reduced, level 27SBL appeared         indicates binding later 32S pre-rRNA, unaffected, shown primer extension stops B1S        A2 cleavage occurs. Nop15p RRM B1L, respectively. exonucleases Rat1p Xrn1p          bind directly pre-rRNA, probably immediately prior <br /> normally digest pre-rRNA site A3 site B1S            cleavage site A2, remains associated pre- <br /> (Henry et al., 1994), loss 27SBS       60S particle cleavage site C2. <br /> consequence defective processing 27SA3.                       test directly Nop15p exhibits RNA binding <br />     Analysis low molecular weight RNAs (Figure 3B)            activity, gel shift assay performed (Figure 4D). showed 7S 6S pre-rRNAs mildly                  labelled pre-rRNA transcript prepared, extends <br /> depleted 8 h strongly depleted 16 h           5&#194;&#162; region ITS1 3&#194;&#162; region ITS2 transfer glucose medium (Figure 3Ba). contrast      includes sites A2, A3, B1 C2. Labelled transcript (10 <br /> 27SB pre-rRNAs, clear differences observed            fmol) incubated 0&#194;&#177;30 fmol recombinant ratios long short forms 7S pre-rRNA        Nop15p, expressed Escherichia coli (7SL 7SS), showing processing 27SBL           thrombin-cleavable GST fusion construct  Materials <br /> 7SL inhibited  Figure 2A). Little depletion       methods). Free bound forms RNA mature 18S, 25S 5.8S rRNA seen late time          resolved native gel electrophoresis (Fatica et al., 2002). <br /> points. conclude Nop15p required                  Nop15p clearly retarded migration RNA, <br /> exonucleolytic processing 27SA3 pre-rRNA               bound material did migrate discrete <br /> 27SBS. alternative form, 27SBL, continues              species. Binding inhibited cold competitor pre- <br /> synthesized fails converted 7SL.                    rRNA transcript, inhibited 200-fold molar <br />     results demonstrate Nop15p bona           excess tRNA. conclude Nop15p RNA <br /> pre-rRNA processing factor speci&#194;&#174;c functions         binding activity. analyses required synthesis 25S 5.8S rRNAs.               determine vitro binding sequence <br /> striking mature rRNAs substantially            speci&#194;&#174;c. <br /> depleted 24 h transfer glucose medium. mature rRNAs stable backgrounds          Cells depleted Nop15p non-viable depleted growth, slow Nop15p-            arrest abnormal morphology <br /> depleted strain late times.                                   association Nop15p pre-rRNAs     test potential defects 60S subunit export,     processing defects seen following depletion clearly <br /> Rpl25p&#194;&#177;enhanced green &#194;&#175;uorescent protein (eGFP)                  demonstrate Nop15p genuine ribosome synthesis <br /> reporter (Gadal et al., 2001) expressed               factor.  sudden dramatic growth arrest <br /> GAL::nop15 strain. nuclear accumulation Rpl25p-            seen depletion Nop15p (Figure 1) expected eGFP observed Nop15p depletion (data                  strains carrying GAL-regulated allele ribosome <br /> shown). Expression Rpl25p-eGFP synergistically                synthesis factor seen inhibits growth mutants ribosome assembly           published analyses. GAL repression complete, defects (Oef&#194;&#174;nger et al., 2002) did alter kinetics   strains generally continue growth reduced rate growth inhibition GAL::nop15 strain (Figure 1).        supported residual ribosome synthesis. <br />      observations indicate growth arrest     growth arrest Nop15p-depleted strain Nop15p-depleted strains consequence               suggests function addition ribosome synthesis, <br />  <br />                                                                                                                         6575 <br />  M.Oef&#194;&#174;nger D.Tollervey <br />  <br />  <br />  <br />  <br />                                                                             Fig. 4. Nop15p associates pre-rRNAs vivo vitro.   <br />                                                                             C) Nop15p-TAP precipitates pre-rRNAs. Lane 1, total RNA <br /> Fig. 3. Analysis pre-rRNA processing.   Northern analysis. Lanes       control (5 mg); lane 2, mock precipitate wild-type control strain; <br /> 1 2, wild-type strain RGS medium 24 h transfer glu-     lane 3, precipitate strain expressing Nop15p-TAP.   Northern <br /> cose; lanes 3&#194;&#177;7, GAL::nop15 strain RGS medium transfer         hybridization high low molecular weight RNAs separated glucose medium times indicated. (B) Northern analysis.           1.2% agarose/formaldehyde gel 8% polyacrylamide/urea gel, <br /> Lane 1, wild-type strain 24 h transfer glucose; lanes 2&#194;&#177;7,         respectively. (B C) Primer extension analyses. Nop15p-TAP GAL::nop15 strain RGS medium transfer glucose med-          immunoprecipitated cell lysates using IgG&#194;&#177;agarose, release <br /> ium times indicated. RNA separated 1.2% agarose        bound RNA&#194;&#177;protein complexes cleavage protein linker <br /> maldehyde gel   8% polyacrylamide/urea gel (B). Probe names            TEV protease. RNA recovered released material, indicated parentheses left. (C) Primer extension using        mock-treated, isogenic wild-type control strain. <br /> oligo 006, hybridizes ITS2, 3&#194;&#162; site C2  3&#194;&#162; end   Oligonucleotides used indicated parentheses. (D) Nop15p binds <br /> 7S pre-rRNA). Primer extension stops sites A2, A3, B1S B1L           pre-rRNA vitro. Gel mobility shift assay performed levels 27SA2, 27SA3, 27SBS 27SBL pre-rRNAs,                 vitro transcribed pre-rRNA fragment, extending 5&#194;&#162; region <br /> respectively. Lanes 1 2, wild-type strain RGS medium 20 h        ITS1 3&#194;&#162; region ITS2. Nop15p expressed E.coli transfer glucose medium; lanes 3 4, GAL::nop15 strain       GST fusion thrombin-sensitive linker eluted RGS medium 20 h transfer glucose medium.                       <span id='am-6' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-7' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span>glutathione</span>&#194;&#177;Sepharose column cleavage thrombin. Lanes 1&#194;&#177;4, <br />                                                                             10 fmol pre-rRNA incubated 0&#194;&#177;30 fmol Nop15p indicated; <br />                                                                             lanes 5&#194;&#177;8, competition experiment. gel shift assay performed <br />                                                                             presence 200-fold molar excess cold pre-rRNA (lane 7) particularly occurs prior substantial depletion             tRNA (lane 8). Reactions lanes 6&#194;&#177;8 contained 10 fmol RNA mature rRNAs.                                                               15 fmol Nop15p. Complexes resolved electrophoresis native <br />                                                                             6% acrylamide/bisacrylamide (80:1) gels containing 0.53 TBE. <br />    Cell viability assessed incubation glucose <br /> medium plating cells solid galactose medium. <br /> Cell numbers calculated counting haemocyt-                        Microscopic inspection GAL::nop15 strain 18 h <br /> ometer slide. 16 h transfer glucose medium,                 transfer glucose medium showed 63% unbudded <br /> viability Nop15p-depleted cells dropped                       cells  600 cells counted), indicating arrest G1. <br /> ~20%, 4-fold lower wild-type GAL::nop15                      37% cells showed distinctive strain grown galactose medium. (Figure 5A).                         unusual elongated shape, observed similar analysis performed GAL::nop1 strain, viabil-                   galactose medium, demonstrating defect cell morph- <br /> ity dropped later depletion (commencing                        ology (Figure 5B). elongated cells contained 60 h), slow decline growth rate continued                    duplicated, separated nuclei shown 4&#194;&#162;,6-diamidino- 80 h depletion (Tollervey et al., 1991).                     2-phenylindole (DAPI) staining marked  <br /> inability cells recover depletion Nop15p                  striction corresponding bud neck (indicated strongly suggested reduced numbers ribosomes                     arrow Figure 5B), cytokinesis major cause growth arrest.                                       occurred. morphology distinct seen  <br /> 6576 <br />                                                                                                            Ribosome synthesis cell division <br />  <br />  <br /> cells defective mitotic exit network (MEN),                      Institute), 35% cells arrested large buds arrest intact, elongated mitotic spindle (Jimenez                     divided nuclei 2 h transfer 37&#194;&#176;C (data et al., 1998). rm distinction, anti-tubulin                       shown), demonstrating phenotype does staining performed wild-type GAL::nop15                            require long incubation restrictive conditions. <br /> cells 18 h glucose medium observe                                Bud neck constriction cytokinesis involves organization microtubules (Figure 5C).                         actin/myosin contractile ring (Bi et al., 1998). ability <br /> elongated GAL::nop15 cells contained mitotic spindle                        Nop15p-depleted cells form ring  &gt;500 cells examined), rounded cells                           assessed using rhodamine-conjugated phalloidin visua- <br /> arrested G1 displayed apparently normal interphase                      lize organization polymerized &#194;&#174;lamentous actin <br /> array microtubules (Figure 5C, d). cells examined                       (F-actin). wild-type (Figure 5D,  d j), F-actin <br /> showed morphology consistent metaphase arrest.                           observed cortical patches concentrated <br />    collection `heat-inducible-degron&apos; strains                   bud neck dividing cells (indicated arrows <br /> described recently (Kanemaki et al., 2003),                     Figure 5D). organization observed target protein rapidly degraded following transfer                  GAL::nop15 strain permissive RGS medium (Figure 5D, <br /> 37&#194;&#176;C. strain expressing degron-tagged Nop15p                         b c) 14 h shift glucose medium (Figure 5D, <br /> construct (generously provided K.Labib, Paterson                           e f). 18 h glucose medium, rounded <br />  <br />  <br />  <br />  <br /> Fig. 5. Depletion Nop15p leads cell cycle defect.   Percentage cells wild-type GAL::nop15 strains remain viable pla- <br /> ted fresh, pre-warmed YRGS plates following incubation glucose medium times indicated. Viability determined comparing total number cells plated number colonies formed time point. (B) Cell morphology. (C) Staining anti-tubulin. (D) Staining rhodamine-conjugated phalloidin detect F-actin. Wild-type GAL::nop15 strains examined growth RGS medium follow- <br /> ing transfer glucose medium 14 18 h. position nucleus visualized DAPI staining. (C) (D), &#194;&#174;gures indicated asterisk phase contrast images cells corresponding immuno&#194;&#175;uorescence images. bar represents 10 mm arrows indicate position bud neck. <br />  <br />  <br />                                                                                                                                               6577 <br />  M.Oef&#194;&#174;nger D.Tollervey <br />  <br />  <br />  <br />  <br /> 6578 <br />                                                                                                            Ribosome synthesis cell division <br />  <br />  <br /> GAL::nop15 cells arrested G1 showed randomly                               presumably leads observed failure undergo <br /> distributed F-actin patches (Figure 5D, l).                           cytokinesis. elongated cells  &gt;500 cells examined) <br /> contained detectable F-actin bud neck (arrow                        nucleolus segregates late Nop15p <br /> Figure 5D, k), indicating actin depolymerized                      partially delocalized nuclear cells.                                                                  separation <br />    analyse cell cycle defect better, wild-type                    Nop15p reported previously localized GAL::nop15 cells examined following synchroniza-                         nucleolus, based exclusion nucleoplasmic <br /> tion. Cell division arrested spindle assembly                      region nucleus (Harnpicharnchai et al., 2001). checkpoint nocodazole treatment (Tolliday et al.,                          rmed localization indirect immuno- <br /> 2002), inhibits microtubule assembly, preventing                        &#194;&#175;uorescence Nop15p-TAP DsRed fusion spindle formation nuclear division. Nocodazole                        characterized nucleolar protein Nop1p, yeast <br /> added 15 h shift glucose medium, time                   homologue human &#194;&#174;brillarin (data shown) (Gadal GAL::nop15 cells cycling, wild-type                    et al., 2001). interphase cells, Nop15p-TAP largely mutant uniformly arrested large-budded cells.                    excluded nucleoplasm, decorated 3 h incubation, nocodazole washed nuclear                     DAPI staining Figure 7A, c). localization separation actin ring formation monitored                            Nop15p dramatically altered mitotic cells, <br /> microscopically. wild-type Nop15p-depleted                        &#194;&#174;lling entire nuclear volume, shown  <br /> cells, mitosis resumed similar kinetics, shown                     localization protein tag DAPI-stained high ef&#194;&#174;ciency nuclear separation 30 min                         region (Figure 7A, d&#194;&#177;f). second striking observation release nocodazole arrest (Figure 6A C).                             late mitotic cells nucleoplasm  wild-type cells carried                         daughter nuclei separated shown DAPI <br />                                                                               staining (Figure 7A, d), Nop15p-stained material formed cytokinesis, Nop15p-depleted cells uniformly arrested <br />                                                                               bridge nuclei (Figure 7A, e). large budded cells (Figure 6A). Wild-type cells <br />                                                                                  determine late segregation Nop15p exhibited peak actin ring formation 30 min                                                                               general feature nucleolar proteins, examined <br /> release nocodazole arrest, ~29% cells <br />                                                                               behaviour Nop1p (Figure 7B). interphase <br /> showing actin ring (Figure 6B C), good                              mitotic cells, Nop1p largely excluded agreement previous reports (Tolliday et al., 2002).                      nucleoplasm. mitotic cells, segregation Nop1p striking contrast, Nop15p-depleted cells F-                     daughter nuclei occurred substantial separation <br /> actin ring detected 30, 45 60 min nocodazole                    nucleoplasm. <br /> release  &gt;500 cell examined) (Figure 6B C).                             conclude nucleolus divided    total level actin present cells assessed                 segregated daughter nuclei separation western blotting (Figure 6D), revealed clear                      nucleoplasm. clearly essential cytokinesis <br /> difference mutant wild-type. ability                      commence daughter nucleoli Nop15p-depleted cells activate MEN                          completely separated. suggests possible rationale <br /> assessed examining localization Cdc14p&#194;&#177;GFP                          linking timing actin ring formation nucleolar <br /> (Figure 6E) [kindly provided E.Schiebel (Hofken                        components. release Nop15p nucleolus Schiebel, 2002)]. wild-type Nop15p-                           time functionally related triggering depleted cells, Cdc14p&#194;&#177;GFP predominantly nucleo-                          cytokinesis, mechanisms involved remain plasmic, indicating Nop15p required                       elucidated. <br /> release nucleolar sequestration. <br />    Nop15p-depleted cells arrested correctly spindle assembly checkpoint subsequently able                        Discussion undergo nuclear separation, activation MEN                      Nop15p ribosome synthesis <br /> disassembly mitotic spindle ef&#194;&#174;ciency                    Nop15p bona  ribosome wild-type, showing non-                        synthesis factor, required 5&#194;&#162; 3&#194;&#162; <br /> speci&#194;&#174;c defect cell cycle progression.                         exonuclease digestion generates 5&#194;&#162; end Nop15p-depleted cells uniformly arrested cytokinesis                       major, short form 5.8S rRNA failed form actin ring bud neck.                          processing 27SB 7S pre-rRNA. Consistent  <br /> conclude Nop15p required, directly indirectly,                     recent proteomic analyses (Harnpicharnchai recruitment actin developing bud neck.                      et al., 2001; Gavin et al., 2002), tagged Nop15p consequent failure form normal contractile ring                      associated RNA components early pre-60S <br />  <br />  <br /> Fig. 6. Cytokinesis actin ring formation inhibited synchronized, Nop15p-depleted cells. Wild-type GAL::nop15 cells shifted glu- <br /> cose medium 15 h treated nocodazole 3 h, arresting cells late M phase. Cells examined nocodazole washout <br /> (0 min samples) times points following release arrest.   Ef&#194;&#174;ciency nuclear separation. (B) Ef&#194;&#174;ciency actin ring formation. <br /> (C) Typical wild-type GAL::nop15 cells visualized 30 min nocodazole release. F-actin visualized staining rhodamine-conjugated <br /> phalloidin (panels b). Nuclei visualized DAPI staining (panels c d). (D) Western blot analysis Act1p wild-type GAL::nop15 cells. cell extracts prepared equal amounts protein separated SDS&#194;&#177;PAGE. Act1p detected western <br /> blotting using primary anti-actin antibody  19) secondary peroxidase-conjugated rabbit-anti goat antibody. (E) Release Cdc14p nucleolus nucleoplasm late mitosis. release Cdc14p&#194;&#177;GFP mitotic exit observed wild-type (panel b) GAL::nop15 (panel e) <br /> cells 30 min release nocodazole arrest. Nuclei visualized DAPI staining (panels d). bar represents 10 mm. <br />  <br />  <br />                                                                                                                                                6579 <br />  M.Oef&#194;&#174;nger D.Tollervey <br />  <br />  <br />                                                                          depletion Nop15p ts-degron strain (Kanemaki et al., <br />                                                                          2003) led similar proportion cells apparently <br />                                                                          arrested cytokinesis 2 h transfer non- <br />                                                                          permissive temperature. Following cell synchronization                                                                          late M-phase nocodazole treatment release,                                                                          defect cytokinesis complete, Nop15p- <br />                                                                          depleted cells uniformly arresting &#194;&#174;rst cytokinesis. <br />                                                                             S.cerevisiae, cortical patches polymerized F-actin <br />                                                                          support ellipsoid outgrowth bud cell wall                                                                          S phase. late mitosis, actin partially relocated                                                                          bud neck form actin/myosin contractile ring (Bi <br />                                                                          et al., 1998) constrict bud neck leading                                                                          cytokinesis. Staining rhodamine-conjugated phalloi- <br />                                                                          din revealed arrested, Nop15p-depleted <br />                                                                          cells assembled actin ring  &gt;500 examined), <br />                                                                          potentially explaining failure complete cytokinesis <br />                                                                          (Bi et al., 1998). General defects actin cytoskeleton <br />                                                                          lead defects cell polarity spindle formation <br />                                                                          (Palmer et al., 1992; Ayscough et al., 1997; McMillan <br /> Fig. 7. nucleolus segregates late yeast cell division.    et al., 1998; Theesfeld et al., 1999), predicted localization nucleolus determined wild-type interphase mitotic cells.   nucleolar marker Nop1p localized anti-   lead metaphase arrest. did Nop15p cells Nop1p antibody. (B) localization Nop15p-TAP. nucleo-          arrested metaphase, Nop15p-depleted cells plasm visualized DAPI staining. bar represents 10 mm.         arrested G1 retained apparently normal distribution <br />                                                                          polymerized actin, indicating defect actin <br /> particles. Weaker precipitation seen U3                   assembly speci&#194;&#174;c cytokinesis. <br /> snoRNA, component 90S pre-ribosomes,                       defect cytokinesis seen strains lacking Nop15p 35S pre-rRNA precipitated detectably.                  reported previously &gt;100 <br /> conclude Nop15p likely associate 90S                 characterized ribosome synthesis factors. speci&#194;&#174;c- <br /> pre-ribosomes initial cleavage site A0                  ally, mutations proteins present pre- <br /> immediately prior cleavage site A2, remaining                      60S particles Nop15p (Harnpicharnchai et al., 2001; <br /> associated early pre-60S complexes (Fatica                  Fatica Tollervey, 2002) observed Tollervey, 2002) processing sites A3, B1S, B1L                 result defects cytokinesis. pre-rRNA processing C2. Nop15p good consensus RRM showed                       defect Nop15p-depleted cells closely resembles <br /> RNA binding activity vitro, likely                seen depletion Nop7p/Yph1p (Adams et al., <br /> bind directly pre-rRNA. Alterations pre-rRNA               2002; Oef&#194;&#174;nger et al., 2002). Depletion Yhp1p does <br /> structure induced binding Nop15p important                  result cell cycle phenotype, defects recruitment 5&#194;&#162; 3&#194;&#162; exonucleases Rat1p               DNA replication (Du Stillman, 2002) clearly <br /> Xrn1p (Henry et al., 1994) generate 5&#194;&#162; end               distinct effects Nop15p depletion. <br /> 5.8S rRNA.                                                                  Defects nucleolar protein, Net1p/C&#194;&#174;1p,                                                                          result cell cycle arrest large-budded cells,                                                                          failure activate MEN (Shou et al., 1999; Straight <br /> Nop15p cell cycle                                                et al., 1999; Visintin et al., 1999). Nop15p-depleted cells cells undergoing depletion Nop15p, growth rate                 lacked characteristic features MEN defects; mildly reduced ~16&#194;&#177;20 h, time                    stabilized mitotic spindle distinctive bud projec- <br /> growth abruptly ceased. accompanied                        tions arise continued spindle elongation <br /> substantial decline viability cells                  (Jimenez et al., 1998). directly, Nop15p-depleted <br /> returned permissive medium.  abundance                   cells shown release Cdc14p&#194;&#177;GFP ef&#194;&#174;ciently ribosomal subunits strongly reduced                nucleolus following release nocodazole arrest, <br /> time growth ceased. behaviour                       demonstrating MEN activation. <br /> common cells depleted ribosome synthesis factors,                    Components septum formation machinery, usually slow, progressive increase                       including septins Myo1p, localized doubling time, growth rate falls match reduced                 future bud site early cell cycle (Lippincott Li, <br /> numbers ribosomes. lack cytoplasmic ribosomes                 1998). contrast, actin relocated form actin/ unlikely principal cause growth               myosin ring late cell cycle, following <br /> arrest Nop15p-depleted cells.                                         nucleoplasmic separation shortly disassembly <br />    Microscopic examination revealed distinct terminal                mitotic spindle. proteins required phenotypes Nop15p-depleted cells. 63%                   actin ring assembly identi&#194;&#174;ed (Osman et al., <br /> cells rounded unbudded, consistent                      2002; Tolliday et al., 2002), mechanism timing <br /> inability pass `Start&apos; cell cycle checkpoint.                 recruitment remain poorly understood. observed <br /> remaining 37% showed unusual, elongated morphology                    segregation nucleolar markers Nop15p duplicated separated nuclei. constriction                 Nop1p dividing yeast cells lags separation present marking site bud formation, cells                 nucleoplasm. Dividing cells readily detected clearly unable complete cytokinesis. Rapid                   nuclei appeared separated DAPI <br />  <br /> 6580 <br />                                                                                                             Ribosome synthesis cell division <br />  <br />  <br /> staining, fact bridged nucleolar material.                        pre-rRNA rRNA analysis following oligonucleotides                                                                                used: 002, 5&#194;&#162;-GCTCTTTGCTCTTGCC; 003, 5&#194;&#162;-TGTTACCTCTGGGC- <br /> Late segregation nucleolus seen previously <br />                                                                                CC; 006, 5&#194;&#162;-AGATTAGCCGCAGTTGG; 007, 5&#194;&#162;-CTCCGCTTATTGA- &#194;&#174;ssion yeast (Toda et al., 1981; Hirano et al., 1989),                   TATGC; 008, 5&#194;&#162;-CATGGCTTAATCTTTGAGAC; 017, 5&#194;&#162;-GCGTTGT- conserved feature. mitotic cells, Nop15p                      TCATCGATGC; 020; 5&#194;&#162;-TGAGAAGGAAATGACGCT; 033, 5&#194;&#162;- CGC- partially delocalized nucleolus, staining                    TGCTCACCAATGG; 041, 5&#194;&#162;-CTACTCGGTCAGGCTC; 200, 5&#194;&#162;- <br /> seen nucleoplasm. regulatory net-                           UUAUGGGACUUGUU. <br /> works implicated release Cdc14p                                                                                Immuno&#194;&#175;uorescence <br /> nucleolus. FEAR network (Cdc fourteen early release)                       localization Nop15p, Nop1p tubulin, cells &#194;&#174;xed required initial release, MEN                         incubation 4% (v/v) formaldehyde 30 min 25&#194;&#176;C, activated Cdc14p required prevent relocation                      spheroplasted. Immuno&#194;&#175;uorescence performed described <br /> Cdc14p nucleolus (Shou et al., 1999; Straight                      (Grandi et al., 1993; Berge&#195;&#65533;s et al., 1994). stain nuclear DNA, DAPI <br /> et al., 1999; Visintin et al., 1999). roles                       included mounting medium (Vectashield, Vector <br />                                                                                Laboratories). Nop15p-TAP detected anti-protein antibody <br /> networks triggering nucleolar separation                      coupled &#194;&#175;uorescein isothiocyanate (FITC). Nop1p detected release Nop15p remain determined. late                            mouse anti-Nop1p antibody secondary goat anti-mouse antibody <br /> nucleolar separation release Nop15p                        coupled FITC (Sigma). Tubulin detected rabbit anti-tubulin appropriate timing markers initiation actin                      antibody (Sigma) secondary goat anti-rabbit antibody coupled                                                                                FITC (Sigma) 1:1000 1:200 dilution, respectively. F-actin <br /> ring formation.                                                                staining, cells &#194;&#174;xed 1% glutaraldehyde 10 min 25&#194;&#176;C. Cells <br />    basic mechanism cytokinesis factors                      washed twice phosphate-buffered saline (PBS), resuspended involved septum formation closure conserved                         PBS/0.1% Triton X-100 incubated 10 min 25&#194;&#176;C. Rhodamine- yeast animal cells (Field et al., 1999). closest                   conjugated phalloidin (6 U) PBS added incubated 45 min human Nop15p homologue NIFK, nucleolar                          25&#194;&#176;C. Cells washed PBS incubated poly-<span id='am-1' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span>L-lysine</span>- <br />                                                                                coated coverslips mounted slides using moviol, <br /> interphase associates condensed mitotic chromo-                       containing DAPI. Cells containing pYE195-Rpl25-eGFP grown somes (Takagi et al., 2001). Functions Nop15p,                           SGSR-URA mid-exponential phase, transferred SD-URA various <br /> addition role pre-rRNA processing,                     times &#194;&#174;xed 4% (v/v) formaldehyde 30 min pelleted. Cells conserved humans.                                                 resuspended 100 mM KH2Ac/K2HAc/1.1 M sorbitol mounted <br />                                                                                slides using moviol, containing DAPI. synchronization late M- <br />                                                                                phase, cells treated nocodazole (10 mg/ml) 3 h, commencing <br />                                                                                15 h shift glucose-containing medium. Cells released Materials methods                                                          washing pre-warmed medium stained F-actin described <br />                                                                                 Images obtained Smart Capture VP. <br /> Strains <br /> Growth handling S.cerevisiae standard techniques. GAL- <br />                                                                                Immunoprecipitation Nop15p-TAP <br /> regulated strains pre-grown RGS medium, containing 2% <br />                                                                                immunoprecipitation Nop15p-TAP, cells grown YPgal raf&#194;&#174;nose, 2% galactose 2% sucrose, harvested intervals <br />                                                                                OD600 = 2 lysed buffer (150 mM KAc, 20 mM Tris-Ac pH 7.5, <br /> following shift medium containing 2% glucose. Strains pulse&#194;&#177; <br />                                                                                5 mM MgAc) 1 mM dithiothreitol (DTT), 0.5% Triton X-100, <br /> chase analysis pre-grown minimal RGS medium lacking uracil <br />                                                                                2.5 mM vanadyl-ribonucleoside complexes (VRC) 5 mM phenyl- shifted minimal glucose medium lacking uracil. Strains                                                                                methylsulfonyl &#194;&#175;uoride (PMSF) 4&#194;&#176;C using glass beads (Sigma). <br /> immuno&#194;&#175;uorescence studies grown minimal glucose medium <br />                                                                                Immunoprecipitation rabbit IgG&#194;&#177;agarose beads, TEV cleavage lacking leucine. degron-Nop15p fusion strain (generously provided <br />                                                                                RNA extraction performed described (Rigaut et al., 1999). K.Labib, Paterson Institute) grown described (Kanemaki et al., <br /> 2003). <br />    Yeast strains used constructed study listed table 1   Pulse&#194;&#177;chase labelling experiments Supplementary data available EMBO Journal Online.                   Pulse&#194;&#177;chase labelling pre-rRNA performed previously <br /> Conditional mutants control repressible GAL10 promoter        described (Tollervey, 1987b) using 100 mCi [5,6-3H]uracil generated step PCR strategy strains YDL401            (Amersham) 2 min 30&#194;&#176;C. Total RNA extracted buffer <br /> BMA64 (Lafontaine Tollervey, 1996). Transformants selected            AE/phenol&#194;&#177;chloroform ethanol precipitated (Schmitt et al., 1990). <br />                                                                                3H-Labelled pre-rRNA rRNA resolved 1.2% agarose gels  prototrophy screened PCR. TAP-tagged strains constructed step PCR strategy GAL-mediated strain              high molecular weight 8% acrylamide&#194;&#177;8 M urea gels low <br /> GAL::nop15 (Rigaut et al., 1999). Transformants screened               molecular weight RNAs. RNA transferred Hybond-N+ Nylon <br /> immunoblotting PCR. TAP-tagged strains transformed               membranes (Amersham), dried exposed X-ray &#194;&#174;lm 10 days pUN100DsRedNOP1 (kindly provided E.Hurt, Heidelberg) allow               &#194;&#177;80&#194;&#176;C intensifying screen. <br /> ready identi&#194;&#174;cation nucleolus. <br />                                                                                Mobility shift assays <br /> Viability assay                                                                PCR fragment corresponding YNL110c (NOP15) ORF GAL::nop15 wild-type strains pre-grown RGS medium,                 ampli&#194;&#174;ed cloned BamHI EcoRI sites pGEX-2T vector <br /> containing 2% raf&#194;&#174;nose, 2% galactose 2% sucrose,              (Pharmacia) obtain pGEX-NOP15 vector. Nop15p expressed <br /> shifted medium containing 2% glucose. viability assay                 E.coli strain BL21, puri&#194;&#174;ed using <span id='am-31' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutathione</span>&#194;&#177;Sepharose <br /> performed plating ~1, 2.5 5 3 103 cells samples taken 0,          according manufacturer&apos;s protocol released cleavage 8, 16 24 h shift pre-warmed YGSR plates. Colonies           thrombin (Sigma). vitro transcription pre-rRNA (5&#194;&#162;-ITS1 3&#194;&#162;-ITS2) counted 2 days growth percentage viability           performed PCR products ampli&#194;&#174;ed rDNA plasmid determined comparison number cells plated.                      carrying T7 promoter region. binding reaction performed                                                                                30 mM Tris&#194;&#177;HCl pH 7.4, 150 mM KCl, 2 mM MgCl2, 0.1% Triton X- <br /> RNA extraction, northern hybridization primer                              100, 20% glycerol 1 mM DTT, presence tRNA (1 mg/ml), <br /> extension                                                                      10 fmol 32P-labelled pre-rRNA 0&#194;&#177;30 fmol recombinant protein <br /> RNA extracted described previously (Tollervey, 1987a). high         reaction volume 15 ml. RNA heat denatured 65&#194;&#176;C 10 min, <br /> molecular weight RNA analysis, 7 mg total RNA separated            followed slow cooling room temperature, prior binding <br /> 1.2% agarose gel containing formaldehyde transferred northern          reaction. competition experiments, 200-fold molar excess cold <br /> hybridization described previously (Tollervey, 1987b). Standard 6, 8     competitor RNA added. Reactions incubated room <br /> 12% acrylamide&#194;&#177;8 M urea gels used analyse low molecular                temperature 30 min loaded 6% native acrylamide/ <br /> weight RNA species primer extension reactions. Primer extensions           bisacrylamide (80:1)/4% glycerol gel 0.53 TBE buffer. Prior performed described previously (Beltrame Tollervey, 1992)          loading, gel pre-run 1 h run 5 h 250 V 5 mg total RNA using primers 006, 007 033.                           cold room. <br /> </body></html>