<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                       Dyskerin Ablation Mouse Liver Inhibits rRNA <br />                                Processing Cell Division <br />  <br />              Dyskerin component small nucleolar ribonucleoprotein complexes acts pseudouridine <br />           synthase modify newly synthesized ribosomal, spliceosomal, possibly RNAs. encoded           DKC1 gene, gene mutated X-linked dyskeratosis congenita, telomerase complex. <br />           yeast ortholog, Cbf5, essential protein, mammals effect dyskerin ablation cellular <br />           level known. mouse hepatocytes survive induction Dkc1 deletion.           absence dyskerin, rRNA processing inhibited accumulation large precursors, fibrillarin <br />           does accumulate nucleoli. low rate apoptosis induced hepatocytes, induction <br />           p53-dependent cell cycle checkpoint pathway. Signs liver damage including increase serum <br />           alanine aminotransferase activity disordered structure histological macroscopic levels           observed. response carbon tetrachloride administration, wild-type hepatocytes mount rapid <br />           proliferative response, dyskerin divide. conclude hepatocytes survive           dyskerin role dyskerin RNA modification essential cellular proliferation. <br />  <br />  <br />    Modification specific bases newly synthesized ribo-                   &#226;&#8364;&#339;scaRNPs&#226;&#8364;&#65533; characterized presence scaRNA <br /> somal spliceosomal RNAs early posttranscriptional                    conserved Sca motif (43) function modification <br /> event carried specialized nucleolar complexes,               snRNAs (9), small RNAs essential pre-mRNA splic- <br /> snoRNPs. 2 classes snoRNPs, H/ACA RNPs,                           ing. vertebrates 3&#226;&#172;&#732; domain telomerase RNA, TERC, pseudouridylate specific uridines, C/D RNPs,                   H/ACA RNA (36), RNA, provides catalyze methylation specific residues. cases               template synthesis telomere repeats, RNP <br /> complexes consist guide RNA 4 proteins (1, 12, 55).                complex 4 proteins snoRNPs case H/ACA RNPs guide RNA H/ACA                               telomerase reverse transcriptase proteins (13, <br /> snoRNA uses base pairing guide complex                         37, 48, 49). TERC RNA contains Sca motif specific uridines nascent target RNA (14). 4 pro-                   Cajal bodies (25). Finally, H/ACA RNPs known <br /> teins dyskerin (20, 35), NOP10 (21), NHP2 (21, 52),                    function, integral RNA having known target. GAR1 (16); dyskerin pseudouridine synthase (29). RNA                    case orphan RNA expression appears modification ancient process, snoRNP proteins                    restricted brain (8), opening possibility highly conserved. Despite  role RNA modifica-                  H/ACA RNAs effect tissue-specific gene expression. <br /> tions clearly understood. Pseudouridines frequently                    Drosophila dyskerin ortholog, minifly, essential regions rRNAs functionally important,                 viability (15). Minifly mutations retain function lead thought increase stability folded domains.                  small size, developmental delay, reduced fertility depletion individual snoRNAs does                        defects pseudouridylation rRNA processing. affect growth yeast, yeast ortholog dyskerin, Cbf5,               mice complete knockout gene encoding dyskerin, Dkc1, <br /> essential (26). Interestingly, mutations render Cbf5 unable               embryonic lethal (19). humans hypomorphic mutations form pseudouridine permit cell survival lead severe                DKC1 gene X chromosome cause X-linked dysker- <br /> reduction ribosome synthesis (56). <br />                                                                                atosis congenita (DC) (20, 28), inherited bone marrow <br />    H/ACA snoRNPs cellular functions                                                                                failure syndrome characterized presence hypo- pseudouridylation rRNA (34). involved cleav- <br />                                                                                hyperpigmentation, nail dystrophy, mucosal leukoplakia <br /> age rRNA precursors, class H/ACA RNPs local- <br />                                                                                (50). Interestingly pathological effects dys- <br /> ized Cajal body nucleolus (9).                                                                                kerin mutations DC patients problems <br />                                                                                telomere maintenance, forms DC    * Corresponding author. Mailing address: Dept. Internal Medicine,           caused mutations components telomerase (51) <br /> Div. Hematology, Washington University School Medicine, 660 S.              biochemical investigations decrease levels Euclid Ave., Box 8125, St. Louis, MO 63110. Phone: (314) 362-8814.             telomerase RNA, TERC, telomerase activity  <br /> Fax: (314) 362-8826. E-mail: pmason@im.wustl.edu. <br />    &#226;&#8364;&#161; Supplemental material article http://mcb <br />                                                                                tectable change rRNA processing snoRNA metabolism <br /> .asm.org/.                                                                     (53, 54). Mice, mouse embryonic stem (ES) cells, containing <br />    &#228;&#338;&#164; <br />      Published ahead print 16 November 2009.                             dyskerin <span id='am-38' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-39' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-42' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> changes cause DC humans far <br />  <br />                                                                          413 <br />  414       GE ET AL.                                                                                                                               MOL. CELL. BIOL. <br />  <br />  variable effects, having decreased Terc RNA                           described previously (19). RNA isolated using Trizol (Invitrogen, Carls- <br /> telomerase mild defects ribosome biogenesis (18, 38).                        bad, CA). <br />                                                                                        Nuclear protein extraction performed using NE nuclear cyto- far mice shown pathological features                       plasmic extraction reagents (Pierce, Rockford, . micrograms pro- DC, probably fact laboratory mice                      tein resolved SDS-polyacrylamide gel electrophoresis (PAGE) analysis <br /> long telomeres. complete absence telomerase, mice                         transferred Hybond-ECL nitrocellulose membrane (GE Healthcare, striking defects generation pass                      Piscataway, NJ). Antibodies detected ECL plus Western blotting detec- <br />                                                                                     tion reagents (GE Healthcare) visualized autoradiography. Primary  <br /> shortened telomeres offspring generation 3                         tibodies used study polyclonal antiserum 2 dyskerin peptides, <br /> later generations features similar seen                       DCSNPLKREIGDYIR (aa 74 88) GLLDKHGKPTDNTPAC (aa 382 DC (3, 32). Recently described mouse line produced                          396 &#226;&#171;&#185; C) (38); polyclonal antiserum dyskerin <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> DRKPLQE <br /> gene targeting contained copy human mutation                           DDVAEIQHA (aa 18 33); anti-TATA box binding protein (1:2,000; Abcam, <br />                                                                                     Cambridge, MA); anti-p53 (1:1,000; Abcam); anti-p21 (1:200; Abcam);  <br /> consisting 3&#226;&#172;&#732; truncation Dkc1 gene, encoding                                                                                     tifibrillarin (1:1,000; Abcam). secondary antibody horseradish <br /> protein 21 amino acids (aa) missing (17).                       peroxidase-conjugated goat anti-mouse antibody (1:10,000; Abcam) horserad- <br /> mice showed slower growth proliferative tissues charac-                      ish peroxidase-conjugated goat anti-rabbit antibody (1:10,000; Abcam). <br /> teristic DC. slow growth dependent telomerase,                           Pre-rRNA analysis. Pre-rRNA analysis Northern blotting described <br /> independent telomere length, associated accu-                       previously (44) using synthetic oligonucleotide DNA probes. RNA (10 &#226;&#65533;&#174;g)                                                                                     electrophoresed 1% (wt/vol) agarose-<span id='am-45' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-46' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span> gels blotted Hybond- <br /> mulation DNA damage.                                                             N&#226;&#171;&#185; membranes (GE Healthcare) overnight. Oligonucleotides (10 pmol)    mammalian cells, dyskerin essential embry-                      end labeled [&#226;&#65533;&#165;-32P]<span id='am-22' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> T4 polynucleotide kinase (Fisher, Pittsburgh, <br /> onic development, known essential cell                       PA). temperature hybridization washing determined survival cell division. paper used inducible                   melting temperature (Tm) probe. probes used ITS1, 5&#226;&#172;&#732;-CTC <br />                                                                                     TCACCTCACTCCAGACACCTCGCTCCA-3&#226;&#172;&#732;; ITS2, 5&#226;&#172;&#732;-ACCCACCGCAGC <br /> knockout strategy delete Dkc1 gene adult mice                         GGGTGACGCGATTGATCG-3&#226;&#172;&#732;; 28S, 5&#226;&#172;&#732;-CACCTTTTCTGGGGTCTGA tissues reconstituted cells                    T-3&#226;&#172;&#732;. escaped deletion liver survived weeks                         Sucrose gradient centrifugation. Sucrose gradient centrifugation polysome <br /> majority cells containing deleted gene. gave                     analysis performed previously described (24). gram liver tissue, 200 <br />                                                                                     &#226;&#65533;&#174;l hypotonic buffer (1.5 mM KCl, 2.5 mM MgCl2, 5.0 mM Tris-Cl, pH 7.4), opportunity studying mammalian cells did  <br />                                                                                     200 &#226;&#65533;&#174;l hypotonic lysis buffer (1.5 mM KCl, 2.5 mM MgCl2, 5.0 mM <br /> tain dyskerin. cells, despite long-term survival,                       Tris-Cl, pH 7.4; 2% sodium deoxycholate, 2% Triton X-100, 2.5 mM dithio- defective rRNA synthesis, showed signs cell cycle                      threitol [DTT]) gently homogenized using Dounce homogenizer. checkpoint response induction apoptosis,                        lysates centrifuged 8,000 &#226;&#171;&#187; g 10 min 4&#194;&#176;C. supernatant (&#226;&#172;&#402;800 <br />                                                                                     &#226;&#65533;&#174;l) supplemented 4 &#226;&#65533;&#174;l RNase inhibitor (Invitrogen) 80 &#226;&#65533;&#174;l heparin <br /> defective cell division following liver injury. conclude <br />                                                                                     stored &#226;&#171;&#186;80&#194;&#176;C. Linear 10% 45% sucrose gradients (80 mM NaCl, 5 mM dyskerin required cell survival essential                MgCl2, 20 mM Tris-Cl, pH 7.4, 1 mM DTT) formed using Gradient <br /> cell division.                                                                      Master (BioComp). Gradients centrifuged 38,000 rpm 3 h 4&#194;&#176;C                                                                                     analyzed ISCO fractionator (ISCO). <br />                                                                                        Histology immunohistochemistry. Liver sections age-matched <br />                         MATERIALS METHODS                                       knockout mice control littermates analyzed hematoxylin eosin <br />                                                                                       staining count mitotic bodies characteristic cells M phase cell <br />    Generation Dkc1 conditional KO mice. generation Dkc1lox allele <br />                                                                                     cycle. Indirect immunohistochemistry BrdU used identify proliferating C57BL/6 mice previously described (19). Mice carrying Alb-Cre       cells S phase. Terminal deoxynucleotidyltransferase-mediated <span id='am-33' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="dUTP" datatype="xsd:string"></span>dUTP</span> nick-end <br /> transgene (42) mice carrying Mx1-Cre transgene obtained       labeling (TUNEL) used detect apoptotic nuclei. <br /> Jackson Laboratory (Bar Harbor, . Homozygous floxed dyskerin mice              Double staining dyskerin fibrillarin dyskerin BrdU used bred Alb-Cre mice, male mice (9 weeks old unless specified) carry      determine expression localization. Briefly, 4-&#226;&#65533;&#174;m paraffin sec- floxed dyskerin allele Alb-Cre selected study (Dkc1loxTg:AlbCre,     tions passed xylene graded alcohol rinsed phosphate- <br /> referred knockout [KO] mice). experiments analysis, mice     buffered saline (PBS). Slides treated Diva Decloaking solution genotype Dkc1lox Dkc1&#226;&#171;&#185;Tg:AlbCre used controls referred       (BioCare Medical, Walnut Creek, CA) pressure cooker 3 min. Slides wild type (WT)  Similarly, male Dkc1loxTg:Mx1Cre mice         rinsed PBS times. Slides incubated primary rabbit <br /> selected study (KO) Dkc1lox Dkc1&#226;&#171;&#185;Tg:Mx1Cre littermates      anti-dyskerin antibody (1:200) mouse antifibrillarin (1:1,000) anti- <br /> (WT). induction Mx1-Cre transgene, mice (5 weeks old) injected      dyskerin rat anti-BrdU (1:400; Accurate <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> <span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Scientific Corp, West- poly C) (250 &#226;&#65533;&#174;g intraperitoneally  p.]) day total 3   bury, NY) overnight 4&#194;&#176;C. washes, sections incubated injections. Mice sacrificed 4 weeks injection.                  secondary anti-rabbit Cy3-conjugated antibody (1:500; Jackson ImmunoResearch, <br />    experiments mice performed strict guidelines        West Grove, PA) anti-mouse fluorescein isothiocyanate (FITC)-conjugated <br /> National Institutes Health Institutional Animal Use Care Com-        antibody (Jackson ImmunoResearch) anti-rabbit Cy3 anti-rat FITC- <br /> mittee Washington University. Age- sex-matched conditional knockout          conjugated antibody (1:200; Jackson ImmunoResearch) 1 h room temper- <br /> mice control littermates weighed sacrificed (n &#226;&#171;&#189; 3 5). Livers       ature. Sudan black autofluorescence inhibitor (Chemicon International, Te- isolated, wet weights recorded calculate liver weight-     mecula, CA) used quench background fluorescence. Slides rinsed body weight ratio. Livers used DNA RNA isolation, protein      distilled water (dH2O) counterstained DAPI (4&#226;&#172;&#732;,6-diamidino-2- <br /> extraction, histology, immunohistochemistry.                                    phenylindole). Slides viewed Axioskop 40 (Zeiss) upright <br />    CCl4 treatment. week-old male Dkc1loxTg:AlbCre mice controls       research microscope, digital images obtained Nikon Coolpix <br /> subjected carbon tetrachloride (CCl4) (Sigma-Aldrich, St. Louis, MO) treat-      camera. <br /> ment (2 &#226;&#65533;&#174;g/g body weight; p.) induce liver damage. Equal numbers KO         positive cells counted 5 low-power fields (200&#226;&#171;&#187;) 3 sections WT mice sacrificed <span id='am-17' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span> asphyxiation different times CCl4        BrdU, TUNEL staining higher-power field (1,000&#226;&#171;&#187;; 300 cells) <br /> treatment. Mice injected p. 2.5 mg/mouse bromodeoxyuridine (BrdU;       3 different knockout control livers double staining. <br /> Sigma-Aldrich, St. Louis, MO) 1 h sacrifice. regenerating livers       mRNA processing quantitative reverse transcription-PCR (qRT-PCR). <br /> harvested used DNA isolation, total RNA isolation, protein extraction,      levels pre-mRNA mRNA glyceraldehyde-3-phosphate dehydro- paraffin embedding.                                                             genase (GADPH) nucleolin measured RT-PCR. 28S used    DNA, RNA, protein analysis. Liver tissues WT KO mice            control. SYBR green master mix (Applied Biosystems, Foster City, CA) used isolating purifying DNA RNA. DNA isolated          used follow RNA amplification real-time reaction. Reverse transcrip- <br /> standard sodium dodecyl sulfate (SDS)-proteinase K method. Southern blotting        tion PCRs performed according manufacturer&#226;&#8364;&#8482;s recommenda- distinguish Dkc1lox Dkc1&#226;&#338;&#172;DC1 alleles performed        tions SuperScript III strand synthesis RT-PCR (Invitro- <br />  VOL. 30, 2010                                                                                   DYSKERIN ABLATION MOUSE LIVER                415 <br />  <br />  <br />  <br />  <br />    FIG. 1. Deletion Dkc1 gene various tissues Dkc1loxTg:Mx1Cre transgenic mice different times. Groups 4 mice (6 weeks old) injected poly C)  p.; 250 &#226;&#65533;&#174;g) 3 times day. Genomic DNA prepared indicated organs, Cre-mediated <br /> deletion Dkc1 gene detected Southern blotting.   Diagram Dkc1 gene targeted alleles. Dkc1lox allele lox <br /> element intron 11 IRES Neo selection cassette followed second lox element 3&#226;&#172;&#732; UTR. presence Cre recombinase, lox elements recombine, removing exons 12 15, producing deleted allele Dkc1&#226;&#338;&#172;DC1. (B) Southern blotting proportions Dkc1lox Dkc1&#226;&#338;&#172;DC1 various tissues collected 4 weeks poly C) injection. BM, bone marrow. (C) Quantification Dkc1 recombination <br /> efficiency various tissues Dkc1loxTg:Mx1Cre mice time. surviving cells liver 80% recombined Dkc1 locus, thymus bone marrow 0 &#226;&#172;&#402;20% recombined. (D) Cells various tissues female Dkc1lox/&#226;&#171;&#185; mice consist 50% deleted 50% WT alleles poly C) treatment, showing induction deletion efficient. experiment mice Dkc1DC7 differently placed lox sites (19). <span id='am-50' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-51' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span>CER</span>, cerebellum. (E) Western blot showing decrease dyskerin liver 2 mice, 60% reduction 90% reduction dyskerin. TBP, TATA box binding protein. <br />  <br />  <br />  <br /> gen). primers specifically designed Primer Express 3.0 software        Dkc1lox allele lox element intron 11 internal <br /> (Applied Biosystems). nucleolin, forward primer spliced mRNA         ribosome entry site (IRES)-Neo selection cassette followed 5&#226;&#172;&#732;-CATGTGACTAAGTCCTTCATGACTGA-3&#226;&#172;&#732;, forward primer specific unspliced pre-mRNA 5&#226;&#172;&#732;-GGATGGGAAAAGTAAAGGGATTG-3&#226;&#172;&#732;,                            second lox element 3&#226;&#172;&#732; untranslated region (UTR) (Fig. reverse primer 5&#226;&#172;&#732;-GCCTTTGACCTTTCTCTCCAGTAT-3&#226;&#172;&#732;.                       1A). presence Cre recombinase, lox elements <br /> GAPDH, forward primer spliced mRNA 5&#226;&#172;&#732;-TTGTGGAAGGGCT                      recombine, removing exons 12 15, producing deleted <br /> CATGACCACAGTCCA-3&#226;&#172;&#732;, forward primer unspliced pre-mRNA                                                                                      allele Dkc1&#226;&#338;&#172;DC1. experiments mouse embryonic fibro- <br /> 5&#226;&#172;&#732;-CTCACTCATTGCCCCCGTGTTTTCTA-3&#226;&#172;&#732;, reverse primer 5&#226;&#172;&#732;- <br /> TCAGATCCACGACGGACACATTGGGGG-3&#226;&#172;&#732;. U2, forward primer                           blast (MEF) cells  shown), detect truncated 5&#226;&#172;&#732;-TTTGGCTAAGATCAAGTGTAGTATCTGTT-3&#226;&#172;&#732; reverse                              protein corresponding expected product  <br /> primer 5&#226;&#172;&#732;-CCAACTCCTACTTCCAAAAATCCA-3&#226;&#172;&#732;. 28S, forward                      leted allele, Dkc1&#226;&#338;&#172;DC1 allele essentially null <br /> primer 5&#226;&#172;&#732;-GGTAGCCAAATGCCTCGTCAT-3&#226;&#172;&#732; reverse primer 5&#226;&#172;&#732;-CCCTTGGCTGTGGTTTCG-3&#226;&#172;&#732;. Fluorescence measured using Ap- <br />                                                                                      allele. Having Dkc1&#226;&#338;&#172;DC1 allele embryonic <br /> plied Biosystems 7500 Fast Real-Time PCR (Applied Biosystems).                lethal (19), interested adult tissues    Statistical analysis. unpaired tailed Student t test used compare   survive absence dyskerin. bred male <br /> hepatocellular BrdU incorporation, TUNEL staining, mitotic body index, liver <br />                                                                                      Dkc1loxTg:Mx1Cre mice, taking advantage Mx1 interfer- <br /> weight body weight ratio, alanine aminotransferase (ALT) activity, DNA <br /> recombination efficiency significance set 0.05.                              inducible promoter, conveniently induced                                                                                      double-stranded RNA form poly C). Cre <br />                                                                                      induction mice sacrificed 2 days, 1 week, 1 month,                                   RESULTS <br />                                                                                      3 months, DNA extracted different tissues   Dkc1 knockout mouse models. previously described                            examined Southern blotting proportions Dkc1lox <br /> construction Dkc1lox allele gene targeting (19).                       Dkc1&#226;&#338;&#172;DC1 alleles. results (Fig. 1B C)  416     GE ET AL.                                                                                                                MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />   FIG. 2. Loss dyskerin fibrillarin hepatocyte nucleoli Dkc1loxTg:Mx1Cre transgenic mice immunohistochemistry.   Dyskerin <br /> localizes nucleolus colocalizes C/D snoRNP protein fibrillarin. liver tissue paraffin embedded incubated rabbit <br /> antidyskerin mouse antifibrillarin antibody. nucleoli counterstained DAPI. (B) Western blotting showed nuclear fibrillarin protein KO mice changed. TATA box binding protein (TBP) used loading control. <br />  <br />  tissues proportion surviving cells containing               90% reduction dyskerin. deletion 40% (heart kidney) close zero (bone             antibody used experiment (38) raised marrow, thymus, spleen, lung). liver,                mixture 2 peptides, deleted proportion surviving cells higher, greater 80%              Dkc1&#226;&#338;&#172;DC1 deletion. possible truncated protein animals; reaches maximum 2 days, remains                 present livers deletion detected high 4 weeks, slowly falls  results           antibody. second antibody N-terminal certainly arise competitive outgrowth                peptide,  truncated peptide detected  Fig. S1 <br /> cells escape Dkc1 deletion populating                  supplemental material), indicating peptide adult organs tissues, seen experiments                    unstable. <br /> involving deletion essential genes adult mice (46).               loss dyskerin liver mice  <br /> experiments female Dkc1lox/&#226;&#171;&#185; animals injected,                firmed immunohistochemistry. WT hepatocytes dyskerin tissues consisted 50% deleted 50% WT alleles,                   colocalizes fibrillarin dense spots nucleoli, indicating efficiency deletion close 100%            livers knockout mice dyskerin detectable surviving cells presumably ones WT allele             hepatocytes (Fig. 2A). Interestingly, hepa- active X chromosome (Fig. 1D).  conclude                tocytes lacking dyskerin, fibrillarin localized dense <br /> hematopoietic tissues high rates cellular proliferation             spots  Fibrillarin component C/D box snoRNAs survive dyskerin, tissues lower                 participate methylation nascent rRNA molecules <br /> proliferative rate, uninjured adult liver, tolerate         stages rRNA biogenesis H/ACA <br /> high levels Dkc1 deletion.                                               snoRNAs act (7). ask lack detectable fibrillarin    Deletion Dkc1 mouse liver. wanted investigate             destabilization fibrillarin caused way lack induction Dkc1 deletion liver              dyskerin, looked fibrillarin Western blotting (Fig. Dkc1loxTg:Mx1Cre mice poly C) injection. pro-               2B). difference WT knockout livers, <br /> portion cells deletion reached 90% 2 days             suggesting absence dyskerin C/D snoRNPs remained 90% 1 month injection gradually              maintained nucleoli. <br /> decreasing (Fig. 1C). chose use livers 1 month                 confirm result mice tolerate lack dyskerin injection studies. maximum extent deletion             liver use independent interferon induc- variable measured Southern blotting                  tion, decided induce Dkc1 deletion using Western blotting. Figure 1E shows decrease                Cre recombinase liver-specific promoter, e., dyskerin livers 2 mice, 60% reduction               mouse albumin promoter, AlbCre (41). studied appear- <br />  VOL. 30, 2010                                                                               DYSKERIN ABLATION MOUSE LIVER                 417 <br />  <br />  <br />  <br />  <br />    FIG. 3. Deletion Dkc1 gene liver Dkc1loxTg:AlbCre trans- <br /> genic mice different times.   Dkc1 recombination efficiency mice different ages determined Southern blot analysis. Liver <br /> DNA isolated 5- &#226;&#172;&#402;16-week-old mice subjected Southern blot analysis shown Fig. 1; percentage Dkc1&#226;&#338;&#172;DC1 <br /> alleles reached peak &#226;&#172;&#402;70% 6 12 weeks. (B) Western blotting <br /> showed dyskerin protein liver significantly decreased KO mice. TBP TATA box binding protein used loading <br /> control. <br />  <br />  <br />  <br /> ance Dkc1 deletion male Dkc1loxTg:AlbCre mice. <br /> Southern blot analysis showed peak 70% liver cells <br /> containing deletion 6 weeks level main- <br /> tained 12 weeks declining (Fig. 3A). AlbCre mice express Cre hepatocytes possibly bile duct hepatocytes make 70% liver <br /> cells, calculations underestimate deletion. similar decrease dyskerin detected protein level <br /> (Fig. 3B). Immunofluorescence dyskerin fibrillarin <br /> antibodies similar observed Dkc1loxTg:Mx1Cre               FIG. 4. Dyskerin deletion activates p53-dependent cell cycle ar- <br /> mice.                                                                     rest pathway Dkc1loxTg:AlbCre mice.   staining liver section <br />                                                                           showing fibrotic regions appearance small proliferating <br />    Dkc1loxTg:AlbCre mice signs chronic liver damage               cells portal areas KO mice (200&#226;&#171;&#187;). (B) Detection apoptotic impaired rRNA processing. histology livers                 hepatocytes KO mice TUNEL staining (200&#226;&#171;&#187;; P &#226;&#172;&#65533; 0.05). depleted dyskerin Mx1-Cre induction appeared normal                 arrows positive nuclei. (C) Increased hepatocyte proliferation (data shown).  livers Dkc1loxTg:AlbCre              KO mice BrdU staining (200&#226;&#171;&#187;; P &#226;&#172;&#65533; 0.05). (D) Western blot  <br />                                                                           ing elevation p53 p21 protein livers KO mice. <br /> mice 9 weeks age marked abnormalities observed.                &#226;&#65533;&#164;-Actin used loading control. present 6 weeks (data shown), indicating develop weeks Dkc1 deletion.                                                                           formed analysis hepatocellular BrdU incorporation, example, livers exhibited fibrotic regions small pro- <br />                                                                           revealed 1.3% hepatocytes KO mice liferating cells periportal areas (Fig. 4A). addition                                                                           0.5% WT mice incorporated BrdU, reflecting increased <br /> liver weight body weight KO mice significantly <br />                                                                           cellular turnover KO livers (Fig. 4C). <br /> reduced compared WT mice (Table 1). assess <br /> liver damage mice, measured serum ALT activity bilirubin. Comparable normal levels bilirubin                    TABLE 1. Body weight liver weight 9-week-old WT <br /> observed WT KO mice.  KO mice                                   (Dkc1lox) KO (Dkc1loxTg:AlbCre) mice <br /> showed 3.5-fold increase serum ALT, indicating pres- <br />                                                                                   Variable                   WT                        KO <br /> ence liver damage. used TUNEL staining mea- <br /> sure extent apoptosis liver tissue           Liver wt (g)                  1.310 &#226;&#171;&#190; 0.045            1.178 &#226;&#171;&#190; 0.051a <br /> 0.2 0.3% hepatocytes knockout livers stained pos-            Body wt (g)                  28.503 &#226;&#171;&#190; 0.991           24.620 &#226;&#171;&#190; 0.629a <br />                                                                           Liver/body wt ratio               0.046                    0.049 <br /> itive positive-staining hepatocytes WT age-matched <br /> animals virtually undetectable (Fig. 4B). Finally,                                                                                              P &#226;&#172;&#65533; 0.05. <br />  418     GE ET AL.                                                                                                              MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 5. Dyskerin ablation disrupts rRNA processing Dkc1loxTg:AlbCre mice.   Scheme pre-rRNA processing mouse liver (5). probes ITS1 ITS2 used detect precursors 18S 5.8S, respectively. (B) Northern blot RNA livers male mice different genotypes. ITS1 probe decrease 21S accumulation 30S, precursors mature <br /> 18S RNA, observed KO mice. ITS2 probe decrease precursors 41S, 17S, 12S observed. (C) Effect dyskerin <br /> deletion polysomes. Polysomes liver extracts sedimented 10 45% sucrose gradients, showing decrease 80S ribosomes significant change polysome profile. 40S 60S subunits present wild-type KO livers small <br /> amounts. <br />  <br />  <br />  <br />    impaired ribosome biogenesis, possible effect              processing mouse liver. investigate  hepatic rRNA <br /> dyskerin ablation, known promote p53-dependent cell                  Dkc1loxTg:AlbCre mice wild-type littermates cycle arrest apoptosis (33, 40), quantified                different ages analyzed Northern blotting. results <br /> expression p53 p53 target p21 KO WT livers.              dyskerin deficiency results changes accu- <br /> Figure 4D shows increased levels proteins KO liver,             mulation rRNA precursors including decreased 21S, 41S, <br /> indicating p53 cell cycle arrest pathway activated          17S, 12S RNA. products ITS1 cleavage, hepatocytes lacking dyskerin.                                              30S species increases 32S species altered (Fig. <br />     observations lack dyskerin leads               5A B; Fig. S2 supplemental material). <br /> mislocalization fibrillarin (Fig. 2A) induces p53              Finally analyzed polysome profile WT KO livers, <br /> pendent cell cycle arrest (Fig. 4D) suggested ribosome                showed decrease relative 80S ribo- <br /> biogenesis impaired hepatocytes lacking dys-                 somes KO livers (Fig. 5C). <br /> kerin. effect dyskerin ablation ribosome                   Dkc1loxTg:AlbCre mice exhibit impaired hepatocellular <br /> biogenesis mammalian cells investigated, Cbf5-             proliferative response acute liver damage. data  <br /> deficient yeasts exhibit complete block rRNA synthesis (6).           scribed  showing hepatocytes survive months <br /> Furthermore, yeast Cbf5 mutants fail pseudouridylate               Dkc1 deletion, suggest dyskerin essential RNA display growth defect impaired ribosome produc-                  cell survival (Fig. 1B D).  modestly increased <br /> tion abnormal accumulation rRNA pro-                    liver damage seen mice raises question cessing intermediates (56). Based observations,                slow loss Dkc1-deleted liver cells time result interested effects dyskerin ablation rRNA                relative proliferative advantage wild-type hepatocytes <br />  VOL. 30, 2010                                                                         DYSKERIN ABLATION MOUSE LIVER                     419 <br />  <br />  <br />  <br />  <br />   FIG. 6. Hepatocyte proliferation CCl4 treatment Dkc1loxTg:AlbCre mice.   Similar perivenous damage livers WT KO mice CCl4 treatment staining (200&#226;&#171;&#187;). (B) Impaired hepatocyte proliferation liver KO mice 72 h CCl4 treatment BrdU staining (200&#226;&#171;&#187;). (C) Quantification BrdU-positive hepatocytes CCl4 treatment different time points. time zero, BrdU <br /> incorporation seen 1.3% hepatocytes KO mice 0.5% hepatocytes WT mice. peak response WT liver seen 72 h 24.7% hepatocytes positive, 6.5% hepatocytes positive KO livers (P &#226;&#172;&#65533; 0.05 time zero, 48 h, 72 h). <br /> (D) Decreased mitotic index KO hepatocytes poor proliferative response. <br />  <br />  <br />  Dkc1-null cells hepatic regenerative response             DNA KO livers 48 h CCl4 injury. answer question, examined               treatment shows proportion Dkc1lox allele ex- <br /> gree injury hepatic regenerative response CCl4               pressing WT dyskerin goes 30% 70% (Fig. <br /> Dkc1loxTg:AlbCre Dkc1lox mice. KO WT mice                    7A), consistent possibility. Immunohisto- <br /> showed comparable perivenous damage (Fig. 6A), indicating                 chemical analysis showed 48 h CCl4 treatment dyskerin deletion did affect susceptibility liver         BrdU-positive hepatocytes Dkc1loxTg:AlbCre mice pos- <br /> cells CCl4 damage. contrast, analysis hepatocellular             itive dyskerin expression (Fig. 7B C). data pro- <br /> BrdU incorporation showed significantly (P &#226;&#172;&#65533; 0.05) increased              vide convincing evidence dyskerin essential hepato- <br /> hepatocellular proliferation 48 72 h CCl4 injection          cellular proliferation. WT compared KO mice (Fig. 6B C). reduced                     mRNA processing absence dyskerin. ob- <br /> proliferative index KO cells demonstrated           served dyskerin ablated large proportion significantly diminished mitotic index 72 h CCl4 treat-             mouse hepatocytes overt effect health ment animals (Fig. 6D). Interestingly, proportion            animal. involved ribosome biogenesis cells BrdU positive 48 h mitosis 72 h       telomere maintenance, dyskerin involved indirectly WT KO livers, implying                mRNA splicing, snRNAs modified pseudouridy- <br /> block G2-M transition KO mouse hepatocytes. Figure              lation guided H/ACA scaRNAs. Inhibition mRNA splic- <br /> 6B illustrates presence population nonparen-             ing expected cause severe phenotype chymal periportal liver cells labeled BrdU.           observed. reason wanted examine mRNA splic- <br /> cells abundant KO WT livers.               ing WT KO livers (31). chose 2 genes ex- identified  suspect                 pressed liver, GAPDH nucleolin, measured infiltrating immunological cells. data hepato-            steady-state levels unspliced spliced transcripts cellular proliferation impaired Dkc1loxTg:AlbCre mice.              WT KO animals, reasoning ratio    Dyskerin necessary cell division. observation          reflect efficiency splicing. calculated ratio <br /> hepatocellular proliferation reduced Dkc1loxTg:AlbCre               spliced unspliced transcripts did versus control mice hepatocytes Dkc1loxTg:               significant change gene, implying dyskerin <br /> AlbCre mice escape Dkc1 deletion raises possibility          ablation did affect splicing efficiency (Fig. 8A). small number proliferating hepatocytes KO livers            result expected spliceosomal RNAs, including escaped deletion. Southern blotting liver              U2, contain high proportion modified nucleotides  420     GE ET AL.                                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                               FIG. 8. Dkc1 deletion does affect mRNA processing.   Lev- <br />                                                                            els nucleolin GAPDH pre-mRNA mRNA KO WT <br />                                                                            liver. Dkc1 deletion effect RNA levels pre-mRNA/ <br />                                                                            mRNA ratio. 28S RNA used internal control. (B) Measure- <br />                                                                            ment U2 RNA levels KO WT liver qRT-PCR. <br />  <br />  <br />  <br />                                                                            data indicate gross defect mRNA <br />                                                                            processing absence dyskerin. <br />  <br />  <br />                                                                                                      DISCUSSION <br />                                                                               knowledge dyskerin snoRNPs                                                                            role modification rRNA snRNA comes                                                                            genetic studies yeast structural studies archaebacte- <br />                                                                            ria. Modification using guide RNAs highly conserved pro- <br />                                                                            cess, proteins associated H/ACA RNA C/D <br />                                                                            RNAs highly conserved (22). humans mutations                                                                            DKC1 cause X-linked dyskeratosis congenita (20),                                                                            good evidence major effect pathogenic dys- <br />                                                                            kerin mutations decrease telomerase activity, causing  <br />                                                                            creased telomere shortening exhaustion stem cells                                                                            highly proliferating tissues (50). possible,                                                                             defects snoRNP function contribute X-linked DC <br />                                                                            (38, 45). mice Dkc1 knockout embryonic lethal (19), <br />                                                                            mutations cause variable effects telomerase <br />                                                                            activity rRNA biogenesis (17, 18, 38, 45).                                                                            used inducible deletion strategy knock Dkc1 adult <br />                                                                            mice. tissues survive  <br />    FIG. 7. Dyskerin necessary cell division.   Southern blot liver DNA Dkc1loxTg:AlbCre mice 48 h         dyskerin populated cells escaped <br /> CCl4 treatment. proportion WT Dkc1lox allele KO          deletion, liver survive large proportion liver goes 30% 70% (P &#226;&#172;&#65533; 0.001). (B) Immuno-           Dkc1 genes deleted months. fact rate fluorescent detection presence BrdU dyskerin          turnover hepatocytes livers knockout animals hepatocytes Dkc1loxTg:AlbCre mice 48 h treatment CCl4. <br /> Paraffin-embedded liver sections double stained antidys-         higher wild-type animals. likely kerin anti-BrdU antibodies, nucleoli counterstained       induction apoptosis small percentage cells DAPI. Hepatocellular BrdU incorporation correlated           replacement division hepatocytes  dyskerin dyskerin-null mice. (C) Quantification dyskerin-    presume, escaped deletion. Deletion solid tissues using <br /> BrdU-positive hepatocytes. hepatocytes counted, <br />                                                                            Mx1-Cre complete, achieved incorporated BrdU dyskerin positive. <br />                                                                            maximum possible efficiency deletion 50%                                                                            Dkc1lox/&#226;&#171;&#185; females, assume efficiency deletion                                                                            mice high. case U2 shown vitro pseudouri-                  Ribosome synthesis inhibited absence dyskerin. <br /> dines required efficient splicing (10). exam-              primary objectives determine extent <br /> ined steady-state level U2 RNA WT KO livers                  lack dyskerin affects rRNA processing ribosome bio- qRT-PCR significant difference (Fig. 8B).                  genesis. experiments suggest absence dys- <br />  VOL. 30, 2010                                                                      DYSKERIN ABLATION MOUSE LIVER                              421 <br />  <br />  <br /> kerin ribosome biogenesis proceed. Firstly, total liver   damage induced CCl4.  regeneration accom- <br /> RNA probed rRNA precursors Northern blotting          panied increase relative wild-type DNA marked reduction mature processed rRNA precur-            compared DNA containing Dkc1 deletion, indicating <br /> sors accumulation 30S precursor.              regeneration involved division cells <br /> Dkc1loxTg:AlbCre mice dyskerin gene deleted hepato-       did contain deleted gene. Compelling evidence cytes, comprise 70% liver cells, seen     absence dyskerin incompatible cell division immunofluorescence experiments, hepatocytes es-         observation Alb-Cre mice treated CCl4 <br /> cape deletion.  decrease        allowed recover resulting liver damage RNA precursor likely means absent cells      cells knockout livers replicating DNA deletion. assumption supported differences    assessed BrdU incorporation. Remarkably cells rRNA processing pattern Alb-Cre Mx1-Cre exper-          did incorporate BrdU minority cells iments. Dkc1loxTg:Mx1Cre mice poly C) treat-       livers stained positive dyskerin antibody. <br /> ment, dyskerin deletion extends nonhepatocytes,       Presumably cells escaped deletion. experiments, extreme change pat-         block cell cycle caused nucleolar <br /> tern rRNA processing (data shown). precise nature        disruption activating p53 checkpoint response (39, 41) sequence processing events place        unknown mechanism couples cell division maturation primary 47S transcript mature 18S,         ribosome biogenesis (2, 41) remains determined. <br /> 5.8S, 28S rRNAs mouse completely established            Conclusions. Using conditional deletion mouse Dkc1 <br /> (27, 47).  alternative early events cleavage     gene hepatocytes, produced model 5&#226;&#172;&#732; external transcribed spacer (ETS) 41S precur-      study mammalian cells dyskerin. model <br /> sor cleavage ITS1 30S 32S RNA.       prove useful delineating role dyskerin little 41S RNA detected KO livers, suggesting dys-       H/ACA snoRNAs ribosome biogenesis, telomere mainte- <br /> kerin required early cleavage event 5&#226;&#172;&#732; ETS        nance, mRNA expression. Furthermore, influence (27). 30S 32S precursors normal mouse cells          ribosome biogenesis cell cycle progression amena- processed, 30S processed 21S intermediate       ble experimentation using model. experiments pre- mature 18S RNA 32S processed 17S 12S           sented lack dyskerin blocks rRNA processing <br /> intermediates mature 5.8S RNA. probes used           cleavage primary transcript ITS1 leads (Fig. 5) hybridize 30S, 32S, 17S, 12S, 21S species   disruption ribosome biogenesis. absence dyskerin, early products ITS1 cleavage accumu-         hepatocytes able divide. <br /> late, 12S, 17S, 21S intermediates reduced KO compared WT livers. data suggest                                     absence dyskerin initial ITS1 cleavage event <br />                                                                       <br /> takes place 30S 32S precursors processed         indicating processing requires pseudouri-      <br /> dylation nascent transcripts H/ACA RNA-            <br /> dependent cleavage event. Pseudouridylation rRNA              <br /> thought place early transcripts, prior cleavage    ITS1 (4, 11). results compatible recent studies     <br /> showing cleavage events rRNA processing, <br /> including 5&#226;&#172;&#732; ETS, require participation                                          <br /> snoRNPs, cleavage site ITS1 requires, yeast,     action catalytic RNA RNase MRP (23, 30).              <br />    compelling evidence ribosome biogenesis              <br /> halted cells dyskerin observation        <br /> cells immunofluorescence fibrillarin antibody does        brightly staining nucleolus seen wild-type cells.         <br /> Fibrillarin active methylase present C/D snoRNPs,          ribosome biogenesis active molecules        dyskerin. molecules pseudouridy-         <br /> lated absence dyskerin, results suggest       <br /> lack pseudouridylation affects action C/D snoRNAs.       mechanism underlying apparent linkage                   2 processes subject experiments&#226;&#8364;&#8221;         interesting know lack methylation inhibits       <br /> pseudouridylation.  factors maintain snoRNPs       nucleolus amenable analysis          <br />    Dyskerin essential cell division. second major             <br /> finding experiments absence cell division         cells dyskerin. dyskerin-depleted livers         <br /> studied showed reduced proliferative response liver       <br /> </body></html>