<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                                                                                                                                               BioMed Central <br />  <br />  <br />  <br /> Research                                                                                                                                                                                                                                                       Open Access deducing causality metabolic networks <br /> Chiara Bodei*1, Andrea Bracciali1 Davide Chiarugi2 <br />  <br /> Address: 1Dipartimento di Informatica, Universit&#195;&#160; di Pisa, 56127, Pisa, Italy 2Dipartimento di Scienze Matematiche e Informatiche, <br /> Universit&#195;&#160; di Siena, 53100, Siena, Italy <br /> Email: Chiara Bodei* - chiara@di.unipi  Andrea Bracciali - braccia@di.unipi  Davide Chiarugi - chiarugi3@unisi * Corresponding author <br />  <br />  <br />  <br />  Seventh International Workshop Network Tools Applications Biology (NETTAB 2007) <br /> Pisa, Italy. 12-15 June 2007 <br />  <br /> Published: 25 April 2008 <br /> BMC Bioinformatics 2008, 9(Suppl 4):S8                                                                                                                  doi:10.1186/1471-2105-9-S4-S8 <br />  <br /> &lt;supplement&gt; &lt;title&gt; &lt;p Semantic Web Bioinformatics: Goals, Tools, Systems, Applications&lt;/p&gt; &lt;/title&gt; &lt;editor&gt;Paolo Romano, Michael Schroeder, Nicola Cannata Roberto Marangoni&lt;/editor&gt; &lt;note&gt;Research&lt;/note&gt; &lt;/supplement&gt; <br />  <br />  <br />  <br />  article available  http://www.biomedcentral.com/1471-2105/9/S4/S8 <br /> &#194;&#169; 2008 Bodei et al.; licensee BioMed Central  open access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                                                                 Abstract <br />                                                                 Background: Metabolic networks present complex interconnected structure,                                                                 understanding general non-trivial task. formal approaches developed                                                                 support investigation networks. relevant problems context                                                                 comprehension causality dependencies molecules involved metabolic process. <br />                                                                 Results: apply techniques formal methods computational logic develop                                                                 abstract qualitative model metabolic networks order determine possible causal <br />                                                                 dependencies. Keeping mind expressiveness ease use, aimed providing:                                                                  minimal notation represent causality biochemical interactions, ii) automated tool <br />                                                                 allowing human experts easily vary conditions silico experiments. exploit reading                                                                 <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> reactions terms logical implications: starting description metabolic <br />                                                                 network terms reaction rules initial conditions, chains reactions, causally depending                                                                  automatically deduced. components initial state                                                                 clauses ruling reactions easily varied new trial experiment started, according                                                                 investigation strategy. approach aims exploiting computational logic formal <br />                                                                 modeling framework, available, naturally close human reasoning.                                                                 directly leads executable implementations support, perspective, various reasoning <br />                                                                 schemata.  abstractions supported computational counterpart, based                                                                 Prolog implementation, allows representation language closely correspondent                                                                 adopted chemical abstract notation. proposed approach validated results <br />                                                                 regarding gene knock essentiality model metabolic network Escherichia coli <br />                                                                 K12, relevant coherence available wet-lab experimental data. <br />                                                                 Conclusions: Starting presented work, goal provide effective analysis toolkit, <br />                                                                 supported efficient fledged computational counterpart, aim fruitfully driving                                                                 vitro experiments effectively pruning non promising directions. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                Page 1 10 <br />                                                                                                                                                                                                                                          (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                   http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br />  <br /> Background                                                     source molecule reaction Understanding relationships elements           forth. words, abstract away quantities, <br /> involved biologic interaction networks,         stoichiometric proportions, kinetical thermodynamics <br /> functioning cellular metabolism, relevant problem      parameters involved production P. Systems Biology. words [1], &#226;&#8364;&#339;diagrams          Noticeably, abstract actual dynamics interconnections represent sort static roadmaps,      reactions. Intuitively speaking, project reactions really seek know traffic patterns,      &#226;&#8364;&#339;flat&#226;&#8364;? temporal scale availability M N patterns emerge, control ?.            spoiled production P, metab- <br /> Having formal description interconnections      olites caused M N. abstrac- <br /> methodology perform software simulation              tion quantities, &#226;&#8364;&#339;infinite copies&#226;&#8364;? M N patterns  help orientating vitro        result available. abstraction, experimentation. regard, causality play      reaction leading P actually represents possibility important role finding chains reactions connect     P generated, caused, vivo, presence M parts  e.g. determining      N.  model gives approximation causal correlations molecules appar-        set actual pathways, possibly including ently correlated. general, proposed models com-      pathways actually prevented, instance, <br /> plex systems code lot information determining         lack suitable quantity reactants possible correlations causal dependencies       inadequate temperature. payoff abstraction <br /> straightforward computationally expensive.                  adopted terms expressiveness language                                                                effectiveness computational counterpart. Abstracting focus metabolic networks, e. set       quantitative issues prevent reasoning cellular biochemical pathways involved energy man-          dynamical features (bio)chemical objects. <br /> agement synthesis structural components.          makes possible consideration <br /> Biochemical pathways typically composed chains       aspects systems (typically large biochemical <br /> enzymatically catalyzed chemical reactions inter-      networks) kinetical thermodynamical param- <br /> connected complex way. Starting composi-         eters scarcely known. <br /> tion local, understood behaviour, study overall emerging behaviour metabolic networks               Chains causal reactions  step step, automati- <br /> appears difficult.                                             cally deduced. aim, exploit analogy                                                                logical implications chemical reactions, interpret- apply techniques formal methods               ing reaction M N produces P logical <br /> computational logic order develop abstract qual-      clause stating M N imply P. method sup- <br /> itative model metabolic networks, focussing causal-      ported computational logic counterpart, based ity. Keeping mind expressiveness ease use,      Prolog implementation semantics. aimed providing:  minimal notation represent      allows compute set metabolites causality biochemical interactions, ii) auto-        produced consequences given set rules mated tool allowing human expert easily vary        starting set initially available metabolites. ditions silico experiment. introduce simple     step-wise determined repeatedly adding metabo- skeletal notation, inspired biochemical reactions,      lites immediately caused application emphasize causality aspects concerned       rules set far produced metabolites. choice relying computational logic, &#226;&#8364;&#339;pro-      relate computational construction orig- <br /> vides straightforward intuitive representation        inal model prove convergence properties proc- <br /> human reasoning&#226;&#8364;? [2], appeared particularly suitable       ess. multidisciplinary context work. hand, palatable biologists, close   Despite abstract working hypotheses adopted, biochemical intuition. hand,      framework revealed sufficient provide meaningful <br /> direct computational counterpart, scien-        approximations possible chains reactions tists build order devise needed analysis    experiments.  approach facilitates thinking <br /> tool.                                                          revising biological model itself, making easy                                                                vary components initial state notation, biochemical reactions given            clauses rule reactions. <br /> abstract representation: record rela- <br /> tions source target elements       proposed approach serve sort  ? <br /> single reaction, e.g. molecules M N        analysis, repeatedly exploring different scenarios, molecule P produce. turn, P           derived different set working hypotheses. <br />  <br />  <br />                                                                                                                    Page 2 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                    http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  tool allows rapidly evaluate impact            formalisms developed concurrency <br /> changes hypotheses particular observable        theory applicable Systems Biology, recall come.  obtain interactive effective analy-       relevant purposes. Petri Nets [4] offer sis, used differentiate promising       way graphically represent structure distributed <br /> causal relations deduced, deserve tested         systems. successfully applied mod- <br /> vitro, instead pruned.                   elling metabolic pathways simple genetic networks <br />                                                                  e.g. [5,6] cite . model pathways validated approach studying robust-          networks terms dynamic evolutions. <br /> ness metabolic network E. coli K12. Selected <br /> genes knocked disabling rules              Process calculi interactions communications <br /> regarding corresponding encoded enzymes. Results            independent agents processes. underlying coherent actual biological behaviour,              idea biological seen concurrent <br /> observed vitro reported [3] &#226;&#8364;&#339;Geno          systems. particular, &#207;&#8364;&#226;&#8364;&#8220;calculus [7] Ambient Calcu- <br /> Base&#226;&#8364;? (http://ecoli.aist-nara.ac.jp/), database entirely      lus [8] transferred theoretical dedicated E. coli K-12.                                      science setting biology setting, leading suitably <br />                                                                 extended biological versions  Bio- paper organized follows.  discuss       chemical stochastic &#207;&#8364;-calculus [9,10] BioAmbients <br /> related work present formal framework,             [11]. version CCS, RCCS [12] intro- <br /> illustrating biochemical processes represented       duced address biological issues. calculi inside  Results experiments metabolic         instead specifically defined biological modelling, <br /> network E. coli K-12 metabolic genotype follow.          &#206;&#186;-calculus [13], Brane calculi [14] Beta Bind- <br /> Conclusions followed section Methods,         ers [15]. Chemistry invoked explicitly report techniques used.                               process algebraic context years com- <br />                                                                 ing systems biology. [16], abstract machine based <br /> Related work comparison                                     chemical metaphor introduced: states chem- work included recent research trend           ical solutions floating molecules interact exploits established theories techniques         according reaction rules. Rules specify produce formal methods order support interpreta-         new molecules old ones. <br /> tion big raw biological data available. using logic, slightly diverge line    Closer approach work presented [17], research biological modelling inspired        authors apply causal semantics &#207;&#8364;-calcu- use concurrent models. Concurrency theory offers       lus order biochemical processes. proc- <br /> wide choice models naturally expressing causality,      ess computations obtained quite accurately essential notions.  causality   capture reflect behaviour biological systems <br /> natural notion logic framework. Concurrent          causality key role enhancing precision models focussed description dynamic           simulations. starting point quite similar, behaviour systems. Considering aspects           model description biological systems given computationally demanding. led        terms molecular entities reaction rules implic- <br /> choice abstract model, &#226;&#8364;&#8220; men-          itly code causal relationships, possible <br /> tioned &#226;&#8364;&#8220; notion time abstracted away.        pathways. causal semantics [17] based abstraction similar performed posteriori       enhanced form transition systems [18], makes concurrent model, instance resorting static      possible capture truly concurrent aspects like causality <br /> analysis techniques, offer static approximations       interleaving setting, like process algebraic  dynamic behaviour. Usually, static techniques extract       proposals, introduced information systematically inspecting specification      aim, instance distributed transition systems dynamic behaviour systems. approach            [19]. <br /> instead, want infer information directly inspect- <br /> ing set reactions, modulo abstractions dis-       interesting observe results com- <br /> cussed  skipping specification            parable obtained using quite efficient <br /> chemical dynamics. Resorting static techniques repre-        static technique like Control Flow Analysis (CFA) [20,21], <br /> sents typical way drastically reduce computational     applied &#207;&#8364;-calculus. settings, reaction <br /> cost study possible dynamic evolu-        M N produces P abstractly mod- <br /> tions  price pay amounts loss        elled synchronization process M N precision, usually provide approximations       shared channel c, process P continuation. behaviour.                                               Notice case expressiveness &#207;&#8364;-cal- <br />  <br />  <br />                                                                                                                     Page 3 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                   http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> culus  particular, passing         induction aim learning general rules needed. simplest form, CFA considers effective   abduced hypotheses. representation language communications occur given              ad-hoc devised. predictive accuracy increases <br /> shared channels, disregarding actual viability,      number training examples. methodology instance synchronization dynamics. modelling         richer representation language aims reactions way, CFA consider       addressing different class problems different <br /> possible consumption reactants. regard,      experimental setting. Differently, [2], [27], approach offer analogous result,          based quite general class languages, extended skeletal abstract setting. Having approxima-       deal biological context. language, tion exact behaviour  case   known action event calculi, suited static analysis framework, means     non-monotone evolution dynamical environment, <br /> events prediction does include hap-    specifically biological networks. Abduction pen, included, events happen, e.     used expressive means compute/deduce explana- possible.                                             tions missing information dynamical nature <br />                                                                world. expressive recent proposal shares similarities     approach, proposals deal explicit approach based biochemical abstract      treatment dynamics systems, differently machine BIOCHAM [22], offers formal mod-          proposal strives simplicity order address, <br /> elling environment biochemical processes, oriented         approximately, causality large metabolic net- qualitative aspects. based rule-centered    works. <br /> language specifying biochemical systems. Differently approach, BIOCHAM semantics takes            Finally, important refer wide usage graphs <br /> sideration dynamics systems provides tools      representation language, traditionally close biologist <br /> querying temporal properties systems using         experience. huge set papers adopting <br /> Computation Tree Logic. Refraining dealing           graphs model bio-networks, work closer quantities, BIOCHAM offers explicit controls reactant       use arcs nodes directly represent reactions. consumption reactions  default, pos-     instance, approach quite corresponding logical <br /> sibilities considered. suggests develop-      view reactions &#226;&#8364;&#8220; far abstraction level  <br /> ments approach,  currently, case     cerned &#226;&#8364;&#8220; [28].  starting chosen qualita- consumption admitted. reason choice      tive interpretation biochemistry analogous  effectiveness: way, branching semantics avoided.    authors focus topology metabolic networks, <br />                                                                aiming defining representative measure network close approach BIOCHAM Path-       activity, Synthetic Accessibility  number chemi- <br /> way logic  e.g. [23,24]), rewriting logic used    cal reactions needed transform set initial metabo- modelling biological processes. Rewrite rules     lites set output ones). notion local changes molecular patterns cause       tested predicting viability mutant strains accu- <br /> Rules concurrently applied corresponds      racy results comparable  differ- actual possibility biological compartments inde-     ent perspective. <br /> pendently evolve. offers basis silico experi- <br /> ments advanced forms symbolic analysis.          Results discussion <br /> choose alternative approach, resorting      constructive formalisation metabolic causality <br /> current semantics, accordance aims.    clarify interpretation metabolic causal rela- <br />                                                                tions introducing notation used represent bio- computational logic tradition, largely       chemical reactions. consider assumptions influences proposal, interesting cite        idealised notatation relies, discussing  <br /> recent proposals based Abductive Logic Programming.         according aims, assumptions consid- <br />                                                                ered viable working hypotheses. notation repre- <br /> Complexity bio-networks, understood lack com-         sents adopted formalisation biochemical <br /> plete knowledge, addressed means            reactions.  chosen hypotheses, capability making assumptions provided abduction.        formal account fact metabolite, approach applied gene networks [25],       called reactant, caused network. section motivations similar  proposal based       Methods, present computational counterpart combination Abduction Induction: abduction         formalisation relate computational <br /> allows inference observable effects  [26])       construction adopted model. used generate hypotheses,  <br />  <br />                                                                                                                    Page 4 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                    http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  biochemical reactions causal relations                   let imagine overall process Let consider biochemical reaction written clas-     described includes  steps  upper sical notation:                                                 glycolytic pathway.  set R contain reaction <br />                                                                 rules like: <br />                   mM + nN &#226;&#8224;&#8217; pP + qQ                      (1) <br />                                                                                       glc6p &#226;&#8224;&#8217; fru6p                            (4) <br /> M, N, P Q species involved m, n, p q corresponding stoichiometric coefficients. Expression <br />                                                                                fru6p <span id='am-22' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span> &#226;&#8224;&#8217; fru16p <span id='am-5' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span>                          (5) <br /> (1) indicates  reaction occurs, certain M N certain P Q <br /> according stoichiometric proportions. stoi-                         fru16p &#226;&#8224;&#8217; gap dhap                           (6) <br /> chiometry, order exhaustively characterize chemi- <br /> cal reaction, account number                                 dhap &#226;&#8224;&#8217; gap                              (7) <br /> factors related thermodynamics kinetics repre- <br /> sent propensity reaction occur rate reactants eventually products.                                             gap &#226;&#8224;&#8217; dhap                             (8) <br />  interested investigating causality relation-                    gap nad &#226;&#8224;&#8217; bpg13 <span id='am-11' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>nadh</span>                           (9) <br /> ships  omit description fac- <br /> tors cited   abstract                 acronyms glc6p, fru6p, fru16p, gap, dhap, nad, <br /> representation (1) follows:                               nadh, bpg13 stand Glucose 6-phosphate, Fructose 6- <br />                                                                 phosphate, Fructose1,6-bisphosphate, Glyceraldehyde 3- <br />                       M N&#226;&#8224;&#8217;P Q                            (2)    phosphate, Dihydroxyacetone phosphate, NAD+, NADH <br />                                                                 1,3 Bisphosphoglicerate, respectively. expression reaction rule. simply states presence M N represents possibil-      rules (4), (5), (6), (7) (8)  (9) <br /> ity P Q produced caused. applicable, e.   reaction catalyzed enzymes phos- known premises M N producible         phoglucose isomerase, 6-phosphofructo 1-chinase, fruc- network, P Q producible. Informally         tose bisphosphate aldolase, triose phosphate isomerase, <br /> speaking, standard dynamical reading (2)          glyceraldehyde 3-phosphate dehydrogenase, respectively. <br />                                                                 Note reactant appearing right                   M N&#226;&#8224;&#8217;M N P Q                            (3)    operator &#226;&#8224;&#8217; appear left                                                                 rule.  considering evolution network, <br /> reactants consumed step.                         eventually decompose rules simpler monadic <br />                                                                 rules, e.g. (2) written  description causal relations metabolic net- <br /> work defining set metabolites initially                             M N&#226;&#8224;&#8217;P                                 (10) <br /> present set reaction rules R new metabolites produced. Initial metabolites                                M N &#226;&#8224;&#8217;Q                                (11) <br /> represented form rules premises like &#226;&#8224;&#8217; <br /> P.                                                              transformation causality preserving sense                                                                 following proposition. <br /> Example 1 <br /> Let consider experiment pathway occurring         Proposition 1 environment providing &#206;&#177;-D Glucose, Glycerol           Let M1 &#226;?? &#226;&#8364;&#166; &#226;?? Mm &#226;&#8224;&#8217; P1 &#226;?? &#226;&#8364;&#166; &#226;?? Pn applicable rule, <br /> oxygen. using suitable acronyms like glc, <span id='am-65' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-66' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>gly</span> <span id='am-77' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="rdfs:label" content="O2" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>o2</span>,        e. M1, &#226;&#8364;&#166;, Mn producible, P1, &#226;&#8364;&#166;, Pn pro- <br /> respectively, initial conditions experiment      ducible.  simplified rules M1 &#226;?? &#226;&#8364;&#166; &#226;?? Mm &#226;&#8224;&#8217; include:                                                 Pi, = 1, &#226;&#8364;&#166;, n, applicable order,                                                                 application makes P1, &#226;&#8364;&#166;, Pn producible,  <br />                            &#226;&#8224;&#8217; glc <br />                                                                 proposition holds causality behaves                            &#226;&#8224;&#8217; gly                                monotone way, application subsequent <br />                                                                 rules spoil fact metabolite produci- <br />                            &#226;&#8224;&#8217; o2                                 ble. <br />  <br />  <br />                                                                                                                    Page 5 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                                                              http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  fact reactant caused network pre-                                              Note experimental evidence possible produc- <br /> cise means following <span id='am-16' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> relating                                                    tion metabolite vivo silico, suggest <br /> metabolite chain reactions                                                    need revision adopted biological model. <br /> exist, starting initial state according set rules R.                                                                                               set reactants &#226;&#8364;&#339;motivated&#226;&#8364;?, sense <br />                                                                                                           illustrated, explanation starting ini- <br /> Definition 1                                                                                              tial pool given reactants specific set rules Given set rules R, initial state  reactant let                                          automatically determined. different consider following construction:                                                                   equivalent approaches used, fol- <br />                                                                                                           low logic-deductive interpretation, line              &#226;&#381;&#167;                                     &#226;&#8224;&#8217; &#226;&#710;&#710;                                           explanatory approach adopted. details, sec- <br />              &#226;&#381;&#170; <br /> ER   ) = &#226;&#381;&#168; &#226;&#381;&#163;&#226;&#381;&#161; ER  ( a1 ) ,&#226;&#8364;&#166; , ER  ( )&#226;&#381;&#164;&#226;&#381;&#166; a1 &#226;&#8364;&#166; &#226;&#8224;&#8217; &#226;&#710;&#710; R ER  ( ai ) &#226;&#8240;&#160;&#226;&#352;&#165;  &#226;&#710;&#710;[1..n ]   tion Methods. <br />              &#226;&#381;&#170; &#226;&#352;&#165;                                                  &#226;&#381;&#169; <br />   ER  &#226;&#8240;&#160; &#226;&#352;&#165; explanation R                                               Experiments <br />                                                                                                           applied approach biological model construction non-deterministic  choice                                               based E. coli K-12 metabolic genotype proposed <br /> rule choice cases),                                               [29] [30]. group genes represents subset different ways cause metabolite.                                                        genome E. coli K-12 includes genes <br />                                                                                                           encoding enzymes involved energetic biosynthetic <br /> False positives false negatives: problem approximation                                           metabolism. Using formalism, represented briefly discuss precision approach.                                                 metabolic network composed enzymes explained  approach intended offer                                                  encoded selected gene set metabolites approximation dynamic behaviour meta-                                                      involved catalyzed reactions. obtained bolic pathways. existence explanation gives                                                  list 600 causal rules. performed indications possibility production                                                silico  ? experiments compared obtained <br /> metabolite, according relaxing assumptions                                                    results correspondent vitro counterpart, <br /> adopted. means false positives, e.                                             excerpts reported following. exper- possible predict silico production metabo-                                                iments carried tool  [31]), briefly <br /> lites produced vivo. presence                                                   presented section Methods. <br /> expected consequence abstraction  <br /> approximation.  experimental results                                                          Mutually essential genes <br /> reported final section accuracy rates,                                            silico experimental session performed gene e. measure approximation, comparable                                                    knock mimicking homologous vitro experiment <br /> obtained known approaches litera-                                                    presented [3].  authors silenced target <br /> ture.                                                                                                     genes E. coli K-12 (sucAB sucCD) encode                                                                                                           enzymes involved Krebs cycle (&#206;&#177;-ketoglutarate contrary, adequacy adopted bio-                                                   dehydrogenase succinyl-<span id='am-33' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> synthase, respectively). <br /> logical model, expect false negatives, e. metab-                                              genes &#226;&#8364;&#339;mutually essential&#226;&#8364;? pro- <br /> olites produced vivo                                                           duction <span id='am-38' about='obo:IMR_0200037' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="rdfs:label" content="Succinyl-CoA" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00091" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200037" datatype="xsd:string"></span>succinyl-CoA</span>, e. sucAB sucCD explanation silico, stated following claim.                                                  knocked individually, simultaneously                                                                                                           order achieve Succinyl-CoA production. Succinyl-CoA <br /> Claim                                                                                                     critically important metabolite involved explanation production certain                                                biochemical pathways leading, e.g., energy production <br /> metabolite  producible vivo.                                                        <span id='am-43' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-49' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> biosynthesis  Diaminopimelate). <br />  intuition supporting claim based follow-                                                order simulate gene knock  ing reasoning: ad absurdum metabolite actually                                                removed rules corresponding reactions cata- <br /> producible vivo, certain set biochemi-                                              lysed &#206;&#177;-ketoglutarate dehydrogenase succinyl-CoA <br /> cal reactions applied starting initial set                                            synthase. set starting experimental  metabolites. Starting reactions                                                      ditions, including initial state metabolites <br /> applying abstractions illustrated                                                       cell assumed uptake external envi- lead explanation looking                                                      ronment. Checking presence succinyl-CoA                                                                                                           end computation, metabolite <br />                                                                                                           produced  e. correspondent fact                                                                                                           deduced) target genes  e. rules <br />  <br />  <br />                                                                                                                                                               Page 6 10 <br />                                                                                                                                         (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                       http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> corresponding action encoded enzymes)                [28]. difference taken account calcu- simultaneously turned  reflects actually        lating accuracy defined  <br /> happens vitro. <br />                                                                    Definition 4 silico gene knock                                           Let TP, TN, FP, FN number true positives, true performed silico gene knock-outs              negatives false positives, false negatives, respectively. <br /> compared results information contained             define accuracy &#226;&#8364;&#339;Geno Base&#226;&#8364;? (http://ecoli.aist-nara.ac.jp/), database <br /> entirely dedicated E. coli K-12. database genes                   = (TP + TN ) / (TP + TN + FP + FN ) classified according various criteria, essentiality, e. capability causing cell death,   . turned  silico knock experiments tried test gene essentiality verifying       experiments following results: silico knock mutants exhibit features typically  <br /> taining living cells. assumed characteris-                                         = <br />                                                                                           &#226;&#381;&#167;    TP        102 <br /> tics reasonably include production ATP                                      &#226;&#381;&#170; <br /> (essential cellular energetic metabolism), produc-                                &#226;&#381;&#170;    TN   =    13 <br />                                                                                           &#226;&#381;&#168; <br /> tion reduced coenzymes <span id='am-17' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span> NADH                                          &#226;&#381;&#170;    FP   =    17 <br /> production dispensable structural components,                                      &#226;&#381;&#170;&#226;&#381;&#169;   FN   =    0 cell wall (<span id='am-54' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-55' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-63' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span>murein</span> Biosynthesis).  <br /> formed silico knock experiments sample           reported [28], accuracy Synthetic Accessi- 132 genes set, time removing rules cor-         bility approach presented ranges 60% responding enzyme encoded silenced gene              74%, approach metabolic networks, e. checking presence observed elements          Flux Balance Analysis [30], accuracy ranges end computation. interpret results         62% 86%. 17 false positives, <br /> prediction viability viability knock-         resulting accuracy 87% specific mutant analysis. experimental setting             experiment. differences probably strongly influ- <br />  Table 1), slightly extend notion false posi-        enced way data sets interpreted tives negatives introduced sec-       cases included data sets.  obtained <br /> tion say True Positive occurs knock-          results make confident approach reasonably mutant results viable silico vivo        accurate. <br /> counterpart, True Negative occurs silico <br /> knock mutant viable silico vivo,       intuitive idea experiments, just report <br /> False Positive occurs viable silico knock            Table 2, examples true positive, true negative <br /> mutant vivo counterpart  False        false positive cases (reminding false <br /> Negative occurs silico knock mutants         negatives). symbols + &#226;&#710;&#8217; stand presence viable, vivo counterpart results viable. Note         absence, respectively, observed elements assignment arbitrarily chosen. eval-         silico results. <br /> uate performance method compare similar approaches use performance measure taken              presence false positives  silico, mutant pre- [28], measures number true predictions &#226;&#8364;&#8220;            dicted viable, actually, vivo,  expected positive (viability) negative (non viability) &#226;&#8364;&#8220;        framework, consequence abstraction overall ones. Note definitions false posi-         approximation used model. corre- <br /> tives false negatives differ proposed            sponds fact influence                                                                    viability potentially producible silico, Table 1: True false positives negatives according                                                                    actually produced real life. Finally, note  definition, depending viability knock mutants     experiments carried  false negatives. <br /> silico vivo                                                 makes confident correctness adopted <br />                                                                    biological model. <br />  CASE                      SILICO           VIVO <br />  <br />  True Positive             viable              viable <br />                                                                    Conclusions <br />  True Negative             viable          viable          metabolic networks, metabolites produced  False Positive            viable              viable          set initial metabolites, set chemical reac- <br />  False Negative            viable          viable              tions. reactions produce intermediate metabolites <br />                                                                    products reactants. intro- <br />  <br />  <br />                                                                                                                           Page 7 10 <br />                                                                                                     (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                           http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> Table 2: Gene knock experimental results                       &#226;&#8364;&#339;emphasis hypothesis-driven research biology&#226;&#8364;? <br />                                                                    advocated [1]. <br />  Gene    ATP    NADPH      NADH      cellwall   outcome <br />  <br />  acpS    &#226;&#710;&#8217;      &#226;&#710;&#8217;          &#226;&#710;&#8217;         &#226;&#710;&#8217;          true negative      Methods <br />  lpxA    +      +          +         &#226;&#710;&#8217;          true negative      illustrate methodology, computa- <br />  glk     +      +          +         +          true positive      tional framework represents counterpart  aceA    +      +          +         +          true positive      formalisation metabolic causality, introduced sec- <br />  prsA    +      +          +         +          false positive     tion Results Discussion. Finally, relate compu- <br />  pfk     +      +          +         +          false positive     tational construction adopted formalisation. <br />  <br />                                                                    set reactants caused &#226;&#8364;&#8220; according duced simple skeletal notation net-        formalisation &#226;&#8364;&#8220; explanation starting initial <br /> works terms molecular entities reaction rules            set reactants set rules automatically specify interactions implicitly code pos-      determined. different equivalent <br /> sible causal dependencies reactions. nota-            approaches choose, eventually follow tion permits  hand, chemical              logic-deductive interpretation, line reactions abstract intuitive representation           explanatory approach adopted. Technically, follows <br /> quantitative aspects abstracted away;             consists fragment Horn-based Logic Programming <br /> hand, representation straightforwardly trans-          (having finite set ground predicates) equipped <br /> lated input tool developed, paving          semantics   e.g. [32]).  way silico experiments tool develop-            sake accessibility, theory recast terms ment. aim, exploited analogy           rules  e., clauses) reactants  e., predicates). reaction rules logical implications, allows         following, reactants directly represented, causal <br /> automatically deduce chains causally related reactions          rules model, &#226;?? c &#226;&#8224;&#8217; b say, straightforwardly means logical-based tool.            translated Horn-rules, like  c &#226;&#8224;&#8217; b, according consider dynamic evolution metabolic networks,              following definition. model sufficient information metabolites possibly produced                   Definition 2  hints possible flows reactions.       Let finite set reactants a1,&#226;&#8364;&#166;,  &#226;&#710;&#710;    methodology makes possible reason                                        a1 ,&#226;&#8364;&#166; , &#226;&#8224;&#8217; model itself, allowing vary ini- <br /> tial conditions rules. easy program          n &#226;&#8240;&#165; 0 rule. <br /> modifications evaluating impact changes hypotheses quite immediate, tool quickly            Notice rules premises. reacts queries (typically answer reasona-        rules used representing elements present ble large network returns istantanelously).             initial state  approach satisfies need simulate inves- <br /> tigate behaviour fully known metabolic net-             set consequences given set rules (respec- <br /> works different working hypotheses. particular         tively, semantics logic program) defined <br /> allows perform perturbative experiments,               according step-wise process. applica- <br /> results trivial predict. fact, studied net-   tion rule set reactants causes new reactant <br /> work complex  results unfeasible priori          premises rule verified set. <br /> estimate effects produced local perturbation           Starting set initially available reactants, overall network. Finally, applied method-          set reactants caused obtained <br /> ology metabolic network E. coli K-12 meta-           repeatedly adding reactants immedi- <br /> bolic genotype. silico experiments presented reflect        ately caused application rules. vitro ones. results obtained method underperform analogous ones. Noticeably              Definition 3 ground formalism provides efficient            Let R set rules set reactants, straightforward implementations.                                   immediate consequence operator TR  : 2A &#226;&#8224;&#8217; 2A defined <br />                                                                    ultimate goal investigations sup- <br /> porting heuristic process searching causal explana- <br />                                                                       TR ( )     | a1 ,&#226;&#8364;&#166;, &#226;&#8224;&#8217; &#226;&#710;&#710; R a1 ,&#226;&#8364;&#166;, &#226;&#710;&#710;  &#226;&#710;&#170; tions metabolic phenomena, mind  <br />  <br />                                                                                                                          Page 8 10 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                       http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br /> .                                                                 recall SLD-trees semantics                                                                   Logic Programming). Considering example  TRn ( )    TR (TRn&#226;&#710;&#8217;1 ( ) TRo ( ) .           reactant c caused TR3 ({b}) applica- <br />                                                                   tion rule (12), turn depends rule (13), convergence outlined process guaranteed          production does belong initial set = following result, based observation           {&#226;&#8224;&#8217; b}. Consequently, explanation  trivially, &#226;&#338;&#169;2 , &#226;&#352;&#8224;&#226;&#381;&#167; (finite) complete partial         c b],b] c.  important considered <br /> order(c.p.o.) T(_) continuous (keeping         convergence reactants explanations. <br /> original guarantees monotonicity).           easy observe fourth step set reactants finite &#226;&#8364;&#8220; e. number reactants    explanation b b]]] cycle occurring rules R initial state &#226;&#8364;&#8220;      b. amounts say &#226;&#8364;&#8220; presence cycles &#226;&#8364;&#8220; construction guaranteed converge finite        explanations growth indefinitely, e. convergence <br /> number steps  fix point indicated TRn         occurs infinity.  restricted                                                                   compute reactants produced, instead TR&#226;&#710;&#382;  ).                                           finite process.  following correspondence <br />                                                                   computational results potentially infi- <br /> Proposition 2 <br />                                                                   nite model explanations drawn. <br /> Let R set rules, set reactants, m number reactants model consider-          Theorem 1 <br /> ation.  n &lt;m exists                                                                   Let R set rules, reactant, set reactants                                                                   n     minimum natural number                       TR (TRn ( )) = TRn ( ). <br />                                                                   T (TRn  )) = TRn  ) . following holds <br /> Example 2 <br /> Let consider network reactants = { &#226;&#8224;&#8217; b} ini-                            &#226;&#710;&#710; TRn ( ) &#226;&#8225;&#8221; &#226;&#710;&#402;EI , R  ) &#226;&#8240;&#160;&#226;&#352;&#165; <br /> tially available consisting following simple <br /> rules R:                                                          Proof <br />                                                                   (Outline). <br />                                b &#226;&#8224;&#8217; c                     (12) <br />                                                                   (&#226;&#8225;&#8217;) induction number steps needed firstly <br />                                                                   cause process  rule moti- <br />                                b                         (13) <br />                                                                   vated inclusion used                                                                   rule definition explanation  reac- <br />                                b                         (14)   tants premises rule caused                                                                   steps,  inductive hypothesis, explanations <br />                                c&#226;&#8224;&#8217;d                         (15)   exist, used construct                                                                   explanation . <br /> Applying TR ({b})                                                                   (&#226;&#8225;?) induction structural complexity EI,R .                    TR ({b}) =         b}                         inductive hypothesis, reactants premises                   TR2 ({b}) =  {c ,  b}                         rule explanation belong TRn  ) , <br />                   TR ({b}) = {d , c , b,  <br />                     3 <br />  <br />                                                                   TRn  ) TRn  ) fix point,                   TR4 ({b}) = TR3 ({b}) <br />                                                                   rule clearly applicable cause . <br /> Coherence computational metabolic causality standard, like relate computational           address problem causality metabolic net- <br />                                                                   works, seen  developed software tool based <br /> construction provided original model, e.                                                                   implementation standard seman- <br /> adopted formalisation biochemical reactions, intro- <br />                                                                   tics, running SICTUS Prolog Interpreter  duced section Results Discussion. Let start          [31]). <br /> observe keeping track rule used applying <br /> TR(_) allow reconstruct explanations   <br />  <br />                                                                                                                          Page 9 10 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2008, 9(Suppl 4):S8                                                         http://www.biomedcentral.com/1471-2105/9/S4/S8 <br />  <br />  <br />  <br />  <br /> Competing interests                                                              19.   Degano P, Montanari U: Concurrent Histories: Basis                                                                                        Observing Distributed Systems. Journal authors declare competing interests.                             Sciences 1987, 34(2/3):422-461. <br />                                                                                  20.   Bodei C, Degano P, Nielson F, Riis-Nielson H: Static analysis Authors&apos; contributions                                                                 &#207;&#8364;-calculus application security. Information <br />                                                                                        Computation 2001, 165:68-92. authors equally contributed present work                          21.   Nielson F, Riis-Nielson H, Schuch-Da-Rosa D, Priami C: Static anal- wrote paper. precisely, CB AB taken care                              ysis systems biology. Proceedings workshop Systeomatics computational framework DC carried                                  &#226;&#8364;&#8220; dynamic biological systems informatics Science Press, Trinity <br />                                                                                        College Dublin; 2004:1-6. silico experiments, jointly develop                        22.   Chabrier N, Fages F, Soliman S: Biochemical abstract <br /> malisation metabolic causality analysis                              machine BIOCHAM. Proceedings Computational Methods                                                                                        Systems Biology (CMSB&apos;04), Lecture Notes Science 3082 experimental results.                                                              Paris, France: Springer; 2005:172-191. <br />                                                                                  23.   Eker S, Knapp M, Lincoln P, Laderoute K, Talcott C: Pathway Logic: <br /> Acknowledgements                                                                       Executable Models Biological Network. Proceedings                                                                                        Fourth International Workshop Rewriting Logic Applications article published BMC Bioinformatics Volume 9 Sup-            (WRLA &apos;02), Electronic Notes Theoretical Science 71 Elsevier; <br /> plement 4, 2008: Semantic Web Bioinformatics: Goals, Tools, Sys-                 2002:144-161. <br /> tems, Applications. contents supplement available online     24.   Talcott C, Eker S, Knapp M, Lincoln P, Laderoute K: Pathway Logic http://www.biomedcentral.com/1471-2105/9?issue=S4.                                  Modeling Protein Functional Domains Signal Transduc- <br />                                                                                        tion. Proceedings Pacific Symposium Biocomputing 2004, <br />                                                                                        9:568-580. <br /> References                                                                       25.   Tamaddoni-Nezhad  Chaleil R, Kakas  Muggleton S: Application <br /> 1.    Kitano H: Systems Biology: brief overview. Science 2002,                       abductive ILP learning metabolic network inhibition <br />       295(5560):1662-1664.                                                             temporal data. Machine Learning 2006, 64(1-3):209-230. <br /> 2.    Tran N, Baral C, Nagaraj VJ, Joshi L: Knowledge-based frame-               26.   Papatheodorou  Kakas AC, Sergot M: Inference Gene Rela- <br />       work hypothesis formation biochemical networks. Bio-                      tions Microarray Data Abduction. Proceedings       informatics 2005, 21(Suppl 2):ii213-ii219.                                       Logic Programming Nonmonotonic Reasoning (LPNMR&apos;05), Lecture <br /> 3.    Yu B, Sung B, Lee J, Son S, Kim M, Kim S: sucAB sucCD                    Notes Science 3662 Diamante, Italy; 2005:389-393. <br />       mutually essential genes Escherichia coli. FEMS Microbiology            27.   Grell S, Schaub T, Selbig J: Modelling Biological Networks       Letters 2006, 254(2):245-250.                                                    Action Languages Answer Set Programming. Proceed- <br /> 4.    Reisig W: Petri Nets: introduction Springer-Verlag; 1985.                     ings International Conference Logic Programming (ICLP&apos;06) Lecture <br /> 5.    Goss P, Peccaud J: Quantitative modeling stochastic systems                   Notes Science 4079 Seattle, WA, USA: Springer; <br />       molecular biology using stochastic Petri Nets. Proceed-                    2006:285-299. <br />       ings National Academy Sciences USA 1998, 12:6750-6754.           28.   Wunderlich  Mirny L: Using topology metabolic net- <br /> 6.    Peleg M, Yeh  Altman R: Modeling biological processes using                    works predict viability mutant strains. Biophysical Journal <br />       Workflow Petri Nets models.                     Bioinformatics 2002,         2006, 91:2304-2311. <br />       18:825-837.                                                                29.   Kayser  Weber J, Henkt V, Rinas U: Metabolic flux analysis 7.    Milner R: Communicating mobile systems: &#207;&#8364;-calculus Cambridge             Escherichia coli glucose-limited continuous culture. Trans- <br />       University Press; 1999.                                                          actions Computational Biology VII 2005, 151:693-706. <br /> 8.    Cardelli L, Gordon  Mobile Ambients. Theoretical Sci-           30.   Edwards J, Palsson B: Metabolic flux balance analysis sil- <br />       ence 2000, 240:177-213.                                                          ico analisis Escherichia Coli K-12 gene deletion. BMC Bioin- <br /> 9.    Priami C, Regev  Shapiro E, Silvermann W: Application sto-                formatics 2000, 1(1):1-10. <br />       chastic passing calculus representation simula-                31.   Toolkit Supporting Formal Reasoning Causality       tion molecular processes. Theoretical Science 2004,                  Metabolic Networks.             . [http://www.di.unipi ~braccia/Tool <br />       325:141-167.                                                                     Code]. <br /> 10.   Regev  Silvermann W, Shapiro E: Representation Simula-               32.   Lloyd J: Foundations Logic Programming Second edition edition. <br />       tion Biochemical Processes Using pi-Calculus Process                      Springer; 1987. <br />       Algebra. Proceedings Pacific Symposium Biocomputing World <br />       Scient <br /> </body></html>