<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br /> Salty dog, SLC5 symporter, modulates Drosophila response salt stress <br />  <br />    Stergiopoulos K, Cabrero P, Davies S, Dow JAT. Salty dog,                yeast (24), high osmolarity glycerol stress pathway <br /> SLC5 symporter, modulates Drosophila response salt stress.                  (HOG) control Hog1 kinase (24). Trehalose Physiol Genomics 37: 1&#226;&#8364;&#8220;11, 2009. published November 18,                  key osmolyte bacteria, yeast plants (17); 2008; doi:10.1152/physiolgenomics.90360.2008. regulate                C. elegans, osmotic stress increases expression synthetic <br /> internal environments, organisms adapt varying <span id='am-27' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-28' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> levels     enzyme trehalose 6-phosphate synthase (tps1 tps2), diet. Adult Drosophila exposed dietary salt stress, physiological, survival, gene expression responses monitored. Insects <br />                                                                                corresponding increase trehalose levels (26). <br /> continued feed NaCl-elevated diet, levels &#226;&#172;&#381;4% wt/vol               insects, mechanisms salt tolerance  <br /> ultimately proved fatal. Affymetrix microarray analysis flies fed        stood. aquatic larvae mosquito Aedes aegypti, <br /> diet containing elevated NaCl showed phased response: earliest           interplay salinity, feeding, hemolymph <br /> response widespread upregulation immune genes, followed              levels diuretic amine <span id='am-70' about='obo:IMR_0100123' typeof='owl:Thing obo:IMR_0001693 obo:IMR_0100121 obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001 obo:IMR_0002121'><span id='am-71' property="rdfs:label" content="Serotonin" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="5-Hydroxytryptamine" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="5-Hydroxytryptamine" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="KEGG:C00780" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="serotonin" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="3-(2-Aminoethyl)-1H-indol-5-ol" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0100123" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="3-(2-Aminoethyl)-1H-indol-5-ol" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasAlternativeId" content="IMR:0200412" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="enteramine" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Serotonin" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="enteramine" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>5-hydroxytryptamine</span> (12). upregulation <span id='am-15' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span>carbohydrate</span> metabolism immune response             larval mosquitoes live fresh brackish water, downregulated, finally switch <span id='am-84' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-85' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-90' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> catabolism              specialized uptake ions scarce resource. size <br /> inhibition genes associated reproductive axis. Significantly,      ion-scavenging anal papillae varies inversely online transcriptomic resource FlyAtlas reports           ionic content water, suggesting key <br /> modulated genes predominantly expressed hindgut Malpighian           osmoregulatory organs (13, 31, 42). contrast, species <br /> (renal) tubule, implicating excretory tissues major responders    insect survive saline lakes particular salt stress. genes selected study: SLC5          specializations handling ions (like magnesium) symporter CG2196, GLUT transporter CG6484, transcription <br />                                                                                present massive excess (7). insect body plan factor sugarbabe (previously implicated starvation stress  <br /> sponses). Expression profiles predicted microarray validated        adaptable wide range environmental salt levels. <br /> quantitative PCR (qPCR); expression mapped alimentary                  Drosophila does normally encounter extremes canal situ hybridization. CG2196::eYFP overexpression constructs         salinity, advantages genetic model gene expres- localized basolateral membrane Malpighian (renal)           sion studied manipulated (15, 16) make tubules, RNAi CG2196 improved survival high-salt diet,          attractive species study. Previously, mutation inebriated driven specifically just principal cells Malpighian        putative neurotransmitter/osmolyte transporter (ine) shown tubule, confirming tissue transporter major determi-     confer reduced tolerance dietary salt (21). Overexpression nants survival salt stress. Accordingly, CG2196 renamed            isoforms ine (ine-P1 ine-P2) rescued salty dog (salt).                                                              osmotic stress response defect. osmotic stress-sensitive <br /> Malpighian tubule; microarray; stress response                                 phenotype ine mutants suggested result                                                                                inability accumulate osmolytes Malpighian tubule <br />                                                                                hindgut. Osmotic stress induced dehydra- <br /> SALT STRESS DISRUPT INTERNAL         homeostasis,                  tion rehydration. Drosophila populations selected organisms robustly defend internal milieu                    enhanced desiccation resistance (2, 19), flies excrete fluctuations external salt limited range. Work,                      sodium desiccation retain &#226;&#172;&#402;85% body <br /> particularly yeast (1, 5, 6, 29, 34, 35) plants (3, 20,              sodium content, 83% potassium, 60% chloride (19), <br /> 23, 35, 36, 41), suggests variety mechanisms, particu-                    implying defense desiccation purely loss <br /> larly synthesis regulation osmolytes,                        ionic osmolytes. <br /> glycerol; increase K&#226;&#171;&#185; uptake Na&#226;&#171;&#185; efflux                         aim study investigate response adult <br /> plasma membrane; Na&#226;&#171;&#185; accumulation vacuole                           Drosophila acute salt stress, using Affymetrix microarrays <br /> (involving proton pumping ATPases, Na&#226;&#171;&#185;-efflux pumps,                           subsequent molecular, transgenic, physiological vali- <br /> plasma membrane proteins operate sensors turgor                     dation genes implicated response. <br /> loss signaling cascades regulate responses). <br />    worm, Caenorhabditis elegans, simple multicellular                 MATERIALS METHODS <br /> organism compatible FACS-like high-throughput screen-                        Drosophila. Flies kept standard medium tubes 25&#194;&#176;C, <br /> ing. C. elegans, massive salt loading caused upregulation                12:12 h photoperiod, 55% relative humidity. Wild-type (Oregon <br /> glycerol 3-phosphate dehydrogenase (25), allowing                       R) flies used obtaining genomic DNA, cDNA, accumulation glycerol protective osmolyte, exactly               performing fluid secretion assays. Microinjections produce trans- <br />  <br />  <br />    Address reprint requests correspondence: J.  T. Dow,           costs publication article defrayed payment <br /> Integrative <span id='am-33' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Systems Biology, Faculty Biomedical Life Sciences,        page charges. article marked &#226;&#8364;&#339;advertisement&#226;&#8364;&#65533; <br /> Univ. Glasgow, Glasgow G12 8QQ, UK (e-mail: j t.dow@bio.gla.ac.uk).       accordance 18 U.S.C. Section 1734 solely indicate fact. <br />  <br />                                         1094-8341/09 $8.00 Copyright &#194;&#169; 2009 American Physiological Society                                               1 <br />  2                                                               SALT STRESS DROSOPHILA <br />  <br /> genic lines performed w1118 background. Transgenes                       Microarrays. Seven-day-old flies transferred food <br /> control yeast upstream activating sequences (UAS) driven                      4% NaCl added normal food. flies left 4, 8, 16, <br /> ubiquitously crossing Act5C-GAL4/CyO line (line 4414                        32 h feed, 100 flies treatment Bloomington Drosophila stock center) Malpighian tubule                    anaesthetized ice homogenized 300 &#226;&#65533;&#174;l TRIzol. principal cells c42-GAL4 line (33).                                          procedure repeated times produce    Generation transgenic RNA interference flies. Inserts &#226;&#172;&#402;500                   biological replicates sample. samples stored <br /> bp, directed nonconserved regions genes                   directly &#226;&#171;&#186;80&#194;&#176;C processed according standard protocols. <br /> cloned pWIZ vector (27) germ-line transformed                       Samples 20 &#226;&#65533;&#174;g total RNA reverse-transcribed, vitro <br /> w1118 embryos, according standard procedures. produced                        transcribed, according Affymetrix standard protocols. quality <br /> transgenic flies, hairpin ds-RNA expressed cells                 complementary RNA (cRNA) checked Agilent choice control appropriate GAL4 driver lines (8).                         RNA Bioanalyzer, sample broad cRNA peak <br />    Dietary salt substitution. preparation &#226;&#8364;&#339;salt&#226;&#8364;&#65533; food,                 exceeded height low-molecular-weight degradation peak required salt (NaCl KCl) dissolved 100 ml normal                     taken satisfactory. Samples run Affymetrix <br /> food just preparation, mixed, diet left set.               Drosophila genome array standard conditions: experiment <br /> appropriate, indigo carmine dye  concentration 1.66 g/l)                  comprised 24 arrays. Quality control performed described added immediately diet preparation,                     previous paper lab (39). <br /> food set, provide visible marker ingestion. Diet freshly                     Bioinformatics. routine analysis Affymetrix propri- <br /> prepared avoid changes concentration salts                   etary software (MAS 5.0), array data analyzed using FunAlyse, evaporation. dissection, anaesthetized flies chilling                    pipeline based Bioconductor package, using robust briefly ice dissecting tubules Schneider&#226;&#8364;&#8482;s                      multichip average low-level normalization subsequent calcu- <br /> medium (Invitrogen).                                                                  lation rank products random permutations. method <br />  <br />  <br />  <br />  <br /> Fig. 1. Continued ingestion survival rates NaCl-supplemented diet.  example Indigo carmine gut 3 h feeding 4% NaCl food (left) <br /> compared 6 h gut scored clear (right). B: ingestion high-salt food time. Flies 0, 4, 5% NaCl food, contained blue <br /> dye Indigo Carmine, dissected 1, 3, 6 h scored blue gut contents. C: survival rates days flies fed 0, 3, 4, 5, 6, 7.5, 9% NaCl <br /> compared starvation. Increasing NaCl significantly reduces survival [Kaplan-Meier analysis log-rank test trend, taking P &#226;&#171;&#189; 0.05 critical level <br /> (2-tailed)]. *Significantly different starvation control. <br />  <br />                                                      Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                           SALT STRESS DROSOPHILA                                                                        3 <br /> provides reliable estimates fold change, significance, false             RESULTS <br /> discovery rate (FDR), sharply improved performance exper- <br /> iments limited numbers arrays (10). Significant changes               flies eat salty food? studying effects dietary <br /> assessed pairs samples (salt vs. normal diet) time point,        salt loading, important establish diet remains gene ontology terms overrepresented gene lists identified             palatable;  effects salt loading ob- <br /> iterative group analysis (iGA) (9).                                             scured starvation desiccation. Accordingly, flies <br />    Quantitative PCR. Real-time quantitative reverse transcription               exposed diet labeled blue dye containing <br /> PCR (qPCR) used validation microarray results.           various loadings NaCl, various times scored blue <br /> Seven-day-old adult flies collected placed 1) normal food,          color guts. results experiment seen <br /> 2) 4% NaCl food, 3) vials food. 8 h, flies        Fig. 1B. 1 h food containing extra NaCl, <br /> collected anaesthetized ice, tissue              approximately half flies blue dye gut (Fig. <br /> dissected. Total RNA extracted (Qiagen RNeasy Kit) cDNA                 1A). Flies food containing 4% 5% NaCl appeared <br /> prepared according standard protocols (Invitrogen, Superscript II).          blue dye gut, means eaten <br /> Experiments controlled genomic DNA contamination                   food. 3 h, flies blue dye guts. reverse-transcriptase sample normalized              6 h, flies food NaCl supplementa- <br /> rp49. samples replicated independently times.             tion, half 4 5% NaCl food Data expressed fold difference stress-treated samples <br />                                                                                 blue gut (Fig. 1A), confirming  food intake compared controls &#226;&#171;&#190; SE. <br />                                                                                 reduced high-salt diet, unpalatability major issue. <br />    Cell type-specific RNA interference knockdown gene expression. downregulation CG2196 expression, CG2196-RNA interfer- <br />                                                                                    Survival high-NaCl diet. Having established flies <br /> ence (RNAi) flies crossed c42-GAL4 (tubule principal cell           continue ingest diet containing high levels NaCl, specific) actin-GAL4/CyO (ubiquitous expression) driver lines,           appropriate examine survival. results shown offspring crosses used validate extent              Fig. 1C. graph evident flies fed 3 4% <br /> knockdown qPCR. repeats used different              NaCl food survive better starved. <br /> fly lines, experiments ware repeated twice                   results comparable reported previously (21), <br /> biological repeats) total. Normalization results              work reports greater range salt concentrations <br /> formed rp49 controls.                                                   monitors survival time course, just 4 <br />    situ hybridizations. expressed sequence tag (EST) clone             days. Additionally, wild-type stock survived marginally CG2196 gene obtained (GH19680 EDM1 133-6918917                          better high salt; 5% NaCl (wt/vol) equivalent 0.85 M, <br /> Drosophila Gene Collection Clone 1) used prepare digoxyge-               50% flies survived 4 days compared 0% reported <br /> nin-labeled DNA probes described previously (18). situ hybrid-            previously 0.8 M (21). difference signifi- <br /> ization performed described previously (18).                             (probably reflecting strain differences), experiments <br />    Survival RNAi flies. RNAi male flies crossed c42                similar pattern reduction survival increasing <br /> virgin female flies cages 10 days, eggs collected          NaCl concentration. Flies fed NaCl food 5% survive <br /> morning, counted, transferred vials food containing 4%              significantly better starved flies, exposed NaCl. percentage flies surviving long reach adult          higher salt levels die fast flies deprived food <br /> stage compared parental lines. 12 vials              altogether. suggests short-term salt-handling mecha- line collected. repeats experiment                                                                                 nisms offer partial protection salt levels elevated performed. <br />                                                                                 5% days. previous <br />    Generation CG2196::eYFP-overexpressing flies. Flies overex- <br /> pressing construct CG2196 gene tagged enhanced yellow <br />                                                                                 results, behavioral adaptation high salt apparent: insects <br /> fluorescent protein (eYFP) generated using Gateway              provided high-salt diet continue feed  survive entry destination vectors used obtained                   better insects food) feed avidly flies <br /> Drosophila Gateway Vector collection (37). open reading frame               normal diet. subsequent microarray time-course gene obtained EST clone inserted frame Gateway destination vector pTWV, encoding fluorescently <br /> COOH-terminal tagged CG2196, flies germ-line transformed described previously. construct control UAS <br /> promoter, allowing cell-specific expression control appro- <br /> priate GAL4 driver lines. <br />    Expression S2 cells. Drosophila S2 cells grown Schnei- <br /> der&#226;&#8364;&#8482;s medium according standard protocols. Genes cloned destination vector pAWV (Drosophila Gateway Vector <br /> Collection). vector contains constitutive ActinC5 promoter. S2 <br /> cells transfected according Invitrogen protocol  <br /> scribed previously (32). Approximately 3 &#226;&#171;&#187; 106 cells used transfection. cells harvested PBS allowed attach poly-<span id='am-22' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-23' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span>L-lysine</span>-treated glass slides. Cells counter- <br /> stained nuclear stain DAPI washed repeatedly viewed Zeiss 510 Meta confocal microscope. <br />    Statistics. fluid secretion experiments, significance results determined unpaired Student&#226;&#8364;&#8482;s t-test, taking critical <br /> value P 0.05  tailed). microarray experiments, FDR            Fig. 2. Numbers genes  downregulated different times start significance taken 5%.                                               salt exposure, 1 5% false discovery rate (FDR). <br />  <br />                                                 Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  4                                                          SALT STRESS DROSOPHILA <br />  <br /> Table 1. Iterative Group Analysis gene families  downregulated dietary salt stress <br />                                                                                                                   % Gene Family Changed Sample Time, h <br />  <br />                              Gene Ontology Code                                             Group            4               8             16               32 <br />  <br />                                                                Upregulated gene families <br /> 0008745-N-acetylmuramoyl-<span id='am-57' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-58' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span>L-alanine</span> amidase activity                                       immune             25 <br /> 0006955-immune response                                                                   immune             11 <br /> 0045087-innate immune response                                                            immune             14 <br /> 0042742-defense response bacterium                                                     immune             14 <br /> 0050829-defense response Gram-negative bacterium                                       immune             18 <br /> 0050830-defense response Gram-positive bacterium                                       immune              9 <br /> 0019731-antibacterial humoral response                                                    immune             25 <br /> 0009253-<span id='am-91' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-92' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-100' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span>peptidoglycan</span> catabolic process                                                   immune             25 <br /> 0042834-peptidoglycan binding                                                             immune             25 <br /> 0030342-1 25-dihydroxyvitamin D3 (1,25-(OH)2D3) 24-hydroxylase activity                                   100 <br /> 0004558-alpha-glucosidase activity                                                        sugar              25              42             42               50 <br /> 0043169-cation binding                                                                                       10              10             16               14 <br /> 0004613-phosphoenolpyruvate carboxykinase (<span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span>GTP</span>) activity                                                    100             100            100              100 <br /> 0004611-phosphoenolpyruvate carboxykinase activity                                                          100             100            100              100 <br /> 0006094-gluconeogenesis                                                                   sugar                              40 <br /> 0017076-purine <span id='am-62' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-63' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-69' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> binding                                                                                            40 <br /> 0004364-<span id='am-40' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-47' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span>glutathione</span> transferase activity                                                                                     13 <br /> 0008643-carbohydrate transport                                                            sugar                               9              9 <br /> 0005355-glucose transmembrane transporter activity                                        sugar                              18             24 <br /> 0005351-sugar:hydrogen ion symporter activity                                                                                11             14 <br /> 0006006-glucose metabolic process                                                         sugar                              27             27               27 <br /> 0005792-microsome                                                                                                                            4 <br /> 0009063-amino acid catabolic process                                                        AA                                              17 <br /> 0016798-hydrolase activity, acting glycosyl bonds                                      sugar                                             11 <br /> 0005996-<span id='am-107' about='obo:IMR_0002125' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasDbXref" content="MeSH:D009005" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:id" content="IMR:0002125" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="monosaccharide" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasDbXref" content="CHEBI:35381" datatype="xsd:string"></span>monosaccharide</span> metabolic process                                                  sugar                                             22               17 <br /> 0009113-purine base biosynthetic process                                                                                                    75               75 <br /> 0006144-purine base metabolic process                                                                                                        9               15 <br /> 0003939-L-iditol 2-dehydrogenase activity                                                                                                  100              100 <br /> 0006164-purine nucleotide biosynthetic process                                                                                              40               50 <br /> 0006189- novo&#226;&#8364;&#8482; <span id='am-113' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-114' property="rdfs:label" content="IMP" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span>IMP</span> biosynthetic process                                                                                                  43               57 <br /> 0006012-galactose metabolic process                                                       sugar                                             50               50 <br /> 0008652-amino acid biosynthetic process                                                     AA                                                               10 <br /> 0004477-methenyltetrahydrofolate cyclohydrolase activity                                                                                                    100 <br /> 0009072-aromatic amino acid family metabolic process                                        AA                                                               38 <br /> 0004488-methylenetetrahydrofolate dehydrogenase. (NADP&#226;&#171;&#185;) activity                                                                                           100 <br /> 0006559-<span id='am-52' about='obo:IMR_0200274' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-53' property="oboInOwl:id" content="IMR:0200274" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00079" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="L-Phenylalanine" datatype="xsd:string"></span>L-phenylalanine</span> catabolic process                                                   AA                                                               50 <br />                                                             B. Downregulated gene families <br /> 0005730-nucleolus                                                                                           12 <br /> 0008143-poly  binding                                                                                     20 <br /> 0005616-larval serum protein complex                                                                        33 <br /> 0005344-oxygen transporter activity                                                                         14 <br /> 0045735-nutrient reservoir activity                                                                         25 <br /> 0046870-cadmium ion binding                                                                                 67              67 <br /> 0005786-signal recognition particle, endoplasmic reticulum targeting                                                         27 <br /> 0004035-alkaline phosphatase activity                                                                                        23 <br /> 0005316-high affinity inorganic phosphate:sodium symporter activity                                                          22 <br /> 0005048-signal sequence binding                                                                                              75 <br /> 0006621-protein retention ER                                                                                             100 <br /> 0050830-defense response Gram-positive bacterium                                       immune                             17 <br /> 0045087-innate immune response                                                            immune                             15            10 <br /> 0006955-immune response                                                                   immune                             13             7 <br /> 0042742-defense response bacterium                                                     immune                             16            11 <br /> 0050829-defense response Gram-negative bacterium                                       immune                             27            18 <br /> 0019731-antibacterial humoral response                                                    immune                             32            21 <br /> 0005784-translocon complex                                                                                                   43            43 <br /> 0042562-hormone binding                                                                                                      67                             67 <br /> 0007321-sperm displacement                                                                 reprod.                           50                             75 <br /> 0045861-negative regulation proteolysis                                                                                   50                             63 <br /> 0007610-behavior                                                                                                             13                             23 <br /> 0004182-carboxypeptidase activity                                                                                                        17 <br /> 0007305-vitelline membrane formation chorion-containing eggshell                        reprod.                                          57 <br /> 0007343-egg activation                                                                     reprod.                                         33 <br /> 0008316-structural constituent vitelline membrane                                       reprod.                                         100 <br /> 0004181-metallocarboxypeptidase activity                                                                                                   33 <br />                                                                                                                                                    Continued <br />  <br />                                                   Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                  SALT STRESS DROSOPHILA                                                                        5 <br /> Table 1. &#226;&#8364;&#8221;Continued <br />                                                                                                                       % Gene Family Changed Sample Time, h <br />  <br />                                 Gene Ontology Code                                              Group             4              8              16              32 <br />  <br /> 0004263-chymotrypsin activity                                                                                                                   18              23 <br /> 0004806-triacylglycerol lipase activity                                                                                                         11              16 <br /> 0008970-phospholipase A1 activity                                                                                                               30              20 <br /> 0016045-detection bacterium                                                                immune                                                            33 <br /> 0004867-serine-type endopeptidase inhibitor activity                                                                                                            12 <br /> 0016298-lipase activity                                                                                                                                         16 <br /> 0046692-sperm competition                                                                     reprod.                                                           50 <br /> 0042600-chorion                                                                               reprod.                                                           38 <br /> 0005213-structural constituent chorion                                                     reprod.                                                           78 <br /> 0007306-eggshell chorion formation                                                            reprod.                                                           36 <br />   Gene families significantly  iterative group analysis, Ref. 9) overrepresented  downregulated groups ordered time onset highlit following names: immune, immune/defence response; reprod, reproduction; sugar, carbohydrate metabolism; AA, amino acid <br /> metabolism. Note  limitations hierarchical nature gene ontology annotation, substantial overlap genes reported related headings. Nonetheless, trends clear. <br />  <br />  <br /> experiment, shorter times (0 &#226;&#8364;&#8220; 8 h) taken repre-                       expression synthetic enzyme trehalose 6-phosphate <br /> sentative acute salt exposure, longer time points                          synthase (tps1 tps2), corresponding increase (16 &#226;&#8364;&#8220;32 h) expected additional effects                            trehalose levels. Interestingly, early changes reduced food intake.                                                                  trehalose metabolism array,    Microarray acute NaCl stress. results,                             significant changes 16 h: synthetic enzyme trehalose <br /> microarray experiment devised comparing matched sets                           6-phosphate synthase 1 (tps1) downregulated, flies transferred normal diet normal 4% salt                        catabolic &#226;&#65533;&#163;-trehalase (CG16965) upregulated. Accord- <br /> diet sampled 4, 8, 16, 32 h. numbers genes                       ingly, trehalose does appear key osmolyte  downregulated increased steadily time,                           response acute salt stress insect. Rapid conversion <br /> summarized Fig. 2. changed genes listed Supple-                         glucose trehalose passage insect midgut <br /> mentary Table S1,1 iGA changed gene families                            classically seen way maintaining favor- <br /> shown Table 1. iGA gene list shows compelling                             able concentration gradient glucose absorption features. viewed time series, response                           gut (38), possible  insects, trehalose lost <br /> resolved phases. Initially, prominent                         primitive role osmolyte. <br /> immune response, multiple genes (metchnikowin; droso-                               key change osmotically stressed worms cin; diptericin; attacin; immune-induced molecules 1, 10,                         upregulation chaperones heat shock protein (hsp) <br /> 23; PGRP-SB1) upregulated 4 h downregulated 8                           family, presumably maintain protein folding changing <br /> 16 h. 8 16 h, major response upregulation                         ionic strength.  hsps underrepresented genes associated carbohydrate metabolism. Conspicu-                              change list Drosophila (Supplementary Table S1): ously, half Drosophila &#226;&#65533;&#163;-glucosidase (maltase)                           hsp68 change significantly gene family upregulated, &#226;&#65533;&#163;-amylase                        single time point. impression gained Dro- <br /> zinc finger transcription factor sugarbabe (sug), transcription                     sophila, relatively sophisticated osmoregulatory <br /> factor previously implicated starvation                           defend internal environment effectively <br /> sugar stress responses (45). Sugarbabe upregulation feed-                       dietary salt loading extended period  ing thought downregulate sugar metabolism (45),                          days), death ensues. <br /> downregulation consistent upregulation carbo-                            Ion transport salt loading. Ion transport genes, hydrate metabolism.                                                                   obvious candidates change, relatively mod- <br />    Sugar metabolism general response salt loading.                        estly represented; putative sodium/halide cotransporter marked changes genes involved carbohydrate metabolism                           upregulated, does change hugely.  evocative studies high-salt diet species.                       major insect epithelial transport genes [Na&#226;&#171;&#185;, K&#226;&#171;&#185; ATPase, yeast C. elegans, salt loading causes accumulation                       V-ATPase, Na&#226;&#171;&#185;/H&#226;&#171;&#185; exchangers, Na&#226;&#171;&#185;/K&#226;&#171;&#185;/2Cl&#226;&#171;&#186; cotransporters glycerol protective osmolyte (25) (24). yeast (24)                     (40)] feature list. <br /> HOG stress pathway control Hog1 kinase (24).                             tissues critical salt response?. mi- Drosophila Hog1 homolog mpk2 does change                             croarray experiments performed flies; al- <br /> significantly experiment (Supplementary Table S1),                                                                                       design reduces sensitivity experiment <br /> does preclude posttranscriptional modulation glycerol <br />                                                                                       tissue-specific changes expression (11), avoids <br /> accumulation, absolutely exclude mecha- <br />                                                                                       preconceptions major tissues involved salt <br /> nism insects. <br />                                                                                       response.  work performed,    Trehalose key osmolyte bacteria, yeast,                                                                                       FlyAtlas online resource available, allowing <br /> plants (17); C. elegans, osmotic stress increases <br />                                                                                       basal expression levels ascertained Drosophila <br />                                                                                       gene 14 tissues life stages (11).    1 <br />       online version article contains supplemental material.              original data obtained flies, possible                                                       Physiol Genomics &#226;&#8364;&#162; VOL   37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  6                                                                     SALT STRESS DROSOPHILA <br />  <br /> Table 2. genes modulated dietary salt predominantly expressed tubule hindgut <br />                                                             Number Genes Maximal Expression Tissue <br />  <br />     Treatment     Brain     Head      Crop      Midgut        Tubule      Hindgut      Ovary         Testis      Acc       l_fat         tag    car    Fly       Total       &#226;&#65533;&#185;2 <br />  FlyAtlas     1,783      463      1,134      1,324           791           571      2,563         2,655     2,386       1,518         900   1,856   144      18,088 <br /> 4 h               0        0          0          2            25            22          0             0         1           0           5       7     2          64        426 <br /> 8 h               0        0          0          2            25            22          0             0         1           0           4       7     2          63        432 <br /> 16 h              0        2          1         10            33            40          5             0         2           0          16      26     0         135        467 <br /> 32 h              1        7          1         14            37            54          6             0         4           0          26      43     0         193        534 <br /> 4 h             1        1          1          0             0             6          2             0         0           0           1       4     0          16         52 <br /> 8 h             1       12          2          3            36            23         15             8         4          32          24      52   129         341      3,273 <br /> 16 h            1        7          1          1            48            12          3             8         0          10           1       8     4         104        638 <br /> 32 h            0        6          0          0            49            10          3            33         0          19           1       6     5         132        545 <br />   time point, probe set significantly changed (false discovery rate &#226;&#172;&#65533; 5%) expression screened Flyatlas.org expression dataset, tissue maximum expression probe set identified. total scores tissue compared profile obtained genes FlyAtlas, &#226;&#65533;&#185;2 calculated. 12 d.f., critical value &#226;&#65533;&#185;2 21, genes  downregulated time points significantly <br /> nonrandomly drawn different tissues: specifically, excretory (tubule hindgut, shown bold) genes massively overrepresented. <br /> Abbreviations tissues: tag, thoracicoabdominal ganglion; acc, accessory gland; car, carcass; l_fat, larval fat body; fly, fly. Details FlyAtlas dataset published previously (11). <br />  <br />  expression FlyAtlas 19 sig-                                      sively upregulated midgut, sugarbabe nificantly changed genes  called MAS 5.0) house-                                    widely expressed. expression patterns CG2196 keeping (generally expressed), tissue                                     CG6486 validated qPCR, confirming mas- <br /> specific, particular tissues emerge com-                                   sive expression genes tubules midgut <br /> mon candidates salt response. results (Table 2)                                   respectively (Table 3B). fly quantitative PCRs remarkable;  downregulated genes                                 genes performed 8 h time point com- <br /> highly tissue-specifically expressed,                                pared time-course microarray data (Fig. 3). highest expression hindgut Malpighian (renal)                                cases, reasonable agreement. CG2196 CG6484 <br /> tubules, key osmoregulatory tissues excretory                                  consistently increasingly upregulated time, <br /> cycle (4), specific midgut. model                                 sugarbabe consistently downregulated.  response high levels salt,  massive                                  based limited sample array, FlyAtlas qPCR data <br /> upregulation carbohydrate metabolism  reflecting                                   concordant. <br /> sharply increased energy demands) hindgut Mal-                                        Role CG2196/salty dog. CG2196 selected pighian tubules, control sugarbabe  tran-                                  study, predicted represent sodium/halide sym- <br /> scription factor previously implicated dietary stress                                   porter annotation Drosophila genome, <br /> sponse, predominantly expressed midgut,                                   highly specifically expressed Mal- <br /> tubules, hindgut, fat body), tissues rela-                                pighian tubules, key tissues response salt <br /> tively unaffected.                                                                            (Table 2). According Drosophila tradition whimsical    Validation selected genes. encompass range                                  informative names, gene named salty dog (salt). <br /> responses induced salt loading, genes selected                                  Firstly, expression salt tubule established study: SLC5 cotransporter CG2196 (annotated                                   situ hybridization (Fig. 4). Expression seen FlyBase sodium/halide symporter), GLUT4/8-like                                    principal cells  stellate cells) length sugar transporter CG6484, transcription factor sugarb-                                tubule particularly main segment. initial segment <br /> abe. According FlyAtlas (Table 3A), basal expression                                    stained. localization reasonable, CG2196 virtually tubule-specific, CG6484 mas-                                   principal cells major active ion-transporting cell type  <br /> Table 3. Genes selected study <br />                                                                 FlyAtlas expression levels multiple tissues <br />  <br />      Gene         Brain       Head        Tag        Crop           Midgut          Tubule          Hindgut        Ovary           Testis      Acc     Car       L_fat        Fly <br />  <br /> CG2196              7           4          5            7                 8         7,248             70               4             6          11      21          5         275 <br /> CG6484              6          22          4            7             2,738            12             32               4             8         523     161          6         178 <br /> sugarbabe          10         197         10            8               584           455             95               5             6           8     371        550         137 <br />  <br />                                        B. qPCR validation CG2196 CG6484 expression Malpighian tubules, guts, heads <br />  <br />                                                                                              Relative Expression Level (w.r.t. rp49) <br />  <br />     Gene                                        Head                                                          Midgut                                         Malpighian tubules <br />                                                 &#226;&#171;&#186;5 <br /> CG2196                                 4.8&#226;&#171;&#187;10 &#226;&#171;&#190;1.2&#226;&#171;&#187;10&#226;&#171;&#186;5                                             3.6&#226;&#171;&#187;10&#226;&#171;&#186;5&#226;&#171;&#190;4.5&#226;&#171;&#187;10&#226;&#171;&#186;6                                        0.129&#226;&#171;&#190;0.008 <br /> CG6484                               2.38&#226;&#171;&#187;10&#226;&#171;&#186;7&#226;&#171;&#190;1.8&#226;&#171;&#187;10&#226;&#171;&#186;8                                             8.7&#226;&#171;&#187;10&#226;&#171;&#186;5&#226;&#171;&#190;1.5&#226;&#171;&#187;10&#226;&#171;&#186;5                                    3.3&#226;&#171;&#187;10&#226;&#171;&#186;8&#226;&#171;&#190;4.2&#226;&#171;&#187;10&#226;&#171;&#186;9 <br />    FlyAtlas expression profile multiple tissues shown. maximal signal gene shown bold. Details FlyAtlas data set published previously (11). B, data shown means &#226;&#171;&#190; SE (n &#226;&#171;&#189; 3 independent biological replicates). <br />  <br />                                                          Physiol Genomics &#226;&#8364;&#162; VOL       37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                   SALT STRESS DROSOPHILA                                                                          7 <br />                                                                                        pressed. S2 cells, CG2196-eYFP <br />                                                                                        protein membrane protein appears mainly                                                                                        plasma membrane. <br />                                                                                           Homozygous CG2196-eYFP flies (insertion chro- <br />                                                                                        mosome) crossed c42-GAL4 Act5C-GAL4 flies, <br />                                                                                        expression chimeric protein assessed (Fig. 5). <br />                                                                                        GAL4 lines drive expression UAS constructs                                                                                        tubule principal cells  salt normally expressed (Fig. <br />                                                                                        4)] ubiquitously, respectively. CG2196-eYFP                                                                                        driven tubule principal cells main lower segment <br />                                                                                        c42-GAL4, directed basolateral membrane infoldings <br />                                                                                        (Fig. 5, B C). promoter does drive expression <br />                                                                                        stellate cells initial segment tubule,                                                                                        staining seen  <br />                                                                                           Act5C-Gal4 promoter, staining prin- <br />                                                                                        cipal cells exactly c42 promoter.                                                                                        progeny cross (Act5C-GAL4 flies) &#226;&#171;&#187; (CG2196-eYFP <br />                                                                                        flies) expresses CG2196-eYFP construct principal <br />                                                                                        cells main lower segment foldings  <br />  <br />  <br />  <br /> Fig. 3. Time course expression selected genes validation 8 h. gene (CG2196, sugarless, CG6844), mean fold change <br /> (salt relative normal diet) plotted 4, 8, 16, 32 h. Bars heavy <br /> lines significantly different change hypothesis FDR &#226;&#172;&#65533;5%. Array <br /> fold changes compared qPCR measurements (shaded boxes) 8 h (means &#226;&#171;&#190; SE, n &#226;&#171;&#189; 3; error bars visible, small plot). <br />  <br />  <br />  tissue (15). staining seen tissues sense control. <br />    Salt localizes basolateral plasma membrane. Dro-                             Fig. 4. situ hybridizations CG2196 Malpighian tubules adult <br /> sophila S2 cells transiently transfected                                   Oregon R flies. Staining anti-sense probe. B: clear staining appears <br /> salt::eYFP construct using calcium phosphate transfec-                             ureter principal cells main lower segment tion method. S2 cells present clear staining                          posterior anterior Malpighian tubules. cases, staining <br />                                                                                        main segment tubule stronger compared lower <br /> plasma membrane (Fig. 5A).  stain-                              segment. initial segment stained pairs tubules. C ing vesicles conglomerations protein inside                           D: sense controls. photographs, diameter tubule taken <br /> cell, frequently observed proteins overex-                                 35 &#226;&#65533;&#174;m. <br />  <br />                                                       Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  8                                                               SALT STRESS DROSOPHILA <br />  <br />  <br />  <br />  <br /> Fig. 5. Expression pattern CG2196-eYFP vitro vivo.  Drosophila S2 cells transfected CG2196::eYFP construct control actin 5C promoter. clear staining plasma membrane vesicles conglomerations protein inside cell. B&#226;&#8364;&#8220;G: CG2196::eYFP <br /> construct flies. B: overview expression driven tubule principal cells c42 GAL4 line. C: construct c42 promoter appears principal <br /> cells infoldings basolateral plasma membrane. Nuclei counterstained blue DAPI C. D&#226;&#8364;&#8220;F: CG2196::eYFP construct Act5C-GAL4 line, drives ubiquitous expression. Stellate cells middle segment express construct (D). E F focus cells <br /> differs; E focused midway tubular epithelium, F focused tubule. photographs, diameter tubule taken 35 &#226;&#65533;&#174;m. <br />  <br />  basolateral membrane                                 iment shown Fig. 6. construct produces signif- <br /> endoplasmic reticulum, stellate cells                             icant (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; 0.05 significant) <br /> middle segment staining observed (Fig. 5D). salt                           knockdown salt expression, Act5C&#226;&#172;&#381;salt-RNAi expressed ion-transporting cells Malpighian                          (18.8 times lower parental RNAi flies), c42&#226;&#172;&#381;salt- <br /> tubule, basolateral membrane, consistent                              RNAi (17.7 times salt expression RNAi flies <br /> presumed function uptake symporter.                                             10.7 times salt expression c42 flies). <br />    Modulation salt expression tubules impacts                                   Salt upregulated response salt challenge, does <br /> organismal survival salt challenge. Flies transgenic                        RNAi-mediated salt knockdown just tubule reduce UAS-salt-RNAi construct generated, extent                               survival organism challenged di- <br /> knockdown assessed qPCR. UAS-salt-RNAi flies                                   etary salt? progeny overexpression knock- crossed Act5C-GAL4 c42-GAL4 driver                                  crosses principal cell-specific driver c42 lines, expression levels salt Malpighian tubules                            raised food containing 4% NaCl, survival compared <br /> compared parental controls. results exper-                           parental controls. results shown Fig. 7.                                                      Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />                                                                 SALT STRESS DROSOPHILA                                                          9 <br />                                                                                         Salt highly tubule-specific gene, expressed                                                                                      basolateral plasma membrane tubule,                                                                                      strongly regulated response dietary salt.  <br />                                                                                      response inappropriate, c42&#226;&#172;&#381;salt-RNAi flies <br />                                                                                      actually survive better high salt. probable <br />                                                                                      reason behavior overexpression salt <br />                                                                                      salt diet miscalculated response organism. <br />                                                                                      inappropriate responses commonly encountered                                                                                      human health disease: example, cytokine storm <br />                                                                                      elicited healthy humans exposed influenza                                                                                      dangerous original infection. speculate flies <br />                                                                                      sense increasing Na&#226;&#171;&#185; levels associated dietary <br />                                                                                      salt stress increase Na&#226;&#171;&#185; excretion tubule                                                                                      upregulating basolateral Na&#226;&#171;&#185; uptake.  possible <br />                                                                                      osmolyte cotransported Na&#226;&#171;&#185; salt                                                                                      handled tubule, RNAi salt actually <br />                                                                                      improves situation. <br />                                                                                         does salt/salty dog actually encode?                                                                                      evidently member characterized SLC5 (sodi- <br />                                                                                      um/solute symporter) gene family (43, 44), answer Fig. 6. Quantification CG2196-RNA interference (RNAi) induced knock-              surprisingly vague. Drosophila genome project (fly- CG2196 mRNA levels. UAS-CG2196-RNAi flies crossed ubiquitous (Act5C-GAL4) Malpighian tubule principal cell-specific              base.bio.indiana.edu) annotates sodium/iodide sym- <br /> (c42) GAL4 driver tubule mRNA levels measured qPCR relative            porter. NCBI BLASTP search human sequences <br /> rp49 reference gene parentals progeny. Act5C-GAL4              produces closest match (E value 3e-87) SLC5A12, flies expression levels higher, c42 slightly lower,         second extremely close match (E value 2e-82) SLC5A8. <br /> RNAi parents, drive RNAi highly efficiently: CG2196 expression Act5C-GAL4&#226;&#172;&#381;progeny 19&#226;&#171;&#187;  c42-GAL4&#226;&#172;&#381;RNAi progeny 18&#226;&#171;&#187; <br />                                                                                      cotransports short chain fatty acids, lactate,  RNAi flies. RNAi/CyO control progeny express CG2196 levels          nicotinate sodium kidney, small intes- exactly RNAi flies. Data expressed relatively      tine, skeletal muscle; cotransports <br /> CG2196 expression CG2196-RNAi parental line normalized            short chain fatty acids, lactate, nicotinate, iodide rp49 reference gene. *Significant change control  CG2196-RNAi           sodium thyroid, kidney, intestine (43, <br /> parental line) (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; 0.05 significant). <br />                                                                                      44).  CLUSTALX alignment protein se- <br />                                                                                      quences human SLC5 members (Fig. 8) places salt <br />                                                                                      SLC5A6, sodium/<span id='am-10' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>vitamin</span> cotransporter, seen results, repeats overexpres-                          SLC5A7, sodium/choline cotransporter. list sion salt consistent effect survival. Surpris-                          exhaustive, SLC5 members thought ingly, c42&#226;&#172;&#381;salt-RNAi flies survive better food containing                         facilitate transfer water organic solutes 4% NaCl parental line (RNAi c42).                                 <span id='am-118' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-119' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span>.  based structure, salt considered percentage RNAi-c42 flies surviving long                           Na&#226;&#171;&#185;/solute cotransport ascribed particular double of parents, result                        transport substrate. Clearly,  exact function salty <br /> statistically significant (Student&#226;&#8364;&#8482;s t-test, tailed).                         dog require extensive work, functional <br /> surprisingly, upregulation salt salt challenge ob-                         workup SLC5 family members known problem- <br /> served microarray qPCR appears counteradap-                             atic (43, 44),  SLC5 members uptake trans- <br /> tive.                                                                                porters, salty dog localized basolateral mem- <br />                                                                                      brane Malpighian tubule, upregulation response <br /> DISCUSSION                                                                           dietary salt clearly increase sodium clearance                                                                                      blood. overexpression does appear <br />    results provide comprehensive view                              sufficient confer enhanced survival organism salt stress insect <span id='am-9' about='oboInOwl:date' typeof='owl:Thing'>date</span>. Previous studies associated                        salt challenge (Fig. 7), observation knockdown salt stress (NaCl KCl) inebriated neurotrans-                           gene actually beneficial (Fig. 7) intriguing. <br /> mitter/osmolyte transporter (ine) (21), studies                              compelling aspect work dehydration rehydration (2, 19) demonstrated Dro-                           clear phasing different classes response sophila melanogaster strong osmoregulator.                            experimental period (Table 2), particularly early induc- <br /> dietary salt stress induced clear reproducible sequence                        tion, repression, immune response. tubule, changes gene expression involving immune/stress                                classically considered fluid-transporting response, carbohydrate metabolism, ion transport path-                           osmoregulating tissue (14), actually plays multiple roles ways. longer term (&#226;&#172;&#381;16 h), effects analogous                               organism (16). Recently, shown mount starvation seen, reproductive axis shut                         robust, autonomous immune response pathogens (28), males females. interesting sugarbabe,                       mediated canonical Toll/imd immune pathways (22). transcription factor associated dietary sugar loading,                        machinery immune response changing study; suggests sug play                         place tubules; intriguing invoked general role response dietary stress.                                          early salt stress response. speculate  <br />                                                     Physiol Genomics &#226;&#8364;&#162; VOL    37 &#226;&#8364;&#162;   www.physiolgenomics.org <br />  10                                                                SALT STRESS DROSOPHILA <br />  <br />  <br />  <br />  <br /> Fig. 7. Effect modulation tubule CG2196 <br /> expression fly survival salt chal- <br /> lenge. Flies allowed lay eggs 4% NaCl <br /> diet, numbers progeny reaching adult- <br /> hood counted compared survival RNAi, eYFP, c42 parental lines. c42 line <br /> drives expression selectively tubule principal <br /> cells. experimental condition, 2 indepen- <br /> dent replicates shown. B: tubule-specific <br /> RNAi CG2196 enhances survival 4% <br /> salt. C D: tubule-specific overexpression eYFP-tagged CG2196 significant impact expression relative parental controls. *Significant <br /> change control (Student&#226;&#8364;&#8482;s t-test, taking P &#226;&#172;&#65533; <br /> 0.05 significant). <br />  <br />  <br />  <br />  <br /> ing implies generality stress signaling,                              course, nature Drosophila does face quite tissue; irrespective precise nature stress, set                          problems excess salt  example, mosquito larvae key adaptive genes quickly upregulated, selec-                                live hypersaline lakes.  importantly, results <br /> tively turned nature stress app-                               provide baseline assess specialized <br /> arent.                                                                                   responses insects evolutionary history equipped <br />  <br />  <br />  <br />  <br /> Fig. 8. human SLC5 sodium/solute sym- <br /> porter similar CG2196? reference <br /> protein sequence 12 human SLC5 <br /> members downloaded NCBI, CG2196. aligned CLUSTALX default parameters, PHYLIP tree gener- <br /> ated 1,000 bootstrapping. tree viewed Treeview (30). <br />  <br />                                                                      11 ability osmoregulate thrive contin-                       <br /> uous osmotic stress.                                                                   <br />   <br /> </body></html>