<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, 2008, p. 3444&#226;&#8364;&#8220;3455                                                                            Vol. 190,  10 <br /> 0021-9193/08/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.00141-08 <br /> Copyright &#194;&#169; 2008, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />          Ortholog OxyR Legionella pneumophila Expressed <br />             Postexponentially Negatively Regulates Alkyl <br />                  Hydroperoxide Reductase (ahpC2D) Operon&#228;&#338;&#164; <br />                            Jason J. LeBlanc,1,2 Ann Karen C. Brassinga,4 Fanny Ewann,4 <br />                                  Ross J. Davidson,1,2,3 Paul S. Hoffman2,3,4,5* <br />               Department Pathology Laboratory Medicine, Queen Elizabeth II Health Sciences Center, Halifax, <br />                  Nova Scotia, Canada1; Department Microbiology Immunology2 Medicine,3 Faculty                   Medicine, Dalhousie University, Halifax, Nova Scotia B3H 4H7, Canada; Department                      Medicine, Division Infectious Diseases International Health4 Department                                     Microbiology,5 University Virginia School Medicine, <br />                                                Charlottesville, Virginia 22908-1340 <br />                                            Received 28 January 2008/Accepted 11 March 2008 <br />  <br />             Legionella pneumophila expresses peroxide-scavenging alkyl hydroperoxide reductase systems (AhpC1          AhpC2D) expressed differentially bacterial growth cycle. Functional loss postexponentially <br />          expressed AhpC1 compensated increased expression exponentially expressed AhpC2D <br />           study, used acrylamide capture DNA-bound complexes (ACDC) technique mass <br />          spectrometry identify proteins bind promoter region ahpC2D operon. major protein <br />          captured ortholog OxyR (OxyRLp). Genetic studies indicated oxyRLp essential gene expressed <br />          postexponentially partially complemented Escherichia coli oxyR mutant (GS077). Gel shift assays <br />          confirmed specific binding OxyRLp ahpC2D promoter sequences, promoters ahpC1 oxyRLp; <br />           OxyRLp weakly bound E. coli OxyR-regulated promoters (katG, oxyR, ahpCF). DNase protection <br />          studies showed OxyRLp binding motif spanned promoter transcriptional start sequences ahpC2 <br />          protected region unchanged treatments reducing agents hydrogen peroxide (<span id='am-69' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span>H2O2</span>). <br />           OxyRLp (pBADLpoxyR)-mediated repression ahpC2-gfp reporter construct E. coli GS077           oxyR mutant) reversed H2O2 challenge. Alignments OxyR proteins revealed <span id='am-51' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-52' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> <br />          substitutions predicted ablate thiol oxidation conformational changes required activation. suggest          mutations locked OxyRLp active DNA-binding conformation, permitted divergence          function regulator oxidative stress cell cycle regulator, controlling gene expression          postexponential differentiation. <br />  <br />  <br />    Legionella pneumophila aquatic moist soil                phagocytes (3, 6, 8, 19, 27) important environments intracellular parasite protozoa (12, 14,              environmental persistence (25). <br /> 15). Legionnaires&#226;&#8364;&#8482; disease results aerosol transmission                previously characterized peroxide-scavenging alkyl <br /> bacteria environment susceptible humans. nat-                hydroperoxide reductase systems L. pneumophila: AhpC1, <br /> ural hosts  mammalian cell lines), L. pneumophila                  similar AhpC-thioredoxin Helicobacter pylori <br /> displays dimorphic lifestyle vegetative replicating              (30, 49), AhpC2D, similar AhpC AhpD bacilli differentiate metabolically dormant cyst-like plank-           mycobacteria (30, 42). studies demonstrated <br /> tonic forms (17). Analogous small-cell variants Cox-              systems contributed efficient scavenging H2O2 <br /> iella burnetii elementary bodies Chlamydia spp.,            organic peroxides  <br /> highly resilient cyst-like forms L. pneumophila enable               quired viability.  genes inversely ex- <br /> organism survive extracellularly extended periods                 pressed growth cycle; ahpC2D induced early time highly infectious state. cyst form likely contrib-            exponential phase repressed postexponentially, concomi- <br /> utes noted resilience L. pneumophila actions            tant induction ahpC1. Finally, compensatory increase <br /> oxidizing agents (chlorine), chloramines, biocides, used                                                                             expression ahpC2D observed ahpC1 mutant, <br /> control infection man aquatic systems (17).                                                                             suggesting ahpC2D operon upregulated lines evidence suggest antioxidant enzymes play crucial <br />                                                                             response changes cellular redox status increases roles early stages infection protozoa human <br />                                                                             intracellular peroxides.  previous studies anti- <br />                                                                             oxidant defense genes L. pneumophila (sodB, sodC, katA, <br />                                                                             katB, ahpC1, ahpC2D) concluded   * Corresponding author. Mailing address: Division Infectious           genes activated response oxidative stress (4, 5, 30, <br /> Diseases International Health, Room 2146, MR-4 Bldg., 409 Lane          39, 43). contrast, studies revealed Road, University Virginia Health Systems, Charlottesville, VA <br />                                                                             antioxidant defense genes, exception sodB (consti- <br /> 22908-1340. Phone: (434) 924-2893. Fax: (434) 924-0075. E-mail: <br /> psh2n@virginia.edu.                                                         tutive), growth stage regulated. <br />   &#228;&#338;&#164; <br />     Published ahead print 21 March 2008.                                 emerging pattern growth stage-regulated antioxidant <br />  <br />                                                                      3444 <br />  VOL. 190, 2008                                                               OxyRLp REGULATION ahpC2D LEGIONELLA                                 3445 <br />  <br />                                           TABLE 1. Strains, genotypes, plasmids used study <br /> Bacterial strain, genotype, <br />                                                                  Relevant property(ies)                                                 Source reference <br />         plasmid <br />  <br /> E. coli <br />   DH5&#226;?&#163;                        F&#226;&#171;&#186; &#226;?&#190;80dlacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;(lacZYA-argF)U169 endA1 recA1 hsdR17 deoR thi-1 supE44 gyrA96 relA1                       Clontech <br />   BL21 Codon Plus             F&#226;&#171;&#186; ompT hsdS (rB&#226;&#171;&#186; mB&#226;&#171;&#186;) dcm&#226;&#171;&#185; Tetr gal&#226;?&#173; (DE3) endA Hte &#229;&#8230;&#179;argU ileY leuW Camr&#229;&#8230;&#180;                                 Stratagene <br />   MG1655                      K-12, F&#226;&#171;&#186; &#226;?&#173;&#226;&#171;&#186;                                                                                                ATCC 47076 <br />   GS077                       MG1655-derived oxyR::kan                                                                                   51 <br />  <br /> L. pneumophila <br />   Lp02                        Philadelphia-1 derivative, rpsL hsdR thyA (Smr) mutant                                                     7 <br />   ahpC1::kan                  Lp02 ahpC1::kan mutant                                                                                     30 <br />   ahpC2D::kan                 Lp02 ahpC2D::kan mutant                                                                                    30 <br />   rpoS::kan                   Lp02 strain MB 379::kan                                                                                    2 <br />   letA::kan                   Lp02 strain MB 414::kan                                                                                    22 <br />   himA::gm himB::kan          Lp02 integration host factor double mutant (lacking &#226;?&#163; &#226;?&#164; subunits); Kanr Gmr                       laboratory <br />  <br />  <br /> Plasmids <br />   pET29b                      IPTG-inducible T7 expression vector                                                                        Novagen <br />   petLpoxyR                   pET29b-derived overexpression vector His6-OxyRLp                                                       study <br />   pBH6119                     RSF1010 ori, promoterless gfpmut3 td (Ampr)                                                              21 <br />   pC2gfp                      PahpC2 region (&#226;&#171;&#186;199 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     30 <br />   pC2(&#226;&#171;&#186;179/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;179 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     study <br />   pC2(&#226;&#171;&#186;156/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;156 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     study <br />   pC2(&#226;&#171;&#186;132/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;132 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     study <br />   pC2(&#226;&#171;&#186;109/&#226;&#171;&#185;27)gfp            PahpC2 region (&#226;&#171;&#186;109 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                     study <br />   pC2(&#226;&#171;&#186;91/&#226;&#171;&#185;27)gfp             PahpC2 region (&#226;&#171;&#186;91 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                      study <br />   pC2(&#226;&#171;&#186;66/&#226;&#171;&#185;27)gfp             PahpC2 region (&#226;&#171;&#186;66 &#226;&#171;&#185;27 bp upstream ahpC2) cloned upstream gfpmut3 pBH6119                      study <br />   poxyRgfp                    oxyR promoter region cloned upstream gfpmut3 pBH6119                                                 study <br />   pBAD22                      Arabinose-inducible expression vector (Ampr)                                                               18 <br />   pBADEcoxyR                  pBAD22-derived expression vector E. coli oxyR                                                          study <br />   pBADLpoxyR                  pBAD22-derived expression vector L. pneumophila oxyR                                                   study <br />  <br />  <br />  <br />  <br /> defense genes L. pneumophila postexpo-                weak binding OxyR-regulated promoters E. coli gel <br /> nential expression transmission (virulence) traits consis-           shift experiments, protein longer redox active tent dimorphic lifestyle organism (17). sup-           repression ahpC2D reversed H2O2. Fi- <br /> port view, microarray analyses comparing sessile               nally, apparent essentiality oxyRLp post- <br /> planktonic cells L. pneumophila biofilm conditions                exponential expression, concomitant repression revealed ahpC2 ahpD highly induced genes                ahpC2D, suggests regulatory roles unrelated oxida- sessile cells (25). Related studies similarly shown           tive stress. <br /> expression ahpC2D operon increases significantly intracellular growth protozoa (8). Despite findings,                                      MATERIALS METHODS <br /> little known regulatory systems controlling ahpC2D                  Bacterial strains growth conditions. Bacterial strains plasmids used expression antioxidant defense genes L. pneumo-                study listed Table 1 oligonucleotides Table 2. L. pneumophila <br /> phila. exception OxyR homolog (described                    strains grown aerobically 37&#194;&#176;C buffered charcoal yeast extract agar                                                                            buffered yeast extract (BYE) broth, supplemented &#226;?&#163;-ketoglutaric acid (1 , typical regulators oxidative stress defenses                                                                             mg/ml), ferric <span id='am-1' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span>pyrophosphate</span> (250 &#226;?&#174;g/ml), <span id='am-12' about='obo:IMR_0200467' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00097" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="L-Cysteine" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200467" datatype="xsd:string"></span>L-cysteine</span> (40 &#226;?&#174;g/ml), <span id='am-64' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span> (100 <br /> SoxRS, OhrR, PerR) absent L. pneumophila                    &#226;?&#174;g/ml), antibiotics required. E. coli strains DH5&#226;?&#163;, MG1655, genome. Growth phase-dependent regulation genes associ-                 GS077 grown 37&#194;&#176;C Luria-Bertani (LB) agar LB broth supple- <br /> ated virulence transmission traits attributed            mented appropriate antibiotics. studies, E. coli GS077 global regulators, stationary-phase sigma           grown 37&#194;&#176;C anaerobic conditions using BD GasPak EZ (Becton <br />                                                                            Dickinson, Oakville, Ontario) AnaeroGen anaerobic atmosphere generation <br /> factor (RpoS) (2, 20), carbon storage regulator (CsrA) (13,            (Oxoid, , Nepean, Ontario). Antibiotics (Sigma-Aldrich Canada , <br /> 33), LetAS, CpxRA, PmrBA component reg-                    Oakville, Ontario) added media following concentrations, ulatory systems (13, 16, 22, 32, 52);  predicted               appropriate: streptomycin (100 &#226;?&#174;g/ml), kanamycin (40 &#226;?&#174;g/ml), chloramphenicol <br /> hierarchal regulatory cascade incomplete,             (20 &#226;?&#174;g/ml), ampicillin (100 &#226;?&#174;g/ml). strains stored &#226;&#171;&#186;85&#194;&#176;C                                                                            nutrient broth containing 10% dimethyl sulfoxide. <br /> regulators linked oxidative stress defenses. <br />                                                                               Standard molecular techniques. Preparation DNA, restriction enzyme di- <br />    report use acrylamide capture DNA-                 gestions, molecular techniques described (40). <br /> bound complexes (ACDC) technique (35) tandem mass                      Plasmid isolation performed QIAprep spin miniprep kit (Qiagen <br /> spectrometry ahpC2D promoter-bound proteins identify                 , Mississauga, Ontario) using standard alkaline lysis method (40). Puri- previously uncharacterized putative regulator oxidative               fication DNA fragments agarose gels subcloning carried                                                                            QIAquick gel purification kit (Qiagen). PCR performed conditions <br /> stress L. pneumophila, designated OxyRLp. studies                 described LeBlanc et al. (30) using oligonucleotides synthesized Integrated oxyRLp able partly complement               DNA Technologies,  (Coralville, IA). DNA protein sequencing oxyR mutant Escherichia coli purified OxyRLp exhibited              carried DalGen Microbial Genetics Center (Halifax, Nova Scotia, <br />  3446       LEBLANC ET AL.                                                                                                                            J. BACTERIOL. <br />  <br />  <br />         TABLE 2. Oligonucleotide primers used study                        addition 1.25 &#226;?&#174;l 5% ammonium persulfate 1.25 &#226;?&#174;l diluted <br />                                                                                    N,N,N&#226;&#172;&#732;,N&#226;&#172;&#732;&#226;&#172;&#732;-tetramethylethylenediamine (1:30 buffer D). reactions        Primer                                  Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;) <br />                                                                                    immediately loaded wells precast nondenaturing 5% polyacrylamide <br /> Ac-FC2PROM ......................GGATCCACCAATATTCCTCCAC                            gel (0.5&#226;&#171;&#187; Tris-borate-EDTA [TBE], Bio-Rad Laboratories). polymeriza- <br />                                            AGGC                                    tion (approximately 5 min), electrophoresis performed 0.5&#226;&#171;&#187; TBE 1 h <br /> RC2PROM ............................GAATTCCTGGAAATTTGTTGCCT                        125 V remove nonspecific proteins. excised gel, <br />                                            ACCG                                    transferred microtube containing 100 &#226;?&#174;l 10% sodium dodecyl sulfate <br /> FECOXYRPBAD ................GGCGAATTCATTATGAATCGTGAT                               (SDS) sample buffer (NEB) 5% &#226;?&#164;-mercaptoethanol heated 3 min                                            CTTGAG                                  94&#194;&#176;C remove captured proteins. brief centrifugation, 50-&#226;?&#174;l aliquots <br /> RECOXYRPBAD................GGCAAGCTTAAACCGCCTGTTTTA                                loaded precast 4 15% Tris-HCl polyacrylamide gel (Bio-Rad <br />                                            AAACTT                                  Laboratories) subjected SDS-polyacrylamide gel electrophoresis (SDS- <br /> FLPOXYRPBAD .................GGCGAATTCATTATGAATTTAAGA                              PAGE). silver staining, proteins distinct DNA controls                                            GATTTACATTAT                            excised submitted mass spectrometry analysis (DalGen Microbial <br /> RLPOXYRPBAD ................GGCAAGCTTCTATGATAATTTGGA                               Genomics Center). <br />                                            TTGAACATT                                  Cloning overexpression OxyRLp E. coli GS077. complementation <br /> FBLA......................................TCCGTGTCGCCCTTATTCC                      E. coli strain GS077 (oxyR::kan) (generously donated Gisela Storz, <br /> RBLA .....................................AACTACGATACGGGAGGGC                      National Institutes Health, Bethesda, MD), E. coli strain K-12 oxyR (oxyREc) <br /> FLPOXYRPET.....................CATATGAATTTAAGAGATTTACA                             oxyRLp expressed using arabinose-inducible expression systems <br />                                            TTAT                                    pBAD22 (18). Amplicons generated PCRs using primer pairs <br /> RLPOXYRPET ....................CTCGAGTGCTAATTTGGATTGAAC                            FECOXYRPBAD RECOXYRPBAD oxyREc FLPOXYRPBAD                                            ATTTT                                   RLPOXYRPBAD oxyRLp, digested EcoRI HindIII subcloned <br /> FC1PROM.............................GAATTCGTGCCATTACTGAGCG                         pBAD22. resulting constructs (pBADEcoxyR pBADLpoxyR)                                            ATTC                                    transformed E. coli DH5&#226;?&#163; verified PCR DNA sequencing. <br /> RC1PROM ............................GGATCCGAGTGCCTTTCATGTA                         Plasmids extracted transformed electrocompetent E. coli GS077. <br />                                            GAGC                                    Ampicillin-resistant GS077 transformants obtained anaerobically  <br /> FLPOXYRPROM ................GAATTCGATGGCACAAAGAGT                                  firmed PCR analysis termed GS077 pBADEcoxyR GS077 pBADL- <br />                                            TGCA                                    poxyR. pBAD22 vector transformed wild-type MG1655 RLPOXYRPROM ...............CTGGGATCCTGCTTTACATCTGC                                 GS077 strains, verified PCR analysis using primers ampicillin resis- <br />                                            CAGG                                    tance cassette FBLA RBLA, termed MG1655 pBAD22 GS077 <br /> FECAHPCPROM ................GGCACTGAAGATACCAAAGG                                   pBAD22, respectively. <br /> RECAHPCPROM................TCGATGAGATGTAAGGTAACC                                      Disk diffusion assays. Strains grown anaerobic conditions mid- <br /> FECKATGPROM................CGAAATGAGGGCGGGAAAAT                                    exponential phase (optical density 620 nm [OD620] 0.4), expression RECKATGPROM ...............CGATACACAGCGTTAGAGAG                                    oxyREc oxyRLp respective pBAD22-derived plasmids induced FECOXYRPROM ...............GGTGCCGCTCCGTTTCTG                                      various concentrations arabinose (0, 0.02, 0.2%) repressed using 0.2% <br /> RECOXYRPROM...............GCTGGCTAACGTGGCAGG                                       glucose. Samples analyzed SDS-PAGE confirm expression FC2PROM179.......................GGCGAATTCCCTTCTTTAAAAGTA                          OxyREc OxyRLp. Cells corresponding 0.1 OD620 inoculated 4 ml <br />                                            GTTTATAAAA                              prewarmed LB 0.75% agar, mixed inversion, poured LB <br /> FC2PROM156.......................GGCGAATTCAAACATAAAAGCCA                           media proper antibiotics presence arabinose glucose.                                            AATATTTTGC                              solidified, 10 &#226;?&#174;l 30% H2O2 placed sterile fourth-inch antibiotic <br /> FC2PROM132.......................GGCGAATTCAAAATGGATTAATA                           disks center plate. Following overnight incubation 37&#194;&#176;C                                            ATTCTTATATTG                            anaerobic conditions, diameters zones inhibition measured <br /> FC2PROM109.......................GGCGAATTCTTGATTGACTTTATT                          reported independent experiments. <br />                                            TATCGTCTT                                  Overproduction purification OxyRLp. primer pair FLPOXYRPET <br /> FC2PROM91.........................GGCGAATTCCGTCTTTATAAAAAC                         RLPOXYRPET used generate amplicon containing NdeI                                            AATTGATTTT                              XhoI restriction sites, respectively. Following digestion, NdeI-XhoI fragment <br /> FC2PROM66.........................GGCGAATTCATTTATTAAATGATT                         subcloned corresponding sites pET29b, generating pETLpoxyR. <br />                                            TCACCTAAATT                             frame fusion L. pneumophila oxyR C-terminal hexameric <br /> RGFP .....................................ACCATAACCGAAAGTAGTGAC                    histidine (His6) tag confirmed DNA sequencing. overexpression,                                                                                    overnight culture E. coli BL21(DE3) Codon Plus harboring pETLpoxyR                                                                                    inoculated 1% (vol/vol) 200 ml fresh LB broth grown OD620                                                                                    0.4. Isopropyl-&#226;?&#164;-D-thiogalactopyranoside added final concentration 1 <br /> Canada). Protein concentrations estimated using Bio-Rad protein assay       mM, incubation continued 60 min 37&#194;&#176;C. Bacteria (Bio-Rad, Hercules, CA).                                                           harvested centrifugation (5,800 &#226;&#171;&#187; g 6 min), resuspended (5 ml/g [wet <br />    ACDC. identify transcription factors bind promoter region     weight]) suspension buffer (20 mM Tris-HCl, pH 7.9; 500 mM NaCl; 5 mM <br /> ahpC2D (PahpC2), ACDC performed according modified version          imidazole; 1 mM PMSF), lysed ice sonication  bursts 10 s methods described Nelson et al. (35). Briefly, PCRs performed using    200 W). Following clarification centrifugation (10,000 &#226;&#171;&#187; g 15 min, 4&#194;&#176;C), <br /> forward primers containing 5&#226;&#172;&#732; Acrydite (Ac) moiety, Ac-FC2PROM,             His6-OxyRLp purified nickel interaction chromatography previously <br /> modified reverse primer RC2PROM. Amplicons purified using MinElute          described (41). Samples taken step purification analyzed gel extraction kit (Qiagen). Protein lysates obtained follows: bacteria    SDS-PAGE, purified protein fractions pooled, aliquoted, harvested (4,800 &#226;&#171;&#187; g, 6 min, 4&#194;&#176;C) 500-ml, 24-h culture L.           stored &#226;&#171;&#186;85&#194;&#176;C. single band excised submitted mass spectrometry <br /> pneumophila. pellet washed cold 50 mM Tris-HCl, pH 7.5,           analysis. Purified OxyRLp protein used study considered wholly <br /> resuspended buffer consisting 50 mM Tris-HCl, pH 7.5; 1 mM EDTA, pH          active. <br /> 8.0; 50 mM NaCl; 10% glycerol 0.1 mM phenylmethylsulfonyl fluoride               EMSA. electrophoretic mobility shift assay (EMSA), amplicons spanning <br /> (PMSF); 1 mM dithiothreitol (DTT). Cells lysed using French press       promoter regions ahpC1, ahpC2, oxyR (PahpC1, PahpC2, PoxyR) subjected ultracentrifugation 100,000 &#226;&#171;&#187; g. Protein lysates ali-     generated PCR amplification using primers sets FC1PROM RC1PROM, <br /> quoted stored &#226;&#171;&#186;85&#194;&#176;C. use, protein lysates diluted buffer   FC2PROM RC2PROM, FLPOXYRPROM RLPOXYRPROM, <br /> consisting 100 mM Tris-HCl, pH 7.5, 10 mM MgCl2 treated 15          respectively. binding assays E. coli ahpCF, katG, oxyREc promot- <br /> min room temperature 100 mM <span id='am-17' about='obo:IMR_0200266' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200266" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00227" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Acetyl phosphate" datatype="xsd:string"></span>acetyl phosphate</span> (phosphorylate             ers, primer pairs FECAHPCPROM RECAHPCPROM, FECKATGPROM <br /> DNA-binding proteins). Binding reactions ACDC assay performed         RECKATGPROM, FECOXYRPROM RECOXYRPROM 25-&#226;?&#174;l volume consisted 0  DNA control), 2, 4 8 &#226;?&#174;g         used generate respective amplicons. Fifty nanograms DNA probes Ac-DNA, 1 &#226;?&#174;g protein lysate, 7.5 &#226;?&#174;l buffer D (20 mM HEPES, pH 7.8; 100          mixed various amounts His6-OxyRLp (0, 3.9, 7.8, 15.6, 31.3, 62.5, 125, 250, <br /> mM KCl; 10% glycerol; 1 mM DTT), 1 &#226;?&#174;g poly(dI-dC), 3.5 &#226;?&#174;l 30%           500 ng) 20-&#226;?&#174;l reaction mixture containing 20 mM Tris-HCl, pH 8.0; 5 <br /> acrylamide. Reactions incubated 15 min room temperature         mM MgCl; 20 mM KCl; 50 &#226;?&#174;g bovine serum albumin; 20% glycerol; 1 mM <br />  VOL. 190, 2008                                                                        OxyRLp REGULATION ahpC2D LEGIONELLA                                 3447 <br />  <br /> PMSF; 0 200 mM DTT (Promega, Madison, WI) oxidizing reducing           obtained.  glucose arabinose added final concentration conditions, respectively. 30 min incubation room temperature, pro-       0.02%, cultures incubated additional hour. cases, strains <br /> tein-DNA complexes separated electrophoresis approximately 1 h        challenged 30 min 100 &#226;?&#174;M H2O2. Fluorometric detection <br /> 125 V 5% nondenaturing polyacrylamide gel 0.5&#226;&#171;&#187; TBE buffer. Gels         samples taken LB-grown bacteria performed described previously soaked 10,000-fold diluted Sybr green nucleic acid stain (Invitrogen, La   (30). Values expressed RFU OD620 represent triplicate values <br /> Jolla, CA) washed twice distilled water, DNA visualized using       obtained independent experiments. <br /> Typhoon 9410 (GE Healthcare) blue light excitation 488 nm using 520-nm emission filter. Densitometry performed using ImageQuant <br /> version 5.2 (Molecular Dynamics, Sunnyvale, CA).                                                                      RESULTS <br />    Mapping PahpC2. EMSA performed described using 500 ng His6-OxyRLp 50 ng variously sized 5&#226;&#172;&#732;-end deletion DNA fragments               Acrylamide capture ahpC2D DNA-bound complexes. generated PCR using common reverse primer, RC2PROM, different              employed ACDC technique (Fig. 1) screen transcrip- <br /> forward primers: FC2PROM, FC2PROM179, FC2PROM156, FC2PROM132,                        tion factors bound promoter region ahpC2D. FC2PROM109, FC2PROM91, FC2PROM66. promoter fragments subcloned pBH6119 electroporated L. pneumophila Lp02                                                                                      principle method relies capture DNA-binding ahpC1::kan mutant (30) evaluate promoter activity using green fluo-       proteins L. pneumophila lysates interaction rescence protein (GFP) reporter assay.                                               ahpC2D promoter DNA sequences immobilized poly- <br />    DNase footprinting. DNase protection experiments performed using         acrylamide matrix covalent linkage 5&#226;&#172;&#732; Ac moiety. 226-bp DNA fragment PahpC2 generated PCRs using primer pair                Proteins bind Ac-DNA removed FC2PROM RC2PROM (containing EcoRI BamHI restriction sites, <br /> respectively). digested DNA (EcoRI strand BamHI                                                                                      standard nondenaturing electrophoresis. Following SDS- strand) treated calf intestinal phosphatase (New England        PAGE, silver staining, mass spectrometry analysis pro- <br /> BioLabs, Ipswich, MA), reactions purified using QIAquick gel ex-          teins captured wells native acrylamide gel, pro- <br /> traction kit (Qiagen). End-labeled DNA prepared using [&#226;?&#165;-32P]<span id='am-22' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span>dATP</span>            teins able bind target DNA sequences identified. <br /> Ready T4 polynucleotide kinase (GE Healthcare) described                                                                                      Using increasing concentrations Ac-labeled ahpC2D pro- <br /> manufacturer purified QIAquick <span id='am-43' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-44' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> removal kit (Qiagen). <br /> Binding reactions performed incubating approximately 20,000 cpm           moter DNA (Ac-PahpC2) ACDC assay (Fig. 1 inset), labeled probe 10 &#226;?&#174;g OxyRLp protein assay buffer consisting 20       distinct protein captured immobilized Ac-DNA, <br /> mM Tris-HCl, pH 8.0; 10 mM MgCl2; 100 mM KCl; 1 mM CaCl2; 10% glycerol;              bands seen absence Ac-DNA. <br /> 0.05 mg/ml bovine serum albumin; 4 &#226;?&#174;g/ml poly(dI-dC); H2O2 (0 50 mM)          Peptides identified tandem mass spectroscopy analysis cor- oxidizing 200 mM DTT reducing conditions. cases peroxi- <br /> dation abolished DNA binding, replicates treated 200 mM DTT                                                                                      responded 181 296 amino acids (61% sequence coverage) <br /> restore DNA binding activity. 20 min incubation, 0.5 1 U DNase        previously uncharacterized protein annotated hy- (Sigma) added. reaction continued 90 s 100 &#226;?&#174;l stop     drogen peroxide-inducible gene activator, OxyR, L. pneu- <br /> solution (200 mM EDTA, pH 8.0; 5 M ammonium acetate; 100 &#226;?&#174;g salmon            mophila Philadelphia-1. protein, designated OxyRLp, <br /> sperm DNA) added. DNA ethanol precipitated following phenol-chlo- <br />                                                                                      shared 45 50% identity 63 69% similarity  <br /> roform extraction. Dry pellets mixed formamide loading dye loaded 6% sequencing gel. DNA ladder performed cycle               characterized OxyR regulators Xanthomonas campestris, E. <br /> sequencing using &#226;?&#163;-35S-dATP <span id='am-63' about='rdfs:label' typeof='owl:Thing'>label</span> according SequiTherm Excel II            coli, Salmonella enterica serovar Typhimurium, Haemophilus <br /> DNA sequencing kit (Epicenter Technologies).                                         influenzae, C. burnetii BLAST analysis. <br />    Primer extension. Total RNA obtained exponentially grown L.                 identification OxyRLp putative regulator pneumophila automated Maxwell-16 protocol (Promega), primer ex- <br /> tension reactions performed follows. Briefly, 2 pmol [&#226;?&#165;-32P]<span id='am-32' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> <br />                                                                                      ahpC2D unexpected, OxyR mediates oxidative <br /> end-labeled RC2PROM primer added 25 &#226;?&#174;g total RNA allowed               stress induction alkyl hydroperoxide reductase catalase <br /> anneal dropping temperature 80&#194;&#176;C 41&#194;&#176;C period 30 min.        genes organisms (1, 26, 31, 36).  cDNA extension products generated 41&#194;&#176;C 1 h reaction mixture         organisms expressing ahpC2D operon, including close rel- <br /> containing 1 &#226;?&#174;l 200 U/&#226;?&#174;l SuperScript reverse transcriptase (Invitrogen)                                                                                      ative C. burnetii, oxyR usually located immediately upstream <br /> subsequently treated RNase (Sigma) eliminate residual RNA. Samples loaded 6% polyacrylamide <span id='am-74' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> sequencing gel              shares promoter regulatory sequences.  corresponding DNA sequence ladder obtained cycle sequencing using              oxyRLp gene located upstream ahpC2D, &#226;?&#163;-35S-dATP label appropriate primer accordance Sequi-           shares divergent promoter region gene annotated Therm Excel II DNA sequencing kit protocol.                                          major facilitator transporter, downstream gene    GFP reporter assay. GFP-transcriptional fusion PoxyR constructed <br />                                                                                      unknown function oriented opposite direction.  <br /> using primer pair FLPOXYRPROM RLPOXYRPROM cloned EcoRI BamHI sites pBH6119 (21). Plasmid constructs trans-          tempts knock oxyRLp allelic replacement drug <br /> formed E. coli DH5&#226;?&#163; strains, correct orientation determined          resistance cassette (kanamycin) methods previously used PCR using respective combination FLPOXYRPROM RGFP primers.                obtain ahpC1 ahpC2D mutants (30) proved unsuccessful. pBH6119 plasmid derivatives electroporated wild-type,           catalase supplementation media enabled recovery <br /> ahpC1, ahpC2D, rpoS, letA, himAB mutants L. pneumophila. Transformants <br /> capable growth thymidine verified respective plasmids                                                                                      oxyR mutants Pseudomonas aeruginosa (23), buffered <br /> PCR analysis using primer pairs FLPOXYRPROM RLPOXYRPROM                       charcoal yeast extract agar supplemented catalase <br /> FBLA RBLA vector controls. Fluorometric detection               (10,000 units);  oxyRLp mutants recovered. <br /> samples taken BYE-grown bacteria performed described previously          apparent essentiality oxyRLp unexpected, (30). Values expressed relative fluorescence units (RFU) OD620                                                                                      genomic organization uncomplicated overlapping reg- <br /> represent triplicate values obtained independent experiments. <br />    OxyRLp regulation ahpC2-gfp E. coli GS077. DNA fragments containing         ulatory elements downstream polarity effects. Possible func- <br /> gfp  promoter) PahpC2gfp excised EcoRI PstI plasmids        tional divergence considered, previous studies pBH6119 pC2gfp, respectively, subcloned pMMB206.               indicated genes typically regulated response oxidative <br /> resulting constructs, pgfp pahpC2gfp, respectively, electroporated     stress bacteria (ahpC1, ahpC2D, katA, katB, sodB, GS077 pBAD22 GS077 pBADLpoxyR. Ampicillin- chloramphen- <br /> icol-resistant transformants confirmed PCR analysis subjected                                                                                      sodC) similarly regulated L. pneumophila (4, 5, 30, <br /> GFP reporter assay (30). Overnight cultures, grown 37&#194;&#176;C LB broth         38, 39, 43).  genes exhibited growth <br /> anaerobic conditions, inoculated 1% fresh media OD620 0.4      stage-dependent expression (4, 5, 30, 43). <br />  3448     LEBLANC ET AL.                                                                                                                    J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Acrylamide capture proteins interacting PahpC2. ACDC assay performed adding 1 &#226;?&#174;g protein (cell extract) <br /> prepared BYE-grown L. pneumophila various amounts Ac-DNA (0, 2, 4, 8 &#226;?&#174;g). Nonspecific proteins removed nondena- <br /> turing electrophoresis 0.5&#226;&#171;&#187; TBE-5% polyacrylamide gel. Captured proteins (inset) excised wells, heated SDS sample buffer <br /> containing &#226;?&#164;-mercaptoethanol, following SDS-PAGE, visualized silver staining. dsDNA, double-stranded DNA; APS, ammonium persul- <br /> fate; TEMED, N,N,N&#226;&#172;&#732;,N&#226;&#172;&#732;&#226;&#172;&#732;-tetramethylethylenediamine. <br />  <br />  <br />  <br />    Complementation E. coli oxyR mutant. Functional                  bound state. differences binding ob- <br /> complementation assessed cloning (pBAD22) ex-               served oxidizing  added reducing agent) reducing <br /> pressing oxyRLp gene E. coli oxyR mutant strain GS077.          conditions (200 mM DTT)  Fig. 2B). Similar experiments <br /> Studies conducted anaerobic conditions              carried PahpC1 PoxyR;  DNA strain highly susceptible peroxide killing (compared        binding observed (data shown). EMSAs wild-type strain MG1655) loss OxyR regula-            performed amplicons containing promoter regions tory functions. inducing conditions (arabinose)            known members E. coli OxyR regulon (ahpCF, katG, <br /> pressing conditions (glucose), presence pBAD22 did          oxyR) oxidizing conditions. seen Fig. 2C, <br /> alter peroxide susceptibility (Table 3). Complementation GS077 pBADEcoxyR fully restored peroxide resistance wild-type E. coli MG1655 levels inducing conditions,                                                                         TABLE 3. Zones inhibition (mm) disk diffusion assay using repressing conditions. contrast, overexpression                              10 &#226;?&#174;l 3% H2O2 <br /> pBADLpoxyR partially rescued peroxide sensitivity E. <br />                                                                                                                  Zones inhibition (mm)  <br /> coli GS077, suggesting OxyRLp recognizes OxyR-regu- <br /> lated promoters. noted partial complementation                            Straina                    Glucose (%)              Arabinose (%)b <br /> OxyRLp DNA sequence differences OxyR                                                    0           0.2          0.02           0.2 <br /> binding motifs organisms. <br />    Interaction OxyRLp promoter region ahpC2.            K-12&#226;&#8364;&#160;                            21 &#226;&#171;&#190; 1       23 &#226;&#171;&#190; 1       21 &#226;&#171;&#190; 1         21 &#226;&#171;&#190; 1 <br />                                                                        K-12 pBAD22&#226;&#8364;&#161;                     21 &#226;&#171;&#190; 1       22 &#226;&#171;&#190; 1       22 &#226;&#171;&#190; 1         22 &#226;&#171;&#190; 1 confirm ACDC results extend functional studies,            GS077&#194;&#167;                           38 &#226;&#171;&#190; 2       40 &#226;&#171;&#190; 1       38 &#226;&#171;&#190; 1         37 &#226;&#171;&#190; 2 <br /> Ni2&#226;&#171;&#185;-nitrilotriacetic acid-purified OxyRLp (overexpression             GS077 pBAD22&#226;&#8364;&#161;                    39 &#226;&#171;&#190; 1       41 &#226;&#171;&#190; 1       38 &#226;&#171;&#190; 1         38 &#226;&#171;&#190; 1 pETLpoxyR) assessed EMSA DNA-binding                   GS077 pBADEcoxyR**               37 &#226;&#171;&#190; 2       40 &#226;&#171;&#190; 1      19* &#226;&#171;&#190; 1        18* &#226;&#171;&#190; 1 <br /> specificity promoter regions (PahpC2, PahpC1, PoxyR)        GS077 pBADLpoxyR**               39 &#226;&#171;&#190; 1       39 &#226;&#171;&#190; 1      29* &#226;&#171;&#190; 2        29* &#226;&#171;&#190; 2 genes. seen Fig. 2A, OxyRLp bound 226-bp                                                                                          Triplicate experiments performed anaerobic conditions Muel- <br /> DNA fragment containing PahpC2 (&#226;&#171;&#186;199 &#226;&#171;&#185;27 relative            ler-Hinton agar antibiotics (&#226;&#8364;&#160;) supplemented 100 &#226;?&#174;g/ml ampi- <br />                                                                        cillin (&#226;&#8364;&#161;), 40 &#226;?&#174;g/ml kanamycin (&#194;&#167;), 100 &#226;?&#174;g/ml ampicillin 40 &#226;?&#174;g/ml <br /> ahpC2 coding sequence). evident sample              kanamycin (**). <br /> taining 500 ng His6-OxyRLp, 84% DNA probe                 b <br />                                                                             *, significance (P &#226;&#172;? 0.01) determined relation strain GS077. <br />  VOL. 190, 2008                                                                OxyRLp REGULATION ahpC2D LEGIONELLA                         3449 <br />  <br />  <br />  <br />  <br />    FIG. 2. Interaction OxyRLp PahpC2.   Fifty ng 226-bp PahpC2 fragment spanning region &#226;&#171;&#186;199 &#226;&#171;&#185;27 (relative L. <br /> pneumophila ahpC2 initiation codon) incubated presence various amounts His6-OxyRLp. Lanes: 1, low-mass ladder (Invitrogen); <br /> 2, DNA protein; 3 10, DNA presence 3.9, 7.8, 15.6, 31.3, 62.5, 125, 250, 500 ng His6-OxyRLp, respectively. Arrows indicate <br /> bands representing DNA probe (P) mobility shift (S). Densitometry analysis reported percentage DNA probe (P [%]) shift (S <br /> [%]) relative total intensity lane. (B) Mobility shift assay performed presence 25 ng PahpC2 OxyRLp reducing <br /> (200 mM DTT, OxyRLp-red) oxidizing  DTT, OxyRLp-ox) conditions. (C) Mobility shift assay performed presence 50 ng E. <br /> coli PahpCF, PkatG, PoxyR presence (&#226;&#171;&#185;) absence (&#226;&#171;&#186;) 500 ng His6-OxyRLp nonreducing conditions. <br />  <br />  <br />  <br />  <br /> OxyRLp weakly bound promoters highest                  albeit lower expression level (data shown). parallel <br /> concentrations protein used (500 ng). EMSA results                    series experiments, mobility shift assays performed <br /> indicate partial complementation E. coli oxyR                 various PahpC2 fragments (Fig. 3C). Deletion nucleo- <br /> mutant oxyRLp combination OxyRLp                     tides &#226;&#171;&#186;199 &#226;&#171;&#186;109 effect OxyRLp binding (Fig. 3C, <br /> overexpression partial activation OxyR-regulated pro-                 lanes 2 5).  deletion nucleotides &#226;&#171;&#186;91 moters GS077.                                                             &#226;&#171;&#186;66  ablated transcription) did completely abol- <br />    Mapping OxyR binding site PahpC2. identify               ish DNA binding, suggesting OxyRLp binding motif(s) <br /> upstream DNA sequences necessary ahpC2 expression,                       remained. alignment DNA sequences &#226;&#171;&#186;105 PCR-generated 5&#226;&#172;&#732;-end deletion fragments PahpC2                       &#226;&#171;&#186;55 region PahpC2 genome sequences L. <br /> cloned upstream gfpmut3, encoding GFP. promoter                       pneumophila sequences OxyR-regulated <br /> fragments shared common 3&#226;&#172;&#732; end &#226;&#171;&#185;27 (relative                     genes E. coli species reveals conservation <br /> ahpC2 initiation codon) variable 5&#226;&#172;&#732; end ranging                   E. coli OxyR consensus (Fig. 3D). Taken  <br /> &#226;&#171;&#186;199 &#226;&#171;&#186;66 (Fig. 3A). ability PahpC2 fragments                    findings suggest OxyRLp-protected region  <br /> express GFP assessed ahpC1 mutant (Fig. 3B),                tains OxyR motif include promoter sequences. <br /> ahpC2 levels increased mutant (30). 5&#226;&#172;&#732;-end                      DNase footprinting primer extension.  <br /> deletions PahpC2 base pairs &#226;&#171;&#186;179 &#226;&#171;&#186;109 showed                      lineate specific DNA binding sequence PahpC2 recog- <br /> transcriptional activity similar previously studied           nized OxyRLp, untreated reduced (200 mM DTT) length fragment (&#226;&#171;&#186;199 &#226;&#171;&#185;27) (30).  PahpC2                     forms OxyR (OxyRLp-ox OxyRLp-red, respectively) deletions &#226;&#171;&#186;91 &#226;&#171;&#186;66 longer expressed GFP.                used footprinting assay. OxyRLp-red OxyRLp-ox showed <br /> noted similar results obtained wild-type strain,            similar DNA footprints (Fig. 4A B) confirmed  3450     LEBLANC ET AL.                                                                                                            J. BACTERIOL. <br />  <br />  <br />  <br />  <br />    FIG. 3. Promoter deletion analysis OxyRLp binding site PahpC2.   Schematic representation 5&#226;&#172;&#732;-end deletions PahpC2 <br /> cloned upstream promoter gfp. (B) Fluorescence detected ahpC1::kan mutants harboring plasmids PahpC2 promoter DNA-GFP <br /> constructs. (C) Mobility shift assay performed 5&#226;&#172;&#732;-end deletions PahpC2. lane contained 50 ng DNA fragments incubated (right panel) (left panel) 500 ng His6-OxyRLp. gels, lanes follows (symbols shown panel B): 1, low-mass ladder <br /> (Invitrogen); 2, PahpC2 fragment &#226;&#171;&#186;199 &#226;&#171;&#185;27 (f); 3, PahpC2 fragment &#226;&#171;&#186;179 &#226;&#171;&#185;27 (&#227;&#8364;&#171;); 4, PahpC2 fragment &#226;&#171;&#186;156 &#226;&#171;&#185;27 (&#197;&#8217;); 5, PahpC2 fragment <br /> &#226;&#171;&#186;132 &#226;&#171;&#185;27 (*); 6, PahpC2 fragment &#226;&#171;&#186;109 &#226;&#171;&#185;27 (E); 7, PahpC2 fragment &#226;&#171;&#186;91 &#226;&#171;&#185;27 (&#226;&#8212;?); 8, PahpC2 fragment &#226;&#171;&#186;66 &#226;&#171;&#185;27 (&#226;&#8364;&#353;); 9, DNA <br /> control (}). DNA fragments annotated relative ahpC2 initiation codon. (D) DNA alignments ahpC2 upstream region (&#226;&#171;&#186;115 &#226;&#171;&#186;55) L. pneumophila Philadelphia-1, L. pneumophila Paris, L. pneumophila Lens; ahpC Xanthomonas campestris (Xc) P. <br /> aeruginosa (Pa); ahpC, katG, oxyR, oxyS E. coli. consensus sequence E. coli provided, matching motifs underlined. <br />  <br />  <br />  <br /> protected region included &#226;&#171;&#186;91 &#226;&#171;&#186;66 regions. Primer                  purification process, typical OxyR <br /> extension analysis (Fig. 4C) revealed transcription               proteins (47, 48), oxidation H2O2 <br /> start site &#226;&#171;&#186;10 hexamer located               reveal typical redox-mediated DNA-binding dif- <br /> OxyRLp-protected region (Fig. 4D). location pro-                ferences. <br /> moter region transcription start site OxyRLp-                 test possibility, treated OxyRLp range protected region unusual, E. coli, reduced OxyR                peroxide concentrations (10 &#226;?&#174;M 50 mM) determined <br /> binds upstream promoter region OxyR-regulated                    1 mM completely abolished DNA binding (Fig. 4A, lane <br /> genes represses transcription overlapping &#226;&#171;&#186;35 hex-              4). shown, treatment OxyRLp 500 &#226;?&#174;M H2O2 <br /> amer (9, 31, 34, 36, 45, 46). case PahpC2, &#226;&#171;&#186;35               reduced DNA binding 50% did alter protected <br /> hexamer outside OxyRLp-protected region, suggesting              area relative untreated OxyRLp controls. distinguish different regulatory mechanism reported                    reversible irreversible oxidation OxyRLp, OxyREc (46, 47).  OxyRLp fully oxidized                 followed 1 mM H2O2 treatment 200 mM DTT treatment <br />  VOL. 190, 2008                                                               OxyRLp REGULATION ahpC2D LEGIONELLA                        3451 <br />  <br />  <br />  <br />  <br />    FIG. 4. OxyRLp binding PahpC2 ahpC2 transcript start site. Shaded bars indicate OxyRLp DNA footprints strand (end labeled BamHI)   strand (end labeled EcoRI) (B). OxyRLp binding PahpC2 occurred following conditions: lanes 1 6, protein control reactions; lane 2, oxidizing conditions  DTT); lane 3, 200 mM DTT; lanes 4, 1 mM H2O2; lane 5, treated 1 mM H2O2 200 mM DTT recover DNA-binding activity. panels C, asterisk denotes ahpC2 transcript start site indicated alignment primer extension product corresponding DNA sequence ladder. (D) Schematic PahpC2. Arrows indicate FC2PROM RC2PROM <br /> primers, open boxes denote OxyRLp sites determined DNase protection assays shown panels B, boldface indicates open reading <br /> frames ahpC2 lpg2351, ribosome-binding site (RBS) ahpC2 underlined. transcription start site indicated asterisk, &#226;&#171;&#186;10 sequence indicated bold. numbers parentheses depict numbering sites deletions shown Fig. 3. <br />  <br />  <br />  reduce disulfide bonds, seen Fig. 4A, lane 5,              polymerase complex, sought confirm OxyRLp subsequent reduction restore DNA binding activity.                repressor. introduced pahpC2gfp E. coli studies indicate OxyRLp footprint unchanged               GS077  oxyR mutant) noted constitutive expression treatments H2O2 concentrations                    GFP (Fig. 5). introduced pBADLpoxyR pBAD22 <br /> sufficient fully oxidize protein.                                    vector controls GS077 strains harboring pahpC2gfp   OxyRLp represses ahpC2-gfp expression. OxyRLp                   pgfp controls induced arabinose glu- <br /> footprint ahpC2 appears block binding RNA                 cose. seen Fig. 5, arabinose-dependent induction  3452     LEBLANC ET AL.                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Regulation ahpC2-gfp E. coli GS077 OxyRLp. <br /> Strains grown LB broth subjected repressing (glucose) inducing (arabinose) conditions. Treatment 100 &#226;?&#174;M H2O2 indicated appropriate. RFUs normalized 1.0 OD620 <br /> unit. Results means &#226;&#171;&#190; standard deviations indepen- <br /> dent experiments. <br />  <br />  <br />  <br /> OxyRLp pBADLpoxyR repressed fluorescence pahpC2gfp, consistent hypothesis binding OxyRLp promoter ahpC2-gfp blocked transcription.  change ahpC2-gfp expression fol- <br /> lowing challenge 100 &#226;?&#174;M H2O2 (oxidative stress), evidence OxyRLp redox-active regulator. <br />    Cell cycle expression oxyRLp. determine temporal <br /> expression oxyR vitro growth, GFP reporter assay          FIG. 6. Cell cycle-dependent oxyR expression L. pneumophila. <br />                                                                          GFP reporter assays (B) bacterial growth curves PoxyR performed. Figure 6 shows expression oxyRLp         performed BYE broth, fluorescence determined sam- <br /> growth cycle-dependent, increasing mid- postexpo-           ples taken 3 h, following growth wild type (}) nential phases decreasing slightly late-stationary          ahpC2D (&#226;&#8212;?), rpoS (&#226;&#8364;&#353;), letA (&#197;&#8217;), himAB (encoding integration host <br /> phase. Similar trends observed peroxide-sensitive              factor) (E) mutant strains. RFUs normalized 1.0 OD620 unit. <br /> ahpC1 ahpC2D mutants. attempt identify possible           Results means &#226;&#171;&#190; standard deviations independent <br />                                                                        experiments. <br /> regulators oxyR expression, poxyRgfp reporter construct transformed mutant strains lacking known regulators L. pneumophila virulence, rpoS, letA, himAB (encoding <br /> integration host factor). differences oxyR expression            genes, including oxyRLp, aligned growth stage, increas- observed mutants analyzed (Fig. 6), indi-           ing dramatically mid-exponential phase peaking cating postexponential expression OxyRLp            early stationary phase, consistent view oxidative <br /> activated RpoS LetA/S regulatory cascade.                    stress acute stationary-phase bacteria (4, 39).                                                                        demonstrating OxyRLp repressed transcription                           DISCUSSION                                   ahpC2D, partly resolved basis reciprocal <br />                                                                        gene expression noted ahpC1 ahpC2D. Studies <br />    laboratory characterized alkyl hydroperoxide            progress identify regulatory mechanisms associated <br /> reductases (AhpC1 AhpC2D) expressed differ-            postexponential activation ahpC1 oxyR. <br /> ent stages bacterial growth provide essential peroxidatic          OxyRLp appears functionally different functions L. pneumophila. Loss AhpC1 function results         OxyR proteins shown function redox- <br /> increased expression ahpC2D, suggesting compensatory              active repressors (47, 48). present lines response cellular oxidative stress (30). Using ACDC              evidence suggest OxyRLp longer function assay, identified OxyRLp regulatory factor controlling         global regulator antioxidant defenses:   oxyRLp expression <br /> expression ahpC2D operon. OxyRLp bound ahpC2D                growth stage dependent, (ii) OxyRLp function apparently <br /> promoter sequences did bind promoter regions            essential, (iii) DNA-binding properties OxyRLp  ahpC1 itself (autoregulation). Members OxyR               dox dependent, (iv) OxyRLp contains key amino acid substitu- <br /> family characterized, redox-active transcriptional ac-        tions likely ablate disulfide-bond formation conforma- <br /> tivators genes associated defense oxidative            tional changes required activation, (v) H2O2 <br /> stress. contrast, antioxidant defense genes             treatment did increase ahpC2-gfp levels L. pneumophila <br /> (sodB, sodC, katA, katB, ahpC1 ahpC2D) L. pneu-              E. coli oxyR mutant (GS077) expressing oxyRLp.  <br /> mophila appear activated response oxidative stress          findings exclude possibility OxyRLp <br /> (4, 5, 30, 39, 43).  expression          retains DNA sequence recognition OxyR-regulated <br />  VOL. 190, 2008                                                        OxyRLp REGULATION ahpC2D LEGIONELLA                  3453 <br />  <br />  <br />  <br />  <br />   FIG. 7. Clustal W alignment selected OxyR amino acid sequences. alignments L. pneumophila Philadelphia-1 (LpnOxyR <br /> [OxyRLp]), Xanthomonas campestris (XanOxyR [OxyRXc]), E. coli (EcoliOxyR [OxyREc]), C. burnetii (CbOxyR [OxyRCb]). DNA binding <br /> helix-turn-helix (HTH) shaded, relevant amino acid positions numbered bolded. <br />  <br />  <br /> promoters E. coli, indicated complementation            form disulfide bonds (redox potential &#226;&#171;&#186;185 mV) EMSA studies. studies indicate OxyRLp                protein purification (1, 50). organisms, Deino- <br /> locked activated DNA-binding conformation (29), inde-          coccus radiodurans, oxidation single (peroxidatic) cysteine <br /> pendent activation peroxide sufficient ac-      residue OxyR sufficient promote activation (10).  <br /> tivate antioxidant defense E. coli.                    regardless single cysteine oxidized    E. coli, OxyR binds DNA dimer dimers        disulfide bond formed, studies suggest reducing oxidizing conditions, promotes      redox-sensing domain OxyRLp longer signals conforma- <br /> expression target genes (47). Treatment E. coli OxyR       tional changes promote activation. millimolar amounts DTT shown promote                 Members LysR family  includes OxyR) use binding OxyR contact sites blocked &#226;&#171;&#186;35 sequences       ligand-binding conformational change mechanism mediate <br /> OxyR-regulated promoters, peroxide oxidation              DNA binding (11). Mutation-based studies identified key <br /> OxyR led conformational change protein          amino acids required function OxyR. example, <br /> tracted DNA footprint permitted strong induction          E. coli OxyR H125I H218D mutations resulted wild- promoters (44, 46, 47). case L. pneumophila,         type DNA binding loss redox-mediated regulation <br /> treatment OxyRLp micromolar concentrations H2O2           (11). amino acid substitutions occur naturally did alter footprint, suggested OxyRLp      OxyRLp proteins strains L. pneumophila fully oxidized protein purification DNA           genomes sequenced (125 218) (Fig. 7). binding independent oxidation state. millimolar         redox-active cysteine residues involved OxyR acti- <br /> concentrations H2O2 abolished DNA binding,             vation (Cys199 Cys208) conserved OxyRLp, result considered physiologically relevant biological   substantial amino acid sequence divergence activation <br /> activity recoverable treatments 200 mM DTT.          domain affect redox activity. example, key regard, functional studies shown redox-        arginine position 220 OxyREc (Fig. 7) predicted active thiols (C199 C208) OxyREc spontaneously oxidize        interact catalytic cysteines peroxide (28) changed  3454     LEBLANC ET AL.                                                                                                                J. BACTERIOL. <br />  <br />  <br /> threonine OxyRLp proteins. Interestingly, OxyR pro-           upregulation ahpC2D ahpC1 mutant postexpo- <br /> teins close relative C. burnetii Francisella tularensis,      nential activation ahpC1 (30). L. pneumophila, transition <br /> Arg220 replaced lysine, suggest                exponential phase stationary phase/cyst differentiation <br /> traditional function organisms. comparisons            controlled postexponential regulators RpoS, LetA, <br /> revealed amino acid sequence variation helix-turn-           HimAB (2, 13, 20, 22, 32, 33, 52).  oxyRLp-gfp <br /> helix region account differences DNA-             gene expression studies rpoS, letA, himAB mutants binding specificity noted OxyRLp OxyREc                  indistinguishable wild type. di- <br /> complementation experiments EMSAs. suggest                  rectly examined, carbon storage regulator CsrA, important loss redox-sensing function OxyRLp occurred           repressor stationary-phase genes exponential phase <br /> naturally specific amino acid substitutions           (13, 33), involved regulating oxyRLp, CsrA demonstrated empirical studies alter disulfide          repressed postexponential expression oxyRLp bond formation conformational changes required ac-              letA rpoS mutants. Future studies employ ACDC <br /> tivation.                                                               strategy capture putative regulatory factors associated    contrast bacteria, F. tularensis, C. burnetii,   control oxyRLp expression. Streptomyces species, oxyR located upstream                summary, shown OxyRLp represses ahpC2D divergently expressed ahpC2D  oxyRLp            expression oxyRLp induced postexponentially. gene located chromosome. conceivable          studies establish OxyRLp longer functions present oxyRLp acquired laterally,             oxidative response regulator L. pneumophila,  <br /> comitantly deletion loss ancestral upstream           sistent previous observations L. pneumophila mounts <br /> ortholog, necessary event associated evolu-          response oxidative stress bacteria tion dimorphic lifestyle.                                          resistant oxidative damage, phagocytic cells <br />    unlikely function OxyRLp            (30). propose OxyRLp adapted repress ahpC2D growth transition stationary               oxidative stress response regulator growth stage-specific <br /> phase cyst morphogenesis. downregulation              regulator genes mediating transition vegetative ahpC2D consistent metabolic changes shift          resilient cyst-like transmissible forms. <br /> <span id='am-27' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>-thioredoxin AhpC1 occur transition stationary phase, ahpC2D deletion mutants                                      ACKNOWLEDGMENTS viable infectious (30).  high-level ahpC2D <br />                                                                            thank Gisela Storz Michele Swanson providing bacterial <br /> expression, noted ahpC1 deletion mutant, sug-             strains plasmids. <br /> gested postexponential presence AhpC2D                 work supported CIHR grant MOP 14443 NIH grant <br /> toxic.  OxyRLp provide essential regulatory             AI066058 P.S.H. CIHR postdoctoral fellowship K.C.B. <br /> functions L. pneumophila, mutant obtained methods routinely used generate mutants                                                  REFERENCES <br /> (30). used pattern search option L. pneumophila            1. Aslund, F., M. Zheng, J. Beckwith, G. Storz. 1999. Regulation                                                                             OxyR transcription factor hydrogen peroxide cellular thiol-disul- <br /> Paris Lens genome website (http://genolist.pasteur.fr                   fide status. Proc. Natl. Acad. Sci. USA 96:6161&#226;&#8364;&#8220;6165. <br /> /LegioList/) screen additional genes, using motif patterns        2. Bachman, M. , M. S. Swanson. 2001. RpoS operates deduced comparisons E. coli L. pneu-                factors induce Legionella pneumophila virulence stationary phase. <br />                                                                             Mol. Microbiol. 40:1201&#226;&#8364;&#8220;1214. <br /> mophila OxyR binding motif. motifs relatively            3. Bandyopadhyay, P., B. Byrne, Y. Chan, M. S. Swanson, H. M. Steinman. <br /> degenerate, binding sites observed upstream approxi-                2003. Legionella pneumophila catalase-peroxidases required proper <br />                                                                             trafficking growth primary macrophages. Infect. Immun. 71:4526&#226;&#8364;&#8220; <br /> mately 50 genes, including dps homolog, recently              4535. prevent Fenton-mediated DNA damage se-                  4. Bandyopadhyay, P., H. M. Steinman. 2000. Catalase-peroxidases questering <span id='am-58' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span>iron</span> (37). included promoters genes                 Legionella pneumophila: cloning katA gene studies KatA func- <br />                                                                             tion. J. Bacteriol. 182:6679&#226;&#8364;&#8220;6686. <br /> coding efflux pumps  exclude redox-cycling compounds                5. Bandyopadhyay, P., H. M. Steinman. 1998. Legionella pneumophila organic solvents); metal <span id='am-6' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> transporters  regulate               catalase-peroxidases: cloning katB gene studies KatB function. <br /> rates Fenton chemistry); components respiratory                   J. Bacteriol. 180:5369&#226;&#8364;&#8220;5374. <br />                                                                          6. Bandyopadhyay, P., H. Xiao, H.  Coleman,  Price-Whelan, H. M. <br /> chain, cytochrome oxidases, DNA repair, modifica-                Steinman. 2004. Icm/Dot-independent entry Legionella pneumophila tion enzymes; known virulence factors, zinc met-                amoeba macrophage hosts. Infect. Immun. 72:4541&#226;&#8364;&#8220;4551. <br />                                                                          7. Berger, K. H., R. R. Isberg. 1993. distinct defects intracellular <br /> alloproteinase (shown inhibit oxidative burst); sub-             growth complemented single genetic locus Legionella pneumophila. <br /> strates Dot/Icm type IVB secretion (SidG, SdhB,               Mol. Microbiol. 7:7&#226;&#8364;&#8220;19. SdeA). Finally, OxyR binding motifs               8. Bruggemann, H.,  Hagman, M. Jules, O. Sismeiro, M.  Dillies, C. <br />                                                                             Gouyette, F. Kunst, M. Steinert, K. Heuner, J. Y. Coppee, C. Buchrieser. <br /> stream genes encoding numerous transcriptional regulators,               2006. Virulence strategies infecting phagocytes deduced vivo <br /> like Fur, FleQ, FleN, members LysR family,                 transcriptional program Legionella pneumophila. Cell. Microbiol. 8:1228&#226;&#8364;&#8220; <br /> suggesting cross talk regulatory             1240. <br />                                                                          9. Charoenlap, N., W. Eiamphungporn, N. Chauvatcharin, S. Utamapongchai, <br /> pathways. Interestingly, unlike E. coli, fur essential gene        P. Vattanaviboon, S. Mongkolsuk. 2005. OxyR mediated compensatory L. pneumophila (24). studies required                 expression ahpC katA significance ahpC protec- <br />                                                                             tion hydrogen peroxide Xanthomonas campestris. FEMS Microbiol. <br /> dissect additional regulatory functions associated essenti-            Lett. 249:73&#226;&#8364;&#8220;78. <br /> ality.                                                                  10. Chen, H., G. Xu, Y. Zhao, B. Tian, H. Lu, X. Yu,  Xu, N. Ying, S. Hu,    studies predict participation additional regulatory           Y. Hua. 2008. novel OxyR sensor regulator hydrogen peroxide <br />                                                                             stress cysteine residue Deinococcus radiodurans. PLoS elements controlling expression ahpC2D,                  13:e1602. <br /> pression ahpC2D OxyRLp does fully explain                 11. Choi, H., S. Kim, P. Mukhopadhyay, S. Cho, J. Woo, G. Storz, S. Ryu. <br />  VOL. 190, 2008                                                                             OxyRLp REGULATION ahpC2D LEGIONELLA                                   3455 <br />  <br />       2001. Structural basis redox switch OxyR transcription factor.        32. Lynch, D., N. Fieser, K. Gloggler, V. Forsbach-Birk, R. Marre. 2003. <br />       Cell 105:103&#226;&#8364;&#8220;113.                                                                       response regulator LetA regulates stationary-phase stress response <br /> 12.   Cirillo, J. D., S. Falkow, L. S. Tompkins. 1994. Growth Legionella               Legionella pneumophila required efficient infection Acan- <br />       pneumophila Acanthamoeba castellanii enhances invasion. Infect. Immun.               thamoeba castellanii. FEMS Microbiol. Lett. 219:241&#226;&#8364;&#8220;248. <br />       62:3254&#226;&#8364;&#8220;3261.                                                                       33. Molofsky,  B., M. S. Swanson. 2003. Legionella pneumophila CsrA 13.   Fettes, P. S., V. Forsbach-Birk, D. Lynch, R. Marre. 2001. Overexpress-             pivotal repressor transmission traits activator replication. Mol. <br />       sion Legionella pneumophila homologue E. coli regulator csrA                Microbiol. 50:445&#226;&#8364;&#8220;461. <br />       affects cell size, flagellation, pigmentation. Int. J. Med. Microbiol. 291:     34. Mongkolsuk, S., W. Whangsuk, P. Vattanaviboon, S. Loprasert, M. <br />       353&#226;&#8364;&#8220;360.                                                                                Fuangthong. 2000. Xanthomonas alkyl hydroperoxide reductase subunit C <br /> 14.   Fields, B. S., R. F. Benson, R. E. Besser. 2002. Legionella Legion-             (ahpC) mutant showed altered peroxide stress response complex <br />       naires&#226;&#8364;&#8482; disease: 25 years investigation. Clin. Microbiol. Rev. 15:506&#226;&#8364;&#8220;526.           regulation compensatory response peroxide detoxification enzymes. <br /> 15.   Fraser, D. W., T. R. Tsai, W. Orenstein, W. E. Parkin, H. J. Beecham, R. G.             J. Bacteriol. 182:6845&#226;&#8364;&#8220;6849. <br />       Sharrar, J. Harris, G. F. Mallison, S. M. Martin, J. E. McDade, C. C.               35. Nelson, C., S. Hendy, K. Reid, J. Cavanagh. 2002. Acrylamide capture <br />       Shepard, P. S. Brachman. 1977. Legionnaires&#226;&#8364;&#8482; disease: description             DNA-bound complexes: electrophoretic purification transcription fac- <br />       epidemic pneumonia. N. Engl. J. Med. 297:1189&#226;&#8364;&#8220;1197.                                  tors. BioTechniques 32:808&#226;&#8364;&#8220;810, 812, 814&#226;&#8364;&#8220;815. <br /> 16.   Gal-Mor, O., G. Segal. 2003. Identification CpxR positive regu-         36. Ochsner, U. , M. L. Vasil, E. Alsabbagh, K. Parvatiyar, D. J. Hassett. <br />       lator icm dot virulence genes Legionella pneumophila. J. Bacteriol.           2000. Role Pseudomonas aeruginosa oxyR-recG operon oxidative <br />       185:4908&#226;&#8364;&#8220;4919.                                                                          stress defense DNA repair: OxyR-dependent regulation katB-ankB, <br /> 17.   Gardun  &#203;&#339; o, R. , E. Garduno, M. Hiltz, P. S. Hoffman. 2002. Intracellular        ahpB, ahpC-ahpF. J. Bacteriol. 182:4533&#226;&#8364;&#8220;4544. <br />       growth Legionella pneumophila gives rise differentiated form dissim-        37. Park, M., S. T. Yun, S.-Y. Hwang, C.  Chun, T.  Ahn. 2006. dps <br />       ilar stationary-phase forms. Infect. Immun. 70:6273&#226;&#8364;&#8220;6283.                            gene symbiotic &#226;&#8364;&#339;Candidatus Legionella jeonii&#226;&#8364;? Amoeba proteus  <br /> 18.   Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu-               sponds hydrogen peroxide phagocytosis. J. Bacteriol. 188:7572&#226;&#8364;&#8220;7580. <br />       lation, modulation, high-level expression vectors containing             38. Rankin, S.,  Li, R. R. Isberg. 2002. Macrophage-induced genes       arabinose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130.                                   Legionella pneumophila: protection reactive intermediates solute <br /> 19.   Halablab, M. , M. Bazin, L. Richards, J. Pacy. 1990. Ultra-structure              imbalance intracellular growth. Infect. Immun. 70:3 <br /> </body></html>