<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> prediction essential genes integration network topology, cellular localization biological process <br /> information <br />  <br />  <br />  <br />                  Abstract <br />                  Background: identification essential genes important understanding                  minimal requirements cellular life practical purposes, drug design.                   experimental techniques essential genes discovery labor-intensive time-consuming. <br />                  Considering experimental constraints, computational approach capable accurately <br />                  predicting essential genes great value. present machine learning- <br />                  based computational approach relying network topological features, cellular localization                  biological process information prediction essential genes. <br />                  Results: constructed decision tree-based meta-classifier trained datasets                  individual grouped attributes-network topological features, cellular compartments                  biological processes generate various predictors essential genes. showed                  predictors better performances generated datasets integrated attributes. <br />                  Using predictor attributes, e., network topological features, cellular compartments                  biological processes, obtained best predictor essential genes used                  classify yeast genes unknown essentiality status. Finally, generated decision trees training <br />                  J48 algorithm datasets network topological features, cellular localization                  biological process information discover cellular rules essentiality. number <br />                  protein physical interactions, nuclear localization proteins number regulating <br />                  transcription factors important factors determining gene essentiality. <br />                  Conclusion: able demonstrate network topological features, cellular localization <br />                  biological process information reliable predictors essential genes.                   constructing decision trees based data, discover cellular rules governing <br />                  essentiality. <br />  <br />  <br />  <br />  <br /> Background                                                                      organism regardless presence remaining genes. <br /> Essential genes genes required growth                         functions encoded essential genes rich medium, e., medium containing nutrients                              crucial survival considered founda- <br /> required growth. deletion                          tion life itself [1,2]. identification essential genes <br /> genes sufficient confer lethal phenotype                          important understanding mini- <br />  <br />                                                                                                                                        Page 1 18 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                            http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> mal requirements cellular life, practical      bility essential genes. integration purposes. example, antibiotics target essen-     network topology, cellular localization biological <br /> tial cellular processes, essential gene products microbial   process information single predictor increased cells promising new targets drugs [3]. pre-    predictability essential genes comparison diction discovery essential genes          predictor containing network topological features. <br /> formed experimental procedures single gene            observed predictability essential <br /> knockouts [4], RNA interference [5] conditional             genes integration cellular localization biological <br /> knockouts [6], techniques require large invest-     process data single predictor comparable ment time resources feasi-       predictor containing network topological features. <br /> ble. Considering experimental constraints, computational approach capable accurately predict            Finally, addition study predictability essential <br /> essential genes great value.                        genes, tried define general rules governing <br />                                                                 essentiality S. cerevisiae analyzing decision trees gen- prediction essential genes, investigators      erated machine learning-based technique. Using net- <br /> implemented computational approaches              work topology, cellular localization biological based sequence features genes proteins       process information training attributes, discovered homology comparison [7,8]. accu-            essentiality depends number protein physi- <br /> mulation data derived experimental small-scale          cal interactions, nuclear localization proteins studies high-throughput techniques,           number regulating transcription factors. Taken possible construct networks gene proteins          findings integration interaction investigate topological        network analysis cellular localization bio- <br /> properties networks useful predict-       logical process information powerful tool ing essential genes. interaction networks         predicting biological characteristics genes, built <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> [9-12], studies relating        essentiality, discovering biological determinants <br /> essentiality topological properties networks      phenotypes. limited indicate topological properties predictive essentiality instead using pre-    Results Discussion <br /> dictors essential genes [9,13]. previously           Integrated network gene interactions S. cerevisiae <br /> shown feasibility using network topological fea-         calculation topological features <br /> tures predicting essential genes bacterium           obtaining network topological features used Escherichia coli [14]. chosen E. coli starting       training data predicting essential genes,  <br /> point evaluating prediction performance essen-       structed integrated network gene interactions (INGI) <br /> tial genes network topological features rea-      Saccharomyces cerevisiae simultaneously containing <br /> sons: completeness catalog E. coli essential      experimentally verified protein physical interactions, met- <br /> genes [15] vast interaction data availa-      abolic interactions transcriptional regulation interac- <br /> ble organism. present work, sought       tions (definitions type interaction detailed <br /> evaluate network topological features used       &quot;Methods&quot;). network comprised 5,667 genes predictors essential genes yeast S. cerevisiae     interacting 42,893 protein physical genes classified         interactions, 11,192 metabolic interactions 18,721 <br /> essential non-essential [4] copious            transcriptional regulation interactions. 5,667 genes amounts available interaction data organism.        network, 5,637 (99,5%) protein-coding genes <br />                                                                 (including transposable elements), 15 (0.26%) trans- purpose, constructed S. cerevisiae integrated     fer RNA-coding genes, 13 (0.23%) small nucleolar <br /> network gene interactions containing simultaneously          RNA-coding genes 2 (0.01%) RNA-coding genes <br /> protein physical, metabolic transcriptional regulation      unknown function. Regarding protein-coding genes, <br /> interactions used topological features net-     including transposable elements, network contains <br /> work learning attributes machine learning-based         96% total 5,884 protein-coding genes S. cerevisiae <br /> prediction  tested individual grouped net-        according current status yeast genome pro- <br /> work topological features predictors essential genes      vided Saccharomyces Genome Database (SGD) [16]. showed essential genes best predicted inte- <br /> grating topological features single predictor.         calculated 12 different topological features prediction performance topological fea-         gene INGI, including degree centralities tures shown acceptable, added set       type interaction, clustering coefficient, betweenness <br /> learning attributes data cellular localization bio-      centralities type interaction, closeness central- <br /> logical process genes order increase predicta-     ity identicalness. detailed description  <br />  <br />                                                                                                                     Page 2 18 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> topological features calculated            (ROC) Curve (AUC) values balanced data- Additional file 1 &quot;Methods&quot;.                    sets (Figure 1 Additional file 2), verified                                                                  prediction performances statistically different Comparison classification performance               evaluated nonparametric statistical method based balanced datasets                                                Mann-Whitney U-statistic [18]  details performance machine learning-based approaches             &quot;Methods&quot;). Based results, selected known affected imbalanced data [17].          balanced datasets perform following analyses. <br /> dataset containing yeast genes classified essential non-essential genes intended used training data         Prediction essential genes network topological classifier imbalanced dataset, used          features <br /> undersampling scheme generate balanced datasets           started analyzes assessing predictability original data  &quot;Methods&quot;). balanced            essential genes 12 network topological fea- <br /> dataset contains different subsets non-essential genes     tures (computed described &quot; Methods&quot;) 12 result sampling approach. different        network topological features integrated single predic- <br /> subsets non-essential genes,  statistically      tor. purpose, trained classifier bal- <br /> compared prediction performance balanced data-            anced dataset network topological features sets assessing predictability essential genes      training data dataset containing different features. trained classifier     network topological features training data  &quot;Meth- balanced dataset available training data         ods&quot; detailed information construction bal- <br /> (network topological features cellular localization          anced datasets). ROC plot shown Figure 2 indicates biological process information) evaluated            integration networks topological features prediction performance balanced dataset. Com-            single predictor outperforms predictability essential <br /> paring Area receiver operating characteristic      genes individual network topological features.  <br />   <br />  <br /> ROC <br /> Figurecurves <br />         1    AUC values classifiers trained balanced datasets available learning attributes <br /> ROC curves AUC values classifiers trained balanced datasets available learning <br /> attributes. Balanced datasets 1-10: datasets available learning attributes prepared undersampling scheme described &quot;Methods&quot;. <br />  <br />  <br />  <br /> comparing AUC values grouped individual net-               topological features. combination work topological features, verified AUC value             protein physical interactions-related features meta- grouped network topological features (AUC = 0.773)              bolic interactions-related features perform- <br /> statistically significantly higher (P &lt; 0.002) AUC               ance (AUC = 0.765, P = 0.302; Additional file 2 value individual network topological feature (Fig-             Figure 2) seen predictor containing grouped <br /> ure 2 Additional file 2).                                         network topological features (AUC = 0.773).                                                                        combination protein physical interactions-related fea- verified different combinations grouped net-            tures clustering coefficient, identicalness work topological features prediction perform-              betweenness closeness centralities pre- <br /> ances comparable grouped network                       diction performance (AUC = 0.763, P = 0.071; Addi- <br />  <br />                      1.0 <br />  <br />  <br />  <br />  <br />                      0.8 <br />  <br />  <br />  <br />  <br />                                                                                           net (AUC = 0.773) <br />                      0.6 <br /> True Positive Rate <br />  <br />  <br />  <br />  <br />                                                                                           only_ppi_metabolic_features (AUC = 0.765) <br />                                                                                           only_ppi_other_features (AUC = 0.763) <br />                                                                                           only_ppi_reg_features (AUC = 0.757) <br />                                                                                           ppi inbetppi (AUC = 0.748) <br />                                                                                           ppi (AUC = 0.747) <br />                      0.4                                                                  inbetppi (AUC = 0.701) <br />                                                                                           inbet (AUC = 0.628) <br />                                                                                           c (AUC = 0.615) <br />                                                                                           cent (AUC = 0.559) <br />                                                                                           regin (AUC = 0.547) <br />                                                                                           metin (AUC = 0.518) <br />                      0.2 <br />                                                                                           regout (AUC = 0.514) <br />                                                                                           metout (AUC = 0.513) <br />                                                                                           inbetmet (AUC = 0.512) <br />                                                                                           ident (AUC = 0.508) <br />                                                                                           inbetreg (AUC = 0.502) <br />                      0.0 <br />  <br />                            0.0   0.2                   0.4                    0.6                   0.8                     1.0 <br />                                                              False Positive Rate <br />  <br /> features <br /> ROC <br /> Figure curves <br />           2    AUC values classifiers trained balanced datasets individual grouped network topological <br /> ROC curves AUC values classifiers trained balanced datasets individual grouped net- <br /> work topological features. ROC curves AUC values classifiers trained balanced dataset 9  Figure 1) groups network topological features learning attributes follows: &quot;net&quot;: network topological features learning <br /> attributes; &quot;ppi&quot;, &quot;inbetppi&quot;, &quot;inbet&quot;, &quot;c&quot;, &quot;cent&quot;, &quot;regin&quot;, &quot;metin&quot;, &quot;regout&quot;. &quot;metout&quot;, &quot;inbetmet&quot;, &quot;ident&quot; &quot;inbetreg&quot;: datasets following network topological features learning attribute: number protein physical interactions (ppi), <br /> betweenness centrality protein physical interactions (inbetppi), betweenness centrality types interactions <br /> (inbet), clustering coefficient (c), closeness centrality (cent, number regulating transcription factor (regin), number reac- <br /> tants participating metabolic reaction catalyzed enzyme encoded gene (metin), number genes regulated transcription factor encoded gene (regout), number products generated metabolic reaction catalyzed enzyme encoded gene (metout), betweenness centrality metabolic interactions (inbetmet), number genes identical topological features (ident) betweenness centrality transcriptional regulation interactions (inbetreg).  _ppi_metabolic_features&quot; &quot;only_ppi_reg_features&quot;: datasets containing protein physical interactions-related features (ppi inbetppi)  respectively, metabolic (met, metin, metout inbetmet) transcriptional regulatory interactions-related <br /> features (reg, regin, regout inbetreg). &quot;only_ppi_other_features&quot;: dataset containing protein physical interactions-related fea- <br /> tures (ppi inbetppi) c, ident, cent inbet. &quot;only_ppi_inbetppi&quot;: dataset containing indicated network topologi- <br /> cal features learning attributes. details network topological features, Additional file 1. <br />  <br />  <br />  <br />                                                                                                                             Page 4 18 <br />                                                                                                       (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                            http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> tional file 2 Figure 2) observed grouped            grating cellular localization biological process data <br /> network topological features (AUC = 0.773).           network topological data, assessed individual <br /> smaller sets network topological features used     performance cellular component biolog- <br /> predict essential genes, making calculation     ical process, collective performance topological features dispensable.                               cellular components biological processes pre- <br />                                                                 dicting essential genes, order verify individual verify predictive power grouped network        feature grouped features related cellular localization <br /> topological features improved exclusion          biological process good predictors essential <br /> topological features marginal AUC values, e., AUC        genes. <br /> values ranging 0.500 0.600, compared pre- <br /> diction performance grouped network topological          Regarding cellular localization, trained classifier <br /> features (AUC = 0.773) combinations        balanced datasets cellular compartments features feature small set features    training data (cytoplasm, endoplasmic reticulum, mito- <br /> excluded  correspondent ROC curves Addi-         chondrion, nucleus localization) datasets <br /> tional file 3 pairwise comparison predictors         containing cellular compartments train- p-values AUC differences pair       ing data. observe ROC plot shown Figure <br /> predictors Additional file 2). discovered        3 best predictor essential genes prediction performance grouped network topologi-         integrated set cellular compartments. confirmed <br /> cal features improved removal individ-     statistical comparison AUC value inte- <br /> ual small sets topological features  Additional       grated set cellular compartments individ- <br /> files 2 3). expected, exclusion grouped fea-      ual cellular compartments: AUC value grouped <br /> tures related metabolic interactions grouped features     cellular compartments (AUC = 0.703) significantly (P &lt; <br /> related protein physical interactions diminishes (AUC        10-5) higher AUC values individual cellular <br /> = 0.764; P = 0.002 metabolic interaction-related        compartment (Figure 3 Additional file 2), features AUC = 0.749; P = 0.001 protein physical        AUC value characterizes set cellular compo- <br /> interaction-related features) prediction performance        nents fair predictors essential gene prediction. grouped network topological features (AUC =              regard biological processes, trained classifier 0.773).                                                         balanced datasets biological processes training <br />                                                                 data (cell cycle, metabolic process, signal transduction, individual network topological features,          transcription, transport process) datasets <br /> number protein physical interactions        containing biological processes training <br /> best predicts essential genes (AUC = 0.747). dis-    data. ROC curves biological processes (Figure 4) <br /> cussed &quot;Cellular rules essentiality&quot;, investiga-   behavior observed prediction tors shown number physical interactions        essential genes network topological features indicative essentiality [9,19,20]. knowledge,      cellular compartment: integration attributes compare number protein physical      single predictor increases predictability essential <br /> interactions network topological features.           genes comparison predictability individual <br /> Despite good performance number protein phys-         attributes. AUC value integrated set biologi- <br /> ical interactions predicting essential genes           cal processes (AUC = 0.667) statistically significantly (P individual network topological features, best         &lt; 0.001) higher AUC values individual biolog- <br /> predictors integrating groups topologi-      ical process (Figure 4 Additional file 2). AUC <br /> cal features number protein physical interac-       value 0.667,  set biological processes <br /> tions. indicates essentiality depends    considered poor predictor essential genes. network topological feature   gene location network important          moderate poor performances cellular localiza- <br /> determining essentiality.                                   tion biological processes predictors essential <br />                                                                 genes, respectively, suggest essentiality, dis- <br /> Prediction essential genes cellular localization      cussed &quot;Cellular rules gene essentiality&quot;, probably <br /> biological process data                                         result multiple factors, reinforcing prediction performance integrated           analyzing prediction performance network topo- <br /> network topological features single predictor       logical features.  decided evaluate pre- <br /> considered acceptable predicting essential genes,        diction performance integration cellular <br /> decided check addition information cellular     localization biological process information single <br /> localization biological process training data      predictor. trained classifier balanced data- <br /> increase predictability essential genes. inte-    sets cellular compartments biological proc- <br />  <br />  <br />  <br /> Figure <br /> ROC   curves <br />          3     AUC values classifiers trained balanced datasets individual grouped cellular compartments <br /> ROC curves AUC values classifiers trained balanced datasets individual grouped cellular <br /> compartments. ROC curves AUC values classifiers trained balanced dataset 9  Figure 1) cellu- <br /> lar compartments learning attributes.  compartments&quot; dataset cellular compartments learning attributes; <br /> &quot;nucleus&quot;, &quot;cytoplasm&quot;, &quot;mitochondrion&quot;,  compartment&quot; &quot;endoplasmic reticulum&quot; datasets respec- <br /> tive cellular compartment learning attribute. <br />  <br />  <br />  <br /> esses training data. Figure 5 indicates               features integrated single predictor cel- <br /> performance integration cellular localization           lular compartments biological processes inte- <br /> biological process data predicting essential genes          grated single predictor. worth mention better predictors. fact, AUC value pre-      choose predictor containing network <br /> dictor containing cellular localization biological        topological features perform following compari- <br /> processes data (AUC = 0.753) statistically higher (P &lt;         sons, sets containing protein physical interactions- <br /> 10-5) AUC values predictors  Additional         related features metabolic interactions-related fea- <br /> file 2).                                                          tures features  &quot;Prediction essential genes <br />                                                                   network topological features&quot; details) Prediction essential genes integrating network              used prediction performances comparable <br /> topological features, cellular localization biological        grouped network topological features. <br /> process information determining predictive power individual          evaluating integration data single predic- <br /> grouped cellular localization biological process data,        tor comparing predictor containing sought verify integration network topological         cellular localization biological process information <br /> features cellular localization biological process        predictor containing network topologi- <br /> data single predictor improve predictability        cal features, trained classifier balanced datasets <br /> essential genes.  sought compare          available data training data, cellular com- <br /> predictability essential genes network topological      partments biological processes training data  <br /> ROC <br /> Figure curves <br />          4    AUC values classifiers trained balanced datasets individual grouped biological processes <br /> ROC curves AUC values classifiers trained balanced datasets individual grouped biolog- <br /> ical processes. ROC curves AUC values classifiers trained balanced dataset 9  Figure 1) biolog- <br /> ical processes learning attributes.  processes&quot; dataset biological processes learning attributes; &quot;metabolic <br /> process&quot;,  process&quot;, &quot;cell cycle&quot;, &quot;transcription&quot; &quot;transport&quot; datasets respective biological process learning attribute. <br />  <br />  <br />  <br /> network topological features, cellular components                Regarding comparison predictive power inte- <br /> biological processes training data. expected,              grated topological network features integrated <br /> ROC curves Figure 6 indicate integration net-         cellular localization biological process data, work topological features cellular compartments             observed difference AUC value biological processes information single predictor               predictor containing cellular compartments bio- <br /> increases predictability essential genes compari-          logical processes information (AUC = 0.753) AUC <br /> son predictors containing network topological              value predictor containing network topological fea- <br /> features cellular compartments biological proc-               tures (AUC = 0.773) statistically significant (P = <br /> esses information.  comparing AUC value                0.269)  Additional file 2). Considering func- <br /> predictor containing network topological features            tion protein intimately linked cellular locali- cellular compartments biological processes infor-            zation [21] biological process mation predictor containing network                protein involved cellular localization topological features cellular compartments biolog-            protein acts predictable network topological fea- <br /> ical processes information, confirmed predictabil-           tures [10,22], surprising predictabilities ity essential genes integrated predictor (AUC =            essential genes predictor containing net- <br /> 0.808) statistically significantly (P &lt; 10-4) higher         work topological features predictor containing predictors (Figure 6 Additional file 2).          cellular localization biological process data simi- <br />                                                                      lar. <br />  <br />  <br /> Figure <br /> ROC curves <br />          5    AUC values integrated predictors cellular localization biological process information <br /> ROC curves AUC values integrated predictors cellular localization biological process <br /> information. ROC curves AUC values classifiers trained balanced dataset 9  Figure 1) biological proc- <br /> esses ( processes&quot;), cellular compartments ( compartments&quot;) biological processes cellular compartments <br /> ( processes compartments&quot;) learning attributes. <br />  <br />  <br /> Classification yeast genes known essential          predict gene essential, defined essentiality obtained list genes classified essential        score 0.654 cutoff value, e., genes essenti- <br /> non-essential used training classifier Giaever      ality score 0.654 considered essential. <br /> et al. [4]  &quot;Methods&quot;). Giaever et al. systemati-       cutoff value based optimal threshold, <br /> cally constructed nearly complete collection yeast          score value leads maximal accu- <br /> gene-deletion mutants covering 96% genes.           racy classification, calculated software StAR [24]  430 genes collection                 predictor containing features (network topolog- <br /> removed yeast genome comprehensive              ical, cellular component biological process; Figure <br /> reannotation process S. cerevisiae genome             6 Additional file 2). 514 genes formed 2006 [23]. addition, new genes anno-           essentiality status defined Giaever et al., 44 genes <br /> tated yeast genome result reannotation           predicted essential (Table 1). Analyzing process. order classify genes analyzed        genes, 9 genes previously dem- <br /> Giaever et al., used best classifier,     onstrated essential (YHR165C, YHR089C, containing network topological features, cellular       YHR052W, YCR042C, YDR320C  YHR169W, YKL138C- <br /> components biological processes information                YGL106W YHR099W) 14 genes <br /> training attributes. gene, predictor output     (YGR252W, YHR027C, YOL012C, YNL147W, YGL100W, <br /> probability classifying essential non-essential,    YNL096C, YOL148C, YFL007W, YOL145C, YBR111W  called, respectively, &quot;essentiality score&quot;          YNL055C, YHR216W, YBL071W YHR039C  &quot;non-essentiality score&quot;.                                        previously demonstrated non-essential  <br />  <br />  <br /> ROC <br /> Figure curves <br />           6 AUC values integrated predictors available data <br /> ROC curves AUC values integrated predictors available data. ROC curves AUC values clas- <br /> sifiers trained balanced dataset 9  Figure 1) network topological features, cellular compartments biological <br /> processes ( data&quot;), biological processes cellular compartments ( processes compartments&quot;) network <br /> topological features (&quot;network&quot;) learning attributes. <br />  <br />  investigators small-scale gene deletion             hypothesize classifier assigned high essentiality <br /> experiments functional characterization studies [25-36]        score genes following features:   (Table 1). non-essential genes, 10 genes                    encoded proteins interact 12 pro- <br /> (YGR252W, YHR027C, YOL012C, YNL147W, YNL096C,                     teins, (ii) regulated 4 transcription <br /> YOL148C, YOL145C, YBR111W  YNL055C                          factors (iii) encoded proteins located YHR039C  shown impair substantially            nucleus. characteristics accordance growth S. cerevisiae completely deleted          cellular rules essentiality discovered approach <br /> [33,36-40], 4 remaining non-essential genes           demonstrated section &quot;Cellular rules gene <br /> (YGL100W, YFL007W, YHR216W YBL071W                     essentiality&quot;: proteins interact 7 shown affect growth phenotype yeast            proteins located nucleus, genes encoding deleted [34,35,41,42]. roughly 1/          likely essential genes regulated 3 genes predicted essential          3 transcription factors tend non-essential. <br /> previously classified non-essential, complete dele- <br /> tions of shown severely reduce          44 genes predicted essential, 21 genes fitness organisms [33,36-40], suggesting          investigated essentiality date <br /> predictor, directly contradicted exper-        (Table 1). genes YER029C imental findings, nonetheless identify genes impor-           encoded protein (Yer029cp) binds 6 proteins tant cellular function. Regarding 4 non-essential          form heteroheptameric complex required genes deletion shown affect             biogenesis spliceosomal U1, U2, U4, U5 <br /> growth phenotype yeast (YFL007W YGL100W),               snRNPs [43]. spliceosomal snRNPs involved  <br />  <br />  <br />  <br /> Table 1: List 44 yeast genes predicted essential S. cerevisiae <br />  <br />  Rank      Gene                Essentiality Score            Essentiality Status         Deletion phenotype                Reference <br />  <br />  1         YHR165C             0.940                         essential                   lethality                         [32] <br />  2         YGR252W             0.939                         non-essential               defective growth                  [33] <br />  3         YHR089C             0.937                         essential                   lethality                         [25] <br />  4         YHR052W             0.065                         essential                   lethality                         [26] <br />  5         YER029C             0.930                         defined                 defined                       - <br />  6         YHR027C             0.930                         non-essential               defective growth                  [37] <br />  7         YHR099W             0.929                         essential                   lethality                         [27] <br />  8         YOL012C             0.925                         non-essential               defective growth                  [33] <br />  9         YHR169W             0.921                         essential                   lethality                         [28] <br />  10        YCR042C             0.920                         essential                   lethality                         [29] <br />  11        YDR320C           0.897                         essential                   lethality                         [30] <br />  12        YNL147W             0.885                         non-essential               defective growth                  [33] <br />  13        YGL100W             0.866                         non-essential               related growth             [41] <br />  14        YNL096C             0.865                         non-essential               defective growth                  [33] <br />  15        YOL148C             0.859                         non-essential               defective growth                  [38] <br />  16        YOR145C             0.856                         essential                   lethality                         [31] <br />  17        YFL007W             0.839                         non-essential               related growth             [42] <br />  18        YKL138C           0.837                         essential                   lethality                         [30] <br />  19        YOL145C             0.824                         non-essential               defective growth                  [39] <br />  20        YBR111W           0.822                         non-essential               defective growth                  [40] <br />  21        YLL022C             0.816                         defined                 defined                       - <br />  22        YNL209W             0.816                         defined                 defined                       - <br />  23        YGL106W             0.813                         defined                 defined                       - <br />  24        YPR080W             0.813                         defined                 defined                       - <br />  25        YER105C             0.794                         defined                 defined <br />  26        YNL055C             0.783                         non-essential               defective growth                  [33] <br />  27        YOL142W             0.781                         defined                 defined                       - <br />  28        YAL024C             0.770                         defined                 defined <br />  29        YHR216W             0.768                         non-essential               defective growth                  [34] <br />  30        YHL004W             0.743                         defined                 defined                       - <br />  31        YHR072W           0.741                         defined                 defined                       - <br />  32        YGL190C             0.738                         defined                 defined                       - <br />  33        YDR079C           0.731                         defined                 defined                       - <br />  34        YNL186W             0.731                         defined                 defined                       - <br />  35        YJR132W             0.716                         defined                 defined                       - <br />  36        YDR261W           0.713                         non-essential               defective growth                  [33] <br />  37        YHR119W             0.696                         defined                 defined                       - <br />  38        YBL071W           0.693                         non-essential               defective growth                  [35] <br />  39        YDR261W-B           0.682                         non-essential               defective growth                  [34] <br />  40        YHR039C           0.680                         non-essential               defective growth                  [36] <br />  41        YHR090C             0.680                         defined                 defined                       - <br />  42        YER026C             0.675                         defined                 defined                       - <br />  43        YHR056C             0.665                         defined                 defined                       - <br />  44        YCL019W             0.659                         defined                 defined                       - <br />  <br />  <br />  <br /> splicing nuclear pre-mRNAs [44], essential biologi-                strated non-essential genes 5 cal process cell viability,  interestingly, proteins         demonstrated essential genes.  4% forming heteroheptameric complex                          genes known essentiality status predicted Yer029cp demonstrated essential [4].                     non-essential actually essential genes (Additional file  presence gene ones predicted                4). Providing 38% (9 14; Table 2) genes essential reinforces fact predictor able           known essentiality status predicted essential identify genes important cellular function.               actually essential genes, predictor integrating                                                                          available features (network topological, cellular compo- <br /> Finally, regarding remaining 470 genes predicted                  nent biological process; Figure 6 Additional <br /> non-essential, verified 129 genes                  file 2) leads enrichment actual essential genes previously tested essentiality studies             set genes predicted essential. suggests Additional file 4).  124 demon- <br />  <br />                                                                                                                              Page 10 18 <br />                                                                                                         (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                             http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  predictor committed minimize false negative       vious studies demonstrated number rate avoiding loss essential genes.                  protein physical interactions indicative essentiality <br />                                                                  [9,19,20]. hypotheses connection <br /> Cellular rules gene essentiality                             gene essentiality number protein physical prediction capability, machine learning tech-         interactions proposed. Coulomb et al. [47] <br /> niques used knowledge acquisition order         suggested relationship network patterns datasets. machine learning algo-        feature gene essentiality partly biases rithms used knowledge acquisition        interaction data enriched small-scale experi- <br /> generate decision trees. Decision trees decision sup-        ments partial essential genes. port tools inferred training data use graph      hand, Zotenko et al. [48] recently hypothesized conditions possible consequences. struc-        connection gene essentiality ture decision tree consists root node representing     number protein physical interactions likely important condition discriminating classes,         involvement proteins encoded essential genes internal nodes representing additional conditions            subnetworks densely connected proteins shared <br /> class discrimination main condition, leaf          biological functions enriched proteins encoded <br /> nodes representing final classification.          essential genes. <br /> learn conditions classifying instances given <br /> class following path root node leaf       Following path root node leaf node <br /> node [45].                                                       right branch (Figure 7), follow- <br />                                                                  ing rule gene essentiality: proteins interact  order discover rules gene essential-    7 proteins (average number interactions <br /> ity S. cerevisiae, analyzed decision trees generated    ranging 6 12 decision trees) located <br /> training J48 algorithm, WEKA&apos;s implementation           nucleus, genes encoding likely essen- C4.5 algorithm [46]  details, &quot;Methods&quot;),       tial. rule observed 9 10 decision trees balanced datasets containing network topo-        128 instances leaf 8 10 decision trees logical features, cellular components biological proc-       64 instances leaf  Additional file 5). pro- <br /> esses training data  construction balanced             teins located cellular compartments datasets detailed &quot;Methods&quot;). decision trees gen-      nucleus, essentiality corresponding genes depends <br /> erated balanced datasets slightly differ-      conditions particular decision tree (Figure 7 <br /> ent undersampling scheme             Additional file 5). path root node leaf <br /> used balance original set classified genes-       nodes left branch (Figure 7) drove dis- <br /> balanced dataset contains different set 1,024 non-          cover rule gene essentiality: proteins interact <br /> essential genes, 1/8 total original         6 fewer proteins participate metabolic <br /> imbalanced dataset- generated detailed (64                process inside nucleus, genes encoding proteins <br /> instances leaf) simplified (128 instances        likely essential. rule observed 7 leaf) decision tree balanced dataset  &quot;Meth-        10 decision trees 128 64 instances leaf <br /> ods&quot; details) manually inspected         (Additional file 5). <br /> order discover general rules gene essentiality. <br />                                                                  According rules, ultimate condition gene 20 slightly different generated decision trees,      essentiality localization proteins nucleus, able devise general rules gene essentiality      suggesting cellular component S. cerevisiae. Figure 7 shows decision tree best     important essentiality. importance nucleus illustrates general rules gene essentiality  deci-   essentiality suggested Seringhaus et al. [7] <br /> sion trees available text format Additional file   shown nuclear localization strong- <br /> 5). observe Figure 7, root node deci-        est positive correlation essentiality cel- <br /> sion tree number protein physical interactions         lular components. relationship nucleus  generated decision trees exhibit feature; Addi-    essentiality explained fact roughly tional file 5);  attribute considered    nuclear proteins encoded essential genes <br /> important feature network topological features         essential biological processes cell viability cellular localization biological process informa-        place nucleus [49].  partici- <br /> tion gene essentiality. Accordingly, predictor       pation proteins nuclear-localized essential <br /> taining number protein physical interaction       processes, DNA replication, transcription training feature best predicts (AUC =            DNA repair, pivotal condition essentiality <br /> 0.747) essential genes individual features       rules defined paths left right observe Figure 2. concert pre-      branches decision tree. worth mention   <br />  <br />                                                                                                                     Page 11 18 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                                http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br />  <br /> Figure <br /> Decision7tree generated training J48 algorithm balanced dataset 8 available data <br /> Decision tree generated training J48 algorithm balanced dataset 8 available data. decision tree generated training J48 algorithm balanced dataset 8 available data  &quot;Methods&quot;). uppermost ellipse node root tree represents important condition discriminating essential genes non-essential genes. case, condition number protein physical interactions (ppi). remaining ellipses internal nodes represent additional conditions considering gene essential non-essential. left branch tree, conditions involvement metabolic process (met. proc.) nuclear localization (nucleus). right branch, conditions nuclear localization (nucleus) number regulating transcription factors (regin). rectangles leaf nodes represent final classification. Red green rectangles depict genes  certain conditions (repre- <br /> sented root node internal nodes), respectively predominantly classified essential (E) non-essential <br /> (N). round brackets inside rectangles, number slash indicates total number genes actually <br /> essential non-essential number slash indicates genes incorrectly predicted. <br />  <br />  <br /> result annotation method used  details           number instances leaf set 64  &quot;Methods&quot; &quot;Methods&quot;), nuclear-localized essential processes          details Additional file 5). finding corrobo- embedded biological process &quot;metabolic proc-             rated Yu et al. [50] genes regulated <br /> ess&quot;, conditions essentiality             &gt; 10 transcription factors likely essential <br /> nuclear localization number protein physical inter-          regulated 2-9 transcription factors, actions equal 6 rule defined path        genes likely essential left branch decision tree (Figure 7). rule        transcription factor. glimpse, fact defined path right branch, essen-           essential genes tend regulated transcription <br /> tiality apparently dependent involvement           factors contradictory expect proteins metabolic processes inside nucleus,             gene essentiality correlated high level tran- <br /> nuclear proteins encoded genes classified essential           scriptional regulation.  essential genes <br /> according rule actually involved                encode housekeeping proteins, e., proteins involved nuclear metabolic process. case,                housekeeping functions, rRNA metabolic process <br /> involvement nuclear metabolic processes                  transcription initiation [48]. housekeeping func- <br /> whelmed number protein physical interactions.             tions basic important functions                                                                     cell, genes encoding housekeeping proteins ubiqui- discovered additional interesting rule gene               tously expressed  consequently, tend regu- <br /> essentiality yeast: genes regulated 3 tran-         lated fewer transcription factors genes encoding <br /> scription factors tend non-essential (Figure 7).         non-housekeeping proteins.  phenomenon <br /> rule observed 6 10 decision trees 128             likely enrichment genes encoding house- <br /> instances leaf decision trees               keeping proteins set essential genes. <br />  <br />  <br />  <br />  <br />                                                                                                                         Page 12 18 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br />  <br /> Conclusion                                                       findings previously demon- identification essential genes largely        strated investigators [7,9,19,20,50],  <br /> experimental effort performed time-consuming           ing notice able obtain results genome knockout experiments. effort               simply inspecting decision tree generated shown <br /> accelerate pace discovery essential genes,          section &quot;Cellular rules gene essentiality&quot;.  deci- <br /> designed machine learning-based computational                  sion trees useful tools extracting knowledge approach relies network topological features, cel-       complex biological data. <br /> lular localization biological process information predicting essential genes evaluated yeast Sac-    confirming previous findings, exploration charomyces cerevisiae.                                           decision trees lead new discoveries.                                                                  exemplified additional analysis performed constructed integrated network gene           referee&apos; s suggestion regarding nuclear locali- <br /> interactions S. cerevisiae containing protein physical,      zation essential proteins. referee suggested metabolic transcriptional regulation interactions        analyze influence children terms computed 12 different network topological features            term &quot;nucleus&quot; nuclear localization-related gene <br /> described Additional file 1 &quot;Methods&quot;)          essentiality. purpose, generated decision tree <br /> individually collectively evaluated ability     training J48 algorithm balanced <br /> predict essential genes. showed predictors           datasets  &quot;Methods&quot; details) features plus <br /> containing 12 network topological features different      terms &quot;nucleolus&quot;, &quot;nucleoplasm&quot;, &quot;nuclear chro- <br /> combinations protein physical interactions-related fea-       mosome&quot; &quot;nuclear envelope&quot;  observed <br /> tures groups topological features training      Additional file 5, entirely new rule data reliable predictors (AUC = 0.763-0.773) essen-       devised generated decision tree: nucleolar <br /> tial genes S. cerevisiae, reinforcing fact   localization proteins important factor integrated network gene interactions useful         gene essentiality. did mention potential tool prediction essential genes.                      interesting rule gene essentiality section &quot;Cellu- <br />                                                                  lar rules gene essentiality&quot; rule se inter- performance predictors containing           esting deserve exhaustive analysis network topological features considered acceptable        reported future paper. predicting essential genes, decided check addition cellular localization biological process         Albeit good prediction performance ability information predictors increase pre-          discover cellular rules essentiality, approach suf- <br /> dictability essential genes. fact, verified    fers limitations.  depends existing <br /> performance predictor containing network          Gene Ontology annotation protein physical interac- <br /> ological features, cellular localization biological proc-    tion data likely enriched small-scale <br /> ess information training data better     experiments involving essential genes. Second,  <br /> predictors containing network topological features          struction integrated network gene interactions cellular localization biological process infor-      requires large experimental interaction data <br /> mation. Interestingly, showed prediction        currently available limited number performances predictor containing network            organisms. <br /> topological predictions predictor containing cellular localization biological process information          prediction essential genes newly similar. knowledge, time       sequenced organisms, example, impractical Gene Ontology terms related cellular localization         approach.  integration approach biological process shown useful predictors            computational-based methods gene annotation <br /> essential genes.                                                 (ii) computational-based methods construc- <br />                                                                  tion integrated networks predicted gene interactions addition prediction essential genes,      type interaction (protein physical, meta- <br /> devise cellular rules gene essentiality using       bolic transcriptional regulation interactions) network topological features, cellular localization          distinguished rise purely <br /> biological process information training data gener-       silico network topology, cellular localization bio- <br /> ation decision trees  details section &quot;Cellular        logical process information-based methodology pre- <br /> rules gene essentiality&quot;). discovered            diction essential genes. methodology number protein physical interactions, nuclear local-      totally independent experimental interaction data  <br /> ization number regulating transcription factors       accordingly, unbiased essential genes-driven experi- important factors determining gene essentiality.             ments. <br />  <br />  <br />                                                                                                                      Page 13 18 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                               http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  summary, despite limitations discussed           connected metabolites (<span id='am-2' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span>, <span id='am-18' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>, H+, <span id='am-29' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span>, NADP+, demonstrate integration network topo-          <span id='am-13' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span>, <span id='am-34' about='obo:IMR_0200102' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:id" content="IMR:0200102" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Orthophosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00009" datatype="xsd:string"></span>orthophosphate</span> <span id='am-8' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span>pyrophosphate</span>) logical features, cellular localization biological proc-     original metabolic model iND750. <br /> ess information capable predict essential genes. work, tested predictive performance          final integrated network gene interactions (INGI) <br /> integration S. cerevisiae, envisage   yeast result integration protein physical, <br /> useful predict essential genes organism     metabolic transcriptional regulation interactions <br /> purely computational-based prediction approach, sug-          datasets genes common datasets. gested  used.                                           performing integration, converted yeast gene <br />                                                                  names systematic names- provided Sac- <br /> Methods                                                          charomyces Genome Database (SGD) Nomenclature <br /> Generation set training features                       Conventions [23]- avoid creation false interac- <br /> Network topological features                                     tions gene ambiguity. Genes classified order compute network topological features used        dubious, e., genes unlikely encode expressed pro- training features predicting essential genes,    tein considered biologically significant SGD, <br /> constructed integrated network gene interactions        removed final INGI. <br /> S. cerevisiae based assumption genes, g1 g2, <br /> coding respectively proteins p1 p2, interacting      gene g yeast INGI, computed genes   p1 p2 interact physically (protein physical     network topological features listed Additional file 1. <br /> interaction), (ii) transcription factor p1 directly regu-    Briefly, degree centrality defined number links <br /> lates transcription gene g2, e., p1 binds pro-   node  case, gene). considered type moter region g2 (transcriptional regulation interaction),     interaction distinct measure degree described (iii) enzymes p1 p2 share metabolites, e.,        Additional file 1. Clustering coefficient (c) node  product generated reaction catalyzed enzyme p1        case, gene) quantifies close node used reactant reaction catalyzed enzyme p2            neighbors clique, e., nodes connected <br /> (metabolic interaction).                                         nodes. yeast INGI, c defined proportion <br />                                                                  links genes neighborhood g <br /> Yeast protein physical interactions data obtained           divided number links possibly exist Biological General Repository Interaction            Betweenness centrality reflects role <br /> Datasets (BioGRID) database, repository literature-         played node  case, gene) global net- <br /> curated protein physical genetic interactions [51].       work architecture  yeast INGI, defined downloaded database release 2.0.42 July 2008          fraction shortest paths gi gj passing <br /> removed entries related genetic interactions. Yeast       g. computed betweenness centrality based <br /> transcriptional regulation interactions obtained            shortest paths types interaction (inbet) Yeast Search Transcriptional Regulators         based shortest paths type interaction <br /> Consensus Tracking (YEASTRACT) database, curated               (inbetppi, inbetmet inbetreg). Closeness centrality (cent) <br /> repository regulatory associations transcrip-         measures close node  case, gene) tion factors target genes Saccharomyces cerevisiae        network  yeast INGI, defined [52]. using utility &quot;Generate Matrix Regulation&quot;       mean shortest path g genes YEASTRACT website, generated downloaded             reachable  Identicalness number genes <br /> regulation matrix containing documented transcrip-          identical network topological characteristics. tional regulation interactions determined direct exper-       network topological features, imental evidence.                                                betweenness centrality-related features, calculated                                                                  program written Mathematica&#194;&#174; 6.0 notebook. Yeast metabolic interactions extracted met-        betweenness centrality-related features calculated abolic model iND750 Saccharomyces cerevisiae [11]        Python package NetworkX [54]. <br /> code implemented Mathematica&#194;&#174; 6.0 (Wolfram <br /> Research, ). excluded metabolic interactions        Cellular localization biological process annotation yeast genes <br /> generated called &quot;currency metabolites&quot;, abun-         determined cellular component yeast <br /> dant molecular species present cell          gene product acts biological process time   unlikely impose           yeast gene involved using Saccharomyces <br /> straints dynamics metabolic reactions.          Genome Database (SGD) Gene Ontology   Slim feature currency metabolites, functionality       Mapper [55]. SGD Slim Mapper maps annota- network better represented             tions group genes general terms. <br /> [53]. considered currency metabolites          Slim sets available SGD, selected cellular <br />  <br />  <br />                                                                                                                         Page 14 18 <br />                                                                                                    (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                              http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> component biological process terms Super             tion essential genes. classifiers evalu- Slim set, collection high-level terms. cel-        ated, provided best performance lular localization annotation, genes annotated terms           ensemble decision tree algorithms using &quot;cytoplasm&quot;, &quot;endoplasmic reticulum&quot;,                 meta-classifier &quot;Vote&quot;, WEKA&apos;s implementation &quot;mitochondrion&quot; &quot;nucleus&quot; reannotated             voting algorithm combines output predictions of terms new term named  localiza-            classifier different rules [57]. combined tion&quot;. biological process annotation, genes annotated         classifiers average rule, output predic- terms &quot;cell cycle&quot;, &quot;metabolic process&quot;, &quot;sig-     tions computed individual classifiers class <br /> nal transduction&quot;, &quot;transcription&quot; &quot;transport&quot;           averaged average used decision [57]. <br /> reannotated terms new term named          classifiers composing model  (1) REPtree <br />  process&quot;.                                                  [56], (2) naive bayes tree [58], (3) random tree [56], (4) <br />                                                                   random forest [59], (5) J48, WEKA&apos;s implementation Classifier design, training evaluation                        C4.5 decision tree [46], minimum number 32 <br /> Construction datasets classifier training evaluation   instances leaf, (6) best decision tree mini- defined &quot;essential genes&quot; genes dele-           mum number 32 instances terminal nodes [60], <br /> tion leads inviable yeast organism cultured rich         (7) logistic model tree [61] (8) alternating decision <br /> glucose medium. obtained dataset containing            tree 25 boost iterations [62]. addition, applied <br /> classification yeast genes essential non-essential       bootstrap aggregating (bagging) approach [63] Giaever et al. [4]. downloading dataset,        classifier. Parameters values classifier pro- <br /> removed genes classified dubious SGD            vided Additional file 6. <br /> converted remaining genes systematic <br /> names provided SGD Nomenclature Conven-                 Classifier training evaluation <br /> tions [23].                                                       10 balanced datasets, trained clas- <br />                                                                   sifier half entries half used eval- dataset classified genes imbalanced dataset,     uate classifier performance, totaling 10 runs e., number non-essential genes larger       training evaluation. runs, generated number essential genes, known         receiver operating characteristic (ROC) curve calcu- <br /> data imbalance degrades performance machine                lated area ROC curve (AUC). ROC curve <br /> learning algorithms [17], built balanced datasets         plot true positive rate versus false positive rate original imbalanced dataset using undersam-             indicates probability true positive prediction <br /> pling scheme follows: (1)  split dataset       function probability false positive predic- <br /> classified genes subsets: &quot;essential genes set&quot;,         tion possible threshold values [64]. AUC widely <br /> containing 1,024 essential gene entries, &quot;non-essen-          used summary measure ROC curve equiva- <br /> tial genes set&quot;, containing 4,097 non-essential gene              lent probability randomly chosen negative <br /> entries; (2) second, selected entries essen-      example  case, non-essential gene) tial genes set (1,024 entries) randomly selected 1,024        smaller estimated probability belonging positive <br /> entries non-essential genes set; (3) cre-        class randomly chosen positive example  ated balanced dataset containing 2,048 selected           case, essential gene) [65]. <br /> entries random distribution essential gene non-essential gene entries. procedure repeated           used web server version StAR (Statistical <br /> 10 times order generate 10 different balanced data-         Analysis ROC curves) software [24] calculating sets containing different sets non-essential gene entries.     true false positive rates AUC values                                                                   generating ROC curves. statistical comparison compare predictability essential genes indi-         AUC values derived different datasets vidual training features different groups         performed StAR means nonparametric statisti- <br /> training features, generated, balanced data-          cal method based Mann-Whitney U-statistic sets, different subsets containing different combinations         comparing distributions values samples [18] training features detailed Additional file 2.            significance level (P) 0.01. <br />  <br /> Classifier design                                                 Determination rules gene essentiality used WEKA (Waikato Environment Knowledge                   determination rules gene essentiality  <br /> Analysis) software package, collection machine learn-        formed analyzing decision trees generated ing algorithms data mining tasks [56], designing,         training J48 algorithm balanced datasets containing <br /> training evaluating classifiers applied predic-        training data. used different values  <br />  <br />  <br />                                                                                                                      Page 15 18 <br />                                                                                                 (page number citation purposes) <br />  BMC Bioinformatics 2009, 10:290                                                                   http://www.biomedcentral.com/1471-2105/10/290 <br />  <br />  <br />  <br /> parameter &quot;number objects leaf&quot; J48 algorithm generating different types decision trees: 64                          Additional file 5 detailed trees 128 simplified trees                                 J48 decision trees. file contains 10 decision trees generated [56]. balanced dataset,  obtained                                training J48 algorithm 10 balanced datasets available <br />                                                                                       data learning attributes. Decision trees represented text format <br /> decision trees (detailed simplified) manually                                 (raw output generated WEKA). <br /> inspected 20 generated decision trees determin-                           Click file <br /> ing general rules gene essentiality. remaining                            [http://www.biomedcentral.com/content/supplementary/1471- <br /> parameters values producing decision trees J48                                 2105-10-290-S5.PDF] <br /> algorithm training provided Additional file 6 decision trees provided text format                             Additional file 6 <br /> Additional file 5.                                                                    Parameters used train meta-classifier J48. File containing <br />                                                                                       parameters values used train meta-classifier essential gene <br />                                                                                       prediction parameters values used train J48 algorithm Authors&apos; contributions                                                                generate decision trees discovery cellular rules essentiality. <br /> MLA obtained interaction data, constructed net-                               Click file <br /> work, designed analyzed classifier performance,                               [http://www.biomedcentral.com/content/supplementary/1471- <br /> pursued biological interpretation results                                  2105-10-290-S6.PDF] <br /> drafted manuscript. NL conceived, designed directed project implemented program cal- <br /> culation network topological features. authors read approved final manuscript. <br />  <br />  individual small sets network topological features. File containing <br />                                                                                 <br />  ROC curves classifiers trained datasets learning attributes            <br />  different sets network topological features set lacks     <br />  topological features small group 2-4 topological features.        <br />  Click file                                                                 <br />  [http://www.biomedcentral.com/content/supplementary/1471-                <br />  2105-10-290-S3.PDF]                                                             <br />  Additional file 4                                                                    <br />  List 470 yeast genes predicted non-essential. Tab-limited               <br />  text file containing 470 genes classified non-essential            <br />  essentiality scores, actual essentiality statuses  applicable,               <br />  Pubmed references showing essentiality statuses.                                <br />  Click file                                                                    <br />   <br /> </body></html>