<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Anal Biochem. 2009 November 1; 394(1): 39&#226;&#8364;&#8220;47. doi:10.1016/j.ab.2009.06.037. <br />  <br />  <br />  <br />                            homogenous resonance energy transfer assay                            phosphopantetheinyl transferase <br />  <br />                            Timothy L. Foley Michael D. Burkart* <br />                            Department Chemistry <span id='am-16' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Biochemistry; University California, San Diego; 9500 Gilman Drive; <br />                            La Jolla, CA 92093-0358. telephone (85)534-5673, fax (858) 882-1174 <br />  <br />                            Abstract <br />                                 Phosphopantetheinyl transferase plays essential role activating fatty acid, polyketide                                 nonribosomal <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> biosynthetic pathways, catalyzing covalent attachment 4&#226;&#8364;&#8482;- <br />                                 phosphopantetheinyl group conserved residue carrier protein domains. enzyme                                 validated essential gene primary metabolism presents target identification <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 antibiotics new mode action. report development homogenous resonance <br />                                 energy transfer assay utilizing fluorescent coenzyme derivatives surrogate peptide substrate <br />                                 serve identify inhibitors enzyme class. assay lays blueprint translation <br />                                 techniques transferase enzymes accept fluorescent substrate analogues. <br />  <br />  <br />                            Keywords <br />                                 phosphopantetheinyl transferase; assay development; high throughput screen; FRET assay; <br />                                 fluorescent analogue; Coenzyme  Acyl carrier protein; Peptidyl carrier protein; secondary <br />                                 metabolism <br />  <br />  <br />                            Introduction <br />                                               unifying characteristic biosynthesis fatty acid, non-ribosomal peptide,                                               polyketide compounds tethering nascent polymer small carrier protein domains <br />                                               synthases thioester linkage phosphopantetheinyl (4&#226;&#8364;&#8482;-PP) arm. 4&#226;&#8364;&#8482;- <br />                                               PP installed proteins post-translationally coenzyme (<span id='am-28' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>) 2 conserved <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               serine residue action phosphopantetheinyl transferase (PPTase) enzymes, converting <br />                                               apo- 1 holo- 4 form (Fig. 1). modification essential synthase <br />                                               activity, ablation PPTase gene precludes natural product production [1&#226;&#8364;&#8220;4],                                               case fatty acid biosynthesis, renders organism unviable [3&#226;&#8364;&#8220;5]. bacteria,                                               exist major classes enzymes PPTase superfamily: AcpS-type Sfp- <br />                                               type [1]. Grouping designations based primary sequence,                                               canonical representatives, AcpS Escherichia coli Sfp Bacillus subtilis, structurally <br />                                               distinct [6; 7]. <br />  <br />                                               demonstrated acpS essential E. coli gene [8; 9], validating                                               target inhibitor development potential treat multi-drug resistance.   <br />  <br />                             correspondence addressed: mburkart@ucsd.edu. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Foley Burkart                                                                                                 Page 2 <br />  <br />  <br />                                                number groups begun focused programs develop AcpS inhibitors [10&#226;&#8364;&#8220;14]                                                candidates recently discussed [10&#226;&#8364;&#8220;12]. addition fatty acids, number <br />                                                compounds produced 4&#226;&#8364;&#8482;-PP dependent pathways identified NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                virulence factors, disruption biosynthesis received attention new <br />                                                angle therapeutic development [15&#226;&#8364;&#8220;19]. intrigued central role                                                phosphopantetheinylation metabolic pathways, interested studying                                                potential effects PPTase inhibitors coordinate attenuation numerous <br />                                                aspects pathogenicity. <br />  <br />                                                AcpS-directed inhibitor development reported [10&#226;&#8364;&#8220;14], work generally <br />                                                omitted screening protocols. described method utilizes homogenous time-resolved <br />                                                fluorescence resonance energy transfer (HT-RF) means activity determination [11]. <br />                                                replication technique capabilities limitations imposed                                                instrumentation, desired use readily available affordable biochemical <br />                                                reagents. report development homogenous fluorescence resonance energy <br />                                                transfer (FRET) screen canonical PPTase representatives, AcpS Sfp.                                                method simple, requiring addition reagents reaction wells microtiter plate, <br />                                                validated process identify inhibitors enzymes. Furthermore,                                                details led successful development screen, serve blueprint <br />                                                assay design transferase enzymes accept reporter-modified substrate analogues. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Materials Methods <br />                               General <br />                                                7-dimethylamino-4-methyl-coumarin-3-maleimide (DACM) N,N,N&#226;&#8364;&#8482;,N&#226;&#8364;&#8482;- <br />                                                tetramethylrhodamine-5-maleimide (TAMRA) purchased Invitrogen Corporation <br />                                                (Carlsbad, CA). Coenzyme trilithium salt purchased EMD biochemicals (San <br />                                                Diego, CA). 3&#226;&#8364;&#8482;-phosphoadenosine-5&#226;&#8364;&#8482;-phosphate disodium salt (PAP), 4-(2-hydroxyethyl)-1- <br />                                                piperazine ethanesulfonic acid (HEPES), dimethylsulfoxide (DMSO, Hybri-Max grade),                                                fluorescein-isothiocyanate isomer (FITC) purchased Sigma (Saint Louis, MO). <br />                                                Concentrations fluorescently labeled reagents quantified UV-VIS spectroscopy <br />                                                Agilent 8354 diode array spectrophotometer (Agilent Technologies, Santa Clara, CA) <br />                                                using following extinction coefficients: DACM: 23,000 cm&#226;&#710;&#8217;1 M&#226;&#710;&#8217;1, FITC: 77,000 cm&#226;&#710;&#8217;1 <br />                                                M&#226;&#710;&#8217;1, TAMRA: 95,000 cm&#226;&#710;&#8217;1 M&#226;&#710;&#8217;1. <br />  <br />                               Enzyme expression &amp; Purification <br />                                                Sfp expressed purified described previously [2]. enzyme concentrated                                                40 mg/mL, diluted equal volume 75 % glycerol, stored 100 &#194;&#181;L aliqouts                                                &#226;&#710;&#8217;80&#194;&#176;C. routine use, stock (20 mg/mL, 765 &#194;&#181;M) diluted 1 mg/mL Sfp storage <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                buffer (50 mM NaHEPES, 120 mM NaCl, 33 % v/v glycerol) supplemented 0.1% w/v <br />                                                BSA stored &#226;&#710;&#8217;20&#194;&#176;C. conditions, noticeable reduction                                                enzymatic activity storage periods longer 6 months. <br />  <br />                                                E. coli AcpS expressed purified native protein pDPJ according published <br />                                                procedures [20]. protein concentration final preparation adjusted 10 mg/mL <br />                                                addition 2X storage buffer, equal volume glycerol added 3 portions, aliquots <br />                                                stored &#226;&#710;&#8217;80&#194;&#176;C. routine work, single tubes (200 &#194;&#181;L portions) stored &#226;&#710;&#8217;20&#194;&#176;C,                                                degradation enzymatic activity observed 1 year storage. <br />  <br />                               Synthesis assay components <br />                                                exploratory quantity (ca. 8 mg) fluorescein-5-isothiocyanate-modified YbbR peptide <br />                                                (FITC-YbbR) 8 (sequence: Fluorescein-Ahx-DSLEFIASKLA-OH) initially purchased <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                            Page 3 <br />  <br />  <br />                                                GL Biochem (Shanghai, China). final screen evaluation, peptide prepared <br />                                                0.2 mmol scale using automated solid phase peptide synthesizer (Applied Biosystems <br />                                                Pioneer) using standard 9-fluorenylmethyloxycarbonyl (FMOC) chemistry 2-(1H-7- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Azabenzotriazol-1-yl)-1,1,3,3-tetramethyl uronium hexafluorophosphate (HATU) activation <br />                                                (Fig. 3A) [21]. sequence appended N-terminal N-FMOC-&#206;&#181;-aminocaproic acid <br />                                                spacer, deprotected, coupled overnight FITC. Following cleavage solid <br />                                                support, product HPLC purified yield 84 mg FITC-YbbR 8 identity verified <br />                                                ESI-MS. <br />  <br />                                                Fluorescent reporter CoA (mCoA) analogues 11 12 prepared reaction reduced <br />                                                CoA trilithium salt 2 (1 mg/mL 50 mM NaH2PO4, pH 7.4) 1.1 equivalents maleimide <br />                                                bearing probe 9 10, respectively  dissolved 1 mg/mL methanol) (Fig. 3B).                                                reaction followed completion HPLC, determined disappearance CoA peak. <br />                                                Excess 9 10 removed extraction times ethyl acetate dichloromethane, <br />                                                respectively. resultant aqueous phase placed vacuum (&lt; 2 mm/Hg) 2h                                                remove residual organic solvent. purity mCoA analogues 11 12 verified                                                &gt;95% HPLC. <br />  <br />                               Kinetic evaluation fluorescent substrates <br />                                                Kinetic parameters FITC-YbbR 8 TAMRA-mCoA 12 determined HPLC. <br />                                                Reactions conducted final volume 50 &#194;&#181;L buffer containing 10 mM MgCl2, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                50 mM Na-HEPES, pH 7.6, 0.1 mg/mL BSA. Reactions initiated addition 0.5 <br />                                                &#194;&#181;M enzyme, allowed progress 15 minutes, quenched addition 50 &#194;&#181;L 50 mM <br />                                                sodium ethylenediaminetetraacetic acid, pH 8.0. reaction mixtures separated                                                reversed phase chromatography Agilent 1100 instrument fitted diode array <br />                                                detector (Agilent Technologies, Santa Clara, CA) using OD5 C18 column (250 &#195;&#8212; 4.6 mm, <br />                                                product number 9575, Burdick &amp; Jackson, Morristown, NJ). separation performed <br />                                                following conditions: Buffer  10 mM ammonium acetate, Buffer B: Acetonitrile, <br />                                                flow rate: 1.5 mL/min. injection, run initiated 2 minute isocratic flow 10% <br />                                                Buffer B, followed sample elution 10 minute linear gradient 10% 60% Buffer <br />                                                B. column regenerated 2 minute isocratic flow 100% Buffer B,                                                column equilibrated 10% Buffer B. <br />  <br />                               Fluorescence spectroscopy <br />                                                Steady state single sample fluorescence spectra recorded QuantaMaster 2000 <br />                                                spectrofluorometer (Photon Technologies International, Princeton, NJ), using excitation                                                emission slit widths 4 nm integration times 0.1 sec. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               FRET screen conditions <br />                                                PAP parent inhibitor plates dissolving compound dry DMSO                                                concentration 10 mM serial diluting (2-fold) DMSO. final protocol (Table <br />                                                1) follows: 2.5 &#194;&#181;L parent DMSO solutions  DMSO negative control)                                                transferred individual wells black polystyrene 96 plate (Costar # 3694) followed <br />                                                37.5 &#194;&#181;L 1.33 X Enzyme solution [16.62 nM Sfp 66.6 nM AcpS 1.33 X PPTase <br />                                                assay buffer (66 mM Na-HEPES, 13.3 mM MgCl2, 1.33 mg/mL BSA, pH 7.6). Reactions                                                initiated addition 10 &#194;&#181;L 5X reagent solution (50 &#194;&#181;M TAMRA-mCoA 12, 25 &#194;&#181;M <br />                                                FITC-YbbR 8, 10 mM NaH2PO4, pH 7.0). reaction monitored continuously (cycle <br />                                                time 2 minutes) 15 cycles Perkin Elmer HTS7000plus microtiter plate reader                                                excitation filter &#206;&#187; = 485 nm, emission filter &#206;&#187; = 535 nm. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 4 <br />  <br />  <br />                               Data analysis <br />                                                Kinetic data processed Microsoft Excel. Assay statistics evaluated according <br />                                                standard equations [22]. Plotting nonlinear regression performed GraphPad <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Prism version 5.00 (GraphPad  La Jolla, CA), IC50 curves fit  <br />                                                parameter dose response equation using ordinary  squares) setting. cases, error <br />                                                bars reported error values represent standard deviation. Molecular graphics images <br />                                                (Fig. 1 &amp; Fig. 2) renditions apo-actinorhodin acyl carrier protein (PDB identifier: <br />                                                2K0Y) generated using UCSF Chimera package [23]. <br />  <br />                               Results &amp; Discussion <br />                               Design high-throughput PPTase assay <br />                                                Traditionally, PPTase bioassays conducted low throughput manner,                                                monitoring radiolabel incorporation precipitated protein mass [3H]-CoA [4]                                                HPLC separation apo- holo-states carrier protein substrate [3]. systems <br />                                                utilize centrifugation chromatographic separations, respectively, readily <br />                                                amenable high throughput screening. designing high throughput screen PPTase <br />                                                activity, desired allow direct monitoring reaction progress                                                eliminate probability false-positive hits arrive inhibition coupled enzyme <br />                                                systems. accomplish  chose exploit synthetic aspect PPTase reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                conjunction fluorescent reporter-CoA (mCoA) technology developed previously                                                laboratory [24].  fluorophore-appended carrier protein domains/mimics 5 <br />                                                generate FRET signal conversion thiol-blocked crypto-form 7 modification <br />                                                mCoA 6 analogs PPTase (Fig. 2). <br />  <br />                               Fluorescent probe selection substrate synthesis <br />                                                chose make fluorescein-5-isothiocyanate (FITC) modification protein-based <br />                                                substrate, fluorophore possesses absorption emission centered visible light <br />                                                spectrum, allowing function FRET donor <span id='am-90' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-91' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span>, depending identity <br />                                                compounds chosen mCoA 6 preparation (Fig. 2). allow probe                                                modes FRET singly prepared pool reagent. turning select carrier protein <br />                                                domain function acceptor substrate, initially considered E. coli fatty acid <br />                                                synthase acyl carrier protein (ACP) candidate, previous reports note ACP contains <br />                                                single tyrosine residue present C-terminus &#206;&#177;-helix 3 modification residue <br />                                                dansyl-moiety does hinder function [25;26]. protocol                                                tyrosine-modifying techniques [27;28] provide low yields fluorescein-modified proteins <br />                                                insolubility FITC derivatives low pH reaction conditions (data                                                shown), purification labeled material insufficient needs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Subsequently, chose investigate use residue YbbR peptide (sequence <br />                                                H-DSKLEFIASKLA-OH) identified Yin et al. undergoes modification PPTases, <br />                                                serving ACP surrogate [29]. choice strengthened fact solid phase <br />                                                peptide synthesis (SPPS) allows access large quantities uniformly labeled material,                                                crucial requirement FRET applications, avoids potential batch batch <br />                                                variability. selecting placement <span id='am-84' about='rdfs:label' typeof='owl:Thing'>label</span>, noted YbbR isolated                                                collection N-terminal extensions consensus, suggesting site modification                                                abrogate activity.  chose attach FITC YbbR consensus 6- <br />                                                aminocaproic acid spacer unit sufficiently distance molecule central motif (Fig. <br />                                                3A) impart number freely rotatable bonds, ensuring random spatial orientation <br />                                                FRET-pair assembly (vide infra). <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                             Page 5 <br />  <br />  <br />                                                selecting complimentary probes containing modest spectral overlap FITC mCoA <br />                                                6 production, sought maleimide-bearing compounds amenable organic <br />                                                extraction reaction CoA (Fig. 3B), circumvent HPLC purification; NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                characteristic make procedure easily scaleable high screening volume <br />                                                application. mind, dimethylaminocoumarin (DACM) 9 tetramethylrhodamine <br />                                                (TAMRA) 10 chosen used prepare DACM-mCoA 11 TAMRA-mCoA 12 (Fig. <br />                                                3B) evaluated FRET donor (Fig. 4A) FRET acceptor (Fig. 4E), respectively. <br />  <br />                               HPLC analysis labeled substrates <br />                                                labeled substrates evaluated HPLC probe effects modification                                                substrates kinetic parameters displayed enzymes. Evaluation                                                FITC-YbbR 8 AcpS Sfp, saturating concentration CoA 2 returned Km values <br />                                                86 101 &#194;&#181;M Kcat values 4.4 12.1 &#194;&#181;M, respectively. agreement <br />                                                values previously determined unmodified substrate (Km values 200 123 <br />                                                &#194;&#181;M AcpS Sfp, respectively) [29;30]. Additionally, kinetic parameters TAMRA- <br />                                                mCoA 12 determined saturating concentration FITC-YbbR 8 gave Km values <br />                                                22 6 &#194;&#181;M AcpS Sfp, Kcat values 3.2 8.9, respectively.                                                 values agreement previously determined parameters [29;30]                                                demonstrate modifications substrates tolerated enzymes. <br />  <br />                               Determination F&#195;&#182;rster Radius <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                order assess viability donor acceptor substrates chosen, sought                                                determine theoretical Forster&#226;&#8364;&#8482;s radius, R0, pairs. value, distance                                                transfer efficiency 50%, related donor quantum yield &#208;&#164;D, relative orientation                                                donor acceptor transition dipoles &#206;&#186;, J(&#206;&#187;), overlap integral donor emission <br />                                                FD absorption acceptor &#206;&#181;A [31]: <br />  <br />  <br />  <br />                                                                                                                                             (1) <br />  <br />  <br />  <br />                                                &#206;&#186; value 2/3 assumed corresponds random orientation fluorophores <br />                                                space, given linked numerous freely rotatable bonds. &#206;&#183;, index                                                refraction medium, assumed 1.44 biological samples [31] NA, avagadro&#226;&#8364;&#8482;s <br />                                                number, constant, leaving R0 determined &#208;&#164;D, J(&#206;&#187;). Quantum yields                                                fluorophores previously determined [32], J(&#206;&#187;) expressed  <br />  <br />                                                                                                                                             (2) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Simple equations written FD(&#206;&#187;) &#206;&#181;A(&#206;&#187;),  J(&#206;&#187;) calculated                                                summation [33]: <br />  <br />  <br />  <br />                                                                                                                                             (3) <br />  <br />  <br />  <br />                                                Absorption emission spectra recorded conjugated substrates 8, 11                                                12 (Fig. 4B &amp; F) summation values wavelength according Eq. (3) use <br />                                                J(&#206;&#187;) values Eq. (1) gave F&#195;&#182;rster&#226;&#8364;&#8482;s radii 38 &#195;&#8230; 56 &#195;&#8230; DACM-FITC (Fig. <br />                                                4A) TAMRA-FITC pairs (Fig. 4E), respectively. <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 6 <br />  <br />  <br />                               Photophysical evaluation phosphopantetheinylated peptide efficiency transfer <br />                               determination <br />                                                theoretical F&#195;&#182;rster&#226;&#8364;&#8482;s radii, mathematically calculated maximum distance NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                fluorophores achieve crypto-FITC-YbbR product (exemplified 7, Fig. 2) <br />                                                summation average bond lengths, determined 42 &#195;&#8230; (radius, r, 21 &#195;&#8230;)                                                rigid linear extension occur. suggests high theoretical FRET efficiency, E, <br />                                                strictly related radius separating fluorophores, r,  <br />  <br />  <br />                                                                                                                                              (4) <br />  <br />  <br />  <br />                                                rapid decay E r exceeds R0. Using R0 values determined Eq. (1)                                                calculated r 21&#195;&#8230;; Eq. (4) gives theoretical E values 97 % 99 % DACM- <br />                                                FITC TAMRA-FITC pairs, respectively.  r exists distribution                                                number freely rotatable bonds, transfer efficiency experimentally <br />                                                determined measurement donor fluorescence absence (FD) presence (FDA)                                                acceptor fitting Eq. (5) experimental data: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                              (5) <br />  <br />  <br />  <br />                                                Reactions set mCoA analogues 11 12, Sfp, FITC-YbbR 8 followed <br />                                                completion HPLC. corresponding crypto-FITC-YbbR 7 peptides isolated <br />                                                semi-preparative HPLC. products used prepare equimolar mixtures                                                probes varying degrees modification. Fluorescent emission spectra mixtures <br />                                                recorded (Fig. 4C &amp; G). Analysis data (Fig. 4D &amp; H) Eq. (5) yields efficiency <br />                                                transfers 0.82 0.99 DACM-FITC (Fig. 4A) TAMRA-FITC (Fig. 4E) pairs, <br />                                                respectively. <br />  <br />                               Mode observation <br />                                                Data Fig. 4 (C &amp; G) informative regard mode monitoring                                                biochemical  FRET pairs offer flexibility assembly detected                                                quench donor emission  arrows) sensitization acceptor fluorescence <br />                                                shorter excitation wavelengths  arrows). Evaluation data systems shows                                                greater response quenching mode, 5-fold 10-fold reduction donor <br />                                                fluorescence conversion crypto-FITC-YbbR 7 DACM-FITC TAMRA- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                FITC pairs, respectively. Evaluation acceptor sensitized emission shorter excitation <br />                                                wavelengths showed increase 2-fold complete conversion, indicating                                                greater signal background ratio monitoring quench donor <br />                                                fluorescence. screen function optimally substrate-consumption assay. <br />  <br />                                                FRET systems, TAMRA-mCoA probe 12 possessed better FRET characteristics <br />                                                 e. attenuation donor fluorescence complete conversion) chosen                                                carried forward screen implementation. <br />  <br />                               Determination screen conditions <br />                                                arrived initial assay conditions based requirement end point need <br />                                                complete attenuation donor fluorescence; requiring TAMRA-mCoA 12 present                                                excess FITC-YbbR 8. mind, probed linearity detector response                                                respect FITC-YbbR 8 concentration. identify maximal concentration <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 7 <br />  <br />  <br />                                                obtained direct correlations relative fluorescence units                                                substrate concentration obscured inner filter effect, phenomenon                                                incedent light irradiated sample absorbed does sufficiently transition NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                target molecules excited state. plate reader (Perkin Elmer HTS7000)                                                delineation occur concentrations exceeding 10 &#194;&#181;M, chose hold FITC-YbbR <br />                                                8 concentration 5 &#194;&#181;M. <br />  <br />                                                sought determine effects increasing concentration TAMRA-mCoA <br />                                                12 FITC-YbbR 8 signal. purpose analysis acceptably high <br />                                                concentration TAMRA-mCoA 12 did impart effect FITC-YbbR 8 emission, <br />                                                displays absorption FITC-YbbR 8 excitation wavelengths. signal <br />                                                maintained concentrations 12.5 &#194;&#181;M arrived final concentration                                                screen TAMRA-mCoA 12 10 &#194;&#181;M. <br />  <br />                               Assay optimization: Requirement BSA enzyme stability <br />                                                screen developed initiating reactions 96 plates varied enzyme <br />                                                concentration. experiments monitored continuously using standard fluorescein <br />                                                optics, analysis progress curves revealed reaction rate did follow linearly <br />                                                respect enzyme concentration. identified complication time- <br />                                                dependent inactivation enzyme using Selwyn&#226;&#8364;&#8482;s method [34] (data shown). number <br />                                                reaction additives, including buffer salt composition, ionic strength, detergents NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                screened attempt stabilize enzyme, inclusion bovine serum <br />                                                albumin (BSA) concentration 1 mg/mL (15 &#194;&#181;M) stabilized enzyme eliminated <br />                                                complication. <br />  <br />                                                modification, stable reaction progress plots observed (Fig. 5A &amp; C)                                                displayed linear relationship rate enzyme concentration (Fig. 5B &amp; D).                                                data, determined optimal enzyme concentrations 50 12.5 nM AcpS                                                Sfp, respectively, based obtaining complete progress curve terminating approximately <br />                                                30% substrate consumption 30 minute time interval (vide infra). noteworthy                                                inclusion BSA traditionally considered unattractive screening campaign, <br />                                                carrier provides excellent source amine thiol moieties serve repress <br />                                                inhibition enzyme nonspecific electrophilic species present diverse compound <br />                                                libraries. <br />  <br />                               DMSO tolerance <br />                                                compound libraries stocked solutions DMSO, sought evaluate                                                tolerance screen organic solvent. Inclusion organic solvent reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                caused immediate 10% increase RFU signal substrate mixture did                                                deteriorate observed rate reaching concentration greater 10% total volume (data <br />                                                shown). routine screening conditions, maintained final DMSO concentration <br />                                                5% v/v. <br />  <br />                               Progress curve analysis signal statistics <br />                                                final assay conditions hand, screening protocol developed presented                                                Table 1. Following protocol, recorded progress curves analysed data <br />                                                statistically determine appropriate time analysis. known inhibitors PPTases <br />                                                commercially available, enzyme-free reaction used fully inhibited <br />                                                (negative) control. Analysis data (presented Fig. 6A &amp; B) demonstrated assay <br />                                                performance relied heavily substrate consumption allowed occur, <br />                                                parameter values better consumption (Table 2).  <br />                                                excessive substrate consumption lead erroneous determination inhibition <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                             Page 8 <br />  <br />  <br />                                                characteristics [35], progression ~20% (Table 2, boxed values) chosen balance <br />                                                conditions minimum substrate consumed assay statistics reached <br />                                                acceptable values, including  values 0.7, signal noise 20 (Table 2). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Subsequently, experiment repeated separate days assess robustness                                                method, data presented Table 3.  factor [36] used evaluate <br />                                                suitability screening method high throughput implementation, values greater <br />                                                0.5 considered satisfactory. assay performed  gave  values 0.72 <br />                                                &#194;&#177; 0.01 0.75 &#194;&#177; 0.02 Sfp AcpS, respectively. <br />  <br />                               Screen validation inhibitor identification <br />                                                demonstrate ability screen identify inhibitors PPTases, evaluated                                                effects product inhibition 3&#226;&#8364;&#8482;-phosphoadenosine-5&#226;&#8364;&#8482;-phosphate (PAP) 3 (Fig. 1),                                                <span id='am-33' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> product released enzyme, utilized McAllister et al. studies                                                reaction mechanism AcpS [13]. PAP serial diluted (2-fold) concentration                                                10 mM DMSO stocked polypropylene plates, evaluated assay                                                enzymes separate days (Fig. 7), demonstrate reproducibility screen. <br />                                                method proved yield consistent inhibition curves, IC50 values 12.3 &#194;&#177; 1.1 &#194;&#181;M                                                92 &#194;&#177; 6 &#194;&#181;M Sfp AcpS, respectively; Hill coefficients analyses &#226;&#710;&#8217;1.06 &#194;&#177; <br />                                                0.09 Sfp &#226;&#710;&#8217;0.98 &#194;&#177; 0.11 AcpS. Taken  data support suitability                                                method determine inhibitory characteristics compounds PPTase enzymes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Conclusions <br />                                                summary, developed homogenous screen PPTase requires minimum <br />                                                number liquid handling steps, meets criteria set forth NIH <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> Genomics <br />                                                Center acceptable automation [37]. Importantly, method utilizes simple reagents <br />                                                prepared traditional techniques [24;38], pure chemical entities                                                require interbatch standardization, luxury afforded complex biochemical <br />                                                reagents. anticipated method used evaluate crossreactivity                                                currently known AcpS inhibitors, identify new inhibitor architectures Sfp AcpS, <br />                                                work currently progress. <br />  <br />                               Abbreviations used <br />                                                4&#226;&#8364;&#8482;-PP, 4&#226;&#8364;&#8482;-phosphopantetheine <br />                                                ACP, acyl carrier protein <br />                                                CoA, coenzyme                                                mCoA, modified-coenzyme NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                PPTase, phosphopantetheinyl transferase <br />                                                FRET, fluorescence resonance energy transfer <br />                                                DACM, 7-dimethylamino-4-methyl-coumarin-3-maleimide <br />                                                TAMRA, tetrametylrhodamine-5-maleimide <br />                                                PAP, 3&#226;&#8364;&#8482;-phosphoadeosine-5&#226;&#8364;&#8482;-phosphate <br />                                                UV-VIS, ultraviolet-visible <br />                                                FITC, fluorescein-5-isothiocyanate <br />                                                HEPES, 4-(2-hydroxyethyl)-1-piperazine ethanesulfonic acid <br />                                                BSA, bovine serum albumen <br />                                                FMOC, 9-Fluorenylmethyloxycarbonyl <br />                                                HATU, 2-(1H-7-Azabenzotriazol-1-yl)-1,1,3,3-tetramethyl uronium hexafluorophosphate <br />                                                DMSO, dimethyl sulfoxide <br />                                                RFU, relative fluorescence unit <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                          Page 9 <br />  <br />  <br />                               Acknowledgements <br />                                                work funded NIH R01GM075797 1R03MH083266. thank Christopher T. Walsh (Harvard <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Medical School) plasmids containing Sfp AcpS expression systems, Elizabeth  Komives guidance <br />                                                assistance peptide synthesis, Adam Yasgar Anton Simeonov (NIH Chemical Genomics Center)                                                helpful discussions. <br />  <br />  <br />                               References <br />                                                1. Lambalot RH, Gehring  Flugel RS, Zuber P, LaCelle M, Marahiel MA, Reid R, Khosla C, Walsh <br />                                                   CT. new enzyme superfamily - phosphopantetheinyl transferases. Chem Biol 1996;3:923&#226;&#8364;&#8220;936. <br />                                                   [PubMed: 8939709] <br />                                                2. Quadri LE, Weinreb PH, Lei M, Nakano MM, Zuber P, Walsh CT. Characterization Sfp Bacillus <br />                                                   subtilis phosphopantetheinyl transferase peptidyl carrier protein domains peptide synthetases. <br />                                                   Biochemistry 1998;37:1585&#226;&#8364;&#8220;1595. [PubMed: 9484229] <br />                                                3. Barekzi N, Joshi S, Irwin S, Ontl T, Schweizer HP. Genetic characterization encoding                                                   multifunctional phosphopantetheinyl transferase Pseudomonas aeruginosa. Microbiology <br />                                                   2004;150:795&#226;&#8364;&#8220;803. [PubMed: 15073290] <br />                                                4. Finking R, Solsbacher J, Konz D, Schobert M, Schafer  Jahn D, Marahiel MA. Characterization                                                   new type phosphopantetheinyl transferase fatty acid siderophore synthesis Pseudomonas <br />                                                   aeruginosa. J Biol Chem 2002;277:50293&#226;&#8364;&#8220;50302. [PubMed: 12381736] <br />                                                5. Lambalot RH, Walsh CT. Cloning, overproduction, characterization Escherichia coli holo- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   acyl carrier protein synthase. J Biol Chem 1995;270:24658&#226;&#8364;&#8220;24661. [PubMed: 7559576] <br />                                                6. Parris KD, Lin L, Tam  Mathew R, Hixon J, Stahl M, Fritz CC, Seehra J, Somers WS. Crystal <br />                                                   structures substrate binding Bacillus subtilis holo-(acyl carrier protein) synthase reveal novel <br />                                                   trimeric arrangement molecules resulting active sites. Structure Folding &amp; Design <br />                                                   2000;8:883&#226;&#8364;&#8220;895. [PubMed: 10997907] <br />                                                7. Reuter K, Mofid MR, Marahiel MA, Ficner R. Crystal structure surfactin synthetase-activating <br />                                                   enzyme Sfp: prototype 4 &#226;&#8364;&#732;-phosphopantetheinyl transferase superfamily. Embo Journal <br />                                                   1999;18:6823&#226;&#8364;&#8220;6831. [PubMed: 10581256] <br />                                                8. Lam HM, Tancula E, Dempsey WB, Winkler  Suppression Insertions Complex Pdxj <br />                                                   Operon Escherichia-Coli K-12 <span id='am-10' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>Ion</span> Mutations. Journal Bacteriology <br />                                                   1992;174:1554&#226;&#8364;&#8220;1567. [PubMed: 1537800] <br />                                                9. Takiff  Baker T, Copeland T, Chen SM, Court DL. Locating Essential Escherichia-Coli Genes                                                   Using Mini-Tn10 Transposons - Pdxj Operon. Journal Bacteriology 1992;174:1544&#226;&#8364;&#8220;1553. <br />                                                   [PubMed: 1537799] <br />                                                10. Chu M, Mierzwa R, Xu L, Yang SW, L, Patel M, Stafford J, Macinga D, Black T, Chan TM, <br />                                                     Gullo V. Structure elucidation Sch 538415 novel acyl carrier protein synthase inhibitor                                                     microorganism. Bioorg Med Chem Lett 2003;13:3827&#226;&#8364;&#8220;3829. [PubMed: 14552789] <br />                                                11. Gilbert  Kirisits M, Toy P, Nunn DS, Failli  Dushin  Novikova E, Petersen PJ, Joseph- <br />                                                     McCarthy D, McFadyen  Fritz CC. Anthranilate 4H&#226;&#8364;&#8220;oxazol-5-ones: novel small molecule <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     antibacterial acyl carrier protein synthase (AcpS) inhibitors. Bioorg Med Chem Lett 2004;14:37&#226;&#8364;&#8220;41. <br />                                                     [PubMed: 14684293] <br />                                                12. Joseph-McCarthy D, Parris K, Huang  Failli  Quagliato D, Dushin  Novikova E, Severina <br />                                                     E, Tuckman M, Petersen PJ, Dean C, Fritz CC, Meshulam T, DeCenzo M, Dick L, McFadyen IJ, <br />                                                     Somers WS, Lovering F, Gilbert  Use structure-based drug design approaches obtain novel <br />                                                     <span id='am-85' about='obo:IMR_0200312' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-86' property="rdfs:label" content="Anthranilic acid" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0200312" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasDbXref" content="KEGG:C00108" datatype="xsd:string"></span>anthranilic acid</span> acyl carrier protein synthase inhibitors. J Med Chem 2005;48:7960&#226;&#8364;&#8220;7969. [PubMed: <br />                                                     16335920] <br />                                                13. McAllister KA, Peery RB, Meier TI, Fischl  Zhao G. Biochemical molecular analyses                                                     Streptococcus pneumoniae acyl carrier protein synthase enzyme essential fatty acid <br />                                                     biosynthesis. J Biol Chem 2000;275:30864&#226;&#8364;&#8220;30872. [PubMed: 10903317] <br />                                                14. Payne DJ, Gwynn MN, Holmes DJ, Pompliano DL. Drugs bad bugs: confronting challenges <br />                                                     antibacterial discovery. Nat Rev Drug Discov 2007;6:29&#226;&#8364;&#8220;40. [PubMed: 17159923] <br />                                                15. Neres J, Labello NP, Somu RV, Boshoff HI, Wilson DJ, Vannada J, Chen L, Barry CE, Bennett EM, <br />                                                     Aldrich CC. Inhibition siderophore biosynthesis Mycobacterium tuberculosis <span id='am-41' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span>nucleoside</span> <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                      Page 10 <br />  <br />  <br />                                                     bisubstrate analogues: Structure-activity relationships nucleobase domain 5 &#226;&#8364;&#732;-O-[N-(salicyl) <br />                                                     sulfamoyl]<span id='am-53' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-54' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-56' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span>adenosine</span>. Journal Medicinal Chemistry 2008;51:5349&#226;&#8364;&#8220;5370. [PubMed: 18690677] <br />                                                16. Stirrett KL, Ferreras JA, Jayaprakash V, Sinha BN, Ren T, Quadri LEN. Small molecules NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     structural similarities siderophores novel antimicrobials Mycobacterium tuberculosis <br />                                                     Yersinia pestis. Bioorganic &amp; Medicinal Chemistry Letters 2008;18:2662&#226;&#8364;&#8220;2668. [PubMed: <br />                                                     18394884] <br />                                                17. Ferreras JA, Stirrett KL, Lu XQ, Ryu JS, Soll CE, Tan DS, Quadri LEN. Mycobacterial phenolic <br />                                                     glycolipid virulence factor biosynthesis: Mechanism small-molecule inhibition polyketide <br />                                                     chain initiation. Chemistry &amp; Biology 2008;15:51&#226;&#8364;&#8220;61. [PubMed: 18158259] <br />                                                18. Cisar JS, Ferreras JA, Soni RK, Quadri LEN, Tan DS. Exploiting ligand conformation selective <br />                                                     inhibition non-ribosomal peptide synthetase <span id='am-46' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-52' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> adenylation designed macrocyclic <br />                                                     small molecules. Journal American Chemical Society 2007;1297752-+ <br />                                                19. Meier JL, Barrows-Yano T, Foley TL, Wike CL, Burkart MD. unusual macrocycle forming <br />                                                     thioesterase mycolactone. Molecular Biosystems 2008;4:663&#226;&#8364;&#8220;671. [PubMed: 18493665] <br />                                                20. Lambalot RH, Walsh CT. Holo-[acyl-carrier-protein] synthase Escherichia coli, Vitamins                                                     Coenzymes. 1997;(Pt :254&#226;&#8364;&#8220;262. <br />                                                21. Glover KJ, Martini PM, Vold RR, Komives EA. Preparation insoluble transmembrane peptides: <br />                                                     Glycophorin  prion (110&#226;&#8364;&#8220;137 FGFR (368&#226;&#8364;&#8220;397). Analytical Biochemistry 1999;272:270&#226;&#8364;&#8220;274. <br />                                                     [PubMed: 10415099] <br />                                                22. Inglese J, Johnson RL, Simeonov  Xia MH, Zheng W, Austin CP, Auld DS. High-throughput <br />                                                     screening assays identification chemical probes. Nature Chemical Biology 2007;3:466&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     479. <br />                                                23. Pettersen EF, Goddard TD, Huang CC, Couch GS, Greenblatt DM, Meng EC, Ferrin TE. UCSF <br />                                                     chimera - visualization exploratory research analysis. Journal Computational <br />                                                     Chemistry 2004;25:1605&#226;&#8364;&#8220;1612. [PubMed: 15264254] <br />                                                24. La Clair JJ, Foley TL, Schegg TR, Regan CM, Burkart MD. Manipulation carrier proteins                                                     antibiotic biosynthesis. Chem Biol 2004;11:195&#226;&#8364;&#8220;201. [PubMed: 15123281] <br />                                                25. Haas JA, Frederick MA, Fox BG. Chemical posttranslational modification Escherichia coli <br />                                                     acyl carrier protein preparation dansyl-acyl carrier proteins&#226;&#8364;&#8482;. Protein Expression                                                     Purification 2000;20:274&#226;&#8364;&#8220;284. [PubMed: 11049751] <br />                                                26. Blommel PG, Fox BG. Fluorescence anisotropy assay proteolysis specifically labeled fusion <br />                                                     proteins. Analytical Biochemistry 2005;336:75&#226;&#8364;&#8220;86. [PubMed: 15582561] <br />                                                27. Joshi NS, Whitaker LR, Francis MB. component Mannich-type reaction selective tyrosine <br />                                                     bioconjugation. Journal American Chemical Society 2004;126:15942&#226;&#8364;&#8220;15943. [PubMed: <br />                                                     15584710] <br />                                                28. Schlick TL, Ding ZB, Kovacs EW, Francis MB. Dual-surface modification tobacco mosaic <br />                                                     virus. Journal American Chemical Society 2005;127:3718&#226;&#8364;&#8220;3723. [PubMed: 15771505] <br />                                                29. Yin J, Straight PD, McLoughlin SM, Zhou  Lin AJ, Golan  Kelleher NL, Kolter R, Walsh CT. <br />                                                     Genetically encoded short peptide tag versatile protein labeling Sfp phosphopantetheinyl <br />                                                     transferase. Proc Natl Acad Sci U S 2005;102:15815&#226;&#8364;&#8220;15820. [PubMed: 16236721] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                30. Zhou  Cironi P, Lin AJ, Xu YQ, Hrvatin S, Golan  Silver PA, Walsh CT, Yin J. Genetically <br />                                                     encoded short peptide tags orthogonal protein labeling sfp AcpS phosphopantetheinyl <br />                                                     transferases. Acs Chemical Biology 2007;2:337&#226;&#8364;&#8220;346. [PubMed: 17465518] <br />                                                31. Lakowicz, JR. Principles fluorescence spectroscopy. New York: Kluwer Academic/Plenum; 1999. <br />                                                32. Machida M, Ushijima N, Machida MI, Kanaoka Y. Fluorescent Thiol Reagents.9 N-(7- <br />                                                     Dimethylamino-4-Methyl-Coumariny)Maleimides (Dacm) - Novel Fluorescent Thiol Reagents. <br />                                                     Chemical &amp; Pharmaceutical Bulletin 1975;23:1385&#226;&#8364;&#8220;1386. <br />                                                33. Campbell, <span id='am-83' about='oboInOwl:id' typeof='owl:Thing'>ID</span>.; Dwek, RA.; Dwek, RA. Biological spectroscopy. Menlo Park, Calif: Benjamin/ <br />                                                     Cummings Pub.  1984. <br />                                                34. Selwyn MJ. Simple Test Inactivation Enzyme Assay. Biochimica Et Biophysica <br />                                                     Acta 1965;105:193&#226;&#8364;&#8220;194. [PubMed: 4221326] <br />                                                35. Wu G, Yuan Y, Hodge CN. Determining appropriate substrate conversion enzymatic assays                                                     high-throughput screening. Journal Biomolecular Screening 2003;8:694&#226;&#8364;&#8220;700. [PubMed: <br />                                                     14711395] <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                                Page 11 <br />  <br />  <br />                                                36. Zhang JH, Chung TDY, Oldenburg KR. simple statistical parameter use evaluation                                                    validation high throughput screening assays. Journal Biomolecular Screening 1999;4:67&#226;&#8364;&#8220;73. <br />                                                    [PubMed: 10838414] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                37. NCGC, Assay Guidance Criteria. 2009. <br />                                                    http://www.ncgc.nih.gov/guidance/HTS_Assay_Guidance_Criteria.html March 31 2009 <br />                                                38. Wellings DA, Atherton E. Standard Fmoc protocols. Methods Enzymology 1997;289:44&#226;&#8364;&#8220;67. <br />                                                    [PubMed: 9353717] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 1. Post-translational modification carrier proteins <br />                                                translated apo-carrier proteins 1 coenzyme 2 react PPTase generate 3&#226;&#8364;&#8482;- <br />                                                phosphoadenosine-5&#226;&#8364;&#8482;-phosphate (PAP) 3 holo-carrier proteins 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                              Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 2. Fluorescence resonance energy transfer assay phosphopantetheinyl transferase <br />                                                Action PPTase fluorescently labeled acceptor substrate 5, conjunction                                                fluorphore-modified coenzyme analogue (mCoA) 6, assembles FRET pair conversion <br />                                                thiol-blocked crypto-acceptor substrate 7. assembly FRET pair NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                detected fluorescence microplate reader. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 3. Synthesis assay components <br />                                                  Beginning <span id='am-23' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span>L-alanine</span> loaded polystyrene resin, standard SPPS procedures afford NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                YbbR sequence appended &#206;&#181;-aminocaproic acid residue. Reaction FITC, followed <br />                                                cleavage HPLC purification gives FITC-YbbR peptide substrate 8. (B) CoA 2                                                reacted maleimide bearing probes 9 10 produce fluorescent mCoA derivatives <br />                                                11 12, respectively. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                            Page 15 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 4. Probe selection photophysical evaluation crypto-YbbR peptides <br />                                                DACM-mCoA 11 TAMRA-mCoA 12 probes evaluated FRET <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                characteristics FITC-YbbR 8.   DACM-mCoA 11 acts FRET donor FITC-YbbR <br />                                                8. (B) Excitation emission spectra probes 11 8 recorded normalized                                                determine spectral overlap 11 emission 8 excitation calculate overlap integral <br />                                                J(&#206;&#187;) [Eq. (3)] theoretical Forster&#226;&#8364;&#8482;s radius R0 [Eq. (2)]  Summation                                                normalized data gave J(&#206;&#187;) 1.60 &#195;&#8212; 10&#226;&#710;&#8217;13cm4 R0 =38&#195;&#8230;. (C) FRET characterization                                                crypto-DACM-FITC-YbbR peptide. Equimolar solutions 11 8 prepared                                                varying degree modification crypto-DACM-FITC-YbbR 7, fluorescent <br />                                                spectra recorded excitation 403nm. spectra overlayed, increase 10% <br />                                                increments modification 0 100%; starting maximal emission observed                                                462nm minimum wavelength. (D) Eq. (5) fitted emission data (C)                                                gave efficiency transfer value 0.82. (E) FITC-YbbR 8 acts FRET donor                                                TAMRA-mCoA 12. (F) Excitation emission spectra probes 11 12 recorded <br />                                                normalized determine spectral overlap 11 emission 12 excitation calculate <br />                                                overlap integral J(&#206;&#187;) [Eq. (3)] theoretical Forster&#226;&#8364;&#8482;s radius R0 [Eq. (2)]  <br />                                                Summation normalized data gave J(&#206;&#187;,) 3.01 &#195;&#8212; 10&#226;&#710;&#8217;13cm4 R0 =56&#195;&#8230;. (G) FRET <br />                                                characterization crypto-TAMRA-YbbR peptide. Equimolar solutions 11 12                                                prepared varying degree modification crypto-TAMRA-YbbR 7,                                                fluorescent spectra recorded excitation 475nm. spectra overlayed, increase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                10% increments modification 0 100%; starting maximal emission <br />                                                observed 513nm spectrum minimum value wavelength. <br />                                                (H) Eq. (5) fitted emission data 513nm (G) gave efficiency transfer <br />                                                value 0.99. <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                          Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 5. Reaction progress enzyme titration <br />                                                  Reaction progress plots concentrations AcpS. Rates determined                                                experiments intersect dashed threshold 32,000 RFU. (B) Reaction rates                                                progress curves plotted AcpS concentration demonstrate linear <br />                                                response. (C) Reaction progress curves concentrations Sfp. Rates determined <br />                                                experiments intersect dashed threshold 32,000 RFU. (D) Reaction rates <br />                                                progress curves C plotted Sfp concentration demonstrate linear <br />                                                response. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                               Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 6. Determination analysis timepoint  <br />                                                  Progress curve 96 plate containing 48 reactions presense (&#226;&#8212;?) absence <br />                                                (&#226;&#8211;&#170;) 50 nM AcpS data timepoint analysed statistically presented                                                Table 2. 24 minute time point selected routine analysis, gave acceptable assay <br />                                                statistics linear range screen. (B) Change relative fluorescence units                                                24 minute time point (Table 2, boxed data) plotted number.  value                                                data 0.72. (C) Progress curve 96 plate containing 48 reactions presense <br />                                                (&#226;&#8212;?) absence (&#226;&#8211;&#170;) 12.5 nM Sfp. data timepoint analysed statistically                                                presented Table 2. 16 minute time point selected routine analysis, gave <br />                                                acceptable assay statistics linear range screen. (B) Change relative <br />                                                fluorescence units 16 minute time point (Table 2, boxed data) plotted                                                number.  value data 0.71. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Anal Biochem. Author manuscript; available PMC 2010 November 1. <br />                             Foley Burkart                                                                                       Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Fig. 7. Reproducibility IC50 values 3&#226;&#8364;&#8482;-phosphoadenosine-5&#226;&#8364;&#8482;-phosphate <br />                                                  Sfp screened 12 concentrations PAP ranging 244 nM 500 &#194;&#181;M                                                separate days. data fit parameter dose response curve using <br />                                                GraphPad returned IC50 values 13, 11, 13 &#194;&#181;M. (B) Replication                                                experiment ) using AcpS enzyme. curves returned IC50 values 86, <br />                                                 <br /> </body></html>