<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Developmental Biology                                                                                                                BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access unexpectedly high degree specialization widespread <br /> involvement sterol metabolism C. elegans putative <br /> aminophospholipid translocases <br /> Nicholas N Lyssenko*1,3,4, Yana Miteva1,5, Simon Gilroy2,6, Wendy Hanna- <br /> Rose1 Robert Schlegel1 <br />  <br /> Address: 1Department Biochemistry Molecular Biology, Pennsylvania State University, University Park, Pennsylvania, 16802, USA, <br /> 2Department Biology, Pennsylvania State University, University Park, Pennsylvania, 16802, USA, 3The Huck Institutes Life Sciences <br />  Pennsylvania State University, University Park, Pennsylvania, 16802, USA, 4Department Cell Biology, NC10, Lerner Research Institute, <br /> 9500 Euclid Avenue, Cleveland Clinic Foundation, Cleveland, Ohio, 44195, USA, 5Department Physiology, University Pennsylvania, <br /> Philadelphia, Pennsylvania, 19104, USA 6Department Botany, University Wisconsin, B117 Birge Hall, Madison, Wisconsin, 53706, USA <br /> Email: Nicholas N Lyssenko* - lyssenn@ccf.org; Yana Miteva - miteva@mail.med.upenn.edu; Simon Gilroy - sgilroy@wisc.edu; Wendy Hanna- <br /> Rose - wxh21@psu.edu; Robert Schlegel - ur3@psu.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 2 October 2008                                                     Received: 30 July 2008 <br />                                                                               Accepted: 2 October 2008 <br /> BMC Developmental Biology 2008, 8:96   doi:10.1186/1471-213X-8-96 article available  http://www.biomedcentral.com/1471-213X/8/96 <br /> &#194;&#169; 2008 Lyssenko et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />         Abstract <br />         Background: P-type ATPases subfamily IV exclusively eukaryotic transmembrane proteins         proposed directly translocate aminophospholipids <span id='am-15' about='obo:IMR_0001358' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-16' property="oboInOwl:id" content="IMR:0001358" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200250" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C02737" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:18303" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="phosphatidylserine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span>phosphatidylserine</span> phosphatidylethanolamine         exofacial cytofacial monolayer plasma membrane. Eukaryotic genomes contain genes <br />         encoding members subfamily. present unclear genes kind organism <br />         individual roles genes perform organism development. <br />         Results: systematically investigated expression developmental function  tat-1 6, <br />         subfamily IV P-type ATPase genes encoded Caenorhabditis elegans genome. tat-5 ubiquitously- <br />         expressed essential gene group. tat-6 poorly-transcribed recent duplicate tat-5. tat-2 4 exhibit <br />         tissue-specific developmentally-regulated expression patterns. Strong expression tat-2 tat-4 occurs         intestine certain cells alimentary  expressed uterus,         spermatogenesis fully-formed spermatheca. tat-2 expressed pharyngeal gland cells,         excretory cells developing vulva. expression pattern tat-3 completely <br />         different tat-2 tat-4. tat-3 expression detectable steroidogenic tissues: hypodermis <br />         XXX cells, cells pharynx  gland), various tissues reproductive <br />          uterus spermatheca) seam cells. Deletion tat-1 4 individually interferes little <br />         regular progression organism growth development normal conditions. <br />          tat-2 4 essential reproductive growth sterol starvation. <br />         Conclusion: tat-5 likely encodes housekeeping protein performs proposed aminophospholipid <br />         translocase function routinely. individually dispensable, tat-1 4 partly <br />         redundant. Expression patterns sterol deprivation hypersensitivity deletion phenotype tat-2 4 <br />         suggest genes carry subtle metabolic functions, fine-tuning sterol metabolism digestive <br />         steroidogenic tissues. findings uncover unexpectedly high degree specialization widespread <br />         involvement sterol metabolism genes encoding putative aminophospholipid translocases. <br />  <br />  <br />  <br />                                                                                                                                        Page 1 17 <br />                                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Background                                                       IV P-type ATPases work concert efficiently <br /> Subfamily IV P-type ATPase superfamily group         sequester aminophospholipids cytofacial exclusively eukaryotic large multipass transmembrane          leaflet [17]. extrapolated multicellular organ- <br /> proteins appear function inward &#226;&#8364;&#8220; exo-       isms, loss aminophospholipid transmem- <br /> facial cytofacial monolayer &#226;&#8364;&#8220; translocases         brane asymmetry PS appearance cell surface <br /> aminophospholipids phosphatidylserine (PS) phos-             fatal [18], yeast paradigm predicts phatidylethanolamine (PE) choline <span id='am-24' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-25' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> phos-         somatic cell express individually nones- <br /> phatidylcholine (PC) [1,2]. PS, PE PC             sential ubiquitous P-type ATPases subfamily IV: <br /> unexpected substrates proteins &quot;clas-      Golgi apparatus plasma mem- <br /> sical&quot; P-type ATPases subfamilies  II III pump          brane. <br /> metal cations protons [3]. lines evidence sug- <br /> gest members subfamily IV translocate amino-             systematically investigated  tat-1 phospholipids.  biochemical investigations            6, P-type ATPase subfamily IV genes expressed mul- <br /> determined heterologously expressed purified            ticellular organism Caenorhabditis elegans subfamily IV P-type ATPases progress catalytic       expression patterns deletion phenotypes cycle hydrolyze <span id='am-36' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span> presence specifically PS       genes inconsistent predictions yeast PE [4,5]. second, genetic studies consist-          paradigm. does mean C. elegans P-type <br /> ently deletion genes encoding subfamily IV         ATPases subfamily IV translocate PS PE. members increases exposure cell surface endog-         fact, recent report shows loss tat-1 leads enous PS PE, normally concealed,            appearance PS surface germline certain diminishes translocation cytofacial leaflet        somatic cells [6].  findings suggest individ- <br /> exogenously-introduced labeled PS PE,              ually nonessential subfamily IV members, tat-1 4, <br /> normally quickly internalized [6-10]. Evidence PC       accomplish bulk aminophospholipid <br /> transported substrate extensive, consisting      internalization group  instead, observations labeled PC internalization Saccharomy-        specialized particular tissues, genes <br /> ces cerevisiae [10]. skepticism sub-    subtly regulate sterol metabolism   adjusting <br /> family IV P-type ATPases directly translocate          transbilayer lipid distribution. housekeeping amino- <br /> phospholipids  opposed pumping sub-             <span id='am-10' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>phospholipid</span> translocase encoded tat-5, stance concentration gradient drives phospholipid          homolog NEO1. <br /> flip actual translocase) [11] directly <br /> responsible internalization PS, PE PC              Results <br /> opposed facilitating vesicular traffic required proper    C. elegans animals express subfamily IV P-type <br /> working actual translocase) [12].         ATPases alternative candidate transported substrate         C. elegans genome encodes predicted members ATPases strong evidence altered vesicular traffic     P-type ATPase subfamily IV. genes named cause PS, PE PC exposure cell surface       transbilayer amphipath transporter (tat) 1 6 (Addi- <br /> subfamily IV P-type ATPase mutants far emerged.         tional file 1). detected splice isoforms tat-1 dif- <br />                                                                  fer respect final exons (Figure 1A). Eukaryotic genomes contain genes encoding P-type            isoform distinct stop codon predicted gen- <br /> ATPases subfamily IV (14 mice humans) [13].            erate product divergent C terminus (Additional brings questions: eukaryotes require          file 2). detected splice isoforms tat-2 differ putative aminophospholipid translocases,            respect penultimate exons are individual functions genes,             predicted generate products sequence genes divided subgroups order         variability N C termini (Figure 1B). study  Investigations single-cell fungus S. <span id='am-67' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-68' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span>cer</span>-     slightly different isoforms tat-3 identified <br /> evisiae offer answers. S. cerevisiae genome             (Figure 1C). product longer isoform contains includes subfamily IV P-type ATPase genes.           extra C-terminal amino acids, absent  NEO1, essential [14]. remaining &#226;&#8364;&#8220; DRS2,       shorter version. tat-4 locus includes open reading <br /> DNF1, DNF2 DNF3 &#226;&#8364;&#8220; individually dispensable               frames (ORFs), tat-4 T24H7.6, appear form comprise essential subgroup [15].                operon (Figure 1D). T24H7.6, tat-4, cDNA Drs2p resides predominantly Golgi appa-          amplified using splice leader 2 (SL2) primer. ratus [16] Dnf1p Dnf2p reside predominantly           C. elegans, trans-splicing SL2 usually indicates plasma membrane [10], deleted          gene occupies subordinate position operon [19]. highest levels PS PE exposure cell surface     tat-4 stop codon resides exon [10,12]. findings imply number subfamily         weak polyadenylation signals (Additional file 2). <br />  <br />  <br />                                                                                                                       Page 2 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                              http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />                                                                     clones [20] low cDNA amplification yield (data                                                                     shown), suggests gene recent poorly- <br />                                                                     expressed duplicate strongly expressed tat-5. <br />                                                                     reasons, tat-6 characterized  <br />  <br />                                                                     P-type ATPases subfamily IV customarily divided <br />                                                                     classes [13]. phylogenetic analysis                                                                     ATPases expressed S. cerevisiae, C. elegans humans <br />                                                                     reveals substantial evolutionary dissimilarity                                                                     class 2 classes (Figure 2). early split <br />                                                                     branch leading class 2 branch lead- <br />                                                                     ing rest classes, class 2 genes dupli- <br />                                                                     cated significantly.  yeast express single gene                                                                     class (NEO1), C. elegans humans express                                                                     class 2 genes (tat-5 tat-6 nematode).                                                                     branch subfamily, contrast, quickly <br />                                                                     undergone multiplication diversification. differ- <br />                                                                     ence extent evolutionary expansion indicate <br />                                                                     strictly preserved class 2 proteins perform Figure <br /> Detected 1 transcripts tat-1 4                           essential function conserved evolution Detected transcripts tat-1 4. tat-1              eukaryotes, frequently duplicating ATPases tat-2 (B) consist generally shorter exons undergo        classes rapidly evolve new roles. <br /> significant alternative splicing, tat-3 (C) tat-4 (D) include <br /> somewhat longer exons encode essentially ver-          tat-5 housekeeping gene <br /> sion product.                                                tat-5 expression cassettes constructed (Figure <br />                                                                     3A). tat-5b::nls::gfp cassette, region operon <br />                                                                     promoter drives transcription sequence encoding Bicistronic tat-4 T24H7.6 messages detected           nuclear localization signal (NLS)-tagged green fluorescent <br /> terminate sole T24H7.6 polyadenilation signal              protein (NLS-GFP). tat-5a/c::nls::gfp, NLS-GFP likely arise tat-4 polyadenylation sig-          control fragment spanning second tat-5 <br /> nals fail induce poly  tail addition. choice            intron. cassettes introduced nematode <br /> polyadenylation site affects translation tat-4       genome particle bombardment. transgenic lines &#226;&#8364;&#8220; <br /> T24H7.6.                                                            integrated (lcIs481.4) extrachromosomal <br />                                                                     (lcEx481.2) &#226;&#8364;&#8220; carrying tat-5b::nls::gfp transgenes iso- alternative transcripts tat-5 detected. tat-5b        lated (Figure 3A). lines GFP fluorescence ema- <br /> begins sequence short exons                   nates broadly inspected tissues located close proximity upstream ORF                developmental stages, early embryos undergoes trans-splicing SL2 (Figure 3A). tat-5a tat-        germline (Figure 3B&#226;&#8364;&#8220;F). integrated line (lcIs461.3) <br /> 5c identical  start         carrying tat-5a/c::nls::gfp transgenes identified. GFP <br /> exon located 3 kilobases downstream             signal detected lcIs461.3 embryos  tat-5b exons spliced SL1 SL2. addi-           maphrodites. findings suggest operon pro- <br /> tion trans-splicing SL2, subordinate cistrons           moter controls tat-5 transcription. ubiquitous <br /> operon usually reside close previous ORF [19].          pattern tat-5 expression revealed using reporter criteria, tat-5b subordinate cistron        fully supported situ staining data nematode <br /> operon. status tat-5a/c certain. long            expression pattern database (NEXTDB [20]).  tat-5 sequence end previous ORF start           ubiquitously expressed gene. <br /> tat-5a/c transcripts conceivably hold  addition operon promoter) cis-acting regulatory              N2 (wild-type) animals fed tat-5 double-stranded RNA <br /> element drives transcription isoforms.            (dsRNA) order suppress TAT-5 RNA interference <br />                                                                     (RNAi) bore dead embryos showing signs extensive <br /> tat-6 73% identical tat-5. tat-6 deletion mutant          necrosis (Figure 3G). eggs uterus tat- <br /> (ok1984, large middle portion protein product              5(RNAi) hermaphrodites disintegrate <br /> removed; C. elegans Gene Knockout Consortium)               (data shown). Similar tat-5(RNAi) phenotypes   reportedly viable. Comparatively weak expression tat-         maphrodite sterility embryonic lethality) 6, evidenced paucity publicly available cDNA          detected systematic RNAi screens genes sup- <br />  <br />  <br />                                                                                                                          Page 3 17 <br />                                                                                                    (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  Figure <br />   phylogenetic <br />         2      tree S. cerevisiae, C. elegans human P-type ATPases subfamily IV phylogenetic tree S. cerevisiae, C. elegans human P-type ATPases subfamily IV. tree assembled <br /> using ClustalW2 [48]. outgroup Drosophila melanogaster calcium-transporting P-type ATPase. <br />  <br />  <br /> pression causes clear morphological developmental           tat-2 reporter clearly detectable 2-fold stage <br /> abnormalities [21,22]. deletion mutants tat-5      embryos sets pharyngeal cells, developing <br /> (tm1823, tm1772 tm1741) reportedly                 pharyngeal-intestinal valve set cells poste- <br /> homozygous lethal  National Bioresource Project         rior (Figure 4B). larval (L1) stage, GFP fluores- Nematode, Tokyo, Japan). ubiquitously                 cence appears intestine (Figure 4C). L4 expressed essential survival, tat-5 charac-     adult animals exhibit reporter signals unidentified cells <br /> teristics housekeeping gene.                               pharyngeal procorpus, gland cells located                                                                 posterior bulb pharynx, pharyngeal-intestinal <br /> tat-2 4 exhibit developmentally-regulated tissue-       valve, rectal gland cells, intestine cells specific expression patterns                                    excretory (Figure 4D 4E, data shown). integrated extrachromosomal transgenic            tat-2 reporter signals seen L4 larvae pri- <br /> lines carrying tat-2::nls::gfp expression cassettes   mary vulval lineage vulE vulF cells proxi- particle bombardment (Figure 4A). exception         mal gonad (Figure 4F 4G). vulval fluorescence instances ectopic transcription extrachro-   vanishes moderately strong uterine signal appears <br /> mosomal lines, generally exhibit nearly identical      uterine-vulval connection complete adults <br /> patterns reporter expression. Curiously, GFP fluores-        (data shown). gonadal signal, emanating cence tat-2::nls::gfp transgenic nematodes emanates      spermatids, migrates spermatheca time nucleus, expected NLS-         ovulation (Figure 4G 4H). <br /> tagged reporter, plasma membrane <br /> region (Figure 4B&#226;&#8364;&#8220;H). Apparently, short TAT-2-coding        integrated extrachromosomal transgenic <br /> fragment retained tat-2 expression cassette      lines carrying tat-3::nls::gfp cassettes derived (Figure <br /> &quot;overpowers&quot; NLS directs chimeric reporter          5A).  integrated extrachro- <br /> <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> plasma membrane compartment.                     mosomal) investigated exhibit <br />                                                                 nearly identical expression patterns. tat-3 reporter signal <br />                                                                 appears embryos developing pharynx (data <br />  <br />  <br />                                                                                                                     Page 4 17 <br />                                                                                               (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-5 a3housekeeping gene <br /> tat-5 housekeeping gene. 5&apos; end tat-5 locus structure tat-5 expression cassettes  . Expres- <br /> sion tat-5b::nls::gfp lcIs481.4 line embryos (B C) head region adult (D), lcEx481.2 isolate head-intestine junction (E) developing somatic gonad (F). Necrotic death tat-5(RNAi) embryos (G). Abbrevia- <br /> tions: DTC &#226;&#8364;&#8220; distal tip cell; &#226;&#8364;&#8220; intestinal nucleus; sg &#226;&#8364;&#8220; somatic gonad. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 5 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-2 reporter <br />          4       expression pattern <br /> tat-2 reporter expression pattern. 5&apos; end tat-2 locus, locations deleted regions tm1634 tm1773 <br /> mutants (bars) structure tat-2 expression cassette  . Expression tat-2 reporter lcIs982.4.2 line embryos (B), intestine L2 larvae (C) gland cells pharynx, pharyngeal-intestinal valve, excretory <br /> gland cell intestine young adult (D). GFP fluorescence lcEx982.2.4.5 line excretory pore excretory <br /> canal cells (E). Reporter signal outlining vulF cells early lcIs982.4.2 L4 larvae  frame series shows  <br /> lay pseudocolored UV image bright light image, enlarged sampling using Photoshop; dashes extend contact surface tissues) (F). tat-2 reporter expression spermatogenesis (G) adult sper- <br /> matheca (H) lcIs982.4.2 animals. Abbreviations: &#226;&#8364;&#8220; excretory gland cell; epc &#226;&#8364;&#8220; excretory pore cell; g &#226;&#8364;&#8220; pharyngeal gland cell; <br /> ga &#226;&#8364;&#8220; gonad arm; ph &#226;&#8364;&#8220; pharynx; pv &#226;&#8364;&#8220; pharyngeal-intestinal valve; oo &#226;&#8364;&#8220; oocyte; r &#226;&#8364;&#8220; rectum; sp &#226;&#8364;&#8220; spermatheca. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 6 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-3  reporter <br />           5      expression pattern <br /> tat-3 reporter expression pattern. 5&apos; end tat-3 locus, location deleted region tm1275 mutants (bar) structure tat-3 expression cassette  . tat-3 reporter expression pharyngeal-intestinal valve muscle marginal, gland, cells pharynx lcIs471.12 line (B). GFP fluorescence muscle buccal epithelial cells pharynx procorpus XXX cells lcIs471.4B line (C). tat-3 reporter signals rectum tail region <br /> (D), seam cells hypodermis (E), DTC (F) AC (G) lcIs471.4B animals. GFP staining vulva late <br /> L4 stage lcIs881.3.2 larvae (H). Reporter expression adult lcIs471.4B vulva  . Cell labels: AC &#226;&#8364;&#8220; anchor cell; DTC &#226;&#8364;&#8220; dis- <br /> tal tip cell; e1 2 &#226;&#8364;&#8220; buccal epithelial cells; g1 &#226;&#8364;&#8220; gland cell; mc &#226;&#8364;&#8220; marginal cell; pm2, 5, 6 7 &#226;&#8364;&#8220; pharyngeal muscle cells; vul &#226;&#8364;&#8220; <br /> vulval cells; utse &#226;&#8364;&#8220; uterine seam cell; XXX &#226;&#8364;&#8220; XXX cells. Abbreviations: hn &#226;&#8364;&#8220; hypodermal nucleus; pv &#226;&#8364;&#8220; pharyngeal-intestinal <br /> valve; sn &#226;&#8364;&#8220; seam cell nucleus. <br />  <br />  <br />  <br />  <br />                                                                                                                            Page 7 17 <br />                                                                                                      (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                            http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  shown). fully formed alimentary             germ line apoptotic cells annexin V-GFP, peptide <br /> strong GFP fluorescence observed muscle, mar-            binds specifically PS, altered tat-1(RNAi)  <br /> ginal buccal epithelial cells pharynx, pha-         maphrodites [23], suggesting RNAi tat-1 did <br /> ryngeal-intestinal valve  lesser intensity,           suppress target. <br /> rectal epithelial cells (Figure 5B&#226;&#8364;&#8220;D). Seam cells display strong fluorescence soon lineage           RNAi studies conducted, deletion <br /> established embryonic development (Figure 5E).              mutants tat-2 4 available. tat- adults, moderate weak fluorescence arise            2(tm1773) (frame shift, splicing <span id='am-57' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span>acceptor</span> deleted), tat- XXX cells, unidentified cells head            3(tm1275) (frame shift) tat-4(tm1801)  basal pro- tail regions hypodermis (Figure 5C 5E).            moter region large portion coding sequence reproductive  tat-3 reporter expression              deleted) likely null (Figure 4A, Figure 5A Fig- <br /> begins distal tip cells (DTC) L1 anchor       ure 6A). tat-2(tm1634) lacks N-terminal exon encoding <br /> cell (AC) early L3 (Figure 5F 5G). GFP signal later      44 amino acids present isoforms, visible dividing progeny vulval precursor cells      partly functional (Figure 4A). tat-2(tm1773), tat- <br /> (VPCs). late L4, anchor cell fuses uterine         2(tm1634), tat-3(tm1275) tat-4(tm1801) single seam cell (utse), does express reporter (Fig-        tat-4(tm1801); tat-3(tm1275) double mutants  <br /> ure 5H). vulval cells continue exhibiting moderate             ble. <br /> fluorescence adulthood (Figure 5H 5I). <br />                                                                    determine deletion tat-2 4 exerts integrated extrachromosomal tat-4::nls::gfp           negative effect nematode growth reproduction, <br /> transgenic lines derived (Figure 6A).           synchronized mutant N2 larvae followed <br /> lines (2 integrated 3 extrachromosomal) investi-          developmental stages, viable progeny gated  Notable tat-4 reporter expression begins       adult hermaphrodites counted. Wild-type, tat- <br /> 2&#226;&#8364;&#8220;3 fold embryos developing pharyngeal-intestinal           3(tm1275), tat-4(tm1801) tat-4(tm1801); tat- <br /> valve unidentified cells posterior (Figure 6B).      3(tm1275) animals essentially indistinguishable fully formed alimentary  GFP fluorescence               timing progression <br /> emanates strongly pharyngeal-intestinal valve,            developmental stages number  <br /> rectal gland cells intestine (Figure 6C 6D). tat-      ble progeny produced hermaphrodites (Figure 7). tat- <br /> 4 reporter expressed uterus, sperma-         2(tm1634) mutants passed development slightly <br /> togenesis proximal gonad spermatheca             slower mutated larvae. evident previously ovulated adults (Figure 6E&#226;&#8364;&#8220;6G).                      lower higher number tat-2(tm1634) offspring pro- <br />                                                                    duced  respectively, sampling available NEXTDB [20] situ staining images             period, comparison numbers N2 progeny. <br /> corroborate expression tat-3 pharynx vulva,          20% (n = 10) tat-2(tm1773) hermaphrodites tat-4 spermatheca, intestine, pharyngeal-intesti-        notably fewer progeny rest animals nal valve uterus tat-2 intestine. tat-1          genotype. reflected large standard devi- <br /> expression pattern obtained             ation value mutant.  pair-wise statistical <br /> near perfect inverted repeats located 5&apos; end         analysis (ANOVA) shows total number viable <br /> tat-1 locus destabilized expression cassette vectors (data         hatchlings tat-2(tm1634) tat-2(tm1773) shown).  images NEXTDB                  mutants significantly different correspond- <br /> tat-1 expressed tissue-specifically.  tat-2, tat-3,   ing number N2 animals (P = 0.22 P = 0.30, respec- <br /> tat-4  likely, tat-1 appear expressed devel-      tively). Overall, deletion tat-1, tat-2, tat-3 tat-4 <br /> opmentally regulated tissues-specific patterns (Table 1).          individually tat-3 tat-4 does                                                                    impair nematode development reproduction sig- <br /> tat-1 4 nonessential regular growth              nificant degree regular growth conditions. <br /> conditions <br /> TAT-2 4 expression patterns pro-           tat-2 tat-4 mutant animals hypersensitive sterol <br /> teins present critical tissues key periods        deprivation nematode development. determine cur-                C. elegans sterol heterotroph uptakes various <br /> tailing expression tat-1, tat-2, tat-3 tat-4 lead      exogenous sterols converts compounds 7- gross morphological developmental abnormali-                dehydrocholesterol [24-26]. metabolite pre- <br /> ties, N2 animals fed dsRNA genes.            cursor dafachronic acids, hormone promotes <br /> tat-1(RNAi), tat-2(RNAi), tat-3(RNAi) tat-4(RNAi)              reproductive growth [27]. Sterol uptake conversion animals did exhibit notable developmental mor-            7-dehydrocholesterol occurs intestine [28-30], <br /> phological defect (data shown).  staining           dafachronic acids synthesized primarily  <br />  <br />                                                                                                                       Page 8 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                            http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figure <br /> tat-4 reporter <br />          6      expression pattern <br /> tat-4 reporter expression pattern. 5&apos; end tat-4 locus, location deleted region tm1801 mutants (bar) structure tat-4 expression cassette  . Reporter expression pharyngeal-intestinal valve, intestine rectal <br /> gland cells 3-fold embryos (B) advanced stage larvae (C D) lcIs911.30 line. GFP fluorescence uterus <br /> (E), spermatogenesis (F) spermatheca previously ovulated hermaphrodites (G) lcIs911.30 animals. <br /> Abbreviations: pv &#226;&#8364;&#8220; pharyngeal-intestinal valve; r &#226;&#8364;&#8220; rectum; rgc &#226;&#8364;&#8220; rectal gland cells; sp &#226;&#8364;&#8220; spermatheca. <br />  <br />  <br /> hypodermis [31]. subfamily IV P-type ATPases TAT-2             solid support medium routine nematode growth 4 facilitated step sterol           contains sterols substances used prepara- <br /> transport pathway &#226;&#8364;&#8220; site exogenous sterol             tion supplemented <span id='am-62' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-63' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cholesterol</span> final <br /> uptake site 7-dehydrocholesterol conversion          concentration 5000 ng/ml [32]. combined dafachronic acids, tat-2 4 mutants             sterol medium sufficient opti- <br /> exhibit sterol deprivation hypersensitivity evident            mal nematode growth: N2 animals grow just decreased reproductive growth.                                    plates enriched cholesterol 1000 ng/ml [30]. <br />                                                                   determine tat-2 4 mutants hyper- <br />  <br />  <br />                                                                                                                        Page 9 17 <br />                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                        http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> Table 1: Expression tat-2, tat-3, tat-4 tat-5 C. elegans <br /> tissues. <br />  <br />  Cells tissues                            tat-2   tat-3   tat-4   tat-5 <br />  <br />  somatic tissues                                                  +++ <br />  Germline                                       -       -       -      - <br />  <br />  Alimentary  Pharyngeal gland cells                       +++       -      - <br />  Pharyngeal muscle cells                       -      +++      - <br />  Pharyngeal marginal cells                     -      +++      - <br />  Buccal epithelium                             -      +++      - <br />  Pharyngeal-intestinal valve                  +++     +++     +++ <br />  Intestine                                    +++       -     +++ <br />  Rectal gland cells                           +++       -     +++ <br />  Rectal epithelial cells                       -       ++      - <br />  <br />  Reproductive  DTCs                                           -     +++      - <br />  AC                                             -     +++      - <br />  Developing vulva: vulE vulF cells   +++     +++      - <br />  Developing vulva: VPC progeny                  -     +++      -              Figure   7 using <br />                                                                               Reproduction <br />                                                                               growth <br />                                                                               performed <br />                                                                               tions) conditions  mutant <br />                                                                                                   (n = 10 <br />                                                                                               ANOVA;                                                                                                             error <br />                                                                                                         N2               regular <br />                                                                                                                 bars statistical <br />                                                                                                              12 nematodes standard <br />                                                                                                                 animals;             analysis <br />                                                                                                                                      devia- <br />  Adult vulva                                    -      ++      -              Reproduction mutant N2 nematodes  Uterus                                        ++       -     +++             regular growth conditions (n = 10 12 animals; statisti- <br />  Spermatogenesis                              +++       -     +++             cal analysis performed using ANOVA; error bars stand- <br />  Adult spermatheca                            +++       -     +++             ard deviations). <br />  Excretory                             +++       -       - <br />  <br />  Epithelial                                                            nutrient [29]. second generation lacked sterol reserves <br />  Seam cells                                     -     +++       -             exhibited notable effects sterol deprivation. <br />  Hypodermis                                     -      ++       - <br />                                                                               Growth N2 animals test plates proportional <br />   Head tail region cells/XXX cells           -     ++        -             cholesterol medium (Figure 8).                                                                               fifth day, N2 populations 1000 ng/ml 100 ng/ <br />  Expression pattern tat-1 determined method            ml cholesterol plates cleared bacterial food began <br />  choice, tat-6 excluded gene appears          starving. indicates generation animals <br />  recent poorly expressed duplicate tat-5.                                  produced plenty viable progeny second <br />                                                                               generation grew quickly significant mortality. N2 <br /> sensitive cholesterol deprivation, test plates spe-                   populations 10 ng/ml cholesterol plates cleared food <br /> cially prepared eliminate exogenous sources sterol                  day 7. time, N2 1 ng/ml cholesterol plates supplemented cholesterol final                       contained plenty food fewer smaller <br /> centrations 1000 ng/ml, 100 ng/ml, 10 ng/ml 1 ng/                       second-generation animals. <br /> ml. OP50 strain Escherichia coli cultures grown synthetic defined medium sterol, supple-                         tat-3(tm1275) populations grew test plates mented cholesterol concentrations                     pace wild-type populations. contrast, tat- <br /> plates. Test plates spotted bacterial culture                  2(tm1634) tat-4(tm1801) populations exhibited cholesterol concentration. resultant food                        dramatic retardation growth (Figure 8). <br /> lawns test plates identical size. Eggs                  Food tat-2(tm1634) tat-4(tm1801) 1000 ng/ml gravid hermaphrodites grown regular sterol                       cholesterol plates cleared day 7: days later <br /> plates collected using alkaline hypochlorite                         cholesterol concentration N2 tat- <br /> method, hatched overnight sterol plates,                       3(tm1275) plates. Significantly, 100 ng/ml lower hatchlings transferred test plates, 30 plate                  cholesterol concentration plates, growth second <br />  day . generation larvae test plates                  generation tat-2(tm1634) tat-4(tm1801) larvae pro- <br /> grew maturity reproduced animals                         gressed minimally. evident slight,  changes <br /> contained reserve sterol deposited oocytes                        size tat-2(tm1634) tat-4(tm1801) popula- <br /> mothers maintained regular high supply                          tions individual animals day 5 7 (Figure 8). <br />                                                                               tat-2(tm1634) tat-4(tm1801) mutants did appear <br />  <br />  <br />                                                                                                                                   Page 10 17 <br />                                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                               http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Figuredeprivation <br /> Sterol   8           sensitivity mutant N2 nematodes <br /> Sterol deprivation sensitivity mutant N2 nematodes. Notations: arrows indicate edge food lawn plates bacterial food; arrow heads point generation animals; stars identify plates began starving. dark lines photographs numbers written underside plates. <br />  <br />  <br />  <br />  <br />                                                                                                                          Page 11 17 <br />                                                                                                     (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                        http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  suffer particular developmental morphologi-       sented indicate tat-5 encodes sole <br /> cal defect respond sterol limitation dra-   ubiquitously expressed essential subfamily IV P-type <br /> matically N2 tat-3 animals. Furthermore,          ATPase. tat-5(RNAi) phenotype dramatic: sterility developmental delay exhibited tat-2(tm1634) animals         embryonic lethality; furthermore, tissues tat-5(RNAi) sterol-limited test plates severe         embryos exhibit extensive necrotic cell death. obser- mild lag development mutants showed          vations suggest TAT-5 performs critical house- regular plates. tat-2(tm1773) populations exhib-       keeping function. yeast Neo1p [14], <br /> ited severe growth retardation sterol-limited test plates   essential, TAT-5 segregates class 2, (data shown). unlikely independ-    ancient class P-type ATPases subfamily IV. <br /> ently derived deletion alleles related genes       class 2 genes encode highly specialized <br /> contain secondary mutations cause identical phe-       subfamily IV P-type ATPases execute critically <br /> notype, mutagenesis specifically tat-2 tat-4      important housekeeping function. induce hypersensitivity sterol deprivation. <br />                                                                Individual deletion remaining C. elegans subfamily <br /> tat-4(tm1801); tat-3(tm1275) double mutants           IV P-type ATPase genes, tat-1 4, (excluding tat-6, <br /> sensitive sterol availability tat-4(tm1801) single     recent poorly expressed duplication tat-5) does mutants                                                        grossly impair C. elegans growth development. data suggest tat-3 major player     consistent high degree redundancy <br /> sterol metabolism;  gene expressed       genes.  analysis expression <br /> hypodermis, major steroidogenic tissue [28],      involvement sterol metabolism exhibited tat-2 <br /> pharynx, potential sterol storage organ [32]. test        4 suggests different explanation. tat-4 tat-3 contributes marginally sterol metabo-         expressed tissues tat-2, tat-3 expression <br /> lism, tat-3(tm1275) mutants crossed tat-             occurs completely different tissues 4(tm1801) mutants derive double mutant         expressing tat-2 tat-4 tat-2 (Table 1). <br /> genes. tat-3 tat-4 expression patterns overlap     Furthermore, tat-4; tat-3 double mutants just viable pharyngeal-intestinal valve (Table 1), terms     tat-4 tat-3 single mutants wild-type animals. <br /> development reproduction regular conditions,         observations indicate closely related <br /> tat-4(tm1801); tat-3(tm1275) animals indistinguisha-        class 5), tat-3 tat-4 redundant. <br /> ble N2 nematodes (Figure 7). sterol-deprivation        closely related tat-2 (class 1b) tat-4 <br /> test plates, double mutants genes performed         expressed tissues, deletion worse tat-4(tm1801) nematodes (Figure 9). tat-            confers hypersensitivity sterol deprivation 4(tm1801) populations noticeably larger tat-         extent, suggests partly <br /> 4(tm1801); tat-3(tm1275) populations day 7 1000          redundant. tat-2 gene expressed ng/ml cholesterol concentration plates. day 8        excretory  Curiously, tat-3 expression plates, tat-4(tm1801) animals cleared food,     occurs VPC progeny vulval development plenty bacteria consume dou-    continues adult vulva, tat-2 expression <br /> ble mutants. Growth minimal 100 ng/ml               takes place specifically vulE vulF cells lower cholesterol concentration plates tat-           vulval development  suggests genes <br /> 4(tm1801); tat-3(tm1275) tat-4(tm1801) animals             perform different roles development vulva. 9-day observation period.  dele-      Expression pattern tat-1 (class 1a) tion tat-3 exacerbates sterol-deprivation hypersensitiv-    obtained method choice (transcriptional <br /> ity tat-4 mutants.                                          reporters stably introduced genome using particle <br />                                                                bombardment).  data presented Darland- <br /> Discussion                                                     Ransom et al. [6] indicate gene expressed Specialization redundancy C. elegans             gonad body-wall muscle cells &#226;&#8364;&#8220; tissues tat-2 <br /> putative aminophospholipid translocases                        4 expressed. Overall, lack gross <br />                                                                growth development impairment tat-1 4 <br /> Eukaryotic genomes contain genes encoding sub-            mutants does stem high level redun- <br /> family IV P-type ATPases [13]. need abun-      dancy genes. <br /> dance genes organism individual <br /> functions presently poorly understood. Closely related     individually dispensable S. cerevisiae subfamily <br /> genes partly redundant          IV P-type ATPase genes (DRS2, DNF1, DNF2 DNF3) <br /> functionally highly specialized. C. elegans genome         essential group [17], individually dispensable <br /> includes  tat-1 6, genes protein prod-       C. elegans subfamily IV P-type ATPase genes ucts P-type ATPases subfamily IV. data pre-         dispensable group. tat-1 dispen- <br />  <br />  <br />                                                                                                                   Page 12 17 <br />                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> mutantssensitivity <br /> Higher <br /> Figure  9          sterol limitation tat-4(tm1801); tat-3(tm1275) double mutants comparison tat-4(tm1801) single <br /> Higher sensitivity sterol limitation tat-4(tm1801); tat-3(tm1275) double mutants comparison tat- <br /> 4(tm1801) single mutants. dark lines photographs numbers written underside plates. <br />  <br />  <br />  <br />  <br />                                                                                                                      Page 13 17 <br />                                                                                                 (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                          http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> sable subfamily IV P-type ATPase expressed body-          fine-tune aspects sterol processing sterol- <br /> wall muscle cells [6]. tat-1 null animals lose extra       metabolizing tissues. tat-2 homolog mamma- <br /> muscle cells, does translate notable         lian Atp8b1/ATP8B1, encodes subfamily IV P-type <br /> growth defect [6]. suggests class    ATPase localizes apical membrane hepato- <br /> 1 5 subfamily IV P-type ATPases nonessential         cytes, enterocytes polarized cells mam- <br /> routine growth development C. elegans cells.              malian digestive [35,36]. Future investigations <br />                                                                  determine TAT-2 localizes api- <br /> Investigations S. cerevisiae suggest individually dis-   cal membrane TAT-4 resides pensable subfamily IV P-type ATPases redundant,              membrane TAT-2. tat-3 expressed hypodermis <br /> especially respect internalization PS PE          XXX cells, steroidogenic tissues 7- <br /> [17].  does case indi-       dehydrocholesterol converted dafachromic acids <br /> vidually dispensable C. elegans subfamily IV P-type              [28].  likely product gene medi- <br /> ATPases. Considering C. elegans germline                ates, partly, turnover 7-dehydrocholesterol somatic cells normally conceal PS [23] loss tat-     release metabolite intestine. Future <br /> 1 leads PS appearance surface germline          research test hypothesis, determine <br /> certain somatic cells [6], contradiction         membrane compartment TAT-3 resides. <br /> yeast nematode paradigms does imply             Overall, lack notable phenotype tat-1(RNAi) <br /> subfamily IV P-type ATPases organism        tat-2 4 null mutants growth translocate aminophospholipids.  considering         normal conditions inability tat-2 tat-4 <br /> class 2 subfamily IV members execute bio-           mutants grow sterol limiting conditions indicate <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> functions subfamily IV members              critical function dispensable subfamily IV <br /> classes [14] loss asymmetric PS PE distri-       P-type ATPases fine tune metabolism sterol <br /> bution plasma membrane incompatible             lipids. <br /> survival [33], findings suggest  nematode, <br /> TAT-5 carries bulk aminophospholipids inter-          Recent reports indicate yeast DRS2 nalization, TAT-1 4 adjust cross-bilayer           involved sterol (ergosterol) metabolism [37-39]. <br /> lipid distribution particular tissues.                        DRS2&#206;&#8221; mutants exhibit reduced ability uptake exoge- <br />                                                                  nous sterol. Atp8b1/ATP8B1 mediates sterol (cholesterol) <br /> C. elegans subfamily IV P-type ATPases fine-tune sterol          metabolism mammals [40,41]. Mutants Atp8b1 <br /> metabolism                                                       secrete cholesterol bile [40]. homozygous <br /> C. elegans sterol heterotroph employs steroid-       individuals compromised ATP8B1 exhibit reduced <br /> mediated developmental program. nematode takes               levels high-density lipoprotein (HDL)-cholesterol exogenous sterols intestine converts          greatly increased levels oxidized cholesterol [41]. 7-dehydrocholesterol [28]. metabolite         previous findings yeast mam- used synthesis reproductive development-pro-       mals, involvement tat-2 4 sterol metabolism <br /> moting hormone, dafachronic acids, hypodermis             C. elegans suggests role performed sub- <br /> [27,28]. unfavorable growth conditions, production         family IV P-type ATPases process conserved hormone stops nematode larvae embark              eukaryotes far greater currently recog- <br /> dauer (dispersal) developmental pathway. Complete                nized. <br /> sterol deprivation mimics cessation hormone syn- <br /> thesis favorable growth environment [34].        Conclusion sterol supplies optimal, nematode development           presented study shows individually dispensable reproduction proportional sterol concentration        C. elegans putative aminophospholipid translocase genes medium [30]. result, C. elegans reproductive         exhibit high degree specialization. finding  <br /> growth sensitive indicator sterol level       tradicts expectation derived investigations hypodermis lipid taken intestine        yeast genes broadly redundant traveled animal body.                       answers question genes                                                                  kind organism. study partly answers data tat-2 tat-4 required growth       question individual roles performed putative ami- sterol starvation deletion tat-3 exacer-      nophospholipid translocases uncovering widespread <br /> bates sterol hypersensitivity tat-4 mutants.         involvement members group sterol metabo- <br /> late stages embryonic development rest         lism. nematode life cycle, tat-2 tat-4 strongly <br /> expressed intestine cells alimentary  likely protein products genes <br />  <br />  <br />                                                                                                                     Page 14 17 <br />                                                                                                (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br />  <br /> Methods                                                         Chromosomal fragments contain promoter regions <br /> C. elegans culture genetics                                 tat-2 5 cloned YAC Y17G9 following strains mutants C. elegans used:      cosmids T24H7, W09D10 F36H2 pPD95.69, var. Bristol strain N2 (wild type), unc-119(-), tat-            promoter NLS-GFP expression vector 2(tm1634) IV, tat-2(1773) IV, tat-3(tm1275) III, tat-           vector kit, using standard cloning methods  Addi- <br /> 4(tm1801) II tat-4(tm1801) II; tat-3(tm1275) III.       tional file 4). DNA expression vector precip- <br /> 4 tat mutants isolated var. Bristol strain N2       itated DNA vector carrying unc-119(+) <br /> background National Bioresource Project          (pDP#MM016B [44]) gold particles described Experimental Animal Nematode [42]. mutants             Tucker et al. [45], coated particles bom- crossed 2 8 times N2 animals.            barded gonad unc-119(-) nematodes noted, C. elegans animals maintained standard           described Praitis et al. [46] modifications <br /> nematode growth medium plates spotted E. coli               Additional file 4). bombardment, nematodes <br /> OP50 20&#194;&#176;C [32].                                              washed plate transferred 2 3 fresh <br />                                                                 plates. Following incubation period, stably transgenic <br /> Gene transcription analysis                                     lines  exhibiting Unc phenotype) selected <br /> Mixed-stage N2 animals grown enriched peptone           plates: hermaphrodite plate <br /> plates. Populations approaching starvation col-            non-Unc animals. Integrated lines identified lected, combined, suspended Trizol&#194;&#174; (Invitrogen,          animals homozygous trangene Carlsbad, CA), freeze-thawed times treated         derived. <br /> 20 strokes pestle 40 ml Dounce homogenizer. <br /> Total RNA isolated recommended Tri-          RNA interference <br /> zol manufacturer. mRNA purified total RNA          500 900 bp-long 5&apos; fragments tat-1 5 cDNA Poly Pure kit (Ambion, Austin, TX). cDNA           clones sub-cloned pPD129.36 (inverted-T7-pro- <br /> synthesized using dT(18) adaptor-dT(18)       moter vector vector kit). dsRNA production <br /> primers  Additional file 4). 5&apos; RACE, splice          nematode feeding conducted described leader-specific primer gene-specific primer (GSP)         Kamath et al. [47]  Additional file 4). employed; 3&apos; RACE, GSP adaptor- <br /> dT(18) primer used. Products cDNA amplification         Growth reproduction regular conditions resolved aragose gels, purified cloned       Gravid N2 mutant hermaphrodites allowed TA helper vector. number clones distinct          lay eggs fresh plates 2 h. animals band agarose gels sequenced  Additional         removed, eggs left hatch overnight. file 4). following ESTs kindly provided Dr. Kohara       day, 10&#226;&#8364;&#8220;12 larvae genotype transferred indi- sequenced: yk1228h06, yk34c11, yk1496c06,                  vidual plates: larva plate. 94 118 h later, yk209d5, yk36b8, yk214f10 yk126b10.                         adult nematodes transferred new plates.                                                                 old plates incubated overnight permit viable <br /> Gene expression analysis                                        eggs hatch, chilled ice halt nematode movement, <br /> Nucleus-targeted transcriptional reporters used         larvae counted. 0&#226;&#8364;&#8220;94 h, 94&#226;&#8364;&#8220;118 h visualize tat-2 5 expression. approach         118&#226;&#8364;&#8220;154 h sampling periods   <br /> chosen tats long genes; tag desta-     respectively. <br /> bilizes P-type ATPases subfamily IV ([14] Lyssenko, <br /> unpublished observations); yeast mammals, mem-           Sterol deprivation assay <br /> bers subfamily exit Golgi apparatus        mixture agarose (8 g/l) ether-extracted peptone <br /> heterodimer proteins Cdc50p/Lam3p group             water autoclaved 30 min, cooled 65&#194;&#176;C [7,12], expression tats lead accumula-       water bath enriched desired concentrations <br /> tion paired peptide Golgi apparatus            5 mg/ml cholesterol ethanol. OP50 strain E. <br /> potentially induction unfolded protein response,         coli culture grown minimal medium (20 mM trigger apoptosis reporter-             NH4Cl, 0.2% w/v <span id='am-47' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span>D-glucose</span>, 2 mg/ml <span id='am-52' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> M9 buffer) <br /> expressing cells; finally, concentrating reporters   overnight 37&#194;&#176;C. culture divided 4 aliq- <br /> nucleus facilitates cell identification [43]. Note    uots, centrifuged remove supernatant, long portions (long code 65             suspended volume peptone solution <br /> amino acids) tat-2 tat-4 translated sequences       (0.5% w/v peptone M9 buffer) enriched cho- <br /> included expression cassettes, cassettes did     lesterol 4 final concentrations plates. <br /> assemble extrachromosomal arrays did inte-         suspensions pipetted carefully cor- <br /> grate genome (Additional file 3).                      responding concentration plates produce uniform <br />                                                                 spots. Nematodes synchronized L1 hatching <br />  <br />  <br />                                                                                                                   Page 15 17 <br />                                                                                              (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                         http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> eggs sterol plates. L1 larvae transferred                 4.    Ding J, Wu  Crider BP, Ma Y, Li X, Slaughter C, Gong L, Xie XS: <br />                                                                                    Identification functional expression isoforms spotted plates, exactly 30 animals plate. 3 replica                            ATPase II, putative aminophospholipid translocase. <br /> plates seeded cholesterol concentration gen-                          Effect isoform variation ATPase activity phos- <br /> otype. populations allowed grow 20&#194;&#176;C                            pholipid specificity. J Biol Chem 2000, 275:23378-23386. <br />                                                                              5.    Paterson JK, Renkema K, Burden L, Halleck MS, Schlegel RA, William- <br /> 1.5&#226;&#8364;&#8220;2 weeks periodic observations.                                            son P, Daleke DL: Lipid specific activation murine P4- <br />                                                                                    ATPase Atp8a1 (ATPase II). Biochemistry 2006, 45:5367-5376. <br />                                                                              6.    Darland-Ransom M, Wang X, Sun CL, Mapes J, Gengyo-Ando K, <br /> Authors&apos; contributions                                                             Mitani S, Xue D: Role C. elegans TAT-1 protein maintain- <br /> NNL conceived project, designed experiments,                               ing plasma membrane phosphatidylserine asymmetry. Sci- <br /> formed experiments, analyzed data wrote manu-                              ence 2008, 320:528-531. <br />                                                                              7.    Paulusma CC, Folmer  Ho-Mok KS, Waart DR, Hilarius PM, <br /> script. YN performed experiments analyzed data. SG,                            Verhoeven AJ, Oude Elferink RP: ATP8B1 requires accessory <br /> WH RAS contributed experimental design, ana-                                protein endoplasmic reticulum exit plasma mem- <br /> lyzed data edited manuscript. authors read                         brane lipid flippase activity. Hepatology 2008, 47(1):268-78. <br />                                                                              8.    Iwamoto K, Kobayashi S, Fukuda R, Umeda M, Kobayashi T, Ohta  <br /> approved final manuscript.                                                     Local exposure phosphatidylethanolamine yeast <br />                                                                                    plasma membrane implicated cell polarity. Genes Cells <br />                                                                                    2004, 9:891-903. <br /> Additional material                                                          9.    Wang L, Beserra C, Garbers DL: novel aminophospholipid <br />                                                                                    transporter exclusively expressed spermatozoa                                                                                    required membrane lipid asymmetry normal fertili- <br />      Additional file 1                                                             zation. Dev Biol 2004, 267:203-215. <br />      WormBase IDs given names C. elegans ORFs encoding P-     10.   Pomorski T, Lombardi R, Riezman H, Devaux PF, van Meer G, <br />                                                                                    Holthuis JC: Drs2p-related P-type ATPases Dnf1p Dnf2p <br />      type ATPases subfamily IV. <br />                                                                                    required phospholipid translocation yeast <br />      Click file                                                           plasma membrane serve role endocytosis. Mol Biol <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Cell 2003, 14:1240-1254. <br />      213X-8-96-S1.doc]                                                       11.   Kuhlbrandt W: Biology, structure mechanism P-type <br />                                                                                    ATPases. Nat Rev Mol Cell Biol 2004, 5:282-295. <br />                                                                              12.   Chen S, Wang J, Muthusamy BP, Liu K, Zare S, Andersen RJ, Graham <br />      Additional file 2                                                             TR: Roles Drs2p-Cdc50p complex protein trans- <br />      Peculiarities tat-1 tat-4 transcription.                               port phosphatidylserine asymmetry yeast plasma <br />      Click file                                                           membrane. Traffic 2006, 7:1503-1417. <br />      [http://www.biomedcentral.com/content/supplementary/1471-               13.   Halleck MS, Schlegel RA, Williamson PL: Reanalysis ATP11B,      213X-8-96-S2.doc]                                                             type IV P-type ATPase. J Biol Chem 2002, 277:9736-9740. <br />                                                                              14.   Wicky S, Schwarz H, Singer-Kr&#195;&#188;ger B: Molecular interactions                                                                                    yeast Neo1p, essential member Drs2 family ami- <br />      Additional file 3                                                             nophospholipid translocases, role membrane traf- <br />      Fusions GFP long N-terminal portions TAT-2 TAT-4         ficking endomembrane  Mol Cell Biol 2004, <br />      expressed apparent toxicity.                                  24:7402-7418. <br />                                                                              15.   Hua  Fatheddin P, Graham TR: essential subfamily Drs2p- <br />      Click file                                                           related P-type ATPases required protein trafficking <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Golgi complex endosomal/vacuolar sys- <br />      213X-8-96-S3.doc]                                                             tem. Mol Biol Cell 2002, 13:3162-3177. <br />                                                                              16.   Natarajan P, Wang J, Hua  Graham TR: Drs2p-coupled amino- <br />                                                                                    phospholipid translocase activity yeast Golgi membranes <br />      Additional file 4                                                             relationship vivo function. Proc Natl Acad Sci USA 2004, <br />      Supplementary materials methods.                                          101:10614-10619. <br />      Click file                                                     17.   Graham TR: Flippases vesicle-mediated protein transport. <br />      [http://www.biomedcentral.com/content/supplementary/1471-                     Trends Cell Biol 2004, 14:670-677. <br />      213X-8-96-S4.doc]                                                       18.   Schlegel RA, Williamson P: Phosphatidylserine, death knell. <br />                                                                                    Cell Death Differ 2001, 8:551-563. <br />                                                                              19.   Blumenthal T: Trans-splicing polycistronic transcription                                                                                    Caenorhabditis elegans operons. Trends Genet 1995, 11:132-136. <br />                                                                              20.   Shin T, Kohara Y: NEXTDB: expression pattern map <br />                                                                                    database C. elegans. Genome Informatics 1999, 10:213-214. <br /> Acknowledgements                                                             21.   Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann like express gratitude Dr. Graham Thomas sharing            M, Ahringer J: Functional genomic analysis C. elegans chro- <br /> equipment, Dr. Peggy Halleck providing technical expertise Katie           mosome systemic RNA interference. Nature 2000, <br />                                                                                    408:325-330. <br /> Estes helping identify gene expression patterns. NNL supported <br />                                                                              22.   Maeda  Kohara Y, Tamamoto M, Sugimoto  Large-scale analysis consecutive pre-doctoral fellowships American Heart                gene function Caenorhabditis elegans high  <br /> Association.                                                                       RNAi. Curr Biol 2001, 11:171-176. <br />                                                                              23.   Z&#195;&#188;llig S, Neukomm LJ, Jovanovic M, Charette SJ, Lyssenko NN, Hal- <br />                                                                                    leck MS, Reutelingsperger CP, Schlegel RA, Hengartner MO: Amino- <br /> References                                                                         phospholipid            translocase       TAT-1         promotes <br /> 1.      Lenoir G, Williamson P, Holthuis JCM: origin lipid               phosphatidylserine exposure C. elegans apoptosis. <br />         asymmetry: flip <span id='am-30' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> transport. Curr Opin Chem Biol             Curr Biol 2007, 17:994-999. <br />         2007, 11(6):654-61.                                                  24.   Chitwood DJ: Biochemistry function nematode ster- <br /> 2.      Tang X, Halleck MS, Schlegel RA, Williamson P: subfamily P-           oids. Crit Rev Biochem Mol Biol 1999, 34:273-284. <br />         type ATPases aminophospholipid transporting activity.           25.   Kurzchalia TV, Ward S: worms need cholesterol? Nat <br />         Science 1996, 272:1495-1497.                                               Cell Biol 2003, 5:684-688. <br /> 3.      Axelsen KB, Palmgren MG: Evolution substrate specificities         P-type ATPase superfamily. J Mol Evol 1998, 46:84-101. <br />  <br />  <br />  <br />                                                                                                                                       Page 16 17 <br />                                                                                                                  (page number citation purposes) <br />  BMC Developmental Biology 2008, 8:96                                                           http://www.biomedcentral.com/1471-213X/8/96 <br />  <br />  <br />  <br /> 26.   Merris M, Kraeft J, Tint GS, Lenard J: Long-term effects sterol    44.   Maduro M, Pilgrim D: Identification cloning unc-119,       depletion C. elegans: sterol content synchronized wild-               gene expressed Caenorhabditis elegans nervous sys- <br />       type mutant populations. <br /> </body></html>