<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A calpain unique alveolates essential Plasmodium falciparum knockdown reveals involvement pre-S-phase development <br /> Ilaria Russo, Anna Oksman, Barbara Vaupel, Daniel E. Goldberg1 <br /> Departments Medicine Molecular Microbiology, Howard Hughes Medical Institute, Washington University School Medicine, <br /> 660 South Euclid Avenue, St. Louis, MO 63110 <br />  <br /> Edited Anthony Cerami, Warren Pharmaceuticals, Ossining, NY, approved November 21, 2008 (received review July 18, 2008) <br />  <br /> Plasmodium falciparum encodes single calpain distinct           undertook genetic characterization assess essentiality domain composition restricted alveolates. evaluate                 gain insight function cell. <br /> potential protein drug target, assessed essen-              report Pcalp distinct type calpain restricted tiality. gene disruption double cross gene trun-            alveolates. Using multiple approaches, gene cation single cross recombination failed.               essential intraerythrocytic parasites. developing conditional <br /> unable achieve allelic replacement using missense mutation           knockdown  able examine gene function. catalytic cysteine codon, obtain synon-             Pcalp plays critical role cell cycle progression tropho- <br /> ymous allelic replacement parasites. results suggested            zoite development. calpain gene proteolytic activity important optimal parasite growth. gain insight biological         Results <br /> role, used FKBP degradation domain generate               Pcalp Distinct Type Calpain Restricted Alveolates. fusion protein stability transfected parasites             identified single calpain gene (Pcalp) P. falciparum, <br /> modulated small FKBP ligand, Shield1 (Shld1).                 MAL13P1.310 locus. predicted coding region 6,147 bp, <br /> calpain-GFP-FKBP fusion single cross integration             encodes 242-kDa protein, confirmed extensive se- endogenous calpain locus. Calpain levels knocked               quence analysis genomic DNA cDNA amplified parasite growth greatly impaired absence Shld1.            mature mRNA (SI Materials Methods). Orthologs Pcalp Parasites delayed ability transition ring        Plasmodium species sequence data stage ability progress S phase. Calpain            available (Table S1). phylogenetic analysis confirmed required cell cycle progression Plasmodium parasites              Pcalp greatest homology Aspergillus PalB (2), limited appears attractive drug target. shown                  catalytic domain.  low reliability node derivation <br /> regulated knockdowns possible P. falciparum                indicates substantial divergence (Fig. 1). fact, aside central <br /> useful evaluating essentiality function.                             calpain catalytic domain (IIa-b) C-terminal domain homol- <br />                                                                              ogous calpain domain III, Pcalp possesses long N terminus cell cycle &#229;&#8230;&#169; cysteine protease &#229;&#8230;&#169; essentiality &#229;&#8230;&#169; inducible &#229;&#8230;&#169; malaria          significant homology known protein. Pcalp distinct <br />                                                                              type calpain domain composition, common  <br /> P    lasmodium falciparum, causative agent severe malaria, <br />      complex life cycle involving numerous biological events, migration, host cell invasion, metabolism, cell cycle <br />                                                                              identified calpain class (Fig. 1). <br />                                                                                sequenced apicomplexan parasite genomes encode single <br />                                                                              calpain N-terminal homology, progression egress. Cysteine proteases implicated           gene predictions result shorter encoded N terminus (Fig. 1). processes (1), function                 ciliate calpains, Tetrahymena Paramecium, estimated 35 members protease class encoded genome            N termini homology Pcalp. Bootstrap analysis defined. single calpain-like protease           domain composition indicate Pcalp belongs distinct clade <br /> recognized (2).                                                              calpains restricted Apicomplexa alveolates <br />    Calpains belong C2 family cysteine proteases. Calpains          (Fig. 1). diatom Thalassiosira pseudonana, belongs organisms bacteria mammals exhibit great            chromalveolate kingdom, encoded calpain distant <br /> divergence sequence domain structure homologous              N-terminal homology alveolate sequences  shown). <br /> catalytic domains. Calpains implicated diverse pro- <br /> cesses (3&#226;&#8364;&#8220;6), muscle function, cell signaling, migration         Pcalp Expressed Early Intraerythrocytic Stages. pub- <br /> attachment, death, transformation, cell-cycle regulation, differen-          lished Pcalp mRNA assessments differ (9, 10), tiation development, fungal alkali tolerance,           analyzed cDNA semiquantitative RT-PCR asexual <br /> precise physiological role calpains poorly         development. Amplifying N-terminal C-terminal <br /> understood (3, 7). <br />    Calpains divided typical (resembling calpain 1) atypical (lacking domain IV) (4, 5). typical calpains, domain IV          Author contributions: R. D.E.G. designed research; R., O., B.V. performed <br /> mediates Ca2&#226;&#171;&#185; binding consequently activity regulation               research; R. D.E.G. analyzed data; R. D.E.G. wrote paper. <br />  <br /> dimerization (8). P. falciparum, 1 calpain gene (Pcalp)           authors declare conflict  <br /> apparent genome (2), atypical. unusually          article PNAS Direct Submission. <br /> long coding sequence, outside catalytic domain,          Freely available online PNAS open access option. <br /> distant homology characterized calpains. Pcalp            Data deposition: sequences reported paper deposited GenBank <br /> contains subdomains highly conserved related             database (accession nos. EF432831, EF432832, EF432833). <br /> species confer uniqueness protein. deter-             1To   correspondence addressed. E-mail: goldberg@borcim.wustl.edu. <br /> mines nucleolar localization, regulates calpain             article contains supporting information online www.pnas.org/cgi/content  <br /> ment nucleus reversible palmitoylation             0806926106/DCSupplemental. <br />  R., unpublished data). Surprised location Pcalp,             &#194;&#169; 2009 National Academy Sciences USA <br />  <br />  <br />  <br /> 1554 &#226;&#8364;&#8220;1559 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; February 3, 2009 &#229;&#8230;&#169; vol. 106 &#229;&#8230;&#169;  5                                                         www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0806926106 <br />                                                                                                           B <br />                    0.05                                    Domains-&gt; N-terminal Catalytic <br />                                                                                 IIa IIb     III   IV <br />                                               Drosoph_CalpC <br />                                        Schisto_Calp                                                                                                                    Domains-&gt; N-terminal Catalytic <br />                                                                                                                                                                                              IIa IIb III <br />                          68            Human_Calp9                                                             0.05 <br />                                                                                                                                     100                      P. falciparum <br />                    77                 Human_Calp3                                                                            100 <br />                         62     58      Murine_Calp8                                                                                                         P. reichenowi <br />                              95        Human_Calp2                                                                                                          P. gallinaceum <br />          63 <br />                                       Human_Calp1                                                                            99 <br />               63         72                                                                                                                                 P. chabaudii <br />                     50                 Human_Calp11                                                                                 100 <br />                                 100    Drosoph_CalpA                                                                         55                              P. yoelii <br />               40                       Drosoph_CalpB                                                                                100                      P. berghei <br />                     37                 C.elegans_clp-7 <br />         80 <br />                                        Human_Calp12                                                                                100                         P. vivax <br />              52                          Human_Calp13                                                                  100                                   P. knowlesi <br />                                         C.elegans_Tra3                                            C2 <br />                98 <br />                                         Human_Calp5                                                                                 100                              N. caninum <br />     48                                                                                                                       100 <br />                    66                      Human_Calp6                                                                                                            T. gondii <br />                                            Human_Calp10 <br />                                                                     ZnF_RBZ               SOLH                    98                                              E. tenella <br />                          100             Drosoph_CalpD <br />          62                              Human_SOLH TMs                                                                 70                                        B.bigemina <br />                                                                                                                              97 <br />                                        Arabid._Dek1                                                                                                                  T. annulata <br />                                                                                           LyM                                                               100 <br />         26                               Entamoeba_Calp <br />          64                                Trypanosome_Calp                                                                                                           T. parva <br />                                            Human_Calp7                   MIT               PBH <br />                     99                                                                                                                                       C.parvum <br />            58 <br />                                             C.elegans_clp-2 <br />          56                                                                                                     100                      P. tetraurelia <br />                                             PCalp <br />         91                                  Asperg_PalB                                                                                                                  T. therm.                  ? <br />                                                   C.neo_PalB <br />    18                                               Candidal_PalB <br />               80                                     S.cerevisiae_Rim13 <br />                                               Gloeob._Glr4140          HemolysinCabind <br />  <br />          62                                      P.ging_Tpr <br />                                             C.elegans_F44F1.3 <br />  <br /> Fig. 1. Phylogenetic analysis reveals Pcalp distinct type clade calpains.   Phylogenetic analysis Pcalp (catalytic domain) aligned representatives documented type calpain shown (left  domain composition protein (right . Bootstrap values shown tree joints. <br /> Domains color coded named appearance (SMART code). Pcalp N terminus subdomains conserved Plasmodium <br /> species (green)  R., O., D.E.G., unpublished data). Scale fractional change unit distance. (B) Phylogenetic analysis domain composition alveolate <br /> calpains. Crossed boxes denote incomplete sequence information. Sequence data listed Tables S1 S2 alignments Figs. S1 S2. <br />  <br />  <br />  <br /> coding regions normalizing actin, peak                                       S phase 30 hpi (Fig. 2B), corresponding peak induction (2&#226;&#171;&#187;) late ring/early trophozoite stages [18&#226;&#8364;&#8220;24                                        Pcalp. <br /> hours postinvasion (hpi), Fig. 2A], consistent data Bozdech et al. (9).                                                                                    Pcalp Gene Activity Essential Optimal Growth.    Despite low expression levels, able detect Pcalp                                    assessed essentiality Pcalp optimal P. falciparum asexual Western blotting immunoprecipitation large num-                                        growth using multiple approaches (Fig. 3A). ber parasites (5 &#226;&#171;&#187; 108). used parasite line (C3)                                  classical gene knockout approach using double cross                                                                                                        positive/negative selection (11). used double positive selection <br /> endogenous calpain ORF tagged GFP  Materials                                                                                                        fusing HcRed hDHFR obtained red fluorescent parasites <br /> Methods), anti-GFP antibody used visual-                                      stably maintained plasmid drug pressure (Fig. S4). <br /> ization. asexual cycle monitored morphology DNA                                         unable recover integrants negative <br /> content (Fig. 2B). Pcalp detected rings late                                          selection despite multiple attempts, assayed Southern blotting <br /> trophozoites, showing peak cell midtropho-                                      (Fig. 3B) nested PCR (data shown). <br /> zoites concentration (normalized BiP) rings (Fig. 2 C                                        gene disruption 5&#226;&#172;&#732; single cross  transfected 3D7 D). experiments, majority cells entered                                    parasites constructs containing 1.3 1.6 kb 5&#226;&#172;&#732; end  <br />  <br /> Fig. 2. Expression profiles Pcalp asexual <br /> cycle.   Semiquantitative RT-PCR analysis. Expression <br />                                                                 12 <br />                                                                                                                         ER   LR <br />                                                                                                                                             C <br />                                                                                                                                      ET MT MT/LT Sc <br />                                                                                                                                                       IP: R- -PCalp <br />                                                                 10 <br />                                                                                                                                                       WB: M- -GFP <br />                                                                           calpain/actin <br />  <br />  <br />  <br />  Pcalp mRNA asexual stages measured                                                                            rel intensity <br />  <br />  <br />  <br />  <br />                                                                   8 <br /> quantifying amplified regions encoding N (light                                                                                                      kDa <br />                                                                   6                                                                                      250 <br /> gray) C (dark gray) termini (map region Fig. <br />                                                                   4 <br /> S3). signals normalized actin. ER LR,                                                                                                    75 <br />                                                                   2                                                                                   IP inputs <br /> early late rings; ET LT, early late tropho-                                                                                               WB:   R - -BiP <br />                                                                   0                                                 hpi    9 19    25    29   33  43 <br /> zoites; Sc, schizonts. (B) Time course Pcalp-GFP                    ER      LR      ET         LT  LT/Sc   Sc <br />  <br /> (clone C3) cycle. Parasites sampled 30 min analyzed flow cytometry DNA content.               100         B ER      LR    ET MT MT/LT   Sc <br />                                                                                                                        8 <br />                                                                                                                                           D <br />                                                                           % total parasites <br />  <br />  <br />  <br />  <br />                                                                90 <br /> Populations selected analysis shown Fig.        80                                                      7 <br />                                                                                                                                           arbitrary units <br />  <br />  <br />  <br />  <br /> S3. percentage total parasites pre-S phase, S        70                                                      6 <br />                                                                60                                                      5 <br /> phase, mature schizogony represented                50                                                      4 <br /> black, red, green lines, respectively. Zero time           40                                                      3 <br /> corresponds peak mature schizonts. Dots        30 <br />                                                                                                                                                                                                            MICROBIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                                                        2 <br />                                                                20 <br /> indicate harvest times protein analysis (MT, midtro-       10                                                       1 <br /> phozoites) arrow marks approximate start            0 <br />                                                                  -5   0   5  10 15 20 25 30 35 40 45 50 55 <br />                                                                                                                        0 <br />                                                                                                                          8 12 16 20 24 28 32 36 40 44 S phase. (C) C3 parasites harvested                                     time (hpi)                                  time (hpi) <br /> indicated time points, Pcalp immunoprecipitated rabbit anti-Pcalp antiserum protein blotted mouse anti-GFP antibody. BiP content measured. lane corresponds equivalent cell numbers different stages. (D) Pcalp levels quantified blots plotted time absolute values (red) ratio calpain BiP (black). point ratio omitted minimal BiP expression early rings. <br />  <br />  <br /> Russo et al.                                                                                                                 PNAS &#229;&#8230;&#169; February 3, 2009 &#229;&#8230;&#169; vol. 106 &#229;&#8230;&#169;  5 &#229;&#8230;&#169; 1555 <br />                                                                                    Bi <br />  <br />  <br />  <br />  <br />                                                                                                                   m <br />                                                                                                         f. <br />                                                                                                                            ii                                                                           v <br />  <br />  <br />  <br />  <br />                                                                                                       ns <br />                                                                                                                                                   iii                             iv <br />  <br />  <br />  <br />  <br />                                                                                                                   <br />  <br />  <br />  <br />                                                                                                                                                                                        7 <br />                       5&#226;&#8364;&#8482;region            HcRed-hDHFR      3&#226;&#8364;&#8482; region <br />  <br />  <br />  <br />  <br />                                                                                               7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          7 <br />                                                                                                                                             kb                                                                         kb <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       3D <br />                                                                                                                       kb                                                     kb <br />  <br />  <br />  <br />  <br />                                                                                           3D <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      3D <br />                                                                                                   tr  <br />  <br />  <br />  <br />                                                                                                          pl <br />                                                                                                                       11                     8                                8                                         8 <br />                                      TK       X                                                                                              7                                7                                         7 <br />                                                                                                                        9                     6                                                                          6 <br />                                                                                                                        8                                                      6 <br />                                                                                                                        7                    5                                                                           5 <br />                     nat 5&#226;&#8364;&#8482; N-term          GFP Ahsp86-3&#226;&#8364;&#8482;   ,hDHFR                                                                                                             5 <br /> ii              S                                  N                   S                                               6                    4                                 4                                         4 <br />             1                                                                                                          5 <br />                       *                                                             Ci     NS          NS(II)                S        3D7                 ii           NS            S       Plasmids 3D7 <br />                     BstUI             Pcalp 3&apos;                                                                                                        kb                                                               kb <br /> iii                                                                                   2                                                              1.6         1                                                    1.6 <br />                                                                                                                                                      1.0                                                              1.0 <br />                                  2        1                   S                                                                                      0.8                                                              0.8 <br />       N P                                                                       P <br /> iv <br />                                                   *                                                                                                  0.5 <br />                                                                                                                                                      0.3                                                              0.5 <br />                                                                                                                                                                                                                       0.3 <br />        nat 5&#226;&#8364;&#8482;                                 Pcalp                                                                                                  0.1 <br />                                                                                           -       +          -        +    -       +    -     +     BstUI            -        +   -        +    -   +       -      + BstUI <br />         D          Q G F V/V D C/C S F L S S L T                       b.    Q G F V D C S F L S S L T                                                iii      clone 1 clone 2 clone 3 clone 4                    kb <br />                     CAAGGTTTTGTTGCGGATTGTTCTTTTCTGTCTTCATTAAC                   CAAGGTTTTGTTGCGGATTGTTCTTTTCTGTCTTCATTAAC                                        2                                                     1.6 <br />                                C        C                  S                                                                           NS <br />                                                                                                                                                                                                                      1.0 <br />                                                                                                                                                                                                                      0.8 <br />                                                                                                                                                                                                                      0.65 <br />                                                                                                                                                                                                                      0.5 <br />                                                                                                                                                                          -    +    -       +   -    +       -     + BstUI <br />  <br /> Fig. 3. Attempted genetic strategies disrupt calpain functionality.   Diagrams vectors.   double cross gene knockout, positive selection cassette <br /> containing HcRed-hDHFR fusion flanked Pcalp ORF 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends inserted <span id='am-16' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span>thymidine</span> kinase (TK) negative selection vector. (ii) single cross gene disruption, sequence homologous Pcalp ORF 5&#226;&#172;&#732; end cloned upstream hDHFR cassette. different lengths (1.3 1.6 kb) used. Control <br /> constructs identical contain 733 bp upstream sequence (nat 5&#226;&#172;&#732;, brackets). (iii) allelic replacement, sequence homologous 3.8 kb Pcalp ORF 3&#226;&#172;&#732; <br /> end used homologous recombination. plasmids created. mutation creating new restriction site (BstUI). codons downstream active site cysteine codon (*), synonymous change, nonsynonymous change. Predicted products integration ii iii shown Fig. S4. (iv) reference Pcalp gene shown. Primers used PCR/restriction allelic replacement strategy indicated (semiarrows). Restriction <br /> sites: X, XmaI; N, NsiI; S, SphI; P, PacI. (B) Southern blot analysis.   Double cross  Transfected parasite DNA (X/P digested) shows uninterrupted endogenous <br /> gene (open arrow) episomal plasmid (gray arrow), evidence integration event (filled arrows). panel different exposure time. (ii&#226;&#8364;&#8220;v) Single <br /> cross  DNA restricted N/S. 4 transfections control vector predicted integrations (ii iii; panels 2 blots). 4 <br /> transfections vectors gene disruption (iv v) evidence integration events (panels blot). Arrows  1.6-kb 5&#226;&#172;&#732; ORF ii iv; 1.3-kb 5&#226;&#172;&#732; ORF iii v; nat 5&#226;&#172;&#732; ii iii. 3D7, genomic DNA parental isolate. (C) PCR restriction screening. DNA prepared parasite culture <br /> transfected synonymous (S) nonsynonymous (NS) vector selected integrants.   PCR performed using primer 2 amplify transcribed locus plasmids. Product incubated (&#226;&#171;&#185;) (&#226;&#171;&#186;) BstUI   arrows indicate expected fragments. S vector transfectants showed evidence upstream cross (introduction restriction site), NS vector transfectant pools did. (ii) PCR performed using primer 1 amplify copies <br /> (including plasmid) transfectants isolated plasmids (1:1 mix S NS). isolated transfected episomal plasmids restrictable. 3D7, B. <br /> (iii) Examples isolated S clones. PCR performed primer 2; 1 4 clones shown crossed upstream. (D) Sequencing PCR products amplified S   NS (ii) integrant pools. Results control sequence analysis plasmids wild-type genomic amplicons Fig. S4. <br />  <br />  <br />  <br /> Pcalp (Fig. 3Aii). controls used second pair constructs,                                          active site, S mutation pool identical previous ones including additional 733 bp                                            alteration codon (Fig. 3D). control, amplified upstream Pcalp ORF. controls, integration,                                                 digested active site region parental 3D7 genome, recapitulate functional 5&#226;&#172;&#732; UTR ahead Pcalp ORF,                                                2 plasmids isolated DNA episomes transfections disruption constructs designed generate                                                 (Fig. 3Cii). confirmed absence restrictability promoterless ORF addition truncated copy. transfec-                                            parental genome, introduction BstUI sensitivity tions repeated twice resulted transformants                                            uniquely plasmids, similarity DNA content subjected selection integrants. promoter-                                           selected transfectants. <br /> containing constructs showed clear integration (Fig. 3Bii iii),                                          completed study isolating clones integration detectable promoterless constructs (Fig.                                             transfections better quantify proportion upstream inte- <br /> 3Biv v). results strongly suggest Pcalp essential                                      grants. isolated nearly 100 clones screened PCR product <br /> parasite growth. data imply 733 bp upstream                                             restrictability, described (examples Fig. 3Ciii). 62 ORF adequate drive calpain expression maintain                                              NS mutation plasmid-transfected clones, detected viability.                                                                                                   upstream integration, &#226;&#172;&#8225;21% isolated S <br />                                                                                                              mutation transfection carried newly introduced restriction site    assess Pcalp essentiality determine proteolytic <br />                                                                                                              consequently mutated cysteine codon. data indicate activity enzyme required, generated 2 identical <br />                                                                                                              essentiality Pcalp depends proteolytic activity. Results <br /> allelic replacement plasmids carrying 3&#226;&#172;&#732; thirds                                                                                                              attempts summarized Table 1. <br /> Pcalp ORF (Fig. 3Aiii). mutations   <br /> common plasmids, silent mutation creates new                                            Conditional Knockdown Pcalp. disruption calpain <br /> restriction site. second mutation, 2 codons downstream                                            gene unachievable, study function protein vivo <br /> catalytic cysteine triplet, synonymous (S) nonsynony-                                          attempted generate conditional knockdown, taking advan- <br /> mous (NS) (Cys&#226;&#172;&#381;Ala). transfected 3D7 parasites selected                                               tage FKBP destabilization domain (12). 10-kDa integrants. Cross occur upstream downstream                                           FKBP domain, fused protein  targets catalytic cysteine codon. Assuming cross random                                            degradation.  presence small-molecule FKBP 3,815 bp homologous sequence, probability integration                                          ligand, Shld1, degradation mitigated. shown <br /> upstream new restriction site 22%, downstream                                               work Plasmodium episomally expressed constructs (13). active site codon 78%, 0.16%.                                                 create conditional protein knockdown, tagged Pcalp C <br /> specifically PCR-amplified transcribed calpain locus,                                           terminus GFP-FKBP homologous recombination 3&#226;&#172;&#732; <br /> BstUI-restrictable product derived S mutation integrant                                             end endogenous locus (Fig. 4A). drug cycling pool (Fig. 3Ci), indicating upstream cross occurs                                             culture constantly exposed 0.5 &#226;?&#174;M Shld1, obtained successful active site cysteine changed. directly                                               integration (Fig. 4B). Clones isolated representative <br /> sequenced amplicons did detect NS mutation                                                 A7, chosen study. <br />  <br /> 1556 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0806926106                                                                                                                                                             Russo et al. <br />                          Table 1. Calpain gene disruption attempts <br />                         Method                                      Purpose                          Outcome                       Attempts <br />                         Double cross                    Gene knockout                      0 integrations                         6 <br />                         Single cross                    Gene recapitulation                4 integrations                         4 <br />                                                              Gene disruption                    0 integrations                         4 <br />                         Allelic replacement                  Synonymous mutation                7 mutations/33 clones                  1 <br />                                                              Nonsynonymous mutation             0 mutations/62 clones                  2 <br />  <br />  <br />  <br />  <br />    analyzed Pcalp-GFP-FKBP obtained                              fixed cells. calculated ratio percent dead <br /> synchronized midtrophozoites grown presence absence                       parasites detected versus Shld1 (Fig. 5F). A7, Shld1. Despite low expression level, able                          dead cells absence Shld1 consistently &#226;&#172;&#8225;1.5 <br /> visualize drop calpain removal Shld1 (Fig. 4C).                       times culture maintained Shld1, experiment repeated 3 times reproducible.                              C3 control culture ratio steady &#226;&#172;&#8225;1.0. immunoprecipitation judged successful Shld1-minus lane                      difference confirmed 3 techniques correlates IgG precipitated comparable                       decreased parasitemia detected. Shld1-plus lane. low molecular weight GFP-                              analyzed microscopy appearance dead <br /> positive bands indicated degradation sample                     parasites. Topro3-positive parasites, detected cultures workup  shown). control used C3, Pcalp-GFP                       Shld1, mature schizonts stochastically did <br /> integrant lacks degradation domain. Calpain content                    egress properly (data shown). Surprisingly, A7 calpain <br /> control cells affected. results strategy                  destabilized removal Shld1, detected appearance used create conditional knockdowns P. falciparum.                         type dead cell (Fig. 5G). cells trophozoites <br />                                                                                        appeared morphologically normal extraerythrocytic <br /> Evidence Important Regulatory Function Pcalp G1. Growth                   inside RBC ghosts, indicating failure homeostasis asynchronous Pcalp-GFP-FKBP-expressing clones depen-                            development. <br /> dent Shld1. removed Shld1 cultures                             surviving parasites A7 culture Shld1 observed growth inhibition 40&#226;&#8364;&#8220;60% 4 days (Fig. 5                       look morphologically abnormal slightly delayed B). control culture (Pcalp-GFP, C3) showed slight <br />                                                                                        transition ring trophozoite (Fig. 5H).  did <br /> long-term toxicity drug. increasing concentrations                                                                                        careful monitoring progression asexual cycle. Using <br /> Shld1, growth improved, reaching maximum &#226;&#172;&#8225;0.4 &#226;?&#174;M (Fig. <br />                                                                                        automated collector, harvested fixed synchronized <br /> 5C), remained constant 1 &#226;?&#174;M (data shown). <br />                                                                                        culture A7 30 min multiple cycles. samples    phenotypic analysis extended synchronized cultures                                                                                        analyzed DNA content (Fig. S5) counted percent A7 C3 clones. Shld1-maintained early schizonts split 2 portions  Shld1) closely monitored                       parasites G1 (1N), S phase (2&#226;&#8364;&#8220;15N), late schizonts (&#226;&#172;&#381;15N). cell cycle flow cytometry (Fig. 5 D E). differences                   Fig. 5I shows 2 representative cycles culture presence <br /> invasion efficiency observed.  A7 culture                          (Upper) absence (Lower) Shld1. substantial Shld1, growth defect evident trophozoite stage                    difference cycle length detected, start S phase (&#226;&#172;&#8225;20% decrease parasitemia). loss parasitemia                            absence Shld1 consistently delayed 3&#226;&#8364;&#8220;3.5 h. correlates <br /> maintained rest cycle detected                     peak enzyme expression (Fig. 2). data indicate cycle. defect detected control                        Pcalp knockdown leads cell death delay transitioning culture.                                                                               G1 phase trophozoite development. <br />    better understand drop parasitemia ring/ <br /> trophozoite transition, tested viability A7 C3                       Discussion <br /> cultures cycle. different measurements                  characterized distinct type calpain using combinations dyes stain metabolically active                     Apicomplexa alveolates. unique N-terminal <br /> cells (CM green), live dead cells (Syto-59 Acridine Orange                     domain, constitutes half entire protein. marked <br /> [AcOr]), dead cells (Topro3). signals pair                      divergence mammalian calpains presence number <br /> dyes (Syto&#226;&#171;&#185;CM, Topro&#226;&#171;&#185;CM, Topro&#226;&#171;&#185;AcOr) detected                                  clinically relevant apicomplexans make enzyme attractive <br /> FITC APC channels compared staining obtained                          target studied. <br />  <br />  <br />                                                                               B     3D7 Sel1 Sel2 Sel3      clones              C IP: R-   -GFP WB: M- -GFP <br />                                                                                                                              kb             A7           C3 <br />       N                                          S              N   500bp                                                               -        +   -        + Shield-1 <br />                                                                                                                             6.0 <br />                               Pcalp                                                                                         5.0 <br />                                       S                               N                                                     4.0 <br />                                                                                                                                                                    kDa <br />                           hDHFR                         GFP-FKBP                                                            3.0 <br />                                           Integration                                                                                                             250 <br />   N                                            S                            N                                                      IP inputs: <br />                                                                                                                                                                    75 <br />                     N                                                                                                       2.0                                   -BiP <br />                                                      probe     GFP-FKBP <br />                                                                                                                                                                   -PM2 <br />            C-term                                                                                                           1.0 <br />                                                                                                                                                                    25 <br />                                                                                                                                                                            MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> Fig. 4. Knockdown Pcalp.   Creation calpain-GFP-FKBP chimera homologous recombination. diagram shows strategy create C-terminally <br /> tagged calpain integration endogenous locus. plasmid contains sequence Pcalp ORF 3&#226;&#172;&#732; end frame GFP-FKBP. Relative positions NsiI (N) SphI (S) restriction sites, probe indicated. (B) Southern blot N/S restricted DNA 3 drug cycles (Sel1&#226;&#8364;&#8220;3) 3 clones. Arrows: <br /> endogenous gene (black), plasmid (gray), modified calpain locus (open). (C) Western blot immunoprecipitated Pcalp-GFP (C3) Pcalp-GFP-FKBP (A7) <br /> parasites grown presence absence Shld1. Samples processed Fig. 2B. lane corresponds equivalent numbers mid/late trophozoites. <br />  <br />  <br /> Russo et al.                                                                                                PNAS &#229;&#8230;&#169; February 3, 2009 &#229;&#8230;&#169; vol. 106 &#229;&#8230;&#169;  5 &#229;&#8230;&#169; 1557 <br />            13.5                                                      D 1211 <br />   A12 A7                                                                                              A7                 Ring <br />  <br />                                                                                                 10                                                                   G         merges    Topro3 Hoechst        Phase         H   - Shld1      + Shld1 <br />           10.5                                                                                                                          Ring <br />                                                                                                  9                                                                         <br />   parasitemia (%) <br />  <br />  <br />  <br />  <br />                                                                         parasitemia (%) <br />              9                                                                                   8 <br />                          7.5                                                                     7 <br />                                                                                                  6 <br />                            6                                                                     5 <br />                                                                                                  4 Ring <br />                                                                                                                                                                           b. <br />                          4.5 <br />                            3                                                                     3 <br />                                                                                                  2                              Troph <br />                          1.5                                                                                  Troph <br />                                                                                                  1 <br />                                                                                                    E Schiz <br />                           0                                                                      0 <br />  <br />  <br />   B 109 C3                                                          E 1211                            C3                 Ring           Ring      I100                                                                                          +Shld1 <br />                                                                                                 10                                                                   80 <br />                           8 <br />                                                                                                  9 <br />                                                                         parasitemia (%) <br />   parasitemia (%) <br />  <br />  <br />  <br />  <br />                           7                                                                      8 <br />                           6                                                                      7                                                                   60                            pre S-phase = 25.5 0.5h <br />                           5                                                                      6 <br />                           4                                                                      5                                                                   40 <br />                                                                                                      Ring                                                                                         Cycle time = 42.5 0.5h <br />  <br />  <br />  <br />  <br />                                                                                                                                                 total parasites <br />                                                                                                  4 <br />                           3 <br />                                                                                                  3 <br />                           2                                                                      2                              Troph                         20 <br />                           1                                                                      1            Troph <br />                                                                                                     E Schiz                                                    0 <br />                           0                                                                      0 <br />                            0.5 1    1.5      2 2.5 3      3.5   4                                 0      20      40     60      80       100 <br />                                           time (days)                                                           time (hours)                                 100                                                                                -Shld1 <br />                                                                                                                                                               80 <br />  C100 <br />    90 <br />                                                                      F1.82 <br />                                                                     dead par (%) -/+ Shld1 <br />  <br />  <br />  <br />  <br />                          80                                                                     1.6 <br />                                                                                                                                                                      60                            pre S-phase = 28.5 1h <br />   % total parasites <br />  <br />  <br />  <br />  <br />                          70                                                                     1.4 <br />                          60                                                                     1.2                                                                  40                           Cycle time = 42 0.5h <br />                          50                                                                       1 <br />                          40                                                                     0.8                                                                  20 <br />                          30                                                                     0.6 <br />                          20                                                                     0.4 <br />                                                                                                 0.2 <br />                                                                                                                                                                       0 <br />                          10                                                                                                                                           -15 -10       -5    0   5      10 15 20 25 30 35 40                  45 50 55 60 <br />                           0                                                                       0 <br />                            0       0.1       0.2    0.3     0.4                                       Syto+CM     Topro+CM      Topro+AcOr                                                                 time (hpi) <br />                                          Shld1 ( M) <br />  <br /> Fig. 5. Analysis Pcalp knockdown phenotypes.  B) Asynchronous cultures representative clone A7   C3 (B) grown (F) (E) <br /> 0.4 &#226;?&#174;M Shld1 monitored time flow cytometry. X axis B. (C) Growth Pcalp-GFP-FKBP (mean 3 clones, F) C3 (&#228;&#352;?) 3 days different Shld1 concentrations. (D E) Fine analysis Shld1 growth phenotype synchronized cultures A7 (D) C3 (E). Schizonts culture <br /> grown 0.2 &#226;?&#174;M Shld1 isolated allowed reinvade fresh RBCs presence (F) absence (E) Shld1. breaks indicate equal subculture <br /> event culture. X axis D E. (F) Viability measurements using different dyes measure dead parasites A7 (light) C3 (dark) cultures. ratio &#226;&#171;&#186;/&#226;&#171;&#185; 0.2 &#226;?&#174;M Shld1 plotted. Flow profiles Fig. S5. (G) Appearance A7 dead parasites (Topro3-positive) fluorescence microscopy. absence Shld1 unique dead species detected: trophozoites extraerythrocytic   RBC ghosts (b). (H) Giemsa-stained smears <br /> showed delay morphological transition ring trophozoite synchronized A7 cultures absence Shld1.   Representative cycles A7 (Upper) (Lower) Shld1. Cycle points acquired 30 min &#226;&#172;&#8225;6 days, samples fixed, DNA content analyzed flow cytometry <br /> (Fig. S3). Green, mature schizonts; red, S phase; black, pre-S phase (G1). cycle time shown distance 2 peaks schizonts; length pre-S <br /> phase time separating initial slope half maxima pre-S-phase S-phase peaks. Red vertical lines indicate S-phase maxima &#226;&#171;&#185;Shld1 visual <br /> comparison &#226;&#171;&#186;Shld1 culture. consistent delay start S phase detected calpain destabilized. <br />  <br />  <br />  <br />     Multiple efforts generate parasite clone lacking calpain                                                                                         calpain stability regulated small molecule Shld1. <br /> expression failed, strongly suggesting absence gene                                                                                      Destabilization Pcalp yielded growth defect 40&#226;&#8364;&#8220;60% 2 incompatible optimal viability blood stages. used                                                                                       cycles. measured delay entry S phase. Morphologically, <br /> 3 different techniques. double cross gene knockout                                                                                               transition ring trophozoite delayed. Concom- <br /> approach failed generate recombinants. Disruption single                                                                                            itantly detected increased number nonviable parasites. <br /> cross 5&#226;&#172;&#732; end endogenous locus, including control                                                                                     Peaks Pcalp transcription translation normal parasites homologous recombination, new approach.                                                                                            seen pre-S-phase stages (ring mid trophozoite), native promoter recapitulated did recombination result                                                                                         sudden drop levels detected. correlates nicely <br /> viable parasites. possible technical problems led                                                                                    observed knockdown phenotype. <br /> failure isolate gene-disrupted recombinants 2 proce-                                                                                            absence Shld1, fusion protein degraded, resulting <br /> dures.  designed approach, allelic replace-                                                                                       knockdown effect. remarkably scarce transcript barely <br /> ment, overcome limitation. readily able isolate                                                                                        detectable protein levels provided challenges detection parasites cross-overs upstream active cysteine codon,                                                                                      study calpain. normal level calpain low predicted frequency, vector contained synonymous                                                                                           quantify precisely extent protein diminution.  <br /> active site mutation. contrast, recombinants downstream                                                                                        FKBP fusions shown order magnitude nonsynonymous mutation vector used. Unlike                                                                                               range response Shld1 (ref. 13 R., unpublished results). 2 strategies, allelic replacement method generates recom-                                                                                       Assuming small number calpain molecules normal <br /> binants missense mutation vector. site                                                                                         Gaussian distribution cells, attribute death phenotype <br /> recombination informs gene essentiality. addition,                                                                                      cells  20% parasites) number importance active site residue informs Pcalp                                                                                           calpain molecules insufficient overcome efficiency activity necessary asexual cycle development.                                                                                                      degradation process.  cells die loss     studies insight essentiality help                                                                                    calpain function. rest cells produce just understand role enzyme cell. pursue                                                                                          calpain survive, defect pre-S-phase <br /> developed method, based FKBP degradation domain (12,                                                                                             development. total cell cycle time normal surviving <br /> 13), generate regulated knockdown Pcalp. fusing                                                                                              parasites, suggesting able catch  cycle <br /> GFP-FKBP tag endogenous locus generated clonal lines                                                                                            clock able proceed normal time,  <br /> 1558 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.0806926106                                                                                                                                                                                             Russo et al. <br />  Pcalp affects previous  Involvement calpain-like activity                              determined according time morphological analysis Giemsa staining. G1 3 S transitions suggested systems                                  RNA treated DNase (Gibco) checked PCR purity. Pcalp cDNA <br /> basis indirect evidence (14, 15). S-phase delay                               amplified step RT-PCR SuperScript (Invitrogen). RT-PCR <br />                                                                                                   products 2 independent reactions time point analyzed <br /> observed P. berghei parasites lacking 1 2 eEF1a genes,                                                                                                   ethidium bromide/agarose gels. Band intensity linear range. Primers case, total cell cycle prolonged (16). Given role                             Table S3. <br /> cell cycle progression, interesting enzyme nucleolus  R., O., D.E.G., unpublished data),                                Southern Blots. Southern blots, 1 &#226;?&#174;g DNA digested analyzed addition rRNA synthesis ribosome assembly, key                                      previously described (25). double cross integrations screened functions cell cycle regulation (17). conclusion, developing                             PacI/XmaI digestion probed using Pcalp ORF 5&#226;&#172;&#732; end. single cross method regulated knockdown Pcalp, begin define                                 integrations screened NsiI/SphI digestion probing Pcalp ORF 5&#226;&#172;&#732; cellular role develop interesting target                                 3&#226;&#172;&#732; end. <br /> antiparasitic chemotherapy. <br />                                                                                                   Western Blots. Protein analysis detect Pcalp immunoprecipitation Materials Methods                                                                             conducted described SI Materials Methods. <br /> Reagents. Polyclonal antiserum   19) raised rabbits Pcalp <br /> <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> 41&#226;&#8364;&#8220;54. used rabbit anti-GFP ab6556 (Abcam), mouse anti-GFP JL8                    Microscopy Techniques. Live parasites observed presence Hoechst <br /> (BD), rabbit anti-BiP (MR4), rabbit anti-plasmepsin II (18) antibodies.             33342 Topro3. Images collected Axioskop epifluorescence <br /> reagents purchased Sigma NEB unless indicated.                                microscope (Zeiss) described (25). <br />  <br /> Cell Cultures Transfection. P. falciparum (3D7) cultured previously                    Flow Cytometry. cell cycle analysis, highly synchronous 2% hematocrit cul- <br /> described (19). Parasite synchronization obtained using 5% <span id='am-6' about='obo:IMR_0200462' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="D-Sorbitol" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200462" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00794" datatype="xsd:string"></span>D-sorbitol</span>                      tures, 1% parasitemia, cultivated gently rocked Roboflasks (Corning) <br /> treatment (20), Percoll (21), magnetic separation (22). transfections, 160                ensure cell suspension. automated  controlled 3 timers &#226;?&#174;l 50% RBCs transfected electroporation (23) &#226;&#172;&#8225;100 &#226;?&#174;g purified                   composed 3 synchronized peristaltic pumps fraction collector, set  <br /> vector DNA infected 3D7 schizonts. 72&#226;&#8364;&#8220;90 h, 10 nM WR99210                     30 min, &#226;&#172;&#8225;150 &#226;?&#174;l sample culture extracted added medium. select integration, parasites cycled twice                   flask septum, sample fixed collected 4 &#194;&#176;C 96 drug (24). Double cross recombination selected described                    plate. subculture, fresh, prewarmed RBCs (2% hematocrit) added <br /> ref. 11.                                                                                          septum removing flask incubator. fixative <br />                                                                                                   solution (1.5 vol sample) 3.2% <span id='am-11' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span> 0.01% glutaraldehyde <br /> Sequence Phylogenetic Analysis. SI Materials Methods.                                 PBS. 5min permeabilization 0.1% Triton X-100 15 min                                                                                                   incubation &#226;&#172;&#8225;75 &#226;?&#174;g/ml RNase  cell DNA stained 0.5&#226;&#8364;&#8220;1 &#226;?&#174;M Topro3. <br /> Construction Vectors. Genomic DNAs extracted P. falciparum                        analysis conducted BD FACSCanto flow cytometer, monitoring <br /> using Blood Mini Kit (Qiagen). Primers restriction sites listed Table S3.            fluorescence profiles infected RBC (Fig. S3). cloning steps confirmed sequencing. gene knockout strategy,                      viability analysis, used following dyes pairs: 1.25 &#226;?&#174;g/ml Cell plasmid pHHTTK (11) modified inserting HcRed coding region                         tracker-green (CM-green), 2.5 &#226;?&#174;M Syto59, 0.5 &#226;?&#174;M Topro3, 0.4 &#226;?&#174;g/ml AcOr <br /> upstream hDHFR. backbone, cloned 5&#226;&#172;&#732; 3&#226;&#172;&#732; ends Pcalp                  (Molecular Probes). Metabolically active cells incorporated CM-green ORF. constructs gene disruption single cross                       15-min incubation 37 &#194;&#176;C. analyzed flow cytometry calpain N-terminal coding regions (1.3 1.6 kb) 733 bp 5&#226;&#172;&#732;                   presence DNA dyes, cell-permeant Syto59, cell-impermeant Topro3. UTR PCR amplified genomic DNA cloned pPM2GT (25).                            viability measurement based use 2 DNA dyes time, <br /> vectors allelic replacement generated cloning 3.8 kb Pcalp 3&#226;&#172;&#732; ORF                  AcOr Topro3. Control dead cells produced fixing culture. pPM2GT (25). Mutagenesis allelic replacement vectors                    analysis conducted recording green far-red fluorescence profiles QuikChange kit (Stratagene). vectors 3&#226;&#172;&#732; tagging constructed                   infected RBCs (Fig. S5). <br /> cloning 1.1 kb 3&#226;&#172;&#732; Pcalp ORF pPM2GT (25) generate Pcalp-GFP replacing GFP GFP-FKBP generate Pcalp-GFP-FKBP. FKBP domain                       ACKNOWLEDGMENTS. thank Sanger Broad Institutes DNA <br /> amplified pBMN YFP-L106P (12).                                                               sequences; Jacobus Pharmaceutical WR99210; T. Wandless (Stanford Uni- <br />   <br /> </body></html>