<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />  <br />  <br />                            Systematic genetic analysis virulence human fungal <br />                            pathogen Cryptococcus neoformans <br />  <br />                             SUMMARY <br />                                 fungus Cryptococcus neoformans leading cause mortality morbidity HIV- <br />                                 infected individuals. utilized completed genome sequence optimized methods                                 homologous DNA replacement using high-velocity particle bombardment engineer 1,201 gene <br />                                 knockout mutants. screened resource vivo proliferation murine lung tissue                                 vitro recognized virulence attributes &#226;&#8364;&#8221; <span id='am-6' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span>polysaccharide</span> capsule formation, <br />                                 melanization, growth body temperature. identified dozens previously uncharacterized <br />                                 genes affect known attributes 40 infectivity mutants obvious defects                                 traits. mutants affect predicted regulatory factors, secreted proteins, immune- <br />                                 related factors, represent powerful tools elucidating novel virulence mechanisms.                                 particular, GATA family transcription factor inhibits phagocytosis murine <br />                                 macrophages independently capsule, indicating previously unknown mechanism innate <br />                                 immune modulation. <br />  <br />  <br />                            INTRODUCTION <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Systematic profiling gene deletion strain collections essential                                               comprehensive study biological processes nonpathogenic yeast Saccharomyces <br />                                               cerevisiae (Bader et al., 2003). Likewise, forward genetic screens large mutant collections <br />                                               bacterial pathogens extremely fruitful identifying virulence pathways                                               prokaryotic organisms (Saenz Dehio, 2005). contrast, study fungal pathogenesis <br />                                               historically hindered relative lack genetic tools techniques                                               eukaryotic species. release complete genome sequences numerous fungal <br />                                               pathogens (Jones et al., 2004; Loftus et al., 2005; Nierman et al., 2005),                                               unprecedented opportunities create genome-scale resources apply systematic methods <br />                                               study fungal virulence. <br />  <br />  <br />  <br />                            4 Correspondence, hiten@biochem.ucsf.edu tel (415) 514-0594, fax (415) 502-4315. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Liu et al.                                                                                                 Page 2 <br />  <br />  <br />                                            Cryptococcus neoformans common cause systemic fungal infections people <br />                                            infected HIV (Bicanic Harrison, 2004). primary pulmonary infections                                            typically mildly symptomatic self-resolving patients intact immune systems, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            infections commonly disseminate bloodstream infect meninges HIV- <br />                                            positive patients. sub-Saharan Africa Southeast Asia, 80% world&#226;&#8364;&#8482;s <br />                                            HIV-infected population lives, studies indicate 10&#226;&#8364;&#8220;40% HIV-related deaths caused <br />                                            cryptococcal meningitis (Corbett et al., 2002; French et al., 2002). high cost limited <br />                                            availability antifungal drugs prohibit effective medical treatment areas <br />                                            (Warnock, 2006) mortality rates parts sub-Saharan African approach 100% (Bekondi <br />                                            et al., 2006). <br />  <br />                                            established mediators C. neoformans virulence: polysaccharide <br />                                            capsule, cell wall-associated melanin, ability grow human body temperature.                                            capsule shown anti-phagocytic immunomodulatory properties (Del <br />                                            Poeta, 2004; Monari et al., 2006). Melanin believed protect fungus broad range <br />                                            toxic insults, environment host (Nosanchuk Casadevall, 2003). <br />  <br />                                            Molecular studies C. neoformans virulence pathways focused exclusively                                            regulation capsule melanization (Table S1). Attempts identify novel virulence <br />                                            mechanisms using random insertional mutagenesis hampered biases                                            insertion sites, chromosomal rearrangements, relatively low ORF hit rates (Idnurm et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2004;Nelson et al., 2001). release complete C. neoformans genome sequence                                            makes possible use directed gene knockout mutants screen novel virulence <br />                                            mechanisms. end, constructed large library 1,201 signature-tagged, targeted <br />                                            gene deletion strains.  insights gained systematic profiling                                            library murine infection model vitro assays virulence-associated <br />                                            attributes (Figure 1). detailed analysis key transcriptional regulator                                            pointed existence unsuspected capsule-independent mechanism inhibition <br />                                            phagocytosis. <br />  <br />                                RESULTS <br />                                Generation signature-tagged C. neoformans gene deletion collection <br />                                            Approximately 1,500 genes C. neoformans targeted deletion homologous <br />                                            recombination. majority (~900) selected absence obvious homologs <br />                                            nonpathogenic fungus S. cerevisiae. remainder encode proteins sequence motifs <br />                                            implicating broad variety cellular processes. targets chosen <br />                                            prior availability detailed gene models based BLAST-based annotations <br />                                            (Chow et al., 2007), bias genes C. neoformans-specific. utilized <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            fusion PCR create deletion constructs high-throughput manner optimized biolistic <br />                                            transformation efficient recombination. reasons described  mutant strain <br />                                            contains 48 unique signature tag DNA sequences adjacent deletion marker.                                            resulting set 1,201 verified mutants represents deletions 1,180 genes  cases, <br />                                            different domains gene targeted separately). comparison, genome                                            C. neoformans predicted contain ~6,500 genes (Loftus et al., 2005), ~190 targeted <br />                                            gene deletions C. neoformans previously reported literature (Table S1). <br />                                            deletion collection currently available restriction Fungal Genetic <br />                                            Stock Center (www.fgsc.org) ATCC (www.atcc.org). <br />  <br />                                Quantitative screen infectivity mutants murine inhalation model disease <br />                                            identify C. neoformans genes involved infectivity, adopted signature-tagged <br />                                            mutagenesis (STM), strategy enables high-throughput, parallel analysis pathogen <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                     Page 3 <br />  <br />  <br />                                            mutants experimental infections (Saenz Dehio, 2005). method, individual mutants <br />                                            marked genomes unique DNA sequence (termed signature tag barcode). <br />                                            allows pool mutants tested single animal infection comparing NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            relative representation mutant pool selection host.                                            practice, accomplished isolating DNA mutant pathogen pools                                            infection quantifying abundance tag sample. <br />  <br />                                            developed robust, quantitative STM assay utilizing previously established mouse model <br />                                            pulmonary cryptococcal infection (Cox et al., 2000). C. neoformans deletion mutants                                            grown independently pooled groups 48 intranasal infection mice. <br />                                            Infections allowed progress mice displayed signs significant morbidity <br />                                            (~5&#226;&#8364;&#8220;6 weeks), time mice sacrificed C. neoformans cells recovered <br />                                            lung tissue. Quantitative PCR using tag-specific primers common <br />                                            primer gene deletion cassette used measure relative abundance mutant <br />                                            input recovered pools. ratio values ([recovered] / [input])                                            calculated log2 scale termed STM score  Experimental Procedures). shown <br />                                            Figure 2A, STM scores demonstrated remarkable consistency mice infected <br />                                            pool signature-tagged mutants. total, 1,100 deletion mutants screened <br />                                            using STM assay. <br />  <br />                                            determined STM score given mutant affected composition NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants pool. 104 mutants removed original pools sorted new <br />                                            pools containing different mutants. Infections repeated STM scores recalculated <br />                                            mutant. original STM scores highly correlated pooled STM <br />                                            scores (R2=0.84), demonstrating STM scores largely independent identities <br />                                            competing mutants highly reproducible pool pool (Figure 2B). <br />  <br />                                            additional control reproducibility assay, constructed 43 independent <br />                                            disruptions gene SXI1, required mating dispensable virulence (Hull <br />                                            et al., 2004); mutant marked different signature tag. 43 sxi1&#206;&#8221; mutant <br />                                            strains tested total 293 times STM screen  expected,                                            distribution STM scores sxi1&#206;&#8221; strains centered near zero (median=0.1)                                            showed tight range (SD=0.5) (Figure 2C). distribution STM scores deletion <br />                                            library centered zero (median=0.0);  case,                                            distribution scores broader (SD=3.3), suggesting wide range fitness                                            assay. <br />  <br />                                STM phenotypes correlate virulence <br />                                            STM screens designed measure relative competitive fitness mutants mixed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            infection.  term &#226;&#8364;&#339;infectivity&#226;&#8364;&#65533; context refers ability proliferate/survive <br />                                            host tissue. &#226;&#8364;&#339;Virulence,&#226;&#8364;&#65533; hand, classically defined ability                                            individual strain proliferate, cause disease  assess ability                                            STM screen phenotypes predict virulence defects, performed additional tests. <br />  <br />                                             asked previously identified virulence mutants included deletion <br />                                            collection detected screen. 24 previously-reported virulence mutants, 21 <br />                                            displayed STM scores significantly zero, ranging &#226;&#710;&#8217;12.8 &#226;&#710;&#8217;1.9 (labeled x- <br />                                            axis Figure 2C Table S2). published virulence mutants, lac1&#206;&#8221; ure1&#206;&#8221;,                                            normal STM scores. mutants,  known replicate normally mouse lungs <br />                                            late defects dissemination (Noverr et al., 2004; Olszewski et al., 2004);                                            mutants anticipated defects screen. final virulence mutant, <br />                                            scp1&#206;&#8221;, retested assay using different pool mutants, significantly lower <br />                                            STM score detected (&#226;&#710;&#8217;3.6 vs. &#226;&#710;&#8217;0.8). results demonstrate robust sensitivity  <br />  <br />                                                     Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 4 <br />  <br />  <br />                                            STM screen virulence genes known affect propagation pulmonary tissue (21 mutants <br />                                            identified 22 mutants known lung defects). Conversely, sxi1&#206;&#8221; STM data described <br />                                            did yield single negative outlier seven mutants collection containing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            deletions genes known dispensable virulence normal STM scores suggesting <br />                                            screen specificity infectivity defects approaching 100% (Figure 2C, Table S2). <br />  <br />                                            second test, conducted classical time endpoint survival analyses seven <br />                                            previously-uncharacterized mutants wide range STM scores. experiments, <br />                                            virulence directly measured determining rate mice infected single <br />                                            strain C. neoformans reach defined disease endpoint  Supplementary Materials). <br />                                            Monotypic intranasal infections 8&#226;&#8364;&#8220;10 mice mutant wild-type control                                            performed time significant morbidity assessed mouse. Survival curves <br />                                            plotted mutant strains compared wild-type control. mutants <br />                                            STM scores close zero indistinguishable wild-type survival curves <br />                                            (Figure 2D). mutants STM scores significantly zero attenuated                                            virulence (Figure 2E) degree attenuation closely correlated magnitude                                            STM defect. Interestingly, mutant STM score significantly zero <br />                                            (zap103&#206;&#8221;) hypervirulent (Figure 2E).  studies suggest STM assay <br />                                            sensitive specific measure infectivity pulmonary infection model <br />                                            cryptococcosis, altered infectivity correlates altered virulence. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Based results, chose STM scores &lt; &#226;&#710;&#8217;2.5 &gt; +2.5 cutoff points                                            reduced infectivity increased infectivity, respectively. Application criteria                                            data yielded 164 (15%) mutants category 33 (3%)  <br />  <br />                                Systematic vitro phenotyping deletion collection identifies mutants defective                                known virulence attributes <br />                                            complement vivo STM screen, systematically assayed deletion collection                                            vitro known virulence-associated phenotypes: 1) growth minimal medium 37&#194;&#176; <br />                                            C, 2) melanization, 3) synthesis polysaccharide capsule. <br />  <br />                                            high-throughput, 96 format assay developed measure growth mutant <br />                                            synthetic minimal medium (YNB) 37&#194;&#176;C. expected, large overlap existed                                            mutants displayed poor growth 37&#194;&#176;C mutants STM defects (P=8.2E-30) (Figure <br />                                            2F). Using hypergeometric distribution analysis, determined mutants displayed <br />                                            growth scores greater &#226;&#710;&#8217;0.5 did significant enrichment mutants                                            virulence phenotypes (P &gt; 0.01) (Figure S1);  used growth score &#226;&#710;&#8217;0.5 cutoff <br />                                            determining mutants categorized having significant growth defects.                                            mutants displayed severe growth defects STM defects. attributed portion <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            inconsistency fact minimal medium does perfectly mimic nutritional <br />                                            environment host. example, mutants peroxisome function (pex1&#206;&#8221;                                            pex6&#206;&#8221;) shown previously normal virulence despite unable grow                                            YNB glucose sole carbon source (Idnurm et al., 2007). <br />  <br />                                            identify mutants defective melanin synthesis, grew deletion collection agar <br />                                            plates containing <span id='am-62' about='obo:IMR_0200290' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00355" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0200290" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="L-Dopa" datatype="xsd:string"></span>L-DOPA</span>  diphenolic substrate). Wild-type cells turned dark brown                                            2&#226;&#8364;&#8220;3 days synthesized accumulated melanin. Thirty deletion mutants <br />                                            displayed decreased levels melanization compared wild-type, including known melanin <br />                                            mutants lac1&#206;&#8221;, cac1&#206;&#8221;, pka1&#206;&#8221;.(Alspaugh et al., 2002; D&apos;Souza et al., 2001; Noverr <br />                                            et al., 2004). noted  lac1&#206;&#8221; mutant  lacks laccase primarily responsible <br />                                            melanin synthesis) late defects dissemination lungs grows normally                                            lung tissue (Noverr et al., 2004);  identified STM screen. Likewise, <br />                                            mutant containing deletion second C. neoformans laccase gene, LAC2,  <br />  <br />                                                    Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                               Page 5 <br />  <br />  <br />                                           double knockout mutant, lac1&#206;&#8221; lac2&#206;&#8221;, did display STM phenotypes (Table S2).                                           consistent melanization playing role dissemination, lung infectivity.                                            surprised note overall significant overlap mutants NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           melanization defects STM defects lungs (Figure 2F). association <br />                                           remained significant (P =1.4E-11) excluding mutants growth defects                                           analysis. Given laccases dispensable lung infection, additional defects                                           poor pigment production proposed subset melanin-defective mutants                                           account lung infectivity defects. deficits affect regulated pathways                                           function independently redundantly melanization promote lung proliferation. <br />  <br />                                           identify mutants defective polysaccharide capsule formation, screened mutants <br />                                           dry colony appearance rich medium (YPAD) agar plates. dry colony morphology <br />                                           characteristic known C. neoformans mutants absent greatly reduced amounts <br />                                           capsule (Griffith et al., 2004; Moyrand et al., 2004; Moyrand Janbon, 2004). Using                                           simple assay, identified potentially capsule-defective mutants including strains <br />                                           (cap59&#206;&#8221;, cap64&#206;&#8221;, cap60&#206;&#8221;, cap10&#206;&#8221;, cas35&#206;&#8221;, nrg1&#206;&#8221;) contain deletions genes <br />                                           previously implicated capsule synthesis (Cramer et al., 2006; Moyrand et al., 2004; <br />                                           Okabayashi et al., 2007). addition, PBX1 PBX2 identified screen; <br />                                           deletion PBX1 PBX2 affects fidelity polysaccharide synthesis detailed <br />                                           analysis described (Liu et al., 2007). expected, mutants dry <br />                                           colony morphologies STM defects, consistent established importance <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           capsule causing disease (Figure 2F). <br />  <br />                                           Certain mutants, pka1&#206;&#8221; cac1&#206;&#8221;, known produce reduced structurally- <br />                                           altered capsules (Alspaugh et al., 2002; D&apos;Souza et al., 2001), dry colony <br />                                           morphologies. identify class mutant, tested subset mutants collection <br />                                           ability induce capsule formation using India ink assay. stain yields                                           characteristic halo cells enlarged capsules  note analysis does rule                                           subtle capsule structure defects detectable NMR analysis). <br />                                           tested 82 mutants displayed melanin STM defects normal colony <br />                                           morphology. mutant, containing deletion GAT201  encodes GATA- <br />                                           family zinc finger DNA-binding transcriptional regulator, Figure S2), produced visible <br />                                           capsule India ink staining capsule-inducing conditions (Figure 4C).                                           investigation role GAT201 C. neoformans pathogenesis described  <br />  <br />                                           total, 164 mutants STM defects, 85 defects growth 37&#194;&#176;C, <br />                                           polysaccharide capsule formation, melanization. remaining 79 mutants appeared <br />                                           normal vitro assays (Figure 2F). verify observed phenotypes                                           linked targeted deletions, reconstructed 107 deletion strains retested NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           original phenotype: 5 5 reconstructed strains reproduced original capsule phenotype; 33 <br />                                           35 reproduced original melanin phenotype; 53 67 reproduced STM score <br />                                           significantly zero; 8 11 reproduced STM score significantly zero (Figure <br />                                           S3). results overall indicate strong linkage targeted mutation observed <br />                                           phenotypes. Slightly higher variability reproduction STM phenotypes reflect                                           higher variability assay potential larger number unlinked mutations                                           affect phenotype. <br />  <br />                                Novel melanin capsule genes <br />                                           Thirty novel genes regulate melanization novel genes regulate capsule <br />                                           formation identified verified knockout reconstruction retesting (Figure 3, <br />                                           Table 1). described previously (Chow et al., 2007), gene names assigned based                                           homology known S. cerevisiae genes available. melanin genes capsule <br />                                           gene named using approach  instead, designated MLN1&#226;&#8364;&#8220;MLN4  <br />                                                   Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 6 <br />  <br />  <br />                                            CPL1 respectively. melanin capsule genes identified screens implicate                                            number novel pathways regulation virulence attributes. example, <br />                                            mutants lack transcription factor Rim101 predicted upstream regulator Rim20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            display melanization defects (Figure S3). Similarly, deletion components Set3 <br />                                            deacetylase complex resulted significantly larger capsules compared wild-type                                            defects melanization (Figure S3&#226;&#8364;&#8220;S4). <br />  <br />                                Novel infectivity genes <br />                                            particularly interesting category mutants identified screens defects lung <br />                                            infectivity normal growth, capsule, melanin. mutants                                            verified knockout reconstruction retesting using STM assay (Figure 3, Table 2). <br />                                             gene names assigned based homology known S. cerevisiae genes                                            available. unnamed genes designated LIV1&#226;&#8364;&#8220;LIV15 (Lung InfectiVity). <br />  <br />                                            addition capsule, melanin, growth, number studied factors                                            implicated C. neoformans virulence based genetic studies. include urease, <br />                                            alternative oxidase, phospholipase, reductive <span id='am-52' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>iron</span> uptake, glucosylceramide synthesis,                                            resistance nitrogen oxygen species (Akhter et al., 2003; Cox et al., 2003; Cox et al., <br />                                            2001; Cox et al., 2000; Jesus-Berrios et al., 2003; Jung et al., 2008; Shea et al., 2006).                                            cases, mutants pathways display growth defects certain media. example, <br />                                            mutants lacking phospholipase gene (PLB1) sensitive SDS (Siafakas et al., 2007), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mutants lacking flavohemoglobin gene (FHB1) sensitive acidified nitrite,                                            mutants lacking alternative oxidase gene (AOX1) sensitive potassium cyanide (KCN). <br />                                             order identify potential roles infectivity genes, systematically tested <br />                                            40 mutants described growth media conditions urease <br />                                            activity using enzymatic assay. analysis revealed mutants sensitivity                                            acidified nitrite, sensitivity SDS, sensitivity KCN (Table 2, Figure <br />                                            S5&#226;&#8364;&#8220;S7). contrast, mutants displayed decreased growth pH 4 media (Figure <br />                                            S5) low ferrous iron conditions (Figure S8) mutants displayed defect <br />                                            urease activity (Figures S9). <br />  <br />                                            mutants displayed increased sensitivity acidified nitrite contain deletions                                            DNA repair genes (rad54&#206;&#8221; rad23&#206;&#8221;), suggesting unrepaired DNA damage,                                            free radical attack phagocytes, deleterious infection (Figure S5). Notably,                                            mutant containing deletion LIV8, encodes RGS (Regulator G-protein <br />                                            Signaling) domain, exquisitely sensitive acidified nitrite, SDS, KCN (Figures S5&#226;&#8364;&#8220; <br />                                            S7). note media assays relatively nonspecific, sensitivities observed <br />                                            explain virulence defects particular mutants. example, mutant <br />                                            containing deletion virulence gene SRE1, shown involved NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ergosterol biosynthesis (Chang et al., 2007; Chun et al., 2007), displayed sensitivity                                            SDS KCN (Figures S6&#226;&#8364;&#8220;S7). <br />  <br />                                            Nonetheless, 33 mutants did display phenotypes vitro assays <br />                                            represent genes strongest candidates novel C. neoformans virulence factors. <br />                                            Seven genes category encode predicted sequence-specific transcription factors.                                            genes encode factors likely involved chromatin regulation. particular note,                                            genes encode factors strong homology proteins previously implicated pathogenesis <br />                                            regulation human immune response (Figure 3, Table 2). LIV5 encodes protein <br />                                            homologous required pathogenesis plant fungal pathogen Magnaporthe <br />                                            grisea (Fujimoto et al., 2002). LIV7 encodes putative secreted protein homologous                                            human protein induced <span id='am-27' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-28' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span>lipopolysaccharide</span>. Finally, LIV6 encodes protein highly <br />                                            homologous human T-cell immunomodulatory protein (TIP). Secreted TIP                                            shown directly regulate cytokine production T-cells (Fiscella et al., 2003). Protein <br />  <br />                                                    Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                   Page 7 <br />  <br />  <br />                                            structure prediction indicated Liv6 contains FG-GAP repeats, commonly                                            alpha integrins, signal <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> sequence, suggesting secreted <br />                                            (Figure S2). Deletion LIV5 LIV6 confirmed result virulence defects NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            survival curve analysis (Figure 2E, Figure S10). <br />  <br />                                            identified mutants reproducibly showed significant increases STM assay. <br />                                            mutants confirmed knockout reconstruction retesting display enhanced <br />                                            colonization lung tissue (Table 2). include deletions genes encoding members <br />                                            Rho signaling pathways (RGD1 RHO104), kinases (RCK2 KSP1),                                            transcription factor (ZAP103). unnamed gene designated LIV16. Deletions                                            ZAP103 RGD1 confirmed result increased virulence survival curve analysis <br />                                            (Figure 2E, Figure S10). <br />  <br />                                Gat201: key regulator virulence capsule-independent anti-phagocytic mechanisms <br />                                            mutants normal growth 37&#194;&#176;C, strain containing deletion GAT201 <br />                                            exhibited largest decrease STM screen (STM score = &#226;&#710;&#8217;18.3).  gat201&#206;&#8221; <br />                                            mutant strain avirulent monotypic infection mice using standard inoculum (Figure <br />                                            4A). mutant chosen investigation. <br />  <br />                                            described  capsules gat201&#206;&#8221; cells visualized India ink staining <br />                                            capsule-inducing conditions (Figure 4B).  sensitive assay using NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            monoclonal antibody stain main capsular polysaccharide, glucoronoxylomannan, <br />                                            detected capsule polysaccharide cell surface gat201&#206;&#8221; cells (Figure 5B). data <br />                                            suggest GAT201 required production basal levels capsule polysaccharide                                            needed proper induction large capsules. gat201&#206;&#8221; cells hypermelanized                                            grown L-DOPA plates (Figure 4C). Previously-identified hypermelanized mutants                                            associated increased decreased virulence display additional <br />                                            phenotypes altered capsule production (Bahn et al., 2005; Wang et al., 2004). Similarly, <br />                                            studies indicate GAT201 participates directly indirectly regulation capsule <br />                                            induction melanization. <br />  <br />                                            Consistent notion, overexpression GAT201 wild-type cells using promoter <br />                                            constitutive ribosomal protein gene (pRPL2b) sufficient induce large capsule <br />                                            formation, normally non-inducing growth conditions (Figure 4B, Figure S11).                                            shown Figure 4C, pRPL2b::GAT201 strain displayed slight melanization defect. <br />                                            pRPL2b::GAT201 strain hypervirulent mice monotypic intranasal infection <br />                                            (Figure 4A), demonstrating overexpression GAT201 sufficient activate key <br />                                            virulence pathways. reconstructed gat201&#206;&#8221; strain recapitulated phenotypes original <br />                                            gat201&#206;&#8221; strain assays (Figure 4A&#226;&#8364;&#8220;4C). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            genome, microarray-based expression profiling pRPL2b::GAT201 strain <br />                                            identified 543 genes significantly upregulated 24 genes significantly <br />                                            downregulated compared wild-type strain (Figure 4D). Consistent enhanced <br />                                            nutritional metabolic requirements necessary ectopic polysaccharide capsule <br />                                            formation, induced set contained substantial number genes encoding proteins involved <br />                                            <span id='am-10' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span>carbohydrate</span> regulation, including sugar transporters (40 genes), glycosyl hydrolases (8 <br />                                            genes), glycosyl transferases (5 genes). addition, genes encode DNA- <br />                                            binding transcription factors upregulated pRPL2b::GAT201 strain, suggesting                                            Gat201 exerts portion effects activating expression transcription <br />                                            factors. Using previously described annotation database (Chow et al., 2007), identified <br />                                            C. neoformans genes homologs sequenced ascomycetes                                            sequenced basidiomycetes. 29% Gat201-upregulated genes (159 genes) fall                                            C. neoformans-specific gene class, represents significant enrichment general <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                              Page 8 <br />  <br />  <br />                                         representation microarray (P =1.2E-35). Sixteen genes predicted produce <br />                                         proteins signal sequences transmembrane domain; potential secreted effectors <br />                                         include members unique gene family (CNF2230, CNF2240, CNF2250, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         CND5620). <br />  <br />                                         probe role Gat201 virulence, examined interaction gat201&#206;&#8221; cells <br />                                         RAW264.7 mouse macrophages. Phagocytic cells play central role host immune <br />                                         response C. neoformans, ability C. neoformans avoid phagocytosis                                         killing cells crucial pathogenesis (Del Poeta, 2004). contrast fungi, <br />                                         C. albicans S. cerevisiae, wild-type C. neoformans undergoes negligible <br />                                         phagocytosis cultured macrophages absence opsonizing antibodies complement <br />                                         (Kozel Mastroianni, 1976). anti-phagocytic properties C. neoformans largely <br />                                         attributed polysaccharide capsule (Del Poeta, 2004). recapitulated results <br />                                         wild-type C. neoformans, observing robust induction capsule low rate <br />                                         phagocytosis 24 hours culturing RAW264.7 macrophages (&lt;2%                                         macrophages contained internalized wild-type C. neoformans, Figures 5A&#226;&#8364;&#8220;B, Figure S12). <br />                                         Deletion capsule-associated genes CAP10, CAP60, CAP64 resulted approximately <br />                                          seven-fold increase number macrophages containing C. neoformans (4&#226;&#8364;&#8220; <br />                                         14%). previously reported (Moyrand Janbon, 2004), cap10&#206;&#8221;, cap60&#206;&#8221;, cap64&#206;&#8221; <br />                                         mutants detectable capsular material, antibody India ink staining (Figure <br />                                         5C). Unexpectedly, dramatic increase phagocytosis seen NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         RAW264.7 macrophages cultured gat201&#206;&#8221; cells; 66% macrophages contained <br />                                         internalized C. neoformans 24 hours (Figure 5A&#226;&#8364;&#8220;B). gat201&#206;&#8221; mutants                                         phagocytosed readily CAP gene mutants, displayed weaker capsule defect,                                         phagocytosis phenotype gat201&#206;&#8221; cells easily explained hypocapsular <br />                                         phenotype. examine role GAT201 absence capsule, asked double <br />                                         mutants lacking GAT201 CAP genes phagocytosed greater extent <br />                                         CAP gene single mutants. Strikingly, case: high levels                                         phagocytosis seen gat201&#206;&#8221; cap10&#206;&#8221; (47%), gat201&#206;&#8221; cap60&#206;&#8221; (57%), gat201&#206;&#8221; <br />                                         cap64&#206;&#8221; (52%) double mutant strains (Figure 5B). Unlike gat201&#206;&#8221; single mutants, double <br />                                         mutant strains detectable capsules antibody India ink staining displayed <br />                                         clumpy phenotypes characteristic cap10&#206;&#8221;, cap60&#206;&#8221;, cap64&#206;&#8221; single mutants (Figure 5C), <br />                                         indicating capsule basis difference phagocytosis                                         gat201&#206;&#8221; CAP mutants. <br />  <br />                                DISCUSSION <br />                                         incidence invasive fungal infections humans risen dramatically                                         decades (Warnock, 2006), understanding molecular mechanisms relevant NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         fungal pathogenesis remains limited. analysis C. neoformans gene deletion <br />                                         collection described demonstrates potential harnessing genome sequencing efforts <br />                                         implement systematic approaches identifying dissecting fungal virulence <br />                                         mechanisms. biolistic gene knockout methods C. neoformans introduced                                         1993, ~190 targeted deletions reported literature (Table S1 summarizes                                         phenotypes previously-described gene deletions). report construction analysis <br />                                         1,201 targeted gene deletion strains, significantly extending genetic resources                                         pathogen. Systematic profiling deletion collection yielded number insights C. <br />                                         neoformans virulence: <br />                                             1.   Melanization: identifying analyzing numerous novel factors required                                                  melanization, obtained strong evidence virulence melanization  <br />                                                  regulated, lung proliferation defects observed large subset melanin <br />                                                  mutants result defects poor melanization. <br />  <br />  <br />                                                  Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                   Page 9 <br />  <br />  <br />                                                2.   Polysaccharide capsule: confirmed role polysaccharide capsule                                                     virulence. screen identified novel hypocapsular mutants,                                                     displayed defects infection. Gat201, following Nrg1 (Cramer et al., 2006), NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     second DNA-binding transcription factor regulate capsule formation. Gat201 <br />                                                     normally limiting synthesis virulence factor overexpression                                                     GAT201 sufficient induce capsule synthesis classically non-inducing <br />                                                     conditions. Gat201 controls melanization, inhibition phagocytosis,                                                     virulence, appears central point regulation cryptococcal pathogenesis. <br />                                                3.   Capsule-independent anti-phagocytic pathways: anti-phagocytic properties                                                     C. neoformans largely attributed polysaccharide capsule (Del Poeta, <br />                                                     2004). analysis Gat201 transcriptional regulator identified previously <br />                                                     unsuspected capsule-independent mechanism anti-phagocytosis deletion                                                     GAT201, wild-type capsule-deficient genetic backgrounds, resulted                                                     dramatic increase phagocytosis untreated C. neoformans macrophages.                                                     CAP mutants produce minor capsule polysaccharide, galactoxylomannan <br />                                                     (GalXM), detectable culture supernatant (Vaishnav et al., 1998).                                                     GAT201 functions controlling synthesis GalXM, protein factors,                                                     lipids, effect seen gat201&#206;&#8221; mutants lack anti- <br />                                                     phagocytic factor presence pro-phagocytic factor remain unclear. <br />                                                     Transcriptional profiling GAT201 overexpression strain detected significant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     enrichment (P =1.2E-35) transcripts C. neoformans genes                                                     sequenced fungi including number predicted secreted proteins                                                     contribute C. neoformans-specific virulence pathways, including ones <br />                                                     involved inhibiting phagocytosis macrophages. Finally, 25 genes encoding <br />                                                     DNA-binding transcription factors upregulated overexpression strain, <br />                                                     suggesting Gat201 control large, virulence-associated transcriptional <br />                                                     regulatory circuit. <br />                                                4.   Potential novel virulence pathways: Molecular studies C. neoformans virulence <br />                                                     focused primarily regulation capsule melanization.                                                     discovery 48 previously undescribed genes (40 mutants decreased infectivity, <br />                                                     8 mutants increased infectivity), affect proliferation lung tissue,                                                     capsule, growth melanin, strongly suggests existence additional unknown <br />                                                     virulence mechanisms provides tools identification. principle,                                                     genes code 1) regulators novel virulence pathways 2) direct effectors <br />                                                     virulence. Potential regulators include seven predicted transcription factors,                                                     targets presumably include pathogenesis factors. Possible direct effectors virulence <br />                                                     include factors homology proteins previously linked pathogenesis <br />                                                     immune modulation (Liv5, Liv6 Liv7). substantial number proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     required infectivity identified function inferred                                                     sequence relationships, indicating novel molecular functions assigned <br />                                                     specific proteins organism likely play role fungal virulence. Taken <br />                                                      identification categorization infectivity mutants large deletion <br />                                                     library highlight advantages using multipronged, systematic genetic analysis <br />                                                     dissect pathogenic mechanisms. <br />  <br />                                EXPERIMENTAL PROCEDURES <br />                                Generation gene deletion strains <br />                                            Cryptococcus neoformans var. grubii serotype clinical strain H99  gift J. Lodge)                                            used wild-type. previously described strategy gene deletion (Chun et al., 2007) utilizing <br />                                            overlap fusion PCR biolistic techniques used generate knockout strains   <br />  <br />                                                     Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                 Page 10 <br />  <br />  <br />                                             Supplementary Materials). Details deletion collection including targeted regions                                             mutant Table S4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Signature-tagged mutagenesis <br />                                             Groups 48 C. neoformans deletion strains individually grown liquid YPAD                                             combined single pool. pooled inocula used intranasally infect mice <br />                                              aliquots inoculum (~5 &#195;&#8212; 105 cells) plated Sabouraud agar <br />                                             plates containing 40 &#194;&#181;g/mL gentamicin 50 &#194;&#181;g/mL carbenicillin. Genomic DNA (Input <br />                                             DNA) purified independently plate using hexadecyltrimethyl ammonium <br />                                             bromide (CTAB, Sigma). Infected mice monitored sacrificed disease endpoint. <br />                                             Lungs removed homogenized 5 mL sterile PBS. Serial dilutions organ <br />                                             sample plated Sabouraud agar plates containing 40 &#194;&#181;g/mL gentamicin 50 &#194;&#181;g/mL <br />                                             carbenicillin. Genomic DNA (Output DNA) set lungs purified using CTAB. <br />  <br />                                             amounts signature tag genomic DNA prep measured using 48 <br />                                             simultaneous quantitative PCR reactions. reaction identical different <br />                                             tag-specific primer (Table S7). threshold cycle (Ct) determined reaction                                             STM score calculated mutant using variation 2&#226;&#710;&#8217;&#206;&#8221;&#206;&#8221;Ct method                                             quantitation analysis (Livak Schmittgen, 2001). STM scores mice                                             averaged determine final STM score mutant. detailed description STM <br />                                             method Supplementary Materials. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                vitro phenotyping <br />                                             Polysaccharide capsule formation, melanin synthesis, urease activity, growth acidified <br />                                             nitrite, SDS, potassium cyanide, pH 4, low iron medium assayed using standard <br />                                             conditions  Supplementary Materials). Growth minimal medium quantified                                             measuring change optical density 600 nm (OD600) 24 hours growth using                                             optical plate reader (Molecular Devices). Growth scores normalized median growth <br />                                             groups 48  Supplementary Materials). <br />  <br />                                Statistical analysis <br />                                             Significances overlaps STM, capsule, melanin phenotypes measured <br />                                             using hypergeometric test. Correlation STM growth defects analyzed                                             measuring enrichment STM defects 25-mutant moving average growth scores <br />                                             using hypergeometric test. <br />  <br />                                             Significances mutant STM scores calculated measuring divergence mutant <br />                                             STM score distribution sxi&#206;&#8221; STM scores using score calculation. retesting <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             STM phenotypes reconstructed knockouts, STM score &#226;&#710;&#8217;1.5 used threshold <br />                                             significance (P &lt; .001). <br />  <br />                                             Survival curves analyzed using Kaplan-Meier method log rank test. <br />  <br />                                Macrophage assays <br />                                             test phagocytosis untreated C. neoformans, RAW264.7 macrophages (4&#195;&#8212;104 cells/ <br />                                              seeded 96 tissue culture-treated glass plates (Nunc) DMEM media <br />                                             allowed adhere overnight. C. neoformans cells overnight cultures grown liquid <br />                                             YPAD washed times PBS lightly sonicated using Sonifer 450 <br />                                             equipped microtip (Branson, second pulses power setting 2) break                                             clumps cells prior incubation macrophages. Sonication did affect viability <br />                                             cells similar results obtained phagocytosis assays utilized wild-type                                             gat201&#206;&#8221; cells sonicated (Figure S13). yeast cells resuspended PBS <br />  <br />                                                       Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                                                      Page 11 <br />  <br />  <br />                                              concentration 107 cells/mL, 10 &#194;&#181;L (105 cells) incubated RAW264.7 <br />                                              cells  200 &#194;&#181;L fresh DMEM  MOI ~ 2:1. Following 24 hours incubation, <br />                                              unphagocytosed C. neoformans cells removed washes PBS, yeast NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              macrophages fixed 1% <span id='am-57' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>/PBS. Remaining extracellular yeast                                              labeled mixture anti-glucoronxylomannan antibody, mAb 339 (Belay et al., <br />                                              1997), anti-beta-glucan mouse monoclonal antibody (Biosupplies Australia) PBS,                                              combination labels encapsulated acapsular strains. Following washes <br />                                              PBS, cells incubated FITC-conjugated donkey anti-mouse secondary <br />                                              antibody (Jackson ImmunoResearch) PBS. Phagocytosis C. neoformans quantified <br />                                              Axiovert 200M (Zeiss) microscope running Axiovision software. Percentage                                              macrophages phagocytosed yeast determined counting number macrophages <br />                                              internalized yeast  defined absence FITC-labeling). 200 macrophages <br />                                              assayed  strain tested triplicate. <br />  <br />                                Microarrays <br />                                              replicate cultures H99 pRPL2b::GAT201 grown overnight saturation <br />                                              YNB 37&#194;&#176;C. Cultures diluted OD600 0.1 YNB (pre-warmed 37&#194;&#176;C), <br />                                              grown OD600 1.0, harvested centrifugation snap freezing. RNA <br />                                              isolation, array hybridizations, data analysis using SAM analysis performed                                              previously described (Chun et al., 2007, Supplementary Materials). Microarray data                                              Gene Expression Omnibus (GSE10750) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         Schematic overview systematic phenotypic profiling strategy utilized work. <br />  <br />  <br />  <br />  <br />                                              <br />  <br />  <br />  <br />                                         Figure 2. vivo signature-tagged mutagenesis (STM) analysis gene deletions <br />                                           Mutants pooled groups 48 used intranasally infect mice. Quantitative <br />                                         PCR signature tag-specific primers used calculate STM score mutant. <br />                                         Red-dotted lines indicate cutoffs used identify STM phenotypes  (B) 104 mutants <br />                                         assayed STM sorted new pools containing different competitors retested. <br />                                         Original STM scores strong correlation repooled STM scores. (C) frequencies <br />                                         STM scores observed screen plotted deletion collection                                         sxi1&#206;&#8221; control strains. Red-dotted lines indicate cutoffs used identify STM phenotypes                                          Indicated STM scores gene knockouts collection previously <br />                                         reported (red) (black) virulence defects. (D&#226;&#8364;&#8220;E) Survival curve analyses <br />  <br />                                                 Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                          Page 17 <br />  <br />  <br />                                         mice infected individual strains demonstrate correlation STM phenotypes <br />                                         virulence phenotypes. hypovirulent hypervirulent strains accurately <br />                                         identified STM. P-values indicate significance virulence phenotype. (F) Shown <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         distribution vivo vitro phenotypes deletion collection. Significant <br />                                         overlaps seen mutants defects virulence attributes mutants <br />                                         vivo infectivity defects. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                              Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. Proteins identified work regulate capsule formation, melanization,                                         infectivity <br />                                         Factors previously reported affect phenotypes factors affect growth YNB <br />                                         included. Proteins predicted encode signal peptides outlined red.                                         relationships melanization, dissemination, lung infectivity discussed                                         text. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                           Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. Gat201 regulator multiple virulence pathways <br />                                           gat201&#206;&#8221; mutant strains  independent knockout strains) avirulent                                         monotypic infection GAT201 overexpression strain hypervirulent. (B) gat201&#206;&#8221; <br />                                         mutant strains did induce observable capsule India ink staining capsule-inducing <br />                                         conditions, capsule polysaccharide cell surface detected antibody staining <br />                                         (Figure 5B). GAT201 overexpression resulted ectopic capsule formation normally non- <br />                                         inducing conditions. Quantification capsule size shown Figure S11. (C) gat201&#206;&#8221; <br />                                         mutants strains showed increased melanization GAT201 overexpression strain showed <br />                                         decreased melanization. (D) Transcriptional profiles pRPL2b::GAT201 wild-type <br />  <br />  <br />                                                 Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                 Page 20 <br />  <br />  <br />                                         grown 37&#194;&#176;C minimal medium compared using genome microarrays. Values <br />                                         represent average microarrays. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Cell. Author manuscript; available PMC 2009 October 3. <br />                             Liu et al.                                                                                          Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 5. Gat201 inhibits phagocytosis independent capsule <br />                                          B) Phagocytosis untreated wild-type (WT) gat201&#206;&#8221; cells RAW264.7 <br />                                         macrophages measured WT capsule-deficient backgrounds. percentage <br />                                         macrophages containing internalized C. neoformans cells determined 24 hours                                         incubation. Data represented mean +/&#226;&#710;&#8217; SD triplicate s. (C) GXM                                         detected cell surface gat201&#206;&#8221; cells, single double mutants containing <br />                                         deletions CAP genes. (D) Model GAT201 regulation phagocytosis. &#226;&#8364;&#339;X&#226;&#8364;&#65533; represents                                         novel capsule-independent mechanism inhibiting phagocytosis macrophages. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Cell. Author manuscript; available PMC 2009 October 3. <br />                                                              NIH-PA Author Manuscript                                     NIH-PA Author Manuscript                                       NIH-PA Author Manuscript <br />                                                                                                                                                 TABLE 1 <br />                                                                                  Novel capsule melanin mutants <br />                                                                                  Genes identified work regulate capsule formation, melanization, infectivity. Previously characterized genes                                                                                  Table S2. <br />                                                                                                                                                                                                                                        Liu et al. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                  Gene Deletion Phenotype <br />  <br />                                                               KO#                 Gene           Broad Annotation   Description                                                 STM Score   Growth Score      Capsule   Melanin <br />  <br />  <br />                                                               Capsule <br />                                                               Transcription <br />                                                                D429                GAT201             CNAG_01551         GATA family DNA-binding regulator                           &#226;&#710;&#8217;18.3&#226;&#8224;&#8220;*     &#226;&#710;&#8217;0.18                &#226;&#710;&#8217;        ++ <br />                                                                D226                SSN801             CNAG_00440         Cyclin subunit Mediator subcomplex                       &#226;&#710;&#8217;1.4        0.18                 ++       ++ <br />                                                               Chromatin <br />                                                                D110                HOS2               CNAG_05563         Class II HDAC; component Set3 complex S. cerevisiae   &#226;&#710;&#8217;16.1&#226;&#8224;&#8220;      &#226;&#710;&#8217;1.24&#226;&#8224;&#8220;               ++       &#226;&#710;&#8217; <br />                                                                D1487               SET302             CNAG_06591         SET domain protein; homolog S. cerevisiae Set3           &#226;&#710;&#8217;10.9&#226;&#8224;&#8220;      &#226;&#710;&#8217;0.86&#226;&#8224;&#8220;               ++       &#226;&#710;&#8217; <br />                                                               Putative secreted proteins <br />                                                                D667                CPL1               CNAG_02797&#226;&#8364;&#160;        Potential secreted protein unknown function              &#226;&#710;&#8217;5.3&#226;&#8224;&#8220;*      0.04                 &#226;&#710;&#8217;        + <br />  <br />                                                               Melanin <br />                                                               Transcription <br />                                                                D1248              ZAP104              CNAG_05392         Zinc finger DNA-binding regulator                           &#226;&#710;&#8217;7.1&#226;&#8224;&#8220;*      0.06                 +         &#226;&#710;&#8217; <br />                                                                D1512              MBP102              CNAG_06721         APSES family DNA-binding regulator                          &#226;&#710;&#8217;3.5&#226;&#8224;&#8220;       0.08                 +         &#226;&#710;&#8217; <br />                                                                D528               NHP6B02             CNAG_02115         Nonhistone chromosomal protein                              &#226;&#710;&#8217;0.9        &#226;&#710;&#8217;1.86&#226;&#8224;&#8220;               +        +/&#226;&#710;&#8217; <br />                                                                D1218              MLN1                CNAG_04837         bHLH family DNA-binding regulator                           &#226;&#710;&#8217;0.3        &#226;&#710;&#8217;0.04                +         &#226;&#710;&#8217; <br />                                                                D228               NHP6B01             CNAG_00445         Nonhistone chromosomal protein                              1.1         0.00                 +         &#226;&#710;&#8217; <br />                                                                D356               PAN1                CNAG_01173         LAG1 family DNA-binding regulator                           1.4         &#226;&#710;&#8217;0.40                +         &#226;&#710;&#8217; <br />                                                                D1258              RIM101              CNAG_05431         Zn finger DNA-binding regulator regulated proteolysis    2.8&#226;&#8224;&#8216;        &#226;&#710;&#8217;0.07                +         &#226;&#710;&#8217; <br />                                                               Chromatin <br />                                                                D1498              RTF1                CNAG_06648         Component PAF complex required histone H2B           &#226;&#710;&#8217;12.7&#226;&#8224;&#8220;      &#226;&#710;&#8217;1.66&#226;&#8224;&#8220;               +        &#226;&#710;&#8217; <br />                                                                                                                          monoubiquitination <br />                                                                D78                 SET202             CNAG_03188         Histone H3 lysine 36 methyltransferase                      &#226;&#710;&#8217;8.1&#226;&#8224;&#8220;       &#226;&#710;&#8217;0.14                +         &#226;&#710;&#8217; <br />                                                                D373                SET101             CNAG_01243         Histone H3 lysine 4 methyltransferase                       &#226;&#710;&#8217;4.5&#226;&#8224;&#8220;*      &#226;&#710;&#8217;0.38                +        +/&#226;&#710;&#8217; <br />                                                               Protein sorting / Protein degradation <br />                                                                D282                DOA4               CNAG_00757         Ubiquitin hydrolase                                         &#226;&#710;&#8217;15.6&#226;&#8224;&#8220;      &#226;&#710;&#8217;0.06                +         &#226;&#710;&#8217; <br />                                                                D1250               VAM6               CNAG_05395         Vacuolar membrane fusion factor                             &#226;&#710;&#8217;14.3&#226;&#8224;&#8220;      &#226;&#710;&#8217;0.61&#226;&#8224;&#8220;               +         &#226;&#710;&#8217; <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available PMC 2009 October 3. <br />                                                                D1222               VPS25              CNAG_04863         ESCRT-II Complex subunit                                    &#226;&#710;&#8217;13.8&#226;&#8224;&#8220;      &#226;&#710;&#8217;0.06                +         &#226;&#710;&#8217; <br />                                                                D1314               UBP14              CNAG_05708         Ubiquitin carboxyl-terminal hydrolase                       &#226;&#710;&#8217;12.3&#226;&#8224;&#8220;      0.27                 +        +/&#226;&#710;&#8217; <br />                                                                D454                CSN1201            CNAG_01697         COP9 signalosome subunit 12; PINT domain                    &#226;&#710;&#8217;8.9&#226;&#8224;&#8220;       &#226;&#710;&#8217;0.58&#226;&#8224;&#8220;               +        +/&#226;&#710;&#8217; <br />                                                                D936                CSN7               CNAG_04177         COP9 signalosome subunit 7; PINT domain                     &#226;&#710;&#8217;1.7        &#226;&#710;&#8217;0.07                +        +/&#226;&#710;&#8217; <br />                                                                D598                CSN1               CNAG_02513         COP9 signalosome subunit 1; PINT domain                     &#226;&#710;&#8217;1.1        &#226;&#710;&#8217;0.16                +        +/&#226;&#710;&#8217; <br />                                                                D737                CSN4               CNAG_03151         COP9 signalosome subunit 4; PINT domain                     &#226;&#710;&#8217;0.2        0.00                 +        +/&#226;&#710;&#8217; <br />                                                                D1349               HSE102             CNAG_05882         Class E VPS factor involved endosomal sorting            &#226;&#710;&#8217;0.2        0.00                 +         &#226;&#710;&#8217; <br />                                                                D671                RPS3102            CNAG_02827         NEDD8 ubiquitin-like protein regulates Cullins         4.1&#226;&#8224;&#8216;        &#226;&#710;&#8217;1.68&#226;&#8224;&#8220;               +         &#226;&#710;&#8217; <br />                                                               Signaling <br />                                                                D245                YCK2               CNAG_00556         Casein kinase homolog involved endocytosis               &#226;&#710;&#8217;15.4&#226;&#8224;&#8220;      &#226;&#710;&#8217;0.44                +         &#226;&#710;&#8217; <br />                                                                D1476               SNF102             CNAG_06552         <span id='am-37' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="AMP" datatype="xsd:string"></span>AMP</span>-activated kinase; ortholog S. cerevisiae Snf1        &#226;&#710;&#8217;15.0&#226;&#8224;&#8220;*     &#226;&#710;&#8217;0.34                +         &#226;&#710;&#8217; <br />                                                                D731                CHO2               CNAG_03139         Putative protein-S-isoprenylcysteine methyltransferase      &#226;&#710;&#8217;12.3&#226;&#8224;&#8220;      0.27                 +         &#226;&#710;&#8217; <br />                                                                D221                CDC2801            CNAG_00415         Cyclin-dependent kinase PITSLRE subfamily                &#226;&#710;&#8217;9.9&#226;&#8224;&#8220;       &#226;&#710;&#8217;0.11                +        +/&#226;&#710;&#8217; <br />                                                                D825                RIM20              CNAG_03582         ESCRT-III-associated regulator Rim101                    &#226;&#710;&#8217;0.3        0.09                 +         &#226;&#710;&#8217; <br />                                                               Nuclear pore <br />                                                                D891                NUP75              CNAG_03917         Nuclear pore protein homologous human Nup75              &#226;&#710;&#8217;14.2&#226;&#8224;&#8220;      &#226;&#710;&#8217;1.83&#226;&#8224;&#8220;               +        +/&#226;&#710;&#8217; <br />                                                               Unknown function <br />                                                                D1408               MLN2               CNAG_06224         CS domain protein unknown function                       &#226;&#710;&#8217;1.4        &#226;&#710;&#8217;1.24&#226;&#8224;&#8220;               +         &#226;&#710;&#8217; <br />                                                                D1453               MLN3               CNAG_06430         Mus7 domain-containing protein conserved fungi           &#226;&#710;&#8217;1.6        &#226;&#710;&#8217;0.78&#226;&#8224;&#8220;               +        +/&#226;&#710;&#8217; <br />                                                                                                                                                                                                                                        Page 22 <br />                                                              NIH-PA Author Manuscript                                            NIH-PA Author Manuscript                                 NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                                   Gene Deletion Phenotype <br />  <br />                                                               KO#                  Gene           Broad Annotation          Description                                          STM Score   Growth Score      Capsule   Melanin <br />                                                                                                                                                                                                                                         Liu et al. <br />  <br />  <br />  <br />  <br />                                                                D445                MLN4                CNAG_01644                Protein unknown function; C. neoformansspecific   &#226;&#710;&#8217;1.4        &#226;&#710;&#8217;0.18                +        &#226;&#710;&#8217; <br />  <br />  <br />                                                               arrows (&#226;&#8224;&#8220;) indicate significant decreases compared wild-type. <br />  <br />                                                               arrows (&#226;&#8224;&#8216;) indicate significant increases compared wild-type. <br />  <br />                                                               Asterisks (*) indicate STM phenotype verified reconstructed knockout strain. <br />  <br />                                                               Crosses (&#226;&#8364;&#160;) indicate gene models predicted encode signal peptides. <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available PMC 2009 October 3. <br />                                                                                                                                                                                                                                         Page 23 <br />                                                              NIH-PA Author Manuscript                                        NIH-PA Author Manuscript                                        NIH-PA Author Manuscript <br />                                                                                                                                                 TABLE 2 <br />                                                                                  Infectivity mutants defects capsule formation, melanization, growth YNB <br />                                                                                  Table 2 includes infectivity mutants did display defects capsule formation, melanization, growth YNB. mutants <br />                                                                                  tested additional phenotypes: decreased urease activity sensitivity acidified nitrite  , SDS, potassium cyanide <br />                                                                                  (KCN), pH 4, low-iron. <br />                                                                                                                                                                                                                                                 Liu et al. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                   Gene Deletion Phenotype <br />  <br />                                                               KO#                    Gene           Broad Annotation       Description                                       STM Score      Growth Score         Additional Phenotypes <br />  <br />  <br />                                                               Decreased Infectivity (STM Score &lt; &#226;&#710;&#8217;2.5) <br />                                                               Transcription <br />                                                                D883                  PDR802              CNAG_03894             Binuclear cluster DNA-binding regulator           &#226;&#710;&#8217;7.2&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.08                      detected <br />                                                                D231                  LIV1                CNAG_00460             bHLH family DNA-binding regulator                 &#226;&#710;&#8217;6.1&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.35                      detected <br />                                                                D725                  HCM1                CNAG_03116             Forkhead DNA-binding regulator                    &#226;&#710;&#8217;4.2&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.46                      detected <br />                                                                D140                  SNT1                CNAG_00051             SANT domain protein related human NCOR         &#226;&#710;&#8217;4.0&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.10                      detected <br />                                                                                                                                 corepressor subunit <br />                                                                D177                   LIV2               CNAG_00201             LAG1 family DNA-binding regulator                 &#226;&#710;&#8217;3.9&#226;&#8224;&#8220;*         0.06                       detected <br />                                                                D1338                  LIV3               CNAG_05835             WOR1 family DNA-binding regulator                 &#226;&#710;&#8217;3.7&#226;&#8224;&#8220;*         0.03                       detected <br />                                                                D1420                  LIV4               CNAG_03116             Myb family DNA-binding regulator                  &#226;&#710;&#8217;2.5&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.03                      detected <br />                                                               Chromatin <br />                                                                D111                   RPD304             CNAG_05690             Class II histone deacetylase                      &#226;&#710;&#8217;11.5&#226;&#8224;&#8220;*        &#226;&#710;&#8217;0.3                      detected <br />                                                                D1296                  RPH1               CNAG_05622             Histone H3 lysine 36 demethylase                  &#226;&#710;&#8217;8.0&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.25                     detected <br />                                                                D933                   HIRA               CNAG_04158             Histone chaperone                                 &#226;&#710;&#8217;6.7&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.15                  sensitivity                                                                D790                   SPP101             CNAG_03406             PHD domain protein unknown function            &#226;&#710;&#8217;6.1&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.09                     detected <br />                                                                D1428                  SAS3               CNAG_06322             Histone acetyltransferase                         &#226;&#710;&#8217;4.4&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.18                     detected <br />                                                                D62                    HST302             CNAG_02085             Homolog Sir2; Class III histone deacetylase    &#226;&#710;&#8217;2.8&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.11                     detected <br />                                                               Protein sorting / Protein degradation <br />                                                                D1266                  HRD1               CNAG_05469             Transmembrane ubiquitin ligase required       &#226;&#710;&#8217;8.0&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.38                  sensitivity                                                                                                                                 ER-associated degradation <br />                                                                D317                  UBA4                CNAG_00986             Urm1 activating protein                           &#226;&#710;&#8217;6.0&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.2                       detected <br />                                                                D399                  RMD5                CNAG_01350             Ubiquitin ligase unknown function              &#226;&#710;&#8217;4.3&#226;&#8224;&#8220;*         0.02                       detected <br />                                                                D729                  CUL3                CNAG_03138             Cullin subunit SCF-like ubiquitin ligase       &#226;&#710;&#8217;3.6&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.11                      detected <br />                                                                D1166                 UBC8                CNAG_04611             Ubiquitin conjugating enzyme                      &#226;&#710;&#8217;3.5&#226;&#8224;&#8220;*         0.16                       detected <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available PMC 2009 October 3. <br />                                                                D1278                 JJJ1                CNAG_05538             DnaJ-like cochaperone; contains U1 snRNA-         &#226;&#710;&#8217;3.4&#226;&#8224;&#8220;*         0.11                       detected <br />                                                                                                                                 type Zn finger <br />                                                               DNA Repair <br />                                                                D352                  RAD54               CNAG_01163             SWI/SNF family ATPase promotes               &#226;&#710;&#8217;9.3&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.49                  sensitivity                                                                                                                                 recombinational DNA repair <br />                                                                D288                  RAD23               CNAG_00772             Ubiquitin-binding protein required            &#226;&#710;&#8217;7.6&#226;&#8224;&#8220;*         0.09                   sensitivity                                                                                                                                 <span id='am-44' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-45' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-51' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> excision repair <br />                                                                D832                  YKU80               CNAG_03637             Double strand break repair factor silencing   &#226;&#710;&#8217;4.9&#226;&#8224;&#8220;*         0.04                       detected <br />                                                                                                                                 regulator <br />                                                                D632                  RAD502              CNAG_02630             SWI/SNF family ATPase promotes               &#226;&#710;&#8217;2.7&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.04                      detected <br />                                                                                                                                 recombinational DNA repair <br />                                                                D855                  RINT1               CNAG_03760             Rad50-interacting protein implicated DNA       &#226;&#710;&#8217;2.5&#226;&#8224;&#8220;*         0.01                  sensitivity KCN <br />                                                                                                                                 repair <br />                                                               Pathogenesis / Immunity <br />                                                                D576                 LIV5                 CNAG_02409             TB2/DP1 family member homologous M.            &#226;&#710;&#8217;12.6&#226;&#8224;&#8220;*        &#226;&#710;&#8217;0.09                      detected <br />                                                                                                                                 grisea pathogenicity protein <br />                                                                D166                  LIV6                CNAG_00170&#226;&#8364;&#160;            Related human T-cell immunomodulatory          &#226;&#710;&#8217;6.2&#226;&#8224;&#8220;*         0.14                       detected <br />                                                                                                                                 protein; FG-GAP repeats <br />                                                                                                                                                                                                                                                 Page 24 <br />                                                              NIH-PA Author Manuscript                                            NIH-PA Author Manuscript                                              NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                                                                             Gene Deletion Phenotype <br />  <br />                                                               KO#                    Gene              Broad Annotation             Description                                        STM Score      Growth Score         Additional Phenotypes <br />                                                                                                                                                                                                                                                              Liu et al. <br />  <br />  <br />  <br />  <br />                                                                D1466                 LIV7                   CNAG_06464&#226;&#8364;&#160;                  Protein unknown function; human homolog         &#226;&#710;&#8217;3.2&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.17                      detected <br />                                                                                                                                          <span id='am-17' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-18' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-26' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span>LPS</span>-inducible <br />                                                               Signaling <br />                                                                D972                  HSV2                   CNAG_04371                   WD40 protein related 3-phosphorylated           &#226;&#710;&#8217;15.9&#226;&#8224;&#8220;*        0.18                       detected <br />                                                                                                                                          inositide-binding protein <br />                                                                D280                  HRK1                   CNAG_00745                   Protein kinase; S. cerevisiae ortholog regulates   &#226;&#710;&#8217;6.8&#226;&#8224;&#8220;*         0.14                  sensitivity SDS <br />                                                                                                                                          metabolite transport <br />                                                                D437                  LIV8                   CNAG_01611                   RGS protein homolog - G protein GAP                &#226;&#710;&#8217;2.7&#226;&#8224;&#8220;*         0.10                sensitivity  SDS, <br />                                                                                                                                                                                                                                           KCN <br />                                                               Unknown <br />                                                                D70                   HOS4                   CNAG_02727                   Ankyrin repeat protein unknown function         &#226;&#710;&#8217;7.6&#226;&#8224;&#8220;*         0.09                       detected <br />                                                                D736                  LIV9                   CNAG_03149                   RING ubiquitin-binding protein            &#226;&#710;&#8217;7.3&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.04                      detected <br />                                                                                                                                          BRAP2 domain <br />                                                                D686                  LIV10                  CNAG_02930                   DUF1765 domain protein unknown function         &#226;&#710;&#8217;7.1&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.19                      detected <br />                                                                D171                  FYV10                  CNAG_00181                   LisH, CTLH domain protein unknown               &#226;&#710;&#8217;6.5&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.06                      detected <br />                                                                                                                                          function <br />                                                                D1256                 LIV11                  CNAG_05422                   Protein unknown function                        &#226;&#710;&#8217;5.0&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.49                      detected <br />                                                                D1035                 LIV12                  CNAG_05105                   Protein unknown function                        &#226;&#710;&#8217;3.9&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.11                      detected <br />                                                                D660                  LIV13                  CNAG_02753                   UPF0121 domain protein unknown function         &#226;&#710;&#8217;3.2&#226;&#8224;&#8220;*         &#226;&#710;&#8217;0.34                      detected <br />                                                                D1448                 LIV14                  CNAG_06397                   Protein unknown function                        &#226;&#710;&#8217;2.9&#226;&#8224;&#8220;*         0.06                       detected <br />                                                                D1357                 LIV15                  CNAG_05934                   Translin domain protein unknown function        &#226;&#710;&#8217;2.8&#226;&#8224;&#8220;*         0.09                       detected <br />                                                               Transporter <br />                                                                D1091                 ENA2                   CNAG_00531                   Sodium pump                                        &#226;&#710;&#8217;11.1&#226;&#8224;&#8220;*        0.11                       detected <br />  <br />                                                               Increased Infectivity (STM Score &gt; 2.5) <br />                                                               Transcription <br />                                                                D88                   ZAP103                 CNAG_04352                   Zinc finger DNA-binding regulator                  3.2&#226;&#8224;&#8216;*          &#226;&#710;&#8217;0.14 <br />                                                               Signaling <br />                                                                D1340                 RGD1                   CNAG_05838                   Rho GAP homolog                                    5.3&#226;&#8224;&#8216;*          0.07 <br />                                                                D1490                 RHO104                 CNAG_06606                   Rho family GTPase                                  4.4&#226;&#8224;&#8216;*          &#226;&#710;&#8217;0.33 <br />                                                                D1156                 RCK2                   CNAG_00130                   Ca-CaM kinase homolog                              3.7&#226;&#8224;&#8216;*          0.03 <br />                                                                D488                  KSP1                   CNAG_01905                   Protein kinase                                     3.4&#226;&#8224;&#8216;*          &#226;&#710;&#8217;0.04 <br />  <br />  <br />  <br />  <br /> Cell. Author manuscript; available PMC 2009 October 3. <br />                                                               Cell wall <br />                                                                D61                   BNI4                   CNAG_02071                   S. cerevisiae ortholog scaffold protein       4.8&#226;&#8224;&#8216;*          0.27 <br />                                                                                                                                          involved cell wall assembly <br />                                                               Actin-related <br />                                                                D359                  LSB1                   CNAG_01183                   BAR-SH3 domain; S. cerevisiae homolog binds        3.6&#226;&#8224;&#8216;*          0.11 <br />                                                                                                                                          actin nucleator WASP <br />                                                               Unknown <br />                                                                D1179                 LIV16                  CNAG_07193&#226;&#8364;&#160;                  Conserved protein unknown function              3.8&#226;&#8224;&#8216;*          0.05 <br />  <br />  <br />                                                               arrows (&#226;&#8224;&#8220;) indicate significant decreases compared wild-type. <br />  <br />                                                               arrows (&#226;&#8224;&#8216;) indicate significant increases compared wild-type. <br />  <br />                                                               Asterisks (*) indicate STM phenotype verified reconstructed knockout strain. <br />  <br />                                                               Crosses (&#226;&#8364;&#160;) indicate gene models predicted encode signal peptides. <br />                                                                                                                                                                                                                                                              Page 25 <br />   <br /> </body></html>