<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A Network Conserved Damage Survival Pathways <br /> Revealed Genomic RNAi Screen <br /> Dashnamoorthy Ravi1, Amy M. Wiles1, Selvaraj Bhavani1, Jianhua Ruan2, Philip Leder3, Alexander J. R. <br /> Bishop1,3* <br /> 1 Greehey Children&#226;&#8364;&#8482;s Cancer Research Institute, University Texas Health Science Center San Antonio, San Antonio, Texas, United States America, 2 Department Science, University Texas San Antonio, San Antonio, Texas, United States America, 3 Harvard Medical School, Department Genetics, Harvard <br /> University, Boston, Massachusetts, United States America <br />  <br />  <br />  <br />      Abstract <br />      Damage initiates pleiotropic cellular response aimed cellular survival appropriate. identify genes required      damage survival, used cell-based RNAi screen Drosophila genome alkylating agent methyl <br />      methanesulphonate (MMS). Similar studies performed model organisms report damage response involve <br />      pleiotropic cellular processes central DNA repair components, intuitive systems level view      cellular components required damage survival, interrelationship, contextual importance lacking. <br />       comparing data different model organisms, identification conserved presumably core survival <br />      components forthcoming. identified 307 genes, representing 13 signaling, metabolic, enzymatic pathways, <br />      affecting cellular survival MMS&#226;&#8364;&#8220;induced damage. expected, majority pathways involved DNA repair; <br />       pathways diverse biological functions identified, including TOR pathway, <br />      transcription, translation, proteasome, <span id='am-12' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-13' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutathione</span> synthesis, <span id='am-36' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span> synthesis, Notch signaling, equally <br />      important damage survival. Comparison genomic screen data Saccharomyces cerevisiae revealed overlap <br />      enrichment individual genes species, conservation pathways. demonstrate functional <br />      conservation pathways, tested Drosophila mouse cells, pathway responding alkylation <br />      damage species. Using protein interactome, significant level connectivity observed Drosophila <br />      MMS survival proteins, suggesting higher order relationship. connectivity dramatically improved incorporating <br />      components 13 identified pathways network. Grouping proteins &#226;&#8364;&#732;&#226;&#8364;&#732;pathway nodes&#226;&#8364;&#8482;&#226;&#8364;&#8482; qualitatively <br />      improved interactome organization, revealing highly organized &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival network.&#226;&#8364;&#8482;&#226;&#8364;&#8482; conclude      identification pathways facilitate comparative biology analysis direct gene/orthologue comparisons fail.      biologically intuitive, highly interconnected MMS survival network revealed incorporated pathway data      interactome analysis. <br />  <br />   Citation: Ravi D, Wiles  Bhavani S, Ruan J, Leder P, et al. (2009) Network Conserved Damage Survival Pathways Revealed Genomic RNAi Screen. PLoS <br />   Genet 5(6): e1000527. doi:10.1371/journal.pgen.1000527 <br />   Editor: Amy Kiger, University California San Diego, United States America <br />   Received January 5, 2009; Accepted 19, 2009; Published June 19, 2009 <br />   Copyright: &#195;&#376; 2009 Ravi et al. open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported NIEHS (K22-ES12264; AJRB). JR supported NIH grant 1SC3GM086305-01. thank Drosophila RNAi <br />   Screening Center (http://flyrnai.org/), supported NIGMS (2R01-GM067761-05). funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: bishopa@uthscsa.edu <br />  <br />  <br />  <br />  <br /> Introduction                                                                            function consequences dysfunction. <br />                                                                                         Recent reports suggest DDR involve pleiotropic cellular <br />    Cellular damage normal component life, constant                         processes central DDR components [3,4], damage exposure endogenous exogenous sources.                             intuitive systems level view cellular components required Damage DNA considered biologically relevant                        damage survival, interrelationship, contextual <br /> lesion potential mutagenic results,                             importance lacking. comprehensive attempt exogenous agents potential damage components                           understanding interrelationship damage response cell. Responding appropriately insults,                           survival components yeast systems level mapping <br /> mitigating cellular toxicity initiating appropriate cell death                    identified genes network integrating general biological <br /> response, critical, particularly multi-cellular organisms.                        processes distinct modules [5]. suggested Inappropriate responses facilitate deleterious effects,                     inclusion defined biological pathways protein networks destabilized genome diseases cancer [1].                          provide better understanding biological interactions <br /> DNA damage response (DDR) components critical suppressors                           [6,7]. deleterious effects genotoxic agents controlling cell cycle                       order identify genes involved damage response progression, DNA repair, apoptosis [2]. reason,                      unbiased manner functional context, used investigations using variety model organisms                        RNAi library [8] knock predicted protein identify components DDR subsequently determine                             Drosophila melanogaster genome assessed knock  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                        1                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                         Damage Survival Pathways <br />  <br />  <br />  <br />   Author Summary                                                           Results <br />   Cellular damage known elicit pleiotropic response,               Genome-wide RNAi screen validation <br />   relative importance constituent compo-                       RNAi screen designed identify genes   nents cell survival poorly understood. provide               modulate cellular survival following exposure level MMS- <br />   unbiased identification proteins utilized damage               induced damage resulted limited cell <br />   survival, performed RNAi survival screen fly cells              death. linear decrease cell viability observed 0.002% <br />   methyl methanesulfonate (MMS). genes identi-                    (w/v) MMS 0.008% (w/v) MMS (Figure S1). dose   fied involved 13 biologically diverse pathways.                   0.004% (w/v; 40 mg/mL) MMS chosen, resulted   Comparison analogous yeast data demonstrated                      statistically significant decrease approximately 65% viability, <br />   lack conservation individual MMS survival genes                allowing additional, measurable decrease cell survival. <br />   conservation pathways. went                            RNAi screens performed using Drosophila RNAi <br />   demonstrate MMS responsiveness representa-                  Screening Center (DRSC) version 1 library  23,000 D. <br />   tive pathways fly mouse cells. conclude <br />                                                                            melanogaster open reading frames) [8]. Kc167 Drosophila cells   identification pathways facilitate comparative <br />   biology analysis direct gene/orthologue compari-                    exposed days dsRNA allow protein knock  <br />   sons fail. Incorporation pathway data interactome                  followed days growth absence presence <br />   analysis improved connectivity  importantly,               MMS exposure (Figure 1A). <br />   revealed biologically intuitive, highly inter-connected                   identify genes required cell viability following <br />   &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival network.&#226;&#8364;&#8482;&#226;&#8364;&#8482; pathway conservation                  MMS treatment, viability results MMS treated RNAi <br />   inter-connectivity implies extensive interaction                 screen compared untreated (control) screen <br />   pathways; diseases cancer, crosstalk                previously described [18]. 1,398 different open reading frames <br />   dictate disparate cellular responses necessarily expect-             identified affected MMS survival continuous <br />   ed confound treatments tailored                  distribution cell survival values  [18] http://gccri. <br />   individual molecular context.                                            uthscsa.edu/ABPublished_Data.asp original data),                                                                            996 unique assigned FlyBase gene number (FBgn; denotes <br />                                                                            known genes)  Figure 1B Table 1). 996 genes, individual proteins significantly altered cell viability following         537 selected validation analysis previously <br /> methyl methanesulfonate (MMS) exposure. MMS prototypical              described, stringent validation method [18]. [18] alkylating agent attacks nucleophilic groups,           validated normalization methods, independently validated nucleic acid [9]. resultant base methylation                  individual genes using dsRNA targeting different region destabilizes glycosidic bond, facilitating production       gene. 202 protein knock-downs validated significant MMS biologically relevant cellular lesion MMS, abasic site       viability effect, 55 notable trend effect DNA. common sources alkylation damage include                     meeting stringent statistical criteria (Table S1 Table S2). <br /> endogenous S-adenoysyl methionine [10], tobacco carcinogen             examination gene ontology   202 validated MMS <br /> N-nitorsoamine [11] chemotherapeutics temozolo-                survival genes revealed significant enrichment genes involved <br /> mide [12], carmustine (BCNU) [9], cyclophosphamide [13].               DNA metabolism, gene transcription, cellular metabolism <br /> Considering physiological relevance alkylation damage,              (Table 2). overall variety categories observed quite recent studies using yeast model organism investigated          broad, consistent findings reported analogous yeast global effects alkylation damage induced MMS. Viability          screen [3] (data shown). Surprisingly  significant <br /> [14], gene transcription [15] protein expression [16]             enrichment gene orthologues organisms <br /> measured studies, provided insights         observed (G-test p-values 0.29 0.057 Fisher Exact <br /> diverse nature biological responses alkylation damage.               test p-values 0.37 0.08, yeast fly, respectively). transcriptional responses various environmen-              test  examined MMS sensitivity following <br /> tal stresses compared mammalian yeast cells               knock 183 fly genes orthologues 118 yeast <br /> [17], Murray et al. reported clear distinctions suggested         MMS survival genes [3], conservation MMS result different selective pressures          survival phenotype 20 (Table S3). apparent lack gene <br /> multi-cellular organisms single-cell organisms.                 enrichment species reported attempt understand diverse responses required damage                comparing transcriptional profiles mammalian yeast <br /> survival evolutionarily closer mammals              cells response variety stresses [17]. Overall, results yeast, performed genome-wide, RNAi-based screen                  suggest conservation biological processes <br /> using cells derived Drosophila melanogaster. experiments          respond MMS individual genes. based &#226;&#8364;&#732;&#226;&#8364;&#732;loss function&#226;&#8364;&#8482;&#226;&#8364;&#8482; assessment cellular <br /> viability following exposure MMS. present results               Identification pathways responsive MMS MMS survival screen, pathways                         Assuming biological function informative <br /> identified, comparative analysis pathways understand           measure damage response requirement individual <br /> conservation pathways yeast mouse cells.                   genes, endeavoured identify MMS survival <br /> Additionally, present novel approach assembling                  proteins known signaling, metabolic, enzymatic <br /> protein interaction network based defined biological path-              pathways. Using priori knowledge KegArray, ways, facilitates network consolidation. including                identified 13 pathways included 41 MMS survival <br /> biological pathways genomic data protein network                proteins (Figures 1, 2, 3 Figures S2, S3, S4, S5),  <br /> analysis, able determine commonality                 DNA repair pathways (Base Excision Repair; <span id='am-52' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-53' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>Nucleotide</span> <br /> species, obscured direct orthologue comparison,               Excision Repair; Mismatch Repair; Homologous Recombination provide simplified representation global survival             Repair; RECQ). pathways statistically <br /> responses identified.                                              significant number MMS survival proteins (Table S5),  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             2                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                           Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 1. experimental design Drosophila RNAi MMS survival screen summary workflow results.   schematic representation experimental design used screen genes involved survival MMS exposure using RNAi Drosophila. (B) schematic representation workflow initial RNAi screen, normalization validation results, follow work conducted, summary number dsRNA amplicons genes (FBgn; FlyBase gene number) involved step process. provided final numbers MMS survival proteins MMS survival pathways (BER: base excision repair; NER: nucleotide excision repair; MMR: mismatch <br /> repair; HRR: homologous recombination repair; RECQ: RecQ helicases; DDR: DNA damage response; Proteasome; GSH: <span id='am-77' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-78' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span>glutathione</span> synthesis; TOR: <br /> TOR pathway; Transcription: basal transcription; Ribosome; ATPase; Notch: Notch signaling pathway). <br /> doi:10.1371/journal.pgen.1000527.g001 <br />  <br />  accounting pathway &#226;&#8364;&#732;&#226;&#8364;&#732;branching&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;subdivision.&#226;&#8364;&#8482;&#226;&#8364;&#8482;          hypothesis identified pathways involved Considering likely role pathways MMS survival,            MMS survival. result highlights possibility false <br /> used RNAi assay method, sensitive                  negative screen data, compared previous false negative rate <br /> stringent original screen [18], determine          23.6% randomly selected data, observe significant <br /> members 13 identified pathway affected MMS survival              enrichment false negatives pertinent pathways ((x2 <br /> (Table S1). individually interrogating 346 pathway                    p = 3.9E-5). total identified 146 MMS survival genes 13 <br /> members 13 pathways identified                   &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival pathways,&#226;&#8364;&#8482;&#226;&#8364;&#8482; encompassing 25% 86% non- <br /> original screen, additional 105 MMS survival genes                   essential genes pathway (Table S4). <br /> discovered. observation significantly enriched number               Using genes mapped MMS survival pathways, MMS survival genes 13 pathways (Figures 1, 2, 3               compared pathways identified Drosophila MMS Figures S2, S3, S4, S5) provides additional support           screen analogous screen performed yeast [14] using <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              3                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                               Damage Survival Pathways <br />  <br />  <br />  <br />  Table 1. Summary MMS survival genes identified                              component XRCC1 resulted increased AP sites <br />  genome-wide RNAi screen.                                                          itself (p#3E-3), following MMS treatment,                                                                                    AP sites absence XRCC1 increased <br />                                                                                    (Figure 4A; compared untreated XRCC1 knock- <br />  Normalized RNAi screen results                                                     p#3.3E-8; compared MMS treated luciferase (Luc) <br />                                                                                    control, p#3.7E-3); MMS produces expected form <br />                      Death           Resistance              Total <br />                                                                                    DNA damage BER appropriately responds. p53,  BS_QN               1183            40                      1223                  central component DDR pathway, regulated  cellHTS2            594             40                      634                   expression, protein stabilization, posttranslational modifica- <br />                                                                                    tion levels. expected, p53 gene expression increased response <br />  total               1326            72                      1398 <br />                                                                                    MMS exposure (Figure 4B; p#1.6E-3). <br />  FBgn                935             61                      996 <br />                                                                                       examined MMS survival pathways  Numbers death resistance hits background subtraction              canonical DDR DNA repair process: glutathione <br />  followed quantile normalization (BSQN) cellHTS2 given. total       metabolism, proteasome, TOR pathway. increased <br />  number unique hits total number hits amplicons map         activity glutathione synthesis pathway following MMS <br />  genes FlyBase gene number (FBgn) given.                             exposure demonstrated measuring total glutathione <br />  doi:10.1371/journal.pgen.1000527.t001 <br />                                                                                    content milligram protein lysate (Figure 4C; p#3.4E-3), <br />                                                                                    examining expression rate-limiting enzyme <br /> genes responsive MMS. previously noted,                     glutathione synthesis, GCLc [22] (p#8.8E-4).  did observe significant overlap screens                   expected, knock protein, GCLc, significantly comparing MMS survival gene orthologues,                         reduced total glutathione present compared pathway comparison, mapped orthologues yeast MMS                        control (p#3.4E-3), knock prevented cells <br /> survival genes 10 13 Drosophila MMS survival pathways                    increasing level glutathione response MMS <br /> (Figures 2 3; Figures S2, S3, S4, S5; Table S6).                     exposure (Figure 4C). glutathione synthesis considered Drosophila MMS survival pathways yeast MMS survival                        oxidative stress response, demonstrated MMS <br /> genes conserved yeast                  resulted dose-dependent increase level 8-oxoguanine, <br /> pathway components essential viability yeast,                      DNA lesion normally associated oxidative damage (Figure <br /> making refractory MMS viability analysis (Table S5).                   S7 Text S1). appears MMS results gene enrichment pathway species highly                        alkylation damage damage oxidative stress. <br /> supportive conservation processes involved MMS                             analysis proteasome degradation pathway, survival. notable absentee Drosophila screen,                   measured proteasome activity increased <br /> observed yeast, 3-methyladenine-DNA-glycosylase (MAG1/                       following MMS treatment (Figure 4D; p#4.2E-2). Protein <br /> AlkA), principal protection MMS-induced DNA                            knock proteasome components Rpn2 Pros26.4 <br /> damage yeast, direct orthologues exist animals.                   significantly reduced proteasome activity compared cells glycosylase Drosophila act step               knock unexposed (Rpn2: BER, Thd1, required survival                   p#1.7E-2; Pros26.4: p#3.7E-3) exposed MMS (Rpn2: <br /> MMS treatment. Furthermore, pathways observed                       p#1.4E-4; Pros26.4: p#7.0E-5). Following Rpn2 knock  species, proteasome [19], TOR pathway [20],                    cells unable mount detectable increase proteasome DNA repair pathways [21], shown                                activity following MMS exposure, unable functionally responsive MMS yeast. Altogether, results                 demonstrate Pros26.4 knock (p#1.7E-3). Overall, suggest conservation pathway function, individual genes,               appears MMS exposure results increased proteasome response MMS species.                                                activity. <br />    demonstrate identified Drosophila pathways                         Finally, investigate TOR pathway activity, S6K phosphor- <br /> functioning expected manner response MMS,                   ylation status monitored. TOR kinase phosphorylates <br /> selected examination &#226;&#8364;&#8220; base excision repair (BER),                     S6K promote growth ribosome biogenesis DDR, glutathione metabolism, proteolysis proteasomal degra-                     negatively regulated tumor suppressor protein Tsc1 [23]. <br /> dation (proteasome), TOR pathway.                             MMS exposure resulted dose-dependent decrease S6K <br /> pathways, BER DDR, expected play role MMS                          phosphorylation, suggesting inhibition TOR activity <br /> survival [9], selected apparent                    (Figure 4E). MMS-induced decrease S6K phosphorylation <br /> breadth function non-canonical role damage                         dependent Tsc1 (Figure 4E). MMS exposure survival. pathway, appropriate functional assay                    elicited regulation growth promoting TOR <br /> chosen, possible, appropriate upstream MMS                             pathway, suggesting pathway coordinated <br /> survival protein pathway knocked (Figure                      component DDR similar observations yeast [20]. S6A) demonstrate modulation MMS induced response.                        supports previously published observation Matsuoka et al. [4], <br />                                                                                    showed components mammalian TOR <br /> Functional conservation MMS survival pathways                                pathway phosphorylated following ionizing radiation expo- <br /> Drosophila mouse                                                               sure. conclusion, results functional assays validate <br />     tested expected MMS survival pathways BER                     identification &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival pathways&#226;&#8364;&#8482;&#226;&#8364;&#8482; RNAi DDR. MMS-induced DNA damage results production                          screening functionality pathways, lack apurinic/apyrimidic (AP) sites, DNA damage typically                          thereof, affects MMS survival. <br /> repaired BER [9]. quantified AP                         Having identified 13 Drosophila MMS survival pathways, sites microgram genomic DNA following MMS treatment                         went investigate functional conservation mammals. observed statistically significant increase compared                      Using primary mouse embryonic fibroblasts (MEFs), examined <br /> control (Figure 4A; p#7.6E-8). Knock BER                               pathways modulated following MMS <br />  <br />  <br />          PLoS Genetics | www.plosgenetics.org                                  4                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                       Damage Survival Pathways <br />  <br />  <br />  <br /> Table 2. Categories Gene Ontology enrichment validated MMS survival genes. <br />  <br />  <br /> Gene Ontology enrichment <br />  <br /> Biological Process <br />  <br /> Adjusted P-value               Attribute <br />  <br /> ,0.001                         0006139: nucleobase, <span id='am-60' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-61' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span>, nucleotide nucleic acid metabolic process <br /> ,0.001                         0006351: transcription, DNA-dependent <br /> ,0.001                         0032774: RNA biosynthetic process <br /> ,0.001                         0016070: RNA metabolic process <br /> ,0.001                         0045449: regulation transcription <br /> ,0.001                         0065004: protein-DNA complex assembly <br /> ,0.001                         0016043: cellular component organization biogenesis <br /> ,0.001                         0051252: regulation RNA metabolic process <br /> ,0.001                         0006367: transcription initiation RNA polymerase II promoter <br /> ,0.001                         0006355: regulation transcription, DNA-dependent <br /> ,0.001                         0065003: macromolecular complex assembly <br /> ,0.001                         0008283: cell proliferation <br /> 0.001                          0035080: heat shock-mediated polytene chromosome puffing <br /> 0.014                          0006911: phagocytosis, engulfment <br /> 0.018                          0045165: cell fate commitment <br /> 0.034                          0030154: cell differentiation <br /> 0.049                          0006259: DNA metabolic process <br />  <br /> Molecular Function <br />  <br /> Adjusted P-value               Attribute <br />  <br /> ,0.001                         0030528: transcription regulator activity <br /> ,0.001                         0003676: nucleic acid binding <br /> ,0.001                         0016251: general RNA polymerase II transcription factor activity <br /> ,0.001                         0003702: RNA polymerase II transcription factor activity <br /> 0.003                          0005515: protein binding <br /> 0.003                          0043565: sequence-specific DNA binding <br />  <br /> Cellular Component <br />  <br /> Adjusted P-value               Attribute <br />  <br /> ,0.001                         0005634: nucleus <br /> ,0.001                         0005654: nucleoplasm <br /> ,0.001                         0043227: membrane-bounded organelle <br /> ,0.001                         0031981: nuclear lumen <br /> ,0.001                         0016591: DNA-directed RNA polymerase II, holoenzyme <br /> ,0.001                         0031974: membrane-enclosed lumen <br /> ,0.001                         0043234: protein complex <br /> ,0.001                         0032991: macromolecular complex <br /> ,0.001                         0005667: transcription factor complex <br /> 0.003                          0000119: mediator complex <br /> 0.012                          0008540: proteasome regulatory particle, base subcomplex <br /> 0.015                          0005665: DNA-directed RNA polymerase II, core complex <br /> 0.032                          0005675: holo TFIIH complex <br /> 0.038                          0000123: histone acetyltransferase complex <br />  examination 202 validated screen MMS survival genes category enrichment Gene Ontologies; Biological Process, 46 genes having functional <br /> annotation; Molecular Function, 40 genes having functional annotation; Cellular Component, 68 genes having cellular component annotation, using <br /> FuncAssociate. <br /> doi:10.1371/journal.pgen.1000527.t002 <br />  <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                    5                               June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                             Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   6   June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                            Damage Survival Pathways <br />  <br />  <br />  <br /> Figure 2. Base excision repair, DNA damage response, proteasome pathways utilized MMS survival. CG numbers given Drosophila pathway component, protein names complex names human orthologues. Pathway entry points noted Roman numerals  end points  key following symbols provided. Symbols encircled lines <br /> represent proteins act complex, symbols encircled lines represent paralogues proteins substitute  Proteins affect MMS survival noted (death) triangles (resistance). Statistically significant proteins indicated black triangles, trend hits indicated grey triangles. Essential genes noted bar downwards upwards pointing boxes validated conferring death resistance, respectively, MMS knock  Shaded squares proteins hits validation, open squares tested validation. Yeast orthologues previously required MMS survival [3] noted dot symbol.   Base Excision Repair. (B) DNA Damage Response. (C) Proteasome. <br /> doi:10.1371/journal.pgen.1000527.g002 <br />  <br /> exposure Drosophila. general observed comparable results             activator (RBP-Jk) [26], result demonstrates MEFs Drosophila (compare Figures 4 5). MMS increased                functional Notch pathway response alkylation damage expo- AP sites MEFs (Figure 5A); absence                 sure. Taken  data suggest functional conservation <br /> XRCC1,  observed significant increase AP sites                MMS survival pathways responses similarly <br /> following MMS exposure (Figure 5A; p#3.7E-4). DDR                      acting agents. <br /> MEFs, assessed phosphorylation status Chk1  kinase  phosphorylated, phosphorylates p53) [24], total p53                Network integration MMS survival pathways <br /> protein, phosphorylation status p53 (Figure 5B),               307 identified MMS survival genes (202 validated screen <br /> examine p53 expression levels, obtain result equivalent           hits plus 105 pathway analyses), able assign 146 Drosophila result: DDR activated MMS exposure.             13 pathways (Figures 1, 2, 3 Figures S2, S3, S4, S5).  similar observation Drosophila cells, MEFs                161 MMS survival protein remain unassigned, examined increased proteasome activity following MMS treatment                         inter-relationship identified MMS survival proteins <br /> (Figure 5D; p#2.0E-4); knock proteasome                        using currently available Drosophila protein:protein interactome <br /> component, PSMC1 PSMD1 (orthologues Rpn2                            (PPI) map [27&#226;&#8364;&#8220;29]. measure connectivity MMS <br /> Pros26.4, respectively), resulted decreased proteasome activity            survival genes, took largest connected components following MMS exposure (PSMC1: p#8.1E-5; PSMD1: p#4.1E-                       PPI network discarded  smaller components. 4). results comparable Drosophila: clear            largest connected component contained 7364 7504 proteins <br /> proteasome activity responsive MMS MEFs.              original PPI network. Taking 202 original validated noted  able demonstrate dose-              MMS screen hits unbiased starting point, determined dependent decrease S6K phosphorylation following MMS                       number proteins directly connected  <br /> exposure MEFs seen Drosophila cells (Figure 5E)             compared random set number proteins knock TSC1 disrupted MMS-induced effect,                     PPI, observed significant enrichment (Figure 6A disruption evident observed Drosophila, probably           Table S7; p#2.1E-10). Similar results obtained inefficient knock TSC1 protein MEFs (data               assessments network connectivity (Table S7). MMS shown). Having observed functionality pathways            screen hits connections average set random <br /> response MMS ability siRNA modulate                 genes, biased analysis, MMS response respective pathway, examined                compared connectivity MMS screen hits real PPI effect knock MMS survival MEFs.                  network connectivity set proteins protein knock-downs, GCLC, PSMC1 PSMD1,                            randomly rewired PPI network. Degrees preserved affected MMS survival MEFs (data shown). Overall,                   random rewiring process. shown Table S7, MMS screen appear Drosophila used accurately predict              hits statistically significantly direct interactions MMS survival pathways mouse.                                               expected randomly rewired network (p = 0.01).                                                                               hand, randomized network smaller average <br /> Utilization MMS survival pathways response                       distance higher global efficiency real network, Temozolomide                                                                  attributed known property real-world <br />    Given MMS causes alkylation damage,                       networks: usually slightly longer average distances  reasonable suppose identified biological pathways                  correspondingly, lower global coefficients) degree- <br /> generally involved response alkylating agents.                preserving randomly rewired counterparts [30]. <br /> Temozolomide alkylating agent used clinically                       unable assess effect essential proteins <br /> treatment cancer [12,25]. demonstrated                 (proteins knock resulted cell death regardless BER TOR necessary Temozolomide survival                      treatment) MMS survival, repeated connectivity analysis <br /> [12,25]. Temozolomide causes DNA damage increasing                     including essential proteins connected number AP sites, similar MMS results, inhibition             MMS survival proteins, ones BER increases sensitivity temozolomide [12].              likely functional role MMS survival. increased <br /> reported <span id='am-1' about='obo:IMR_0002818' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="rdfs:label" content="rapamycin" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0002818" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C07909" datatype="xsd:string"></span>rapamycin</span> inhibition TOR pathway increased               size significantly improved connectivity cellular sensitivity temozolomide [25]. demonstrate             resultant network (Figure 6B Table S7; p#1.3E-26). utilization MMS survival pathways response                       pathway analysis, identified additional 105 MMS pathway <br /> temozolomide, examined DDR pathway, glutathione                        hits, validated. order expand network levels, proteasome activity MEFs observed                 include proteins relevant pathways, hits responses observed following MMS exposure (Figure S8).                     incorporated larger, equally connected  using HEK293 cells, observed MMS                    network observed (Figure 6C Table S7; p#4.6E-26). <br /> temozolomide exposure repressed Notch reporter activity (Figure               Considering apparent importance pathways S9 Text S1). Absence functional Notch protein                 individual genes, included components 13 <br /> shown repress activity downstream transcriptional                 identified MMS survival pathways observed  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               7                            June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                 Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 3. Glutathione TOR pathways utilized MMS survival. CG numbers given Drosophila pathway component, protein names complex names human orthologues. Pathway entry points noted Roman numerals  end <br /> points  key following symbols provided. Symbols encircled lines represent proteins act complex, symbols encircled lines represent paralogues proteins substitute  Proteins affect MMS <br /> survival noted (death) triangles (resistance). Statistically significant proteins indicated black triangles, trend hits indicated grey triangles. Essential genes noted bar downwards upwards pointing boxes validated conferring death resistance, respectively, MMS knock  Shaded squares proteins hits validation, open <br /> squares tested validation. Yeast orthologues previously required MMS survival [3] noted dot symbol.   Glutathione  expanded list GST family members given Table S4) (B) TOR. <br /> doi:10.1371/journal.pgen.1000527.g003 <br />  <br />  <br /> connectivity resultant enlarged network significantly                 essential genes pathway hits randomly rewired <br /> improved (Figure 6D Table S7; p#4.6E-52). Similar                  network consisting nodes. lower analysis 202 validated MMS screen hits,                    statistical significance, general trend observed (Table <br /> compared connectivity subnetworks containing                      S7). inclusive network allows view interactions <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                  8                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> Figure 4. examination functionality pathways utilized MMS survival Drosophila cells.   MMS exposure results increase AP sites. dsRNA knock BER component XRCC1 increases quantity AP sites observed following MMS exposure. (B) MMS <br /> exposure results increased p53 expression quantitative real-time PCR analysis; expression level provided fold change compared endogenous control (CG6905). (C) MMS results increased total glutathione, increase dependent rate limiting <br /> glutathione metabolizing enzyme glutamate cysteine ligase (GCLc). intracellular glutathione concentration expressed mmoles/L normalized mg protein. GCLc dependency correlates increased level GCLc expression observed following MMS exposure. (D) Proteasome activity increased following MMS exposure manner dependent proteasome components Rpn2 Pros26.4. Proteasome activity expressed units activity/cell, normalizing activity number cells using parallel viability assessment. (E) MMS exposure results dose-dependent <br /> decrease phosphorylation dTor component p70S6K. decrease dependent negative regulator dTor, Tsc1. Statistically <br /> significant differences unexposed MMS exposed cells denoted (*; **; ***; p-values ,0.05; ,0.01; ,0.001 respectively) differences Luc control dsRNA knock (#; ##; ###; p-values ,0.05; ,0.01; ,0.001 respectively). <br /> doi:10.1371/journal.pgen.1000527.g004 <br />  <br />  <br />  <br />  <br /> Figure 5. Functional conservation mouse embryonic fibroblasts pathways utilized MMS survival Drosophila cells.   MMS exposure results increase AP sites, siRNA based XRCC1 knock results increased quantity AP sites <br /> following MMS exposure. (B) MMS exposure results phosphorylation CHK1 p53 accumulation total p53 levels. (C) MMS exposure <br /> results increased total glutathione increase partially dependent GCLc. intracellular glutathione concentration expressed mmoles/L normalized mg protein. GCLc dependency correlates increased level GCLc expression observed following <br /> MMS exposure. (D) Proteasome activity increased following MMS exposure manner dependent proteasome components Pmsc1 Psmd1. Proteasome activity expressed units activity/cell, normalizing activity number cells using parallel viability <br /> assessment. (E) MMS exposure results dose-dependent decrease phosphorylation p70S6K. Statistically significant differences unexposed MMS exposed cells denoted (*; **; ***; p-values ,0.05; ,0.01; ,0.001 respectively) differences NT <br /> control siRNA knock (#; ##; ###; p-values ,0.05; ,0.01; ,0.001 respectively). <br /> doi:10.1371/journal.pgen.1000527.g005 <br />  <br />        PLoS Genetics | www.plosgenetics.org                                 9                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                              Damage Survival Pathways <br />  <br />  <br />  <br />  <br /> PLoS Genetics | www.plosgenetics.org   10   June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                             Damage Survival Pathways <br />  <br />  <br />  <br /> Figure 6. Protein network MMS survival proteins reveals integrated, highly connected MMS survival network.   150 validated MMS survival proteins (red nodes) represented known Drosophila interactome, protein:protein interactions represented grey edges; 104 validated hits &#226;&#8364;&#732;&#226;&#8364;&#732;orphans,&#226;&#8364;&#8482;&#226;&#8364;&#8482; connected proteins. (B) Nodes orphans  , including 11 proteins <br /> essential viability (black nodes) connected validated hits, improves connectivity DNA damage survival network. network, 93 hits orphans larger network 36 proteins, containing 29 validated hits, observed. (C) Nodes orphans (B), including additional pathway hits screen (green nodes). network, 86 screen hits additional 54 pathway hits orphans larger network 63 proteins, containing 33 validated screen hits, 20 additional pathway hits 10 essential proteins. (D) Nodes orphans (C), including non-hit members 13 MMS survival pathways (open nodes). network, 60 screen hits additional 40 pathway <br /> hits orphans larger network 247 proteins, containing 79 validated screen hits (26 pathways), 35 additional pathway <br /> hits, 101 non-hit pathway proteins, 32 essential proteins. (E) Network proteins validated required MMS treatment (red nodes), <br /> proteins essential viability (black nodes), 13 pathways (blue pathway nodes) validated involved MMS survival. proteins pathway, regardless hit, represented metanode, node size representing protein number. pathway components used create network, 54 proteins orphan, 96 proteins contained main interconnected network. Network <br /> connectivity coefficients given lower right-hand corner  D). <br /> doi:10.1371/journal.pgen.1000527.g006 <br />  connecting pathway, pathway                   recent studies demonstrated role proteasome components critical survival MMS.               regulating DNA repair pathways (reviewed [34]), <br /> qualitatively visualize result, protein known             supporting observation increased proteasome activity pathway grouped assigned &#226;&#8364;&#732;&#226;&#8364;&#732;pathway node&#226;&#8364;&#8482;&#226;&#8364;&#8482;                response MMS.  screen pathway identification  single node interactome retains interactions            revealed conserved set MMS survival pathways. constitutive proteins proteins external                Drosophila based study provided novel insights pathway). resulted highly connected interactome,                  global cellular response alkylation damage identifying <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival network&#226;&#8364;&#8482;&#226;&#8364;&#8482; (Figure 6E), encompassed 179                 biological pathways functions required survival 233 MMS survival proteins present PPI                   damage. analogous genome-wide, loss  <br /> network (77%). remaining 54 orphan proteins, 47                function screen MMS survival genes performed yeast protein removed MMS survival network,                 [14]. study highlighted general biological processes <br /> proteins away, connected  step           required MMS survival based gene ontology integrated analysis, observed improved network, enlarged             identified proteins disorganized network [14]. <br /> better connected, comparing entire set                    Considering pathways, signaling, metabolic, proteins network random set proteins PPI,           enzymatic, long identifiable entities, logical randomizing additional proteins added step               consider units network. experiments <br /> (data shown). Overall, observations MMS survival              focused identifying pathways required exposure network suggest despite general interconnectivity              damage validating biological responsiveness pathways <br /> protein interactomes, pathway analysis highly relevant            following damage exposure. results confirmed improve interactome connectivity simplifies systems            biological pathways functional yeast, Drosophila, biology overview.                                                              mouse cells functional contribution                                                                                biological pathways pertinent damage response network <br /> Discussion                                                                     representation. <br />                                                                                   addition functional conservation survival <br />    Studies performed using yeast model organism predict                pathway response alkylation damage, biological <br /> network response MMS reported astounding involvement                     pathways appear roles response types of diverse biological pathways [31]. Considering genes                damages. interesting compare results elegant respond environmental stress differ mammalian                  study Matsuoka et al. [4], identified proteins yeast cells [17], presumed damage response                   phosphorylated following ionizing radiation human cells. different complex higher eukaryotes, especially                  study identified proteins components 13 <br /> considering presence paralogues increased                      MMS survival pathways, including DNA repair <br /> genetic redundancy. performed genome-wide,                      pathways, DDR, mTOR, proteasome, basal transcription, RNAi based screen Drosophila cells investigate                   ribosome. results suggest different types damage, genes essential survival following damage exposure                just alkylation damage, utilize different components DNA <br /> MMS. able identify validate 307 MMS survival                    damage survival network functionally conserved manner genes, majority previously associated                reemphasize functional conservation pathways, alkylation damage survival. genes, 146                      individual genes, species. <br /> components 13 different MMS survival pathways.                        emphasis reorganize MMS survival network based pathways examined  observed                   pathways facilitate observation biologically <br /> functionally conserved yeast conserved mouse                relevant interactions. protein:protein interaction networks regard utilization following MMS treatment                       appear chaotic, interrogated simple sub- <br /> (Figure 5). yeast, experimental validation response                   networks associated protein(s) pathways [21]. <br /> MMS proteasome [19], TOR pathway [20], DNA                           working pleiotropic responses repair pathways [21] previously reported. Similarly,                       encompass different biological processes, glutathione response MMS observed mammalian cells                    DDR, chaotic network representation appears non-intuitive <br /> [32], observation increase GCLc expression                    (Figure 6D).  integration pathways provides underlying mechanism phenomenon.                      conceptual framework systems biology networking logical. <br /> demonstration dose-dependent increase 8-oxoguanine               additional advantage including pathways highlighted MMS exposure indicates MMS results oxidative stress               demonstration MMS survival protein enrichment damage, previous studies suggested [33]. Additionally,              detailed examination pathways. detailed <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                               11                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                        Damage Survival Pathways <br />  <br />  <br /> analysis convincing evidence pathways               Methods <br /> functioning expected, able assign protein pathway &#226;&#8364;&#732;&#226;&#8364;&#732;MMS survival protein.&#226;&#8364;&#8482;&#226;&#8364;&#8482;                Tissue culture possible explanations  nonetheless, considering               Kc167 cells grown Schneider medium (Invitrogen, pathways provide framework                       Carlsbad, CA) supplemented 10% heat inactivated fetal <br /> network highlights novel interactions, cross-talk,                  bovine serum (65uC, 10 minutes), penicillin streptomycin identified proteins mapped canonical pathway,                   22uC humidified chamber. <br /> present network, unlikely observed                     Primary MEFs obtained harvesting 14.5 day C57BL/6 <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;chaotic network view&#226;&#8364;&#8482;&#226;&#8364;&#8482; encourages investigation.            embryos previously described [48]. Briefly, fetal liver head <br /> approach similar computational clustering networks                  removed remainder embryo mechanically <br /> based signaling pathways using interactome datasets [35],             disaggregated plating medium. suspension single-cells approach includes identified proteins pathway                   plated Dulbecco&#226;&#8364;&#8482;s Modified Eagle&#226;&#8364;&#8482;s Medium (DMEM) <br /> components. representation simplified, using single                 supplemented 10% fetal calf serum, 2 mM glutamine, <br /> node represent entire pathway complex                   100 U/mL penicillin, 100 mg/mL streptomycin. MEFs display interactions component pathway [36].               grown passages freezing aliquots. Aliquots simplified pathway inclusive representation reveals highly            taken expanded needed experiment. <br /> organized network, consistent requirement pathway cellular survival (Figure 6E), provides                   Genome-wide viability screen following MMS exposure <br /> effective strategy integrate modular components                     Methods adapted [49], 1.26104 Kc167 cells <br /> network [37] inferring biological properties [38].               suspended 10 mL serum-free Schneider medium (Invitrogen) <br />    interconnectivity survival pathways                       added 384 plates. (Figure 5E) suggests potential pathway cross-talk. cross-            contains 0.25 mg double-stranded (ds) RNA, talk exists, highly pertinent cancer therapy.                 22,915 individual dsRNA represented library <br /> Recent studies demonstrated utility global level               (Version 1 dsRNA). Cells incubated 1 h 22uC, allowed <br /> analysis, allowing identification altered pathway function             phagocytic uptake dsRNA, 20 mL serum (15% heat- <br /> complex diseases Notch pathway pancreatic                     inactivated fetal bovine serum) containing medium added <br /> cancer [39] similarly DDR pathway breast                      plates incubated 72 h. Medium colorectal cancers [40]. Considering identification Notch,            exchanged fresh serum containing medium, TOR, DDR, proteasome &#226;&#8364;&#732;&#226;&#8364;&#732;survival pathways,&#226;&#8364;&#8482;&#226;&#8364;&#8482;                0.004% (w/v) MMS (Sigma-Aldrich, St. Louis, MO). Following currently explored targets cancer therapy             72 h additional incubation, medium removed number <br /> [41,42], identified survival network suggest                   viable cells assessed using pre-diluted Celltiter-Glo possibility combining pathway-specific pharmacological agents             manufacturers instructions (Promega, Madison, WI). Screens cancer therapy. pathway connections                       performed duplicate, replicate screen initiated potential cross-talk represented survival network                     different day. <br /> (Figure 6E) observed. example, protein <br /> phosphatase, PP2A, downstream component TOR                         Production dsRNA <br /> pathway [23], interacts DDR component regulate                      dsRNA produced similarly previously described [8,18]. <br /> phosphorylation ATM ATR [43] vice versa [4];                      Briefly, dsRNA synthesized using cDNA prepared gene <br /> DDR interacts BER CHK2 XRCC1 [44]; Notch                        specific amplification reverse transcribed cDNA. cDNA <br /> interacts DDR Mastermind p53 [45];                          preparation, RNA harvested adult D. melanogaster using <br /> glutathione pathway interacts nucleotide excision repair            Trizol reagent. gene specific amplification, primers designed <br /> pathway (NER) [46]; proteasome interacts various                DRSC target genes custom synthesized <br /> DNA repair components [34,47].  appear                overhanging T7 promoter sequences (Invitrogen). DRSC model integrated network conserved damage                       validation primer sequences obtained DRSC survival pathways valid biologically relevant.                   targeting genes  oligonucleotides purchased <br />    conclusion, identified network pathways              (Invitrogen). round PCR amplifications performed using functional role damage response affecting viability;         gene specific primers, amplified products gel demonstrated functional conservation species             purified, using agarose gel purification kit (Qiagen, Valencia, MMS survival pathways. considering protein                        CA) 96 gel isolation kit (Invitrogen). purified PCR <br /> interactions MMS survival proteins incorpo-               products used second round amplification using <br /> rating MMS survival pathways, highly interconnected                    gene specific primers; amplified products purified <br /> damage survival network observed encompasses                using Millipore PCR purification 96 plate (Millipore, <br /> 58% identified MMS survival proteins directly.                   Billercia, MA). purified products used dsRNA <br /> interconnectivity suggests strong functional interrelationship             synthesis using T7 Ribomax express (Promega) constitutive components survival network              purified using PCR purification plate (Millipore). possibility pathway cross-talk coordination level greater        quality purified dsRNA verified agarose gel electropho- just instigation DDR pathway.               resis  following spectrophotometric quantification, stored MMS survival pathways implicated MMS                    220uC. <br /> damage response, identified seemingly disparate <br /> pathways single screen, network analysis,             Validation analysis <br /> lends direct connection pathways response                    Validation MMS survival genes performed previously <br /> damage. pleiotropic effects alkylation damage               described [18], exception DRSC &#226;&#8364;&#732;&#226;&#8364;&#732;validation&#226;&#8364;&#8482;&#226;&#8364;&#8482; <br /> require wide-variety functioning pathways order cell         dsRNA (targeting gene mRNA transcript different location survive.                                                                  library amplicon), designed DRSC produced <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                              12                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                           Damage Survival Pathways <br />  <br />  house, preferentially used study. Briefly, dsRNA              Bradford&#226;&#8364;&#8482;s reagent (Biorad). estimation glutathione, equal validated quadruplicate, set plates treated          amounts protein lysate proteinized using metapho- <br /> MMS set used untreated control. Plates                  sphoric acid (MPA) final concentration MPA adjusted MMS dsRNA compared.                           5%. precipitated proteins cleared centrifugation    Genes required MMS treatment genes knock-                  4000 g 10 min 4uC, supernatant used conferred resistance MMS validated validation                 assay. <span id='am-24' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="glutathione" datatype="xsd:string"></span>Reduced glutathione</span> (Sigma) used standard, amplicons, available. validation amplicons                   samples arrayed 96 plate. step reaction available, amplicon used original library used             conducted, thioesterification intracellular thiols using 4-choloro- <br /> validation. instances, validation library                    1-methyl-7-trifluoromethyl-quinolinium methylsulfate followed amplicons used validation experiments. cases,                alkaline conversion glutathione-thioester chromophoric knock result validation amplicon preferred              thione, followed detection total glutathione absorbtion library amplicon, difference evident.        400 nm. <br /> gene annotated pathway determined involved MMS response, gene                       Quantitative Reverse-Transcriptase PCR <br /> considered hit regardless version amplicon validated.              dsRNA used target gene  dsRNA                                                                               used validate genes knock  Gene ontology                                                                 level RNAi mediated silencing gene expression    Gene ontology retrieved FlyBase (http://flybase.org/).            monitored quantitative real time RT&#226;&#8364;&#8220;PCR using QuantiTect <br /> Enrichment analysis conducted using FuncAssociate (http://                SYBR Green RT-PCR kit (Qiagen GmbH) ABI 7500 Real <br /> llama.med.harvard.edu/cgi/func/funcassociate/) ontol-               Time PCR (Applied Biosystems, Foster City, CA). ogies represented p-value 0.05.                experiments, 18.616104 Kc167 cells dispensed 24- <br /> ontology categories overlapped consolidated.                        tissue culture plate containing 6.2 mg dsRNA  <br />                                                                               Following 1 h incubation serum-free condition allow Westerns                                                                      uptake dsRNA, serum replenished final concentration <br />    Protein lysates prepared using radioimmunoprecipitation               10%. Day 3, RNA isolated using RNeasy Mini kit <br /> assay (RIPA) buffer containing 5% sodium deoxycholate, 0.1%                   (Qiagen) quantified using ND-1000 Spectrophotometer <br /> SDS, 0.1% Igepal PBS cocktail protease inhibitors                (Nanodrop, Wilmington, . PCR amplifications, distinct <br /> (1 mM PMSF, 1 mM Sodium Orthovandate 30 uL/mL                             primers encompassed dsRNA used Aprotinin (Sigma)). Protein concentration determined using                target gene used. gene expression analysis following <br /> Bradford Protein Assay Reagent (Biorad, Hercules, CA). Equal                  MMS exposure, RNA isolated Day 4. experiments <br /> amounts protein resolved using 10% SDS polyacrylamide                 Kc167 cells, CG6905, expression remained <br /> gel transferred nitrocellulose membrane (Hybond-                   unaltered following MMS exposure, used endogenous <br /> ECL, GE Healthcare Lifesciences, Piscataway, NJ). Using                       control. experiments MEFs, isolated RNA appropriate dilutions primary secondary antibodies,                    reverse transcribed using ImProm-II Reverse Transcription <br /> immunodetection protein performed using ECL                    (Promega) cDNA used TaqMan Gene <br /> plus (GE Healthcare Lifesciences). Primary antibodies                  Expression Assay kit GCLc, purchased (Applied <br /> include P-Chk1 (T68) (Abcam) total CHK1 (Abcam), P-                       Biosystems, CA) TaqMan Universal Master Mix (Applied <br /> p70S6K (T389) (Cell Signaling Technology) total p70S6K                    Biosystems). Mouse b-Actin TaqMan Gene Expression Assay kit <br /> (Cell Signaling Technology), P-p53 (S15) (Cell Signaling                  used endogenous control. level gene expression <br /> Technology) total p53 (Cell Signaling Technology).                        determined using nnCt method [50]. <br />  <br /> Apurinic sites measurement                                                    Proteasome measurement <br />    Apurinic apyrmidinic sites (AP) generated following              Experiments performed 384 plates using DNA damage measured using colorimetric assay kit                   layout timing described RNAi validation <br /> DNA damage quantification (Oxford Biomedical Science, Oxford,                 experiments, [18], proteasome activity measured <br /> MI), following manufacturer&#226;&#8364;&#8482;s protocol. Briefly, DNA isolated             Day 4 experiment, 24 h MMS exposure. experimental cells using Genomic DNA isolation kit                 proteasome measurement, Proteasome-Glo assay kit used <br /> (Oxford Biomedical Science). Equal amounts DNA                   (Promega), instructions. normalize proteasome <br /> labelled using biotinylated aldehyde reactive probe. Labelling              activity cell density, parallel experiment performed followed purification colorimetric quantification using            separate 384 plate, cell density estimated using <br /> streptavidin-horse radish peroxidase (HRP) conjugate HRP-                 CellTiter-Glo (Promega) described  Proteasome activity <br /> dependent substrate supplied kit. aldehyde reactive              normalized relative number cells present. <br /> probe labelled DNA standard, supplied kit, used determine number AP sites 100 kilobase pairs DNA                Orthologue identification experimental samples.                                                    Orthologues Drosophila genes human, mouse, <br />                                                                               yeast, obtained Ensembl49 (http://ensembl.org/). <br /> Glutathione measurement <br />    Intracellular glutathione levels measured using colori-             siRNA targeting MEFs <br /> metric assay kit glutathione (Oxford Biomedical Science),                    SMARTPool siRNAs purchased Dharmacon RNA <br /> following manufacturer&#226;&#8364;&#8482;s protocol. Briefly, cells RNAi                   technologies (Lafayette, . siRNA transfection MEFs, <br /> experiments resuspended ice-cold PBS homogenized                  cells harvested trypsinization, washed serum free <br /> using sonicator; lysate cleared centrifugation           medium, cell density adjusted 16106 cells 100 mL protein concentration supernatant determined using                 MEF-2 Nucleofector Solution (Amaxa, Gaithersburg, MD), <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              13                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                Damage Survival Pathways <br />  <br />  <br /> containing 0.5 mg siRNA, followed transfection                            itself) edges  clustering coefficient vi <br /> electroporation using Nucleofector II Device (Amaxa).                          measured total number edges gi divided transfected cells seeded 96 384                    maximum number edges possibly exist gi. tissue culture plate, required experiment. non-                      clustering coefficient network given average specific control, scrambled, non-targeting siRNA used.                     clustering coefficient vertex, high clustering <br /> fluorescent probe siGLO red (Dharmacon) used monitor                        coefficient indicating distinction real network efficiency siRNA uptake, efficiency protein                      random  assess statistical significance knock determined western blot analysis (Figure S3B).                   measurement, number proteins proteins pairs, Additional validation SMARTPool siRNA experiments                         appropriate, randomly sampled PPI PPI <br /> performed using minimum independent duplex siRNA                         subnetwork. sampling repeated 1,000 times estimate gene (Figure S10).                                                        p-value difference expected chance. <br />  <br /> Pathway identification                                                             Statistical analyses <br />    Pathway analysis conducted priori identification                 determine knock genes resulted increased <br /> protein/pathway relationships KegArray, yielding                    sensitivity MMS, raw data obtained viability assays included DNA damage response, glutathione metabolism,                       normalized statistically analyzed described previously <br /> TOR pathway, proteasome, DNA repair pathways.                            [18]. T-test performed normalized quadruplicates <br /> systematic analysis MMS hits, KegArray  default                        determine significant difference treated settings) used (http://genome.jp/download/), yielding Notch                    untreated wells. Percent control survival MMS treatment <br /> signaling, ATPase, basal transcription, ribosome, proteasome                   estimated experimental gene knock glutathione metabolism. pathways examined                       described previously [18], second T-test performed number hits identified pathway versus number total                percent viability compared luciferase control pathway components. Pathways included prior <br />                                                                                    plate described [18]. analyses, <br /> knowledge pathways retrieved KegArray                                                                                    significant hits selected death hits greater relatively small size completeness KEGG <br />                                                                                    55% viability treated wells compared untreated p- <br /> database fly. <br />                                                                                    value 0.05 resulted T-tests.                                                                                    genes essential phenotype 40% viability <br /> Interactome analysis                                                               non-targeting dsRNA, stringent requirements    Protein interactome data obtained IntAct [27]                         viability effect MMS treatment, 30&#226;&#8364;&#8220;40% <br /> (http://www.ebi.ac.uk/intact/), Database Interacting                        viability untreated needed 15% viability MMS <br /> Proteins [28] (DIP, http://dip.doe-mbi.ucla.edu/) Bio- <br />                                                                                    treatment, 20&#226;&#8364;&#8220;30% needed 10% treatment, 10&#226;&#8364;&#8220;20% needed <br /> molecular Interaction Network Database [29] (BIND, http:// <br />                                                                                    5% treatment, 0&#226;&#8364;&#8220;10% dead determine bond.unleashedinformatics.com/). Data databases <br />                                                                                    MMS effect. Genes considered &#226;&#8364;&#732;&#226;&#8364;&#732;trend death hits&#226;&#8364;&#8482;&#226;&#8364;&#8482; combined single interactome, using CG number                                                                                    exhibited 55% viability treatment did gene identifier protein. Interactomes                                                                                    p-value significance exhibited 55&#226;&#8364;&#8220;65% <br /> visualized using Cytoscape 2.6 (http://www.cytoscape.org/). <br />                                                                                    viability MMS treatment significant p-value. Knock- <br /> Pathway nodes created external Cytoscape renaming <br />                                                                                    genes resulting resistance MMS confirmed nodes representing protein pathways                                                                                    greater 85% viability treatment compared pathway. protein multiple pathways, represented relevant pathway nodes. Interactions                    untreated wells, p-values 0. 0001 <br /> pathways trimmed interaction                    viability treatment 60% non- <br /> existed pathways. instance, protein:protein                   targeting knock  <br /> interaction occurs members NER pathway,                            determine gene essential viability, T-test proteins exist BER pathway,                     performed untreated wells comparing values non-targeting <br /> interaction likely pathway specific cross-talk               dsRNA luciferase dsRNA targeting gene. pathways removed.                                        targeting dsRNA resulted 70% viability luciferase <br />                                                                                    p-value 0.00001, gene deemed essential. <br /> Connectivity analyses                                                                 Comparisons predicted observed numbers    measurements connectivity  (1) Counting                   MMS survival genes yeast Drosophila studies number pairs MMS survival proteins directly interact                 standard G-test [52]. G-test equivalent interaction network, (2) computing average geodesic                        contingency Chi-square test allows classes zero <br /> distance  e., number edges shortest path)               events. <br /> pair MMS survival proteins interaction network [51], (3)                    determine protein enrichment pathway global efficiency network [51], (4) clustering                  unbalanced distribution data numbers coefficient [51]. number direct interactions provides                    proteins pathway compared total number genes <br /> intuitive measure connectivity subnetwork,                   genome, Fisher&#226;&#8364;&#8482;s Exact test employed [53]. <br /> average distance measures global connectivity sub-                         connectivity measurements, case clustering <br /> network. Global efficiency provides similar measure average                   coefficient, p-values estimated using test, given distance, allows disconnected                                              connectivity measurement random subnetworks approxi- <br />                                  1 <br />                                       P components.    1 <br />                                                                global <br /> efficiency measured N &#195;&#176;N{1   &#195;&#382;    j~1::N, j dij , n        mately follows normal distribution (Table S6). p-value number vertices network dij geodesic distance                 clustering coefficient measurement estimated simply vertices vi vj. clustering coefficient measure                counting frequency clustering coefficient local connectivity network. vertex vi, let gi            randomly sampled sub-network high subnetwork consists direct neighbours vi (excluding vi                  real sub-network. <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                   14                           June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                        Damage Survival Pathways <br />  <br />  <br />  <br /> Supporting Information                                                      knock resulted significant difference MMS <br />                                                                             survival control shown black (death) stripped <br /> Figure S1 MMS dose response measured cell survival.               (resistance), trend effect shown grey. Viability Drosophila Kc167 cells following exposure increasing        complete list proteins human yeast <br /> dose MMS determined using CellTiter-Glo (relative light units            orthologues given Table S6.   Basal Transcription. (B) <br /> (ordinate) increasing dose MMS (abscissa)).                          Ribosome.  doi:10.1371/journal.pgen.1000527.s001 (0.54 MB EPS)                doi:10.1371/journal.pgen.1000527.s003 (3.26 MB EPS) <br /> Figure S2 additional pathways utilized                Figure S4 additional pathways utilized MMS survival. CG numbers given Drosophila pathway              MMS survival. CG numbers given Drosophila pathway <br /> component, protein names complex names                component, protein names complex names human orthologues. Pathway entry points noted                human orthologues. Pathway entry points noted Roman numerals  end points               Roman numerals  end points  key following symbols provided. Symbols encircled           key following symbols provided. Symbols encircled lines represent proteins act complex,           lines represent proteins act complex, symbols encircled lines represent paralogues             symbols encircled lines represent paralogues proteins substitute  Proteins             proteins substitute  Proteins affect MMS survival noted (death) triangles               affect MMS survival noted (death) triangles <br /> (resistance). Statistically significant proteins indicated         (resistance). Statistically significant proteins indicated black triangles, trend hits indicated grey triangles.        black triangles, trend hits indicated grey triangles. <br /> Essential genes noted bar                     Essential genes noted bar downwards upwards pointing boxes validated                  downwards upwards pointing boxes validated conferring death resistance, respectively, MMS knock-            conferring death resistance, respectively, MMS knock-  Shaded squares proteins hits                 Shaded squares proteins hits validation, open squares tested validation.             validation, open squares tested validation. <br /> Yeast orthologues previously required MMS                   Yeast orthologues previously required MMS <br /> survival [3] noted dot symbol. example           survival [3] noted dot symbol. example average percent untreated control survival validated hits          average percent untreated control survival validated hits shown pathway, represent               shown pathway, represent actual control data point graph. Error bars         actual control data point graph. Error bars standard deviation quadruplicates. Survival control cells         standard deviation quadruplicates. Survival control cells dsRNA targeting luciferase shown open bar, protein            dsRNA targeting luciferase shown open bar, protein <br /> knock resulted significant difference MMS                 knock resulted significant difference MMS <br /> survival control shown black (death) stripped           survival control shown black (death) stripped <br /> (resistance), trend effect shown grey.            (resistance), trend effect shown grey. complete list proteins human yeast                   complete list proteins human yeast <br /> orthologues given Table S6.   Nucleotide Excision Repair.           orthologues given Table S6.   ATPases. (B) Notch. <br /> (B) Mismatch Repair. (C) Homologous Recombination Repair. (D)                doi:10.1371/journal.pgen.1000527.s004 (2.47 MB EPS) <br /> RecQ Helicases. <br />                                                                             Figure S5 Average percent untreated control survival  doi:10.1371/journal.pgen.1000527.s002 (1.40 MB EPS) <br />                                                                             validated hits pathway represented Figures 2 3. <br /> Figure S3 additional pathways utilized                 Error bars standard deviation quadruplicates. Survival <br /> MMS survival. CG numbers given Drosophila pathway              control cells dsRNA targeting luciferase shown component, protein names complex names                open bar average plates general reference. human orthologues. Pathway entry points noted                noted plate luciferase controls <br /> Roman numerals  end points               plate values compared. Protein knock key following symbols provided. Symbols encircled           resulted significant difference MMS survival lines represent proteins act complex,           internal plate control shown black (death) dashed symbols encircled lines represent paralogues             (resistance), trend effect shown grey. proteins substitute  Proteins             complete list proteins human yeast <br /> affect MMS survival noted (death) triangles               orthologues given Table S6. <br /> (resistance). Statistically significant proteins indicated          doi:10.1371/journal.pgen.1000527.s005 (4.48 MB EPS) <br /> black triangles, trend hits indicated grey triangles. <br />                                                                             Figure S6 Efficiency RNAi mediated silencing gene <br /> Essential genes noted bar                                                                             expression.   RNAi transfection resulted decreased expression <br /> downwards upwards pointing boxes validated                                                                             target mRNA Drosophila Kc167 cells, measured quantitative <br /> conferring death resistance, respectively, MMS knock- <br />                                                                             real time PCR indicated percent control gene expression  Shaded squares proteins hits                                                                             (ordinate) genes (abscissa) tested, expression <br /> validation, open squares tested validation. <br />                                                                             normalized endogenous control (CG6905). (B) Western blot <br /> Yeast orthologues previously required MMS <br />                                                                             analysis efficiency RNAi siRNA transfected primary <br /> survival [3] noted dot symbol. example                                                                             mouse embryonic fibroblast cells. <br /> average percent untreated control survival validated hits                                                                              doi:10.1371/journal.pgen.1000527.s006 (0.74 MB EPS) <br /> shown pathway, represent actual control data point graph. Error bars         Figure S7 MMS exposure results dose-dependent increase standard deviation quadruplicates. Survival control cells         8-oxoguanine DNA modifications. MMS-dependent dsRNA targeting luciferase shown open bar, protein            increase 8-oxoguanine   observed qualitatively  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             15                          June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                              Damage Survival Pathways <br />  <br />  <br /> microscopic examination 8-oxoguanine containing cells                    Table S3 List yeast MMS hits fly orthologues. Yeast <br /> observed fluorescence FITC channel (B)                 MMS hits determined Begley et al. [3], fly quantified percentage fluorescent field                   orthologues hit MMS screen (ordinate) increasing dose MMS (abscissa).                             pathway identified screen. given fly  doi:10.1371/journal.pgen.1000527.s007 (7.71 MB EPS)                 gene essential, resistance hit, death hit, death tread <br />                                                                               validation independent dsRNA. <br /> Figure S8      Temozolomide exposure results functional  <br />                                                                                doi:10.1371/journal.pgen.1000527.s013 (0.04 MB <br /> sponse MMS survival pathways mouse embryonic <br />                                                                               XLS) <br /> fibroblasts.   Temozolomide exposure results phosphorylation p53 accumulation total p53 levels. (B)                        Table S4 Drosophila GST family members involvement <br /> Temozolomide exposure results increased total                 MMS survival. GST families component <br /> glutathione. intracellular glutathione concentration ex-               members. Provided gene  corresponding CG number, <br /> pressed units activity/cell. (C) Proteasome activity                 validated significant MMS survival gene (death), <br /> increased following temozolomide exposure, normalizing                    trend (death trend) involved MMS survival  . activity number cells using parallel viability assessment.        validation observed effect, type amplicon, <br /> Proteasome activity expressed units activity/cell,                   library validation given. noted validated <br /> normalizing activity number cells using parallel              MMS survival gene effect observed viability assessment.                                                         screen. validation performed noted   doi:10.1371/journal.pgen.1000527.s008 (0.71 MB EPS)                 tested). <br />                                                                                doi:10.1371/journal.pgen.1000527.s014 (0.02 MB <br /> Figure S9 MMS Temozolomide exposure results                            XLS) <br /> functional response notch signalling pathway HEK 293 <br /> cells. MMS temozolomide exposure results decreased                     Table S5 Summary number MMS survival genes <br /> luciferase activity notch reporter RBP-Jk, normalized              identified MMS survival pathway (BER: base excision repair; <br /> transfection efficiency HEK 293 cells using renilla luciferase.            DDR: DNA damage response; Proteasome; GSH: glutathione  doi:10.1371/journal.pgen.1000527.s009 (0.41 MB EPS)                 synthesis; TOR: TOR pathway; NER: nucleotide excision repair; <br />                                                                               MMR: mismatch repair; HRR: homologous recombination <br /> Figure S10     Validation pathway functions additional                repair; RECQ: RecQ helicases; Transcription: basal transcription; <br /> siRNA knock mouse embryonic fibroblasts.   Knock-                   Ribosome; ATPase; Notch: Notch signaling pathway), GCLc different siRNA results decreased                   percentage non-essential genes pathway GCLc expression mouse cells, quantitative real-time PCR                 represents. Protein enrichment pathway compared <br /> analysis; expression level provided fold-change compared             number genes validated (202 13826 prior endogenous control (mouse b-actin). (B) MMS results                  pathway analysis 307 13826 including pathway analysis) increased total glutathione, increase                   determined using Fisher&#226;&#8364;&#8482;s Exact Test (NS: Significant; NA: <br /> dependent rate limiting glutathione metabolizing enzyme              Applicable). <br /> glutamate-cysteine ligase (GCLc). (C) Knock proteasome                 doi:10.1371/journal.pgen.1000527.s015 (0.51 MB PDF) <br /> components Psmc1 Psmd1 different siRNA results decreased gene expression mouse cells, quantitative real-time           Table S6 orthologous relationship MMS <br /> PCR analysis; expression level provided fold-change                     survival genes identified Drosophila yeast MMS survival <br /> compared endogenous control (mouse b-actin). (D)                        screens. Provided 13 MMS survival pathway, gene <br /> Proteasome activity increased following MMS exposure                  names Drosophila component corresponding CG <br /> manner dependent proteasome components Pmsc1                     number. Drosophila genes, yeast ORF Psmd1.                                                                    provided identifiable yeast orthologue  doi:10.1371/journal.pgen.1000527.s010 (0.81 MB EPS)                 observed involved MMS survival (hit) S. cerevisiae                                                                               Begley et al. [3], identified  hit) essential <br /> Table S1 Summary MMS screen hits pathway genes                         (essential). orthologues species known selected validation. 534 Drosophila MMS screen hits                   taking account yeast essential genes selected validation analysis 298 pathway genes tested             assayed effect MMS survival, clear <br /> pathway analysis, provided FlyBase gene number, corre-                   enrichment MMS survival genes majority sponding CG number, dsRNA used validation, noted                  pathways examined species. complete list DRSC identification number, library dsRNA                        proteins human orthologues given. <br /> validation dsRNA. genes validation                       doi:10.1371/journal.pgen.1000527.s016 (0.11 MB <br /> amplicon designed library amplicon                      XLS) <br /> potential target effects 19 nt, data [18] used <br /> (asterisks). dsRNA, significant death, trend death                Table S7 Connectivity analysis MMS survival genes identified <br /> significant resistance MMS treatment noted (MMS survival).              RNAi genomic screen following validation analyses. <br />                                                                               Provided number proteins identified, number  doi:10.1371/journal.pgen.1000527.s011 (0.94 MB <br />                                                                               proteins protein:protein interactome <br /> XLS) <br />                                                                               (PPI), number direct connections proteins, <br /> Table S2    Raw, normalized, survival data validation                 average distance possible pair proteins experiment. data normalization, raw data untreated control             network compared expected values, global efficiency MMS experiments normalized using luciferase (Luc)                clustering coefficient. measurement expected <br /> high MMS controls statistical significance determined              number direct interactions derived number described [18].                                                               proteins randomly selected 1000 times PPI. Analyses  doi:10.1371/journal.pgen.1000527.s012 (0.23 MB                      provided proteins identified MMS screen prior XLS)                                                                          validation (Screen hits), screen hits actually validated <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                              16                             June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                                  Damage Survival Pathways <br />  <br />  <br /> (Validated hits), validated hits essential proteins                      Acknowledgments connected hits (Validated+essential connec- <br /> tors), total number validated MMS survival proteins                                   thank Thomas Begley, University Albany sharing yeast MMS <br />                                                                                              screen data. UTHSCSA Center Epidemiology Biostatistics, <br /> identified screen pathway validation (Validated+path- <br />                                                                                              thank Gary Chisholm performed initial statistical analyses way hits), Validated pathway hits essential                                  John Cornell Jonathon Gelfond discussions guidance. proteins connected hits (Validated+path-                             thank Fen Zhou Montserrat Michelman Leder laboratory <br /> way hits+essential connectors), total number validated MMS                            cell line maintenance. thank Kent Nybakken, Ram Dasgupta, survival proteins components 13 MMS survival                            Herve&#194;&#180; Agaisse Norbert Perrimon&#226;&#8364;&#8482;s laboratory cells, plasmids, pathways (Validated pathways), finally total number                           technical discussions Lisa Ching helping production of validated MMS survival proteins, components                              dsRNA. David Rodrigez GCCRI, UTHSCSA, provided support 13 MMS survival pathways essential proteins                                 production line art. Thank members Bishop connected hits pathway proteins (Validated                             Leder laboratories, Dr. Don McEwen particularly Dr. <br />                                                                                              Vivienne Rebel constructive criticism manuscript. pathways+essential connectors). Analyses provided compare connectivity real randomly rewired PPI.  doi:10.1371/journal.pgen.1000527.s017 (0.19 MB                                     Author Contributions <br /> DOC)                                                                                         Conceived designed experiments: DR PL AJRB. Performed                                                                                              experiments: DR SB AJRB. Analyzed data: DR AMW SB JR AJRB. <br /> Text S1       Methods 8-oxo-<span id='am-72' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span> assay notch reporter                             Contributed reagents/materials/analysis tools: PL AJRB. Wrote paper: <br /> analysis.                                                                                    DR AMW JR AJRB. Reviewed manuscript: PL.  doi:10.1371/journal.pgen.1000527.s018 (0.03 MB <br /> DOC) <br />  <br /> References <br />  1. Altieri F, Grillo C, Maceroni M, Chichiarelli S (2008) DNA damage repair:            20. Shen C, Lancaster CS, Shi B, Guo H, Thimmaiah P, et al. (2007) TOR <br />     molecular mechanisms health implications. Antioxid Redox Signal 10:                  signaling determinant cell survival response DNA damage. Mol Cell <br />     891&#226;&#8364;&#8220;937.                                                                                     Biol 27: 7007&#226;&#8364;&#8220;7017. <br />  2. Shimada M, Nakanishi M (2006) DNA damage checkpoints cancer. J Mol                   21. Rusyn  Fry RC, Begley TJ, Klapacz J, Svensson JP, et al. (2007) <br />     Histol 37: 253&#226;&#8364;&#8220;260.                                                                          Transcriptional networks S. cerevisiae linked accumulation base <br />  3. Begley TJ, Rosenbach  Ideker T, Samson LD (2004) Hot spots                            excision repair intermediates. PLoS 2: e1252. doi:10.1371/journal.- <br />     modulating toxicity identified genomic phenotyping localization                       pone.0001252. <br />     mapping. Mol Cell 16: 117&#226;&#8364;&#8220;125.                                                           22. Dickinson DA, Levonen AL, Moellering DR, Arnold EK, Zhang H, et al. (2004) <br />  4. Matsuoka S, Ballif BA, Smogorzewska  McDonald ER 3rd, Hurov KE, et al.                     Human glutamate cysteine ligase gene regulation electrophile <br />     (2007) ATM ATR substrate analysis reveals extensive protein networks                     response element. Free Radic Biol Med 37: 1152&#226;&#8364;&#8220;1159. <br />     responsive DNA damage. Science 316: 1160&#226;&#8364;&#8220;1166.                                        23. Dann SG, Thomas G (2006) <span id='am-65' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-66' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> sensitive TOR pathway yeast <br />  5. Workman CT, Mak HC, McCuine S, Tagne JB, Agarwal M, et al. (2006)                          mammals. FEBS Lett 580: 2821&#226;&#8364;&#8220;2829. <br />     systems approach mapping DNA damage response pathways. Science 312:                   24. Ou YH, Chung PH, Sun TP, Shieh SY (2005) p53 C-terminal phosphorylation <br />     1054&#226;&#8364;&#8220;1059.                                                                                   CHK1 CHK2 participates regulation DNA-damage-induced C- <br />  6. Kelley R, Ideker T (2005) Systematic interpretation genetic interactions using            terminal acetylation. Mol Biol Cell 16: 1684&#226;&#8364;&#8220;1695. <br />     protein networks. Nat Biotechnol 23: 561&#226;&#8364;&#8220;566.                                            25. Sinnberg T, Lasithiotakis K, Niessner H, Schittek B, Flaherty KT, et al. (2008) <br />  7. Ng  Bursteinas B, Gao Q, Mollison E, Zvelebil M (2006) Resources                       Inhibition PI3K-AKT-mTOR Signaling Sensitizes Melanoma Cells     integrative systems biology: data databases networks                     Cisplatin Temozolomide. J Invest Dermatol. <br />     dynamic models. Brief Bioinform 7: 318&#226;&#8364;&#8220;330.                                       26. Kao HY, Ordentlich P, Koyano-Nakagawa N, Tang  Downes M, et al. (1998) <br />  8. Ramadan N, Flockhart  Booker M, Perrimon N, Mathey-Prevot B (2007)                         histone deacetylase corepressor complex regulates Notch signal <br />     Design implementation high-throughput RNAi screens cultured                        transduction pathway. Genes Dev 12: 2269&#226;&#8364;&#8220;2277. <br />     Drosophila cells. Nat Protoc 2: 2245&#226;&#8364;&#8220;2264.                                               27. Kerrien S, Alam-Faruque Y, Aranda B, Bancarz  Bridge  et al. (2007) <br />                                                                                                  IntAct&#226;&#8364;&#8220;open source resource molecular interaction data. Nucleic Acids Res <br />  9. Drablos F, Feyzi E, Aas PA, Vaagbo CB, Kavli B, et al. (2004) Alkylation <br />                                                                                                  35: D561&#226;&#8364;&#8220;565. <br />     damage DNA RNA&#226;&#8364;&#8220;repair mechanisms medical significance. DNA <br />                                                                                              28. Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, et al. (2004)     Repair (Amst) 3: 1389&#226;&#8364;&#8220;1407. <br />                                                                                                  Database Interacting Proteins: 2004 update. Nucleic Acids Res 32: <br /> 10. Rydberg B, Lindahl T (1982) Nonenzymatic methylation DNA                                                                                                  D449&#226;&#8364;&#8220;451. <br />     intracellular methyl group donor <span id='am-47' about='obo:IMR_0200054' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-48' property="rdfs:label" content="S-Adenosyl-L-methionine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200054" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00019" datatype="xsd:string"></span>S-adenosyl-L-methionine</span> potentially <br />                                                                                              29. Alfarano C, Andrade CE, Anthony K, Bahroos N, Bajec M, et al. (2005)     mutagenic reaction. Embo J 1: 211&#226;&#8364;&#8220;216. <br />                                                                                                  Biomolecular Interaction Network Database related tools 2005 update. <br /> 11. Hecht SS (1999) DNA adduct formation tobacco-specific N-nitrosamines.                   Nucleic Acids Res 33: D418&#226;&#8364;&#8220;424. <br />     Mutat Res 424: 127&#226;&#8364;&#8220;142.                                                                  30. Watts DJ, Strogatz SH (1998) Collective dynamics &#226;&#8364;&#732;small-world&#226;&#8364;&#8482; networks. <br /> 12. Liu L, Taverna P, Whitacre CM, Chatterjee S, Gerson SL (1999) Pharmacologic                  Nature 393: 440&#226;&#8364;&#8220;442. <br />     disruption base excision repair sensitizes mismatch repair-deficient -            31. Begley TJ, Samson LD (2004) Network responses DNA damaging agents. <br />     proficient colon cancer cells methylating agents. Clin Cancer Res 5:                      DNA Repair (Amst) 3: 1123&#226;&#8364;&#8220;1132. <br />     2908&#226;&#8364;&#8220;2917.                                                                               32. Lynn S, Yew FH, Hwang JW, Tseng MJ, Jan KY (1994) Glutathione rescue <br /> 13. Malet-Martino M, Gilard V, Martino R (1999) analysis cyclophospha-                    inhibitory effects nickel DNA ligation repair synthesis. <br />     mide metabolites. Curr Pharm Des 5: 561&#226;&#8364;&#8220;586.                                         Carcinogenesis 15: 2811&#226;&#8364;&#8220;2816. <br /> 14. Begley TJ, Rosenbach  Ideker T, Samson LD (2002) Damage recovery                      33. Mizumoto K, Glascott PA Jr, Farber JL (1993) Roles oxidative stress     pathways Saccharomyces cerevisiae revealed genomic phenotyping                     poly(<span id='am-6' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>-ribosyl)ation killing cultured hepatocytes methyl <br />     interactome mapping. Mol Cancer Res 1: 103&#226;&#8364;&#8220;112.                                              methanesulfonate. Biochem Pharmacol 46: 1811&#226;&#8364;&#8220;1818. <br /> 15. Jelinsky SA, Samson LD (1999) Global response Saccharomyces cerevisiae             34. Harper JW, Elledge SJ (2007) DNA damage response: years  Mol <br />     alkylating agent. Proc Natl Acad Sci U S 96: 1486&#226;&#8364;&#8220;1491.                                 Cell 28: 739&#226;&#8364;&#8220;745. <br /> 16. Lee MW, Kim BJ, Choi HK, Ryu MJ, Kim SB, et al. (2007) Global protein                    35. Baudot  Angelelli JB, Guenoche  Jacq B, Brun C (2008) Defining modular <br />     expression profiling budding yeast response DNA damage. Yeast 24:                   signalling network fly interactome. BMC Syst Biol 2: 45. <br />     145&#226;&#8364;&#8220;154.                                                                                 36. Barrios-Rodiles M, Brown KR, Ozdamar B, Bose R, Liu  et al. (2005) High- <br /> 17. Murray JI, Whitfield ML, Trinklein ND, Myers RM, Brown PO, et al. (2004)                     throughput mapping dynamic signaling network mammalian cells. <br />     Diverse specific gene expression responses stresses cultured human                 Science 307: 1621&#226;&#8364;&#8220;1625. <br />     cells. Mol Biol Cell 15: 2361&#226;&#8364;&#8220;2374.                                                      37. Hwang D, Smith JJ, Leslie DM, Weston AD, Rust AG, et al. (2005) data <br /> 18. Wiles  Ravi D, Bhavani S, Bishop AJ (2008) analysis normalization                   integration methodology systems biology: experimental verification. Proc <br />     methods Drosophila RNAi genomic screens development robust                      Natl Acad Sci U S 102: 17302&#226;&#8364;&#8220;17307. <br />     validation scheme. J Biomol Screen 13: 777&#226;&#8364;&#8220;784.                                          38. Vidal M (2005) Interactome modeling. FEBS Lett 579: 1834&#226;&#8364;&#8220;1838. <br /> 19. Burgis NE, Samson LD (2007) protein degradation response                          39. Jones S, Zhang X, Parsons DW, Lin JC, Leary RJ, et al. (2008) Core signaling <br />     Saccharomyces cerevisiae classical DNA-damaging agents. Chem Res                          pathways human pancreatic cancers revealed global genomic analyses. <br />     Toxicol 20: 1843&#226;&#8364;&#8220;1853.                                                                       Science 321: 1801&#226;&#8364;&#8220;1806. <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                            17                                June 2009 | Volume 5 | Issue 6 | e1000527 <br />                                                                                                                                                Damage Survival Pathways <br />  <br />  <br /> 40. Lin J, Gan CM, Zhang X, Jones S, Sjoblom T, et al.  <br /> </body></html>