<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>RNase Y, novel endoribonuclease, initiates <br /> riboswitch turnover Bacillus subtilis <br />  <br />                                              pyrophosphatase RppH shown rate <br />                                                                  limiting step 50 end dependent action RNase E                                                                                         mRNAs (Deana et al, 2008). Inactivation RNase E increases <br />  <br />                                     bulk mRNA stability (Babitzke Kushner, 1991)                                                                                         abundance 440% E. coli mRNAs (Lee et al, 2002). contrast Escherichia coli, initiation mRNA decay                                B. subtilis ortholog RNase E essential <br /> Gram-positive organisms poorly understood. studied                                RNase J1 (rnjA) paralog RNase J2 (rnjB) endo- fate highly structured RNAs generated pre-                                nucleolytic cleavage specificity similar RNase E <br /> mature transcription termination S-adenosylmethionine                                 et al, 2005). 50 &#226;&#8364;&#8220;30 exonucleolytic <br /> (SAM)-dependent riboswitches Bacillus subtilis.                                   activity (Mathy et al, 2007) sensitive phos- <br /> essential protein earlier unknown function, YmdA,                                    phorylation state 50 end, monophosphorylated identified novel endoribonuclease  called                                  hydroxylated 50 ends preferred substrates  et al, <br /> RNase Y) capable preferential cleaving vitro                             2005; Mathy et al, 2007; Li la Sierra-Gallay et al, 2008). 50 monophosphorylated yitJ riboswitch upstream                                   X-ray crystallographic structure RNase J1 showed pre- SAM-binding aptamer domain. Antiterminated                                  sence mononucleotide-binding pocket <span id='am-41' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-42' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-44' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> <br /> length yitJ mRNA substrate RNase Y                                        distance catalytic site accommodate vivo vitro, transcripts capable forming                                <span id='am-36' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span> group (Li la Sierra-Gallay et al, 2008). <br /> antiterminator cleaved presence SAM.                                    RNases J1 J2 overlapping substrate specificities <br /> Turnover 10 SAM-dependent riboswitches                                     affect expression levels hundreds genes initiated RNase Y. Depletion ribonuclease                               (Ma&#194;&#168;der et al, 2008). Individual case studies shown increased half-life bulk mRNA fold.                                RNases J1/J2 effectively involved degradation indicates RNase Y important                                 specific structural mRNAs fine-tuning expression riboswitch RNA turnover key player                             polycistronic transcript processing event initiation mRNA decay B. subtilis. 40%                               generates matured transcripts differential stabilities <br /> sequenced eubacterial species RNase Y orthologue.                               (Ma&#194;&#168;der et al, 2008). theory, polyvalence RNases EMBO Journal (2009) 28, 3523&#226;&#8364;&#8220;3533. doi:10.1038/                                     J1/J2 explain characteristics described mRNA <br /> emboj.2009.283; Published online 24 September 2009                                      degradation B. subtilis, including decay-initiating clea- <br /> Subject Categories: RNA; cellular metabolism                                            vage followed 50 exonuclease degradation.  Keywords: Bacillus subtilis; endoribonuclease; mRNA degra-                              rnjA/rnjB double mutant (RNase J1 depleted) half- <br /> dation; riboswitch; RNase Y                                                             life global mRNA slightly increased  et al, <br />                                                                                         2005) observation does support idea                                                                                         important function RNases J1/J2 initiating global <br />                                                                                         mRNA decay. generally, direct evidence endonu- <br />                                                                                         cleolytic cleavage major function mRNA decay Introduction                                                                            B. subtilis elusive. degradation mRNA vital process organisms.                                Secondary structures untranslated leader regulating process, cells control gene expression                           common feature B. subtilis mRNAs. gain react rapidly changing growth conditions. Bacterial                              insight events initiating decay structured mRNA <br /> mRNA degradation best studied model                                 studied fate S-adenosylmethionine (SAM)- <br /> organisms Escherichia coli Bacillus subtilis                                dependent riboswitches. conserved genetic revealed surprising differences enzymes pathways                                 (S-box) controls expression 11 transcriptional units Gram-negative Gram-positive bacteria.                                        26 genes) involved biosynthesis transport E. coli, mRNA decay initiated endonucleolytic                                  methionine SAM B. subtilis (Grundy Henkin, <br /> cleavage followed 30 &#226;&#8364;&#8220;50 exonucleolytic degradation.                                 1998, 2003; Winkler Breaker, 2005; Henkin, 2008). major player process RNase E, essential                               Regulation occurs level premature transcription <br /> endoribonuclease significant preference 50 mono-                             termination (Grundy Henkin, 1998). presence phosphorylated substrates (Carpousis et al, 2009).                                  methionine  SAM levels high) SAM binds directly removal 50 triphosphate primary transcripts                              leader RNA. causes structural rearrangement                                                                                         prevents formation antiterminator  forms <br /> *Corresponding author. IBPC, CNRS UPR9073, Institut Biologie                         preferentially absence effector) stabilizing Physico-Chimique, 13, rue Pierre et Marie Curie, Paris 75005, France.                   antiantiterminator structure, called SAM aptamer (posi- <br /> Tel.: &#195;&#190; 33 1 58415127; Fax: &#195;&#190; 33 1 58415020;                                            tions &#195;&#190; 58 &#195;&#190; 177, Figure 1A). favours formation <br /> E-mail: putzer@ibpc.fr                                                                  terminator structure provokes premature termina- <br /> Received: 19 March 2009; accepted: 24 August 2009; published                            tion transcription (Epshtein et al, 2003; McDaniel et al, <br /> online: 24 September 2009                                                               2003, 2005; Winkler et al, 2003). Terminator read  <br /> <span id='am-13' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; 2009 European Molecular Biology Organization                                                                       EMBO Journal   VOL 28 | 22 | 2009 3523 <br />        RNase Y initiates mRNA decay B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />  <br />       Figure 1 Model regulation S-box gene expression response SAM.   presence methionine (high SAM level), binding       SAM (represented encircled S) stabilizes antiantiterminator structure favouring formation terminator structure,       causes premature termination transcription. (B) SAM levels low, default antiterminator structure forms base pairing       sequences antiantiterminator (grey, named  terminator (hatched, named II), allowing expression downstream coding <br />       region. positions oligonucleotides shown. Relevant positions transcript numbered transcription start point <br />       (determined Figure 2C). yitJ AUG start codon indicated. <br />  <br />  <br />  <br />       induced response starvation methionine SAM               Initiation riboswitch degradation requires       levels low (Figure 1B). noninducing conditions                essential gene unknown function <br />       (SAM excess), transcription antitermination pro-            straight-forward hypothesis apparently <br />       duces large amounts prematurely terminated mRNA               efficient turnover yitJ riboswitch RNA initial <br />       constitutive promoter. Efficient degradation highly            cleavage preparing structured RNA exonucleolytic <br />       structured RNAs bound SAM (Tomsic et al, 2008)              digestion. Obvious candidates decay-initiating cleavage <br />       important nucleotide recovery recycling           RNase III RNases J1/J2. double <br />       SAM, costly metabolite.                                            strand-specific endoribonuclease cleave          work, examined degradation com-              long structured 50 noncoding region. RNases J1/J2       plex structures. identified novel endoribonuclease,           shown function expression various <br />       encoded essential gene earlier unknown function,               structured leader mRNAs  et al, 2005; Choonee et al, <br />       ymdA responsible initiating decay SAM-           2007; Deikus et al, 2008; Yao et al, 2008). carried       dependent riboswitches B. subtilis. provide evidence              Northern blot analysis using oligonucleotide probe <br />       enzyme, called RNase Y, likely key player          (HP1400) hybridizing close putative 50 end yitJ <br />       initiating mRNA decay organism discuss                mRNA (positions 28&#226;&#8364;&#8220;58, Figure 1) (Grundy Henkin, <br />       RNases Y J act                                   1998). rnjA, rnjB double mutant, expressed                                                                               RNase J2 (gene deletion) showed 20-fold reduced <br />                                                                               level RNase J1 Pxyl-rnjA gene absence <br />       Results                                                                 inducer (Ma&#194;&#168;der et al, 2008), unable detect <br />                                                                               prematurely terminated transcripts (Figure 2B, lanes 2 <br />       Turnover yitJ riboswitch efficient <br />                                                                               3). similar result obtained rnc null mutant <br />       Expression S-box gene yitJ reported specifi- <br />       cally induced starvation methionine (Grundy                  (Figure 2B, lane 6), suggesting RNase III <br />       Henkin, 1998). Northern blot analysis carried                   RNases J1 J2 involved initial event <br />       monitor presence terminated read                  leading degradation riboswitch.       transcripts methionine auxotroph grown presence             turned attention ymdA gene,       absence methionine. Total RNA gel separated               remaining essential genes (Kobayashi et al, 2003)       hybridized RNA probe complementary 30 end                function remained unknown. constructed strain <br />       yitJ leader beginning structural gene             (SSB447) ymdA chromosomal copy       (positions 152&#226;&#8364;&#8220;346, Figure 1, Materials methods).               control IPTG-inducible Pspac promoter. Total RNA       Care taken exclude yitJ leader sequences prone                extracted strain grown presence 2 mM <br />       cross-hybridize S-box gene leaders. expected,             IPTG resuspension medium IPTG <br />       length transcript 2.1 kb detected         deplete YmdA. RNA separated polyacryla- <br />       cells starved methionine (Figure 2A). yitJ                 mide gel probed oligonucleotide probe <br />       promoter constitutively active required               (HP1400) described  absence IPTG,       methionine regulation yitJ expression (Grundy                    detected strong signal 220 nucleotides. likely <br />       Henkin, 1998). presence methionine                  corresponded yitJ leader mRNA prematurely termi- <br />       expected detect prematurely terminated                 nated leader terminator interaction SAM       leader transcript 220 nt.  failed detect         riboswitch. presence IPTG, riboswitch RNA <br />       transcript size (Figure 2A B, lane 1).             undetectable, suggesting ymdA gene product <br />       observation indicated efficient mechanism oper-            encode ribonuclease initiating decay       ate initiate decay highly structured transcript.         riboswitch (Figure 2B, lanes 4 5). <br />  <br /> 3524 EMBO Journal VOL 28 | 22 | 2009                                                       &amp; 2009 European Molecular Biology Organization <br />                                                                                                      RNase Y initiates mRNA decay B. subtilis <br />                                                                                                                             K Shahbabian et al <br />  <br />  <br />  <br />                                                                          respect yitJ start codon originates Sigma                                                                          type promoter located upstream (Figure 2C, lane 3                                                                          Figure 2E). transcription start site 30 nucleotides <br />                                                                          upstream respect earlier predicted 50 end                                                                          yitJ mRNA (Grundy Henkin, 1998; Tomsic et al, 2008). <br />                                                                          expected length leader transcript, originating                                                                          position terminating immediately downstream                                                                          terminator 220 nucleotides. corresponds                                                                          prematurely terminated leader transcript stabilized                                                                          YmdA depleted strain (Figure 2B, lane 5). <br />                                                                             yitJ riboswitch efficiently degraded pre- <br />                                                                          sence YmdA, wanted know read                                                                          transcripts contain complete leader sequence.                                                                          purpose, total RNA extracted methionine <br />                                                                          auxotrophic strain  wild type ymdA) grown                                                                          presence withdrawal methionine. RNA                                                                          used primer extension oligonucleotide comple- <br />                                                                          mentary sequences yitJ ORF (HP1056, <br />                                                                          Figure 1B). detected major signals corresponding                                                                          length transcript (346 nt) transcript                                                                          125 nt, corresponds premature arrest reverse <br />                                                                          transcriptase terminator structure (Figure 2D, lane 2). <br />                                                                          arrest leader terminator odds                                                                          notion read transcript cleaved <br />                                                                          exists antiterminator conformation. <br />                                                                           RNA used extending primer vitro                                                                          denatured reaction reasonable assume <br />                                                                          conditions transcript refold <br />                                                                          terminator conformation. signal detected                                                                          position potentially corresponding cleavage site.                                                                          suggested riboswitch antiterminator confor- <br />                                                                          mation, folds stable termi- <br />                                                                          nator conformation (McDaniel et al, 2003; Winkler et al, <br /> Figure 2 Analysis yitJ leader RNA vivo.   Northern blot          2003), form efficient substrate <br /> analysis yitJ mRNA. Total RNA methionine auxotrophic         YmdA. turn implies YmdA strain SSB441, isolated cultures grown presence           significant impact expression level yitJ  absence methionine separated denaturing 1.2%                Discussion). <br /> agarose gel. RNA probed uniformly labelled anti- <br /> sense 195 nt RNA covering 30 sequences leader transcript extending yitJ coding region (positions 152&#226;&#8364;&#8220;346 Figure 1B       YmdA endoribonuclease <br /> delimited oligonucleotides HP1231 HP1056). RNA se-             truncated form YmdA lacking 25 N-terminal amino <br /> quence ladder run parallel size marker. Migration 23S     acids comprising membrane-binding domain purified 16S ribosomal RNAs indicated. (B) High resolution Northern <br /> analysis yitJ leader. Total RNA RNase J1/J2 double        affinity chromatography  Materials methods mutant (rnjA inducible), rnc mutant ymdA mutant                 Discussion). characterize enzymatic activity YmdA, <br /> (inducible), isolated presence absence inducer,         performed vitro assays riboswitch RNAs synthe- <br /> separated 8% polyacrylamide gel probed leader-        sized T7 RNA polymerase. substrate contained <br /> specific oligonucleotide HP1400 (Figure 1A). [32P]-labelled DNA <br /> ladder transferred membrane RNA.            leader sequences position 1 196 (30 end defined <br /> (C) Identification yitJ transcription start. Oligonucleotide      primer HP1238, Figure 1A), corresponds HP1238 (Figure 1A) hybridized extended total RNA           nucleotide immediately upstream terminator stem <br /> isolated wild type Pspac-ymdA mutant strain (&#194;&#177;IPTG).          (Figure 3C). RNA fold antiterminator primer used generating sequence ladder. <br />  &#195;&#190; 1 indicates band corresponding yitJ transcription start;   conformation (Figure 1B), increasing potential transcribed nucleotide yitJ mRNA encircled.          form SAM-binding aptamer (Figure 1A). configura- <br /> (D) Primer extension analysis total RNA isolated methio-       tion corresponds terminated leader transcript, auxotrophic strain grown presence absence              likely substrate YmdA vivo. <br /> methionine. T indicates region corresponding terminator <br /> stem. (E) Sequence yitJ sigma type promoter preceding       Incubation 50 -labelled monophosphorylated transcript <br /> new transcription start point identified (C).                         YmdA produced major fragment resulting  <br />                                                                          donucleolytic cleavage residue position 52 <br />                                                                          (Figure 3A C). minor reproducible cleavages <br />                                                                          observed positions 124  apical loop    used strain depleted YmdA determine                     aptamer) 185  aptamer termi- <br /> experimentally transcription start point yitJ primer           nator, Figure 3A). <br /> extension using oligonucleotide HP1238 (Figure 1).                          Binding SAM riboswitch aptamer precludes  <br /> Transcription starts G residue position&#226;&#8364;&#8221;265               mation default antiterminator conformation (Epshtein <br />  <br /> &amp; 2009 European Molecular Biology Organization                                                 EMBO Journal     VOL 28 | 22 | 2009 3525 <br />        RNase Y initiates mRNA decay B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />                                                                             structural context cleavage site provides                                                                             rationale fact leader transcript,                                                                             antiterminated length transcript, good sub- <br />                                                                             strate YmdA. <br />                                                                                considered YmdA endoribonuclease <br />                                                                             renamed RNase Y (rny). Cleavage RNase Y required <br />                                                                             presence Mg &#195;&#190; 2 ions, replaced Mn &#195;&#190; 2 <br />                                                                             Zn &#195;&#190; 2 (data shown). <br />  <br />                                                                             RNase Y activity sensitive 5 0 phosphorylation <br />                                                                             state substrate <br />                                                                             endoribonucleolytic activity RNase Y  <br />                                                                             tected 50 monophosphorylated substrate (Figure 3).                                                                             wanted determine cleavage RNase Y                                                                             altered nature substrate 50 end.                                                                             transcript described (30 end defined primer <br />                                                                             1238, Figure 3C) carrying tri- monophosphorylated <br />                                                                             50 end uniformly labelled [32P]<span id='am-61' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-62' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span> tested <br />                                                                             standard conditions. expected, 50 monopho- <br />                                                                             sphorylated substrate cleaved producing fragments <br />                                                                             145 52 nt (Figure 4A, lanes 3 4) identical <br />                                                                             yield, respect number U residues present                                                                             fragment (data shown), plus additional pro- <br />                                                                             ducts, significant yields. <br />                                                                                contrast, transcript carrying 50 PPP group                                                                             resistant cleavage conditions used (Figure 4A, <br />                                                                             lanes 1 2). indicated RNase Y sensitive                                                                             50 phosphorylation state substrate requires 50 P                                                                             significant cleavage occur. kinetic analysis using 30 end- <br />                                                                             labelled mono- triphosphorylated substrates confirmed <br />                                                                             preference RNase Y transcripts carrying mono- <br />                                                                             phosphate group 50 end (Figure 4B). initial reaction <br />       Figure 3 Endonucleolytic cleavage yitJ leader transcripts       YmdA.   Cleavage 50 end-labelled monophosphorylated            rate observed 50 P substrate 20-fold faster <br />       transcript (30 end defined primer HP1238). major cleavage      observed 50 PPP substrate 90 min <br />       site mapped sequence ladder generated primer          similar cleavage product produced       50 end corresponded transcription start point.           substrates (Figure 4C). <br />       (B) Effect increasing SAM concentrations (25 80 mM)       cleavage 50 end-labelled monophosphorylated transcript (30 <br />       end defined primer HP1356). (C) Cartoon riboswitch          Degradation yitJ riboswitch vivo requires <br />       leader. vertical arrow indicates cleavage site YmdA    RNase Y, RNase J1, PNPase RNase R <br />       leader mRNA identified panel  position         shown turnover yitJ riboswitch <br />       primers delimiting transcript 30 ends PCR templates       shown. encircled S stands SAM.                                depends presence RNase Y vivo (Figure 2B, <br />                                                                             lanes 4 5). discovery RNase Y endoribo- <br />                                                                             nuclease capable cleaving vitro riboswitch leader                                                                             terminator conformation provided rationale       et al, 2003; McDaniel et al, 2003; Winkler et al, 2003).           nuclease initiate decay riboswitch vivo. <br />       tried mimic vivo situation using              performed primer extension analysis using <br />       substrate 12 nt longer, contained sequences              primer HP1134 (Figure 5D) total RNA isolated       quired formation antiterminator  position        number endo- exoribonuclease mutants. expected, <br />       208, oligonucleotide HP1356, Figure 3D). transcript            length leader transcript greatly stabilized       priori good substrate RNase Y           RNase Y expression reduced (Figure 5A, lanes 6 7). <br />       absence SAM, shown fold preferentially            strain depleted RNase J1, detected weak signal       antiterminator conformation (Epshtein et al, 2003;           position &#195;&#190; 50 (indicated scissors symbol labelled 1       McDaniel et al, 2003).  RNA                 Figure 5A, lane 3), corresponds quite RNase <br />       accessible cleavage addition SAM, shifts           Y cleavage site identified vitro (position 52, Figure 3C),       RNA structure terminator conformation.                 number smaller degradation intermediates (Figure 5A, <br />       shown Figure 3B, cleavage YmdA major cleavage            lane 3), implying RNase Y cleaved mRNA       site (position &#195;&#190; 52) barely detectable absence          efficiently degraded 50 end conditions. <br />       SAM stimulated significantly increasing            observed significant accumulation       physiological range concentration metabolite,              degradation intermediates strains devoid major 30 &#226;&#8364;&#8220;50 <br />       vivo reaches B300 mM presence methio-              exoribonucleases, polynucleotide phosphorylase ribonu- <br />       (Tomsic et al, 2008). weak cleavage              clease R (Figure 5A, lanes 4 5). accumulating <br />       observed position 54. indicates YmdA sensitive        transcripts 50 ends corresponding original <br />  <br /> 3526 EMBO Journal VOL 28 | 22 | 2009                                                    &amp; 2009 European Molecular Biology Organization <br />                                                                                                      RNase Y initiates mRNA decay B. subtilis <br />                                                                                                                             K Shahbabian et al <br />  <br />  <br />  <br />                                                                          transcription start position &#195;&#190; 58 (Figure 5D). quite <br />                                                                          dramatic effect 30 &#226;&#8364;&#8220;50 exonuclease mutations                                                                          stabilization riboswitch degradation intermediates <br />                                                                          suggested leader transcript cleaved  <br />                                                                          create entry site nucleases. <br />                                                                          Extension primer HP1238 complementary sequences <br />                                                                          immediately upstream terminator (Figure 1A) failed                                                                          detect degradation intermediates total RNA isolated <br />                                                                          pnp/rnr rnjA mutants (data shown).                                                                          indicated riboswitch RNA cleaved second <br />                                                                          site primers HP1134 HP1238.                                                                          performed 50 /30 RACE experiment ligating total RNA <br />                                                                          wild type, rnjA/B pnp/rnr double-mutant <br />                                                                          strains. RT&#226;&#8364;&#8220;PCR junction circularized mRNA <br />                                                                          using primers HP1394 (positions &#195;&#190; 146 &#195;&#190; 121)                                                                          HP1395 (positions &#195;&#190; 148 &#195;&#190; 166, Figure 5D) specifically <br />                                                                          produced fragments B130 bp mutant strains <br />                                                                          (Figure 5B, lanes 1 2). size expected                                                                          transcript cleaved aptamer domain <br />                                                                          (Figure 5D). RT&#226;&#8364;&#8220;PCR products purified cloned <br />                                                                          20 recombinant plasmids sequenced.  <br />                                                                          firmed site RNase Y cleavage upstream aptamer <br />                                                                          (Figure 5D, cleavage 1) agreement primer <br />                                                                          extension product observed RNase J1/J2 mutant <br />                                                                          (Figure 5A, lane 3, position &#195;&#190; 50). second cleavage                                                                          RNase Y  Discussion) occurred residue &#195;&#190; 182, <br />                                                                          single-stranded region aptamer                                                                          terminator structures (Figure 5D). confirmation                                                                          site 2 cleavage obtained S1 mapping analysis <br />                                                                          downstream cleavage product containing transcrip- <br />                                                                          tion terminator. fragment expected size (25 nucleotides, <br />                                                                          legend Figure 5C) accumulated cells depleted                                                                          RNase J1 presumably 50 &#226;&#8364;&#8220;30 exonucleolytic degrada- <br />                                                                          tion fragment inefficient conditions <br />                                                                          (Figure 5C, lane 3). basis data, turnover                                                                          yitJ riboswitch terminator conformation likely proceeds <br />                                                                          illustrated schematically Figure 5D  Discussion). <br />  <br />                                                                          RNase Y acts riboswitch controlled S-box gene <br />                                                                          family members <br /> Figure 4 RNase Y cleavage yitJ riboswitch sensitive 50 phosphorylation state substrate.   Equal amounts                                                                          investigated RNase Y initiated decay [32P] uniformly labelled tri- monophosphorylated yitJ tran-          SAM-dependent riboswitches S-box genes. scripts (30 end defined primer HP1238) used substrates        purpose, performed Northern blot analysis post-transcriptional cleavage assays RNase Y described     10 SAM riboswitches present B. subtilis. mRNA <br /> Material methods. Reaction products separated 8% <br /> polyacrylamide gel. major cleavage site indicated scissors <br />                                                                          prepared strain SSB447, wild type cleavage products identified schematic drawings           methionine biosynthesis contains IPTG-inducible right. [32P]-labelled 50 bp DNA ladder used size       rny gene. Specific oligonucleotide probes (Supplementary <br /> marker. (B) Kinetic analysis RNase Y cleavage 30 end-labelled      Table  riboswitch used avoid potential <br /> tri- monophosphorylated yitJ transcripts (30 end defined                                                                          cross-hybridization. exception cysH, pre- <br /> primer HP1238). Samples scaled cleavage assays taken indicated time points. reaction products           maturely terminated riboswitch transcript detected separated 5% polyacrylamide gel. RNase Y-dependent              genes culture depleted RNase Y (Figure 6). <br /> cleavage product 150 bases indicated arrow. band         cysH operon shown earlier regulated corresponds spontaneous cleavage transcript                                                                          primarily level transcription initiation response purification observed addition enzyme. <br /> (C) Graph kinetic analysis RNase Y cleavage yitJ transcripts    O-acetyl-serine (Mansilla et al, 2000) prematurely <br /> bearing mono- triphosphorylated 50 ends. reactions          terminated leader transcript expected present. <br /> carried (B) incubated 90 min detect cleavage   11 S-box genes analysed RNase J1/J2 <br /> 50 tri-phosphorylated transcripts. gels containing   double mutant. Using gene-specific probes, observed cleavage products shown (50 -P) graph (50 -PPP), respectively. numbers Y axis represent       effect absence/depletion RNases J1/J2 arbitrary units generated quantification Image J             expected length leader RNA levels of S-box <br /> program.                                                                 genes (data shown). Taken  indicates                                                                          RNase Y key enzyme turnover SAM- <br />                                                                          dependent riboswitches B. subtilis. <br />  <br /> &amp; 2009 European Molecular Biology Organization                                                 EMBO Journal     VOL 28 | 22 | 2009 3527 <br />        RNase Y initiates mRNA decay B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       Bulk mRNA stability increased RNase Y depleted    IPTG-inducible rny copy) grown presence absence <br />       strain                                                     IPTG inducer. measured mRNA half-lives       investigated RNase Y impact      6.1&#194;&#177;1.3 min ( IPTG) 3.5&#194;&#177;0.4 min ( &#195;&#190; IPTG), respectively, <br />       global mRNA stability. <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> decay rate pulse- <br />       labelled RNA determined strain SSB447 (containing <br />  <br />  <br />  <br />  <br />                                                                  Figure 6 Turnover functional S-box riboswitches B. subtilis <br />                                                                  initiated RNase Y. Northern blot analysis leader tran- <br />                                                                  scripts 10 S-box genes. Total RNA isolated rny (ymdA) <br />                                                                  depletion strain SSB447 grown presence absence IPTG <br />                                                                  separated 8% acrylamide gels probed leader- <br />                                                                  specific oligonucleotides (Supplementary Table . putative  <br />                                                                  length leader transcripts marked arrow head. shorter <br />                                                                  transcripts likely degradation intermediates. [32P]-labelled <br />                                                                  DNA ladder transferred membrane RNA. <br />  <br />  <br />                                                                  Figure 5 Degradation yitJ riboswitch vivo requires RNase <br />                                                                  Y, RNase J1, PNPase RNase R.   Extension primer HP1134 <br />                                                                   panel D) total RNA isolated wild-type different <br />                                                                  RNase mutant strains (pnpA rnr encode polynucleotide phos- <br />                                                                  phorylase RNase R, respectively). scissors symbol J        1 <br />                                                                  indicates RNase Y vivo cleavage site upstream aptamer <br />                                                                  (position &#195;&#190; 50, panel D, text). Extension products correspond- <br />                                                                  ing transcription start ( &#195;&#190; 1) degradation intermediate <br />                                                                  ( &#195;&#190; 58) stabilized pnpA rnr mutants (lanes 4 5)                                                                  indicated. (B) Mapping RNase Y cleavage sites vivo                                                                  50 /30 RACE. Total RNA isolated wild-type, pnpA/rnr                                                                  rnjB/rnjA double-mutant strains, ligated create circular RNA <br />                                                                  potential degradation intermediates. RT&#226;&#8364;&#8220;PCR used                                                                  amplify sequences 50 /30 junction. amplified <br />                                                                  products expected size B130 bp  present                                                                  wild type) gel purified, cloned sequenced. cleavage <br />                                                                  sites deduced sequences representing majority                                                                  clones mutants (20 independent clones).                                                                  sequences remaining clones identified ligated junctions, <br />                                                                   probably partial degradation, located                                                                  aptamer domain (data shown). products size                                                                  B100 bp present lanes generated PCR amplification <br />                                                                  tandemly ligated aptamer RNAs relevant (data                                                                  shown). (C) Detection 30 terminal fragment yitJ leader <br />                                                                  created RNase Y cleavage 2. Total RNA wild-type                                                                  rnjB/rnjA double mutant subjected S1 mapping analysis <br />                                                                  using oligonucleotide HP1442 (complementary positions &#195;&#190; 207 <br />                                                                  &#195;&#190; 131 mRNA). band corresponding size                                                                  expected 30 terminal fragment indicated. (D) Scheme yitJ <br />                                                                  riboswitch RNA RNase Y cleavage sites                                                                   determined 50 &#226;&#8364;&#8220;30 RACE) aptamer domain <br />                                                                  indicated. degradation pathway riboswitch following <br />                                                                  initial cleavage RNase Y illustrated based data shown                                                                  panels C  Discussion). <br />  <br /> 3528 EMBO Journal VOL 28 | 22 | 2009                                          &amp; 2009 European Molecular Biology Organization <br />                                                                                                    RNase Y initiates mRNA decay B. subtilis <br />                                                                                                                           K Shahbabian et al <br />  <br />  <br />  <br />                                                                        turnover transcript dependent <br />                                                                       product ymdA gene (Figure 2B). vitro <br />                                                                       experiments characterized YmdA protein endoribo- <br />                                                                       nuclease (named RNase Y) capable cleaving yitJ <br />                                                                       riboswitch nucleotides upstream aptamer structure <br />                                                                       (Figure 3A C). Interestingly, RNase Y appeared                                                                       sensitive RNA conformation downstream clea- <br />                                                                       vage site. Consequently, vitro transcript including 30 <br />                                                                       sequences, allowing formation antiterminator                                                                       terminator (Figure 3C, primer HP1356), poor <br />                                                                       substrate RNase Y.  efficient cleavage Figure 7 RNase Y domain structure effect point mutations       obtained adding increasing amounts SAM (Figure 3B), RNase Y activity.   KH HD domains indicated          known favour terminator conformation HisAsp doublet conserved HD domain <br /> proteins. TMD indicates N-terminal transmembrane domain.          binding stabilizing mutually exclusive aptamer  <br /> (B) Cleavage yitJ leader transcript wild-type RNase Y   main (McDaniel et al, 2005). suggested principal <br /> H368A D369A mutant proteins. Cleavage products sepa-         substrate RNase Y riboswitch complexed SAM. <br /> rated 8% polyacrylamide gel (similar shown Figure      Corroborating hypothesis, extension primer specific <br /> 3A B). portion gel corresponding major <br /> cleavage product (52 nt) shown.                                    read (antiterminated) transcripts present RNA <br />                                                                       isolated methionine starvation conditions, detected <br />                                                                       essentially transcripts, contained complete leader <br /> compared 2.8&#194;&#177;0.2 min wild-type strain.            sequences (Figure 2D). case, stable anti- <br /> RNase Y depletion strain, global mRNA decay slowed            terminator conformation probably remains predominant fold compared wild type.                       inhibits cleavage read transcripts.                                                                       cleavage actually counterproductive RNase Y widely conserved HD domain protein                       destabilize structural mRNA encoding YitJ protein basis presence conserved HD domain,          reduce effect transcriptional antitermination. YmdA protein assigned superfamily metal-           support notion, depletion RNase Y signifi- <br /> dependent phosphohydrolases (Aravind Koonin, 1998).               impact induced expression yitJ-lacZ fusion presence RNA-binding KH domain (Grishin, 2001)              methionine starvation (data shown). HD domain (Figure 7A) suggested               Initiation yitJ riboswitch decay vivo turned primary function protein nucleic acid metabolism.       complex anticipated vitro data. putative metalloenzyme RNase Y possesses distinctive           confirm RNase Y cleaves riboswitch upstream <br /> combination metal-chelating residues, essentially histi-           aptamer domain vivo, nucleotides upstream dines aspartates (Aravind Koonin, 1998). assess            cleavage site identified vitro (Figures 3C 5D). Small importance HD domain RNase Y activity,                 differences cleavage site specificity comparing mutated His368 Asp369 constitute highly               vivo vitro conditions uncommon probably <br /> conserved HD motif. shown Figure 7B, mutated                reflect extent, conformational differences proteins severely impaired cleavage yitJ               structured RNA presented nuclease. sur- <br /> riboswitch, indicating HD domain important            prising fact vivo second efficient endonucleolytic activity RNase Y.                              cleavage occurred downstream aptamer domain <br />                                                                       (14 nt upstream terminator structure, Figure 5D). <br />                                                                       cleavage 1  observed vitro) Discussion                                                            50% efficient  estimated primer extension signals expression genes B. subtilis Gram-                 obtained pnpA-rnr mutants, Figure 5A, lanes 4 5), <br /> positive organisms general controlled transcription          cleavage aptamer terminator vir- <br /> antitermination mechanisms (Merino Yanofsky, 2005;                tually 100% efficient prematurely terminated <br /> Grundy Henkin, 2006), produce large amounts                 transcripts. degradation intermediates observed prematurely terminated transcripts strong constitu-           pnpA-rnr mutant strains accumulated higher levels <br /> tive promoters. SAM-dependent riboswitch controlling              intact leader transcript RNase Y depleted yitJ gene belongs class (Tomsic et al, 2008)          strain (Figure 5A, lanes 4, 5 7). difference forms highly structured mRNA, especially terminator          probably fact PNPase RNase R conformation, complexed SAM.                 completely absent pnpA-rnr knock mutants, <br /> surprising presumably stable transcript        RNase Y depleted, procedure virtually undetectable wild-type strain (Figure 2B,          residual enzyme remains cell. apta- <br /> lane 1). meant efficient               mer domain predominantly degraded 30 &#226;&#8364;&#8220;50 <br /> mechanism initiate decay, providing, priori, perfect       exoribonucleases PNPase RNase R (Figure 5A, lanes 4 <br /> role RNases J1/J2. enzymes, particularly RNase J1,          5).  presence degradation intermediates dual endo- 50 &#226;&#8364;&#8220;30 exoribonucleolytic activities        RNase J1 depleted strain (Figure 5A, lane 3) suggests thought constitute heart RNA degradation            degradation aptamer domain occurs <br /> machinery Gram-positive organisms.               RNase J1 50 &#226;&#8364;&#8220;30 exonucleolytic activity attacking involved rate-determining initiating           riboswitch RNA cleaved site 1. major <br /> events SAM riboswitch decay (Figure 2B, lane 3). Instead,          degradation intermediates identified pnpA-rnr mutants <br />  <br /> &amp; 2009 European Molecular Biology Organization                                              EMBO Journal      VOL 28 | 22 | 2009 3529 <br />        RNase Y initiates mRNA decay B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       50 end position &#195;&#190; 58 (Figure 5A, lanes 4 5).          RNase Y appeared cleave single-stranded AU rich <br />       corresponds nucleotide immediately upstream            sequences upstream secondary structure, context <br />       highly structured aptamer domain (Figure 5D) presum-             resembling cleavage specificities RNases J1/J2 <br />       ably constitutes stall site 50 &#226;&#8364;&#8220;30 exoribonucleolytic      (Deikus et al, 2008; et al, 2005; Yao et al, 2007)       progression RNase J1.                                             E. coli RNase E (Callaghan et al, 2005). requirement <br />          riboswitch leader terminator conformation              downstream secondary structure explain       strongly accumulated RNase Y depleted strain                   cleavage site 1 occurs presence SAM:       (Figure 2B, lane 5), indicating terminator efficiently      lower stem aptamer domain (Figure 3C) forms       protects leader riboswitch 30 50 exonucleolytic          binding metabolite (McDaniel et al, 2005). sug- <br />       attack. Degradation fragment containing terminator        gested cleavage site selection RNase Y extremely <br />       occur principally 50 &#226;&#8364;&#8220;30 exonucleolytic           specific.  SAM needed form       activity RNase J1 using cleavage site 2 entry        (unspecific) secondary structure 30 cleavage site,       site. confirmed S1 nuclease mapping,          role played terminator structure downstream       showed accumulation 25 nt terminal fragment          cleavage site 2. case, cleavage specificity       RNase J1 depleted strain (Figure 5C, lane 3). Taken         RNase Y compatible general function <br />       data suggest SAM riboswitch turnover initiates         mRNA metabolism. <br />       rate determining endonucleolytic cleavages RNase Y,                  RNase Y preferred 50 monophosphorylated substrate       likely dependent intracellular SAM concen-          depletion significantly increased bulk mRNA stability,       tration determining RNA folding. cleavage products          characteristics shares RNase E. Reduced cleavage       substrates 30 &#226;&#8364;&#8220;50 exonucleases PNPase             50 triphosphorylated RNA avoids attack nascent RNA <br />       RNase R  aptamer domain probably 50 nt 50                transcripts priori useful property enzyme <br />       terminal leader fragment) 50 &#226;&#8364;&#8220;30 exoribonucleolytic          responsible setting decay rate mRNAs. <br />       activity RNase J1. required degradation      RNase Y able discriminate       30 terminal fragment accessory            distinct RNA conformations located downstream       aptamer domain.                                                      cleavage site. poor cleavage efficiency observed <br />          able reproducibly vitro           vitro reflect sensitivity. remains       RNase Y responsible site 2 cleavage           seen extent nature 50 end RNA <br />       riboswitch RNA. unfavourable struc-         structure, respectively, contribute substrate specificity <br />       tural conformation obtained vitro transcripts chosen     enzyme activity. <br />       optimize cleavage site 1. length terminated           recent study using bacterial hybrid showed <br />       riboswitch leader detected RNase Y mutant          PNPase, RNase J1 glycolytic enzymes 6-phos- <br />       (Figure 2B, lane 5), likely RNase Y     phofructokinase enolase interact ymdA gene <br />       cleaves sites. alternative, cleavage site 1   product potentially form degradosome-like complex <br />       obligatory create structural context recognized           vivo (Commichau et al, 2009). interesting       RNase.  case cleavage        complex purified addition       site 2 expected efficient cleavage      glycolytic enzymes improve alter RNase Y activity       site 1. observed contrary  .                vitro. authors showed earlier reported <br />          RNase Y initiated turnover SAM-dependent               processing B. subtilis gapA operon (Meinken et al, <br />       riboswitches B. subtilis (Figure 6) exception         2003) diminished vivo ymdA expression reduced <br />       cysH, shown earlier regulated level        (Commichau et al, 2009). Testing known RNA clea- <br />       transcription initiation (Mansilla et al, 2000). sizes    vages RNase Y depleted strain rapidly lead       stabilized length leader transcripts detected RNase      discovery new potential substrates RNase Y.       Y depleted strain correlated quite transcription       earlier systematic inactivation analysis unknown es- <br />       initiation sites inferred sequence-based search           sential genes shown depletion YmdA resulted       promoters (Grundy Henkin, 1998 #1760).                effects cell chromosome morphology (Hunt <br />       remains analysed degradation pathway            et al, 2006). Interestingly, H368A mutation  caused       yitJ riboswitch elucidated applicable        strong decrease endonucleolytic activity, Figure 7B) <br />       S-box riboswitches. RNA&#226;&#8364;&#8220;SAM complexes               showed phenotype similar depleted strain, suggesting <br />       stable half-lives ranging 4 11 min (Tomsic         impaired endonucleolytic activity RNase Y <br />       et al, 2008), efficient turnover large amounts             responsible observed effects. <br />       riboswitch RNA produced important                    Gfp-fusion experiments localized YmdA protein       nucleotide recycling liberating bound SAM,              cell periphery (Hunt et al, 2006), consistent       costly metabolite.                                                 presence predicted N-terminal transmembrane domain <br />          chemical decay rate bulk mRNA slowed              (Figure 7A). hands, affinity purified, length <br />       fold RNase Y depletion strain compared          tagged protein suitable vitro assays       wild-type strain (6.1 versus 2.8 min, respectively).         considerable contamination large number proteins <br />       effect RNase Y global mRNA stability significant       presumably associated membrane fragments.  <br />       distinctly greater 30% increase mRNA                 solved problem deleting 25 N-terminal amino <br />       stability observed RNase J1/J2 double-mutant strain             acids comprising transmembrane domain using  <br />        et al, 2005). suggests mRNA cleavage              tein technology purification. <br />       RNase Y major function initiation                Localization RNase Y  complex       mRNA degradation B. subtilis.                                     proteins) membrane, extends analogy  <br /> 3530 EMBO Journal VOL 28 | 22 | 2009                                                   &amp; 2009 European Molecular Biology Organization <br />                                                                                                   RNase Y initiates mRNA decay B. subtilis <br />                                                                                                                          K Shahbabian et al <br />  <br />  <br />  <br /> E. coli RNase E. fact, RNase E contains short domain      Table Occurrence RNase Y prokaryotes propensity form amphipathic helix neces- <br />                                                                                                               RNase <br /> sary sufficient bind membranes (Khemici et al, <br /> 2008), providing rationale cytoskeletal-like structure                                  E/G              J              Y degradosomal proteins cell (Taghbalout            Archaea <br /> Rothfield, 2007).  consequence localiza-           Crenarchaeota                                       tion RNA processing degradation clear,                                                                        Euryarchaeota                         30/36            RNase E RNase Y. suggested membrane serve storehouse RNase E                   Nanoarchaeota           active form enzyme small proportion       Bacteria <br /> molecules cytoplasmic (Khemici et al, 2008).                                                                        Actinobacteria          <br /> conserved pseudo-compartmentalization RNase Y RNase E different organisms (B. subtilis               Bacteroidetes         <br /> E. coli, respectively) clearly suggests evolutionary advantage      Chlamydiae               configuration. membrane bound nucleases                 Cyanobacteria          provide anchor assembling potentially transient <br />                                                                        Firmicutes <br /> degradosome-type complexes fine tuning degrada- <br /> tion machinery various physiological conditions.                     Bacillales         <br />     RNase Y member new class ribonucleases            Clostridia        constitute subgroup superfamily HD <br />                                                                            Lactobacillales       <br /> proteins combining metal-chelating HD domain RNA-binding KH domain (Aravind Koonin, 1998).                  Mollicutes              <br /> Single <span id='am-49' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-50' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-53' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> substitutions highly conserved HD             Proteobacteria <br /> doublet greatly reduced ribonucleolytic activity RNase <br />                                                                            Alpha                <br /> Y, indicating catalytic activity resides HD <br /> domain. cleavage D369A mutant occurred                    Beta                second site 2&#226;&#8364;&#8220;3 nucleotides upstream corresponding                    Delta               <br /> closely vivo cleavage site (Figure 7, lane 4).                   Epsilon              <br />     Orthologs RNase Y occur 40% currently <br />                                                                            Gamma               <br /> sequenced eubacteria (Table , similarities ranging 56 99%. RNase Y absent archea                  Spirochaetales          <br /> eukaryotic organisms single exception, Drosophila <br /> willistoni. Organisms containing RNase Y RNase E/G like enzyme RNase J. bacteria                 direction. property especially useful degrading <br /> actually enzymes, like Frankia               transcripts 30 ends protected secondary <br /> Salinispora species Actinobacteria 25&#226;&#8364;&#8220;30%             structure. mind, 50 &#226;&#8364;&#8220;30 exonucleolytic activity <br /> Bacillales (e.g. Listeria) Clostridia (Table . raises    RNase J evolved organisms like B. subtilis <br /> interesting questions sharing functions         contrast E. coli, does appear efficient enzymes usually essential            polyadenylation pathway allow 30 &#226;&#8364;&#8220;50 degradation struc- <br /> RNases dispensable context.            tured RNAs. B. subtilis, 50 road blocks (structural hand, single eubacterial species (Sulcia            formed bound ribosomes, ) substantially stabilize <br /> muelleri Bacteroidetes)           nontranslated downstream sequences, implying enzymes. case Crenarcheota         endoribonuclease like RNase Y efficiently cleave <br /> majority Euryarcheota novel key ribonu-             body mRNA. ignorant exact <br /> cleases probably remain identified.                           parameters determine substrate specificity RNase Y <br />     does RNase Y fit current understanding           matter, endonucleolytic activity mRNA processing degradation bacteria? intriguing       RNase J. characterisation RNase Y include completely unrelated enzymes like RNase Y              analysis relative importance preference RNase E, Gram-positive (B. subtilis) Gram-nega-          RNase Y 50 phosphorylation state conformation, <br /> tive organism (E. coli) thought follow             respectively, RNA substrate. Interestingly, different strategies concerning mRNA metabolism, share           characteristics potential control access features including large effects global mRNA stabi-         RNase Y nascent transcripts.  easily <br /> lity 50 end preference mono phosphorylated sub-           provide specificity necessary reconcile activity <br /> strates. raises possibility mRNA processing       RNase Y current understanding mRNA <br /> degradation similar B. subtilis           metabolism Gram-positive bacteria. <br /> E. coli presently assumed, endonucleolytic clea- <br /> vage crucial step initiating mRNA decay organisms. RNases J1/J2 smaller effects bulk mRNA           Materials methods RNase Y  et al, 2005), suggesting enzymes initiating decay large scale             Bacterial strains growth conditions <br />                                                                     prototrophic B. subtilis 168 strain 1A2 (BGSC) parent <br /> endo- exonucleolytically) involved             strain used construction mutants. RNase J1/J2 <br /> degrading cleavage products exonucleolytically 50 &#226;&#8364;&#8220;30          double-mutant strain SSB357 (rnjBHspc, PxylHrnjA) described <br />  <br /> &amp; 2009 European Molecular Biology Organization                                             EMBO Journal      VOL 28 | 22 | 2009 3531 <br />        RNase Y initiates mRNA decay B. subtilis <br />       K Shahbabian et al <br />  <br />  <br />  <br />       earlier  et al, 2005) RNase III mutant strain BG322        S1 nuclease mapping <br />       (Herskowitz Bechhofer, 2000) provided D Bechhofer.           measure 30 mg total RNA mixed 50 end-labelled <br />       strain SSB447, ymdA gene control Pspac     oligonucleotide HP1442 (105 cpm, B10 ng), precipitated,       promoter Campbell-type integration plasmid pHMD1. Induc-          resuspended 40 mM Pipes pH 6.4, 1 mM EDTA pH8, 0.4 M NaCl <br />       tion yitJ expression methionine starvation carried     80% deionized formamide. Hybridization carried       strain SSB441 (metC2, lys1) provided T Leighton. E. coli strain      incubating mixture 481C overnight S1 nuclease <br />       SSC91, used overexpression YmdA, constructed          (Fermentas) digestion (200 U) carried according       lysogenisation strain IBPC852 (Drnb201Htet, rph1, thyA715,           manufacturer&#226;&#8364;&#8482;s instructions 371C 30 min. reaction <br />       rne1) kindly provided P Regnier lDE3. B. subtilis           products resolved 5% polyacrylamide gel. <br />       grown SMS minimal medium (Spizizen, 1958) supplemented <br />       0.1% <span id='am-20' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-21' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-glutamine</span>. Methionine added final concentra-       5 0 /3 0 RACE <br />       tion 2 mM required. Antibiotics added,              Total RNA (5 mg) circularized RNA ligase (Biolabs)       following concentrations: ampicillin (200 mg/ml), kanamycin             subjected RT&#226;&#8364;&#8220;PCR 50 /30 junction, using oligonucleo- <br />       (5 mg/ml), chloramphenicol (4 mg/ml). depletion essential    tides HP1134 (RT) HP1394 HP1395 (PCR). PCR <br />       RNases J1 Y, strains bearing inducible copies genes        fragments B130 bp isolated agarose gels, digested <br />       (Pxyl Pspac, respectively) grown overnight              EcoRI BamHI cloned plasmid pJRD184 <br />       presence inducer. cells centrifuged resuspended         (Heusterspreute Thi, 1985) sequencing. clones       medium inducer. Growth monitored 30 min            mutant sequenced. <br />       slowing growth absence inducer, compared <br />       culture grown inducer, detectable OD600        Assay RNase Y activity <br />       B0.4. Cells harvested immediately detectable       Reactions carried using 10 nM RNA substrate       change growth rate.                                                  0.25 mM RNase Y reaction volume 10 ml. buffer contained <br />                                                                               20 mM Hepes pH 8, 8 mM MgCl2, 0.48 U/ml RNAsin (Promega), <br />       Construction YmdA depletion strain                               100 mM NaCl reactions incubated 10 min 301C. <br />       Plasmid pHMD1 used make ymdA depletion strain:             time course experiments, samples scaled reactions (50 ml) <br />                                                                               taken indicated times. Reactions stopped       500 bp fragment PCR amplified using oligonucleotides HP1113/ <br />                                                                               addition 5 ml stop buffer (87.5% formamide, 0.05% xylene <br />       HP1114, digested SalI SphI ligated respective sites <br />       plasmid pDG648 (kindly provided P Stragier) downstream            cyanol, 0.05% bromophenol blue, 5 mM EDTA) samples       Pspac promoter. Campbell-type integration pHMD1            loaded 5 8% denaturing polyacrylmide gels. RNA <br />       B. subtilis chromosome rendered ymdA expression IPTG dependent.         substrates synthesized T7 RNA polymerase PCR <br />                                                                               products (primer pairs HP1198/HP1238 HP1198/HP1356).                                                                               cases, mRNA transcripts contained additional 50 G <br />       Purification YmdA                                                    residue compared wild-type mRNA. vitro <br />       Plasmid pHMD3 used overexpress YmdA: coding                  transcripts 50 end-labelled g-[32P] <span id='am-25' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> using T4 <br />       sequence ymdA gene excluding 25 residues               polynucleotide kinase following dephosphorylation, continuously <br />       comprising transmembrane domain PCR amplified using               labelled [32P] UTP 30 end-labelled [32P]-pCp using T4 <br />       oligonucleotides HP1182/HP754 cloned phase intein       RNA ligase. 50 monophosphorylated transcripts synthesized <br />       domain plasmid pKYB1 (New England Biolabs) plasmid           including 50-fold excess <span id='am-56' about='obo:IMR_0200292' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-57' property="rdfs:label" content="GMP" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200292" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00144" datatype="xsd:string"></span>GMP</span> <span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> reaction. <br />       pHMD3. YmdA isolated intein fusion protein using                test effect SAM mRNA cleavage, SAM (25       IMPACT (New England Biolabs). Overexpression              80 mM) added RNA substrate mixture       recombinant proteins carried strain SSC91. Purified          incubated 701C 10 min slowly cooled 371C,       YmdA heated 10 min 451C use inactivate             addition RNase Y. <br />       thermosensitive RNase E potentially contaminate             50 end-labelled 50 bp DNA ladder used size marker       preparation.                                                            experiments. <br />  <br />       Site-directed mutagenesis                                               Chemical decay bulk mRNA <br />       H368A D369A mutations YmdA HD domain                Total mRNA decay measured described earlier (Wang       introduced plasmid pHMD3 using Quikchange (Stratagene)           Bechhofer, 1996; et al, 2005). <br />       strategy (oligonucleotides HP1298, HP1299, HP1300, HP1301)       KOD DNA polymerase (Novagen).                                            <br />       Northern blot analysis <br />       Northern blot analysis carried described (Putzer et al, <br />       1992) using 6 mg total RNA separated 1.2% agarose    <br />       8% acrylamide gel. continuously [32P] UTP-labelled riboprobe        <br />       generated T7 vitro transcription PCR fragment       <br />       primers HP1231 (50 end corresponds position &#195;&#190; 152)         <br />       HP1058 (corresponds HP1056 plus upstream T7 promoter          <br />       sequence) used detect terminated yitJ leader        <br />       length transcript. 50 [32P]-labelled oligonucleotides (listed   <br />       Supplementary Table  used probes specifically detect      <br />       leader transcripts 11 S-box genes.                        <br />  <br />       Primer extension analysis <br />       measure 5 mg total RNA subjected reverse               <br />       transcription analysis extending 50 [32P]-labelled primers <br />       Superscript III reverse transcriptase (Invitrogen).               <br />  <br />  <br />  <br /> </body></html>