<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization binding surface translocated intimin receptor, essential protein  <br /> Abstract translocated intimin receptor (TIR) enteropathogenic enterohemorrhagic Escherichia coli <br /> (EPEC EHEC) required EPEC EHEC infections, cause widespread illness globe. TIR translocated type-III secretion intestinal epithelial cell membrane, serves anchor E. coli attachment binding partner intimin. aspects EPEC EHEC infection understood, importance intermolecular contacts intimin TIR thoroughly investigated. report site-directed <br /> mutagenesis studies intimin-binding domain EPEC TIR, mutations affect TIR- <br /> intimin association, analyzed isothermal titration calorimetry circular dichroism. results factors govern TIR&#226;&#8364;&#8482;s binding intimin: residue TIR hot spot identified largely mediates interaction, mutants alter b-hairpin structure TIR severely diminish <br /> binding affinity. addition, peptides incorporating key TIR residues identified mutagenesis incapable binding intimin. results indicate hot spot residues structural orientation/ <br /> preorganization required EPEC, likely EHEC, TIR-intimin binding. <br /> Keywords: EPEC; EHEC; intimin; translocated intimin receptor (TIR) <br /> Supplemental material: www.proteinscience.org <br />  <br />  <br /> Intimin translocated intimin receptor (TIR)                         Knutton et al. 1997). TIR, expression, localized enteropathogenic enterohemorrhagic Escherichia coli                        E. coli cell membrane translocated (EPEC EHEC) play integral role attachment                         type-III secretion needle complex (Kenny et al. 1997; <br /> intestinal epithelial cells (Marches et al. 2000; Vallance                     Gauthier Finlay 2003) host cell. TIR Finlay 2000). proteins encoded                        immobilized intestinal epithelium locus enterocyte effacement expressed (TIR)                      intimin binding domain (IBD) displayed intestinal regulated (intimin) EPEC/EHEC cells come                          cell surface. Intimin, transmembrane protein close contact host cell (McDaniel et al. 1995;                          extracellular domain (ECD) displayed E. coli <br />                                                                                surface, binds TIR-IBD, intimately attaching E. coli <br />                                                                                cell intestinal epithelium (Kenny et al. 1997).    Reprint requests  Benjamin Miller, University Rochester,              attachment ultimately leads actin recruitment Department Dermatology, 601 Elmwood Avenue, Box 697, Roches-                epithelial cell formation attaching ter, NY 14642, USA; e-mail: Benjamin_Miller@urmc.rochester.edu;                effacing lesions (Knutton et al. 1987). Lesion formation fax: (585) 273-1346. <br />    Article publication http://www.proteinscience.org/cgi/doi/       precursor intestinal cell membrane disruption 10.1110/ps.073128607.                                                          diarrhea infected individuals (Knutton et al. 1987, <br />  <br />  <br /> Protein Science (2007), 16:2677&#226;&#8364;&#8220;2683. Published Cold Spring Harbor Laboratory Press. Copyright &#195;&#8220; 2007 Protein Society         2677 <br />  Ross Miller <br />  <br />  <br />  <br /> 1997). severe cases, infection leads hemolytic                         strate short peptides derived intimin- <br /> uremic syndrome, disease kills 12%                           binding region TIR-IBD ability bind <br /> infected (Gerber et al. 2002; Garg et al. 2003).                             intimin-ECD. peptides support hypothesis    structure intimin-ECD solved NMR                       sequential, unstructured mimics sufficient <br /> (Batchelor et al. 2000) X-ray crystallography (Luo                       bind intimin-ECD, additional structural  <br /> et al. 2000). addition, cocrystal structure                        quirements necessary modulate protein&#226;&#8364;&#8220;protein <br /> intimin-ECD bound TIR-IBD determined                        interaction. 85% sequence identity (Fig. 1; Luo et al. 2000). structures                       EPEC EHEC TIR b-hairpin region <br /> great impact understanding proteins,                     (DeVinney et al. 1999), results apply EHEC <br /> including elucidation binding interface.                    TIR   limited additional work conducted determine specific residues pro- <br />                                                                              Results <br /> teins responsible interaction. knowledge, studies lines                      Alanine point mutants generated nonalanine <br /> reported Leong group, carried random                          residues b-hairpin TIR-IBD chains <br /> mutagenesis studies ECD intimin (Liu et al.                        5  &#203;&#353; intimin, determined based 1999, 2002).  mutants                          inspection Luo et al. (2000) X-ray crystal structure <br /> generated binding surface, vast majority                  (Protein Data Bank [PDB] code 1F02). mutants <br /> protein&#226;&#8364;&#8220;protein interface   remained                           screened ability bind intimin-ECD <br /> unstudied.                                                                   ITC. results thermodynamic analysis    article, report systematic mutation                 11 mutants shown Tables 1 2 (results nonalanine residues b-hairpin EPEC                        average 6 SD independent titrations <br /> TIR-IBD lie close proximity intimin,                      TIR-IBD mutant). Figure 2A depicts location <br /> providing complete understanding TIR-                        of residues b-hairpin TIR. intimin binding interface <span id='am-6' about='oboInOwl:date' typeof='owl:Thing'>date</span>. ability of                    chains color-coded according respective point mutants bind intimin-ECD                          DDG values intimin-ECD binding relative wild- <br /> characterized isothermal titration calorimetry (ITC)                      type TIR-IBD. secondary structures examined circular                             alanine point mutants, V299A N304A, <br /> dichroism (CD). study 11 point mutants,                      significant effect TIR-IBD/intimin-ECD bind- identified residue TIR-IBD binding hot spot.                    ing (DDG &#194;&#188;  0.5 +0.5 kcal/mol). Mutation I301 mutants implicate key structural require-                      N306 alanine modest effect (DDG &#194;&#188; +0.5 <br /> ments TIR-intimin association. Finally, demon-                        +1.0 kcal/mol), D302A caused substantial <br />                                                                              change affinity (DDG &#194;&#188; 1.37 kcal/mol). alanine <br />                                                                              mutants (G305, E312, K298, N300) large <br />                                                                              impact binding interaction, DDG values  <br />                                                                              tween 1.75 3.00 kcal/mol. <br />                                                                                 secondary structure of mutants                                                                              examined using CD spectroscopy ensure                                                                              changes binding affinity observed ITC                                                                              chain interactions  changes                                                                              structure protein. seen Figure <br />                                                                              2B, mutants appear similar CD <br />                                                                              spectra wild-type TIR-IBD, exception <br />                                                                              G305A K298A. mutants,                                                                              designed probe structural importance TIR-IBD&#226;&#8364;&#8482;s <br />                                                                              b-hairpin, altered overall structure TIR-IBD                                                                              expected. loss binding affinity mutation                                                                              G305 K298 likely reflects significance                                                                              b-hairpin&#226;&#8364;&#8482;s conformation, positioning relative                                                                              rest TIR-IBD. <br />                                                                                 Mutation asparagine-300 alanine Figure 1.   Depiction intimin-ECD (surface) TIR-IBD (wire) cocrystallized form. (B) Close representation intimin-ECD/   significant impact TIR-IBD&#226;&#8364;&#8482;s ability <br /> TIR-IBD interface showing b-hairpin TIR-IBD examined          bind intimin-ECD altering global fold <br /> study.                                                                       (DDG &#194;&#188; 2.81 kcal/mol). effort probe  <br />  <br /> 2678     Protein Science, vol. 16 <br />                                                                                                                             binding hot spot TIR <br />  <br />  <br /> Table 1. Analysis native mutant TIR-IBD proteins ITC <br /> Mutant              Kd (nM)              DG (kcal/mol)           DH (kcal/mol)              DS (cal/mol*K)              TDS (kcal/mol)           N-value <br />       Native             577   6   90           8.50   6   0.07        21.80   6   0.74            44.66   6   2.30            13.43   6   0.70      0.75 6 0.12 <br /> E312A           21,586   6   5486         6.39   6   0.15        29.40   6   7.12            77.22   6   24.38           23.01   6   7.27      1.20 6 0.19 <br /> N306A            1,667   6   83           7.89   6   0.03        27.46   6   0.77            65.67   6   2.66            19.57   6   0.79      0.87 6 0.01 <br /> G305A           13,167   6   1303         6.68   6   0.06        36.61   6   4.56           100.44   6   15.39           29.93   6   4.59      0.93 6 0.15 <br /> N304A              390   6   23           8.75   6   0.04        23.08   6   0.42            48.10   6   1.51            14.33   6   0.45      0.93 6 0.02 <br /> D302A            5,760   6   194          7.16   6   0.02        27.19   6   1.19            67.22   6   4.04            20.03   6   1.21      1.04 6 0.05 <br /> I301A            2,189   6   142          7.74   6   0.04        30.56   6   0.73            76.59   6   2.56            22.82   6   0.76      0.98 6 0.03 <br /> N300A           69,881   6   17354        5.69   6   0.14        23.66   6   3.03            60.31   6   10.59           17.97   6   3.16          1.00 <br /> N300D            5,528   6   312          7.18   6   0.03        29.06   6   1.95            73.43   6   6.61            21.88   6   1.97      0.77 6 0.06 <br /> N300Q           20,204   6   878          6.41   6   0.03        19.67   6   0.43            44.50   6   1.53            13.26   6   0.46          1.00 <br /> V299A              291   6   46           8.93   6   0.10        28.73   6   0.68            66.43   6   2.58            19.80   6   0.77      0.94 6 0.02 <br /> K298A           75,114   6   4742         5.64   6   0.04        32.35   6   1.60            89.62   6   5.51            26.71   6   1.64          1.00 <br />  thermodynamic parameters governing TIR-IBD/intimin-ECD binding shown standard deviations average values. data set represents <br /> dilution controlled experimental results, repeated triplicate. N300A, N300Q, K298A, necessary set N-value 1.00 order curve fitting process converge.   Native TIR-IBD/intimin-ECD titrations average 6 SD seven independent titrations (Ross et al. 2007). <br />  <br />  <br />  <br /> interaction, additional mutations  N300D                         asparagine-304 valine-299, N300Q showed significant decrease binding                        significant effect TIR-IBD&#226;&#8364;&#8482;s ability bind intimin- <br /> free energy (DDG &#194;&#188; 1.32 2.09 kcal/mol, respectively)                       ECD. N304 points directly bulk solvent, away <br /> compared wild-type TIR-IBD, supporting                            intimin, cocrystal structure pivotal role asparagine-300 interaction                         proteins. Accordingly, surprising mutation <br /> (Tables 1, 2).                                                                 residue did pronounced effect    order examine importance confor-                       binding. Valine-299 slightly surprising, mational constraint TIR-IBD&#226;&#8364;&#8482;s ability bind intimin-                      chain lies directly surface intimin.  <br /> ECD, synthesized tested short peptides.                            does fit pocket surface peptides (INV, AINV, INVK) designed                             feature, replacing hydrophobic residue represent potential minimal binding domains TIR,                         hydrophobic residue, albeit smaller chain, <br /> incorporating key residues identified mutagenesis                           did reduce binding. minimizing total length effort reduce                          mutants, N306A I301A, modestly <br /> entropic penalty binding. showed measur-                          decreased binding free energy (DDG &#194;&#188; +0.5 +1.0 kcal/mol) <br /> able affinity intimin-ECD, determined ITC (data                      relative wild-type TIR-IBD. Asparagine-306 shown).                                                                    hydrogen-bonding distance lysine-919 intimin,  <br />  <br /> Discussion binding interface TIR intimin relatively                         Table 2. effect mutating given residue compared small, burying 1335  &#203;&#353; 2 solvent-exposed surface area                     wild-type TIR-IBD/intimin-ECD binding <br /> (Luo et al. 2000; Headd et al. 2007).                                        DDG                DDH              DDS          TDDS <br /> surprising residues TIR located near                       Mutant         (kcal/mol)         (kcal/mol)      (cal/mol*K)    (kcal/mol) interface impact intimin binding.                             Nativea           0.00               0.00               0.00         0.00  study implicates key residues, aspar-                      E312A             2.11               7.60              32.56         9.58 <br /> agine-300, isoleucine-301, <span id='am-14' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-15' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span>glutamic acid</span>-312,                       N306A             0.61               5.66              21.01         6.14 <br /> binding hot spot (Clackson Wells 1995; Jones                           G305A             1.82              14.81              55.78        16.50 <br /> Thornton 1997) protein&#226;&#8364;&#8220;protein interaction inter-                      N304A             0.25               1.28               3.44         0.90 <br />                                                                                D302A             1.37               5.39              22.56         6.60 <br /> face. Additionally, conservative mutations N300                        I301A             0.76               8.76              31.93         9.39 <br /> allowed observation ability                           N300A             2.81               1.86              15.65         4.54 <br /> asparagine form intermolecular bonds,                         N300D             1.32               7.26              28.77         8.45 <br /> placement, affects binding intimin-ECD.                                     N300Q             2.09               2.13               0.16         0.17 <br />    Mutagenesis revealed residues contribute                      V299A             0.43               6.93              21.77         6.37 <br />                                                                                K298A             2.86              10.55              44.96        13.28 binding interaction lesser extent,                                                                                appear participate binding  residues,                         Native TIR-IBD/intimin-ECD values Ross et al. (2007). <br />  <br />  <br />                                                                                                                         www.proteinscience.org      2679 <br />  Ross Miller <br />  <br />  <br />  <br />                                                                                removal entropic penalty required freezing <br />                                                                                I301 chain rotation (Harpaz et al. 1994; Stites <br />                                                                                1997; Lee et al. 2000). <br />                                                                                   Mutation D302 alanine produces DDG                                                                                binding 1.37 kcal/mol. initially somewhat <br />                                                                                puzzling, D302 does appear                                                                                direct interactions intimin. chain oxygen <br />                                                                                does appear make cross-strand hydrogen bond                                                                                backbone amide TIR N306.  mutation                                                                                residue alanine did alter CD spectrum                                                                                TIR-IBD. Aspartic acid cross-strand hydrogen bonds                                                                                documented, stabilizing factors type-II b-turns <br />                                                                                (Hutchinson Thornton 1994);  mutation                                                                                residue likely caused small changes orientation                                                                                conformational dynamics b-hairpin (accounting                                                                                change binding free energy), altering <br />                                                                                global fold protein. <br />                                                                                   amino acids greater 1.5 kcal/mol <br />                                                                                effect DG mutated alanine.  lysine- <br />                                                                                298, participates intramolecular salt bridge <br />                                                                                TIR aspartic acid-315, buries methylene <br />                                                                                groups surface intimin (Fig. 2A). Mutation                                                                                K298 alanine resulted DDG 2.86 kcal/mol. Figure 2.   TIR-IBD residues studied alanine scanning mutagenesis         salt bridge K298 D315 located exactly shown colors corresponding effect binding free energy <br /> change, DDG (green indicates  0.50 +0.50 kcal/mol; yellow, +0.50                                                                                boarder b-hairpin helix TIR-IBD. +1.0 kcal/mol; orange, +1.0 +1.75 kcal/mol; red, +1.75 +3.0 kcal/        bridge serve maintain angle positioning mol). Alanine residues b-hairpin mutated     b-hairpin TIR-IBD respect rest study, colored gray. (B) Circular dichroism results   protein. fact, bond broken  mutation mutants studied. data normalized concentration         lysine alanine), random coil features buffer subtracted. majority mutations global fold native TIR-IBD, K298A G305A greater <br />                                                                                predominant component secondary structure contribution overall fold random coil.                           protein (Fig. 2B). Additionally, K298A mutation <br />                                                                                causes large decrease TDDS ( 13.28 kcal/mol), <br />                                                                                consistent greater binding-induced loss  absence H-bonding capability account                         formational freedom K298A mutant. Accordingly, 0.61 kcal/mol loss DG observed N306A.                            alteration structure alignment b-hairpin investigators estimated energetic value                     TIR likely responsible majority loss hydrogen bond protein association,                            binding affinity. <br /> generally accepted range 0.6&#226;&#8364;&#8220;1.0 kcal/mol                               second mutation pronounced effect <br /> (DG) reasonable approximation (Myers Pace                             TIR-IBD/intimin-ECD binding <span id='am-26' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-27' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span>-305. 1996; Klostermeier Millar 2002). I301A DDG                           unusual mutate <span id='am-38' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span>glycine</span> residue conducting 0.76 kcal/mol. Isoleucine-301 buries chain                    alanine scanning mutagenesis, interested deter- <br /> pocket surface intimin (Fig. 2A), potentially                        mining converting G305 alanine disrupt <br /> serving &#226;&#8364;&#732;&#226;&#8364;&#732;set register&#226;&#8364;&#8482;&#226;&#8364;&#8482; binding interaction                type-II b-hairpin, serving indicator hydrophobic burial (Chothia Janin 1975; Tsai                          structural importance.  mutation alanine et al. 1997). low temperature factors residue                     position 305 caused loss binding affinity (DDG) 1.82 <br /> (Supplemental Fig. S2) suggest I301 critical                      kcal/mol altered overall fold protein TIR-intimin interaction.   does mutating                      (Fig. 2B). structural mutation caused large <br /> residue alanine result DDG 0.76 kcal/mol?                      entropic penalty binding (TDDS &#194;&#188;  16.50 kcal/mol). possible alanine methyl group able                        Taken  K298A G305A mutants <br /> provide registration function isoleucine                    indicate fold orientation b-hairpin chain native structure. Additionally,                       TIR-IBD essential submicromolar binding speculate favorable binding entropy                          intimin-ECD. <br /> I301A binding intimin largely differences                         K298A G305A significant effects solvent release, partially compensated                     binding altering structure TIR-IBD,  <br />  <br /> 2680      Protein Science, vol. 16 <br />                                                                                                              binding hot spot TIR <br />  <br />  <br />  <br /> residues exhibit large effects binding                      loss affinity displayed N300A mutant, large <br /> disrupting fold protein. Glutamic acid-312                        Second, asparagine-300 changed appears make direct salt bridge lysine-927                       aspartic acid. anticipated intimin forms pocket surface                      smaller effect DG N300A mutation, TIR occupied intimin K927.                         direct H-bond lost. case, <br /> unsurprising E312A mutant destabilizing                      DDG 1.32 kcal/mol compared wild-type <br /> effect 2.11 kcal/mol. noteworthy E312                 TIR-IBD. Interestingly, change binding free key residue differentiating EPEC TIR-IBD                           energy closely resembles expected <br /> EHEC TIR-IBD (DeVinney et al. 1999; Luo et al. 2000).                       H-bonds broken, providing evidence EHEC TIR-IBD, glutamic acid replaced valine                     H-bond N300 intimin N932 serve position 312. makes sense E312A                       orient N932 way chain oxygen mutant roughly matches affinity EHEC TIR-IBD                     H-bond I301 (Fig. 3). <br /> EPEC intimin (DeVinney et al. 1999).                                           importance asparagine-300 reinforced <br />    Mutation asparagine-300 alanine                            temperature factors cocrystal structure destructive effect TIR-IBD binding affinity                         intimin-ECD TIR-IBD examined (Supplemen- <br /> (DDG &#194;&#188; 2.81 kcal/mol), affecting overall                        tal Fig. S2). comparison rest TIR-IBD, <br /> fold protein. detailed examination                       N300 atoms low B-values (Luo et al. 2000). crystal structure revealed chain aspar-                    appears asparagine-300 complex <br /> agine-300 likely participates direct contacts                   group interactions intimin-ECD TIR-IBD, <br /> intimin-ECD (Fig. 3). chain amino group ap-                        serve central anchoring point TIR- <br /> pears form hydrogen bond glutamic acid-930,                       IBD. idea supported recent review chain oxygen H-bonds chain                         protein&#226;&#8364;&#8220;protein interactions (Headd et al. 2007), amino group asparagine-932. hydrogen bond                           common theme key contact areas N932 orient chain N932                      presence backbone&#226;&#8364;&#8220;backbone H-bonds, way oxygen accept hydrogen bond                            case region TIR-IBD intimin- backbone amide nitrogen isoleucine-301                       ECD (Fig. 3). <br /> TIR-IBD.                                                                       mutagenesis results summed    investigate N300, additional mutations                    overall hypothesis follows: key hydrogen-bonding   asparagine-300 mutated gluta-                      interactions provided asparagine-300, lock key effort determine importance                        type fit isoleucine-301, salt bridge glutamic <br /> precise placement chain. unimportant,                  acid-312 EPEC TIR serve hot spot intimin- argue asparagine-300 involved                      TIR binding. consistent results highly coordinated network hydrogen bonds                           Liu et al. (1999), observed asparagine-932 intimin TIR. N300Q substantial effect                     EHEC intimin  interact N300 EPEC/ binding (DDG &#194;&#188; 2.09 kcal/mol), slightly                    EHEC TIR) important TIR-IBD binding.                                                                             complete lack affinity demonstrated synthetic <br />                                                                             peptides INV, AINV, INVK hot spot <br />                                                                             sequence confer intimin binding <br />                                                                             ability, consistent results secondary <br />                                                                             structure-altering mutants K298A G305A.                                                                             residues important preorganizing TIR-IBD                                                                             binding-competent conformation. Glycine-305                                                                             lysine-298 allow proper type-II b-turn,                                                                             positioning, respectively. high sequence <br />                                                                             identity b-hairpin EPEC EHEC TIR <br />                                                                             (85%), likely interactions                                                                              exception E312 explained earlier) govern                                                                             binding proteins cognate receptors.  <br /> Figure 3. Key bonds N300 TIR-IBD intimin-ECD             time, provides detailed explanation shown black dashed lines. Proposed hydrogen bonds          critical protein&#226;&#8364;&#8220;protein interaction event controlled <br /> backbone TIR-IBD intimin-ECD shown gray dashed lines.         enteropathogenic enterohemorrhagic E. coli. Mutagenesis, CD, peptidomimetic screening results indicate N300, I301, E312 integral residues TIR-IBD/intimin-ECD          anticipate information prove useful association, structural preorganization b-hairpin residues TIR-   design peptidomimetics able interfere IBD required binding.                                                intimin-TIR binding. <br />  <br />  <br />                                                                                                           www.proteinscience.org   2681 <br />  Ross Miller <br />  <br />  <br />  <br /> Materials Methods                                                experiments, calorimeter sample cell injection needle <br />                                                                      cleaned 500 mL distilled, deionized <br />                                                                      water, followed final cleaning dialysis buffer. Molecular biology                                                    protein samples degassed stirring 25&#194;&#176;C                                                                      5-min intervals. Data fit using MicroCal Origin 7.0 <br /> Intimin-ECD (amino acids 658&#226;&#8364;&#8220;939) TIR-IBD (272&#226;&#8364;&#8220;363) <br />                                                                      software ITC instrument. Binding curves analyzed <br /> constructs kindly provided Professors Natalie <br />                                                                      using site binding model Origin (v 7.0; Origin Labs). <br /> Strynadka Brett Finlay (University British Columbia) <br />                                                                      Representative isotherms generated TIR-IBD mutant pET-28a(+) vectors prepared described previously (Luo <br />                                                                      available Supplemental material (Supplemental <br /> et al. 2000). constructs transformed E. coli <br />                                                                      Fig. S1). Reported binding constants average values ob- <br /> BL21(DE3)pLysS BL21(DE3) cell lines, respectively. Cells <br />                                                                      tained independent measurements. grown 37&#194;&#176;C Lauria-Bertani media 25 mg/mL <br /> kanamycin optical density (595 nm) 0.5&#226;&#8364;&#8220;0.8 reached. shifted room temperature, isopropyl b-D-1-thiogalactopyranoside (IPTG) added          Circular dichroism <br /> final concentration 1 mM. Following 24-h induction period,      CD experiments conducted using AVIV 202 spectrom- <br /> cells pelleted 3000 rpm Beckman GS-6R swinging          eter 25&#194;&#176;C (AVIV Biomedical). Scans conducted bucket centrifuge (Beckman Coulter). Cell pellets          260&#226;&#8364;&#8220;200 nm 0.5-nm increments 5-sec sampling time resuspended buffer (20 mM HEPES, 500 mM NaCl, pH 7.5)             0.5 nm, using 1-mm path length cell (Hellma). Spectra sonicated 15 min. suspension pelleted        normalized, truncated 210 nm  strong buffer signal <br /> 15,000 rpm 1 h, filtered 0.45-mm filter,           started obscure protein signal), plotted using Excel <br /> applied Pharmacia Hi-trap nickel chelating column using       (Microsoft Office 2002; Microsoft). <br /> peristaltic pump flow rate 0.5 mL/min. column washed 20 mM HEPES, 500 mM NaCl, 50 mM <br /> imidazole (pH 7.5), remove nonspecifically bound pro-         <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>Peptide</span> synthesis <br /> tein. Bound protein eluted 20 mM HEPES, 500 mM <br /> NaCl, 300 mM imidazole, pH 7.5, examined purity          Unless stated chemicals reagents used 12% SDS-PAGE. ITC CD experiments, pro-             peptide synthesis purchased Sigma-Aldrich used <br /> tein samples dialyzed 24 h buffer change           obtained manufacturer. INV, AINV, INVK 200 volumes dialysis buffer (50 mM HEPES, 150 mM          peptides synthesized para-methyl-benzhydrylamine <br /> NaCl, pH 7.2). Following dialysis, concentrations deter-        resin (0.4 mmol/g loading) (Advanced ChemTech). Resin mined UV-vis spectrophotometry using calculated molar         initially swollen methylene chloride (DCM; Mallinckrodt <br /> extinction coefficients 37,080 1280 (M 1cm 1)             Baker) treated 10% N,N9-diisopropylethylamine <br /> intimin-ECD TIR-IBD, respectively (Ross et al. 2007).            (DIPEA; Advanced ChemTech) DCM. Resin    TIR-IBD point mutants prepared using Stratagene            washed sequence 33 DCM, 33 N,N-dimethylform- <br /> QuikChange mutagenesis kit. residue TIR-IBD              amide (DMF; Mallinckrodt Baker), 33 methanol (BDH/VWR), <br /> b-hairpin analyzed determine chain     33 DCM, 13 DMF, nutating 5 min wash step. Couplings <br />                                &#203;&#353; ) intimin. determined <br /> interacting distance (limit: 5                                     conducted using 2-(1H-Benzotriazole-1-yl)-1,1,3,3-tetra- <br /> using previously published cocrystal structure intimin-       methyluronium hexafluorophosphate (HBTU; 3.0 equivalents <br /> ECD TIR-IBD (Luo et al. 2000) visualized PyMOL           respect resin; GenScript), N (9-fluorenylmethyloxy- <br /> (v. 0.99; DeLano Scientific). Primers selected mutants       carbonyl; Fmoc)&#226;&#8364;&#8220;protected amino acids (3.0 eq; Advanced designed according kit recommendations, sequences        ChemTech), DIPEA (5.0 eq) DMF 30 min (3 min available Supplemental material (Supple-         HBTU, Fmoc-<span id='am-7' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span>, DIPEA premixing DMF). <br /> mental Table S1). sequences individual mutants           Amino acids double-coupled washed  <br /> confirmed University Rochester Functional Genomics         Fmoc protecting group removed cycles Center (Rochester, NY). Mutants desired sequences           20% piperidine/DMF (30 min , followed washing transformed BL21(DE3) E. coli cells, protein        cycle double-couple, double-deprotect prepared, dialyzed (50 mM HEPES, 150 mM NaCl, pH 7.2),           repeated synthesis completed. coupling <br /> analyzed purity using 12% SDS-PAGE manner            deprotection, small (;2 mg) aliquot resin heated wild-type TIR-IBD. Concentrations determined using           ninhydrin stain determine free amines present. molar extinction coefficient used native TIR-        cases, desired presence lack free amines IBD. buffer used experiments discussed        observed. used protein dialysis.                                  Following end chain elongation, global deprotection <br />                                                                      undertaken remove Fmoc group, followed                                                                      washing removal trityl chain protecting group <br /> Isothermal titration calorimetry                                     (2% trifluoroacetic acid [TFA], 2% triethylsilane [TES; Oak- <br />                                                                      wood Products], DCM, 2 3 30 min). final wash se- <br /> ITC experiments carried using MicroCal VP-ITC             quence, peptides liberated resin 5% TES <br /> isothermal titration calorimeter. experiments carried       TFA (2 mL/100 mg resin) 10 min 0&#194;&#176;C, followed 25&#194;&#176;C (unless stated) dialysis buffer           addition trifluoromethanesulfonic acid (TFMSA; 200 mL/100 <br /> (described  reference cell. titrations         mg resin). reaction vessels allowed cool, reference subtracted control heats dilution,      resin solvent mixture stirred room temperature cases negligible. experiments performed       2 h. Resin solvent filtered glass wool, <br /> 10 mcal/sec reference power, 290 rpm stir rate, 60 5-mL          peptides precipitated cold ether (;5 volumes; <br /> injections 300 sec spacing injections.      EMD Biosciences). Acid/ether solutions left  20&#194;&#176;C <br />  <br /> overnight, precipitates centrifuged 2000 rpm                    30 min 4&#194;&#176;C. acid/ether solution removed,                   <br /> pellets washed cold ether centrifuged               <br /> Pellets washed fashion times (total)            <br /> allowed dry 6 h placed high vacuum                    <br /> overnight. Peptides solubilized protein dialysis buffer,              final concentration 5 mM. screened                 <br /> ability bind 100 mM intimin-ECD ITC.                                <br /> </body></html>