<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                                   JCB: Report <br />  <br />  <br />  <br />  <br />                               NuRD chromatin&#226;&#8364;&#8220;remodeling complex regulates <br />                               signaling repair DNA damage <br />                               Godelieve Smeenk,1 Wouter W. Wiegant,1 Hans Vrolijk,2 Aldo P. Solari,3 Albert Pastink,1 Haico van Attikum1 <br />                               1 <br />                               Department Toxicogenetics, 2Department Molecular Cell Biology, 3Department Medical Statistics, Leiden University Medical Center, Leiden 2300RC, Netherlands <br />  <br />  <br />  <br />  <br />                               C <br />                                        ells respond ionizing radiation  &#226;&#8364;&#8220;induced                              catalytic subunit (<span id='am-37' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-49' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenosine</span> triphosphatase [ATPase]) <br />                                        DNA double-strand breaks (DSBs) orches-                                   NuRD, leads accumulation spontaneous DNA <br />                                        trating events coordinate cell cycle pro-                               damage increased sensitivity. MTA2 CHD4 <br />                               gression DNA repair. cells signal repair                                  accumulate DSB-containing chromatin tracks gener- JOURNAL CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                               DSBs fully understood. genome-wide RNA                                   ated laser microirradiation. Directly DSBs, CHD4 <br />                               interference screen Caenorhabditis elegans identified                              stimulates RNF8/RNF168-dependent formation ubiq- <br />                               egr-1 factor protects worm cells                             uitin conjugates facilitate accrual RNF168                               human homologue egr-1, MTA2 (metastasis-associated                                 BRCA1. Finally, CHD4 promotes DSB repair <br />                               protein 2), subunit nucleosome-remodeling                                 checkpoint activation response                                 histone deacetylation (NuRD) chromatin-remodeling                                 NuRD chromatin&#226;&#8364;&#8220;remodeling complex novel regu- <br />                               complex. knockdown MTA2 CHD4                                      lator DNA damage responses orchestrates proper <br />                               (chromo&#194;&#173;domain helicase DNA-binding protein 4),                                   signaling repair DSBs. <br />  <br />  <br />  <br />  <br />                               Introduction <br />                               efficient accurate repair chromosomal double-strand                        arrive DSBs MDC1, directly binds &#238;?&#167;H2AX (Stucki <br />                               breaks (DSBs), arise exposure agents                           et al., 2005). allows recruitment E2 ubiq- <br />                               ionizing radiation  , critical maintaining genome sta-                    uitin conjugase UBC13 E3 ubiquitin ligase RNF8, <br />                               bility preventing cell death carcinogenesis. avoid                         binds MDC1 (Huen et al., 2007; Kolas <br />                               deleterious effects DSBs, eukaryotic cells activate signaling                      et al., 2007; Mailand et al., 2007). RNF8/UBC13 promote                               cascades, called checkpoints, coordinate rapid detection                        ubiquitylation histones H2A/H2AX, leading recruit- <br />                               DNA breaks temporal arrest cell cycle progression                        ment E3 ubiquitin ligase, RNF168, associates <br />                               activation DNA repair mechanisms (Hoeijmakers, 2001;                           RNF8-ubiquitylated histones ubiquitin-interacting <br />                               Khanna Jackson, 2001).                                                            motifs (Huen et al., 2007; Kolas et al., 2007; Mailand et al., <br />                                     cellular response DSBs predominantly                               2007; Doil et al., 2009; Stewart et al., 2009). RNF168 coop- <br />                               ordinated <span id='am-8' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-9' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span>phosphatidylinositol</span> 3-kinase&#226;&#8364;&#8220;like kinases                           erates UBC13 amplify RNF8-mediated histone ubiq- <br />                               ataxia telangiectasia mutated (ATM) ataxia telangiectasia                         uitylation threshold required accumulation                               Rad3 related (ATR). ATM phosphorylates histone H2AX                               checkpoint repair proteins, including BRCA1, 53BP1, <br />                               (&#238;?&#167;H2AX) DSB-flanking chromatin create environment                            RAD18, PTIP, DSB-flanking chromatin compart- <br />                               allows spatiotemporal redistribution accumu-                         ment (Wang Elledge, 2007; Doil et al., 2009; Gong et al., <br />                               lation checkpoint DNA repair factors DNA breaks                             2009; Huang et al., 2009; Stewart et al., 2009). addition                               (van Attikum Gasser, 2009). proteins                           ATM-driven events, DSB ends undergo extensive resec- <br />                                                                                                                     tion, leading formation replication protein coated <br />                               Correspondence Haico van Attikum: h.van.attikum@lumc.nl                            single-stranded DNA subsequent assembly activation <br />                               Abbreviations used paper: ATM, ataxia telangiectasia mutated; <br />                                                                                                                     &#194;&#169; 2010 Smeenk et al.&#226;&#8364;&#402; article distributed terms Attribution&#226;&#8364;&#8220; <br />                               ATR, ataxia telangiectasia Rad3 related; DDR, DNA damage response;                Noncommercial&#226;&#8364;&#8220;Share Alike Mirror Sites license months pub- <br />                               DSB, double-strand break;  ionizing radiation; IRIF, induced foci; NuRD,        lication <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>  http://www.rupress.org/terms). months available                               nucleosome remodeling histone deacetylation; TERT, telomerase reverse             Creative Commons License (Attribution&#226;&#8364;&#8220;Noncommercial&#226;&#8364;&#8220;Share Alike 3.0 Unported license, <br />                               transcriptase; WCE, cell extract.                                               described http://creativecommons.org/licenses nc-sa/3.0/). <br />  <br />  <br />  <br />  <br />                               Rockefeller University Press&#226;&#8364;&#402;&#226;&#8364;&#352;$30.00 <br />                               J. Cell Biol. Vol. 190  5&#226;&#8364;&#402; 741&#226;&#8364;&#8220;749 <br />                               www.jcb.org/cgi/doi/10.1083/jcb.201001048                                                                                                                                    JCB   741 <br />            ATR (Zou Elledge, 2003; Jazayeri et al., 2006; Dubrana        Zhang et al., 1999). biochemical study indicated MTA2 <br />           et al., 2007). Finally, ATM ATR amplify signals gener-       modulates histone deacetylation activity NuRD (Zhang <br />           ated DSBs phosphorylating regulatory proteins,         et al., 1999). chromatin-remodeling activity complex <br />           including SMC1, CHK1, CHK2, p53, p21, coordinate            resides subunit, CHD4 (chromodomain helicase <br />           cell cycle progression induce apoptosis (Khanna Jackson,      DNA-binding protein 4), identified derma- <br />           2001; Shiloh, 2003).                                                 tomyositis-specific autoantigen (Seelig et al., 1995). CHD4                  cross talk histone modifications (e.g., phos-     member SNF2 family ATPases possesses intrinsic <br />           phorylation ubiquitylation) DSB-flanking chromatin        <span id='am-26' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>-dependent nucleosome-remodeling activity (Wang           trols ATM/ATR-dependent signaling repair DSBs,         Zhang, 2001). thought NuRD represses transcription <br />           unclear achieved (van Attikum Gasser, 2009).      regulating chromatin structure (Denslow Wade, 2007). <br />           possible mechanism ATP-dependent chromatin                 recent study showed loss NuRD <br />           remodeling performed ATPases SNF2 superfamily              components results chromatin defects associated <br />           (Clapier Cairns, 2009). previously dem-       DNA damage accumulation aging (Pegoraro et al., <br />           onstrated budding yeast chromatin-remodeling         2009).  NuRD preserves genome stability           complexes (INO80, SWR1, SWI/SNF, RSC) recruited              regulates DDR remained unclear. <br />           DSBs, change chromatin structure distinct                  investigate  transfected U2OS cells siRNAs <br />           ways regulate cell cycle progression DSB repair            luciferase CHD4 counted cells 2, 3, 4 d           (Morrison et al., 2004; van Attikum et al., 2004, 2007; Chai         siRNA treatment. CHD4 knockdown cells proliferated           et al., 2005; Shim et al., 2005, 2007; Tsukuda et al., 2005;         slower control cells (Fig. 1, B). Flow cytometric analy&#194;&#173; <br />           Papamichos-Chronakis et al., 2006; Liang et al., 2007).       sis cells did significant changes cell <br />           trast budding yeast, human cells, role chromatin        cycle distribution.  morphological changes marked <br />           remodeling cellular response DSBs just begin-       sub-G1 peaks, indicative apoptosis, observed 2&#226;&#8364;&#8220;4 d           ning emerge. TIP60 SWI/SNF chromatin-remodeling           siRNA transfection (Fig. 1, B C). Consistently, levels <br />           complexes, example, implicated DNA repair           p53, phosphorylated p53 (S15p), p53 effector p21, <br />           promoting histone acetylation H2AX phosphorylation            coordinate cell cycle progression apoptosis, sig- <br />           DSBs, respectively (Murr et al., 2006; Park et al., 2006; Ikura   nificantly increased absence CHD4 (Fig. 1 D),           et al., 2007), ALC1 recently shown assist        agreement earlier study implicating role NuRD <br />           poly(<span id='am-2' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span>ADP</span>-ribose) polymerase&#226;&#8364;&#8220;dependent chromatin remod-            apoptosis p53/p21 regulation (Luo et al., 2000).  <br />           eling sites DNA damage (Ahel et al., 2009; Gottschalk          vestigated apoptosis induced loss CHD4           et al., 2009).  unclear interplay      related spontaneous occurrence DNA lesions.  <br />           chromatin remodeling histone modifications DSBs            CHD4 knockdown cells showed increased levels &#238;?&#167;H2AX           ordinates cell cycle progression DNA repair.                     early 2 d siRNA transfection (Fig. 1 D), corroborating <br />                  study, identify nucleosome-remodeling      findings recent study (Pegoraro et al., 2009).  CHD4 <br />           histone deacetylation (NuRD) complex novel regulator         depletion leads accumulation spontaneous DNA dam- <br />           DSB response human cells. NuRD prevents accumu-           age activation apoptotic p53/p21 program. infer <br />           lation spontaneous DNA damage regu&#194;&#173;lates apoptotic            NuRD prevents genome instability apoptosis. <br />           responses p53 p21.  NuRD rapidly  <br />           cruited DSBs, promotes RNF8/RNF168-mediated              CHD4 MTA2 protect cells           histone ubiquitylation ubiquitin-dependent accumula-         clastogenic effects           tion RNF168 BRCA1. Consequently, loss NuRD                 EGR-1 (MTA2) protects worm cells (van Haaften et <br />           components causes defects DNA repair checkpoint acti-         al., 2006). examine MTA2 protects human cells <br />           vation, rendering cells hypersensitive   NuRD          tested depletion affects clonogenic <br />           chromatin&#226;&#8364;&#8220;remodeling complex novel DNA damage response          survival VH10-SV40 cells. Loss MTA2 led increase <br />           (DDR) factor helps preserve genome stability regulating   sensitivity comparable observed           signaling repair DNA damage.                                  XRCC4 knockdown cells, impaired DSB repair                                                                                nonhomologous end joining (Fig. 1, E F; Grawunder et al., <br />           Results discussion                                               1998). addition, CHD4-depleted cells  <br />                                                                                creased sensitivity (Fig. 1, E G).  MTA2           CHD4 preserves genome stability                                  CHD4 protect cells effects  implicating role <br />           prevents apoptosis                                                   NuRD cellular response DSBs. Furthermore, MTA2 <br />           previously performed genome-wide RNAi screen             protects worm human cells  sug- <br />           nematode Caenorhabditis elegans identified 45 genes         gest putative role DDR conserved. <br />           protect worms (van Haaften et al., 2006).           genes known DDR factors novel            CHD4 controls p53/p21 axis           genes, including egr-1/lin-40. egr-1 encodes protein   induced DDR <br />           homologous MTA2 (metastasis-associated protein 2),          investigate role NuRD DDR, examined           component human NuRD complex (Xue et al., 1998;          CHD4 depletion affects ATM/ATR-dependent phosphorylation <br />  <br />  <br />     742   JCB &#226;&#8364;&#162; VOLUME 190 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2010 <br />                                                                                            Figure 1.&#226;&#8364;&#402; MTA2 CHD4 depletion renders <br />                                                                                           cells sensitive    Depletion CHD4 <br />                                                                                           reduces cell proliferation. U2OS cells                                                                                           transfected indicated siRNAs. Cells <br />                                                                                           counted 0, 2, 3, 4 d siRNA <br />                                                                                           transfection. (B) Pictures representative <br />                                                                                           areas cell dishes shown. <br />                                                                                           Bar, 200 &#194;&#181;m. (C) FACS analysis cells <br />                                                                                            PI, propidium iodide. (D) CHD4 deple- <br />                                                                                           tion leads enhanced levels phosphory- <br />                                                                                           lated p53 (S15p), p53, p21, &#238;?&#167;H2AX. <br />                                                                                           Protein levels monitored Western blot <br />                                                                                           analysis using WCEs cells  SMC1                                                                                           loading control. (E) CHD4, MTA2, XRCC4 <br />                                                                                           levels monitored Western blot analy- <br />                                                                                           sis using WCEs cells F G. Tubulin                                                                                           loading control. (F) MTA2 depletion renders <br />                                                                                           cells hypersensitive  VH10-SV40 cells <br />                                                                                           transfected indicated siRNAs. <br />                                                                                           (G) CHD4 depletion renders cells hypersensi- <br />                                                                                           tive  F, siRNAs                                                                                           CHD4 used. Graphs represent mean &#194;&#177; <br />                                                                                           SEM independent experiments. <br />  <br />  <br />  <br />  DDR components response  Knockdown CHD4            phosphorylation status SMC1 CHK1, reg- <br /> did impair induced ATM activation &#238;?&#167;H2AX             ulators induced intra&#226;&#8364;&#8220;S phase checkpoints, weak <br /> mation, led increased levels &#238;?&#167;H2AX unirradiated      effect cell cycle progression (Fig. 2, B C).  loss <br /> cells, corroborating previous result (Fig. 1 D, Fig. 2      CHD4 enhanced levels total p53 phosphorylated Fig. S1). investigated CHD4 mediates          p53 exposure  accompanied increase <br /> ATM/ATR-dependent activation downstream effectors SMC1         p21 levels 24 h treatment (Fig. 2 . p53 p21 <br /> (S966p), CHK1 (S317p), CHK2 (S19p), p53 (S15p), p21           play prominent role G1 checkpoint response (Khanna DDR. repeatedly observed small increase        Jackson, 2001; Shiloh, 2003). Accordingly, detected phosphorylation SMC1 CHK1, CHK2          arrest CHD4-depleted cells G1 phase maintained 30 min exposure (unpublished data). addi-   exposure (Fig. 2, B C). suggests CHD4 <br /> tion, observed small accumulation CHD4-depleted           controls p53/p21-dependent G1 checkpoint responses induced <br /> cells mid&#226;&#8364;&#8220;S phase, suggesting aberration       <br />  <br />  <br />                                                                   NuRD regulates DNA damage response &#226;&#8364;&#162; Smeenk et al.                    743 <br />                                                                                         NuRD rapidly accumulates                                                                                        DSB-flanking chromatin <br />                                                                                        DSB response characterized accumulation                                                                                        checkpoint DNA repair proteins DSB-flanking chroma- <br />                                                                                        tin. investigate NuRD plays direct role DDR, <br />                                                                                        examined recruitment subunits (CHD4, <br />                                                                                        MTA2, MBD3) sites DNA damage. GFP- <br />                                                                                        tagged CHD4, MTA2, MBD3 accumulate microlaser- <br />                                                                                        generated DSB tracks span entire chromatin region <br />                                                                                        marked &#238;?&#167;H2AX (Fig. 3 Fig. S2 , charac- <br />                                                                                        teristic shared known DSB-associated factors (Bekker- <br />                                                                                        Jensen et al., 2006). Furthermore, rapidly recruited                                                                                        DSB tracks accumulated similar kinetics. Accumulation <br />                                                                                        detectable 30 s, reaching half-maximum 40 s                                                                                        steady-state levels 3 min (Fig. 3, B C; Fig. S2, B C). <br />                                                                                        Mailand et al. (2007) recently defined distinct kinetic groups                                                                                        proteins arrive early (e.g., MDC1 RNF8) late <br />                                                                                        (e.g., BRCA1 53BP1) DSB track. demonstrated                                                                                        GFP-CHD4, MTA2-GFP, GFP-MBD3 arrive DSB track <br />                                                                                        early GFP-MDC1 (Fig. 3 C Fig. S2 C). Interestingly,                                                                                        reached steady-state levels accumulation significantly <br />                                                                                        faster MDC1 BRCA1. suggest NuRD <br />                                                                                        subunits occupy DSB-containing chromatin                                                                                        level core components DSB response (Fig. 3 C                                                                                        Fig. S2 C). conclude NuRD factors                                                                                        assemble early DSB-flanking chromatin. <br />  <br />                                                                                        CHD4 regulates DSB-associated <br />                                                                                        ubiquitylation orchestrate                                                                                        accumulation RNF168 BRCA1 <br />                                                                                        examine NuRD modulates early events DSB <br />                                                                                        response, analyzed induced foci (IRIF) formation                                                                                        &#238;?&#167;H2AX histone mark acts docking site                                                                                        MDC1/RNF8 DSBs (Stucki et al., 2005).  CHD4 <br />                                                                                        knockdown did affect &#238;?&#167;H2AX IRIF formation (Fig. 4 , <br />                                                                                        corroborates Western blot analysis showing proper <br />                                                                                        induced &#238;?&#167;H2AX formation absence CHD4 (Fig. 2                                                                                        Fig. S1). Accordingly, MDC1 RNF8 IRIF formation <br />                                                                                        affected loss CHD4 (Fig. 4, B). <br />                                                                                               contrast, accumulation conjugated ubiquitin <br />                                                                                        IRIF impaired CHD4-depleted cells  twofold; <br />                                                                                        Fig. 4, C D). Consistently, observed distinct reduction                                                                                        IRIF formation RNF168 BRCA1, shown <br />                                                                                        bind DSB-associated ubiquitin moieties (Fig. 4, C D), <br />                                                                                        formation require CHD4.                                                                                         CHD4 knockdown, like RNF8 knockdown, significantly <br />           Figure 2.&#226;&#8364;&#402; CHD4 controls induced p53/p21 responses.   CHD4           decreased level &#238;?&#167;H2AX mono- diubiquitylation <br />           pletion increases induced p53 phosphorylation (S15p), p53, p21        (Fig. S3, B; Huen et al., 2007). total <br />           levels. U2OS cells transfected indicated siRNAs ex- <br />                                                                                        levels endogenous MDC1, RNF8, RNF168, BRCA1 <br />           posed 6 Gy  WCEs prepared indicated time points           &#238;?&#167;H2AX, ATM S1981p, ATM, CHD4, p53 S15p, p53, p21 levels             af&#194;&#173;fected CHD4 depletion, conclude           monitored Western blot analysis. Histone H3 SMC1 loading         observed defects IRIF formation resulted reduction <br />           controls. (B) CHD4- MTA2-depleted cells accumulate G1 phase                                                                                        RNF8/RNF168-dependent ubiquitylation subsequent <br />           remain G1 arrest exposure  U2OS cells transfected           indicated siRNAs 72 h, exposed 6 Gy  12 h later,        RNF168 BRCA1 accumulation DSBs (Fig. S1).  <br />           subjected FACS. (C) B, cells stained BrdU.    expression dominant-negative, ATPase-dead form CHD4 <br />           PI, propidium iodide. <br />                                                                                        (GFP-CHD4 K757R) reduced RNF168 IRIF formation (Fig. S3, <br />                                                                                        C D), suggesting ATP-dependent chromatin remodeling <br />                                                                                        driven CHD4 ATPase triggers process. <br />  <br />  <br />     744   JCB &#226;&#8364;&#162; VOLUME 190 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2010 <br />                                                                                    change nucleosome structures inacces- <br />                                                                                   sible RNF8/RNF168 H2A-type histone targets ame- <br />                                                                                   nable ubiquitylation. <br />  <br />                                                                                   CHD4 MTA2 promote DSB repair                                                                                   G2/M checkpoint activation <br />                                                                                   RNF168 BRCA1 promote DSB repair G2/M check- <br />                                                                                   point activation response (Moynahan et al., 1999; Xu <br />                                                                                   et al., 2001; Xie et al., 2007; Doil et al., 2009; Stewart et al., <br />                                                                                   2009). Given CHD4 facilitates accumulation                                                                                   proteins DSB-flanking chromatin, examined role                                                                                   DSB repair G2/M checkpoint activation. <br />                                                                                          Neutral comet assays performed determine                                                                                   effect CHD4 knockdown rejoining induced <br />                                                                                   DSBs VH10&#226;&#8364;&#8220;telomerase reverse transcriptase (TERT) cells. <br />                                                                                   shown Fig. 5  B), detected pronounced increase <br />                                                                                   level DSBs unirradiated CHD4-depleted cells.                                                                                   consistent aforementioned elevated levels total <br />                                                                                   &#238;?&#167;H2AX (Fig. 1 D Fig. 2  implies spontaneous <br />                                                                                   DSBs accumulate absence CHD4 (Fig. 1 D). Importantly, <br />                                                                                   level induced DSBs remained higher 2 h CHD4 <br />                                                                                   knockdown cells returned basal levels 6 h. suggests <br />                                                                                   CHD4 promotes proper DSB repair (Fig. 5, B). <br />                                                                                           tested effect CHD4 MTA2 depletion                                                                                   induced G2/M checkpoint activation. CHD4- <br />                                                                                   MTA2-depleted cells, like depleted BRCA1 RNF8 <br />                                                                                   (Xu et al., 2001; Huen et al., 2007; Kolas et al., 2007), failed                                                                                   fully activate G2/M checkpoint continued enter mitosis <br />                                                                                   (Fig. 5, C D), indicating CHD4 MTA2 facilitate  <br />                                                                                   scale activation G2/M checkpoint. <br />                                                                                          conclusion, report identification charac- <br />                                                                                   terization NuRD chromatin&#226;&#8364;&#8220;remodeling complex                                                                                   novel factor involved genome surveillance. lack CHD4 <br />                                                                                   consistently led increased levels spontaneous DNA dam- <br />                                                                                   age, phenotype associated activation                                                                                   p53 p21 responses, reduced cell proliferation, apoptosis. <br />                                                                                   propose loss chromatin remodeling NuRD  <br />                                                                                   duces genome-wide chromatin alterations, render chro- <br />                                                                                   matin susceptible spontaneous DNA breaks (Fig. 5 E). <br />                                                                                   Support scenario comes recent study,                                                                                   demonstrated loss NuRD components                                                                                   premature normal aging associated changes                                                                                   higher-order chromatin structure, including loss heterochro- <br />                                                                                   matic regions, accumulation spontaneous DNA dam- <br /> Figure 3.&#226;&#8364;&#402; CHD4 MTA2 rapidly accumulate sites DNA damage. <br />   CHD4 MTA2 accumulate DSB-containing regions marked                                                                                   age (Pegoraro et al., 2009). <br /> &#238;?&#167;H2AX. U2OS cells transiently expressing GFP-CHD4 MTA2-GFP                       provide evidence direct role NuRD subjected laser microirradiation. 15 min, cells immuno&#194;&#173;             DSB response, NuRD components (CHD4, MTA2, <br /> stained &#238;?&#167;H2AX. (B) GFP-CHD4 MTA2-GFP, like GFP-MDC1, rapidly <br /> accumulate DSB-containing regions. U2OS cells transiently expressing <br />                                                                                   MBD3) assemble DSBs. Loss CHD4 uncoupled GFP-CHD4, MTA2-GFP, GFP-MDC1 microirradiated                  DSB response level RNF8/RNF168-mediated histone <br /> subjected real-time recording protein assembly damaged area.         ubiquitylation, leading defects assembly RNF168 <br /> (C) Quantitative representation results B. U2OS cells transiently ex- <br /> pressing GFP-BRCA1 included. Relative fluorescence units (RFU)                                                                                   BRCA1. likely attenuated DSB repair activa- <br /> plotted time scale. Graphs represent mean &#194;&#177; SEM 10           tion G2/M checkpoint contributed sensitiv- <br /> individual cells independent experiments. Bars, 10 &#194;&#181;m.          ity observed CHD4- MTA2-depleted cells (Fig. 5 E).  <br />                                                                                   NuRD novel factor preserves genome stability mod- <br />        NuRD-mediated chromatin remodeling facilitates                        ulating chromatin structure   genome-wide prevent sponta- <br /> RNF8/RNF168-dependent histone ubiquitylation orches-                           neous DNA damage (b) DNA breaks orchestrate trate accumulation RNF168 BRCA1. NuRD                              proper DSB response  Larsen et al. issue). work <br />  <br />  <br />                                                                                   NuRD regulates DNA damage response &#226;&#8364;&#162; Smeenk et al.                745 <br />            Figure 4.&#226;&#8364;&#402; CHD4 promotes histone ubiquitylation DSBs orchestrate accumulation RNF168 BRCA1.   CHD4 depletion does alter <br />           &#238;?&#167;H2AX, MDC1, RNF8 IRIF formation. Cells transfected indicated siRNAs, exposed 1 Gy  30 60 min later, <br />           immunostained &#238;?&#167;H2AX, MDC1, RNF8. (B) Quantitative analysis &#238;?&#167;H2AX, MDC1, RNF8 IRIF formation. 150 nuclei cells           scored time point independent experiments. (C) CHD4 depletion impairs ubiquitin, RNF168, BRCA1 IRIF formation.  <br />           cells immunostained conjugated ubiquitin (FK2), RNF168, BRCA1. (D) Quantitative analysis ubiquitin, RNF168, BRCA1 <br />           IRIF formation  B). Error bars indicate mean &#194;&#177; SEM. Bars, 10 &#194;&#181;m. <br />  <br />  <br />  <br />  <br />     746   JCB &#226;&#8364;&#162; VOLUME 190 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2010 <br />  Figure 5.&#226;&#8364;&#402; CHD4 promotes DSB repair induced activation G2/M checkpoint.   CHD4 depletion impairs DSB repair. VH10-TERT cells transfected indicated siRNAs 72 h, exposed 20 Gy  subjected neutral comet analysis indicated time points. Representa- <br /> tive images shown. Bar, 30 &#194;&#181;m. (B) Quantification tail moments using cells  Tail moments condition calculated minimum 300 cells data point. Results independent experiments shown box whisker plot. ordinate cube root scale. Data statistically analyzed using way analysis variance, revealed significant delay DSB repair CHD4-depleted cells 2 h (P &lt; 0.0001). (C) CHD4 MTA2 depletion impairs induced G2/M checkpoint activation. U2OS cells transfected indicated siRNAs, <br /> exposed 3 Gy  1 h later, immunostained phosphorylated histone H3 (S10p). Mitotic indexes determined FACS. representative <br /> experiment shown. PI, propidium iodide. Red circle indicates fraction M phase cells. (D) Graphical representation relative mitotic index values. ratio index values irradiated unirradiated cells calculated normalized control cells, set 1. mean &#194;&#177; <br /> SEM experiments shown. (E) Model role CHD4 maintenance genome stability. paragraph Results discus- <br /> sion details. <br />  <br />  <br />                                                                                   NuRD regulates DNA damage response &#226;&#8364;&#162; Smeenk et al.                        747 <br />            provides framework future studies gain                       selected global thresholding DAPI images. Contour tracing           insight cross talk chromatin remodeling                        performed, pixels labeled unique object index                                                                                             determines nucleus pixels belong. actual detection foci           mechanisms involved genome maintenance.                                        performed Alexa Fluor 488 Cy3 images.  hat transformation <br />                                                                                             performed reduce influence variation background staining <br />                                                                                             nuclei. resulting image, watershed algorithm performed <br />           Materials methods                                                             determine location foci image providing unique spot <br />                                                                                             index pixels focus. combining foci indices object indices <br />           Cell culture chemicals                                                        obtained DAPI image, foci assigned nuclei           Human U2OS, VH10-SV40&#226;&#8364;&#8220;, VH10-TERT&#226;&#8364;&#8220;immortalized fibroblasts                    detected, allowing calculation number foci nucleus. <br />           grown DME (Invitrogen) containing 10% FCS (Bodinco BV). siRNA <br />           plasmid transfections performed using HiPerfect (QIAGEN)             Chromatin fractionation Western blotting <br />           JetPEI (Polyplus Transfection), respectively, according manufacturer&#226;&#8364;&#8482;s     Chromatin-enriched extracts prepared used immunoprecipita- <br />           instructions. following siRNA sequences used: (luciferase), 5&#238;&#8364;&#185;-CGUA&#194;&#173;     tion described previously (Huen et al., 2007). cell extracts (WCEs) <br />           CGCGGAAUACUUCGA-3&#238;&#8364;&#185;; (CHD4-1; Thermo Fisher Scientific), 5&#238;&#8364;&#185;-CCAA&#194;&#173;G&#194;&#173;                prepared cell lysis Laemmli buffer. Proteins separated Bis- <br />           GACCUGAAUGAUGA-3&#238;&#8364;&#185;; (CHD4-2; Thermo Fisher Scientific), 5&#238;&#8364;&#185;-CAAA&#194;&#173;                   Tris-HCl&#226;&#8364;&#8220;buffered acrylamide gels (Invitrogen) blotted PVDF <br />           GGUGCUGCUGAUGUA-3&#238;&#8364;&#185;; (MTA2; Thermo Fisher Scientific), 5&#238;&#8364;&#185;-CAAA-                    (Millipore) nitrocellulose (GE Healthcare). Membranes incubated <br />           GUCUCUCUCCUUACAUU-3&#238;&#8364;&#185;; (RNF8; Thermo Fisher Scientific), 5&#238;&#8364;&#185;-GAGG&#194;&#173;                  primary antibody indicated Figs. 1&#226;&#8364;&#8220;5 followed incubation           GCCAAUGGACAAUUA-3&#238;&#8364;&#185;; (XRCC4), 5&#238;&#8364;&#185;-AUAUGUUGGUGAA&#194;&#173;CU&#194;&#173;                             secondary antibody (Odyssey IRDye; LI-COR Biosciences). Odyssey <br />           G&#194;&#173;AGA-3&#226;&#8364;&#8482; (Sartori et al., 2007). Cells examined 48 h siRNA             imager (LI-COR Biosciences) equipped Odyssey software (version 3.0) <br />           transfection unless stated. cDNAs human CHD4 (Thermo            used scan membranes analyze fluorescence signals. <br />           Fisher Scientific) MTA2 (Imagenes) cloned pEGFP-C1           pEGFP-N1, respectively (Takara Bio ). point mutation K757A            FACS <br />           introduced pEGFP-C1-CHD4 site-directed mutagenesis. pGFP-             G2/M checkpoint analysis, cells fixed 70% ethanol           MBD3, GFP-BRCA1, GFP-MDC1 constructs provided  Bird                 stained rabbit antibody histone H3 S10p (Millipore) followed  <br />           (University Edinburgh, Edinburgh, Scotland, UK), O. Sibon (University       cubation conjugated anti&#226;&#8364;&#8220;rabbit IgG (Alexa Fluor 488). cell cycle <br />           Groningen, Groningen, Netherlands), J. Lukas (Danish Cancer Soci-             analysis, cells pulse labeled 10 &#194;&#181;M BrdU 1 h, fixed 70% <br />           ety, Copenhagen, Denmark) respectively.                                           ethanol, denatured 2 M HCl, stained mouse antibody BrdU <br />                                                                                             followed incubation conjugated anti&#226;&#8364;&#8220;mouse IgG (Alexa Fluor 488) <br />           Generation DSBs                                                                DNA staining 0.1 mg/ml propidium iodide. Cell sorting           DSBs induced  delivered x-ray generator                performed flow cytometer (LSRII; BD) using FACSDiva software (ver- <br />           (200 kV; 4 mA; 1.1 Gy/min dose rate; YXLON International).                        sion 5.0.3; BD). Quantifications performed using WinMDI software <br />                                                                                             (version 2.9; J. Trotter). <br />           Cell survival assay <br />           VH10-SV40 cells transfected 48 h, trypsinized, seeded low             Neutral comet assay <br />           density, exposed  7 d later, cells washed 0.9% NaCl           DSBs measured VH10-TERT cells using Comet Assay           stained methylene blue. Colonies &gt;20 cells scored.               (Trevigen) according manufacturer&#226;&#8364;&#8482;s instructions. Comet tail moments <br />                                                                                             scored using Comet Score software (TriTek). <br />           Microscopy laser microirradiation <br />           Brightfield pictures taken fluorescence microscope (EVOS fl AMG;      Online supplemental material <br />           Westover Scientific) using 4&#195;&#8212; Ph objective. Laser microirradiation    Fig. S1 shows depletion CHD4 does alter expression level           formed confocal microscope (TCS SP5/AOBS; Leica) equipped            DDR proteins  Fig. S2 shows MBD3-GFP rapidly accumulates <br />           environmental chamber set 37&#194;&#176;C 5% <span id='am-21' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span>CO2</span>. Cells grown glass-          sites laser-induced DNA damage. Fig. S3 shows CHD4 depletion <br />           dishes (MatTek) colorless medium containing 10% FCS (Invitrogen),       reduces induced &#238;?&#167;H2AX ubiquitylation CHD4 ATPase activity           Hoechst 33342 added medium concentration 0.5 &#194;&#181;g/ml         required RNF168 IRIF formation. Online supplemental material avail- <br />           laser irradiation. 0.5 &#195;&#8212; 10 &#194;&#181;m DSB-containing tracks generated        able http://www.jcb.org/cgi/content jcb.201001048/DC1. <br />           diode laser (&#238;?&#172; = 405 nm; 30% laser power; 0.75-s irradiation time;         thank Jiri Lukas, Dan Durocher, Adrian Bird, Ody Sibon, Mauro Modesti <br />           2,220 &#194;&#181;W) using UV-transmitting 63&#195;&#8212; 1.4 NA oil immersion objective (HCX         providing reagents. acknowledge Ron Romeijn constructing pEGFP- <br />           PL APO; Leica). Confocal images recorded laser irradi-      C1-MTA2, Antoine Morr&#195;&#169;e technical advice, Karien Wiesmeijer, Annelies <br />           ation 20- 30-s time intervals period 5 12 min              van der Laan, Roeland Dirks, Przemek Krawczyk, Jacob Aten assistance <br />           analyzed using LAS-AF software (Leica). Fluorescence intensities cor-        microlaser irradiation, Jiri Lukas sharing results publication. <br />           rected background normalized prebleach values determine                  work supported Vidi grant Netherlands Organi- <br />           cruitment kinetics.                                                               zation Scientific Research (NWO). <br />           Antibodies <br />                                                                                             Submitted: 11 January 2010 <br />           IRIF Western blot analysis performed using antibodies &#238;?&#167;H2AX <br />           (Millipore), &#238;?&#161;-tubulin (Sigma-Aldrich), ATM (GeneTex), ubiquitin (FK2; Enzo       Accepted: 6 August 2010 <br />           Life Sciences, ), BRCA1 (EMD), histone H3, ATM S1981p, MDC1, MTA2, <br />           CHD4, RNF8 (Abcam), p53 p21 (Santa Cruz Biotechnology, ), <br />           p53 S15p (Cell Signaling Technology), SMC1 (Bethyl Laboratories). <br />                                                                                             References <br />           antibodies RNF168 XRCC4 provided D. Durocher (Uni-             Ahel, D.,  Horejs&#195;&#173;, N. Wiechens, S.E. Polo, E. Garcia-Wilson,  Ahel, H. <br />           versity Toronto, Toronto, Ontario, Canada) M. Modesti (Universite                 Flynn, M. Skehel, S.C. West, S.P. Jackson, et al. 2009. Poly(ADP-ribose)- <br />           d&#226;&#8364;&#8482;Aix-Marseille, Marseille, France), respectively.                                       dependent regulation DNA repair chromatin modeling  <br />                                                                                                    zyme ALC1. Science. 325:1240&#226;&#8364;&#8220;1243. doi:10.1126/science.1177321 <br />           IRIF analysis                                                                     Bekker-Jensen, S., C. Lukas, R. Kitagawa, F. Melander, M.B. Kastan, J. Bartek, <br />           Cells grown glass coverslips treated indicated Figs. 1&#226;&#8364;&#8220;5.              J. Lukas. 2006. Spatial organization mammalian genome sur- <br />           Subsequently, cells washed PBS, fixed 2% <span id='am-67' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span>                 veillance machinery response DNA strand breaks. J. Cell Biol. <br />           0.25% Triton X-100 PBS, incubated appropriate primary                    173:195&#226;&#8364;&#8220;206. doi:10.1083/jcb.200510130 <br />           secondary antibodies (Alexa Fluor 488 [Invitrogen] goat anti&#226;&#8364;&#8220;rabbit IgG        Chai, B., J. Huang, B.R. Cairns, B.C. Laurent. 2005. Distinct roles RSC <br />           conjugated Cy3 [Jackson ImmunoResearch Laboratories, ]) DAPI.                 Swi/Snf ATP-dependent chromatin remodelers DNA double-strand <br />           additional extraction 0.25% Triton X-100 CSK buffer (10 mM                    break repair. Genes Dev. 19:1656&#226;&#8364;&#8220;1661. doi:10.1101/gad.1273105 <br />           Hepes, 300 mM sucrose, 100 mM NaCl, 3 mM MgCl2) performed                 Clapier, C.R., B.R. Cairns. 2009. biology chromatin remodeling com- <br />           fixation incubation primary antibodies RNF168                plexes. Annu. Rev. Biochem. 78:273&#226;&#8364;&#8220;304. doi:10.1146/annurev.biochem <br />           BRCA1. Images recorded microscope (Axioplan; Carl Zeiss, )               .77.062706.153223 <br />           using 63&#195;&#8212; 1.25 NA oil objective camera (AxioCam mRm; Carl Zeiss,          Denslow, S , P  Wade. 2007. human Mi-2/NuRD complex gene <br />           ) analyzed using home Stacks software. Nuclei detected                 regulation. Oncogene. 26:5433&#226;&#8364;&#8220;5438. doi:10.1038/sj.onc.1210611 <br />  <br />  <br />     748   JCB &#226;&#8364;&#162; VOLUME 190 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2010 <br />  Doil, C., N. Mailand, S. Bekker-Jensen, P. Menard, D.H. Larsen, R. Pepperkok,          Pegoraro, G., N. Kubben, U. Wickert, H. G&#195;&#182;hler, K. Hoffmann, T. Misteli. <br />        J. Ellenberg, S. Panier, D. Durocher, J. Bartek, et al. 2009. RNF168                    2009. Ageing-related chromatin defects loss NURD com- <br />        binds amplifies ubiquitin conjugates damaged chromosomes                      plex. Nat. Cell Biol. 11:1261&#226;&#8364;&#8220;1267. doi:10.1038/ncb1971 <br />        allow accumulation repair proteins. Cell. 136:435&#226;&#8364;&#8220;446. doi:10.1016/          Sartori, , C. Lukas, J. Coates, M. Mistrik, S. Fu, J. Bartek, R. Baer, J. Lukas, <br />        j.cell.2008.12.041                                                                      S.P. Jackson. 2007. Human CtIP promotes DNA end resection. <br /> Dubrana, K., H. van Attikum, F. Hediger, S.M. Gasser. 2007. processing                 Nature. 450:509&#226;&#8364;&#8220;514. doi:10.1038/nature06337 <br />       double-strand breaks binding single-strand-binding proteins            Seelig, H.P.,  Moosbrugger, H. Ehrfeld, T. Fink, M. Renz, E. Genth. 1995. <br />       RPA Rad51 modulate formation ATR-kinase foci yeast.                        major dermatomyositis-specific Mi-2 autoantigen presumed <br />       J. Cell Sci. 120:4209&#226;&#8364;&#8220;4220. doi:10.1242/jcs.018366                                       helicase involved transcriptional activation. Arthritis Rheum. 38:1389&#226;&#8364;&#8220; <br /> Gong, , Y.W. Cho, J.E. Kim, K. Ge, J. Chen. 2009. Accumulation Pax2                   1399. doi:10.1002/art.1780381006 <br />       transactivation domain interaction protein (PTIP) sites DNA breaks         Shiloh, Y. 2003. ATM related protein kinases: safeguarding genome integ- <br />       RNF8-dependent pathway required cell survival DNA                       rity. Nat. Rev. Cancer. 3:155&#226;&#8364;&#8220;168. doi:10.1038/nrc1011 <br />       damage. J. Biol. Chem. 11:7284&#226;&#8364;&#8220;7293. doi:10.1074/jbc.M809158200 <br />                                                                                        Shim, E.Y., J.L. Ma, J.H. Oum, Y. Yanez, S.E. Lee. 2005. yeast chro- <br /> Gottschalk, J., G. Timinszky, S.E. Kong, J. Jin, Y. Cai, S.K. Swanson, M.P.                  matin remodeler RSC complex facilitates end joining repair DNA <br />        Washburn, L. Florens, G. Ladurner, J.W. Conaway, R.C. Conaway.                    double-strand breaks. Mol. Cell. Biol. 25:3934&#226;&#8364;&#8220;3944. doi:10.1128/MCB <br />        2009. Poly(ADP-ribosyl)ation directs recruitment activation ATP-              .25.10.3934-3944.2005 <br />        dependent chromatin remodeler. Proc. Natl. Acad. Sci. USA. 106:13770&#226;&#8364;&#8220; <br />        13774. doi:10.1073/pnas.0906920106                                              Shim, E.Y., S.J. Hong, J.H. Oum, Y. Yanez, Y. Zhang, S.E. Lee. 2007. RSC <br />                                                                                                mobilizes nucleosomes improve accessibility repair machinery Grawunder, U., D. Zimmer, S. Fugmann, K. Schwarz, M.R. Lieber. 1998.                       damaged chromatin. Mol. Cell. Biol. 27:1602&#226;&#8364;&#8220;1613. doi:10.1128/ <br />      DNA ligase IV essential V(D)J recombination DNA double-                        MCB.01956-06 <br />      strand break repair human precursor lymphocytes. Mol. Cell. 2:477&#226;&#8364;&#8220; <br />                                                                                        Stewart, G.S., S. Panier, K. Townsend, K. Al-Hakim, N.K. Kolas, E.S. Miller, <br />      484. doi:10.1016/S1097-2765(00)80147-1 <br />                                                                                                S. Nakada, J. Ylanko, S. Olivarius, M. Mendez, et al. 2009. RIDDLE <br /> Hoeijmakers, J.H. 2001. Genome maintenance mechanisms preventing                           syndrome protein mediates ubiquitin-dependent signaling cascade sites <br />       cancer. Nature. 411:366&#226;&#8364;&#8220;374. doi:10.1038/35077232                                        DNA damage. Cell. 136:420&#226;&#8364;&#8220;434. doi:10.1016/j.cell.2008.12.042 <br /> Huang, J., M.S. Huen, H. Kim, C.C. Leung, J.N. Glover, X. Yu, J. Chen.             Stucki, M., J  Clapperton, D. Mohammad, M.B. Yaffe, S.J. Smerdon, S.P. <br />       2009. RAD18 transmits DNA damage signalling elicit homologous                      Jackson. 2005. MDC1 directly binds phosphorylated histone H2AX       combination repair. Nat. Cell Biol. 11:592&#226;&#8364;&#8220;603. doi:10.1038/ncb1865                      regulate cellular responses DNA double-strand breaks. Cell. 123:1213&#226;&#8364;&#8220; <br /> Huen, M.S., R. Grant,  Manke, K. Minn, X. Yu, M.B. Yaffe, J. Chen. 2007. RNF8            1226. doi:10.1016/j.cell.2005.09.038 <br />       transduces DNA-damage signal histone ubiquitylation check&#194;&#173;           Tsukuda, T., B. Fleming, J  Nickoloff, M  Osley. 2005. Chromatin  <br />       point protein assembly. Cell. 131:901&#226;&#8364;&#8220;914. doi:10.1016/j.cell.2007.09.041                modelling DNA double-strand break site Saccharomyces cerevi- <br /> Ikura, T., S. Tashiro,  Kakino, H. Shima, N. Jacob, R. Amunugama, K. Yoder,                  siae. Nature. 438:379&#226;&#8364;&#8220;383. doi:10.1038/nature04148 <br />         S. Izumi,  Kuraoka, K. Tanaka, et al. 2007. DNA damage-dependent             van Attikum, H., S.M. Gasser. 2009. Crosstalk histone modifica- <br />         acetylation ubiquitination H2AX enhances chromatin dynamics.                    tions DNA damage response. Trends Cell Biol. 19:207&#226;&#8364;&#8220;217. <br />         Mol. Cell. Biol. 27:7028&#226;&#8364;&#8220;7040. doi:10.1128/MCB.00579-07                                doi:10.1016/j.tcb.2009.03.001 <br /> Jazayeri, , J. Falck, C. Lukas, J. Bartek, G.C. Smith, J. Lukas, S.P. Jackson.   van Attikum, H., O. Fritsch, B. Hohn, S.M. Gasser. 2004. Recruitment        2006. ATM- cell cycle-dependent regulation ATR response                    INO80 complex H2A phosphorylation links ATP-dependent chroma- <br />        DNA double-strand breaks. Nat. Cell Biol. 8:37&#226;&#8364;&#8220;45. doi:10.1038/ncb1337                  tin remodeling DNA double-strand break repair. Cell. 119:777&#226;&#8364;&#8220;788. <br />                                                                                                doi:10.1016/j.cell.2004.11.033 <br /> Khanna, K.K., S.P. Jackson. 2001. DNA double-strand breaks: signal- <br />      ing, repair cancer connection. Nat. Genet. 27:247&#226;&#8364;&#8220;254. doi:10             van Attikum, H., O. Fritsch, S.M. Gasser. 2007. Distinct roles SWR1      .1038/85798                                                                               INO80 chromatin remodeling complexes chromosomal double-strand <br />                                                                                                breaks. EMBO J. 26:4113&#226;&#8364;&#8220;4125. doi:10.1038/sj.emboj.7601835 <br /> Kolas, N.K., J.R. Chapman, S. Nakada, J. Ylanko, R. Chahwan, F.D. Sweeney, S. <br />        Panier, M. Mendez, J. Wildenhain, T.M. Thomson, et al. 2007. Orches&#194;&#173;            van Haaften, G., R. Romeijn, J. Pothof, W. Koole, L.H. Mullenders,  Pastink, <br />        tration DNA-damage response RNF8 ubiquitin ligase.                        R.H. Plasterk, M. Tijsterman. 2006. Identification conserved path- <br />        Science. 318:1637&#226;&#8364;&#8220;1640. doi:10.1126/science.1150034                                     ways DNA-damage response radiation protection genome-wide <br />                                                                                                RNAi. Curr. Biol. 16:1344&#226;&#8364;&#8220;1350. doi:10.1016/j.cub.2006.05.047 <br /> Larsen, D.H., C. Poinsignon, T. Gudjonsson, C. Dinant, M.R. Payne, F.J. Hari, <br />        J.M.R. Danielsen, P. Menard, J.C. Sand, M. Stucki, et al. 2010.             Wang, B., S.J. Elledge. 2007. Ubc13/Rnf8 ubiquitin ligases control foci  <br />        chromatin-remodeling factor CHD4 coordinates signaling repair                       mation Rap80/Abraxas/Brca1/Brcc36 complex response DNA <br />        DNA damage. J. Cell Biol. 190:731&#226;&#8364;&#8220;740.                                            damage. Proc. Natl. Acad. Sci. USA. 104:20759&#226;&#8364;&#8220;20763. doi:10.1073/pnas <br />                                                                                                .0710061104 <br /> Liang, B., J. Qiu, K. Ratnakumar, B.C. Laurent. 2007. RSC functions        early double-strand-break sensor cell&#226;&#8364;&#8482;s response DNA damage.          Wang, H.B., Y. Zhang. 2001. Mi2, auto-antigen dermatomyositis,        Curr. Biol. 17:1432&#226;&#8364;&#8220;1437. doi:10.1016/j.cub.2007.07.035                                 ATP-dependent nucleosome remodeling factor. Nucleic Acids Res. <br />                                                                                                29:2517&#226;&#8364;&#8220;2521. doi:10.1093/nar/29.12.2517 <br /> Luo, J., F. Su, D. Chen,  Shiloh, W. Gu. 2000. Deacetylation p53 modu- <br />         lates effect cell growth apoptosis. Nature. 408:377&#226;&#8364;&#8220;381. doi:10     Xie, ,  Hartlerode, M. Stucki, S. Odate, N. Puget,  Kwok, G. Nagaraju, C. <br />         .1038/35042612                                                                         Yan, F.W. Alt, J. Chen, et al. 2007. Distinct roles chromatin-associated <br />                                                                                                proteins MDC1 53BP1 mammalian double-strand break repair. <br /> Mailand, N., S. Bekker-Jensen, H. Faustrup, F. Melander, J. Bartek, C. Lukas,                  Mol. Cell. 28:1045&#226;&#8364;&#8220;1057. doi:10.1016/j.molcel.2007.12.005 <br />       J. Lukas. 2007. RNF8 ubiquitylates histones DNA double-strand <br />       breaks promotes assembly repair proteins. Cell. 131:887&#226;&#8364;&#8220;900.              Xu, B., Kim St, M.B. Kastan. 2001. Involvement Brca1 S-phase       doi:10.1016/j.cell.2007.09.040                                                           G(2)-phase checkpoints ionizing irradiation. Mol. Cell. Biol. 21: <br />                                                                                                3445&#226;&#8364;&#8220;3450. doi:10.1128/MCB.21.10.3445-3450.2001 <br /> Morrison, J., J. Highland, N.J. Krogan,  Arbel-Eden, J.F. Greenblatt, J.E. <br />       Haber, X. Shen. 2004. INO80 gamma-H2AX interaction links                 Xue, Y., J. Wong, G.T. Moreno, M.K. Young, J. C&#195;&#180;t&#195;&#169;, W. Wang. 1998. <br />       ATP-dependent chromatin remodeling DNA damage repair. Cell. 119:                      NURD, novel complex ATP-dependent chromatin-remodeling <br />       767&#226;&#8364;&#8220;775. doi:10.1016/j.cell.2004.11.037                                                  histone deacetylase activities. Mol. Cell. 2:851&#226;&#8364;&#8220;861. doi:10.1016/ <br />                                                                                                S1097-2765(00)80299-3 <br /> Moynahan, M.E., J.W. Chiu, B.H. Koller, M. Jasin. 1999. Brca1 controls <br />      homology-directed DNA repair. Mol. Cell. 4:511&#226;&#8364;&#8220;518. doi:10.1016/                  Zhang, Y., H.H. Ng, H. Erdjument-Bromage, P. Tempst,  Bird, D. Reinberg. <br />      S1097-2765(00)80202-6                                                                     1999. Analysis NuRD subunits reveals histone deacetylase core <br />                                                                                                complex connection DNA methylation. Genes Dev. 13:1924&#226;&#8364;&#8220; <br /> Murr, R., J  Loizou, Y.G. Yang, C. Cuenin, H. Li, Q. Wang,  Herceg.                  1935. doi:10.1101/gad.13.15.1924 <br />       2006. Histone acetylation Trrap-Tip60 modulates loading repair <br />                                                                                        Zou, L., S.J. Elledge. 2003. Sensing DNA damage ATRIP recogni- <br />       proteins repair DNA double-strand breaks. Nat. Cell Biol. 8:91&#226;&#8364;&#8220;99. <br />                                                                                             <br /> </body></html>