<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Nmd3p Crm1p-dependent Adapter Protein Nuclear Export Large Ribosomal Subunit <br />  <br /> Abstract. eukaryotic cells, nuclear export nascent                      nucleus coimmunoprecipitated ribosomal subunits nuclear pore complex                           NES-deficient Nmd3p. 60S subunit biogenesis ex- <br /> depends small GTPase Ran.                              port truncated Nmd3p restored addition nuclear export signals (NESs) ribosomal                           exogenous NES. identify export receptor <br /> subunits receptor proteins, recognize                       Nmd3p Nmd3p shuttling 60S ex- <br /> NESs mediate export subunits,                            port blocked Crm1p-specific inhibitor lepto- <br /> identified. showed previously Nmd3p es-                         mycin B. results identify Crm1p receptor <br /> sential protein yeast required late step                   Nmd3p export.  export 60S subunit biogenesis large (60S) ribosomal subunit.                           mediated adapter protein Nmd3p Crm1p-  Nmd3p shuttles deletion                        dependent pathway. NES Nmd3p leads nuclear accumulation mutant protein, inhibition 60S subunit bio-                        Key words: nuclear export &#226;&#8364;&#162; ribosome &#226;&#8364;&#162; Crm1p &#226;&#8364;&#162; <br /> genesis, inhibition nuclear export 60S sub-                     Nmd3p &#226;&#8364;&#162; Saccharomyces cerevisiae <br /> units.  60S subunits accumulate  <br />  <br />  <br /> Introduction <br /> Eukaryotic ribosome biogenesis requires transport                      pore complex mediate vectorial transfer nu- <br /> cargo nucleus nuclear pore                    clear envelope. <br /> complex. Ribosomal proteins, synthesized cyto-                            Ribosomal subunit export major cellular activity plasm, imported nucleus, assem-                      rapidly growing cells nuclear/cytoplasmic transport bly ribosomal subunits nucleolus coordinated                   macromolecules field great   extensive processing rRNAs (Kressler et al.,                      theless, little known nuclear export ribosomal <br /> 1999; Venema Tollervey, 1999). assembled sub-                         subunits. Microinjection experiments Xenopus oocytes <br /> units exported site function                  showed subunit export energy dependent  <br /> cytoplasm. nuclear export events depend                           ceptor mediated (Bataille et al., 1990). yeast Saccha- <br /> small GTPase Ran specific transport receptors                        romyces cerevisiae, mutations affecting function recognize nuclear export signals (NESs)1 cargo                         Gsp1p (Ran) inhibit ribosomal subunit export (Hutchison <br /> molecules  reviews Rout et al., 1997; Mattaj                      et al., 1969; Hurt et al., 1999; Moy Silver, 1999). Muta- <br /> Englmeier, 1998; G&#195;&#182;rlich Kutay, 1999; Nakielny                        tions nucleoporins, structural proteins Dreyfuss, 1999; Pemberton et al., 1998). NESs inte-                    nuclear pore complex, overexpression gral components cargo molecules pro-                         tRNA export factor Los1p (Sarkar Hopper, 1998), <br /> vided trans adapter proteins bridge interac-                   affect ribosomal subunit export (Hurt et al., 1999; <br /> tion cargo receptors (Mattaj Englmeier,                       Moy Silver, 1999). export factors yeast  <br /> 1998). receptors interact nuclear                    clude Crm1p(Xpo1p), required export leucine- <br />                                                                               rich NES-containing proteins, Mex67p, essential <br /> Address correspondence Arlen Johnson, Section Molecular Genet-          mediator mRNA export (Stade et al., 1997; Segref et al., <br /> ics Microbiology, ESB325, University Texas Austin, Austin,      1997; Sarkar Hopper, 1998).  previous work <br /> TX 78712-1095. Tel.: (512) 475-6350. Fax: (512) 471-7088. E-mail:             demonstrated transport proteins  <br /> arlen@mail.utexas.edu                                                         quired ribosomal subunit transport (Hurt et al., 1999; <br />    1 <br />      Abbreviations used paper: aa, <span id='am-6' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span>; DAPI, 4&#226;&#172;&#732;-6&#226;&#172;&#732;-diamino- <br /> dino-2-phenylindole; GFP, green fluorescent protein; NES, nuclear export <br />                                                                               Moy Silver, 1999). addition receptors ri- <br /> signal; NLS, nuclear localization sequence; oligo; oligonucleotide; PKI,      bosomal subunit export, export signals riboso- <br /> <span id='am-18' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span>cAMP</span>-dependent protein kinase inhibitor.                                      mal subunits identified. Furthermore,  <br />  <br /> &#239;&#163;&#169; Rockefeller University Press, 0021-9525/2000/11/1057/10 $5.00 Journal Cell Biology, Volume 151, Number 5, November 27, 2000 1057&#226;&#8364;&#8220;1066 <br /> http://www.jcb.org/cgi/content 151/5/1057                                1057 <br />  known NESs ribosomal subunit export                     PCR products digested BamHI HindIII cloned intrinsic subunits provided trans adapter              sites pTD125 (GFP CEN URA3) make green fluorescent pro- <br />                                                                            tein (GFP)&#226;&#8364;&#8220;NLS GFP&#226;&#8364;&#8220;NLS&#226;&#8364;&#8220;NES fusions, respectively. Gene fusions <br /> proteins.                                                                  PCR express cAMP-dependent protein kinase inhibi- <br />    studying role Nmd3p 60S sub-                     tor(PKI)WT NES PKIP12 NES fused NMD3&#226;&#338;&#172;100. 5&#226;&#172;&#732;-oligos <br /> unit biogenesis S. cerevisiae. showed previously                CGCGAATTCTCCAATGAATTAGCCTTGAAATTAGCAGGTCT- temperature sensitive nmd3 mutant failed accumulate                   TGATATCAACAAGACAATGGAATTTACACCTATAGATCCG- <br />                                                                            CAC CGCGAATTCTCCAATGAATTAGCCTTGAAATTAGC- <br /> 60S subunits nonpermissive temperature (Ho                          AGGTGCTGATATCAACAAGACAATGGAATTTACACCTATAG- <br /> Johnson, 1999). kinetics rRNA pro-                    ATCCGCAC 3&#226;&#172;&#732;-oligo CTGCATCCAGTATACACACCCA- <br /> cessing mutant similar kinetics observed                used amplify NMD3&#226;&#338;&#172;100. resulting PCR products, containing wild-type cells, nascent 25S rRNA ex-                       PKIWT NES PKIP12 NES 5&#226;&#172;&#732;-end NMD3, digested tremely unstable, displaying half-life min-                 EcoRI ligated c-myc&#226;&#8364;&#8220;tagged NMD3&#226;&#338;&#172;100. Plasmid pAJ412, ex- <br />                                                                            pressing c-myc&#226;&#8364;&#8220;tagged length Nmd3p URA3 centromeric vec- <br /> utes. hand, 60S subunits                    tor, ligating NMD3-containing EheI HindIII fragment <br /> fore shift nonpermissive temperature stable                 pAJ408 SmaI HindIII sites pRS416 (Sikorski nonpermissive temperature (Ho Johnson, 1999).                       Hieter, 1989). Plasmid pAJ235 expressing GST-Nmd3p, control interpreted results indicate Nmd3p                    galactose-inducible GAL1 promoter, ligating NMD3- <br />                                                                            containing XhoI HindIII fragment pAJ118 SalI quired late step 60S biogenesis pathway                 HindIII sites pEG(KT) (Mitchell et al., 1993). <br /> initial subunit assembly nucleolus,  <br /> quired maintaining integrity mature cytoplasmic                 Fluorescence Experiments <br /> subunits involved translation. <br />                                                                            Indirect immunofluorescence carried described previously <br />    Nmd3p cosediments sucrose gradients position <br />                                                                            (Heyer et al., 1995), using anti&#226;&#8364;&#8220;c-myc mAb (Covance) Cy3-conju- free 60S subunits. recently, shown                    gated goat anti&#226;&#8364;&#8220;mouse antibody (Amersham Pharmacia Biotech) pri- <br /> Nmd3p binds directly 60S subunits 60S sub-                     mary secondary antibodies, respectively. Overnight cell cultures units coimmunoprecipitated Nmd3p (Ho et                        diluted fresh medium grown 4 h fixation indirect <br /> al., 2000). coimmunoprecipitation specific free                immunofluorescence direct visualization GFP fluorescence. ex- <br />                                                                            periments requiring galactose induction, cells grown overnight 60S subunits; 40S subunits coimmunoprecipitated                    medium containing 2% raffinose. Cultures diluted fresh Nmd3p. pulse-chase&#226;&#8364;&#8220;labeling ribosomal proteins,                    medium containing 2% raffinose grown additional 4 h followed coimmunoprecipitation 60S subunits                     addition galactose 2%. Cells stained 4&#226;&#172;&#732;-6&#226;&#172;&#732;-diaminodino- <br /> Nmd3p, shown nascent 60S subunits                         2-phenylindole (DAPI) grown 40 min presence DAPI                                                                            concentration 2.5 &#226;&#65533;&#174;g/ml. Fluorescence visualized using ZEISS <br /> bound Nmd3p (Ho et al., 2000), consistent                                                                            Axiophot microscope fitted 100&#226;&#171;&#187; objective Princeton Elec- role biogenesis 60S subunit.               tronics MicroMAX CCD camera controlled IPLab Spectrum P <br /> Nmd3p shuttles essential adapter protein                 software package Signal Analytics Corp. Captured images provides NES direct nuclear export nascent                  prepared using Adobe Photoshop&#194;&#174; 5.0. <br /> 60S subunits Crm1p pathway. <br />                                                                            Immunoprecipitation <br />                                                                            50-ml cultures cells grown induced described legend <br /> Materials Methods                                                      Fig. 3. Extracts prepared 20 mM Tris-HCl, pH 7.5, 20 mM <br />                                                                            NaCl, 10% glycerol, 0.1% NP-40, 1 mM PMSF clarified cen- <br />                                                                            trifugation. Extracts preincubated protein beads 30 min <br /> Strains Media                                                          beads removed. Anti&#226;&#8364;&#8220;c-myc mAb added following strains used study: CH1305 (MATa ade2 ade3      samples mixed 4&#226;&#172;&#353;C 2 h. Protein beads added leu2 lys2-801 ura3-52) (Kranz Holm, 1990), MNY7 (MATa CRM1::           samples mixed additional 30 min. beads pelleted KANr leu2&#226;&#171;&#186; his3&#226;&#171;&#186; trp1&#226;&#171;&#186; ura3&#226;&#171;&#186;/ pDC-CRM1), MNY8 (MATa CRM1::             centrifugation, washed times extraction buffer, bound pro- <br /> KANr leu2&#226;&#171;&#186; his3&#226;&#171;&#186; trp1&#226;&#171;&#186; ura3&#226;&#171;&#186;/pDC-CRM1T539C) (Neville Rosbash,          teins eluted heating SDS-PAGE loading buffer. <br /> 1999). Standard yeast genetic methods selective growth media described previously (Kaiser et al., 1994). <br />                                                                            Results <br /> Plasmid Constructs <br />                                                                            Nmd3p essential 59-kD protein (Belk et al., 1999; Ho 5&#226;&#172;&#732; oligonucleotide (oligo) CTAGTCTAGACTCGAGAAAATG- <br />                                                                            Johnson, 1999). NH2-terminal 43 kD Nmd3p CATCATCATCATCATCATTCCATGGAATTCACACCTATAGA- <br /> TCC 3&#226;&#172;&#732;-deletion oligos CGCGGATCCGTTTGTATAGGT-                      conserved Archaea Eukarya, Eubacteria, <br /> TAACACT, CGCGGATCCGAGCCATTCTCTTTAACTT, CGC-                            contains putative Zn2&#226;&#171;&#185; binding motifs similar GGATCCGCACAAGAACTACATCAGG used PCR                            ribosomal proteins (Rivlin et al., <br /> wild-type NMD3 template amplify NMD3&#226;&#338;&#172;50, NMD3&#226;&#338;&#172;100,               1999). eukaryotic members protein family  <br /> NMD3&#226;&#338;&#172;120, respectively. products digested BamHI li- <br /> gated pAJ408 (NMD3::13-myc pRS315 [CEN LEU2]) (Ho et al., <br />                                                                            tain COOH-terminal extensions consisting highly basic <br /> 2000), replaces corresponding fragment wild-type NMD3.        cluster followed hydrophilic acidic domains (Fig. 1 <br /> RPL25 amplified PCR genomic DNA using oligos 5&#226;&#172;&#732;-CGCG-          . Based motifs, constructed series GATCCAAAATGGCTCCATCTGGTAT 5&#226;&#172;&#732;-CGCTCTAGAAAT-                             COOH-terminal deletions Nmd3p. Nmd3&#226;&#338;&#172;50 lacked GTAACCGATTCTGTTAG. resulting product digested                                                                            COOH-terminal acidic 50 amino acid (aa). Nmd3&#226;&#338;&#172;100 <br /> BamHI XbaI ligated sites pTS395 (GAL10::GFP <br /> URA3 CEN). GAL10::NMD3&#226;&#338;&#172;100 replacing HpaI               lacked COOH-terminal 100 aa, retained basic <br /> BglII fragment NMD3&#226;&#338;&#172;100 CEN LEU2 plasmid GAL10::            cluster. Nmd3&#226;&#338;&#172;120 lacked COOH-terminal 120 aa,  <br /> NMD3-containing SmaI BglII fragment pAJ118 (Ho                 cluding 20 aa basic cluster. mutants tagged Johnson, 1999). 5&#226;&#172;&#732;-oligo CGCGGATCCTTAAATATCGATTAT-                     COOH-terminal, 13 tandem copies c-myc <br /> GTGCC 3&#226;&#172;&#732;-oligos GCGAAGCTTATTGAGCTCTTGAATTG- <br /> TAATCT CGCAAGCTTGCGGCCGCTTACTGCTGAGATTCAA- <br />                                                                            epitope (Longtine et al., 1998), expressed low- <br /> CGGGTGT used PCR reactions amplify nuclear localiza-        copy centromeric plasmids NMD3 promoter.  <br /> tion sequence (NLS) NLS plus NES NMD3, respectively.        length Nmd3p containing epitope tag functional <br />  <br />  Journal Cell Biology, Volume 151, 2000                              1058 <br />                                                                                 Figure 1. Phenotype Nmd3p COOH-terminal <br />                                                                                truncation mutants.   diagram Nmd3p <br />                                                                                wild-type deletion mutants showing                                                                                amino-terminal domain conserved archaebac- <br />                                                                                terial eukaryotic proteins (gray) containing <br />                                                                                Zn2&#226;&#171;&#185;-binding motifs (cys-x2-cys), basic <br />                                                                                cluster (aa 399&#226;&#8364;&#8220;419, stippled), acidic <br />                                                                                COOH terminus (aa 476&#226;&#8364;&#8220;518, hatched). NLS                                                                                NES indicate approximate positions NLS                                                                                NES, respectively. Numbers indicate aa positions. <br />                                                                                Complementation assayed plasmid shuffle <br />                                                                                indicates ability replace plasmid- <br />                                                                                borne wild-type NMD3 nmd3::TRP1 disrup- <br />                                                                                tion strain. (b) proteins depicted panel                                                                                tagged c-myc epitope expressed <br />                                                                                centromeric vectors wild-type yeast <br />                                                                                strain CH1305 (Kranz Holm, 1990). Transfor- <br />                                                                                mants streaked single colonies selec- <br />                                                                                tive plate incubated 2 d 30&#226;&#172;&#353;C. <br />  <br />  <br /> (Ho et al., 2000) bound 60S subunits vivo, deter-        examined intracellular localization mined coimmunoprecipitation assay (Ho et al., 2000).       length truncated proteins. Wild-type Nmd3p truncation mutants unable complement          cytoplasmic, reported previously (Ho John- nmd3 null mutation plasmid shuffle assay (Fig. 1 .    son, 1999). Surprisingly, Nmd3&#226;&#338;&#172;50 Nmd3&#226;&#338;&#172;100                                                                 localized nucleus (Fig. 2 b, &#226;&#338;&#172;50 &#226;&#338;&#172;100, respec- <br />                                                                 tively), Nmd3&#226;&#338;&#172;120 cytoplasmic (Fig. 2 b, <br /> Dominant Mutant Nmd3 Proteins Localized              Nmd3&#226;&#338;&#172;120). results suggested COOH ter- <br /> Nucleus Inhibit 60S Subunit Biogenesis                      minus Nmd3p contains NES, nuclear local- <br />                                                                 ization mutant proteins lacking COOH-terminal <br /> Nmd3&#226;&#338;&#172;50 Nmd3&#226;&#338;&#172;100 inhibited growth ex- <br />                                                                 50 100 aa resulted loss NES. Furthermore, <br /> pressed low levels NMD3 wild-type cells (Fig. 1 b). <br />                                                                 nuclear localization mutant proteins indicated <br /> Untagged versions truncated proteins inhibited <br />                                                                 Nmd3p contains NLS. Nmd3&#226;&#338;&#172;120 cy- <br /> growth overexpressed (data shown), consis- <br />                                                                 toplasmic (Fig. 2 b), 20 aa basic cluster, deleted tent earlier observation overexpression                                                                 Nmd3&#226;&#338;&#172;120 (KKLYQRKSKKSRHWKLKRMA), com- <br /> Nmd3p lacking COOH-terminal 100 aa inhibited <br />                                                                 prises NLS Nmd3p  . consid- <br /> growth (Belk et al., 1999). addition                                                                 ered possibility Nmd3p does normally enter <br /> multiple c-myc tag deletion mutants enhanced                                                                 nucleus, Nmd3&#226;&#338;&#172;50 Nmd3&#226;&#338;&#172;100 dominant-negative phenotype, specific                                                                 gain function mutants cryptic NLS  <br /> c-myc epitope, GFP-tagged Nmd3p truncations  <br />                                                                 vealed.  addition strong NLS SV- <br /> haved similarly (data shown). possible                                                                 40 large T antigen length Nmd3p did result COOH-terminal fusions stabilize truncated proteins, <br />                                                                 nuclear accumulation dominant-negative phenotype, <br /> leading increased cellular levels proteins. Ex- <br />                                                                 control protein containing NLS did accu- <br /> pression wild-type Nmd3p Nmd3&#226;&#338;&#172;120 low-copy <br />                                                                 mulate nucleus (data shown). lack nu- <br /> plasmids did adversely affect growth wild-type <br />                                                                 clear accumulation Nmd3p containing SV-40 NLS <br /> cells.  Nmd3&#226;&#338;&#172;120 did complement                                                                 probably reflects rapid export Nmd3p temperature sensitivity nmd3 null mutant, indicating <br />                                                                 NES. Consequently, conclude dominant phe- nonfunctional. <br />                                                                 notype Nmd3&#226;&#338;&#172;50 Nmd3&#226;&#338;&#172;100 gain <br />    Nmd3p required late step 60S biogenesis (Ho <br />                                                                 function cryptic NLS. analysis trun- Johnson, 1999). determine dominant-nega- <br />                                                                 cation mutants, Nmd3p appears contain NLS <br /> tive mutants affected 60S subunit biogenesis, car- <br />                                                                 NES, making capable shuttling. <br /> ried polysome analysis sucrose density gradients. <br /> Wild-type strains expressing dominant-negative alleles <br />                                                                 Dominant Mutant Nmd3 Proteins Inhibit 60S <br /> Nmd3&#226;&#338;&#172;50 Nmd3&#226;&#338;&#172;100 low-copy vectors showed <br />                                                                 Subunit Export <br /> reduced 60S subunit levels (Fig. 2  &#226;&#338;&#172;50 &#226;&#338;&#172;100, respec- <br /> tively) indicate defect 60S subunit biogenesis. Sim-   results far suggested deletion NES <br /> ilar results reported overexpression         Nmd3p led retention truncated Nmd3p tagged Nmd3&#226;&#338;&#172;100 (Belk et al., 1999). Polysomes cells       nucleus, resulting inhibition 60S subunit biogene- <br /> bearing Nmd3&#226;&#338;&#172;120 identical cells         sis. Nmd3p binds directly 60S subunits (Ho et al., <br /> taining vector (data shown), indicating      2000) required late step 60S subunit biogen- Nmd3&#226;&#338;&#172;120 nonfunctional. production        esis, plausible failure export Nmd3p <br /> 60S 40S subunits tightly coordinated (Warner, 1989),     cytoplasm blocked ribosome export nu- specific drop 60S, 40S subunit levels, indi-     cleus.  export itself biogenesis step requir- <br /> cated defect biogenesis large subunit,   ing Nmd3p. assay 60S ribosomal subunit export general downregulation ribosome biogenesis.           yeast described recently utilizes fusion  <br />  <br /> Ho et al. Nmd3p Provides NES 60S Export                  1059 <br />                                                                                Figure 2. Dominant Nmd3p truncation mutants <br />                                                                               inhibit 60S biogenesis localized                                                                               nucleus.   Extracts, prepared cultures                                                                               transformants shown Fig. 1 b, frac- <br />                                                                               tionated ultracentrifugation 7&#226;&#8364;&#8220;47% sucrose <br />                                                                               gradients, described previously (Ho et al., <br />                                                                               2000). positions free 40S 60S subunits <br />                                                                               indicated. relative peak heights free <br />                                                                               subunits sensitive indicator total subunit <br />                                                                               levels   biogenesis defects. (b) Indi- <br />                                                                               rect immunofluorescence c-myc&#226;&#8364;&#8220;tagged <br />                                                                               proteins yeast. Cy3, rhodamine channel                                                                               c-myc&#226;&#8364;&#8220;tagged proteins; DAPI, UV channel indi- <br />                                                                               cating position nuclei. <br />  <br />  <br /> GFP large ribosomal subunit protein L25 (L25&#226;&#8364;&#8220;           temperature shift (data shown). nmd3-4 mutants <br /> GFP) (Hurt et al., 1999). fusion protein functional    affect stability nascent 60S subunits, L25&#226;&#8364;&#8220;GFP incorporated ribosomal subunits, providing       unstable conditions prevent incorporation <br /> means monitoring ribosome localization. reasoned         stable subunits. Nmd3&#226;&#338;&#172;100 trapped nascent 60S subunits nu-          binding Nmd3p 60S subunits direct does <br /> cleus, able observe nuclear retention       require additional proteins bridge interaction <br /> L25&#226;&#8364;&#8220;GFP presence Nmd3&#226;&#338;&#172;100. focused             (Ho et al., 2000). used 60S-binding activity Nmd3&#226;&#338;&#172;100, displayed stronger dominant pheno-        Nmd3p coimmunoprecipitate free 60S subunits bound <br /> type Nmd3&#226;&#338;&#172;50 (data shown). Nmd3&#226;&#338;&#172;100               c-myc&#226;&#8364;&#8220;tagged Nmd3p cell extracts (Ho et al., <br /> L25&#226;&#8364;&#8220;GFP control galactose-induc-         2000). determine L25&#226;&#8364;&#8220;GFP incorporated ible GAL10 promoter introduced plasmids            ribosomal subunits bound Nmd3&#226;&#338;&#172;100, carried wild-type yeast. Cells coexpressing L25&#226;&#8364;&#8220;GFP wild-          immunoprecipitation experiment Nmd3&#226;&#338;&#172;100. <br /> type Nmd3p served control. Protein expression         Extracts prepared cells coexpressing L25&#226;&#8364;&#8220;GFP <br /> induced addition galactose, localization     Nmd3&#226;&#338;&#172;100, c-myc tag. L25&#226;&#8364;&#8220;GFP monitored fluorescence microscopy.           Nmd3&#226;&#338;&#172;100 immunoprecipitated pellet frac- induction Nmd3&#226;&#338;&#172;100 led accumulation           tion analyzed SDS-PAGE Western blot L25&#226;&#8364;&#8220;GFP nucleus (Fig. 3 . contrast, L25&#226;&#8364;&#8220;GFP        presence Nmd3&#226;&#338;&#172;100, L25&#226;&#8364;&#8220;GFP, wild-type retained nucleus evident         60S subunit protein L12. L12 cytoplasm cells expressing wild-type Nmd3p (Fig. 3          L25&#226;&#8364;&#8220;GFP coimmunoprecipitated c-myc&#226;&#8364;&#8220;tagged <br /> c). results demonstrate induction            Nmd3&#226;&#338;&#172;100, extracts containing Nmd3&#226;&#338;&#172;100 <br /> Nmd3&#226;&#338;&#172;100, lacks NES accumu-             epitope tag (Fig. 4 . Coomassie blue staining <br /> lates nucleus, blocked export L25&#226;&#8364;&#8220;GFP. L25&#226;&#8364;&#8220;      gel-separated proteins showed typical profile GFP functional protein incorporated       low molecular weight 60S subunit proteins (Fig. 4 b). 60S subunits (Hurt et al., 1999). Furthermore, ribosomal       identification proteins 60S subunit proteins proteins incorporated subunits     described previously (Ho et al., 2000). results <br /> unstable (Woolford Warner, 1991).  nuclear        provide physical evidence    Nmd3&#226;&#338;&#172;100 binds retention L25&#226;&#8364;&#8220;GFP likely reflects retention       60S subunits, (b) L25&#226;&#8364;&#8220;GFP incorporated riboso- <br /> tire 60S subunit  . support conclusion,   mal subunits, (c) L25&#226;&#8364;&#8220;GFP&#226;&#8364;&#8220;containing subunits L25&#226;&#8364;&#8220;GFP expression induced           trapped nucleus bound Nmd3&#226;&#338;&#172;100. temperature-sensitive nmd3-4 mutant cells shift       Nmd3&#226;&#338;&#172;100 blocked export, binds 60S sub- <br /> nonpermissive temperature, L25&#226;&#8364;&#8220;GFP signal ob-           units, likely nuclear retention 60S sub- <br /> served, L25&#226;&#8364;&#8220;GFP stably incorporated cy-       units physical interaction truncated <br /> toplasmic ribosomes expression induced       Nmd3&#226;&#338;&#172;100 protein trapped nucleus. <br />  <br />  Journal Cell Biology, Volume 151, 2000                  1060 <br />                                          Figure 3. Nmd3&#226;&#338;&#172;100 traps <br />                                         60S ribosomal subunits                                         nucleus. Galactose- <br />                                         inducible L25&#226;&#8364;&#8220;GFP                                         Nmd3&#226;&#338;&#172;100 (13-myc&#226;&#8364;&#8220;tagged) <br />                                         length Nmd3p                                         introduced wild-type <br />                                         cells (CH1305) plasmids. <br />                                         Cultures grown                                         presence raffinose  <br />                                         duced addition ga- <br />                                         <span id='am-1' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Lactose" datatype="xsd:string"></span>lactose</span>. DAPI added                                         time 2.5 &#226;&#65533;&#174;g/ml. 3 h af- <br />                                         ter induction, localiza- <br />                                         tion  c) L25&#226;&#8364;&#8220;GFP                                         (b d) DAPI visual- <br />                                         ized fluorescence micros-      Figure 4. Nmd3&#226;&#338;&#172;100 coimmunoprecipitates 60S subunits contain- <br />                                         copy.  b) Cells coex-       ing L25&#226;&#8364;&#8220;GFP.   Extracts prepared cells coexpressing <br />                                         pressing Nmd3&#226;&#338;&#172;100             L25&#226;&#8364;&#8220;GFP Nmd3&#226;&#338;&#172;100 13-myc (&#226;&#171;&#185;) Nmd3&#226;&#338;&#172;100                                         L25&#226;&#8364;&#8220;GFP. (c d) Cells       13-myc tag (&#226;&#171;&#186;). total protein ex- <br />                                         expressing length Nmd3p      tract used. Immunoprecipitation carried addi- <br />                                         L25&#226;&#8364;&#8220;GFP. Similar           tion anti&#226;&#8364;&#8220;c-myc antibody protein beads, immuno- <br />                                         sults obtained          precipitated samples analyzed SDS-PAGE Western <br />                                         Nmd3&#226;&#338;&#172;100 epitope          blot analyisis. Nmd3, Nmd3&#226;&#338;&#172;100; GFP, L25&#226;&#8364;&#8220;GFP; L12, wild-type <br />                                         tag (data shown).             60S subunit protein L12. (b) Gel-separated proteins panel                                                                           stained Coomassie blue. Nmd3&#226;&#338;&#172;100, immunoglobulin <br />                                                                           heavy light chains (IgG), ribosomal proteins (asterisks) <br /> Nmd3p Shuttles                                                            indicated. identity low molecular weight proteins                                                                           60S subunit proteins confirmed direct comparison pu- test putative transport signals Nmd3p di-              rified 60S subunits SDS-PAGE Western blot analysis <br /> rect cellular localization proteins, fused            multiple antibodies specific 60S proteins (data shown). <br /> putative NLS (aa 387&#226;&#8364;&#8220;435) putative NLS plus NES <br /> (aa 387&#226;&#8364;&#8220;518) GFP. GFP containing putative NLS <br />                                                                           scent 60S subunits bound mutant Nmd3&#226;&#338;&#172;100 (GFP-NLS) predominantly nuclear (Fig. 5), <br />                                                                           trapped nucleus lack export signal. free GFP present cells (data                                                                           Nmd3&#226;&#338;&#172;100 sequesters nascent 60S subunits nu- <br /> shown).  aa 387&#226;&#8364;&#8220;435 sufficient directing                                                                           cleus, able simultaneously restore export <br /> nuclear localization proteins comprise NLS.                                                                           Nmd3&#226;&#338;&#172;100 60S subunit production appending addition putative NES-containing COOH terminus <br />                                                                           heterologous NES Nmd3&#226;&#338;&#172;100. fused leucine- Nmd3p GFP-NLS (GFP&#226;&#171;&#185;NLS&#226;&#171;&#185;NES) relocalized <br />                                                                           rich NES cAMP-dependent protein kinase inhibitor protein cytoplasm (Fig. 5), suggesting                                                                           (PKIWT) (Wen et al., 1995; Stade et al., 1997) NH2 <br /> COOH terminus Nmd3p act NES.                     terminus Nmd3&#226;&#338;&#172;100 (Fig. 6 . used mutant PKI <br /> argued experiments putative                     NES (PKIP12), contained single aa change  <br /> NES Nmd3p simply masked function NLS                        activates NES function (Wen et al., 1995), negative   did act NES.  subse-                    control. Nmd3&#226;&#338;&#172;100 containing PKIWT NES local- <br /> quent experiments showed length Nmd3p                      ized cytoplasm (Fig. 6 b), Nmd3&#226;&#338;&#172;100, GFP&#226;&#171;&#185;NLS&#226;&#171;&#185;NES trapped nucleus                      contained PKIP12 NES remained nucleus (Fig. 6 <br /> Crm1p-dependent fashion  , demonstrating                   b). importantly, Nmd3&#226;&#338;&#172;100, contained func- proteins shuttle contain functional NES.                      tional NES, complemented nmd3-4 temperature-sensi- <br />                                                                           tive mutant (Fig. 7  able replace NMD3 Rescue 60S Biogenesis Addition                             nmd3::TRP1 disruption mutant (data shown). <br /> Heterologous NES Nmd3&#226;&#338;&#172;100                                                 nmd3-4 mutant unable produce 60S sub- dominant inhibition 60S subunit biogenesis                      units nonpermissive temperature (Ho Johnson, <br /> Nmd3&#226;&#338;&#172;100, overexpressing mutant protein,                      1999), complementation mutant suggested likely competition truncated mu-                   Nmd3&#226;&#338;&#172;100, containing functional NES PKI, sup- <br /> tant protein wild-type Nmd3p binding nascent                   ported 60S biogenesis. determine true, 60S subunits nucleus. Nmd3&#226;&#338;&#172;100 accumu-                     examined 60S subunit levels sucrose gradients. lated nucleus, localized concentration greater             incubation nonpermissive temperature 3 h, nmd3-4 wild-type Nmd3p, nucleus                cells containing vector displayed significant  <br /> transiently, allowing efficient competition                 crease 60S subunit levels compared cells bearing overexpression. (Nmd3&#226;&#338;&#172;120 dominant nega-                      wild-type copy NMD3 (Fig. 7 b, vector compared tive, presumably lacked NLS NES                   NMD3). 60S subunit defect enhanced longer enter nucleus.) surmised na-                    incubation times resulted arrest cell growth <br />  <br />  <br /> Ho et al. Nmd3p Provides NES 60S Export                            1061 <br />  Figure 5. Localization GFP directed NLS NES Nmd3p. putative NLS Nmd3p (aa 387&#226;&#8364;&#8220;435) NLS <br /> plus NES (aa 387&#226;&#8364;&#8220;518) fused GFP introduced wild-type cells (CH1305). localization GFP fusion   Figure 6. addition functional NES Nmd3&#226;&#338;&#172;100 relo- <br /> proteins determined direct fluorescence (GFP), DNA       calizes protein cytoplasm.   cartoon shown stained DAPI living cells. multiple small DAPI-     addition heterologous NES NH 2 terminus stained spots mitochondrial DNA.                                Nmd3&#226;&#338;&#172;100. (b) fusion proteins expressed centro- <br />                                                                     meric plasmids wild-type strain (CH1305). localization <br />                                                                     proteins determined indirect immunofluorescence <br />                                                                     (Cy3), DNA stained DAPI. <br /> (data shown). shown previously drop 60S subunit levels rapid turnover nascent subunits result instability mature        export receptor Crm1p (Fornerod et al., 1997; <br /> subunits (Ho Johnson, 1999). contrast, cells             Stade et al., 1997). prompted examine  <br /> taining PKIWT&#226;&#171;&#186;Nmd3&#226;&#338;&#172;100 supported 60S biogenesis (Fig.               pendence Nmd3p shuttling Crm1p. ex- <br /> 7 b, PKIWT), indicated increased ratio           pressed c-myc&#226;&#8364;&#8220;tagged length Nmd3p GFP- <br /> 60S 40S absence halfmers compared                reporter protein fused NLS NES Nmd3p vector  elevated level free 40S conse-         temperature-sensitive crm1(xpo1-1) mutant. quence depressed 60S levels typical mutants            unable detect nuclear accumulation proteins defects 60S biogenesis. 60S subunit levels         strain nonpermissive temperature (data shown). slightly lower cells expressing PKIWT&#226;&#171;&#186;Nmd3&#226;&#338;&#172;100                 temperature shifts global effects cell <br /> compared wild-type, polysome profile stable           growth rise transient inhibition ribosome time nonpermissive temperature (Fig. 7; data            biogenesis (Warner, 1999), decided examine Nmd3p shown). Cells containing PKIWT&#226;&#171;&#186;Nmd3&#226;&#338;&#172;100 showed                  localization conditions Crm1p specifi- slight growth defect compared wild-type cells           cally inhibited. Leptomycin B antibiotic inhibitor (data shown), consistent slightly lower         Crm1p eukaryotic cells (Nishi et al., 1994; Kudo et <br /> 60S levels PKIWT&#226;&#171;&#186;Nmd3&#226;&#338;&#172;100&#226;&#8364;&#8220;containing cells                   al., 1998). wild-type S. cerevisiae sensitive <br /> compared wild-type. simultaneous restoration            leptomycin B, single aa change S. cerevisiae <br /> 60S subunit biogenesis cytoplasmic relocalization            Crm1p (threonine 359 cysteine) renders cells sensitive <br /> Nmd3p, addition functional NES Nmd3                  leptomycin B (Neville Rosbash, 1999).  <br /> &#226;&#338;&#172;100, strongly suggests Nmd3p mediates export               troduced plasmids expressing c-myc&#226;&#8364;&#8220;tagged length <br /> providing NES nascent 60S subunits. Nmd3&#226;&#338;&#172;100 did             Nmd3p GFP containing NLS NES Nmd3p <br /> assemble nascent 60S subunits, able              leptomycin-sensitive cells. Cultures cells coimmunoprecipitate subunits Nmd3&#226;&#338;&#172;100.                treated leptomycin B, localization Nmd3p  addition functional NES required              GFP monitored fluorescence microscopy. restore function truncated protein, indicating       addition leptomycin B led accumulation primary deficiency mutant Nmd3&#226;&#338;&#172;100 protein               Nmd3p nucleus majority cells lack export signal.                                   leptomycin B&#226;&#8364;&#8220;sensitive strain (Fig. 8 b). Nmd3p remained <br />                                                                     cytoplasmic wild-type leptomycin B&#226;&#8364;&#8220;resistant cells (Fig. <br /> Nmd3p Export Requires Crm1p                                         8  leptomycin B&#226;&#8364;&#8220;sensitive strain ab- COOH-terminal 50 aa Nmd3p, aa 491&#226;&#8364;&#8220;500                 sence antibiotic (Fig. 8 c). nuclear retention (INIDELLDEL) highly conserved predicted                 Nmd3p observed 15 min addition lep- form amphipathic helix isoleucine leucine            tomycin B persisted 1 h (data predominantly face. structure characteris-         shown). Similar results obtained GFP&#226;&#171;&#185; <br /> tic leucine-rich NES (Rittinger et al., 1999), ligand      NLS&#226;&#171;&#185;NES reporter (Fig. 9). GFP reporter  <br />  <br />  Journal Cell Biology, Volume 151, 2000                       1062 <br />  Figure 7. Rescue Nmd3&#226;&#338;&#172;100 function heterologous <br /> NES.   Plasmids expressing wild-type Nmd3p (NMD3), pro- <br /> tein (vector), Nmd3&#226;&#338;&#172;100 containing PKIWT mutant <br /> PKIP12 NES introduced temperature-sensitive <br /> nmd3-4 mutant. Transformants patched selective  <br /> dium grown 2 d 37&#226;&#172;&#353;C. Similar results obtained nmd3::TRP1 disruption strain (data shown). domi- <br /> nant-negative effect Nmd3&#226;&#338;&#172;100 containing mutant NES <br /> (PKIP12) severely impaired cell growth permissive tem- <br /> perature, leading phenotype severe nmd3-4       Figure 8. Nmd3p localization sensitive Crm1p inhibitor <br /> cells containing vector. (b) Sucrose gradient analysis        leptomycin B.  d) wild-type strain MNY7 (CRM1) performed. Cultures transformants shown panel          (b, c, e, f) leptomycin-sensitive strain MNY8 grown permissive temperature, shifted 37&#226;&#172;&#353;C 3 h,         (CRM1T539C) transformed plasmid pAJ412 expressing status ribosomal subunits polysomes analyzed        c-myc&#226;&#8364;&#8220;tagged Nmd3p. Cells grown early log phase sucrose gradient sedimentation, described previously (Ho         treated   b, d, e) (&#226;&#171;&#185;LMB) leptomycin B (0.1 &#226;&#65533;&#174;g/ml) et al., 2000). Nmd3&#226;&#338;&#172;100 containing mutant NES (PKI P12)        (c f) controls (&#226;&#171;&#186;LMB) leptomycin B incubated analyzed severe growth defect. positions         15 min. Cells fixed prepared indirect im- <br /> free 40S, free 60S, halfmers (polysomes containing           munofluorescence. Cy3, rhodamine channel c-myc&#226;&#8364;&#8220;tagged <br /> joined 40S subunit initiation site resulting deficit   Nmd3p; DAPI, UV channel DNA. localization Nmd3p free 60S subunits) shown.                                       CRM1 cells, absence leptomycin B, similar                                                                        CRM1 cells presence leptomycin B (data shown). <br />  <br /> tained shuttling signals Nmd3p, nuclear  <br /> tention protein likely reflects direct interac- <br /> tion NES Nmd3p Crm1p                     strong nuclear accumulation L25&#226;&#8364;&#8220;GFP leptomy- <br /> indirect effect mediated Crm1p-dependent                    cin-sensitive cells treated leptomycin B (Fig. 10, <br /> protein.  inhibition Crm1p addition lep-             CRM1T539C). experiment, detect signal <br /> tomycin B, temperature shift, resulted nu-             L25&#226;&#8364;&#8220;GFP nucleus 2 h (data shown), <br /> clear accumulation Nmd3p. results suggest                signal considerably intense 4 h. <br /> Crm1p export receptor Nmd3p.                                leptomycin-insensitive strain, L25&#226;&#8364;&#8220;GFP accumu- <br />    Nmd3p Crm1p-dependent adapter 60S subunit               lated cytoplasm evident nucleus <br /> export, 60S subunit export itself dependent               (Fig. 10, CRM1). L25&#226;&#8364;&#8220;GFP accumulated cytoplasm <br /> Crm1p. 60S subunit export reported                   strains absence leptomycin B (data sensitive temperature shifts temperature-sensi-              shown).  like Nmd3p export, export 60S sub- <br /> tive crm1(xpo1-1) mutant (Hurt et al., 1999).                  units blocked leptomycin B&#226;&#8364;&#8220;sensitive manner. <br /> Nmd3p retained nucleus              nuclear retention L25&#226;&#8364;&#8220;GFP Crm1p-dependent <br /> ditions, trapped nucleus treatment              manner suggests 60S subunit export itself mediated Crm1p inhibitor leptomycin B. Consequently, ex-                Crm1p. Taken  results provide compel- <br /> amined localization 60S subunits presence             ling evidence Nmd3p acts Crm1p-dependent <br /> leptomycin B. experiment simultaneously                     adapter export 60S ribosomal subunit. <br /> treated cells leptomycin B induced expres- <br /> sion L25&#226;&#8364;&#8220;GFP monitor localization nascent-                Discussion <br /> 60S subunits. L25&#226;&#8364;&#8220;GFP incorporated nascent sub- <br /> units  , giving rise functional 60S subunits              delineation Nmd3p- Crm1p-dependent ex- <br /> (Hurt et al., 1999). 4 h treatment, observed             port pathway 60S ribosomal subunit  <br />  <br />  <br /> Ho et al. Nmd3p Provides NES 60S Export                         1063 <br />                                                                       Figure 10. L25&#226;&#8364;&#8220;GFP retained nucleus presence                                                                      leptomycin B.   leptomycin-sensitive strain MNY8 <br />                                                                      (CRM1T539C) (b) wild-type strain MNY7 (CRM1)                                                                      transformed plasmid pAJ369, expresses L25&#226;&#8364;&#8220;GFP. <br /> Figure 9. GFP containing NLS NES Nmd3p              Overnight cultures grown medium lacking <span id='am-13' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span>, raffi- <br /> stricted nucleus, presence leptomycin B.          nose carbon source, diluted medium <br /> MNY7 (CRM1) (b c) leptomycin-sensitive strain            grown 4 h 30&#226;&#172;&#353;C. Leptomycin B galactose MNY8 (CRM1T539C) cells transformed plasmid                 added 0.1 &#226;&#65533;&#174;g/ml 1%, respectively. 4 h incubation <br /> pAJ629 expressing galactose-inducible GFP fused NLS           30&#226;&#172;&#353;C, GFP fluorescence visualized. absence plus NES Nmd3p (aa 387&#226;&#8364;&#8220;518). Cells grown early log        added leptomycin B, L25&#226;&#8364;&#8220;GFP cytoplasmic strains <br /> phase galactose added final concentration 1%      (data shown). <br /> induce GFP expression. 1 h induction, leptomycin B added 0.1 &#226;&#65533;&#174;g/ml (&#226;&#171;&#185;LMB). (c) controls, leptomycin B omitted (&#226;&#171;&#186;LMB). GFP visualized direct fluorescence mi-        eukaryotes (Johnson, , unpublished results). <br /> croscopy 1 h addition leptomycin B. GFP localization <br />                                                                      Interestingly, related proteins predicted archaebac- nucleus evident 15 min addition lepto- <br /> mycin (data shown).                                              teria   archaebacterial proteins lack <br />                                                                      shuttling sequences defined Nmd3p. <br />                                                                       eukaryotic Nmd3 proteins appear evolved <br /> port nuclear export pathway ribosomal subunits.             addition NLS NES, adapting NES Nmd3p essential 60S subunit                archaea-like protein nuclear shuttling. <br /> biogenesis export, Nmd3p appears principal                COOH-terminal 50 aa Nmd3p (aa 469&#226;&#8364;&#8220;518) <br /> protein providing export signal large ribosomal          necessary nuclear export, domain protein <br /> subunit. Nmd3p additional role               likely contains NES. region, aa 491&#226;&#8364;&#8220;500 60S subunit  , ability modulate 60S             (INIDELLDEL) highly conserved predicted <br /> subunit export presence absence NES               form amphipathic helix isoleucine leucine <br /> Nmd3p clearly demonstrates essential function               predominantly face, similar leucine-rich NES Nmd3p provide NES 60S export.                  (Rittinger et al., 1999). export Nmd3p depends <br /> Nmd3p acts adapter protein bridge interaction           Crm1p, receptor leucine-rich NES&#226;&#8364;&#8220;contain- 60S subunit export receptors.            ing proteins, tentatively conclude aa 491&#226;&#8364;&#8220;500 com-  demonstration Crm1p receptor                 prise NES Nmd3p. Experiments determine Nmd3p mediate 60S subunit transport evi-             minimal NES Nmd3p underway. note dence Crm1p involved ribosomal subunit export.            Nmd3&#226;&#338;&#172;100 displayed stronger dominant-negative phe- <br />    Nmd3p highly conserved protein. Similar proteins             notype Nmd3&#226;&#338;&#172;50. larger deletion Nmd3&#226;&#338;&#172;100 eukaryotes eukaryotic            encompassed aa 419&#226;&#8364;&#8220;468, contains additional <br /> proteins high degree conservation shut-             highly conserved domain. Preliminary results indicate tling signals identified Nmd3p.               region necessary function, act addi- recently cloned human homologue, CGI-07,                tively aa 469&#226;&#8364;&#8220;518 (Johnson, , unpublished results). complements temperature-sensitive nmd3               region encode second, weaker NES, mutant, suggesting conservation function Nmd3p                 signal intranuclear localization. Consequently, pos- <br />  <br />  Journal Cell Biology, Volume 151, 2000                        1064 <br />  sible redundancy export signal pos-    Johnson, unpublished results),  NMD3 sibly export pathway. Determination intranu-      high-copy suppressor mex67-5. high-copy <br /> clear localization mutant Nmd3 proteins deleted          MEX67 PAB1 partially suppressed growth defect various signals elucidate respective     nmd3-1 cells, did reverse 60S subunit defi- <br /> tribution Nmd3p localization.                                cit. Consequently, MEX67 PAB1 unlikely    Leptomycin B antibiotic specific Crm1p          directly involved 60S export. possible inhibi- <br /> nearly eukaryotic cells (Nishi et al., 1994; Kudo et al.,   tion 60S export indirectly affects export mRNA lead- <br /> 1999). Wild-type S. cerevisiae cells resistant lepto-    ing condition mRNA partially limiting mycin B.  single aa change Crm1p ren-          cells. link mRNA transport nucleolus, <br /> ders S. cerevisiae sensitive antibiotic (Neville     site ribosome biogenesis, suggested previ- <br /> Rosbash, 1999). Nmd3p export inhibited lep-        ously (Schneiter et al., 1995). possible overexpres- <br /> tomycin B, conclude Crm1p receptor           sion MEX67 partially bypasses block, Nmd3p. preliminary vitro experiments,        overexpression PAB1 stabilize mRNAs (Caponi- <br /> observed Ran binding presence Crm1p          gro Parker, 1995) conditions mRNA Nmd3p (Kallstrom, G.,  Johnson, unpublished            limiting cell. work needed determine <br /> sults), suggests cooperative interaction Ran       basis genetic interactions. Crm1p formation export complex   <br /> nerod et al., 1997; Kutay et al., 1998). Furthermore,        Fundamental Function Nmd3p? <br /> showed Crm1p needed efficient 60S subunit ex-       Eukaryotes appear adapted archaeal Nmd3p- <br /> port. contrary previous report temper-    like protein transporting 60S subunit nu- <br /> ature-sensitive crm1(xpo1-1) mutant cells did inhibit       clear envelope. presence Nmd3p-like proteins 60S subunit export shifted restrictive temperature      archaebacteria, lack nuclei, suggests Nmd3p <br /> (Hurt et al., 1999). Nmd3p did ac-       additional function ancient nuclear ex- <br /> cumulate nucleus crm1(xpo1-1) cells restrictive    port. role biogenesis step large <br /> temperature. Preliminary results suggest failure      ribosomal subunit distinct  cou- observe Nmd3p accumulation nucleus            pled  export subunit. requirement Nmd3p inhibition Nmd3p import restrictive tem-       ultimate maturation step, export, pro- <br /> perature (Ho, J.,  Johnson, unpublished results).         vide mechanism control 60S export.  transient inhibition ribosome biogenesis          Nmd3p provide quality control mechanism ri- <br /> temperature shifts (Warner, 1999) likely complicates        bosomal subunit biogenesis analogous role nu- <br /> use crm1(xpo1-1) mutants examining effects ri-        clear aminoacylation tRNAs required tRNA export <br /> bosome export. possible alternative      (Lund Dahlberg, 1998). note truncated <br /> Crm1p-independent pathway acts elevated temperature          Nmd3p, lacking export signal, binds nuclear 60S sub- bypass Crm1p-dependent pathway.             units, sufficiently stable accumulate nu- did observe strong nuclear accumulation Nmd3p           cleus.  temperature-sensitive nmd3 mutant 60S subunits leptomycin B&#226;&#8364;&#8220;sensitive cells           does allow nuclear accumulation nascent 60S <br /> treated leptomycin B.  Crm1p export          subunits severe instability.  Nmd3p ceptor Nmd3p mediate 60S subunit export.                 provides function subunit biogenesis necessary <br />    shuttling Nmd3p nucleus          stabilization nascent 60S subunit. suggest <br /> pends recognition import export signals     eukaryotes adapted ribosome biogenesis factor <br /> ceptor proteins. Nmd3p binds 60S subunits        transport large ribosomal subunit. <br /> nucleus, NES displayed recognition Crm1p. Nmd3p binds mature 60S subunits cyto-         <br /> plasm (Ho Johnson, 1999; Ho et al., 2000). Conse-           <br /> quently, NLS Nmd3p bound 60S subunits cy-      <br /> toplasm masked prevent retrograde transport     <br /> mature subunits nucleus. similar proposal   ribosomal proteins (Rout et al., 1997).      <br /> Nmd3p predominantly cytoplasmic, binds ma- <br /> ture free 60S subunits (Ho et al., 2000), ratio Nmd3p   free 60S subunits cytoplasm determine        <br /> availability Nmd3p shuttling nucleus.         <br />  <br />  <br /> Does Inhibition 60S Subunit Export Affect              <br /> Transport Pathways?                                         screen high-copy suppressors growth defect         nmd3-1 mutant (Ho Johnson, 1999) identi-          <br /> fied MEX67, encoding mRNA transport factor (Segref         <br /> et al., 1997; Hurt et al., 2000), PAB1, encoding              <br /> poly  binding protein (Kallstrom, G.,  Johnson,        <br /> unpublished results). addition, mex67-5 nmd3-1             <br /> mutations synthetic lethal (Kallstrom, G.,         <br /> </body></html>