<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTEIUOLOGY, Apr. 1976, p. 38-47                                                     Vol. 126,  1 <br /> Copyright &#194;&#169;) 1976 American Society Microbiology                                           Printed U.S  <br />  <br />  <br />   Method Isolation Escherichia coli K-12 Mutants <br />                 Deficient Essential Genes <br />                        KAREN  ARMSTRONG&apos;* ROBERT K. HERMAN <br />       Department Genetics Cell Biology, University Minnesota, St. Paul, Minnesota 55108 <br />                                   Received publication 6 November 1975 <br />  <br />              developed general procedure induction identification           mutations chromosomal essential genes located diploid region           Escherichia coli K-12. partial diploidy conferred episome           temperature sensitive replication mutant strain form microco- <br />           lonies 42 C complete media essential chromosomal gene diploid <br />           region defective. Mutations identified procedure classified           cistrons complementation method devised purpose. verify           procedure works practice, fused episome covering rif region <br />           Fjlac+ used resulting temperature-sensitive episome iden- <br />           tify chromosomal mutations essential functions near rif. expected,           certain proportion mutations rif gene, essential gene           codes ,3 subunit ribonucleic acid polymerase. <br />  <br />    Essentials genes Escherichia coli               suppressors general sup- functions essential growth                 press low efficiency, suppressor complete broth media (5, 22).                tRNA insert <span id='am-38' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-42' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> unable obvious importance,             confer proper function protein product. produce analyze mutations essential                  procedure Beckman Cooper <br /> genes. haploid organisms, common                  permit isolation mutations <br /> method isolating mutants affected essen-               essential genes. <br /> tial genes involved use conditionally                 Murgola Adelberg (27) used streptomy- <br /> lethal mutations. conditionally lethal                   cin suppression isolate mutations essen- <br /> strain, mutant gene product functions                tial genes E. coli. Streptomycin causes mis- certain (permissive) conditions               reading nonspecific suppression active (nonpermissive) circum-                    nonsense codons (31, 38).  isolates <br /> stances. likely,  genes               lesion essential gene grew easily mutated form                  complete medium streptomycin code product functions                  added complete medium experimentally desirable conditions.                   streptomycin. mutants defi- <br /> stance, Edgar Lielausis (11)                cient syntheses protein, <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span>, large sample temperature-sensitive mu-                   deoxyribonucleic acid (DNA) isolated, <br /> tants bacteriophage T4D, mutations                grew poorly permissive distributed randomly: genes mu-                 condition partial nature tated times, mutated                     phenotypic suppression.  suggests                       avoid problems inherent studies genes conditionally lethal muta-                discussed  used partially diploid <br /> tions induced. Furthermore,                    strain E. coli K-12 isolate essential gene conditionally lethal mutations ob-                   mutants. F&apos; episome carries du- <br /> tained, low levels activity gene product            plicate set genes, functions essential permissive conditions obscure                  chromosomal genes diploid region analyses (15).                                                completely abolished loss viability    Beckman Cooper (5) used strain E.                 mutant strain. Partial diploids coli carrying temperature-sensitive suppres-                used, example, obtain recessive lethal <br /> sor transfer ribonucleic acid (tRNA) (14) ob-              mutations ribosomal protein genes (28) tain essential gene mutations chromo-                  gene, called rif (3), specifies /- <br />      Present address: Department Biology, University                                                               polypeptide RNA polymerase (17, 30).  California, San Diego, La Jolla, Calif. 92093. (Send reprint cases, known phenotypic responses  requests University Minnesota address.)            particular drugs used select                                                        38 <br />  VOL. 126, 1975                                            MUTATIONS ESSENTIAL GENES                     39 <br /> desired mutants. proposing gen-           Plates complete media (lac-tet Pen- <br /> eral approach wild-type alleles         assay agar) stored dark use episome removed unmask            avoid lethality RecA- strains (36).  recessive lethal mutation.                          quired, media supplemented L-amino <br />     chose use F&apos; factor     acids, purines, pyrimidines, 20 ,tg/ml. <br />                                                         exception <span id='am-33' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Thymine" datatype="xsd:string"></span>thymine</span>, added <br /> temperature sensitive replication.          50 ,ug/ml strains requiring high concentra- <br /> Cells mutations essential genes          tions thymine. Rifampin streptomycin chromosome dependent              added sterile media final concentrations 100 <br /> continued presence F&apos; factor growth.         250 ,ug/ml, respectively. media containing 28 C cells grow normally         rifampin stored dark. <br /> temperature-sensitive episome replicates effi-              Chemicals. ICR-191, {3-chloro-7-methoxy-9-(3- <br /> ciently.  episomal replication          [chloroethyllamino propylamino) acridine dihydro- <br /> inhibited incubation 42 C, mutants            chloride} generous gift H. J. Creech. grow poorly form microcolonies.          Rifampin (B grade, Calbiochem) rifamycin SV <br />                                                         sodium salt (Schwarz/Mann) used parts <br /> Nonmutant bacteria grow normally              work. results obtained both temperatures, F&apos; factor             drugs  designation rifampin maintained 28 C lost 42 C. micro-         used text apply antibiotic. N- <br /> colonies formed mutant strains 42 C           methyl-N&apos;-nitro-N-nitrosoguanidine (NG) ob- analogous minute colonies pro-         tained Aldrich <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> , trimeth- <br /> duced unilinear transmission bac-          oprim Sigma Chemical  <br /> teriophage abortive transduction (29; B.               Nomenclature. Abbreviations symbols ge- <br /> D. Stocker, Heredity, 9:290-291, 1955).                  netic markers accordance proposed <br />     determine method mutant           Demerec et al. (10) used Taylor Trot- <br />                                                          ter (35). ess used signify mutation isolation successful, selected F&apos;         essential gene. Alleles rif gene denoted factor carried known essential gene,           convention used Austin Scaife (3): rif certain chromosomal mu-                confers sensitivity drug rifampin, rif confers <br /> tants defective essential gene          resistance drug, rif&#194;&#176; results complete <br />  recovered. chose F&apos; factor             loss rif gene function. <br />  F110 carries essential gene rif.             Bacterial strains. bacterial strains used  Strains deficient rif gene character-       study listed Tables 1 2. <br />  ized previously (4) known rir mu-                Mutagenesis procedures. construction  tants. rif&quot; mutants inviable haploids,         various strains (Table 1), auxotrophs induced <br />                                                          treating exponentially growing cultures NG <br />  characteristic expected essential mutant.         described Hoess Herman (20). muta- <br />     make F110 temperature sensitive,           genesis, cells collected centrifugation, <br />  fused vivo character-           washed twice M63 buffer-salts, suspended <br />  ized temperature-sensitive episome, Fu52lac+            glucose minimal medium. culture <br />  (8). mutagenized strain ICR-191,          reached stationary phase, penicillin selection (16) <br />  frameshift mutagen usually produces mu-            performed enrich auxotrophs. <br />  tations virtually abolish gene function (1,            Mutations essential genes induced  32). isolated mutants defective chro-        treating exponential culture strain KAF1/ <br />  mosomal essential genes covered tem-             KA17 (Table 2) ICR-191 (19) 4 8 h                                                          <span id='am-16' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span>lactose</span>-threonine-tryptophan-methionine medium <br />  perature-sensitive episome. predicted,           28 C using 0, 10, 15, 20 jig ICR-191/ml. <br />  mutants deficient chromo-              4 8 h, cells collected washed  somal rif gene function.                                 NG mutagenesis suspended M63 <br />     developed method transfer            buffer-salts. resuspended cells diluted <br />  chromosomal mutant rif allele muta-           1:100 lactose-threonine-tryptophan-methionine <br />  tions essential genes F&apos; factor          medium incubated shaking 28 C  combination. procedure important             stationary phase reached. <br />  cause allowed perform complementation            efficiency ICR-191 mutagenesis  tests classify mutations cistrons.          monitored examining number colonies                                                          strain KAF1/KA17 grew minimal plates <br />                                                          containing rifampin. purpose, station- <br />            MATERIALS METHODS                         ary cultures segregated cells diluted 500-fold <br />    Media. M63 minimal liquid medium agar,            plated incubated 28 C. cul- <br />  lactose-tetrazolium agar (lac-tet), Penassay        tures showed highest frequency rifampin- <br />  broth described previously (18, 19). Penassay      resistant colonies varied somewhat ex- <br />  agar contains 25.5 g Antibiotic medium 2 (Difco)     periment experiment - screened mutants <br />  5 mg thiamine liter. Dilution medium,        deficient essential genes. <br />  M63 buffer-salts, contained following liter:       Thy- mutants. Spontaneous Thy- mutants  KH2PO4, 5.3 g; K2HPO4 3H20, 13.93 g; (NH4)2SO4,         selected using trimethoprim described  2.0 g; MgSQ4-7H20, 0.2 g; FeSO4 7H20, 0.5 mg.       Hoess Herman (20). <br />  40        ARMSTRONG HERMAN                                                                        J. BACTERIOL. <br />                                             TABLE 1. Bacterial strains <br />          Strain             Sex                    Genotype&quot;                                   Derivation <br /> Ft,62jac+I20OPS            F&apos;       F.,26ac+/lacZ+, lacY-                       F. Jacob (8) <br /> Ft,2ac+/KA9                F&apos;       Ft.621ac+largE, met, pur, str, thy+,    FU62Jac+/200PS     x KA9 <br />                                       pro, lac, recAl <br /> JC5029                     Hfr      ilv-38, thr-300, recA+                      CGSCb <br /> JC5029 Thy-                Hfr      ilu-38, thr-300, recA+, thy                 Spontaneous mutant JC5029 <br /> JC5029 Thy- Leu-           Hfr      ilv-38, thr-300, recA+, thy, leu            NG-induced mutant JC5029 <br />                                                                                   Thy- <br /> KAlc                       F-    argE, met                                      NG-induced mutant 254 <br /> KA2c                       F-    argE, met, str                                 Spontaneous mutant KA1 <br /> KA3C                       F-    argE, met, str, thy                            Spontaneous mutant KA2 <br /> KASC                       F-    argE, met, pur, str, thy,  pro, lac        NG-induced mutant KA3 <br /> KA9c                       F-    argE, met, pur, str, thy+,  pro, lac,      KA8 x KL16-99 Thy- <br />                                     recAl <br /> KA12c                   F-       argE, met, pur, lac                       NG-induced mutant KA1 <br /> KA13r                   F-       argE, met, pur, lac, thy                  Spontaneoqs mutant KA12 <br /> KA14c                   F-       argE, met, pur, lac, thy+, recAl          KA13 x KL16-99 Thy- <br /> KA14 RiF c              F-       argE, met, pur, lac, recAl, rif&apos;          Spontaneous mutant KA14 <br /> KA14 Thy-c              F-       argE, met, pur, lac, recAl, thy           Spontaneous mutant KA14 <br /> KA16c                   F-       argE, met, pur, lac, thy, thr, trp        NG-induced mutant KA13 <br /> KA17c                   F-       argE, met, pur, lac, thy+, thr, trp, KA16 x KL16-99 Thy <br />                                     recAl <br /> KL16-99 Thy-            Hfr      thy, recAl                                J.  Clark (24) <br /> KLF10/JC1553            F&apos;       FilO metB+IargG6, metBi, 1,            CGSC<span id='am-21' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; <br />                                     leu-6, recAl, str-104, malB16 <br /> KLF23/KL181             F&apos;        F123 trp+IpyrD34, 68, trp-45,        &apos;CGSCb <br />                                     recAl <br /> 254                     F-       argE                                       P. Rogers (12) <br /> 254Rifr                  F-      argE, rif&apos;                                 Spontaneous mutant 254 <br />   genetic-markers relevant current study listed. <br />   b*E. coli Genetic Stock Center, Department Human Genetics, Yale University, New Haven, Conn. <br />     met pur markers strains selected complemented episome F110. <br />                             TABLE 2. Construction Ft,6Aact+-FIO rif met (KAFJ) <br />               Strain               Sex                    Genotype&quot;                              Derivation <br /> F110 rifr/254 Rifr                  F&apos;    F110 argE+, rifr/argE, rifr               KLF10/JC1553 x 254 RiFr <br /> F110 rif/KA14 Rif                   Ft    F110 argE+, rifilargE, met, pur, lac,     F110 rifd/254 Rip x DA14 Rifr <br />                                              reCAl, rif <br /> Ft6a21ac+-F110 rif&apos;/KA9             F&apos;     Ft0bac+-F110 argE+, rif largE,           F110 rifr/KA14 Rifr x F- <br />                                              met, pur, strr, thy+,  pro, lac,     phenocopies Fr,6.Iac+/ <br />                                              recAl                                      KA9 <br /> Ftb621aC+-F110 rif&apos;/KA1             F&apos;     Fta621ac+-F110 argE+, rifr/argE, met,Ft62lac+-F110 rifKA9 x <br />                                              recA +                               KA1 <br /> FtS62jac+tF110 rif met/KA1          F&apos;     F,621ac+-F110 argE+, rifr, metlargE, Homogenote formation                                              met, recA +                          preceeding strain <br /> FtI62lac+-F110 rif&apos; metP/KA9        F&apos;     Ft,621ac+-F110 argE+, rifr, met/ Ft1621ac+-F110 rif met/KAl x <br />                                              argE, met, pur, thy+,  pro, lac,       KA9 <br />                                              recAl <br /> FtSunbac+tF110 rif met/KA17         F&apos;     Ft,621ac+-F110 argE+, rifg, met/ Ft,6Sac+-F110 rifr met/KA9 <br />                                             argE, met, pur, lac, thy+, thr, trp,        x KA17 <br />                                             recAl <br />      genetic markers relevant current study listed. <br />      b <br />        Ftlac+-F110 rif&apos; met designated KAF1. <br />    Identification recA+ recA strains.              Mating procedures. Aside exceptions  <br />  Rec- phenotype detected sensitivity          scribed  standard mating procedure  ultraviolet light, using published procedures (20).        dilute overnight cultures donors recipients, <br />     Preparation F- phenocopies. Male strains             grown minimal medium broth, sepa- <br />  converted F- phenocopies standard           rately 1:40 Penassay broth, incubate  procedure (26).                                            shaking 37 C 2.5 4 h, mix  VOL. 126, 1975                                              MUTATIONS ESSENTIAL GENES                  41 <br /> 1:1 ratio. mating mixtures incubated           chromosomal rir gene, shaking 1 2 h 37 C.                     wild-type allele, mutated.    mating protocols employed        began experiments F&apos; factor F110 involved temperature-sensitive        carries rif gene (3). <br /> F&apos; factors need screening large numbers F&apos;-containing colonies.                                      F&apos;riflrif strain usually sensitive    Donors containing F.l,,21ac+ fused episomes     rifampin (3).  chromosomal rir <br /> derived F&apos; factor grown stationary      allele mutated rifr rio,  <br /> phase 28 C Penassay broth (Ft1621ac+)     sulting F&apos;riflrif F&apos;rif/rift strain pheno- <br /> liquid minimal medium  temperature-sensitive,         typically rifampin resistant (3).   <br /> fused episomes). diluted 1:20         struct F&apos;rifr/rif strain, use ap- <br /> Penassay broth, shaken 34 C 5 h,         pearance Rif&apos; phenotype score directly <br /> mated shaking 2 h 34 C described        chromosomal mutations rifr rip;   Plates spread mating mixtures            use frequency appearance incubated 28 C. <br />    Replica matings used screen large num-         Rifr mutants estimate success bers isolates ability donate episome   mutagenesis ICR-191. F110 carries normal replication machinery strain       rif allele, step convert carrying episome temperature sensitive        allele rifr. transferring replication. Stationary donor cultures            F110 cell carrying rif gene grown minimal medium 37 C diluted          chromosome selecting homogenate plated minimal plates approximately        formation. <br /> 100 colonies plate. plates incubated         identification mutations chromo- <br /> 37 C colonies approximately 1.5 mm         somal essential gene required episome diameter  48 56 h) replica plated selective plates spread 0.15 ml culture    carrying gene temperature sensitive recipient strain containing temperature-       replication microcolony formation sensitive episome. recipient grown        42 C used identify mutants. <br /> Penassay broth 28 C 5 7 h. selective         second step fuse F110 rif <br /> plate contained minimal medium selecting        Ft.621ac+ obtain episome rif growth parents permitting             gene F110 temperature-sensitive <br /> growth recipients received non-tem-       replication machinery Ft,621ac+. tech- <br /> perature-sensitive episome. crucial        nique fusing F&apos; factors (37) ob- <br /> success procedure donor plates      tain Fts62lac+-F110 rif allowed isolate replica mating plates fresh, e., dried overnight 2 days room temperature.              temperature-sensitive episome relatively easily <br />    Spot matings used test ability      compared alternative mutagenizing strain transfer episomes recipient         isolating temperature-sensitive mutants conjugation. Bacteria used crosses          F110 rif. Ft,62lac+ used replica- <br /> grown minimal medium 37 C stationary             tion shown thermolabile <br /> phase cells did contain temperature-         Fts,14lac+, episome com- <br /> sensitive episome. bacteria did possess tem-     monly used (8, 26). <br /> perature-sensitive episome, cells grown          Finally, wanted F&apos; factor carry 28 C Penassay broth matings. Approxi-          auxotrophic marker carried chromo- <br /> mately 0.01 ml culture strain spotted selective plate. spot dried, 0.01    selection used replace <br /> ml strain deposited       resident F&apos; factor F&apos; factor <br /> mixed spot permitted dry plate      carrying wild-type allele marker. <br /> incubated.                                                replacement important                                                           complementation tests mapping mu- <br />                                                           tations chromosomal essential genes   <br />                     RESULTS                               low accompanying paper [2]). chose <br />    Outline construction F,1621ac+-              met marker purpose used homo- <br /> F110 rifr met episome. discussed              genate formation transfer met marker <br /> wished develop method isolating mu-              KA1 episome. Strain KA1 tants deficient essential genes E. coli.        recA + carries met marker plan induce mutations strains               KA17, chromosome carried duplicate copy essential            used search ess mutants. <br /> gene F&apos; factor temperature sensitive               summary, episome construction neces- <br /> replication. test procedure, chose         sary test procedure isolating determine isolate mutant            mutants deficient essential genes involved: <br /> deficient specific chromosomal essential               isolating F110 episome carrying rifl <br /> gene, rif. reason choosing rif gene           mutation, (ii) fusing F&apos; factor simple tests used deter-             Ft,621ac+, (iii) isolating met variant  42      ARMSTRONG HERMAN                                                              J. BACTERIOL. <br /> fused episome met allele          glucose plates containing methionine. chromosome.               subsequent growth, recombination occur <br /> procedures outlined Table 2 dis-        produce met homogenates, iden- <br /> cussed                                 tified follows. Single-colony isolates    Construction FilO rifr. F110 trans-          selective plates grown stationary phase <br /> ferred conjugation strain 254 Rifr (argE       28 C liquid medium containing glucose <br /> rif recA+; Table 2). Rif&quot; merodiploids            methionine. prevent recombina- <br /> formed recombination subsequent              tion occurring episome growth identified ability         chromosome, episome transferred grow rifampin-containing plates. prevent         KA9, F-recA strain carries occurrence changes epi-          met allele strains KA1 KA17. recombination chromosomal                 conjugation transfer, streptomycin markers, transferred KA14 Rifr,            used select donor.  selec- <br /> recA strain. resulting strain resistant        tion medium contained lactose defi- rifampin, expected. Proof F&apos; fac-       cient arginine <span id='am-28' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> select tor carried rifr allele  rift)       KA9. incubation 28 C, plates segregation rifampin-resistant recombi-         replicated similar plates lacking methionine, <br /> nants episome transferred             colonies didn&apos;t grow replica <br /> KA2, rifs recA + strain.                             plates identified met homozygous mero- <br />    Fusion F110 rir Ft,621ac+. F110 rif/        diploids. colonies purified twice KA14 rifr recA grown stationary phase           streaking obtain single colonies selective glucose minimal medium, diluted 1:40           plates containing methionine 28 C. Penassay broth, grown exponential phase          isolates contained desired episome <br /> 37 C, mated 34. C 2 h F- pheno-        Ft,621ac+-FllO rifr met, desig- <br /> copies Ft,621ac+/KA9 grown stationary            nated KAF1. mutations essential genes <br /> phase 28 C. Undiluted aliquots mating        identified using fused episome mixture spread lactose plates            chromosomal genes duplicated F110 <br /> taining streptomycin select          chromosomal genes carried  plates supplemented             FtS621ac+ dispensable (7). <br /> histidine proline, amino acids required           Chromosomal genotype strain used recipient. Colonies grew incu-       isolating mutations essential genes. <br />  bation 28 C candidates containing         chromosomal genotype constructed  episome produced fusion FilO rill        experiments (Table 1) contained follow- <br />  Flsf2lac+ growth medium           ing mutations: argE, met, pur, thr, trp,  required argE+, met+, pur+ genes         recAl. recA mutation needed pre- <br />  FilO rif lac+ gene Fts62lac+,     vent loss ess mutations chromo- <br />  F&apos; factor exist         recombination episome, <br />  stably cell (9, 33). deletion        carries ess+ alleles.  neces- <br />  DNA occur episomal fu-           sary chromosome strain carry <br />  sion, probably incompatibility      auxotrophic alleles genes carried  sets F replication genes (37), rifr gene   episome selection continued pres- <br />  fused episome rif lies       ence episome exerted. KAF1  argE+ pur+, genes required            lost unless episomal genes required <br />  growth selective plates. putative      growth. argE, pur, lac muta- <br />  Ft,62lac+-F 10 rifr/KA9 strains grew      tions served purpose. functions  minimal plates purified twice streak-         chromosomal markers (met, thr, trp) <br />  ing selective medium           apparent later comple- <br />  tested mating 34 C KA17, F-             mentation procedure discussed.   strain. mating mixtures plated          final strain used mutant isolation <br />  selective medium test transfer           KAF1l81ac+ argE+ rift met pur+/argE met <br />  FtS61lac+-F110 rip/KA9 genes lac+, argE+,      pur lac thr trp recAl. strain obtained <br />  met+, pur+. way obtained            conjugating KAF1/KA9 KA17 (Table <br />  episome did        2). <br />  genes temperature sensitive              Isolation mutants defective essential <br />  replication. fused episome FtS621ac+-F110          genes. Strain KAF1/KA17 mutagenized <br />  rifr carried KA9  pro) transferred       ICR-191 grown minimal medium  KA1 (argE met recA +) conjugation. Dilu-            28 C allow chromosomal segregation. Sta- <br />  tions mating mixture spread           tionary cultures segregated cells  VOL. 126, 1975                                         MUTATIONS ESSENTIAL GENES                 43 <br /> diluted M63 buffer-salts 100        parent strain recipient consistently gave <br /> colonies plate lac-tet agar. plates       numerous Trp+ merodiploids spot matings incubated 28 C 32 34 h       strain KLF23/KL181. mutant isolates <br /> replicated medium          showed parental phenotype forma- <br /> prewarmed 42 C. replica plates           tion viable Trp+ merodiploids concluded <br /> incubated 42 C approximately 50 h.       defective chromosomal essential <br /> colonies, including formed unmuta-          genes examined  <br /> genized bacteria, grew normally             Isolation chromosomal rite mutants. <br /> master replica plates. Rare colonies        KAF1 carries rifr gene, expected <br /> formed,  grew normally           chromosomal mutations essen- <br /> 28 C master plate formed tiny clusters      tial genes rir mutations. Theoretically, microcolonies, called spotty colo-       possible mutated chromosomal <br /> nies, 42 C plate. colonies tenta-    rip gene rifr rifo, tively identified having mutations essen-      identified rif&apos; type essential <br /> tial genes chromosome. mutants           mutant rif&apos; class mutants dependent presence          accept F123 trp+ place KAF1. <br /> episome KAF1 continued growth,           F&apos;rifrir mutants easily identi- <br /> replication episome temperature sensi-     fied rifampin-resistant phenotype. <br /> tive.                                                 mutant strains unable <br />    verify bacteria spotty colo-   form Trp+ merodiploids transferred <br /> nies defective chromosomal          minimal plates rifampin <br /> essential gene diploid region, cells     test rifampin resistance. plates purified corresponding colony        incubated 28 C. rifampin-resistant <br /> 28 C master plate spot-mated         strains independently <br /> strain carrying F123 trp+ episome (strain        isolated mutants KAF1  <br /> KLF23/KL181, Table 1). Trpt merodiploids             placed F110 met+ F123 trp+. selected glucose-arginine-methionine-        KAF1 replaced FllO met+ rifs, mero- <br /> adenine-threonine minimal plates incubated        diploids sensitive rifampin expected <br /> 37 C. failure isolate form viable       rif mutation chromosomal rif gene. <br /> Trp+ colonies mating suggested       Furthermore, mutations KAF1 replaced F123 trp+,       strains mapped rif portion F&apos; factors stably maintained        diploid region (2). conclude cell;  isolate pre-     mutant strains type sumably required continued presence           predicted isolate. seven mu- <br /> KAF1 isolate contained lesion       tants discussed accompanying essential chromosomal gene func-          paper (2). <br /> tion coded episome.                     Construction F110 episomes carrying es- <br />    alternative possibility strain      sential mutations. able classify formed spotty colony 42 C         essential gene mutations complementation <br /> longer accept F&apos; factor conjugation.       groups, wanted transfer F110 <br /> ascertain true, strain        met+ episomes. shall illustrate proce- <br /> forming spotty colony 42 C mated       dure (Fig. 1) using rir muta- strain containing F110 met+ episome.           tions. described, tests F110 met+ carries genes KAF1       identification mutations essential genes  able replace KAF1       involved replacement KAF1 F110 episome carried gene essential   met+. resulting strain following <br /> growth. Replacement KAF1 accom-               genotype: F110 met+ ess+/argE met pur lac thr <br /> plished mating strain KLF10/JC1553           trp ess recAl, designated strain C Fig. 1. spotty colony isolates KAF1/KA17 select-      basic procedure introduce recA + gene <br /> ing Met+ merodiploids. existence          strain episome pick <br /> Met+ colonies mating indicated       ess mutation chromosome inability form Trp+ colonies mating     recombination: F110 met+ ess +/met ess F123 trp+ F123 trp+ did         recAl heterogenotes (strain C) code wild-type allele mutant chro-      F- phenocopies mated recA+ <br /> mosomal essential gene KAF1/         Hfr strain, JC5029 Thy- Leu- (Table 1). KA17 unable accept F&apos; factors          matings 100- <br /> mating.                                              fold efficient recipient grown    Control matings using unmutagenized           28 C instead 37 C liquid minimal medium <br />  44           ARMSTRONG HERMAN                                                                        J. BACTERtIOL. <br /> Strain <br /> desig-                                       Genotype                                              Derivation <br /> nation <br />                 KAF1 met ess+largE met pur lac thr trp ess+ recAl                      Strain KAF1/KA17 (Table 2) <br />  <br />   B                              KAF1 met esst/  ess recAl                             NG ICR-191 mutagenesis; <br />                                                                                            selection Ess- mutants <br />  <br />   C <br />                                                  1r <br />                                  F110 mett ess+l  ess recAl <br />                                                                                                      text) <br />                                                                                         Strain B x KLF10/JC1553; se- <br />                                                                                            lection Met+  Leu+ <br />                                                                                                      colonies <br />      D                                    F- phenocopies                                              (26) <br />  <br />      E                                         &apos;r <br />                   F11O mett esst/argE met pur lac thr trpt ess recAt+                   Strain D x JC5029 Thy- Leu-; <br />                                                                                             selection Trp+, Thy+, <br />                                                                                           Leu+ recombinants; UV irra- <br />                                                                                             diation identify RecA+ <br />                                                                                                      isolates <br />                            1 <br />                            4,                                           1 <br />   F                        (1)                                         (2)               growth &#194;&#177; FUdR treat- <br />                       F&apos;met+                                         F&apos;met+                          ment <br />                        ess+                                           ess <br />  <br />  <br />      G                     (1)                                      (2)                 Strain F (1 + 2) x KA14 Thy- <br />                    F&apos;met+/ (b)                                 F&apos;mett+          (b)         recAl; selection Arg+ <br />                    ess+ / recAI                                ess      IrecAl              Mett Thr+ Pur+ isolates <br />  <br />  <br />      H               (1)                                (2)                               Strain G (1 + 2) x strain  <br />                                                                                            selection Met+ Thy+ colo- <br />                                                                                                         nies <br />               F&apos;met+ /   esst+              F&apos;met+     /     ess+ <br />                 ess+ / recAI                     ess    /   recAl <br />                   VIABLE                              VIABLE <br />                                                                         - <br />                                                                             &lt; <br />                                    (1)                                (2)               Strain G (1 + 2) x strain B; <br />                             F&apos;met+    ess <br />                                       /           F&apos;met+ /   ess            selection Met+ Thy+ colo- <br />                          esst / recALl              ess / recAl                            nies <br />                             VIABLE                  NONVIABLE <br />   FIG. 1. Procedure transfer chromosomal ess mutations F110 episomes. symbol   denotes markers argE met pur lac thr trp, (B) denotes argE met pur lac thy. <br />  F- phenocopies.                         episome  Tlpt RecA+ <br /> mating 37 C 2 h, Trp+ Leu+ Thy+ recom-                        strains usually treated fluorodeoxyu- <br /> binants selected minimal agar 37 C.                       ridine (FUdR) 90 150 min method recA+ gene enters trp+                       Gallant Spottswood (13).  <br /> transfer DNA Hfr strain (6), approxi-                      combinant ess episomes identified, mately 50% Trp+ recombinants inherit                          sample episomes culture recA + gene Hfr. Trp+ recom-                      transferred F-KA14 Thy- (recAl) conju- <br /> binant strains twice purified streaking                      gation prevent recombination. type selective plates, grown                     crosses immediately FUdR <br /> minimal liquid medium 37 C, irradiated                        treatments. 60 90 min mating ultraviolet light (UV) identify UV                        37 C, mating mixtures cells strains F-1 <br /> sistant (RecA+) isolates (strain E).                                 F-2 KA14 Thy- recA diluted 1:100 <br />   increase recombination chromo-                          liquid minimal glucose-thyniine medium <br />  VOL. 126, 1975                                          MUTATIONS ESSENTIAL GENES                 45 <br />  enrich KA14 Thy- recA cells          described Hong Ames (21),  received episome donor strain            method involved temperature-sensitive <br /> (strains G-1 G-2). growth station-       mutations, advantage ary phase 37 C, culture diluted        method used mutations spread glucose-thymine plates form sin-       simply lead inactive gene products. F&apos; factors <br /> gle colonies. colony strains G-1 G-2      available desired region tested presence F&apos;argE+        E. coli chromosome (25), applicability <br /> met+ pur+ ess episome. mat-          method obviously extended. <br /> ing original ess mutant parent:            Naturally, like able iden- <br /> KAF1 met ess+IargE met pur lac thr trp recAl          tify biochemical defects essential <br /> ess (strain B) selecting Met+ Thy+ (Arg+      gene mutants. suggest possible <br /> Pur+) merodiploids. Nonrecombinant (ess  epi-       approaches doing  involve mak- <br /> somes carried strain G-1 replace           ing guess particular proteins resident episome recipient viable      involved. guesses based Met+ Thy+ merodiploids (strain 1), ess          genes known map episomes carried strain G-2 (strain       region studied. procedure involves <br /> 2). ess episomes rare (1 3%        assaying specific protein mutant strain FUdR-treated cultures 0. 1%        shift 42 C. 42 C untreated cultures), replica matings          differential rate synthesis protein <br /> used screen large numbers colonies        reduced compared essen- inability produce Met+ Thy+ merodip-       tial proteins chromosomal gene loids. Candidates carrying ess episome         protein inactive. 42 C culture mutant <br /> identified test twice purified        cells does completely lack essential gene <br /> selective plates spot-mated        function. cells culture contain chromosomal ess + parent (strain , KAF1         unreplicating episome, presumably met ess+/ess+ KA17, chromosomal ess           cells ess + gene product continues mutant derivative (strain B). Isolates gave      synthesized. F-, cells continually segre- <br /> Met+ Thy+ colonies mated chro-          gated,  cells contain mosomal ess+ strain, Met+ Thy+ colonies        nearly normal ess+ gene product mated chromosomal ess strain,           segregated, growth deduced following desired gen-       cells descendants gradu- <br /> otype: F&apos;argE+ met+pur+ esslargE met pur lac          ally slow ess+ gene product  <br /> recA1 thy.                                            comes diluted rate-limiting growth. <br />    method, constructed F&apos;met+            second type biochemical analysis rif&apos; episomes KAFl/met rif&quot; strains.          applied mutants isolated predicted, episomes form Met+ Thy+ mer-         immunological technique described odiploids matings ess+ parent             Horvitz (23). Briefly, supernatant extract KAFl/met rir matings rif&quot;         obtained radioactively labeled cells mutant parent.                                        incubated antibody purified protein. <br />                                                       antigen-antibody complex analyzed <br />                   DISCUSSION                          sodium dodecyl sulfate-polyacrylamide slab <br />    paper, delineated procedure      gel electrophoresis. Horvitz (23) used proce- isolating mutants deficient chromosomal        dure demonstrate sequential phosphoryla- <br /> essential genes partially diploid strain E.   tions subunit RNA polymerase. coli K-12. Knowing episome          extracts mutants, look poly- <br /> ferred partial diploidy carried essential         <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> fragments, frameshift mutations <br /> gene rif, predicted essential     frequently lead premature termination mutants isolated defective         translation (32). gel pattern extract <br /> chromosomal rif gene.  mutants          mutant strain extra type isolated.                              band, polypeptide fragment,    feature method developed       present gel pattern wild-type concentrates limited region ge-        strain. approach  course, nome region covered F&apos; used.            work polypeptide fragment al- <br /> cause method enables classify          tered longer precipitated mutations genes complementation               antibody. <br /> tests, feasible saturate particu-      possible approach involves isola- <br /> lar chromosomal region mutations es-          tion partial revertant strains viable <br /> sential genes. idea localized mutagenesis      absence fused episome. hap- <br />  46       ARMSTRONG HERMAN                                                                                   J. BACTERIOL. <br /> loid strains assayed enzyme activ-                         transfer ribonucleic acid synthetase mutants ofEsch- <br /> ity exhibit specific activity lower                        erichia coli. J. Bacteriol. 112:102-113. <br />                                                                 13.   Gallant, J., T. Spottswood. 1965. recombino- -wild-type F- parent.                            genic effect thymidilate starvation Escherichia <br /> partial revertants, carry                         coli merodiploids. Genetics 52:107-110. <br /> tragenic intergenic suppressor mutations,                    14.   Gallucci, E., G. Pacchetti, S. Zangrossi. 1970. analogous leaky mutants                              Genetic studies temperature sensitive nonsense <br />                                                                         suppression. Mol. Gen. Genet. 106:362-370. <br /> scribed Schlessinger (34).                                   15.   Gefter, M. L., Y. Hirota, T. Kornberg, J.  Wechsler, <br />    conclusi <br /> </body></html>