<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Sept. 1992, p. 5814-5819                                                                      Vol. 174,  18 <br /> 0021-9193/92/185814-06$02.00/0 <br /> Copyright &#194;&#169; 1992, American Society Microbiology <br />  <br />  <br />        Cloning Erwinia herbicola Gene Necessary Gluconic <br />             Acid Production Enhanced Mineral Phosphate <br />            Solubilization Escherichia coli HB101: <span id='am-7' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>Nucleotide</span> <br />            Sequence Probable Involvement Biosynthesis <br />                 Coenzyme Pyrroloquinoline Quinone <br />          SHIH-TUNG LIU,&apos;* LAN-YING LEE,&apos; CHIN-YING TAI,1 CHIEN-HUI HUNG,&apos; YU-SUN CHANG,&apos; <br />                      JAMES H. WOLFRAM,2 ROBERT ROGERS,2 ALAN H. GOLDSTEIN3* <br />         Department Microbiology Immunology, Chang-Gung Medical College, Kwei-ShanTaoyuan, Taiwan&apos;; <br />               Biotechnology Group, Idaho National Engineering Laboratory, Idaho Falls, Idaho 834152;                            Department Biology, California State University Los Angeles, <br />                               5151 State University Drive, Los Angeles, California 900323 <br />                                            Received 28 April 1992/Accepted 16 July 1992 <br />  <br />            Escherichia coli capable synthesizing apo-glucose dehydrogenase enzyme (GDH) cofactor <br />          pyrroloquinoline quinone (PQQ), essential formation holoenzyme.  absence <br />          exogenous PQQ, E. coli does produce gluconic acid. Evidence presented expression <br />          Erwinia herbicola gene E. coli HB101(pMCG898) resulted production gluconic acid,  <br />          turn, implied PQQ biosynthesis. Transposon mutagenesis showed essential gene locus          1.8-kb region 4.5-kb insert plasmid pMCG898. 1.8-kb region contained apparent <br />          open reading frame. paper, present nucleotide sequence open reading frame, 1,134-bp <br />          DNA fragment coding protein Mr 42,160. deduced sequence protein high <br />          degree homology gene III (Mr, 43,600) PQQ synthase gene complex Acinetobacter <br />          calcoaceticus previously identified Goosen et al. (J. Bacteriol. 171:447-455, 1989). minicell analysis, <br />          pMCG898 encoded protein Mr 41,000. data indicate E. coli HB101(pMCG898) <br />          produced GDH-PQQ holoenzyme,  turn, catalyzed oxidation glucose gluconic acid          periplasmic space. result gluconic acid production, E. coli HB101(pMCG898) showed enhanced <br />          mineral phosphate-solubilizing phenotype acid dissolution hydroxyapatite substrate. <br />  <br />    Quinoproteins play major role regulation                  synthase genes E. coli resulted GDH activity bioenergetic processes gram-negative bacteria,                absence exogenous PQQ. used slight modifi- <br /> cluding Erwinia spp. (2). species Erwinia,               cation identify PQQ synthase gene nonphosphorylating oxidation pathway primary mech-                Erwinia herbicola EHO10 (18). experimental approach <br /> anism aldose sugar utilization (2). quinoprotein                 included unique phenotypic screening based glucose dehydrogenase (GDH) controls unique step                  elucidation metabolic basis mineral metabolic pathway.  GDH plays direct role              phosphate solubilization (Mps+) phenotype gram-negative generation transmembrane proton motive force              bacteria. oxidation aldose sugars (5). generally                     Poorly soluble mineral phosphates hydroxyapatite <br /> accepted  gram-negative bacteria, membrane-                  (HAP) dissolved acidification (7, 8). bacterial <br /> bound quinoprotein GDH present periplasmic                Mps+ phenotype assayed addition HAP cytoplasmic membrane. GDH member                      poorly soluble phosphates culture medium. largest group quinoproteins, use cofactor              agar plates, solubilization visualized directly 2,7,9-tricarboxyl-lH-pyrrolo[2,3-flquinoline-4,5-dione (PQQ)             clearing zones, liquid culture level Pi (5).                                                                     determined analytically. expected, bacteria pro- <br />   Escherichia coli does synthesize PQQ, does                  duce large amounts gluconic acid, E. herbicola, <br /> synthesize apo-GDH dependent uptake                  high levels mineral phosphate solubilization, PQQ environment culture medium (9). Bind-                 E. coli strains usually poor solubilizers. ing cofactor presumably simplified location          strategy clone genes E. herbicola E. apoenzyme outer face cytoplasmic mem-                  coli order study encoded gene products involved brane. GDH holoenzyme formed E. coli                  Mps+ phenotype. presumed genes responsi- <br /> functional genes PQQ biosynthesis introduced.               ble Mps+ phenotype genes controlling organic elegantly exploited Goosen et al. (9)              acid production secretion identical order identify isolate PQQ synthase genes                    closely related (7, 8). <br /> Acinetobacter calcoaceticus basis expression                paper, report construction plasmid plasmids cloned E. coli. expression PQQ                   containing functional mineral phosphate-solubilizing gene <br />                                                                          defined HAP-solubilizing phenotype. Data                                                                          strongly suggest metabolic function protein <br />   * Corresponding authors.                                               encoded gene biosynthesis PQQ,                                                                    5814 <br />  VOL. 174, 1992                                      CLONING ERWINLA HERBICOLA PQQ SYNTHASE GENE                                       5815 <br />  <br />                                                TABLE 1. Bacterial strains plasmids <br />       Strain plasmid                                  Characteristics                                       Source reference <br /> Strains <br />   Envinia herbicola EHO10        Wild type                                                           18 <br />   Escherichia coli <br />     HB101                        F- hsdS20 (rB- mB-) supE44 recA13 ara-14proA2                       4 <br />                                    rpsL20(Strr) xyl-S mtl-ql galK2 lacYl X <br />     MG1063                       recA F+::TnlOOO                                                     10 <br />     P678-54                      F- thr leu thi supE lacYfluA gal mal xyl ara mtl min                C.  Kado laboratory, University <br />                                                                                                         California Davis <br />     Eco898                        HB101::TnS                                                         S.-T. Liu laboratory <br />  <br /> Plasmids <br />   pSL775                          cloning vector using pigment genes E. herbicola          14 <br />                                     selection marker; 7.0 kb <br />   pMCG898                         deletion derivative pIMPS1 containing complete              study <br />                                     PQQ synthase gene <br />  <br />  <br /> turn, results assembly GDH holoenzyme,               lysed, proteins analyzed sodium dodecyl <br /> presented. report complete nucleotide sequence               sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) ac- gene deduced <span id='am-15' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> sequence                cording method Laemmli (13). <br /> shows high degree homology gene III               DNA sequencing. Supercoiled plasmids sequenced gene PQQ synthase complex  calcoaceticus <br /> identified Goosen et al. (9). <br />  <br />                 MATERIALS METHODS                                                         O- Hindlil <br />    Bacterial strains, plasmids, culture conditions. Bacte-                                     8amHI                  BamHI <br /> rial strains plasmids used study listed Table <br /> 1. E. coli cultured Luria-Bertani (LB) broth (16) 37&#194;&#176;C. E. herbicola cultured medium 28&#194;&#176;C. antibiotics used selection ampicillin (Ap) (100 <br /> ,ug/ml) kanamycin (50 ,ug/ml). <br />    Subcloning. Plasmids screened according                         Hindlil <br /> method Kado Liu (12). Restriction enzymes, T4 DNA <br /> ligase, E. coli DNA polymerase (Klenow fragment)                     pIMPS1 (55kb) purchased Boehringer Mannheim Biologicals used according methods described Maniatis et <br /> al. (16). HindIII fragments insert cosmid pIMPS1 <br /> (8) subcloned HindIII site pBR322- <br /> derived vector pSL775 (14). clone pB2 (27.6 kb)                                                                               ,   Banit <br /> taining Mps+ activity selected M9-HAP agar. plasmid deleted BamHI (pB8, 13.3 kb),                                                                                      7.0 <br /> digested BanII, religated, resulted                                                                                   &apos;BgIll <br /> pMCG898 (Fig. 1), retained ability confer HAP-solubilizing phenotype recipient E. coli. <br />    TnlOOO mutagenesis. TnlO00 insertional mutagenesis                                                                  pB8(1 3.3kb) <br /> (gamma-delta) carried according method Guyer (10). Plasmid pMCG898 transformed                                                                      Banil <br /> MG1063. pMCG898::Tnl000 insertional mutants trans- <br /> ferred Eco890 conjugation. resulting transcon- <br /> jugants cultured LB broth kanamycin plated LB agar containing Ap kanamycin.                                                                        Banil <br /> locations orientations TnlOOO insertions determined restriction mapping BamHI.                                                                        AP(      Bgll <br />    Preparation minicells analysis plasmid-encoded                                                5.0 vector  PMC0981 <br />                                                                                                                         9.5kb  gSU 4. 5 <br /> proteins. plasmids pMCG898 pBR322 trans-                                                          NdeI <br /> formed minicell-producing strain P678-54 (17). Cul-                                                                              Kpnt <br /> tures grown overnight LB-Ap broth 37&#194;&#176;C. Minicells                                                      EooRV                   del purified sucrose density-gradient centrifugation,                                                                                                                               BamHI <br /> plasmid-encoded proteins labelled 45 pCi                    FIG. 1. Subcloning strategy map pMCG898 carrying L-[35S]methionine (1,020 Ci/mmol; Amersham Corp.) accord-              putative E. herbicola PQQ synthase gene. gene originally <br /> ing method Meagher et al. (17). minicells           cloned 55-kb insert cosmid vector pHC79 (8). <br />   5816     LIU ET AL.                                                                                                      J. BACTEkRIOL. <br />  <br />  according chain termination method Sanger et al. <br />  (19). DNA sequenced gamma delta                 BamHI                                               Banil <br />  primers according method Liu et al. (15). Gaps  filled primers synthesized results obtained                gamma-delta sequencing (primer walking). DNA <br />  strands sequenced. primers synthe- <br />  sized Cruachem PS250 DNA synthesizer. Sequence                                                         analysis, translation, comparisons previously iden-                                                                      tified PQQ synthase genes using Intellige-             0          1          2          3           4          5 kb <br />  netics Suite program (11).                                         FIG. 2. Transposon TnlOOO mutagenic analysis region     Changes medium pH bacterial growth. Bacteria        pMCG898 necessary gluconic acid production Mps+ <br />  cultured overnight LB-Ap. Cells harvested        phenotype. mutants shown figure contained <br />  centrifugation washed times LB-Ap.           approximately 1.8-kb region. insertions simulta- <br />  Cells  5-ml culture) inoculated 250 ml       neously eliminated gluconic acid production Mps+ pheno- <br />  M9WP culture medium. Media (20 ml) sampled 6          type. Insertions outside region did affect GDH+  h, cells centrifuged, supernatants         Mps+ phenotype. Insertions gamma-delta orientation (0) <br />  passed Millipore filters. pH values deter-      delta-gamma orientation (-) indicated. <br /> mined, 100 ,ul supernatants spotted M9WP-HAP medium determination HAP solubi- <br /> lization. M9WP-HAP modified M9 (16) minimal medium,         tions covered approximately 1.8-kb segment, soluble Pi replaced HAP previously described (8),     shown Fig. 2. Data based size insert region, <br />  1% glucose carbon source, appropriate amino acids      restriction map, sequencing results strongly antibiotics (8). M9WP medium             indicated Mps- insertions located gene. <br /> added HAP.                                                       did Mps- mutants insertions <br />    Mineral phosphate solubilization determination      outside 1.8-kb region. <br /> ganic acids bacterial growth. Liquid medium (GMS            Data shown Fig. 3 indicate proteins <br /> medium) consisted  1 liter) 10 g glucose, 2 g       encoded genes carried pMCG898. Plasmid <br /> (NH4)2S04, 0.3 g MgCl2. 6H20, 0.001 g MnCl2- 4H20,         pSL775 originally derived pBR322 (14). 0.0006 g FeSO4. 7H20, 0.0006 g NaMoO4. Amino           pMCG898, pSL775-specific sequence  <br /> acids added 20 ,ug/ml, Ap added 50 ,ug/ml.    leted pBR322 used control minicell <br /> Tricalcium phosphate present 20 mg/ml. 50-ml <br /> volume media inoculated 5 ml 48-h-old <br /> (stationary phase) liquid culture various bacteria.                                         B    C    D inoculated, flasks placed shaker grown 30&#194;&#176;C (E. herbicola) 37&#194;&#176;C (E. coli).  <br /> milliliter samples taken daily. concentration soluble Pi plate counts determined samples. Organic acids separated high-performance <br /> liquid chromatography Beckman Interaction <span id='am-1' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>Ion</span> 300 <br /> column. Detection A220. identification  <br /> known organic acids based comparison elution <br /> times unknowns elution times multiple stan-                              41   - <br />  <br />  <br /> dards. detection limit gluconic acid 10 ppm.                                33 <br />    Nucleotide sequence accession number. GenBank ac-                                32   -       cession number nucleotide sequence PQQ <br /> synthase gene E. herbicola EHO10 M94448. <br />  <br />                 RESULTS DISCUSSION <br />    screening strategy isolation clones based fact  plate assays, HB101 visibly <br /> solubilize poorly soluble mineral phosphates HAP tricalcium phosphate, E. herbicola (8). <br /> Plasmid pMCG898 (insert size, 4.5 kb) (Fig. 1) originally <br /> derived cosmid clone pIMPS1, contained 55-kb insert E. herbicola DNA. cosmid identified Goldstein Liu ability confer HAP- <br /> solubilizing phenotype HB101 (8). <br />    shown Fig. 1 2, combination subcloning transposon mutagenesis data indicated 1.8-kb region         FIG. 3. SDS-PAGE [35SJmethionine-labelled proteins  E. herbicola DNA sufficient confer GDH+            coded pMCG898 minicells. Proteins encoded pBR322 (lane <br /> Mps+ phenotype E. coli. basis selection         synthesis directed pMCG898 (lane B) shown. M9-HAP agar appropriate supplements, screened          Klenow fragment repair-induced base insertion BamHl site total 113 insertion mutants copy           pMCG898 caused loss 33-kDa protein (lane C),                                                                  similar insertion BanlI site did affect synthesis pECO898::Tn1O00. Mps-, rest             protein (lane D). indicated proteins Mrs 41,000 Mps+. 25 mutants insertions loca-         32,000 apparently encoded 4.5-kb insert. <br />  VOL. 174, 1992                                      CLONING ERWINIA HERBICOLA PQQ SYNTHASE GENE                                 5817 <br />  <br />                                   RTGRCTGNCC   RCGCGCCCRC    GGTGRRCCCR    CCGCTCTGGC TGCTGGCGGR GCTGRCCTRC <br />                            60    CGCTGCCCGC    TGCRGTGCCC    CTRCTGCTCC    RRCCCRCTCG RTTTCGCCGC CCRAGRRRRR <br />                            120   GRGCTGRGTR    CCGRGCRGTG    GRTCGRGGTR    TTTCGCCRGG CRCGCGCGRT GGGCRGCGTG <br />                            180   CRGTTGGGCT    TCTCGGCGGG    CGRGCCGCTG    GTSCGTRRRG RTCTCCCCGR GCTGRTCGCC <br />                            240   GCCGCGCGCG    RTCTCGGCTT    TTRCRCCRRC    CTGRTCRCCT CCGGCRTCGG CCTGRCCGRG <br />                            300   RRGRRGCTGC    RGRCCTTCGC    CGRTGCCGGG    CTGGRCCRTR TTCRGRTCRG CTTTCRRGCC <br />                            360   RGCGRCGRGR    CGTTRRRCGC    GGCTCTGGCT    GGCTCTGCCR RRGCRTTTCR GCRGRRGCTG <br />                            420   RCRRTGGCCR    RRGCGGTTRR    GGCGCTCGGC    TRTCCGRTGG TGCTCRRCTT CGTGCTGCRT <br />                            480    CSCCRTRRTR   TCGRTCRGRT    TGRCCGTRTC    RTCGRGCTGG CGRTCCRGCT GGRTGCCGRT <br />                            540    GRCGTRGRGC   TGGCRRCCTG    CCRGTTCTRC    GGCTGGGCGC RCCTGRRTCG CGRRGGCCTG <br />                            600    CTTCCCRCCC   GCGRRCRGRT    CGCCRRRGCR    GRGRGCGTGG TCCRGCGCTR CCGCGRRCGC <br />                            660   RTGGCCGTCG    RCGGCRRGCT    GGCTRRTCTG    CTGTTTGTCR CGCCGGRCTR CTRTGRGGRG <br />                            720   CGGCCRRRRG    GCTGTRTGGG     CGGTTGGGGR    GCCRTTTTCR TGRGCGTGRC GCCGGRRGGR <br />                            780   RTGGCCCTGC    CCTGCCRCRG     TGCCCGTCRG    CTGCCGGTGR RGTTTCCGTC GGTGCTGGRR <br />                             840 CRCGRTCTCG     RGCRTRTCTG     GTRTCRCTCC    TTCGGCTTTR RCCGTTRCCG CGGCRRTGCG <br />                             900  TGGRTGCCGG    RGCCGTGCCG     RTCCTGCCCG    GRGRRRGRGR RRGRCTTCGG CGGCTCRTGC <br />                             960  TGCCRGGCCT    TCRTGCTCRC     CGGCGRCGCT    GRTRRTGCCG RCCCGGTGTG TGCTRRRTCG <br />                             1020 CCCCRTCRCG    GCRCCRTTTT     GGCGGCGCGR    GRRGCGGCRR RCCGCRCCCR SGCTCGGCRTC <br />                             1080 GRTCRGCTGC     GTTTCCGCRR    TCRGGCCRRC    TCCCGGCTGT TTTRCRRRGG CTRR <br />  <br />   FIG. 4. DNA sequence open reading frame (starting +1) presumed coding mps gene contained 1.8-kb region <br /> identified transposon mutagenesis essential gluconic acid production E. coli HB101(pMCG898). <br />  <br />  <br /> analyses. proteins apparent Mrs 41,000,                solubilizing 24 32% tricalcium phosphate <br /> 33,000, 32,000 synthesized pMCG898            solubilized E. herbicola time period. <br /> pBR322. base insertion generated BamHI             value represents average separate experi- <br /> site pMCG898 Klenow repair, protein Mr          ments. Numbers CFUs similar various <br /> 33,000 longer synthesized (Fig. 3, lane C).           bacterial strains  shown). base addition BanIl site did eliminate                Table 2, seen gluconic acid synthesis protein (Fig. 3, lane D), suggesting      detectable media E. herbicola proteins Mrs 41,000 32,000 encoded           HB1O1(pMCG898) cells media HB101 E. herbicola DNA insert, protein Mr             HB1O1(pBR322) cells. E. herbicola carried 33,000 encoded fusion sequence vector          mineral phase solubilization day 3. Likewise, insert. discussed  open reading frame <br /> identified 1.8-kb region pMCG898 essential gluconic acid production encoded protein deduced                  MTIHAPPPLWLLAELTYRCPLQCPYCSNPLDFAAQEKELSTEQWIE VF <br /> Mr 42,160, compares favorably protein                       MTEGVGLPLWLLAELTYRCPLQCPYCSNPLDYAQHKNELTTQEWFDVF Mr 41,000 seen minicell analysis. <br />    obtained 1,960 bp sequence covering entire                    POARAMGSVQLGFSAGEPLVRK DLPELI AAA RDL G FYTNUTSG    GLTEKKLQ <br /> 1.8-kb region identified TnlO00 mutagenesis essential                DAQMGQVLGFSGGEPLVPaXE-LVAHA HQQGFYTNLITSGMGLTEQRIA GDH+ Mps+ phenotype. Figure 4 shows com- <br /> plete sequence significant open reading frame                  TFAD AGLDHIQ SFQASDETLNA ALAGSAK AFQQKLT MAKAVKALGYPMVLN <br /> contained fragment. open reading frame                 DLKQ AGLDHIQVSFQASDPVVNDALAGSKH  AFEQKYEMCRLVKKYDYPMVLN <br /> 1,134-bp long codes protein deduced Mr 42,160. Figure 5 compares deduced amino acid sequence                  FVLHRHN IDQ IDR ELAJQLDADDVELATCOFYGWAHLNREGLL PTREQ IAKA putative PQQ synthase gene gene III multigene complex involved synthesis PQQ                    FV IHRHN IDO IEQ I ELClENADTVELA ICQFYGWAF LNRQGLLPTQECLTRA <br /> calcoaceticus (9). seen  overall, 60% <br /> identity E. herbicola gene gene III                   ESWQRYRERMAVDGKLANL LFVrPDYYEERPKGCMGGWGA IFMSVTPEG <br /> calcoaceticus.                                                             ER TNEYREKL KAONHPCKLI FWPDY YEERPKCMNGWGK IFFT VAPDG <br />    end 24 h liquid culture, E. herbicola  lowered pH media 3.5. HB1O1(pMCG898)                        MALPCHSARCLPVKFPSVL EHDLEHIWYHSFGFNRYRGNAWMPEPCRSCP <br />  lowered pH approximately 4.0 end                MALPCH/RtLP FPNVREHKLSDI WYKSTGFNHFRGDAWMPEGCRSCP <br />  time period. HB1O1(pMCG898-50)  pMCG898:Tnl000 mu- <br />  tant insertion outside 1.8-kb Mps region) gave  similar phenotype. HBBO1(pBR322) capable low-                  EKEKDFGGSCCQAF MLTGDADNADPVCAKS PHHGT AAREAANRTOLGID <br />  ering pH media 5.2                        DKDRDFGGCRCQAYKLTGDAANADPVCGKS PYHQMIEQARAESOLVAPLON <br />  conditions. HBBO1(pMCG898-51) (TnlOOO insertion  1.8-kb Mps region) unable lower pH <br />  5.2 (pH data shown).                                            QLRFRNQANSRLFYKG <br />    Table 2 shows soluble Pi released                    LVFRNSRNSKS LSATQNIPVHTITDI <br />  tricalcium phosphate medium various bacteria.            FIG. 5. Alignment deduced amino acid sequences E. <br />  seen  day 7, HB1O1(pMCG898) solubilized               herbicola mps gene (upper) gene III multigene complex <br />  approximately 81% tricalcium phosphate E.                 involved synthesis PQQ  calcoaceticus. Regions  herbicola. HB101 HB1O1(pBR322) capable                   identity indicated connecting dots. <br />  5818       LIU ET AL.                                                                                                                   J. BACTEDRIOL. <br />  <br />    TABLE 2. Production gluconic acid release Pi                basis data, propose      tricalcium phosphate different bacterial strains             cloned characterized PQQ synthase gene E. <br />                          liquid culturea                                    herbicola gluconic acid produced GDH-PQQ <br />                                                            Gluconic acid    holoenzyme provides molecular basis Mps+ phe- <br />         Strain             Days culture   P1 released <br />                                                  g/ml)    (integrated     notype shown E. herbicola recombinant E. coli.                                                             area   units)   gram-negative bacteria, gluconic acid undergo <br /> EHO10                            3              350            8.65         extracellular oxidation 2,5-diketogluconic acid (2). <br /> HB101(pMCG898)                   7              300            4.12         pathways provide basis metabolic <br /> HB101                            7               88            NDb          engineering Mps+ phenotype accomplish effica- <br /> HB1O1(pBR322)                    7              115            ND           cious biotransformation rock phosphates. <br /> Blankc                           7                8            ND <br />   Data averages separate experiments.                                           ACKNOWLEDGMENTS <br />   bND, detected.                                                          work supported biological research grant NSC78- <br />   c bacteria present.                                                    0418-B169-03 S.T.L. National Science Council                                                                             Republic China research contract DEAC07-76ID01570                                                                             J.H.W., R.R., H.G. USDOE Office Industrial <br />  gluconic acid peak detected day 3. appearance                Processes, Division Advanced Industrial Concepts. <br />  gluconic acid peak HB101(pMCG898) delayed <br />  later experiment peaked day 7, did                                              REFERENCES <br />  maximum mineral phosphate solubilization. HB101(pMCG                        1. Ameyama, M., K. Matsushita, Y. Ohno, E. Shinagawa, 0. <br />  898) produced approximately 48% gluconic acid                           Adachi. 1981. <span id='am-22' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span>D-Glucose</span> dehydrogenase  <br /> </body></html>