<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                            DNA damage-induced alterations chromatin contribute                            genomic integrity age-related changes gene expression <br />  <br />                            Abstract <br />                                 Genomic instability alterations gene expression hallmarks eukaryotic aging.                                 yeast histone deacetylase Sir2 silences transcription stabilizes repetitive DNA,                                 aging response DNA break, Sir complex relocalizes sites genomic instability, <br />                                 resulting desilencing genes cause sterility, characteristic yeast aging. Using <br />                                 embryonic stem cells, mammalian Sir2, SIRT1, represses repetitive DNA                                 functionally diverse set genes mouse genome. response DNA damage, SIRT1 <br />                                 dissociates loci relocalizes DNA breaks promote repair, resulting                                 transcriptional changes parallel aging mouse brain. Increased SIRT1 expression <br />                                 promotes survival mouse model genomic instability suppresses age-dependent <br />                                 transcriptional changes.  DNA damage-induced redistribution SIRT1 chromatin <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 modifying proteins conserved mechanism aging eukaryotes. <br />  <br />  <br />                            Keywords <br />                                 epigenetics; sirtuin; chromatin; histone; DNA repair <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2008 Elsevier  rights reserved. <br />                            * <br />                              Contact: David_Sinclair@hms.harvard.edu, Phone: 617-432-3931, Fax 617-432-6225. <br />                            7Present address: Max-Planck-Institute Experimental Medicine, 37075 G&#195;&#182;ttingen, Germany <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Oberdoerffer et al.                                                                                               Page 2 <br />  <br />  <br />                               Introduction <br />                                                  Chromosomes arguably difficult structure organism maintain NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lifetime. Chromosomes break, mutations accumulate, youthful gene expression patterns <br />                                                  progressively lost. changes gene expression interpreted beneficial <br />                                                  responses cellular damage (Narita et al., 2006; Niedernhofer et al., 2006).                                                   numerous stochastic changes gene expression apparent long-term benefit <br />                                                  organism fact detrimental (Bahar et al., 2006). &quot;Heterochromatin <br />                                                  Island Hypothesis&quot; related hypotheses propose alterations chromatin                                                  resulting gene expression changes drive aging process, evidence lacking <br />                                                  (Cutler, 1995; Imai Kitano, 1998; Vijg, 2004; Villeponteau, 1997). <br />  <br />                                                  Potential clues relationship epigenetic changes aging come                                                  studies Saccharomyces cerevisiae, epigenetic changes primary cause                                                  aged phenotype. SIR2 gene, encoding nutrient-responsive NAD+-dependent histone <br />                                                  deacetylase, emerged key regulator health lifespan yeast                                                  organisms (Haigis Guarente, 2006). major functions Sir2 repress gene <br />                                                  expression silent mating type loci HML HMR (Klar et al., 1979; Rine et al., 1979) <br />                                                  suppress recombination ribosomal DNA (rDNA) locus, gives rise                                                  toxic rDNA circles (ERCs) (Sinclair Guarente, 1997). yeast cells age, Sir protein <br />                                                  complex dissociates HM loci moves nucleolus response ERC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  accumulation causing sterility, hallmark yeast aging (Kennedy et al., 1997; Sinclair                                                  Guarente, 1997; Smeal et al., 1996).  redistribution chromatin modifying factors <br />                                                  resulting changes transcription cause primary yeast aging phenotypes. <br />  <br />                                                  Aging stimulus causes yeast Sir proteins relocalize. DNA breakage <br />                                                  causes Sir proteins dissociate HM loci relocate DNA breaks, DNA <br />                                                  damage checkpoint dependent manner (Martin et al., 1999; McAinsh et al., 1999; Mills et <br />                                                  al., 1999). effect relocalization appears fold:   expression HM genes <br />                                                  promotes DNA repair (ii) Sir proteins directly modify chromatin surrounding break <br />                                                  site, possibly facilitate DNA repair (Lee et al., 1999; Tamburini Tyler, 2005). <br />  <br />                                                  evidence related processes occur mammals.  cells damaged                                                  oxidative stress vitro undergo stochastic transcriptional changes parallel aged <br />                                                  heart tissue (Bahar et al., 2006). Second, deficiency DNA repair factor ERCC1 <br />                                                  accelerates aging phenotypes generates gene expression profiles reminiscent aged <br />                                                  animals (Niedernhofer et al., 2006).  cells senesce replicative aging vitro <br />                                                  aged tissues vivo exhibit alterations heterochromatin (Herbig et al., 2006; Narita et <br />                                                  al., 2006) secrete growth factors drive tumorigenesis (Campisi, 2005). Finally, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  oxidative DNA damage promoters correlates gene repression aging human <br />                                                  brain (Lu et al., 2004) linked transcriptional epigenetic changes <br />                                                  contribute Alzheimer disease (Wu et al., 2008). <br />  <br />                                                  <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, study tested Sir2-mediated alterations chromatin contribute                                                  aging mammals. observations,  consistent possibility.                                                  mammalian ortholog Sir2, SIRT1, regulates expression individual genes <br />                                                  (Picard et al., 2004; Pruitt et al., 2006; Vaquero et al., 2004) formation facultative <br />                                                  heterochromatin (Vaquero et al., 2007). SIRT1 linked DNA damage <br />                                                  response regulation p53 (Luo et al., 2001; Vaziri et al., 2001) interaction                                                  Nbs1, component DNA damage sensor complex MRN (Mre11-Rad50-Nbs1) (Yuan <br />                                                  et al., 2007). Furthermore, SIRT1 recently implicated regulation DNA <br />                                                  methylation patterns damaged CpG-rich DNA (O&apos;Hagan et al., 2008). Deletion                                                  Sir2 homolog, SIRT6, reduces base excision DNA repair causes accelerated aging <br />  <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                             Page 3 <br />  <br />  <br />                                                  phenotype mice (Mostoslavsky et al., 2006). study, map interaction <br />                                                  SIRT1 mouse genome identify evolutionarily conserved DNA <br />                                                  damage response drive changes gene expression aging. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results <br />                               Loss Sir2 SIRT1-dependent silencing response oxidative stress <br />                                                  previous studies relocalization yeast Sir2 utilized highly artificial means                                                  induce DNA damage, EcoRI yeast HO endonuclease (Mills et al., 1999).                                                  test stress relevant aging results desilencing mating-type loci, yeast <br />                                                  strain carrying GFP reporter HMR locus (HMR::GFP) exposed oxidative <br />                                                  stress  e. <span id='am-42' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span>H2O2</span>). tight correlation H2O2 levels HMR derepression <br />                                                  (Figure 1A Figure S1A). <br />  <br />                                                  additional copy SIR2 (2xSIR2) extends replicative lifespan (Kaeberlein et al., 1999), <br />                                                  effect oxidative stress resistance loss silencing unknown. treated <br />                                                  H2O2, strains increased Sir2 levels maintained HMR silencing stable <br />                                                  rDNA compared wild type cells (Figures 1A B). non-lethal concentrations                                                  H2O2, 2xSIR2 strain significantly greater replicative lifespan (Figure 1C                                                  Figure S1B).  additional SIR2 suppresses toxicity, genomic instability,                                                  desilencing caused genotoxic stress. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  sought examine findings relevant mammals. tested <br />                                                  SIRT1 associates highly repetitive DNA pericentromeric major satellite <br />                                                  repeats mouse ES cells, cell type previously used study satellite repeat <br />                                                  silencing (Kanellopoulou et al., 2005). Histone H1 acetylation lysine 26 (H1AcK26) <br />                                                  served read SIRT1 deacetylase activity (Vaquero et al., 2004). Using chromatin <br />                                                  immunoprecipitation (ChIP) quantitative PCR (q-PCR), detected association <br />                                                  SIRT1 major satellite repeats disrupted pansirtuin inhibitor <br />                                                  <span id='am-8' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nicotinamide</span> (NAM) (Figure 1D 1E). coincided increase repeat <br />                                                  transcripts (Figure 1F) H1AcK26 (Figure 1E). Consistent known redundancy <br />                                                  histone deacetylases (Zhu et al., 2004), trichostatin (TSA), NAM sirtinol, pan- <br />                                                  class II pan-class III (sirtuin) inhibitors, respectively, caused desilencing major <br />                                                  satellite repeats (Figure 1F Figures S1D E, (Kanellopoulou et al., 2005)),                                                  significant increase repeat transcripts seen stable knock SIRT1 <br />                                                  (Figure S1C data shown). Ongoing work aimed identifying role                                                  HDACs contribute satellite repeat silencing. <br />  <br />                                                  accordance observations yeast, treatment cells non-cytotoxic levels NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  H2O2 greatly decreased SIRT1 bound repeats, coinciding increase <br />                                                  H1AcK26 (Figure 1E, Figure S2 S3). Similar effect NAM TSA, <br />                                                  oxidative stress increased transcription loci (Figure 1F), effect                                                  counteracted overexpression SIRT1 (Figure 1G).  data indicate                                                  SIRT1 binds contribute silencing major satellite repeats. <br />  <br />                               Global changes promoter-associated SIRT1 response oxidative stress <br />                                                  Given role yeast Sir2 silencing HM loci, hypothesized mammalian SIRT1 <br />                                                  regulate number protein-coding genes. test  used ChIP combination <br />                                                  genome-wide promoter tiling array (ChIP chip) identify putative SIRT1 target <br />                                                  genes. Immunoprecipitated DNA untreated H2O2-treated cells hybridized <br />                                                  NimbleGen MM5 array detect SIRT1- H1AcK26-associated promoter segments <br />                                                  (Figure 2A). Based gene ontology  , overrepresented groups included genes                                                  chromatin assembly transcriptional repression (e.g. methyl-CpG binding proteins 2, -3 <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                           Page 4 <br />  <br />  <br />                                                  -4, SIRT7, histone genes), ubiquitin-regulated protein degradation, cell <br />                                                  cycle regulation (Figure 2B, S4A Table S1). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   determined SIRT1-binding H1 acetylation patterns change response                                                  oxidative stress. Paralleling yeast response, oxidative stress caused redistribution                                                  SIRT1 chromatin level (Figure 2A), 10% SIRT1-associated <br />                                                  promoters overlapped untreated H2O2-treated cells (Figure S4B),                                                  resulting binding pattern did cluster functional groups, indicating shift                                                  random SIRT1 distribution (Figure 2B). significant negative correlation <br />                                                  loss SIRT1 binding H1K26 acetylation (&#207;&#8225;2=12.12, P&lt;0.001, Figure 2C), <br />                                                  supporting notion SIRT1 regulates genes,  H1 <br />                                                  deacetylation.  observations indicate SIRT1 associates considerably <br />                                                  genes currently known oxidative stress causes major change SIRT1 <br />                                                  distribution genome. <br />  <br />                               Oxidative stress causes SIRT1 dependent transcriptional deregulation <br />                                                  determine redistribution promoter-associated SIRT1 affects transcription,                                                  performed combination microarray-based transcriptional profiling (Table S2), q- <br />                                                  RT-PCR analysis SIRT1-associated non-associated genes, comparing standard <br />                                                  growth conditions H2O2 treatment, SIRT1 overexpression.                                                  identified diverse set SIRT1-associated genes significantly increased expression <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  H2O2 treatment, coincident SIRT1 release, including regulators metabolism <br />                                                  (pisd, prkag3, gstz1), <span id='am-2' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-3' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="ion" datatype="xsd:string"></span>ion</span> transport (slc9a9), cell motility (tcte3) G-protein signaling <br />                                                  (farp2). genes, 75% repressed modest overexpression SIRT1 (Figure 3) <br />                                                  H1K26 acetylation inversely correlated SIRT1 binding (Figure S5A). <br />                                                  Demonstrating specificity, SIRT1 overexpression did repress induction genes                                                  associated SIRT1 (Figure 3B). reduction SIRT1 binding                                                  confirmed random selection promoters q-PCR using distinct types                                                  genotoxic stress: H2O2 methyl-methane-sulfonate (MMS) (Figure S5B). Inactivation                                                  SIRT1 resulted transcriptional deregulation SIRT1-associated loci, <br />                                                  corroborating regulatory role SIRT1 sites (Figure S5C). SIRT1                                                  recently shown negatively regulate HDAC1   function transcriptional <br />                                                  activator repressor (Binda et al., 2008), explain knockdown SIRT1 <br />                                                  does alter gene expression direction oxidative stress.                                                   results indicate DNA damage induces change SIRT1 distribution affects                                                  expression individual genes, effect suppressed increasing SIRT1 <br />                                                  levels. <br />  <br />                               SIRT1 recruited DNA double strand breaks <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Given yeast Sir complex redistributes silent loci sites DNA repair (Martin <br />                                                  et al., 1999; Mills et al., 1999), wondered SIRT1 behaves way.                                                  isolating chromatin-bound non-chromatin-bound protein fractions untreated                                                  H2O2-treated cells, observed substantial, transient, dose-dependent increase                                                  chromatin-associated SIRT1. Treatment MMS produced similar effect, supporting                                                  idea SIRT1 recruited damaged DNA promoters soluble nuclear <br />                                                  pool (Figure 4A, Figures S6A S6B). <br />  <br />                                                  yeast, recruitment Sir proteins DNA double-strand break (DSB) requires DNA <br />                                                  damage signaling MEC1, ortholog mammalian PI3-kinases ATR/ATM <br />                                                  (Martin et al., 1999; McAinsh et al., 1999; Mills et al., 1999). test DNA damage <br />                                                  signaling required SIRT1 redistribution, cells treated PI3 kinase <br />                                                  inhibitor wortmannin ATM inhibitor KU55933 prior H2O2 exposure. increase <br />  <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                          Page 5 <br />  <br />  <br />                                                  chromatin-associated SIRT1 strongly reduced compounds (Figure 4B).                                                  investigated SIRT1 recruitment dependent histone H2AX,                                                  immediate targets ATM (Burma et al., 2001). H2AX-deficient ES cells 2&#226;&#8364;&#8220;3 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  fold reduction chromatin-bound SIRT1 response H2O2 MMS <br />                                                  compared cells reconstituted wild-type H2AX (Figure 4C Figure S6C, (Xie et <br />                                                  al., 2004)).  efficient recruitment SIRT1 damaged DNA requires DNA damage <br />                                                  signaling ATM H2AX. <br />  <br />                                                  Rad51, critical component homologous DSB repair (HR) process, recruited                                                  chromatin concomitant SIRT1 response H2O2 exposure DSB-inducing <br />                                                  ionizing irradiation  . Importantly, Rad51 recruitment impaired absence                                                  SIRT1 (Figure 4D&#226;&#8364;&#8220;E). data indicated SIRT1 physically associate                                                  DSBs perform key step DNA repair process. <br />  <br />                                                  test  employed cell-based DSB induced transient <br />                                                  transfection vector encoding endonuclease SceI (Weinstock et al., 2006).                                                  used previously identify DSB-binding patterns number DNA <br />                                                  repair enzymes (Rodrigue et al., 2006). Nbs1, component MRN complex served                                                  positive control binding (Berkovich et al., 2007). Concomitant recruitment Nbs1, <br />                                                  SIRT1 binding detected 24 h transfection SceI DSB site. Similar <br />                                                  kinetics recently shown interaction SIRT1 DSBs CpG-rich <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  DNA (O&apos;Hagan et al., 2008). <br />  <br />                                                  Interestingly, association Nbs1 break site delayed strongly reduced <br />                                                  absence SIRT1 (Figures 5A B). similar effect observed recruitment <br />                                                  Rad51 (Figure 5C). data SIRT1 physically associates sites DNA <br />                                                  damage agree recent studies indicating chromatin-modifying enzymes                                                  recruited DSB prepare site incoming DNA repair factors (Botuyan et al., <br />                                                  2006; Tamburini Tyler, 2005). <br />  <br />                               SIRT1 required efficient DSB repair genomic stability <br />                                                  DSB repair occurs major pathways: HR non-homologous end-joining <br />                                                  (NHEJ). Rad51 data suggested role SIRT1 HR, measured                                                  determining repair frequency defective GFP gene restored wild-type <br />                                                  SceI transfection  Figure S7A). Inhibition SIRT1 activity NAM,                                                  specific SIRT1 inhibitor S91211 (Solomon et al., 2006), stable knockdown SIRT1 <br />                                                  resulted 25% 50% reduction GFP+ cells, indicating SIRT1 necessary                                                  efficient HR-mediated repair (Figure 5D, Figure S7B). Repair NHEJ, prominent DSB <br />                                                  repair pathway G1 post-mitotic cells, reduced assay  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lesser extent (Figure S7C). <br />  <br />                                                  test SIRT1 necessary maintenance genomic stability, SIRT1 knock- <br />                                                  ES cell lines transiently exposed H2O2 analyzed metaphase <br />                                                  aberrations. absence genotoxic stress, significant difference                                                  chromosomal stability SIRT1-deficient control cells, consistent previous <br />                                                  findings (Chua et al., 2005). Strikingly, H2O2 treatment caused significant increase                                                  chromosomal aberrations specifically SIRT1 deficient cells (Figure 5E). frequency                                                  chromatid breaks comparable knock control ES cells,                                                  number chromosomal fusions, particular dicentric chromosomes Robertsonian <br />                                                  translocations, significantly higher absence SIRT1 (Table S3). Chromosome <br />                                                  fusions generally result aberrantly repaired DNA breaks, supporting role <br />                                                  SIRT1 DSB repair. general DSB defect increase chromatid <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                            Page 6 <br />  <br />  <br />                                                  breaks, predominantly detected stably inherited aberrations fusions                                                  metaphases analyzed 48 h (approx. divisions) exposure H2O2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Increased SIRT1 levels protect irradiation-induced cancer mice <br />                                                  test SIRT1 promotes genomic stability vivo, used p53+/&#226;&#710;&#8217; mice (Jacks et al., <br />                                                  1994),  exposed  high incidence malignant thymic lymphoma <br />                                                  arising loss heterozygosity (LOH) p53 locus (Kemp et al., 1994). explore <br />                                                  role SIRT1 damage-induced LOH, p53+/&#226;&#710;&#8217; mice fed SIRT1 activator <br />                                                  resveratrol (Baur Sinclair, 2006) weeks prior irradiation. Resveratrol-treated <br />                                                  animals showed 24% increase survival (&#207;&#8225;2=5.0, P = 0.025, Wilcoxon rank sum test)                                                  45% reduction frequency fatal thymic lymphomas (Figures 6A B), resulting <br />                                                  tumor spectrum highly reminiscent non-irradiated p53+/&#226;&#710;&#8217; mice (Donehower et al., <br />                                                  1992; Jacks et al., 1994), <br />  <br />                                                  Given effects resveratrol limited SIRT1 activation, generated                                                  SIRT1 transgenic mouse strain (SIRT1STOP), allows Cre-mediated temporal                                                  tissue-specific overexpression SIRT1 deletion transcriptional STOP cassette <br />                                                  (Firestein et al., 2008). SIRT1STOP strain crossed p53+/&#226;&#710;&#8217; mice carrying                                                  interferon (IFN) type inducible Mx-cre transgene (Kuhn et al., 1995). Mice Mx-cre- <br />                                                  dependent, IFN-inducible SIRT1 overexpression referred &quot;MISTO mice&quot;. Injection <br />                                                  IFN inducer poly poly(C), MISTO mice increased expression SIRT1 bone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  marrow lymphocyte progenitors (3&#226;&#8364;&#8220;4 fold) mature B T cells (Figures 6C                                                  6D). Thymocytes intrinsically high levels SIRT1. weeks SIRT1 induction, <br />                                                  p53+/&#226;&#710;&#8217; MISTO mice p53+/&#226;&#710;&#8217; controls exposed 4 Gy &#206;&#179;-irradiation, monitored <br />                                                  tumor-related deaths. Deletion STOP cassette tumor tissues examined                                                  Southern blot q-PCR (Figure S8A B). <br />  <br />                                                  mean survival MISTO mice ~46% greater control animals (Figure 6E, <br />                                                  &#207;&#8225;2=5.68, P=0.017, Log-Rank test; &#207;&#8225;2=4.9, P=0.027, Wilcoxon rank sum test). Furthermore, <br />                                                  frequency fatal thymic lymphoma reduced 45% MISTO mice, consistent <br />                                                  finding resveratrol-treated animals (Figure 6F). Tumor cells exhibited LOH                                                  p53 locus control MISTO mice (Figure S8C D).  data <br />                                                  indicate increasing SIRT1 activity quantity increase genomic stability vivo                                                  suppress tumorigenesis. <br />  <br />                               SIRT1-associated genes deregulated aged brain <br />                                                  observations indicated SIRT1, like yeast counterpart, recruited DSBs                                                  response genotoxic stress, resulting loss silencing repetitive DNA elements NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  SIRT1-regulated genes. test transcriptional changes relevant aging,                                                  compared transcriptional changes caused oxidative stress associated                                                  aging using q-RT-PCR microarrays. chose examine neocortex age- <br />                                                  related gene expression changes characterized tissue (Lee et al., 2000; <br />                                                  Lu et al., 2004). Interestingly, thirds SIRT1-bound genes derepressed                                                  oxidative stress vitro (Figure 3) derepressed aging (Figures 7A D). <br />                                                   SIRT1-target genes identified ChIP chip vitro  Figure 2)                                                  significantly overrepresented age-upregulated genes (&#207;&#8225;2=7.3, P=0.0055; Figure 7B <br />                                                  Table S4). abundance major satellite repeat transcripts increased <br />                                                  significantly age (Figure S9A). <br />  <br />                                                  gain mechanistic insights, tested overexpression SIRT1 brain                                                  delay transcriptional changes, paralleling ability SIR2 suppress expression <br />                                                  yeast mating type genes. SIRT1STOP transgenic mouse crossed brain- <br />  <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                           Page 7 <br />  <br />  <br />                                                  specific Cre-driver (Nestin-cre) generate Nestin-cre; SIRT1STOP mice, referred                                                  &quot;NeSTO mice.&quot; Transcript levels SIRT1-bound genes age-upregulated non- <br />                                                  SIRT1 bound (control) genes examined NeSTO mice comparable SIRT1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  overexpression (~10 fold, Figure 7C, Figure S9B C) 8&#226;&#8364;&#8220;10 18&#226;&#8364;&#8220;19 months. <br />                                                  Strikingly, transcriptional derepression exclusively observed SIRT1- <br />                                                  associated genes completely suppressed aged NeSTO mice (Figure 7D).                                                  non-SIRT1 associated genes change ~19 months age inflammatory <br />                                                  markers associated gliosis, characteristic brain aging (Nichols et al., 1995).                                                  upregulation repressed NeSTO mice, indicating secondary, <br />                                                  beneficial effects SIRT1 overexpression. effect SIRT1 overexpression mice <br />                                                  older 19 months age known.  results confirm SIRT1- <br />                                                  bound genes derepressed aging brain SIRT1 overexpression suppress <br />                                                  age-related changes. <br />  <br />                               Discussion <br />                                                  discovery yeast Sir complex relocalizes aging response DNA <br />                                                  damage (Kennedy et al., 1995; Kennedy et al., 1997; Martin et al., 1999; McAinsh et al., <br />                                                  1999; Mills et al., 1999) led propose model aging caused,                                                   DNA-damage induced reorganization chromatin, phenomenon termed                                                  &quot;RCM response,&quot; redistribution chromatin modifiers (Imai Kitano, 1998; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Oberdoerffer Sinclair, 2007; Villeponteau, 1997).  present evidence                                                  RCM response exists mammals, contribute age-related changes gene <br />                                                  expression. <br />  <br />                               SIRT1 recruited DSBs required efficient DNA repair <br />                                                  role chromatin modifiers DNA repair convincingly shown yeast <br />                                                  (Tamburini Tyler, 2005) mammals (Bassing et al., 2002; Bhaskara et al., 2008; <br />                                                  Botuyan et al., 2006; Celeste et al., 2002; Xie et al., 2004).  SIRT1 binds <br />                                                  hundreds promoters mouse genome, binding pattern altered                                                  response genotoxic stress, coincident relocalization SIRT1 damaged DNA. <br />  <br />                                                  recruitment SIRT1 DSBs reminiscent yeast Sir2 histone modifying <br />                                                  enzymes bind DSB, resulting epigenetic changes surrounding break site <br />                                                  (Chen et al., 2008; Tamburini Tyler, 2005). shown methylation H4                                                  lysine 20, chromatin alterations surrounding DSB promote recruitment DNA <br />                                                  repair factors 53BP1 (Botuyan et al., 2006). DSB associated SIRT1 serve                                                  deacetylate histones DNA repair factors. Consistent  SIRT1 shown                                                  directly interact Nbs1 (Yuan et al., 2007) proteins coexist <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  break site (Figure 5B). SIRT1 recruitment DSBs initiated molecular <br />                                                  level require study, data indicate ATM-mediated signaling                                                  H2AX phosphorylation important (Figure 4B 4C). demonstrate SIRT1                                                  required efficient DSB repair genome maintenance response oxidative stress. <br />                                                  direct association DSBs finding SIRT1-deficient cells                                                  checkpoint proficient suggest DNA repair defect checkpoint defect (Cheng et <br />                                                  al., 2003). Consistent role SIRT1 DNA repair, high doses                                                  reported induce cell cycle delay SIRT1- deficient fibroblasts (Yuan et al., 2007), <br />                                                  reminiscent SIRT6-deficient cells, DNA repair defect prolonged S <br />                                                  phase checkpoint defect (Mostoslavsky et al., 2006). <br />  <br />                                                  Based studies showing SIRT1 deacetylate inactivate p53, researchers <br />                                                  predicted SIRT1 promote tumorigenesis vivo (Lim, 2006). contrast, p53+/&#226;&#710;&#8217; <br />                                                  mice increased SIRT1 activity susceptible irradiation-induced thymic <br />  <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                            Page 8 <br />  <br />  <br />                                                  lymphoma 20&#226;&#8364;&#8220;46% greater mean lifespan (Figure 6). appealing speculate                                                  increased SIRT1 levels protect irradiation-induced LOH increasing DSB repair <br />                                                  efficiency. finding MISTO transgenic mice overexpressed SIRT1 early <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  lymphocyte precursors later stages thymic T cell development (Figures 6C <br />                                                  6D) consistent hypothesis argues protective role SIRT1                                                  later stages tumor progression. rule   additional protective <br />                                                  mechanisms contributed protection tumorigenesis. <br />  <br />                               Derepression SIRT1-associated loci response oxidative stress aging <br />                                                  SIRT1 previously reported contribute formation facultative <br />                                                  heterochromatin (Vaquero et al., 2007). data SIRT1-mediated repression                                                  occur constitutive heterochromatic regions pericentromeric DNA,                                                  number specific genes, prominently regulators chromatin assembly                                                  transcription. Oxidative stress reduces association SIRT1 repetitive loci <br />                                                  (Figure 1E) individual genes (Figure 2). surprisingly, change SIRT1 <br />                                                  localization associated functional consequences silencing heterochromatic <br />                                                  repetitive DNA (Figure 1F) expression individual genes (Figure 3). cells <br />                                                  aging brain, majority changes counteracted overexpressing <br />                                                  SIRT1 (Figure 3 Figure 7D). Loss SIRT1 binding,  did lead                                                  transcriptional derepression, indicating chromatin alterations necessary                                                  sufficient cause transcriptional deregulation. work required NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  identify chromatin modifiers transcription factors involved RCM <br />                                                  response. HDAC1 HDAC2 candidates RCM response given increased <br />                                                  binding chromatin DNA damage  Figure S10). <br />  <br />                                                  wealth data changes gene expression age catalogued                                                  recent years, debate physiological relevance (Oberdoerffer                                                  Sinclair, 2007; Vijg, 2004). Transcriptional changes elderly beneficial <br />                                                  defense response cellular damage (Niedernhofer et al., 2006). Conversely, age-related <br />                                                  changes gene expression deleterious, exist weak forces                                                  natural selection older ages.  transient RCM response young individuals                                                  likely beneficial, constitutive triggering RCM actually contribute aging.                                                  duality clearly evidenced yeast, transient derepression silent HM loci <br />                                                  increases HR resistance DNA damaging agents (Lee et al., 1999) constitutive <br />                                                  derepression HM loci old cells cause sterility (Smeal et al., 1996). mammals, <br />                                                  defective DNA repair associated premature aging (Lombard et al., 2005). <br />                                                  Conversely, lack DNA damage response beneficial situations chronic <br />                                                  DNA damage telomere dysfunction (Choudhury et al., 2007; Schaetzlein et al., 2007). <br />                                                  Furthermore, exposure genotoxic stress early life accelerate changes gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  expression associate age-related diseases amyloidogenesis (Wu <br />                                                  et al., 2008). Interestingly, constitutive overexpression set age- <br />                                                  deregulated SIRT1 target genes promotes apoptosis primary neurons (Figure S11), <br />                                                  work needed determine physiological relevance observation. <br />  <br />                               Perspective <br />                                                  identified SIRT1 participant stress response provide direct link <br />                                                  DNA damage gene expression changes occur aging. DNA <br />                                                  damage previously suggested directly inhibit gene repression (Lu et al., 2004), <br />                                                  data explains ostensibly undamaged genes deregulated time.                                                  speculate RCM response cause permanent changes chromatin <br />                                                  structure sites repair, leading stable transcriptional changes accumulate                                                  lifetime (Oberdoerffer Sinclair, 2007).  recent report showed SIRT1 <br />  <br />  <br />                                                           Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                          Page 9 <br />  <br />  <br />                                                  recruitment DNA break CpG islands result DNA methylation changes                                                  heritable gene silencing (O&apos;Hagan et al., 2008). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  age-related transcriptional changes limited SIRT1-regulated loci <br />                                                  (Botuyan et al., 2006; Tamburini Tyler, 2005), multiple mechanisms involving                                                  chromatin modifiers likely involved. sets SIRT1 apart link calorie <br />                                                  restriction (CR), dietary regimen slows aging mammals (Sinclair, 2005). Given                                                  increased SIR2/SIRT1 expression suppress genomic instability gene expression <br />                                                  alterations, CR promotes genomic stability delays aging mammals                                                  similar mechanism. <br />  <br />                               Experimental Procedures <br />                               Yeast experiments <br />                                                  experiments log phase yeast growing YPD (2% glucose). HMR::GFP cells                                                  HMR::GFP 2xSIR2 cells (Park et al., 1999) exposed H2O2 30 min, followed                                                  4 h recovery period, analyzed FACS. Replicative lifespans rDNA recombination <br />                                                  analyses performed described (Lamming et al., 2005). rDNA recombination, <br />                                                  WT Sir2o/e W303AR cells treated 30 min H2O2 (1.5 mM) prior plating. <br />                                                  lifespan analysis, WT 2xSIR2 cells plated regular YPD agar agar <br />                                                  supplemented H2O2 (1 mM). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Cell culture treatments <br />                                                  Mouse ES cells cultured gelatinized tissue culture dishes described <br />                                                  (Kanellopoulou et al., 2005). Stable SIRT1 overexpressing V6.5&#226;&#8364;&#8220;C10 ES cells                                                  obtained SIRT1STOP ES cells (Firestein et al., 2008) Cre-mediated deletion                                                  loxP-flanked STOP cassette. SIRT1 knock ES cells generated lentiviral <br />                                                  infection V6.5&#226;&#8364;&#8220;C10 ES cells (Beard et al., 2006). Cells infected                                                  luciferase-specific SIRT1-specific shRNA lentiviral vector (Araki et al., 2004). <br />                                                  Generation SIRT1 knock ES cells respective wild-type cells described (Chua et <br />                                                  al., 2005). Cells &#206;&#179;-irradiated (4 Gy, 137Cs irradiator, Shepherd Associates)                                                  treated H2O2 MMS 1 h 37&#194;&#176;C. Treatment 50 &#194;&#181;M wortmannin (Sigma)                                                  25 &#194;&#181;M KU55933 (AstraZeneca) started 2&#226;&#8364;&#8220;3 h prior treatments. NAM (Sigma, <br />                                                  25 mM), TSA (Sigma, 0.1 &#194;&#181;M) sirtinol (100 &#194;&#181;M) added indicated times. <br />  <br />                                                  DRGFP-transgenic U2OS cells SceI- Ds-red-encoding plasmids pCBASceI                                                  pCAGGS-Dsred described (Weinstock et al., 2006). Stable SIRT1 knock                                                  control lines generated lentiviral gene transfer using shRNA vectors Open <br />                                                  Biosystems. Transfection pCBASce pCAGGS-Dsred performed using Fugene 6 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  transfection reagent (Roche). indicated, cells treated NAM (10 mM)                                                  S91211/EX-527 (50 &#194;&#181;M, (Solomon et al., 2006)) starting 2 h prior transfection. 48 <br />                                                  h, cells analyzed FACS. <br />  <br />                               Chromatin immunoprecipitation (ChIP) <br />                                                  Approximately 107 cells cross-linked 1% <span id='am-37' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> 15&#226;&#8364;&#8220;20 minutes                                                  37&#194;&#176;C quenched <span id='am-13' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-14' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span>. Cell lysates sonicated (Branson sonifier)                                                  incubated overnight rabbit &#206;&#177;-Sir2&#206;&#177; (Upstate), &#206;&#177;-Nbs1 (Novus), &#206;&#177;-Rad51 (Calbiochem) <br />                                                  &#206;&#177;-H1AcK26 (Vaquero et al., 2004). Immunoprecipitation performed described <br />                                                  (Upstate) eluates purified using QIAgen PCR purification, followed (q-) PCR <br />                                                  analysis. <br />  <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 10 <br />  <br />  <br />                               RNA isolation, reverse transcription, PCR analysis <br />                                                  Total RNA isolated using Trizol reagent (Invitrogen), followed DNase treatment <br />                                                  (Turbo Dnase free, Ambion). RNA reverse transcribed Invitrogen&#226;&#8364;&#8482;s Thermoscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  RT PCR using combination random hexamers oligo dT primers. PCR                                                  performed using Taq Platinum (Roche). q-PCR performed using SYBR green-based <br />                                                  detection Roche Light Cycler Roche LC480. Table S5 primers PCR <br />                                                  conditions. <br />  <br />                               Microarray analysis statistics <br />                                                  Nimblegen promoter tiling array analysis (Nimblegen-Roche), ChIP DNA                                                  amplified ligation-mediated PCR. IP input DNA samples labeled using 9mer <br />                                                  Cy3 Cy5-labeled primers. IP total DNAs hybridized NimbleGen <br />                                                  MM5 minimal promoter tiling array analyzed using NimbleScan software (NimbleGen). <br />                                                  Peak data files generated searching 4 probes significant <br />                                                  enrichment using 500 bp sliding window. peak assigned false discovery rate <br />                                                  (FDR) score based randomization. Gene ontology cluster analysis SIRT1-associated <br />                                                  promoters (FDR &#226;&#8240;&#164; 0.1) performed using BiNGO plug Cytoscape v2.5. <br />  <br />                                                  Affymetrix expression analysis, total RNA hybridized mouse genome 430 2.0 <br />                                                  array. CEL files analyzed significance fold changes experimental <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  groups using DChip software. comparison Nimblegen Affymetrix array data, <br />                                                  Gene IDs arrays matched using DAVID. Analysis limited genes                                                  highly significant SIRT1 promoter enrichment (FDR &lt; 0.005) transcriptional increase                                                   &#226;&#8240;&#165; 10%, P &#226;&#8240;&#164; 0.005). &#207;&#8225;2-based P values calculated using Pearson&apos;s Chi-squared test                                                  Yates&apos; continuity correction. Microarray data deposited NCBI&apos;s Gene <br />                                                  Expression Omnibus accessible GEO Series accession number GSE13121 <br />                                                  (http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE13121). <br />  <br />                               Cellular fractionation Western blotting <br />                                                  Chromatin-bound protein purified described (Cha et al., 2005). Primary antibodies <br />                                                  rabbit &#206;&#177;-Sir2&#206;&#177;, &#206;&#177;-histone H3, &#206;&#177;-Histone H4 (Upstate), rabbit &#206;&#177;-YY1 (Santa Cruz), <br />                                                  mouse &#206;&#177;-GAPDH (Chemicon) rabbit &#206;&#177;-Rad51 (Dr. R. Scully); &#206;&#177;-rabbit &#206;&#177;-mouse <br />                                                  HRP-coupled secondary antibodies GE Healthcare. <br />  <br />                               Metaphase analysis <br />                                                  Metaphase spreads performed previously described (Mostoslavsky et al., 2006).                                                  80 metaphases genotype scored experiment. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Mouse breeding treatments <br />                                                  mice housed pathogen-free. SIRT1STOP (Firestein et al., 2008), Mx-cre (gift                                                  Dr. K. Rajewsky), Nestin-cre p53&#226;&#710;&#8217;/&#226;&#710;&#8217; mice (Jackson Laboratory) crossed                                                  obtain indicated genotypes. Experimental animals C57BL/6 &#195;&#8212; 129/Sv mixed <br />                                                  genetic background. Mx-cre induction, mice injected 400 &#194;&#181;g poly poly(C) <br />                                                  (Amersham) 6&#226;&#8364;&#8220;10 weeks age (Kuhn et al., 1995). tumor studies, mice &#206;&#179;- <br />                                                  irradiated 10&#226;&#8364;&#8220;14 days single dose 4 Gy. Animals sacrificed                                                  moribund. obvious, mice submitted necropsy identify cause                                                  death. Deaths related tumors mice decomposed analysis censored. <br />                                                  Kaplan-Meier survival curves generated separate cohorts irradiated <br />                                                  animals using JMP7 software. Resveratrol fed 2.4 mg/kg food previously <br />                                                  described (Baur et al., 2006). cohorts pooled survival analysis. <br />  <br />  <br />  <br />                                                          Cell. Author manuscript; available PMC 2010 April 13. <br />  <br />                                                  Figure 1. <br />                                                  Oxidative stress reduces Sir2/SIRT1-mediated repetitive DNA silencing yeast                                                  mammalian cells.  C). Effects H2O2 yeast aging.   H2O2 causes loss silencing <br />                                                  mating type loci. Shown fraction GFP+ cells 2xSIR2 WT cells. Unless <br />                                                  noted, P values based Student&#226;&#8364;&#8482;s tailed t test #P&#226;&#8240;&#164; 0.1, *P&#226;&#8240;&#164; 0.05, <br />                                                  **P&#226;&#8240;&#164; 0.01, ***P&#226;&#8240;&#164; 0.001. (B) Frequency H2O2-induced rDNA recombination                                                  absence presence extra Sir2 (WT Sir2 o/e). (C) Replicative lifespan WT                                                  2xSIR2 cells grown absence presence 1 mM H2O2. (D) Mouse SIRT1 binds                                                  repetitive genomic DNA. ChIP SIRT1 control immunoglobulin (Ig) major satellite <br />                                                  repeats absence presence NAM (25 mM) (E) q-PCR analysis ChIP <br />                                                  experiments using antibodies specific SIRT1 H1AcK26. ES cells left untreated <br />                                                  treated H2O2 (2 mM) NAM 1 h. (F) Oxidative stress increases transcription                                                  satellite repeat DNA. ES cells treated NAM 24 h H2O2 1 h, <br />                                                  followed 23 h recovery. (G) Cells targeted extra copy SIRT1 control cells <br />                                                  treated H2O2 analyzed (F). inset shows Western blot SIRT1                                                  WT (black) SIRT1 overexpressing ES cells (white). Data represented mean +/&#226;&#710;&#8217; <br />                                                  SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 2. <br />                                                  Oxidative stress causes major redistribution SIRT1.   Distribution SIRT1                                                  H1AcK26 representative chromosome. ES cells untreated treated                                                  H2O2 1 h, followed ChIP SIRT1, H1AcK26 control Ig. x-axis depicts <br />                                                  probe sets spanning promoters annotated ORFs, signals y-axis reflect                                                  log2 change IP DNA input DNA. (B) Significance overrepresentation selected <br />                                                  gene ontology   groups SIRT1-associated genes H2O2 <br />                                                  treatment. Asterisks indicate significance Benjamini- Hochberg false discovery rate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  correction. (C) Loss SIRT1 binding correlates increased H1K26 acetylation. <br />                                                  Promoters &#226;&#8240;&#165; 2log2 enrichment SIRT1 prior H2O2 treatment, signal <br />                                                  compared non-SIRT1 associated promoters. y-axis shows fraction <br />                                                  promoters &#226;&#8240;&#165; 2-fold increase H1AcK26 enrichment exposure H2O2. <br />                                                  SIRT1 H1AcK26 probe set binding shown promoters. GAPDH <br />                                                  served negative control. <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                       Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 3. <br />                                                  Transcriptional deregulation SIRT1-associated genes response oxidative stress                                                  repressed increasing SIRT1 levels.   SIRT1 expression wild-type (WT) SIRT1 <br />                                                  overexpressing (OE) ES cells. (B) q-RT-PCR analysis putative SIRT1 target genes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  (SIRT1-bound), non-SIRT1-bound control genes &#206;&#178;-actin wild-type (open bars)                                                  SIRT1 overexpressing ES cells (closed bars). Shown fold change expression <br />                                                  compared untreated samples. Data represented mean +/&#226;&#710;&#8217; SEM. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                      Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  SIRT1 recruited chromatin DNA damage ATM-dependent manner.   <br />                                                  Western blot analysis SIRT1 indicated control proteins chromatin-bound                                                  non-chromatin-bound protein fractions ES cells left untreated                                                  treated H2O2 MMS 1 h. (B C) Western blot analysis chromatin-bound <br />                                                  SIRT1 response H2O2 pretreatment wortmannin KU55933 (B), <br />                                                  presence absence H2AX (C). (D E) Recruitment SIRT1 Rad51                                                  chromatin WT SIRT1-deficient (SIRT1-&#206;&#8221;ex4) cells response H2O2  <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                        Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 5. <br />                                                  SIRT1 recruited DNA breaks required efficient DSB repair.   SIRT1 <br />                                                  expression parental U2OS-DRGFP cells, independent shSIRT1 expressing cell lines <br />                                                  (SIRT1-KD) shRFP expressing controls. (B) ChIP analysis SIRT1 Nbs1 binding <br />                                                  DSB shRFP (control) SIRT1 KD cells  . (C) DSB ChIP analysis                                                  Rad51 recruitment using Ds-red-transfected (&#226;&#710;&#8217;) SceI transfected cells (+) (B) 24h <br />                                                  transfection. (D) SIRT1 knock results reduced DSB repair measured                                                  GFP rexpression  Figure S7A). Control SIRT1-KD cell lines                                                    transfected (C) analyzed FACS. (E) Loss SIRT1 causes increased genomic <br />                                                  instability oxidative stress. Untreated H2O2 treated shSIRT1- control shRNA- <br />                                                  expressing ES cells subjected Q-FISH analysis. Telomeres shown red. Arrows <br />                                                  indicate chromatid break (left) fused centromere (right). fraction aberrant <br />                                                  metaphases shown. Data represented mean +/&#226;&#710;&#8217; SEM. <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                        Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 6. <br />                                                  Increased SIRT1 activity alters tumor spectrum increases survival irradiated p53+/&#226;&#710;&#8217; <br />                                                  mice.   Survival irradiated p53+/&#226;&#710;&#8217; mice fed normal resveratrol-supplemented chow <br />                                                  (n=19 25, respectively). Tumor-related deaths recorded days irradiation. <br />                                                  (B) Tumor spectrum mice  , legend lists dominant tumor time death;                                                   tumors detected necropsy. (C) Western blot analysis MACS-purified thymic CD4/CD8 <br />                                                  double-positive (DP) T cells splenic CD8+ T cells MISTO mice (+ Mx-cre)                                                  littermate controls (&#226;&#710;&#8217; Mx-cre). Mice analyzed 14 days Mx-cre induction; Thy: <br />                                                  Thymus, Spl: Spleen. (D) SIRT1 mRNA expression lymphocyte subsets MISTO <br />                                                  mice (closed bars) littermate controls (open bars). BM: Bone marrow, Lin+: Lineage- <br />                                                  positive. (E) Survival MISTO control p53+/&#226;&#710;&#8217; mice (n=12 16, respectively) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  response single dose 4 Gy &#206;&#179;-irradiation. Tumor-related deaths recorded days <br />                                                  irradiation. (F) Tumor spectrum mice (E), legend (B). <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />                             Oberdoerffer et al.                                                                                         Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Figure 7. <br />                                                  Transcriptional deregulation SIRT1 target genes occurs normal aging.   <br />                                                  Expression SIRT1-associated gene Figure 3 neocortex young (5 months) <br />                                                  old (30 months) B6C3F1 mice (n &#226;&#8240;&#165; 6 group), analyzed q-PCR. Shown fold <br />                                                  change expression old relative young mice. P values based student&#226;&#8364;&#8482;s  <br />                                                  tailed t-test. (B) Microarray expression analysis SIRT1 target genes neocortex                                                  young old B6C3F1 mice (n = 5 group). SIRT1 target genes significantly <br />                                                  overrepresented age-upregulated genes (&#207;&#8225;2 = 7.28, P = 0.0055). (C) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Immunofluorescence analysis SIRT1 NeuN expression neocortex NeSTO <br />                                                  mice Nestin-cre littermate controls. (D) q-RT-PCR analysis SIRT1-bound genes                                                    non-SIRT1-bound genes including housekeeping genes (rps16, hprt) genes <br />                                                  upregulated 30 months old mice microarray. NeSTO mice Nestin-cre littermate <br />                                                  controls (Control) analyzed 8&#226;&#8364;&#8220;10 months (young, n = 3&#226;&#8364;&#8220;5 6, respectively) 18&#226;&#8364;&#8220; <br />                                                  19 months age (old, n = 4 5, respectively). P values based student&#226;&#8364;&#8482;s tailed <br />                                                  t-test. Data represented mean +/&#226;&#710;&#8217; SEM. <br />  <br />  <br />  <br />  <br />                                                              Cell. Author manuscript; available PMC 2010 April 13. <br />   <br /> </body></html>