<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, OCt. 1987, p. 4637-4645                                                                       Vol. 169,  10 <br /> 0021-9193/87/104637-09$02.00/0 <br /> Copyright &#194;&#169; 1987, American Society Microbiology <br />  <br />  <br />               dnaA,          Essential Host Gene, TnS Transposition <br />                                     JERRY C. P. YINt WILLIAM S. REZNIKOFF* <br />         Department    Biochemistry,   College Agricultural Life Sciences, University Wisconsin-Madison, <br />                                                    Madison, Wisconsin 53706 <br />                                           Received 6 February 1987/Accepted 17 July 1987 <br />  <br />             Mutations dnaA, essential gene Escherichia coli, decrease frequency transposition Tn5.          insertion mutation dnaA gene does affect Tn5 gene expression.  DnaA protein plays          role transposition reaction itself type cellular regulation transposition. Analysis <br />          mutation DnaA box, outside end IS50, consistent direct interaction          protein bases. IS50 transposition, utilizes end containing DnaA box,          affected dnaA mutations. Overproduction DnaA protein does increase transposition frequencies <br />          wild-type cells, transposase overproduced. <br />  <br />  <br />    procaryotic transposable elements encode protein,              required transposition proficiency. DnaA protein transposase, necessary transposition            does affect pattern TnS gene expression.  <br /> element (27). Transposition requires cellular factors addi-           fore, postulate probably plays direct role tion transposase. complex small, histonelike                 transposition reaction. <br /> proteins shown necessary Mu transposi- <br /> tion vitro (10). complex probably plays mechanistic                          MATERIALS METHODS <br /> role Mu transposition (10). Transposition o<span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;  TnJO <br /> appears require complex host proteins,                    Media, supplies, general techniques. bacteria integration host factor (IHF), clear             grown    LB broth M9 medium supplemented role IHF plays ISJ transposition (14; D. Morisato,                appropriate amino acids vitamins (33). Antibiotic  <br /> personal communication). IHF consists small pro-                  centrations used follows: kanamycin, 40 jig/ml; <br /> teins homology histone protein Hi (11, 26). IHF              tetracycline, 5 ,ig/ml; gentamicin, 5 ,ug/ml; chloramphenicol, involved site-specific integration-excision            10 pLg/ml; ampicillin, 100 ,xg/ml. Antibiotics pur- <br /> bacteriophage k (26).                                                    chased Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span>  radioactive compounds <br />    TnS compound transposon. consists insertion            Amersham Corp. Restriction enzymes sequences IS5OR IS50L, inverted orientation rela-              New England BioLabs, , Bethesda Research Laborato- <br /> tive  flanking region contains antibiotic          ries, , Promega Biotec. Si nuclease calf intestinal <br /> resistance genes. transposase encoded IS50R (35).              alkaline phosphatase Boehringer Mannheim TnS transposes, outer ends compound                      Biochemicals. T4 polynucleotide kinase P-L <br /> transposon used. bases constitute minimal                Biochemicals,  large fragment DNA polymerase functional outer-end sequence defined (22).               T4 DNA ligase gifts M. Cox, F. Pugh, B. insertion sequence IS50 transposes, outer end                Schutte. recombinant DNA experiments  inner end IS50 used substrates.                    formed described previously (31). <br /> sequence requirements inner end                       Strains, bacteriophages, plasmids. bacterial <br /> examined (38).                                                           strains, bacteriophages, plasmids used described    host proteins implicated TnS transposi-              Table 1. general, antibiotic selection maintained tion. DNA gyrase shown needed                 times bacterial strains containing plasmids insertions <br /> target molecule transposition reaction supercoiled                transposons chromosome. Strains constructed <br /> (20). DNA polymerase required,                  described Miller (33) noted. <br /> certain experimental conditions (39, 44). unclear               parental strain D47.00 (CSH26 [33]). RNase functions DNA polymerase needed TnS                  H::Tn3 (18) marker introduced P1 transduction, <br /> transposition (44).                                                      selecting ampicillin resistance. dnaA::TnJO marker <br />    sequences outer end Tn5 necessary              moved mediated transduction described previ- transposition (22) contain homology repeated se-                ously (28). Tetracycline-sensitive derivatives selected <br /> quence Escherichia coli origin replication (45).          technique Bochner et al. (7). <br /> sequence referred DnaA box. Purified DnaA                   dnaA mutants monitored loss function <br /> protein, product essential host gene (17), binds            combination techniques.   Tetracycline-sensitive sequence, E. coli origin outer end           derivatives selected shown incapable of TnS (13).                                                             maintaining pSC101. (ii) fraction II crude extract    paper investigate requirement DnaA             prepared used measure oriC replication programmed <br /> protein TnS transposition vivo.                 M13oriC template (13). dependency DnaA                                                                          shown addition purified DnaA protein (kindly <br />   * <br />     Corresponding author.                                                provided R. Fuller). (iii) Southern blot analysis   t Present address: Sino-American Biotechnology Company,                performed described previously (28), Luoyang, Henan, People&apos;s Republic China.                              plasmid pRB100 (8) used probe. <br />                                                                   4637 <br />  4638       YIN REZNIKOFF                                                                                             J. BACTERIOL. <br />  <br />                                       TABLE 1. Strains, phage, plasmids used study <br />  Strain, phage, plasmid                                      Genotype description                                     Source <br /> Strains <br />   D47.00                     pro-lac) ara                                                                              J. Miller <br />   D47.20                     D47.00 RNase H::Tn3                                                                         study <br />   D47.22                     D47.20 dnaA::850 TnIO                                                                       study <br />   D47.26                     D47.22 tetracycline-sensitive derivative                                                    study <br />   D47.30                     D47.20 ilv::Tn5                                                                             study <br />   D47.32                     D47.30 dnaA::850 TnIO                                                                       study <br />   D47.33                     D47.32 tetracycline-sensitive derivative                                                    study <br />   D48.09                     D47.00 dam::Tn9 ilv::Tn5 (X84)                                                              study <br />   D48.10                     D47.20 dam::Tn9                                                                             study <br />   D48.15                     D48.10 dnaA::850 TnJO                                                                       study <br />   NS294                      AlacX74 bglR ilvO rpsL att::P211                                                             Wright <br />   NS366                      NS294 tna::TnJO dnaA366                                                                      Wright <br />   DC204                      pro-lac) leu thy <span id='am-17' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-18' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>gly</span> drm CycA supE42 Hfr (81&apos;) tna::TnIO                                  study <br />   DC204 dnaA                 DC204 dnaA::850 TnlO                                                                 study <br />   12.00                      Hfl-                                                                                        B. Belfort <br />   168.01                     galK (X c1857 ABam AM1 A8) ilv::TnS/pOXgen                                              study <br />   169.01                     168.01 X c1857 N7 N53 replaces X c1857 ABam AM1 A8                                   study <br />   B7.21                      D47.00 rpsL rpsE gyrA polA XT                                                               R. Johnson <br />   RZ211/F,lq::TnI5         D47.00 rpsL recA56/F&apos; pro lacJq::TnS (X b519 b515 cI857 nin5 S7)                            R. Johnson <br /> Phages <br />   X84                        X b519 b515 cI857                                                                           M. Syvanen <br />   X299                       X 121 ninS  EcoRI sites)                                                                 J. Way <br />   X500                       X b221 cI857 029 P80 rex::TnS                                                               N. Kleckner <br /> ColEl-based plasmids <br />   pRZ102                     ColE1::Tn5 (25)                                                                             R. Jorgenson <br />   pRZ305                     Protein fusion p, P2 ,B-galactosidase                                             R. Johnson <br />   pRZ464                     IS50R outer (1 15) base pairs IS50L flanking tetracycline resistance (22)     R. Johnson <br />   pRZ466                     IS50R outer (1 21) base pairs IS50L flanking tetracycline resistance (22)     R. Johnson <br />   pRZ466-11A                 pRZ466 mutation dnaA box                                                      R. Johnson <br />   pRZ914                     Derivative pRZ102 fully functional contains gene codes          M. Krebs <br />                                gentamicin resistance <br /> pBR322-based plasmids <br />   pBR25                      lac promoter driving functional dnaA gene (8)                                               R. Braun <br />   pRB36                      lac promoter driving nonfunctional dnaA gene (8)                                            R. Braun <br />   pRB100                     Clone functional dnaA gene (8)                                                       R. Braun <br />   pRB101                     Clone nonfunctional dnaA gene (8)                                                      R. Braun <br />   pRZ1095                    Gentamicin-resistant derivative pRB100                                                   study <br />   pRZ1096                    Gentamicin-resistant derivative pRB101                                                   study <br />   pBF110                     pL-driven dnaA gene (13)                                                                    R. Fuller <br />   pBF1509                    pL-driven dnaA gene (B. Funnell, personal communication)                                    B. Funnell <br />   pRZ1073                    Protein-defective derivative IS50 outer end inner end flanking            study <br />                                tetracycline resistance <br />   pRZ1113                    Protein-defective derivative IS50 outer ends flanking tetracycline resistance   study <br />   pRZ986                     Derivative pBR322::TnS wild-type promoters 1S50R replaced K PR        M. Krebs <br />                                promoter. defective element carrying gene codes gentamicin resistance <br />                                located cis transposase-coding region complemented transposition <br />                                assays. <br /> RSF1010-based plasmids <br />   RSF1010::TnS               RSF1010 TnS transposed                                                         study <br />   pRZ1054                    Transposase-defective derivative RSF1010::TnS                                            study <br />  <br />  <br />    Strains dam growth P1 phage strain             Wisconsin, Madison, 1981) cloned unique PstI contained Tn9 insertion dam gene.                     site plasmid. pRZ1054 derivative RSF1010:: <br /> Transductants selected chloramphenicol resistance             Tn5 does synthesize transposase screened loss methylase function examining               internal deletion coding region. <br /> digests plasmid DNA MboI.                                         pRZ464 pRZ466 described previously (22). <br />    pRZ102 RSF1O1O::TnS described previ-                  pRZ466-11A derivative pRZ466 contains ously (23, 25), pRB100, pRB101, pRB25, pRB36               single T transversion DnaA box end (8). pBF110 pBF1509 kindly provided R. Fuller.             TnS element. created oligonucleotide-directed <br /> pRZ1095 pRZ1096 derivatives pRB100                      mutagenesis, followed sequencing subcloning pRB101, respectively; PstI fragment carrying gentamicin            fragment contained outer-end sequence Tn5 (R. <br /> resistance gene (S. Kagan, Ph.D. thesis, University                 Johnson, personal communication). <br />  VOL. 169, 1987                                                                     dnaA Tn5 TRANSPOSITION               4639 <br />  <br />   pRZ1113 derivative pBR322::ISSOR. inner end        antibiotic-resistant exconjugants subsequent bacterial IS50R parent plasmid replaced small         mating experiment. mating transposition assay fragment contained outer-end sequences IS50.        performed described previously (23), cells tetracycline resistance gene exchanged           grown 34&#194;&#176;C mating mixture DNA internal IS50. resulting plasmid carries            incubated 34&#194;&#176;C. 10-RI portion freshly saturated, <br /> defective element comprising outer ends IS50 flanking     washed culture male female strains mixed gene confers tetracycline resistance. pRZ1073         1 ml broth incubated slow shaking 6 12 h. similar pRZ1113, original inner-end      cells diluted plated appropriate media. sequence IS5OR left intact. plasmid carries     pOXgen derivative F factor used. exconju- defective element; case outer end inner      gants selected LB agar plates containing 200 pug end flank gene confers tetracycline resistance.          spectinomycin ml 10 ,g gentamicin ml. Cells <br /> pRZ914 deleted derivative pRZ102 (M. Krebs,          containing- transposition prodiicts measured LB <br /> sonal communication). contains functional TnS element       agar plates containing 200 jig spectinomycin ml, 50 ,g encodes gentamicin resistance.                             nalidixic acid ml, 5 ,ug gentamicin ml, 40 jig <br />   pRZ986 derivative pBR322::TnS contains         kanamycin ml. Strain B7.21 used female <br /> functional TnS transposase-coding region X        strain matings. <br /> PR promoter control (M. Krebs, personal communication).             mating mixtures male female cells diluted plasmid used cis complementation experi-         plated yield approximately 50 500 colonies ment  Table 5).                                              plate. variation set assays usually   Transposition assays. lambda infection assay mea-          twofold. assays performed triplicate sures TnS transposition frequency X500, replica-      given day, numbers quoted represent averages tion-defective, integration-defective lambda phage         sets assays. <br /> carries wild-type TnS. absorption, cells diluted      Si nuclease mapping. RNA isolated strains <br /> LB broth allowed grow 37&#194;&#176;C equivalent       D47.20, D47.22, D48.10, D48.15 carrying pRZ102. cell doublings. mixture plated LB agar       Si mapping procedure, probes, polyacrylamide gel <br /> plates containing kanamycin incubated 37&#194;&#176;C 24 h.      analysis identical described number transposition events measured count-        (J. C. P. Yin, M. P. Krebs, W. S. Reznikoff, J. Mol. <br /> ing number kanamycin-resistant colonies present.      Biol., press). <br /> number drug-resistant colonies linear num- <br /> ber input phage particles. Transposition frequencies                                  RESULTS fraction input plaque-forming particles yield <br /> kanamycin-resistant colonies. multiplicity infection         dnaA mutations TnS transposition. Figure 1 shows adjusted yield 50 500 kanamycin-              features TnS known play direct role resistant colonies plate. standard variation    transposition process. transposase (Pi) encoded factor 3, numbers quoted represent         IS50R (35). base pairs outer end IS50L <br /> average assays triplicate.                   ISSOR sufficient transposition Tn5 (22). <br />   steady-state assay assay X84 infected    required 21 base pairs sequence similar cells carry transposon, serves        sequence appears times E. coli origin target transposition event. Cells grown satu-     replication (45). sequence ISS0 oriC ration LB broth containing antibiotic maltose (0.2%).     shown binding site DnaA protein vitro (12). <br /> Phage lysates mixing fresh plaque             determine dnaA required transposition <br /> phage (containing ca. 106 phage) freshly saturated cells,   TnS vivo, transposition frequencies measured buffer, molten agar. mixture poured TYE        series E. coli strains mutations dnaA gene. plates (33) incubated 8 16 h 37&#194;&#176;C. lysates     dnaA mutants grow, contain extra- collected spun, titer supernatant    genic suppressor dnaA mutation (2, 27, 28, 32). determined (yields typically .1010 phage). trans-       different suppressors dnaA lethality used: position frequency performed measuring fraction       rnh::Tn3 mutation R factor phage P2 plaque-forming phage acquired particular         integrative suppressor (2, 27, 28, 32). Table 1 lists drug resistance marker transposon. measure-          relevant strains, mutations, suppressors. indepen- <br /> ment performed mixing constant number plaque-        dent assays  Materials Methods) used forming phage (typically 107 108) 5 x 108 I2.00 cells    confirm lack functional DnaA protein appropri- plating selective antibiotic-containing plates 60   ate strains. <br /> min growth. frequency lysogeny                data Table 2 effect mutations conditions approached 40%. number antibiotic-             dnaA gene TnS transposition. insertion TnWO resistant colonies shown linearly related       dnaA structural gene decreased transposition 25-fold input phage.                                           compared transposition wild-type strain (lines 1   lambda induction assay determines frequency         3). 13-fold decrease transposition transposition Tn5 prophage measured        RNase H mutant isogenic dnaA+ strain <br /> abundance :TnS-transducing phage infectious unit        (lines 2 3). RNase H mutants extragenic suppressors prophage induction. assay described          dnaA mutations allow secondary replication origins <br /> previously (23), measurement :TnS deter-        used dnaA-independent manner (28). RNase H <br /> mined described  assay used      mutation itself negligible effect TnS trans- <br /> experiments test effect overproducing DnaA         position (lines 1 2). <br /> protein.                                                            following results (Table 2) support conclusion   mating assay determines frequency trans-        dnaA directly affects Tn5 transposition.  mutations position F factor measured occurrence       dnaA decreased transposition frequency, regardless  4640        YIN REZNIKOFF                                                                                                              J. BACTERIOL. <br />  <br />        S&apos; CTGACTdRTGTaC <br />  <br />                                                                                                                  .~~~~~~~~~~~~~~~~~~~~~~~~~~~S <br />                                 IS500                                                                                                                                                               IS50R <br />                                         aTC     ACTATTCTCTGT <br />                                                                                                                                              pi <br />  <br />                                                        OUTER END        5&apos;   CTGACTCTATACAkGTAG <br />                                                        INNER END        S&apos;   CTGtCTCTT&amp;LtCA&amp;AtcTtG <br />                                                        oriC REPEATS              #1 TTATCCACA <br />                                                                                  #2   TTATGACACA <br />                                                                                  #3 TTATCC~AA <br />                                                                                  #4 TTATCCACA <br />   FIG. 1. TnS sequence. corresponding outer-end inner-end sequences IS50 shown compared repeat sequences oriC region E. coli. dnaA sequence present outer end boxed, dam methylation sequences inner <br /> end underlined. transposase protein shown diagramatically. <br />  site TnS transposes. Transposition TnS                        greater effect dnaA mutation infection multicopy plasmids pRZ102 RSF1010::TnS,                            assay  steady-state assay. experiments, derivatives ColE1 RSF1010, affected                        dnaA mutations qualitative mutations dnaA gene (lines 2 3; lines 4 5).                   effect TnS transposition indicated exam- Tn5 transposed bacterial chromosome, dnaA                         ined assay, lambda induction assay (data affected transposition frequency (lines 6 8).                       shown). <br />   Second, effect dnaA demonstrated using                          decrease transposition frequency occurred transposition assays. data presented lines 1 8                  suppressors dnaA mutation. transpo- collected using steady-state assay. assay,                  sition assays repeated integrated R factor (lines <br /> lambda phage, serves target transposition                   11 12) lysogen phage P2 (lines 13 14) reaction, grown lytically cells contain TnS. Lines                  suppress dnaA mutation (30, 34). suppressor <br /> 9 10 results obtained using donor assay                      backgrounds, TnS transposition decreased presence transposition infecting phage molecule                       mutation dnaA gene. bacterial chromosome measured (24).                             Fourth, different null alleles dnaA similar effects steady-state assay (lines 1 8) infection assay (lines                transposition. strain uses P2 integrative <br /> 9 10), transposition decreased dnaA mutants.                     suppressor contains amber mutation dnaA gene (40) know biological significance                     instead TnJO insertion. mutation  <br />                                                    TABLE 2. dnaA mutations transposition <br />    Strain                  Relevant genotype               Suppressor        Starting location                      Transposition                 Relative <br />                                                                                   TnO                              frequency                   frequency <br /> 1. D47.00            Wild type                                          pRZ102                              (3.9 &#194;&#177; 2.0) x 10-7                   1.0 <br /> 2. D47.20            rnh::Tn3                              RNase H           pRZ102                              (2.1 &#194;&#177; 1.3) x 10-7                  0.54 <br /> 3. D47.22            rnh::Tn3 dnaA::TnIO                   RNase H           pRZ102                              (1.5 &#194;&#177; 0.8) X 10-8                  0.04 <br /> 4. D47.20            rnh::Tn3                                  RNase H                RSF1010;:Tn5               (4.4 &#194;&#177; 0.3) x 10-7                  1.0 <br /> 5. D47.22            rnh::Tn3 dnaA::TnIO                       RNase H                RSF1010::Tn5               (1.1 &#194;&#177; 0.9) x i0-8                  0.03 <br /> 6. D47.30            rnh::Tn3                                  RNase H                ilv&apos;::TnS                  (1.2 &#194;&#177; 0.2) x 10-7                  1.0 <br /> 7. D47.32            rnh::Tn3 dnaA::TnlO                       RNase H                ilv::Tn5                   (1.5 &#194;&#177; 0.9) x 10-8                  0.13 <br /> 8. D47.33            rnh::Tn3 dnaA::TnIO Tets                  RNase H                ilv::Tn5                   (1.7 &#194;&#177; 1.5) x lo-8                  0.14 <br /> 9. D47.20            rnh::Tn3                                  RNase H                Lambda                     (1.2 &#194;&#177; 0.2) x 10-4,                 1.0 <br /> 10. D47.22           rnh::Tn3 dnaA::TnJO                       RNase H                Lambda                     (1.0 &#194;&#177; 0.1) X 10-7a               &lt;0.01 <br /> 11. DC204            Hfr (81&apos;) tna::TnIO                       R100.1                 pRZ102                     (8.4 &#194;&#177; 3.6) x 10-6                  1.0 <br /> 12. DC204-           Hfr (81&apos;) dnaA::TnJO                      R100.1                 pRZ102                     (9.2 &#194;&#177; 5.8) x 10-7                  0.11 <br /> 13. NS294            att::P211                                 P2                     pRZ102                            8.4 x 10-6&quot;                  1.0 <br /> 14. NS366            att::P211 dnaA366                      P2                     pRZ102                            2.2 x 10-8b                &lt;0.01 <br />  lambda infection assay used measure transposition. assays steady-state assay. <br />  b Average duplicate experiments. values averages experiments. <br />  VOL. 169, 1987                                                                                           dnaA Tn5 TRANSPOSITION                        4641 <br />  <br />                TABLE 3. dnaA trans complementation                                  dnaA mutation Tn5 transposition primarily               Chromo-                      Gene                                     inactivation dnaA gene itself. <br />                                                            Transposition               Direct effect TnS. effect dnaA <br />  Strain         somal       Plasmid&apos;                        frencyb <br />               genotype                    plasmid              frequency            mutation Tn5 transposition mediated DnaA box <br /> D47.20        dnaA         pRZ1095        dnaA+         (1.5   &#194;&#177; 0.5) x 10-6        outer end TnS, effect single point mutation <br /> D47.20        dnaA+        pRZ1096        dnaA          (1.7   &#194;&#177; 0.1) x 10-6        dnaA consensus sequence examined. D47.22        dnaA         pRZ1095        dnaA+         (1.1   &#194;&#177; 1.0) X 10-6        conserved bases sequence homology (46) mutated <br /> D47.22        dnaA         pRZ1096        dnaA          (1.0   &#194;&#177; 0.3) x 10-7        sequence shown Table 4. assay transposition <br />     cells carry plasmid RSF1010::Tn5, source Tn5         mutant, used compound elements contain transposition assay. Cells carry compatible coresident plasmid           wild-type IS50 sequence outer-end sequences contains cloned copy functional nonfunctional dnaA gene.            IS5OL  mutation). sequences flank <br />   bThe steady-state assay used measure transposition.                        selectable marker, tetracycline resistance gene. <br />                                                                                        effect DnaA box mutation transposition                                                                                     presented Table 4. cells containing DnaA protein, <br /> affected TnS transposition (lines 13 14), effect                         point mutation reduced transposition 50-fold (lines 1 different dnaA null alleles transposition appears                          2). residual transposition mutant <br /> similar.                                                                            shows background frequency dele- <br />    results conclude mutations dnaA                            tion removed bases DnaA box <br /> decrease frequency Tn5 transposition. effect                         (lines 2 3). absence DnaA protein, trans- <br /> independent starting location Tn5, transposition                      position wild-type ends decreased eightfold (lines 1 assay used, second-site suppressor used permit cell                          4), shown (Table 2). cells lack <br /> growth, particular null allele dnaA gene.                            dnaA contain mutant DnaA box, transposition <br />    dnaA mutation does affect Tn5 transposition                              decreased additional sixfold (lines 4 5).  indirectly affecting absorption. possibility ruled                     DnaA box mutation decreased transposition following evidence.  dnaA                             DnaA protein present. presence qualitative effect TnS transposition X induction                          mutation DnaA box, additional effect <br /> assay (data shown). assay measures transposition                           DnaA protein removed mutation (lines 2 induced prophage independent possible                         5). <br /> adsorption problems. Second, Tn3 TnJO transposition                               effect DnaA box mutation (50-fold), showed dnaA sensitivity set assays (data                          present end element, greater shown).  phage growth consistently better                          effect removing DnaA protein (8-fold). simplest dnaA mutant background, unlikely result                        interpretation results single-base change absorption defect.                                                               DnaA box affects participation different <br />    Control polarity. dnaA gene                          components transposition machinery, DnaA pro- <br /> operon (36), inactivation structural gene                            tein additional factor factors. Presumably total insertion mutation amber mutation result                        effect mutation DnaA box product polarity expression distal gene dnaN.                        effect removing DnaA protein effect  determined effect dnaA                             additional factor factors. <br /> mutation Tn5 transposition inactivation                          dnaA IS50 transposition. DnaA box appears dnaA gene itself possible polar effect distal                        outer-end sequences ISSO (Fig. 1), reasonable <br /> gene expression.                                                                    question transposition IS50  uses    Transposition experiments performed wild-type                          outer end inner end) requires DnaA protein. dnaA mutant cells contained plasmid clones                         Experiments performed defective elements functional defective dnaA gene (8). func-                          outer ends outer end inner end tional defective dnaA gene, plasmids contained                            substrates complemented trans transposase chromosomal genes. cloned copy wild-type                             donor. <br /> dnaA gene reversed effect TnJO insertion                              Table 5, lines 1 3, shows eightfold <br /> chromosomal gene, cloned copy inactive dnaA                           decrease efficiency complementation element <br /> gene did (Table 3). result shows effect                    outer ends transposition measured  <br />  <br />                                                         TABLE 4. Effect dnaA site mutation&quot; <br />  Strain               Relevant                Plasmid                          Element description                      Transposition                  Relative <br />                       genotype                                                                                           frequency                    frequency <br /> D47.20                dnaA+                pRZ466                     Wild type                                     (2.3 &#194;&#177; 1.7) x 10-7                    1.0 <br /> D47.20                dnaA+                pRZ466-11A                 Point mutation                                (4.0 &#194;&#177; 2.0) x 10-9                    0.02 <br /> D47.20                dnaA+                pRZ464                     Deletion dnaA box                      &apos;1.0 x 10-9                     &lt;0.01 <br /> D47.26                dnaA                 pRZ466                     Wild type                                     (2.8 &#194;&#177; 1.6) x 10-8                    0.12 <br /> D47.26                dnaA                 pRZ466-11A                 Point mutation                                (4.0 &#194;&#177; 2.0) x 10-9                    0.02 <br /> D47.26                dnaA   pRZ464                                   Deletion dnaA box                      &apos;1.0   x 10-9                   &lt;0.01 <br />   &apos;Wild type 5&apos;-CTGACTCTTATACACAAGTAG <br />     466-11A       CTGACTCTTAAACACAAGTAG <br />                                                X <br />     464          CTGACTCTTATACACCGGGAT <br />     steady-state assay used measure transposition. &lt; denotes frequency transposition event transposition assay plate. <br />  4642        YIN REZNIKOFF                                                                                                                   J. BACTERIOL. <br />  <br />                                                                            TABLE 5. dnaA end usage <br />    Strain                     Relevant <br />                               genotype                          Transposase <br />                                                                donor plasmid&quot;               Target                                                                                         complementationb               Transposition <br />                                                                                                                         frequencyc                    Relative <br />                                                                                                                                                      frequency <br /> 1. D47.20                      dnaA+                           RSF1O01::Tn5          pRZ1113 (outer-outer)         (4.2 &#194;&#177; 0.8) x 10-8                    1.0 <br /> 2. D47.20                      dnaA+                           pRZ1O54               pRZ1113 (outer-outer)             &lt;4.7 x 10-9                     &lt;0.1 <br /> 3. D47.26                      dnaA                            RSFiOlO::Tn5          pRZ1i13 (outer-outer)         (5.4 &#194;&#177; 4.6) x 10-9                    0.13 <br /> 4. D47.26                      dnaA                            pRZ1054               pRZi113 (outer-outer)             &lt;2.0 x 10-9                     &lt;0.1 <br /> 5. D47.26                      dnaA+                           RSFi010::Tn5          pRZ1073 (outer-inner)         (3.4 &#194;&#177; 2.4) x 10-8                    1.0 <br /> 6. D47.20                      dnaA+                           pRZ1054               pRZ1073 (outer-inner)             s2.0 x 10-9                     &lt;0.1 <br /> 7. D47.26                      dnaA                            RSF101O:TnS           pRZ1073 (outer-inner)         (2.0 &#194;&#177; 2.0) x 10-8                    0.59 <br /> 8. D47.26                      dnaA                            pRZ1054               pRZ1073 (outer-inner)            s2.0 x 10-9                      &lt;0.1 <br />   aRSF1010::TnS encodes functional transposase, pRZ1054 does  <br />   b second, coresident plasmid contains element cdmolemented Trans. pkZl13 contains element outer ends flanking gene codes tetracycline resistance. pRZ1073 contains element outer end inner end flanking gene codes tetracycline resistance. <br />   c steady-state assay used measure transposition. s denotes frequency transposition event transposition assay plate. <br />  <br />  <br /> dnaA strain.  element outer end                                   shown previously Pi translated Ti inner end examined, significant                                   transcript, P2 translated T2 transcript (29). <br /> decrease transposition frequency dnaA+                                   DnaA box approximately 13 base pairs away dnaA strain (lines 5 7).                                                        putative -35 region Ti promoter    dnaA Tn5 gene expression. Genetic evidence suggests                                45 base pairs away -35 region T2 promoter possible direct interaction DnaA protein                                (29), DnaA protein affect transcription RNA polymerase (2, 3, 40). examined                                      promoter. <br /> possible role interaction transcription                                 Si mapping experiments RNA isolated Tri genes. transposition frequency TnS                                   wild-type dnaA mutant cells showed difference shown function concentration                              pattern protection  major transcripts Tn5. <br /> transposase, Pi (Yin et al., press), inhibitor, P2 (19,                        Figure 2 autoradiogram gel Si analysis. <br /> 24; submitted publication), DnaA protein                             levels Ti T2 transcripts did appreciably <br /> stimulate expression Pi depress expression P2.                            change mutant cells (compare lanes b).                                                                                           dam methylation mutation added strains,                                                                                           effect dnaA mutation Ti T2 <br />                                                                                           transcripts (lanes c d), showed char- <br />                      &apos;j          b c d                                                    acteristic response tnethylation mutation (e.g., com- <br />                                                                                           pare lanes c lanes b d). <br />                                                                                              Furthermore, DnaA protein required TnS trans- <br />                                                                                           position transposase expressed                                                                                           heterologous promoter. dependence transposition                                                                                           dnaA measured cells transposase mRNA <br />                                                                                           placed control X PR promoter (M. <br />                                                                                           Krebs, personal communication). requirement                                                                                           element DnaA protein (Table 6, lines 3 4)                           *_;,      ........ <br />  <br />  <br />                                          :.....r           b <br />                                                                                           similar requirement wild-type element <br />                                                                                           (lines 1 2). Taken  experiments                                                      TRT <br />                                                  ..... <br />                                                                                           dnaA does affect TnS transposition affecting <br />                                                                                           transcriptional initiation element. <br />                                                                                              Effect overproducing DnaA. decreasing                                                                                           DnaA protein decreases transposition,  <br />                                   ..                                                      creasing DnaA protein increase                                                                                           transposition frequency. Table 7 presents transposition data <br />                                                                                           various clones overproduce DnaA protein. <br />                           ........................                                        clones place dnaA gene control                                                                                           X PL promoter (13; R. Fuller, personal communication). <br />                                                                                           fully inducing conditions, pBF11O overproduces                                                                                           DnaA protein 200-fold (13). transposition experi- <br />                                                                                           ments involved clones performed <br />                                                                                           fully inducing conditions,   FIG. 2. SI mapping transcripts. autoradiograph 8%                           overproduction DnaA protein measured. polyacrylamide-7 M <span id='am-29' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> gel shown. RNAs prepared D47.20                           clone overproduces DnaA protein places dnaA <br /> (RNase H), D47.22 (RNase H dnaA), D48.10 (RNase H dam),                               gene control <span id='am-5' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span>lactose</span> promoter (8). Transposition <br /> D48.15 (RNase H dnaA dam) cells containing plasmid pRZiO2                                 experiments overproducer strain <br /> correspond lanes d, respectively. end-labeled probes used Si reactions, hybridized                             deficient lactose repressor. range j3-lactamase gene (located chromosome)                            overproduction clones exhibit exper- detected Tn5-specific mRNA species. relevant Tn5                                imental conditions, did appear stimula- <br /> transcripts shown right gel. RT denotes                             tion Tn5 transposition. experiments pre- <br /> readthrough transcript enters element outside.                             sented Table 7 performed dam mutants (lines 1 <br />  VOL. 169, 1987                                                                                                dnaA Tn5 TRANSPOSITION                  4643 <br />  <br />                               TABLE 6. Effect dnaA transposase expressed heteologous promoter <br />  <br />  Strain                   Relevant                    P <br />                                                       Plasmid                       Control                       Transposition                    Relative <br />                           genotype                                                 transposase                         frequency                     frequency <br />                                                                                     synthesis <br /> D47.20                     dnaA+                      pRZ914                       Wild type                     (1.2 &#194;&#177; 0.5) x 10-7                    1.0 <br /> D47.22                     dnaA                       pRZ914                       Wild type                     (5.0 + 1.0) x 10-9                    0.04 <br /> D47.20                     dnaA+                      pRZ986                       PR                          (3.3 &#194;&#177; 0.3) x 10-5a                   1.0 <br /> D47.22                     dnaA                       pRZ986                       k PR                          (3.7 &#194;&#177; 0.7) x 10-6a                   0.11 <br />         Steady-state transpositions   X299   assayed 34&#194;&#176;C. transpositions X84 assayed 37&#194;&#176;C. <br />  3), transposase levels increased wild-                             fected mutations dnaA gene (data shown). <br /> type levels (Yin et al., press). Increasing                        elements contains DnaA box sequences. <br /> transposase did make level DnaA protein                              data indicate mutations dnaA gene rate limiting transposition.                                                        affect Tn5 transcriptional initiation. effect                                                                                         levels major transcripts Ti T2,                                                                                         readthrough transcription affected. comple- <br />                               DISCUSSION                                                mentation experiment presented Table 6 consistent <br />                                                                                         interpretation mutations dnaA effect <br />                                                                                         TnS transcriptional initiation.  requirement <br />    dnaA TnS transposition. experiments described                             DnaA protein probably level TnS gene paper involve use different mutations                            expression. <br /> dnaA gene (28, 40) investigate requirement                                  Role dnaA cell. DnaA protein known DnaA protein TnS transposition. TnJO insertion                                functions cell: repression transcriptional amber mutation result substantial decrease                               initiation (8, 43; D. Smith, personal communication) transposition. requirement DnaA protein TnS                              role initiation DNA replication (12, 15-17). transposition absolute, decrease                                functions thought involve binding protein <br /> transposition frequency 10-fold (Table 2).                                DnaA boxes, binding purified protein Mutations transposase sequences required                               sites shown (8, 12). requirement DnaA outer end TnS effects 100-fold (22, 35).                        protein TnS transposition related    Control experiments effect largely inde-                           functions dnaA known perform? <br /> pendent transposition assay, type suppressor                                 Repression. clear Si mapping data allows cell growth absence DnaA protein,                            complementation experiment presented Table 6 donor site TnS starts. cloned copy                              dnaA mutation does affect initiation transcription dnaA gene reverses effect, showing directly                            TnS. readthrough transcript detected Si <br /> indirectly, requirement DnaA protein itself.                             mapping experiment affected dnaA muta- <br /> Analysis single point mutation consensus DnaA                               tion.  transcriptional elongation entering ele- <br /> box present outside end TnS consistent                            ment probably affected DnaA protein. Tran- <br /> interpretation DnaA protein interacts site.                          scriptional fusion experiments measure genetic evidence, taken results                             transcription traversing DnaA box direction vitro binding experiment (12), suggests dnaA exerts                            dnaA effect (data shown).  <br /> effect TnS transposition acting                              quality transcript, quantity, site. transposition IS50 does require DnaA                             important factor TnS transposition. possible protein, protein function interacting                          RNA polymerase-DnaA protein complex formed DnaA box sequences located outer end Tn5.                                RNA polymerase crosses DnaA box com- additional piece evidence supports interpre-                            plex important species TnS transposition. tation transposition Tn3, Tn9, TnJO unaf-                             precedence hypothesis transcriptional <br />  <br />  <br />                                                               TABLE 7. Overproduction DnaA <br />         Strain                            Relevant                            Plasmid                         Promoter/                       Transposition <br />                                          geneotype                         overproducer                         gene                            frequency <br /> 1. D48.09                                dam                                 pBF110                       X   pLjdnaA&apos;                    (3.6 &#194;&#177; 2.4) x 10-5a <br /> 2. D48.09                                dam                                 pRB25                        lac/dnaA+                       (6.6 &#194;&#177; 4.2) x 10-5a <br /> 3. D48.09                                dam                                 pRB36                        lacldnaA                        (2.8 &#194;&#177; 1.2) x 10-5a <br /> 4. 168.01                                N+ host                                                                                    3.1 x 10-5b <br /> 5. I68.01                                N+ host                             pBF110                       k pL/dnaA&apos;                          4.2 x 1o-5b <br /> 6. I68.01                                N+ host                             pBF1509                      k   pL/dnaA&apos;                    (2.4 &#194;&#177; 0.6) x 10-5b <br /> 7. 169.01                                N host                              pBF1509                      X   PLIdnaA&apos;                    (2.6 &#194;&#177; 1.8) x 10-5b <br /> 8. RZ211/F&apos;lq (K)                        recA                                pRB322                                                  (2.0 &#194;&#177; 0.1) x 10-6a <br /> 9. RZ211/F&apos;Iq (K)                        recA                                pRB25                        lac/dnaA+                       (6.3 &#194;&#177; 2.7) x 10-6a <br /> 10. RZ211/F&apos;lq (K)                       recA                                pRB36                        lacldnaA                        (2.7 &#194;&#177; 0.7) x 10-6a <br />       lambda induction assay used measure transposition. <br />   b mating assay used measure transposition. <br />  4644     YIN REZNIKOFF                                                                                                 J. BACTERIOL. <br />  <br /> activation model oriC replication (43). data      involved initiation chromosomal replication (13, <br /> support refute model.                                      17). initiation event regulated process occurs <br />    Replication. direct evidence exten-      cell division.  DnaA protein  <br /> sive replication needed Tn5 transposition process     lieved synthesized constant rate (6, 21). elegant genetic evidence shows TnJO        cell cycle (37). form temporal regulation <br /> transposes conservative mechanism (4) does exist          exists controls timing initiation event, TnS. evidence does exist suggests TnS           controlling activity DnaA protein. does transpose obligate, cointegrate structure      transposition requires active DnaA protein, transposi- intermediate (5, 21). cointegrate replicative       tion occur limited cell cycle. <br /> intermediate Shapiro model transposition (41). <br />    Interaction ends. possible role DnaA <br /> protein TnS transposition bind DnaA box                               ACKNOWLEDGMENTS outer end assist properly aligning ends          thank M. Wickens careful reading manuscript, M. type nucleoprotein structure. require-         Krebs P. Lambert discussions aid,  Wright, R. <br /> ment DnaA protein TnS transposition              Braun, R. Johnson, T. Kogoma strains. B. Fuller absolute, likely transposase itself   helpful initial stages work. <br /> capable bringing outer ends         work supported Public Health Service grant efficiently. model true, IS50 transposition,    GM19670 National Institutes Health. J.C.P.Y. independent dnaA, rely transposase         supported Public Health Service Training Grant GM07133 <br /> itself align outer end inner end proper        National Institutes Health. <br /> structure. <br />   related possibility DnaA protein facilitates                          LITERATURE CITED <br /> binding function transposase outer end. spatial relationship presumed transposase-              1. Abeles,  L., K. M. Snyder, D. K. Chattoraj. 1984. P1 <br /> binding site, 5&apos;-CTGACTCTT-3&apos;, DnaA box                plasmid replication: replicon structure. J. Mol. Biol. 173: <br />                                                                        307-324. <br /> interesting, actually overlap (Fig. 1). overlap     2. Atlung, T. 1981. Analysis seven dnaA suppressor loci binding sites similar relationship       Escherichia coli. ICN-UCLA Symp. Mol. Cell. Biol. 22: <br /> presumed transposase-binding site  IHF-                  297-314. <br /> binding site (14). tempting speculate host    3. Bagdasarian, M. M., M. Izakowska, M. Bagdasarian. 1977. <br /> factors performing analogous roles transposi-            Suppression DnaA phenotype mutations rpoB <br /> tion process elements.                                    cistron ribonucleic acid polymerase Salmonella typhy- <br />    Interaction dam methylation DnaA protein.                murium Escherichia coli. J. Bacteriol. 130:577-582. evidence dam methylation           4. Bender, J., N. Kleckner. 1986. Genetic evidence TnlO <br /> DnaA protein probably interact. DnaA protein represses             transposes non-replicative mechanism. Cell 45:801-815. <br />                                                                     5. Berg, D. E. 1983. Structural requirement IS50-mediated gene <br /> expression dam gene (D. Smith, personal communi-                transposition. Proc. Natl. Acad. Sci. USA 80:792-796. <br /> cation), recently shown         6. Berg, D. E., C. M. Berg. 1983. prokaryotic transposable <br /> dnaA structural gene promoters activated dam meth-               element TnS. Bio/Technology 1:427-435. <br /> ylation (9).  genes gene products          7. Bochner, B., H.-C. Huang, G. Schieven, B. N. Ames. 1980. <br /> form autoregulatory circuit. juxtaposition          Positive selection loss tetracycline resistance. J. Bacte- DnaA boxes dam methylation sites oriC region             riol. 143:926-933. <br /> (45) ori-r region phage P1 (1).        8. Braun, R. E., K. O&apos;Day,  Wright. 1985. Autoregulation origins require dnaA function. oriC region                 DNA replication gene dnaA Escherichia coli K-12. Cell <br /> requires methylated bases function (32, 42).                        40:159-169. <br />                                                                     9. Braun, R. E.,  Wright. 1986. DNA methylation differen- <br />    Transposition IS50 sensitive dam methylation           tially enhances expression E. coli dnaA mechanism increases usage inner               promoters vivo vitro. Mol. Gen. Genet. 202:246- <br /> end substrate undermethylated (Yin et al.,          250. <br /> press). Usage outer ends TnS transposition       10. Craigie, R., D. J. Arndt-Jovin, K. Mizuuchi. 1985. defined independent dam methylation (Yin et al.,           strand-transfer reaction initiation press) stimulated dnaA. possible        bacteriophage Mu transposition. Proc. Natl. Acad. Sci. USA <br /> forms stimulation, undermethylation acting IS50 inner            82:7570-7574. <br /> ends DnaA protein acting TnS outer ends,            11. Flamm, E. L., R.  Weisberg. 1985. Primary structure functionally analogous. example, transposition                 hip gene Escherichia coli product, subunit <br /> require proper positioning ends element          integration host factor. J. Mol. Biol. 183:117-128. <br />                                                                    12. Fuller, R. S., B. E. Funnell,  Kornberg. 1984. dnaA transposase. IS50 transposes,               protein complex E. coli chromosomal replication origin <br /> transposase bound outer end inner end             (oriC) DNA sites. Cell 38:889-900. <br /> bring ends form complex. Undermethyla-        13. Fuller, R. S.,  Kornberg. 1983. Purified dnaA protein <br /> tion inner end stimulate binding               initiation replication Escherichia coli origin replica- <br /> transposase site. TnS transposes, DnaA                tion. Proc. Natl. Acad. Sci. USA 80:5817-5821. <br /> protein bound outer end bring ends           14. Gamas, P., D. Galas, M. Chandler. 1985. DNA sequence facilitate performance function             end  required transposition. Nature (London) <br /> transposase.  usage outer end inner            317:458-460. <br /> end facilitated interacting genes        15. Hansen, E. B., M. B. Yarmolinsky. 1986. Host participation <br />                                                                        plasmid maintenance: dependence dnaA replicons products.                                                        derived P1 F. Proc. Natl. Acad. Sci. USA 83:4423- <br />    Celiular regulation TnS transposition. formally            4427. <br /> possible DnaA protein involved regulating       16. Hasunuma, K., M. Sekiguchi. 1977. Replication plasmid <br /> timing TnS transposition. DnaA prote <br /> </body></html>