<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Feb. i991, p. 1376-1381                                                                      Vol. 173,  4 <br /> 0021-9193/91/041376-06$02.00/0 <br /> Copyright &#194;&#169; 1991, American Society Microbiology <br />  <br />  <br />          RNase T Affects Escherichia coli Growth Recovery                               Metabolic Stress <br />                          KRISTINE PALAS PADMANABHA MURRAY P. DEUTSCHER* <br />             Department Biochemistry, University Connecticut Health Center, Farmington, Connecticut 06432 <br />                                         Received 21 September 1990/Accepted 5 December 1990 <br />  <br />              determine essentialty role RNase T RNA metabolism, constructed Escherichia col <br />            chromosomal rnt.:kan mutation using gene replacement disrupted, plasmid-borne copy rnt <br />            gene. Cell extracts strain mutations RNases BN, D, H, interuppted rnt gene            devoid RNase T activity, retained low level (&lt;10%) exonucleolytic activity            tRNA-C-C-[]4CJA unidentified RNases. mutant lacking tRNA nucleotidyltanserase            addition aforementioned RNases accumulated 5% defective tRNA did RNase <br />            T-positive cells, indicating RNase responsible essentialiy tRNA end turnover E. coi. tRNA <br />            rnt::kan strains displayed slightly reduced capacity aminoacylated, raising possibility            RNase T participate tRNA processing. Strains devoid RNase T displayed slower growth rates <br />            did wild type, phenotype accentuated absence exoribonludeases.            strain lacking RNase T RNases displayed  normal response UV irradiation growth            bacteriophages severely affected ability recover starvation regimen. data <br />            demonstrate absence RNase T affects normal functioning E. coil, compensated <br />            degree presence RNases. Possible roles RNase T RNA metabolism            discussed. <br />  <br />    RNase T seven Escherichia coli exoribonu-                              MATERIALS METHODS <br /> cleases identified <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> (6, 7). enzyme,           Bacterial strains plasmids. Wild-type E. coli CA265 <br /> purified homogeneity, exists a2 dimer 50 kDa (11).       RNase-deficient derivatives (BN deficient), vitro RNase T specifically catalyzes removal           CAN20-12E (BN, II, D, deficient), CAN20-12E/ <br /> terminal <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>AMP</span> residue uncharged tRNA releases                18-11 (BN, II, D,  T deficient) described <br /> tRNA-C-C product (12). gene encoding RNase T                previously (13). Strain MC1061 [F- araD139 ara-leu)7697 <br /> (rnt) recently cloned localized 36 min           Alac galU galK strA] used construction rnt E. coli genetic map (3).                                         insertion mutant. Strain UT481  lac-pro) hsdS (r- m-) <br />    Previously, rnt mutant strain retained        laCdq] routinely used transformations. pRNT14  <br /> 30% RNase T activity cell extracts isolated (13).       sists 1. 1skb DNA fragment containing rnt gene <br /> studies mutant strain, deficient           cloned BamHI HindIII sites pBR328 (3). <br /> tRNA nucleotidyltransferase (cca), shown RNase           pUC4K obtained Pharmacia. pMAK700 (16) T involved end turnover tRNA. cca mutant                gift S. Kushner (University Georgia). pMAK700, <br /> strains accumulate defective tRNA  result,             thermosensitive DNA replication, contains <br /> dramatically reduced viability (21).  introduction           unique HindIII, BamHI, SphI sites carries gene rnt mutation cca mutant strains prevents             encoding chloramphenicol acetyltransferase (16). <br /> formation defective tRNA relieves slow-growth                  Materials. RNase T substrate   tRNA-C-C-[&quot;4C phenotype.                                                            prepared [14C]AMP incorporation tRNA-C-C E. <br />    mentioned studies demonstrated             coli tRNA nucleotidyltransferase (10). tRNA isolated rnt involved end turnover, did answer                                                                       phenol extraction isopropanol fractionation described <br />                                                                       previously (9). [14C]<span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span> obtained ICN. Ultrogel <br /> question gene essential E. coli,         AcA54 purchased LKB. Restriction endonu- <br /> residual RNase T activity present mutant.         cleases, mung bean nuclease, T4 DNA ligase addition, rnt mutation isolated background         purchased New England BioLabs used accor- lacked RNases (13), possible         dance directions manufacturer. salts used <br /> study effect mutation isolation. reason,       reagent grade. constructed rnt: :kan insertion mutant using             Growth conditions. Cells grown YT media (19). <br /> gene replacement method (16) moved allele           solid medium, 2% agar (Difco) added. YT broth different genetic backgrounds study physiological              YT agar supplemented ampicillin (50 ,ug/ml), <br /> properties strains lacking RNase T. results indicate           kanamycin (50 p.g/ml), chloramphenicol (20 ,ug/ml) RNase T essential normal growth E. coli        needed. recovery starvation.                                        DNA techniques. E. coli chromosomal DNA isolated <br />                                                                       described Silhavy et al. (20). Large-scale isolation                                                                       plasmid DNA method Birnboim Doly <br />                                                                       (1). Rapid plasmid screening modification                                                                       procedure Ish-Horowitz Burke (17) small <br />    *   Corresponding author.                                             volumes cell cultures. <br />                                                                   1376 <br />  VOL. 173, 1991                                                                 PHYSIOLOGICAL ROLE RNase T                  1377 <br />  <br />    DNA probes Southern blots labeled                noacylation assays performed described previously <br /> [oa-32P]<span id='am-9' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-10' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span>dATP</span> (New England Nuclear) random primer           (4) mixture amino acids E. coli aminoacyl- <br /> method Feinberg Vogelstein (15). Southern blotting         tRNA synthetases. E. coli aminoacyl-tRNA synthetases performed described Maniatis et al. (18).               devoid tRNA nucleotidyltransferase prepared    Plasmid constructions. pRNT14, contains 1.1-kb         described Deutscher Evans (8). Protein concentra- <br /> insert carrying rnt gene (3), used construct        tions determined colorimetrically method rnt::kan insertion mutation. kanamycin cassette           Bradford (2). <br /> obtained digestion pUC4K (Pharmacia) Sall. single-stranded DNA ends kanamycin DNA                                         RESULTS <br /> fragment removed mung bean nuclease. fragment ligated pRNT14, linearized            Subcloning overexpression rnt. rnt gene partial digestion HincII. pRNT14 contains             initially cloned screening cosmid library containing <br /> HinclI site rnt additional sites vector.    large fragments E. coli genome plasmid pHC79 (3). plasmid clone, pRNT17, contained single insertion        rnt gene plasmid resulted twofold kan rnt gene, confirmed restriction endonu-       elevation RNase T activity (3). rnt gene subse- <br /> clease cleavage plasmid, obtained             quently subcloned 2.5-kb fragment pHC79, pBR328, <br /> ligation.                                                         pUC18 assess level overexpression    rnt::kan gene inserted plasmid pMAK700            determine effect elevated RNase T activity cell <br /> (16). 2.4-kb BamHI-HindIII DNA fragment pRNT17             growth. copy numbers plasmids containing rnt::kan cloned corresponding          thought different, levels RNase T activity cells <br /> striction sites pMAK700. resulting plasmid, pRNT18,        harboring different plasmid subclones nearly used construct rnt::kan chromosomal mutation.           RNase T activity elevated 12.3-, 13.9-,    Construction rnt::kan chromosomal mutation.         15.0-fold CA265 containing rnt plasmids pHC79, <br /> method used described Hamilton et al.       pUC18, pBR328, respectively. maximum elevation <br /> (16). pRNT18 transformed strain MC1061.              RNase T activity cells apparently tolerate <br /> resulting strain grown 30&#194;&#176;C mid-log phase,      15-fold. fact elevated activity specif- <br /> dilutions culture plated YT agar plates          ically removed terminal AMP residue tRNAC-C- <br /> supplemented kanamycin chloramphenicol.              [&apos;4C] confirmed presence rnt gene plates incubated 44&#194;&#176;C select cells       plasmid subclones. <br /> integrated plasmid chromosome. cointe-          Strain CA265 harboring rnt cloned pHC79 displayed grates selected grown cycles 300C.         growth rate identical untransformed strain. temperature, cointegrates unstable tended       contrast, considerable lag  hours) recombine free plasmid, carrying        onset exponential growth CA265 contained rnt <br /> wild-type rnt::kan allele, generated      cloned high-copy-number plasmid pUC18 (data cell (16). Strains containing plasmid resolution products         shown). growth curves CA265 strains car- selected screening colonies 300C culture         rying pUC18 containing rnt pHC79 containing rnt temperature-sensitive growth chloramphenicol plates.       dramatically different, levels RNase T activity Plasmid DNAs 13 temperature-sensitive colonies          log-phase cultures strains  prepared analyzed agarose gel (data shown).         . explanation findings high <br /> DNAs strains size pRNT18.             level RNase T activity expected remaining strains contained plasmid DNAs           present initially strains carrying pUC18 containing rnt <br /> smaller pRNT18, indicating wild-type rnt gene       deleterious normal growth.  replaced rnt::kan construct plasmid. Strains       strong selection cells reduced containing smaller rnt plasmids kanamycin resis-         RNase T activity consequently grow faster. Slow growth <br /> tant chloramphenicol sensitive 440C, indicating       strains elevated RNase T activity kan integrated chromosome. plasmid          high level tRNA end turnover cellular levels cured strains growth liquid cultures        tRNA nucleotidyltransferase sufficient repair 44&#194;&#176;C absence chloramphenicol.                           defective tRNA molecules. support explana- <br />    Analysis RNase T activity cell extracts. Cell extracts   tion, attempts transformation CCA-negative strain prepared 30-ml cultures grown optical             rnt gene subcloned pHC79 unsuccessful, <br /> density 550 nm (OD550) 1 37&#194;&#176;C. Cells harvested      CCA-positive strains transformed normally centrifugation, resuspended 1/5 volume sonication      (data shown). <br /> buffer (20 mM <span id='am-32' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-33' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span>-NaOH [pH 8.9], 0.1 mM EDTA, 1                 Southern blots wild-type rnt::kan strains. Southern <br /> mM dithiothreitol), sonicated ice 60 s.          blots chromosomal DNAs MC1061 MC1061 <br /> centrifugation remove cell debris, 25-,u aliquots         containing rnt::kan (MC1061-rnt::kan) prepared assayed RNase T activity 20 min reaction mixture     probed labeled 1.1-kb HindIII-BamHI DNA frag- <br /> (0.2 ml) containing 20 mM <span id='am-44' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-45' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycine</span>-NaOH (pH 8.9), 5 mM             ment containing complete rnt gene (Fig. 1B). blot <br /> MgCl2, 30 ,ug tRNA-C-C-[&apos;4C  Reactions              MC1061-rnt: :kan DNA, bands seen stopped addition 0.3 ml 0.5% yeast tRNA 0.3      autoradiograph PstI PstI-BamHI <br /> ml 20% trichloracetic acid. precipitate removed        digests;  single band obtained centrifugation, radioactivity half       corresponding digests MC1061 DNA (Fig. 1A). PstI <br /> supernatant fluid determined.                                 digest MC1061 DNA single band 7 kb (Fig. 1A, <br />   methods. tRNA defective 3&apos;             lane 1), corresponding digest MC1061-rnt::kan <br /> termini various strains determined isolating tRNA       DNA bands 4.4 2.6 kb (Fig. 1A, lane 3). band measuring level [&apos;4C]AMP incorporation          3.2 kb obtained BamHI-PstI digest MC1061 <br /> presence E. coli tRNA nucleoltidyltransferase (9). Ami-        DNA (Fig. IA, lane 2). contrast, double BamHI-PstI <br />  1378     PADMANABHA DEUTSCHER <br />  <br />                 1   2 3 <br />  <br />  <br />                       ; <br />                           _ ~ 9.4 <br />                              4 <br />  <br />                                    23.1 <br />                                     6.5 <br />                                  - 4.3 <br />                                    2.3 <br />                                    2.0 <br />  <br />                                  - .56 <br />                                              B <br />  <br />  <br />  <br />  <br />                                           length (kb) <br />                                                       (n <br />                                                      ra <br />  <br />  <br />                                                             <br />                                                            L <br />                                                                   0L <br />  <br />  <br />  <br />  <br />                                                                  2.1 <br />                                                                             2.1 <br />  <br />  <br />  <br />  <br />                                                                             l <br />                                                                                 C <br />                                                                                 . <br />                                                                                 = m <br />                                                                                     tfi nL <br />  <br />                                                                                     0.5 <br />                                                                                              -ac <br />  <br />  <br />  <br />  <br />                                                                                               - <br />                                                                                              1.3 <br />                                                                                                    _ <br />                                                                                                         c <br />  <br />  <br />                                                                                                        1-L <br />                                                                                                        11 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                <br />  <br />  <br />  <br />                                                                                                               :ECZ <br />  <br />  <br />                                                                                                              0.6 <br />                                                                                                                    m <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                       CC <br />  <br />                                                                                                                      LX. <br />                                                                                                                       `7 `7 <br />                                                                                                                                3.8 <br />  <br />  <br />  <br />  <br />                                                                                                                               3.8 <br />                                                                                                                                      I_e <br />  <br />  <br />  <br />  <br />                                                                                                                                       r, <br />                                                                                                                                            M(; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                       rr! <br />                                                                                                                                                                                                                                                                                                       J. BACTERIOL. <br />  <br />  <br />  <br />  <br />                                                                                                                                                      jD6 <br />  <br />  <br />  <br />  <br />                                                                                                                                                      06 <br />  <br />  <br />                                                                                                                                                      kar- <br />                                                                                                                                                            1 <br />  <br />  <br />  <br />  <br />                                                                E7Z       r nt   probe <br />                                                                L&apos;,&apos;- _chromosomal DNA outside probe regior <br />                                                                     _M kanamycin cassette <br />   FIG. 1. Southern blot chromosomal DNA MC1061-rnt::kan MC1061.   Lanes: 1 2, MC1061 chromosomal DNAs <br /> digested PstI PstI-BamHI, respectively; 3 4, MC1061-rnt::kan chromosomal DNAs digested PstI PstI-BamHl, <br /> respectively. markers right lane 4 indicate migration positions lambda HindlIl size markers  thousands). (B) <br /> Location kanamycin cassette rnt gene positions restriction sites. Note kanamycin cassette contains flanking PstI sites; PstI sites rnt. <br />  <br />  <br /> digest MC1061-rnt::kan DNA yielded bands 2.6                                    rnt-encoded activity. Examination effect kan 0.6 kb (Fig. 1A, lane 4). data agree previ-                                insertion chromosomal copy rnt gene indicated <br /> ously mapped restriction sites surrounding rnt                                 RNase T activity strain CA265 reduced gene (3). results, taken genetic                                  twofold comparison strain evidence described Materials Methods, indicated                               uninterrupted gene.  wild-type strain, kan cassette located middle rnt gene                             enzyme enzymes present ability <br /> MC1061-rnt::kan (Fig. 1B).                                                                remove terminal AMP residue tRNA-C-C     RNase T activity strains carrying rnt::kcw insertion.                           hand, strain CAN20-12E, RNase T activity rnt::kan construct MC1061-rnt::kan trans-                                    reduced 10-fold interruption rnt gene. <br /> duced wild-type strain, CA265, strain                                    strain, remaining 3&apos; exonucleolytic <br /> CAN20-12E, deficient multiple RNase activities                                activity intact tRNA observed assay condi- <br /> (BN, II, D, , strain CAN20-12E/18-11,                                tions encoded rnt. deficient multiple RNase activities (BN, II, D,                                 Previously, rnt mutant RNase-deficient strain T). Cell extracts strains controls                             CAN20-12E isolated nitrosoguanidine mutagenesis. prepared assayed RNase T activity (Table 1).                                 resulting strain, CAN20-12E/18-11, 30% introduction intact rnt gene strain CAN20-                                 RNase T activity parent (13). activity 12E/18-11 plasmid pRNT14 led 15-fold elevation                               CAN20-12E containing rnt::kan (CAN20-12E-rnt::kan) RNase T activity. rnt gene interrupted                                threefold lower CAN20-12E/18-11,  <br /> plasmid, pRNT17, RNase T activity overpro-                                   firming rnt mutation nitrosoguanidine <br /> duced. insertion,  completely abolished                                 mutagenesis leaky does abolish rnt-encoded <br />                                                                                           activity (Table 1). <br />                                                                                              Analysis RNase activity CAN20-12E CAN20- <br />        TABLE 1. Analysis RNase T activity rnt+                                                                                           12E-rnt.*:kan strains gel filtration chromatography.                       rnt::kan strainsa                                                   analyze residual activity putative rmnt null mutant, <br />                                                                                           prepared cell extracts strains CAN20-12E                                                            [14C]AMP released              CAN20-12E-rnt::kan resolved AcA54 gel <br />         Strain                   Phenotype                    (nmol/mg                 filtration column. assay fractions tRNA-C-C- <br />                                                                 protein)                  [&apos;4C1 (Fig. 2) revealed single peak CAN20-12E <br /> CAN20-12E/18-11           Deficient RNases                     1.80                    extract corresponding size 50 kDa, known <br />                             BN, II, D,  T                                            size RNase T. contrast, CAN20-12E-rnt: :kan <br /> CAN20-12E/18-11 +         Deficient RNases                   27.0                      extract showed small peaks activity, 66 <br />   pRNT14                    BN, II, D,  T                                           kDa 35 kDa, confirming RNase T CAN20-12E/18-11 +         Deficient RNases                     1.90                    absent. identities residual activities   pRNT17                    BN, II, D,  T                                           known. <br /> CA265                     Wild type                               8.30 <br /> CA265-rnt::kan            Deficient RNase T                    4.90                       End turnover tRNA rwt::kan strins. tRNA CAN20-12E                 Deficient RNases                     5.80                    isolated rnt::kan control strains analyzed <br />                             BN, II, D,                                              acceptance AMP tRNA nucleotidyltrans- <br /> CAN20-12E-rnt::kan        Deficient RNases                     0.60                     ferase assess end turnover described previously (9).                              BN, II, D,  T                                           reported previously (13), little terminally defective <br />   Cell extracts prepared assayed described Materials                  tRNA strains CAN20-12E/18-11 (0.10 0.09 <br /> Methods. pRNT14 carries wild-type rnt gene, pRNT17 carries                     nmol (14C]AMP incorporated mg tRNA, respec- <br /> interrupted rnt gene.                                                                      tively) presence wild-type cca gene.  VOL. 173, 1991                                                                             PHYSIOLOGICAL ROLE RNase T                          1379 <br />  <br />                                                                           TABLE 2. Aminoacylation tRNA isolated wild-type                                                                                           RNase T-deficient strainsa <br />                                                                                   Strain                    Phenotype                  Relative <br />                                                                                                                                   aminoacylation (%) <br />                                                                          CA265                      Wild type                            100 <br />        EI <br />                                                                          CA265-rnt::kan             Deficient RNase T              91 &#194;&#177; 3.0 <br />        1f 600     - <br />  <br />                                                                          CAN20-12E                  Deficient RNases               95 &#194;&#177; 2.5 <br />                                                                                                       BN, II, D,            400    -                           CAN20-12E                  CAN20-12E-rnt::kan         Deficient RNases               85 &#194;&#177; 0.7 <br />                                                 ~~~~rnt::kan                                          BN, II, D,  T <br />            200-                                                            14C-<span id='am-25' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> incorporated 100 ,ug tRNA                                                                          determined described Materials Methods. percentages represent <br />               0 <br />                                                                          results experiments. <br />                   25   30      35     40     45      50     55 <br />                               Fraction <br />    FIG. 2. Gel filtration chromotagraphy cell extracts           tRNA precursors. work necessary evalu- <br /> CAN20-12E CAN20-12E-rnt::kan strains. cells grown           ate possible role RNase T. OD550 1 YT media 37&#194;&#176;C. Cells (500 ml) harvested,        Growth rates rnt::kan strains. growth rates washed sonication buffer, resuspended 3 ml buffer,     wild-type rnt::kan strains measured rich media sonicated ice 3 5 min  1-min bursts) disruption   37&#194;&#176;C (Table 3). presence rnt: :kan mutation appeared complete. Cell debris removed low-speed centrif-         small, reproducible, effect growth rate ugation 12,000 x g. extract centrifuged high speed <br /> (100,000 x g) 3 h. 35 mg (3 ml) extract layered     wild-type strain CA265 genetic background (30 versus 25 AcA54 column (84 1.5 cm) eluted buffer containing        min). pronounced effect growth observed 20 mM Tris chloride (pH 7.4), 0.5 mM EDTA-0.1 mM phenylmeth-             RNase-deficient background strain CAN20-12E: ylsulfonyl fluoride, 0.1 mM dithiothreitol, 10% glycerol, 0.2 M      mutant doubling time 45 min, compared 30 <br /> NaCl. Fractions 1.5 ml collected, 25-,ul aliquots       min parental strain. results indicate RNase <br /> assayed RNase T activity described Materials Meth-         T essential normal growth E. coli  ods. arrows graph indicate positions    potential compensatory RNases absent, effect molecular mass standards  left right): catalase   232   growth quite pronounced. <br /> kDa, eluted void volume), bovine serum albumin (67 kDa),             Recovery rnt:.:kan strains starvation. CA265, <br /> ovalbumin (43 kDa), trypsin inhibitor (21 kDa). <br />                                                                          CA265-rnt: :kan, CAN20-12E, CAN20-12E-rnt: :kan <br />                                                                          grown mid-log phase YT media                                                                          starved Tris buffer (pH 7.4) 16 h. starvation,  introduction cca mutation strains              cells returned YT media, rate recovery <br /> CAN20-12E/18-11 led large increase terminally defec-             measured. Strains CA265, CA265-rnt::kan, CAN20- <br /> tive tRNA levels inability repair                  12E started recover rich media 2-h lag <br /> defective molecules. Defective tRNA levels                    (Fig. 3). contrast, growth experiments fivefold lower CAN20-12E/18-11 derivative             arrested CAN20-12E-rnt::kan culture 4 h derivative, contains RNase T (0.70 3.30                    prior resumption growth. starvation <br /> nmol [14C]AMP incorporated mg tRNA, respec-                    period lengthened 20 h, recovery tively), confirming RNase T participates end turnover            CAN20-12E-rnt::kan observed, 6 h rich <br /> (13). Interruption rnt gene led additional           media (data shown). addition, starved threefold-lower defective tRNA (0.30 nmol                   20-12E-rnt::kan cells required 2 days colonies [14C]AMP incorporated mg tRNA), compared                  visible YT plates, strains showed strain residual RNase T activity.              colonies 1 day incubation (data shown). rnt-encoded activity abolished, small                 results suggest RNase T, combination 6%) defective tRNA present.  data                RNases, required normal recovery cell <br /> indicate &gt;90% end turnover tRNA occurs               growth starvation. E. coli RNase T small                    Additional physiological properties RNase T-deficient <br /> defective tRNA arises pathway.                                mutants. CA265, CA265-rnt::kan, CAN20-12E,    Aminoacylation tRNA wild-type rnt::kan <br /> strains. determine RNase T plays additional <br /> role tRNA metabolism, isolated tRNA wild-type rnt::kan strains tested ability accept                  TABLE 3. Doubling times wild-type RNase <br /> amino acid mixture amino acids E. coli                                          T-deficient strainsa <br /> aminoacyl-tRNA synthetases (Table 2). tRNA                             Strain                         Phenotype                  Doubling <br /> rnt::kan strains, CA265 containing rnt::kan (CA265-rnt::kan) CAN20-12E-rnt::kan, slightly reduced capacity               CA265                           Wild type                            25 <br /> accept amino acid comparison tRNA                    CA265-rnt::kan                  Deficient RNase T                 30 <br /> wild-type CAN20-12E strains. strains                CAN20-12E                       Deficient RNases                  30 wild type cca, decreased <span id='am-56' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-57' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span> activity                                                    BN, II, D, incomplete 3&apos; termini                    CAN20-12E-rnt::kan              Deficient RNases                  45 .  lower levels aminoacylation                                                BN, II, D,  T unprocessed 3&apos; tRNA ends.  RNase T                   Overnight cultures diluted 100-fold YT media grown 37&apos;C limited, detectable, role trimming 3&apos; ends          shaking. OD550 culture measured 30-min intervals. <br />  1380     PADMANABHA DEUTSCHER                                                                                             J. BACTERIOL. <br />  <br />                                                                         end-turnover process known affect small fraction <br />                                                                         E. coli tRNAs (10), process appears futile <br />                                                                         cycle.  difficult imagine absence                                                                         process affect growth. function RNase T <br />                                                                           likely. <br /> E   1074                                           CAN20-12E              possibility function 3&apos; processing                                                                         tRNA precursor(s). idea consistent                                                                         finding (Table 2) level functional tRNA slightly <br />                                                                         decreased  10%o) RNase T-deficient mutants n                                            CAN20-1 2E-rnt::kan        conditions CCA sequence intact <br />                                                                          10% decrease functional tRNA appear <br />                                                                         minimal, represent major defect                                                                         tRNA species affected). suggestion                                                                         consistent ability RNase T act synthetic <br />                                                                         tRNA precurs <br /> </body></html>