<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Methods Enzymol. 2010 ; 470: 181&#226;&#8364;&#8220;204. doi:10.1016/S0076-6879(10)70008-2. <br />  <br />  <br />  <br />                            Making temperature-sensitive mutants <br />  <br />                            Shay Ben-Aroya1,*, Xuewen Pan2,*, Jef D. Boeke3, Philip Hieter1 <br />                            1 Michael Smith Laboratories, Department Medical Genetics, University British Columbia, <br />  <br />                            Vancouver, BC Canada <br />                            2Verna Marrs McLean Department Biochemistry Molecular Biology, Department                            Molecular Human Genetics, Baylor College Medicine, Baylor Plaza, Houston, Texas <br />                            77030, USA <br />                            3High Throughput Biology Center, Johns Hopkins University school Medicine, Baltimore MD <br />                            21205 <br />  <br />                            Abstract <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  study temperature sensitive (Ts) mutant phenotypes fundamental gene identification                                  dissecting essential gene function. chapter &#226;&#8364;&#339;shuffling&#226;&#8364;? methods                                  producing Ts mutants using combination PCR, vivo recombination, transformation                                  diploid strains heterozygous knockout desired mutation. main difference                                  methods type strain produced. &#226;&#8364;&#339;plasmid&#226;&#8364;? version, product knockout <br />                                  mutant carrying centromeric plasmid carrying Ts mutant. &#226;&#8364;&#339;chromosomal&#226;&#8364;? version,                                  Ts allele integrated directly endogenous locus, albeit entirely native configuration. <br />                                  variations strengths weaknesses, discussed  <br />  <br />  <br />                            Introduction <br />                                              study temperature sensitive (Ts) mutant phenotypes proven fundamental <br />                                              approach identification gene sets essential various aspects biology                                              obtaining detailed understanding essential gene function. observation                                              temperature sensitive mutations represent general class mutation recognized                                              &#226;&#8364;&#732;50s (Horowitz, 1950), targeted screen, isolation, analysis Ts mutants (382 <br />                                              mutations located 37 genes scattered widely bacteriophage T4 genome) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Edgar Lielausis 1963 (Edgar Lielausis, 1964). Lee Hartwell, 1967, reported                                              isolation 400 Ts mutations S. cerevisiae, caused defects essential processes <br />                                              including cell division, protein, RNA, DNA synthesis (Hartwell, 1967). past <br />                                              40 years, isolation analysis Ts mutations essential genes linchpin <br />                                              technology investigating genetics molecular biology essential processes                                              experimental organisms. <br />  <br />                                              Ts mutations typically missense mutations, retain function specific essential <br />                                              gene standard (permissive) low temperature, lack function defined high (non- <br />                                              permissive) temperature, exhibit partial (hypomorphic) function intermediate (semi- <br />                                              permissive) temperature. mutants make possible analysis physiologic changes                                              follow controlled inactivation gene gene product shifting cells non-permissive <br />                                              temperature, offering powerful approach analysis gene function. <br />  <br />  <br />  <br />  <br />                             authors contributed equally work <br />                             Ben-Aroya et al.                                                                                               Page 2 <br />  <br />  <br />                                               Essential genes, <span id='am-13' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, encode critical cellular functions buffered                                               redundant functions pathways (Hartman et al., 2001). Essential genes shown                                               highly dense hubs genetic interaction networks involved aspects NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               basic cellular function (Jeong et al., 2001). Furthermore, essential genes tend highly <br />                                               conserved evolution; 38% essential yeast proteins easily identifiable counterparts <br />                                               humans, versus 20% nonessential genes (Hughes, 2002). <br />  <br />                                               Despite importance, functions essential yeast proteins studied. <br />                                                absence essential gene representation genome wide haploid <br />                                               mutant collections, cover ~5000 non-essential yeast genes.                                                comparable systematic haploid mutant collection currently exists ~1000 essential genes <br />                                               S. cerevisiae. frequency sites mutable reduced conditional function highly <br />                                               gene-specific; example, highly conserved proteins, random single missense mutation <br />                                               expected, vast majority positions protein, cause complete loss <br />                                               function mutated.  genetic screens using random mutagenesis approach <br />                                               rarely reach saturation &#226;&#8364;&#339;mutability&#226;&#8364;? varies widely genes. <br />  <br />                                               report detailed protocols methodologies allow systematic isolation                                               Ts alleles essential genes  method plasmid-based, second                                               genome integration based, specific advantages depending application. <br />                                               methods exploit features &#226;&#8364;&#339;haploid convertible&#226;&#8364;? heterozygous diploid collection, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               allows introduction library mutagenized essential gene copies                                               heterozygous diploid subsequent direct selection haploids deleted                                               target essential gene carry individual members mutagenized essential gene <br />                                               library, using &#226;&#8364;&#339;diploid shuffle&#226;&#8364;? technique  . useful corollary <br />                                               methods transferring extant Ts alleles specific gene constructs   fusion proteins)                                               presented. Finally, note laboratories process generating complete <br />                                               set Ts alleles essential genes S. cerevisiae, distributed                                               resource scientific community completed  Ben Aroya etal 2008 details). <br />                                               specific essential genes study individual laboratories,  useful, <br />                                               using methods described  generate additional series independent Ts alleles                                               detailed functional analysis. Furthermore, mutagenized libraries specific essential non- <br />                                               essential genes screened conditional viability variety conditions                                               normally sub-lethal wildtype strain   sublethal doses drugs) using methods <br />                                               described  <br />  <br />                                Diploid shuffle &#226;&#8364;&#8220; plasmid method <br />                               General description diploid shuffle &#226;&#8364;&#8220; plasmid method <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               like traditional plasmid-shuffling methods, mutants generated version                                               diploid shuffle plasmid-borne alleles easily transferred tested                                               different strain backgrounds. experimental procedure outlined Fig. 1.                                                endogenous promoter (including 5&#226;&#8364;&#178; untranslated region, 5&#226;&#8364;&#178;UTR, ~500bp) terminator <br />                                               (3&#226;&#8364;&#178;UTR, ~500bp) gene  favorite gene, YFG) PCR amplified                                               cloned tandem centromere-based yeast-E. coli shuttle vector, contains URA3 <br />                                               selectable marker yeast cells, pRS416. resultant promoter/terminator clone <br />                                               subsequently linearized endonuclease, typically NotI, cuts site pre- <br />                                               engineered promoter terminator. Simultaneously, sequence YFG, <br />                                               including open reading frame (ORF), complete promoter terminator <br />                                               regions, randomly mutagenized error prone PCR. linearized promoter/terminator <br />                                               plasmid PCR products combined transformed haploid-convertible <br />                                               heterozygous diploid deletion mutant (YFG/yfg&#206;&#8221;::kanMX4) gene                                               mutagenized. mutagenized PCR product cloned URA3 plasmid  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 3 <br />  <br />  <br />                                               recombination mediated terminal homologous DNA sequences PCR products <br />                                               linearized vector. Ura+ transformants subsequently cultured sporulation <br />                                               medium converted haploid cells growing medium allow growth NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               haploid MATa G418R Ura+ cells. cells, chromosomal wild-type copy YFG                                               deleted, allowing direct observation phenotypes plasmid-borne alleles. screen <br />                                               conditional alleles, haploid cells grown permissive condition                                               low temperature. Colonies formed subsequently replica-plated fresh plates permissive <br />                                               nonpermissive conditions. Conditional alleles identified grow                                               permissive non-permissive condition subsequently verified. method                                               used create thermosensitive (Ts) alleles multiple essential genes large <br />                                               collection methyl methanesulfonate (MMS) hypersensitive alleles POL30 (Huang et al., <br />                                               2008;Lin et al., 2008). outline detailed methods generating verifying <br />                                               Ts alleles essential gene. <br />  <br />                               Materials <br />                                               Media&#226;&#8364;&#8221;Haploid selection synthetic medium SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg+G418  dextrose, <br />                                               20 g/L; yeast nitrogen base amino acids ammonium sulfate, 1.7 g/L; SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220; <br />                                               Leu dropout mix, 2g/L; sodium glutamate, 1g/L; G418, 200 mg/L; L-canavanine <br />                                               (Sigma, Cat# C1625), 60 mg/L; Agar, 2%. sodium glutamate substituted ammonium <br />                                               sulfate nitrogen source makes G418 selection reliable minimal <br />                                               medium. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               SC&#226;&#8364;&#8220;Ura: dextrose, 20g/L; yeast nitrogen base amino acids ammonium sulfate, <br />                                               1.7g/L; SC&#226;&#8364;&#8220;Ura dropout mix, 2g/L; ammonium sulfate, 5g/L; Agar, 2%. <br />  <br />                                               Liquid YPD: yeast extract, 10g/L; peptone, 20g/L; dextrose, 20g/L. <br />  <br />                                               Solid liquid sporulation medium: potassium acetate, 10g/L; zinc acetate 0.05g/L,                                               2% agar respectively. <br />  <br />                                               Solid Luria Broth (LB) plus carbenicillin: yeast extract, 10g/L; Tryptone, 5g/L; sodium <br />                                               chrolide, 10g/L; carbenicillin, 50mg/L; Agar, 2%. <br />  <br />                                               Liquid LB plus ampicillin: yeast extract, 10g/L; Tryptone, 5g/L; sodium chrolide, 10g/L; <br />                                               ampicillin, 50mg/L. <br />  <br />                                               Yeast Strains haploid-convertible heterozygous diploid knockout mutants (MAT a&#206;&#177;/ <br />                                               ura3&#206;&#8221;0/ura3&#206;&#8221;0 leu2&#206;&#8221;0/leu2&#206;&#8221;0 his3&#206;&#8221;1/his3&#206;&#8221;1 lys2&#206;&#8221;0/LYS2 met15&#206;&#8221;0/MET15 can1&#206;&#8221;::LEU2- <br />                                               MFA1pr::His3/CAN1 YFG/yfg&#206;&#8221;::KanMX; OpenBiosystems Cat# YSC4428) (Pan et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               2006) used screen Ts alleles. Chemically competent DH5&#206;&#177; cells prepared described <br />                                               (Inoue et al., 1990) used cloning plasmid recovering yeast. <br />  <br />                                               Plasmids essential method use plasmid vector contains YFG <br />                                               promoter terminator separated unique endonuclease (typically NotI) recognition site. <br />                                               limited auxotrophic markers available haploid-convertible heterozygous <br />                                               diploid knockout mutants, normally use plasmids containing URA3 selectable marker <br />                                               pRS416 (Brachmann et al., 1998; Sikorski Hieter, 1989) YCplac33 (Gietz                                               Sugino, 1988). URA3 CEN vectors work. <br />  <br />                                               Yeast genomic DNA genomic DNA sample isolated wild-type yeast strain <br />                                               BY4743 MATa/&#206;&#177; (Brachmann et al., 1998) used template cloning promoter <br />                                               terminator YFG mutagenizing entire sequence PCR. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 4 <br />  <br />  <br />                               Methods <br />                                               Constructing promoter/terminator clone past, used <br />                                               endonuclease restriction enzyme digestion ligation construct promoter/terminator <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               clone.  promoter terminator YFG separately PCR-amplified using primers <br />                                               contain endonuclease recognition sites, example, HindIII/NotI promoter                                               NotI/BamHI terminator. PCR products digested HindIII/NotI                                               NotI/BamHI, respectively, ligated pRS416  YCplac33) digested HindIII/BamHI <br />                                               3-piece ligation reaction. ligation products transformed DH5&#206;&#177; competent cells <br />                                               candidate clones selected solid LB plus carbenicillin. recently,                                               adopted modified version sequence ligation independent cloning (SLIC) procedure <br />                                               (Li Elledge, 2007) (Figure 2). method does require endonuclease digestion                                               inserts greatly simplifies primer designs experimental procedures, especially <br />                                               large number genes processed simultaneously. relatively new <br />                                               cloning technique described greater  <br />                                                   1. Design PCR primers: promoter forward (PF), promoter reverse (PR), terminator <br />                                                   forward (TF), terminator reverse (TR). addition gene specific sequences                                                   3&#226;&#8364;&#178; termini, PF TR primers contains 30 bp sequence 5&#226;&#8364;&#178; end                                                   identical complimentary ends pRS416  YCplac33) linearized                                                   multicloning site (Fig. 2). PR TF primers completely complementary                                                   contain parts: ~20 bp promoter- terminator-specific sequences ends NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   NotI recognition site (8 bp) middle (Fig. 2). <br />                                                   2a. PCR-amplify promoter terminator according conditions listed Table <br />                                                   1. genomic DNA BY4743a/&#206;&#177; used PCR template. Platinum Pfx DNA <br />                                                   polymerase (Invitrogen, Cat# 11708) used relative robustness high fidelity. <br />                                                   enzymes similar features Phusion (New England Biolabs, NEB, <br />                                                   Cat# F-540) KOD (Novagen, Cat# 71085-3) DNA polymerases used. <br />                                                   2b. Digest ~1ug pRS416, YCplac33, URA3 plasmid endonuclease <br />                                                   multicloning site generate ends competent cloning PCR products <br />                                                   SLIC. <br />                                                   3. Gel-purify PCR products digested vector DNA using gel extraction kit <br />                                                   (Qiagen, Cat# 28706 equivalent) following manufacturer&#226;&#8364;&#8482;s instruction.                                                   PCR products promoter terminator combined purification. Elute <br />                                                   purified sample 25&#206;&#188;l provided elution buffer. <br />                                                   4. Set reaction according Table 2. DNA resection 3&#226;&#8364;&#178; 5&#226;&#8364;&#178; direction                                                    T4 DNA polymerase, NEB, Cat# M0203) annealing complementary single <br />                                                   strand overhangs occurs reaction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   5. Immediately use 2&#206;&#188;l reaction transform 20&#206;&#188;l chemically competent <br />                                                   DH5&#206;&#177; cells following standard protocol. includes incubating cell/DNA <br />                                                   mixture sequentially ice 30 minutes, 42&#194;&#176;C 90 seconds, ice 2 <br />                                                   minutes, 37&#194;&#176;C 30 minutes  presence 200 &#206;&#188;l liquid LB medium). <br />                                                   6. Plate 100&#206;&#188;l transformation mixture solid LB plus carbenicillin select                                                   single colony transformants. <br />                                                   7. Screen positive clones using colony-PCR PF PR primers according                                                   Table 1 subtle modifications. cells single colonies instead yeast <br />                                                   genomic DNA used provide PCR templates. <br />                                                   8. Prepare plasmid DNA samples 2&#226;&#8364;&#8220;3 positive clones using mini-spin kit (Qiagen, <br />                                                   Cat# 27106 equivalent) following manufacture&#226;&#8364;&#8482;s instructions. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 5 <br />  <br />  <br />                                                   9. Verify plasmid DNA sequencing using primers read                                                   inserts (promoter terminator) directions vector backbone. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Mutagenizing YFG error prone PCR&#226;&#8364;&#8221;Mutagenesis YFG using error-prone PCR <br />                                               performed essentially described previously (Leung et al, 1989).  genomic DNA <br />                                               sample wild-type yeast strain BY4743a/&#206;&#177; used DNA template. PF TR <br />                                               primers described used amplify length gene ~500bp flanking <br />                                               sequences. TaKaRa Ex Taq (Cat# RR001A) LA Taq (Cat# RR002B), ~4 times <br />                                               accurate normal Taq polymerase, used robustness. Induction <br />                                               mutation rates achieved adding Mn2+ PCR final concentration 10&#226;&#8364;&#8220;150 <br />                                               &#206;&#188;M arbitrarily defined, higher concentrations smaller genes (genes&#226;&#8364;&#8482; sizes range <br />                                               0.5&#226;&#8364;&#8220;5kb). <br />                                                   1.   Set independent reactions gene according Table 3 reduce <br />                                                        potential founder effects. <br />                                                   2.   PCR, pool samples. <br />                                                   3.   Examine PCR products agarose gel electrophoresis using 1&#226;&#8364;&#8220;2 &#206;&#188;l sample <br />                                                        ensure successful PCR amplification. <br />  <br />                                               Linearizing plasmid DNA promoter/terminator clone &#206;&#188;g plasmid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               DNA promoter/terminator clone digested NotI (NEB, Cat# R0189) NEB buffer <br />                                               3 20&#206;&#188;l reaction. reaction incubated 37&#194;&#176;C overnight subsequently                                               65&#194;&#176;C 20 minutes inactivate NotI. small aliquot digestion product examined <br />                                               agarose gel electrophoresis ensure complete digestion plasmid. <br />  <br />                                               Combining concentrating PCR products digested vector                                               mutagenized PCR products (approximately 10&#226;&#8364;&#8220;20 &#206;&#188;g 200 &#206;&#188;l) linearized plasmid DNA <br />                                               promoter/terminator clone (~2 &#206;&#188;g) combined concentrated ethanol <br />                                               precipitation. <br />                                                   1.   Transfer mutagenesis PCR products NotI-digested promoter/terminator <br />                                                        plasmid DNA 1.7 ml microcentrifuge tube adjust volume ~200 &#206;&#188;l                                                        adding ddH2O. <br />                                                   2.   Add 5 &#206;&#188;l 4M ammonium acetate (pH 7.0) 500 &#206;&#188;l 100% ethanol DNA <br />                                                        sample mix briefly vortexing. Place ice 10 minutes. <br />                                                   3.   Precipitate DNA spinning &gt;12,000rpm microcentrifuge 7 minutes.                                                        tiny whitish DNA pellet tube. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   4.   Carefully aspirate liquid wash DNA pellet 300 &#206;&#188;l 70% <br />                                                        ethanol. <br />                                                   5.   Spin &gt;12,000rpm microcentrifuge 3 minutes carefully aspirate ethanol. <br />                                                   6.   Dry DNA pellet speed vac. <br />                                                   7.   Resuspend DNA 28 &#206;&#188;l sterile ddH2O. <br />  <br />                                               Transforming yeast cells concentrated DNA sample PCR products                                               linearized vector transformed corresponding haploid-convertible heterozygous <br />                                               diploid yeast knockout mutant create mutagenized library YFG. <br />                                                   1.   Inoculate haploid-convertible YFG/yfg&#206;&#8221;::kanMX4 heterozygous diploid mutant <br />                                                        5 ml liquid YPD incubate 30&#194;&#176;C overnight roller drum. <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 6 <br />  <br />  <br />                                                   2.   Transfer aliquot overnight culture 50 ml YPD liquid (starting 0.125 <br />                                                        OD600nm/ml 250 ml Erlenmeyer flask) incubate 30&#194;&#176;C shaking  200 <br />                                                        rpm) cell density ~0.5 OD600nm/ml. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   3.   Harvest culture 50 ml conical tube spinning 4000 rpm 3 minutes                                                        bench centrifuge discard medium. <br />                                                   4.   Resuspend cells 10 ml sterile ddH2O, centrifuge described previous <br />                                                        step, discard supernatant. <br />                                                   5.   Resuspend cells 10 ml 0.1 M lithium acetate (LiOAc), centrifuge, discard <br />                                                        supernatant  <br />                                                   6.   Resuspend cells residual 0.1 M LiOAc total volume 100 &#206;&#188;l 1.7 ml <br />                                                        microcentrifuge tube. <br />                                                   7.   Make transformation mixture order mix  480 &#206;&#188;l 50% polyethylene <br />                                                        glycol (PEG-3350, JTBaker, Cat# JTU221-9), 72 &#206;&#188;l 1M lithium acetate, 40 &#206;&#188;l                                                        heat-denatured herring sperm DNA (10mg/ml, Sigma, Cat# D6898) 28 &#206;&#188;l DNA <br />                                                        sample transformed (error-prone PCR products linearized promoter/ <br />                                                        terminator clone). <br />                                                   8.   Add transformation mixture yeast competent cells prepared step 6                                                        immediately mixed pipetting P1000 pipettor followed vortexing <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        (VMR, Vortexer 2) speed 5&#226;&#8364;&#8220;10 seconds. <br />                                                   9.   Incubate transformation reaction 30&#194;&#176;C incubator 30 minutes. <br />                                                   10. Add 72 &#206;&#188;l dimethyl sulfoxide (DMSO; Qbiogene DMSO0001, Molecular Biology <br />                                                       Grade) transformation reaction immediately mixed thoroughly vortexing <br />                                                       speed 5&#226;&#8364;&#8220;10 seconds. DMSO intrinsically sterile sterilization <br />                                                       needed. <br />                                                   11. Incubate transformation reaction 42&#194;&#176;C water bath 13 minutes.                                                   12. Spin cells 3600 rpm microcentrifuge 30 seconds pellet cells. <br />                                                   13. Aspirate supernatant resuspend cells 1 ml sterile ddH2O. <br />                                                   14. 0.2 &#206;&#188;l (1/5000) resuspended cells plate SC&#226;&#8364;&#8220;Ura plate determine <br />                                                       transformation yield. successful reaction typically yields library 105 106 <br />                                                       independent Ura+ transformants, &gt;90% recombinants PCR <br />                                                       products linearized plasmid DNA promoter/terminator clone.                                                       remainder primarily vector molecules cut place <br />                                                       rejoined nonhomologous end joining. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Sporulation rest yeast transformation reaction incubated 50 ml                                               fresh liquid SC&#226;&#8364;&#8220;Ura 30&#194;&#176;C 2 days allow propagation library sporulated <br />                                               immediately convert transformants library haploid spores harbor mutant <br />                                               alleles YFG plasmid  . <br />                                                   1.   Grow yeast transformants nonselectively 50 ml liquid YPD incubating                                                        30&#194;&#176;C shaking (180&#226;&#8364;&#8220;200 rpm) 3 hours refresh cells. <br />                                                   2.   Harvest cells 50 ml conical tube spinning 4000 rpm 3 minutes bench <br />                                                        centrifuge discard medium. <br />                                                   3.   Resuspend cells 40 ml sterile ddH2O spin 4000 rpm 3 minutes. <br />                                                   4.   Discard supernatant resuspend cells 50 ml liquid sporulation medium. <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 7 <br />  <br />  <br />                                                   5.   Incubate sporulation culture 250 ml flask 25&#194;&#176;C 4&#226;&#8364;&#8220;6 days shaking <br />                                                        (180&#226;&#8364;&#8220;200rpm). typically rise 20&#226;&#8364;&#8220;40% sporulation efficiency                                                        checked microscope. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   6.   Repeat Steps 2 3. <br />                                                   7.   Discard supernatant resuspend cells 10 ml sterile ddH2O. <br />                                                   8.   Spread aliquots (200 &#206;&#188;l) 10 &#195;&#8212; serial dilutions sporulation culture                                                        individual plates haploid selection medium SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg+G418 <br />                                                         incubate 25&#194;&#176;C 2&#226;&#8364;&#8220;3 days determine efficiency producing <br />                                                        MATa G418R Ura+ haploid cells. <br />                                                   9.   Store rest spores ddH2O 4&#194;&#176;C later use. viability spores                                                        maintained weeks way. <br />  <br />                                               Screening Ts alleles titer MATa G418R Ura+ haploid cells determined, <br />                                               library screened potential Ts mutants. typically screen ~4,000 clones                                               gene. <br />                                                   1.   Spread spores solid SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg+G418   aiming density <br />                                                        ~400 MATa G418R Ura+ haploid colonies formed plates. Store                                                        rest spores 4&#194;&#176;C backup. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   2.   Incubate plates 25&#194;&#176;C 3 days allow formation colonies ~2 mm                                                        diameter. <br />                                                   3.   Replica-plate colonies plate fresh plates haploid <br />                                                        selection medium mark orientations plates. best pre-warm                                                        &#226;&#8364;&#339;nonpermissive&#226;&#8364;? plate 37&#194;&#176;C prior replica plating. <br />                                                   4.   Incubate daughter plate 25&#194;&#176;C 37&#194;&#176;C 1 day. <br />                                                   5.   Compare growth colony plates assess potential Ts phenotype                                                        select alleles form relatively robust colonies 25&#194;&#176;C ghost colonies 37&#194;&#176;C <br />                                                        candidate Ts mutants. Ts colony detected, need screen                                                        clones using backup spores stored 4&#194;&#176;C, using 38&#194;&#176;C restrictive <br />                                                        temperature,  <br />  <br />                                               Confirming Ts mutants&#226;&#8364;&#8221;Candidate mutants picked streaked                                               haploid selection media tested Ts phenotypes incubating 25&#194;&#176;C 37&#194;&#176;C. <br />                                               plasmids recovered confirmed Ts initial assay  <br />                                               introduced individually haploid-convertible heterozygous diploid mutant test <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Ts phenotype linked plasmids. <br />                                                   1.   Grow candidate Ts mutant 1.5 ml liquid SC&#226;&#8364;&#8220;Ura 25&#194;&#176;C saturated, <br />                                                        typically takes 1&#226;&#8364;&#8220;2 days, depending particular alleles. <br />                                                   2.   Harvest cells strain microcentrifuge tube spinning 4000 rpm 1 <br />                                                        minute discard medium. <br />                                                   3.   Resuspend cells 500&#226;&#8364;&#8220;1000 &#206;&#188;l sterile ddH2O repeat Step 2. <br />                                                   4.   Resuspend cells 40 &#206;&#188;l Lysis Buffer (50 mM Tris-HCl 7.5, 10 mM EDTA) <br />                                                        containing 5 mg/ml Zymolyase 100T (MP Biomedicals, Cat# 320931) incubate <br />                                                        37&#194;&#176;C 1 hour shaking 15-minute intervals. <br />                                                   5.   Add 40 &#206;&#188;l 10% SDS mix vortexing pipetting. <br />                                                   6.   Add 160 &#206;&#188;l 7.5M ammonium acetate mix vortexing pipetting. <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                            Page 8 <br />  <br />  <br />                                                   7.   Incubate sample &#226;&#710;&#8217;80&#194;&#176;C 15 minutes. <br />                                                   8.   Centrifuge &gt;12,000rpm 5 minutes 4&#194;&#176;C. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   9.   Carefully transfer 100 &#206;&#188;l clear supernatant new tube contains 75 &#206;&#188;l                                                        isopropanol mix inverting tube times. <br />                                                   10. Centrifuge &gt;12,000rpm 7 minutes room temperature precipitate DNA. <br />                                                   11. Wash DNA pellet 100 &#206;&#188;l 70% ethanol. <br />                                                   12. Carefully aspirate ethanol dry DNA pellet speed vac. <br />                                                   13. Resuspend DNA pellet 20 &#206;&#188;l sterile ddH2O. <br />                                                   14. Use 2 &#206;&#188;l DNA sample transform chemically competent DH5&#206;&#177; cells                                                       mentioned &#226;&#8364;&#339;constructing promoter/terminator clone&#226;&#8364;? section. <br />                                                   15. Purify plasmid DNA representative bacterial transformant using kit (Qiagen, <br />                                                       Cat# 27106 equivalent). <br />                                                   16. Individually transform plasmid haploid-convertible heterozygous <br />                                                       diploid knockout mutant described previously (Gietz et al., 1995) select                                                       Ura+ transformants. <br />                                                   17. Patch representative Ura+ transformants plasmid solid SC&#226;&#8364;&#8220;Ura NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                       incubate 30&#194;&#176;C overnight. <br />                                                   18. Transfer cells sporulation plates incubate room temperature 4&#226;&#8364;&#8220;6 days. <br />                                                   19. Resuspend cells sporulated culture sterile ddH2O. <br />                                                   20. Spot aliquots 10x serial dilutions SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg+G418                                                       plates. <br />                                                   21. Incubate plate 25&#194;&#176;C 37&#194;&#176;C 2&#226;&#8364;&#8220;3 days. <br />                                                   22. Compare growth MATa G418R Ura+ cells temperatures. True Ts alleles <br />                                                       form colonies 25&#194;&#176;C 37&#194;&#176;C. <br />  <br />                               II. Diploid shuffle &#226;&#8364;&#8220; chromosome method <br />                               general description <br />                                               &#226;&#8364;&#339;diploid shuffle&#226;&#8364;? chromosome method used systematically screen                                               missense mutations result temperature sensitive (Ts) alleles hundreds essential <br />                                               genes, allele directly integrated endogenous chromosomal location flanked <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               &#226;&#8364;&#339;barcodes&#226;&#8364;? corresponding yeast knockout mutant (Fig 3; (Ben-Aroya et al., <br />                                               2008), unpublished data).  YFG, including promoter terminator regions,                                               mutagenized error-prone PCR (Fig. 3A). mutagenized PCR product cloned <br />                                               SB221+Topo-TA (Fig. 3B). plasmid contains URA3 gene flanked 5&#226;&#8364;&#178;                                               3&#226;&#8364;&#178; regions KanMX. Topo-TA cloning site (Invitrogen) inserted                                               KanMX 5&#226;&#8364;&#178; end URA3 gene. site allows direct cloning PCR products, <br />                                               need modifications. result cloning step library                                               mutagenized YFG, transformed E. coli, digested release linear <br />                                               fragments (following DNA purification) (Fig. 3B). linear fragments directly <br />                                               transformed corresponding strain haploid-convertible heterozygous YFG/ <br />                                               yfg&#206;&#8221;::kanMX diploid YKO collection selecting Ura+ transformants (Fig. 3C).                                               ~700bp KanMX5&#226;&#8364;&#178; KanMX3&#226;&#8364;&#178; fragments direct mutagenized YFG library                                               yfg&#206;&#8221;::KanMX genomic locus homologous recombination, retention original bar <br />                                               codes flanking gene (Fig. 3C). Pools Ura+ cells containing mutant alleles  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                Page 9 <br />  <br />  <br />                                               sporulated (Fig. 3E). Spores formed spread haploid selective medium                                               incubated 25&#194;&#176;C colony formation. haploid MATa Ura+ spores containing                                               integrated mutant allele YFG grow medium. Colonies formed selective <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               medium replica-plated incubated 37&#194;&#176;C (Fig. 3F). Colonies growing 25&#194;&#176;C                                               37&#194;&#176;C selected potential Ts alleles, retested. summary, final product                                               diploid shuffle approach confirmed MATa strain YKO collection genetic <br />                                               background containing URA3 marked Ts allele specific gene integrated                                               endogenous locus flanked barcodes. addition, strain contains LEU2- <br />                                               MFA1pr-HIS3 reporter integrated CAN1 locus. <br />  <br />                                               addition creating Ts alleles, diploid shuffle-chromosomal method used                                               transfer existing alleles knockout strain background strain backgrounds                                               vice versa. Using Topo-TA plasmid protocol described Fig. 1, extant Ts allele <br />                                               easily transferred deletion collection genetic background (referred &#226;&#8364;&#339;allele <br />                                               transfer ?). result integrated allele, marked URA3, flanked appropriate <br />                                               barcodes.  using Topo-TA plasmid, PCR product (mutant allele, fusion <br />                                               protein, heterologous gene expression cassette, ) introduced 6000 <br />                                               genomic sites carrying KanMX replacement cassette integration site, depending                                               specific deletion mutant chosen recipient strain. using primers external                                               primers used original mutagenesis, URA3 marked Ts-allele gene construct <br />                                               easily transferred deletion set genetic background strain  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               replace wild-type copy recipient strain homologous recombination (referred <br />                                               &#226;&#8364;&#339;allele transfer ?, Fig. 4).  Ts mutation gene construct analyzed <br />                                               specific phenotypes variety genetic contexts. <br />  <br />                               Materials <br />                                               Media&#226;&#8364;&#8221;Haploid selection medium SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg  dextrose, 20g/L; yeast <br />                                               nitrogen base amino acids ammonium sulfate, 1.7g/L; SC-Ura-Leu Arg <br />                                               dropout mix, 2g/L; sodium glutamate, 1g/L; L-canavanine, 60mg/L; Agar, 2%. YPD: yeast <br />                                               extract, 10g/L; peptone, 20g/L; dextrose, 20g/L; Agar, 2%. YPD+G418: yeast extract, 10g/L; <br />                                               peptone, 20g/L; dextrose, 20g/L; Agar, 2%; G418, 200 mg/L. <br />  <br />                                               Diploid selection medium SC-Ura+ClonNAT: dextrose, 20 g/L; yeast nitrogen base                                               amino acids ammonium sulfate, 1.7 g/L; SC&#226;&#8364;&#8220;Ura dropout mix, 2g/L; sodium glutamate, <br />                                               1g/L; ClonNAT, 200 mg/L; Agar, 2%. sodium glutamate substituted ammonium <br />                                               sulfate nitrogen source makes ClonNAT selection reliable minimal <br />                                               medium. <br />  <br />                                               described  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Strains genotype haploid-convertible heterozygous diploid strains similar                                               described  Haploid Ts strains following genotype: MATa ura3&#206;&#8221;0 <br />                                               leu2&#206;&#8221;0 his3&#206;&#8221;1 lys2&#206;&#8221;0  LYS2) met15&#206;&#8221;0  MET15) can1&#206;&#8221;::LEU2-MFA1pr::HIS3 yfg- <br />                                               ts::URA3. <br />  <br />                                               BY4742-ade2101-NatMX MAT&#206;&#177; wild-type haploid strain NatMX gene                                               linked ade2-101 ochre mutation. strain used confirm MATa Ura+ Ts <br />                                               candidate: MAT&#206;&#177; his3&#206;&#8221;1 leu2&#206;&#8221;0 lys2&#206;&#8221;0 ura3&#206;&#8221;0 ade2-101-NatMX <br />  <br />                                               OneShot&#194;&#174; 10 Electrocomp Cells (Invitrogen Cat #C4040-52) <br />  <br />                                               Plasmids&#226;&#8364;&#8221;SB221 derived M4758 (Voth et al., 2003). M4758, BglII/KpnI <br />                                               SphI/EcoRI fragments contain TEF promoter (388bp) terminator (262bp) <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                               Page 10 <br />  <br />  <br />                                               respectively Ashbya gossypii. SB221 fragments replaced BamHI/KpnI <br />                                               PCR fragment (731bp) containing TEF promoter plus half KanMX gene, SphI/ <br />                                               EcoRI fragment (751bp) containing half KanMX gene TEF terminator. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               cases template PCR products KanMX gene used constructing                                               heterozygous diploid collection. Finally BamHI site SB221, adjusted Topo- <br />                                               TA site (invitrogen) create SB221-Topo-TA. <br />  <br />                               Methods <br />                                               PCR mutagenesis carried described                                                 successfully used slightly different condition experiments using diploid shuffle <br />                                               chromosomal method. exclusively used LA Taq DNA polymerase (Cat# <br />                                               RR002B) 150 &#206;&#188;M MnCl2. PCR 50 &#206;&#188;l, instead  normally set                                               gene. primers, allow amplification entire coding region, 250&#226;&#8364;&#8220;300bp                                               5&#226;&#8364;&#178;UTR, 150&#226;&#8364;&#8220;200bp 3&#226;&#8364;&#178;UTR gene, used PCR. <br />  <br />                                               Cloning PCR products mutagenized PCR products purified cloned                                               E. coli cells electroporation. <br />                                                   1.   Purify PCR product using ChargeSwitch&#194;&#174; Clean Kit (Invitrogen Cat <br />                                                        #CS12000). <br />                                                   2.   Set ligation reaction includes following components: 0.5&#206;&#188;l SB221 TOPO- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        TA, 1.0 &#206;&#188;l ligation Buffer (300mM NaCl, 15mM MgCl2), purified PCR product <br />                                                        (100ng/1kb), ddH2O total volume 6 &#206;&#188;l. <br />                                                   3.   Incubate reaction room temperature overnight. <br />                                                   4.   Store reaction 4&#194;&#176;C  using soon) (&#226;&#710;&#8217;20&#194;&#176;C). <br />                                                   5.   1.5 &#206;&#188;l ligation products transform aliquot 50 &#206;&#188;l <br />                                                        OneShot&#194;&#174; 10 Electrocomp Cells (Invitrogen Cat #C4040-52)                                                        electroporation. <br />                                                   6.   Add 200 &#206;&#188;l LB transformation incubate 37&#194;&#176;C 1 hour. <br />                                                   7.   Plate 1 &#206;&#188;l aliquot LB plus ampicillin (LB-<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>) plate estimate                                                        transformation efficiency. <br />                                                   8.   Transfer 100 &#206;&#188;l transformation suspension flask containing 50 ml LB- <br />                                                        amp liquid media incubate 37&#194;&#176;C ~12h amplify plasmid library.                                                        necessary, store rest transformation suspension (approximately 150 &#206;&#188;l)                                                        20% glycerol &#226;&#710;&#8217;70&#194;&#176;C later use. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   9.   Count number colonies 1 &#206;&#188;l plate multiply microliters <br />                                                        inoculated LB-amp liquid media estimate complexity library. <br />                                                         total number colonies ~150,000&#226;&#8364;&#8220;200,000) <br />                                                   10. Purify plasmid DNA library using Plasmid Midi prep kit (Qiagen Cat #12745). <br />                                                   11. Digest 2 &#206;&#188;l 10x diluted library DNA sample NotI (10U/&#206;&#188;l, NEB, Cat#R018S) <br />                                                       10 &#206;&#188;l reaction examine digests agarose gel electrophoresis.                                                       allow estimation overall ligation efficiency. 2.5kb+2.7kb fragments represent <br />                                                       vector (Fig. 5A), successful cloning reaction indicated band <br />                                                       shift 2.7kb fragment  accordance insert size) (Fig. 5B). <br />  <br />                                               Yeast transformation&#226;&#8364;&#8221;Purified plasmid DNA sample random mutagenesis library <br />                                               digested NotI EcoRI (NEB, Cat#R0101T, 100u/&#206;&#188;l) transformed  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                               Page 11 <br />  <br />  <br />                                               corresponding haploid-convertible heterozygous knockout diploid mutant. mutant alleles <br />                                               integrated endogenous locus homologous recombination. <br />                                                   1.   Digest 40 &#206;&#188;g library plasmid DNA  night 37&#194;&#176;C) total volume 400ul <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        using restriction enzymes mentioned  <br />                                                   2.   Reduce reaction volume 100 &#206;&#188;l using speed-vac. <br />                                                   3.   Use digested library DNA transform corresponding heterozygous diploid <br />                                                        strain. Split digested DNA 50 &#206;&#188;l aliquots, use prepare <br />                                                        separate transformation suspension described (Pan et al., 2004). <br />                                                   4.   Combine transformation suspensions (~1284 &#206;&#188;l) plate 1 &#206;&#188;l SC&#226;&#8364;&#8220; <br />                                                        Ura plate estimate transformation yield. <br />                                                   5.   Transfer rest transformation suspension 50 ml liquid SC&#226;&#8364;&#8220;Ura incubate <br />                                                        30&#194;&#176;C 2 days amplify yeast library. <br />                                                   6.   Count number colonies 1 &#206;&#188;l plate multiply microliters <br />                                                        inoculated SC&#226;&#8364;&#8220;Ura estimate total transformation yield. number <br />                                                        15,000.  repeat yeast transformation. <br />  <br />                                               Yeast sporulation&#226;&#8364;&#8221;Amplified yeast library sporulated convert heterozygous <br />                                               diploid haploid spores similarly described   slightly different haploid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               selection medium, SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg  used determine efficiency                                               producing haploid MATa Ura+ cells sporulation culture. <br />  <br />                                               Screening Ts alleles plating efficiency determined, typically ~6000 <br />                                               haploid MATa Ura+ colonies screened candidate Ts alleles gene. <br />                                                   1.   Spread sporulation culture 15 plates SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg ~400 <br />                                                        colonies plate. <br />                                                   2.   Incubate plates 25&#194;&#176;C 3&#226;&#8364;&#8220;4 days. <br />                                                   3.   Replica-plate colonies plate fresh plate haploid selection <br />                                                        media mark orientation mother daughter plates. <br />                                                   4.   Incubate mother plates 25&#194;&#176;C daughter plates 37&#194;&#176;C. <br />                                                   5.   Assess Ts phenotype clone comparing growth mother                                                        daughter plates day. <br />  <br />                                               Confirming Ts phenotypes&#226;&#8364;&#8221;Candidate Ts mutants streaked haploid <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               selection media plates, incubate 25&#194;&#176;C 37&#194;&#176;C. Ts phenotype confirmed, <br />                                               backcross MATa Ura+ Ts candidate wild-type BY4742-ade2101-NatMX MAT&#206;&#177; strain. <br />                                               strain NatMX gene  provides resistance drug ClonNAT) linked                                               ade2-101 ochre mutation. Diploid cells selected streaking SC-Ura+ClonNAT <br />                                               media, sporulated, dissected. dissected tetrads replicated YPD (25&#194;&#176;C 37&#194;&#176; <br />                                               C), SC-Ura (25&#194;&#176;C), YPD supplemented G418 (25&#194;&#176;C). confirms  1)                                               temperature sensitivity segregates Mendelian manner (2:2), indicates Ts <br />                                               phenotype depends single mutated gene, 2) Ts phenotype linked URA3 gene, <br />                                               segregates mutated PCR product, 3) mutagenized PCR product <br />                                               integrated correct genomic locus rendering cells G418 sensitive (5&#226;&#8364;&#178;kanMX::yfg- <br />                                               ts-URA3::3&#226;&#8364;&#178;kanMX). <br />  <br />                                               Allele transfer carried similarly described subtle <br />                                               modifications. pre-existing Ts allele  gene construct) PCR-amplified  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                Page 12 <br />  <br />  <br />                                               appropriate plasmid genomic DNA template using proofreading-competent <br />                                               polymerase (e.g. LA Taq DNA polymerase) generate  ? overhang 3&#226;&#8364;&#178; end                                               PCR product. PCR products cloned SB221 using Topo TA cloning. cloned <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               PCR products released URA3 marker KanMX5&#226;&#8364;&#178;                                               KanMX3&#226;&#8364;&#178; fragments vector backbone trans formed corresponding haploid <br />                                               convertible heterozygous diploid mutant. Ura+ yeast transformants sporulated                                               population plated SC&#226;&#8364;&#8220;Ura&#226;&#8364;&#8220;Leu Arg select haploid MATa Ura+ cells <br />                                               appropriate colony density. Single colonies tested phenotype                                               (e.g. temperature sensitivity). Candidate clones backcrossed wild-type MAT&#206;&#177; <br />                                               strain analyzed tetrad dissection confirm phenotype. <br />  <br />                                               Allele transfer transfer Ts allele generated diploid <br />                                               shuffle method ura3 strain background. Unless stated, methodologies <br />                                               mentioned  <br />                                                   1.   Design PCR primers amplify 5&#226;&#8364;&#178;kanMX::yfg-ts-URA3::3&#226;&#8364;&#178;kanMX <br />                                                        cassette. primers specific 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions YFG recipient <br />                                                        strain. external primers originally used generate                                                        Ts allele donor strain (Fig. 4A). <br />                                                   2.   Set PCR using genomic DNA Ts mutant DNA template. <br />                                                        Transform PCR product strain select Ura+ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   3. <br />                                                        transformants. <br />                                                   4.   recipient strain haploid, screen colonies Ts phenotype. <br />                                                   5.   Backcross wild-type strain opposite mating type. <br />                                                   6.   Verify Ts phenotype linked Ura+ tetrad analysis. <br />                                               recipient strain diploid. case, Ura+ transformants selected Step <br />                                               3, sporulated, characterized tetrad analysis ensure Ts Ura+ phenotypes <br />                                               segregate. allow testing Ts allele viable particular genetic <br />                                               context recipient strain. possible allele longer Ts strain <br />                                               background.  representative Ura+ transformants need characterized                                               diagnostic PCR sequencing ensure 5&#226;&#8364;&#178;kanMX::yfg-ts-URA3::3&#226;&#8364;&#178;kanMX cassette                                               integrated right locus. <br />  <br />  <br />                                               Perspectives <br />                                               variations &#226;&#8364;&#339;diploid shuffle&#226;&#8364;? highly efficient methods making Ts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               mutants. essentially possible, screening mutants using methods <br />                                               outlined  alleles. adaptation methods outlined                                               required making Ts mutants large essential genes  case, gene                                               mutagenized sections). relative advantages methods described                                               summarized Table 4. <br />                                               chosen forward chromosomal method generation                                               genome-wide collection Ts mutants community resource felt                                               users prefer integrated copies fluctuate lost                                               subpopulation cells division episome. <br />                                               ability generate genome wide collection compares favorably attempts <br />                                               generate genome wide resources study essential genes, Tet-regulated <br />                                               alleles (Hartman et al., 2001; Mnaimneh et al., 2004) <span id='am-8' about='obo:IMR_0200160' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:id" content="IMR:0200160" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="dAMP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00360" datatype="xsd:string"></span>dAMP</span> (Schuldiner et al., 2005). <br />                                               approaches, having distinct advantages, produced  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                    Page 13 <br />  <br />  <br />                                               behaved alleles 30% cases. Disadvantages Ts mutants include fact <br />                                               uniform regard nonpermissive temperature leakiness, fact <br />                                               needed temperature shifts induce heat shocks effects NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               potentially cloud phenotypic analyses. alleles bastion                                               traditional genetic analyses essential genes. Ts alleles sequenced include                                               mix single <span id='am-14' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> substituions multi amino acid substitutions;                                                sequenced develop extensive statistics  Studies collections <br />                                               Ts mutants variety genes allowed empirical determination typical <br />                                               characteristics. basis, schemes predicting Ts mutants developed (Ye, <br />                                               P., Dymond, J., Shi, X., Lin, Y.-Y., Pan, X., Boeke J.D. Bader, J.S., submitted).                                               likely prove useful designing Ts mutants use organisms                                               extensive screening impractical. <br />  <br />  <br />                               References <br />                                               Ben-Aroya S, et al. Comprehensive Temperature-Sensitive Mutant Repository Essential <br />                                                  Genes Saccharomyces cerevisiae. Mol Cell 2008;30:248&#226;&#8364;&#8220;58. [PubMed: 18439903] <br />                                               Brachmann CB, et al. Designer deletion strains derived Saccharomyces cerevisiae S288C: useful <br />                                                  set strains plasmids PCR-mediated gene disruption applications. Yeast <br />                                                  1998;14:115&#226;&#8364;&#8220;32. [PubMed: 9483801] <br />                                               Edgar RS, Lielausis  Temperature-Sensitive Mutants Bacteriophage T4d: Isolation Genetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Characterization. Genetics 1964;49:649&#226;&#8364;&#8220;62. [PubMed: 14156925] <br />                                               Gietz RD, et al. Studies transformation intact yeast cells LiAc/SS-DNA/PEG procedure. <br />                                                  Yeast 1995;11:355&#226;&#8364;&#8220;60. [PubMed: 7785336] <br />                                               Gietz RD, Sugino  New yeast-Escherichia coli shuttle vectors constructed vitro mutagenized <br />                                                  yeast genes lacking base pair restriction sites. Gene 1988;74:527&#226;&#8364;&#8220;34. [PubMed: 3073106] <br />                                               Hartman, JLt, et al. Principles buffering genetic variation. Science 2001;291:1001&#226;&#8364;&#8220;4. [PubMed: <br />                                                  11232561] <br />                                               Hartwell LH. Macromolecule synthesis temperature-sensitive mutants yeast. J Bacteriol <br />                                                  1967;93:1662&#226;&#8364;&#8220;70. [PubMed: 5337848] <br />                                               Horowitz NH. Biochemical genetics Neurospora. Adv Genet 1950;3:33&#226;&#8364;&#8220;71. [PubMed: 15425388] <br />                                               Huang  et al. Plasmid-chromosome shuffling non-deletion alleles yeast. Nat Methods <br />                                                  2008;5:167&#226;&#8364;&#8220;9. [PubMed: 18193055] <br />                                               Hughes TR. Yeast drug discovery. Funct Integr Genomics 2002;2:199&#226;&#8364;&#8220;211. [PubMed: 12192593] <br />                                               Inoue H, et al. High efficiency transformation Escherichia coli plasmids. Gene 1990;96:23&#226;&#8364;&#8220;8. <br />                                                   [PubMed: 2265755] <br />                                               Jeong H, et al. Lethality centrality protein networks. Nature 2001;411:41&#226;&#8364;&#8220;2. [PubMed: 11333967] <br />                                               Li MZ, Elledge SJ. Harnessing homologous recombination vitro generate recombinant DNA NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   SLIC. Nat Methods 2007;4:251&#226;&#8364;&#8220;6. [PubMed: 17293868] <br />                                               Leung DW, Chen E, Goeddel DV. method random mutagenesis defined DNA segment using <br />                                                   modified polymerase chain reaction. J Cell Mol Biol 1989;1:11&#226;&#8364;&#8220;15. <br />                                               Lin YY, et al. comprehensive synthetic genetic interaction network governing yeast histone acetylation <br />                                                   deacetylation. Genes Dev 2008;22:2062&#226;&#8364;&#8220;74. [PubMed: 18676811] <br />                                               Mnaimneh S, et al. Exploration essential gene functions titratable promoter alleles. Cell <br />                                                   2004;118:31&#226;&#8364;&#8220;44. [PubMed: 15242642] <br />                                               Pan X, et al. DNA integrity network yeast Saccharomyces cerevisiae. Cell 2006;124:1069&#226;&#8364;&#8220;81. <br />                                                   [PubMed: 16487579] <br />                                               Pan X, et al. robust toolkit functional profiling yeast genome. Mol Cell 2004;16:487&#226;&#8364;&#8220;96. <br />                                                   [PubMed: 15525520] <br />                                               Schuldiner M, et al. Exploration function organization yeast early secretory pathway <br />                                                   epistatic miniarray profile. Cell 2005;123:507&#226;&#8364;&#8220;19. [PubMed: 16269340] <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                              Page 14 <br />  <br />  <br />                                               Sikorski RS, Hieter P. shuttle vectors yeast host strains designed efficient <br />                                                   manipulation DNA Saccharomyces cerevisiae. Genetics 1989;122:19&#226;&#8364;&#8220;27. [PubMed: 2659436] <br />                                               Voth WP, et al. New &#226;&#8364;&#732;marker swap&#226;&#8364;&#8482; plasmids converting selectable markers budding yeast gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   disruptions plasmids. Yeast 2003;20:985&#226;&#8364;&#8220;93. [PubMed: 12898713] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 1. schematic creating conditional alleles using plasmid-chromosome shuffle <br />                                               promoter (5&#226;&#8364;&#178;) terminator (3&#226;&#8364;&#178;) YFG separately PCR-amplified primer pairs <br />                                               PF/PR TF/TR, respectively, cloned centromeric (CEN) yeast-E. coli <br />                                               shuttling vector. PF stands promoter forward, PR promoter reverse, TF                                               terminator forward, TR terminator reverse. NotI recognition site engineered <br />                                               promoter terminator. resultant promoter/terminator clone linearized <br />                                               NotI digestion.  entire sequence YFG gene, including coding <br />                                               region promoter terminator sequences, mutagenized using error-prone PCR                                               primer pair PF/TR. mutagenesis PCR products linearized promoter/terminator <br />                                               plasmid DNA mixed transformed haploid-convertible heterozygous <br />                                               diploid knockout mutant gene (MATa/&#206;&#177; YFG/yfg&#206;&#8221;::kanMX4 CAN1/can1&#206;&#8221;::LEU2- <br />                                               MFA1pr-HIS3). linearized promoter/terminator clones repaired inside yeast cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               homologous recombination using transformed mutagenesis products                                                YFG* alleles) templates. extensive homology ends PCR <br />                                               product vector, &gt;105 recombinant clones easily generated. pool                                               recombinants sporulated. Haploid MATa G418R Ura+ cells selected                                               permissive condition solid SC-Ura-Leu Arg+G418 medium single colonies, <br />                                               replica-plated fresh plates incubated permissive non- <br />                                               permissive conditions. Candidate alleles grow permissive condition                                               non-permissive condition. <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                            Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 2. Constructing promoter/terminator clone using sequence ligation independent cloning <br />                                               (SLIC) <br />                                               promoter (5&#226;&#8364;&#178;) terminator (3&#226;&#8364;&#178;) YFG separately PCR-amplified yeast genomic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               DNA primer pairs PF/PR TF/TR, respectively.  centromeric yeast- <br />                                               E. coli shuttling vector linearized endonuclease digestion multicloning site <br />                                               (MCS). PCR products linear vector plasmid mixed processed                                               T4 DNA polymerase create 5&#226;&#8364;&#178; single-stranded overhangs. PCR primers designed                                               way PCR products vector assembled homology-mediated <br />                                               single strand annealing process. NotI site engineered cloned promoter                                               terminator. aliquot annealing reaction transformed E. coli competent cells. <br />                                               modified version SLIC procedure originally described Li Elledge <br />                                               (2007). <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                             Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 3. diagram &#226;&#8364;&#339;Diploid shuffle&#226;&#8364;? method generating temperature sensitive alleles <br />                                                ) Genomic DNA containing YFG 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions used template PCR <br />                                               mutagenesis. black horizontal arrows represent gene-specific primers used.                                               mutagenized PCR products cloned vector SB221+Topo-TA (mutations                                               represented black stars). Topo-TA cloning site represented black T,                                               overhang protruding PCR product represented black  Left gray bar represents <br />                                               5&#226;&#8364;&#178; half KanMX selectable marker (Kan), right gray bar represents half <br />                                               KanMX selectable marker (MX). NotI restriction sites indicated 2 diagonal <br />                                               black arrows. (B.) product cloning step library mutagenized YFEG.                                               library transformed E. coli, digested NotI release linear fragments <br />                                               (following DNA purification). (C.) linearized library transformed corresponding <br />                                               heterozygous diploid strain. Bars flank KanMX knockout represent Barcodes. <br />                                               (D.) Heterozygous diploid transformants sporulated (following meiosis), MATa Ura+ <br />                                               haploids spores selected haploid selective medium 25&#194;&#176;C. (E.) Selection temperature <br />                                               sensitive candidates following replica plating incubating 25&#194;&#176;C C 37&#194;&#176;C. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               arrows identify potential Ts allele. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                 Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 4. Allele transfer                                               Unless stated, symbols Fig. 3  ) specific example, genomic <br />                                               DNA containing Ts allele generated diploid shuffle method PCR amplified. <br />                                               black arrows represent primers used amplification. primers specific                                               5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions YFG recipient strain. external primers <br />                                               originally used generate Ts allele donor strain (represented broken arrows). <br />                                               (B.) PCR product transformed strain  specific example shows allele <br />                                               transfer haploid strain.  desirable transfer allele diploid strain <br />                                                followed sporulation tetrad dissection, Ts allele transferred inviable <br />                                               specific genetic context  Ts allele replaces wild-type YFG                                               homologous recombination (represented dashed lines) rise Ura+ transformants <br />                                               (indicated Ts phenotype). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                         Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Fig. 5. Testing Topo-TA cloning efficiency SB221 <br />                                               Unless stated, symbols Fig. 3  ) SB221 vector                                               URA3 gene flanked fragment containing TEF promoter plus half KanMX <br />                                               gene (Kan), fragment containing half KanMX gene TEF <br />                                               terminator (Mx). 2.7kb fragment excised 2.5kb backbone restriction <br />                                               digestion NotI. (B.) Restriction digest NotI following successful Topo-TA cloning <br />                                               reaction indicated 2.5kb backbone band shift 2.7kb fragment,                                               accordance insert size. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                     Page 20 <br />  <br />  <br />  <br />  <br />                                                                                               Table 1 <br />                            PCR amplification promoters terminators. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                                       Volume/reaction <br />                             10 &#195;&#8212; Platinum Pfx DNA Polymerase buffer               2.5 &#206;&#188;l <br />  <br />                             dNTP (2.5 mM                                      2 &#206;&#188;l <br />  <br />                             Primer mix (5 &#206;&#188;M                                  2 &#206;&#188;l <br />  <br />                             Yeast genomic DNA (~200 ng/&#206;&#188;l)                         1 &#206;&#188;l <br />  <br />                             ddH2O                                                17.25 &#206;&#188;l <br />  <br />                             Platinum Pfx DNA Polymerase (2.5 units/&#206;&#188;l)           0.25 &#206;&#188;l <br />  <br />                             Total                                                 25 &#206;&#188;l <br />  <br />                            PCR conditions: 94&#194;&#176;C 4 min; 30 &#195;&#8212; (94&#194;&#176;C 30 sec, 55&#194;&#176;C 30 sec, 72&#194;&#176;C 45 sec); 72&#194;&#176;C 7 min; hold 4&#194;&#176;C. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                         Page 21 <br />  <br />  <br />  <br />  <br />                                                                                                    Table 2 <br />                            T4 DNA polymerase resection reaction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                              Volume <br />                             10 &#195;&#8212; NEB BamHI Buffer                    2 &#206;&#188;l <br />  <br />                             10 &#195;&#8212; BSA                                 2 &#206;&#188;l <br />  <br />                             ddH2O                                   3.7 &#206;&#188;l <br />  <br />                             T4 DNA polymerase (3 units/&#206;&#188;l)          0.3 &#206;&#188;l <br />  <br />                             Vector (100 ng/&#206;&#188;l)                       2 &#206;&#188;l <br />  <br />                             PCR product (50 ng/&#206;&#188;l)                  10 &#206;&#188;l <br />  <br />                             Total                                   20 &#206;&#188;l <br />  <br />                            set  reaction incubated 25&#194;&#176;C 30 minutes.  sit ice E. coli transformation. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                        Methods Enzymol. Author manuscript; available PMC 2010 October 20. <br />                             Ben-Aroya et al.                                                                                                                      Page 22 <br />  <br />  <br />  <br />  <br />                                                                                                 Table 3 <br />                            Error-prone PCR conditions. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Component                                 Volume/reaction <br />  <br />                             10 &#195;&#8212; Ex Taq buffer                               5 &#206;&#188;l <br />  <br />                             dNTP (2.5 mM                                4 &#206;&#188;l <br />  <br />                             Primer mixtur <br /> </body></html>