<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Concurrent Growth Rate Transcript Analyses Reveal <br /> Essential Gene Stringency Escherichia coli <br /> Shan Goh1,2, Jaroslaw M. Boberek2, Nobutaka Nakashima3, Jem Stach4, Liam Good1,2* <br /> 1 Department Cell Molecular Biology, Karolinska Institute, Stockholm, Sweden, 2 Department Pathology Infectious Diseases, Royal Veterinary College, <br /> University London, London, United Kingdom, 3 Research Institute Genome-based biofactory, Toyohira-Ku, Sapporo, Japan, 4 School Biology, University Newcastle, Newcastle Tyne, United Kingdom <br />  <br />  <br />  <br />      Abstract <br />      Background: Genes essential bacterial growth particular scientific  putative essential genes      identified predicted species,  little known gene expression requirement stringency, <br />      important aspect bacterial physiology likely determining factor drug target development. <br />  <br />      Methodology/Principal Findings: Working premise essential genes differ absolute requirement growth, <br />      silencing putative essential genes E. coli obtain titration declining growth rates transcript levels <br />      using antisense <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> nucleic acids (PNA) expressed antisense RNA. relationship mRNA decline      growth rate decline reflects degree essentiality, stringency, essential gene, defined      minimum transcript level 50% reduction growth rate (MTL50). applied growth essential genes,      RNA silencing methods resulted MTL50 values reveal acpP stringently required genes <br />      examined, ftsZ stringently required. established antibacterial targets murA fabI      stringently required. <br />  <br />      Conclusions: RNA silencing reveal stringent requirements gene expression respect growth. method      used validate existing essential genes quantify drug target requirement. <br />  <br />   Citation: Goh S, Boberek JM, Nakashima N, Stach J, Good L (2009) Concurrent Growth Rate Transcript Analyses Reveal Essential Gene Stringency   Escherichia coli. PLoS 4(6): e6061. doi:10.1371/journal.pone.0006061 <br />   Editor: Laurent nia, BMSI STAR, Singapore <br />   Received November 3, 2008; Accepted June 2, 2009; Published June 26, 2009 <br />   Copyright: &#195;&#376; 2009 Goh et al. open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: Swedish Institute Guest Scholarship SG, Bloomsbury Colleges scholarship JMB, Swedish Research Council, University London <br />   Central Research Fund, Sasakawa Foundation. funders role study design, data collection analysis, decision publish, preparation   manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: lgood@rvc.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                             hypothesize growth essential genes differ require- <br />                                                                                       ment stringencies, differences revealed    Progress antimicrobial discovery slow recent                    measuring relationships specific mRNA decreases decades [1] despite large-scale efforts identify genes essential               bacterial growth rate decline. aim study measure <br /> growth conserved pathways Escherichia coli [2] Staphylococcus               growth stringency requirement E. coli genes using antisense <br /> aureus [3]. drug discovery process involves essential gene                        gene silencing, evaluate degree essentiality cell <br /> identification various methods chromosomal                            viability. antisense gene control strategies bacteria deletions [2] experimentally reduced genomes [4], followed                        used - synthetic antisense peptide nucleic acids (PNAs) plasmid studies gene product interactions tandem affinity                       expression antisense RNA sequences. methods capable <br /> purification [5] mathematical models [6]. Potential drug                       modulating essential genes situ E. coli [11,12] S. aureus <br /> targets subjected high-throughput screening                           [13,14], certain antisense agents inhibit Mycobacteria <br /> inhibitors, fail cellular level &#226;&#8364;&#732;&#226;&#8364;&#732;hit lead&#226;&#8364;&#8482;&#226;&#8364;&#8482;                 [15,16].  RNA silencing using specific antisense <br /> stages development factors [7], gene                        sequences provides approach measure gene requirement <br /> function redundancy [8].  certain essential genes suspect                 stringency. low degree gene expression requirement                        genes, acpP, fabI, ftsZ murA selected study <br /> bacterial viability. Despite evidence differential requirement                 following reasons:   gene shown <br /> growth essential genes [9], initial efforts scoring essentiality               experimentally essential growth E. coli <br /> [2,10], quantitative method determine                     [2,4,17,18,19,20], (b) studied antimicrobial <br /> expression requirement essential genes stringently                   targets [21,22,23], (c) gene products different cellular <br /> required targets considered drug development.                    functions (d) gene silenced potentially study focuses measuring differences suspected essential                    downstream effects [24,25]. Interactions FabI genes terms degree requirement cell viability,                     specific inhibitor, triclosan, understood [26]. fabI <br /> referred stringency, E. coli.                                           gene encodes enoyl ACP reductase, catalyzes fatty acid <br />  <br />  <br />        PLoS | www.plosone.org                                                 1                                    June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                               E. coli Gene Stringency <br />  <br />  <br /> elongation [17]. studied drug target <span id='am-23' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="UDP" datatype="xsd:string"></span>UDP</span>-N-                    antisense expressed plasmids, constructed acetylglucosamine enolpyruvyl transferase (encoded murA),               DH5a strains containing pBAD-acpP+pHNA, pBAD- catalyzes synthesis <span id='am-12' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-13' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> N-acetylglu-               fabI+pHN682, pBAD-ftsZ+pHNZ pBAD-murA+pHNM. cosamine acid phosphoenoylpyruvate [10,18,27]. MurA                     strain grown presence IPTG induction specifically inhibited phosphomycin [28], mechanism                 expressed antisense, addition L- inhibition understood [29]. protein function FtsZ            arabinose induce target gene expression. L- studied  FtsZ focus assay           arabinose, growth strains inhibited, L-arabinose <br /> development discover effective inhibitors [30]. ftsZ gene               addition complemented gene silencing allowed strains encodes tubulin-like protein polymerizes form ring            grow presence expressed antisense (Figure 1). results scaffold cell division [31]. ACP acyl carrier protein              rescue experiments demonstrate silencers <br /> encoded acpP. ACP interesting drug target                     display gene selective growth inhibition. shown central bacterial fatty acid <br /> biosynthesis [32,33], specific protein inhibitors            Titration essential gene expression effect discovered.  acpP shown effective                    growth rate <br /> antisense target treatment bacterial infections vivo [34,35]            test hypothesis essential genes differentially gene product inhibited [36]. <br />                                                                                required maintain similar level growth, titrated    principle, extent partial mRNA inhibition limits            growth rate E. coli using antisense PNA expressed <br /> growth reflect stringently required targeted mRNA               antisense determined mRNA levels targeted essential bacterial viability. specificities antisense PNA             gene. enable parallel analyses multiple genes study, <br /> expressed antisense RNA silencing selected target genes <br />                                                                                used RT-PCR quantify mRNA levels assume assessed conducting rescue experiments,  <br />                                                                                proxy protein measure bacteria [39]. PNA-mediated <br /> expression target genes different strains comple- <br />                                                                                antisense effects, used hyper-permeable E. coli strain AS19 <br /> mented effects cognate silencers. titrate                                                                                obtain efficient uptake [40]. doses antisense PNA <br /> essential gene expression obtain measured responses E. coli, <br />                                                                                selected order achieve titration E. coli growth rate <br /> growth, used gene specific antisense PNA IPTG-induced <br />                                                                                reduction.  dose range antisense PNA expression antisense sequences. concept enables quantita- <br />                                                                                unique, resulting growth inhibition dose dependent manner <br /> tive analyses mRNA silencing growth reduction,                                                                                (Figure 2). expressed antisense, plasmids transformed possible using knockout approaches.                                                                                TOP10 E. coli cells (Table 2) antisense RNA expression <br /> essential genes differed mRNA inhibition growth inhibition <br />                                                                                induced using IPTG concentrations provided profiles, demonstrating relationship transcript <br />                                                                                titration decreasing growth rates  shown). <br /> levels growth rate reflects requirement stringencies                                                                                   antisense PNA expressed antisense specifically <br /> essential genes. <br />                                                                                silenced mRNA essential genes acpP, fabI, ftsZ murA,                                                                                determined analyses transcript levels appropriate <br /> Results                                                                        range inhibitory lethal doses (Figure 3). Decreases Specificity antisense RNA PNA expressed                                 essential gene transcripts corresponded decreases bacterial <br />                                                                                growth rates (PNA data: R2 acpP, fabI, ftsZ, murA = 0.87, 0.63, <br /> antisense RNA <br />                                                                                0.89, 0.45), response non-linear.    silence selected target genes using different <br />                                                                                noted growth profiles observed ftsZ-inhibited cultures, <br /> silencing methods, needed design, test validate <br />                                                                                displayed rise fall culture turbidity, new silencers. Fortunately, able use previously <br />                                                                                consistent cell-division effect observed previously, developed silencers acpP fabI, used similar design <br />                                                                                cells elongate dividing [41,42]. Phenotypic effects ftsZ- <br /> guidelines additional silencers needed study.                                                                                inhibited cultures confirmed microscopy, revealing silencers able inhibit E. coli growth.  <br />                                                                                extremely elongated cells compared untreated cells cells PNA expressed antisense RNA silencers displayed gene <br />                                                                                treated PNAs. Cell phenotypes antisense-expressing sequence selective effects.  ensure                                                                                cells similar antisense PNA-treated cells (Figure 4). <br /> silence mainly target gene, carried set rescue <br /> experiments RNA silencers used conditions inhibited growth tested growth                    Essential gene stringencies minimum level transcript <br /> rescued expression target gene open reading frame               50 measure <br /> (ORF) plasmid. transcomplementation strategy                          relationship decrease mRNA growth <br /> provides strict control specificity relative controls             rate determined silencing methods indicates involve sense, scrambled irrelevant sequences,              requirement essential genes unequal, takes account transcript sequences present bacterial           genes stringently required   expressing <br /> cell effector sequences remain unchanged. designed                  relative growth rate function relative mRNA level (Figure 5) <br /> antisense PNA specific acpP (Ec108), fabI (Ec107), ftsZ (Ec326),           showed minimum level mRNA needed 50% cell murA (Ec330) using optimal parameters gene silencing                   viability order acpP .ftsZ.murA.fabI antisense PNA. <br /> [37,38] (Table 1). test PNA specificity, constructed                 large number acpP transcripts (approximately 0.65 <br /> DH5a strains containing pBAD-acpP, pBAD-fabI, pBAD-                     relative value 65% normal cell) required maintain 50% <br /> ftsZ pBAD-murA. strain grown presence                    cell viability, compared ftsZ (38%), murA (30%) fabI (20%) <br /> antisense PNA specific target gene cloned pBAD,                transcripts. words, small decrease acpP mRNA led addition L-arabinose target gene             large decrease bacterial growth rates, decreases ftsZ expression. L-arabinose, strains did grow                 mRNA proportional effects growth rates large induction L-arabinose, resistant growth                    decreases fabI murA mRNA required obtain inhibitory effects PNA (Figure 1). test specificity               small decrease growth rates. Gene stringency order <br />  <br />  <br />        PLoS | www.plosone.org                                          2                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                            E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 1. Transcomplementation antisense PNA expressed antisense RNA effects growth target gene expression. <br /> Essential gene expression induced L-arabinose indicated concentrations. PNAs added final concentration 4 mM. <br /> Concentrations IPTG used induction acpP-, fabI-, ftsZ- murA-antisense expression 200 mM, 1 mM, 100 mM 75 mM, respectively. <br /> doi:10.1371/journal.pone.0006061.g001 <br />  <br />  <br />        PLoS | www.plosone.org                                      3                               June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                E. coli Gene Stringency <br />  <br />  <br />  <br />  Table 1. Properties PNA used study. <br />  <br />  <br />  PNA         Sequence                                       Target         Target location* length                 Reference/source <br />  <br />  Ec107       (KFF)3K-eg1-cccatagctt                         fabI           25 +5 (10 nt.)                           [11] <br />  Ec108       (KFF)3K-eg1-ctcatactct                         acpP           25 +5 (10 nt.)                           [57] <br />  Ec326       (KFF)3K-eg1-tcaaacatag                         ftsZ           22 +8 (10 nt.)                           study <br />  Ec330       (KFF)3K 1-tttagtttgt                        murA           29 +1 (10 nt.)                           study <br />  <br />  * <br />   Antisense target locations indicated relative start codon. <br />  doi:10.1371/journal.pone.0006061.t001 <br />  <br />  <br /> acpP.ftsZ.fabI.murA expressed antisense (Figure 5),                   dsRNA-mediated cleavage target site. Ideally, protein <br /> differs PNA data murA stringently                quantification carried parallel RT-PCR, <br /> required essential gene growth fabI.                            biochemical assays confirm antisense inhibition    Based data, propose minimum transcript level                acpP, fabI, ftsZ murA carried technical <br /> 50 (MTL50) measure stringent requirement, somewhat                     difficulties involved harvesting sufficient culture biomass similar IC50 measure enzyme inhibition. MTL50 values                 quantitative protein assays lacking genes  <br /> directly reflect relationship level particular            Current methods limited use antibodies [33,43] mRNA growth rate decline, indicating stringently                  intracellular peptide expression [44].  correlation <br /> required essential gene growth. plotting values            mRNA protein levels bacteria relative growth rate relative mRNA level gene              demonstrated genes [25,39,45,46,47], suggesting graph, able estimate value relative                  MTL values reported provide useful measure gene <br /> mRNA (MTL50) 50% growth rate (Figure 5). observed                   requirement. shift right indicates greater stringency minimum                  genes compared study acpP does <br /> level transcript needed sustain 50% cell viability increases.            encode enzyme.  finding acpP  high MTL50 value indicates high gene stringency.              critically required ftsZ, fabI murA entirely value indicates minimum required level                      surprising. ACP known interact large number transcript viability, minimum inhibition level                  proteins fatty acid biosynthesis pathway [5] reduction <br /> transcript, differs IC50 respect. Compar-               levels active ACP results cell toxicity [33,36]. isons MTL50 values, determined different gene-               addition, ACP interacts proteins involved <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> <br /> silencing methods, shown Figure 6. general, MTL50                    synthesis [48].  ACP needed large quantities values higher expressed antisense cultures compared             slight decrease ACP levels amplified antisense PNA treated cultures. methods               interacting partners, leading widespread physiological effects. <br /> stringently required essential gene acpP, followed ftsZ.                 acpP transcripts protein products abundant E. <br /> Antisense PNA silencing showed fabI stringent,                 coli [49,50]. FtsZ recently GTPase activity expressed antisense silencing murA significantly             [51],  interacts proteins stringently required, relative essential genes          component complex divisome [52]. point examined (Figures 5 6).                                                    view gene product regulation, FtsZ concentrations dictate                                                                                initiation cell division interaction FtsA FtsQ <br /> Discussion                                                                     [53]. contrast, FabI MurA interact fewer proteins <br />                                                                                [5] possibly catalyze specific reactions.  FabI    Cell growth mRNA profiles following expressed antisense                 catalytic end point fatty acid elongation subject RNA silencing differed somewhat following antisense                 feedback inhibition acyl-ACP fatty acid elongation <br /> PNA silencing, possibly reflecting different mechanisms                     cycle [54], MurA catalytic starting point silencing.  data reject null hypothesis equal          peptidoglycan assembly [27] regulated stringent requirement growth essential genes support               feedback inhibition UDP-N-acetylmuramic acid,  <br /> premise essential genes differ level requirement          stream product pathway [55].  argued <br /> growth.  RNA silencing methods indicate significant                changes ACP FtsZ levels tolerated differences gene stringency reveal acpP                     changes FabI MurA levels. <br /> stringently required essential gene, followed ftsZ,                 Specific control single gene expression bacterial studied. MTL50 value gene                     operon remains challenge polycistronic nature calculated relative appropriate unsilenced controls, differences            transcripts.  antisense PNAs targeting genes lac basal expression different essential genes affect       operon polar effects transcript stability translation <br /> stringency ranking.                                                            [39]. avoid problem, selected genes likely <br />    stringency ranking murA fabI differed PNA-               affected: fabI murA predicted RegulonDB <br /> treated cells antisense-expressing cells, probably              (regulondb.ccg.unam.mx) EcoCyc (http://ecocyc.org/) differences PNA RNA chemistry, size differences                    individually transcribed, ftsZ, 39 end operon types silencers subtle differences               [24], acpP, transcribed fabF [25], <br /> mechanisms gene silencing. example, antisense PNAs act                  non-essential gene involved thermal regulation fatty acid <br /> directly steric hindrance translation initiation            biosynthesis [56]. note potency acpP-specific <br /> indirectly trigger decay repressed transcripts, antisense           PNA inhibition fabF expression  RNAs act steric hindrance direct activation              unlikely, reported tandem mutation <br />  <br />  <br />          PLoS | www.plosone.org                                        4                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 2. Dose dependent growth inhibition antisense PNA-treated AS19 cells. Overnight AS19 cultures sub-cultured fresh media <br /> containing different PNA doses gene, growth monitored turbidity. doses chosen PNA, obtain titration growth inhibition. Growth profiles terminated cultures harvested DOD550,0.1 untreated culture, transcript analyses. <br /> doi:10.1371/journal.pone.0006061.g002 <br />  <br />  <br />        PLoS | www.plosone.org                                         5                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                              E. coli Gene Stringency <br />  <br />  <br />  <br />  Table 2. Plasmids used study. <br />  <br />  <br />                                                                                                              Antisense target location*             Reference/ <br />  Plasmid           Relevant characteristic                         Purpose                                   length                             source <br />  <br />  pBAD/HisA         Arabinose inducible promoter (PBAD), AmpR       Inducible expression essential genes   n                                    Invitrogen <br />  pBAD-fabI         fabI ORF                                        Inducible expression fabI         n                                    study <br />  pBAD-acpP         acpP ORF                                        Inducible expression acpP         n                                    study <br />  pBAD-ftsZ         ftsZ ORF                                        Inducible expression ftsZ         n                                    study <br />  pBAD-murA         murA ORF                                        Inducible expression murA         n                                    study <br />  pHN678            IPTG inducible promoter (Ptrc),                 Stabilized antisense expression vector    n                                    [12] <br />                    paired-termini flanking MCS, CamR <br />  pHN682            fabI antisense insert                           Inducible expression fabI antisense    274 +86 fabI (160 nt.)           [12] <br />  pHNA              acpP antisense insert                           Inducible expression acpP antisense    242 +85 acpP (127 nt.)           study <br />  pHNZ              ftsZ antisense insert                           Inducible expression ftsZ antisense    253 +76 ftsZ (129 nt.)           study <br />  pHNM              murA antisense insert                           Inducible expression murA antisense    254 +76 murA (130 nt.)           study <br />  <br />  * <br />   Antisense target locations indicated relative start codon. <br />  doi:10.1371/journal.pone.0006061.t002 <br />  <br />  gene (fabB) needed growth inhibition [56],                       Plasmids strains used testing specificity anti-acpP effects rescued plasmid acpP transcomplementa-                        antisense agents <br /> tion [37].                                                                                ORFs essential genes cloned    summary, results essential genes differentially                     multiple cloning site (MCS) pBAD/HisA vector (Invitrogen), <br /> required requirement level appears reflect extent                     allowing gene expression L-arabinose-inducible predicted protein-protein interactions. PNA                                promoter. Sequences amplified K12 genomic DNA <br /> expressed antisense gene silencing methods gave similar results,                       using primers acpP R, ftsZ R, murA R fabI- <br /> despite having different inhibitory mechanisms. RNA                          R, specific acpP, ftsZ, murA fabI, respectively (Table <br /> silencing used confirm quantify essential gene                           S1). Amplicons digested NcoI XhoI (Fermentas) requirement serve molecular pre-screen,                                cloned pBAD/HisA vector containing similarly digested <br /> investing search specific inhibitors. provide                     ends. Recombinant plasmids transformed E. coli DH5a useful tool identifying reference genes quantitative                         cells selected LB plates supplemented ampicillin <br /> transcript profiling.                                                                  (100 mg/ml). Clones used testing specificity                                                                                        PNA treatment. testing specificity expressed antisense, <br /> Materials Methods                                                                  clones containing pBAD-fabI, pBAD-acpP, pBAD-ftsZ                                                                                        pBAD-murA transformed respective antisense- <br /> Bacteria growth conditions specificity testing                              carrying plasmid, e. pHN682, pHNA, pHNZ pHNM (Table 2 <br /> antisense agents                                                                       , selected LB plates supplemented    E. coli strain DH5a (Invitrogen) transformed                               ampicillin (100 mg/ml) chloramphenicol (30 mg/ml). <br /> recombinant plasmids pBAD-fabI, pBAD-acpP, pBAD-ftsZ pBAD-murA (Table 2  clones used testing                         Bacteria, growth conditions, PNA treatment specificity gene silencing PNA treatment                           antisense expression gene silencing <br /> antisense expression. Overnight E. coli cultures standardized                        E. coli strain AS19 hyper-permeable phenotype used OD550 readings approximately 26104 cfu/ml. test                           PNA treatments (total 6 doses; Table 1) [40]. AS19 cultures <br /> specificity antisense PNA, clones grown Mueller-Hinton                      Mueller-Hinton broth (DIFCO) used PNA (Panagene, <br /> broth (MHB; Oxoid) supplemented ampicillin (100 mg/ml)                            Korea) treatments prepared described previously [57], 4 mM gene-specific antisense PNA aqueous solution,                           PNA concentrations (0&#226;&#8364;&#8220;110 nM) optimized titration 0.02&#226;&#8364;&#8220;0.2% L-arabinose (Sigma)                               growth inhibition. TOP10 E. coli (Invitrogen) used induction essential gene expression. Concentrations L-                      transformation reactions plasmids expressing antisense sequenc- <br /> arabinose optimized clone.                                               es (Table 2). Antisense expression induced various <br />    test expressed antisense specificity, clones containing                     concentrations IPTG (total 6 doses) added LB <br /> essential gene expressing plasmid antisense-expressing                        supplemented chloramphenicol (30 mg/ml; Sigma).  <br /> plasmid   grown MHB supplemented                                night cultures antisense-expressing clones standardized ampicillin (100 mg/ml) plus chloramphenicol (30 mg/ml) 0.1&#226;&#8364;&#8220;                        OD550 readings approximately 26105 cfu/ml IPTG <br /> 75 mM IPTG  induction expressed antisense),                      induction. antisense PNA treatment IPTG induction, 0.02&#226;&#8364;&#8220;0.2% L-arabinose induction essential gene                          20 ml appropriate PNA IPTG concentration aqueous expression. Concentrations IPTG L-arabinose                           solution deposited 96 plate optimized clone. Bacterial cultures grown Bio-                      addition E. coli liquid culture final volume 200 ml. <br /> Tek PowerWave X340 spectrophotometer 37uC agitation                            Bacterial cultures grown VERSAmax spectropho- 5 min 200 ml volumes 96 plate. Growth                           tometer 37uC agitation 5 s 5 min. Growth monitored OD550 readings 5 min. experiment                           monitored OD550 readings 5 min cultures performed triplicate.                                                               harvested untreated control increased turbidity  <br />  <br />         PLoS | www.plosone.org                                                 6                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 3. Gene silencing E. coli obtain titration declining growth rates transcript levels antisense PNA expressed <br /> antisense. E. coli AS19 treated PNA doses, TOP10 clones containing antisense-expressing plasmids induced various IPTG <br /> concentrations shown Figure 2, obtain growth inhibition dose-dependent manner. Total RNA E. coli cultures harvested time control culture increased OD550 0.1. RNA used cDNA synthesis quantitative real-time PCR determine mRNA levels normalization reference gene rpoA, calculated relative unsilenced (untreated) control. Growth rates determined OD550 <br /> readings time. relationship mRNA growth rate decline indicated growth requirement stringency gene. <br /> doi:10.1371/journal.pone.0006061.g003 <br />  <br />  <br />        PLoS | www.plosone.org                                         7                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                 E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 4. Effect gene silencers cell morphology. Cells stained DAPI fluorescent microscopy. Left panel: AS19 untreated treated antisense PNA Ec108 (20 nM) targeting acpP, Ec107 (80 nM) targeting fabI, Ec326 (70 nM) targeting ftsZ Ec330 (55 nM) targeting <br /> murA. Gross elongation cells observed cultures treated ftsZ-specific PNA, Ec326. Right panel: TOP10 clones uninduced induced IPTG antisense RNA expression. Concentrations IPTG used induction acpP-, fabI-, ftsZ- murA-antisense expression 160 mM, <br /> 130 mM, 80 mM 80 mM, respectively. Cells expressing ftsZ-antisense grossly elongated compared cells. <br /> doi:10.1371/journal.pone.0006061.g004 <br />  <br />  <br />  <br />  <br /> Figure 5. Determining MTL50 value gene. gene, mean relative mRNA growth rate values Figure 3 transformed mean relative mRNA values plotted corresponding mean values relative growth rates obtain curve. curve gene allows estimation transcript value  x-axis) growth reduction 50% (y = 0.5),  MTL50 value gene  MTL50value reveals stringency essential gene growth. <br /> doi:10.1371/journal.pone.0006061.g005 <br />  <br />  <br />        PLoS | www.plosone.org                                         8                                June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                           E. coli Gene Stringency <br />  <br />  <br />  <br />  <br /> Figure 6. Analysis essential gene MTL50 values. curves triplicate experiments gene plotted obtain MTL50 values statistical analyses. Significant difference (** = p,0.01, * = p,0.05) MTL50 genes determined way ANOVA Tukey HSD Test. Antisense PNA silencing (left panel) showed MTL50 acpP significantly greater genes. Expressed <br /> antisense RNA silencing (right panel) showed MTL50 acpP ftsZ significantly greater murA  <br /> doi:10.1371/journal.pone.0006061.g006 <br />  <br /> ,0.1. Wells containing treatment pooled harvest           PNA viewed fluorescence microscopy. Similarly, E. cells pelleted RNA extraction. Growth rate                coli TOP10 cultures left uninduced induced IPTG <br /> calculated DOD/Dt.                                                      moderate (dose 3) high expression (dose 6) antisense <br />                                                                            sequences viewed microscopically. Cells grown Design antisense PNA gene silencing                                 harvested described  1 ml cells pooled    Antisense PNA specific acpP (Ec108), fabI (Ec107), ftsZ             pelleted. Cells washed equal volume 16PBS, <br /> (Ec326) murA (Ec330) designed using optimal parameters            suspended 100 ml 16PBS, stained DAPI (1 mM) gene silencing [37,38] (Table 1). Briefly, parameters        5 min. Cells viewed 10006magnification Leica <br /> 9&#226;&#8364;&#8220;12 mer PNA antisense sequence targeting region              DMRA2 microscope images captured processed <br /> 210 +10 start codon conjugated cell wall-           using Openlab software version 3.1.4. <br /> permeating peptide (KFF)3K. Candidate antisense sequences stated start codon region checked sequence <br /> uniqueness similar regions using E. coli database (http:// <br />                                                                            RNA extraction, reverse transcription qPCR <br /> genolist.pasteur.fr/Colibri/) testing antisense inhibition          Extraction RNA bacterial cells, followed DNase growth gene expression, described                   treatment carried using RiboPure Bacteria Kit <br />                                                                            (Ambion) according manufacturer&#226;&#8364;&#8482;s protocol. RNA <br />                                                                            (200&#226;&#8364;&#8220;500 ng) converted cDNA 25 ml reaction <br /> Plasmid expressing antisense gene silencing                            consisting 16RT reaction buffer, 5.5 mM MgCl2, 0.5 mM <br />    Antisense sequences chosen based length, location (120&#226;&#8364;&#8220;         dNTP, 2.5 mM random hexamers, 0.4 U/ml RNase <br /> 160 nt start codon include promoter coding               inhibitor, 1.25 U/ml MultiScribe Reverse Transcriptase <br /> regions) [12] minimal secondary structures,             (Applied Biosystems). 25 ml PCR reaction contained <br /> predicted RNAfold (http://www.tbi.univie.ac RNA/RNA- <br />                                                                            12.5 ml SYBR Green PCR buffer (Eurogentec), 100 nM fold.html). Antisense sequences cloned MCS                                                                            primer (Biomers) 5 ml cDNA. Relative qPCR pHN678, flanked 38 bp paired-termini stabilize <br />                                                                            carried primers acpP-F/R, fabI-F/R, ftsZ-F/R, <br /> antisense RNA inserts [12]. Antisense sequences essential genes <br />                                                                            murA-F/R specific acpP, fabI, ftsZ murA, respectively, amplified K12 genomic DNA primers acpP-XF1/R2, <br />                                                                            target gene, primers rpoA-F/R specific rpoA ftsZ-XF5/R5 murA-XF/R specific acpP, ftsZ murA, <br />                                                                            reference gene (Table S1). Target reference primer <br /> respectively (Table S1). Amplicons digested NcoI                                                                            pairs validated amplification efficiencies real time <br /> XhoI cloned pHN678 similarly digested ends. <br />                                                                            data analyses, 2-DDCT method [58] primers <br /> Recombinant plasmids transformed TOP10 cells selected <br />                                                                            similar efficiencies, 2DCT (target)/2DCT (reference) LB supplemented chloramphenicol (30 mg/ml). fabI <br />                                                                            method primers different efficiencies [59] relative <br /> antisense expressing plasmid pHN682 described [12]. <br />                                                                            qPCR. Quantitation acpP, fabI, ftsZ murA mRNA                                                                            normalized rpoA mRNA calculated relative Fluorescence microscopy                                                    untreated sample. Mean values error bars, representing <br />   E. coli AS19 cultures left untreated treated         standard deviation experimental replicates, sub-inhibitory (dose 3) inhibitory (dose 6) concentrations          plotted PNA IPTG doses. <br />  <br />  <br />       PLoS | www.plosone.org                                       9                              June 2009 | Volume 4 | Issue 6 | e6061 <br />                                                                                                                                                               E. coli Gene Stringency <br />  <br />  <br />  <br /> MTL50 calculation                                                                               Supporting Information <br />    gene, data points DOD/Dt PNA IPTG <br /> concentration (Figure 2) plotted relative mRNA                                 Table S1 Primers used study. <br /> PNA IPTG concentration (Figure 2) new graph.                                        doi:10.1371/journal.pone.0006061.s001 (0.06 MB <br /> results curve relative DOD/Dt relative mRNA                                 DOC) gene, MTL50 estimated. x-axis value curve intersects y = 0.5  50% reduction growth)                                Acknowledgments <br /> gives MTL50 value. experiment carried                                    thank Swedish Institute Guest Scholarship SG, Bloomsbury <br /> triplicate, gene curves PNA treatment                                   Colleges scholarship JMB Swedish Research Council, curves expressed antisense experiments. simplicity,                              University London Central Research Fund, Sasakawa mean data resulting average curve gene                                   Foundation support. thank Rikard Dryselius, Mark Schreiber, <br /> shown Figure 3. Triplicate MTL50 values, estimated                                      Abbas Nikravesh, Omid Faradani help comments. <br /> triplicate curves, used statistical analysis Figure 6. <br />                                                                                                 Author Contributions <br /> Statistical analysis                                                                            Conceived designed experiments: LG. Performed experiments: <br />   Statistical analyses R2 correlation, mean6SD                                       SG. Analyzed data: SG LG. Wrote paper: SG LG. Carried ANOVA carried using Excel. Tukey HSD Test                                          cloning fabI, ftsZ murA expressing strains antisense <br /> post-ANOVA comparisons calculated http://faculty.vassar.                                 RNA rescue experiments: JMB. Carried cloning acpP  <br /> edu/lowry/hsd.html                                                                              expressing strain: NN. Helped design supervise rescue experiments: JS. <br />  <br /> References <br />  1. Vergidis PI, Falagas (2008) Multidrug-resistant Gram-negative bacterial                  20. Lay NR, Cronan JE (2006) Gene-specific random mutagenesis Escherichia <br />     infections: emerging threat potential novel treatment options. Curr Opin                coli vivo: isolation temperature-sensitive mutations acyl carrier protein <br />     Investig Drugs 9: 176&#226;&#8364;&#8220;183.                                                                      fatty acid synthesis. J Bacteriol 188: 287&#226;&#8364;&#8220;296. <br />  2. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                   21. Eschenburg S, Priestman MA, Abdul-Latif FA, Delachaume C, Fassy F, et al. <br />     Escherichia coli K-12 frame, single-gene knockout mutants: Keio                       (2005) novel inhibitor suspends induced fit mechanism UDP-N- <br />     collection. Mol Syst Biol 2: 2006 0008.                                                         acetylglucosamine enolpyruvyl transferase (MurA). J Biol Chem 280: <br />  3. Forsyth RA, Haselbeck RJ, Ohlsen KL, Yamamoto RT, Xu H, et al. (2002)                         14070&#226;&#8364;&#8220;14075. <br />     genome-wide strategy identification essential genes Staphylococcus            22. Jaiswal R, Beuria TK, Mohan R, Mahajan SK, Panda D (2007) Totarol inhibits <br />     aureus. Mol Microbiol 43: 1387&#226;&#8364;&#8220;1400.                                                            bacterial cytokinesis perturbing assembly dynamics FtsZ. Biochemistry <br />  4. Yu BJ, Sung BH, Koob MD, Lee CH, Lee JH, et al. (2002) Minimization                      46: 4211&#226;&#8364;&#8220;4220. <br />     Escherichia coli genome using Tn5-targeted Cre/loxP excision  Nat                  23. Tasdemir D, Topaloglu B, Perozzo R, Brun R, O&#226;&#8364;&#8482;Neill R, et al. (2007) Marine <br />     Biotechnol 20: 1018&#226;&#8364;&#8220;1023.                                                                       natural products Turkish sponge Agelas oroides inhibit enoyl <br />  5. Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, et al. (2005) Interaction                 reductases Plasmodium falciparum, Mycobacterium tuberculosis Escherichia coli. <br />     network containing conserved essential protein complexes Escherichia coli.               Bioorg Med Chem 15: 6834&#226;&#8364;&#8220;6845. <br />     Nature 433: 531&#226;&#8364;&#8220;537.                                                                        24. Flardh K, Garrido T, Vicente M (1997) Contribution individual promoters  6. Yellaboina S, Goyal K, Mande SC (2007) Inferring genome-wide functional                         ddlB-ftsZ region transcription essential cell-division gene ftsZ     linkages E. coli combining improved genome context methods: comparison                    Escherichia coli. Mol Microbiol 24: 927&#226;&#8364;&#8220;936. <br />     high-throughput experimental data. Genome Res 17: 527&#226;&#8364;&#8220;535.                             25. Zhang Y, Cronan JE Jr (1996) Polar allele duplication transcriptional analysis <br />  7. Cheng AC, Coleman RG, Smyth KT, Cao Q, Soulard P, et al. (2007) Structure-                      consecutive essential genes: application cluster Escherichia coli fatty acid <br />     based maximal affinity model predicts small-molecule druggability. Nat                          biosynthetic genes. J Bacteriol 178: 3614&#226;&#8364;&#8220;3620. <br />     Biotechnol 25: 71&#226;&#8364;&#8220;75.                                                                       26. Heath RJ, Rubin JR, Holland DR, Zhang E, Snow  et al. (1999) Mechanism <br />  8. Myllykallio H, Leduc D, Filee J, Liebl U (2003) Life dihydrofolate                      triclosan inhibition bacterial fatty acid synthesis. J Biol Chem 274: <br />                                                                                                     11110&#226;&#8364;&#8220;11114. <br />     reductase FolA. Trends Microbiol 11: 220&#226;&#8364;&#8220;223. <br />                                                                                                 27. Marquardt JL, Siegele DA, Kolter R, Walsh CT (1992) Cloning sequencing <br />  9. Badarinarayana V, Estep PW 3rd, Shendure J, Edwards J, Tavazoie S, et al. <br />                                                                                                     Escherichia coli murZ purification product, UDP-N-acetylglucosa- <br />     (2001) Selection analyses insertional mutants using subgenic-resolution arrays. <br />                                                                                                     enolpyruvyl transferase. J Bacteriol 174: 5748&#226;&#8364;&#8220;5752. <br />     Nat Biotechnol 19: 1060&#226;&#8364;&#8220;1065. <br />                                                                                                 28. Marquardt JL, Brown ED, Lane WS, Haley TM, Ichikawa Y, et al. (1994) <br /> 10. Herring CD, Blattner FR (2004) Conditional lethal amber mutations essential <br />                                                                                                     Kinetics, stoichiometry, identification reactive thiolate     Escherichia coli genes. J Bacteriol 186: 2673&#226;&#8364;&#8220;2681. <br />                                                                                                     inactivation UDP-GlcNAc enolpyruvoyl transferase antibiotic <br /> 11. Dryselius R, Nekhotiaeva N, Good L (2005) Antimicrobial synergy                         fosfomycin. Biochemistry 33: 10646&#226;&#8364;&#8220;10651. <br />     mRNA- protein-level inhibitors. J Antimicrob Chemother 56: 97&#226;&#8364;&#8220;103.                      29. Schonbrunn E, Eschenburg S, Krekel F, Luger K, Amrhein N (2000) Role 12. Nakashima N, Tamura T, Good L (2006) Paired termini stabilize antisense                         loop containing residue 115 induced-fit mechanism bacterial cell <br />     RNAs enhance conditional gene silencing Escherichia coli. Nucleic Acids                  wall biosynthetic enzyme MurA. Biochemistry 39: 2164&#226;&#8364;&#8220;2173. <br />     Res 34: e138.                                                                               30. Tsao DH, Sutherland AG, Jennings LD, Li Y, Rush TS 3rd, et al. (2006) <br /> 13. Nekhotiaeva N, Awasthi SK, Nielsen PE, Good L (2004) Inhibition                              Discovery novel inhibitors ZipA/FtsZ complex NMR fragment <br />     Staphylococcus aureus gene expression growth using antisense peptide nucleic                screening coupled structure-based design. Bioorg Med Chem 14: <br />     acids. Mol Ther 10: 652&#226;&#8364;&#8220;659.                                                                    7953&#226;&#8364;&#8220;7961. <br /> 14. Pragman AA, Ji Y, Schlievert PM (2007) Repression Staphylococcus aureus SrrAB            31. Bi EF, Lutkenhaus J (1991) FtsZ ring structure associated division     using inducible antisense srrA alters growth virulence factor transcript levels.            Escherichia coli. Nature 354: 161&#226;&#8364;&#8220;164. <br />     Biochemistry 46: 314&#226;&#8364;&#8220;321.                                                                   32. Majerus PW, Alberts AW, Vagelos PR (1964) acyl carrier protein fatty <br /> 15. Kulyte  Nekhotiaeva N, Awasthi SK, Good L (2005) Inhibition                                acid synthesis: purification, physical properties, substrate binding site. Proc <br />     Mycobacterium smegmatis gene expression growth using antisense peptide                      Natl Acad Sci U S 51: 1231&#226;&#8364;&#8220;1238. <br />     nucleic acids. J Mol Microbiol Biotechnol 9: 101&#226;&#8364;&#8220;109.                                       33. Jackowski S, Rock (1983) Ratio active inactive forms acyl carrier <br /> 16. Harth G, Zamecnik PC, Tabatadze D, Pierson K, Horwitz MA (2007) Hairpin                         protein Escherichia coli. J Biol Chem 258: 15186&#226;&#8364;&#8220;15191. <br />     extensions enhance efficacy mycolyl transferase-specific antisense                   34. Tan XX, Actor JK, Chen Y (2005) Peptide nucleic acid antisense oligomer     oligonucleotides targeting Mycobacterium tuberculosis. Proc Natl Acad Sci U S                 therapeutic strategy bacterial infection: proof principle using mouse <br />     104: 7199&#226;&#8364;&#8220;7204.                                                                                 intraperitoneal infection. Antimicrob Agents Chemother 49: 3203&#226;&#8364;&#8220;3207. <br /> 17. Heath RJ, Rock (1995) Enoyl-acyl carrier protein reductase (FabI) plays                35. Tilley LD, Mellbye BL, Puckett SE, Iversen PL, Geller BL (2007) Antisense <br />     determinant role completing cycles fatty acid elo <br /> </body></html>