<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />    QIP, Component Vegetative RNA Silencing Pathway, Essential <br />       Meiosis Suppresses Meiotic Silencing Neurospora crassa <br />  <br />   <br />                                                               ABSTRACT <br />                 processes play essential roles genome defense organism survival              involved chromosome comparison. acutely active meiotic cells Neurospora crassa, <br />              evaluate mutual identity homologs process trans-sensing. non- <br />              symmetrical regions  silenced. known molecular components meiotic <br />              silencing machinery related RNA-dependent RNA polymerases, Argonautes Dicers, suggesting <br />              mechanisms heterologous chromosomal regions silenced involves, stage,              production small interfering RNAs. Neurospora active clearly distinct RNA interference <br />              pathways: quelling (vegetative specific) meiotic silencing (meiosis specific). pathways require              common set protein types like RNA-dependent RNA polymerases, Argonautes Dicers. work <br />              demonstrate involvement quelling defective-2 interacting protein (qip1), Neurospora gene              function essential silencing quelling, meiotic silencing, normal sexual development.              observations reinforce molecular connection silencing pathways. <br />  <br />  <br />  <br />  <br /> E    UKARYOTIC genomes require activity com- <br />        plex mechanisms aimed maintaining pre- <br /> serving molecular integrity. distinct <br />                                                                             vation mutants carrying deletion                                                                             transcription factor Ascospore maturation-1 (asm-11)                                                                             viable,  crossed wild type, progeny potentially interrelated processes known                       composed dead sexual spores (Aramayo Neurospora crassa: DNA methylation, quelling, repeat-                    Metzenberg 1996; Aramayo et al. 1996). connec- <br /> induced point mutation (RIP), meiotic silencing                         tion position-effect phenomenon RNA <br /> (Galagan et al. 2003; Borkovich et al. 2004).                               silencing characterization suppres- <br />    Quelling, vegetative RNA silencing, observed                      sor meiotic silencing, Suppressor ascus dominance-1 dawn RNA silencing field (Cogoni                         (Sad-1) (Shiu et al. 2001), implicated SAD-1, <br /> et al. 1996). Like organisms, discovery                    RNA-dependent RNA polymerase pathway. <br /> born need explain having copies                        characterization pathway revealed  gene equivalent having (Cogoni et al.                         volvement Argonaute Suppressor meiotic silencing- <br /> 1994). Genetic dissection pathway uncovered                     2 (Sms-2) dicer-like-1/Suppressor meiotic silencing-3 <br /> genes quelling defective-1 (qde-11), -2 (qde-21), -3 (qde-31),              (Dcl-1/Sms-3) (Galagan et al. 2003; Lee et al. 2003b; <br /> encoding RNA-dependent RNA polymerase                                Alexander et al. 2008). Unlike quelling, molecular <br /> (QDE-1), Argonaute (QDE-2), RecQ DNA                               properties required trigger meiotic silencing helicase (QDE-3), respectively (Cogoni Macino                           understood (Kutil et al. 2003; Lee et al. 2003b; Lee et al. <br /> 1997, 1999a, 1999b; Catalanotto et al. 2000).                               2004; Pratt et al. 2004). Exactly symmetry- <br />    Like discovery quelling, meiotic silencing                    evaluation process works mystery, know born need explain odd observa-                        involves evaluation identity tion: gene dominant absent?                         detailed level (Pratt et al. 2004). model meiotic <br /> (Aramayo Metzenberg 1996). initial obser-                           silencing described previously (Aramayo                                                                             Pratt 2010). involves recognition nonhomol- <br />                                                                             ogy homologous chromosomes unknown <br />   Supporting information available online http://www.genetics.org/ <br />                                                                             molecular players. identified, heterologous  <br /> cgi/content genetics.110.118422/DC1.                                   gions predicted produce form aberrant <br />   work dedicated memory David Dexter Perkins (1919&#226;&#8364;&#8220;       RNA, converted dsRNA action 2007), leadership spirit guided inspired work. David,     RdRP, SAD-1. produced, dsRNA triggers ini- light shall illuminate work forever. <br />   1 <br />                                                                             tiation step pathway, predicted involve <br />    Corresponding author: Department Biology, College Science, Texas M University, Room 415, Bldg. BSBW, College Station, TX 77843-            conversion dsRNA trigger siRNAs. 3258. E-mail: raramayo@tamu.edu                                             step likely executed DCL-1/SMS-3 Dicer. <br /> Genetics 186: 127&#226;&#8364;&#8220;133 (September 2010) <br />  128                                               D. W. Lee, R. Millimaki R. Aramayo <br />  <br />                                                                     TABLE 1 <br />                                                        Fungal strains used study <br />  <br /> Namea                                          Genotypeb                                                           Origina,c <br /> 367-4                          QipDThph1                                                  Yi Liu (Maiti et al. 2007) <br /> DLNCR517                       f l; Asm-1-Ama-11Thph1                                     Progeny DLNCR455 3 DLNCR469 <br /> DLNCR518                       f l; Asm-1-Ama-11Thph1                                     Progeny DLNCR455 3 DLNCR469 <br /> DLNCR553                       gle-11-sgfp1Thph1                                          Progeny RANCR05A 3 DLNCT545 <br /> DLNCR554                       gle-11-sgfp1Thph1                                          Progeny RANCR05A 3 DLNCT545 <br /> DLNCR707                                                                                  Progeny RANCR05A 3 RANCR49A <br /> DLNCR708                                                                                  Progeny RANCR05A 3 RANCR49A <br /> DLNCR750                       QipDThph1                                                  Progeny 367-4 3 KBNCR06A <br /> DLNCR769                       ridRIP1; qip1-sgfp1Thph1, mus-53RIP                        Progeny DLNCT756 3 DLNCR708 <br /> DLNCR770                       qip1-sgfp1Thph1, mus-53RIP                                 Progeny DLNCT756 3 DLNCR708 <br /> DLNCT807                       ridRIP1, 31Tccg-1(p)-qde-2-sgfp1; inl                  Transformation DLNCR243 pDLAM384 <br /> DLNCT808                       ridRIP2, 31Tccg-1(p)-qde-2-sgfp1; inl                  Transformation DLNCR244 pDLAM384 <br /> KBNCR05A                       3, RspRIP93; f l                                       Progeny DLNCR93 3 RANCR50A <br /> KBNCR06A                       RspRIP93; f l                                              Progeny DLNCR93 3 RANCR50A <br /> RMNCR113                       Qip DThph1; gle-11-sgfp1Thph1                              Progeny 367-4 3 DLNCR554 <br /> RMNCR114                       Qip DThph1; gle-11-sgfp1Thph1                              Progeny 367-4 3 DLNCR554 <br />         DLNC, KBNC, RANC, RMNC indicate strains constructed provided study Dong W. Lee, Kevin Baker, Rodolfo <br /> Aramayo Ryan Millimaki, respectively. <br />    b <br />      Allele numbers designations  mating type   mating type   Ascospore maturation-africana-1, Ama-1; Ascospore maturation-1, <br /> Asm-1; clock-controlled gene-1 promoter, ccg-1(p); fluffy, fl (P); <span id='am-2' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span>-leucine-phenylalanine-<span id='am-14' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-15' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> lethal-1, gle-1; histidine-3, 3 (1-234- <br /> 723); hygromycin B phosphotransferase, hph; inositol, inl (89601); mutagen-sensitive-53, mus-53; quelling-deficient-2, qde-2; quelling-deficient- <br /> 2-interacting protein, qip; repeat-induced-deficient, rid; Roundspore, Rsp; S65T green fluorescent protein, sgfp. <br />    c <br />      Information regarding construction parental strains plasmids used provided request. <br />  <br />  maintenance silencing probably involves ampli-                        sponding protein complexes involved share <br /> fication, SAD-1, degradation                         molecular components possible. target mRNAs RNA-induced silencing complex                            Neurospora genome available (RISC), Argonaute-like protein SMS-2                         components involved quelling correspond- <br /> predicted essential component. Interestingly,                       ing paralogs,  predicted, SAD-2 protein required meiotic silencing,                    involved meiotic silencing (Lee et al. 2003b). role appears anchor recruit SAD-1                    unexpected presence paralogs explained RdRP cytoplasmic face nuclear periphery                        complexity meiotic silencing. <br /> (Shiu et al. 2006) (D. W. Lee R. Aramayo,                                common hallmark conserved RNA interfer- <br /> published results). indicates SAD-1&#226;&#8364;&#8482;s RdRP                         ence pathways (RNAi) organisms activation <br /> activity required outside nucleus,                  RISC Argonaute proteins, process completely dissolved Neurospora meiosis. RNAs                             requires separation siRNA duplex single processed SAD-1 transit nuclear                          strands (Paroo et al. 2007). Neurospora quelling <br /> pores sort quality check. case, RdRP activity                 pathway, QIP, product quelling defective-2 likely downstream effector meiotic silencing                     interacting protein (qip1) gene, essential pur- component trans-sensing mechanism                          pose acts exonuclease cleaves  necessarily nuclear. &#226;&#8364;&#732;&#226;&#8364;&#732;RNA quality&#226;&#8364;&#8482;&#226;&#8364;&#8482; monitor-                     moves nicked passenger strand siRNA <br /> ing   implies deficient pairing                          duplex QDE-2-dependent manner (Maiti et al. <br /> triggers production aberrant transcripts alert                     2007). report involvement qip1 meiotic  proficient recognition                            silencing sexual development. establishes QIP <br /> pairing suppresses transcription.                                       common component quelling meiotic silenc- <br />    Clearly, presence mechanisms evolved                       ing reinforces notion quelling meiotic <br /> preserve genome integrity profound evolu-                           silencing intrinsically connected (Catalanotto et al. <br /> tionary consequences Neurospora shaped                           2000; Cogoni 2001; Lee et al. 2004). <br /> genome. Currently, Neurospora lowest number paralogous genes comparison organisms <br /> (Galagan et al. 2003; Borkovich et al. 2004), suggest- <br /> ing related molecular mechanisms operate                                          MATERIALS METHODS <br /> different stages life cycle (e.g., RNA silencing op-                    Molecular biology: Basic procedures DNA cloning, <br /> erating vegetative cells vs. meiotic cells), corre-                   analysis, sequencing, Southern blot, nucleic acid <br />                                                  Exonucleases Sex Meiotic Silencing                                             129 <br />  <br />                                                                  TABLE 2 <br />                                                     QIP involved meiotic silencing <br />  <br />       Linkage <br />        group:                                                                     Total    Wild-type                       Suppression <br /> Cross Relevant              III        VI                                    ascospores ascospores                             genotypea        rsp      qip       asm-1                Parents           examined      (%)b         Observations      silencing? <br />                       1         1        1 <br /> 1                  rsp        qip    asm-1          DLNCR707 3 DLNCR708           5161      86.8 6 0.5 Control                 n/ac <br />                    rsp1       qip1   asm-11 <br /> 2                  RspRIP93   qip1   asm-11         KBNCR05 3 KBNCR06             2541         0        Control                n/ac <br />                    RspRIP93   qip1   asm-11 <br /> 3                  rsp1       qip1   asm-11         DLNCR707 3 KBNCR06            2743       6.1 6 0.9 Meiotic silencing                          RspRIP93   qip1   asm-11 <br /> 4                  rsp1       QipD   asm-11         367-4 3 KBNCR06               4139      14.1 6 1.1 Experimental            Yes <br />                    RspRIP93   qip1   asm-11 <br /> 5                  rsp1       qip1   Asm-1-Ama-11   DLNCR517 3 DLNCR518           8382      73.1 6 1.3 Control                 n/ac <br />                    rsp1       qip1   Asm-1-Ama-11 <br /> 6                  rsp1       qip1   Asm-1-Ama-11   DLNCR517 3 DLNCR708           7187       9.2 6 5.4 Meiotic silencing                          rsp1       qip1   asm-11 <br /> 7                  rsp1       QipD   asm-11         367-4 3 DLNCR518              7034      36.6 6 2.1 Experimental            Yes <br />                    rsp1       qip1   Asm-1-Ama-11 <br />           Complete genotype described Table 1. <br />     b <br />       Scoring described Figure 1. <br />     c <br />       applicable meiotic silencing induced. <br />  <br />  <br /> manipulations performed described (Pratt                     transferred hemocytometer covered glass cover <br /> Aramayo 2002; Pratt et al. 2004).                                        slip. pictures taken cross num- <br />    Strain description construction: Escherichia coli K12             ber round spindle ascospores counted. XL1-Blue MR (Stratagene) host bacterial                  Asm-1- Rsp-silencing experiments, duplicate <br /> manipulations. N. crassa strains used study              crosses performed experiment. Multiple duplicate <br /> described Table 1. formulas Vogel&#226;&#8364;&#8482;s medium N,             crosses performed account plate plate variation. Westergaard&#226;&#8364;&#8482;s medium, sugar mixture Brock-                percentage wild-type spores averaged man Serres described Davis                    crosses standard deviation determined provide <br /> Serres (1970). Similarly, growth conditions, conidial sphero-            error bars. <br /> plast preparation, fungal transformation performed described (Pratt Aramayo 2002). Homokaryon <br /> purification performed described (Pratt Aramayo                               RESULTS DISCUSSION <br /> 2002; Lee et al. 2003a). <br />    Preparation perithecial tissues microscopy: Perithe-              QIP involved meiotic silencing: test role <br /> cia collected (4 days fertilization), fixed (90 mm            QIP meiotic silencing used strain carrying Pipes pH 6.9, 10 mm EGTA pH 7.5, 10 mm MgSO4, 0.3% Triton                deletion gene derivatives (Table 1). Testing <br /> X-100, 3.7% <span id='am-26' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span> 30 min), washed (twice                  involvement homozygous loss function  13 PBS), observed  suspended 13 PBS, 15% <br /> glycerol, pH 7.0 solution containing 0.75 mg/ml DAPI)                 dition proved impossible barren <br /> 6003 magnification. Pictures taken Photometrics                phenotype crosses  . CoolSnap HQ2 camera using RSimage software Roper                    assayed meiotic silencing heterozygous condition <br /> Scientific.                                                              following established protocol (Lee et al. <br />    Assaying meiotic silencing: crosses listed Table 2.        2003b, Figure 1, Table 2). used quantification Asm-1 silencing, progeny direc- <br /> tional crosses allowed shoot petri dish lids.            best-characterized sensitive inducers meiotic <br /> 15 days postfertilization (dpf), spores collected          silencing, Roundspore (Rsp) allele RspRIP93 (Pratt <br /> lid 1 ml water. aliquot population placed         et al. 2004) chimeric fusion promoter hemocytometer covered glass cover slip.              Ascospore maturation-1 (Asm-1) gene N. crassa pictures taken cross number white              fused coding trailer region Ascospore black spores determined pictures. Quan- <br /> tification Rsp silencing follows: 15 dpf,              maturation-1 (Ama-1) gene N. africana (Asm-1-Ama-11). <br /> ascospores collected 1 ml water lid              Unlike RspRIP93 allele, nonfunctional plate transferred microfuge tube. Round-shaped                 methylated, Asm-1-Ama-11 allele functional <br /> spores ejected efficiently wild-type spores            methylated (data shown), induces <br /> tend ooze perithecia, surface crossing           meiotic silencing series polymorphisms <br /> plate scraped using glass rod 3 ml water. Collected <br /> spores combined filtered Miracloth.                 present coding 39-untranslated regions. <br /> combined spores centrifuged 13,500 rpm 10 min             expected, crosses &#226;&#8364;&#732;&#226;&#8364;&#732;wild-type&#226;&#8364;&#8482;&#226;&#8364;&#8482; strains <br /> concentrate. aliquot ascospores cross                rsp1 3 rsp1 Asm-1-Ama-11 3 Asm-1-Ama-11 produced <br />  130                                          D. W. Lee, R. Millimaki R. Aramayo <br />  <br />                                                                                               Figure 1.&#226;&#8364;&#8221;QIP required effi- <br />                                                                                            cient meiotic silencing. involve- <br />                                                                                            ment qip1 silencing induced <br />                                                                                            homeologous indel alleles                                                                                            determined quantifying ratio                                                                                            wild-type mutant ascospores                                                                                            cross. Efficient meiotic silencing results <br />                                                                                            production ascospores                                                                                            round instead spindle shaped Rsp <br />                                                                                            white instead black Asm-1. <br />                                                                                            weaker silencing higher <br />                                                                                            percentage wild-type ascospores <br />                                                                                            produced cross. Note ab- <br />                                                                                            sence spindle spores cross 2                                                                                            silencing loss Rsp <br /> function. Similarly, number wild-type ascospores observed cross 5 nature Asm-1&#226;&#8364;&#8220;Ama-11 fusion allele. <br /> Crosses described Table 2. text Pratt et al. (2004) details. <br />  <br /> ascospores wild-type phenotype (86.8 6 0.5%                  3) chromosomal location (Freitag et al. 2004). 73.1 6 1.3%; Figure 1, crosses 1 5, respectively),           homozygous crosses unable observe perinu- crosses mutant strains RspRIP93 3                  clear localization QDE-2&#226;&#8364;&#8220;SGFP meiosis (support- <br /> RspRIP93 produced round ascospores (Figure 1, cross             ing information, Figure S1, H). Taken  2). Unpairing Rsp wild-type background               observations consistent QIP, Argonaute- <br /> results production ascospores mutant                interacting protein, working perinuclear region round (instead wild-type spindle) phenotype                   meiosis, region Argonaute SMS-2, <br /> (Pratt et al. 2004) (6.1 6 0.5%; Figure 1, cross 3).                 QDE-2, located. observations  <br /> Silencing compromised RspRIP93 unpaired                inforce notion QIP&#226;&#8364;&#8220;sGFP working presence unpaired copy Qip (14.1 6 1.1%;                SMS-2, establish direct physical <br /> Figure 1, cross 4). induced meiotic silencing             interaction. <br /> crossing strains carrying Asm-1-Ama-11 allele                  QIP essential early sexual development: carrying wild-type asm-11. heterozygous                 homozygous barren phenotype qip loss function <br /> crosses  e., asm-11 3 Asm-1-Ama-11), Asm-1 unpairing               mutants investigated. Sexual development <br /> results efficient silencing measured low <br /> production black, mature ascospores (9.2 6 5.4%; <br /> Figure 1, cross 6). expected, silencing sup- <br /> pressed unpairing reporter gene presence unpaired copy Qip (36.6 6 2.1%; <br /> Figure 1, cross 7). Combined, results suggest  conditions, QIP involved meiotic <br /> silencing. <br />    QIP localization perinuclear meiosis: Previous <br /> studies established components meiotic <br /> silencing machinery perinuclearly localized (Shiu <br /> et al. 2006; Alexander et al. 2008). common lo- <br /> calization function suggests work  <br /> possibly direct interactions (Bardiya et al. <br /> 2008). previously established QIP <br /> interacts QDE-2 (Maiti et al. 2007). understand QIP works meiotic silencing determined subcellular localization meiosis. constructed fusion qip1 reporter gene sgfp1 inserted resulting fusion (qip1&#226;&#8364;&#8220;sgfp1) canon- <br /> ical chromosomal location gene replacement. Ho- <br /> mozygous crosses strains fertile,                   Figure 2.&#226;&#8364;&#8221;Localization QIP&#226;&#8364;&#8220;sGFP meiosis. Pictures <br /> suggesting fusion functional. observed                representative asci photographed pachytene stage <br /> QIP&#226;&#8364;&#8220;sGFP perinuclear localization meiosis                 meiosis  Columns 1 2 QIP&#226;&#8364;&#8220;sGFP (green) <br /> (Figure 2, K). determine QDE-2 similar                 DAPI (red) signals, respectively. Column 3 shows cor- <br />                                                                      responding merged images columns 1 2. Tissues localization, constructed qde-21&#226;&#8364;&#8220;sgfp1 fusion              harvested 4 days postfertilization processed described <br /> gene control clock controlled gene-1                materials methods bar  column 1, <br /> promoter (ccg-1(p)) inserted histidine-3               equals 20 mm. <br />                                              Exonucleases Sex Meiotic Silencing                                            131 <br />  <br />                                                                                                             Figure 3.&#226;&#8364;&#8221;QIP essen- <br />                                                                                                          tial completion                                                                                                          early sexual development. <br />                                                                                                            Sexual development                                                                                                          N. crassa. cartoon dia- <br />                                                                                                          gram represents                                                                                                          main stages (early develop- <br />                                                                                                          ment, middle development, <br />                                                                                                          late development), <br />                                                                                                          fertilization asco- <br />                                                                                                          spore maturation observed <br />                                                                                                          sexual develop- <br />                                                                                                          ment. Drawings                                                                                                          scale. Cartoon inspired <br />                                                                                                          work Raju <br />                                                                                                          (1980). (B). Early wild-type <br />                                                                                                          development.       Maternal <br />                                                                                                          ascogenous tissue cor- <br />                                                                                                          responding wild-type <br />                                                                                                          cross     (DLNCR553       3 <br />                                                                                                          DLNCR554; Table 2)                                                                                                          examined. 1A, 1B,                                                                                                          2A 8A, GLE-1&#226;&#8364;&#8220;sGFP signal <br />                                                                                                          green, DAPI sig- <br />                                                                                                          nal red. 1A shows repre- <br />                                                                                                          sentative maternal tissue. <br />                                                                                                          1B shows 103 enlarge- <br />                                                                                                          ment nuclei <br />                                                                                                          present 1A. 2A 8A                                                                                                          merge images develop- <br />                                                                                                          ing croziers                                                                                                          formation, conju- <br />                                                                                                          gate mitosis, karyogamy, <br />                                                                                                          early meiosis  2B 8B <br />                                                                                                          light images. (C). <br />                                                                                                          Early QipD development. <br />                                                                                                          Maternal ascogenous <br />                                                                                                          tissue corresponding                                                                                                          mutant Qip loss function <br />                                                                                                          cross     (RMNCR113       3 <br />                                                                                                          RMNCR114; Table 1)                                                                                                          examined. 1A, 1B, 1C, 1D 2A, 2B 9A, 9B, GLE-1&#226;&#8364;&#8220;sGFP signal green. 1A, 1B, 1C, 1D 2A, 2C 9A, 9C, DAPI signal red. <br /> 1A 1C representative maternal tissue days 3 5 postfertilization, respectively. 1B 1D 103 enlargement nuclei present 1A 1C, respectively. 2A 9A merge images developing croziers  <br /> velopment. 2B 9B GLE-1&#226;&#8364;&#8220;sGFP localization. 2C 9C DAPI signal. 2D 9D light images. tissues harvested 3 4 days postfertilization, unless noted, processed described materials methods. Pictures taken 6003. bar 1A equals 20 mm. <br />  <br />  Neurospora complex divided               tissue (Figure 3A, stage  Figure 3B, 1A 1B), stages (Figure 3A). Following fertilization early sexual             large nuclei observed ascogenous tissue (Figure 3A, <br /> development difficult follow.               stages III, Figure 3B[2A 8A]). nuclei reach <br /> syncytial nature maternal ascogenous                 maximal size pachytene stage meiosis tissue involved colocalization inside               (Figure 4[1] ). Following meiosis II (Figure 4[2&#226;&#8364;&#8220; <br /> incipient perithecium. distinguish                  9]), nuclei regain &#226;&#8364;&#732;&#226;&#8364;&#732;normal&#226;&#8364;&#8482;&#226;&#8364;&#8482; GLE-1 signal mitosis tissues corresponding nuclei, constructed               (Figure 4[10]). contrast wild-type condition, <br /> strains carrying canonical fusions glycine&#226;&#8364;&#8220;leucine&#226;&#8364;&#8220;           crosses homozygous strains carrying GLE-1&#226;&#8364;&#8220;sGFP phenylalanine&#226;&#8364;&#8220;glycine lethal-1 (gle-11, NCU01198) nucleo-            qipD mutant background (Table 1) severely <br /> pore complex component gene (Murphy Wente                        developmentally compromised. circumstances <br /> 1996) sgfp1. GLE-1 predicted localized           did observe formation large asci ascospores. <br /> nuclear membrane confirmed tissues               formation maternal tissue appeared examined (Figures 3 4). Note sexual                  normal (Figure 3C[1]), development ascoge- <br /> development size nucleus undergoes dramatic               nous tissue compromised early stage (Figure <br /> changes; small nuclei observed maternal                  3C[2&#226;&#8364;&#8220;9]). hallmark ascogenous tissue  132                                          D. W. Lee, R. Millimaki R. Aramayo <br />  <br />                                                                   efficient repeat-inactivating mechanism. redun- <br />                                                                   dancy suggests quelling meiotic silencing <br />                                                                   biological functions simple <br />                                                                   repeat recognition silencing. evolutionary terms, <br />                                                                   safe think meiosis &#226;&#8364;&#732;&#226;&#8364;&#732;late&#226;&#8364;&#8482;&#226;&#8364;&#8482; evolutionary <br />                                                                   invention  e., meiosis vs. mitosis).  likely <br />                                                                   meiotic silencing machinery built                                                                   existing RNA silencing apparatus.                                                                   exact evolutionary order emergence                                                                   primitive RNA silencing machineries RIP                                                                   known, evolutionary pressures function imposed <br />                                                                   RIP selected presence pro- <br />                                                                   teins specific pathway favoring sharing                                                                   proteins pathways. finding ad- <br />                                                                   ditional common component reinforces hypothesis. <br />  <br />   Figure 4.&#226;&#8364;&#8221;Localization GLE-1&#226;&#8364;&#8220;sGFP normal meiosis.      cellular localization GLE-1&#226;&#8364;&#8220;sGFP followed          <br /> meiosis cross strains (DLNCR553 3 DLNCR554;          <br /> Table 1), representing cells (1) pachytene, (2) metaphase,          <br /> (3) telophase, (4) interphase, (5) prophase II, (6) anaphase II       telophase II, (7) early interphase II, (8) late inter-        <br /> phase II, (9) metaphase IV, (10) postmitosis II. Stages        assigned following Raju (1980). stages           <br /> merge image GLE-1&#226;&#8364;&#8220;sGFP (green) DAPI (red) signals.             <br /> Tissues harvested 5 7 days postfertilization           <br /> processed described materials methods.             <br /> bar l 1 equals 20 mm. Pictures taken 6003.         <br /> formation crozier, celled apparatus           <br /> sisting basal, tip, penultimate cell  Figure         <br /> 3A details). observations suggest                 formed, crozier development com-             <br /> promised qip loss function mutants. summary,             results QIP essential early sexual           <br /> development suggest required           <br /> meiotic silencing.                                            <br />    substantial component eukaryotic genomes                  <br /> studied <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> formed repeated sequences (Hsieh             2000). genome N. crassa intriguing                regard mechanisms, like quelling               meiotic silencing, arguably evolved              <br /> cope  respond  presence genomic-            <br /> repeated elements. despite presence RIP,               <br />  <br />  <br />  <br /> QIP, Component Vegetative RNA Silencing Pathway, Essential <br />    Meiosis Suppresses Meiotic Silencing Neurospora crassa <br />  <br />  <br />  <br />              Dong Whan Lee, Ryan Millimaki Rodolfo Aramayo <br />  <br />  <br />  <br />  <br />             Copyright &#195;&#8220; 2010 Genetics Society America <br />                     DOI: 10.1534/genetics.110.118422 <br />  2 SI                                                   D. W. Lee et al. <br />  <br />  <br />  <br />  <br />    FIGURE S1.&#226;&#8364;&#8221;Localization QDE-2-sGFP Meiosis. Picture shows representative asci photographed Pachytene stage Meiosis (Panels H). Columns 1 2 GLE-1-sGFP (green), DAPI (red) signals, respectively. Column 3 <br /> shows corresponding merged images Columns 1 2. Perithecia collected, fixed, washed observed described Materials Methods. bar present Panel 1A equals 20 &#206;&#188;. <br />   <br /> </body></html>