<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>OsMADS51 Short-Day Flowering Promoter Functions Upstream Ehd1, OsMADS14, Hd3a1[W][OA] <br /> Song Lim Kim2, Shinyoung Lee2, Hyo Jung Kim, Hong Gil Nam, Gynheung  <br /> Division Molecular Life Sciences Biotechnology Research Center, Pohang University Science Technology, Pohang 790&#226;&#8364;&#8220;784, Republic Korea <br />  <br />  flowering regulatory mechanisms extensively studied Arabidopsis (Arabidopsis thaliana), species elucidated.  investigated role OsMADS51, type MADS-box gene short-day <br /> (SD) promotion pathway rice (Oryza sativa). SDs OsMADS51 null mutants flowered 2 weeks later normal, long days loss OsMADS51 little effect flowering. Transcript levels flowering regulators&#226;&#8364;&#8221;Ehd1, OsMADS14, Hd3a decreased mutants, OsGI Hd1 unchanged. Ectopic expression OsMADS51 caused flowering occur 7 d earlier SDs. ectopic expression lines, transcript levels Ehd1, <br /> OsMADS14, Hd3a increased, OsGI Hd1 remained  results indicate OsMADS51 flowering promoter, particularly SDs, gene functions upstream Ehd1, OsMADS14, Hd3a. investigate relationship flowering promoters, generated transgenic plants expression Ehd1 OsGI suppressed. Ehd1 RNA interference plants, OsMADS51 expression affected, supporting conclusion MADS-box gene functions upstream Ehd1.  OsGI antisense plants, OsMADS51 transcript level reduced. addition, circadian expression pattern MADS-box gene similar OsGI. results <br /> demonstrate OsMADS51 functions downstream OsGI. summary, OsMADS51 novel flowering promoter transmits SD promotion signal OsGI Ehd1. <br />  <br />  <br />  <br />    ability adjust timing flowering                    little effect short days (SDs). GI expression important factor trying adapt plants                        regulated circadian clock, peak tran- <br /> gional environments control harvest time.                       script levels 8 10 h dawn. Timing, height, <br /> Regulatory mechanisms control flowering                         duration peak influenced daylength studied extensively Arabidopsis (Arabidopsis                      (Fowler et al., 1999). GI protein acts thaliana). long-day (LD) species, flowering                    circadian oscillator promote flowering promoted regulatory genes, including                           increasing FT mRNA abundance (Mizoguchi <br /> GIGANTEA (GI), CONSTANS  , FLOWERING                                   et al., 2005). putative transcription factor  <br /> LOCUS T (FT), SUPPRESSOR OVEREXPRESSION                                taining B-box zinc fingers C-terminal CCT CO1 (SOC1), APETALA1 (AP1).                                        domain (Putterill et al., 1995). promotes flowering <br />    GI encodes nuclear protein defines proper                   response LDs SDs (Putterill et al., 1995). <br /> amplitude period length circadian rhythms                          protein activates expression early <br /> (Park et al., 1999; Mizoguchi et al., 2005). Arabidop-                 target genes (e.g. SOC1 FT [Samach et al., 2000; <br /> sis, mutations GI delay flowering LDs,                     Suarez-Lopez et al., 2001]). FT, encodes pro- <br />                                                                           tein similar phosphatidylethanolamine-binding pro- <br />    1                                                                      teins Raf kinase inhibitor proteins, promotes <br />      work supported Crop Functional <br />                                                                           flowering LDs (Kardailsky et al., 1999; Kobayashi <br /> Genomic Center, 21st Century Frontier Program (grant  <br /> CG1111); Biogreen 21 Program, Rural Development Administra- <br />                                                                           et al., 1999). FT primarily expressed vascular tis- <br /> tion; Korea Science Engineering Foundation (KOSEF)                sues, interactor, FD, predominantly ex- National Research Laboratory Program funded            pressed shoot apex (Abe et al., 2005; Wigge et al., <br /> Ministry Science Technology (grant  M10600000270&#226;&#8364;&#8220;               2005). SOC1 integrates vernalization GA signals <br /> 06J0000&#226;&#8364;&#8220;27010); Korean government (KOSEF grant                 flowering (Moon et al., 2005). expression af- <br /> R15&#226;&#8364;&#8220;2004&#226;&#8364;&#8220;033&#226;&#8364;&#8220;05002&#226;&#8364;&#8220;0).                                                    fected loss FT, activation-tagged FT <br />    2 <br />      authors contributed equally article.                    mutants shown induce SOC1 regulation <br />    * Corresponding author; e-mail genean@postech.ac.kr.                   (Schmid et al., 2003; Moon et al., 2005). contrast, FT <br />    author responsible distribution materials integral   expression remains unaffected loss gain findings presented article accordance policy <br />                                                                           SOC1 expression. response FT-mediated floral <br /> described Instructions Authors (www.plantphysiol.org)  <br /> Gynheung (genean@postech.ac.kr). <br />                                                                           induction, AP1 contributes inflorescence meri- <br />    [W] <br />        online version article contains Web data.         stem committed flowering (Huang et al., <br />    [OA] <br />         Open Access articles viewed online sub-          2005). <br /> scription.                                                                   contrast Arabidopsis, rice (Oryza sativa) SD <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.107.103291                     plant flowering inhibited early <br /> 1484       Plant Physiology, December 2007, Vol. 145, pp. 1484&#226;&#8364;&#8220;1494, www.plantphysiol.org &#195;&#8220; 2007 American Society Plant Biologists <br />                                                                                        OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br />                                                                           high night (Izawa et al., 2002; <br />                                                                           Kojima et al., 2002; Hayama et al., 2003). contrast                                                                            Hd1 double role regulating downstream <br />                                                                           flowering activators&#226;&#8364;&#8221;repression LDs promo- <br />                                                                           tion SDs (Yano et al., 2000). downstream <br />                                                                           targets Hd3a, FT ortholog, acts preferentially <br />                                                                           inductive SDs (Kojima et al., 2002). addition                                                                           photoperiod-controlled pathway, rice contains                                                                           alternative inductive pathway appears function <br />                                                                           independently Hd1. Ehd1, encoding B-type  <br />                                                                           sponse regulator, promotes flowering inducing                                                                           expressions Hd3a, FTL1, OsMADS14 (Doi et al., <br />                                                                           2004). results indicate different signal- <br />                                                                           ing pathways intrinsically involved photo- <br />                                                                           periodic flowering pathway rice. <br />                                                                              relationship AP1 group MADS-box <br />                                                                           genes differ rice Arabidopsis. <br />                                                                           example, AP1 related genes determine <br />                                                                           floral meristem identity downstream FT.  <br />                                                                           rice, OsMASD14, member AP1 group, <br />                                                                           apparently functions upstream Hd3a flowering <br />                                                                           activator. ectopic expression causes significant early <br />                                                                           flowering tissue culture (Jeon et al., 2000),                                                                           transcription Hd3a regulated OsMADS14- <br />                                                                           overexpressing cultured cells (S. Lee G.   <br />                                                                           published data). <br />                                                                              plants eventually flower, nonper- <br />                                                                           missive conditions. autonomous pathway reg- <br /> Figure 1. OsMADS51 genomic structure, positions T-DNA inser-           ulated mainly SOC1 Arabidopsis. gene acts <br /> tional mutant alleles, osmads51 mutant phenotypes.  Schematic       flowering inducer, result converging photo- <br /> diagram T-DNA insertional position OsMADS51. gene              period, autonomous, vernalization, GA pathways <br /> consists exons (black boxes) introns (lines      (Araki, 2001; Mouradov et al., 2002; Simpson Dean, <br /> black boxes). T-DNA located 9,606 bp (osmads51-1) 15,927 <br />                                                                           2002). OsMASD50, Arabidopsis SOC1 homolog, <br /> bp (osmads51-2) translation start site. F1 R1 primers used RT-PCR real-time PCR analyses. F2, R2, Gus2R primers <br />                                                                           acts flowering activator similar SOC1. Ectopic genotyping osmads51-1. F3, R2, LB2R primers genotyp-      expression causes dramatic early flowering  <br /> ing osmads51-2. B, Semiquantitative RT-PCR OsMADS51 transcript         generating callus stage, OsMASD50 mutants flower osmads51 mutants (HO) wild-type segregants (WT). Actin1        month late (Lee et al., 2004; Lee  2007). <br /> used normalization cDNA quantity. PCR cycles numbered 28           MADS-box genes regulate flowering time <br /> OsMADS51 25 Actin1. C, Phenotype OsMADS51 mutants               belong type II, group contains conserved <br /> (HO) compared wild-type segregants (WT). Plants cultivated      MEF2-like MADS-box K regions. addition,  growth room SD conditions (10 h light/14 h dark). Photo-       termediate   C-terminal (C) regions present. <br /> graph taken 75 <span id='am-14' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span>DAG</span>. Arrows indicate panicles emerging            termed MIKC-type genes (Munster <br /> heading stage. <br />                                                                           et al., 1997). possible roles group, type <br />                                                                           MADS-box genes flowering time, developmental stages, induced day-                           understood. Type group proteins contain conserved <br /> length shortens fall. roles genes                  SRF-like MADS domain, C-terminal region involved photoperiod pathway generally                         clearly defined. Furthermore, type proteins conserved rice Arabidopsis, differ-                      carry conserved K-box region (Alvarez-Buylla <br /> ences exist. OsGI, GI ortholog, regulates flowering <br /> time response photoperiodic conditions. Muta- <br /> tions gene delay flowering inductive SDs, <br />                                                                           Table  Days heading osmads51 flowering marginally induced LDs <br /> (Hayama et al., 2003). comparison, overexpression                       Days heading scored panicle appeared. WT, <br />                                                                           Wild type; HO, osmads51/osmads51 homozygotes. OsGI transgenic rice causes late flowering SDs LDs. Expression Hd1, ortholog                                                   Days Heading    Days Heading <br />                                                                                 Alleles     Genotypes <br />                                                                                                            SD           LD <br /> affected plants, indicates OsGI <br /> functions regulate Hd1, gene shows                            osmads51-1       WT           75    6   2.2      96   6   3.8 <br /> circadian rhythm (Hayama et al., 2003).                                           HO           89    6   5.2     100   6   3.2 <br /> expression pattern differs OsGI. Hd1                            osmads51-2       WT           75    6   2.2      97   6   2.5 <br />                                                                                               HO           90    6   3.8     101   6   4.0 <br /> transcript levels lowest midday Plant Physiol. Vol. 145, 2007                                                                                                                1485 <br />  Kim et al. <br />  <br />  <br />                                                                           RESULTS <br />                                                                           Isolation T-DNA Insertional Mutants OsMADS51 <br />                                                                              previously reported generation                                                                           T-DNA insertional mutant population japonica rice <br />                                                                           (Jeon  2001; Jeong et al., 2002, 2006; et al., <br />                                                                           2003, 2005b; Ryu et al., 2004). sequence database                                                                           open public (www.postech.ac.kr/life/pfg/risd) <br />                                                                           seeds available scientific community. <br />                                                                           database, identified indepen- <br />                                                                           dent T-DNA insertional mutant alleles, osmads51-1                                                                           osmads51-2, T-DNAs inserted                                                                           intron alleles, 9,606 15,927 bp                                                                           start codon osmads51-1 osmads51-2, respectively <br />                                                                           (Fig. 1A). Homozygous (osmads51/osmads51) plants <br />                                                                           did accumulate OsMADS51 transcript (Fig. 1B). <br />                                                                           SDs, flowering homozygous progeny                                                                           alleles 2 weeks later wild-type <br />                                                                           segregants (Fig. 1C; Table . contrast, little change                                                                           flowering time noted LDs (Table .                                                                           results indicate OsMADS51 flowering pro- <br />                                                                           moter functions preferentially SDs. <br />  <br />  <br />  <br />                                                                           Ectopic Expression OsMADS51 Causes <br />                                                                           Early Flowering <br />  <br /> Figure 2. OsMADS51 overexpression.  Schematic diagram UbiT              elucidate functional roles OsMADS51, <br /> OsMADS51 construct. OsMADS51 length cDNA clone                   constructed chimeric binary vector (Ubi: <br /> placed Ubi promoter (Pubi) nos terminator (Tnos). B,          OsMADS51) placing OsMADS51 length <br /> Semiquantitative RT-PCR analyses UbiTOsMADS51 transgenic (OX)          cDNA maize ubiquitin (Ubi) promoter (Fig. <br /> lines 3 4 segregating wild-type controls (WT). Actin1   2A). construct introduced Agrobacterium <br /> used normalization cDNA quantity. RT-PCR performed        transferred plant cells cocultivation  28 cycles OsMADS51 25 cycles Actin1. C, Phenotype <br />                                                                           et al., 1985; Hiei et al., 1997). Seven primary (T1) trans- <br /> UbiTOsMADS51 transgenic plants (OX) segregating wild- <br /> type controls (WT) cultivated 62 d SD conditions. Arrows <br />                                                                           genic plants obtained planted paddy <br /> indicate heading panicles.                                                field. showed early flowering, albeit varying  <br />                                                                           grees. example, lines flowered 2 weeks early, <br />                                                                           lines 10 d sooner, lines just <br /> et al., 2000). overexpression type                        3 d compared control transgenic plants. <br /> MADS-box gene, AGAMOUS-LIKE28 (AGL28), pro-                               extent early flowering correlated motes flowering autonomous pathway                          OsMADS51 expression levels confirmed Arabidopsis (Yoo et al., 2006), roles MADS-                   northern-blot analyses (data shown) box genes type remain unknown.                                    chose lines (3 4) manifested early flower- <br />     elucidated functional role                      ing 10 d selected homozygous plants type MADS-box gene, OsMADS51, rice. gene                        segregating wild type using hygromycin selection. originally identified large-scale cDNA                        T3 progeny reared growth room analysis rice genome research program                          SDs (10 h light) LDs (14 h light) elucidate (Shinozuka et al., 1999). OsMADS51 unique rice; homologous gene lacking Arabidopsis.  <br /> homologous genes occur, 74% identity wheat (Triticum aestivum; accession  ABF57941)                                                                           Table II. Days heading OsMADS51-overexpressing plants <br /> maize (Zea mays; accession  CAD23412).  <br />                                                                            Days heading scored panicle appeared. WT, roles elucidated. OsMADS51                                                                           Wild type; OX, homozygous UbiTOsMADS51. <br /> composed 164 amino acids contains SRF-like <br /> MADS domain.  like type MADS-box                                                             Days Heading   Days Heading <br />                                                                              Transgenic Plants   Genotypes <br />                                                                                                                 SD          LD <br /> genes, K-box region conserved C-terminal region short. Expression ubiquitous                          Line 3            WT          72   6   3.8      93   6   3.1 organs (Shinozuka et al., 1999). study,                                             OX          63   6   4.0      91   6   2.4 investigated function OsMADS51                                  Line 4            WT          73   6   1.7      94   6   2.4 <br />                                                                                                    OX          66   6   2.3      93   6   3.3 <br /> regard flowering time. <br /> 1486                                                                                                            Plant Physiol. Vol. 145, 2007 <br />                                                                     OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> daylength-dependent roles OsMADS51. Homozy-            investigated expression OsGI, Hd1, Ehd1, <br /> gous plants exhibited early flowering 9 d (line 3)     OsMADS14, Hd3a osmads51 mutants 7 d (line 4) SDs.  significant pheno-   wild-type segregants grown continuous SDs <br /> typic difference observed homozygous          LDs. 55 d germination (DAG), leaf blades <br /> transgenic lines wild-type plants grown LDs        harvested transcript levels measured <br /> (Table II). results support conclusion     reverse transcription (RT)-PCR quantitative real- <br /> OsMADS51 SD flowering activator.                     time PCR (Fig. 3; Supplemental Fig. S1). Expression <br />                                                           OsGI Hd1 unchanged OsMADS51 <br /> Relationship OsMADS51 Various                     mutants lighting conditions (Fig. 3,  B, D, <br /> Flowering-Time Regulators                                 E).  transcript levels Ehd1, OsMADS14, <br />                                                           Hd3a greatly reduced osmads51 mutants <br />   evaluate relationship OsMADS51           SDs LDs (Fig. 3,  B, F, G, H). results previously identified flowering regulators,        indicate Ehd1, Hd3a, OsMADS14 probably <br />  <br />                                                                         Figure 3. Expression profiles various flowering <br />                                                                         regulators OsMADS51 mutants. Shown semi- <br />                                                                         quantitative RT-PCR analyses OsMADS51, OsGI, <br />                                                                         Hd1, Ehd1, OsMADS14, Hd3a wild-type (WT) <br />                                                                         OsMADS51 mutant plants (HO) grown 55 d <br />                                                                         SD   LD (B) conditions. Actin1 used <br />                                                                         normalization cDNA quantity. RT-PCR cycles <br />                                                                         numbered 28 flowering regulators 25                                                                         Actin1. independent experiments, <br />                                                                         reproducible data presented. Real-time <br />                                                                         PCR analyses OsMADS51 (C), OsGI (D), Hd1 (E), <br />                                                                         Ehd1 (F), OsMADS14 (G), Hd3a (H) WT                                                                         HO plants grown 55 d SD LD conditions. <br />                                                                         RNA prepared leaf blades harvested 0 h <br />                                                                         light turned  data point                                                                         average independent experiments. <br />                                                                         y axis, Relative values transcript levels                                                                         regulatory genes Ubi; vertical bars, standard <br />                                                                         deviation. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                          1487 <br />  Kim et al. <br />  <br />  <br /> function downstream OsMADS51. expected,         (Fig. 4; Supplemental Fig. S2). used 35-DAG plants <br /> transcript levels OsMADS51, OsGI, Ehd1, OsMADS14,      flowered early   Hd3a higher SD-grown plants.         regulatory genes expressed earlier. wild-type <br /> ingly, expression difference SDs LDs     control plants, transcript levels Ehd1, OsMADS14, greatest OsMADS14 Hd3a.          Hd3a low 35 DAG LDs (Fig. 4), <br /> appears LD-specific inhibitors coordinately regu-    transcripts detected 55 DAG (Fig. 3), <br /> late downstream genes. Hd1 candidate         showing age-dependent regulation patterns. proposed negative regulator   expected knockout experiment, Ehd1, Hd3a, LDs (Yano et al., 2000), transcript    OsMADS14 transcript levels increased (Fig. 4,  <br /> level higher LDs compared SDs.                 B, F, G, H) SD conditions,    Transcript levels examined   OsGI Hd1 significantly changed genes overexpressing OsMADS51 transgenic plants        UbiTOsMADS51 transgenic plants daylength <br />  <br />  <br />  <br /> Figure 4. Expression profiles various flowering <br /> regulators OsMADS51-overexpressing lines. <br /> Shown semiquantitative RT-PCR analyses OsMADS51, OsGI, Hd1, Ehd1, OsMADS14, Hd3a WT UbiTOsMADS51 plants (OX) grown 35 d SD   LD (B) conditions. Actin1 used normalization cDNA quantity. RT- <br /> PCR cycles numbered 28 flowering regulators SD; 32 Ehd1, OsMADS14, Hd3a; 28 OsMADS51, OsGI, Hd1 LD. Actin1, 25 cycles performed. independent experiments, reproducible data presented. Real-time PCR analyses OsMADS51 <br /> (C), OsGI (D), Hd1 (E), Ehd1 (F), OsMADS14 (G), Hd3a (H) WT OsMADS51 mutant plants (HO) <br /> grown 35 d SD LD conditions. RNA prepared leaf blades harvested 0 h light turned  data point average independent experiments. y axis, Relative <br /> values transcript levels regulatory genes Ubi; vertical bars, standard deviation. <br />  <br />  <br />  <br />  <br /> 1488                                                                                  Plant Physiol. Vol. 145, 2007 <br />                                                                      OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> (Fig. 4,  B, D, E). results support       Hd3a demonstrated function  <br /> conclusion OsMADS51 functions upstream            stream OsGI (Hayama et al., 2003; Doi et al., 2004). <br /> Ehd1, Hd3a, OsMADS14.                                 expected, transcript levels reduced                                                           OsGI antisense plants (Fig. 5, E&#226;&#8364;&#8220;G). results <br /> Expression Analyses OsMADS51 OsGI                   indicate OsGI does function pathway <br /> Antisense Plants                                          OsMADS51, OsGI occurring upstream. <br />  <br />    osmads51 mutants UbiTOsMADS51 trans-            Expression Analyses OsMADS51 Ehd1-RNAi Plants <br /> genic plants, transcript levels OsGI unchanged <br /> (Figs. 3 4).  believe OsGI      osmads51 plants, Ehd1 transcript level functions upstream OsMADS51 inde-     significantly reduced (Fig. 3, F), pendent pathway. Hayama et al. (2003) previ-         notably increased UbiTOsMADS51 plants (Fig. <br /> ously reported OsGI RNAi plants delayed         4, F).  likely Ehd1 located <br /> flowering SDs, observed late-          downstream OsMADS51. confirm hypothe- <br /> flowering phenotype OsGI antisense plants.         sis, generated transgenic plants carrying Ehd1-  SD-specific effect suggests OsGI     RNA interference (RNAi) construct (Fig. 6B).  30 OsMADS51 function path-           independently transformed plants obtained, <br /> way. investigate relationship    20 flowered later control. genes, generated OsGI antisense plants       observed Ehd1 transcript levels reduced OsMADS51 transcript level reduced         transgenics delayed. RNAi plants <br /> plants grown SDs LDs (Fig. 5A).        selected showed significant change <br /> controls, examined expression levels         flowering time. SD conditions, flowering regulators&#226;&#8364;&#8221;Ehd1, OsMADS14,            flowered 7 d later controls. Ehd1 transcript <br />  <br />                                                                         Figure 5. Expression level analyses flowering reg- <br />                                                                         ulators OsGi antisense plants.  Schematic dia- <br />                                                                         gram OsGi. F1, F2, R1, R2 primers used                                                                         RT-PCR real-time PCR analyses. B, Schematic <br />                                                                         diagram OsGi antisense construct. C, RT-PCR <br />                                                                         analysis OsGi antisense (OsGi  plants                                                                         wild-type segregants (WT). OsGi plants                                                                         grown 55 d SDs LDs. RNA prepared <br />                                                                         leaf blades harvested 0 h light                                                                         turned  Endogenous OsGI transcript ampli- <br />                                                                         fied F1 R1 primer sets, OsGI                                                                         endogenous transcripts amplified                                                                         F2 R2 primers. PCR cycles numbered 28 OsGI <br />                                                                         25 Actin1. Actin1 used normalization <br />                                                                         cDNA quantity. Transcript amounts OsMADS51 <br />                                                                         (D), Ehd1 (E), OsMADS14 (F), Hd3a (G)                                                                         analyzed quantitative real-time PCR. data <br />                                                                         point average independent <br />                                                                         experiments. y axis, Relative values tran- <br />                                                                         script levels regulatory genes Ubi; vertical <br />                                                                         bars, standard deviation. <br />  <br />  <br />  <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                           1489 <br />  Kim et al. <br />  <br />  <br /> levels reduced late-flowering transgenic   level low beginning day (Fig. 7A), plants significant RNAi tran-       increased light reached maximum script accumulated (Fig. 6C). primers used   end day. darkness, level continu- detecting RNAi transcript amplified       ously reduced minimal  similar rhythm <br /> dogenous Ehd1 transcript  PCR    observed LDs (data shown). SDs, band wild type.                                   similar induction curve observed OsMADS51 <br />    Ehd1-RNAi plants, OsGI OsMADS51 tran-      (Fig. 7B), increasing light decreasing script levels changed (Fig. 6, D E),    dark.  change transcript amounts firming function upstream Ehd1.          great OsGI. LD, previously reported (Doi et al., 2004), expression    rhythm (data shown). <br /> Hd3a significantly reduced Ehd1-RNAi <br /> plants (Fig. 6G). OsMADS14 transcript level      Relationship OsMADS51 OsMADS50 reduced RNAi plants, confirming pos- <br /> sibility flowering regulators           relationship OsMADS51 OsMADS50 <br /> stream OsGI (Fig. 6F).                                investigated. previously reported <br />                                                          osmads50 plants late-flowering phenotype <br /> Diurnal Rhythms OsMADS51 OsGI                     (delay 1 month), overexpressors                                                          OsMADS50 display extreme early-flowering phe- <br />   OsMAD51 functions OsGI             notype (Lee et al., 2004).  mutations  <br /> Ehd1, transcript level MADS-box gene        regulated expression OsMADS14 Hd3a, <br /> oscillate OsGI Ehd1.  measured    suggesting OsMADS50 functions upstream OsMADS51 transcription leaf blades 55-DAG        OsMADS14. evaluate OsMADS50 plants grown SDs LDs (Fig. 7). reported      flowering activator influence OsMADS51, previously (Hayama et al., 2003), OsGI transcript    examined OsMADS51 expression altered <br />  <br /> Figure 6. Expression level analysis flowering reg- <br /> ulators Ehd1-RNAi plants.  Schematic diagram Ehd1. F1, F2, R1, R2 primers used RT-PCR real-time PCR analyses. B, Schematic diagram Ehd1-RNAi construct. C, RT-PCR analysis Ehd1- <br /> RNAi plants wild-type segregants (WT). <br /> Endogenous Ehd1 transcript amplified F1 R1 primer sets Ehd1-RNAi endog- <br /> enous transcripts amplified F2 R2 <br /> primers. PCR cycles numbered 28 Ehd1 25 Actin1. Actin1 used normalization cDNA <br /> quantity. D G, Real-time PCR analysis OsMADS51 <br /> (D), OsGI (E), OsMADS14 (F), Hd3a (G) Ehd1- <br /> RNAi plants. Samples RT-PCR real-time PCR harvested 0 h light turned  data point average independent <br /> experiments. y axis, Relative values tran- <br /> script levels regulatory genes Ubi; vertical <br /> bars, standard deviation. <br />  <br />  <br />  <br />  <br /> 1490                                                                                   Plant Physiol. Vol. 145, 2007 <br />                                                                                     OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br />                                                                        rice Arabidopsis (Izawa et al., 2003). <br />                                                                         regulatory factors just                                                                        model systems. example, FLC present <br />                                                                        Arabidopsis, Ehd1 exclusive rice <br />                                                                        (Michaels Amasino, 1999; Doi et al., 2004).  <br />                                                                        fore, understanding mechanisms regulate <br />                                                                        flowering involves focusing species-specific <br />                                                                        genes gain essential information elucidate <br />                                                                        details diversification species.                                                                        fact, existence FLC family Arabidopsis <br />                                                                        makes possible explain vernalization pathway <br />                                                                        crucifers, does exist rice (Michaels                                                                        Amasino, 1999). <br />                                                                             speculated MADS-box <br />                                                                        genes diversified control gene-specific <br />                                                                        flowering mechanisms. OsMADS51 selected                                                                        investigation regard MADS-box genes <br />                                                                        group grass species. <br />                                                                        OsMADS51 identified differential analy- <br />                                                                        sis transcripts mature spikelets (Ciaffi et al., <br />                                                                        2005). expression pattern OsMADS51 <br />                                                                        studied (Shinozuka et al., 1999),                                                                        report concerns functional roles gene. <br />                                                                           transgenic analyses showed OsMADS51                                                                        SD-dependent flowering activator. SDs, <br />                                                                        OsMADS51 mutants flower 2 weeks later com- <br /> Figure 7. Diurnal rhythms OsMADS51   OsGI (B) SD        pared control, transformants conditions. Plants grown SDs 10 h light (0&#226;&#8364;&#8220;10 h, white   overexpress OsMADS51 flower 1 2 weeks early. <br /> box) 14 h dark (10&#226;&#8364;&#8220;24 h, black box). RNA prepared leaf <br />                                                                        LDs, effects milder, OsMADS51 <br /> blades 50-DAG plants 0, 4, 8, 12, 16, 20 h light turned  data point average independent <br />                                                                        mutants flowering 3 4 d later overexpressors <br /> experiments. y axis, Relative values transcript levels      flowering 1 2 d early. rice, SD <br /> regulatory genes Ubi; vertical bars, standard deviation.           promotion pathways proposed medi- <br />                                                                        ated Hd1 Ehd1 (Yano et al., 2000; <br />                                                                        Doi et al., 2004). pathways converge induce osmads50 plants.  proved                                                                        Hd3a resultant protein product moved case (Supplemental Fig. S3).                                                                        shoot apical meristem induce flowering (Tamaki <br /> levels OsMADS50 unchanged osmads51 <br /> plants (Supplemental Fig. S3). results indicate MADS-box genes function independently control flowering time. <br /> Analysis Transcript Levels OsMADS51 According Developmental Stage <br />    OsMADS51 early-acting activator, measured transcript levels various developmen- <br /> tal stages remain constant change plants matured. 0 h (immediately light turned , sampled leaf blades plants grown SDs. Real-time PCR analyses showed transcription high early develop- <br /> mental stages, day 43, declined remained low remaining period (Fig. 8). indicates OsMADS51 functions early 2 <br /> months growth rice. <br />                                                                        Figure 8. Changes OsMADS51 transcript levels growth <br />                                                                        phase. Plants reared SD conditions (10 h light/14 h dark). <br /> DISCUSSION                                                             RNA prepared leaf blades 13-, 28-, 43-, 54-, 69-, 81-,                                                                        97-DAG plants 0 h light turned  data point   Intensive molecular genetics studies regula-                  average independent experiments. y axis, Relative <br /> tory mechanisms control flowering time                       values transcript levels regulatory genes Ubi; vertical <br /> revealed major players conserved                    bars, standard deviation. <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                                           1491 <br />  Kim et al. <br />  <br />  <br />                                                                          target OsGI, levels OsMADS51                                                                          Ehd1 regulated OsGI antisense plants. <br />                                                                          results suggest OsMADS51 acts  <br />                                                                          stream OsGI. daily rhythm observed                                                                          OsMADS51 supports hypothesis                                                                          rhythm similar OsGI, showing                                                                          peak middle day (Fig. 7).                                                                           SD promotion pathway mediated OsGI                                                                          divided routes. known Hd1- <br />                                                                          mediated pathway, OsMADS51- <br />                                                                          Ehd1-mediated pathway. finding consistent <br />                                                                          previous report Hd1 works regardless                                                                          function Ehd1 (Doi et al., 2004). Furthermore, <br />                                                                          study confirms Hd1 Ehd1 operate inde- <br />                                                                          pendently provides additional evidence Figure 9. Model flowering-signal pathways rice photo-     OsGI common upstream regulator Hd1 Ehd1 <br /> period signal translated OsGI, divided inde-   (Fig. 9). <br /> pendent pathways. mediated Hd1; OsMADS51             Ehd1 OsMADS51 occur orphan genes Ehd1. OsMADS50 generates independent signal turns                                                                          family rice genome putative <br /> OsMADS14. signals finally activate Hd3a, produces putative florigen. <br />                                                                          orthologs Arabidopsis genome. <br />                                                                           unique genes probably function dif- <br />                                                                          ferentiate SD-inducible rice LD-inducible Arabi- <br />                                                                          dopsis.  conclude OsMADS51 plays <br /> et al., 2007). investigate relationship                  important role transmitting flowering signal <br /> regulators OsMADS51, studied expres-                       OsGI Ehd1 SD promotion pathway sion patterns OsMADS51 mutants. Transcript levels                     route independent previously repor- OsMADS14 examined                            ted pathways mediated Hd1 OsMADS50 (Fig. 9). <br /> gene regulated Ehd1 mutant  <br /> ground (Fig. 3) ectopic expression causes <br /> significant early flowering (Fig. 4). <br />    RT-PCR quantitative real-time PCR analyses                        MATERIALS METHODS <br /> demonstrated Ehd1 expression regulated                      Plant Materials Growth OsMADS51 mutants regulated Ubi: <br /> OsMADS51 plants. suggests OsMADS51                                T-DNA-tagging lines generated rice (Oryza sativa) var. japonica <br />                                                                          &#226;&#8364;&#732;Dongjin&#226;&#8364;&#8482; &#226;&#8364;&#732;Hwayoung&#226;&#8364;&#8482; (Jeon et al., 2000; Jeong et al., 2002). Insertion sites <br /> acts upstream Ehd1. explore relation-                 determined inverse PCR  et al., 2003; Ryu et al., 2004; Jeong et al., <br /> ship OsMADS51 Ehd1, generated                             2006). T-DNA-tagged mutants searched flanking sequence <br /> Ehd1 RNAi plants measured OsMADS51 mRNA                              database (www.postech.ac.kr/life/pfg/risd), linked Salk <br /> levels RNAi plants. previously identified Ehd1                    Institute Genomic Analysis Laboratory (http://signal.salk.edu/cgi-bin/RiceGE; <br />                                                                          et al., 2005a, 2005b). Plants reared growth rooms mutant allele <span id='am-7' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> change reduces <br />                                                                          continuous SDs (10 h light/14 h dark) LDs (14 h light/10 h dark). <br /> DNA-binding ability reduces Ehd1 function (Doi <br /> et al., 2004). RNAi results support role Ehd1 flowering activator. Ehd1 RNAi                       Screening OsMADS51 Mutants plants, OsMADS51 mRNA levels similar                         T-DNA-Tagging Lines <br /> RNAi control plants, OsMADS14                                osmads51 homozygous progeny screened T1 seeds mu- <br /> Hd3a expression affected mutation.                      tants. Genotyping performed 35 cycles 95&#194;&#176;C 15 s, 55&#194;&#176;C 30 s, <br /> fore, combined results OsMADS51                        72&#194;&#176;C 40 s, using combination specific T-DNA border primer sets <br />                                                                          follows. primer sets osmads51-1 included F2 primer (5#-CAACAACT- <br /> mutants, conclude OsMADS51 transmits                           GACGAGGAGCAC-3#), R2 primer (5#-TAGATCATGACCTCCAGTAC-3#), flowering signal Ehd1, turns                       Gus2R primer T-DNA (5#-TTGGGGTTTCTACAGGACGTAAC-3#). OsMADS14 Hd3a. OsMADS51 regulates Ehd1,                              osmads51-2 primer sets F3 primer (5#-CCTCCCATGTTGTGTTCCTT-3#), <br /> OsMADS14, Hd3a SDs                           R3 primer (5#-AAATGGCAAACCACTCGAAC-3#), LB2R primer T-DNA <br />                                                                          (5#-ACGTCCGCAATGTGTTATTAAG-3#). Gus2R LB2R primers LDs.  absolute amounts flower- <br />                                                                          used obtain T-DNA flanking sequences inverse PCR  et al., 2003; <br /> ing activators quite low LDs, probably                   Jeong et al., 2006). sufficient inducing flowering signal. <br />    contrast, levels Hd1 OsGI notably <br />                                                                          Vector Construction Transformation <br /> changed OsMADS51 mutants overexpressing <br /> plants, implying OsMADS51 acts                           length cDNA clone OsMADS51 (GenBank accession  <br /> downstream genes independently. dis-                      AB003327; Shinozuka et al., 1999) amplified PCR using forward <br />                                                                          primer (5#-aagcttggggagctagggtttttcg-3#) reverse primer (5#-ggtaccaaact- <br /> sect possibilities, applied antisense technol-                  tatgcacaggacacc-3#). primers contained restriction enzyme <br /> ogy evaluate transgenic plants suppress OsGI.                    sites, HindIII Asp-718 (underlined primer sequences), subsequent level Hd3a, known downstream                           cloning. PCR product cloned pBluescript SK2 (Stratagene). <br />  <br /> 1492                                                                                                                Plant Physiol. Vol. 145, 2007 <br />                                                                                                 OsMADS51, Short-Day Flowering Promoter Rice <br />  <br />  <br /> Afterward, cDNA subcloned pGA1611 binary vector (Lee et al.,         ACKNOWLEDGMENTS <br /> 1999; Kim et al., 2003) maize (Zea mays) Ubi promoter nopaline synthase (nos) terminator generate UbiTOsMADS51 construct.           express thanks Ko Shimamoto providing pANDA create Ehd1-RNAi construct, PCR amplified 346-bp Ehd1 cDNA          (p2K-1) binary vector, Chung-Hwan Ryu Dong-Hoon Jung valu- <br /> fragment (724&#226;&#8364;&#8220;1,069 bp translation initiation site Ehd1 cDNA      able discussions. acknowledge Priscilla Licht critical reading clone; GenBank accession AB092506), using forward primer (5#-gtt-            manuscript. <br /> gccagtcatctgcag-3#) reverse primer (5#-ggatgtggatcatgagacat-3#).                                                                                   Received June 4, 2007; accepted October 9, 2007; published October 19, 2007. <br /> PCR product cloned pGA2960, Gateway vector modified pENTR 1A (Invitrogen). Afterward, product subcloned pANDA (p2K-1) binary vector clonase reaction (Miki Shimamoto, <br /> 2004). OsGI antisense construct generated inserting partial <br /> cDNA clone binary vector pGA1611 antisense orientation.                                                                                   LITERATURE CITED <br /> cDNA clone OsGI (GenBank accession  AJ133787mRNA) amplified             Abe M, Kobayashi Y, Yamamoto S, Daimon Y, Yamaguchi  Ikeda Y, PCR using forward primer (5#-agacagcaaatttgactgcggcag-3#)                 Ichinoki H, Notaguchi M, Goto K, Araki T (2005) FD, bZIP protein <br /> reverse primer (5#-gacatgctgagtgcacggacatgcg-3#). PCR product          mediating signals floral pathway integrator FT shoot <br /> cloned pGEM-T vector (Promega). OsGI cDNA product               apex. Science 309: 1052&#226;&#8364;&#8220;1056 <br /> digested restriction enzymes XhoI (991 bp start codon) XbaI         Alvarez-Buylla ER, Pelaz S, Liljegren SJ, Gold SE, Burgeff C, Ditta GS, <br /> (2,374 bp start codon). digested product cloned pBluescript        Ribas Pouplana L, Martinez-Castilla L, Yanofsky MF (2000) SK2 subcloned pGA1611 binary vector using KpnI SacI. Rice           ancestral MADS-box gene duplication occurred divergence transformation performed according Agrobacterium-mediated coculti-            plants animals. Proc Natl Acad Sci USA 97: 5328&#226;&#8364;&#8220;5333 <br /> vation methods (Lee et al., 1999). transgenic rice plants grown   G, Jeong DH, Jung KH, Lee S (2005a) Reverse genetic approaches greenhouse transferred confined paddy field.                           functional genomics rice. Plant Mol Biol 59: 111&#226;&#8364;&#8220;123 <br />                                                                                   G, Lee S, Kim SH, Kim SR (2005b) Molecular genetics using T-DNA                                                                                      rice. Plant Cell Physiol 46: 14&#226;&#8364;&#8220;22 <br /> RNA Isolation Semiquantitative RT-PCR                                         G, Watson BD, Stachel S, Gordon MP, Nester EW (1985) New cloning <br />                                                                                      vehicles transformation higher plants. EMBO J 4: 277&#226;&#8364;&#8220;284 <br />     Total RNA isolated Tri Reagent (MRC). strand cDNA          S, Park S, Jeong DH, Lee DY, Kang HG, Yu JH, Hur J, Kim SR, Kim <br /> synthesized 2 mg total RNA, using Moloney murine leukemia virus              YH, Lee M, et al (2003) Generation analysis end sequence <br /> reverse transcriptase (Promega), 10 ng oligo(dT)15 primer            database T-DNA tagging lines rice. Plant Physiol 133: 2040&#226;&#8364;&#8220;2047 <br /> 2.5 mM deoxyriboneucleotide <span id='am-2' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span>, described Han et al. (2006).      Araki T (2001) Transition vegetative reproductive phase. Curr Opin <br /> Actin1 used normalization cDNA quantity. Primers included             Plant Biol 4: 63&#226;&#8364;&#8220;68 <br /> 5#-GTTTGCTCTGCTCCTACTC-3# 5#-ACTCCTCCTCCAGCATTGAA-3#                          Ciaffi M, Paolacci AR, D&#226;&#8364;&#8482;Aloisio E, Tanzarella OA, Porceddu E (2005) OsMADS51; 5#-TGGAGAAAGGTTGTGGATGC-3# 5#-CATAGACGG-                           Identification characterization gene sequences expressed CACTTCAGCAGAT-3# OsGI; 5#-GTTGCCAGTCATCTGCAGAA-3#                            wheat spikelets heading stage. Gene 346: 221&#226;&#8364;&#8220;230 <br /> 5#-GGATGTGGATCATGAGACAT-3# Ehd1; 5#-TTCTCCTCTCCAAA-                           Doi K, Izawa T, Fuse T, Yamanouchi U, Kubo T, Shimatani  Yano M, <br /> GATTCC-3# 5#-CATACGCCTTTCTTGTTTCA-3# Hd1; 5#-TCCTATG-                        Yoshimura (2004) Ehd1, B-type response regulator rice, confers <br /> CAGAAAAGGTCCTT-3# 5#-GGACGAAGCCAAAATATACAC-3#                                short-day promotion flowering controls FT-like gene expression <br /> OsMADS14; 5#-ATGGCCGGAAGTGGCAGGGAC-3# 5#-ATCGATCGG-                              independently Hd1. Genes Dev 18: 926&#226;&#8364;&#8220;936 <br /> GATCATCGTTAG-3# Hd3a; 5#-GTATCCATGAGACTACATACATA-                         Fowler S, Lee K, Onouchi H, Samach  Richardson K, Morris B, <br /> CAACT-3# 5#-TACTCAGCCTTGGCAATCCACA-3# Actin1.                            Coupland G, Putterill J (1999) GIGANTEA: circadian clock-controlled <br /> reactions included initial 5 min denaturation 95&#194;&#176;C, followed             gene regulates photoperiodic flowering Arabidopsis encodes <br /> 95&#194;&#176;C 15 s, 55&#194;&#176;C 30 s, 72&#194;&#176;C 40 s. PCR cycles numbered 28      protein possible membrane-spanning domains. EMBO J <br /> flowering-regulator genes 25 Actin1.                                         18: 4679&#226;&#8364;&#8220;4688 <br />                                                                                   Han MJ, Jung KH, Yi G, Lee DY, G (2006) Rice Immature Pollen 1 (RIP1)                                                                                      regulator late pollen development. Plant Cell Physiol 47: 1457&#226;&#8364;&#8220;1472 <br />                                                                                   Hayama R, Yokoi S, Tamaki S, Yano M, Shimamoto K (2003) Adaptation Quantitative Real-Time RT-PCR <br />                                                                                      photoperiodic control pathways produces short-day flowering rice. <br />     Real-time PCR performed using Roche LightCycler II previously           Nature 422: 719&#226;&#8364;&#8220;722 <br /> described (Han et al., 2006). cDNA prepared leaf blades plants    Hiei Y, Komari T, Kubo T (1997) Transformation rice mediated 35 55 DAG. Ubi mRNA level used normalize expression             Agrobacterium tumefaciens. Plant Mol Biol 35: 205&#226;&#8364;&#8220;218 <br /> ratio gene. Primers included 5#-AACCAAGATCGGCAGTATGG-3#              Huang T, Bohlenius H, Eriksson S, Parcy F, Nilsson O (2005) mRNA <br /> 5#-GATTGATTGCTCCAGCAGGT-3# Hd1; 5#-GCCAACTAATGCTCGA-                             Arabidopsis gene FT moves leaf shoot apex induces <br /> GTCC-3# 5#-CAGCTGCTGCAGGGTAGTTA-3# OsMADS14; 5#-AGC-                         flowering. Science 309: 1694&#226;&#8364;&#8220;1696 <br /> CCAAGTGACCCTAACCT-3# 5#-GTTGTAGAGCTCGGCGAAGT-3#                           Izawa T, Oikawa T, Sugiyama N, Tanisaka T, Yano M, Shimamoto K <br /> Hd3a; 5#-CACGGTTCAACAACATCCAG-3# 5#-TGAAGACCCTGA-                            (2002) Phytochrome mediates external light signal repress FT <br /> CTGGGAAG-3# Ubi. Changes gene expression calculated              orthologs photoperiodic flowering rice. Genes Dev 16: 2006&#226;&#8364;&#8220;2020 <br /> DDCt method.                                                                      Izawa T, Takahashi Y, Yano M (2003) Comparative biology comes                                                                                      bloom: genomic genetic comparison flowering pathways rice <br />     Sequence data article GenBank/EMBL data            Arabidopsis. Curr Opin Plant Biol 6: 113&#226;&#8364;&#8220;120 <br /> libraries accession numbers AB003327, AB092506, AJ133787.               Jeon J, G (2001) Gene tagging rice: high throughput                                                                                      functional genomics. Plant Sci 161: 211&#226;&#8364;&#8220;219 <br />                                                                                   Jeon JS, Lee S, Jung KH, Jun SH, Jeong DH, Lee J, Kim C, Jang S, Yang K, <br />                                                                                      Nam J, et al (2000) T-DNA insertional mutagenesis functional <br /> Supplemental Data                                                                    genomics rice. Plant J 22: 561&#226;&#8364;&#8220;570 <br />                                                                                   Jeon JS, Lee S, Jung KH, Yang WS, Yi GH, Oh BG, GH (2000) Production    following materials available online version article. <br />                                                                                      transgenic rice plants showing reduced heading <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> plant height    Supplemental Figure S1. Expression profiles various flowering reg-            ectopic expression rice MADS-box genes. Mol Breed 6: 581&#226;&#8364;&#8220;592 <br />      ulators 55-DAG osmads51-2 plants.                                         Jeong DH, S, Kang HG, Moon S, Han JJ, Park S, Lee HS, K, G <br />                                                                                      (2002) T-DNA insertional mutagenesis activation tagging rice. <br />    Supplemental Figure S2. Expression profiles various flowering reg- <br />                                                                                      Plant Physiol 130: 1636&#226;&#8364;&#8220;1644 <br />      ulators 35-DAG UbiTosmads51 plants line 4. <br />                                                                                   Jeong DH, S, Park S, Kang HG, Park GG, Kim SR, Sim J, Kim YO, Kim <br />    Supplemental Figure S3. Analyses relationship OsMADS51                 MK, Kim SR, et al (2006) Generation flanking sequence-tag <br />      OsMADS50.                                                                   database activation-tagging lines japonica rice. Plant J 45: 123&#226;&#8364;&#8220;132 <br />  <br /> Plant Physiol. Vol. 145, 2007                                                                                                                             1493 <br />  Kim et al. <br />  <br />  <br /> Kardailsky  Shukla VK, Ahn JH, Dagenais N, Christensen SK, Nguyen          Park DH, Somers  Kim YS, Choy YH, Lim HK, Soh MS, Kim HJ, Kay <br />    JT, Chory J, Harrison MJ, Weigel D (1999) Activation tagging          SA, Nam HG (1999) Control circadian rhythms photoperiodic <br />    floral inducer FT. Science 286: 1962&#226;&#8364;&#8220;1965                                    flowering Arabidopsis GIGANTEA gene. Science 285: 1579&#226;&#8364;&#8220;1582 <br /> Kim SR, Lee S, Kang HG, Jeon JS, Kim KM, G (2003) complete              Putterill J, Robson F, Lee K, Simon R, Coupland G (1995) CONSTANS <br />    sequence pGA1611 binary vector. J PIant Biol 16: 211&#226;&#8364;&#8220;214              gene Arabidopsis promotes flowering encodes protein showing <br /> Kobayashi Y, Kaya H, Goto K, Iwabuchi M, Araki T (1999) pair               similarities zinc finger transcription factors. Cell 80: 847&#226;&#8364;&#8220;857 <br />    related genes antagonistic roles mediating flowering signals.     Ryu CH, JH, Kang HG, Hur J, Kim YH, Ha <br /> </body></html>