<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>APPLIED ENVIRONMENTAL MICROBIOLOGY, Jan. 2009, p. 483&#226;&#8364;&#8220;492                                                                        Vol. 75,  2 <br /> 0099-2240/09/$08.00&#226;&#171;&#185;0 doi:10.1128/AEM.01616-08 <br /> Copyright &#194;&#169; 2009, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br /> Overproduction Exopolysaccharides Escherichia coli K-12 rpoS <br />               Mutant Response Osmotic Stress&#228;&#338;&#164;&#226;&#8364;&#160; <br />                                             Michael Ionescu Shimshon Belkin* <br />           Department Plant Environmental Sciences, Institute Life Sciences, Hebrew University Jerusalem, <br />                                                     Jerusalem 91904, Israel <br />                                                Received 15 July 2008/Accepted 30 October 2008 <br />  <br />              yjbEFGH operon implicated production exopolysaccharide unknown function           induced osmotic stress negatively regulated general stress response sigma factor RpoS. Despite <br />           obvious importance RpoS, negative selection rpoS reported place starved cultures, <br />           suggesting adaptive occurrence allowing overexpression RpoD-dependent uptake nutrient- <br />           scavenging systems. trade RpoS-dependent functions improved nutrient utilization abilities <br />           makes bacterium sensitive environmental stressors, e.g., osmotic stress. work, addressed <br />           hypothesis overinduction genes rpoS-deficient strains indicates essentiality. Using DNA <br />           microarrays, real-time PCR, transcriptional fusions, genes wca operon, implicated           production colanic acid exopolysaccharide, previously shown induced osmotic stress,           negatively controlled RpoS. exopolysaccharides synthesis yjb wca involved           overproduced rpoS mutant osmotic stress. operons essential           rpoS-deficient strain wild type; promoters operons constitutively active yjb rpoS <br />           mutants; strain produces extremely mucoid colonies, forms long filaments, exhibits reduced growth <br />           capability. addition, wca rpoS mutant&#226;&#8364;&#8482;s growth inhibited osmotic stress. results indicate           induced wild type, operons valuable rpoS-deficient strain, suggesting <br />           overproduction exopolysaccharides adaptive action. <br />  <br />  <br />    role general stress response sigma factor RpoS (&#226;?&#180;38)             overexpression RpoD-dependent nutrient-sensing Escherichia coli&#226;&#8364;&#8482;s physiology extensively studied.                 uptake systems. studies focused characterization genes                  previously shown (24, 44) E. coli genes <br /> positively controlled RpoS physiological implica-                shown significantly induced osmotic stress, gene, <br /> tions underexpression genes rpoS-deficient                  yjbF, stood promoter drives induction <br /> strains (22, 61), refer genes negatively regulated            appeared negatively regulated RpoS. gene, mem- <br /> RpoS consequences overexpression.                          ber gene operon (yjbEFGH), listed (15)    Inactivation rpoS common; mutations rpoS gene                  genes positively dependent RcsC, inner mem- detected laboratory bacterial stocks (25, 52),              brane sensor kinase Rcs phosphorelay  Rcs environmental clinical isolates pathogenic               controls variety physiological functions prokaryotes, <br /> commensal enteric bacteria (1, 43), suggesting certain              extracellular <span id='am-5' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span>polysaccharide</span> (EPS) synthesis (27, 42, 51, <br /> conditions, loss attenuation RpoS activity                 58), biofilm formation (10, 15, 41), cell division (3), motility <br /> adaptive value. reports indicated rpoS gene tends               (17, 53). shown (16) genes undergo frequent mutations lead loss activity           yjbEFGH operon transcribed  operon mutated forms appear spread dominant                      induced growth solid surfaces, involved glucose-limited chemostat cultures (39) incubation                production unknown EPS. <br /> stationary phase (5, 66). shown rate rpoS                 E. coli K-12 possesses known sets genes promoting <br /> mutations spreading population decreased                   EPS production:   genes involved O-antigen synthesis (46); <br /> osmotic stress applied (14, 28). selection               (ii) wca operon, responsible colanic acid (CA)-EPS <br /> pressure rpoS gene thought largely                    synthesis (19); (iii) pgaABCD operon encodes genes <br /> competition sigma factors RpoS RpoD                    involved production poly-&#226;?&#164;-1,6-N-acetyl-D-glucos- <br /> limited number RNA polymerase core subunits (13, 32).                  amine (PGA), polysaccharide shown crucial biofilm <br /> suggests rpoS inactivation beneficial starved cell            formation (59); (iv) dfc pseudo-operon (comprised                                                                                gfcABCDE, etp, etk), responsible production type <br />                                                                                IV capsule E. coli O127:H6  operon nonfunctional                                                                                E. coli K-12 presence IS1 element    * Corresponding author. Mailing address: Department Plant            promoter region [40]); (v) yjbEFGH operon, paral- <br /> Environmental Sciences, Institute Life Sciences, Hebrew Uni-            ogue dfcABCD (15). <br /> versity Jerusalem, Jerusalem 91904, Israel. Phone: 972-2-6584192.              propose overinduction yjb operon  possibly, <br /> Fax: 972-2-6585559. E-mail: shimshon@vms.huji.ac  <br />    &#226;&#8364;&#160; Supplemental material article http://aem <br />                                                                                genes involved EPS production adap- <br /> .asm.org/.                                                                     tive response osmotic stress rpoS-deficient background <br />    &#228;&#338;&#164; <br />      Published ahead print 7 November 2008.                              overinduced genes special value  <br />                                                                          483 <br />  484       IONESCU BELKIN                                                                                                          APPL. ENVIRON. MICROBIOL. <br />  <br />           TABLE 1. Strains plasmids used study                           Bio-X-Act DNA polymerase (Bioline) presence 200 nM template                                                                                      primers using manufacturer&#226;&#8364;&#8482;s reagents instructions. &#226;&#172;&#402;1,500-bp prod- <br />                                                                       Source      uct gel purified using QIAquick gel extraction kit (Qiagen), digested  Strain plasmid                     Genotype <br />                                                                      reference(s)    DpnI eliminate template plasmid, desalted using kit. Strains                                                                              PCR product transformed DY378 (64), recombination-permis- <br />   MG1655                  F&#226;&#171;&#186; &#226;?&#173;&#226;&#171;&#186;                                     Laboratory       sive strain harboring lysogenically defective lambda phage. overnight cul- <br />                                                                       stock          ture grown 30&#194;&#176;C regrown OD600 0.6, heat shocked 15 min   QC2410                  MG1655 rpoS::Tn10                         31               induce lambda PL promoter (controlling lambda recombination promoting <br />   MK1399                  MG1655 &#226;&#338;&#172;yjbEFGH(kanR)                     work        factors), cooled ice, washed times cold double-distilled water (DDW), <br />   MK1310                  MG1655 &#226;&#338;&#172;yjbE(kanR)                        work        finally resuspended 1 ml DDW. Aliquots (100 &#226;?&#174;l) mixed ice   MK1351                  MG1655 &#226;&#338;&#172;Pwza(kanR) (allele                work        50 &#226;?&#174;l purified PCR product. Electrical DNA transformation carried <br />                             introduced MS1651                                   2.5 mV (Electro Cell manipulator ECM 935; BTX, United States),                             &#229;&#8230;&#179;40&#229;&#8230;&#180;)                                                    cells grown 1 h 37&#194;&#176;C prewarmed LB broth. Positive recombi- <br />   MK2499                  QC2410 &#226;&#338;&#172;yjbEFGH(kanR)                     work        nants selected LB agar plates supplemented kanamycin verified <br />   MK2431                  QC2410 &#226;&#338;&#172;yjbE(yjbE::kanR)                  study       PCR sequencing. <br />   MK2411                  QC2410 &#226;&#338;&#172;Pwza(kanR)                        work           Allele transduction P1 kc coliphage. new inactive alleles trans- <br />   CF6343                  MG1655 &#226;&#338;&#172;lacIZ(MluI)                       8                duced MG1655 P1 kc transduction described previously (48).   SK1150                  CF6343 rcsA::kanR                         17               rpoS::Tn10 allele QC2410 transduced MK1399, MK1310,   SK1158                  CF6343 rcsB::Tn10                         8                MK1351 produce double mutants MK2499, MK2431, MK2411. <br />   SK1154                  CF6343 rcsC::Tn10                         8                Briefly, overnight donor bacterial strain culture (carrying defective allele) <br />   MS1651                  EPEC O126:H6 &#226;&#338;&#172;Pwza(kanR)                  40               regrown 10 ml LB supplemented 5 mM CaCl2 40 min   DY378                   W3110 &#226;?&#173;cI857&#226;&#338;&#172;(cro-bioA)                   64               addition 100 &#226;?&#174;l P1 kc coliphage (produced MG1655) incubated <br />                             (permissive strain gene                              37&#194;&#176;C shaking (200 rpm) 3 h culture completely lysed                             inactivation)                                            phage. Cells survived lysis killed chloroform, cell debris <br />                                                                                      removed centrifugation. supernatant containing new phages Plasmids                                                                             collected, various volumes incubated 37&#194;&#176;C 40 min   pDEW201                 Promoterless; luxCDABE ori                56, 57           overnight MG1655 culture suspended 10 mM MgSO4, 5 mM CaCl2. P1 <br />                              pMB1(pBR322) rop ampR                                   infection stopped adding 100 &#226;?&#174;l 0.1 M Na3 citrate sample,   pDEW/yjbF               pDEW201; yjbF&#226;&#172;&#732;::luxCDABE                  56, 57           positive transductants selected LB agar plates supplemented     (pDEW609)                (pgi-yjbF)                                              appropriate antibiotics. <br />   pDEW/wza                pDEW201; wza&#226;&#172;&#732;::luxCDABE                   study          Cloning wza promoter. MG1655 genomic DNA (extracted   pDEW/osmY               pDEW201; osmY&#226;&#172;&#732;::luxCDABE                  56, 57           DNeasy plant mini kit [Qiagen]) used template amplifying wza <br />   pDEW/lon                pDEW201; lon&#226;&#172;&#732;::luxCDABE                   56, 57           promoter primers designed calculated annealing temperature   pKD13                   Template plasmid kanR                 11               72&#194;&#176;C target site tail containing restriction enzyme digestion site <br />                              amplification                                           followed 8 mismatched nucleotides far 5&#226;&#172;&#732; end. pDEW201 (56, 57)   pES2                    recA&#226;&#172;&#732;::gfpUV ori pUC ampR                 45               used <span id='am-54' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acceptor</span> new DNA fragment. DNAs (plasmid PCR <br />   pATC400                 pBR322 rcsA&#226;&#171;&#185; ampR                         54               product) digested EcoRI KpnI desalted using QIAquick <br />                                                                                      gel extraction kit (Qiagen). Ligation carried T4 ligase (Roche) <br />                                                                                      according manufacturer&#226;&#8364;&#8482;s instructions treating linear vector                                                                                      shrimp alkaline phosphatase (Fermentas). <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> transformation                                                                                      AG1688 competent cells carried  Positive plasmids selected rpoS-deficient strain. results indicate alternative stress <br />                                                                                      colony PCR verified sequencing. <br /> response strategies come play absence RpoS                        DNA microarray rRT-PCR analysis. Single colonies strains MG1655 activity diminished, allowing cells survive                   QC2410 (rpoS::Tn10) grown (37&#194;&#176;C) overnight LB broth regrown proliferate general stress protection.                              fresh LB broth containing 0.09 0.7 M NaCl 90 180 min, respectively, <br />                                                                                      wild-type strain reached OD600 0.3. cultures diluted <br />                                                                                      density obtained rpoS mutant (OD600 &#226;&#171;&#189; 0.15), 50 ml                         MATERIALS METHODS                                        strain subjected total RNA extraction using RNeasy midi kit (Qiagen) <br />    Strains plasmids. strains used study listed Table 1.   DNase (Qiagen) treatment. RNA concentration determined strains grown Luria-Bertani medium (LB broth; 5 g liter&#226;&#171;&#186;1 yeast extract,     using ND-1000 spectrophotometer (NanoDrop Technologies, , Wilming- <br /> 10 g liter&#226;&#171;&#186;1 Bacto-tryptone, 5 g liter&#226;&#171;&#186;1 NaCl) supplemented appro-      ton, . micrograms total RNA reverse transcribed priate antibiotics (100 &#226;?&#174;g ml&#226;&#171;&#186;1 ampicillin, 10 &#226;?&#174;g ml&#226;&#171;&#186;1 tetracycline, 50 &#226;?&#174;g ml&#226;&#171;&#186;1      poly(T18) primer using RevertAid strand cDNA synthesis kit (Fermentas), <br /> kanamycin) optical density 600 nm (OD600) 1 2 (Spekol 1200;           mRNA expression assayed using Affymetrix GeneChip E. coli <br /> Analytikjene, Germany), mixed 50% glycerol, kept &#226;&#171;&#186;80&#194;&#176;C. Cells       antisense genome array (Weizmann Institute Science [Rehovot, Israel] <br /> recovered freezing overnight growth LB agar plates. Plates kept      MicroArray Unit) according manufacturer&#226;&#8364;&#8482;s instructions. DNA mi- 1 month 4&#194;&#176;C.                                                            croarray results confirmed relative real-time PCR (rRT-PCR) analysis <br />    Detection rpoS wca mutants. RpoS deficiency routinely checked          (30). nanograms cDNA mixed Sybr green reaction <br /> qualitatively catalase activity assay. drop 5 &#226;?&#174;l 32% <span id='am-59' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-60' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O2</span> placed    mixture (ABI) assayed ABI Prism 7000 sequence detection colony; immediate vigorous bubbling indicated wild-type RpoS activity (39),     (Applied Biosystems) relative quantification abundance lack bubbling interpreted RpoS deficiency. Deficiency wca           following genes: yjbE, yjbF, yjbG, yjbH, wzc, gmd, manC, wcaH, wcaD, (&#226;&#338;&#172;Pwza) tested transforming strain plasmid pATC400 (rcsA&#226;&#171;&#185;)           rcsA. additional genes similarly amplified RpoS-depen- <br /> (54). Wild-type strains extremely mucoid (CA-EPS overproduction),         dent osmY served negative control RpoS-independent house- <br /> &#226;&#338;&#172;Pwza strains did                                                                keeping gene rpmA served reference. gene yielded average <br />    Construction new inactive alleles. Nonpolar gene deletions carried    threshold cycle (CT) values (n &#226;&#171;&#189; 3) 17.8 &#226;&#171;&#190; 0.7 (mean &#226;&#171;&#190; standard deviation) described previously (11, 64), using primers listed Table 2. Briefly,      17.5 &#226;&#171;&#190; 0.6 MG1655 QC2410, respectively. reaction conditions <br /> pKD13 (11) used template DNA PCR primer pairs 60 bases             follows: 50&#194;&#176;C 2 min, hot start 95&#194;&#176;C 10 min, 40 cycles 95&#194;&#176;C long designed amplify kanamycin resistance gene plasmid (20          15 s, 60&#194;&#176;C 1 min. dissociation test (12) performed end bases 3&#226;&#172;&#732; end primer) undergo recombination process      run ensure single PCR product reaction mixture. Results edges chromosomal target site (40 bases 5&#226;&#172;&#732; primer     obtained SDS analysis program (ABI) using relative quantification <br /> identical recombination sites chromosome). &#226;&#338;&#172;Pwza allele        algorithm automatic CT calculation. &#226;&#338;&#172;CT calculated gene directly amplified MS1651 genome (40), extracted           &#226;&#338;&#172;CT &#226;&#171;&#189; CT(studied gene) &#226;&#171;&#186; CT(rpmA). &#226;&#338;&#172;&#226;&#338;&#172;CT calculated different <br /> DNeasy plant mini kit (Qiagen) according manufacturer&#226;&#8364;&#8482;s instructions.         ways.   RpoS dependency calculated equation &#226;&#338;&#172;&#226;&#338;&#172;CT &#226;&#171;&#189; <br /> PCR (TGradient; Biometra, United States) carried proofreading           &#226;&#338;&#172;CT(rpoS::Tn10) &#226;&#171;&#186; &#226;&#338;&#172;CT(MG1655). (ii) relative expression wca genes <br />  VOL. 75, 2009                                                                                  EPS OVERPRODUCTION rpoS MUTANT                                 485 <br />  <br />  <br />                                                                 TABLE 2. Primers used study <br />       Primer  purpose                                              Sequence (5&#226;&#172;&#732; 3 3&#226;&#172;&#732;                                                       Use <br />  <br /> Gene inactivation <br />  Pwza-F                                     CGTTGGTGATCGCCTGTTGACCGG                                                            Transfer Pwza allele                                                                                                                                   MS1651 MG1655 <br />   Pwza-R                                    GCCGTTCGTTCAGCTCCATCGTGG                                                            Transfer Pwza allele                                                                                                                                   MS1651 MG1655 <br />   yjbE-F                                    ATTTTTGCCATATCTGCGCTTGCGGCGACTTCTGCGT                                               Inactivation yjbEFGH yjbE <br />                                              GGGCGTGTAGGCTGGAGCTGCTTC <br />   yjbE-R                                    GTCGTGGTTGTGGTGGTCCCGGTATTAGAACCA                                                   Inactivation yjbE <br />                                              TCACCGCATTCCGGGGATCCGTCGACC <br />   yjbH-R                                    CGCACGGCTGCGTGTCGGGCCAGACGAGAAGAGAT                                                 Inactivation yjbEFGH <br />                                              CCAACGGTACATTCCGGGGATCCGTCGACC <br />  <br /> Cloning (pDEW201 based) <br />   wza-F                                     TATACATGGAATTCCGATTAACCCGGCCCAGATAGACG                                              EcoRI-mediated ligation <br />   wza-R                                     ATCAGTGGGGTACCCCCCCGCCCTCGCCTTTCAGCG                                                KpnI-mediated ligation <br />  <br /> rRT-PCR analysis <br />   yjbE-F                                    TTTGCCATATCTGCGCTTG <br />   yjbE-R                                    TGGTGGTCCCGGTATTAGAA <br />   yjbF-F                                    AAGCGACCTGCACTCATTCT <br />   yjbF-R                                    AGGCCAGCACCACAAATAAC <br />   yjbG-F                                    TATTGTCGCGTTGCTTTTGA <br />   yjbG-R                                    CCAGCTCTTCGCTAATCACC <br />   yjbH-F                                    GCTTGCTACGGCGAAACATA <br />   yjbH-R                                    GGGAAGAGTTGCACTGAAGC <br />   wzc-F                                     CCTCGATATTGCAGTGAGCA <br />   wzc-R                                     CGCTGCTCAGGGTGTAGTTT <br />  <br />   wcaD-F                                    CCCATCACCATCGTCACTTT <br />   wcaD-R                                    CCACACCATGCCAATAATGA <br />   gmd-F                                     ATGGCGACCTGAGTGATACC <br />   gmd-R                                     TTTCTTTTTCCAGACCGAGGA <br />   wcaH-F                                    AGGAAGACTTTGCCACGGTA <br />   wcaH-R                                    CTTCCAGCGTTTCGTCTTTC <br />   manC-F                                    ATGAGCAGCACCGCTTTATT <br />   manC-R                                    CCGCCAATACCAGCATTAAC <br />   rcsA-F                                    CGACATTGAAACCGTTGATG <br />   rcsA-R                                    GCAATTGCCATAAAAACGAT <br />  <br />   osmY-F                                    TGCTGGCTGTAATGTTGACC <br />   osmY-R                                    TCGGTGCTCTTGATGTTGTC <br />   rpmA-F                                    TAACGGTCGCGATTCAGAAG <br />   rpmA-R                                    GCAAACAGAGTGTGGTCACG <br />         Underlined letters indicate restriction enzyme digestion sites. <br />  <br />  <br />  <br />  <br /> compared yjb genes calculated equation &#226;&#338;&#172;&#226;&#338;&#172;CT &#226;&#171;&#189; [averaged                NaCl-exposed cells did change. Bioluminescence data normalized <br /> &#226;&#338;&#172;CT(wzc, gmd, manC, wcaH, wcaD)] &#226;&#171;&#186; [averaged &#226;&#338;&#172;CT(yjbE, yjbF, yjbG, yjbH)].             uniform cell density dividing measured light intensity (relative light <br /> RQ (relative RNA quantification) calculated RQ &#226;&#171;&#189; 2&#226;&#171;&#186;&#226;&#338;&#172;&#226;&#338;&#172;CT                 units) OD600 value measured time point. <br /> results experiment. Results presented averaged RQ values (n &#226;&#171;&#189; 3).       Maximum bioluminescence refers peak activity obtained course <br />    Bioluminescence monitoring promoter induction. Induction promoters            300 min measurements. routinely monitored following luminescence E. coli strains bearing         EPS-related phenotype characterization. Single colonies spread LB <br /> pDEW201-derived plasmids (56, 57) (Table 1) harboring promoter::luxCDABE               agar plates supplemented 0.09 0.7 M NaCl 150 &#226;?&#174;g <br /> (Photorhabdus luminescens luxCDABE bioluminescence genes) transcriptional              ml&#226;&#171;&#186;1 Congo red (16) allowed grow 24 h 37&#194;&#176;C. Plates photo- <br /> fusions 96 microtiter plates. experiments carried LB broth     graphed black background  Congo red) light screen  supplemented 0.09, 0.3, 0.5, 0.7 M NaCl. ensure similar plasmid copy        Congo red) Olympus Stylus-770 SW digital camera. mucoid appear- <br /> numbers (24), lag phase eliminated initiating experiments        ance indicated overproduction CA-EPS wca operon; red-stained cells <br /> growing culture. single colony grown OD600 0.5 LB broth              indicated presence EPS driven yjb operon. <br /> supplemented appropriate antibiotic 37&#194;&#176;C, diluted 1:10 fresh              Light/fluorescence microscopy. microliters culture mounted medium, regrown OD600 0.2. Fifty microliters bacterial            microscopic slide Neubauer counting chamber (400-&#226;?&#174;m2 10-&#226;?&#174;m-deep <br /> culture added wells opaque, white, clear 96 micro-        Clay Adams; Becton Dickinson, NJ), covered coverslip, analyzed titer plate containing 50 &#226;?&#174;l fresh LB broth supplemented           epifluorescence microscope (Axivert 135TV; Zeiss, Germany) light mi- <br /> twice required NaCl concentration 0.09 M NaCl (basal growth medium              croscope (Eclipse N100; Nikon, Japan). Photographs acquired concentration). plate sealed transparent cover incubated       mounted Canon PowerShot A95 digital camera. 37&#194;&#176;C 300 min microtiter plate reader (Victor2; Wallac, Finland).             Total <span id='am-17' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span>carbohydrate</span> determination. Overnight colonies grown LB agar <br /> Bioluminescence (reported instrument&#226;&#8364;&#8482;s arbitrary relative light units)      plates supplemented 0.7 0.09 M NaCl collected suspended OD600 measured intervals 15 min, following 5-s shaking.          2% Na2SO4, vigorously mixed 10 min, calibrated uniform cell hours following exposure NaCl microtiter plate, OD600         concentration 1010 ml&#226;&#171;&#186;1. following analytical protocol used (65): 500 <br />  486       IONESCU BELKIN                                                                                                      APPL. ENVIRON. MICROBIOL. <br />  <br /> &#226;?&#174;l cells extracted addition 800 &#226;?&#174;l chloroform, vigorous mixing,    TABLE 3. Expression ratios obtained DNA microarray analysis phase separation centrifugation (14,000 rpm 10 min). 300-&#226;?&#174;l                   rRT-PCR gene members various EPS upper (aqueous) phase mixed 200 &#226;?&#174;l DDW 1                                production systemsa <br /> ml anthrone reagent (Sigma) stock solution (1 mg ml&#226;&#171;&#186;1 concentrated H2SO4 <br /> [95 98%, wt/vol]). mixture incubated 90&#194;&#176;C 10 min,                  Microarray analysis ratio    rRT-PCR <br /> OD630 sample determined. Total carbohydrate content calculated                Osmotic-                       RpoS <br />                                                                                     Gene                     RpoS                               <span id='am-48' about='rdfs:comment' typeof='owl:Thing'>Comment</span> trehalose (Sigma) calibration curve.                                                    stress                    dependency <br />                                                                                                           dependencyc       ratioc <br />    Characterization growth capability. Single colonies grown LB                   responseb <br /> broth supplemented appropriate antibiotics OD600 0.5, diluted <br /> 1:10 fresh LB broth supplemented final concentration 0.7   yjbE       33.2           1.7         7.1 &#226;&#171;&#190; 0.4     yjb operon <br /> 0.09 M NaCl, incubated 37&#194;&#176;C shaking (200 rpm) 6 h. OD600        yjbF        3.9           1.6         5.3 &#226;&#171;&#190; 0.4 monitored 2-h intervals. verify correlation OD measurements     yjbG        1.9           1.8         4.1 &#226;&#171;&#190; 0.4 real cell concentration, determination growth performed        yjbH        2.2           1.7         4.1 &#226;&#171;&#190; 0.3 time points direct microscopic counts (Eclipse N100; Nikon, Japan).         wza         5             2.7            ND         wca operon <br />                                                                                     wzb         3.8           1.7            ND <br />                                                                                     wzc         4.4           2             5 &#226;&#171;&#190; 0.3 <br />                                                                                     wcaA        3.1           2.1            ND <br />                                  RESULTS                                            wcaB        9             2.7            ND <br />                                                                                     wcaC        8.8           1.9            ND <br />    Genes wca operon induced osmotic stress                        wcaD       27             2.6         4.4 &#226;&#171;&#190; 0.7 repressed RpoS. previously reported (24, 44),                         wcaG        4             1.8            ND <br /> promoter regulating induction yjbEFGH operon                          gmd         5.1           1.6         7.9 &#226;&#171;&#190; 0.3 <br /> activated osmotic stress repressed RpoS.                         wcaH        5.7           2.2         8.4 &#226;&#171;&#190; 0.4 <br />                                                                                     wcaI        2.6           1.7            ND <br /> attempt identify additional EPS synthesis-related genes                                                                                     manC       18             1.4         6.1 &#226;&#171;&#190; 0.5 coregulated yjbEFGH, screened genes                         wzxC        4.3           1.8            ND share regulatory features: induction NaCl                         wcaK        4             1.5            ND overinduction rpoS::Tn10 mutant.                                          wcaM        3.2           1.6            ND <br />    purpose, MG1655 wild-type strain                            rcsA        5.9           1.9         4.1 &#226;&#171;&#190; 0.7     Transcription activator <br />                                                                                                                                           wca/yjb operons <br /> rpoS::Tn10 derivative grown LB supplemented                       osmY         6.3          0.3         0.3 &#226;&#171;&#190; 0.4     RpoS dependent 0.09 0.7 M NaCl wild-type cultures reached                                                                                       OD600 0.3. samples diluted OD                          Expression ratios wild type rpoS mutant obtained fol- <br />                                                                                     lowing growth LB supplemented 0.7 0.09 M NaCl. <br /> subjected total RNA extraction, followed cell DNA                         b <br />                                                                                         MG1655 (0.7 M NaCl)/MG1655 (0.09 M NaCl). <br />                                                                                       c <br /> microarray analysis. Sampling times chosen based                            rpoS::Tn10 (0.7 M NaCl)/MG1655 (0.7 M NaCl). ND, determined. <br /> activity yjbF&#226;&#172;&#732;::luxCDABE transcriptional fusion strain NaCl dose; 180 min, order mag- <br /> nitude higher rpoS mutant wild type                    yhaL, yhaK  Table S1 supplemental material). <br /> grown LB broth supplemented 0.7 M NaCl.                                     previously reported (18), functional EPS produc- <br />    members yjb operon, yjbE, yjbF, yjbH, dis-                      tion operon, pgaABCD, induced osmotic stress. played osmotic-stress response ratios 33 (result low                   experiment, members displayed osmotic-stress response <br /> background), 3.9, 2.2 (Table 3). previous reports                 ratios 1.6 6.1 exhibited positive negative <br /> DNA microarray studies investigated response E.                    dependency RpoS. <br /> coli osmotic shock (e.g., reference 60), members yjb                         apparently similar induction characteristics (osmotic <br /> operon listed osmotically regulated                         stress RpoS dependency) yjb wca confirmed genes. attributed lower NaCl doses (&#226;&#172;?0.5 M)                      relative real-time PCR analysis, applying sample <br /> applied studies, short duration expo-                 preparation procedure employed DNA microarray <br /> sure (&#226;&#172;?30 min). present study, based observed                         analysis. RpoS-dependent gene osmY served induction characteristics yjbF&#226;&#172;&#732;::luxCDABE transcriptional                    negative control.  shown Table 3, genes <br /> fusion, experimental conditions harsher. DNA                        tested  members yjb operon, members microarray, yjbE, yjbF, yjbH displayed RpoS dependency ra-                      wca operon, common regulator rcsA) displayed RpoS <br /> tios 1.7, 1.6, 1.7, respectively; RpoS dependency thresh-                dependency ratios ranging 4 8, confirming general <br /> old ratio set 1.5.                                                 trend obtained DNA microarray analysis. expected, <br />    774 genes demonstrated &#226;&#177;&#8211;2-fold overexpres-                            RpoS-dependent gene osmY induced NaCl sion wild type rpoS mutant                            repressed rpoS mutant. <br /> presence 0.7 M NaCl, 187 displayed &#226;&#177;&#8211;1.5-fold-                           yjb wca promoters display similar induction pat- <br /> increased level expression rpoS mutant comparison                      terns. study induction characteristics EPS level expression wild type  Table S1                  biosynthesis operons, wca yjb, cloned pro- <br /> supplemental material). selected double criterion                      moter wza, gene wca operon (50), produce 14 21 known genes wca operon, display-                       wza&#226;&#172;&#732;::luxCDABE transcriptional fusion. activity ing osmotic-shock response ratios 3.2 27 RpoS                          construct compared previously described <br /> pendency ratios 1.5 2.8 (Table 3). wca operon                         (24) yjbF&#226;&#172;&#732;::luxCDABE presence different NaCl  <br /> previously reported (49) induced osmotic stress,                   centrations, wild type rpoS mutant (Fig. <br /> negative dependence rpoS demonstrated.                             1). rRT-PCR analysis, RpoS-dependent pro- <br /> additional RcsC-dependent genes, defined previously (15),                        moter osmY (osmY&#226;&#172;&#732;::luxCDABE) (56, 57) served behaved similarly, including osmB, rcsA, ugd, ykfE, ygaC, ymgD,                     control. clearly evident panels  D) <br />  VOL. 75, 2009                                                                     EPS OVERPRODUCTION rpoS MUTANT                       487 <br />  <br />  <br />  <br />  <br />   FIG. 1. Activities yjbF&#226;&#172;&#732;::luxCDABE   B), wza&#226;&#172;&#732;::luxCDABE (C, D), osmY&#226;&#172;&#732;::luxCDABE (E, F), lon&#226;&#172;&#732;::luxCDABE (G, H) transcriptional <br /> fusions response different NaCl concentrations wild-type strain   C, E, G) rpoS::Tn10 mutant (B, D, F, H). RLU, <br /> relative light units. <br />  <br />  Fig. 1, yjb wca promoters exhibit similar patterns           moters similarly enhanced rpoS mutation induction response different NaCl concentrations,              peaked NaCl concentration (0.7 M) (Fig. 2A). wild type (Fig. 1A C) rpoS::Tn10                difference observed induction promoters <br /> mutant (B D); furthermore, activation pro-                 intensity; bioluminescence exhibited wza&#226;&#172;&#732; <br />  488     IONESCU BELKIN                                                                                    APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                         ducted temperatures, 27&#194;&#176;C 37&#194;&#176;C, previ- <br />                                                                         ously shown (49) wca operon active                                                                         lower temperature.  induction promoters <br />                                                                         exhibited similar patterns different intensities.                                                                         completely inhibited rcsC rcsB mutants, <br />                                                                         displayed higher activity lower temperature <br />                                                                         wild type. rcsA mutant, inhibition activity <br />                                                                         temperature dependent: significantly inhibited (74- <br />                                                                         fold wca 49-fold yjb) 37&#194;&#176;C moderately <br />                                                                         reduced (4.2-fold wca 2.8-fold yjb) 27&#194;&#176;C. <br />                                                                            Overproduction EPS rpoS mutants. CA-EPS produc- <br />                                                                         tion (mucoid appearance) unnoticeable wild-type colonies <br />                                                                         grown 37&#194;&#176;C LB agar plates supplemented 0.09                                                                         0.7 M NaCl (Fig. 3A);  0.7 M NaCl, rpoS::Tn10 <br />                                                                         mutant produced moderate mucosity. Inactivation wca <br />                                                                         promoter rpoS mutation (&#226;&#338;&#172;Pwza rpoS::Tn10) appeared <br />                                                                         limit growth somewhat clearly abolished mucosity (Fig. <br />                                                                         3A). parallel manner, true yjb-depen- <br />                                                                         dent EPS (Fig. 3B) characterized Congo red staining:                                                                         observed wild type apparent rpoS mutant. <br />                                                                         Inactivating yjb operon rpoS mutant (&#226;&#338;&#172;yjbEFGH <br />                                                                         rpoS::Tn10) suppressed overproduction EPS stained <br />                                                                         Congo red  addition, gave rise extreme mucosity. <br />                                                                         carbohydrate contents cultures (Table 4) support <br />                                                                         visual observation. carbohydrate concentrations                                                                         strains higher strains grown LB agar plates <br />                                                                         supplemented 0.7 M NaCl. increase carbohydrate                                                                         wild-type strain enhanced activity                                                                         EPS production systems known induced osmotic stress: <br />                                                                         PGA, yjb-dependent EPS, CA-EPS. rpoS mutant,                                                                         expected overproduced; data                                                                         Table 4 roughly estimated mixture dominated <br />                                                                         CA-EPS. <br />   FIG. 2.   NaCl dose dependency yjbF&#226;&#172;&#732;::luxCDABE                  &#226;&#338;&#172;yjb rpoS::Tn10 double mutants filamentous.  <br /> wza&#226;&#172;&#732;::luxCDABE induction wild-type strain rpoS::Tn10     letion yjbEFGH, rpoS::Tn10 strain lost red <br /> mutant. (B) Induction yjbF&#226;&#172;&#732;::luxCDABE wza&#226;&#172;&#732;::luxCDABE tran-       staining described extremely mucoid <br /> scriptional fusions response 0.7 M NaCl rcsA, rcsB, rcsC   (Fig. 3A B). addition, cells elongated filaments <br /> mutants ancestral wild type 27&#194;&#176;C 37&#194;&#176;C. Error bars standard deviations. Max, maximum; RLU, relative light units. <br />                                                                         200 &#226;?&#174;m long (Fig. 4A). phenotypes longer <br />                                                                         dependent osmotic stress observed                                                                         growth regular LB agar plates (0.09 M NaCl). better <br />                                                                         visualize phenomenon, cells transformed fusion approximately fivefold higher wild type          plasmid containing green fluorescent protein gene (pES2, rpoS mutant. true rRT-PCR                  recA&#226;&#172;&#732;::gfpUV) (45). Furthermore, &#226;&#338;&#172;yjbE rpoS::Tn10 double <br /> analysis; averaged RQ values  Materials Methods)            mutant exhibit identical colony  shown) wca gene expression compared expression yjb               cellular (Fig. 4B) appearance. phenotypes negated <br /> genes 4.7 &#226;&#171;&#190; 2.2 5.1 &#226;&#171;&#190; 1.1 wild type            strain transformed plasmid pDEW/yjbF, <br /> rpoS mutant, respectively. expected, osmY promoter               harbors active yjbE gene, confirming nonpolar <br /> exhibited strong positive response osmotic stress,         deletion yjbE. cellular aspect complemented <br /> activity strongly inhibited rpoS::Tn10 mutant (Fig.          phenotype shown Fig. 4B. <br /> 1E F). RpoS-independent gene promoter (lon,                      Activity yjb wca promoters enhanced RpoH-dependent gene) exhibited similar activity patterns            &#226;&#338;&#172;yjbEFGH rpoS::Tn10 mutant. explain NaCl-indepen- <br /> intensities strains (Fig. 1G H).                            dent mucoid colony appearance exhibited &#226;&#338;&#172;yjbEFGH <br />    yjb operon listed (14) genes controlled             rpoS::Tn10 mutant, activities promoters (yjb Rcs phosphorelay RcsA-dependent manner                 wca) assayed wild type mutants, <br /> (16). authors pointed sequence highly          &#226;&#338;&#172;yjbEFGH, rpoS::Tn10, &#226;&#338;&#172;yjbEFGH rpoS::Tn10.  <br /> similar RcsAB box (62) located yjbE promoter           ground activity promoters rpoS mutant  <br /> region. confirmed findings assessing osmotic            sixfold higher wild-type strain, suggesting (0.7 M NaCl) induction activity yjbF&#226;&#172;&#732;::luxCDABE              molecular mechanism promotes overinductions wza&#226;&#172;&#732;::luxCDABE transcriptional fusions rcsA-, rcsB-,             osmotic-stress independent. expected,  rcsC-inactive strains (Fig. 2B). experiments           ground activity  0.09 M NaCl) promoters  VOL. 75, 2009                                                                EPS OVERPRODUCTION rpoS MUTANT                     489 <br />  <br />  <br />  <br />  <br />   FIG. 3.   Appearance culture 24 h growth LB agar plates supplemented 0.09 M 0.7 M NaCl. (B) Congo red staining cultures grown LB agar plates supplemented 0.7 M NaCl. Representative colonies shown insets. <br />  <br />  <br />  <br /> higher &#226;&#338;&#172;yjbEFGH rpoS::Tn10 double mutant rpoS::Tn10 ancestor (Fig. 5). induced 0.7 M <br /> NaCl, maximal activity similar (yjb) twice high (wza) background activity. observation ap- <br /> pears significant spite large standard deviation wza&#226;&#172;&#732;::lux background measurements. case, response double mutant does fully explain 10- <br /> fold-higher carbohydrate content (CA-EPS) strain <br /> grown 0.7 M NaCl (Table 4). <br />  <br />  <br />  <br /> TABLE 4. Carbohydrate content cells grown LB agar plates <br />      supplemented different concentrations NaCl <br />                                      Carbohydrate content &#226;&#171;&#190; SD <br />          Genotype                          (&#226;?&#174;g/109 cells) <br />  <br />                                 0.09 M NaCl            0.7 M NaCl <br />  <br /> Wild type                        3.4 &#226;&#171;&#190; 2.6            11.6 &#226;&#171;&#190; 6 <br /> &#226;&#338;&#172;Pwza                            2.7 &#226;&#171;&#190; 2.4             9.1 &#226;&#171;&#190; 4.5 <br /> &#226;&#338;&#172;yjbEFGH                         2.4 &#226;&#171;&#190; 1.4            16.1 &#226;&#171;&#190; 9.5 <br /> rpoS::Tn10                       0.9 &#226;&#171;&#190; 0.3             393 &#226;&#171;&#190; 124a,b <br /> &#226;&#338;&#172;Pwza rpoS::Tn10                 1.7 &#226;&#171;&#190; 1.1             136 &#226;&#171;&#190; 16b <br /> &#226;&#338;&#172;yjbEFGH rpoS::Tn10               79 &#226;&#171;&#190; 39a            806 &#226;&#171;&#190; 136a         FIG. 4.   Fluorescence microscopy rpoS::Tn10 &#226;&#338;&#172;yjbEFGH <br />                                                                       rpoS::Tn10 mutants harboring pES2 (recA&#226;&#172;&#732;::gfp). (B) Light microscopy <br />       Mucoid.                                                          &#226;&#338;&#172;yjbE rpoS::Tn10 harboring pDEW201 (vector  pDEW/yjbF <br />  b <br />      Congo red positive.                                              (containing yjbE upstream yjbF&#226;&#172;&#732;). <br />  490     IONESCU BELKIN                                                                                     APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />   FIG. 5. Maximal activities yjbF&#226;&#172;&#732;::luxCDABE   wza&#226;&#172;&#732;::luxCDABE (B) transcriptional fusions wild type &#226;&#338;&#172;yjbEFGH, rpoS::Tn10, &#226;&#338;&#172;yjbEFGH rpoS::Tn10 mutants. Error bars standard deviations. Max, maximum; RLU, relative light units. <br />  <br />                                                                           FIG. 6. Growth LB medium supplemented 0.09 M      Inactivation yjb wca impaired growth             0.7 M (B) NaCl. strains listed panel  Results shown                                                                        averaged curves results repeat experiments. <br /> rpoS-deficient strain. growth capabilities strains tested LB broth supplemented 0.09 M (Fig. 6A) 0.7 M (Fig. 6B) NaCl. wild-type strain &#226;&#338;&#172;Pwza &#226;&#338;&#172;yjbEFGH mutants exhibited differences             detected laboratory stocks enteric bacteria growth characteristics, rpoS mutant grew slowly        natural isolates. fully partially inactive rpoS alleles saline medium. derived double mutants, &#226;&#338;&#172;Pwza rpoS::Tn10        common, hypothesized aspects &#226;&#338;&#172;yjbEFGH rpoS::Tn10, exhibited slower growth: &#226;&#338;&#172;Pwza           low-RpoS RpoS-free physiology evolved compen- <br /> affected growth presence 0.7 M NaCl,          sate loss physiological functions normally <br /> effect &#226;&#338;&#172;yjbEFGH observed low NaCl medium.           RpoS dependent. effect osmotic stress &#226;&#338;&#172;Pwza rpoS::Tn10 growth           RpoS shown important player E. coli&#226;&#8364;&#8482;s <br /> shown Fig. 3B, mutant appears reduced           osmotic-stress response (22, 61). activates transcription <br /> comparison strains plate.                          large number genes provide osmoprotection,    Direct microscopic counts indicated measured               cross-protection stress factors. <br /> OD600 values displayed Fig. 6 valid comparisons             synthesis uptake compatible solutes, tested strains: OD600 1 represents 2.8 &#226;&#171;&#187; 108         important responses cells osmotic stress, partially <br /> (rpoS::Tn10), 2.6 &#226;&#171;&#187; 108 (&#226;&#338;&#172;Pwza rpoS::Tn10), 2.2 &#226;&#171;&#187; 108              mediated RpoS-dependent genes. include otsA, otsB, <br /> (&#226;&#338;&#172;yjbEFGH rpoS::Tn10) cells ml&#226;&#171;&#186;1. filamentous pheno-               treA (21), involved trehalose synthesis metabolism. <br /> type described (Fig. 4) started emerge              growth rpoS mutants hyperosmotic <br /> &#226;&#338;&#172;yjbEFGH rpoS::Tn10 double-mutant culture entry              medium, slower, demonstrated attrib- stationary phase unnoticeable             uted function organic compatible solutes trans- <br /> course experiment.                                              port systems ProP ProU, high-affinity K&#226;&#171;&#185; <br />                                                                        transport KdpFABC BetTIBA im- <br />                                                                        plicated choline metabolism (9, 63). systems <br />                           DISCUSSION <br />                                                                        expressed RpoS-independent manner response <br />   Natural selection loss RpoS activity shown        osmotic stress. data clear rpoS-deficient <br /> occur course nutrient-limited growth            strains overproduce EPS response osmotic stress  VOL. 75, 2009                                                                  EPS OVERPRODUCTION rpoS MUTANT                                  491 <br />  <br />  deletion yjb wca operons attenuates im-          example mechanism evolved allow cells paired osmotolerance mutant.                                    rpoS deficient grow cope osmotic upshifts <br />    CA-EPS reported endow cells                enjoying better nutrient-scavenging capability stress protection, including extreme osmotic              overexpression RpoD-dependent uptake nutrient utiliza- <br /> stress E. coli O157:H7 (6, 34, 35). demonstrated        tion systems. <br /> (6) CA-EPS-deficient strain lost viability faster                                                                                                         REFERENCES <br /> wild-type strain course 2-day exposure 1.5 M                                                                         1. Bhagwat,  , J. Tan, M. Sharma, M. Kothary, S. Low, B. D. Tall, M. NaCl. demonstrated CA-EPS overpro-                   Bhagwat. 2006. Functional heterogeneity RpoS stress tolerance entero- <br /> duced rpoS mutant response osmotic stress             hemorrhagic Escherichia coli strains. Appl. Environ. Microbiol. 72:4978&#226;&#8364;&#8220; deficiency limited growth saline medium                4986. <br />                                                                         2. Bi, E. F., J. Lutkenhaus. 1991. FtsZ ring structure associated rpoS-deficient strain. clinical isolates enteric bacteria          division Escherichia coli. Nature 354:161&#226;&#8364;&#8220;164. mucoid CA-EPS production (20).                  3. Carballes, F., C. Bertrand, J. P. Bouche, K. Cam. 1999. Regulation Furthermore, Escherichia coli O157:H7 isolates                Escherichia coli cell division genes ftsA ftsZ component <br />                                                                            rcsC-rcsB. Mol. Microbiol. 34:442&#226;&#8364;&#8220;450. <br /> ported mucoid growth medium containing               4. Charette, M., G. W. Henderson,  Markovitz. 1981. <span id='am-29' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span> hydrolysis- <br /> high salt concentrations (26). view increasing reports           dependent activity lon(capR) protein E. coli K12. Proc. Natl. Acad. <br />                                                                            Sci. USA 78:4728&#226;&#8364;&#8220;4732. prevalence mutated rpoS genotypes similar               5. Chen, G., C. L. Patten, H. E. Schellhorn. 2004. Positive selection loss <br /> isolates, speculated phenomena, mu-               RpoS function Escherichia coli. Mutat. Res. 455:193&#226;&#8364;&#8220;203. <br /> cosity loss rpoS functionality, coupled,         6. Chen, J., S. M. Lee, Y. Mao. 2004. Protective effect exopolysaccharide <br />                                                                            colanic acid Escherichia coli osmotic oxidative stress. Int. J. Food mucoid appearance consequence rpoS mutation.               Microbiol. 93:281&#226;&#8364;&#8220;286. <br />    studies reported varied osmotolerance clin-          7. Chung, C. H.,  L. Goldberg. 1981. product lon (capR) gene <br /> ical E. coli isolates (29). shown (9) E. coli          Escherichia coli ATP-dependent protease, protease La. Proc. Natl. <br />                                                                            Acad. Sci. USA 78:4931&#226;&#8364;&#8220;4935. <br /> CFT073 lacking rpoS, proP, proU simultaneously               8. Conter, , R. Sturny, C. Gutierrez, K. Cam. 2002. RcsCB Asp <br /> limited growth virulence high-osmolality human urine,             phosphorelay essential overcome chlorpromazine-induced stress <br />                                                                            Escherichia coli. J. Bacteriol. 184:2850&#226;&#8364;&#8220;2853. <br /> suggesting possesses additional osmoregulatory systems.         9. Culham, D. E.,  Lu, M. Jishage, K.  Krogfelt,  Ishihama, J. M. interesting determine deletion yjb           Wood. 2001. osmotic stress response virulence pyelonephritis <br /> wca affect CFT073&#226;&#8364;&#8482;s growth ability virulence.              isolates Escherichia coli: contributions RpoS, ProP, ProU                                                                            systems. Microbiology 147:1657&#226;&#8364;&#8220;1670. <br />    Overproduction CA-EPS results mucoid colony            10. Danese, P. N., L.  Pratt, R. Kolter. 2000. Exopolysaccharide produc- <br /> appearance reported emerge response                  tion required development Escherichia coli K-12 biofilm architec- <br /> inactivation lon (36), treatment resulted UV             ture. J. Bacteriol. 182:3593&#226;&#8364;&#8220;3596. <br />                                                                        11. Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo- <br /> sensitivity cell filamentation UV treatment. Lon           somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> ATP-dependent protease (4, 7)  addition role             Sci. USA 97:6640&#226;&#8364;&#8220;6645. <br />                                                                        12. Dhar,  K., M. M. Roux, K. R. Klimpel. 2001. Detection quanti- degradation misfolded proteins following heat shock,                fication infectious hypodermal hematopoietic necrosis virus white regulatory function (55). cell elongation mu-               spot virus shrimp using real-time quantitative PCR SYBR green <br /> coid colony phenotype linked lon-specific target pro-              chemistry. J. Clin. Microbiol. 39:2835&#226;&#8364;&#8220;2845. <br />                                                                        13. Farewell, , K. Kvint, T. Nystrom. 1998. Negative regulation RpoS: <br /> teins RcsA (54) SulA (37). RcsA unstable auxiliary              case sigma factor competition. Mol. Microbiol. 29:1039&#226;&#8364;&#8220;1051. <br /> DNA binding protein Rcs (33, 54). SulA cell         14. Ferenci, T. 2008. spread beneficial mutation experimental bac- <br /> division inhibitor induced SOS DNA repair                 terial population: influence environment genotype                                                                            fixation rpoS mutations. Heredity 100:446&#226;&#8364;&#8220;452. <br /> response (47). binds FtsZ, key protein cell division        15. Ferrie`res, L., D. J. Clarke. 2003. RcsC sensor kinase required responsible septum formation (2, 23),                  normal biofilm formation Escherichia coli K-12 controls expres- <br />                                                                            sion regulon response growth solid surface. Mol. Microbiol. <br /> presses cell division (38).  likely yjb rpoS       50:1665&#226;&#8364;&#8220;1682. <br /> mutants, phenotypes mediated Lon protease.             16. Ferrie`res, L., S. N. Aslam, R. M. Cooper, D. J. Clarke. 2007. sequenced lon allele double mutant,            yjbEFGH locus Escherichia coli K-12 operon encoding proteins <br />                                                                            involved exopolysaccharide production. Microbiology 153:1070&#226;&#8364;&#8220;1080. assayed lon&#226;&#172;&#732;::lux activity  shown), did alter-     17. Francez-Charlot, , B. Laugel,  Van Gemert, N. Dubarry, F. Wiorowski, <br /> ation sequence reduced induction.                                    M. P. Castanie&#194;&#180;-Cornet, C. Gutierrez, K. Cam. 2003. RcsCDB Asp <br />    results indicate group genes            phosphorelay negatively regulates flhDC operon Escherichia <br />                                                                            coli. Mol. Microbiol. 49:823&#226;&#8364;&#8220;832. greater importance stress resistance               18. Goller, C., X. Wang, Y. Itoh, T. Romeo. 2006. cation-responsive <br /> growth rpoS mutant wild type. present             protein NhaR Escherichia coli activates pgaABCD transcription, required <br />                                                                            production biofilm adhesion poly-&#226;?&#164;-1,6-N-acetyl-<span id='am-24' about='obo:IMR_0200485' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:id" content="IMR:0200485" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00329" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="D-Glucosamine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>D-glucosamine</span>. J. <br /> communication, demonstrated EPS produc-                 Bacteriol. 188:8022&#226;&#8364;&#8220;8032. <br /> tion operons, wca yjb. operons similarly             19. Gottesman, S., P. Trisler,  Torres-Cabassa. 1985. Regulation cap- <br /> regulated, expression significantly enhanced           sular polysaccharide synthesis Escherichia coli K-12: characterization                                                                            regulatory genes. J. Bacteriol. 162:1111&#226;&#8364;&#8220;1119. <br /> rpoS mutant. stress resistance capabilities endowed          20. Grant, W. D.,  W. Sutherland, J. F. Wilkinson. 1969. Exopolysaccha- EPS overproduction unclear (6), results clearly            ride colanic acid occurrence Enterobacteriaceae. J. Bacteriol. growth mutants impaired EPS production                   100:1187&#226;&#8364;&#8220;1193. <br />                                                                        21. Hengge-Aronis, R., W. Klein, R. Lange, M. Rimmele, W. Boos. 1991. <br /> inhibited saline  wca yjb) salt-free (yjb)             Trehalose synthesis genes controlled putative sigma factor  <br /> medium.                                                                    coded rpoS involved stationary-phase thermotolerance                                                                            Escherichia coli. J. Bacteriol. 173:7918&#226;&#8364;&#8220;7924. <br />    tempting hypothesize operons,        22. Hengge-Aronis, R., R. Lange, N. Henneberg, D. Fischer. 1993. Osmotic stress-responsive RpoS-independent systems, evolved             regulation rpoS-dependent genes Escherichia coli. J. Bacteriol. 175:259&#226;&#8364;&#8220; order cope prevalence low levels RpoS rpoS         265. <br />                                                                        23. Higashitani, , N. Higashitani, K. Horiuchi. 1995. cell division <br /> mutations necessitated need im-              inhibitor SulA Escherichia coli directly interacts FtsZ <span id='am-9' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span>GTP</span> <br /> proved nutrient scavenging (14). EPS overproduction              hydrolysis. Biochem. Biophys. Res. Commun. 209:198&#226;&#8364;&#8220;204. <br />  492       IONESCU BELKIN                                                                                                              APPL. ENVIRON. MICROBIOL. <br />  <br /> 24. Ionescu, M.,  Franchini, T. Egli, S. Belkin. 2008. Induction                 Ulber, O. Lev, T. Scheper, S. Belkin. 2003. Fluorescence biolumi- <br />     yjbEFGH operon regulated growth rate oxygen concentration.                      nescence reporter functions genetically modified bacterial sensor strains. <br />     Arch. Microbiol. 189:219&#226;&#8364;&#8220;226.                                                             Sens. Actuators 90:2&#226;&#8364;&#8220;8. <br /> 25. Jishage, M.,  Ishihama. 1997. Variation RNA polymerase sigma               46.   Samuel, S., P. Reeves. 2003. Biosynthesis O-antigens: genes     subunit composition different stocks Escherichia coli W3110. J.                 pathways involved <span id='am-40' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-41' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-43' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> sugar precursor synthesis O-antigen <br />     Bacteriol. 179:959&#226;&#8364;&#8220;963.                                                                   assembly. Carbohydr. Res. 338:2503&#226;&#8364;&#8220;2519. <br /> 26. Junkins,  D., M. P. Doyle. 1992. Demonstration exopolysaccharide           47.   Schoemaker, J. M., R. C. Gayada,  Markovitz. 1984. Regulation cell <br />     production enterohemorrhagic Escherichia coli. Curr. Microbiol. 25:9&#226;&#8364;&#8220;17.               division Escherichia coli: SOS induction cellular location sulA <br /> 27. Kelm, O., C. Kiecker, K. Geider, F. Bernhard. 1997. Interaction                protein, key lon-associated filamentation death. J. Bacteriol. 158: <br />     regulator proteins RcsA RcsB promoter operon                      551&#226;&#8364;&#8220;561. <br />     amylovoran biosynthesis Erwinia amylovora. Mol. Gen. Genet. 256:72&#226;&#8364;&#8220;83.           48.   Silhavy, T. J., M. L. Berman, L. W. Enquist. 1984. Experiments 28. King, T., S. Seeto, T. Ferenci. 2006. Genotyping environment inter-                gene fusions. Cold Spring Harbor Laboratory Press, Cold Spring Harbor, <br />     action influencing emergence rpoS mutations Escherichia coli pop-               NY. <br />     ulation. Genetics 172:2071&#226;&#8364;&#8220;2079.                                                    49.   Sledjeski, D., S. Gottesman. 1996. Osmotic shock induction capsular <br /> 29. Kunin, C. M., T. H. Hua, L. Van Arsdale White, M. Villarejo. 1992.                    synthesis Escherichia coli K <br /> </body></html>