<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Apr. 1990, p. 2055-2064                                                                        Vol. 172,  4 <br /> 0021-9193/90/042055-10$02.00/0 <br /> Copyright C 1990, American Society Microbiology <br />  <br />  <br />  Identification Characterization New Escherichia coli Gene <br />              Dosage-Dependent Suppressor                        dnaK Deletion Mutation <br />                                     PIL JUNG KANG ELIZABETH  CRAIG* <br />             Department Physiological Chemistry, University Wisconsin-Madison, Madison, Wisconsin 53706 <br />                                          Received 4 October 1989/Accepted 10 January 1990 <br />  <br />             report isolation characterization previously unidentified Eschenichia coli gene           suppresses temperature-sensitive growth filamentation dnaK deletion mutant strain. Introduction <br />           multicopy plasmid carrying wild-type gene dnaK deletion mutant strain rescued           temperature-sensitive growth dnaK deletion mutant strain 40.5&#194;&#176;C filamentation, fully 37&#194;&#176;C <br />           partially 40.5&#194;&#176;C.  inability dnaK mutant cells support bacteriophage growth           suppressed. gene able suppress temperature-sensitive growth grpE280 mutant strain <br />           41&#194;&#176;C. Filamentation grpE280 mutant strain suppressed 37&#194;&#176;C 41&#194;&#176;C. dnaK <br />           suppressor gene, designated dksA, maps near mrcB gene (3.7 min E. coli chromosome). DNA <br />           sequence analysis vivo experiments showed dksA encodes 17,500-Mr polypeptide. Gene disruption <br />           experiments indicated dksA essential gene. <br />  <br />   heat shock response highly               revealed synthesis DNA RNA rapidly shut <br /> served biological responses environmental changes (19,             (15) cell division blocked (10, 32) 20, 24). organisms far examined, including               nonpermissive temperature. Recent isolation new dnaK <br /> bacterial, plant, animal species, induce synthesis         allele, dnaKI I, showed dnaKI I unable initiate proteins encoded HSP60, HSP70, HSP90 gene                  new round DNA replication nonpermissive temper- <br /> families response elevated temperatures.         ature termination round progress (28). DnaK proteins induced heat induced variety             known involved directly indirectly stresses. heat shock proteins               phosphorylation aminoacyl-tRNA synthetases (34). highly conserved proteins known. DNA sequencing                  Furthermore, DnaK negative modulator heat shock <br /> studies shown <span id='am-33' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-34' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> sequence Esche-             response. Cells carrying dnaK756 mutation showed high <br /> richia coli groEL product 54% identical yeast hsp60         level synthesis heat shock proteins 30&#194;&#176;C protein (25). E. coli DnaK 50% identical              failed turn heat shock response 42&#194;&#176;C (31). <br /> eucaryotic hsp70 proteins (5), E. coli C62.5 heat                study cellular function DnaK protein, shock protein 41% identical hsp83 protein                constructed new dnaK deletion mutant strain. Phenotypic <br /> Drosophila melanogaster (4). universality heat             analysis strain showed deletion mutant shock response high degree conservation heat            temperature cold sensitive growth highly fila- <br /> shock proteins imply proteins fundamental,            mentous permissive temperature. way common roles various organisms.                                    study cellular function DnaK protein, isolated    eucaryotic organisms contain family HSP70-               analyzed suppressors overcame dnaK deletion mu- <br /> related genes. example, Saccharomyces cerevisiae              tant phenotypes. paper, report isolation genome contains genes related               newly identified gene, dksA, suppresses tempera- <br /> HSP70 higher eucaryotes (19).  dnaK gene            ture-sensitive growth filamentation dnaK deletion HSP70-like gene E. coli (5). dnaK gene forms          mutant strain overexpressed. dksA gene operon heat shock gene, dnaJ,                cloned sequenced, location E. coli <br /> structure promoter-dnaK-dnaJ.                                         chromosome determined. <br />   dnaK gene originally identified essential bacteriophage X growth (14, 27). Genetic                                MATERIALS METHODS <br /> biochemical studies suggest DnaK protein required                    Bacterial strains, phages, media. bacterial strains initiation X DNA replication facilitating               phages used genotypes listed Table 1. <br /> dissociation P protein K DNA replication                    LB broth M9 minimal medium prepared  <br /> initiation complex, allowing DnaB protein  helicase)                 scribed Miller (23). necessary, antibiotics function (12). DnaK, 69,000-Mr protein, exten- <br /> sively characterized HSP70 heat shock protein. Purified                   supplemented following final concentrations: ampicil- <br /> DnaK, binds <span id='am-14' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> tightly, weak DNA-indepen-                    lin, 100 ,ug/ml; tetracycline, 20 ,ug/ml; kanamycin, 100 <br /> dent ATPase  generally, 5&apos;-nucleotidase) activity (6,                ,ug/ml. Amino acids supplemented 40 ,ug/ml. <br /> 37). purified protein autophosphorylates                  Plasmids. Plasmids obtained study briefly  <br /> threonine residues (37). DnaK biochemically                   scribed Table 2, details given text. characterized, little known cellular function.             Genetic manipulations. Linear transformation DNA vivo analysis temperature-sensitive dnaK mutants                    recB recC sbcB strain performed described                                                                           Winans et al. (35). Selections k lysogens P1 trans- <br />                                                                           ductions described Silhavy et al. (30),   * <br />       Corresponding author.                                               transductions dnaK deletion allele. Growth phage <br />                                                                    2055 <br />  2056       KANG CRAIG                                                                                                 J. BACTERIOL. <br />  <br />              TABLE 1. Bacterial phage strains                               TABLE 2. Plasmids obtained study <br />  <br />   Strain                   Genotype <br />                                                        Source                                                        reference      Plasmid                   Description                   Suppressing <br />                                                                                                                                  ability <br /> JC7623      AB1157 recB21 recC22 sbcBJS sbcC201 21                    pJK53     Originally isolated suppressor clone              + <br /> MG1655      F-                                        CGSCa           pJK72     Originally isolated suppressor clone              + <br /> DS410       F- ara Azir tonA lacY minA minB xyl       C. Gross&apos;       pJK711    Originally isolated suppressor clone              + <br />               mtl thi malA strA                                       pJK530    Clal-digested pJK53, religated (large frag-       - <br /> JM109       recAl endAl gyrA96 thi hsdR17 supE44 36                               ment) <br />               relAl      lac-proAB)(F&apos; traD36                       pJK533    1.2-kb PvuII fragment pf pJK53 cloned <br />               proAB lacIqZAM]5)                                                   EcoRV, PvuII sites pBR322 <br /> CAG5051     HfrH, nad::TnlO thi-1 spoT supQ80         C. Gross        pJK537    1.8-kb PvuII fragment pf pJK53 cloned             + <br /> CAG5052     KL227, btu::TnlO metBi relA               C. Gross                    EcoRV, PvuII sites pBR322 <br /> CAG5053     KL208, zbc::TnlO relA                     C. Gross        pJK538    1.7-kb PvuII-EcoRV fragment pJK53               + <br /> CAG5054     KL96, trp: :TnlO thi-J relA               C. Gross                    cloned EcoRV, PvuII sites CAG5055     KL16, zed::TnlO thi-l relA                C. Gross                     pBR322 <br /> CAG8160     KL14, leu thi-l relA rpoD+-TnlO           C. Gross        pJK539    ClaI site pBR322 pJK538                + <br /> CAG8209     KL228, leu thi-l gal lacY supE44          C. Gross                     inactivated Klenow fragment <br />               rpoD+-TnlO                                              pJK540    pJK539 cut ClaI, inactivated CAG18436    MG1655, zae-502::TnJO                     C. Gross                     Klenow fragment, religated <br /> CAG12025    MG1655, zad-220::TnlO                     C. Gross        pJK541    Clal-digested pJK538, religated (large <br /> CAG9253     C600, X901 F&apos; X 90, grpE280 Tetr          C. Gross                     fragment) <br /> MF6         thr-l dapD4 4 rpsL136 leuB6 eda       9               pJK542    176-bp PvuII-BclI deletion pJK538 <br />                thi-l lacYl Agal-1 ara-14 xyl-1S mtl-l                 pJK543    NdeI site pJK538 inactivated                tonA supE                                                           Klenow fragment <br /> MF6R        MF6, rpoB                                 9               pJK333    pJK533 X17-181 <br /> RX800       MF6, lpxA                                 9               pJK334    pJK333 opposite orientation <br /> JCR20       MF6R, firA                                9               pJK337    pJK537 X17-181                 + <br /> DC1         thi-1 4 rpsL136 leuB6 eda thr-J       9               pJK21     408-bp ClaI-PvuII fragment pJK538 <br />                lacYl lpxBI pgsA Agal-1 ara-14 xyl-                                 replaced Kanr cassette                15 mtl-l tonA supE                                                  pUC4K <br /> PK5         MG1655, grpE280 (CAG9253 x                study      pJK15     DnaJ-expressing plasmid (dnaJ fused                MG1655, selection Tetr, screening                               dnaK promoter) <br />                X&apos;) <br /> PK9         MG1655, zae-502::TniO dksA::kan           study <br />                (CAG18436 x PK201, selection                       compare mutant phenotypes DnaK- DnaJ-                Kanr Tet)                                              DnaJ, dnaJ deletion constructed. dnaJ <br /> PK101       MG1655, dnaKi4 dnaJI4                                     deletion mutant strain, PK102, carries deletion PK102       MG1655, dnaJi5 <br /> PK113       Spontaneous Ts&apos; revertant PK101                        majority dnaJ coding sequence, retaining PK201       MG1655, dksA::kan                                         sequences encoding 23 30 amino acids PK202       MG1655, dksA::kan dnaKI4 dnaJ14                           DnaJ  named allele dnaJJ5). details imm2&apos;                                                H. Uchidac     construction dnaK dnaJ deletion mutant strains <br />   dnaK+                                                               described  Plasmid pJK21,   dnaJ+                                                               carries deletion 408-bp ClaI-PvuII fragment X imm2&apos;                                               H. Uchida       pJK538, constructed replacing region   dnaK+                                                               kanamycin resistance (Kanr) cassette pUC4K (33) X17-181                                               F. Blattnerb    make dksA deletion mutant strain. <br />   CGSC, E. coli Genetic Stock Center, Yale University, New Haven,      Construction genomic library isolation suppres- <br /> Conn. <br />   b University Wisconsin-Madison. <br />                                                                       sor clone. multicopy genomic library constructed     University Tokyo, Tokyo, Japan.                                cloning partial Sau3AI digestion chromosomal DNA <br />                                                                       spontaneous temperature-resistant suppressor strain, <br />                                                                       PK113, unique BamHI site pBR322. DNA <br />                                                                       used transform dnaK deletion strain, PK101, P1 dnaK deletion strain poor; P1                  transformants selected 42&#194;&#176;C LB-ampicillin <br /> transduction efficiency extremely low. increase            plates. Plasmid DNA isolated temperature- <br /> P1 transduction efficiency, P1 lysates prepared X           resistant clones used transform PK101 second time <br /> imm21dnaK&apos; dnaJ+ lysogen strain PK101, lysates             confirm suppression temperature-sensitive growth treated Lambda Sorb (Promega Biotec) remove              PK101. <br /> X phages P1 lysate. Transduction efficiency                   Mapping. map dksA gene E. coli chromo- <br /> increased 1,000-fold way.                            Kanr cassette pUC4K inserted    Construction dnaK, dnaJ, dksA deletion mutant           EcoRV sites pJK53. Kanr marker intro- <br /> strains. basic strategy construction chromo-             duced chromosome linear transformation somal deletion construct deletion mutations dnaK,           mapped 10-min interval set Hfr strains <br /> dnaJ, dksA recombinant plasmids insert             origins distributed chromosome (ob- mutations E. coli chromosome linear                 tained C. Gross, University Wisconsin-Madison). <br /> transformation. dnaK deletion mutant strain, PK101,               Kanr marker tested P1 cotransduction carries deletion entire dnaK coding sequence             mapped TnJO insertions region described 155 base pairs (bp) 5&apos; end dnaJ gene.        Results. Subsequent transduction analysis performed dnaJ gene deleted, resulting construction         using nearby selectable markers. Fine mapping DnaK- DnaJ-  named allele dnaKJ4 dnaJJ4).               Southern hybridization, using 1.2-kilobase-pair (kb) <br />  VOL. 172, 1990                                               CHARACTERIZATION dnaK SUPPRESSOR GENE                                        2057 <br />  <br /> PvuII fragment pJK53 probe DNA                  TABLE 3. Plating efficiency wild type (MG1655) covering region (X17-181, obtained F. Blattner,            dnaK deletion strain (PK101) carrying dnaK suppressor <br /> University Wisconsin-Madison).                                        clone (pJK537) control plasmid (pJK333) <br />    Recombinant DNA techniques. procedures compiled                                                                  Plating efficiencya Maniatis et al. (22) generally used minor                   Strain <br />                                                                                                                37&#194;&#176;C                           40.50C <br /> modifications. Isolation DNA carried using <br /> Lambda Sorb (Promega Biotec).                                      Wild type                                   99.4                           96.2 <br />    Si nuclease mapping. identify start site dksA         PK101(pJK537)                               95.6                           91.3 <br /> RNA transcript, Si nuclease mapping previ-             PK101(pJK333)                               53.7                            0.08 <br /> ously described (13). 5&apos;-end-labeled probe prepared                                                                                 strains considered   100%o plating efficiency 30&#194;&#176;C. pJK539 cleaving PvuII, followed alkaline <br /> phosphatase kinase treatment. appropriate labeled <br /> DNA probe digested HincII purified agarose gel. Total RNA isolated mid-log-phase           identify suppressors dnaK deletion, spontaneously <br /> culture strain MG1655 carrying pJK537 described           arising mutants PK101 able grow 42&#194;&#176;C isolated. <br /> Aiba et al. (1). Si digestion reaction, 50 F.g total   spontaneous reversion frequency 10-&apos;. RNA 100 pg labeled probe hybridized 55&#194;&#176;C,           strains, PK113, chosen analysis. 200 U Si nuclease treated. reaction mixture              Isolation temperature-resistant suppressor clone. containing 30 ,ug yeast tRNA included control      began experiments designed isolate suppressor  assay. DNA fragments electrophoresed  6%            clone, expected isolate mutant gene polyacrylamide gel containing 7 M <span id='am-51' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-52' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span>             suppress temperature-sensitive growth dnaK <br /> 32P-end-labeled HaeIII digest pBR322 DNA marker             deletion strain.  later results indicated fragments.                                                         isolated suppressor clone described contain&apos;ed wild- <br />    DNA sequencing. DNA sequences determined            type gene, mutant gene, suggesting overexpression <br />  dideoxy-chain termination method (29), using Sequenase            wild-type gene suppressed temperature-sensitive <br />  (U.S. Biochemical Corp.) strain JM109 host        growth dnaK deletion strain PK101. <br />  M13mpl8 M13mpl9 derivatives (36).                                 multicopy genomic library constructed cloning <br />     Analysis plasmid-encoded polypeptides. Polypeptides         partial Sau3AI digest chromosomal DNA  encoded plasmids analyzed using minicells (11).        spontaneous temperature-resistant suppressor strain, <br />  Minicells prepared cells carrying plasmids        PK113, unique BamHI site pBR322. temper- <br />  tested labeled L-[355]methionine (1,114 Ci/mmol; 30      ature-sensitive dnaK deletion strain, PK101, transformed <br />  ,uCi/ml, final concentration; Dupont, NEN Research Prod-          genomic library DNA, temperature-resistant <br />  ucts). Labeled protein samples analyzed sodium            transformants selected 42&#194;&#176;C LB-ampicillin plates. <br />  dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-          Plasmid DNA isolated temperature-resistant <br />  PAGE) (15% acrylamide, 0. 195% bisacrylamide); gel        transformants used transform strain PK101  stained Coomassie blue treated 1 M              second time confirm suppression temperature- <br />  sodium salicylate. gel dried fluorographed.       sensitive growth. second screening, indepen- <br />     Plating efficiency. Cells grown overnight 30&#194;&#176;C          dent clones isolated. Restriction endonuclease analysis <br />  diluted 1:100 LB broth grown optical density        showed identical clones  600 nm 0.5. Appropriate dilutions LB plates          clones overlapping DNA segments. <br />  incubated overnight 30, 37, 40.5, 41&#194;&#176;C, colonies            initial screening, isolated dnaK deletion cells <br />  counted.                                                      transformed suppressor plasmids grow     Phase-contrast microscopy. Cells fresh colonies            42&#194;&#176;C.  additional studies showed P1 transduc- <br />  grown LB agar plates 30, 37, 40.5, 41&#194;&#176;C              tion dnaK deletion allele wild-type strain <br />  prepared glass microscope slides. Zeiss photomicro-           carrying isolated suppressor clone resulted forma- <br />   scope x40 phase objective lens Kodak PanX film           tion viable colonies temperatures high  used photography.                                      40.5&#194;&#176;C. result indicated original dnaK deletion <br />                                                                     mutant strain contained additional mutation <br />                             RESULTS                                  combination suppressor clone, allowed <br />                                                                     growth 42&#194;&#176;C. <br />    Isolation suppressor dnaK deletion. dnaK              Mapping suppressor locus. map suppressor <br />  deletion strain PK101, containing alleles dnaKJ4           locus E. coli chromosome, Kanr cassette  dnaJ14, temperature sensitive growth 40&apos;.5&#194;&#176;C.        pUC4K inserted EcoRV sites pJK53. <br />  plating efficiency PK101 40.5&#194;&#176;C 0.08%       Kanr marker introduced E. coli chromo- <br />  30&#194;&#176;C (Table 3). dnaK deletion strain showed            linear transformation mapped 96 <br />  cold-sensitive growth 17&#194;&#176;C. P1 transductants         6 min set Hfr strains origins  30&#194;&#176;C replated P1 transduction          distributed chromosome. delineate  17&#194;&#176;C, colonies formed, 10 days incubation.       location suppressor, number strains carrying TnlO <br />  Cells extremely filamentous 30&#194;&#176;C            insertions region used. Kanr marker  defective cell division. previously mentioned,      tested P1 cotransduction mapped TnJO insertions. <br />  dnaK deletion strain PK101 actually DnaK- DnaJ-.              strains carrying TnJO insertion near 4 min <br />  check DnaK- phenotypes, PK101 transformed              (zae-502::TnJO) showed 80 90% cotransductiona ofstrain   DnaJ-expressing plasmid (pJK15). resulting trans-           tetracycline resistance (Tetr) Kanr markers; <br />  formants showed temperature-sensitive growth               carrying TnJO insertion near 3 min (zad-220::TnJO) gave <br />  filamentation. DnaJ-expressing plasmid          18% cotransduction. Subsequent P1 transduction analysis <br />  stably maintained dnaK deletion strain PK101,             nearby selectable markers showed 83% cotransduction <br />  PK101&apos; used studies reported  genetically        Kanr marker dapD locus (Fig. 1). relative <br />  2058     KANG CRAIG                                                                                                     J. BACTERIOL. <br />  <br />                            minute        2                      3.5                                    5 5 <br />                                                                                                                41 <br />                            leu <br />                                                                Kan     mr         hu   d8ph          fiA             <br />  <br />  <br />  <br />                                               &lt;.002        - <br />                                                                       ,.-   .83        - <br />                                                                       |            18 <br />                                                                   |                        .13 .04 <br />  <br />                                                                                 4 (published data; ref.8) <br />                                                                            .8-.9 <br />   FIG. 1. Linkage dnaK suppressor gene number reference loci 2- 4-min segment standard E. coli linkage map. Kanr marker inserted just outside dnaK suppressor gene EcoRV site pJK53 indicated + Fig. 3 used selectable marker. <br />  <br /> cotransduction frequencies Kanr marker            hybridized probe. result reconfirmed markers suggested Kanr marker located           originally isolated suppressor clones near mrcB gene.                                                overlapping DNA segments. <br />   Comparison restriction map suppressor clone           Fine localization suppressor gene. define published restriction map E. coli chromosome       suppressor gene, sequential subcloning originally <br /> (17) revealed similar pattern restriction sites near      isolated suppressor clone  subclone mrcB gene. Subsequently, fine mapping South-           tested ability allow growth dnaK deletion <br /> ern hybridization analysis. 1.2-kb PvuII fragment iso-         strain PK101 40.5&#194;&#176;C. temperature-sensitive growth lated suppressor clone (pJK53) hybridized        strain PK101 suppressed plasmids pJK537 restriction enzyme-digested DNA X transducing phage           pJK538 pJK530 pJK533 (Fig. 3). (X17-181) covering region (Fig. 2). result indicated     results suggested dnaK suppressor gene located suppressor located upstream mrcB gene          1.7-kb DNA fragment PvuII site <br /> (3.7 min E. coli chromosome). restriction frag-         EcoRV site (*s Fig. 3). <br /> ments suppressor clones (pJK72 pJK711)               Overexpression wild-type gene suppresses dnaK dele- <br />                                                                    tion. suppressor gene localized upstream                                                                    mrcB gene, wild-type gene cloned X <br />      EcoRI Hindj    Pv u ll Psji!      Hildii                    transducing phage DNA covers region. pJK337, <br />                 RV PYIU                                            carried 1.7-kb PvuII-EcoRV DNA fragment     I1      ,,,_, <br />                 ... 1 R      11.11l       -- <br />                                           .P   ,- <br />                                                               xDNA <br />                                                                    phage, constructed used transform dnaK <br />                                                                    deletion strain, PK101. PK101 transformed pJK337 <br />                        l                1 1 <br />                                                    PsC Ps <br />                                                                    formed    colonies 40.5&#194;&#176;C, did PK101 transformed                EcoFRV <br />                   Clal       }Spat EcnRsV <br />                                        PstI <br />                                                                    pJK537,    plasmid carrying original suppressor clone. <br />                                         ,mrc <br />                                                  2Kb <br />                                                                    result implied originally isolated suppressor <br />                                                                    clone wild-type gene. Subsequent genetic experiments <br />                        probe                                       supported conclusion. determine mutation <br /> pJK53                                                              isolated gene responsible suppression pJ K72                                                             original temperature-resistant suppressor strain, asked <br /> pJK71 1                                                            temperature resistance PK113, spontane- <br />                                                                    ously isolated suppressor strain, cotransduced                                                                    Tetr marker present near suppressor gene. PK113                      z_1 <br />                    o.&apos;           .7 .p <br />                                ill            P&apos;                   transduced    P1 vir grown strain CAG18436,                            -,    --c <br />                                11- <br />                               1. d&apos; <br />                                        - <br />  <br />                                             <br />                                                                    contained    zae-502::TnlO.   zae- 502::TnO deter- <br />                                                                    mined 80 90%o cotransduced Kanr marker <br />                                                                    near suppressor gene, expected suppres- <br />                                                  .*   ).)2k b <br />                                                                     sor gene actual site mutation, 70 80%                                                       4. k r, <br />                                                                     Tetr transductants temperature sensitive.  <br />                                                  .4 <br />                                                                     95% Tetr transductants                                                                    temperature resistant. results suggested orig- <br />                                                                     inally isolated suppressor clones carried wild-type gene                                                                    overexpression gene suppressed temperature- <br />                                                                     sensitive growth dnaK deletion strain. <br />                                                                        DNA sequence determination. <span id='am-25' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-26' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> sequence                                                                     1.7-kb PvuII-EcoRV fragment (* Fig. 3) deter- <br />                                                                     mined (Fig. 4). Northern (RNA) blot analysis using labeled <br />                                                                     single-stranded DNA probes showed transcription oc- <br />    FIG. 2. Southern hybrdization 1.2-kb PvuII fragment    curred direction PvuII EcoRV site. <br /> pJK53 X17-181, pJK72, pJK711. 32P-labeled probe      Analysis sequence data revealed prepared pJK53 hybridized PstI-HindIII-digested X DNA,  nonoverlapping open reading frames sense strand <br /> PstI-HindIII-digested pJK711, EcoRV-PstI-digested pJK72.        region larger 42 codons. shorter open <br />  VOL. 172, 1990                                                                    CHARACTERIZATION dnaK SUPPRESSOR GENE                                    2059 <br />  <br />                        Hpal   PuuII*         Clal                       ECORU*                 EcoRU+ <br />  <br />  <br />                           PstI                          Puu            Puul  <br />                                                                                                                               suppressing <br />                                              ORFI            ORF2             4           kb       &gt;                                  ability <br />  <br />                                                              __________________                          pJK 53 <br />                                                                                                               pJK533 <br />                                                                                                                  pJK537               + <br />                                                                                                                  pJK538               + <br />                                              _____________                                                       pJK54t               + <br />                                                                                                                  pJK541 <br />                                                                                                                  pJK542 <br />                                                              ___                                                 pJK543 <br />   FIG. 3. Fine localization suppressor gene. originally isolated suppressor clone, pJK53, subcloned, mutants constructed vitro described Materials Methods. clone tested suppressing ability testing ability transformants PK101 form colonies 40.5&#194;&#176;C. pJK540 pJK543 +1 frameshift mutations. pJK542 frame deletion middle ORF2. ORF1 ORF2 open reading frames determined region analysis DNA sequencing data (Fig. 4). <br />  <br />  <br />  <br />  <br />     1   GACGAAAGAGGCTATCCTTAATGAATCAATTTCAGAACTGTCAGGCTATAGCTCGCTGAAAAGCGAAGTAAAATACGGCGCAGAACGCAGCCGTATTGAC                                                     100 <br />  <br />  <br />   101   TTTATGTTGCAGGCGGATTCGCGTCCAGACTGCTATATTAAGTGAAATCGGTTAQGTGCGGAGAACGAACAGGGATATTTTCCCGATGCGGTCACTG                                                        200 <br />                                                                     -35                            -10 <br />  <br />   201   AACGAGGTCAGAAACACTTCEaGUTTGATGAGCGTAGCGGCTGAAGGCCAGCGTGCGGTTATCTTTTTCGCCGTGCTGCATTCAGCCATTACACGGTTT                                                      300 <br />                              S.D    M S V E G Q R V F F V L H S T R F <br />                               ORFP--&gt; <br />   301   TCACCCGCGCGCCACATCGATGAGAAATACGCGCAACTATTGTCAGAAGCTCAACAGAGGGGGGTAGAAATTCTGGCTTACAAAGCGGAAATTTCTGCTG                                                     400 <br />          S P R H D E K Y Q L L S E Q Q R G V E L Y K E S  <br />   401   AAGGCATGGCTCTTAAAAAATCACTGCCGGTTACATTGTAGTAAAGTAAGTAACTGGTTAATTTACATTCTGGTCGCGTGCGCAAATACGCTTTTCCTCA                                                     500 <br />         E G M L K K S L P V T L <br />  <br />   501   CACAGTTGTCAAGTGTTACGTTTAGATAATTGCTATCCGGAAAAGCATCTGCTATTTATAGCGGCCTCATTTTTCCCCCGAACATGGGGATCGATAGTGC                                                     600 <br />  <br />  <br />   601   GTGTTAAGE<span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>;aLAGCAACATGCAAGAAGGGCAAAACCGTAAAACATCGTCCCTGAGTATTCTCGCCATCGCTGGGGTGGAACCATATCAGGAGAAGCCGG                                                     700 <br />                  S.D             M   Q   E   G      Q    N    R     K     T       S   S    L   S       L          G   V   E    P   Y    Q   E   K   P <br />                      0RF2--&gt; <br />   701   GCGAAGAGTATATGAATGAAGCCCAGCTGGCGCACTTCCGTCGTATTCTGGAAGCATGGCGTAATCAACTCAGGGATGAAGTCGATCGCACCGTTACACA                                                     800 <br />         G E E Y M N E Q L H F R R L E W R N Q L R D E V D R T V T H <br />  <br />   801   TATGCAGGATGAAGCAGCCAACTTCCCGGACCCGGTAGACCGTGCAGCCCAGGAAGAAGAGTTCAGCCTCGAACTGCGTAACCGCGATCGCGAGCGTAAC                                                     900 <br />           M Q D E N F P D P V D R Q E E E F S L E L R N R D R E R N <br />  <br />   901   GTGATCAAAAAGATCGAGAAGACGCTGAAAAAAGTGGAAGACGAAGATTTCGGCTACTGCGAATCCTGCGGTGTTGAAATTGGTATTCGCCGTCTGGAAG                                                    1000 <br />          V K K E K T L K K V E D E D F G Y C E S C G V E G R R L E <br />  <br />  1001   CGCGCCCGACAGCCGATCTGTGCATCGACTGCAAAACGCTGGCTGAAATTCGCGAAAAACAGATGGCTGGCTAATTACAGCCGTTCCATCACGTTTACCA                                                    1100 <br />          R P T D L C D C K T L E R E K Q M G <br />  <br />  1101   CACGGGGAAATCGTCCCGCCTTATTTTTTGTTCAAAGAGATGACAGACACACAGTATATTGGCCTGTCGCCCCTCTCTTCCGGCGAGCTTCATTTTGGCT                                                    1200 <br />  <br />  <br />  1201   CTCTGATCGCTACGCTCGGCAGCTATTTGCACGTCGCGCCCGGCAAGGTCGCTGGCTGGTACGCATAGAAGAT 1273 <br />   FIG. 4. Nucleotide sequence dksA region. sequence sense strand amino acid sequences open reading <br /> frames shown. sequence numbered HincII (left) EcoRV site (Fig. 3). -10 -35 regions possible promoter indicated. Putative Shine-Dalgamo (S.D) sequences underlined. <br />  2060     KANG CRAIG                                                                                                                    J. BACTERIOL. <br />  <br />                                                                              reading frame (ORF1) encode polypeptide 70 <br />                                                                              amino acids (Mr, 7,650), longer (ORF2)                                                                              encode polypeptide 151 amino acids (Mr, 17,480).                                                                              predicted ORF2 gene product contains 38% charged residues <br />                                                                              distributed length protein, net <br />                                                                              charge -6. ORF1 preceded putative ribosome- <br />                                                                              binding site, GGGAG, located 4 bp upstream initiation <br />                                                                              codon. ORF2 preceded putative ribosome- <br />                                ,r,~08e.^100i                                 binding site, GGAGA, located 6 bp upstream initiation <br />                                                                              codon. sequence shares homology consensus <br />                                                                              E. coli promoter sequence (26) located upstream                                             600                              ORF1. organization suggested ORF1 ORF2 <br />                                       h <br />                                                                              comprise operon. <br />                                                                                 Analysis RNA transcript start site. estimate                                                                              size mRNA encoded ORF1 ORF2, Northern <br />                                                                              hybridization experiments  RNAs isolated                                                                              wild-type strain MG1655 MG1655 carrying                                                                              suppressor clone, pJK537, separated electrophore- <br />                            2      3                                          sis hybridized labeled 408-bp ClaI-PvuII frag- <br />                                                                              ment pJK538. major RNA transcript approximately <br />   FIG. 5. Analysis RNA transcripts dksA region. Total            500 nucleotides detected RNA preparations RNAs prepared wild-type cells, wild-type cells carrying        strain suppressor clone preparations <br /> dksA clone (pJK537), cells carrying control plasmid (pJK333),          transformed strain. addition, relatively abun- <br /> electrophoresed 1.3% agarose gel, blotted. blot hybridized labeled 408-bp ClaI-PvuII fragment pJK538.            dant 600-nucleotide RNA identified transformed <br /> Lanes: 1, MG1655; 2, MG1655(pJK537); 3, MG1655(pJK333).                      strain. largest transcript hybridized probe <br />                                                                              1,000 nucleotides length (Fig. 5). <br />                                                                                 determine 5&apos; end RNA transcript, Si nuclease <br />                                                                              mapping carried  5&apos;-end-labeled probe pre- <br />                                                                              pared noncoding strand 726-bp HincII-PvuII <br />  <br />  <br />                       _          -      undigested probe <br />                                    -      560 nt <br />  <br />  <br />  <br />                                                              Hpol PuuII*               C1al <br />                                                                                                        Ndel <br />                                                                                                                       EcoRiJ*        ECoRV+ <br />                                                                                                                                             Hincl] <br />                                                                                                          l                l <br />                                    -      255 nt                    w-                                 1       1                9 <br />  <br />  <br />  <br />  <br />                                                                 Psti <br />                                                                                   PL-             PuulI <br />                                                                                                        Bcll                   l <br />                                                                                                                             Puull <br />  <br />                                                                                                                                     1 kb <br />                                                                                                                                               M- <br />                                                                                                                                               w- <br />  <br />  <br />  <br />  <br />                                                                                            ORFI        ORF2 <br />  <br />  <br />  <br />                                    -- 1 6 0 <br />                                                                                                     160 nt <br />                                                                                                   -~255 nt <br />                                                                                        -      -    -               560 nt <br />  <br />  <br />  <br />  <br />               1 12 3 14 <br />   FIG. 6. S1 analysis dksA transcripts. Si mapping carried described text, using total cellular RNA 32P-end- <br /> labeled noncoding strand HincII-PvuII fragment. sizes protected fragments  nucleotides) determined running <br /> end-labeled pBR322 digested HaeIII marker. 5&apos; end protected fragment shown restriction map region. P, Putative promoter site. Lane 1, pBR322-HaeIII marker; lane 3, undigested probe marker. reaction mixtures contained RNA MG1655(pJK537) (lane 2) yeast tRNA control (lane 4). <br />  VOL. 172, 1990                                              CHARACTERIZATION dnaK SUPPRESSOR GENE                                             2061 <br />                                                                        Mr. <br />  <br /> fragment (Fig. 6). probe hybridized total RNA               68- <br />  <br /> isolated log-phase cells strain MG1655 carrying    <br /> suppressor clone, pJK537. major protected fragments approximate lengths 560, 255, 160 nucleotides                4 2- <br />  <br />  observed (Fig. 6). Fragments lengths <br />                                                                                                                                              TET detected using RNA isolated untrans-                                                                                       BLA <br />  <br /> formed cells (data shown). cases, 160- <br /> nucleotide fragment abundant. 5&apos; ends                18 <br />  <br />  major protected fragments approximately                      1   5 <br />  <br /> mapped nucleotide positions 166, 501, 566        <br /> sequence shown Fig. 4. Position 166 located upstream ORF1   close consensus E. coli promoter sequence.                                       M.            B <br />  <br /> Positions 501 566,  close proximity                      piK <br />                                                                                    537    333 <br />                                                                                                 marKe   pBR <br />                                                                                                         322 <br />                                                                                                               pK   pJ K <br />                                                                                                               537 334 <br />                                                                                                                           pBR <br />                                                                                                                           322 <br />                                                                                                                                 p1K <br />                                                                                                                                 537 <br />                                                                                                                                       p_l( <br />                                                                                                                                        334 consensus E. coli promoter sequence. 160- <br /> nucleotide fragment abundant ORF2                                                                      FIG. 7. Polypeptides encoded plasmids overexpressing <br /> completely included pJK530, pJK530 sup-          cells minicells. Proteins analyzed 15% SDS-PAGE.   <br /> press  temperature-sensitive phenotype.          Coomassie blue-stained gel total cell extracts strain <br /> propose position 166 actual RNA transcription         MG1655, MG1655 carrying suppressor clone (pJK537), start site longest RNA transcript processed    MG1655 carrying control plasmid (pJK333); (B) Coomassie blue- smaller RNA molecules. 1-kb RNA observed          stained gel lysed 35S-labeled minicells prepared strain DS410 <br /> analysis RNA described probably length       carrying parental plasmid (pBR322), suppressor clone <br /> transcript.                                                       (pJK537), control plasmid (pJK334; pJK334    ORF2 encodes dnaK suppressor gene, dksA,        DNA fragment opposite orientation pJK333); (C) <br /> previously unidentified gene. nonoverlapping        autoradiogram panel B. Molecular weights  thousands)                                                                   shown left. Notation: BLA, P-lactamase; TET, tetracycline <br /> open reading frames sense strand putative dnaK <br />                                                                   resistance protein; *, putative dksA gene product. upper band suppressor gene region large encode       lane pJK334 C P-lactamase fusion protein initiated 42 amino acids (Fig. 3). pJK530 pJK533,          promoter activity DNA insert pJK334. covers ORF1 ORF2 region, respectively, suppressing activity. Si mapping DNA sequence analy- <br /> sis suggested RNA transcript starts position     growth defect dnaK deletion mutant. dksA clone 166 original RNA transcript region. identify    originally selected ability suppress temperature- open reading frame actually encodes suppressor       sensitive growth dnaK deletion mutant strain PK101. <br /> dnaK deletion, mutant clones constructed vitro        dnaK deletion strain extremely filamentous <br /> tested suppressing activity (Fig. 3). mutation inactivat-   30&#194;&#176;C, dksA clone tested determine <br /> ing  ORF1    (pJK540) did affect suppressing activity,   suppress filamentous phenotype. mutations inactivating ORF2 (pJK542               Cells carrying dksA clone, pJK537, showed greatly  <br /> pJK543) lost ability suppress. pJK541, carries       duced cell length 30 37&#194;&#176;C partially reduced entire ORF2 ORF1, showed suppressing           filamentation 40.5&#194;&#176;C (Fig. 8). Cells carrying control <br /> activity. results indicated ORF2 encodes           plasmid, pJK333, did reduction filamentation <br /> suppressor dnaK deletion. think pJK541 does        30&#194;&#176;C. result indicated dksA clone <br />  <br /> suppress lacks transcription initiation site.        suppressed temperature-sensitive growth nucleotide amino acid sequence deduced           filamentous phenotype dnaK deletion strain. <br /> ORF2 showed significant homology protein sequence             dnaK deletion mutant strain unable NBRF data base DNA sequence GenBank.             support growth phage X, dksA clone tested designated newly identified gene dksA (dnaK          suppress phage growth defect. suppressor).                                                      dnaK deletion mutant strain PK101, carrying dksA clone, <br />    Identification dksA gene product. identify       pJK537, support phage X growth (data shown). <br /> dksA gene product, polypeptides encoded plasmids           result suggested role DnaK inA growth pJK537 pJK334 analyzed cells carrying dksA        separable host growth. <br /> gene multicopy plasmid minicells (Fig. 7).             dksA suppresses grpE280, dnaK756, dnaJ <br />  strong band position expected 18,000-Mr            deletion mutant        phenotypes.            shown             <br /> polypeptide detected MG1655 carrying pJK537           biochemically genetically DnaK GrpE <br />  total cell extracts analyzed SDS-PAGE. contrast,      proteins interact (16), ability dksA clone  MG1655 carrying plasmid carrying control            suppress temperature-sensitive growth filamen- <br />  plasmid, pJK333, strong band position ob-         tation grpE280 mutant cells tested. grpE280 <br />  served. polypeptides encoded suppressor clone,        mutant strain carrying dksA clone showed 7,000- <br />  pJK537, analyzed minicells. Minicells          fold-higher plating efficiency 41&#194;&#176;C did grpE280 <br />  prepared strain DS410 carrying parental plasmid         strain carrying control plasmid. dksA clone suppressed <br />  pBR322, suppressor clone pJK537, control plasmid,       filamentation grpE280 37&#194;&#176;C 41&#194;&#176;C (Fig. 9). <br />  pJK334. minicells labeled L-[35S]methionine        phage X growth defect grpE280 mutant  10 min. Minicells carrying suppressor clone, pJK537,     suppressed dksA clone (data shown). <br />  showed labeled 18,000-Mr polypeptide comigrated              dnaK deletion strain PK101 used study  overexpressed 18,000-Mr protein synthesized          actually DnaK- DnaJ-, dnaJ deletion mutant strain <br />  cells carrying suppressor clone. observed          PK102 tested dnaJ deletion <br />  molecular weight consistent deduced       mutant phenotypes suppressed dnaK suppressor <br />  DNA sequence dksA gene.                                   clone. dnaJ deletion mutant strain PK102, carrying     dksA suppresses filamentation phageA     dksA clone pJK537, temperature-resistant  2062    KANG CRAIG                                                                                                  J. BACTERIOL. <br />  <br />  <br />  <br />  <br /> PKIO1/ <br />  pJK537 <br />  <br />  <br />  <br />  <br /> P K 1 0 1/ <br />  pJK333 <br />  <br />  <br />  <br />  <br />                             300C                                         370C                                       40.5&#194;&#176;C <br />   FIG. 8. Suppression filamentation dnaK deletion strain dnaK suppressor clone (pJK537). pJK333 control plasmid. <br />  <br /> 44.5&#194;&#176;C, strain PK102 carrying control plasmid                recB recC sbcB strain poor, second P1 lysate pJK333 viable temperature. dksA clone            prepared Kanr Tetr transductants (PK9), suppressed cell elongation dnaJ deletion             MG1655 cells transduced. transduc- <br /> mutant strain 44.5&#194;&#176;C (data shown).                       tants selected Kanr, 70% Tetr. similar <br />    dksA multicopy suppressor originally iso-           value (77% cotransduction) obtained transduc- <br /> lated basis suppressing dnaKJ4 dnaJ14 deletion         tants selected Tetr. result suggested mutation, ability dksA suppress dnaK756                dksA essential gene.  eliminate <br /> missense mutation tested. case dnaK14           possibility presence closely linked mutation <br /> dnaJ14 deletion mutation, temperature-sensitive growth           suppress dksA loss. <br /> cell elongation caused dnaK756 mutation sup-               dksA deletion strain PK201 grow nearly <br /> pressed dksA clone, phage growth defect             wild-type cells rich media 30, 37, 42&#194;&#176;C. (data shown).                                             grew poorly M9 minimal medium <br />    determine dksA suppress heat               supplemented 20 amino acids vitamins tem- <br /> shock protein mutant strains, lon::TnJO, groEL140,           peratures. poor growth overcome changing <br /> groES30 mutant strains tested. growth phe-             carbon sources. Phage growth normal dksA <br /> notypes strains suppressed                deletion strain. double mutant dnaK deletion overexpression dksA gene (data shown).              dksA deletion constructed. phenotypic <br /> fore, dksA does ability suppress heat            difference observed dnaK deletion strain <br /> shock mutants specific dnaK, dnaJ,             dnaK dksA double-deletion strain (data shown). <br /> grpE mutant strains. <br />    dksA essential gene. dksA newly                                     DISCUSSION <br /> identified gene, constructed dksA mutant determine effect lack dksA function. plasmid carrying          isolated previously unidentified E. coli gene, <br /> deletion dksA, pJK21, constructed removing             dksA, suppress temperature-sensitive growth <br /> 408-bp ClaI-PvuII fragment pJK538 replacing           dnaK deletion mutant dosage-dependent manner. Kanr cassette. vitro-mutated copy dksA            filamentation dnaK deletion mutant strain integrated E. coli chromosome linear transfor-           suppressed, defect growth  mation strain TnJO 70 80% linked dksA              dksA clone isolated genomic library DNA (zae-502::TnJO). Transformants selected Kanr            spontaneous temperature-resistant suppressor strain, screened ampicillin sensitivity (Amps) confirm recom-         dksA locus original suppressor <br /> bination chromosome. P1 vir grown             mutation site, dksA clone isolated wild-type Kanr Tetr Amps transformants, wild-type strain           strain rescued temperature-sensitive growth MG1655 transduced Kanr Tetr. P1 growth           filamentation dnaK deletion strain PK101  <br />  VOL. 172, 1990                                                 CHARACTERIZATION dnaK SUPPRESSOR GENE                         2063 <br />  <br />  <br />  <br />  <br /> PK5/                                                                       4.4 <br />  pJK537&quot;&apos;                                                                                * <br />  <br />  <br />                                  521                                             01l_;.i6Fte&gt;                                                                                                       ll <br />  <br />  <br />  <br />  <br />                                                                                                              jlk&apos;~~f <br />  <br />  <br />  <br />  <br /> PK5/ <br />  pJK333 <br />  <br />  <br />  <br />                             300C                                          370C                                         410C <br />   FIG. 9. Suppression filamentation grpE280 mutant strain dnaK suppressor clone (pJK537). pJK333 control plasmid. <br />  <br /> expressed. Examples dosage-dependent suppression               clear overexpression dksA suppresses reported previously. Kusukawa et al. (18) showed           temperature-sensitive growth filamentation dnaK <br /> overproduction DnaK GroE protein suppressed tem-               deletion mutant strain. possible DksA bypasses <br /> perature-sensitive growth rpoH deletion mutant                DnaK protein function(s). dksA <br /> strain. Aldea et al. (2) cloned newly identified gene,        gene dispensable gene, DksA protein <br /> bolA, suppressed wee amber mutation                   important cell growth high temperature overexpressed.                                                       dnaK mutant background. dksA mRNA level    dnaK deletion mutant strain formed colonies          dnaK deletion mutant strain PK101 threefold lower 30&#194;&#176;C, original transductants grew extremely slowly            wild-type strain. addition, dksA mRNA level filamentous.  cells                spontaneously isolated suppressor strain PK113 fourfold <br /> filamentous generations growth 30&#194;&#176;C.             higher deletion strain (unpublished result).  <br /> Suppressor mutations appear accumulate dnaK                fore, possible dnaK mutant phenotypes <br /> deletion mutant strain 30&#194;&#176;C. Bukau Walker (7)            lowered expression dksA suppres- similar observations. Additional observations              sion phenotypes original suppressor strain laboratory support idea. initial           increased expression dksA. screening suppressor clone, isolated dnaK deletion         possible transcription certain genes inhibited cells transformed suppressor plasmid             dnaK mutant strain. Mutation(s) overexpression <span id='am-40' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-41' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cer</span>- <br /> grow 42&#194;&#176;C.  additional studies shown            tain proteins results increase transcription P1 transduction dnaK deletion allele wild-type         RNAs bypasses function encoded <br /> strain carrying isolated suppressor clone resulted        proteins suppress dnaK mutant phenotypes. <br /> formation viable colonies temperatures high            interesting overexpression dksA suppressed <br /> 40.5&#194;&#176;C. failure transductants grow 42&#194;&#176;C indi-          temperature-sensitive growth dnaK deletion <br /> cated additional mutations dnaK               mutant strain, defect phage k growth. result <br /> deletion mutant strain affected growth high tempera-         suggests functions DnaK host growth tures. fact, genetic evidence         growth separable. Multiple copies dksA suppress <br /> mutations, tightly linked dnaK,            phenotypes temperature-sensitive grpE <br /> originally isolated spontaneous temperature-resistant sup-           mutant, allowing growth grpE280 mutant 41&#194;&#176;C pressor strain PK113 required temperature-resistant          suppressing filamentation caused mutation. growth (unpublished result). possible       suppression specific sense heat <br /> mutations sufficient suppress temperature-           shock genes, groEL, groES, <span id='am-1' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>Ion</span>, affected. DnaK <br /> sensitive growth present multiple copies.          GrpE interact vivo vitro (16). explain isolated wild-type gene           dksA bypasses functions carried dnaK grpE <br /> mutant genes responsible growth high temperatures.            needed growth high temperatures normal cell <br />  2064        KANG CRAIG                                                                                                      J. BACTERIOL. <br />  <br /> division. Recently, Ang Georgopoulos reported                     70K heat shock gene related genes Saccharomyces <br /> grpE deletion mutation viable dnaK mutant                         cerevisiae. Mol. Cell. Biol. 4:1454-1459. <br /> strains grpE normally essential gene (3).              14. Georgopoulos, C. P. 1977. new bacterial gene (groPC) viability suppressors dnaK mutant                     affects lambda DNA replication. Mol. Gen. Genet. 151:35-39. <br />                                                                          15. Itikawa, H., J.  Ryu. 1979. Isolation characterization <br /> strains, dnaK mutation itself. results                      <br /> </body></html>