<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, 1991, p. 3191-3198                                                                              Vol. 173,  10 <br /> 0021-9193/91/103191-08$02.00/0 <br /> Copyright C 1991, American Society Microbiology <br />  <br />  <br />   Molecular Cloning, <span id='am-20' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>Nucleotide</span> Sequence, Characterization  Bacillus subtilis Gene Encoding DNA-Binding Protein HBsu <br />          BIRGIT MICKA,&apos; NICOLAS GROCH,2 UDO HEINEMANN,2 MOHAMED  MARAHIELl* <br />  Fachbereich Chemie/Biochemie, Philipps-Universitdt Marburg, Hans-Meerwein-Strasse, D-3550 Marburg,&apos; Institut <br />             fur Kristallographie, Freie Universitat Berlin, D-1000 Berlin 33,2 Federal Republic Germany <br />                                            Received 5 October 1990/Accepted 15 March 1991 <br />  <br />               homologous class histonelike proteins believed wrap DNA condense            chromosome highly folded nucleoid structures identified different bacterial species. Bacillus <br />            subtilis encodes homodimeric DNA-binding protein called HBsu. cloned corresponding gene <br />            (hbs) 3.8-kb fragment. gene subcloned 1-kb fragment, sequenced, characterized.            encodes 92-amino-acid protein predicted molecular mass 9,884 Da. Fortunately, analysis DNA <br />            sequence downstream 3&apos; end hbs revealed location 19 <span id='am-28' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> residues MtrA.            finding located hbs gene unequivocally 5&apos; end mtr operon 2040 standard genetic <br />            map B. subtilis. Northern (RNA) blot analysis primer extension studies indicated presence            distinct hbs transcripts, initiated different sites located 160 bases apart. <br />            attempts replace hbs gene B. subtilis chromosome cat-interrupted copy (hbs::cat) <br />            marker replacement recombination unsuccessful. order study hbs essential <br />            gene, constructed strain containing truncated copy gene promoter            intact copy control isopropyl-,-D-thiogalactopyranoside (IPTG)-inducible spac-1 pro- <br />            moter. strain (BM19), normal growth depend IPTG, absence IPTG, <br />            growth severely affected. results suggest essential role hbs gene product normal growth <br />            B. subtilis. <br />  <br />    procaryotes, number abundant, small, basic,                  protein (49). identity nucleoid proteins B. <br /> heat-stable proteins identified. proteins                   subtilis (34, 39) HBsu remains unproven. <br /> wrap DNA obvious sequence specificity                      initial step understanding physiological <br /> homologous eucaryotic histones  reviews,                   role B. subtilis protein, cloned, sequenced, <br /> references 6, 11, 37, 41, 46). bacteria,                      characterized corresponding gene. gene primary structures highly conserved.                     identified genomic library constructed Lambda Zap <br /> designated histonelike DNA-binding proteins (41).                    II using radiolabeled DNA probe synthesized best-studied abundant protein HU, present                    polymerase chain reaction (PCR). <br /> Escherichia coli. exists predominantly heterodimer <br /> (HU2 HU1) encoded genes hupA hupB (25,                                     MATERIALS METHODS <br /> 26, 30, 32). subunits consist 90 amino acid <br /> residues 70% identical residues. HU                       Bacterial strains, plasmids, culture media. Bacterial <br /> protein alters structure topology bound DNA (3,                  strains listed Table 1. Plasmid pDH88 6, 7, 37, 38), affects cellular processes binding          described Henner (16). 2xYT NZY media DNA. role DNA replication especially docu-                   SM buffer described Short et al. (43). <br /> mented (5, 10, 35, 42).  function accessory                      Construction oligonucleotides. DNA sequence protein stimulating DNA recognition proteins postulated (9). addition, recent studies hupA hupB null mutants revealed growth, lysogeny, transpo- <br /> sition, cell division affected absence HU                                   TABLE 1. Bacterial strains <br /> protein (19, 35).                                                               Strain*                                            ~~~~~~~~~~~~Source <br />                                                                                                                                        rceor <br />    proteins properties structures highly                                                Genotype                   reference <br /> homologous HU protein isolated                   B. subtilis bacteria bacteriophages. Bacillus                        JH642       trpC2 pheAl                             J. Hoch <br /> species (B. caldolyticus, B. globigii, B. stearothermophilus,                 BM04        trpC2 pheAl hbs+ hbs::cat               paper B. subtilis), protein highly homologous HU, classi-                  BM19        trpC2 pheAl PhbS-hbs&apos;::cat::Pspac-hbs   paper <br /> fied HB, isolated characterized (1, 4, 21, 27).                                  hbs+ <br /> contrast HU, HB exists homodimeric protein (44). protein binds single-stranded double-                      E. coli <br /> stranded DNA RNA (4, 21, 22, 48). HB protein B.                    XL-1 Blue recAl endAl gyrA96 thi hsdRJ7 (rK-        43 <br /> subtilis, HBsu, consists 92 amino acid residues differs                            MK+) supE44 relAl X- lac [F&apos; sequence position B. globigii HB                                     proAB lacIqZAM15 TnJO (Tetr)] <br />                                                                               JM105     endAl (lac pro) thi-J strA sbcBJ5         J. Messing <br />                                                                                           hsdR4 (F&apos; traD36 proA+B+ <br />                                                                                             lacPZAM15) <br />   * <br />       Corresponding author. <br />                                                                      3191 <br />  3192     MICKA ET AL.                                                                                                          J. BACTERIOL. <br />  <br />                                       B                               method Krause et al. (28). Genomic DNA                                                     1 2 3               template mixed oligonucleotides prim- <br />                   1 2 3                                                 ers. PCR carried 50-,ul volume containing 1 ,ug <br />           kb                            kb                              template DNA, 200 pmol primer (100 pmol                                                                         oligonucleotide), 50 mM Tris-HCl (pH 9.0 25&#194;&#176;C), 15 mM <br />                                                                         ammonium sulfate, 7 mM magnesium chloride, 50 mM <br />          23 -                                                           potassium chloride, 1 mM   deoxynucleoside triphos- <br />           9A -                                                          phates, 170 ,ug bovine serum albumin ml, 1 U           6.5 -                                                         Thermus aquaticus DNA polymerase (Anglian Biotec).                                         5.4&apos; _-                         DNA denatured 93&#194;&#176;C 1 min, annealed 50&#194;&#176;C           4.3-                          4.3       - <br />                                                                         3 min, extended 72&#194;&#176;C 3 min. Following 30 cycles <br />                                                                         amplification, 2 ptl 50-pld reaction mixture           2.3-                                                          electrophoresed 1% agarose gel. product           2.0-                                                          amplification DNA fragment 241 bp length. <br />                                                                            Hybridization conditions PCR product. Nitrocellulose <br />                                                                         filters prehybridized 6x SSC (lx SSC 0.15 M <br />                                                                         NaCl plus 0.015 M sodium citrate)-10 mM EDTA (pH <br />                                                                         8.0)-100 p,g denatured salmon sperm DNA ml 1 h <br />                                                                         65&#194;&#176;C. PCR probe labeled Multiprime <br />                                                                         DNA-labeling (Amersham). probe added                                                                         prehybridization mix hybridized 5 h 65C. <br />   FIG. 1. Identification hbs gene B. subtilis.   Restric-   Southern blots plaque- colony-screening mem- <br /> tion analysis genomic DNA isolated B. subtilis JH642.           branes subjected washes: 2x SSC Chromosomal DNA digested restriction enzymes          0.5% sodium dodecyl sulfate (SDS) 15 min room <br /> HindIII (lane 1), EcoRI (lane 2), PstI (lane 3). cleaved        temperature 2x SSC 0.1% SDS 15 min products separated visualized 0.8% agarose gel. (B)       65&#194;&#176;C. Filters exposed Kodak X-Omat film. <br /> Southern blot analysis revealed major band digestion        Construction genomic library. Genomic DNA B. <br /> HindlIl (lane 1), EcoRI (lane 2), PstI (lane 3) restriction         subtilis JH642 partially digested restriction <br /> enzymes hybridization PCR probe. probe recog-          enzyme Sau3A. Bacteriophage Lambda Zap II used nized fragment 4.3 kb HindIll digest        vector clone genomic DNA according instructions <br /> fragment 5.4 kb EcoRI PstI digests. <br />                                                                         supplier (Stratagene). micrograms vector DNA <br />                                                                         linearized restriction enzyme XhoI. <br />                                                                            ligate complementary ends, <span id='am-35' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="dCTP" datatype="xsd:string"></span>dCTP</span> <span id='am-10' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dTTP</span> used <br /> oligonucleotides used PCR based amino acid             XhoI-digested vector, <span id='am-15' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span>dATP</span> <span id='am-5' about='obo:IMR_0200286' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00286" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200286" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="dGTP" datatype="xsd:string"></span>dGTP</span> <br /> sequence HBsu protein (49). oligomers represent              used Sau3A-digested chromosomal DNA. <br /> distinct parts chemically synthesized hbs gene (14).             polymerization, 1 pLg Lambda Zap II vector 0.6 oligonucleotides synthesized Applied Bio-               jig insert DNA packaged Gigapack II Gold <br /> systems automatic DNA synthesizer purified hydro-                packaging extract (Stratagene). ratio nonrecombinant <br /> phobic interaction anion-exchange fast protein liquid               (blue) recombinant (white) plaques determined chromatography. Final purification achieved poly-                plating 0.1 pu 500-pdl DNA library 200 pl XL-1 <br /> acrylamide gel electrophoresis. oligomer, 26-oligonu-             Blue host cells NZY plates containing 10 mM isopropyl- <br /> cleotide sequence, 5&apos;-CATTAACGCCGTTGCCGAAGCC                            ,B-D-thiogalactopyranoside (IPTG) 6.25 mg X-Gal <br /> AGCG-3&apos;, matches N-terminal HBsu sequence,                   (5-bromo-4-chloro-3-indolyl-p-D-galactopyranoside) ml oligomer, 50-mer, 5&apos;-TTTT-lAAAGCTTTGCCC                     (43). JH642 library contained 4. 106 PFU/pug DNA <br /> GGTTTAAAGGCCGGTACCTTGCTGGCCGGAATT                                       1. 10&apos; blue background plaques. <br /> T-3&apos;, matches C-terminal amino acid sequence.                      library amplified, phages transferred   Amplification bacterial DNA PCR. PCR used                nitrocellulose filters. Phage DNA released amplify genomic DNA fragments B. subtilis. Chromo-                   phage head denatured standard procedures (2). <br /> somal DNA JH642 (trpC2 pheAl) prepared according                 Methods hybridization wash conditions described <br />  <br />  <br />               pHBS1                                                                                                    2 <br />                            .C <br />                4s <br />                                                                                                                          3I&quot; <br />                     <br />                                                                         1                          31 <br />                4T._c                          1                         2                          3                     3.8kb <br />           B       pHBS2 =                     &#194;&#162; <br />                      _    .                   0 <br />               --51 &apos; #                     ok <br />  <br />                                               1kb <br />   FIG. 2. Restriction map DNA vicinity hbs gene.   Restriction map 3.8-kb DNA insert plasmid pHBS1. (B) <br /> Restriction map 1.0-kb DNA insert subcloned plasmid pHBS2. location orientation hbs coding sequence indicated. <br />  VOL. 173, 1991                                                                                                                         DNA-BINDING PROTEIN B. SUBTILIS                                3193 <br />  <br />                                                    -35                                                     -10                         r-P1 <br />                           1    AAATGC gATGGCTTTTTATATGTGTTACTACATACAGAAATTCTTCACTTTGTT                                                                                                       60 <br />                         61     GGACAAACATTCCTCAGAGTGCAGTTTTTCTTAAAAAGCCGTTTAATTGCCTTTCTCTTA                                                                                                 120 <br />                        121     CTTGCTCTCATTTTTTTCTGAGACAGGTTTAGAATCAGACTGAACTGTGAAGAAATGATA                                                                                                 180 <br />  <br />                                                    r-P2                m-hbs <br />                        181     ATAAACGAACTGAATGTATCCTTTTGGGAGGTGAAAGGCATGAACAAAACAGAACTT                                                                                                    240 <br />                                                                                                            SD                              M       N       K       T       E       L <br />                        241     ATCAATGCGGTTGCAGAAGCAAGCGAATTGTCTAAAAAAGACGCTACAAAAGCAGTTGAC                                                                                                 300 <br />                                      N             V             E             S       E   L       S       K K             D             T       K             V       D <br />                        301     TCTGTTTTTGATACGATCTTAGATGCACTTAAAAACGGTGATAAAATCCAACTGATCGGT                                                                                                 360 <br />                                S       V       F       D       T             L       D         L K N                   G       D       K             Q       L             G <br />                        361     TTTGGTAACTTCGAGGTGCGTGAACGTTCTGCACGTAAAGGACGCAATCCTCAAACAGGT                                                                                                 420 <br />                                F       G       N       F       E       V       R       E       R   S             R       K       G       R&apos; N            P       Q       T       G <br />                        421     GAAGAAATCGAAATTCCAGCAAGCAAAGTACCTGCTTTCAAACCAGGTAAAGCGCTTAAA                                                                                                 480 <br />                                E       E             E             P             S       K   V       P             F       K       P       G       K             L       K <br />                        481     GACGCAGTTGCCGGAAAATAATTGTGAATATAGATCGTGTATGCATCTAGCTTACATACA                                                                                                 540 <br />                                D             V *           G       K                       *           * <br />                        541     CTTTATTTCTTCACAGAAAAGCCCTxCT6AGGGGTTTATATTTCAAGAGCATGGG                                                                                                      600 <br />                        601     CTTCCTGACAGGGCATTCACTTTGCTTTTAGCGGGGCATATGTGCTAGAATCGAAATTAA                                                                                                 660 <br />                                                                                                        -   mtrA <br />                        661     ATGTATTCATTGGTGGAGGACATAGAACATGAAAGAAGTTAATAAAGAGCAAATCGAACA                                                                                                 720 <br />                                                                                                        M       K       E       V       N       K       E       Q             E       Q <br />                        721     AGCTGTTCGTCAAATTTTAGAAGCGA                                                                                                                                   746 <br />                                          V       R       Q             L       E          FIG. 3. DNA sequence hbs gene deduced amino acid sequence. hbs gene 279 bp length encodes protein 92 amino acid residues. -10 -35 regions, putative ribosome binding site (SD), sequence potential transcription <br /> terminator underlined. Arrows indicate 5&apos; ends (P1 P2) mapped primer extension. 5&apos; sequence mtrA gene 19 amino acid residues shown. <br />  <br />   Promising plaques      isolated agar plate                                                    tary sequence located nucleotides 111 transferred 500 RI     SM buffer containing 20 ,ul                                                    158. Reverse transcriptase nucleotides dATP, dGTP, <br /> chloroform (43).                                                                                                   dTTP, radiolabeled  32P]dCTP used extend <br />   vivo excision pBluescript plasmid Lambda Zap II                                                       primer produce cDNA, used vector. Bacillus library constructed Lambda                                                        mapping 5&apos; termini mRNA hbs. <br /> Zap II use facility vivo excision cloned                                                          Strain constructions interrupt chromosomal copy insert vector. phagemid, pBluescript <br /> [SK(-)], excised fl helper phage Lambda <br /> Zap II incorporates signals initiation termi- <br /> nation DNA synthesis fl origin replication                                                                                                                                  12 <br /> (43). vivo excision lambda vector, colonies                                                                                        kb appeared plates transferred nylon filters denatured standard procedures (2). Methods hybrid- <br /> ization wash conditions described previously <br /> (40). <br />    DNA sequencing. DNA sequencing cloned plasmid <br /> DNA denatured alkali carried chain                                                                                                                                              -23S <br /> termination reaction (United States Biochemical). chain-terminated products separated 6% sequencing <br /> gels visualized overnight autoradiography                                                                                                                                             -16S <br /> Kodak X-Omat film. <br />    RNA preparation Northern (RNA) blot. Fifty-milliliter                                                                                       0.540- <br /> cell cultures B. subtilis collected different                                                                                           0.38 w                      &apos; <br /> times vegetative growth. Total cellular RNA prepared procedure Penn et al. (36) modified Igo Losick (20). RNA pellets suspended 50 ,ul  diethylpyrocarbonate-treated water. transfer RNA                                                                                                                                          -5S <br />  gel blot performed method Igo                                                          FIG. 4. Northern blot analysis total cellular RNA (5 ,ug) B. <br />  Losick (20).                                                                                                  subtilis prepared different periods vegetative growth <br />    Primer extension. Primer extension used map 5&apos;                                                       (optical densities 600 nm 0.5 1.0) (lanes 1 2, respec- <br />  termini mRNA described reference 40. RNA                                                             tively). PCR probe, transcripts 0.54 0.38 kb <br />  hybridized excess single-stranded DNA                                                              identified. right indicated locations  primer. primer, 50-mer oligonucleotide, complemen-                                                       rRNAs. <br />                                                                   3194    MICKA ET AL.                                                                                                         J. BACTERIOL. <br />  <br />  <br />                                                                                                  hbs&apos; <br />                                         pDH 88                       + <br />                                                                             HindNi                    <span id='am-45' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-46' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span>Sph</span>                           pspac <br />                               Hind Sph  <br />  <br />                                                                        <br />                                                           Phbs <br />                HindNi EcoRI                                                                                          HindM EcoRI <br />  <br />  <br />                                                                                                                 hbS&apos;                         p                        hbS <br />                       phbs                                                 spac   Ib                             1 <br />                HindlilEcoRI                        Sphl                    HindM                                     HindM EcoRI <br />                                                                         FIG. 5.   Construction integration pDH88hbs&apos; homol- <br />          B                                                           ogous recombination. plasmid indicated solid lines,                           1 2                                        chromosomal DNA indicated dotted lines. box indicates                                                                      hbs coding region, jagged ends hbs&apos; indicate truncated <br />              kb                               Kt                     copy gene. positions promoters shown Pspac <br />                                                                      Phbs. locations restriction sites HindIII, EcoRl,                                                                      SphI shown. figure scale. (B) Characterization B. <br />                                                                      subtilis BM19 Southern hybridization. Chromosomal DNAs B. <br />                                                                      subtilis wild type (lane 1) BM19 (lane 2) digested                                                                      HindIII-EcoRI. cleaved products separated 0.8% <br />                                                                      agarose gel, transferred nitrocellulose filter, hybridized                                                                      PCR probe. Arrowheads indicate locations hbs gene                                                                      wild type (2.3 kb) BM19 strain containing truncated <br />                                                                      copy (6.5 kb) Pspac-hbs fusion (1.8 kb). <br />  <br />              2.3        4W                                           promoter integrative plasmid pDH88, 191-bp <br />                                                                      fragment flanked restriction sites Hindlll SphI <br />                                                                      synthesized PCR. HindIII-SphI PCR fragment <br />                                                                      contained natural ribosomal-binding site (SD) hbs gene place control spac-)             59 92 codons hbs. truncated copy pDH88 <br /> promoter. Plasmid pHBS2 linearized hbs                digested HindlIl SphI, ligated, trans- <br /> gene using restriction enzyme HincII. cat gene            formed competent cells E. coli JM105. new <br /> localized 1.3-kb EcoRI fragment pZA327 (50)              construct, pDH88hbs&apos;, transformed competent <br /> isolated. ligate blunt-ended fragments, dATP dTTP             cells B. subtilis JH642. Transformants selected filled EcoRI sites. ligation, hbs::cat       2xYT plates containing 5 ,ug chloramphenicol ml plasmid (pHBS2cat) transformed competent cells           1 mM IPTG. <br /> E. coli JM105 (40). 2 tg linearized pHBS2cat               Antisera. Antisera HB protein B. globigii, <br /> used transform competent cells B. subtilis JH642 (15).         HBgl, raised rabbits used dilution 1:500 <br /> Transformants selected 2xYT plates containing 5 ,g           purification. Polyclonal anti-HBgl antibodies chloramphenicol ml.                                           cross-reacted B. subtilis HBsu protein. Total pro- <br />   place hbs gene control spac-J              tein extracts B. subtilis E. coli cells containing  VOL. 173, 1991                                                                     DNA-BINDING PROTEIN B. SUBTILIS                     3195 <br />  <br />        2o                                                                                                           B <br />                                                                                5     4    3   2        1                      1   2 3   4 5 <br />  <br />        1.0 <br />                                                                                     5kl   _ <br />  <br />  <br />  <br />  <br />        0.5                                                                                        P~~~~ffM   --- <br />                                                                                                                    HtIB P-- <br />  <br />  <br />                                                                           FIG. 7. Western blot analysis HBsu protein B. subtilis.   <br />                                                                        Coomassie brilliant blue R 250 stain total cellular protein analyzed <br />                                                                        15% SDS-PAGE B. subtilis JH642 cells (lane 5), E. coli cells <br />                                                                        bearing recombinant plasmid pHBS1 (lane 4), vector DNA (lane <br />                                                                        3), plasmid (lane 2). (B) Proteins transferred                                                                        nitrocellulose filter incubated anti-HBgl antibodies                                                                        chromogenic substrate 5-bromo-4-chloro-3-indolyl-phosphate. Puri- <br />  0 <br />   <br />        0.1                                                             fied HBgl protein (lane 1) used internal control. <br />  <br />  C.3 <br />                                                                        4.3 kb HindIII digest. suggested presence        0.05X                                                           hbs gene fragments. <br />                                                                           Cloning, sequencing, transcriptional mapping                                                                        hbs gene. efficiency constructed genomic library <br />                                                                        high, 1% PFU                                                                        inserted DNA. 1 RI (0.002%) packaged material <br />                                                                        amplified gave rise 8_ 103 PFU/,ul.                                                                        plaques transferred nylon membranes screened <br />                                                                        PCR probe. plaques (L-1 L-5) identi- <br />                                                                        fied. purification reinfection, (L-1)                                                                        subjected excision process. using 105 phage <br />        0.01-                                                       particles L-1, ampicillin-resistant colonies ap- <br />              0      1        2         3        4      5        6      peared. colony filter hybridization screening                                  TIME (hours)                          PCR probe, promising clone identified.                                                                        plasmid, designated pHBS1, containing 3.8-kb DNA insert <br />   FIG. 6. Effect IPTG growth B. subtilis BM19 bearing     identified. Southern blot analysis, hbs gene Pspac-hbs fusion. Cultures  grown 2xYT 5 F&#194;&#177;g chloramphenicol ml presence (U) absence     assigned 900-bp EcoRI-NdeI fragment 3.8-kb <br /> 1 mM IPTG.                                                             insert pHBS1 (Fig. 2A). Plasmid pHBS2, bearing hbs <br />                                                                        gene 1-kb insert, constructed elimination                                                                        2.8-kb EcoRI insert pHBS1 (Fig. 2B). <br />                                                                           using T7 primer synthetic oligonucleotides, recombinant plasmid pHBS1 pHBS2 analyzed                    sequence major inserted DNA pHBS2 SDS-polyacrylamide gel electrophoresis (PAGE) (15% poly-               determined. hbs gene 279 bp length encodes acrylamide) described Laemmli (29). Gels stained            protein 92 amino acid residues molecular mass Coomassie brilliant blue R 250 transferred nitro-           9,884 Da (Fig. 3). strong potential ribosome-binding site <br /> cellulose immunodetection analysis (Western blot)               (AGGAGGT) AG -18.8 kcal (1 cal = 4.184J) described Towbin et al. (45).                                       located 7 bp upstream initiation codon AUG (33).    Nucleotide sequence accession number. Sequence data              hbs open reading frame ends stop codons fol- work assigned EMBL GenBank data base                    lowed hairpin loop structure similar rho-independent <br /> accession number X52418.                                               terminators. deduced amino acid sequence hbs <br />                                                                        gene identical purified HBsu <br />                                                                        protein (49).  sequence open reading <br />                              RESULTS                                   frame located downstream hbs gene determined. <br />                                                                        sequence identical 5&apos; end <br />   Construction PCR probe hybridization               mtrA gene determined Gollnick et al. <br /> chromosomal DNA. verify quality successfully             (12). shown Fig. 3, determined sequence mtrA, <br /> amplified PCR probe  241-bp fragment chromosomal              located 3&apos; end hbs, comprises 58 nucleo- <br /> hbs gene), labeled probe hybridized chromo-               tides encode 19 amino acid residues somal DNA blot washed stringent conditions                MtrA protein (12). sequence determined Gollnick et <br /> indicated Materials Methods. Chromosomal DNA                    al. 5&apos; end mtrA gene overlaps sequence cleaved restriction enzymes HindIII,            region 453 bp. region comprises 3&apos; end EcoRI, PstI revealed corresponding Southern blot            hbs, 5&apos; end mtrA, region 175 bp located presence major band digest (Fig. 1).             data revealed exact location hbs <br /> probe recognized band 5.4 kb EcoRI             gene 5&apos; end mtr operon located 2040 PstI chromosomal digests smaller fragment               Bacillus chromosome. <br />  31%      MICKA ET AL.                                                                                                       J. BACTERIOL. <br />  <br />               1          10          20           30           40          50           60           70          80           90 <br />  <br />   HBsu <br />   HBgl <br />   HBst        ------------QT-G---------- --NS-T--RK---V--------- ---------- ----------------K---LKY <br />   HBca        --------T--- G---------- - &quot;-TZ ---K---V------------ -----------N--------------------K <br />   TF-1        -------K QDT--TQVSVS-JLA-FEKITTLWTAK---V--T--L-3[KPVA-Q----F----Q-AL--AP-VGVSV--- ES--K-AE-LKYED <br />  <br />  <br />                        di                        e2                                         arm region                     63 <br />   FIG. 8. Comparison amino acid sequences HB proteins genus Bacillus. Proteins obtained B. subtilis (HBsu), <br /> B. globigii (HBgl), B. stearothermophilus (HBst), B. caldolyticus (HBca), B. subtilis bacteriophage SPOl (TF-1). lower  locations conserved helices potential DNA-binding arm region shown. amino acid residues different B. subtilis shown proteins. <br />  <br />  <br />    order determine level size hbs                  corresponds HBsu protein molec- <br /> transcript, Northern blot primer extension analyses                 ular mass purified HBgl protein (Fig. 7, lane 1). carried  Northern blot analysis total cellular              contrast, anti-HBgl antibodies did cross-react pro- <br /> RNA prepared different stages vegetative growth                   tein extract E. coli control strain (Fig. 7, lane 2) E. <br /> (optical densities 600 nm 0.5 1.0) indicated              coli containing pBluescript (Fig. 7, lane 3).  presence distinct hbs transcripts constant             immunoreactive polypeptide detected protein ex- <br /> level (Fig. 4). approximate lengths transcripts             tracts E. coli JM105 cells bearing recombinant plas- estimated 0.54 0.38 kb. addition,               mids pHBS1 (Fig. 7, lane 4) pHBS2 (data shown). primer extension studies  shown) confirmed 5&apos;             polypeptide synthesized E. coli strains carrying ends transcripts located 160 bases apart.              recombinant plasmids molecular mass 10 kDa, <br /> Inspection DNA sequence (Fig. 3) upstream                 identical B. subtilis HBsu protein. putative initiation site P1 -10 -35 revealed               expression independent external vector promoter; <br /> consensus sequences recognized                      plasmids, pHBS1 pHBS2, lacZ promoter <br /> vegetative form RNA polymerase (EorA) B. subtilis.             located 3&apos; end hbs gene. <br /> contrast, -10 -35 regions upstream P2  underlined Fig. 3) good homology conserved                                     DISCUSSION <br /> consensus sequence recognized minor form RNA <br /> polymerase containing sigma-32   (24).  addi-                  cloned, sequenced, characterized hbs <br /> tional studies needed direct evaluation              gene encoding B. subtilis histonelike protein HBsu. <br /> putative promoters.                                                     According knowledge, hbs represents identi- <br />    Interruption placing hbs gene control           fied gene histonelike protein Bacillus spe- <br /> spac-l promoter. attempts interrupt hbs gene                cies. gene encodes protein 92 amino acid residues <br /> directly double-crossover recombination event using                molecular mass 9,884 Da. deduced amino acid linearized pHBS2cat plasmid  Materials Meth-                sequence hbs gene identical determined ods) resulted production chloramphenicol-                purified B. subtilis protein differs B. globigii <br /> resistant hbs-partial diploids. Southern hybridization exper-           protein (HBgl) amino acid position 40. aspartic <br /> iments  shown) using digested chromosomal DNA                    acid residue HBsu replaced lysine HBgl (49). transformants indicated hbs: :cat construct           relationship reflects immunological cross-reactivity  <br /> integrated Campbell-like single-crossover recombina-               tween anti-HBgl antibodies HBsu protein.  <br /> tion event.                                                             trast, E. coli HU proteins (HU2 HU1)    failure interrupt hbs chromosome B.                 lower degree homology (52 57% identity) HBsu <br /> subtilis suggests essential role gene product.            protein cross-react anti-HBgl antibodies (21). <br /> test possibility placed hbs gene             HB proteins different Bacillus species far <br /> control spac-J promoter (Fig. SA) tested cell            sequenced, conservation 80% (Fig. 8).  <br /> viability relation IPTG induction. shown Fig. 6,             homology B. subtilis bacteriophage SPO1- <br /> normal cell growth strain BM19 depends highly                 encoded TF-1 protein HB proteins different Bacillus <br /> presence IPTG. absence IPTG, cell growth                species 50% (13). TF-1 isolated infected B. <br /> strongly affected. limited growth switched normal           subtilis cells shares 41% identical residues HBsu <br /> level addition 1 mM IPTG, derepresses                 (13). amino acid substitutions expression hbs. noted cells contain-          conserved helix region (Fig. 8). TF-1 binds preferentially <br /> ing Pspac-hbs fusion able grow absence            5-hydroxymethyl-<span id='am-40' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>-containing DNA present  IPTG consequence leakiness control               SPOl DNA (23), divergence amino acid <br />    integration pDH88hbs&apos; homologous recombi-                sequence Bacillus HB proteins  nation event confirmed Southern hybridization ex-               arisen selective pressure adapt novel func- <br />  periments. shown Fig. SB, HindIII-EcoRI digest             tion.  nucleotide sequence TF-1 <br />  wild-type DNA, hbs located single 2.3-kb frag-              gene (13) compared hbs, 57% identity  ment. strain BM19, fragment split 6.5-kb             observed. indicates higher level conservation fragment carrying truncated gene hbs 1.8                   DNA level observed amino acid sequence <br />  kb-fragment containing Pspac-hbs fusion.                           level. suggest common ancestor genes. <br />     HBsu expression E. coli. Anti-HBgl antibodies cross-             hand, histonelike proteins lysine  reacted specifically single band 10 kDa           arginine rich devoid cysteine, tryptophan,  total B. subtilis protein extract (Fig. 7, lane 5). band          tyrosine, generally. <br />  VOL. 173, 1991                                                                    DNA-BINDING PROTEIN B. SUBTILIS                     3197 <br />  <br />    data obtained Southern hybridization                  2. Berger, S. L., G. M. Wahl. 1987. Screening colonies sequence analysis suggest presence single copy                   plaques radioactive nucleic acid probes. Methods  hbs gene B. subtilis chromosome. copy                     zymol. 152:415-423. <br /> located 5&apos; end mtrA gene maps 2040              3. Broyles, S. S., D. E. PettiJohn. 1986. Interaction standard genetic map B. subtilis. determined                  Escherichia coli HU protein DNA: evidence formation <br />                                                                              nucleosome-like structures altered DNA helical pitch. J. DNA sequence (Fig. 3), mtrA initiation codon                      Mol. Biol. 187:47-60. <br /> placed 175 bp downstream hbs stop codon.                 4. DiJk, J., S. W. White, K. S. Wilson, K. Appelt. 1983. hbs gene located 900-bp EcoRI-NdeI chromosomal                       DNA binding protein II Bacillus stearothermophilus.  <br /> fragment. E. coli Salmonella typhimurium, genes                   Purification, studies solution, crystallization. J. Biol. encode closely related proteins (HU2                       Chem. 258:4003-4006. <br /> HU1) form heterodimeric protein HU (17, 25, 26,                  5. Dixon, N. E.,  Kornberg. 1984. Protein HU 30-32). availability genes encoding HU E. coli                 enzymatic replication chromosomal origin Escherichia S. typhimurium facilitated studies biological role             coli. Proc. Natl. Acad. Sci. USA 81:424-428. <br /> vivo. Recently, HU null mutants constructed                      6. Drlica, K., J. Rouvibre-Yaniv. 1987. Histonelike proteins                                                                              bacteria. Microbiol. Rev. 51:301-319. <br /> shown defective growth, cell division, lysogeny,             7. Durrenberger, M., M.  Bjornsti, T. Uetz, J.  Hobot, E. <br /> transposition bacteriophage Mu (18, 19, 35, 47).                     Kellenberger. 1988. Intracellular location histonelike <br /> pleiotropy suggests important role HU protein                  protein HU Escherichia coli. J. Bacteriol. 170:4757-4768. physiology E. coli S. typhimurium.                             8. Ferrari, F. , K. Trach, D. LeCoq, J. Spence, E. Ferrari,    order understand vivo physiological role              J.  Hoch. 1985. Characterization spoOA locus HBsu protein growth development B. subtilis,                   deduced product. Proc. Natl. Acad. Sci. USA 82:2647-2651. constructed vitro hbs::cat mutation.                   9. Flashner, Y., J. D. Gralla. 1988. DNA dynamic flexibility  unable replace hbs chromosomal copy                     protein recognition: differential stimulation bacterial hbs::cat construct double-crossover recombina-                   histone-like protein HU. Cell 54:713-721. <br />                                                                          10. Funnell, B. E., T.  Baker,  Kornberg. 1987. vitro <br /> tion event. transformation efficiency                  assembly prepriming complex region Escherichia <br /> low, chloramphenicol-resistant transformants                    coli chromosome. J. Biol. Chem. 262:10327-10334. <br /> analyzed Southern hybridization screen gene                11. Geider, K., H. Hoffmann-Berling. 1981. Proteins controlling <br /> interruption. transformants tested                      helical structure DNA. Annu. Rev. Biochem. 50:233-260. <br /> partial diploids; contain intact interrupted            12. Gollnick, P., S. Ishino, M.  Kuroda, D. J. Henner, C. <br /> copy hbs, arisen Campbell-like                        Yanofsky. 1990. mtr locus gene operon required integration mechanism.                                                       transcription attenuation trp operon Bacillus subtilis. <br />    failure interrupt hbs B. subtilis chromo-               Proc. Natl. Acad. Sci. USA 87:8726-8730. <br />                                                                          13. Greene, J. R., S. M. Brennan, D. J. Andrew, C. C. Thompson, homologous recombination using hbs::cat                      S. H. Richards, R. L. Heinrikson, E. P. Geiduschek. 1984. <br /> struct prompted use strategy control hbs                    Sequence bacteriophage SPOl gene coding transcrip- <br /> expression vivo. constructed strain bearing                   tion factor 1, viral homologue bacterial type DNA <br /> truncated hbs copy control promoter                     binding proteins. Proc. Natl. Acad. Sci. USA 81:7031-7035. second complete copy spac promoter                       14. Groch, N., R. Quaas, U. Hahn, U. Heinemann. 1988. <br /> control (Fig. 5A). strain, BM19, normal growth                   <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> synthesis cloning gene coding Bacillus depend IPTG induction, indicating essen-                    subtilis HBsu protein. Nucleosides Nucleotides 7:817-820. <br /> tial role intact HBsu protein cell growth. Cells              15. Gryczan, T. J., D. Dubnau. 1978. Construction prop- <br /> containing Pspac-hbs fusion grew threefold                     erties chimeric plasmids Bacillus subtilis. Proc. Natl. <br /> slowly cells presence 1 mM IPTG.                     Acad. Sci. USA 75:1428-1432. <br />                                                                          16. Henner, D. J. 1990. Inducible expression regulatory genes reduced growth absence IPTG                      Bacillus subtilis. Methods Enzymol. 185:223-228. leakiness inducible spac promoter. Similar results            17. Higgins, N. P., D. Hillyard. 1988. Primary structure obtained spoOA gene fused                      mapping hupA gene Salmonella typhimurium. J. <br /> spac promoter vector (8).                                      Bacteriol. 170:5751-5758. <br />    hbs essential gene viability B. subtilis?            18. Hillyard, D. R., M. Edlund, K. T. Hughes, M. Marsh, N. P. <br /> results mentioned  including   failure interrupt              Higgins. 1990. Subunit-specific phenotype Salmonella typhi- <br /> hbs homologous recombination, (ii) IPTG-dependent                     murium HU mutant. J. Bacteriol. 172:5402-5407. <br /> growth strain BM19 containing hbs gene spac                 19. Huisman, O., M. Faelen, D. Girard,  Jaffe,  Toussaint, promoter control, (iii) fact gene expressed              J. Rouvi Yaniv. 1989. Multiple defects Escherichia coli distinct promoters growth, suggest                   mutants lacking HU protein. J. Bacteriol. 171:3704-3712. <br />                                                                          20. Igo, M. M., R. Losick. 1986. Regulation promotor important essential physiological role HBsu                 utilized minor forms RNA polymerase holoenzyme B. <br /> protein B. subtilis.  final proof                  subtilis. J. Mol. Biol. 191:615-624. <br /> HBsu essential growth development awaits addi-                21. Imber, R., H. Bachinger, T.  Bickle. 1982. Purification tional detailed studies, way.                                characterization small DNA-binding protein, HB,                                                                              Bacillus globigii. Eur. J. Biochem. 122:627-632. <br />                       ACKNOWLEDGMENTS                                    22. Imber, R., M. Kimura, N. Groch, U. Heinemann. 1987. <br />   thank L. Diederich S. Borchert assistance PCR            DNA-binding properties primary structure HB protein <br /> library construction, respectively. thank G. Willimsky                Bacillus globigii. Eur. J. Biochem. 165:547-552. <br /> discussion J. Bittner excellent technical assistance.    23. Johnson, G. G., E. P. Geiduschek. 1977. Specificity thank D. J. Henner providing pDH88 sharing sequence              weak binding phage SPOl transcription-inhibitory <br /> data mtr operon.                                                      protein, TF-1 SPOl DNA. Biochemistry 16:1473-1495. <br />                                                                          24. Johnson, W. C., C. P. Moran, R. Losick. 1983. RNA <br />                            REFERENCES                                         polymerase sigma factors Bacillus subtilis discriminate <br />  1. Beck, , J. DiJk, R. Reinhardt. 1987. Ribosomal proteins            overlapping promoters developmentally regulated <br />     DNA-binding protein II extreme thermophile                  gene. Nature (London) 302:800-804. <br />     Bacillus caldolyticus. Biol. Chem. Hoppe-Seyler 368:121-130.         25. Kano, Y., K. Osato, M. Wada, F. Imamoto. 1987. Cloning <br />  3198       MICKA ET AL.                                                                                                              J. BACTERIOL. <br />  <br />       sequencing HU-2 gene Escherichia coli. Mol. Gen.           structure circular double-stranded DNA. Cell 17:265-274. <br />       Genet. 209:408-410.                                                  39. Salti, V., F. Le Hegarat, L. Hirschbein. 1985. Isolation 26.   Kano, Y., S. Yoshino, M. Wada, K. Yokoyama, M. Nobuhara,                 characterization small heat-stable acid-soluble DNA-binding <br />       F. Imamoto. 1985. Molecular cloning nucleotide se-               proteins Bacillus subtilis nucleoids. J. Gen. Microbiol. <br />       quence HU-1 gene Escherichia coli. Mol. Gen. Genet.            131:581-590. <br />       201:360-362.                                                         40. Sambrook, J., E. F. Fritsch, T. Maniatis. 1989. Molecular <br /> 27.   Kimura, M., K. S. Wilson. 1983. DNA binding                   cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />       protein II Bacillus stearothermophilus. lI. amino acid          Cold Spring Harbor, N.Y. <br />       sequence relation homologous proteins       41. Schmidt, M. B. 1988. Structure function bacterial <br />       procaryotes. J. Biol. Chem. 258:4007-4011.                         chromosome. Trends Biochem. Sci. 13:131-135. <br /> 28.   Krause, M., M.  Marahiel, H. von Dohren, H.  <br /> </body></html>