<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Yeast ribosomal protein L10 helps coordinate tRNA <br /> movement large subunit <br />  <br /> ABSTRACT                                                                        sequentially bind EF-Tu aa-tRNA <span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> (ternary com- <br />                                                                                 plex TC; eukaryotes elongation factor called <br /> Yeast ribosomal protein L10 (E. coli L16) located                            &#226;&#8364;&#732;eEF1A&#226;&#8364;&#8482;), stimulate GTP hydrolysis EF-Tu, accommo- center topological nexus connects                              <span id='am-13' about='oboInOwl:date' typeof='owl:Thing'>date</span> aa-tRNA large subunit, e&#239;&#172;&#8364;ect peptidyltrans- functional regions large subunit.                              fer, bind EF-G (eEF2 eukaryotes), stimulate GTP <br /> essential protein previously implicated                             hydrolysis EF-G translocate mRNA processes diverse ribosome biogenesis,                                    codon. TC EF-G structurally similar (1), <br /> translational fidelity mRNA stability.                             large degree overlap ribo- <br /> inability maintain yeast Killer virus used                           binding sites. yeast, example, eEF2 contacts proxy large subunit defects identify                              sarcin-ricin loop (SRL, H95 25S rRNA), series L10 mutants. mapped roughly                                  GTPase-associated center (GAC composed ribosomal discrete regions protein. detailed ana-                           protein L12, H43, H44), Helices 33, 34, 69 89, ribo- <br /> lysis mutants located N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482;                            somal proteins L9, L12 P proteins (P0, P1a, P1b, <br /> L10, inserts bulge 25S rRNA helix                             P2a, P2b) large ribosomal subunit,                                                                                 interacts elements small subunit includ- <br /> 89, revealed strong effects rRNA structure corre- <br />                                                                                 ing h5, h15, h33, h34, h44 head body domains, <br /> sponding entire path taken tRNA 3&#226;&#8364;&#8482; end                            ribosomal protein S23 (2). interactions moves large subunit                                factors ribosome initially char- <br /> elongation cycle. mutant-induced structural                                 acterized using <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> footprinting methods (3). cryo- <br /> changes wide-ranging, affecting ribosome bio-                               electron microscopy study TC stalled genesis, elongation factor binding, drug resistance/                            Escherichia coli 70S ribosome kirromycin revealed hypersensitivity, translational fidelity virus main-                        similarly complex set interactions included tenance. importance L10 potential trans-                            SRL GAC large subunit (4). <br /> ducer information ribosome,                                using yeast-based dissect possible role N-terminal domain switching                             ribosome coordinates TC binding, aa-tRNA accom- pre- post-translocational states                                modation, peptidyltransfer eEF2 binding. Molecular discussed.                                                                  genetics biochemical studies using mutant rRNAs                                                                                 ribosomal proteins identifying important elements                                                                                 allosteric rearrangements involved processes (5&#226;&#8364;&#8220; <br />                                                                                 11). studies led focus structural ele- <br />                                                                                 ments vicinity &#226;&#8364;&#732;accommodation corridor&#226;&#8364;&#8482;, INTRODUCTION                                                                    passageway located interface H89 ribosome complex macromolecular machine                           H90&#226;&#8364;&#8220;H92 structure large subunit rRNA (12). <br /> orchestrates multiple reactions highly coordinated                         Previously, demonstrated involvement central <br /> manner. atomic level ribosome structures                               extension ribosomal protein L3 coordinating TC, <br /> reveal topologies functional centers,                      eEF2 binding peptidyltransfer helping open <br /> exchange information coordinate activities                            close structure (9). <br /> remains active &#239;&#172;&#65533;eld inquiry. example,                             Ribosomal protein L3 located H90&#226;&#8364;&#8220;H92 translation elongation cycle ribosome                              accommodation corridor. Examination  <br />  correspondence addressed. Tel: +1 301 405 0918; Fax: +1 301 314 9489; Email: dinman@umd.edu <br /> Present addresses: <br /> Alexey N. Petrov, Department Structural Biology, Stanford University School Medicine. D105 Fairchild Science Building, 299 Campus Drive <br /> West, Stanford, CA 94305-5126, USA <br /> Sara C. Roshwalb, University Tennessee College Veterinary Medicine. 2407 River Dr Knoxville, TN 37996, USA <br />  <br /> &#195;&#376; 2008 Author(s) Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  6188 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br /> region reveals ribosomal protein, L10 (L16       MATERIALS METHODS <br /> E. coli L10e Haloarcula marismortui) located        Strains, media genetic methods H89 corridor. L10 extends solvent <br /> accessible near factor-binding site         Escherichia coli strain DH5a used amplify plas- peptidyltransferase center (PTC) core          mids, E. coli transformations performed using <br /> large subunit. N-terminal quarters yeast            high-e&#239;&#172;&#402;ciency transformation method (24). hap- <br /> L10 sandwiched helices 89 38, appears         loid S. cerevisiae rpl10 gene deletion strain AJY1437 largely structurally conserved bacterial      (MATa met15D0 leu2D0 ura3D0 his3D0 RPL10::kanR <br /> archael homologs (13&#226;&#8364;&#8220;16). Inspection structures         pAJ392) kind gift Dr. W. Johnson. reveals proteins represent extension H39        L M1 dsRNA viruses introduced large subunit rRNA. cryo-EM reconstitutions,           strain cytoplasmic mixing (cytoduction) (25) using eukaryote speci&#239;&#172;&#65533;c C-terminal region L10 appears        strain JD759 [MAT  kar1-1 arg1 thr x) (L HN M1)] <br /> form large unresolved mass adjacent L7/L12 stalk        cytoplasm donor. resulting strain desig- association Helix 38 (17). Saccharomyces cere-        nated JD1293 [MATa rpl10::Kan met15D0 leu2D0 <br /> visiae, L10 encoded essential single copy RPL10        ura3D0 his3D0 pRPL10-URA3-2  (L HN M1)]. Yeast <br /> gene (18). L10 plays critical role 60S subunit biogen-      cells transformed using alkali cation method (26). <br /> esis: incorporation L10 large subunit constitu-      YPAD synthetic complete medium (H-), tes step 60S subunit biogenesis (19). Speci&#239;&#172;&#65533;cally,   YPG, SD 4.7 MB plates used testing killer <br /> immature large subunits lacking L10 exported            phenotype prepared used described previously nucleus cytoplasm Crm1p pathway            (27). Oligonucleotide primers purchased IDT <br /> complex export adapter protein Nmd3p,             (Coralville, IA). Dual luciferase assays quantitatively L10 brought ribosome complex              monitor translational recoding yeast performed Sqt1p, promoting release Nmd3p followed               previously described (28). involved use incorporation L10. Depletion studies demonstrate              0-frame control reporter combination  1 riboso- large subunits lacking L10 unable form 80S           mal frameshift, nonsense suppression test reporter  <br /> ribosomes translationally inactive (20,21).         structs. Recoding e&#239;&#172;&#402;ciencies statistical analyses L10 key component required &#239;&#172;&#65533;nal activation        performed previously described (28,29). fold dilu- <br /> step ribosomes cytoplasm. addition,        tion spot assays monitor sensitivity anisomycin (20 rpl10 allele shown inhibit degradation         g/ml) paromomycin (3 mg/ml) performed pre- <br /> nonsense-containing mRNAs, suggesting L10               viously described (30). <br /> play role translational &#239;&#172;&#65533;delity (22).  L10 important ribosome assembly post-transcrip-         Generation rpl10 alleles <br /> tional regulatory processes.                                     library plasmid-borne rpl10 mutants constructed <br />    goal furthering understanding            using error-prone PCR gap repair method (31) <br /> L10, primary library randomly mutagenized rpl10             based pJD589. create pJD589, 1-kb fragment <br /> alleles screened inability maintain           containing 50 promoter, open reading frame 30 <br /> yeast &#226;&#8364;&#732;Killer&#226;&#8364;&#8482; phenotype, caused endogen-         untranslated region (UTR) sequences RPL10 sub- <br /> ous dsRNA virus highly sensitive broad array        cloned URA3-2  vector pAJ392 BamHI changes translational apparatus (23).         sites pRS313 (32). Silent mutations added ward genetic screen identi&#239;&#172;&#65533;ed 56 new rpl10 alleles.              codons 4, 5, 220 221 RPL10 ORF create mutants generally clustered regions         StuI restriction sites. Mutagenesis primers (70 nt) L10 structure:   &#226;&#8364;&#732;hook&#226;&#8364;&#8482; L10 inserts          PCR designed complementary 50 30 bulge base Helix 89; (ii) cluster           UTRs RPL10 include RPL10 translational <br /> appears form bridge Helices 38 39; (iii)        start stop codons: (forward 50 TTCCGCAAGTGC series amino acids appear form plane        TTTTGGAGTGGGACTTTCAAACTTTAAAGTACA face L10 interacts distal regions           GTATATCAAATAACTAATTCAAGATGGCTAGAA Helices 89 38; (iv) amino acids present        GG30 , reverse 50 AATTACTGTTTAATAAACTAGAAT <br /> unresolved region protein directly               TTAAATCAAAAAAATTTCTCTTTTAAGTTAGTTC <br /> interact GTPase-associated center. current          AAATGTTTGAAAAGAACTTAGG30 ). Random muta- <br /> study focuses N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; region L10.            genesis performed GeneMorph II PCR series genetic biochemical studies led             random mutagenesis kit template concentrations <br /> detailed analysis e&#239;&#172;&#8364;ects mutants           optimized generate mutations <br /> region 25S rRNA structure. pro-           RPL10 coding sequence. pJD589 digested mote structural changes map path taken         StuI, linearized plasmid lacking RPL10 <br /> tRNAs elongation cycle. considered              coding sequence puri&#239;&#172;&#65533;ed Tris&#226;&#8364;&#8220;acetate&#226;&#8364;&#8220;EDTA&#226;&#8364;&#8220; physical location L10 large         agarose gel electrophoresis. cotransformed subunit role ribosome biogenesis, &#239;&#172;&#65533;nding          randomly PCR-mutagenized RPL10 coding sequences <br /> presented used build model L10             JD1293 cells. Recombinants selected growth <br /> plays important role coordinating tRNA passage            medium lacking histidine ( , cells having lost ribosome.                                            wild-type RPL10-containing plasmids selected <br />                                                                        Nucleic Acids Research, 2008, Vol. 36,  19 6189 <br />  <br />  replica plating 5-&#239;&#172;&#8218;uoroorotic acid (5-FOA)-         70S ribosome complexed tRNAs 2.8  resolu- <br /> containing medium (33). Note recombination                 tion (PDB 2J00 2J01) (14), T. thermophilus 70S PCR product linearized vector               ribosome complexed model mRNA designed silent mutations revert           tRNAs 3.7  resolution (PDB 2I1C 1VSA) (15), <br /> wild-type sequence recombinant clones.                   E. coli ribosome complexed tRNAs 3.5  <br />                                                                 (2AW4) (13), T. thermophilus 70S ribosome Identification rpl10 alleles unable support yeast     model mRNA tRNAs 5.5  (2HGU) (39) killer virus                                                    cryo-EM reconstruction D. radiodurans ribosome <br /> Approximately 3000  colonies arrayed          complexed thiostrepton 3.3&#226;&#8364;&#8220;3.7  (2ZJR) (40) <br /> plates, replica plated 4.7 MB plates seeded           visualized using PyMOL (DeLano Scienti&#239;&#172;&#65533;c LLC). <br /> 5   47 killer indicator cells, incubated 208C iden- <br /> tify Killer  (K&#226;&#8364;&#8220;) colonies previously described (34). <br /> Plasmids harboring rpl10 alleles rescued E. coli      RESULTS yeast strains lost killer phenotype, rein-    Identification translationally defective rpl10 alleles <br /> troduced JD1238 cells rescored inability <br />                                                                 Prior studies implicated L10 broad range maintain killer phenotype. procedure  <br />                                                                 ribosome-associated functions, including large subunit <br /> formed times order prevent identi&#239;&#172;&#65533;cation                                                                 maturation, subunit association NMD. facts, <br /> false positive strains spontaneous Killer virus loss. <br />                                                                 combination centralized location L10 RNA blot analyses                                               3-D structure large subunit, provided                                                                 motive pursue depth molecular genetic RNA (northern) blotting performed previously             biochemical analysis L10 function. end, <br /> described (35). Total RNA extracted acid               library randomly mutagenized rpl10 alleles phenol/chlorophorm (pH = 4.5) mid-logarithmic              screened inability maintain yeast killer phe- <br /> cell cultures previously described (36). Equal amounts       notype. killer yeast composed L RNA (2.5  g) separated 1% agarose-            helper, M1 satellite dsRNA viruses (41). <span id='am-33' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> gel, RNA transferred Hybond-N             genome L virus encodes open reading <br /> membrane (Amersham) UV cross linked mem-             frames &#239;&#172;&#65533;rst encodes Gag (capsid) protein, <br /> brane. Nonspeci&#239;&#172;&#65533;c binding sites membrane            second encodes viral RNA-dependent RNA <br /> blocked using ULTRAhyb-Oligo bu&#239;&#172;&#8364;er (Ambion), incu-              polymerase (RDRP), production depen- <br /> bated [32P] 50 -end-labeled oligonucleotide (50 C        dent programmed &#226;&#8364;&#8220;1 ribosomal frameshift (&#226;&#8364;&#8220;1 <br /> CGGGGTGCTTTCTGTGCTTACCGATACGACCTT                               PRF) (42,43). single open reading frame M1 <br /> TACCGGCTG30 ) complementary 50 end                satellite virus encodes protein toxin exported CYH2 CDS, subsequently washed according man-             surrounding media kills uninfected cells, pro- <br /> ufacturer&#226;&#8364;&#8482;s instructions. Hybridizing species identi-      ducing easily detectable Killer+ (K+) phenotype, &#239;&#172;&#65533;ed using GeneStorm phosphoimager (Bio-Rad),              maintenance requires L supply capsid proteins <br /> quanti&#239;&#172;&#65533;ed using QuantifyOne software Bio-Rad.              RDRP. M1 highly sensitive small perturbations <br />                                                                 translational apparatus, including changes &#226;&#8364;&#8220;1 <br /> Ribosome biochemistry visualization                         PRF e&#239;&#172;&#402;ciency (34), ribosome biogenesis defects (44) Lysates cycloheximide arrested yeast cells sedi-        decreased overall protein synthesis (45).  killer <br /> mented 7&#226;&#8364;&#8220;47% sucrose gradients polysome             assay provides simple rapid method screen pro&#239;&#172;&#65533;les determined monitoring A254 nm pre-           translational defects. <br /> viously described (37). S. cerevisiae ribosomes iso-          coding region RPL10 subjected random <br /> lated, yeast aminoacyl-tRNA synthetases puri&#239;&#172;&#65533;ed            PCR mutagenesis screened loss killer phe- yeast phenylalanyl-tRNAs aminoacylated            notype described Materials methods section. <br /> [14C]Phe puri&#239;&#172;&#65533;ed HPLC previously described            Mutant rpl10-HIS3 plasmids rescued Killer <br /> (6). Single-round peptidyltransfer assays using Ac-             minus (K&#226;&#8364;&#8220;) strains, passaged E. coli, reintroduced <br /> [14C]Phe-tRNA puromycin, equilibrium binding            rpl10D cells rescored abilities promote <br /> studies [14C]Phe-tRNA binding ribosomal site        M1 virus loss. signi&#239;&#172;&#65533;cantly high intrinsic rate carried using poly(U) primed ribosomes pre-         killer loss JD1293 strain background ( 10%), viously described (6). Equilibrium binding studies eEF-      plasmid rescue reintroduction procedure repeated <br /> 2 performed previously described (9). data          times eliminate false-positive results. rpl10 &#239;&#172;&#65533;tted site-binding model ligand deple-       mutations responsible conferring K  phenotype <br /> tion using Prism Graph Pad software. rRNA structure             identi&#239;&#172;&#65533;ed DNA sequence analysis. &gt;3000 <br /> analysis using DMS, kethoxal CMCT performed            colonies tested, total 56 unique mutants identi- previously described (9). X-ray crystal structure     &#239;&#172;&#65533;ed. addition,  15% mutants lethal H. marismortui 50S ribosomal subunit (1VQ6) (38),           determined inability grow sole allele cryo-electron microscopy (cryo-EM) reconstruction           rpl10 (data shown). collection K&#226;&#8364;&#8220; mutants  S. cerevisiae ribosomal proteins threaded           tained alleles harboring 35 single mutations, 20 double <br /> X-ray crystal structure H. marismortui 50S ribo-         mutations triple mutant. summarized <br /> somal subunit (PDB IS1I) (2), Thermus thermophilus          Supplementary Table 1. large overlap <br />  6190 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  observed single double mutants.                       groups, speci&#239;&#172;&#65533;cally:   hook-like structure located high degree overlap single double                          N-terminal region L10 inserts major <br /> mutants led conclude mutagenesis                       groove helix 89; (ii) space appears link <br /> formed near saturation, large number alleles                    helices 38 39; (iii) plane protein lying single <span id='am-26' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> substitutions allowed exclusion                     &#226;&#8364;&#732;atop&#226;&#8364;&#8482; helix 89; (iv) unresolved C-terminal <br /> alleles multiple mutations subsequent analysis.                     domain L10 appears extend outward single mutants, multiple substitutions                         GTPase-associated center. Given apparent structural amino acid residues 7, 9, 40, 93, 94 145.                        complexity L10, chose focus characterizing Areas L10 high frequencies mutations                          N-terminal &#226;&#8364;&#732;hook domain&#226;&#8364;&#8482; mutants study. <br /> clustered regions protein. regions <br /> spanned amino acids 7&#226;&#8364;&#8220;17, 59&#226;&#8364;&#8220;94 144&#226;&#8364;&#8220;152.                                  N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants affect cell growth <br /> Consistent &#239;&#172;&#65533;ndings Johnson lab, viable <br />                                                                               Ribosome-associated defects commonly promote de&#239;&#172;&#65533;cien- <br /> mutants mapping unstructured central loop (amino <br />                                                                               cies cellular growth ambient temperature. semi- <br /> acids 102&#226;&#8364;&#8220;112) identi&#239;&#172;&#65533;ed current study,                       quantitative monitor  10-fold dilution spot assays <br /> sistent suggestion loop play critical                 performed rich medium 308C, revealing allele- <br /> function peptidyltransfer (46). diagramed                            speci&#239;&#172;&#65533;c growth defects R7Q Y11C mutants <br /> Figure 1, mutants appear cluster distinct                    (Figure 2A). Note large colonies Y11C <br />                                                                               sample appear escape mutants. Changes ribosome <br />         5S                                                                    structure function alter growth characteristics <br />    L4 <br />                    L10                                                        cells decreased elevated temperatures.  <br />                                                                               mutants assayed regard resistance <br />                                   H38                  4 C-term <br />              PTC                                                              sensitivity low (158C) high (378C) temperatures. <br />                      L3                                    P160T              shown Figure 2B, mutations tyrosines                                                            P179S <br />   L2                                                                          N-terminal region protein bridging helices 89                                                            F199I <br />                                                            S201F <br />                                                            K202 Q <br />                                                                               39, e. Y9H, Y9N Y11C promoted enhanced growth <br />                                                                               low temperature. contrast, mutants <br />                           2 <br />                                                                               a&#239;&#172;&#8364;ected growth 378C (Figure 2C). fold serial dilu- <br />                                                                               tion assays used probe e&#239;&#172;&#8364;ects small <br />         H39                                                  3                molecule protein synthesis inhibitors translational <br />                                                                               apparatus. Anisomycin competitive inhibitor site <br />                                                                               binding sterically hinders positioning <span id='am-39' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span>acceptor</span> <br />                                                                               end site tRNA PTC (47).                               1                                               mutants hypersensitive determined                                                                               reduced growth 15  g/ml anisomycin (R7Q, Y9H, <br />                                         H89                                   Y9N Y11C), resistant evidenced ability <br />                                                                               grow medium containing 20  g/ml anisomycin <br />                                                                               (R7L, R7P Y9C) (Figure 2D E, summarized <br /> Figure 1. Localization L10 mutations identi&#239;&#172;&#65533;ed study.          Table 1). data indicate high degree allele- <br /> Insert shows &#226;&#8364;&#732;crown view&#226;&#8364;&#8482;, general location L10 relative salient features large subunit. Large picture shows       speci&#239;&#172;&#65533;city regard drug. small mole- <br /> L10 mutants mapped structure yeast L10 taken (2).            cule drug, paromomycin, binds decoding center spatially de&#239;&#172;&#65533;ned groups L10 mutants circled               small subunit promotes conformational changes <br /> labeled, group 1 represents located N-terminus          associated formation codon&#226;&#8364;&#8220;anticodon helix protein. Group 4, indicated pink circle, map structu- <br /> rally unde&#239;&#172;&#65533;ned C-terminal region yeast L10. Neighboring 25S rRNA           mRNA incoming site tRNA (48).  <br /> helical structures indicated, GAC indicates GTPase-               paromomycin used probe defects associated <br /> associated center large subunit.                                       interactions small large subunits. <br />  <br />                  30&#194;&#176;C                 B       15&#194;&#176;C                C      37&#194;&#176;C              D      Aniso (15&#194;&#181;g/ml)         E    Aniso (20&#194;&#181;g/ml) <br />      WT <br />     R7L <br />     R7P <br />     R7Q <br />     Y9C <br />     Y9H <br />     Y9N <br />    Y11C <br />  <br /> Figure 2. Genetic phenotypes L10 N-terminal hook mutants. fold dilutions cells harboring indicated rpl10 alleles spotted complete synthetic medium lacking histidine ( , incubated indicated temperatures  left panels). right panels, cells spotted  medium containing indicated concentrations anisomycin. <br />                                                                                    Nucleic Acids Research, 2008, Vol. 36,  19 6191 <br />  <br />  mutants promoted paromomycin-speci&#239;&#172;&#65533;c                            rpl10 alleles a&#239;&#172;&#8364;ected nonsense- <br /> changes growth (data shown), indicating                    suppression, control isogenic wild-type observed e&#239;&#172;&#8364;ects cell growth conferred                        upf1D cells. CYH2 pre-mRNA stabilized <br /> changes large subunit.                                               upf1D strain, mutants appeared sta- <br />                                                                             bilize species relative isogenic wild-type controls <br /> Effects translational fidelity                                           (Figure 3B), con&#239;&#172;&#65533;rming e&#239;&#172;&#8364;ects nonsense- <br /> Alterations ribosome structure a&#239;&#172;&#8364;ect &#239;&#172;&#65533;delity                  suppression. <br /> protein translation. Changes &#226;&#8364;&#8220;1 PRF shown interfere virus propagation altering ratio <br /> viral Gag Gag-pol proteins (34), altered &#226;&#8364;&#8220;1                     L10 N-terminal hook mutants affect 60S <br /> PRF frequencies possible reason underlying                     subunit biogenesis K  phenotypes rpl10 mutants. vivo dual-                    L10 protein incorporated large subunit <br /> luciferase reporter assay statistical analysis package                  required formation active ribosomes (50). <br /> (28,29) used quantitatively assess &#226;&#8364;&#8220;1 PRF                     translation initiation, 43S complex recruits <br /> wild-type RPL10, isogenic rpl10 strains. wild-type                   mRNA form 48S complex, cells, &#226;&#8364;&#8220;1PRF 8.3   0.16%. Surprisingly, &#226;&#8364;&#8220;1PRF e&#239;&#172;&#402;- <br />                                                                             60S subunit recruited. Insu&#239;&#172;&#402;cient levels 60S subunits, <br /> ciency strongly a&#239;&#172;&#8364;ected rpl10 mutant <br />                                                                             60S subunits de&#239;&#172;&#65533;cient ability form 80S ribo- <br /> strains (Figure 3A, white bars) suggesting &#226;&#8364;&#8220;1 PRF <br />                                                                             somes, tend promote accumulation 48S&#226;&#8364;&#8220;mRNA <br /> defects major cause K  phenotypes. <br />                                                                             complexes, additional mass results intermedi- dual luciferase assay used quantitatively <br /> access e&#239;&#172;&#8364;ects rpl10 alleles ability ribo-              ate sedimentation coe&#239;&#172;&#402;cients monitored appear- <br /> somes recognize termination codons (nonsense suppres-                    ance &#226;&#8364;&#732;halfmers shoulders&#226;&#8364;&#8482; denser sides sion). analysis revealed strong allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects                  polysome tracings. L10 mutants a&#239;&#172;&#8364;ecting inclusion (Figure 3A, gray bars). nonsense suppression                      protein large subunits shown pro- <br /> 0.27%   0.01% wild-type cells, stimulated                      mote appearance halfmer shoulders poly- <br />  1.5-fold R7L mutant                        pro&#239;&#172;&#65533;les (21), 60S ribosomal subunit biogenesis <br /> mutants position. contrast Y9 mutants                 defects linked defects killer virus main- <br /> appeared promote hyperaccurate termination codon                         tenance (44). Sucrose gradient analyses revealed recognition, decreasing rates nonsense-suppression                    Y9H Y11C mutants promoted formation 1.8-fold (Y9C) 2.5-fold (Y9N). translational &#239;&#172;&#65533;del-                  halfmers (Figure 3C), greater amplitude ity data summarized Table 1.                                         previously observed temperature sensitive mutants <br />    previously shown grc5-1 allele                   (21). Additionally, observation decreased amplitude RPL10 promoted defective turnover nonsense-contain-                      60S relative 40S peaks, 80S relative 40S ing mRNAs (22). bicistronic reporters                          60S peaks, increase material near <br /> internally control apparent changes translational                    gradient associated 60S sub- <br /> &#239;&#172;&#65533;delity e&#239;&#172;&#8364;ects mRNA stability (28,49),                         unit biogenesis defects. e&#239;&#172;&#8364;ects observed exclude possibility determine                   R7L, R7Q, Y9C, Y9H, Y9N Y11C mutants (sum- new rpl10 alleles a&#239;&#172;&#8364;ected nonsense-mediated                      marized Table 1). results point 60S subunit <br /> mRNA decay, levels ine&#239;&#172;&#402;ciently spliced CYH2                          ribosome biogenesis defects likely cause <br /> precursor-mRNA assayed northern blot analy-                         inability rpl10 mutants maintain sis using total RNA extracted cells expressing                         killer virus. <br />  <br />  <br /> Table 1. Summary L10 N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants examined study <br />  <br /> Strain       Percentage             Nons. Supp.             Anisoc             aa-tRNA Kd,           eEF2 Kd, nM           Ribo biogenesis <br />               1PRF (fold-WT          (fold-WT)b                                 nM (fold-WT)d         (fold-WT)e            defectsf <br />  <br /> WT           <br /> R7L        <br /> R7P           <br /> R7Q           <br /> Y9C          <br /> Y9H       <br /> Y9N            <br /> Y11C          <br />  <br />   Denotes standard errors    E&#239;&#172;&#402;ciency  1 ribosomal frameshifting. Numbers parentheses denote fold wild-type levels  <br /> b <br />   Percent rate suppression frame UAA codon. <br /> c <br />   Resistance hypersensitivity anisomycin relative wild-type. <br /> d <br />   Dissociation constants ribosomes aa-tRNA eEF1A GTP Ternary Complex (nM). <br /> e <br />   Dissociation constants ribosomes eEF2 (nM). <br /> f <br />  Indicates e&#239;&#172;&#8364;ects indicated L10 mutants ribosome biogenesis/subunit joining. <br />  6192 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br />                                              1.8 <br />                                                                              -1PRF     Nons Supp <br />                                                1.6 <br />                                                                                                              B <br />                                                1.4                                                                  f1 T 7L 9C 9H <br />                                                                                                                   W R Y Y <br />  <br />                           Recoding (Fold-WT) <br />   <br />                                                 0 <br />                                                      WT   R7L    R7P    R7Q      Y9C    Y9H     Y9N   Y11C <br />  <br />  <br />                           C WT              R7L                                                 R7P                 R7Q <br />                               40S 80S                                           halfmers               halfmers             halfmers <br />                                60S polysomes <br />  <br />  <br />  <br />  <br />                                                Y9C                     Y9H                 Y9N                     Y11C <br />                                                       halfmers                                          halfmers <br />  <br />  <br />  <br />  <br /> Figure 3. L10 N-terminal hook mutants promote allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects nonsense codon recognition ribosome biogenesis/subunit joining. <br />   Allele-speci&#239;&#172;&#65533;c e&#239;&#172;&#8364;ects nonsense codon recognition programmed &#226;&#8364;&#8220;1 ribosomal frameshifting (&#226;&#8364;&#8220;1 PRF). Isogenic yeast cells expressing wild-type mutant forms L10 transformed dual luciferase reporter control plasmids (28) rates translational recoding determined. &#226;&#8364;&#8220;1 PRF indicates programmed &#226;&#8364;&#8220;1 ribosomal frameshifting promoted yeast L virus frameshift signal. Nons-supp. denotes ability ribosomes able suppress frame UAA termination codon located Renilla &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase reporter genes. <br /> Relative di&#239;&#172;&#8364;erences recoding e&#239;&#172;&#402;ciencies mutants depicted fold wild-type. Error bars denote standard error previously described <br /> (29). (B) mutants a&#239;&#172;&#8364;ect nonsense-mediated mRNA decay. northern blot total RNA extracted isogenic wild-type indicated <br /> L10 mutants probed [32P]-labeled synthetic oligonucleotide complementary yeast CYH2 coding sequence previously described <br /> (62). Locations unspliced CYH2 pre-mRNA mature CYH2 mRNA indicated. (C) L10 mutants promote strong 60S <br /> biogenesis subunit joining defects. Cytoplasmic extracts isogenic strains loaded 7&#226;&#8364;&#8220;47% sucrose gradients, centrifuged SW41 <br /> rotor 40 000 r.p.m. 180 min 48C, fractionated analyzed continuous monitoring A254 (63). locations 40S, 60S, 80S, polysome <br /> fractions halfmers labeled. presence halfmers typically subunit joining defects lower abundance 60S subunits consequence 60S biogenesis defects. <br />  <br />  <br /> Ribosome biochemistry                                                                         e&#239;&#172;&#8364;ects parameters. Y9C mutant pro- <br />                                                                                               moted  8-fold increase Kd aa-tRNA, Previous studies laboratory using mutants                                                                                                50% decrease Kd eEF2. Conversely, Kds ribosomal protein L3 demonstrated inverse relation- <br />                                                                                               Y11C ribosomes aa-tRNA 0.86- 1.54-fold ship a&#239;&#172;&#402;nity ribosomes aa-tRNA                                                                                               wild-type. signi&#239;&#172;&#65533;cant changes peptidyltransferase <br /> eEF2 (9). Equilibrium dissociation constants deter- <br />                                                                                               activities observed wild-type mutant <br /> mined aa-tRNA using [14C]Phe-tRNA non-salt- <br />                                                                                               ribosomes (data shown). <br /> washed ribosomes, eEF2 monitoring ability diphtheria toxin <span id='am-38' about='rdfs:label' typeof='owl:Thing'>label</span> ribosome bound eEF2 [14C]NAD+. inverse relationship aa-tRNA                                           Chemical protection studies: Y9C Y11C forms eEF2 a&#239;&#172;&#402;nities observed L10 mutants                                      L10 produce profound local long-distance changes (Figure 4, Table 1 Supplementary Figure 1).                                        25S rRNA structure <br /> cases, increased a&#239;&#172;&#402;nity aa-tRNA matched                                            light strong opposing e&#239;&#172;&#8364;ects aa-tRNA <br /> decreased a&#239;&#172;&#402;nity eEF2 vice versa. particular,                                      eEF2 binding, e&#239;&#172;&#8364;ects Y9C Y11C Y9C Y11C mutants showed dramatically opposing                                         mutants 25S rRNA structure probed using                                                                                                                Nucleic Acids Research, 2008, Vol. 36,  19 6193 <br />  <br />  <br />                             800                                            20                           L10 rRNA helical structures. <br />                             700                                            18                           Interestingly, changes H38 H89 similar                                                                                                         promoted rRNA mutants located Helix 38 (7). <br />  aa-tRNA binding (Kd, nM) <br />  <br />  <br />  <br />  <br />                             600                                            16 <br />  <br />  <br />  <br />  <br />                                                                                 eEF2 binding (Kd, nM) <br />                                                                                                         additional series long-distance changes observed <br />                             500                                            14 <br />                               ~ <br />                             300                         ~                  12 <br />                                                                                                         extending line base peptidyltrans- <br />                                                                                                         ferase center (G2813&#226;&#8364;&#8220;G2815) Helix 74                             250                                            10 <br />                                                                                                         Helix 88: discussed  follows path taken <br />                                                                                                         30 end deacylated tRNA moves                             200                                            8                            P-site E-site exits ribosome. a&#239;&#172;&#8364;ected <br />                             150                                            6                            bases cluster tend hyperprotected com- <br />                             100                                            4                            pared wild-type ribosomes, suggesting Y11C <br />                             50 <br />                                                                                                         promoted disruption local interactions L10 <br />                                                                            2 <br />                                                                                                         Helices 38 89 caused collapse                              0                                             0                            distantly located structures. pattern simi- <br />                                                    7Q <br />   <br />                                                                                                         lar observed rRNA mutant, 	2922C, <br />                              located base Helix 92 loop (11). <br /> Figure 4. inverse relationship ribosome binding                                     Additionally, base tip Helix 91 (A2901, <br /> aa-tRNA eEF2. Dissociation constants generated analysis                                     E. coli U2583), abuts SRL, hyperpro- single-site-binding isotherms eEF-1A stimulated binding [14C]Phe-tRNA ribosomal sites presence GTP <br />                                                                                                         tected mutant. Attention directed speci&#239;&#172;&#65533;c <br /> (aa-tRNA, gray bars), eEF2 (black bars) determined                                        bases.  U2860 (E. coli U2483) helps form monitoring fraction [14C]-<span id='am-14' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span>ADP</span>-ribosylation ribosome asso-                                     &#239;&#172;&#65533;rst &#226;&#8364;&#732;gate&#226;&#8364;&#8482; aa-tRNA passes <br /> ciated eEF2 diphtheria toxin. Error bars denote standard deviation.                                  process accommodation (12). Second, equilibrium binding curves data generated shown Supplementary Figure 1. <br />                                                                                                         A2818 (E. coli A2450) located core peptidyl- <br />                                                                                                         transferase center.  G2815 (E. coli G2447) hyper- <br />                                                                                                         protected mutant discussed context base-speci&#239;&#172;&#65533;c solvent-accessible reagents: dimethylsulfate                                               anisomycin hypersensitivity. Fourth, G2777 (E. coli <br /> (DMS), kethoxal carbodiimide metho-p-toluenesulfo-                                                  A2406) normally base-paired C311 (E. coli <br /> nate (CMCT). rRNAs extracted, modi&#239;&#172;&#65533;ed bases                                                    U416) helix 22 deprotected, C2775 identi&#239;&#172;&#65533;ed primer extension using reverse tran-                                                  (E. coli U2404) displayed enhanced line cleavage pat- <br /> scriptase detect methylation N3 position uri-                                              tern, suggesting opening proposed E-site gate <br /> dines N1 position guanosines (CMCT),                                                  responsible movement tRNA P-site N1 N2 positions guanosines (kethoxal),                                                E-site release deacylated tRNA ribosome. <br /> N1 position adenosines N3 position cytidines                                                     Examination e&#239;&#172;&#8364;ects Y9C mutant 25S <br /> (DMS) (51). primers used designed probe                                                     rRNA chemical protection patterns reveals distinctly <br /> functional regions domain V (including site                                                di&#239;&#172;&#8364;erent picture marked local changes P-site loops, PTC, helices adjacent                                                aa-tRNA accommodation corridor (Figure. 5A, <br /> structures, e. Helices 73, 74, 88&#226;&#8364;&#8220;93), Helices 94&#226;&#8364;&#8220;96                                              C E). Proximal mutant amino acid, U2828 <br /> Helix 38 (9,11).                                                                                        (E. coli U2460) hyperprotected, suggesting    Autoradiograms generated experiments                                                  interaction L10 signi&#239;&#172;&#65533;cantly altered. <br /> shown Figure 5A. Figure 5B C 2-dimensional                                                  Deprotection bases Helix 38 (A1947, A1048) maps relevant regions yeast 25S rRNA                                                     Helix 89 (A2844, A2850) appear interact nucleotides having altered chemical protection pat-                                               parts L10 suggests local change terns Y11C Y9C mutants mapped.                                                     radiated outward mutated residue disrupt- information turn used map modi&#239;&#172;&#65533;ed                                                    ing interaction L10 helices. <br /> bases structure yeast ribosome (2)                                                   Hyperprotection bases aa-tRNA accommo- <br /> Figure 5D E. chemical protection patterns                                                   dation corridor, particular U2861 (E. coli U2492) mapped &#239;&#172;&#65533;ve additional high-resolution ribo-                                                   U2923 (E. coli U2555), extending outward regions structures (Supplementary Figure 2).                                                   Helix 95, suggests collapse structure signi&#239;&#172;&#65533;cant di&#239;&#172;&#8364;erences observed locations                                                 corridor. Finally, strong deprotection A2818 bases chemical modi&#239;&#172;&#65533;cation changes, discussed                                                   A2819 (E. coli A2450 A2451) indicates structural  analysis did reveal di&#239;&#172;&#8364;erences locations                                             changes peptidyltransferase center. possible sig- N-terminal hook domain. Examination data                                                  ni&#239;&#172;&#65533;cance chemical protection patterns observed reveals mutants confer signi&#239;&#172;&#65533;cant changes                                              Y9C Y11C mutant ribosomes discussed  25S rRNA structure. Y11C mutant marked signi&#239;&#172;&#65533;cant changes radiating outward parallel L10 site mutation Helices 89 38                                               DISCUSSION <br /> expansion segment 12, present atomic                                                study describes generation characterization <br /> resolution structure) (Figure 5A, B D). a&#239;&#172;&#8364;ected                                                 new set rpl10 yeast alleles based ability bases structures deprotected, suggest-                                              promote loss Killer phenotype. mutations pre- <br /> ing Y11C mutant destabilizes interactions                                                      dominantly clustered regions. mutants <br />  6194 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br /> Figure 5. Structure probing wild-type mutant ribosomes.   Autoradiograms reverse transcriptase primer extension reactions spanning <br /> sequences helix 38 expansion segment 12, helices 88&#226;&#8364;&#8220;92 helices 94&#226;&#8364;&#8220;96. Sequencing reactions (left sides panels) labeled corresponding rRNA sense strand. Mutants indicated  bases altered chemical protection patterns indicated left. Atop panel lane, &#195;&#184; indicates untreated ribosomes, D DMS, C denotes CMCT K stands kethoxal. Open arrowheads indicate bases hyperprotected chemical attack relative wild-type, &#239;&#172;&#65533;lled arrowheads denote bases susceptible chemical attack. black <br /> arrow C2775 denotes increased line cleavage site Y11C mutant. Yeast 25S rRNA base numbering used  (B C) <br /> Localization bases yeast 25S rRNA modi&#239;&#172;&#65533;cation patterns a&#239;&#172;&#8364;ected Y11C (B) Y9C (C) mutants. Protected deprotected bases indicated open &#239;&#172;&#65533;lled arrowheads, respectively, arrow C2775 shows enhanced line cleavage. (D E) <br /> rRNA protection data mapped yeast large subunit crystal structure threaded H. marismortui (2). Base numbering follows S. cerevisiae sequence shown (B) (C). Hyperprotected bases colored gray, deprotected bases shown black. panel E, A2818 <br /> (E. coli A2451) colored dark ochre. path taken 30 end aa-tRNA accommodates large subunit denoted orange <br /> arrows, path takes peptidyltransferase center traced green arrow (D). <br />                                                                         Nucleic Acids Research, 2008, Vol. 36,  19 6195 <br />  <br />  identi&#239;&#172;&#65533;ed N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; region composed          particular regard coordination elongation residues R7, Y9 Y11. residues        factor interactions. Speci&#239;&#172;&#65533;cally, Y9C mutant, hyper- <br /> tact PTC-proximal bulge H89 nearby            protection Gate 1 bases (U2860 U2923, E. coli <br /> residues. second group, A64, K74, N144, K145             U2492 U2555), accompanied increased a&#239;&#172;&#402;nity L152, appear interact H89. Group appears        eEF2 decreased a&#239;&#172;&#402;nity aa-tRNA consistent <br /> directly contact close proximity bulge      &#226;&#8364;&#732;closed&#226;&#8364;&#8482; accommodation corridor/eEF2-binding <br /> H38 includes mutants positions (K15,            site. Conversely Y11C mutant, deprotection Y17, V26, Q59, P93 F94). particular, K15              U2861 (E. coli U2493) coupled increased a&#239;&#172;&#402;nity <br /> Y17 link Helices 38 39, appearing make           aa-tRNA decreased a&#239;&#172;&#402;nity eEF2 consistent <br /> tact structures. fourth group located        &#226;&#8364;&#732;open&#226;&#8364;&#8482; corridor/TC-binding conformation. C-terminal region L10, structure          addition, deprotection G2777 (E. coli A2406) currently unresolved. tip H38,         enhanced line cleavage C2775 (E. coli U2404) sug- <br /> region protein forms extension approaches         gests &#226;&#8364;&#732;E-site gate&#226;&#8364;&#8482;, deacylated L7/L12 stalk. attribute K40 G81positions           tRNA passes way P-site E-site group topologically represent base      ribosome (39,54&#226;&#8364;&#8220;57), open extension. Consistent previous report (46),       mutant. e&#239;&#172;&#8364;ects mutants ribosome struc- <br /> mutations unstructured loop L10        ture entire path traversed tRNAs thought closely approach PTC, indi-          elongation cycle leads propose L10 plays rectly suggesting loop important     important role coordinating tRNA movement ribosome biogenesis, peptidyltransfer translational       large subunit. <br /> &#239;&#172;&#65533;delity (unlikely), important mutants region inviable  likely). analysis   Anisomycin resistance/hypersensitivity mutants identi&#239;&#172;&#65533;ed current study      ribosome structure <br /> scope single manuscript.  chose focus deeply located N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482;            L10 mutants examined current study displayed <br /> region L10 current study order establish       range phenotypes related anisomycin, ranging connection ribosomal protein L3  accompanying           strong drug resistance (R7L, R7P Y9C) hypersen- <br /> manuscript Meskauskas  Dinman,J.D.)             sitive (Y9H, Y9N Y11C). Anisomycin resistance emphasis ribosome structure informs function.             ribosomal protein L3 mutants previously proposed                                                                  opening accommodation corridor, <br />                                                                  increasing di&#239;&#172;&#8364;usion rates relatively large aa-tRNA <br /> Ribosome structure elongation factor interactions: <br />                                                                  PTC relative smaller drug (6,9). allosteric model <br />                                                                  model explain observations L10 TC eEF2 structural mimics             mutants current study, Y9C (corridor closed) bind essentially site ribosome (1).       drug resistant Y11C (corridor open) hypersen- does ribosome coordinate interactions           sitive. recent studies point changes ligands correct phases elonga-      drug-binding site responsible anisomycin resis- <br /> tion cycle? Dontsova group used mutants Helix 42          tance (11,58). Mutation E. coli C2452 U (yeast E. coli demonstrate activation EF-G            C2820U, H. marismortui 2487U) confers strong ani- <br /> involve allosteric signaling peptidyltransferase        somycin resistance. Chemical protection studies center GAC, di&#239;&#172;&#8364;erence            yeast mutant showed anisomycin unable pro- <br /> EF-G TC-binding sites di&#239;&#172;&#8364;erences            tect base chemical attack, suggesting aniso- positioning GAC relative SRL (52,53).          mycin interact U position. X-ray subsequently proposed complementary model sug-              crystal structure analogous H. marismortui mutant <br /> gesting ribosomal protein L3 functions &#226;&#8364;&#732;gate-        con&#239;&#172;&#65533;rmed model. current study, did keeper site&#226;&#8364;&#8482; scheme (9). model         detect changes sensitivity base chemi- expanded re&#239;&#172;&#65533;ned accompanying                 cal modifying agents. increased sensitivity manuscript. studies propose L3 plays role           50 adjacent bases (A2818 A2819, E. coli A2450 coordinating orderly binding elongation fac-       A2451) account resistance Y9C <br /> tors peptidyltransferase activity participating      mutant drug. does address question series local allosteric changes rRNA structure.            Y11C, A2818 (E. coli A2450) depro- <br /> Speci&#239;&#172;&#65533;cally proposed &#226;&#8364;&#732;open&#226;&#8364;&#8482; aa-tRNA accom-           tected, anisomycin hypersensitive. potential <br /> modation corridor conformation de&#239;&#172;&#65533;nes TC-binding             answer gleaned anisomycin-resistant <br /> site, &#226;&#8364;&#732;closed&#226;&#8364;&#8482; conformation favors eEF2 binding.       G2447C G2447U H. marismortui mutants (58). Examination atomic scale ribosome structures              base contacts anisomycin K+, reveals L10 lies opposite accommo-       mutants cause drug resistance altering position dation corridor PTC L3 (e.g. Figure 5D),            <span id='am-20' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-21' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span>. yeast, corresponding base G2815; data accompanying manuscript indicate            base 50 neighbors deprotected Y11C <br /> bases H89 interact N-terminal hook           mutant (Figure 5). suggest mutant result <br /> L10 involved process. data current      change region anisomycin-binding <br /> study closely complement studies L3,           pocket confers increased a&#239;&#172;&#402;nity anisomycin, <br />  6196 Nucleic Acids Research, 2008, Vol. 36,  19 <br />  <br />  <br /> resulting drug hypersensitivity. idea supported     impact discrete physical pathways information increased a&#239;&#172;&#402;nity Y11C aa-tRNA, 30 end        &#239;&#172;&#8218;ow ribosome. partially overlaps anisomycin-binding           Lastly, L10 involved process ribosome ratch- <br /> pocket.                                                        eting pre- post-translocational states <br />                                                                (61)? Comparison L10 (bacterial L16, archae <br /> Ribosome structure, translational fidelity virus           L10e) N-terminal regions seven di&#239;&#172;&#8364;erent high- <br /> maintenance                                                    resolution ribosome structures suggest assume L10 &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants strong e&#239;&#172;&#8364;ects ter-       di&#239;&#172;&#8364;erent conformations: interacting Helix 89 mination codon recognition (Figure 3A). speculate      Helix 38  Supplementary Figure 2). possible explained open/closed nature accommo-        interpretation comparisons hook inter- <br /> dation corridor. closed conformation, e.g. Y9C,      acts Helix 89 site unoccupied  post- <br /> decrease aa-tRNA accommodation rates, providing           translocation state), interacts Helix 38 time ribosomes reject suppressor tRNAs       site occupied (pre-translocation state). possi- <br /> proofreading phase translation elongation (59).             ble N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; L10 participate Conversely, &#226;&#8364;&#732;open&#226;&#8364;&#8482; mutants (Y11C, R7L)         process interactions PTC prox- <br />                                                                imal loop Helix 89 (post-state) Helix 38 (pre-state). inhibit proofreading allowing faster accommoda- <br />                                                                model, interaction L10 hook bases tion incorrect tRNAs. &#226;&#8364;&#8220;1 PRF best                                                                PTC-proximal bulge Helix 89 help open marginally a&#239;&#172;&#8364;ected mutants examined study,                                                                aa-tRNA accommodation corridor. site tRNA likely killer virus maintenance defects                                                                occupancy, L10 hook &#239;&#172;&#8218;ip interact L10 N-terminal &#226;&#8364;&#732;hook&#226;&#8364;&#8482; mutants strong <br />                                                                H38, releasing H89 gate base interact  <br /> e&#239;&#172;&#8364;ects 60S biogenesis previously described (44,45). <br />                                                                ner base Helix 92, controlled L10 information flow ribosome                  L3 &#226;&#8364;&#732;rocker switch&#226;&#8364;&#8482;  accompanying manuscript                                                                Meskauskas  Dinman,J.D.). model  <br /> Inspection Figure 5D shows L10 lies plane      Y9C mutant thermodynamically favor formed Helix 38, Helix 39/L10, Helix 89, Helix 91       closed conformation Y11C drive Helix 95. compared &#239;&#172;&#65533;ve &#239;&#172;&#65533;ngers, L10      open state. Alternatively, possible proteinaceous prosthesis Helix 39. addition, L10       switch exists, di&#239;&#172;&#8364;erences atomic resolu- <br /> interacts D-loop 5S rRNA (data shown).         tion structures phylogenetic di&#239;&#172;&#8364;erences nature select removable protein             (eukaryotes archae versus bacteria), di&#239;&#172;&#8364;er- <br /> occupy space simply extending Helix 39?       ent crystallization conditions. answers ques- <br /> Examination placement reveals L10        tions await availability high-resolution structures center topological nexus interacts  5S          eukaryotic ribosomes bacterial ribosomes com- <br /> rRNA, connecting B1b B1c intersubunit       plexed EF-G. <br /> bridges crown large subunit; GTPase- <br /> associated center, connecting elongation factor acti- <br /> vation; site &#239;&#172;&#65533;nger (Helix 38), connecting     <br /> decoding center B1a bridge; peptidyl-     <br /> transferase center. suggest information &#239;&#172;&#8218;ow region large subunit complex monitored simple rRNA helix, necessitating         evolution informationally complex mole-         <br /> cule protein occupy site.  suggest   L10 functions collect, distribute coordinate     <br /> information large subunit.                      <br />    observed e&#239;&#172;&#8364;ects L10 mutants ribosome      <br /> structure function explained consequent         <br /> 60S mis-assembly defects. notion supported        <br /> studies prokaryotic ortholog L16 showing         important late assembly steps    <br /> involved ribosomal functions (60).  support model, note mis-assembly major issue, Y9C Y11 C mutants           displayed similar changes chemical protection pat-     <br /> terns. case. Instead, chemical protec-    <br /> tion patterns observed Y9C mutants closely      <br /> resembled observed various mutants riboso-         <br /> mal protein L3 [(9) accompanying manuscript],       Y11C mutants closely resembled      <br /> observed mutants Helix 38 loop (7,11), <br /> suggesting Y9C Y11C mutants independently     <br /> </body></html>