<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Bioinformatics                                                                                                                       BioMed Central <br />  <br />  <br />  <br /> Database                                                                                                                               Open Access PATO-compliant zebrafish screening database (MODB): <br /> management morpholino knockdown screen information <br /> Michelle N Knowlton*1,6, Tongbin Li2, Yongliang Ren2, Brent R Bill1, <br /> Lynda BM Ellis3,4 Stephen C Ekker1,4,5 <br />  <br /> Address: 1Department Genetics, Cell Biology Development, University Minnesota, Minneapolis, MN 55455, USA, 2Department Neuroscience, University Minnesota, Minneapolis, MN 55455, USA, 3Department Laboratory Medicine Pathology, University Minneapolis, Minneapolis, MN 55455, USA, 4University Minnesota Arnold Mabel Beckman Center Transposon Research, Minneapolis, <br /> MN 55455, USA, 5Department Biochemistry Molecular Biology, Mayo Clinic, Rochester, MN 55905, USA 6Box 45, Jackson <br /> Laboratory, 600 Main Street, Bar Harbor, Maine 04609, USA <br /> Email: Michelle N Knowlton* - mnk@informatics.jax.org; Tongbin Li - toli@biocompute.umn.edu; <br /> Yongliang Ren - yongliang@biocompute.umn.edu; Brent R - bill0115@umn.edu; Lynda BM Ellis - lynda@umn.edu; <br /> Stephen C Ekker - ekker.stephen@mayo.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 7 January 2008                                                    Received: 28 April 2007 <br />                                                                              Accepted: 7 January 2008 <br /> BMC Bioinformatics 2008, 9:7   doi:10.1186/1471-2105-9-7 article available  http://www.biomedcentral.com/1471-2105/9/7 <br /> &#194;&#169; 2008 Knowlton et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: zebrafish powerful model vertebrate amenable high throughput vivo <br />                  genetic analyses. Examples include reverse genetic screens using morpholino knockdown, <br />                  expression-based screening using enhancer trapping forward genetic screening using <br />                  transposon insertional mutagenesis. created database facilitate web-based <br />                  distribution data genetic studies. <br />                  Description: MOrpholino DataBase MySQL relational database online, PHP <br />                  interface. Multiple quality control levels allow differential access data raw finished formats. <br />                  MODBv1 includes sequence information relating 800 morpholinos targets                  phenotypic data regarding dose effect morpholino (mortality, toxicity defects).                  improve searchability database, incorporated fixed-vocabulary defect <br />                  ontology allows organization morpholino affects based anatomical structure <br />                  affected defect produced. allows comparison species utilizing Phenotypic <br />                  Attribute Trait Ontology (PATO) designated terminology. MODB cross-linked ZFIN, <br />                  allowing searches databases. MODB offers users ability retrieve <br />                  morpholino data sequence morpholino target, target, anatomical structure <br />                  affected defect produced. <br />                  Conclusion: MODB data used functional genomic analysis morpholino design                  maximize efficacy minimize toxicity. MODB serves template future sequence- <br />                  based functional genetic screen databases, currently used model creation <br />                  mutagenic insertional transposon database. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 10 <br />                                                                                                                  (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Background                                                     pholinos. Ensembl limited ability store dis- <br /> Advances scientific technology availability      play experimental data, ZFIN contains wealth <br /> genomic sequence increasing number species           information published results lacks specific infor- led paradigm shift identifying sequence      mation dosage searchable phenotype data gene product function. Large functional genetic screens        published morpholinos phenotypic data largely increasingly prevalent require new          unpublished morpholinos. <br /> way thinking data facilitate cross-species com- <br /> parison deciphering gene product function differ-       model organism communities developed <br /> ent organisms. Zebrafish particularly amenable          online databases access results mutagenic, knock- <br /> large screens tremendous ability repro-      expression studies. mouse <br /> duce, embryos finish development         knockout mutant [15,16] C. elegans RNAi [17,18] <br /> organ rudiments 2 days post-fertilization [1].        experimental result databases phenotypic content <br /> semi-transparent embryos ready availability         exist, morpholino-specific database available transgenic lines facilitate functional annotation       pilot Xenopus morpholino screen 202 genes genes required numerous organ systems             targeted subsequent phenotypes displayed pathways. historically common use          webpage [19,20]. Xenopus tropicalis Morpholino <br /> genetic screenings primarily restricted ENU        Screen database flat format website allows mutagenesis screens [2-4] retroviral mutagenesis [5],      records accessed viewed gene targeted, syn- availability morpholinos [6], antisense knockdown       phenotype group specific defect. Notably database <br /> oligonucleotides, insertional mutagenic transposons        does include search functions does offer [7] opened new avenues functional genetic             means input outside sources.  research.                                                      intent database stated authors display <br />                                                                results specific screen [19,20]. <br /> Recently, morpholino screens pub- <br /> lished collaboration Ekker laboratory [8,9].       Given intent database display large largest  Secretome Screen [9,10],           data generated going collabora- <br /> identified likely secreted genes available transcrip-     tive screens, developed scalable, centralized <br /> tome databases using bioinformatics approach. Ver-       easily accessible worldwide, MOrpholino DataBase <br /> tebrate Secretome DataBase (VSDB) [4] established       (MODB) [3] store facilitate retrieval zebrafish <br /> identify translationally translocated (CTT) proteins     morpholino screening results. Additional access model organism species [10]. Using combina-          provided active links MODB ZFIN. <br /> tion signal sequence, transmembrane domain, ini- <br /> tiation site predictors homology known           MODB scalable, searchable readily adaptable <br /> secreted proteins, subset zebrafish CTT genes        online MySQL database PHP interface. MODB <br /> identified (0.3&#195;&#8212; genome coverage based zebrafish            includes storage retrieval results generated loaded predicted protein sequences CTTome           screens public access published unpublished <br /> database Genescan gene prediction estimates         information morpholinos MODB allows dis- <br /> zebrafish genome [4,5]). 150 gene targets selected     semination target-specific information effect random morpholino design, carried using             knockdown, facilitates analysis morpholino- <br /> Assisted MOrpholino Design tool (AMOD) [11,12].           specific data toxicity, dosage, mortality gene knocked zebrafish morpholino-             design. MODB new, online, publicly accessible data- <br /> injection screened defects general morphology      base data-mining zebrafish morpholino knockdown organ systems including ocular, renal, pig-        effects morpholino non-specific effects. <br /> ment, vascular, hematopoietic, cranio-facial, motor <br /> [9]. second morpholino knockdown screen, Hemat-          Construction Content <br /> opoietic Stem Cell Screen, conducted using zebrafish       Construction <br /> homologues human genes differentially expressed          MODB web project currently running Apache <br /> hematopoietic stem cell differentiation microarray          web server [21] PHP [1] Linux (Intel <br /> analysis [8].                                                  Xeon, 2.8 GHz, CentOS release 4.4). project architec- <br />                                                                ture developed according Model-View-Control- <br /> Currently, zebrafish community stores published        ler (MVC) design paradigm. Model (database tables) <br /> sequence-related results web-based databases.       stored MySQL relational database [22]. Major tables principal database zebrafish-related research   include Morpholino, Staff, Lab, Test, Test_Result Zebrafish Information Network (ZFIN) [13,14]. Ensembl          store information pertinent morpholino [5], likewise, stores sequences targeted published mor-     users (Figure 1). View developed help  <br />  <br />                                                                                                                    Page 2 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                    http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 1 <br /> Entity-Relationship diagram MODB <br /> Entity-Relationship diagram MODB. organization data contained MODB centered unique mor- <br /> pholino sequence. Tables pertaining morpholino-specific information located upper left corner (solid box). tables lower left corner establish relations laboratories, users security level designation (dashed <br /> line box). tables right half diagram represent tests test components anatomical structure, <br /> screen defect observed (unboxed area). <br />  <br />  <br />  <br />  <br /> Smarty [23], template engine PHP helps             anatomy browser written PHP incorporated designer make modifications layout break-         MODB. <br /> ing PHP logic. PHP scripts control connections Model View. GD graphics library          Content <br />  PHP module) [24] used graphic functions dis-       unique identifier MODB sequence played summary page.                                      morpholino. morpholino targets screens                                                                  derived transcriptome sequence data Insti- <br /> NCBI BLAST 2.2.13 [25] invoked PHP sys-            tute Genomic Research (TIGR) [27] GenBank [28] <br /> tem calls sequence search functions. MODB             subject change new sequence data BLAST target database updated automatically            available. unique morpholino entered new morpholino sequence entered database.            database, accompanying set tables devoted ZFIN anatomy ontology [26] downloaded                    information pertaining morpholino (release <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> 08/16/2007) ZFIN web site           designator, laboratory created  contact person, <br /> OBO format [26] transferred database.            target  target sequence, gene targeted, reference <br />  <br />  <br />                                                                                                                      Page 3 10 <br />                                                                                                (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                   http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 2 <br /> Morpholino-specific information <br /> Morpholino-specific information. Information morpholino sequence, laboratory morpholino designa- <br /> tor (box number), creating laboratory, initiating user reference images associated morpholino displayed initial morpholino page morpholino. <br />  <br />  <br />  <br />  <br /> images number tests created (Figure 2).         tests, caused toxic effects high cell <br /> cise format allows key information regarding mor-           death mortality, produced pleiotropic effects pholino viewed                                    decipher. information use sci- <br />                                                                 entific community optimizing morpholino design Early 2007, MODB contained 700 unique mor-              learning dosage data overall effect previously <br /> pholino sequences generated screens, indi-         designed tested morpholinos. <br /> vidual projects control morpholinos. generation Secretome Screen [9] largest overall      MODB living database, new data regularly <br /> tributor morpholino sequences. 150 candidate            submitted. coming year 50 150 <br /> genes sequences, 209 morpholinos designed.            morpholinos added database contin- <br /> morpholino subjected variety morphological          uing Hematopoietic Stem Cell secretome screens. molecular assays determine effect reg-       allow release data defined formats quality, <br /> ulating particular gene product 5-days    MODB data sharing quality control levels. Users <br /> zebrafish development [9]. Secretome Screen identi-        capable accessing modifying data related fied 26/150 gene targets produced reproducible             designated morpholinos. facilitate collaborative <br /> effects specific aspects early development early       efforts, shared designation developed allow <br /> morphology variety organ systems including vis-        members consortium laboratories access data related <br /> ual, renal, cardiovascular, cranio-facial, hematopoiesis,       morpholinos designed user laborato- pigment. focused Hematopoietic Stem                ries connected relationship MODB. <br /> Cells Screen identified 16/64 targets necessary    designation (&apos;public&apos;) allows morpholino data <br /> proper blood development [8]. bulk data              used generate summary reports available entered MODB (124/150 target gene morpholinos           open scientific community MODB website. Secretome Screen 48/64 Hematopoietic             Nearly morpholinos (779/784) MODB, Stem Cell Screen) consists morpholinos           exception non-zebrafish morpholinos, designated <br /> observed developmentally specific effect screening       public MODBv1. <br />  <br />  <br />                                                                                                                     Page 4 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 3 defect check box <br /> Common <br /> Common defect check box. Entry observations following morpholino injection zebrafish embryos cata- <br /> logued using fixed vocabulary checkbox form. Penetrance observed defect recorded subsequent analysis. <br />  <br />  <br />  <br />  <br />                                                                                                                  Page 5 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Figure 4 <br /> Correspondence    defect description PATO designation <br /> Correspondence defect description PATO designation. work ongoing establishing cross- <br /> species phenotypic vocabulary PATO, terminology constantly evolving. Defect names used maintain  <br /> sistency rooted PATO designations translation table. PATO term exists adequately defect observed, proposed term submitted NCBO ZFIN. <br />  <br />  <br />  <br /> Utility                                                       administrators enter new listings provide basic <br /> Data Entry                                                    information morpholino. Authorized users sub- major function MODB data compilation mor-          mit morpholino sequence results specific mor- <br /> pholino-specific information generated primary             phological molecular experiments given <br /> research. main ways add data. Currently,     observation point dose. Researchers apply  <br />  <br />                                                                                                                  Page 6 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> authorized-user identification password  author        search keyword, anatomical structure fixed defects <br /> contact information).                                          term allows future integration fixed- <br />                                                                vocabulary phenotype databases WormBase results called &apos;tests&apos;. early 2007, MODB      Mouse Phenome database [15,29]. utilize data <br /> tained 9000 individual test entries. free text notes    entered MODB meaningful manner, devel- <br /> field permits detailed description defects observed mor-    oped variety search functions. addition pholino-injected embryos fixed vocabulary defect        parameters mentioned (laboratory designator, <br /> term selected. defects ontology check box         morpholino  morpholino target  gene tar- established common scientific terms used       geted, laboratory created  keyword), MODB screening protocols (Figure 3). Specific laboratory sci-    queried controlled defects vocabulary. defects <br /> entific descriptions correspond established defects         ontology developed conjunction ZFIN ontology. reduction free text promotion      model current status Phenotypic Attribute Trait <br /> fixed vocabularies increases searchability MODB     Ontology (PATO) developed National Center  <br />  <br />  <br />  <br /> Figure search <br /> MODB   5      functions <br /> MODB search functions.  Data retrieval MODB simplified morpholino gene targeted. Results sorted morpholino gene targeted. B. additional search public interface allows searches based PATO <br /> term anatomical structure affected. C. additional sequence-based search using morpholino target sequence performed. <br />  <br />  <br />  <br />  <br />                                                                                                                    Page 7 10 <br />                                                                                              (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                 http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> Biomedical Ontology (NCBO) [30]. facilitate use        nology (Figure 4). Initially, PATO designed common language, MODB users assess phenotypes         defect observed anatomical structure  using defect descriptions linked current PATO termi-       entity, e.g. tail) using trait (e.g. relative length)  <br />  <br />  <br />  <br /> Figuresummary <br /> Public  6       page <br /> Public summary page. publicly accessible summary page available released morpholinos MODB. page  <br /> tains morpholino target information  sequence  review knockdown results dosing mortality normalcy. list tests performed, defects observed images displayed. <br />  <br />  <br />  <br />  <br />                                                                                                                  Page 8 10 <br />                                                                                            (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                  http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br /> attribute trait (e.g. short length).   information (reviewed [2,6,7,31-33]). Efficient data <br /> subsequently replaced entity quality          storage critical allow retrieval results mean- <br /> designation (e.g. tail, short length). MODB&apos;s defects          ingful fashion facilitate data mining. MODB ontology contains version PATO.          non-proprietary, relational database online accessi- <br /> PATO term exists adequately defect             bility acts storehouse critical information <br /> observed, proposed term submitted NCBO ZFIN        relating sequence-specific morpholino knockdown assist development cross-species phenotypic     screens. MODB  knowledge, unique ontology. importance common phenotypic                available databases offers searchable mor- <br /> vocabulary resides ability perform cross-species     pholino-specific data fixed-vocabulary defects  <br /> analysis gene function conserved pathways.              logical cataloging morphants allows cross- <br />                                                                species analysis using common, PATO-derived ontol- <br /> Data retrieval                                                 ogy. Cross-linking ZFIN provides additional value usefulness data stored database       MODB research community. <br /> subject retrieval methods available. MODB designed numerous search sort functions          Continued screening efforts zebrafish using mor- allow retrieval morpholino sequence, mor-          pholinos sequence-specific knockdown pholino gene targeted (Figure 5A).                 knockout technologies expand knowledge advanced search feature public interface allows         gene function. MODB provides central repository users search anatomy phenotype identify           morpholino information. schema potentially <br /> morpholinos given defect (Figure 5B). sequence-       applicable sequence-based screening approach based search available, using target mor-   zebrafish related systems. example, MODB cur- <br /> pholino sequence sequence morpholino             rently adapted creation transposon data- <br /> (Figure 5C). offering numerous search sorting           base includes insertional mutagenesis expression <br /> options, results nearly 800 morpholinos       analysis data. <br /> 9000 tests accessed analyzed scien- <br /> tific insight knockdown effect specific mor-     Conclusion <br /> pholino target, genes effecting developmental         offering public access schema data  <br /> pathways, non-specific morpholino effects.                 tained MODB, hope aid advancement <br />                                                                functional genetic studies gene annotation MODB v1 Public Interface                                       improvements morpholino analysis design. facilitate display digestion large     dissemination valuable knowledge gained large <br /> data stored MODB, summary sheet mor-         screens Secretome [9] Hematopoietic <br /> pholino created  construction content)      Stem Cell [8] screens goes published gene target <br /> publicly available. morpholino summary contains           characterization provides foundation future <br /> sequence target information survival        research. <br /> normalcy (percent normal embryos observed overall specific screen types) curves based dose (Figure 6).    Availability requirements search functions used access infor-     MODB accessible web http://www.secre <br /> mation including gene targeted, BLAST                tomes.umn.edu/MODB/. added authorized <br /> advanced search function facilitates searching        user, contact corresponding author M.K. S.C.E. <br /> morpholinos PATO designation anatomical struc-          Morpholinos readily available purchase ture affected (Figure 5B). MODB presents users       Gene Tools [34] Open Biosystems [35]. authors valuable knowledge, mortality dose             affiliation, financial  cor- <br /> curves, effectiveness morphant phenotype. Currently,       porate entity. <br /> 101 morpholinos contain hyperlinks specific mor- <br /> pholino entries ZFIN hyperlinks provided        Authors&apos; contributions <br /> gene targets MODBv1.                                        MK designed, tested managed database. TL                                                                YR responsible programming, debugging Discussion                                                     informatics support. BB aided conception, design advent genome sequencing technology,           testing database. EL SE aided concep- <br /> greater emphasis gene function emerging. Func-           tion design database coordinated tional genomic screening tools available zebrafish     project  authors read approved final ENU mutagenesis, GripNAs, RNAi, insertional muta-           manuscript. <br /> genesis morpholinos allow large functional gene <br /> screens produce enormous biological <br />  <br />  <br />                                                                                                                     Page 9 10 <br />                                                                                               (page number citation purposes) <br />  BMC Bioinformatics 2008, 9:7                                                                       http://www.biomedcentral.com/1471-2105/9/7 <br />  <br />  <br />  <br />  <br /> Acknowledgements                                                              20.   Rana AA, Collart C, Gilchrist MJ, Smith JC: Defining synphenotype authors like acknowledge numerous collaborations              groups Xenopus tropicalis use antisense morpholino <br />                                                                                     oligonucleotides. PLoS Genet 2006, 2(11):e193. contributed development MODB laboratories       21.   Apache Web Server [http://www.apache.org] <br /> Drs. Farber, Hammerschmidt, Pickart, Schimmenti, Sivasubbu, Verfail-      22.   Nasevicius  Ekker SC: Effective targeted gene &apos;knockdown&apos; lie. like acknowledge bioinformaticians               zebrafish. Nat Genet 2000, 26(2):216-220. <br /> worked project including Dr. Eric Klee Kyong Jin Shim. Funding    23.   Smarty [http://smarty.php.net] <br />                                                                               24.   GD Graphics [http://www.boutell.com/gd/] project obtained NIH SCE (GM63904                25.   Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ: Basic local <br /> CA65493). work supported University Minnesota            alignment search tool. J Mol Biol 1990, 215(3):403-410. <br /> Supercomputing Institute.                                                     26.   ZFIN Anatomy Ontology                  [http://www.geneontology.org/ <br />                                                                                     format.shtml] <br />                                                                               27.   Institute Genomic Research (TIGR)                    [http:// <br /> References                                                                          www.tigr.org] <br /> 1.    Kimmel CB, Ballard WW, Kimmel SR, Ullmann B, Schilling TF: Stages       28.   GenBank [http://www.ncbi.nlm.nih.gov/Genbank/index.html] <br />       embryonic development zebrafish. Dev Dyn 1995,                29.   WormBase [http://www.wormbase.org/] <br />       203(3):253-310.                                                         30.   National Center Biomedical Ontology (NCBO) [http:// <br /> 2.    Geisler R, Rauch GJ, Geiger-Rudolph S, Albrecht  van Bebber F,              www.bioontology.org/] <br />       Berger  Busch-Nentwich E, Dahm R, Dekens MP, Dooley C, et al.:        31.   Amsterdam  Insertional mutagenesis zebrafish: genes       Large-scale mapping mutations affecting zebrafish devel-                   development, genes disease. Brief Funct Genomic Proteomic <br />       opment. BMC Genomics 2007, 8(1):11.                                           2006, 5(1):19-23. <br /> 3.    Draper BW, McCallum CM, Stout JL, Slade AJ, Moens CB: high-           32.   Amsterdam  Hopkins N: Mutagenesis strategies zebrafish <br />       throughput method identifying N-ethyl-N-nitrosourea                       identifying genes involved development disease. <br />       (ENU)-induced point mutations zebrafish. Methods Cell Biol                 Trends Genet 2006, 22(9):473-478. <br />       2004, 77:91-112.                                                        33.   Dahm R, Geisler R: Learning small fry: zebrafish 4.    Wienholds E, van Eeden F, Kosters M, Mudde J, Plasterk RH, Cuppen             genetic model organism aquaculture fish species. Mar Bio- <br />       E: Efficient target-selected mutagenesis zebrafish. Genome                 technol (NY) 2006, 8(4):329-345. <br />       Res 2003, 13(12):2700-2707.                                             34.   Gene Tools [http://www.gene-tools.com/] <br /> 5.    Amsterdam  Burgess S, Golling G, Chen W, Sun  Townsend K,           35.   Open Biosystems [http://www.openbiosystems.com/] <br />       Farrington S, Haldi M, Hopkins N: large-scale insertional muta- <br />       genesis screen zebrafish. Genes Dev 1999, 13(20):2713-2724. <br /> 6.    Pickart MA, Sivasubbu S, Nielsen AL, Shriram S, King RA, Ekker SC: <br />       Functional genomics tools analysis zebrafish pig- <br />       ment. Pigment Cell Res 2004, 17(5):461-470. <br /> 7.    Taylor MR, Kikkawa S, Diez-Juan  Ramamurthy V, Kawakami K, <br />       Carmeliet P, Brockerhoff SE: zebrafish pob gene encodes       novel protein required survival red cone photo- <br />       receptor cells. Genetics 2005, 170(1):263-273. <br /> 8.    Eckfeldt CE, Mendenhall EM, Flynn CM, Wang TF, Pickart MA, Grin- <br />       dle SM, Ekker SC, Verfaillie CM: Functional analysis human <br />       hematopoietic stem cell gene expression using zebrafish. <br />       PLoS Biol 2005, 3(8):e254. <br /> 9.    Michael  Pickart EWK, Nielsen Aubrey L, Sivasubbu Sridhar, <br />       Mendenhall Eric M, Brent R, Chen Eleanor, Eckfeldt Craig E, <br />       Knowlton Michelle, Robu Mara E, Larson Jon D, Deng Yun, Schim- <br />       menti Lisa  Ellis Lynda BM, Verfaillie Catherine M, Hammerschmidt <br />       Matthias, Farber Steven  Ekker Stephen C: Genome-Wide <br />       Reverse Genetics Framework Identify Novel Functions       Vertebrate Secretome. PLoS 2006, 1(1):e104. <br /> 10.   Klee EW, Ellis L BM: Evaluating eukaryotic secreted protein <br />       prediction. BMC Bioinformatics 2005, 6:256. <br /> 11.   Sadl VS, Sing  Mar L, Jin F, Cordes SP: Analysis hindbrain pat- <br />       terning defects caused kreisler(enu) mutation reveals <br />       multiple roles Kreisler hindbrain segmentation. Dev Dyn <br />       2003, 227(1):134-142. <br /> 12.   Klee EW, Shim KJ, Pickart MA, Ekker SC, Ellis LB: AMOD: mor- <br />       pholino oligonucleotide selection tool. Nucleic Acids Res <br />       2005:W506-511. <br /> 13.   Miller-Bertoglio V, Carmany-Rampey  Furthauer M, Gonzalez EM, <br />       Thisse C, Thisse B, Halpern  Solnica-Krezel L: Maternal       zygotic activity zebrafish ogon locus antagonizes BMP <br />       signaling. Dev Biol 1999, 214(1):72-86. <br />                                                                                         Publish Bio Med Central 14.   Sprague J, Bayraktaroglu L, Clements D, Conlin T, Fashena D, Frazer              scientist read work free charge <br />       K, Haendel M, Howe DG, Mani P, Ramachandran S, et al.:                    &quot;BioMed Central significant development       Zebrafish Information Network: zebrafish model organ- <br />                                                                                     disseminating results biomedical researc h lifetime.&quot; <br />       ism database. Nucleic Acids Res 2006:D581-585. <br /> 15.   Mouse Phenome Database [http://phenome.jax.org/pub-cgi/phe                        Sir Paul Nurse, Cancer Research UK <br />       nome/mpdcgi?rtn=docs/home] <br /> 16.   Bogue MA, Grubb SC, Maddatu TP, Bult CJ: Mouse Phenome                          research papers  <br />       Database (MPD). Nucleic Acids Res 2007:D643-649.                                  available free charge entire biomedical community <br /> 17.   Horn T, Arziman  Berger J, Boutros M: GenomeRNAi: data-                       peer reviewed published immediately acceptance <br />       base cell-based RNAi phenotypes. Nucleic Acids Res <br />       2007:D492-497.                                                                    cited PubMed archived PubMed Central <br /> 18.   GenomeRNAi [http://www.dkfz signaling2/rnai/ernai.html]                        &#226;&#8364;&#8221; copyright <br /> 19.   Xenopus tropicalis Morpholino Screen                  [http://www.gur <br />       don.cam.ac.uk/~smithlab/screens/Xenopus-morpholino-pilot/mor             Submit manuscript                                 BioMedcentral <br />       pholino-screen-lists.html#synphenotypes]                                 http://www.biomedcentral.com/info/publishing_adv.asp <br />  <br />  <br />  <br />  <br />                                                                <br /> </body></html>