<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Divergent Mechanisms Controlling Hypoxic Sensitivity Lifespan DAF-2/Insulin/IGF-Receptor Pathway <br /> Meghann E. Mabon1,3, Barbara  Scott1, C. Michael Crowder1,2,3* <br /> 1 Department Anesthesiology, Washington University School Medicine, St. Louis, Missouri, United States America, 2 Department Developmental Biology, <br /> Washington University School Medicine, St. Louis, Missouri, United States America, 3 Division Biology <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; Biomedical Sciences, Program Developmental <br /> Biology, Washington University School Medicine, St. Louis, Missouri, United States America <br />  <br />  <br />  <br />      Abstract <br />      Organisms cells vary greatly tolerance low oxygen environments (hypoxia). delineation      determinants hypoxia tolerance incomplete, despite intense implications diseases stroke      myocardial infarction. insulin/IGF-1 receptor (IGFR) signaling pathway controls survival Caenorhabditis elegans      variety stressors including aging, hyperthermia, hypoxia. daf-2 encodes C. elegans IGFR homolog primary <br />      signaling pathway modulates activity FOXO transcription factor DAF-16. DAF-16 regulates transcription      large number genes, shown control aging. identify genes selectively regulate <br />      hypoxic sensitivity, compared organismal transcriptomes daf-2 reduction function alleles,      hypoxia resistant, thermotolerant, long lived, differ rank severities phenotypes.      transcript levels 172 genes increased hypoxia resistant daf-2 allele, e1370, relative alleles <br />      transcripts 10 genes decreased abundance. RNAi knockdown 6 10 genes produced      significant increase organismal survival hypoxic exposure expected regulation genes <br />      e1370 mutation responsible hypoxia resistance.  RNAi knockdown genes did prolong <br />      lifespan. genes definitively separate mechanisms hypoxic sensitivity lifespan identify biological <br />      strategies survive hypoxic injury. <br />  <br />   Citation: Mabon  Scott BA, Crowder CM (2009) Divergent Mechanisms Controlling Hypoxic Sensitivity Lifespan DAF-2/Insulin/IGF-Receptor <br />   Pathway. PLoS 4(11): e7937. doi:10.1371/journal.pone.0007937 <br />   Editor: Immo  Hansen, New Mexico State University, United States America <br />   Received August 25, 2009; Accepted October 22, 2009; Published November 20, 2009 <br />   Copyright: &#195;&#376; 2009 Mabon et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported National Institute Neurological Disorders Stroke - R01 NS045905 (http://www.ninds.nih.gov/), American Heart <br />   Association Established Investigator Award (http://www.americanheart.org/presenter.jhtml?identifier = 3004142), McKnight Endowment Fund   Neuroscience - Neuroscience Brain Disorders Award (http://www.mcknight.org/neuroscience/). funders role study design, data collection   analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: crowderm@morpheus.wustl.edu <br />  <br />  <br />  <br /> Introduction                                                                        various methods used studies different <br />                                                                                     phenotypes screened produced modestly overlapping gene <br />    Low ambient oxygen concentrations induce cell death.                         sets.  DAF-16 appears directly indirectly control  cells vary greatly level duration hypoxia                    expression large number diverse genes influence required produce death.                       specific daf-2 phenotype. <br /> organism hypoxic sensitivity range widely. humans,                              Multiple daf-2 alleles isolated, primarily screens neurons cardiac myocytes exquisitely hypoxia sensitive,                     mutants constitutively form dauers. Gems et al. measured lead stroke myocardial infarction.                            phenotypes sixteen daf-2 alleles lack concordance <br /> identification determinants hypoxic sensitivity                  severity different phenotypes cases [13]. <br /> biological medical significance.                                                example, daf-2(e1370) moderately long lived dauer <br />    previously shown reduction function mutations                    constitutive thermotolerant alleles tested. daf-2 gene confer powerful protection hypoxic injury                 hand, daf-2(e1368) considerably thermotol- nematode Caenorhabditis elegans [1]. daf-2 encodes insulin/                  erant dauer constitutive e1370 IGF-receptor homolog signals conserved PI-3                          median lifespan 50% longer e1370. measured kinase signaling cascade regulate negatively activity                   hypoxic sensitivity thirteen daf-2 alleles wide range <br /> FOXO transcription factor, DAF-16 [2,3]. daf-2 reduction                         sensitivities [1]. example, e1370 hypoxia <br /> function mutants hypoxia resistant                        resistant daf-2 allele tested e1368 weakly resistant. <br /> thermotolerant, long lived, propensity form dauer                     mechanism underlying divergence phenotypic <br /> larvae, capable long periods hibernation                         severities alleles obscure. <br /> response environmental stress [4].                              current study utilizing cDNA microarrays double <br /> interesting biological processes controlled daf-2 &#226;&#8364;&#8220; daf-16                   stranded RNA-mediated interference (RNAi) expression pathway, considerable effort identify                          candidate genes, advantage daf-2 allelic variations downstream targets pathway [5,6,7,8,9,10,11,12].                         identify genes regulated daf-2 selectively alter hypoxic <br />  <br />  <br />        PLoS | www.plosone.org                                               1                          November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                  daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  <br /> Figure 1. daf-2 allelic series hypoxia resistance does correlate daf-2 phenotypes.   Survival following recovery 20 hour hypoxia insult (plotted data mean6SD independent experiments). (B) Wild-type (N2) daf-2 mutants assayed thermotolerance (mean trials plotted); e1370 (red line), m596 (purple line), e1368 (blue line) N2 (black line). (C) Lifespans N2 daf-2 alleles. survival curve generated minimum 77 animals. Animals bagged crawled plate censored. <br /> (D, E, F) Dauer arrest measured 20u (C), 23u (D), 25u N2 daf-2 mutants. Black bars represent adults white bars represent dauers. strains statistically different hypoxic survival, thermotolerance, dauer formation  N2 e1368 D,E m596 e1370 F), p,0.01  tailed t-test  linear regression B, Mantel-Cox Log-rank test C, Fisher&#226;&#8364;&#8482;s exact x2 D, E, F. <br /> doi:10.1371/journal.pone.0007937.g001 <br />  <br /> sensitivity affecting lifespan. identity genes           weaker allele (Figure 1D,E,F). results generally consistent <br /> implicates novel pathways selective regulation hypoxic             previously reported Gems et al. [13]. authors <br /> sensitivity.                                                                  reported e1370 thermotolerant, followed closely                                                                               m596, distantly e1368. culture temperature Results                                                                       22.5uC, m596 longest lifespan followed e1368                                                                               e1370. dauer constitutive phenotype, order severity <br /> daf-2 Alleles Reveal Divergent Outputs Hypoxic                             m596.e1370.e1368. <br /> Survival Lifespan <br />     chose daf-2 alleles study e1370, m596, e1368.        Transcriptional Profiling daf-2 Alleles previously reported recovery 20 hours                      transcriptional output daf-2 - daf-16 pathway hypoxic incubation, organism survival alleles             distinct daf-2 allelic series hypoxia versus lifespan dauer <br /> 96%, 53%, 23%, respectively, compared 4% wild <br /> type strain N2 [1]. Gems et al. scored dauer constitutive, <br /> thermotolerance, long lifespan phenotypes alleles [13].            Table 1. Lifespan analysis daf-2 strains.  severities daf-2 phenotypes varied consider- <br /> ably labs, particularly lifespan.  scored lifespan, thermotolerance, dauer constitutive, hypoxia resistant            Strain               Mean (d)             Maximum (d)          n <br /> phenotypes daf-2 alleles contemporaneously confirm              N2                   14.9                 29                   87 phenotypic divergence. e1370                                                                                 Daf-2(e1368)         18.8*                37*                  86 <br /> strongly hypoxia resistant, m596 moderately resistant, e1368 <br /> weakly (Figure 1A). order severities alleles              Daf-2(m596)          31.3*                51*                  77 <br /> quite similar thermotolerance (Figure 1B).  lifespan            Daf-2(e1370)         22.5*{               51*                  86 <br /> m596 strongest allele having mean lifespan 31.3 days <br />                                                                                 Day 1 begins day adulthood. <br /> compared 22.5 days e1370 18.8 e1368 (Figure 1C               * <br />                                                                                  p,0.01 vs N2; <br /> Table 1). dauer constitutive phenotype, m596               { <br />                                                                                   - p,0.01 vs e1368 m596. <br /> severe allele; e1370 close second e1368                  doi:10.1371/journal.pone.0007937.t001 <br />  <br />  <br />  <br />        PLoS | www.plosone.org                                         2                           November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                            daf-2 Hypoxic Death Targets <br />  <br />  <br /> formation offer means identify genes daf-2 - daf-16                  expression relative m596 relative e1368. results <br /> pathway specifically regulate hypoxia. hybridized                  suggest majority genes regulate lifespan, genome oligonucleotide microarrays cDNA derived                        trait e1370 diverges m596 e1368. e1370, m596, e1368 L4 stage larval animals grown                        expression 182 genes followed hypoxic <br /> normal conditions. Comparisons transcriptome                       phenotypic allelic series,  regulated alleles revealed 182 genes statistically different                     order e1370, m596, e1368 (Table 3).   <br /> (p,0.01) e1370 m596 e1368 (Figure 2A);                        C17C3.12, encodes strong mammalian homolog, mitochron- <br /> Raw data deposited NCBI Gene Expression                         drial <span id='am-8' about='obo:IMR_0200065' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:id" content="IMR:0200065" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00040" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Acyl-CoA" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acyl-CoA</span> dehydrogenase, functions catabolically Omnibus database (http://www.ncbi.nlm.nih.gov/geo/), acces-                      mammals fatty acid oxidation. general, genes involved sion number GSE18601. 172 genes regulated e1370                       regulation metabolic processes highly represented compared m596 e1368 (Table S1), 10                            182 gene list (Figure 2C). Transcriptional/DNA/RNA processing <br /> relatively regulated e1370 (Table 2, Figure 2B). vast                genes abundant. <br /> majority e1370-upregulated genes greater increase                    number previous studies identified genes                                                                                  expression regulated daf-2 [7,8,9,10,11,12,14,15].                                                                                  studies implicated large number genes regulated <br />                                                                                  daf-2 - daf-16 pathway.  overlap                                                                                  datasets relatively modest presumably distinct <br />                                                                                  methodologies employed study.                                                                                  experimental design microarrays sought gene <br />                                                                                  expression differences daf-2 alleles, overlap                                                                                  182 genes identified studies                                                                                  modest seven genes common (Table 4).                                                                                  seven genes expression changes qualitatively <br />                                                                                  similar previously. example, Y54G11A.13/ <br />                                                                                  ctl-3 gene encodes C. elegans catalases                                                                                  predicted reduce peroxide free radicals. ctl-3 transcript                                                                                  increased e1370 compared e1368 m596. Likewise <br />                                                                                  using quantitative mass spectrometry methods, Dong et al.                                                                                  CTL-3 protein increased e1370 compared wild type <br />                                                                                  worms [14]. Similarly, data Murphy et al. <br />                                                                                  [9] increased expression e1370 mutants unc-38,                                                                                  encodes subunit nicotinic <span id='am-29' about='obo:IMR_0100111' typeof='owl:Thing obo:IMR_0001693 obo:IMR_0100121 obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0002121'><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="acetylcholine" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C01996" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0100111" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="O-Acetylcholine" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="O-Acetylcholine" datatype="xsd:string"></span>acetylcholine</span> receptor. <br />                                                                                      genes regulated e1370 (die) compared                                                                                  alleles (Table 2, Figure 2B). genes include akt-2,                                                                                  homolog serine/threonine protein kinase Akt/PKB. akt-2 <br />                                                                                  paralog, akt-1, functions downstream daf-2                                                                                  daf-2 - daf-16 pathway. Knockdown akt-1 akt-2 produce                                                                                  partially redundant dauer constitutive phenotype [16], akt- <br />                                                                                  1 gain function mutant partially suppresses hypoxia <br />                                                                                  resistance daf-2(1370) [1]. role AKT-1/2 downstream <br />                                                                                  DAF-2 thought phosphorylation DAF-16, <br />                                                                                  inhibiting nuclear localization transcription factor <br />                                                                                  activity [17,18]. new data suggest positive feedback <br />                                                                                  loop exist enhances reduction function phenotype <br />                                                                                  e1370 compared m596 e1368 reducing levels                                                                                  akt-2 mRNA. akt-2, die genes predicted                                                                                  kinase phosphatase activity, substrates                                                                                  unknown. particularly interesting die genes include acdh-2, <br />                                                                                  noted expression differences alleles                                                                                  follow hypoxic sensitivity allelic series, F54G2.1 - isoform                                                                                  presynaptic vesicle fusion protein UNC-13, F36F2.3 - E3 <br />                                                                                  ubiquitin ligase, T04A11.6 - <span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span>-dependent DNA <br />                                                                                  helicase. previously genome RNAi screen <br />                                                                                  genes promote hypoxic death knockdown                                                                                  genes, predicted control chromatin structure, produce <br />                                                                                  significant hypoxia resistance [19]. current data                                                                                  daf-2 pathway acts regulation DNA/chromatin <br />                                                                                  modifying enzymes control hypoxic sensitivity.  Figure 2. Expression 182 genes differentially regulated                   unclear protects organism hypoxic injury. daf-2 alleles.   Heat map expression 182 genes,  regulated e1370 m596 e1368, p,0.01, Way ANOVA. (B) Genes expression downregulated                                                                                  Increased Hypoxic Survival die Gene Knockdown <br /> e1370 (die) relative m596 e1368, p,0.01. (C) Relative number              Genes expression increased e1370 relative daf-2 regulated genes fall various functional categories.       alleles candidates encode hypoxia protective <br /> doi:10.1371/journal.pone.0007937.g002                                            factors. chose 10 genes highest normalized relative <br />  <br />  <br />        PLoS | www.plosone.org                                            3                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                          daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  Table 2. Genes transcriptionally regulated e1370 vs m596 e1368 &#226;&#8364;&#8220; die genes. <br />  <br />  <br />  Sequence Gene                                                                                                                          e1370vs     e1370vs   e1368vs <br />              Protein/Homology                                 Biological Process         Expression                 p-value   m596        e1368     m596 <br />  <br />  F36F2.3               Predicted E3 ubiquitin ligase                    Protein Degradation/       ND                         0.0019    26.0        22.6      22.3 <br />                                                                         Modification/Turnover <br />  ZK262.8               synthetic lethal Argonaute mutant           Unknown                    ND                         0.0021    23.6        21.9      21.9 <br />  T22C1.8               Protein tyrosine phosphatase                     Signaling                  ND                         0.0027    25.2        22.5      22.1 <br />  F54D11.3              C. elegans-specific                              Unknown                    ND                         0.0048    27.1        22.6      22.7 <br />  F58D2.1               ZYG-11-like serine/threonine protein kinases     Unknown                    ND                         0.0059    22.2        21.5      21.5 <br />  C17C3.12    acdh-2    Mitochondrial short-chain acyl-<span id='am-24' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span>CoA</span>               Metabolism                 ND                         0.0060    22.4        23.6      21.5 <br />                        dehydrogenase <br />  F28H6.1     akt-2     Homolog serine/threonine kinase           Signaling                  Intestine, Neuronal,       0.0061    22.7        21.3      22.1 <br />                        Akt/PKB                                                                     Hypodermal, Pharyngeal <br />  F54G2.1               Isoform 1 protein UNC-13 homolog D            Unknown                    Intestine, Neuronal        0.0072    22.5        21.6      21.6 <br />  T20D4.18    srab-21   Sra family integral membrane protein             Unknown                    Hypodermal, Neuronal       0.0077    22.1        21.4      21.5 <br />  T04A11.6    6     ATP-dependent DNA helicase                       Transcription/DNA/RNA      Primarily Germline         0.0086    22.9        21.5      21.9 <br />                                                                         Processing <br />  <br />  Annotations WormBase (www.wormbase.org); p-value way ANOVA e1370 vs m596. <br />  doi:10.1371/journal.pone.0007937.t002 <br />  <br />  <br /> fold change e1370 compared m596 test hypoxia                               e1368 background compared vector control <br /> protection (Table S1). used feeding RNAi reduce                             (Figure 3B). results reduction function expression e1370 background prediction                         genes capable converting weakly hypoxia suppress partially hypoxia resistant phenotype.                      resistant e1368 animals strongly resistant ones.   did observe significant change hypoxia                             sensitized background e1368 revealed F58D2.1 does resistance treatment RNAis (data shown).                       fact encode hypoxic sensitivity promoting activity negative result explained number ways.                            apparent wild type background. <br /> Obviously, inadequate knockdown gene expression way.                             Given expression die gene transcripts regulated daf- <br /> Given 172 genes regulated, likely                      2, hypothesized hypoxia resistance phenotype die <br /> possibility knockdown single gene sufficient                        RNAi depend DAF-16, primary target DAF-2. <br /> produce measurable phenotype. Alternatively, regulated                          hypoxia resistance phenotypes die genes genes tested mediate responses maladaptive                       suppressed null mutation daf-16 (Figure 3C).  unrelated survival hypoxia;  knockdown                        suppression hypoxia resistance incomplete suppress e1370&#226;&#8364;&#8482;s hypoxia resistance.                                                    RNAi knockdowns.  genes regulate hypoxic <br />    die genes candidates promote daf-2-regulated hypoxic                      sensitivity entirely DAF-16 dependent pathway. <br /> sensitivity;  die gene reduction function produce <br /> hypoxia resistance. test hypothesis, used RNAi                                                                                         die Genes Equally Required Thermotolerance <br /> knockdown die genes; tested,  <br /> C17C3.12 F28H6.1, library,                                Lifespan Extension <br /> T22C1.8, did grow streaked, fourth, F36F2.3,                              experimental design microarrays devised enrich <br /> produced sick animals absence hypoxia. RNAi                       genes selectively regulate hypoxic sensitivity. test treatments conferred significant hypoxia resistance                        selective regulation, measured thermotolerance lifespan <br /> wild type animals (Figure 3A). tested effect knockdown die genes e1368 background. <br /> Knockdown genes produced strong hypoxia resistance                            Table 4. Genes overlap datasets. <br />  <br />  <br />  Table 3. Genes regulated according allelic series.                               Sequence       Overlapping Expression e1370     Expression e1370 <br />                                                                                                    Study       relative m596, e1368 relative wild type* <br />  <br />                                                                                          C09G4.5        Murphy           Increased                Decreased <br />  Sequence         Common           e1370*     m596*         e1368* <br />                                                                                          C47E8.5        Dong             Increased                Decreased <br />  C14A4.8                                     2.19       0.90          0.85 <br />                                                                                          F15E11.1       Dong             Increased                Increased <br />  Y37H2A.6              fbxa-211              1.20       0.92          0.90 <br />                                                                                          F17C11.9       Dong             Increased                Decreased <br />  F45D11.4                                    3.64       0.62          0.34 <br />                                                                                          F21F3.5        Murphy           Increased                Increased <br />  F45D11.14                                   3.70       0.54          0.28 <br />                                                                                          F45D11.14      Dong             Increased                Increased <br />  C17C3.12              acdh-2                0.79       1.88          2.86 <br />                                                                                          Y54G11A.13 Dong                 Increased                Increased <br />  T26H5.1                                     0.83       1.17          1.20 <br />                                                                                          Murphy et al. Nature 424:277-83, 2003. Dong et al. Science 317:660-3, 2007. <br />  *                                                                                       * <br />   Mean normalized expression, p,0.01.                                                  - data overlapping study. <br />  doi:10.1371/journal.pone.0007937.t003                                                   doi:10.1371/journal.pone.0007937.t004 <br />  <br />  <br />  <br />       PLoS | www.plosone.org                                                    4                            November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                             daf-2 Hypoxic Death Targets <br />  <br />  <br />                                                                                  (Figure 5 Table S2).  results consistent                                                                                  poor correlation lifespan hypoxic sensitivity <br />                                                                                  phenotypes daf-2 alleles [1]. <br />                                                                                     possibility strong correlation thermotoler- <br />                                                                                  ance hypoxia resistance 26.5uC temperature                                                                                  hypoxic incubations performed. models hypoxic <br />                                                                                  injury, sensitivity C. elegans hypoxia temperature <br />                                                                                  dependent [1]; higher temperature severe                                                                                  hypoxic injury.  number observations suggest                                                                                  hypoxic sensitivity differences daf-2 alleles simply <br />                                                                                  manifestation varied thermotolerance.  daf-2 alleles, <br />                                                                                  e979 e1369 moderate hypoxia resistance similar                                                                                  m596, thermotolerances high e1370 [13]. <br />                                                                                  Second, hypoxic incubations performed 20uC, e1370 <br />                                                                                  m579, highly hypoxia resistant assay <br />                                                                                  conditions [1], highly hypoxia resistant, m596                                                                                  [20]. Finally, asked high temperature incubation <br />                                                                                  buffer room air environment produced significant wild <br />                                                                                  type animal death account daf-2 allelic <br />                                                                                  differences hypoxic sensitivity higher temperature.  <br />                                                                                  observed significant death wild type animals                                                                                  20 hour normoxic incubation 28uC (Figure S1). 71 hour <br />                                                                                  incubation killed 5% animals. <br />  <br />                                                                                  Discussion <br />                                                                                      Utilizing daf-2 alleles distinct orders phenotypic severity <br />                                                                                  hypoxic sensitivity versus phenotypes basis                                                                                  DNA microarray experiments, identified 182 genes                                                                                  differentially expressed animals varying levels hypoxia <br />                                                                                  resistance. expression genes increased                                                                                  hypoxia resistant daf-2(e1370). genes candidates                                                                                  antagonizing hypoxic death.  unable detect <br />                                                                                  hypoxic hypersensitivity RNAi knockdown subset                                                                                  genes. general, hypoxic hypersensitivity                                                                                  difficult produce model; reflect genetic <br />                                                                                  redundancy reminiscent relatively uncommon <br />                                                                                  phenotype shortened lifespan compared prolonged <br /> Figure 3. Survival hypoxia RNAi knockdown die                         lifespan.  genes decreased daf- <br /> genes. Genetic background strains indicated closed (wild type -         2(e1370), able test N2) open bars (mutant &#226;&#8364;&#8220; daf-2(e1368) daf-16(mu86)). Note            knockdown produced hypoxia resistance. survival data 22 hour hypoxic incubation upper lower panels 24 hour hypoxic incubation                                                                                  genes fact control hypoxic <br /> middle panel.   Survival recovery 22 hour hypoxic                 sensitivity relatively specific manner incubation N2 wild type animals treated bacteria transfected            having detectable influence lifespan. regard, L4440 vector RNAi plasmids die genes.                 ZK262.8 intriguing unlike die <br /> * p,0.01 compared L4440  tailed t-test, 6 trials/RNAi .30            genes hypoxia resistance knockdown phenotype, ZK262.8 <br /> worms/trial). (B) Effect RNAi treatments survival 24 hour          knockdown does confer thermotolerance. ZK262.8 poorly <br /> hypoxic incubation wild type daf-2(e1368) backgrounds. RNAi&#226;&#8364;&#8482;s resistance e1368 background compared L4440 &#226;&#8364;&#8220; p,0.01              characterized gene clear homology genes  tailed t-test, 3 trials/RNAi .30 worms/trial) (C) Effect RNAi       organisms. implicated microRNA processing treatments survival 22 hour hypoxic incubation wild type           synthetic lethal phenotype mutation Argonaut daf-16(mu86 null) backgrounds. Hypoxia resistance significantly              homolog gene, alg-2 [21]. ALG-2 expressed   <br /> suppressed mu86 background RNAi&#226;&#8364;&#8482;s &#226;&#8364;&#8220; p,0.01 (paired t-test, 6          cells C. elegans embryonic adult stages trials/RNAi .30 worms/trial). <br /> doi:10.1371/journal.pone.0007937.g003 <br />                                                                                  hypothesized facilitate loading pre-miRNAs Dicer <br />                                                                                  complex [21].  unclear role ZK262.8 die genes knockdown produced hypoxia resistance                gene product miRNA processing. wild type background. Knockdown die genes                   hypoxic death promoting die gene identified srab-21, <br /> produced significant thermotolerance (Figure 4). ZK262.8                    encodes serpentine receptor likely function RNAi did protect thermal stress (Figure 4B). results              chemosensation [22]. genome RNAi screen genes consistent strong correlation thermotolerance               promoting hypoxic death, identified serpentine receptor hypoxia resistance observed various daf-2 alleles [1].                 genes knockdown produced hypoxia resistance [19].  mechanisms thermal injury                genes represented 5% genes identified screen. <br /> hypoxic injury C. elegans identical. Despite considerable           Given genes thought encode chemosensory <br /> increase mean maximum lifespan daf-2 RNAi,                   receptors, normal chemosensory function appears promote die gene RNAi&#226;&#8364;&#8482;s resulted significant alteration lifespan          hypoxic sensitivity C. elegans. Likewise aging, mutations  <br />  <br />        PLoS | www.plosone.org                                            5                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                    daf-2 Hypoxic Death Targets <br />  <br />  <br />  <br />  <br /> Figure 4. Increased thermotolerance die gene RNAi knockdown. Survival recovery incubations 37u various times worms raised L4440 vector denoted RNAi. RNAi&#226;&#8364;&#8482;s panel C-F produce significantly increased thermotolerance RNAi panel B produces significantly decreased thermotolerance &#226;&#8364;&#8220; p,0.01, F-test nonlinear regression simultaneous curve-fitting. minimum 353 animals/curve 18 animals/data point scored. <br /> doi:10.1371/journal.pone.0007937.g004 <br />  <br /> genes required chemosensation prolong lifespan [23].             expression somatic cells possible. Germ cells 6 mutant <br /> prolonged lifespan mutants suppressed               animals increased sensitivity ionizing radiation, mutations daf-16, model proposed activation         surprisingly decreased germ cell apoptosis genotoxic chemosensory neurons increases insulin-like hormone signaling         stress [25].  possible hypoxia resistance limits lifespan.  fact srab-21              secondary defects apoptosis remains determined. <br /> knockdown does prolong lifespan suggests simply altering           identified significant number genes ligand activation DAF-2 unlikely mechanism.                     regulate hypoxic sensitivity [1,19,26,27,28,29]. case,    identification ATP-dependent DNA helicase gene         complete understanding mechanism(s) gene <br /> 6 die gene promotes hypoxic death consistent           protects organism cells hypoxic injury, previous results. genome RNAi screen, identified       cases, genes identified  mechanistic <br /> large number DNA processing genes knockdown                     information  Future studies necessarily focus phenotype hypoxia resistance [19].  nucleic acid binding       promising hypoxic sensitivity genes. genes processing enzyme genes represented largest functional           clear mammalian orthologs, preferably considerable <br /> category helicases  6 expressed           understanding cellular function defined. Addi- <br /> primarily germline assessed microarray experiments          tionally, promising genes able modulated <br /> comparing animals germline [24].            control hypoxic injury producing effects. <br />  <br />  <br />       PLoS | www.plosone.org                                     6                        November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                             daf-2 Hypoxic Death Targets <br />  <br />   studies described important                  worms placed NGM plates spotted OP50 bacteria <br /> order identify specific pathways target hypoxic protection.          using glass Pasteur pipets recovered 20u 16220 hours <br />                                                                                prior scoring. Animals scored dead pharyngeal <br /> Materials Methods                                                          pumping spontaneous evoked movement (touching                                                                                platinum wire) observed. minimum triplicate plates Nematode Maintenance                                                           trial trials/RNAi scored. <br />   strain maintenance worms cultured 20u nematode growth medium (NGM) agar plates seeded OP50                      Feeding RNAi <br /> bacteria [30]. following strains utilized study: N2              generation feeding RNAi performed described <br /> (wild-type), CB1370 daf-2(e1370) III, DR1565 daf-2(m596) III,                  previously [19,31]. Briefly, gravid adult worms left DR1572 daf-2(e1368) III, CF1038 daf-16(mu86)  Strains               RNAi plates 2 day old bacterial lawns 3 hours obtain <br /> obtained Caenorhabditis Genetics Center funded                 30250 eggs plate. RNAi plates composed NGM agar <br /> NIH NCRR.                                                                      supplemented 50 mg/ml carbenicillin 1 mM IPTG                                                                                seeded appropriate RNAi bacterial strain cultured Hypoxia Assay                                                                  2xYT 50 mg/ml carbenicillin, 10 mg/ml tetracycline   Worms subjected hypoxia described previously [1].                 0.8 mM IPTG. Animals grown 20uC young adulthood, <br /> Briefly, plate washed 1.5 mL tube 1 mL               day past L4 stage phenotyped. <br /> M9 buffer (22 mM KH2PO4, 22 mM Na2HPO4, 85 mM NaCl, <br /> 1 mM MgSO4) [30]. Worms allowed settle gravity,                 Lifespan Assays <br /> 800 mL M9 removed. tubes placed                       Lifespan assays performed OP50 bacteria <br /> anaerobic chamber (Forma Scientific) 26.5u 20, 22,                   (Figure 1C) RNAi bacteria containing L4440 24 hours, depending assay. Following hypoxic insult,                vector control plasmid particular RNAi plasmid (Figure 5). <br />  <br />  <br />  <br />  <br /> Figure 5. effects lifespan die gene RNAi knockdown. Lifespan survival curves animals grown L4440 vector bacteria <br /> versus RNAi bacteria.   effect daf-2 RNAi lifespan shown positive control significantly different L4440 lifespan &#226;&#8364;&#8220; p%0.01 <br /> Mantel-Cox Log-rank test. (B-F) RNAis significantly lengthen lifespan p,0.01 level. Lifespan assays performed 120 <br /> animals/RNAi grown 20uC. <br /> doi:10.1371/journal.pone.0007937.g005 <br />  <br />  <br />        PLoS | www.plosone.org                                          7                         November 2009 | Volume 4 | Issue 11 | e7937 <br />                                                                                                                                            daf-2 Hypoxic Death Targets <br />  <br />  <br /> Gravid adult worms allowed lay eggs hours                          ter, Washington University St. Louis). Slides scanned OP50 RNAi bacteria picked  animals cultured                           ScanArray 3000 (Perkin Elmer, Waltham, MA) 10 micrometer 20uC reaching L4 stage placed                             resolution. biological replicates (independent RNA isola- <br /> density 30 worms/plate. daf-2 allele lifespan data, L4                     tions), technical replicate (dye swap) performed <br /> animals transferred fresh NGM plates seeded OP50.                            condition. Raw microarray fluorescent intensity data die gene RNAi lifespan experiments, L4 animals                               processed analyzed GeneSpring GX 7.3.1 software <br /> transferred new RNAi plates. Worms transferred daily                              (Agilent Technologies, Santa Clara, CA). time point egg-laying transferred days                         subjected Lowess normalization  spot array) <br /> plates freshly seeded OP50 RNAi bacteria. Worms                              separately. flag filter applied include genes scored daily survival censored bagging, exploding,                        hybridized trials narrowed gene list crawling plate.                                                                   14,317. way ANOVA (p,0.01) performed genes <br />                                                                                           statistically different e1370 Thermotolerance Assay                                                                     daf-2 alleles, m596 e1368. gene list included 182 genes. <br />    Thermotolerance assays performed 37u water bath.                             Raw data deposited NCBI Gene Expression <br /> Young adult worms, day post L4 stage, washed tubes                          Omnibus database (http://www.ncbi.nlm.nih.gov/geo/), acces- <br /> using M9 buffer allowed settle gravity. Excess M9                           sion number GSE18601. <br /> removed 100 mL. M9 buffer preheated 37u added tube final volume 500 mL. tubes incubated                         Supporting Information <br /> water bath according time course. Worms removed tubes appropriate time using glass Pasteur pipette                      Table S1 List genes upregulated daf-2(e1370). allowed recover 20uC 16220 hours NGM plates                               doi:10.1371/journal.pone.0007937.s001 (0.06 MB <br /> spotted OP50 bacteria scored survival.                                  XLS) <br />                                                                                           Table S2 Lifespan analysis die gene RNAi knockdown. <br /> Dauer Arrest Assay                                                                         doi:10.1371/journal.pone.0007937.s002 (0.03 MB <br />    gravid adult animals allowed lay eggs hour                         XLS) picked  Plates containing 30250 eggs 20uC, <br /> 23uC 25uC. Animals scored dauer arrest                               Figure S1 Comparison normoxic hypoxic lethality. <br /> non-dauer animals reached adulthood, 72 h 96 h later.                              Young adult N2 animals incubated 28 degrees M9 <br /> fraction non-adult animals partial dauers arrested                        buffer incubator room air atmosphere (normoxia) <br /> non-dauer larvae formally scored;  large                              incubator ,0.3% oxygen (hypoxia). % dead <br /> majority non-adult animals true dauers                          animals scored recovery various incubation times. <br /> strains previously reported [13].                                                      point represents mean +/2 sd minimum trials <br />                                                                                           16 animals/trial. <br /> cDNA Microarrays                                                                           doi:10.1371/journal.pone.0007937.s003 (2.48 MB TIF) <br />    Animals synchronous egg lay cultured 20uC NGM plates seeded OP50, total RNA isolated                              Acknowledgments <br /> L4 staged daf-2(e1370), daf-2(m596) daf-2(e1368) animals using <br />                                                                                           thank Elaine Mardis Genome Center Washington <br /> Trizol reagent (Invitrogen). Dauer animals separated                                                                                           University St. Louis help design performance L4 larvae differential pelleting M9. RNA quality                                DNA microarray experiments. <br /> measured using Nanodrop-1000 spectrophotometer (Thermo- <br /> Scientific). High quality RNA samples (OD 260/280.1.9) used inputs microarrays. micrograms RNA                                                                                           Author Contributions <br /> subjected reverse transcription subsequent hybridization                           Conceived designed experiments: MEM CMC. Performed using 3DNA Array 350 Kit (Genisphere,  Hatfield, PA). Cy3-                            experiments: MEM BAS. Analyzed data: MEM BAS CMC. Cy5-labeled samples hybridized C. elegans                               Contributed reagents/materials/analysis tools: CMC. Wrote paper: <br />                                                                                           MEM BAS CMC. <br /> genome oligonucleotide microarrays (Genome Sequencing Cen- <br />  <br /> References <br />  1. Scott BA, Avidan MS, Crowder CM (2002) Regulation hypoxic death C.               8. Lamitina ST, Strange K (2005) Transcriptional targets DAF-16 insulin <br />     elegans insulin/IGF receptor homolog DAF-2. Science 296: 2388&#226;&#8364;&#8220;2391.                signaling pathway protect C. elegans extreme hypertonic stress. <br />  2. Ogg S, Paradis S, Gottlieb S, Patterson GI, Lee L, et al. (1997) Fork head            J Physiol Cell Physiol 288: C467&#226;&#8364;&#8220;474. <br />     transcription factor DAF-16 transduces insulin-like metabolic longevity            9. Murphy CT, McCarroll SA, Bargmann CI, Fraser  Kamath RS, et al. (2003) <br />     signals C. elegans. Nature 389: 994&#226;&#8364;&#8220;999.                                               Genes act downstream DAF-16 influence lifespan  3. Lin K, Dorman JB, Rodan  Kenyon C (1997) daf-16: HNF-3/forkhead                      Caenorhabditis elegans. Nature 424: 277&#226;&#8364;&#8220;283. <br />     family member function double life-span Caenorhabditis             10. McElwee J, Bubb K, Thomas JH (2003) Transcriptional outputs     elegans. Science 278: 1319&#226;&#8364;&#8220;1322.                                                          Caenorhabditis elegans forkhead protein DAF-16. Aging Cell 2: 111&#226;&#8364;&#8220;121. <br />  4. Riddle DL, Albert PS (1997) Genetic environmental regulation dauer             11. Lee SS, Kennedy S, Tolonen AC, Ruvkun G (2003) DAF-16 target genes     larva development.  Riddle DL, Blumenthal T, Meyer BJ, Priess JR, eds                  control C. elegans life-span metabolism. Science 300: 644&#226;&#8364;&#8220;647. <br />     (1997) C elegans II. Cold Spring Harbor: Cold Spring Harbor Laboratory Press.         12. Yu H, Larsen PL (2001) DAF-16-dependent independent expression targets <br />     pp 739&#226;&#8364;&#8220;768.                                                                               DAF-2 insulin receptor-like pathway Caenorhabditis elegans include <br />  5. Samuelson AV, Carr CE, Ruvkun G (2007) Gene activities mediate                       FKBPs. J Mol Biol 314: 1017&#226;&#8364;&#8220;1028. <br />     increased life span C. elegans insulin-like signaling mutants. Genes Dev 21:       13. Gems D, Sutton AJ, Sundermeyer ML, Albert PS, King KV, et al. (1998)     2976&#226;&#8364;&#8220;2994.                                                                                pleiotropic classes daf-2 mutation affect larval arrest, adult behavior, <br />  6. Pinkston-Gosse J, Kenyon C (2007) DAF-16/FOXO targets genes regulate                 reproduction longevity Caenorhabditis elegans. Genetics 150: 129&#226;&#8364;&#8220; <br />     tumor growth Caenorhabditis elegans. Nat Genet advanced online publication.            155. <br />  7. Oh SW, Mukhopadhyay  Dixit BL, Raha T, Green MR, et al. (2006)                      14. Dong M-Q, Venable JD, Au N, Xu T, Park SK, et al. (2007) Quantitative Mass <br />     Identification direct DAF-16 targets controlling longevity, metabolism             Spectrometry Identifies Insulin Signaling Targets C. elegans. Science 317: <br />     diapause chromatin immunoprecipitation. N <br /> </body></html>