<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 4 November 2008                                                     Nucleic Acids Research, 2008, Vol. 36,  22 e151 <br />                                                                                                                   doi:10.1093/nar/gkn811 <br />  <br />  novel method generate unmarked <br /> gene deletions intracellular pathogen <br /> Rhodococcus equi using 5-fluorocytosine <br /> conditional lethality <br /> R. van der Geize1,*, W. Jong1, G.  Hessels1,  W. F. Grommen2,   C. Jacobs2 L. Dijkhuizen1 <br /> 1 <br />  Groningen Biomolecular Sciences Biotechnology Institute (GBB), Department Microbiology, University Groningen, Kerklaan 30, 9751 NN Haren 2Intervet Schering-Plough Animal Health, Bacteriological R<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>;D, <br /> Wim Ko&#194;&#168;rverstraat 35, Postbus 31, 5830 AA Boxmeer, Netherlands <br />  <br /> Received September 16, 2008; Revised Accepted October 10, 2008 <br />  <br />  <br />  <br /> ABSTRACT                                                                        INTRODUCTION novel method efficiently generate unmarked                                 important molecular tool functional genomics stu- frame gene deletions Rhodococcus equi                                 dies targeted inactivation gene  <br /> developed, exploiting cytotoxic effect                                   Ideally, unmarked gene deletions constructed using 5-fluorocytosine (5-FC) action cytosine                               positive selection step rare second recombination <br /> deaminase (CD) <span id='am-43' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> phosphoribosyltransfer-                               event. sacB gene Bacillus subtilis                                                                                 widely used suicide genes, conferring sucrose sensitivity <br /> ase (UPRT) enzymes. opportunistic, intracellular <br />                                                                                 Gram-negative bacteria (1,2). Previously, pathogen R. equi resistant high concentrations <br />                                                                                 reported use sacB counter-selection e&#239;&#172;&#402;ciently 5-FC. Introduction Escherichia coli genes                                 generate unmarked gene deletions Rhodococcus erythro- <br /> encoding CD UPRT conferred conditional                                      polis (3). method subsequently applied lethality R. equi cells incubated 5-FC.                                 Rhodococcus species (4&#226;&#8364;&#8220;6). Sucrose sensitivity exemplify use codA::upp cassette                               sacB reported mycolic acid  <br /> counter-selectable marker, unmarked frame                                 taining actinobacteria, e. Corynebacterium glutamicum <br /> gene deletion mutant R. equi constructed.                                Mycobacterium sp. (7). Despite considerable e&#239;&#172;&#8364;orts, supA supB genes, putative cho-                                unable apply sacB counter-selectable <br /> lesterol catabolic gene cluster, efficiently                               marker Rhodococcus equi, lack sucrose sen- <br /> deleted R. equi wild-type genome. Pheno-                               sitivity. Similar observations reported typic analysis generated &quot;supAB mutant                                   Streptomyces lividans (7). sacB counter-selection <br /> confirmed supAB essential growth                                generally applicable Gram-positive <br /> R. equi <span id='am-48' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span>cholesterol</span>. Macrophage survival assays                              bacteria, like B. subtilis. B. subtilis, mazF, encoding <br /> revealed &quot;supAB mutant able                                      toxin, used suicide marker generate <br />                                                                                 unmarked gene deletion mutants (8). method relies <br /> survive proliferate macrophages comparable <br />                                                                                 availability tightly regulated, inducible pro- wild type.  cholesterol metabolism does                                 moter, limiting applicability bacteria devel- appear essential macrophage                                       oped molecular toolboxes. use alternative suicide <br /> survival R. equi. CD-UPRT based 5-FC                                     genes reported  e.g. glkA, pyrF, upp counter-selection useful asset                                  rpsL (9&#226;&#8364;&#8220;12). major drawback markers,  generation unmarked frame gene deletions                               function glkA, pyrF, upp actinobacteria  actinobacteria                              strA null mutants, respectively, necessitating construc- <br /> generally appear 5-FC resistant 5-FU                                  tion, availability, null mutant strain sensitive.                                                                      mutated. PCR-targeted gene replacement strategy  <br />  <br />  correspondence addressed. Tel: +31 50 363 2257; Fax: +31 50 363 2254; Email: r.van.der.geize@rug.nl <br />  authors wish known  opinion, authors regarded joint Authors. <br />  <br /> &#195;&#376; 2008 Author(s) Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  e151 Nucleic Acids Research, 2008, Vol. 36,  22                                                             PAGE 2 10 <br />  <br />  reported streptomycetes relies avail-       M. tuberculosis H37Rv, mce4 cluster ability ordered genomic libraries e&#239;&#172;&#402;ciently generate       H37Rv essential growth cholesterol. <br /> mutants (13). method,  requires labo-                R. equi strain 103S genome sequence recently rious extra round mutagenesis remove resist-           available (http://www.sanger.ac.uk/Projects/ <br /> ance marker integrated genome generate               R_equi/). Proper genome annotation identi&#239;&#172;?cation <br /> unmarked mutants. Recently, simpli&#239;&#172;?ed method               pathogenicity genes requires simple methods gene <br /> marker removal actinomycetes using Flp recombinase            deletion mutagenesis. method isolation gene described (14).                                         deletion mutants R. equi double homologous <br />    Cytosine deaminase (CD, EC 3.5.4.5) uracil phos-          recombination strategy &#239;&#172;?rst reported Navas et al. <br /> phoribosyltransferase (UPRT, EC 2.4.2.9) enzymes             (27). improved method, using lacZ counter- <br /> involved pyrimidine salvage pathway, converting           selectable marker, subsequently reported Jain <br /> cytosine uracil <span id='am-38' about='obo:IMR_0200008' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00365" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="dUMP" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200008" datatype="xsd:string"></span>dUMP</span>. CD activity              et al. (33). methods,  involve screen- certain prokaryotes lower eukaryotes.           ing handling large numbers colonies select genes encoding activities Escherichia coli, codA        rare second recombination event (34,35). upp, respectively, cloned characterized        introduction codA::upp cassette confers <br /> (15,16). Interestingly, microorganisms expressing CD         5-FC sensitivity R. equi allowing positive selection vert innocuous cytosine analog 5-&#239;&#172;&#8218;uorocytosine               targeted gene deletion mutants. simple e&#239;&#172;&#402;cient <br /> (5-FC) 5-&#239;&#172;&#8218;uorouracil (5-FU), highly toxic com-            procedure generate unmarked frame gene deletions <br /> pound lethal living cells. cytotoxicity largely        R. equi reported, exempli&#239;&#172;?ed construction exerted following UPRT mediated conversion 5-FU               characterization  supAB mutant. 5-&#239;&#172;&#8218;uoro-dUMP, irreversibly inactivates thymi- <br /> dylate synthase inhibiting RNA DNA synthesis <br /> (17). Heterologous expression E. coli codA shown          MATERIALS METHODS confer 5-FC sensitivity mammalian cells, ordinarily        Culture media growth conditions R. equi strain RE1 producing CD (18). Concomitant expression E. coli <br />                                                                  Virulent R. equi wild type strain RE1 isolated CD UPRT fusion protein, encoded codA::upp,           foal pyogranulomatous pneumonia September shown enhance 5-FC cytotoxicity (19).         2007 Netherlands. R. equi strains grown recent study pyrimidine salvage Streptomyces spe-        308C (200 r.p.m.) Luria-Bertani (LB) medium cies indicated lack CD activity sensitivity     mineral acetate medium (MM-Ac). MM-Ac contained <br /> 5-FU (20). actinobacteria, like Rhodococcus           K2HPO4 (4.65 g/l), NaH2PO4   <span id='am-13' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span> (1.5 g/l), sodium- <br /> species, shown 5-FU sensitive (21,22).           acetate (2 g/l), NH4Cl (3 g/l), MgSO4   7H2O (1 g/l), thia- Nocardia species certain Mycobacterium strains,         (40 mg/l, &#239;&#172;?lter sterile) Vishniac stock solution  5-FU resistant, mycobacteria           (1 ml/l). Vishniac stock solution prepared follows <br /> strains highly 5-FU sensitive (21).                         [modi&#239;&#172;?ed (36)]: EDTA (10 g/l) ZnSO4   7H2O <br />    R. equi facultative intracellular pathogen causes   (4.4 g/l) dissolved distilled water (pH 8 using <br /> fatal pyogranulomatous bronchopneumonia young                 2 M KOH).  CaCl2 2H2O (1.47 g/l), MnCl2 7 <br /> foals. emerging opportunistic pathogen          H2O (1 g/l), FeSO4 7H2O (1 g/l), (NH4)6 Mo7O24 4 H2O <br /> immuno-compromised humans, particularly HIV infected             (0.22 g/l), CuSO4 5H2O (0.315 g/l) CoCl2 6 H2O <br /> patients (23,24). addition pathogenic life-style, R.   (0.32 g/l) added order pH 6 &#239;&#172;?nally <br /> equi common soil-dwelling microorganism capable          stored pH 4. growth solid media Bacto-agar <br /> rapid growth soil manure, using plant animal          (15 g/l) added. 5-Fluorocytosine (Sigma-Aldrich <br /> sterols sole carbon energy sources (25,26).               Chemie, Zwijndrecht, Netherlands) 5-&#239;&#172;&#8218;uorouracil <br /> Knowledge sterol metabolism R. equi extremely           (Sigma-Aldrich) stock solutions (10 mg/ml) prepared <br /> limited. far, gene encoding proposed &#239;&#172;?rst        distilled water, dissolved heating 508C, &#239;&#172;?lter- <br /> step cholesterol degradation, cholesterol oxidase             sterilized added autoclaved media. <br /> (choE), identi&#239;&#172;?ed inactivated R. equi               growth cholesterol, R. equi strains inocu- <br /> (27,28). cholesterol catabolic gene cluster           lated 25 ml MM-Ac liquid medium grown 24 h <br /> identi&#239;&#172;?ed closely related Rhodococcus jostii RHA1         200 r.p.m. 308C. pre-culture (0.5 ml) used <br /> (6,29). cluster encodes putative cholesterol         inoculate 50 ml MM medium containing 0.5 g/l cho- <br /> uptake  designated mce4 operon. &#226;&#8364;&#732;Mammalian                lesterol (Sigma-Aldrich) sole carbon energy source <br /> cell entry&#226;&#8364;&#8482; (mce) genes critical virulence factors    &#239;&#172;?nely dispersed sonication. Regular <br /> intracellular pathogen Mycobacterium tuberculosis (30)           turbidity (OD600nm) measurements cholesterol grown  interestingly, mce1 mce4 strains M. tuber-           cultures possible high background culosis H37Rv showed attenuated survival mice (31).           cholesterol suspension. Protein content culture supAB genes mce4 operon              used measure biomass formation encode permease subunits cholesterol uptake           determined follows. sample (0.5 ml) culture (6). supAB mce4 genes shown               pelleted centrifugation (5 min 12 000g) thor- essential growth R. jostii RHA1 cholesterol         oughly resuspended 0.1 ml Bacterial Protein Extraction <br /> (6). Pandey Sassetti (32) recently con&#239;&#172;?rmed             Reagent (B  Pierce, PerBio Science Nederland B.V., <br /> cholesterol used carbon energy source             Etten-Leur, Netherlands).  0.4 ml distilled water <br />  PAGE 3 10                                                           Nucleic Acids Research, 2008, Vol. 36,  22 e151 <br />  <br />  added suspension vortexed incubated          Construction plasmids pSET-Pkan-codA, room temperature 5&#226;&#8364;&#8220;10 min. aliquot 160 ml           pSET-Pkan-codAupp pSelAct mixed 640 ml distilled water 200 ml           aphII promoter region ampli&#239;&#172;?ed pRESQ <br /> protein assay reagent (BioRad) added. Protein            (37) using PCR primers Pkan-F Pkan-E5-R <br /> tent sample determined using bovine serum             (Table 1). obtained PCR product 367 bp albumin (BSA) standard described                  blunt-ligated EcoRV digested pBluescript(II)KS <br /> manufacturer.                                                    (Stratagene), resulting pBs-Pkan. SalI/NotI restric- <br />                                                                  tion released 431 bp fragment comprising aphII pro- <br />                                                                  moter cloned SalI/NotI digested <br /> Cloning, PCR chromosomal DNA isolation <br />                                                                  pORF-codA::upp (InvivoGen, San Diego, USA), yielding <br /> E. coli DH5a used host cloning procedures.        plasmid pORF-Pkan-codAupp. Pkan-codA::upp <br /> Restriction enzymes obtained Fermentas                 cassette subsequently isolated pORF-Pkan- <br /> GmbH (St Leon-Rot, Germany). PCR performed                codAupp 2.4 kb SmaI/NheI fragment ligated reaction mixture (25 ml) consisting Tris&#226;&#8364;&#8220;HCl (10 mM,        EcoRV/XbaI digested pSET152 resulting plasmid <br /> pH 8), 10   High-Fidelity polymerase bu&#239;&#172;&#8364;er (Fermentas),          pSET-Pkan-codAupp (Figure 2). Pkan-codA cassette <br /> dNTPs (0.2 mM), DMSO (2%), PCR primers (10 ng/ml                 (1733 bp) ampli&#239;&#172;?ed pSET-Pkan-codAupp using  Table 1) High-Fidelity polymerase enzyme               primers Pkan-F codA-R2 (Table 1) ligated (1&#226;&#8364;&#8220;2 U, Fermentas). colony PCR, cell material            EcoRV digested pSET152, resulting pSET-Pkan-codA <br /> mixed 100 ml chloroform 100 ml 10 mM              (Figure 2). Suicide plasmid pSelAct (Figure 3)  <br /> Tris&#226;&#8364;&#8220;HCl pH 8, vortexed vigorously centrifuged               structed ligating 2.4 kb Klenow-treated EcoRI/NheI <br /> (2 min 14 000g). sample upper water phase            fragment pORF-Pkan-codAupp SspI digested <br /> (1 ml) subsequently used template PCR. Chro-          pBs-Apra-ori dephosphoryated alkaline phospha- <br /> mosomal DNA R. equi cell cultures isolated using          tase. Plasmid pBs-Apra-ori constructed GenElute Bacterial Genomics DNA Kit (Sigma-                  pBluescript(II)KS bla cassette removed <br /> Aldrich) according instructions manufacturer.      BspHI, followed Klenow treatment replaced <br />                                                                  apramycin-oriT cassette obtained 1.3 kb XbaI <br />                                                                  fragment (Klenow treated) pIJ773 (13). <br /> Electrotransformation R. equi strain RE1 <br />                                                                  Macrophage survival test <br /> R. equi strain RE1 cells transformed electropora- <br /> tion essentially described (27). Brie&#239;&#172;&#8218;y, cell cultures   human monocyte cell line U937 (38) used test <br /> grown 50 ml LB 308C OD600 reached 0.8&#226;&#8364;&#8220;1.0.           survival R. equi strains. monocytes grown cells pelleted (20 min 4500g) washed             RPMI 1640 (Invitrogen) + NaHCO3 (1 g/l) + sodium <br /> twice 10% ice-cold glycerol. Pelleted cells            pyruvate (0.11 g/l) + glucose medium (4.5 g/l) (RPMI suspended 0.5&#226;&#8364;&#8220;1 ml ice-cold 10% glycerol               1640 medium), bu&#239;&#172;&#8364;ered 10 mM HEPES (Hopax <br /> 200 ml aliquots ice. MilliQ-eluted plasmid           &#239;&#172;?ne chemicals, Taiwan) supplemented penicillin <br /> DNA (5&#226;&#8364;&#8220;10 ml; GenElute Plasmid Miniprep Kit, Sigma-              (200 IU/ml), streptomycin (200 IU/ml) 10% fetal <br /> Aldrich) added 200 ml cells 2 mm gapped cuv-           bovine serum (FBS). cells grown suspension <br /> ettes. Electroporation performed single pulse      378C 5% <span id='am-8' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span>CO2</span>. macrophage survival assay, <br /> 12.5 kV/cm, 1000  <br />  25 mF. Electroporated cells          monocytes grown days described <br /> gently mixed 1 ml LB medium allowed recover           culture medium replaced fresh cul- 2 h 378C 200 r.p.m. Aliquots (200 ml)            ture medium cells activated overnight plated LB agar medium containing apramycin                  phorbol 12-myristate 13-acetate (60 ng/ml, PMA, Sigma- <br /> (50 mg/ml; Duchefa Biochemie, Haarlem, Nether-               Aldrich) induce di&#239;&#172;&#8364;erentiation macrophages. <br />                                                                  di&#239;&#172;&#8364;erentiated cells spun (5 min 200g) <br /> lands). Transformants appeared 2&#226;&#8364;&#8220;3 days incuba- <br />                                                                  pellet suspended fresh, antibiotic free <br /> tion 308C. transformation e&#239;&#172;&#402;ciency                                                                  RPMI 1640 medium 10% FBS. strain non-replicative plasmids integrating homologous <br />                                                                  tested, tube containing 10 ml cell suspension <br /> recombination   10 transformants/mg plasmid DNA. <br />                                                                  ( 106 cells/ml) inoculated R. equi, pre-grown                                                                  nutrient broth (Difco, Detroit, MI., USA) 378C, 5-Fluorocytosine positive selection R. equi                   multiplicity infection (MOI) approximately 10 bac- <br />                                                                  teria macrophage. bacteria incubated R. equi transformants inoculated LB liquid               macrophages 1 h 378C 5% CO2. medium (25 ml) grown overnight (20&#226;&#8364;&#8220;24 h) 308C             medium replaced 10 ml RPMI 1640 medium sup- 200 r.p.m. 5-FC selection performed plating           plemented 10% FBS 100 mg/ml gentamycin 100 ml aliquots dilution series (10 1 10 3 MM-Ac      incubated 1 h kill extra-cellular bacteria. <br /> medium) grown culture MM-Ac agar plates              macrophages  internalized R. equi) spun <br /> supplemented 5-FC (100 mg/ml). Dilution cul-         (5 min 200g) pellet suspended ture prior plating crucial e&#239;&#172;&#8364;ective 5-FC selec-       40 ml RPMI 1640 medium, bu&#239;&#172;&#8364;ered 10 mM HEPES <br /> tion. 5-FC resistant colonies appeared 2&#226;&#8364;&#8220;3 days         supplemented 10% FBS gentamycin <br /> incubation 308C.                                              (10 mg/ml). suspension divided culture <br />  e151 Nucleic Acids Research, 2008, Vol. 36,  22                                                                   PAGE 4 10 <br />  <br />  <br /> bottles (10 ml  incubated 378C 5% CO2. 4, 28, 52 76 h macrophages  culture <br /> bottle strain) spun (5 min 200g) pellet washed twice 1 ml antibiotic free RPMI <br /> 1640 medium. Finally pellet lysed 1% <br /> Triton X-100 (Sigma-Aldrich) 0.01 M phosphate buf- <br /> fered saline, followed live count determination (plate <br /> counting). <br />  <br /> Culture media growth conditions 5-FC selection actinobacteria <br /> Actinobacterial strains (Table 2) grown late <br /> exponential phase shaken liquid cultures complex <br /> medium 308C, Amycolatopsis methanolica Mycobacterium smegmatis grown 378C. Tryptic Soy Broth (TSB) used Corynebacter- <br /> ium glutamicum, Arthrobacter globiformis, Amycolatopsis <br /> orientalis,  methanolica M. smegmatis. TSB supple- <br /> mented 2.5% NaCl used Salinospora tropica, YEME:TSB (1:1) (39) Streptomyces strains Saccharopolyspora erythrea. LB used R. equi Rhodococcus rhodochrous LBP (3) used           Figure 1. Introduction E. coli codA::upp cassette, encoding CD R. erythropolis R. jostii. Minimal regeneration medium     UPRT, confers 5-FC sensitivity R. equi strain RE1. Panels wild <br />                                                                type strain RE1, recombinant strain RE1 containing pSET152, recom- <br /> (MRM) consisted K2SO4 (0.25 g/l), (NH4)2SO4 (2 g/l),        binant strain RE1 containing plasmid pSET-codA::upp streaked MgCl2 6H20 (0.6 g/l), 2-[(hydroxyl-1, 1-bis (hydroxy-          mineral acetate agar (MM-Ac) containing   addition, (B) 5-FC <br /> methyl)ethyl)amino] ethanesulfonic acid; TES (5.73 g/l)        (100 mg/ml) (C) 5-FU (50 mg/ml) (D) LB agar containing trace elements (39). Sucrose (10.3% w/v) <span id='am-33' about='obo:IMR_0200183' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="L-Proline" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00148" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200183" datatype="xsd:string"></span>L-proline</span>     5-FC (100 mg/ml). <br /> (0.3% w/v) added MRM medium Streptomyces <br /> strains, omitted strains. Glucose (20 mM) used carbon energy source, Rhodococcus strains grown presence acetate (2 g/l). Autoclaved MRM supplemented KH2PO4 (0.1 g/l), CaCl2 2H20 (3 g/l) NaOH (0.28 g/l). agar plates 2% granulated agar added. Filter- <br /> sterilized thiamine (40 mg/ml) added autoclaved <br /> MRM medium growth Rhodococcus strains. 5-FC 5-FU freshly prepared 10 mg/ml stocks distilled water, dissolved heating 508C added autoclaved medium. <br />                                                                Figure 2. pSET152 derived integrative plasmids used introduce <br /> RESULTS DISCUSSION                                           codA (B) codA::upp cassette R. equi RE1, expressed <br />                                                                control aphII kanamycin resistance cassette promoter <br /> CD-UPRT mediated 5-fluorocytosine sensitivity                  (PaphII). apramycin resistance cassette (aac(3)IV), R. equi strain RE1                                          Streptomyces PhiC31 integrase gene (int) RP4 origin transfer <br />                                                                (oriT) indicated. obvious prerequisite applicability 5-FC <br /> based conditionally lethal positive selection nat- <br /> ural resistance R. equi strain RE1 5-FC. test  R. equi RE1 streaked acetate mineral <br /> (MM-Ac) agar plates supplemented 5-FC 5-FU incubated 3 days 308C. Examination plates revealed R. equi resistant high concen- <br /> trations 5-FC (100 mg/ml), highly sensitive lower concentration 5-FU (50 mg/ml), indicating feasibility developing 5-FC based positive selection R. equi (Figure 1).  examined expression E. coli genes codA upp R. equi confer sensitivity 5-FC act suicide marker. integrative E. coli-Streptomyces shut-     Figure 3. Schematic representation non-replicative plasmid <br /> tle vector pSET152 provided stable convenient vehi-      pSelAct used generate unmarked frame gene deletions R. equi <br /> cle introduce codA functional codA::upp fusion    RE1. Restriction sites indicated unique. <br />  PAGE 5 10                                                         Nucleic Acids Research, 2008, Vol. 36,  22 e151 <br />  <br />  R. equi genome (40,41). aphII promoter region       SupA 54 amino acids SupB, separated kanamycin resistance cassette (Pkan) used           6 bp SmaI restriction site. &#239;&#172;?rst supA <br /> drive expression codA codA::upp, respectively.       supB, respectively, left intact ensure proper resulting integrative plasmids, pSET-Pkan-codA             expression downstream genes mce4 operon. pSET-Pkan-codAupp (Figure 2), mobilized sepa-         Introduction plasmid pSelAct- supAB R. equi <br /> rately electroporation R. equi RE1. Transformants        RE1, selecting apramycin resistance, resulted 29 <br /> harboring pSET-Pkan-codA pSET-Pkan-                  transformants selected grown non- <br /> codAupp streaked MM-Ac agar plates                   selectively overnight 20 h LB medium  e. lacking <br /> (Figure 1A), MM-Ac agar plates supplemented               apramycin 5-FC). Aliquots (100 ml) overnight <br /> 100 mg/ml 5-FC (Figure 1B), MM-Ac agar plates sup-          cultures plated 10 1 10 3 dilutions MM- <br /> plemented 50 mg/ml 5-FU (Figure 1C) incubated         Ac agar plates supplemented 100 mg/ml 5-FC 3 days 308C. presence codA::upp cassette     incubated 3 days 308C. Con&#239;&#172;&#8218;uent growth 5-FC <br /> rendered R. equi sensitive 5-FC, expression      selection plates obtained transformants. <br /> codA did result 5-FC sensitivity (Figure 1B).     Conceivably, codA::upp suicide cassette inac- <br /> Growth R. equi strain carrying codA::upp cassette   tivated transformant spontaneous mutation nonselective conditions similar wild      transposon insertion, reported occur type (Figure 1A), indicating expression condition-     sacB counter-selection (3,42). Typically, &gt;102 colonies <br /> ally lethal detrimental cells 5-FC          grew 10 2 plate 50 colonies replica <br /> omitted medium. Importantly, 5-FC selection        picked LB agar apramycin select observed complex agar media (LB) used             5-FCR/ApraS colonies. frequency 5-FCR/ApraS <br /> (Figure 1D). repression pyrimi-      colonies transformants varied <br /> dine salvage pathway a&#239;&#172;&#8364;ecting 5-FC uptake.                     70&#226;&#8364;&#8220;90%, indicating suicide cassette    enable generation unmarked frame gene          inactivated 10&#226;&#8364;&#8220;30% 5-FCR/ApraS colonies. <br /> deletions R. equi using CD-UPRT based 5-FC condi-           transformant, eighteen 5-FCR/ApraS colonies <br /> tional lethality, developed non-replicative sui-     checked colony PCR presence cide vector designated pSelAct (Figure 3; Materials        mutant  supAB genotype using PCR oligonucleo- Methods section). Vector pSelAct small (5.6 kb)       tides 6 7 amplifying supAB genes (Table 1, <br /> conjugative plasmid based pBlueScript(II)KS, harbor-        Figure 4). eighteen FCR/ApraS colonies gave <br /> ing aac(3)IV cassette apramycin resistance    PCR product expected size (231 bp) used E. coli R. equi. vector contains     selected characterization. Genomic DNA MCS unique restriction sites cloning          isolated  supAB mutants subjected &#239;&#172;&#8218;anking regions gene using lacZ depen-      PCR analysis supAB locus   <br /> dent blue-white screening E. coli.                          stream &#239;&#172;&#8218;anking regions supAB (Figure 5, Table 1                                                                oligonucleotides used). con&#239;&#172;?rmed presence Unmarked frame gene deletion supAB R. equi            genuine supAB gene deletion cases revealed                                                                aberrant genomic rearrangements supAB locus BLAST search supA (ro04696), supB (ro04697)         (Figure 5). mutant strain chosen, designated R. mce4 (ro04698-ro04703) genes R. jostii RHA1,            equi RE1 supAB, used predicted encode cholesterol uptake machinery           characterization. <br /> (6), revealed genes apparent operonic <br /> organization highly conserved R. equi 103S <br />                                                                Phenotypical analysis functional complementation <br /> genome (Figure 4; http://www.sanger.ac.uk/Projects/ <br />                                                                R. equi RE1&quot;supAB mutant <br /> R_equi/). Highest <span id='am-26' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> identities observed SupA (94%) SupB (89%), mce4                R. equi RE1 wild type RE1 supAB mutant strain <br /> encoded proteins Mce4A-Mce4F showed lower identities           grown MM-Ac liquid medium used inocu- <br /> (57&#226;&#8364;&#8220;71%) (Figure 4).                                           late MM-cholesterol liquid medium. RE1 supAB <br />    supA supB genes &#239;&#172;?rst genes       mutant completely blocked growth cholesterol <br /> mce4 operon, encode cholesterol permease           sole carbon energy source (Figure 6). Growth subunits (Figure 4). demonstrate speci&#239;&#172;?c role          acetate steroid substrate 4-androstene-3,17-dione <br /> supAB cholesterol catabolism, aimed selec-     (AD) una&#239;&#172;&#8364;ected comparable wild type <br /> tive inactivation, a&#239;&#172;&#8364;ecting expression          strain (data shown). indicated supAB downstream genes. prompted develop              essential cholesterol catabolism, probably acting unmarked frame gene deletion R. equi             permease subunits cholesterol ABC transporter. <br /> based counter-selection 5-FC. upstream         results fully consistent phenotype downstream &#239;&#172;&#8218;anking regions ( 1.5 kb) supAB genes        supAB mutant strain RHA1 (6). ampli&#239;&#172;?ed PCR ligated pSelAct generat-            ensure growth de&#239;&#172;?ciency ing pSelAct- supAB (Figure 4; Table 1 primers).       RE1 supAB mutant solely deletion construct used generate  supAB mutant,             supAB genes, 3163-bp DNA fragment carrying <br /> reducing wild type supA (765 nt) supB (852 nt)         wild-type supAB genes introduced electropora- <br /> genes, encompassing 1637 bp, single frame open-        tion RE1 supAB mutant. DNA fragment <br /> reading-frame 231 bp, encoding &#239;&#172;?rst 20 amino acid       obtained PCR strain RE1 wild type DNA <br />  e151 Nucleic Acids Research, 2008, Vol. 36,  22                                                                               PAGE 6 10 <br />  <br />  <br />  <br />  <br /> Figure 4.   Schematic overview molecular organization mce4 gene clusters R. jostii RHA1 R. equi 103S. Percentages indicate <br /> amino acid sequence identities Mce4 proteins R. jostii RHA1 R. equi RE1. (B) Construction mutagenic plasmid pSelAct- <br />  supAB used generate unmarked supAB gene deletion mutant R. equi RE1. Small black arrows numbers indicate PCR oligonu- <br /> cleotides used study (Table 1) site annealing. (C) Molecular organization mce4 locus R. equi RE1 following <br /> integration pSelAct- supAB single homologous recombination, second homologous recombinant event 5-FC counter-selection, <br /> resulting supAB gene deletion. <br />  PAGE 7 10                                                                        Nucleic Acids Research, 2008, Vol. 36,  22 e151 <br />  <br />  <br /> Table 1. Oligonucleotides used study <br />    PCR amplicon                                             Size (bp)         Oligonucleotide sequence (nr.) <br />  <br /> 1    Upstream region supAB (deletion construct)               1592              supABequiUP-F ATCGCGAGGTCAGCTTGGAG (1) <br />                                                                                 supABequiUP-R CCCGGGCGCCGACATCGCGAAGAATC (2) <br /> 2    Downstream region supAB (deletion construct)             1470              supABequiDOWN-F CCCGGGCTCATCCACACCTACTACGG (3) <br />                                                                                 supABequiDOWN-R ACTAGTGAGCTGCTGAATCTGAACTGG (4) <br /> 3    Upstream region supAB (con&#239;&#172;?rmation deletion mutant)      1786 (wt: 3192)   supABequiContrUP-F CGGGAGTGCGTAGATGAGTGCA (5) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 4    Downstream region supAB (con&#239;&#172;?rmation deletion mutant)    1570 (wt: 2976)   supABequiContr-F GTGGTCGACCTCCTCGAGGTAC (7) <br />                                                                                 supABequiContrDOWN-R GGACTTGAGCCCGGAGGCATCG (8) <br /> 5    supAB genes (con&#239;&#172;?rmation deletion mutant)                 231 (wt: 1637)   supABequiContr-F GTGGTCGACCTCCTCGAGGTAC (7) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 6    supAB genes (complementation  supAB mutant)              3163              supABequiUP-F ATCGCGAGGTCAGCTTGGAG (1) <br />                                                                                 supABequiContr-R TTATCCCGAAAGGTTGAAGTTG (6) <br /> 7    aphII promoter region                                     367              Pkan-F AGCTTCACGCTGCCGCAAGCACT <br />                                                                                 Pkan-E5-R GATATCATGCGAAACGATCCTCATCCTG <br /> 8    Pkan-codA cassette                                       1733              Pkan-F AGCTTCACGCTGCCGCAAGCACT <br />                                                                                 codA-R2 GTCAACGTTTGTAATCGATGGCTTCT <br /> 9    vapA                                                      408              vapA-F GCAGCAGTGCGATTCTCAATAG (9) <br />                                                                                 vapA-R TAACTCCACCGGACTGGATATG (10) <br />  <br /> Primer numbering used text shown brackets following <span id='am-18' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span> sequence. <br />  <br />  <br />  <br />  <br /> Figure 5. PCR analysis wild type R. equi strain RE1 (wt)  supAB gene deletion mutants R. equi strain RE1 (1 2). Panels ampli&#239;&#172;?cation   supAB locus [6 + 7], (II) upstream region supAB [5 + 6], (III) downstream region supAB [7 + 8], (IV) vapA <br /> virulence gene [9 + 10]. oligonucleotides used mentioned brackets described Table 1 Figure 4. <br />  <br />  <br /> using oligonucleotides 1 6 (Table 1, Figure 4) cloned integrative plasmid SET152, resulting pSET-supAB. cholesterol growth negative phenotype RE1 supAB mutant harboring pSET-supAB fully complemented, restoring growth cholesterol levels comparable wild type (Figure 6). conclude supAB deletion did exert polar e&#239;&#172;&#8364;ects expression genes mce4 operon. <br />    data developed novel, simple e&#239;&#172;&#402;cient method generate unmarked frame gene <br /> deletions R. equi, based 5-FC lethality pre- <br /> sence E. coli codA::upp fusion gene. <br />  <br /> Deletion supAB does affect R. equi RE1 survival macrophages <br /> Intracellular survival proliferation R. equi <br /> RE1 supAB mutant human monocyte cell line <br /> U937 compared wild type strain RE1 <br /> (Figure 7). avirulent, plasmid free strain R. equi <br /> 103 (43) included negative control macro-                      Figure 6. Growth curves mineral medium supplemented choles- <br /> phage survival (Figure 7). results revealed                    terol (0.5 g/l) R. equi RE1 wild-type (&#239;&#172;?lled diamond), RE1  supAB <br /> RE1 supAB mutant able survive proliferate                      mutant strain (&#239;&#172;?lled square), RE1  supAB + pSET-supAB complemen- <br />                                                                             ted strain (&#239;&#172;?lled triangle), RE1  supAB + pSET152 control strain <br /> macrophages comparable wild-type parent strain                       (x). Protein content (mg/l) culture used measure RE1. contrast, avirulent strain 103 failed pro-                   biomass formation. data represent averages indepen- <br /> liferate, resulting reduced numbers intracellular                     dent experiments. <br />  e151 Nucleic Acids Research, 2008, Vol. 36,  22                                                                       PAGE 8 10 <br />  <br />  <br /> bacteria time (Figure 7). results indicate               Actinobacteria generally 5-FC resistant <br /> cholesterol metabolism essential macrophage                 CD-UPRT based 5-FC selection survival R. equi RE1 suggest cholesterol                   useful asset generation unmarked frame gene <br /> metabolism important virulence R. equi                   deletions actinobacteria. examine RE1 vivo. observations consistent                CD-UPRT selection potentially e&#239;&#172;&#8364;ective counter- <br /> &#239;&#172;?nding ChoE important virulence                  selectable marker family microorganisms, R. equi (28).                                                          tested sensitivity actinobacteria                                                                        5-FC 5-FU. wide selection actinobacterial strains <br />                                                                        grown complex medium late exponential/ <br />                                                                        early stationary phase plated 10-fold dilutions <br />                                                                        mineral selection media containing increasing  <br />                                                                        centrations 5-FC 5-FU (Table 2). tested strains <br />                                                                        generally resistant high concentrations 5-FC <br />                                                                        (100 mg/ml), highly sensitive lower concentrations <br />                                                                        5-FU (20&#226;&#8364;&#8220;50 mg/ml), indicating feasibility devel- <br />                                                                        oping 5-FC based positive selection                                                                        actinobacteria. <br />  <br />                                                                        ACKNOWLEDGEMENTS <br />                                                                        sequence data R. equi 103S produced                                                                        R. equi Sequencing Group Sanger Institute                                                                        obtained ftp://ftp.sanger.ac.uk/pub/pathogens  <br />                                                                        thank J  Salas (Oviedo, Spain), E.M.H. <br />                                                                        Wellington (Warwick, UK) M. Fukuda (Nagaoka, <br />                                                                        Japan) providing strains. <br />  <br /> Figure 7. Survival proliferation R. equi strains human <br /> monocyte cell line U937. Macrophage cell suspensions infected <br />                                                                        FUNDING wild type virulent strain R. equi RE1 (&#239;&#172;?lled diamond), mutant     Intervet Schering-Plough Animal Health, Boxmeer, <br /> strain R. equi RE1 supAB (&#239;&#172;?lled square) non-virulent (control) <br /> strain R. equi 103 (&#239;&#172;?lled triangle). Following 1-h incubation                                                                        Netherlands. Funding open access charge: <br /> allow phagocytosis, cells washed treated gentamycin   Groningen Biomolecular Sciences Biotechnology <br /> kill remaining extra-cellular bacteria. numbers intracellular   Institute (GBB), University Groningen, bacteria determined plate counts following macrophage lysis.   Netherlands. data represent averages independent experiments. <br /> Plate counts carried duplicate.                            Con&#239;&#172;&#8218;ict statement. declared. <br />  <br />  <br /> Table 2. Growth actinobacteria MRM mineral agar media supplemented di&#239;&#172;&#8364;erent concentrations 5-FC 5-FU <br />  <br /> Strain                                                  5-FC (mg/ml)                 5-FU (mg/ml)          Reference/origin <br />  <br />                                              0         20       50     100      20       50         100 <br />  <br /> Amycolatopsis methanolica NCIB11946          ++        +                        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (44) <br /> Amycolatopsis orientalis ATCC 19795          ++        ++       +      +                                   ATCC <br /> Arthrobacter globiformis DSM 20124           ++        ++       +      +        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Corynebacterium glutamicum ATCC 13032        ++        +                                 &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      ATCC <br /> Mycobacterium smegmatis mc2155               ++        +                        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (45) <br /> Rhodococcus rhodochrous DSM 43269            ++        ++       ++     ++       +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Rhodococcus erythropolis SQ1                 ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (46) <br /> Rhodococcus jostii RHA1                      ++        ++       ++     ++       +                   &#226;&#8364;&#8220;      M. Fukuda, Nagaoka (Japan) <br /> Rhodococcus equi RE1                         ++        ++       ++     ++       +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      Schering-Plough, Netherlands <br /> Salinospora tropica                          ++        ++       ++     ++                                  DSMZ <br /> Saccharopolyspora erythraea                  ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Streptomyces albus J1074                     ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (47) <br /> Streptomyces avermitilis MA-4680             ++        ++       ++     ++       ++       ++         ++     ATCC (ATCC 31267) <br /> Streptomyces coelicolor M145                 ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (39) <br /> Streptomyces griseus DSM40236                ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      DSMZ <br /> Streptomyces lividans TK23                   ++        ++       ++     ++       &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (39) <br /> Streptomyces scabies ISP5078                 ++        ++       ++     +        &#226;&#8364;&#8220;        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      Wellington, Warwick (UK) <br /> Streptomyces tendae Tu&#194;&#168; 901/8c               ++        ++       ++     +        +        &#226;&#8364;&#8220;          &#226;&#8364;&#8220;      (48) <br />  <br /> Symbols indicate growth (++), moderate growth (+), slight/minor growth ( ) growth (&#226;&#8364;&#8220;). <br /> ATCC, American Type Culture Collection; DSMZ, Deutsche Sammlung von Mikroorganismen und Zellkulturen. <br />  PAGE 9 10                                                                      Nucleic Acids Research, 2008, Vol. 36,  22 e151 <br />  <br />  <br /> REFERENCES                                                                     cytotoxicity 5-&#239;&#172;&#8218;uorocytosine. . FEMS Microbiol. Lett., <br />                                                                                167, 41&#226;&#8364;&#8220;49. <br />  1. Ried,J.L. Collmer  (1987) nptI-sacB-sacR cartridge        20. Hughes,L.E., Beck,D  O&#226;&#8364;&#8482;Donovan,G  (2005) <br />     constructing directed, unmarked mutations gram-negative                 Pathways pyrimidine salvage Streptomyces. Curr. Microbiol., <br />     bacteria marker exchange-eviction mutagenesis. Gene,                    50, 8&#226;&#8364;&#8220;10. <br />     57, 239&#226;&#8364;&#8220;246.                                                           21. Tsukamura,M. (1981) Di&#239;&#172;&#8364;erentiation genera <br />  2. Reyrat,J.M., Pelicic,V., Gicquel,B. Rappuoli,R. (1998)                 Mycobacterium, Rhodococcus Nocardia susceptibility     Counterselectable markers: untapped tools bacterial genetics           5-&#239;&#172;&#8218;uorouracil. J. Gen. Microbiol., 125, 205&#226;&#8364;&#8220;208. <br />     pathogenesis. Infect. Immun., 66, 4011&#226;&#8364;&#8220;4017.                       22. Tsukamura,M., Komatsuzaki,C., Sakai,R., Kaneda,K., Kudo,T. <br />  3. Van der Geize,R., Hessels,G , Van Gerwen,R., Van der Meijden,P.          Seino  (1988) Mesenteric lymphadenitis swine caused     Dijkhuizen,L. (2001) Unmarked gene deletion mutagenesis             Rhodococcus sputi. J. Clin. Microbiol., 26, 155&#226;&#8364;&#8220;157. <br />     kstD, encoding 3-ketosteroid  1-dehydrogenase, Rhodococcus          23. Prescott,J.F. (1991) Rhodococcus equi: animal human <br />     erythropolis SQ1 using sacB counter-selectable marker. FEMS             pathogen. Clin. Microbiol. Rev., 4, 20&#226;&#8364;&#8220;34. <br />     Microbiol. Lett., 205, 197&#226;&#8364;&#8220;202.                                        24. Meijer,W.G. Prescott,J.F. (2004) Rhodococcus equi. Vet. Res., <br />  4. Nga,D.P., Altenbuchner,J. Heiss,G.S. (2004) NpdR, repressor          35, 383&#226;&#8364;&#8220;396. <br />     involved 2,4,6-trinitrophenol degradation Rhodococcus opacus     25. Ahmad,S., Roy,P.K., Basu,S.K. Johri,B.N. (1993) Cholesterol <br />     HL PM-1. J. Bacteriol., 186, 98&#226;&#8364;&#8220;103.                                       chain cleavage immobilized cells Rhodococcus equi <br />  5. Sharp,J.O., Sales,C.M., LeBlanc,J.C., Liu,J., Wood,T.K.,                   DSM89-133. Indian J. Exp. Biol., 31, 319&#226;&#8364;&#8220;322. <br />     Eltis,L.D., Mohn,W.W. Alvarez-Cohen,L. (2007) inducible         26. Murohisa,T. Iida,M. (1993) new intermediates micro- <br />     propane monooxygenase responsible N-nitrosodimethylamine            bial chain degradation b-sitosterol. J. Ferment. Bioeng., <br />     degradation Rhodococcus sp. strain RHAI. Appl. Environ.                 76, 174&#226;&#8364;&#8220;177. <br />     Microbiol., 73, 6930&#226;&#8364;&#8220;6938.                                             27. Navas,J., Gonza&#194;&#180;lez-Zorn,B., Ladro&#194;&#180;n,N., Garrido,P. Va&#194;&#180;zquez- <br />  6. Van der Geize,R., Yam,K., Heuser,T., Wilbrink,M.H., Hara,H.,               Boland,J  (2001) Identi&#239;&#172;?cation mutagenesis allelic <br />     Anderton,M.C., Sim,E., Dijkhuizen,L., Davies,J.E., Mohn,W.W.               exchange choE, encoding cholesterol oxidase     et al. (2007) gene cluster encoding cholesterol catabolism          intracellular pathogen Rhodococcus equi. J. Bacteriol., <br />     soil actinomycete provides insight Mycobacterium tuberculosis         183, 4796&#226;&#8364;&#8220;4805. <br />     survival macrophages. Proc. Natl Acad. Sci. USA,                    28. Pei,Y., Dupont,C., Sydor,T., Haas  Prescott,J.F. (2006) <br />     104, 1947&#226;&#8364;&#8220;1952.                                                            Cholesterol oxidase (ChoE) important virulence  7. Ja&#194;&#168;ger,W., Scha&#194;&#168;fer , Pu&#194;&#168;hler , Labes,G. Wohlleben,W. (1992)       Rhodococcus equi. Vet. Microbiol., 118, 240&#226;&#8364;&#8220;246. <br />     Expression Bacillus subtilis sacB gene leads sucrose         29. McLeod,M.P., Warren,R.L., Hsiao,W.W., Araki,N., Myhre,M., <br />     sensitivity gram-positive bacterium Corynebacterium                 Fernandes,C., Miyazawa,D., Wong,W., Lillquist L., Wang,D. <br />     glutamicum Streptomyces lividans. J. Bacteriol.,                et al. (2006) complete genome Rhodococcus sp. RHA1 <br />     174, 5462&#226;&#8364;&#8220;5465.                                                            provides insights catabolic powerhouse. Proc. Natl Acad. Sci. <br />  8. Zhang,X , Yan,X., Cui L., Hong,Q. Li,S.P. (2006) mazF,           USA, 103, 15582&#226;&#8364;&#8220;15587. <br />     novel counter-selectable marker unmarked chromosomal               30. Rengarajan,J., Bloom,B.R. Rubin,E.J. (2005) Genome-wide <br />     manipulation Bacillus subtilis. Nucleic Acids Res., 34, e71.            requirements Mycobacterium tuberculosis adaptation  9. Van Wezel,G.P. Bibb,M.J. (1996) novel plasmid vector            survival macrophages. Proc. Natl Acad. Sci. USA, <br />     uses glucose kinase gene (glkA) positive selection          102, 8327&#226;&#8364;&#8220;8332. <br />     stable gene disruptants Streptomyces. Gene, 182, 229&#226;&#8364;&#8220;230.           31. Joshi,S.M., Pandey K., Capite,N., Fortune,S.M., Rubin,E.J. 10. Hosted,T.J. Baltz,R.H. (1997) Use rpsL dominance                Sassetti,C.M. (2006) Characterization mycobacterial virulence <br />     selection gene replacement Streptomyces roseosporus. J.             genes genetic interaction mapping. Proc. Natl Acad. Sci. <br />     Bacteriol., 179, 180&#226;&#8364;&#8220;186.                                                  USA, 103, 11760&#226;&#8364;&#8220;11765. <br /> 11. Knipfer,N., Seth  Shrader,T.E. (1997) Unmarked gene inte-        32. Pandey K. Sassetti,C.M. (2008) Mycobacterial persistence <br />     gration chromosome Mycobacterium smegmatis                 requires utilization host cholesterol. Proc. Natl Acad. Sci. <br />     precise replacement pyrF gene. Plasmid, 37, 129&#226;&#8364;&#8220;140.                USA, 105, 4376&#226;&#8364;&#8220;4380. <br /> 12. Fabret,C., Ehrlich,S.D. Noirot,P. (2002) new mutation            33. Jain,S., Bloom,B.R. Hondalus,M.K. (2003) Deletion     delivery genome-scale approaches Bacillus subtilis.          vapA encoding Virulence Associated Protein attenuates     Mol. Microbiol., 46, 25&#226;&#8364;&#8220;36.                                                intracellular actinomycete Rhodococcus equi. Mol. Microbiol., <br /> 13. Gust,B., Challis,G.L., Fowler,K., Kieser,T. Chater,K.F. (2003)         50, 115&#226;&#8364;&#8220;128. <br />     PCR-targeted Streptomyces gene replacement identi&#239;&#172;?es protein         34. Pei,Y., Nicholson,V., Woods,K. Prescott,J.F. (2007) <br />     domain needed biosynthesis sesquiterpene soil odor              Immunization intrabronchial administration 1-week-old foals <br />     geosmin. Proc. Natl Acad. Sci. USA, 100, 1541&#226;&#8364;&#8220;1546.                        unmarked double gene disruption strain Rhodococcus equi <br /> 14. Fedoryshyn,M., Petzke,L., Welle,E., Bechthold                        strain 103+. Vet. Microbiol., 125, 100&#226;&#8364;&#8220;110. <br />     Luzhetskyy  (2008) Marker removal actinomycetes genome          35. Pei,Y., Parreira,V., Nicholson,V.M. Prescott,J.F. (2007) <br />     using Flp recombinase. Gene, 419, 43&#226;&#8364;&#8220;47.                                   Mutation virulence assessment chromosomal genes 15. Andersen,L., Kilstrup,M. Neuhard,J. (1989) Pyrimidine, purine          Rhodococcus equi 103.  J. Vet. Res., 71, 1&#226;&#8364;&#8220;7. <br />     nitrogen control cytosine deaminase synthesis Escherichia    36. Vishniac,W. Santer,M. (1957) thiobacilli. Bacteriol. Rev., <br />     coli K 12. Involvement glnLG purR genes regu-            21, 195&#226;&#8364;&#8220;213. <br />     lation codA expression. Arch. Microbiol., 152, 115&#226;&#8364;&#8220;118.             37. Van der Geize,R., Hessels,G , Van Gerwen,R., Van der Meijden,P. <br /> 16. Danielsen,S., Kilstrup,M., Barilla,K., Jochimsen,B. Neuhard,J.         Dijkhuizen,L. (2002) Molecular functional characterization <br />     (1992) Characterization Escherichia coli codBA operon               kshA kshB, encoding components 3-ketosteroid <br />     encoding cytosine permease cytosine deaminase. Mol.                    9a-hydroxylase, class IA monooxygenase, Rhodococcus <br />     Microbiol., 6, 1335&#226;&#8364;&#8220;1344.                                                  erythropolis SQ1. Mol. Microbiol., 45, 1007&#226;&#8364;&#8220;1018. <br /> 17. Heidelberger,C., Danenberg,P.V. Moran,R.G. (1983)                  38. Sundstrom,C. Nilsson,K. (1976) Establishment character- <br />     Fluorinated pyrimidines nucleosides. Adv. Enzymol. Relat.        ization human histiocytic lymphoma cell line (U937). Int. J. <br />     Areas Mol. Biol., 54, 58&#226;&#8364;&#8220;119.                                              Cancer, 17, 565&#226;&#8364;&#8220;577. <br /> 18. Mullen,C., Kilstrup,M. Blaese,R.M. (1992) Transfer          39. Kieser,T., Bibb,M.J., Buttner,M.J., Chater,K.F.     bacterial gene cytosine deaminase mammalian cells confers           Hopwood,D  (2000) Practical Streptomyces Genetics, John Innes <br />     lethal sensitivity 5-&#239;&#172;&#8218;uorocytosine: negative selection         Foundation, Norwich, UK. <br />     Proc. Natl Acad. Sci. USA, 89, 33&#226;&#8364;&#8220;37.                                  40. Bierman,M., Logan,R., O&#226;&#8364;&#8482;Brien,K., Seno,E.T., Rao,R.N. 19. Tiraby,M., Cazaux,C., Baron,M., Drocourt,D., Reynes,J.P.               Schoner,B.E. (1992) Plasmid cloning vectors conjugal <br />     Tiraby,G. (1998) Concomitant expression E. coli cytosine                transfer DNA Escherichia coli Streptomyces spp. Gene, <br />     deaminase uracil phosphoribosyltransferase improves                116, 43&#226;&#8364;&#8220;4 <br /> </body></html>