<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization PMT Gene Family Cryptococcus <br /> neoformans <br /> Sven D. Willger1,2*, Joachim F. Ernst1, J. Andrew Alspaugh3, Klaus B. Lengeler1 <br /> 1 Institut fu&#194;&#168;r Mikrobiologie, Molekulare Mykologie, Heinrich-Heine-Universita&#194;&#168;t, Du&#194;&#168;sseldorf, Germany, 2 Department Veterinary Molecular Biology, Montana State <br /> University, Bozeman, Montana, United States America, 3 Departments Medicine Molecular Genetics/Microbiology, Duke University Medical Center, Durham, <br /> North Carolina, United States America <br />  <br />  <br />  <br />      Abstract <br />      Background: Protein-O-mannosyltransferases (Pmt&#226;&#8364;&#8482;s) catalyze initial step protein-O-glycosylation, addition      mannose residues serine threonine residues target proteins. <br />  <br />      Methodology/Principal Findings: Based protein similarities, highly conserved protein family divided      subfamilies: Pmt1 sub-family, Pmt2 sub-family Pmt4 sub-family. contrast Saccharomyces <br />      cerevisiae Candida albicans, similar filamentous fungi, putative PMT genes (PMT1, PMT2, PMT4)      identified genome human fungal pathogen Cryptococcus neoformans. Similar Schizosaccharomyces pombe <br />      C. albicans, C. neoformans PMT2 essential gene. contrast, pmt1 pmt4 single mutants viable;  <br />      pmt1/pmt4 deletions synthetically lethal. Mutation PMT1 PMT4 resulted distinct defects cell morphology <br />      cell integrity. pmt1 mutant susceptible SDS medium wild-type strains mutant cells      enlarged. pmt4 mutant grew poorly high salt medium demonstrated abnormal septum formation defects      cell separation. Interestingly, pmt1 pmt4 mutants demonstrated variety-specific differences levels      susceptibility osmotic cell wall stress. Delayed melanin production pmt4 mutant alteration      classical virulence-associated phenotypes.  pmt1 pmt4 mutants showed attenuated virulence murine <br />      inhalation model cryptococcosis. <br />  <br />      Conclusion/Significance: findings suggest C. neoformans protein-O-mannosyltransferases play crucial role      maintaining cell morphology, reduced protein-O-glycosylation leads alterations stress resistance, cell wall <br />      composition, cell integrity, survival host. <br />  <br />   Citation: Willger SD, Ernst JF, Alspaugh JA, Lengeler KB (2009) Characterization PMT Gene Family Cryptococcus neoformans. PLoS 4(7): e6321. <br />   doi:10.1371/journal.pone.0006321 <br />   Editor: Xiaorong Lin, Texas <span id='am-21' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; M University, United States America <br />   Received 8, 2009; Accepted June 23, 2009; Published July 27, 2009 <br />   Copyright: &#195;&#376; 2009 Willger et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: study supported DAAD-short-term-scholarship (Deutscher Akademischer Austausch Dienst, http://www.daad index.html) <br />   Deutsche Forschungsgemeinschaft (SPP1160)(http://www.dfg . funders role study design, data collection analysis, decision   publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: sven.willger@montana.edu <br />  <br />  <br />  <br /> Introduction                                                                        Broader phylogenetic analyses reveal protein-O-mannosyltrans- <br />                                                                                     ferases grouped major subfamilies, corresponding     Protein-O-glycosylation essential, evolutionary conserved                 S. cerevisiae Pmt1p, Pmt2p Pmt4p. fungal species contain <br /> protein modification studied extensively yeasts                PMT genes genome, subfamily. <br /> Saccharomyces cerevisiae Candida albicans. process                   S. cerevisiae, O-glycosylation affects stability, localization, identified fungal species [1&#226;&#8364;&#8220;3], higher eukaryotes                 function proteins, preventing exportation misfolded <br /> [4&#226;&#8364;&#8220;6] certain bacterial genera [7,8]. yeasts fungi,                   proteins ER [12&#226;&#8364;&#8220;14]. Similarly, human pathogenic <br /> protein-O-glycosylation initiated luminal                     fungus C. albicans, correct O-mannosylation important endoplasmic reticulum (ER) addition mannosyl residue                    morphogenesis, adherence host cells, virulence [15]. C. specific serine/threonine residues proteins entering                      albicans proteins modified Pmt&#226;&#8364;&#8482;s include chitinases, <br /> secretory pathway [9,10]. modification derived                   proteases, proteins involved glucan synthesis, heat-shock proteins, polyisoprenoid carrier <span id='am-5' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> dolichyl phosphate-activated                       cell-surface antigens important virulence (reviewed [2,3]). <br /> mannose (Dol-P-Man), followed addition short, linear,                     addition secreted proteins, proper function various <br /> mannosyl-rich glycans. Maturation modification                   receptors requires intact protein-O-glycosylation activity [14,16]. <br /> glycosyl chains occur Golgi apparatus.                                          protein-O-mannosylation abundant     initial step O-glycosylation catalysed highly conserved          higher eukaryotes, defects process result human disease, <br /> family integral ER membrane proteins, protein-O-mannosyl-                    muscle-eye-brain disease (MEB) Walker&#226;&#8364;&#8220;Warburg <br /> transferases (Pmt&#226;&#8364;&#8482;s) [11]. Pmt family initially identified S.            Syndrome (WWS) [17]. Furthermore, targeted deletion cerevisiae, seven PMT genes identified [3,11].                   POMT1, causes WWS mice resulted embryonic <br />  <br />  <br />        PLoS | www.plosone.org                                               1                                    July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                       C. neoformans PMT Gene Family <br />  <br />  <br /> lethality [18]. Mutation Drosophila PMT homologs alters muscle              growth conditions. wild-type strains JEC21 (serotype D) structures alignment adult cuticle [19,20].               H99 (serotype  incubated mid-logarithmic phase rich analysis PMT-gene family different species              medium (YPD) 30uC 37uC, salt stress (YPD+1 M NaCl), years revealed protein-O-mannosylation activity               capsule-inducing conditions (DMEM). Northern blots involved central developmental growth processes                 quantitative real-time PCR demonstrated little variation uni- multicellular eukaryotes.                                             expression PMT genes strain variety <br />     C. neoformans opportunistic human fungal pathogen causing            conditions (data shown).  C. <br /> life-threatening meningoencephalitis. fungus produces             neoformans PMT genes constitutively expressed extracellular factors important virulence, including           growth conditions. data support confirm recent <br /> extensive <span id='am-1' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span>polysaccharide</span> capsule, secreted hydrolytic                  observation PMT4A gene induced changes enzymes, cell wall-targeted pigment melanin (reviewed                  temperature nutrient availability [22]. [21]). protein-O-glycosylation predominantly affects <br /> extracellular proteins, defect biological process              Protein-O-glycosylation essential C. neoformans <br /> affect interface pathogenic microorganism host.                    used targeted gene disruption characterise     PMT4, putative Pmt orthologs basidiomycete             biological functions C. neoformans PMT genes C. neoformans, recently identified. Disruption C.              serotype backgrounds. shown organisms <br /> neoformans PMT4 gene results dramatic effects virulence [22].            pmt mutant strains highly sensitive cell wall addition, pmt4 mutant strains morphological defects                destabilizing drugs drugs targeting protein synthesis, including <br /> alterations cell wall, possibly changes glycan                used C. neoformans dominant selectable markers  composition/synthesis.  relative roles C. neoformans            hygromycin B), used ADE2 URA5 genes selectable <br /> PMT1 PMT2 elucidated. paper                   markers perform experiments. selectable marker <br /> report continuing analysis complete C. neoformans PMT                 cassettes inserted loop region PMT genes, <br /> gene family biologically distinct varieties, var. grubii var.       predicted essential enzyme function [27]. <br /> neoformans. identified C. neoformans PMT genes,                   pmt1 pmt4 single mutant strains easily begun define overlapping distinct functions               isolated serotypes, able isolate pmt2 mutant <br /> stress response, cell wall integrity, survival host.                strains serotype, rounds                                                                                transformation (.300 transformants). finding suggested Results                                                                        PMT2 essential gene C. neoformans, S. pombe <br />                                                                                C. albicans [15,28]. confirm hypothesis, PMT genes present C. neoformans                                   isolated homozygous ade2/ade2 diploid strain cross    Fungal genomes typically contain multiple genes encoding                    strains JEC156 (MATa ade2 ura5) JEC157 (MATalpha ade2 ura5 <br /> protein-O-mannosyltransferases. example, S. cerevisiae contains            lys1) according methods Sia et al. [29]. transforming seven PMT genes (PMT1-7), PMT genes present              pmt2::ADE2 disruption construct diploid strain, genome human pathogenic fungus C. albicans. putative              able isolate heterozygous PMT2/pmt2::ADE2 strains similar <br /> PMT genes recently identified basidiomycetous yeast C.             frequencies (,10%) compared pmt1 pmt4 single <br /> neoformans variety neoformans (serotype D), major sub-            mutants. allowed independent heterozygous diploid <br /> family: PMT1 (CND06150 chromosome 4), PMT2 (CNJ01930                        mutants sporulate. sporulation, isolated 75 chromosome 10), PMT4 (CND01240 chromosome 4)                         ADE+haploid progeny, wild-type PMT2 locus. [23]. C. neoformans Pmt1 shares 41% <span id='am-43' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-44' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-49' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> identity 59%                contrast, genetic markers (mating type, LYS1) demon- <br /> similarity S. cerevisiae Pmt1; CnPmt2 47% identity 65%              strated expected levels recombination. assume similarity ScPmt2; CnPmt4 42% identity 61%                      ADE+progeny pmt2, ectopic integration <br /> similarity ScPmt4. direct amino acid sequence compar-                pmt2::ADE recombination restores ade2 allele <br /> ison, detailed phylogenetic sequence analysis places            wild-type. determine isolated 75 ADE+haploid C. neoformans PMT genes major PMT              progeny wild-type PMT2 locus performed Southern <br /> gene families. Hydrophobicity analysis predicted Pmt protein            Blot analyses diploid strains sporulated showing <br /> sequences revealed seven-transmembrane helical structure                     PMT2 pmt2::ADE2 alleles present (data commonly predicted class proteins [9].                             shown). progeny subsequently analyzed colony <br />    closely related, C. neoformans var. neoformans var.            PCR nature PMT2 allele present strains <br /> grubii strains (serotypes D  respectively) distinct                 amplifying short fragment spanning region ADE2 <br /> characteristics terms intracellular signaling cellular               marker integrated (data shown). failure isolate <br /> physiology [24&#226;&#8364;&#8220;26]. scientific models, var. grubii strains             haploid pmt2 mutants sporulation PMT2/pmt2 used extensively pathogenesis experiments, var.               diploid strongly suggests PMT2 gene essential. <br /> neoformans strains tractable mating genetic systems.                verify insertion selectable marker cassette PMT genes present var. grubii genome,                resulted loss transcript pmt1 pmt 4 mutants <br /> encoding proteins 97&#226;&#8364;&#8220;98% identical amino acid                  performed real time PCRs PMT1, PMT2, PMT4 <br /> level var. neoformans. clarity, chosen            specific primers cDNA derived serotype D mutants <br /> refer var. grubii (serotype  PMT genes PMT1A, PMT2A,               grown 30uC. control used cDNA derived PMT4A; var. neoformans (serotype D) genes PMT1D,                serotype D wild-type strain JEC21. couldn&#226;&#8364;&#8482;t generate PMT2D, PMT4D.                                                              pmt2 mutant normalized data strain PMT2 <br />                                                                                expression. results confirm PMT genes PMT genes C. neoformans constitutively expressed                        constitutively expressed little variation various growth conditions                                                wild-type strain JEC21 (Figure 1). insertion mutants    characterized transcriptional regulation entire C.            transcript PMT1 PMT4 completely abolished neoformans PMT gene family various physiologically relevant              respective mutant. real time PCR data clearly  <br />  <br />        PLoS | www.plosone.org                                          2                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                     C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 1. PMTs differentially expressed loss pmt gene compensated overexpression PMT <br /> gene. wild-type (JEC21), Dpmt1 Dpmt4 mutant strains incubated mid-logarithmic phase YPD medium total RNA isolated. <br /> Quantitative real-time PCR performed corresponding cDNA samples assess relative PMT1, PMT2, PMT4 transcript abundance <br /> compared wild-type strain6one standard deviation. expression values strains normalized PMT2 expression. <br /> doi:10.1371/journal.pone.0006321.g001 <br />  PMT genes differentially expressed               properly mother cells, unusually large amounts cell <br /> mutants compensate loss PMT gene (Figure 1).                wall chitin area (Figure 4, arrow). results confirmed <br />                                                                               recently published data pmt4A mutant cell &#226;&#8364;&#732;&#226;&#8364;&#732;Pmt holoenzyme&#226;&#8364;&#8482;&#226;&#8364;&#8482; functions dimer multimer                       aggregates connected degraded cell wall material individual Pmt proteins                                                demonstrated transmission electron microscopy [22]. <br />    shown yeasts Pmt2 forms heterodimer            single cell retained single nucleus shown Pmt1 proteins dimers high protein-O-               DAPI staining (Figure 4) obvious effect mannosylation activity [28,30]. Based finding                     pmt4 mutation cell cycle. <br /> interaction studies Pmts higher eukaryotes, e. human,                 contrast pmt4 mutant strains, pmt1A pmt1D POMT1 (member PMT4 family) POMT2                             mutants did prominent cell aggregation grown <br /> (member PMT2 family) form heterodimers [31], predicted              conditions (Figure 3A), calcofluor white Cryptococcus Pmt2 forms heterodimers Pmts.            staining revealed normal pattern chitin deposition cell model protein function predict pmt1 pmt4               separation (Figure 4). incubated 37uC, pmt1 pmt4 <br /> double mutation synthetically lethal            mutants demonstrated prominent morphological changes com- <br />                                                                               pared wild type. mutants grew large, dysmorphic cells Pmt protein interact Pmt2. test hypothesis, <br />                                                                               subject spontaneous lysis. DAPI staining revealed crossed MATalpha pmt1D strain MATa pmt4D strain <br />                                                                               single nuclei strains, nuclei pmt1 pmt4 mutants isolated individual basidiospores microdissection. tested <br />                                                                               displaced cell periphery large central resulting progeny strains following genetic markers: <br />                                                                               structure. structure demonstrated enlarged <br /> mating type, PMT1, PMT4, ADE2, URA5. marker <br />                                                                               vacuole staining FM4-64 (Figure 4). large <br /> segregation pattern demonstrated isolated spores                                                                               vacuole present cells 30uC, prominent result meiotic recombination (Figure 2). importantly, <br />                                                                               cells incubated 37uC. contrast, wild-type cells displayed 15 dissected spores able identify pmt1 <br />                                                                               multiple, small vacuoles incubated similar conditions. <br /> pmt4 double mutant, strongly suggesting pmt1 pmt4 <br /> mutations synthetically lethal. results suggest protein- <br /> O-glycosylation essential process C. neoformans,            pmt mutants sensitive different cell stresses Pmt1 Pmt4 interact Pmt2 provide cell                 Pmt proteins play important roles cell wall architecture, optimal protein-O-mannosylation activity Pmt1                mutations proteins result increased sensitivity Pmt4 common essential target independent Pmt2.                      various cell stresses elevated growth temperature, osmotic <br />                                                                               stress cell wall destabilising agents. Consistent                                                                               temperature-dependent cell morphology changes pmt <br /> pmt1 pmt4 mutant strains altered cell                                mutants, pmt1A pmt4A strains displayed marked defect <br /> morphology                                                                    growth elevated temperatures (Figure 5). incubated    yeasts filamentous fungi, loss certain                YPD medium 30uC, mutant strains grew similar wild <br /> protein-O-mannosyltransferases results cell morphology defects.            type.  37uC, pmt mutants displayed reduced contrast wild-type cells grow single yeasts simple         colony size, consistent profound temperature-sensitive <br /> buds, C. neoformans pmt4A pmt4D mutant strains                   growth defect (Figure 5). Formal growth curves, measuring <br /> exhibited cell aggregation phenotype cells grown              changes optical density log-phase cultures liquid <br /> 24 h 48 h YPD SD medium 30uC. cells               YPD, confirmed observations solid media (data shown). associated multi-cell aggregates (Figure 3A). addition,           contrast wild type reconstituted strains, cell aggregates resolved vortexing sonication          mutant strain grew 39uC (Figure 5). pmt1D pmt4D <br /> suggesting Dpmt4 strain defect cell                    mutant strains displayed similar severe temperature- <br /> separation, altered cell adherence (Figure 3B).                   dependent growth defect. <br /> Calcofluor white staining cell wall chitin pmt4 mutant                 pmt mutant strains serotypes displayed <br /> strains demonstrated failure daughter cells separate                  increased susceptibility salt stress (Figure 6 . incubated <br />  <br />  <br />        PLoS | www.plosone.org                                         3                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                      C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 2. pmt1 pmt4 double deletion lethal serotype D. Serotype D strains pmt1D (MATa pmt1D::URA5 ade2-27) pmt4D (MATa <br /> pmt4D::ADE2 ura5) crossed standard V8 mating media individual spores isolated days micromanipulation.  15 <br /> individual progeny analyzed colony PCR presence PMT1 (upper panel) PMT4 alleles (lower panel). Position respective <br /> wild-type disruption alleles indicated right. Wild-type strain JEC21 used control. B: 15 strains spotted indicated plates, grown 2&#226;&#8364;&#8220;3 days 30uC subsequently analyzed auxotrophic marker distribution. Strains spotted left (#1) right (wild-type control). C: Summary genotypes identified B respect PMT alleles mating-type. Mating-type individual spores determined standard mating reactions using wild-type strains JEC20 (MATa) JEC21 (MATa) tester strains. <br /> doi:10.1371/journal.pone.0006321.g002 <br />  presence 0.5 M KCl, pmt4A pmt4D strains grew               contrast, pmt1D mutant grows poorly presence slower wild-type/reconstituted strains. pmt4 mutant                sorbitol (Figure 6B), growth unaffected SDS <br /> strains demonstrated striking growth inhibition                  (Figure 6C). pmt4D strain inhibited high sorbitol <br /> addition 0.7 M 1 M NaCl medium (Figure 6A).                 concentrations, contrast corresponding pmt4A strain; <br /> pmt1A pmt1D strains inhibited NaCl,                 growth pmt4D inhibited SDS.  <br /> lesser extent pmt4 strains. cell wall destabilising             sorbitol SDS different cell surface destabilizing <br /> agents Congo red, caffeine, calcofluor white significant          effects pmt mutants C. neoformans varieties, <br /> effect growth pmt mutants variety (data          suggesting Pmt proteins play distinct roles related <br /> shown).                                                                      divergent strain backgrounds. <br />    contrast concordant effects salt temperature various pmt mutants different varieties, pmt1         Virulence factors C. neoformans differentially <br /> pmt4 mutants demonstrated variety-specific differences levels         affected pmt mutations susceptibility osmotic cell wall stress. pmt4A              ability grow 37uC, virulence fungal <br /> mutant susceptible corresponding pmt1A strain            pathogen C. neoformans determined set specific virulence <br /> sorbitol (2 M 2.5 M) (Figure 6B), pmt1A mutant                factors, including secretion various hydrolytic enzymes, susceptible effects 0.1% SDS (Figure 6C).                  ability produce polysaccharide capsule, expression  <br />  <br />        PLoS | www.plosone.org                                        4                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                 C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 3. pmt1 pmt4 disruption strains altered cell morphology.  Serotype wild-type strain H99 mutant strains <br /> pmt1A (pmt1A::URA5) pmt4A (pmt4A::URA5) incubated YPD 30uC 37uC OD600 1, cells subsequently analyzed light microscopy (DIC). B: Serotype pmt4A strain incubated YPD 30uC OD600 1, cell suspension analyzed light <br /> microscopy (DIC) treatment 1 min vortexing sonication. <br /> doi:10.1371/journal.pone.0006321.g003 <br />  antioxidant melanin pigment. common feature                 diameter cell itself [32]. average capsule ratio 100 <br /> factors involvement extracellular components.              wild-type cells 1.8 (+/20.02), ratio similar protein glycosylation mainly affects extracellular surface-             pmt1 mutant 1.75+/20.05). contrast, capsule ratio exposed proteins, hypothesized virulence             pmt4 mutant 1.45+/20.03, indicative slightly reduced <br /> factors affected mutations genes affecting protein-         capsule size (data shown). Results identical serotype O-mannosylation. Compared wild type, pmt1 pmt4                  serotype D strains, reduction capsule size mutants demonstrated changes activity secreted            pmt4 strains complemented wild-type PMT4 allele. <br /> enzymes urease phospholipase B (data shown), linked virulence C. neoformans.                      Protein-O-mannosylation mating <br />    secreted protein essential virulence C.              effect Pmt&#226;&#8364;&#8482;s mating analyzed serotype D <br /> neoformans enzyme laccase, phenoloxidase catalyses       strains crosses vigorous serotype  <br /> rate-limiting step melanin production. pmt1 mutant                mating type associated virulence strains did defect melanin production, pmt4            strain background [33,34]. pmt1 mutant strains did mutants  serotype serotype D) delayed melanin           obvious defect unilateral (wild-type x pmt1) bilateral <br /> production. melanin production delay complemented               (pmt1 x pmt1) crosses. contrast, pmt4 mutation showed pmt4+PMT4 reconstituted strain (Figure 7). finding                notable effects mating. Unilateral matings pmt4 strains contrasts prior observation melanin production            wild-type testers behaved similar wild-type control crosses (data <br /> serotype pmt4 mutant [22]. difference small           shown). hand, bilateral cross pmt4 strains <br /> differences disruption constructs used studies          revealed delayed mating reaction, reduced filament <br /> Discussion).                                                               formation 48 h comparison wild-type controls <br />    virulence-associated phenotype C. neoformans          (Figure 8). addition, aerial hyphae produced ability produce large polysaccharide capsule. Using standard        cross, indicated lack white mycelial rim <br /> India ink counter-stain, assessed capsule formation          surrounding mating patch (Figure 8; small picture). Micro- pmt mutant strains. Microscopic analyses various                scopic analysis hyphal structures produced bilateral <br /> serotype D mutant strains revealed strains             pmt4 mating reaction showed irregularly shaped thickened <br /> capsule incubated DMEM capsule-inducing medium.                 filaments swollen distal tips, phenotypes reminiscent control cell size variation mutant wild-type           mutations swoA locus Aspergillus nidulans, gene cells, quantified relative capsule size determining total        interestingly recently encode PMT <br /> cell diameter cells surrounding capsule compared        genes present  nidulans [35]. <br />  <br />  <br />       PLoS | www.plosone.org                                       5                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                      C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 4. pmt1 pmt4 mutant strains altered cell wall phenotypes vacuole distribution. Wild-type serotype strain H99 pmt mutant strains pmt1A (pmt1A::URA5) pmt4A (pmt4A::URA5) incubated YPD 30uC OD600 1. Cells fixed 30 min 10% <span id='am-55' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-56' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> subsequently stained described chitin staining dye calcofluor white, DNA dye DAPI vacuolar dye FM4-64. <br /> Pictures taken 400x using Zeiss Axioskop 2 Plus Fluorescence Microscope AxioCam MRM digital camera. <br /> doi:10.1371/journal.pone.0006321.g004 <br />  <br /> pmt mutant strains severely attenuated virulence                       modify proteins targeted secretion, process essential <br />    Considering effect different pmt mutations growth         fungi [3,28,36]. role fungal pathogenesis virulence-associated phenotypes, likely mutations         explored C. albicans C. neoformans [22,37]. negative effect virulence C. neoformans.            report, extend prior studies Pmt function human tested pathogenicity serotype strains isolated       pathogenic yeast C. neoformans.  demonstrated C. <br /> macrophages animals. incubated                       neoformans contains PMT genes, gene encodes <br /> activated J774A.1 macrophages, pmt1A pmt4A strains                     Pmt enzyme major classes proteins. demonstrated reduced survival 24 hours compared wild-type                contrast, fungi S. cerevisiae undergone <br /> (6-fold 40-fold reduction, respectively) (Figure 9). relative        paralogous duplication PMT genes classes, differences macrophage survival likely differences            developing novel functions various Pmt proteins. <br /> high-temperature growth virulence factor expression.                          addition conservation sequence similarity PMT <br />    murine model inhaled cryptococcosis, mutant strains            genes, demonstrated basic function C. neoformans significantly reduced virulence. model, 105 C.            Pmt enzymes likely similarly conserved. example, current <br /> neoformans colony-forming units intranasally inoculated               models Pmt enzyme activity suggest Pmt holoenzymes <br /> complement-defective Jcr mice, simulating natural respiratory            function optimally heterodimers [28,30,38], activity route infection humans. Animals infected wild-type        Pmt2 class proteins required component protein <br /> reconstituted strains median survival 18&#226;&#8364;&#8220;19 days,              complexes [15,28]. Consistent working model developed <br /> mice survived past 21 days infection. median survival             fungi, demonstrated PMT2 gene essential <br /> mice infected pmt4A mutant 40 days (p,0.001 compared              viability C. neoformans. Additionally, simultaneous wild-type), mice infected pmt1A strain died            mutation C. neoformans PMT1 PMT4 genes infection course 45-day experiment (Figure 9).           synthetically lethal, predicted model Pmt2 <br />                                                                                function concert Pmt proteins. functional <br />                                                                                conservation suggests strategies block fungal Pmt function <br /> Discussion <br />                                                                                inhibit growth diverse fungal species. <br />    past years, Pmt-mediated process protein-O-                essential process, demonstrated glycosylation defined species. Pmt&#226;&#8364;&#8482;s primarily             protein-O-glycosylation significant effect virulence  <br />  <br />        PLoS | www.plosone.org                                          6                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                    C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 5. pmt mutant strains growth defects high growth temperatures. night cultures wild-type, pmt mutant corresponding reconstituted strains serotype (upper panel) serotype D (lower panel) adjusted OD600 0.1 diluted 10- <br /> fold serial dilutions 1024 dilution. 5 ml dilution step spotted YPD plates, plates incubated indicated <br /> temperature 2&#226;&#8364;&#8220;3 days. Strains used serotype strains H99 (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), pmt1A+PMT1A <br /> (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR), respective serotype D strains JEC21 (wild-type), pmt1D <br /> (pmt1D::URA5), pmt4D (pmt4D::URA5), pmt1D+PMT1D (pmt1D::URA5 PMT1D-NeoR) pmt4D+PMT4D (pmt4D::URA5 PMT4D-NeoR). <br /> doi:10.1371/journal.pone.0006321.g005 <br />  <br /> C. neoformans, instances required             dramatic reduction melanin production. observation viability. demonstrated mutations PMT1              reduced melanin serotype D pmt4 mutants different <br /> PMT4 genes result dramatic attenuation virulence             prior report melanin defect observed macrophage killing assay mouse inhalation model                     serotype pmt4 mutant [22]. difference pmt4 <br /> cryptococcosis, models assess different aspects                  mutants method gene mutation. 5th 6th <br /> cryptococcal pathogenesis. studies role Pmt4 C.             transmembrane domains, Pmt proteins characterized neoformans pathogenesis consistent prior reports [22].              extensive loop 5 essential mannosyl-transferase activity  experiments demonstrate pmt1 mutant               [27]. Loop 5 contains conserved motifs  C) attenuated survival vivo compared pmt4                   important enzyme activity, domain C C- <br /> mutants, despite similar vitro temperature- cell wall-sensitive       terminal domain region protein. Olson et al. disrupted <br /> phenotypes. Interestingly, pmt1 pmt4 mutants high              Pmt4 downstream motif C; contrast, pmt4 mutations <br /> temperature sensitivity 39uC, great impact            resulted deletion domain C loop 5, supposedly <br /> infectivity mouse model. investigate role                   resulting complete inactivation Pmt4 enzyme. <br /> temperature sensitivity virulence attenuation test               addition melanin delay, pmt1 pmt4 mutant cells virulence mutants heterologous host model,            display aberrant cell morphology pronounced cell aggrega- wax moths (Galleria mellonella), does require high              tion. morphological changes linked cell wall- <br /> temperature infection.                                                temperature-sensitive phenotypes identified vitro, resulting <br />    Virulence basidiomycete C. neoformans linked           cell lysis stress. Similarly, hyperflocculant dysmorphic defined phenotypes, including production                C. neoformans cells demonstrated reduced virulence mouse <br /> export extracellular factors melanin, capsule,                model, increased susceptibility complement-activated <br /> various lytic enzymes [39&#226;&#8364;&#8220;43]. classic virulence-                   phagocytosis macrophages [45]. <br /> associated phenotypes, melanin production altered                   temperature sensitivity pmt1 pmt4 mutants likely strains, pmt4 mutant. C. neoformans strains        explains altered virulence strains. C. <br /> defective melanin production attenuated virulence,                neoformans strains altered growth mammalian physiological <br /> including strains mutations laccase gene LAC1,                   temperatures display similar, predictable virulence defects [46]. <br /> encoding rate limiting enzyme melanin production                      Interestingly, serotype strains, serotype D, pmt1 <br /> [39,40].  delay melanin production likely plays             mutant showed significant growth defect 37uC liquid role reduced virulence pmt4 mutant strain.                   medium solid medium. differential growth effect    C. neoformans laccase Lac1 direct target             liquid medium caused higher shearing forces Pmt4. Lac1 N-glycosylated, cell wall associated protein [44],          present growth solid medium. N-glycosylation precedes O-glycosylation. Altered Pmt4                 previous reports demonstrated pmt4 deficient <br /> activity lead mislocalization Lac1             strains showed dramatic differences overall pattern  <br />  <br />        PLoS | www.plosone.org                                        7                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                        C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 6. pmt mutant strains sensitive hyper-osmotic stresses. night cultures wild-type, pmt mutant corresponding <br /> reconstituted strains serotype (upper panel) serotype D (lower panel) adjusted OD600 0.1 diluted 10-fold serial <br /> dilutions 1024 dilution. 5 ml dilution step spotted YPD plates containing stress agents, plates incubated indicated temperature 2&#226;&#8364;&#8220;3 days. Strains used serotype strains H99 (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), <br /> pmt1A+PMT1A (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR), respective serotype D strains JEC21 (wild-type), <br /> pmt1D (pmt1D::URA5), pmt4D (pmt4D::URA5), pmt1D+PMT1D (pmt1D::URA5 PMT1D-NeoR) pmt4D+PMT4D (pmt4D::URA5 PMT4D-NeoR). Strains spotted YPD plates supplemented various salts  , sorbitol (B) SDS (C) indicated concentrations. test G418 resistance (20 mg/ml) <br /> (C), night cultures initially diluted OD600 1.0 versus 0.1 used experiments. Plates incubated 30uC 2&#226;&#8364;&#8220;3 days. <br /> doi:10.1371/journal.pone.0006321.g006 <br />  <br />  <br />  <br />        PLoS | www.plosone.org                                          8                                 July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                  C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 7. pmt4 pmt1 mutant strains delayed melanin synthesis.  10 ml night cultures serotype strains H99 <br /> (wild-type), pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR) spotted standard Niger seed- <br /> plates, plates incubated days 30uC. B: Cells 2 ml night cultures harvested resuspended 2 ml glucose-free asparagine-medium supplemented <span id='am-60' about='obo:IMR_0200290' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-61' property="rdfs:label" content="L-Dopa" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00355" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0200290" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-DOPA</span>, cultures shaken 30uC night. Subsequently cells pelleted, cell pellet supernatant photographed. C: OD480 supernatants cultures B determined graphically displayed <br /> using Prism 5 (GraphPad, San Diego, Calif.). Graph shows overall result independent experiments. <br /> doi:10.1371/journal.pone.0006321.g007 <br />  <br /> mannosylated proteins 2D gel electrophoresis [22]. Extracel-              composition proteins specifically affected lular mannoproteins important regarding immunological                    pmt1 pmt4 mutation. identification immediate targets <br /> aspects C. neoformans-host interactions including T-cell activation       individual cryptococcal Pmt proteins provide [47&#226;&#8364;&#8220;49].  defects protein-O-glycosylation              explanations defects pathogenicity pmt1 <br /> impair virulence C. neoformans altering immunological              pmt4 mutants C. neoformans. <br /> response microbial cells affect pathogenesis.             Targets Pmt enzyme complex defined interesting analyze pmt1               various fungal species. fact, Fks1 protein modified <br /> mutation results similar differences cell wall mannoprotein            C. neoformans Pmt4 [22]. Fks1 plays major role  <br />  <br />        PLoS | www.plosone.org                                        9                             July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                       C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 8. pmt4D mutant strains serotype-dependent filamentation defects. perform matings wild-type pmt mutant strains, respective strains grown night YPD 30uC. Cells harvested, washed twice sterile 0.9% saline, incubated saline shaker hour 30uC. 10 ml respective strains mixed, cells spotted standard solid V8 agar plates, subsequently incubated RT dark. Edges colonies colonies (small pictures) photographed 48 h. Shown bilateral matings serotype D strains JEC20 JEC21 (wild-type), SW5 (MATa) SW6 (MATa) (pmt1D mutants), SW8 (MATa) SW9 <br /> (MATa) (pmt4D mutants), respectively. <br /> doi:10.1371/journal.pone.0006321.g008 <br />  <br /> synthesis glucans, important components fungal                 proteins candidates involved cell walls, observation offers potential mechanism cell         remodelling cell wall including chitin-deacetylases (Cda1- <br /> wall defects described pmt4 mutant strain [22]. Given              3) potential endoglucanases (AAW45003, AAW46063 <br /> similar distinct phenotypes pmt1 pmt4 mutants,            AAW46065). interesting likely Pmt proteins possess different affinities            proteins identified direct target Pmt-dependent O- <br /> Pmt target proteins.                                                           glycosylation deletions corresponding genes <br />    Given role protein-O-glycosylation plays microbial              result morphological phenotypes similar pmt mutants. <br /> pathogenesis, investigators explored Pmt proteins potential               conclusion, PMT gene encodes group proteins <br /> targets therapeutic intervention. Derivatives rhodanine-3-              critical biology virulence C. neoformans. Given high <br /> <span id='am-65' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span> recently identified specifically inhibit        degree conservation gene family pathogenic <br /> Pmt1 isoform C. albicans. Treating wild-type cells               fungi, gene family ideal antifungal drug target. Future <br /> inhibitors resulted phenotypic transcriptional changes                  studies explore possibility define reminiscent C. albicans pmt1 deletion strains [15,50,51].             mechanisms PMT family mutant defects. <br /> Pmt2 Pmt4 orthologs present mammals, possibility developing Pmt isoform-specific inhibitors renders Pmt proteins             Materials Methods attractive target antifungal drug development. Additionally, <br /> targeting specific Pmt isoforms inhibitory compounds                  Strains media useful killing phytopathogenic fungi, Pmt proteins               Reference strains used study congenic serotype identified plants [3].                                       D strains JEC20 (MATa) JEC21 (MATa), serotype    cell wall defect pmt1 pmt4 mutants striking.              strain H99. pmt mutant strains constructed transforming mechanism proposed Pmt4 control C. neoformans cell              JEC20/21 derivatives JEC155 (MATa, ade2 ura5), JEC156 (MATa, <br /> wall integrity recently provided. stress related induction             ade2 ura5), F99, respectively, 5-FOA resistant derivative of Fks1 gene, encoding catalytic subunit b-1,3-glucan               H99. strains used study listed Table 1. Yeast- <br /> synthase, dependent Pmt4 [22]. b-1,3-glucan major             peptone-dextrose (YPD) yeast nitrogen base media, synthetic <br /> component fungal cell walls, altered glucan synthesis                 (SD) medium, V8 agar mating, filament agar, Niger seed likely result altered cell integrity stress conditions [52].          melanin production, serum-free Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s  defects C. neoformans PKC/cell integrity signalling               medium capsule induction prepared previously <br /> resulted phenotypes similar pmt mutant strains              reported [32,53,55,56]. <br /> [53]. phenotypes included morphological defects, defects vacuolar biogenesis, growth defects elevated growth tempera-                Plasmid construction <br /> tures low SDS concentrations, higher sensitivity                      PMT homologs identified tblastn searches using S. <br /> osmotic stresses. interesting shown                 cerevisiae Pmt1 C. albicans Pmt1 protein sequences proper PKC/cell integrity signalling necessary melanin                  National Center Biotechnology Information (http://www.ncbi. <br /> production; pkc1 mutants improper laccase localization                nlm.nih.gov). generating PMT disruption constructs PMT <br /> reduced melanin production. cell wall abnormalities,                   open reading frames (ORF) amplified PCR resulting changes PKC signalling, explain melanin              reference strains H99 JEC21, fragments obtained production delay pmt4 mutants.                                          cloned standard cloning vectors. PMT1A BglII sites <br />    Levitz Specht identified 55 potential GPI anchored                      integrated PMT ORFs using oligonucleotide based <br /> membrane proteins serine/threonine rich regions                  site directed mutagenesis (Stratagene Quick Change site directed <br /> thought potential O-glycosylation targets [54].             mutagenesis kit). Subsequently, ADE2 URA5 containing BamHI <br />  <br />  <br />        PLoS | www.plosone.org                                         10                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                 C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  <br /> Figure 9. pmt mutant strains serotype attenuated virulence.  macrophage killing assay performed serotype wild-type strain H99 pmt mutant strains SW13 (pmt1A::URA5) SW14 (pmt4A::URA5) previously described. CFU independent <br /> experiments repetitions strain shown. B: murine inhalation model performed serotype strains H99 (wild-type), <br /> pmt1A (pmt1A::URA5), pmt4A (pmt4A::URA5), pmt1A+PMT1A (pmt1A::URA5 PMT1A-NeoR) pmt4A+PMT4A (pmt4A::URA5 PMT4A-NeoR) previously <br /> described. Briefly, strain 10 Jcr mice infected intranasally 105 CFU, survival time post infection determined. <br /> doi:10.1371/journal.pone.0006321.g009 <br />  <br /> fragments cloned BglII sites disrupt respective        amplified PCR primers containing BamHI restriction <br /> ORFs amino acid E468 (2A), K461 (4A), W315 (1D), N508                 sites. Corresponding BamHI fragments subsequently cloned <br /> (2D) Q412 (4D), respectively. PMT1A existing EcoRV               BamHI restriction site vector pJAF1 containing Tn5 <br /> restriction site used generate disruption constructs.         derived NeoR resistance marker [59]. case ADE2 URA5 containing SmaI fragments used                primers utilized plasmid construction presented disrupting PMT1A ORF position D439. constructs                 Table S1. PCR amplified generate linear disruption allele gel purified use biolistic transformation appropriate           Strain construction <br /> auxotroph strain [57]. Biolistic transformations conducted                generate homozygous serotype D diploid ade2 strain, <br /> using established methods [58]. reconstituting generated            strains JEC156 JEC157 crossed according Sia et al. <br /> disruption strains PMT ORFs plus additional 500&#226;&#8364;&#8220;700 bp               (2000) random spores isolated SD solid media downstream respective START STOP codons                      supplemented 20 mg/ml <span id='am-33' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span>. Plates grown  <br />  <br />       PLoS | www.plosone.org                                       11                              July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                    C. neoformans PMT Gene Family <br />  <br />  <br />  <br />  Table 1. Strains used study. <br />  <br />  <br />  Serotype strains                             Genotype                                                             Source/reference <br />  H99                                            wild-type MATa                                                       [67] <br />  F99                                            MATa ura5 (5-FOA resistent isolate H99)                           Ping Wang (Heitman-laboratory) <br />  SW13                                           MATa pmt1::URA5                                                      study <br />  SW14                                           MATa pmt4::URA5                                                      study <br />  SW15                                           MATa pmt1::URA5 PMT1-NeoR                                            study <br />                                                                             R <br />  SW16                                           MATa pmt4::URA5 PMT4-Neo                                             study <br />  Serotype D strain                              Genotype                                                             Source/reference <br />  JEC20                                          wild-type MATa                                                       [68] <br />  JEC21                                          wild-type MATa                                                       [68] <br />  JEC155                                         MATa ade2-27 ura5                                                    Jeff Edman <br />  JEC156                                         MATa ade2-27 ura5                                                    Jeff Edman <br />  JEC157                                         MATa ade2 ura5 lys1                                                  Jeff Edman <br />  JEC34                                          MATa ura5                                                            Jeff Edman <br />  JEC43                                          MATa ura5                                                            Jeff Edman <br />  JEC50                                          MATa ade2-27                                                         Jeff Edman <br />  SW1                                            MATa pmt1::ADE2                                                      study <br />  SW2                                            MATa pmt1::ADE2                                                      study <br />  SW4                                            MATa pmt1::URA5 ade2-27                                              study <br />  SW5                                            MATa pmt1::URA5                                                      study <br />  SW6                                            MATa pmt1::URA5                                                      study <br />  SW7                                            MATa pmt4::ADE2 ura5                                                 study <br />  SW8                                            MATa pmt4::ADE2                                                      study <br />  SW9                                            MATa pmt4::ADE2                                                      study <br />  SW10                                           MATa pmt4::URA5 ade2-27                                              study <br />  SW11                                           MATa pmt4::URA5                                                      study <br />  SW12                                           MATa pmt4::URA5                                                      study <br />  SW17                                           MATa pmt1::URA5 PMT1-NeoR                                            study <br />  SW18                                           MATa pmt4::URA5 PMT4-NeoR                                            study <br />  SW19                                           MATa pmt4::ADE2 ura5                                                 study <br />  <br />  doi:10.1371/journal.pone.0006321.t001 <br />  <br />  days 37uC prevent filament formation usually seen             strand synthesis reverse transcription kit (Invitrogen). resulting diploid strains. second step, reddish Ade2 strains           cDNA used template quantitative real-time PCR using <br /> isolated streaked plates, incubated RT          iQ SYBR Green Supermix (Bio-Rad) according manufac- induce filament basidiospore formation typical diploid            turer&#226;&#8364;&#8482;s specifications. iCycler iQ multicolor real-time <br /> Cryptococcus strains conditions. generate                  detection used fluorescence detector heterozygous pmt2/PMT2 serotype D strain, resulting diploid               following PCR conditions: initial denaturing cycle 95uC ade2/ade2 diploid strain transformed ADE2 based                 3 min 40 cycles denaturation 95uC 10 s pmt2D-disruption construct using biolistic transformation. Hetero-          annealing/extension 58uC 330 s. cycles followed <br /> zygous mutants initially identified colony PCR                  standard melting curve 53uC 93uC fluorescent <br /> subsequently confirmed Southern analysis.                                monitoring 0.5uC. data confirmed amplification                                                                             single product primer pair lack primer <br /> RNA preparation                                                             dimerization. Reactions performed triplicate, data <br />    Strains incubated mid-logarithmic phase 30uC               expressed average cycle threshold, 6 standard <br /> YPD medium. Cells collected centrifugation flash                deviation. Standard PCRs run fivefold dilutions frozen dry ice. Total RNA extracted lyophilized cells           cDNA template determine optimal template using TRIzol reagent (Invitrogen Life Technologies, Carlsbad,           optimal annealing temperature experimental reference <br /> CA).                                                                        reactions, using 500 nM primer. <br />                                                                                Gene amplification strain condition normalized <br /> Real-time PCR                                                               constitutively expressed GPD gene [60]. Induction (n- <br />   Total RNA isolated relevant strains described             fold) calculated relative wild-type strain JEC21 using  RNA treated RNase-free DNase, cDNA                  Bio-Rad iCycler software  utilizes comparative synthesized using oligo(dT) primers SuperScript          cycle threshold statistical methods previously described [61] <br />  <br />  <br />          PLoS | www.plosone.org                                    12                                July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                   C. neoformans PMT Gene Family <br />  <br />  <br />  <br /> Mating analyses haploid filamentation                                   vitro virulence assays <br />    determine defects mating, various pmt mutant                  determine capsule production, strains grown <br /> tester strains pre-grown YPD plates 30uC 2&#226;&#8364;&#8220;3 days,           night Dulbecco&#226;&#8364;&#8482;s modified Eagles medium 3 days small growth removed using sterile tooth pick             30uC [32]. Subsequently, 5 ml culture mixed spotted solid V8 plates (pH 5.0 7.0),           equal volume India ink, cells analyzed light <br /> mixed respective tester strain. Plates incubated             microscopy. Melanin production determined different <br /> room temperature (RT) dark 3 days mating            techniques.  10 ml night cultures spotted haploid filamentation assessed light microscopy.                 standard Niger seed plates, plates incubated 30uC                                                                             2&#226;&#8364;&#8220;4 days controls showed brownish colour indicative                                                                             melanin production. Second, cells 25 ml YPD night <br /> Basidiospore isolation <br />                                                                             cultures harvested resuspended 2 ml glucose-free <br />    isolate basidiospores genetic crosses, strains                                                                             asparagine-medium supplemented L-DOPA, cell sus- <br /> crossed solid V8 plates described                                                                             pensions incubated 24 h 30uC rotary shaker. <br /> mating analyses incubated RT dark                                                                             Subsequently cells pelleted, melanin production basidiospore formation noticeable. Areas showing basidio- <br />                                                                             documented photographing cell pellet, determining spores excised single basidiospores transferred                                                                             OD480 supernatants [64]. Urease activity tests wild-type <br /> fresh YPD plates using micromanipulator (single spore isolation).         pmt mutant strains performed following way. Alternatively, excised filaments transferred 1.5 ml               colony strain added sterile deionized <span id='am-38' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span> reaction tube containing 500 ml water, vortexed vigorously,            microcentrifuge tube vortexed vigorously. BBL TAX- spores subsequently plated fresh YPD plates                  OTM urease differentiation disk (Becton, Dickinson, &amp; , Sparks, <br /> respective selective media (random spore isolation). isolate             MD) added tube using sterile techniques, haploid Ade2+ strains heterozygote serotype D pmt2/                incubated 30uC checked 10 min 30 min PMT2 strain random spore analysis spreading spores               urease activity according manufacturer&#226;&#8364;&#8482;s instructions. SD plates containing 20 mg/ml adenine, plates incubated 37uC prevent filamentation. Subsequently,                                                                             vivo virulence analyses <br /> white colonies streaked SD plates                                                                                Overall virulence C. neoformans strains tested incubated RT. Strains filament                                                                             macrophage killing assays murine inhalation model. <br /> conditions discarded filamentation indicate                                                                             macrophage killing assay 50 ml freshly grown J77A.1 strains diploid nature. <br />                                                                             macrophages (,16105 J774A.1 macrophages DMEM)                                                                             pipetted 96 microtiter plates activated adding <br /> Serial dilution patch tests                                                 50 ml DMEM supplemented INF-c (100 U/ml) <span id='am-11' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-12' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span>LPS</span> <br />    Standard serial dilution patch tests performed follows:          (0.6 mg/ml). Negative controls &#226;&#8364;&#732;&#226;&#8364;&#732;activated&#226;&#8364;&#8482;&#226;&#8364;&#8482; DMEM <br /> strains tested pre-grown night 5 ml liquid YPD           containing additional supplements. Microtiter plates medium 30uC stationary phase. OD600 determined              incubated 12&#226;&#8364;&#8220;18 h 37uC 5% <span id='am-28' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span> atmosphere day, cultures diluted OD600 1, tenfold serial        generate macrophage cell-layer. C. neoformans strains tested <br /> dilutions YPD dilution 1024. Finally,             grown night liquid YPD medium, cells 5 ml various dilutions spotted respective solid        harvested washed times PBS. Cells media incubated indicated temperatures.                          resuspended DMEM containing 1 mg/ml monoclonal antibody <br />                                                                             mAb18B7 cell titer 16106 living cells ml, determined                                                                             Trypan Blue staining, incubated 1 h 37uC 5% CO2. <br /> Microscopy                                                                  Subsequently, 100 ml C. neoformans cells pipetted    Light, DIC fluorescence microscopy pictures taken               macrophage cell layers, microtiter plates incubated using Zeiss Axioskop 2 Plus Fluorescence Microscope mounted               37uC 5% CO2 hour. Non incorporated cryptococcal AxioCam MRM digital camera, alternatively                  cells removed carefully washing macrophage cell <br /> Nikon Eclipse E400 microscope Nikon CoolPix 990 digital               layers times PBS. Finally cell layers covered camera. Cell wall material genomic DNA stained using               150 ml fresh DMEM culture medium incubated Fluorescent Brightener 28 (calcofluor white; Sigma Aldrich)             24 h 37uC 5% CO2. determine survival rates 4,6-diamidinO-2-phenylindole (DAPI; Molecular Probes).                   cryptococcal cells covering culture medium removed  C. neoformans strains grown OD600 ,1, cells            transferred fresh reaction tube. Macrophages titer plates harvested fixed 30 min 10% formaldehyde.                    incubated 100 ml 0.5% SDS solution ,5 min RT, <br /> Subsequently, cells washed times PBS               cells lysed pipetting cells times, permeabilized 1% Triton X-100 10 min. Finally, cells               cell lysate combined respective culture washed times PBS stained                 supernatant. Finally, wells washed 200 ml PBS indicated dyes. visualize vacuoles, cells stained          solution, subsequently added respective cell lipophilic dye N-(3-triethylammoniumpropyl)-4-(6-(4-                    lysate. Cell lysates diluted 300 fold, cryptococcal CFU <br /> (diethylamino) phenyl) hexatrienyl) pyridinium dibromide (FM                determined standard techniques solid YPD plates, <br /> 4&#226;&#8364;&#8220;64; T-3166; Invitrogen Corporation, Carlsbad, CA)                      incubated 30uC. Macrophage killing assays repeated described previously [62,63]. Briefly, logarithmically growing              times [65]. murine inhalation model systemic <br /> C. neoformans cells corresponding 1 ml OD600 ,0.1             Cryptococcosis performed described [66]. Briefly, <br /> harvested resuspended 100 ml YPD. Subsequently,                  groups female Jcr mice (,20 g body weight) vacuolar stain FM4-64 added final concentration                 infected intra-nasally 105 cryptococcal CFU resuspended 40 mM, cells incubated rotary shaker 45 min               PBS. determine survival rates, mice inspected vitality 30uC. microscopy cells washed times             twice day, individuals showing ,15% weight loss, <br /> YPD.                                                                        neurological abnormalities extreme anorexia sacrificed <br />  <br />  <br />       PLoS | www.plosone.org                                       13                               July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                                              C. neoformans PMT Gene Family <br />  <br />  <br /> immediately according animal protection regulations                             doi:10.1371/journal.pone.0006321.s001 (0.02 MB <br /> Duke University Medical Center Animal Care &amp; Use Program.                                    DOC) <br /> Mice maintained Research Institute Children <br /> Animal Facility accordance American Association                               Acknowledgments <br /> Accreditation Laboratory Animal Care guidelines. <br />                                                                                              wish thank J. Heitman providing C. neoformans strains                                                                                              plasmids. special thanks members Ernst/Lengeler, <br /> Statistical analysis                                                                         Alspaugh laboratories Robert Cramer technical assistance   software program Prism 5 (GraphPad, San Diego, Calif.)                                 helpful discussion. used statistical tests. Log-rank tests utilized determine significance survival animal studies. <br />                                                                                              Author Contributions <br /> Supporting Information                                                                       Conceived designed experiments: SDW AA KL. Performed                                                                                              experiments: SDW. Analyzed data: SDW. Contributed reagents/ <br /> Table S1 Plasmids used study                                                         materials/analysis tools: JE AA KL. Wrote paper: SDW AA KL. <br />  <br /> References <br />  1. Tanner W, Lehle L (1987) Protein glycosylation yeast. Biochim Biophys Acta            21. Perfect JR, Casadevall (2002) Cryptococcosis. Infect Dis Clin North 16: <br />     906: 81&#226;&#8364;&#8220;99.                                                                                  837&#226;&#8364;&#8220;874, v&#226;&#8364;&#8220;vi. <br />  2. Strahl-Bolsinger S, Gentzsch M, Tanner W (1999) Protein O-mannosylation.                 22. Olson GM, Fox DS, Wang P, Alspaugh JA, Buchanan KL (2007) Role     Biochim Biophys Acta 1426: 297&#226;&#8364;&#8220;307.                                                          protein O-mannosyltransferase Pmt4 morphogenesis virulence  3. Ernst JF, Prill SK (2001) O-glycosylation. Med Mycol 39 Suppl 1: 67&#226;&#8364;&#8220;74.                      Cryptococcus neoformans. Eukaryot Cell 6: 222&#226;&#8364;&#8220;234. <br />  4. Jurado LA, Coloma  Cruces J (1999) Identification human homolog            23. Loftus BJ, Fung E, Roncaglia P, Rowley D, Amedeo P, et al. (2005) genome <br />     Drosophila rotated abdomen gene (POMT1) encoding putative protein                          basidiomycetous yeast human pathogen Cryptococcus neoformans. <br />     O-mannosyl-transferase, assignment human chromosome 9q34.1.                           Science 307: 1321&#226;&#8364;&#8220;1324. <br />     Genomics 58: 171&#226;&#8364;&#8220;180.                                                                    24. Franzot SP, Fries BC, Cleare W, Casadevall (1998) Genetic relationship <br />  5. Willer T, Amselgruber W, Deutzmann R, Strahl S (2002) Characterization                    Cryptococcus neoformans var. neoformans strains serotypes D. J Clin <br />     POMT2, novel member PMT protein O-mannosyltransferase family                        Microbiol 36: 2200&#226;&#8364;&#8220;2204. <br />     specifically localized acrosome mammalian spermatids. Glycobiology             25. Tanaka R, Imanishi Y, Nishimura K (2003) Difference FKS1 gene sequences <br />     12: 771&#226;&#8364;&#8220;783.                                                                                 serotypes D Cryptococcus neoformans. J Clin Microbiol 41: <br />  6. Lyalin D, Koles K, Roosendaal SD, Repnikova E, Van Wechel L, et al. (2006)                   4457&#226;&#8364;&#8220;4459. <br />     twisted gene encodes Drosophila protein O-mannosyltransferase 2                  26. Sugita T, Ikeda R, Shinoda T (2001) Diversity strains Cryptococcus <br />     genetically interacts rotated abdomen gene encoding Drosophila                      neoformans var. gattii revealed sequence analysis multiple genes     protein O-mannosyltransferase 1. Genetics 172: 343&#226;&#8364;&#8220;353.                                      chemotype analysis capsular polysaccharide. Microbiol Immunol 45: <br />  7. VanderVen BC, Harder JD, Crick DC, Belisle JT (2005) Export-mediated                         757&#226;&#8364;&#8220;768. <br />     assembly mycobacterial glycoproteins parallels eukaryotic pathways. Science           27. Girrbach V, Zeller T, Priesmeier M, Strahl-Bolsinger S (2000) Structure- <br />     309: 941&#226;&#8364;&#8220;943.                                                                                function analysis dolichyl phosphate-mannose: protein O-mannosyltrans- <br />  8. Mahne M, Tauch  Puhler  Kalinowski J (2006) Corynebacterium                          ferase ScPmt1p. J Biol Chem 275: 19288&#226;&#8364;&#8220;19296. <br />     glutamicum gene pmt encoding glycosyltransferase related eukaryotic protein-        28. Willer T, Brandl M, Sipiczki M, Strahl S (2005) Protein O-mannosylation     O-mannosyltransferases essential glycosylation resuscitation                   crucial cell wall integrity, septation viability fission yeast. Mol <br />     promoting factor (Rpf2) secreted proteins. FEMS Microbiol Lett                     Microbiol 57: 156&#226;&#8364;&#8220;170. <br />     259: 226&#226;&#8364;&#8220;233.                                                                            29. Sia RA, Lengeler KB, Heitman J (2000) Diploid strains pathogenic <br />  9. Strahl-Bolsinger S, Scheinost (1999) Transmembrane topology pmt1p,                    basidiomycete Cryptococcus neoformans thermally dimorphic. Fungal Genet Biol <br />     member evolutionarily conserved family protein O-mannosyltrans-                     29: 153&#226;&#8364;&#8220;163. <br />     ferases. J Biol Chem 274: 9068&#226;&#8364;&#8220;9075.                                                     30. Girrbach V, Strahl S (2003) Members evolutionarily conserved PMT <br /> 10. Willer T, Valero MC, Tanner W, Cruces J, Strahl S (2003) O-mannosyl glycans:                 family protein O-mannosyltransferases form distinct protein complexes     yeast novel associations human disease. Curr Opin Struct Biol 13:                J Biol Chem 278: 12554&#226;&#8364;&#8220;12562. <br />     621&#226;&#8364;&#8220;630.                                                                                 31. Akasaka-Manya K, Manya H, Nakajima  Kawakita M, Endo T (2006) <br /> 11. Gentzsch M, Tanner W (1997) Protein-O-glycosylation yeast: protein-specific               Physical functional association human protein O-mannosyltransferases 1 <br />     mannosyltransferases. Glycobiology 7: 481&#226;&#8364;&#8220;486.                                               2. J Biol Chem 281: 19339&#226;&#8364;&#8220;19345. <br /> 12. Bourdineaud JP, van der Vaart JM, Donzeau M, Sampaio G, Verrips CT, et                32. Granger DL, Perfect JR, Durack DT (1985) Virulence Cryptococcus neoformans: <br />     al. (1998) Pmt1 mannosyl transferase involved cell wall incorporation               regulation capsule synthesis carbon dioxide. J Clin Invest 76: 508&#226;&#8364;&#8220;516. <br />     proteins Saccharomyces cerevisiae. Mol Microbiol 27: 85&#226;&#8364;&#8220;98.                   33. Kwon-Chung KJ, Edman JC, Wickes BL (1992) Genetic association mating <br /> 13. Harty C, Strahl S, Romisch K (2001) O-mannosylation protects mutant alpha-                   types virulence Cryptococcus neoformans. Infect Immun 60: 602&#226;&#8364;&#8220;605. <br />     factor precursor endoplasmic reticulum-associated degradation. Mol Biol             34. Nielsen K, Cox GM, Wang P, Toffaletti DL, Perfect JR, et al. (2003) Sexual <br />     Cell 12: 1093&#226;&#8364;&#8220;1101.                                                                          cycle Cryptococcus neoformans var. grubii virulence congenic alpha <br /> 14. Proszynski TJ, Simons K, Bagnat M (2004) O-glycosylation sorting                        isolates. Infect Immun 71: 4831&#226;&#8364;&#8220;4841. <br />     determinant cell surface delivery yeast. Mol Biol Cell 15: 1533&#226;&#8364;&#8220;1543.             35. Shaw BD, Momany M (2002) Aspergillus nidulans polarity mutant swoA 15. Prill SK, Klinkert B, Timpel C, Gale CA, Schroppel K, et al. (2005) PMT family               complemented protein O-mannosyltransferase pmtA. Fungal Genet Biol 37: <br />     Candida albicans: protein mannosyltransferase isoforms affect growth,                263&#226;&#8364;&#8220;270. <br />     morphogenesis antifungal resistance. Mol Microbiol 55: 546&#226;&#8364;&#8220;560.                      36. Gentzsch M, Tanner W (1996) PMT gene family: protein O-glycosylation 16. Lommel M, Bagnat M, Strahl S (2004) Aberrant processing WSC family                    Saccharomyces cerevisiae vital. Embo J 15: 5752&#226;&#8364;&#8220;5759. <br />     Mid2p cell surface sensors results cell death Saccharomyces cerevisiae         37. Rouabhia M, Schaller M, Corbucci C, Vecchiarelli  Prill SK, et al. (2005) <br />     O-mannosylation mutants. Mol Cell Biol 24: 46&#226;&#8364;&#8220;57.                                            Virulence fungal pathogen Candida albicans requires isoforms 17. Beltran-Valero Bernabe D, Currier S, Steinbrecher  Celli J, van                         protein mannosyltransferases. Infect Immun 73: 4571&#226;&#8364;&#8220;4580. <br />     Beusekom E, et al. (2002) Mutations O-mannosyltransferase gene                    38. Gentzsch M, Immervoll T, Tanner W (1995) Protein O-glycosylation     POMT1 rise severe neuronal migration disorder Walker-Warburg                     Saccharomyces cerevisiae: protein O-mannosyltransferases Pmt1p Pmt2p <br />     syndrome. J Hum Genet 71: 1033&#226;&#8364;&#8220;1043.                                                      function heterodimer. FEBS Lett 377: 128&#226;&#8364;&#8220;130. <br /> 18. Willer T, Prados B, Falcon-Perez JM, Renner-Muller  Przemeck GK, et al.                39. Rhodes JC, Polacheck  Kwon-Chung KJ (1982) Phenoloxidase activity     (2004) Targeted disruption Walker-Warburg syndrome gene Pomt1                      virulence isogenic strains Cryptococcus neoformans. Infect Immun 36: <br />     mouse results embryonic lethality. Proc Natl Acad Sci U S 101:                          1175&#226;&#8364;&#8220;1184. <br />     14126&#226;&#8364;&#8220;14131.                                                                             40. Kwon-Chung KJ, Polacheck  Popkin TJ (1982) Melanin-lacking mutants 19. Martin-Blanco E, Garcia-Bellido (1996) Mutations rotated abdomen                    Cryptococcus neoformans virulence mice. J Bacteriol 150: 1414&#226;&#8364;&#8220;1421. <br />     locus affect muscle development reveal intrinsic asymmetry                     41. Chang YC, Kwon-Chung KJ (1994) Complementation capsule-deficient <br />     Drosophila. Proc Natl Acad Sci U S 93: 6048&#226;&#8364;&#8220;6052.                                          mutation Cryptococcus neoformans restores virulence. Mol Cell Biol 14: <br /> 20. Ichimiya T, Manya H, Ohmae Y, Yoshida H, Takahashi K, et al. (2004)                      4912&#226;&#8364;&#8220;4919. <br />     twisted abdomen phenotype Drosophila POMT1 POMT2 mutants                          42. Chen SC, Muller M, Zhou JZ, Wright LC, Sorrell TC (1997) Phospholipase <br />     coincides heterophilic protein O-mannosyltransferase activity. J Biol             activity Cryptococcus neoformans: new virulence factor? J Infect Dis 175: <br />     Chem 279: 42638&#226;&#8364;&#8220;42647.                                                                       414&#226;&#8364;&#8220;420. <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                                      14                                       July 2009 | Volume 4 | Issue 7 | e6321 <br />                                                                                                                                                  C. neoformans PMT Gene Family <br />  <br />  <br /> 43. Chen SC, Wright LC, Santangelo RT, Muller M, Moran VR, et al. (1997)                         56. Vartivarian SE, Anaissie EJ, Cowart  Sprigg HA, Tingler MJ, et al. (1993) <br />     Identification extracellular phospholipase B, lysophospholipase, acyl-                    Regulation cryptococcal capsular polysaccharide <span id='am-50' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>iron</span>. J Infect Dis 167: <br />     transferase produced Cryptococcus neoformans. Infect Immun 65: 405&#226;&#8364;&#8220;411.                       186&#226;&#8364;&#8220;190. <br /> 44. Zhu X, Gibbons J, Garcia-Rivera J, Casadevall  Williamson PR (2001)                        57. Davidson RC, Blankenship JR, Kraus PR, Jesus Berrios M, Hull CM, et al. <br />     Laccase Cryptococcus neoformans cell wall-associated virulence factor. Infect            (2002) PCR-based strategy generate integrative targeting alleles large <br />     Immun 69: 5589&#226;&#8364;&#8220;5596.                                                                             regions homology. Microbiology 148: 2607&#226;&#8364;&#8220;261 <br /> </body></html>