<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Pac Symp Biocomput. 2009 ; : 391&#226;&#8364;&#8220;402. <br />  <br />  <br />  <br />                            HIGH THROUGHPUT INTERACTION DATA REVEALS DEGREE <br />                            CONSERVATION HUB PROTEINS <br />                             Fox1,*, D. Taylor1, D. K. Slonim1,2 <br />                            1Department Science, Tufts University, 161 College Avenue, Medford, MA 02155, <br />  <br />                            USA <br />                            2Department         Pathology, Tufts University School Medicine, 145 Harrison Ave., Boston, MA <br />                            02111, USA <br />  <br />                            Abstract <br />                                    Research model organisms relies unspoken assumptions conservation protein- <br />                                    protein interactions species, analyses suggest conservation limited. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                    Fortunately, purposes crucial issue global conservation interactions,                                    preferential conservation functionally important ones. observed bias essentiality                                    highly-connected proteins implies functional importance &#226;&#8364;&#339;hubs&#226;&#8364;?. define <br />                                    notion degree-conservation demonstrate hubs preferentially degree-conserved. <br />                                    protein likely hub high-degree ortholog,                                    protein hub, tends remain  identify positive correlation                                    average degree protein conservation interaction partners,                                    conservation individual hub interactions surprisingly high. work important <br />                                    implications prediction protein function, computational inference PPIs, interpretation <br />                                    data model organisms. <br />  <br />  <br />                            1 Introduction <br />                                                 power comparative genomic research grown steadily availability                                                 genomic sequence annotation increasing numbers organisms. variety                                                 techniques solving diverse problems motif discovery[1], gene expression <br />                                                 analysis[2], regulatory network inference[3] interactome discovery[4] rely NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 assumption functionally important protein-DNA protein-protein interactions (PPIs) <br />                                                 preferentially conserved species. remains fully understand <br />                                                 conservation protein interaction modules functions[5]. studies                                                 &#226;&#8364;&#732;interologs&#226;&#8364;&#8482; (orthologous pairs proteins interactions conserved)[6] report <br />                                                 preferential conservation interactions highly similar orthologous protein pairs[7], <br />                                                 Mika Rost[8] point inference PPIs based sequence homology                                                 species continues quite inaccurate, high levels sequence identity. <br />  <br />                                                 interpretation data model organisms, low interolog conservation                                                 acceptable, provided functionally important interactions                                                 orthologs exist species conserved acceptably high rate.                                                 remains shown. Recent work[9] shows functional modules better conserved <br />                                                 yeast fly expected chance. Furthermore, previous work                                                 studying transcriptional regulation shows despite preferential sequence conservation <br />  <br />  <br />  <br />                            * <br />                                afox01@cs.tufts.edu. <br />                             Fox et al.                                                                                              Page 2 <br />  <br />  <br />                                          functionally important proteins, functional regulatory modules especially                                          conserved[10]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           focus conservation interactions proteins,                                          highly-connected proteins, &#226;&#8364;&#339;hubs&#226;&#8364;? species&#226;&#8364;&#8482; interaction network. select hub <br />                                          proteins precisely high connectivity serve indication functional <br />                                          importance.  proteins interactions likely essential[11],                                           hubs connected hubs essential 12]. hub proteins <br />                                          likely functionally important, roles hubs preferentially <br />                                          conserved, individual interactions  investigate <br />                                          property hub protein conserved organisms,                                          determine level PPI conservation hubs neighbors.                                          interactions high degree proteins preferentially conserved                                          specific interactions hubs lost gained, high degree hub protein                                          nonetheless conserved.  greater hope functional importance                                          hubs conserved  <br />  <br />                                          point ample reason expect priori high degree hub <br />                                          proteins preferentially conserved evolution[13]. based                                          argument protein evolved functional interactions proteins, <br />                                          dramatic change affect partners, likely NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          evolutionarily disadvantageous.  given recent work[9, 10] casts doubt                                          modular functional conservation, hypothesis requires proof. <br />  <br />                                          reasons skeptical quality quantity available PPI data. <br />                                          Estimates false-positive rates high throughput interaction data sets range                                          45% 90%[14, 15], coverage estimates human yeast interactomes                                          10% 50%, respectively[14, 16]. <br />  <br />                                           attempts identify high degree proteins affected noise, coverage,                                          testing bias[16]. Methods estimating true degree interactions[17] available,                                          directly applicable data sets studied  cases clear <br />                                          interactions tested. Nonetheless, modest data filtering, obtain clear <br />                                          evidence important trends. <br />  <br />                                          results provide additional evidence identifying biologically relevant high-degree <br />                                          proteins model organisms shed light human response. work                                          important implications protein function prediction, comparative genomics, network <br />                                          inference analysis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2 Background Definitions <br />                               2.1 Data Sources <br />                                          PPIs Homo sapiens, Drosophila melanogaster, Caenorhabditis elegans,                                          Saccharomyces cerevisiae downloaded BioGRID[18], IntAct[19]                                          MINT[20] online databases July, 2008 combined form single large data set. <br />  <br />                                          map orthologous proteins species, used InParanoid database[21]                                          discriminates true orthologs   paralogs differentiated relevant <br />                                          speciation event   paralogs arose speciation. <br />  <br />  <br />  <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                  Page 3 <br />  <br />  <br />                               2.2 Filtering PPI Noise <br />                                           improve reliability data[17], built PPI graph assay method,                                           directed edges (b, p) indicating roles protein (b)ait (p)rey NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           appropriate. define baitrank protein p species s assay type                                           fraction proteins species s having (non-zero) degree p&#226;&#8364;&#8482;s degree                                           assay  preyrank defined analogously degree. given degree  <br />                                           degree zero corresponding baitrank preyrank undefined. <br />  <br />                                           concluded assay type inconsistent protein p p&#226;&#8364;&#8482;s baitrank                                           preyrank defined |baitrank(p) &#226;&#710;&#8217; preyrank(p)| &gt; 0.1. conditions                                           removed data assay type p. filtering, degrees estimated using <br />                                           naive method[17] remaining data. Overall, approach removed 36%                                           data, led stronger results. datasets descriptive statistics (Supplemental File 1) <br />                                           available http://bcb.cs.tufts.edu/hubsPSB09. <br />  <br />                                2.3 Ranking Protein Degrees <br />                                           Degrees proteins different species directly compared species <br />                                           different degree distribution. address issue, ranked proteins                                           species degree filtered dataset species, worked                                           ranks instead raw degree. define r, rank protein p degree d species s, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           fraction proteins species s having non-zero degree d. <br />  <br />                                           Individual proteins labeled hubs using straightforward threshold rank.                                           hub threshold t defined interval [0.5,1) intuitively, makes little sense talk <br />                                           protein rank 0.5 hub. <br />  <br />                               2.4 Degree Conservation <br />                                           protein considered evolutionarily conserved                                           orthologous proteins species consideration.                                           literature conservation hub proteins focused type sequence <br />                                           conservation[22, 23]. end spectrum, consider specific <br />                                           PPIs conserved, given pairs proteins known orthologs species. <br />                                            conserved interactions frequently referred interologs[6].) <br />  <br />                                           paper, focus middle ground - looking interaction degree                                           protein conserved. conservation occur specific interologs                                           maintained. define pair orthologous proteins degree-conserved hub threshold t <br />                                           proteins hubs hub threshold t respective species. <span id='am-1' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           naturally extended species. <br />  <br />                               3 Having Orthologous Hub Increases Hub Likelihood <br />                               3.1 Degree Conservation Ortholog Pairs <br />                                           analyzed pairs orthologous proteins species. created                                           dataset D containing proteins filtered data species                                           proteins orthologous. interested determining proteins                                           hub ortholog significantly higher probability hub protein. <br />  <br />                                            compare observed posterior probability protein hub, given                                           ortholog species hub, prior probability protein                                           hub data set. simple way define prior probability hub just (1 &#226;&#710;&#8217; t), <br />                                           t hub threshold. uninformed prior, U(t).   <br />  <br />                                           Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                     Page 4 <br />  <br />  <br />                                          considering proteins orthologs 1 3 species consideration,                                          shown proteins distant orthologs higher average degree[24], <br />                                          true prior higher.  compute informed prior, t). Let hubt(p) indicate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          p hub threshold t let orth(x, y) indicate x y orthologs. t) = <br />                                          Pr[hubt(p) = 1], represents observed fraction proteins data set                                          hubs. compute t) dividing number proteins D rank &#226;&#8240;&#165; t total size <br />                                          D. <br />  <br />                                          define posterior probability P(t) = Pr[hubt(pi) = 1| &#226;&#710;&#402; pj s pecies, s.t. <br />                                          orth(pi, pj) &#226;&#710;&#167; hubt (pj) = 1]. compute P(t), create set St containing proteins <br />                                          ortholog species hub threshold t. P(t) = |{pi &#226;&#710;&#710; St | <br />                                          hubt (pi) = 1}|/|St|. Intuitively, P(t) fraction proteins St hubs threshold t. <br />                                          Figure 1a shows P(t), t) U(t). <br />  <br />                                          compare observed distribution P informed prior using Kolmogorov- <br />                                          Smirnov (KS) Test[25]. used 1-sided KS Test determine [P &gt;                                           intervals t &#226;&#710;&#710; [x, 1) x &#226;&#710;&#710; {0.5, 0.55,&#226;&#8364;&#166;, 0.95}. Tests significant p &lt; 10&#226;&#710;&#8217;6                                          intervals. results support hypothesis probability protein hub                                          higher ortholog hub, implying hub proteins degree-conserved                                          expected chance. way, determined significantly greater                                          U, expected reason described  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          determine increase hub probability given existence orthologous hub,                                          plot fold-change probabilities F(t) = P(t) t) Figure 1b. graph F(t)                                          probability given protein hub greater hub ortholog                                          magnitude effect increases higher hub threshold.  F(t) peaks                                          approximately t = &#206;&#179; = 0.99. t &gt; &#206;&#179; expected number conserved hubs quickly <br />                                          approaches zero, diminishing expectation number degree-conserved <br />                                          pairs data set  numerator P(t)) t &gt; &#206;&#179;. hub thresholds t &#226;&#8240;&#710; 0.93                                          fold increase hub likelihood. <br />  <br />                               3.2 Degree Conservation Species <br />                                          extend pairwise ortholog analysis look ortholog groups                                          species. create 4-component rank vectors xi = [rank(yi), rank(wi), rank(fi), rank(hi)] <br />                                          yi, wi, fi hi proteins form ortholog group yeast, worm, fly                                          human. Let nT total number rank vectors, n(t) number rank vectors                                          exhibit degree-conservation hub threshold t, O(t) = n(t)/nT proportion                                          ortholog groups exhibit hub conservation hub threshold t. compute                                          expected rate conservation degree conserved orthology E(t). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          E(t) = &#206;&#160;c&#226;&#710;&#352;M Pr[r &gt; t|r &#226;&#710;&#710; c], M matrix rank 4-vectors generated                                          orthology groups c represents column ranks specific particular species. O(t) <br />                                          significantly greater E(t) conclude degree fact conserved                                          orthology. <br />  <br />                                          compare observed rate 4-species degree-conservation O(t) expected rate <br />                                          E(t) using series KS tests. tested interval t &#226;&#710;&#710;   b] &#226;&#710;&#710; {0.5, 0.51,&#226;&#8364;&#166;, <br />                                          0.95} b &#226;&#710;&#710; {0.55, 0.56,&#226;&#8364;&#166;, 1.0} &#226;&#8240;&#164; (b &#226;&#710;&#8217; 0.05). KS test results showed [O &gt; <br />                                          E] t-intervals fully subsumed interval t &#226;&#710;&#710; [0.66, 0.95] (p &lt; 10&#226;&#710;&#8217;6)                                          conclude high degree conserved considered species. <br />  <br />                                          lack 4-species degree-conservation t &gt; 0.95 attributed incomplete <br />                                          interaction data, low overlap interactome coverage species[26], combination <br />                                          effects. <br />  <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                Page 5 <br />  <br />  <br />                                          gain understanding functional classes proteins degree-conserved,                                          performed functional enrichment analysis (DAVID[27]) yeast proteins degree- <br />                                          conserved (t = 0.8) 1 3 species. Enriched functions included primary <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          metabolism, protein synthesis, splicing, DNA repair regulation cell-cycle (FDR &lt; <br />                                          0.05). Enrichment basic cellular processes expected given hubs required <br />                                          degree-conserved yeast higher eukaryotes. contrast, degree-conserved <br />                                          (t = 0.8) proteins human fly enriched processes cell signalling <br />                                           p53, Wnt, ErbB, Notch TGF-&#206;&#178; pathways), cell differentiation, cellular <br />                                          developmental processes, post-translational protein modification regulation                                          protein kinase activity (FDR &lt; 0.05). details functional enrichment results <br />                                          (Supplemental File 2) available online http://bcb.cs.tufts.edu/hubsPSB09. <br />  <br />                               4 Hub, Hub <br />                                          hypothesis hubs exhibit degree-conservation relies intuition                                          protein evolved connections, difficult protein lose importance <br />                                          neighbors affected change.  analyses far <br />                                          fail account orthologs diverged proteins evolved                                          observed &#226;&#8364;&#339;hub&#226;&#8364;? role. simple evolutionary model expects hubs remain highly <br />                                          connected account larger fraction data.                                           hub, hub&#226;&#8364;? hypothesis. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          test hypothesis, simple phylogenetic tree (Figure 2a) relating                                          species data set. seek determine closely data matches                                          phylogenetic model. ortholog group spanning species, form rank <br />                                          vector Section 3.2. create 4-bit bitstring describing hub status                                          component protein species.  given protein&#226;&#8364;&#8482;s rank greater hub <br />                                          threshold t, species&#226;&#8364;&#8482; component bitstring 1; 0.) ortholog <br />                                          group considered consistent model  assumption parsimony) bitstring <br />                                          explained evolution hub property point tree (Figure <br />                                          2b). <br />  <br />                                          consider possible ortholog groups species. Figure 3 shows                                          fraction ortholog groups consistent model grows hub threshold increases. <br />                                          t &gt; 0.73, majority data explained straightforward phylogenetic <br />                                          model hub property selective pressure retained                                          evolved. <br />  <br />                                          caveat sufficiently high t, data zero bitstring                                          consistent model.  potentially growth fraction NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          consistent ortholog groups category.                                           case repeating analysis excluding zero data (Figure 3). details                                          consistency analysis (Supplemental File 3) provided http://bcb.cs.tufts.edu/ <br />                                          hubsPSB09. <br />  <br />                                5 Hub Interactions Preferentially Conserved <br />                                          Finally, examine retention hub property retention                                          individual PPIs, simply general importance high degree proteins. consider <br />                                          models.  interologs conserved sufficiently - high degree <br />                                          proteins - &#226;&#8364;&#339;module&#226;&#8364;? structure hub neighbors retained. suggests <br />                                          function protein partners likely conserved                                           alternative hypothesis assumes protein evolved large number partners,                                          pressure evolve slowly[24]. protein&#226;&#8364;&#8482;s interaction <br />  <br />                                          Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                             Page 6 <br />  <br />  <br />                                         partners lost time, high sequence stability hubs allow new <br />                                         interactions arise retained easily. Note hypothesis                                         somewhat odds growing body work suggesting interologs conserved NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         moderate level[7, 26], functional modules generally conserved intact[9]. <br />  <br />                                         address question, analyzed conservation individual interactions                                         function protein interaction degree. measure conservation interactions                                         given protein p species s1, designated second species s2, rely following <br />                                         definitions.  let G1 protein-protein interaction graph species s1: G1 = <br />                                         &#226;&#338;&#169;V1,E1&#226;&#338;&#170;, let G2 similarly defined species s2. Furthermore, let relation orth, <br />                                         defined V1 &#195;&#8212; V2, denote orthologous pairs proteins species. Let                                         potential number conserved interactions p, Poten(p) number protein <br />                                         interaction partners q p s1 orthologs p q exist s2. let                                         actual number conserved interactions p, Conserv(p) number protein <br />                                         interactions p s1 conserved s2. Formally, protein p &#226;&#710;&#710; V1, define <br />                                         Poten(p) = |{q &#226;&#710;&#710; V1|&#226;&#338;&#169;p, q&#226;&#338;&#170; &#226;&#710;&#710; E1 &#226;&#710;&#167; (&#226;&#710;&#402;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#710;&#710; V2|orth(p, p&#226;&#8364;&#178;) &#226;&#710;&#167; orth(q, q&#226;&#8364;&#178;))}| Conserv(p) = <br />                                         |{q &#226;&#710;&#710; V1|&#226;&#338;&#169;p, q&#226;&#338;&#170; &#226;&#710;&#710; E1 &#226;&#710;&#167; (&#226;&#710;&#402;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#710;&#710; V2|orth(p, p&#226;&#8364;&#178;) &#226;&#710;&#167;orth(q, q&#226;&#8364;&#178;) &#226;&#710;&#167; &#226;&#338;&#169;p&#226;&#8364;&#178;, q&#226;&#8364;&#178; &#226;&#338;&#170; &#226;&#710;&#710; E2)}|.                                         compute conservation rate R(p) = Conserv(p)/Poten(p) proportion                                         interactions conserved interactions possibly conserved. <br />  <br />                                         measured conservation rate R(p) protein pair species compared <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         rank(p). Results fly proteins shown Figure 4. plot reveals                                         proportion protein&#226;&#8364;&#8482;s interactions conserved increases protein rank  degree) <br />                                         increases. words, hubs preferential conservation individual protein <br />                                         interactions non-hubs. <br />  <br />                                         species pairs, graphs similar shape (Supplemental File 4),                                         overall conservation highest degree proteins lower (65%&#226;&#8364;&#8220;85%).                                         especially high conservation fly human interologs reflect bias                                         protein interactions chosen test assays particular species.                                         similar species pairs,  high hub thresholds, half                                         interologs conserved. contrasts sharply general analysis Mika                                         Rost[8], shows interolog conservation hovering 25% highest levels <br />                                         sequence similarity, using data overlaps significantly  conclude                                         interolog conservation hub proteins simply fact                                         highly-connected proteins higher sequence conservation, interactions                                         truly preferentially conserved. <br />  <br />                                         note number species pairs, significant fraction  40%) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         interactions conserved high hub thresholds. suggests degree- <br />                                         conservation hubs rely solely conservation individual pairwise <br />                                         interactions, functional roles hub proteins  <br />  <br />                               6 Discussion <br />                                         demonstrated important novel facts conservation protein <br />                                         interactions function.  likelihood protein hub increases                                         ortholog hub. Second, protein evolves hub, tends remain  <br />                                         specific interaction partners gained lost. Finally, rate interolog <br />                                         conservation hub proteins higher low-degree proteins. <br />  <br />                                         results imply  said functional conservation modules <br />                                         defined various clustering methods (e.g., Wang Zhang[9]), functional modules <br />                                         network hubs likely persist species. result crucial implications <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                              Page 7 <br />  <br />  <br />                                         unspoken assumption underlying research model organisms:                                         functional relationships key proteins conserved humans model <br />                                         organisms studied. Furthermore,  hub, hub&#226;&#8364;? property implies NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         possible infer identity hub proteins organisms sufficient <br />                                         protein interaction data sets available. example, orthologous proteins <br />                                         worm fly PPI hubs, reasonable assume evolution                                         hub property occurred ancestor worm fly;  orthologous <br />                                         protein bees likely hub  <br />  <br />                                         addition, stronger expected conservation individual interactions hub <br />                                         proteins important implications computational prediction protein-protein <br />                                         interactions. growth experimentally-generated protein interaction data                                         substantial, interactomes organisms likely remain largely incomplete                                         time, computational inference interactions continue play important <br />                                         role. Furthermore, suggested Matthews et al.[4], computational predictions                                         used guide selection experimental assays, higher predictive accuracy reduce <br />                                         cost screening, allowing generation data. <br />  <br />                                         Despite results, issues surrounding data require caution                                         interpreting results based existing protein interaction networks. importantly,                                         data noisy extraordinarily incomplete. different subsets NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         interactomes species investigated, possible bias species <br />                                         coverage affect conclusions based data currently available. addition,                                         methods identifying protein interactions  coimmunoprecipitation affinity <br />                                         precipitation[9]) identify protein complexes pairwise interactions. pairwise <br />                                         interactions inferred participants complexes[28], potentially <br />                                         creating high-degree nodes erroneously.  informative revisit                                         conclusions quality quantity interactome data grow. <br />  <br />                               Supplementary Material <br />                                         Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                         research supported award number R21LM009411 National Library Medicine. content <br />                                         solely responsibility authors does necessarily represent official views National <br />                                         Library Medicine National Institutes Health. thank members Tufts BCB research group <br />                                         helpful discussions comments. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               References <br />                                         1. Blanchette M, Tompa M. Genome Res. 2002; 12(5):739. [PubMed: 11997340] <br />                                         2. Eisen M, Spellman P, Brown P, Botstein D. PNAS. 1998; 95:14863. [PubMed: 9843981] <br />                                         3. Yu J, Smith V, Wang P, Hartemink  Jarvis E. Bioinformatics. 2001; 20:3594. [PubMed: <br />                                            15284094] <br />                                         4. Matthews L, Vaglio P, Reboul J, Ge H, Davis B, Garrels J, Vincent S, Vidal M. Genome Res. 2001; <br />                                            11:2120. [PubMed: 11731503] <br />                                         5. Han J, Bertin N, Hao T, Goldberg D, Berriz G, Dupuy D, Walhout  Cusick M, Roth F, Vidal M. <br />                                            Nature. 2004; 88:6995. <br />                                         6. Walhout  Sordella R, Lu X, Hartley J, Temple G, Brasch M, Thierry-Mieg N, Vidal M. Science. <br />                                            2000; 287:116. [PubMed: 10615043] <br />                                         7. Yu H, Luscombe N, Lu H, Zhu X, Xia Y, Han J. Genome Res. 2004; 14 <br />                                         8. Mika S, Rost B. PLoS Comput. Biol. 2006; 2:e79. [PubMed: 16854211] <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                                Page 8 <br />  <br />  <br />                                         9. Wang  Zhang J. PLoS Comp Biol. 2007; 3 <br />                                         10. Babu M, Luscombe N, Aravind L, Gerstein M. Curr Opin Cell Biol. 2004; 14:283. <br />                                         11. Jeong H, Mason S, Barabasi  Oltvai  Nature. 2001; 411:41. [PubMed: 11333967] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         12. Batada N, Reguly T, Breitkreutz  Boucher L, Breitkreutz B, Hurst L, Tyers M. PLoS Biol. 2006; <br />                                             4:e317. [PubMed: 16984220] <br />                                         13. Wuchty S, Barabasi  Ferdig M. BMC Evol Biol. 2006; 6:1471. <br />                                         14. Hart G, Ramani  Marcotte E. Genome Biol. 2006; 7:120. [PubMed: 17147767] <br />                                         15. D&#226;&#8364;&#8482;haeseleer P, Church G. Proc IEEE Comp Syst Bioinformatics. 2004 Aug.216 <br />                                         16. Chiang T, Scholtens D, Sarkar D, et al. Genome Biol. 2007; 8:186. <br />                                         17. Scholtens D, Chiang T, Huber W, Gentleman R. Bioinformatics. 2008; 24:218. [PubMed: <br />                                             18025006] <br />                                         18. Stark C, Breitkreutz B, Reguly T, Boucher L, Breitkreutz  Tyers M. Nucleic Acids Res. 2006 <br />                                             Jan.34:D535. [PubMed: 16381927] <br />                                         19. Hermjakob H, Montecci-Palazzi L, Lewington C, Mudali S, Kerrien S, et al. Nucleic Acids Res. <br />                                             2004; 32:D452. [PubMed: 14681455] <br />                                         20. Zanzoni  Montecci-Palazzi L, Quondam M, Ausiello G, Helmer-Citterich M, et al. FEBS <br />                                             Letters. 2002; 513:135. [PubMed: 11911893] <br />                                         21. Remm M, Storm C, Sonnhammer E. JMB. 2001; 314:1041. <br />                                         22. Fraser H, Hirsh  Steinmetz L, Scharfe C, Felman M. Science. 2002; 296:750. [PubMed: <br />                                             11976460] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         23. Jordan  Wolf  Koonin E. BMC Evol Biol. 2003; 3:1. [PubMed: 12515583] <br />                                         24. Brown K, Jurisica  Genome Biol. 2007; 8 <br />                                         25. Press, W.; Teukolsky, S.; Vetterling, W.; Flannery, B. Numerical Recipes C, Art                                             Scientific Computing. Cambridge Univeristy Press; 1992. <br />                                         26. Gandhi T, Zhong J, Mathivanan S, et al. Nature Gen. 2006:38. <br />                                         27. D G Jr, Sherman B, Hosack D, et al. Genome Biol. 2003; 4:3. <br />                                         28. Bader G, Hogue C. Nature Biotechnol. 2002; 20:991. [PubMed: 12355115] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                            Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 1. <br />                                         Hub likelihood plots.  Conditional probability hub given ortholog                                         hub, compared prior probability hub. solid line represents P(t),                                         posterior (conditional) hub probability; dot-dash line U(t) uninformed prior <br />                                         probability, dashed line t) informed prior based observed data. b) Fold <br />                                         Increase F hub likelihood, given orthologous hub protein, function hub <br />                                         threshold t. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 2. <br />                                          Simple phylogenetic model species data set, based data                                         Tree Life database (www.tol.org). b) Consistency hub-status bitstrings <br />                                         (yeast,worm,fly,human)  hub, hub&#226;&#8364;&#8482; hypothesis. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 3. <br />                                         Consistency  hub, hub&#226;&#8364;? hypothesis, function hub threshold. <br />                                         y-axis shows percentage ortholog groups consistent hypothesis, <br />                                         given phylogenetic model Figure 2. solid line represents consistent ortholog <br />                                         groups; dashed line excludes zero vectors (proteins hubs                                         species) computation. <br />  <br />  <br />  <br />  <br />                                         Pac Symp Biocomput. Author manuscript; available PMC 2009 December 17. <br />                             Fox et al.                                                                                          Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Figure 4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                         Interolog conservation function protein interaction degree rank. data shown                                         fly proteins orthologs species. Similar-rank <br />                                         proteins binned conservation rates averaged trend seen clearly. <br />                                         Conservation interologs low low-degree proteins, increases function              <br /> </body></html>