<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Low-Oxygen-Induced NAC Domain Transcription <br /> Factor ANAC102 Affects Viability Arabidopsis Seeds <br /> following Low-Oxygen Treatment1[W][OA] <br />  <br /> Jed  Christianson, Iain W. Wilson*, Danny J. Llewellyn, Elizabeth S. Dennis <br /> CSIRO Plant Industry, Canberra, Australian Capital Territory 2601, Australia <br />  <br />  <br /> Low-oxygen stress imposed field waterlogging impediment plant germination growth. Plants respond waterlogging complex set physiological responses regulated transcriptional, cellular, tissue levels. Arabidopsis (Arabidopsis thaliana) NAC domain-containing gene ANAC102 shown induced 0.1% oxygen 30 <br /> min roots shoots 0.1% oxygen-treated germinating seeds. Overexpression ANAC102 altered expression number genes, including previously identified low-oxygen responsive. Decreasing ANAC102 <br /> expression effect global gene transcription plants did alter expression patterns low-oxygen-stressed seeds. <br /> Increasing decreasing expression ANAC102 did affect adult plant survival low-oxygen stress. Decreased ANAC102 <br /> expression significantly decreased germination efficiency following 0.1% oxygen treatment, increased expression effect germination. protective role germination appeared specific low-oxygen stress, implicating ANAC102 important regulator seed germination flooding. <br />  <br />  <br />    Transient waterlogging, impose low-                          observed Brassica oleracea, close <br /> oxygen stress established plants, shown                       relative Arabidopsis (Arabidopsis thaliana; Finch- reduce yield number crops, including cotton                    Savage et al., 2005). <br /> (Gossypium hirsutum; Hodgson Chan, 1982), wheat                          effects low oxygen seeds vary (Triticum aestivum; Collaku Harrison, 2002), barley                   plant species. species, lack oxygen <br /> (Hordeum vulgare; Setter Waters, 2003), maize (Zea                    prevents germination. cereals, barley, <br /> mays; Mason et al., 1987), canola (Brassica napus;                    rely limiting oxygen availability embryos Cannell Belford, 1980). rice (Oryza sativa),                     mechanism imposing maintaining dormancy adapted growing partially underwater,                    (Benech-Arnold et al., 2006). contrast, rice seeds adversely affected entire plant sub-                       able germinate anoxia, capability <br /> merged (Singh et al., 2001). growing                        dependent ethanolic fermentation pathways <br /> plants, waterlogging poses significant problem                       (Kato-Noguchi, 2001).  little known <br /> seeds prior emergence. Germination rates decline                       effects low oxygen seed viability dramatically oxygen concentrations reduced                     Arabidopsis. Germination did occur wild-type <br /> 12 plant species, including Brassica vegetable species,                   Arabidopsis seeds anoxic environment soybean (<span id='am-38' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Glycine</span> max), pea (Pisum sativum), wheat,                        severely reduced treated seeds following maize, rice (Al-Ani et al., 1985). Soil-based exper-                  recovery period (Mattana et al., 2007). impair- <br /> iments shown large impacts seed viability                    ment germination partially ameliorated following 4 d flooding stress oat (Avena sativa),                   overexpression rice transcription factor Mybleu <br /> triticale (x Triticosecale), wheat, barley (Setter                (Mattana et al., 2007). Arabidopsis germination Waters, 2003). Declines seed germination num-                     reduced significantly ALCO- <br /> ber low-oxygen environments (1%215% oxygen)                            HOL DEHYDROGENASE1 (ADH1) mutants                                                                           wild-type plants following anoxic treatments (Jacobs <br />                                                                           et al., 1988), suggests role fermentative <br />    1 <br />      work supported CottTech, research alliance             metabolism sustaining seeds low oxygen. Commonwealth Scientific Industrial Research                  Following perception lack oxygen, changes Organization, Cotton Seed Distributors, Cotton Research           gene transcripts, proteins, metabolism rapidly Development Corporation.                                              result. set 20 anaerobically induced poly- <br />    * Corresponding author; e-mail iain.wilson@csiro.au.                   peptides, majority involved <span id='am-26' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-27' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>gly</span>- <br />    author responsible distribution materials integral   colysis fermentation, identified findings presented article accordance policy <br /> described Instructions Authors (www.plantphysiol.org)  <br />                                                                           low-oxygen response number plant <br /> Iain W. Wilson (iain.wilson@csiro.au).                                    species (Sachs et al., 1980; Kelley Freeling, 1984a, <br />    [W] <br />        online version article contains Web data.         1984b; Dennis et al., 2000). <br />    [OA] <br />         Open Access articles viewed online sub-             Functional approaches studying effects low <br /> scription.                                                                oxygen focused adult plants involved <br />    www.plantphysiol.org/cgi/doi/10.1104/pp.108.131912                     altering expression enzymes fermenta- <br /> 1724            Plant Physiology, April 2009, Vol. 149, pp. 1724&#226;&#8364;&#8220;1738, www.plantphysiol.org &#195;&#8220; 2009 American Society Plant Biologists <br />                                                                       ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br /> tion glycolysis. genes SUCROSE SYNTHASE1              ANC055, ANAC072 shown increase <br /> (SUS1) SUS4, ADH1, PYRUVATE DECAR-                    drought tolerance Arabidopsis (Tran et al., 2004). BOXYLASE1 (PDC1) shown vital               addition modulating lateral root formation Arab- <br /> Arabidopsis tolerance low oxygen, knocking          idopsis, AtNAC2 salt responsive  et al., function leads reduction plant growth       2005). NTL8 gene shown influence survival (Ellis et al., 1999; Rahman et al., 2001;         flowering time salt-stress conditions (Kim et al., <br /> Ismond et al., 2003; Kursteiner et al., 2003; Bieniawska      2007). ANAC102 belongs ATAF subfamily et al., 2007). Overexpression PDC, considered        NAC domain genes. Members ATAF subfamily, <br /> rate limiting ethanol fermentation, shown         ATAF1 ATAF2, negatively regulate responses increase Arabidopsis survival (Ellis et al., 2000;         drought wounding, respectively (Delessert et al., <br /> Ismond et al., 2003). Increased root survival        2005; Lu et al., 2007). <br /> observed LACTATE DEHYDROGENASE (LDH)-                          investigated role ANAC102 plant <br /> overexpressing Arabidopsis lines (Dolferus et al.,            responses low oxygen provide evidence 2008). Altering expression ALANINE AMINO-              role gene transcriptional regulation TRANSFERASE, controls Ala fermentation              low-oxygen response important positive <br /> pathway, did affect tolerance low oxygen            regulator seed viability low-oxygen conditions. <br /> Arabidopsis (Dolferus et al., 2003; Miyashita et al., <br /> 2007). candidate gene approach yielded promising results, overexpression transcrip-          RESULTS <br /> tion factors alter expression stress <br />                                                               ANAC102 Regulated Shoots, Roots, response networks greater impact.                                                               Imbibed Seeds Response 0.1% Oxygen <br /> SUB1A ERF/AP2-type transcription factor rice proven capable increasing survival fully sub-             ANAC102 significantly regulated early time <br /> merged conditions regulating transcription        points hairy root cultures exposed oxygen stress <br /> suite genes associated <span id='am-7' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span>carbohydrate</span> consump-          (Klok et al., 2002). confirm ANAC102 induced <br /> tion, ethanolic fermentation, cell expansion       plants, 3-week-old Arabidopsis plants leading decreased growth, chlorophyll degradation,         subjected 0.1% oxygen 0, 0.5, 2, 4, 8, 24 h carbohydrate consumption (Fukao et al., 2006; Xu          (Fig. 1A). root shoot tissues showed similar <br /> et al., 2006).                                                response, increased expression detected 0.5 h <br />    Global gene expression studies Arabidopsis         remaining high 24 h (Fig. 1A). revealed widespread complex responses low              substantial difference root- shoot- <br /> oxygen typically significant changes       specific expression untreated plants (1.2-fold). Low- <br /> approximately 5% 10% genes assayed (Klok        oxygen induction gene expression occurred et al., 2002; Liu et al., 2005; Loreti et al., 2005). Early   presence 10 mM cycloheximide (17.0-fold; data <br /> microarray experiments low-oxygen responses             shown), demonstrating new protein syn- <br /> Arabidopsis hairy root cultures (Klok et al., 2002)           thesis required ANAC102 induction. <br /> identified known putative transcription factors             classical hypoxia-induced genes, expression induced early low-oxygen              ADH, LDH, AlaAT, exhibit increased expression stress, including NAC domain-containing gene              germinating seeds following exposure low oxygen <br /> ARABIDOPSIS NAC DOMAIN-CONTAINING PRO-                        levels (Ricoult et al., 2005). Imbibed stratified <br /> TEIN102 (ANAC102; At5g63790).                                 seeds, unstressed subjected 0.1% oxygen    NAC domain genes plant-specific class             light period 6 d  germination occurs <br /> transcription factors functions development           seeds 0.1% oxygen), assayed stress responses  review, Olsen et al., 2005).    ANAC102 ADH1 expression. ANAC102 Arabidopsis contains 105 genes NAC domains,              ADH1 expression low-oxygen-stressed seeds divided major groups                increased unstressed seeds (2.5-fold 11.9- partitioned 18 subgroups (Ooka et al.,           fold, respectively), expression level 2003). ANAC transcription factors characterized           ANAC102 ADH1 unstressed seeds lower N-terminal DNA-binding NAC domain com-                  unstressed adult tissue (3.2-fold 2-fold <br /> posed subelements variable C-terminal           lower expression seeds roots; Fig. 1B). <br /> transcriptional activator region (Ooka et al., 2003). NAC domain identified characterized          ANAC102 Expressed Roots Early petunia (Petunia hybrida) APICAL MERISTEM,             Rosettes, Concentrated Root Cortex ATAF1, ATAF2, CUP-SHAPED COTYLEDON                        Root Caps <br /> genes (Souer et al., 1996; Aida et al., 1997). Numerous <br /> members ANAC gene family shown               characterize spatial developmental pat- involved response stresses: ANAC019,                 terns ANAC102 expression Arabidopsis, lines <br /> ANAC055, ANAC072 regulated               transformed ANAC102 promoterTGUS fusion <br /> sponse drought, high salinity, abscisic acid           generated (Fig. 2). GUS staining clearly <br /> (ABA), overexpression ANAC019,              visible parts root  aerial portion <br /> Plant Physiol. Vol. 149, 2009                                                                                       1725 <br />  Christianson et al. <br />  <br />  <br />                                                                             tional efficiency low-oxygen conditions (Bailey- <br />                                                                             Serres Dawe, 1996). ANAC102 promoterT <br />                                                                             GUS construct used does contain ANAC102 <br />                                                                             untranslated region sequences,                                                                             GUS mRNA produced low oxygen translated <br />                                                                             decreased efficiency, resulting lower amounts <br />                                                                             protein formed despite having higher levels                                                                             transcript. <br />  <br />                                                                             Arabidopsis Lines Altered ANAC102 Levels                                                                             Gross Phenotypic Differences                                                                             Wild Type <br />  <br />                                                                                independent Columbia (Col-0) lines carrying <br />                                                                             insertions second exon ANAC102 gene                                                                             predicted eliminate protein function                                                                             obtained designated KO-1 (SALK_030702)                                                                             KO-2 (SALK_094437). ANAC102 mRNA expression                                                                             lines QRT-PCR                                                                             subsequent microarray experiments 3.5- <br />                                                                             15-fold lower KO-1 approximately 15-fold <br />                                                                             lower KO-2 wild type (Supplemental Fig. <br />                                                                             S1).  independent ANAC102-overexpressing <br />                                                                             lines (OX-1 OX-2) generated C24 <br />                                                                             ecotype, ecotype susceptible low- <br /> Figure 1. Expression ANAC102 response low oxygen.  Time          oxygen stress Col-0 similar ANAC102 <br /> course ANAC102 expression subjected 0.1% oxygen. Plants                                                                             expression profiles (data shown), potentially mak- <br /> ecotype Col-0 liquid MS medium placed chambers <br /> containing 0.1% oxygen 0.5, 2, 4, 8, 24 h. Expression changes <br />                                                                             ing easier detect increase tolerance derived monitored root (black triangles) shoot (white triangles)   ANAC102. OX line 25- 30-fold higher <br /> tissues. Data expression different times relative 0-h time    expression ANAC102 wild type point (6SE; n 5 9). B, Expression ANAC102 ADH1 imbibed            3-week-old plants (Supplemental Fig. S1). gross <br /> seeds ambient 0.1% oxygen (6 d exposure) relative           phenotypic differences observed 3-week-old root tissue ambient oxygen (6SE; n 5 9).                      knockout overexpressing lines respective <br />                                                                             parental ecotypes grown standard laboratory <br />                                                                             conditions, slightly lighter green leaf color plant, faint staining detected older sepals                   ANAC102-overexpressing lines. Lighter green leaves. public Arabidopsis gene expression                          coloration observed closely related <br /> database Genevestigator (Zimmermann et al., 2004)                           ATAF2 gene (ANAC081; .92% <span id='am-19' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> similarity largely corroborated expression patterns observed                       ANAC102) overexpressed C24 (Delessert et al., ANAC102 promoterTGUS fusion lines, indicat-                          2005);  ATAF2 overexpres- <br /> ing high expression lateral root cap, stele,                         sion phenotypes, increased leaf size, wrinkled <br /> epidermal atrichoblasts, endodermis, cortex. Pub-                       leaves, increased biomass, observed lic expression data indicated ANAC102                          ANAC102-overexpressing lines. ANAC102 relatively highly expressed stem sepal.                          highly expressed root tips, obvious <br /> Sepal expression observed ANAC102                                differences root morphology quantity (data promoterTGUS lines (Fig. 2D), stem expression                           shown). low diffuse observed. GUS <br /> expression seen imbibed seeds, expres-                       Overexpression ANAC102 Modifies Expression sion visible radicle seedlings 2 d                   Downstream Genes, ANAC102 Knockout Lines emergence (Fig. 2E). induction GUS expression                         Affect Global Gene Transcription detected following treatment 0.1% oxy-                        Growing Plants <br /> gen 4 h, GUS signal appeared weaker treated plants (Fig. 2, G H). contrast                   ANAC102 putative transcription factor, visible GUS staining, quantitative real-time (QRT)-                         global impact overexpressing knocking PCR demonstrated transcript levels                         ANAC102 Arabidopsis transcriptome ex- <br /> native ANAC102 gene ANAC102 promoterT                               amined using Affymetrix ATH1 Arabidopsis <br /> GUS fusion gene increased plants fol-                         arrays. RNA extracted untreated 4-h <br /> lowing 4-h 0.1% oxygen treatment (Fig. 2I).                        low-oxygen-treated 3-week-old seedlings wild observed maize ADH1 5# 3# untrans-                            type (Col-0), KO-1, KO-2 lines used lated regions important maintaining transla-                        microarray analysis. comparisons 1726                                                                                                       Plant Physiol. Vol. 149, 2009 <br />                                                                                        ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br />  <br />  <br />          Figure 2. Tissue-specific expression, induction ANAC102, vital staining low-oxygen-treated seeds. ANAC102 <br />          promoterTGUS fusion line constructed plants stained 5-bromo-4-chloro-3-indolyl-b-glucuronic acid          localize ANAC102 expression.  Untreated plant. B, Untreated roots. C, Closeup untreated roots. D, Untreated leaf <br />          developing flowers. E, Seedling 2 d germination. F, Vital staining Arabidopsis embryo failed germinate <br />          following 6-d treatment 0.1% oxygen. embryo isolated seed coat prior staining. Green tissue alive,          red-stained tissue dead. C, Embryonic cotyledons; R, embryonic radicle. G, Untreated 3-week-old plant stained 8 h. H, <br />          week-old plant treated 0.1% oxygen 4 h, stained 8 h.  Expression native ANAC102 gene          ANAC102TGUS fusion relative ANAC102 expression untreated plants. Black bars represent expression untreated plants, <br />          hatched bars represent expression plants treated 0.1% oxygen 4 h. Error bars represent 1 SE (n 5 9). <br />  <br />  <br />  <br /> untreated KO lines wild type comparisons                         significantly affected ANAC102 overexpres- low-oxygen-treated KO lines wild type                        sion compared QRT-PCR KO-1, OX-1, showed differences gene expression fold                          wild-type lines time points following expo- <br /> change cutoff 1.5 adjusted P value ,0.05,                       sure 0.1% oxygen (Fig. 3; Supplemental Fig. S2). save ANAC102 itself, underexpressed                         genes examined changed response low <br /> KO lines circumstances (17- 50-fold; Table .                    oxygen levels, agreement previous reports. <br /> RNA extracted 3-week-old seedlings                           11 selected genes identified wild type (C24), OX-1, OX-2 lines used                           microarray regulated OX-1, microarray analysis. total 113 genes                            (At1g02850 [glycosyl hydrolase] At2g43820 [<span id='am-62' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="UDP" datatype="xsd:string"></span>UDP</span>- <br /> regulated 1.5-fold adjusted P value                       glucosyl transferase]) shown QRT-PCR #0.05, 98 genes significantly                           higher expression OX-1 time points. regulated overexpressing line (Supplemen-                        genes highly <br /> tal Table S1). Seventy regulated genes                    expressed OX-1 initial stages low- <br /> 61 regulated genes identified                          oxygen exposure, expression wild differentially regulated low-oxygen                        type increased match OX-1 (Supplemental <br /> microarray experiments (Klok et al., 2002; Branco-Price                      Fig. S2). selected gene identified et al., 2005; Liu et al., 2005; Loreti et al., 2005). Notably,               microarray regulated OX-1 ADH1 SUS1, long recognized                            QRT-PCR regulated compar- <br /> key components low-oxygen response plants,                         ison wild-type line time points. Taken regulated ANAC102 overexpression (2.9-                             overexpression ANAC102 increased fold 1.6-fold, respectively; Table .                                   preinduced expression low-oxygen- <br />    Relative expression levels adult plants 11 genes                    inducible genes decreased delayed induc- <br /> reported low-oxygen responsive shown                          tion  <br /> Plant Physiol. Vol. 149, 2009                                                                                                                 1727 <br />  Christianson et al. <br />  <br />  <br />  <br /> Table  Microarray-derived expression ratios selected genes differential expression ANAC102 mutant lines wild type <br />                                                                                               Expression Ratio   Expression Ratio <br />         Arabidopsis                                          Expression        Expression <br />                                                                                                KO-1 (0.1%       KO-1 Seeds     NAC       NAC <br />           Genome              Gene Title/Description        Ratio OX-1     Ratio KO-1 <br />                                                                                                   Oxygen)         (0.1% Oxygen)      Strictb Consensusc <br />        Initiative  <br />                                                            Ratioa      P      Ratioa    P     Ratioa        P    Ratioa       P <br />  <br />       Selected genes differential expression OX plants <br />         AT5G63790 ANAC102 (Arabidopsis NAC                  18      ,0.01     0.06     0.4    0.02      0.04     0.03       ,0.01      1         5 <br />                         domain-containing protein <br />                         102), transcription factor; <br />                         similar ATAF2 <br />         AT1G02850 Glycosyl hydrolase family 1 protein        8.6    ,0.01     0.67     1      0.53      0.9      0.85         0.11     0         2 <br />         AT2G04040 MATE efflux family protein                 5      ,0.01     0.86     1      0.48      0.9      0.63         0.11     0         2 <br />         AT1G77120 Alcohol dehydrogenase (ADH)                2.9    ,0.01     1.2      1      0.99      1        1            0.9      0         0 <br />         AT2G43820 UDP-glucoronosyl/UDP-glucosyl              5.6    ,0.01     0.57     1      0.74      0.9      0.68         0.01     0         2 <br />                         transferase family protein <br />         AT1G26770 Expansin, putative (EXP10)                 5.1    ,0.01     0.94     1      1.1       0.96     0.96         0.76     1         2 <br />         AT3G59140 ABC transporter family protein             3.4    ,0.01     0.81     1      0.89      0.93     0.93         0.6      1         4 <br />         AT5G43580 Protease inhibitor, putative               2.6     0.01     0.58     1      1.2       0.9      1.1          0.45     1         3 <br />         AT1G72680 Cinnamyl-alcohol                           2.7     0.01     0.66     1      0.71      0.9      0.77         0.09     0         3 <br />                         dehydrogenase, putative <br />         AT2G30140 UDP-glucoronosyl/UDP-glucosyl              2.4    ,0.01     0.68     1      0.66      0.9      0.98         0.94     0         1 <br />                         transferase family protein <br />         AT1G68570 Proton-dependent oligopeptide              2.5    ,0.01     0.86     1      1         1        0.9          0.41     1         4 <br />                         transport (POT) family protein <br />         AT3G56710 sigA-binding protein                       0.4    ,0.01     1.2      1      0.86      0.9      1.1          0.38     1         1 <br />       Selected genes differential expression KO seeds   0.1% oxygen <br />         AT3G22370 Alternative oxidase 1a,                    0.94      0.81 0.86       1      0.95      0.97     1.8        ,0.01      0         2 <br />                         mitochondrial (AOX1A) <br />         AT3G23150 Ethylene receptor, putative (ETR2)         1         0.91   1.1      1      1.2       0.9      0.51       ,0.01      0         0 <br />         AT1G11580 Pectin methylesterase, putative            0.91      0.68   1.2      1      1.1       0.95     0.48       ,0.01      0         3 <br />         AT1G03090 3-Methylcrotonyl-<span id='am-14' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-15' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span>CoA</span>                       1.2       0.28   0.97     1      0.83      0.9      0.47       ,0.05      1         3 <br />                         carboxylase 1 (MCCA) <br />         AT2G40220 Abscisic acid-insensitive 4 (ABI4)         0.95      0.85   1.2      1      1         1        0.6        ,0.01      0         0 <br />       Selected genes differential expression ANAC102     OX plants 0.1% oxygen-treated ANAC102 KO seeds <br />         AT3G46670 UDP-glucoronosyl/UDP-glucosyl              2.4       0.01 0.87      1     0.94      0.96  0.58    0.01               0         0 <br />                         transferase family protein <br />         AT5G61440 Thioredoxin family protein                 2         0.02   1.2      1      0.91      0.9      0.47       ,0.01      0         0 <br />         AT2G02120 Plant defensin-fusion protein,             1.9       0.01   1        1      0.95      0.98     2           0.01      0         4 <br />                         putative (PDF2.1) <br />         AT4G27830 Glycosyl hydrolase family 1 protein 1.8              0.03   0.85     1      0.77      0.89     0.62         0.01     0         2 <br />         AT5G20830 Suc synthase/Suc-UDP                       1.6       0.04   1        1      0.97      0.98     0.55         0.03     0         0 <br />                         glucosyltransferase (ASUS1) <br />         AT2G43590 Chitinase, putative                        0.66      0.04   1.1      1      0.89      0.96     2           0.03      0         0 <br />         AT5G60950 Phytochelatin synthetase-related           0.55      0.05   1.2      1      0.95      0.95     0.65        0.02      0         1 <br />         AT1G02640 Glycosyl hydrolase family 3 protein 0.48             0.04   1.1      1      1.2       0.93     0.48       ,0.01      0         2 <br />         AT1G73480 Hydrolase, b-fold family protein         0.05      0.01   0.79     1      0.7       0.9      0.62        0.02      0         2 <br />      Relative expression ratio OX-1 KO-1 lines compared wild type. OX-1, wild-type background C24; KO-1, wild-type <br />                                                                                                                b <br /> background Col-0. Expression ratios significantly different wild type highlighted boldface.  Number occurrences strict <br />                                                                                                                                        c <br /> NAC domain-binding site motif TTNCGTA 1,000-bp upstream sequence given gene, defined Olsen et al. (2005).           Number occurrences strict NAC domain-binding site motif [TA][TG][TAGC]CGT[GA] 1,000-bp upstream sequence given gene, defined Olsen et al. (2005). <br />  <br />  <br />  <br />    promoter regions genes showing                             copy element. genes differential expression OX-1 analyzed evi-                          regulated OX-1 enriched dence conserved motifs. Analysis Athena                             AGC box enhancer element identified  <br /> visualization tool (O&#226;&#8364;&#8482;Connor et al., 2005),                           bacco (Nicotiana tabacum) class b-1,3-glucanase GLB <br /> identify transcription factor-binding motifs pro-                            gene (Hart et al., 1993), LS7 promoter element <br /> moter sequences, showed enrichment                                 required salicylic acid induction Arabidopsis <br /> ABA-responsive element (ABRE)-like binding site mo-                             PR-1 (Despres et al., 2000), JASE1 motif  <br /> tif (Shinozaki Yamaguchi-Shinozaki, 2000)                            quired leaf senescence jasmonic acid induction regulated genes, 34 113 genes containing                         Arabidopsis OPR1  Gan, 2001). Athena <br /> 1728                                                                                                                      Plant Physiol. Vol. 149, 2009 <br />                                                                                        ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br />                                                                                                       Figure 3. QRT-PCR analysis selected genes <br />                                                                                                       regulated ANAC102-modified expres- <br />                                                                                                       sion lines. Changes expression mon- <br />                                                                                                       itored set genes 3-week-old <br />                                                                                                       Arabidopsis plants Col-0, C24, <br />                                                                                                       KO-1 (white squares), OX-1 (black <br />                                                                                                       squares) lines. Plants grown exposed                                                                                                       0.1% oxygen anaerobic chambers                                                                                                       dark various times specified. Data <br />                                                                                                       presented expression log2-transformed ra- <br />                                                                                                       tios KO-1 OX-1 relative                                                                                                       wild-type lines, Col-0 C24, respectively, <br />                                                                                                       time point. expanded version                                                                                                       figure, including expression data Col-0 <br />                                                                                                       C24, available Supplemental Fig- <br />                                                                                                       ure S2. <br />  <br />  <br />  <br />  <br /> does identify NAC domain transcription factor                              random set Arabidopsis promoter sequences, <br /> binding sites, Toucan analysis tool,                             confirmed strict general consensus <br /> perform searches specified motifs, used                             NAC-binding sites statistically overrepresented <br /> search sequences. set genes                           promoters genes affected ANAC102 overex- <br /> regulated OX-1 set genes regulated                          pression (P , 0.0001; Kankainen Holm, 2004). OX-1 higher expected incidences NAC-binding site motif: 37.8% 48.7% pro-                             Overexpression Underexpression ANAC102 moters data set contained instances                        Significant Effect Adult Plant Tolerance strict NAC core-binding site (TTNCGTA),                             Low Oxygen <br /> 90.9% 96.5% promoters data set <br /> contained instances general                               determine ANAC102 important sensus NAC-binding site ([TA][GT][TACG]CGT[GA];                                plant survival low oxygen, KO OX lines <br /> Olsen et al., 2005; Table II; Supplemental Table S1).                      subjected severe low-oxygen stress. Web-based tool POBO, compares frequency                              separate experiments, plants subjected 0.1% given motif set promoter sequences                           oxygen, 24-h 5% oxygen pretreat- <br />  <br /> Table II. Promoter region analysis NAC-binding sites genes affected ANAC102 expression <br />   1,000-bp upstream regions genes identified significantly affected ANAC102 retrieved Arabidopsis Information <br /> Resource (www.arabidopsis.org). <br />              Gene Seta         Genes   NAC Motif Strictb   POBO T Statisticc   POBO P     NAC Consensusd      POBO T Statisticc    POBO P <br />  <br />       OX-1   regulated         113            48.7%                 46.5          ,0.0001         96.5%                83.1          ,0.0001 <br />       OX-1   regulated        98            37.8%                 49.7          ,0.0001         90.9%                34.0          ,0.0001 <br />       KO-1   regulated          94            38.3%                 41.7          ,0.0001         89.4%                88.1          ,0.0001 <br />       KO-1   regulated       113            38.9%                 43.6          ,0.0001         82.3%                36.6          ,0.0001 <br />                                                                         b <br />    gene sets used Supplemental Tables S1 S4.       strict NAC-binding site motif TTNCGTA, defined Olsen et al. <br />            c <br /> (2005).     T statistic calculated promoter analysis program POBO (Kankainen Holm, 2004) measure difference frequency searched motif given sequence set compared randomly generated background set                d <br /> sequences.      consensus NAC-binding site motif [TA][GT][TACG]CGT[GA], defined Olsen et al. (2005). <br />  <br /> Plant Physiol. Vol. 149, 2009                                                                                                                   1729 <br />  Christianson et al. <br />  <br />  <br />                                                                            Seed Viability Impaired ANAC102 Knockout Lines <br />                                                                            following Low-Oxygen Treatment <br />                                                                               months harvest, seeds line                                                                            overexpressing knocked ANAC102                                                                            parental wild types, collected plants grown                                                                            time growth cabinet, subjected <br />                                                                            6 d exposure 0.1% oxygen light.                                                                            exposure 0.1% oxygen, germination oc- <br />                                                                            curred, contrast seeds exposed ambient <br />                                                                            atmosphere, reached maximum germination <br />                                                                            2 4 d. Germination percentages normal <br />                                                                            atmosphere lower seeds C24  <br />                                                                            ground plants (63%280%; Fig. 5A) seeds <br />                                                                            Col-0 background plants (92%297%; Fig. <br />                                                                            5A), appears feature C24 germination <br />                                                                            unsupplemented agarose, seeds  <br />  <br />  <br />  <br /> Figure 4. Survival exposure 0.1% oxygen. Plants T-DNA <br /> insertions ANAC102 gene (KO-1, KO-2, Col-0  <br /> ground) plants overexpressing ANAC102 gene (OX-1, OX-2, C24 background) respective parental ecotypes subjected low-oxygen treatments.  Plants treated 0.1% <br /> oxygen dark 3 d. Survival percentages averages experiments (6SE; n 5 15). Data points different lowercase letters <br /> denote lines survival percentages significantly different determined ANOVA Tukey&#226;&#8364;&#8482;s honestly significant <br /> difference test (P , 0.05). B, Plants treated 1 d 5% <br /> oxygen, followed 3 d 0.1% oxygen. Survival percentages averages experiments (6SE; n 5 9). <br />  <br />  <br /> ment, dark 3 d scored survival, <br /> shoot weight, root weight 1 week removal stress condition. Survival counts variable <br /> (Supplemental Fig. S3), significant differences survival weight measures KO lines OX lines respective wild types experiments 5% oxygen pretreat- <br /> ment. expected,  significant <br /> difference survival Col-0 ecotype lines C24 ecotype lines experiments 5% oxygen pretreatment used (Fig. 4; accumulated                       Figure 5. Effects germination various abiotic stresses. Seeds analysis deviance, 5 degrees freedom [df]; devi-                     scored germination intervals 1 week. Data represent average <br /> ance ratio 5 6.77; approximate F probability , 0.001).                     numbers germinated seeds replicates containing 20 50 <br /> Alteration ANAC102 expression did appear                         seeds (6SE; n 5 3). Black symbols represent data untreated effect recovery growth plants surviv-                    seeds, white symbols represent treated seeds. Data points                                                                            different lowercase letters denote lines germination percentages <br /> ing low-oxygen treatment. Plants survived                                                                            significantly different  differences lines <br /> low-oxygen stress measured root shoot                         treatment shown) determined ANOVA Tukey&#226;&#8364;&#8482;s <br /> tissue fresh weights. difference root shoot                       honestly significant difference test (P , 0.05).  Seeds treated weight observed OX KO lines                        0.1% oxygen 6 d. Germination scored removal ambient wild-type counterparts following 0.1% oxygen                         oxygen levels. B, Seeds sown medium containing 200 mM NaCl. C, <br /> treatment 5% oxygen pretreatment                           Seeds sown medium containing 5% (w/v) mannitol. D, Seeds sown <br /> (Supplemental Table S2).                                                   medium containing 15 mM ABA. <br />  <br /> 1730                                                                                                             Plant Physiol. Vol. 149, 2009 <br />                                                                      ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br /> germinate media. Modification ANAC102               determine decreased germination <br /> expression appeared effect seed germi-         observed low oxygen KO lines nation absence stress, signif-     specific low-oxygen treatments generalized <br /> icant difference germination ANAC102 KO           stress response, subjected Arabidopsis wild-type OX seeds respective wild types            mutant lines stresses germination. <br /> normal conditions. Following removal 0.1% oxy-          Salt osmotic stresses chosen exper- <br /> gen, OX-1, OX-2, wild type showed              iments, public data available Genevestigator <br /> reduction germination percentage, signifi-        Web site indicated ANAC102 expression differences response lines (Fig. 5),     inducible salt osmotic stresses (Zimmermann marked difference                et al., 2004). role inhibiting germination ANAC102 knockout lines wild type. Following          Arabidopsis, tested ANAC102 mutant lines low-oxygen treatment, wild-type germination          differences ABA sensitivity germination. <br /> centages reduced 65.4% 80.0%          germination KO lines transiently untreated control, ANAC102 knockout            lower wild type subjected 200 mM <br /> lines KO-1 KO-2 germination percentages              NaCl, 5% mannitol 15 mM ABA appeared reduced 26% 38% untreated            stronger effect KO lines compared controls (Fig. 5A). Germination percentages KO-1,         wild type (Fig. 5, B&#226;&#8364;&#8220;D). results suggest KO-2, wild type untreated 0.1% oxygen-           decreases germination observed KO lines treated seeds 7 d stress ANOVA        likely specific low-oxygen stress. highly significant treatment line differ- <br /> ences, highly significant interaction         ANAC102 Knockout Lines Changes Global <br /> treatment line (treatment, df 5 1, F 5 140.0662,         Gene Expression 0.1% Oxygen-Treated Seeds <br /> P 5 9 3 10211; line, df 5 2, F 5 13.4125, P 5 2 3 1024; <br /> treatment 3 line, df 5 2, F 5 9.25, P 5 0.001; Fig. 5A).        Affymetrix ATH1 microarrays used interro- <br />    determine 0.1% oxygen treatment            gate RNA samples derived wild-type (Col-0) killed imposed secondary dormancy              KO-1 lines assess impact global gene expres- <br />  seeds viable nondormant remain          sion loss ANAC102 function. comparisons <br /> viable refuse germinate) seeds      using 3-week-old plants ambient germinated, ungerminated seeds       atmosphere 0.1% oxygen 4 h, genes assay placed 4&#194;&#176;C 1 week (486 seeds          ANAC102 itself  decreased 17- 50-fold; <br /> total). small proportion (2%) ungermi-         Table  significantly different <br /> nated seeds germinated following cold treatment,         expression (greater 1.5-fold change indicating secondary dormancy            KO-1 wild type, adjusted P value induced seeds (Supplemental Table S3).           #0.05). contrast, KO-1 wild-type imbibed set ungerminated seeds separate exper-       seeds subjected 6 d 0.1% oxygen iment stained tetrazolium salts (510 seeds         assayed differences gene expression, 94 genes <br /> total); 95% seeds turned pink, indicating      highly expressed 113 respiration occurring (Supplemental Table S3).           lowly expressed KO-1 wild type <br /> Vital staining propidium iodide fluorescein         (greater 1.5-fold change adjusted P value diacetate performed set low-oxygen-        ,0.05; Supplemental Table S4). case treated, ungerminated seeds, consisting unger-       ANAC102 OX lines, large proportion genes <br /> minated seeds KO-1, KO-2, wild          altered expression ANAC102 KO seeds <br /> type. Endosperm tissues alive, cases     previously reported responsive low <br /> save  seeds KO-1, mixture live dead        oxygen; 29 94 regulated genes 58 cells seen embryo (Fig. 2F). remaining       113 regulated genes listed having <br /> KO-1 embryo appeared completely viable.                significant expression changes following low oxygen <br />    examined germination signals           microarray studies (Klok et al., 2002; blocked seeds using ATGA3OX2                 Branco-Price et al., 2005; Liu et al., 2005; Loreti et al., <br /> (At1g80340) expression marker germination.          2005). little overlap genes Arabidopsis, biosynthesis gibberellins        altered expression ANAC102 OX lines air <br /> crucial step initiating germination (Koornneef        genes altered expression ANAC102 KO Vanderveen, 1980; Nambara et al., 1991).          seeds low oxygen, genes responsive final steps production active gibberellin   instances (Table . <br /> catalyzed ATGA3OX2, expression                   Promoters genes altered expression KO-1 <br /> creases germinating seeds (Yamaguchi et al., 1998).       seeds low oxygen analyzed using QRT-PCR assays ATGA3OX2 expression                Athena visualization tool (O&#226;&#8364;&#8482;Connor et al., 2005). wild-type KO-1 lines treated 0.1% oxygen            analysis identified following related <br /> showed significant reduction ATGA3OX2 ex-              motifs  share core sequence ACGTG): ABRE- <br /> pression KO-1 lines (1.2-fold induction KO-1       like-binding site motif (Shinozaki Yamaguchi- <br /> line; t test; tailed P 5 0.17).                          Shinozaki, 2000), ABRE-binding site motif (Choi <br /> Plant Physiol. Vol. 149, 2009                                                                                      1731 <br />  Christianson et al. <br />  <br />  <br /> et al., 2000), ABF-binding site motif, G-box motif    ANAC102 KO-1 regulated regulated <br /> CACGTG (Menkens et al., 1995), box motif (Ha            genes, respectively, contain general NAC-binding  1988), GA regulation motif (Ogawa           domain promoters (Table II; Supplemental Table <br /> et al., 2003) 60 promoters 94 ANAC102 KO-1          S4). cis-element analysis tool POBO confirmed regulated genes. Analysis promoters          strict general NAC domain binding se- <br /> 113 genes regulated ANAC102 KO-1 seeds                quences overrepresented promoter sequences <br /> identified 39 DREB1A/CBF3 DRE core                P , 0.0001. <br /> motif stress-inducible genes (Maruyama             genes, including ANAC102, identified differ- <br /> et al., 2004). Searching promoter sequences Toucan       entially regulated wild-type KO-1 seeds <br /> (Aerts et al., 2003, 2005) revealed 38.3% 38.9%   following 6 d exposure 0.1% oxygen, assayed ANAC102 KO-1 regulated regulated              QRT-PCR determine relative expression <br /> genes, respectively, contain strict NAC-binding           levels wild-type, KO-1, OX-1 lines 4 d domain promoters 89.4% 82.3%              0.1% oxygen 6 d (Fig. 6). genes tested <br />  <br />  <br /> Figure 6. Relative expression selected genes seeds exposed 0.1% oxygen 4 6 d. Seeds line imbibed, cold stratified, placed 0.1% oxygen atmosphere 4 6 d. Expression ratios selected genes given <br /> relative expression levels untreated wild- <br /> type (Col-0 Col-0 KO-1; C24 C24 OX-1) imbibed seeds. Bars represent expression <br /> ratios Col-0 (black columns), KO-1 (dotted <br /> columns), C24 (shaded columns), OX-1 (di- <br /> agonal lined columns; 6SE; n 5 9). Data points different letters denote lines relative <br /> expression ratios significantly different determined way ANOVA (line 3 <br /> time) Tukey&#226;&#8364;&#8482;s honestly significant difference <br /> test (P , 0.05). Uppercase letters differentiate Col-0 KO-1 different times, lowercase letters differentiate C24 OX-1 different times. comparisons lines Col-0  <br /> ground lines C24 background. <br />  <br />  <br />  <br />  <br /> 1732                                                                                       Plant Physiol. Vol. 149, 2009 <br />                                                                     ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br /> showed greater response low oxygen 6 d        possibility ANAC102 act bifunctionally 0.1% oxygen compared 4 d. SUS1, key low-          activator repressor transcription. oxygen-responsive gene, showed induction low-            type bifunctionality observed oxygen conditions time points tested,      NFYA5 transcription factor, overexpression induction significantly lessened KO-1 seeds.     affected abundance mRNA number gene (AOX; At3g22370) sig-         genes containing NFYA5 recognition site CCAAT, <br /> nificant difference expression KO-1          inducing repressing (Li et al., 2008). <br /> Col-0 4 d 0.1% oxygen (Fig. 6). Differences     Overrepresentation WRKY-binding motif expression KO-1 Col-0            observed promoters genes  <br /> apparent genes 6 d 0.1% oxygen (Fig. 6).   duced repressed overexpression WRKY70 <br /> Overexpression ANAC102 significant            (Kankainen Holm, 2004). thirds genes <br /> impact gene expression 0.1% oxygen (Fig.           induced repressed ANAC102-overexpressing <br /> 6). Surprisingly, overexpression ANAC102           lines previously identified low- <br /> similar impact gene expression reduced expres-        oxygen responsive (Supplemental Table S1), including <br /> sion ANAC102;  gene expression         ADH1 SUS1, known vital decreased KO-1 compared Col-0,       adult plant tolerance low oxygen (Ellis et al., 1999; <br /> decreased OX-1 compared C24.                     Bieniawska et al., 2007). contrast, ANAC102 KO <br />                                                             lines showed transcriptional differences                                                             wild type normal low-oxygen conditions. <br /> DISCUSSION                                                  ANAC102 OX KO lines showed dif- <br />                                                             ferences growth survival adult plants <br />    ANAC102 required normal growth            subjected low-oxygen stress. ANAC102 <br /> development, primary role Arabidopsis         OX microarray results normal conditions, likely respond stress.        QRT-PCR results low-oxygen conditions sug- <br /> independent ANAC102 knockout lines showed               gest ANAC102 role modifying transcrip- <br /> apparent phenotype nonstressed conditions,            tional responses low oxygen, survival assays <br /> including global changes gene expression.             ANAC102-mediated responses suf- <br /> Knockouts stress-inducible NAC genes ATAF2           ficient protect plants  case ADH1 <br /> (Delessert et al., 2005), ATAF1 (Lu et al., 2007),      overexpression, rate limiting survival <br /> AtNAC2  et al., 2005) similarly showed pheno-         low oxygen (Baxter-Burrell et al., 2002; Ismond <br /> type normal growth conditions.                        et al., 2003). <br />    ANAC102 expression increases early following           ANAC102 KO microarray survival assay imposition low oxygen levels. rapid           results demonstrate ANAC102 function sponse ANAC102 low oxygen coupled            required transcriptional phenotypic response <br /> ability respond low oxygen presence            low oxygen adult plants.  ANAC102 KO cycloheximide, blocks protein synthesis,           lines impaired germination 0.1% oxygen <br /> suggest ANAC102 responds directly low-              treatment, comparisons wild-type oxygen stress does require induction            KO-1 seeds showed expression differences 207 upstream low-oxygen-responsive transcription            genes. Loss ANAC102 did alter germination factors. ANAC102 classed unstable            unstressed seeds, effect germination <br /> transcript half-life 60 min       observed salt, osmotic, ABA stress. normal conditions (Gutierrez et al., 2002). increase    indicates ANAC102 essential ANAC102 transcript levels low oxygen           adult tolerance low oxygen, important partially increase transcript stability     tolerance low-oxygen levels germination. increase transcription,      possible explanation difference adult <br /> ANAC102 promoter capable driving increase       seed low-oxygen phenotypes redundancy GUS transcript response low oxygen (Fig. 2I).         ANAC102 gene function. total 23 NAC <br />    contrast lines overexpressing ATAF2 (Delessert     domain transcription factors possess significantly <br /> et al., 2005), AtNAC2  et al., 2005), ANAC055         altered expression low oxygen, (Tran et al., 2004), overexpression ANAC102          high sequence similarity ANAC102, particularly <br /> little effect plant phenotype mild          ANAC002 (ATAF1) ANAC032 (Ooka et al., 2003). <br /> yellowing leaves. Overexpression ANAC102          adult plants, ANAC102 highly expressed resulted altered expression 211 genes           lateral root cap (Fig. 2; Supplemental Fig. S4), normal conditions. (96.5%) genes           ANAC002 ANAC032 (Supplemental Fig. S4). <br /> altered expression OX-1 contained general            ANAC102 expressed relatively high levels <br /> consensus NAC-binding site, indicating           radicle;  ANAC002 ANAC032 genes targets ANAC102 binding.              strongly expressed tissue (Supplemental <br /> genes regulated regulated ANAC102-           Fig. S4). Differences tissue-specific expression  <br /> overexpressing lines high frequencies NAC        tween highly similar genes account domain consensus-binding sequence, suggesting           difference adult survival/germination rates Plant Physiol. Vol. 149, 2009                                                                                     1733 <br />  Christianson et al. <br />  <br />  <br /> low-oxygen conditions lack transcriptome         gation, adventitious root growth species results observed ANAC102 mutant lines adult       maize rice  et al., 1994; Rzewuski Sauter, <br /> tissue.                                                     2008). possible decreased seed viability <br />    little known Arabidopsis seeds         phenotype observed ANAC102 KO seeds respond low oxygen, unclear disrup-        result changes combination tion ANAC102 function compromises seed tolerance         low-oxygen response mechanisms. Decreased SUS1 low oxygen. thirds genes regulated      expression lead decreased throughput stressed ANAC102 KO seeds core motif ABA-         glycolysis pathway lower energy levels responsive elements promoters, ABA         low-oxygen-stressed embryos. decreased ethylene <br /> response gene ABI4 shows lower wild-type ex-           sensitivity brought decreased ETR2 expres- <br /> pression KO lines following exposure low oxygen       sion result weaker ethylene-mediated low- <br /> (Table  Fig. 6), implying role ABA           oxygen response seeds. <br /> ANAC102-mediated response 0.1% oxygen seeds.             set genes identified differentially expressed <br /> ABA oxygen levels interact regulate dormancy         KO wild-type seeds single low- barley, glumella limits oxygen diffusion       oxygen time point microarray assayed barley embryo, resulting increased           OX KO lines using QRT-PCR time points <br /> sensitivity ABA (Benech-Arnold et al., 2006).        low-oxygen stress. genes   germination experiments media supple-           assayed), lower levels expression mented ABA, KO OX lines showed             observed OX KO lines change wild type ABA sensitivity.           wild-type counterparts 6 d exposure low <br /> Furthermore, Arabidopsis seeds used work        oxygen, raising possibility excess ANAC102 nondormant, low oxygen did induce            expression somewhat detrimental seed <br /> secondary dormancy, seeds failed germinate       responses low oxygen. differences low-oxygen treatment did germinate        statistically significant, OX <br /> dormancy-breaking cold treatment.                           lines did lower germination rates wild <br />    Vital staining seeds failed germinate         type following low-oxygen stress (Fig. 5A). <br /> indicated portions embryo            increased expression putative low-oxygen- <br /> killed low-oxygen stress. damage            responsive genes ANAC102 overexpressors sufficient prevent radicle      reduced germination rates ANAC102 knockout lines <br /> able break seed coat. alternative         response low-oxygen treatment suggest hypothesis mobilization energy reserves       ANAC102 positively regulates response low <br /> affected ANAC102 knockout lines way         oxygen, lack global gene expression leave seed little energy able      change adult plant stage ANAC102 knockouts <br /> germinate following return normal oxygen concen-         lack significant adult plant survival differ- <br /> trations. Interestingly, SUS1, Suc synthase gene,         ences ANAC102 OX KO lines shows lower expression ANAC102 KO-1 seeds           wild type suggest role ANAC102 wild-type seeds (Table  Fig. 6). SUS1 induced       functionally redundant adult plant stage. low-oxygen conditions, responsible pro-        features direct contrast mode viding Glc units glycolysis, double knockouts    action ATAF1 ATAF2, negatively <br /> SUS1 closely related ASUS4 reduced        regulate responses drought wounding, respec- <br /> root growth following flooding (Bieniawska et al.,          tively, loss function 2007). alternative oxidase gene AOX1            genes results readily observable phenotype highly expressed ANAC102 KO-1 seeds low            stress conditions (Delessert et al., 2005; Lu et al., <br /> oxygen compared wild type. AOX1             2007). proposed possible sensor oxygen levels           ANAC102 important Arabidopsis response <br /> (Rhoads Subbaiah, 2007) shown           low-oxygen levels germination. work expression regulated anoxia reoxygen-       illustrates responses environmental stresses, ation barley roots rice seedlings,      case low oxygen, dependent developmental protecting oxidative damage overreduc-        stage mechanisms tolerance differ- <br /> tion mitochondrial electron transport chain (Szal    ent different stages. ANAC102 required toler- <br /> et al., 2003; Millar et al., 2004). ethylene receptor   ance low oxygen seed stage gene ETR2 shows reduced expression ANAC102          early adult plant stage. transcriptional responses <br /> KO-1 seeds, relative wild type, low oxy-       low oxygen differ developmental <br /> gen. ETR2 induced low-oxygen conditions,        stages, KO lines genes differences ACC synthase ACC oxidase genes (Klok et al.,        transcript levels adult plant microarrays 207 <br /> 2002; Branco-Price et al., 2005; Liu et al., 2005; Loreti   genes significant expression differences seeds. <br /> et al., 2005; Peng et al., 2005). Ethylene production    end developmental scale, stimulated wide variety plant species subjected     recently reported 12-week-old Arabidopsis low oxygen responsible triggering         plants form aerenchyma response waterlogging <br /> responses aerenchyma formation, stem elon-          stress (Muehlenbock et al., 2007). response 1734                                                                                       Plant Physiol. Vol. 149, 2009 <br />                                                                                                ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br /> waterlogging previously thought occur                                 data. model assessed variation Arabidopsis line differ- <br />                                                                                     ences, blocked variation experiments. Predicted probabilities Arabidopsis, prior research                                    survival compared pair lines using LSD ducted younger, rosette stage plants form                            comparison significance level 0.33% (5% Bonferroni correction). <br /> aerenchyma. Taken  findings highlight different responses stress different develop- <br /> mental growth stages.                                                               QRT-PCR <br />                                                                                        QRT-PCRs performed triplicate Corbett Rotor Gene <br />                                                                                     (Corbett Life Sciences) Applied Biosystems 7900HT (Applied Biosys- <br />                                                                                     tems)  Product detected fluorescence incorporated SYBR <br /> MATERIALS METHODS <br />                                                                                     Green. data normalized expression At4g26410, Plant Material                                                                      identified highly stable broad range conditions <br />                                                                                     (Czechowski et al., 2005) change expression     Arabidopsis (Arabidopsis thaliana) lines carrying T-DNA insertions           response low oxygen microarray experiments conducted. Primer <br /> ANAC102 (At5g63790; SALK_030702 SALK_094437) identified using              sequences listed Supplemental Table S5. cases, relative expression SIGnAL Web site (http://signal.salk.edu). insertion lines          levels calculated using DDCt method Pfaffl et al. (2002). Expression <br /> obtained Arabidopsis Biological Resource Center (Alonso, 2003).            levels various experiments analyzed using tailed t test assuming <br /> Insertions verified homozygote lines selected using PCR according          equal variances identify statistically significant changes expression levels. protocols provided SIGnAL Web site. KO-1 derivative SALK_030702, KO-2 derivative SALK_094437. insertions second exon ANAC102, segregated single-locus              Low-Oxygen Treatments Arabidopsis Seedlings <br /> insertions. Ecotype Col-0 used wild type comparisons         Seeds insertion mutant lines. C24 susceptible low-oxygen stress Col-0, ANAC102 coding sequence control cauliflower           Low-oxygen treatments microarray QRT-PCR experiments mosaic virus 35S promoter transformed ecotype C24 make easier        carried manner low-oxygen survival assays. Plants detect increase low-oxygen tolerance. independent, highly             grown solid MS medium 3 weeks,  leaf <br /> expressing lines  assayed QRT-PCR), designated OX-1 OX-2,            stage, transferred liquid medium 1 d prior stress. low- <br /> chosen use experiments. Ecotype C24 used wild type       oxygen treatments, plants place 3.5-L anaerobic chambers (Oxoid) comparisons ANAC102 overexpression lines. promoterTGUS               purged 0.1% oxygen/balance nitrogen <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-6' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span>gas</span> mixture 20 min, fusion line created amplifying 1,611 bp upstream ANAC102              plants left 0.1% oxygen atmosphere dark. transcription start site cloning amplicon BamHI/HindIII-digested       cycloheximide treatment, 3-week-old plants moved liquid medium <br /> pBI101, binary plant transformation vector, transformed C24       containing 10 mM cycloheximide 1 h, medium refreshed new <br /> floral dip method.                                                                  medium plus 10 mM cycloheximide, plants low-oxygen treated 4 <br />     plants grown Murashige Skoog (MS) medium 3% Suc            h. treatment, plants flash frozen ground liquid nitrogen. 0.8% agar. seed germination experiments, seeds plated        RNA extracted using Trizol buffer (Invitrogen) following manufac- <br /> 0.6% agarose water half-strength MS medium. Plants seeds           turer&#226;&#8364;&#8482;s instructions. seed microarray real-time PCR experiments, <br /> experiments kept growth room 22&#194;&#176;C day/night period           seeds plated 0.6% agarose water, allowed imbibe approx- <br /> 16/8 h fluorescent lighting levels approximately 75  gene             imately 4 h, cold stratified 4&#194;&#176;C overnight. Following stratification, <br /> expression low-oxygen assays, plants transferred liquid MS         plated seeds allowed equilibrate room temperature, placed 3% Suc. obtain seeds germination experiments, plants grown           anaerobic chambers purged 0.1% oxygen gas mixture 20 <br /> soil 22&#194;&#176;C 16/8-h day/night cycle, lines tested grown     min. Seeds left 0.1% oxygen atmosphere 6 d. Following time location minimize environmental influences         treatment, seeds frozen ground liquid nitrogen. RNA seed development subsequent germination.                                     extracted using hot borate method (Cadman et al., 2006). <br />  <br />  <br /> Low-Oxygen Survival Assay                                                           Microarray Analysis Gene Expression <br />     Low-oxygen survival assays performed outlined previously (Ellis            plant seed RNA sent Australian Genome Research <br /> et al., 1999), argon used flush anaerobic chambers       Facility labeling hybridization Affymetrix Arabidopsis ATH1 plants subjected anoxia mild hypoxia (5%              genome arrays (22,500 probes). unstressed microarray experiments, oxygen, balance nitrogen) severe hypoxia (0.1% oxygen, balance               biological replicates used wild type, KO-1, OX-1 nitrogen). Briefly, plants grown  leaf stage solid       replicate used KO-2 OX-2. differences medium transferred liquid medium 1 d prior treatment.            KO-1 KO-2 OX-1 OX-2, data presented Immediately prior treatment, plants transferred liquid medium       KO-1 OX-1 comparisons  3 weeks old, plants line sparged 5% oxygen, pretreated plants, 0.1%              replicate plate bulked, flash frozen liquid nitrogen, ground. <br /> oxygen, nonpretreated plants. Plants placed 3.5-L anaerobic        RNA extracted using Trizol buffer (Invitrogen) following manufac- <br /> chambers (Oxoid) purged 5% 0.1% oxygen flow rate         turer&#226;&#8364;&#8482;s instructions. low-oxygen stress microarrays plant seed <br /> approximately 10 L min21 20 min. Pretreated plants left 5%              stages, biological replicates wild type KO-1 used. <br /> oxygen, dark gentle shaking, 24 h purged 0.1%         experiments seeds, seeds plated 0.6% agarose water oxygen left 3 d. Nonpretreated plants purged            low-oxygen treated described  Following treatment, seeds 0.1% oxygen left dark gentle shaking 3 d. Nontreated plants     frozen ground liquid nitrogen. RNA extracted using hot borate placed anaerobic chambers left exposed ambient atmo-             method (Cadman et al., 2006). <br /> sphere left dark gentle shaking 3 d. Following treatment,          Resulting signal data analyzed using limma Bioconductor pack- <br /> plants given fresh liquid medium allowed recover orbital          age R. Array data normalized using EXPRESSO function (Gautier <br /> shaker normal growth cabinet conditions 1 week survival scoring       et al., 2005). robust multichip average calculated using quantile weight measurements.                                                            normalization, background correction, median polish method     Survival data independent experiments pooled ana-            recommended Bolstad et al. (2003) Irizarry et al. (2003). <br /> lyzed differences lines response low oxygen. low- <br /> oxygen survival data approximated assumption normality. Consequently, data analyzed binomial general-          Promoter Analysis <br /> ized linear model logit link. dispersion factor initially fixed 1, analysis flagged large number large standardized                  Promoter regions selected genes screened common motifs. <br /> residuals, data reanalyzed, allowing dispersion estimated     different Web-based analysis tools used: Athena (O&#226;&#8364;&#8482;Connor et al., <br />  <br /> Plant Physiol. Vol. 149, 2009                                                                                                                                    1735 <br />  Christianson et al. <br />  <br />  <br /> 2005), Toucan (Aerts et al., 2003, 2005), POBO (Kankainen Holm,                  Supplemental Table S1. Genes identified significant changes 2004). case, 1,000-bp upstream regions selected genes                  expression OX-1 adult plants. <br /> used analysis. POBO analysis (Kankainen Holm, 2004), <br />                                                                                          Supplemental Table S2. Average root shoot weights Arabidopsis <br /> Arabidopsis_thaliana_clean used background organism sequence set, <br />                                                                                            lines following 2-week recovery low-oxygen stress treatments. number sequences pick set equal number input <br /> sequences, number samples generate sequence length               Supplemental Table S3. Germination low-oxygen-treated seeds fol- <br /> set 1,000.                                                                              lowing cold treatment seed mortality determined tetrazolium <br />                                                                                            staining. <br />                                                                                          Supplemental Table S4. Genes identified significant changes Germination Assays <br />                                                                                            expression low-oxygen-treated KO-1 seeds. <br />     Seeds line tested harvested time             Supplemental Table S5. Sequences primers used. plants grown environmentally controlled growth room imbibed 1.5-mL microfuge tubes approximately 2 h room tempera- <br /> ture. 50 seeds line plated approximately 0.5 <br /> cm apart 0.6% agarose placed 4&#194;&#176;C 1 week. low-oxygen <br /> treatments, seeds placed 3.5-L anaerobic chambers purged           ACKNOWLEDGMENTS 0.1% oxygen flow rate 10 L min21 20 min exposed                                                                                         thank Jun Yang technical assistance Donna Bond providing <br /> ambient atmosphere. Seeds left exposed ambient air scored                                                                                       cDNA samples. <br /> germination daily. Seeds treated 0.1% oxygen remained sealed <br /> anaerobic chambers 0.1% oxygen 6 d, time seed                  Received October 30, 2008; accepted January 23, 2009; published January 28, <br /> germination occurred. Following removal 0.1% oxygen, seed germina-               2009. <br /> tion scored daily. stress treatments, seeds plated half-strength MS medium containing 200 mM NaCl, 5% (w/v) mannitol, 15 <br /> mM ABA. cases, germination scored intervals 1 week. cases, seeds counted germinated radicle penetrated seed          LITERATURE CITED <br /> coat. Data proportion seeds germinated 7 d replicate <br />                                                                                       Aerts S, Thijs G, Coessens B, Staes M, Moreau Y, Moor BD (2003) Toucan: <br /> experiments analyzed way ANOVA blocked replicate <br />                                                                                          deciphering cis-regulatory logic coregulated genes. Nucleic Acids <br /> experiments detect significant differences lines. Arabidopsis <br />                                                                                          Res 31: 1753&#226;&#8364;&#8220;1764 <br /> lines grouped basis germination percentage using Tukey&#226;&#8364;&#8482;s <br />                                                                                       Aerts S, Van Loo P, Thijs G, Mayer H, Martin R, Moreau Y, Moor B <br /> honestly significant difference. <br />                                                                                          (2005) TOUCAN 2: inclusive open source workbench regu- <br />                                                                                          latory sequence analysis. Nucleic Acids Res 33: W393&#226;&#8364;&#8220;W396 <br />                                                                                       Aida M, Ishida T, Fukaki H, Fujisawa H, Tasaka M (1997) Genes involved <br /> Seed Staining                                                                            organ separation Arabidopsis: analysis cup-shaped <br />                                                                                          cotyledon mutant. Plant Cell 9: 841&#226;&#8364;&#8220;857 <br />     GUS staining, plant material stained immersed solution <br />                                                                                       Al-Ani  Bruzau F, Raymond P, Saint-Ges V, Leblanc JM, Pradet (1985) <br /> composed 3 mM 5-bromo-4-chloro-3-indolyl-b-glucuronic acid, 10 mM <br />                                                                                          Germination, respiration, adenylate energy charge seeds EDTA, 100 mM NaPO4 buffer, pH 7.2, 0.3% Triton X, 500 mM ferricyanide, <br />                                                                                          various oxygen partial pressures. Plant Physiol 79: 885&#226;&#8364;&#8220;890 <br /> 500 mM ferrocyanide, 10% methanol, vacuum infiltrated, incubated                                                                                       Alonso JM (2003) Genome-wide insertional mutagenesis Arabidopsis <br /> 37&#194;&#176;C color developed. Following staining, plant material washed                                                                                          thaliana. Science 301: 1849 <br /> distilled water cleared washes 70% ethanol. Ungerminated <br />                                                                                       Bailey-Serres J, Dawe RK (1996) 5# 3# sequences maize adh1 <br /> seeds low-oxygen germination assays assayed viability <br />                                                                                          mRNA required enhanced translation low-oxygen condi- <br /> using vital staining methods, using tetrazolium salts turn                                                                                          tions. Plant Physiol 112: 685&#226;&#8364;&#8220;695 <br /> colorless pink presence cellular respiration (Cottrell, 1947),                                                                                       Baxter-Burrell  Yang ZB, Springer PS, Bailey-Serres J (2002) RopGAP4- utilizing combination propidium iodide, permeate <br />                                                                                          dependent Rop GTPase rheostat control Arabidopsis oxygen depriva- <br /> living cells, fluorescein diacetate,  cleaved cytoplas- <br />                                                                                          tion tolerance. Science 296: 2026&#226;&#8364;&#8220;2028 <br /> mic esterases yield fluorescein (Jones Senft, 1985). tetrazolium <br />                                                                                       Benech-Arnold RL, Gualano N, Leymarie J, Come D, Corbineau F (2006) <br /> staining, seeds tested scored 21-gauge needle break                                                                                          Hypoxia interferes ABA metabolism increases ABA sensitivity <br /> seed coat. seeds placed filter paper soaked 1% <br />                                                                                          embryos dormant barley grains. J Exp Bot 57: 1423&#226;&#8364;&#8220;1430 <br /> solution tetrazolium salts incubated dark overnight. vital <br />                                                                                       Bieniawska  Barratt DHP, Garlick AP, Thole V, Kruger NJ, Martin C, <br /> staining propidium iodide/fluorescein diacetate, embryos dis- <br />                                                                                          Zrenner R, Smith (2007) Analysis sucrose synthase gene <br /> sected seeds embryos seed coats stained 10 mg <br />                                                                                          family Arabidopsis. Plant J 49: 810&#226;&#8364;&#8220;828 <br /> mL21 propidium iodide 0.5 mg mL21 fluorescein diacetate 5 10 min <br />                                                                                       Bolstad BM, Irizarry RA, Astrand M, Speed TP (2003) comparison observation fluorescent microscope. <br />                                                                                          normalization methods high density oligonucleotide array data <br />                                                                                          based variance bias. Bioinformatics 19: 185&#226;&#8364;&#8220;193 <br />    Microarray data article submitted public National <br />                                                                                       Branco-Price C, Kawaguchi R, Ferreira RB, Bailey-Serres J (2005) <br /> Center Biotechnology Information Gene Expression Omnibus database <br />                                                                                          Genome-wide analysis transcript abundance translation (GEO accession  GSE14420). <br />                                                                                          Arabidopsis seedlings subjected oxygen deprivation. Ann Bot (Lond) <br />                                                                                          96: 647&#226;&#8364;&#8220;660 <br /> Supplemental Data                                                                     Cadman CSC, Toorop PE, Hilhorst HWM, Finch-Savage (2006) Gene <br />                                                                                          expression profiles Arabidopsis Cvi seeds dormancy cycling <br />    following materials available online version article.          indicate common underlying dormancy control mechanism. Plant J 46: <br />    Supplemental Figure S1. Expression levels ANAC102 knockout                  805&#226;&#8364;&#8220;822 <br />      overexpressing lines.                                                            Cannell RQ, Belford RK (1980) Effects waterlogging different stages <br />                                                                                          development growth yield winter oilseed rape (Brassica <br />    Supplemental Figure S2. Expanded version Figure 3 showing expres-                  napus L). J Sci Food Agric 31: 963&#226;&#8364;&#8220;965 <br />      sion selected genes time course Col-0, C24, KO-1,           Choi H, Hong J, Ha J, Kang J, Kim S (2000) ABFs, family ABA- <br />      OX-1.                                                                           responsive element binding factors. J Biol Chem 275: 1723&#226;&#8364;&#8220;1730 <br />                                                                                       Collaku  Harrison SA (2002) Losses wheat waterlogging. Crop <br />    Supplemental Figure S3. Survival exposure 0.1% oxygen                                                                                          Sci 42: 444&#226;&#8364;&#8220;450 <br />      independent experiments. <br />                                                                                       Cottrell HJ (1947) Tetrazolium salt seed germination indicator. Nature <br />    Supplemental Figure S4. Heat map gene expression 23 low-oxygen-                159: 748 <br />      induced NAC genes.                                                               Czechowski T, Stitt M, Altmann T, Udvardi MK, Scheible WR (2005) <br />  <br /> 1736                                                                                                                          Plant Physiol. Vol. 149, 2009 <br />                                                                                           ANAC102 Required Seed Viability Low Oxygen <br />  <br />  <br />     Genome-wide identification testing superior reference genes         primary root growth germinating rice anoxia. Plant Growth <br />     transcript normalization Arabidopsis. Plant Physiol 139: 5&#226;&#8364;&#8220;17               Regul 35: 181&#226;&#8364;&#8220;185 <br /> Delessert C, Kazan K, Wilson IW, Van Der Straeten D, Manners J, Dennis          Kelley PM, Freeling M (1984a) Anaerobic expression maize glucose <br />     ES, Dolferus R (2005) transcription factor ATAF2 represses             phosphate isomerase  J Biol Chem 259: 673&#226;&#8364;&#8220;677 <br />     expression pathogenesis-related genes Arabidopsis. Plant J 43:        Kelley PM, Freeling M (1984b) Anaerobic expression maize fructose-1,6- <br />     745&#226;&#8364;&#8220;757                                                                        diphosphate aldolase. J Biol Chem 259: 4180&#226;&#8364;&#8220;4183 <br /> Dennis ES, Dolferus R, Ellis M, Rahman M, Wu Y, Hoeren FU, Grover             Kim SG, Kim SY, Park CM (2007) membrane-associated NAC tran- <br />     Ismond KP, Good AG, Peacock WJ (2000) Molecular strategies                 scription factor regulates salt-responsive flowering FLOWERING <br />     improving waterlogging tolerance plants. J Exp Bot 51: 89&#226;&#8364;&#8220;97                LOCUS T Arabidopsis. Planta 226: 647&#226;&#8364;&#8220;654 <br /> Despres C, DeLong C, Glaze S, Liu E, Fobert PR (2000) Arabidopsis           Klok EJ, Wilson IW, Wilson D, Chapman SC, Ewing RM, Somerville SC, <br />     NPR1/NIM1 protein enhances DNA binding activity subgroup              Peacock WJ, Dolferus R, Dennis ES (2002) Expression profile analysis <br />     TGA family bZIP transcription factors. Plant Cell 12: 279&#226;&#8364;&#8220;290        low-oxygen response Arabidopsis root cultures. Plant Cell 14: <br /> Dolferus R, Klok EJ, Delessert C, Wilson S, Ismond KP, Good AG,                    2481&#226;&#8364;&#8220;2494 <br />     Peacock WJ, Dennis ES (2003) Enhancing anaerobic response. Ann          Koornneef M, Vanderveen JH (1980) Induction analysis gibberellin <br />     Bot (Lond) 91: 111&#226;&#8364;&#8220;117                                                         sensitive mutants Arabidopsis thaliana (L) Heynh. Theor Appl Genet <br /> Dolferus R, Wolansky M, Carroll R, Miyashita Y, Ismond K, Good                   58: 257&#226;&#8364;&#8220;263 <br />     (2008) Functional analysis lactate dehydrogenase hypoxic          Kursteiner O, Dupuis  Kuhlemeier C (2003) Pyruvate decarboxylase1 <br />     stress Arabidopsis. Funct Plant Biol 35: 131&#226;&#8364;&#8220;140                            gene Arabidopsis required anoxia, environ- <br /> Ellis MH, Dennis ES, Peacock WJ (1999) Arabidopsis roots shoots                mental stresses. Plant Physiol 132: 968&#226;&#8364;&#8220;978 <br />     different mechanisms hypoxic stress tolerance. Plant Physiol       Li WX, Oono Y, Zhu JH, XJ, Wu JM, Iida K, Lu XY, Cui XP, Jin HL, Zhu <br />     119: 57&#226;&#8364;&#8220;64                                                                     JK (2008) Arabidopsis NFYA5 transcription factor regulated <br /> Ellis MH, Millar AA, Llewellyn DJ, Peacock WJ, Dennis ES (2000)                    transcriptionally posttranscriptionally promote drought resis- <br />     Transgenic cotton (Gossypium hirsutum) expressing alcohol dehy-           tance. Plant Cell 20: 2238&#226;&#8364;&#8220;2251 <br />     drogenase shows increased ethanol fermentation increase           Liu FL, Vantoai T, Moy LP, Bock G, Linford LD, Quackenbush J (2005) <br />     tolerance oxygen deficiency. Aust J Plant Physiol 27: 1041&#226;&#8364;&#8220;1050             Global transcription profiling reveals comprehensive insights hyp- <br /> Finch-Savage  Come D, Lynn JR, Corbineau F (2005) Sensitivity                oxic response Arabidopsis. Plant Physiol 137: 1115&#226;&#8364;&#8220;1129 <br />     Brassica oleracea seed germination hypoxia: QTL analysis. Plant Sci    Loreti E, Poggi  Novi G, Alpi  Perata P (2005) genome-wide analysis <br />     169: 753&#226;&#8364;&#8220;759                                                                   effects sucrose gene expression Arabidopsis seedlings <br /> Fukao T, Xu KN, Ronald PC, Bailey-Serres J (2006) variable cluster            anoxia. Plant Physiol 137: 1130&#226;&#8364;&#8220;1138 <br />     ethylene response factor-like genes regulates metabolic develop-        Lu PL, Chen NZ, R, Su  Qi BS, Ren F, Chen J, Wang XC (2007)     mental acclimation responses submergence rice. Plant Cell 18:            novel drought-inducible gene, ATAF1, encodes NAC family protein <br />     2021&#226;&#8364;&#8220;2034                                                                      negatively regulates expression stress-responsive genes Gautier L, Irizarry R, Cope L, Bolstad B (2005) Description affy.               Arabidopsis. Plant Mol Biol 63: 289&#226;&#8364;&#8220;305 <br />     Bioconductor Vignettes. http://bioconductor.org/packages/2.2/bioc/          Maruyama K, Sakuma Y, Kasuga M, Ito Y, Seki M, Goda H, Shimada Y, <br />     vignettes/affy/inst/doc/affy.pdf (April 17, 2008)                              Yoshida S, Shinozaki K, Yamaguchi-Shinozaki K (2004) Identification <br /> Gutierrez RA, Ewing RM, Cherry JM, Green PJ (2002) Identification               cold-inducible downstream genes Arabidopsis DREB1A/CBF3 <br />     unstable transcripts Arabidopsis cDNA microarray analysis: rapid         transcriptional factor using microarray systems. Plant J 38: 982&#226;&#8364;&#8220;993 <br />     decay associated group touch- specific clock-controlled    Mason WK, Pritchard KE, Small DR (1987) Effects early season water- <br />     genes. Proc Natl Acad Sci USA 99: 11513&#226;&#8364;&#8220;11518                                  logging maize growth yield. Aust J Agric Res 38: 27&#226;&#8364;&#8220;35 <br /> Ha SB, G (1988) Identification upstream regulatory elements            Mattana M, Vannini C, Espen L, Bracale M, Genga  Marsoni M, Iriti M, <br />     volved developmental expression Arabidopsis thaliana cab1        Bonazza V, Romagnoli F, Baldoni E, et al (2007) rice Mybleu <br />     gene. Proc Natl Acad Sci USA 85: 8017&#226;&#8364;&#8220;8021                                     transcription factor increases tolerance oxygen deprivation Arabi- <br /> Hart CM, Nagy F, Meins F Jr (1993) 61 bp enhancer element                 dopsis plants. Physiol Plant 131: 106&#226;&#8364;&#8220;121 <br />     tobacco beta -1,3-glucanase N gene interacts regulated     Menkens AE, Schindler U, Cashmore AR (1995) G-box: ubiquitous <br />     nuclear proteins. Plant Mol Biol 21: 121&#226;&#8364;&#8220;131                                   regulatory DNA element plants bound GBF family bZIP CJ, Drew MC, Morgan PW (1994) Induction enzymes associated              proteins. Trends Biochem Sci 20: 506&#226;&#8364;&#8220;510 <br />     lysigenous aerenchyma formation roots Zea mays hypoxia         Millar AH, Trend AE, Heazlewood JL (2004) Changes mitochondrial <br />     nitrogen starvation. Plant Physiol 105: 861&#226;&#8364;&#8220;865                             proteome anoxia air transition rice focus X, Mu R, Cao W, Zhang  Zhang J, Chen S (2005) AtNAC2,                       cytochrome-containing respiratory complexes. J Biol Chem 279: 39471&#226;&#8364;&#8220; <br />     transcription factor downstream ethylene auxin signaling path-          39478 <br />     ways, involved salt stress response lateral root development.     Miyashita Y, Dolferus R, Ismond KP, Good AG (2007) Alanine amino- <br />     Plant J 44: 903&#226;&#8364;&#8220;916                                                            transferase catalyses breakdown alanine hypoxia Arabi- Y, Gan S (2001) Identical promoter elements involved regulation          dopsis thaliana. Plant J 49: 1108&#226;&#8364;&#8220;1121 <br />     OPR1 gene senescence jasmonic acid Arabidopsis. Plant      Muehlenbock P, Plaszczyca M, Plaszczyca M, Mellerowicz E, Karpinski S <br />     Mol Biol 47: 595&#226;&#8364;&#8220;605                                                           (2007) Lysigenous aerenchyma formation Arabidopsis controlled Hodgson  Chan KY (1982) effect short-term waterlogging            LESION SIMULATING DISEASE1. Plant Cell 19: 3819&#226;&#8364;&#8220;3830 <br />     furrow irrigation cotton cracking grey clay. Aust J Agric Res 33:   Nambara E, Akazawa T, McCourt P (1991) Effects gibberellin <br />     109&#226;&#8364;&#8220;116                                                                        biosynthetic inhibitor uniconazol mutants Arabidopsis. Plant <br /> Irizarry RA, Hobbs B, Collin F, Beazer-Barclay YD, Antonellis KJ, Scherf           Physiol 97: 736&#226;&#8364;&#8220;738 <br />     U, Speed TP (2003) Exploration, normalization, summaries high        O&#226;&#8364;&#8482;Connor TR, Dyreson C, Wyrick JJ (2005) Athena: resource rapid <br />     density oligonucleotide array probe level data. Biostatistics 4: 249&#226;&#8364;&#8220;264       visualization systematic analysis Arabidopsis promoter se- <br /> Ismond KP, Dolferus R, Pauw M, Dennis ES, Good AG (2003)                        quences. Bioinformatics 21: 4411&#226;&#8364;&#8220;4413 <br />     Enhanced low oxygen survival Arabidopsis increased met-          Ogawa M, Hanada  Yamauchi Y, Kuwahara  Kamiya Y, Yamaguchi S <br />     abolic flux fermentative pathway. Plant Physiol 132: 1292&#226;&#8364;&#8220;1302          (2003) Gibberellin biosynthesis response Arabidopsis seed <br /> Jacobs M, Dolferus R, Vandenbossche D (1988) Isolation biochemical             germination. Plant Cell 15: 1591&#226;&#8364;&#8220;1604 <br />     analysis ethyl methanesulfonate induced alcohol dehydrogenase null       Olsen  Ernst HA, Lo Leggio L, Skriver K (2005) DNA-binding <br />     mutants Arabidopsis thaliana (L) Heynh. Biochem Genet 26: 105&#226;&#8364;&#8220;122           specificity molecular functions NAC transcription factors. Plant <br /> Jones KH, Senft JA (1985) improved method determine cell viability           Sci 169: 785&#226;&#8364;&#8220;797 <br />     simultaneous staining fluorescein diacetate propidium iodide.       Ooka H, Satoh K, Doi K, Nagata T, Otomo Y, Murakami K, Matsubara K, <br />     J Histochem Cytochem 33: 77&#226;&#8364;&#8220;79                                                 Osato N, Kawai J, Carninci P, et al (2003) Comprehensive analysis Kankainen M, Holm L (2004) POBO, transcription factor binding site                 NAC family genes Oryza sativa Arabidopsis thaliana. DNA Res 10: <br />     verification bootstrapping. Nucleic Acids Res 32: W222&#226;&#8364;&#8220;W229               239&#226;&#8364;&#8220;247 <br /> Kato-Noguchi H (2001) importance ethanolic fermentation              Peng HP, Lin TY, Wang NN, Shih MC (2005) Differential expression  <br /> Plant Physiol. Vol. 149, 2009                                                                                                                        1737 <br />  Christianson et al. <br />  <br />  <br />    genes encoding 1-aminocyclopropane-1-carboxylate synthase Arabi-          lowland rice: search physiological marker traits. J Plant Physiol 158: <br />    dopsis hypoxia. Plant Mol Biol 58: 15&#226;&#8364;&#8220;25                              883&#226;&#8364;&#8220;889 <br /> Pfaffl MW, Horgan GW, Dempfle L (2002) Relative Expression Software          Souer E, vanHouwelingen  Kloos D, Mol J, Koes R (1996) apical <br />    Tool (REST (c)) group-wise comparison statistical analysis        meristem gene petunia required pattern formation embryos <br />    relative expression results real-time PCR. Nucleic Acids Res 30: e36      flowers expressed meristem primordia boundaries. <br /> Rahman M, Grover  Peacock WJ, Dennis ES, Ellis MH (2001) Effects           Cell 85: 159&#226;&#8364;&#8220;170 <br />    manipulation pyruvate decarboxylase alcohol dehydrogenase          Szal B, Jolivet Y, Hasenfratz-Sauder MP, Dizengremel P, Rychter    levels submergence tolerance rice. Aust J Plant Physiol 28:        (2003) Oxygen concentration regulates alternative oxidase expression <br />    1231&#226;&#8364;&#8220;1241                                                                    barley roots hypoxia post-hypoxia. Physiol Plant 119: <br /> Rhoads DM, Subbaiah CC (2007) Mitochondrial retrograde regulation            494&#226;&#8364;&#8220;502 <br />    plants. Mitochondrion 7: 177&#226;&#8364;&#8220;194                                          Tran LS, Nakashima K, Sakuma Y, Simpson SD, Fujita Y, Maruyama K, <br /> Ricoult C, Cliquet JB, Limami (2005) Stimulation alanine amino <br />                                                                                 Fujita M, Seki M, Shinozaki K, Yamaguchi-Shinozaki K (2004) Isola- <br />    transferase (AlaAT) gene expression alanine accumulation em- <br />                                                                                 tion functiona <br /> </body></html>