<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, Mar. 2009, p. 1722&#226;&#8364;&#8220;1724                                                                                  Vol. 191,  5 <br /> 0021-9193/09/$08.00&#226;&#171;&#185;0 doi:10.1128/JB.01573-08 <br /> Copyright &#194;&#169; 2009, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />           Conditional Essentiality csrA Gene Escherichia coli&#228;&#338;&#164; <br />                                      Johan Timmermans Laurence Van Melderen* <br />   Laboratoire Ge&#194;&#180;ne&#194;&#180;tique et Physiologie Bacte&#194;&#180;rienne, Institut Biologie et decine Mole&#194;&#180;culaires, Faculte&#194;&#180; des Sciences, <br />                 Universite&#194;&#180; Libre Bruxelles, 12 rue des Professeurs Jeener et Brachet, B:6041 Gosselies, Belgium <br />                                             Received 5 November 2008/Accepted 10 December 2008 <br />  <br />              CsrA global posttranscriptional regulator numerous physiological processes, glycogenesis           glycolysis.  csrA gene Escherichia coli essential growth LB synthetic <br />           medium containing glycolytic carbon sources.  csrA necessary growth synthetic medium <br />           containing pyruvate, showing Krebs cycle functional csrA::cat deletion mutant. Deletion           glgCAP operon csrA::cat mutant restored ability grow LB synthetic medium containing <br />           glycolytic carbon sources, showing growth inhibition excess glycogen synthesis. <br />  <br />  <br />    Escherichia coli csrA (carbon storage regulator  gene                   growth defect presented double csrA::kan rpoS mutant <br /> encodes global regulator controlling wide variety biolog-                 associated high level glycogen accumulation (27). <br /> ical processes, glycogen synthesis, glycolysis, mo-                  Acetate sensitivity appears require csrA::kan tility cyclic-di-<span id='am-18' about='obo:IMR_0200292' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00144" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200292" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="GMP" datatype="xsd:string"></span>GMP</span> synthesis biofilm formation                  rpoS mutations single mutants sensitive (2, 7, 14, 19, 21, 22, 26, 28). CsrA regulates gene expression                   acetate did evolve suppressor mutations (27). positively negatively posttranscriptional level                     E. coli csrA gene essential certain growth  modulating translation. binds homodimer spe-                     ditions. attempt construct csrA deletion mutant E. <br /> cific sequence (consensus sequence, ACANGGANGA)                               coli, large series homologous recombination experiments mRNA targets small regulator RNAs (1, 2, 7,                     carried DJ624 strain (MG1655 lacX74 <br /> 18, 23, 26). Binding CsrA leader sequence                       malP::lacIq) (12) replace csrA open reading frame <br /> mRNA target blocks ribosome binding Shine-Dalgarno                        (ORF) chloramphenicol resistance cassette (6). Despite <br /> sequence (2, 7, 26) likely leads rapid degrada-                  attempts various conditions, deletion mutants <br /> tion (18). positive regulator, CsrA enhances translation                    obtained, suggesting csrA essential growth. efficiency stabilizing mRNA targets (28).                                 test hypothesis, various rescue plasmids constructed. <br />    Homologues csrA identified, role                    csrA gene cloned pBAD33 (11) pKK223-3 bacterial species studied  review,                    (4) expression vectors. expression rescue copy reference 15). csrA homologues share regulatory net-                       csrA induced LB medium addition arabinose <br /> works identified E. coli, motility (17)                (0.4%) isopropyl &#226;?&#164;-D-1-thiogalactopyranoside (IPTG) (1 <br /> biofilm formation (8). homologues appear reg-                      mM). constructs checked sequencing, ulate processes, virulence Salmonella enterica,                 ability complement csrA::kan mutation tested. Using <br /> Legionella pneumophila, Helicobacter pylori, Proteus mirabilis,                  plasmids csrA rescue copy, attempts delete csrA Yersinia pseudotuberculosis (3, 9, 13, 17, 24); quorum sens-                 DJ624 successful, indicating high expression lev- <br /> ing Erwinia carotovora Vibrio cholerae (5, 16); oxi-                  els CsrA deleterious conditions. dative stress resistance Campylobacter jejuni (8). <br />                                                                                  bypass problem, csrA gene cloned low-copy- <br />    E. coli, csrA gene identified transposon mu- <br />                                                                                  number pWSK129 vector control lac pro- <br /> tagenesis screen mutants altered glycogen accu- <br />                                                                                  moter (25). DJ624 transformed construct, csrA <br /> mulation. kanamycin transposon insertion site located                                                                                  expression induced addition 0.1 mM IPTG codon 51 csrA coding sequence  encodes protein <br />                                                                                  culture medium. conditions, csrA deletion mu- 61 amino acids). addition increased glycogen accu- <br />                                                                                  tants obtained LB plates containing chloramphenicol <br /> mulation, csrA::kan mutant affected motility, biofilm <br />                                                                                  (20 &#226;?&#174;g/ml) kanamycin (50 &#226;?&#174;g/ml) 37&#194;&#176;C (&#226;&#338;&#172;csrA::cat). formation, glycolysis (2, 19, 21, 22, 28). association                                                                                  replacement csrA ORF chloramphenicol resis- rpoS defective mutation, csrA::kan mutation led                                                                                  tance cassette checked PCR using primers flanking extended lag phase growth acetate (27). Suppressor <br />                                                                                  csrA ORF. &#226;&#338;&#172;csrA::cat mutant strain containing pWSK- <br /> mutations restoring growth double mutant acetate <br />                                                                                  csrA plasmid able grow presence 0.1 mM mapped glg locus (27). indicates                                                                                  IPTG. growth observed LB plates IPTG, <br />                                                                                  showing csrA gene essential growth LB  <br />    * Corresponding author. Mailing address: Laboratoire Ge&#194;&#180;ne&#194;&#180;t-              dium (Fig. 1, upper panels). <br /> ique et Physiologie Bacte&#194;&#180;rienne, Institut Biologie et -                CsrA1-50 <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> partially active. kan transposon <br /> decine Mole&#194;&#180;culaires, Faculte&#194;&#180; des Sciences, Universite&#194;&#180; Libre Brux-          inserted codon 51 csrA gene (21). inser- <br /> elles, 12 rue des Professeurs Jeener et Brachet, B:6041 Gosselies, <br /> Belgium. Phone: 32 2 650 97 78. Fax: 32 2 650 97 70. E-mail: lvmelder <br />                                                                                  tion mutant viable LB medium, leaves possibility <br /> @ulb.ac                                                                       50 amino-terminal amino acids retain    &#228;&#338;&#164; <br />      Published ahead print 19 December 2008.                               partial activity proposed Romeo coworkers (21).  <br />                                                                           1722 <br />  VOL. 191, 2009                                                                                                                    NOTES        1723 <br />  <br />  <br />  <br />  <br />    FIG. 1. csrA gene essential E. coli. &#226;&#338;&#172;csrA::cat mutant <br /> containing pWSK-csrA (upper panels) pKK-csrA1-50 <br /> (lower panels) plasmids grown LB plates containing ap- <br /> propriate antibiotics 0.1 mM IPTG (left panels) IPTG (right panels). <br />  <br />  <br />  <br /> test hypothesis, 150 5&#226;&#172;&#732; nucleotides csrA <br /> coding sequence, followed stop codon, cloned pKK223-3. csrA deletion mutant containing pWSK- <br /> csrA plasmid transformed pKK-csrA1-50 plasmid. displace pWSK-csrA plasmid, used incompati- <br /> bility properties pWSK129 pMLO59  plasmids <br />                                                                             FIG. 2. CsrA-mediated regulation central carbon flux. CsrA pSC101 replication origin different antibiotic resis-          positively regulates glycolysis negatively regulates gluconeogenesis <br /> tance genes; pWSK129 confers resistance kanamycin                  glycogen synthesis. Negative positive regulations target <br /> pMLO59 spectinomycin). &#226;&#338;&#172;csrA::cat pWSK-csrA pKK-                   genes indicated blocked arrows (        ) arrows, respectively. <br /> csrA1-50 strain transformed pMLO59 vector,             Adapted reference 22. <br /> resulting transformants plated LB plates containing <br /> spectinomycin (100 &#226;?&#174;g/ml), chloramphenicol (20 &#226;?&#174;g/ml), 0.1 mM IPTG induce expression csrA1-50 polypep-             amino acids glycolytic carbon sources. &#226;&#338;&#172;csrA::cat mu- <br /> tide. Transformants checked kanamycin sensitivity                tant unable grow glucose fructose-6-phos- <br /> (loss pWSK-csrA plasmid). &#226;&#338;&#172;csrA::cat mutant                phate sole carbon sources (Fig. 3B C). Interestingly, taining pKK-csrA1-50 plasmid able form colonies             able grow pyruvate, carbon source LB plates containing 0.1 mM IPTG plates                junction glycolysis Krebs cycle (Fig. 3D). IPTG (Fig. 1, lower panels), showing CsrA1-50 peptide            fructose-6-phosphate plates, small colonies appeared functional. IPTG plates, colonies abnormal               16 h 37&#194;&#176;C  small seen Fig. 3C). data <br /> phenotype (forming heterogeneous &#226;&#8364;&#339;sticky&#226;&#8364;? colonies).              indicate early steps glycolysis fully functional <br /> molecular mechanisms underlying phenotype                     &#226;&#338;&#172;csrA::cat mutant. glucose fructose-6-phos- <br /> known. phenotype does appear dependent                  phate sole carbon sources, carbon flux essen- CsrA1-50  observed higher            tially directed glycogen synthesis, impairing glycol- <br /> IPTG concentrations (data shown). carboxy-termi-               ysis Krebs cycle. tested deletion nal CsrA (amino acids 51 54) involved RNA                glgCAP operon, involved steps binding (20), truncated form CsrA reduced             glycogen synthesis, restore ability grow glyco- <br /> affinity RNA  RNA binding site mapped                lytic carbon sources. glgCAP operon deleted amino-terminal region CsrA). provide              DJ624 strain using method described reference 6. explanation phenotype csrA::kan mutant.                    csrA::cat deletion transduced P1 &#226;&#338;&#172;glgCAP <br />    excess glycogen accumulation impairs growth               strain containing pWSK-csrA plasmid presence 0.1 csrA::cat mutant. CsrA controls intracellular carbon              mM IPTG. Transductants obtained checked PCR <br /> flux positively regulating glycolysis pathway nega-            primers flanking csrA ORF. Interestingly, double <br /> tively regulating gluconeogenesis pathway glycogen                &#226;&#338;&#172;glgCAP&#226;&#338;&#172;csrA::cat pWSK-csrA mutant able form colo- <br /> synthesis (Fig. 2) (22). absence CsrA, glycogen syn-            nies LB plates IPTG (Fig. 3E) glucose <br /> thesis strongly favored.  csrA::kan insertion mu-           fructose-6-phosphate plates (Fig. 3F G), unlike tation resulted 20-fold-higher levels glycogen (21).            &#226;&#338;&#172;csrA::cat pWSK-csrA mutant (Fig. 3A, B, C). strains <br /> csrA::cat mutant unable direct carbon flux           able grow presence pyruvate sole Krebs cycle, reduced glycolysis                carbon source (Fig. 3D H). data enhanced glycogenesis. test hypothesis, &#226;&#338;&#172;csrA::cat            csrA::cat mutant, glycogen synthesis favored glyco- <br /> mutant plated Ceria synthetic medium (10) containing               gen accumulation impairs viability. absence <span id='am-6' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-7' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>gly</span>- <br />  1724         NOTES                                                                                                                                       J. BACTERIOL. <br />  <br />                                                                                           Identification global repressor gene, rsmA, Erwinia carotovora subsp. <br />                                                                                           carotovora controls extracellular enzymes, N-(3-oxohexanoyl)-L-homo- <br />                                                                                           serine lactone, pathogenicity soft-rotting Erwinia spp. J. Bacteriol. <br />                                                                                           177:5108&#226;&#8364;&#8220;5115. <br />                                                                                      6.   Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo- <br />                                                                                           somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br />                                                                                           Sci. USA 97:6640&#226;&#8364;&#8220;6645. <br />                                                                                      7.   Dubey,  K., C. S. Baker, K. Suzuki,  D. Jones, P. Pandit, T. Romeo,                                                                                           P. Babitzke. 2003. CsrA regulates translation Escherichia coli carbon <br />                                                                                           starvation gene, cstA, blocking ribosome access cstA transcript. J. <br />                                                                                           Bacteriol. 185:4450&#226;&#8364;&#8220;4460. <br />                                                                                      8.   Fields, J. , S.  Thompson. 2008. Campylobacter jejuni CsrA mediates <br />                                                                                           oxidative stress responses, biofilm formation, host cell invasion. J. Bac- <br />                                                                                           teriol. 190:3411&#226;&#8364;&#8220;3416. <br />                                                                                      9.   Forsbach-Birk, V., T. McNealy, C. Shi, D. Lynch, R. Marre. 2004. <br />                                                                                           Reduced expression global regulator protein CsrA Legionella <br />                                                                                           pneumophila affects virulence-associated regulators growth Acanth- <br />                                                                                           amoeba castellanii. Int. J. Med. Microbiol. 294:15&#226;&#8364;&#8220;25. <br />                                                                                     10.   Glansdorff, N. 1965. Topography cotransducible arginine mutations                                                                                           Escherichia coli K-12. Genetics 51:167&#226;&#8364;&#8220;179. <br />                                                                                     11.   Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu- <br />                                                                                           lation, modulation, high-level expression vectors containing                                                                                           arabinose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130. <br />                                                                                     12.   Hansen,  M., H. Lehnherr, X. Wang, V. Mobley, D. J. Jin. 2003. <br />                                                                                           Escherichia coli SspA transcription activator bacteriophage P1 late <br />                                                                                           genes. Mol. Microbiol. 48:1621&#226;&#8364;&#8220;1631. <br />                                                                                     13.   Heroven,  K., K. Bohme, M. Rohde, P. Dersch. 2008. Csr-type <br />                                                                                           regulatory  including small non-coding RNAs, regulates global <br />                                                                                           virulence regulator RovA Yersinia pseudotuberculosis Ro <br /> </body></html>