<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Plaimas et al. BMC Systems Biology 2010, 4:56 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />     METHODOLOGY ARTICLE                                                                                                                               Open Access <br />  <br /> Identifying essential genes bacterial metabolic <br /> Methodology article <br />  <br /> networks machine learning methods <br /> Kitiporn Plaimas1,2, Roland Eils*1,2 Rainer K&#195;&#182;nig1,2 <br />  <br />  <br />  <br />     Abstract <br />     Background: Identifying essential genes bacteria supports identify potential drug targets understanding <br />     minimal requirements synthetic cell.  experimentally assaying essentiality coding genes     resource intensive feasible bacterial organisms, particular infective. <br />     Results: developed machine learning technique identify essential genes using experimental data     genome-wide knock screens bacterial organism infer essential genes related bacterial <br />     organism. used broad variety topological features, sequence characteristics expression properties <br />     potentially associated essentiality, flux deviations, centrality, codon frequencies sequences,  <br />     regulation phyletic retention. organism-wise cross-validation bacterial species yielded reliable results     good accuracies (area receiver-operator-curve 75% - 81%). Finally, applied drug target <br />     predictions Salmonella typhimurium. compared predictions viability experimental knock-outs S. <br />     typhimurium identified 35 enzymes, highly relevant considered potential drug targets. <br />     Specifically, detected promising drug targets non-mevalonate pathway. <br />     Conclusions: Using elaborated features characterizing network topology, sequence information microarray data <br />     enables predict essential genes bacterial reference organism related query organism     knowledge essentiality genes query organism. general, method beneficial inferring <br />     drug targets experimental data genome-wide knockout screens available investigated <br />     organism. <br />  <br />  <br /> Background                                                                                  fully targeted antibiotics inhibiting essential processes <span id='am-40' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, essential proteins cellular organism                                bacterial genomes [1]. <br /> necessary live replicate, attrac-                                    computational techniques developed <br /> tive targets antimicrobial treatments.  exper-                                  identify essential genes silico. Flux balance analyses <br /> imentally assaying essentiality coding genes                                (FBA) widely used assess essentiality genes [2]. <br /> error prone high throughput. Additionally,                                      FBA approaches need clear definitions nutri- <br /> experimental screens resource intensive feasi-                                  tion availability biomass production specifi- <br /> ble organisms, typically, gene knock-                                 cally given environmental conditions [3]. Descriptors strain needs constructed.  patho-                                    enzymes metabolic network graph <br /> genic bacterial organisms hazardous cultivate                                    theoretical approaches used identify drug need higher laboratorial efforts. metabo-                                     targets micro-organisms. Concepts choke-points <br /> lism cell substantial maintaining life                                      load-points successfully applied estimate growth, metabolic enzymes success-                                      essentiality enzyme [4-6]. term &apos;damage&apos;                                                                                             used assess enzymes serve drug targets <br />                                                                                             inhibition influences substantial number * Correspondence: r.eils@dkfz                                                            downstream reactions products [7]. previous <br /> 1Department Bioinformatics Functional Genomics, Institute                         study, examined ability network obtain <br /> Pharmacy Molecular Biotechnology, Bioquant, University Heidelberg, Im                products knocked reaction educts <br /> Neuenheimer Feld 267, 69120 Heidelberg, Germany list author information available end article                      alternative pathways used method detect <br />                                             &#194;&#169; 2010 Plaimas et al; licensee BioMed Central  Open Access article distributed terms Creative Commons <br />             BioMed Central Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction                                             medium, provided original work properly cited. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                     Page 2 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> potential drug targets P. falciparum [6]. Various            screen, defined 35 enzymes drug targets S. <br /> descriptors centrality node network           typhimurium, 23 described previ- successfully applied supported detecting essen-         ously drug targets micro-organisms. sug- <br /> tial proteins protein-protein interaction networks [8-        gest remaining potential new drug <br /> 12]. shown proteins central        targets organism studied (S. typhimurium). <br /> position evolve slowly likely essential survival [9]. sequence features like          Results discussion <br /> codon usage, GC-content localization signals            Predicting essential genes organism-wise cross <br /> used predicting essential genes. success-          validation <br /> fully applied inferring essential genes S. cerevi-      general workflow depicted Figure 1. recon- <br /> siae studied yeast strain S. mikatae [13].           structed metabolic networks investigated <br /> Additionally, phyletic retention          organisms bipartite graph consisting alternating <br /> valuable predictive feature gene essentiality E. coli     reactions compounds. reactions linked S. cerevisiae [14,15]. protein-protein inter-       compound compound product reac- <br /> action networks provide global view cellular sig-       tion substrate  machine learning <br /> naling, interested identifying drug            trained validated large set fea- <br /> targets pathogens inferred properties mal-            tures. Local topology based features used qualita- <br /> functional metabolism having knocked enzy-          tively possible flux deviations. Choke load <br /> matic function.                                                  points defined damage used    previous study, developed applied inte-        qualitative flux load stream effects grative machine learning method combined              knocked reaction. Centrality features calcu- <br /> topology based methods validate experimental               lated additionally estimate load reactions. <br /> knock screen Escherichia coli [16]. used           considered existence homologous genes basic concepts strategy enable predicting         corresponding knocked genes expected <br /> essential genes organism experimental         function. regulated genes  <br /> training data available. defining essentiality   sidered estimate existence possible analogous <br /> gene, integrated sequence characteristics         genes. Phyletic retention calculated observing <br /> codon usages, length sequences phyletic reten-        phylogenetic conservation gene tion. Furthermore, incorporated centrality-           knocked  Codon usages calculated gene <br /> descriptors node network. used experimen-          used features. features listed Table 1 tal datasets comprehensive genome wide knock              described methods. started pre- <br /> screens Escherichia coli [17,18] Pseudomonas              dicting essential genes E. coli.  trained <br /> aeruginosa [19,20] train machines large vari-      classifiers (machines) experimental data ety attributes including topology characteristics          genome-wide knock screens Pseudomonas aerugi- <br /> mentioned  developments evaluating pos-             nosa (datasets paeJ paeL experimental studies <br /> sible flux deviations [6], genomic transcriptomic        Jacobs workers [19] Liberati work- <br /> information. develop classification          ers [20], respectively). datasets taken readily applicable predicting essential genes new       gold standard defining true positives true negatives <br /> query organism, needs make accurate pre-           essential genes metabolism training <br /> dictions organism trained.            organism (P. aeruginosa). trained (100) classi-  performed cross-validation            fiers essential genes equal ran- <br /> organisms E. coli P. aeruginosa, e. trained     domly selected non-essential genes (stratification E. coli validated P. aeruginosa  vice versa)     training data). trained machines applied obtain quality performance approach.          predict essential genes query organism (E. coli). applied trained validated classifiers         output machines summed used pathogenic bacterium Salmonella typhimurium.              voting score represented propensity gene compared results literature experimen-          lethal cell. turn, scheme tal data large knock study S. typhimurium           applied predict essential genes P. aeruginosa [21]. Furthermore, analyzed predictions gene         classifiers trained datasets <br /> set enrichment tests metabolic pathways identi-          E. coli (ecoB Baba workers&apos; study [17] <br /> fied proteins entire non-mevalonate pathway         ecoG Gerdes workers&apos; study [18], <br /> relevant targeting drugs. reactions showed          respectively). organism-wise cross-validation typically topological characteristics essential reactions.    applied estimate performance classifiers. Using prediction results experimental knock-         compared datasets genome. 79 essen- <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                     Page 3 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 1: List features. <br />  Short form           Explanation <br />  <br />  <br />  Topology features <br />                        Deviation <br />  RUP                  Reachable/Unreachable Products (RUP): equals products produced blocking reaction, <br />                       zero <br />  PUP                  Percentage Unreachable Products (PUP): percentage products produced blocking                       reaction <br />  ND                   Number Deviations (ND) <br />  APL                  Average Path Length (APL): average path length deviations <br />  LSP                  Length Shortest Path (LSP): length shortest path deviations <br />                       b) Local topology <br />  NS                   Number Substrates (NS) <br />  NP                   Number Products (NP) <br />  NNR                  Number Neighboring Reactions (NNR) <br />  NNNR                 Number Neighbors Neighboring Reactions (NNNR) <br />  CCV                  Clustering Coefficient Value (CCV): clustering coefficient reaction <br />  DIR                  Directionality reaction (DIR) <br />                       c) Choke points load scores <br />  CP                   Choke Point (CP): reaction choke point (Rahman et al, 2006) <br />  LS                   Load Score (LS): load score reaction (Rahman et al, 2006) <br />                       d) Damage <br />  NDR                  Number Damaged Reactions (NDR) (Lemke et al, 2004) <br />  NDC                  Number Damaged Compounds (NDC) (Lemke et al, 2004) <br />  NDRD                 Number Damaged Reactions having Deviations (NDRD): number damaged reactions                       alternative paths reached blocking reaction <br />  NDCD                 Number Damaged Compounds having Deviations (NDCD): number damaged compounds                       alternative paths reached blocking reaction <br />  NDCR                 Number Damaged Choke point Reactions (NDCR) <br />  NDCC                 Number Damaged Choke point Compounds (NDCC) <br />  NDCRD                Number Damaged Choke point Reactions having Deviations (NDCRD): number damaged choke point <br />                       reactions alternative paths reached blocking reaction <br />  NDCCD                Number Damaged Choke point Compounds having Deviations (NDCCD): number damaged choke point <br />                       compounds alternative paths reached blocking reaction <br />                       e) Centrality <br />  BW                   Betweenness centrality <br />  CN                   Closeness centrality <br />  EC                   Eccentricity centrality <br />  EV                   Eigenvector centrality <br />  Genomic transcriptomic features <br />                       f) Homologs <br />  NAR                  Number Associated Reactions (NAR): number reactions base knocked gene <br />  Hn                   Homology different expectation values: number homologous genes e-value cutoff 10-30,10-20,10-10,10-7,10- <br />                       5,10-3 (H30, H20, H10, H7, H5, H3) <br />  <br />                       g) Gene expression <br />  NGSE                 Number Genes having Similar Expression (NGSE): number genes similar expression (correlation <br />                       coefficient &gt;0.8) <br />  MCC                  Maximum Correlation Coefficients (MCC): maximum value correlation coefficients neighboring genes <br />                       h) Phyletic retention <br />  PR                   Phyletic Retention (PR): number orthologs prokaryotes <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                          Page 4 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 1: List features. (Continued) <br />                         Codon usage <br />  Nc                    Number codons <br />  N3s                   Base composition silent sites (T3s, C3s, A3s, G3s) <br />  glt                   frequency amino acids glutamine (exemplarily) <br />  <br />  <br /> tial genes common ecoB ecoG, 92 com-                          predicting essential genes P. aeruginosa. turn, <br /> mon paeL paeJ. machines                                yielded precision 61% (accuracy: 87%, sensitivity: <br /> trained different training-sets knock                        27%, validating ecoB) 65% (accuracy: 80%, sensi- <br /> screen. Votes training sets organism                          tivity: 18%, validating ecoG) E. coli. yielded summed defined stringency. high                              best classifier results using features, com- <br /> number votes essentiality led high specificity,                      parison classification performance using lower numbers led higher sensitivity. result-                     individual sets features  Additional file 1: <br /> ing receiver operator curves (ROC) classifiers                      SupplementS1 details). <br /> shown Figure 2A predicting P. aeruginosa Fig- <br /> ure 2B predicting E. coli. predicting essential                        Examining features <br /> genes P. aeruginosa yielded area                           wanted obtain estimate correlations curve (AUC) 0.80 0.79 compared                              features essentiality gene.  experimental datasets paeL paeJ, respectively.                          calculated Pearson&apos;s correlation coefficients essen- <br /> turn, E. coli yielded AUC 0.81 0.75                      tiality class gene (1 = essential, 0 = non-essential) <br /> compared ecoB ecoG, respectively. wanted                          corresponding feature values. Figure 3 gives obtain reliable list potential drug targets.                     overview features  Additional file 2: <br /> predictions essential genes needed low number                         SupplementS2 correlation coefficients features). <br /> false positives.  set high stringency calcu-                    following, major results cor- <br /> lated precision (true predictions predictions                   relation study. essentiality) high selection criterion                                                                                 Topology features <br /> 195 200 votes). yielded precision 67% <br />                                                                                efficiency flux deviations estimated fea- <br /> (accuracy: 87%, sensitivity: 7%, validating paeL)                                                                                tures RUP PUP gave estimate products <br /> 100% (accuracy: 80%, sensitivity: 3%, validating paeJ) <br />  <br />  <br />  <br />                                                            	 <br />   <br />        	  <br />        <br />   <br />                   <br />       <br />                        	 <br />     <br />   <br />    <br /> 	                                                <br />              <br /> 	                                                    <br />                                                  	   <br />     <br />               <br />                                                       <br />                 <br />                                                      <br />           <br />                                                                              <br />                                                             	     <br />                                                                      <br />                                                          <br />                                                 	    <br />       <br />  <br />                                                              	  <br />         <br />          <br />                                      	 <br />                                                                        	 <br /> 	  <br />         <br />                                                     	 <br /> 	 	  <br />  	     <br />  <br />                                                                                                                     <br />   	 	  <br />                                                             	 <br /> 	  	  	 <br />  	        <br />      	  <br />                                               <br />                                                                	     	 	 <br />     <br />  <br />  <br />                                                                                                                           <br />       	  <br />                                                                                                              <br />       	 <br />  Figure 1 workflow. Essentiality data organism taken experimental genome wide knock screen. used train  machine learning using features based topology metabolic network genomic transcriptomic data. trained clas- <br />  sifier applied organism (query organism) essentiality gene metabolism predicted. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                               Page 5 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                                                                <br />  <br />  <br />  <br />  <br />                                                                                                                 <br />      <br />  <br />  <br />  <br />  <br />                                                                                                 	 <br /> 	     <br />        	 <br /> 	 	 <br />    <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                	 <br />                                               <br />                                                                                                                                                    	 <br />    <br />                                                                                                                                                    	 <br />      <br />                                                                   <br />                                                                                                                                                                 <br />                                                                                                                                     	 <br /> 	      <br />    <br />         	 	 	 <br />    <br />  <br />  <br />  <br />  <br />                                                                                                                 <br />                                                                 <br />  <br />  <br />  <br />                                                                                                 	 <br /> 	     <br />        	 <br /> 	 	 <br />    <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                                                                   <br />                      <br />                                                                 <br />                                                                   <br />                                                                                                                                                   	 <br />    <br />   <br />                                                                                                                                                   	 <br />      <br />                                                                  <br />                                                                                                                                                                 <br />                                                                                                                                     	 <br /> 	      <br />    <br />         	 	 	 <br />    <br />  <br />  <br />  Figure 2 ROC curves prediction performances.   100 Support Vector Machines trained datasets ecoB ecoG, respectively, <br />  queried using datasets P. aeruginosa (union datasets paeL paeJ). number machines predicting essentiality <br />  summed (voting score). Results varying thresholds voting score compared experimental results paeL paeJ yield- <br />  ing ROC curves (area curve: 0.80 0.79, respectively). (B) Similar   machines trained datasets P. <br />  aeruginosa queried datasets E. coli resulting ROC curves AUC = 0.81 0.75 datasets ecoB ecoG, respectively. <br />  <br />  knocked reaction produced                         covered alternative pathways. number reaction (RUP) large percentage non-                       substrates products reactions knocked <br /> producible products (PUP)  RUP Boolean fea-                         gene (NS, NP) positively correlated gene <br /> ture observe mutant produce products                      essentiality (P = 4.3E-06 P = 0.0172, respectively) knocked reaction. RUP = 1 set                      showing essential enzymes metabolize differ- <br /> stream products produced mutant                         ent compounds. Interestingly, number neighboring <br /> RUP set zero downstream product                        reactions (NNR) number neighbors neigh- produced. number reachable prod-                          boring reactions (NNNR) showed weak negative corre- <br /> ucts (RUP) highly negative correlated                         lation essentiality (P = 0.14 P = 0.091, respectively). <br /> centage unreachable products (PUP) highly positive                         reasonable reaction high number correlated essentiality gene (P = 1.2E-10 P                   neighboring reactions metabolites = 2.4E-09, respectively) shown Figure 3A.                     products produced alternative enzymes. <br /> silico) mutant produce                             clustering coefficients (CCV) showed  <br /> stream products, RUP zero percentage                          dency (negatively correlated, P = 0.018) pointing unreachable products increased comparison                        advantageous alternative pathways. situation products produced.                          estimated feasibility possible flux deviations higher percentage unreachable products                      set features describing alternative pathways. mutant, products knocked enzyme                           number alternative pathways (ND), average path <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                 Page 6 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                                                                  	 <br />    <br /> 	    <br />  <br /> 	  <br />            	    <br />      <br />                 	     <br />  <br />     	   <br />                                       <br />                                        <br />                                                      <br />  <br />  <br />  <br />  <br />                                                                                                                                         <br />                                                                                                                                         <br />                                                                                                                                         <br />                                                                                                                                       	 <br />  <br />  <br />  <br />  <br />                                                              <br />                                                              <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />                                                               <br />                                                              <br />  <br />  <br />                                                              <br />                                                              <br />  <br />                                                            	  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                             <br />  <br />  <br />  <br />                                                               <br />                                                              <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />                                                           <br />    <br />                                                              <br />                                                              <br />  <br />                                                              <br />                                                              <br />  <br />  <br />  <br />  <br />                                                               <br />  <br />  <br />  <br />  <br />                                                  	  <br />    	  <br />             	           	          <br />  <br />     	     <br />       <br />                                      <br />               	     <br />  <br />     	   <br />                                      <br />                                      <br />                                                     <br />  <br />  <br />  <br />  <br />                                                                                                                                           <br />                                                                                                                                           <br />                                                                                                                                           <br />                                                                                                                                         	 <br />                                                               <br />                                                                <br />                                                                <br />                                                               <br />                                                                <br />                                                            !   <br />  <br />  <br />                                                                <br />                                                                <br />                                                               <br />                                                               <br />                                                               <br />                                                               <br />  <br />                                                                 <br />                                                              <br />                                                              <br />                                                              <br />                                                            	 <br />                                                             <br />  <br />                                                              <br />                                                             <br />  <br />  <br />  <br />  <br />                                                               <br />                                                               <br />                                                                 <br />                                                                <br />                                                             <br />  <br />  <br />  <br />  <br />                                                            <br />  <br />                                                              <br />                                                              <br />                                                              <br />                                                              <br />                                                             <br />                                                              <br />                                                                <br />                                                              <br />  <br />  <br />  <br />  <br />                                                              <br />  <br />  <br />  <br />  <br />  Figure 3 Correlation features essentiality. feature-values gene correlated essentiality gene (1 = essential, <br />  0 = non-essential).   shows correlation coefficients topology features, (B) genomic transcriptomic features. High values indi- <br />  cate feature positively correlated essentiality  Additional file 2: SupplementS2 correlation coefficients). values  obtained gold standards (ecoB, ecoG E. coli paeJ, paeL P. aeruginosa). <br />  <br />  <br /> length deviations (APL) length                           alternative pathways. expected, nega- <br /> shortest alternative path (LSP) described feasibility                   tively correlated essentiality (P = 0.15, P = 3.4E-04  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                      Page 7 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> P = 0.0063, respectively), e. knocked enzymes           redundant position genetic code. alternative pathways existed likely            Matching mRNA tRNA codon nucleotides cause lethal phenotype knocked  Choke-points             robust position, translational errors (CP) uniquely consumed produced compounds               likely occur position.  metabolism showed positive correlation             essential genes need stable protected essentiality (P = 2.8E-04) choke-points diffi-       sequence. <span id='am-70' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span>Thymine</span> genetic code cope cult replaced rest metabolism. Load-          shown thymine protected DNA scores (LS) estimate reaction           improved efficiency DNA replication [22].  <br /> involved metabolic processes. posi-             served genes likely essential [23] thy- <br /> tively correlated essentiality (P = 9.4E-04).          3rd codon position facilitates stable genetic <br /> ness centrality (BW) eccentricity (EC) strongly          inheritance springs cellular replicates. Inter- <br /> positive correlated essentiality (P = 1.3E-14 7.6E-        estingly, observed larger difference T3s E. coli <br /> 08, respectively) showing enzymes higher              compared P. aeruginosa. large <br /> influence vitality placed center network.     average G C content codon position Closeness centrality (CN) showed positive correla-         common genes P. aeruginosa [24]. results <br /> tion (P = 0.0020). Interestingly, eigenvector centrality      low T content codon position (EV) showed negative correlation (P = 0.0013).                  observed explain larger difference T3s Betweenness, closeness eccentricity centrality            essential non-essential genes E. coli compared P. <br /> global centrality measures considering network          aeruginosa  Additional file 3: SupplementS3 eigenvector centrality measure local           histograms). <br /> centrality computed neighbors. Note             decision trees performed inferior com- <br /> typically node high value eigenvector central-        pared Support Vector Machines (data shown), ity hub (node high connectivity) hubs        used decision trees alternative approach deter- <br /> connected   flux deviations               discriminating features gene essential- <br /> likely local hubs hubs vicinity mak-       ity. Decision trees trained datasets ecoB, <br /> ing node replaceable global central nodes             ecoG, paeL paeJ. known decision generally substantial maintaining            decision tree performed best discriminating <br /> metabolic flow network. eigenvector          feature [25].  counted occurrence centrality network topology          features position run. 400 runs, <br /> sense clustering coefficient, specifically respect      got T3s (137&#195;&#8212;), BW (81&#195;&#8212;), Nc (38&#195;&#8212;), H3 (21&#195;&#8212;), H5 likelihood alternative pathways.                        (15&#195;&#8212;), val (12&#195;&#8212;), H7 (11&#195;&#8212;), NGSE (11&#195;&#8212;), PUP (10&#195;&#8212;)                                                                   decision (features 10 counts Genomic transcriptomic features                               shown). showed feature T3s sub- expected, number homologous genes (H30,                 stantially supported classification. <br /> H20, H10, H7, H5, H3) showed negative correlation essentiality (P = 3.2E-04, 6.3E-04, 1.4E-06, 4.7E-09, 1.1E-       Identifying drug targets S. typhimurium <br /> 10, 1.5E-09, respectively). Interestingly, E-value cut     applied trained machines datasets 10-5 (H5) worked best showing non-perfectly          (ecoB, ecoG, paeL, paeJ) predict essential genes matching sequences functions                 S. typhimurium obtained votes knocked gene. number genes having similar              machines gene S. typhimurium essential. <br /> expression (NGSE) exhibited negative correlation           obtain reasonable threshold number votes essentiality (P = 1.7E-04)              predicting gene essential, compared num- <br /> expression genes analogous function. fea-         ber essentially predicted genes numbers ture phyletic retention (PR), number prokaryotes           training sets E. coli P. aeruginosa. E. coli 104 <br /> having orthologs knocked gene showed posi-           147 genes essential corresponding data- <br /> tive correlation essentiality (P = 2.1E-16) supporting         sets ecoB ecoG, respectively, P. aeruginosa, 92 findings previous study conserved genes          150 (corresponding datasets paeL paeJ, <br /> evolution hint essentiality [14].                       respectively).  set threshold 350 votes <br />   analyzed codon usage gene related            400 machines) classify gene essential S. essentiality gene. genes        typhimurium obtained comparable 128 high number <span id='am-62' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-63' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-67' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> thymine         predicted essential genes. complete list genes <br /> position codons likely essential       predicted essential given supplement <br /> cell viability (feature T3s Figure 3, Additional file 3:    Additional file 4: SupplementS4). compared <br /> SupplementS3 histograms). codon position            results experimental data Knuth  <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                   Page 8 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> workers performed large knock study S.            * pyrH: gene uridylate kinase essential typhimurium [21]. detected 6% open reading        Mycobacterium tuberculosis [32]. <br /> frames essential including 53 essential genes          ** uppS: Undecaprenyl <span id='am-15' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-16' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span>pyrophosphate</span> synthase (UPPS) <br /> coding enzymes metabolism. remaining           novel antibacterial target Streptococcus pneumo- <br /> open reading frames genome didn&apos;t make        niae [33]. <br /> prediction, including 711 genes enzymes metabo-          ** accA: Acetyl-<span id='am-57' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span> carboxylase drug target lism. compared list essential genes Knuth        anti-obesity antibiotic drugs [34,35]. workers predictions 27         * hemH: Ferrochelatase essential multiplication <br /> predicted genes list Knuth workers yield-    intracellular survival Brucella abortus [36]. <br /> ing precision 21%, accuracy 83% sensitivity      * kdsB: Analogs 3-deoxy-D-manno-octulosonate 51%. note experimental screen Knuth   (KDO) designed inhibit <span id='am-139' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-140' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CTP</span>:<span id='am-129' about='obo:IMR_0200494' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-130' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="KEGG:C00055" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:id" content="IMR:0200494" datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="CMP" datatype="xsd:string"></span>CMP</span>-KDO cytidy- workers comprehensive; authors             lyltransferase (CMP-KDO synthetase) [37]. poten- <br /> stated article genes pre-     tial target Haemophilus influenzae [38] E.coli [39]. <br /> dicted essential, couldn&apos;t conclude          * fabD: Acyl carrier protein S-malonyltransferase genes definitely non-essential.  novel      potential target Mycobacterium bovis BCG [38]. <br /> predictions suit potential new targets       ** fabG: 3-ketoacyl-(acyl-carrier-protein) reductase investigations. conservative robust estimate      known drug target E. coli, B. subtilis, S. aureus <br /> essential genes S. typhimurium, defined corre-     [40]. <br /> sponding enzymes genes experimentally             * pssA: <span id='am-6' about='obo:IMR_0001358' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-7' property="oboInOwl:id" content="IMR:0001358" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C02737" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:18303" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasAlternativeId" content="IMR:0200250" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="phosphatidylserine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="O-Phosphatidyl-L-serine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Phosphatidylserine</span> synthetase required determined  Knuth workers) recog-          motility chemotaxis E. coli [41]. Furthermore, <br /> nized classifiers. searched litera-     mutants Escherichia coli K12 defective ture drug treatments enzymes       phosphatidylserine synthetase, isolated tempera- <br /> micro-organisms. results listed following.     ture-sensitive, conditional lethals [42]. asterisks (**) mark gene enzyme              * kdtA: 3-deoxy-D-manno-octulosonic-acid trans- clear evidence drug target micro-         ferase enzyme kdtA. E. coli, essential organism. asterisk (*) set gene         cell growth accounts conditional lethality associ- reasonable evidence enzyme serve        ated mutations KDO biosynthesis [43]. <br /> drug target micro-organism.  enzymes           ** dfp: Pantothenate kinase uptream phosphopan- asterisk serve finding new drug targets       tothenoylcysteine decarboxylase biosynthesis enzymes asterisks transferring drug targets      pantothenate CoA. known target anti- bacterial diseases disease studied (S.   microbial drugs E. coli Mycobacterium tuber- <br /> typhimurium). Table 2 gives overview results.       culosis [44]. compared open reading frames predicted            * rep: <span id='am-46' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-47' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span>-dependent DNA helicase Rep. deletion <br /> genes human transcripts did detect sig-      lethal B. subtilis [45] Stapphylo- <br /> nificant homologs (using BLAST [26] ENSEMBL               coccus aureus [46]. <br /> cDNA transcripts [27]). E-values best hits given     * rpoB: DNA-directed RNA polymerase promising supplement  Additional file 5: SupplementS5).      target discovery new antimicrobial agents brief summary literature evidences      E. coli [47].  <br />    ** murE: <span id='am-134' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-135' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-N-acetylmuramoylalanyl-D-glutamate-           non-mevalonate pathway fatty acid biosynthesis <br /> 2,6-diaminopimelate ligase essential enzyme       highly enriched essential genes S. typhimurium known target bacterial cell walls Staphylo-   performed gene set enrichment tests (Fisher&apos;s exact <br /> coccus aureus [28].                                           tests) pathways KEGG [48] sig- <br />    * murG: N-acetylglucosaminyl transferase potential    nificant enrichment essential genes non- <br /> antibiotic targeting biosynthesis bacterial peptido-   mevalonate pathway (P = 9.2E-06) fatty acid <br /> glycans.  difficult design inhibitors    biosynthesis pathway (P = 3.8E-04). genes enzyme. Identifying inhibitors current          pathways essential (8 9 genes research [29,30].                                             non-mevalonate pathway 8 12 genes    ** murC: UDP-N-acetylmuramate-<span id='am-41' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span>L-alanine</span> ligase cat-        fatty acid biosynthesis pathway. non-mevalonate <br /> alyzes essential step pathway synthesizing      pathway (Figure 4) produces <span id='am-20' about='obo:IMR_0200287' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Isopentenyl diphosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200287" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00129" datatype="xsd:string"></span>isopentenyl diphosphate</span> <br /> <span id='am-75' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-76' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-82' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span>peptidoglycan</span> precursors. Recently, new inhibitors     (IPP) dimethylallyl pyrophosphate (DMAPP) MurC enzyme successfully tested Escheri-        serve basis production sterols, dolichols, <br /> chia coli, Proteus mirabilis Klebsiella pneumoniae        <span id='am-30' about='obo:IMR_0200483' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="rdfs:label" content="Ubiquinone" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00399" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200483" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ubiquinone</span> components macromole- <br /> [31].                                                         cules prenyl groups proteins [49]. pathway <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                           Page 9 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> Table 2: Predicted essential genes potential drug targets. <br />  <br />   ORF              Gene Symbol         EC                 Enzyme                                                       Evidence <br />  <br />  <br />    Intersection predictions experimental screen <br />   STM0123          murE                6.3.2.13           UDP-N-acetylmuramoylalanyl-D-glutamate-2,6-                     ** <br />                                                           diaminopimelate ligase <br />   STM0128          murG                2.4.1.227          N-acetylglucosaminyl transferase                                * <br />   STM0129          murC                6.3.2.8            UDP-N-acetylmuramate-L-alanine ligase                           ** <br />   STM0154          lpdA                1.8.1.4            Dihydrolipoamide dehydrogenase <br />   STM0218          pyrH                2.7.4.22           Uridylate kinase                                                * <br />   STM0221          uppS                2.5.1.31           Undecaprenyl pyrophosphate synthase                             ** <br />   STM0222          cdsA                2.7.7.41           <span id='am-25' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="CDP" datatype="xsd:string"></span>CDP</span>-<span id='am-118' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-119' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-123' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span>diglyceride</span> synthase <br />   STM0228          lpxA                2.3.1.129          UDP-N-acetylglucosamine acyltransferase <br />   STM0232          accA                6.4.1.2            <span id='am-113' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span>Acetyl-CoA</span> carboxylase                                          ** <br />   STM0489          hemH                4.99.1.1           Ferrochelatase                                                  * <br />   STM0535          lpxH                                   UDP-2,3-diacylglucosamine hydrolase <br />   STM0542          folD                1.5.1.5, 3.5.4.9   Bifunctional 5,10-methylene-tetrahydrofolate dehydrogenase <br />   STM0988          kdsB                2.7.7.38           CTP:CMP-KDO cytidylyltransferase                                * <br />   STM1194          fabD                2.3.1.39           Acyl carrier protein S-malonyltransferase                       * <br />   STM1195          fabG                1.1.1.100          3-ketoacyl-(acyl-carrier-protein) reductase                     ** <br />   STM1200          tmk                 2.7.4.9            Thymidylate kinase <br />   STM1700          fabI                1.3.1.10           Enoyl-(acyl carrier protein) reductase <br />   STM2483          dapE                3.5.1.18           Succinyl-diaminopimelate desuccinylase <br />   STM2652          pssA                2.7.8.8            Phosphatidylserine synthase                                     * <br />   STM3090          metK                2.5.1.6 <br />   STM3415          rpoA                2.7.7.6            DNA-directed RNA polymerase subunit alpha <br />   STM3724          kdtA                                   3-deoxy-D-manno-octulosonic-acid transferase                    * <br />   STM3730          dfp                 4.1.1.36           Pantothenate kinase                                             ** <br />   STM3912          rep                 3.6.1.-            ATP-dependent DNA helicase Rep                                  * <br />   STM3978          yigC                                   3-octaprenyl-4-hydroxybenzoate decarboxylase <br />   STM4153          rpoB                2.7.7.6            DNA-directed RNA polymerase subunit beta                        * <br />   STM4154          rpoC                2.7.7.6            DNA-directed RNA polymerase subunit beta&apos; <br />   b) Predictions non-mevalonate pathway <br />   STM0049          ispH, lytB          1.17.1.2           4-hydroxy-3-methylbut-2-enyl diphosphate reductase              * <br />   STM0220          dxr                 1.1.1.267          1-deoxy-<span id='am-103' about='obo:IMR_0200169' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-104' property="oboInOwl:hasDbXref" content="KEGG:C00231" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="D-Xylulose 5-phosphate" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:id" content="IMR:0200169" datatype="xsd:string"></span>D-xylulose 5-phosphate</span> reductoisomerase                 * <br />   STM0422          dxs                 2.2.1.7            1-deoxy-D-xylulose-5-phosphate synthase                         ** <br />   STM0423          ispA                2.5.1.10           geranyltranstransferase                                         * <br />   STM1779          ispE, ipk           2.7.1.149          4-diphosphocytidyl-2-C-methyl-D-erythritol kinase               ** <br />   STM2523          ispG, gcpE          1.17.7.1           4-hydroxy-3-methylbut-2 1-yl diphosphate synthase            * <br />   STM2929          ispF                4.6.1.12           2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase           * <br />   STM2930          ispD                2.7.7.60           2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase        * <br />   ** clear evidence drug target <br />   * reasonable evidence serve good drug target <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                       Page 10 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br />                 	 <br />       <br />                                              	 <br />       <br />    <br />                            	 <br />       <br />                     <br />                                                         <br />     <br />     <br />                                                                                                           <br />  <br />                                               <br />                                                                                       	    <br />  <br />                                                                                                               	   <br />   <br />                                                      <br />                                                 <br />   <br />   <br />     <br />                                                                                        	  &quot; <br />                                                                                                          <br />           <br />          <br />         <br />   <br />    <br />       <br />                                                                                 <br />   <br />   <br />     <br />                    <br />                                                                                       	   <br /> ! <br />                                                                                                                     <br />   <br />   <br />     <br />                                                                                                          <br />           <br />       <br />  <br />                 <br />                                                                         	 <br />    <br />                                                                                                       <br />   <br />   <br />                   <br />                                                                                                       <br />   <br />   <br />            <br />       <br />                                 <br />                                                                                                        <br />       <br />                                                                                       	   <br />  <br />             <br />                                                                                               <br />           <br />       <br />                                                                                                                <br />   <br />   <br />     <br />                                                                                       	   <br /> # <br />                                                                                                                  <br />       <br />           <br />           <br />                                                                                                  !	            <br />   <br />   <br />     <br />                                                                                       	 <br />   <br />                                 <br />  <br />  <br />                                                                        	 	                        <br />             <br />   <br />   <br />     <br />                          	 <br />     <br />     <br />     <br />  <br />                                                                                         	 <br />                  <br />    	      <br />    <br />   <br />    <br />                                           	 <br />    <br />        <br />  <br />  <br />  Figure 4 non-mevalonate pathway. non-mevalonate pathway produces isopentenyl diphosphate (IPP). alternative pathway bac- <br />  teria does exist human host uses mevalonate pathway produce IPP. non-melavonate pathway highly enriched <br />  genes predicted essential. Reactions given EC number gene symbol genes corresponding  <br />  zymes. <br />  <br />  non-mevalonate biosynthesis considered                        irreversibility reactions previously attractive targets novel antibiotics                     direction overall flux. linear pathway bacteria [50,51] including S. typhimurium [52,53].                 starts 1-deoxy-D-xylulose-5-phosphate-synthase (EC <br /> Figure 4 shows non-mevalonate pathway                          2.2.1.7) corresponding gene dxs identi- <br /> essential enzymes S. typhimurium. Note                        fied essential experimental knock arrows figure represent information                    study Knuth workers [21].  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                   Page 11 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> enzymes downstream predicted essential            separated internal nodes external nodes method. enzyme pathway            reduce computational complexity [54]). geranyltranstransferase (EC 2.5.1.10). catalyzes         method served estimate genes reaction produce farnesyl-diphosphate. Recently, Cor-        corresponding enzymes bacterial organisms E. <br /> nish workers performed elaborated mutagene-           coli, P. aeruginosa S. typhimurium. Inferring essenti- <br /> sis study non-melavonate pathway S.                   ality information organism organism <br /> typhimurium genes essential (ispD,         facilitated investigating closely related organ- <br /> ispE, ispF, ispG, ispH) [52]. propose         ism did S. typhimurium closely <br /> enzymes pathway promising potential drug            related E. coli. apply method micro- <br /> targets S. typhimurium. searched literature       organisms, metabolism need indicated findings asterisks          described, method need adaptation described  genes clear evidences         studied organisms special metabolic capa- <br />  asterisks) genes reasonable evidences          bilities. applying method eukaryotic genomes, <br />  asterisks) code drug targets  Additional file   compartments cell reaction occurs <br /> 5: SupplementS5).                                               need considered. challenging                                                                 apply method inferring multiple drug targets <br /> Conclusions                                                     using experimental double knock screens e.g. established machine learning approach pre-            synthetic lethal project eSGA [55].  attri- <br /> dicts essentiality genes organism         butes related single players (like e.g. sequence <br /> experimental knock data available. classifiers       features) relevance specifically net- trained essentiality information genes         work features related synergistic organism (e.g. E. coli) employed predict        knock effects. challenge future remains essential genes organism (e.g. P. aeruginosa).     integrate topological descriptive approaches predictions did depend essentiality infor-         genetic information systematically explore network <br /> mation query organism predictions          effects enzyme treatments combinations thereof.  solely features calculated metabolic network genomic transcrip-           Methods <br /> tomic information query organism. data           Network reconstruction <br /> abundantly available pathogenic bacteria.           metabolic networks E. coli, P. aeruginosa, S. <br /> applied method predict essential genes Salmo-        typhimurium reconstructed using database nella typhimurium query organism         KEGG [48]. Unspecific compounds water, ATP, <br /> proposed 35 potential drug targets. 27 targets resulted          discarded. Additionally, main com- intersection predictions            pounds reactions annotated KGML files experimental study [21] 8 targets non-               KEGG used. centrality features mevalonate pathway statistical              clustering coefficients, topology features calcu- <br /> enrichment analysis. non-mevalonate pathway              lated representation network bipartite <br /> essential algae, plants eubacteria including     graph consisting metabolites reactions alternat- <br /> pathogenic bacteria. Enzymes pathway          ing nodes. calculating centrality features described potential targets development        clustering coefficients, represented metabolic net- <br /> novel antibiotics herbicidal agents [50,52]. S.         work undirected graph, known reaction- <br /> typhimurium, inferred machine learn-        pair network. consisted reactions nodes ing approach. discovered interesting correlations         metabolites edges connecting reactions features essentiality gene. Various features    graph. reactions connected metabolite describing network topology served machine           metabolite product reactions select reactions showed possible pathways flux      substrate reaction. Edges discarded <br /> deviations, e.g. linear non-mevalonate pathway.       network loops intelligent combination features seen        edge different nodes. Reactions alternative approach established methods        mapped enzymes enzymes mapped corre- <br /> flux balance analyses (FBA) elementary flux modes           sponding genes using association tables KEGG. <br /> (EFM) detailed growth nutrient information            Genes corresponded death-end reactions lacking  needed FBA, [3]          network included datasets training <br /> depth refinement metabolic network considered         validation. gene corresponded labor intensive  EFM enzymes need        reaction, mean value reaction features                                                                 taken. Boolean features (RUP, DIR, CP,  <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                      Page 12 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  used Boolean operation, e. gene feature         defined knocked reaction sub- <br /> set reaction feature equaled        stantial producing downstream metabolites                                                                   products produced pathways. gold standards                                                investigation tested knocked reaction train validate predictions, used published          performed following algorithm: <br /> datasets genome wide experimental knock                      metabolites acting input nodes (substrates) screens. performed LB rich medium.                output nodes (products) knocked reaction <br /> datasets E. coli P. aeruginosa. Addi-          selected. set substrates S defined input <br /> tionally used data knock study S. typh-          nodes set products P defined output nodes. <br /> imurium performed LB rich medium.               broader list available substrates integrated datasets taken NMPDR database [56].             substrates S. included substrates E. coli, used KEIO collection Baba           upstream reactions products downstream <br /> workers [17] denoted &apos;ecoB&apos;. consisted          reactions sets S P, respectively. Substrates 104 essential 641 non-essential genes meta-           reactions substrates S sub- <br /> bolic network. dataset E. coli              strate included S.  substrates reac- <br /> Gerdes workers [18] denoted &apos;ecoG&apos;.            tions metabolite P substrate consisted 147 essential genes 533 non-essential         included S. <br /> genes network. P. aeruginosa used data            ii. Reactions selected used available Liberati et al. [20] denoted &apos;paeL&apos;. consisted 92     compounds substrates. <br /> essential genes 615 non-essential genes net-             iii. selected reactions products work. dataset P. aeruginosa taken          incorporated list discovered reactions study Jacobs et al. [19]. denoted &apos;paeJ&apos;.   products. products set newly available <br /> consisted 150 essential genes 579 non-essential            metabolites network. <br /> genes. experimental dataset S. typhimurium               iv. Steps ii iii repeated reac- Knuth workers [21] based insertion-            tions identified. <br /> duplication mutagenesis (IDM). Small, randomly gener-                v. output nodes produced ated genomic fragments cloned conditionally           counted (reachable products P). <br /> replicating vector, resulting library single S.           finishing process, used number typhimurium clones grown permissive condi-              defined output nodes produced tions. switching non-permissive temperature,              mutated network features, e. quality feature <br /> discrimination lethal non-lethal insertions           defining products produced (RUP, reach- <br /> following homologous recombination allowed trap-              able/unreachable products), percentage prod- <br /> ping genes essential functions. method,         ucts produced (PUP, percentage genes detected indispensable growth.           unreachable products). run breadth  non-essential genes determined.             search network estimate possible deviations. metabolism, 53 genes essential           Starting S, breadth search explored net- remaining 711 essentiality           work finding direct products knocked determined method [21].                                   reaction. algorithm visited products,                                                                   stored corresponding pathway continued Defining features <br />                                                                   search alternative paths network <br /> Features obtained network topology proper- <br />                                                                   entirely explored maximal path length 10 <br /> ties genomic transcriptomic information. Table 1 <br />                                                                   reactions reached. organism shows overview features abbreviations. <br />                                                                   pathways produce products making Topology based features                                           robust.  counted number possible  Deviation features                                             alternative paths yielding feature ND (ND, number following features, used metabolic net-            deviations). took average path length (APL, aver- <br /> work representation bipartite graph consisting        age path length) shortest path length (LSP, length different alternating nodes, e. metabolites          shortest path) deviations features clas- <br /> reactions. reported recently [6,16], implemented          sifier. deviation features used alternative <br /> breadth algorithm investigate network         pathways produce products knocked reac- <br /> single reaction blocked. defined reaction             tion substrates S. metabolic network, essential survival basically mutated network         substrates consumed reactions yield- yield products reaction                 ing products   kept track alter- <br /> upstream substrates reaction.  features  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                         Page 13 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> native paths network potential           reactions [10]. betweenness centrality Cb(v) organism survive reaction blocked.               reaction v given b) Local topology number substrates products knocked                                             d ij (v) reaction counted (NS: number substrates; <br /> NP: number products). defined features                                                                       C b ( v) =      &#226;&#710;&#8216; <br />                                                                                    &#226;&#8240;&#160; j &#226;&#8240;&#160; v&#226;&#710;&#710;V <br />                                                                                                   d ij <br />                                                                                                             ,                                  (1) <br />  <br /> number Neighboring reactions (NNR) number Neighbors Neighboring reactions (NNNR). cal- <br />                                                                  dij number shortest paths reac- <br /> culated clustering coefficient (CCV) described [57,58] knocked reaction estimate local      tion reaction j, dij(v) number shortest <br /> density network. reaction direction (DIR,           paths j pass reaction v. sum directionality reactions) taken KEGG set       composed pairs   j) reactions network. reversible information available.           Closeness centrality approximates edges c) Choke-points load-scores                                required access reaction given reac- reaction uniquely consumes produces certain        tion [10]. defined inverse average length <br /> metabolite metabolic network considered            shortest paths reactions. close- <br /> choke point. reaction shows high potential          ness centrality Cc(v) reaction v given essentiality [4,5]. used feature (CP, choke <br /> points). According concept [4], load scores                                    n &#226;&#710;&#8217;1 <br /> defined hot spots network based ratio             C c ( v) = <br />                                                                                          &#226;&#710;&#8216; d vi                                                (2) number k-shortest paths passing reac-                              &#226;&#8240;&#160; v  V <br /> tion, number nearest neighbor links attached  ratio compared average load value                                                                  n number reactions network. network. <br /> d) Damage <br />                                                                Eccentricity longest distance given reac- <br />                                                                tion reaction [61]. eccentricity Ce(v) damage defined determine potentially <br /> effected metabolites reactions downstream           vertex v given knocked reaction. used definition damaged <br /> compounds reactions Lemke workers                                           n &#226;&#710;&#8217;1 <br />                                                                       C e ( v) =                   .                                           (3) <br /> [7] yielding features NDR (NDR, number damaged                                  max (d vi ) <br /> reactions) NDC (number damaged compounds).                                   &#226;&#8240;&#160; v  V half reactions annotated revers- <br /> ible, compounds reactions                Eigenvector centrality based assumption inferred damage actually just traced alter-    utility reaction determined utility native pathways.  calculated number        neighboring reactions [62]. scores reaction higher damaged compounds reactions network               connected high-scoring reactions. defined alternative pathways discarded yielding         principal eigenvector adjacency matrix net- <br /> NDRD (number damaged reactions devia-               work. Let xi denote score reaction  Let Aij tions) NDCD (number damaged compounds              adjacency matrix network. Aij = 1 deviations). addition, calculated number                                                                edge reactions j, Aij = 0  <br /> damaged choke points (NDCR, number damaged <br /> choke point reactions; NDCC, number damaged choke           reaction  centrality score proportional point compounds; NDCRD, number damaged choke                average centralities s network neighbors: <br /> point reactions deviations; NDCCD, number                                                                                                                         n <br /> damaged choke point compounds deviations). <br />                                                                                        &#226;&#710;&#8216;                                 x , <br />                                                                              1                                      1 <br /> e) Centrality                                                         xi =                          xj          =              ij j            (4) <br />                                                                              l                                      l features, used network-representation                          j&#226;&#710;&#710;Neighbor                         j =1 <br />  reaction-pair network. computed centrality fea- <br /> tures using R package &apos;igraph&apos; [59] consisting         Neighbor  set neighboring reactions <br /> betweenness centrality (BC), closeness centrality (CN),        reaction  n total number reactions &#206;&#187; eccentricity (EC) [10], eigenvector centrality (EV)        constant. leads directly known eigenvec- <br /> [60]. Let G(V, E) simple undirected graph n ver-     tor equation, Ax = &#206;&#187;x. Normally, different eigen- <br /> tices (reactions). Betweenness measures frequency       values &#206;&#187; eigenvector solution exists. reaction (node) shortest path pairs   According Perron-Frobenius theorem  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                          Page 14 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> eigenvector largest eigenvalue eigenvector            T3s, C3s, A3s, G3s thymine, cytosine, <span id='am-35' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> centrality [60].                                                    <span id='am-108' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-109' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span>, respectively. Additionally, number codons <br />                                                                     coding encoded amino acids (phe, ser, tyr, cys, leu, <br /> Genomic transcriptomic features                                 trp, pro,  arg, gln, ile, met, thr, asn, lys, val, ala, asp, <br /> f) Homologs                                                         glu, <span id='am-91' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-92' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>gly</span>) counted. codon counts normalized calculated number homologous genes                   division total number codons (Nc). Nc taken function knocked               used feature. <br /> gene. Homologous genes searched using BLAST <br /> [26] open reading frames respective              machine learning organism (E. coli, P. aeruginosa, S. typhimurium).               used Support Vector Machines R package <br /> used different E-value cutoffs, e. 10-3, 10-5, 10-7, 10-10, 10-   &apos;e1071&apos; classify essential non-essential genes 20, 10-30 obtain features H3, H5, H7, H10, H20           metabolism http://www.r-project.org. radial basis H30, respectively. Sequences open reading                function used kernel. Parameter optimization <br /> frames taken NCBI database (http://                   performed training data regularization <br /> www.ncbi.nlm.nih.gov/, E. coli: [GenBank:NC_000913],                term kernel width. regularization term <br /> P. aeruginosa: [GenBank:NC_002516], S. typhimu-                 defined costs false classifications opti- <br /> rium: [GenBank:NC_003197]).                                         mized using values 2n n = -4, -2, 0, 2, 4. g) Gene expression                                                  range taken kernel width. optimization collected gene expression data investi-            realized training grid search combi- <br /> gated organisms public resources. datasets                 nations parameters. sizes classes selected respect unspecific regu-          differed considerably data sets (essential genes: 8 - <br /> lation, e. treatments affecting small band         15%, non-essential genes: 85 - 92%). broad spec- broad range metabolic pathways. E. coli, used           trum different sensitivities specificities, applied <br /> gene expression data study regulation           voting scheme. trained 100 Support Vector oxygen deprivation investigated [63], P.             Machines (SVMs) essential genes equal <br /> aeruginosa study observing response agma-             randomly selected non-essential genes. tine <span id='am-86' about='obo:IMR_0200028' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:id" content="IMR:0200028" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Putrescine" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00134" datatype="xsd:string"></span>putrescine</span> treatment [64] study               stratified training data. classification <br /> quorum-sensing response environmental conditions                 query gene, output machines summed <br /> [65]. S. typhimurium used data cells treated              used voting score gene essential nutrient limitation different time points [66]          cell. <br /> data study captured regulatory response environment host [67]. data               Defining discriminating features decision <br /> malized variance stabilization normalization [68].               trees <br /> Genes similar functionality pathway                decision decision trees applies best dis- regulation [69].  maximum               criminating feature [25].  used decision <br /> correlation coefficient (MCC) neighboring reac-              trees alternative approach defining dis- <br /> tions knocked reaction used feature.            criminating features. applied method decision <br /> Additionally, calculated number reactions            trees using R package &apos;rpart&apos; http://www.r-project.org <br /> similar gene expression (NGSE, correlation coefficient &gt;            classify essential non-essential genes metab- <br /> 0.8) used features estimate regulated        olism. Gini impurity [70] used splitting data. <br /> analogous genes.                                                    minimum number observations exist <br /> h) Phyletic retention                                               node 20 maximum depth 30. We selected 177 prokaryotic organisms  E. coli, P.           stratified training data manner aeruginosa, S. typhimurium) described                     training Support Vector Machines. Gustafson et al. [14] counted number            decision trees generated gold standard organisms having open reading frame                  (ecoB, ecoG, paeL paeJ). discriminat- <br /> homologous sequence knocked gene.               ing features, run decision tree performed E. coli, P. aeruginosa, S. typhimu-          selected. <br /> rium using bi-directional best BLAST hits (E-value cutoff <br />                                                                     Performance measures statistics 0.1). <br />                                                                     assessing performance classifier, predic-  Codon usage <br />                                                                     tions compared gold standard exper- <br /> Codons counted investigated gene                                                                     imental screens. prediction true positive (tp, <br /> coding region. counted base compositions silent <br />                                                                     prediction: essential, gold standard: essential), false posi- <br /> sites  position codons) yielding features <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                                                                                                                                                                                                                              Page 15 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br />  <br /> tive (fp, prediction: essential, gold standard: non-essen-                                                                                                                                                                                                                               5.    Yeh  Hanekamp T, Tsoka S, Karp PD, Altman RB: Computational analysis <br />                                                                                                                                                                                                                                                                                                Plasmodium falciparum metabolism: organizing genomic <br /> tial), true negative (tn, prediction: non-essential, gold                                                                                                                                                                                                                                      information facilitate drug discovery. Genome Res 2004, 14:917-924. <br /> standard: non-essential) false negative (fn, prediction:                                                                                                                                                                                                                              6.    Fatumo S, Plaimas K, Mallm JP, Schramm G, Adebiyi E, Oswald M, Eils R, <br /> non-essential, gold standard: essential). calculated                                                                                                                                                                                                                                    Konig R: Estimating novel potential drug targets Plasmodium <br />                                                                                                                                                                                                                                                                                                falciparum analysing metabolic network knock strains standard measures accuracy = (tp+tn)/(tp+tn+fp+fn),                                                                                                                                                                                                                                            silico. Infect Genet Evol 2009, 9:351-358. <br /> sensitivity = tp/(tp+fn), specificity = tn/(tn+fp).                                                                                                                                                                                                                                7.    Lemke N, Heredia F, Barcellos CK, Dos Reis  Mombach JC: Essentiality <br /> receiver operator characteristics (ROC-curve) used                                                                                                                                                                                                                                         damage metabolic networks. Bioinformatics 2004, 20:115-119. <br />                                                                                                                                                                                                                                                                                          8.    Gursoy  Keskin O, Nussinov R: Topological properties protein measure performance classifier                                                                                                                                                                                                                                        interaction networks structural perspective. Biochem Soc Trans <br /> various thresholds. ROC-curve sensitivity                                                                                                                                                                                                                                        2008, 36:1398-1403. <br /> plotted 1 - specificity area                                                                                                                                                                                                                                   9.    Hahn MW, Kern AD: Comparative genomics centrality                                                                                                                                                                                                                                                                                                essentiality eukaryotic protein-interaction networks. Mol Biol <br /> curve (AUC) yields performance estimate                                                                                                                                                                                                                                           Evol 2005, 22:803-806. <br /> entire range thresholds. P-values Pearson&apos;s correla-                                                                                                                                                                                                                              10.   Estrada E: Virtual identification essential proteins protein <br /> tion coefficients features calculated                                                                                                                                                                                                                                          interaction network yeast. Proteomics 2006, 6:35-40. <br />                                                                                                                                                                                                                                                                                          11.   Przulj N, Wigle DA, Jurisica  Functional topology network protein <br /> described [71].                                                                                                                                                                                                                                                                             interactions. Bioinformatics 2004, 20:340-348. <br />                                                                                                                                                                                                                                                                                          12.   Acencio ML, Lemke N: prediction essential genes Additional material                                                                                                                                                                                                                                                                            integration network topology, cellular localization biological <br />                                                                                                                                                                                                                                                                                                process information. BMC Bioinformatics 2009, 10:290. <br />                                                                                                                                                                                                                                                                                          13.   Seringhaus M, Paccanaro  Borneman  Snyder M, Gerstein M: <br />       Additional file 1 ROC curves essential gene predictions                                                                                                                                                                                                                     Predicting essential genes fungal genomes. Genome Res 2006, <br />       subsets features.                                                                                                                                                                                                                                                                     16:1126-1135. <br />       Additional file 2 Correlation coefficients features gene                                                                                                                                                                                                                 14.   Gustafson  Snitkin ES, Parker SC, DeLisi C, Kasif S:       essentiality.                                                                                                                                                                                                                                                                            identification essential genes using targeted genome sequencing <br />       Additional file 3 Histograms frequency T3s essential                                                                                                                                                                                                                       comparative analysis. BMC Genomics 2006, 7:265. <br />       genes non-essential genes.                                                                                                                                                                                                                                                     15.   Hwang YC, Lin CC, Chang JY, Mori H, Juan HF, Huang HC: Predicting <br />       Additional file 4 examined metabolic genes S. typhimurium                                                                                                                                                                                                                         essential genes based network sequence analysis. Mol Biosyst <br />       computational predictions experimental results                                                                                                                                                                                                                       2009, 5:1672-8. <br />       Knuth et al. (2004).                                                                                                                                                                                                                                                               16.   Plaimas K, Mallm JP, Oswald M, Svara F, Sourjik V, Eils R, Konig R: Machine <br />       Additional file 5 Predicted essential genes potential drug targets                                                                                                                                                                                                                   learning based analyses metabolic networks supports high- <br />       S. typhimurium literature evidences.                                                                                                                                                                                                                                       throughput knockout screens. BMC Syst Biol 2008, 2:67. <br />                                                                                                                                                                                                                                                                                          17.   Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br />                                                                                                                                                                                                                                                                                                Tomita M, Wanner BL, Mori H: Construction Escherichia coli K-12  <br /> Authors&apos; contributions                                                                                                                                                                                                                                                                         frame, single-gene knockout mutants: Keio collection. Mol Syst Biol <br /> KP RK general concept design study. KP carried                                                                                                                                                                                                                   2006, 2:2006 0008. data analysis. KP RK drafted manuscript. authors read                                                                                                                                                                                                                18.   Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD, <br /> approved final manuscript.                                                                                                                                                                                                                                                                 Somera AL, Kyrpides NC, Anderson  Gelfand MS, et al.: Experimental <br />                                                                                                                                                                                                                                                                                                determination level analysis essential genes Acknowledgements                                                                                                                                                                                                                                                                               Escherichia coli MG1655. J Bacteriol 2003, 185:5673-5684. work funded BMBF-FORSYS consortium Viroquant                                                                                                                                                                                                                         19.   Jacobs MA, Alwood  Thaipisuttikul  Spencer D, Haugen E, Ernst S, O, <br /> [#0313923]; Nationales Genom-Forschungs-Netz (NGFN+) neuro-                                                                                                                                                                                                                        Kaul R, Raymond C, Levy R, et al.: Comprehensive transposon mutant <br /> blastoma project ENGINE, Commission Higher Education (CHE)                                                                                                                                                                                                                       library Pseudomonas aeruginosa. Proc Natl Acad Sci USA 2003, <br /> Thailand.                                                                                                                                                                                                                                                                                      100:14339-14344. <br />                                                                                                                                                                                                                                                                                          20.   Liberati NT, Urbach JM, Miyata S, Lee DG, Drenkard E, Wu G, Villanueva J, <br /> Author Details                                                                                                                                                                                                                                                                                 Wei T, Ausubel FM: ordered, nonredundant library Pseudomonas <br /> 1Department Bioinformatics Functional Genomics, Institute                                                                                                                                                                                                                            aeruginosa strain PA14 transposon insertion mutants. Proc Natl Acad <br /> Pharmacy Molecular Biotechnology, Bioquant, University Heidelberg, Im                                                                                                                                                                                                                   Sci USA 2006, 103:2833-2838. <br /> Neuenheimer Feld 267, 69120 Heidelberg, Germany 2B080 Division                                                                                                                                                                                                                    21.   Knuth K, Niesalla H, Hueck CJ, Fuchs TM: Large-scale identification Theoretical Bioinformatics, German Cancer Research Center (DKFZ), Im                                                                                                                                                                                                                           essential Salmonella genes trapping lethal insertions. Mol Microbiol <br /> Neuenheimer Feld 280, 69120 Heidelberg, Germany                                                                                                                                                                                                                                                2004, 51:1729-1744. <br />                                                                                                                                                                                                                                                                                          22.   Leon PE: Inhibition ribozymes deoxyribonucleotides Received: 17 December 2009 Accepted: 3 2010                                                                                                                                                                                                                                                origin DNA. J Mol Evol 1998, 47:122-126. <br /> Published: 3 2010                                                                                                                                                                                                                                                                    23.   Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau- <br /> &#194;&#169; 2010 BMC  Open <br />      article <br />       Systems <br />          Plaimas   Access <br />              Biology <br />                 available <br />                    et al; 2010, <br />                              article <br />                           licensee <br />                                 BioMed Central <br />                                      distributed <br />                                   4:56 <br />                                     http://www.biomedcentral.com/1752-0509/4/56 <br />                                                                                                       <br />                                                        terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />                                                                                                                                                                                                                                                                                                Danila  Anderson K, Andre B, et al.: Functional profiling References                                                                                                                                                                                                                                                                                     Saccharomyces cerevisiae genome. Nature 2002, 418:387-391. <br /> 1. Hopkins AL, Groom CR: druggable genome. Nat Rev Drug Discov                                                                                                                                                                                                                       24.   Grocock RJ, Sharp PM: Synonymous codon usage Pseudomonas <br />     2002, 1:727-730.                                                                                                                                                                                                                                                                           aeruginosa PA01. Gene 2002, 289:131-139. <br /> 2. Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD,                                                                                                                                                                                                                       25.   Witten IH, Frank E: Data Mining, Practical Machine Learning Tools     Broadbelt LJ, Hatzimanikatis V, Palsson BO: genome-scale metabolic                                                                                                                                                                                                                       Techniques San Francisco: Morgan Kaufmann; 2005. <br />     reconstruction Escherichia coli K-12 MG1655 accounts 1260                                                                                                                                                                                                               26.   Altschul SF, Madden TL, Schaffer AA, Zhang J, Zhang  Miller W, Lipman <br />     ORFs thermodynamic information. Mol Syst Biol 2007, 3:121.                                                                                                                                                                                                                             DJ: Gapped BLAST PSI-BLAST: new generation protein <br /> 3. Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objective                                                                                                                                                                                                                           database search programs. Nucleic Acids Res 1997, 25:3389-3402. <br />     functions predicting intracellular fluxes Escherichia coli. Mol Syst                                                                                                                                                                                                          27.   Hubbard TJ, Aken BL, Beal K, Ballester B, Caccamo M, Chen Y, Clarke L, <br />     Biol 2007, 3:119.                                                                                                                                                                                                                                                                          Coates G, Cunningham F, Cutts T, et al.: Ensembl 2007. Nucleic Acids Res <br /> 4. Rahman SA, Schomburg D: Observing local global properties                                                                                                                                                                                                                            2007, 35:D610-617. <br />     metabolic pathways: &apos;load points&apos; &apos;choke points&apos; metabolic                                                                                                                                                                                                                28.   Bratkovic T, Lunder M, Urleb U, Strukelj B: <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>Peptide</span> inhibitors MurD     networks. Bioinformatics 2006, 22:1767-1774.                                                                                                                                                                                                                                               MurE, essential enzymes bacterial cell wall biosynthesis. J Basic <br />                                                                                                                                                                                                                                                                                                Microbiol 2008, 48:202-206. <br />  Plaimas et al. BMC Systems Biology 2010, 4:56                                                                                                           Page 16 16 <br /> http://www.biomedcentral.com/1752-0509/4/56 <br />  <br />  <br />  <br /> 29. Kotnik M, Anderluh PS, Prezelj  Development novel inhibitors                    xylulose-5-phosphate reductoisomerase (DXR) enzyme. Curr Pharm <br />     targeting intracellular steps peptidoglycan biosynthesis. Curr Pharm              Des 2007, 13:1161-1177. <br />     Des 2007, 13:2283-2309.                                                        52.   Cornish RM, Roth JR, Poulter CD: Lethal mutations isoprenoid <br /> 30. Helm JS, Hu Y, Chen L, Gross B, Walker S: Identification active-site              pathway Salmonella enterica. J Bacteriol 2006, 188:1444-1450. <br />     inhibitors MurG using generalizable, high-throughput                      53.   Testa CA, Cornish RM, Poulter CD: sorbitol phosphotransferase <br />     glycosyltransferase screen. J Chem Soc 2003, 125:11168-11169.                     responsible transport 2-C-methyl-D-erythritol 31. Zawadzke LE, Norcia M, Desbonnet CR, Wang H, Freeman-Cook K,                         Salmonella enterica serovar typhimurium. J Bacteriol 2004, <br />     Doughert <br /> </body></html>