<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL      BACTERIOLOGY, Mar. 1989,   p.   1631-1637                                                               Vol. 171,  3 <br /> 0021-9193/89/031631-07$02.00/0 <br /> Copyright C) 1989, American Society Microbiology <br />  <br />  <br /> <span id='am-10' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> Sequence Expression cheF, Essential Gene                  Chemotaxis Bacillus subtilis <br />                                            CHINGWEN YING GEORGE W. ORDAL* <br />                        Department Biochemistry, School <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Sciences College Medicine, <br />                                           University Illinois, Urbana, Illinois 61801 <br />                                            Received 23 September 1988/Accepted 9 December 1988 <br />  <br />               cheF gene, involved chemotaxis Bacillus subtilis, cloned, expressed,            sequenced. gene contained 0.7-kilobase PstI DNA fragment isolated Charon 4A <br />            B. subtilis chromosomal DNA library. fragment subcloned expression vector pSI-1 shown <br />            complement cheF mutation chemotaxis methanol production response addition <br />            attractants. Plasmid-encoded DNA expression B. subtilis maxicells indicated membrane-associated <br />            polypeptide 20-kilodaltons expressed 0.7-kilobase DNA. nucleotide sequence DNA <br />            fragment determined, open reading frame capable encoding putative 175-amino-acid protein <br />            (M, 20,002) identified. effort understand function cheF protein, dosage cheF <br />            gene product varied altering concentration IPTG (isopropyl-gB-D-thiogalactopyranoside)            growth. presence high concentrations IPTG, chemotaxis inhibited methanol production <br />            impaired. <br />  <br />  <br />  <br />    Bacterial chemotaxis primitive sensory mechanism                             MATERIALS METHODS bacteria swim higher concentrations attrac- <br /> tant lower concentrations repellent.                      Bacterial strains, plasmids, phages. E. coli strains <br /> knowledge pathway information processing                  JM103 DHaS used propagating plasmids. <br /> bacterial chemotaxis result work                    Strains JM103 NM522 hosts M13 phage <br /> gram-negative bacteria Escherichia coli Salmonella ty-                vectors pTZ phagemids, respectively. Complementation <br /> phimurium (17, 21).  recent experiments (26a)                expression studies performed isogenic strains suggested pathway chemotactic signal processing               B. subtilis strain 011925. plasmid used subcloning quite different gram-positive Bacillus subtilis.          pSI-1 (31). shuttle plasmid contains spac Previous work suggests methyl groups transferred                 promoter lacI gene, shown intermediate carriers methyl-                    function B. subtilis (32). DNA fragments subcloned <br /> accepting chemotaxis proteins (MCPs) form methanol (26,                M13mpl8 M13mpl9 phage vectors nucleotide <br /> 26a). Current evidence suggests methanol evolves di-                 sequence analysis. phagemids, pTZ18U pTZ19U, <br />                                                                           used vectors DNA fragments rectly MCPs E. coli (21). B. subtilis, methanol generated response addition removal attrac-              sequencing study. phages (X-14.9 X-11.7) harboring <br /> tants repellents (26; M. Thoelke G. Ordal, unpub-                 clustered che genes isolated X Charon 4A library <br /> lished data). explore chemotactic mechanism B.                  (19). 7.7-kilobase EcoRI DNA fragment X-11.7 subtilis, number chemotactic mutants isolated               subcloned pFH7 shown capable comple- <br /> characterized (19). 20 complementation groups                   menting cheF mutations (20). cheF mutants defective shown involved (20). mutants motile,                                                                           general chemotactic stimulation identified various <br />                                                                           chemotactic assays (20). 7.7-kilobase EcoRI DNA corresponding genes cheR located                fragment cloned pUC18, restriction region chromosome, thyA pyrD. <br />                                                                           map fragment determined (C. Ying G. Ordal, <br /> Mutation cheF locus results failure respond               unpublished results). chimeric plasmid, designated <br /> chemotactic stimuli (20). region coding cheF                  pGO101, subsequently used subcloning experi- <br /> protein previously localized 7.7-kilobase segment                ment. strains plasmids used listed Table 1. <br /> isolated Charon 4A B. subtilis chromosomal DNA                      Construction B. subtilis strains. chemotactic wild- <br /> library (20). continuing investigation              type strain, 011085, used previous study (29) shown <br /> molecular mechanism involved chemotaxis, cheF gene                 poor host plasmid DNA transformation. strain subcloned expression vector gene prod-                offered reasonable transformational frequency uct identified. procedure allowed detection               crucial subcloning complementation. 011878, subcellular location cheF protein. nucleotide                  derivative 011085, constructed  Table 1 sequence cheF gene determined.                      construction procedure). strain transformed <br /> analyze gene product, dosage cheF protein              plasmid DNAs high frequency. cheFJ41 allele <br /> varied vivo, effects chemotaxis methanol                subsequently recE4 allele transferred production quantitated.                                              011878  constructions, Table 1). strain, 011925, <br />                                                                           used cheF mutant following studies. <br />                                                                              Culture media growth conditions. Unless                                                                           stated, E. coli B. subtilis grown L broth   * <br />       Corresponding author.                                               tryptone plates containing ampicillin chloramphenicol (35 <br />                                                                    1631 <br />  1632        YING ORDAL                                                                                                       J. BACTERIOL. <br />  <br />             TABLE 1. Bacterial strains plasmids used                         plast, methyltransferase, chemotaxis buffers pre- <br />  Bacterial strain                                                                pared according method Ullah Ordal (29). <br />                                Genotype                 Source reference         Transformation. Transformation B. subtilis plas- <br />      plasmid <br />                                                                                  mid DNA performed according method Dub- <br /> Bacterial strain                                                                 nau Davidoff-Abelson (3). method Maniatis et al. <br />   B. subtilis                                                                    (12) used transform E. coli plasmid DNA. <br />      011085           metC trpF7 hisH2 Che+            29 <br />      M1112            leuA8 argi5 thrA recE4           25                           Chemotactic assays B. subtilis strains. Fresh transfor- <br />                         rm- mm                                                   mants picked plates toothpicks  <br />        01363          ilvCI leu-l                      18                        night incubation spotted tryptone swarm plates <br />        011878         leu-l trpF7                      work&apos;                containing S jig chloramphenicol ml. Plates        M141           metC trpF7 hisH2                 19                        incubated wet incubator 5 12 h. swarm <br />                         cheF141                                                  phenotypes colony observed. Capillary assays <br />        IA302          argC4 hisAl recE4                Bacillus Genetic          performed described previously (20). Capillary tubes <br />                                                          Stock Center <br />        011881         trpF7 cheF141                    work&apos;                filled 3 x 10-5 M mannitol 102 M aspartate        011925         trpF7 cheF141 recE4              workc                placed cell suspension. Reactions continued        OI1929         trpF7 cheF141 recE4              work                 45 min 37&#194;&#176;C. Bacteria accumulated inside capillary <br />                         pGO102                                                   tube plated. overnight incubation, number        011933         trpF7 cheF141 recE4              work                 colonies plate counted. <br />                         pSI-1                                                       Changes methanol evolution addition attrac- <br />   E. coli                                                                        tants detected Conway diffusion cells (29). Bacteria <br />     JM103             supE thi lac-proAB)             15 <br />                         (F&apos; traD36 proAB                                         grown L broth harvested cell density <br />                         lacIq lacZAM15)                                          180 Klett units. Cells washed chemotaxis <br />        NM522          hsdA5 lac-proAB)               6                         buffer suspended protoplast buffer density A6. <br />                         supE thi (F&apos; proAB                                       = 1.0. Protoplasts cells prepared adding 1 <br />                         lacl&quot;ZAM15)                                              mg lysozyme ml buffer. protoplasts        DH5a           F- endAl hsdR17                  9                         labeled L-[methyl-3H]methionine 5 min 37&#194;&#176;C                         (rK MK ) supE44 thil                                     mixed 2.3 x 10-2 M aspartate Conway diffusion <br />                         X- recAl gyrA96 Al <br />                         ,80dlacZAM15                                             cells. reaction stopped, methanol <br />                         lacZYA-argF)U169                                       released cells quantitated according method <br />                                                                                  Thoelke et al.  press). <br /> Plasmid                                                                             Plasmid-encoded DNA expression B. subtis maxiceDls. B. <br />   pGO101              Apr                              G. W. Ordal,           subtilis 011925 used host study. Cells                                                           published data         prepared UV irradiated according Shimotsu et al. (23) <br />   pSI-1               Cmr                              33                        modifications. irradiation, cells   pGO102              Cmr; 700-bp PstI               work                 incubated 37&#194;&#176;C 30 min followed addition 20 ,g                         fragment containing <br />                         cheF inserted                                   D-cycloserine ml, incubation continued 16                         pSI-1 pGO101                                        24 h. Cells labeled [35S]methionine lysed <br />   pTZ18U              Apr                       13                               described Hirochika et al. (10). Labeled proteins   pTZ19U              Apr                       13                               visualized autoradiography. <br />   pJH101              Apr Cmr                   4                                   separate membrane fraction cytoplasmic <br />  a011085 competent cells transformed Met&apos; chromosomal               fraction plasmid-encoded proteins maxicells, DNA M1112. transformants high transformation               modified method used. Bacteria grown, irradiated <br /> frequency pSI-1 cotransfornation rM- genetic locus.   UV light, labeled [35S]methionine described strain subsequently  Leu- using chromosomal DNA              Protoplasts cells adding 1 mg 01363. <br />   b cheF141 mutation transferred M141 chromosomal DNA               lysozyme ml cells suspended 1 ml OI1878 competent cells, characteristic chemotactic phenotype     protoplast buffer. Cells incubated 37&#194;&#176;C 45 min cheF141 mutation constructed strain, 011881, checked          harvested. Cell pellets lysed chemotaxis assays.                                                               50 ,ul water 4&#194;&#176;C. Membrane cytoplasm fractions <br />   c <br />     allele recE4 introduced 011881 competent cells chromosomal DNA IA302 gene cong plasmid pSI-1. Colonies          separated centrifugation airfuge 100,000 x g Cmr screened plates containing 0.6 jLl ethyl methane-      45 min 4&#194;&#176;C. membrane fraction precipitated sulfonate 5 iLg chloramphenicol ml. Ethyl methanesulfonate-           pellet suspended 50 RI 1 M NaCl prepared sensistive colonies identified checked swarm plates containing       methyltransferase buffer solution centrifuged  concentration chloramphenicol. colonies showed cheF141 phenotype chosen. Plasmid pSI-1 segre-           45 min 100,000 x g 4&#194;&#176;C. centrifugation, <br /> gated strain. manner, 011925, contained cheFI4l      membrane-associated molecules remained recE4 alleles, constructed.                                              supernatant separated membranes. fraction <br />                                                                                  suspended sodium dodecyl sulfate solubilizer (62 mM <br />                                                                                  Tris, 2% sodium dodecyl sulfate, 10%o glycerol, 5% f- 5 ,ug/ml, respectively) needed plasmid selection.                    mercaptoethanol, pH 6.8), cell debris removed. resuspension medium Sterlini Mandelstam (24)                          Proteins fraction fractionated sodium dode- used maxicell experiments. Media used transfor-                        cyl sulfate-polyacrylamide gels. <br /> mation B. subtilis SPI SPII (3). capillary                         DNA preparation analysis. chromosomal DNA assays, bacteria grown minimal medium supple-                            B. subtilis prepared standard methods. Plasmid <br /> mented 0.2% sorbitol amino acids required                           DNAs isolated E. coli B. subtilis described <br /> growth (20). Tryptone swarm plates mannitol swarm                            Maniatis et al. (12). Restriction endonucleases T4 <br /> plates used described previously (19). proto-                        DNA ligase obtained Bethesda Research Labora- <br />  VOL. 171, 1989                                                 SEQUENCE EXPRESSION cheF B. SUBTILIS                        1633 <br />  <br />                                                                        primer sequencing purchased Bethesda  <br />                                                                        search Laboratories. [35S]<span id='am-5' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dATP</span> supplied Amersham <br />                                                                        Corp. (Arlington Heights, Ill.). chemicals nucle- <br />                                                                        otides sequencing reaction purchased kit <br />                                                                        U.S. Biochemical. <br />                                                                                                  RESULTS <br />                                                                           Subcloning cheF gene. plasmid DNA pGO101 <br />                                                                        digested PstI, liberated DNA fragments. <br />                                                                        Inserts ligated linearized pSI-1. Competent <br />                                                                        cells OI1925 transformed ligated DNA <br />                                                                        mixtures. Transformants selected, swarm <br />   FIG. 1. Complementation cheF mutation pGO102.              phenotype transformants screened. swarm <br /> cheF mutants containing pSI-1   pGO102 (B) spotted tryptone swarm plate supplemented 5 jig         rate cheF mutants shown slow compared <br /> chloramphenicol ml.                                                wild-type strains (20). Che+ transformants                                                                        identified (Fig. 1) basis wild-type swarm <br />                                                                        phenotype, plasmid DNAs prepared tories,  (Gaithersburg, Md.) used recommended               cells. Che+ transformants carried 700-base-pair PstI supplier.                                                       fragment cloned pSI-1. pSI-1 derivative desig- <br />    DNA sequencing techniques. M13mpl8, M13mpl9 (30),                   nated pGO102 (Fig. 2). Cells containing pSI-1 did  phagemids pTZ18U pTZ19U (13, 14) used                  vert Che- phenotype Che+. distinguish vectors sequencing study. nucleotide sequences              Che+ phenotype transformants resulted inserts recombinant clones determined                pGO102 complementation reversion cheFJ41, dideoxynucleotide chain termination method Sanger et al.            following tests performed. Plasmids segregated <br /> (22). Sequenase (U.S. Biochemical Corp.) used              Che+ transformants growing transformants <br /> chain elongation sequencing reactions. 15-mer universal           absence chloramphenicol period time.  <br />                                                                                                  Pstl <br />                                                                                   Pst1                    Pst l <br />                                                                              Pstli <br />  <br />  <br />                                            Pstl <br />                                   spac              lac <br />  <br />                                                           BamHl          EcoR l                          EcoR                                            PSI-1 <br />                            pUBllO                    pBR322 ori <br />  <br />                                             CAT <br />                                                                                                  Pstl <br />  <br />  <br />  <br />  <br />                                                                      Pst 1 <br />                                                                                   st <br />                                                             spac                                                                                        act <br />                                                                   PGOIm       2           amHl <br />  <br />  <br />  <br />                                                                   CAT <br />   FIG. 2. Construction pGO102. 700-base-pair PstI fragment (dotted box pGO101) inserted PstI site pSI-1. solid <br /> box pGO101 represents 7.7-kilobase B. subtilis chromosomal DNA pUC18 vector, shown line. <br />  1634       YING ORDAL                                                                                                                               J. BACTERIOL. <br />  <br />                                                                                              TABLE 2. Chemotaxis various levels cheF gene product                                                   B                                                      IPTG         Swarming           Relative          % Increase <br />                                                                                              Strain        concn           rate         aulatioe             methanol <br />                                                                                                            (mM)           (cm/h)                             production&apos; <br />  <br />                                                              12 34                          011929          0 <br />                                                                                                             0.01 <br />                                                                                                                           0.038 <br />                                                                                                                           0.104 <br />                                                                                                                                              3.0 <br />                                                                                                                                              3.14 <br />                                                                                                                                                                   30 <br />                                                                                                                                                                  ND&apos; <br />   31- <br />                                                                                                             0.1           0.092              6.9                  68 <br />                                                                                                             0.75          0.142              ND                  105 <br />                                                     31-                                                     1.0           0.107             21.5                  86 <br />                    ....                                             .:&apos;  <br />                                                                                                            10.0           0.106             13.4                  35 <br />                                                                                                           100.0           0.105              3.0                 ND <br />                    m2                                                                       011925          0             0.02               0.7                  10 <br />                                                                                             011878          0             0.24              23.1                 125 <br />                     .........~.                                       c <br />                                                                                               Expressed number cells accumulated capillary tube containing <br />                                                                                             3 x 10-&apos; M mannitol/number accumulated tube containing buffer.                                                                                             value mean replications. <br />                                                                                               b generated addition 2.3 x 10-2 M aspartate versus                                                                                             generated buffer added. <br />                                                                                               C ND, determined. <br />   FIG. 3. Expression localization cheF gene product. <br />   Plasmid-encoded polypeptides p00102 (lane 1)                             pSI-1 <br /> (lane 2) expressed B. subtilis maxicells. Strains containing                        possible band represented modified form p00102           pSI-1         UV irradiated labeled [355]          major protein. <br /> methionine. Proteins   fractionated 12.5% sodium dodecyl                             localize cheF protein, labeled maxicells sulfate-polyacrylamide gel visualized autoradiography.                             ruptured fractionated membrane, membrane-associ- <br /> 250-ng protein loaded lane. Molecular mass                            ated, cytoplasm fractions ultracentrifugation. standards  kilodaltons) shown left. (B) cheF protein                      cheF gene product behaved like membrane-associated labeled UV programming. Extracts separated                                                                                             protein, removed membrane fraction <br /> cytoplasm (lane 1) membrane (lane 2) fractions ultracen- <br /> trifugation. membrane fraction lane 2 washed                                                                                             salt wash membranes (Fig. 3B). <br /> 1 M NaCl centrifuged. centrifugation, cheF <br />                                                                                                Expression level cheF product chemotaxis. protein appeared supernatant fraction (lane 3)                               study effect altered cheF expression chemotaxis, <br /> removed membrane fraction (lane 4). Methods separat-                           gene placed control spac pro- <br /> ing fraction described Materials Methods. Approx-                           moter, regulated addition imately 350 ng protein loaded lane; molecular mass                           inducer, IPTG. level cheF gene product standards  kilodaltons)                shown left.                            altered vivo exposing cells various concentra- <br />                                                                                             tions IPTG swarm plates. <br />                                                                                                absence inducer, slow swarming rate was    cells <br />                            regained swarm phenotype,                                    conferred plasmid (Table 2). swarming rate      indistinguishable cheF mutants. <br />                                                                                         cells reached maximum 0.75 mM IPTG 011925 retransformed p00102 plasmid DNA,                                       decreased. expected, IPTG did affect swarming swarm phenotype transformants examined.                                      wild type (OI1878).  swarming rate inhib- <br /> transformants checked                       Che&apos;.     confirm          ited overproduction cheF protein.   Pstl   DNA         fragment complement                cheF mutations,             investigate effect elevated dosage chemotaxis assay                        used. Chemotaxis 011925              cheF protein behavior, capillary assay used. 011929, contained                       p00102,             analyzed    Cells grown various IPTG concentrations    capillary   assay. assay,                   011925        011929    washed buffer assay performed. Bacte-     grown              presence         1   mM      IPTG       (isopropyl-   ria responded mannitol poorly grown  13-D-galactopyranoside) induce expression plasmid-                               medium IPTG (Table 2). responded best encoded   polypeptides (32). 011925 showed poor taxis                                  grown 1 mM IPTG. maximal accumulation attractant aspartate capillary assay (20).                                achieved wild-type control, 011878. relative accumulation 011925 10-2 M aspartate                                    observed high IPITG concentration medium <br /> 1.49. 011929, hand, responded concen-                                   impaired chemotaxis OI1929. expression level trationgradient aspartate did wild type.                                  cheF showed similar effect capillary assays aspartate <br /> 011925 011878 (wild type) showed relative accumulation                                  (data shown). <br /> values 11.5 10.9, respectively, concen-                                    effect altered cheF expression levels methanol <br /> tration aspartate.                                                                       production explored. Labeled cells   Identification localization cheF gene                            product    grown presence different IPTG concentrations using   B. subtilis maxicells. established cheF                             mixed aspartate Conway diffusion cells. metha- <br /> structural gene              contained        pG0102.       determine         nol released cells result aspartate stimulation <br /> size cheF  protein, plasmid-encoded polypeptides                                 quantitated (Table 2). optimal expression level expressed p00102 vivo using B. subtilis                                     cheF gene product methanol formation occurred maxicells (Fig. 3). Strains 011925, 011933, 011929                                 0.75 mM IPTG. methanol evolved IPTG concen- <br /> UV irradiated, newly synthesized proteins labeled                                  tration 80% evolved wild type (011878).    [35S]methionine              described Materials             Methods..    effect elevated cheF protein dosage methanol  predominant protein band unique p00102 ob-                                      production similar accumulation served molecular weight 20 kilodaltons (Fig. 3A).                                 capillary assay. <br /> second faint, slightly larger protein band observed;                                  Nucleotide sequence cheF gene. PstI DNA <br />                                                                                  Pst1          BgIll Oral          Dral Hindlil Pstl <br />                                                                                 momiA <br />  <br />  <br />  <br />                                                  4- <br />  <br />  <br />                                                                 4- <br />  <br />                                                                            .4 <br />                B                                                      Met Ser Asn Ile Ser <br />                                                                                                                        50 <br />                                   CTGCAGTTAACGTACTAAAAAGCATCAGCAGGGTG ATG TCA AAC ATT TCA <br />                                   PstI <br />                                                                                101 <br />                Thr Lys Gln Leu Asp Ala Ala Asn Lys Phe Arg Glu Leu Leu Ser Thr <br />                ACA AAA CAG CAT CTG GAT GCG GCA AAT AAA TTC CGT GAA CTT TTA TCG ACA <br />                                                                                                                      152 <br />                Tyr Gln Asn Pro <span id='am-25' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-26' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>Gly</span> ASp Leu Phe Asn Ile Gly Ala Tyr Lys Arg Gly Ser <br />                TAC CAA AAT CCG GGA GAT CTC TTT AAT ATC GGA GOT TAO AAA AGA GGA TCA <br />  <br />                                                                                                                      203 <br />                Ser Arg Glu Ile Asp Ala Ala Ile Gln Phe Tyr Pro Gln Leu Ile Gln Phe <br />                TCC AGA GAA ATT GAT GCA GCG ATA CAA TTT TAT CCG CAG CTT ATT CAA TTT <br />  <br />                                                                                                                      254 <br />                Leu Lys Gln Gly Thr Asp Glu Pro Ala Leu Leu Glu Glu Ser Ile Ala Ala <br />                TTA AAG CAG GGA ACA GAC GAA CCG GCT TTA TTG GAA GAA AGC ATT GCT GCA <br />                                                                                                                      305 <br />                Lys Thr Ser Leu Thr Gly Asn Glu Glu Glu Thr Trp Leu Asn Gln Phe Tyr <br />                AAA ACT AGT TTG ACA GGA AAT GAG GAA GAA ACG TGG CTT AAT CAA TTT TAC <br />                                                                                                                      356 <br />                Pro leu Ala Lys Ser Cys Trp Asn Tyr Lys Lys Met Lys Lys Lys Asn Pro <br />                CCA TTG GCC AAA AGC TGC TGG AAC TAC AAG AAA ATG AAA AAG AAA AAT CCT <br />                                                                                407 <br />                Tyr Pro Asn Ile Ser Asn Gln Phe Leu Asn Trp Lys Thr Leu Leu Leu Lys <br />                TAT CCG AAT ATC AGC AAT CAG TTT CTG AAT TGG AAA ACG TTG CTT TTA AAA <br />                                                                                                                      458 <br />                Leu Tyr Glu Asn Met Ser Lys Lys Glu Leu Ala Gly Thr Lys Ile Arg Lys <br />                TTA TAT GAG AAT ATG AGC AAA AAG GAA TTG GCT GGA ACC AAA ATA AGG AAA <br />                                                                                                                      509 <br />                Arg Asn Ser Lys Ala Ala Gly Val Tyr Lys Lys Val GLy Thr Thr Asn Asn <br />                AGA AAT TCA AAA GCG GCA GGA GTG TAC AAG AAA GTA GGC ACT ACC AAC AAT <br />                                                                                                                      560 <br />                Leu Ser Gln Ile Leu Ile Ile Arg Ser Ile Thr Ile Lys Ser Leu Ser Leu <br />                TTG TCT CAA ATC TTG ATA ATA CGA TCT ATC ACT ATC AAA AGC TTG TCA TTA <br />                                                                                                                      627 <br />                TGATGAAAAGAAATCAAATGAAATCAAAAAACAAGAAATTTTGACAGAGAAAAACATTGAAGTGAAA <br />                                                                                                             688 <br />                AAGTTTGAAAAAATGCGTGAAAAACAGTTTAAAATGTTTGCTCTTGAAGATAAAGCTGCAG <br />                                                                                                           PstI <br />   FIG. 4. Sequencing strategy 700-base-pair PstI fragment nucleotide sequence cheF gene.   Restriction sites used sequencing experiments. fragments inserts (*) constructed M13 phages; phagemids used vectors fragments. (B) Nucleotide sequence. Nucleotides numbered taking position nucleotide PstI site +1. asterisks mark proposed translational stop codon. <br />  <br />                                                                    1635 <br />  1636     YING ORDAL                                                                                                J. BACTERIOL. <br />  <br /> fragment containing cheF gene sequenced (Fig. 4).          ribosome-binding site cheF gene lower analysis DNA sequence revealed open             average calculated free energy interaction 3&apos; end <br /> reading frame (ORF) 528 bases (Fig. 4B).          16S RNA. Taken  results imply potential initiation codons 5&apos; end ORF.       low levels cheF proteins required cells perform codons frame reading frame                chemotaxis normally. cheF product overex- <br /> identified cheF gene. potential ribosome-binding         pressed presence high concentrations IPTG, <br /> site located base region ATG codon        chemotaxis methanol production impaired. position 36. seven bases (AGXAGGG) bases          20 chemotaxis proteins, cheF ATG codon complementary 3&apos;-OH end            protein titrate causing inhibition. 16S rRNA (8). basis rules Tinoco et al.          hydropathy profile cheF protein indicates (27) Borer et al. (1), calculated AG interaction   C-terminal protein hydrophobic region. -14.2 kcal (ca. -59.4 KJ)/mol. GTG codon                 region short membrane spanning, position 33,  preceded recognizable            results suggest cheF protein membrane associated <br /> ribosome-binding site.  ATG codon appears            normal conditions removed likely translational initiation codon. ORF     membrane high-salt wash. cheF protein capable encoding 175-amino-acid polypeptide         calculated pI 9.81, basic nature protein calculated molecular weight 20,002, agreement       apparent <span id='am-18' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> sequence. molecular weight polypeptide expressed                proteins MCPs involved chemotaxis B. subtilis <br /> pGO102 maxicell experiments. confirm            characterized methyltransferase ORF encodes functional cheF protein, mutations            methylesterase. proteins methylate demethylate, generated shifting reading frame pre-           respectively, MCPs vitro. proteins sumed coding region. pGO102 digested BglII,               basic adhere membranes (2, 5). remains located putative ORF. Klenow fragment          determined cheF protein proteins used remove sticky ends.      interact membrane. <br /> new hybrid plasmids introduced 011925.              deduced amino acid sequence cheF protein <br /> complementation plasmids observed.            compared sequences proteins Furthermore, frame translational fusion cheF pro-        GenBank (E. Kofoid J. S. Parkinson, personal commu- <br /> tein P-galactosidase successfully constructed. West-        nication). significant homology cheF protein <br /> ern blot (immunoblot) experiments confirmed fusion,        known proteins observed. cheF mutants fused protein purified (C. Ying G. Ordal,              impaired release methanol caused addition manuscript preparation).                                        attractant removal repellent (20). strong <br />    deduced amino acid sequence cheF gene                evidence methyl groups released directly product appeared include large group lysine residues,      B. subtilis MCPs instead transferred  methyl 21 lysines 175 amino acids; pl value cheF protein     methoxy groups) carrier(s) (26, 26a).  9.81 calculated. hydropathy profile           fore, tempting speculate actual substrate duced cheF amino acid sequence (11) suggests cheF         methyl carriers. clear cheF product <br /> protein composed quite different portions.       functions pathway unique B. subtilis, sequence characterized long stretches hydrophilic     E. coli. <br /> residues, carboxyl-terminal end contains short          Determination substrate cheF protein hydrophobic region (159LSQILIIRSITIKSLSL175).             elucidation mechanism associates shown cheF protein appears membrane             membrane possible protein puri- <br /> associated, possible means interaction            fied. able express protein cheF protein membrane: cheF                relatively high level control spac pro- <br /> protein contact membrane hydrophobic         moter, attempting raise antibodies carboxyl terminus use lysines bind          cheF protein aid process analyzing role anionic residues membrane proteins.                       protein molecular mechanism chemotaxis. <br />  <br />                          DISCUSSION                                                     ACKNOWLEDGMENTS <br />                                                                      thank D. Henner gift pSI-1 J. Kaput valuable <br />   paper reports nucleotide sequence gene       suggestions course investigation. thank required chemotaxis B. subtilis. cheF gene      Aamir Zuberi help preparation manuscript. Comparison <br /> cloned expression plasmid complements cheF               deduced cheF protein proteins GenBank  <br /> mutation chemotactic deficiency normal        formed Eric Kofoid laboratory J. S. Parkinson. <br /> methanol production response addition attrac-           work supported Public Health Service grant Al 0336 <br /> tants.  chemotaxis methanol formation result-         National Institutes Health National Science Foun- <br /> ing stimulation attractants reach optimal         dation grant DCB 85-01604. <br /> levels cheF gene product induced 0.75                                LITERATURE CITED <br /> mM IPTG. results evidence close relation- <br /> ship methanol production chemotaxis B.               1. Borer, P. N., B. Dengler,  Tinoco, Jr., 0. C. Uhlenbeck. <br /> subtilis.                                                              1974. Stability ribonucleic acid double-stranded helices. J. <br />    Analysis nucleotide sequence cheF gene                Mol. Biol. 86:843-853. <br />                                                                     2. Burgess-Cassler, ,  H. J. Ullah, G. W. Ordal. 1982. <br /> suggests gene capable encoding protein          Purification characterization Bacillus subtilis methyl- molecular mass 20 kilodaltons, agreement              accepting chemotaxis protein methyltransferase 11. J. Biol. <br /> finding protein 20 kilodaltons B. subtilis maxicells.       Chem. 257:8412-8417. cheF gene probably expressed low levels vivo,       3. Dubnau, D., R. Davidoff-Abelson. 1971. Fate transform- <br /> indicated codon usage gene (7). potential          ing DNA following uptake competent Bacillus subtilis. 1. <br />  VOL. 171, 1989                                                    SEQUENCE EXPRESSION cheF B. SUBTILIS                         1637 <br />  <br />       Formation properties integrable plasmid Bacillus         cloning chemotaxis genes. J. Bacteriol. 154:1088-1097. <br />       subtilis. J. Mol. Biol. 56:209-221.                                20. Ordal, G. W., H. M. Parker, J. R. Kirby. 1985. Comple- <br /> 4.    Ferrari, F.  <br /> </body></html>