<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Biotechnology                                                                                                                        BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access <br /> Plasmid selection Escherichia coli using endogenous essential <br /> gene marker <br /> Shan Goh1 Liam Good*1,2 <br />  <br /> Address: 1Department Cell Molecular Biology, Karolinska Institute, Stockholm, SE-17177, Sweden 2Department Pathology Infectious Diseases, Royal Veterinary College, University London, AL9 7TF, UK <br /> Email: Shan Goh - shan.goh@ki.se; Liam Good* - lgood@rvc.ac.uk <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 11 August 2008                                                     Received: 4 April 2008 <br />                                                                               Accepted: 11 August 2008 <br /> BMC Biotechnology 2008, 8:61   doi:10.1186/1472-6750-8-61 article available  http://www.biomedcentral.com/1472-6750/8/61 <br /> &#194;&#169; 2008 Goh Good; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Antibiotic resistance genes widely used selection recombinant bacteria, <br />                  use risks contributing spread antibiotic resistance. particular, practice                  inappropriate intrinsically resistant bacteria vaccine production, costly                  industrial scale production. Non-antibiotic systems available, require mutant host strains, <br />                  defined media expensive reagents. unexplored concept expression host essential <br />                  gene enable selection presence <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> inhibitor gene product. test                  idea E. coli, used growth essential target gene fabI plasmid-borne marker                  biocide triclosan selective agent. <br />                  Results: new cloning vector, pFab, enabled selection triclosan 1 &#206;&#188;M. Interestingly, pFab <br />                  performed parent pUC19-ampicillin cell growth, plasmid stability plasmid <br />                  yield.  pFab toxic host cells way reversed triclosan.  pFab <br />                  triclosan toxic used combination enhance growth plasmid <br />                  production gene-inhibitor interaction. <br />                  Conclusion: fabI-triclosan model provides alternative plasmid selection method <br />                  based essential gene expression, use antibiotic-resistance genes                  conventional antibiotics. <br />  <br />  <br />  <br />  <br /> Background                                                                    tem avoid antibiotic resistance markers thera- <br /> Antibiotic resistance marker genes commonly used                       peutic antibiotics, cost effective, convenient, <br /> select maintain recombinant bacteria presence                      robust flexible. antibiotics.  use antibiotics undesira- <br /> ble manufacturing gene therapy products [1,2].                       variety strategies antibiotic-free selection introduction antibiotic resistance genes bio-                     developed, widely used bacteria. hazardous strains recommended [3], antibiotic                      reported non-antibiotic involves auxo- <br /> selection fails bacteria naturally resistant [3,4].               trophic bacterial strain complementation using Finally, use antibiotics costly industrial                   plasmid-encoded biosynthesis gene, trans- <br /> scale production, especially case enzymatic                         formants grow defined media lacking nutrient [5]. <br /> depletion antibiotic culture. alternative sys-                   termed repressor-titration, lac <br />  <br />  <br />                                                                                                                                          Page 1 9 <br />                                                                                                                  (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                   http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> operator functions vector-borne selection marker,          pBAD18s (National Institute Genetics, Japan). Media represses modified chromosomal essential               SOC LB (GIBCOBRL) supplemented <span id='am-27' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>- <br /> gene [6]. general approach alter expres-        icillin (LBA, 100 &#206;&#188;g/ml ampicillin, (Sigma)), triclosan <br /> sion growth essential genes using synthetic [7]             (LBT, 1 &#206;&#188;M triclosan; Ciba) arabinose (Sigma). Tri- <br /> expressed antisense sequences [8]. Unfortunately,           closan used 1 M stock DMSO. existing non-antibiotic systems suffer need mutant host strains expensive reagents,        Construction pFab, pUCFA pBFab cases low efficiency.                                        EagI site created <span id='am-39' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-40' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> position 1621                                                                   pUC19 (New England Biolabs), immediately downstream observed expression growth             bla, PCR primers (5&apos;cgtcggccgttaccaatgcttaatcag <br /> essential gene results reduced susceptibility gene      5&apos;cgccggccggaccaagtttactcatat). amplicon product inhibitor [9]. considered strategy based        digested EagI (New England Biolabs), ligated T4 essential gene plasmid-borne marker            DNA ligase transformed DH5&#206;&#177; propagation. <br /> specific protein level inhibitor selective agent.       construct pFab, bla gene excised pUC19 <br /> test idea, chose fabI-triclosan model combina-        SspI EagI, replaced fabI tion studied gene-inhibitor pair: FabI       promoter, amplified K12 genomic DNA <br /> (enoyl ACP reductase) catalyzes fatty acid elongation [10]        using primers 5&apos;ccggatatcgtgctggagaatattcg confers reduced susceptibility triclosan         5&apos;gcgcggccgttatttcagttcgagttcgtt. amplicon expressed E. coli [11,12]; triclosan inhibits FabI     digested EcoRV EagI used create pFab <br /> binding ACP-enoyl substrate site, forming stable         vector. E. coli DH5&#206;&#177; transformed pFab FabI/NAD+/triclosan ternary complex [13]. addition,            plated LB 0.5 &#226;&#8364;&#8220; 5 &#206;&#188;M triclosan. Transformants <br /> triclosan biocide fulfils criteria non-anti-   subsequently maintained 1 &#206;&#188;M triclosan (LBT). <br /> biotic [14]. stable, easy handle, inexpensive        fabI gene cloned pUC19 SphI approved use hygiene, household indus-            BamHI MCS (pUCFA) similar way using <br /> trial applications [15,16].                                       primers 5&apos;ccggcatgcgtgctggagaatattcg 5&apos;ccggatc- <br />                                                                   cgattatttcagttcgagt. Competent E. coli DH5&#206;&#177; trans- triclosan used systemic therapeutic,      formed pUCFA plated LBA LBT. important consider possible risks associated new applications. importantly, concern         induce expression fabI PBAD, fabI amplicon <br /> triclosan use contribute antibiotic resistance [17];       generated primers (5&apos;cggaattcgaatgggttttctttccgg  resistant mutants generated labora-         5&apos;cctctagagattatttcagttcgagt) digested EcoRI tory [18-20].  studies bacteria non-labora-         XbaI (New England Biolabs) cloned pBAD18s, <br /> tory conditions exposed biocide-concentrations              similarly digested, yield pBFab1. Expression <br /> triclosan did correlation antibiotic           fabI pBAD18 required Shine Dalgarno sequence, <br /> resistance decreased triclosan susceptibility [21-23].        predicted uaagga position -13 relative <br /> Regulatory agencies continue approve use tri-           start codon. Primers (5&apos;cggaattctcaacaataaggat- <br /> closan domestic clinical products [24,25],          taaagc 5&apos;cctctagagattatttcagttcgagt) used appears comparatively safe choice selection          amplification fabI Shine Dalgarno sequence, <br /> agent.                                                            cloned pBAD18 yield pBFab6. E. coli DH5&#206;&#177; <br />                                                                   transformed pBFab plasmids plated study, tested fabI-marker plasmid selection         LBA, LBT LBT 0.2% (w/v) arabinose. <br /> triclosan observed efficient stable selection. new performed parent antibiotic          Plasmid transformant properties <br /> growth plasmid production presence tri-             Transformation efficiencies chemically competent <br /> closan. Cells containing fabI-marker plasmid dis-             DH5&#206;&#177; cells determined recommended played toxic effects absence triclosan, suggesting      manufacturer (Invitrogen). &quot;addictive&quot; effect, aid plasmid contain- <br /> ment.                                                             Plasmid yields determined clones                                                                   pUC19 pFab. Plasmids isolated ml <br /> Methods                                                           overnight cultures grown selection using mini- <br /> Bacterial strains, plasmids media                             prep kit (Qiagen) quantified OD260 readings. Plas- E. coli strains used study DH5&#206;&#177; (Invitro-        mids (50 ng) digested BamHI gen), XL1-Blue (Strategene), HB101, BL21 (Strategene)             electrophoresed 1% agarose gel. K12 (Coli Genetics Stock Center, Yale U.). Plasmids pUC19 (New England Biolabs), pBAD18  <br />  <br />                                                                                                                        Page 2 9 <br />                                                                                                (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> Plasmid stability assayed presence        determined plasmid extraction, digestion 100 ng <br /> absence selection. Overnight cultures grown           DNA BamHI fractionation 1% agarose gel. <br /> selection 37&#194;&#176;C shaking diluted 1000 &#195;&#8212; 5 <br /> ml LB selection, aliquots time      Inducible expression fabI <br /> zero cultures diluted plated LB plates       Clones pBFab1 pBFab6 grown LBA 16 <br /> taining X-Gal (20 &#206;&#188;g/ml, Saveen). time zero cultures       h, diluted approximately 5 &#226;&#8364;&#8220; 9 &#195;&#8212; 106 cfu/ml LBT grown diluted 24 48 h, aliq-      aliquots 180 &#206;&#188;l added wells 96 plate. <br /> uots 48 h cultures plated. ratios blue      Arabinose added final concentration 0 &#226;&#8364;&#8220; 5% <br /> colonies total colonies LB plates X-Gal (20 &#206;&#188;g/     final volume 200 &#206;&#188;l ml) determined 0 h 48 h, % plas-        water. Clones pBAD18 pBAD18s included mid-bearing cells 48 h calculated. independ-      controls. Cultures grown 24 h VERSAmax <br /> ent clones pUC19 pFab used provide              spectrophotometer (Molecular Devices) agitation replicates.                                                    5 s 5 min, followed OD550 reading. growth <br />                                                                rate arabinose concentration calculated Plasmid abundance determined ways.         described  <br /> compare band intensities genomic (gDNA) plasmid <br /> DNA (pDNA) agarose gel, total genomic DNA            Cell viability <br /> extracted different clones pUC19 pFab         DH5&#206;&#177;/pFab grown overnight absence pres- <br /> clones grown selection 16 h. cultures        ence 0.5 &#226;&#8364;&#8220; 2 &#206;&#188;M triclosan subjected SYTOX green <br /> K12, derived single colonies, grown LB       staining flow cytometry, previously described [29]. DNA extraction using Bacterial GenElute         DH5&#206;&#177;/pUC19 included control determine <br /> (Sigma) ml overnight culture. Total DNA (10         staining levels live heat-treated dead cells. Samples <br /> &#206;&#188;l) electrophoresed 1% agarose gel, stained      excited 488 nm air-cooled argon <span id='am-16' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ion" datatype="xsd:string"></span>ion</span> laser ethidium bromide scanned using Typhoon 9400              CyFlow SL flow cytometer (Partec GmbH). Threshold <br /> (Amersham Biosciences). Band intensities deter-           settings enabled forward scatter exclude cell <br /> mined using ImageQuant software (Amersham Bio-              debris. forward scatter dot plot used sciences). Second, relative quantitative PCR (qPCR)        identify gate cell populations. Fluorescence meas- <br /> carried using plasmid lacZ&#206;&#177; gene target      ured 520 nm. Viable dead cell populations gene single copy chromosomal dxs reference          counted using Partec, FloMax software version 2.4e. <br /> gene [26]. K12 gDNA containing single copy lacZ&#206;&#177; dxs used calibrator. Primers amplifying       Growth competition <br /> target gene (5&apos;gtgctgcaaggcgattaagtt 5&apos; cactggccgtcgttt-   Growth competition DH5&#206;&#177; plasmid bearing <br /> tacaa), reference gene (5&apos;cgagaaactggcgatcctta         cells carried previously described, modifi- <br /> 5&apos;cttcatcaagcggtttcaca) validated similar amplifi-    cations [30]. Overnight (24 h) cultures DH5&#206;&#177;, DH5&#206;&#177;/ <br /> cation efficiencies. Real time data analyses carried      pUC19 DH5&#206;&#177;/pFab prepared LB, LBA 2-&#206;&#8221;&#206;&#8221;CT method relative qPCR [27]. Total         LBT, respectively. Equal volumes DH5&#206;&#177; plasmid <br /> DNA concentrations determined OD260 absorb-            bearing cultures mixed diluted 1:100 10 ml <br /> ance qPCR. 25 &#206;&#188;l PCR reaction contained 12.5       fresh LB growth. aliquot diluted <br /> &#206;&#188;l SYBR Green PCR buffer (Eurogentec), 100 nM            mixed culture simultaneously plated selective primer (Biomers) 5 ng total DNA.                      non-selective media cell counts. mixed <br />                                                                culture incubated 24 h shaking 37&#194;&#176;C, Growth rates calculated exponential phase        diluted 1:100 10 ml fresh LB growth growth [28], monitored increased OD550         plated,  procedure repeated 5 times time VERSAmax spectrophotometer (Molecu-           days. numbers ampicillin triclosan <br /> lar Devices). overnight culture (16 h), standardized     resistant colonies scored relative total CFUs. <br /> OD550 yield approximately 7 &#195;&#8212; 105 cfu/ml, grown 200 &#206;&#188;l volumes 96 plate 24 h    Results <br /> agitation 5 s 5 min, OD550 readings        Vectors containing fabI enable triclosan selection <br /> taken. Triclosan added 0 &#226;&#8364;&#8220; 2 &#206;&#188;M Triclosan      test potential fabI selective marker clon- <br /> 1% DMSO final concentration.                                   ing, vectors derived pUC19 constructed. <br />                                                                 constructed pFab, fabI host range pFab commonly used E. coli            native promoter replaces ampicillin resistance gene <br /> cloning strains tested transformation XL1-Blue       (bla) pUC19. Second, enable selection triclosan <br /> (Stratagene), HB101 BL21, followed selection         ampicillin, constructed pUCFA, contains LBT plates. Plasmid DNA integrity abundance            fabI cassette cloned pUC19 MCS (Figure 1A). E. <br />  <br />  <br />                                                                                                                      Page 3 9 <br />                                                                                              (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                                                                                                                   http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               Vector <br />                                                                                                                                                                                 Figure construction <br />                                                                                                                                                                                          1               triclosan selection <br />                                                                                                                                                                                 Vector construction triclosan selection.                                                                                                                                                                                   bla gene pUC19, confers ampicillin resistance,                             EcoRV                                    AatII                                                           BamHI                                      replaced fabI promoter region (pFab).                                      Pbla <br />                                    SspI                                                                                                                                         pUC19 multiple cloning site (MCS) retained, Hin- <br />                                                                                                     lacZa <br />                                                                                                      BamHI                                                                      cII, HindIII PstI unique pFab. fabI cassette    (triclosan resistance) <br />  <br />  <br />  <br />  <br />                                            (ampicillin resistance) <br />  <br />  <br />  <br />  <br />                                                                                                         MCS                                                                     pFab transferred pUC-derived plasmids using <br />  <br />  <br />  <br />  <br />                                                                                                                                                    fabI <br />                                                                                                        SphI <br />                                                                      pUC19 <br /> fabI <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 AatII AlwNI restriction sites. fabI cassette                                      bla <br />  <br />  <br />  <br />  <br />                                                                                                        Plac <br />  <br />                                                                      2686 bp                                                                                                    inserted MCS pUC19 obtain pUCFA.                                                                                                                                                                                 plasmids available authors. (B) Growth pFab <br />                                                                                                                                    SphI                                         pUCFA clones LBT LBA plates. (C) Plasmids <br />                               EagI     EagI                                                  ori                                                                                propagated different E. coli hosts digested                                                                                           AlwNI <br />                                                                                                                                                                                 BamHI analyzed gel electrophoresis. <br />  <br />                                    AatII                                                                                                 AatII <br />                                                                                   lacZa                              Plac                                                       coli strain DH5&#206;&#177; transformed pFab, pUCFA                                                                                      BamHI                                                                         BamHI <br />                                                                                            MCS <br />                                                                                                                                                                                 pUC19 transformants selected LB plates  <br />                                                                                                           bla <br />                                                                                            SphI                                                                                 taining triclosan (LBT) ampicillin (LBA). antici- <br />                                 pFab                                                        Plac                               pUCFA                                            pated, fabI inserts enabled selection triclosan <br />   fabI <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          fabI <br />  <br />  <br />                                2912 bp                                                                                         3771 bp                                          containing plates. Colonies formed LBT                                                                                                                                                                                 variable size colonies LBA.  triclosan- <br />                                                                                                          AlwNI <br />                                                                                                                                                                                 resistant colonies sizes maintained resistance (Figure <br />                                                                                ori                            ori                                                 SphI <br />                             EagI                                             AlwNI                                                                               Plac           1B) displayed uniform colony morphologies  <br />                                                                                                                                                                                 plating. contrast pUC19, did observe plas- <br />                                                                                                                                                                                 mid-free colonies satellite colonies using pFab <br /> B                                                                                                                                                                               pUCFA.  fabI-triclosan enables <br />                                                                                                                                                                                 non-antibiotic selection maintains stable recom- <br />                                                                      DH5&#206;&#177;/                                                                 DH5&#206;&#177;/ <br />                                                                                                                                             DH5&#206;&#177;/                               binant strains. <br />                             DH5&#206;&#177;                                     pUC19                               DH5&#206;&#177; <br />                                                                                                          DH5&#206;&#177;                              pUC19 <br />                                                                                                                                             pUC19 <br />                                                                                                                                                                                 test fabI-triclosan selection function <br />                                                                                                                                                                                 E. coli strains, pFab transformed                                                                                                                                                                                 strains BL21, HB101 XL1-blue. Similar results <br />                                                                                                                                                                                 DH5&#206;&#177;, observed efficient selection high yield <br />                                                                                                                                                                                 plasmid production (Figure 1C). test fabI                                                                                                                                                                                 enable triclosan selection vectors, pFab cas- <br />                             DH5&#206;&#177;/ DH5&#206;&#177;/                                                                  DH5&#206;&#177;/ DH5&#206;&#177;/                                                            sette inserted multiple cloning site pGEM- <br />                             pFab pUCFA                                                                   pFab pUCFA                                                             3Zf used replace ampicillin resistance gene <br />                                                                                                                                                                                 low copy number vector pBR322. cases, tri- <br />                             Triclosan (LBT)                                                              Ampicillin (LBA)                                                       closan resistant colonies selected (data shown). <br />                                                                                                                                                                                  fabI-triclosan enables efficient selec- <br />                                                                                                                                                                                 tion commonly used vectors E. coli strains. <br />  C <br />                                                                                                                                                  XL1-Blue/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 confirm fabI expression main mechanism <br />                                                                              DH5&#206;&#177;/pUC19 <br />  <br />  <br />  <br />  <br />                                                                                                                             HB101/pFab <br />                                                                                              DH5&#206;&#177;/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 mediating triclosan resistance point mutations <br />                                                                                                          BL21/pFab <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                 chromosomal fabI [12], expression fabI                                                                                                                                                                                 placed control PBAD promoter                             kb                                                                                                                                                  pBAD18s (pBFab1) pBAD18 (pBFab6) [31].                                                                                                                                                                                 pBFab1 pBFab6 strains tested resistance fol- <br />                             3.5                                                                                                                                                 lowing fabI induction. absence inducer arab- <br />                               3                                                                                                                                          2.9    inose, growth observed LBT broth, growth <br />                             2.5                                                                                                                                          2.7    rates increased increasing arabinose concentrations <br />                               2                                                                                                                                                 0.4% (Figure 2).  pFab-mediated resist- <br />                                                                                                                                                                                 ance triclosan expression fabI. <br /> Figure 1 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                         Page 4 9 <br />                                                                                                                                                                                                                 (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />                                                                                                                                      DH5&#206;&#177;/pUC19          DH5&#206;&#177;/pFab <br />                                                                 kb <br />                                                                                                                      genomic <br />                                                                 10 <br />                                                                  4                                                   DNA <br />                                                                  2                                                   plasmid <br />                                                                                                                      DNA <br />  <br />  <br />                                                                B <br />  <br /> Figureof2fabI induction triclosan resistance <br /> Effect <br /> Effect fabI induction triclosan resistance. Tri- <br /> closan resistance mediated arabinose-induced  <br /> expression fabI. Exponential growth rates DH5&#206;&#177;/ <br /> pBFab1 DH5&#206;&#177;/pBFab6 relative DH5&#206;&#177;/pBAD18s DH5&#206;&#177;/pBAD18, respectively. <br />  <br />  <br /> Characterization pFab selection pFab transformants triclosan, characterized general properties pFab cloning <br /> vector, pUC19 included comparison (Table 1).          Plasmid <br />                                                                Figure 3 production pFab transformants <br /> Plasmid preparation yield pFab 43% greater         Plasmid production pFab transformants.   Agar- <br /> pUC19 (Table 1) E. coli DH5&#206;&#177; transformants.        ose gel electrophoresis total DNA isolated copy number pFab 38% 40% greater          clones pUC19 pFab. Bands quantified using pUC19, measured qPCR plasmid genomic           ImageQuant software calculate ratio pDNA gDNA <br /> DNA abundance (Table 1; Figure 3A &amp; B), respectively.          measure copy number pUC19 pFab. (B) Mean  pFab displayed higher yield copy number         ratios pDNA:gDNA pUC19 pFab  . <br /> relative parent vector. <br />  assess plasmid stability, scored number     observed selection (Figure 4A; Table 1). triclosan resistant colony forming units relative total     determined DH5&#206;&#177;/pFab culture growth rates pres- <br /> colony forming units. Surprisingly, observed col-      ence range triclosan concentrations (0 &#226;&#8364;&#8220; 2 &#206;&#188;M). onies LBT LB plates. indicated high plas-      pFab unstable triclosan, DH5&#206;&#177;/pUCFA <br /> mid stability presence triclosan, form      grown LBA included prevent growth com- pFab-mediated toxicity absence triclosan.         peting plasmid-free cells. Growth rates pUCFA  measurements plasmid stability using alter-      pFab clones lowest triclosan increased <br /> native &#206;&#177;-complementation assay revealed pFab          triclosan addition 1 &#206;&#188;M, indicating triclosan- stable pUC19 selection (Table 1).        dependent fitness rescue cells carry pFab (Figure <br /> fore, pFab expression clearly conferred tri-        4B). Similarly, observed proportion dead <br /> closan resistance, appeared confer requirement    cells decreased increasing triclosan concentrations. biocide.                                               Fluorescence microscopy SYTOX stained cells [29] <br />                                                                revealed elongated dead cells absence tri- <br /> Effects pFab triclosan E. coli growth, survival      closan, elongated viable cells 1 &#206;&#188;M triclosan fitness                                                    (data shown). ratio live dead cells indicated investigate interaction fabI tri-   SYTOX staining quantified flow cytometry <br /> closan, examined culture growth profiles.       [29]. results showed triclosan reduced ampicillin little effect DH5&#206;&#177;/pUC19 cultures,       number dead cells, indicating pFab transformants <br /> final optical density DH5&#206;&#177;/pFab highest          rescued triclosan (Figure 4C).  pFab presence triclosan exponential phase growth         toxic toxicity suppressed triclosan. <br /> rates DH5&#206;&#177;/pFab cultures faster DH5&#206;&#177;/ <br /> pUC19 cultures selection, inverse  <br />  <br />                                                                                                                     Page 5 9 <br />                                                                                             (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> Table 1: Properties pFab pUC19 <br />  <br />  Parametersa                                                                    pUC19                                   pFab <br />  <br />  Transformation efficiencyb (CFU/&#206;&#188;g)                                     5.5 &#195;&#8212; 107 &#194;&#177; 1.2 &#195;&#8212; 107                  2.2 &#195;&#8212; 107 &#194;&#177; 3.6 &#195;&#8212; 106 <br />  Plasmid yieldc (&#206;&#188;g/ml)                                                        30 &#194;&#177; 2.1                              42.9 &#194;&#177; 3.5 <br />  Copy numberd                                                                  141 &#194;&#177; 25                               200 &#194;&#177; 33 <br />  Stability (%) <br />     selectione                                                           85.3 &#194;&#177; 10.8                            99.5 &#194;&#177; 1.5 <br />     selectionf                                                        56.7 &#194;&#177; 17.7                            54.5 &#194;&#177; 14.9 <br />  Relative growth rateg (&#206;&#8221;OD/&#206;&#8221;t) <br />     selection                                                                 1                                 1.2 &#194;&#177; 0.07 <br />     selection                                                         1.1 &#194;&#177; 0.05                             0.6 &#194;&#177; 0.02 <br />  <br />  Determined   replicate cultures. <br />  b chemically competent DH5&#206;&#177; cells. <br />  c Plasmid yield 1 ml 18 h culture, determined OD <br />                                                                260 absorbance. <br />  d Determined copy ratio lacZ&#206;&#177; dxs qPCR. <br />  e Percentage plasmid-bearing cells 48 h cultures grown selection. <br />  f Percentage plasmid-bearing cells 48 h cultures grown selection. <br />  g Change OD <br />                   550 time DH5&#206;&#177;/pUC19 cultures LB DH5&#206;&#177;/pFab LB LBT, relative control DH5&#206;&#177;/pUC19 cultures LBA. <br />  <br />  test triclosan resistance likely persist            practice [21-23,39,40]. laboratory, intended applications, assessed fitness                   spontaneous triclosan resistance E. coli resulting DH5/pUC19 DH5/pFab strains mixed culture                     exposure low triclosan concentration plasmid-free parent strain. competitive fitness assay,             observed, point mutations fabI increased <br /> conducted absence selection, showed pFab                  MIC 95 fold [12]. mutations arise dur- <br /> persistence weaker pUC19 cul-                 ing triclosan selection.  experiments, tri- <br /> ture days (Figure 4D). words, rate              closan resistance dependent expression pFab loss faster pUC19 loss                plasmid-borne fabI (Figure 2), blue-white selection absence selection, pFab stable competi-              recombinant E. coli indicated pFab-carrying cells <br /> tive pUC19.                                                         (blue) spontaneous resistant mutants (Table <br />                                                                          1). environment, triclosan resistance slow <br /> Discussion                                                               emerge compared antibiotic resistance [21,23], pos- expression growth essential gene                   sibly poor solubility triclosan [41], rapid degra- <br /> conferring resistance specific protein inhibitor               dation triclosan [22,42,43], low competitive fitness plasmid selection E. coli, using fabI triclosan            FabI mutants [12], tripartite nature FabI/ example. avoiding use antibiotic                 <span id='am-34' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>/triclosan complex. Furthermore, spread resistance genes antibiotics, pFab transformants                     pFab outside intended applications limited <br /> showed improved growth, yield gene containment.                      low plasmid stability, poor competitive fitness improvements appear mechanism                     cell toxicity absence triclosan.  FabI inhibition [13] balance toxic gene-inhibitor                 stringently regulated fatty acid biosynthesis <br /> levels required cell survival [32] selection [33].               pathway [32], regulation inhibits cell growth Similar interactions systems suggest gene-                 viability [44,45].  possible pFab <br /> inhibitor reciprocal suppression common mech-                   transfer horizontally induce resistance anism. example, reciprocal mutual suppression                 triclosan wild-type bacteria, standard precautions described genes [34] chemical                       handling genetically modified microorganisms <br /> inhibitors [35].  antibiotic-dependent strains                 maintained. described bacteria isolated laboratories <br /> [36,37] clinics [38].                                                large scale production proteins, plasmid stability <br />                                                                          selection pre-requisite residual anti- new selection relies endogenous E. coli                 biotics undesirable. regard, finding gene widely used biocide.  relative               pFab requires triclosan plasmid stability safety triclosan fabI application               disadvantage.  triclosan required <br /> considered. Triclosan approved regulatory authori-                 compared antibiotic selection (typ- <br /> ties EU USA applications [24,25]                 ically 2%), level residual triclosan permis- association bacterial triclosan resistance                sible, given approval use non-prescription antibiotic susceptibility, suggested,                 medicines hygiene products.  need  <br />  <br />                                                                                                                                    Page 6 9 <br />                                                                                                           (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                 http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br />  <br />                                     formants <br />                                Effect <br />                                Figure  of4triclosan growth, viability fitness pFab trans- <br />                                Effect triclosan growth, viability fitness <br />                                pFab transformants.   Growth curves DH5&#206;&#177;/ <br />                                pUC19 DH5&#206;&#177;/pFab cultures grown                                selection. Growth rates determined exponen- <br />                                tial phase, indicated arrows. (B) Reduced fitness fabI <br />                                expression suppression triclosan. Growth <br />                                rates pFab pUCFA clones LB LBA, respectively, <br />                                calculated relative DH5&#206;&#177;/pUC19 LBA. (C) Flow <br />                                cytometric determination dead cell numbers, measure <br />                                cell toxicity, DH5&#206;&#177;/pFab cultures grown presence <br />                                range triclosan concentrations. (D) Growth competi- <br />                                tion DH5&#206;&#177; DH5&#206;&#177;/pUC19 DH5&#206;&#177;/pFab.                                log10 ratio plasmid-bearing cells total number cells <br />      B                         time represents rate plasmid loss mixed cell <br />                                populations. data representative independent <br />                                experiments. <br />  <br />  <br />  <br />                                stability absence selection prove impor- <br />                                tant   pFab stability                                potential problem vector construct process adjust- <br />                                ments needed scale  <br />  <br />                                Conclusion <br />                                summary, study provides example essen- <br />                                tial genes used combination non-antibiotic <br />       C                        inhibitors select maintain recombinant bacteria.                                possible apply triclosan selection bac- <br />                                teria Staphylococcus aureus [46] Mycobacterium <br />                                smegmatis [47], homologues FabI.                                inhibitors FabI described [48,49]  <br />                                depending bacterial resistance development, certain <br />                                triclosan analogues preferred use plas- <br />                                mid selection  hand, bacteria                                divergent enoyl-ACP reductase, Bacillus subti- <br />                                lis (FabL) [50] Streptococcus pneumoniae (FabK) [51], <br />                                species intrinsically resistant triclosan, <br />                                Pseudomonas aeruginosa [18], suitable                                fabI-triclosan plasmid selection. species,                                interesting test additional essential gene inhibitor <br />       D                        combinations expand plasmid selection strategy <br />                                described  pFab triclosan poten- <br />                                tially attractive production recombinant proteins, <br />                                used increase plasmid copy number <br />                                yield. addition, pFab derivatives                                suitable manufacturing biopharmaceuticals gene <br />                                therapy products, applications require <br />                                absence antibiotic resistance sequences antibiotic <br />                                residues. <br />  <br />                                Authors&apos; contributions <br />                                SG carried experimental work data analyses <br /> Figure 4 <br />                                flow cytometry, participated design study <br />                                drafted manuscript. LG conceived study, car- <br />  <br />  <br />                                                                                        Page 7 9 <br />                                                                (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                    http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> ried flow cytometry FACS analysis, helped                          19.   Braoudaki M, Hilton AC: Adaptive resistance biocides Sal- <br />                                                                                      monella enterica Escherichia coli O157 cross-resist- design drafting manuscript. authors                            ance antimicrobial agents. J Clin Microbiol 2004, 42:73-78. <br /> read approved final manuscript.                                        20.   Karatzas KA, Webber MA, Jorgensen F, Woodward MJ, Piddock LJ, <br />                                                                                      Humphrey TJ: Prolonged treatment Salmonella enterica <br />                                                                                      serovar Typhimurium commercial disinfectants selects <br /> Acknowledgements                                                                     multiple antibiotic resistance, increased efflux thank Swedish Institute Guest Scholarship SG                reduced invasiveness. J Antimicrob Chemother 2007, 60:947-955. <br /> Swedish Research Council University London Central Research             21.   Cole EC, Addison RM, Rubino JR, Leese KE, Dulaney PD, Newell MS, <br /> Fund support. thank Thomas Bentin, Omid Faridani, Abbas                  Wilkins J, Gaber DJ, Wineinger T, Criger DA: Investigation anti- <br />                                                                                      biotic antibacterial agent cross-resistance target bac- <br /> Nikravesh, M&#195;&#179;nica Marta, Joshua Fink Rikard Dryselius help               teria homes antibacterial product users comments.                                                                            nonusers. J Appl Microbiol 2003, 95:664-676. <br />                                                                                22.   McBain AJ, Bartolo RG, Catrenich CE, Charbonneau D, Ledder RG, <br />                                                                                      Price BB, Gilbert P: Exposure sink drain microcosms tri- <br /> References                                                                           closan: population dynamics antimicrobial susceptibility. <br /> 1.    Points consider production testing new drugs                  Appl Environ Microbiol 2003, 69:5433-5442. <br />       biologicals produced recombinant DNA technology                   23.   Beier RC, Duke SE, Ziprin RL, Harvey RB, Hume  Poole TL, Scott <br />       [http://www.fda.gov/cber/gdlns/ptcdna.htm]                                     HM, Highfield LD, Alali WQ, Andrews K, et al.: Antibiotic Dis- <br /> 2.    Validation growth-based rapid microbiological methods                       infectant Susceptibility Profiles Vancomycin-Resistant <br />       sterility testing cellular gene therapy products                    Enterococcus faecium (VRE) Isolated Community <br />       [http://www.fda.gov/cber/gdlns/stercgtp.htm]                                   Wastewater Texas. Bull Environ Contam Toxicol 2008. <br /> 3.    Titball RW, Sjostedt  Pavelka MS Jr, Nano FE: Biosafety            24.   Opinion triclosan resistance             [http://www.europa.eu.int/ <br />       selectable markers. Ann N Y Acad Sci 2007, 1105:405-417.                       comm/food/fs/sc/ssc/out269_en.pdf] <br /> 4.    Fajardo  Martinez-Martin N, Mercadillo M, Galan JC, Ghysels B,         25.   Drug approvals [http://www.fda.gov/cder/da/da1201.htm] <br />       Matthijs S, Cornelis P, Wiehlmann L, Tummler B, Baquero F, Martinez      26.   Lee C, Kim J, Shin SG, Hwang S: Absolute relative QPCR <br />       JL: neglected intrinsic resistome bacterial pathogens.                  quantification plasmid copy number Escherichia coli. J <br />       PLoS 2008, 3:e1619.                                                        Biotechnol 2006, 123:273-280. <br /> 5.    Gonzalez  Davila G, Calva E: Cloning DNA sequence            27.   Livak KJ, Schmittgen TD: Analysis relative gene expression <br />       complements glutamine auxotrophy Saccharomyces cere-                        data using real-time quantitative PCR 2(-Delta Delta <br />       visiae. Gene 1985, 36:123-129.                                                 C(T)) Method. Methods 2001, 25:402-408. <br /> 6.    Cranenburgh RM, Hanak JA, Williams SG, Sherratt DJ: Escherichia          28.   Nilsson AI, Zorzet  Kanth  Dahlstrom S, Berg OG, Andersson DI: <br />       coli strains allow antibiotic-free plasmid selection                  Reducing fitness cost antibiotic resistance amplifi- <br />       maintenance repressor titration. Nucleic Acids Res 2001,                    cation initiator tRNA genes. Proc Natl Acad Sci USA 2006, <br />       29:E26.                                                                        103:6976-6981. <br /> 7.    Dryselius R, Nekhotiaeva N, Nielsen PE, Good L: Antibiotic-free          29.   Roth BL, Poot M, Yue ST, Millard PJ: Bacterial viability anti- <br />       bacterial strain selection using antisense <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> nucleic                     biotic susceptibility testing SYTOX green nucleic acid <br />       acid. Biotechniques 2003, 35:1060-1064.                                        stain. Appl Environ Microbiol 1997, 63:2421-2431. <br /> 8.    Mairhofer J, Pfaffenzeller  Merz D, Grabherr R: novel antibiotic     30.   Lenski  Simpson SC, Nguyen TT: Genetic analysis plas- <br />       free plasmid selection  advances safe efficient                  mid-encoded, host genotype-specific enhancement bacte- <br />       DNA therapy. Biotechnol J 2008, 3:83-89.                                       rial fitness. J Bacteriol 1994, 176:3140-3147. <br /> 9.    Xu HH, Real L, Bailey MW: array Escherichia coli clones            31.   Guzman LM, Belin D, Carson MJ, Beckwith J: Tight regulation, <br />       expressing essential proteins: new strategy identi-                  modulation, high-level expression vectors containing <br />       fying cellular targets potent antibacterial compounds. Bio-                 arabinose PBAD promoter. J Bacteriol 1995, 177:4121-4130. <br />       chem Biophys Res Commun 2006, 349:1250-1257.                             32.   Heath RJ, Rock  Regulation fatty acid elongation ini- <br /> 10.   Bergler H, Wallner P, Ebeling  Leitinger B, Fuchsbichler S, Aschauer         tiation acyl-acyl carrier protein Escherichia coli. J Biol <br />       H, Kollenz G, Hogenauer G, Turnowsky F: Protein EnvM                    Chem 1996, 271:1833-1836. <br />       <span id='am-22' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>NADH</span>-dependent enoyl-ACP reductase (FabI)                             33.   Ward WH, Holdgate GA, Rowsell S, McLean  Pauptit RA, Clayton <br />       Escherichia coli. J Biol Chem 1994, 269:5493-5496.                             E, Nichols WW, Colls JG, Minshull CA, Jude DA, et al.: Kinetic 11.   Heath RJ, Yu YT, Shapiro MA, Olson E, Rock  Broad spectrum                  structural characteristics inhibition enoyl (acyl car- <br />       antimicrobial biocides target FabI component fatty                      rier protein) reductase triclosan.               Biochemistry 1999, <br />       acid synthesis. J Biol Chem 1998, 273:30316-30320.                             38:12514-12525. <br /> 12.   McMurry LM, Oethinger M, Levy SB: Triclosan targets <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span> syn-           34.   Weitao T, Nordstrom K, Dasgupta S: Mutual suppression       thesis. Nature 1998, 394:531-532.                                              mukB seqA phenotypes arise opposing <br /> 13.   Heath RJ, Rubin JR, Holland DR, Zhang E, Snow  Rock  Mech-               influences Escherichia coli nucleoid structure. Mol <br />       anism triclosan inhibition bacterial fatty acid synthesis.               Microbiol 1999, 34:157-168. <br />       J Biol Chem 1999, 274:11110-11114.                                       35.   Yue TF, Dayton PG, Gutman AB: Mutual Suppression Uri- <br /> 14.   Russell AD: Biocide use antibiotic resistance: rele-                   cosuric Effects Sulfinpyrazone Salicylate: Study       vance laboratory findings clinical environmental                     Interactions Drugs. J Clin Invest 1963, 42:1330-1339. <br />       situations. Lancet Infect Dis 2003, 3:794-803.                           36.   Wild DG: Reversion erythromycin dependence 15.   Panagakos FS, Volpe AR, Petrone  DeVizio W, Davies RM, Proskin              Escherichia coli: strains altered ribosomal sub-unit associa- <br />       HM: Advanced oral antibacterial/anti-inflammatory technol-                     tion ribosome assembly.                  J Gen Microbiol 1988, <br />       ogy: comprehensive review clinical benefits tri-                 134:1251-1263. <br />       closan/copolymer/fluoride dentifrice.            J Clin Dent 2005,       37.   Goldstein F, Perutka J, Cuirolo  Plata K, Faccone D, Morris J, Sour- <br />       16(Suppl):S1-19.                                                               nia  Kitzis MD, Ly  Archer G, Rosato AE: Identification 16.   Wohlrab J, Jost G, Abeck D: Antiseptic efficacy low-dosed                 phenotypic characterization beta-lactam-dependent, <br />       topical triclosan/chlorhexidine combination therapy                         methicillin-resistant Staphylococcus aureus strain. Antimicrob <br />       atopic dermatitis. Skin Pharmacol Physiol 2007, 20:71-76.                      Agents Chemother 2007, 51:2514-2522. <br /> 17.   Aiello AE, Larson E: Antibacterial cleaning hygiene prod-            38.   Van Bambeke F, Chauvel M, Reynolds PE, Fraimow HS, Courvalin P: <br />       ucts emerging risk factor antibiotic resistance               Vancomycin-dependent Enterococcus faecalis clinical isolates <br />       community. Lancet Infect Dis 2003, 3:501-506.                                  revertant mutants. Antimicrob Agents Chemother 1999, <br /> 18.   Chuanchuen R, Beinlich K, Hoang TT, Becher  Karkhoff-Schweizer               43:41-47. <br />       RR, Schweizer HP: Cross-resistance triclosan                 39.   Aiello AE, Marshall B, Levy SB, Della-Latta P, Larson E: Relationship <br />       antibiotics Pseudomonas aeruginosa mediated multid-                   triclosan susceptibilities bacteria isolated <br />       rug efflux pumps: exposure susceptible mutant strain                   hands community. Antimicrob Agents Chemother 2004, <br />       triclosan selects nfxB mutants overexpressing MexCD-OprJ.                      48:2973-2979. <br />       Antimicrob Agents Chemother 2001, 45:428-432. <br />  <br />  <br />                                                                                                                                             Page 8 9 <br />                                                                                                                     (page number citation purposes) <br />  BMC Biotechnology 2008, 8:61                                                                  http://www.biomedcentral.com/1472-6750/8/61 <br />  <br />  <br />  <br /> 40.   Gilbert P, McBain  Sreenivasan P: Common therapeutic <br />       approaches control oral biofilms: microbiological <br />       safety efficacy. Clin Microbiol Infect 2007, 13(Suppl 4):17-24. <br /> 41.   Orvos DR, Versteeg DJ, Inauen J, Capdevielle M, Rothenstein  Cun- <br />       ningham V: Aquatic toxicity triclosan. Environ Toxicol Chem <br />       2002, 21:1338-1349. <br /> 42.   Federle TW, Kaiser SK, Nuck BA: Fate effects triclosan       activated sludge. Environ Toxicol Chem 2002, 21:1330-1337. <br /> 43.   Aranami K, Readman JW: Photolytic degradation triclosan       freshwater seawater. Chemosphere 2007, 66:1052-1056. <br /> 44.   Keating DH, Carey MR, Cronan JE Jr: unmodified (apo) form <br />       Escherichia coli acyl carrier protein potent inhibitor       cell growth. J Biol Chem 1995, 270:22229-22235. <br /> 45.   Subrahmanyam S, Cronan JE Jr: Overproduction functional <br />       fatty acid biosynthetic enzyme blocks fatty acid synthesis       Escherichia coli. J Bacteriol 1998, 180:4596-4602. <br /> 46.   Heath RJ, Li J, Roland GE, Rock  Inhibition Staphyloco- <br />       ccus aureus NADPH-dependent enoyl-acyl carrier protein <br />       reductase triclosan hexachlorophene. J Biol Chem 2000, <br />       275:4654-4659. <br /> 47.   McMurry LM, McDermott PF, Levy SB: Genetic evidence       InhA Mycobacterium smegmatis target triclosan. <br />       Antimicrob Agents Chemother 1999, 43:711-713. <br /> 48.   Heerding DA, Chan G, DeWolf  Fosberry AP, Janson CA, Jawor- <br />       ski DD, McManus E, Miller WH, Moore TD, Payne DJ, et al.: 1,4-Dis- <br />       ubstituted imidazoles potential antibacterial agents <br />       functioning inhibitors enoyl acyl carrier protein reduct- <br />       ase (FabI). Bioorg Med Chem Lett 2001, 11:2061-2065. <br /> 49.   Lu H, Tonge PJ: Inhibitors FabI, Enzyme Drug Target       Bacterial Fatty Acid Biosynthesis Pathway. Acc Chem Res <br />       2008, 41:11-20. <br /> 50.   Heath RJ, Rock  triclosan-resistant bacterial enzyme. <br />       Nature 2000, 406:145-146. <br /> 51.   Marrakchi H, Dewolf Jr, Quinn C, West <br /> </body></html>