<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br />       Structural basis recruitment ERCC1-XPF <br />       <span id='am-22' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> excision repair complexes XPA <br />  <br />  <br />                                                   Introduction <br />                                                     repair <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> insults DNA caused UV light                                                              mutagens essential coping successfully                                                    intrinsic reactivity DNA preserving genetic informa- <br />                                                                   tion. Inherited diseases resulting failure correct <br />       <br />        Department Biological Chemistry Molecular Pharmacology,                         spontaneous environmentally induced damage typi- <br />       Harvard Medical School, Boston, MA, USA, 2Department                                 cally associated genomic instability predisposition <br />       Pharmacological Sciences, Stony Brook University, Stony Brook, NY,                      various cancers (Friedberg et al, 2005). Contrarily, DNA <br />       USA, 3Department Chemistry, Stony Brook University, Stony Brook, <br />       NY, USA 4Department Biochemistry Molecular Biophysics,                       repair undesirable DNA-damaging agents used <br />       Washington University School Medicine, St Louis, MO, USA                             chemotherapy cancer diseases.                                                                                               settings, ability modulate DNA repair activities       nucleotide excision repair (NER) pathway corrects                                   cells targeted destruction desirable goal (Ding et al, <br />       DNA damage caused sunlight, environmental mutagens                                   2006). nucleotide excision repair (NER) pathway       certain antitumor agents. multistep DNA repair                                 essential normal genomic maintenance, removing bulky <br />       reaction operates sequential assembly protein                                 chemical adducts DNA mutagenic       factors sites DNA damage. efficient recognition                            pose lethal blocks replication. <br />       DNA damage repair orchestrated specific                                     NER involves 30 proteins recognize damaged <br />       protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220;DNA interactions NER                                 sites DNA excise oligonucleotide containing       complexes. investigated essential protein&#226;&#8364;&#8220;                                   damage  Laat et al, 1999; Gillet Scha&#194;&#168;rer, 2006). <br />       protein interaction NER pathway, binding                                  Following excision damaged DNA segment, result- <br />       XPA protein ERCC1 subunit repair                                      ing gap filled templated DNA synthesis ligase seals <br />       endonuclease ERCC1-XPF. structure ERCC1 com-                                  nick complete repair. Cell biological biochem- <br />       plex XPA <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> shows small region                                 ical studies shown NER operates sequential <br />       XPA interacts ERCC1 form stable complex                                    assembly protein factors sites DNA damage,       exhibiting submicromolar binding affinity.                                 action preformed repairosome <br />       XPA peptide potent inhibitor NER activity cell-                            (Houtsmuller et al, 1999; Volker et al, 2001). recruitment <br />       free assay, blocking excision cisplatin adduct                            NER factors protein&#226;&#8364;&#8220;DNA ensembles guided       DNA. structure peptide inhibitor bound                                step numerous protein&#226;&#8364;&#8220;protein interactions (Araujo et al, <br />       target site reveals binding interface amenable                             2001), imparting specificity recognition verification <br />       development small molecule peptidomimetics                                  damaged sites. Damage recognition NER culminates       used modulate NER repair activities                                         incision DNA 50 30 lesion site ERCC1-XPF <br />       vivo.                                                                                XPG, respectively, release 24&#226;&#8364;&#8220;32 nucleotide segment <br />       EMBO Journal (2007) 26, 4768&#226;&#8364;&#8220;4776. doi:10.1038/                                     containing damage  Laat et al, 1999; Gillet       sj.emboj.7601894; Published online 18 October 2007                                      Scha&#194;&#168;rer, 2006). <br />       Subject Categories: genome stability <span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; dynamics; structural                                NER pathway, DNA cleavage ERCC1-XPF  <br />       biology                                                                                 quires physical interaction XPA, scaffold protein       Keywords: DNA repair; ERCC1; NMR; nucleotide excision                                   interacts DNA repair proteins, including <br />       repair; XPA                                                                             RPA, TFIIH ERCC1 subunit ERCC1-XPF (Li et al, <br />                                                                                               1994, 1995; Park Sancar, 1994; Saijo et al, 1996). <br />                                                                                               XPA originally described DNA damage- <br />                                                                                               specific sensor verification protein, recent work suggests <br />                                                                                               XPA instead recognizes DNA structural intermediates <br />                                                                                               arising processing NER (Jones Wood, 1993; <br />                                                                                               Camenisch et al, 2006). XPA recruits ERCC1-XPF NER <br />                                                                                               complexes (Volker et al, 2001), positioning XPF nuclease <br />       *Corresponding author. Department Biochemistry Molecular                         domain 50 damage site (Enzlin Scha&#194;&#168;rer, <br />       Biophysics, Washington University, Washington University School                      2002). ERCC1-XPF roles DNA metabolism  <br />       Medicine, 660 S Euclid, Campus Box 8231, St Louis, MO 63110, USA. <br />       Tel.: &#195;&#190; 1 314 747 8893; Fax: &#195;&#190; 1 314 632 4432; <br />                                                                                               NER, notably interstrand crosslink repair       E-mail: tome@biochem.wustl.edu                                                          homologous recombination (Hoy et al, 1985; Niedernhofer <br />       5 <br />        Present address: Department Medicinal Chemistry, University                      et al, 2001). importance additional, NER-inde- <br />       Michigan, Ann Arbor, MI 48109, USA                                                      pendent functions ERCC1-XPF underscored       6 <br />        authors contributed equally work <br />                                                                                               pronounced sensitivity crosslinking agents caused       Received: 19 2007; accepted: 25 September 2007; published                           mutations ERCC1 XPF mice humans (McWhir <br />       online: 18 October 2007                                                                 et al, 1993; Weeda et al, 1997; Niedernhofer et al, 2006). <br />  <br /> 4768 EMBO Journal VOL 26 | 22 | 2007                                                                           &amp; 2007 European Molecular Biology Organization <br />                                                                                                                       Recruitment ERCC1-XPF NER complexes XPA <br />                                                                                                                                                       OV Tsodikov et al <br />  <br />  <br />   exact biochemical role(s) ERCC1-XPF                                          Scha&#194;&#168;rer, 2002; Nishino et al, 2003; Newman et al, 2005), <br /> crosslink repair remain discovered.                                                       ERCC1&#226;&#8364;&#8482;s groove lacks catalytic residues    Li et al (1994, 1995) identified residues 59&#226;&#8364;&#8220;114 XPA                                   nuclease active site instead populated basic site interaction ERCC1, showed deletion                                    aromatic residues (Gaillard Wood, 2001). previously conserved glycines (Gly72, Gly73, Gly74) abrogates                                     showed central domain ERCC1 binds single- XPA&#226;&#8364;&#8220;ERCC1 interaction ability XPA                                     stranded DNA vitro (Tsodikov et al, 2005), proposed <br /> protein confer UV resistance XP cells. Furthermore,                                     V-shaped groove DNA-binding site. expression truncated protein comprising residues                                          observations prompted investigate struc- <br /> 59&#226;&#8364;&#8220;114 XPA renders cells sensitive UV light cisplatin                                 tural functional basis interaction XPA (Rosenberg et al, 2001), suggesting region suffi-                                  ERCC1, role recruiting XPF-ERCC1 endonu- <br /> cient disrupt interaction native XPA protein                                 clease sites NER.  structure ERCC1-XPF. Conversely, inferred pre-                                     short peptide motif XPA complex central <br /> vious studies residues 92&#226;&#8364;&#8220;119 ERCC1 necessary                                   domain ERCC1. XPA peptide specifically interaction XPA (Li et al, 1994; Sijbers et al, 1996;                                  inhibits NER reaction vitro, creating opportunity <br /> Gaillard Wood, 2001).                                                                       structure-based design NER inhibitors targeting    Following seminal studies, understanding                                        protein&#226;&#8364;&#8220;protein interaction. Point mutations correspond- <br /> biochemical structural basis XPA&#226;&#8364;&#8482;s interaction                                     ing region XPA abolish NER activity vitro, underscoring <br /> ERCC1 advanced, known                                        importance XPA&#226;&#8364;&#8220;ERCC1 interactions NER. individual proteins. XPA contains defined central                                     addition providing insights protein&#226;&#8364;&#8220;protein inter- <br /> main (residues 98&#226;&#8364;&#8220;219; Figure 1A), remainder                                    actions mediate progression NER pathways, protein including ERCC1 interaction domain appears                                      studies provide blueprint develop small molecules poorly structured (Buchko et al, 1998, 2001; Ikegami                                      intercept interaction XPA ERCC1. et al, 1998; Iakoucheva et al, 2001). Residues 92&#226;&#8364;&#8220;119                                        molecules valuable studying biochemical <br /> ERCC1 fall central domain ERCC1 (Tsodikov                                         functions ERCC1-XPF NER repair pathways <br /> et al, 2005) structurally homologous nuclease                                    including DNA interstrand crosslink repair homologous <br /> domain archaeal XPF-like proteins Aeropyrum pernix                                       recombination selectively inhibiting NER. <br /> Mus81/XPF (Newman et al, 2005) Pyrococcus furiosus <br /> Hef (Nishino et al, 2003). V-shaped groove surface ERCC1 corresponds nuclease active site XPF (Enzlin <br />                                                                                                 Results <br />                                                                                                 Induced fit XPA peptide interaction                                                                                                 ERCC1 <br />                                                                                                 Previous reports suggested ERCC1-interacting <br />                  1         58 67   77 98   129    138             219   226         273       region XPA (Figure 1A) unfolded solution, based              XPA     Dispensable ERCC1 Zn/RPA                 DNA             TFIIH             NMR studies sensitivity proteolytic cleavage <br />                                                                                                 (Buchko et al, 2001; Iakoucheva et al, 2001). investigate <br />   B          100                                                                                structure XPA ligand, collected HSQC NMR <br />                                                                                                 spectra 15N-labeled XPA59 93 peptide              105                                                                                complex unlabeled central domain ERCC1                                                                                                  ERCC192 214 protein; Figure 1B). absence ERCC1, <br />              110                                                                                resonance signals XPA cluster narrow range                                                                                                 chemical shifts (Figure 1B, inset) characteristic    N (ppm) <br />  <br />  <br />  <br />  <br />              115 <br />                                                                                                 unstructured polypeptide poor spectral dispersion.                                                    100                                          complex ERCC1, subset XPA backbone amides <br />   15 <br />  <br />  <br />  <br />  <br />                                                    105 <br />                                                                                                 dispersed peaks broader.              120                                   110 <br />                                                    115 <br />                                                                                                 changes indicative structured region                                                    120                                          bound XPA peptide. resonance peaks              125                                   125                                          markedly perturbed XPA59 93 bound ERCC1,                                                    130 <br />                                                          10    9    8     7     6      5 <br />                                                                                                  <span id='am-42' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-43' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> residues (assigned Gly72, <br />              130                                                                                Gly73 Gly74) strongly perturbed complex.                 10           9           8                7               6                 5 <br />                                              1H   (ppm)                                         order overcome peak broadening observed NMR <br />                                                                                                 spectra XPA59 93 peptide concentrations Figure 1 XPA domain organization structure ERCC1- <br /> binding peptide.   ERCC1-binding region XPA (residues                                  0.1 mM, sought identify shorter XPA peptide ligands <br /> 67&#226;&#8364;&#8220;77) located central domain (Zn2 &#195;&#190; -binding                                ERCC1. minimal, 14-residue XPA67 80 peptide   <br /> DNA-binding subdomains; residues 98&#226;&#8364;&#8220;219) N-terminal                                      scribed  identified examining series  <br /> region (residues 1&#226;&#8364;&#8220;58) dispensable functional comple-                               lapping XPA fragments region previously shown mentation NER cell extracts XP mutant cells <br /> (Miyamoto et al, 1992) TFIIH-binding region (Park et al, <br />                                                                                                 interact ERCC1 (Li et al, 1994). <br /> 1995). (B) 15N HSQC spectrum 15N-labeled XPA59 93 complex unlabeled ERCC1, unbound state (inset).                                     structure XPA complex ERCC1 <br /> spectrum unbound XPA59 93 (inset) characteristic                                synthetic XPA67 80 peptide amino-acid sequence <br /> unfolded peptide. appearance new dispersed NMR <br /> peaks XPA spectrum addition ERCC192 214 (shown                                   KIIDTGGGFILEEE forms stable complex ERCC196 214 larger spectrum) indicates portion XPA peptide                             purified gel filtration chromatography. Like <br /> adopts defined conformation complex ERCC1.                                            length XPA protein, XPA59 93 XPA67 80 <br />  <br /> &amp; 2007 European Molecular Biology Organization                                                                        EMBO Journal       VOL 26 | 22 | 2007 4769 <br />        Recruitment ERCC1-XPF NER complexes XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       peptides behave similarly efficiently purify ERCC1,                  complex unlabeled XPA67 80 (red, Figure 3).  <br />       suggesting XPA67 80 contains significant binding                        complexes ERCC196 214 XPA67 80 XPA59 93 <br />       determinants. confirmed XPA ERCC1 form                             identical, suggesting shorter XPA peptide makes <br />       stoichiometric 1:1 complex estimating                           significant binding contacts. 15N HSQC spec- <br />       subunit purified complex using Edman                           trum ERCC1&#226;&#8364;&#8220;XPA complex consistent slow- <br />       gradation reaction, analytical centrifugation                      exchange regime, implying dissociation equilibrium  <br />       complex. Equilibrium sedimentation data complex                          stant 1 mM complex. XPA-binding site       (Supplementary Figure 1C) best fit expected                          ERCC1 central domain identified using backbone <br />       masses 1:1 complex XPA59 93 ERCC192 214                             assignments ERCC196 214 complex XPA <br />       (Mw &#194;&#188; (19.471.2) kDa)            unbound        ERCC192 214                   Materials methods). comparison 15N HSQC <br />       (Mw &#194;&#188; (15.071.0) kDa). confirmed ERCC196 214                             spectra ERCC1 presence absence XPA reveals <br />       binds stoichiometrically XPA67 80 peptide KD                        exception, prominent changes <br />       0.78 mM (Supplementary Figure 2). structure                          chemical shifts involve cluster residues       XPA67 80&#226;&#8364;&#8220;ERCC196 214 complex (Figure 2A) determined                          V-shaped groove ERCC1 central domain (Figures 2B <br />       combination NMR-derived distance restraints X-                       3). <br />                                           &#203;&#353; resolution (Table 1       ray diffraction data extending 4                                               bound XPA peptide fits snugly V-shaped <br />       Supplementary data Materials methods) described                       groove ERCC1 (Figure 2) previously speculated <br />                                                                                  binding site single-stranded DNA (Tsodikov <br />                                                                                        et al, 2005). consecutive glycines (Gly72, <span id='am-30' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-31' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>Gly</span> 73, <br />       Identification ERCC1-binding site complex                              Gly74) XPA peptide insert groove, making       XPA                                                                         U-turn close steric complementary binding site. <br />       binding site XPA surface ERCC1 (Figure 2B)                     conserved glycines previously <br />       identified using dimensional HSQC experiments.                           reported essential interaction XPA       spectrum unliganded 15N-labeled ERCC192 214 (blue,                        ERCC1 required functional complementation <br />       Figure 3) showed significant differences                        XP cells (Li et al, 1994, 1995). total 1039  &#203;&#353; 2  <br />  <br />  <br />  <br />  <br />  <br />  <br />       Figure 2 Structure XPA&#226;&#8364;&#8220;ERCC1 complex.   XPA67 80 peptide (orange) bound V-shaped groove central domain       ERCC196 214 (green). orthogonal view bound XPA peptide (left  shown comparison peptide complex ERCC1 <br />       (right-hand . (B) XPA-binding site surface ERCC1 (colored red) identified resonance perturbations larger 0.2 ppm <br />       indicative direct interactions XPA. <br />  <br />  <br /> 4770 EMBO Journal VOL 26 | 22 | 2007                                                                      &amp; 2007 European Molecular Biology Organization <br />                                                                                                                                    Recruitment ERCC1-XPF NER complexes XPA <br />                                                                                                                                                                    OV Tsodikov et al <br />  <br /> Combined average chemical shift <br />  <br />  <br />  <br /> Figure 3 XPA67 80 binds shallow groove ERCC1.                                             Figure 4 XPA67 80 peptide effective inhibitor comparison dimensional HSQC spectra 15N-labeled                                        NER activity.   XPA67 80 inhibits vitro NER reaction, <br />                                                                                                                                      F75A <br /> ERCC192 214 presence absence unlabeled XPA67 80                                      mutant XPA67 80            peptide effect. HeLa <br /> peptide. 15N HSQC spectra reveal significant chemical shift                                       cell extracts incubated plasmid containing 1,3- <br /> changes ERCC1 residues absence (blue) presence                                     intrastrand cisplatin adduct presence increasing concen- <br />                                                                                                                                           F75A <br /> (red) unlabeled XPA67 80. (B) Combined average chemical shift                                      trations XPA67 80 XPA67 80  (lane 1, XPA; lanes 2 7, <br /> perturbations calculated Dwave &#194;&#188; (((Dw1H)2 &#195;&#190; (Dw15N/5)2)/2)1/2                                 46 nM XPA peptide; lanes 3 8, 460 nM; lanes 4 9, 4.6 mM; backbone amide ERCC1 shown histogram.                                            lanes 5 10, 46 mM; lanes 6 11, 92 mM). Products                                                                                                       visualized reaction following annealing oligonu- <br />                                                                                                       cleotide complementary excision product 4-nt <br />                                                                                                       overhang (Shivji et al, 1999). marker DNA ladder labeled <br />                                                                                                                                                       F75A <br />                                                                                                       LMW DNA ladder. (B) XPA67 80 XPA67 80             affect                                                                                                       intrinsic nuclease activity ERCC1-XPF. stem12-loop22 sub- <br /> accessible surface area XPA peptide buried                                             strate (6.6 nM) incubated different concentrations complex ERCC1, accounting 61% solvent                                                 ERCC1-XPF (lanes 2, 4 6: 6.7 nM ERCC1-XPF; lanes 3, 5 <br />                                                                                                       7: 26.8 nM) 0.4 mM MnCl2 presence peptide <br /> accessible surface area XPA residues 67&#226;&#8364;&#8220;77,                                                                                                               F75A <br />                                                                                                       (lanes 1&#226;&#8364;&#8220;3), 92 mM XPA67 80 (lanes 4 5), 92 mM XPA67 80 <br /> close proximity binding site. XPA ligand derives                                           (lanes 6 7). DNA substrate cleavage products interactions core sequence motif (shown                                              indicated. <br /> boldface; KIIDTGGGFILEEE) XPA67 80 peptide. chains Phe75, Leu77 Thr71 XPA clustered mouth V-shaped groove (Figure 2A)                                              directly observed NMR experiments Phe75 stacks Asn110 ERCC1, Ile76                                                                                                 &#203;&#353; ) X-ray <br />                                                                                                       reliably confirmed low-resolution (4 chain packs aliphatic portion ERCC1                                          diffraction data (Table 1 Supplementary data). Based <br /> chains Arg144 Leu148. binding groove ERCC1                                              proximity atoms modeled complex, infer <br /> capped XPA Leu77.                                                                                  carbonyl oxygen Gly74 bond main <br />    <span id='am-54' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-55' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span>-rich loop XPA67 80 extends far                                             chain amide Ser142 ERCC1. orientations groove ERCC1 main chain atoms XPA                                                   Tyr145 Tyr152 chains ERCC1 permit <br /> residues stack chains Tyr145 Tyr152                                           hydroxyl groups make hydrogen-bonding interactions ERCC1 (Figure 2A). main chain amides                                                backbone carbonyls Thr71 Gly73, respec- <br /> glycines participate hydrogen-bonding interactions                                           tively. chain XPA Asp70 participate ERCC1-binding site, interactions                                              electrostatic interactions chain His149  <br /> &amp; 2007 European Molecular Biology Organization                                                                                     EMBO Journal            VOL 26 | 22 | 2007 4771 <br />        Recruitment ERCC1-XPF NER complexes XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       ERCC1. notable solvent-exposed salt bridge            activities wild-type XPA protein. ability       chains Asp129 Arg156 ERCC1              XPA proteins mediate NER activity tested       (PDB code 2A1I; Tsodikov et al, 2005)               incubating plasmid containing 1,3-cisplatin intrastrand <br />       completely buried XPA bound.                               crosslink cell-free extract generated XPA-deficient <br />          Phe75 XPA completely buried ERCC1-             cells supplemented purified wild-type mutant XPA <br />       binding site (Figure 2A). tested alanine             protein (Shivji et al, 1999). Addition wild-type XPA protein <br />       substitution position interferes binding 15N-      mixture resulted robust NER activity, evidenced <br />       labeled ERCC1 measuring chemical shifts 15N HSQC         formation characteristic excision products 24&#226;&#8364;&#8220;32 <br />       spectra presence mutant peptide designated           nts length (Figure 5A, lanes 1&#226;&#8364;&#8220;2). contrast, NER <br />            F75A <br />       XPA67 80  . Addition mutant peptide failed perturb       activity observed following addition F75A       chemical shifts ERCC1 seen addition wild-type       G73D/G74D mutants, G73D single deletion mutant <br />       XPA67 80 (data shown), indicating mutant pep-         displayed marginal NER activity. <br />       tide does bind ERCC1. TGGGFI-binding motif              test XPA mutations affected binding       XPA ligand corresponding residues ERCC1-        ERCC1, compared DNA-binding activities wild- <br />       binding site strictly conserved higher eukaryotes.       type mutant XPA proteins. investigated binding       lower eukaryotes, corresponding sequences pro-         wild-type mutant XPA DNA way junction, <br />       teins diverged consensus, indicating        representing high-affinity target XPA band-shift assays <br />       coevolution proteins functions.         (Missura et al, 2001). wild-type, F75A, G73D G73D/ <br />                                                                          G74D XPA proteins bound similar affinity  <br />       XPA peptide inhibits NER mammalian cell                     way junction (Figure 5B), indicating mutant proteins <br />       extracts                                                           fully proficient DNA binding unlikely       direct interaction XPA67 80 peptide ERCC1-         misfolded inactive. results       binding pocket raised possibility peptide      single point mutations XPA result defect NER <br />       specifically interfere recruitment ERCC1-XPF <br />       nuclease NER reaction pathway. investigated                                                 F75A <br />       effect XPA67 80 mutant XPA67 80       peptide                                                                                                                        G73&#226;&#710;&#8224; <br />       dual incision DNA lesion NER cell-free extracts. <br />                                                                                             WT       F75A      G73&#226;&#710;&#8224;      G74&#226;&#710;&#8224; <br />       plasmid containing single site-specific 1,3-cisplatin  <br />       trastrand crosslink incubated HeLa cell-free extract <br />       presence increasing concentrations XPA peptide <br />       (Shivji et al, 1999). absence XPA peptide,       characteristic NER excision products 28&#226;&#8364;&#8220;33 nucleotides <br />       containing lesion evident (Figure 4A, lane 1). <br />       Increasing concentrations XPA67 80 interfered exci- <br />       sion oligonucleotide, complete inhibition                    25 <br />       achieved concentration XPA peptide low <br />       micromolar range (Figure 4A, lanes 2&#226;&#8364;&#8220;6). contrast,                         1        2   3    4    5     6    7     8 <br />                         F75A <br />       addition XPA67 80   did affect NER activity concentra- <br />       tions 92 mM, maximum concentration tested                B                                                          &#226;&#710;&#8224;G73 <br />       (Figure 4A, lanes 7&#226;&#8364;&#8220;11).                                                         WT            F75A             &#226;&#710;&#8224;G73          &#226;&#710;&#8224;G74 <br />                                                                               &#226;&#710;&#8217; <br />          XPA peptide inhibit NER activity vitro       directly interfering endonuclease activity ERCC1- <br />       XPF, instead blocking interaction XPA. account       x <br />                                                                           d <br />       possibility, tested effects XPA <br />       peptides DNA incision reaction catalyzed purified         d <br />       ERCC1-XPF using stem-loop DNA substrate  Laat et al, <br />       1998). ERCC1-XPF efficiently cleaves 50 loop         1 2 3 4 5 6 7 8 9 10 11 12 13 14 15 16 17 181920 21 <br />       XPA peptide effect activity (Figure 4B)     Figure 5 Mutation ERCC1-binding epitope XPA abolishes <br />       concentration (92 mM) completely abolishes          NER DNA-binding activity.   XP (XP2OS) cell extracts <br />       NER activity (Figure 4A). conclude inhibitory          incubated plasmid containing 1,3-intrastrand cisplatin <br />       effect XPA67 80 NER reaction results disrupting     adduct presence wild-type XPA (XPA-WT) mutant XPA <br />                                                                          proteins (XPA-F75A, XPA-G73D XPA-G73D/G74D). reaction <br />       interaction ERCC1 XPA, essential protein&#226;&#8364;&#8220;           products visualized reaction annealing       protein interaction dual incision DNA NER        excision product complementary oligonucleotide 4-nt <br />       pathway.                                                           overhang (Shivji et al, 1999). Different XPA concentrations                                                                          200 nM (lanes 1, 3, 5 7) 800 nM (lanes 2, 4, 6 8) <br />                                                                          tested. position 25 mer LMW DNA ladder       Mutations ERCC1-binding epitope XPA abolish              indicated. (B) 50 -labeled DNA way junction (1 nM)       NER                                                                incubated wild-type mutant XPA proteins 30 min       specificity inhibition NER XPA67 80 suggested         room temperature, XPA-bound (xd) free DNA (d) <br />       mutations single residues F75 diminish       oligonucleotides separated 8% native polyacrylamide <br />                                                                          gel. reaction products generated different concentrations <br />       NER activity XPA protein. generated mutant           XPA shown: 0 (lane 1), 4 nM (lanes 2, 7, 13, 17), 10 nM (lanes <br />       XPA proteins containing F75A mutation DG73 single           3, 8, 13, 18), 25 nM (lanes 4, 9, 14, 19), 60 nM (lanes 5, 10, 15, 20) <br />       DG73/DG74 double deletion, compared                  150 nM (lanes 6, 11, 16, 21). <br />  <br />  <br /> 4772 EMBO Journal VOL 26 | 22 | 2007                                                      &amp; 2007 European Molecular Biology Organization <br />                                                                                          Recruitment ERCC1-XPF NER complexes XPA <br />                                                                                                                          OV Tsodikov et al <br />  <br />  <br />  <br /> activity weakening interaction ERCC1           associated entropic penalty binding. peptide&#226;&#8364;&#8220; <br /> XPA. highly cooperative nature NER (Moggs           protein interaction sufficient block NER activity cell- <br /> et al, 1996), NER functions interactions            free extracts (Figure 4A), raising possibility peptido- disrupted result blocking recruitment           mimetic ligands developed specifically block NER <br /> XPF&#226;&#8364;&#8220;ERCC1.                                                        activity vivo. competing XPA peptide pre- <br />                                                                   vents double incision lesioned DNA NER, XPA competes single-stranded DNA binding                 peptide does interfere cleavage model DNA ERCC1                                                          substrate purified ERCC1&#226;&#8364;&#8220;XPF (Figure 4B). results XPA binds groove ERCC1 (Figure 2)          XPA peptide does block nuclease previously implicated DNA-binding activity (Tsodikov       activity XPF-ERCC1 instead likely interfere et al, 2005), directly tested XPA competes      recruitment nuclease NER protein complex. single-stranded DNA binding ERCC1. DNA-bind-          Mutations ERCC1-binding domain XPA similarly <br /> ing activity measured monitoring fluorescence aniso-       abolish NER activity affecting intrinsic DNA- <br /> tropy, using single-stranded 40-mer oligonucleotide labeled       binding activity XPA. Intriguingly, central domain 50 end 6-carboxyfluorescein. XPA67 80 pep-        ERCC1 binds single-stranded DNA vitro (Tsodikov et al, <br /> tide does detectably bind DNA  shown),     2005) DNA-binding activity blocked XPA <br /> does compete DNA binding ERCC1                        (Supplementary Figure 3). competing activ- <br /> (Supplementary Figure 3). result confirms           ities appear contradictory, cleavage assays em- <br /> DNA-binding site ERCC1&#226;&#8364;&#8482;s central domain overlaps          ployed length ERCC1 protein complex XPF, XPA-binding site. EC50 binding XPA67 80      vitro binding studies used central domain micromolar range, quenching fluorescent probe      ERCC1. ERCC1-XPF multiple DNA-binding high concentrations XPA precluded accurate mea-          sites (Newman et al, 2005; Tripsianes et al, 2005; Tsodikov <br /> surement binding constant. previously reported       et al, 2005; Nishino et al, 2005a, b), likely equilibrium-binding constant 1.5 mM DNA binding         DNA-binding surfaces XPF&#226;&#8364;&#8220;ERCC1 hetero- central domain ERCC1 (Tsodikov et al, 2005). fitting    dimer compensate interference XPA XPA competition titration data competitive binding       peptide. respect overall NER reaction, model (Equation 9 Supplementary data), obtain       conceivable DNA XPA protein alternatively <br /> estimated binding constant Kd &#194;&#188; (5407280) nM           bind site ERCC1 different steps XPA&#226;&#8364;&#8220;ERCC1 complex. result agrees affinity      repair process. regard, molecular handoff ERCC1 <br /> determined directly interaction (Supplementary           XPA strand unwound DNA substrate Figure 2).  XPA binds central domain ERCC1         envisioned multiple, individually weak interac- approximately fold higher affinity single-        tions drive progression NER pathway stranded DNA.                                                     concerted fashion (Stauffer Chazin, 2004; Gillet                                                                   Scha&#194;&#168;rer, 2006). <br />                                                                       Homologs XPA ERCC1 proteins Discussion                                                        eukaryotic organisms, despite presence homodi- removal bulky helix-distorting DNA lesions      meric XPF-like endonucleases Archaea (Nishino et al, <br /> NER pathway requires coordinated assembly large          2003). ERCC1 residues constituting XPA-binding site <br /> multiprotein complex (Houtsmuller et al, 1999; Volker et al,      poorly conserved Saccharomyces cerivisiae (Rad10 <br /> 2001) exposes damaged DNA strand excises          protein) S. pombe (Swi10 protein), XPA homo- <br /> oligonucleotide containing lesion (Gillet Scha&#194;&#168;rer,       logs (S. saccharomyces Rad14; S. pombe Rhp14) highly <br /> 2006). investigated essential protein&#226;&#8364;&#8220;         divergent mammalian XPA.  different inter- <br /> protein interactions pathway. specific interaction    action site recently reported Rad14 Rad10/ XPA ERCC1 responsible recruitment           Rad1, respective yeast homologs XPA ERCC1&#226;&#8364;&#8220;XPF <br /> ERCC1-XPF nuclease DNA repair complex (Li et al,           (Guzder et al, 2006). observations suggest XPA <br /> 1994). structural studies defined XPA ligand    ERCC1 coevolved interact specifically TGGGFI sequence motif inserts pocket           higher eukaryotes, response central domain ERCC1 (Figure 2). previously             added complexity distinct functional organization shown deletion GGG triplet motif          eukaryotic NER pathway. Correspondingly, BLAST search <br /> abolishes interaction XPA ERCC1 (Li et al,            does identify TGGGFI motif XPA ligand 1995). glycines insert deep ERCC1-binding          mammalian protein. <br /> site likely make hydrogen-bonding interactions             conclusion, established short <br /> using main chain atoms (Figure 2). binding site ERCC1      peptide segment XPA sufficient form stable mainly nonpolar surface punctuated        specific 1:1 complex ERCC1. interactions large aromatic chains (Phe145, Phe152) buried salt     consecutive glycines (Gly72, Gly73, Gly74) flank- <br /> bridge Arg156 Asp129. single             ing residues XPA complement V-shaped, hydrophobic <br /> point mutations XPA (F75A G73D) effectively abolish         groove central domain ERCC1. protein&#226;&#8364;&#8220;protein <br /> NER activity vitro, underscoring high specificity   interaction essential NER activity, XPA peptide <br /> binding interaction ERCC1 XPA.                        effective inhibitor NER activity cell-free reaction. <br />    XPA peptide ligand unstructured solution             work paves way development specific NER <br /> (Figure 1B). remarkable short peptide      inhibitors targeting surface ERCC1 involved XPA <br /> segment binds ERCC1 submicromolar affinity, given         binding. ERCC1 served molecular marker clinical <br />  <br /> &amp; 2007 European Molecular Biology Organization                                           EMBO Journal       VOL 26 | 22 | 2007 4773 <br />        Recruitment ERCC1-XPF NER complexes XPA <br />       OV Tsodikov et al <br />  <br />  <br />  <br />       resistance cisplatin-based chemotherapy (Reed, 2006),                    Structural information ERCC1&#226;&#8364;&#8220;XPA complex obtained <br />                                                                                differentially labeled sample ERCC1 15N- <br />       raising possibility using ERCC1 antagonists sensitiz- <br />                                                                                labeled perdeuterated synthetic XPA fragment       ing agents tumors resistant DNA-dama-              unlabeled (D,N-ERCC1/U-XPA) (Walters et al, 1997, 2001).       ging agents treatment cancer.                                  assignment XPA peptide sample performed <br />                                                                                using homonuclear 2D NOESY 2D TOCSY experiments acquired <br />                                                                                <span id='am-17' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span>H2O</span> D2O buffers. total 92 intramolecular <br />       Materials methods                                                    distance constraints XPA peptide derived 2D <br />                                                                                NOESY experiment acquired H2O 100 ms mixing time. <br />       Peptide DNA                                                          Intermolecular distance restraints derived 15N- <br />       XPA67 80 peptide corresponding residues 67&#226;&#8364;&#8220;80 XPA          dispersed NOE-HSQC experiment acquired D,15N-ERCC1/ <br />                                             F75A <br />       protein mutant peptide XPA67 80    synthesized solid     U-XPA sample using 200 ms mixing time. total 23 intermole- <br />       phase methods, HPLC-purified Molecular Biology Core          cular distance restraints amide protons ERCC1       Facility Tufts University (Boston, MA). 40-mer DNA oligomer         protons XPA derived experiment. structure <br />       50 -CCGGTGGCCAGCGCTCGGCGT20&#226;&#8364;&#8220;30 50 6FAM <span id='am-66' about='rdfs:label' typeof='owl:Thing'>label</span> (Inte-               ERCC1&#226;&#8364;&#8220;XPA complex calculated using simulated <br />       grated DNA Technologies) gel-purified conventional                annealing procedure XPLOR-NIH (Schwieters et al, 2003).       techniques.                                                              total energy term used calculation incorporated                                                                                NMR-derived distance restraints 4  X-ray data.                                                                                lowest energy structures 100 calculated deposited       Protein expression purification <br />                                                                                PDB accession code 2JNW. solvent accessible <br />       central domain ERCC1 (constructs ERCC192 214                                                                                surface areas calculated lowest energy structure using <br />       ERCC196 214) N-terminal His6 tag expressed       purified previously described (Tsodikov et al, 2005). Fragments       Surface Racer 4.0 (Tsodikov et al, 2002) solvent probe <br />                                                                                              &#203;&#353;. <br />                                                                                radius 1.4       XPA protein (XPA1 273, XPA59 273, XPA59 219, XPA59 93)       cloned pET19bpps, N-terminal  10 tag       separated XPA sequences Prescission protease               Competitive binding equilibrium titrations <br />       cleavage site (gift Dr Tapan Biswas), NdeI XhoI           Fluorescence anisotropy measurements performed pre- <br />       sites. length XPA protein expressed bacteria            viously reported (Tsodikov et al, 2005). equilibrium titrations <br />       pET15b-XPA purified Ni2 &#195;&#190; -NTA, gel filtration, heparin       analysis described supplementary data. <br />       chromatography. Protein expression purification described <br />       Supplementary data. <br />                                                                                Construction expression mutant XPA proteins <br />       Analytical ultracentrifugation                                           Site-directed mutagenesis using QuikChange kit (Stratagene) <br />       Sedimentation equilibrium experiments ERCC192 214           introduced point mutations expression vector pET15b-XPA <br />       complex ERCC192 214&#226;&#8364;&#8220;XPA59 93 performed using Beckman                performed. pET15b-XPA served template oligonucleo- <br />       XLA Analytical Centrifuge. cases, proteins               tide primers used generate mutations contained desired <br />       concentrations 0.3&#226;&#8364;&#8220;0.5 mg/ml NMR Buffer (20 mM Tris buffer         mutation, marker restriction site selection. following <br />       pH 7.2, 50 mM NaCl, 2 mM b-mercaptoethanol 0.1 mM EDTA).             primers used (restriction site underlined indicated, <br />       Sedimentation equilibrium data analyzed described         modified nucleotides shown italics): <br />       Supplementary data.                                                         XPA-F75A: GACACAGGAGGAGGCGCCATTCTAGAAGAGGAAGAAG <br />                                                                                (XbaI) <br />       Crystallization XPA&#226;&#8364;&#8220;ERCC1 complex, data collection                   XPA-DG74:      GACACAGGAGGATTCATTCTAGAAGAGGAAGAAG <br />       analysis                                                                 (XbaI) <br />       Protein crystallization X-ray data collection described      XPA-DG73/74: GATAATTGACACAGGATTCATTCTAGAAGAGGAA <br />       Supplementary data. complete redundant X-ray data set              GAAG (XbaI). <br />       collected processed using HKL2000 (Otwinowski                   Positive clones fully sequenced rule introduction <br />       Minor, 1997). crystals belong space group I4132          additional mutations. Mutant XPA proteins expressed       ERCC1&#226;&#8364;&#8220;XPA complex asymmetric unit. structure              Escherichia coli BL21(DE3)pLyS cells purified chromato- <br />       determined molecular replacement (MR) methods using               graphy nickel-NTA, gel filtration heparin columns. <br />       program PHASER (McCoy et al, 2005) crystallographic <br />       model ERCC1 central domain (Tsodikov et al, 2005; PDB code <br />       2A1I) residues C-terminal residue 214 deleted.      Nuclease assay <br />       difference (Fo Fc) electron density map calculated phases         ERCC1-XPF purified nuclease assays using stem-loop <br />       MR solution revealed bound XPA peptide. XPA             substrate carried described previously (Enzlin       peptide built difference density using distance             Scha&#194;&#168;rer, 2002)  Supplementary data details). <br />       restraint information NMR experiments structure       complex refined described  strong <br />                                                                                DNA-binding assays <br />       geometric restraints imposed ERCC1 subunit                                                                                way junction DNA substrate described previously <br />       low-resolution diffraction data absence intramolecular <br />                                                                                (substrate 7 Table 1 Hohl et al, 2003) 50 -32P-end labeled <br />       distance information ERCC1. experimental XPA&#226;&#8364;&#8220;ERCC1 <br />                                                                                incubated 1 nM concentration various amounts       distance restraints accommodated violations <br />                                                                                XPA EMSA buffer (25 mM HepesKOH pH 8.0, 30 mM KCl, 10% <br />       using structure unbound ERCC1, suggesting ERCC1 <br />                                                                                glycerol, 1 mM DTT, 1 mM EDTA, 0.1 mg/ml BSA) reaction <br />       does undergo significant conformational changes binding <br />                                                                                volume 15 ml. equilibration room temperature 30 min, <br />       XPA. <br />                                                                                samples loaded 5% (37.5:1) native polyacrylamide <br />                                                                                gel containing 0.5   TBE electrophoresed 90 V 2 h. <br />       NMR experiments determination structure                       Gels dried radioactive bands visualized autoradio- <br />       XPA&#226;&#8364;&#8220;ERCC1 complex                                                     graphy. <br />       NMR data acquired NMR Buffer described  <br />       protein concentrations 0.25 mM free ERCC196 214       ERCC192 214  behaved similarly experiments), 0.25 mM        vitro NER assay <br />       ERCC192 214 complex synthetic XPA67 80 peptide         assay performed using established protocol (Shivji <br />       0.1 mM ERCC192 214 complex XPA59 93 fragment. Higher            et al, 1999), described Supplementary data. <br />       protein concentrations resulted line broadening lower quality <br />       NMR spectra. Backbone assignments free ERCC192 214       ERCC1&#226;&#8364;&#8220;XPA67 80 complex performed using standard set                <br />       triple-resonance experiments: HNCA/HN CA, HN(CA)CB/                <br />       HN(COCA)CB HNCO/HN(CA                                            <br />  <br /> </body></html>