<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL VIROLOGY, OCt. 1993, p. 5922-5931                                                                            Vol. 67,  10 <br /> 0022-538X/93/105922-10$02.00/0 <br /> Copyright X 1993, American Society Microbiology <br />  <br />  <br />              Regions Herpes Simplex Virus Type 1 UL42 <br />                Protein Required Functional Interaction <br />                         Viral DNA Polymerase <br />                        STEVEN J. MONAHAN,&apos; TAMAR F. BARLAM,2 CLYDE S. CRUMPACKER,2 <br />                                          DEBORAH S. PARRIS&quot;* <br />                   Department Medical Microbiology Immunology Comprehensive Cancer Center, <br />                      Ohio State University, Columbus, Ohio 43210,1 Charles  Dana Research <br />                           Institute Harvard-Thorndike Laboratory Beth Israel Hospital                             Department Medicine, Division Infectious Disease, Beth Israel <br />                            Hospital Harvard Medical School, Boston, Massachusetts 022152 <br />                                              Received 24 March 1993/Accepted 12 July 1993 <br />  <br />               essential gene products herpes simplex virus type 1, viral DNA polymerase (pol) UL42,            accessory protein, physically functionally interact form core viral DNA replication complex. <br />            Understanding essential interaction provide basis develop novel anti-herpesvirus <br />            agents. previously shown coexpressed vitro transcription-translation  UL42 <br />            stimulates pol activity (M. L. Gallo, D.  Dorsky, C. S. Crumpacker, D. S. Parris, J. Virol. 63:5023-5029, <br />            1989). analyzing various insertion, deletion, frameshift mutations UL42             C-terminal 149 amino acids dispensable ability protein stimulate pol activity. addition, <br />            distinct internal regions UL42 required pol stimulation. Regions II            defined lie <span id='am-21' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> residues 129 163 residues 202 337, respectively.            physical association examined antibody UL42, pol coimmunoprecipitate            level expressed UL42 mutant protein lacking region wild-type UL42.  mere <br />            physical association insufficient stimulation pol activity. Deletion region H reduced eliminated <br />            coimmunoprecipitation pol. Interestingly, antibody pol specific residues 1216 1224 <br />            coimmunoprecipitated UL42 proteins synthesized baculovirus expression            rabbit reticulocyte lysates. results indicate   portion region recognized            pol antiserum accessible pol-UL42 heterodimer (ii) immunoprecipitation results products <br />            different expression systems vary.  distinct regions UL42 essential            functional interaction pol.  results point UL42 region function physical <br />            association pol. <br />  <br />    Herpes simplex virus type 1 (HSV-1), replicates              DNA-protein interactions (32), interactions nuclei eukaryotic cells, provides unique model                  likely involved eukaryotic DNA replication.  eukaryotic DNA replication                protein-protein DNA-protein interactions involving proteins required genome replication virus encoded               seven genes required HSV-1 dependent replication <br /> (4), sequence entire 152-kbp genome known                  documented. product UL9 sequence- <br /> (34), abundance viral mutants (46) lends            specific DNA-binding protein binds HSV-1 powerful genetic analysis. HSV-1 DNA replica-                sequences (16, 37) possesses helicase activity important <br /> tion reconstituted vitro, transient transfec-            initiation (1, 17). gene products (UL5, UL8, <br /> tion mammalian cells plasmids containing HSV-1                UL52) form stable protein-protein complex origin replication   various HSV-1 genes                   possesses helicase primase activities (3, 10), revealed seven viral genes necessary sufficient             products coimmunoprecipitated HSV-1- amplify containing plasmids (53). Analysis host-                infected cells specific antibody UL5 protein (55). <br /> range temperature-sensitive mutants defects                  UL5 UL52 sufficient helicase-primase enzymatic genes confirmed seven gene                  activities (3, 14), UL8 involved stabilizing <br /> products essential viral DNA replication host cells           interaction primers DNA template (48). <br />  review, reference 46).  likely             remaining proteins DNA polymerase (pol, viral cellular proteins required infected            product UL30 [27, 42]), 65-kDa double-stranded DNA- <br /> cells nucleic acid metabolism cleavage            binding protein (65KDBP, product UL42 [33, 41]), packaging unit-length genomes (5, 44).                                single-stranded major DNA-binding protein (ICP8, product <br />    proteins involved DNA replication charac-              UL29 [8, 52]) reported physically <br /> terized prokaryotic systems, Eschenchia coli onC <br /> plasmids T4 T7 bacteriophages, form multipro-                interact (19, 50, 51). Results laboratory tein complexes held protein-protein                   demonstrated copurification UL42 <br />                                                                          pol standard biochemical chromatography                                                                          immunoaffinity chromatography employing monoclonal  <br />                                                                          tibodies (MAbs) ULA2 (9, 19, 50). cases, ICP8 <br />   * <br />       Corresponding author.                                              copurified pol UL42 (19, 50). HSV-1 <br />                                                                   5922 <br />  VOL. 67, 1993                                                     FUNCTIONAL INTERACrION HSV-1 PROTEINS                  5923 <br />  <br /> pol expressed heterologous systems possesses catalytic         frame insertion mutations HpaI NdeI sites, <br /> activity (15, 18, 19, 24-26, 31), UL42 stimulates pol activity    respectively.  sevenfold (18), increases processivity pol        Deletion mutations ULA2 gene primed single-stranded DNA templates (24, 26),            removing internal restriction fragments. d129-163 mu- defined pol accessory protein (18, 24, 26). ICP8    tation created digestion pLBN 19A NdeI reported stimulate pol activity twofold (36, 45),    AatII, removal single-stranded overhangs Si group reported required UL42        nuclease, religation. d141-206 mutation  <br /> increase processivity pol (26).  pro-       structed digestion pLBN 19A NdeI HpaI, <br /> teins colocalize replication compartments          removal single-stranded overhangs Si nuclease, productively infected cells (2, 11, 22, 23, 43).                  religation. deletion mutation d37-282  desig- <br />    shown previously carboxy-terminal         nated APst) described previously (18). create <br /> (C-terminal) 227 amino acids pol involved           d202-337 deletion mutation, BamHI EcoRI frag- <br /> sufficient complex formation UL42 basis        ment polylinker pLBN 19A removed coimmunoprecipitation vitro transcription-translation       delete SstI site vector, 398-bp SstI <br /> products pol-specific antibody (13). groups         fragment UL42 gene removed form pLB <br /> attempted recently delineate C-terminal pol        ASst. removal SstI fragment placed C-ter- <br /> domain involved physical association UL42.       minal region frame, BamHI linker introduced Tenney et al. (49) shown 19           unique SstI site create frame d202-337 muta- <br /> amino acids pol required functional physical    tion. <br /> association UL42, C-terminal 19 40 residues              Nested deletions UILA2 open reading frame <br /> stabilize physical association UL42,      created unique restriction sites digestion residues required pol activity stimulated   exonuclease III (United States Biochemical, Cleveland, UL42. contrast, Digard et al. (12) failed observe        Ohio), removal single-stranded ends mung bean <br /> coimmunoprecipitation truncated pol polypeptides lack-         nuclease, ligation T4 DNA ligase. Deletion d140 <br /> ing C-terminal 19 residues UL42 produced           generated NdeI site pLBN 19A. vitro translation.  exact region pol         sequences constructs regions altered sub- <br /> required complex formation UL42 remains unclear.         cloned confirmed dideoxy chain termination  domains UL42 required functionally            method Sequenase 2.0 according instructions interact pol defined. determine         manufacturer (United States Biochemical). <br /> domains UL42 required functional physical            vitro transcription translation. Linearized plasmids <br /> association pol, exploited fact cotrans-    transcribed T7 RNA polymerase (Promega Bio- <br /> lation products UL42 pol transcripts rabbit reticu-     tec, Madison, Wis.) 1 h 40&#194;&#176;C. Transcripts locyte lysates possess sevenfold pol activity     translated rabbit reticulocyte lysates (Promega Biotec) translation products pol transcripts (18).        1 h 30&#194;&#176;C final concentration 5 20 ng RNA present study, localized distinct regions          ,l. Cotranslation reaction mixtures contained approximately <br /> UL42 required stimulation pol activity. results        equal amounts UL42 pol RNA, judged gels <br /> demonstrate domains particularly sensi-      containing serial twofold dilutions RNA species. tive disruption mutation required functional     experiments, L-[35S]methionine (specific activity, physical interaction pol.                            1,000 1,300 Ci/mmol; Amersham, Arlington Heights, Ill.) <br />                                                                   added translation reaction mixtures final concen- <br />                 MATERIALS METHODS                             tration 1 ,uCi/,ul. Labelled translation products                                                                   analyzed electrophoresis denaturing, polyacryl- <br />    Plasmids. HSV-1 pol UL42 genes subcloned          amide gradient (10 20%) gels, gels impreg- phagemid vectors sense orientation downstream         nated En3Hance (DuPont, Boston, Mass.), dried, T7 RNA polymerase promoter. Plasmid pGEM2-               visualized fluorography (33). <br /> 702, encodes length pol, amino-terminal           pol assays. HSV-1 pol activity measured deletion mutants, pT7-7.1 pT7-7.2, encode        incorporation [3H]<span id='am-11' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span>dTTP</span> (43 82 Ci/mmol; ICN 67 27 amino acids, respectively,          Amersham) trichloroacetic acid-insoluble radioactivity <br /> described previously (15). Plasmid pLBN 19A,             described previously (15, 18). assay vitro translation <br /> codes length UL42 (18), used wild-type UL42       products, reactions containing 2 10 p,g maximally <br /> construct, mutant forms UL42 derived          activated calf thymus DNA template initiated plasmids routinely propagated E. coli JM109        25 pl translation reaction mixtures. relative stability <br /> HB101 purified use Quiagen Maxi-Prep col-           UL42 mutant protein assayed incubation umns (Quiagen , Chatworth, Calif.) according           labelled translation mixtures pol assay conditions <br /> instructions manufacturer.                                  omission bovine serum albumin [BSA], deoxy- <br />    Construction UL42 gene mutations. C-terminal deletions      <span id='am-16' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span>nucleoside</span> triphosphates, template) 37&#194;&#176;C 30 min, UL42 expressed runoff transcripts produced           followed sodium dodecyl sulfate-polyacrylamide gel elec- wild-type UL42 plasmid linearized      trophoresis (SDS-PAGE) analysis. <br /> UL42 gene mutagenized insertion BamHI linkers            Preparation proteins expressed recombinant baculo- <br /> (8-, 10-, 12-mer; New England Biolabs, Beverly, Mass.)        viruses. Recombinant baculoviruses (Autographica califor- introduce frameshift (fs) mutations. Mutant constructs         nica) express HSV-1 UL42 HSV-1 <br /> N459(fs), N248(fs), N206(fs), N140(fs) encode polypep-        pol gene polyhedrin promoter kind gifts tides containing wild-type UL42 amino acids amino        Mark Challberg (National Institutes Health, Bethesda, <br /> terminus (N terminus) residue indicated         Md.) Robert Lehman (Stanford University, Stanford, <br /> frameshifts created NruI, NotI, HpaI, NdeI             Calif.), respectively. wild-type strain baculovirus restriction sites, respectively. Plasmids i206 i140 contain   Sf9 insect cell line used propagate  5924    MONAHAN ET AL.                                                                                                             J. VIROL. <br />  <br /> baculovirus strains kindly provided Fred Hink             TABLE 1. pol activities amino-terminal deletion mutants <br /> (Ohio State University, Columbus). Sf9 cells routinely                UL30 coexpressed vitro UL42&apos; <br /> propagated Grace medium (GIBCO-BRL, Grand Island, <br /> N.Y.) supplemented 10% <span id='am-6' about='obo:IMR_0200277' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00243" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Lactose" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200277" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lactose</span> hydrolysate (Difco               UL30         Polypeptide         Pol activity (units)&apos;        Stimulation <br />                                                                    construct      composition        -UL42           +UL42           indexc <br /> Laboratories, Detroit, Mich), 10% tissue culture yeastolate <br /> (Difco Laboratories), 100 U penicillin ml, 100 ,ug     pGEM2-702           1-1235       116 &#194;&#177; 62        1080 &#194;&#177; 674           9.3 <br /> streptomycin sulfate ml, 10% fetal bovine serum          pT7-7.2            28-1235       418 &#194;&#177; 121       2416 &#194;&#177; 277           5.8 <br /> (Flow Laboratories, McLean, Va.) 28&#194;&#176;C infected       pT7-7.1            68-1235       311 &#194;&#177; 109       2251 &#194;&#177; 185           7.2 baculovirus input multiplicity 5 PFU cell.       UL30 RNA transcribed linearized plasmid translated prepare proteins immunoprecipitation, L-[35S]methio-      UL42 RNA rabbit reticulocyte lysates 60 min 30&#194;&#176;C. added 12 h postinfection final concentration      b Units pol activity defined femtomoles [3H]dTTP 50 ,uCi/ml, high salt nuclear extracts prepared      incorporated activated calf thymus DNA 37&#194;&#176;C 30 min. Activities essentially described Gallo et al. (18) cells         expressed means + standard deviations. pol assays performed harvested 36 48 h postinfection, 10 strokes    triplicates independent experiments pGEM2-702 pT7-7.2                                                                  experiments pT7-7.1. Dounce homogenization used disrupt cyto-            c (Mean pol activity + UL42)/(mean pol activity - UL42). <br /> plasmic membranes Sf9 cells. <br />   Antibodies coimmunoprecipitation. MAb 6898, specific UL42, described previously (41) kindly <br /> provided mouse ascites fluid Howard Marsden (Uni-          pared activity translation products pol RNA <br /> versity Glasgow, Glasgow, Scotland). epitope recog-       (18). UL42 mutant RNA encoding nized MAb 6898 shown immunodominant            product lacking 28 C-terminal residues resulted wild- maps amino acids 363 369 (35). addition,          type levels pol stimulation, failure large amino- <br /> polyclonal antibody residues region UL42         terminal (N-terminal) C-terminal deletion mutants raised rabbits. 40-amino-acid <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> synthe-       UL42 stimulate pol activity study (18) suggested <br /> sized contained UL42 residues 360 377 linked     useful delineating amino terminus promiscuous T-cell epitope mea-         domains UL42 required functional interaction sles virus (47) 4-residue linker consisting Leu-<span id='am-28' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-29' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Gly" datatype="xsd:string"></span>Gly</span>-    pol.  sensitivity screen <br /> Pro-Ser. linked B- T-cell epitopes capable       large numbers UL42 mutations hindered low <br /> eliciting antibody response rabbits          variable activities translation products wild- <br /> coupled protein carrier, resulting antibody 834.       type pol RNA. <br /> synthetic peptide corresponding residues 1216 1224         suggested pol production regulated HSV-1 pol coupled BSA (Sigma, St. Louis, Mo.)         HSV-infected cells posttranscriptionally atten- injected rabbits produce pol-specific anti-      uation mechanism resulting secondary-structure forma- <br /> serum 771. peptides synthesized Peptide         tion N-terminal end pol RNA (54). sec- Protein Engineering Laboratory Ohio State Univer-         ondary-structure formation affect ability sity (Columbus). specificity antipeptide serum       RNA translated vitro, tested N-terminal confirmed immunoprecipitation presence         deletion mutants pol abilities stimulated absence cognate peptide.                                  wild-type UL42 vitro transcription-translation <br />    pol UL42 transcripts (0.5 3 ,ug RNA)       (Table 1). Transcripts mutant pol  <br /> translated 70 ,u rabbit reticulocyte lysate described   structs previously reported translated  removal 7-pI aliquot, 3 volumes B2          greater efficiency wild-type, translation <br /> buffer (10 mM Tris-HCl [pH 8.0], 10% glycerol) added.       products possessed pol activity (15). results solution, equal volume 2x Zweig&apos;s buffer (0.2      present study demonstrate wild-type pol construct <br /> M Tris-HCl [pH 8.0], 20% glycerol, 1.0% Nonidet P-40,            previously used, pGEM2 702, encodes 1.0% sodium deoxycholate, 0.4 mM phenylmethylsulfonyl            length (1,235-amino-acid) protein, yielded barely <br /> fluoride) added. Samples precleared incubation       detectable pol activity  particular room temperature 0.1 volume 50% (wt/vol)          assay activity stimulated 9.3-fold RNA <br /> slurry protein Sepharose CL-4B (Sigma) 1 x Zweig&apos;s       cotranslated wild-type ULA2 RNA (Table 1). buffer containing 0.02% sodium azide. Supernatants          mutant pol transcripts encode proteins split 1.5-ml tubes gently mixed 20 25 pI    27 (pT7-7.2) 67. (pT7-7.1) amino acids deleted antiserum overnight 4&#194;&#176;C. Immune complexes             yielded higher pol activities wild-type trans- <br /> recovered addition 0.2 volume protein          lated  confirming earlier report (15). Furthermore, Sepharose 1 h 4&#194;&#176;C. Sepharose beads             pol activities pT7-7.1 pT7-7.2 transcripts washed times buffer containing 0.1 M Tris-HCl (pH      stimulated 7.2- 5.8-fold, respectively, cotranslation <br /> 8.0), 0.5 M NaCl, 1.0% 2-mercaptoethanol. Bound              UL42 transcripts (Table 1). observed protein eluted boiled 3 min 25 RI         variability experiment experiment pol activities <br /> dissociation buffer (0.125 M Tris-HCl [pH 6.8], 2% SDS, 2%       produced transcripts mutated pol genes glycerol, 5% 2-mercaptoethanol, 0.004% bromophenol blue)         produced transcripts wild-type gene. analyzed SDS-PAGE.                                    basis somewhat higher stimulation indices observed <br />                                                                  pT7-7.1 construct, selected func- <br />                           RESULTS                                tional domain mapping studies. <br />                                                                     C-terminal deletions UL42. determined   Analysis  pol constructs. previous report, ob-      C-terminal region UL42 protein served  sevenfold stimulation pol activity     required ability stimulate pol activity. C-ter- <br /> HSV-1 pol RNA transcribed vitro cotranslated           minal mutant forms UIA2 protein (Fig. 1) rabbit reticulocyte lysates wild-type UL42 RNA com-         generated ways:   cleavage restriction <br />  VOL. 67, 1993                                                                         FUNCTIONAL INTERACTION HSV-1 PROTEINS                    5925 <br />  <br />  <br />                                                                                   V      0.0        E          -    - <br />                                                                                      IZZ         C/)        x <br />                                  pLBN19A         F <br />  <br />  <br />                                                UL42 ORF                                                            ) (488AA) <br />                                                       wt <br />                                                       N459 <br />                                                       N339 <br />                                                       N248 <br />                                                       N206 <br />                                                       N139 <br />   FIG. 1. Generation C-terminal deletion mutants ULA2. Schematic diagram ULA2 gene cloned phagemid vector pTZ <br /> 19U downstream T7 RNA polymerase promoter. Transcripts encoding C-terminal deletions produced   linearizing open reading frame (ORF) restriction sites indicated produce runoff transcripts (ii) inserting linkers sites introduce <br /> frameshift mutations amino acid (AA) indicated. wt, wild type. <br />  <br />  <br /> enzyme indicated open reading frame                                 translation mixtures containing pol RNA consis- <br /> wild-type gene, followed production runoff transcripts;                         tent competition UL42 RNAs limited translation <br /> (ii) deletion sequences encoding C-terminal end;                         machinery observed previously (18). fact, (iii) insertion linkers various sizes frame                    observed reduction approximately twofold indicated sites. method produces C-termi-                            pol polypeptide synthesized presence nal tails nonrelated sequence, linkers inserted                            wild-type mutant UL42 RNA, compared create different frame mutations. activities                        observed absence competing RNA (results products pT7-7.1 transcripts translated (pol                         shown). C-terminal mutant protein generated trunca- <br /> control), wild-type UL42 transcripts (normalized                          tion UL42 gene SmaI site (N339) produced <br /> 100%), ULA2 mutant transcripts                                wild-type levels pol stimulation, did N459 muta- <br /> indicated Table 2. results indicate UL42 proteins                         tions (Table 2). results demonstrate C-termi- <br /> containing 139 (140 generated frameshift), 206,                      nal 149 amino acids UL42 dispensable protein&apos;s 248 N-terminal residues, regardless method                               ability stimulate pol activity. generated, failed stimulate pol activity                              failure mutant UL42 proteins stimulate pol observed absence mutant UIA2 protein.                            activity instability mutant fact, pol activities approximately twofold                        proteins.  relative stability mutant <br /> presence mutant proteins absence                          protein measured expression rabbit reticulocyte <br /> UL42 transcript. reduction pol activity 50%                         lysates. mutant proteins incubated pol <br />                                                                                       reaction conditions (+) compared proteins                                                                                       reactions held 0&#194;&#176;C absence pol reaction buffer <br />     TABLE 2. pol stimulation C-terminal deletion mutants                           (-). Figure 2 shows profile polypeptides                           UL42                                                     representative samples analyzed SDS-PAGE.                                            pol stimulation (%)                        mutant polypeptide species indicated small open <br /> Construct                                                                             circles. results indicate mutations <br />                            Truncation&apos;                  Frameshift mutationc          generated linearizing DNA template UL42 <br /> Wild type                     lood                               lood                 open reading frame relatively unstable pol reac- <br /> N459                          110                                 81                  tion conditions (compare lane 4 lane 5 lane 10 N339                          100                                NDe                  lane 11). frameshift mutations generating N248                             11                                8                  UL42 N-terminal sequences,  stable (com- <br /> N206                             13                                7                  pare lanes 6 7 lanes 12 13). results suggest <br /> N139                              7                               11                  UL42 proteins possessing C-terminal truncations       pol stimulation calculated ratio pol activity measured       greater 149 residues unstable stabilized <br /> vitro cotranslation products pol mutant ULA2 RNAs measured             presence polypeptide nonrelated sequence cotranslation pol wild-type UL42 RNAs. pol (UL30) RNA                  C terminus. stable polypeptides produced transcribed linearized pT7-7.1  translated  yielded average activity 239 units.                                                        N140(fs), N206(fs), N248(fs) frameshift mutations <br />   b Mutant UIA2 RNAs generated linearizing UIA2 open               yielded stimulation pol activity (Table 2), reading frame.                                                                        conclude sequences insufficient functional <br />   c Mutant UL42 RNAs generated frameshift mutations linearized              interaction pol. <br /> downstream ULA2 gene. <br />   d (Activity pol mutant ULA2 translation products/activity pol            multiple polypeptide species produced wild- <br /> wild-type UL42) x 100.                                                                type mutant transcripts observed previously <br />   &apos; ND,                                                                      (13, 15, 18) likely initiation translation <br />  5926      MONAHAN ET AL.                                                                                                              J. VIROL. <br />  <br />                                                                               cleaved DNA N140(fs) mutant NotI site <br />                                                                               (wild-type residue 248) instead XbaI site (Fig. 2B, <br />                                                                               lane 2). cleavage NotI site predicted <br />                                                                               affect length product resulting N140(fs),            *(6 <br />            6A    1slf                       1                - -- <br />                                                                               shortened product produced internal <br />                                                                               initiation methionine codon. size                                                                               predicted frameshifted product did change,                                                                               loss larger polypeptide species, predicted arise <br />                                                                         Ow.   internal initiation (Fig. 2B, compare lanes 1 2). <br />                                                                                likely larger polypeptide species <br />                                                                               observed N140(fs) mutant results internal <br />                                                                               initiation translation residue 158, downstream                                                                               frameshift mutation. Despite presence substantial <br />                         21       3   4           (6   7      ()               levels wild-type C-terminal peptide translation prod- <br />                                                                               ucts N140(fs) mutant RNA generated template <br />                                      NI 4()Ofi    NlI3)(1)                    cleaved XbaI, observed stimulation pol <br />                                                                             activity.  polypeptide corresponding residues <br />                                                                               158 488 UL42 insufficient functional interac- <br />                                                                               tion pol. <br />                                                                                  Internal insertion deletion mutations. means                                                                               defining critical domains UL42 required                                                                            functional interaction pol, created frame <br />                                                                               insertion deletion mutations UL42 tested                                                                               stability (data shown) abilities stimulate pol <br />                                                                               activity described (Fig. 3). deletion <br />                                                                               mutants contain insertions novel amino acids, indi- <br />                                                                               cated, result mutation itself result                              c                                                insertion additional sequences place frame <br />                                                                               deletion mutation frame. mutant forms <br />                                                                               indicated stable pol reaction conditions (results <br />    FIG. 2.   stability C-terminal deletion mutants            shown). confirmed d37-282 deletion mutant, <br /> UL42. Transcripts encoding representative deletion mutants               used previous study wild-type pol construct <br /> Table 1 translated rabbit reticulocyte lysates presence        (18), failed stimulate activity pT7-7.1 pol [35S]methionine frozen immediately (-) incubated             construct (Fig. 3). smaller deletion mutants polymerase assay conditions (+) prior SDS-PAGE 10 20% gradient polyacrylamide gels. Mutants designated               region UL42 failed stimulate pol UL42 amino acid truncation (t), frameshift (fs),               activity, including nonoverlapping mutations, d129-163 frame insertion mutation  . position length             d202-337. Insertion novel amino acids product denoted circle. (B) Internal initiation products           residue 206 (i206) little effect ability protein <br /> UL42 expressed downstream frameshift mutations. Tran-                    stimulate pol activity compared wild-type UL42 <br /> scripts produced pLBN 19A linearized downstream                   protein.  insertion residues amino <br /> UL42 gene (lane 5) internal NdeI site (lane 3)                 acid 140 (i140) completely abolished pol-stimulating <br /> translated described panel  Transcripts produced           activity UL42. sequence inserted i140 <br /> template encoding frameshift residue 140 linearized          encodes proline residues, pro- <br /> XbaI (X) downstream UL42 gene (lane 1) NotI (N) <br /> site gene downstream frameshift mutation (lane          effects secondary tertiary structures 2); transcripts translated analyzed SDS-PAGE                   distance site insertion, created subtle <br /> described panel  product denoted produced                mutation site deleting residue 140.  <br /> initiation AUG UL42 gene. wt, wild type.                  ingly, d140 stimulated pol activity, consistently                                                                               half level observed wild-type UL42. Taken <br />                                                                                results indicate separable <br />                                                                               domains required UL42 stimulate pol activity. internal AUG codons.  apparent molecular                     referred region  maps masses consistent hypothesis. observed                    amino acids 129 163 sensitive pertubations <br /> unexpected species apparent molecular mass 44                      produced insertions deletions region. kDa (small arrow) translation products N140(fs)                   domain, denoted region II, maps residues 202 <br /> mutant transcriptional template linearized                   337. XbaI site polylinker downstream UL42                        Coimmunoprecipitation pol UL42 proteins. gene (Fig. 2B, lane 1). species size apparently                    ability UL42 stimulate pol activity result <br /> larger predicted mutant protein,                     multiple attributes functions UL42 protein, comigrated product predicted initiated                  investigated second property UL42: ability phys- AUG codon wild-type protein.                  ically associate pol. previously used methionine codon (amino acid 158) located                       coimmunoprecipitation assay pol antiserum anti- <br /> stream frameshift mutation N140(fs), internal trans-                serum UL42 determine domains pol required lation initiation site yield protein                     physical association UL42 (12, 13, 49). likely <br /> wild-type sequence amino acids 158 488, lacking                   interfering pol-UL42 complex formation N terminus. determine product                     abolish UL42&apos;s pol-accessory functions. assess initiation translation AUG codon,                     region II deletions abolished ability UL42  VOL. 67, 1993                                                             FUNCTIONAL INTERACITION HSV-1 PROTEINS                      5927 <br />  <br />                                               UL42 Mutations                        Pol Activity Stimulation <br />                                                                                          (%/) <br />  <br />                         wt                                                                100                   + <br />  <br />                         i206                                                             85 <br />                                                      RGSA <br />                         i140                                                                                                               16 <br />                                               PDPV <br />                         d140                     n                                         50                 +l- <br />                         dl 29-163               :                                          17 <br />                         d141-206                                                           14 <br />                                                        RI <br />                         d37-282                        Rl                                   11 <br />  <br />                         d202-337                                  ,1                     15 <br />                         _    .   _                                LRDPA <br />                         Pol Control                                                        20 <br />  <br />  <br />                                                            II <br />    FIG. 3. Stimulation pol activity insertion deletion mutants UL42. Schematic diagram insertion mutants inserting frame linkers convenient restriction sites deletion mutants constructed described Materials Methods. pol activities vitro translation products produced coexpression RNA encoding UL30  pT7-7.1) UL42 mutant reported percentages activity produced pol wild-type (wt) UL42 transcripts. UL42 mutants stimulated pol activity greater 50% level measured wild-type UL42 considered positive (+). Activities averages experiments. <br />  <br />  <br />  <br />  <br /> stimulate pol activity abolishing physical interaction                 region region II UL42 mutants (lanes 14 pol, utilized coimmunoprecipitation assay detect                       11). Likewise, pol antibody 771 immunoprecipitated complex formation. pol UL42 transcripts cotrans-                      140-kDa length cognate protein pol- <br /> lated immunoprecipitated antibody 834, directed                    related species (lane 6). UL42 antibody pos- <br /> UL42 residues 360 377, antibody 771, directed                    sesses cross-reactivity pol, pol residues 1216 1224 (Fig. 4). specificity                     low level pol adheres nonspecifically protein <br /> UL42 antipeptide serum 834 demonstrated precip-                      Sepharose, absence antibody itation 63-kDa product translations wild-type                 extensive preclearing protein Sepharose (results UL42 RNA (lane 2) 61- 49-kDa products                       shown). nonspecific adherence accounts low <br />                                                                                level pol polypeptide immunoprecipitated                                                                                absence UL42 (Fig. 4, lane 5). transcripts encoding <br />                U        P       P    U    d202-337 P   d129-163 P              pol region mutant, d129-163, coexpressed            -  (-PI: <br />                uU   - U (Pi -        (   (P    U P        c IP                 immunoprecipitated antibody UL42, pol coim- <br />    Pol                                      Pol~~~~~~-((                       munoprecipitated (lane 14) levels comparable                                                                                observed wild-type UL42 (lane 8).  region does <br /> UL42 3-                                                                        appear required pol-UL42 complex formation. <br />                                                                                   contrast, observed extremely low level                mK                                                ~~~~~~~~~~~~~~~- <br />                                                                    AUL42 <br />                                                                                pol polypeptide immunoprecipitated UL42 <br />        1    2 3     4  5 6    7 8 9       100    12   13 &apos;4 15 <br />    FIG. 4. Immunoprecipitation pol-UL42 complex formed               region II mutant protein antibody 834 (Fig. 4, lane 11). <br /> vitro. Translation products produced UL42 RNA (U), pol               level pol polypeptide precipitated RNA (P), UL42 pol RNA expressed (U/P), UL42                       presence region II mutant similar region mutant pol RNA (d129-163/P), UIA2 region II                  nonspecific level observed, relatively low sensitivity mutant pol RNA (d202-337/P). Translation products                    assay precludes definitive determination role electrophoresed immunoprecipitation (-)                 region II complex formation.  regions <br /> immunoprecipitated polyclonal antibody 834 UL42 (aU)                required functional interaction UL42 antibody 771 pol (cl). Translation products treated                pol. <br /> antibody (lanes 1, 4, 7, 10, 13) represent fifth sample <br /> immunoprecipitated. linotronix print derived                seen Fig. 4, UL42 coimmuno- <br /> exposure gels run parallel obtained scanning             precipitated pol-UL42 cotranslation products manipulating Adobe Photoshop 2.01 software (Adobe                     pol antibody 771 directed residues 1216 1224. Systems, , Mountain View, Calif.).                                         ability antibodies specific region coprecipitate <br />  5928         MONAHAN ET AL.                                                                                                                                       J. VIROL. <br />  <br />                      EXTRACTS                         6898              771       6898 771:                complex contains 1:1 ratio pol UL42 (9, 24). <br />                  ! <br />  <br />  <br />                  Y <br />                       F <br />  <br />  <br />                      MA. :j   1 <br />                                             {L1   .                                   CL; r <br />                                                                                                            viral proteins essential viral DNA <br />        200   - <br />                                                                                                            replication vivo (6, 27-29, 31, 42, 53). pol         97   - <br />                                                                                                            inherent enzymatic activity (15, 18, 24, 25, 30), UL42 <br />        69                                                                                                  stimulates activity pol (18, 49), acting processivity <br />                      1ffi: <br />                      *  _                                 *                           &quot;        ~~~~-UL42   factor vitro (24, 26) increasing affinity                   - <br />                  ._                       _ ~~~~~~~~~~~~~~~P <br />        46    - <br />                                                                                                            pol binds DNA template. heterologous <br />                                                                                                            expression systems, pol-UL42 complex forms        30 <br />                                                                                                            detectable immunoprecipitation antibodies pol <br />                                                                                                            UL42 (12, 13, 21, 49) (Fig. 4 5). examined <br />                      2           3    4     5     6   7   8    9   10         12 13    14     15 <br />                                                                                                            physical functional interactions UL30 (pol) <br />   FIG. 5. Coimmunoprecipitation pol-UL42 complex ex-                                                UL42 vitro expression  coexpressing <br /> pressed baculovirus recombinants. Nuclear extracts                                               pol mutant forms UL42 vitro rabbit reticulocyte <br /> prepared Sf9 cells metabolically labelled 12 48 h                                             lysates, identified distinct domains UL42 <br /> postinfection 50 ,uCi [35S]methionine ml, mock infected (M) infected wild-type  californica (W),                                         critical stimulation pol activity. <br /> pol-containing recombinant (P), UL42-containing recombinant                                                 ensure mutant forms UL42 failed (U), recombinants (U/P). Nuclear extracts electro-                                            stimulate pol lacked critical domains simply <br /> phoresed immunoprecipitation (lanes 1 5) immu-                                          degraded rapidly, UL42 mutants incubated noprecipitated MAb 6898 UL42 (lanes 6 9 14)                                              polymerase reaction conditions (Fig. 2) assess stability, <br /> polyclonal antibody 771 pol (lanes 10 13 15). Nuclear                                            proteins fold properly suscep- <br /> extracts treated antibody (lanes 1 5) represent tenth                                      tible increased proteolysis (40). sample immunoprecipitated.                                                                          exception C-terminal deletion mutants generated <br />                                                                                                            linearized template, mutant forms UL42                                                                                                            stable (Fig. 2).  deletion <br /> UL42 debate                                                 mutants created linker mutagenesis stable, exact C-terminal residues pol required                                                addition nonsense peptide C-terminal end physical association UL42 (12, 49). possible                                                     deletion mutants sufficient stability. fact interpretations failure pol antibody copre-                                              deletion insertion mutations UL42 <br /> cipitate UL42   antibodies pol sterically                                                resulted unstable polypeptides suggests muta- <br /> interfered ability pol UL42 associate                                               tions did result large perturbations global folding (ii) epitope recognized 771 masked pol                                                protein. UL42 physically associated, pol free.                                                 domains UL42 critical coimmunoprecipitate small amounts                                                   ability stimulate pol activity located amino <br /> pol UL42 vitro translation products                                                   thirds protein. C-terminal 149 amino acids UL42 antibody highly stringent wash conditions (500                                                  UL42 deleted affecting stability mM NaCl), appeared small percentage UL42                                                 ability stimulate pol activity (Table 2). coexpressing pol formed stable complexes rabbit reticulocyte                                                     vitro transcription-translation reactions internal insertion <br /> lysates. Furthermore, pol-specific antibody 771                                                deletion mutants UL42 pol (Fig. 3), avid 834 UL42-specific antibody (unpub-                                                   physically separable regions critical pol <br /> lished results), failure detect coprecipitation UL42                                             stimulation. Region located amino acids 129 pol antiserum reflected lack                                                      163, region II located amino acids 202 sensitivity assay   tested                                                 337. relatively large number amino acid <br /> ability pol antibody 771 coprecipitate                                                   residues encompassing region II, exclude proteins coexpressed high levels insect cells coinfected                                             possibility contains functional domain. pol UL42 baculovirus recombinants (Fig. 5).                                               information contained amino-terminal 157 resi- <br /> experiment, MAb 6898, recognizes essentially                                                     dues essential UL42 function, indicated region 834, employed UL42 antibody.                                                     failure initiation products AUG codon specificity antibody confirmed ability                                              stimulate pol activity (Fig. 2; Table 2). precipitate protein cells singly infected                                                Small alterations introduced region recombinant expressing cognate protein despite                                                  UL42 define residues critical presence large amounts labelled proteins                                                functional interaction pol. Deletion residue 140 extracts.  large amounts pol coimmu-                                                   (isoleucine) decreased pol stimulation twofold. Fur- <br /> noprecipitated MAb 6898 (ca-UL42) extracts                                                    thermore, insertion mutation amino acid 140 failed <br /> cells coinfected UL42 pol recombi-                                                   stimulate pol.  region close amino acid <br /> nants. apparent small UL42                                               140 particularly sensitive perturbations. cur- coimmunoprecipitated pol using 771 serum                                                    rently introducing small deletions amino acid <br />  pol). results demonstrate residues recog-                                                140 point mutations region define <br /> nized pol antiserum (1216 1224) accessible                                             residues critical functional interaction subpopulation complexes pol UL42                                                     pol. <br /> suggest essential physical                                                       determine region II involved interaction pol UL42.                                                                              physical association pol ULA2, mutant wild-type <br />                                                                                                            UL42 cotranslated pol vitro immunopre- <br />                                                 DISCUSSION                                                 cipitated antibody specific ULA2 (Fig. 4). pol                                                                                                            coimmunoprecipitated wild-type UL42 region   Analogously prokaryotic eukaryotic replica-                                                  mutant, d129-163, indicating region required tion systems, HSV-1 DNA polymerase multiprotein                                                   physical association pol. pol detected <br />  VOL. 67, 1993                                                      FUNCTIONAL INTERACTION HSV-1 PROTEINS                       5929 <br />  background levels coexpressed region              region essential physical interactions <br /> II mutant, nonspecific adherence pol poor            pol, required stimulation pol activity vitro. <br /> sensitivity assay make difficult assign        Disruption functional domains affecting protein- <br /> region II required complex formation.  low        protein interactions reported component level coimmunoprecipitation pol UL42 observed          HSV-1 helicase-primase complex. Single amino acid <br /> cotranslations wild-type UL42 pol           substitutions introduced conserved motifs UL5 dis- <br /> specific vitro translation used.      rupted function viral replication did affect <br /> noted wash conditions utilized immu-           UL5-UL8-UL52 complex formation (56). possible noprecipitations stringent (containing 500 mM             region mediates binding UL42 NaCl) used investigators (12, 13, 21, 49).      pol-UL42 complex DNA template. Chou-Fasman (7)  stringent conditions, ob-              Robson-Garnier (20) algorithms predict residues 130 <br /> served quantitative coimmunoprecipitation            139, 140 144, 145 162 form helix-loop-helix <br /> pol UL42 antibody UL42 extracts cells             motif. similar motif directly involved DNA coexpressed pol UL42 recombinant baculo-             binding prokaryotic regulatory proteins eukary- <br /> viruses (Fig. 5). differences observed ability    otic transcription factors (39). possible region proteins coimmunoprecipitated different          required maintain conformational integrity expression systems differences             UL42, perturbations result antibodies used, observed low-level coprecip-         mutations tested sufficient alter global folding <br /> itation pol UL42 vitro translations MAb        lead increased proteolysis. 6898 (results shown). possible differences       ability UL42 bind double-stranded DNA observed reflect poor ability proteins expressed        apparent sequence specificity (19, 19a, 33), neces- rabbit reticulocyte lysates assume conformation            sary purify mutant UL42 proteins determine <br /> necessary proteins form stable physical            abilities bind DNA altered. <br /> association. possible Kd pol-UL42         shown ULA2 multiple, separable  <br /> heterodimer low level protein expressed      mains required pol-accessory functions vitro.  <br /> rabbit reticulocyte lysates prevents formation signifi-       necessary define critical residues amounts stable pol-UL42.  associ-          region region II ULA2. ation proteins translated vitro sufficient      synthetic peptides corresponding linear arrays residues <br /> enable UL42 stimulate pol activity.                          ULA2 inhibit pol activity abilities <br />    possible explanation poor ability      interact DNA abilities inhibit pol UL42 coimmunoprecipitated vitro               interaction pol UL42 (38), care exer- <br /> translation products presence truncated poly-         cised defining peptide domains interfere peptides produced rabbit reticulocyte lysates result     ability pol UL42 functionally physically <br /> initiation translation internal AUG codons.            interact.  important gain complete <br /> products present high levels nuclear extracts      understanding function residues overall <br /> cells infected recombinant baculoviruses.            process viral DNA replication provide basis possible polypeptides sufficient      design peptidomimetic antiviral drugs. <br /> stimulate pol activity, compete extent wild-type UL42 protein physical association                                  ACKNOWLEDGMENTS pol. able observe               thank Mark Challberg Bob Lehman recombinant dominant negative effects competing polypeptides         baculovirus constructs, Howard Marsden Ann Cross MAb ability wild-type UL42 stimulate pol activity         6898, Fred Hink advice propagation baculoviruses (unpublished results), exclude possibility       insect cells, Pravin Kaumaya advice help designing polypeptides destabilize interaction          immunogenic peptides. <br /> wild-type UL42 pol.                                               work supported grant GM34930    ability detect coimmunoprecipitation UL42         National Institutes Health (D.S.P.) OSU Comprehensive <br /> pol antiserum residues 1216 1224 nuclear extracts        Cancer Center core grant CA16958 National Cancer cells coexpress proteins recombinant              Institute. S.J.M. supported National Research <br /> baculoviruses, vitro translation products,          Service award T32 CA09498-06 National Cancer Institute <br /> indicates caution exercised interpreting          fellowship F32 A108566-01 National Institutes                                                                     Health. T.F.B. supported National Institute Allergy negative immunoprecipitation results experiments            Infectious Diseases training grant 5T32 AI07387-03 AIDS  <br /> volving vitro translation products.  failure       search. <br /> pol polypeptide lacking 19 C-terminal residues copre- <br /> cipitate UL42 translation reactions (12)                                    REFERENCES <br /> success mutant proteins immunoprecipitated             1. Bruckner, R. C., J. J. Crute, M. S. Dodson,  R. Lehman. UL42 expressed Saccharomyces cerevisiae                    1991. herpes simplex virus 1 origin binding protein: DNA <br /> (49) accounted relatively unstable inter-             helicase. J. Biol. Chem. 266:2669-2674. <br /> action proteins present rabbit reticulocyte       2. Bush, M., D. R. Yager, M. Gao, K. Weisshart,   Marcy, <br /> lysates, protein  differences                  D. M. Coen, D. M. Knipe. 1991. Correct intranuclear <br /> posttranslational modifications expression systems.               localization herpes simplex virus DNA polymerase requires  observed significant levels             viral ICP8 DNA-binding protein. J. Virol. 65:1082-1089. <br />                                                                      3. Chadler, J. M., N. D. Stow. 1990. Herpes simplex virus coimmunoprecipitation region II UL42 mutant                   helicase-primase: UL8 protein required DNA- <br /> protein d202-337 cotranslations pol (Fig. 4),           dependent ATPase DNA helicase activities. Nucleic Acids <br /> await construction baculovirus recombinant ex-                 Res. 18:3573-3578. <br /> pressing mutations region definitive coim-          4. Challberg, M. D. 1986. method identifying viral genes <br /> munoprecipitation analysis.                                             required herpesvirus DNA replication. Proc. Natl. Acad. <br />  5930     MONAHAN ET AL.                                                                                                           J. VIROL. <br />  <br />     Sci. USA 83:9094-9098.                                              24. Gottlieb, J.,   Marcy, D. M. Coen, M. D. Challberg. <br />  5. Challberg, M. D., T. J. Kelly. 1989. Animal virus DNA               1990. herpes simplex virus type 1 UL42 gene product:     replication. Annu. Rev. Biochem. 58:671-717.                            subunit DNA polymerase functions increase proces- <br />  6. Chartrand, P., N. D. Stow, M. C. Timbury, N. M. Wilkie.             sivity. J. Virol. 64:5976-5987. <br />     1979. Physical mapping paar mutations herpes simplex          25. Haffey, M. L., J. T. Stevens, B. J. Terry, D.  Dorsky, C. S. <br />     virus type 1 type 2 intertypic marker rescue. J. Virol.          Crumpacker, S. M. Wietstock, W. T. Ruyechan,  K. Field. <br />     31:265-276.                                                             1988. Expression herpes simplex virus type 1 DNA poly- <br />  7. Chou, P. Y., G. D. Fasman. 1978. Empirical predictions           merase Saccharomyces cerevisiae detection virus- <br />     protein conformation. Annu. Rev. Biochem. 47:251-276.                   specific enzyme activity cell-free lysates. J. Virol. 62:4493- <br />  8. Conley,  J., D. M. Knipe, P. C. Jones, B. Roizman. 1981.          4498. <br />     Molecular genetics herpes simplex virus. VII. Characteriza-      26. Hernandez, T. R.,  R. Lehman. 1990. Functional interac- <br />     tion temperature-sensitive mutant produced vitro             tion herpes simplex-1 DNA polymerase UL42 <br />     mutagenesis defective DNA synthesis accumulation             protein. J. Biol. Chem. 265:11227-11232. <br />     y polypeptides. J. Virol. 37:191-206.                            27. Jofre, J. T., P.  Schaffer, D. S. Parris. 1977. Genetics  9. Crute, J. J.,  R. Lehman. 1989. Herpes simplex-1 DNA              resistance phosphonoacetic acid strain KOS herpes <br />     polymerase. Identification intrinsic 5&apos;-3&apos; exonuclease       simplex virus type 1. J. Virol. 23:833-836. <br />     ribonuclease H activity. J. Biol. Chem. 264:19266-19270.            28. Johnson, P. , M. G. Best, T. Friedmann, D. S. Parris. <br /> 10. Crute, J. J., T. Tsurumi, L. Zhu, S. K. Weller, P. D. Olivo,            1991. Isolation herpes simplex virus type 1 mutant deleted <br />     M. D. Challberg, E. S. Mocarski,  R. Lehman. 1989.                essential UL42 gene characterization null <br />     Herpes simplex virus 1 helicase-primase: complex             phenotype. J. Virol. 65:700-710. <br />     herpes-encoded gene products. Proc. Natl. Acad. Sci. USA            29. Marchetti, M. M., C.  Smith, P.  Schaffer. 1988.     86:2186-2189.                                                           temperature-sensitive mutation herpes simplex virus type 1 <br /> 11. Bruyn Kops, , D. M. Knipe. 1988. Formation DNA              gene required viral DNA synthesis maps coordinates <br />     replication structures herpes virus-infected cells requires        0.609 0.614 UL. J. Virol. 62:715-721. <br />     viral DNA binding protein. Cell 55:857-868.                         30. Marcy,  , P. D. Olivo, M. D. Challberg, D. M. Coen. <br /> 12. Digard, P., C. S. Chow, L. Pirrit, D. M. Coen. 1993.                1990. Enzymatic activities overexpressed herpes simplex <br />     Functional analysis herpes simplex virus UL42 protein. J.        virus DNA polymerase purified recombinant baculovirus- <br />     Virol. 67:1159-1168.                                                    infected insect cells. Nucleic Acids Res. 18:1207-1215. <br /> 13. Digard, P., D. M. Coen. 1990. novel functional domain      31. Marcy,  , D. R. Yager, D. M. Coen. 1990. Isolation     alpha-like DNA polymerase. binding site herpes            characterization herpes simplex virus mutants containing <br />     simplex virus polymerase viral UL42 protein. J. Biol.           engineered mutations DNA polymerase locus. J. Virol. <br />     Chem. 265:17393-17396.                                                  64:2208-2216. <br /> 14. Dodson, M. S.,  R. Lehman. 1991. Association DNA           32. Marians, K. J. 1992. Prokaryotic DNA replication. Annu. Rev. <br />     helicase primase activities subassembly herpes        Biochem. 61:673-719. <br />     simplex virus 1 helicase-primase composed UL5            33. Marsden, H. S., M. E. M. Campbell, L. Haarr, M. C. Frame, <br />     UL52 gene products. Proc. Natl. Acad. Sci. USA 88:1105-1109.            D. S. Parris, M. Murphy, R. G. Hope, M. T. Muller, C. M. <br /> 15. Dorsky, D. , C. S. Crumpacker. 1988. Expression                Preston. 1987. 65,000-Mr DNA-binding virion trans- <br />     herpes simplex virus type 1 DNA polymerase gene vitro             inducing proteins herpes simplex virus type 1. J. Virol. <br />     translation effects gene deletions activity. J. Virol.        61:2428-2437. <br />     62:3224-3232.                                                       34. McGeoch, D. J., M.  Dalrymple,  J. Davison,  Dolan, <br /> 16. Elias, P., M. E. O&apos;Donnell, E. S. Mocarski,  R. Lehman.           M. C. Frame, D. McNab, L. J. Perry, J. E. Scott, P. Taylor. <br />     1986. DNA binding protein specific origin replication       1988. complete DNA sequence long unique region     herpes simplex virus type 1. Proc. Natl. Acad. Sci. USA              genome herpes simplex virus type 1. J. Gen. Virol. <br />     83:6322-6326.                                                           69:1531-1574. <br /> 17. Fierer, D. S., M. D. Challberg. 1992. Purification          35. Murphy, M., P. Schenk, H. M. Lankinen,  M. Cross, P. <br />     characterization UL9, herpes simplex virus type 1 origin-        Taylor,  Owsianka, R. G. Hope, H. Ludwig, H. S. <br />     binding protein. J. Virol. 66:3986-3995.                                Marsden. 1989. Mapping epitopes 65K DNA-binding <br /> 18. Gallo, M. L., D.  Dorsky, C. S. Crumpacker, D. S. Parris.         protein herpes simplex virus type 1. J. Gen. Virol. 70:2357- <br />     1989. essential 65-kilodalton DNA-binding protein            2364. <br />     pes simplex virus stimulates virus-encoded DNA poly-            36. O&apos;Donnell, M. E., P. Elias, B. E. Funnell,  R. Lehman. <br />     merase. J. Virol. 63:5023-5029.                                         1987. Interaction DNA polymerase single- <br /> 19. Gallo, M. L., D. H. Jackwood, M. Murphy, H. S. Marsden,             stranded DNA-binding protein (infected cell protein 8) herpes <br />     D. S. Parris. 1988. Purification herpes simplex virus type       simplex virus 1. J. Biol. Chem. 262:4260-4266. <br />     1 65-kilodalton DNA-binding protein: properties protein      37. Olivo, P. D., N. J. Nelson, M. D. Challberg. 1988. Herpes <br />     evidence association virus-encoded DNA              simplex virus DNA replication: UL9 gene encodes origin <br />     polymerase. J. Virol. 62:2874-2883.                                     binding protein. Proc. Natl. Acad. Sci. USA 85:5414-5418. <br /> 19a.Gallo, M. L., D. S. Parris. Unpublished observations.           38. Owsianka,  M., G. Hart, M. Murphy, J. Gottlieb, R. Boehme, <br /> 20. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis         M. Challberg, H. S. Marsden. 1993. Inhibition herpes <br />     accuracy implications simple methods predicting          simplex virus type 1 DNA polymerase activity peptides     secondary structure globular proteins. J. Mol. Biol.             UL42 accessory protein largely nonspecific. J. Virol. <br />     120:97-120.                                                             67:258-264. <br /> 21. Gibbs, J. S., K. Weisshart, P. Digard,  deBruyn Kops, D. M.       39. Pabo, C. O., R. T. Sauer. 1992. Transcription factors: <br />     Knipe, D. M. Coen. 1991. Polymerization activity              structural families principles DNA recognition. Annu. <br />     alpha-like DNA polymerase requires conserved 3&apos;-5&apos; exonu-             Rev. Biochem. 61:1053-1095. <br />     clease active site. Mol. Cell. Biol. 11:4786-4795.                  40. Pakula,  , R. T. Sauer. 1989. Genetic analysis protein <br /> 22. Goodrich, L. D., F. J. Rixon, D. S. Parris. 1989. Kinetics       stability function. Annu. Rev. Biochem. 23:289-310. <br />     expression gene encoding 65-kilodalton DNA-binding       41. Parris, D. S.,  Cross, L. Haarr,  Orr, M. C. Frame, M. <br />     protein herpes simplex virus type 1. J. Virol. 63:137-147.           Murphy, D. J. McGeoch, H. S. Marsden. 1988. Identifica- <br /> 23. Goodrich, L. D., P.  Schaffer, D.  Dorsky, C. S. Crumpacker,        tion gene encoding 65-kilodalton DNA-binding protein <br />     D. S. Parris. 1990. Localization herpes simplex virus        herpes simplex virus type 1. J. Virol. 62:818-825. <br />     type 1 65-kilodalton DNA-binding protein DNA polymerase         42. Purifoy, D. J. M., R. B. Lewis, K. L. Powell. 1977. <br />     presence absence viral DNA synthesis. J. Virol.           Identification herpes sim <br /> </body></html>