<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             DNA Repair (Amst). 2009 October 2; 8(10): 1207&#226;&#8364;&#8220;1214. doi:10.1016/j.dnarep.2009.06.007. <br />  <br />  <br />  <br />                            Cellular Pathways DNA Repair Damage Tolerance                            <span id='am-21' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>Formaldehyde</span>-Induced DNA-Protein Crosslinks <br />  <br />                            Bendert Graaf, Adam Clore, Amanda K. McCullough* <br />                            Department Molecular Medical Genetics, Center Research Occupational                            Environmental Toxicology, Oregon Health <span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; Science University, 3181 SW Sam Jackson Park Road, <br />                            Portland, 97239; USA <br />  <br />                            Abstract <br />                                 established DNA-protein crosslinks formed consequence cellular <br />                                 exposure agents formaldehyde, transplatin, ionizing ultraviolet radiation,                                 biochemical pathways promote cellular survival repair tolerance lesions poorly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 understood. investigate mechanisms function limit DNA-protein crosslink-induced <br />                                 cytotoxicity, Saccharomyces cerevisiae non-essential gene deletion library screened                                 increased sensitivity formaldehyde exposure. Following low-dose, chronic exposure, strains <br />                                 containing deletions genes mediating homologous recombination showed greatest sensitivity, <br />                                 exposure conditions, deletions genes associated <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> excision <br />                                 repair conferred low moderate sensitivities.  exposure regime changed <br />                                 high-dose acute (short-term) formaldehyde treatment, genes conferred maximal survival <br />                                 switched nucleotide excision repair pathway, little contribution homologous <br />                                 recombination genes. Data presented suggest following acute formaldehyde exposure, <br />                                 repair tolerance DNA-protein crosslinks proceeds formation nucleotide excision <br />                                 repair-dependent single-strand break intermediates detectable accumulation double- <br />                                 strand breaks. data clearly demonstrate differential pathway response chronic versus acute <br />                                 formaldehyde exposures significance implications risk extrapolation human <br />                                 exposure studies. <br />  <br />  <br />                            1. Introduction <br />                                               Despite evidence DNA-protein crosslinks (DPCs) threat genomic integrity, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               biochemical pathways promote cellular survival repair tolerance lesions                                               poorly understood. reports implicate specific pathways                                               process, nucleotide excision repair (NER) [1-3], proteasomal degradation [2,4], <br />                                               Fanconi/BRCA pathway [5], homologous recombination (HR) [1,5].                                                accumulated data given rise clear defined role pathway,                                               instances, findings inconsistent concerning relative contributions process. <br />                                               example, transplatin-induced DPCs persistent NER-deficient <br />                                               xeroderma pigmentosum (XP) group fibroblasts compared normal cells, <br />                                               suggesting critical role NER removal lesions [3].  NER-centric <br />  <br />  <br />                            *Corresponding Author: mcculloa@ohsu.edu, 503-494-9958 (voice), 503-494-6831 (fax). <br />                            Conflict Statement: authors declare conflicts                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Graaf et al.                                                                                               Page 2 <br />  <br />  <br />                                              model does explain data Gantt [6], showed rapidly-proliferating, SV40- <br />                                              transformed XPA cells proficient repair transplatin-induced DPCs,                                              slow-growing, non-transformed parental line deficient repair lesions [6]. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              addition, studies demonstrated sensitivity formaldehyde-induced crosslinks <br />                                              significantly higher XPA XPF cells [4,7]. studies suggested                                              undefined &#226;&#8364;&#339;active repair process&#226;&#8364;? control removal formaldehyde-induced DPCs <br />                                              [4]. complexity DPC repair exemplified studies demonstrating                                              inhibition proteasomal activity mammalian cells resulted decreased repair                                              formaldehyde-induced DPCs. suggested DPC removal dependent proteasomal <br />                                              degradation cross-linked protein [2,4]. <br />  <br />                                              Biochemical studies implicated NER repair DPCs. Reconstituted NER <br />                                              complexes shown initiate repair DPCs DNA-<span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> crosslinks.                                              biochemical study investigate NER pathway remove intact covalently <br />                                              linked proteins, Minko et al created site-specific &#226;&#710;&#188;16 kDa protein adduct DNA linked <br />                                              reduced abasic site [8]. prokaryotic NER complex, UvrABC, able catalyze <br />                                              dual incisions DPC-containing strand, efficiency comparable measured                                              polycyclic aromatic hydrocarbon-modified DNA substrates. Subsequent analyses UvrABC <br />                                              incision DNA-peptide crosslinks revealed increased kinetics incision lesion <br />                                              small oligopeptide [9]. data suggested proteolytic degradation protein <br />                                              portion DPC absolutely essential E. coli NER initiate repair. recently, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Nakano et al examined efficiency E. coli UvrABC incision function                                              oligopeptide-adduct size [1]. investigation demonstrated bacterial NER                                              able repair DPCs containing cross-linked proteins 12-14 kDa size. <br />                                              Interestingly, larger DPCs appeared processed E. coli recBCD-dependent pathway, <br />                                              suggesting formation HR intermediates course repair [1]. contrast,                                              reconstituted mammalian NER incapable excision intact 16 kDa DNA <br />                                              crosslinked protein, efficient removal DNA-peptide substrates observed [10]. <br />  <br />                                              order clearly define cellular pathways proteins involved repair tolerance <br />                                              DPC adducts, genome-wide approach utilized.  results reported                                              screening Saccharomyces cerevisiae non-essential gene deletion library (&#226;&#710;&#188;5000 genes) <br />                                              deletions enhance cytotoxicity following exposure DPC-inducing agent, <br />                                              formaldehyde. described, dramatic differences DNA repair                                              tolerance pathways confer formaldehyde resistance following chronic versus acute <br />                                              exposures. <br />  <br />                               2. Materials methods <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               2.1 Yeast strains chemicals <br />                                              Chemicals purchased Sigma unless noted. MAT (BY4741) S. <br />                                              cerevisiae deletion strain library obtained European S. cerevisiae archives                                              functional analysis (EUROSCARF). Additional pso2&#206;&#8221; S. cerevisiae strains, LBY9 (W303-1A- <br />                                              background) KGY212 (A364A-background) obtained Peter McHugh (U.                                              Oxford) [11] Robb Moses (OHSU) [12], respectively. <br />  <br />                               2.2 Parameters determining chronic acute formaldehyde screening conditions <br />                                              formaldehyde concentration used chronic exposure determined growing <br />                                              wild-type, rad4&#206;&#8221; rad52&#206;&#8221; strains YPD (yeast extract, peptone, dextrose) media <br />                                              containing 0-10 mM formaldehyde 48 hr. Additionally, exponentially growing cultures <br />                                              serially diluted aliquots spotted agar-YPD plates containing formaldehyde <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                              Page 3 <br />  <br />  <br />                                              concentrations ranging 0-10 mM. Colonies grew 48 hr optimal concentrations                                              differential cytotoxicity determined. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              determine conditions acute formaldehyde exposure, yeast strains                                              described harvested exponential growth phase resuspended media <br />                                              containing 20-80 mM formaldehyde 15 min. Cells collected centrifugation, <br />                                              resuspended fresh media aliquots serially diluted cultures spotted agar-YPD plates. <br />                                              Differential survival assessed 48 hr 30&#194;&#176;C. <br />  <br />                               2.3 Formaldehyde sensitivity screen non-essential gene library <br />                                              master MAT (BY4741) S. cerevisiae deletion strain library stored -80&#194;&#176;C YPD <br />                                              + 15% glycerol. Duplicates liquid cultures growing yeast YPD media <br />                                              48 hr 30&#194;&#176;C follows: cells transferred (&#226;&#710;&#188;1&#206;&#188;l) 96 Floating-Pin Replicator <br />                                              rich YPD agar plates containing G418 (200 &#206;&#188;g/ml). liquid cultures transferred <br />                                              using floating-pin replicator triplicate plate (Omnitray, NUNC plates) <br />                                              containing formaldehyde plate containing 1.5 mM formaldehyde. Strains                                              grown 30&#194;&#176;C imaged 24 48 hr plating using AlphaEase FC Imaging  <br />                                              predetermined formaldehyde stable conditions plates                                              used 1 week initial preparation evidenced comparable levels                                              formaldehyde-induced cytotoxicity. Strains classified sensitive limited                                              growth observed replicates 1.5 mM formaldehyde plating. entire <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              library independently screened twice deletion strain assayed triplicate                                              screen. Strains exhibited sensitivity formaldehyde relative wild-type strain                                              individually assayed freezer stocks verify formaldehyde sensitivity. <br />  <br />                               2.4 Cell survival assays <br />                                              rapid semi-quantitative survival assessments, cells cultured YPD overnight                                              diluted 1&#195;&#8212;107 cells/ml. Aliquots (2 &#206;&#188;l) serial 10-fold dilutions cells spotted                                              YPD agar plates containing indicated concentrations formaldehyde ranging 0-2.0 <br />                                              mM. Cells cultured 30&#194;&#176;C 2 days (chronic exposure). acute formaldehyde <br />                                              exposure, cells overnight culture harvested centrifugation, resuspended 1 <br />                                              ml water exposed 60 mM formaldehyde 15 min. exposure, cells                                              collected centrifugation, washed twice water, resuspended YPD concentration <br />                                              1&#195;&#8212;107 cells/ml. strain serially diluted (1:10) YPD plating 2 &#206;&#188;l                                              suspension YPD agar formaldehyde. Cells grown 2 days 30&#194;&#176;C                                              images captured AlphaEase FC imaging  <br />  <br />                                              order assess relative survival, strains representing major DNA repair pathways NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              chosen colony forming assays performed. chronic acute exposure, yeast <br />                                              strains grown overnight 30&#194;&#176;C, vigorous shaking, resulting log-phase culture. <br />                                              Cells diluted following formaldehyde exposure, total number surviving <br />                                              colonies ranged 30-300 cells plate unexposed conditions. chronic <br />                                              exposures, formaldehyde added plates various concentrations (0-2.0 mM),                                              colonies counted 2-3 days growth 30&#194;&#176;C. acute exposures, cells pelleted <br />                                              centrifugation, resuspended, exposed indicated formaldehyde (0-80 mM) 15 <br />                                              min, pelleted 4,000 rpm 2 min washed twice. Cells resuspended media                                              plated YPD agar formaldehyde. experiments repeated                                              independent times. <br />  <br />                               2.5 Pulsed field gel electrophoresis analyses (PFGE) <br />                                              PFGE analyses performed determine strand breaks accumulate following <br />                                              formaldehyde exposure. Assays performed using CHEF genomic DNA plug kit (Bio- <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                              Page 4 <br />  <br />  <br />                                              Rad). Yeast cells grown overnight 30&#194;&#176;C 48 hr presence 1.5 mM <br />                                              formaldehyde. acute exposure, yeast strains incubated 60 mM formaldehyde                                              15 min. exposure, cells washed, pelleted resuspended YPD media. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Additionally, PFGE performed cells arrested G1 using &#206;&#177;-mating factor. arrest <br />                                              (&gt;90%) confirmed visually using light microscope (Nikon, Eclipse E 200, 100&#195;&#8212; <br />                                              magnification). Following arrest, cells exposed 60 mM formaldehyde 15 min, <br />                                              pelleted, washed twice, resuspended YPD containing &#206;&#177;-mating factor (20 &#206;&#188;g/ml) order <br />                                              maintain G1 arrest. inoculated media, 2&#195;&#8212;108 cells used plug. Plugs <br />                                              unexposed cells aliquots harvested 0, 4, 6, 8 hr exposure. <br />                                              cells processed PFGE analyses according manufacturer&apos;s protocol.                                              yeast chromosomes separated 2% pulsed field certified agarose gel 0.5&#195;&#8212; TBE (90 <br />                                              mM Tris base, 89 mM boric acid, 10 mM EDTA pH 8.0), recirculating 14&#194;&#176;C, 20 hr                                              6.5 V/cm 60 120 sec switch time ramp included angle 120&#194;&#176;. Images                                              captured AlphaEase FC imaging  <br />  <br />                               2.6 SDS/KCl precipitation detect DPCs <br />                                              chronic acute treatment, 1&#195;&#8212;107 cells pelleted centrifugation, resuspended                                              washed 1 ml TE (10 mM Tris-HCl, 1 mM EDTA, pH 7.8). Cells resuspended                                              0.25 ml lyticase buffer (BioRad), 8 &#206;&#188;l lyticase (BioRad), 1 &#206;&#188;l yeast protease <br />                                              inhibitor cocktail 30 min 30&#194;&#176;C. confirming spheroplast formation microscopy, <br />                                              0.25 ml 4% SDS added cells frozen thawed 55&#194;&#176;C 5 min. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              suspension, 0.5 ml 200 mM KCl, 20 mM Tris-HCl (pH 7.5) added DNA sheared <br />                                              repeated pipetting. solution cooled ice 5 min. precipitate pelleted <br />                                              3500 &#195;&#8212; g 5 min supernatant saved measure unbound DNA. pellet                                              washed 20 mM Tris-HCl (pH 7.5), 200 mM KCl, incubated 55&#194;&#176;C 5 min, cooled                                              ice 5 min, followed centrifugation 3600 &#195;&#8212; g 4 min. wash procedure                                              repeated 3 times final resuspension 0.5 ml buffer. Proteins digested <br />                                              adding 0.2 mg/ml proteinase K incubating 55&#194;&#176;C 45 min. solution                                              cooled ice 50 &#206;&#188;l 10 mg/ml bovine serum albumin (BSA) (New England BioLabs) <br />                                              added placed ice 5 min. precipitate pelleted 3500 &#195;&#8212; g 5 min. <br />                                              Individually, final supernatant (10 &#206;&#188;l) supernatant wash (5 &#206;&#188;l)                                              separated 0.7% agarose gel stained ethidium bromide (0.001%) 1&#195;&#8212; solution <br />                                              Sybr Gold&#226;&#8222;&#162; (Invitrogen). Images captured AlphaEase FC imaging                                              DNA lane quantified ImageQuant software (GE Healthcare <br />                                              Lifesciences). control potential differential efficiencies extraction DNA                                              formaldehyde-treated cells, cross-linked DNA calculated ratio SDS/ <br />                                              KCl precipitated DNA unbound DNA. limited quantities cross-linked DNA, <br />                                              amounts recoverable unbound DNAs 100-fold excess cross- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              linked DNAs. <br />  <br />                               2.7 Velocity sedimentation ultracentrifugation <br />                                              Overnight cultures yeast cells diluted 1:2 fresh media grown 4 hr. Cells                                              arrested &#206;&#177;-mating factor (20 &#206;&#188;g/ml) 2 hr prior formaldehyde treatment. Cells                                              treated 60 mM formaldehyde 15 min, washed twice resuspended YPD                                              grown additional 4 hr YPD containing &#206;&#177;-mating factor. &#206;&#178;-mercaptoethanol (0.2%) <br />                                              45 mg lyticase added cells incubated 30&#194;&#176;C 45 min. Cells                                              centrifuged 1500 &#195;&#8212; g 10 minutes resuspended 2 ml TE. Sucrose gradients, 15                                              30%, mixing appropriate sucrose buffer consisting 15 <br />                                              mM EDTA, 10 mM Tris-HCl pH 7.4, 0.9 M NaCl, 0.1 N NaOH. Linear gradients                                              poured gradient mixer Sorvall PA ultracentrifuge tubes. <br />  <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                  Page 5 <br />  <br />  <br />                                              Lyticase digested cells poured gradient 50 &#206;&#188;l 10% SDS added. <br />                                              gradients placed 4&#194;&#176;C 10 min lyse cells, centrifuged 4&#194;&#176;C, 11,500 rpm <br />                                              20 hr Sorvall AH627 rotor. centrifugation, fractions (800 &#206;&#188;l) removed NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              tubes automated fractionator 24 &#206;&#188;l 2N HCl added                                              neutralize solution. total 50 &#206;&#188;l fraction added 100 &#206;&#188;l TE containing <br />                                              1.5&#195;&#8212; Sybr Gold&#226;&#8222;&#162; 96 plate. plate scanned Spectramax Gemini XS <br />                                              plate reader using excitation wavelength 490 nm emission wavelength 540 nm. <br />  <br />                               3. Results <br />                               3.1 Formaldehyde sensitivity screen <br />                                              order identify cellular pathways involved enhancing cellular resistance DPC- <br />                                              inducing agents, S. cerevisiae MAT non-essential gene library consisting &#226;&#710;&#188;5000 <br />                                              individual strains [13] screened formaldehyde sensitivity conditions                                              continuous (chronic), low-dose exposure. Prior screening entire library, necessary <br />                                              establish appropriate assay conditions differential cytotoxicity observed <br />                                              wild-type representative strains major DNA repair pathways,                                              NER deficient rad4&#206;&#8221; HR deficient rad52&#206;&#8221;. Aliquots exponentially growing cultures <br />                                              spotted agar plates contained concentrations formaldehyde ranging 0-10 <br />                                              mM. Wild-type cells showed marked decrease survival 2.0 mM concentration,                                              virtually survival observed concentrations &gt; 3 mM (data shown). contrast, repair- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              deficient strains showed greater cytotoxicity compared wild-type strain 1.0-1.5 mM <br />                                              formaldehyde (data shown). <br />  <br />                                              order address formaldehyde stable conditions used, plates <br />                                              containing 1.5 mM formaldehyde prepared stored various times subsequently <br />                                              used screening assays. data revealed highly reproducible results obtained <br />                                              using plates stored 7 days, suggesting effective concentration                                              formaldehyde does significantly change interactions YPD media. ensure <br />                                              doses formaldehyde cytotoxic, agar plugs containing yeast spots lacking <br />                                              colonies removed formaldehyde-containing plates placed liquid YPD                                              allowed grow 48 hr. growth observed (data shown). <br />  <br />                                              Having established assay conditions gave robust signal noise ratio, complete <br />                                              library screened independent times strain assayed triplicate screen. <br />                                              representative set plates screen shown Supplementary Fig. 1. Increased <br />                                              cytotoxicity chronic exposure conditions observed 44 deletion strains,                                              majority genes involved cell cycle DNA repair. Genes involved metabolism, <br />                                              transcription, protein fate cellular transport identified (Table 1). strains listed <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              sensitive Table 1 independently confirmed assaying individual strains using <br />                                              serial dilution spot assays chronic formaldehyde exposure. deletion strains <br />                                              showing significant sensitivity listed Table 1. Interestingly, genome-wide screen <br />                                              suggested strains involved DNA repair tolerance mechanisms, strains deleted <br />                                              HR genes, NER, appeared sensitive formaldehyde. <br />  <br />                               3.2 Homologous recombination protects chronic formaldehyde exposure <br />                                              Following initial genome-wide chronic screen, relative contributions DNA <br />                                              repair tolerance pathway protecting cell formaldehyde-induced toxicity                                              assessed  subset strains previously characterized                                              involved DNA damage repair tolerance mechanisms chosen detailed <br />                                              cytotoxicity analyses based sensitivity formaldehyde. addition strains <br />                                              identified screen, additional strains NER, base excision repair (BER), HR  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                               Page 6 <br />  <br />  <br />                                              non-homologous end-joining (NHEJ) pathways identified sensitive                                              genome-wide screen selected controls (Fig. 1 Supplementary Fig. 2). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cells cultured YPD serial dilutions exponentially growing cells                                              spotted YPD agar plates containing 1.25 mM 1.5 mM formaldehyde grown 30&#194;&#176; <br />                                              C 48 hr. Consistent library screen, highest sensitivities observed                                              strains gene deletions HR pathway, including rad50&#206;&#8221;, rad52&#206;&#8221;, mre11&#206;&#8221;                                              xrs2&#206;&#8221;. strains showed high sensitivities exposed 1.25 mM formaldehyde <br />                                              (Fig. 1A). contrast, strains deleted rad51, rad54 rad55 displayed moderate <br />                                              sensitivity, rdh54&#206;&#8221;, rad59&#206;&#8221; rad57&#206;&#8221; sensitive conditions. <br />                                              Comparatively, strains deleted NER mutants, rad1, rad4, rad14 demonstrated                                              low moderate sensitivities low dose formaldehyde, strains deleted rad2, rad7, <br />                                              rad1, rad17, rad23, rad24 closely comparable wild-type strain (Fig. 1B). <br />                                              Strains deletions genes traditionally defined RAD6 epistasis group (post- <br />                                              replication repair) sensitive formaldehyde conditions,                                              exception moderate high sensitivity rad5 strain (Fig. 1C). confirm                                              data, colony-forming assays carried subset strains (Fig. 1D).                                              anticipated based data Fig. 1A-C, rad18&#206;&#8221; comparable wild-type strain, <br />                                              rad4&#206;&#8221; rad51&#206;&#8221; moderately sensitive mre11&#206;&#8221; rad52&#206;&#8221; showed marked <br />                                              sensitivity. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.3 Nucleotide excision repair protects cells acute formaldehyde exposure <br />                                              determine length time concentration formaldehyde exposure affected                                              pathway used repair DPCs, strains representing major DNA repair damage <br />                                              tolerance pathways assayed sensitivity formaldehyde acute exposure (60 mM, <br />                                              15 min) compared chronic exposure (1.5 mM, 48 hr) (Fig. 2A). noted <br />                                              original screen &#206;&#8221;sgs1 &#206;&#8221;top3 strains highly sensitive chronic <br />                                              formaldehyde exposure. Sgs1/Top3 complex involved maintenance replisome <br />                                              stability Sgs1 RecQ-family helicase implicated double-strand break <br />                                              processing [14,15].  examine relative sensitivity                                              deletion strains following chronic versus acute exposure. <br />  <br />                                              Analyses data revealed dramatic differences cellular pathway responses                                              conditions. shown Fig. 2A, left panel (control) strains grew similar                                              wild-type strain non-exposed conditions, modest slow growth phenotype <br />                                              shown mre11&#206;&#8221; strain. exposed chronic conditions (1.5 mM; middle panel), <br />                                              increased formaldehyde sensitivity observed strains carrying deletions genes                                              HR pathway, including rad52&#206;&#8221;, rad50&#206;&#8221;, mre11&#206;&#8221;, xrs2&#206;&#8221;. addition, sgs1&#206;&#8221;                                              top3&#206;&#8221; showed marked sensitivities, implicating proteins processing DPCs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Consistent data  modest sensitivity seen NER mutants, <br />                                              rad1&#206;&#8221; rad4&#206;&#8221;, sensitivity post-replication repair mutants, rev3&#206;&#8221;, rad6&#206;&#8221;,                                              rad18&#206;&#8221;. <br />  <br />                                              contrast, following acute exposures, NER strains (rad1&#206;&#8221; rad4&#206;&#8221;) demonstrated                                              highest sensitivity, HR deletion strains (mre11&#206;&#8221;, xrs2&#206;&#8221;, rad50&#206;&#8221;, rad52&#206;&#8221;)                                              sgs1&#206;&#8221; top3&#206;&#8221; strains exhibited moderate sensitivity (Fig. 2A; right panel). data <br />                                              suggest relative contribution DNA repair pathways protection                                              formaldehyde-induced DPCs dependent exposure conditions. order confirm                                              findings, colony forming assays carried subset strains acute <br />                                              conditions. agreement data shown Fig. 2A, &#206;&#8221;rad4 showed marked sensitivity, <br />                                              colony forming ability rad52&#206;&#8221;, mre11&#206;&#8221; top3&#206;&#8221; comparable                                              slightly better wild-type strain. <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                 Page 7 <br />  <br />  <br />                                              differences seen response pathways following formaldehyde exposure,                                              directly detect DPCs yeast DNA following chronic acute formaldehyde <br />                                              exposures. assess dose-dependent formation DPCs, SDS/KCl precipitation proteins <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              cell lysates adapted measurement DPC formation previously <br />                                              described [16]. assay relies ability cross-linked DNA precipitate                                              proteins. Following proteinase digestion, extent precipitation DNA measured <br />                                              agarose gel electrophoresis DNA. shown Fig. 3, wild-type cells exposed                                              20-40 mM formaldehyde 15 min showed dose-dependent accumulation DPCs.                                              DPCs detected cells exposed chronic doses (1.5-2 mM) formaldehyde (data                                              shown). Based acute dose experiments, apparent lower limit detection <br />                                              assay 10-20 mM exposures.  DPCs forming chronic doses, <br />                                              detectable method. <br />  <br />                               3.4 role nucleotide excision repair removal formaldehyde-induced DPCs <br />                                              establishing DPCs accumulate dose-dependant manner cells treated                                              formaldehyde, removal DPCs observed 0, 4 6 hr post-exposure wild- <br />                                              type NER-deficient cells. Using SDS/KCl assay described  observed                                              DPCs decrease linear rate cells following acute exposure formaldehyde, reaching                                              level background approximately 6 hr post-exposure. Similar previous results                                              mammalian cells [4], NER-deficient yeast cells similar rate removal DPCs                                              wild-type cells despite large difference sensitivity (Fig. 4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               3.5 Formation single-strand breaks (SSBs) following acute formaldehyde exposure                               NER-dependent <br />                                              NER-deficient cells showed differences rate DPC removal SDS/ <br />                                              KCl assay described  accumulation SSBs acute formaldehyde exposure                                              measured alkaline sucrose gradient sedimentation analyses. DNAs isolated wild-type <br />                                              yeast 4 hr 15 min exposure 60 mM formaldehyde accumulation SSBs                                              evidenced loss high molecular weight DNA lower fractions compared                                              untreated controls (Fig. 5A). NER deficient strains, rad1&#206;&#8221; rad4&#206;&#8221;,                                              accumulation SSBs conditions (Fig. 5B 5C, respectively). <br />  <br />                               3.6 DPCs repaired different pathway DNA-DNA interstrand crosslinks <br />                                              findings HR NER pathways, components pathways, involved                                              repair tolerance DPCs reminiscent potential interplay                                              pathways repair DNA-DNA interstrand crosslinks (ICL). fact, significant <br />                                              overlap strains implicated repair ICLs identified                                              formaldehyde screen. strains sensitive mitomycin C-induced ICLs NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              sensitive formaldehyde (rad1&#206;&#8221;, rad4&#206;&#8221;, rad51&#206;&#8221;, rad52&#206;&#8221;) [11,12];                                               mitomycin C-sensitive deletion strains rev3&#206;&#8221; exo1&#206;&#8221; sensitive formaldehyde <br />                                              (Fig. 1,Fig. 2 Supplementary Fig. 2, respectively). Processing ICLs eukaryotic <br />                                              systems associated formation double-strand breaks (DSBs) hypothesized <br />                                              formed sites stalled replication forks [17]. PSO2, 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; exonuclease, significantly <br />                                              contributes cellular resistance ICL damage. Interestingly, pso2&#206;&#8221; mutants appear                                              proficient initial incision events ICL repair, fail repair ICL-induced DSBs <br />                                              [11]. Additionally, mutants decreased rates homologous recombination, <br />                                              suggesting PSO2 nuclease acts downstream NER processing ICLs                                              provide suitable substrate HR [11,12]. current models DPC repair predict <br />                                              DSB intermediates formed replication fork collapse strains <br />                                              deleted genes NER HR pathways sensitive formaldehyde exposures,                                              determine PSO2 involved DPC repair. pso2&#206;&#8221; strain  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                               Page 8 <br />  <br />  <br />                                              deletion library sensitive formaldehyde, additional pso2&#206;&#8221; strains different <br />                                              backgrounds, previously shown sensitive ICL-inducing agents, obtained                                              Dr. Peter McHugh (University Oxford) Dr. Robb Moses (Oregon Health &amp; Science <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              University) tested identical conditions described  pso2&#206;&#8221; strains <br />                                              did sensitivity formaldehyde conditions (data shown). data <br />                                              suggest ICL DPC-inducing agents lead stalled replication forks, <br />                                              possibly different intermediate structures result fundamentally different <br />                                              mechanisms limit cytotoxicity response agents. <br />  <br />                               3.7 DSBs accumulate following formaldehyde exposure <br />                                              Based sensitivity HR deletion strains, plausible model invokes DNA break <br />                                              intermediate order perform single-strand invasion process DPCs [18]. test                                              model, pulsed field gel electrophoresis (PFGE) analyses performed determine SSBs, <br />                                              DSBs, formed processing formaldehyde-induced DPCs. PFGE <br />                                              analyses performed addition S1-nuclease, added                                              convert SSBs DSBs subsequently detected PFGE. Analyses performed <br />                                              cells exposed chronic acute doses formaldehyde. <br />  <br />                                              serve controls induction DSBs SSBs, wild-type cells treated                                              bleomycin hydrogen peroxide, respectively. Following control experimental exposures, <br />                                              cells digested lyticase, lysates subsequently incubated proteinase <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              K, DNAs subjected PFGE. bleomycin treatment resulted DSB formation (Fig. <br />                                              6A, left panel), hydrogen peroxide treatment produced DSBs SSBs (Fig. 6A, <br />                                              middle panel), results consistent previously published data [19].  cells <br />                                              exposed chronic formaldehyde showed evidence SSB DSB accumulation <br />                                              (Fig. 6A, right panel). <br />  <br />                                              possible interpretation formaldehyde exposure data SSBs DSBs occur <br />                                              transiently, intermediates repair process, breaks detectable <br />                                              previous conditions. increase number breaks occurring time, <br />                                              cells arrested G1 phase &#206;&#177;-mating factor prior formaldehyde treatment.                                              acute exposures tested arrested cells duration chronic exposure. <br />                                              Following treatment, cells assayed formation SSBs DSBs 24 hr <br />                                              period. shown Fig. 6B, DSBs observed background levels PFGE                                              exposure given time point. Likewise, DSB intermediates observed accumulating <br />                                              background levels rad52 mre11 deletion strains (Supplementary Fig. 3). <br />  <br />                                              contrast, SSBs observed immediately following exposure formaldehyde                                              observed greater numbers 4 hr exposure.  8 hr, SSBs NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              completely repaired (Fig. 6B). cells counted post-exposure time point <br />                                              significant increase cell number occurred, conclude dilution <br />                                              damage cell growth. accumulation SSBs following formaldehyde exposure <br />                                              evident rad4 deletion strain (Fig. 5 Fig. 6), suggesting process                                              dependent NER. <br />  <br />                               4. Discussion <br />                                              order identify characterize biochemical mechanisms repair formaldehyde- <br />                                              induced DPCs, study utilized S. cerevisiae haploid non-essential gene deletion library <br />                                              screen genes deleted, confer formaldehyde sensitivity. Specific biochemical <br />                                              pathways identified protect cells cytotoxic effects acute chronic <br />                                              formaldehyde exposures. investigations shown large portion                                              formaldehyde-sensitive genotypes pathways DNA repair, DNA damage tolerance, <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                Page 9 <br />  <br />  <br />                                              chromatin remodeling (Table 1). Interestingly, S. cerevisiae cells respond fundamentally <br />                                              different ways induction DPCs dependent exposure                                              formaldehyde given chronic, low dose acute high dose. Following chronic exposure, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              cell survival conferred proteins HR pathway including, Rad50, Rad51, Rad52, <br />                                              Rad54, Rad55, Mre11 Xrs2, proteins associated resolution stalled <br />                                              replication fork structures, Sgs1, Top3 Rad5. data suggest successful <br />                                              tolerance repair DPCs proceed intermediates substrates HR.                                              comparison, moderate sensitivity observed NER deletion strains, suggesting                                              critical contribution NER survival following chronic exposure. possible                                              formaldehyde exposures, DPCs encountered leading lagging strands                                              DNA synthesis dealt different manners. DPCs leading strand block <br />                                              progression helicase complex precedes replisome  replication <br />                                              complexes stall advance polymerase reaching site DNA adduct. <br />                                              allow NER repair DPC possibly explain moderate sensitivity                                              NER deletion strains. lagging strand, stalled replication complexes temporarily <br />                                              uncouple synthesis strands lagging strand synthesis occur.                                              point, fork regression occur, leading recombinogenic structure,                                              processed effectively allow damage tolerance replication restart. time, <br />                                              exact nature intermediates remains undetermined;  DSBs SSBs                                              detected following chronic exposures. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              case acute short-term exposures, cellular survival strategies appear switch                                              NER-dependent response, Rad1, Rad4, Rad14 having essential roles conferring <br />                                              enhanced survival. data suggest signal transition HR-dependent <br />                                              pathway NER-dependent pathway related rapid induction large number                                              DPCs. data reveal SSBs readily detected conditions accumulate <br />                                              post-exposure, indicating breaks intermediate processing DPCs.                                              formaldehyde-induced SSBs appear NER-dependent breaks accumulate <br />                                              post-formaldehyde exposure rad4&#206;&#8221; rad1&#206;&#8221; strains, suggesting SSBs                                              intermediates generated repair process. results consistent previous <br />                                              report showing Rad3-dependent accumulation SSBs following formaldehyde exposure <br />                                              [20]. PFGE analyses suggest SSBs dependent presence Mre11 <br />                                              (Supplementary Fig. 3). <br />  <br />                                              Based data showing similar rates DPC removal wild type NER-deficient arrested <br />                                              cells, suggest acute high-dose exposure, proteolytic degradation precedes NER <br />                                              recognition degradation NER-independent active repair process, initiated <br />                                              stalled replication forks (replication-independent). replication-independent, NER- <br />                                              dependent model DPC repair consistent data reported E. coli [1]. addition, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              similar rates removal DPCs measured Fig. 4 WT NER-deficient strains <br />                                              indicative pre-NER processing event degrades protein crosslink peptide <br />                                              detectable assay. <br />  <br />                                              shown Fig. 2, strains deficient mre11 rad52 sensitivity formaldehyde <br />                                              wild-type strain. enhanced resistance formaldehyde seen consistently                                              mre11 rad52 deletion strains, suggesting acute conditions, HR                                              advantageous fact, attempted processing formaldehyde-induced DNA adducts HR <br />                                              produce cytotoxic intermediate interfere NER processing DPCs. <br />  <br />                                              course study, reports demonstrated possible role HR pathway <br />                                              tolerance DPCs [1,5]. E. coli study study using isogenic DT40 <br />                                              chicken cell lines, authors proposed models DPC processing proceeding DSB <br />                                              intermediates. data demonstrating prominent role HR led speculate  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                 Page 10 <br />  <br />  <br />                                              DSBs likely intermediate DPC processing pathway.  PFGE analyses                                              neutral centrifugation analyses showed detectable DSBs following exposure                                              formaldehyde, HR-deficient background (Fig. 5 Supplementary Fig. 3). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              data consistent PFGE analyses E. coli comet assays performed V79 Hamster <br />                                              cells following formaldehyde exposure [1,7]. <br />  <br />                                              Interestingly, DSBs detected cells treated <span id='am-26' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span>thymidine</span>, known block <br />                                              DNA replication. Previously, shown mammalian cells respond hydroxyurea <br />                                              replication arrest differently thymidine block [21], hydroxyurea induced DSBs <br />                                              repair strand breaks proceeded NHEJ HR pathways.                                              contrast, thymidine arrest involved HR detectable DSBs. authors <br />                                              concluded cytotoxic lesion following thymidine treatment persistent                                              transient DSBs, different, undetermined substrate HR [21]. Additionally,                                              alternative non-DSB dependent model HR initiated DNA nick single- <br />                                              strand gap generated replication past polymerase stalling DPC lesion.                                              plausibility model supported observation mitotic HR S. <br />                                              cerevisiae, initiating lesion DSB potentially SSB single-strand gap [22]. <br />                                              consistent data demonstrating lack DSBs, accumulation SSBs <br />                                              following formaldehyde exposure. noted observations rule                                              possibility small transient DSBs occur detectable                                              methods. possible single-strand break repair pathway role NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              preventing cytotoxicity DPC processing intermediates.  rad27 (Fen1) <br />                                              deletion strain sensitive formaldehyde chronic acute conditions <br />                                              (Supplementary Fig 3). definitely rule role pathway genes <br />                                              major ligase (cdc9) polymerase (pol&#206;&#180;) pathway essential                                              included screen. <br />  <br />                                              addition genome-wide screen examining formaldehyde cytotoxicity,                                              study highlights exposure conditions (duration concentration) significantly <br />                                              affect spectra gene deletion strains identified sensitive resistant. data <br />                                              clearly demonstrate differential pathway response chronic versus acute formaldehyde <br />                                              exposures significance risk extrapolation human exposure studies.                                              recognized validity extrapolations challenged capacity                                              cells activate detoxify chemicals, steady-state DNA repair capacity                                              effectively remove damages occur. Consistent dose-dependent pathway responses, <br />                                              rat formaldehyde inhalation studies shown steep dose-dependent transitions cell <br />                                              proliferation assays tumorigenesis [23]. increased understanding molecular basis <br />                                              transitions cellular response pathways elucidate details models dose- <br />                                              dependent transitions mechanisms formaldehyde toxicity. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              summary, shown NER HR critical survival following <br />                                              formaldehyde exposure.  relative contribution pathway differs depending <br />                                              dose duration exposure.  genome-wide screen identified                                              genes contribute significantly protection genotoxic effects formaldehyde. <br />                                              relationship genes/pathways NER HR investigated                                              context newly defined DPC-specific repair/tolerance pathway. <br />  <br />                               Supplementary Material <br />                                              Refer Web version PubMed Central supplementary material. <br />  <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                                  Page 11 <br />  <br />  <br />                               Acknowledgments <br />                                              thank Robb Moses (OHSU), Peter McHugh (University Oxford) Leona Samson (MIT) generous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              gifts yeast strains. like thank Christian Dan Erik Bom technical assistance. grateful <br />                                              R. Stephen Lloyd, Irina Minko, Anuradha Kumari valuable comments critically reading manuscript. <br />                                              work supported NIH RO1 CA106858 NIH PO1 ES05355. <br />  <br />  <br />                               References <br />                                              1. Nakano T, et al. Nucleotide excision repair homologous recombination systems commit <br />                                                 differentially repair DNA-protein crosslinks. Mol Cell 2007;28(1):147&#226;&#8364;&#8220;58. [PubMed: <br />                                                 17936711] <br />                                              2. Baker DJ, et al. Nucleotide excision repair eliminates unique DNA-protein cross-links                                                 mammalian cells. J Biol Chem 2007;282(31):22592&#226;&#8364;&#8220;604. [PubMed: 17507378] <br />                                              3. Fornace AJ Jr, Seres DS. Repair trans-Pt(II) diamminedichloride DNA-protein crosslinks normal <br />                                                 excision-deficient human cells. Mutat Res 1982;94(2):277&#226;&#8364;&#8220;84. [PubMed: 7202118] <br />                                              4. Quievryn G, Zhitkovich  Loss DNA-protein crosslinks formaldehyde-exposed cells occurs <br />                                                 spontaneous hydrolysis active repair process linked proteosome function. <br />                                                 Carcinogenesis 2000;21(8):1573&#226;&#8364;&#8220;80. [PubMed: 10910961] <br />                                              5. Ridpath JR, et al. Cells deficient FANC/BRCA pathway hypersensitive plasma levels                                                 formaldehyde. Cancer Res 2007;67(23):11117&#226;&#8364;&#8220;22. [PubMed: 18056434] <br />                                              6. Gantt R. cell cycle-associated pathway repair DNA-protein crosslinks mammalian cells. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mutat Res 1987;183(1):75&#226;&#8364;&#8220;87. [PubMed: 3025724] <br />                                              7. Speit G, Schutz P, Merk O. Induction repair formaldehyde-induced DNA-protein crosslinks                                                 repair-deficient human cell lines. Mutagenesis 2000;15(1):85&#226;&#8364;&#8220;90. [PubMed: 10640535] <br />                                              8. Minko IG, Zou Y, Lloyd RS. Incision DNA-protein crosslinks UvrABC nuclease suggests                                                 potential repair pathway involving nucleotide excision repair. Proc Natl Acad Sci U S 2002;99(4): <br />                                                 1905&#226;&#8364;&#8220;9. [PubMed: 11842222] <br />                                              9. Minko IG, et al. Initiation repair DNA-polypeptide cross-links UvrABC nuclease. <br />                                                 Biochemistry 2005;44(8):3000&#226;&#8364;&#8220;9. [PubMed: 15723543] <br />                                              10. Reardon JT, Sancar  Repair DNA-polypeptide crosslinks human excision nuclease. Proc Natl <br />                                                   Acad Sci U S 2006;103(11):4056&#226;&#8364;&#8220;61. [PubMed: 16537484] <br />                                              11. Barber LJ, et al. DNA interstrand cross-link repair Saccharomyces cerevisiae cell cycle: <br />                                                   overlapping roles PSO2 (SNM1) MutS factors EXO1 S phase. Mol Cell Biol <br />                                                   2005;25(6):2297&#226;&#8364;&#8220;309. [PubMed: 15743825] <br />                                              12. Grossmann KF, et al. S. cerevisiae pathways DNA interstrand crosslink repair. Mutat <br />                                                   Res 2001;487(34):73&#226;&#8364;&#8220;83. [PubMed: 11738934] <br />                                              13. Kelly  Lamb DC, Kelly SL. Genome-wide generation yeast gene deletion strains. Comp Funct <br />                                                   Genomics 2001;2(4):236&#226;&#8364;&#8220;42. [PubMed: 18628917] <br />                                              14. Mimitou EP, Symington LS. Sae2, Exo1 Sgs1 collaborate DNA double-strand break <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   processing. Nature 2008;455(7214):770&#226;&#8364;&#8220;4. [PubMed: 18806779] <br />                                              15. Bjergbaek L, et al. Mechanistically distinct roles Sgs1p checkpoint activation replication <br />                                                   fork maintenance. EMBO J 2005;24(2):405&#226;&#8364;&#8220;17. [PubMed: 15616582] <br />                                              16. Trask DK, DiDonato JA, Muller MT. Rapid detection isolation covalent DNA/protein <br />                                                   complexes: application topoisomerase II. EMBO J 1984;3(3):671&#226;&#8364;&#8220;6. [PubMed: 6325181] <br />                                              17. Bergstralh DT, Sekelsky J. Interstrand crosslink repair: XPF-ERCC1 let hook? Trends <br />                                                   Genet 2008;24(2):70&#226;&#8364;&#8220;6. [PubMed: 18192062] <br />                                              18. Baldwin EL, et al. Mms22p protects Saccharomyces cerevisiae DNA damage induced                                                   topoisomerase II. Nucleic Acids Res 2005;33(3):1021&#226;&#8364;&#8220;30. [PubMed: 15718301] <br />                                              19. Ribeiro GF, Corte-Real M, Johansson B. Characterization DNA damage yeast apoptosis induced <br />                                                   hydrogen peroxide, <span id='am-31' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span>acetic acid</span>, hyperosmotic shock. Mol Biol Cell 2006;17(10):4584&#226;&#8364;&#8220;91. <br />                                                   [PubMed: 16899507] <br />  <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                                 Page 12 <br />  <br />  <br />                                              20. Magana-Schwencke N, Ekert B, Moustacchi E. Biochemical analysis damage induced yeast                                                  formaldehyde.  Induction single-strand breaks DNA repair. Mutat Res 1978;50(2): <br />                                                  181&#226;&#8364;&#8220;93. [PubMed: 349372] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              21. Lundin C, et al. Different roles nonhomologous end joining homologous recombination <br />                                                  following replication arrest mammalian cells. Mol Cell Biol 2002;22(16):5869&#226;&#8364;&#8220;78. [PubMed: <br />                                                  12138197] <br />                                              22. Lettier G, et al. role DNA double-strand breaks spontaneous homologous recombination                                                  S. cerevisiae. PLoS Genet 2006;2(11):e194. [PubMed: 17096599] <br />                                              23. Monticello TM, et al. Correlation regional nonlinear formaldehyde-induced nasal cancer                                                  proliferating populations cells. Cancer Res 1996;56(5):1012&#226;&#8364;&#8220;22. [PubMed: 8640755] <br />  <br />  <br />                               Abbreviations <br />                                              BER <br />                                                             base excision repair <br />  <br />                                              DPC <br />                                                             DNA-protein crosslink <br />  <br />                                              DSB <br />                                                             double strand break <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              NER <br />                                                             nucleotide excision repair <br />  <br />                                              HR <br />                                                             homologous recombination <br />  <br />                                              PFGE <br />                                                             pulsed field gel electrophoresis <br />  <br />                                              SSB <br />                                                             single-strand break <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                DNA Repair (Amst). Author manuscript; available PMC 2010 October 2. <br />                             Graaf et al.                                                                                           Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Fig. 1. Survival Analyses Yeast Deletion Strains Exposed Chronic Low Dose Formaldehyde <br />                                                HR deletion strains, (B) NER cell cycle-associated deletion strains, (C) Translesion <br />                                              synthesis (post replication repair) strains differential growth following formaldehyde <br />                                              exposure. strains shown grown exponentially YPD media 2 &#206;&#188;l aliquots NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              serially diluted cells (10- <br /> </body></html>