<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Escherichia coli-Salmonella typhimurium Hybrid nusA Genes: <br />              Identification Short Motif Required Action                  N Transcription Antitermination Protein <br />  <br />  <br />              Escherichia coli nusA gene, nusA,E, encodes essential protein influences transcription elongation. <br />           Derivatives E. coli SalmoneUla typhimurium nusA gene, nusAs, replaced nusA$4 viable. <br />            NusAst substitute NusAEz supporting essential bacterial activities.  hybrid E. coli <br />           strains nus4st substitution effectively support transcription antitermination mediated N <br />           gene product phage X. report DNA sequence nusAst, showing derived <span id='am-16' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> sequence <br />           95% identical derived amino acid sequence nus4,c. alignment amino acid sequences <br />           reveals scattered single amino acid differences region significant heterogeneity. region, called <br />           449, NusAF, amino acids NusAst amino acids. Functional studies hybrid ns<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>;4 genes, <br />           constructed nusAEc nus4st, 449 region NusAEc protein important X <br />           N-mediated transcription antitermination. hybrid substitution E. coli codons           S. typhimurium codons, nusAs, supports action N antitermination protein.           449 region  presumably, adjacent sequences appear compose functional domain NusAE important <br />           action N transcription antitermination protein phage X. <br />  <br />    nusA gene Escherichia coli encodes essential                      terminators proceeds Q gene. Q gene <br /> 54,400-Mr protein functions transcription elongation                   product, regulates late gene expression, (reviewed references 20, 29, 83) termination (16, 37,               antitermination factor requires NusA (67). <br /> 49, 75, 81). Consistent roles, NusA shown                     Isolation nonsense (80) thermo- cold- influence pausing RNA polymerase (16, 17, 47, 50, 75, 76,                sensitive (63, 73) mutations nusA demonstrates NusA <br /> 85). demonstration association NusA                      provides essential bacterial function. essential <br /> RNA polymerase (31, 35, 43, 46, 74) suggests NusA                     role(s) NusA cellular growth identified, <br /> influence transcription direct interaction polymerase.               mutations nusA reduce effectiveness transcrip- <br />    NusA identified role regulated expres-               tion terminators (62, 81). <br /> sion phage X genes (reviewed references 13, 20, 23,                     addition RNA polymerase N, NusA 67). Early transcription initiating promoters PL PR                   shown bind Rho termination protein (74) RNA <br /> partially terminates terminators tLl (15, 70) tRl (10),                 (79). facts binds number factors respectively. escaping transcripts case termi-               relatively large protein (36, 44, 68) suggest possibility nate completely collection terminators nin region               NusA multiple functional domains. func- <br /> (8, 9, 11, 48, 52) (Fig. 1). N gene product, encoded                tionally important regions NusA located 5&apos; thirds <br /> PL operon, acts host Nus proteins modify RNA                           protein, plasmid-based nusA gene polymerase termination-resistant form nut sites (1, 14,              3&apos;-terminal deleted (nusAA324) complements nusA <br /> 32, 53-55, 64, 69). Mutations affecting N action identified               mutations fail support X N action bacterial E. coli nus genes (reviewed references 20, 22, 26,                  growth low temperatures (73).  plasmid-based <br /> 67). example, nusAl mutation (19), missense muta-                    nusAA324 gene fails complement nusAII(Ts) mutation. <br /> tion resulting single base change (68), causes failure                product nusAA&amp;324 gene unable support support N-mediated antitermination (14, 24, 42, 82). N                  bacterial growth high temperatures (61). <br /> protein binds NusA (34), vitro transcription studies                    Experiments hybrid bacterium, genome confirmed Nus proteins required N action <br /> (14, 42, 54, 82). properly modified N Nus                          derived primarily E. coli, provide additional evidence <br /> proteins, RNA polymerase resistant termination                      supporting role NusA X N action. hybrid (2), signals proceeds downstream terminators.                   small exchange DNA resulted replacement case PR, N-modified transcription passes nin                nusAE,, nusA gene E. coli, nusAst, nusA gene <br />                                                                                closely related enteric bacterium Salmonella typhi- <br />                                                                                murium (66, 72). addition nusA gene, additional <br />   * <br />     Corresponding author. Phone: (313) 763-3142. Fax: (313) 764- <br />                                                                                uncharacterized genetic information 5&apos; nusA 3562. Electronic mail address: David.Friedman@um.cc.umich.edu.                 replaced. viability hybrid bacterium, priori, <br />   t Present address: Roche Institute Molecular Biology, Nutley, NJ          demonstrates NusAst substitute NusAEc sup- <br /> 07110-1199.                                                                    porting cellular processes E. coli genetic background. <br />   t Present address: Department Microbiology, University                  NusAst fails support X N action (2, 21, 73). <br /> Texas, Austin, TX 78712-1095.                                                     failure nusAst support X N action, condi- <br />                                                                         1394 <br />  VOL. 176, 1994                                                                                                          HYBRID nusA GENES                  1395 <br />  <br />                                                                                                                                nin5 <br />                     att int                        dll        N                d          cro                   cllOP                  0   Q <br />                                                                                                          0   xs                     *          <br />                                                                                                                                                   4 <br />  <br />  <br />          ti                                          tLl nutL                                nutR         tRI                42 43 %4 qut tRI <br />                                                                     PL <br />                                                                          -N, nut                                                                                                                                                                     - <br />                                                                           Nus <br />          ~                                                           +N, nut <br />                                                                          Nus <br />   FIG. 1. Map early region genome.  line) Shown locations relevant genes positions aut site nin region, shown relevant transcription signals, including promoters (p), terminators (t), antitermination signals (nut qut). (Middle line) Patterns transcription PL PR absence N.  line) Patterns transcription PR PL <br /> following N modification RNA polymerase. <br />  <br />  <br /> tions nusAEc does, suggested structural                            EOP. Methods determining efficiency plating (EOP) <br /> differences proteins reflected                            qualitative assessment phage growth differences primary amino acid sequences.                                 described previously (3, 59). <br /> determined DNA sequence nusA, compare                                        Genetic techniques.   Transfer nusAhy3O2 hybrid <br /> derived amino acid sequence derived DNA                               gene pNAX302 E. coli chromosome homologous <br /> sequence nusA,E (12, 45). comparison, revealed                             recombination. rec+ nusAlO(Cs) strain, K1456, containing small differences proteins, provided                            pNAX302 passed rounds growth LB <br /> information enabling construct series hybrid nusA                             broth 40&#194;&#176;C absence ampicillin. Colonies genes various amounts genetic information                              chromosomal nusAhy302 gene selected growth 30&#194;&#176;C <br /> E. coli S. typhimurium. basis studies                           LB plates presence XcI - Xc -h80; colony <br /> hybrid genes, identified region nusA gene                          formed recombinant expected cold required X N action.                                                              resistant unable support growth X. Plasmid-free <br />                                                                                          bacteria identified screening ampicillin sensitivity <br />                           MATERIALS METHODS                                          Amp Screen (Bethesda Research Laboratories). P1 trans- <br />                                                                                          duction (77) used transfer nusAhy3O2 allele    Strains. Bacteria phages listed Tables 1 2,                            colony appropriate phenotype K37, yielding strain <br /> respectively.                                                                            K4092. <br />    Media. media described previously (56, 57).                                (ii) Creation chromosomal nusAhy3O2-449 allele. <br /> Bacterial strains plasmids cultivated presence                          Plasmid pSES28, derivative pUC18 840-bp frag- appropriate antibiotic, ampicillin 30 ,ug/ml                         ment composed internal portion nusAs, tetracycline 15 ,ug/ml.                                                               nucleotides encoding amino acids nus4EC 449 <br />    Plasmids. Plasmid manipulations essentially                                sequence replaced nucleotides encoding scribed Sambrook et al. (71). Table 3 lists plasmids used                         amino acids nusAst 449 sequence, transformed study sources.                                                  strain K6600. K6600 nusAhy3O2 gene defective <br />  <br />                                                                           TABLE 1. Bacterial strains <br />  Straina                                                 Relevant genotype                                Parent                        Source reference <br /> K37                                    Wild type                                                                                    laboratory (19) <br /> K95                                    nusAI                                                              K37                       laboratory (19) <br /> K1102                                  nusAhyllO2                                                                                   L. S. Baron (21) <br /> K1456                                  argG::Tn5                                                          K37                       laboratory (19, 73) <br /> K1914                                  argG::Tn5 nusAlO(Cs)                                               K1456                     laboratory <br /> K2415                                  recA nusAlO(Cs)                                                    K1914                     L. Baron <br /> OR1150                                 XcI857pO-IS2-gal                                                                             S. Adhya (65) <br /> K4012                                  nusAll4(Ts)                                                        K37                       68 <br /> K4092                                  argG::Tn5 nusAhy3O2                                                K37                       paper <br /> K6600                                  argG::Tn5 nusAhy3O2-449 XcI857pR-IS2-gal                                                     paper <br /> K6906                                  argG::TnS nus4hy3O2-449                                            K37                       paper <br /> K7346                                  nusA134  recA::TnlO                                                                       Y. Nakamura (80) <br /> LT2                                    Wild type                                                                                    G. Jones <br />     strains     E. coli LT2, S. typhimurium. <br />  1396        CRAVEN ET AL.                                                                                                        J. BACT1E1RIOL. <br />  <br />                               TABLE 2. Phages                                   glucose plates. 2 days, large haloed plaques                                Relevant genotype                 Source                                                                                 observed thinly growing bacterial lawn. bacteria <br />        Phage                                                                    grow infecting phage carried expressed <br />                                phenotype                  reference <br />                                                                                 argG gene S. typhimurium. centers plaques <br /> AcI857                   clts                                 NIHa collection   picked, phage purified bacterial lawns <br /> XcI60                    cI-                                  NIH collection    formed conditions original isolation. Phage <br /> AcI857bio256sex          clts, altered N                      NIH collection    obtained plaques identified second isolation XcI -h80                 cI-, adsorption specificity          NIH collection <br />                             48O                                              used make lysates. resulting lysates screened XninS                    N independent                        NIH collection    nusAst testing complementation argG::TnS <br /> M13mp8                   NAb                                  84                nusAlO(Cs) mutations. nusAst gene X1059 <br /> M13mp9                   NA                                   84                derivative isolated partial Sau3AI digestion Pi                       NA                                   NIH collection    phage DNA cloned BamHI site   NIH, National Institutes Health.                                         pACYC184. plasmid complemented nusAJO(Cs) <br />   b NA, applicable.                                                         mutation isolated; plasmid, pNAS1000, contains                                                                                 2.9-kb insert includes nusA5s, <br />                                                                                    DNA sequencing. dideoxy chain termination method <br />                                                                                 described Biggin et al. (4) employed. Primers XcI857 prophage fused bacterial gal operon                   sequencing obtained University Michigan <br /> gal operon expressed &apos;APR promoter. presence                    Oligonucleotide Synthesis Facility Bethesda Research intervening termination signals PR gal                   Laboratories. expression X repressor cI gene cI857ts                       nusA,st gene sequenced using collection allele results expression gal operon dependent                  overlapping subclones pNAS1000 constructed ligating functional phage N antitermination read                   sonication-generated DNA fragments M13mp8. terminators growth high temperature                   regions overlapping subclones obtained, remove repression. nusAhy302 allele fails support                  sequence determined directly pNAS1000 using <br /> N-mediated transcription antitermination, K6600 Gal -                     nusA-specific primers sequences derived high temperatures. Gal&apos; segregants isolated, plating                   M13mp8 clones. appropriate X derivatives identified                        PCR. Reactions carried thermocycler 30 <br /> distinctive phenotype nusAhy449 gene plasmid.                       cycles. Cycle durations 1 min 94&#194;&#176;C, 1 min 44&#194;&#176;C, derivative chosen study. putative                       1 min 72&#194;&#176;C. reaction sample consisted 0.4 ,ug nusAhy3O2-449 allele transduced K37, creating strain                     DNA template (pNAS1000) 80 pmol primers. <br /> K6906, DNA sequencing PCR-amplified fragment                              Oligonucleotides used priming PCRs prepared confirmed structure.                                                        University Michigan Oligonucleotide Synthesis Facility. <br />    (iii) Isolation nusAst gene. X1059-Salmonella clone                 sequences primers nusAs, nucleotides bank, constructed R. Mauer kindly provided P.                         bind follows: SaI128, 5&apos; TCGATGTTCGT <br /> Youderian, used isolate nusAs, gene according                   GTAGAA 3&apos; (binds nucleotides 128 144 method suggested P. Youderian. underlying assump-                        strand); Las971, 5&apos; ACGTTCTGACCATTACG 3&apos; (binds tion arrangement S. typhimurium genome                   nucleotides 971 955 strand); SaI483, 5&apos; GGATQT nusA region E. coli                  GGGCAACAACGCTGAAGCGGTGATTC[G 3&apos; (binds genome; e., nusA adjacent argG. culture E.                   nucleotides 504 484 strand); Las457, 5&apos; <br /> coli K-12 strain K1456 (K37 argG::TnS) mixed                           GITGCCCAGATCCAGAGAGATA1TGTCGCGGTTC 3&apos; <br /> different dilutions phage bank, infected bacteria                (binds nucleotides 457 435 strand). Primers plated 2.5 ml agar M9 minimal glucose                       Sa1128 Las971 perfectly match nusAs, sequences. <br /> medium supplemented 0.1 ml LB broth M9 minimal                       Primers Sal483 Las457 designed 13 bases                                                                                 5&apos; ends complementary                                                                                 hybridize. (shown underlined) 13 bases corre- <br />                            TABLE 3. Plasmids                                    spond nusA,E nucleotides replace nucleotides 458 <br />                                                                                 483 nusAst DNA sequence. <br />   Plasmid            Relevant marker(s)              Parent         Source                                                                     reference <br />                                                                                    Construction hybrid nusA genes. nusAhy449 gene                                                                                 created PCR &quot;gene splicing overlap extension&quot; <br /> pBR322   NA&quot;                                       NA              5            technique (41). rounds PCR used, separate <br /> pACYC184 NA                                        NA              7            reactions, primers Sal128 Las457 primers Sal483 pUC19    NA                                        NA              84           Las971 amplify portion nusA st DNA, pNAS1000 pWR324   nusAA324                                  pBR322          73           template. fragments gel purified com- <br /> pNAG15   nusAEc                                    pBR322          work    bined primers Sal128 Las971, primers pNAG2010       nusAEc                              pBR322          work    round PCR. fragments overlap 13 nucle- <br /> pNAS1000       nusAs,                              pACYC184        work <br /> pNAS2001       nusA s                              pBR322          work    otides,  frequency, anneal  <br /> pN2/EuS        nusAA50L-nusA,s                     pBR322          work    annealed strands acted templates polymerase pNAX2          nusAhy2                             pBR322          work    generated spliced recombinant DNA. presence pNAX3          nusAhy3                             pBR322          work    Sal128 Las971 allowed amplification recombi- <br /> pNAX302        nusAhy3O2                           pBR322          work    nant subsequent cycles. 840-bp fragment 26 <br /> pSES28         840-bp fragment nusA,s           pUC18           work    bp S. typhimurium DNA replaced 11 bp E. coli <br />                  449 region                                             DNA digested PvuI used replace analo- <br />                  nusAEc                                                         gous fragment nusA st gene pNAS1000. resulting <br /> pNAX449        nusAhy449                           pBR322          work    plasmid, pHSE19, sequenced PvuI restriction <br />  NA, applicable.                                                          sites DNA insert confirm inserted fragment <br />  VOL. 176, 1994 <br />  <br />  <br />  <br />  <br />     nusA <br />  <br />  <br />  <br />    nusAEC <br />  <br />  <br />  <br />  <br /> nusAhyllO2 <br />  <br />   nusAhy2 <br />  <br />  <br />   nusAhy3 <br />  <br />  <br />  nusAhy3O2 <br />  <br />  <br /> nusAhy3O2-449 <br />  <br />  <br />   nusAhy449 <br />  <br />   nusA A324 <br />                   v- <br />  <br />  <br />  <br />  <br />                     * ********(4/9) * * <br />  <br />  <br />  <br />  <br />                                    9 <br />  <br />  <br />  <br />                                    9 <br />  <br />  <br />                                    9 <br />  <br />  <br />  <br />  <br />                                   4li <br />                                         b&gt;&apos;, <br />                                         _S__ <br />  <br />                                         __ <br />  <br />  <br />  <br />  <br />                                         __ <br />                                              * <br />                                                  ~~~~~~~~~~-4 <br />                                                   8 <br />  <br />                                                       *** * ** <br />  <br />  <br />  <br />  <br />    FIG. 2. Structures nusA genes. bar shows                                                                  * *** <br />  <br />  <br />  <br />  <br /> relative positions relevant restriction sites shared nusAFs nusAs, genes. site marked 4/9 position 449 region major heterology, asterisks indicate regions open reading frames encode different amino acids. Aligned schematic representations hybrid genes constructed nusAEc (solid bars) nusAs, (hatched bars). 4 9 boxed <br /> area indicates derivation heterologous region: 4 indicates codons E. coli, 9 indicates codons S. <br /> typhimurium. amino acids position NusAE, protein DLGN, amino acids NusAs, <br /> protein EIKSEGMAG. small region parallel horizontal <br /> lines representation nusAhy1lO2 indicates region AvaI BglII restriction sites nusAst recombination events assumed occurred. <br />                                                                             hybrids constructed method nusAhy2                                                                             nusAhy3O2. <br />                                                                                                            HYBRID nusA GENES <br />  <br />  <br />  <br />                                                                                Method  depicted Fig. 4, relies selection                                                                             functional hybrid nusA genes complementation                                                                             nusAlO(Cs), conditionally lethal mutation (73). Plasmid <br />                                                                             pN2/EuS tandem arrangement nusA4Ec nusAs, <br />                                                                             genes. recognition sites restriction enzyme PvuI                                                                             positions nusA genes, cleavage                                                                             enzyme employed delete DNA                                                                             PvuI sites. Ligation remaining plasmid DNA generates <br />                                                                             hybrid nusA gene. nusAhy3 gene constructed                                                                             method. <br />                                                                                Southern Western blot analyses. Southern blots                                                                             run essentially outlined Sambrook et al. (71). Western <br />                                                                             blots (immunoblots) run essentially according                                                                             method employed Schauer et al. (73), antibody                                                                             identified ECL kit Amersham. <br />                                                                                <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. DNA sequence <br />                                                                             nusA,st deposited GenBank given accession <br />                                                                             number M61008. <br />  <br />  <br />  <br />  <br />                                                                             acid residues. <br />                                                                                                        RESULTS <br />                                                                                Nucleotide sequence nusAst. DNA sequence                                                                             nusAs, determined described Materials Methods. <br />                                                                             nusAEc sequence used comparison based                                                                             original sequencing data subsequent corrections (12, <br />                                                                             44, 45, 68). DNA sequences nusA genes                                                                             extremely similar, varying 156 1,488 nucleotides <br />                                                                             (data shown). predicted primary sequences                                                                             NusAs5 NusAEc proteins compared Fig. 5A.                                                                             expected comparison DNA sequences,                                                                             proteins similar, varying 27 495 amino <br />                                                                                alignment derived amino acid sequences                                                                             NusA proteins, shown Fig. 5A, reveals region  <br />                                                                             sequence amino acids, EIKSEGMAG,                                                                             amino acids, DLGN. small region NusA amino acid <br />                                                                                                                                         1397 <br />  <br />  <br />  <br />  <br />                                                                             significant heterogeneity located amino acids <br />                                                                             152 162 nusA,s sequence. region, NusAst <br />                                                                             corresponding site NusAEc sequence different <br />                                                                             mismatch referred 449 region reflect amino <br />                                                                             acid heterogeneity. second distinguishing feature, set                                                                             tandem repeats carboxy ends NusA proteins,                                                                             revealed dot plot comparison amino acid <br />                                                                             sequences. locations repeats shown Fig. 5A, <br />                                                                             alignment repeats shown Fig. SB.                                                                             identified functional role repeat.  desired deletion-substitution additional                repeat amino acid sequence implicated changes introduced PCR cloning steps.                       TATA recognition eukaryotic transcription factor <br />    hybrid genes constructed                 TFIID (40, 60). <br /> vitro methods outlined  methods exploit restriction                 determine amino acid sequence differences <br /> enzyme recognition sequences occupy identical locations                relate functional differences NusA proteins, reading frames nusAEc nusAst. structures               created series hybrid genes nusAEc nusA,s hybrid genes shown Fig. 2 described                express hybrid NusA proteins. constructs used study <br /> plasmids containing genes described Table 3).                  sufficient genetic material nusA genes include    Method  depicted Fig. 3, permits isolation recom-              known promoters nusAEc operon binants employing selective pressure functional               identified sequence analysis nusA,s operon. Com- <br /> nusA gene. recipient plasmid pNAS2001 tetR gene                   parison DNA sequence obtained region <br /> adjacent cloned nusA gene. portion nusA gene               upstream nusA4s obtained analogous region resected replaced fragment donor                     upstream nusAEc reveals high degree homology; S. <br /> plasmid contains analogous portion heterolo-                typhimurium genome  upstream nusA, open <br /> gous nusA gene downstream DNA replaced                     reading frame corresponding protein 15 kDa, tetR gene BamHI site. cloning creates               fMet-tRNA, similar set promoter sequences (32a, 33, hybrid nusA gene results recombinant plasmid,             38, 72a). constructions, DNA upstream like donor, containing 3&apos; portion tetR gene.              nusA derives source 5&apos; end nusA <br />  1398      CRAVEN ET AL.                                                                                                              J. BAC-FERIOL. <br />  <br />                                  BomHI                                                                                      BomHI <br />                   EcoRI <br />  <br />                                                                                   BomHI <br />  <br />  <br />        Nco                                          Stul <br />  <br />                                                     NcoI - <br />                           Eco RI <br />  <br />                      x Nco                                                                                         x NcoI <br />                      x Bam HI                                             Eco RI                                     x Bom HI <br />                               <br />                     purify f ragment                          -,combine       +   ligate <br />    FIG. 3. Construction nusAhy2. Plasmid pNAG15 (containing nusAE) digested restriction enzymes BamHI NcoI. fragment <br /> containing 3&apos; end nusAE, isolated ligated linearized pNAS2001 completely digested BamHI partially digested NcoI. construction created hybrid nusA gene. Like constructs employed study, cloned nusA gene <br /> included associated promoter nusA operon. Materials Methods details. <br />  <br />  <br /> hybrid; e.g., upstream region derives S. typhimurium               action. plasmid-cloned hybrid nusA genes tested 5&apos; end nusA derives S. typhimurium.                        supported growth bacteria thermosensitive <br />    Structures hybrid nus4 genes. initial studies,             cold-sensitive conditionally lethal nusA alleles (data hybrid nusA genes constructed according               shown).  resulting hybrid NusA proteins strategies relied vitro splicing, detailed Mate-           support essential bacterial functions require NusA action. <br /> rials Methods. structures hybrid genes                     Complementation nusAl mutation hybrid nusA <br /> shown Fig. 2 described  plasmids containing                genes. determine specific region NusAEc genes detailed Table 3).                                       protein necessary N-mediated antitermination, plas- <br />    nusAhy2 gene contains nusAst sequences 5&apos;                     mid-based hybrid nusA genes tested support N <br /> (N-terminal) thirds, contains nusAEc sequences               action nusAl mutant host. complementation test <br /> 3&apos; (C-terminal)  reading frame. nusAhy3                employed based observations.  higher <br /> gene virtually identical nusAst, nusAEc DNA              temperatures wild-type X grows poorly E. coli derivative 5&apos; end encodes sequence single amino acid difference             carrying nusAl mutation, failure N- amino acid sequence derived nusA5s                     mediated antitermination (19). Second, wild-type allele region.  derivation associated 5&apos; regulatory             dominant nusA1/nusA+ heterozygote (21). <br /> region case nusAhy3 E. coli,                  Derivatives X mutations various genes nucleic <br /> nusAst  obviously, S. typhimurium.                                  acid sites involved termination antitermination    nusAhy3O2 gene nusA,E sequences 5&apos; 3&apos;                 transcription provide means assaying host support N <br /> ends contains internal segment derived                  action (28). X derivatives exhibit different degrees <br /> nusAs5. homologies E. coli chromosome               dependence host-encoded Nus factors employed S. typhimurium sequences, nusAhy3O2                   assess N-mediated antitermination:   Abio256sex, fails recombined E. coli chromosome. result-                form plaques lawns formed nus mutants low ing hybrid chromosome E. coli small                      high temperatures exhibits maximal dependence <br /> replacement nuSAEc nusAst sequences.                   (73); (ii) wild-type X, forms plaques lawns low <br /> able unambiguously distinguish effects nusAst                 high temperatures considered reading frame effects expression              normal dependence (21); (iii) Xnin, forms plaques upstream downstream elements S. typhimurium                  lawns formed nus mutants low high <br /> genome.                                                                     temperatures  terms N action, exhibits Nus <br />    Complementation conditionally defective nusA mutations                independence. basis considerations, group hybrid nus4 genes. nusA,E nusAst gene                  phages employed test abilities nusA genes <br /> expressed plasmids complements different condition-                cloned plasmids support N-mediated antitermination ally defective nusA mutations E. coli, cold sensitive                host chromosomal nusAl allele. growth [nusAlO(Cs)] thermosensitive [nusAll(Ts)]                     tester phage heterozygote assessed 40&#194;&#176;C, (data shown).  like chromosomal progen-                 chromosomally encoded mutant NusA fails itors, cloned genes express products functional              support N action. restriction X growth supporting essential bacterial processes requiring NusA                  occurs lower temperatures (40 42&#194;&#176;C)  <br />    FIG. 4. Construction nusAhy3. Plasmid pN 2/EuS contains copy carboxy-truncated nusAEc gene complete nusAs, gene <br /> oriented direction separated short intervening sequence. Cleavage PvuI sites (indicated arrows) followed ligation generates hybrid nusA gene. Following partial digestion PvuI ligation, functional hybrids selected complementation nusAlO(Cs) mutation. Materials Methods details construction. <br />  <br />  <br />  <br /> nusAl mutant host carries pBR322-based plasmid (Table 3                 failure support N-mediated antitermination E. coli data shown).  restrictive temperature               derivatives containing plasmid-based hybrid nusA <br /> used study 40&#194;&#176;C.                                              genes explained expression low levels NusA <br />    growth X derivatives quantitatively assayed              protein. assess levels NusA expression various <br /> nusAl/nusAhy heterozygotes measuring EOP 40&#194;&#176;C (Table                hybrid nusA constructs, measured steady-state levels 4). nusA + bacterium supports growth              NusA protein employing immunoblots. E. coli  <br /> phages, Xnin5 able grow 40&#194;&#176;C nusAl host           quires NusA growth wild-type NusA expressed carries pBR322, vector employed cloning                 chromosome interfere measuring NusA ex- <br /> various nusA genes. plasmid containing nusAEc fully com-                pressed plasmids, employed derivative E. coli, <br /> plements nusAl mutation; phages tested grow            K7346, nusA134  allele (80). absence nusAl/nusAEc heterozygote nusA+ bacterium.                  amber suppressor, strain expresses truncated NusA <br /> plasmid containing nusAst,  fails complemen-              protein supports bacterial growth low temperatures. <br /> tation X growth.                                                       K7346 provides means identifying NusA protein <br />    plasmid truncated nusAA324 gene shows                     expressed plasmid, provided plasmid-encoded <br /> complementation assessed growth X derivative              NusA differs size NusA amber fragment. wild-type Nus factor requirements, showing impor-                immunoblot Fig. 6 shows NusA protein tance 5&apos; region nusA N action.  unlike              K7346 runs lower gel wild-type protein case complete nusA gene, plasmid              position expected amber fragment (lane 1).  <br /> truncated gene, strain heterozygotic nusA              protein identified anti-NusA antibody observed locus fails support growth Xbio256sex. Plasmids            position wild-type NusA protein lanes <br /> cloned hybrid genes nusAhy2, nusAhy3, nusAhy3O2                identifies protein expressed plasmids. complement nusAl  shown Table 4 plasmids pNAX2,                 plasmid-based hybrid genes express roughly equal levels pNAX3, pNAX302). Growth tester phages                   respective NusA proteins.  functional <br /> heterozygotes plasmids indistinguishable               differences observed different nusA-bearing plas- observed nusAlInusAst heterozygote. central               mids likely differences levels regions genes derived nusAst (Fig. 2).                  expression respective NusA proteins. <br />    Levels expression plasmid-based nusA genes.                     Chromosomal hybrid nusA genes. Previous studies  1400         CRAVEN ET AL.                                                                                                                     J. BAC-rERIOL. <br />  <br />                                                                                 TABLE 4. Complementation nusAI mutation hybrid <br />                                                                                                    nusA genes <br />     S.t.    MNKEILAVVEAVSNEKALPREKIFEALESALATATKKKYEQEIDVRVEID 50 <br />     E. c. <br />             111 11111111111111111111111111111111111111111111:11 <br />             MNKEILAVVEAVSNEKALPREKIFEALESALATATKKKYEQEIDVRVQID 50 <br />                                                                                Plasmid             Episomal <br />                                                                                                   nusA gene                           EOPa <br />                                                                                                                                        Anin5       Xbio256sex <br />     S.t.    RKSGDFDTFRRWLIVEEVTMPTKEITLEAARFEDESLNVGDYVEDQIESV 100           pBR322                               10-6             -1             &lt;lo-8 <br />             1111111111111:1:111 11111111111:111111:11111111111               pNAG2010            nusAE,,                -1              -1              -1 <br />     E. c.   RKSGDFDTFRRWLVVDEVTQPTKEITLEAARYEDESLNLGDYVEDQIESV 100           pNAS2001            nusAst               &lt;10-7             -1             &lt;10-8 <br />                                                                              pWR324              nusAA324                 -1            -1             &lt;10-8 <br />     S.t.    TFDRITTQTAKQVIVQKVREAERAMVVDQFRDQEGEIVTGVVKKVNRDNI 150           pNAX2               nusAhy2              &lt;10-7             -1             &lt;10-8 <br />              l l l ll llI 11l <br />                             11ll <br />                               ll ll ll ll ll l: : ll l:1l                    pNAX3               nusAhy3              &lt;10-7             -1             &lt;10-8 <br />     E. c.   TFDRITTQTAKQVIVQKVREAERAMVVDQFREHEGEIITGVVKKVNRDNI 150           pNAX302             nusAhy3O2            &lt;10-7             -1             &lt;10-8 <br />                                                                                              pNAX449             nusAhy449                -1            -1             &lt;10-6 <br />     S.t.    SLEIKSEGMAGNAEAVILREDMLPRENFRPGDRIRGVLYAVRPEARGAQL 200             Calculated dividing phage titer 40&#194;&#176;C indicated strain             I:    : : . II II II II II   1I  1I1                   lawn titer phage permissive strain K37 carrying <br />     E. c.   SL _      jNAEAVILREDMLPRENFRPGDRVRGVLYSVRPEARGAQL 195 <br />                                                   InusAl                pBR322 lawn. <br />                                                        R <br />     S.t.    FVTRSKPEMLIELFRIEVPEIGEEVIEIKAAARDPGSRAKIAVKTNDKRI 250 <br />             11I111111111111111111111111111111111111111111111111 <br />     E. c.   FVTRSKPEMLIELFRIEVPEIGEEVIEIKAAARDPGSRAKIAVKTNDKRI 245           nusAs,  employed hybrid bacterium, Ki 102, defective                                                                              support N action assayed X growth (21, 73).     S.t.    DPVGACVGMRRARVQAVSTELGGERIDIVLWDDNPAQFVINAMAPADVAS 300           genome K1102, derived primarily E. coli, small <br />             111111I1111 111111111111111111111111111111111111111              substitution S. typhimurium genetic information     E. c.   DPVGACVGMRGARVQAVSTELGGERIDIVLWDDNPAQFVINAMAPADVAS 295           thought include entire nusAst gene (2, 21).  <br />                                                                              direct evidence structure nusA <br />     S.t.    IVVDEDKHTMDIAVEAGNLAQAIGRNGQNVRLASQLSGWELNVMTVDDLQ 350           gene bacterium.  employed Southern <br />             11111111111111111111111111111111111111111111111111               analysis chromosomal DNA digested restriction  <br />     E. c.   IVVDEDKHTMDIAVEAGNLAQAIGRNGQNVRLASQLSGWELNVMTVDDLQ 345 <br />                                                                              zymes determine structure nusA gene.     S.t.    AKHQAEAHAAIEIFTKYLDIDEEFATVLVEEGFSTLEELAYVPMKELLEI 400 <br />                                                                              studies (data shown) indicate crossover event <br />             11111111111:. 111111111:111111111111111111 111111111             generating hybrid junctions occurred <br />     E. c.   AKHQAEAHAAIDTFTKYLDIDEDFATVLVEEGFSTLEELAYVPMKELLEI 395           3&apos; segments nusA genes (Fig. 2). position <br />                                 V                                  4         crossover located sites AvaI (nucleotide <br />     S.t.    DGLDEPTVEALRERAKNALATLAQDQEASLGDNKPADDLLNLEGVDRDMA 450           1172 nusAst), present nusA gene K1102, <br />             :11111111111111111111:11.11.11111111111111111111:1               BglI (nucleotide 1280 nusAs,), absent  E. c.      EGLDEPTVEALRERAKNALATIAQAQEESLGDNKPADDLLNLEGVDRDLA 445 <br />                                                         V <br />     S.t.    FKLAARGVCTLEDLAQQGIDHLADIEGLTDEKAGALIIAARNICWFGDEA 500 <br />             111111111II111111:1111.1111111111111111:111111111111                                                      1        2   3    4      5   6     7 <br />  E. c.      FKLAARGVCTLEDLAEQGIDDLADIEGLTDEKAGALIMAARNICWFGDEA 495 <br />                                                                                                 84kd - <br /> B <br /> Ec 364-414       DIDEDFATVLVEEGFSTLEELAYVPMKELLEIEGLDEPTVEALRERAKNAL <br />                   :1 11 1      1 111:11    : :1 :1111:   : 11   I <br />                                                                                            sized NusA <br /> Ec 439-489       GVDRDLAFKLAARGVCTLEDLAEQGIDDLADIEGLTDEKAGALIMAARNIC <br />                   :1:  1  1    111:11      : :  :1:11 : : 11 <br /> St 369-489       DIDEEFATVLVEEGFSTLEELAYVPMKELLEIDGLDEPTVEALRERAKNAL                           58kd - <br />                   :1: 1 1      1 111:11    :    :1:11: : 11     I <br /> St 444-494       GVDRDNAFKLAARGVCTLEDLAQQGIDHLADIEGLTDEKAGALIIAARNIC <br />                      D            D      E   D E       D                          48.5kd - <br />                                                                               Amber fragment NusA <br /> Consensus         <span id='am-23' about='oboInOwl:id' typeof='owl:Thing'>ID</span> -L---G--TLEILA-- JIIGL -l-AL-- N--                                                  40M <br />                                                                                                                        q*M             : <br />                  V     E            E      M D   E D       E   V                                                                        gz* <br />    FIG. 5. Analysis NusA proteins. Single-letter abbreviations used amino acids.   Comparison S. typhimurium E. coli <br /> NusA protein sequences. optimal alignment requires gap amino acids residues 153 154 E. coli sequence.                        36.5 kd - line deduced S. typhimurium (S.t.) NusA protein sequence, line deduced E. coli (E.c.) NusA protein sequence. symbols placed sequences indicate relatedness amino acids: identical amino acids indicated solid line, <br /> chemically similar amino acids indicated dots  similar) dot (similar), unrelated amino acids indicated       FIG. 6. Western blot analysis assess NusA expression blank space. 449 region labeled, location shown       plasmids. Proteins isolated analyzed essentially outlined lines sequence. amino acid change             Schauer et al. (73). E. coli strain used study K7346, <br /> nusAl mutation (68) shown downstream 449 region.               nusA134  mutation expresses truncated NusA <br /> positions repeated sequence indicated horizontal arrows     protein. Positions molecular mass markers, sized NusA protein, starts triangles endpoints. junctions hybrid   truncated NusA amber fragment indicated left. genes created restriction enzyme sites shared nusA genes     lanes (numbered  loaded lysate prepared indicated vertical arrows ( T, PvuI; T T, NcoI; T t ,           K7430 derivatives containing following pBR322 <br /> BglII). (B) Alignment partial repeats carboxy ends      derivative, nusA gene listed parentheses: 1, plasmid NusA proteins. Related amino acids indicated             (control); 2, pNAX2010 (nusAE); 3, pNAS2001 (nusAsj); 4, pNAX449 <br /> symbols described  consensus sequence derived            (nusAhy302-449); 5, pNAX2 (nusAhy2); 6, pNAX3 (nusAhy3); 7, <br /> alignment shown line. Positions alternative        pNAX302 (nusAhy302). Slight differences reflect differences total <br /> related unrelated (-) amino acids indicated.                  protein loaded. <br />  VOL. 176, 1994                                                                                       HYBRID nusA GENES           1401 <br />  <br /> nusA gene K1102. crossover creating              wide variety conditions, included temperatures <br /> hybrid gene occurred 5&apos; nusA gene.          ranging 16 42&#194;&#176;C defined minimal rich position crossover nusA gene,       media anaerobiosis rich medium growth named allele nusAhyllO2.                                        mouse gut (data shown). <br />   second chromosomal hybrid nusA gene created crossing nusAhy302 allele plasmid pNAX302 chromosome  Materials Methods details) gen-                                    DISCUSSION <br /> erate K4091. portion chromosome K4091 derived S. typhimunium sequences extend             Variations closely related species long nucleotide 148 926, extent S.         used tools assess functional consequences subtle <br /> typhimurium DNA nusAhy302. chromosomal                        genetic differences (58). present study, exam- <br /> nusAhy302 allele, like nusAhyllO2 allele, fails support       ined structural basis differences actions nusA <br /> N-mediated antitermination (data shown).                         gene products closely related species enteric <br />   Defining region NusA required N action.             bacteria, E. coli (nusA,) S. typhimurium (nusAst) (66, 72). <br /> nusAEc-nusAst hybrids support action N nUSAEc           demonstration E. coli hybrid small substi- <br /> sequences include 449 region, encodes amino           tution S. typhimurium DNA nusA region failed acids positions 153 156. region         support N-mediated antitermination (21) suggested S. <br /> amino acids NusAE, (DLGN) amino acids                 typhimurium nusA gene, product unable support <br /> NusAst (EIKSEGMAG).                                                  N action. NusA provides essential function,    assess functional significance 449 region,       likely S. typhimurium nusA gene. basis created hybrid gene primarily nusAs,             current study cloning sequencing nusA,s <br /> substitution 449 region; nucleotides encoding         gene comparison derived amino acid amino acids nusAE, 449 region replaced               sequence nusAEc. comparison revealed nucleotides encoding amino acids NusAst.             features consider significant:   sequences <br /> hybrid gene named nusAhy449 reflect switch.              extensive homologies, (ii) alignment amino <br />    effectiveness nusAhy449 gene product support        acid sequences reveals region significant X N-mediated antitermination reaction assessed         variation NusA proteins. <br /> using previously described complementation assay.          NusAst functions genetic background E. coli <br /> study employed K6210, chromosomal nusAl             K-12, surprising derived amino acid <br /> allele pBR322 derivative nusAhy449 allele             sequences nusA genes extensive homologies. <br /> (pNAX449). plates EOP 1 heterozygotic           vivo vitro recombination employed strain 40&#194;&#176;C (Table 4). Bearing mind host       create series hybrid nusA genes. hybrid genes nusAl allele fails support growth 40&#194;&#176;C specifically          used relate structure function examining failure N action, result establishes   effectiveness supporting bacterial growth N-mediated <br /> product plasmid-based nusAhy449 gene supports N               antitermination. <br /> action.  low EOP Xbio256sex diploid           hybrids used study constructed 40&#194;&#176;C    10-6) indicates potency nusAhy449 allele support N action significantly <br />                                                                      include upstream promoter regions. observation <br />                                                                      plasmid constructs hybrid nusA genes <br /> greater nusAst, wild-type       complemented nusA mutations bacterial growth demon- <br />  nusAEc.                                                             strates plasmid-based hybrid genes ex- <br />    determined expression nusAhy449         pressed.  Western blot analysis shows homozygous state sufficient support N action.         constructs express similar levels NusA protein. <br /> accomplished creating chromosomal gene struc-             Support N action. Hybrid nusA genes proximal <br /> ture nusAhy449. end, central region                central portion nusAst fail support X <br /> nusAhy449 crossed chromosomally located nusAhy3O2           N-mediated antitermination. portion NusA, sur- <br /> allele  Materials Methods). previously discussed,         prisingly, includes 449 region identified nusAhy3O2, like nusAst, fails support X N action.             alignment NusA amino acid sequences. resulting hybrid gene, nusAhy302-449, differs nusAhy449,        region NusAst amino acids covering region E. coli sequences 5&apos; 3&apos; ends.      NusAEc different amino acids. test functional  EOP studies showed chromosomally based             significance heterology, created hybrid nusA gene <br /> nusAhy3O2-449 allele, like plasmid-based nusAhy449 allele,       encoded protein NusAst region supports growth X wild-type N growth            heterogeneity, amino acids NusAEc Xbio256sex (data shown).                                         substituted NusAst. <br />    Effect nusA allele bacterial growth. observa-          substitution 449 regions, creating nusAhy449, tion X N functions NusAEc NusAst              sufficient create hybrid protein supports N action. indicates proteins differ functional    diploid chromosomal nusA1 allele plasmid domain. domain facilitate NusA interaction(s)           nusA449 supports growth X derivative wild-type Nus bacterial factor(s), conceivably      dependence conditions nusAl haploid <br /> conditions NusAEc functions effectively E.          parent fails support growth phage.  coli NusAst. assess relative physiological effective-    heterozygote does support growth Abio256sex, phage <br /> ness nusA gene products E. coli background,         expresses low levels altered N protein. compared growth isogenic strains E. coli       proposed significant impairment N <br /> differ nusA genes. strains K1456,      activity, phage requires effective interaction nusAEc, K4092, derivative K1456           components make antitermination complex <br /> nusAhy3O2 allele. Studies monitoring growth cultures       (73).  inability nusAhy449 gene product strains failed reveal significant differences   support growth Abio256sex indicates NusA protein <br />  1402      CRAVEN ET AL.                                                                                                           J. BACTERIOL. <br />  <br />  effective support N action wild-type nus4E&quot;,   acids 449 region supports wild-type N action.  gene product.                                                          nusAhy3O2-449 allele active single copy, phenotype <br />     Lambdoid phages 21 P22 antitermination systems            does vary observed multiple copies; <br />  analogous N X, similarly located N genes        e.g., supports growth X growth Xbio256sex. nut target sites (18, 30, 39, 51). curious,      Conclusion. studies identify 449 region important <br /> phage 21, E. coli phage, N products, unlike      NusA support N action fail provide sufficient X, function nusAs5.  N product uniquely          information elucidate specific role region N <br /> requires 449 region E. coli.  requirement         action determine importance NusA X N NusA protein E. coli 449 region       interactions E. coli. NusA structure probably did <br /> immutable; X derivative mutations N gene         evolve foster interaction X N, likely nutR signal grow hybrid E. coli strain        449 region serves enhance interactions bacterium <br /> nusAhyllO2 (25). N mutations, punAl punA133,                   identified. results current studies <br /> change, respectively, amino acids 45 (Lys Arg) 50 (Ser          effects altering amino acids 449 region Asp) N protein (73). nut mutations, box,A4        surrounding regions provide information formulate boxA , changes base pairs create boxA               definitive unified model. <br /> sequences nutR conform closely consensus <br /> box sequence (27).                                                                          <br />     Considering findings light vitro studies <br /> demonstrating N binding NusA (34), plausible                 <br /> assume interaction X N influenced amino         <br /> acids 449 region. 449 region NusAEr             <br /> positions X N proper contacts NusA              proteins nucleic acid sites. ensure     <br /> formation contacts necessary establish function-          <br /> ing antitermination complex. finding 449 region             NusAE, sufficient X N action consistent       argument additional contacts NusA. Asso-             <br /> ciated 449 region run amino acids immediately          <br /> downstream appears functionally important.            <br /> 26 amino acids 449 region mutations influ- <br /> encing NusA support N action (nusAl) bacterial                                         <br /> growth [nusAll(Ts)] phenotypes (63, 73).                   <br /> region nusA Ec nusAs, unusually rich arginines;         8 conserved arginines run 35 amino acids.                arginines associated regions proteins            interact RNA (6,51, 78), conceivable 449         <br /> region associated arginine-rich region represent          active domain NusA (79). nusAl mutation results              change leucine arginine NusA protein.               extra arginine residue run arginines cause            <br /> NusA bind N Nus factor effectively                <br /> position unfavorably antitermination complex.                    <br />     Chromosomally located nusA hybrids. phenotype                   <br /> nusAst single copy studied using chromosomal hybrid           <br /> genes nusAhyllO2 nusAhy3O2. hybrid                     <br /> genes different proportions nusAEr nusAst,                fail support N action. nusAhyllO2 allele           <br /> assumed complete nusAst gene (21, 73). analysis                 <br /> demonstrates crossover events substituted             <br /> S. typhimurium DNA E. coli DNA K1102 occurred                     3&apos; regions nusA genes. nusAst portion             <br /> nusAhyl1O2, nusAhy3O2, includes 449           <br /> region.  think logical assume studies       <br /> N action chromosomally based hybrid                   <br /> genes reflect phenotype expected complete                <br /> nusAst gene.                                                             <br />     Finally, focus nusAhy3O2. hybrid nusA,Ec           5&apos; 3&apos; ends, S. typhimurium DNA             <br /> bacterium internal nusA gene.  regula-            <br /> tory signals (transcriptional translational) 5&apos;        3&apos; nusA gene E. coli. functional          <br /> features distinguishing nusAst nusAEc                <br /> primarily reflect structural differences. chromosomally                <br /> located nusAhy302 allele, like plasmid-based hybrid, fails          <br /> support N action.  derivative nusAhy3O2    NusAEC amino acids instead NusA5t amino               <br /> </body></html>