<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Impact Extracellularity Evolutionary Rate Mammalian Proteins <br />  <br />  <br />  <br /> Abstract fundamental importance understand determinants rate protein evolution. Eukaryotic extracellular <br /> proteins known evolve faster intracellular proteins. rate difference appears lower <br /> essentiality extracellular proteins intracellular proteins yeast,  mammals, impact extracellularity independent impact gene essentiality. partial correlation analysis indicated impact extracellularity mammalian protein evolutionary rate independent tissue-specificity, expression <br /> level, gene compactness, number protein&#226;&#8364;&#8220;protein interactions  surprisingly, strongest factors examined. Similar results principal component regression analysis. findings suggest different rules govern pace protein sequence evolution mammals yeasts. <br /> Key words: evolutionary rate, subcellular localization, gene essentiality, gene expression level, mammal, yeast. <br />  <br />  <br />  great molecular evolutionists                         ness genomic (Waterston et al. 2002) transcrip- identify factors explain large variation evo-                    tomic (Su et al. 2004) data. Based &#226;&#8364;&#732;&#226;&#8364;&#732;cellular <br /> lutionary rate proteins encoded genome (Fraser et al.                       component&#226;&#8364;&#8482;&#226;&#8364;&#8482; terms Gene Ontology   www. <br /> 2002; Subramanian Kumar 2004; Drummond et al.                                   geneontology.org), treated proteins exclusively located <br /> 2005; Zhang 2005; Liao et al. 2006; Makino                               outside inside cell membrane extracellular Gojobori 2006). Extracellular proteins, known se-                           intracellular proteins, respectively  supplementary <br /> creted proteins, shown exhibit elevated rates                          fig S1, Supplementary Material online Materials nonsynonymous substitutions yeasts mam-                              Methods). Following previous work (Liao et al. 2006), <br /> mals (Winter et al. 2004; Julenius Pedersen 2006; Dean                          defined mouse essential genes knockout phenotypes <br /> et al. 2008), control gene expression level                       premature death sterility  Materials Method). number protein interactions (Julenius Pedersen                           study impact factor protein evolutionary <br /> 2006). yeast,  evolutionary rate longer                        rate, compare closely rated species (Zhang significantly different extra- intracellular                            2005) gene properties (e.g., subcellular localiza- <br /> proteins control gene essentiality (Julenius                      tion essentiality; Liao Zhang 2008; Qian Zhang <br /> Pedersen 2006), suggesting extracellularity does di-                       2009) evolutionary rates change evolution. rectly influence protein evolutionary rate.                       used orthologs mouse rat case mammals. importantly, impact                       (Rattus norvegicus) estimate rates synonymous extracellularity mammalian protein evolutionary rate                       (dS) nonsynonymous (dN) substitutions. Secreted independent gene essentiality                        proteins contain rapidly evolving signal <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> greatest factors examined.                                        (Williams et al. 2000). avoid overestimating substitution <br />     investigate influence extracellularity evo-                    rates extracellular proteins, signal peptides removed <br /> lutionary rate mammalian proteins, chose mouse                                prior estimating dN dS. extracellular <br /> (Mus musculus) focal species comprehensive-                          proteins enriched proteins related immune <br />  <br />  <br /> &#194;&#170; 2010 Authors Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses nc/ <br /> 2.5), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br /> Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010                                                        39 <br />  Liao et al.                                                                                                                             GBE <br />  <br />  <br />  <br />  <br />      FIG. 1.&#226;&#8364;&#8221;Comparison   dN, (B) dS, (C) dN/dS extracellular intracellular proteins. upper quartile, median, lower <br /> quartile indicated box; bars indicate semiquartile ranges. P values (U test) hypothesis difference extracellular intracellular proteins shown panel. <br />  <br /> response (P 5 2.54e 28, v2 test). immune-                          stantially reduced control factors (table 2). <br /> related proteins subject positive selection (Hughes                      Partial correlation analysis limitations <span id='am-6' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-7' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cer</span>- <br /> 1999) X-linked genes tend fast evolving                       tain conditions (Drummond et al. 2006; Kim Yi <br /> (Vicoso Charlesworth 2006), proteins ex-                         2007). conducted principle component regres- <br /> cluded. final data set subsequent analysis included                     sion analysis data. Consistent results <br /> 3,069 mouse&#226;&#8364;&#8220;rat orthologs information gene                           partial correlation analysis (tables 1 2), essentiality subcellular localization mouse.                       extracellularity contributes prin-  1,740 intracellular 288 extracellular                         ciple component explains variance mouse&#226;&#8364;&#8220;rat dN, <br />  supplementary fig. S1, Supplementary Material online).                     dS, dN/dS (supplementary table S1, Supplementary Ma- <br />     extracellular proteins average                        terial online). possible evolutionary rate differ- <br /> mouse&#226;&#8364;&#8220;rat dN (0.047) 61% higher intracellular                      ence intracellular extracellular proteins proteins (0.029; P 5 7.1E-12, Mann&#226;&#8364;&#8220;Whitney U test;                              product different distributions terms fig. 1A). average dS extracellular (0.24)                       groups proteins.  higher dN/dS ex- intracellular (0.21) proteins differ 13.5% (fig.                    tracellular proteins intracellular proteins observed <br /> 1B), elevated mutation rate does fully explain differ-                  compared proteins terms <br /> ence dN. Significantly higher dN/dS extracellular pro-                    (fig 2A C) excluding genes differentially <br /> teins (average 5 0.202) intracellular proteins (0.132;                     distributed terms (fig. 2B D). fig. 1C) suggests extracellular proteins subject                    categories significantly differently distributed  frequent stronger positive selection relaxed pu-                     tracellular extracellular proteins contain rifying selection. Mammalian essential proteins tend                         25 essential extracellular proteins, 25 nonessential extracellu- <br /> evolve slowly (Liao et al. 2006). intracellular                   lar proteins, 25 essential intracellular proteins, 25 nones- <br /> proteins contain higher proportion essential genes                         sential intracellular proteins (supplementary table S2, <br /> (1104/1740 5 63.5%) extracellular proteins (125/                           Supplementary Material online). exception 288 5 43.4%; P 5 1.13e 10, v2 test).  dN                            categories, median dN/dS extracellular proteins sig- <br /> dN/dS extracellular proteins significantly greater                 nificantly higher intracellular proteins pro- intracellular proteins (fig. 1A C),                     teins essentiality compared essential nonessential genes considered. Clearly,                   categories (supplementary table S2, Supplementary Ma- <br /> extracellularity impacts rate mammalian protein                          terial online). repeated analysis removing pro- <br /> evolution independently gene essentiality.                                 teins unknown molecular functions, proteins involved <br />     addition essentiality extracellularity, determi-                  known biological process, proteins located syn- <br /> nants rate mammalian protein evolution                        apse obtained essentially results (supple- <br /> clude expression level, tissue-specificity, number                           mentary figs. S3 S4, Supplementary Material online). <br /> interacting proteins, gene compactness  e., length                         results indicate extracellularity ma- introns untranslated regions, UTRs; Liao et al.                       jor, likely direct, impact mammalian protein evolution. <br /> 2006; Liang Li 2007). compared relative impor-                           influence extracellularity protein evolutionary <br /> tance factors performing Spearman&#226;&#8364;&#8482;s rank                            rate differs greatly mammals <br /> correlation dN factors. Im-                       reported yeasts (Julenius Pedersen <br /> portant factors expected stronger rank correla-                     2006). examine difference dif- <br /> tions dN (Xia et al. 2009). extracellularity                 ferent analytical approaches used, applied  important factor determining dN dN/dS                        alytical procedures orthologs yeast species factors examined (table 1). Furthermore,                          Saccharomyces cerevisiae S. paradoxus  Materials correlation extracellularity dN sub-                     Methods). results yeasts consistent  <br />  <br /> 40            Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010 <br />  Protein Localization Evolutionary Rate                                                                                                    GBE <br />  <br /> Table 1                                                                                      reduction sample size resulted Rank Correlations Various Factors dN dN/dS                                        weaker correlations, changed relative <br />   Gene                     r (P value)                      r (P value)              importance different factors shown tables 1 2. <br /> properties               correlation dN                correlation dN/dS               different impacts extracellularity protein evolu- <br /> Mammals                                                                                      tionary rates yeasts mammals potentially ex- <br />   Extracell               0.177   (6.28e 11)                   0.166    (1.17e 09)           plained ways.  extracellularity qualitatively <br />   5#UTR                   0.144   (9.75e 08)                   0.133    (9.71e 07)           different meanings species  yeasts, se- <br />   Essen                   0.128   (2.21e 06)                   0.101    (1.88e 4) <br />                                                                                              creted proteins outside organisms,   TissSpcf                0.122   (6.88e 06)                   0.096    (4.23e 4) <br />   KPPI                    0.104   (1.21e 4)                    0.103    (1.42e 4) <br />                                                                                              mammals, largely inside organisms.  <br />   3#UTR                   0.085   (1.71e 3)                    0.079    (3.71e 3)            difference implies properties extracellular  <br />   Intron                  0.075   (5.72e 3)                    0.077    (4.48e 3)            tracellular proteins similar mammals   ExpLev                  0.038   (0.165)                      0.060    (0.028)              yeasts, does explain observation rate <br /> Yeasts                                                                                       protein evolution. Second, secreted proteins involved   ExpLev                  0.541   (2.65e 215)                  0.473    (1.22e 158)          biological processes present mammals   Essen                   0.197   (2.73e 26)                   0.202    (1.57e 27) <br />                                                                                              yeasts evolve rapidly.  figure 2 supple- <br />   KPPI                    0.122   (5.80e 11)                   0.151    (7.36e 16) <br />   Extracell               0.022   (0.246)                      0.022    (0.232) <br />                                                                                              mentary table S2 (Supplementary Material online) showed <br />                                                                                               functional categories, extracel- <br />       NOTE.&#226;&#8364;&#8221;Extracell 1 extracellular proteins 0 intracellular proteins. <br /> Essen 1 essential genes 0 nonessential genes. &#226;&#8364;&#732;&#226;&#8364;&#732;UTR&#226;&#8364;&#8482;&#226;&#8364;&#8482; UTR length                                                                                              lular proteins evolve faster intracellular proteins, sug- <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Intron&#226;&#8364;&#8482;&#226;&#8364;&#8482; average length intron. &#226;&#8364;&#732;&#226;&#8364;&#732;KPPI&#226;&#8364;&#8482;&#226;&#8364;&#8482; number interacting proteins.     gesting faster evolution extracellular proteins <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;TissSpcf&#226;&#8364;&#8482;&#226;&#8364;&#8482; tissue-specificity. &#226;&#8364;&#732;&#226;&#8364;&#732;ExpLev&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene expression level. P values                                                                                              attributable special functions proteins.  <br /> probabilities observations hypothesis correlation. analysis based 1,350 mouse&#226;&#8364;&#8220;rat orthologs 2,840 Saccharomyces cerevisiae&#226;&#8364;&#8220;S. paradoxus            cause small fraction mammalian genes subject <br /> orthologs.                                                                                   recurrent positive selection genes likely                                                                                              subject selection  e., immunity genes) previously published (table 1).  extracellularity                              removed analysis, observed evolutionary rate effect yeast protein evolutionary rate                                   difference extracellular intracellular proteins <br /> control gene essentiality (Julenius Pedersen 2006),                                   likely owing differed purifying selection acting expression level important rate determinant                                    exact biological factors cause yeast (Drummond et al. 2006). noted                                     difference purifying selection remain explored. <br /> compared reported previously (Drummond Wilke 2008), observed weaker correlation  <br /> tween expression level dN mammalian proteins                                         Materials Methods <br /> (tables 1 2). difference probably small-                              annotations, sequences, orthologous relationships er number genes used  fewer genes                                  mouse rat genes retrieved Ensembl version 53 information needed partial correlation analysis.                              (www.ensembl.org), yeast genes <br />                                                                                              obtained Saccharomyces Genome Database <br /> Table 2                                                                                      (www.yeastgenome.org). Based  immune-related <br /> Partial Rank Correlations Various Factors dN dN/dS                                proteins annotation 0002376 (immune                                                                                              process). Extracellularity defined terms cel- <br /> Extracell j controlled          r (P value)                 r (P value)       property                correlation dN           correlation dN/dS           lular component  supplementary fig. S1, Supplementary <br /> Mammals <br />                                                                                              Material online). term includes child terms. <br />   Extracell   j   Intron        0.175   (6.50e 11)              0.163   (1.30e 09)           Essentiality mouse genes defined based Mouse Ge- <br />   Extracell   j   ExpLev        0.174   (9.55e 11)              0.159   (3.16e 09)           nome Informatics 4.21 (www.informatics.jax.org), following <br />   Extracell   j   3#UTR         0.172   (1.63e 10)              0.160   (2.79e 09)           Liao Zhang (2007). Essentialities expression levels <br />   Extracell   j   KPPI          0.166   (6.21e 10)              0.154   (1.04e 08)           yeast genes obtained Zhang (2005). <br />   Extracell   j   5#UTR         0.166   (7.03e 10)              0.154   (9.99e 09)           zero fitness gene deletion used define essential <br />   Extracell   j   Essen         0.160   (2.96e 09)              0.151   (1.98e 08) <br />                                                                                              genes yeast mouse. Properties mouse <br />   Extracell   j   TissSpcf      0.158   (4.67e 09)              0.150   (2.63e 08) <br /> Yeasts <br />                                                                                              gene expression defined based microarray data <br />   Extracell   j ExpLev          0.043 (0.023)                   0.040 (0.035)                61 mouse tissues (Su et al. 2004). Expression level cal- <br />   Extracell   j KPPI            0.015 (0.415)                   0.015 (0.439)                culated averaging expression signals 61 tissues, <br />   Extracell   j Essen           0.011 (0.562)                   0.011 (0.547)                tissue-specificity (s), ranges 0 1 (high- <br />      NOTE. note table 1 Extracell, Essen, UTR, Intron, KPPI, TissSpcf,        er values indicate stronger tissue-specificity), calculated <br /> ExpLev. factor &#226;&#8364;&#732;&#226;&#8364;&#732;j&#226;&#8364;&#8482;&#226;&#8364;&#8482; factor examined &#226;&#8364;&#732;&#226;&#8364;&#732;j&#226;&#8364;&#8482;&#226;&#8364;&#8482;     according Liao et al. (2006). Experimentally verified yeast <br /> factor controlled  P values probabilities observations hypothesis correlation. analysis based 1,350 mouse&#226;&#8364;&#8220;rat orthologs                                                                                              protein&#226;&#8364;&#8220;protein interaction (PPI) data obtained 2,840 Saccharomyces cerevisiae&#226;&#8364;&#8220;S. paradoxus orthologs.                                       Batada et al. (2007), thosefor humanwere compiledfrom <br />  <br />  <br /> Genome Biol. Evol. 2(1):39&#226;&#8364;&#8220;43. 2010 doi:10.1093/gbe/evp058 Advance Access publication January 6, 2010                                                     41 <br />  Liao et al.                                                                                                                              GBE <br />  <br />  <br />  <br />  <br />      FIG. 2.&#226;&#8364;&#8221;Higher dN/dS extracellular proteins intracellular proteins observed control biological processes   B) molecular functions (C, D).   (C) performed comparing proteins GOs level 3 (left right second <br /> differentially enriched terms, respectively), (B) (D) performed removing genes significantly differentially enriched terms level 3 (left) level 4 (right). legend figure 1 details boxplot supplementary figure S2 (Supplementary Material online) results dN dS. <br />  sources: Human Protein Reference Database (www                      <br /> .hprd.org), Munich Information center Protein Sequences              <br /> (mips.helmholtz-muenchen , Molecular INTeraction data-                         <br /> base (mint.bio.uniroma2 , Reactome (www.reactome.org),              <br /> IntAct (www.ebi.ac.uk), Database Interacting Proteins                      <br /> (dip.doe-mbi.ucla.edu). human ortholog&#226;&#8364;&#8482;s number                    <br /> interacting proteins (KPPI) used proxy mouse                      <br /> protein&#226;&#8364;&#8482;s KPPI.                                                                    <br />    calculate mammalian yeast protein evolutionary <br />                                                                              <br /> rate, signal peptides, annotated SPdb (proline.bic.nus                           <br /> .edu.sg/spdb/), removed. Orthologous coding sequen-                         <br /> ces signal peptides aligned following                       <br /> protein alignment ClustalW (www.ebi.ac.uk/clustalw/).                             gene multiple isoforms, longest isoform                              used. Values dN dS mouse rat                              S. cerevisiae S. paradoxus computed                           <br /> using PAML 4 (Yang 2007). <br />   <br /> </body></html>