<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Mol Biol. 2010 March 26; 397(2): 481&#226;&#8364;&#8220;494. doi:10.1016/j.jmb.2010.01.051. <br />  <br />  <br />  <br />                            Crystal structures trypanosomal histidyl-tRNA synthetase <br />                            illuminate differences eukaryotic prokaryotic <br />                            homologs <br />  <br />                            Ethan Merritta,b,*, Tracy L Arakakia,b, J Robert Gillespiea,c, Eric T Larsona,b, Angela <br />                            Kelleya,c, Natascha Muellera,c, Alberto J Napulia,c, Jessica Kima,b, Li Zhanga,b, Christophe <br />                            L M J Verlindea,b, Erkang Fana,b, Frank Zuckera,b, Frederick S Bucknera,c, Wesley C Van <br />                            Voorhisa,c, Wim G J Hola,b <br />                            Medical Structural Genomics Pathogenic Protozoa http://www.msgpp.org <br />  <br />                            b   Department Biochemistry, University Washington, Seattle, WA 98195 USA <br />                            c   Department Medicine, University Washington, Seattle, WA 98195 USA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Abstract <br />                                  Crystal structures histidyl-tRNA synthetase eukaryotic parasites Trypanosoma brucei <br />                                  Trypanosoma cruzi provide structural view eukaryotic form enzyme, reveal <br />                                  differences bacterial homologs. Histidyl-tRNA synthetases general contain extra domain <br />                                  inserted conserved motifs 2 3 Class II aminoacyl-tRNA synthetase catalytic core. <br />                                  current structures dimensional topology domain different                                  bacterial archaeal/eukaryotic forms enzyme. Comparison apo histidine-bound <br />                                  trypanosomal structures indicates substantial active site rearrangement histidine binding,                                  relatively little subsequent rearrangement reaction histidine <span id='am-45' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-46' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> form enzyme&#226;&#8364;&#8482;s <br />                                  reaction product, histidyladenylate. specific residues involved forming binding pocket <br />                                  <span id='am-40' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> moiety differ substantially previously characterized binding site                                  bacterial structures homologous residues human histidyl-tRNA synthetases.                                  essentiality single histidyl-tRNA synthetase gene T. brucei shown severe depression <br />                                  parasite growth rate results partial suppression expression RNA interference. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Keywords <br />                                  aminoacyl-tRNA synthetase; protozoa; drug target; tropical disease; RNAi <br />  <br />  <br />                            Introduction <br />                                               Aminoacyl-tRNA synthetases (aaRS) mediate key biological function translating                                               RNA message corresponding protein sequence. specific aminoacyl-tRNA <br />                                               synthetase responsible attaching particular <span id='am-64' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-65' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> proper cognate tRNA.  <br />  <br />                            *Corresponding author: Phone: 206-543-1421 Fax: 206-685-7002, merritt@u.washington.edu. <br />                            Accession Numbers <br />                            Coordinates structure factors deposited Protein Data Bank accession numbers 3HRI, 3HRJ, 3HRK. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service customers <br />                            providing early version manuscript. manuscript undergo copyediting, typesetting, review resulting <br />                            proof published final citable form. note production process errors discovered                            affect content, legal disclaimers apply journal pertain. <br />                             Merritt et al.                                                                                               Page 2 <br />  <br />  <br />                                             proceeds separate reactions, catalyzed active site.  amino acid <br />                                             adenylated reaction ATP yield aminoacyladenylate + <span id='am-17' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span>pyrophosphate</span>. second <br />                                             reaction, adenylate activated amino acid replaced terminal <span id='am-71' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-72' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-79' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenosine</span> residue <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             tRNA <span id='am-137' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-138' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-141' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span>acceptor</span> stem. general results charged tRNA uniquely associates <br />                                             anticodon sequence carries anticodon loop amino acid carries                                             acceptor stem. exceptional cases require subsequent additional <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> modification <br />                                             attached amino acid1. expected molecule key role, aminoacyl- <br />                                             tRNA synthetases living organisms. group classes defined                                             sequence phylogeny consequent structural homology core domains.                                             Class catalytic domain contains Rossmann fold highly conserved sequence motifs. <br />                                             Class II catalytic domain contains core antiparallel &#206;&#178;-sheet surrounded &#206;&#177;-helices,                                             uniquely identified conserved sequence motifs. subdivision Class II                                             subclasses  b, c based nature anticodon binding domain. aaRS                                             amino acid lysine belongs consistently Class Class II forms <br />                                             life. Despite broad conservation core structural functional features,                                             individual aaRS evolved idiosyncratic structural features, additional inserted domains, <br />                                             new biological functions supplement essential role.                                             combination essential biological function cross-species variation suggests                                             individual aaRS useful targets drugs targeting infectious disease [review: Hurdle <br />                                             et al.2]. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             chose trypanosomal Histidyl-tRNA synthetase (HisRS) study broader effort <br />                                             Medical Structural Genomics Pathogenic Protozoa collaborative investigate <br />                                             aminoacyl-tRNA synthetases possible targets drug discovery3. target <br />                                             pathogens Trypanosoma brucei, causative agent African sleeping sickness,                                             Trypanosoma cruzi, causative agent Chagas disease. chronic forms diseases <br />                                             fatal, existing therapeutics unsatisfactory poor effectiveness toxicity. <br />                                             higher eukaryotes, including mammals, separate cytosolic mitochondrial forms <br />                                             aminoacyl-tRNA synthetase. contrast, genomes lower eukaryotes, <br />                                             including trypanosomatids, contain aaRS sequences cytosolic grouping.                                             complication, trypanosomatids evolved separate mitochondrial forms                                             specific aaRS, apparently gene duplication cytosolic sequence4,5.  <br />                                             HisRS  T. brucei T. cruzi genomes contain single HisRS,                                             typical eukaryotic cytosolic type, used cytosolic mitochondrial <br />                                             translation. <br />  <br />                                             HisRS Class IIa aminoacyl-tRNA synthetase containing conserved domains.                                             catalytic domain, core fold shared Class II aaRS. second                                             C-terminal domain, &#206;&#177;/&#206;&#178; fold shared Class IIa enzymes HisRS, ThrRS, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             ProRS, eukaryotic/archaeal GlyRS6. C-terminal domain holds binding <br />                                             site tRNA anticodon loop, anticodon itself primary specificity <br />                                             determinant recognizing tRNAHis. Cognate tRNA recognition instead believed                                             mediated residues Class II motif 2 loop catalytic domain7. structure <br />                                             reported HisRS:tRNAHis complex, tRNA binding modeled <br />                                             approximately based structures bacterial threonyl- prolyl-tRNA synthetase:tRNA <br />                                             complexes8,9. conserved domain unique HisRS, consists 50&#226;&#8364;&#8220;75 residue <br />                                             &#206;&#177;-helical insertion conserved motifs 2 3 catalytic domain. Prokaryotic <br />                                             sequences insertion domain contain highly conserved residues surface <br />                                             near active site proposed interact 3&#226;&#8364;&#178; strand backbone tRNA <br />                                             acceptor stem10. Guth et al.11 insertion domain important                                             histidine adenylation reaction absence tRNA, fluorescent resonance transfer <br />                                             experiments Escherichia coli HisRS suggest forward rate adenylation                                             limited concomitant rigid body rotation insertion domain. Furthermore,  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 3 <br />  <br />  <br />                                             conformational change introduces asymmetry reaction rate monomers <br />                                             making HisRS dimer. equivalent characterization insertion domain                                             reported eukaryotic HisRS. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             addition conserved features, HisRS eukaryotes including humans <br />                                             contains copies short, N-terminal WHEP domain6. WHEP domains named <br />                                             presence non-homologous aaRS (tryptophanyl, histidyl, glutamyl, <br />                                             prolyl), mediate various non-canonical biological activities promoting <br />                                             association multiple aaRS molecules larger assembly12,13. HisRS WHEP domain <br />                                             implicated major epitope human autoimmune disease14. NMR studies <br />                                             shown single WHEP domain human HisRS forms antiparallel 2-helix <br />                                             bundle [unpublished; PDB accession 1&#195;&#8212;59]. structure observed individual <br />                                             modules repeated WHEP domain sequences aaRS15. N-terminal <br />                                             sequence trypanosomal HisRS equivalent length human WHEP domain                                             low sequence similarity. known N-terminus trypanosomal HisRS <br />                                             mediates secondary biological function. <br />  <br />                                             mammalian HisRS genes, HARS HARS2  known HARSL),                                             arose inverted gene duplication16,17. human HisRS gene products 79% sequence <br />                                             identical outside WHEP domain. trypanosomal human <br />                                             HisRS sequences belong  eukaryotic branch HisRS phylogenetic tree18, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             maximum pairwise sequence identity human genes                                             trypanosomal homolog discussed 30%. pairwise sequence identity <br />                                             eukaryotic sequences previously studied bacterial HisRS homologs                                             30%. <br />  <br />                                             Crystal structures bacterial HisRS published, offering insight                                             specific residues binding interactions involved binding substrates histidine                                             ATP10,19,20,9, intermediate product histidyladenylate (HAM)10,21. apo structure <br />                                             archaeal HisRS, Thermococcus acidophilum, determined <br />                                             crystallographically [unpublished; PDB accession 1wu7]. structure eukaryotic HisRS <br />                                             reported <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span>. <br />  <br />                                             report crystal structure apo HisRS T. brucei 2.85 &#195;&#8230; resolution,                                             crystal structures HisRS T. cruzi complex amino acid substrate, histidine, <br />                                             product activation reaction, histidyladenylate, 1.8 &#195;&#8230; 3.05 &#195;&#8230; resolution <br />                                             respectively. structures eukaryotic HisRS insertion domain                                             eukaryotic HisRS adopts different dimensional structure bacterial insertion <br />                                             domain. reveal significant differences binding site histidyladenylate                                             compared previously reported crystal structures bacterial homologs19,10. residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             contributing differences active site distinguish trypanosomal HisRS <br />                                             human HisRS homologs, suggesting possible design selective <br />                                             inhibitors trypanosomal enzymes. <br />  <br />                                Results Discussion <br />                                Comparison overall structure HisRS organisms <br />                                             dimensional structures T. cruzi T. brucei HisRS homologs similar, <br />                                             reflecting fact 84% identical sequence (Figures 1, 2). sequence identity <br />                                             trypanosomal HisrRS previously studied bacterial homologs order <br />                                             25% catalytic domain lower  entire trypanosomal HisRS protein <br />                                             visible crystal structures exceptions N-terminus, truncated <br />                                             expressed proteins begin residue 45 (T. cruzi numbering), residues 456&#226;&#8364;&#8220;467 <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 4 <br />  <br />  <br />                                             near C-terminus anticodon binding domain. Residues 158&#226;&#8364;&#8220;164 conserved Class <br />                                             II aaRS motif 2 ordered apo structure histidyladenylate complex,                                             visible higher resolution histidine complex. Conversely, residues 220&#226;&#8364;&#8220;241, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             comprising half insertion domain, poorly ordered apo structure                                             defined ligand-bound structures. overall folds catalytic anticodon <br />                                             binding domains previously seen studied bacterial homologs,                                             differences residues forming active site discussed separately <br />                                              catalytic anticodon binding domains connected extended stretch                                             residues, 376&#226;&#8364;&#8220;383, following Class II motif 3. <br />  <br />                                             HisRS sequences characterized additional domain inserted conserved <br />                                             motifs 2 3 Class II catalytic core. insertion tends larger eukaryotes                                             prokaryotes. Sequence conservation eukaryotic, archaeal, prokaryotic insertions <br />                                             low, eukaryotes higher. trypanosomal human HisRS sequences                                             roughly 30% identical region (Figure 2). insertion domain proposed                                             play role tRNA recognition21,10, specific model developed. present <br />                                             work reveals eukaryotic prokaryotic insertion domains different                                             dimensional structures (Figure 3). superposition structurally similar eukaryotic <br />                                             prokaryotic core domains, evident correspondence position                                             secondary structural elements belonging respective insertion domains. Given                                             divergence sequence structure region, unlikely single model <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             interaction insertion domain tRNA apply prokaryotic                                             eukaryotic HisRS. <br />  <br />                                             N-terminus eukaryotic HisRS folds 2-helix bundle (WHEP domain)15,22. <br />                                             Truncation 60 N-terminal residues human HisRS results inactive protein                                             measured production tRNAHis&#226;&#8364;&#8240;23, clear loss                                             45-residue WHEP domain absence following 15 residues belonging start <br />                                             catalytic domain. trypansomal proteins present work N-terminal <br />                                             truncations equivalent removal WHEP domain. 44-residue N-terminal <br />                                             truncation T. cruzi HisRS studied retains ability activate histidine,                                             assayed ability charge tRNAHis. obviously present crystal <br />                                             structures, interested evidence WHEP-equivalent domain                                             N-terminus trypanosomal HisRS. 10% sequence identity                                             trypanosomal human sequences 46 residues, corresponding helices <br />                                             human WHEP domain (Figure 2). evaluate likelihood trypanosomal <br />                                             HisRS N-terminus forms helical bundle, examined trypanosomal sequences                                             helix propensity. different secondary structure predictors indicate strong propensity                                             helix formation residues 10&#226;&#8364;&#8220;21 (Jpred24) 5&#226;&#8364;&#8220;28 (Porter25) predict second <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             helical region pair form WHEP-like 2-helix bundle. Equivalent <br />                                             predictions obtained N-terminal sequence HisRS Leishmania major, <br />                                             related pathogenic protozoan. <br />  <br />                                Conformational changes associated substrate binding <br />                                             base active site Class II aaRS formed central strands conserved <br />                                             &#206;&#178;-sheet. &#206;&#178;6, &#206;&#178;10, &#206;&#178;11 present structures (Figure 4). binding surface <br />                                             amino acid ATP substrates largely contributed residues conserved <br />                                             Class II sequence motifs 2 3. case HisRS, histidine recognition binding                                             mediated additional motifs, His B, shared aaRS21. <br />                                             Class II aaRS bind ATP coordination divalent metal <span id='am-22' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> helps                                             position &#206;&#178;- &#206;&#179;- phosphates acts electrophile adenylation reaction, HisRS <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                 Page 5 <br />  <br />  <br />                                             atypical binding ATP catalyzing adenylation involvement divalent <br />                                             cation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS atypical extent substrate binding proceeds induced-fit <br />                                             mechanism. catalytic domain undergoes substantial conformational change                                             histidine binding (Figure 4). Helix &#206;&#177;3 shifts 10 &#195;&#8230; active site. Helices <br />                                             &#206;&#177;8, &#206;&#177;9, &#206;&#177;10, &#206;&#177;11 insertion domain shift concert approximately 3 &#195;&#8230;                                             active site. HisRS-specific histidine recognition sequence  motif), residues 314&#226;&#8364;&#8220; <br />                                             318 T. cruzi enzyme, shifts 3 &#195;&#8230; interact directly histidine.                                             effect repositioning Arg 314 placed interact &#206;&#177;-phosphate                                             ATP, probably contributes increased affnity ATP histidine bound (Figure <br />                                             5). discussed later, attribute positioning &#206;&#178;- &#206;&#179;-phosphates coordination                                             additional arginine residues, conserved HisRS sequences. <br />                                             Somewhat unexpectedly, portion conserved Class II motif 2 interacts                                             <span id='am-86' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-87' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-93' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> ring  residues 158&#226;&#8364;&#8220;165 T. cruzi sequence, ordered apo <br />                                             structure histidine-bound structure. observed lack order                                             different crystal form complex  detailed , making likely                                             true consequence binding histidine artifact crystal lattice packing. <br />  <br />                                             conformational change histidine substrate-bound enzyme                                             histidyladenylate product-bound enzyme. Helix &#206;&#177;3 shifts slightly farther active site. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             addition, motif 2 residues 158&#226;&#8364;&#8220;165 fully ordered interact directly ring <br />                                             adenyl moiety. constitutes shift approximately 7 &#195;&#8230; position                                             C&#206;&#177; atoms residues tip loop compared position apo <br />                                             crystal structure. discussed  specific residues making overall binding <br />                                             pocket adenine ring significantly different current structures                                             equivalents bacterial archaeal homologs previously studied. little <br />                                             shift insertion domain trypanosomal histidine- histidyladenylate- <br />                                             bound structures. course does preclude movement insertion domain                                             course reaction; domain flexibility necessary accommodate                                             recognition binding ATP subsequent release &#206;&#178;- &#206;&#179;- phosphates                                             catalytic displacement previously bound histidine. <br />  <br />                                Order asymmetry substrate binding <br />                                             structural details tRNA binding HisRS directly observed,                                             analogy binding mode seen Class IIa aaRS, cognate tRNA believed                                             interact monomer HisRS dimer8,9. geometry binding                                             tRNA molecules bind simultaneously, monomer dimer.                                             monomers act independently,  Francklyn coworkers extensively <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             characterized reaction ordering kinetics E. coli HisRS dimer. E. coli enzyme <br />                                             interacts weakly,  ATP alone11. presence tRNA, activation histidine <br />                                             second monomer dependent prior completion aminoacyl transfer tRNA                                             active site monomer26. absence tRNA, rate formation                                             histidyladenylate different active sites11. HisRS dimer repurified size <br />                                             exclusion chromatography reaction, recovered complex molar ratio                                             bound histidyladenylate dimer27. led model activation histidine <br />                                             ATP site induces rate-limiting conformational change, possibly rigid-body rotation <br />                                             insertion domain, interferes ATP binding reactivity second <br />                                             site11. <br />  <br />                                             conformational difference apo histidine-bound trypanosomal HisRS <br />                                             crystal structures substantial, provides structural basis sequential binding                                             substrates, histidine ATP second. Thermal melt data trypanosomal HisRS <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 6 <br />  <br />  <br />                                             homologs agreement order binding solution (Figure 5).                                             case addition <span id='am-28' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>AMP</span>, <span id='am-11' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span>, ATP does stabilize HisRS dimer solution, <br />                                             addition histidine shifts TM protein 4 6&#194;&#176; C. presence NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             histidine, addition AMP ADP causes small significant increase stability. Addition <br />                                             histidine ATP leads substantial stabilization protein corresponding <br />                                             &gt;7&#194;&#176; C increase TM relative apo protein. enzymes active, probably <br />                                             represents stability complex protein reaction product, <br />                                             histidyladenylate. ordering &#206;&#8221;TM observed consistently T. cruzi T. <br />                                             brucei protein constructs used current structural analysis, length                                              N-terminal truncation) HisRS related trypanosomatid Leishmania naiffi. <br />  <br />                                              weak evidence structural asymmetry active sites                                             trypanosomal HisRS activation histidine ATP. monomers making <br />                                             crystallographically observed apo bound dimers structurally  <br />                                              crystals T. cruzi HisRS complex contain single monomer                                             asymmetric unit; monomers biological dimer related perfect <br />                                             crystallographic 2-fold symmetry. crystallographically independent monomers <br />                                             making dimer T. cruzi HisRS:HAM complex identical                                             precision model; superposition monomers, RMSD 414 paired <br />                                             C&#206;&#177; atoms model 0.17 &#195;&#8230;. does preclude existence slight differences,                                             effect atomic geometry active site monomers evident NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             crystal structure. <br />  <br />                                             dimers seen previously characterized complexes T. thermophilus histidine <br />                                             (PDB accession 1adj) HAM (PDB accession 1ady) symmetric.                                             structures, monomers dimer superimpose rmsd 0.50 &#195;&#8230;                                             C&#206;&#177; atoms insertion domain, superposition insertion domains                                             paired monomers differ rotation leads displacement distal helices                                             4 &#195;&#8230;. insertion domain single archaeal homolog studied crystallographically, T. <br />                                             acidophilum, structurally like current eukaryotic structures                                             bacterial homologs. apo form archaeal structure reported (PDB <br />                                             1wu7), insertion domains monomers dimer differ                                             rotation leads displacement roughly 4 &#195;&#8230; distal end. <br />  <br />                                             Notwithstanding symmetric geometry monomers T. cruzi HisRS:HAM <br />                                             complex, evidence differential binding histidyladenylate                                             active sites. Electron density corresponding complete molecule histidyladenylate <br />                                             present sites (Figure 6).  electron density monomer greater                                             monomer B. Furthermore, monomer density adenine moiety better <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             resolved density histidine, monomer B density histidine moiety <br />                                             stronger AMP moiety. indicate reaction went completion <br />                                             monomer  fraction monomer B copies present crystal                                             histidine remains unactivated. <br />  <br />                                trypanosomal HisRS active site <br />                                             noted  ATP binding site HisRS fully formed                                             conformational rearrangement induced prior binding histidine. E. coli HisRS <br />                                             complex histidinol ATP19, ATP phosphates positioned interaction <br />                                             guanidinium groups arginine residues. &#206;&#177;-phosphate positioned                                             association bacterial residues Arg 113 Arg 259. homologous T. cruzi residues <br />                                             Arg 156 Arg 314, analogous interactions &#206;&#177;-phosphate seen                                             present complex histidyladenylate. Arg 314 believed act electrophile <br />                                             catalysis, substituting divalent metal cation present aaRS. absence <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                             Page 7 <br />  <br />  <br />                                             ATP-bound structure eukaryotic enzyme, used E. coli complex model <br />                                             interactions &#206;&#178;- &#206;&#179;-phosphates. &#206;&#178;-phosphate interacts bacterial residue Arg <br />                                             113. &#206;&#179;-phosphate positioned bacterial residues Arg 121 Arg 311. infer NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             sequence structure alignment role bacterial Arg 121 played homologous <br />                                             trypanosomal Arg 164.  arginine similar residue trypanosomal <br />                                             sequence alignment equivalent bacterial Arg 311. spatial proximity sidechain <br />                                             trypanosomal residue Arg 334 current structures suggests role positioning <br />                                             ATP &#206;&#179;-phosphate prior catalysis played instead residue non-conserved <br />                                             region sequence preceding B motif (Figure 2). E. coli human homologs <br />                                             <span id='am-113' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-114' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-124' property="rdfs:label" content="Gly" datatype="xsd:string"></span>glycine</span> position, equivalent interaction possible. <br />  <br />                                             hydrogen-bonding network secures sugar moiety histidyladenylate                                             conserved bacterial trypanosomal complexes. sidechain strongly <br />                                             conserved Gln 170 bridges ribose O4&#226;&#8364;&#178; histidyl O. ribose <span id='am-142' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-143' property="rdfs:label" content="O2" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span>O2</span>&#226;&#8364;&#178; donates hydrogen <br />                                             bond backbone carbonyl oxygen Leu 336 accepts hydrogen bond                                             backbone N strongly conserved residue <span id='am-101' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-102' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="Gly" datatype="xsd:string"></span>Gly</span> 363. ribose O3&#226;&#8364;&#178; donates hydrogen <br />                                             bond backbone carbonyl oxygen Ala 335. conserved hydrogen bond <br />                                             terminal O&#206;&#181; strongly conserved residue Glu 158 N6 adenine ring. <br />  <br />                                              binding pocket occupied adenine ring current T. cruzi <br />                                             complex histidyladenylate formed intriguingly different set residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously observed ATP histidyladenylate complexes bacterial homologs. <br />                                             adenine ring seen E. coli T. thermophilus HAM complexes sandwiched <br />                                             Phe 125 Arg 311 (E. coli numbering; Figure 7a). trypanosomal homologs, <br />                                             residue equivalent Phe 125 168 (T. cruzi numbering), histidine ring does <br />                                             perform equivalent role stacking adenine ring (Figure 7b). <br />                                             sequence structural trypanosomal equivalent bacterial Arg 311.                                             binding pocket bacterial Arg 311 extends, trypanosomal <br />                                             enzymes instead pair residues, Cys 365 Val 366, sidechains provide                                             hydrophobic surface adjacent adenine. role stacking face                                             adenine ring instead played sidechain Arg 164, extends                                             opposite binding pocket (Figure 7). <br />  <br />                                             Unlike structural difference bacterial trypanosomal insertion domains,                                             differences ATP binding mode demonstrated T. cruzi E. coli HisRS                                             represent generic difference eukaryotic bacterial forms HisRS active site. <br />                                             corresponding set key residues eukaryotic HisRS sequences, including <br />                                             particular human homologs HARS HARS2, directly equivalent                                             E. coli enzyme T. cruzi enzyme (Table 1, Figure 2).  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             details ATP coordination deduced studies bacterial HisRS21,10,7 likely                                             valid HisRS homologs including higher eukaryotes, trypanosomal <br />                                             homologs deviate quite remarkably  <br />  <br />                                RNA interference HisRS gene T. brucei <br />                                             trypanosomes possess single gene coding histidyl-tRNA synthetase,                                             expected gene product essential protein. able confirm RNAi <br />                                             silencing partial suppression HisRS expression profound effect growth <br />                                             bloodstream-form T. brucei. Targeted knockdown single HisRS gene T. brucei                                             performed using established RNA interference methods T. brucei 28. Cells induced                                             express RNAi construct showed profound suppression growth, factor &gt;103                                             day 4  compared uninduced cultures (Figure 8a). Interestingly, degree <br />                                             mRNA knockdown far complete indicated quantitative PCR mRNA <br />                                             (Figure 8b). likely complete gene knockdown lead  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 8 <br />  <br />  <br />                                             complete growth arrest possibly death. profound growth phenotype partial <br />                                             knockdown mRNA supports selection HisRS drug target, pharmacological <br />                                             inhibition enzyme complete effectively stop growth possibly <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             kill cells. <br />  <br />                                Potential HisRS anti-protozoan drug target <br />                                             Humans HisRS genes. homologous trypanosomal HisRS,                                             overall sequence identity trypanosomal enzymes human homolog                                             30%. current structures, lack sequence conservation extends <br />                                             residues making active site. distinctive differences adenine <br />                                             binding pocket trypanosomal HisRS compared human homologs offer <br />                                             opportunity identification selective inhibitors provide basis anti- <br />                                             trypanosomal drug design. <br />  <br />                                             order determine opportunity extends pathogens, reexamined <br />                                             HisRS sequences protozoa representative higher eukaryotes. particular                                             looked presence cluster residues equivalent T. cruzi [Arg 165, Arg 334, Cys <br />                                             365, Val 366] human E. coli [Phe/Tyr, Gly, Glu, Arg] corresponding <br />                                             positions. cluster single HisRS closely related trypanosomatid <br />                                             Leishmania major apicoplast-specific HisRS sequences various species                                             apicomplexan parasites Plasmodium Toxoplasma.  cytosolic HisRS <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             homologs identified apicoplexan genomes instead matched cluster residues <br />                                             human HisRS sequences, did sequences identified Entamoeba, Giardia, <br />                                             Cryptosporidia. remains open question hypothetical drug specific                                             T. cruzi-like HisRS homologs apicoplast effective apicomplexans <br />                                             possessing second, cytosolic HisRS. <br />  <br />                                Summary <br />                                             trypanosomal histidyl-tRNA synthetase structures presented illuminate differences <br />                                             eukaryotic prokaryotic HisRS homologs, highlight differences                                             trypanosomal human enzymes. insertion domain histidyl-tRNA synthetase                                             eukary-otes seen adopt different dimensional structure reported                                             bacterial homologs. insertion domain bacterial HisRS implicated                                             mechanisms tRNA binding aminoacylation10,11, leaves open question                                             equivalent eukaryotic mechanisms differ extent.                                             informative structure eukaryotic HisRS:tRNAHis complex, date                                             lacking. details substrate binding differ eukaryotic HisRS <br />                                             homologs. substrate binding pocket accepts adenine ring ATP differs <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             significantly character trypanosomal structures reported previously <br />                                             characterized bacterial archaeal structures. surface binding pocket adjacent                                             adenine ring formed pair hydrophobic residues trypanosomes, Cys 365                                             Val 366, replace arginine sequence previously studied homologs,                                             bacterial human. previously assigned functions missing arginine taken                                             Arg 164, observe stack adenine ring, Arg 311,                                             infer coordinate ATP &#206;&#179;-phosphate. peculiarity trypanosomes related <br />                                             protozoa, human HisRS sequences key regions like                                             bacteria trypanosomes. evolutionary basis difference,                                             constitutes possibly exploitable target identification selective anti-protozoan drugs <br />                                             targeting essential enzyme histidyl-tRNA synthetase. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                   Page 9 <br />  <br />  <br />                                Materials Methods <br />                                Choice expression constructs crystallization strategy <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Following standard MSGPP procedure, length truncated variants target T. <br />                                             brucei T. cruzi sequences selected expression. Based multiple sequence <br />                                             alignment HisRS sequence family, priority given truncation boundary <br />                                             extra N-terminal (WHEP) domain conserved HisRS core domains.                                             variants proteins cloned E. coli expression vector AVA0421, derived <br />                                             vector pET14b 29. AVA0421 vector contains N-terminal tag followed                                             protease 3C cleavage site, giving option purifying cleaved uncleaved protein                                             crystallization trials. event, expression observed length T. cruzi variant. <br />                                             cleaved uncleaved forms length T. brucei variant subject                                             crystallization trials addition histidine crystallant, crystals <br />                                             obtained diffracted better 20 &#195;&#8230; resolution. Crystallization trials                                             uncleaved truncation variants screened crystallization histidine,                                             ATP. crystals diffracting better 3 &#195;&#8230; obtained apo <br />                                             T. brucei histidine-bound T. cruzi proteins, focused optimizing                                             exploring additional truncation variants tag cleavage. <br />  <br />                                Protein production crystallography <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             T. brucei HisRS sequence corresponding GenBank accession  AAZ11768.1, <br />                                             coding residues 44&#226;&#8364;&#8220;477 477 residue T. brucei HisRS, PCR amplified                                             genomic DNA T. brucei strain TREU927 GUTat 10.1, cloned AVA0421 vector, <br />                                             expressed E. coli. Protein purified Ni-NTA affnity column followed XK <br />                                             26/60 Superdex 75 column (Amersham Pharmacia Biotech) using SGPP standard buffer <br />                                             (20mM HEPES, 0.5 M NaCl, 2 mM &#206;&#178;-mercaptoethanol, 5% glycerol, 0.025% NaN3 pH 7.5) <br />                                             30. Purified protein retained 22 residue expression tag. Crystals used data collection <br />                                             grown vapor diffusion sitting drops equilibrated reservoir containing <br />                                             2 M lithium sulfate, 0.1 M HEPES buffer pH 7.3, 1 mM TCEP. initial crystallization <br />                                             drops consisted 0.2 &#206;&#188;l protein 22 mg/ml standard buffer plus 0.2 &#206;&#188;l reservoir <br />                                             solution. Crystals soaked 20% ethylene glycol prior frozen liquid nitrogen. <br />  <br />                                             Crystallographic diffraction images collected single crystal beamline 9&#226;&#8364;&#8220;2                                             Stanford Synchrotron Radiation Lightsource processed using Mosflm31. initial <br />                                             structural model available previous molecular replacement experiment using lower <br />                                             resolution data collected crystals different crystal form (data shown). model <br />                                             generated program Phaser using monomer 35% sequence identical <br />                                             HisRS archaeote Thermoplasma acidophilum (PDB 1wu7) probe.  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             relatively poor model did refine major portions later rebuilt using                                             higher resolution structure T. cruzi HisRS complex guide. <br />  <br />                                             T. cruzi HisRS sequence corresponding residues 45&#226;&#8364;&#8220;478 GenBank sequence <br />                                             accession  EAN89395.1, coding 478 residue T. cruzi HisRS, cloned                                             genomic DNA strain TCCL Brener expressed E. coli. Protein purified  <br />                                             Purified protein retained 22 residue expression tag. Crystals grown vapor diffusion <br />                                             sitting drops equilibrated reservoir containing 0.2 M lithium sulfate, 0.1 M <br />                                             BisTris buffer pH 5.5, 26&#226;&#8364;&#8220;28% w/v PEG 3350, 1 mM TCEP, 10 mM histidine.                                             initial crystallization drops consisted 0.15 &#206;&#188;l protein 24 mg/ml standard buffer plus <br />                                             0.15 &#206;&#188;l reservoir solution. Crystals differing unit cells space groups                                             grow conditions. Diffraction data 1.8 &#195;&#8230; resolution single crystal <br />                                             HisRS complex belonging space group C2 collected SSRL beamline 9&#226;&#8364;&#8220; <br />                                             2. Integration scaling performed Mosflm31 Scala32. initial molecular <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                              Page 10 <br />  <br />  <br />                                             replacement model generated BALBES33, used multiple fragments T. <br />                                             acidophilum homolog (1wu7) probes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             histidyladenylate:HisRS complex (HAM:HisRS) generated allowing enzyme <br />                                             catalyze aminoacyl activation reaction <br />  <br />  <br />  <br />  <br />                                             crystals HisRS complex. reaction buffer prepared adding 0.2 &#206;&#188;l 250 <br />                                             mM ATP standard buffer 0.2 &#206;&#188;l 250 mM MgCl2 standard buffer 5 &#206;&#188;l                                             crystallization reservoir solution neighboring containing slightly higher PEG <br />                                             concentration (30% w/v). 2 &#206;&#188;l reaction buffer added directly                                             crystallization drop containing grown crystals. crystals crumbled immediately, <br />                                             survived formed. left equilibrate original reservoir <br />                                             solution 85 minutes frozen directly liquid nitrogen. Diffraction data                                             single crystal collected SSRL beamline 9-2, integrated scaled using <br />                                             HKL200034. space group HAM:HisRS crystal different                                             determined unsoaked HisRS crystals (Table 2). treated crystal diffracted                                             3 &#195;&#8230; resolution, substantial degradation diffraction quality untreated crystals.                                             unclear crystal represents minority crystal form better tolerated exposure <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             reaction buffer, morphed new crystal lattice reformed rapidly <br />                                             reaction-induced disruption original lattice. <br />  <br />                                             Crystallographic data handling project management used CCP4 program suite32,35. <br />                                             Structural superpositions performed using Coot SSM36. Refinement                                             structural models proceeded using alternating cycles refinement Refmac537 model <br />                                             building Coot38. Moderate non-crystallographic symmetry (NCS) restraints applied                                             apo (6 NCS-related chains) HAM:HisRS complex (2 NCS-related chains) models <br />                                             refinement. Model validation performed using Molprobity39. refined <br />                                             crystallographic model structure included multi-segment model atomic <br />                                             displacements generated TLSMD40. Crystallographic statistics Protein Data Bank <br />                                             accession codes given Table 2. <br />  <br />                                Thermal melt assays <br />                                             Protein stability measured solution function temperature adding hydrophobic <br />                                             dye, Sypro Orange (SigmaAldrich). dye emits low fluorescence aqueous solution <br />                                             surrounding folded protein gives increased fluorescent signal consequence NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             binding hydrophobic patches exposed protein releases bound ligand eventually <br />                                             denatures. presence high-affnity ligand, protein general resistant <br />                                             thermal denaturation, resulting shift &#206;&#8221;TM inflection point melting <br />                                             curve41. effect substrate binding trypanosomal histidyl-tRNA synthetases                                             assayed adding substrate potential ligands 0.5 mg/ml protein standard buffer <br />                                             monitoring fluorescence temperature range 20&#226;&#8364;&#8220;90&#194;&#176; C. assay carried                                             96 trays DNA Engine Opticon 2 RT-PCR machine (BioRad). Melting curve shifts <br />                                             measured T. cruzi T. brucei N-terminally truncated HisRS proteins used                                             structural studies, length HisRS trypanosomatid, Leishmania <br />                                             naiffi. <br />  <br />                                RNA interference T. brucei HisRS <br />                                             confirm HisRS gene product essential, T. brucei gene subjected RNAi <br />                                             knockdown. region gene coding sequence used RNAi selected <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                                Page 11 <br />  <br />  <br />                                               using RNA target selection program RNAit42 ensure significant <br />                                               sequence homology genes genome. sequence T. brucei <br />                                               genomic DNA used construction RNAi plasmids taken GeneDB <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               accession number Tb927.6.2060. Bases 467 1097 amplified T. brucei 927 <br />                                               genomic DNA using primers 5&#226;&#8364;&#178;-ATGAGGCAATTACTCGTGGG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                               ATAACGCAATCACCAAAGCC-3&#226;&#8364;&#178;. resulting amplicon ligated TA cloning                                               vector p2T7TABlue  gift D. Horn, London School Hygiene Tropical Medicine) <br />                                               28, sequence confirmed <span id='am-56' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-57' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-61' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence analysis insert. construct <br />                                               linearized NotI restriction enzyme (New England Biolabs, Ipswich, MA). T. <br />                                               brucei bloodstream-form parasites (provided G. Cross, Rockefeller University) expressing <br />                                               T7 RNA polymerase Tet repressor single selection marker (SM), G418 <br />                                               resistance, cultured HMI-9 medium 10% heat-inactivated fetal bovine serum                                               G418 2.5 &#206;&#188;g/ml 37&#194;&#176;C 5% <span id='am-35' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span> atmosphere43. micrograms linearized <br />                                               p2T7TABlue containing HisRS RNAi fragment electroporated mid-log-phase T. <br />                                               brucei (2.5 &#195;&#8212; 107 cells) suspended 500 &#206;&#188;l cytomix (120 mM KCl, 0.15 mM CaCl2, 10 <br />                                               mM K2HPO4-KH2PO4, pH 7.6, 25 mM HEPES, 2 mM disodium EDTA, 5 mM MgCl2). <br />                                               Electroporation 4-mm gap cuvettes 1.6 kV 24 &#206;&#169; resistance, described <br />                                               previously44. cells resuspended HMI-9 medium supplemented 2.5 &#206;&#188;g/ml <br />                                               hygromycin 2.5 &#206;&#188;g/ml G418. Stable individual clones selected 5 7 days                                               transfection. Selected cultures diluted 1 &#195;&#8212; 105 cells/ml induced express double- <br />                                               stranded RNA addition 1 &#206;&#188;g/ml tetracycline. Parasite growth measured period <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               8 days following induction RNAi cultures passed daily 1:2 1:20 dilution                                               needed, monitored ATPLite luminescence ATP detection assay (catalog <br />                                                6016941; Perkin-Elmer, Waltham, MA). quantification gene knockdown, cDNA <br />                                               prepared messenger RNA collected 0h, 24h, 48h RNAi induction                                               tetracycline. mRNA signal knock analyzed quantitative PCR cDNA using <br />                                               primers 5&#226;&#8364;&#178;-TCAATTAAGGCGTCTGATAGCG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                               TGTCGTGCAAGTGATGGTGTC-3&#226;&#8364;&#178;, amplified separate region gene                                               used RNAi construct. amplified products 23 PCR cycles                                               quantified densitometry (normalized &#206;&#178;-tubulin). <br />  <br />                                Acknowledgments <br />                                               work funded NIAID award P01AI067921 (Medical Structural Genomics Pathogenic Protozoa).                                               thank Angela Criswell Rigaku Americas Corporation collecting X-ray data used initial characterization                                               crystals T. cruzi HisRS. Portions research carried Stanford Synchrotron Radiation <br />                                               Lightsource, national user facility operated Stanford University behalf U.S. Department Energy, <br />                                               Office Basic Energy Sciences. <br />  <br />  <br />                                Abbreviations <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS          histidyl-tRNA synthetase <br />                                             HAM            histidyl-adenosinemonophosphate (histidyladenylate) <br />                                             aaRS           aminoacyl-tRNA synthetase <br />  <br />                                References <br />                                               1. Sheppard K, Yuan J, Hohn MJ, Jester B, Devine KM, Soll D. amino acid  tRNA- <br />                                                  dependent amino acid biosynthesis. Nucleic Acids Res 2008;36:1813&#226;&#8364;&#8220;25. [PubMed: 18252769] <br />                                               2. Hurdle JG, O&#226;&#8364;&#8482;Neill AJ, Chopra  Prospects aminoacyl-tRNA synthetase inhibitors new <br />                                                  antimicrobial agents. Antimicrob Agents Chemother 2005;49:4821&#226;&#8364;&#8220;33. [PubMed: 16304142] <br />                                               3. Fan E, Baker D, Gelb MH, Buckner FS, Van Voorhis WC, Phizicky E, Dumont M, Mehlin C, Grayhack <br />                                                  EJ, Sullivan M, Verlinde CL, DeTitta G, Meldrum D, Merritt EA, Earnest TN, Soltis M, Zucker F, <br />  <br />  <br />                                                      J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                        Page 12 <br />  <br />  <br />                                                Myler P, Schoenfeld L, Kim D, Worthey EA, LaCount D, Vignali M, Li J, Mondal S, Massey  <br />                                                Carroll B, Gulde S, Luft JR, DeSoto L, Holl M, Caruthers JM, Bosch J, Robien MA, Arakaki T, Holmes <br />                                                MA, LeTrong  Hol  Structural genomics pathogenic protozoa: overview. Methods NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Molecular Biology 2008;426:497&#226;&#8364;&#8220;513. [PubMed: 18542886] <br />                                             4. Charriere F, O&#226;&#8364;&#8482;Donoghue P, Helgadottir S, Marechal-Drouard L, Cristodero M, Horn EK, Soll D, <br />                                                Schneider  Dual Targeting tRNAAsp Requires Different Aspartyl-tRNA Synthetases                                                Trypanosoma brucei. J Biol Chem 2009;284:16210&#226;&#8364;&#8220;7. [PubMed: 19386587] <br />                                             5. Charriere F, Helgadottir S, Horn EK, Soll D, Schneider  Dual targeting single tRNA(Trp) requires <br />                                                different tryptophanyl-tRNA synthetases Trypanosoma brucei. Proc Natl Acad Sci U S                                                2006;103:6847&#226;&#8364;&#8220;52. [PubMed: 16636268] <br />                                             6. Wolf YI, Aravind L, Grishin NV, Koonin EV. Evolution aminoacyl-tRNA synthetases&#226;&#8364;&#8220;analysis                                                unique domain architectures phylogenetic trees reveals complex history horizontal gene <br />                                                transfer events. Genome Res 1999;9:689&#226;&#8364;&#8220;710. [PubMed: 10447505] <br />                                             7. Hawko SA, Francklyn CS. Covariation specificity-determining structural motif aminoacyl- <br />                                                tRNA synthetase tRNA identity element. Biochemistry 2001;40:1930&#226;&#8364;&#8220;6. [PubMed: 11329259] <br />                                             8. Sankaranarayanan R, Dock-Bregeon AC, Romby P, Caillet J, Springer M, Rees B, Ehresmann C, <br />                                                Ehresmann B, Moras D. structure threonyl-tRNA synthetase-tRNA(Thr) complex enlightens <br />                                                repressor activity reveals essential zinc ion active site. Cell 1999;97:371&#226;&#8364;&#8220;81. [PubMed: <br />                                                10319817] <br />                                             9. Yaremchuk  Tukalo M, Grotli M, Cusack S. succession substrate induced conformational <br />                                                changes ensures amino acid specificity Thermus thermophilus prolyl-tRNA synthetase: <br />                                                comparison histidyl-tRNA synthetase. J Mol Biol 2001;309:989&#226;&#8364;&#8220;1002. [PubMed: 11399074] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             10. Aberg  Yaremchuk  Tukalo M, Rasmussen B, Cusack S. Crystal structure analysis                                                  activation histidine Thermus thermophilus histidyl-tRNA synthetase. Biochemistry <br />                                                  1997;36:3084&#226;&#8364;&#8220;94. [PubMed: 9115984] <br />                                             11. Guth E, Farris M, Bovee M, Francklyn CS. Asymmetric Amino Acid Activation Class II Histidyl- <br />                                                  tRNA Synthetase Escherichia coli. J Biol Chem 2009;284:20753&#226;&#8364;&#8220;62. [PubMed: 19487703] <br />                                             12. Rho SB, Lee KH, Kim JW, Shiba K, Jo YJ, Kim S. Interaction human tRNA synthetases <br />                                                  involves repeated sequence elements. Proc Natl Acad Sci U S 1996;93:10128&#226;&#8364;&#8220;33. [PubMed: <br />                                                  8816763] <br />                                             13. Jia J, Arif  Ray PS, Fox PL. WHEP domains direct noncanonical function glutamyl-Prolyl tRNA <br />                                                  synthetase translational control gene expression. Mol Cell 2008;29:679&#226;&#8364;&#8220;90. [PubMed: <br />                                                  18374644] <br />                                             14. Mathews MB, Bernstein RM. Myositis autoantibody inhibits histidyl-tRNA synthetase: model                                                  autoimmunity. Nature 1983;304:177&#226;&#8364;&#8220;9. [PubMed: 6866113] <br />                                             15. Cahuzac B, Berthonneau E, Birlirakis N, Guittet E, Mirande M. recurrent RNA-binding domain                                                  appended eukaryotic aminoacyl-tRNA synthetases. EMBO J 2000;19:445&#226;&#8364;&#8220;52. [PubMed: <br />                                                  10654942] <br />                                             16. O&#226;&#8364;&#8482;Hanlon TP, Raben N, Miller FW. novel gene oriented head head configuration                                                  human histidyl-tRNA synthetase (HRS) gene encodes mRNA predicts polypeptide <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  homologous HRS. Biochem Biophys Res Commun 1995;210:556&#226;&#8364;&#8220;66. [PubMed: 7755634] <br />                                             17. O&#226;&#8364;&#8482;Hanlon TP, Miller FW. Genomic organization, transcriptional mapping, evolutionary <br />                                                  implications human bi-directional histidyl-tRNA synthetase locus (HARS/HARSL). Biochem <br />                                                  Biophys Res Commun 2002;294:609&#226;&#8364;&#8220;14. [PubMed: 12056811] <br />                                             18. Brindefalk B, Viklund J, Larsson D, Thollesson M, Andersson SGE. Origin evolution                                                  mitochondrial aminoacyl-tRNA synthetases. Mol Biol Evol 2007;24:743&#226;&#8364;&#8220;56. [PubMed: 17182897] <br />                                             19. Arnez JG, Augustine JG, Moras D, Francklyn CS. step aminoacylation atomic level <br />                                                  histidyl-tRNA synthetase. Proc Natl Acad Sci U S 1997;94:7144&#226;&#8364;&#8220;9. [PubMed: 9207058] <br />                                             20. Qiu X, Janson CA, Blackburn MN, Chhohan IK, Hibbs M, Abdel-Meguid SS. Cooperative structural <br />                                                  dynamics novel fidelity mechanism histidyl-tRNA synthetases. Biochemistry <br />                                                  1999;38:12296&#226;&#8364;&#8220;304. [PubMed: 10493797] <br />                                             21. Arnez JG, Harris DC, Mitschler  Rees B, Francklyn CS, Moras D. Crystal structure histidyl- <br />                                                  tRNA synthetase Escherichia coli complexed histidyl-adenylate. EMBO J 1995;14:4143&#226;&#8364;&#8220; <br />                                                  55. [PubMed: 7556055] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                       Page 13 <br />  <br />  <br />                                             22. Shiba K. Intron positions delineate evolutionary path pervasively appended <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>                                                 human aminoacyl-tRNA synthetases. J Mol Evol 2002;55:727&#226;&#8364;&#8220;33. [PubMed: 12486531] <br />                                             23. Raben N, Nichols R, Dohlman J, McPhie P, Sridhar V, Hyde C, Le3 R, Plotz P. motif human <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 histidyl-tRNA synthetase shared aminoacyl-tRNA synthetases coiled- <br />                                                 coil essential enzymatic activity contains major autoantigenic epitope. J Biol Chem <br />                                                 1994;269:24277&#226;&#8364;&#8220;83. [PubMed: 7523371] <br />                                             24. Cole C, Barber JD, Barton GJ. jpred 3 secondary structure prediction server. Nucleic acids <br />                                                 research 2008;36:W197&#226;&#8364;&#8220;201. [PubMed: 18463136] <br />                                             25. Pollastri G, McLysaght  Porter: new, accurate server protein secondary structure prediction. <br />                                                 Bioinformatics 2005;21:1719&#226;&#8364;&#8220;20. [PubMed: 15585524] <br />                                             26. Guth EC, Francklyn CS. Kinetic discrimination tRNA identity conserved motif 2 loop                                                 class II aminoacyl-tRNA synthetase. Mol Cell 2007;25:531&#226;&#8364;&#8220;42. [PubMed: 17317626] <br />                                             27. Guth E, Connolly SH, Bovee M, Francklyn CS. substrate-assisted concerted mechanism                                                 aminoacylation class II aminoacyl-tRNA synthetase. Biochemistry 2005;44:3785&#226;&#8364;&#8220;94. [PubMed: <br />                                                 15751955] <br />                                             28. Alibu VP, Storm L, Haile S, Clayton C, Horn D. doubly inducible RNA interference <br />                                                 rapid RNAi plasmid construction Trypanosoma brucei. Mol Biochem Parasitol 2005;139:75&#226;&#8364;&#8220; <br />                                                 82. [PubMed: 15610821] <br />                                             29. Alexandrov  Vignali M, LaCount DJ, Quartley E, deVries C, Rosa DD, Babulski J, Mitchell SF, <br />                                                 Schoenfeld LW, Fields S, Hol  Dumont  Phizicky EM, Grayhack EJ. facile method                                                 high-throughput expression protein pairs. Mol Cell Proteomics 2004;3:934&#226;&#8364;&#8220;938. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 15240823] <br />                                             30. Mehlin C, Boni E, Buckner FS, Engel L, Feist T, Gelb M, Haji L, Kim D, Liu C, Mueller N, Myler <br />                                                 PJ, Reddy JT, Sampson JN, Subramanian E, Van Voorhis WC, Worthey E, Zucker F, Hol WGJ. <br />                                                 Heterologous expression proteins plasmodium falciparum: results 1000 genes. <br />                                                 Molecular Biochemical Parasitology 2006;148:144&#226;&#8364;&#8220;160. [PubMed: 16644028] <br />                                             31. Leslie AGW. Recent changes mosflm package processing film image plate data. Joint <br />                                                 CCP4 ESF-EACBM Newsletters Protein Crystallography 1992;26:x. <br />                                             32. Collaborative Computational Project  4. ccp4 suite: programs protein crystallography. <br />                                                 Acta Crystallographica Section D 1994;50:760&#226;&#8364;&#8220;763. <br />                                             33. Long F, Vagin AA, Young P, Murshudov GN. BALBES: molecular-replacement pipeline. Acta <br />                                                 Crystallographica Section D 2008;64:125&#226;&#8364;&#8220;132. <br />                                             34. Otwinowski, ; Minor, W. Processing X-ray diffraction data collected oscillation mode.  <br />                                                 Carter, CW., Jr; Sweet, RM., editors. Methods Enzymol.  Vol. 276. Academic Press; New <br />                                                 York: 1997. p. 307-326.Macromolecular Crystallography <br />                                             35. Potterton L, McNicholas S, Krissinel E, Gruber J, Cowtan K, Emsley P, Murshudov GN, Cohen S, <br />                                                 Perrakis  Noble M. Developments ccp4 molecular-graphics project. Acta Crystallographica <br />                                                 Section D 2004;60:2288&#226;&#8364;&#8220;94. <br />                                             36. Krissinel E, Henrick K. Secondary-structure matching (SSM), new tool fast protein structure <br />                                                 alignment dimensions. Acta Crystallographica Section D 2004;60:2256&#226;&#8364;&#8220;2268. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             37. Murshudov GN, Vagin AA, Dodson EJ. Refinement macromolecular structures maximum- <br />                                                 likelihood method. Acta Crystallographica Section D 1997;53:240&#226;&#8364;&#8220;255. <br />                                             38. Emsley P, Cowtan K. Coot: model-building tools molecular graphics. Acta Crystallographica <br />                                                 Section D 2004;60:2126&#226;&#8364;&#8220;32. <br />                                             39. Lovell S, Davis  Arendall WB III, deBakker P, Word J, Prisant M, Richardson J, Richardson D. <br />                                                 Structure validation C&#206;&#177; geometry: &#207;&#8224;, &#207;&#710; C&#206;&#178; deviation. Proteins: Structure, Function,                                                 Genetics 2003;50:437&#226;&#8364;&#8220;450. <br />                                             40. Painter J, Merritt EA. Optimal description protein structure terms multiple groups undergoing <br />                                                 tls motion. Acta Crystallographica Section D 2006;62:439&#226;&#8364;&#8220;450. <br />                                             41. Lo MC, Aulabaugh  Jin G, Cowling R, Bard J, Malamas M, Ellestad G. Evaluation fluorescence- <br />                                                 based thermal shift assays hit identification drug discovery. Anal Biochem 2004;332:153&#226;&#8364;&#8220;9. <br />                                                 [PubMed: 15301960] <br />                                             42. Redmond S, Vadivelu J, Field MC. RNAit: automated web-based tool selection RNAi <br />                                                 targets Trypanosoma brucei. Mol Biochem Parasitol 2003;128:115&#226;&#8364;&#8220;8. [PubMed: 12706807] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                   Page 14 <br />  <br />  <br />                                             43. Hirumi H, Hirumi K. Continuous cultivation Trypanosoma brucei blood stream forms medium <br />                                                 containing low concentration serum protein feeder cell layers. J Parasitol 1989;75:985&#226;&#8364;&#8220; <br />                                                 9. [PubMed: 2614608] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             44. Wirtz E, Leal S, Ochatt C, Cross GA. tightly regulated inducible expression conditional <br />                                                 gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol Biochem Parasitol <br />                                                 1999;99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br />                                             45. Eriani G, Delarue M, Poch O, Ganglo3 J, Moras D. Partition tRNA synthetases classes <br />                                                 based mutually exclusive sets sequence motifs. Nature 1990;347:203&#226;&#8364;&#8220;6. [PubMed: 2203971] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                           Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. dimer T. cruzi HisRS bound histidyladenylate <br />                                             Monomer shown dark brown (catalytic domain), yellow (anticodon binding domain), <br />                                             light brown (insertion domain). Monomer B similarly shown blue. active site                                             monomer occupied histidyladenylate, product half-reaction catalyzed <br />                                             enzyme. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                              Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. Sequence alignment HisRS T. brucei T. cruzi human cytosolic (HARS) <br />                                             mitochondrial (HARS2) homologs, E. coli homolog <br />                                             conserved characteristic Class II aaRS catalytic domain motifs defined Eriani <br />                                             et al.45 indicated sequence, conserved histidine recognition motifs <br />                                             specific HisRS. Secondary structure elements observed T. cruzi HisRS structure <br />                                             shown sequences. &#206;&#177;-helices N-terminal WHEP domain seen NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             NMR studies human eukaryotic homologs indicated human HARS2 <br />                                             sequence. domain included expression constructs trypanosomal <br />                                             proteins studied  region E. coli sequence highlighted orange indicates                                             portion insertion domain structurally equivalent bacterial archaeal/ <br />                                             eukaryotic structures. Residues interacting directly histidyladenylate T. cruzi <br />                                             structure indicated blue letters sequence: H/P/S signify interaction                                             histidyl/phosphate/sugar/adenine moieties respectively. red P residue 334 indicates <br />                                             inferred interaction &#206;&#179;-phosphate ATP. Points relevant sequence difference <br />                                             trypanosomal human sequences highlighted cyan. residues <br />                                             contribute difference shape hydrophobic character binding surface                                             active site. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 3. insertion domain eukaryotic bacterial HisRS structures <br />                                             HisRS contain additional domain inserted motifs 2 3 conserved <br />                                             Class II catalytic domains.  insertion domains eukaryotic bacterial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             HisRS homologs similar sequence structure. figures generated                                             superimposing T. cruzi (3hrk) T. thermophilus (1ady) dimers separating                                             respective insertion domains pure translation horizontal. grey helix lower <br />                                             left structure structurally equivalent element following insertion domain. <br />                                               T. cruzi insertion domain, residues 210&#226;&#8364;&#8220;285, followed helix &#206;&#177; 13 conserved <br />                                             catalytic domain. (b) T. thermophilus insertion domain, residues 164&#226;&#8364;&#8220;235, followed                                             homologous catalytic domain helix. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 4. Superposition apo HisRS histidyladenylate:HisRS complex <br />                                             protein undergoes substantial conformational change binding histidine. apo <br />                                             form shown light colors; histidyladenylate:HisRS complex shown dark colors. <br />                                             superposition apo form T. brucei histidine-bound form                                             T. cruzi;  sequence identity trypanosomal HisRS 84% believe <br />                                             highlighted conformational change attributed histidine binding                                             sequence differences. apo form little affnity ATP. rearrangement residues <br />                                             314&#226;&#8364;&#8220;317, motif, bind histidine acts reposition Arg314 oriented <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             interact &#206;&#177;-phosphate ATP. structural elements undergo substantial <br />                                             conformational shift. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                               Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 5. Effect substrate binding thermal melt curves <br />                                               Fluorescence function temperature. curve normalized maximum <br />                                             peak height. replicate curves shown experimental condition. (b) Shift                                             TM, defined change inflection point melting curve presence ligand, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             relative inflection point melting curve protein ligand. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                    Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                             Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 6. Difference electron density active sites T. cruzi HAM:HisRS complex <br />                                               Site  Difference density contoured 2.4&#207;&#402; (grey cages) 4.5&#207;&#402; (green cages)                                             sigma weighted omit map. (b) Site B. Difference density contoured 2.0&#207;&#402; 3.6&#207;&#402;                                             sigma weighted omit map. view angle slightly different   order                                             clearly better density histidyl moiety. <br />  <br />  <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                    Page 22 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                         Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 7. Adenine binding pocket <br />                                               binding mode histidyladenylate seen previously structures T. <br />                                             thermophilus (1ady) E. coli HisRS (1kmm, shown). face adenine ring                                             stacked sidechain Arg 311. (b) binding mode histidyladenylate seen <br />                                             structure T. cruzi HisRS (3hrk). dot surface surrounds hydrophobic residues <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cys 365 Val 366, occur sequence position equivalent bacterial residue <br />                                             Arg 311. trypanosomal HisRS, adenine ring stacked instead sidechain <br />                                             Arg 164, extends opposite face binding pocket. Residue 334, visible <br />                                             left figure, far away interact directly adenine ring,                                             hypothesized coordinate &#206;&#179;-phosphate ATP. <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                        Page 24 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 8. RNAi silencing T. brucei HisRS <br />                                               Growth bloodstream T. brucei cells culture 1 day intervals following RNAi <br />                                             induction. RNA interference HisRS induced tetracycline (TCN +) compared <br />                                             cells RNAi induction (TCN &#226;&#710;&#8217;). growth rate compared &#226;&#8364;&#732;single <br />                                             marker&#226;&#8364;&#8482; (SM) strain T. brucei transfected RNAi expression vector. <br />                                             Cumulative cell densities shown log-scale product cell number                                             total dilution. clone observed nearly identical phenotype (data                                             shown). (b) Knockdown T. brucei HisRS mRNA 0, 24, 48 hours induction                                             RNAi tetracycline. PCR products 23 cycles imaged staining agarose gel <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                        Page 25 <br />  <br />  <br />                                             ethidium bromide. loading demonstrated PCR &#206;&#178;-tubulin gene.                                             intensity bands 24h 48h 42% 71% 0h control, respectively,                                             measured densitometry. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2011 March 26. <br />                             Merritt et al.                                                                                                                                 Page 26 <br />  <br />  <br />  <br />  <br />                                                                                                  Table 1 <br />                            Residues involved ATP binding <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                T. cruzi (3HRK)                             E. coli (1KMN)       human HARS(2) <br />  <br />                             H-bond adenine N6               Glu 158                                     Glu 114              Asp(Glu) 159 <br />  <br />                             sandwiches Adenine ring            168                                     Phe 125              Phe 171 <br />                                                                Arg 164  structurally homologous)       Arg 311              Arg 388 <br />  <br />                             coordinates &#206;&#177; phosphate            Arg 156                                     Arg 113              Arg 157 <br />                                                                Arg 314                                     Arg 259              Arg 326 <br />  <br />                             coordinates &#206;&#178; phosphate            Arg 156 (inferred)                          Arg 113              Arg 157 <br />  <br />                             coordinates &#206;&#179; phosphate            Arg 164 (inferred)                          Arg 121              Arg 167 <br />                                                                Arg 334  structurally homologous)       Arg 311              Arg 388 <br />  <br />                             provides hydrophobic surface       Cys 365 Val 366                             equivalent        equivalent <br />  <br />                            residues tabulated T. cruzi observed HisRS:HAM complex, coordination &#206;&#178;- &#206;&#179;-phosphates inferred. <br />                            residue numbering bacterial HisRS E. coli sequence. structure available human HisRS, set residues shown, <br />                            human HARS HARS2 sequences align exactly E. coli sequence (Figure 2). Residue numbering human HARS. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                            J Mol Biol. Author manuscript <br /> </body></html>