<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Systems Biology                                                                                                                      BioMed Central <br />  <br />  <br />  <br /> Research article                                                                                                                       Open Access effects protein interactions, gene essentiality regulatory <br /> regions expression variation <br /> Linqi Zhou, Xiaotu Ma Fengzhu Sun* <br />  <br /> Address: Molecular Computational Biology Program, Department Biological Sciences, University Southern California, Los Angeles, CA <br /> 90089-2913, USA <br /> Email: Linqi Zhou - linqizho@usc.edu; Xiaotu Ma - xma@usc.edu; Fengzhu Sun* - fsun@usc.edu <br /> * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 26 June 2008                                                       Received: 5 February 2008 <br />                                                                               Accepted: 26 June 2008 <br /> BMC Systems Biology 2008, 2:54   doi:10.1186/1752-0509-2-54 article available  http://www.biomedcentral.com/1752-0509/2/54 <br /> &#194;&#169; 2008 Zhou et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Identifying factors affecting gene expression variation challenging problem                  genetics. Previous studies shown presence TATA box, number cis-regulatory <br />                  elements, gene essentiality, protein interactions significantly affect gene expression variation. <br />                  Nonetheless, need obtain complete understanding factors                  interactions influence gene expression variation remains challenge. growth rates yeast cells <br />                  DNA-damaging conditions studied gene&apos;s toxicity degree defined <br />                  number conditions growth rate yeast deletion strain significantly <br />                  affected. toxicity degree reflects gene&apos;s importance cell survival DNA-damaging <br />                  conditions, expect negatively associated gene expression variation. Mutations                  cis-regulatory elements transcription factors (TF) regulating gene affect gene&apos;s <br />                  expression study relationship gene expression variation number <br />                  TFs regulating gene. importantly study factors interact                  influencing gene expression variation. <br />                  Results: Using yeast model  evaluated effects separate factors                  interactions gene expression variation: protein interaction degree, toxicity degree, number                  TFs, presence TATA box. Results showed 1) gene expression variation negatively <br />                  correlated protein interaction degree protein interaction network, 2) essential <br />                  genes tend expression variation non-essential genes gene expression variation <br />                  decreases toxicity degree, 3) number TFs regulating gene important <br />                  factor influencing gene expression variation (R2 = 8&#226;&#8364;&#8220;14%). addition, number TFs regulating <br />                  gene important factor influencing gene expression variation TATA- <br />                  containing non-TATA-containing genes, different association strength.  <br />                  gene expression variation significantly negatively correlated toxicity degree                  TATA-containing genes. <br />                  Conclusion: finding distinct mechanisms influence gene expression variation                  TATA-containing non-TATA-containing genes, provides new insights mechanisms                  underlie evolution gene expression. <br />  <br />  <br />  <br />  <br />                                                                                                                                        Page 1 12 <br />                                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Background                                                      studied effect protein-protein interactions pro- <br /> Gene expression variation studied dif-        tein length evolutionary variation (variation ferent levels: single cells common environment         strains species). evolutionary varia- <br /> [1], species variety different envi-     tion negatively correlated protein-protein interac- <br /> ronments [2,3], different species/strains,     tions Saccharomyces cerevisiae Drosophila melanogaster referred evolutionary variation [4-8].   [4] negatively correlated protein length Dro- <br /> paper, study genetic factors affecting gene expression       sophila melanogaster [5]. studies highlighted variation species different envi-        importance protein interactions gene regulatory <br /> ronmental conditions. Broadly, genetic factors affect-      regions gene expression variation. <br /> ing gene expression primarily include binding regulatory proteins cis-elements upstream      studies,  integrated differ- <br /> gene, physical genetic interactions         ent data sources way collectively identifies genes. availability gene expression      interprets key factors affecting gene expression varia- <br /> profiles, protein interaction networks, gene regula-        tion.  conducted studies proteomic tory networks, possible study gene             genomic factors marginally collectively influencing <br /> expression variation associated network fea-       gene expression variation different perturbation <br /> tures genomic factors. case protein interac-      conditions species: yeast. <br /> tion networks, interaction degree, e., number interacting partners given protein, fac-    Protein interactions play important role gene expres- <br /> tors. presence absence TATA box               sion variation. Protein-protein interactions key bio- <br /> number transcription factors (TF) regulating gene          logical events living cell, proteins cell interact <br /> provide examples genomic factors influencing gene            perform certain functions. High <br /> expression variation.                                           throughput technologies, including yeast hybrid sys- <br />                                                                 tems mass spectrometry, generated large studies focused individual factors affecting       protein interactions yeast. Computational <br /> gene expression variation. instance, Newman et al. [1]      methods developed study reliability <br /> developed experimental technique study protein            observed interactions [10,11] build reliable <br /> expression noise single cells showed chromo-        protein interaction networks. efforts resulted <br /> somal distance genes mRNA-half life            development protein interaction data- <br /> associated expression noise.  did         bases, albeit differing degrees reliability, including relationship protein expression noise        MIPS [12], DIP [13] BioGrid [14]. evolution- <br /> protein-protein interactions. Recently, using com-       ary point view, expression profiles neighboring <br /> plete interaction dataset, Batada et al. [9] pro-    genes target gene protein interaction network <br /> tein expression variation negatively correlated         constraints target gene&apos;s expression. <br /> interaction degree protein abundance control-           protein interaction network, interacting <br /> led using data Newman et al. [1]. relationship      partners specific protein affect corresponding <br /> continues hold viable genes. groups       gene&apos;s expression.  protein physical interaction <br /> investigated mRNA expression variation species.          degree, e., number interacting partners given example, Nelson et al. [2] studied relationship         protein, significantly affect gene expression variation. number tissues body parts (expression         present study, gene expression varia- <br /> variation), gene expressed, gene spacing       tion decreases protein interaction degree C. elegans D. melanogaster. gene         protein interaction degree accounts 1&#226;&#8364;&#8220;2% expression variation increases relation intergenic    expression variation model organism yeast, result <br /> distances genes. Walther et al. [3] posi-       consistent previous studies [4,5]. <br /> tive correlation frequency gene&apos;s differen- <br /> tial expression number cis-regulatory elements       key factor affecting gene expression variation gene  thaliana. Furthermore, groups        gene essentiality. Genes classified essential studied gene expression variation, known           non-essential genes based fitness phenotype evolutionary variation, different species/strains.       yeast cell gene deleted normal <br /> Using gene expression data yeast species [7],      growth conditions [15]. Essential genes  different strains derived mutation-        deleted, render yeast cell non-viable. Non- <br /> accumulation experiments [6],  instance,       essential genes classified phenotype interspecies/interstrain variation gene expres-     toxicity-modulating genes based fitness phe- <br /> sion significantly correlated presence/absence      notype yeast cell gene deleted TATA box promoter region. Lemos et al. [4,5]      conditions DNA-damaging treatments [16]. Spe- <br />  <br />  <br />                                                                                                                      Page 2 12 <br />                                                                                                (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                  http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> cifically, define gene&apos;s toxicity modulation degree        influencing TATA-containing gene expression variation number DNA-damaging treatments significantly                complicated non-TATA-contain- <br /> affecting deletion strain&apos;s fitness (toxicity modulation       ing genes. example, TATA-containing genes degree = 0  phenotype), 1, 2, 3, 4). higher         likely epigenetic regulated [17,18]. <br /> toxicity modulation degree, important gene             study gives complete analysis factors relation cell survival.  toxicity degree        interaction affecting gene expression variation <br /> gives quantitative measurement gene&apos;s importance            previous studies. yeast cell survival. measure gene&apos;s functional <br /> importance relation cell survival essentiality        Results Discussion essential genes toxicity modulation degree          present results based MIPS protein physical non-essential genes. expression genes              interaction data [12] yeast gene expression pro- <br /> important cell survival generally stable        files 40 Ca Na exposure conditions [19]. different stimuli fluctuate extensively,             results based interaction datasets (MIPS [12], DIP <br /> hypothesize expression variation essen-           [13], BioGrid [14]) gene expression <br /> tial genes lower non-essential genes           datasets (chemostat (nutritional stress) [20], environmen- <br /> decreases toxicity degree non-essential genes.         tal stress [21], oxidative stress [22], combined gene <br />                                                                    expression dataset 1,500 conditions [7]) number cis-elements shown posi-              given Additional Files 1 2. study tively associated gene expression variation [6].          expression data individually order minimize var- <br /> number cis-elements usually approximated using               iation different laboratories. doing  computational approaches contain false posi-              confirm results based different gene <br /> tive negative predictions. Theoretically, given gene&apos;s       expression data consistent. Consistency results <br /> expression pattern increasingly complex            using variety different datasets adds confidence increasing number transcription factors regu-          conclusions. manuscript, use genes pro- <br /> late gene, directly indirectly. study,   teins interchangeably. declare statistical significance hypothesize number TFs significant predic-        p-value 0.05 adjusting multiple <br /> tor expression variation number            comparisons. study, conducted exploratory <br /> TFs regulating gene (hereinafter referred &apos;number       study factors affecting gene expression variation. TFs&apos;) accounts 8&#226;&#8364;&#8220;14% expression variation,              epidemiological studies, did adjust p-values higher explained number            multiple comparisons.  findings <br /> cis-elements (0.3&#226;&#8364;&#8220;1.7%). implies importance            need tested datasets. <br /> indirect trans-effect expression variation. <br />                                                                    Gene expression variation versus protein interaction TATA box conserved element eukaryotic              degree <br /> promoter region usually bound TATA-binding               measured gene expression variation logarithm <br /> proteins. presence TATA box shown            variance expression levels gene important factors contributing gene             40 Ca Na exposure conditions [19]. stud- <br /> expression variation [6,7]. analysis individ-       ied relationship expression variation ual genomic proteomic factors affecting gene expres-           protein interaction degree using LOWESS function sion indicates distinct                    R [23] fit data. bases, obvious mechanisms specifically influence gene expression             gene expression variation decreases degree variation TATA-containing non-TATA-containing               protein physical interaction (Figure 1A). decreasing <br /> genes. importantly, significant nega-            trend especially significant interaction degree <br /> tive correlation expression variation toxicity         relatively low (&#226;&#8240;&#164; 20). contrast, interaction <br /> degree present TATA-containing genes          degree greater 40, decreasing trend toxicity degree accounts 1.3&#226;&#8364;&#8220;2.6% expression            obvious. Biologically, explained fact <br /> variation. contrast, relationship expression        gene expression variation stabilizes interac- <br /> variation toxicity degree absent non-TATA           tion degree given threshold. potential <br /> taining genes. fact TATA-containing genes             explanation interactions proteins high <br /> enriched stress-related genes [17] explain dif-        degrees simply reliable [24]. large number ference. number TFs significantly posi-         reliable interactions proteins high degrees <br /> tively correlated expression variation TATA-         skew true relationship gene expression non-TATA-containing genes, association strength            variation interaction degrees. true underly- higher non-TATA containing genes TATA-             ing mechanism phenomenon clear, lim- <br /> containing genes. results imply mechanism           ited analysis proteins interaction <br />  <br />  <br />                                                                                                                        Page 3 12 <br />                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                    http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Figure <br /> Gene expression <br />          1         variation negatively correlated protein interaction degree <br /> Gene expression variation negatively correlated protein interaction degree. x-axis represents protein <br /> physical interaction degree, y-axis represents gene expression variation.  LOWESS fit gene expression <br /> variation. B) Bar-plot expression variation genes given protein interaction degree lin- <br /> ear regression fit gene expression variation relation interaction degree. linear coefficient &#206;&#178; = -0.0302, R2 = <br /> 1.41%, p-value = 9.704e-14. red dots mean expression variation genes given protein physical inter- <br /> action (PPI) degree. bar represents standard deviation gene expression variation given PPI degree. scale gene expression variation figures, range y-axis -2.5 0.5. <br />  <br />  <br />  <br />  <br /> degree 20. Protein physical interaction             Gene expression variation versus essentiality, toxicity <br /> degree negatively associated              modulation, interaction degrees <br /> gene expression variation single cells [9] evolu-           noted  divided genes classes: essen- <br /> tionary expression variation [4,5] different                tial non-essential genes. Essential genes likely <br /> strains/species. result relationship             perturbed non-essential genes, significant <br /> gene expression variation interacting degree S.          perturbations essential genes  example, render <br /> cerevisiae consistent findings.                       yeast cell non-viable. classified non- <br />                                                                     essential genes groups  phenotype, 0, tox- <br /> Accordingly, used linear regression fit              icity-modulating proteins degrees 1, 2, 3, 4, <br /> expression variation proteins maximal physical           respectively) according cell&apos;s fitness phenotype <br /> interaction degree 20:                                           changes DNA-damaging agents (methylating <br />                                                                     agent methyl methanesulfonate (MMS), bulky alkylat- <br />                           v = &#206;&#177; + &#206;&#178;d                                ing agent 4-nitroquinoline-N-oxide (4NQO), oxidiz- <br />                                                                     ing agent tert-butyl hydroperoxide (t-BuOOH), 254- v gene expression variation d inter-        nm UV radiation) non-essential genes action degree. &#206;&#177; &#206;&#178; parameters. fitted line          knocked [16]. toxicity degree reflects func- corresponding bar-plot expression variation             tional importance genes relation cell survival shown Figure 1B. gene expression variation            DNA-damaging perturbations, expected <br /> significantly negatively correlated protein inter-         gene expression variation decrease tox- <br /> action degree (&#226;&#8240;&#164; 20) (R2 = 1.41%, &#206;&#178; = -0.0302, p-value =            icity degree increases. number genes tox- <br /> 9.704e-14). negative correlation expression             icity degree 4 small (n = 32), combined variation interaction degree implies protein          group having toxicity degree 3. referred high interaction degrees tolerate extensive expres-          essential genes group toxicity degree 4. Figure <br /> sion variation protein need precise control           2A shows bar-plot linear regression fit gene expression organism function normally.            gene expression variation respect toxicity degree. <br />  <br />  <br />                                                                                                                          Page 4 12 <br />                                                                                                    (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                   http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br />  <br /> Figure  effect <br />          2 essentiality, toxicity degree, protein interaction degree gene expression variation effect essentiality, toxicity degree, protein interaction degree gene expression variation.  Bar- <br /> plot expression variation genes given toxicity degree linear regression fit expression variation genes relation toxicity degree. linear coefficient &#206;&#178; = -0.0629, R2 = 0.75%, p- <br /> value = 4.73e-08. B) mean expression variation linear regression fit expression variation respect PPI <br /> degree non-essential genes stratified according toxicity degree essential genes. &#206;&#178; values -0.0172, - <br /> 0.0230, -0.0304, -0.0164 -0.0460 toxicity degree 0, 1, 2, 3, essential genes, respectively. corresponding <br /> p-values 0.0313, 0.0141, 0.0011, 0.2248 0.0013, respectively. R2 0.31%, 0.75%, 2.75%, 0.8% 4.41%, respectively. labels Figure 1. <br />  <br />   significant negative association gene             conversely, caused positive correlation expression variation toxicity degree observed (R2          interaction degree toxicity degree. consequently <br /> = 0.75%, &#206;&#178; = -0.0629, p-value = 4.73e-08). study         studied relationship gene expression varia- relationship gene essentiality evolution-          tion protein interaction degree gene groups <br /> ary expression variation conducted Tirosh Barkai            stratified according toxicity degrees (Figure 2B). <br /> [8] Choi et al. [25], essential genes          Using Ca Na exposure gene expression data [19], tend lower variation non-essential genes.             significant decreasing trend gene expression consistent result demonstrates         variation respect interaction degree strata variation essential genes (toxicity degree = 4)          toxicity degree 3. correspond- <br /> lower non-essential genes (combine pro-           ing (R2, &#206;&#178;, p-value) (0.31%, -0.017, 0.03), (0.75%, - <br /> teins toxicity degree = 0, 1, 2, 3) (p-value = 0.027).        0.023, 0.014), (2.75%, -0.030, 0.001), (0.8%, -0.016,  study differs studies          0.2248), (4.41%, -0.046, 0.001) toxicity degrees 0, classified genes according toxicity        1, 2, 3 4, respectively. fraction expression vari- <br /> degree negative association gene                 ation explained protein interaction degree expression toxicity degree.                                    increase toxicity degree increases. <br />  observed positive correlation interaction             analyses, toxicity protein interaction <br /> degree toxicity degree (data shown)              degrees negatively associated gene expression var- <br /> fore, asked observed negative correlation              iation.  important gene survival gene expression variation interaction degree           yeast cell, variation expression <br />  <br />  <br />                                                                                                                         Page 5 12 <br />                                                                                                   (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                 http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> levels different conditions. Similarly,          variation toxicity degree observed TATA- <br /> higher interaction degree gene, stability      containing group (Figure 3A) (R2 = 2.59%, &#206;&#178; = -0.1674, p- observed expression levels. Biologically, gene    value = 7.174e-06).  association important cell&apos;s survival participates        expression variation toxicity degree non-TATA- important biological processes. perturbation         containing group marginally significant (R2 = gene&apos;s expression likely cause deleterious effect   0.13%, &#206;&#178; = -0.02338, p-value = 0.0413). Consistent corresponding biological process renders        gene expression datasets, chemostat [20] <br /> cell non-viable. evolutionary consequence             environmental stress [21], combined <br /> hypothesis genes important cell survival appear       gene expression data [7]  Additional File 2: Sup- robust expression levels.                                plementary Table 7), highly significant negative <br />                                                                  relationship gene expression variation toxic- <br /> Expression variation versus gene regulatory regions: TATA        ity degree observed TATA-containing genes, <br /> box, number TFs, toxicity degree                          non-TATA-containing genes.  Previous studies established relationship               effect toxicity degree gene expression variation dif- gene expression variation regulatory             ferent TATA-containing genes versus non-TATA  <br /> regions, including presence/absence TATA box              taining genes. relative small R2 value gene <br /> [6,7], length intergenic regions [2], number      expression variation toxicity degree cis-regulatory elements [3]. asked        explained absence association observed relationship gene expression variation          non-TATA-containing genes. toxicity degree TATA-containing <br /> genes non-TATA-containing genes. accomplish               Previous studies identified number cis-element goal, stratified yeast genes based      motifs contributing gene expression variation  thal- <br /> presence/absence TATA boxes reanalyzed rela-          iana [3], Landry et al. [6] number cis- <br /> tionship gene expression variation toxicity          elements marginally affects gene expression varia- <br /> degree. Consistent previous findings [6,7], clear     tion yeast. studied factors  gene expression variation TATA-containing            number cis-elements number TFs) relation <br /> genes higher non-TATA-containing            gene expression variation. analyzed rela- <br /> genes fixed toxicity degree (p-value &lt; 2.2e-16).        tionship gene expression variation number <br /> Significant negative association gene expression         cis-elements [26] using linear model. Similar  <br />  <br />  <br />  <br /> Figure effect <br />         3 TATA box, number TFs, toxicity degree gene expression variation effect TATA box, number TFs, toxicity degree gene expression variation.  relationship expression variation toxicity degree stratified presence/absence TATA box (R2 = 2.59%, &#206;&#178; = - <br /> 0.1674, p-value = 7.174e-06 TATA-containing gene set; R2 = 0.13%, &#206;&#178; = -0.0234, p-value = 0.0413 non-TATA- <br /> containing gene set). B) relationship expression variation number TFs 25 (R2 = 8.28%, &#206;&#178; = <br /> 0.0654, p-value &lt; 2.2e-16). labels Figure 1. <br />  <br />  <br />  <br />  <br />                                                                                                                      Page 6 12 <br />                                                                                                (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> MacIsaac et al. [26], defined cis-elements according                          Overall analysis factors affecting gene expression <br /> binding probability (p &lt; 0.0001) classified                                variation <br /> according conservation yeast species.                         enumerated  identified factors results presented Table 1 different gene                         influencing gene expression variation S. cerevisiae. expression data. Ca Na exposure data [19],                          addition presence/absence TATA box identified <br /> highest R2 1.33% (p-value = 4.45e-08) cis-ele-                              previous studies [6,7], gene expression <br /> ments conserved species used.                          variation decreases protein interaction studied relationship gene expression                            toxicity modulation degrees increase. findings variation number TFs influence target gene                          consistent studies expression variation expression based gene regulatory network devel-                              single cells [9] evolutionary expression variation <br /> oped Hu et al. [27]. highly significant positive corre-                        different strains/species [4,5,25]. lation gene expression variation number                             gene expression variation increases number of TFs observed (R2 = 8.28%, &#206;&#178; = 0.0654, p-value &lt;                              TFs number cis-elements increases 2.2e-16) (Figure 3B). fraction variation explained                           number TFs regulating gene better predictor number TFs (R2 = 8.28%) higher                               expression variation number cis-elements. number cis-elements indicating                              studied contribution factor number TFs better predictor gene expression var-                         interactions gene expression variation taking <br /> iation number cis-elements. result implies                         factors consideration using Akaike importance trans-effect gene expression varia-                           Information Criterion (AIC) [28]. retained model <br /> tion.                                                                               smallest AIC. Table 2 gives results factors <br /> Table 1: relationship gene expression variation number cis-elements. <br />  <br />  Cis-elements identified binding p &lt; 0.0001 conservation 2 yeast species <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0589         1.17e-06        1.25% <br />  Chemostat                                                                                                         0.0188         0.0067          0.39% <br />  Environmental Stress                                                                                              0.0507         5.48e-07        1.33% <br />  Oxidative Stress                                                                                                  0.0025         0.725           0.01% <br />  <br />  Cis-elements identified binding p &lt; 0.0001 conservation 1 yeast <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0539         4.45e-08        1.33% <br />  Chemostat                                                                                                         0.0161         0.0046          0.36% <br />  Environmental Stress                                                                                              0.0516         3.56e-10        1.74% <br />  Oxidative Stress                                                                                                  0.0013         0.826           0 <br />  <br />  Cis-elements identified binding p &lt; 0.0001 Conservation Criteria <br />  <br />  Gene expression data-set                                                                                          Linear regression              R2 <br />  <br />                                                                                                                    &#206;&#178;              p value <br />  <br />  Ca_Na exposure                                                                                                    0.0393         1.05e-07        1.02% <br />  Chemostat                                                                                                         0.0151         0.0004          0.45% <br />  Environmental Stress                                                                                              0.0346         1.68e-08        1.15% <br />  Oxidative Stress                                                                                                  0.0028         0.526           0.01% <br />  <br />  Cis-elements identified different criteria according conservation species. &#206;&#178; linear coefficient linear <br />  model, p-value related null hypothesis &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0, R2 fraction variation explained number cis- <br />  elements. <br />  <br />  <br />                                                                                                                                              Page 7 12 <br />                                                                                                                        (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                     http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> Table 2: Analysis factors interactions affecting expression variation using stepwise selection AIC. <br />  <br />   variable            Ca_Na_exposure                        Chemostat                        Environmental Stress                 Oxidative Stress <br />  <br />                model       p value        R2       model       p value       R2       model        p value      R2        model        p value         R2 <br />  <br />     x1            &#226;&#710;&#353;        0.4252       0.05%        &#226;&#710;&#353;         0.1582      0.16%         &#226;&#710;&#353;        0.0647      0.28%          &#226;&#710;&#353;         0.1075        0.21% <br />     x2            &#226;&#710;&#353;        0.3721       0.06%        &#226;&#710;&#353;         0.0635      0.28%         &#226;&#710;&#353;        0.0861      0.24%                    0.8729        0.002% <br />     x3            &#226;&#710;&#353;       1.16E-20      6.76%        &#226;&#710;&#353;        4.59E-09     2.73%         &#226;&#710;&#353;        &lt; 2e-16     13.42%         &#226;&#710;&#353;        7.30E-09        2.65% <br />     x4            &#226;&#710;&#353;       3.22E-12      3.83%        &#226;&#710;&#353;        7.96E-10     3.00%         &#226;&#710;&#353;        &lt; 2e-16      6.92%         &#226;&#710;&#353;         0.8841        0.002% <br />    x1*x2 <br />    x1*x3 <br />    x1*x4          &#226;&#710;&#353;        0.0581       0.29%        &#226;&#710;&#353;        0.0100       0.53%                                             &#226;&#710;&#353;         0.0348        0.36% <br />    x2*x3                                             &#226;&#710;&#353;        0.0422       0.33% <br />    x2*x4          &#226;&#710;&#353;        0.0226       0.41%        &#226;&#710;&#353;        0.0036       0.68%         &#226;&#710;&#353;        0.0108      0.53% <br />    x3*x4          &#226;&#710;&#353;        0.0039       0.67%        &#226;&#710;&#353;        0.0185       0.45%         &#226;&#710;&#353;        6.2e-09     2.71% <br />   R2model                  16.36%                             12.73%                               22.39%                               4.43% <br />  <br />  main factors include protein interaction degree (x1), toxicity degree (x2: treat essential genes ones toxicity degree 4), number  TFs (x3), presence TATA box (x4: 1-TATA containing genes, 0-non-TATA containing genes). protein interaction data used  analysis based MIPS dataset. column marked &quot;&#226;&#710;&#353;&quot; indicates inclusion final linear model. multiple linear regression  based final linear model, respectively. p-value related null hypothesis &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. R2 variation explained  model independent variable, respectively. <br />  <br /> included final linear model using MIPS interac-                        non-TATA-containing genes 1.5-fold higher tion data [12] expression profiles stepwise                       corresponding values TATA-containing genes. <br /> selection AIC. studied effect                             Accordingly, p-values non-TATA-containing <br /> selected factors expression variation using linear                             genes orders magnitude smaller regression, corresponding p-values R2 values                          corresponding p-values TATA-containing gene given Table 2. results showed protein inter-                      set. noted TATA-containing genes tend action degree explained 1% variation                            higher number TFs non-TATA-containing genes adjusted main factors                            (p-value = 1.184e-10), explain obser- <br /> interaction terms. oxidative stress gene                           vation association strength expression <br /> expression dataset [22], consistently selected model                          variation number TFs TATA-contain- <br /> contained main factors including protein interaction                         ing genes lower non-TATA-contain- <br /> degree, toxicity degree, number TFs, TATA box,                         ing genes. possible explanation presence interaction terms, e., interaction TATA                         TATA-box weakens effect number TFs gene toxicity degree TATA number TFs.                           expression evolution. Second, toxicity degree interaction terms statistically significant                       significant contributor gene expression varia- expression datasets: Ca Na exposure [19],                        tion TATA-containing genes Ca-Na expo- <br /> chemostat [20], environmental stress [21] (p-value &lt;                          sure [19] Chemostat [20] gene expression dataset, <br /> 0.05) explained 0.4% &#226;&#8364;&#8220; 2.7% variation.                                 significant contributor gene expression <br />                                                                                   variation non-TATA-containing gene set. TATA box interact toxicity                             environmental stress gene expression dataset [21], degree number TFs influencing gene expression                          effect toxicity degree gene expression variation variation, reanalyzed contributions toxicity                            statistically significant.  did observe degree number TFs gene expression variation                             decreasing trend gene expression variation respect <br /> stratified presence/absence TATA box using                          toxicity degree TATA-containing genes  linear model. protein physical interaction degree                         Additional File 1, Supplemental Figure 5). included analysis interaction TATA box physical interaction degree                              toxicity degree gene measures tolerance detected. Table 3 shows different results TATA                        yeast cell different external stress conditions non-TATA gene sets.  number TFs                           gene knocked   expected significant factor gene expression variation                          different relationship gene expression varia- TATA- non-TATA-containing genes p-val-                              tion toxicity degree TATA-containing genes ues 5e-6 datasets.  asso-                       non-TATA-containing genes enrichment ciation strength number TFs gene                               stress-related genes TATA-containing genes, expression variation measured coefficient &#206;&#178;                        [17]. test hypothesis, used set genes related <br />  <br />  <br />                                                                                                                                              Page 8 12 <br />                                                                                                                        (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> Table 3: effect factors expression variation stratified presence/absence TATA box. <br />  <br />  TATA dataset <br />  <br />    variable             Ca_Na_exposure                        Chemostat                       Environmental Stress                 Oxidative Stress <br />  <br />                     &#206;&#178;         p-value      R2          &#206;&#178;          p-value      R2         &#206;&#178;           p-value     R2           &#206;&#178;          p-value      R2 <br />  <br />       x2         -0.1436      0.0024     1.95%      -0.0764       0.0034     1.81%     -0.0510         0.156    0.43%       0.0081         0.778      0.02% <br />       x3         0.0312       5.4e-10    7.90%       0.0168       1.5e-09    7.50%      0.0283        2.6e-13   10.79%      0.0138        4.8e-06     4.37% <br />   R2 (model)                  9.47%                               8.97%                               11.09%                               4.39% <br />  <br />  Non-TATA dataset <br />  <br />    variable             Ca_Na_exposure                        Chemostat                       Environmental Stress                 Oxidative Stress <br />  <br />                     &#206;&#178;         p-value      R2          &#206;&#178;          p-value      R2         &#206;&#178;           p-value     R2           &#206;&#178;          p-value      R2 <br />  <br />       x2         -0.0167       0.402     0.06%      -0.0026       0.8189     0.005%     0.0258         0.086     0.26%      0.0038         0.747     0.009% <br />       x3         0.0502       &lt; 2e-16    8.70%       0.0300       &lt; 2e-16     9.56%     0.0468        &lt; 2e-16    12.6%      0.0181        4.4e-10    3.40% <br />   R2 (model)                  8.85%                                9.62%                              12.67%                               3.39% <br />  <br />  linear model includes toxicity degree (x2) number TFs (x3) built gene expression datasets, <br />  respectively. R2 variation explained independent factor model, respectively. &#206;&#178; linear coefficient linear model,  p-value related null hypothesis &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. Environmental Stress Response (ESR) [21].                              non-TATA-containing genes [17]. Basehoar et al. [17] <br /> hypothesis true, expect higher association                         suggested distinct regulatory mechanisms expression variation toxicity degree                          present TATA- TATA promoters. results ESR genes non-ESR genes.                            Table 3 support findings.  data shows &#206;&#178; values similar groups genes (Table 4). explain                        results based oxidative stress gene expression <br /> interaction TATA toxicity degree                              dataset [22] consistent results based enrichment ESR genes TATA-containing genes.                            gene expression datasets. observation <br /> biological mechanisms underlying observed interac-                           relatively small gene expression varia- <br /> tion clear need studied.                               tion data. example, range variance                                                                                  expression levels oxidative stress dataset, did analysis average gene expres-                       (0.07, 5.34), smaller corresponding <br /> sion variation expression datasets (Ca                       ranges, (0.02, 10.59), (0.17, 9.18), (0.09, 11.07), Na exposure [19], chemostat [20], environmental stress                           Ca Na exposure [19], chemostat [20], envi- <br /> [21], oxidative stress [22]) combined gene                           ronmental stress conditions [21], respectively. <br /> expression data Landry et al. [7], results pre- <br /> sented Additional File 2. conclusions                         studied contributing factors gene expres- <br /> obtained indicating robustness results. Previous                      sion variation using DIP [13] BioGrid interactions <br /> studies showed TATA- non-TATA-containing                                [14], results given Additional File 1 File <br /> genes recruit different coactivator complexes                          2. Similar conclusions based MIPS interac- <br /> gene expression [17]. TATA-containing genes                            tion data [12] obtained. consistency subject greater nucleosomal regulation                            results using different combinations protein interaction <br /> Table 4: effect toxicity degree expression variation stratified set environmental stress response (ESR). <br />  <br />  Gene group                   Ca_Na_exposure                       Chemostat                     Environmental Stress                Oxidative Stress <br />  <br />                              &#206;&#178;            p-value             &#206;&#178;             p-value              &#206;&#178;              p-value               &#206;&#178;             p-value <br />  <br />  ESR                       -0.0710        0.0053           -0.0314           0.0532           -0.0176            0.3640            0.0026           0.8708 <br />  Non-ESR                   -0.0848       2.09e-13          -0.0354          1.11e-07          -0.0579           1.28e-11           -0.0091          0.2086 <br />  <br />  linear model built gene expression datasets, respectively. &#206;&#178; linear coefficient linear model p-value  related null hypothesis &#206;&#178; &#226;&#8240;&#160; 0 versus &#206;&#178; = 0. <br />  <br />  <br />                                                                                                                                               Page 9 12 <br />                                                                                                                        (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                              http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> data sets gene expression profiles showed robust-     40 conditions. datasets include yeast gene <br /> ness conclusions.  fraction gene        expression profiles 40 Ca Na exposure condi- <br /> expression variation explained factors    tions [19], chemostat  e., nutritional stress) 100  <br /> 25%. possible explanation measurement         ditions [20], environmental stress 156 conditions [21] gene expression changes factors, including       oxidative stress 70 conditions [22]. data toxicity degree interaction degree,    analyzed separately ensure laboratory <br /> noisy. expect true R2 higher        variation minimized. combined gene expression observed study.                                  profile 1,500 conditions collected <br />                                                               [7]. responsiveness gene Conclusion                                                    1,500 conditions calculated [7] used analy- implemented wide analysis proteomic            sis expression variation. genomic factors affecting gene expression variation. different factors (protein interaction degree,     Protein interaction data <br /> toxicity degree, TATA box, number TFs), TATA-box       downloaded yeast protein interaction data number TFs important factors          different data sources. MIPS (Munich Information <br /> influencing gene expression variation. influence       Center Protein Sequences) [12] dataset (version: <br /> TATA-box evolutionary gene expression variation        PPI_18052006.tab) contains 11,124 protein physical extensively studied computationally exper-      interactions involving 4,404 proteins. DIP core inter- <br /> imentally [6,8], results consistent    action dataset [13] (version: ScereCR20070107) contains <br /> findings. intuitive number TFs     5,738 protein interactions involving 2,161 proteins. regulating gene significant effect     DIP core interactions assessed number qual- <br /> gene&apos;s expression variation, magnitude influence   ity tests supposed highly reliable [11]. studied large scale expression datasets    BioGrid [14] dataset (version 2.0.34) contains 59,317 best knowledge. findings demonstrated          protein physical interactions involving 5,054 proteins. gene regulation main factor affecting gene <br /> expression variation. Protein interaction degree toxic-   Essential toxicity modulating genes <br /> ity degree account variation          Large scale gene deletion studies identified compared influence number TFs        17&#226;&#8364;&#8220;20% genes essential yeast cell survival [15] <br /> TATA-box.                                                     normal conditions. class non- <br />                                                               essential genes, gene&apos;s importance relation cell sur- overall analysis, interactions       vival  studies classified non- TATA-box toxicity degree           essential genes based cell&apos;s fitness phenotypes <br /> number TFs influence expression variation.     different DNA damage perturbations study stratified TATA-box indicated TATA-contain-     gene knocked [16]. toxicity modulating genes <br /> ing genes non-TATA containing genes behave differ-        defined significantly affecting cell&apos;s fit- <br /> ently relation toxicity degree number    ness phenotype knocked  defined toxic- <br /> TFs. effect number TFs expression varia-     ity degree gene number perturbations tion TATA-containing genes lower      significantly affected deletion strain&apos;s fitness. Essential non-TATA-containing genes. hand,         genes downloaded SGD website [29], toxicity degree associated expression variation       toxicity degrees non-essential genes calculated TATA-containing genes  findings         [16]. <br /> suggest regulatory mechanism complicated TATA-containing genes non-TATA           Gene Regulatory Network <br /> containing genes.                                             Studies shown gene expression variation pos- <br />                                                               itively correlated number cis-regulatory ele- <br /> Methods                                                       ments length intergenic region order study factors affecting gene expression varia-    organisms. cis-elements control expression tion, collected data gene expression profiles, pro-     genes interaction TFs, interesting tein physical interactions, gene regulatory networks,         study number TFs regulating gene effect <br /> essentiality toxicity resistance. Details data   gene expression variation. mapping cis-ele- given                                               ments genes obtained using motif discovery algo- <br />                                                               rithms, PhyloCon Converge, binding p-value <br /> Gene expression profiles                                      0.001 conservation 0, 1 2 large number gene expression studies available.   yeast species [26]. mapping TFs genes study, chose gene expression studies containing    obtained Hu et al. [27]. <br />  <br />  <br />                                                                                                                   Page 10 12 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                       http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> TATA-containing genes                                           clear. factors highly correlated, TATA box DNA sequence (cis-element)         treat independent factors joint analysis. <br /> promoter region eukaryotic genes. TATA  <br /> sensus sequence identified TATA T T) G)          overall analysis, used stepwise selection [17]. TATA box identified impor-         model gives smallest AIC (Akaike informa- <br /> tant factor gene expression variation. relationship     tion criterion) = 2*K+n*ln(SSE/n), K number yeast genes TATA box downloaded             parameters model; n number observa- [17]. 1090 6278 genes pre-      tions; SSE residual sum squares [30]. dicted TATA box. analysis used 1090         used linear regression analyze relationship genes TATA-containing genes genes non-          gene expression variation retained factors TATA-containing genes.  note 607 genes          interactions. corresponding p-values R2 val- <br /> classified [17], results essentially    ues reported Table 2. genes considered (data shown).) <br />                                                                 Authors&apos; contributions <br /> Statistical Analysis                                            authors contributed design coordination Gene expression variation measured logarithm         study. LZ carried calculations performed variance gene expression levels various     statistical analysis. FS drafted original manuscript, <br /> conditions. distribution variance        revised XM LZ. authors read mal. addition, standard deviations resulting      approved final manuscript. <br /> distributions conditional independent variables <br /> (protein physical interaction degree, toxicity degree, TATA     Additional material <br /> box, number TFs) differed widely, making linear <br /> model variance invalid. avoid problems, measured gene expression variation loga- <br />                                                                      Additional file 1 <br />                                                                      statistical analysis individual factors gene expression variation <br /> rithm variance. resulting distributions           using expression datasets, respectively. <br /> fit conditions linear model.                Click file <br /> study, used linear model study relationship              [http://www.biomedcentral.com/content/supplementary/1752- expression variation factor.             0509-2-54-S1.pdf] <br /> study relationship gene expression var- <br /> iation interaction degrees, used LOWESS             Additional file 2 <br /> function R [23] fit data. approximate linear            Overall analysis main factors (protein interaction degree, toxicity <br />                                                                      degree, number TFs presence TATA box) interac- <br /> relationship gene expression variation inter-            tions. <br /> action degree observed interaction degree               Click file 20. proceeded use linear regres-            [http://www.biomedcentral.com/content/supplementary/1752- <br /> sion fit data interaction degree 20.                    0509-2-54-S2.pdf] <br />  <br />  <br />                          v = &#206;&#177; + &#206;&#178;d <br />  v gene expression variation d inter-    Acknowledgements <br /> action degree. &#206;&#177; &#206;&#178; parameters. tested sta-       thank Mr. David Martin carefully reading manuscript tistical significance relationship gene         suggestions significantly improved presentation paper.                                                                 work partly supported NIH/NSF Joint Mathematical Biology Initi- <br /> expression variation interaction degree based                                                                 ative DMS-0241102 NIH P50 HG 002790. sincerely thank linear regression model.                                        anonymous reviewers pointing important references                                                                 missed original version. thank reviewers sug- joint analysis expression variation         gestions significantly improved presentation paper. respect factors (protein interaction degree, <br /> toxicity degree, number TFs, TATA box), tested        References factors highly correlated. calculated    1.      Newman JR, Ghaemmaghami S, Ihmels J, Breslow DK, Noble M, <br />                                                                         DeRisi JL, Weissman JS: Single-cell proteomic analysis S. <span id='am-1' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span>cer</span>- <br /> correlation matrix given Addi-                evisiae reveals architecture biological noise. Nature <br /> tional File 2 (Supplementary Table 9). correlation              2006, 441:840-6. <br /> coefficients smaller 0.3 indicating      2.      Nelson CE, Hersh BM, Carroll SB: regulatory content                                                                         intergenic DNA shapes genome architecture. Genome Biol highly correlated. compu-                 2004, 5:R25. <br /> tationally reasonable principal compo-        3.      Walther D, Brunnemann R, Selbig J: regulatory code tran- <br /> nents factors analyze data using                  scriptional response diversity relation genome <br />                                                                         structural properties  thaliana. PLoS Genet 2007, 3:e11. <br /> linear regression, interpretation final result  <br />  <br />                                                                                                                                Page 11 12 <br />                                                                                                           (page number citation purposes) <br />  BMC Systems Biology 2008, 2:54                                                                   http://www.biomedcentral.com/1752-0509/2/54 <br />  <br />  <br />  <br /> 4.    Lemos B, Meiklejohn CD, Hartl DL: Regulatory evolution           24.   Bader JS, Chaudhuri  Rothberg JM, Chant J: Gaining confidence <br />       protein interaction network. Nat Genet 2004, 36:1059-60.                  high-throughput protein interaction networks. Nat Biotech- <br /> 5.    Lemos B, Bettencourt BR, Meiklejohn CD, Hartl DL: Evolution                nol 2004, 22:78-85. <br />       proteins gene expression levels coupled Dro-                 25.   Choi JK, Kim SC, Seo J, Kim S, Bhak J: Impact transcriptional <br />       sophila independently associated mRNA abun-                      properties essentiality evolutionary rate. Genetics <br />       dance, protein length, number protein-protein                          2007, 175:199-206. <br />       interactions. Mol Biol Evol 2005, 22:1345-54.                           26.   MacIsaac KD, Wang T, Gordon DB, Gifford DK, Stormo GD, Fraen- <br /> 6.    Landry CR, Lemos B, Rifkin SA, Dickinson WJ, Hartl DL: Genetic                kel E: improved map conserved regulatory sites Sac- <br />       properties influencing evolvability gene expression.                   charomyces cerevisiae. BMC Bioinformatics 2006, 7:113. <br />       Science 2007, 317:118-21.                                               27.   Hu  Killion PJ, Iyer VR: Genetic reconstruction functional <br /> 7.    Tirosh  Weinberger  Carmi M, Barkai N: genetic signature             transcriptional regulatory network.                 Nat Genet 2007, <br />       interspecies variations gene expression. Nat Genet 2006,                   39(5):683-7. <br />       38:830-4.                                                               28.   Akaike H: Information theory extension maxi- <br /> 8.    Tirosh  Barkai N: Evolution gene sequence gene                       mum likelihood principle. Proceedings Second International <br />       expression correlated yeast. Trends Genet 2008,                    Symposium Information Theory Edited  Petrov BN, Csaki F. <br />       24:109-13.                                                                    Akademiai Kiado, Budapest; 1973:267-281. <br /> 9.    Batada NN, Reguly T, Breitkreutz  Boucher L, Breitkreutz BJ, Hurst    29.   Winzeler EA, Shoemaker DD, Astromoff  Liang H, Anderson K, <br />       LD, Tyers M: Stratus altocumulus: new view yeast                 Andre B, Bangham R, Benito R, Boeke JD, Bussey H, Chu   <br />       protein interaction network. PLoS Biol 2006, 4(10):e317.                      nelly C, Davis K, Dietrich F, Dow SW, El Bakkoury M, Foury F, Friend <br /> 10.   Deng MH, Sun FZ, Chen T: Assessment reliability pro-                SH, Gentalen E, Giaever G, Hegemann JH, Jones T, Laub M, Liao H, <br />       tein-protein interactions protein function prediction.                    Liebundguth N, Lockhart DJ, Lucau-Danila  Lussier M, M&apos;Rabet N, <br />       Pac Symp Biocomput 2003:140-151.                                              Menard P, Mittmann M, Pai C, Rebischung C, Revuelta JL, Riles L, Rob- <br /> 11.   Deane CM, Salwinski L, Xenarios, Eisenberg D: Protein interac-              erts CJ, Ross-MacDonald P, Scherens B, Snyder M, Sookhai-Mahadeo <br />       tions: methods assessment reliability high                  S, Storms RK, V&#195;&#169;ronneau S, Voet M, Volckaert G, Ward TR, Wysocki <br />       throughput observations. Mol Cell Proteomics 2002, 1:349-56.                  R, Yen GS, Yu K, Zimmermann K, Philippsen P, Johnston M, Davis <br /> 12.   Mewes HW, Amid C, Arnold R, Frishman D, G&#195;&#188;ldener U, Mannhaupt                 RW: Functional characterization S. cerevisiae genome <br />       G, M&#195;&#188;nsterk&#195;&#182;tter M, Pagel P, Strack N, St&#195;&#188;mpflen V, Warfsmann J,              gene deletion parallel analysis. Science 1999, 285:901-6. <br />       Ruepp  MIPS: analysis annotation proteins                 30.   Burnham KP, Anderson DR: Model selection multimodel inference: <br />       genomes. Nucleic Acids Res 2004:D41-44.                                 practical-theoretic approach 2nd edition. Springer-Verlag; 2002. <br /> 13.   Salwinski L, Miller CS, Smith AJ, Pettit FK, Bowie JU, Eisenberg D: <br />       Database Interacting Proteins: 2004 update. Nucleic <br />       Acids Res 2004:D449-451. <br /> 14.   Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz  Tyers M: <br />       BioGRID: general repository interaction datasets. <br />       Nucleic Acids Res 2006, 34 (Database issue):D535-9. <br /> 15.   Giaever G, Chu  Ni L, Connelly C, Riles L, V&#195;&#169;ronneau S, Dow S, <br />       Lucau-Danila  Anderson K, Andr&#195;&#169; B, Arkin AP, Astromoff  El- <br />       Bakkoury M, Bangham R, Benito R, Brachat S, Campanaro S, Curtiss <br />       M, Davis K, Deutschbauer  Entian KD, Flaherty P, Foury F, Garfinkel <br />       DJ, Gerstein M, Gotte D, G&#195;&#188;ldener U, Hegemann JH, Hempel S,  <br />       man  Jaramillo DF, Kelly  Kelly SL, K&#195;&#182;tter P, LaBonte D, Lamb <br />       DC, Lan N, Liang H, Liao H, Liu L, Luo C, Lussier M, Mao R, Menard <br />       P, Ooi SL, Revuelta JL, Roberts CJ, Rose M, Ross-Macdonald P, Sche- <br />       rens B, Schimmack G, Shafer B, Shoemaker DD, Sookhai-Mahadeo S, <br />       Storms RK, Strathern JN, Valle G, Voet M, Volckaert G, Wang CY, <br />       Ward TR, Wilhelmy J, Winzeler EA, Yang Y, Yen G, Youngman E, Yu <br />       K, Bussey H, Boeke JD, Snyder M, Philippsen P, Davis RW, Johnston <br />       M: Functional profiling Saccharomyces cerevisiae <br />       genome. Nature 2002, 418:387-91. <br /> 16.   Said MR, Begley TJ, Oppenheim AV, Lauffenburger DA, Samson LD: <br />       Global network analysis phenotypic effects: protein net- <br />       works toxicity modulation Saccharomyces cerevisiae. <br />       Proc Natl Acad Sci USA 2004, 101:18006-11. <br /> 17.   Basehoar AD, Zanton SJ, Pugh BF: Identification distinct reg- <br />       ulation yeast TATA box-containing genes. Cell 2004, <br />       116:699-709. <br /> 18.   Choi JK, Kim YJ: Epigenetic regulation variability       gene expression. Nat Genet 2008, 40:141-7. <br /> 19.   Yoshimoto H, Saltsman K, Gasch AP, Li HX, Ogawa N, Botstein D, <br />       Brown PO, Cyert MS: Genome-wide analysis gene expres- <br />       sion regulated calcineurin/Crz1p signaling pathway                      Publish Bio Med Central       Saccharomyces cerevisiae. J Biol Chem 2002, 277:31079-88. <br /> 20.   Saldanha AJ, Brauer MJ, Botstein D: Nutritional homeostasis                   scientist read work free charge <br />       batch steady-state culture yeast. Mol Biol Cell 2004,                  &quot;BioMed Central significant development       15:4089-104.                                                                  disseminating results biomedical researc h lifetime.&quot; <br /> 21.   Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz <br />       G, Botstein D, Brown PO: Genomic expression programs                       Sir Paul Nurse, Cancer Research UK <br />       response yeast cells environmental changes. Mol Biol Cell                 research papers  <br />       2000, 11:4241-57. <br /> 22.   Shapira M, Segal E, Botstein D: Disruption yeast forkhead-                     available free charge entire biomedical community <br />       associated cell cycle transcription oxidative stress. Mol Biol                 peer reviewed published immediately acceptance <br />       Cell 2004, 15:5659-69. <br /> 23.   R Project Statistical Computing                [http://www.r-             cited PubMed archived PubMed Central <br />       project.org/]                                                                     &#226;&#8364;&#8221; copyright <br />  <br />                                                                                Submit manuscript                                 BioMedcentral <br />                                                                                http://www.biomedcentral.com/info/publishing_adv.asp <br />  <br />  <br />  <br />  <br />                                                                                                                                               Page 12 12 <br />                                                                                                                          (page number citation purposes) <br />   <br /> </body></html>