<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Extracytoplasmic Stress Factor, sE, Required Maintain Cell Envelope Integrity Escherichia coli <br /> Jennifer D. Hayden, Sarah E. Ades* <br /> Department Biochemistry Molecular Biology, Pennsylvania State University, University Park, Pennsylvania, United States America <br />  <br />  <br />  <br />      Abstract <br />      Extracytoplasmic function ECF sigma factors abundant class alternative sigma factors bacteria. <br />      Members rpoE subclass ECF sigma factors implicated sensing stress cell envelope Gram-negative <br />      bacteria required virulence pathogens. best-studied member family rpoE Escherichia <br />      coli, encoding sE protein. sE studied role combating extracytoplasmic stress, members <br />      regulon largely defined. sE required viability E. coli, studies <span id='am-5' about='oboInOwl:date' typeof='owl:Thing'>date</span> explain sE <br />      essential seemingly unstressed cells. work investigate essential role sE E. coli analyzing      phenotypes associated loss sE activity isolating suppressors allow cells live absence sE.      demonstrate sE inhibited, cell envelope stress increases envelope integrity lost. cells lyse      develop blebs containing cytoplasmic material sides. better understand connection      transcription sE cell envelope integrity, identified multicopy suppressors essentiality sE, ptsN      yhbW. yhbW gene unknown function, ptsN member sE regulon. Overexpression ptsN lowers      basal level multiple envelope stress responses, cytoplasmic stress response. results consistent <br />      model overexpression ptsN reduces stress cell envelope, promoting survival absence <br />      sE. <br />  <br />   Citation: Hayden JD, Ades SE (2008) Extracytoplasmic Stress Factor, sE, Required Maintain Cell Envelope Integrity Escherichia coli. PLoS 3(2): <br />   e1573. doi:10.1371/journal.pone.0001573 <br />   Editor: Steve Sandler, University Massachusetts, United States America <br />   Received December 18, 2007; Accepted January 13, 2008; Published February 6, 2008 <br />   Copyright: &#195;&#376; 2008 Hayden, Ades. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported grant MCB-0347302 National Science Foundation. funding agency did play role study   preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   *E-mail: ades@psu.edu <br />  <br />  <br />  <br /> Introduction                                                                         sE-dependent response E. coli, rpoE gene, encodes <br />                                                                                      sE, essential viability [7]. Despite wealth information <br />    organisms stress responses allow sense                  role sE response cell envelope stress respond damaging conditions altering gene expression.                       identification sE regulon, essential role sE additional level complexity introduced inducing                       unclear [8,9]. suppressor DrpoE lethality, deletion signal sensed membrane information                      ydcQ gene, identified [10].  function communicated membrane response                                                                                      gene understood clear suppresses generated. Gram-negative bacteria intercompartmental <br />                                                                                      essentiality rpoE. <br /> signaling required maintain cell envelope, consists <br />                                                                                         sE activated stresses interfere folding outer inner outer membranes, periplasmic space,                                                                                      membrane proteins (OMPs) heat shock, overexpression <span id='am-77' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-78' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-79' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span>peptidoglycan</span> layer [1]. <br />                                                                                      OMP genes, mutations genes encoding chaperones <br />    cell envelope complex, dynamic compartment                                                                                      required OMP folding [11&#226;&#8364;&#8220;14]. unstressed cells, sE activity <br /> crucial survival cell. static structure remodeled response environmental conditions.                            low sE sequestered inner membrane <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> environment cell envelope distinct                   antisigma factor RseA (Fig. 1) [15,16]. RseA single-pass inner cytoplasm. envelope lacks <span id='am-66' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-67' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>, oxidizing,                      membrane protein binds sE prevents sE subject fluctuations ionic strength passive diffusion                interacting RNA polymerase [15&#226;&#8364;&#8220;17]. envelope stress, <br /> small molecules outer membrane porins [2].                           RseA degraded response unfolded porins sequential <br /> Gram-negative bacteria possess stress responses uniquely                    action inner membrane proteases, DegS RseP, followed <br /> targeted cell envelope. stress responses include                    cytoplasmic protease ClpXP (Fig. 1) [18&#226;&#8364;&#8220;22]. periplasmic <br /> CpxAR (Cpx), BaeRS (Bae), Rcs phosphorelays, response                        protein, RseB, binds periplasmic domain RseA governed alternative sigma factor sE, phage shock                     enhances inhibition RseA, protecting proteolysis (Fig. 1) <br /> (PSP) response [3&#226;&#8364;&#8220;6]. responses activated following                 [15,16,18,23]. Continual degradation RseA required perturbation particular components envelope.                               provide cell sufficient free sE support viability,    stress responses important reacting                           deletion degS rseP toxic stabilization damaging conditions, stress proteins play important                        RseA consequent sequestration sE [19,20,23]. sE roles basic cellular physiology. particularly true                activated independently RseA-dependent stress-signaling <br />  <br />  <br />        PLoS | www.plosone.org                                                1                               February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                       sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 1. sE-dependent extracytoplasmic stress response. sE held membrane antisigma factor RseA. RseB binds periplasmic domain RseA protects RseA proteolysis. Unfolded OMPs activate protease DegS, cleaves periplasmic domain RseA. partially degraded RseA substrate RseP. RseP cleaves RseA  releasing cytoplasmic domain RseA bound sE. remaining domain RseA degraded ClpXP freeing sE interact RNA polymerase direct transcription regulon. major <br /> classes known genes sE regulon indicated. alarmone ppGpp protein DksA activate sE-dependent transcription sE released RseA, shown clarity. (IM inner membrane, OM outer membrane). <br /> doi:10.1371/journal.pone.0001573.g001 <br />  <br /> pathway cytoplasmic alarmone ppGpp, levels change               transcribe genes. Alternatively, loss response nutrient availability [24].                                   expression multiple non-essential genes dependent sE    expected role stress response, sE regulon          expression lethal.  genes sE <br /> includes genes encoding periplasmic foldases, proteases,                 regulon transcribed s70. Chromatin immoprecipitation <br /> chaperones aid OMP folding. addition, sE transcribes              followed genomic microarray (ChIP-chip) experiments exam- array biosynthetic enzymes involved phospho-               ining location s70 E. coli chromosome suggest <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipid</span>, fatty acid, <span id='am-19' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span>LPS</span>, membrane-derived oligosaccharide                 approximately 40% sE-regulated promoters overlap s70 <br /> synthesis transport, number cell envelope                 promoters [32]. number likely underestimation proteins including lipoproteins, inner membrane proteins,                extent dual regulation, does include genes envelope proteins unknown function [9,25,26]. sE regulon              s70 promoters farther away sE promoter [32]. includes genes encode major components                   sE essential genes regulon complexes responsible ushering LPS OMPs                   transcribed s70 misregulated lethal effect periplasm properly assembling outer membrane                 absence sE. <br /> [27,28]. Recently sE shown transcribe small                 work took approaches better understand RNAs decrease expression OMPs, providing                       essential role sE. approach characterize mechanism decrease flux OMPs envelope                phenotypes cells sE activity inhibited gain insights <br /> times stress [29&#226;&#8364;&#8220;31]. addition envelope-associated              cellular functions impacted. demonstrate proteins, sE-dependent promoters upstream genes                 sE required maintain integrity cell envelope; involved cytoplasmic processes transcription, transla-            absence sE cells developed blebs sides lysed. <br /> tion, DNA replication, DNA/RNA modification [9,25,26].                   second approach isolate multicopy suppressors    sE regulon contains number essential genes,              allow cells live absence sE activity. multicopy <br /> simplest explanation sE essential required       suppressors, ptsN yhbW, isolated suppress  <br />  <br />        PLoS | www.plosone.org                                        2                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                           sE Cell Envelope Integrity <br />  <br />  <br /> lethality inhibition sE activity essentiality rpoE gene. ptsN member sE regulon, yhbW  gene previously associated cell envelope <br /> functions. Overexpression ptsN lowered basal activity envelope-sensing pathways, implying suppress <br /> DrpoE lethality reducing stress cell envelope. <br />  <br /> Results <br /> Inhibition sE activity toxic increases envelope <br /> stress <br />     gain better understanding role particular gene cellular physiology, informative examine phenotypes deletion mutant lacking gene  sE encoded essential gene, possible examine phenotype DrpoE strain. examined effects loss sE activity inducing overexpression inhibitors, rseA rseB, encoded IPTG-inducible trc <br /> promoter pTrc99a plasmid, pRseAB. method <br /> efficiently inhibits sE (Fig. 2A) preventing association RNA polymerase [15&#226;&#8364;&#8220;17]. Overexpression rseA rseB consequent sequestration sE lethal bacterium. liquid culture, cfu/ml began decrease 2 hours (,2.5 <br /> generations) following addition IPTG optical <br /> density stopped increasing began decrease slightly 3 hours (,3.5 generation) (Fig. 2B [15]). solid media, plating efficiency presence IPTG reduced orders magnitude compared non-inducing conditions <br /> (Table 1). ensure observed phenotypes inhibition sE overproduction RseA RseB, introduced point mutation rseA gene plasmid <br /> encoding rseA rseB, changing aspartate residue position <br /> 11 RseA histidine. mutation abrogates antisigma <br /> factor activity RseA (Fig. 2A [16]). overproduced                Figure 2. Inhibition sE activity caused cell death. Strains <br />                                                                              SEA007 (squares, pRseAB), SEA008 (circles, pTrc99a vector control), RseB, RseAD11H protein reached similar steady-                SEA4044 (triangles, pRseAD11H) grown LB 30uC shaking <br /> state levels wild-type protein, properly localized         gyrotary waterbath. IPTG added OD600 ,0.1 induce <br /> inner membrane, did inhibit sE activity, did induce lysis,           overexpression rseA rseB, indicated arrow.   sE did reduce plating efficiency (Fig. 2, Table 1, data         inhibited overexpression rseA rseB, rseAD11H shown).                                                                  rseB, shown b-galactosidase activity measured sE- <br />     sE known respond envelope stress direct             dependent rpoHP3-lacZ reporter. Miller units (open symbols, right axis) <br />                                                                              OD600 (closed symbols, left axis) plotted time point. (B) synthesis components cell envelope, likely          Overexpression rseA rseB, rseAD11H rseB, caused compartment negatively impacted envelope               reduction colony forming units optical density. OD600 (closed <br /> stress increase sE activity blocked. measured                symbols, left axis) cfu/ml (open symbols, right axis) measured <br /> activation reporter genes Cpx, Bae, Rcs envelope              growth curve. representative experiment shown stress responses (cpxP-lacZ activated CpxR, spy-lacZ activated         panels. <br />                                                                              doi:10.1371/journal.pone.0001573.g002 <br /> BaeR CpxR, rprA-lacZ activated RcsB, Fig. 3). b- <br /> galactosidase activity envelope stress reporters increased 2.5 hours following inhibition sE (Fig. 4). activation        altered. Samples taken cultures 2.5 hours following stress responses general increase stress          overexpression rseA rseB. time point chosen associated cell death, monitored activation              analysis cell envelope stress responses activated, <br /> cytoplasmic stress response mediated s32 using s32-                   cfu/ml started decrease, morphological defects <br /> regulated htpG-lacZ reporter fusion. s32 activity increased relatively <br /> little (,2-fold) following inhibition sE compared larger          Table 1. Plating efficiencies following inhibition sE <br /> increases seen envelope stress responses (3.5-fold Cpx, <br /> 8-fold Rcs, 12-fold Bae/Cpx-dependent reporters). results suggest loss sE activity does increase stress        strain               plasmid                      plating efficiencya cell envelope, does increase cytoplasmic stress            SEA008               pTrc99a                      1.0 6 0.3 extent.                                                                  SEA007               pRseAB                       4.9 6 2.7 6 1023 <br />                                                                                                            D11H <br />                                                                               SEA4044              pRseA      B                 0.9 6 0.1 <br /> Loss sE activity does cause major changes                                                                               SEA4256              pRseAB+pPtsN                 0.9 6 0.2 <br /> composition membranes <br />                                                                               SEA4271              pRseAB+pYhbW                 0.6 6 0.3 <br />    inhibition sE increases envelope stress sE transcribe genes localized inner outer                                                                                                   plating efficiency: cfu/ml IPTG divided cfu/ml IPTG <br /> membranes, asked envelope membranes                       doi:10.1371/journal.pone.0001573.t001 <br />  <br />  <br />  <br />        PLoS | www.plosone.org                                        3                             February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                             sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 3. Diagram stress regulators respective <br /> regulon members used reporters work. reporter <br /> constructs present lacZ fusions integrated single-copy chromosome. rpoHP3-lacZ fusion regulated sE. degP-lacZ <br /> fusion regulated sE CpxR. cpxP-lacZ fusion regulated CpxR. spy-lacZ fusion regulated CpxR BaeR. rprA-lacZ <br /> fusion regulated RcsB. htpG-lacZ fusion regulated s32. sE, <br /> Cpx, Bae, Rcs monitor envelope stress, s32 monitors <br /> cytoplasmic stress. <br /> doi:10.1371/journal.pone.0001573.g003 <br />  <br />  <br /> described evident. Cell lysates fractionated centrifugation discontinuous sucrose density gradients assays used detect fractions containing inner outer membrane components. assays results cultures sE activity inhibited similar control cultures. alterations overall protein <br /> content fractions detected Coomassie-stained SDS <br /> polyacrylamide gels (Fig. 5A). inner outer membrane <br /> fractions separated, determined western blots <br /> probed antibodies inner membrane protein, FtsH,  <br />                                                                                  Figure 5. Fractionation strain SEA007 lysates                                                                                  inhibition sE sucrose density gradients.                                                                                  2.5 hours following inhibition sE, cells harvested lysates <br />                                                                                  separated discontinuous sucrose density gradient centrifugation. <br />                                                                                  Lysates comparable numbers cells  determined OD600) <br />                                                                                  loaded gradients.   Coomassie-stained 12% SDS- <br />                                                                                  polyacrylamide gels gradient fractions control culture <br />                                                                                    culture sE activity inhibited  . M denotes <br />                                                                                  molecular weight markers. (B) b-<span id='am-44' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADH</span> oxidase activity (closed symbols) <br />                                                                                  specific gravity (open symbols) fractions  , inhibition                                                                                  sE (circles) control (squares). (C) Western blots using polyclonal <br />                                                                                  antibodies detect FtsH (inner membrane) FepA (outer <br />                                                                                  membrane) fractions  , control  panels) inhibition <br />                                                                                  sE  panels). L denotes cell lysates fractionation. <br />                                                                                  doi:10.1371/journal.pone.0001573.g005 <br />  <br /> Figure 4. Envelope stress responses activated following <br /> inhibition sE. b-galactosidase activity measured htpG- <br /> lacZ (s32, SEA4183), cpxP-lacZ (Cpx, SEA4177), rprA-lacZ (Rcs, SEA4190),         outer membrane protein, FepA (Fig. 5C). Fractions containing spy-lacZ (Cpx Bae, SEA4187) reporters. fold change Miller <br /> units (M.U.) rseA rseB overexpression (M.U.                                                                                  inner membrane identified using b-NADH oxidase <br /> IPTG/M.U. ITPG) shown samples taken 0 (white), 1 (grey), 2          activity assay (Fig. 5B).  loss sE activity does result (blue), 2.5 (red), 3 (black) hours addition IPTG.                   gross defects steady-state protein composition density doi:10.1371/journal.pone.0001573.g004                                            inner outer membranes. <br />  <br />  <br />        PLoS | www.plosone.org                                            4                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                         sE Cell Envelope Integrity <br />  <br />  <br />  <br />  <br /> Figure 6. Cell envelope defects resulting inhibition sE. Cultures SEA007 SEA008 grown Fig. 2. Samples taken <br /> approximately 2.5 hrs. induction rseA rseB overexpression.  D) Images live cells shown using differential interference microscopy <br /> (DIC, column 1) fluorescence microscopy following addition FM4-64 visualize membranes (red, column 2), DAPI visualize DNA (blue, <br /> column 3), expression YFP visualize cytoplasm (green, column 4). fluorescent micrographs overlaid column 5.   <br /> Images SEA008 control strain sE inhibited shown. (B&#226;&#8364;&#8220;D) Images SEA007 following sE inhibition, reveal blebs contain YFP stain DAPI (B C) contain YFP stain DAPI (D). (D), arrow marks bleb lacking DAPI staining arrowhead marks lysed cell retained DAPI staining, lost YFP. Scale bars 2 mm. 1,000 cells examined fluorescence <br /> microscopy typical micrographs represented  Scanning electron micrographs (E) transmission electron micrographs (F) SEA007 <br /> following sE inhibition. Scale bars 1 mm. blebs seen cells control cultures. <br /> doi:10.1371/journal.pone.0001573.g006 <br />  <br />  <br /> Morphological changes following inhibition sE                              cells culture blebs (Fig. 6) number ghosted <br />    examined morphology cells following inhibition              cells increased cellular debris, indicative sE. 2 hours addition IPTG overexpress rseA              lysis. typical experiment, approximately 100 cells rseB  time cfu/ml began drop), phenotypes            viewed time point. location blebs timing <br /> evident phase contrast microscopy. Ghosted cells cells            appearance respect loss viability blebs began appear culture. blebs formed primarily              reproducible, suggesting phenotypes related. 3&#226;&#8364;&#8220; lateral wall cells frequently         4 hours addition IPTG cells additional blebs, poles septum. Usually bleb formed              number ghosted cells cell debris increased, <br /> cell. 2.5 hours addition IPTG, approximately 20%               cells clumped  Similar phenotypes  <br />  <br />        PLoS | www.plosone.org                                         5                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                            sE Cell Envelope Integrity <br />  <br />  <br /> observed sE inhibited proteolytic stabilization                 Table 2. Cotransduction nadB::Tn10 rpoE::kan <br /> RseA depletion DegS RseP proteases (data shown) following overexpression rseAD11H rseB pRseAD11HB, providing evidence                  recipient               plasmida                nadB::Tn10b     rpoE::kan   c <br />  <br /> phenotypes inhibition sE overexpression rseA rseB. addition, phenotypes seen                 SEA001 (wild type)      _                       159             1 cells grown glucose glycerol minimal media                      CAG43113 (sup+)         _                       184             184 <br /> supplemented amino acids indicating phenotypes                   SEA4270                 pYhbW                   48              48 function growth rate medium composition.                      SEA4254                 pPtsN                   153             153 <br />    probe morphology cells following inhibition            SEA4106                 pPtsNH73A               24              0 <br /> sE, fluorescent probes used detect cytoplasm, nucleic <br />                                                                                SEA4110                 pPtsNH73A/H120A         24              0 <br /> acids, membranes live cells. visualized cytoplasm expressing soluble fluorescent protein, YFP,                       SEA4230 (DtrkA)         _                       11              0 <br /> expression construct integrated chromosome. Membranes                   SEA4234 (DtrkA)         pPtsN                   23              23 nucleic acids visualized respectively FM4-64                 SEA4144 (DptsP)         _                       20              0 DAPI fluorescent dyes, added directly                    SEA4156 (DptsP)         pPtsN                   12              12 <br /> cultures. dyes cytoplasmic expression YFP <br />                                                                                SEA4143 (DptsO)         _                       12              0 <br /> altered phenotypes associated inhibition sE. cells blebs outlined FM4-64, cells              SEA4155 (DptsO)         pPtsN                   15              15 blebs contained YFP FM4-64 boundary,                   SEA4131 (DptsN)         _                       24              0 <br /> exception ghosted cells (Fig. 6A&#226;&#8364;&#8220;D). ghosted cells                                                                                              &#226;&#8364;&#732;&#226;&#8364;&#732;_&#226;&#8364;&#8482;&#226;&#8364;&#8482; denotes plasmid <br /> outlined FM4-64, longer contained YFP (Fig. 6D,                    b <br />                                                                                  number colonies isolated P1 transduction selecting nadB::Tn10 <br /> indicated arrowhead). YFP proteolytically labile,                c <br />                                                                                 number nadB::Tn10 transductants shown rpoE::kan lack YFP signal ghosted cells likely        doi:10.1371/journal.pone.0001573.t002 release cytoplasmic contents lysis. cells DAPI staining, compaction nucleoids number nucleoids cell did differ significantly            encoded ASKA plasmid. approximately 10,000 <br /> control cells, suggesting loss sE activity does cause             transformants, 44 putative suppressors recovered. <br /> major defects chromosome replication segregation (Fig. 6A&#226;&#8364;&#8220;                 classes genes isolated selection, D). Approximately 50% cells blebs contained DAPI                  encoding genes restore sE activity longer <br /> staining bleb, addition YFP (Fig. 6B,C).                  require sE growth. distinguished classes <br /> hypothesize blebs contain DAPI staining (Fig. 6D,            using chromosomally encoded sE-dependent lacZ reporter <br /> indicated arrow), nucleoid simply               gene (rpoHP3-lacZ [36]) strain SEA007 detected sE activity drawn blebs, nucleic acids              plating presence Xgal IPTG. 44 clones <br /> present visualized DAPI.                                           isolated selection, 26 formed blue colonies, indicating    Electron microscopy employed determine                overexpression gene plasmid restored sE activity, additional changes cell morphology detectable                 18 formed white colonies, indicating sE activity light microscopy. Scanning electron micrographs revealed                   restored longer required growth. significant alterations cells blebs ghosted cells         class contained plasmids potential suppressors (Fig. 6E). surface cells blebs ghosted cells         requirement sE viability. 18 plasmids, <br /> similar cells control cultures overexpres-           verified suppression linked plasmid, plating <br /> sion rseA rseB. Using transmission electron microscopy                 efficiencies rseA rseB overexpressed longer <br /> (TEM) obtained images containing sections                reduced, morphological defects significantly reduced. <br /> bleb. images, inner outer membranes                    selection identified plasmids permit growth clearly traced bleb, cytoplasmic material                  presence overexpression rseA rseB.  present bleb (Fig. 6F). evidence outer membrane                  strains low levels sE activity contribute vesiculation, associated alterations sE                viability. determined rpoE gene activity [33,34], gross morphological defects                   deleted potential suppressor genes overexpressed <br /> observed TEM SEM images.                                            using cotransduction assay. nadB::Tn10 allele immediately <br />                                                                               upstream rpoE::kan deletion introduced P1 transduction <br /> Isolation multicopy suppressors requirement                      wild-type strains containing 18 plasmids, sE activity viability                                                     nadB::Tn10 transductants selected. genes    phenotypic studies demonstrate sE activity                tightly linked, rpoE::kan allele cotransduced unless rpoE depleted, cell envelope integrity compromised, resulting lysis          essential (Table 2). rpoE deletion cotransduced strains bleb formation.  phenotypes readily                 carrying 2 18 plasmids (Table 2). remainder explained loss expression gene(s) sE regulon          strains, rpoE deletion cotransduced, indicating Discussion). gain additional information sE                   rpoE gene required viability. multicopy <br /> essential, turned genetics isolated multicopy suppressors           suppressors rpoE essentiality yhbW ptsN (Tables 1 allowed E. coli live absence sE activity. Cells             2). yhbW encodes putative luciferase-like monooxygenase containing pRseAB (strain SEA007) transformed pools                 unknown function. ptsN encodes EIIANtr, protein related plasmids ASKA ORF library, E. coli                  enzyme IIA components PTS <span id='am-12' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>carbohydrate</span> transport systems <br /> ORF cloned control IPTG-inducible                         [37].  EIIANtr known associated promoter [35]. Transformants plated presence                   known transporters E. coli Ntr phosphoryl relay IPTG induce overexpression rseA rseB gene                   instead thought play role signaling [37&#226;&#8364;&#8220;39]. ptsN  <br />  <br />        PLoS | www.plosone.org                                         6                                February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                        sE Cell Envelope Integrity <br />  <br />  <br /> member rpoE regulon [9] focused studies. <br />  <br /> Phosphorylation EIIANtr required suppression <br />     EIIANtr phosphoryl relay phosphorylation state EIIANtr important activity <br /> [40,41], determined phosphorylation EIIANtr important suppression. EIIANtr phosphorylated His73 possibly His120 [41]. constructed variants, ptsNH73A ptsNH73A/H120A, tested ability suppress requirement sE viability cotransduction assay. rpoE::kan allele cotransduced nadB::Tn10 <br /> (Table 2) suggesting phosphorylation EIIANtr required survival absence sE. Overexpression EIIANtr <br /> variants detected Coomassie-stained protein gels                   Figure 7. Overexpression ptsN reduces cell envelope stress. <br /> indicating lack suppression owed instability             b-galactosidase activity rpoHP3-lacZ (sE, SEA4254), degP-lacZ <br />                                                                                (sE Cpx, SEA4181), cpxP-lacZ (Cpx, SEA4179), rprA-lacZ (Rcs, variant proteins (data shown). <br />                                                                                SEA4199), spy-lacZ (Cpx Bae, SEA4189), htpG-lacZ (s32, <br />     ptsP ptsO genes encode EINtr NPr, respectively,            SEA4185) fusions measured overnight cultures, form phosphoryl relay transfers phosphate                        overexpression ptsN. Activity lacZ fusion normalized phosphoenol pyruvate (PEP) EINtr NPr EIIANtr [37,39].                 cultures ptsN overexpression set 100%. Average phosphorylation EIIANtr required suppression,              values standard deviations minimum 3 experiments asked components phosphoryl relay                shown. <br />                                                                                doi:10.1371/journal.pone.0001573.g007 required determining overexpression ptsN suppress essentiality rpoE strains lacking ptsP ptsO. genes required suppression overexpressed                 cytoplasmic stress (Fig. 3). Overexpression ptsN significantly <br /> ptsN; nadB::Tn10 rpoE::kan cotransductants obtained               lowered expression lacZ fusions regulated strains (Table 2). Additionally, rpoE gene deleted            envelope stress response factors, effect s32- DptsN, DptsO, DptsP strains overexpression ptsN.              dependent cytoplasmic stress reporter (Fig. 7). result results point mutations ptsN indicate               indicates overexpression ptsN does lower b-galactosi- <br /> phosphorylation EIIANtr required suppression, likely           dase activity se, does dampening effect overall paralogous proteins phosphoryl relays                    gene expression. <br /> compensate loss Ntr components phosphorylate                    ptsN overexpression reduce basal levels envelope <br /> EIIANtr absence.                                                      stress activate stress signaling pathways trigger     PtsN shown bind inhibit potassium uptake             cytoplasmic signal downregulates effectors signaling potassium transporter TrkA [40]. K+ major cation             pathways independently cell envelope stress. idea E. coli cytoplasm participates variety processes related           correct, ptsN lower reporter activity independently known functions sE including adaptation osmotic stress                envelope signal transduction pathways. strain lacking maintenance turgor pressure [42&#226;&#8364;&#8220;44]. addition, K+ binds              antisigma factor RseA, sE activity high unresponsive regulates number intracellular enzymes including RNA                  known regulatory signals originate cell envelope, polymerase  potassium glutamate) [45].  overex-                responsive levels cytoplasmic signaling factor, ppGpp <br /> pression ptsN suppresses requirement sE viability             [24]. Overexpression ptsN did lower activity rpoHP3- <br /> inhibiting TrkA, rpoE essential DtrkA strain.               lacZ reporter DrseA strain, suggesting signal transduction  DtrkA strain used recipient                  RseA required ptsN lower sE activity (Fig. 8A). <br /> cotransduction assay, nadB::Tn10 transductants ob-                       ptsN overexpression does fact lower basal level tained indicating DtrkA suppressor (Table 2).                    envelope stress, decrease reporter gene expression <br /> Furthermore, trkA gene required suppression                 dependent transcriptional activators associated <br /> ptsN, cotransductants readily obtained DtrkA strain               envelope stress pathways. Interestingly, DydcQ ptsN overexpressed (Table 2).                                         strain decrease reporter gene expression Cpx- <br />                                                                                regulated cpxR-lacZ reporter dependent response <br /> Overexpression ptsN lowers envelope stress                                  regulator CpxR [10]. measured effects overexpression    basal levels envelope stress responses lower          ptsN Cpx-dependent cpxP-lacZ Rcs-dependent rprA-lacZ DydcQ strain, characterized suppressor DrpoE lethality             reporter fusions strains lacking respective transcriptional <br /> [10]. addition, sE-dependent rpoHP3-lacZ activity low               regulators, CpxR RcsB. WT strain, overproduction strain containing unmapped DrpoE suppressor [46]. Consistent                ptsN lowered expression rprA-lacZ reporter 3.2-fold results, overexpression ptsN lowered activity             compared 1.8-fold DrcsB strain (Fig. 8B) lowered <br /> sE-dependent rpoHP3-lacZ reporter wild-type rpoE+ strain                  expression cpxP-lacZ reporter 4.3-fold compared 2.2- <br /> (Fig. 7 8A). determine effect specific              fold cpxR::Vspec strain (Fig. 8C). reported <br /> sE, ptsN overexpression affected envelope stress                   previously cpxP-lacZ reporter expressed DcpxR <br /> responses, measured effects ptsN overexpression                   variant E. coli strain MC4100 [10,47].  reporters following envelope stress responses (Fig. 3): Cpx            able reproducibly measure b-galactosidase activity (cpxP-lacZ reporter), sE Cpx (degP-lacZ reporter), Bae Cpx             reporter cpxR::Vspec variant strain MG1655, (spy-lacZ reporter), Rcs (rprA-lacZ reporter). strain carrying         expression quite low (,250-fold lower WT <br /> reporter s32-dependent cytoplasmic heat shock response                 background). results indicate decreased activity (htpG-lacZ) included assay ptsN lowered                    reporters ptsN overexpression partially dependent <br />  <br />  <br />        PLoS | www.plosone.org                                          7                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                         sE Cell Envelope Integrity <br />  <br />  <br />  <br />                                                                              r <br />  <br />                                                                              fusion WT DrseA strains (SEA4254 SEA4228), (B) rprA-lacZ <br />                                                                              fusion WT DrcsB strains (SEA4199 SEA4203), (C) cpxP- <br />                                                                              lacZ fusion WT cpxR::Vspec strains (SEA 4179 SEA4251).                                                                              inset C shows cpxR::Vspec strain ptsN <br />                                                                              overexpression. basal level expression .250-fold lower                                                                              cpxR::Vspec cells WT cells easily seen main plot. <br />                                                                              doi:10.1371/journal.pone.0001573.g008 <br />  <br />                                                                              cpxR rcsB, suggesting EIIANtr lowers envelope stress <br />                                                                              sensed pathways. <br />  <br />                                                                              Discussion <br />                                                                                 Despite extensive knowledge regulatory pathways                                                                              control sE activity, stresses activate pathways,                                                                              genes sE transcribes, essential role sE cell                                                                              remained elusive. work demonstrate sE required <br />                                                                              maintenance cell envelope integrity. sE activity                                                                              inhibited, envelope stress sensed stress responses <br />                                                                              increased, indicating cell envelope impacted.                                                                              cells lysed developed blebs containing cytoplasmic <br />                                                                              material sides. Interestingly, phenotypes                                                                              accompanied major alterations composition                                                                              envelope membranes. identified multicopy suppressors, <br />                                                                              ptsN yhbW, allowed cells live absence rpoE <br />                                                                              gene. ptsN member sE regulon [9] lowers basal <br />                                                                              level activity envelope stress responses unknown <br />                                                                              mechanism overexpressed. <br />  <br />                                                                              does loss sE activity lead bleb formation                                                                              lysis? <br />                                                                                 blebs form following inhibition sE resemble bulges <br />                                                                              formed E. coli treated penicillin-like antibiotics                                                                              inhibit peptidoglycan synthesis, inactive variant                                                                              peptidoglycan synthase PBP 1B overproduced [48,49].                                                                              treatments lytic, lysis thought triggered holes <br />                                                                              formed peptidoglycan layer <span id='am-88' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-92' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="PGN" datatype="xsd:string"></span>murein</span> hydrolases acting                                                                              absence peptidoglycan synthesis [48,49]. similarity                                                                              phenotypes suggests inhibition sE leads weakening                                                                              defects peptidoglycan layer. defects potentially <br />                                                                              major outcomes. hole forms peptidoglycan <br />                                                                              large accommodate extrusion inner <br />                                                                              membrane cytoplasm, evidenced YFP signal                                                                              blebs, cell lyses, evidenced loss YFP signal ghosted <br />                                                                              cells. observation blebs form 20% cells                                                                              likely inherent stochastic nature process. <br />                                                                              cells lyse bleb form. support                                                                              model inhibition sE leads defects peptidoglycan <br />                                                                              layer comes preliminary results strain lacking                                                                              enzymes involved peptidoglycan synthesis, PBPs 4, 5, 6, 7, <br />                                                                              results irregular peptidoglycan synthesis [50]. strain <br />                                                                              develops blebs lyses quickly following inhibition sE <br />                                                                              isogenic WT strain (data shown). peptidoglycan                                                                              affected, able reduce cell death growing cells <br />                                                                              high osmolarity medium. Unfortunately, experiment                                                                              technically feasible. High osmolarity induced sE response Figure 8. ptsN requires known regulators reduce stress                    sE longer inhibited RseA, RseA sensed sE, Rcs, Cpx envelope stress pathways.                  overexpressed (data shown). panel b-galactosidase activity measured           phenotypes consistent model <br /> overexpression ptsN WT strain strain lacking known <br /> regulators respective stress responses. Measurements                                                                              loss sE activity leads alterations peptidoglycan <br /> expressed percentage reporter gene activity WT strain       layer resulting bleb formation lysis, eliminate ptsN overexpression value noted columns.       possibility bleb formation caused different event, Relative reporter gene activities shown   rpoHP3-lacZ          aberrant cell division. Future experiments examining  <br />  <br />        PLoS | www.plosone.org                                        8                            February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                        sE Cell Envelope Integrity <br />  <br />  <br /> integrity peptidoglycan layer                  suppression ptsN, DtrkA suppressor, indicating localization cell division proteins following inhibition sE            EIIANtr additional functions, discovered, help distinguish possibilities.                         responsible suppression DrpoE lethality. discussed  <br />                                                                              results indicate novel activity likely involves regulation <br /> Relationship phenotypes known                            cell envelope stress responses. <br /> functions sE regulon members <br />    reported function sE direct                     Lowered envelope stress responses, characteristic sE <br /> transcription, altered regulation genes             suppressors sE regulon ultimately lead bleb formation loss             basal level activity multiple envelope stress responses viability sE activity blocked.  given known              reduced presence sE suppressors data functions sE regulon members, point specific             available, overexpression ptsN, DydcQ [10] uncharacter- <br /> gene set genes regulon obviously associated        ized suppressor [46], suggesting lower activity envelope phenotypes. Depletion phenotypes characterized               stress responses important suppression essential sE-regulated genes (yaeT, yfiO, <span id='am-109' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="IMP" datatype="xsd:string"></span>imp</span>, lptA       mechanism. Preliminary results indicate basal levels lptB), encode components complexes assemble                cell envelope stress responses lower following overexpres- <br /> LPS OMPs outer membrane [28,51&#226;&#8364;&#8220;54]. Depletion                   sion yhbW, multicopy suppressor isolated. genes causes striking defects cell envelope,            light results, model suppression observed following inhibition sE [28,51&#226;&#8364;&#8220;55].                        suppressors strengthen cell envelope loss sE <br />    blebbing phenotype suggests peptidoglycan cell                activity resultant impact cell envelope longer <br /> division affected loss sE. sE does transcribe          lethal. model correct, suppressors lower known murein hydrolases biosynthetic enzymes directly                 envelope stress sensed signal transduction pathways involved peptidoglycan synthesis, does transcribe bacA,          monitor cell envelope. effects ptsN <br /> encodes undecaprenyl <span id='am-39' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span>pyrophosphate</span> phosphatase [9,56].                    overexpression reporter gene expression Rcs Cpx <br /> BacA hydrolyzes phosphate moiety undecaprenyl pyro-                   pathways partially dependent respective <br /> phosphate generate undecaprenyl phosphate, lipid               stress regulators. addition, sE activity altered ptsN <br /> carrier required synthesis peptidoglycan cell wall          overexpression strain lacking rseA, known envelope <br /> polymers [56].  E. coli possesses additional          sensor pathway. possible EIIANtr undecaprenyl pyrophosphate phosphatase enzymes cells                     unable overcome high level activation sE DrseA <br /> lacking bacA obvious morphological phenotypes                  strain background, think unlikely based observations [56]. sE-dependent promoter upstream cell             regulator sE, does act RseA. sE <br /> division protein FtsZ [9], promoter multiple              activity activated reduced DrseA strain promoters contribute complex regulation ftsZ                  response cytoplasmic regulator ppGpp [24], suggesting expression [57], making unlikely loss sE activity           effects ptsN overexpression discernible. large effect non-stressed growth conditions.                    contrast, reduction activity Cpx reporter gene <br />    direct connection convincingly                   examined DydcQ strain dependent cpxR [10]. sE regulon members observed phenotypes,                    epistasis experiments reported think likely phenotypes caused indirect effects           reporter, known activity cell <br /> resulting loss altered regulation sE-dependent genes.          envelope stress responses reduced DydcQ background example, sE transcribes genes required synthesis              independently respective signal transduction pathways <br /> lipid component LPS, lpxA, lpxB, lpxD [9]. LpxA catalyzes           [10]. step lipid synthesis uses substrate, <span id='am-114' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-115' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-118' property="rdfs:label" content="UDP" datatype="xsd:string"></span>UDP</span>- <br /> N-acetyl-<span id='am-56' about='obo:IMR_0200485' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00329" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200485" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="D-Glucosamine" datatype="xsd:string"></span>D-glucosamine</span>, used dedicated step            sE essential E. coli? <br /> peptidoglycan biosynthesis [58]. Changes lpxA lpxD                       functions core members sE regulon conserved <br /> expression absence sE alter flux substrates           number bacterial species [9].  sE essential pathways, affecting peptidoglycan synthesis            bacteria including closely related species integrity peptidoglycan layer.                                        Salmonella typhimurium [59]. observation led,                                                                               model proposing sE essential required Suppressors DrpoE lethality                                               transcribe particular gene set genes [10]. Instead, sE    work genes identified overexpression               essential bacteria overreact absence  suppresses loss viability following inhibition sE         unknown mechanism requires ydcQ gene product, requirement rpoE gene viability. known           activate unknown cell death pathway kills cells [10]. <br /> suppressors DrpoE lethality (DydcQ [10] overexpression             data consistent model.  suggest ptsN yhbW), gene known roles cell            functions core regulon members conserved <br /> ptsN. ptsN member sE regulon, suggests             bacterial species sE essential [9], significant <br /> plays role cell envelope processes [9]. functions          differences exist properties cell envelopes associated nitrogen PTS  ptsN             different species strains species member, including balancing nitrogen carbon metabolism,                  determine bacteria survive absence sE. <br /> suppression temperature-sensitive allele essential               example, majority work sE, including GTPase Era DptsN mutant, inhibition potassium                presented paper, performed K12 strains E. <br /> transporter TrkA [37,38,40]. role ptsN held          coli, lack O-antigen LPS [60]. variation promise providing insights mechanism suppression             significantly alter properties bacterial cell envelope, K+ important osmotic regulation regulate              making bacteria sensitive disruptions cell envelope <br /> RNA polymerase, activities associated sE [42,44,45].                integrity sE essential.  propose  demonstrated trkA required                       sE essential required transcribe genes  <br />  <br />        PLoS | www.plosone.org                                        9                           February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                       sE Cell Envelope Integrity <br />  <br />  regulon. absence sE genes properly                2% osmium tetroxide, 0.1% uranyl acetate acetone 290uC <br /> regulated, leading disruption cell envelope integrity lysis.        3 days. Samples slowly warmed 260uC 14 hours, <br /> Future experiments delving greater depth phenotypes            230uC 14 hours, 0uC 1 hour. Samples cells lacking sE, roles sE regulon members unknown                  transferred acetone, brought room temperature, function, mechanism suppression help                     infiltrated resin. Ultrathin sections obtained using distinguish models provide new insights            Reichart-Jung Ultracut E microtome diamond knife. <br /> role important cell envelope stress response.                         Sections mounted hexagonal mesh open slot copper <br />                                                                               grids stained 2% uranyl acetate 50% ethanol Materials Methods                                                         5 minutes followed Reynold&#226;&#8364;&#8482;s lead stain 2 minutes                                                                               washed water. drying, grids visualized Media, strains, plasmids <br />                                                                               JEOL JEM 1200 EXII transmission electron microscope. Images <br />    Strains derivatives MG1655, unless noted,                                                                               taken digital TEITZ camera. listed Table 3. Cultures grown Luria Bertani (LB) <br /> broth 30uC shaking gyrotary water bath. Antibiotics used following concentrations: ampicillin (<span id='am-49' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>)                   Fluorescence microscopy <br /> 100 mg/ml, kanamycin (kan) 30 mg/ml, tetracyline (tet) 10 mg/                     fluorescence microscopy, location cytoplasm ml, spectinomycin (spec) 50 mg/ml, chloramphenicol (cam)                  visualized strains SEA4287 SEA4288, express soluble <br /> 20 mg/ml. Altered alleles stress response reporters              YFP cytoplasm, periplasm. Overnight <br /> transferred appropriate recipient strains transduction            cultures grown LB/Amp subcultured media containing P1vir according standard methods [61], exception             1 ng/ml anhydrous tetracycline, induces expression YFP. sodium citrate used toxic cells             IPTG added cells reached OD600 0.1&#226;&#8364;&#8220;0.2                                                                               samples taken microscopy 1.5 3.5 hours later. compromised membranes. rseAD11H mutation                                                                               FM4-64 (Molecular Probes T-3166) 4,6-diaminidino-2-pheny- pRseAB plasmid ptsN H73A H73A/ <br />                                                                               lindole, DAPI, added directly samples respective final <br /> H12A mutations pPtsN plasmid using QuikChange site- <br />                                                                               concentrations 5 mg/ml 1 mg/mL. Cells mounted directed mutagenesis kit (Stratagene) verified sequencing. <br />                                                                               freshly prepared polylysine coated coverslips visualized using ptsN::kan, ptsO::kan, ptsP::kan, trkA::kan alleles                                                                               Olympus BX-61 epi-fluorescent microscope 1006 phase oil <br /> obtained Keio collection [62] moved P1 <br />                                                                               objective (NA 1.3), using Olympus 41028 filter cube YFP, <br /> transduction SEA001. kan allele removed                                                                               Olympus 31000v2 filter cube DAPI, Olympus 41001 <br /> FLP recombinase method Datsenko Wanner [63]                                                                               filter cube FM4-64. Images captured Hamamatsu <br /> create strains SEA4131, SEA4143, SEA4144, SEA4230. <br />                                                                               Orca-ER camera using Slidebook 4.1 software (Compix, ). <br />                                                                               Deconvolution performed using AutoQuant software <br /> b-galactosidase assays                                                        package (Media Cybernetics). Generally, 10 iterations <br />   b-galactosidase assays performed largely described [24].            necessary deconvolve images. <br /> Overnight cultures diluted OD600 0.02 grown aeration 30uC. rseA rseB overexpression induced             Sucrose density gradient fractionation <br /> addition IPTG 1 mM cultures reached OD600                       Cultures SEA007 grown OD600 0.1 0.1&#226;&#8364;&#8220;0.2 experiments shown Figs. 2 4. Figs. 7 8, b-          point IPTG added 1 mM culture induce rseA galactosidase activity determined cultures grown 12&#226;&#8364;&#8220;             rseB overexpression. 2.5 hours growth, cells 14 hours 1mM IPTG. Experiments                          harvested centrifugation, resuspended 10 mM HEPES <br /> exponential phase cultures yielded similar results. minimum              pH 7.8, lysed French press. Discontinuous sucrose density independent experiments performed strain                  gradients prepared layering 860 ml 50%, 45%, 40%, condition.                                                                35%, 30% sucrose solutions 200 ml 55% sucrose <br />                                                                               cushion. Lysates equal numbers cells treated Electron microscopy                                                           control cultures loaded gradients, based    Cells SEA007 SEA008 used electron                         OD600 cultures time sampling. centrifugation <br /> microscopy grown LB OD600 0.1&#226;&#8364;&#8220;0.2,                  50,000 rpm 15 hours, 150 ml fractions collected point IPTG added induce rseA rseB overexpression                gradients. specific gravity fraction SEA007. Cells harvested 2.5 hours later centrifugation.               determined measuring refractive index. b-NADH oxidase <br /> Samples scanning electron microscopy (SEM) subjected                 activity fraction determined previously reported primary fixation step 6.25% glutaraldehyde Sorenson&#226;&#8364;&#8482;s              [64]. Total protein content analyzed 12% SDS- <br /> Buffer 1.5 hours room temperature washed 0.1 M                 polyacrylamide gels Coomassie blue staining. Western <br /> cacodylate buffer. secondary fixation step performed                 blotting performed polyclonal antibodies outer <br /> treating samples 2% osmium tetroxide 0.1 M                        membrane protein FepA (gift Kathleen Postle) inner <br /> cacodylate buffer, followed washing dehydration              membrane protein FtsH (gift Christophe Herman) detected ethanol. Samples dried, sputter coated imaged                horseradish peroxidase-conjugated donkey anti-rabbit IgG <br /> JEOL JSM 5400 scanning electron microscope PGT Prism                   (GE Healthcare) using ECL Plus kit (GE Healthcare). <br /> light element detector. <br />    Samples transmission electron microscopy (TEM)                    Genetic selections <br /> prepared cryo-fixation freeze substitution. samples                   genetic selection carried identify genes washed 0.1 M cacodylate buffer incubated 18%                   overexpression allowed cells live sE activity inhibited. <br /> glycerol (v/v) 0.1 M cacodylate 20 minutes room                     Strain SEA007 (carrying pRseAB) transformed electropo- <br /> temperature cryo-protection. sample pelleted                   ration 5 pools plasmids ASKA plasmid library <br /> centrifugation 10 ml pellet blotted filter paper          [35], cells plated LB agar 1 mM IPTG, Amp frozen liquid nitrogen. Freeze substitution carried             select pRseAB), Cam  select library plasmids). <br />  <br />  <br />        PLoS | www.plosone.org                                        10                          February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                   sE Cell Envelope Integrity <br />  <br />  <br />  <br /> Table 3. Strains Plasmids <br />  <br />  <br /> Strain                                  Genotype                                                 Reference <br />  <br /> CAG43113                                MC1061 sup+, CAG41001                            [20] <br /> CAG45146                                MG1655 DlacX74 Wl[htpG P1-lacZ]                          [65] <br /> DH300                                   MG1655 D(argF-lac)U169; rprA142-lacZ                     [66] <br /> SEA001                                  MG1655 DlacX74 Wl[rpoHP3-lacZ]                           [67] <br /> SEA007                                  SEA001 pRseAB                                            [24] <br /> SEA008                                  SEA001 pTrc99a                                           work <br /> SEA2000                                 SEA001 nadB::Tn10 DrseA                                  [24] <br /> SEA4041                                 CAG43113 rpoE::kan                                       work <br />                                                          D11H <br /> SEA4044                                 SEA001 pRseA        B                                    work <br /> SEA4106                                 SEA001 pPtsNH73A                                         work <br /> SEA4110                                 SEA001 pPtsNH73A/H120A                                   work <br /> SEA4114                                 SEA4041 nadB::Tn10                                       work <br /> SEA4131                                 SEA001 DptsN                                             work <br /> SEA4143                                 SEA001 DptsO                                             work <br /> SEA4144                                 SEA001 DptsP                                             work <br /> SEA4155                                 SEA4143 pPtsN                                            work <br /> SEA4156                                 SEA4144 pPtsN                                            work <br /> SEA4177                                 MG1655 DlacX74 lRS88[cpxP-lacZ] pRseAB                   work, reporter [47] <br /> SEA4179                                 MG1655 DlacX74 lRS88[cpxP-lacZ] pPtsN                    work, reporter [47] <br /> SEA4181                                 MG1655 DlacX74 lRS88[degP-lacZ] pPtsN                    work, reporter [68] <br /> SEA4183                                 CAG45146 pRseAB                                          work <br /> SEA4185                                 CAG45146 pPtsN                                           work <br /> SEA4187                                 MG1655 DlacX74 lRS88[spy-lacZ] pRseAB                    work, reporter [69] <br /> SEA4189                                 MG1655 DlacX74 lRS88[spy-lacZ] pPtsN                     work, reporter [69] <br /> SEA4190                                 DH300 pRseAB                                             work <br /> SEA4199                                 DH300 pPtsN                                              work <br /> SEA4203                                 4199 rcsB::kan                                           work, rcsB::kan [66] <br /> SEA4204                                 SEA4181 nadB::Tn10 DrseA pPtsN                           work <br /> SEA4228                                 SEA2000 pPtsN                                            work <br /> SEA4230                                 SEA001 DtrkA                                             work <br /> SEA4234                                 SEA4230 pPtsN                                            work <br /> SEA4251                                 SEA4179 cpxR::Vspec                                      work, cpxR::Vspec [68] <br /> SEA4254                                 SEA001 pPtsN                                             work <br /> SEA4256                                 SEA007 pPtsN                                             work <br /> SEA4270                                 SEA001 pYhbW                                             work <br /> SEA4271                                 SEA007 pYhbW                                             work <br /> SEA4287                                 SEA007 D(xylAxylB)::tetR tetA-yfp                        work, yfp allele Goulian <br />                                                                                                  lab collection <br /> SEA4288                                 SEA008 D(xylAxylB)::tetR tetA-yfp                        work, yfp allele Goulian <br />                                                                                                  lab collection <br /> Plasmids <br /> pTrc99a                                 vector, pBR322 ori AmpR                                  Pharmacia <br /> pRseAB                                  rseA rseB pTrc99a, pLC253                 [15] <br /> pRseAD11HB                              RseAD11H pRseAB                                       work <br /> pPtsN                                   ptsN pCA24N, ASKA plasmid , CamR                      work [35] <br /> pPtsNH73A                               PtsNH73A pPtsN                                        work <br /> pPtsNH73A/H120A                         PtsNH73A/H120A pPtsN                                  work <br /> pYhbW                                   yhbW pCA24N, ASKA plasmid , Camr                      work [35] <br />  <br /> doi:10.1371/journal.pone.0001573.t003 <br />  <br />  <br />  <br />  <br />         PLoS | www.plosone.org                               11                February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                                                    sE Cell Envelope Integrity <br />  <br />  ASKA plasmid library  GFP tag) used                                    restoring sE activity transformed SEA001. studies consists plasmids containing E. coli ORF&#226;&#8364;&#8482;s cloned                              resulting strains transduced P1 lysate SEA4114. control IPTG-inducible promoter [35]. Colonies                                   Colonies plated LB/Tet/IPTG select grew IPTG streaked LB/IPTG/Amp/Cam                                          nadB::Tn10 overexpress putative suppressor gene confirm growth. Plasmids isolated, library plasmids                               plasmid. Transductants streaked LB/Kan/IPTG <br /> purified transformation DH5a selection LB/Cam.                                  determine rpoE deleted. PCR used verify selected ASKA plasmids transformed                                      presence nadB::Tn10 rpoE::kan alleles absence SEA007 verify conferred growth IPTG. Cells                                 rpoE gene. Deletion rpoE gene verified grown plates containing Xgal determine sE                                    western blotting polyclonal antibodies specific sE. activity restored (blue colonies) inhibited (white                             cotransduction experiment, SEA001 CAG43113 used colonies). selected genes encoded ASKA plasmids                                 negative positive controls, respectively. <br /> identified sequencing. <br />                                                                                                 Acknowledgments <br /> Determination essentiality rpoE <br />    nadB::Tn10 rpoE::kan linked marker construct used                                     thank Susan Gottesman, Tracy Raivio, Carol Gross, Mark Goulian, <br />                                                                                                 Hirotada Mori sharing strains; Mary Laubacher making strains; <br /> cotransduction assays strain contains                                                                                                 Kathleen Postle, Marta Manning, Charles Bulathsinghala unidentified suppressor DrpoE (sup, CAG43113). rpoE gene                                 assistance sucrose gradients FepA antibody. fluorescence deleted  using method Datsenko Wanner [63]                                 microscopy Cytometry Facility, University Park (Huck generate strain SEA4041. nadB::Tn10 insertion                                       Institutes Life Sciences, Penn State University) facility SEA2000 moved rpoE::kan strain P1                                          funded,  grant Pennsylvania Department Health <br /> transduction, selecting resistance Kan Tet generate                               using Tobacco Settlement Funds. Department specifically disclaims <br /> strain SEA4114. Transductants verified PCR ensure                                    responsibility analyses, interpretations, conclusions. electron kan allele present rpoE gene absent.                               microscopy Electron Microscopy Facility Huck <br />                                                                                                 Institutes Life Sciences, Penn State University. <br /> essentiality rpoE determined assessing cotransduction <br /> frequency nadB::Tn10 rpoE::kan recipient strains. markers tightly linked, nadB::Tn10 transduced                                  Author Contributions <br /> independently strains rpoE essential. Plasmids                              Conceived designed experiments: SA JH. Performed suppressed lethality overexpression rseA rseB                                experiments: JH. Analyzed data: SA JH. Wrote paper: SA JH. <br />  <br /> References <br />  1. Kellenberger E (1996) Structure Function Subcellular Level.                         factor activity RseA, RseB RseC proteins. Mol Microbiol 24: <br />     Neidhardt FC, Curtiss R III, Ingraham JL, Lin ECC, Low KB, Magasanik B,                           355&#226;&#8364;&#8220;371. <br />     Reznikoff WS, Riley M, Schaechter M, Umbarger  eds. Escherichia coli                 17.   Campbell EA, Tupy JL, Gruber TM, Wang S, Sharp MM, et al. (2003) Crystal <br />     Salmonella: Cellular Molecular Biology. 2nd ed. Washington, DC: ASM                           structure Escherichia coli sE cytoplasmic domain anti-sigma <br />     Press. pp 17&#226;&#8364;&#8220;28.                                                                                  RseA. Mol Cell 11: 1067&#226;&#8364;&#8220;1078. <br />  2. Nikaido H (1994) Porins specific diffusion channels bacterial outer                  18.   Ades SE, Connolly LE, Alba BM, Gross CA (1999) Escherichia coli sE- <br />     membranes. J Biol Chem 269: 3905&#226;&#8364;&#8220;3908.                                                            dependent extracytoplasmic stress response controlled regulated <br />  3. Majdalani N, Gottesman S (2005) Rcs phosphorelay: complex signal                            proteolysis anti-sigma factor. Genes Dev 13: 2449&#226;&#8364;&#8220;2461. <br />     transduction  Annu Rev Microbiol 59: 379&#226;&#8364;&#8220;405.                                        19.   Alba BM, Leeds JA, Onufryk C, Lu CZ, Gross CA (2002) DegS YaeL <br />  4. Raivio TL (2005) Envelope stress responses Gram-negative bacterial                            participate sequentially cleavage RseA activate sE-dependent <br />     pathogenesis. Mol Microbiol 56: 1119&#226;&#8364;&#8220;1128.                                                        extracytoplasmic stress response. Genes Dev 16: 2156&#226;&#8364;&#8220;2168. <br />  5. Rowley G, Spector M, Kormanec J, Roberts M (2006) Pushing envelope:                     20.   Alba BM, Zhong HJ, Pelayo JC, Gross CA (2001) degS (hhoB) essential <br />     extracytoplasmic stress responses bacterial pathogens. Nat Rev Microbiol 4:                    Escherichia coli gene indispensable function provide sE activity. Mol <br />     383&#226;&#8364;&#8220;394.                                                                                          Microbiol 40: 1323&#226;&#8364;&#8220;1333. <br />  6. Ruiz N, Silhavy TJ (2005) Sensing external stress: watchdogs Escherichia coli        21.   Flynn JM, Levchenko  Sauer RT, Baker TA (2004) Modulating substrate <br />     cell envelope. Curr Opin Microbiol 8: 122&#226;&#8364;&#8220;126.                                                    choice: SspB adaptor delivers regulator extracytoplasmic-stress <br />  7. Las Penas  Connolly L, Gross CA (1997) SigmaE essential sigma                          response AAA+ protease ClpXP degradation. Genes Dev 18: <br />     factor Escherichia coli. J Bacteriol 179: 6862&#226;&#8364;&#8220;6864.                                           2292&#226;&#8364;&#8220;2301. <br />                                                                                                 22.   Kanehara K, Ito K, Akiyama Y (2002) YaeL (EcfE) activates sE pathway  8. Alba BM, Gross CA (2004) Regulation Escherichia coli sigma-dependent <br />                                                                                                       stress response site-2 cleavage anti-sE, RseA. Genes Dev 16: <br />     envelope stress response. Mol Microbiol 52: 613&#226;&#8364;&#8220;619. <br />                                                                                                       2147&#226;&#8364;&#8220;2155. <br />  9. Rhodius VA, Suh WC, Nonaka G, West J, Gross CA (2006) Conserved                                                                                                 23.   Grigorova  Chaba R, Zhong HJ, Alba BM, Rhodius V, et al. (2004) Fine- <br />     variable functions sE stress response related genomes. PLoS Biol 4: e2. <br />                                                                                                       tuning Escherichia coli sE envelope stress response relies multiple <br /> 10. Button JE, Silhavy TJ, Ruiz N (2007) suppressor cell death caused                     mechanisms inhibit signal-independent proteolysis transmembrane <br />     loss sE downregulates extracytoplasmic stress responses outer membrane                     anti-sigma factor, RseA. Genes Dev 18: 2686&#226;&#8364;&#8220;2697. <br />     vesicle production Escherichia coli. J Bacteriol 189: 1523&#226;&#8364;&#8220;1530.                         24.   Costanzo  Ades SE (2006) Growth phase-dependent regulation 11. Mecsas J, Welch R, Erickson JW, Gross CA (1995) Identification                                extracytoplasmic stress factor, sE, <span id='am-61' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-62' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span>guanosine</span> 39,59-bispyrophosphate <br />     characterization outer membrane protein, OmpX, Escherichia coli                     (ppGpp). J Bacteriol 188: 4627&#226;&#8364;&#8220;4634. <br />     homologous family outer membrane proteins including Ail Yersinia              25.   Dartigalongue C, Missiakas D, Raina S (2001) Characterization Escherichia <br />     enterocolitica. J Bacteriol 177: 799&#226;&#8364;&#8220;804.                                                         coli sE regulon. J Biol Chem 276: 20866&#226;&#8364;&#8220;20875. <br /> 12. Missiakas D, Betton JM, Raina S (1996) New components protein folding                 26.   Rezuchova B, Miticka H, Homerova D, Roberts M, Kormanec J (2003) New <br />     extracytoplasmic compartments Escherichia coli SurA, FkpA Skp/OmpH.                        members Escherichia coli sE regulon identified plasmid  <br />     Mol Microbiol 21: 871&#226;&#8364;&#8220;884.                                                                        FEMS Microbiol Lett 225: 1&#226;&#8364;&#8220;7. <br /> 13. Rouviere PE, Las Penas  Mecsas J, Lu CZ, Rudd KE, et al. (1995) rpoE,                  27.   Ruiz N, Kahne D, Silhavy TJ (2006) Advances understanding bacterial outer- <br />     gene encoding second heat-shock sigma factor, sE, Escherichia coli.                    membrane biogenesis. Nat Rev Microbiol 4: 57&#226;&#8364;&#8220;66. <br />     Embo J 14: 1032&#226;&#8364;&#8220;1042.                                                                       28.   Sperandeo P, Cescutti R, Villa R, Di Benedetto C, Candia D, et al. (2007) <br /> 14. Rouviere PE, Gross CA (1996) SurA, periplasmic protein peptidyl-prolyl                     Characterization lptA lptB, essential genes implicated lipopolysac- <br />     isomerase activity, participates assembly outer membrane porins. Genes                  charide transport outer membrane Escherichia coli. J Bacteriol 189: <br />     Dev 10: 3170&#226;&#8364;&#8220;3182.                                                                                244&#226;&#8364;&#8220;253. <br /> 15. Las Penas  Connolly L, Gross CA (1997) sE-mediated response                     29.   Johansen J, Rasmussen AA, Overgaard M, Valentin-Hansen P (2006) <br />     extracytoplasmic stress Escherichia coli transduced RseA RseB,                   Conserved small non-coding RNAs belong sE regulon: role     negative regulators sE. Mol Microbiol 24: 373&#226;&#8364;&#8220;385.                                             regulation outer membrane proteins. J Mol Biol 364: 1&#226;&#8364;&#8220;8. <br /> 16. Missiakas D, Mayer MP, Lemaire M, Georgopoulos C, Raina S (1997)                            30.   Thompson KM, Rhodius VA, Gottesman S (2007) sE regulates regulated <br />     Modulation Escherichia coli sE (RpoE) heat-shock transcription-                            small RNA Escherichia coli. J Bacteriol 189: 4243&#226;&#8364;&#8220;4256. <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                                         12                                 February 2008 | Volume 3 | Issue 2 | e1573 <br />                                                                                                                                                      sE Cell Envelope Integrity <br />  <br />  <br /> 31. Udekwu KI, Wagner (2007) sE controls biogenesis antisense RNA                           viability, characteristics, implications peptidoglycan synthesis. J Bacteriol <br />     MicA. Nucleic Acids Res 35: 1279&#226;&#8364;&#8220;1288.                                                            181: 3981&#226;&#8364;&#8220;3993. <br /> 32. Wade JT, Roa DC, Grainger DC, Hurd D, Busby SJ, et al. (2006) Extensive                     51.   Braun M, Silhavy TJ (2002) Imp/OstA required cell envelope biogenesis     functional overlap sigma factors Escherichia coli. Nat Struct Mol Biol                 Escherichia coli. Mol Microbiol 45: 1289&#226;&#8364;&#8220;1302. <br />     13: 806&#226;&#8364;&#8220;814.                                                                                52.   Charlson ES, Werner JN, Misra R (2006) Differential effects yfgL mutation 33. McBroom AJ, Johnson AP, Vemulapalli S, Kuehn MJ (2006) Outer membrane                             Escherichia coli outer membrane proteins <span id='am-29' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipopolysaccharide</span>. J Bacteriol 188: <br />     vesicle production Escherichia coli independent membrane instability.                    7186&#226;&#8364;&#8220;7194. <br />     J Bacteriol 188: 5385&#226;&#8364;&#8220;5392.                                                                 53.   Doerrler WT, Raetz CR (2005) Loss outer membrane proteins 34. McBroom AJ, Kuehn MJ (2007) Release outer membrane vesicles Gram-                           inhibition lipid export Escherichia coli YaeT mutant. J Biol Chem 280: <br />     negative bacteria novel envelope stress response. Mol Microbiol 63:                          27679&#226;&#8364;&#8220;27687. <br />     545&#226;&#8364;&#8220;558.                                                                                    54.   Malinverni JC, Werner J, Kim S, Sklar JG, Kahne D, et al. (2006) YfiO <br /> 35. Kitagawa M, Ara T, Arifuzzaman M, Ioka-Nakamichi T, Inamoto E,                                    stabilizes YaeT complex essential outer membrane protein <br />     Toyonaga H, Mori H (2005) Complete set ORF clones Escherichia coli                          assembly Escherichia coli. Mol Microbiol 61: 151&#226;&#8364;&#8220;164. <br />     ASKA library  complete set E. coli K-12 ORF archive): unique resources             55.   Wu T, McCandlish AC, Gronenberg LS, Chng SS, Silhavy TJ, et al. (2006) <br />     biological research. DNA Res 12: 291&#226;&#8364;&#8220;299.                                                         Identification protein complex assembles lipopolysaccharide 36. Mecsas J, Rouviere PE, Erickson JW, Donohue TJ, Gross CA (1993)                               outer membrane Escherichia coli. Proc Natl Acad Sci USA 103: 11754&#226;&#8364;&#8220;11759. <br />     activity sE, Escherichia coli heat-inducible sigma-factor, modulated            56.   El Ghachi M, Derbise  Bouhss  Mengin-Lecreulx D (2005) Identification     expression outer membrane proteins. Genes Dev 7: 2618&#226;&#8364;&#8220;2628.                                    multiple genes encoding membrane proteins undecaprenyl pyrophosphate <br /> 37. Powell BS, Court DL, Inada T, Nakamura Y, Michotey V, et al. (1995) Novel                         phosphatase (UppP) activity Escherichia coli. J Biol Chem 280: 18689&#226;&#8364;&#8220;18695. <br />     proteins phosphotransferase encoded rpoN operon                 57.   Flardh K, Garrido T, Vicente M (1997) Contribution individual promoters     Escherichia coli. Enzyme IIANtr affects growth organic nitrogen                        ddlB-ftsZ region transcription essential cell-division gene ftsZ     conditional lethality erats mutant. J Biol Chem 270: 4822&#226;&#8364;&#8220;4839.                             Escherichia coli. Mol Microbiol 24: 927&#226;&#8364;&#8220;936. <br /> 38. Commichau FM, Forchhammer K, Stulke J (2006) Regulatory links                       58.   Anderson MS, Raetz CR (1987) Biosynthesis lipid precursors Escherichia <br />     carbon nitrogen metabolism. Curr Opin Microbiol 9: 167&#226;&#8364;&#8220;172.                                   coli. cytoplasmic acyltransferase converts UDP-N-acetylglucosamine 39. Rabus R, Reizer J, Paulsen  Saier MH Jr (1999) Enzyme Ntr) Escherichia                   UDP-3-O-(R-3-hydroxymyristoyl)-N-acetylglucosamine. J Biol Chem 262: <br />     coli. novel enzyme phosphoenolpyruvate-dependent phosphotransferase                      5159&#226;&#8364;&#8220;5169. <br />     exhibiting strict specificity phosphoryl <span id='am-104' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-105' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span>acceptor</span>, NPr. J Biol Chem          59.   Humphreys S, Stevenson  Bacon  Weinhardt AB, Roberts M (1999)     274: 26185&#226;&#8364;&#8220;26191.                                                                                 alternative sigma factor, sE, critically important virulence Salmonella <br />                                                                                                       typhimurium. Infect Immun 67: 1560&#226;&#8364;&#8220;1568. <br /> 40. Lee CR, Cho SH, Yoon MJ, Peterkofsky  Seok YJ (2007) Escherichia coli <br />                                                                                                 60.   Liu D, Reeves PR (1994) Escherichia coli K12 regains O antigen. Microbiology <br />     enzyme IIANtr regulates K+ transporter TrkA. Proc Natl Acad Sci USA 104: <br />                                                                                                       140: 49&#226;&#8364;&#8220;57. <br />     4124&#226;&#8364;&#8220;4129. <br />                                                                                                 61.   Miller JH (1972) Experiments Molecular Genetics. Cold Spring Harbor, NY: <br /> 41. Lee CR, Koo BM, Cho SH, Kim YJ, Yoon MJ, et al. (2005) Req <br /> </body></html>