<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>BMC Systems Biology                                                                                                                      BioMed Central <br />  <br />  <br />  <br /> Methodology article                                                                                                                    Open Access <br /> Machine learning based analyses metabolic networks supports <br /> high-throughput knockout screens <br /> Kitiporn Plaimas1,2, Jan-Phillip Mallm1, Marcus Oswald3, Fabian Svara4, <br /> Victor Sourjik4, Roland Eils*1,2 Rainer K&#195;&#182;nig1,2 <br />  <br /> Address: 1Department Bioinformatics Functional Genomics, Institute Pharmacy Molecular Biotechnology, Bioquant, University Heidelberg, Im Neuenheimer Feld 267, 69120 Heidelberg, Germany, 2Division Theoretical Bioinformatics, German Cancer Research Center <br /> (DKFZ), Im Neuenheimer Feld 280, 69120 Heidelberg, Germany, 3Department Discrete Optimization, Interdisciplinary Center Scientific <br /> Computing, University Heidelberg, Im Neuenheimer Feld 368, 69120 Heidelberg, Germany 4Center Molecular Biology Heidelberg <br /> (ZMBH), Im Neuenheimer Feld 282, 69120 Heidelberg, Germany <br /> Email: Kitiporn Plaimas - k.plaimas@dkfz  Jan-Phillip Mallm - jpmallm@gmx  Marcus Oswald - Marcus.Oswald@Informatik.Uni- <br /> Heidelberg  Fabian Svara - fabian.svara@urz.uni-heidelberg  Victor Sourjik - v.sourjik@zmbh.uni-heidelberg  <br /> Roland Eils* - r.eils@dkfz  Rainer K&#195;&#182;nig - r.koenig@dkfz * Corresponding author <br />  <br />  <br />  <br />  <br /> Published: 24 July 2008                                                       Received: 29 April 2008 <br />                                                                               Accepted: 24 July 2008 <br /> BMC Systems Biology 2008, 2:67   doi:10.1186/1752-0509-2-67 article available  http://www.biomedcentral.com/1752-0509/2/67 <br /> &#194;&#169; 2008 Plaimas et al; licensee BioMed Central  Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />  <br />  <br />  <br />  <br />                  Abstract <br />                  Background: Computational identification new drug targets major goal pharmaceutical <br />                  bioinformatics. <br />                  Results: paper presents machine learning strategy study validate essential enzymes <br />                  metabolic network. single enzyme characterized local network topology, gene <br />                  homologies expression, flux balance analyses. machine learning trained <br />                  distinguish essential non-essential reactions. validated comprehensive <br />                  experimental dataset, consists phenotypic outcomes single knockout mutants                  Escherichia coli (KEIO collection). yielded reliable results high accuracy (93%)                  precision (90%). topologic, genomic transcriptomic features describing                  network sufficient defining essentiality reaction. features substantially <br />                  depend specific media conditions enabled apply approach specific <br />                  media conditions, like lysogeny broth rich medium. <br />                  Conclusion: analysis feasible validate experimental knockout data high throughput <br />                  screens, used improve flux balance analyses supports experimental knockout screens <br />                  define drug targets. <br />  <br />  <br />  <br />  <br /> Background                                                                     silico helps identify enzymes essential Defining drug targets drug design major                      survival organism [2,3]. general model goals biomedical research. particular, metabolic                         metabolic network described graph theoreti- <br /> enzymes successfully targeted specific drugs                      cal approaches applied identify drug targets inhibit essential processes pathogenic organisms                      pathogenic organisms [4]. term &apos;damage&apos; used human host [1]. Analyzing metabolic network                         assess enzymes serve drug targets  <br />  <br />                                                                                                                                        Page 1 11 <br />                                                                                                                  (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                              http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> inhibition influences substantial number             targets. Supporting claim, 19 37 predictions stream metabolic reactions products [5].           novel targets literature reported  concepts choke points load points            experimental evidence. <br /> successfully applied estimate essentiality enzyme [2,3]. Load points defined hot spots     Methods <br /> metabolic network (enzymes/metabolites) based           Network reconstruction <br /> ratio number k-shortest paths passing      data metabolic network taken pre- <br /> metabolite/enzyme  , number nearest          vious study reconstructed way (iAF1260, <br /> neighbor links   attached  ratio com-    Feist workers [8]). Basically, metabolic net- <br /> pared average load value network [2]. Choke      work represented undirected bipartite graph <br /> points uniquely consume produce certain metabolite,       consisting metabolites reactions alternating make indispensable. example,             nodes. network taken flux balance analy- <br /> Plasmodium falciparum d-aminolevulinate dehydratase            ses. analyses, unspecific compounds (ALAD) considered choke point [3]        water, <span id='am-19' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span>ATP</span>,  discarded. <br /> proven experimentally serve valid antimalarial tar- [6].                                                       gold standard <br />                                                                order demonstrate efficiency approach Flux balance analyses (FBA) widely used          used data KEIO collection [10] gold stand- <br /> established method assess essentiality genes         ard. dataset consisted phenotypic outcomes <br /> [7,8].  FBA approaches need clear definitions       set knockout mutants single genes nutrition availability biomass production spe-       used define classes &quot;essential&quot; &quot;non-essential&quot; <br /> cifically given environmental conditions  good      reactions. Genes knocked frame <br /> view aspects e.g. [9]). High-throughput           replacement PCR product containing kanamycin <br /> experiments performed investigate essen-      resistance gene. start-codon stream trans- <br /> tiality major portion genes organism [10-    lational signal replaced fully intact. 12]. Escherichia coli, essentiality virtually   kanamycin treatment, frame single gene deletions open reading frames observed comprehensive            verified PCR loci specific primers. knockout screen (KEIO collection [10]). data enables      unable create mutant formed colonies plate, test performance silico metabolic model        mutated gene considered essential. Knock- predicts essential genes. Analyzing flux balances         experiments performed LB rich medium aerobic glucose condition using COBRA tool-          glucose minimal medium, resulting datasets <br /> box [13] newly reconstructed metabolic network        (denoted rich medium glucose minimal medium, <br /> E. coli yielded 92% accuracy predicting essenti-      respectively). rich medium, 4,288 tested <br /> ality genes [8]. Feist workers compared        genes, 303 genes mutants predictions KEIO collection yielded 88%       defined essential. Genes considered rich media conditions. study, FBA cor-      essential rich medium condition  <br /> responding experimental knockout screen performed          sidered essential glucose minimal medium  study opportunistic behavior pathogen Pseu-      dition. Additionally, genes, 119 genes domonas aeruginosa systems view [14].                   assigned essential glucose minimal medium                                                                showed slow growth minimal media (growth paper propose integrative machine learning       rate &#226;&#8240;&#164; 0.0926 24 hours). Experimental criteria gene <br /> approach applying broad list described tools.     essentiality glucose minimal medium described machine learning supplied qualitative          [8,12]. Genes mapped corresponding quantitative descriptors derived biochemical          proteins, enzymes reactions using gene-protein- <br /> knowledge, genomic transcriptomic data, flux           reaction Table Feist et al. [8]. reaction(s) associ- <br /> balance analyses. Using KEIO collection [10]        ated gene defined essential non- <br /> gold standard, yielded overall accuracy 93%       essential way activate reac- <br /> rich media conditions. Comparative analysis        tion(s) genes coding gene experi- <br /> flux balance approach machine learning                 mentally essential non-essential, respectively. <br /> approach yielded improvements FBA,          discarded training test- <br /> consider aminyl-tRNA reactions modeling. Predictions        ing analysis. Furthermore, 133 reactions discarded contradicted KEIO collection experimentally      analysis, corresponding genes couldn&apos;t tested successfully used detect errors experi-   defined. Finally, 303 essential genes determined <br /> mental data. Predicted reactions matching experimen-       set 231 essential 1125 non-essential reactions <br /> tal screen strengthen candidacy potential drug        rich medium. 1125 non-essential reac- <br />  <br />  <br />                                                                                                                    Page 2 11 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> tions rich medium, 107 reactions defined                       Topology based features <br /> essential glucose minimal medium. total, 1356                       set breadth algorithm investigate net- <br /> reactions used experimental results (KEIO)                      work single reaction blocked. defined essentiality taken class labels reac-               reaction essential survival basically tions (samples) training validating classifiers.                 mutated network yield products reac- <br /> Note  didn&apos;t use experimental data fea-                 tion upstream substrates reaction.  fea- <br /> tures reactions.                                                      tures defined knocked reaction <br />                                                                              substantial producing downstream metabolites <br /> Defining features                                                        products produced path- list relevant features obtained differ-                  ways. investigation tested knocked reac- <br /> ent aspects: network topology, genomics flux balance                     tion performed following algorithm. <br /> analyses. Table 1 gives overview features abbreviations. <br /> Table 1: List features <br />  <br />  Short form Explanation <br />  <br />                 Topology features: local structures <br />  RUPa           Reachable/Unreachable Products (RUP): equal product produced blocking reaction <br />  PUP            Percentage Unreachable Products (PUP): percentage products produced blocking reaction <br />  NSa            Number Substrates (NS) <br />  NPa            Number Products (NP) <br />  NNRa           Number Neighbouring Reactions (NNR) <br />  NNNRa          Number Neighbours Neighbouring Reactions (NNNR) <br />  CCVa           Clustering Coefficient Value (CCV): clustering coefficient reaction <br />  DIRa           Directionality reaction (DIR) <br />  <br />                 Topology features: deviations, choke points, load scores damage <br />  ND             Number Deviations (ND) <br />  APL            Average Path Length (APL): average path length deviations <br />  LSPa           Length Shortest Path (LSP): length shortest path deviations <br />  CP             Choke Point (CP): reaction choke point (Rahman et al, 2006) <br />  LSa            Load Score (LS): load score reaction (Rahman et al, 2006) <br />  NDRa           Number Damaged Reactions (NDR): number damaged reactions blocking reaction (Lemke et al, 2004) <br />  NDCa           Number Damaged Compounds (NDC): number damaged compounds blocking reaction (Lemke et al, 2004) <br />  NDRDa          Number Damaged Reactions having Deviations (NDRD): number damaged reactions alternative paths <br />                 reached blocking reaction <br />  NDCDa          Number Damaged Compounds having Deviations (NDCD): number damaged compounds alternative <br />                 paths reached blocking reaction <br />  NDCRa          Number Damaged Choke point Reactions (NDCR): number damaged choke point reactions blocking reaction <br />  NDCCa          Number Damaged Choke point Compounds (NDCC): number damaged choke point compounds blocking reaction <br />  NDCRDa         Number Damaged Choke point Reactions having Deviations (NDCRD): number damaged choke point reactions                 alternative paths reached blocking reaction <br />  NDCCDa         Number Damaged Choke point Compounds having Deviations (NDCCD): number damaged choke point compounds <br />                 alternative paths reached blocking reaction <br />  <br />                 Gene expression data, genomic data miscellaneous <br />  NCGa           Number Coding Genes (NCG): number coding genes reaction <br />  H10a           Homology 10-10 (H10): number homologous genes e-value cutoff 10-10 <br />  H7             Homology 10-7 (H7): number homologous genes e-value cutoff 10-7 <br />  H5a            Homology 10-5 (H5): number homologous genes e-value cutoff 10-5 <br />  H3a            Homology 10-3 (H3): number homologous genes e-value cutoff 10-3 <br />  NRSGa,b        Number Reactions Genes (NRSG): number reactions derived genes <br />  NRSEa,b        Number Reaction having Similar Expression <br />                 (NRSE): number reactions similar expression (correlation coefficient &gt;0.8) <br />  MCCa,b         Maximum Correlation Coefficients (MCC): maximum value correlation coefficients neighbouring reactions <br />  <br />                 Flux distribution <br />  BFVa           Biomass Flux Value (BFV): biomass flux value blocking reaction  aerobic glucose condition) <br />  <br />  athe   optimized features, bfeatures gene expression <br />  <br />  <br />                                                                                                                                       Page 3 11 <br />                                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />   metabolites acting input nodes (substrates)       blocked. organism pathways pro- <br /> output nodes (products) knocked reaction        duce products making robust.  <br /> selected. set substrates S defined input nodes       counted number possible alternative paths set products P defined output nodes.      yielding feature ND (ND, number deviations). broader list available substrates integrated substrates S. included substrates      Choke points, load points damage <br /> upstream reactions products downstream           reaction uniquely consumes produces certain <br /> reactions sets S P, respectively. Substrates    metabolite metabolic network considered reactions substrates S sub-   choke point. reaction shows high potential strate included S.  substrates reactions    essentiality [2,3]. checked observed reaction metabolite P substrate         choke point (CP, choke points). According  <br /> included S.                                                cept load scores [2], computed load score                                                                 reaction average number pathways passing <br /> ii. Reactions selected used available com-      reaction, comparison number pounds substrates.                                           pathways metabolites network. used                                                                 <span id='am-18' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> damaged compounds/reactions reported iii. selected reactions products incor-    [5]. Basically, damage defined determining porated network. products set new        potentially effected metabolites reactions  <br /> available metabolites network.                           stream knocked reaction. applied def- <br />                                                                 inition calculating features NDR (NDR, number iv. Steps ii iii repeated reactions   damaged reactions) NDC (number damaged com- identified incorporation.                          pounds). turn, damaged compounds/reactions <br />                                                                 produced alternative pathways. <br /> v. output nodes produced counted          calculated number damaged com- <br /> (reachable products P).                                         pounds/reactions did alternative way                                                                 reached substrates knocked reac- finishing process, used number defined      tion (NDRD, number damaged reactions having output nodes produced mutated          deviations; NDCD, number damaged compounds hav- <br /> network features, e. quality feature defining    ing deviations). addition analysis dam- product produced (RUP, reacha-        aged compounds/reactions, included number <br /> ble/unreachable products), percentage prod-          damaged choke points (NDCR, number damaged <br /> ucts produced (PUP, percentage             choke point reactions; NDCC, number damaged choke <br /> unreachable products).                                          point compounds; NDCRD, number damaged choke <br />                                                                 point reactions having deviations; NDCCD, number run breadth search network esti-     damaged choke point compounds having deviations). <br /> mate possible deviations. time focused rele- <br /> vant pathways using similarity measure          Local topology features <br /> SIMCOMP software [15]. SIMCOMP used define               number substrates products knocked relevant substrates products reac-         reaction counted (NS, number substrates, NP, <br /> tion. Starting S, breadth search explored    number products, respectively).  defined <br /> network finding direct products knocked      features number neighbouring reactions (NNR, <br /> reaction. algorithm visited products,         number neighbouring reactions), number neigh- <br /> stored corresponding pathway continued              bours neighbouring reactions (NNNR, number search alternative paths network      neighbours neighbouring reactions) clustering entirely explored maximal path length 10 reac-      coefficient (CCV, clustering coefficient values) [16,17] tions reached. took average path length (APL,        knocked reaction. reaction direction (DIR, <br /> average path length) shortest path length (LSP,         directionality reaction) taken model <br /> length shortest path) deviations features      Feist et al.[8]. classifier. deviation features used alternative pathways produce products knocked         Gene expression data, genomic data miscellaneous reaction substrates S. metabolic network,     case study, collected gene expression data substrates consumed reactions        study observing regulation oxygen depriva- <br /> yielding products   kept track        tion [18]. dataset taken unspe- <br /> alternative paths metabolic network poten-       cific regulation, e. small band broad <br /> tial organism survive reaction             range effected metabolic pathways. gene expression <br />  <br />  <br />                                                                                                                    Page 4 11 <br />                                                                                              (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                 http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> data data-set mapped corresponding          diction value (number true negatives/number nega- <br /> reactions. reaction catalysed complex        tively predicted reactions). proteins, took mean gene expression val- <br /> ues corresponding genes  details            Feature selection <br /> [19]). Genes pathway regula-           feature selection approach. <br /> tion [20].  maximum correlation coefficient        trained Support Vector Machines terms max- neighboring reactions knocked reaction         imizing overall accuracy using features. single <br /> (MCC, maximum correlation coefficient) number            feature discarded data set perform- reactions having similar gene expression (correlation         ance machine observed. Testing perform- <br /> coefficient &gt; 0.8) calculated (NRSE, number reac-        ance machine leave cross <br /> tions having similar expression).              validation. accuracies machines missing number reactions coming gene (NRSG,             feature compared best machine kept number reactions genes), features             iteration. repeated accuracy did served machine estimating knocked reac-       increase. machine best accuracy selected <br /> tion biosynthesis degradation pathway.            best classifier features optimized fea- included number homologous genes                ture set. taken function knocked gene. Homologous genes searched using Blast [21]            Experimental protocol knockout verification open reading frames E. coli differ-     Knockout mutations verified PCR amplification <br /> ent e-value cutoffs, e. 10-3, 10-5, 10-7, 10-10 yielding   genomic loci expected contain 1327 base pair features H3, H5, H7 H10, respectively.               gene replacement cassette specific primers (Table S4 <br /> method flux balance simulations described           Additional file 1). Primers chosen equal <br /> Results discussion (section Comparing perform-           predicted melting temperatures ~60&#194;&#176;C hybridised <br /> ance performance flux balance analyses).               specific distances upstream downstream tar- <br />                                                                  gene. PCR reactions performed directly Machine learning                                                 freshly grown bacterial colonies 30 cycles applied Support Vector Machines R package            annealing temperature 54&#194;&#176;C. product sizes <br /> e1071 [22] classify essential non-essential       obtained KEIO collection strains compared <br /> reactions metabolic network. radial basis func-         wild-type E. coli strain MG1655 1% <br /> tion used kernel function. Parameter optimiza-        agarose gels. <br /> tion performed regularization term defined costs false classifications (5 steps    Assembling list drug targets <br /> range: 2n, n = -4, -2, 0, 2, 4). range taken    map enzymes drug targets, drugs corre- kernel width &#206;&#179;. optimization realized            sponding drug targets selected drug data- <br /> training grid search combinations       base Drugbank [23]. took drugs account parameters [22]. sizes classes differed sig-      affected organism excepting humans mam- <br /> nificantly data set (essential: 17%, non-essential:       mals. Entries metabolites reaction <br /> 83%). broad spectrum different precisions           KEGG database [24] discarded restrict sensitivities, varied weight factor positive      drug list non-endogenous compounds. targets&apos; <br /> instances data set optimized feature set       annotated EC numbers remaining drugs col- range 0.1 5.0. performed leave         lected validated drug targets. <br /> cross validation measure effectiveness machine learning method. single reaction selected          Results discussion validation data predicted remaining         overview machine learning procedure shown <br /> reactions training data. repeated       Figure 1. machine learning trained reaction data set. assessing performance       validated large set features. Firstly, local topology classifiers, calculated standard measures accu-       based features used qualitatively possi- <br /> racy (number correctly predicted reactions/number          ble flux deviations. Secondly, choke load points predicted reactions), sensitivity (number true posi-      defined damage used qualitative <br /> tives/(number true positives + number false nega-          flux load stream effects knocked tives)), specificity (number true negatives/(number        reaction. Thirdly, functional genomics data,  <br /> true negatives + number false positives)), positive pre-      expression genes  stream reactions <br /> diction value precision (number true positives/            used indicate conjoint reactions pathway. number positively predicted reactions), negative pre-         addition features, considered existence                                                                  homologous genes corresponding knockout reac- <br />  <br />  <br />                                                                                                                       Page 5 11 <br />                                                                                                 (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                     http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />  <br />                        Metabolic Network                           1) Topology based features <br />                                                                              - Local topology <br />                                                                              - Deviations <br />                                                                              - Choke load points <br />                                                                              - Damages <br />                                                                    2) Genomic transcriptomic data <br />                                                                                              ATCGAGTTGCAAT <br />                                                                                              ATAGAA &#226;&#8364;&#8220; TGCA-T <br />                                                                                             - Gene expression <br />                                                                                             - Homologous genes <br />                          reaction                                 3) Flux balance analysis <br />                          metabolite <br />                                                                                        - Biomass objective flux <br />  <br />  <br />                                                                              Feature creation <br />                                                                                     Features <br />                                                                                            R <br />                                                                                                           Essentiality <br />                                                                              P S P NR NN V             classes <br />                                                                            RU P N N N N CC . . . <br />                                                                        1                                    E   E: Essential <br />                                                                                                             N   N: Non-essential <br />                                                                                                           N <br />                                                            Reactions <br />                                                                          ...   3 1    4   ...             E <br />                                                                                                             . <br />                                                                                                             . <br />                                                                        . <br />                                                                                                             . <br />                                                                        . <br />                                                                        . <br />  <br />                                                                                  Reaction profile <br />  <br />  <br />       machine learning (SVM) <br />  <br />              Essential             &#207;&#8224; <br />                o o                                           o                                    Trained classifier <br />                 o o x                                     oo <br />               o                                         o o o <br />                        x x                                x xx <br />               x ox                                                                             Prediction                         Non-essential                    x x <br />                                                                                             reaction essential <br />                  Input space                           Space kernel <br />  <br />  <br /> Figure 1 overview method <br /> Schematic <br /> Schematic overview method. reaction, features calculated describing 1) local topology metabolic network, 2) genomic transcriptomic relations, 3) biomass production using flux balance analyses discarding reaction. E.g. given reaction educts, product 4 neighbors, feature NS (number sub- <br /> strates), NP (number products), NNR (number neighboring reactions) set 3, 1 4, respectively. table reaction profiles created used training classifier distinguish essential non-essential reac- <br /> tions. Support Vector Machine (SVM) applied classifier optimal separating hyperplane. <br />  <br />  <br />  <br />  <br />                                                                                                                              Page 6 11 <br />                                                                                                        (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                   http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br /> tion, expected function           positive instances recognized knocked gene. Finally, used flux balance anal-          weight factor increased. highest specificity (99%) yses estimate network&apos;s biomass production follow-          best precision (95%) yielded data <br /> ing knocking reaction. list features given    point weight factor 0.1. beneficial Table 1.                                                           predicting drug targets high reliability. Alternatively, <br />                                                                    avoid overlooking potential targets, increased sensitiv- <br /> Performance machine learning algorithm                      ity achieved raising weight factor 1.0 small data set 1356 reactions training           (sensitivity = 75%). following, analyses validation, performed leave cross vali-           performed weight factor  <br /> dation measure effectiveness machine learn- <br /> ing  Using KEIO-collection data rich                 Identifying drug targets <br /> medium reference, gained overall accuracy          compared enzymes predictions 92% features taken (Table 2). increase            results KEIO collection comprehensive list performance, did systematic feature reduction             valid drug targets Drugbank database [23] (Fig- procedure. yielded better result            ure 3). Surprisingly, 80% drug target enzymes optimized feature set 25 features (accuracy =           KEIO high-throughput screen 93%, Table 2). 25 features regarded            machine. fact drug dominating factors leading good performance.          targets broad range organisms having different relevant,              topologies metabolic networks dif- <br /> started procedure, stopping          ferent alternative pathways corresponding drug tar- <br /> step. features, compared accuracy          gets. noted study focused classifier lacking feature, respectively. turned     reactions essential rich media conditions. loosing feature NNNR yielded worst classifi-          suggest 37 promising drug target enzymes cation performance (accuracy -0.89 compared clas-           Drugbank database, validated inter- <br /> sifier features) hinting      section predictions results exper- relevant feature. feature followed NRSE           imental KEIO screen. list reactions (-0.82), BFV (-0.74), NNR (-0.52) H10 (-0.52). Inter-          references reported experimental evidences given estingly, features span set     Table S1  Additional file 1]. feature categories (NNNR NNR: network <br /> toplogy; NRSE: gene expression, genomics; H10: homol-              Comparing performance machine learning <br /> ogy, genomics; BFV: flux balance analysis).                    approach flux balance analyses <br />                                                                    performed single reaction deletion network sizes classes &quot;essential reactions&quot; &quot;non-       calculated flux values FBA using Cobra Tool- <br /> essential reactions&quot; differed significantly data set        box [13] assess essential reactions aerobic glu- <br /> (essential: 17%, non-essential: 83%). obtaining differ-        cose minimal media conditions  described ent stringencies, weighted positive instances          supplementary material Feist et al. 2007). anal- <br /> factor ranging 0.1 5.0 step size 0.1 (Fig-       ysis, reaction assessed essential respective <br /> ure 2). sensitivity increased significantly smaller       prediction mutated network&apos;s maximal biomass higher weights, reaching plateau weight factors        production &lt; 1% wildtype&apos;s biomass produc- <br /> 1.0  expected, smaller weight classi-        tion. biomass objective function used analysis <br /> fier tended overwhelmed large negative class.         taken explained [8]. Note simulating <br />  <br /> Table 2: Performance machine learning based predictions rich media condition <br />  <br />                                                                          Machine Learning                          Machine Learning <br />                                                                            30 features)                   (25 optimized features) <br />  <br />  true positives                                                                         168                                        174 <br />  true negatives                                                                        1078                                       1092 <br />  false positives                                                                         47                                         33 <br />  false negatives                                                                         63                                         57 <br />  sensitivity (recall)                                                                72.73%                                     75.32% <br />  specificity                                                                         95.82%                                     97.07% <br />  positive predictive values (precision)                                              78.14%                                     84.06% <br />  negative predictive values                                                          94.48%                                     95.04% <br />  overall accuracy                                                                    91.89%                                     93.36% <br />  <br />  <br />  <br />  <br />                                                                                                                         Page 7 11 <br />                                                                                                   (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                         http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />                                                                        medium according experimental criteria gene <br />                                               ROC Plot                 essentiality glucose minimal media KEIO col- <br />                     90%                                                lection [8,10,12]. 996 reactions identified non- <br />                     80%                                                essential. remaining reactions associated gene, <br />                     70%                                                exchange reactions, clearly identi- <br />  sensitivity (%) <br />  <br />  <br />  <br />  <br />                     60%                                                fied. FBA approach detected essentiality reac- <br />                                                                        tion aerobic glucose minimal condition                     50%                                                accuracy 86%, sensitivity 52% specificity                     40%                                                98%. performed machine learning glucose <br />                     30%                                                minimal conditions BFV (Biomass flux <br />                     20%                                                value FBA simulation) BFV improve <br />                     10%                                                results (Table 3). BFV, approach yielded 90% <br />                                                                        precision 79% recall experimental results, com- <br />                      0% <br />                         0% 1% 2% 3% 4% 5% 6% 7%                        pared FBA results 87% precision 51% recall. <br />                                                                        Results reactions given Table S2 [Additional <br />                                       100 &#226;&#8364;&#8220; specificity (%)            file 2]. <br />  <br /> ROC-curve <br /> Figurefactors <br /> weight         positive <br />          2 showing    prediction <br />                             instances results different           Improving flux balance simulations <br /> ROC-curve showing prediction results dif-                     Figure 4 shows comparison method FBA ferent weight factors positive instances. (blue)              categorizing results according KEGG pathways <br /> diamond shows result different weight. left          [24]. essential reactions right, weight increased 0.1 5.0, step size <br />                                                                        machine learning approach FBA 0.1. weight factors higher 1.0, sensitivity remained constant. dotted line manually            reactions <span id='am-38' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-39' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-42' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> metabolism <span id='am-5' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> <br /> fitted.                                                                metabolism. amino acid metabolism, tRNA trans- <br />                                                                        ferases amino acids essen- <br />                                                                        tial machine learning approach FBA. rich media conditions challenging, difficult            improved FBA simulations adding corresponding <br /> characterize uptake rates compound rich              aminyl-tRNA reactions products biomass <br /> medium (Adam Feist, personal communication 2008).                      objective function. ensured tRNA transferases  compared performances                   predicted essential FBA method. approach FBA glucose minimal media. 338                    simulations subsequently gained better results cor- <br /> reactions essential glucose minimal                rectly predicting essentiality aminyl-tRNA reac- <br />                                                                        tions. <br />  <br />                                                                        Using approach means validate                                             Machine Learning <br />                                                                        experimental knockout screen <br />                                  8                                     Predicting different outcome experimental high <br />                                                                        throughput screen (KEIO) error  <br />                             37         17                              Table 3: Comparison machine learning method Flux <br />                                  15                                    Balance Analyses glucose minimal media condition <br />                                                                  183 <br />                        22        16         125                         Performance                         ML\BFV1          ML2         FBA3 <br />  <br />                                                                         true positives                          192          266          174 <br />                                                   Drug targets          true negatives                          932          968          971 <br />                    KEIO (rich medium)                                   false positives                          64           28           25 <br />                                                                         false negatives                         146           72          164 <br />                                                                         sensitivity                          56.80%       78.70%       51.48% Figureinvestigated <br />           3        enzymes                                              specificity                          93.57%       97.19%       97.49% investigated enzymes. sets shown: essen-                  positive predictive values           75.00%       90.48%       87.44% <br /> tial enzymes machine learning approach                 negative predictive values           86.46%       93.08%       85.55% <br /> circle), essential enzymes KEIO knockout                   overall accuracy                     84.26%       92.50%       85.83% <br /> screen [10]  left circle) enzymes valid <br />                                                                         1machine learning feature BFV (biomass flux value drug targets taken database Drugbank [23] (bot- <br /> tom-right).                                                             FBA). <br />                                                                         2machine learning including feature BFV <br />                                                                         3Flux Balance Analysis <br />  <br />  <br />  <br />  <br />                                                                                                                                Page 8 11 <br />                                                                                                          (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                       http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  <br />                                                                         4 genes list, mutations                                                      FBA <br />                                                         Non essential   stated 100% correct Baba et al.                                                          Essential <br />                                                      ML                 genes (aspC, epd, luxS, thiE) correct PCR <br />           Metabolism                                 Non essential <br />   Cofactors Vitamins                                Essential <br />                                                                         product corresponding knockout allele   Amino Acid Metabolism                              Experiment         observed. <br />                                                        Nonessential <br />    <span id='am-30' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-31' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> Metabolism                               Essential <br />                                                                         Conclusion <br />         Lipid Metabolism <br />                                                                         Defining drug targets challenging task. experi- <br />  <span id='am-11' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>Carbohydrate</span> Metabolism                                                ments rely conditional essentiality screen genes       pathway                                                define associated enzymes possible drug targets. <br />                                                                         Machine learning methods help validate exper- <br />                                                                                          imental data. approach used experimental knock- <br />                            0   50 100 150 200 250 300 350               data E. coli KEIO [10]. machine                                Number reactions                      trained data predicted quite accurately                                                                         experimental outcomes. methods based graphi- <br /> Figure <br /> Comparison experimental <br />         4  data,  according <br />                      machine  learning <br />                                    different <br />                                        predictions, <br />                                                 pathways <br />                                                     FBA             cal networks aim finding weak points net- <br /> Comparison machine learning predictions,                         work. set machine learning integrates <br /> FBA experimental data, according differ-                     features describing network topology functional <br /> ent pathways. pathway KEGG [24], lowest                 genomics properties elaborated way. bars represent experimental result (KEIO), reactions                                                                         gained valuable insights. Firstly, grouped non-essential (left, blue) essential (right, <br /> magenta). mid bars prediction results              topologic, genomic transcriptomic data describing machine learning approach flux balance analyses,             network attributes sufficient defining respectively. Larger differences machine learn-             essentiality certain reaction. pathogens ing prediction FBA amino acid metabo-               hard define environmental parameters lism lipid metabolism.                                              complex changeable e.g. intestinal infections. <br />                                                                         approach  principle, handle media condi- <br />                                                                         tions, shown rich media conditions study. algorithm, error experimental knock-               Rich media conditions better reflect situation screen. examined lists false positives                pathogens host (like e.g. gut), compar- <br /> false negatives experimental set-ups. list                ison minimal media conditions clearly defined <br /> false negatives contained 71 genes algorithm                  carbon sources flux balance analyses predicted non-essential glucose minimal                 adapted. second benefit study experimental <br /> dition contradiction outcome KEIO exper-               validation support estimations potential drug <br /> iment [10]. 33 obtained corresponding                    targets. regarding intersection results knockout clones KEIO library (growing rich                  KEIO collection, 37 potential targets media), grew M9 glucose medium.                      novel drugs, 19 able grow 9 33 clones good growth                reported experimental evidence literature. rates (OD600 &#226;&#8240;&#165; 0.2 48 hours) 3 clones rea-               advantage machine learning approaches easily <br /> sonable growth rates (OD600 0.07 0.2                  change stringency parameter, e.g. increasing preci- <br /> 48 hours). complete list given Table S3  Addi-             sion avoid loosing potential candidates, weight fac- <br /> tional file 1]. turn, tested list false posi-         tor positive instances increased. used <br /> tives, algorithm predicted 33 genes                 gene expression data E. coli wild-type single <br /> essential, contrast experimental high throughput              knock strains. single knock outs regulators <br /> screen. assumed genes weren&apos;t                     respiration effecting large number genes knocked correctly. Baba et al. (2006) provided valid-             treatment unspecific (growth oxygen rich <br /> ity estimation clones. compared results             deprived conditions).  large portion net- estimations selected 6 genes, mutants               work pathways metabolic network differen- estimated equal 37.5% correct.               tially expressed [25]. presented approach, data 5 6 genes (alaS, coaA, coaE, glyS                  pathway unspecific examinations suited let <br /> hemE) PCR specific primer pairs (Table S4 Addi-                 classifier learn neighboring enzymes jointly <br /> tional file 1) yielded products sizes correspond-              work   multiple gene expres- <br /> ing wild-type knockout alleles, respectively.               sion datasets variety conditions suit indicated genes correctly knocked                 approach.  needs investigated wild-type gene present. PCR product                gene expression data suits best optimize  observed ileS knockout. Additionally tested              formance. <br />  <br />  <br />                                                                                                                             Page 9 11 <br />                                                                                                       (page number citation purposes) <br />  BMC Systems Biology 2008, 2:67                                                               http://www.biomedcentral.com/1752-0509/2/67 <br />  <br />  <br />  presented broadly applied                    SYS consortium Viroquant (# 0313923), Deutscher Akademischer Aus- systems seeking potential drug targets variety                  landsdienst, Helmholtz Alliance Systems Biology Cancer substantial bacterial infections organisms.                   Commission Higher Education (CHE) Thailand. <br /> E. coli benefited rich data pool including elaborated metabolic network, genome wide knock                                                                             References <br />                                                                             1.    Hopkins AL, Groom CR: druggable genome. Nature reviews <br /> viability screen, genome sequence feasible gene                         2002, 1(9):727-730. <br /> expression dataset. Nowadays, genomic sequence                      2.    Rahman SA, Schomburg D: Observing local global proper- <br />                                                                                   ties metabolic pathways: &apos;load points&apos; &apos;choke points&apos; limiting factor applications                             metabolic networks. Bioinformatics 2006, 22(14):1767-1774. <br /> remarkable number genomes sequenced                          3.    Yeh  Hanekamp T, Tsoka S, Karp PD, Altman RB: Computational sequenced future. approach uses                        analysis Plasmodium falciparum metabolism: organizing <br />                                                                                   genomic information facilitate drug discovery. Genome Res <br /> unspecific gene expression data obtained                         2004, 14(5):917-924. publically available resources obtained                   4.    Schuster S, Fell DA, Dandekar T: general definition meta- <br /> straightforward experiments. elaborated meta-                           bolic pathways useful systematic organization analy- <br />                                                                                   sis complex metabolic networks. Nat Biotechnol 2000, <br /> bolic networks assembled organisms                             18(3):326-332. <br /> (e.g. B. subtilis [11], H. pylori [26], M. barkeri [27], M.                 5.    Lemke N, Heredia F, Barcellos CK, Dos Reis  Mombach JC: <br />                                                                                   Essentiality damage metabolic networks. Bioinformatics <br /> tuberculosis [28], S. cerevisiae [29]) expect                    2004, 20(1):115-119. method transferred major difficulties.                   6.    Bonday ZQ, Dhanasekaran S, Rangarajan PN, Padmanaban G: Import networks received large                            host delta-aminolevulinate dehydratase malarial <br />                                                                                   parasite: identification new drug target. Nat Med 2000, <br /> organisms existing excellent databases like BioCyc                           6(8):898-903. <br /> [30] Kegg [24]. challenging exploit                 7.    Edwards JS, Palsson BO: Metabolic flux balance analysis                                                                                   silico analysis Escherichia coli K-12 gene deletions. BMC <br /> networks method. Finally,                              Bioinformatics 2000, 1:1. <br /> approach genome wide essentiality screen sub-                  8.    Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> stantial laborious. methodological challenging                         Broadbelt LJ, Hatzimanikatis V, Palsson BO: genome-scale met- <br />                                                                                   abolic reconstruction Escherichia coli K-12 MG1655 task remains employ approach different                              accounts 1260 ORFs thermodynamic information. <br /> organisms, e.g. using essentiality screen                           Molecular systems biology 2007, 3:121. <br /> organism infer information                                9.    Schuetz R, Kuepfer L, Sauer U: Systematic evaluation objec- <br />                                                                                   tive functions predicting intracellular fluxes Escherichia <br />                                                                                   coli. Molecular systems biology 2007, 3:119. <br /> Authors&apos; contributions                                                      10.   Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko <br />                                                                                   KA, Tomita M, Wanner BL, Mori H: Construction Escherichia <br /> KP, J&#226;&#8364;&#8220;PM, RK general concept                                coli K-12 frame, single-gene knockout mutants: Keio <br /> design study. KP carried data analysis. MO                         collection. Molecular systems biology 2006, 2:2006 0008. KP conceptualized implemented breath                      11.   Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome- <br />                                                                                   scale reconstruction metabolic network Bacillus subtilis <br /> search algorithm. FS VS validated model predic-                           based high-throughput phenotyping gene essentiality <br /> tions experimentally lab. KP RK drafted                            data. J Biol Chem 2007, 282(39):28791-28799. <br />                                                                             12.   Joyce AR, Reed JL, White  Edwards R, Osterman  Baba T, Mori <br /> manuscript. authors read approved final man-                          H, Lesely SA, Palsson BO, Agarwalla S: Experimental compu- <br /> uscript.                                                                          tational assessment conditionally essential genes                                                                                   Escherichia coli. J Bacteriol 2006, 188(23):8259-8271. <br />                                                                             13.   Becker SA, Feist  Mo ML, Hannum G, Palsson BO, Herrgard MJ: <br /> Additional material                                                               Quantitative prediction cellular metabolism  <br />                                                                                   straint-based models: COBRA Toolbox. Nature protocols <br />                                                                                   2007, 2(3):727-738. <br />  Additional File 1                                                          14.   Oberhardt MA, Puchalka J, Fryer KE, Dos Santos VA, Papin JA: <br />                                                                                   Genome-scale metabolic network analysis opportun- <br />  Supplementary tables. Tables S1, S3 S4. <br />                                                                                   istic pathogen Pseudomonas aeruginosa PAO1. J Bacteriol 2008. <br />  Click file                                                        15.   Hattori M, Okuno Y, Goto S, Kanehisa M: Development  [http://www.biomedcentral.com/content/supplementary/1752-                        <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> structure comparison method integrated anal- <br />  0509-2-67-S1.pdf]                                                                ysis chemical genomic information metabolic <br />                                      <br /> </body></html>