<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />  <br />                          novel pathway regulates memory plasticity SIRT1                          miR-134 <br />  <br />  <br />                          Abstract <br />                                NAD-dependent deacetylase Sir2 initially identified mediator replicative lifespan <br />                                budding yeast subsequently shown modulate longevity worms flies1,2.                                mammalian homologue, SIRT1, appears evolved complex systemic roles cardiac <br />                                function, DNA repair, genomic stability. Recent studies suggest functional relevance                                SIRT1 normal brain physiology neurological disorders.  unknown SIRT1 <br />                                plays role higher-order brain functions. report SIRT1 modulates synaptic plasticity <br />                                memory formation microRNA-mediated mechanism. Activation SIRT1 enhances, <br />                                loss function impairs, synaptic plasticity. Surprisingly, effects mediated                                post-transcriptional regulation CREB expression brain-specific microRNA, miR-134. <br />                                SIRT1 normally functions limit expression miR-134 repressor complex containing                                transcription factor YY1, unchecked miR-134 expression following SIRT1 deficiency results <br />                                regulated expression CREB BDNF, impairing synaptic plasticity. <br />                                findings demonstrate novel role SIRT1 cognition previously unknown <br />                                microRNA-based mechanism SIRT1 regulates processes. Furthermore,                                results separate branch SIRT1 signaling, SIRT1 direct role                                regulating normal brain function manner disparate cell survival functions, <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                demonstrating value potential therapeutic target treatment CNS disorders. <br />  <br />  <br />                                       mammalian Sir2 homolog SIRT1 involved variety complex processes relevant <br />                                       aging, including regulation oxidative stress, metabolism control, circadian <br />                                       rhythms1&#226;&#8364;&#8220;4. previously demonstrated SIRT1 gain function neuroprotective                                       overactive Cdk5 mutant SOD models neurodegeneration, relevant                                       Alzheimer&#226;&#8364;&#8482;s disease ALS, respectively5.  SIRT1 recently implicated <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       molecular pathways regulated cocaine6, suggesting  addition involvement <br />                                       neurogenesis neuroprotection, SIRT1 functions brain                                       described. <br />  <br />                                       directly evaluate physiological role SIRT1 learning memory, mutant mice <br />                                       lacking SIRT1 catalytic activity brain-specific manner (SIRT1&#206;&#8221;) generated                                       crossing mice carrying floxed SIRT1&#206;&#8221;ex4 allele7,8 Nestin-Cre transgenic mice.                                       tests associative memory, SIRT1&#206;&#8221; mice exhibited significant decrease freezing <br />                                       behavior evaluated contextual (Fig. 1a) tone-dependent (Fig. S2a) fear <br />                                       conditioning paradigms. Shock sensitivity locomotor activity did differ                                       SIRT1&#206;&#8221; mice littermate controls (Figs. 1b, S2b). SIRT1&#206;&#8221; mice showed similarly <br />                                       decreased memory performance novel object recognition task (Fig. 1c), relies <br />                                       hippocampus cortex9. time spent exploring objects training did <br />                                       significantly differ groups (Fig 1d). Morris water maze, SIRT1&#206;&#8221; mice <br />                                       displayed significantly increased escape latencies hidden platform paradigm (Fig. 1e), <br />                                       spending time target quadrant probe trial (Fig. 1f) compared HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       control mice, suggesting SIRT1 plays role spatial learning. Visual function                                       swimming ability affected SIRT1&#206;&#8221; mice (Figs. S2c,d).                                        results SIRT1 important role forms memory. <br />  <br />                                        used long-term potentiation (LTP) paradigm directly determine role                                       SIRT1 synaptic plasticity. LTP hippocampal CA1 neurons, induced &#206;&#184; burst <br />                                       (2&#195;&#8212;TBS) stimulation Schaffer collaterals control mice, abrogated SIRT1&#206;&#8221; <br />                                       mice, demonstrating requirement SIRT1 synaptic plasticity (Fig. 1g). CA1 neurons                                       SIRT1&#206;&#8221; mice exhibited normal basal synaptic transmission (Figs. S3a,b) compared                                       control mice. results demonstrate LTP deficits caused SIRT1 inactivation <br />                                       impaired synaptic transmission. <br />  <br />                                       brains SIRT1&#206;&#8221; mice grossly normal anatomy (data shown).  <br />                                       experiments using antibody synaptophysin (SVP), labels presynaptic <br />                                       terminals functional synapses10, revealed significant decreases SVP immunoreactivity <br />                                       hippocampal striatum radiatum SIRT1&#206;&#8221; mice, reduced SVP protein <br />                                       content SIRT1&#206;&#8221; hippocampus, compared controls (Figs. 1h 1i). Golgi <br />                                       impregnation demonstrated dendritic spine density CA1 pyramidal neurons HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       significantly decreased hippocampus SIRT1&#206;&#8221; mice (Fig. 1j). results suggest <br />                                       SIRT1 regulates synapse formation, synaptic plasticity, memory formation. <br />  <br />                                       Brain-derived neurotrophic factor (BDNF) <span id='am-1' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span>cAMP</span> response binding protein (CREB)                                       genes play critical roles synaptic plasticity modulating synapse formation11&#226;&#8364;&#8220; <br />                                       13. mRNA protein levels BDNF significantly decreased SIRT1&#206;&#8221; <br />                                       hippocampi compared controls (Fig. 2a). CREB binds BDNF promoters                                       plays key role activity-dependent regulation BDNF expression14&#226;&#8364;&#8220;17.                                       performed chromatin immunoprecipitation (ChIP) examine association                                       CREB BDNF promoters. Baseline binding CREB BDNF promoters 1, 2, 4 <br />                                       reduced SIRT1&#206;&#8221; hippocampus (Fig. 2b). reported previously15,17, binding                                       CREB BDNF promoters 1 4, increased contextual fear conditioning <br />                                       training. SIRT1&#206;&#8221; mice,  training-related increase BDNF promoter 1 4 <br />                                       binding CREB abolished (Fig. S4a). <br />  <br />  <br />                                             Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                              Page 3 <br />  <br />  <br />                                       possible explanation decreased CREB binding BDNF promoters observed                                       SIRT1&#206;&#8221; mice CREB itself downregulated. Consistent notion, CREB <br />                                       protein levels significantly reduced SIRT1&#206;&#8221; hippocampi (Fig. 2c, left).                                       overall CREB levels reduced SIRT1&#206;&#8221; mice, CREB phosphorylation HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       increased fear conditioning training (Fig. S4b).  contrast BDNF, mRNA <br />                                       levels CREB altered (Fig. 2c, right), suggesting CREB protein levels                                       downregulated SIRT1&#206;&#8221; brains posttranscriptional mechanisms. <br />  <br />                                       investigate mechanisms, considered known means posttranscriptional <br />                                       regulation gene expression, inhibition translation microRNA (miRNA). <br />                                       miRNAs expressed high levels brain18,19, involvement miRNA                                       numerous aspects normal abnormal brain function reported19,20.                                       compared expression miRNAs SIRT1&#206;&#8221; hippocampi littermate control <br />                                       hippocampi using miRNA microarray (Exqion , number brain- <br />                                       enriched miRNAs differed expression groups (Fig. S5). evaluated <br />                                       expression miRNAs appeared significantly altered microarray <br />                                       analysis, including upregulated downregulated miRNAs, using mature <br />                                       miRNA-specific quantitative PCR (qPCR). Consistent microarray, qPCR <br />                                       analyses confirmed significant changes expression miRNAs (Fig. 2d). <br />  <br />                                       miRNAs, miRNA-134 particular  specifically expressed                                       brain demonstrated negatively regulate dendritic spine formation HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       vitro21. Comparative genomic analyses 3&#226;&#8364;&#8482;UTR mouse CREB gene revealed <br />                                       partial complementary binding sites miR-134, site conserved <br />                                       (Fig. S6a). determine miR-134 directly binds inhibits translation                                       CREB mRNA, carried CREB activity luciferase reporter assay luciferase <br />                                       driven minimal promoter downstream CREB-binding elements. transfection                                       miR-134 CRE-Luc resulted significant decrease CREB activity cultured CAD <br />                                       cells, neural cell line (Fig. 2e). negative control, created miR-134 mutant <br />                                       containing mutations miR-134 seed region (mut-miR-134). Expression mut- <br />                                       miR-134 did affect CREB activity (Fig. 2e). expression miR-34b-5p miR-34c, <br />                                       miRNAs upregulated brains SIRT1&#206;&#8221; mice, effect CREB activity <br />                                       (Fig. S6b), indicating specific regulation CREB miR-134. Finally, miR-134 loss  <br />                                       function using locked-nucleic-acid (LNA)-modified oligonucleotide probe (LNA- <br />                                       miR-134), achieves specific knockdown endogenous miR-134, significantly <br />                                       enhanced CREB activity (Fig. 2e). results suggest miR-134 regulate CREB <br />                                       protein expression post-transcriptional mechanism. <br />  <br />                                       verify direct nature CREB inhibition miR-134, generated luciferase <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       reporter construct 3&#226;&#8364;&#8482;UTR CREB, containing predicted miR-134 <br />                                       target sequences, inserted downstream luciferase expression cassette (WT-CREB). <br />                                       expected, expression miR-134 WT-CREB significantly attenuated reporter <br />                                       expression (Fig. 2f). luciferase reporter construct linked 3&#226;&#8364;&#8482;UTR CREB <br />                                       containing multiple mutations miR-134 target sequences (mut-CREB)                                       refractory knockdown miR-134 (Fig. 2f). Expression mut-CREB resulted                                       significant increase luciferase activity, indicating repression reporter construct <br />                                       endogenous miR-134 (Fig. 2f). results indicate miR-134 attenuates CREB <br />                                       expression specific interaction target regions 3&#226;&#8364;&#8482;UTR CREB. <br />                                        overexpression miR-134, mut-miR-134, cultured CAD cells resulted <br />                                       reduced levels CREB protein relative scrambled miR control (Scr-miR; Fig. 2g).                                       contrast, delivery LNA-miR-134 resulted increased CREB protein levels compared                                       LNA-scrambled-miR (LNA-scr-miR) control (Fig. 2g), suggesting  normal <br />                                       conditions, miR-134 limiting effect CREB translation. <br />  <br />  <br />                                              Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                            Page 4 <br />  <br />  <br />                                       investigate SIRT1 modulates miR-134 transcription, conducted ChIP                                       anti-SIRT1 antibody examined association SIRT1 DNA sequences spanning <br />                                       ~5 Kb upstream Pre-miR-134 sequence. regions, R3 R7,                                       specifically amplified SIRT1 ChIP, suggesting SIRT1 associated HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       potential regulatory elements upstream miR-134 (Fig. 3a). specificity                                       interaction verified ChIP analyses SIRT1 constitutive knockout (KO) mouse <br />                                       brain tissue8, yielded dramatically reduced pull R3 R7 fragments <br />                                       (Fig. 3b). determine transcriptional modulating activities sites, cloned R3, <br />                                       R7, control R5 fragment upstream minimal promoter pCL3-promoter <br />                                       luciferase reporter construct. Interestingly, expression SIRT1 reporter constructs <br />                                       containing R3 R7, R5, significantly reduced, SIRT1 shRNA <br />                                       increased, expression luciferase reporter (Fig. 3c). Collectively, results <br />                                       indicate SIRT1 inhibits miR-134 expression directly binding distal inhibitory <br />                                       elements. <br />  <br />                                       verify contribution miR-134 modulation CREB SIRT1, created <br />                                       SIRT1 loss function CAD cells shRNA-mediated knockdown (shRNA- <br />                                       SIRT1) overexpression catalytically-inactive mutant SIRT1 (H363Y)22,                                       elevated level miR-134 cells (Fig. 3d). loss SIRT1 markedly <br />                                       inhibited CREB activity, effect reversed administration LNA-miR-134 <br />                                       (Fig. 3e). results provide strong evidence SIRT1 loss function attenuates <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       CREB activity miR-134-mediated posttranscriptional mechanism. searched R3 <br />                                       R7 regions transcription factor consensus binding motifs discovered Yin Yang 1 <br />                                       (YY1) binding sites R3 R7 fragments (Fig. 3f, ,                                       fragments upstream miR-134 coding region. YY1 ubiquitous highly- <br />                                       conserved transcription factor activate repress gene expression, depending                                       cellular context23. determine YY1 binds miR-134 regulatory sequences,                                       performed ChIP experiments using anti-YY1 anti-SIRT1 antibodies                                       overexpression knockdown YY1 CAD cells. anti-YY1 antibody <br />                                       immunoprecipitated R3 R7, R5, fragments, suggesting association                                       YY1 DNA elements fragments (Fig. 3f). knockdown YY1                                       reduced YY1 binding R3 R7 miR-134 promoter regions cultured CAD cells <br />                                       (Fig. 3f, lower left).  binding SIRT1 R3 R7 impaired YY1 <br />                                       knockdown (Fig. 3f, lower right), suggesting SIRT1 cooperates YY1 binding                                       upstream regulatory elements miR-134. Consistent idea, SIRT1 <br />                                       constitutive KO mice, YY1 binding R3 R7 miR-134 reduced (Fig. 3g).                                       determine functional consequence YY1 binding R3 R7, used pCL3- <br />                                       promoter luciferase reporter constructs containing R3, R5, R7 described Fig 3c. HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       YY1 overexpression repressed, YY1 knockdown potentiated,                                       luciferase activity driven R3 R7 (Fig. 3h). Conversely, YY1 abundance                                       influence luciferase activity driven R5. results suggest binding                                       YY1 R3 R7 represses transcription. investigated influence YY1                                       miR-134 CREB protein abundance CAD cells, finding overexpression YY1 <br />                                       reduced miR-134 expression, miR-134 abundance increased following                                       shRNA-mediated knockdown YY1 (Fig. 3i). Furthermore, CREB levels positively <br />                                       regulated YY1 expression (Fig. 3j). Collectively, observations support concept <br />                                       SIRT1 recruited YY1 DNA binding elements proteins collaborate <br />                                       suppress miR-134 expression. <br />  <br />                                       examined role miR-134 synaptic plasticity memory formation. <br />                                       miR-134 overexpressed area CA1 hippocampus lentiviral-mediated <br />                                       delivery (Fig. S7a). observed SIRT1&#206;&#8221; mice, overexpression miR-134 abrogated <br />                                       LTP CA1 neurons (Fig. 4a), basal synaptic transmission impaired (Fig. <br />  <br />  <br />                                              Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                            Page 5 <br />  <br />  <br />                                       S7b). miR-134 overexpression area CA1 resulted significant impairment long- <br />                                       term memory formation contextual fear-conditioning paradigm (Fig. 4b),                                       tone-dependent fear conditioning paradigm (Fig. S7c), task reliant amygdalar <br />                                       function24. miR-134 overexpression did affect shock sensitivity (Fig. S7d).  <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       miR-134 overexpression hippocampus closely mimics effects SIRT1 loss. <br />                                       data demonstrate upregulation miR-134 plays role synaptic <br />                                       plasticity impairment observed SIRT1&#206;&#8221; mice. <br />  <br />                                       assess, vivo, miR-134 upregulation underlies impairment LTP observed <br />                                       SIRT1&#206;&#8221; mice, examined effect miR-134 knockdown using injections                                       LNA-miR-134 probe hippocampal CA1 region (Figs. S8a,b). knockdown <br />                                       miR-134 restored LTP acute hippocampal slices SIRT1&#206;&#8221; hippocampi (Fig. 4c), <br />                                       LNA probe containing scrambled sequence (LNA-scr-miR) failed                                       ameliorate LTP defects SIRT1&#206;&#8221; mice.  injections LNA-miR-134,                                       LNA-scr-miR, SIRT1&#206;&#8221; mouse hippocampus markedly rescued contextual (Fig. 4d),                                       tone-dependent (Fig. S8c), fear memory formation. groups exhibited normal shock <br />                                       sensitivity (Fig. S8d). Examination hippocampal lysates harvested control                                       SIRT1&#206;&#8221; mice injected LNA-scr-miR revealed CREB BDNF protein <br />                                       levels reduced SIRT1&#206;&#8221; mice (Fig 4e).  LNA-miR-134 treatment restored <br />                                       CREB BDNF protein levels comparable control mice (Fig. 4e). <br />  <br />                                       data indicate impairments memory synaptic plasticity observed HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       SIRT1&#206;&#8221; mice   partly mediated regulation miR-134 consequent <br />                                       translational inhibition miR-134 target genes. SIRT1 normally functions cooperation <br />                                       YY1, potentially additional proteins, restrict expression miR-134  <br />                                       SIRT1 loss function, higher levels miR-134 negatively regulate synaptic <br />                                       plasticity translational block key plasticity proteins CREB (Fig. S1), <br />                                       subsequently mediates various synaptic plasticity impairments observed following <br />                                       SIRT1 loss function. <br />  <br />                                       previously demonstrated SIRT1 promotes neuronal survival age-dependent <br />                                       neurodegenerative disorders5. SIRT1 activity promotes <br />                                       plasticity memory direct manner mechanism distinct established <br />                                       neuroprotective activity. result demonstrates multi-faceted role SIRT1 brain, <br />                                       highlighting potential target treatment neurodegeneration                                       conditions impaired cognition, implications wider range CNS disorders. <br />  <br />                             Methods Summary <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       detailed methods, supplemental materials. SIRT1 KO                                       SIRT1&#206;&#8221;ex4/Nestin-Cre mice provided laboratory Leonard Guarente.                                       plasmids LNA-miRNA (Ambion) transfected mouse CAD cells using <br />                                       Lipofectamine 2000 (Invitrogen) according manufacturer&#226;&#8364;&#8482;s instructions. RNA <br />                                       extraction, purification, quantitative PCR performed according                                       manufacturer&apos;s protocols. Tissue cell lysis, protein concentrations, western blot <br />                                       analyses prepared described previously5. Immunoblot data quantified                                       measuring band intensity NIH imaging software SCAN gel digitizing <br />                                       software (Silk Scientific). Immunostaining performed described previously3                                       LSMeta10 software confocal microscope (Zeiss). behavioral testing                                       performed described previously5  data analyzed unpaired <br />                                       Student&#226;&#8364;&#8482;s t-test. way ANOVA used compare differences groups                                       time points. <br />  <br />  <br />  <br />  <br />  <br />  <br />                                       Figure 1. SIRT1 loss function impairs memory synaptic plasticity <br />                                         Freezing time SIRT1&#206;&#8221; mice littermate controls (Cont) 24 hr contextual fear <br />                                       conditioning training. (b) Shock sensitivities did differ control SIRT1&#206;&#8221; <br />                                       mice. (c) SIRT1&#206;&#8221;and control mice tested novel object discrimination 24 hrs                                       training. (d) Pre-test exploration times equal object. (e) Escape latencies                                       SIRT1&#206;&#8221; control mice examined Morris water maze hidden platform test. <br />                                       (f) Left panel: swimming time spent quadrant probe trial day 5 (T, <br />                                       target; L, left; O, opposite; R, right). Right panel: representative path tracings probe <br />                                       test. (g) LTP measurements performed CA1 region acute slices SIRT1&#206;&#8221; <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       mice controls. (h) SVP immunoreactivity SIRT1&#206;&#8221; control hippocampi.   <br />                                       Western blots hippocampal lysates examined SVP SIRT1&#206;&#8221; control mice. (j) <br />                                       density Golgi-impregnated hippocampal neuronal dendritic spines measured                                       SIRT1&#206;&#8221; control mice. Scale bar: 10 &#206;&#188;m. TBS: theta-burst stimulation, SVP: <br />                                       synaptophysin, OD: optical density. *p &lt; 0.05, **p &lt; 0.01, ***p &lt; 0.001. histograms <br />                                       represent average &#194;&#177; SEM. <br />  <br />  <br />  <br />  <br />                                              Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                      Page 9 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 2. BDNF CREB downregulated, miRNA-134 regulated, SIRT1&#206;&#8221; <br />                                       mice <br />                                         BDNF SIRT1&#206;&#8221; control mouse hippocampus. Left: mRNA levels. Right: <br />                                       Western blot. (b) Chromatin immunoprecipitation (ChIP) anti-CREB antibody                                       followed qPCR BDNF promoters 1, 2, 4. (c) CREB SIRT1&#206;&#8221; control <br />                                       mouse hippocampus. Left: Western blot. Right: mRNA levels. (d) qPCR selected <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       miRNAs hippocampi SIRT1&#206;&#8221; control mice. (e) CAD cells transfected <br />                                       plasmids LNA indicated CRE-Luc. (f) Luciferase reporter <br />                                       constructs containing wildtype (WT-CREB) mutated (mut-CREB) CREB <br />                                       3&#226;&#8364;&#8482;UTR region, cotransfected miR-134 control. (g) CREB protein expression                                       CAD cells transfection indicated constructs LNAs. Scr: scrambled, mut: <br />                                       mutant, LNA: locked-nucleic-acid (LNA), CRE-Luc: CREB activity reporter construct. **p <br />                                       &lt; 0.01; ***p &lt; 0.001; NS, significant. <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                        Page 10 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 3. SIRT1 regulates CREB miR-134 <br />                                         SIRT1 ChIP, genomic regions, R3 R7, corresponding base pairs <br />                                       1418~830 3427~2901 upstream miR-134, respectively, amplified wild <br />                                       type, SIRT1 total knockout (SIRT1KO), brain tissue. (b) miR-134 promoter <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       regions R3 R7 quantified SIRT1 ChIP qPCR. (c) Reporter constructs <br />                                       containing R3, R5, R7 regions upstream minimal promoter luciferase reporter <br />                                       transfected SIRT1, SIRT1 shRNA, vector. (d) qPCR miR-134                                       CAD cells transfection indicated plasmids. (e) CRE-luciferase reporter assay <br />                                       CAD cells. (f) ChIP performed CAD cells anti-YY1 anti-SIRT1 <br />                                       antibodies. (g) Anti-YY1 ChIP followed qPCR regions R3 R7. (h) Luciferase <br />                                       reporter constructs containing R3, R5, R7 regions transfected indicated <br />                                       plasmids.   miR-134 levels measured CAD cells qPCR transfection                                       indicated plasmids. (j) Western blotting measured CREB protein levels CAD cells <br />                                       transfections indicated plasmids. shRNA Cont: scrambled shRNA control. *p <br />                                       &lt; 0.05; **p &lt; 0.01; ***p &lt; 0.001; NS: significant. histograms represent average &#194;&#177; <br />                                       SEM. <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available PMC 2011 February 1. <br />                           Gao et al.                                                                                      Page 11 <br /> HHMI Author Manuscript <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                       Figure 4. MiR-134 knockdown rescues LTP memory impairments caused SIRT1 <br />                                       deficiency <br />                                         LTP measured acute hippocampal slices mice weeks injection                                       Lv-miR-134 Lv-Scr-miR. (b) Lv-miR-134 Lv-Scr-miR-injected mice tested <br />                                       contextual fear conditioning task. (c) Following hippocampal injections LNA- <br />                                       miR-134 LNA-scr-miR, LTP measured SIRT1&#206;&#8221; control mouse acute <br />                                       hippocampal slices. (d) Freezing behavior contextual fear conditioning task                                       examined SIRT1&#206;&#8221; control mice hippocampal injections LNA-miR-134                                       LNA-scr-miR. (e). Western blot CREB BDNF brain lysate vivo miR-134 <br />                                       knockdown. Control scrambled miR lentivirus: Lv-Scr-miR. **p &lt; 0.01, NS: significant. <br /> HHMI Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Nature. Author manuscript; available PMC 2011 February 1. <br />   <br /> </body></html>