<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2009 Genetics Society America <br /> DOI: 10.1534/genetics.109.106716 <br />  <br />  <br />  <br />     MAP Kinase Signaling Antagonizes PAR-1 Function Polarization <br />                 Early Caenorhabditis elegans Embryo <br />  <br />             Annina C. Spilker,*,1 Alexia Rabilotta,&#226;&#8364;&#160; Caroline Zbinden,* Jean-Claude Labbe&#194;&#180;&#226;&#8364;&#160;,2 <br />                                          Monica Gotta*,&#226;&#8364;&#161;,2,3 <br />     *Swiss Federal Institute Technology, Institute Biochemistry, ETH Ho&#194;&#168;nggerberg, 8093 Zu&#194;&#168;rich, Switzerland, &#226;&#8364;&#160;Institute Research <br />       Immunology Cancer, Department Pathology Cell Biology, Universite&#194;&#180; Montre&#194;&#180;al, Quebec H3C 3J7, Canada          &#226;&#8364;&#161; <br />           Department Genetic Medicine Development, Faculty Medicine, University Geneva, 1211 Geneva, Switzerland <br />                                                          Manuscript received June 26, 2009 <br />                                                       Accepted publication August 26, 2009 <br />  <br />  <br />                                                              ABSTRACT <br />                 PAR proteins (partitioning defective) major regulators cell polarity asymmetric cell division. <br />              par genes, par-1, encodes Ser/Thr kinase conserved yeast mammals.              Caenorhabditis elegans, par-1 governs asymmetric cell division ensuring polar distribution cell fate <br />              determinants. precise mechanisms PAR-1 regulates asymmetric cell division C. <br />              elegans remain elucidated. performed genomewide RNAi screen identified genes              specifically suppress embryonic lethal phenotype associated mutations par-1.              suppressors mpk-1, C. elegans homolog conserved mitogen activated protein (MAP) kinase <br />              ERK. Loss function mpk-1 restored embryonic viability, asynchronous cell divisions, asymmetric <br />              distribution cell fate specification markers, distribution PAR-1 protein par-1 mutant <br />              embryos, indicating genetic interaction functionally relevant embryonic development. <br />              Furthermore, disrupting function components MAPK signaling pathway resulted              suppression par-1 embryonic lethality. data indicates MAP kinase signaling <br />              antagonizes PAR-1 signaling early C. elegans embryonic polarization. <br />  <br />  <br />  <br />     SYMMETRIC cell division, process       mother cell divides different daughter cells, fundamental mechanism achieve cell diver- <br />                                                                             form complex PKC-3 (Etemad-Moghadam <br />                                                                             et al. 1995; Izumi et al. 1998; Tabuse et al. 1998; Hung <br />                                                                             Kemphues 1999). complex restricted <br /> sity development. use early embryo                         anterior cortex embryo response Caenorhabditis elegans model study asym-                     spatially defined actomyosin contractions occurring metric cell division. C. elegans cell embryo                        embryo fertilization (Goldstein Hird <br /> divides asymmetrically anteroposterior axis,                      1996; Munro et al. 2004). posterior cortex generating cells different sizes fates: larger               embryo devoid anterior PAR <br /> anterior daughter cell generate somatic tissues                  proteins occupied RING protein PAR-2 smaller posterior daughter cell generate                       Ser/Thr kinase PAR-1 (Guo Kemphues 1995; <br /> germline (Sulston et al. 1983).                                             Boyd et al. 1996; Cuenca et al. 2003). polarized, <br />    group proteins called PAR proteins (partitioning                    anterior posterior PAR proteins mutually <br /> defective) required asymmetric cell division C.                   exclude respective cortices <br /> elegans (Kemphues et al. 1988). Depletion                     (Etemad-Moghadam et al. 1995; Boyd et al. 1996; <br /> seven par genes (par-1 -6 pkc-3) leads defects                    Cuenca et al. 2003; Hao et al. 2006). Loss function asymmetric cell division embryonic lethality                         gene par-1, opposed loss par <br /> (Kemphues et al. 1988; Kirby et al. 1990; Tabuse et al.                     genes, results embryos exhibit subtle effects <br /> 1998; Hung Kemphues 1999; Hao et al. 2006). PAR-3                       polarized cortical domains occupied PAR-6 conserved proteins contain PDZ-domains                   PAR proteins (Cuenca et al. 2003). defects                                                                             gene associated symmetric division <br />   Supporting information available online http://www.genetics.org/    size, aberrant distribution cell fate specification <br /> cgi/content genetics.109.106716/DC1.                                   markers, altered cell fates daughter cells   1 <br />    Present address: Goodman Cancer Centre, McGill University, Montre&#194;&#180;al,    embryo, ultimately embryonic lethality (Kemphues <br /> Quebec H3A 1A3, Canada.                                                     et al. 1988; Guo Kemphues 1995). <br />   2 <br />     authors contributed equally work. <br />   3 <br />                                                                                PAR-1 controls asymmetric cell division cell fate <br />    Corresponding author: Universite&#194;&#180; Gene`ve, CMU/GEDEV, Rue <br /> Michel Servet, 1211 Geneva 4, Switzerland.                                  specification regulating localization E-mail: Monica.Gotta@medecine.unige.ch                                      highly similar CCCH-type zinc-finger proteins MEX-5 <br />  <br /> Genetics 183: 965&#226;&#8364;&#8220;977 (November 2009) <br />  966                                                    C. Spilker et al. <br />  MEX-6 (referred MEX-5/6). MEX-5 MEX-6                function, suppress embryonic lethal phenotype 70% identical <span id='am-30' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence                 par-1 mutant embryos. genes mpk-1, <br /> fulfill partially redundant functions embryo                C. elegans homolog highly conserved MAP <br /> (Schubert et al. 2000). wild-type animals, endoge-              kinase ERK. Closer analysis subsequently showed nous MEX-5 GFP fusions MEX-6 localize primar-               reduction function mpk-1 increases <br /> ily anterior embryo proteins          viability par-1 mutant embryos, reverts evenly distributed par-1 mutant embryos (Schubert               polarity phenotypes associated loss function et al. 2000; Cuenca et al. 2003). suggests wild-      par-1. data indicate mpk-1 antagonizes par-1 <br /> type animals, PAR-1 acts restricting MEX-5              activity regulate polarization asymmetric cell MEX-6 anterior embryo. precise               divisions early embryo. <br /> mechanism regulation known, elegant study performed MEX-5 indicates differential protein mobility anterior poste-                          MATERIALS METHODS <br /> rior cytoplasm cell embryo contributes                                                                       Strains: Strains maintained 15&#194;&#176; unless noted  <br /> asymmetry (Tenlen et al. 2008). increased mo-                wise cultured according Brenner (1974). N2 (Bristol) <br /> bility posterior cell embryo correlates          used wild-type strain following mutant par-1- par-4-dependent phosphorylation               alleles: LGI, dhc-1(or195); air-2(or207); LGIII, unc-79(e1068); <br /> MEX-5, kinase directly phosphorylating MEX-5                   mpk-1(ga111); LGV, sqt-3(sc8); par-1(it90); par-1(zu310). remains identified (Tenlen et al. 2008).                     statistical analyses carried using Student&#226;&#8364;&#8482;s t-test. <br />                                                                       Sequencing par-1(zu310): sequence coding region <br />    phenotypes associated loss par-1            par-1(zu310) animals lysed 1 hr 60&#194;&#176; buffer (50 <br /> function dependent function mex-5                mm KCl; 2.5 mm MgCl2; 10 mm Tris HCl pH 8.3; 0.45% NP40; <br /> mex-6.  loss function par-1 leads decreased       0.45% Tween 20; 0.01% gelatin) supplemented 0.4 mg/ml <br /> stability aberrant localization posterior cell          proteinase K, followed inactivation enzyme 15 <br /> fate specification marker PIE-1, protein usually         min 95&#194;&#176;. exons gene par-1 (WormBase 2008)                                                                    amplified using Taq polymerase according standard proce- <br /> inherited posterior daughter cell wild-type              dures. independent lysates sequenced animals ensures correct specification               amplified stretch. deviant region par-1(zu310) germline (Mello et al. 1996; Seydoux et al. 1996).            amplified independent wild-type par-1(zu310) <br /> decreased stability dependent mex-5/6 function            mutant lysates using following primers: 59-GTCGGAAA <br /> PIE-1 levels restored, albeit symmetrical distri-         TACGAAACAGCC (forward) 59-CCATCAAATGGTAGAC <br />                                                                    TTCCG (reverse). <br /> bution, mex-6(RNAi); mex-5(RNAi); par-1(b274) em-                  DIC microscopy: time-lapse microscopy, embryos bryos (Schubert et al. 2000; Cuenca et al. 2003;                   dissected gravid hermaphrodites mounted 2% <br /> Derenzo et al. 2003). Second, embryos lacking par-1                agarose pad M9 buffer. PE94 temperature-controlled stage <br /> function exhibit decreased amounts P granules                (Linkam Scientific) used ensure constant tempera- cell embryo, markers present               ture 22&#194;&#176; acquisition. Images acquired                                                                    Hamamatsu Orca ER digital camera mounted Leica mex-6(pk440); mex-5(zu199); par-1(RNAi) embryos              DM6000B microscope (Leica Microsystems), acquisi- <br /> comparable age (Cheeks et al. 2004).                      tion controlled Openlab software (Improvision). <br /> par-1(RNAi) cell embryos posterior cortical                Images acquired 10-sec intervals using Plan Apochro- <br /> domain occupied polarity protein PAR-2                   mat 63X/1.4 NA objectives. Image analysis performed <br /> extended anteriorly, compared wild-type em-                using ImageJ software (National Institutes Health, NIH). <br />                                                                       Genomewide RNAi-based screen: RNAi-based screen <br /> bryos (Cuenca et al. 2003). anterior extension             performed liquid medium using Biomek FX robot <br /> rescued embryos deficient par-1 mex-5/6            (Beckman Coulter, courtesy M. Peter) following method- <br /> (Cuenca et al. 2003). Taken  results                ology described previously (Labbe&#194;&#180; et al. 2006). embryonic <br /> indicate par-1 acts embryo                 lethality par-1(zu310) animals assessed incubation regulating localization activity             22&#194;&#176;. RNAi screen performed duplicate                                                                    clone yielded 1639 1389 candidates, respectively. proteins MEX-5 MEX-6.  remains                  candidates, 286 identified assays unclear proteins modulate PAR-1                  kept analysis. 286 bacterial clones tested function affect MEX-5/6 activity.                               feeding bacterial clones NGM plates containing 1 mm IPTG <br />    gain insight mechanisms par-1 function           par-1(zu310) animals L1 stage  Plates  embryo, sought identify genes act                cubated 15&#194;&#176; 2.5 days  L3/L4 stage) sub- <br />                                                                    sequently shifted 22&#194;&#176;. Adults removed day par-1 embryonic development.               progeny examined viability day removal. <br /> performed RNAi-based screen genetic interactors             Using method 286 clones tested duplicate temperature-sensitive allele par-1(zu310), using            feeding bacterial clones L1 larvae agar plates. assay embryonic lethal phenotype mutant                 yielded 60 candidates estimated readout. method proven successful previous             hatching progeny compared par-1(zu310) animals <br />                                                                    fed vector  60 candidates subsequently <br /> screens identify genes involved early embryonic              retested independent assays  plates assay) processes (Labbe&#194;&#180; et al. 2006; O&#226;&#8364;&#8482;Rourke et al. 2007).           embryonic lethal phenotype determined counting able identify genes  disruption           hatched vs. unhatched progeny compared control <br />                                            MPK-1 Antagonizes PAR-1 Function Polarity                                         967 <br />  <br /> plates (data shown). candidates resulted         targeting 45-kDa 55-kDa isoform mpk-1 viability significantly higher     used (Kamath et al. 2003). mek-2, lin-45, ksr-2 <br /> vector control (P , 0.05) kept analysis.    targeting feeding constructs generated cloning  <br /> remaining clones tested assays       length PCR fragments generated cDNA libraries  mek-2 <br /> plates . resulted confirmation clones    lin-45) cloned cDNA (clone yk343d6, kindly significantly suppressed par-1(zu310) (Table 1),      provided Y. Kohara, encoding ksr-2a) gateway- confirmed molecular identity sequencing.       adapted L4440 vector. <br /> Viability strains assessed using approach.      Confocal microscopy: Confocal imaging performed  assays performed alleles dhc-1(or195)        using SP2 confocal microscope (Leica Microsystems) equip- air-2(or207) carried 20&#194;&#176; 18&#194;&#176;, respectively.    ped Leica software. Antibodies recognizing PAR-1 (kindly <br />     Generation double mutants: generate mutant           provided J. Ahringer) PAR-3 (Labbe&#194;&#180; et al. 2006) strains unc-79(e1068) mpk-1(ga111); par-1(zu310) unc-          generated previously described (Etemad-Moghadam et al. <br /> 79(e1068); par-1(zu310), males allele par-1(zu310)     1995; Guo Kemphues 1995) affinity purified mated unc-79(e1068) mpk-1(ga111) unc-79(e1068)         serum nitrocellulose strip containing bacterially expressed <br /> maphrodites permissive temperature. animals        antigen. mouse anti-PIE-1 (Tenenhaus et al. 1998) resulting F1 progeny singled allowed lay eggs.         mouse anti-MEX-5 (Schubert et al. 2000) antibodies Subsequently 30 F2 progeny singled allowed lay         generously shared J. Priess. Oic1d4, mouse antibody <br /> eggs permissive temperature transferred         recognizing P granules, P4A1, mouse antibody recog- <br /> 25&#194;&#176;. par-1(zu310) homozygous F2 animals threw dead embryos         nizing PAR-3, obtained Developmental Studies restrictive temperature (escaping larvae developed    Hybridoma Bank (University Iowa). rabbit anti-PAR-1, <br /> sterile adults) determined dissecting scope. unc-       rabbit anti-PAR-3, mouse P4A1, Oic1D4 staining, samples <br /> 79(e1068) mpk-1(ga111); par-1(zu310) unc-79(e1068); par-        snap frozen liquid nitrogen subsequently fixed <br /> 1(zu310) homozygous animals identified F3           20 min MeOH. anti-PIE-1 anti-MEX-5 stainings <br /> progeny basis sluggish, uncoordinated         contrast, samples squashed 20 min 3% PFA <br /> ment. presence homozygous par-1(zu310) allele         solution humid chamber freeze-crack MeOH <br /> reconfirmed double mutant background sequenc-         incubation. <br /> ing  . test viability double mutants,         MeOH fixation, slides washed times PBS, <br /> staged L1 larvae indicated genotypes 22&#194;&#176;.       supplemented 0.2% Tween 20 (later referred 2.5&#226;&#8364;&#8220;3 days, adults cloned new plates    PBST) incubation primary antibodies overnight allowed lay eggs 12 hr removal. viability    4&#194;&#176;. washes PBST, slides incubated progeny counted 24 hr removal adults.      secondary antibody solution (including DAPI) 45 min     assess embryonic viability 25&#194;&#176;, L3/L4 animals       37&#194;&#176;. Slides subsequently washed times shifted 15&#194;&#176; 25&#194;&#176; 24 hr  circumvent sterility     mounted Mowiol (6 g glycerol, 2.4 g Mowiol (Calbiochem <br /> conferred mpk-1ts allele elevated temperature), trans-   475904), 6 ml H20, 12 ml 0.2 m Tris pH 8.5, 0.1% DABCO). ferred new plate, allowed lay eggs  12 hr   staining, second antibody used sample <br /> removal mothers. Embryonic viability counted $18        validate antibody accessibility imaged embryos. hr removal mothers.                                   secondary antibodies used Alexa488 Alexa568-coupled <br />     phenotypic analysis, staged L1 animals shifted     anti-rabbit anti-mouse antibodies (Molecular Probes). <br /> 22&#194;&#176; 2.5 days. reaching adulthood, animals           Quantifications stainings performed using ImageJ <br /> sacrificed examine progeny. examination ster-      software (NIH). Quantifications MEX-5 PIE-1 fluores- <br /> ility, progeny allowed develop adulthood         cence intensities performed cell embryos examined DIC microscopy presence          polarization P1, measuring half-moon shaped areas (spar- <br /> absence oocytes embryos. phenotypic analysis,       ing nucleus) dorsal ventral cytoplasm AB <br /> wild-type animals compared animals follow-        P1. Background areas comparable size measured <br /> ing genotypes: par-1(zu310) single mutants, unc-79(e1068)          outside embryo subtracted average fluores- <br /> mpk-1(ga111) double mutants, unc-79(e1068) mpk-1(ga111);        cence intensity. corrected values, ratios par-1(zu310) triple mutants. unc-79 encodes protein         AB:P1 (MEX-5) P1:AB (PIE-1) fluorescence intensities localizes&#226;&#8364;&#8221;according GFP reporter constructs ner-         calculated. distribution P granules vous ventral nerve cord (WormBase 2008).            judged eye. PAR-1 domain size PAR-3 domain size  unc-79 required locomotion animals         individual stainings determined imaging middle sensitivity specific volatile anesthetics (WormBase     plane embryo measuring respective length 2008). unc-79(e1068) did influence embryonic lethality      domain embryo, compared total embryo par-1(zu310) background  shown Table 2),           length. domain borders determined eye. PAR-1 <br /> conclude unc-79 influencing par-1 function.            PAR-3 domain size quantified separate experi- <br />    RNAi: RNAi experiments targeting different compo-           ments antibodies produced rabbit. visualize <br /> nents MAP kinase pathway feeding, staged animals         domain overlap, costainings rabbit anti-PAR-1 allowed develop L3/L4 stage 2.5 days 15&#194;&#176;      mouse anti-PAR-3 (P4A1) embryos perfor- standard NGM plates seeded bacterial strain OP50.      med imaged upper cortex embryos. subsequently washed times M9 remove           PAR-1 asymmetry cell stage assessed measur- <br /> bacteria transferred 1-mm IPTG plates seeded           ing PAR-1 average fluorescence intensity line HT115(DE3) bacteria transformed RNAi constructs.          cortex EMS/P2 cortex ABp/ <br /> Worms fed plates 1&#226;&#8364;&#8220;1.5 days progeny        P2. Values background corrected using comparable lines laid subsequently removed plates. Survival       cytoplasm ABp subtracted cortex values. progeny scored 24 hr removal parents.      gradient determined calculating ratio Constructs RNA-mediated interference components          cortices. MAP kinase pathway taken genomewide                 Germline phenotypes posterior gonad arm collection feeding constructs (Kamath et al. 2003), unless      observed DIC microscopy F1 progeny sedated <br /> noted  RNAi mpk-1, feeding construct          M9 buffer, supplemented 10 mm NaN3. generate F1 <br />  968                                                      C. Spilker et al. <br />  <br />                                                              TABLE 1 <br />                                            Embryonic viability candidate RNAi <br />  <br /> RNAia          par-1(zu310)      par-1(it90)      air-2(or207)      dhc-1(or195)                   NCBI KOGsb <br /> asd-2            34 6 20*           060            14 6 11           53 6 14*        RNA-binding protein, KH domain <br /> D2030.4          20 6 10*           060             769               8 6 6**        <span id='am-1' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>NADH</span>:<span id='am-6' about='obo:IMR_0200483' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00399" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Ubiquinone" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200483" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ubiquinone</span> oxidoreductase subunit, <br />                                                                                        NDUFB7/ B18 <br /> F36A2.7          34 6 18*           060             8 6 8**          17 6 13         Unknown <br /> C34B2.8          34 6 12*           060             4 6 2**           5 6 3**        NADH:ubiquinone oxidoreductase, subunit <br />                                                                                        B16.6, cell death-regulatory protein <br /> ntl-2            47   6   15*       060             1 6 1**           6 6 4**        Predicted transcriptional regulator <br /> F23H11.3         39   6   22*       060            13 6 10           46 6 22*        Succinyl-<span id='am-12' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> synthetase, subunit <br /> mpk-1            23   6   9*        060            15 6 12           25 6 15         Mitogen activated protein kinase <br /> K03B4.4          18   6   10*       060            12 6 14           17 6 12         Unknown <br /> mex-5             2   6   2          ND               ND                ND <br /> mex-6            35   6   20*        ND               ND                ND <br /> Vector            7   6   5         060            13 6 13           18 6 13 <br />    Values correspond number hatched progeny divided total progeny (n $ 9 plates assays) 6 standard  <br /> viation, par-1(it90) assays (n &#194;&#188; 3 plates assay), mex-5 mex-6 assays (n &#194;&#188; 6 plates assays). <br />         Gene cosmid number genes targeted RNAi. <br />    b <br />      NCBI KOGs (conserved orthologous groups) determined reciprocal best BlastP hits domains eukaryotic <br /> proteins highly divergent species. NCBI KOGS taken WormBase (WormBase 2008). <br />    * Values significantly higher vector control (Student&#226;&#8364;&#8482;s unpaired t-test: P , 0.05). <br />    ** Values lower vector control (Student&#226;&#8364;&#8482;s unpaired t-test: P , 0.05). <br />  <br />  <br />  <br /> progeny, mother animals kept 22&#194;&#176; L1 stage            temperature allow identify weak par-1 egg laying. resulting progeny kept 22&#194;&#176;       suppressors. Like previously published alleles par-1, <br /> adulthood (1 day post L4 stage). representative images <br />                                                                      par-1(zu310) mutant embryos exhibit symmetric <br /> shown acquired 633 objective DM6000B <br /> Leica microscope (Leica Microsystems, Heerbrugg, Switzerland)        division cell embryo (Figure 1, B; <br /> microscope using Openlab software (Improvision/Perkin                Figure 3A), synchronous cell divisions  <br /> Elmer, Coventry, U.K.) subsequently assembled          cell stage (Figure 1, C D; Figure 3B) aberrant <br /> manually using Photoshop.                                            localization different cell fate specification markers <br />                                                                      (Figure 1, E F; Figure 4). Shifting adult mothers                                                                      embryos permissive restrictive temperature                              RESULTS <br />                                                                      vice versa) allowed determine tempera- <br />    Characterization allele par-1(zu310): iden-             ture-sensitive period allele par-1(zu310) starts <br /> tify genes act par-1, sought                prior fertilization does end 28-cell <br /> perform RNAi-based screen suppressors                  stage (Figure S1). indicates allele par- <br /> embryonic lethality caused loss par-1 function. par-1          1(zu310) suitable tool study function par-1 <br /> gene function provided maternally, par-1                  early embryogenesis. <br /> homozygous mutant worms viable,                      gain insight molecular lesion underlying <br /> produce dead progeny.  chose                   par-1(zu310) mutation, sequenced coding <br /> temperature-sensitive allele par-1(zu310) (Kao et al.                region par-1 gene mutant identified 2004), use conditional alleles embryonic                single <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> change region encoding lethal genes facilitates growth large scale.            kinase domain. change T position 917 <br /> embryonic phenotypes par-1(zu310)                    respect cDNA prediction par-1a) causes characterized, performed phenotypic analysis              isoleucine asparagine change amino acid allele. par-1(zu310) mutant                    sequence, affecting predicted splice variants <br /> animals exhibit phenotypes closely resembling                  (Figure 1G) (WormBase 2008). Consistent pre- <br /> reported  previously characterized par-1 alleles           vious reports alleles par-1 carry point <br /> (Kemphues et al. 1988; Guo Kemphues 1995; Schubert               mutations kinase domain (Guo Kemphues <br /> et al. 2000). Embryos produced par-1(zu310) mutant                1995), PAR-1 protein detectable Western <br /> mothers  referred par-1 embryos)                 blot analysis embryonic extracts par-1(zu310) <br /> inviable 25&#194;&#176; (supporting information, Table S2)               animals elevated completely restrictive <br /> nearly inviable 22&#194;&#176; compared wild-type                    temperature (Figure S2 data shown). embryos (Table 1). semi-restrictive temperature               results consistent par-1(zu310) having residual <br /> 22&#194;&#176; chosen subsequent analyses screening              activity PAR-1 semi-restrictive temperature. <br />                                            MPK-1 Antagonizes PAR-1 Function Polarity                                            969 <br />  <br />                                                                                    Figure 1.&#226;&#8364;&#8221;Characterization allele par- <br />                                                                                 1(zu310). allele par-1(zu310) exhibits pheno- <br />                                                                                 types previously described alleles                                                                                 par-1 bears lesion par-1 coding sequence. <br />                                                                                  B) Images taken timelapse record- <br />                                                                                 ings, showing embryos undergoing divi- <br />                                                                                 sion. Note pronounced asymmetry cell <br />                                                                                 size daughter cells generated par- <br />                                                                                 1(zu310) mutant embryos compared                                                                                 wild-type embryos. (C D) Images time <br />                                                                                 lapse recordings showing cell embryos  <br />                                                                                 dergoing asynchronous division (wild type)                                                                                 synchronous division (par-1(zu310)). Asterisks <br />                                                                                 mark spindle poles, circles mark centrosomes  <br />                                                                                 fore nuclear envelope breakdown. (E F)                                                                                 cell-embryos stained anti-PIE-1 antibodies. <br />                                                                                 PIE-1 enriched posterior daughter cell <br />                                                                                 wild-type embryos evenly distrib- <br />                                                                                 uted par-1(zu310) mutants. Embryo length                                                                                  50 mm. (G) Sequence alignment                                                                                 kinase domain PAR-1 human KP78, <br />                                                                                 rat MARK2, mouse EMK, rat AMPK, yeast <br />                                                                                 SNF1 KIN1 according Guo Kemphues <br />                                                                                 (1995). par-1(zu310) contains lesion ki- <br />                                                                                 nase domain leading I306N exchange. orthologs carry nonpolar residues position opposed par-1(zu310). alignment generated ClustalW soft- <br /> ware. BoxShade backgrounds identical residues solid; similar residues shaded. Borders catalytic loop activation <br /> segment assigned according Panneerselvam et al. (2006). <br />  <br />  <br />    Identification genetic interactors par-1:                 sensitive mutant alleles affect embryonic develop- <br /> performed RNAi-based screen genes sup-                    ment, air-2(or207) dhc-1(or195) (Severson et al. <br /> press embryonic lethality caused temperature-              2000; Hamill et al. 2002). air-2 encodes C. elegans <br /> sensitive allele par-1(zu310), testing 93% clones           homolog Aurora B kinase dhc-1 encodes C. RNAi library 16,757 bacterial strains ex-                  elegans dynein heavy chain, loss function pressing dsRNA corresponding C. elegans gene                   gene affects early embryonic development (Kamath et al. 2003). 15,671 clones analyzed                    obvious consequence polarity cell fate specifi- <br /> duplicate, 286 candidates suppressed embryonic                    cation. suppressors identified screen, <br /> lethal phenotype par-1(zu310) mutant animals                RNAi disruption assays  Table S1).                                           suppress lethality air-2(or207) mutants,    verify candidates reliable                   RNAi disruption  asd-2 F23H11.3, <br /> suppressors embryonic lethal phenotype par-                 partially restored embryonic viability dhc-1(or195) <br /> 1(zu310), tested multiple assays feeding                mutants (Table 1). conclude bacterial clones agar plates par-1(zu310) mutant                effect asd-2 F23H11.3 par-1(zu310) embryonic <br /> L1 larvae  materials methods). led                viability unspecific. consistent identification bacterial clones result              previously published data indicating asd-2(RNAi) significant (P , 0.05) increase viability              able act genetic suppressor variety mutants <br /> par-1(zu310) mutant animals (Table 1).                                alleles, including dhc-1(or195), lit-1(or131) spn- <br />    suppression assays performed semi-                   4(or191) (O&#226;&#8364;&#8482;Rourke et al. 2007). <br /> restrictive temperature par-1(zu310). examine                  genes specifically suppress embryonic identified clones suppress strong loss par-1             lethal phenotype par-1(zu310) mutant, function, tested suppressors restore               K03B4.4, previously reported exhibit <br /> viability animals mutant par-1(it90),               embryonic lethality depletion RNAi <br /> presumptive null allele par-1 (Guo Kemphues                    wild-type RNAi-sensitive mutant backgrounds <br /> 1995). clone restore viability              (Fraser et al. 2000; Kamath et al. 2003; Ceron et al. <br /> par-1(it90) animals conditions (Table 1).              2007). consistent notion genes <br /> results suggest suppressors identified                like par-1, required maturation germ cells bypass requirement par-1.                              embryonic development. conditions <br />    investigate identified suppressors specific          used suppression  knockdown par-1 conditional mutant background, assessed                function genes leads,  subtle targeting candidates RNAi alters                embryonic lethality wild-type animals (data embryonic lethal phenotype temperature-              shown). <br />  970                                                  C. Spilker et al. <br />  <br />                                                                                Figure 2.&#226;&#8364;&#8221;Genetic interaction par-1 <br />                                                                             MAP kinase pathway.   Schematic rep- <br />                                                                             resentation genes involved MAP kinase sig- <br />                                                                             naling required vulva formation C. elegans <br />                                                                             (adapted Sundaram 2006). clarity,                                                                             core components depicted. Human ho- <br />                                                                             mologs C. elegans genes indicated paren- <br />                                                                             theses. (B) Reduction function                                                                             genes MAP kinase pathway increases viabil- <br />                                                                             ity par-1(zu310) mutant background. Values <br />                                                                             correspond average viability progeny <br />                                                                             percentage total progeny 6 standard devia- <br />                                                                             tion (n &#194;&#188; 9 plates independent assays, <br />                                                                             let-23(RNAi), corresponds                                                                             plates assay). Values significantly dif- <br />                                                                             ferent par-1(zu310) control animals                                                                             marked circle (P # 0.03). <br />  <br />  <br />    suppressors form heterogeneous group                 PAR-1-related kinase interacts MAP kinase <br /> genes judged information available          scaffold Ksr mammalian cells (Muller et al. 2001), <br /> molecular nature (Table 1). genes, F36A2.7           suggesting PAR-1 negatively regulates MAP kinase <br /> K03B4.4, clear homolog identified             signaling contexts. consistent species molecular function                 results reduction mpk-1 function leads proposed. D2030.4 C34B2.8 encode subunits                 suppression embryonic lethality exhibited NADH:ubiquinone oxidoreductase, component                  par-1(zu310) mutants suggests genetic mitochondrial electron transport chain,           interaction par-1 mpk-1 embryonic develop- key players cellular respiration. ntl-2 enc-        ment conserved mechanism. decided <br /> odes protein similar NOT2, component               address mpk-1 acts par-1 transcriptional regulatory complex CCR4  Reduc-              embryonic development C. elegans using genetic <br /> tion function ntl-2 synthetic lethal mu-         cell biological approaches. <br /> tation ksr-1, gene MAP kinase signaling                 RNA interference result target effects <br /> pathway, indicating gene impinge MAP            (Rual et al. 2007), confirmed genetic <br /> kinase signaling (Rocheleau et al. 2008). Finally, mpk-1,        interaction observed mpk-1(RNAi) par- gene encoding central kinase highly           1(zu310) generating double mutant strain carrying <br /> served MAP kinase pathway (Figure 2A), suppressed           allele par-1(zu310) temperature- <br /> embryonic lethality par-1 mutant background.                sensitive loss function allele mpk-1(ga111) (Lackner <br />    Taken  RNAi targeting            Kim 1998). allele mpk-1(ga111) carries point <br /> genes suppresses lethal phenotype par-1(zu310)            mutation near predicted MEK binding site animals. genes, specific       encoded MPK-1 protein proposed prone suppressing par-1(zu310).                                        activation wild-type counterpart (Lackner    mpk-1, suppressor par-1(zu310) embryonic                 Kim 1998). 4 6 4% unc- <br /> lethality: focused efforts defining           79(e1068); par-1(zu310) mutant embryos escaped embry- <br /> relationships par-1 suppressors           onic lethality semi-restrictive temperature, 38 6 9% identified, mpk-1. MAP kinase pathway essential           unc-79(e1068) mpk-1(ga111); par-1(zu310) mutant regulation variety different cellular pro-       embryos viable (P &#194;&#188; 3 3 10 8) (Table 2). Consistent <br /> cesses yeast mammals (Schlessinger 2000)             RNAi results, allele mpk- <br /> aberrant regulation pathway largely             1(ga111) suppress lethality par-1(zu310) <br /> implicated development progression             animals fully restrictive temperature (Table S2), <span id='am-37' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-38' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span>cer</span> (Dhillon et al. 2007). C. elegans, MAP kinase             phenotypic analysis demonstrated par-1-depen- <br /> signaling best characterized role cell       dent embryonic phenotypes suppressed fate specification formation vulva             mpk-1(ga111); par-1(RNAi) animals temperature <br /> (Lackner et al. 1994; Wu Han 1994). Interestingly,           (Figure S3). suggests residual PAR-1 <br /> abrogating function genes positively       activity required mpk-1 efficiently suppress par- <br /> negatively regulate MAP kinase signaling impairs em-             1(lf) mutants. results indicate functional <br /> bryonic development (Page et al. 2001; Hajnal                interaction par-1 mpk-1 important Berset 2002; Kao et al. 2004; Sonnichsen et al. 2005).           early embryonic development C. elegans.  gene par-1 reported act antag-                 components MAP kinase pathway <br /> onistically MAP kinase signaling vulva              genetically interact par-1: MPK-1 functions <br /> velopment (Kao et al. 2004; Yoder et al. 2004)             downstream conserved signaling pathway,                                             MPK-1 Antagonizes PAR-1 Function Polarity                                           971 <br />  <br />                            TABLE 2                                    efficiently abrogate function. conclude <br />         mpk-1(ga111) restores embryonic viability                  knockdown genes MAP kinase <br />                    par-1(zu310) mutants                               pathway suppresses embryonic lethal phenotype                                                                       allele par-1(zu310), indicating par-1(zu310) <br /> Genotype                                          Viability (%)       functionally interacts MAP kinase signaling <br />                                                                       pathway early embryonic development. <br /> Wild type                                            99   6   1 <br /> par-1(zu310)                                          4   6   4 <br />                                                                          Loss MPK-1 function restores cell division <br /> unc-79(e1076) mpk-1(ga111)                           97   6   2       asynchrony par-1(zu310) animals: Reduction func- <br /> unc-79(e1076)                                        93   6   2       tion gene mpk-1 genes MAP kinase <br /> unc-79(e1076); par-1(zu310)                           4   6   4       pathway leads significant increase viability unc-79(e1076) mpk-1(ga111); par-1(zu310)             38   6   9       par-1(zu310) background. address   Values average viability progeny 22&#194;&#176;          function mpk-1 early embryogenesis percentage total progeny 6 standard deviation (n &#194;&#188; 9               asymmetric cell division, examined reduc- <br /> plates assays). unc-79(e1068) mpk-1(ga111); par-             tion function gene mpk-1 restore polarity- <br /> 1(zu310) exhibits increased viability compared par-           dependent processes par-1(zu310) mutant animals. <br /> 1(zu310) (P &#194;&#188; 3 3 10 8) unc-79(e1068); par-1(zu310)                division C. elegans embryo, (2 3 10 8). <br />                                                                       mitotic spindle displaced posterior                                                                       embryo, leading generation daughter cells <br /> sought determine disrupting function                differ size. cells subsequently divide components MAP kinase pathway,                   asynchronously, anterior AB cell dividing  2 <br /> mpk-1, increase embryonic viability par-1(zu310)             min posterior P1 cell. Mutations par-1 <br /> animals. test  par-1(zu310) mutant animals               result defects processes, treated dsRNA targeting components                blastomeres cell stage equal size <br /> MAP kinase pathway. Disrupting function                    divide synchronously (Kemphues et al. 1988). MAP kinase pathway core components lin-45, mek-2,                 determined par-1(zu310) animals exhibit <br /> mpk-1 RNAi resulted robust increase                    similar early embryonic defects par-1 mutants <br /> viability par-1(zu310) mutant embryos (Figure 2B).                 grown semi-restrictive temperature 22&#194;&#176;. observed modest significant (P , 0.05; n &#194;&#188;                Quantitative analysis revealed anterior cell 3) increase viability disrupted function               significantly smaller par-1(zu310) embryos (56 6 2%, lin-3, let-60, ksr-2 RNAi (Figure 2B). contrast,         n &#194;&#188; 22) compared wild type (57 6 1%, n &#194;&#188; 24, <br /> RNAi directed components                P &#194;&#188; 0.04) (Figure 3A). phenotype sup- <br /> let-23, sem-5, sos-1, ksr-1 did alter viability par-        pressed mpk-1(ga111); par-1(zu310) double mutant <br /> 1(zu310) mutant embryos significant degree                embryos (56 6 2%, n &#194;&#188; 28) compared par- assays. genes impinge par-1               1(zu310) single mutants (P &#194;&#188; 0.96). suggests function early embryo, possible             loss function mpk-1 restore cell size cases, RNAi directed genes did             asymmetry par-1 mutants  given expected <br />  <br />  <br />                                                                                       Figure 3.&#226;&#8364;&#8221;Asymmetric cell division asyn- <br />                                                                                    chrony cell cycle timing par-1 mpk-1; <br />                                                                                    par-1 mutant embryos.   panel: Asymmetry <br />                                                                                    cell division determined size                                                                                    AB cell relative embryo length.                                                                                     ratio distance  <br />                                                                                    terior pole cleavage furrow total <br />                                                                                    length embryo determined time <br />                                                                                    cleavage completion. panel: ante- <br />                                                                                    rior cell par-1(zu310) significantly smaller <br />                                                                                    wild type. mpk-1(ga111); par-1(zu310)                                                                                    significantly different par-1(zu310). <br />                                                                                    (B) panel: Cell cycle timing assessed <br />                                                                                    measuring time completion                                                                                    cleavage furrow nuclear envelope <br />                                                                                    breakdown anterior (AB) posterior <br />                                                                                    (P1) blastomeres. panel: Quantification <br />                                                                                    cell cycle timing AB (light shading)                                                                                    P1 (dark shading) daughter blastomeres. par- <br />                                                                                    1(zu310) cells divide synchronously com- <br /> pared wild type. phenotype restored mpk-1(ga111); par-1(zu310) animals. values shown averages 6 standard <br /> deviation (n $ 20). Values significantly differed (P , 0.05) wild type marked circle, values different par-1 marked asterisk. <br />  972                                                   C. Spilker et al. <br />  <br />  <br />  <br />  <br />                                                                                 Figure 4.&#226;&#8364;&#8221;Cell fate specification partially <br />                                                                              restored mpk-1; par-1 double mutant embryos. <br />                                                                               L) Localization endogenous MEX-5                                                                               panel) PIE-1 (middle panel) cell stage <br />                                                                              embryos localization PAR-1 (red) P <br />                                                                              granules (green) cell stage embryos dur- <br />                                                                              ing centration  panel). Wild-type em- <br />                                                                              bryos mpk-1 embryos asymmetric <br />                                                                              localization MEX-5, PIE-1, P granules. <br />                                                                              par-1 embryos exhibit pronounced gradient <br />                                                                              MEX-5 PIE-1 daughter <br />                                                                              blastomeres P granules absent cell <br />                                                                              stage embryos. mpk-1; par-1 animals partially <br />                                                                              restored asymmetries MEX-5 PIE-1. P <br />                                                                              granules present, localized pos- <br />                                                                              terior. Embryo length  50 mm. (M N) <br />                                                                              Quantification fluorescence intensity ratio  <br />                                                                              tween cytoplasm AB P1 MEX-5                                                                              PIE-1 stained embryos. Circles indicate values dif- <br />                                                                              ferent wild type asterisks mark values <br />                                                                              different par-1 mutants (P , 0.05). <br />  <br />  <br />  <br />  <br /> suppression frequency 30&#226;&#8364;&#8220;40%,               link mpk-1 function PAR <br /> data acquired sufficient reveal suppres-           protein-dependent features asymmetric cell division. <br /> sion.  quantitative analysis cell division                Disrupting mpk-1 function restores cell fate specifi- <br /> timing cell stage indicated mpk-1(ga111)          cation par-1(zu310) mutants: PAR-1 required suppress synchronous division par-1(zu310)             segregation number cell fate determinants <br /> mutant embryos semi-restrictive temperature. wild-          anteroposterior axis cell embryo <br /> type embryos, nuclear envelope breakdown               regulating anterior localization MEX-5/6 <br /> blastomeres occurs 7:27 min (AB) 9:45 min (P1)                (Schubert et al. 2000; Cuenca et al. 2003; Tenlen et al. end cytokinesis daughter        2008). Consistent  MEX-5 localizes <br /> cells divide synchronously par-1(zu310) embryos           daughter cells par-1(zu310) embryos semi- <br /> (8:25 min 8:28 min), AB slightly slower            restrictive temperature (Figure 4, B). mea- P1 faster wild type (Figure 3B).                sured average ratio cytoplasmic MEX-5 fluores- <br /> Interestingly, phenotype P1 restored mpk-          cence intensity blastomeres cell stage <br /> 1(ga111); par-1(zu310) AB nucleus broke              wild-type embryos enrichment 2.2 6 8:30 min, P1 nuclear envelope breakdown             0.5 (n &#194;&#188; 35) anterior blastomere compared occurred 12 min, making division              posterior  ratio reduced 1.5 6 0.3 (n &#194;&#188; <br /> asynchronous wild-type embryos. assay,            35) par-1(zu310) embryos, value significantly differ- <br /> mpk-1(ga111) mutants behaved asynchro-                 ent wild type (P &#194;&#188; 10 7) (Figure 4M). Consistent <br /> nous wild type division timings 7:48 min             previously published data (Tenlen et al. 2008), <br /> (AB) 10:48 min (P1), consistent previously               MEX-5 distribution mpk-1(ga111) mutants did published results, suggesting reduction mpk-1             differ significantly wild-type animals (Figure 4, C <br /> function RNAi leads asynchronous division              M), fully restrictive temperature (Sonnichsen et al. 2005).  effect mpk-1           allele ga111 (Figure S5), MEX-5 fluorescence cell cycle timing unlikely primary cause       ratio measured mpk-1; par-1 double mutant embryos <br /> par-1(zu310) suppression, did correla-         (1.8 6 0.4; n &#194;&#188; 35) significantly higher par-1 <br /> tion cell cycle timing par-1 suppression       embryos (P &#194;&#188; 0.004) (Figure 4, B, D, M).  suppressors identified (data shown), slow-          fluorescence ratio significantly lower ing cell cycle P1 disrupting function       ratio measured wild-type embryos (P &#194;&#188; 0.001). unrelated genes (F33H2.5 div-1, Figure S4A) did           indicates disruption mpk-1 partially suppress par-1(zu310) phenotypes (Figure S4, B            restore localization MEX-5 par-1(zu310) ani- <br /> C; . results indicate mpk-1              mals MPK-1 acts upstream mechanism <br /> restore cell cycle timing asynchrony defect par-1          contributing MEX-5 localization stage. <br />                                        MPK-1 Antagonizes PAR-1 Function Polarity                                     973 <br />  <br />    MEX-5 required restrict germline determi-        regulation mex-5 mex-6 par-1(zu310) mutant <br /> nant PIE-1 posterior cell embryo        embryos  . <br /> subsequently P1 blastomere cell stage,         localization P granules PIE-1 pos- <br /> leading reciprocal pattern PIE-1 MEX-5            terior cell cell stage important gener- <br /> localization wild-type embryos (Figures 4, E)        ation germline fertility (Schubert et al. 2000). Consistent localization      adult animal. Accordingly, 81% (n &#194;&#188; 136) <br /> pattern MEX-5, posterior enrichment       par-1(zu310) animals escaped embryonic PIE-1 pronounced par-1(zu310) embryos         lethality semi-restrictive temperature developed (1.8 6 3, n &#194;&#188; 39) wild-type embryos (6.1 6 1.4, n &#194;&#188;     sterile adults, having visible germline (21%) <br /> 39, P &#194;&#188; 7 3 10 31) (Figure 4, E, F, N). observed       exhibiting cellular material body cavity posterior enrichment PIE-1 partially           resembled oocytes germ cells (60%) <br /> stored mpk-1; par-1 double mutants (2.5 6 0.9) (n .        correspond somatic gonad (Figure S6). <br /> 38, P &#194;&#188; 4 3 10 5) (Figure 4, F, H, N).          data consistent agametic phenotype reduction mpk-1 partially restore asymmetric       previously described alleles par-1 (Guo <br /> localization PIE-1 par-1 mutant embryos,       Kemphues 1995). cell fate specification consistent restored localization upstream     partially restored mpk-1(ga111); par-1(zu310) embryos <br /> localizing factor MEX-5.                                      compared par-1(zu310), asked    assessed localization P granules,     fertility likewise restored. contrast par-1(zu310) <br /> consist ribonucleoprotein particles  PIE-1,        animals, 31% mpk-1(ga111); <br /> localize posterior embryo segregate      par-1(zu310) double mutant animals escaped em- <br /> germline blastomeres (Seydoux Strome 1999;                bryonic lethality sterile (n &#194;&#188; 132), Schisa et al. 2001). P granules proposed           animals genotype contained oocytes destabilized anterior embryo MEX-5/6         embryos germline (Figure S6). data sug- <br /> activity (Schubert et al. 2000) PAR-1 suggested       gest restoring asymmetric localization PIE-1 mediate posterior stabilization restricting       early embryos enrichment P granules MEX-5/6 anterior (Cheeks et al. 2004). P       germ cell precursors late mpk-1; par-1 double mutant <br /> granule distribution highly dynamic       embryos physiologically relevant, correlates mitosis, scored localization prophase,     increased fertility progeny double mu- time centration nucleo-centrosomal com-         tants compared progeny par-1 animals. <br /> plex. par-1(zu310) cell embryos            Disrupting specifically MEX-6 function suppresses <br /> largely devoid P granules semi-restrictive temper-      par-1(zu310) lethality: finding mpk-1 restores <br /> ature, consistent patterns previously        mex-5/6-dependent phenotypes par-1(zu310) <br /> ported par-1 alleles: detected P granules     embryos prompted test disrupting mex-5 <br /> 24% (n &#194;&#188; 32) par-1(zu310) mutant embryos          mex-6 RNAi suppress par-1(zu310) mutants. <br /> observed 90% (n &#194;&#188; 38) wild-type controls       mex-5 mex-6 partially redundant (Figure 4, J). Interestingly, 69%         share sequence identity, specifically disrupted <br /> (n &#194;&#188; 34) mpk-1; par-1 embryos contained P granules,        function using RNAi targeting unique regions particles restricted           genes, previously described (Schubert et al. <br /> posterior (20/24) double mutant            2000). Interestingly, disrupting mex-5 <br /> cell embryos (Figure 4L). indicates mpk-1     did suppress par-1(zu310) phenotypes, disruption restore P granule stability par-1 mutant cell      mex-6 suppressed embryonic lethality par- <br /> embryos, P granules exhibit apolar distribution       1(zu310) mutant embryos semi-restrictive temper- <br /> mpk-1(ga111); par-1(zu310) mutant background, phe-          ature 22&#194;&#176; (Table 1). unclear effect notype observed mpk-1(ga111)        specific mex-6. genes partially <br /> par-1(zu310) single mutant embryos (Figure 4, J, K,           redundant functions, possibility disrupting L). phenotype reminiscent cell      mex-6 partially compromise, abrogate MEX-5/ <br /> embryos lacking function par-1; mex-5; mex-6           6 activity, remaining activity sufficient (Cheeks et al. 2004), ectopically             ensure normal embryogenesis downstream PAR-1. <br /> stabilized, symmetrically localized P granules.        Alternatively, exclude uncover terestingly, P granules restricted      new, specific function MEX-6 indepen- cells later stage mpk-1; par-1 double mutant    dent MEX-5. finding RNAi disruption embryos, indicating mpk-1 act asymmetrically      mpk-1 mex-6 results similar effects par-1(zu310) <br /> segregate P granules later step embryonic         animals consistent genes functioning development (data shown). Taken              direction regulate par-1-dependent processes. <br /> data reduction function mpk-1 restores           mpk-1 regulates size cortical PAR protein <br /> MEX-5 localization mex-5/6-dependent       domains zygote: Previous studies reported <br /> phenotypes, indicating mpk-1 act         par-1 required asymmetric <br />  974                                                       C. Spilker et al. <br />  <br />                                                                                             Figure 5.&#226;&#8364;&#8221;PAR-1 localization  <br />                                                                                          cell embryos   Quantification <br />                                                                                          PAR domain sizes cell embryos <br />                                                                                          stained PAR-3 PAR-1 ana- <br />                                                                                          phase telophase. PAR-3 domain size <br />                                                                                          (green bars) PAR-1 domain size (red <br />                                                                                          bars) determined independent ex- <br />                                                                                          periments plotted bar <br />                                                                                          diagram comparison. Zero percent cor- <br />                                                                                          responds anterior embryo, <br />                                                                                          100% corresponds posterior <br />                                                                                          pole. Values shown correspond domain <br />                                                                                          size averages 6 standard deviation. Values <br />                                                                                          significantly different wild <br />                                                                                          type marked circle values sig- <br />                                                                                          nificantly different par-1 aster- <br />                                                                                          isk. anterior posterior <br />                                                                                          domains extended par-1(zu310) ani- <br />                                                                                          mals. (B) Confocal images embryos dou- <br />                                                                                          ble stained PAR-3 PAR-1                                                                                          visualize domain extension em- <br />                                                                                          bryos. single-stained embryos <br />                                                                                          quantified  domains PAR-3 <br />                                                                                          PAR-1 extend par-1 mutant <br />                                                                                          embryos compared wild type (n &#194;&#188; 6). <br />                                                                                          Quantifications using mid- <br />                                                                                          plane confocal sections  panels). (C&#226;&#8364;&#8220; <br />                                                                                          G) Dorsoventral PAR-1 asymmetry                                                                                          restored mpk-1(ga111); par-1(zu310) mu- <br /> tant embryos. (C&#226;&#8364;&#8220;F) Confocal images anti-PAR-1 staining (red) late cell stage embryos. Anti-tubulin staining shown green, DAPI staining shown blue. (C, E, G) wild-type embryos mpk-1(ga111) mutants, PAR-1 localizes asymmet- <br /> rically cortex P2 (cell marked asterisk), protein localizing P2/EMS P2/ABp. <br /> (D, F, G) par-1(zu310) mpk-1(ga111); par-1(zu310) animals, PAR-1 localizes symmetrically dorsoventral axis. <br /> Embryo length  50 mm. (G) panel: Schematic representation cell stage embryo. cortices measured quantifications shown graph depicted red. (G) panel: Quantification ratio fluores- <br /> cence intensities cortex P2/EMS cortex P2/ABp. Values shown correspond average intensity <br /> 6 standard deviation. Values significantly different (P , 0.05) wild type marked circle. <br />  <br />  <br /> localization anterior posterior PAR proteins             PAR-3 extends posterior par- cell stage embryo, disrupting function leads            1(zu310) embryos: size PAR-3 domain expansion PAR-2 cortical domain                     58 6 5% (n &#194;&#188; 22) total embryo length par-1(zu310) <br /> anterior embryonic polarization (Cuenca et al.                 compared 52 6 7% (n &#194;&#188; 39) wild-type animals (P &#194;&#188; <br /> 2003). phenotype depends activity MEX-                 0.004) (Figure 5, B).  contrary 5/6, suggesting feedback loop functions                 observed PAR-1, localization PAR-3 polarization embryo (Cuenca et al. 2003).               anterior cortex restored mpk-1(ga111); par- determined localization PAR proteins                 1(zu310) double mutants compared par- asymmetric divisions C. elegans               1(zu310) embryos (57 6 5%; n &#194;&#188; 19; P &#194;&#188; 0.9) (Figure <br /> embryo quantifying length cortical PAR pro-                 5A). mpk-1(ga111) mutant embryos, domain size <br /> tein domains middle focal plane embryos fixed               PAR-1 (35 6 8%, n &#194;&#188; 13) PAR-3 (52 6 7%, n &#194;&#188; 19) anaphase telophase stages (Figure 5, B).              did significantly differ wild type (P . 0.5) agreement previous observations local-                 (Figure 5A). results indicate ization PAR-2 (Cuenca et al. 2003),                  localization PAR-1 PAR-3 perturbed par- <br /> PAR-1 extends anterior par-1(zu310)                     1(zu310) mutant embryos anterior exten- <br /> mutants: size PAR-1 domain average                  sion PAR-1, posterior extension PAR-3, <br /> 44 6 7% (n &#194;&#188; 22) total embryo length par-1(zu310)               depends mpk-1 activity, suggesting mpk-1 func- <br /> mutant embryos compared 32 6 4% wild-type                       tions upstream par-1 regulate localization embryos (n &#194;&#188; 22) (P &#194;&#188; 3 3 10 8) (Figure 5, B).                  posterior PAR proteins. <br /> Disrupting mpk-1 function partially suppress                  Following division, PAR proteins required <br /> phenotype size PAR-1 domain mpk-                     asymmetric division germline blastomeres 1(ga111); par-1(zu310) embryos 39 6 5% (n &#194;&#188; 23),                  adopt polarized distributions cells (Guo significantly smaller par-1(zu310)                   Kemphues 1995).  PAR polarity dif- <br /> embryos (P &#194;&#188; 0.01) (Figure 5A). Interestingly,                ferentially established cell embryo compared <br />                                         MPK-1 Antagonizes PAR-1 Function Polarity                                    975 <br />  <br />                                                                Taken  results provide novel link                                                                PAR-1 MAP kinase signaling regulation                                                                asymmetric division C. elegans embryo. <br />                                                                   understand loss mpk-1 function suppresses <br />                                                                par-1, analyzed par-1 phenotypes embryos <br />                                                                mutant par-1 mpk-1. Importantly, precise <br />                                                                regulation MAP kinase signaling required    Figure 6.&#226;&#8364;&#8221;Models depicting possible genetic interactions    proper oocyte maturation embryonic cell fate mpk-1 polarity regulators. mpk-1 act      specification (Page et al. 2001; Kao et al. 2004). Loss <br /> stream  , downstream (B), parallel (C) par-1     mpk-1 function embryo leads embryonic antagonize function.                                    lethality defects cell cycle timing (Kamath et al. <br />                                                                2003; Simmer et al. 2003; Sonnichsen et al. 2005) later cells P lineage, genes         indicating mpk-1 function required oogenesis <br /> ported required PAR polarity cell         embryogenesis. mpk-1 fulfills functions <br /> stage, cell stage embryos (Basham           remains elucidated. analysis shows <br /> Rose 1999). tested mpk-1 sup-             embryos mutant mpk-1 par-1, presses par-1 phenotypes later embryonic stages          polarity phenotypes par-1 partially restored, <br /> determining restore proper PAR-1                indicating mpk-1 antagonizes par-1-dependent sig- <br /> localization P2 blastomere cell stage par-      naling embryonic polarity links mpk-1 <br /> 1(zu310) mutants. assessed quantifying         function polarity. PAR-1 levels comparable fluorescence intensity PAR-1 cortices        wild type mpk-1(ga111); par-1(zu310) embryos,  <br /> P2 shares neighboring blastomeres (Figure 5G,         dicating MPK-1 does suppress par-1(zu310) . levels PAR-1 wild-type      regulating protein levels (Figure S2). Notably, mpk-1 <br /> cell stage embryos 3.1 6 0.9 (n &#194;&#188; 42) higher        mutant embryos did exhibit apparent defects <br /> cortex P2 EMS cortex           cell fate specification exception cell cycle <br /> P2 ABp (Figure 5, C G). difference lost       asynchrony. suggests mpk-1 par-1(zu310) animals, displayed ratio 1.1 6      negative regulator par-1 activity polarity. <br /> 0.4 (n &#194;&#188; 42; P &#194;&#188; 3 3 10 10) (Figure 5, D G).                  data indicate mpk-1 impinges PAR-1 <br /> Interestingly, asymmetric enrichment PAR-1           localization cell stage par-1(zu310) mutant em- P2/EMS boundary restored mpk-1(ga111);          bryos loss mpk-1 function bypass par-1(zu310) double mutant embryos, exhibited          requirement PAR-1 activity, asymmetric distribu- <br /> fluorescence ratio 1.1 6 0.5 (n &#194;&#188; 38; P &#194;&#188; 0.9) mpk-     tion MEX-5 restored mpk-1(ga111); par- <br /> 1(ga111) mutants embryos did differ significantly          1(zu310) embryos fully restrictive temperature (data wild-type embryos assays (2.8 6 1.8, n &#194;&#188; 35,     shown). Interestingly, loss function mpk-1 P &#194;&#188; 0.5) (Figure 5, E&#226;&#8364;&#8220;G). conclude           semi-restrictive temperature par-1(zu310) animals disruption mpk-1 restore PAR-1 protein            partially restored asymmetric distribution localization cell stage par-1(zu310) embryos,        MEX-5, suppressed phenotypes does modulate PAR-1 protein distribution later          reported depend genes mex-5 stage embryos.                                                 mex-6, distribution PIE-1 precise <br />                                                                <span id='am-11' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> cortical domains occupied                                                                posterior PAR proteins cell stage.  <br />                       DISCUSSION <br />                                                                mpk-1(ga111); par-1(zu310) mutant embryos exhibited <br />    performed genomewide screen genetic             ectopic stabilization apolar distribution P gran- <br /> teractors par-1 mutant allele better define        ules, phenotype reminiscent mex-6(pk440); mex-5(zu199); <br /> function PAR-1 embryonic polarity. ap-           par-1(RNAi) embryos (Cheeks et al. 2004). Taken  <br /> proach, identified genes partially             gether, results support model mpk-1 <br /> specifically suppress conditional mutant allele       act upstream par-1 impinge distribu- <br /> gene par-1, suggesting act       tion activity MEX-5/6 (Figure 6A). loss par-1 fulfilling functions. According mole-    mpk-1 function does suppress phenotypes <br /> cular nature, suppressors cluster            associated reduction PAR-1 function, prominent pathway, suggesting act          model implies cellular processes different mechanisms suppress par-1 mutant.         sensitive reduction PAR-1 activity.  genetic interactors par-1 identified mpk-    results compatible alternative models 1, gene encoding MAP kinase. par-1           mpk-1 act downstream parallel par-1 <br /> genetically linked MAP kinase signaling pathway         (Figure 6, B C), consistent interaction mpk-1 par-1               previous reports indicating MPK-1/Erk activation <br /> impinges aspects asymmetric cell division.       depends PAR-1/C-TAK1 activity (Muller et al. 2001; <br />  976                                                              C. Spilker et al. <br />  <br /> Yoder et al. 2004). results mpk-1                           grateful Anne Pacquelet sharing initial mex-5/6 suppression data <br /> regulates PAR-1 localization, models entail                      insightful advice, Alexander Schultze technical help,                                                                              Marc Therrien Christian Rocheleau comments regulation occurs indirectly negative                        manuscript. thank laboratories M. Gotta, J.-C. Labbe&#194;&#180;,  <br /> feedback loop MEX-5/6,                               Hajnal, M. Hengartner, M. Peter stimulating discussions. reported control PAR protein localization (Schubert                       strains used study provided Caenorhabditis <br /> et al. 2000; Cuenca et al. 2003). Biochemical analysis                  Genetics Center, funded National Institutes Health. required resolve details epistatic                         work supported Boehringer Ingelheim Foundation <br />                                                                              studentship S., Cole Foundation studentship R., relationship.                                                                Canada Research Chair Cell Division Differentiation    Interestingly, embryos mutant E2F DP                          Canadian Institutes Health Research grant  158715 J.-C.L., <br /> transcription factor-related genes efl-1 dpl-1,                    funds Swiss Federal Institute Technology (ETH) lead sustained activation MPK-1 C. elegans                      Swiss National Science Foundation M.G. Institute germline, exhibit Mex (Muscle excess) phenotype                         Research Immunology Cancer supported                                                                              Canadian Center Excellence Commercialization Research, <br /> miniscent mutations mex-5 mex-6 (Page et al.                        Canada Foundation Innovation, Fonds Recherche <br /> 2001). morphological defects efl-1/dpl-1                     Sante&#194;&#180; du Que&#194;&#180;bec. <br /> mex-5 mutants partially suppressed reduction function components MAP kinase pathway <br /> (Page et al. 2001). contrast results, indi- <br />                                                                                                    LITERATURE CITED <br /> cates increase MAP kinase activity efl-1 <br /> phenocopies loss function mex-5/6, suggesting                        Basham, S. E., L. S. Rose, 1999 Mutations ooc-5 ooc-3 dis- <br />                                                                                  rupt oocyte formation reestablishment asymmetric mpk-1 mex-5/6 act antagonistically em-                           PAR protein localization cell Caenorhabditis elegans em- <br /> bryonic development. possible explanation                           bryos. Dev. Biol. 215: 253&#226;&#8364;&#8220;263. <br /> apparent discrepancy data                         Benton, R., D. St Johnston, 2003 Drosophila PAR-1 14&#226;&#8364;&#8220; <br />                                                                                  3-3 inhibit Bazooka/PAR-3 establish complementary cortical gain loss function mpk-1                         domains polarized cells. Cell 115: 691&#226;&#8364;&#8220;704. phenotypic outcome. Alternatively, recent study                       Boyd, L., S. Guo, D. Levitan, D. T. Stinchcomb K. J. Kemphues, <br /> using gene expression profiling proposed muta-                              1996 PAR-2 asymmetrically distributed promotes associ- <br />                                                                                  ation P granules PAR-1 cortex C. elegans em- <br /> tions mpk-1 efl-1/dpl-1 actually control dif-                         bryos. Development 122: 3075&#226;&#8364;&#8220;3084. <br /> ferent target genes C. elegans germline (Leacock                      Brenner, S., 1974 genetics Caenorhabditis elegans. Genetics Reinke 2006), suggesting mpk-1 efl-1 act                        77: 71&#226;&#8364;&#8220;94. <br />                                                                              Ceron, J., J. F. Rual,  Chandra, D. Dupuy, M. Vidal et al., <br /> different mechanisms. Future identification addi-                             2007 Large-scale RNAi screens identify novel genes interact <br /> tional genes regulate mpk-1- mex-5/6-depen-                          C. elegans retinoblastoma pathway splicing- <br /> dent phenotypes help address  Taken                                 related components synMuv B activity. BMC Dev. Biol. 7: 30. <br />                                                                              Cheeks, R. J., J. C. Canman, W. N. Gabriel, N. Meyer, S. Strome  indicates interplay                              et al., 2004 C. elegans PAR proteins function mobilizing <br /> MPK-1 activity MEX-5/6 function complex                            stabilizing asymmetrically localized protein complexes. Curr. warrants analysis.                                                   Biol. 14: 851&#226;&#8364;&#8220;862. <br />                                                                              Cuenca,  ,  Schetter, D. Aceto, K. Kemphues G. Seydoux, <br />    par-1 previously shown function genetically                            2003 Polarization C. elegans zygote proceeds distinct <br /> antagonistic MAP kinase signaling                               establishment maintenance phases. Development 130: <br /> velopment C. elegans vulva (Kao et al. 2004; Yoder                        1255&#226;&#8364;&#8220;1265. <br />                                                                              DeRenzo, C., K. J. Reese G. Seydoux, 2003 Exclusion germ <br /> et al. 2004) homolog kinases species                           plasm proteins somatic lineages cullin-dependent degra- reported target MAP kinase pathway compo-                                dation. Nature 424: 685&#226;&#8364;&#8220;689. <br /> nents (Muller et al. 2001; Benton St Johnston                            Dhillon,  S., S. Hagan, O. Rath W. Kolch, 2007 MAP ki- <br />                                                                                  nase signalling pathways cancer. Oncogene 26: 3279&#226;&#8364;&#8220;3290. <br /> 2003). antagonistic relationship consistent                     Etemad-Moghadam, B., S. Guo K. J. Kemphues, results reduction function mpk-1                        1995 Asymmetrically distributed PAR-3 protein contributes compensate loss function par-1                          cell polarity spindle alignment early C. elegans embryos. <br />                                                                                  Cell 83: 743&#226;&#8364;&#8220;752. <br /> embryonic development indicate                        Fraser,  G., R. S. Kamath, P. Zipperlen, M. Martinez-Campos, M. antagonistic behavior par-1 MAP                              Sohrmann et al., 2000 Functional genomic analysis C. ele- <br /> kinase pathway reflects conserved mechanism.                               gans chromosome systematic RNA interference. Nature <br />                                                                                  408: 325&#226;&#8364;&#8220;330. <br /> core components transmitting MAP kinase signaling                        Goldstein, B., S. N. Hird, 1996 Specification anteroposte- <br /> conserved different cell types develop-                              rior axis Caenorhabditis elegans. Development 122: 1467&#226;&#8364;&#8220;1474. <br /> mental processes C. elegans. work high-                        Guo, S., K. J. Kemphues, 1995 par-1, gene required estab- <br />                                                                                  lishing polarity C. elegans embryos, encodes putative Ser/ <br /> lights contribution MAP kinase signaling                             Thr kinase asymmetrically distributed. Cell 81: 611&#226;&#8364;&#8220;620. <br /> asymmetric cell division, probe                    Hajnal, , T. Berset, 2002 C. elegans MAPK phosphatase mechanistic relationships MAP kinase                                 LIP-1 required G(2)/M meiotic arrest developing oo- <br />                                                                                  cytes. EMBO J. 21: 4317&#226;&#8364;&#8220;4326. <br /> signaling pathways controlling cell polarity.                      Hamill, D. R.,  F. Severson, J. C. Carter B. Bowerman, <br />                                                                                  2002 Centrosome maturation mitotic spindle assembly    thank Ken Kemphues sharing par-1 mutant strains; Jim Priess            C. elegans require SPD-5, protein multiple coiled-coil  Julie Ahringer providing anti-MEX-5, anti-PIE-1, anti-PAR-           mains. Dev. Cell 3: 673&#226;&#8364;&#8220;684. <br /> 1 antibodies; Yuji Kohara sharing cDNA encoding ksr-2;        Hao, Y., L. Boyd G. Seydoux, 2006 Stabilization cell polarity <br /> Matthias Peter generously providing access robot unit.         C. elegans RING protein PAR-2. Dev. Cell 10: 199&#226;&#8364;&#8220;208. <br />                                                MPK-1 Antagonizes PAR-1 Function Polarity                                                977 <br />  <br /> Hung, T. J., K. J. Kemphues, 1999 PAR-6 conserved               Rual, J. F., N. Klitgord G. Achaz, 2007 Novel insights RNAi <br />     PDZ domain-containing protein colocalizes PAR-3                target effects using C. elegans paralogs. BMC Genomics 8: 106. <br />     Caenorhabditis elegans embryos. Development 126: 127&#226;&#8364;&#8220;135.         Schisa, J. , J. N. Pitt J. R. Priess, 2001 Analysis RNA asso- <br /> Izumi, Y., T. Hirose, Y. Tamai, S. Hirai, Y. Nagashima et al.,               ciated P granules germ cells C. elegans adults. Devel- <br />     1998 atypical PKC directly associates colocalizes              opment 128: 1287&#226;&#8364;&#8220;1298. <br />     epithelial tight junction ASIP, mammalian homologue       Schlessinger, J., 2000 Cell signaling receptor tyrosine kinases. <br />     Caenorhabditis elegans polarity protein PAR-3. J. Cell Biol.          Cell 103: 211&#226;&#8364;&#8220;225. <br />     143: 95&#226;&#8364;&#8220;106.                                                         Schubert, C. M., R. Lin, C. J. Vries, R. H. Plasterk J. R. <br /> Kamath, R. S.,  G. Fraser, Y. Dong, G. Poulin, R. Durbin et al.,           Priess, 2000 MEX-5 MEX-6 function establish soma/ <br />     2003 Systematic functional analysis Caenorhabditis ele-           germline asymmetry early C. elegans embryos. Mol. Cell 5: <br />     gans genome using RNAi. Nature 421: 231&#226;&#8364;&#8220;237.                             671&#226;&#8364;&#8220;682. <br /> Kao, G., S. Tuck, D. Baillie M. V. Sundaram, 2004 C. elegans         Severson,  F., D. R. Hamill, J. C. Carter, J. Schumacher B. <br />     SUR-6/PR55 cooperates LET-92/protein phosphatase 2A                 Bowerman, 2000 aurora-related kinase AIR-2 recruits <br />     promotes Raf activity independently inhibitory Akt phos-          ZEN-4/CeMKLP1 mitotic spindle metaphase  <br />     phorylation sites. Development 131: 755&#226;&#8364;&#8220;765.                             quired cytokinesis. Curr. Biol. 10: 1162&#226;&#8364;&#8220;1171. <br /> Kemphues, K. J., J. R. Priess, D. G. Morton N. S. Cheng,             Seydoux, G., C. C. Mello, J. Pettitt, W. B. Wood, J. R. Priess et al., <br />     1988 Identification genes required cytoplasmic localiza-          1996 Repression gene expression embryonic germ lin- <br />     tion early C. elegans embryos. Cell 52: 311&#226;&#8364;&#8220;320.                      eage C. elegans. Nature 382: 713&#226;&#8364;&#8220;716. <br /> Kirby, C., M. Kusch K. Kemphues, 1990 Mutations par           Seydoux, G., S. Strome, 1999 Launching germline Cae- <br />     genes Caenorhabditis elegans affect cytoplasmic reorganiza-           norhabditis elegans: regulation gene expression early germ <br />     tion cell cycle. Dev. Biol. 142: 203&#226;&#8364;&#8220;215.               cells. Development 126: 3275&#226;&#8364;&#8220;3283. <br /> Labbe&#194;&#180;, J. C.,  Pacquelet, T. Marty M. Gotta, 2006 ge-           Simmer, F., C. Moorman,  M. van der Linden, E. Kuijk, P. V. van <br />     nomewide screen suppressors par-2 uncovers potential reg-         den Berghe et al., 2003 Genome-wide RNAi C. elegans using <br />     ulators PAR protein-dependent cell polarity Caenorhabditis         hypersensitive rrf-3 strain reveals novel gene functions. PLoS <br />     elegans. Genetics 174: 285&#226;&#8364;&#8220;295.                                          Biol. 1: E12. <br /> Lackner, M. R., S. K. Kim, 1998 Genetic analysis Caeno-       Sonnichsen, B., L. B. Koski,  Walsh, P. Marschall, B. Neumann <br />     rhabditis elegans MAP kinase gene mpk-1. Genetics 150: 103&#226;&#8364;&#8220;117.          et al., 2005 genome RNAi profiling early embryogenesis <br /> Lackner, M. R., K. Kornfeld, L. M. Miller, H. R. Horvitz S. K.           Caenorhabditis elegans. Nature 434: 462&#226;&#8364;&#8220;469. <br />     Kim, 1994 MAP kinase homolog, mpk-1, involved ras-           Sulston, J. E., E. Schierenberg, J. G. White J. N. Thomson, <br />     mediated induction vulval cell fates Caenorhabditis elegans.       1983 embryonic cell lineage nematode Caenorhab- <br />     Genes Dev. 8: 160&#226;&#8364;&#8220;173.                                                   ditis elegans. Dev. Biol. 100: 64&#226;&#8364;&#8220;119. <br /> Leacock, S. W., V. Reinke, 2006 Expression profiling MAP          Sundaram, M. V., 2006 RTK/Ras/MAPK signaling. WormBook, Ed. <br />     kinase-mediated meiotic progression Caenorhabditis elegans.           C. elegans Research Community, WormBook, doi/10.1895/ <br />     PLoS Genet. 2: e174.                                                     wormbook.1.80.1, http://www.wormbook.org. <br /> Mello, C. C., C. Schubert, B. Draper, W. Zhang, R. Lobel et al.,         Tabuse, Y., Y. Izumi, F. Piano, K. J. Kemphues, J. Miwa et al., <br />     1996 PIE-1 protein germline specification C.                  1998 Atypical protein kinase C cooperates PAR-3 estab- <br />     elegans embryos. Nature 382: 710&#226;&#8364;&#8220;712.                                    lish embryonic polarity Caenorhabditis elegans. Development <br /> Muller, J., S. Ory, T. Copeland, H. Piwnica-Worms D. K.                  125: 3607&#226;&#8364;&#8220;3614. <br />     Morrison, 2001 C-TAK1 regulates Ras signaling phosphor-           Tenenhaus, C., C. Schubert G. Seydoux, 1998 Genetic  <br />     ylating MAPK scaffold, KSR1. Mol. Cell 8: 983&#226;&#8364;&#8220;993.                   quirements PIE-1 localization inhibition gene expres- <br /> Munro, E., J. Nance J. R. Priess, 2004 Cortical flows powered            sion embryonic germ lineage Caenorhabditis elegans. <br />     asymmetrical contraction transport PAR proteins establish          Dev. Biol. 200: 212&#226;&#8364;&#8220;224. <br />     maintain anterior-posterior polarity early C. elegans     Tenlen, J. R., J. N. Molk, N. London, B. D. Page J. R. Priess, <br />     embryo. Dev. Cell 7: 413&#226;&#8364;&#8220;424.                                            2008 MEX-5 asymmetry cell C. elegans embryos requires <br /> O&#226;&#8364;&#8482;Rourke, S. M., M. D. Dorfman, J. C. Carter B. Bowerman,                PAR-4- PAR-1-dependent phosphorylation. Development <br />     2007 Dynein modifiers C. elegans: light chains suppress           135: 3665&#226;&#8364;&#8220;3675. <br />     ditional heavy chain mutants. PLoS Genet. 3: e128.                   WormBase, 2008 Available http://www.wormbase.org; Release <br /> Page, B. D., S. Guedes, D. Waring J. R. Priess, 2001 C.              WS200, 16, 2008. <br />     elegans E2F- DP-related proteins required embryonic      Wu, Y., M. Han, 1994 Suppression activated Let-60 ras pro- <br />     asymmetry negatively regulate Ras/MAPK signaling. Mol.               tein defines role Caenorhabditis elegans Sur-1 MAP kinase <br />     Cell 7: 451&#226;&#8364;&#8220;460.                                                         vulval differentiation. Genes Dev. 8: 147&#226;&#8364;&#8220;159. <br /> Panneerselvam, S.,  Marx, E. M. Mandelkow E. Mandelkow,            Yoder, J. H., H. Chong, K. L. Guan M. Han, 2004 Modulation <br />     2006 Structure catalytic ubiquitin-associated domains         KSR activity Caenorhabditis elegans Zn ions, PAR-1 ki- <br />     protein kinase MARK/Par-1. Structure 14: 173&#226;&#8364;&#8220;183.                 nase PP2A phosphatase. EMBO J. 23: 111&#226;&#8364;&#8220;119. <br /> Rocheleau, C. E., K. Cullison, K. Huang, Y. Bernstein,  C. <br />     Spilker et al., 2008 Caenorhabditis elegans ekl (enhancer <br />     ksr-1 lethality) genes include putative components germ- <br />     line small RNA pathway. Genetics 178: 1431&#226;&#8364;&#8220;1443.                                                    Communicating editor: K. Kemphues <br />                       Supporting Information <br /> http://www.genetics.org/cgi/content genetics.109.106716/DC1 <br />  <br />  <br />  MAP Kinase Signaling Antagonizes PAR-1 Function   Polarization Early Caenorhabditis elegans Embryo <br />  <br />  <br />  Annina C. Spilker, Alexia Rabilotta, Caroline Zbinden, Jean-Claude Labb&#195;&#169; <br />                              Monica Gotta <br />  <br />  <br />         Copyright &#194;&#169; 2009 Genetics Society America <br />                 DOI: 10.1534/genetics.109.106716 <br />                       Supporting Information <br /> http://www.genetics.org/cgi/content genetics.109.106716/DC1 <br />  <br />  <br />  MAP Kinase Signaling Antagonizes PAR-1 Function   Polarization Early Caenorhabditis elegans Embryo <br />  <br />  <br />  Annina C. Spilker, Alexia Rabilotta, Caroline Zbinden, Jean-Claude Labb&#195;&#169; <br />                              Monica Gotta <br />  <br />  <br />         Copyright &#194;&#169; 2009 Genetics Society America <br />                 DOI: 10.1534/genetics.109.106716 <br />  2 SI                                                      C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S1. temperature sensitive period par-1(zu310) starts cell stage extends 28-cell <br /> stage. Embryos dissected gravid hermaphrodites grown permissive temperature shifted restrictive temperature <br />  vice versa) various developmental stages. shifted embryos left develop final temperature survival assessed day. analysis revealed par-1 requirement par-1(zu310) animals starts cell stage ends embryonic stage 28 nuclei visible middle focal plane embryo. Upper panel: Schematic <br /> representation experiment. Lower panel: Bar graphs representing percentage hatching non-hatching embryos, <br /> shifted   particular stage (n&gt;7 stage). Morphogenesis scored presence absence embryo twitching. <br />                                                          C. Spilker et al.                                                3 SI <br />  <br />  <br />  <br />  <br />    FIGURE S2.&#226;&#8364;&#8221;mpk-1 does regulate PAR-1 levels. Western blot analysis PAR-1 protein levels present embryonic <br /> extracts obtained animals indicated genotypes grown 22&#194;&#186;C. graph shows relative intensity PAR-1 levels various extracts &#194;&#177; SD 2 independent experiments reveals levels PAR-1 similar conditions tested. <br /> Tubulin used reference. <br />  4 SI                                                     C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S3.&#226;&#8364;&#8221;mpk-1(ga111) does suppress strong loss par-1 function. Quantification cell cycle timing AB (light grey) P1 (dark grey) daughter blastomeres par-1(RNAi) (n=11) mpk-1(ga111); par-1(RNAi) (n=13) embryos. synchronous cell <br /> division phenotype par-1(RNAi) 2-cell stage embryos suppressed mpk-1(ga111). <br />                                                          C. Spilker et al.                                                 5 SI <br />  <br />  <br />  <br />  <br />    FIGURE S4.&#226;&#8364;&#8221;Slowing cell cycle does result suppression par-1(zu310).   Quantification cell cycle timing AB (light grey) P1 (dark grey) daughter blastomeres control (n=9), F33H2.5(RNAi) (n=10) div-1(RNAi) (n=10) embryos. <br /> Cell cycle timing AB P1 div-1(RNAi) embryos significantly different control (p&lt;0.05). division P1 F33H2.5(RNAi) embryos slightly slower control, significant. (B) Localization endogenous <br /> MEX-5 cell stage embryos 22&#194;&#186;C. wild-type embryos asymmetric localization MEX-5, par-1 mutant <br /> embryos exhibit pronounced gradient MEX-5 daughter blastomeres. MEX-5 localization restored RNAi depletion F33H2.5 div-1. Embryo length ~50 &#206;&#188;m. (C) Quantification fluorescence intensity ratio cytoplasm AB P1 MEX-5 stained embryos. number embryos analyzed ranges 5 7. <br />  6 SI                                                  C. Spilker et al. <br />  <br />  <br />  <br />  <br />    FIGURE S5. strong loss mpk-1 function does affect MEX-5 levels. images localization endogenous MEX- <br /> 5 cell stage embryos. Quantification revealed MEX-5 localized asymmetrically wild-type (n=4) mpk- <br /> 1(ga111) (n=7) embryos grown 25&#194;&#186;C. Embryo length ~50 &#206;&#188;m. <br />                                                           C. Spilker et al.                                                 7 SI <br />  <br />  <br />  <br />  <br />    FIGURE S6.&#226;&#8364;&#8221;Fertility mpk-1; par-1 double mutant F1 progeny partially restored. DIC images posterior F1 animals 24hs post larval stage L4. Posterior right ventral images. wild-type, mpk-1 mpk-1; par-1 animals <br /> embryos oocytes visible ventrally. par-1 animals gut visible body cavity (lower image) gut cellular material present. cellular material reminiscent somatic gonad. <br />  8 SI                                    C. Spilker et al. <br />  <br />  <br />  <br />  <br />                                           TABLE S1 <br />  <br />                              Candidate Suppressors par-1(zu310) <br />  <br />  <br />        M01D7.7         C16C4.1                     T21B4.1           T20G5.6 <br />        C32E8.4         R52.10                      Y54G9A.2          Y39A1C.2 <br />        T03F1.2         D2062.9                     Y54G9A.5          Y56A3A.29 <br />        C18E3.6         Y27F2A.g                    F26H11.3          Y49E10.25 <br />        T03A1.3         F14D2.3                     Y46G5.ee          F53A2.1 <br />        ZK973.f         Y49F6C.7                    Y46G5.kk          F53A2.6 <br />        F21F3.5         F12E12                    C29F9.2           C51G7.d <br />        C46H11.10       C17A2.8                     T24C4.1           Y119D3_457.b <br />        C32E12.4        B0286.4                     F23H11.3          Y119D3_465.y <br />        F48C1.3         T05C1.5                     R06B10.2          Y53G8A_9248.d <br />        F26B1.7         C34F11.3                    T20H9.4           Y55B1A_116        C55B7.2         C34F11.5                 <br /> </body></html>