<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Doyle et al. BMC Genomics 2010, 11:222 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />     RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> Drug target prediction prioritization: using <br /> Research article <br />  <br /> orthology predict essentiality parasite <br /> genomes <br /> Maria Doyle1,3, Robin B Gasser*2, Ben J Woodcroft1, Ross S Hall2 Stuart Ralph*1 <br />  <br />  <br />  <br />     Abstract <br />     Background: New drug targets urgently needed parasites socio-economic importance. Genes     essential parasite survival highly desirable targets, information genes lacking, gene knockouts <br />     knockdowns difficult perform species parasites. examined applicability large-scale <br />     essentiality information model eukaryotes, Caenorhabditis elegans, Drosophila melanogaster, Mus musculus <br />     Saccharomyces cerevisiae, discover essential genes genomes. Parasite genes lack <br />     orthologues host desirable selective targets, examined prediction essential genes     subset. <br />     Results: Cross-species analyses showed evolutionary conservation genes presence essential <br />     orthologues strong predictors essentiality eukaryotes. Absence paralogues     general predictor increased relative essentiality. combining orthology essentiality criteria     select gene sets fold enrichment essential genes compared random selection.     quantitative application criteria used predict ranked list potential drug targets Ancylostoma <br />     caninum Haemonchus contortus - blood-feeding strongylid nematodes, presently limited <br />     sequence data functional genomic tools. <br />     Conclusions: present study demonstrates utility using orthology information multiple, diverse <br />     eukaryotes predict essential genes. data emphasize challenge identifying essential genes     parasite absent host. <br />  <br />  <br /> Background                                                                                  potential drug targets potentially accessible recently, search novel drugs parasites                                organisms. major challenge carried mainly using approaches                                           identify putative targets, prioritize  <br /> directly screen inhibition parasite growth lethal-                                available resources focused ity. current industry regulatory focus target-                                   likely lead effective treatments/drugs. based drug development meant search                                        aspect pressing neglected infectious diseases <br /> new anti-parasitic compounds moved target-                                    cause disproportionate burden developing <br /> based paradigm. completion genome projects                                       countries costs drug develop- <br /> large-scale expressed sequence tag (EST) surveys                                      ment process deterred investment pharma- <br /> range parasites means tens thousands                                      ceutical industry [1].  considered approaches <br />                                                                                             predict promising targets identify <br /> * Correspondence: robinbg@unimelb.edu.au, saralph@unimelb.edu.au <br /> 2 <br />                                                                                             likely essential required increase <br />   Department Veterinary Science, University Melbourne, 250 Princes <br /> Highway, Werribee, Victoria, 3030, Australia                                                likelihood lead compounds proceed commer- <br /> 1 Department Biochemistry <span id='am-12' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Molecular Biology, Bio21 Molecular Science                   cial development [2]. Biotechnology Institute, University Melbourne, Victoria, 3010,                     Identifying essential genes pathogens important <br /> Australia list author information available end article                      <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> inhibition non-essential genes                                                                                             unlikely result death infectious agent, <br />                                             &#194;&#169; 2010 Doyle et al; licensee BioMed Central  Open Access article distributed terms Creative Commons <br />             BioMed Central Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction                                             medium, provided original work properly cited. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                              Page 2 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  use non-essential molecules, example,         screening uncertain compared organisms vaccine targets rapidly select mutants        screens, future screening single molecule targets <br /> evade protective immune responses. Essentiality,             likely measure loss function linked lethality,  difficult define application neces-   searching modulators agonists.  <br /> sarily restricted experimental conditions           fore, prediction identification essential genes <br /> tested. Despite limitation, indispensability        parasitic helminths considered important step <br /> defined laboratory condition useful starting            prioritization drug targets nematodes. <br /> point identifying biological processes likely      significant growth crucial survival organism natural         sequence data nematodes, enabling prediction conditions. proliferation genome sequencing proj-         numerous potential drug targets. focus ects followed systematic analyses gene            sequencing ESTs [17-23]   diffi- <br /> knockouts knockdowns determine essentiality            culties producing sufficient quantities genetically number model eubacteria eukaryotes,          homogenous material heterologous worm popula- <br /> Escherichia coli, Saccharomyces cerevisiae                   tions genomic sequencing.  progress <br /> Caenorhabditis elegans [3-8]. Unfortunately, infec-          genome sequencing (e.g., Brugia <br /> tious agents currently amenable genome              malayi) [24], program recently received fund- <br /> approaches experimental testing essentiality,         ing sequencing range socio-economically promising candidate drug targets indi-         important nematodes order Strongylida [25]. vidually validated genetically. particularly true     data emerging sequencing projects eukaryotic parasites, genetic tools char-       spurred identification drug targets parasitic <br /> acterize drug vaccine targets limited,          nematodes using bioinformatic techniques (e.g., [26]), absent                                           testing essentiality targets generally <br />    particularly challenging area development         absent. Members Strongylida prop- <br /> novel compounds combat parasitic helminths ani-            agated  entire life cycle) culture vitro. mals. helminths major socio-economic                 addition, gene silencing (double-stranded RNA interfer- <br /> importance; billion people infected         ence, RNAi) does work effectively parasitic <br /> soil-transmitted worms (= geohelminths),             nematodes order assessed far,  cur- <br /> blood-feeding hookworms Ancylostoma duodenale                rently, major limitations conducting func- <br /> Necator americanus, common roundworm, Ascaris                tional studies different developmental stages lumbricoides, whipworm, Trichuris trichiura              parasites [27-29] order pre-validate potential drug <br /> [9,10]. parasites impose worldwide annual          targets. <br /> burden 39 million Disability Adjusted Life Years                paucity experimental tools characterize drug <br /> (DALYs) cause adverse effects human               targets Strongylida intensifies need predict <br /> health, particularly children [10]. Similarly, parasitic      putative targets likely essential; nematodes livestock, cattle sheep,           case, predictions necessarily based inference <br /> cause substantial economic losses diseases            orthologues model organisms.  ratio- cause, billions dollars spent annually       nal method applying knowledge essentiality treatment gastro-intestinal nematodes  Resis-           orthologues species lacking. Taking advantage <br /> tance emerging main classes anthelmint-      recent advances genome orthology mapping <br /> ics used combat parasitism [11-14], creating        [30] availability large-scale eukaryotic essenti- <br /> need new classes compounds.                               ality datasets [31-33], created database anal- <br />    desirable, anthelmintic targets neces-        yse essentiality predicted existing data <br /> sarily need encoded essential genes. example        model eukaryotes. designation essentiality number anthelmintics  aldicarb trichlo-         diverse organisms means quite different things. rfon) target acetylcholinesterases, perturbations         yeast, essentiality refer failure grow culture, <br /> silencing C. elegans acetylcholinesterase genes           mice encompass failure embryos <br /> result viable phenotye  drug             progress live offspring. <br /> resistant phenotypes) [15]. Likewise, levamisole targets           present study, tested disparate <br /> <span id='am-27' about='obo:IMR_0100111' typeof='owl:Thing obo:IMR_0001693 obo:IMR_0100121 obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0002121'><span id='am-28' property="oboInOwl:hasExactSynonym" content="O-Acetylcholine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C01996" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0100111" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="O-Acetylcholine" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="acetylcholine" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acetylcholine</span> receptors, disruption           forms essentiality predictive corresponding <br /> subunits receptors leads viable levami-       phenotypes organisms definitions <br /> sole resistant phenotype[16]. Despite precedence         essentiality strongly predictive essentiality successful drugs act agonist relationships          relatively distantly related eukaryotes. applied les- non-essential molecules, essentiality desirable        sons analyses prediction essential <br /> character genes pursued target-centric             genes parasitic nematodes. Members order <br /> approach. relative success target-based          Strongylida considered relatively closely related <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                         Page 3 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  Caenorhabditis elegans, clade V [34],     genetic experiments involving mouse genes did considered appropriate infer gene function       higher percentage reports lethality/essentiality abundant functional genomic data (including         organisms studied (Figure 1), reflect genome RNAi knockdown) exist C. elegans         selection bias. Nonetheless, patterns observed [26,35,36]. addition rational prediction essenti-   prediction essentiality orthology M. mus- <br /> ality orthology essentiality data, integrated      culus dataset paralleled organisms <br /> recently developed gene network connectivity data [37]         assessed, supporting inclusion present analy- predict key, essential genes offer         ses. <br /> promising testable drug targets. <br />   criterion employed predic-       Prediction essentiality based phyletic characteristics - <br /> tion drug targets infectious organisms absence    presence paralogues corresponding host orthologue, maximise          interrogated database determine par- <br /> opportunities pharmacological selectivity.         alogy useful predictor essentiality. Intuitively, successful exceptions criterion exist;        belonging paralogous family suggest example, genuine targets differentially regulated       degree redundancy compensate dis- pathogen host [38], protein turnover        rupted gene;  previous studies C. elegans S. rapid host pathogen [39], targets         cerevisiae indicated genes paralogues active proliferating cancerous cells     likely essential [42,43].  mouse, host [40], sufficiently structurally different   recent analysis [44] revealed difference genes host pathogen allow selectivity [41].        paralogues. studies used different <br /> investigated effect excluding genes possess        criteria define paralogues  cases, included <br /> mammalian orthologues discovered remain-          genes cause sterility essential, making dif- <br /> ing genes likely essential.      ficult compare results studies.  developed flexible prioritization methods         compare effect paralogy likelihood lethal- limited requirement genes lack host      ity different species, used single method orthologues. present study, applied tech-      <span id='am-19' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> paralogue (OrthoMCL), combined <br /> niques predict essential drug targets large EST       existing large-scale essentiality data C. elegans, <br /> data sets representing blood-feeding strongylid nem-       D. melanogaster, M. musculus S. cerevisiae. pres- <br /> atodes major socio-economic importance,              ence paralogues inferred OrthoMCL groups; <br /> Haemonchus contortus  barber&apos;s pole worm small          gene species ruminants) Ancylostoma caninum (canine hook-               OrthoMCL group, genes classified genes <br /> worm). combined prioritization strategy           paralogues. probability essentiality genes criteria predict molecules drug targets haematophagous nematodes proof principle <br /> application using orthology predict essentiality. <br />  <br /> Results <br /> Database essentiality enable comparison phenotypes model organisms - Caenorhabditis elegans, Drosophila <br /> melanogaster, Mus musculus Saccharomyces cerevi- <br /> siae - database constructed allow mapping phenotype data orthologues. Orthology determined using OrthoMCL database [30], populated phenotype data species. case C. elegans, D. melanogaster S. cerevisiae,      Figure 1 Effect paralogy probability essentiality genes. <br /> phenotype information available 94%, 75%            graph shows probability essentiality genes 88% protein-coding genes respectively. M. mus-       paralogues C. elegans, D. melanogaster, M. musculus S. <br /> culus, gene knockouts/knockdowns represented 20%             cerevisiae. Presence paralogues means gene                                                                 species present OrthoMCL group. Percentages num- protein-coding genes. genes chosen                                                                 bers genes lethal phenotypes expressed fraction genes <br /> knockout analysis mice selected address spe-         available phenotype information species. Differences cific phenotypic questions, possible non-       essentiality genes versus paralogues highly sta- <br /> random collection introduce bias            tistically significant (P &lt; 10-04), determined using Fisher&apos;s exact test <br /> away genes expected essential.       (Additional File 2). <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                            Page 4 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  paralogues shown Figure 1              tor essentiality conservation Additional File 1: Table S1.                                     organisms diverged recently.   C. elegans S. cerevisiae, genes          tested conservation orthologues paralogues significantly likely          multiple species improve probability essen- <br /> essential genes paralogues (Figure 1). C.          tiality particular species. nearly combina- <br /> elegans, 1.6-fold greater probability essenti-    tions, case  Figure 2). ality genes paralogues (P &lt; 10-33, Fisher&apos;s          example, 51% C. elegans genes orthologues S. <br /> exact test, Additional File 2), S. cerevisiae,       cerevisiae essential, 53% C. elegans genes 3.8-fold greater probability essentiality (P &lt;    orthologues S. cerevisiae D. melano- <br /> 10-51). D. melanogaster, genes paralogues            gaster essential. exceptions likely essential (1.4-fold, P &lt; 10-12),     causes unclear- e.g., genes present species, lesser extent C. elegans S. cerevi-     C. elegans, D. melanogaster, S. cerevisiae M. muscu- <br /> siae. incomplete dataset M. musculus differed             lus, likely essential knocked species genes         Drosophila present  paralogues (0.9-fold, P &lt; 10-04) likely      mer species (Figure 2). <br /> essential genes paralogues. <br />                                                                  Prediction based orthology mapping existing <br /> Prediction essentiality based phyletic characteristics -   essentiality data species <br /> presence orthologues                                          Subsequently, tested presence Genes shared species hypothe-           essential orthologue predictive essentiality <br /> sized previously likely          gene presence non-essential orthologue. <br /> essential [45].  evidence pro-     explored previously bacterium <br /> posal tested quantitatively eukaryotes,         Pseudomonas aeruginosa, essential genes unaware published studies demonstrated           predicted identifying genes homologous effect breadth extent conservation        essential genes unicellular micro-organisms, <br /> large evolutionary distances probability essenti-      Mycoplasma genitalium, Haemophilus influen- <br /> ality. Indications gene conservation used       zae, Vibrio cholerae, Staphylococcus aureus, E. coli S. predictor essentiality eukaryotes comes           cerevisiae [47]. set genes predicted essential <br /> study 315 essential genes Danio rerio (zebrafish)          agreement genes shown experimen- <br /> [46], essential genes likely       tally essential.  essentiality conserved non-essential genes. test effect        orthologue used predict essentiality multi- conservation genes essentiality, analysed        cellular organisms tested. C. elegans, D. <br /> frequency essentiality genes orthologues           melanogaster, M. musculus S. cerevisiae, assessed C. elegans, D. melanogaster, M. musculus S. cerevi-       essentiality species used pre- <br /> siae. species, tested effect      dictor essentiality second species. conservation gene species essenti-        query species  C. elegans, D. melanogaster, M. <br /> ality (query) species. example, examine        musculus S. cerevisiae), assessed probability presence orthologue S. cerevisiae          essentiality genes lethal phenotypes predicts essentiality C. elegans, identified         species. comparing phenotypes linked <br /> orthology groups comprising genes spe-            particular genes species, limited analysis <br /> cies,  C. elegans genes groups,        genes paralogues species calculated percent essential.          compared; conducted avoid difficulty ings shown Figure 2 Additional File 1: Table S2.      comparing phenotypes individual paralogues   genes species, existence     associated different phenotypes. test orthologue species increased probability         having essential orthologues S. cerevisiae predicted <br /> essentiality, distant source ortho-      essentiality corresponding C. elegans genes, logue, greater effect. C. elegans, probabil-    identified orthology groups containing non-paralogous <br /> ity essentiality gene paralogues selected       C. elegans S. cerevisiae genes subjected random 18%; probability increased 38%          essentiality analysis. compared frequency gene orthologue D. melanogaster             essentiality C. elegans genes grouped 51% orthologue existed distant relative,       essential non-essential S. cerevisiae orthologues. S. cerevisiae (Figure 2). Conservation greater evo-       tested essentiality multiple species <br /> lutionary distance appears better indica-      improved probability essentiality. results  Doyle et al. BMC Genomics 2010, 11:222                                                                                                          Page 5 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />                                <br />            	  <br />  <br />  <br />  <br />  <br />                                <br />                               <br />                               <br />                                <br />                               <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />           	 <br />     <br />    <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                               <br />                               <br />                                <br />                               <br />                                                                                                                                               <br />           <br />                                                                                                                                               <br />   <br />          <br />                           	 <br />   <br />   <br />                                <br />                <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                               <br />                               <br />                               <br />                                <br />  <br />  <br />                           	 <br />   <br />   <br />                                <br />                  <br />  <br />  <br />  <br />  <br />                                <br />                                <br />                                <br />                                <br />                                <br />                                 <br />  <br />                                                                                                                                      <br />  <br />  Figure 2 Effect gene conservation probability essentiality genes. graph shows probability essentiality genes C. <br />  elegans, D. melanogaster, M. musculus S. cerevisiae respectively, modified phyletic profile. percentages number genes  lethal phenotypes divided total number genes orthologues species compared. Specificity refers proportion genes <br />  selected subset essential sensitivity refers proportion total number essential genes species recov- <br />  ered selected subset. cases, essentiality assessed genes paralogues. probability essentiality genes  specified subsets versus gene selected random query genome statistically significant (P &lt; 10-05), determined using Fisher&apos;s <br />  exact test (Additional File 2). C = C. elegans, D = D. melanogaster, M = M. musculus, D = D. melanogaster, S = S. cerevisiae. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                            Page 6 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> predictions C. elegans species                     OrthoMCL group did gene shown Figure 3 Additional File 1: Table S3, respec-                     mammals considered  e., Homo sapiens, M. mus- <br /> tively.                                                                        culus Rattus norvegicus). calculated    analysis showed gene orthologue                          probability essentiality genes. C. elegans genes lethal knockout/knockdown phenotype                            lacked mammalian orthologues likely <br /> likely essential orthologue non-                        essential; probability essentiality decreased <br /> essential; distant orthologue, greater                        16% 8% gene selected random, was probability essentiality.  existence                    6% gene paralogues (Table 1, Figure 3). orthologue associated lethal phenotype S.                         substantial decrease probability essentiality <br /> cerevisiae highly predictive lethality C. elegans                    important consequences drug target prediction <br /> (72%), greater presence orthologue                     explicit previously. Researchers <br /> S. cerevisiae (51%). Combining presence essen-                       quite likely examine genes exactly category <br /> tial orthologue evolutionarily distant relative                      e. present pathogen absent host), (S. cerevisiae) additional, closely-related species                    low &quot;raw&quot; probability essentiality  e., D. melanogaster) increased probability -                   genes (6-8%) highlights challenge prediction C. elegans genes orthologue D. melano-                         essential drug targets subset, gaster S. cerevisiae, lethal phenotype,                    address  81% probability lethality. <br />                                                                                Prediction drug targets Anyclostoma caninum effect drug-target selectivity predictions                        Haemonchus contortus: combining &apos;assayability&apos; essentiality                                                                   prediction essentiality <br /> Anti-parasitic drugs need selectively inhibit kill                   present findings applied prediction parasite causing unacceptable toxicity                          drug targets haematophagous parasitic nematodes, host. way pursuing targets drugs                         outlined flow diagram (Figure 4). parasites Ancy- <br /> focus parasite genes lack host orthologues.                     lostoma caninum Haemonchus contortus aim identify targets parasitic nematodes                       selected represent parasitic nematodes order <br /> infect mammals, examined possible effects                            Strongylida (&quot;clade V&quot; [34]), significant EST <br /> essentiality focusing parasite genes lack mam-                      sequence data exist. available EST data  caninum <br /> malian orthologues.  analysed probabil-                      H. contortus (comprising 21,975 80,551 ESTs, <br /> ity essentiality C. elegans genes                           respectively) downloaded NCBI database. <br /> possessed lacked mammalian orthologues. obtain                           ESTs mapped OrthoMCL orthol- set C. elegans genes mammalian ortho-                           ogy groups BLASTX analysis individual EST <br /> logues, identified C. elegans genes                         sequences OrthoMCL proteins using                                                                                OrthoSelect method [48]. caveat EST-based <br />                                                                                analysis incompleteness genomes                                                                                parasitic nematodes means poorer iden- <br />                                                                                tification paralogues orthologues. analysis <br />                                                                                followed extraction C. elegans orthologues <br />                                                                                orthology groups. Previously, showed                                                                                having essential orthologue better predictor                                                                                essentiality just having orthologue (e.g., C. ele- <br />                                                                                gans, gene orthologue essential D. <br />                                                                                melanogaster 1.6-fold likely essential                                                                                C. elegans gene just orthologue D. melano- <br />                                                                                gaster). focused subset C. elegans <br />                                                                                orthologues parasitic nematodes essen- <br />                                                                                tial  e., genes lethal RNAi phenotypes C. ele- <br />  Figure 3 Effect conservation essential genes probability             gans). <br />  essentiality. graph shows likelihood essentiality C. el- <br />  egans genes essential orthologues organ-           Assayability requirement: enzymes, <span id='am-6' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> channels  isms; D. melanogaster, M. musculus S. cerevisiae. percentages         GPCRs <br />  number genes lethal phenotypes divided total           select targets amenable vitro <br />  number genes orthologues species. Essentiality  <br />                                                                                screening growth inhibition lethality, focused <br />  sessed genes paralogues. C = C. elegans, D = D. melano- <br />  gaster, M = M. musculus, S = S. cerevisiae.                                   genes encoding proteins functional assays <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                          Page 7 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> Table 1: Combining essentiality selectivity <br />  <br />                                                                           Probability essentiality <br />  <br />  <br />                                        Genes                 paralogues                   paralogues <br />                                             paralogues <br />  <br />  <br />            C. elegans                            16%                                 11%                                18% <br />             genes)                    (2941/18860 genes)                   (656/5988 genes)                  (2285/12872 genes) <br />  <br />  <br />    C. elegans (excluding genes                   8%                                  6%                                 9% <br />   mammalian orthologues)             (1090/13307 genes)                   (249/4431 genes)                   (841/8876 genes) <br />   table demonstrates effect excluding genes mammalian orthologues probability essentiality. Percentages shown <br />   number genes lethal RNAi (gene knock  phenotypes divided total number genes category.   numbers genes given brackets. <br />  <br />  <br /> exist.  examined subset C. elegans genes                Increasing probability essentiality incorporating <br /> encoding proteins inferred enzymatic, ion chan-                information network connectivity <br /> nel, GPCR activity, determined term annota-                Recently, Lee et al. [37] constructed probabilistic func- <br /> tion  e., 0003824, 0005216 0004930,                 tional interaction network C. elegans, largest subterm thereof ). current drugs clinical use           network <span id='am-5' about='oboInOwl:date' typeof='owl:Thing'>date</span>, comprising ~82% C. elegans genes. <br /> nematodes target protein catego-               network integrates data proteins genes (e.g., <br /> ries. GPCRs targets 50% pharma-                microarray data interaction association informa- <br /> ceutical drugs [49], making attractive targets            tion) using information C. elegans genes parasitic nematodes. enzymes                   orthologues species. Using network, <br /> likely essential non-enzymes C. ele-               authors showed gene connectivity predictive gans S. cerevisiae - C. elegans, 16% genes              essentiality - genes highly connected net- essential, 21% enzymes essential;                 work likely essential; shown S. cerevisiae, 19% genes essential, 24%           feature conserved eukaryotes <br /> enzymes essential -  focusing enzymes,                (including S. cerevisiae M. musculus). indepen- increased likelihood identifying essential              dent study Hwang colleagues showed genes.                                                                 genes encoding proteins higher connectivity pro- <br />                                                                        tein-protein interaction networks likely Essentiality requirement: quantitative formula                     essential E. coli S. cerevisiae [50]. genes maximum prediction essentiality based present                higher network connectivity scores  assigned Lee et <br /> analysis                                                               al) likely essential lower applied quantitative weighting scheme genes                scores [37], gave greater weighting genes met criterion assayability, according              higher connectivity scores. <br /> present analysis relative importance feature              combined quantitative weighting scheme determining essentiality. example, showed               network connectivity scores produce prioritized list <br /> having paralogue increased probability essenti-              putative drug targets  caninum H. contortus <br /> ality 1.6-fold C. elegans (Table 1), having ortho-            (Additional file 3). represent predicted C. elegans <br /> logue paralogues S. cerevisiae D.                        orthologues  caninum H. contortus ESTs melanogaster increased probability 3.3 2.4-fold,                chosen assayability (enzyme, ion-channel respectively. (Figure 3). Furthermore, having ortho-                GPCR), quantitative ranking scheme applied logue paralogues essential S. cerevi-              predict essentiality. Genes included list siae D. melanogaster increased probability                  related lethal RNAi phenotype C. elegans. <br /> essentiality 5.2-fold. fold increases sta-             list comes incomplete EST snap- <br /> tistically significant (P &lt; 0.0005), determined using               shot haematophagous helminths, Fisher&apos;s exact test (Additional File 2). weighting                recovers validated anthelmintic drug targets; beta- score genes numerically order prob-              tubulin, target Benzimidazole [51]; glutamate-gated <br /> ability genes essential.                                 chloride channel, target avermectin/milbemycin <br />                                                                        anthelmintics [52] succinate dehydrogenase/fumar- <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                         Page 8 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                                                                             exact probability essentiality uncharacterized <br />    Orthologue C. elegans?                                                gene, weightings estimation rel- <br />                                                                             ative likelihood essential genes                                     yes                                                                                              considered candidate targets. <br />    Enzymes, Ion channels &amp; GPCRs? <br />                                                                             Discussion <br />                                     yes                                  Defining essential genes central strategy priori- <br />                                                                             tization drug targets infectious organisms.    Lethal phenotype C. elegans? <br />                                                                             non-essential genes act drug targets  <br />                                     yes                                  verting non-toxic precursors toxic products,                                                                             anti-microbials act inhibiting function essen- <br />                         Lacks paralogues?                                   tial process, leading death pathogen,                                                                             direct absence essential process  indi- <br />               yes                                                        rectly, stress-related response. number                   Orthologue distant relative <br />                                                                             anthelmintic drugs (e.g. levamisole [16] ivermectin <br />                                                                             [54]) appear function agonists non-essential <br />               yes                                                        genes  review anthelmintic modes action                                                                             [55]). search drug targets essen- <br />             Essential orthologue distant relative                        tial genes useful approach prioritization tar- <br />                                                                             gets, inability methodology predict               yes                                                                                                                  successful anti-parasitic drug targets limitation            Presence orthologue close relative                         practice. <br />                                                                               attempts define predict <br />               yes                                                        essential genes infectious agents; cases,                                                                             sophisticated functional genomic tools avail- <br />              Essential orthologue close relative                         able, possible systematic dis- <br />               yes                                                        ruption candidate genes, example, Streptococcus <br />                                                                             pneumoniae [56], Candida albicans [57], Aspergillus <br />                     Network connectivity score                              fumigatus [58]) Mycobacterium tuberculosis [7,8]. <br />                                                                              vast majority infectious organisms, <br />               yes                                                        high-throughput experimental interrogation essential- <br />                                                                             ity currently possible, efforts define essenti- <br />      Prioritized list targets parasitic nematodes                    ality based bioinformatic predictions [59- <br />                                                                             63]. <br />  Figure 4 Prioritizing essential nematode drug targets. flow di-          attempted define general characters <br />  agram outlines approach drug target prediction prioritiza- <br />  tion parasitic nematodes. steps clear  reject&quot; <br />                                                                             predictive essentiality eukaryotic genes <br />  decisions genes failing criteria easily   orthology. Previous studies shown  <br />  amenable development unacceptable probabili-         servation genomes itself predictive feature  ty essential. steps used quantitatively refine   essential genes. Studies Butland, [64] Hwang [50] Kro- <br />  weighting produce ranked list essential drug targets.              gan [65] colleagues showed essential genes S. <br />                                                                             cerevisiae E. coli widely phyletically <br /> ate reductase, C. elegans equivalent helminth                    distributed, connected physical <br /> target thiabendazole [53].                                               functional protein-protein interactions. data <br />   likely application prioritization method                  broad evolutionary conservation genes focus particular subset enzymes                    multicellular eukaryotes) predictive essenti- <br /> investigator expertise. example, aim                  ality, essentiality orthologues identify promising kinase drug targets, kinases                    strong predictor. Combining individual criteria ranked list extracted appraised.                     independently predictive essentiality increases 49 predicted kinases (Figure 5) H. contortus                 specificity prediction - example, C. elegans gene  caninum EST datasets. list                       selected random 16% likely essential, expected likely essential                      C. elegans genes lack paralogues parasitic nematodes, based present anal-                   essential orthologues S. cerevisiae D. melanogaster <br /> yses. numerical weighting                       81% likely essential. dramatic increase  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                                                                                                              Page 9 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />                                                                                                                                                         	    <br />                                                                                                                       	                                                            <br />      	  <br />                                                                                                                                                  <br />     <br />   <br />     	  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                            <br />     <br />  <br />  <br />  <br />  <br />                                                                                                                             	      <br />  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                	      <br />  <br />                                                                                                                                                                                                                             	  	    <br />  <br />  <br />  <br />  <br />                                                                                                                	 <br /> 	   <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                   	 <br /> 	   <br />                                                                                                                                                            <br />   <br />   	   <br />                   	 <br />       	      <br />                                     <br />   	       	  	 <br />                                                                                                                                                   <br />      	 	    <br />  <br />  <br />  <br />  <br />                                                                                                                                               <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                  	 <br />  <br />                                                                                                                                                                                                                           	    <br />                                                                                                                                                                                                                                           <br />    <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        <br />     <br />                                                                                                                                                       <br />                       	        %&amp;           %&apos;%    &amp;    &amp;   (    ) (&quot; *                                                                                                                                                    <br />             <br />    	 <br />                    <br />   		          <br />   <br />  	       +,-(*   ) (+,-*                                                                                                                                                                <br />              <br />      <br />                                              . /  0,   <br />        &amp;()     0 1  /    (  /    (    )                                                                                                                                               <br />                    <br />    	           <br />   <br />    <br />      +,-(*   ) (+,-* <br />                                                                                                                                                                               	 <br />                                     <br />           /    (    )                                                                                                                                                                    <br />           	     <br />                                     <br />          1/) 1   &amp;&apos;   /)  / (      )                                                                                                                                                                  <br /> 	 <br />                        <br />         2    <br />          $#(*   )                                                                                                                                                                                  <br />     <br />   <br />                     	 	        3 4  <br />          &apos; /)   (    ) 5( # (6  1   7(  %   / (8 2  &apos;                                                                                                                                                  <br />   <br />                        <br />             	  <br />        1/) 1/2 3  /    (    )                                                                                                                                                                         <br />                        <br />         %&amp;   <br />         %&apos;%    &amp;    &amp;   (    )                                                                                                                                                                     <br />   <br />    <br />                                     <br />         % )   (    ) (,                                                                                                                                                                                  <br />                      <br />   <br />             	  <br />       9 # &quot; #(    ) (                                                                                                                                                                                  <br />                                             % )   (    ) (,(6  )  / 0&amp;    7                                                                                                                                                                  <br />                                %&amp;   <br />         %&apos;%   (&amp;    &amp;   (    ) ( <br /> (1/2/ /:                                                                                                                                                                <br />                                %&amp;  	        %&apos;%   (% &amp;    &amp;   (    ) (&quot; *	                                                                                                                                                              <br />      <br />                      	         )    <br />         ;+(# /    (*   )                                                                                                                                                                            	 	  <br />                                     <br />         %&apos;%    &amp;    &amp;   (    ) (    <br />                                                                                                                                                                      <br />                        <br />         4.4  <br />         2  /  %(%1 %  /   ()     0 1  /    (  /    (    )                                                                                                                                              <br />                    	  <br />  <br />          )             4/)/2  (  /    (; (    ) 0(+;*    (    ) (1/2/ /:                                                                                                                                         <br />   	  <br />                                )    <br />         $+*(  /    (    ) (8 2  &apos;                                                                                                                                                                        <br />                       <br />               <br />         )     0 1  /    (    ) (6*,&lt; <br /> 0;&lt;  <br /> 0&lt; 2 <br /> ().48 2  &apos;7                                                                                                                                               <br />                    <br />             %)    <br />        % )   (    ) (6)     0 1  /    0 &apos; /)   (  /    (    ) 7                                                                                                                                     <br />     <br />                     <br />   	        &amp;    <br />         2  /:    %  3   &amp;(  /    (    ) (    ) (    )                                                                                                                                                   <br />                      <br />                        )     0 1  /    (    )                                                                                                                                                                          <br />                      <br />  	         1   <br />          1&apos;2 &amp;   (    )                                                                                                                                                                                 <br />                                     <br />          1/ 4  &amp;  :(;  0=1 (    )                                                                                                                                                        <br />                <br />                      <br />              <br />   <br />          &apos; .3   (    )                                                                                                                                                                   <br />                 <br />                    <br />  	          !&quot;    	        :     (    )                                                                                                                                                                   <br />                <br />                       	        %1   <br />         &quot; * <br />      ()     ( 1  /    (  /    (    )                                                                                                                                                         <br />                    <br />                 <br />           /    (    )                                                                                                                                                                                  <br />  <br />  <br />                     <br />                 <br />   <br />         /    (    )                                                                                                                                                                                <br />  	  <br />                        <br />                          :         &amp;(    )                                                                                                                                                                        <br />   <br />   <br />                      <br />  <br />             <br />     <br />        .%  /) &amp; (&amp;  1/) 1   (    )                                                                                                                                                   <br />                   <br />                     <br />            :)           : &apos;%/:  ()&apos;  1 ) (    )                                                                                                                                                        <br />             	    <br />  <br />                      <br />          24           &amp;.   )  % 8 %  &apos;(    <br />        &amp;(    )                                                                                                                                            <br />                 <br />   <br />                                &quot;   	  <br />       % )   (    ) (6)     0 1  /    0 &apos; /)   (  /    (    ) 7                                                                                                                       <br />                   <br />                    	             <br />            .  &amp;&apos;    (    ) 0 &amp;  &apos;    (    )                                                                                                                                               <br />               <br />   <br />   <br />                    	 	           <br />            1 &gt;/    )                                                                                                                                                                      <br />                 <br /> 	 <br />                      <br />                <br />          /.)  &amp;     (  /    (    )                                                                                                                                                     <br />                  <br />                       	             <br />          . /  0,   <br />        &amp;()     0 1  /    (  /    (    )                                                                                                                             <br />                <br />   <br />                    <br />    <br />              <br /> #    <br />     1 &gt;/    )                                                                                                                                                                      <br />              <br />    <br />  <br />                    <br />             %&amp;           %&apos;%    &amp;    &amp;   (    ) (&quot; *                                                                                                                                                    <br />                  <br />                      <br />   <br />                      % )   (    ) (,,(   1 5(%    &apos;  %().4.                                                                                                                                         <br />                   <br />                      <br />              <br />   <br />  $  <br />      1/) 1/8 .% /    )                                                                                                                                                             <br />               <br />     <br />                     <br />  	         8    <br />          /    %   / ( &apos; /)   (    )                                                                                                                                                    <br />              <br />      <br />                      <br /> 	              <br />         %$ # &amp;    &amp;   (  /    (    )                                                                                                                                                   <br />             <br />  	   <br />  <br />                      	         ):   <br />         )     0 1  /    (  /    (    )                                                                                                                                                 <br />                 <br />   <br />                                :)           )     0 1  /    (    )                                                                                                                                                         <br />                   <br />                    <br />              ) )  <br />          4/)/2  (  /    (; (    )                                                                                                                                                     <br />              <br />      <br />  <br />  <br />  <br />  Figure 5 Prioritized kinases parasitic nematodes  caninum H. contortus. shows C. elegans predicted orthologues  <br />  caninum H. contortus ESTs, identified present study. genes predicted kinases, lethal RNAi phenotype C. elegans. Targets <br />  prioritized essentiality weighting assigned present study, Wormnet connectivity score. ranked kinases <br />   bold) thirty ranked targets ESTs (Additional File 3). Present mammals defined having orthologue  mammalian species OrthoMCL 2.0 database (human, rat, mouse). Essential orthologue yeast, drosophila mouse: ? = orthologue <br />   paralogues) associated lethal phenotype, ? = orthologue  paralogues) non-lethal phenotypes recorded, ? =  <br />  thologue  paralogues) phenotype information available. Presence paralogues C. elegans determined OrthoMCL <br />  grouping. Presence paralogues  caninum H. contortus determined described Materials methods. network connectivity <br />  score sum interaction scores (log-likelihood scores) gene Wormnet core network [37]. <br />  <br />  specificity prediction increases probability                                             scriptive queries. present example, 258 <br /> essentiality dishearteningly low level,                                              candidates remained set 2941 C. elegans genes <br /> randomly chosen drug target candidate likely                                             determined experimentally essential. practice, <br /> non-essential, reassuring level prioritized                                             investigator balance desired specificity didates likely essential.                                                   query desired pool size selected tar- <br /> increase specificity associated predictions                                           gets, highly weight specific predictors, accompanied decrease sensitivity - gen-                                              genes likely essential ranked  <br /> uinely essential genes discarded highly pre- <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                           Page 10 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br />    discovery exclusion genes mamma-         host. quantitatively applied conclu- <br /> lian orthologues leads sets genes       sions analysis prediction essential <br /> likely essential major implications,        genes blood-feeding parasitic nematodes, combining <br /> highlights additional challenge identification     requirement assayability identify tar- <br /> drug targets allow selectivity. decreased         gets readily amenable investigation. analy- <br /> essentiality readily rationalised, considering        sis led prioritization tractable number analyses relationship orthology        potential drug targets leading development essentiality. case, excluded set genes      nematocidal agents. <br /> shared orthology worms mammals. resultant set, conserved fewer organisms,            Methods <br /> tained higher proportion non-essential genes.               Orthology data <br />    focused enzymes, ion channels G-protein-            Orthology data obtained OrthoMCL [30], <br /> coupled receptors (GPCRs) vast majority           available http://www.orthomcl.org. OrthoMCL classi- drugs target classes [49,66] chemicals             fies genes 87 genomes orthology groups. Files exist interrogate targets. Furthermore,       containing protein sequence data orthology group proportion essential genes greater      information downloaded OrthoMCL version <br /> enzymes protein-coding genes, C.         2. dataset contained 20084, 14034, 24435 6718 <br /> elegans S. cerevisiae.                                       proteins C. elegans, D. melanogaster, M. musculus <br />    study, applied semi- quantitative methods       S. cerevisiae, respectively includes genes assigned <br /> identify essential genes parasitic helminths.           orthology groups OrthoMCL dataset methods equally applicable prediction        assigned group (annotated OrthoMCL <br /> essential genes eukaryotic pathogens EST            &apos;no_group&apos;). Orthologues genes deriving genome projects. shown genes           ancestral gene separated speciation event, <br /> essential orthologues greatly increased probability       paralogues genes separated gene essentiality researcher identify genes      duplication event genome. Subsequent specia- parasite essential C. elegans,    tion lead paralogous relationships genes <br /> D. melanogaster, M. musculus S. cerevisiae.            different genomes (referred paralogues) species distantly related protozoan spe-        [67]. practice, identification relationships <br /> cies, essentiality species      non-trivial robust complete genome <br /> highly suggestive essentiality protozoan parasite        data sequences useful phylogenetically informa- <br /> versus genes essential model        tive characters allow unambiguous phylogenetic trees organisms.                                                       constructed. currently computationally    addition identifying drug targets, identification    practically feasible multi-eukaryote, genome <br /> promising vaccine candidates priority para-        analyses. OrthoMCL clustering method provides sitic diseases. essentiality likely         convenient, necessarily imperfect means estimat- <br /> important characteristic vaccine targets, large variety     ing orthology paralogy. paralogues paral- characters desirable currently diffi-      ogues separated recent duplication event cult establish bioinformatically, particularly         normally clustered OrthoMCL group incomplete genome. characters                 represent gene/protein family, ancient tractable near future, current            duplications likely result separate clusters <br /> approach appropriate identifying vaccine anti-        [68]. purposes paper, genes different <br /> gens.                                                            species share OrthoMCL cluster  <br />                                                                  sidered orthologues, multiple genes Conclusions                                                      species share OrthoMCL study, undertook quantitative assessment         cluster considered paralogues. Additional File 4 dem- <br /> essentiality inference using orthology. analy-      onstrates example phylogenetic trees representa- <br /> sis demonstrates utility examining multiple, diverse      tive types orthologue clusters. <br /> organisms predicting essentiality, shows essentiality gene highly predictive essentiality   Phenotype data orthologues, organisms extremely           Phenotype information obtained C. elegans, M. <br /> different life cycles. data highlight poten-      musculus S. cerevisiae databases Wormbase <br /> tial undesirably enriching non-essential genes           WS191 http://www.wormbase.org[69], Mouse Genome prioritizing genes pathogen absent            Informatics (http://www.informatics.jax.org; June 2008 <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                             Page 11 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> [70]) Saccharomyces Genome Database (http://                   phenotype data available 20% (5008/24435) www.yeastgenome.org; Aug 2008) [71], respectively.             M. musculus OrthoMCL genes. <br /> D. melanogaster used results genome-wide <br /> RNAi screen [33]). order focus phenotypes            Computational predictions essentiality <br /> result disruption/inhibition proteins, opposed          Custom Ruby scripts written query database overexpression phenotypes, restricted analysis           Ruby Rails framework. Genes phenotypes following experiment/mutant                 paralogues defined genes OrthoMCL <br /> types: C. elegans (RNAi), S. cerevisiae (null, reduction        groups gene query species, function, repressible), D. melanogaster (RNAi) M.              genes query species OrthoMCL dataset <br /> musculus (included phenotypes following                   assigned &apos;no_group&apos;. number genes experiments: Chemically radiation induced, Chemi-              paralogues obtained subtracting number cally induced (ENU), Chemically induced  , Gene              genes paralogues total number genes <br /> trapped, Radiation induced, Spontaneous, Targeted                  query species. <br /> (knock , Targeted (Reporter), Targeted (Floxed/Flt),             analyze effect presence orthologues Transgenic (random, gene disruption)). Essential genes             probability essentiality, example, effect hav- defined follows; C. elegans M. musculus            ing yeast orthologue probability essentiality C. genes phenotype description contain-             elegans genes, OrthoMCL groups contained ing term &apos;lethal&apos;  Additional File 5 list   gene species identified. phenotypes), S. cerevisiae genes           subset groups selected contained phenotype &apos;inviable&apos;, D. melanogaster               gene S. cerevisiae C. elegans, genes showed lethality Adult, Pupal,               genes paralogues species.  Pupal Eclosion life-stage [33].           C. elegans genes groups number genes <br /> exclude possibility phenotypes           lethal phenotypes calculated expressed M. musculus result protein disruption/                percent total number C. elegans genes inhibition, example, possible chemical           orthologues S. cerevisiae (numbers genes given <br /> spontaneous mutation result constitutive activa-          Additional File 1: Table S2). <br /> tion protein gain function.  includ-            analyze frequency essentiality genes <br /> ing phenotypes result                  essential orthologues C. elegans, example, conservative prediction essentiality           examine presence essential orthologue <br /> mean M. musculus genes                   S. cerevisiae predicts essentiality C. elegans, falsely classed essential orthologues used          OrthoMCL groups contained gene spe- <br /> predict essentiality C. elegans lead lower           cies identified. subset groups probability essentiality C. elegans.                        selected contained gene S. cerevi- <br />                                                                    siae C. elegans, followed identifying Construction database genome comparisons            subset groups essential S. cerevisiae ortho- essentiality                                                    logue. C. elegans genes groups enable comparison phenotypes              number genes lethal phenotypes calculated <br /> species, relational database constructed map              expressed percent total number C. ele- <br /> phenotype data species OrthoMCL                 gans genes essential orthologue S. cerevisiae <br /> genes groups. database PostgreSQL data-             (numbers genes given Additional File 1: Table <br /> base http://www.postgresql.org used Ruby                S3). <br /> Rails framework http://www.rubyonrails.org. Custom                   set C. elegans genes mammalian ortho- <br /> tables designed store data OrthoMCL            logues genes OrthoMCL <br /> proteins mapped genes phenotype files           group did contain gene using gene identifiers present datasets (identifi-      mammalian species OrthoMCL (Homo sapiens, M. <br /> ers type WBGene00000001 C. elegans, CG3095              musculus Rattus norvegicus) C. elegans genes D. melanogaster, ENSMUSG00000020473 M.                     assigned &apos;no_group&apos; OrthoMCL. <br /> musculus YNL214W S. cerevisiae). software <br /> used process available http://github.com/            Prediction drug targets parasitic nematodes <br /> wwood/essentiality. Phenotype data available              available ESTs H. contortus  caninum 94% (18860/20084), 75% (10572/14034), 88% (5908/               downloaded National Center Biotechnology <br /> 6718) C. elegans, D. melanogaster S. cerevisiae             Information (NCBI) comprised 21,975 80,551 <br /> genes OrthoMCL dataset, respectively. contrast,          ESTs respectively. ESTs cleaned vector using <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                    Page 12 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />  <br /> SeqClean, repeats masked Repeatmasker, using        genes enzymes identifier catalytic <br /> default parameters programs. initially used       activity  0003824) specific sub- CAP3 program clustering ESTs,             terms (e.g., 0016301 kinase activity). Similarly, manual inspection results led discover   script written identify ion channels GPCRs - getting incorrect clustering - clustering ESTs        genes identifier ion channel activity similar identical genes contig      0005216) G-protein coupled receptor activity example ESTs large gene families colla-       0004930) specific subterms <br /> gens, lectins, cysteine proteases. tried increasing    (e.g., 0001584 rhodopsin-like receptor activity). percent identity cutoff CAP3 program          software written Ruby http://www.ruby-lang.org/ <br /> default 80% 95% incorrect clus-     using RSRuby library http://rubyforge.org/projects/ <br /> tering manual inspection. ESTs          rsruby/ communicate R http://www.r-project.org <br /> individually mapped OrthoMCL orthology groups              db Bioconductor package http://www.bio- <br /> using initial steps OrthoSelect method             conductor.org/packages/data/annotation/html/ <br /> described Schreiber colleagues [48]. Briefly, ESTs     db.html. Using approach, reference list assigned orthology group sufficient       terms did need loaded database BLASTX e-value, averaged best hit          instead installed db. software available species, default score cutoff used.     GPLv3 license http://github.com/wwood/ <br /> ESTs frequently represent small fragments genes,         goruby. <br /> caveat type analysis shorter length      C. elegans genes predicted ortho- <br /> makes assignment orthologue groups robust.            logues  caninum H. contortus ESTs iden- <br />   OrthoMCL group defined containing paral-          tified met assayability requirement, <br /> ogues parasitic nematodes ESTs       followed identification identified TBLASTX hit (e-value &lt; 1e-10)     essential C. elegans. resulted 421 C. imperfect BLASTN hit (identity &lt; 95%). conser-        elegans genes final drug target list, 163 ortho- <br /> vative e-value used avoid missing potential paralo-    logues common  caninum H. contortus, 124 <br /> gous relationships cost flagging potential false    present H. contortus 134 pres- <br /> positive paralogues pairs manually checked.     ent larger dataset  caninum. genes Additionally, OrthoMCL groups containing          list enzymes ion channels GPCRs met C. elegans proteins considered containing paral-      selection criteria. quantitative weighting scheme <br /> ogues. C. elegans gene(s) group         applied according present analysis rela- <br /> classed orthologues parasite EST drug       tive importance feature determining essential- <br /> target selection criteria applied C. elegans    ity - absence paralogues, presence orthologue  genes. caveat OrthoSelect/OrthoMCL cluster-         paralogues) D. melanogaster M. musculus S. <br /> ing approach members recently diverged gene      cerevisiae, presence essential orthologue  par- <br /> family, genes share common domain,        alogues) D. melanogaster, M. musculus S. cere- classified paralogues,         visiae. distinct roles functionally redun-         Network connectivity data C. elegans genes dant. improved preliminary EST-           obtained Wormnet v1 (available http:// <br /> clustering step, included future analy-    www.functionalnet.org/wormnet/); [37] integrated <br /> ses EST datasets amenable clustering.         database mapping genes Wormnet  caninum ESTs assigned 2313                 C. elegans OrthoMCL genes using gene identifiers <br /> OrthoMCL groups, resulting 2743 C. elegans genes           Wormbase gff3 file. &quot;core&quot; higher confi- predicted orthologues  caninum ESTs.        dence set linkages LLS &#226;&#8240;&#165; H. contortus ESTs assigned 2230               0.405465108108 (ln1.5). C. elegans gene LLS <br /> OrthoMCL groups, resulting 2650 C. elegans genes           linkages core network added predicted orthologues H. contortus ESTs.      resulted 10,041 genes network score greater <br />   assayability requirement enzymes, ion channels      zero. approach identification drug GPCRs identified using gene ontology              targets parasitic nematodes summarised Figure 4. <br /> terms. terms C. elegans genes obtained Amigo CVS database, using version 1.127                Additional material <br /> gene_association.wb.gz included database <br /> (retrieved Feb 2009 http://cvsweb.geneontol-              Additional file 1 Numbers genes orthology analysis.doc. file <br />                                                                shows numbers genes meeting criteria orthology <br /> ogy.org/cgi-bin/cvsweb.cgi/). custom software library,       comparisons essentiality. <br /> named GORuby, developed classify C. elegans <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                                                                                                                                                                                                                       Page 13 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                           5.    Winzeler EA, Shoemaker DD, Astromoff  Liang H, Anderson K, Andre B, <br />       Additional file 2 Fisher&apos;s exact test results orthology analysis.xls. <br />                                                                                                                                                                                                                                                                                                 Bangham R, Benito R, Boeke JD, Bussey H, et al.: Functional <br />       file contains results Fisher&apos;s exact test applied comparisons                                                                                                                                                                                                                                                                                                 characterization S. cerevisiae genome gene deletion       essentiality model organisms. <br />                                                                                                                                                                                                                                                                                                 parallel analysis. Science 1999, 285(5429):901-906. <br />       Additional file 3 Prioritized list drug targets  caninum H.                                                                                                                                                                                                            6.    Kang Y, Durfee T, Glasner JD, Qiu Y, Frisch D, Winterberg KM, Blattner FR: <br />       contortus.xls. genes list C. elegans predicted orthologues                                                                                                                                                                                                            Systematic mutagenesis Escherichia coli genome. J Bacteriol <br />        caninum H. contortus ESTs, identified present study.                                                                                                                                                                                                              2004, 186(15):4921-4930. <br />       Genes correspond validated anthelmintic drug targets shown                                                                                                                                                                                                           7.    Sassetti CM, Boyd DH, Rubin EJ: Comprehensive identification       bold. genes list predicted enzymes  term)                                                                                                                                                                                                              conditionally essential genes mycobacteria. Proc Natl Acad Sci USA <br />       lethal RNAi phenotype C. elegans. Targets prioritized                                                                                                                                                                                                              2001, 98(22):12712-12717. <br />       essentiality weighting assigned present study, network                                                                                                                                                                                                             8.    Sassetti CM, Boyd DH, Rubin EJ: Genes required mycobacterial <br />       connectivity score. &quot;Present mammals&quot; defined having ortho-                                                                                                                                                                                                                   growth defined high density mutagenesis. Mol Microbiol 2003, <br />       logue mammalian species OrthoMCL 2.0 data-                                                                                                                                                                                                                   48(1):77-84. <br />       set (human, rat, mouse). Essential orthologue yeast, drosophila mouse:                                                                                                                                                                                                        9.    Bethony J, Brooker S, Albonico M, Geiger SM, Loukas  Diemert D, Hotez <br />       ? = orthologue  paralogues) associated lethal phenotype, ? =                                                                                                                                                                                                               PJ: Soil-transmitted helminth infections: ascariasis, trichuriasis,       orthologue  paralogues) non-lethal phenotypes                                                                                                                                                                                                                       hookworm. Lancet 2006, 367(9521):1521-1532. <br />       recorded, ? = orthologue  paralogues) phenotype informa-                                                                                                                                                                                                             10.   Chan MS: global burden intestinal nematode infections--fifty <br />       tion available. Presence paralogues C. elegans determined                                                                                                                                                                                                                   years  Parasitol Today 1997, 13(11):438-443. <br />       OrthoMCL grouping. Presence paralogues  caninum H.                                                                                                                                                                                                             11.   Wolstenholme AJ, Fairweather  Prichard R, von Samson-Himmelstjerna G, <br />       contortus determined described Materials methods. net-                                                                                                                                                                                                                  Sangster NC: Drug resistance veterinary helminths. Trends Parasitol <br />       work connectivity score sum interaction scores (log-likelihood                                                                                                                                                                                                              2004, 20(10):469-476. <br />       scores) gene Wormnet core network [37].                                                                                                                                                                                                                              12.   Pomroy  Anthelmintic resistance New Zealand: perspective       Additional file 4 Phylogenetic trees nematode OrthoMCL clus-                                                                                                                                                                                                                       recent findings options future. New Zealand Veterinary <br />       ters. file contains phylogenetic trees illustrative gene clusters                                                                                                                                                                                                            Journal 2006, 54(December):265-270. <br />       include ESTs parasitic nematodes.                                                                                                                                                                                                                                              13.   Besier B: New anthelmintics livestock: time right. Trends <br />       Additional file 5 Essential phenotypes C. elegans M. muscu-                                                                                                                                                                                                                       Parasitol 2007, 23(1):21-24. <br />       lus.doc. file contains phenotype descriptions C. elegans                                                                                                                                                                                                       14.   Gilleard JS, Beech RN: Population genetics anthelmintic resistance       M. musculus genes classed essential genes current analy-                                                                                                                                                                                                              parasitic nematodes. Parasitology 2007, 134(Pt 8):1133-1147. <br />       sis.                                                                                                                                                                                                                                                                                15.   Nguyen M, Alfonso  Johnson CD, Rand JB: Caenorhabditis elegans <br />                                                                                                                                                                                                                                                                                                 mutants resistant inhibitors acetylcholinesterase. Genetics 1995, <br /> Abbreviations                                                                                                                                                                                                                                                                                   140(2):527-535. <br /> GPCR: G-protein coupled receptor; EST: expressed sequence tag;  Gene                                                                                                                                                                                                                   16.   Lewis JA, Wu CH, Levine JH, Berg H: Levamisole-resistant mutants Ontology; LLS: log-likelihood score.                                                                                                                                                                                                                                                            nematode Caenorhabditis elegans appear lack pharmacological <br />                                                                                                                                                                                                                                                                                                 acetylcholine receptors. Neuroscience 1980, 5(6):967-989. <br /> Authors&apos; contributions                                                                                                                                                                                                                                                                    17.   Campbell  Nagaraj SH, Hu M, Zhong W, Sternberg PW, Ong EK, Loukas <br /> Project conceived SAR RBG. MAD BJW performed bioinfor-                                                                                                                                                                                                                        Ranganathan S, Beveridge  McInnes RL, et al.: Gender-enriched <br /> matic experiments, BJW assisted creation development data-                                                                                                                                                                                                                      transcripts Haemonchus contortus--predicted functions genetic <br /> base, MAD, BJW, RSH, SAR RBG analysed data. SAR, RBG MAD wrote                                                                                                                                                                                                                      interactions based comparative analyses Caenorhabditis <br /> manuscript, authors read approved final manuscript.                                                                                                                                                                                                                                 elegans. Int J Parasitol 2008, 38(1):65-83. <br />                                                                                                                                                                                                                                                                                           18.   Hoekstra R, Visser  Otsen M, Tibben J, Lenstra JA, Roos MH: EST <br /> Acknowledgements                                                                                                                                                                                                                                                                                sequencing parasitic nematode Haemonchus contortus authors wish thank Wormbase, Flybase MGI teams assis-                                                                                                                                                                                                                        suggests shift gene expression transition parasitic <br /> tance data access. SAR supported Future Fellowship (FT0990350)                                                                                                                                                                                                                     stages. Mol Biochem Parasitol 2000, 110(1):53-68. Australian Research Council (ARC). Financial support present                                                                                                                                                                                                              19.   McCarter JP, Mitreva MD, Martin J, Dante M, Wylie T, Rao U, Pape D, <br /> study provided grants RBG ARC.                                                                                                                                                                                                                                               Bowers Y, Theising B, Murphy CV, et al.: Analysis functional <br />                                                                                                                                                                                                                                                                                                 classification transcripts nematode Meloidogyne <br />                                                                                                                                                                                                                                                                                                 incognita. Genome Biol 2003, 4(4):R26. <br /> Author Details <br /> 1Department                                                                                                                                                                                                                                                                               20.   Mitreva M, Jasmer DP, Appleton J, Martin J, Dante M, Wylie T, Clifton SW, <br />               Biochemistry &amp; Molecular Biology, Bio21 Molecular Science <br />                                                                                                                                                                                                                                                                                                 Waterston RH, McCarter JP: Gene discovery adenophorean Biotechnology Institute, University Melbourne, Victoria, 3010, <br />                                                                                                                                                                                                                                                                                                 nematode Trichinella spiralis: analysis transcription Australia, 2Department Veterinary Science, University Melbourne, 250 <br />                                                                                                                                                                                                                                                                                                 life cycle stages. Mol Biochem Parasitol 2004, 137(2):277-291. <br /> Princes Highway, Werribee, Victoria, 3030, Australia 3Current address: Peter <br />                                                                                                                                                                                                                                                                                           21.   Nisbet AJ, Gasser RB: Profiling gender-specific gene expression MacCallum Cancer Centre, St Andrew&apos;s Place, East Melbourne Victoria, 3002, <br />                                                                                                                                                                                                                                                                                                 Trichostrongylus vitrinus (Nematoda: Strongylida) microarray <br /> Australia <br />                                                                                                                                                                                                                                                                                                 analysis expressed sequence tag libraries constructed Received: 11 September 2009 Accepted: 3 April 2010                                                                                                                                                                                                                                              suppressive-subtractive hybridisation. Int J Parasitol 2004, <br /> Published: 3 April 2010                                                                                                                                                                                                                                                                         34(5):633-643. &#194;&#169; <br /> BMC <br />   2010 <br />           article available <br />                    Access <br />       Genomics <br />          Doyle <br />            Open  et2010,    articleBioMed <br />                                      distributed <br />                     al; licensee <br />                            11:222 <br />                                        Central terms Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction medium, provided original work properly cited. <br />                                    http://www.biomedcentral.com/1471-2164/11/222 <br />                                                                                                                                                                                                                                                                                                                                            22.   Thompson FJ, Mitreva M, Barker GL, Martin J, Waterston RH, McCarter JP, <br />                                                                                                                                                                                                                                                                                                 Viney  expressed sequence tag analysis life-cycle References                                                                                                                                                                                                                                                                                      parasitic nematode Strongyloides ratti. Mol Biochem Parasitol 2005, <br /> 1. Trouiller P, Olliaro P, Torreele E, Orbinski J, Laing R, Ford N: Drug                                                                                                                                                                                                                        142(1):32-46. <br />     development neglected diseases: deficient market public-                                                                                                                                                                                                                  23.   Yin Y, Martin J, McCarter JP, Clifton SW, Wilson RK, Mitreva M: <br />     health policy failure. Lancet 2002, 359(9324):2188-2194.                                                                                                                                                                                                                                    Identification analysis genes expressed adult filarial <br /> 2. Nwaka S, Hudson  Innovative lead discovery strategies tropical                                                                                                                                                                                                                         parasitic nematode Dirofilaria immitis. Int J Parasitol 2006, <br />     diseases. Nat Rev Drug Discov 2006, 5(11):941-955.                                                                                                                                                                                                                                          36(7):829-839. <br /> 3. Gerdes SY, Scholle MD, Campbell JW, Balazsi G, Ravasz E, Daugherty MD,                                                                                                                                                                                                                 24.   Ghedin E, Wang S, Spiro D, Caler E, Zhao Q, Crabtree J, Allen JE, Delcher AL, <br />     Somera AL, Kyrpides NC, Anderson  Gelfand MS, et al.: Experimental                                                                                                                                                                                                                        Guiliano DB, Miranda-Saavedra D, et al.: Draft genome filarial <br />     determination level analysis essential genes                                                                                                                                                                                                                               nematode parasite Brugia malayi. Science 2007, 317(5845):1756-1760. <br />     Escherichia coli MG1655. J Bacteriol 2003, 185(19):5673-5684.                                                                                                                                                                                                                         25.   Mitreva M, Zarlenga DS, McCarter JP, Jasmer DP: Parasitic nematodes - <br /> 4. Giaever G, Chu  Ni L, Connelly C, Riles L, Veronneau S, Dow S, Lucau-                                                                                                                                                                                                                     genomes control. Vet Parasitol 2007, 148(1):31-42. <br />     Danila  Anderson K, Andre B, et al.: Functional profiling                                                                                                                                                                                                                    26.   Gilleard JS, Woods DJ, Dow JAT: Model-organism genomics veterinary <br />     Saccharomyces cerevisiae genome. Nature 2002, 418(6896):387-391.                                                                                                                                                                                                                            parasite drug-discovery. Trends Parasitol 2005, 21(7):302-305. <br />  Doyle et al. BMC Genomics 2010, 11:222                                                                                                                 Page 14 14 <br /> http://www.biomedcentral.com/1471-2164/11/222 <br />  <br />  <br />  <br /> 27. Geldhof P, Murray L, Couthier  Gilleard JS, McLauchlan G, Knox DP,                 <span id='am-20' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-21' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> 200 beta-tubulin isotype 1. Mol Biochem Parasitol 1994, <br />     Britton C: Testing efficacy RNA interference Haemonchus                    63(2):299-303. <br />     contortus. Int J Parasitol 2006, 36(7):801-810.                                52.   Cully DF, Vassilatis DK, Liu KK, Paress PS, Ploeg LH Van der, Schaeffer JM, <br /> 28. Geldhof P, Visser  Clark D, Saunders G, Britton C, Gilleard J, Berriman M,         Arena JP: Cloning avermectin-sensitive glutamate-gated chloride <br />     Knox D: RNA interference parasitic helminths: current situation,                  channel Caenorhabditis elegans. Nature 1994, 371(6499):707-711. <br />     potential pitfalls future prospects. Parasitology 2007, 134(Pt             53.   Prichard RK: Mode action anthelminthic thiabendazole     5):609-619.                                                                          Haemonchus contortus. Nature 1970, 228(5272):684-685. <br /> 29. Knox DP, Geldhof P, Visser  Britton C: RNA interference parasitic         54.   Dent JA, Smith MM, Vassilatis DK, Avery L: genetics ivermectin <br />     nematodes animals: reality check? Trends Parasitol 2007,                        resistance Caenorhabditis elegans. Proc Natl Acad Sci USA 2000, <br />     23(3):105-107.                                                                       97(6):2674-2679. <br /> 30. Chen F, Mackey AJ, Stoeckert CJ Jr, Roos DS: OrthoMCL-DB: querying           55.   Martin RJ: Modes action anthelmintic drugs. Vet J 1997, <br />     comprehensive multi-species collection ortholog groups. Nucleic                   154(1):11-34. <br />     Acids Res 2006:D363-368.                                                       56.   Thanassi JA, Hartman-Neumann SL, Dougherty TJ, Dougherty BA, Pucci <br /> 31. Kamath RS, Ahringer J: Genome-wide RNAi screening Caenorhabditis                  MJ: Identification 113 conserved essential genes using high- <br />     elegans. Methods 2003, 30(4):313-321.                                                throughput gene disruption Streptococcus pneumoniae. <br /> 32. Kamath RS, Fraser AG, Dong Y, Poulin G, Durbin R, Gotta M, Kanapin  Le             Nucleic Acids Res 2002, 30(14):3152-3162. <br />     Bot N, Moreno S, Sohrmann M, et al.: Systematic functional analysis         57.   Roemer T, Jiang B, Davison J, Ketela T, Veillette K, Breton  Tandia F, <br />     Caenorhabditis elegans genome using RNAi. Nature 2003,                           Linteau  Sillaots S, Marta C, et al.: Large-scale essential gene <br />     421(6920):231-237.                                                                   identification Candida albicans applications antifungal drug <br /> 33. Mummery-Widmer JL, Yamazaki M, Stoeger T, Novatchkova M, Bhalerao S,                 discovery. Mol Microbiol 2003, 50(1):167-181. <br />     Chen D, Dietzl G, Dickson BJ, Knoblich JA: Genome-wide analysis             58.   Hu W, Sillaots S, Lemieux S, Davison J, Kauffman S, Breton  Linteau  Xin <br />     Notch signalling Drosophila transgenic RNAi. Nature 2009,                      C, Bowman J, Becker J, et al.: Essential gene identification drug <br />     458(7241):987-992.                                                                   target prioritization Aspergillus fumigatus. PLoS Pathog 2007, <br /> 34. Blaxter M: Caenorhabditis elegans nematode. Science 1998,                       3(3):e24. <br />     282(5396):2041-2046.                                                           59.   Aguero F, Al-Lazikani B, Aslett M, Berriman M, Buckner FS, Campbell RK, <br /> 35. Gilleard JS: use Caenorhabditis elegans parasitic nematode                 Carmona S, Carruthers IM, Chan AW, Chen F, et al.: Genomic-scale <br />     research. Parasitology 2004, 128(Suppl 1):49-70.                                     prioritization drug targets: TDR Targets database. Nat Rev Drug <br /> 36. Zhong W, Sternberg PW: Genome-wide prediction C. elegans genetic                  Discov 2008, 7(11):900-907. <br />     interactions. Science 2006, 311(5766):1481-1484.                               60.   Joyce AR, Palsson BO: Predicting gene essentiality using genome-scale <br /> 37. Lee  Lehner B, Crombie C, Wong W, Fraser AG, Marcotte EM: single                 silico models. Methods Mol Biol 2008, 416:433-457. <br />     gene network accurately predicts phenotypic effects gene                    61.   Lamichhane G, Zignol M, Blades NJ, Geiman  Dougherty  Grosset J, <br />     perturbation Caenorhabditis elegans. Nat Genet 2008, 40(2):181-188.               Broman KW, Bishai WR: postgenomic method predicting essential <br /> 38. Zhang K, Rathod PK: Divergent regulation dihydrofolate reductase                  genes subsaturation levels mutagenesis: application     malaria parasite human host. Science 2002,                               Mycobacterium tuberculosis. Proc Natl Acad Sci USA 2003, <br />     296(5567):545-547.                                                                   100(12):7213-7218. <br /> 39. Bacchi CJ, Nathan HC, Hutner SH, McCann PP, Sjoerdsma  Polyamine             62.   Seringhaus M, Paccanaro  Borneman  Snyder M, Gerstein M: <br />     metabolism: potential therapeutic target trypanosomes. Science                  Predicting essential genes fungal genomes. Genome Res 2006, <br />     1980, 210(4467):332-334.                                                             16(9):1126-1135. <br /> 40. Kraus JM, Verlinde CL, Karimi M, Lepesheva GI, Gelb MH, Buckner FS:            63.   Volker C, Brown JR: Bioinformatics discovery novel anti- <br />     Rational modification candidate cancer drug use                     microbial targets. Curr Drug Targets Infect Disord 2002, 2(4):279-290. <br />     Chagas disease. J Med Chem 2009, 52(6):1639-1647.                              64.   Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, Canadien V, <br /> 41. Silvian LF, Wang J, Steitz TA: Insights editing ile-tRNA                Starostine  Richards D, Beattie B, Krogan N, et al.: Interaction network <br />     synthetase structure tRNAile mupirocin. Science 1999,                       containing conserved essential protein complexes Escherichia <br />     285(5430):1074-1077.                                                                 coli. Nature 2005, 433(7025):531-537. <br /> 42. Conant GC, Wagner  Duplicate genes robustness transient               65.   Krogan NJ, Cagney G, Yu H, Zhong G, Guo X, Ignatchenko  Li J, Pu S, <br />     gene knock-downs Caenorhabditis elegans. Proc Biol Sci 2004,                      Datta N, Tikuisis AP, et al.: Global landscape protein complexes     271(1534):89-96.                                                                     yeast Saccharomyces cerevisiae. Nature 2006, 440(7084):637-643. <br /> 43. Gu X: Evolution duplicate genes versus genetic robustness           66.   Overington JP, Al-Lazikani B, Hopkins AL: drug targets     null mutations. Trends Genet 2003, 19(7):354-356.                                     Nat Rev Drug Discov 2006, 5(12):993-996. <br /> 44. Liao B-Y, Zhang J: Null mutations human mouse orthologs                 67.   Sonnhammer EL, Koonin EV: Orthology, paralogy proposed <br />     frequently result different phenotypes. Proc Natl Acad Sci USA 2008,              classification paralog subtypes. Trends Genet 2002, 18(12):619-620. <br />     105(19):6987-6992.                                                             68.   Li L, Stoeckert CJ Jr, Roos DS: OrthoMCL: identification ortholog <br /> 45. Mushegian AR, Koonin EV: minimal gene set cellular life derived             groups eukaryotic genomes. Genome Res 2003, 13(9):2178-2189. <br />     comparison complete bacterial genomes. Proc Natl Acad Sci USA               69.   Harris TW, Antoshechkin  Bieri T, Blasiar D, Chan J, Chen WJ, La Cruz N, <br />     1996, 93(19):10268-10273.                                                            Davis P, Duesbury M, Fang R, et al.: WormBase: comprehensive <br /> 46. Amsterdam  Nissen RM, Sun  Swindell EC, Farrington S, Hopkins N:                 resource nematode research. Nucleic Acids Res 2010:D463-467. <br />     Identification 315 genes essential early zebrafish development.         70.   Bult CJ, Kadin JA, Richardson JE, Blake JA, Eppig JT: Mouse Genome <br />     Proc Natl Acad Sci USA 2004, 101(35):12792-12797.         <br /> </body></html>