<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                                     JCB: ARTICLE <br />  <br />  <br />  <br />  <br />                               Excess Mcm2&#226;&#8364;&#8220;7 license dormant origins <br />                               replication used conditions <br />                               replicative stress <br />                               Anna M. Woodward,1 Thomas G&#195;&#182;hler,1 M. Gloria Luciani,1 Maren Oehlmann,1 Xinquan Ge,1 Anton Gartner,1 <br />                               Dean  Jackson,2 J. Julian Blow1 <br />                               1 <br />                                Wellcome Trust Biocentre, University Dundee, Dundee DD1 5EH, Scotland, UK <br />                               2 <br />                                Department Biomolecular Sciences, University Manchester Institute Science Technology, Manchester, M60 1QD, England, UK <br />  <br />  <br />  <br />  <br />                                                                n late mitosis early G1, replication origins                             allow complete genome replication presence                                  licensed subsequent use loading complexes                          replication inhibitors. results suggest metazoan JOURNAL CELL BIOLOGY <br />  <br />  <br />  <br />  <br />                                  minichromosome maintenance proteins 2&#226;&#8364;&#8220;7 (Mcm2&#226;&#8364;&#8220;7).                                replication origins actually comprised  <br />                               number Mcm2&#226;&#8364;&#8220;7 complexes loaded DNA                                      didate origins, normally remain dormant <br />                               greatly exceeds number replication origins used                              unless cells experience replicative stress. Consistent                               S phase, function excess Mcm2&#226;&#8364;&#8220;7                               model, using Caenorhabditis elegans,                               unknown. Using Xenopus laevis egg extracts,                              partial RNAi-based knockdown MCMs                               excess Mcm2&#226;&#8364;&#8220;7 complexes license additional                               observable effect normal conditions causes lethality <br />                               dormant origins &#239;&#172; unperturbed                                  treatment low, nontoxic, levels                               S phases suppression caffeine-sensitive                             replication inhibitor hydroxyurea. <br />                               checkpoint pathway. Use additional origins  <br />  <br />  <br />  <br />                               Introduction <br />                               eukaryotes, precise duplication genome S phase                     new origins prevented late G1 midmitosis <br />                               achieved initiation replication numerous ori-                  (Diffley, 2004; Blow Dutta, 2005). means  <br />                               gins distributed DNA. late mitosis early                      problems encountered replication forks                               G1, origins licensed replication loading complexes                       S phase, previously licensed sites used.  <br />                               minichromosome maintenance proteins 2&#226;&#8364;&#8220;7 (Mcm2&#226;&#8364;&#8220;7),                            verging replication forks stall irreversibly S phase,                                forming &#226;&#8364;&#339;prereplicative complex&#226;&#8364;? (Diffley, 2004; Blow                       DNA probably remain unreplicated, poten- <br />                               Dutta, 2005). Licensing involves coordinated action                      tially leading cell death chromosome rearrangement. <br />                               origin recognition complex (ORC) Cdc6 Cdt1 proteins                          crucial cells organize replication <br />                               (Gillespie et al., 2001), probably act clamp Mcm2&#226;&#8364;&#8220;7                        machinery minimize risk happening. <br />                               DNA. Mcm2&#226;&#8364;&#8220;7 displaced replication ori-                                 Previous work range eukaryotes, including <br />                               gins initiate, suggested provide                    Saccharomyces cerevisiae, humans, Xenopus laevis,                               helicase activity unwind DNA ahead replication fork                       demonstrated Mcm2&#226;&#8364;&#8220;7 complexes loaded DNA                               (Labib Diffley, 2001). prevent replication origins                       20-fold excess number DNA-bound ORC mol- <br />                               firing single S phase, ability license                   ecules number replication origins (Burkhart et al., <br />                                                                                                                   1995; Lei et al., 1996; Rowles et al., 1996; Donovan et al., <br />                               Correspondence J. Julian Blow: j.j.blow@dundee.ac.uk                             1997; Mahbubani et al., 1997; Edwards et al., 2002).  <br />                               M.G. Luciani&#226;&#8364;&#8482;s present address Dept. Medicine, Vienna General Hospital,       normal replication rates maintained number                               1090 Vienna, Austria.                                                               Mcm2&#226;&#8364;&#8220;7 molecules reduced 1&#226;&#8364;&#8220;2 origin (Mahbubani <br />                               M. Oehlmann&#226;&#8364;&#8482;s present address Dept. Biochemistry, National University         et al., 1997; Edwards et al., 2002; Cortez et al., 2004; Oehlmann <br />                               Ireland, Galway, Ireland. <br />                               Abbreviations used paper: HU, hydroxyurea; MCM, minichromosome <br />                                                                                                                   et al., 2004; Tsao et al., 2004). observation, termed                               maintenance; ODP, origin decision point; ORC, origin recognition complex.           &#226;&#8364;&#339;MCM paradox&#226;&#8364;? (Hyrien et al., 2003), led speculation                               online version article contains supplemental material.                  possible function extra complexes, particularly  <br />                               &#194;&#169; Rockefeller University Press $8.00 <br />                               Journal Cell Biology, Vol. 173,  5, June 5, 2006 673&#226;&#8364;&#8220;683 <br />                               http://www.jcb.org/cgi/doi/10.1083/jcb.200602108 <br />                                                                                                                                                                                   JCB   673 <br />        MCM proteins exclusively colocalize sites DNA <br />       synthesis S phase (Madine et al., 1995; Krude et al., 1996; <br />       Dimitrova et al., 1999). <br />              evidence Mcm2&#226;&#8364;&#8220;7 complexes       DNA significant distances ORC bound (Ritzi <br />       et al., 1998; Edwards et al., 2002; Harvey Newport, 2003; <br />       Danis et al., 2004). suggested distant <br />       multiple MCM complexes cooperatively pump double- <br />       stranded DNA   unwind (Laskey Madine, <br />       2003). recently, role excess MCM proteins       checkpoint activation proposed, based obser- <br />       vation cells partially depleted Mcm7 display reduc- <br />       tion replication checkpoint signaling (Cortez et al., 2004; <br />       Tsao et al., 2004). possibility suggested <br />       excess Mcm2&#226;&#8364;&#8220;7 provide replication origins use <br />       certain contingencies, incomplete DNA replica- <br />       tion (Lucas et al., 2000; Edwards et al., 2002; Hyrien et al., <br />       2003; Blow Dutta, 2005). different proposals       mutually exclusive. <br />              study, demonstrate role excess Mcm2&#226;&#8364;&#8220;7 <br />       complexes licensing &#226;&#8364;&#339;dormant&#226;&#8364;? replication origins.       dormant origins effectively suppressed unperturbed         Figure 1. Replication characteristics minimally licensed DNA.   Sperm <br />       DNA replication, support initiation replication          nuclei incubated X. laevis egg extract supplemented biotin-16- <br />                                                                             <span id='am-16' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="dUTP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span>dUTP</span>. produce minimally licensed DNA, extract supplemented <br />       forks stalled response range replicative stresses,     geminin shortly sperm addition. 30 min, DNA isolated <br />       including replication inhibition aphidicolin, mitomycin C,         spread glass slides, biotin-labeled tracks detected <br />       etoposide, actinomycin D. support model, dem-        &#239;&#172;&#8218;uorescent antibodies. distance center points                                                                             adjacent tracks labeled DNA measured. (B) Sperm nuclei       onstrate Caenorhabditis elegans, partial knockdown            incubated X. laevis egg extract containing &#206;&#177;-[32P]<span id='am-11' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span>dATP</span>, plus minus <br />       MCMs induces hypersensitivity nontoxic levels         aphidicolin caffeine. produce minimally licensed DNA, extract <br />       hydroxyurea (HU).                                                  supplemented geminin shortly sperm addition. indi- <br />                                                                             cated times, total DNA synthesized measured. <br />  <br />  <br />       Results <br />                                                                             mum licensing; 17.1 kb minimum licensing). normal <br />       Minimally licensed chromatin replicates                               circumstances, clusters 3&#226;&#8364;&#8220;7 adjacent origins initiate simi- <br />       poorly presence aphidicolin                                 lar times, different clusters activated different <br />       caffeine                                                          stages S phase (Blow et al., 2001). observed major <br />       demembranated sperm nuclei incubated X. laevis            difference minimally maximally licensed DNA <br />       egg extracts, average 10&#226;&#8364;&#8220;20 Mcm2&#226;&#8364;&#8220;7 complexes                clustering replication origins, average 6.1 <br />       loaded replication origin entry S phase         origins cluster maximally licensed DNA 4.8       (Mahbubani et al., 1997; Edwards et al., 2002; Oehlmann et al.,       minimally licensed DNA.  conclude minimum <br />       2004); DNA &#226;&#8364;&#339;maximally licensed.&#226;&#8364;? Cdt1             licensing does significantly change replication origin usage <br />       inhibitor geminin added extract shortly sperm     normal conditions. <br />       DNA, number DNA-bound MCM complexes                             examined excess Mcm2&#226;&#8364;&#8220;7 complexes <br />       limited minimum required support approximately              important replication forks stress       normal replication kinetics (Oehlmann et al., 2004);     supplementing extracts low concentrations DNA <br />       DNA &#226;&#8364;&#339;minimally licensed.&#226;&#8364;? examine role excess               polymerase inhibitor aphidicolin. Fig. 1 B (crosses closed <br />       Mcm2&#226;&#8364;&#8220;7 complexes, investigated minimum               circles) shows replication minimally maximally li- <br />       licensing causes alteration spacing adjacent        censed DNA inhibited similar extents 10 &#206;&#188;M aphidicolin. <br />       replication origins. Nascent DNA labeled biotin-16-          previously shown  concentration, aphidico- <br />       dUTP early S phase, DNA isolated,          lin slows replication forks approximately threefold,       spread microscope slides, analyzed fluorescence             major effect replication induce ATR-dependent <br />       microscopy. Short fluorescent tracks seen,            checkpoint suppresses initiation events (Luciani <br />       caused bidirectional replication origins center     et al., 2004).  supplemented extracts       track, spacing replication origins           caffeine, ATR kinase inhibitor (Fig. 1 B, crossed <br />       determined measuring distance centers           boxes). Consistent previous study (Luciani et al., 2004), <br />       adjacent tracks (Herrick et al., 2000; Blow et al., 2001). Fig. 1   addition caffeine completely rescued aphidicolin- <br />       shows significant difference average         induced inhibition replication maximally licensed <br />       interorigin distance samples (15.8 kb maxi-       DNA. contrast, caffeine partially rescued replication <br />  <br />  <br /> 674   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  minimally licensed DNA replicated presence aphidicolin. suggests excess Mcm2&#226;&#8364;&#8220;7 complexes required way allow rescue DNA replication inhibition replication fork progression. <br />  <br /> Maximally licensed chromatin use <br /> extra origins replication <br /> Replication forks X. laevis normally progress  10 bp/s <br /> (Callan, 1972; Mahbubani et al., 1992; Strausfeld et al., 1994; <br /> Walter Newport, 1997), slowed approximately <br /> threefold 10 &#206;&#188;M aphidicolin (Luciani et al., 2004). Fork rate significantly affected presence caffeine (Luciani <br /> et al., 2004). replicate normal rates presence aphidicolin caffeine, maximally licensed DNA use replication forks normal. Minimally licensed DNA, contrast, does appear capable using replication <br /> forks.  assessed number active forks measuring quantity Cdc45 loaded chromatin. Cdc45 <br /> loads replication forks (probably binding Mcm2&#226;&#8364;&#8220;7) just initiation moves forks elongate <br /> (Hopwood Dalton, 1996; Aparicio et al., 1997, 1999; Zou Stillman, 1998; Mimura Takisawa, 1998). Fig. 2 shows maximally minimally licensed chromatin <br /> contained similar quantities Cdc45 undisturbed <br /> S phase, expected similar replication rates.  <br /> sistent previous study (Edwards et al., 2002), Cdc45 maximally licensed chromatin increased  20-fold           Figure 2. Caffeine allows initiation extra forks maximally extract treated combination aphidicolin            licensed DNA.   Sperm nuclei incubated X. laevis egg extract, caffeine. contrast, minimally licensed chromatin showed        plus minus aphidicolin caffeine. produce minimally licensed <br />                                                                       DNA, extract supplemented geminin shortly sperm addition.  threefold increase similar conditions.           40 min, chromatin isolated immunoblotted Cdc45. slight increase chromatin-bound Cdc45 seen         Coomassie-stained histones shown loading control. (B) Sperm maximally minimally licensed chromatin treated          nuclei (400 ng DNA) incubated different volumes egg extract <br />                                                                       supplemented 40 &#206;&#188;M aphidicolin plus caffeine. produce minimally caffeine, consistent acceleration       licensed DNA, extract supplemented geminin shortly sperm <br /> origin firing caffeine causes (Marheineke Hyrien, 2004;      addition. 40 min, chromatin isolated immunoblotted Shechter et al., 2004).                                               Cdc45. (C) Sperm nuclei incubated egg extract supplemented                                                                       biotin-16-dUTP caffeine. produce minimally licensed DNA, extract <br />         performed quantitative immunoblotting esti-        supplemented geminin shortly sperm addition. 30 min, <br /> mate Cdc45 loaded chromatin            DNA isolated spread glass slides, biotin-labeled tracks <br /> conditions (Fig. 2 B). fixed quantity X. laevis sperm nuclei     detected &#239;&#172;&#8218;uorescent antibodies. distance center <br />                                                                       points adjacent tracks labeled DNA measured. incubated increasing quantities egg extract supple- <br /> mented aphidicolin caffeine. Extracts optionally <br /> supplemented geminin shortly sperm addition                   results suggest maximally licensed DNA create minimally licensed chromatin. absence added          contain increased density nascent strands conditions <br /> geminin (maximum licensing), quantity chromatin-bound          increased number forks. recent <br /> Cdc45 increased increasing extract volume (Fig. 2 B,             studies shown caffeine accelerate origin firing <br /> shaded bars), caused,  increased quantities Mcm2&#226;&#8364;&#8220;7      (Marheineke Hyrien, 2004; Shechter et al., 2004),  loaded DNA (Mahbubani et al., 1997).            peated fluorescent fiber-labeling experiments using extracts <br /> minimally licensed chromatin,  Cdc45 remained              supplemented caffeine. Fig. 2 C (left) shows fairly constant level 3&#226;&#8364;&#8220;5 ng (Fig. 2 B, open bars). level   maximally licensed chromatin replicated extract contain- Cdc45 minimally licensed chromatin corresponds 1&#226;&#8364;&#8220;2           ing caffeine average origin spacing declined dramatically <br /> molecules Cdc45 10-kb DNA, 1&#226;&#8364;&#8220;2 molecules Cdc45           (compare Fig. 1 , observed spacing na- molecule ORC (Rowles et al., 1996). single       scent strands falling near resolution limit technique <br /> molecule Cdc45 replication fork, results sug-        (1&#226;&#8364;&#8220;2 kb). fiber-labeling experiments repeated <br /> gest normal conditions single molecule ORC            minimally licensed chromatin, reduction spacing allows, average, single pair replication forks initiate;   pronounced maximally licensed DNA (Fig. number forks increased certain circum-            2 C, right). data suggest extra Mcm2&#226;&#8364;&#8220;7 molecules <br /> stances, excess Mcm2&#226;&#8364;&#8220;7 molecules present        maximally licensed DNA license dormant replication origins DNA.                                                           used presence caffeine. <br />  <br />  <br />                                                           EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                  675 <br />                                                                                                correct, nascent strands addi- <br />                                                                                        tional origins rapidly fuse high molecular weight <br />                                                                                        products observable alkaline agarose gels (Fig. 3  <br />                                                                                        left). minimally licensed DNA, interorigin distances                                                                                        larger, nascent strand fusion occur later (Fig. 3  <br />                                                                                        right). test  maximally minimally licensed DNA sam- <br />                                                                                        ples pulsed &#206;&#177;-[32P]dATP beginning S phase, <br />                                                                                        chased unlabeled dATP different periods. <br />                                                                                        Fig. 3 B  iv) shows maximally minimally licensed <br />                                                                                        DNA gave similar profiles absence drug <br />                                                                                        treatment. long smear nascent strands makes                                                                                        accurate rate determination impossible, modal size na- <br />                                                                                        scent strands increased rate  25 nt/s. expected <br />                                                                                        error, consistent strand elongated                                                                                        forks, moving 10 nt/s, previously reported (Callan, <br />                                                                                        1972; Mahbubani et al., 1992; Strausfeld et al., 1994; Walter <br />                                                                                        Newport, 1997; Luciani et al., 2004). <br />                                                                                               contrast, behavior maximally minimally <br />                                                                                        licensed DNA presence aphidicolin caffeine (Fig. 3 B, <br />                                                                                        ii v) showed dramatic difference consistent                                                                                        model shown Fig. 3  highlighted Fig. 3 C, <br />                                                                                        relative abundance nascent strands sample <br />                                                                                        lane corresponding alkaline gel plotted,                                                                                        earliest sample lane plotted nearest  maximally                                                                                        minimally licensed nascent strands started uni- <br />                                                                                        form size untreated sample aphidicolin                                                                                        limited rate elongation. minutes,                                                                                        maximally minimally licensed nascent strands increased                                                                                        size  6 nt/s ( 3 nt/s fork). minimally licensed <br />                                                                                        sample (Fig. 3, B [v] C [v]), strands maintained                                                                                        rate time course. later samples,                                                                                         strands doubled size, consistent fusion                                                                                        adjacent strands (Fig. 3 B, v, arrows). larger strands                                                                                        visible  10 kb, consistent                                                                                        fusions smallest replicons  5 kb seen Fig. 1  <br />                                                                                        different pattern seen maximally licensed DNA <br />                                                                                        (Fig. 3, B [ii] C [ii]). smear higher molecular weight <br />                                                                                        strands built rapidly did minimally <br />                                                                                        licensed sample, middle time series ma- <br />                                                                                        jority strands shifted higher molecular weight <br />                                                                                        form. end time series, majority strands <br />                                                                                        exclusion limit gel. Unlike case                                                                                        minimally licensed DNA, obvious lower size limit <br />                                                                                        strand fusion events occurred. results suggest <br />  <br />  <br />       Figure 3. Maximally licensed DNA undergo premature nascent <br />       strand fusion presence caffeine.   Model use addi-        supplemented &#206;&#177;-[32P]dATP, chased unlabeled <br />       tional dormant origins extract treated aphidicolin caffeine.         dATP 2 min   iii, iv, vi) 5 min (ii v). different <br />       small segment ( 30 kb) chromosomal DNA shown, normally             times DNA isolated, separated alkaline agarose gel <br />       supports initiation origins. Candidate replication origins       electrophoresis, autoradiographed. times minute <br />       bound Mcm2&#226;&#8364;&#8220;7 shown gray circles. Nascent strands shown             27&#226;&#8364;&#8220;36 min. times ii 31, 33, minute 35&#226;&#8364;&#8220;45, <br />       double-headed arrows. maximally licensed DNA (left), nascent strands          48, 51, 54 min. times iii minute 23&#226;&#8364;&#8220;32 min. <br />       initiate dormant origins rapidly fuse. minimally licensed chro-      times iv minute 23&#226;&#8364;&#8220;30, 32, 34, 36, 40 min. <br />       matin (right), dormant origins &#239;&#172;?rst strand fusion events     times v 2 min 32&#226;&#8364;&#8220;50, 55, 60 min. times <br />       occur closely spaced origins. (B C) Sperm nuclei            vi min 23&#226;&#8364;&#8220;30, 32, 34, 36, 40 min. Molecular <br />       incubated X. laevis egg extract, plus minus aphidicolin caffeine.   weight markers (&#206;&#187;-HindIII) shown left (sizes kilobases).       produce minimally licensed DNA, extract supplemented                 autoradiographs shown B. x-ray &#239;&#172;?lm scanned       geminin shortly sperm addition. iii, maximally licensed DNA; iv&#226;&#8364;&#8220;vi,      density <span id='am-34' about='rdfs:label' typeof='owl:Thing'>label</span> lane quanti&#239;&#172;?ed. scans lane shown <br />       minimally licensed DNA. iv, additions; ii v, plus caffeine      C (earliest times  slowest migration right). migration <br />       aphidicolin; iii vi, plus caffeine. start S phase, extract     &#206;&#187;-HindIII shown ticks  <br />  <br />  <br /> 676   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />                                                                                 presence aphidicolin caffeine, maximally li- <br />                                                                                censed DNA displays dramatic reduction replicon size,                                                                                effect seen minimally licensed DNA. Note <br />                                                                                 consistent previous work (Luciani et al., 2004), <br />                                                                                sign caffeine reducing fork stability, did <br />                                                                                observe buildup low molecular weight products. <br />                                                                                       extracts supplemented caffeine  <br />                                                                                obvious difference maximally mini- <br />                                                                                mally licensed DNA, samples yielding fully repli- <br />                                                                                cated product (Fig. 3 B, iii vi). fiber-labeling experiments <br />                                                                                caffeine (Fig. 2 C) predict caffeine increase <br />                                                                                number active forks DNA maximally li- <br />                                                                                censed.  fork-slowing effect aphidico- <br />                                                                                lin, rapid fusion nascent strands arising                                                                                additional origins dormant absence caf- <br />                                                                                feine. initial fusion events taken place                                                                                maximally licensed sample, maximally minimally li- <br />                                                                                censed samples behave similarly. effect explains                                                                                high levels Cdc45 loading maximally <br />                                                                                licensed DNA treated caffeine (Fig. 2 . <br />  <br />                                                                                extra origins normally &#239;&#172;                                                                                results far suggest excess Mcm2&#226;&#8364;&#8220;7 complexes li- <br />                                                                                cense dormant origins, used normal condi- <br />                                                                                tions, rescue DNA synthesis response <br />                                                                                replication fork inhibition. known Cdks required <br />                                                                                initiation replication forks X. laevis                                                                                 fork elongation (Strausfeld et al., 1994; Luciani et al., <br />                                                                                2004). known Cdks act individual origins                                                                                shortly origin fires (Strausfeld et al., 1994; Luciani <br />                                                                                et al., 2004).  predicted Cdks inhib- <br />                                                                                ited start S phase, origins                                                                                fired, dormant origin firing prevented rescue <br />                                                                                DNA replication place. inhibit Cdks, used <br />                                                                                roscovitine, purine analogue inhibits initiation <br />                                                                                X. laevis (Meijer et al., 1997; Luciani et al., <br />                                                                                2004). model predicted behavior shown Fig. 4  <br />                                                                                model left shows initiation occurring dormant <br />                                                                                origins replication occurs presence aphidicolin <br />                                                                                caffeine. model right shows roscovitine <br />                                                                                added shortly origins fired, initiation <br />                                                                                occur dormant origins, replication profile <br />                                                                                resembles minimally licensed DNA treated aphidi- <br />                                                                                colin caffeine. <br />                                                                                       Fig. 4 (B C) experimental results support <br /> Figure 4. Roscovitine blocks premature nascent strand fusion.   Model        model. Fig. 4 B (left) shows effect DNA replication effect roscovitine dormant origin &#239;&#172;?ring. model maximally licensed DNA shown Fig. 3  roscovitine       different combinations caffeine, aphidicolin, added shortly &#239;&#172;?rst origins &#239;&#172; (right), preventing    roscovitine added X. laevis egg extract start dormant origins &#239;&#172;?ring. (B) Sperm nuclei incubated X. laevis       S phase. Consistent previous work (Luciani et al., 2004), <br /> egg extract supplemented &#206;&#177;-[32P]dATP. 25 min (left), 30 min <br /> (right), extract optionally supplemented aphidicolin, caffeine,       addition roscovitine early S phase allowed existing roscovitine. indicated times, total DNA synthesis        forks elongate, replication rate subsequently trailed <br /> measured. (C) Sperm nuclei incubated X. laevis egg extract supple-     clustered nascent strands fused  <br /> mented aphidicolin caffeine. 25 min, extract supple- <br /> mented &#206;&#177;-[32P]dATP minus  iii) plus (ii iv) roscovitine. 30 min, extract supplemented unlabeled dATP. indicated <br /> times, DNA isolated, separated alkaline agarose gel electrophoresis, autoradiographed. End-labeled &#206;&#187;-HindIII run molecular               scans lane shown iii iv (earliest times  <br /> weight standards. autoradiographs shown ii. x-ray &#239;&#172;?lm     slowest migration right). migration &#206;&#187;-HindIII shown ticks scanned density label lane quanti&#239;&#172;?ed.       <br />  <br />  <br />                                                                   EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                      677 <br />        Figure 5. Minimally licensed DNA sensitive range <br />       replication inhibitors. Sperm nuclei incubated       X. laevis egg extract supplemented &#206;&#177;-[32P]dATP plus <br />       minus caffeine. produce minimally licensed DNA, <br />       extract supplemented geminin shortly sperm <br />       addition. 5 min, extract optionally supplemented <br />       500 &#206;&#188;M mitomycin C, 200 &#206;&#188;M etoposide, 4 ng/&#206;&#188;l <br />       actinomycin D. indicated times, total       DNA synthesized measured. <br />  <br />  <br />  <br />  <br />       expected maximally licensed DNA, aphidicolin inhib-             consistent fusion replicons 5&#226;&#8364;&#8220;10 kb size <br />       ited replication inhibition completely res-          (Fig. 4 C, ii, arrows). profile resembles minimally <br />       cued addition caffeine.  roscovitine            licensed DNA treated aphidicolin plus caffeine (Fig. 3, <br />       added aphidicolin caffeine,         B [v] C [v]). predicted, inhibiting Cdk-dependent initi- <br />       rescue replication. related experiment shown Fig.           ation dormant origins maximally licensed DNA, pattern <br />       4 B (right), difference inhibitors added       nascent strand labeling resemble minimally <br />       later S phase. time, addition roscovitine      licensed DNA, dormant origins exist. <br />       significant effect replication kinetics, suggesting       virtually normal replication origins fired         Dormant origins activated response <br />       time inhibitors added.  inhibition           range replication stresses <br />       replication aphidicolin rescued coaddition           investigated activation dormant <br />       caffeine, rescue substantially dependent fur-       origins maximally licensed DNA useful recovering <br />       ther Cdk activity. roscovitine-resistant rescue caf-           replication-induced stresses aphidicolin <br />       feine observed, possibly reflecting additional recovery            (Fig. 5). mitomycin C  cross-linking agent) etopo- <br />       pathways, restart previously stalled forks.          topoisomerase II inhibitor) induced block replica- <br />       significance version experiment require-      tion efficiently rescued caffeine       ment Cdks seen late stage S phase ori-        excess Mcm2&#226;&#8364;&#8220;7 present DNA. similar       gins initiated, suggesting complete replication      results aphidicolin (Fig. 1  suggests general <br />       dependent use origins normally           requirement dormant origins rescue DNA synthesis <br />       fired (dormant origins).                                          replication block. Actinomycin D, acts              confirm interpretation results,          DNA intercalator inhibitor primase compo- <br />       formed alkaline gel analysis nascent strands. Nascent           nent DNA polymerase &#206;&#177; showed dramatic <br />       DNA labeled &#206;&#177;-[32P]dATP beginning S phase,          effect. Replication slightly inhibited maxi- <br />       presence aphidicolin caffeine. 32P          mally licensed DNA treated 4 ng/&#206;&#188;l actinomycin D, <br />       chased unlabeled dATP,  time, extract           inhibition severe minimally <br />       supplemented minus (Fig. 4 C, iii) plus roscovitine       licensed DNA. small inhibition       (Fig. 4 C, ii iv). different times  labeled DNA      maximally licensed DNA fully reversed caffeine, <br />       analyzed alkaline gels. sample treated aphidi-         virtually rescue caffeine observed mini- <br />       colin caffeine (Fig. 4 C,  closely resembled corre-          mally licensed DNA. result explained inabil- <br />       sponding sample shown Fig. 3 B (ii), nascent DNA               ity actinomycin D activate strong ATR-dependent <br />       increasing rapidly size migrated exclusion      checkpoint response, despite strong inhibition replication <br />       limit gel end time series. Addition rosco-     (Hekmat-Nejad et al., 2000). Dormant origins cases <br />       vitine (Fig. 4 C, ii iv) completely inhibited        promote significant recovery replication <br />       crease modal strand length, strands increased       inhibition need checkpoint alleviation       size constant rate  6 nt/s ( 3 nt/s fork). later time   caffeine. Consistent interpretation, treatment       points, small number strands migrated higher molecu-         X. laevis egg extracts actinomycin D greatly enhances <br />       lar weight smear approximately twice size majority,      Cdc45 loading (Edwards et al., 2002). <br />  <br />  <br /> 678   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  Figure 6. Knockdown Mcm7 C. elegans causes hypersensitivity HU. C. elegans grown plates plus minus bacteria expressing anti-Mcm7 <br /> siRNA (2%) &#194;&#177;9.5 mM HU. 7 d, plates examined worm growth.   number F1 (&#239;&#172;?rst) F2 (second) generation worms standardized areas counted, &#194;&#177;SD. (B) Representative picture plate, case showing adult F1 worm. Closed arrowhead, F1 adult; open <br /> arrowheads, F2 larvae. Note absence F2 worms sample containing HU anti-Mcm7 siRNA. Bar, 0.2 mm. <br />  <br />  <br /> Excess Mcm2&#226;&#8364;&#8220;7 required C. elegans <br />                                                                             Discussion survive HU large amounts replicative stress need use          results presented work provide solution MCM <br /> X. laevis cell-free activate global checkpoint pathways              paradox showing excess Mcm2&#226;&#8364;&#8220;7 license dormant prevent replication completing. test physiologi-           replication origins normally used sup- <br /> cal significance findings, turned vivo            pression caffeine-sensitive pathway. X. laevis, effect lower levels replicative stress              10 dormant origins active origin, dormant reliably assessed. took advantage ability lower             origin activation permitted treatment caffeine, av- <br /> specific gene expression C. elegans providing siRNA-                  erage origin spacing drops 5- 10-fold. caffeine sensitivity <br /> expressing bacteria food source. mixed different              suggests involvement ATR-dependent checkpoint ratios bacteria expressing MCM7 siRNA bacteria                       suppressing activation dormant origins. conclusion <br /> expressing vector determine maximum quantity                consistent results mammalian cells, MCM7 siRNA observable effect C. elegans                  suppression ATM&#226;&#8364;&#8220;ATR Chk1 checkpoint pathways leads <br /> life cycle (Table S1, available http://www.jcb.org/cgi/content/          dramatic reduction average spacing adjacent jcb.200602108/DC1). creates state function-             replication origins (unpublished data). ally equivalent minimally licensed state X. laevis.               dormant origins used allow complete replication  determined low concentration HU (9.5 mM),                der conditions replicative stress, propose observable effect C. elegans life cycle (Table S1).           reason existence. <br /> HU lowers dNTP pools inhibiting ribonucleotide reductase                        Detailed analysis effect performed inhibits DNA synthesis. Fig. 6 shows treat-              X. laevis egg extracts, tested key consequence <br /> ment combination MCM7 siRNA 9.5 mM                     model lowering MCM levels C. elegans opera- <br /> HU completely abrogated C. elegans proliferation.               tionally mimic minimally licensed state. Reduction MCM <br /> dramatic reduction number adult worms                levels causes dramatic hypersensitivity HU, showing <br /> generation fed siRNA. adult worms                obvious defect absence replicative stress. generation appeared sterile  depicted), pro-             published evidence effect operate mammals. <br /> duced virtually eggs larvae (Fig. 6 B). Similar results          DNA fiber autoradiography studies shown increase obtained using knockdown MCM5 MCM6 (Tables S2                        density replication origins replication inhib- S3). experiments reduction Mcm2&#226;&#8364;&#8220;7                  ited FdUrd, thymidylate synthase inhibitor (Ockey <br /> proteins causes dramatic hypersensitivity HU,               Saffhill, 1976; Taylor, 1977), treatment ultraviolet <br /> ing obvious defect absence replicative stress.                 light (Griffiths Ling, 1985, 1987, 1989; Painter, 1985). Simi- possible explanations                   larly, reduced <span id='am-21' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> availability shown increase <br /> happening C. elegans experiments, results fully            frequency initiation origins normally used <br /> consistent results obtained X. laevis idea         (Anglana et al., 2003).  believe dormant ori- <br /> excess Mcm2&#226;&#8364;&#8220;7 license dormant origins used dur-                gins present metazoans, potentially <br /> ing undisturbed S phases.                                                   provide role responding replicative stresses. <br />  <br />  <br />                                                              EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                   679 <br />        Figure 7. Model use dormant origins chromosomal DNA.   small segment chromosomal DNA shown single molecule       ORC bound  Distributed ORC Mcm2&#226;&#8364;&#8220;7 double hexamers (hexagons), represent candidate origin. collec- <br />       tion Mcm2&#226;&#8364;&#8220;7 ORC designated single license group. (Bi) initiation occurs given license group generates weak <br />       caffeine-sensitive checkpoint signal inhibits initiation Mcm2&#226;&#8364;&#8220;7 license group. (Ci) fork replicates DNA, uniniti- <br />       ated Mcm2&#226;&#8364;&#8220;7 dormant origins displaced DNA. (Bii) rightward fork stalls, potentially generating strong global checkpoint signal. <br />       (Cii) dormant origins license group escapes inhibition initiates,  allowing DNA right stalled fork       replicated. escape inhibition stochastic, occur result attenuation checkpoint signal DNA repair. (D) Completion <br />       DNA replication. <br />  <br />  <br />       model behavior                                                        prolonged, forks rapidly replicate surrounding DNA, <br />       dormant origins                                                              displacing bound Mcm2&#226;&#8364;&#8220;7 molecules elimi- <br />       results suggest classically defined replication origins            nating dormant origins (Fig. 7, Ci D). Abolishing       actually comprised candidate origins,                  inhibitory signal caffeine absence replication <br />       normally used, outlined Fig. 7. known                  stress causes decrease origin spacing, does signifi- <br />       work X. laevis replication forks initiate               cantly increase overall replication rates. contrast Shechter <br />       sites Mcm2&#226;&#8364;&#8220;7, ORC, bound (Hua Newport,                    et al. (2004), restrict analysis extracts <br />       1998; Rowles et al., 1999; Edwards et al., 2002; Harvey                     replicate DNA efficiently  case extracts <br />       Newport, 2003; Danis et al., 2004). Based previous studies,                  recapitulate behavior intact eggs), inhibition ATM       Mcm2&#226;&#8364;&#8220;7 spread kilobases ORC                      ATR kinases does greatly accelerate replication-timing <br />       loaded  form term &#226;&#8364;&#339;license group&#226;&#8364;? (Ritzi et al.,              program (Luciani et al., 2004; study). Dormant origins       1998; Edwards et al., 2002; Harvey Newport, 2003; Danis                     just late-firing origins, confirmed experi- <br />       et al., 2004). site containing bound Mcm2&#226;&#8364;&#8220;7 candidate                 ments roscovitine (Fig. 4), replication-timing pro- <br />       replication origin potentially initiate replication (Fig.              gram appears operate independently checkpoint signals <br />       7 . wide dispersal Mcm2&#226;&#8364;&#8220;7 increases usefulness                  (unpublished data). results rule role Mcm2&#226;&#8364;&#8220;7 <br />       candidate origins dealing replicative stress.                     checkpoint activation;  recently reported phos- <br />               Fig. 7 (Bi&#226;&#8364;&#8220;Ci) shows events occurring absence                    phorylation Mcm2&#226;&#8364;&#8220;7 proteins ATR (Cortez et al., 2004; <br />       replicative stress. Initiation occurs candidate                Tsao et al., 2004) contribute inhibition dormant <br />       origins  primary origin). Consistent previous studies                  origins propose. <br />       (Edwards et al., 2002; Marheineke Hyrien, 2004; Shechter                           Fig. 7 (Bii&#226;&#8364;&#8220;Cii) shows events occurring origin       et al., 2004), results caffeine suggest role ATM&#226;&#8364;&#8220;ATR             forks experiences replicative stress stalled       preventing origin license group initi-               slowed. Fig. 7 Bii, rightward fork stalls dor- <br />       ating replication. speculate initiation taken                  mant origins license group eliminated.       place primary origin, ATM&#226;&#8364;&#8220;ATR activated,                     forks stall  occurred experiments using replication <br />       haps locally, inhibit initiation remaining candidate ori-             inhibitors X. laevis), global checkpoint signal activated <br />       gins license group (Fig. 7 Bi). prevent initiation               represses initiation replication origins       dormant origins, inhibition need absolute               genome.  use dormant origins  <br />  <br /> 680   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  physiologically important levels fork stalling        tightly localized origins multiple Mcm2&#226;&#8364;&#8220;7 complexes <br /> strongly activate global checkpoint response. Fig. 7 Cii           associated  mechanism reproducibly <br /> shows fork stalled, nearby dormant          selects initiate form primary origin. <br /> origins undergo initiation, ultimately allowing              remains unclear certain candidate origins DNA region replicated (Fig. 7 D).        selected preferential initiation, difference <br /> potential explanations dormant origin escape          underlying chromatin attractive possibility inhibitory signal emitted stalled fork. possibility,   (Machida et al., 2005). case X. laevis, initiation mentioned previous paragraph, inhibitory             occur variable distance away ORC signal weak (e.g., forks stalled         bound (Harvey Newport, 2003) directed  global suppression initiation), sto-    ward specific DNA sequences DNA-bound transcription <br /> chastic fashion extended lifespan dormant origin pro-        factors (Danis et al., 2004). occurs vides sufficient length time undergo initiation.         changing localization ORC, presumably happens possibility DNA repair processes operating           increasing initiation probability certain candidate stalled fork attenuate checkpoint signal. possi-        origins. Interestingly, transcription factor binding induced  <br /> bility certain types replication inhibition     tone acetylation favored origins (Danis et al., 2004), <br /> induced actinomycin D) robustly activate check-          supporting case chromatin modification involved <br /> point  leaving dormant origins free        candidate origin selection. Somatic cells undergo transi- <br /> controls allow dormant origins used               tion called origin decision point (ODP) middle nearby forks encounter problems,  lowering probability         G1, initiation directed occur specific preferred converging forks stall irreversibly leave         sequences randomly distributed sites (Wu <br /> DNA unable replicate.                                   Gilbert, 1996). ODP occurs Mcm2&#226;&#8364;&#8220;7 <br />        model emphasizes counterintuitive nature          bound DNA origins functionally <br /> intra&#226;&#8364;&#8220;S phase checkpoint globally blocks origin fir-       licensed (Dimitrova et al., 2002), possible ODP <br /> ing response replication stress           represents stage certain candidate origins selected <br /> ditions better cells respond using         preferentially support initiation. <br /> origins, fewer. possible explanation global intra&#226;&#8364;&#8220;S phase checkpoint provides strategy respond- <br />                                                                         Materials methods <br /> ing severe replicative stresses affect replication <br /> forks transient.                                           Preparation use X. laevis egg extracts <br />                                                                         X. laevis egg extracts prepared previously described (Chong <br />                                                                         et al., 1997) supplemented 250 &#206;&#188;g/ml cycloheximide, 25 mM replication origin?                                           phosphocreatine, 15 &#206;&#188;g/ml <span id='am-6' about='obo:IMR_0200144' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:id" content="IMR:0200144" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Creatine" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00300" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>creatine</span> phosphokinase, 300 &#206;&#188;M CaCl2 model suggests normal conditions firing           use. Demembranated X. laevis sperm nuclei (Chong et al., 1997) <br />                                                                         added extract &#239;&#172;?nal concentration 10&#226;&#8364;&#8220;15 ng DNA/&#206;&#188;l primary origin represses origins license group. <br />                                                                         incubated 23&#194;&#176;C. replication assays, extracts supplemented  maximally minimally licensed DNA                &#206;&#177;-[32P]dATP, DNA synthesis measured trichloroacetic <br /> overall replication kinetics average origin spacing.       acid precipitation, previously described (Chong et al., 1997). obtain <br />                                                                         minimally licensed DNA samples, recombinant geminin lacking Origin spacing largely independent number                                                                         destruction box (McGarry Kirschner, 1998; Tada et al., 2001) Mcm2&#226;&#8364;&#8220;7 complexes bound DNA determined                  added &#239;&#172;?nal concentration 10 ng/&#206;&#188;l. appropriate geminin <br /> location ORC binding. X. laevis, regular ORC spacing (Blow        addition time determined extract produce minimal licens- <br />                                                                         ing, previously described (Oehlmann et al., 2004). Chromatin et al., 2001; Oehlmann et al., 2004), coupled repression <br />                                                                         isolated subjected Western blot analysis, previously described excess Mcm2&#226;&#8364;&#8220;7 license groups, reliably support            (Oehlmann et al., 2004). <br /> complete genome replication, dormant origins used backup problems encountered forks originating            Reagents antibodies <br />                                                                         Aphidicolin (Sigma-Aldrich) dissolved DMSO 18 mM used primary origins. reminiscent &#226;&#8364;&#339;Jesuit model&#226;&#8364;?         &#239;&#172;?nal concentration 10 &#206;&#188;M extract. Caffeine (Sigma-Aldrich) selection replication initiation sites, suggests      dissolved H20 100 mM used &#239;&#172;?nal concentration DNA contains potential initiation sites, sites        5 mM. Roscovitine (Calbiochem) dissolved DMSO 400 mM                                                                         used &#239;&#172;?nal concentration 500 &#206;&#188;M. Mitomycin C dissolved repressed, activated (DePamphilis, 1999).              H20 5 mM used &#239;&#172;?nal concentration 500 &#206;&#188;M. Etoposide         Replication origins metazoans different         dissolved DMSO 25 mM used &#239;&#172;?nal concentration forms (DePamphilis, 1999; Gilbert, 2001; Machida et al., 2005).         200 &#206;&#188;M. Actinomycin D dissolved DMSO 1 &#206;&#188;g/&#206;&#188;l used <br />                                                                         &#239;&#172;?nal concentration 4 ng/&#206;&#188;l. Anti-Cdc45 antibody (Mimura replication origins, origin near lamin          Takisawa, 1998) gift H. Takisawa (Osaka University, Osaka, <br /> B2 gene, replication initiates reproducibly tightly defined        Japan). Geminin gift  Ferenbach (University Dundee, <br /> locus; origins, origin near dihydrofolate      Dundee, Scotland). <br /> reductase gene, replication initiate potential sites <br />                                                                         DNA &#239;&#172;?ber labeling <br /> distributed initiation zone. case origins       X. laevis sperm DNA incubated extract supplemented 50 &#206;&#188;M large initiation zones, dihydrofolate reductase        biotin-16-dUTP (Roche) desired conditions. Reactions (20 &#206;&#188;l) <br /> origin, zone consist multiple sites containing bound        stopped resuspension 350 &#206;&#188;l ice-cold NIB (50 mM <br />                                                                         Hepes-KOH, pH 7.6, 50 mM KCl, 5 mM MgCl2, 2 mM DTT, 0.5 mM <br /> Mcm2&#226;&#8364;&#8220;7 (Alexandrow et al., 2002),            <span id='am-29' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span>spermidine</span>-3HCl, 0.15 mM <span id='am-1' about='obo:IMR_0200092' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00750" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200092" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Spermine" datatype="xsd:string"></span>spermine</span>-4HCl, 100 &#206;&#188;M PMSF, 1 &#206;&#188;g/ml <br /> didate origins initiating S phase. possible       aprotinin, leupeptin, pepstatin) containing 30 &#206;&#188;M aphidicolin. <br />  <br />  <br />                                                            EXCESS MCM2&#226;&#8364;&#8220;7 LICENSE DORMANT ORIGINS &#226;&#8364;&#162; WOODWARD ET AL.                                    681 <br />        resuspended extract underlayered 100 &#206;&#188;l NIB containing 30 &#206;&#188;M            work supported Cancer Research UK (CR-UK) grants <br />       aphidicolin 10% (wt/vol) sucrose spun 2,000 g swinging        C303/A3135 STU063/001 J.J. Blow CR-UK grant CDA A5991 <br />       bucket rotor 5 min 4&#194;&#176;C. supernatant removed leave       Gartner. <br />       sucrose cushion, cushion washed twice                authors &#239;&#172;?nancial con&#239;&#172;&#8218;icts work. <br />       100 &#206;&#188;l NIB plus 30 &#206;&#188;M aphidicolin. cushion       carefully removed leave 40 &#206;&#188;l, resuspended 400 &#206;&#188;l             Submitted: 17 February 2006 <br />       ice-cold PBS plus 0.1% (vol/vol) Triton X-100. underlayered                                                                                     Accepted: 3 2006 <br />       PBS plus 10% sucrose spun like resuspended extract. super- <br />       natant removed cushion washed twice       100 &#206;&#188;l PBS. cushion carefully removed leave pelleted <br />       nuclei, gently resuspended 50 &#206;&#188;l PBS freezing         References <br />       dry ice. <br />              DNA spread glass slides according previously          Alexandrow, M.G., M. Ritzi,  Pemov, J.L. Hamlin. 2002. potential <br />       scribed standard conditions (Blow et al., 2001). Slides rehydrated               role mini-chromosome maintenance (MCM) proteins initia- <br />       HPLC pure water incubated 2 h blocking solution contain-              tion dihydrofolate reductase replication origin. J. Biol. Chem. <br />       ing PBS, 1% (wt/vol) BSA, 0.05% (vol/vol) TWEEN 20. Slides                   277:2702&#226;&#8364;&#8220;2708. <br />       incubated anti-biotin (B7653; Sigma-Aldrich) 20 &#206;&#188;g/ml         Anglana, M., F. Apiou,  Bensimon, M. Debatisse. 2003. Dynamics       buffer 2 h, washed extensively (10 changes buffer 2 h),               DNA replication mammalian somatic cells: nucleotide pool modulates <br />                                                                                             origin choice interorigin spacing. Cell. 114:385&#226;&#8364;&#8220;394. <br />       labeled 1 &#206;&#188;g/ml cy3-conjugated anti&#226;&#8364;&#8220;mouse antibody (715-165-150; <br />       Jackson ImmunoResearch Laboratories) buffer 2 h. Samples      Aparicio, O.M., D.M. Weinstein, S.P. Bell. 1997. Components dynamics <br />       washed extensively (10 changes buffer 2 h),                     DNA replication complexes S. cerevisiae: redistribution MCM <br />                                                                                             proteins Cdc45p S phase. Cell. 91:59&#226;&#8364;&#8220;69. <br />       washed twice PBS DNA stained YOYO (Y-3601 diluted       1:10,000 1-mM stock; Invitrogen) 10 min. Samples              Aparicio, O.M., M. Stout, S.P. Bell. 1999. Differential assembly Cdc45p <br />       rinsed &#239;&#172;?ve times PBS mounted Vectashield. Images cap-                  DNA polymerases early late origins DNA replication. Proc. <br />                                                                                             Natl. Acad. Sci. USA. 96:9130&#226;&#8364;&#8220;9135. <br />       tured using microscope (LSM510-META; Carl Zeiss MicroImaging, ) <br />       measurements using associated LSM software.                 Blow, J.J.,  Dutta. 2005. Preventing replication chromosomal DNA. <br />       Random &#239;&#172;?elds selected using YOYO staining ensure                    Nat. Rev. Mol. Cell Biol. 6:476&#226;&#8364;&#8220;486. <br />       single DNA &#239;&#172;?bers, &#239;&#172;?ber bundles, scored. Fields rich           Blow, J.J., P.J. Gillespie, D. Francis, D  Jackson. 2001. Replication origins <br />       unambiguous single &#239;&#172;?bers recorded using 100&#195;&#8212;, NA 1.4, lens                 Xenopus egg extract 5&#226;&#8364;&#8220;15 kilobases apart activated clus- <br />       optimal resolution sensitivity. sample minimum 200                  ters different times. J. Cell Biol. 152:15&#226;&#8364;&#8220;25. <br />       measurements                                                         Burkhart, R., D. Schulte, D. Hu, C. Musahl, F. Gohring, R. Knippers. 1995. <br />                                                                                             Interactions human nuclear proteins P1Mcm3 P1Cdc46. Eur. J. <br />       Pulse-chase experiments                                                               Biochem. 228:431&#226;&#8364;&#8220;438. <br />       Pulse-chase experiments performed incubating X. laevis sperm          Callan, H.G. 1972. Replication DNA chromosomes eukaryotes. Proc. <br />       DNA interphase egg extract, inhibitors added appropriate.              R. Soc. Lond. B. Biol. Sci. 181:19&#226;&#8364;&#8220;41. <br />       beginning S phase  exact timing determined indepen-       Chong, J.P., P. Th&#195;&#182;mmes,  Rowles, H.M. Mahbubani, J.J. Blow. 1997. <br />       dently extract), extract supplemented 0.2 mCi/ml                Characterization Xenopus replication licensing  Methods <br />       &#206;&#177;-[32P]dATP  stock 10 mCi/ml 3,000 Ci/mmol; GE Health-                  Enzymol. 283:549&#226;&#8364;&#8220;564. <br />       care). 2-min pulse (5 min samples containing aphidicolin),    Cortez, D., G. Glick, S.J. Elledge. 2004. Minichromosome maintenance pro- <br />       radioactive nucleotide chased addition 2.5 mM unlabeled                     teins direct targets ATM ATR checkpoint kinases. Proc. <br />       Mg-dATP. Aliquots (10 &#206;&#188;l) subsequently removed sample            Natl. Acad. Sci. USA. 101:10078&#226;&#8364;&#8220;10083. <br />       desired time points, reactions stopped, DNA processed        Danis, E., K. Brodolin, S. Menut, D. Maiorano, C. Girard-Reydet, M. <br />       electrophoresis 0.7% agarose gels, previously described (Luciani            Mechali. 2004. Specification DNA replication origin transcrip- <br />       et al., 2004). HindIII-digested &#206;&#187;-phage DNA (New England Biolabs)                 tion complex. Nat. Cell Biol. 6:721&#226;&#8364;&#8220;730. <br />       end-labeled &#206;&#177;-[32P]dATP loaded comparison (12.5 ng           DePamphilis, M.L. 1999. Replication origins metazoan chromosomes: fact       lane). Alkaline agarose gels exposed autoradiography &#239;&#172;?lm                      fiction? Bioessays. 21:5&#226;&#8364;&#8220;16. <br />       (Kodak), scanned using scanner (CanoScan 9900F; Canon)            Diffley, J.F. 2004. Regulation early events chromosome replication. Curr. <br />       Photoshop software (Adobe). Lane densities analyzed using                    Biol. 14:R778&#226;&#8364;&#8220;R786. <br />       GelEval software (FrogDance Software).                                        Dimitrova, D.S., T. Todorov, T. Melendy, D.M. Gilbert. 1999. Mcm2,                                                                                             RPA, component mammalian early G1-phase prereplication <br />       C. elegans work                                                                       complex. J. Cell Biol. 146:709&#226;&#8364;&#8220;722. <br />       C. elegans RNAi feeding performed previously described, using          Dimitrova, D.S., T  Prokhorova, J.J. Blow, T. Todorov, D.M. Gilbert. <br />       MCM5 (R10E4.4), MCM6 (ZK632.1), MCM7 (F32D1.10)                          2002. Mammalian nuclei licensed DNA replication       genome-wide RNAi library (Fraser et al., 2000; Kamath et al., 2003).                  late telophase. J. Cell Sci. 115:51&#226;&#8364;&#8220;59. <br />       MCM RNAi-expressing bacteria titrated bacteria expressing           Donovan, S., J. Harwood, L.S. Drury, J.F. Diffley. 1997. Cdc6p-dependent <br />       RNAi GFP. P0 worms placed RNAi plates containing                loading Mcm proteins pre-replicative chromatin budding yeast. <br />       100 &#206;&#188;g/ml ampicillin, 5 mM IPTG, indicated concentration HU                Proc. Natl. Acad. Sci. USA. 94:5611&#226;&#8364;&#8220;5616. <br />       L4 larval stage, resulting adult worms taken    Edwards, M.C., V. Tutter, C. Cvetic, C.H. Gilbert, T  Prokhorova, J.C. <br />       plates 40 h later. 4 d, number adult F1 worms scored,            Walter. 2002. MCM2-7 complexes bind chromatin distributed pat- <br />       sterile worms determined absence eggs body.               tern surrounding origin recognition complex Xenopus egg extracts. <br />       number viability F2 animals scored 2&#226;&#8364;&#8220;3 d later. stage,             J. Biol. Chem. 277:33049&#226;&#8364;&#8220;33057. <br />       wild-type conditions, plates starved exces-     Fraser, G., R.S. Kamath, P. Zipperlen, M. Martinez-Campos, M. Sohrmann, <br />       sive F2 larvae. Plates viewed microscope (Stemi SV11; Carl                J. Ahringer. 2000. Functional genomic analysis C. elegans chro- <br />       Zeiss MicroImaging, ) using 20&#195;&#8212; objective. Representative images                mosome systematic RNA interference. Nature. 408:325&#226;&#8364;&#8220;330. <br />       captured using Openlab 3.1.2 software (Improvision) exported         Gilbert, D.M. 2001. Making sense eukaryotic DNA replication origins. <br />       tiff &#239;&#172;?les.                                                                         Science. 294:96&#226;&#8364;&#8220;100. <br />                                                                                     Gillespie, P.J.,  Li, J.J. Blow. 2001. Reconstitution licensed replication <br />       Online supplemental material                                                          origins Xenopus sperm nuclei using purified proteins. BMC Biochem. <br />       Tables S1&#226;&#8364;&#8220;S3 number C. elegans adult F1 worms observed                 2:15. <br />       different HU concentrations different amounts siRNA directed   Griffiths, T.D., S.Y. Ling. 1985. Effect ultraviolet light <span id='am-35' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-36' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span>thymidine</span> <br />       MCM7 (Table S1), MCM5 (Table S2), MCM6 (Table S3). Online                         incorporation, DNA chain elongation replicon initiation wild-type <br />       supplemental material available http://www.jcb.org/cgi/content/                 excision-deficient Chinese hamster ovary cells. Biochim. Biophys. <br />       jcb.200602108/DC1.                                                               Acta. 826:121&#226;&#8364;&#8220;128. <br />                                                                                     Griffiths, T.D., S.Y. Ling. 1987. Activation alternative sites replicon <br />       Thanks Andrew Ferenbach recombinant geminin, Neil Perkins,                 initiation Chinese hamster cells exposed ultraviolet light. Mutat. <br />       Anatoliy Li, Margret Michalski-Blow comments manuscript.               Res. 184:39&#226;&#8364;&#8220;46. <br />  <br />  <br /> 682   JCB &#226;&#8364;&#162; VOLUME 173 &#226;&#8364;&#162; NUMBER 5 &#226;&#8364;&#162; 2006 <br />  Griffiths, T.D., S.Y. Ling. 1989. Effects UV light DNA chain growth       Ritzi, M., M. Baack, C. Musahl, P. Romanowski, R  Laskey, R. Knippers. <br />         replicon initiation human cells. Mutat. Res. 218:87&#226;&#8364;&#8220;94.                     1998. Human minichromosome maintenance proteins human <br /> Harvey, K.J., J. Newport. 2003. CpG methylation DNA restricts pre-                  origin recognition complex 2 protein chromatin. J. Biol. Chem. <br />         replication complex assembly Xenopus egg extracts. Mol. Cell. Biol.             273:24543&#226;&#8364;&#8220;24549. <br />         23:6769&#226;&#8364;&#8220;6779.                                                               Rowles, , J.P. Chong, L. Brown, M. Howell, G  Evan, J.J. Blow. 1996. <br /> Hekmat-Nejad, M.,   M.C. Yee, J.W. Newport, K  Cimprich. 2000.                 Interaction origin recognition complex replication <br />         Xenopus ATR replication-dependent chromatin-binding protein                licensing Xenopus. Cell. 87:287&#226;&#8364;&#8220;296. <br />         quired DNA replication checkpoint. Curr. Biol. 10:1565&#226;&#8364;&#8220;1573.        Rowles, , S. Tada, J.J. Blow. 1999. Changes association Xenopus <br /> Herrick, J., P. Stanislawski, O. Hyrien,  Bensimon. 2000. Replication fork           origin recognition complex chromatin licensing replication <br />         density increases DNA synthesis X. laevis egg extracts. J. Mol.          origins. J. Cell Sci. 112:2011&#226;&#8364;&#8220;2018. <br />         Biol. 300:1133&#226;&#8364;&#8220;1142.                                                        Shechter, D., V. Costanzo, J. Gautier. 2004. ATR ATM regulate Hopwood, B., S. Dalton. 1996. Cdc45p assembles complex                     timing DNA replication origin firing. Nat. Cell Biol. 6:648&#226;&#8364;&#8220;655. <br />         Cdc46p/Mcm5p, required minichromosome maintenance,            Strausfeld, U.P., M. Howell, R. Rempel, J.L. Maller, T. Hunt, J.J. Blow. <br />         essential chromosomal DNA replication. Proc. Natl. Acad. Sci. USA.             1994. Cip1 blocks initiation DNA replication Xenopus extracts <br />         93:12309&#226;&#8364;&#8220;12314.                                                                    inhibition cyclin-dependent kinases. Curr. Biol. 4:876&#226;&#8364;&#8220;883. <br /> Hua, X.H., J. Newport. 1998. Identification preinitiation step DNA      Tada, S.,  Li, D. Maiorano, M. Mechali, J.J. Blow. 2001. Repression         replication independent origin recognition complex cdc6,            origin assembly metaphase depends inhibition RLF-B/Cdt1         dependent cdk2. J. Cell Biol. 140:271&#226;&#8364;&#8220;281 <br /> </body></html>