<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 30 October 2008                                  Nucleic Acids Research, 2009, Vol. 37, Database issue D455&#226;&#8364;&#8220;D458 <br />                                                                                                               doi:10.1093/nar/gkn858 <br />  <br />  <br /> DEG 5.0, database essential genes prokaryotes eukaryotes <br /> Ren Zhang1,* Yan Lin2 <br /> 1 <br />     Department Epidemiology Biostatistics, Tianjin Cancer Institute Hospital, Tianjin 300060 2 <br />     Department Physics, Tianjin University, Tianjin 300072, China <br />  <br /> Received September 15, 2008; Revised October 14, 2008; Accepted October 16, 2008 <br />  <br />  <br />  <br /> ABSTRACT                                                                        emerging &#239;&#172;?eld, synthetic biology (4). Essential-gene <br />                                                                                 studies practical reasons  Essential genes indispensable                                 instance, essential genes, lethality survival organism, functions                                disruptions, attractive targets antibiotics (5). considered foundation life. Determi-                             essential genes conserved species nation minimal gene set needed sustain                                candidates broad-spectrum drug targets, life form, fundamental question biology, plays                             speci&#239;&#172;?c bacterium candidates spe- key role emerging field, synthetic biology.                            cies-speci&#239;&#172;?c ones. years constructed DEG, database                                 2004, constructed DEG 1.0, database essential <br /> essential genes, DEG 5.0 significant advances                               genes (6). past &#239;&#172;?ve years, fueled accumulation 2004 version number essen-                              sequenced genomes, sophisticated genome-wide muta- <br /> tial genes number organisms                                 genesis techniques (7), burgeoning &#239;&#172;?eld synthetic genes determined. number prokar-                               biology (8&#226;&#8364;&#8220;10), signi&#239;&#172; advances                                                                                 determining essential genes wide range organisms. <br /> yotic essential genes DEG increased                                                                                 paper represents update, DEG 5.0, database 10-fold, mainly owing genome-wide gene essen- <br />                                                                                 essential genes prokaryotes eukaryotes. <br /> tiality screens performed wide range bacteria. number eukaryotic essential genes increased 5-fold, DEG 1.0                                SUMMARY DATABASE UPDATES yeast ones, DEG 5.0 humans, mice, worms, fruit flies, zebrafish                             parallel rapid progress essential-gene &#239;&#172;?eld, <br /> plant Arabidopsis thaliana. updates                              DEG 5.0 signi&#239;&#172; advances DEG 1.0                                                                                 following changes: <br /> represent significant advances DEG, represent rapid progress essential-                                  (1) number prokaryotic essential genes gene field. DEG freely available website                                  increased 10-fold, 543 5260 (Table 1). <br /> http://tubic.tju.edu.cn/deg http://www.essential                                   DEG 1.0, essential genes, e.g. gene.org.                                                                           Escherichia coli, collected literature <br />                                                                                     searches, DEG 5.0 records replaced <br />                                                                                     determined genome-wide studies using                                                                                     genetic footprinting technique (11) systematic gene <br /> INTRODUCTION <br />                                                                                     knockout experiments (12), (ii) DEG 1.0, Essential genes indispensable survival                         essential genes, e.g. Haemophilus in&#239;&#172;&#8218;uenzae organism certain conditions, functions                             determined theoretical prediction com- encode considered foundation life.                          parative genomics studies (13), DEG 5.0 Essential genes organism constitute minimal gene                          records replaced determined genome- <br /> set, smallest possible group genes                             wide studies using global transposon mutagenesis (14) su&#239;&#172;&#402;cient sustain functioning cellular life                            (iii) 2004, genome-wide studies form favorable conditions (1&#226;&#8364;&#8220;3). Determi-                            identifying bacterial essential genes  nation minimal gene set organism addresses                            12 &#239;&#172;?nished. conceptually important question: basic                           (2) number essential genes eukaryotes functions needed sustain life form,                              increased 5-fold, 878 4808, minimal-gene-set concept plays key role                                DEG 1.0 yeast essential genes, <br />  <br />  correspondence addressed. Tel: +86 22 2740 2987; Fax: +86 22 2335 8329; Email: rzhang.cn@gmail.com <br />  <br /> &#195;&#376; 2008 Author(s) Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/2.0/uk/) permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  D456 Nucleic Acids Research, 2009, Vol. 37, Database issue <br />  <br />  <br /> Table 1. Contents DEG version 5.0a <br />     Kingdom      Organism                      Essential   Method                                                      Saturated     References <br />                                                  gene                                                                 near saturated <br />  <br /> 1     Prokaryote   Acinetobacter baylyi           499        Single-gene deletions                                       Y                (23) <br /> 2     Prokaryote   Bacillus subtilis              271        Single-gene deletions                                       Y                (24) <br /> 3     Prokaryote   Escherichia coli               712        Genetic footprinting single-gene deletions              Y                (11,12) <br /> 4     Prokaryote   Francisella novicida           392        Transposon mutagenesis                                      Y                (25) <br /> 5     Prokaryote   Haemophilus in&#239;&#172;&#8218;uenzae          642        Transposon mutagenesis                                      Y                (14) <br /> 6     Prokaryote   Helicobacter pylori            323        Transposon mutagenesis                                      Y                (26) <br /> 7     Prokaryote   Mycobacterium tuberculosis     614        Transposon mutagenesis                                      Y                (27) <br /> 8     Prokaryote   Mycoplasma genitalium          381        Transposon mutagenesis                                      Y                (18) <br /> 9     Prokaryote   Mycoplasma pulmonis            310        Transposon mutagenesis                                      Y                (28) <br /> 10    Prokaryote   Pseudomonas aeruginosa         335        Transposon mutagenesis                                      Y                (29) <br /> 11    Prokaryote   Salmonella typhimurium         230        Insertional-duplication mutagenesis                         Y                (31) <br /> 12    Prokaryote   Staphylococcus aureus          302        Antisense RNA                                               N                (21,22) <br /> 13    Prokaryote   Streptococcus pneumoniae       244        Single-gene deletions allelic replacement mutagenesis   N                (19,20) <br /> 14    Prokaryote   Vibrio cholerae                  5        Transposon mutagenesis                                      N                (40) <br /> 15    Eukaryote    Arabidopsis thaliana           777        T-DNA insertion                                             N                (36) <br /> 16    Eukaryote    Caenorhabditis elegans         294        RNA interference                                            N                (34) <br /> 17    Eukaryote    Danio rerio                    288        Insertion mutagenesis                                       N                (35) <br /> 18    Eukaryote    Drosophila melanogaster        339        P-element insertion mutagenesis                             N                (33) <br /> 19    Eukaryote    Homo sapiens                   118        Literature search                                           N                (38) <br /> 20    Eukaryote    Mus musculus                  2114        Literature search                                           N                (37) <br /> 21    Eukaryote    Saccharomyces cerevisiae       878        Single-gene deletions                                       Y                (32)  cases, slight di&#239;&#172;&#8364;erence essential-gene number reported DEG. mainly annotation changes latest versions genomes records  reported results contain identical records partially <br /> published. <br />  <br />  <br />  <br />       DEG 5.0 humans, mice, worms,                     applied H. in&#239;&#172;&#8218;uenzae (14), essential genes       fruit &#239;&#172;&#8218;ies, zebra&#239;&#172;?sh plant Arabidopsis                         obtained replaced corresponding records DEG 1.0, <br />       thaliana.                                                              determined comparative genomics (13). <br />                                                                                 DEG 1.0, essential genes E. coli collected                                                                              http://magpie.genome.wisc.edu/ chris/essential.html,                                                                              essential genes obtained searching related <br /> DATABASE DESCRIPTION                                                         literatures. Using genetic footprinting technique, <br /> Essential genes prokaryotes                                               Gerdes et al. (11) conducted genome-wide, comprehen- <br />                                                                              sive experimental assessment E. coli genes necessary <br /> Determination minimal gene set cellular life                    robust aerobic growth, consequently, 620 genes possible availability &#239;&#172;?rst comple-                    identi&#239;&#172;?ed essential. addition, Keio collec- <br /> tely sequenced genomes bacteria Mycoplasma                          tion contains 303 essential genes determined genitalium (15) H. in&#239;&#172;&#8218;uenzae (16). attempt                         systematic single-gene knockout experiments (12).  <br /> determine minimal gene set pioneered Koonin                         fore, DEG 5.0, essential gene records obtained litera- coworkers comparing sequenced gen-                          ture search replaced obtained omes belong ancient bacterial lineages, based                    genome-wide mutagenesis studies (11) systematic notion genes conserved                       single-gene knockout experiments (12), likely essential cellular functions (13).                            copy retained 205 genes overlap    1999, Venter&#226;&#8364;&#8482;s group performed &#239;&#172;?rst global trans-                  studies. <br /> poson mutagenesis M. genitalium experimentally                            100 Streptococcus pneumoniae essential genes <br /> address question minimal gene set                     determined high-throughput gene disruption living organism (17), 300 genes esti-                       (19). Later, 133 essential genes determined <br /> mated essential included DEG 1.0.                          allelic replacement mutagenesis (20). DEG 5.0,  concept global transposon mutagenesis                        results combined removing redundant fact based identi&#239;&#172;?cation non-essential genes,                records, resulting 244 essential genes S. pneumoniae. e. disrupted transposons identi&#239;&#172;?ed,                       DEG 1.0 contained 65 Staphylococcus aureus essential disrupted considered essential.                      genes determined using antisense RNA technique gain proof gene dispensability M. genitalium,                   (21), DEG 5.0 contains 302 S. aureus essential <br /> Venter&#226;&#8364;&#8482;s group isolated characterized Tn4001                       genes combining results studies using insertion mutants present individual colonies                   rapid shotgun antisense RNA method (22). <br /> picked agar plates (18). Consequently, 382 genes                           past years, genome-wide mutagen- demonstrated essential, genes                      esis studies performed wide range bac- <br /> included DEG 5.0 replacing version 1.0.                       teria. addition mentioned  DEG 5.0 high-density transposon mutagenesis strategy                      contains essential genes determined large-scale <br />                                                                Nucleic Acids Research, 2009, Vol. 37, Database issue D457 <br />  <br />  <br /> single-gene deletion studies Acinetobacter baylyi (23)       open-source database management  MySQL, Bacillus subtilis (24), determined global          allows rapid data retrieval. ways <br /> transposon mutagenesis Francisella novicida (25),            users access data. Users Helicobacter pylori (26), Mycobacterium tuberculosis (27),      browse essential gene records, search <br /> Mycoplasma pulmonis (28) Pseudomonas aeruginosa             essential genes names, functions, accession <br /> (29,30), determined trapping lethal insertions     numbers organisms. addition, users  Salmonella typhimurium (31).                                 form BLAST searches DEG query DNA                                                                 protein sequences. database composed Essential genes eukaryotes                                   subdatabases, e. prokaryotes eukar- major improvement DEG 5.0 inclusion           yotes, users need perform functions Browse, essential genes eukaryotes, including animals        Search BLAST individual databases. addition, plant  thaliana, eukaryotic          database downloaded request. <br /> species DEG 1.0 Saccharomyces cerevisiae (32). goal determining bacterial minimal gene set                                                                 CONCLUSION FUTURE DEVELOPMENT <br /> applies eukaryotes, e. de&#239;&#172;?ne minimal gene set <br /> needed produce living multicellular organism          DEG 5.0 signi&#239;&#172; advances DEG 1.0 viable plant. goal obviously ambitious     number essential genes number organ- current stage, e&#239;&#172;&#8364;ort devoted       isms genes determined. updates identi&#239;&#172;?cation essential genes eukaryotes.          represent signi&#239;&#172; advances 2004 <br />    Drosophila genome, 25% genes            version DEG, represent rapid progress disrupted P-element insertions Berkeley               essential-gene &#239;&#172;?eld. future, prokaryotes, fueled <br /> Drosophila Genome Project (33), genes           availability complete genomes <br /> disruption lethal phenotypes collected DEG          emerging &#239;&#172;?eld, synthetic biology, expected <br /> 5.0. Caenorhabditis elegans genome, using RNA        increase essential gene number accel- <br /> interference, Kamath et al. (34) inhibited activity      erate, eukaryotic model organisms, 86% genes, characterized pheno-          gene essentiality screens far saturated, types, genes inhibition lethal              number essential genes expected grow. included DEG 5.0. Hopkins coworkers conducted            advances re&#239;&#172;&#8218;ected timely DEG future updates. large-scale insertional mutagenesis zebra&#239;&#172;?sh iden-      welcome users&#226;&#8364;&#8482; comments, corrections new infor- <br /> tify genes essential embryonic early larval devel-      mation, used updating. DEG freely <br /> opment (35), identi&#239;&#172;?ed essential genes             available website http://tubic.tju.edu.cn/deg collected DEG 5.0. &#239;&#172;?rst large-scale identi&#239;&#172;?cation        http://www.essentialgene.org. essential genes &#239;&#172;&#8218;owering plant performed Meinke coworker  thaliana characterizing large number T-DNA insertion lines (36), iden-       ACKNOWLEDGEMENTS <br /> ti&#239;&#172;?ed essential genes collected DEG 5.0.                like thank anonymous referees    Large-scale gene inactivation studies      constructive comments. <br /> formed mice, likely technical di&#239;&#172;&#402;culties labor intensiveness,  mice probably important model organism, large number            FUNDING <br /> genes inactivated individual labora- <br /> tories. study comparing essentiality duplicate     present work supported National <br /> genes singleton genes, Liao Zhang (37) analyzed         Natural Science Foundation China (NNSF 90408028). <br /> nearly 3900 individually inactivated mouse genes,           Funding open access charge: NNSF 90408028. 55% essential singletons          Con&#239;&#172;&#8218;ict statement. declared. duplicates. essential genes analyzed study collected DEG 5.0. study comparing <br /> human mouse essential genes, Liao Zhang (38)            REFERENCES <br /> extensively reviewed literatures &#239;&#172;?nd genes null <br /> mutations humans lethal, human essential        1. Koonin,E.V. (2000) genes make cell: minimal- <br />                                                                     gene-set concept. Annu. Rev. Genomics Hum. Genet., 1, 99&#226;&#8364;&#8220;116. <br /> genes collected DEG 5.0.                            2. Koonin,E.V. (2003) Comparative genomics, minimal gene-sets                                                                     universal common ancestor. Nat. Rev. Microbiol., 1, <br /> User interface data access                                      127&#226;&#8364;&#8220;136. <br />                                                                  3. Gerdes,S., Edwards,R., Kubal,M., Fonstein,M., Stevens,R. database divided subdatabases,                Osterman  (2006) Essential genes metabolic maps. Curr. Opin. prokaryotic eukaryotic essential genes.                Biotechnol., 17, 448&#226;&#8364;&#8220;456. entry unique DEG identi&#239;&#172;?cation number,                4. Lartigue,C., Glass,J , Alperovich,N., Pieper,R., Parmar,P.P., <br /> gene  gene reference number, gene function,                Hutchison,C  3rd, Smith,H.O. Venter,J.C. (2007) Genome <br />                                                                     transplantation bacteria: changing species  <br /> DNA protein sequences. prokaryotic essential                Science, 317, 632&#226;&#8364;&#8220;638. <br /> genes, link COG information (39) pro-           5. Galperin,M.Y. Koonin,E.V. (1999) Searching drug targets <br /> vided. information stored operated                 microbial genomes. Curr. Opin. Biotechnol., 10, 571&#226;&#8364;&#8220;578. <br />  D458 Nucleic Acids Research, 2009, Vol. 37, Database issue <br />  <br />  <br />  6. Zhang,R., Ou,H.Y. Zhang,C.T. (2004) DEG: database             23. Berardinis,V., Vallenet,D., Castelli,V., Besnard,M., Pinet , <br />     essential genes. Nucleic Acids Res., 32, D271&#226;&#8364;&#8220;D272.                        Cruaud,C., Samair,S., Lechaplais,C., Gyapay,G., Richez,C. et al. <br />  7. Judson,N. Mekalanos,J.J. (2000) Transposon-based                       (2008) complete collection single-gene deletion mutants     approaches identify essential bacterial genes. Trends Microbiol.,       Acinetobacter baylyi ADP1. Mol. Syst. Biol., 4, 174. <br />     8, 521&#226;&#8364;&#8220;526.                                                            24. Kobayashi,K., Ehrlich,S.D., Albertini , Amati,G., <br />  8. Benner,S  Sismour M. (2005) Synthetic biology. Nat. Rev.           Andersen,K.K., Arnaud,M., Asai,K., Ashikaga,S., Aymerich,S., <br />     Genet., 6, 533&#226;&#8364;&#8220;543.                                                        Bessieres,P. et al. (2003) Essential Bacillus subtilis genes. Proc. Natl <br />  9. Andrianantoandro,E., Basu,S., Karig,D.K. Weiss,R. (2006)               Acad. Sci. USA, 100, 4678&#226;&#8364;&#8220;4683. <br />     Synthetic biology: new engineering rules emerging discipline.   25. Gallagher,L , Ramage,E., Jacobs,M , Kaul,R., Brittnacher,M. <br />     Mol. Syst. Biol., 2, 2006 0028.                                            Manoil,C. (2007) comprehensive transposon mutant library <br /> 10. Galperin,M.Y. (2008) dawn synthetic genomics. Environ.              Francisella novicida, bioweapon surrogate. Proc. Natl Acad. <br />     Microbiol., 10, 821&#226;&#8364;&#8220;825.                                                   Sci. USA, 104, 1009&#226;&#8364;&#8220;1014. <br /> 11. Gerdes,S.Y., Scholle,M.D., Campbell,J.W., Balazsi,G., Ravasz,E.,       26. Salama,N.R., Shepherd,B. Falkow,S. (2004) Global transposon <br />     Daugherty,M.D., Somera L., Kyrpides,N.C., Anderson ,                   mutagenesis essential gene analysis Helicobacter pylori. <br />     Gelfand,M.S. et al. (2003) Experimental determination           J. Bacteriol., 186, 7926&#226;&#8364;&#8220;7935. <br />     level analysis essential genes Escherichia coli MG1655.          27. Sassetti,C.M., Boyd,D.H. Rubin,E.J. (2003) Genes required     J. Bacteriol., 185, 5673&#226;&#8364;&#8220;5684.                                             mycobacterial growth de&#239;&#172;?ned high density mutagenesis. <br /> 12. Baba,T., Ara,T., Hasegawa,M., Takai,Y., Okumura,Y., Baba,M.,               Mol. Microbiol., 48, 77&#226;&#8364;&#8220;84. <br />     Datsenko,K , Tomita,M., Wanner,B.L. Mori,H. (2006)               28. French,C.T., Lao,P., Loraine E., Matthews,B.T., Yu,H.     Construction Escherichia coli K-12 frame, single-gene knock-         Dybvig,K. (2008) Large-scale transposon mutagenesis     mutants: Keio collection. Mol. Syst. Biol., 2, 2006 0008.          Mycoplasma pulmonis. Mol. Microbiol., 69, 67&#226;&#8364;&#8220;76. <br /> 13. Mushegian R. Koonin,E.V. (1996) minimal gene set           29. Liberati,N.T., Urbach,J.M., Miyata,S., Lee,D.G., Drenkard,E., <br />     cellular life derived comparison complete bacterial genomes.         Wu,G., Villanueva,J., Wei,T. Ausubel,F.M. (2006) ordered, <br />     Proc. Natl Acad. Sci. USA, 93, 10268&#226;&#8364;&#8220;10273.                                nonredundant library Pseudomonas aeruginosa strain PA14 <br /> 14. Akerley,B.J., Rubin,E.J., Novick,V.L., Amaya,K., Judson,N.             transposon insertion mutants. Proc. Natl Acad. Sci. USA, 103, <br />     Mekalanos,J.J. (2002) genome-scale analysis identi&#239;&#172;?cation         2833&#226;&#8364;&#8220;2838. <br />     genes required growth survival Haemophilus in&#239;&#172;&#8218;uenzae.        30. Jacobs,M , Alwood , Thaipisuttikul , Spencer,D., Haugen,E., <br />     Proc. Natl Acad. Sci. USA, 99, 966&#226;&#8364;&#8220;971.                                    Ernst,S., O., Kaul,R., Raymond,C., Levy,R. et al. (2003) <br /> 15. Fraser,C.M., Gocayne,J.D., White,O., Adams,M.D., Clayton,R ,             Comprehensive transposon mutant library Pseudomonas aerugi- <br />     Fleischmann,R.D., Bult,C.J., Kerlavage R., Sutton,G.,                    nosa. Proc. Natl Acad. Sci. USA, 100, 14339&#226;&#8364;&#8220;14344. <br />                                                                            31. Knuth,K., Niesalla,H., Hueck,C.J. Fuchs,T.M. (2004) Large- <br />     Kelley,J.M. et al. (1995) minimal gene complement                                                                                scale identi&#239;&#172;?cation essential Salmonella genes trapping lethal <br />     Mycoplasma genitalium. Science, 270, 397&#226;&#8364;&#8220;403. <br />                                                                                insertions. Mol. Microbiol., 51, 1729&#226;&#8364;&#8220;1744. <br /> 16. Fleischmann,R.D., Adams,M.D., White,O., Clayton,R , <br />                                                                            32. Mewes,H.W., Frishman,D., Guldener,U., Mannhaupt,G., <br />     Kirkness,E.F., Kerlavage R., Bult,C.J., Tomb,J.F., <br />                                                                                Mayer,K., Mokrejs,M., Morgenstern,B., Munsterkotter,M., <br />     Dougherty,B , Merrick,J.M. et al. (1995) genome random <br />                                                                                Rudd,S. Weil,B. (2002) MIPS: database genomes     sequencing assembly Haemophilus in&#239;&#172;&#8218;uenzae Rd. Science, <br />                                                                                protein sequences. Nucleic Acids Res., 30, 31&#226;&#8364;&#8220;34. <br />     269, 496&#226;&#8364;&#8220;512.                                                          33. Spradling C., Stern,D., Beaton , Rhem,E.J., Laverty,T., <br /> 17. Hutchison,C , Peterson,S.N., Gill,S.R., Cline,R.T., White,O.,            Mozden,N., Misra,S. Rubin,G.M. (1999) Berkeley <br />     Fraser,C.M., Smith,H.O.  <br /> </body></html>