<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Copyright &#195;&#8220; 2008 Genetics Society America <br /> DOI: 10.1534/genetics.107.083782 <br />  <br />  <br />  <br />               Identification Drosophila Mutants Altering Defense                        Endurance Listeria monocytogenes Infection <br />  <br />                              Janelle S. Ayres,* Nancy Freitag&#226;&#8364;&#160; David S. Schneider*,1 <br />                       *Department Microbiology Immunology, Stanford University, Stanford, California 94305                          &#226;&#8364;&#160; <br />                           Department Microbiology Immunology, University Illinois, Chicago, Illinois 60612 <br />                                                     Manuscript received October 24, 2007 <br />                                                  Accepted publication December 19, 2007 <br />  <br />  <br />                                                                ABSTRACT <br />                 extended use Drosophila model signaling pathways required pattern <br />              recognition immune signaling fly used identify genes required patho- <br />              genesis host&#226;&#8364;&#8220;pathogen interactions. performed forward genetic screen identify Drosophila <br />              mutations altering sensitivity intracellular pathogen Listeria monocytogenes. recovered 18 mutants <br />              increased susceptibility infection, previously shown function Drosophila <br />              immune response. Using secondary screens, divided mutants groups: group, <br />              mutants reduced endurance infections change bacterial growth. new fly <br />              immunity phenotype commonly studied. second group, mutants typical defense <br />              defect bacterial growth increased survival decreased. challenging mutant flies <br />              L. monocytogenes mutants, identified subgroups fly mutants affect specific stages L. <br />              monocytogenes life cycle, exit vacuole, actin-based movement. overlap              genes hundreds genes identified Drosophila S2 cells fighting L. monocytogenes infection, <br />              using genomewide RNAi screens vitro. using animal model screening host survival, <br />              revealed genes involved physiologies different previous screens,              defects defensive immune signaling. <br />  <br />  <br />  <br />   NTRACELLULAR pathogens responsible     large group infectious diseases; example, .500 <br /> million people worldwide suffer tuberculosis, <br />                                                                           (Cossart et al. 2003). Entry nonphagocytic cells <br />                                                                           dependent surface proteins called internalins <br />                                                                           (Cossart Lecuit 1998; Lecuit et al. 1999, 2001). <br /> AIDS, malaria year (http://www int).                      entry, single-membrane vacuole forms residing host cell, pathogens protect                     bacterium (Cossart Lecuit 1998). Secretion <br /> selves host immune responses drug                           pore-forming cytotoxin, listeriolysin O (LLO), dis- <br /> therapies. ability enter survive host                 rupts phagosome membrane, freeing bacterium <br /> cell requires close intricate interaction                   cytosol grow divide (Kuhn et al. <br /> pathogen host; manipulating host processes,                        1988; Portnoy et al. 1988; Gedde et al. 2000; O&#226;&#8364;&#8482;Riordan <br /> pathogens prevent immune responses subvert                         Portnoy 2002). Expression actin-nucleating <br /> host processes aid infection.                                       protein ActA facilitates polymerization host actin    Listeria monocytogenes gram-positive, facultative              pole bacterium, resulting directional  <br /> tracellular bacterium cause listeriosis,                  ment cytosol. Bacteria able spread food-borne disease. L. monocytogenes                     neighboring cells escape second double- <br /> widely used model pathogen better understand                      membrane phagosome cytosol new host molecular cellular aspects intracellular path-                 cell (Tilney Portnoy 1989; Tilney et al. 1990; <br /> ogenesis mammalian cell-mediated immunity.                         Domann et al. 1992; Kocks et al. 1992, 1993). <br /> vitro studies using cultured cells model host defined                   development RNA interference (RNAi) technol- L. monocytogenes intracellular life cycle virulence               ogy provided researchers new approach studying <br /> factors (Cossart Lecuit 1998; Portnoy et al. 1988;                    host factors involved host&#226;&#8364;&#8220;pathogen interactions Kreft Vazquez-Boland 2001; Vazquez-Boland                             fly (Ramet et al. 2002; Foley O&#226;&#8364;&#8482;Farrell 2004; Ayres <br /> et al. 2001) demonstrated L. monocytogenes                   Schneider 2006). recent reports use <br /> enter professional phagocytes nonphagocytic cells                      RNAi Drosophila S2 cells (embryo-derived phagocytic <br />                                                                           cells Drosophila) identify host factors involved   1 <br />                                                                           L. monocytogenes infection; screens helped clarify    Corresponding author: Department Microbiology Immunology, <br /> Stanford University, D333 Fairchild Bldg., 299 Campus Dr., Stanford, CA   cell biology interactions L. monocytogenes 94305. E-mail: dschneider@stanford.edu                                    professional phagocytes (Agaisse et al. 2005; Cheng et al. <br />  <br /> Genetics 178: 1807&#226;&#8364;&#8220;1815 (March 2008) <br />  1808                                        J. S. Ayres, N. Freitag D. S. Schneider <br />  <br /> 2005). assays used vitro screens measured           Imd pathways (Meng et al. 1999; Leulier et al. 2000; <br /> bacterial loads qualitative changes bacterial patterns.       Rutschmann et al. 2000; Wu et al. 2001; Khush et al. limited scope screens identification         2002; Gesellchen et al. 2005; Yagi Ip 2005; Kambris fly genes regulating bacterial life cycle. Surprisingly,     et al. 2006). screen monitored fly survival screens did identify genes affecting             bacterial growth mutant flies. focusing phe- <br /> characterized pattern recognition pathways, Toll Imd.           notypes <span id='am-11' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span> transcription, anticipated  vitro systems obviously limited           identify host genes affecting wider array ability explore animal physiologies              immunological pathological functions. <br /> interact infecting microbe L. monocyto-                identified 18 mutants increased sensitivity genes enter survive wide variety cell types.        L. monocytogenes infection. genes    overcome limitations, performed                described previously immune pathogenesis <br /> ward genetic screen Drosophila identify host            functions. Secondary screens grouped mutants <br /> genes required survive infections. goals.           phenotypic classes. flies split classes  wanted cast wider net pre-             basis ability control growth L. <br /> viously used measure immunity Drosophila.                monocytogenes. define group having defects know fly immunity deciphered              endurance suspect difficulties control- <br /> using extracellular microbes immunocompromised                   ling pathogenesis mutant flies die faster flies. using L. monocytogenes study able          wild-type flies bacterial load identify host genes important survival intra-          wild-type flies. second class appears lack  <br /> cellular infection. genes specific L.             fense functions. define group immunocom- <br /> monocytogenes infection, intracellular pathogens              promised shows increased fly death general, variety pathogens. coupling survival         correlated rapid growth L. monocytogenes. bacterial proliferation output phenotypes,            mutants split sensitive wild- able identify mutants defense defects          type classes basis response Staphylococ- lines pathological defects die              cus aureus Salmonella typhimurium. Secondary screens disease faster altered bacterial load. hy-           used L. monocytogenes mutants allowed  <br /> pothesize second class mutants able         termine fly mutants acting endure stress infection. Second, wanted             vacuole cytoplasmic stages bacteria life <br /> determine genes required survive infection            cycle.  screening survival endpoint flies identified cultured           instead monitoring transcriptional output Toll <br /> cells RNAi screens. doing  screen              imd signaling, identified host genes involved reveal differences host genes involved inter-         immunity pathogenesis identi- <br /> actions animal physiology microbe,            fied previously. host genes involved interactions cell type infecting microbe.  <br />                                                                                    MATERIALS METHODS wanted probe host contributions infec- <br /> tion studies involving host&#226;&#8364;&#8220;pathogen inter-               Flies, bacterial strains, media: Exelixis Drosophila <br /> actions intracellular pathogen focused              melanogaster piggyBac insertion mutant collection used identifying microbial factors required infection.               survival screen. serve wild-type control w1118 <br />                                                                     line used generate isogenic collection used.    Drosophila innate immune response                  viable lines available Bloomington Drosophila Stock <br /> arms:  cellular immune response depends                Center beginning screen obtained (1231 <br /> circulating phagocytic cells engulf clear              lines total). Flies kept standard fly bottles containing <br /> foreign microbes. Second, melanization response                 dextrose medium. L. monocytogenes strains 10403S, DP-L2161 <br /> produces melanin toxic reactive oxygen species               (Dhly), DP-L3078 (DactA) stored 80&#194;&#176; brain&#226;&#8364;&#8220; <br />                                                                     heart infusion (BHI) broth containing 15% glycerol. L. <br /> wounds sites infection. branch              monocytogenes mutant strains derived 10403S <br /> humoral immune response, involves produc-                 parent strain generated laboratory Daniel  <br /> tion antimicrobial peptides (AMP) fat body.               Portnoy (University California, Berkeley, CA). S. typhimu- AMPs produced largely control                rium SL1344 stored 80&#194;&#176; LB media containing 15% Toll Imd pathways. pathways               glycerol. S. aureus clinical isolate Clinical <br />                                                                     Microbiological Laboratory Stanford University activated microbial elicitors disruption            stored 80&#194;&#176; BHI media containing 15% glycerol. <br /> pathways immunocompromises fly fly                   Survival screen: infection adult flies, L. monocytogenes sensitive normally nonpathogenic bacteria                grown overnight BHI medium 37&#194;&#176; shaking. like Escherichia coli (Lemaitre et al. 1995, 1996, 1997;            total 10 adult flies, aged 5&#226;&#8364;&#8220;7 days old, mutant line Kopp Medzhitov 1999, 2003; Khush et al. 2001).                  injected 50 nl culture using picospritzer pulled <br />                                                                     glass needle. Flies injected anterior abdomen Genetic screens flies primarily concen-               ventrolateral surface  1000 colony-forming units <br /> trated AMP signaling output, strengthening                 (CFUs). Mutant lines tested groups minimum understanding signaling Toll                 20 lines, addition wild-type control. infected, flies <br />                                           Survival-Based Screen Defense Endurance                                       1809 <br />  <br />  <br />  <br />  <br />                                                                                      Figure 1.&#226;&#8364;&#8221;L. monocytogenes monocytogenes infec- <br />                                                                                   tion Drosophila melanogaster. Wild-type flies                                                                                   injected 103 CFUs wild-type mu- <br />                                                                                   tant L. monocytogenes. Survival growth (growth <br />                                                                                   wild-type bacteria  monitored                                                                                   course infection.   L. monocytogenes <br />                                                                                   strain 10403S (wild type) infection. Wild-type flies <br />                                                                                   exhibit median time death (MTD)  5 days <br />                                                                                   postinfection. (B) Growth 10403S flies. Bac- <br />                                                                                   teria reach levels .104 CFUs 48 hr postinfec- <br />                                                                                   tion. **P , 0.005 (t-test). (C) Dhly infection                                                                                   flies. Wild-type flies exhibit MTD  20 days <br />                                                                                   postinfection. (D) DactA infection flies.  <br />                                                                                   fected flies exhibit MTD  10 days postinfec- <br />                                                                                   tion. Statistical analysis survival curves                                                                                   using log-rank analysis lines P , <br />                                                                                   0.05 considered statistically significant. <br />  <br />  <br />  <br />  transferred vials containing dextrose medium              overnight shaking. Thirty 7-day-old male flies incubated 29&#194;&#176; number dead flies line       mutant line infected independent replicates counted 24 hr postinfection flies dead.      mutant fly line flies injected  1000 CFUs. <br /> median time death (MTD), time postinfection               death rate line directly compared wild- <br /> 50% infected flies succumbed infection,       type control analyzed using Graphpad Prism software line determined exhibiting extreme             described  <br /> phenotypes flagged testing. Mutant lines           Verification PiggyBac insertion site: Inverse PCR failed exhibit phenotype initial test fur-       determine insertion site PiggyBac (PBac) <br /> ther tested. Candidate lines retested indepen-          element following protocol provided Exelixis. Briefly, <br /> dent experiments eliminate false positives. Thirty        Genomic DNA mutant fly line isolated using 7-day-old male flies line infected         QIAGEN (Valencia, CA) DNeasy kit. 59- 39-end digestions <br /> death rate mutant flies line compared             using Sau3AI HinPI, respectively,  <br /> directly death rate wild-type control flies, using          cubated 37&#194;&#176; 3 hr. Ligations using T4 DNA <br /> Graphpad Prism software. Using log-rank analysis, P-value         ligase 4&#194;&#176; overnight. PCR reactions performed using determined. Lines exhibited death rate              primer sequences reaction conditions provided Ex- <br /> P-value ,0.05 retests considered positive          elixis (http://flystocks.bio.indiana.edu). <br /> mutants. <br />    Secondary screens: S. aureus S. typhimurium secondary <br /> screens: infection adult flies S. aureus, bacteria                 RESULTS DISCUSSION <br /> grown overnight BHI medium 37&#194;&#176; shaking. Ap- <br /> proximately 100 CFUs injected fly.                 Screening host genes important survival: fection S. typhimurium, bacteria grown LB medium         search host genes important surviving 37&#194;&#176;  10,000 CFUs injected fly.              infection, conducted forward genetic screen infections, 35 7-day-old male flies mutant line infected mentioned death rate line          mutant population Drosophila monitored death directly compared wild-type flies analyzed      rates following challenge L. monocytogenes. L. <br /> previously mentioned.                                                 monocytogenes establishes lethal infection injec- <br />    Quantification CFUs flies: Infected flies homog-        tion Drosophila (Figure 1A) (Mansfield et al. <br /> enized BHI media supplemented 1% Triton X-100.                2003). Wild-type flies injected 103 CFUs exhibited Homogenates serially diluted plated BHI agar incubated overnight 37&#194;&#176;. number CFUs                MTD 4&#226;&#8364;&#8220;5 days postinfection. predictable death <br /> mutant line compared wild-type flies using             kinetics allowed use survival output pheno- <br /> Graphpad Prism software independent experiments.            type screen. hypothesized screening flies <br /> Using unpaired t-test, P-value determined. Mutant          ability survive L. monocytogenes infections <br /> lines exhibited P-value ,0.05 retests       allow determine immune mechanisms used <br /> considered significantly different bacterial growth <br /> compared wild-type control.                                    fight microbe, host mechanisms exploited    L. monocytogenes mutants: infection adult flies        pathogen, physiologies drive pathogenesis DP-L2161 DP-L3078, bacteria grown 37&#194;&#176;                fly. <br />  1810                                       J. S. Ayres, N. Freitag D. S. Schneider <br />  <br />                           TABLE 1                                                              TABLE 2 <br />                Distribution mutant fly lines                        Screening genes affecting defense endurance <br />  <br /> Mean time                                       PBac                                     death (day)                                          lines         Retest           lines      positive     negative      % positive <br />                                                                                 tested       lines         lines         lines <br /> 1                                                     85 <br /> 2                                                    145           Initial test     1231         101           &#226;&#8364;&#8221;              8.2 <br /> 3                                                    249           1                 101          35           66            34.6 <br /> 4                                                    510           2                  35          25           10            71.4 <br /> 5                                                    168           3                  25          18            7            72 <br /> 6                                                     58 <br /> 7                                                     16              1200 PBac insertion mutant lines tested                                                                    ability survive L. monocytogenes infection MTD    initial test, MTD mutant Drosophila       line determined. mutant lines exhibited <br /> line determined. majority mutant lines exhibit        extreme phenotypes MTDs day 1 day <br /> MTD 4 days postinfection extreme pheno-            7 ( 100 mutants) considered positive hits tested fur- <br /> types observed 1 7 days postinfection. Mutants ex-        ther eliminate false positives. candidate mutants <br /> hibiting extreme phenotypes flagged              tested independent experiments, death testing eliminate false positives.                      curves compared wild-type control, <br />                                                                    log-rank statistical analysis  Lines exhibited <br />                                                                    death rate P-value ,0.05 retests  <br />    initial screen tested 1231 viable mutants            sidered positive mutants. rounds retesting <br />                                                                    .80% mutants considered false positives publicly available collection Exelixis PBac            total 18 mutants, increased sensitivity infection, <br /> homozygous transposon insertion lines. tested              identified positive mutants affect host susceptibil- <br /> homozygous viable lines number represents                 ity L. monocytogenes infection. <br />  8% coverage Drosophila genome. Age-matched <br /> male flies mutant line injected 103            reduce possibility background effects host <br /> CFUs L. monocytogenes survival monitored.               survival. Using computed gene (CG) numbers Approximately half mutants tested exhibited               genes determined gene ontology   terms <br /> MTD 4 days postinfection (Table 1). mutant lines            FlyBase (http://flybase.bio.indiana.edu), exhibited extreme phenotypes MTDs               listed Table 3. day 1 day 7 ( 100 mutants) tested              Ubiquitination components: screen eliminate false positives. candidate mutants               identified mutant lines candidate genes tested independent experiments               categorized genes involved ubiquitina- <br /> death curves compared wild-type               tion process. crossbronx, CG15120, CG2247. <br /> control statistical analysis              surprising previous studies demonstrated <br /> rounds retesting .80% mutants                     importance ubiquitin-mediated protein degrada- <br /> considered false positives total 18 mutants,          tion regulation Toll Imd pathways. increased sensitivity infection, identified        Activation Toll pathway requires ubiquitination <br /> positive mutants affect host susceptibility L.             degradation cactus, inhibitor NF-kb-like <br /> monocytogenes infection (Table 2). study,               transcription factor DIF. Lys-63 polyubiquitination define fly exhibits reproducible, significantly         TRAF6 necessary signaling Imd path- <br /> different faster death rate compared wild-type flies            way (Zhou et al. 2005). Additionally, ubiquitination sensitive fly slower death rate com-             important controlling cytotoxicity LLO pared wild type resistant. injected positive          release L. monocytogenes vacuole mutants media confirm increased             infection. <br /> mortality infection dependent (data shown).                   cbx characterized ubiquitin-conjugating E2 <br />    Identification genes: benefit using Exelixis        enzyme. studied processes including axon <br /> PBac collection insertion sites trans-          pruning, sperm development, sperm individualiza- <br /> posons mutants collection publicly              tion. context fly immunity, cbx iden- <br /> available. Bloomington Drosophila Stock Center                 tified genetic screen possibly role (http://flystocks.bio.indiana.edu) verified             crystal cell development, type immune cell fly <br /> ported insertion sites. additional verification,            (Milchanowski et al. 2004). identification identified PBac-element insertion site          gene reveal role crystal cells host  <br /> positive mutants using iPCR BLAST sequence                  fense bacterial infection. <br /> analysis flanking DNA sequences. gene iden-                CG15120 homolog Parkin coregulated gene <br /> tities agree reported Exelixis (Table 3).         protein, PACRG, involved ubiquitination; outcrossed 18 mutants parental strain               polymorphisms PACRG related challenged F4 generation L. monocytogenes             gene, PARK2, linked increased susceptibility                                           Survival-Based Screen Defense Endurance                                     1811 <br />  <br />                                                              TABLE 3 <br />                                                Summary positive Drosophila lines <br />  <br /> PBac line          CG           Symbol        PBac orientation relative gene         Gene ontology/molecular function <br /> 10036        CG11489a                           fifth intron CG11489          <span id='am-18' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span> binding, receptor signaling, <br />                                                                                           protein serine/threonine kinase activity <br /> 10067        CG10536a              cbx          intron cbx              Ubiquitin-conjugating enzyme activity <br /> 10269        CG13564a                           59-UTR CG13564 <br /> 10496        CG14899a                           59-UTR CG14899 <br /> 10618        CG11293a                           400 bp CG11293 <br /> 10675        CG15120/CG16926                    39 end CG15120 <br />                                                   CG16926 <br /> 10743        CG13788a              gr28b        intron Gr28b            Taste receptor activity <br /> 10881        CG32706a                           100 bp 39 end CG32706      mRNA binding <br /> 17872        CG4857a,b                          intron CG4857 <br /> 18050        CG2247a                            intron CG2247 <br /> 18270        CG3527b                            59-UTR CG3527                 rRNA processing, ribosomal small <br />                                                                                           subunit processing <br /> 18398        CG2152a,b             pcmt         200 bp 59 end pcmt         Protein-l-isoaspartate (d-aspartate) <br />                                                                                           O-methyltransferase activity <br /> 18609        CG12487a              bobA         coding region BobA            Notch signaling pathway, <br />                                                                                           cell fate specification <br /> 18678        CG2914a,b             ets21C       intron Ets21C           DNA binding, transcription factor activity <br /> 19181        CG6128b                            coding region CG6128          l-fucosidase activity, fucose metabolism <br /> 19207        CG2397b               cyp6a13      coding region Cyp6a13         Electron transporter activity, <br />                                                                                           oxidoreductase activity <br /> 19220        CG3056b                            39 end CG3056                 mRNA binding <br /> 19305        CG7408b                            59-UTR CG7408                 N-acetylgalactosamine-4-sulfatase activity <br />    insertion sites closest gene PBac element mutant line listed. insertion sites determined Exelixis using iPCR verified FlyBase publicly available. <br />         verified insertion sites lines iPCR. proposed molecular functions genes provided FlyBase listed. <br />    b <br />      Genes tested RNAi screens Cheng et al. (2005). <br />  <br />  <br />  intracellular pathogen Mycobacterium leprae hu-               pression upregulated Drosophila S2 cells  <br /> mans (Lorenzetti et al. 2004; Schurr et al. 2006).                   sponse <span id='am-1' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-2' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span>LPS</span> challenge regulation dependent <br /> Polymorphisms PACRG PARK2 linked                    activation JNK signaling Imd pathway susceptibility humans pathogens S. typhi            (Boutros et al. 2002; Park et al. 2004). immune S. paratyphi (Ali et al. 2006). ubiquitin-             function gene demonstrated. <br /> related gene, CG2247 represented line 18050,                     gr28b seven-pass transmembrane gustatory  <br /> tains F-box domain facilitates interactions              ceptor important taste perception. Expression stud- proteins polyubiquitination. iden-                ies shown protein expressed bitter <br /> tification genes reveal new processes                taste cells sensilla fly taste organs. Iden- <br /> host defense pathogenesis ubiquitin-                    tifying gene involved taste perception context <br /> mediated protein degradation involved.                            immunity pathogenesis somewhat surprising. <br />    RNA binding/processing components: second                     Future experiments reveal feeding behavior main class mutants identified screen                  fly affected, affecting immune response <br /> includes proteins involved various processes                   fly. Alternatively, gr28b serving function <br /> volving RNA. CG3527 involved rRNA processing,                  taste perception fly. <br /> specifically processing small ribosomal subunit.                 Phenotypic characterization mutants&#226;&#8364;&#8221;defense <br /> CG32706 CG3056 reported mRNA-                        defects vs. endurance defects: distinguish binding activity. Expression mutation analyses              mutants defects immunity vs. pathogenesis, published genes.                                  performed secondary screens.  mea- <br />    Additional interesting genes recovered: addi-                 sured L. monocytogenes growth determine flies <br /> tional genes recovered great             defect preventing growth microbe. ets21C gr28b. ets21C DNA-binding protein                Second, measured survival infected S. transcription factor activity. Previous studies            aureus determine mutants general <br /> demonstrated ets21C immune regulated: ex-                defect preventing gram-positive bacterial growth <br />  1812                                       J. S. Ayres, N. Freitag D. S. Schneider <br />  <br />                                                                TABLE 4 <br />                 Characterization Drosophila mutants&#226;&#8364;&#8221;sensitivity S. aureus S. typhimurium growth <br />                                                     L. monocytogenes <br />  <br />         PBac line               CG                 Symbol          S. aureus        S. typhimurium         Bacterial load <br />         10067             CG10536                     cbx                1                   0                      1 <br />         10675             CG15120/CG16926                                1                   0                      1 <br />         10881             CG32706                                        1                   0                      1 <br />         17872             CG4857                                         1                   0                      1a <br />         18050             CG2247                                         1                   0                      1 <br />         19207             CG2397                      cyp6a13            1                   0                      1b <br />         19220             CG3056                                         1                   0                      1 <br />         10036             CG11489                                        1                   0                      1 <br />         18609             CG12487                     bobA               1                   0                      1b <br />         18270             CG3527                                         1                   0                      0 <br />         18398             CG2152                      pcmt               1                   0                      0 <br />         19305             CG7408                                         1                   0                      0 <br />         10496             CG14899                                        0                   0                      1 <br />         19181             CG6128                                         0                   0                      1 <br />         18678             CG2914                      ets21C             0                   0                      0 <br />         10269             CG13564                                        0                   0                      0 <br />         10618             CG11293                                        1                   1                      0 <br />         10743             CG13788                     gr28b              1                                          1 <br />            Mutant lines injected 10 CFUs S. aureus 104 CFUs S. typhimurium survival flies <br />                                               2 <br />  <br />         monitored course infection. growth L. monocytogenes mutant lines determined <br />         described materials methods. 1 indicates increased growth sensitivity compared wild-type <br />         flies, indicates resistance decreased growth compared wild-type flies. 0 indicates change sensitivity <br />         growth compared wild type. Statistical analysis survival curves using log-rank analysis lines <br />         P , 0.05 considered statistically significant. Statistical analysis bacterial growth using t- <br />         test lines P , 0.05 considered statistically significant. <br />                         Increased growth 24 hr postinfection  <br />            b <br />              Increased growth 48 hr postinfection  <br />  <br />  <br /> (Toll mutants sensitive microbes).            flies able endure infection  measured survival infected S.                    wild-type flies bacterial levels similar. <br /> typhimurium determine mutants sen-                   studies examining Drosophila immune <br /> sitive gram-negative bacteria  expected               response gram-positive bacteria involve challenges Imd pathway mutants) (Table 4).                                  nonpathogens used specific molecular <br />    monitor L. monocytogenes growth, flies                 readouts AMP production assess immune <br /> fected, homogenized 0, 24, 48 hr postinfection,               activity (Michel et al. 2001; Degregorio et al. 2002; plated count viable bacteria. mutants                  Rutschmann et al. 2002; Tauszig-Delamasure et al. increased L. monocytogenes growth compared                  2002). Defects signaling resulted inability wild-type flies. consider nature muta-            mount immune response tested gram-positive <br /> tions immunocompromising fly                    microbes. basis examination mu- allow increased growth L. monocytogenes.                  tants, clear Drosophila response gram- <br /> alternative explanation mutants              positive bacteria complex previously environment pathogen mutant                appreciated. mutants sensitive fly favorable L. monocytogenes growth             second gram-positive pathogen, S. aureus. presence <br /> mutation affects physiology outside                mutant lines clearly immunocom- <br /> mally considered immune  remaining                 promised respect L. monocytogenes mutants exhibited significant difference L. mono-              change S. aureus susceptibility indicates cytogenes levels compared wild type. new              multiple mechanisms pathways used fight gram- <br /> class mutant phenotype sense                positive pathogenic bacteria. <br /> commonly studied field fly immunity.                    determine mutations causing increased <br /> microbial proliferation unchanged group,                  sensitivity L. monocytogenes infection affect mutants regarded immunocom-                   infections gram-negative bacteria, challenged <br /> promised; instead, suggest flies               flies S. typhimurium. gene, CG11293, shows <br /> altered physiology makes sensitive                 increased sensitivity S. typhimurium. mutant ex- pathology induced L. monocytogenes infection.             hibits increased sensitivity L. monocytogenes S. aureus <br />                                        Survival-Based Screen Defense Endurance                                   1813 <br />  <br />                           TABLE 5                                 mutants Dhly DactA determine         Characterization L. monocytogenes mutants           Drosophila mutants involved specific steps                        Drosophila lines                           bacterial life cycle. hypothesized fly <br />                                                                   mutations involved specific stages life cycle, <br /> PBac line           CG            Symbol     Dhly      DactA   fly susceptibility mutant bacterial infections <br />                                                                   different compared wild-type bacterial <br /> 18270         CG3527 <br />                                                                   infection. <br /> 10067         CG10536                cbx <br /> 10269         CG13564                                                14 mutant fly lines, mutant L. <br /> 10496         CG14899                                             monocytogenes strains acted wild-type flies, <br /> 10618         CG11293                                             killing flies slowly wild-type L. monocyto- <br /> 10675         CG15120/CG16926                                     genes (Table 5) (individual death curves provided 18050         CG2247                                              supplemental materials). 4 fly mutants <br /> 19181         CG6128                                              (CG11489, CG4857, CG32706, gr28b) flies 19207         CG2397                 cyp6a13 <br />                                                                   equally sensitive L. monocytogenes mutant <br /> 19220         CG3056 <br /> 19305         CG7408                                              wild-type L. monocytogenes. mutants, CG11489 <br /> 18678         CG2914                 ets21C                       CG4857, just sensitive infection DactA <br /> 18609         CG12487                bobA                         infection wild-type bacteria 10036         CG11489                                      0      sensitive Dhly mutants. 17872         CG4857                                       0      words, DactA L. monocytogenes  <br /> 10881         CG32706                             0               fective cell cell spread, mutant fly lines 18398         CG2152                 pcmt <br />                                                                   just sensitive infected wild- <br /> 10743         CG13788                gr28b        0        0 <br />                                                                   type bacteria.  mutant fly lines respond <br />    Fly lines infected Dhly DactA mutant lines     like wild-type flies infection L. monocytogenes <br /> described materials methods. indicates resistance          lacking LLO resistant. basis compared 10403S infection. 0 indicates change sen- <br /> sitivity compared 10403S infection. Statistical analysis    RNAi screen results, Cheng et al. (2005) suggested <br /> survival curves using log-rank analysis lines   mutants like sensitive LLO <br /> P , 0.05 considered statistically significant.               knocked genes involved                                                                   controlling LLO toxicity. model apply infection unaltered L. monocytogenes growth. sug-          mutants;  mutants gest flies general pathogenesis defect          sensitive S. aureus presumably sensitivity S. sensitive different pathogenic          aureus caused perturbed process bacteria bacterial load comparable          result sensitivity LLO. mutant wild-type flies. gr28b exhibits increased resistance      affects general process involved sensing bacteria. S. typhimurium infection sensitive            CG32706 sensitive Dhly mutants compara- <br /> S. aureus. increased resistance unexpected            tively insensitive DactA infection.  phenotype; aware fly mutations            Dhly strain defective vacuole escape, kills cause resistance gram-negative bacteria               mutant fly line similar kinetics wild-type L. <br /> increased sensitivity gram-positive bacteria.                  monocytogenes. fly line resistant L. mono- <br />    Characterization L. monocytogenes mutants                cytogenes lack ActA defective cell cell <br /> Drosophila mutants: chose L. monocytogenes              spread. flies LLO required L. <br /> pathogen screen life cycle viru-          monocytogenes released cytosol. Similar <br /> lence factors characterized,  using         phenotypes observed vitro screens bacterial mutants, test changes pathogen-           genes involved later stages vesicular traffick- <br /> esis mutant flies. taken host          ing knocked (Cheng et al. 2005). <br /> cell, L. monocytogenes secretes LLO lyse phago-               gr28b sensitive Dhly DactA L. monocytogenes. membrane release bacteria                   shown  gr28b sensitive L. monocytogenes cytoplasm. protein ActA important nucleating           S. aureus, increased resistance S. typhimurium. <br /> host actin pole bacterium. Actin polymer-           mutant strains L. monocytogenes ization drives motility L. monocytogenes cyto-          attenuated fly line, suggest LLO ActA <br /> plasm infected cell enables bacteria             required bacteria elicit pathogenic host cell entering           effects fly line. words, L. monocytogenes <br /> extracellular space, avoiding immune defenses              lacking factors required normal intracellular patho- <br /> host. Dhly (LLO deletion) DactA                 genesis attenuated, indicating L. monocytogenes <br /> mutants strongly attenuated wild-type Drosophila,          gr28b mutant extracellular population. MTD 18&#226;&#8364;&#8220;20 days 11&#226;&#8364;&#8220;12 days, respectively                Members immunity pathways identified: <br /> (Figure 1, C D). examined susceptibility                known immunity genes present fly mutants infection L. monocytogenes            collection mutant flies tested (imd kenny). <br />  1814                                     J. S. Ayres, N. Freitag D. S. Schneider <br />  previous studies fly immunity primarily          bacteria host genes involved focused Toll Imd pathways, anticipated             infection. Attempts overcome limitations genes identified                RNAi screens S2 cells infected <br /> screen. mutants median times death               L. monocytogenes. screens revealed im- <br /> 2 days, tested            portant host factors required internalization outside cutoff median time death 1 day.        growth L. monocytogenes host cells screened relatively small number mutants,       capable identifying immunity- pathogenesis-related <br /> possible did hit members Toll             genes. screening host survival fly model, <br /> signaling pathway small sample size.           probe host contributions infection addition, Toll pathway mutants meet               extend processes screens viability requirement excluded.              uncovered genes discovered gardless, screen demonstrates fighting infec-       methods. <br /> tion interactions occur host               thank Marc Dionne, Linh Pham, Eric Mabery, Madeleine <br /> pathogen complex, processes addition             Moule critical review manuscript. DP-L3078 (DactA) DP- pathways regulating AMP transcription involved.       L2161 (Dhly) kindly provided Julie Theriot (Stanford <br />    importance performing vivo                 University). S. aureus strain provided Ellen Jo Baron.                                                                  thank Stanley Falkow, Michael Simon, Julie Theriot, members vitro screens: compared list mutant genes <br />                                                                  Schneider lab helpful discussions. especially thank hundreds genes identified RNAi screens            Adam, Baxter, Buster Ayres support. work monitoring L. monocytogenes growth cultured Dro-              supported National Institutes Health grants RO1 A1053080-01 <br /> sophila cells. mutations essential               (D.S.S.) A1055651 (D.S.S.) National Science Founda- <br /> functions lethal phenotype             tion graduate research program ( J.S ). <br /> fly, anticipated genes fly identified vitro. Conversely, expected genes important                                 LITERATURE CITED animal tissue culture. Nonetheless,                                                                  Agaisse, H., L. S. Burrack, J. Philips, E. J. Rubin, N. Perrimon et al., <br /> expected significant overlap sets genes                2005 Genome wide RNAi screen host factors required isolated screens; instead,                   intracellular bacterial infection. Science 309: 1248&#226;&#8364;&#8220;1251. <br /> overlap. screen Agaisse et al. (2005)               Ali, S.,  M. Vollaard, S. Widjaja, C. Surjadi, E. van Vosse et al., <br />                                                                       2006 PARK2/PACRG polymorphisms susceptibility ty- <br /> genomewide screen theoretically mutants                phoid paratyphoid fever. Clin. Exp. Immunol. 144: 425&#226;&#8364;&#8220;431. tested screen. dsRNA                Ayres, J. S., D. S. Schneider, 2006 Genomic dissection mi- <br /> library used Cheng et al. (2005) contained                crobial pathogenesis cultured Drosophila cells. Trends Micro- <br />                                                                       biol. 14: 101&#226;&#8364;&#8220;104. positive mutants, identified              Boutros, M., H. Agaisse N. Perrimon, 2002 Sequential activa- vitro screen  Table 3).               tion signaling pathways innate immune responses vitro screens selected bacterial phenotypes,                Drosophila. Dev. Cell 3: 711&#226;&#8364;&#8220;722. <br />                                                                  Cheng, L. W., J. P. Viala, N. Stuurman, U. Wiedermann, R. D. Vale <br /> cluding growth vacuole escape, utilized cell              et al., 2005 Use RNA interference Drosophila S2 cells type. majority genes identified vitro            identify host pathways controlling compartmentalization  <br /> screens  surprisingly, involved vesicular traf-            tracellular pathogen. Proc. Natl. Acad. Sci. USA 102: 13646&#226;&#8364;&#8220;13651. <br />                                                                  Cossart, P., M. Lecuit, 1998 Interactions Listeria monocyto- <br /> ficking phagocytosis. screening host survival              genes mammalian cells entry actin-based  animal model, able identify different           ment: bacterial factors, cellular ligands signaling. EMBO J. <br /> host genes involved variety processes infec-           17: 3797&#226;&#8364;&#8220;3806. <br />                                                                  Cossart, P., J. Pizarro-Cerda M. Lecuit, 2003 Invasion tion, including immune response pathogenesis.                 mammalian cells Listeria monocytogenes: functional mimicry <br />    Conclusions: study demonstrate power                subvert cellular functions. Trends Cell Biol. 13: 23&#226;&#8364;&#8220;31. combining genetically tractable organisms,            Degregorio, E., P. T. Spellman, P. Tzou, G. M. Rubin B. <br />                                                                       Lemaitre, 2002 Toll Imd pathways major reg- <br /> host pathogen, reveal previously unchar-                 ulators immune response Drosophila. EMBO J. 21: <br /> acterized group genes involved immunity                     2568&#226;&#8364;&#8220;2579. <br /> pathogenesis. Previous genetic studies using             Domann, E., J. Wehland, M. Rohde, S. Pistor, M. Hartl et al., <br />                                                                       1992 novel bacterial virulence gene Listeria monocytogenes <br /> ganism separately greatly limited scope genes              required host cell microfilament interaction homology <br /> processes identified. use Drosophila               proline-rich region vinculin. EMBO J. 11: 1981&#226;&#8364;&#8220;1990. study innate immune response infection                 Foley, E., P. H. O&#226;&#8364;&#8482;Farrell, 2004 Functional dissection  <br />                                                                       nate immune response genome-wide RNAi screen. PLoS <br /> facilitated understanding signaling events                Biol. 2: E203. <br /> lead production AMPs. Previous screens          Gedde, M. M., D. E. Higgins, T. G. Tilney D.  Portnoy, <br /> fly S2 cells focused identifying genes                 2000 Role listeriolysin O cell cell spread Listeria mono- <br />                                                                       cytogenes. Infect. Immun. 68: 999&#226;&#8364;&#8220;1003. <br /> involved signaling pathways, did           Gesellchen, V., D. Kuttenkeuler, M. Steckel, N. Pelte M. <br /> look process. vitro studies using L.                   Boutros, 2005 RNA interference screen identifies Inhibi- <br /> monocytogenes expanded knowledge better                   tor Apoptosis Protein 2 regulator innate immune sig- <br />                                                                       naling Drosophila. EMBO Rep. 6: 979&#226;&#8364;&#8220;984. <br /> understand pathogenic mechanisms used intracellu-             Kambris, , S. Brun,  H. Jang, H. J. Nam, Y. Romeo et al., <br /> lar pathogens. genetic studies focused                 2006 Drosophila immunity: large-scale vivo RNAi screen <br />                                               Survival-Based Screen Defense Endurance                                                1815 <br />  <br />     identifies serine proteases required Toll activation. Curr.     Michel, T., J. M. Reichart, J.  Hoffmann J. Royet, 2001 Dro- <br />     Biol. 16: 808&#226;&#8364;&#8220;813.                    <br /> </body></html>