<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cytometry  Author manuscript; available PMC 2009 July 1. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cytometry  2008 July ; 73(7): 637&#226;&#8364;&#8220;642. doi:10.1002/cyto 20579. <br />  <br />  <br />  <br />                            long-term flow cytometry assay analyze role specific <br />                            genes Drosophila melanogaster S2 cells surviving <br />                            genotoxic stress <br />                            Xia Yi1,2, Willy Lemstra1, Michel J. Vos1, Yongfeng Shang2, Harm H. Kampinga1, Tin Tin <br />                            Su3, Ody C.M. Sibon1,* <br />  <br />                            1Department Radiation Stress Cell Biology, Division Cell Biology, University Medical <br />                            Center Groningen, University Groningen, Netherlands 2 Department Biochemistry                            Molecular Biology, Peking University Health Science Center, Beijing 100083, China 3 Department <br />                            Molecular, Cellular Developmental Biology University Colorado, Boulder,                            80309-0347 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Abstract <br />                                  Background&#226;&#8364;&#8221;Drosophila S2 cells easy manipulate culture versatile <br />                                  model high throughput screens genome-wide siRNA screens genes <br />                                  involved stress therapy resistance screening large compound libraries                                  identify cytotoxins. Clonogenic assays considered gold-standard investigate                                  cytotoxicity specific treatments compare sensitivity various cell types specific <br />                                  treatment.  assay used Drosophila S2 cells virtually unable <br />                                  grow distinct colonies. <br />                                  Methods designed novel fluorescence-based flow cytometry assay study long-term <br />                                  proliferation S2 cells various conditions presence specific gene products                                  downregulation specific gene products. <br />                                  Results validate assay used novel method investigate role                                  checkpoint genes grapes/Dchk1 DmChk2 cell survival responses. <br />                                  Conclusions data demonstrate Grapes/Dchk1 DmChk2 required survive <br />                                  hydroxyurea. assay use investigate long-term effects various treatments S2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  cells evaluate role specific proteins  <br />  <br />                            Keywords <br />                                  viability; sensitivity; Drosophila S2 cells; FCM; DNA damage; survival assay; RNAi technology; <br />                                  grapes/Dchk1; DmChk2 <br />  <br />  <br />                            Introduction <br />                                             ability cells form clones certain treatment used study survival <br />                                             potential specific cells (1). called clonogenic assay method determine <br />                                             proliferation capacity assay determines actual ability cell repeatedly <br />                                             divide form visible colony. clonogenic assays, cells investigation <br />  <br />  <br />                            * <br />                             Corresponding author: address: Ant. Deusinglaan 1, 9713 AV Groningen, Netherlands, e-mail: o.c.m.sibon@med.umcg.nl; <br />                            telephone: +31(0)50 363 2559; fax: +31(0)50 363 2913. <br />                             Yi et al.                                                                                                    Page 2 <br />  <br />  <br />                                               able form colonies solid surface  adherent cells) fluid medium                                               substrates agar non-adherent cells. clonogenic assays                                               alternative methods evaluate proliferation viability cells specific conditions NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               specific treatments. assays, incorporation rates radiolabeled nucleotides <br />                                               3H-<span id='am-6' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>thymidine</span> used indication proliferation (2). known <br />                                               assays quantify viability, MTT assay (3) MTS assay (4), measure                                               concentration products generated converted viable cells. Using alternative <br />                                               assays possible distinguish actual cell death delayed cell death, <br />                                               temporary growth inhibition permanent growth inhibition hamper proper <br />                                               data analysis.   time-consuming) long-term clonogenic assay                                               considered golden standard investigate proliferation capacity dividing cells. <br />                                               Drosophila melanogaster Schneider line 2 (S2) cells derived primary culture                                               20 24-hour-old Drosophila embryos. cells relatively easy culture                                               cells grow room temperature controlled <span id='am-1' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CO2</span> levels. S2 cells normally grow                                               semisuspension need trypsinization prior passaging. powerful <br />                                               characteristic S2 cells multiple transcripts simultaneously downregulated <br />                                               near 100 % efficiency using RNAi technology transient assays (5,6).                                               addition, S2 cells highly permeable expensive transfection carriers needed                                               perform RNAi studies. Finally, S2 cells induce interferon response interferes <br />                                               RNAi gene silencing (7). Consequently, S2 cells relatively easy manipulate                                               provide powerful model investigate pro- anti- survival functions NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               specific genes. Unfortunately, clonogenic assays used S2 cells                                               adherent grow discrete colonies.  present simple method <br />                                               using flow cytometry (FCM) accurately compare proliferation capacity S2 cells                                               response specific treatments presence absence specific proteins                                               extended period time. unique feature assay continual adjustment cell <br />                                               density various samples analysis, using methods. use &#226;&#8364;&#339;feeder cells&#226;&#8364;? <br />                                                differentially marked experimental cells) equalize total cell densities                                               cultures subjected different treatments order rule indirect effects survival                                               differences cell number. Second, protocol includes adjustment cell densities <br />                                               days order extend period exponential growth allowing                                               analyze long-term effects. addition established methods, approach useful                                               study long-term effects suspension cells  especially cells                                               analyzed clonogenic assays. <br />  <br />                                Materials methods <br />                                Cell culture <br />                                               Drosophila Schneider&#226;&#8364;&#8482;s S2 cells (S2 cells), S2 cells stably express nuclear luciferase- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               eGFP S2 cells stably express H2B-GFP cultured Schneider&#226;&#8364;&#8482;s Drosophila <br />                                               medium (GIBCO, Paisley, UK) supplemented 10% heat-inactivated Fetal Bovine <br />                                               Serum (Greiner, Netherlands), 100 units/ml penicillin 100 &#206;&#188;g/ml streptomycin T25 <br />                                               flasks 25&#194;&#176;C. Nuclear luciferase-eGFP (eGFP-tagged firefly luciferase fused Large T <br />                                               Antigen nuclear localization signal) expressing cells generated according                                               protocol manufacturer (Drosophila expression  Invitrogen). H2B-GFP <br />                                               expressing S2 cells generous gift P.H. O&#226;&#8364;&#8482;Farrell (San Francisco, California, USA). <br />                                               order continue exponential cell growth, cell density kept 3&#195;&#8212;105 cells/ml <br />                                               3&#195;&#8212;106 cells/ml. experimental use cells cultured 6 plates 35 <br />                                               mm dishes. <br />  <br />  <br />  <br />  <br />                                                     Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                               Page 3 <br />  <br />  <br />                                RNA interference <br />                                            RNA interference performed previously described (5,8). short, grp/Dchk1                                            DmChk2 DNA fragment 700~800 bp length containing coding sequence grp/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Dchk (GenBank accession  AF057041, sense primer 493&#226;&#8364;&#8220;509 anti-sense primer <br />                                            1185&#226;&#8364;&#8220;1200) DmChk2 cDNA (Gen Bank accession  NM-165318: sense primer 666&#226;&#8364;&#8220; <br />                                            683 anti-sense primer 1395&#226;&#8364;&#8220;1413) amplified using PCR. primer used                                            PCR contained 5&#226;&#8364;&#178; T7 RNA polymerase binding site <br />                                            (GAATTAATACGACTCACTATAGGGAGA) followed sequence specific grp/ <br />                                            Dchk1 DmChk2. dsRNA generated using MEGASCRIPT T7 transcription kit <br />                                            (Ambion, Austin, TX) stored &#226;&#710;&#8217;20&#194;&#176;C. Nuclear luciferase-eGFP-expressing S2 cells, <br />                                            H2B-GFP-expressing S2 cells S2 cells diluted final concentration 1&#195;&#8212;106 <br />                                            cells/ml Drosophila Serum Free Medium (GIBCO, Paisley, UK). 1 ml cells plated <br />                                            35 mm dishes 10 &#206;&#188;g dsRNA added Serum Free Medium dish.                                            additional transfection agent added, demonstrated previously                                            transfection carriers required perform successful RNAi experiments S2 cells (5,6). <br />                                            cells incubated 1 hour 25&#194;&#176;C followed addition 2 ml complete <br />                                            Schneider&#226;&#8364;&#8482;s medium. Cells incubated 96 hours downregulate Grp/Dchk1                                            DmChk2 proteins. Downregulation Grp/Dchk1 DmChk2 RNAi controlled <br />                                            Western Blot analysis using antibodies specific Grp/Dchk1 DmChk2                                            course experiment (Supplementary Figure 1). control cells  referred                                            mock RNAi-treated cells), GFP-expressing S2 cells transfected non-relevant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            dsRNA construct dsRNA added medium. non-relevant dsRNA <br />                                            construct, fragment coding sequence stonewall used, stonewall gene <br />                                            required oogenesis (9) supplementary file). Kinetics downregulation Grp/ <br />                                            Dchk1 DmChk2 comparable nuclear luciferase-eGFP-expressing cells, H2B- <br />                                            GFP-expressing cells S2 cells. <br />  <br />                                Proliferation Assay <br />                                            GFP-expressing-S2 cells (Figure 1) transfected dsRNA (e.g. dsRNA                                            Grp/Dchk1 DmChk2). 4 days, RNAi-treated GFP-S2 cells mixed 1:1                                            non-GFP expressing S2 cells  referred feeder cells). experiments shown <br />                                            manuscript performed using Nuclear luciferase-eGFP expressing cells H2B- <br />                                            GFP-expressing cells. overnight incubation, mixed cells treated 10 mM <br />                                            HU (Sigma, St Louis, MO, USA) 6 hours left untreated. time point                                            considered day 0 experiment. Subsequently, fraction living cells                                            calculated staining trypan blue detects dead cells (note: immediately HU <br />                                            treatment, increase observed dead cells compared untreated <br />                                            cells). condition following protocol used. mixed populations cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (containing GFP-expressing cells feeder cells) subcultured split                                            fractions: cohort Ia cohort Ib. Cohort Ia seeded density 3 &#195;&#8212; 105 living cells <br />                                            ml medium cultured 6 plates FCM analysis day 2 performed. <br />                                            detailed describtion FCM analysis, supplementary Figure 3. Cohort Ib seeded <br />                                            density 3 &#195;&#8212; 105 living cells ml cultured T25 flask FCM analysis <br />                                            later time points (day 4, 6, 9, 11, 13). day 2 (48 hours HU treatment),                                            percentage GFP-expressing cells cohort Ia measured using FCM analysis. Cells                                            cohort Ib split fractions (cohort IIa cohort IIb), suspended                                            3 &#195;&#8212; 105 (cohort IIa) viable cells ml medium seeded 6 plates FCM <br />                                            analysis day 4 3 &#195;&#8212; 105 viable cells (cohort IIb) ml cultured T25 flask <br />                                            FCM analysis later time points (day 6, 9, 11, 13)  course                                            experiment, 2 days density cells adjusted diluting mixed <br />                                            populations cells fresh medium density 3 &#195;&#8212; 105 living cells ml                                            reached. percentage viable cells calculated using trypan blue staining cell <br />  <br />  <br />                                                   Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                  Page 4 <br />  <br />  <br />                                            density corrected resulting constant narrow range viable cell density <br />                                            experiment  supplementary file details).                                            condition, 3 independent experiments performed data presented mean &#194;&#177; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            standard deviation mean (SDM). <br />  <br />                                FCM analysis <br />                                            Cells harvested centrifugation, washed phosphate-buffered saline (PBS), <br />                                            cell pellets resuspended 0.2&#226;&#8364;&#8220;0.5 ml PBS. percentage GFP-expressing <br />                                            cells determined using FACSCalibur Flow Cytometer (Becton Dickinson, San Jose, <br />                                            CA), using 488 nm Argon laser (15 mWatt) standard filters. 10,000 events                                            measured sample. FL1 used detect GFP fluorescence FL2                                            autofluorescence. Data analysis WinList (version 6.0) Verity Software <br />                                            House (Topsham,   supplementary file). typical experiment                                            condition independently performed measured triplicate. <br />  <br />                                Western blot analysis <br />                                            Western blot analysis S2 cells prepared previously described using primary rabbit <br />                                            polyclonal antibodies Grp/Dchk1 DmChk2 (5)commercial available secondary <br />                                            antibodies ECL detection reagents (Amersham Biosciences, Amersham Place, UK). &#206;&#179;- <br />                                            tubulin (Sigma T6557, St Louis, MO, USA) used loading control  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            supplementary file). <br />  <br />                                Results <br />                                            Drosophila S2 cells widely used study function specific genes (6,10&#226;&#8364;&#8220;12). <br />                                            Recently, demonstrated S2 cells possess functional checkpoint delays entry <br />                                            cells mitosis DNA damaged ionizing radiation   DNA <br />                                            replication inhibited hydroxyurea (HU) (5). order investigate role                                            specific Drosophila genes responses DNA damaging agents, long-term survival <br />                                            assay, applicable S2 cells, required. Firstly, investigated S2 cell survival <br />                                            investigated using clonogenic assay. Despite attempts unable                                            obtain reproducible results using clonogenic assays S2 cells did form colonies                                            grown various types agar (data shown). <br />  <br />                                            (mammalian) tissue culture cells, S2 cells investigation                                            cultured conditions cell densities extensively fluctuate                                            cells grow exponentially. laboratory S2 cells routinely cultured cell density <br />                                            3 &#195;&#8212; 105 cells/ml 3 &#195;&#8212; 106 cells/ml. S2 cell doubling time hampered                                            density lower 3 &#195;&#8212; 105 cells/ml higher 3 &#195;&#8212; 106 cells/ml. cell doubling time <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            S2 cells approximately 24 hours. survival responses need investigated <br />                                            3&#226;&#8364;&#8220;4 days, cells grow exponentially. aim study design <br />                                            long-term (&gt;4 days) survival assay S2 cells provide method study function <br />                                            specific genes  grapes/Dchk1  referred grp/Dchk1) DmChk2)                                            survival response various DNA damaging treatments. cell density influences S2 <br />                                            cell proliferation, long-term assay, cell density needs kept strict <br />                                            control allowing cells grow exponentially. <br />  <br />                                long-term survival assay investigate role Grp/Dchk1 DNA damage response <br />                                pathways <br />                                            alternative method clonogenic assay designed method depicted Figure <br />                                            1. assay proliferation rate specific cell type compared                                            proliferation rate specific cell type protein downregulated (e.g. <br />  <br />  <br />                                                  Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                           Page 5 <br />  <br />  <br />                                        Grp/DmChk1). measured presence absence specific treatment (e.g. <br />                                        HU). assay types cells mixed 1:1 ratio: GFP-expressing cells                                        specific proteins downregulated non-GFP-expressing cells serve NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        compensate cell density  referred feeder cells). GFP-expressing <br />                                        cells, nuclear luciferase-eGFP-expressing cells used. percentage GFP- <br />                                        expressing cells mixture feeder cells measured FCM analysis.                                        control GFP-expressing cells mock transfected transfected non-relevant <br />                                        dsRNA. control cells mixed type feeder cells.                                        validate method, effect proliferation capacity Grp/Dchk1-downregulated <br />                                        cells compared survival mock-RNAi treated (= Grp/Dchk1 containing) cells <br />                                        control conditions treatment DNA synthesis inhibitor HU. Grp/ <br />                                        Dchk1 required cell cycle checkpoint regulation response HU (5). absence <br />                                        Grp/Dchk1 presence replication inhibition, cells cell cycle <br />                                        delay, enter mitosis presence incompletely replicated DNA abnormal <br />                                        cytokineses observed (5,13). Based  prediction Grp/Dchk1- <br />                                        downregulated cells decrease proliferative capacity HU treatment compared <br />                                        mock-RNAi treated cells. <br />  <br />                                        Firstly, efficiency RNAi treatment examined Western blot analysis 4 <br />                                        days start RNAi treatment, Grp/Dchk1 levels undetectable                                        remained undetectable 9 days (Figure 2). Grp/Dchk1 protein levels <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        undetectable 4 days start RENAi treatment, experiments mixed <br />                                        populations cells started time point (4 days RNAi treatment                                        corresponds 0 days HU treatment). Subsequently, tested Grp/ <br />                                        Dchk1-downregulation affects proliferation rate cells. Hereto percentage mock <br />                                        RNAi-treated GFP-expressing cells  mixture feeder cells) measured <br />                                        various times compared percentage Grp/Dchk1-downregulated GFP- <br />                                        expressing cells  mixture feeder cells). presented Figure 3A,                                        large difference proliferation rate mock-RNAi treated GFP-expressing cells <br />                                        Grp/Dchk1-RNAi treated GFP-expressing cells  supplementary file statistical <br />                                        analysis).  tested role Grp/Dchk1 surviving HU treatment. Hereto                                        percentage GFP-positive cells HU treatment measured various time points. <br />                                        Figure 3A demonstrates HU reduces proliferation larger extent Grp/Dchk1- <br />                                        downregulated cells compared mock-RNAi treated cells. order exclude                                        observed decrease proliferation capacity Grp/Dchk1-downregulated cells HU                                        specific nuclear luciferase-eGFP-expressing cells, Grp/Dchk1                                        downregulated H2B-GFP-expressing cells  absolute ratios                                        different) resulted comparable outcome cells Grp/Dchk1                                        downregulated increased sensitivity HU compared cells Grp/Dchk1 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        downregulated (Figure 3B). Note regardless normal Grp/Dchk1 <br />                                        expression downregulation Grp/Dchk1, GFP-expressing cells appear                                        growth advantage feeder cells percentage fluorescent cells increased <br />                                        50% course 13 days.  does affect outcome <br />                                        experiments reverse experiment performed Grp/Dchk1                                        downregulated non-GFP expressing cells  GFP-expressing cells served feeder <br />                                        cells), resulted outcome Grp/Dchk1-downregulated cells showed                                        impaired proliferation capacity HU treatment compared mock-RNAi treated cells <br />                                        (Supplementary Figure 4). data consistent previous observations <br />                                        demonstrating grp/Dchk1 mutant larvae sensitive HU (13&#226;&#8364;&#8220;15) data                                        Grp/Dchk1 required prevent mitotic catastrophe presence HU (5), <br />                                        demonstrating validity assay. <br />  <br />  <br />  <br />  <br />                                              Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                 Page 6 <br />  <br />  <br />                                DmChk2 required survive HU treatment <br />                                            validation survival assay, tested effect downregulation                                            protein response HU. DmChk2  Dmnk) conserved checkpoint kinase involved NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            various aspects DNA damage responses. DmChk2 mediates p53-mediated transcription                                            DNA repair genes (16), required cell cycle checkpoint function cycle 14                                            embryogenesis (17) involved apoptosis (16).  previously demonstrated <br />                                            DmChk2 depleted cells normal cell cycle delay response HU Grp/ <br />                                            Dchk1 DmChk2 required G2/M checkpoint function presence                                            damaged incompletely replicated DNA (5). <br />  <br />                                            Firstly, investigated downregulation DmChk2 successful                                            course experiment demonstrated DmChk2 detectable day <br />                                            10 RNAi treatment (Figure 4). Comparable Grp/Dchk1 RNAi treatment, <br />                                            downregulation DmChk2 did change proliferation rate cells cultured <br />                                            control conditions (Figure 3A). Remarkably, HU treatment, proliferation rate <br />                                            DmChk2-downregulated cells comparable HU-treated control cells DmChk2 <br />                                            downregulated cells sensitive HU compared Grp/Dchk1-downregulated cells <br />                                            (Figure 3A). <br />  <br />                                            Finally, investigated proliferative capacity cells Grp/Dchk1                                            DmChk2 downregulated. Double downregulation Grp/Dchk1 DmChk2 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            efficient downregulation single protein (Figure 5). Downregulation Grp/Dchk1                                            DmChk2 downregulation Grp/Dchk1 DmChk2 simultaneously did alter <br />                                            cell proliferation S2 cells normal culturing conditions (Figure 3A). Double <br />                                            downregulation Grp/Dchk1 DmChk2 did result similar decrease cell <br />                                            proliferation HU treatment compared downregulation Grp/Dchk1 (Figure <br />                                            3A).  data demonstrate Grp/Dchk1  far lesser <br />                                            extent) DmChk2 required maintenance proliferation HU-treated cells                                            DmChk2 plays multiple roles DNA damage responses (16,17),                                            DmChk2-dependent responses required S2 cells recover HU treatment. <br />  <br />                                Discussion <br />                                            addition existing methods, survival assay measure                                            proliferative capacity cells presence specific proteins downregulation                                            proteins, control culturing conditions specific treatments.                                            characteristics assay course experiment cell density                                            compensated means; 1) RNAi-treated  mock RNAi-treated) GFP-positive cells                                            mixed feeder cells 2) course experiment, viable <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cells condition adjusted 2 days. adjustment cell density overcomes                                            complication low cell density hampers growth affects outcome survival <br />                                            experiments. Compensating cell density especially important cells                                            investigation need cultured following strict density protocols. cell density                                            compensated  theory, possible specific treatment eliminating large <br />                                            proportion viable cells remaining cells unable grow resulting <br />                                            decrease density critical concentration. inability proliferate                                            treatment itself low density. assay cell density                                            compensated used distinguish possibilities.                                            contrast, assay described applied investigate proliferation capacity                                            cells treatments eliminate large proportions cells. <br />  <br />                                            presented assay percentage GFP-expressing (RNAi-treated) cells                                            mixture feeder cells measured time. case percentage GFP expressing <br />  <br />  <br />                                                  Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                             Page 7 <br />  <br />  <br />                                          cells increasing time, indicates GFP-expressing cells growth <br />                                          advantage compared feeder cells. normal culturing conditions  absence <br />                                          HU) percentage GFP-expressing cells increases time, likely reflecting NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          slightly shorter doubling time GFP expressing cells. phenomenon observed                                          control GFP-expressing cells, mock RNAi-treated, Grp/Dchk1- DmChk2- <br />                                          downregulated cells. phenomenon observed HU treatment control GFP- <br />                                          expressing cells, mock-RNAi treated GFP-expressing cells DmChk2-downregulated <br />                                          GFP-expressing cells, demonstrating growth advantage GFP expressing cells                                          influenced HU presence absence DmChk2.                                           downregulation Grp/Dchk1 presence HU, percentage GFP-expressing <br />                                          cells decreasing time, indicative growth disadvantage Grp/Dchk1- <br />                                          downregulated cells. Possible explanations growth disadvantage increased cell <br />                                          death, induced cell cycle delay cell cycle arrest. observed increased sensitivity                                          Grp/Dchk1-downregulated cells exclusive phenomenon GFP-expressing cells <br />                                          Grp/Dchk1-depleted non-GFP-expressing cells sensitive HU <br />                                          compared mock-RNAi treated DmChk2-downregulated cells (Supplementary Figure <br />                                          4). increased sensitivity Grp/Dchk1-downregulated cells consistent previous <br />                                          data published (5,15,18).  observation downregulation                                          DmChk2 does influence proliferation capacity HU remarkable. DmChk2 plays                                          role various cell cycle checkpoints DNA damage responses. Based                                           prediction DmChk2-downregulated cells hypersensitive DNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                          damaging insults.  possible role cell checkpoint regulation                                          demonstrated specific stages embryogenesis (19,20) S2 cells (5).                                          downregulation DmChk2 did decrease proliferation capacity Grp/ <br />                                          Dchk1-downregulated cells, indicating DmChk2 S2 cells does play role <br />                                          recovering HU-induced impaired DNA integrity. <br />  <br />                                          summary, presented assay useful tool evaluate proliferation capacities cells <br />                                          suspension provides effective method investigate function proteins <br />                                          (individual simultaneously) involved responses various treatments. specific <br />                                          proteins downregulated GFP-expressing cells, cells subsequently                                          sorted FACS various biochemical assays expression profile studies                                          performed analyze consequences treatments specific gene products                                          downregulated. effect single treatments analyzed survival                                          investigated long period time, effect repeated treatments investigated <br />                                          role specific genes required survival repeated treatments.                                          proteins stably downregulated  regulated), role proteins                                          survival responses analyzed extended period time. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Acknowledgments <br />                                          thank Geert Mesander Henk Moes assistence FCM analysis. work supported                                          UMCG Bernouilli Bursary grant (809003) X.Y, grant National Institute Health (GM66441)                                          T.T.S VIDI grant Netherlands Organization Scientific Research NWO (917-36-400)                                          O.C.M.S. <br />  <br />  <br />                                Reference List <br />                                          1. Baroffio  Dupin E, Le Douarin NM. Clone-forming ability differentiation potential                                             migratory neural crest cells. Proc Natl Acad Sci USA. 1988; 85:5325&#226;&#8364;&#8220;5329. [PubMed: 2455901] <br />                                          2. Bodycote J, Wolff S. Metabolic breakdown [3H]thymidine inability measure human <br />                                             lymphocyte proliferation incorporation radioactivity. Proc Natl Acad Sci USA. 1986; <br />                                             83:4749&#226;&#8364;&#8220;4753. [PubMed: 3460068] <br />  <br />  <br />  <br />  <br />                                                 Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                                   Page 8 <br />  <br />  <br />                                        3. Ferrari M, Fornasiero MC, Isetta  MTT colorimetric assay testing macrophage cytotoxic <br />                                           activity vitro. J Immunol Methods. 1990; 131:165&#226;&#8364;&#8220;172. [PubMed: 2391427] <br />                                        4. Buttke TM, McCubrey JA, Owen TC. Use aqueous soluble tetrazolium/formazan assay NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           measure viability proliferation lymphokine-dependent cell lines. J Immunol Methods. 1993; <br />                                           157:233&#226;&#8364;&#8220;240. [PubMed: 8423368] <br />                                        5. Vries HI, Uyetake L, Lemstra W, Brunsting JF, Su TT, Kampinga HH, Sibon OC. Grp/DChk1                                           required G2-M checkpoint activation Drosophila S2 cells, Dmnk/DChk2                                           dispensable. J Cell Sci. 2005; 118:1833&#226;&#8364;&#8220;1842. [PubMed: 15860729] <br />                                        6. Goshima G, Vale RD. roles microtubule-based motor proteins mitosis: comprehensive <br />                                           RNAi analysis Drosophila S2 cell line. J Cell Biol. 2003; 162:1003&#226;&#8364;&#8220;1016. [PubMed: <br />                                           12975346] <br />                                        7. Dykxhoorn DM, Novina CD, Sharp PA. Killing messenger: short RNAs silence gene <br />                                           expression. Nat Rev Mol Cell Biol. 2003; 4:457&#226;&#8364;&#8220;467. [PubMed: 12778125] <br />                                        8. Clemens JC, Worby CA, Simonson-Leff N, Muda M, Maehama T, Hemmings BA, Dixon JE. Use <br />                                           double-stranded RNA interference Drosophila cell lines dissect signal transduction <br />                                           pathways. Proc Natl Acad Sci USA. 2000; 97:6499&#226;&#8364;&#8220;6503. [PubMed: 10823906] <br />                                        9. Clark KA, McKearin DM. Drosophila stonewall gene encodes putative transcription factor <br />                                           essential germ cell development. Development. 1996; 122:937&#226;&#8364;&#8220;950. [PubMed: 8631271] <br />                                        10. Goshima G, Wollman R, Goodwin SS, Zhang N, Scholey JM, Vale RD, Stuurman N. Genes <br />                                             required mitotic spindle assembly Drosophila S2 cells. Science. 2007; 316:417&#226;&#8364;&#8220;421. <br />                                             [PubMed: 17412918] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        11. Lents NH, Baldassare JJ. RNA interference takes flight: new RNAi screen reveals cell cycle <br />                                             regulators Drosophila cells. Trends Endocrinol Metab. 2006; 17:173&#226;&#8364;&#8220;174. [PubMed: 16723254] <br />                                        12. Radyuk SN, Klichko VI, Spinola B, Sohal RS, Orr WC. peroxiredoxin gene family                                             Drosophila melanogaster. Free Radic Biol Med. 2001; 31:1090&#226;&#8364;&#8220;1100. [PubMed: 11677042] <br />                                        13. Sibon OC, Stevenson VA, Theurkauf  DNA-replication checkpoint control Drosophila <br />                                             midblastula transition. Nature. 1997; 388:93&#226;&#8364;&#8220;97. [PubMed: 9214509] <br />                                        14. Sibon OC, Laurencon  Hawley R, Theurkauf  Drosophila ATM homologue Mei-41                                             essential checkpoint function midblastula transition. Curr Biol. 1999; 9:302&#226;&#8364;&#8220;312. <br />                                             [PubMed: 10209095] <br />                                        15. Radcliffe CM, Silva EA, Campbell SD. method assaying sensitivity Drosophila <br />                                             replication checkpoint mutants anti-cancer DNA-damaging drugs. Genome. 2002; 45:881&#226;&#8364;&#8220; <br />                                             889. [PubMed: 12416620] <br />                                        16. Brodsky MH, Weinert BT, Tsang G, Rong YS, McGinnis NM, Golic KG, Rio DC, Rubin GM. <br />                                             Drosophila melanogaster MNK/Chk2 p53 regulate multiple DNA repair apoptotic <br />                                             pathways following DNA damage. Mol Cell Biol. 2004; 24:1219&#226;&#8364;&#8220;1231. [PubMed: 14729967] <br />                                        17. Masrouha N, Yang L, Hijal S, Larochelle S, Suter B. Drosophila chk2 gene loki essential                                             embryonic DNA double-strand-break checkpoints induced S phase G2. Genetics. 2003; <br />                                             163:973&#226;&#8364;&#8220;982. [PubMed: 12663536] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        18. Jaklevic BR, Su TT. Relative contribution DNA repair, cell cycle checkpoints, cell death                                             survival DNA damage Drosophila larvae. Curr Biol. 2004; 14:23&#226;&#8364;&#8220;32. [PubMed: <br />                                             14711410] <br />                                        19. Takada S, Kelkar  Theurkauf  Drosophila checkpoint kinase 2 couples centrosome function <br />                                             spindle assembly genomic integrity. Cell. 2003; 113:87&#226;&#8364;&#8220;99. [PubMed: 12679037] <br />                                        20. Xu J, Xin S, Du W. Drosophila Chk2 required DNA damage-mediated cell cycle arrest                                             apoptosis. FEBS Lett. 2001; 508:394&#226;&#8364;&#8220;398. [PubMed: 11728459] <br />                                        21. Oishi  Sugiyama S, Otani H, Yamamura H, Nishida Y, Minami Y. (1998) novel Drosophila <br />                                             nuclear protein serine/threonine kinase expressed germline establishment. Mech <br />                                             Dev. 1998; 71:49&#226;&#8364;&#8220;63. [PubMed: 9507063] <br />  <br />  <br />  <br />  <br />                                               Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                         Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 1. Long-term survival assay <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Schematic representation survival assay. <br />  <br />  <br />  <br />  <br />                                              Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 2. Downregulation Grp/Dchk1 RNAi treatment <br />                                        Grp/Dchk1 protein levels downregulated using RNAi protein levels analyzed <br />                                        using polyclonal rabbit antibody specifically binding Grp/DChk1 protein (migrating <br />                                        58kD)(5) Supplementary Figure 1). 4 days RNAi treatment Grp/Dchk1 <br />                                        protein levels detection levels. long-term survival assays,                                        RNAi treated cells mixed feeder cells HU added 4 days                                        RNAI treatment. 4 days RNA treatment corresponds 0 days HU <br />                                        treatment. Grp/Dchk1 remain undetectable day 13 RNAi treatment. data <br />                                        indicate long-term survival assays Grp/Dchk1 efficiently downregulated <br />                                        RNAi treated cells untill 9 days HU treatment. &#206;&#179;-tubulin used loading <br />                                        control. &#226;&#710;&#8217; = presence mock RNAi treatment; + = presence Grp/Dchk1 <br />                                        RNAi constructs. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 3. Decreased long-term survival HU treatment Grp/Dchk1 downregulated cells <br />                                         Nuclear-luciferase GFP-expressing cells (NucLuc cells) mock transfected                                        transfected specific Grp/Dchk1 dsRNAi constructs, DmChk2 dsRNAi constructs                                        Grp/Dchk1 DmChk2 dsRNAi constructs 4 days start                                        experiment. NucLuc cells mixed non-GFP-expressing cells 1:1 ratio. <br />                                        percentage GFP-expressing cells different conditions measured time <br />                                        using FCM analysis (explained Figure 1) depicted y-axis (day 0 4 days <br />                                        start RNAi treatment, protein levels detection). <br />                                        B) H2B GFP-expressing cells (H2B cells) mock transfected transfected Grp/ <br />                                        Dchk1 dsRNAi constructs Stonewall dsRNAi constructs  control) 4 days                                        start experiment. H2B GFP-expressing cells mixed non-GFP- <br />                                        expressing cells 1:1 ratio. percentage GFP-expressing cells different <br />                                        conditions measured time using FCM analysis depicted y-axis (day <br />                                        0 4 days start RNAi treatment, protein levels detection). <br />                                        statistical analysis supplementary file. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                        Page 12 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 4. Downregulation DmChk2 RNAi treatment <br />                                        DmChk2 protein levels downregulated using RNAi protein levels analyzed <br />                                        using polyclonal rabbit anti-Dmnk-L antibody (21) recognizing DmChk2 (54 kD). 4 days <br />                                        RNAi treatment DmChk2 protein levels detection levels.                                        long-term survival assays, RNAi treated cells mixed feeder cells HU <br />                                        added 4 days RNAi treatment. 4 days RNA treatment corresponds <br />                                        0 days HU treatment. DmChk2 remain undetectable day 10                                        RNAi treatment. data indicate long-term survival assays DmChk2                                        efficiently downregulated RNAi treated cells untill 6 days HU treatment. &#206;&#179;-tubulin <br />                                        used loading control. &#226;&#710;&#8217; = presence mock RNAi treatment; + =                                        presence Grp/Dchk1 RNAi constructs. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Cytometry  Author manuscript; available PMC 2009 July 1. <br />                             Yi et al.                                                                                         Page 13 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                        Figure 5. Downregulation Grp/Dchk1 DmChk2 RNAi treatment <br />                                        DmChk2 protein (54kD) levels Grp/Dchk1 (58kD) protei <br /> </body></html>