<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Sept. 1997, p. 5146&#226;&#8364;&#8220;5155                                                                                  Vol. 17,  9 <br /> 0270-7306/97/$04.0010 <br /> Copyright &#194;&#169; 1997, American Society Microbiology <br />  <br />  <br />  <br />            SQT1, Encodes Essential WD Domain Protein             Saccharomyces cerevisiae, Suppresses Dominant-Negative <br />                Mutations Ribosomal Protein Gene QSR1 <br />           DOMINIC P. EISINGER,&#226;&#8364;&#160; FREDERICK  DICK, ELKE DENKE,                                   BERNARD L. TRUMPOWER* <br />                      Department Biochemistry, Dartmouth Medical School, Hanover, New Hampshire 03755 <br />                          Received 14 January 1997/Returned modification 12 March 1997/Accepted 3 June 1997 <br />  <br />              QSR1 essential Saccharomyces cerevisiae gene, encodes 60S ribosomal subunit protein required <br />           joining 40S 60S subunits. Truncations QSR1 predicted encode C-terminally truncated forms           Qsr1p substitute QSR1 act dominant negative mutations, inhibiting growth yeast <br />           expressed inducible promoter. dominant negative mutants exhibit polysome profile <br />           characterized &#226;&#8364;&#732;half-mer&#226;&#8364;&#8482; polysomes, indicative subunit joining defect like seen qsr1 mutants <br />           (D. P. Eisinger, F.  Dick, B. L. Trumpower, Mol. Cell. Biol. 17:5136&#226;&#8364;&#8220;5145, 1997.) screening high-copy <br />           yeast genomic library, isolated clones containing overlapping inserts novel gene rescues           slow-growth phenotype dominant negative qsr1 truncations. suppressor qsr1 truncation mutants, <br />           SQT1, essential gene, encodes 47.1-kDa protein containing multiple WD repeats           interacts strongly Qsr1p yeast hybrid  SQT1 restores growth &#226;&#8364;&#339;half-mer&#226;&#8364;? polysome <br />           profile dominant negative qsr1 mutants normal, does rescue temperature-sensitive qsr1 <br />           mutants original qsr1-1 missense allele. yeast cell lysates, Sqt1p fractionates oligomeric <br />           protein complex loosely associated ribosomes distinct known eukaryotic initiation <br />           factor complexes. Loss SQT1 function regulation inducible promoter results formation <br />           half-mer polyribosomes decreased Qsr1p levels free 60S subunits. Sqt1p appears involved <br />           late step 60S subunit assembly modification cytoplasm. <br />  <br />  <br />    functions individual eukaryotic ribosomal proteins                 physically interact Qsr1p. SQT1 encodes cytosolic pro- understood. undertaken biochemical                    tein appears loosely associated ribosomes genetic approach elucidate function recently               involved late stage large ribosomal subunit assembly discovered ribosomal protein, Qsr1p, Saccharomyces cerevi-                maturation. <br /> siae (13). Qsr1p eukaryotic 60S ribosomal protein encoded essential single-copy S. cerevisiae gene QSR1 (13, 54)                                        MATERIALS METHODS mammalian homolog rat ribosomal protein L10                       Construction plasmids. Plasmid pDEGQ2 (CEN URA3 GALUAS::QSR1), <br /> (9).                                                                         GAL1-10 upstream activating sequence (UAS) modulating    Characterization Qsr1p temperature-sensitive qsr1               expression QSR1 minimal promoter (14), used construct mutants revealed Qsr1p required joining 40S               C-terminal truncations QSR1-encoded protein follows. pDEGQ2                                                                              digested HindIII remove internal 474-bp fragment QSR1 open 60S subunits active 80S ribosomes inactive                reading frame encoding amino acids 64 221, immediately preceding stop <br /> 80S couples (14). Mutant alleles temperature                 codon. Self-ligation remaining plasmid backbone generated pDEGQ64, sensitive conditional synthesis Qsr1p 60S                 N-terminal 64 amino acids QSR1-encoded protein retained. <br /> subunits devoid Qsr1p unable join                truncation referred qsr1-64D. Constructs pDEGQ133                                                                              pDEGQ187 ligating HindIII-digested QSR1 PCR fragments 40S subunits. conditional mutants 60S ribo-                  HindIII pDEGQ2 backbone. fragment making pDEGQ133 somal protein genes defective biogenesis 60S                   synthesized PCR primers DEQ133HIII (59-GCGTAAGCTTGACCAA <br /> subunits (11, 12, 35, 38, 39, 45, 47, 48), precludes insights          TGTCGACA [coding strand]) FDYGST59 (59-AACGGATCCAAGATGG postassembly functions, biogenesis 60S subunits              CTAGAAGACC [noncoding strand]). resulting construct N-termi- <br />                                                                              nal 133 amino acids encoded open reading frame referred slightly impaired qsr1 mutants (14). investi-             qsr1-133D. PCR fragment making pDEGQ187 synthesized gation Qsr1p likely lead better understanding             primers DEQ187HIII (59-CCTCACAAGCTTCTCTTC [coding strand]) late stages translation initiation ribosomal subunit            FDYGST59 (59-AACGGATCCAAGATGGCTAGAAGACC [noncoding strand]). <br /> joining, little currently known.                                 resulting construct N-terminal 187 amino acids encoded QSR1 <br />                                                                              open reading frame referred qsr1-187D. <br />    learn function Qsr1p, constructed                    1.8-kb PCR fragment encompassing SQT1 open reading frame dominant-negative qsr1 mutants isolated novel essential                High Fidelity Taq polymerase (Boehringer Mannheim) gene (SQT1) high-copy suppressor dominant-neg-                   primers DESQTP3 (59-CAAGGATCCCCATTCAAACCCCATC [coding strand]) <br /> ative slow-growth phenotype. dominant-negative mutants                   DESQTP1 (59-AGAGAAGGATCCCCAAAG [noncoding strand])                                                                              contain BamHI sites cloning. PCR fragment 277 bp 39-flanking defect ribosomal subunit joining rescued                sequence 258 bp 59-flanking sequence cut BamHI cloned <br /> SQT1. directed hybrid analysis, Sqt1p                BamHI sites pFL36 pFL46S (5) generate pDESQT2 (SQT1 <br />                                                                              CEN LEU2) pDESQT1 (SQT1 2mm LEU2). <br />                                                                                 make myc-tagged version SQT1-encoded protein, primers                                                                              synthesized long-inverse PCR (15) myc epitope recognized    * Corresponding author. Mailing address: Department Biochem- <br />                                                                              monoclonal antibody 9E10 (31) added C terminus Sqt1p. <br /> istry, Dartmouth Medical School, Hanover, NH 03755. Phone: (603)             Primer DSQTM1 (59-AAGCTTTTGTTCACCACCGTTTGGTACCTCGAAC <br /> 650-1621. Fax: (603) 650-1389. E-mail: Trumpower@Dartmouth.edu.              AC [noncoding strand]) primer DSQTM2 (59-ATTTCTGAAGAAGACT <br />    &#226;&#8364;&#160; Present address: Argonex , c/o W. Alton Jones Cell Science          TGTGATAGATAGTCGCGTATACATA [coding strand]) add myc epitope <br /> Center, Lake Placid, NY 12946.                                               (EQKLISEEDL) preceded glycines C-terminal asparagine <br />  <br />                                                                       5146 <br />  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                             5147 <br />  <br />  <br />                                                          TABLE 1. Yeast strains used study <br />      Strain                                                                       Genotype                                                                     Source <br />  <br /> W303-1A                    ade2-1 his3-11,15 trp1-1 leu2-3,112 ura3-1 can1-100                                                                             54 <br /> W303-1B                    ade2-1 his3-11,15 trp1-1 leu2-3,112 ura3-1 can1-100                                                                             54 <br /> HFF10                      W303 markers, QSR1/qsr1D1::HIS3                                                                                               54 <br /> DEH64                      W303 markers, QSR1, 1 pDEGQ64 (GAL1-10UAS-qsr1-64D CEN URA3)                                                                      study <br /> DEH133                     W303 markers, QSR1, 1 pDEGQ133 (GAL1-10UAS-qsr1-133D CEN URA3)                                                                    study <br /> DEH187                     W303 markers, QSR1, 1 pDEGQ187 (GAL1-10UAS-qsr1-187D CEN URA3)                                                                    study <br /> DEH2212                    W303 markers, QSR1, 1 pDEGQ2 (GAL1-10UAS-QSR1 CEN URA3)                                                                           study <br /> DESQT1B                    DEH64, 1 pSQT1B (SQT1 2mm LEU2)                                                                                                   study <br /> DEQSR4                     DEH64, 1 pQSR-4 (QSR1 2mm LEU2)                                                                                                   study <br /> YPH500 X 499               ura3-52 lys2-801 ade2-101 trp1-D63 his3-D200 leu2-D1                                                                          49 <br /> DPE1                       YPH markers, SQT1/sqt1D1::HIS3                                                                                                  study <br /> DPESM1                     DPE1 1 pSQTMYC1 (SQT1-myc 2mm LEU2)                                                                                             study <br /> DPESM2-1A                  YPH markers, sqt1D1::HIS3 1 pSQTMYC2 (SQT1-myc CEN LEU2)                                                                          study <br /> FAD13-2B                   YPH markers, sqt1D1::HIS3 1 pFAD30 (GAL-SQT1-myc CEN URA3)                                                                        study <br /> EGY48                      atrp1 ura3-52 his3 leu2 LEU2::pLexAop6-Leu2                                                                                       20 <br /> Y190                       aleu2-3,112 ura3-52 trp1-901 his3-D200 ade2-101 gal4Dgal80D (URA3GAL-lacZ LYSGAL-HIS3 Cyhr)                                       22 <br />  <br />  <br />  stop codon SQT1. pDESQT1 pDESQT2 used templates                plating density estimated 15,000 transformants 10-cm petri addition myc tag, creating 2mm (pSQTMYC1) CEN (pSQT-                   dish. 4 days 30&#194;&#176;C, 15 large colonies restreaked fresh plates. Total <br /> MYC2) versions, respectively, myc-tagged SQT1. Primers FDSQTP1                     yeast DNA purified transformed E. coli, purified plasmids (59-GGATCCCCAAAGAACTTACC [coding strand]) FDSQTP2 (59-GG                          retransformed DEH64 link rescue growth inhibition caused ATCCGCGTAATGGAACCTCAAG [noncoding strand]), contain                        galactose-induced qsr1-64D library-derived plasmid. clones rescued <br /> BamHI sites, used amplify SQT1-myc plasmid pSQTMYC2, using               growth defect analyzed restriction mapping partial se- High Fidelity Taq polymerase (Boehringer Mannheim). resulting 1.6-kb          quencing. Deletions chromosome 9 genomic clone 12 long- <br /> fragment ligated pCR2.1 TA cloning kit (Invitrogen).            inverse PCR (15) follows. 12.6-kb pSQT12A blunt-end <br /> 1.6-kb insert excised BamHI ligated unique BamHI site        self-ligation PCR product vector primers YEPB3 (59-GGAGGA <br /> pBM258. resulting plasmid, contains SQT1-MYC control           TCCACAGGACGGGTGTGGTC) DESQTP1  , 11-kb GAL1-10 promoter, named pFAD30. plasmids amplified            pSQT12B primers YEPB3 DESQTP2 (59-CTAGGATCCT Escherichia coli DH5a (21), DNA constructs verified sequenc-       GCTCAACTTCTTCGG [noncoding strand]). <br /> ing Applied Biosystems  373 DNA sequencer.                                    Polysome ribosome subunit analysis. Ribosomal subunits, 80S ribosomes, <br />    S. cerevisiae strains growth conditions. Yeast cells grown yeast       polysomes fractionated sucrose velocity gradients (14). cells <br /> extract-peptone-dextrose (YPD), synthetic complete (SC), synthetic dropout         treated cycloheximide arrest translating ribosomes, gradi- <br /> (SD) medium relevant nutritional supplements omitted.               ents run Mg cycloheximide. dissociate ribosomal subunits, indicated, galactose substituted dextrose carbon source.         cells collected cycloheximide treatment, Mg cycloheximide <br /> lithium acetate procedure used transform yeast cells (1). yeast        omitted solutions. <br /> strains used listed Table 1 constructed described  Haploid           Directed hybrid interaction analysis SQT1 QSR1. yeast strain <br /> strains wild type chromosomal QSR1 harboring galactose-inducible C-           EGY48, LexA fusion plasmid pEG202, positive control LexA-GAL4 <br /> terminal qsr1 truncation mutants separately transforming             plasmid, LexAop-lacZ reporter plasmid pSH18-34 provided centromeric plasmid pDEGQ64, pDEGQ133, pDEGQ187 HFF10,                      Roger Brent (1, 20). Strain Y190 (22) gift Steve Elledge. GAL4 <br /> hemizygous QSR1 diploid, isolating Ura1 His2 spores generate strains           DNA binding domain vector pAS2-1 GAL4 activation domain vector <br /> DEH64, DEH133, DEH187, respectively. DEH2212 isolated                  pACT2-1 Clontech Laboratories,  pEG202-QSR1, pAS2-1-QSR1, manner length galactose-inducible wild-type QSR1 gene.            pACT2-QSR1 contain BamHI-BglII PCR fragment encoding  <br /> DESQT1B derived DEH64 additional plasmid (pSQT1B)                  length QSR1 protein. fragment generated PCR amplification contains SQT1 locus isolated high-copy genomic             pMY7 (54) primers HFF1 (59-CTGGATCCAGATGGCTCAAGACC) library. DESQR4 derived DEH64 plasmid (pDEQSR4)                HFF2 (59-GAAAAGATCTTAAGCTTGAGCAGC [noncoding strand]) cre- contains QSR1 locus isolated genomic library.              ate BamHI BglII sites, respectively, underlined primer <br />    null allele SQT1 isogenic diploid YPH499 3 YPH500          sequences. BamHI-BglII fragment ligated BamHI sites targeted deletion SQT1 open reading frame (3). primers,          pEG202, pAS2-1, pACT2. construct pEG202-SQT1, pAS2-1-SQT1, 36 bp (34) SQT1 sequence flanking HIS3 gene sequence, used amplify         pACT2-SQT1, PCR fragment generated plasmid pSQT1B prim- HIS3 gene pRS303 (49). Primer SQT1HIS1 (59-CAAGACGTTCCC                      ers DESQTP5 (59-CAAGGATCCCTCAAGAAGAGTTTATAAC) DESQ <br /> GTTGATATTGAAGGAGAGAATGACGGCCTCCTCTAGTACACTC [cod-                                     TP6 (59-CTTGGATCCTTGGTACCTCGA.ACACCA [noncoding strand]) ing strand]) contains nucleotides 73 108 SQT1 fused 59 HIS3 se-          create BamHI sites, fragment cloned pEG202, pAS2-1, <br /> quence (shown italics). Primer SQT1HIS2 (59-TCAGTTTGGTACCTCGAA                     pACT2. open reading frames proteins translational <br /> CACCAGAGATACACCTTCGCGCGCCTCGTTCAGAATG [noncoding                                      phase DNA binding activation domains vectors. accuracy <br /> strand]) contains nucleotides 1296 1261 SQT1 fused 39 HIS3 se-           reading frames verified sequencing junctions. <br /> quence (shown italics). amplified fragment transformed directly          Cells prepared exponentially growing cultures. monitor inter- <br /> strain YPH499 3 YPH500 (49), His1 transformants selected                 action fusion proteins Y190, 50 mM 3-amino-1,2,4-triazole added <br /> screened colony PCR (3) generate strain DPE1. Proper integration         dropout plates inhibit low levels leaky HIS3 expression HIS3 gene SQT1 locus confirmed PCR genotyping                 reporter strain. Colony color b-galactosidase assays performed di- <br /> predicted unique fragments generated internal HIS3 primers exter-         rectly plates containing 5-bromo-4-chloro-3-indolyl-b-D-galactopyranoside <br /> nal SQT1 primers.                                                                     (X-Gal) (1) colony lift filter assay (Clontech). b-Galactosidase activity    Strain DPESM2-1A, plasmid pSQTMYC2 (SQT1-MYC CEN LEU2)                    solution assayed o-nitrophenyl-b-D-galactopyranoside method (Clon- <br /> contains SQT1-MYC tag allele covering chromosomal SQT1 deletion               tech) cells permeabilized freezing liquid nitrogen thawing <br /> (sqt1D1::HIS3), isolating Leu1 His1 spore derived diploid          37&#194;&#176;C. b-Galactosidase activity expressed units described Miller (37). <br /> strain DPE1 transformed pSQTMYC2. Strain DPESM1 diploid strain                   Cell fractionation immunoblot analysis proteins. Yeast cells DPE1 plasmid pSQTMYC1 (SQT1-MYC 2mm LEU2). DPE1 trans-                       recovered 25-ml cultures centrifugation absorbance 600 nm <br /> formed pFAD30 sporulated. spores dissected YPGal                 1.0. cells washed 1.5 ml 20 mM Tris (pH 7.5)&#226;&#8364;&#8220;50 mM <br /> plates, spore carrying sqt1D1::HIS3 allele complemented GAL1-        NaCl&#226;&#8364;&#8220;0.4 M sorbitol (lysis buffer); resuspended 250 ml lysis buffer plus 10 <br /> 10-controlled SQT1-MYC identified named FAD13-2B.                             mM leupeptin, 1 mg pepstatin ml, 1 mM diisopropyl fluorophos- <br />    Isolation high-copy suppressor dominant-negative qsr1 truncation.        phate; glass beads added 50% total volume. cells Strain DEH64 grown SD medium lacking <span id='am-41' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span>, transformed                broken vortexing 5 min 4&#194;&#176;C, lysates cleared centrifu- <br /> LEU2-marked YEp13 yeast genomic library (43), allowed recover YPD           gation 500 3 g 5 min 4&#194;&#176;C. 30-ml volume supernatant kept <br /> medium 1 h 30&#194;&#176;C plating SD galactose plates lacking uracil       cell lysate. remainder centrifuged 14,000 rpm 20 min, leucine. transformation efficiency 1.2 3 105 mg library plasmid,       supernatant retained crude cytosolic fraction. pellet washed <br />  5148        EISINGER ET AL.                                                                                                                           MOL. CELL. BIOL. <br />  <br />  resuspended 200 ml lysis buffer recentrifuged, super- <br /> natant discarded. crude membrane pellet resuspended 30 ml lysis buffer. Protein concentrations determined dye-binding assay (7). <br /> Protein fractions denatured electrophoresis incubation sodium <br /> dodecyl sulfate-polyacrylamide gel electrophoresis (SDS-PAGE) sample buffer <br /> (32) 55&#194;&#176;C 15 min. <br />    Fractionation yeast cells isolate nuclei carried essentially described Wise (57). proteins resolved 10% SDS-PAGE gels (1), <br /> unless stated, analyzed Western blotting (13). electroblotted nitrocellulose, membrane blocked 5% nonfat dry <br /> milk 20 mM Tris (pH 7.5)&#226;&#8364;&#8220;150 mM NaCl&#226;&#8364;&#8220;0.3% Tween 20 (TTBS) 1 h incubated 1 h primary antibody diluted 1:2,000 1% milk TTBS. primary antibodies used rabbit polyclonal anti-Qsr1p (13), <br /> rabbit polyclonal anti-Prt1p (10), rabbit polyclonal anti-Nop1p (52), mouse <br /> monoclonal anti-Myc (31). blots washed TTBS, incubated horseradish peroxidase-conjugated goat anti-rabbit antibodies 1:2,500 dilution horse anti-mouse antibodies 1:10,000 dilution 1% milk TTBS 1 h, washed  visualized Lumiglo <br /> chemiluminescent (Kirkegaard <span id='am-19' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; Perry, Gaithersburg, Md.) exposure Hyperfilm-ECL (Amersham, Arlington Heights, Ill.). <br />    Gel filtration Sqt1p salt-washed ribosomes. Ribosomes purified modification procedure used isolate translation initiation factors (10). <br /> Strain DPESM2-1A (16 liters) grown YPD absorbance 600 nm 1.2, cells collected centrifugation. cells washed water 20 mM Tris (pH 7.5)&#226;&#8364;&#8220;10 mM MgSO4&#226;&#8364;&#8220;100 mM KCl&#226;&#8364;&#8220;1 mM <br /> dithiothreitol (ribosome lysis buffer), resuspended 20 ml ribosome lysis <br /> buffer, frozen lysed liquid N2 Waring blender (1). <br />    broken cells thawed ice, protease inhibitors added described  lysate cleared sequential centrifugation 5,000 3 <br /> g 15 min 22,000 3 g 15 min Sorvall SS34 rotor, ribosomes pelleted centrifugation 200,000 3 g 2 h Beckman <br /> Ti60 rotor. postribosomal supernatant collected stored 270&#194;&#176;C. ribosomal pellet resuspended 15 ml 20 mM Tris (pH 7.5)&#226;&#8364;&#8220;10 mM                 FIG. 1.   C-terminal truncation mutations 221-amino-acid Qsr1p <br /> MgSO4&#226;&#8364;&#8220;500 mM KCl&#226;&#8364;&#8220;1 mM dithiothreitol (ribosome wash buffer) plus protease               constructed making frame deletions fused residue <br /> inhibitors centrifuged 2 h 200,000 3 g. supernatant (ribo-         stop codon QSR1. (B) Immunoblot analysis dominant-negative qsr1 <br /> somal salt wash) dialyzed overnight 2 liters ribosome lysis buffer       truncation strains anti-Qsr1p antibodies. strains DEH64 (GALUAS- frozen 270&#194;&#176;C use.                                                  qsr1-64D), DEH133 (GALUAS-qsr1-133D), DEH187 (GALUAS-qsr1-187D), <br />    2-mg sample ribosomal salt wash post ribosomal supernatant           DEH221-(GALUAS-QSR1), W303, DESQT1B (DEH64 rescued SQT1 loaded Superose 6 gel filtration column Pharmacia fast protein          expressed 2mm plasmid). Western blots lysates cells grown liquid chromatography equilibrated ribosome lysis buffer.             galactose dextrose shown lanes 1 6 lanes 7 12, respectively. <br /> 0.5-ml fractions collected analyzed Western blotting. column calibrated thyroglobulin (670 kDa), b-galactosidase (540 kDa), gamma <br /> globulin (158 kDa), ovalbumin (44 kDa), myoglobin (17 kDa), <span id='am-14' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-15' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>vitamin</span> B12 <br /> (1.3 kDa). <br />    <span id='am-26' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. SQT1 nucleic acid sequence             culture, doubling times increased 1.5 3.5 h, assigned GenBank accession  U75717.                                             cellular morphology appeared normal (data shown). <br />                                                                                            effort quantitate relative amounts Qsr1p                                    RESULTS                                              truncated Qsr1 proteins, haploid QSR1 strains carrying                                                                                         various dominant-negative alleles plasmids grown    Isolation dominant-negative QSR1 mutants. investi-                             galactose dextrose analyzed Western blotting. gate QSR1-encoded protein essential,                           blot Fig. 1B overexposed trun- <br /> C-terminal truncations Qsr1p constructed deleting                            cated proteins, predicted 21.6, 15.3, 7.5 kDa, portions QSR1 (Fig. 1) placing resulting constructs                          detected induced growth galactose. Furthermore, control GAL1-10 upstream activating se-                                endogenous Qsr1p levels remain unchanged quence (UAS). truncated peptides 64 (encoded                                 extra wild-type copy galactose-inducible QSR1 present. <br /> GALUAS-qsr1-64D), 133 (encoded GALUAS-qsr1-133D),                                apparently invariant Qsr1p consistent 187 (encoded GALUAS-qsr1-187D) residues length com-                               previous findings unassembled ribosomal proteins pared 221 residues (encoded GALUAS-QSR1)                                   short half-lives degraded rapidly (56). length Qsr1p. plasmids transformed                            antibodies detect truncated forms Qsr1p, <br /> HFF10, strain hemizygous QSR1 (54), tetrads                              shortest (qsr1-64Dp) longest (qsr1-187Dp) truncated <br /> dissected sporulated diploid. deletion                             proteins apparently present sufficient quantities exert <br /> alleles cover QSR1 chromosomal deletion,                            dominant-negative effect. failure antibodies length GALUAS-QSR1 allele covered deletion strain                              detect truncated proteins antibodies grown galactose dextrose (data shown).                           recognize C-terminal portion Qsr1p deleted <br /> Examination spores failed grow showed did                          constructs. conserved region divide  indicating nonconserved C-                               protein (15). <br /> terminal 34 residues (16) essential QSR1 function.                                 Isolation characterization SQT1. exploited    noticed diploids haploids harboring long-                             dominant-negative effect Qsr1p truncations growth <br /> est deletion (GALUAS-qsr1-64D) shortest deletion                                search high-copy suppressors genetic screen. Strain <br /> (GALUAS-qsr1-187D) grew slower carrying                                 DEH64 (QSR1 pDEGQ64[GALUAS-qsr1-64D CEN URA3]) length QSR1 spread galactose plates. domi-                             transformed LEU2-marked high-copy genomic li- <br /> nant-negative slow growth phenotype stable                               brary, fast-growing colonies picked. Screening 105 <br /> multiple switches galactose dextrose media                             transformants yielded 16 large colonies. passage occurrence spontaneous revertants, growth                            E. coli, library plasmids recovered fast- <br /> defect evident temperatures 16 37&#194;&#176;C. liquid                           growing colonies suppressed slow-growth phenotype  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                    5149 <br />  <br />  <br />  <br />  <br />    FIG. 2. Rescue dominant-negative growth defect SQT1.   Map SQT1 locus genomic clones carrying SQT1. figure shows open reading frames restriction map vicinity SQT1 long <br /> arm chromosome IX. SQT1 open reading frame separated              FIG. 3.   deduced <span id='am-34' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> sequence Sqt1p sequences <br /> divergently transcribed gene DBF8 (27) 242 bp. Clones 1B, 8, 15, 12   WD repeats underlined. middle repeat (residues 135 183) <br /> isolated complementing slow-growth phenotype qsr1-64D mutant        conforms consensus sequence Neer et al. (44), insertion <br /> strain DEH64. Clones 12A 12B subclones clone 12. ability    10 residues instead 3 initial VH. (B) Tetrad analysis clones rescue slow-growth phenotype strain DEH64 indicated    hemizygous sqt1D1 strain DPE1 showing SQT1 null allele (sqt1D1) <br /> right. restriction map given HindIII (H), EcoRI (E), BamHI (B),      segregates 2:2 lethality. <br /> XhoI (X). (B) Effect qsr1 truncations yeast growth. strains DEH64 <br /> (GALUAS-qsr1-64D), DEH133 (GALUAS-qsr1-133D), DEH187 (GALUAS-qsr1- <br /> 187D), DEH2212 (GALUAS-QSR1), DESQT1B (DEH64 rescued SQT1 <br /> expressed 2mm plasmid), DESQT4 (DEH64 rescued QSR1 <br /> expressed 2mm plasmid).                                                    minimal 1.8-kb fragment encompassing SQT1 <br />                                                                                   open reading frame cloned centromeric vector,                                                                                   suppressed growth defect galactose (data retransformed strain DEH64. Restriction digestion anal-                      shown), suggesting relationship SQT1 QSR1 ysis partial sequencing genomic inserts revealed                  specific, weaker nonspecific interaction plasmids carried identical overlapping                  probably require overexpression typical high-copy plas- <br /> clones QSR1 remaining carried overlapping                         mid. <br /> clones novel DNA fragment mapped region                            SQT1 open reading frame encodes predicted protein long arm chromosome IX (54a). genomic                             431 amino acids molecular mass 47.1 kDa chromosome IX clones subclones depicted Fig.                       calculated pI 4.16 (Fig. 3A). Inspection deduced <br /> 2A single open reading frame                         amino acid sequence search databases reveals named SQT1 (suppressor QSR1 truncations) suppresses                        SQT1 encodes novel protein sequence similarity slow-growth phenotype caused dominant-negative                             b-transducin WD repeats (44). deduced Sqt1p sequence <br /> QSR1 truncations presence chromosomal copy                         includes  possibly  consensus WD  <br /> QSR1. Examination DNA sequence surrounding                             peats (Fig. 3A). <br /> open reading frame reveals TATA-like                          Originally identified b-subunit heterotrimeric <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span>GTP</span>- <br /> elements (consensus TATAA) region 50 120 bp                          binding proteins, WD-repeat motifs present family stream open reading frame (51) putative poly-                        conserved eukaryotic proteins common <br /> adenylation signal (55) centered position 1349.                                function known members regulatory proteins <br />    dominant-negative effect growth GALUAS-                          enzymes (44). Searches nucleic acid qsr1-64D GALUAS-qsr1-187D alleles rescue                           protein data banks revealed significant homologies Sqt1p <br /> GALUAS-qsr1-64D allele high-copy QSR1 SQT1                             limited proteins major WD repeats. <br /> shown Fig. 2B. SQT1 allele-specific suppressor                  homologous proteins include known WD proteins does rescue lethality QSR1 chromosomal dele-                       Tup1p, involved gene regulation (30); Ret1p, tion, does suppress temperature-sensitive qsr1 mutants                     coatamer protein involved vesicular traffic (44); AAMP, original qsr1-1 missense mutation (data shown).                        mammalian mediator heparin-sensitive cell adhesion (4). <br />  5150     EISINGER ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />    SQT1 essential gene. hemizygous SQT1/sqt1D1:: <br /> HIS3 diploid strain (DPE1) created replacing chro- <br /> mosomal copy SQT1 HIS3 gene. Sporulation diploid dissection tetrads yielded viable spores diploid, shown Fig. 3B, viable spores His2 (data shown), indicating SQT1 essen- <br /> tial gene. Microscopic examination spores failed grow revealed germinated failed progress  cell stage. 1.8-kb SQT1 complement- <br /> ing fragment centromeric plasmid (pDESQT2) trans- <br /> formed strain DPE1 rescued lethality frequency <br /> compatible segregation centromeric plasmid hemizygous diploid sporulated (data shown). <br />    SQT1 rescues defect 40S 60S ribosomal-subunit join- <br /> ing caused dominant-negative qsr1-64D mutant. previous studies shown QSR1 essential ability 60S subunits dock 40S subunits form 80S <br /> ribosomes (14), investigated effect dominant- <br /> negative qsr1-64D allele distribution ribosomal sub- <br /> units polysomes. Figure 4 shows polysome profiles qsr1-64D strain repressed dextrose, induced galactose, rescued SQT1. qsr1-64D strain grown dextrose polysome profile typical wild-type cells large 80S peak polysome peaks gradient. dominant-negative qsr1-64D protein induced galactose, 80S polysome peak sizes dramatically reduced, half-mer polysomes seen heavier 80S peak smaller polysomes (Fig. 4B). polysome profile restored dextrose  <br /> dition SQT1 present centromeric plasmid (Fig. <br /> 4C). <br />    Half-mer polysomes result 60S-40S subunit-joining <br /> defect stalled 43S preinitiation complexes mRNA template containing translating 80S ribo- joined 60S subunit (23, 47). defect assembly 60S subunits (stoichiometric imbalance) subunit joining defect leads half-mers. reported half- <br /> mers occur mutants display depleted mutated 60S <br /> ribosomal proteins defective assembling 60S <br /> subunits (11, 12, 35, 38, 39, 45, 47, 48). previously            FIG. 4. Changes polysome profiles resulting expression shown temperature-sensitive mutations QSR1 deple-          dominant-negative qsr1-64D allele rescue SQT1. Cells pretreated <br />                                                                       cycloheximide, lysates fractionated 5 47% sucrose gradients <br /> tion Qsr1p regulation inducible promoter             containing Mg cycloheximide. 40S, 60S, 80S subunit peaks results population 60S subunits lacking Qsr1p        marked absorbance profile. arrows panel B mark location defective subunit joining opposed deficiency 60S        half-mer polysomes.   DEH64, GALUAS-qsr1-64D grown dextrose; (B) <br /> subunits (14). degree half-mer formation greater         DEH64, GALUAS-qsr1-64D grown galactose; (C) DESQT1B, DEH64, <br />                                                                       GALUAS-qsr1-64D grown galactose rescued SQT1 expressed temperature-sensitive strains strain expressing       2mm plasmid. induced dominant-negative qsr1-64D allele. <br />    investigate dominant-negative qsr1-64D mu- <br /> tation causes subunit imbalance, examined ratio 40S 60S subunits running polysome gradients lack           binding protein pEG202 (20) GAL4 activation <br /> Mg, causes 80S ribosomes dissociate (14, 18, 46).         domain pACT2 (17). Combinations different bait shown Table 2, small significant increase    prey plasmids cotransformed LexA-lacZ <br /> ratio 40S 60S subunits qsr1-64D allele induced      reporter plasmid pSH18-34 EGY48 reporter strain, galactose, ratio restored essentially    transformants identified growth selective  <br /> dextrose condition high-copy SQT1 present.           dium. subunit imbalance slight, difficult say      Interaction fusion proteins monitored half-mers result subunit imbalance caused trun-         streaking colonies X-Gal medium performing <br /> cated Qsr1p truncated Qsr1p inhibits subunit           b-galactosidase assays. shown Fig. 5, LexA- <br /> joining mechanisms discussed                      QSR1 fusion LexA-SQT1 fusion coexpressed    SQT1 QSR1 interact hybrid assay. test          GAL4 activation domain  cells showed bas- <br /> interactions Qsr1p Sqt1p, used different           al-level b-galactosidase activity, observed bait vectors expressing fusion proteins LexA          LexA protein expressed GAL4 acti- <br /> DNA-binding protein GAL4 DNA binding domain.                vation domain. basal-level activity corresponded white <br /> express second hybrid, fused open reading frames           colonies X-Gal plates demonstrates LexA <br /> encoding proteins GAL4 activation domain.             hybrids Qsr1p Sqt1p interact GAL4 length Qsr1p Sqt1p fused LexA DNA                portion prey hybrid induce b-galactosidase activity <br />  VOL. 17, 1997                                                            SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                        5151 <br />  <br />  <br /> TABLE 2. Ratio 40S 60S subunits wild-type cells, qsr1-64D <br />        mutants, qsr1-64D mutants rescued SQT1 <br />                                               Medium            Mean 40S 60S <br />                Strain <br />                                             supplement           ratioa 6 SD <br />  <br /> W303-1B                                     Dextrose           0.470 6 0.04 <br /> DEH64 (GALUAS-qsr1-64D)                     Dextrose           0.419 6 0.016 <br /> DEH64 (GALUAS-qsr1-64D)                     Galactose          0.534 6 0.037 <br /> DESQT1B (GALUAS-qsr1-64D,                   Galactose          0.464 6 0.008 <br />  12mm SQT1) <br />       Ratios 40S 60S subunits calculated determining correspond- <br /> ing areas 254-nm absorbance profiles. <br />  <br />  <br />   basal level reporter gene activity observed GAL4-QSR1 GAL4-SQT1 prey plas- <br /> mids coexpressed LexA protein. GAL4-SQT1 prey plasmid introduced cells bearing LexA-QSR1 fusion construct, resulted b-galactosidase <br /> activity approximately 100-fold higher basal levels (Fig. <br /> 5). Coexpression LexA-SQT1 GAL4-QSR1 prey plas- <br /> mids resulted 25-fold increase b-galactosidase activity. cases, colonies blue X-Gal medium. <br />   Similar results obtained QSR1 SQT1 fused GAL4 DNA binding domain bait vector                                 FIG. 6. Sqt1p cytosolic protein. Protein fractions strains expressing <br /> pAS2-1 (data shown). Coexpression strain Y190 ei-                          myc-tagged Sqt1p visualized immunoblotting detection ther QSR1 SQT1 bait fusion GAL4                         9E10 mouse monoclonal anti-myc antibody.   cell lysates (W), cytosol <br /> activation domain did cause expression HIS3                         (C), membrane (M) fractions indicated yeast strains. Western <br />                                                                                      blot probed anti-myc antibodies stripped reprobed lacZ reporter genes. QSR1 bait fusion trans-                         anti-Qsr1p antibodies. Lanes: 1, parent strain DPE1, Sqt1p formed yeast cells bearing SQT1 prey fusion, cells                      myc tagged; 2, strain DPESM1 (SQT1/sqt1D1) expressing Sqt1-myc protein <br /> grew minimal medium histidine formed blue                             2mm plasmid; 3, strain DPESM2-1A (sqt1D1) expressing Sqt1-myc pro- <br /> colonies b-galactosidase filter assay. Expression                      tein CEN plasmid; 4, cytosolic fraction DPESM2-1A; 5, membrane <br />                                                                                      fraction DPESM2-1A. (B) Strain DPESM2-1A fractionated obtain <br />                                                                                      nuclei, blots probed anti-myc anti-Nop1 antibodies. Lanes: 1, <br />                                                                                      cell lysate; 2, postnuclear supernatant; 3, nuclear fraction. <br />  <br />  <br />  <br />  <br />                                                                                      reporter genes observed SQT1 fused                                                                                      DNA binding domain QSR1 coexpressed prey <br />                                                                                      fusion. <br />                                                                                         Sqt1p cytosolic protein affinity ribosomes. <br />                                                                                      Sqt1p myc tagged C terminus fusion myc <br />                                                                                      epitope (31) residue SQT1 open reading <br />                                                                                      frame, CEN plasmid carrying myc-tagged SQT1                                                                                      transformed hemizygous SQT1 strain DPE1.                                                                                      diploid sporulated tetrads dissected,                                                                                      SQT1-myc allele covered SQT1 chromosomal deletion                                                                                      frequency compatible segregation centromeric plas- <br />                                                                                      mid, deletion strain covered CEN plasmid <br />                                                                                      carrying SQT1-myc grew normal doubling time liquid <br />                                                                                      culture 30&#194;&#176;C. <br />                                                                                         Western blot analysis SQT1-myc strains mouse <br />                                                                                      monoclonal anti-myc antibody reveals specific protein                                                                                      present control strain lacking myc-tagged <br />                                                                                      protein (Fig. 6A) abundant strain <br />                                                                                      expressing myc-tagged protein 2mm plasmid.                                                                                      myc-tagged protein migrates approximately 52 kDa, com- <br />                                                                                      pared predicted molecular mass 48.5 kDa. Subcellular <br />    FIG. 5. Directed hybrid analysis QSR1 SQT1. Proteins ex- <br />                                                                                      fractionation cytosolic membrane fractions pressed LexA fusion vector pEG202 derivatives pEG202-QSR1           SQT1-myc strain DPESM2-1A [sqt1D1 pSQTMYC2(SQT1- pEG202-SQT1 GAL4 activation domain fusion vector pACT2              MYC CEN LEU2)] shows Sqt1p predominantly derivatives pACT2-QSR1 pACT2-SQT1. LexA, length LexA                cytosolic fraction (Fig. 6A). levels Qsr1p af- <br /> protein; GAD, GAL4 activation domain. Transformants assayed b- <br /> galactosidase activity, activities means 10 15 independent <br />                                                                                      fected gene dosage SQT1 different strains, transformants, assayed triplicate. Standard errors basal activities   relative Qsr1p membrane fraction 30% depicted. GAL4 activation domain LexA                 greater Sqt1p fraction (Fig. 6A). <br /> plasmid pSH17-4 used positive control showed activity 590 U.      association Qsr1p membrane fraction  <br />  5152      EISINGER ET AL.                                                                                                                   MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />    FIG. 7. Characterization Sqt1p translational machinery.   Analysis polysome profiles distribution Sqt1p, Qsr1p, Prt1p strain <br /> DPESM2-1A. Cell lysates fractionated centrifugation 5 47% sucrose gradients. Immunoblots Sqt1p Prt1p aligned tracings. (B) <br /> Fractionation 0.5 M KCl ribosomal salt wash fraction gel filtration chromatography Superose 6 column. 280-nm absorbance (A280) profile shown location known protein standards. Immunoblots Sqt1p Prt1p aligned corresponding column fractions. <br />  <br />  <br />  <br />  <br /> sistent fact 60S subunits bind endoplasmic                   yeast eIF-3, SDS-PAGE yeast eIF-3 does protein reticulum relatively high affinity (6, 29).                                  molecular mass range (41). monitored distri- <br />    possible explanation rescue subunit-join-                   bution Prt1p, subunit eIF-3 complex (19, 41, 42), ing defect mutants 60S subunit assembly im-                     gradient fractions explore possible colocalization <br /> paired truncated Qsr1p rescued SQT1. 40S                      Sqt1p eIF-3 complex. distribution Sqt1p 60S subunits assembled nucleolus (36, 56, 58),                     does match Prt1p, concentrated investigated Sqt1p nuclear protein. cell                     40S subunit (Fig. 7A). smallest form Prt1p fraction <br /> lysates fractionated recover nuclei postnuclear                       3, agreement 600-kDa eIF-3 bound supernatants, Sqt1p recovered supernatant fraction,                    40S subunits run 540-kDa b-galactosi- Nop1p, nucleolar marker (52), enriched                         dase marker recovered. <br /> nuclear fraction, shown Fig. 6B.                                               test possible association Sqt1p    Sqt1p cytosolic protein physically interacts                   eIF-3 complex, washed ribosomes 0.5 M KCl obtain Qsr1p, 60S ribosomal subunit protein, analyzed                      extract enriched translational initiation factors (10, 41). <br /> distribution Sqt1p polysome gradients. shown Fig.                     quantitative Western blot analysis, estimated 5% 7A, majority Sqt1p recovered 0.5 ml                   Sqt1p cell lysate recovered fraction, <br /> polysome gradient, contains free cytosolic                      remaining 95% postribosomal supernatant <br /> proteins.  Sqt1p does migrate significantly                      (data shown). salt-washed ribosomal extract gradient present 80S ribosome small                        fractionated gel filtration Superose 6 column, Sqt1p <br /> polysome fractions. unlikely nonspecific tailing,                   concentrated fraction 21, corresponding molecular cytosolic markers hexokinase recovered solely                   mass 158 kDa significantly larger 47 gradients (13). Furthermore, strain                    kDa predicted Sqt1p (Fig. 7B). elution profile Sqt1p <br /> (DPESM1) expressing SQT1 high-copy plasmid                             does match Prt1, eluted position <br /> similarly analyzed, excess Sqt1p accumulated                    670-kDa marker, agreement size eIF-3 gradient, lower fractions contained                      complex (10). fractionated postribosomal super- relative amounts Sqt1p observed low-copy                      natant Superose 6 column Sqt1p strain (data shown). properties Sqt1p consis-                    Prt1p localized fractions ribosomal salt <br /> tent loose association ribosomes.                                     wash (data shown). results Sqt1p dis- <br />    apparently loose association Sqt1p ribosomes                       tinct eIF-3 complex. Sqt1p behaves resembles behavior translational initiation factors                   oligomeric associated novel complex 90 150 <br /> (24). Translational initiation factor eIF-3 stable complex                kDa. polypeptides aggregate mass approx-                        Depletion Sqt1p results subunit-joining defect. imately 600 kDa understood transla-                       analyzed phenotype cells depleted Sqt1p construct- <br /> tional initiation factors  review, reference 24). eIF-3                 ing yeast strain (FAD13-2B) SQT1 controlled involved steps initiation pathway;              GAL1-10 promoter plasmid. GAL-SQT1 plasmid factors bind 40S subunit, shifting equi-                  covers SQT1 chromosomal deletion grown galac- <br /> librium 40S subunits away association 60S sub-                       tose, cells covered myc-tagged Sqt1p grow units, thought factor leave subunit                 rate covered Sqt1p (data shown). joining. recent description mammalian eIF-3 subunits                     Western analysis, determined Sqt1p undetect- <br /> lists 47-kDa protein (24), molecular mass Sqt1p,                     able 8 h shift dextrose medium (data shown). report 47-kDa protein                     GAL-SQT1 strain pregrown dextrose 8 h deplete <br />  VOL. 17, 1997                                                          SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                        5153 <br />  <br />  <br />                                                                                   8B Fig. 4B). Western blots 60S subunit fractions                                                                                   wild-type cells GAL-SQT1 strain indicate Qsr1p                                                                                   significantly depleted 60S fraction 16 h depletion                                                                                   Sqt1p (Fig. 8C). <br />  <br />                                                                                                            DISCUSSION <br />                                                                                      Truncations QSR1 unable substitute                                                                                   length gene inhibit growth expressed yeast                                                                                   contain wild-type chromosomal copy QSR1.                                                                                   truncations tested, encoding longest (qsr1- <br />                                                                                   187Dp) shortest (qsr1-64Dp) forms protein acted                                                                                   dominant-negative mutations, truncation predicted                                                                                   encode protein intermediate length (qsr1-133Dp)                                                                                   detectable effect growth, possibly form                                                                                   protein rapidly degraded present sufficient <br />                                                                                   amounts exert negative effect. possible  <br />                                                                                   termine relative amounts truncated proteins, <br />                                                                                   antibodies raised Qsr1p did detect                                                                                   truncated proteins. suspect anti- <br />                                                                                   bodies recognize epitopes nonconserved C terminus <br />                                                                                   (16), absent truncated proteins. <br />                                                                                      QSR1 truncations placed control                                                                                   galactose-inducible promoter, permitting controlled <br />                                                                                   expression. dominant-negative qsr1-64D allele                                                                                   induced growth galactose, examination polysome <br />                                                                                   profile revealed ribosomal subunit-joining defect, evi- <br />                                                                                   denced presence half-mer polysomes. qsr1- <br />                                                                                   64D allele repressed dextrose, wild-type polysome <br />                                                                                   profile restored. half-mer phenotype qsr1-64D <br />                                                                                   allele consistent disruption wild-type Qsr1p function, <br />                                                                                   temperature-sensitive qsr1 mutants original <br />                                                                                   qsr1-1 mutant exhibit similar half-mer polysome profiles (14). <br />                                                                                      screening yeast genomic library, isolated novel <br />                                                                                   suppressor qsr1 truncations, SQT1, rescue                                                                                   growth inhibition half-mer polysome profile caused                                                                                   qsr1-64D allele. SQT1 initially isolated                                                                                   high-copy suppressor, subsequent tests established expres- <br />                                                                                   sion CEN plasmid equally effective rescuing                                                                                   dominant-negative mutants, indicating slight excess                                                                                   Sqt1p addition derived chromosomal gene <br />                                                                                   sufficient reverse toxic effects truncated pro- <br />                                                                                   teins. <br />                                                                                      screen high-copy genomic suppressors revealed <br />    FIG. 8. Loss SQT1 function results half-mer polysomes. FAD13-2B cells    QSR1 reverse dominant-negative effect grown galactose medium followed 8 h preculture dextrose          qsr1-64D allele. suggests excess wild-type <br /> medium, reinoculated fresh dextrose medium. Extracts prepared cycloheximide-arrested cells 0 h   16 h (B) reinoc- <br />                                                                                   protein outcompete dominant-negative truncated pro- <br /> ulation dextrose medium analyzed 5 47% sucrose velocity gradi-     tein. general, dominant-negative polypeptide needs ents. Fractions containing 60S subunits pooled gradients             excess inhibitory effects substantial (25). <br /> FAD13-2B prepared  B) wild-type control (YPH499)            SQT1 rescued dominant-negative qsr1 0- 16-h time points used prepare SDS-PAGE samples, probed Qsr1p L3 (C). <br />                                                                                   truncations, rescue temperature-sensitive qsr1 mu- <br />                                                                                   tants. indicates SQT1 function bypass <br />                                                                                   suppressor temperature-sensitive alleles sub- <br />                                                                                   stitute QSR1. SQT1 rescue original <br /> Sqt1p reinoculated fresh dextrose medium.                         qsr1-1 mutation, G194D missense mutation C terminus <br /> growth defect evident 12 h reinoculation                    protein semidominant QSR1 (54). <br /> dextrose medium, eventually cells ceased grow (data                    SQT1 appears allele-specific suppressor qsr1 shown). Polysome profiles cells immediately rein-                    dominant-negative truncation mutations. qsr1-1 allele oculation indistinguishable wild-type cells                    rescued single chromosomal copy QCR6, gene <br /> (Fig. 8A).  polysome profiles wild-                    encodes subunit 6 mitochondrial cytochrome bc1 <br /> type cells remained unchanged 16 h dextrose (data                    complex (54). known gene protein shown), half-mer phenotype clearly present polysomes                      known location inner mitochondrial membrane GAL-SQT1 cells decrease 80S                        rescue lethal missense mutation cyto- polysome peaks (Fig. 8B). altered polysome profile                        plasmic ribosomal protein. failure SQT1 substitute <br /> mimics dominant-negative qsr1 truncation mutants,                     QCR6 indicates different roles SQT1 QCR6 including slight decrease 60S abundance (compare Fig.                      relationship QSR1. <br />  5154     EISINGER ET AL.                                                                                                          MOL. CELL. BIOL. <br />  <br />  <br />    mechanisms truncations QSR1          SQT1. known WD-repeat proteins participate exert dominant-negative effect suppressed         assembly multiprotein complexes, pro- <br /> QSR1 SQT1. possibility truncated Qsr1p             posed common function WD proteins (44). <br /> causes reduced assembly 60S subunits, resulting stoi-         Consistent interpretation fact defect chiometric imbalance ribosomal subunits formation           translational initiation expected manifested <br /> half-mer polysomes. unlikely,        sooner 12 h disappearance Sqt1p (40). <br /> slight change ratio 40S 60S subunits            Interestingly, pronounced growth defects FAD13-2B <br /> qsr1-64D allele induced decrease proportion      strain Sqt1p depleted 8-h <br /> 60S subunits does sufficient account ap-          preculture starting growth curves. indicates <br /> pearance half-mers, especially comparison previously         Sqt1p function early log-phase growth, reported mutants grossly defective assembly        ribosomal biogenesis occurs (28).  cytosolic loca- <br /> 60S subunits (11, 12, 35, 38, 39, 45, 47, 48).          tion Sqt1p argues role ribosome assembly unlikely SQT1 rescues defect assembly 60S sub-           nucleolus, suggesting important maturation <br /> units. ribosomal subunits assembled nucle-         modification 60S subunits occurs cytoplasm. olus factors affecting assembly subunits         delay assembly Qsr1p immature 60S subunits localized nucleolus (36, 56, 58), Sqt1p         result degradation account slight 40S-60S <br /> excluded nucleus. Additionally, loss SQT1             subunit imbalance observed. explanation applies, function normal genetic background does           cytosolic location Sqt1p imply Qsr1p appear impair assembly 60S subunits degree           small group proteins exchanges ribo- <br /> expected prolonged depletion Sqt1p.                    somes cytoplasm (12, 59). <br />    second explanation dominant-negative effect truncated Qsr1p assembled 60S subunit,                                    ACKNOWLEDGMENTS inactivates sequesters closely associ- <br />                                                                         research supported American Cancer Society grant ated ribosomal proteins falls  leaving 60S subunit <br />                                                                       204. F D. supported American Heart Association predoc- <br /> incapable subunit joining. Sqt1p loosely         toral fellowship (NH/VT affiliate). <br /> associated ribosomes, conceivable stabilizes         thank Alan Hinnebusch antibodies Prt1 Ed Hurt Qsr1p 60S subunit. hypothesis consistent          antibodies Nop1p. observation loss SQT1 function results smaller <br /> amounts Qsr1p free 60S subunits.                                              REFERENCES <br /> obvious SQT1 rescue dominant-negative effect             1. Ausubel, F. M., R. E. Brent, D. D. Moore, J.  Smith, J. G. Seidman, K. qsr1-64D qsr1-187D mutations, correspond-            Struhl. 1996. Current protocols molecular biology. Wiley Interscience, <br />                                                                           New York, N.Y. <br /> ing truncated proteins substitute Qsr1p.                 2. Ballesta, J. P., M. Remacha. 1996. large ribosomal subunit stalk    possibility Sqt1p involved subunit joining       regulatory element eukaryotic translational machinery. Prog. Nu- interacts Qsr1p 60S subunit. case,                cleic Acids Res. Mol. Biol. 55:157&#226;&#8364;&#8220;193. <br /> truncated forms Qsr1p assembled 60S           3. Baudin, , O. Ozier-Kalogeropoulos,  Denouel, F. Lacroute, C. Cul- <br />                                                                           lin. 1993. simple efficient method direct gene deletion Saccha- <br /> subunit sequester Sqt1p prevent interaction             romyces cerevisiae. Nucleic Acids Res. 21:3329&#226;&#8364;&#8220;3330. Qsr1p 60S subunit. distribution Sqt1p            4. Beckner, M. E., H. C. Krutzsch, M. L. Stracke, S. T. Williams, J.  Gal- <br /> polysome gradients interaction Qsr1p              lardo, L.  Liotta. 1995. Identification new immunoglobulin su- <br /> hybrid indicate interacts ribosomes.               perfamily protein expressed blood vessels heparin-binding consen- <br />                                                                           sus sequence. Cancer Res. 55:2140&#226;&#8364;&#8220;2149. <br /> explored possibility Sqt1p subunit known            5. Bonneaud, N., O. Ozier-Kalogeropoulos, G. Li, M. Labouesse, L. Minivielle- <br /> translation initiation factor. Eukaryotic initiation factors          Sebastia, F. Lacroute. 1991. family low high copy replicative, <br /> multiprotein complexes defined biochemically               integrative single-stranded S. cerevisiae/E. coli shuttle vectors. Yeast reconstituting vitro translation. reviews, reference        7:609&#226;&#8364;&#8220;615. <br />                                                                        6. Borgese, N. 1974. Ribosomal-membrane interaction: vitro binding ri- <br /> 24 53. initiation factors associate 40S subunit         bosomes microsomal membranes. J. Mol. Biol. 88:559&#226;&#8364;&#8220;580. specific temporal order  possible exception       7. Bradford, M. M. 1976. rapid sensitive method quantitation eIF-2, dissociate joining 60S subunit. <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span>          microgram quantities protein utilizing principal protein-dye bind- <br /> subunits eIF-1, eIF-1A, eIF-2, eIF-4, eIF-5 known,             ing. Anal. Biochem. 72:248&#226;&#8364;&#8220;254. <br />                                                                        8. Chakravarti, D., U. Maitra. 1993. Eukaryotic initiation factor 5 factors function initiation cycle disso-           Saccharomyces cerevisiae. J. Biol. Chem. 268:10524&#226;&#8364;&#8220;10533. <br /> ciation 40S 60S subunits final 60S joining (8, 26, 33,      9. Chan, Y., J. Diaz, L. Denoroy, J. Madjar,  G. Wool. 1996. primary <br /> 50). possible Sqt1p subunit eIF-3             structure rat ribosomal protein L10: relationship Jun-binding protein <br /> (24), initiation factor peptide composition          putative Wilms&#226;&#8364;&#8482; tumor suppressor. Biochem. Biophys. Res. Com- <br />                                                                           mun. 225:952&#226;&#8364;&#8220;956. <br /> characterized (19, 40&#226;&#8364;&#8220;42).  Sqt1p does colocalize         10. Cigan, M. , M. Foiani, E. M. Hannig,  G. Hinnebusch. 1991. Com- Prt1p, known subunit eIF-3, polysome gradients            plex formation positive negative translational regulators GCN4. size exclusion chromatography behaves            Mol. Cell. Biol. 11:3217&#226;&#8364;&#8220;3228. <br /> oligomeric previously unreported protein complex.        11. Deshmukh, M., J. Stark, L. C. Yeh, J. C. Lee, J. L. Woolford. 1995. <br />                                                                           Multiple regions yeast ribosomal protein L1 important Furthermore, inactivation eIF-3 transcriptional             interaction 5S rRNA assembly ribosomes. J. Biol. Chem. <br /> pression known subunit results large inactive 80S peak          270:30148&#226;&#8364;&#8220;30156. <br /> (40), half-mer polysomes described                        12. Deshmukh, M., Y. Tsay,  G. Paulovich, J. L. Woolford. 1993. Deple- <br />    plausible explanation Sqt1p involved            tion Saccharomyces cerevisiae ribosomal protein L16 causes decrease                                                                           60S ribosomal subunits formation half-mer polyribosomes. Mol. Cell. <br /> assembling Qsr1p 60S subunit late assembly                Biol. 13:2835&#226;&#8364;&#8220;2845. <br /> pathway, truncated Qsr1p competes length            13. Dick, F. , S. Karamanou, B. L. Trumpower. QSR1, essential yeast <br /> Qsr1p assembly 60S subunit. lack func-                gene genetic relationship subunit mitochondrial cyto- <br /> tional Qsr1p result half-mer phenotype,              chrome bc1 complex, codes 60S ribosomal subunit protein. J. Biol. <br />                                                                           Chem. 272:13372&#226;&#8364;&#8220;13379. <br /> case temperature-sensitive qsr1 mutants (14). truncated        14. Eisinger, D. P., F.  Dick, B. L. Trumpower. 1997. Qsr1p, 60S <br /> forms Qsr1p sequester essential Sqt1p, defect               ribosomal subunit protein, required joining 40S 60S subunits. suppressed extra expression QSR1                 Mol. Cell. Biol. 17:5136&#226;&#8364;&#8220;5145. <br />  VOL. 17, 1997                                                            SQT1 SUPPRESSES DOMINANT-NEGATIVE QSR1 MUTATIONS                                         5155 <br />  <br />  <br /> 15. Eisinger, D. P., B. L. Trumpower. Long-inverse PCR generate               ribosomal subunits perturbed temperature-sensitive yeast mutants  <br />     gional peptide libraries codon mutagenesis. BioTechniques, press.             fective ribosomal protein L16. Mol. Cell. Biol. 11:5681&#226;&#8364;&#8220;5692. <br /> 16. Farmer,  , T. M. Loftus,   Mills, K. Y. Sato, J. D. Neil, T. Tron, M.   40. Naranda, T., M. Kainuma, S. E. MacMillan, J. W. B. Hershey. 1997.     Yang, B. L. Trumpower, E. J. Stanbridge. 1994. Extreme evolutionary             39-kilodalton subunit eukaryotic translation initiation factor 3 essential <br />     conservation QM, novel c-Jun associated transcription factor. Hum.             complex&#226;&#8364;&#8482;s integrity cell viability Saccharomyces cerevisiae. <br />     Mol. Genet. 3:723&#226;&#8364;&#8220;728.                                                              Mol. Cell. Biol. 17:145&#226;&#8364;&#8220;153. <br /> 17. Fields, S., O. Song. 1989. novel genetic detect protein-        41. Naranda, T. S., E. MacMillan, J. W. B. Hershey. 1994. Purified yeast <br />     protein interactions. Nature (London) 340:245&#226;&#8364;&#8220;246.                                  translational initiation factor eIF-3 RNA-binding protein complex 18. Fioani, M., M.  Cigan, C. J. Paddon, S. Harashima,  G. Hinnebusch.          contains PRT1 protein. J. Biol. Chem. 269:32286&#226;&#8364;&#8220;32292. <br />     1991. GCD2, translational repressor GCN4 gene, general           42. Naranda, T. S., E. MacMillan, T. F. Donahue, J. W. B. Hershey. 1996. <br />     function initiation protein synthesis Saccharomyces cerevisiae.        SUI1/p16 required activity eukaryotic translation initiation <br />     Mol. Cell. Biol. 11:3203&#226;&#8364;&#8220;3216.                                             <br /> </body></html>