<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Targeted metabolic labeling yeast <br /> N-glycans unnatural sugars <br /> Mark  Breidenbacha, Jennifer E. G. Gallagherb, David S. Kingc, Brian P. Smarta, <br /> Peng Wua,2, Carolyn R. Bertozzia,b,c,d,1  Departments Chemistry; bMolecular Cell Biology; cHoward Hughes Medical Institute, University California, Berkeley, CA 94720; dThe Molecular Foundry, Lawrence Berkeley National Laboratory, Berkeley, CA 94720 <br />  <br /> Edited David  Tirrell, California Institute Technology, Pasadena, CA, approved December 23, 2009 (received review September 30, 2009) <br />  <br /> Metabolic labeling glycans synthetic sugar analogs               mass spectrometric analyses proteins displaying nonna- <br /> emerged attractive means introducing nonnatural chemi-             tural <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> functionality (5, 6). <br /> cal functionality glycoproteins.  complexities              existing glycan labeling methodology suitable stud- <br /> glycan biosynthesis prevent installation nonnatural moieties          ies require stochastic insertion analogs low levels, defined, predictable locations glycoproteins high levels        technique inadequate applications specific incorporation.  demonstrate conserved                    monosaccharides reliably targeted metabolic replace- <br /> N-acetyglucosamine (GlcNAc) residues chitobiose cores              ment. example, high-efficiency predictable installation N-glycans model organism Saccharomyces cerevisiae                 sugar analogs dramatically facilitate biophysical studies specifically targeted metabolic replacement unnatural              glycoprotein structure function site-specific introduction <br /> sugars. introduced exogenous GlcNAc salvage pathway               fluorophores heavy atoms. Unfortunately, site-specific  <br /> yeast, allowing cells metabolize GlcNAc provided supple-             tabolic labeling glycans hindered multiple factors.  <br /> ment culture medium. rendered yeast auxo-                 <span id='am-29' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-30' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span>-sugar analogs necessarily direct competition <br /> trophic production donor nucleotide-sugar <span id='am-37' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Uridine" datatype="xsd:string"></span>uridine</span>-                cell&#226;&#8364;&#8482;s endogenous pool donor sugars, leading diphosphate-GlcNAc (<span id='am-48' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-GlcNAc) deletion essential                 poor levels metabolic incorporation synthetic analog (7). <br /> gene GNA1. demonstrate gna1&#206;&#8221; strains require GlcNAc                 eukaryotes possess epimerase activities supplement expression plasmids containing exoge-               interconvert nucleotide-sugar stereochemistries alter <br /> nous components salvage pathway, GlcNAc transporter                   final metabolic destination given <span id='am-42' about='obo:IMR_0002125' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="MeSH:D009005" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="monosaccharide" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0002125" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:35381" datatype="xsd:string"></span>monosaccharide</span> (8, 9). <br /> NGT1 Candida albicans GlcNAc kinase NAGK Homo                  Finally, glycan assembly genetically encoded sapiens, required rescue context.            inherently prone microheterogeneity, targeting ana- cells successfully incorporate synthetic GlcNAc analogs                 logs specific positions defined class glycoconjugates <br /> N-azidoacetyglucosamine (GlcNAz) N-(4-pentynoyl)-glucosa- <br />                                                                              untenable. (GlcNAl) cell-surface glycans secreted glycoproteins. <br />                                                                                 glycan biosynthesis template-driven,                                                                              glycans share conserved structural features result pro- verify incorporation nonnatural sugars N-glycan core <br />                                                                              cess assembled. Asparagine-linked glycans <br /> positions, endoglycosidase H (endoH)-digested peptides                                                                              (N-glycans) excellent example; antennary regions <br /> purified secretory glycoprotein, Ygp1, analyzed mass <br />                                                                              N-glycans ultimately edited diverse compositions spectrometry. Multiple Ygp1 N-glycosylation sites bearing GlcNAc, <br />                                                                              structures, N-glycan cores derived conserved <br /> isotopically labeled GlcNAc, GlcNAz identified; mod- <br />                                                                              tetradecasaccharide precursor featuring &#206;&#178;1,4-linked GlcNAc <br /> ifications dependent supplement added culture <br />                                                                              chitobiose unit site polypeptide attachment (10). medium. enables production glycoproteins                                                                              GlcNAc monomers chitobiose unit, donated UDP- functionalized specific chemical modifications                                                                              GlcNAc, persist lifespan N-glycan.  <br /> glycosylation sites.                                                         target conserved core GlcNAc residues metabolic  <br />                                                                              placement, effectively sidestepping issue glycan structural <br /> click chemistry &#226;&#710;&#163; GlcNAc &#226;&#710;&#163; metabolic engineering &#226;&#710;&#163; N-glycosylation &#226;&#710;&#163; GNA1 <br />                                                                              heterogeneity (Fig. 1A). N-glycans uniquely suited                                                                              targeted metabolic labeling profoundly influ- <br /> T   introduction unnatural chemical moieties proteins <br />     enormous potential facilitate vivo vitro <br /> studies protein function, mechanism, structure. exam- <br />                                                                              ence protein&#226;&#8364;&#8482;s folding, trafficking, solubility, stability (11); <br />                                                                              types glycosylation, N-glycan occu- <br />                                                                              pancy mandatory function.  occupancy ple, genetically encoded residue-specific approaches                 N-glycosylation sites sufficiently high observe core carbohy- <br /> introduction unique amino acids enabled integration               drate residues crystallographically (12). functionalities spectroscopic probes, bioorthogonal             Targeted glycan labeling dramatically simplified <br /> chemical reporters, metal chelators, photoaffinity labels                working organism relatively characterized <br /> directly recombinant proteins (1, 2). alternative              glycan biosynthesis. study, utilize model eukaryote insertion nonnatural amino acids polypeptide chain,            S. cerevisiae reasons including relatively simple <br /> unique chemical functionality introduced protein&#226;&#8364;&#8482;s posttranslational modifications exemplified                                                                              Author contributions: M B. designed research; M B., J.E.G.G., D.S.K. performed <br /> technique glycan metabolic labeling (3). approach,                research; M B., J.E.G.G., B.P.S. analyzed data; M B., J.E.G.G., C.R.B. wrote <br /> synthetic analogs natural monosaccharides introduced             paper; P.W. contributed new reagents/analytic tools. <br /> cells processed series enzymes generate            authors declare conflict  <br /> activated nucleotide-sugar analogs. sugar analogs, bearing               article PNAS Direct Submission. <br /> unique chemical functionality, subsequently utilized              1 <br />                                                                                  correspondence addressed. E-mail: crb@berkeley.edu. <br /> biosynthesis various cellular glycoconjugates; technique             2 <br />                                                                                  Present address: Department Biochemistry, Albert Einstein College Medicine, vividly illustrated recent glycan imaging studies (4).               Yeshiva University, Bronx, NY 10461. <br /> Additionally, introduction synthetic sugars glycans              article contains supporting information online www.pnas.org/cgi/content  metabolic labeling exploited affinity capture               0911247107/DCSupplemental. <br />  <br />  <br />  <br /> 3988&#226;&#8364;&#8220;3993 &#226;&#710;&#163; PNAS &#226;&#710;&#163; March 2, 2010 &#226;&#710;&#163; vol. 107 &#226;&#710;&#163;  9                                                               www.pnas.org/cgi/doi/10.1073/pnas.0911247107 <br />                                                                                    demonstrate synthetic, azide- alkyne-bearing <br />                                                                                GlcNAc analogs isotopically labeled GlcNAc                                                                                introduced yeast N-glycans unique salvage pathway. <br />                                                                                Azide alkyne functional groups known versatile <br />                                                                                chemical handles bioconjugation proteins, cells, living <br />                                                                                organisms (20).  utilize phosphine (21), alkyne (22),                                                                                azide (23) probes conjunction mass spectrometry verify <br />                                                                                GlcNAc analogs successfully metabolized integrated <br />                                                                                yeast cell surfaces secreted glycoproteins. <br />                                                                                Results Discussion <br />                                                                                Design gna1&#206;&#8221; Strains. step novo UDP-GlcNAc bio- <br />                                                                                synthetic pathway (Fig. 1B) responsible metabolic conver- <br />                                                                                sion fructose-6-phosphate, intermediate glycolysis,                                                                                UDP-GlcNAc. highly conserved enzymes pathway                                                                                essential yeast viability normal growth conditions (24). <br />                                                                                interruption UDP-GlcNAc biosynthetic pathway                                                                                rescued appropriate downstream metabolite delivered <br />                                                                                cells. example, gfa1&#206;&#8221; strains S. cerevisiae capable <br />                                                                                internalizing phosphorylating extracellular glucosamine, <br />                                                                                effectively bypassing GFA1 deletion (25). organisms, <br />                                                                                C. albicans, functional salvage pathways allowing <br /> Fig. 1. Targeting GlcNAc metabolic replacement.   structurally       internalize phosphorylate GlcNAc (18, 19, 26), <br /> conserved GlcNAc2 Man8 core region S. cerevisiae N-glycan shown       S. cerevisiae lacks activities gna1&#206;&#8221; strains tached hypothetical integral membrane protein. N-glycan cores sus-    rescued extracellular GlcNAc (Fig. 2). GlcNAc <br /> ceptible specific cleavage endoH PNGaseF indicated. Synthetic <br /> GlcNAc analogs bearing bioorthogonal chemical groups azides                                                                                analogs modifications C-2 acetamido substituent <br /> alkynes allow bioconjugation diverse probes cargos directly      known tolerated downstream eukaryotic biosynthetic <br /> glycans. (B) strategy bypassing novo UDP-GlcNAc biosynthesis (black   machinery (27), chose interrupt UDP-GlcNAc production <br /> arrows) shown. exogenous salvage pathway (blue arrows) allows extra-     N-acetyltransferase step catalyzed Gna1. Glucosamine, <br /> cellular GlcNAc analogs internalized transporter Ngt1     produced chemical enzymatic deacetylation C. albicans. intracellular GlcNAc  analog) phosphorylated 6   GlcNAc, rescue gna1&#206;&#8221; strains does case position activity human GlcNAc kinase, NAGK (28).           GFA1 disruption. bypass GNA1 disruption, 6-phosphorylated product subsequently converted activated <br />                                                                                necessary bestow cells functional alternative path- <br /> nucleotide-sugar mutase pyrophosphorylase activities Pcm1 Qri1 respectively. <br />                                                                                way, illustrated Fig. 1B. introduced expression plasmids <br />                                                                                bearing human GlcNAc kinase NAGK heterozygous <br />                                                                                GNA1/gna1&#206;&#8221; strain S. cerevisiae primarily NAGK <br /> composition glycans ease yeast                previously shown tolerate GlcNAc analogs vitro <br /> genetically modified. Importantly, UDP-GlcNAc known                      (28). enzyme capable phosphorylating position <br /> provide GlcNAc S. cerevisiae glycoconjugate types                 cytosolic GlcNAc  analogs), delivering required <br />  <br />  <br />  <br />  <br />                                                                                                                                                             CHEMISTRY core chitobiose units N-glycans. UDP-GlcNAc                  metabolite compensate GNA1 disruption. NAGK tran- <br /> nates GlcNAc chitin, linear polymer &#206;&#178;-1,4-GlcNAc (13)                scription placed control constitutive minor component yeast cell wall deposited              glyceraldehyde-3-phosphate dehydrogenase (GPD) promoter ring neck growing bud cell division.             pRS41X vector scaffolds URA3, HIS3, LEU2 selective <br /> Additionally, UDP-GlcNAc donor step GPI                markers (29). <br /> anchor biosynthesis, GlcNAc residue subsequently dea-                  enable utilization extracellular GlcNAc, introduced <br /> cetylated (14). Unlike higher eukaryotes, S. cerevisiae does               plasmids carrying recently characterized C. albicans GlcNAc <br /> incorporate GlcNAc O-glycans (15) antennary regions                    transporter NGT1 GNA1/gna1&#206;&#8221; strain (19).  <br />  <br />  <br />                                                                                                                                                             GENETICS N-glycans (16). Furthermore, S. cerevisiae lacks epimerases                 NGT1 initially cloned pRS41X-GPD scaffolds, act UDP-GlcNAc (17).  distribution GlcNAc                  transformants obtained plasmids containing S. cerevisiae glycoproteome expected restricted             GPD promoters, suggesting S. cerevisiae sensitive chitobiose cores N-glycans.                                             NGT1 expression level. NGT1 subsequently placed    address issue competing endogenous UDP-                       galactose-inducible GAL1 promoter remains largely  <br /> GlcNAc pool, hypothesized genetic disruption novo                pressed glucose-containing medium. combinations UDP-GlcNAc biosynthesis removal essential glucos-                   NGT1 NAGK plasmids transformed GNA1/gna1&#206;&#8221; amine 6-phosphate N-acetyltransferase (GNA1) gene                        summarized Table S1. isolate gna1&#206;&#8221; haploids, subjected <br /> force cells scavenge environmental GlcNAc survive. Disrup-               heterozygous transformants sporulation. Notably, dually <br /> tion S. cerevisiae GNA1 possible viable alternate              transformed strains carrying NGT1 NAGK plasmids  <br /> pathway provided complement loss (18). S. cerevi-              veloped tetrads. GNA1/gna1&#206;&#8221; heterozygotes sporulated <br /> siae does endogenous GlcNAc salvage pathway,                 slowly produced tetrads 7 d sporulation media; <br /> equipped strains component salvage pathway bor-                     supplementation GlcNAc difference. Dissections <br /> rowed organisms (Fig. 1B). Specifically, employed                resulting tetrads demonstrated high rate spore death recently characterized GlcNAc transporter NGT1                        25.5%   2.1% YPD &#195;&#190; GlcNAc. 28.4%   2.3% Candida albicans (19) human GlcNAc kinase NAGK.                       tetrads contained viable spores half (49.8%   5%) functional salvage pathway allows bypass lethal               tetrads viable spores contained wild-type <br /> GNA1 deletion long cells provided GlcNAc-                 GNA1. unclear gna1&#206;&#8221; segregants demonstrated supplement growth medium. modifications                 high rate inviability. GNA1 disruption surviving segregants <br /> place, obtained viable gna1&#206;&#8221; GlcNAc auxotrophs.                             confirmed PCR (Fig. S1). Surviving gna1&#206;&#8221; segregants <br />  <br /> Breidenbach et al.                                                                                 PNAS &#226;&#710;&#163;   March 2, 2010 &#226;&#710;&#163;   vol. 107 &#226;&#710;&#163;    9 &#226;&#710;&#163;   3989 <br />  Fig. 2. Verification exogenous GlcNAc salvage pathway S. cerevisiae. Tenfold serial dilutions cultures GNA1 gna1&#206;&#8221; carrying <br /> combinations NGT NAGK plasmids spotted solid media reveal necessity extracellular GlcNAc source NGT1 NAGK-encoding <br /> plasmids essential gna1&#206;&#8221; rescue. Haploid gna1&#206;&#8221; strains (rows 2, 4, 6) grow rich media supplemented GlcNAc (YPD &#195;&#190; GlcNAc) wild-type cells showed difference growth (rows 1, 3, 5). requirement NAGK NGT1 encoded pRS416 plasmids gna1&#206;&#8221; yeast demonstrated growth 5-FOA &#195;&#190; GlcNAc media. Cells containing pRS413-GAL-NGT1 pRS415-GPD-NAGK (rows 1-2) insensitive 5-FOA, NGT1 NAGK introduced pRS416 plasmid, 5-FOA prevents growth gna1&#206;&#8221;, GNA1 strains (rows 3-6). <br />  <br />  <br /> demonstrated GlcNAc auxotrophy (Fig. 2) doubling                    ment. secreted proteins GlcNAc- GlcNAz- <br /> time approximately twice wild-type haploids.                          supplemented cultures subjected Staudinger ligation <br />    verify plasmids bearing NGT1 NAGK essen-                  (21) phosphine-FLAG (phos-FLAG) chemospecifically <br /> tial GlcNAc salvage gna1&#206;&#8221; rescue, negative selection <br /> scheme implemented. Haploid strains harboring NAGK NGT1 URA3 marked plasmid (pRS416) isolated dissected tetrads. assess growth  <br /> quirements various gna1&#206;&#8221; haploids, strains spotted 5-fluoroorotic acid (5-FOA) media &#195;&#190; GlcNAc <br /> (Fig. 2). Cells containing URA3 marked plasmid convert <br /> 5-FOA 5-fluorouracil grow presence compound, enabling negative selection indivi- <br /> dual component salvage pathway (30). gna1&#206;&#8221; cells grow 5-FOA media NGT1 NAGK encoded pRS416 (URA3) plasmid, genes encoded pRS415 (LEU2) pRS413 (HIS3), respec- <br /> tively, conclude gna1&#206;&#8221; yeast rescued NAGK NGT1 gene products expressed growth <br /> medium supplemented GlcNAc. <br />    NGT1 transcription controlled GAL1 promoter repressed glucose-containing media. Despite pre- <br /> sence glucose rich medium synthetic dropout (SD) <br /> plates, GAL1 promoter fully repressed  <br /> firmed low levels NGT1 NAGK mRNA transcripts cells qPCR. cells grown nonrepressive, galactose- <br /> containing media (YPGAL &#195;&#190; GlcNAc) observed apparent <br /> toxicity Ngt1 gna1&#206;&#8221; cells (Fig. 2). Wild-type cells grown YPGAL &#195;&#190; GlcNAc selective pressure retain NGT1 plasmid able grow YPGAL medium (Fig. 2). observations suggest high <br /> level NGT1 expression toxic S. cerevisiae. <br />  <br /> Incorporation Synthetic Sugars Glycoproteins. Significantly, <br /> exogenous salvage pathways serve convenient routes  <br /> troduce nonnatural sugar analogs glycans. example, artificial fucose salvage pathway recently exploited install <br /> fucose analogs cell-surface polysaccharides E. coli novo biosynthesis donor nucleotide-sugar <span id='am-10' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span>GDP</span>- <br /> fucose disrupted (31). conducted experi- <br />                                                                               Fig. 3. Introduction GlcNAc analogs cell-surface secreted glyco- <br /> ments determine GlcNAc salvage pathway gna1&#206;&#8221; yeast <br />                                                                               proteins.   Secreted glycoproteins (5 &#206;&#188;g total protein load/lane) gna1&#206;&#8221; assimilate unnatural GlcNAc analogs secreted glyco-                cells expressed medium supplemented GlcNAc GlcNAz proteins.  utilized azide-bearing analog GlcNAz.              labeled phos-FLAG analyzed &#206;&#177;-FLAG immunoblotting. PNGaseF <br /> observed cell proliferation gna1&#206;&#8221; cultures administered               treatment totally removes N-glycans prevents phos-FLAG ligation (lanes pure unnatural sugar supplement severely impaired                  2,4). Samples treated endoH (lanes 6, 8) remain reactive phos-FLAG; cells remained viable continued produce                  proteins detected &#206;&#177;-FLAG blotting collectively downshifted molecu- <br /> protein readily proliferated switched GlcNAc supple-              lar weight deglycosylation. Cell-surface azidosugar-bearing glycans <br /> ment, supplemented GlcNAc/analog mix. increase                readily detected fluorescence microscopy following chemos- <br />                                                                               pecific labeling alk-AF488; GlcNAz-supplemented gna1&#206;&#8221; cells (B) yields secreted protein, adopted strategy growing <br />                                                                               strong reactivity alk-AF488 display morphological abnormalities, <br /> cells midlog phase presence GlcNAc pellet-                observed gna1&#206;&#8221; strain supplemented ing, washing, resuspending analog-supplemented culture                 GlcNAc (C). Similarly, GlcNAl-supplemented gna1&#206;&#8221; cells (D) strongly <br /> medium. Conditioned medium gna1&#206;&#8221; yeast cultures                      labeled N3 -AF647, GlcNAc-supplemented cells (E)  Nuclear <br /> collected desalted remove unmetabolized sugar supple-                  mitochondrial DNA indicated DAPI stain. <br />  <br />  <br /> 3990 &#226;&#710;&#163;   www.pnas.org/cgi/doi/10.1073/pnas.0911247107                                                                                     Breidenbach et al. <br />  modify resident azides (Fig. 3A). labeled glycoproteins       Characterization Ygp1 N-glycans. single, highly glycosylated analyzed immunoblotting &#206;&#177;-FLAG anti-             protein purified conditioned growth medium  <br /> body. Notably, secreted proteins GlcNAz-supplemen-          form precise analysis analog location labeling ef- <br /> ted cultures showed immunoreactivity &#206;&#177;-FLAG, indicating          ficiency. YGP1 encodes secretory glycoprotein unknown <br /> azides present glycoproteins.                     function containing 15 potential sites N-glycosylation (36). <br />    verify azide incorporation limited N-glycan cores,        YGP1 placed control GPD promoter subjected secreted protein fractions digestion         multicopy pRS426 vector C-terminal polyhistidine tag, <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>:N-glycosidase F (PNGaseF) endoglycosidase           transformed gna1&#206;&#8221; yeast, expressed presence H (endoH), specifically target N-glycans (33). Consistent       GlcNAc, isotopically labeled GlcNAc, GlcNAz supplements. N-glycan-specific labeling, PNGaseF-digested proteins lost       Ygp1 affinity purified near-homogeneity assessed reactivity phos-FLAG (Fig. 3A).  secreted glyco-         silver-stained SDS-PAGE (Fig. 4A). purified glycoprotein <br /> proteins reacted phos-FLAG following endoH diges-          subjected endoH digestion remove core <br /> tion. observed overall downshift molecular weight            GlcNAc  GlcNAz) residues, causing significant downshift consistent deglycosylation. data confirm dis-     molecular weight. identity Ygp1 confirmed immu- <br /> played azides located N-glycans glycoproteins      noblotting polyhistidine tag. Ygp1 expressed GlcNAz- core GlcNAc residue glycans carries phos-        supplemented medium (Fig. 4A) highly reactive phos- <br /> phine-reactive azide modification.                                    FLAG fully glycosylated endoH-treated samples <br />    GlcNAc analogs displayed cell-surface glycoproteins        readily detected &#206;&#177;-FLAG blotting. <br /> detected fluorescence microscopy. case GlcNAz- <br /> supplemented yeast, utilized alkyne-functionalized Alexa- <br /> fluor-488 (alk-488) dye chemospecifically modify azides Cu <br />  -catalyzed azide-alkyne cycloaddition (CuAAC) (22). GlcNAz- <br /> supplemented gna1&#206;&#8221; yeast reactive alk-488, GlcNAc-supplemented cells (Fig. 3B, C). Surfaces uniformly intense fluorescence, possibly differences cell age. cases, alk-488 strongly labeled bud-neck <br /> regions cells. Notably, GlcNAz-supplemented cells <br /> appeared display morphological cytokinetic defects; <br /> branched, multicell chains frequently observed cul- <br /> tures (Fig. 3B). abnormalities observe cell shape, size, bud-neck morphology strikingly similar observed strains chitin synthases CHS1 CHS2 deleted (34). contrast, GlcNAc-supplemented gna1&#206;&#8221; cells dis- <br /> played little reactivity alk-488 comparatively normal <br /> morphology (Fig. 3C). small minority GlcNAz-supplemen- <br /> ted/alk-488 labeled cells, observed strongly fluorescent ring- <br /> like features cell surfaces (Fig. S2) consistent bud scars, remnants primary septa seen surfaces mother <br /> cells following division. Bud scars primary repository  <br />  <br />  <br />  <br />                                                                                                                                                         CHEMISTRY <br /> chitin, chitin-specific stain calcofluor white (CW) (35) colo- <br /> calized alk-488 bud scars, possibly indicating presence azide-bearing chitinous deposits. frequently,  observed notable decrease CW fluorescence GlcNAz- <br /> supplemented cells relative supplemented GlcNAc. speculate chitin synthases S. cerevisiae rela- <br /> tively intolerant GlcNAz analog defects chitin <br /> production lead observed morphological abnormalities  <br />  <br />  <br />                                                                                                                                                         GENETICS <br /> decreased viability. <br />    Similarly, cultures supplemented alkyne-bearing GlcNAl        Fig. 4. Analysis GlcNAz incorporation Ygp1.   Polyhis-tagged Ygp1 probed azide-functionalized Alexafluor-647 (N3 -647).       overexpressed GlcNAz-supplemented SD-URA medium purified. <br /> GlcNAl-supplemented gna1&#206;&#8221; yeast reactive N3 -647            Identical (1 nanogram) loads Ygp1 subjected detection silver- displayed strong fluorescence surfaces,            stain (lanes 1<span id='am-22' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>;2), immunoblotting &#206;&#177; (lanes 3&amp;4) &#206;&#177;-FLAG <br /> GlcNAc-supplemented cells did (Fig. 3D, E). GlcNAl-supple-        (lanes 5&amp;6). Film exposure times varied generate visibly comparable <br /> mented cells demonstrated morphological abnormalities            Western blots: lane 3, 1 hour; lane 4, 1 min, lanes 5&amp;6, 1 second. required increased size relative GlcNAc-supplement       differences reflect variability epitope accessibility. molar excess                                                                       EndoHf (silverstain band **) used remove N-glycans heavily <br /> cultures. GlcNAz, increased tendency intercellular     glycosylated Ygp1 (silverstain band *) migrates diffuse <br /> connectivity observed GlcNAl supplement; single          high-molecular weight smear (lanes 1 2). Samples probed cells rarely observed cultures.                         &#206;&#177;-FLAG subjected chemospecific phos-FLAG ligation prior    S. cerevisiae, UDP-GlcNAc known provide GlcNAc       blotting; immunodetection FLAG peptide indicates presence GlcNAz. GPI anchor biosynthesis. GlcNAc coupled phos-         (B) Ygp1 expressed culture medium supplemented photidylinositol step GPI anchor biosynthesis,    GlcNAc 13 C6 -GlcNAc. Ygp1 treated endoH, trypsinized, sub- <br /> C-2 acetamido substituent promptly removed essential        jected ESI-FTICR MS analysis. Masses representative glycopeptide, <br />                                                                       spanning Leu 98-Arg 115 glycosylated potential <br /> deacetylase activity Gpi12 (14). expect                                                                       sites (indicated green), shown. Relative intensities GlcNAc- azide alkyne functionality mature GPI anchors      13 C6 -GlcNAc-modified peptides normalized  analog-supplemented cells, explicitly discount       (C) Ygp1 expressed culture medium supplemented GlcNAz <br /> possibility GlcNAc analogs remain unprocessed Gpi12           subjected ESI-FTICR MS analysis. Masses corresponding activity.                                                             glycopeptide (B) shown. <br />  <br />  <br /> Breidenbach et al.                                                                           PNAS &#226;&#710;&#163;    March 2, 2010 &#226;&#710;&#163;    vol. 107 &#226;&#710;&#163;    9 &#226;&#710;&#163;   3991 <br />     verify nonnatural sugar labeling core N-glycan         hydrogen bonds acetamido group transfer- <br /> positions, endoH-treated Ygp1 samples trypsinized              ase essential activity (39), features pre- resulting peptides examined electrospray ionization         served GlcNAz GlcNAl analogs. Importantly, data Fourier-transform <span id='am-16' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> cyclotron resonance mass spectrometry             explicitly confirm analog incorporation Asn-distal site <br /> (ESI-FTICR MS). Tryptic peptides covering 68% Ygp1               N-glycan core. available biochemical data suggest <br /> sequence successfully identified mass spectra              modifications C-2 acetamido group interfere <br /> samples derived cultures supplemented GlcNAc,                 Alg13&#226;&#710;&#8226;14 OT activities, explicit confirmation N-acetyl-&#194;&#189;13 C6  glucosamine &#195;&#176;13 C6 -GlcNAc&#195;&#382;, GlcNAz (Table S3).     analog incorporation Asn-distal site subject future potential N-glycosylation sites observed         Qualitatively, did observe reduction occupied. Specifically, positions Asn 100, 106, 118, 239,        glycosylation level Ygp1 expressed GlcNAz- <br /> 286 glycosylated, tryptic peptides covering        supplemented medium; expect decreased glycosylation potential sites Ygp1 sequence ob-         GlcNAz strongly inhibited dolichol pathway. <br /> served. tryptic peptide spanning Leu 98-Arg 115 contained potential N-glycosylation sites peptide masses corre-           Conclusion <br /> sponding singly- doubly glycosylated forms          High-efficiency, site-specific replacement GlcNAc unna- <br /> peptide detected, unglycosylated peptide                tural analogs chitobiose cores N-glycans allows reli- <br /> observed. Tryptic peptides spanning Val 116-Lys 124 Glu             able predictable introduction unique functionalities 236-Lys 314 observed fully glycosylated.                     N-glycans little risk perturbing glycoprotein structure <br />    stable isotope GlcNAc used unequivocally verify          function. envision GlcNAc analogs convenient means supplement added culture medium metabolically        reliable introduction fluorophores, anomalous x-ray scat- <br /> incorporated N-glycan cores secreted glycoproteins. Ygp1        terers, crosslinkers, affinity tags directly glycoprotein <br /> expressed cultures supplemented 13 C6 -GlcNAc isotope         surfaces. Larger cargos, including peptides, macromolecules, yielded tryptic glycopeptides expected 6 Da shift          cells, tethered N-glycosylated carrier proteins <br /> occupied glycosylation site (Fig. 4B, Table S3). labeling           uniform, predictable sites attachment. Given utility efficiency 13 C6 -GlcNAc appears extremely high;             yeast expression platform glycoprotein biotherapeutics, <br /> GlcNAc-modified peptides detectable cultures                technique use biotechnology applications  <br /> supplemented 13 C6 -GlcNAc. data indicate         tested azide, alkyne, isotopic GlcNAc <br /> possible achieve global replacement GlcNAc unnatural         analogs study, anticipate gna1&#206;&#8221; yeast able <br /> isotopes gna1&#206;&#8221; strains.                                              assimilate additional nonnatural sugars N-glycans, sub- <br />    Similarly, Ygp1 expressed presence GlcNAz ana-         ject future  believe targeted metabolic labeling lyzed ESI-FTICR MS determine high levels non-          N-glycans unnatural sugars afford numerous unique <br /> natural GlcNAc analog occupy N-glycan core positions.           applications previously possible, ultimately facilitat- <br /> case, cultures grown midlog phase GlcNAc supple-         ing studies glycoprotein structure mechanism. <br /> ment switching GlcNAz slow doubling time gna1&#206;&#8221; cells GlcNAz-supplemented medium. resulting             Materials Methods <br /> Ygp1 glycopeptide masses observed consistent             Preparation NAGK, NGT1, YGP1 Expression Vectors. Details pertaining mix GlcNAc GlcNAz occupying core N-glycan posi-              plasmid construction included SI Text. <br /> tions (Fig. 4C, Table S3). mixture GlcNAc GlcNAz <br /> modifications likely results cytosolic reservoir GlcNAc       Isolation gna1&#206;&#8221; Strains Viability Assays. Standard rich SD media <br />                                                                         formulations lithium acetate transformation protocols used persists cells resuspended GlcNAz-supplemented <br />                                                                         gna1&#206;&#8221; cells supplemented 100 &#206;&#188;M GlcNAc  GlcNAz) liquid <br /> culture medium. Unfortunately, possible draw quan-         culture. Solid medium supplemented 200 &#206;&#188;M GlcNAc. heterozy- <br /> titative conclusions GlcNAz&#226;&#710;&#182;GlcNAc ratio               gous GNA1/gna1&#206;&#8221; strain (ATCC#4025635; MAT MAT &#206;&#177;his3&#206;&#8221;&#226;&#710;&#8226;his3&#206;&#8221;leu2&#206;&#8221;&#226;&#710;&#8226; <br /> glycopeptides detected mass spectrometry potential dif-       leu2&#206;&#8221;lys2&#206;&#8221;&#226;&#710;&#8226; &#195;&#190; met15&#206;&#8221;&#226;&#710;&#8226; &#195;&#190; ura3&#206;&#8221;&#226;&#710;&#8226;ura3&#206;&#8221;yf l017c::KanMX4&#226;&#710;&#8226;&#195;&#190;) trans- <br /> ferences ionization efficiency.  likely expres-   formed GAL1-NGT1 GPD-NAGK plasmids (Table S1) sporu- <br /> sion conditions optimized thoroughly deplete cells       lated 2% potassium acetate. Haploid segregants grown overnight internal GlcNAc reservoirs switching non-        media (SD-URA SD LEU) select plasmids. Cells diluted <br /> natural sugar supplement, enabling extremely high labeling         107 mL serially diluted fold. Approximately 5 &#206;&#188;L spotted <br /> efficiencies.                                                           solid media. Cells allowed grow 2&#226;&#8364;&#8220;4 d 30 &#194;&#176;C                                                                         photographed. <br /> Implications Promiscuity Dolichol Pathway. Collectively, <br />                                                                         Microscopy. Procedures used imaging fluorophore-labeled cells data indicate enzymes involved early steps yeast <br />                                                                         described SI Text. <br /> N-glycan biosynthesis tolerate subtle chemical modifications C-2 acetamido moiety GlcNAc.  present strong          Western Blots. Protein electrophoresis performed Criterion tris-HCl <br /> evidence acetamido group Asn-proximal GlcNAc            4&#226;&#8364;&#8220;12% gels (BioRad). Polyhistidine-tagged proteins detected modified correctly processed phosphoglycosyl-           &#206;&#177;-5xHis-peroxidase conjugate kit (Qiagen) specified product literature. <br /> transferase Alg7 later oligosaccharyl transferase         Synthesis detection reagent phos-FLAG described previously (40). <br /> sembly (OT). oligosaccharyl transfer, believed         Fully secreted yeast glycoproteins obtained conditioned medium C-2 acetamido group GlcNAc acts stabilize           small-scale (5 mL) cultures centrifugation remove cells. Proteins oxonium intermediate prior nucleophilic attack            simultaneously concentrated buffer exchanged PBS using Amicon <br /> receiving Asn residue (37); removal substantial electronic       10 kDa net molecular weight cutoff (NMWCO) centrifugal filters (Millipore). <br /> turbation acetamido group, Asn-distal       Total protein concentration measured colorimetric DC protein <br />                                                                         assay (BioRad). Glycoprotein samples labeled Staudinger ligation <br /> GlcNAc, effectively blocks OT activity (38). azido al- <br />                                                                         (21) 12 h, room temperature incubation 500 &#206;&#188;M phos-FLAG. <br /> kynyl additions C-2 acetamido group obviously increase           FLAG-conjugated proteins detected &#206;&#177;-FLAG M2-peroxidase anti- <br /> size, believe modifications interfere      body (Sigma-Aldrich). Peroxidase-conjugated antibodies detected oxonium stabilization transfer. Modifications C-2         chemiluminescence using SuperSignal West Pico substrate (Pierce). <br /> acetamido group Asn-proximal GlcNAc impact activity Alg13&#226;&#710;&#8226;14 enzyme complex transfers         Protein Expression Purification. Procedures used protein expression, <br /> Asn-distal GlcNAc Asn-proximal GlcNAc. Specific                purification enzymatic deglycosylation described SI Text. <br />  <br />  <br /> 3992 &#226;&#710;&#163;   www.pnas.org/cgi/doi/10.1073/pnas.0911247107                                                                                 Breidenbach et al. <br />  Chemical Synthesis. Syntheses GlcNAz GlcNAl described                          ACKNOWLEDGMENTS. wish extend gratitude Dr. T. Starr, <br /> SI Text.                                                                                     J. Baskin, S. Hubbard, Dr. J. Seeliger, Dr. M. Boyce technical assistance, <br />                                                                                              reagents, helpful discussion. work supported National Insti- <br /> Mass Spectrometry. Highly purified Ygp1 treated endoH (New Eng-                     tutes Health Grant GM066047  C.R.B.) . J.E.G.G supported NSF <br /> land Biolabs) deglycosylated products separated SDS-PAGE.                    postdoctoral fellowship DBI-0511799. <br /> Desalted, gel tryptic digests Ygp1 samples analyzed ESI-FTICR <br /> MS (Bruker, 9.4 T magnet). <br />  <br />  1. Link AJ, Mock ML, Tirrell DA (2003) Non-canonical amino acids protein engineering.    23. Chang PV, et al. (2009) Metabolic labeling sialic acids living animals alkynyl <br />     Curr Opin Biotechnol, 14:603&#226;&#8364;&#8220;609.                                                            sugars. Angew Chem Int Ed Engl, 48:4030&#226;&#8364;&#8220;4033. <br />  2. Wang L, Schultz PG (2004) Expanding genetic code. Angew Chem Int Ed Engl,            24. Milewski S, Gabriel  Olchowy J (2006) Enzymes UDP-GlcNAc biosynthesis yeast. <br />     44:34&#226;&#8364;&#8220;66.                                                                                    Yeast, 23:1&#226;&#8364;&#8220;14. <br />  3. Campbell CT, Sampathkumar SG, Yarema KJ (2007) Metabolic oligosaccharide engi-           25. Dummitt B, Micka WS, Chang YH (2005) Yeast glutamine-fructose-6-phosphate <br />     neering: Perspectives, applications, future directions. Mol Biosyst, 3:187&#226;&#8364;&#8220;194.          aminotransferase (Gfa1) requires methionine aminopeptidase activity proper <br />  4. Laughlin ST, Baskin JM, Amacher SL, Bertozzi CR (2008) vivo imaging membrane-          function. J Biol Chem, 280:14356&#226;&#8364;&#8220;14360. <br />     associated glycans developing zebrafish. Science, 320:664&#226;&#8364;&#8220;667.                        26. Yamada-Okabe T, Sakamori Y, Mio T, Yamada-Okabe H (2001) Identification char- <br />  5. Nandi  et al. (2006) Global identification O-GlcNAc-modified proteins. Anal Chem,       acterization genes N-acetylglucosamine kinase N-acetylglucosamine- <br />     78:452&#226;&#8364;&#8220;458.                                                                                  phosphate deacetylase pathogenic fungus Candida albicans. Eur J Biochem, <br />  6. Hanson SR, et al. (2007) Tailored glycoproteomics glycan site mapping using              268:2498&#226;&#8364;&#8220;2505. <br />     saccharide-selective bioorthogonal probes. J Chem Soc, 129:7266&#226;&#8364;&#8220;7267.                 27. Saxon E, et al. (2002) Investigating cellular metabolism synthetic azidosugars  7. Luchansky SJ, Argade S, Hayes BK, Bertozzi CR (2004) Metabolic functionalization          Staudinger ligation. J Chem Soc, 124:14893&#226;&#8364;&#8220;14902. <br />     recombinant glycoproteins. Biochemistry, 43:12358&#226;&#8364;&#8220;12366.                                 28. Vocadlo DJ, Hang HC, Kim EJ, Hanover JA, Bertozzi CR (2003) chemical approach <br />  8. Holden HM, Rayment  Thoden JB (2003) Structure function enzymes              identifying O-GlcNAc-modified proteins cells. Proc Natl Acad Sci USA, <br />     Leloir pathway galactose metabolism. J Biol Chem, 278:43885&#226;&#8364;&#8220;43888.                       100:9116&#226;&#8364;&#8220;9121. <br />  9. Tanner (2005) enzymes sialic acid biosynthesis. Bioorg Chem, 33:216&#226;&#8364;&#8220;228.       29. Mumberg D, Muller R, Funk M (1995) Yeast vectors controlled expression 10. Weerapana E, Imperiali B (2006) Asparagine-linked protein glycosylation: eukar-         heterologous proteins different genetic backgrounds. Gene, 156:119&#226;&#8364;&#8220;122. <br />     yotic prok <br /> </body></html>