<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 10, 2377&#226;&#8364;&#8220;2391, July 1999 <br />  <br />  <br /> Yeast Dam1p Required Maintain Spindle <br /> Integrity Mitosis Interacts Mps1p <br /> Kinase <br /> Michele H. Jones, Jeffrey B. Bachant,* Andrea R. Castillo, <br /> Thomas H. Giddings, Jr., Mark Winey&#226;&#8364;&#160; <br />  <br /> Department Molecular, Cellular, Developmental Biology, University Colorado&#226;&#8364;&#8220;Boulder, <br /> Boulder Colorado 80309-0347 <br />  <br /> Submitted December 28, 1998; Accepted April 26, 1999 <br /> Monitoring Editor: Tim Stearns <br />  <br />  <br />  <br />         identified mutant allele DAM1 gene screen mutations         lethal combination mps1-1 mutation. MPS1 encodes essential protein <br />         kinase required duplication spindle pole body spindle <br />         assembly checkpoint. Mutations different genes lethal combi- <br />         nation mps1-1, DAM1 novel. remaining genes encode         checkpoint protein, Bub1p, chaperone proteins, Sti1p, Hsc82p, Cdc37p,         Ydj1p. DAM1 essential gene encodes protein recently described member <br />         microtubule binding complex. report cells harboring dam1-1 <br />         mutation fail maintain spindle integrity anaphase restrictive tempera- <br />         ture. Consistent phenotype, DAM1 displays genetic interactions STU1, <br />         CIN8, KAR3, genes encoding proteins involved spindle function. ob- <br />         served Dam1p-Myc fusion protein expressed endogenous levels localized <br />         immunofluorescence microscopy, appears evenly distributed short mitotic <br />         spindles spindle poles later times mitosis. <br />  <br />  <br /> INTRODUCTION                                                          Spc97p Spc98p, required mi- <br />                                                                       crotubule nucleation (Knop et al., 1997; Knop mitotic spindle serves segregate replicated                    Schiebel, 1997, 1998; Sundberg Davis, 1997), key <br /> chromosomes progeny cells cell division.                    function SPB. Centrosome duplication crit- sophisticated microtubule-based structure                  ical cell cycle&#226;&#8364;&#8220;regulated event required forma- <br /> ganized spindle poles opposing sides                   tion bipolar spindle. budding yeast, SPB nucleus contain centrosomes equivalent                    duplicated late G1 phase cell cycle (Byers <br /> organelles (Kellogg et al., 1994). budding yeast               Goetsch, 1974, 1975). duplication pathway <br /> Saccharomyces cerevisiae, centrosome-equivalent                described morphologically, ganelle spindle pole body (SPB) (Botstein et al.,              genes required process identified 1997; Winsor Schiebel, 1997). SPB multi-                 mutation (Winey Byers, 1993; Botstein et al., 1997). <br /> layered organelle, remains embedded                      genes required SPB duplication nuclear envelope yeast cell cycle                   MPS1  monopolar spindle), encodes es- <br /> ers Goetsch, 1975). SPB contains g-tubulin                    sential protein kinase (Winey et al., 1991; Lauze et al., <br />  product TUB4 gene) associated                     1995). Analysis strains containing different mutant <br />                                                                       alleles MPS1 reveals distinct SPB duplication <br />   * Present address: Department Biochemistry, Baylor College    defects, suggesting Mps1p dis- <br />     Medicine, 1 Baylor Plaza, Houston, TX 77030.                      tinct functions SPB duplication (Schutz Winey, <br />   &#226;&#8364;&#160; <br />     Corresponding author. E-mail address: Mark.Winey@Colorado. <br />     edu. <br />                                                                       1998). <br />     Abbreviations used: 5-FOA, 5-fluoroorotic acid; GFP, green flu-      SPB duplication, SPBs separate mi- <br />     orescent protein; SPB, spindle pole body.                         totic spindle assembled. Later cell cycle,  <br />  <br /> &#194;&#169; 1999 American Society Cell Biology                                                                            2377 <br />  M.H. Jones et al. <br />  <br />  <br /> spindle segregates chromosomes anaphase           distinct functions SPB dupli- anaphase B movements (Winey et al., 1995;               cation spindle checkpoint activation (Geiser et al., <br /> Straight et al., 1997). number proteins required   1997). Finally, Mps1p hybrid processes, including local-      coimmunoprecipitation experiments bind <br /> ized spindle spindle pole  review,       Mob1p (Luca Winey, 1998). Mob1p required (Hoyt Geiser, 1996; Botstein et al., 1997).       completion mitosis maintenance include variety microtubule-based molecular mo-        ploidy; function requires Mps1p activity tors, kinesin-related proteins (Cin8p, Kar3p,      related role Mps1p SPB duplica- <br /> Kip1p) dynein (Dhc1p). nonmotor          tion. <br /> spindle components Stu1p, Ase1p, Duo1p             report genes identified required spindle function. Finally, compo-     mutations caused inviability combination nents kinetochore, connect chromo-         mps1-1 mutation. genes encodes <br /> somes spindle, contribute spindle formation       Bub1p, kinase  like Mps1p, required integrity (e.g., Mif2p; Brown et al., 1993). Exten-     spindle assembly checkpoint (Hoyt et al., 1991; Roberts <br /> sive genetic interactions detected        et al., 1994). genes encode molec- <br /> alleles various genes required spindle        ular chaperones, suggesting single kinase  <br /> sembly function.                                        quire different chaperones activity.  <br />    assembly mitotic spindle execu-       identified novel gene encodes essential <br /> tion mitosis strict cell cycle control.   coiled-coil protein. gene, DAM1, indepen- SPB duplication separation SPBs      dently identified screen proteins interact <br /> form spindle require CDC28/CLN activity (Winey          spindle-associated protein, Duo1p (Hofmann et Byers, 1993; Winsor Schiebel, 1997). onset      al., 1998; Figure 6), Dam1p anaphase requires degradation Esp1p           shown bind microtubules vitro. findings <br /> inhibitor Pds1p anaphase-promoting complex           indicate Dam1p required spindle integrity <br /> (Cohen-Fix et al., 1996), exit mitosis requires    anaphase B elongation, protein complete inactivation CDC28/CLBs          localized short spindles spindle poles degradation spindle component Ase1p (Juang et        cell cycle. fact mutation MPS1 exhib- <br /> al., 1997). controls executed         genetic interaction allele DAM1 rein- <br /> round cell division serve coordinate activi-      forces idea Mps1p essential roles ties mitotic spindle cell cycle events.   SPB spindle function. distinct set cell cycle controls, termed checkpoint <br /> pathways, inducible function block mitotic <br /> progression DNA spindle damaged          MATERIALS METHODS prerequisite event occurred               Yeast Escherichia coli Culture Genetic <br /> (Elledge, 1996). example, spindle assembly          Techniques <br /> checkpoint pathway delays onset anaphase             yeast strains used study listed Table 1. Yeast spindle function compromised microtu-            growth conditions (including media) yeast genetic techniques <br /> bule-depolymerizing agents mutations af-         described (Guthrie Fink, 1991). strains derive <br /> fect assembly integrity mitotic spindle (Rud-     S288c background. cell synchronization experiments <br />                                                             performed treating cultures (0.4 OD units) 10 mM <br /> ner Murray, 1996; Wells Murray, 1996).           factor  Biological, Swampscott, MA) ;2 h, washing addition role SPB duplication, Mps1p kinase    times, resuspending warmed media. Percentage large required activation spindle assembly        budded cells measured counting cells bud size <br /> checkpoint, likely phosphorylation          greater half size mother cell. Depolymerizaton Mad1p, constituent signaling pathway        microtubules performed described (Hardwick et al., 1996). <br />                                                             Samples prepared 3 h media containing benomyl <br /> (Hardwick et al., 1996; Weiss Winey, 1996).        (DuPont, Wilmington,  nocodazole  Biological) flow <br /> far, conditional mutations MPS1 lie kinase    cytometry (60% large budded; unpublished results) immu- <br /> domain, affect SPB duplication         nofluorescence described  checkpoint functions (Schutz Winey, 1998). <br />    endeavored understand Mps1p               Synthetic Lethal Screen Gene Cloning <br /> functions dual roles identifying factors     Mutant strains require MPS1 normally permissive   interacts, genetically physically.    bient, ;23&#194;&#176;C) mps1-1 temperature (mps1-1 enhancer [moe]) previously reported MPS1 molecular             derived strains, 12a, 15b, 7d (Table 1). strains <br /> chaperone gene CDC37 variety genetic           contain mps1-1 ts allele mutant alleles ade2, ade3, teractions Mps1p requires Cdc37p           ura3. harbor plasmid yeast markers ADE3                                                             URA3 wild-type copy MPS1. ADE3 URA3 yeast <br /> kinase activity (Schutz et al., 1997). MPS1 shows      markers allowed assess requirement strains genetic interactions CIN8, spindle motor gene,       wild-type copy MPS1 using <span id='am-12' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> red&#226;&#8364;&#8220;white sectoring <br /> suggesting potential role Mps1p spindle        assay (Bender Pringle, 1991) resistance 5-fluoroorotic <br />  <br />  <br /> 2378                                                                                              Molecular Biology Cell <br />                                                                                                           mps1-1 Synthetic Lethal Mutations <br />  <br />  <br />  <br /> Table 1. Yeast strains <br />  <br /> Strain                                                                        Genotype <br />  <br /> 7d                         MATa, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 12a                        MATa, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b                        MATa, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 12a30-69                   MATa, moe1, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 12a30-70                   MATa, moe2, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b20-11                   MATa, moe2, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-5                    MATa, moe7, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801(MPS1-URA3,ADE3,CEN) <br /> 15b45-68                   MATa, moe13, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-188                  MATa, moe21, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-279                  MATa, moe27, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-341                  MATa, moe34, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-69                   MATa, moe14(1), mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-248                  MATa, moe25, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-298                  MATa, moe29, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-69                   MATa, moe14(2), mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b30-4                    MATa, moe4, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-59                   MATa, moe12, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-156                  MATa, moe15, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-240                  MATa, moe24, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-262                  MATa, moe26, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-295                  MATa, moe28, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-335                  MATa, moe33, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 15b45-350                  MATa, moe35, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801 (MPS1-URA3,ADE3,CEN) <br /> 7d45-1                     MATa, moe37, mps1-1, ade2, ade3, his3D200, leu2-3,112, ura3-52, lys2-801, (MPS1-URA3,ADE3,CEN) <br /> WX241-20c(295)             MATa, mps1-1, ura3-52, his3D200 <br /> DB8WX257-15ca              MATa, mps1D::HIS3, ura3-52, leu2-3,112, trp1D1, his3D200 (MPS1-URA3,CEN) <br /> WX257-17c(371)             MATa, ura3-52, leu2-3,112, trp1D1, his3D200 <br /> D8bX5cA(375              MATa/MATa, ura3-52/ura3-52, leu2-3,112/leu2-3,112, trp1D1/trp1D1, his3D200/his3D200 <br /> MJYM14-9b(889)             MATa, dam1-1, ura3-52, leu2-3,112, trp1D1, his3D200 <br /> moe1b/1d (908)             MATa/MATa, dam1-1/dam1-1, ura3-52/ura3-52, leu2-3,112/leu2-3,112, TRP/trp1D1, HIS3/his3D200 <br /> MHJ19-5b(1325)             MATa, dam1-1, mad1D2::URA3, ura3-52, leu2-3,112, his3D200, ade2-1 <br /> MHJ39-8c(1491)             MATa, dam1-1, mad2D::URA3, ura3-52, leu2-3,112, trp1D1 <br /> MHJ39-8c(1495)             MATa, dam1-1, mad3D::URA3, ura3-52, leu2-3,112, trp1D1, his3D200, ade2-1, can1-100 <br /> MHJ22-1b(1341)             MATa, dam1-1, cin8D::HIS3, ura3-52, leu2-3,112, trp1D1, his3D200, lys2-801, cyh2R, can1-100 (DAM1-URA3-CEN) <br /> MHJ38-2a(1485)             MATa, dam1-1, kar3D::LEU2, ura3-52, leu2-3,112, his3D200 (KAR3-URA3-2m) <br /> MHJ37-8b(1547)             MATa, DAM1-MYC::HIS3, SPC42-GFP::TRP, SPC42D::LEU2, ura3-52, leu2-3,112, his3D200, trp1-1 <br /> 8bX15a-1(1549)             MATa/MATa, DAM1-MYC::HIS3/DAM1-MYC::HIS3, SPC42-GFP::TRP/SPC42-GFP::TRP, <br />                             SPC42D::LEU2/SPC42D::LEU2, ura3-52/ura3-52, leu2-3,112/leu2-3,112, his3D200/his3D200, trp1-1/trp1-1 <br /> MHJMM (1345)               MATa/MATa, DAM1-MYC::HIS3/DAM1-MYC::HIS3, ura3-52/ura3-52, leu2-3,112/leu2-3,112, his3D200/his3D200, <br />                             TRP/trp1D1 <br /> MHJDDAM1 (950)             MATa/MATa, dam1D::HIS3/DAM1, ura3-52/ura3-52, leu2-3,112/leu2-3,112, his3D200/his3D200, trp1D1/trp1D1, <br /> WX257-14c(372)             MATa, ura3-52, leu2-3,112, trp1D1, his3D200 <br />  strains study aLauze et al. (1995). <br />  <br />  <br />  <br />  <br /> acid (5-FOA, suicide substrate strains URA3 gene         mps1D strain (DB8WX257-5C; Table 1) supported URA3- <br /> product; Boeke et al., 1987). Strains 12a, 15b, 7d mu-         based plasmid harboring wild-type copy MPS1. resulting <br /> tagenized using methane sulfonic acid ethyl ester (Sigma, St. Louis,    diploids struck 5-FOA-containing medium mps1-1 <br /> MO) 40 50% viability (Guthrie Fink, 1991).           permissive temperature determine required Mutagenized cultures plated rich (YPD) media, 93,000        wild-type copy MPS1. diploids crosses <br /> colonies screened solid red colony phenotype      5-FOA sensitive, indicating additional mutation indicate strain unable lose wild-type copy MPS1.      mps1-1 generated chromosomal null; mps1D strains <br /> Seven eighty-seven (0.8%) nonsectoring colonies fur-       behave phenotypically moe strains nonsectoring ther analyzed requirement wild-type MPS1 checking      5-FOA sensitive. 5-FOA sensitivity. strains           nature moe mutations assessed crossing sistently nonsectoring 5-FOA resistant. demonstrate          strains mps1-1 strain (WX241-20c; Table 1). 13 mps1-1 phenotypes requirement strains       homozygous, moe heterozygous diploids 5-FOA resistant, sug- <br /> MSP1, LYS2-based plasmid containing MPS1 transformed              gesting moe mutations recessive. determine strains, 22 subsequently shown sector   synthetic lethal phenotype original moe strains <br /> 5-FOA resistant.                                                        mutation single gene, diploids sporu- <br />    distinguish bona fide moe mutation creation     lated, resulting tetrads dissected. Spores analyzed mps1 chromosomal null strains, crossed     viability 5-FOA medium. 13 moe strains, <br />  <br />  <br /> Vol. 10, July 1999                                                                                                                      2379 <br />  M.H. Jones et al. <br />  <br />  <br />                                                                           Young, 1991), HIS3 gene, 39 untranslated region Table 2. moe mutations                                                    DAM1 176 1148. Histidine prototrophs selected                                                                           analyzed PCR amplification confirm correct integration Complementation                                 Previously identified     Western analysis confirm protein expression described (Luca <br /> group                           Members                genesa             Winey, 1998), using mouse anti-myc 1&#194;&#176; antibody (1:1000; <br />                                                                           Santa Cruz Biotechnology, Santa Cruz, CA) sheep anti-mouse                         dam1-1                YGR113W/DAM1              2&#194;&#176; antibody conjugated HRP (1:20,000; Amersham Pharmacia <br /> II                        moe2                  YDJ1                      Biotech, Piscataway, NJ). <br /> III                       moe3, moe7, moe13     CDC37 <br /> IV                        moe21, moe27, moe34   STI1 <br /> V                         moe14(1), moe25       HSC82                     Cytological Techniques <br /> VI                        moe29                 BUB1                      Flow cytometric analysis cells performed described using <br /> VII                       moe14(2)              identified        DNA stain propidium iodide (Sigma) (Hutter Eipel, 1979). <br /> VIII                      moe4, moe12, moe15,   mps1 null alleles         Samples analyzed Becton Dickinson (Mountain View, <br />                             moe24, moe26,                                 CA) FACScan flow cytometer using CELL QUEST software ob- <br />                             moe28, moe33,                                 tain analyze data (BDIS, San Jose, CA). <br />                             moe35, moe37                                     Fixation techniques immunofluorescence microtubules  <br />                                                                           ing 1&#194;&#176; monoclonal antibody YOL1/34 2&#194;&#176; anti-rat conjugated     text literature citations.                                    FITC Texas Red (Accurate <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> Scientific, Westbury, <br />                                                                           NY) visualization DNA using DAPI stain performed                                                                           described (Pringle et al., 1991). techniques used visualize <br />                                                                           Dam1p-myc involved technique described microtu- tetrads segregated 2:2 synthetic lethality, suggesting   bules <span id='am-17' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>&#226;&#8364;&#8220;methanol&#226;&#8364;&#8220;acetone technique described <br /> moe phenotype mutation single gene.                     (Chial et al., 1998), second case, using mouse anti-myc 1&#194;&#176; <br />    Complementation analyses carried determine             antibody (1:300; Santa Cruz Biotechnology) sheep anti-mouse <br /> number genes represented 13 remaining moe strains.              2&#194;&#176; antibody conjugated Texas Red FITC (1:800; Jackson Immu- moe strains identified mating        noResearch, West Grove, PA). Standard fluorescence microscopy <br /> type, strains crossed mps1-1 strain (WX241-20c; Table       carried using Zeiss (Thornwood, NY) fluorescence <br /> 1) obtain mutation opposite mating type. Spores          microscope Empix charge-coupled device camera sulting cross 5-FOA sensitive,          Metamorph Software (Universal Imaging, Westchester, PA) tained mps1-1 moe mutations, analyzed         Leica DMRXA/RF4/V automated microscope Cooke Sensi- <br /> mating type. MATa MATa moe mutant strains inter-            Cam digital camera Slidebook software (Intelligent Imaging <br /> crossed, resulting diploids replica plated 5-FOA           Innovations, Denver, . Yeast cells prepared sec- <br /> dium determine moe strains complement            tioning described (Byers Goetsch, 1991). Serial sections  analysis revealed seven genes, known       viewed Philips CM10 electron microscope (Philips Elec- <br /> (Table 2).                                                                tronic Instruments, Mahwah, NJ). <br />    Complementation groups  II, III (Table 2) cloned complementation ts phenotype respective mutation transformation strains LEU-based centromeric         RESULTS <br /> yeast genomic library  gift C. Connelly P. Heiter, Uni- <br /> versity British Columbia, Vancouver, Canada)           Synthetic Lethal Screen known genes, DAM1, YDJ1, CDC37. ensure             screen mutations lethal combination <br /> cloned gene responsible mps1-1 synthetic lethal pheno-        mps1-1 mutation conducted identify <br /> type, determined corresponding null strain   <br /> did complement original ts strain. Genes representing             genes products interact Mps1p kinase <br /> complementation groups IV, V, VI cloned restoration           (moe 5 mps enhancer). restrictive temperatures sectoring 5-FOA resistance phenotypes transfor-         mps1-1 allele defective SPB duplication <br /> mation LEU-based centromeric yeast genomic library.          (Winey et al., 1991)  subsequently, spindle <br /> gene corresponding moe142 isolated (VII; Table 2). <br />                                                                           assembly checkpoint (Weiss Winey, 1996).                                                                           mutation allele lies kinase domain DAM1 Plasmids, DAM1 Gene Disruption,                                      greatly decreases protein kinase activity vitro myc-DAM1                                                              (Schutz Winey, 1998). Mutations lethal combi- <br /> Multicopy plasmids containing DAM1 gene constructed           nation mps1-1 permissive temperature inserting 5-kb XbaI&#226;&#8364;&#8220;SalI fragment containing DAM1 gene plasmids pRS424 pRS426 (Sikorski Hieter, 1989) cut           initially identified plasmid-sectoring assay based <br /> Spe1&#226;&#8364;&#8220;SalI. precise deletion DAM1 gene replacement            adenine biosynthetic pathway (Bender HIS3 gene step gene replacement (Baudin          Pringle, 1991). mutagenisis  MATERIALS <br /> et al., 1993) wild-type diploid strain 375 (Table 1) PCR        METHODS) nonsectoring solid red colonies <br /> product containing 42 bp DAM1 gene flanking sequences               selected candidates appeared HIS3 gene. Histidine prototrophs selected, correct integration confirmed using PCR amplification             require MPS1-containing plasmid. inability (Luca Winey, 1998). haploid strain containing DAM1 null         solid red colonies lose wild-type copy allele obtained sporulation dissection. Oligonucleotide        MPS1 confirmed failure grow primers purchased Life Technologies (Gaithersburg,              5-FOA medium selects cells harboring MD). strain containing myc-DAM1 fusion gene  <br /> structed transformation wild-type strain PCR product         plasmid-borne URA3 gene (Boeke et al., 1987). Finally, <br /> containing following sequence: 280 bp stop codon          demonstrate nonsectoring 5-FOA sen- DAM1 gene, triple repeat myc epitope (Kolodziej       sitivity phenotypes strains&#226;&#8364;&#8482; require- <br />  <br />  <br /> 2380                                                                                                           Molecular Biology Cell <br />                                                                                           mps1-1 Synthetic Lethal Mutations <br />  <br />  <br /> ment plasmid-borne wild-type MPS1, second         class Hsp70s (Cyr et al., 1992; Cyr, 1995; Cyr MPS1-containing plasmid transformed            Douglas, 1994). strain contained muta- <br /> didate strains, transformed strains            tions synthetically lethal mps1-1 shown sector grow 5-FOA medium. Twen-             mutation HSC82 unidentified <br /> ty independently isolated strains identified       locus. Interestingly, allele BUB1 gene colonies fail sector fail grow   encodes protein kinase involved spin- <br /> 5-FOA medium, phenotypes             dle assembly checkpoint identified screen <br /> restored MPS1 plasmid introduced           (VI; Table 2; Roberts et al., 1994)). allele BUB1, strain.                                            called bub1-656, behaved similarly previously iden- <br />   Genetic analysis 22 original isolates involved     tified bub1 alleles (Hoyt et al., 1991), exhibited <br /> eliminating chromosomal null alleles MPS1           slow growth benomyl sensitivity, suggesting recovered; MATERIALS METHODS),                 checkpoint activity bub1-656 compromised <br /> screening additional phenotypes, segregation         unpublished observation). complementation analysis. 2 13          conditional allele representing final comple- <br /> remaining strains, synthetic lethality segregated    mentation group   Table 2) rescued non- single mutation. studied         permissive temperature uncharacterized  synthetic lethal phenotype         YGR113W gene chromosome VII. constructed mutations gene.         null allele gene exact gene replacement  second aberrantly segregating strain separable      MATERIALS METHODS) mutations identified, lethal      gene essential viability. gene, named DAM1 <br /> combination mps1-1. 13 strains        Duo1p Mps1p Interactor) indepen- temperature-sensitive growth defects.         dently identified screen proteins interact ts phenotype cosegregated synthetic le-        spindle-associated protein, Duo1p. Dam1p thality strains. Complementation analysis      shown localize spindle  <br /> (Table 2) revealed synthetic lethal        expressed bind microtubules vitro (Hofmann <br /> mutations temperature-sensitive growth         et al., 1998). <br /> fects defines separate complementation groups groups contain noncon- <br /> ditional synthetic lethal alleles. remaining non-       dam1-1 Strains Fail Mitosis <br /> conditional synthetic lethal mutations identified      examined cells containing ts dam1-1 mutation <br /> complementation groups, multiple members           (908; Table 1) nonpermissive temperature (Table 2).                                                  understand role Dam1p cell. <br />                                                             2.5 h nonpermissive temperature (37&#194;&#176;C),                                                             significant number dam1-1 cells arrest large <br /> Identification Genes                                 buds, determined budding index (47%; n 5 400 previously reported alleles yeast            cells), 2C DNA content determined CDC37 gene, encodes molecular chaperone,            flow cytometry (Figure 1A). chromatin lethal combination mps1-1 (Schutz et       cells, visualized DAPI staining, separated al., 1997). tested seven       masses approximately equal staining intensity, <br /> complementation groups synthetic lethal mutations        indicating cells begun anaphase elonga- complemented CDC37. group (III;             tion spindle (Figure 1B, DAPI). anaphase, <br /> Table 2) consists conditional noncondi-         dam1-1 cells permissive temperature (Figure 1B, <br /> tional alleles CDC37. remaining genes    25&#194;&#176;C, Tubulin) wild-type cells  unpublished identified complementation using yeast            observation) contain mitotic spindles interdigi- <br /> genomic library  MATERIALS METHODS).                tated microtubules SPB appear Consistent previous findings Mps1p        solid bar microtubules overlapping <br /> kinase requires chaperone function activity,        DAPI staining region. Strikingly, structure genes identified, HSC82, YDJ1, STI1        absent dam1-1 strains incubated nonpermis- <br /> (V, II, IV; Table 2), encode chaperonins. HSC82         sive temperature (Figure 1B, 37&#194;&#176;C, Tubulin); instead, <br /> encodes S. cerevisiae Hsp90 proteins         cells contain distinct microtubule-staining exhibit substrate specificity kinases (Pratt,      regions appear result loss integrity <br /> 1992; Jakob Buchner, 1994; Nathan Lindquist,        mitotic spindle. &#226;&#8364;&#339;broken spindle&#226;&#8364;? pheno- <br /> 1995). STI1 encodes Hsp70p Ssa subclass      type confirmed electron microscopic examina- complex Hsp90p (Chang          tion spindles cells. cells Lindquist, 1994). chaperone identified    contain SPBs, distinct array screen, Ydj1p, encodes dnaJ homologue            nuclear mictrotubules, microtubules shown interact regulate Ssa sub-       interconnected normal spindle (Figure 2  <br />  <br /> Vol. 10, July 1999                                                                                                   2381 <br />  M.H. Jones et al. <br />  <br />  <br />  <br />  <br />                                                                         Figure 2. Electron micrograph asynchronously growing <br />                                                                         diploid dam1-1 strain shift nonpermissive (37&#194;&#176;C) tem- <br />                                                                         perature 2.5 h. Arrows denote spindle pole bodies. Bar, 0.5 mm. <br />                                                                         spindles examined 13 large-budded cells, 11 discon- <br />                                                                         tinuous shown  2 appeared normal. <br />  <br />  <br />  <br />                                                                         staining (Figure 3B, DAPI). Early mitotic spindles, <br />                                                                         visible separation chromatin, appear                                                                         similar wild-type dam1-1 cells (Figure 3B, . <br />                                                                          cells proceed anaphase elongation <br />                                                                         spindle (Figure 3B, b c), spindle remains <br />                                                                         intact wild-type cells breaks dam1-1 cells, <br />                                                                         yielding distinct microtubule-staining arrays                                                                         partially separated chromatin. 3.3 h non- <br />                                                                         permissive temperature, 78% dam1-1 cells large- <br /> Figure 1. Characterization asynchronously growing diploid        budded, majority containing defective spindles <br /> dam1-1 strain permissive temperature (25&#194;&#176;C) shift    quantitation figure legend). Additional exper- nonpermissive (37&#194;&#176;C) temperature 2.5 h.   Flow cytometric     iments conducted determine fate analysis measure DNA content. 1C 2C, DNA content (pro-           cells monitoring budding index wild-type <br /> pidium iodide fluorescence) DNA replication,  <br /> spectively. histogram represents 5000 cells. (B) Immunofluo-       dam1-1 cells 6 h release non- <br /> rescence staining DNA (DAPI) tubulin  Tubulin) cells        permissive temperature (Figure 4). grown 25 37&#194;&#176;C, described MATERIALS METH-               strains start large percentage unbudded cells <br /> ODS.                                                                    treatment factor, wild-type cells <br />                                                                         second increase unbudded cells 4 h, <br /> adjoining serial sections; unpublished observa-                     presumably normal cytokinesis complete <br /> tion; legend quantitation). Consistent                     cell cycle. contrast, unbudded cell population mitotic abnormalities, significant loss viabil-              reappears dam1-1 cells course exper- <br /> ity (.66% lethality return permissive tem-                  iment; instead increase large multibudded <br /> perature 2.5 h nonpermissive temperature)                  cells observed 4 h. population multibud- observed dam1-1 cells.                                           ded cells increases 6 hours, consistent <br />    examined dam1-1 cells proceeding synchro-                         idea dam1-1 cells undergo cyto- <br /> nously cell cycle nonpermissive                      kinesis. Taken  results suggest direct <br /> temperature determine timing development                      role Dam1p anaphase spindle elongation defective spindle phenotype. Wild-type                       role, possibly indirect, cytokinesis. <br /> dam1-1 strains (372 889; Table 1) arrested G1 phase cell cycle permissive tempera-                   dam1-1 Strains Constitutively Require ture treatment factor released                  Spindle Assembly Checkpoint Require G1 arrest nonpermissive temperature                     Mad1p Accumulate Mitotic Cells MATERIALS METHODS). Cell cycle progres-                             Nonpermissive Temperature <br /> sion monitored time quantitating                        described, mps1-1 strains documented  <br /> number large-budded cells (%LB) quantitating                     fects SPB duplication spindle  <br /> DNA content population cells (Figure 3A).                     sembly checkpoint pathway. failure mitotic mitotic spindles cells monitored                   spindle dam1-1 strains suggested genetic <br /> immunofluorescence staining microtubules (Figure                     interaction MPS1 DAM1 reflect <br /> 3B, Tubulin), DNA detected DAPI                        Mps1p&#226;&#8364;&#8482;s role checkpoint, e., normally <br />  <br />  <br /> 2382                                                                                                         Molecular Biology Cell <br />                                                                                         mps1-1 Synthetic Lethal Mutations <br />  <br />  <br />  <br />  <br />                                                                        Figure 3. Characterization synchronized wild- <br />                                                                        type dam1-1 haploid strains shift                                                                        nonpermissive (37&#194;&#176;C) temperature.   Flow cyto- <br />                                                                        metric analysis measure DNA content,                                                                        Figure 1. Asynch, asynchronously growing cells                                                                        25&#194;&#176;C; 0, 2.2, 3.3, hours 37&#194;&#176;C removal                                                                        factor, %LB, percentage cells bud size <br />                                                                        half size mother cell; 200 &#226;&#8364;&#8220; 400 <br />                                                                        cells counted. (B) Immunofluorescence stain- <br />                                                                        ing DNA (DAPI) tubulin  Tubulin)                                                                        wild-type dam1-1 cells grown 37&#194;&#176;C 2.2 h <br />                                                                         b) 3.3 h (c). spindles large- <br />                                                                        budded dam1-1 cells separated DAPI- <br />                                                                        staining regions 2.2 h, 90% discontinuous                                                                        abnormally  b c, 6%                                                                        abnormal (e.g., associated DNA), 4% <br />                                                                        appear normal (n 5 200). <br />  <br />  <br />  <br />  <br /> nonessential checkpoint role Mps1p essential   seven checkpoint genes causes lethality dam1-1 strain. constitutive checkpoint requirement      mutant strain  lesser effect seen postulated spindle defect caused    MAD3D strain; Geiser et al., 1997). mps1-1 <br /> deletion CIN8 gene, mutation    strains checkpoint defect permis- <br />  <br />  <br /> Vol. 10, July 1999                                                                                                     2383 <br />  M.H. Jones et al. <br />  <br />  <br />  <br />  <br />                                                                          Figure 5. DNA content wild-type, dam1-1, dam1-1mad1null <br />                                                                          cells. Left panels (0 h), flow cytometric data cells factor arrest; <br />                                                                          right panels (3 h), flow cytometric data (performed Figures 1 <br />                                                                          3) cells 3 h release factor nonpermissive <br />                                                                          temperature. Thirty percent (n 5 400 cells) dam1-1, mad1D <br />                                                                          cells multibudded time point compared 0%                                                                          wild-type dam1-1 cells. <br />  <br />  <br />  <br />                                                                          mutation nonpermissive temperature activates                                                                          spindle assembly checkpoint. <br /> Figure 4. Budding indices wild-type dam1-1 cells release factor arrest nonpermissive temperature represented bar graph. sample represents 200 &#226;&#8364;&#8220; 400 cells examined.              dam1-1 Mutation Exhibits Genetic Interactions <br />                                                                          Mutations Spindle Proteins <br />                                                                          requirement Dam1p mitotic spindle function <br /> sive temperature (Weiss Winey, 1996), pos-                  led determine dam1-1 mutation exhib- <br /> sible slight checkpoint defect lethal               genetic interactions mutations genes presence dam1-1 mutation. tested                   encoding proteins involved mitosis, addition model making double mutants dam1-1                        genetic interaction MPS1. interactions  <br /> mutation mad1, mad2, mad3 null mutations                     tected dam1-1 mutation alleles genes <br /> (1325, 1491, 1495; Table 1), defec-                encoding integral SPB components (tub4-1, spc98-2, tive spindle assembly checkpoint (Li Murray,                  cmd1-1), kinetochore components (ndc10-42 mif2-3), <br /> 1991). double mutant strains                    proteins act late mitosis (ase1D viable normal growth characteristics, indicating                    mob1-77)  unpublished observations, mutations dam1-1 strains require spindle assembly                  genes encoding mitotic spindle apparatus;  <br /> checkpoint viability permissive temperature.                  view, Botstein et al., 1997). Given common occur- dam1-1 cells nonpermissive temperature                    rence allele-specific interactions, results clearly display short mitotic spindle duplicated,                  rule possibility genetic interactions <br /> unseparated DNA typical cells arrested                    different alleles genes.  spindle assembly checkpoint pathway, possible                 dam1-1 mutation lethal combination mitotic bias observed dam1-1 cells independent                 deletion kinesin-like motor pro- pathway. tested idea comparing syn-                   teins encoded CIN8 KAR3  deletion <br /> chronized dam1-1 dam1-1, mad1 null double mutant                     DYN1) exacerbate stu1-5, mutation mi- <br /> strains 3 h release nonpermissive temper-                 crotubule-binding protein (Table 3). examined dos- <br /> ature (Figure 5). dam1-1 cells previ-                  age suppression dam1-1 STU1, KAR3, CIN8 <br /> ously described mitotic bias (Figure 5, dam1-1), peak               multicopy plasmids stu1-5 cin8D missing cells lacking Mad1p (Figure 5, dam1-1,                 DAM1 multicopy plasmid, detected  mad1D), suggesting defect caused dam1-1                  unpublished observation). Finally, observed <br />  <br />  <br /> 2384                                                                                                              Molecular Biology Cell <br />                                                                                                        mps1-1 Synthetic Lethal Mutations <br />  <br />  <br />                                                                           spindle defect observed dam1-1 mutant strains <br /> Table 3. Summary synthetic interactions dam1-1                    direct role Dam1p spindle. <br />                                        Relative growth                    chromosomal DAM1 gene haploid strain                                                                           epitope tagged integration triple c-myc epitope <br /> Yeast strain            23&#194;&#176;C          30&#194;&#176;C          34&#194;&#176;C         37&#194;&#176;C     39 end  MATERIALS METHODS). <br />                                                                           C-terminally myc-tagged Dam1p detectable dam1-1                    1           1             1             2 <br /> dam1-1,   mps1-1          2           ND            ND            ND <br />                                                                           Western blots  unpublished observation), dam1-1,   cin8D           2           ND            ND            ND      myc-DAM1 strain (1345, Table 1) grows rate indis- <br /> dam1-1,   kar3D           2           ND            ND            ND      tinguishable wild-type strains tempera- <br /> stu1-5                    1           1             1             6       tures checked (15&#226;&#8364;&#8220;37&#194;&#176;C), suggesting tagged <br /> dam1-1,   stu1-5          1           6             2             2       DAM1 allele fully functional. Initial immunofluores- <br /> 1, size colonies similar wild-type control; 6, size colonies   cence experiments using anti-myc antibody indi- <br /> ,20% wild-type control; 2, growth; ND, determined.            cated myc-Dam1p localizes structures sug- <br />                                                                           gestive spindle poles short mitotic spindles <br />                                                                            unpublished observation). verify localiza- dam1-1 strains hypersensitive growth                       tion, myc-DAM1 allele introduced strain <br /> benomyl, microtubule-destabilizing agent  unpub-                    containing SPC42p&#226;&#8364;&#8220; green fluorescent protein <br /> lished results). Genetic interactions DAM1                    (GFP) fusion (Schutz Winey, 1998) visualize <br /> genes encode spindle components,                   SPBs (1549; Table 1), myc-DAM1 strain dam1-1 phenotype vitro microtubule-binding                     doubly stained anti-myc anti-tubulin <br /> activity Dam1p (Hofmann et al., 1998), consistent                  (Tub1p) antibodies visualize mitotic spindles. Using role Dam1p function mitotic                       deconvolution microscopy myc-DAM1, SPC42- <br /> spindle. addition, Stu1p shown interact                    GFP strain (Figure 7), observed myc-Dam1p local- Mps1p 2-hybrid experiments (Luca Winey,                       ization immediately adjacent SPBs single <br /> personal communication), CIN8 shows genetic                        SPB present (Figure 7a) late stage teractions MPS1 (Geiser et al., 1997), providing                   mitosis (Figure 7d). duplicated SPBs connection DAM1 MPS1. ge-                         close  myc-Dam1p clearly localizes netic interactions described genetic                   (Figure 7b), presumably mitotic spindle, hybrid interactions work                       starts separate discrete foci SPBs <br /> (Hofmann et al., 1998; Luca Winey, personal com-                      apart (Figure 7c). quantitate transition, <br /> munication) summarized Figure 6.                                   measured spindle lengths synchronized pop- <br />                                                                           ulation myc-DAM1, SPC42-GFP cells (1547; Table 1). <br />                                                                           Myc-Dam1p present continuous bar length <br /> Dam1p Localized Spindle Peripheral                                                                           1.1&#226;&#8364;&#8220;2.0 mm SPBs (Figure 8, GROUP 1) Spindle Pole <br />                                                                           begins separate discrete foci dis- investigated subcellular localization Dam1p                     tance $1.9 mm (Figure 8, GROUP 2). spindle <br /> protein endogenous levels determine                     localization confirmed myc-DAM1 strain <br />                                                                           (1345; Table 1) immunofluorescence experiments  <br />                                                                           ing myc Tub1p antibodies (Figure 9).                                                                           short spindle stage (Figure 9A, c), myc-Dam1p <br />                                                                           colocalizes tubulin generally reflects microtu- <br />                                                                           bule density. later stages mitosis, myc-Dam1p stain- <br />                                                                           ing appears strongly spindle poles (Figure <br />                                                                           9A, d e) does appear correlate                                                                           microtubule density. experiment,                                                                           able detect punctate nuclear myc-Dam1p staining <br />                                                                           cell cycle using sensitive FITC- <br />                                                                           conjugated secondary antibody myc antibody <br />                                                                            possible SPC42-GFP strain). Treatment <br />                                                                           microtubule-destabilizing drugs nocodozole <br />                                                                           benomyl abolishes majority microtubule                                                                           myc-Dam1p staining (Figure 9B, left panel, right cell), <br /> Figure 6. Summary genetic hybrid interactions involv-          observed punctate nuclear staining ing DAM1. Jagged lines denote hybrid interactions,                absence microtubules (Figure 9B, right panel). straight lines denote synthetic lethal interactions. CIN8-MPS1 <br /> data Geiser et al. (1997). STU1-MPS1 data           cells residual microtubule staining, remains <br /> Luca Winey (personal communication). DAM1-DUO1 data               proportionally reduced myc-Dam1p staining Hofman et al. (1998).                                            (Figure 9B, left panel, left cell). <br />  <br />  <br /> Vol. 10, July 1999                                                                                                                2385 <br />  M.H. Jones et al. <br />  <br />  <br />  <br />  <br /> Figure 7. Localization myc-Dam1p wild-type yeast cells using immunofluorescence microscopy. DAPI, DNA staining; SPC42-GFP, <br /> autofluorescence GFP-tagged version Spc42p; MYC, anti-myc 1&#194;&#176; antibody plus Texas Red anti-mouse 2&#194;&#176; antibody; MERGED, <br /> merging fluorescence SPC42-GFP (green) MYC (red).   unbudded cells. (b c) Budded cells short spindles anaphase B elongation. (d) focal planes large budded cell late anaphase. Bar, 1 mm. <br />  <br />  <br />  <br />  <br /> 2386                                                                                                      Molecular Biology Cell <br />                                                                                                     mps1-1 Synthetic Lethal Mutations <br />  <br />  <br />                                                                        normally permissive temperatures. Similar alleles                                                                        CDC37, strains harboring mutant alleles chap- <br />                                                                        erones exhibit SPB defects. Cell harboring                                                                        ydj1-10 hsf1-82, defective synthesis                                                                        Hsp90p, arrest large buds                                                                        single focus microtubules restrictive tempera- <br />                                                                        tures. Electron microscopic analysis hsf1-82 cells  <br />                                                                        vealed unduplicated SPB enlarged half- <br />                                                                        bridge, reminiscent mps1-1 SPB defect (Zarzov et <br />                                                                        al., 1997). results suggest phenotype                                                                        instability Mps1p caused decreased Hsp90p <br />                                                                        cell. growth sti1 null mutants com- <br />                                                                        promised restrictive temperatures, detailed cytology <br />                                                                        mutants available (Nicolet Craig, <br />                                                                        1989). Based different terminal SPB phenotypes                                                                        cdc37-1 ydj1-10 hsf1-82 mutants, likely                                                                        genes affect different steps Mps1p activation                                                                        define requirement chaperones Figure 8. Transition myc-Dam1p localization foci. <br /> Bar graph spindle lengths cells following patterns   activate maintain Mps1p kinase activity possibly <br /> myc-Dam1p staining. GROUP 1, myc-Dam1p localization unin-        activity kinases. <br /> terrupted bar SPBs (n 5 19; average 5 1.5 mm; SD 5 0.3);      addition requiring molecular chaperones, GROUP 2, myc-Dam1p localization staining regions adjacent    mps1-1 mutation shows synthetic lethality SPBs (n 5 13; average 5 2.4 mm; SD 5 0.3). Spindles mea- <br /> sured using Slidebook software error 60.1 mm. <br />                                                                        checkpoint-defective allele gene encoding                                                                        Bub1p protein kinase (Roberts et al., 1994).  <br />                                                                        ingly, mps1-1 viable combination mu- <br />                                                                        tations checkpoint pathway (Hardwick et al. <br /> DISCUSSION                                                             1996), suggesting synthetic lethality                                                                        requirement checkpoint proteins gen- reported isolation mutations                   eral instead closer link exist distinct classes genes synthetically lethal                MPS1 BUB1 previously suspected. MPS1 mps1-1 mutation. includes genes                BUB1  BUB3, product forms complex <br /> encode molecular chaperones CDC37, YDJ1,                           Bub1p; Roberts et al., 1994) distinct HSC82, STI1. second class defined BUB1,                  checkpoint genes null mutations gene required spindle assembly checkpoint.                   drastic effects cell viability indicative roles final class contains DAM1 gene encodes                    cell addition nonessential checkpoint essential spindle- spindle pole-localized pro-                  functions (Roberts et al., 1994).  mutations tein required mitotic spindle function.                            MPS1 BUB1 synthetically <br />    previously demonstrated Mps1p activity                   lethal deletion allele CIN8 kinesin-like <br /> quires Cdc37p chaperone (Schutz et al., 1997).             motor (Geiser et al., 1997). Regardless molecular <br /> study, isolated conditional nonconditional                 nature interaction, interesting alleles CDC37 lethal combination               kinases appear interact way mps1-1 mutation. Cells containing conditional allele               role spindle assembly checkpoint. CDC37 arrest G1 unreplicated DNA                        class genes reported represented <br /> nonpermissive temperature previously charac-                 DAM1. Dam1p shows homology genes <br /> terized alleles gene (Schutz et al., 1997; Schutz,             S. cerevisiae organisms, does  <br /> Ph.D. thesis). addition CDC37,                     tain coiled coil domains defined COILS <br /> mps1-1 strains require wild-type activity YDJ1, HSC82,              program (Lupas et al., 1991; unpublished observa- STI1 viability. mentioned previously,                 tion), seen structural proteins. DAM1 HSC82-encoded Hsp90p Sti1p                      independently identified hybrid screen macromolecular chaperone complex, Ydj1p interacts                  DUO1 gene, Dam1p shown bind regulates Sti1p (Cyr et al., 1992; Cyr Doug-              microtubules vitro (Hofmann et al., 1998).  <br /> las, 1994; Cyr, 1995). Mutations members chap-               overexpression DAM1  DUO1) results erone complex CDC37 identified                     spindle defects. Consistent data, report <br /> synthetic lethal screens defective kinases,              Dam1p required integrity spindle <br /> cluding kin28-ts3 cdc28-109 (Valay et al., 1995; Zarzov            anaphase B elongation.  uncov- <br /> et al., 1997). possible group chaperones            ered DAM1 using different approach, functions stabilize compromised kinases                mps1-1-based screen designed interactions <br />  <br /> Vol. 10, July 1999                                                                                                             2387 <br />  M.H. Jones et al. <br />  <br />  <br />  <br />  <br />                     Figure 9. <br />  <br />  <br />  <br />  <br /> 2388                            Molecular Biology Cell <br />                                                                                                          mps1-1 Synthetic Lethal Mutations <br />  <br />  <br /> necessary SPB duplication spindle assem-                     spindle spindle pole. interesting test <br /> bly checkpoint functions. eliminated possibil-                    Dam1p phosphorylated, genetic <br /> ity dam1-1 defect permissive tempera-                    interaction indicative kinase&#226;&#8364;&#8220;substrate rela- <br /> ture leads constitutive requirement                         tionship. <br /> checkpoint pathway demonstrating viability                            shown epitope-tagged version double mutants containing dam1-1 null allele                    Dam1p endogenous levels exhibits interesting pat- checkpoint-related genes.                    tern localization cell cycle, associating nonpermissive temperature, dam1-1 cells appear                     short spindles initially moving spindle poles defective maintenance checkpoint arrest                       spindles elongate mitosis. presence state;  activation checkpoint, step                   Dam1p spindle pole likely mediated Mps1p acts, appears intact (Figure 5).                    microtubule-binding activity (Hofmann et al., 1998);     interaction MPS1 DAM1                   idea consistent dramatic decrease staining <br /> probably role Mps1p SPB duplication                    depolymerization microtubules genetic  uncharacterized role spindle                     teractions observed DAM1 genes encoding <br /> dynamics. role suggested dem-                   spindle components, Stu1p, Cin8p, Kar3p. <br /> onstrated physical interaction Mps1p                         pattern observe subset seen Stu1p (Luca Winey, personal communication);                          overexpression protein appears bind <br /> Stu1p spindle spindle pole component                   entire spindle spindle poles times identified originally extragenic suppressor                   cell cycle (Hofmann et al., 1998). explanation mutation b-tubulin-encoding gene TUB2 (Pasqua-                    difference increasing protein <br /> lone Huffaker, 1994). addition, noncondi-                     drives association spindle, eventually lead- <br /> tional alleles MPS1 shown syntheti-                   ing observed toxicity cell. Interestingly, <br /> cally lethal deletion CIN8 (Geiser et al., 1997),            Duo1p, binding partner Dam1p, spin- <br /> gene encoding kinesin-like motor protein involved                   dles spindle poles endogenous levels spindle assembly maintenance (Hoyt et al., 1992;                     cell cycle (Hofmann et al., 1998). <br /> Roof et al., 1992; Saunders et al., 1995), &#226;&#8364;&#339;rapid&#226;&#8364;?               Mitotic spindles contain general classes micro- <br /> phase (Straight et al., 1998) anaphase B chromosome                   tubules defined functionally morpholog- <br /> separation (Saunders et al., 1997). shown ge-                    ically (Winey et al., 1995; Straight et al., 1997). class netic interactions dam1-1 mutation mu-                   microtubules interdigitate spindle midzone  <br /> tations CIN8, KAR3, STU1 genes (Figure 6)                     fine core bundle lengthen anaphase spindle favor idea DAM1 interacts MPS1                         elongation. microtubule-associated proteins spindle functions SPB duplication                   yeast, Ase1p, appear localized group <br /> cell cycle. idea tested isolation               spindle microtubules (Pellman et al., 1995). mps1 mutations specific roles.                class spindle microtubules makes kinetochore studies involving localization Dam1p                   fibers. budding yeast, kinetochore fiber comprised <br /> mps1-1 mutant strain shed light molecular                     single microtubule joining SPB chromosome <br /> basis dam1-1, mps1-1 synthetic lethality.               (Peterson Ris, 1976; Winey et al., 1995). observe physical interaction Mps1p                         reported proteins bind specifically kinetochore <br /> Dam1p using hybrid coimmunoprecipitation                       microtubules. far, clear Dam1p says  unpublished observation), suggesting                 specifically associated types physical association exists proteins,               microtubules. terminal broken spindle phenotype likely transient limited context               dam1-1 strains occurs anaphase B elongation, <br />                                                                          consistent potential role Dam1p stabilizing <br />                                                                          region overlap spindle midzone, e., Figure 9 (facing page). Colocalization myc-Dam1p microtu-        class microtubles.  redistribution bules using immunofluorescence microscopy.   DAPI, DNA stain- <br /> ing; MYC, anti-myc 1&#194;&#176; antibody plus FITC anti-mouse 2&#194;&#176; antibody;       wild-type Dam1p entire spindle spindle pole Tub1-GFP, anti-tubulin 1&#194;&#176; antibody plus Texas Red anti-rat 2&#194;&#176; anti-    mitotic progression reminiscent body. granular staining using Texas Red present controls       dynamics kinetochore movement, similar pat- <br /> lacking primary antibody considered background. MERGED,           tern localization seen kinetochore proteins <br /> merging MYC (green) Tub1p fluorescence (red).   Unbud- <br /> ded cell. (b c) Budded cells short spindles anaphase B <br />                                                                          Ndc10p (Goh Kilmartin, 1993), Ndc80p (Wigge et al., <br /> elongation. (d e) Large budded cells late anaphase. (B) myc-      1998), Cse4p (Meluh et al., 1998). SPB-proximal <br /> Dam1p localization treatment cells microtubule-depoly-     staining similar kinetochore proteins Ctf19p (Hy- <br /> merizing agents. DIC (differential interference contrast) shows cell     land et al., 1999) Mif2p (Meluh Koshland, 1997). <br /> outline. MERGED, merging DAPI (blue), MYC (green),              Kinetochore fibers occupy significant space Tub1p (red) fluorescence. left panel shows residual colocaliza- <br /> tion (yellow) myc-Dam1p (green) tubulin (red), right      short spindles early mitosis diminish length <br /> panel shows punctate nuclear myc-Dam1p staining (green) ab-       short remnants remain spindle poles <br /> sence microtubules (red). Bar, 1 mm.                                  later stages mitosis (Winey et al., 1995). fact, <br />  <br /> Vol. 10, July 1999                                                                                                                  2389 <br />  M.H. Jones et al. <br />  <br />  reduction anaphase kinetochore micro-                       Byers, B., Goetsch, L. (1974). Duplication spindle plaques tubule number length documented electron                          integration yeast cell cycle. Cold Spring Harb. Symp. Quant. <br />                                                                             Biol. 38, 123&#226;&#8364;&#8220;131. <br /> microscopy (Winey et al., 1995) occurs spindles ;2 <br /> mm length, showing good correlation spindle                     Byers, B., Goetsch, L. (1975). Behavior spindles spindle <br />                                                                             plaques cell cycle conjugation Saccharomyces cerevisiae. <br /> length observe Dam1p concentrating                       J. Bacteriol. 124, 511&#226;&#8364;&#8220;523. <br /> poles. addition, unusual discontinuous spindle <br />                                                                             Byers, B., Goetsch, L. (1991). Preparation yeast cells phenotype observed dam1-1 mutant strain resem-                       section electron microscopy. Methods Enzymol. 194, 602&#226;&#8364;&#8220; 608. <br /> bles, superficially, described mutation <br />                                                                             Chang, H.C., Lindquist, S. (1994). Conservation Hsp90 mac- MIF2 (Brown et al., 1993).                           romolecular complexes Saccharomyces cerevisiae. J. Biol. Chem. <br /> detected genetic interactions DAM1                          269, 24983&#226;&#8364;&#8220;24988. <br /> NDC10 MIF2. difficult envision                                                                             Chial, H.J., Rout, M.P., Giddings, T.H., Jr., Winey, M. (1998). kinetochore microtubule-binding protein affect                      Saccharomyces cerevisiae Ndc1p shared component nuclear <br /> anaphase elongation, possibility dam1-1                   pore complexes spindle pole bodies. J. Cell Biol. 143, 1789 &#226;&#8364;&#8220;1800. <br /> cause defect spindle structure, resulting tension                 Cohen-Fix, O., Peters, J.M., Kirschner, M.W., Koshland, D. <br /> imbalance spindle manifested broken                     (1996). Anaphase initiation Saccharomyces cerevisiae controlled <br /> spindle phenotype.                                                          APC-dependent degradation anaphase inhibitor <br />   higher-resolution analysis Dam1p localization                    Pds1p. Genes <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Dev. 10, 3081&#226;&#8364;&#8220;3093. necessary reveal exact location mitotic                   Cyr, D.M. (1995). Cooperation molecular chaperone Ydj1 spindles. addition, analysis Dam1p                    specific Hsp70 homologs suppress protein aggregation. FEBS <br />                                                                             Lett. 359, 129 &#226;&#8364;&#8220;132. <br /> necessary understand basis genetic inter- <br /> action dam1-1 mps1-1 mutations.                        Cyr, D.M., Douglas, M.G. (1994). Differential regulation analysis yield clearer understanding                        Hsp70 subfamilies eukaryotic DnaJ homologue YDJ1. J. Biol. <br />                                                                             Chem. 269, 9798 &#226;&#8364;&#8220;9804. <br /> function novel microtubule-binding spindle <br /> protein.                                                                    Cyr, D.M., Lu, X., Douglas, M.G. (1992). Regulation Hsp70 function <br />                                                                             eukaryotic DnaJ homolog. J. Biol. Chem. 267, 20927&#226;&#8364;&#8220;20931. <br />                                                                             Elledge, S.J. (1996). Cell cycle checkpoints: preventing identity <br /> ACKNOWLEDGMENTS                                                             crisis. Science 274, 1664 &#226;&#8364;&#8220;1672. thank D. Drubin, G. Barnes,  Cheeseman communicat-             Geiser, J.R., Schott, E.J., Kingsbury, T.J., Cole, N.B., Totis, L.J., Bhatta- <br /> ing results publication, M. Brown, C. Connelly, P. Heiter, T.        charyya, G.,  L., Hoyt, M  (1997). Saccharomyces cerevisiae <br /> Davis, M. Hoyt, K. Hardwick, T. Huffaker, J. Kilmartin, F. Luca, P.         genes required absence CIN8-encoded spindle motor act Meluh, D. Pellman, E. Schiebel,  Straight reagents, L.            functionally diverse mitotic pathways. Mol. Biol. Cell 8, 1035&#226;&#8364;&#8220;1050. <br /> Pillus, T. Su, C. Troxel, J. Yucel critical reading          Goh, P.Y., Kilmartin, J.V. (1993). NDC10: gene involved chromo- <br /> manuscript, reviewers helpful insights, mem-            segregation Saccharomyces cerevisiae. J. Cell Biol. 121, 503&#226;&#8364;&#8220;512. <br /> bers Winey Drubin&#226;&#8364;&#8220;Barnes laboratories helpful dis- <br /> cussion. work supported fellowships National           Guthrie, C., Fink, G.R. (1991). Guide Yeast Genetics Institutes Health (GM-19566 R.C.) Cancer League          Molecular Biology, vol. 194. Methods Enzymology, San Diego: <br /> Colorado  J.B.B.) grants National Institutes          Academic Press. <br /> Health (GM-51312) National Science Foundation (MCB-                 Hardwick, K.G., Weiss, E., Luca, F.C., Winey, M., Murray, W. <br /> 09357033). Deconvolution microscopy possible,              (1996). Activation budding yeast spindle assembly checkpoint gift Virginia Mel Clark.                                      mitotic spindle disruption. Science 273, 953&#226;&#8364;&#8220;956. <br />                                                                             Hofmann, C., Cheeseman, M., Goode, B.L., McDonald, K.L., <br /> REFERENCES                                                                  Barnes, G., Drubin, D.G. (1998). Saccharomyces cerevisiae Duo1p <br />                                                                             Dam1p, novel proteins involved mitotic spindle function. <br /> Baudin, , Ozier-Kalogeropoulos, O., Denouel, , Lacroute, F.,        J. Cell Biol. 143, 1029 &#226;&#8364;&#8220;1040. <br /> Cullin, C. (1993). simple efficient method direct gene deletion <br />                                                                             Hoyt, M , Geiser, J.R. (1996). Genetic analysis mitotic Saccharomyces cerevisiae. Nucleic Acids Res. 21, 3329 &#226;&#8364;&#8220;3330. <br />                                                                             spindle. Annu Rev. Genet. 30, 7&#226;&#8364;&#8220;33. <br /> Bender, , Pringle, J.R. (1991). Use screen synthetic <br />                                                                             Hoyt, M ,  L., Loo, K.K., Saunders, W.S. (1992). lethal multicopy suppresser mutants identify new genes <br />                                                                             Saccharomyces cerevisiae kinesin-related gene products required involved morphogenesis Saccharomyces cerevisiae. Mol. Cell. <br />                                                                             mitotic spindle assembly. J. Cell Biol. 118, 109 &#226;&#8364;&#8220;120. <br /> Biol. 11, 1295&#226;&#8364;&#8220;1305. <br />                                                                             Hoyt, M , Totis, L., Roberts, B.T. (1991). S. cerevisiae genes <br /> Boeke, J.D., Trueheart, J., Natsoulis, G., Fink, G.R. (1987). <br />                                                                             required cell cycle arrest response loss microtubule <br /> 5-Fluoroorotic acid selective agent yeast molecular genetics. <br />                                                                             function. Cell 66, 507&#226;&#8364;&#8220;517. <br /> Methods Enzymol. 154, 164 &#226;&#8364;&#8220;175. <br />                                                                             Hutter, K.J., Eipel, H.E. (1979). Microbial determinations Botstein, D., Amberg, D., Mulholland, J., Huffaker, T., Adams, , <br />                                                                             flow cytometry. J. Gen. Microbiol. 113, 369 &#226;&#8364;&#8220;375. <br /> Drubin, D., Stearns, T. (1997). yeast cytoskeleton.  Molecular Cellular Biology Yeast Saccharomyces: Cell Cycle       Hyland, K.M., Kingsbury, J., Koshland, D., Hieter, P. (1999). Ctf19p: Cell Biology, ed. J.R. Pringle, J.R. Broach, E.W. Jones, Cold       novel kinetochore protein Saccharomyces cerevisiae potential link <br /> Spring Harbor, NY: Cold Spring Harbor Laboratory Press, 1&#226;&#8364;&#8220;90.               kinetochore mitotic spindle. J. Cell Biol. 145, 15&#226;&#8364;&#8220;28. <br /> Brown, M.T., Goetsch, L., Hartwell, L.H. (1993). MIF2             Jakob, U., Buchner, J. (1994). Assisting spontaneity: role quired mitotic spindle integrity anaphase spindle elon-          Hsp90 small Hsps molecular chaperones. Trends Biochem. <br /> gation Saccharomyces cerevisiae. J. Cell Biol. 123, 387&#226;&#8364;&#8220; 403.            Sci. 19, 205&#226;&#8364;&#8220;211. <br />  <br />  <br /> 2390                                                                                                                  Molecular Biology Cell <br />                                                                                                                    mps1-1 Synthetic Lethal Mutations <br />  <br />  <br /> Juang, Y.L., Huang, J., Peters, J.M., McLaughlin, M.E., Tai, C.Y.,         Rudner, D., Murray, W. (1996). spindle assembly <br /> Pellman, D. (1997). APC-mediated proteolysis Ase1 mor-              checkpoint. Curr. Opin. Cell Biol. 8, 773&#226;&#8364;&#8220;780. <br /> phogenesis mitotic spindle. Science 275, 1311&#226;&#8364;&#8220;1314.                     Saunders, W., Lengyel, V., Hoyt, M  (1997). Mitotic spindle <br /> Kellogg, D.R., Moritz, M., Alberts, B.M. (1994). centrosome            function Saccharomyces cerevisiae requires balance dif- cellular organization. Annu. Rev. Biochem. 63, 639 &#226;&#8364;&#8220; 674.                  ferent types kinesin-related motors. Mol. Biol. Cell 8, 1025&#226;&#8364;&#8220;1033. <br /> Knop, M., Pereira, G., Geissler, S., Grein, K., Schiebel, E. (1997).   Saunders, W.S., Koshland, D., Eshel, D., Gibbons, R., Hoyt, <br /> spindle pole body component Spc97p interacts gamma-tubu-              M  (1995). Saccharomyces cerevisiae kinesin- dynein-related <br /> lin Saccharomyces cerevisiae functions microtubule organiza-         proteins required anaphase chromosome segregation. J. Cell <br /> tion spindle pole body duplication. EMBO J. 16, 1550 &#226;&#8364;&#8220;1564.                Biol. 128, 617&#226;&#8364;&#8220; 624. <br /> Knop, M., Schiebel, E. (1997). Spc98p Spc97p yeast              Schutz, R. (1997). MPS1 CDC37: Analysis Genes <br /> gamma-tubulin complex mediate binding spindle pole body                 Required Spindle Pole Body Duplication Budding Yeast. interaction Spc110p. EMBO J. 16, 6985&#226;&#8364;&#8220; 6995.                    Ph.D. Thesis. Boulder,  University Colorado. <br /> Knop, M., Schiebel, E. (1998). Receptors determine cellular            Schutz, R., Giddings, T.H., Jr., Steiner, E., Winey. M. (1997). localization gamma-tubulin complex site                yeast CDC37 gene interacts MPS1 required proper <br /> microtubule formation. EMBO J. 17, 3952&#226;&#8364;&#8220;3967.                                  execution spindle pole body duplication. J. Cell Biol. 136, 969 &#226;&#8364;&#8220;982. <br /> Kolodziej, P , Young, R  (1991). Epitope tagging pro-              Schutz, R., Winey, M. (1998). New alleles yeast MPS1 <br /> tein surveillance. Methods Enzymol. 194, 508 &#226;&#8364;&#8220;519.                             gene reveal multiple requirements spindle pole body duplication. <br />                                                                                Mol. Biol. Cell 9, 759 &#226;&#8364;&#8220;774. <br /> Lauze, E., Stoelcker, B., Luca, F.C., Weiss, E., Schutz, R., Winey, <br /> M. (1995). Yeast spindle pole body duplication gene MPS1 encodes            Sikorski, R.S., Hieter, P. (1989). shuttle vectors essential dual specificity protein kinase. EMBO J. 14, 1655&#226;&#8364;&#8220;1663.              yeast host strains designed efficient manipulation DNA                                                                                Saccharomyces cerevisiae. Genetics 122, 19 &#226;&#8364;&#8220;27. <br /> Li, R., Murray W. (1991). Feedback control mitosis budding yeast. Cell 66, 519 &#226;&#8364;&#8220;531.                                              Straight, F., Marshall, W.F., Sedat, J.W., Murray, W. (1997). <br />                                                                                Mitosis living budding yeast: anaphase metaphase <br /> Luca, F.C., Winey, M. (1998). MOB1, essential yeast gene                plate. Science 277, 574 &#226;&#8364;&#8220;578. <br /> required completion mitosis maintenance ploidy. Mol. <br />                                                                                Straight, F., Sedat, J.W., Murray, W. (1998). Time-lapse <br /> Biol. Cell 9, 29 &#226;&#8364;&#8220; 46. <br />                                                                          <br /> </body></html>