<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> microtubule-associated protein, NUD-1, exhibits <br /> chaperone activity vitro <br />  <br /> Abstract Regulation cell division requires concerted           refolding assay utilized. Following thermal denatur- <br /> function proteins protein complexes properly              ation, rabbit reticulocyte lysate <span id='am-1' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> added mediate cytoskeletal dynamics. NudC evolutionarily              luciferase activity measured. presence NUD-1, <br /> conserved protein undetermined function associates            nearly luciferase activity regained, indicating microtubules interacts key regulators           unfolded intermediates complexed NUD-1 mitosis, polo-kinase 1 (Plk1) dynein. NudC             refolded. studies represent functional essential proper mitotic progression, homologs             evidence member mitotically essential protein identified species ranging fungi                 family having chaperone activity facilitates elucida- <br /> humans. paper, report characterization          tion role proteins play chaperone complexes <br /> Caenorhabditis elegans NudC homolog, NUD-1,                      utilized cell division. C. elegans NUD-1 member protein exhibiting molecular chaperone activity.                  evolutionary conserved protein family unknown <br /> NudC/NUD-1 proteins share conserved p23/HSP20                       function involved regulation cytoskeletal dynam- <br /> domain predicted dimensional modeling [Garcia-               ics. NUD-1 mammalian homolog, NudC, function <br /> Ranea, Mirey, Camonis, Valencia, FEBS Lett 529(2&#226;&#8364;&#8220;                     dynein motor complex ensure proper cell <br /> 3):162&#226;&#8364;&#8220;167, 2002]. demonstrate nematode NUD-1                 division, knockdown overexpression able prevent aggregation substrate proteins,         proteins leads disruption mitosis. paper, citrate synthase (CS) luciferase, stoichiometric               NUD-1 possesses ATP-independent chaperone <br /> concentrations.  NUD-1 protects native               activity comparable small heat shock proteins state CS thermal inactivation significantly                cochaperones changes phosphorylation state <br /> reducing inactivation rate enzyme.             functionally alter chaperone activity phosphomimetic <br /> determine NUD-1/substrate complexes productive                NUD-1 mutant. simply &#226;&#8364;&#339;dead-end&#226;&#8364;&#65533; unfolding intermediates, luciferase <br />                                                                       Keywords Chaperone . Dynein . NudC . nud . p23 . Plk1 <br />  <br />                                                                       Abbreviations <br /> L. M. Faircloth : P. F. Churchill : G.  Caldwell :                  CS     citrate synthase <br /> K.  Caldwell (*)                                                    GR     <span id='am-41' about='obo:IMR_0100036' typeof='owl:Thing obo:IMR_0100033 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C05490" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00735" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C02140" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0100036" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="stress hormone" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="stress hormone" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00762" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="glucocorticoid" datatype="xsd:string"></span>glucocorticoid</span> receptor <br /> Department Biological Sciences, University Alabama, <br />                                                                       nud    nuclear distribution <br /> Tuscaloosa, AL 35487, USA <br /> e-mail: kcaldwel@bama.ua.edu                                          Plk1   polo-kinase                                                                       RRL    rabbit reticulocyte lysate <br /> G.  Caldwell : K.  Caldwell                                       SEC    size exclusion chromatography <br /> Departments Neurology Neurobiology, <br />                                                                       sHsps small heat shock proteins <br /> Center Neurodegeneration Experimental Therapeutics, <br /> University Alabama Birmingham,                                  TPR    tetracopeptide repeat <br /> Birmingham, AL 35294, USA                                             UCS    UNC-45/Cro1/She4p <br />  96                                                                                                         L.M. Faircloth et al. <br />  <br />  <br /> Introduction                                                    contains serine residues noted   <br />                                                                 knockdown nud-1 C. elegans results mislocalization mitosis, regulatory proteins assist synchroni-    PLK-1 longer localized centrosomes <br /> zation cellular events association chaperones,      (Faircloth Caldwell, unpublished observation). <br /> molecular motors, motor-associated proteins, kinases,          Notably, homologs NudC, including NUD-1, proteins. proteins       contain domain similar human p23 protein defined, control mitotic cell division remains        HSP20/alpha-crystallin family heat shock proteins <br /> unresolved problem. Studies molecular motors       (Garcia-Ranea et al. 2002). Proteins contain p23 <br /> cargo provided insights mechanism,         core-folding motif bind Hsp90 region (Zhu <br /> functional understanding mitotic &#226;&#8364;&#339;machinery&#226;&#8364;&#65533;             Tytgat 2004; Catlett Kaplan 2006). domain <br /> remains discerned.                                        revealed using combination dimensional     Nearly 30 years ago, genetic screen performed      hidden Markov modeling identified new <br /> fungus, Aspergillus nidulans, identified number nuclear    protein families. new p23-containing proteins <br /> distribution (nud) genes regulate movement          identified study, Sgt1, yeast kinetochore assembly <br /> nuclei, fundamental process growth development        protein, functionally validated Hsp90 <br /> (Morris 1975).  NudC identified      binding partner (Takahashi et al. 2003; Catlett Kaplan <br /> mutation regulated nuclear movement, cell wall             2006). <br /> deposition, colony growth, viability (Osmani et al.            human Hsp90 interactome recently character- <br /> 1990; Chiu et al. 1997). nud loci encode           ized using proteomic techniques, NudC identified <br /> components proposed regulators minus-end-             novel specific binding partner (Te et al. 2007), directed microtubule-dependent motor protein dynein         providing physical evidence predicted p23 domain activator dynactin (Morris 2000). NudC evolutionarily    NudC. general, p23 proteins HSP20 family <br /> conserved, homologs identified species             members  referred small heat shock proteins <br /> including Caenorhabditis elegans, Drosophila,               (sHsps)] exhibit ATP-independent chaperone activity humans.                                                         cooperate ATP-dependent chaperones Hsp70 <br />     Genetic analyses  nidulans shown NudC        Hsp90 (Ehrnsperger et al. 1997; Felts Toft 2003; functionally linked dynein/dynactin motor complex     Haslbeck et al. 2005b). <br /> (Morris 2000; Han et al. 2001). Complementary functional           study, investigate vitro chaperone <br /> studies performed mammalian cells demonstrated          activity C. elegans NUD-1 prototypic NudC <br /> NudC immunoprecipitated dynein/dynactin             example important, functionally undefined, <br /> complex (Aumais et al. 2001). additional NudC                NudC protein family. results study reveal homolog, termed NudC-like (NudCL), binds               NUD-1 does possess ATP-independent chaperone activity. <br /> dynein complex appears involved                tested aggregation, inactivation, refolding <br /> stabilization dynein intermediate chain (Zhou et al.         assays NUD-1 shown activity equivalent 2006). mammalian NudC homologs increased              exceeding Hsp90. <br /> phosphorylation mitosis (Nishino et al. 2006; Zhou <br /> et al. 2006). kinase responsible phosphorylation NudCL identified.  NudC          Materials methods <br /> known phosphorylated Ser-274 Ser-326 mitotic kinase Plk1 (Zhou et al. 2003). Modulating levels    Plasmid constructs site-directed mutagenesis <br /> mammalian NudC knockdown overexpression results mitotic failure mislocalization Plk1 (Aumais et al.   worm total RNA extracted previously <br /> 2003; Zhou et al. 2003).                                        described (Locke et al. 2006), complementary DNA <br />     characterized sole C. elegans NudC              (cDNA) synthesized using SuperScriptIII (Invitrogen) <br /> homolog, NUD-1, reported expression neurons,         oligo(dT)12&#226;&#8364;&#8220;18 primer. nud-1 cDNA isolated <br /> multinucleated tissues hypodermis,           worm cDNA use Gateway Cloning <br /> mitotically dividing cells (Dawe et al. 2001). RNAi-            (Invitrogen) Nud-1GTWY5 Nud- <br /> mediated knockdown nud-1 C. elegans associated         1GTWY3 primers. primers used listed Table 1. embryonic lethality resulting mitotic cell division   Site-directed mutagenesis performed change defects misoriented mitotic spindles, absence     serine residues positions 263 315 <span id='am-24' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span>glutamic acid</span> <br /> midzone microtubules, cytokinesis furrow regression         residues order mimic phosphorylation sites. <br /> (Aumais et al. 2003). <span id='am-17' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>Amino acid</span> sequence alignment             polymerase chain reactions (PCRs) performed NUD-1 NudC reveals NUD-1                  using nud-1 cDNA accomplish mutations.  NUD-1 exhibits chaperone activity vitro                                                                                        97 <br />  <br /> Table 1 Primers used cloning mutagenesis C. elegans nud-     1 mg/ml solution (Buchner et al. 1998). Luciferase 1 pnud-1 <br />                                                                         luciferase assay purchased Promega.                    Primer sequence                                 Oxaloacetate, acetyl-<span id='am-12' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="CoA" datatype="xsd:string"></span>CoA</span>, IgG, dithio-1,4-nitrobenzoic <br />                                                                         acid purchased Sigma BSA Invitrogen. <br /> Nud-1 GTWY 5            ggggacaagtttgtacaaaaaagcaggcttcat <br />                          gtctcaatatgagcgattcgactcgg <br />                                                                         CS inactivation <br /> Nud-1 GTWY 3            ggggaccactttgtacaagaaagctgggtttta <br />                          tccaatttctgaagtccatttcagga <br /> Nud263E Fwd             gaagccagagaacgagaaattgagcgacttgg                Thermal inactivation CS achieved incubating Nud263E Rev             ccaagtcgctcaatttctcgttctctggcttc                native protein 43&#194;&#176;C. CS (15 &#206;&#188;M) diluted 1:100 <br /> mutnud-1 GTWY 3         Ggggaccactttgtacaagaaagctgggtcttatccaattttagc   40 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), 0.1 mM EDTA                                                                         presence absence additional proteins 25&#194;&#176;C. <br /> Primers listed 5&#226;&#8364;&#178;&#226;&#8224;&#8217;3&#226;&#8364;&#178; direction. <br />                                                                         Inactivation initiated shifting sample 43&#194;&#176;C. <br />                                                                         Aliquots taken specific time points, CS activity <br /> PCR used Nud-1GTWY5 Nud263E Rev primers.                            measured according Srere et al. (1963). <br /> Nud263E Fwd mutNud-1GTWY3 primers used generate second PCR product. PCR                  Aggregation assays <br /> products combined serve template PCR. reaction used Nud-1GTWY5 mutNud-                    measure CS aggregation, CS (15 &#206;&#188;M) diluted 1:100 <br /> 1GTWY3 primers resulted pnud-1 cDNA, encoding                    40 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5) equilibrated 43&#194;&#176;C phosphomimetic NUD-1 protein (pNUD-1). PCR                      presence absence chaperones controls. products cloned pDONR221 create Gateway                  NUD-1 pNUD-1 molar concentrations based entry vector. nud-1 pnud-1 introduced                octameric form. Luciferase (10 &#206;&#188;M) diluted destination vector pDEST17 carries N-                      equilibrated 42&#194;&#176;C chaper- <br /> terminal tag. PCR products inserts                     ones controls. monitor kinetics thermal aggrega- <br /> verified sequencing (Northwoods DNA, ).                          tion, light scattering measured PerkinElmer <br />                                                                         LS50-B luminescence spectrophotometer 370 nm Protein expression/purification                                         spectral bandwidth 2.5 nm. <br />  <br /> Expression recombinant NUD-1 pNUD-1                          NUD-1 analysis SEC <br /> carried Escherichia coli strain BL21-AI (Invi- <br /> trogen). Transformed cells induced 0.1%                       Purified NUD-1 pNUD-1 protein samples spun arabinose 37&#194;&#176;C 4 h. Cells spun                     4&#194;&#176;C 10 min 16,000&#195;&#8212;g filtered 0.45-&#206;&#188;m <br /> resuspended 50 mM sodium phosphate (pH 8.0)                      syringe filter. Samples analyzed Superose 6 10/ <br /> taining protease inhibitor cocktail (Roche) frozen               300 GL column (GE Healthcare; 10&#195;&#8212;300 mm; separation <br /> &#226;&#710;&#8217;80&#194;&#176;C. lysed sonication, 50%                      range 5&#226;&#8364;&#8220;5,000 kDa) mobile phase consisting ammonium sulfate added resulting supernatant.                100 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), 150 mM KCl, running Precipitated proteins pelleted centrifugation                   0.5 ml/min 22&#194;&#176;C. Proteins detected absorbance <br /> (15,000&#195;&#8212;g) resuspended dialyzed                      280 nm. <br /> 50 mM sodium phosphate (pH 8.0). proteins purified using select nickel affinity gel (Sigma)                   Luciferase refolding assay <br /> according manufacturer&#226;&#8364;&#8482;s specifications. purification, proteins dialyzed 40 mM 4-               Luciferase refolding assays performed described (2-hydroxyethyl)-1-piperazineethanesulfonic acid (HEPES)&#226;&#8364;&#8220;               Lee et al. (1997) modified Abdulle et al. (2002). <br /> KOH (pH 7.5), protein concentrations determined                Luciferase (0.2 &#206;&#188;M) incubated various concen- bicinchoninic assays using bovine serum albumin (BSA)                trations NUD-1, pNUD-1, IgG refolding buffer standard.                                                          [5 mM MgCl2, 10 mM KCl, 50 mM HEPES&#226;&#8364;&#8220;KOH <br />                                                                         (pH 7.5), 2 mM DTT] 22&#194;&#176;C 42&#194;&#176;C 15 min. <br /> Materials                                                               prevent luciferase adsorption walls, tubes treated <br />                                                                         1 mg/ml BSA 15 min rinsed water. Citrate synthase (CS) porcine heart mitochondria               microliters mixtures added solution pre- <br /> obtained Roche prepared described previously                incubated 30&#194;&#176;C containing 18 &#206;&#188;l rabbit reticulocyte <br /> (Buchner et al. 1998). CS concentration determined                  lysate (RRL; Promega) 2 &#206;&#188;l 0.1 M ATP. various <br /> using published extinction coefficient 1.78                times, 2 &#206;&#188;l sample removed, added 48 &#206;&#188;l  98                                                                                                              L.M. Faircloth et al. <br />  <br />                                                                      Fig. 2 NUD-1 pNUD-1 <br /> 50 mM HEPES&#226;&#8364;&#8220;KOH (pH 7.5), vortexed, diluted 10-                  form large oligomeric com- <br /> fold luciferase assay reagent (Promega). Luciferase             plexes. analyze oligomeriza- <br /> activity measured Promega GloMAX 20/20                    tion state, NUD-1 luminometer.                                                         pNUD-1 (2 &#206;&#188;M) run                                                                      Superose 6 10/300 GL column <br />                                                                      (GE Healthcare) described Statistics                                                           &#226;&#8364;&#339;Materials methods&#226;&#8364;&#65533; <br />  <br /> Statistical analysis chaperone activity performed <br /> using Student&#226;&#8364;&#8482;s t test compare NUD-1 pNUD-1 <br /> endpoint results various assays. Significance determined P&lt;0.05. <br />  <br />  <br /> Results <br />  <br /> Production analysis recombinant NUD-1 pNUD-1 <br />  nud-1 cDNA fused 6X histidine tag N <br /> terminus expressed bacteria length 6X  <br /> NUD-1 protein. purified nickel column referred NUD-1 (Fig. 1). Notably,             determine size altered phosphor- <br /> mammalian NudC phosphorylated highly                   ylation.  utilized size exclusion chromatogra- <br /> served serine residues Plk1; phosphorylated form           phy (SEC) analyze oligomeric state NUD-1 NudC biologically active cell division (Zhou et al.        pNUD-1. NUD-1 forms large oligomer (~300 kDa)  <br /> 2003). conserved sites present C.              sisting NUD-1 monomers (Fig. 2). evident <br /> elegans homolog, NUD-1, residues 263 315.               phosphorylation effect oligomeric state <br /> phosphorylation known effect sHsps,          NUD-1 shift molecular weight <br /> important investigate role phosphorylation         observed pNUD-1 mutant (Fig. 2). subsequent activity NUD-1.  constructed mutant          assays, concentrations NUD-1 pNUD-1 calcu- <br /> mimic phosphorylation sites using site-directed          lated using octameric form. <br /> mutagenesis change conserved serines glutamic <br /> acid residues. resulting mutant phosphomimetic pro-              NUD-1 suppresses aggregation CS luciferase <br /> tein, pNUD-1, produced purified 6X fusion protein (Fig. 1).                                             proteins contain p23 domain <br />    sHsps known form large oligomeric                     vitro chaperone activity, complexes composition altered phos-                sought determine NUD-1 pNUD-1 exhibited <br /> phorylated (Haslbeck et al. 2005b). NUD-1              activity. used CS luciferase substrates size range sHsps 37 kDa, wanted           common assay examines ability chaperone <br /> know capable forming larger complex            inhibit aggregation (Jakob et al. 1995; Herbst et al. 1997; <br />                                                                      Morrow et al. 2006). substrates aggregate high <br />                                                                      temperatures (&gt;40&#194;&#176;C) aggregation monitored light <br />                                                                      scattering. <br />                                                                         Figure 3 shows thermal aggregation CS                                                                      incubation 43&#194;&#176;C 20 min. equimolar concen- <br />                                                                      trations NUD-1 (1&#195;&#8212;), CS aggregation reduced                                                                      93.5&#194;&#177;3.0% (Fig. 3A). high level protection                                                                      maintained concentration NUD-1 Fig. 1 Expression purification NUD-1 pNUD-1. Crude        decreased fourfold (0.25&#195;&#8212;). nearly absolute protection <br /> extract E. coli BL21-AI cells containing pDEST17            compared excess BSA, negative protein control, <br /> expression vector NUD-1 (lane 1) pNUD-1 (lane 3) <br />                                                                      shows high levels aggregation. determine collected 4 h arabinose induction. NUD-1 (lane 2) pNUD-1 (lane 4) purified nickel affinity column chromatog-   potential phosphorylation affect chaperone <br /> raphy analyzed gel electrophoresis Coomassie staining    activity, pNUD-1 tested (Fig. 3B). Imitating <br />  NUD-1 exhibits chaperone activity vitro                                                                                                        99 <br />  <br />  <br />                                                                             stoichiometric concentrations, capable nearly <br />                                                                             complete suppression aggregation, substoichio- <br />                                                                             metric concentrations, NUD-1 exhibits greater chaperone <br />                                                                             activity pNUD-1. <br />  <br />                                                                             Thermal inactivation CS <br />  <br />                                                                             NUD-1 prevent protein aggregation,                                                                             known protected proteins retain activity.                                                                             characterize interactions NUD-1 unfold- <br />                                                                             ed proteins, examined thermal inactivation CS. <br />                                                                             assay measures CS activity remaining <br />                                                                             exposure heat-denaturing conditions presence <br />                                                                             additional chaperone. example, presence                                                                             excess IgG  negative control), CS rapidly lost activity <br />                                                                             incubation 43&#194;&#176;C, decreasing 10%                                                                             5 min (Fig. 5). inactivation reaction followed <br />                                                                             order kinetics inactivation rate 19.2&#195;&#8212;10&#226;&#8364;&#8220;3 s&#226;&#710;&#8217;1 <br />                                                                             (Table 2). contrast, eightfold molar excess NUD-1 <br />                                                                             decreased inactivation rate 88%; similar                                                                             effect Hsp90 molar ratio (Jakob et al. 1995). <br />  <br />  <br /> Fig. 3 NUD-1 pNUD-1 inhibit heat-induced aggregation citrate synthase (CS). CS (0.15 &#206;&#188;M) incubated 43&#194;&#176;C 20 min (filled circle), presence BSA (0.75 &#206;&#188;M, triangle), <br /> NUD-1 (0.15 &#206;&#188;M, open circle; 0.0375 &#206;&#188;M, filled inverted triangle)  pNUD-1 (0.15 &#206;&#188;M, open circle; 0.0375 &#206;&#188;M, filled inverted <br /> triangle) B. Aggregation determined light scattering 370 nm. Data representative trials calculated percentage maximum aggregation CS 20 min trial expressed mean&#194;&#177;SD <br />  <br /> phosphorylation manner effect (P=0.88) chaperone activity NUD-1 1&#195;&#8212; concentration <br /> (93.2&#194;&#177;2.0%).  lower concentration, chaperone activity pNUD-1 significantly lower NUD-1 (82.2&#194;&#177;3.2%; P=0.01). <br />    demonstrate NUD-1 chaperone activity exhibits <br /> general substrate specificity, verified protein able inhibit thermal aggregation luciferase. <br /> Figure 4 shows aggregation luciferase incuba- <br /> tion 42&#194;&#176;C 20 min. Similar effect CS, equimolar NUD-1 decreased aggregation 98&#194;&#177; <br /> 1.4% (Fig. 4A). reduced concentration NUD-1 effective 87.1&#194;&#177;4.8% protection. Incubation pNUD-1 exhibited similar result (Fig. 4B). equimolar concentration, aggregation decreased                                                                             Fig. 4 NUD-1 pNUD-1 inhibit heat-induced aggregation 94.2&#194;&#177;4.22% significantly different NUD- <br />                                                                             luciferase. Luciferase (0.1 &#206;&#188;M) incubated 42&#194;&#176;C 20 min <br /> 1 (P=0.21).  lower concentration, aggrega-                   (filled circle), presence BSA (0.75 &#206;&#188;M,), NUD-1 <br /> tion reduced 69.9&#194;&#177;5.3%, significantly                       (0.1 &#206;&#188;M, triangle; 0.025 &#206;&#188;M, filled inverted triangle)  pNUD- <br /> lower NUD-1 (P=0.01). data correspond                       1 (0.1 &#206;&#188;M, triangle; 0.025 &#206;&#188;M, filled inverted triangle) B. <br />                                                                             Aggregation determined light scattering 370 nm. Data CS aggregation data demonstrate <br />                                                                             representative trials calculated percentage phosphorylation significant effect ability                maximum aggregation luciferase 20 min trial NUD-1 prevent aggregation. NUD-1 pNUD-1,                       expressed mean&#194;&#177;SD <br />  100                                                                                                                      L.M. Faircloth et al. <br />  <br />  <br />                                                                         showed slight increase activity 102%                                                                         addition RRL, stable refolding <br />                                                                         period (active luciferase, Fig. 6).  luciferase <br />                                                                         incubated 42&#194;&#176;C 15 min absence                                                                         chaperones, excess IgG, activity decreased                                                                         10% recovery activity refolding <br />                                                                         period observed (Fig. 6A, B). presence                                                                         equimolar concentration NUD-1, 88.2&#194;&#177;12.2% luciferase <br />                                                                         activity regained; doubling concentration NUD- <br />                                                                         1 did alter activity recovered (Fig. 6A).                                                                         luciferase denatured presence equimolar <br />                                                                         concentration pNUD-1, 54.7 &#194;&#177; 8.0% activity                                                                         regained  NUD-1, increase pNUD-1 <br />                                                                         concentration effect (Fig. 6B).  <br />                                                                         significant difference chaperone activity <br />                                                                         NUD-1 pNUD-1 concentrations (1&#195;&#8212;, <br />                                                                         P=0.02; 2&#195;&#8212;, P=0.04). data, clear                                                                         NUD-1 pNUD-1 maintain heat-denatured lucifer- <br />  <br />  <br /> Fig. 5 NUD-1 pNUD-1 decrease rate heat-induced <br /> inactivation CS. CS (0.15 &#206;&#188;M) incubated 30 min presence 1.5 &#206;&#188;M IgG (filled inverted triangle), 1.2 &#206;&#188;M NUD-1 <br /> (filled circle), 1.2 &#206;&#188;M pNUD-1 (open circle), CS activity measured various time points. enzymatic activity CS expressed percentage initial CS activity <br />  <br />  presence pNUD-1 resulted greater thermo- <br /> protection 94% decrease inactivation rate. data demonstrate NUD-1 pNUD-1 able bind stabilize early folding intermediates CS thermal denaturing conditions. <br />  <br /> NUD-1 maintain heat-denatured luciferase folding <br /> competent state <br />  verify partially unfolded proteins trapped chaperones refoldable state, vitro <br /> luciferase refolding assay employed. assay, <br /> luciferase heat-denatured presence absence chaperones allowed refold lower temper- <br /> ature presence ATP chaperone <br /> machinery present RRL. Luciferase maintained 25&#194;&#176;C <br />                                                                         Fig. 6 Luciferase refolded presence NUD-1                                                                         pNUD-1. Active luciferase (0.2 &#206;&#188;M, filled circle) measured Table 2 Influence various proteins rate thermal            incubation 25&#194;&#176;C indicated times. measurements <br /> inactivation CS 43&#194;&#176;C                                              incubation 42&#194;&#176;C 15 min. Luciferase (0.2 &#206;&#188;M) <br />                                                                         incubated (open circle) presence IgG <br /> Addition                              Rate constant (&#195;&#8212;10&#226;&#710;&#8217;3 s&#226;&#710;&#8217;1)      (2 &#206;&#188;M, filled square). Luciferase incubated NUD-1 <br /> inactivation buffer                      CS thermal inactivationa    (0.2 &#206;&#188;M, triangle; 0.4 &#206;&#188;M, filled inverted triangle). B Luciferase                                                                         incubated pNUD-1 (0.2 &#206;&#188;M, triangle; 0.4 &#206;&#188;M, filled inverted <br /> 1.50 &#206;&#188;M IgG                                          19.2               triangle). refolding step performed 30&#194;&#176;C 120 min 1.20 &#206;&#188;M NUD-1                                         2.3               rabbit reticulocyte lysate (RRL) supplemented ATP, lucifer- <br /> 1.20 &#206;&#188;M pNUD-1                                        1.2               ase activity determined different time points. Data                                                                         representative trials presented percentage  Rate constants obtained inactivation kinetics shown   luciferase activity 15 min incubation 25&#194;&#176;C. Data Fig. 5A.                                                                expressed mean&#194;&#177;SD <br />  NUD-1 exhibits chaperone activity vitro                                                                                     101 <br />  <br />  <br /> ase refoldable state allows refolded       Abdulle et al. 2002). Notably, pNUD-1 displayed chaperones fully active enzyme, NUD-1           high chaperone activity assay, significantly <br /> exhibits greater chaperone activity.                             lower NUD-1, demonstrating regulation NUD-1 <br />                                                                  chaperone activity phosphorylation. <br />                                                                     years ago, NUD-1 predicted contain Discussion                                                       p23/HSP20 domain dimensional modeling pro- <br />                                                                  grams (Garcia-Ranea et al. 2002). interesting note previously unreported ability NUD-1        assays indicate NUD-1 shares function molecular chaperone, multiple substrates      characteristics proteins sHsp family, variety established assays. purified         distinct differences activity does fit protein forms large oligomer (&gt;300 kDa), corresponding         exemplar profile sHsp p23-like protein octamer NUD-1 subunits; characteristic       exclusively. proteins predominantly consist chaperones sHsp family. features       p23/Hsp20 domain known functional <br /> sHsps oligomeric state altered           domains present (Weikl et al. 1999; Stromer et al. 2003). <br /> phosphorylation, changing chaperone activity            NUD-1, p23/HSP20 domain fused <br /> levels (Sun MacRae 2005).  used                 additional domains   likely provide molecular mimicry serine phosphorylation NUD-1             proper dimensional structure required NUD-1/ substituting glutamate study effects oligomer-      NudC activity (Garcia-Ranea et al. 2002). metazoans, <br /> ization chaperone activity (Eidenmuller et al. 2000;         NudC homologs N-terminal extension consisting Longshaw et al. 2004).                                           coiled-coil domains;  proteins share C- <br />    ability inhibit aggregation heat-denatured          terminal region phosphorylated  binds  <br /> proteins characteristic molecular chaperones        Plk1. p23/Hsp20 domain located (Borges Ramos 2005). NUD-1 pNUD-1                   domains  Fig. 7).  surprising <br /> prevented 90% aggregation CS             NUD-1 appears characteristics multiple <br /> luciferase equimolar concentrations. similar   chaperone types prevents aggregation forms <br /> high activity sHsps, high substrate     oligomers similar sHsps (instead monomers like p23), <br /> binding affinity (Lee et al. 1995; Koteiche McHaourab        displays thermal inactivation properties similar 2003). decrease chaperone activity phosphor-       p23  unlike sHsps). distinct combination ylation seen sHSPs (Rogalla et al. 1999; Ito    observed chaperone traits reflect multidomain <br /> et al. 2001).                                                    architecture NUD-1. <br />    examined NUD-1 chaperone activity using             greater understanding NUD-1 function requires <br /> CS inactivation assay. context, NUD-1 protects CS        investigation unique chaperone properties. thermal denaturation slowing rate inactiva-       contains p23 domain, known bind <br /> tion 80%. result suggestive            Hsp90 (Felts Toft 2003; Lee et al. 2004; Wu et al. <br /> transient interaction NUD-1 early folding CS         2005), relationship NUD-1 Hsp90 intermediates (Jakob et al. 1995). Notably, cochaperone      investigated determine acts cochaperone. <br /> p23 shown reduce inactivation rate       Classification cochaperones traditionally based <br /> CS (Bose et al. 1996). contrast NUD-1 p23, sHsps       secondary structural domains.  considering Hsp25 Hsp30C appear form long-             NUD-1 identified p23 domain-containing <br /> lived complex CS intermediates (Ehrnsperger et al.          protein based dimensional modeling, does 1997; Fernando Heikkila 2000)              conform families defined purely secondary <br /> unable prevent CS inactivation (Ehrnsperger et al.            structure.  functional studies NUD-1 cochaperone <br /> 1997). tight binding sHsps unfolded             activity Hsp90 warranted. substrates, require addition ATP-dependent <br /> chaperone, Hsp70, permissive conditions substrates released (Ehrnsperger et al. 1997; <br /> Lee et al. 1997; Haslbeck et al. 2005a). <br />    Using luciferase refolding assay presence equimolar NUD-1, determined NUD-1 able maintain luciferase folding-competent state;                                                                  Fig. 7 Schematic representation primary domains <br /> addition RRL ATP, ~90% luciferase activity         identified NUD-1. positions conserved C. recovered. demonstrated          elegans serine residues altered mimic PLK-1 phosphorylation sHsps (Bose et al. 1996; Fernando Heikkila 2000;             indicated upper portion figure (amino acids 263 315) <br />  102                                                                                                               L.M. Faircloth et al. <br />  <br />  <br /> scope study, binding NUD-1 Hsp90        cellular dysfunction associated aberrant phosphoryla- confirmed SEC immunoprecipitation               tion NudC Plk1 consequence experiments. Structure&#226;&#8364;&#8220;function studies involving removal       mislocalization chaperone activity. chaper- p23 domain examine specificity     activity NUD-1/pNUD-1 maintain func- binding characterization ability NUD-1      tionality proteins, Plk1, traverse modulate ATPase activity Hsp90 potentially         cellular milieu. <br /> advance understanding proteins.          interaction NudC molecular motor proteins,  interesting determine N- C-      dynein, Plk1, underscores pivotal role <br /> terminal regions NUD-1, involved protein&#226;&#8364;&#8220;protein          NudC plays cell division. speculate NudC interactions PLK-1 binding, respectively, necessary      various homologs function capacity chaperone activity. Previous studies p23          chaperone correctly localize mitotic proteins, shown unstructured C terminus responsible   Hsp90 dynein-mediated mechanism. <br /> chaperone activity (Weikl et al. 1999). <br />     chaperones interact Hsp90  <br />                                                                <br /> characterized Hsp90-binding domains, p23/HSP20         <br /> tetracopeptide repeat (TPR) domains (Lamb et al. 1995;          <br /> Blatch Lassle 1999). Glucocorticoid receptor (GR)          <br /> transport nucleus characterized function Hsp90 requires association immunophilin TPR domain. regard, GR/Hsp90 <br />                                                           <br /> complex linked immunophilin, attaches dynein complex retrograde movement                                                                 <br /> microtubules. interaction chaperone                                                                    <br /> cytoskeletal element unique class                  <br /> chaperones. Additional chaperones interact motor            <br /> proteins cytoskeletal elements; include sHsps,   interact actin filaments tubulin (Gusev et al. <br />                                                                 <br /> 2002; Mitra et al. 2007), UNC-45/Cro1/She4p (UCS)              <br /> family chaperones interact myosin (Hutagalung   <br /> et al. 2002), plethora tubulin chaperones,            <br /> include cofactors chaperonins (Lewis et al. 1997; <br />                                                                     <br /> Lundin et al. 2008).                                           <br />     Studies  nidulans mammalian cells shown          NudC associated microtubule minus-end           <br /> directed dynein/dynactin motor complex (Morris 2000; <br />                                                                <br /> Aumais et al. 2001; Han et al. 2001). complex        <br /> target regulatory effector molecules midzone        cytokinesis phase mitosis. NudC         <br /> knocked using small interfering RNA, cytokinesis <br />                                                                    <br /> defects result, Plk1 longer localized               <br /> centrosomes metaphase midbody           <br /> cytokinesis (Aumais et al. 2003; Zhou et al. 2003).           <br /> cytokinesis defects partially rescued ectopic <br />                                                                  <br /> expression wild-type NudC, expression         <br /> NudC mutations Plk1 phosphorylation sites              <br /> (Zhou et al. 2003). Furthermore, NudC-deficient cells         reconstituted wild-type NudC, proper Plk1 <br />                                                                  <br /> localization restored, phospho-defective             <br /> NudC used (Nishino et al. 2006).                                <br />     Notably, study, discovered NUD-1       <br /> chaperone activity levels regulated phosphoryla- <br />                                                                      <br /> tion. context findings, localization     <br /> activity NudC/NUD-1 dissociated.             <br /> </body></html>