<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>6570&#194;&#177;6577 Nucleic Acids Research, 2003, Vol. 31,  22 <br /> DOI: 10.1093/nar/gkg859 <br />  <br />  <br /> Gene essentiality determines chromosome <br /> organisation bacteria <br /> Eduardo P. C. Rocha1,2,* Antoine Danchin1 <br /> 1 <br />  Unite&#195;&#8218; Ge&#195;&#8218;ne&#195;&#8218;tique des Ge&#195;&#8218;nomes Bacte&#195;&#8218;riens, Institut Pasteur, 28, rue du Dr Roux, 75724 Paris Cedex 15, France 2Atelier Bioinformatique, Universite&#195;&#8218; Pierre et Marie Curie, 12, Rue Cuvier, 75005 Paris, France <br />  <br /> Received August 1, 2003; Revised Accepted September 25, 2003 <br />  <br />  <br /> ABSTRACT                                                                        techniques allowed nition set essential <br />                                                                                 genes bacteria. achieved Escherichia coli Bacillus subtilis, essen-                               different techniques. Saturation transposon mutagenesis, fol- <br /> tiality, expressivity, drives distribution                           lowed sequencing, allowed nition essential genes <br /> genes replicating strands.                                      Mycoplasma genitalium (1) Haemophilus uenzae essential genes tend coded                                (2). Antisense RNA used speci&#194;&#174;cally silence genes <br /> leading replicating strand, underlying selective                            Staphylococccus aureus (3). techniques fast constraints evolutionary extent                                easy implement, especially bacteria hard &#194;&#174;ndings subject comparative                              cultivate.  require important resources, tend <br /> studies.  extend previous analysis                               represent number essential genes genomes low G + C &#194;&#174;rmicutes g-proteo-                                polarity transcription operons insertion biases. bacteria, second step sequenced                                 example, studies identi&#194;&#174;ed 658 essential <br /> bacterial genomes. inference essentiality                             genes S.aureus 478 H uenzae, precise <br />                                                                                 systematic gene inactivation larger genome free- <br /> homology allows essential genes <br />                                                                                 living Bacillus subtilis resulted identi&#194;&#174;cation frequent leading strand                               277 essential genes (4). Escherichia coli, equivalent genes, compared non-                                       published work unavailable, PEC database essential highly expressed genes. Smaller biases                                Japan National Institute Genetics provides extensive genomes obligatory intra-                                  information essentiality thirds genes. <br /> cellular bacteria, assignment essen-                           database classi&#194;&#174;es 206 genes essential E.coli, tiality homology fast growing free-living                               82% classi&#194;&#174;ed essential B.subtilis. bacteria problematic. Cross-comparisons                                 study essentiality provides important information cell <br /> used assess potential errors assignment                            functioning; used develop new antibiotic targets, <br /> essentiality homology revealed                                  substantial evolutionary  example, cases, variations assignment criteria                               currently subject intense debate essentiality plays <br /> little uence overall results. Essential                               important role constraining evolutionary rate                                                                                 proteins (5&#194;&#177;8).  patterns loss `essential&apos; genes genes tend conserved leading <br />                                                                                 process genome degradation important strand average genes, consistent                                                                                 understanding evolution bacteria selection positioning impose                                 hosts (9). <br /> strong constraint chromosomal rearrangements.                                   existence larger number genes leading results indicate essentiality plays                                strand, relative lagging strand, known fundamental role distribution genes                                decades. &#194;&#174;rst observed rDNA ribosomal bacterial genomes.                                                         proteins systematically coded leading strand                                                                                 E.coli chromosome (10). speculated result                                                                                 selection prevent collisions DNA polymerase <br /> INTRODUCTION                                                                    (DNAP) RNA polymerase (RNAP) (11). DNA replication concept `essential&apos; gene ambiguous,                      transcription occur simultaneously DNA <br /> genes needed basic supplies indispensable cell                       molecule. DNAP E.coli proceeds 10&#194;&#177;20 times faster metabolites essential special circumstances.                          RNAP (12), head oriented collisions  uses medium enriched presumably                      occur replicating bacteria.  outcome necessary components, essentiality pertains core                  collision different depending poly- <br /> cell&apos;s functions, particular processes involved                    merases oriented  resulting transcription <br /> information transfer compartmentalisation. avail-                       abortion severe replication slow genes <br /> ability complete genome sequences high-throughput                        lagging strand (13,14).  preferential <br />  <br />  correspondence addressed Atelier Bioinformatique, Universite&#195;&#8218; Pierre et Marie Curie, 12 rue Cuvier, 75005 Paris, France. <br /> Tel: +33 1 44 27 65 36; Fax: +33 1 44 27 63 12; Email: erocha@abi.snv.jussieu.fr <br />  <br /> Nucleic Acids Research, Vol. 31  22 &#195;&#163; Oxford University Press 2003; rights reserved <br />                                                                             Nucleic Acids Research, 2003, Vol. 31,  22       6571 <br />  <br />  <br /> positioning translation-related highly expressed genes           frequent leading strand genes. Thirdly, (rDNA ribosomal proteins) leading strand             tested non-highly expressed essential genes biased widely deemed result selection lower               highly expressed non-essential genes. Fourthly, tested <br /> transcription abortion higher replication rates.            conservation essential genes leading strand observations genes coding rDNA ribosomal               important genes. check proteins leading strand bacterial genomes        dif&#194;&#174;culties associated assignment essentiality rm hypothesis (15,16). selection        homology biasing results, set tests effective highly expressed genes acts        cross-comparisons available experimental data positioning genes leading strand, create     B.subtilis, E.coli, M.genitalium H uenzae. <br /> gene strand bias proportional expression level replicating bacteria. bias proportional       MATERIALS METHODS frequency replication. Fast-growing bacteria higher strand bias.                                Data <br />    lines evidence recently challenged           Sequence data corresponding annotations taken <br /> emphasis model replication slow basis        GenBank Genomes (ftp://ftp.ncbi.nih.gov/genomes/ gene strand bias.  gene strand bias strongly depends    Bacteria/). following genomes low G + C &#194;&#174;rmicutes composition DNAP, bacteria containing               analysed: B.subtilis 168, Bacillus halodurans, <br /> different subunits exhibiting higher strand biases (17).     Clostridium acetobutylicum ATCC824, Clostridium perfrin- bacteria, subunit dedicated       gens 13, Lactococcus lactis IL1403, Listeria inocua replication DNA strand (18), average 78%        Clip11262, Listeria monocytogenes EGD, Oceanobacillus genes coded leading strand,     iheyensis, Staphylococcus aureus N315, Streptococcus aga- highly expressed. Secondly, largest gene            lactiae serotype V, Streptococcus pneumoniae TIGR4, <br /> strand biases slow-growing bacteria.             Streptococcus pyogenes M1 Thermoanaerobacter teng- <br /> bacteria containing different DNAP subunits, slow-        congensis. following genomes g-proteobacteria growing M.genitalium highest biases (80%          analysed: E.coli MG1655, Buchnera Ap, H uenzae Rd, <br /> genes leading strand), fast-growing              Pasteurella multocida PM70, Pseudomonas aeruginosa <br /> B.subtilis lowest (74%). genomes           PA01, Pseudomonas putida KT2440, Salmonella enterica <br /> replicated gene coding DNAP subunit,        Typhimurium LT2, Shewanella oneidensis MR-1, slow-growing Borrelia burgdorferi shows highest bias        Xanthomonas axonopodis, Xanthomonas campestris, Xylella <br /> (65%). Comparing equally sized chromosomes, &#194;&#174;nds           fastidiosa 9a5c Yersinia pestis CO92. different slow-growing Mycobacterium tuberculosis (59%)           strains species linear, biased fast-growing E.coli (55%). Thirdly,         strain species analysed. Essentiality data recently shown gene strand bias B.subtilis E.coli       taken literature following genomes: B.subtilis <br /> stems `essentiality&apos; genes. Controlling         (4), M.genitalium (1), H uenzae (2) E.coli (http:// <br /> essentiality revealed signi role expression levels   www.shigen.nig.ac.jp/ecoli/pec/). Identi&#194;&#174;cation origins gene strand bias (19). suggests anti-oriented             replication leading strands use collisions selected problems associated          linear discriminant analysis GC skews, detailed existence aborted transcripts, replication          (23). <br /> pausing. suggests essentiality major deter- <br /> minant chromosome structure. Rearrangements result-          Analysis orthology <br /> ing important inversions leading strands            Orthologues identi&#194;&#174;ed reciprocal best hits. <br /> extremely deleterious (20,21). strand switch essential        BlastP search identi&#194;&#174;ed 10 best hits protein <br /> genes partly responsible observations.             genome. proteins aligned using    substantiate previous observation concerning essen-       global alignment end gaps facing largest <br /> tial genes, limited fast-growing bacteria     sequence ignored. proteins regarded similar genome sizes, tested essentiality plays similar     orthologues reciprocal best hits role bacterial genomes. Putative essentiality          40% similarity <span id='am-6' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence &lt;20% identi&#194;&#174;ed homology genomes B.subtilis            difference protein length (24). Essentiality derived low G + C &#194;&#174;rmicutes) E.coli  g-proteobacteria).         homology B.subtilis genomes low G + C <br /> working hypothesis, simplifying assump-             &#194;&#174;rmicutes E.coli genomes g-proteobacteria. <br /> tions.  ned set putative orthologues        second step, analysed set complete bacterial <br /> genomes, using reciprocal best hits. Secondly, assumed           genomes origin terminus replication orthologues essential genes likely       reliably identi&#194;&#174;ed. case, essentiality derived essential genomes. Thirdly, ned putative         homology B.subtilis subset. classi&#194;&#174;cations highly expressed genes genome codon adaptation         E.coli B.subtilis genes used reference index (CAI) values computed ribosomal proteins (22).        obtained http://wwwabi.snv.jussieu.fr/~erocha/essential/ point, tested different hypotheses concerning      Supplementary Material NAR Online. <br /> higher frequency genes leading strand.  tested essential genes frequent leading     Analysis codon usage <br /> strands non-essential genes. Secondly, tested      CAI values computed using EMBOSS package non-essential genes, highly expressed genes            (http://www.uk.embnet.org/Software/EMBOSS). reference <br />  6572   Nucleic Acids Research, 2003, Vol. 31,  22 <br />  <br />  <br /> values codon usage highly expressed genes (understand-      tailed c2 tests 2 3 2 contingency tables ing high expression cells growing exponen-       comparisons class orthologous <br /> tially) computed using ribosomal proteins (22).       essential genes versus remaining orthologues. recently shown yeast correlation CAI     type tests multiple tests  ned using ribosomal proteins) transcriptome data       genome), used Bonferroni correction <br /> good, good different transcriptome data sets        taxonomic group. Thirdly, test  experimental conditions)         sequenced genomes group genes biased (25). recent work indicated CAI values       performed single paired t-test. case, <br /> robust small changes learning set (26).         test carried hypothesis previously checked transcriptome proteome data,          Bonferroni correction used. available, rmed analysis gene strand bias CAI values (19).  gene regarded potentially highly expressed CAI 10% <br /> highest values genome. use 10% threshold       RESULTS <br /> somewhat arbitrary, ects results classi- <br /> &#194;&#174;cations authors, using CAI factorial          Distribution essential genes replicating <br /> analysis, indicating 5&#194;&#177;10% genes particularly         strands <br /> highly expressed (27&#194;&#177;29). shown previously           Low G + C &#194;&#174;rmicutes. explored link varying threshold range 5&#194;&#177;15% did change      essentiality gene strand bias genomes closely <br /> results signi&#194;&#174;cantly (19).                                        related low G + C &#194;&#174;rmicutes (excluding distant <br />                                                                   Mycoplasmas).  identi&#194;&#174;ed orthologues Tests CAI <br />                                                                   B.subtilis genomes, assigning essentiality <br /> Codon usage bias good estimator gene              according B.subtilis classi&#194;&#174;cation. performed <br /> expression bacteria (30).  designed set     tests.  tested essential genes tests verify codon usage ribosomal proteins       frequently leading strand non-essential <br /> signi&#194;&#174;cantly biased, codon usage classi&#194;&#174;cation      genes. case genomes (Table 1). genome using set discriminant using        essentiality plays important role gene strand <br /> random set genes.  100 random                 bias low G + C &#194;&#174;rmicutes. Secondly, tested experiments.  randomly took 50 genes (corres-          non-essential genes, highly expressed genes frequent <br /> ponding number ribosomal proteins used           leading strand non-highly expressed genes. previous analysis) build reference table codon usage.   test identi&#194;&#174;ed case signi&#194;&#174;cantly higher frequency <br /> Using table, computed CAI values genes    highly expressed genes leading strand frequency 50 initial genes 10%          (C.acetobutylicum) (Table 1).  expression levels <br /> highest CAI. CAI indicating signi codon usage bias         contribute signi&#194;&#174;cantly gene strand bias, high consistency  e. genes initially    essentiality taken account. Thirdly, tested <br /> hypothesised highly expressed classed          essential non-highly expressed genes biased <br /> highly expressed) genome average value smaller         non-essential highly expressed genes. expected chance  discriminate minority highly        mentioned exception C.acetobutylicum, expressed genes rest genome). Signi          difference statistically signi  test showed <br /> codon usage bias ribosomal proteins met conditions.      generally true.  &gt;40% ribosomal proteins classed highly <br /> expressed 10% CAI values. Secondly, frequency      g-Proteobacteria. analysis g- ribosomal proteins classed 10% CAI values      proteobacteria, starting PEC database E.coli signi&#194;&#174;cantly higher obtained random        essential genes. database contains 1700 genes experiments genes used build codon index           essentiality character unknown, (signi difference taken &gt;3 SDs away       removed simplicity. resulting sample contains 2410 <br /> mean random experiments). Thirdly, average CAI      E.coli genes, used identify orthologous genes genes, CAI computed using ribosomal proteins,     g-proteobacteria. incomplete classi- signi&#194;&#174;cantly lower obtained random       &#194;&#174;cation lack extensive experimental gene <br /> experiments (&gt;3 SDs away).                                        disruption programme E.coli, analysis                                                                   error prone.  results qualitatively similar Statistical tests <br />                                                                   observed low G + C &#194;&#174;rmicutes (Table 1). performed types statistical tests adapted   exception Buchnera, genomes high relative <br /> different situations. test groups different        frequencies leading strand essential genes. analysing <br /> frequency leading strand genes genome, carried     non-essential genes gene strand bias function tailed c2 tests 2 3 2 contingency tables. Strand           expression levels, genomes higher biases conservation groups genes genomes               highly expressed genes, genomes lower biases <br /> computed frequency orthologues          highly expressed genes genomes non- replicating strand genomes. tested    signi differences.  non-essential genes, <br /> value different expected given average           expression does play signi role gene <br /> strand conservation orthologues. test consisted    strand bias. Finally, genomes showed                                                                                               Nucleic Acids Research, 2003, Vol. 31,  22                 6573 <br />  <br />  <br /> Table 1. Distribution genes leading strands genomes <br /> Genome                  Genes                                           % Genes lead                                 Statistical tests <br />                         EH           EnH        nEH        nEnH         EH      EnH          nEH        nEnH         Ess        Exp      EvH         SC       CB <br />  <br /> B.subtilis               78          199         119       3710          94       94         72         74           +          0        +                    + <br /> B.halodurans            240           15        1503        402          93       93         81         79           +          0        +           +        + <br /> C.acetobutylicum        218           10         918        251          93       91         91         80           +          +        0           +        + <br /> C.perfringens           208            9         770        196          95       93         88         85           +          0        +           +        + <br /> L.lactis                225           11         698        186          88       89         74         81           +          0        +           +        + <br /> L.inocua                242           16        1069        255          95       94         75         80           +          0        +           +        + <br /> L.monocytogenes         243           18        1035        243          96       93         73         79           +          0        +           +        + <br /> O.iheyensis             248           14        1366        395          96       93         79         76           +          0        +           +        + <br /> S.agalactiae            219           11         665        186          93       92         76         79           +          0        +           +        + <br /> S.aureus                237           13         956        261          95       89         75         75           +          0        +           +        + <br /> S.pneumoniae            214           10         654        153          93       90         72         80           +          0        +           +        + <br /> S.pyogenes              208            9         575        131          92       93         78         80           +          0        +           +        + <br /> T.tengcongensis         212           11         771        166         100       94         90         87           +          0        +           0        0 <br />  <br /> E.coli                  112           94         249       1955          80       71         57         56           +          0        +                    + <br /> Buchnera Ap             127           33         169        116          70       47         73         61           0          0        &#194;&#177;           0        0 <br /> H uenzae             149           44         389        289          79       67         52         51           +          0        +           +        + <br /> P.multocida             152           44         450        318          67       72         65         54           +          0        +           +        + <br /> P.aeruginosa            150           40         628        437          75       82         51         59           +          &#194;&#177;        +           +        + <br /> P.putida                145           39         615        422          75       78         54         61           +          &#194;&#177;        +           +        + <br /> S.enterica              154           49         967        718          80       70         57         57           +          &#194;&#177;        +           +        + <br /> S.oneidensis            148           40         569        393          81       72         62         57           +          +        +           +        + <br /> X.axonopodis            145           35         474        323          79       77         56         60           +          0        +           +        + <br /> X.campestris            141           34         482        321          79       79         53         59           +          0        +           +        + <br /> X.fastidiosa            139           35         342        216          99       63         98         51           +          +        &#194;&#177;           +        + <br /> Y.pestis                156           46         730        567          80       74         59         53           +          +        +           +        + <br />  classi&#194;&#174;cation low G + C &#194;&#174;rmicutes homology B.subtilis g-proteobacteria E.coli. genes classed according expression level essential phenotype highly expressed (H) versus non-highly expressed (nH) essential (E) versus non-essential (nE). acronyms concatenated classes (e.g. EnH indicates essential non-highly expressed genes). columns `Statistical tests&apos;  tail c2 <br /> tests contingency tables) indicate signi&#194;&#174;cance test P &lt; 0.05, applying Bonferroni correction. + &#194;&#177; signs indicate rejection hypothesis (right- left  respectively), 0 indicates rejection. tests correspond  E &#194;&#185; nE (Ess), nEH &#194;&#185; nEnH (Exp), EnH &#194;&#185; nEH (EvH), <br /> larger strand conservation essential genes (SC) signi codon usage bias ribosomal proteins (CB). <br />  <br />  <br />  <br />  <br /> signi&#194;&#174;cantly higher frequency leading strand essential                      (Buchnera, Chlamydia, Spirochetes Rickettsia), non-highly expressed genes non-essential highly                           obligatory pathogen (X.fastidiosa). <br /> expressed genes. exceptions trend non- <br /> free-living Bacteria Buchnera X.fastidiosa,                                                                                    positioning essential genes leading strand <br /> showing suf&#194;&#174;cient codon usage bias reliably determine <br />                                                                                    highly conserved <br /> expression levels. <br />                                                                                    observations suggest essentiality important Extension procedure. assignment essentiality                      low G + C &#194;&#174;rmicutes g-proteobacteria, possibly <br /> homology distant phylogenetic domains prone                        bacterial domains.  closely related genomes important errors, essentiality relative set                   resemble E.coli B.subtilis phylogenetic functions lifestyle nition                            inertia, common selective pressure orthology increasingly error prone evolutionary                         positioning essential genes leading strand. distances increase. cases, nition                       close genomes, E.coli Y.pestis (31) high expression based CAI problematic.                     S.agalactiae S.pneumoniae (32), severely preliminary analysis entire set sequenced                         shuf&#194;&#175;ed speciation. Interestingly, genome rearrange- <br /> bacteria, including 30 fully sequenced genomes                            ments tend symmetrical origin replication <br /> different species. genome, determined                              change replicating strand gene orthologues B.subtilis. orthologues classed                     coded, disrupt gene order (33,34).  essential non-essential according classi&#194;&#174;cation                  performed test determine strand <br /> B.subtilis. results qualitatively similar                        conservation stronger essential genes presented   essential genes biased                       genes (SC column Table 1). results indicate non-essential genes nearly genomes (Fig. 1). Secondly,                      21 23 genomes, conservation leading strand highly expressed genes, essential genes                           character essential genes signi&#194;&#174;cantly important biased non-essential ones. Thirdly, highly expres-                       genes (P &lt; 0.05, applying sed non-essential genes biased essential                     Bonferroni correction multiple tests) (Table 1). highly expressed genes. analysis, observed                        exceptions concern Buchnera T.tengcongensis. exceptions, concerning intracellular bacteria                           conservation important  genome <br />  6574     Nucleic Acids Research, 2003, Vol. 31,  22 <br />  <br />  <br />  <br />  <br />                                                                                  Figure 2. Comparison essential genes data set E.coli                                                                                  B.subtilis. Upper panel: assignment essentiality orthologues (black <br />                                                                                  squares represent essential genes B.subtilis). Lower panel: presence                                                                                  absence orthologues depending classi&#194;&#174;cation essentiality                                                                                  genomes (black squares represent genes present B.subtilis absent <br />                                                                                  E.coli). area blocks proportional relative frequency                                                                                  category. <br />  <br />  <br />                                                                                  Methodological problems lead erroneous infer- <br />                                                                                  ence orthology gene duplication non- <br />                                                                                  orthologous gene displacement (36).  important <br />                                                                                  test robustness assumptions concerning <br />                                                                                  essentiality. low G + C &#194;&#174;rmicutes sharing fewer <br />                                                                                  orthologues B.subtilis essential genes S.pyogenes <br />                                                                                  C.acetobutylicum (Table 1).  cases,                                                                                  identify orthologues 78% B.subtilis essential genes. <br /> Figure 1. Frequency leading strand genes 53 genomes bacteria.                                                                                  g-proteobacteria, genome containing fewer bacteria chromosome, pooled               orthologues E.coli essential genes Buchnera. Buchnera <br /> chromosomes. Essentiality assigned homology B.subtilis,           relatively close E.coli, evolutionary history genes presenting orthologous gene B.subtilis considered        extensive genome reduction led genome analysis.   Essential versus non-essential genes. (B) Essential versus     approximately 500 genes (37). 13% non-essential genes putative highly expressed genes. (C) Highly <br /> expressed non-essential genes versus essential non-highly expressed genes.       E.coli non-essential genes orthologue Buchnera, diagonal line indicates similar frequency sets. Intracellular   77% essential genes orthologue. obligatory pathogens symbionts indicated open circles          results indicate majority essential genes genomes circles. panels, essential genes     conserved evolutionary distances, biased set (P &lt; 0.001, paired t-tests). <br />                                                                                  genes conserved. turned comparison                                                                                  distantly related E.coli B.subtilis genomes. <br />  <br /> shows highest overall strand bias (35), difference                Comparison essential genes B.subtilis E.coli. signi application Bonferroni correction.                       1129 orthologues E.coli B.subtilis,                                                                                  75% B.subtilis essential genes 26% non- <br /> Tests                                                                            essential genes (Fig. 2).  expected, presence previous analyses strongly suggest majority                   essential genes conserved bacteria, essentiality drives gene strand bias.  assigning                   genes large evolutionary distances.  82% essentiality homology prone errors. example,                   E.coli essential genes classed essential B.subtilis genomes lack orthologues essential B.subtilis                     7% non-essential genes regarded essential <br /> genes, indicating non-essentiality bacteria.                      B.subtilis) (Fig. 2). Given large evolutionary <br />                                                                                          Nucleic Acids Research, 2003, Vol. 31,  22                6575 <br />  <br />  <br />                      Table 2. Cross-comparisons frequency genes leading strand using experimental data <br />                      sets essential genes genomes <br />                                            Experimental data              Homology                                                                   Essential   Non-essential      Essential     Non-essential     Essential     Non-essential <br />  <br />                      M.genitalium (Bs)     82%          75%               85%           78%               86%           77% <br />                      H uenzae (Ec)      55%          55%               70%           51%               73%           53% <br />                      B.subtilis (Ec)       94%          72%               97%           83%               97%           80% <br />                      E.coli (Bs)           76%          56%               75%           56%               84%           56% <br />  <br />                      types information essentiality used compute frequency leading strand essential <br />                      genes:   using experimental data set genome; (ii) homology data set genome <br />                      (Bs, B.subtilis; Ec, E.coli); (iii) making intersection   (ii). example, M.genitalium shows <br />                      82, 85 86% essential genes leading strand essentiality ned respectively                      experimental data, homology essential genes B.subtilis, using intersection                      criteria (essential B.subtilis M.genitalium). <br />  <br />  <br />  <br /> distance bacteria, &#194;&#174;nding                        Table 3. Percentage genes leading strand genomes, <br /> large fraction essential genes conserved                      according essentiality <br /> genomes suggests assigning essentiality homology                    Comparison                             Leading       Leading       Expected <br /> reasonable, free-living bacteria.                                                                    essential      <br /> Comparisons experimental data sets. used                        B.subtilis versus M.genitalium         85%           70%           60% <br /> experimental data sets essential genes B.subtilis, E.coli,               B.subtilis versus E.coli               73%           44%           41% <br />                                                                                E.coli versus H uenzae              60%           35%           35% <br /> M.genitalium H uenzae make set cross- <br /> comparisons. comparison, different                       gene regarded essential experimentally essential <br /> analyses (Table 2). &#194;&#174;rst analysis consists simply                       genomes. <br /> computing frequency leading strand essential non- <br /> essential genes genome, essentiality taken experimental data set [e.g. M.genitalium data set analysis `M.genitalium (Bs)&apos;]. second analysis                      leading strand character essential genes.  consists computing frequency leading strand essential                compared B.subtilis genomes M.genitalium genes genome essentiality assigned                          E.coli, H uenzae. homology genome [e.g. assigned                             genes essential genomes tend occur homology B.subtilis analysis `M.genitalium (Bs)&apos;].                   leading strand higher frequency analysis consists computing frequency                      genes (Table 3). example, comparison B.subtilis <br /> leading strand essential genes essentiality assigned                  E.coli, gene order gene essential experimental data set              conservation (39), shows 73% essential genes leading orthologue essential genome [e.g. gene              strand genomes. similar analysis applied experimentally essential M.genitalium                   remaining genes indicates 44% genes conserved B.subtilis analysis `M.genitalium (Bs)&apos;].                    leading strand, expected frequency, taking analyses consistently results,                        account average gene strand composition genome, <br /> essential genes frequent leading strand                      41%. B.subtilis M.genitalium, difference independently information used ne essentiality.                   smaller (85 versus 70%), expected value analysis H uenzae&apos;s experimental data set                   larger (60%). conservation positioning essential <br /> indicates signi difference essential non-                  genes leading strand consistent existence essential genes. experimental analysis H uenzae                      selective forces displacement essential genes identi&#194;&#174;ed 474 essential genes 1272 analysed  e. ~70%                   lagging strand. genome) (2), twice essential <br /> genes B.subtilis (4), predicted H uenzae computational genome analysis (38). H uenzae,                       DISCUSSION <br /> essentiality determined transposon mutagenesis, likely overestimate number essential genes.                       Essentiality drives gene strand bias <br /> Unsurprisingly, use E.coli data set intersection             discussed  lines evidence indicate data sets indicates signi bias (73%                  positioning certain genes leading strand subject essential leading strand genes . general              selection. Brewer proposed biases caused <br /> trend: highest biases intersection               selection avoidance frequent head collisions <br /> data sets (Table 2), probably case likehood                DNAP RNAP (13).  rmed falsely assigning essentiality minimised.                                previous analyses, showing bacteria, essentiality,                                                                                expressiveness, plays major role gene strand bias (19). <br /> Testing degree gene strand conservation. used                   compatible previous model emphasis experimental data sets test conservation                shifted rate collisions polymerases  6576   Nucleic Acids Research, 2003, Vol. 31,  22 <br />  <br />  <br /> transcript resulting collisions. importantly,     non-essential genes, differences typically high. gene product, expression rate,            exceptions analysis important selective factor.  understanding fate     highly expressed genes versus essential highly <br /> transcript cast light causes gene strand bias.      expressed genes, concern obligatory parasites/ <br /> possibility  transcription abortion occurs       symbionts, obligate intracellular (Fig. 1C). lower rates oriented collisions head collisions    propose expression levels important (14), truncated transcripts lead truncated, non-    genomes. unlikely reasons.  functional, peptides. peptides parts large         bacteria grow slowly expression levels <br /> complexes (e.g. ribosome DNAP), typically       important fast growers. Secondly, basis dominant-negative phenotypes. naturally           used CAI determine high levels expression.  counter-selected essential functions.                    genomes spirochetes (30) Chlamydia (40) exhibit weak <br />    extended previous observations concerning gene             signi  codon usage biases highly <br /> strand bias directions.  shown       expressed genes, Buchnera genomes nearly devoid <br /> spite substantial rearrangements E.coli          biases (41). tests CAI B.subtilis, leading strand positioning essential genes    genomes g-proteobacteria low G + C &#194;&#174;rmicutes <br /> conserved. conserved average       lack signi codon usage bias ribosomal proteins: <br /> orthologue. Secondly, results putatively          T.tengcongensis Buchnera. CAI bad predictor identi&#194;&#174;ed essential genes preferably located leading     expression levels genomes, safely <br /> strand analysed genomes low G + C              attribute atypical bias expression levels. Thirdly, gene <br /> &#194;&#174;rmicutes g-proteobacteria.  collision problems       strand bias putative highly expressed non-essential <br /> associated gene strand bias likely general        genes relatively small.  expression unlikely importance bacterial world. Thirdly,       play important role bacteria essentiality taken account, expression levels        B.subtilis E.coli. Genome rearrangements change signi&#194;&#174;cantly contribute explain gene strand bias        leading strand character gene  frequent, groups. essential genes synthesised         lead lower gene strand bias. genomes high levels, frequency highly expressed genes                                                                     obligatory intracellular bacteria stable, Buchnera <br /> essential genes higher average (e.g. nearly 30% <br />                                                                     suffered extensive rearrangements associated process B.subtilis essential genes). quite natural                                                                     genome reduction, lowered gene strand <br /> previous analyses using rDNA ribosomal proteins                                                                     bias. possible bacteria, leading strand <br /> revealed expressiveness basis gene strand bias: <br />                                                                     positioning essential genes weaker selection, e.g. genes simultaneously essential highly expressed.  essentiality taken account,         drift associated small population <br /> expression plays small,  role determining           sizes. bacteria suffer relaxed selection distribution genes replication strands.        housekeeping functions (9).  depend host results derived analyses orthology assignment       functions deemed essential.  essentiality orthologous relationships.       assignment essentiality homology error <br /> shown, prone errors.  degree             prone functions, especially metabolic ones. <br /> consistency results phylogenetic          hand, intracellular bacteria groups cross-validations performed suggest          essential genes (e.g. transporters metabolites) errors strongly biasing conclusions.          essential survival, essential                                                                     B.subtilis. clearer insight gene strand bias Understanding differences gene strand bias intensity             genomes require availability experimental data sets signi differences terms overall gene           essential genes bacteria (note M.genitalium <br /> strand bias bacteria. discussed (17),        does belong category composition DNAP explain differences.          intracellular bacterium). effect independent genome size                 addition suggesting lower biases obligatory <br /> species ecology relate differential stability     intracellular bacteria, data indicate different levels replication fork. Bacillus subtilis higher general   gene strand bias different bacteria. causes gene strand bias (75%) higher essential gene strand bias      different stabilities replication forks, (94%) E.coli (55 76%, respectively).             discussed  different RNAP processivity, leading effect essentiality gene strand bias adds caused   RNAP robust collisions, different ef&#194;&#174;ciency <br /> differences DNAP composition, effects            mechanisms degrading truncated peptides. likely related. example, speculate         aborted transcript results stalled ribosomes, tmRNA tags having DNAP subunits leads increased asym-              truncated <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> directs protease degradation <br /> metry outcome collisions DNAP RNAP.           machinery (42,43). tmRNA does direct elimination lead higher frequency leading strand genes       corresponding mRNA likely saturated general, essential genes particular.                   replicating bacteria, large number collisions. <br />    Buchnera, essential genes            tmRNA intrinsically ef&#194;&#174;cient, frequent leading strand remaining          expect lower levels gene strand bias bacteria. <br /> genes (Fig. 1A). plots, putative highly              Experimental work necessary tackle expressed genes, distribution essential versus               questions. <br />                                                                                           Nucleic Acids Research, 2003, Vol. 31,  22                 6577 <br />  <br />  <br /> SUPPLEMENTARY MATERIAL                                                          20. Louarn,J.M., Bouche,J.P., Legendre,F., Louarn,J. Patte,J. (1985) <br />                                                                                     Characterization properties large inversions E.coli <br /> Supplementary Material available NAR Online.                                  chromosome origin terminus axis. Mol. Gen. Genet., 201, <br />                                                                                     467&#194;&#177;476. <br />                                                                                 21. Segall , Mahan,M.J. Roth,J.R. (1988) Rearrangement ACKNOWLEDGEMENTS                                                                    bacterial chromosome: forbidden inversions. Science, 241, 1314&#194;&#177;1318. <br />                                                                                 22. Sharp,P.M. Li,W.H. (1987) codon adaptation index&#195; measure thank Joe&#195;&#710;l Pothier comments manuscript,                         directional synonymous codon usage bias potential scientists disrupted studied genes                       applications. Nucleic Acids Res., 15, 1281&#194;&#177;1295. <br /> B.subtilis genome, particular                             23. Rocha,E.P.C. Danchin  (2001) Ongoing evolution strand <br /> enterprise happen S. D. Ehrlich, F. Kunst, N. Ogasawara                         composition bacterial genomes. Mol. Biol. Evol., 18, 1789&#194;&#177;1799. <br />                                                                                 24. Tatusov,R.L. Koonin,E.V. (1997) genomic perspective protein <br /> H. Yoshikawa.                                                                       families. Science, 278, 631&#194;&#177;637. <br />                                                                                 25. Coghlan  Wolfe,K.H. (2000) Relationship codon bias mRNA <br />                                                                                     concentration protein length Saccharomyces cerevisiae. Yeast, 16, <br /> REFERENCES                                                                          1131&#194;&#177;1145. <br />  1. Hutchison,C , Peterson,S.N., Gill,S.R., Cline,R.T., White,O.,             26. Jansen,R., Bussemaker,H.J. Gerstein,M. (2003) Revisiting codon <br />     Fraser,C.M., Smith,H.O. Venter,J.C. (1999) Global transposon                adaptation index genome perspective: analyzing     mutagenesis minimal Mycoplasma genome. Science, 286,                      relationship gene expression codon occurrence yeast <br />     2165&#194;&#177;2169.                                                                      using variety models. Nucleic Acids Res., 31, 2242&#194;&#177;2251. <br />  2. Akerley,B.J., Rubin,E.J., Novick,V.L., Amaya,K., Judson,N.              27. Sharp,P.M. Li,W.-H. (1986) evolutionary perspective     Mekalanos,J.J. (2002) genome-scale analysis identi&#194;&#174;cation              synonymous codon usage unicellular organisms. J. Mol. Evol., 24, <br />     genes required growth survival Haemophilus uenzae.                 28&#194;&#177;38. <br />     Proc. Natl Acad. Sci. USA, 99, 966&#194;&#177;971.                                     28. Andersson,S.G.E. Kurland,C.G. (1990) Codon preferences  3. Forsyth,R , Haselbeck,R.J., Ohlsen,K.L., Yamamoto,R.T., Xu,H.,                free-living microorganisms. Microbiol. Rev., 54, 198&#194;&#177;210. <br />     Trawick,J.D., Wall,D., Wang,L., Brown-Driver,V., Froelich,J.M. et al.       29. Me&#195;&#8218;digue,C., Rouxel,T., Vigier,P., Henaut  Danchin  (1991) <br />     (2002) genome-wide strategy identi&#194;&#174;cation essential genes       Evidence horizontal gene transfer E.coli speciation. J. Mol. Biol., <br />     Staphylococcus aureus. Mol. Microbiol., 43, 1387&#194;&#177;1400.                          222, 851&#194;&#177;856. <br />  4. Kobayashi,K., Ehrlich,S.D., Albertini , Amati,G., Andersen,K.K.,          30. Perriere,G. Thioulouse,J. (2002) Use misuse correspondence <br />     Arnaud,M., Asai,K., Ashikaga,S., Aymerich,S., Bessieres,P. et al. (2003)        analysis codon usage studies. Nucleic Acids Res., 30, 4548&#194;&#177;4555. <br />     Essential Bacillus subtilis genes. Proc. Natl Acad. Sci. USA, 100,          31. Parkhill,J., Wren,B.W., Thomson,N.R., Titball,R.W., Holden,M.T., <br />     4678&#194;&#177;4683.                                                                      Prentice,M.B., Sebaihia,M., James,K.D., Churcher,C., Mungall,K.L. et al. <br />  5. Jordan K., Rogozin B., Wolf,Y  Koonin,E.V. (2002) Essential           (2001) Genome sequence Yersinia pestis, causative agent     genes evolutionarily conserved nonessential genes          plague. Nature, 413, 523&#194;&#177;527. <br />     bacteria. Genome Res., 12, 962&#194;&#177;968.                                         32. Glaser,P., Rusniok,C., Buchrieser,C., Chevalier,F., Frangeul,L., <br />  6. Pal,C., Papp,B. Hurst,L.D. (2003) Rate evolution gene                Msadek,T., Zouine,M., Couve,E., Lalioui,L., Poyart,C. et al. (2002) <br />     dispensability. Nature, 421, 496&#194;&#177;497.                                           Genome sequence Streptococcus agalactiae, pathogen causing <br />  7. Hirsh E. (2003) Rate evolution gene dispensability&#195;?reply.              invasive neonatal disease. Mol. Microbiol., 45, 1499&#194;&#177;1513. <br />     Nature, 421, 497&#194;&#177;498.                                                       33. Tillier,E.R. Collins,R  (2000) Genome rearrangement  8. Rocha,E.P.C. Danchin  (2003) analysis determinants               replication-directed translocation. Nature Genet., 26, 195&#194;&#177;197. <br />     protein substitution rates bacteria. Mol. Biol. Evol., press.         34. Eisen,J , Heidelberg,J.F., White,O. Salzberg,S.L. (2000) Evidence <br />  9. Ochman,H. Moran,N  (2001) Genes lost genes                  <br /> </body></html>