<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                                 Expression purification F-plasmid RepE Crystallization                             preliminary X-ray crystallographic study Communications <br />                                                 complex operator DNA <br />  <br />  <br />  <br />  <br />                   replication initiator factor RepE F plasmid Escherichia coli                             essential protein stringently regulates F-plasmid copy number.                                                 RepE protein dual function: monomer functions replication <br />  <br />    initiator, dimer acts transcriptional repressor repE gene.             wild-type dimeric RepE protein expressed N-terminal histidine-tagged <br />      protein, purified native conditions high salt concentration  <br />                                                 crystallized complex repE operator DNA using sitting-drop <br />                                                                                    &#203;&#353;  <br />                                                 vapour-diffusion technique. crystals diffracted resolution 3.14                         application dehydration crystal annealing belong space group <br />                                                 P21, unit-cell parameters = 60.73, b = 99.32, c = 95.00   <br />  <br />  <br />                                                 1. Introduction <br />                                                 mini-F plasmid, derivative F factor Escher- <br />                                                 ichia coli, replicates strict control, maintaining copy number <br />                                                 1&#226;&#8364;&#8220;2 host chromosome (Kline, 1985). plasmid-encoded <br />                                                 initiator protein RepE (251 amino acids, 29 kDa) plays important <br />                                                 role regulation mini-F replication. distinct oligomeric <br />                                                 states, monomeric dimeric, RepE                                                 functions known different (Ishiai et al., 1994). RepE <br />                                                 monomer replication initiator recognizes specific 19 bp <br />                                                 sequence DNA (iteron; Masson <span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; Ray, 1986, 1988; Tokino et al., <br />                                                 1986). replication origin (ori2), directly repeated <br />                                                 iterons (DR) present bound RepE molecules,                                                 cause local DNA melting 13-mer region assis- <br />                                                 tance histone-like protein HU subsequent initiation                                                 mini-F replication (Kawasaki et al., 1996; Komori, Matsunaga et al., <br />                                                 1999). contrast, RepE dimer autogenous repressor                                                 binds repE promoter/operator region located just upstream                                                 repE gene (Rokeach et al., 1985; Masson &amp; Ray, 1986, 1988; <br />                                                 Tokino et al., 1986). operator repE includes inverted-repeat <br />                                                   sequence, shares 8 bp iterons. RepE dimer <br />                                                 recognizes common 8 bp inhibits transcription <br />                                                 repE itself. RepE exists primarily dimer, order                                                 activate RepE initiator, conversion dimer                                                 monomer accomplished molecular chaperones DnaK, DnaJ <br />                                                 GrpE (Kawasaki et al., 1990, 1991; Wada et al., unpublished <br />                                                 results). crystal structure active initiator form RepE                                                 reported previously (Komori, Matsunaga et al., 1999)                                                 suggested C-terminal domain recognizes specific DNA <br />                                                 sequence N-terminal domain interacts DNA <br />                                                 backbone. Based RepE monomer structure, marked <br />                                                 conformational change required dimerization. Structural <br />                                                 information repressor form RepE,  remains <br />                                                 unknown. initiator proteins generally aggregate easily, <br />                                                 wild-type length initiators targeted struc- <br />                                                 tural studies. successfully expressed purified wild-type <br /> # 2007 International Union Crystallography   RepE report crystallization preliminary X-ray rights reserved                             diffraction analysis RepE bound DNA containing  <br /> 346       doi:10.1107/S1744309107012894                                                                              Acta Cryst. (2007). F63, 346&#226;&#8364;&#8220;349 <br />                                                                                                        crystallization communications <br /> Table 1                                                                                    added His6-RepE solution 1.2-fold molar excess relative Data-collection processing statistics His6-RepE&#226;&#8364;&#8220;DNA1 complex.                   RepE dimer mixture subsequently dialyzed Values parentheses refer highest resolution shell.                               crystallization buffer [100 mM potassium citrate pH 6.2, 100 mM KCl, <br /> Wavelength  &#203;&#353;)                                                1.0000                     10 mM MgCl2, 10%(v/v) glycerol 2 mM dithiothreitol (DTT)] Space group                                                     P21                        277 K concentrated approximately 5 mg ml 1 His6-RepE <br />                       &#203;&#353; ,  ) <br /> Unit-cell parameters                                          = 60.73, b = 99.32,      concentration. crystallization, formation protein&#226;&#8364;&#8220; <br />                                                                    c = 95.00,   = 108.55 <br /> Resolution range  )                                           50.0&#226;&#8364;&#8220;3.14 (3.25&#226;&#8364;&#8220;3.14)      DNA complex confirmed native polyacrylamide gel electro- <br /> Observed reflections                                            42887                      phoresis (PAGE), band corresponding protein&#226;&#8364;&#8220; <br /> Unique reflections                                              16535 <br /> Completeness (%)                                                88.3 (73.6) <br />                                                                                            DNA complex stained Coomassie Brilliant Blue (CBB)                                                             11.0 (3.9)                 Mupid-Blue (Advance, Tokyo, Japan). <br /> Rsym&#226;&#8364;&#160; (%)                                                       6.7 (15.0) <br />         P P                          P P <br /> &#226;&#8364;&#160; Rsym = hkl jIi &#195;&#176;hkl&#195;&#382;   hI&#195;&#176;hkl&#195;&#382;ij= hkl Ii &#195;&#176;hkl&#195;&#382;, Ii(hkl) ith intensity   2.3. Crystallization <br /> measurement reflection hkl hI(hkl average value multiple <br /> measurements.                                                                                 crystallization conditions His6-RepE&#226;&#8364;&#8220;operator <br />                                                                                            complexes screened sitting-drop vapour-diffusion <br /> sequence order elucidate crystal structure dimeric RepE                       method 277 K. Initial conditions using Hydra II Plus complex repE operator DNA.                                                         (Matrix Technologies Corporation, Hudson, NH, USA)                                                                                            commercially available screening kits (Hampton Research, Aliso <br />                                                                                            Viejo, CA, USA; Sigma&#226;&#8364;&#8220;Aldrich, St Louis, MO, USA). drop 2. Materials methods                                                                   prepared mixing equal volumes (0.3 ml initial screening 2.1. Expression purification                                                           1.0 ml optimization) His6-RepE&#226;&#8364;&#8220;operator complex solution <br />                                                                                            reservoir solution, allowing mixture equilibrate    E. coli strain M15 harbouring pREP4 plasmid (Qiagen, Hilden,                            reservoir solution (80 ml initial screening 100 ml Germany) transformed pKV7202 plasmid carrying                                  optimization). presence complex crystal component <br /> N-terminal tagged repE gene (Matsunaga et al., 1995) grown                         identified native PAGE using washed dissolved 310 K Luria broth medium containing 50 mg ml 1 ampicillin                        crystals. <br /> 25 mg ml 1 kanamycin. culture reached mid-log growth <br /> phase (optical density 600 nm = 0.6), culture rapidly cooled ice protein expression induced addition                               2.4. X-ray diffraction analysis <br /> isopropyl  -d-thiogalactopyranoside (IPTG) final concentration <br />                                                                                               His6-RepE&#226;&#8364;&#8220;DNA1 (Fig. 1) crystal mounted nylon loop 0.1 mM. growth 22 h 293 K, cells                          transferred soaked 36 h 277 K dehydration solution <br /> harvested centrifugation 2110g 30 min 277 K.                       containing 1.2-fold higher concentration component following purification steps N-terminal tagged RepE (His6-                          crystallization condition. dehydrated crystal briefly <br /> RepE) performed cooled conditions 277 K.                                 immersed dehydration solution supplemented 25&#226;&#8364;&#8220; <br /> bacterial pellets resuspended lysis buffer [50 mM NaH2PO4                          30%(v/v) glycerol cryoprotectant frozen flash- <br /> pH 8.0, 1 M KCl, 20%(v/v) glycerol 10 mM imidazole] supple-                            cooling stream nitrogen <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="gas" datatype="xsd:string"></span>gas</span> 100 K. Prior X-ray irradia- <br /> mented 1 mg ml 1 lysozyme 0.5 mM 4-(2-aminoethyl)-                                tion, performed crystal annealing (Harp et al., 1998; Heras &amp; <br /> benzenesulfonyl fluoride (p-ABSF) disrupted sonication                           Martin, 2005) blocking cryostream twice 8 s time. <br /> ice. cell lysate centrifuged 27 000g 60 min                         X-ray diffraction data collected using ADSC Quantum 315 <br /> supernatant filtered using membrane filter 0.45 mm <br />                                                                                            CCD detector SPring-8 beamline BL41XU wavelength pore size. Ni2+&#226;&#8364;&#8220;nitrilotriacetic acid (Ni&#226;&#8364;&#8220;NTA) Superflow resin                       1.0000 &#203;&#353; . total 180 frames recorded oscillation <br /> (Qiagen) equilibrated lysis buffer added supernatant                       angle 1.0  , exposure time 3.0 s frame crystal- cell lysate, mixture stirred 60 min loaded                     detector distance 350 mm. diffraction intensities Ni&#226;&#8364;&#8220;NTA Superflow column. column washed                              integrated HKL-2000 (Otwinowski &amp; Minor, 1997) program <br /> stepwise gradient wash buffer [50 mM sodium phosphate pH 8.0, <br /> 1 M KCl 20%(v/v) glycerol] containing 30, 50 100 mM <br /> imidazole. His6-RepE eluted wash buffer containing <br /> 200 mM imidazole. cycles purification means Ni&#226;&#8364;&#8220; <br /> NTA immobilized metal-affinity chromatography (IMAC) performed. stock solution, glycerol added final <br /> concentration 35%(v/v) purified His6-RepE stored small aliquots 193 K. <br />  <br /> 2.2. Preparation RepE&#226;&#8364;&#8220;DNA complex <br />   RepE&#226;&#8364;&#8220;DNA complex formation, oligonucleotides containing inverted-repeat sequence repE operator recognized RepE dimer prepared Hokkaido Science (Sapporo, <br /> Japan). sequences designed based expectation DNA duplexes form pseudo-continuous helix crystal packing <br /> (Joachimiak &amp; Sigler, 1991), shown Fig. 1. DNA duplexes                         Figure 1 <br />                                                                                            DNA duplexes used complex formation crystallization trials. common <br /> prepared essentially described Komori, Sasai et al. (1999),                          8 bp sequences iteron operator underlined directions final concentrations 2.4 mM. DNA duplex                          indicated arrows. <br />  <br />  <br /> Acta Cryst. (2007). F63, 346&#226;&#8364;&#8220;349                                                                                                      Nakamura et al.         RepE    347 <br />  crystallization communications <br /> suite. data-collection processing statistics summarized                    estimated molecular weight  100 kDa gel-filtration <br /> Table 1.                                                                                  chromatography. addition, reported repE <br />                                                                                           operator preferentially bound dimeric form RepE <br />                                                                                           compared monomeric form (Ishiai et al., 1994).  <br /> 3. Results discussion                                                                 concluded complexes constituted RepE <br />                                                                                           dimer DNA duplex. <br /> Wild-type RepE proteins tend aggregate 80%                               crystallization conditions complexes RepE expressed E. coli collected inclusion bodies. RepE                     screened sparse-matrix method. crystals protein previously purified denaturing                           complexes (complexes DNA1, DNA2 DNA3) conditions using guanidine hydrochloride (Kawasaki et al., 1992;                          obtained, crystal  complex DNA1) suitable <br /> Kline et al., 1992; Ishiai et al., 1994; Matsunaga et al., 1997).                 X-ray analysis. presence His6-RepE DNA1 order RepE proteins native fold subsequent                      obtained crystals confirmed (Fig. 2), optimization <br /> crystallization, attempted purify His6-RepE dena-                           crystallization conditions His6-RepE&#226;&#8364;&#8220;DNA1 afforded turing. bacterial culture conditions, temperature shifted                     final condition 100 mM HEPES&#226;&#8364;&#8220;NaOH pH 7.2, 200 mM NaCl, 310 293 K order decrease protein-expression level                       10%(w/v) PEG 4000 5 mM SmCl3. Prismatic crystals grew expression His6-RepE protein. Furthermore,                           weeks (Fig. 3; 0.12   0.04   0.02 mm). Prior X-ray exposure, RepE purification high salt concentration 1 M maintained                      dehydration treatment (Schick &amp; Jurnak, 1994; Heras &amp; Martin, buffers prevent protein aggregation. Owing modifi- <br /> cations, significant improvement yield purified RepE <br /> protein achieved  1 mg 25 mg His6-RepE litre culture). cycles purification IMAC <br /> method, His6-RepE attained 90% purity estimated SDS&#226;&#8364;&#8220; <br /> PAGE followed staining CBB. result gel-filtration <br /> chromatography His6-RepE using wash buffer containing <br /> 200 mM imidazole [eluted 76 ml Hi-Load 16/60 Superdex 200 <br /> (GE Healthcare Biosciences, Tokyo, Japan) flow rate 0.3 ml min 1] indicates presence  80 kDa protein  30 kDa protein, suggesting His6-RepE forms dimers high-salt conditions (1 M). <br />    complexes His6-RepE DNA duplexes <br /> containing (31 33 bp; Fig. 1) prepared complexation confirmed native PAGE described  normal conditions (pH 8.8), His6-RepE unable diffuse native polyacrylamide gel theoretical pI value                      Figure 3 <br />                                                                                           Crystal His6-RepE&#226;&#8364;&#8220;DNA1 complex. scale bar 100 mm length. <br /> 9.5, complexes able gel owing effect acidic oligonucleotides. result staining gels CBB Mupid-Blue independently confirm presence His6-RepE DNA, respectively, specific sharp bands corre- <br /> sponding complexes detected position gels (Fig. 2). corresponding band lanes His6-RepE DNA loaded independently native gel (data shown). complex His6-RepE&#226;&#8364;&#8220;DNA1  <br />  <br />  <br />  <br /> Figure 2 <br /> Native PAGE crystals His6-RepE&#226;&#8364;&#8220;DNA1 complex. Samples loaded 15% polyacrylamide gel (37.5:1 acrylamide:bisacrylamide) electrophoresed. gels stained Mupid-Blue (left) CBB               Figure 4 <br /> (right) detection DNA1 His6-RepE, respectively. arrows labelled        Pseudo-precession image His6-RepE&#226;&#8364;&#8220;DNA1 crystal representing h0l zone <br /> Complex Free DNA indicate positions His6-RepE&#226;&#8364;&#8220;DNA1 complex                                                                             &#203;&#353; . picture                                                                                           reciprocal space. circle indicates resolution 4 free DNA1, respectively. Lane 1, solution used crystallization; lane 2,            generated HKLVIEW (Collaborative Computational Project, Number 4, <br /> crystallization mother liquor; lanes 3&#226;&#8364;&#8220;4, washing solution; lane 5, dissolved crystals.   1994). <br />  <br />  <br /> 2005) process crystal annealing indispensable          <br /> improve resolution limit X-ray diffraction His6-RepE&#226;&#8364;&#8220;         <br /> DNA1 crystals  approximately 8 3   &#203;&#353; resolution). Diffraction      <br /> intensities recorded maximum resolution 3.14     &#203;&#353; using   <br /> synchrotron radiation, diffraction patterns anisotropic,     <br />                                   &#203;&#353; weakest direction (Fig. 4).   <br /> showing diffraction 4.2                                            diffraction symmetry systematic absences                <br /> diffraction data indicated His6-RepE&#226;&#8364;&#8220;DNA1 crystals belong        monoclinic space group P21, unit-cell parameters = 60.73,     <br />                       &#203;&#353; ,   = 108.55  . Assuming presence      <br /> b = 99.32, c = 95.00                                                                         <br /> His6-RepE&#226;&#8364;&#8220;DNA1 complex asymmetric unit, VM value              <br /> (Matthews, 1968) calculated 3.3     &#203;&#353; 3 Da 1, corre-     <br /> sponds solvent content 63%. diffraction data         collected wavelength 1.6000    &#203;&#353; using crystal,       <br /> significant peaks derived Sm atoms             <br /> anomalous difference Patterson map isomorphous differ-           <br /> ence Patterson map. Structure determination currently progress.        <br />   <br /> </body></html>