<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The 53BP1 Homolog C. elegans Influences DNA Repair Promotes Apoptosis Response Ionizing <br /> Radiation <br /> Jin-Sun Ryu., Sang Jo Kang., Hyeon-Sook Koo* <br /> Department Biochemistry, College Life Science Biotechnology, Yonsei University, Seoul, Republic Korea <br />  <br />  <br />  <br />      Abstract <br />      53BP1 contributes activation G2/M checkpoint downstream ATM MDC1 response ionizing radiation <br />      promotes nonhomologous end-joining (NHEJ) mammalian cells. order determine cellular activities <br />      53BP1 conserved model organism C. elegans, analyzed function homolog, HSR-9 response      DNA damage. Deletion Mos1-insertion hsr-9 did affect sensitivity worms double strand DNA breaks (DSBs), <br />      reflected embryonic survival larval development.  hsr-9 mutations, lig-4 deletion, <br />      reversed hypersensitivity rad-54-deficient worms DSBs. addition, oocyte chromosomal aberrations,      increased rad-54 knockdown response DSBs, reduced hsr-9 mutations. hsr-9 mutations did      prevent cell cycle arrest induced DSBs mitotically proliferating germ cells.  attenuated apoptosis <br />      induced DSBs, CEP-1  p53 ortholog) absent, suggesting HSR-9 functions pathway      CEP-1. concluded 53BP1 homolog C. elegans directly involved cell cycle arrest response DSBs, <br />      promotes apoptosis form NHEJ occurs rad-54 deficient. <br />  <br />   Citation: Ryu J-S, Kang SJ, Koo H-S (2013) 53BP1 Homolog C. elegans Influences DNA Repair Promotes Apoptosis Response Ionizing <br />   Radiation. PLoS 8(5): e64028. doi:10.1371/journal.pone.0064028 <br />   Editor: Shawn Ahmed, University North Carolina, United States America <br />   Received September 30, 2012; Accepted April 10, 2013; Published 8, 2013 <br />   Copyright: &#195;&#376; 2013 Ryu et al. open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Nuclear Research <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Development Program (grant code: 2010-0017583) National Research Foundation Korea <br />   (NRF) grant funded Korean government (MEST)   2012-029610) HSK. funders role study design, data collection analysis, decision   publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: kooh@yonsei.ac.kr <br />   . authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                          suppressing homologous recombination permitting NHEJ. <br />                                                                                       Similarly, 53BP1 ortholog S. cerevisiae, Rad9, inhibits DNA <br />    53BP1 (p53 binding protein 1) discovered p53-binding                      resection DSBs uncapped telomeres [17]. unprotected <br /> protein primary role initially thought stabilizing                  telomeres present TRF2 knockout cells greater mobility p53 response ionizing radiation [1,2].                             presence 53BP1, leads telomere fusions important roles cell cycle checkpoints DNA repair                         NHEJ [18]. Similarly, 53BP1 required class-switch <br /> identified later. double-strand DNA breaks (DSBs)                            recombination facilitates long-range DNA end-joining formed, ATM activated help Mre11/Rad50/                            V(D)J recombination lymphocytes [13,19,20]. <br /> NBS1 (MRN) complex phosphorylates histone H2AX, <br />                                                                                          53BP1 number phosphorylation sites [21] contains <br /> MDC1, 53BP1 [3,4]. MDC1 recruits RNF8 (RING-finger <br />                                                                                       dimerization, tandem Tudor, BRCT domains. dimer- <br /> protein 8), ubiquitinates histones H2A H2AX,                                                                                       ization BRCT domains contribute oligomerization 53BP1 [5&#226;&#8364;&#8220;9]. 53BP1 activates CHK2 SMC1, leads                                                                                       53BP1, BRCT domain mediates interaction activation G2/M S phase checkpoint [2,10&#226;&#8364;&#8220;13]. 53BP1 <br />                                                                                       DNA damage response proteins MRN complex [15]. MDC1 mediate checkpoint activation                                                                                       oligomerized tandem Tudor domain sufficient low doses ionizing radiation, later observed                                                                                       recognition DSBs binds di-methylated histones participate maintaining G2/M arrest high doses ionizing radiation [14]. 53BP1 downstream target                     H3K79 H4K20 [22,23]. DOT1L, H3K79 methyltrans- <br /> ATM, stimulate kinase activity ATM [15].                           ferase, needed recruit 53BP1 DSBs human cell lines,    addition role cell cycle arrest, 53BP1 promotes                       Dot1 plays role nuclear localization Rad9  53BP1 <br /> nonhomologous end-joining (NHEJ) suppresses homologous                            ortholog) S. cerevisiae [24]. histone H4K20 methyltransfer- <br /> recombination (HR) [7]. hand, upstream                               ases, SUV4-20, SET8 (PR-SET7/8), MMSET (WHSC1) checkpoint protein, MDC1, mediates homologous recombination                           reported affect 53BP1 accumulation DSBs human cell lines <br /> [7]. Brca1-deficiency (Brca1D11/D11) induces senescence                       [25&#226;&#8364;&#8220;27]. Likewise, histone methylation H4Lys20 mediated death MEF cells, rescued deletion 53BP1 [16].                 Set9 controls recruitment Crb2  53BP1 ortholog) DSBs suggested rescue occurs 53BP1 inhibits                     S. pombe [28,29]. <br /> DNA-end resection CtIP Brca1-deficient cells,  <br />  <br /> PLOS | www.plosone.org                                                        1                                  2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                  53BP1 C. elegans <br />  <br />  <br />    DNA damage checkpoint proteins, ATM, ATR, CHK1,                    survival proliferating meiotic germ cells CHK2, MRE11/RAD50, 9-1-1 complex, RPA,                            DSB formation. <br /> functionally conserved C. elegans [30&#226;&#8364;&#8220;33]. order assess                 NHEJ important DSB repair non-cycling somatic 53BP1 activities related DSBs conserved C.                cells C. elegans 53BP1 promotes NHEJ mammalian cells <br /> elegans homolog HSR-9, investigated roles HSR-9 cell             [7,37], presumed HSR-9, 53BP1 homolog C. elegans, <br /> cycle control, DNA repair, apoptosis DSB formation.                 influence larval growth DSB formation. late <br />                                                                               embryos treated c-rays (90 Gy) cultivated Results                                                                       adulthood, larvae lig-4(ok716) mutant, deletion <br />                                                                               gene ligase 4, developed slowly wild-type C. elegans 53BP1 homolog, HSR-9 nuclear                           larvae, shown Figure 3A. result similar localization                                                                  previous observations C. elegans mutants deficient NHEJ <br />    carrying BLAST search C. elegans &#226;&#8364;&#732;&#226;&#8364;&#732;orfeome&#226;&#8364;&#8482;&#226;&#8364;&#8482;,           proteins CKU-80 (Ku80 ortholog), CKU-70 (Ku70 <br /> identified protein called HSR-9 27% <span id='am-9' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span>                     ortholog), LIG-4 [38]. contrast, brd-1(gk297) worms, <br /> identity 53BP1 C-terminal BRCT region (Hs 53BP1:                  defective HR, did significant retardation growth <br /> amino acids 1678,1965, Ce HSR-9: 871,1154), 21%                           rate, did hsr-9 mutants. lig-4(ok716) worms <br /> identity N-terminal domain (53BP1: 704,1142, HSR-9:                    developed large number morphological abnormalities 111,562) (Figure 1A).  regions needed nuclear                larvae adults, ruptured body protruding vulva, <br /> focus formation (53BP1, 1251,1271; HSR-9, 564,582)                    wild-type N2, brd-1(gk297) hsr-9(ok759) did tandem Tudor folds (53BP1, 1486,1602; HSR-9, 734,854),                        (Figure 3B). results imply contrary expectation, bind methylated histones, conserved [22,34].                 HSR-9 does play significant role NHEJ assess C. elegans HSR-9 functional homolog 53BP1,             postembryonic development. examined intracellular location using antibodies polypeptide fragment HSR-9 (496,655 ) (Figure 1B).                     hsr-9 lig-4 mutations rescue hypersensitivity Immuno-stained HSR-9 observed nuclei proliferating <br />                                                                               DSBs rad-54-deficient germ cells <br /> germ cells normal conditions localization                                                                                  Homologous recombination (HR) major DSB repair prominent c-irradiation (Figure 1C).                                                                               pathway germ cells cycling somatic cells C. elegans, <br /> signal hsr-9(ok759) germ cells c- <br />                                                                               NHEJ major pathway noncycling somatic cells <br /> irradiation (Figure 1C), confirming antibody specific HSR-9. nuclear level HSR-9 protein increased                  [38].  NHEJ influences sensitivity germ cells UV treatment DNA replication inhibition using                       DNA damage particular genetic backgrounds. example, lig- <br /> hydroxyurea (Figure S1 File S1), suggests role                  4 mutation suppresses hypersensitivity fcd-2  FANCD2 <br /> response various types DNA damage.                                      ortholog) mutant germ cells interstrand DNA crosslinks (ICLs) <br />                                                                               suppresses association oocyte chromosomes seen                                                                               double-deficiency strain fcd-2 meiotic recombination <br /> Mutations hsr-9 affect sensitivity DSBs                                                                               gene [39].  examined possibility lig-4 <br /> germ line noncycling somatic cells                                        mutation affect hypersensitivity DSBs worms <br />    order test HSR-9 role DNA damage                defective homologous recombination. Knockdown C. <br /> response expected, obtained hsr-9(ok759) mutant               elegans ortholog RAD54 participates strand invasion <br /> deletion Exon 5 Exon 7 (Figure 1B). used              branch migration HR [38,40&#226;&#8364;&#8220;43], decreased embry- <br /> knockout mutant hsr-9(ttTi14815), Mos1 insertion               onic survival c-irradiation (Figure 4), reported previously open reading frame (Figure 1B). genetic backgrounds                [38]. deletion mutation ok716 LIG-4, involved mutants cleaned outcrossing wild-type N2              NHEJ pathway, expectedly rescued embryonic lethality strain. irradiated L4 stage worms c-rays scored              rad-54(RNAi) worms c-irradiation (Figure 4A). hatching progeny embryos, detect effects hsr-9 <br />                                                                                  tested HSR-9, homolog promotes <br /> mutations sensitivity DSBs (Figure 2A). mutation gk297 <br />                                                                               NHEJ mammalian cells [7,16,18], influences DSB repair brd-1 encodes C. elegans BARD1 homolog, decreased <br />                                                                               worms deficient RAD-54.  hsr-9 mutations, ok759 <br /> embryonic hatching c-irradiation, expected role                                                                               ttTi14815, rescued embryonic lethality rad-54(RNAi) HR [35]. contrast, hatching hsr-9(ok759) deletion mutant <br />                                                                               45 90 Gy (Figure 4B), did lig-4(ok716) mutation. Rescue significantly affected 60 Gy (Student&#226;&#8364;&#8482;s t test, p value <br />                                                                               hsr-9 mutations observed respect oocyte <br /> = 0.48) 120 Gy (p = 0.99) (Figure 2A). did Mos1 <br />                                                                               chromosomal aberrations (defined abnormal numbers insertion mutant, hsr-9(ttTi14815), differ wild-type N2 <br />                                                                               chromosomes) c-ray treatment (60 90 Gy) (Figure 4C): <br /> strain sensitivity c-rays  60 Gy, p = 0.13; 120 Gy, <br />                                                                               percentage abnormal oocytes 2 fold greater (2.0 <br /> p = 0.36). hsr-9 mutations did significantly affect                                                                               fold 60 Gy, 1.7 fold 90 Gy) rad-54(RNAi) wild <br /> survival L4-stage germ cells mitotically proliferating <br />                                                                               type, increase rad-54 knockdown significantly meiotic pachytene stage, worms instead c-irradiated                                                                               hsr-9 mutant backgrounds (,1.4 fold 60 Gy, ,1.1 <br /> L1 stage (Figure 2B). L1 survival assays, radiation-sensitivity mitotically proliferating germ cells measured               fold 90 Gy). <br /> scoring adult fertility number surviving larvae generation [36]. number living larvae significantly                hsr-9 mutations influence normal cell cycle arrest <br /> reduced compared wild type positive control, brd-             response DSBs <br /> 1(gk297).  hsr-9 mutations, ok759                    detect role HSR-9 DNA damage checkpoint, <br /> ttTi14815, significantly decreased number worms produced               gonads stained DAPI 12 h c-ray treatment (75 25 50 Gy irradiation (N2 vs. ok759, p = 0.05 0 Gy,               Gy) number nuclei mitotic region gonad <br /> p.0.5 25 50 Gy; N2 vs. ttTi14815, p.0.5 0, 25, 50               arm counted. Numbers germ cells decreased Gy).  concluded hsr-9 mutations affect           extent hsr-9 mutants wild-type N2 (N2, <br />  <br />  <br /> PLOS | www.plosone.org                                                2                              2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                 53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 1. 53BP1 homolog C. elegans, HSR-9 protein, accumulates nuclei C. elegans germ cells response c-rays.   <br /> HSR-9 similar predicted open reading frames C. elegans human 53BP1 C-terminal BRCT domains (27% amino acid <br /> identity) N-terminus (21% amino acid identity). regions controlling nuclear focus formation (Foci) Tudor domains conserved <br /> [22,34]. (B) deletion hsr-9(ok759) shown blue representation gene, Mos1 insertion brown. polypeptide <br /> fragment used antibody production indicated pink bar. (C) HSR-9 immuno-localized nuclei wild-type germ cells mitotically proliferating region gonads 3 h c-ray (75 Gy) treatment. Scale bar, 10 mm. <br /> doi:10.1371/journal.pone.0064028.g001 <br />  <br /> Dn = 4962(SEM); hsr-9(ok759), Dn = 4562; hsr-9(ttTi14816),                  pachytene stage, detected observing cell <br /> Dn = 4762; p.0.5 N2 hsr-9 mutants),                 corpses Nomarski optics staining condensed <br /> reduction atm-1(ok186) (N2, Dn = 4962(SEM); atm-                chromosomes [47]. L4 stage hermaphrodites treated 1(gk186), Dn = 2562; p,0.03) (Figure 5A). suggests                c-rays (120 Gy), dying germ cells pachytene stage unlike ATM-1, HSR-9 significantly involved cell cycle             scored 12 h intervals staining SYTO-12. hsr-9(ok759) <br /> arrest. quantified extent cell cycle arrest               hsr-9(ttTi14815) mutants contained fewer apoptotic cells measuring diameters germ cell nuclei mitotically              wild-type worms (Figure 6A).  reductions apoptotic <br /> proliferating regions gonads. Germ cells normally enlarge          cell numbers hsr-9(ok759) hsr-9(ttTi14815) c-irradiation G2 arrest subsequent cell growth [44&#226;&#8364;&#8220;46].          cep-1(lg12501), deletion gene coding P53 c-ray treatment, nuclear diameters hsr-9 atm-         ortholog [48,49]. order determine HSR-9 influences <br /> 1 mutants slightly larger wild type N2 (N2,              apoptosis pathway CEP-1, measured apoptotic <br /> average nuclear diameter (d) = 3.6060.02(SEM)mm; hsr-9(ok759),              cells hsr-9(ok759);cep-1(RNAi) c-ray treatment (120 Gy, 24 <br /> 3.8760.02 mm; atm-1(gk186), 3.9260.02 mm). expected,               h later). apoptotic cell number strain similar c-ray treatment, average values (d) increased            cep-1(lg12501) slightly higher cep-1(RNAi) (hsr- <br /> strains (N2, 5.0160.04(SEM) mm; hsr-9(ok759), 4.9460.04 mm;                 9(ok759);cep-1(RNAi) vs. cep-1(lg12501), p = 0.62; hsr-9(ok759);cep- <br /> atm-1(gk186), 4.4260.04 mm).  distribution nucle-            1(RNAi) vs. cep-1(RNAi) p,0.01) (Figure 6B). fact ar sizes hsr-9(ok759) worms closer wild-         reductions apoptotic cell numbers single deficiency <br /> type worms atm-1(ok186) worms, consequently             strains did add reduction double deficiency true averages nuclear diameters (Figure 5B).            strain, suggests HSR-9 functions pathway CEP- result agrees Figure 5A, supporting view         1. confirm  transcripts egl-1, cell-death activator <br /> HSR-9 does play significant role cell cycle arrest           downstream cep-1 [50], measured gene-specific real- <br /> DSB formation, important ATM-1.              time PCR reverse transcribing total RNA (Figure S2 File <br />                                                                             S1). transcripts significantly lower hsr-9(ok759) worms <br /> hsr-9 mutations attenuate apoptosis induced DSBs                         wild-type worms (p = 0.01), low cep- <br />    did detect role HSR-9 cell cycle arrest          1(lg12501) worms. reduced level egl-1 transcripts hsr- response DSBs, examined deletion mutation              9(ok759) likely contributed reduction apoptosis. <br /> affects apoptosis induced DSBs. C. elegans germline, half germ cells spontaneously undergo apoptosis  <br />  <br /> PLOS | www.plosone.org                                              3                               2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                          53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 2. HSR-9 does affect survival L4 stage germ cells proliferation L1 stage germ cells.   Embryonic hatching scored exposing L4 stage worms wild-type N2 hsr-9 mutants c-rays collecting embryos 24 48 h irradiation. <br /> Hatching rates wild-type N2, hsr-9(ok759), hsr-9(ttTi14815) mutant embryos significantly different 60 Gy (Student&#226;&#8364;&#8482;s t test: N2 vs. <br /> ok759, p = 0.48; N2 vs. ttTi14815, p = 0.13) 120 Gy (N2 vs. ok759, p = 0.99; N2 vs. ttTi14815, p = 0.36). (B) P0 worms strain exposed c-rays (25 50 Gy) L1 stage grown 2 days. F1 embryos collected following 24 h, live larvae (L1-L3 stage) hatched embryos counted 24 later. p values N2 hsr-9(ok759 ttTi14815) radiation doses larger 0.5, p = 0.05 0 Gy N2 hsr-9(ok759). <br /> doi:10.1371/journal.pone.0064028.g002 <br />  <br />  <br /> Discussion                                                                       File S1, brc-1;hsr-9 vs. brc-1, p.0.3). difference probably <br />                                                                                  related fact CTIP homolog C. elegans, COM-1,    53BP1-deficient mice growth-retarded, immuno-deficient,                   required RAD-51 loading exogenously-induced DSBs, <br /> radiation-sensitive, cancer prone [51]. contrast,                essential meiotic DSBs generated SPO-11 [52]. C. elegans mutants 53BP1 homolog, HSR-9,                     C. elegans CTIP does appear contribute hypersensitive DSBs measured embryonic survival                     processing exogenously-induced DSBs unlikely larval growth c-ray treatment (Figures 1 2).               interact BRC-1 end-resection. Instead, C. elegans <br /> hsr-9 mutations unexpectedly reversed hypersensitivity DSBs               homolog RAD54, involved strand invasion rad-54-deficient worms, suggesting role DSB                  branch migration homologous recombination, interacts <br /> repair (Figure 4). observation similar finding           53BP1 homolog, HSR-9 (Figures 4B 4C). deletion phenotypes BRCA1-deficient (Brca1D11/D11) mice,                   mutation lig-4, involved NHEJ pathway, rescued <br /> embryonic lethality, senescence, cell death, rescued                  embryonic lethality rad-54 knockdown c-ray <br /> 53BP1 knockout [16]. suppression BRCA1-deficiency <br />                                                                                  treatment (Figure 4A). fact HSR-9 LIG-4 phenotypes 53BP1 mutation explained                                                                                  involved similar genetic interactions RAD-54 suggests mechanism 53BP1 inhibits end-resection DSBs,                                                                                  HSR-9 probably promotes NHEJ context HR defect <br /> shifting DSB repair NHEJ HR [37].  <br />                                                                                  associated RAD-54 deficiency. clear hsr-9 HR compromised BRCA1-deficient cells leads <br />                                                                                  mutation rescues rad-54 knockdown phenotypes, mechanism error-prone repair NHEJ, absence 53BP1 allows DSBs <br />                                                                                  responsible evident. possibility RAD-54 interacts repaired error-free HR. test genetic <br />                                                                                  MRE-11 EXO-1 nucleases end-resection relationship conserved C. elegans, generated double <br />                                                                                  DSBs, BRCA1 does CTIP mammalian cells. mutant brc-1 (BRCA1 homolog) hsr-9.  brc- <br />                                                                                  possibility chromatin modulation RAD-54, SNF2 <br /> 1(tm1145);hsr-9(ok759) mutant hypersensitive DSBs                                                                                  family protein, normally counterbalanced binding HSR-9 <br /> brc-1(tm1145), contrast mammalian situation (Figure S3  <br />  <br /> PLOS | www.plosone.org                                                   4                                 2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                   53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 3. hsr-9 deletion does affect post-embryonic development c-ray treatment. Late embryos wild-type N2, hsr-9(ok759), <br /> lig-4(ok716), brd-1(gk297) treated c-rays (90 Gy).   Larvae hatched treated untreated embryos examined day estimate growth rates, percentages worms different larval stages 48 h irradiation calculated. (B) Developmental <br /> abnormalities treated wild-type (w.t.) mutant worms scored 3rd day adulthood (120 h irradiation). Phenotypes scored ruptured body (rup), protruding vulva (pvl), bag worms (hatched larvae inside mothers) growth arrest (arrest). <br /> doi:10.1371/journal.pone.0064028.g003 <br />  di-methylated histones near DSBs [22,23]: absence               apoptosis response DSBs high-throughput protein <br /> RAD-54 DSBs, abnormal chromatin structures formed                  interaction analysis DNA damage responses C. elegans [56]. result HSR-9 activity, HR proceed.                        contrast effects, did observe effects hsr-9 <br />    pan-nuclear level HSR-9 significantly increased             mutations progeny survival cell cycle arrest.  <br /> germ cells DSB formation (Figure 1C),                hsr-9 mutations did attenuate germline apoptosis DSB <br /> clear foci HSR-9 contrast case mammalian 53BP1.              formation (Figure 6), like hsr-9 knockdown previous report <br /> Similarly, C. elegans ATM-1 did form nuclear foci unlike             [56]. Double deficiency hsr-9 cep-1  p53 ortholog C. <br /> mammalian homolog [53], implying homologs                        elegans) did decrease apoptotic cell numbers mammalian proteins form nuclear foci response DNA                 germline compared single deficiency cep-1, suggesting damage form foci worms. 53BP1 S/                HSR-9 promotes apoptosis induced DSBs TQ sites phosphorylated ionizing UV radiation,             pathway CEP-1. fact egl-1 mRNA expression, which consistent idea ATM ATR catalyze              induced CEP-1, decreased hsr-9 mutation phosphorylation [21].  observed nuclear HSR-9                 confirms proteins function pathway <br /> increased inhibiting DNA replication hydroxyurea              (Figure S2 File S1). HSR-9 probably mediates activation exposure UV&#226;&#8364;&#8220;radiation (Figure S1 File S1). Despite           CEP-1 response DSBs, 53BP1 contributes nuclear accumulation HSR-9, hsr-9 mutations did affect            accumulation p53 mammalian cells [2]. intriguing arrest germ cell proliferation induced DSBs, atm-          RAD-54 MRE-11, acting homologous recombination <br /> 1 mutation did partially prevent arrest (Figure 5).  HSR-           pathway, promote apoptosis response UV-C radiation C. <br /> 9 does participate checkpoint activation, unlike          elegans [41], HSR-9 does response DSBs. <br /> homolog mammalian cells. reason difference                conclude HSR-9 mediates apoptosis induced DSBs fact C. elegans homolog CHK2, downstream                pathway CEP-1.  clear <br /> target 53BP1, does play significant role response         evidence HSR-9 functions cell cycle arrest. Similarly, <br /> DSBs [54,55]. reason high dose c-rays                normal cell cycle arrest observed response DSBs used, need 53BP1 checkpoint activation              knocking CEP-1 [49]. spite fact HSR-9 observed low doses ionizing radiation [2,12].                 appears function DSB repair wild type genetic <br />    Previously, hsr-9 knockdown result reduced                background, HSR-9 mutations reverse hypersensitivity progeny survival, defective cell cycle arrest, attenuated                RAD-54-deficient cells DSBs like LIG-4 deficiency.  <br />  <br /> PLOS | www.plosone.org                                               5                               2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                          53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 4. hsr-9 mutations rescue hypersensitivity rad-54-deficient worms c-rays lig-4 mutations  Wild-type N2, hsr- <br /> 9(ok759), hsr-9(ttTi14815), lig-4(ok716) worms fed E. coli cells containing double-stranded RNA rad-54 L1 stage.   L4 stage <br /> worms deficient lig-4 rad-54, treated c-rays (45 Gy 90 Gy), embryos collected 24 48 h irradiation. p values hatching rates  rad-54 vs. lig-4;rad-54, p = 0.01 45 Gy p = 0.07 90 Gy; lig-4 vs. lig-4;rad-54, p = 0.24 45Gy p = 0.26 90 Gy. (B) L4 stage worms deficient hsr-9 rad-54, treated c-rays  . p values hatching rates 45 Gy  hsr-9(ok759 ttTi14815) vs. rad-54, p,0.01; rad-54 vs. hsr-9;rad-54, p,0.03; hsr-9 vs. hsr-9;rad-54, p.0.20. hatching rates hsr-9 hsr-9;rad-54 did differ significantly 90 Gy (p.0.14). (C) L4 stage worms irradiated 60 90 Gy c-rays fixed 24 <br /> h. staining worms DAPI, number chromosomes oocyte counted fluorescence microscopy (80 oocytes strain). percentage chromosomal aberrations calculated number oocytes containing abnormal number chromosomes  opposed 6 bivalent chromosomes) divided total number oocytes observed. <br /> doi:10.1371/journal.pone.0064028.g004 <br />  <br /> suggests HSR-9 shift choice DSB repair pathway                   brc-1(tm1145) mutants obtained Caenorhabditis HR NHEJ, just 53BP1 promotes NHEJ BRCA1-                           Genetics Center (Minneapolis, MN, USA). obtained deficient cells.  roles 53BP1 mammalian cells,               validated hsr-9(ttTi14815) strain  Mos1 transposon inserted <br /> mediating apoptosis shunting DSB repair NHEJ,                    7th exon hsr-9) Drs. Laurent Se&#194;&#180;galat (Universite&#194;&#180; <br /> degree conserved 53BP1 ortholog C. elegans [16,37].                    Claude Bernard Lyon 1, France) Ma&#196;&#177;&#194;&#168;te&#194;&#180; Carre-Pierrat <br />                                                                                  (Plateforme &#226;&#8364;&#732;&#226;&#8364;&#732;Biologie Caenorhabditis elegans&#226;&#8364;&#8482;&#226;&#8364;&#8482;, France). Materials Methods                                                            generate uniform genetic backgrounds, hsr-9(ok759), hsr- <br />                                                                                  9(ttTi14815), brd-1(gk297), brc-1(tm1145), lig-4(ok716) C. elegans strains                                                               outcrossed wild type N2 worms 4, 3, 3, 6, 2 times, <br />    standard wild-type Bristol N2 strain, hsr-                 respectively. double mutant brc-1(tm1145);hsr-9(ok759) 9(ok759), atm-1(gk186), brd-1(gk297), lig-4(ok716), cep-1(lg12501), <br />  <br />  <br /> PLOS | www.plosone.org                                                   6                                 2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                    53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 5. Cell cycle arrest response c-rays normal hsr-9(ok7599) mutants. Wild-type N2, hsr-9(ok759), hsr-9(ttTi14815), atm-1(gk186) worms treated 75 Gy c-irradiation L4 stage larvae. hours later, gonads isolated stained DAPI.   <br /> Germ cells mitotically proliferating regions gonads  75 mm distal tip cell) counted fluorescence microscope <br /> (30 gonad arms data point). (B) diameters 40 nuclei mitotically proliferating region gonad arm measured (number nuclei = 800 strain irradiation) wild-type N2, hsr-9(ok759), atm-1(gk186) worms. percentages total nuclei different diameters plotted. small (,2 mm) large (.7 mm) nuclei grouped  <br /> doi:10.1371/journal.pone.0064028.g005 <br />  <br /> generated crossing brc-1(tm1145) hsr-9(ok759). atm-               L4-stage gem cell survival oocyte chromosomal <br /> 1(gk186) mutant previously outcrossed 6 times [53].              aberration c-ray treatment <br /> strains maintained feeding E. coli OP50-1 cells                   wild type, hsr-9(ok759), hsr-9(ttTi14815), brd-1(gk297), <br /> 20uC. confirm Mos1 transposon inserted hsr-                    lig-4(ok716), rad-54(RNAi), hsr-9;rad-54(RNAi), lig-4(ok716);- <br /> 9(ttTi14815) spontaneously excised, genomic DNA                   rad-54(RNAi) worms L4 stage exposed 60 Gy inserted transposon amplified hsr-                       120 Gy  45 Gy 90 Gy) c-rays using 137Cs source (IBL <br /> 9(ttTi14815) (Figure S4 File S1). genomic DNA                     437C, CIS Biointernational). Embryos collected 24 <br /> fragment analyzed agarose gel electrophoresis, Mos1                48 h irradiation, hatching scored 24 h <br /> insertion present.                                            later. score oocytes containing abnormal numbers chromo- <br />                                                                               somes, worms fixed 24 h irradiation (60 90 Gy), <br /> Bacteria-mediated RNAi                                                        stained DAPI, observed fluorescence microscope <br />    E. coli HT115(DE3) cells harboring plasmid containing                (DMRHC, Leica). <br /> cDNA cep-1 rad-54 obtained C. elegans RNAi <br /> v1.1 feeding library (Open Biosystems) Ahringer&#226;&#8364;&#8482;s RNAi                    Survival L1-stage germ cells c-ray treatment <br /> feeding library, respectively. E. coli cells cultured                survival L1-stage germ cells measured according liquid LB medium containing ampicillin (50 mg  tetracy-                protocol Craig et al. [36] slight modification. Wild <br /> cline (5 mg  12 h 37uC. Aliquots cultures               type, hsr-9(ok759), hsr-9(ttTi14815) brd-1(gk297) worms spread NGM plates containing 1 mM IPTG carbenicillin                   L1 stage treated 25 Gy 50 Gy c-rays. 48 h, <br /> (25 mg , left room temperature 24 h. rad-54                 worms P0 generation plated 3 NGM <br /> RNAi cep-1 RNAi, C. elegans worms grown plates                plates containing E. coli OP50-1 lawn, embryos collected L1 stage.                                                            24 h. number living worms (F1 generation) counted <br />                                                                               24 h later. <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                                7                               2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                 53BP1 C. elegans <br />  <br />  <br />  <br />  <br /> Figure 6. hsr-9 mutations attenuate apoptosis induced c-rays.   Wild type, hsr-9(ok759), hsr-9(ttTi14815), cep-1(lg12501) L4 stage <br /> worms irradiated c-rays (120 Gy), stained SYTO-12 dye 0, 12, 24, 36 h c-ray treatment. Intensely stained pachytene <br /> stage apoptotic cells scored gonad arm. (B) Wild type hsr-9(ok759) worms fed E. coli cells containing double-stranded RNA cep-1 L1 stage. Wild type, hsr-9(ok759), cep-1(lg12501), cep-1(RNAi), hsr-9(ok759);cep-1(RNAi) L4 stage worms treated c-rays <br /> (120 Gy), apoptotic germ cells counted  . <br /> doi:10.1371/journal.pone.0064028.g006 <br />  <br />  <br /> Measurement developmental abnormalities c-ray                        SYTO-12 staining apoptotic germ cells <br /> treatment                                                                        identify apoptotic cells induced double-strand DNA <br />    adult worms allowed lay eggs 1 h.                 breaks, wild type, hsr-9(ok759), hsr-9(ttTi14815), cep-1(lg12501), incubation 4 h, late-stage embryos irradiated           hsr-9(ok759);cep-1(RNAi) L4 stage worms irradiated c-rays (90 Gy). Hatched worms examined stereomi-                  c-rays (120 Gy). Worms harvested 0, 12, 24, 36 h croscope day estimate growth rates                c-ray treatment, incubated 33 mM SYTO-12 2 h control (untreated wild-type) worms reached L4 stage.                     25uC. 3 washes 16 PTw, allowed recover Developmental abnormalities scored controls                     1 h NGM plates bacterial lawn, mounted reached 3rd adult day. Phenotypes scored ruptured body               glass slides agar pads. Apoptotic cells pachytene germ <br /> (rup), protruding vulva (pvl), bag worms (hatched larvae inside            stage observed fluorescence microscope. mothers) growth arrest (arrest). <br />                                                                               Preparation anti-HSR-9 antibody <br /> Assessment cell cycle arrest c-ray treatment                            Total RNA isolated mixed-stage C. elegans worms <br />    measure effect c-rays cell cycle arrest, wild type,           using easy-BLUE Total RNA Extraction kit (Intron Biotech- <br /> hsr-9(ok759), hsr-9(ttTi14815), atm-1(gk186) larvae treated          nology). cDNA pool synthesized total RNA using L4 stage c-rays (75 Gy). 12 h, gonads            M-MLV reverse transcriptase (Intron Biotechnology) oli- <br /> dissected, stained DAPI, observed fluorescence                dT) primer, length cDNA HSR-9 amplified <br /> microscope (DMRHC, Leica). Cell cycle stages determined                  gene-specific PCR. PCR used amplify cDNA methods.  counted number nuclei                 fragment (nucleotides 1486,1965) open reading frame, <br /> 75 mmof distal tip cells gonad,  mitotically       cloned pGEM-T vector (Promega). forward <br /> proliferating region. Second, measured diameters              reverse primers 59-GGATCCCGTAGATCTA- <br /> 40 nuclei mitotically proliferating region gonad.              CAAGGGCTAAG 59-GAATTCCGAAGTTCCTACA- <br />                                                                               GACGATAC, respectively. recombinant plasmid DNA                                                                               digested BamHI EcoRI sites (underlined), cDNA <br />  <br />  <br /> PLOS | www.plosone.org                                                8                             2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                                     53BP1 C. elegans <br />  <br />  <br /> product recloned pGEX4T-1 vector. resulting                               stained ethidium bromide, visualized UV <br /> plasmid pGEX4T-1/HSR-9 transferred E. coli                                    transilluminator. <br /> BL21(DE3) cells, protein expression induced incubation 1 mM IPTG 4 h 37uC. fusion protein                          Supporting Information isolated used raise antibodies SD rats. <br />                                                                                        File S1     Figure S1, HSR-9 protein accumulates Immunostaining gonads                                                               nuclei C. elegans germ cells response various <br />    heads adult C. elegans worms cut open,                          types DNA damage.   HSR-9 immuno-localized gonads intestines pushed  fixed 3%                               nuclei wild-type germ cells mitotically proliferating region <br /> paraformaldehyde 0.1 M K2HPO4 (pH 7.2) 1 h room                             gonads response hydroxyurea (HU, 60 mM) treatment temperature. washed 5 times 16 PTw                             16 h. (B) Nuclear levels HSR-9 3 h UV (150 J/m2) <br /> placed methanol 220uC overnight post-fixation. 5                       irradiation. Scale bar, 10 mm. Figure S2, hsr-9 deletion <br /> washes 16 PTw, samples left room temperature                        decreases egl-1 mRNA expression c-ray treatment. <br /> 1 h blocking solution  mixture equal volumes goat serum                     Adult wild type, hsr-9(ok759), cep-1(lg12501) worms (Gibco-BRL) 16 PTw). fixed specimens reacted                              irradiated 120 Gy c-rays, total RNA isolated 6 h HSR-9 antiserum diluted blocking solution (1:50) 4uC                        later. cDNA pools prepared reverse transcription, overnight. washes 16PTw, specimens incubated                        relative amounts egl-1 cDNA measured real-time PCR. FITC-conjugated goat anti-rat immunoglobulin (1:1000                              Figure S3, hsr-9 mutation relieve hypersen- <br /> dilution, Molecular Probes) room temperature 1 h.                          sitivity brc-1 mutants c-rays. Wild-type N2, hsr- washed 3 times, stained DAPI 15 min, washed                          9(ok759), brc-1(tm1145), brc-1(tm1145);hsr-9(ok759) worms <br /> twice 16 PTw. placed glass slides coated                        irradiated c-rays (45 Gy 90 Gy) L4 stage. 2% agarose, followed addition antifade reagent                              Embryos collected 24 48 h irradiation, <br /> (Invitrogen), observed fluorescence microscope                              hatching scored 24 h later. brc-1(tm1145) mutation <br /> (DMRHC, Leica).                                                                        hsr-9(ok759) resulted hypersensitivity DSBs.                                                                                        double mutant similar single brc-1 mutant Quantitative RT-PCR egl-1 mRNA                                                      sensitivity DSBs  45 Gy 90 Gy, brc-1 vs. brc-1;hsr-9, <br />    Adult wild type, hsr-9(ok759), cep-1(lg12501) worms                        p.0.3). Figure S4, presence Mos1 insertion irradiated 120 Gy c-rays incubated 6 h. Total                          hsr-9(ttTi14815) mutant confirmed PCR amplifica- <br /> RNA isolated using easy-BLUE Total RNA Extraction kit                           tion genomic DNA fragment. Single worms picked <br /> (Intron Biotechnology). Total RNA (2 mg) reverse-transcribed                       lysed, followed PCR amplification genomic DNA <br /> using M-MLV reverse transcriptase (Intron Biotechnology)                           fragment flanking insertion site. 0.75 kb DNA fragment oligo(dT) primer. Relative amounts egl-1 cDNA measured                         produced wild-type worms, longer DNA real-time PCR using IQ SYBR Green Supermix (BIORAD)                              fragment (2.0 kb) hsr-9(ttTi14815) mutant. CFX96 Detection (BIORAD). Primer pairs                                   (PDF) <br /> designed suggested Hofmann et al [30], c-tubulin used standard calculate relative abundance egl-1                        Acknowledgments <br /> mRNA. primer pairs egl-1, 59-CAGGACTTC- <br /> TCCTCGTGTGAAGATTC 59-GAAGTCATCGCACAT-                                              C. elegans N2, hsr-9(ok759), atm-1(gk186), brd-1(gk297), lig-4(ok716), cep- <br /> TGCTGCTA, c-tubulin, 59-AAGATCTATTGTTC-                                        1(lg12501), brc-1(tm1145) provided C. elegans Genetics <br />                                                                                        Center (St. Paul, MN), funded NIH Office Research <br /> TACCAGGC 59-CTTGAA CTTCTTGTCCTTGAC. <br />                                                                                        Infrastructure Programs (P40 OD010440). hsr-9(ttTi14815)                                                                                        UMS3421 provided Drs. Laurent Se&#194;&#180;galat (Universite&#194;&#180; Claude <br /> Identification Mos1 insertion PCR amplification                               Bernard Lyon 1, France) Ma&#196;&#177;&#194;&#168;te&#194;&#180; Carre-Pierrat (Plateforme &#226;&#8364;&#732;&#226;&#8364;&#732;Biologie genomic DNA                                                                            Caenorhabditis elegans&#226;&#8364;&#8482;&#226;&#8364;&#8482;, France). thank Mi Park (Yonsei Univ.)    detect Mos1 insertion 7th exon hsr-9 gene,                     assistance rad-54 experiments. <br /> single wild type hsr-9(ttTi14815) worms lysed, genomic DNA fragment containing insertion site                                 Author Contributions <br /> amplified. PCR carried 2720 Thermal Cycler                                Conceived designed experiments: HSK. Performed experi- <br /> (Applied Biosystems) using primers, 59-TCTCCTGCTGCTAA-                                 ments: JSR SJK. Analyzed data: JSR SJK HSK. Contributed reagents/ <br /> GAATCG 59-ACGAGTAGCCTCCGATATTGT. Ampli-                                            materials/analysis tools: HSK. Wrote paper: SJK HSK. <br /> fied DNA fragments electrophoresed 1% agarose gel, <br />  <br /> References <br /> 1. Iwabuchi K, Bartel PL, Li B, Marraccino R, Fields S (1994) cellular              6. Mailand N, Bekker-Jensen S, Faustrup H, Melander F, Bartek J, et al. (2007) <br />    proteins bind wild-type mutant p53. Proc Natl Acad Sci U S            RNF8 ubiquitylates histones DNA double-strand breaks promotes <br />    91: 6098&#226;&#8364;&#8220;6102.                                                                          assembly repair proteins. Cell 131: 887&#226;&#8364;&#8220;900. <br /> 2. Wang B, Matsuoka S, Carpenter PB, Elledge SJ (2002) 53BP1, mediator         7. Xie  Hartlerode  Stucki M, Odate S, Puget N, et al. (2007) Distinct roles    DNA damage checkpoint. Science 298: 1435&#226;&#8364;&#8220;1438.                                          chromatin-associated proteins MDC1 53BP1 mammalian double-strand <br /> 3. Anderson L, Henderson C, Adachi Y (2001) Phosphorylation rapid                      break repair. Mol Cell 28: 1045&#226;&#8364;&#8220;1057. <br />    relocalization 53BP1 nuclear foci DNA damage. Mol Cell Biol 21:           8. Minter-Dykhouse K, Ward  Huen MS, Chen J, Lou (2008) Distinct versus <br />    1719&#226;&#8364;&#8220;1729.                                                                              overlapping functions MDC1 53BP1 DNA damage response 4. Rappold  Iwabuchi K, <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>Date</span> T, Chen J (2001) Tumor suppressor p53 binding               tumorigenesis. J Cell Biol 181: 727&#226;&#8364;&#8220;735. <br />    protein 1 (53BP1) involved DNA damage-signaling pathways. J Cell Biol          9. Eliezer Y, Argaman L, Rhie  Doherty AJ, Goldberg M (2009) direct <br />    153: 613&#226;&#8364;&#8220;620.                                                                           interaction 53BP1 MDC1 required recruitment 53BP1 <br /> 5. Huen MS, Grant R, Manke  Minn K, Yu X, et al. (2007) RNF8 transduces              sites damage. J Biol Chem 284: 426&#226;&#8364;&#8220;435. <br />    DNA-damage signal histone ubiquitylation checkpoint protein assembly. <br />    Cell 131: 901&#226;&#8364;&#8220;914. <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                                         9                                   2013 | Volume 8 | Issue 5 | e64028 <br />                                                                                                                                                           53BP1 C. elegans <br />  <br />  <br /> 10. DiTullio RA Jr, Mochan TA, Venere M, Bartkova J, Sehested M, et al. (2002)             34. Zgheib O, Pataky K, Brugger J, Halazonetis TD (2009) oligomerized 53BP1 <br />     53BP1 functions ATM-dependent checkpoint pathway constitu-                   tudor domain suffices recognition DNA double-strand breaks. Mol Cell <br />     tively activated human cancer. Nat Cell Biol 4: 998&#226;&#8364;&#8220;1002.                               Biol 29: 1050&#226;&#8364;&#8220;1058. <br /> 11. Wu J, Prindle MJ, Dressler GR, Yu X (2009) PTIP regulates 53BP1 SMC1               35. Boulton SJ, Martin JS, Polanowska J, Hill  Gartner  et al. (2004) BRCA1/ <br />     DNA damage sites. J Biol Chem 284: 18078&#226;&#8364;&#8220;18084.                                     BARD1 orthologs required DNA repair Caenorhabditis elegans. Curr Biol <br /> 12. Fernandez-Capetillo O, Chen HT, Celeste  Ward  Romanienko PJ, et al.                   14: 33&#226;&#8364;&#8220;39. <br />     (2002) DNA damage-induced G2-M checkpoint activation histone H2AX                   36. Craig AL, Moser SC, Bailly AP, Gartner (2012) Methods studying     53BP1. Nat Cell Biol 4: 993&#226;&#8364;&#8220;997.                                                       DNA damage response Caenorhabdatis elegans germ line. Methods Cell <br /> 13. Ward IM, Reina-San-Martin B, Olaru  Minn K, Tamada K, et al. (2004)                      Biol 107: 321&#226;&#8364;&#8220;352. <br />     53BP1 required class switch recombination. J Cell Biol 165: 459&#226;&#8364;&#8220;464.            37. Bunting SF, Callen E, Wong N, Chen HT, Polato F, et al. (2010) 53BP1 inhibits <br /> 14. Shibata  Barton O, Noon  Dahm K, Deckbar D, et al. (2010) Role                     homologous recombination Brca1-deficient cells blocking resection     ATM damage response mediator proteins 53BP1 MDC1                        DNA breaks. Cell 141: 243&#226;&#8364;&#8220;254. <br />     maintenance G(2)/M checkpoint arrest. Mol Cell Biol 30: 3371&#226;&#8364;&#8220;3383.                  38. Clejan  Boerckel J, Ahmed S (2006) Developmental modulation 15. Lee JH, Goodarzi AA, Jeggo PA, Paull TT (2010) 53BP1 promotes ATM                          nonhomologous end joining Caenorhabditis elegans. Genetics 173: 1301&#226;&#8364;&#8220; <br />     activity direct interactions MRN complex. EMBO J 29: 574&#226;&#8364;&#8220;                 1317. <br />     585.                                                                                   39. Adamo  Collis SJ, Adelman CA, Silva N, Horejsi  et al. (2010) Preventing <br /> 16. Cao L, Xu X, Bunting SF, Liu J, Wang RH, et al. (2009) selective requirement             nonhomologous end joining suppresses DNA repair defects Fanconi anemia. <br />     53BP1 biological response genomic instability induced Brca1               Mol Cell 39: 25&#226;&#8364;&#8220;35. <br />     deficiency. Mol Cell 35: 534&#226;&#8364;&#8220;541.                                                      40. Nimonkar AV, Dombrowski CC, Siino JS, Stasiak AZ, Stasiak  et al. (2012) <br /> 17. Lazzaro F, Sapountzi V, Granata M, Pellicioli  Vaze M, et al. (2008) Histone             Saccharomyces cerevisiae Dmc1 Rad51 proteins preferentially function     methyltransferase Dot1 Rad9 inhibit single-stranded DNA accumulation                Tid1 Rad54 proteins, respectively, promote DNA strand invasion     DSBs uncapped telomeres. EMBO J 27: 1502&#226;&#8364;&#8220;1512.                                         genetic recombination. J Biol Chem 287: 28727&#226;&#8364;&#8220;28737. <br /> 18. Dimitrova N, Chen YC, Spector DL, Lange T (2008) 53BP1 promotes non-                41. Stergiou L, Eberhard R, Doukoumetzidis K, Hengartner MO (2011) NER     homologous end joining telomeres increasing chromatin mobility. Nature               HR pathways act sequentially promote UV-C-in <br /> </body></html>