<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 18 October 2011                                  Nucleic Acids Research, 2012, Vol. 40, Database issue D687&#226;&#8364;&#8220;D694 <br />                                                                                                               doi:10.1093/nar/gkr827 <br />  <br />  <br /> PhenoM: database morphological phenotypes <br /> caused mutation essential genes Saccharomyces cerevisiae <br /> Ke Jin1,2, Jingjing Li1,2,3, Frederick S. Vizeacoumar4, Zhijian Li1,2,3, Renqiang Min1,5, <br /> Lee Zamparo1,5, Franco J. Vizeacoumar1,2, Alessandro Datti4,6, Brenda Andrews1,2,3,*, <br /> Charles Boone1,2,3,* Zhaolei Zhang1,2,3,5,* <br /> 1 <br />  Donnelly Centre Cellular Biomolecular Research, University Toronto, 160 College Street, Toronto,  M5S 3E1, 2Banting Best Department Medical Research, University Toronto, 112 College Street, <br /> Toronto,  M5G 1L6, 3Department Molecular Genetics, University Toronto, 1 King&#226;&#8364;&#8482;s College Circle, <br /> Toronto,  M5S 1A8, 4Samuel Lunenfeld Research Institute, Mount Sinai Hospital, 600 University Avenue, <br /> Toronto,  M5G 1X5, Canada, 5Department Experimental Medicine Biochemical Sciences, University Perugia, del Giochetto, 06122, Perugia, Italy 6Department Experimental Medicine Biochemical <br /> Sciences, University Perugia, Perugia, Italy <br />  <br /> Received July 22, 2011; Revised September 16, 2011; Accepted September 19, 2011 <br />  <br />  <br />  <br /> ABSTRACT                                                                        enrichment analysis gene sets showing similar fifth genes budding yeast                               morphological alterations. current PhenoM essential haploid viability                               version 1.0 contains 78 194 morphological images <br /> functionally assessed using standard genetic                                    1 909 914 cells covering subcellular compart- <br /> approaches gene deletion. facilitate                                 ments structures 775 ts alleles spanning 491 <br /> genetic analysis essential genes,                              essential genes. PhenoM freely available http:// assembled collections yeast strains ex-                                 phenom.ccbr.utoronto.ca/. <br /> pressing temperature-sensitive (ts) alleles essen- <br /> tial genes. explore phenotypes caused                                 INTRODUCTION <br /> essential gene mutation used panel genet-                               Essential genes indispensable survival ically engineered fluorescent markers explore                            organism typically involved fundamental bio- <br /> morphology cells ts strain collection using                           logical processes, cell wall membrane biogen- <br /> high-throughput microscopy.                                esis, ribosome biosynthesis, DNA replication design implementation online database,                                cytoskeletal functions (1). thoroughly <br /> PhenoM (Phenomics yeast Mutants), storing,                               characterized model organisms, budding yeast <br /> retrieving, visualizing data mining quantita-                           Saccharomyces cerevisiae frequently used study <br /> tive single-cell measurements extracted                                    genes involved conserved biological pathways. micrographs ts mutant cells. PhenoM allows                               S. cerevisiae,  18% genes considered es- <br />                                                                                 sential growth rich medium glucose users rapidly search retrieve raw images                                                                                 carbon source (1,2). Essential genes evolutionarily quantified morphological data genes                                conserved yeast humans  database provides                                    non-essential genes, indicating deeper understanding <br /> data-mining tools, including PhenoBlast module                                functions yeast essential genes extend phenotypic comparison mutant strains                                understanding human gene function (3). Previous Gene Ontology module functional                                       studies characterized functions yeast essential <br />  <br />  <br />  <br />  <br />  correspondence addressed. Tel: +1 416 978 8562; Fax: +1 416 978 8287; Email: brenda.andrews@utoronto.ca <br /> Correspondence addressed Charles Boone. Tel: + 416 946 7260; Fax: +1 416 978 8287; Email: charlie.boone@utoronto.ca <br /> Correspondence addressed Zhaolei Zhang. Tel: +416 946 0924; Fax: +1 416 978 8287; Email: zhaolei.zhang@utoronto.ca <br />  <br /> &#195;&#376; Author(s) 2011. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  D688 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br /> genes using variety alleles allow conditional                   facilitate access valuable single-cell morpho- <br /> partial inactivation essential gene function.                      logical data including raw images, implemented mutant strains constructed genetic analysis                  web server integrated database capability essential genes past years, including              allow visualization analysis morphological data. <br /> harbouring tetracycline (tet)-repressible promoter alleles                 PhenoM (Phenomics Yeast Mutants) web-based <br /> (1), decreased abundance mRNA perturbation alleles                      platform contains quantitative single-cell measure- <br /> (4) temperature-sensitive (ts) alleles (5,6).               ments morphological images yeast cells carrying <br /> methods, ts alleles considered powerful                   ts alleles essential genes generated HCS technology <br /> genetic analysis perturb speci&#239;&#172;?c aspects es-                (10&#226;&#8364;&#8220;13) [reviewed (9)]. Figure 1 shows detailed record <br /> sential gene function (5&#226;&#8364;&#8220;8).                                               allele smc4-1, describes basic information    Temperature-sensitive alleles essential genes allow                  gene phenotypic images. developed cell viability permissive temperature                     user-friendly fast accurate search <br /> protein encoded ts allele resembles encoded                    engine embedded data mining tools, PhenoDev normal wild-type allele mutant                           PhenoBlast. PhenoDev set tools calculating <br /> cells morphologically indistinguishable                     phenotypic differences mutant strains wild-type cells.  restrictive (non-                     mutant wild-type strain grown permissive) temperature, ts gene product defective                  different temperatures. tools used result loss function. Consequently                     identify genes  compromised, rise cells growth morphological defects                  speci&#239;&#172;?c abnormal phenotype  conversely, strictive temperature. recently developed                        speci&#239;&#172;?c ts allele, tools quickly identify <br /> method quantify cell morphological changes thou-                    signi&#239;&#172; phenotypic changes associated <br /> sands mutants parallel coupling synthetic genetic                 temperature shift. contrast, tool, <br /> analysis high-content screening (SGA&#226;&#8364;&#8220;HCS) (9,10).                     PhenoBlast, serves quantify morphological similar- used SGA&#226;&#8364;&#8220;HCS study essential yeast genes                         ity multiple ts alleles. tool best used recently published phenotypic pro&#239;&#172;?ling 497 yeast                      discover ts alleles cause abnormal phenotypes <br /> strains carrying ts conditional alleles essential genes (6).            similar ts allele  provides <br />  <br />  <br />  <br />  <br /> Figure 1. example allele interface based allele smc4-1. interface includes basic information allele phenotypic <br /> images allele different subcellular compartments structures. <br />                                                                    Nucleic Acids Research, 2012, Vol. 40, Database issue   D689 <br />  <br />  <br /> tools Gene Ontology   enrichment analysis &#239;&#172;?nd            website (18) customized according gene <br /> common annotation(s) ranked                       set database. <br /> mutants identi&#239;&#172;?ed PhenoDev PhenoBlast. <br />    Currently, PhenoM version 1.0 holds 78 194 morpho-              DATA MINING <br /> logical images 1 909 914 cells, covering yeast <br /> subcellular compartments structures, including actin &#239;&#172;?la-       current PhenoM database contains data 775 ts <br /> ments, DNA damage foci, mitochondria, nucleus, plasma              mutants covering 491 yeast essential genes. membrane mitotic spindle. data assembled            mutant strain, 865 morphological parameters studying strains carrying 775 different ts alleles spanning     catalogued database, corresponding different <br /> 491 essential genes. knowledge, PhenoM &#239;&#172;?rst          temperatures, cellular compartments cell cycle <br /> database phenotypes caused mutation yeast essen-          stages. example, following parameters describing <br /> tial genes contains analyses quantitative data            morphology spindle quanti&#239;&#172;?ed stored <br /> derived single-cell morphological images. PhenoM              database: area, perimeter, length orientation. valuable resource yeast genomics           aid biologists extracting biologically useful informa- <br /> communities large volume data ef&#239;&#172;?cient        tion, designed implemented data mining tools, <br /> analysis tools facilitate study essential genes        PhenoDev PhenoBlast, summarize function. Images available downloadable              phenotypic deviations genes isogenic cell popu- <br /> format facilitate future analysis using                 lations quantify morphological similarities sophisticated software developed compu-          different mutants, respectively. <br /> tational experts.                                                  Phenotypic deviation: PhenoDev <br />                                                                    PhenoDev collection analytical tools DATA SOURCE                                                        help users evaluate phenotypic deviations                                                                    given morphological parameter response condition PhenoM database contains morphological images              changes. consists modules: PhenoTempDev, <br /> quantitative single-cell measurements yeast strains             PhenoMutaDev PhenoCycleDev. provide carrying 775 different ts alleles associated 491 essen-       detailed user manual examples using tools <br /> tial genes, covering following subcellular compart-        PhenoM website; tools user-friendly inter- <br /> ments structures: actin &#239;&#172;?laments, DNA damage foci,              faces extensively tested optimized. <br /> mitochondria, nucleus, plasma membrane mitotic                    phenotypic temperature deviation (PhenoTemp <br /> spindle. described previous report, MATa                 Dev) module help users identify ts strains query strains carrying different &#239;&#172;&#8218;uorescent markers           signi&#239;&#172; changes speci&#239;&#172;?c morphological parameter <br /> mated ts allele array MATa haploid ts strains           grown permissive restrictive temperature <br /> expressing different Green Fluorescent Protein (GFP)            Figure 2 example). example, researcher Red Fluorescent Protein (RFP) fusion proteins              interested asking ts allele affects &#226;&#8364;&#732;spindle <br /> isolated using SGA technology (6,10,14). following             length&#226;&#8364;&#8482;, use PhenoTempDev extract cellular markers used: plasma membrane marker               sets measurements &#226;&#8364;&#732;spindle length&#226;&#8364;&#8482; (Psr1p-GFP), reporter DNA damage (Ddc2p-GFP),                 cell populations. statistical test allows ranking nuclear marker (Mad1p-Nuclear localization Signal-               ts alleles according signi&#239;&#172;?cance difference <br /> RFP), mitotic spindle reporter (GFP-Tub1), mitochon-           tested strains. essence, tool help <br /> drial marker (OM45p-GFP) actin reporter                     researchers identify genes implicated particu- <br /> (Sac6p-GFP). Morphological images generated                lar yeast morphological trait. Phenotypic mutation devi- <br /> using SGA-HCS protocol (9,15) quantitative                 ation (PhenoMutaDev) intended quantify measurements particular morphological feature            phenotypic deviation wild-type strain extracted single-cell level. reporter,    ts mutant strain selected set morphological par- <br /> independent images captured different tempera-             ameters temperature. contrast tures referred &#226;&#8364;&#732;sites&#226;&#8364;&#8482; database.        PhenoTempDev, detects morphological changes quantitative measurements cell                    caused temperature shift, PhenoMutaDev categorized factors temperature          identify mutants signi&#239;&#172;?cantly different pheno- yeast strain grown, cell cycle stage      type wild-type cells. <br /> cell subcellular compartment structure                Finally, phenotypic cell cycle deviation (PhenoCycleDev) feature extracted. yeast cells rela-      allows users examine phenotypic changes single <br /> tively simple ellipsoidal shape, dimensions cells       mutant cell cycle. described  serve proxy determine cell cycle phase (16).        categorized cell cycle phases based PhenoM, categorized cells phases            daughter bud mother cell ratios. queries cell cycle according ratio daughter cell area     contiguous cell cycle transitions:   unbudded mother cell area:   unbudded cell; (ii) small budded cell;       small budded cells (G1-S phase); (ii) small budded (iii) medium budded cell; (iv) large budded cell               medium budded cells (S-G2); (iii) medium <br /> (detailed information provided PhenoM website).              budded large budded cells (G2-M). cell cycle <br /> Gene annotations downloaded SGD (17),                    transition, measurements morphological parameters annotations downloaded                   extracted compared. program ranks  D690 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 2. example &#226;&#8364;&#732;PhenoTempDev  interface. &#239;&#172;?ve alleles presented &#239;&#172;?gure. <br />  <br />  <br />  <br />  <br /> mutants according changes morphological measure-                   colleagues developed software tool called PhenoBlast, ments associated transitions. tool,             analogy BLAST, gene pairs scored based essential mutant, users readily determine              similarities morphological phenotypes caused phenotypic features affected particular cell                knockdown gene expression RNAi. tool, <br /> cycle phase transition.                                                  47 phenotypic parameters used assess morpho- <br />                                                                          logical phenotypes Caenorhabditis elegans (19).                                                                          adopted PhenoBlast comparing yeast morphology <br /> PhenoBlast <br />                                                                          data. algorithm, directly compare rationale using morphological data study gene                 morphology different mutant strains. <br /> function stems recent observation genes                    Instead, mutant strain, morphological meas- annotated similar functions tend                     urements population mutant cells &#239;&#172;?rst rise similar morphological defects perturb-                 compared population wild-type cells. ation (6). huge volume morphology data generated                  used Wilcoxon rank-sum test determine high-content screens catalogued PhenoM                   particular morphological parameter (e.g. spindle length) <br /> database offers unique useful resource search               signi&#239;&#172;?cantly increased (labelled &#226;&#8364;&#732;+1&#226;&#8364;&#8482;), unchanged <br /> alleles similar morphological phenotypes                   (labelled &#226;&#8364;&#732;0&#226;&#8364;&#8482;) signi&#239;&#172;?cantly decreased (labelled similar cellular functions.                                    &#226;&#8364;&#732; 1&#226;&#8364;&#8482;) mutant strain. way, obtain    contrast searching databases genes                vector 865 values consisting &#226;&#8364;&#732;+1&#226;&#8364;&#8482;, &#226;&#8364;&#732;0&#226;&#8364;&#8482; &#226;&#8364;&#732; 1&#226;&#8364;&#8482; sequence similarities, searching morphological                       mutant strain, describes overall morpho- <br /> similarities pair yeast strains computa-               logical change comparison wild-type cells. tionally algorithmically challenging.                     quantify morphological similarities earlier study phenotypic changes Caenorhabditis                    mutant strains calculating similarities elegans RNAi interference experiments, Gunsalus                   corresponding vectors  detailed information, <br />                                                                  Nucleic Acids Research, 2012, Vol. 40, Database issue   D691 <br />  <br />  PhenoM server). algorithm described           mutants number objects single cell implemented optimized                      particular compartment; (iv) searching mutants PhenoBlast. Users query single mutant          certain range particular cell shape parameter. mutants similar phenotypic pro&#239;&#172;?les         options uni&#239;&#172;?ed Morphology Search. <br /> real time. Figure 3 shows example PhenoBlast <br /> comparison used default settings query              enrichment analysis <br /> smc4-1 strain. search identi&#239;&#172;?ed mcd1-73 strain,          Depending users&#226;&#8364;&#8482; objective, running PhenoDev contains ts allele MCD1 gene           PhenoBlast automatically generates list mutants <br /> hit smc4-1 mcd1-73 strains shared similarity          showing phenotypic abnormality. provide 621 morphological parameters [122 parameters signi&#239;&#172;?-             module enrichment analysis test cantly increased, 435 unchanged 64 signi&#239;&#172;?cantly              genes associated phenotypic defects decreased (Figure 4)].                                           enriched known functions pathways. function- <br />                                                                  ality particularly useful gauging biological sig- <br /> Search                                                           ni&#239;&#172;?cance highly ranked genes mutation <br />                                                                  causes extreme phenotypic defects. implement search engine provides options,            analysis, compared tools GoMiner <br /> Quick Search Morphology Search. Quick Search,             (20), GOEAST (21) :TermFinder (22). users directly query mutant         chose use :TermFinder package (v0.86) database specifying gene  open reading frame         PhenoM uses modular design (ORF)  aliases. partial-word-match method           easy integrate  incorporated Quick Search. hand, users want &#239;&#172;?nd mutant strains share morpho- <br /> logical similarity pre-speci&#239;&#172;?ed gene, search         DATABASE IMPLEMENTATION USER <br /> morphology pro&#239;&#172;?les performed. Morphology              INTERFACE <br /> Search utility &#239;&#172;&#8218;exible allows users conduct <br />                                                                  Database implementation <br /> speci&#239;&#172;?c advanced searches including:   searching mutants certain cell cycle phase; (ii) searching        designed implemented PhenoM using Struts- <br /> mutants grown certain temperature; (iii) searching          Spring-Hibernate,   lightweight enterprise-level <br />  <br />  <br />  <br />  <br /> Figure 3. example &#226;&#8364;&#732;Search&#226;&#8364;&#8482; interface PhenoBlast. <br />  D692 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  <br />  <br /> Figure 4. example &#226;&#8364;&#732;PhenoBlast  interface. interface includes analysis result section &#226;&#8364;&#732;Search Result&#226;&#8364;&#8482; section. Smc4-1 used query allele 5 23&#226;&#8364;&#8220;25 alleles presented &#239;&#172;?gure. <br />  <br />  <br />  <br />  <br /> developmental framework (23,24). MySQL (v5.0.77)                        charts. source code development performed used underlying relational database store                     Eclipse v3.4.2 (http://www.eclipse.org). <br /> large volume quantitative measurements links raw morphological images &#239;&#172;?le                    User interface end, Apache Server 2.0 deployed handle user <br />                                                                             PhenoM server provides following primary web <br /> requests forward requests Java servlets <br />                                                                             interfaces, allow users perform focused query Java Server Pages Apache Tomcat 6.0.                                                                             analysis tasks. <br /> provide user-friendly interface, JavaScript AJAX <br />                                                                                &#226;&#8364;&#732;Allele interface&#226;&#8364;&#8482; presents detailed informa- <br /> (Asynchronous Javascript XML) technology                                                                             tion mutant strain. consists parts: <br /> adopted program client functionality Apache TilesTM used create reusable interface                            Basic information cross links databases <br /> components. online graphical visualization                                SGD (17) BioGRID (25); <br /> image manipulation, ImageJ (v1.44) incorporated                       (ii) Phenotypic images statistical information assist users morphological image processing Java                         subcellular compartments struc- <br /> applets, Java open source package JFreeChart                              tures. functional links provided, <br /> (http://www.jfree.org/jfreechart/) used plot bar                           option download original <br />                                                                 Nucleic Acids Research, 2012, Vol. 40, Database issue          D693 <br />  <br />  <br />       micrograph TIFF format, viewing image               (P = 0.04455) (Figure 4). results PhenoBlast <br />       ImageJ, link dataset        consistent known biology (6,29&#226;&#8364;&#8220;31),       particular image. ease analysis, images      believe useful tool suggesting <br />       annotated names correspond             new biological gene associations yeast research <br />       mutant  reporter imaged, temperature         community. <br />       images captured site               Download: PhenoM allows users download       number  indicates set independent            data set mutants single <br />       images used produce particular image           selected mutant tab-delimited format. <br />       mutant) (iii) PhenoBlast results list, containing &#239;&#172;?fty <br />       mutants share similar morphological            AVAILABILITY FUTURE DIRECTION <br />       pro&#239;&#172;?les (Figures 1 4).                                PhenoM freely accessed http://phenom.ccbr. <br />                                                                 utoronto.ca/. future development PhenoM                                                                 include following aspects:   PhenoM include <br />    PhenoDev: series tools measure pheno-        phenotypic information additional yeast strains <br /> typic deviations strains carrying different ts alleles       carrying mutant alleles non-essential essential varying conditions. PhenoTempDev               genes data available groups example navigate interfaces function           (ii) data subcellular compartments (Figure 2). Multiple options, morphological             structures incorporated database help <br /> features certain compartment, bud size                 biologists uncover association cell morph- provided customized. ana-              ology gene function; (iii) data mining lysis default parameters lead              statistical analysis tools developed, especially tools <br /> PhenoTempDev results interface (Figure 2),                integrate types data protein <br /> includes sections:   results analysis       complexes protein interactions. <br /> (ii) deviation details 50 mutants listed         database ultimately aims provide centralized <br /> descending order.                                               platform analysis gene function using quan- <br />    PhenoBlast: comparison tool phenotyp-          titative measurements morphological images ic similarity accessed home          yeast mammalian cells. accomplish goal, <br /> interface PhenoM navigation bar.              PhenoM provide submission collect data <br /> execute PhenoBlast, query mutant                 image-based HCS experiments future. <br /> relevant morphological features speci&#239;&#172;?ed. typing characters query allele box, automatically suggest list similar mutant         ACKNOWLEDGEMENTS <br /> names. facilitate customized search, list 865       like thank Xiaoli Wu helping design morphological features divided parts:   386     style interfaces Dr Yufang Zheng Fudan <br /> features phenotypic mutation deviation; (ii) 193           University critical readings early version features phenotypic temperature deviation;             article. wish thank Dr Judice L.Y. Koh (iii) 286 features phenotypic cell cycle deviation         helpful suggestions. <br /> (detailed information provided PhenoM <br /> server). feature chosen conduct analysis. PhenoBlast results interface dis-     FUNDING <br /> played following execution successful query                                                                 Fudan University - University Toronto Exchange <br /> contains parts:   analysis results (ii)                                                                 Scholarship  K.J.); postdoctoral fellowship <br /> details 50 mutants similar phenotypes. <br />                                                                 Best Foundation  F.J.V.); Canadian <br /> Figures 3 4 shows example navigating                                                                 Institutes Health Research grants (GMX-201237 PhenoBlast (Figures 3 4). SMC4 gene encodes                                                                 B  C.B.) (GMX-211012 B , C.B. member ubiquitous family chromosome-associated <br />                                                                 Z.). Infrastructure automated imaging ATPases (26). Smc4 protein subunit                                                                 Boone Andrews Lab funded condensin complex required chromosome <br />                                                                 Canadian Foundation Innovation (LEF-21475) condensation dynamics (27,28). search                                                                 Ontario Research Fund. Funding open access <br /> mutants sharing similar morphological pro&#239;&#172;?les                                                                 charge: Canadian Institutes Health Research <br /> smc4-1 mutant strain, input smc4-1 query <br />                                                                 (GMX-201237) (GMX-211012). <br /> allele box Search interface PhenoBlast (Figure 3) press button &#226;&#8364;&#732;Run PhenoBlast&#226;&#8364;&#8482; using default        Con&#239;&#172;&#8218;ict statement. declared. <br /> parameters. automatically leads interface PhenoBlast. search result section, &#239;&#172;?rst, &#239;&#172;?fth fourth ranked mutants, mcd1-73,           REFERENCES <br /> smc1-2 smc3-1, respectively, carry mutant alleles <br />                                                                  1. Mnaimneh,S., Davierwala P., Haynes,J., Moffat,J., Peng,W.T., genes encoding condensin subunits list                Zhang,W., Yang,X., Pootoolal,J., Chua,G., Lopez  et al. (2004) <br /> mutants overall enriched DNA metabolic                   Exploration essential gene functions titratable promoter <br /> process (P = 0.01218) sister chromatid cohesion                 alleles. Cell, 118, 31&#226;&#8364;&#8220;44. <br />  D694 Nucleic Acids Research, 2012, Vol. 40, Database issue <br />  <br />  <br />  2. Giaever,G., Chu M., Ni,L., Connelly,C., Riles,L., Veronneau,S.,       17. Hong,E.L., Balakrishnan,R., Dong,Q., Christie,K.R., Park,J., <br />     Dow,S., Lucau-Danila , Anderson,K., Andre,B. et al. (2002)                Binkley,G., Costanzo,M.C., Dwight,S.S., Engel,S.R., Fisk,D.G. <br />     Functional pro&#239;&#172;?ling Saccharomyces cerevisiae genome.                 et al. (2008) Gene Ontology annotations SGD: new data <br />     Nature, 418, 387&#226;&#8364;&#8220;391.                                                       sources annotation methods. Nucleic Acids Res., 36, <br />  3. Hughes,T.R. (2002) Yeast drug discovery. Funct. Integr.                 D577&#226;&#8364;&#8220;D581. <br />     Genomics, 2, 199&#226;&#8364;&#8220;211.                                                   18. Ashburner,M., Ball,C , Blake,J , Botstein,D., Butler,H., <br />  4. Schuldiner,M., Collins,S.R., Thompson,N.J., Denic,V.,                       Cherry,J.M., Davis P., Dolinski,K., Dwight,S.S., Eppig,J.T. <br />     Bhamidipati , Punna,T., Ihmels,J., Andrews,B., Boone,C.,                  et al.; Gene Ontology Consortium (2000) Gene ontology: <br />     Greenblatt,J.F. et al. (2005) Exploration function               tool uni&#239;&#172;?cation biology. Nat. Genet., 25, 25&#226;&#8364;&#8220;29. <br />     organization yeast early secretory pathway            19. Gunsalus,K.C., Yueh,W.C., MacMenamin,P. Piano,F. (2004) <br />     epistatic miniarray pro&#239;&#172;?le. Cell, 123, 507&#226;&#8364;&#8220;519.                             RNAiDB PhenoBlast: web tools genome-wide phenotypic <br />  5. Ben-Aroya,S., Coombes,C., Kwok,T., O&#226;&#8364;&#8482;Donnell,K , Boeke,J.D.               mapping projects. Nucleic Acids Res., 32, D406&#226;&#8364;&#8220;D410. <br />     Hieter,P. (2008) comprehensive                             20. Zeeberg,B.R., Feng,W., Wang,G., Wang,M.D., Fojo T., <br />     temperature-sensitive mutant repository essential genes           Sunshine,M., Narasimhan,S., Kane,D.W., Reinhold,W.C., <br />     Saccharomyces cerevisiae. Mol. Cell, 30, 248&#226;&#8364;&#8220;258.                           Lababidi,S. et al. (2003) GoMiner: resource biological <br />  6. Li , Vizeacoumar,F.J., Bahr,S., Li,J., Warringer,J.,                      interpretation genomic proteomic data. Genome Biol., 4, <br />     Vizeacoumar,F.S., Min,R., Vandersluis,B., Bellay,J., Devit,M.               R28. <br />     et al. (2011) Systematic exploration essential yeast gene            21. Zheng,Q. Wang,X.J. (2008) GOEAST: web-based software <br />     function temperature-sensitive mutants. Nat. Biotechnol., 29,          toolkit Gene Ontology enrichment analysis. Nucleic Acids <br />     361&#226;&#8364;&#8220;367.                                                                    Res., 36, W358&#226;&#8364;&#8220;W363. <br />  7. Hartwell,L.H., Culotti,J. Reid,B. (1970) Genetic control         22. Boyle,E , Weng,S., Gollub,J., Jin,H., Botstein,D., Cherry,J.M. <br />     cell-division cycle yeast.  Detection mutants. Proc. Natl       Sherlock,G. (2004) :TermFinder&#226;&#8364;&#8220;open source software                                                                                 accessing Gene Ontology information &#239;&#172;?nding signi&#239;&#172;?cantly <br />     Acad. Sci. USA, 66, 352&#226;&#8364;&#8220;359. <br />                                                                                 enriched Gene Ontology terms associated list genes. <br />  8. Simchen,G. (1978) Cell cycle mutants. Ann. Rev. Genet., 12, <br />                                                                                 Bioinformatics, 20, 3710&#226;&#8364;&#8220;3715. <br />     161&#226;&#8364;&#8220;191. <br />                                                                             23. Sun,L.V., Jin,K., Liu,Y., Yang,W., Xie,X., Ye,L., Wang,L., <br />  9. Vizeacoumar,F.J., Chong,Y., Boone,C. Andrews,B.J. (2009) <br />                                                                                 Zhu,L., Ding,S., Su,Y. et al. (2008) PBmice: integrated <br />     picture worth thousand words: genomics phenomics                                                                                 database piggyBac (PB) insertional mutations     yeast Saccharomyces cerevisiae. FEBS Lett., 583, 1656&#226;&#8364;&#8220;1661.             characterizations mice. Nucleic Acids Res., 36, D729&#226;&#8364;&#8220;D734. <br /> 10. Vizeacoumar,F.J., van Dyk,N.S., Vizeacoumar,F., Cheung,V.,              24. Yang,W., Jin,K., Xie,X., Li,D., Yang,J., Wang,L., Gu,N., <br />     Li,J., Sydorskyy,Y., Case,N., Li , Datti , Nislow,C. et al.             Zhong,Y. Sun,L.V. (2009) Development database     (2010) Integrating high-throughput genetic interaction mapping              mapping insertional mutations mouse genome     high-content screening explore yeast spindle                         large-scale experimental data. BMC Genomics, 10(Suppl. 3), S7. <br />     morphogenesis. J. Cell. Biol., 188, 69&#226;&#8364;&#8220;81.                              25. Stark,C., Breitkreutz,B.J., Chatr-Aryamontri , Boucher,L., <br /> 11. Bakal,C., Aach,J., Church,G. Perrimon,N. (2007)                         Oughtred,R., Livstone,M.S., Nixon,J., Van Auken,K., Wang,X., <br />     Quantitative morphological signatures de&#239;&#172;?ne local signaling                 Shi,X. et al. <br /> </body></html>