<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Addition Escherichia coli K-12 Growth Observation Gene <br /> Essentiality Data EcoCyc Database <br /> Amanda Mackie Suzanne Paley,b Ingrid M. Keseler,b Alexander Shearer,b Ian T. Paulsen Peter D. Karpb <br /> Department Chemistry Biomolecular Science, Macquarie University, Sydney, Australiaa; Bioinformatics Research Group, SRI International, Menlo Park, California, <br /> USAb <br />  sets compounds support growth organism defined presence transporters metabolic path- <br /> ways convert nutrient sources cellular components energy growth. collection known nutrient sources serve impetus investigating new metabolic pathways transporters reference computa- <br /> tional modeling known metabolic pathways. establish collection Escherichia coli K-12, integrated data growth nongrowth E. coli K-12 obtained published observations using variety individual media high-throughput phenotype microarrays EcoCyc database. assembled collection revealed substantial number discrepancies high-throughput data sets, investigated possible using low-throughput growth assays soft agar liquid culture. integrated data sets describing 16,119 observations growth single-gene <br /> knockout mutants E. coli K-12 EcoCyc, relevant antimicrobial drug design, provide clues regarding roles genes unknown function, useful validating metabolic models. make information easily accessible EcoCyc users, developed software capturing, querying, visualizing cellular growth assays gene essentiality data. <br />  <br />  <br />  <br /> T    diversity <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> environments support organ- <br />      ismal growth fundamental corpus scientific knowledge.  despite decades research large amounts accu- <br />                                                                                       throughput phenotype microarray (PM) data (2); (iv) PM <br />                                                                                       data generated group. PMs measure cellular respiration                                                                                       proxy growth (henceforth, refer growth simplicity) <br /> mulated knowledge, compendium set nutrients Esch-                        sets 96 plates, containing erichia coli K-12 able utilize exists. compendium                 different combination nutrients. integrated basic serve reference com-                   EcoCyc data sets growth single-gene <br /> putational metabolic modeling, requires comprehensive                         knockout mutants E. coli K-12 published set reference growth data sets evaluating accuracy                      recent years. <br /> growth predictions.                                                                       developed software tools capturing, querying, vi- <br />     validating computational metabolic models                    sualizing cellular growth assays, corresponding nutrients experimental data describing essentiality organism&#226;&#8364;&#8482;s                    growth conditions, gene essentiality data. tools genes, metabolic models predict phenotypes                         integrated Pathway Tools software (3),  <br /> gene knockouts. essential gene gene indispensable                  abling use conjunction 2,000 <br /> support life specific set growth conditions.                       genomes contained BioCyc database collection (BioCyc <br /> gene essentiality data useful predicting antibiotic targets               .org) organism databases created Pathway Tools <br /> pathogenic bacteria, guiding design minimal genomes,                       users. providing clues regarding roles genes unknown <br /> function.                                                                             MATERIALS METHODS <br />     large number high-throughput data sets gene ex-                         Bacterial strains. E. coli K-12 MG1655 obtained Yale <br /> pression metabolomics phenotypic measurements                            Coli Genetic Stock Center (CGSC; strain 7740) American <br /> coming available variety organisms. maximize                        Type Culture Collection (ATCC; strain 700926). <br />                                                                                           Laboratory evaluation individual growth media. evaluate <br /> value, data sets captured integrated                                                                                       growth soft agar, following protocol used based modifica- <br /> database published website, tools querying                    tion earlier Salmonella nutrient evaluation method (4). Bacterial analysis ensure data maximally available                   cells grown overnight LB medium 37&#194;&#176;C. overnight culture <br /> scientific community.  integration requires                         washed times phosphate-buffered saline (PBS), resus- <br /> simply collecting multiple data sets common repos-                      pended PBS, diluted 10:1 cooling liquefied 0.6% agar solution <br /> itory. data types, integration include identifying resolving conflicts data sets possible order extract knowledge possible noisy data.                          Received 10 October 2013 Accepted 15 December 2013 <br />     integrated following growth data single collec-                        Published ahead print 20 December 2013 <br /> tion available EcoCyc database (1) supplemental                        Address correspondence Peter D. Karp, pkarp@ai.sri.com. <br /> files:   collection observations growth nongrowth                    Supplemental material article http://dx.doi.org/10.1128 <br /> laboratory strains E. coli K-12 variety media                        /JB.01209-13. obtained low-throughput methods                                Copyright &#194;&#169; 2014, American Society Microbiology. Rights Reserved. <br /> previously published literature; (ii) low-throughput growth                       doi:10.1128/JB.01209-13 <br /> data generated group; (iii) previously published high- <br />  <br />  <br /> 982   jb.asm.org                                       Journal Bacteriology   p. 982&#226;&#8364;&#8220;988                                          March 2014 Volume 196 Number 5 <br />                                                                                                             Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  plated appropriate base plate. Fully cooled        analyzed using Omnilog PM software response calls growth <br /> plates used subsequent nutrient testing. Base plates        (G), growth (NG), low growth (LG) based param- minimal salts medium (1.9 mM potassium sulfate, 25.8 mM              eters maximum height  10th percentile highest value dipotassium phosphate, 11.8 mM monopotassium phosphate, 0.13 mM                values time points) area  sum Omnilog values magnesium sulfate heptahydrate) supplemented 1.5% agar ei-            time points). calls 13 nitrogen source wells, 10 <br /> ther 0.5% succinate  nitrogen source testing) 2.5 mM NH4Cl          sulfur source wells, phosphate sources, carbon source testing). nutrient tested plates (containing     showed ambiguous responses compared negative-control  succinate NH4Cl) placing 2 20 mg nutrient        Data set S6 supplemental material includes raw data PM <br /> center plate. plates incubated 37&#194;&#176;C, growth        experiments conducted course work. &#226;&#8364;&#339;Mackie12&#226;&#8364;? refers <br /> evaluated visually 24, 48, 72 h. plate scored positive     experiments conducted 2012, data currently avail- <br /> growth bacterial lawn &#226;&#177;&#8211;1 cm2 present           able EcoCyc website. results later experiments (dated 2013 <br /> scoring checks. growth conditions tested duplicate.                File S6 supplemental material) conducted compare Biolog PM <br />     evaluate growth liquid culture, strains grown overnight      responses different stock cultures E. coli K-12 MG1655  <br /> M9 minimal media 0.2% (vol/vol) glycerol. overnight culture           cluded future EcoCyc release. washed times modified M9 minimal media containing                 PM data sets added EcoCyc, reviewed carbon nitrogen [48 mM sodium phosphate dibasic, 22 mM monopo-              growth observations difference observations tassium phosphate, 8.5 mM sodium chloride, 2 mM magnesium sulfate,             PM data sets existed, resolved conflict significant <br /> 0.1 mM calcium chloride, 0.01 mM Fe(II) sulfate] transfer new      evidence regarding actual growth outcome  factors <br /> M9 minimal medium starting optical density 600 nm 0.05.           considered included number available observations growth ver- <br /> Carbon sources tested M9 minimal medium final concentra-          sus growth, low-throughput observations tion 0.2% (wt/vol), nitrogen sources tested modified M9       literature given growth condition  cases , minimal medium final concentration 0.2% (wt/vol). carbon           minority data set(s) generally judged reliable <br /> source assays 0.2% sodium succinate. Cultures         based total pairwise conflicts data sets. incubated 37&#194;&#176;C 48 h, growth rates reported means &#226;&#171;&#190;       did resolve conflict, <span id='am-122' about='rdfs:comment' typeof='owl:Thing'>comment</span> usually created EcoCyc standard deviations replicates. appropriate positive neg-    summarize rationale, literature citations recorded appro- <br /> ative controls included experiments. nitrogen source        priate. example, aerobic cell PM1:F2, tests growth says, positive control ammonium chloride; carbon source            <span id='am-10' about='obo:IMR_0200473' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="rdfs:label" content="Citric acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00158" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200473" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>citric acid</span> sole carbon source, resolved favor growth <br /> assays, positive control sodium succinate.                             (5), aerobic cell PM1:G7, tests growth acetoacetic <br />     Biolog PMs. Biolog phenotype microarrays (PMs) commercially          acid sole carbon source, resolved favor growth (6) (mouse <br /> available (Biolog, , Hayward, CA) high-throughput           cells online EcoCyc version Fig. 1 http://ecocyc.org <br />                                                                                /ECOLI/NEW-IMAGE?object&#226;&#171;&#189;Growth-Media explanation global analysis microbial respiration phenotypes (2). PMs use color- <br />                                                                                resolution). <br /> imetric reaction (reduction tetrazolium dye <span id='am-26' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADH</span>) sensitive <br /> indicator microbial respiration response presence individ-     RESULTS <br /> ual nutrients chemicals. PM plates 1 4, contained 190 sole <br /> carbon sources, 95 sole nitrogen sources, 59 sole phosphate sources,       Table 1 summarizes sources nutrient utilization data pres- <br /> 35 sole sulfur sources, used analysis. E. coli K-12 MG1655    ent EcoCyc version 17.5 released October/Novem- <br /> pregrown nutrient agar used inoculate plates according      ber 2013. results low-throughput growth experiments Biolog instructions. data collected analyzed using         reported appear future EcoCyc release. <br /> OmniLogH PM  records color change 15 min                Table 1 includes total 1,860 growth observations 552 96 assay plates. incubations performed       growth conditions. 552 growth conditions include 456 aero- <br /> 37&#194;&#176;C 48 h.                                                                bic conditions 96 anaerobic conditions include 384 PM <br />     assess effect pregrowth conditions carbon-source utiliza-     media (including 5 negative controls) 72 media. tion, bacterial strain pregrown types nutrient agar        example, growth assessed set nutrients, plates&#226;&#8364;&#8221;LB agar, R2A agar, BUG-S (Biolog universal growth agar         aerobic anaerobic conditions, count situation 5% sheep blood)&#226;&#8364;&#8221; inoculation PM plates 1 2.                                                                                growth conditions. <br /> nitrogen, phosphate, sulfur utilization plates (PM3 PM4), bacte- <br /> ria pregrown LB agar  instance, inoculated accord- <br />                                                                                    growth data integrated literature  <br /> ing manufacturer&#226;&#8364;&#8482;s instructions. user-supplied carbon source        cluded following commonly used media: AB, LB medium  assays 20 mM sodium succinate 2 &#226;?&#174;M ferric citrate.            riched, LB Lennox, 3-(N-morpholino)propanesulfonic acid conditions, positive response recorded negative-      (MOPS), M9, M56, M63. media listed control nitrogen assays sulfur assays.    table EcoCyc.org Growth Media site (http: bacterial pregrowth assays nutrient-limited      //biocyc.org/ECOLI/NEW-IMAGE?object&#226;&#171;&#189;Growth-Media). <br /> R2A agar  discussion manufacturer, concentration            Integration PM data sets. integrated aerobic cell inoculum decreased factor 10. use R2A agar      phenotype microarray (PM) data sets produced bacterial pregrowth preferred method minimizing          independently different laboratories refer sponse seen negative-control nitrogen sulfur assays    Bochner12 (B. Bochner, unpublished data), AbuOun09 (7), reported previously (2).                                                  Baumler11 (8), Yoon12 (9), Mackie12. data sets  <br />     Data integration arbitration. Respiration data obtained                                                                                sisted measurements PM plates designated PM1 (95 <br /> PMs used proxy growth. Growth observations  <br /> corded Pathway Tools growth, growth, low (borderline) <br />                                                                                carbon sources), PM2 (95 carbon sources), PM3 (95 nitrogen <br /> growth. attempt unify criteria used distinguish          sources), PM4 (59 phosphorus plus 35 sulfur sources), states imported PM data original            Baumler11, covers PM1  <br /> growth/respiration curve data available data             data set, compared result sets;  used determinations specified providers      result data sets turn. data set. PM experiments (Mackie12), kinetic data        Table 2 summarizes degree pairwise agreement dis- <br />  <br />  <br /> March 2014 Volume 196 Number 5                                                                                                               jb.asm.org 983 <br />  Mackie et al. <br />  <br />  <br />  <br />  <br /> FIG 1 Table showing current display integrated growth observations EcoCyc PM1 carbon sources aerobic conditions. Colors indicate <br /> consensus conflicts  color legend . colored bar cell depicts individual observations, e.g., cell H5 (D-psicose) shows observations growth, observations growth, observation low growth. Mousing bar BioCyc.org version diagram <br /> provides literature reference growth observations. <br />  <br />  <br />  <br /> agreement data sets. percent pairwise agreement                    sets shows significantly pairwise conflicts number matches expressed percentage total                    data sets does data set.  15 16 <br /> number comparisons  example, 86% pairwise com-                       cases data sets recorded growth observa- <br /> parisons AbuOun09 data sets carbon                   tion (meaning Y [growth] N  growth], ignoring low growth) <br /> source PM plates agreement). Data set S1 supple-                    data set recorded opposite observation, outlier <br /> mental material lists pairwise comparisons individ-                  Baumler11. remove Baumler11 data set com- <br /> ual data sets. Table 2 combines pairwise comparison results                    puting  combined&#226;&#8364;? agreement score Table 2, percent single number reflects percentage agreement                     agreement carbon sources increases 82 87%; similarly, data set data sets. pairwise                Y/N conflict percentage carbon sources Table 2 drops <br /> agreement data sets individual growth observa-                     6 2%. <br /> tions ranged 72 87%. conflicts exist, distinguish                      Note PM3 (nitrogen source) plate shows significantly cases data set records growth                     disagreement plates. different data shows growth  &#226;&#8364;&#339;Y/N&#226;&#8364;? conflicts)                      sets similar degrees disagreement. systematic  <br /> cases data set records low borderline growth                   vestigation differences data sets consid- shows growth growth. 379 growth                     eration low-throughput E. coli growth data literature, <br /> conditions surveyed PM data sets, conflict existed                     resolve 73 differences (24 Y/N). result, <br /> tween pair data sets 119 wells, 57             consensus growth calls exist 333 379 conditions <br /> wells Y/N conflicts. <br />                                                                                    surveyed. Unresolved differences prevent growth calls  <br />     Baumler11 data set outlier PM data <br />                                                                                    maining 45  34 represent Y/N differences). Table 3 sum- <br />                                                                                    marizes number conflicts resolved                                                                                    nutrient types aerobic conditions, number TABLE 1 Numbers aerobic anaerobic growth observations <br /> described  grouped data source                                             conflicts remaining case. Table 4 lists compounds                                                                                    involved remaining conflicts. <br />                                                          growth <br />                                                         observations <br />                                                                                        Baumler11 Bochner12 include measurements                                                                                    anaerobic observations, PM1  data sets agreed Type data                                            Aerobic     Anaerobic      45% wells, Y/N conflict rate 49%. did Low-throughput data literature                 23          0              attempt resolve conflicts. <br /> Low-throughput data generated group              49          0                  Low-throughput growth experiments. integration High-throughput PM data literature             1,244       191            PM results EcoCyc, ability 33 compounds sup- <br /> High-throughput PM data generated group          353         0 <br />                                                                                    port growth E. coli K-12 sole carbon nitrogen source <br />  <br />  <br /> 984   jb.asm.org                                                                                                                            Journal Bacteriology <br />                                                                                                                                    Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  <br /> TABLE 2 Concordance aerobic datasetsa <br />                         % Agreement                                                                          % Y/N conflicts <br /> Data set                Carbon           Nitrogen           Phosphorus            Sulfur                 Carbon           Nitrogen           Phosphorus            Sulfur          AbuOun09                86               75                 89                    84             84          5                11                 5                     1               4 <br /> Baumler11               72               NA                 NA                    NA             72          21               NA                 NA                    NA              21 <br /> Bochner12               80               75                 91                    81             81          5                13                 3                     1               6 <br /> Mackie12                87               80                 93                    91             87          4                8                  4                     0               5 <br /> Yoon12                  86               75                 92                    75             84          4                3                  2                     1               3 <br />  combined            83               76                 91                    82             83          6                9                  3                     1               6   left half table shows percent pairwise agreement different datasets sources carbon, nitrogen, phosphorus, sulfur, respectively. right half table shows percent pairwise Y/N (growth growth) conflicts data. NA, applicable. <br />  <br />  <br />  <br /> remained unresolved (Table 4). tested 19 compounds                                     Baba06 (11) knockouts using lambda Red minimal salts soft agar M9 minimal medium                                    recombinase  replacing target genes frame kana- <br /> termine support aerobic growth sole ni-                                mycin resistance genes. later subset trogen source (Table 5), tested compounds deter-                                 mutants contained partial duplications support aerobic growth sole carbon                                cross-contaminated (12). combined data pub- <br /> source (Table 6). information included conflict                                lications EcoCyc. total 324 genes unable dis- <br /> resolutions future release EcoCyc.                                                     rupted predicted essential growth rich media <br />     Integration gene essentiality data. Table 7 lists gene                              37&#194;&#176;C. 3,844 strains containing knockouts genes essentiality data sets present version 17.5 EcoCyc.                             nonessential rich media (henceforth referred data sets include total 16,119 growth observations                              Keio collection) tested growth glucose <br /> knockout strains E. coli. Note numbers listed                                minimal medium (11). <br /> reflect number observations incorporated EcoCyc                                    Joyce et al. (13) profiled Keio collection growth necessarily numbers reported original studies. Dis-                             minimal medium glycerol sole source carbon crepancies numbers caused subsequent                                   energy. total 113 genes nonessential rich  <br /> corrections data  reflected numbers                              dium identified essential growth glycerol. Joyce et al. , gene merges, deletions,                                                            combined observations Baba et al. <br />     Gerdes03 (10) study used genetic footprinting technique                              regarding conditional essentiality mutants grown Tn5-based transposon reported assessment                                  glucose-supplemented minimal media able identify ca. 87% E. coli open reading frames essentiality.                              conserved conditionally essential core 94 genes required study identified 620 genes essential aerobic growth                             E. coli K-12 grow minimal nutritional supplementa- <br /> rich media, actual experimental result transposon                               tion essential growth rich conditions. <br /> insertions identified genes. did consider                                    Feist et al. (14) used experimental data regarding condi- sufficient evidence essentiality incorporated                           tional gene essentiality earlier studies (11, 13) compared EcoCyc observations 3,082 genes trans- <br /> poson insertions identified. transposon  <br /> sertion result null mutation, genes identified non-                            TABLE 4 Nutrients generate conflicting responses phenotype <br /> essential data set fact required growth.                                 microarrays <br />                                                                                                Source                     Nutrients <br />                                                                                                Carbon                     <span id='am-76' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-77' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span>L-Glutamic acid</span>, Tween 20, D-psicose, <br /> TABLE 3 Summary PM data integration conflict <br /> resolutiona                                                                                                                 glucuronamide, &#226;?&#163;-methyl-D-glucoside, pectin, <br />                                                                                                                             &#226;?&#163;-keto-valeric acid, L-alaninamide <br />                                wells                                                     Nitrogen                   <span id='am-31' about='obo:IMR_0200467' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-32' property="rdfs:label" content="L-Cysteine" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00097" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0200467" datatype="xsd:string"></span>L-Cysteine</span>, L-<span id='am-88' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-89' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span>glutamic acid</span>, <span id='am-5' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span>L-lysine</span>, <span id='am-148' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-149' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-151' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-152' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span>L-methionine</span>, <br />  <br /> Parameter status           Carbon          Nitrogen        Phosphorus           Sulfur                                   <span id='am-36' about='obo:IMR_0200274' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-37' property="rdfs:label" content="L-Phenylalanine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0200274" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00079" datatype="xsd:string"></span>L-phenylalanine</span>, <span id='am-56' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span>L-threonine</span>, D-asparagine, <br />                                                                                                                             D-lysine, D-valine, L-homoserine, <span id='am-112' about='obo:IMR_0200028' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-113' property="rdfs:label" content="Putrescine" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:id" content="IMR:0200028" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="KEGG:C00134" datatype="xsd:string"></span>putrescine</span>,  wells                  190             95              59                   35 <br />                                                                                                                             agmatine, D-mannosamine, N-acetyl- growth                     99              38              5                    4 <br />                                                                                                                             D-mannosamine, <span id='am-46' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>, <span id='am-51' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-52' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span>guanosine</span>, <span id='am-143' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-144' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-146' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>, <br /> Low growth                    2               0               0                    0 <br />                                                                                                                             <span id='am-128' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-129' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span>uridine</span>, <span id='am-123' about='obo:IMR_0200439' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-124' property="rdfs:label" content="Xanthine" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="KEGG:C00385" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:id" content="IMR:0200439" datatype="xsd:string"></span>xanthine</span>, <span id='am-66' about='obo:IMR_0200426' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:id" content="IMR:0200426" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Xanthosine" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C01762" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>xanthosine</span>, <span id='am-117' about='obo:IMR_0200029' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-118' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasDbXref" content="KEGG:C00366" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="Uric acid" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:id" content="IMR:0200029" datatype="xsd:string"></span>uric acid</span>, alloxan, <br /> Growth                        80              32              50                   23 <br />                                                                                                                             DL-&#226;?&#163;-amino-N-butyric acid &#226;?&#165;-amino-N-butyric <br /> Remaining conflicts           8 (9)b          25              4                    8 <br />                                                                                                                             acid, DL-&#226;?&#163;-amino-caprylic acid <br /> Resolved conflicts            52              10              6                    5 <br />                                                                                                Phosphorus                 <span id='am-41' about='obo:IMR_0200161' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Cytidine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0200161" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00475" datatype="xsd:string"></span>Cytidine</span> 3=,5=-cyclic mono-P, phosphono <span id='am-153' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-154' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span>acetic acid</span>,   Row 1 indicates total numbers wells nutrient type. Rows 2 4 indicate                                   2-aminoethyl phosphonic acid, methylene numbers wells nutrient type showing different growth levels                                       diphosphonic acid <br /> growth assessed. Row 5 indicates number wells growth state <br />                                                                                                Sulfur                     <span id='am-138' about='obo:IMR_0200417' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-139' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:hasDbXref" content="KEGG:C00506" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="L-Cysteic acid" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:id" content="IMR:0200417" datatype="xsd:string"></span>L-Cysteic acid</span>, cysteamine, S-methyl-L-cysteine, assessed conflicting observations arbitrated. Row 6 <br />                                                                                                                             DL-ethionine, thiourea, 1-thio-6-<span id='am-71' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-72' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>D-glucose</span>, <br /> gives number wells originally contained conflicts resolved                                                                                                                             DL-<span id='am-61' about='obo:IMR_0200456' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00248" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0200456" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Lipoamide" datatype="xsd:string"></span>lipoamide</span>, <span id='am-133' about='obo:IMR_0200004' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-134' property="oboInOwl:hasDbXref" content="KEGG:C05122" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:id" content="IMR:0200004" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="Taurocholic acid" datatype="xsd:string"></span>taurocholic acid</span> <br /> arbitration procedure. sum rows 2 5 equals row 1. <br /> b                                                                                                conflicting PM result recorded dulcitol (galactitol) genetic variation       Nutrients boldface growth growth  opposed growth/low-growth stock cultures E. coli K-12 MG1655.                                                 low-growth growth) conflicts. <br />  <br />  <br />  <br /> March 2014 Volume 196 Number 5                                                                                                                                           jb.asm.org 985 <br />  Mackie et al. <br />  <br />  <br />  <br /> TABLE 5 Growth E. coli K-12 MG1655 minimal salts agar M9                 TABLE 7 Gene essentiality datasets present EcoCyca <br /> minimal medium sole nitrogen sourcesa                                                                                          knockouts <br />                                              Growth  <br />                                                                                                                                                         Data <br />                                              Minimal                 M9 minimal   Data set      Medium                             growth     Growth       unavailable <br /> Nitrogen source                              salts agar              medium       Gerdes03      LB medium enriched,                           3,082        1,419 <br /> L-Cysteine                                   Y                       Y                            aerobic <br /> L-Glutamic      acid                         Y                       Y            Baba06        LB Lennox, medium,                 324        3,844        333 <br /> L-Lysine                                     Y                       Y                            aerobic <br /> Putrescine                                   Y                       Y            Baba06        MOPS &#226;&#171;&#185; 0.4% glucose,               15         3,828        658 <br /> <span id='am-21' about='obo:IMR_0200282' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:id" content="IMR:0200282" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00645" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="N-Acetyl-D-mannosamine" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>N-Acetyl-D-mannosamine</span>                       Y                       Y                            aerobic <br /> Adenine                                      Y                       Y            Feist07       MOPS &#226;&#171;&#185; 0.4% glucose,               107        996          3,398 <br /> Guanosine                                    Y                       NT                           aerobic <br /> Uric acid                                    Y                       N            Patrick07     M9 &#226;&#171;&#185; 0.4% glucose, aerobic         106                     4,395 <br /> DL-&#226;?&#163;-Amino-N-butyric acid                    Y                       N            Joyce06       M9 &#226;&#171;&#185; 1% glycerol, aerobic          113        3,704        684 <br /> L-Threonine                                  Y                       Y                                                                                                data set, list growth conditions gene <br /> L-Methionine                                 N                       Y            essentiality determined, number knockouts exhibiting growth, <br /> L-Phenylalanine                              N                       Y            growth, indeterminate level growth indicated. MOPS, <br /> D-Valine                                     N                       N            morpholinepropanesulfonic acid. <br /> Agmatine                                     N                       Y <br /> Uracil                                       N                       Y <br /> Uridine                                      N                       Y <br /> Xanthine                                     N                       N            chemical compounds comprise single growth medium. &#226;?&#165;-Amino-N-butyric acid                       N                       Y            new schema class &#226;&#8364;&#339;Growth-Observations&#226;&#8364;? defines Alloxan                                      N                       N            measurements growth nongrowth specified   Growth curves M9 minimal medium provided Fig. S4 S5        growth media. class specifies experimental condi- <br /> supplemental material. Y, yes; N,  NT, tested.                             tions temperature, aerobicity, gene knockouts <br />                                                                                   strain studied. <br />                                                                                       Growth media growth observations entered computationally predicted essential genes                    using interactive editing windows Pathway Tools. addi- <br /> genome-scale metabolic reconstruction E. coli K-12. data                  tion, growth observations imported data files, set  includes genes used model) included               importing PM data. data sets imported, EcoCyc facilitate benchmarking computational predic-                     user inspect compare growth data ways tions essentiality EcoCyc model predictions                 Pathway Tools-based web site EcoCyc.org. web site model Feist et al.(14). Based ability Keio col-              command &#226;&#8364;&#339;Search &#194;&#161; Growth Media&#226;&#8364;? enables searching lection single-gene knockout mutants grow rich versus                    growth media according chemical composition growth <br /> fined media, Baba et al. Joyce et al. (11, 13) identified set            outcome. addition, user growth medium conditionally essential genes. Patrick et al. (15) tested subset             navigating page listing growth observations recorded Keio mutants ability form colonies M9-glucose                  organism (http://biocyc.org/ECOLI/NEW-IMAGE?object <br /> agar.                                                                             &#226;&#171;&#189;Growth-Media), contains series diagrams, including <br />     Growth essentiality data Pathway Tools soft-                   shown Fig. 1. Clicking PM image page <br /> ware. extended Pathway Tools software (3) handle                   user page describing composition aspects growth observation data gene essentiality                  corresponding growth medium  Fig. S1 supplemental <br /> data. pathway/genome database (PGDB) EcoCyc                         material). PGDB pages chemical compounds include links capture growth observations wild-type mutant strains                    growth media contain compound. capture descriptions media growth                         growth-medium page  Fig. S1 supplemental ma- <br /> assayed. new schema class &#226;&#8364;&#339;Growth-Media&#226;&#8364;? lists set                     terial) lists composition growth medium forms:                                                                                   recipe substances used create medium (e.g., <br />                                                                                   ions listed salts) listing constituents TABLE 6 Growth E. coli K-12 MG1655 minimal salts agar M9                 medium (e.g., <span id='am-15' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> listed  concentration <br /> minimal medium sole carbon sourcesa                                          summed multiple constituents necessary). growth <br />                                          Growth                                data available, growth medium page lists wild- <br />                                          Minimal                     M9 minimal <br />                                                                                   type E. coli grows medium, growth status Carbon source                            salts agar                  medium       single-gene knockouts listed available. <br />                                                                                       EcoCyc gene pages include table listing growth status <span id='am-100' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-101' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span>L-Glutamate</span>                              N                           N <br /> &#226;?&#163;-Keto-valeric acid                      N                           N <br />                                                                                   knockouts gene growth media data Tween 20                                 Y/N                         N            available  Fig. S2 supplemental material). page <br /> Pectin                                   N                           N            listing growth media, possible generate heatmap com- <br /> &#226;?&#163;-Methyl-D-glucoside                     N                           N            paring gene essentiality growth different nutrient sources. <br /> L-Alaninamide                            N                           N            Currently, EcoCyc, gene essentiality data available   Growth curves M9 minimal medium provided Fig. S4 S5        growth glucose, glycerol, small number sub- <br /> supplemental material. N,  Y/N, ambiguous.                                     strates. PM data gene knockouts available,  <br />  <br /> 986   jb.asm.org                                                                                                                                Journal Bacteriology <br />                                                                                                          Addition Growth Essentiality Data EcoCyc <br />  <br />  <br />  <br /> power tool apparent. Figure S3           preclude possibility genetic variation stock <br /> supplemental material shows example heatmap data               cultures contribute variation PM data. <br /> currently available EcoCyc.                                                 tested different pregrowth agars used                                                                            experimenters (LB, R2A, BUG-S) compared response <br /> DISCUSSION                                                                 PM plates 1 2  Fig. S5 supplemental material). <br /> Differences PM data sets. addition high-                 Despite difference colony appearance agar plates, <br /> throughput Biolog PM data sets EcoCyc highlighted number              difference seen kinetic curves, apparent conflicts data regarding E. coli K-12 MG1655&#226;&#8364;&#8482;s          conclude variation PM response carbon sources <br /> ability use various compounds sole source carbon,             data sets collated present study nitrogen, phosphate, sulfur. integration exercise suc-         differences pregrowth conditions. potential contribu- <br /> cessful identifying conflicting information resolv-         tor disagreement PM data sets different labora- <br /> ing conflicts based low-throughput growth data            tories use different criteria distinguish growth, literature experimental results.           growth, low growth. unable investigate factor <br /> interesting briefly discuss possible reasons disparity       original growth curve data available growth observation PM data highlights consider-             data sets. existence international repository PM <br /> ations arise attempting unify data various               data included growth curves facilitate data reanal- <br /> sources.                                                                   ysis. <br />      clear difficulty integrating data sets             Growth tests minimal salts agar M9 minimal medium <br /> founding effect result slight variations experi-          identified 15 compounds support growth sup- <br /> mental method used laboratory. Potential reasons dis-          plied sole nitrogen source despite generating conflicting  <br /> agreement PM data sets added EcoCyc include                 sults PM data sets (Table 5). Varying incubation <br /> genetic differences E. coli K-12 MG1655 stock cultures, (ii)        time likely confounding factor nitrogen source PM assays. <br /> variation pregrowth conditions bacteria (LB, R2A,         Long lag times particularly noticeable response nitrogen <br /> BUG-S agar), (iii) variation length incubation times used        compounds  Fig. S4 supplemental material, PM3) assay (26 48 h), (iv) variability choice cutoff   contribute higher conflict rate observed nitro- <br /> values assigning positive/negative results respiration      gen source plate. example, nitrogen source PM assays, 6 <br /> PM assays.                                                                 compounds lag time &#226;&#172;&#381;20 h, 10 additional com- <br />      Genetic variation stock cultures E. coli K-12              pounds lag time &#226;&#172;&#381;24 h  Data set S6 supplemen- <br /> MG1655 documented literature (16, 17) likely           tal material). suggests important incubate Biolog implications physiological experiments. predicted,          nitrogen source plates 48 h recommended example, frameshift gatC present stock       manufacturer. nitrogen compounds tested, cultures E. coli K-12 MG1655 result nonfunctional             variation detecting growth minimal <br /> product inability transport D-galactitol (dulcitol),      salts agar compared M9 minimal medium. example, likely differences PM data compound         growth E. coli K-12 using &#226;?&#163;-amino-N-butyrate uric acid genetic variation stock cultures. able identify        sole nitrogen source detected soft agar different stock cultures E. coli K-12 MG1655 used          liquid culture, growth L-methionine, L-phenylalanine, <br /> generate data sets added EcoCyc: came central              agmatine, uracil, uridine, amino-N-butyrate sole nitrogen <br /> repositories (ATCC CGSC), laboratory-held                sources detected liquid minimal media soft <br /> strain. tested stock cultures (ATCC 700926             agar. agar used plate media contain impurities, CGSC7740) PM plates 1 4 compared responses                 contact solid surface neighboring cells result  Fig. S6 supplemental material). Overall, little      differential regulation enzymes. <br /> variation response majo <br /> </body></html>