<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                  Structure function <span id='am-48' about='obo:IMR_0200161' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00475" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Cytidine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0200161" datatype="xsd:string"></span>cytidine</span> <br />                                                   monophosphate kinase                                                   Yersinia pseudotuberculosis, <br /> rsob.royalsocietypublishing.org <br />                                                   essential virulence Research <br />                                                   survival <br /> Cite article: Walker NJ, Clark EA, Ford      Nicola J. Walker1, Elizabeth  Clark2, Donna C. Ford1, Helen <br /> DC, Bullifent HL, McAlister EV, Duffield ML, <br /> Acharya KR, Oyston PCF. 2012 Structure        L. Bullifent1, Erin V. McAlister1,&#226;&#8364;&#160;, Melanie L. Duffield1, <br /> function cytidine monophosphate kinase <br />                                                   K. Ravi Acharya2 Petra C. F. Oyston1 Yersinia pseudotuberculosis, essential virulence survival. Open Biol 2:      1 <br />                                                     Biomedical Sciences, Defence Science Technology Laboratory, Porton  <br /> 120142.                                           Salisbury SP4 0JQ, UK <br /> http://dx.doi.org/10.1098/rsob.120142             2 <br />                                                     Department Biology Biochemistry, University Bath, Claverton  <br />                                                   Bath BA2 7AY, UK <br />  <br /> Received: 2 October 2012 <br /> Accepted: 13 November 2012                        1. Summary <br />                                                   need new antibiotics pressing light emergence                                                   antibiotic-resistant strains human pathogens. Yersinia pestis, causative <br />                                                   agent plague, public health threat agent concern biode- <br /> Subject Area:                                     fence. recently emerged clonal derivative enteric pathogen Yersinia <br /> microbiology                                      pseudotuberculosis. Previously, developed bioinformatic approach ident- <br />                                                   ify proteins suitable targets antimicrobial therapy Keywords:                                         particular treatment plague. target cytidine monophos- <br />                                                   phate (<span id='am-133' about='obo:IMR_0200494' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-134' property="oboInOwl:id" content="IMR:0200494" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="KEGG:C00055" datatype="xsd:string"></span><span id='am-136' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-137' property="rdfs:label" content="CMP" datatype="xsd:string"></span>CMP</span>) kinase, essential gene organisms. Previously, <br /> cytidine monophosphate kinase, crystal <br />                                                   thought CMP kinase essential Y. pseudotuberculosis, structure, virulence, Yersinia                    modification mutagenesis approach, report production                                                   characterization Dcmk mutant. isogenic mutant growth defect <br />                                                   relative parental strain, highly attenuated mice.                                                   elucidated structure CMP kinase 2.32        &#203;&#353; , identified <br /> Author correspondence: <br />                                                   key residues active site essential activity Petra C. F. Oyston <br />                                                   enzyme. findings implications development novel <br /> e-mail: pcoyston@dstl.gov.uk                      CMP kinase inhibitors therapeutic use. <br />  <br />  <br />  <br />  <br />                                                   2. Introduction <br />                                                   Cytidine monophosphate (CMP) kinase member <span id='am-72' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-73' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> <br />                                                   monophosphate (NMP) kinase family, plays crucial role biosynthesis <br />                                                   nucleoside precursors. bacteria, CMP kinase catalyses transfer &#226;&#8364;&#160; <br />                                                   phosphoryl group <span id='am-53' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> CMP <span id='am-113' about='obo:IMR_0200005' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="dCMP" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:id" content="IMR:0200005" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasDbXref" content="KEGG:C00239" datatype="xsd:string"></span>dCMP</span>. differs activity <br />  Present address: Imperial College, London        eukaryotic CMP kinases, catalyse conversion <span id='am-108' about='obo:IMR_0200409' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-109' property="rdfs:label" content="UMP" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="KEGG:C00105" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:id" content="IMR:0200409" datatype="xsd:string"></span>UMP</span> CMP <br /> SW7 2AZ, UK.                                      respective diphosphate form [1],  structurally, bacterial CMP kinases <br />                                                   different members NMP kinase family [2]. CMP kinase <br />                                                   expression suppress lethal effects inactivation UMP <br />  <br />  <br />                                                    <span id='am-36' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-37' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; 2012 Authors. Published Royal Society terms Creative Commons Attribution <br />                                                    License http://creativecommons.org/licenses 3.0/, permits unrestricted use, provided original <br />                                                    author source credited. <br />  kinase, PyrH [3,4], indicated low level UMP kinase           plague, inhibitors bacterial CMP kinase        2 <br /> activity CMP kinase, observation confirmed later [5].             represent novel class broad-spectrum antimicrobials. <br />  <br />  <br />  <br />  <br />                                                                                                                                          rsob.royalsocietypublishing.org <br />      CMP kinase reported essential viability Bacillus subtilis [6] Streptococcus pneumoniae [7].  <br /> inactivation cmk reportedly lethal Escherichia coli        3. Material methods <br /> [8] Salmonella enterica [9]. Haemophilus influenzae, <br />                                                                         Unless stated, chemicals purchased copies cmk [10], reported <br />                                                                         Sigma-Aldrich (Poole, UK), enzymes purchased E. coli.  conflicting data regarding <br />                                                                         Promega (Southampton, UK). cmk essential E. coli, mutagenesis <br /> studies indicate cmk  fact, essential organism <br /> [11,12], essentiality target influenced            3.1. Production Yersinia pseudotuberculosis <br /> strain method used. mutant generated,                                                                              Dcmk mutant <br />  <br />  <br />  <br />  <br />                                                                                                                                          Open Biol 2: 120142 <br /> E. coli mutant significant growth defect demon- <br /> strated cold sensitivity [8]. Previously, attempted           Construction Y. pseudotuberculosis Dcmk mutant create cmk mutant Yersinia pseudotuberculosis, suc-        generated reported previously [13] modification <br /> cess, indicated cmk essential gene             primers designed amplify kanamycin- <br /> member Enterobacteriaceae [13].                                  resistance cassette including cognate promoter <br />      Yersinia pseudotuberculosis human pathogenic       region. primers used  50 -CTGCCGGGGGCAGAC <br /> members genus, Yersinia enterocolitica          AAGAGATTTGCCTACCGAAAGGAGAGATAAATGAGCC Yersinia pestis. Y. pseudotuberculosis                 ATATTCAACGGG-30 (forward) 50 -CGTATTGAGAGCG <br /> Y. enterocolitica enteric pathogens, Y. pestis causes bubonic       CAGAACTGACCGCATTGTCGTTAGAAAAACTCATCGAG pneumonic plague. Yersinia pestis considered             CATC-30 (reverse), sequence italic font hybri- <br /> recently emerged clonal derivative Y. pseudotuberculosis             dized kanamycin-resistance gene. PCR products [14]. evolution Y. pestis enteropathogen              generated using plasmid pK2 [19] template, arthropod-vectored systemic pathogen involved                  excess template digested DpnI. PCR products <br /> gene acquisition loss [15], organisms retain            purified using Millipore Microcon Ultracel YM-100 <br /> high levels genetic similarity. result, Y. pseudotuberculosis   transformed Y. pseudotuberculosis exploited safer model pathogen elucidating            IP32953 [20] pAJD434 [21] electroporation. Following <br /> pathogenic mechanisms Yersinia, proteins retain             overnight incubation 288C Luria Bertani (LB) broth sup- function species, results validated       plemented 0.8 cent arabinose, transformants subsequently plague bacillus focused efforts.            selected LB agar supplemented kanamycin <br />      Approximately 2000 cases plague reported                    (50 mg ml21) trimethoprim (100 mg ml21) 48 h annually World Health Organization. case               288C. Transformants screened PCR using target <br /> suspected plague infection, recommended antibiotics             gene-specific (50 -TTTGCCTACCGAAAGGAGAG-30 50 - <br /> therapy gentamicin doxycycline quinolones [16],              GCGCAGAACTGACCGCATTG-30 ) kanamycin gene- prompt initiation therapeutic regimens essential, par-          specific primers (50 -GCCATATTCAACGGGAAACG-30 ticularly highly acute pneumonic form disease.               50 -AAACTCACCGAGGCAGTTCC-30 ). <br /> Antibiotic resistance reported clinical isolates                 Mutant strains cured pAJD434 plasmid [17], highly efficient transfer resistance genes           growth 378C LB medium supplemented kanamy- midgut flea [18]. human human transmission             cin (50 mg ml21). Cured mutant strains screened risk plague cases, need effective        virulence plasmid pYV PCR genes located <br /> vaccines antibiotics prevent outbreaks arising              plasmid: virF yscC. retention Yersinia <br /> enzootic areas.                                                         virulence plasmid ( pYV) confirmed culture      Previously, reported bioinformatic approach              Congo-red magnesium oxalate plates, plasmid reten- identify potential new targets novel antimicrobials              tion results small red colonies plasmid loss results [13]. database essential genes (DEG) records               large pink colonies. <br /> genes thought essential range bacteria. Using                Mutation cmk confirmed Southern blotting. selection process included conservation                Genomic DNA digested overnight Bpu10I, resolved <br /> DEG, cmk identified potential lethal gene,                0.7 cent agarose gel blotted positively <br /> cmk homologues identified 11 14 prokaryotes                   charged nylon membrane (Roche). Southern hybridization <br /> listed DEG. selection criteria, including size,                performed using digoxigenin-labelled probe amplified <br /> enzymatic function non-membrane protein, provided                   open reading frame located immediately  <br /> additional support CMP kinase &#226;&#8364;&#732;druggable&#226;&#8364;&#8482; target               stream cmk encoding 30S ribosomal protein S1 (rpsA; inhibition active site. study,                primer sequences 50 -GAAAGCTAAGCGTCACGAAG-30 wished definitively prove CMP kinase                  50 -GTCCAAGCCGATGAAGATAC-30 ) Roche DIG <br /> essential locus mutated. elucidated crystal             according manufacturer&#226;&#8364;&#8482;s instructions. <br /> structure CMP kinase Y. pseudotuberculosis vali- <br /> dated function enzyme kinase. homology modelling, identified potentially impor- <br />                                                                         3.2. Expression purification CMP kinase <br /> tant protein &#226;&#8364;&#8220;ligand interactions elucidated                    Y. pseudotuberculosis cmk gene amplified PCR <br /> essentiality number interactions site-directed         genomic DNA using Phusion DNA polymerase (Finn- <br /> mutagenesis. results provide basis           zymes, NEB) oligonucleotide primers GSTcmkfor <br /> research development novel therapeutics                 50 GGATCCATGACGGCGATAGCCCCGGTGATA30 (BamH1 <br />  site indicated) GSTcmkrev 50 GCGGCCGCTTATTTTTT                PHENIX [26]. set reflections (5%) set aside         3 <br /> CAACGGCAAGG30 (Not1 site indicated). amplified                Rfree calculation [27]. <br />  <br />  <br />  <br />  <br />                                                                                                                                   rsob.royalsocietypublishing.org <br /> fragment ligated blunt cloning vector pCR- <br /> Blunt II-TOPO (Invitrogen) using T4 DNA ligase (Roche). cmk insert excised plasmid restriction        3.5. Homology modelling <br /> digestion BamH1 Not1, ligated               model Yersinia CMP kinase created using SWISS <br /> pGEX-6P-1 (GE Healthcare) vector. Fidelity cmk gene        modeller server [28]. model based sequence confirmed sequencing.                                      alignment Yersinia CMP kinase E. coli CMP <br />      Escherichia coli BL21*(DE3) harbouring recombinant           kinase <span id='am-43' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="CDP" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span>CDP</span> bound (PDB code 2CMK). coordinates <br /> pGEX-6P-1 plasmids cultured LB broth supplemen-           2CMK used add CDP model. <br /> ted 1 cent glucose ampicillin (50 mg ml21). <br /> Cultures grown shaking (170 rpm) 378C  <br />  <br />  <br />  <br />                                                                                                                                   Open Biol 2: 120142 optical density 600 nm (OD600) 0.4 reached.          3.6. CMP kinase activity assay <br /> Protein expression induced 1 mM isopropyl-B- <br />                                                                   CMP kinase activity determined using coupled spec- <br /> D-thiogalactoside (Roche) incubation 4 h <br />                                                                   trophotometric assay based Blondin et al. [29]. <br /> followed harvesting centrifugation (10 min, 1700 g). <br />                                                                   Briefly, determination Km CMP, 1 ml Cell pellets resuspended PBS sonicated                                                                   50 mM Tris pH 7.4 containing 2 mM MgCl2, 50 mM KCl, ice water bath. suspension clarified centrifu- <br />                                                                   1 mM phosphoenolpyruvate, 0.2 mM <span id='am-31' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span>NADH</span>, 1 mM ATP, <br /> gation 27 000g 30 min. Supernatants loaded                                                                   units lactate dehydrogenase, pyruvate kinase GSTrap FF column (GE Healthcare) equilibrated                                                                   NDP kinase, various concentrations CMP, PBS. column washed equilibration buffer, <br />                                                                   determination Km ATP 0.1 mM CMP var- <br /> followed cleavage buffer (50 mM Tris, 150 mM NaCl, <br />                                                                   ious concentrations ATP incubated 308C 1 mM EDTA, 1 mM DTT pH 7). PreScission Protease (GE <br />                                                                   absorbance monitored 340 nm. stable absorbance <br /> Healthcare) cleavage buffer added (160 units milli- <br />                                                                   reached (10 mM), CMP kinase added litre column matrix) incubated overnight 48C.                                                                   decrease absorbance recorded. <br /> column washed cleavage buffer, fractions <br /> containing cleaved CMP kinase pooled buffer <br /> exchanged 50 mM HEPES, 150 mM NaCl, 10 mM <br />                                                                   3.7. Generation point mutations CMP kinase <br /> glycerol pH 7. <br />      structural analysis, elution fractions containing        Point mutants Arg188Ala, Arg131Ala Arg110Ala CMP kinase determined SDS&#226;&#8364;&#8220;PAGE analysis                generated using Stratagene Quikchange site-directed <br /> pooled buffer exchanged 50 mM MES pH 6.0,                mutagenesis kit. Mutagenesis carried 150 mM NaCl. Protein concentrated approximately            recombinant cmk-pGEX-6P-1 plasmids, using oligonucleotide <br /> 6.4 mg ml21, using Millipore concentrator.                      primers 50 -GGATAACCGTGATCGTAACGCTGCTGTTGCA <br />                                                                   CCTTTAGTC-30 , 50 -GGTTTAATTGCTGATGGCGCGGATAT <br />                                                                   GGGGACTATCGT-30 50 -CGGCTTTCCCCCGAGTAGCTG <br /> 3.3. Crystallization CMP kinase                                AAGCATTACTGC-30 . Mutated plasmids transformed <br /> Initial screens identify conditions CMP kinase        E. coli XL-1 blue plated LB plates supplemented <br /> crystallized carried using pre-prepared 96          ampicillin (50 mg ml21). Colonies screened PCR <br /> screens (Molecular Dimensions) Art Robbins Phoenix          sequenced using T7 universal primers. <br /> nano-dispenser. Drops 300 nl set using sitting <br /> drop method. Optimization carried 24 plates <br /> using hanging drop method. Improved crystals               3.8. Circular dichroism spectroscopy <br /> CMP kinase grew 7 days 2 ml drop contained         Circular dichroism (CD) studies performed Jasco 1 : 1 volume ratio protein solution mother liquor.        J-600 spectropolarimeter 2 mm circular cell. Samples <br /> Mother liquor contained 100 mM sodium HEPES pH 7.5,               wild-type CMP kinase (0.7 mg ml21) 1.2 M lithium sulphate 100 mM magnesium chloride.             mutants Arg188Ala (0.3 mg ml21), Arg131Ala (0.4 mg ml21) <br />                                                                   Arg110Ala (0.7 mg ml21) prepared 50 mM <br /> 3.4. Structure determination refinement                       HEPES, 150 mM NaCl 10 mM glycerol pH 7.0. Wave- <br />                                                                   length scans performed 208C 195 250 nm X-ray diffraction data collected single cryo-         rate 50 nm min21. <br /> cooled crystal (100 K) 2 M lithium sulphate Diamond Light Source (Didcot, UK) ADSC Q315 <br /> CCD detector station I02 (l &#194;&#188; 0.98 &#203;&#353; ). Diffraction images   3.9. vitro evaluation growth collected oscillation angle 1.08. Data pro- <br /> cessed rhombohedral space group R3 using XDS [22], <br />                                                                        Dcmk mutant data scaled using SCALA [23]. Initial phases             288C agitation, cultures strain IP32953 obtained molecular replacement using             Dcmk mutant grown overnight LB LB sup- <br /> program PHASER [24]. search model used molecu-            plemented 50 mg ml21 kanamycin, respectively. lar replacement model Yersinia CMP kinase                overnight cultures used inoculate fresh pre-warmed <br /> generated based sequence alignment E. coli            medium, cultures incubated  <br /> CMP kinase (PDB code 1CKE). Model building                selected intervals, aliquots removed OD600 <br /> formed COOT [25], refinement performed               measurement enumeration viable counts. <br />      vitro competition index (CI) experiments              2.500                                                                  4 <br /> ducted prior vivo CI evaluation. Cultures  <br />  <br />  <br />  <br />                                                                                                                                              rsob.royalsocietypublishing.org <br /> strains grown LB overnight 288C agitation,                     2.000 diluted OD600 0.1 LB. cultures seri- <br /> ally diluted viable counts determined culture LB                     1.500 <br />  <br />  <br />  <br />  <br />                                                                     OD600 <br /> agar. Wild-type mutant bacterial suspensions mixed 1 : 1 ratio incubated 288C shaking.                     1.000 selected intervals, aliquots removed OD600 <br /> measurement enumeration viable counts. CI                        0.500                                              wt <br /> defined output ratio (mutant/wild-type) divided                                                                         Dcmk input ratio (mutant/wild-type).                                                 0 <br />  <br />  <br />  <br />  <br />                                                                                                                                              Open Biol 2: 120142 <br />                                                                     (b) 1 &#195;&#8212; 1010 <br />  <br /> 3.10. vivo studies Dcmk mutant <br /> Attenuation Y. pseudotuberculosis Dcmk mutant eval-                 1 &#195;&#8212; 109 <br /> uated CI median lethal dose (MLD) evaluation. Mutant <br />  <br />  <br />  <br />  <br />                                                                     cfu ml&#226;&#8364;&#8220;1 wild-type strains grown separately 100 ml LB <br /> broth shaking 288C overnight. cultures                   1 &#195;&#8212; 108 <br /> centrifuged (10 min, 6000g) pellet resuspended 4 ml sterile 10 cent v/v aq. glycerol. Aliquots stored                                                              wt count 2808C required. aliquot strain                                                                      Dcmk count <br />                                                                                1 &#195;&#8212; 107 <br /> defrosted, serially diluted cultured determine cfu ml21.                          0   5        10        15           20 <br />      CI challenge, aliquot strain                                              time (h) <br /> defrosted, diluted PBS approximately 1   109 cfu ml21, wild-type mutant bacterial suspensions mixed                                                                     Figure 1. Growth Y. pseudotuberculosis IP32953 Dcmk mutant, <br /> 1 : 1 ratio. suspension serially diluted ster- <br />                                                                     followed   optical density (b) viable count. <br /> ile PBS. Retrospective viable counts determined plating dilutions triplicate LB agar LB agar sup- <br /> plemented kanamycin determine input ratio.                  pure culture, growth Dcmk mutant appeared <br /> Groups mice dosed 0.1 ml            slower wild-type, eventually <br /> dilutions intravenous route. 3 days, spleens           reached similar final density (figure 1). fitness defect removed passed 70 mm sieves (Becton&#226;&#8364;&#8220;               observed vitro CI, value 0.186 indicated <br /> Dickinson) produce cell suspension 3 ml PBS. Cell        reduced fitness mutant compared wild-type. <br /> suspensions serially diluted sterile PBS plated             Mice challenged p. serial dilutions wild- LB agar, LB agar supplemented kanamycin            type Y. pseudotuberculosis Dcmk mutant. Retrospective <br /> determine output ratio. CI determined          challenge doses showed mice challenged IP32953 <br />      MLD determination, aliquot culture          received 6.4   105 &#226;&#8364;&#8220;0.64 cfu, mice challenged defrosted diluted PBS. Retrospective viable counts           mutant received 2.2   107 &#226;&#8364;&#8220; 22 cfu. published MLD determined plating dilutions triplicate LB        wild-type strain 2200 cfu [31]. study, MLD <br /> agar. Groups mice dosed intraperitoneally  p.)       wild-type strain 940 cfu. Dcmk mutant 0.1 ml dilutions. Mice observed twice            significantly attenuated, MLD 9.28   107 cfu <br /> daily, animals moribund deemed incap-             (figure 2), signs recorded mice receiving <br /> able survival humanely killed. Mice observed           2.2   105 cfu mutant. confirmed 28 days. MLD calculated reported [30].               vivo CI, CI 4.48   1026 indicated significant <br />                                                                     reduction fitness compared wild-type  CI                                                                     0.2 considered significant reduction fitness <br />                                                                     model). <br /> 4. Results <br /> 4.1. Inactivation CMP kinase highly attenuating               4.2. CMP kinase crystal structure Y. pseudotuberculosis cmk gene inactivated replace-      CMP kinase molecule crystallographic <br /> ment gene kanamycin-resistance cassette driven        asymmetric unit, solvent content 48 cent. cmk promoter. Following transformation PCR          final refined structure Rcryst 22.2 cent product, colonies grew LB kanamycin agar,              Rfree 27.8 cent (table 1). final model consists selected screening PCR.               18 water molecules sulphate ions asymmetric <br /> shown mutants lacking cmk gene carrying           unit. Ramachandran plot generated Molprobity <br /> kanamycin cassette. Following curing pAJD434 plas-           shows residues disallowed regions [32]. <br /> mid, confirmation unstable Yersinia virulence          Clear density observed Met1 Ala225 plasmid pYV retained, mutation            exception 12-residue gap Glu180 Ala191. <br /> mutants confirmed Southern blotting, size            N-terminal residues observed. chain <br /> difference approximately 4 kb observed           density observed residues Glu57, Glu58, Arg92 <br /> Dcmk mutant wild-type.                                          Glu122, chains modelled. <br />        6                                                                                      Table 1. Crystallographic data CMP kinase structure. Numbers     5 <br />                                                                                                parentheses upper resolution shell (2.32&#226;&#8364;&#8220; 2.45  ),  <br />  <br />  <br />  <br />                                                                                                                                                                          rsob.royalsocietypublishing.org <br />                        5                                                                       appropriate. <br />  <br />                        4 <br />      survivors <br />  <br />  <br />  <br />  <br />                                                                                                                                  5 b 5 88.64  , c 5 84.29  ; <br />                        3 <br />                                                                                                    cell dimensions               5 b 5 9088, g 5 12088 <br />  <br />                                                                                                    space group                   R3 <br />                        2 <br />                                                                                                    resolution   )              2.32 <br />                        1                                                                           completeness (%)              96.4 (87.4) <br />                                                                                                     re&#239;&#172;&#8218;ections             44 472(4160) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                          Open Biol 2: 120142 <br />                        0 <br />                                WT 6.4 &#195;&#8212; 105             WT 6.4 &#195;&#8212; 104           WT 6.4 &#195;&#8212; 103         unique re&#239;&#172;&#8218;ections      10 266(1370) <br />    (b) 6 <br />                                          WT 640 cfu               WT 64 cfu                        redundancy (%)                4.3 (3.0) <br />                                                                                                    sI                          16.2 (2.8) <br />                        5                                                                           Rmerge (%                   4.7 (46.4) <br />                                                                                                              b <br />                                                                                                    Rcryst (%)                    22.2 <br />                        4                                                                           Rfree (%)c                    27.8 <br />      survivors <br />  <br />  <br />  <br />  <br />                                                                                                    Wilson B-factor   2)        61.6 <br />                        3 <br />                                                                                                    average B-factors main        64.2/67.2 <br />                        2                                                                              chain chain   2) <br />                                                                                                    r.m.s.d. ideal values <br />                        1 <br />                                                                                                       bonds   )                0.008 <br />                                                                                                       angles (8)                 1.168 <br />                        0 <br />                                                                                                                           0             5               10               15              20     Rmerge &#194;&#188; ShklSijIi (hkl 2 kI(hkl)lj/ShklSIi (hkl), kIl                                                  days post-infection                           averaged intensity observations reflection hkl. <br />                                                                                                b <br />                                                                                                  Rcryst &#194;&#188; SjjFoj2jFcjj/SjFoj, Fo Fc observed <br />                                cmk 2.2 &#195;&#8212; 107           cmk 2.2 &#195;&#8212; 106           cmk 2.2 &#195;&#8212; 105 <br />                                                                                                calculated structure factors, respectively. <br />                                cmk   2.2 &#195;&#8212; 104         cmk    2.2 &#195;&#8212; 103        cmk 220 cfu     c <br />                                                                                                 Rfree equal Rcryst random set reflections (5%) <br />                                                        cmk 22 cfu                              used refinement [27]. <br /> Figure 2. Survival mice following challenge   wild-type <br /> Y. pseudotuberculosis IP32953 (b) Dcmk mutant. <br />                                                                                                CMP kinase (PDB code 1CKE), residues 180&#226;&#8364;&#8220;192                                                                                                disordered absence ligand. <br />     Yersinia CMP kinase adopts overall fold                                            structure, anion hole formed <span id='am-84' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-85' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-87' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span>- <br /> related CMP kinases composed helices                                      rich loop forms mononucleotide-binding <br /> seven b-strands connected loops. arranged                                         motif. forms ATP-binding site, Yersinia domains (figure 3). central portion mol-                                 CMP kinase, sulphate <span id='am-25' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span>ion</span> bound. sulphate ion <br /> ecule composed central b-sheet, b1, b2,                                   occupies position previously published <br /> b5, b6 b7, a1 a9. Helix a6 joins central                                       bacterial CMP kinase structures crystallized <br /> domain NMP-binding domain. NMP-binding                                              absence ATP [2,33]. <br /> domain composed anti-parallel b-strands (b3 b4) helices (a2, a3, a4 a5). Yersinia CMP kinase described member class long-NMP kinases <br />                                                                                                4.3. CMP kinase: CDP model <br /> owing insert NMP-binding domain,                                           model Yersinia CMP kinase CDP bound cre- present short variants: structure,                                 ated based sequence alignment Yersinia <br /> insert comprises anti-parallel b-strands, loop region                                    CMP kinase E. coli CMP kinase CDP bound (PDB connects b4 b4, a5. lid domain composed                                     code 2CMK). proteins share 84 cent identity helices a7 a8. Yersinia CMP kinase E. coli CMP                                      94 cent similarity, ligand-interacting resi- <br /> kinase (PDB code 1CKE) superimpose                                        dues conserved (figure 5). compared root mean square deviation (r.m.s.d.) 210 Ca atoms                                       structure Yersinia CMP kinase absence ligand <br /> 0.91  . proteins share 84 cent identity 93                                   model CDP-bound Yersinia CMP kinase cent similarity.                                                                               order propose potential binding-induced changes     ligand-binding site formed pocket                                       Yersinia CMP kinase. <br /> NMP-binding domain. related structures, CMP CDP                                             NMP-binding domain undergoes movement bind cytosine base ligand sits                                     substrate binding, notably anti-parallel b-strands <br /> C-terminal end b5 strand, phosphate moiety                                      b3 b4. a5-helix moves binding site, positioned a2&#226;&#8364;&#8220; b2 boundary [8,9]. sulphate                                  giving closed conformation. homology model <br /> ion bound site (figure 4). Residues Glu180 Ala191                                contains section residues E180 A191, disordered structure. case E. coli                            seen apo-enzyme. homology model, section <br />                                                                                                                                                                     6 <br />                                                                        a5 <br />                                                                                                                           a8 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  rsob.royalsocietypublishing.org <br />                                                                                                                            lid domain <br />                                                                                         a4 <br />                                                                                                                  a7 <br />                                          a3 <br />                                                                             a2 <br />                                                                                              b4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  Open Biol 2: 120142 <br />                                                         a6             b3 <br />                                                                                                       a1 <br />                                                                             b2 <br />                           NMP-binding domain                                             b5 <br />  <br />                                                                                                            b6 <br />                                                                                                                      b7 <br />  <br />                                                                                         b1 <br />                                                                                                                 a9 <br />  <br />  <br />  <br />                                                                    N <br />  <br />                                                                        central domain             C <br /> Figure 3. overall fold CMP kinase shown. helices b-strands numbered. N- C-termini labelled. central domain, lid domain NMP-binding domains coloured pink, blue orange, respectively. figure prepared using PYMOL (v. 0.99; Schro&#194;&#168;dinger, LLC; www.pymol.org). <br />  <br />  <br />                                                                                         Briozzo et al. [2] suggested region <br />                                                                                         ordered binding. notable  <br />                                                                                         chain rearrangements Arg131, Arg110                              R110                                                       Arg41. Arg131 Arg41 chains make hydrogen <br />                                               R110*                                     bonds phosphate moieties ligand. Arg110 <br />                                                                                         chain forms hydrogen bonds cytosine base <br />                                                                                         CDP, does nearby Asp132 chain. Residues <br />                                                                                         Ser36 Ala38 N-terminal end a2 helix pre- <br />                 R188* <br />                                                                                         dicted away ligand-binding site,                                                                     S36*                predicted hydrogen bond Ser36                                                                                         chain cytosine base. Adjacent section, Tyr40 <br />                    R131              CDP                                                moves stacking interaction                                                     S36 <br />                                                                                         cytosine base CDP. phosphate moiety       D185*                                                        G37*                 ligand a2&#226;&#8364;&#8220;b2 boundary Ser14 hydrogen <br />                                                                                         bonded CDP. <br />                        R131* <br />                                                              G37 <br />  <br />                                                                                         4.4. Activity Yersinia CMP kinase <br />                                                                                         activity CMP kinase confirmed vitro <br />                                                              R41*                       assay. constant concentration 1 mM ATP, Km <br />                                           SO42&#226;&#8364;&#8220;                                         kcat values CMP 0.028 mM 91.9 s21, respect- <br />                                                     R41                                 ively, constant concentration 0.1 mM CMP, Km <br />                                                                                         ATP determined 0.04 mM kcat 74.3 s21. <br /> Figure 4. Active site CMP kinase. Green sticks: positioning sulphate          comparison activity data available litera- <br /> ion CMP kinase active site number surrounding residues                  ture (table 2), KCMPm    Yersinia CMP kinase shown. Blue sticks: model CDP-bound CMP kinase shown residue                  similar E. coli B. subtilis. KCMP <br />                                                                                                                                            m    labels denoted asterisk. Predicted hydrogen bonds shown              M. tuberculosis protein,  approximately threefold <br /> black dashes. figure prepared using PYMOL.                                     higher bacteria, kcat twofold <br />                                                                                         lower Yersinia E. coli proteins. Yersinia mainly helical extension a8 helix. Resi-                          E. coli CMP kinases inhibited CMP 0.2 mM; <br /> dues Arg188 Asp185 section interact                                   B. subtilis protein slightly inhibited CDP homology model. case E. coli,                            concentrations 2 mM [37]. Yersinia protein KATP m                                                                                                                                                                      7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                rsob.royalsocietypublishing.org <br />                                                                                                                                                                Open Biol 2: 120142 <br /> Figure 5. Sequence alignment Streptococcus pneumoniae, Staphylococcus aureus, Mycobacterium abscessus, Yersinia pseudotuberculosis E. coli <br /> CMP kinase. Residues mutated Yersinia protein study highlighted. Alignment performed using CLUSTALW sequence alignment <br /> program [34]. <br />  <br />  <br /> Table 2. Summary kinetic data Yersinia CMP kinase bacterial CMP kinases available literature. <br />  <br />                                                                   CMP                                                   ATP <br />  <br />                                        reference                  Km (mM)                    kcat (s21)                 Km (mM)                kcat (s21) <br />  <br />   Y. pseudotuberculosis                study                 0.028                       91.9                      0.04                   74.3 <br />   E. coli                              [5,35]                     0.035                      103                        0.038                  &#226;&#8364;&#8221; <br />   M. tuberculosis                      [36]                       0.12                       52                         &#226;&#8364;&#8221;                      &#226;&#8364;&#8221; <br />   B. subtilis                          [37]                       0.04                       &#226;&#8364;&#8221;                          0.12                   &#226;&#8364;&#8221; <br />  <br />  <br />  similar E. coli protein, B. subtilis                   disease, responsible millions deaths <br /> protein KATP <br />           m    threefold higher.                                                 history. Unfortunately, need new antibiotics     basis structural analysis, site-directed mutants                      pressing Y. pestis organism long generated residues predicted play key role                          concern biodefence. Previously, reported active site. order assess mutants                        identification cmk potentially interesting correctly folded, CD experiments performed wild-                            essential gene target development type CMP kinase mutants Arg131Ala,                                  novel antibiotics [13]. study, modifying Arg110Ala Arg188Ala. CD spectra proteins                            approach mutating target gene, able pro- highly similar, indicating mutants cor-                          duce mutant Y. pseudotuberculosis cmk <br /> rectly folded. mutated proteins assayed                          inactivated. homologous DNA sequences oli- <br /> concentration 10 mM constant concentration                             gonucleotide primers  difference <br /> 1 mM ATP concentrations CMP 5 mM,                                   successful approach kanamycin-resist- <br /> activity seen, showing essential CMP                             ance cassette driven cmk promoter binding active site.                                                         kanamycin cassette promoter. explain  <br />                                                                                     flicting reports essentiality organisms                                                                                     mutant appears highly sensitive approach used <br />                                                                                     production. <br /> 5. Discussion                                                                            Similar E. coli [8], CMP kinase required normal need new antibiotics pressing light                        growth Y. pseudotuberculosis 288C. Owing issues emergence antibiotic-resistant strains human patho-                       virulence plasmid instability 378C, did evaluate <br /> gens. Y. pestis, causative agent plague, public health                  growth defect 378C, assumed <br /> threat parts world multiply antibiotic-                          growth defect temperature <br /> resistant strains reported [17],                       predicted bioinformatic analysis [13], need effective new antibiotics treat notorious                          Y. pseudotuberculosis Dcmk mutant significantly <br />  attenuated vivo, confirming potential target                                similar Km values Y. pseudotuberclosis                            8 novel antimicrobials pathogenic Yersinia.                               protein 0.04 0.028 mM, respectively. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     rsob.royalsocietypublishing.org report vivo characterization Dcmk mutant,                                  structure Yersinia CMP kinase reveals globular instances mutant genera-                                     protein consisting domains. overall <br /> ted published studies primarily focused                                     fold observed related NMP kinases. Yersinia CMP <br /> impact bacterial physiology [8,9]. fundamental                                       kinase contains insert NMP-binding domain, <br /> physiological function CMP kinase phosphorylation                                 characteristic long-NMP kinases. absence CMP produced turnover nucleic acids pro-                                    structure ligand bound, homology modelling <br /> duce di- tri-phosphates, inactivation cmk                                    predicted number changes place results increased internal levels CMP excretion                                ligand binding number protein &#226;&#8364;&#8220;ligand interactions <br /> <span id='am-123' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-124' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> cytidine [8,9]. <span id='am-138' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-139' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-140' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-141' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span>CTP</span> required DNA                                       predicted. interactions based synthesis, loss CMP kinase compensated                                       high identity Yersinia E. coli CMP kinase.  <br />  <br />  <br />  <br />                                                                                                                                                                                     Open Biol 2: 120142 Y. pseudotuberculosis similar way S. enterica,                               importance key arginine residues positions 131, novo synthesis CTP CTP synthetase                                        110 188 particular confirmed site- <br /> encoded pyrG maintains supply CTP [8,9]. addition                                directed mutagenesis, substitution key amino precursor RNA DNA synthesis, CTP                                         acids showed essential CMP binding <span id='am-118' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-119' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span>dCTP</span> involved <span id='am-10' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span>phospholipid</span> biosynthesis.                                    active site. detailed insights stereochemistry  effects inactivation                                     substrate binding CMP kinase underpin future struc- <br /> membrane synthesis, affect growth rate                                     ture-based efforts develop novel inhibitors targeting contribute attenuation. documented                                    CMP-binding site. apparently minor changes membrane compo- <br /> sition, induced mutation PhoPQ <br /> regulon [38], example, large impact virulence. <br />                                                                                             6. Acknowledgements <br />      Yersinia closely related E. coli,                             work supported Ministry Defence members Enterobacteriaceae, NMP kinases                                    Wellcome Trust (UK) equipment grant  088464 K.R  different biochemical physico-<span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> prop-                                   thank scientists station IO3 Diamond Light <br /> erties. example, despite high degree <span id='am-77' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-78' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-79' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span>                                    Source (Didcot, UK) Dr Nethaji Thiyagarajan support <br /> identity conservation <span id='am-64' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-71' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> binding cat-                                X-ray data collection, Rebecca Driesener alytic amino acids, thymidylate kinase Y. pestis                                   University Southampton help enzyme kinetic <br /> E. coli exhibited significant differences thermodynamic                                  data analysis. thank Marc Baylis technical <br /> properties phosphorylation activity [39].                                  support. atomic coordinates structure factors difference observed CMP kinase. reported                                  (code 4E22) deposited Protein Data Bank, <br /> Km E. coli CMP kinase 0.038 ATP phosphoryl                                  Research Collaboratory Structural Bioinformatics, Rutgers <br /> donor 0.035 CMP phosphate <span id='am-128' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-129' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span>acceptor</span> [5].                                      University, New Brunswick, NJ (www.rcsb.org). <br />  <br />  <br />  <br /> References <br /> 1.   Segura-Pena D, Sekulic N, Ort S, Konrad M, Lavie     6.  Sorokin  Serror P, Pujic P, Azevedo V, Ehrlich SD.   11. Kato Ji, Hashimoto M. 2007 Construction      2004 Substrate-induced conformational changes           1995 Bacillus subtilis chromosome region               consecutive deletions Escherichia coli <br />      human UMP/CMP kinase. J. Biol. Chem. 279,                  encoding homologues Escherichia coli mssA           chromosome. Mol. Syst. Biol. 3, 132. <br />      33 882&#226;&#8364;&#8220;33 889. (doi:10.1074/jbc.M401989200)                rpsA gene products. Microbiology 141,              12. Gerdes SY et al. 2003 Experimental determination <br /> 2.   Briozzo P et al. 1998 Structures Escherichia coli       311 &#226;&#8364;&#8220;319. (doi:10.1099/13500872-141-2-311)                 level analysis essential genes      CMP kinase complex CDP: new        7. Yu L et al. 2003 Solution structure function         Escherichia coli MG1655. J. Bacteriol. 185, <br />      fold nucleoside monophosphate binding               essential CMP kinase Streptococcus                   5673&#226;&#8364;&#8220;5684. (doi:10.1128/JB.185.19.5673-5684.2003) <br />      domain insights cytosine nucleotide               pneumoniae. Protein Sci. 12, 2613&#226;&#8364;&#8220;2621. (doi:10.       13. Duffield M, Cooper  McAlister E, Bayliss M, Ford D, <br />      specificity. Struct. Folding Des. 6, 1517&#226;&#8364;&#8220; 1527.           1110/ps.03256803)                                          Oyston P. 2010 Predicting conserved essential genes <br />      (doi:10.1016/S0969-2126(98)00150-6)                    8. Fricke J, Neuhard J, Kelln RA, Pedersen S. 1995         bacteria: silico identification putative drug <br /> 3.   Yamanaka K, Ogura T, Koonin EV, Niki H, Hiraga S.          cmk gene encoding cytidine monophosphate kinase            targets. Mol. Biosyst. 6, 2482&#226;&#8364;&#8220;2489. (doi:10.1039/ <br />      1994 Multicopy suppressors, mssA mssB,           located rpsA operon required          c0mb00001a) <br />      smbA mutation Escherichia coli. Mol. Gen. Genet.        normal replication rate Escherichia coli.           14. Achtman M, Zurth K, Morelli C, Torrea G, Guiyoule <br />      243, 9&#226;&#8364;&#8220;16. (doi:10.1007/BF00283870)                        J. Bacteriol. 177, 517&#226;&#8364;&#8220;523.                                 Carniel E. 1999 Yersinia pestis, cause 4.   Yamanaka K, Ogura T, Niki H, Hiraga S. 1992            9. Beck CF, Neuhard J, Thomassen E, Ingraham JL,               plague, recently emerged clone Yersinia <br />      Identification characterization smbA            Kleker E. 1974 Salmonella typhimurium mutants              pseudotuberculosis. Proc. Natl Acad. Sci. USA 96, <br />      gene, suppressor mukB mull mutant              defective cytidine monophosphate kinase (cmk).          14 043 &#226;&#8364;&#8220;14 048. (doi:10.1073/pnas.96.24.14043) <br />      Escherichia coli. J. Bacteriol. 174, 7517 &#226;&#8364;&#8220;7526.           J. Bacteriol. 120, 1370&#226;&#8364;&#8220;1379.                          15. Parkhill J et al. 2001 Genome sequence Yersinia <br /> 5.   Bucurenci N et al. 1996 CMP kinase Escherichia    10. Fleischmann RD et al. 1995 genome random             pestis, causative agent plague. Nature 413, <br />      coli structurally related nucleoside           sequencing assembly Haemophilus influenzae          523&#226;&#8364;&#8220;527. (doi:10.1038/35097083) <br />      monophosphate kinases. J. Biol. Chem. 271,                 Rd. Science 269, 496&#226;&#8364;&#8220; 512. (doi:10.1126/science.       16. Prentice MB. 2010 Plague: Yersinia pestis. Oxford <br />      2856&#226;&#8364;&#8220;2862. (doi:10.1074/jbc.271.5.2856)                    7542800)                                                   textbook medicine, 5th edn. (eds DA Warrell, TM <br />        CoxJD Firth), pp. 772&#226;&#8364;&#8220; 775. Oxford, UK: Oxford                crystallographic software. J. Appl. Crystallogr. 40,      33. Dhaliwal B, Ren J, Lockyer M, Charles  Hawkins AR,         9 <br />       University Press.                                             658 &#226;&#8364;&#8220;674. (doi:10.1107/S0021889807021206)                     Stammers DK. 2006 Structure Staphylococcus <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              rsob.royalsocietypublishing.org <br /> 17.   Guiyoule et al. 2001 Transferable plasmid-            25.   Emsley P, Lohkamp B, Scott  Cowtan K. 2010                 aureus cytidine monophosphate kinase complex <br />       mediated resistance streptomycin clinical             Features development coot. Acta Crystallogr.           cytidine 5&#226;&#8364;&#8482;-monophosphate. Acta Crystallogr. F <br />       isolate Yersinia pestis. Emerg. Infect. Dis. 7,            D Biol. Crystallogr. 66, 486&#226;&#8364;&#8220;501. (doi:10.1107/               Struct. Biol. Cryst. Commun. 62, 710 &#226;&#8364;&#8220;715. (doi:10. <br />       43 &#226;&#8364;&#8220;48. (doi:10.3201/eid0701.010106)                          S0907444910007493)                                            1107/S174430910602447X) <br /> 18.   Hinnebusch BJ, Rosso ML, Schwan TG, Carniel E.          26.   Adams PD et al. 2010 PHENIX: comprehensive              34. Larkin MA et al. 2007 Clustal W Clustal X <br />       2002 High-frequency conjugative transfer                   Python-based macromolecular structure              version 2.0. Bioinformatics 23, 2947&#226;&#8364;&#8220; 2948. (doi:10. <br />       antibiotic resistance genes Yersinia pestis         solution. Acta Crystallogr. D Biol. Crystallogr. 66,          1093/bioinformatics/btm404) <br />       flea midgut. Mol. Microbiol. 46, 349 &#226;&#8364;&#8220;354. (doi:10.           213 &#226;&#8364;&#8220;221. (doi:10.1107/S0907444909052925)                 35. Bertrand T et al. 2002 Sugar specificity bacterial <br />       1046/j.1365-2958.2002.03159.x)                          27.   Brunger  1992 Free R-value: novel statistical            CMP kinases revealed crystal structures 19.   Taylor VL, Titball RW, Oyston PCF. 2005 Oral                  quantity assessing accuracy crystal-               mutagenesis Escherichia coli enzyme. J. Mol. Biol. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              Open Biol 2: 120142 <br />       immunization dam mutant Yersinia                    structures. Nature 355, 472 &#226;&#8364;&#8220;475. (doi:10.1038/               315, 1099&#226;&#8364;&#8220; 1110. (doi:10.1006/jmbi.2001.5286) <br />       pseudotuberculosis protects plague.                   355472a0)                                                 36. Thum C, Schneider CZ, Palma MS, Santos DS, Basso <br />       Microbiology 151, 1919 &#226;&#8364;&#8220;1926. (doi:10.1099/mic.0.       28.   Schwede T, Kopp J, Guex N, Peitsch MC. 2003                   LA. 2009 Rv1712 locus Mycobacterium <br />       27959-0)                                                      SWISS-MODEL: automated protein homology-                   tuberculosis H37Rv codes functional CMP <br /> 20.   Chain PSG et al. 2004 Insights evolution          modeling server. Nucleic Acids Res. 31, 3381&#226;&#8364;&#8220;3385.            Kinase preferentially phosphorylates dCMP. <br />       Yersinia pestis genome comparison               (doi:10.1093/nar/gkg520)                                      J. Bacteriol. 191, 2884&#226;&#8364;&#8220;2887. (doi:10.1128/JB. <br />       Yersinia pseudotuberculosis. Proc. Natl Acad.      29.   Blondin C, Serina L, Wiesmuller L, Gilles  Barzu           01337-08) <br />       Sci. USA 101, 13 826&#226;&#8364;&#8220; 13 831. (doi:10.1073/pnas.              O. 1994 Improved spectrophotometric assay              37. Schultz CP et al. 1997 Structural catalytic <br />       0404012101)                                                   nucleoside monophosphate kinase activity using            properties CMP kinase Bacillus subtilis: 21.   Maxson  Darwin AJ. 2004 Identification                  pyruvate kinase/lactate dehydrogenase coupling                comparative analysis homologous enzyme <br />       inducers Yersinia enterocolitica phage shock            Anal. Biochem. 220, 219&#226;&#8364;&#8220;221. (doi:10.                 Escherichia coli. Arch. Biochem. Biophys. 340, <br />       protein comparison regulation            1006/abio.1994.1326)                                          144&#226;&#8364;&#8220;153. (doi:10.1006/abbi.1997.9888) <br />       RpoE Cpx extracytoplasmic stress responses.     30.   Reed LJ, Muench H. 1938 simple method               38. Gunn JS, Ryan SS, Van Velkinburgh JC, Ernst RK, Miller <br />       J. Bacteriol. 186, 4199&#226;&#8364;&#8220; 4208. (doi:10.1128/JB.186.           estimating fifty percent endpoints.  J. Hygiene            SI. 2000 Genetic functional analysis PmrA- <br />       13.4199-4208.2004)                                            27, 493&#226;&#8364;&#8220; 497.                                                 PmrB-regulated locus necessary <span id='am-15' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-16' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span>lipopolysaccharide</span> <br /> 22.   Kabsch W. 2010 Xds. Acta Crystallogr. D Biol.           31.   Lavander M, Ericsson SK, Broms JE, Forsberg                 modification, antimicrobial <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> resistance, oral <br />       Crystallogr. 66, 125&#226;&#8364;&#8220;132. (doi:10.1107/                       2006 twin arginine translocation                virulence Salmonella enterica serovar typhimurium. <br />       S0907444909047337)                                            essential virulence Yersinia pseudotuberculosis.       <br /> </body></html>