<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Presence&#226;&#8364;&#8220;Absence Variation  thaliana Primarily <br /> Associated Genomic Signatures Consistent Relaxed Selective Constraints <br /> Stephen J. Bush,1 Atahualpa Castillo-Morales,1 Jaime M. Tovar-Corona,1 Lu Chen,1 Paula X. Kover,1 Araxi O. Urrutia*,1 <br /> 1 <br />  Department Biology Biochemistry, University Bath, Bath, United Kingdom  Present address: Human Genetics, Wellcome Trust Sanger Institute, Genome Campus, Hinxton, UK <br /> *Corresponding author: E-mail: urrutia@bath.ac.uk. <br /> Associate editor: Stephen Wright <br />  <br /> Abstract sequencing multiple genomes plant species revealed polymorphic gene exon loss. Genes <br /> associated disease resistance overrepresented showing structural variations, suggesting adaptive <br /> role gene exon presence&#226;&#8364;&#8220;absence variation (PAV). shed light possible functional relevance poly- <br /> morphic coding region loss mechanisms driving process, characterized genes lost entire exons coding regions 17 fully sequenced Arabidopsis thaliana accessions. significant <br /> enrichment genes associated certain functional categories observed, PAV events largely restricted genes signatures reduced essentiality: PAV genes tend newer additions genome, tissue specific, lowly <br /> expressed. addition, PAV genes located regions lower gene density higher transposable element density. <br /> Partial coding region PAV events associated marginal reduction gene expression level affected <br /> accession occurred genes higher levels alternative splicing Col-0 accession.  results <br /> suggest adaptive scenarios ruled  PAV events explained invoking  <br /> Key words: exon deletion, presence&#226;&#8364;&#8220;absence variation, genome evolution, transposable elements, adaptive <br /> evolution, Arabidopsis. <br />  <br /> Introduction                                                                        intraspecies variation genes, inclusive large <br />                                                                                     deletions (Gan et al. 2011) 775 genes acces- <br /> Intraspecies variation gene content represents impor- <br />                                                                                     sion deletions spanning 50% tant source heterogeneity genome species <br />                                                                                     coding region sequence (Gan et al. 2011). comparison potentially contributes organism&#226;&#8364;&#8482;s adaptability                                                                                     80 Arabidopsis genomes 9% total genes <br /> response external pressures (Feuk et al. 2006). Cataloguing <br />                                                                                      thaliana showed presence&#226;&#8364;&#8220;absence variation (PAV) <br />  <br />  <br />  <br />  <br />                                                                                                                                                         Article <br /> significant gains losses coding regions species allow deeper understanding                            averaging 444 absent genes accession (Tan et al. 2012). <br /> mechanisms underlying molecular evolution genomes                               Characterization coding region PAV shown certain assist identifying functional variation agronom-                      gene categories significantly enriched. instance, 52 ically elite varieties staple crops (Wang,  et al. 2013).                 154 <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>-binding site leucine-rich repeat (NBS-LRR) end, studies examined polymorphic                         R (resistance) genes deleted 1 <br /> partial gene loss plant species. instance,                     50 rice cultivars (Xu et al. 2012). Similar overrepresentation <br /> resequencing 50 rice genomes, 1,327 possible gene loss                        R genes  thaliana observed (Bakker <br /> events (2.4% total gene set) detected relative                          et al. 2006; Shen et al. 2006), soybean, genes Nipponbare reference accession (Xu et al. 2012).                             enriched structural variation likely involved <br /> Significant intraspecies variation gene content                         nucleotide binding biotic defense (McHale et al. 2012). reported maize (Swanson-Wagner et al. 2010), sor-                           Enrichment particular functional gene categories ghum (Zheng et al. 2011), soybean (McHale et al. 2012).                         genes affected structural polymorphism suggests Previous studies model plant Arabidopsis thaliana,                           structural polymorphisms functional role, allow- <br /> using resequencing microarrays Illumina sequencing                           ing accessions better adapted environmental <br /> synthesis reads, shown significant variations total                    conditions face. <br /> nuclear genome sequence naturally occurring strains                               hypothesis explicitly tested. (Clark et al. 2007; Ossowski et al. 2008). recent study                      significant polymorphic deletions adaptive, using 18 fully sequenced  thaliana genomes                             expect affected genes multiple signatures <br /> relative reference accession Col-0, 93.4% proteins                        selection. hand, structural <br />  <br /> &#195;&#376; Author 2013. Published Oxford University Press behalf Society Molecular Biology Evolution. Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http:// <br /> creativecommons.org/licenses nc/3.0/), permits non-commercial use, distribution, reproduction medium, <br /> provided original work properly cited. commercial use, contact journals.permissions@oup.com                 Open Access <br /> Mol. Biol. Evol. 31(1):59&#226;&#8364;&#8220;69 doi:10.1093/molbev/mst166 Advance Access publication September 25, 2013                                              59 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                MBE <br /> polymorphisms affect genes evolving relaxed          GOslim term. proportion poorly characterized genes constraints, adaptive significance           greater CDS-PAV genes, 60% having <br /> questioned.                                                       associated GOslim term biological process.    characterize genes affected PAV spanning            examining genes associated GOslim terms exons  thaliana, investigate genomic          E-PAV CDS-PAV genes significantly enriched <br /> features,  associated polymorphisms.        genes associated signal transduction nucleotide results provide insights likely functional           binding (fig. 1 supplementary fig. S2, Supplementary <br /> impact structural variation protein-coding genes.           Material online). Furthermore, E-PAV genes appear <br />                                                                   significantly enriched genes associated GOslim <br /> Results <br />                                                                   term  binding,&#226;&#8364;? includes proteins bind order characterize PAV  thaliana, examined          lipids, metal ions, <span id='am-2' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span>, cofactors (fig. 1). <br /> previously identified polymorphic deletions 17 fully           Significant overrepresentation functional categories sequenced Arabidopsis accessions transcriptome          PAV genes consistent previous assessment large <br /> data available (Gan et al. 2011)  Materials          coding region indels soybean genome (McHale et al. <br /> Methods). compiled set deletions spanned             2012) gene deletions  thaliana (Tan et al. <br /> entire exons 17 accessions relative Col-0        2012). observed classifying genes using reference genome. subset annotated deletions         broader set &#226;&#8364;&#339;GOslim&#226;&#8364;? terms (supplemen- <br /> experimentally validated (Gan et al. 2011). rule   tary fig. S3, Supplementary Material online). possibility wrongly identifying deletions differ-       classifying genes family, observe overrep- <br /> ences assemblies, exons confirmed missing         resentation members NBS-LRR family&#226;&#8364;&#8221;involved searching homology Col-0 exon           pathogen detection (DeYoung Innes 2006) accessions  Materials Methods).                     E-PAV genes (families &#226;&#8364;&#339;NBS-LRR active TNL,&#226;&#8364;? adjusted <br />    total 794 exons classified missing    P value = 8.57   10 35, &#226;&#8364;&#339;NBS-LRR active CNL,&#226;&#8364;? adjusted 17 accessions, corresponding 411 genes (~1.5%        P value = 4.63   10 5; fig. 1), consistent previous findings total gene set) including 81 genes coding      (Shen et al. 2006). Furthermore, examining 3,753 <br /> region completely absent accession (sup-      Pfam <span id='am-21' about='oboInOwl:id' typeof='owl:Thing'>ID</span> gene associations (supplementary figs. S4 S5, <br /> plementary table S1, Supplementary Material online). Exon         Supplementary Material online), observe overrepresen- <br /> losses uniformly distributed gene:         tation members NB-ARC (APAF-1, R proteins, missing exon sequences near              CED-4) LRR domain containing families (note &#226;&#8364;&#339;NBS- <br /> ends gene (supplementary fig. S1, Supplementary           LRR&#226;&#8364;? refers composite NBS LRR domains Material online).                                                 NBS domain known &#226;&#8364;&#339;NB-ARC&#226;&#8364;? [McHale <br />    Overall, ~0.3% genes accession          et al. 2006]). enrichment particular gene family missing exon, representing 10 50 kb         observed CDS-PAV genes (data shown). missing sequence accession (supplementary table S2,            significant enrichments gene functional Supplementary Material online). total 200 genes          domain annotations line previous findings exon loss affecting accession, consistent           Arabidopsis (Tan et al. 2012) plant species previous study reporting &#226;&#8364;&#339;common history&#226;&#8364;?             (Swanson-Wagner et al. 2010; Zheng et al. 2011; McHale <br /> deletion events  thaliana (Santuari et al. 2010).            et al. 2012) proposed reflect adaptive <br />    partial deletions spanning exons           role large polymorphic deletions. distinct functional implications compared coding region deletions, 330 genes partial coding        Genes Affected PAV Signatures Consistent <br /> region loss spanning exon       Relaxed Selective Constraints <br /> accession (exon PAV [E-PAV]) 81 genes           determine PAV genes generally associated <br /> coding region polymorphic deletions affecting        fast evolving proteins potentially positive selection, <br /> accession  coding DNA sequence PAV [CDS-PAV])                examined rates nonsynonymous synonymous examined separately.                                         changes gene (dN/dS). Using randomization test, <br />                                                                   E-PAV genes significantly higher dN/dS <br /> Genes Involved Signal Transduction                    ratio compared genes exons present, Nucleotide Protein Binding Overrepresented                genes dN/dS ratio 1 (fig. 2 PAV Genes                                                   supplementary tables S1 S3, Supplementary Material order characterize PAV genes, assessed     online). CDS-PAV genes nonsignificant increase dN/ genes overrepresented particular gene classes       dS compared intact genes  affected dele- gene ontology   categories.  used          tions spanning exon accession; fig. 2 classification schemes:  &#226;&#8364;? condensed set terms         supplementary table S3, Supplementary Material online). <br /> (GOslim), Pfam protein domain database family            examine selective pressures associated classification scheme (Gan et al. 2011)  Materials     PAV genes, examined nucleotide diversity. considered <br /> Methods). 330 E-PAV genes,             nucleotide diversity replacement sites silent sites <br /> poorly characterized 50% having associated        (defined noncoding sites synonymous sites  <br /> 60 <br />  Presence&#226;&#8364;&#8220;Absence Variation  thaliana . doi:10.1093/molbev/mst166                                                                  MBE <br />                                                                        (b) <br />  <br />  <br />  <br />  <br />      (c)                                                                 (d) <br />  <br />  <br />  <br />  <br /> FIG. 1. Distribution E-PAV genes (n = 330)   exons missing accession GOslim categories molecular function  , biological process (b) cellular component (c), family (d). expected observed number E-PAV genes category represented bar. significant enrichment (P   0.05) observed expected E-PAV genes particular category, asterisk shown bars. categories E-PAV gene shown. <br />  <br />  <br /> protein-coding regions) gene, according Gan et al.                 genes) D &gt; 2 (randomization test P = 0.93 P = 1 E- <br /> (2011). PAV genes associated higher                      CDS-PAV genes, respectively). demographic charac- <br /> nucleotide diversity silent replacement sites (sup-                teristics Arabidopsis population result shift plementary table S3, Supplementary Material online).                           average Tajima&#226;&#8364;&#8482;s D general pool genes,    higher dN/dS nucleotide diversity                          possible hard thresholds informative. <br /> suggestive relaxed selective constraints, pattern                    intact genes Arabidopsis consistent scenario positive balancing                   average Tajima&#226;&#8364;&#8482;s D estimate shifted negative values. <br /> selection. differentiate possible scenarios,                   PAV genes fall short hard threshold + 2 <br /> Tajima&#226;&#8364;&#8482;s D calculated gene  Materials                     higher D estimate general pool Methods). threshold &#194;&#177;2 considered point                        genes, suggestive balancing selection.  E-PAV D significantly departs null expectation                     genes significant differences Tajima&#226;&#8364;&#8482;s D esti- neutral evolution given gene. 330 E-PAV                      mates compared intact genes CDS-PAV genes  <br /> genes E-PAV, 24 D &lt;  2 2 D &gt; 2                         fact, significantly lower estimate D (fig. 2 supple- <br /> (AT1G12180, D = 2.17, AT5G35460, D = 2.05,                         mentary tables S1 S3, Supplementary Material online). functionally uncharacterized). CDS-PAV                         possible PAV genes higher range genes, seven D &lt;  2 D &gt; 2.                             D values compared intact genes, hiding higher propor- <br /> Compared set intact genes, signifi-                   tion genes positive balancing selection differences proportion PAV genes                    reflected overall changes mean. D &lt; 2 (randomization test P = 1 E- CDS-PAV                        test  compared distributions Tajima&#226;&#8364;&#8482;s D <br />  <br />                                                                                                                                                  61 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                                                  MBE <br />  <br />                                                                                                                                   20 <br />  <br />                                                                          4 <br />  <br />                   0.4                                                                                                             15 <br />  <br />  <br />  <br />  <br />                                                                                                               Exp. level (RPKM) <br />                                                                          3 <br />      dN/dS <br />  <br />  <br />  <br />  <br />                                                        Age <br />                                                                                                                                   10 <br />                                                                          2 <br />                   0.2 <br />  <br />                                                                                                                                    5 <br />                                                                          1 <br />  <br />  <br />  <br />                   0.0                                                    0                                                         0 <br />  <br />                           Intact     E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                           Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                                     Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV <br />  <br />  <br />  <br />                    0.00 <br />                                                                                                                                   0.5 <br />  <br />  <br />                                                                          9 <br />                   &#226;&#710;&#8217;0.25                                                                                                           0.4 <br />                                                         paralogs <br />      Tajima&apos;s D <br />  <br />  <br />  <br />  <br />                                                                                                                                   0.3 <br />                                                                          6 <br />  <br />  <br />                                                                                                               Tau <br />                   &#226;&#710;&#8217;0.50 <br />  <br />  <br />                                                                                                                                   0.2 <br />  <br />                   &#226;&#710;&#8217;0.75                                                  3 <br />                                                                                                                                   0.1 <br />  <br />  <br />                   &#226;&#710;&#8217;1.00 <br />                                                                          0                                                        0.0 <br />  <br />                             Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                           Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV                                     Intact   E&#226;&#710;&#8217;PAV CDS&#226;&#710;&#8217;PAV <br />  <br /> FIG. 2. Genetic features associated intact (having exons P variation), E-PAV (having   exons missing accession), CDS-PAV (having entire CDS missing accession) genes. left right,  dN/dS, age, expression <br /> level, Tajima&#226;&#8364;&#8482;s D, number paralogs, tau. supplementary table S3, Supplementary Material online, values means statistical analysis. <br />  <br />  <br /> estimates sets genes (intact, E-PAV, CDS-                           (3.32%) observe CDS-PAV genes <br /> PAV).  did observe evidence                                   (8.82%). finding suggest significant propor- <br /> creased dispersion D PAV genes (supplementary                                  tion CDS-PAV genes undergoing stronger puri- <br /> fig. S6, Supplementary Material online). examine                             fying positive selection relative intact genes.  possibility, examined proportion PAV genes                               results suggest rule fifth 95th percentile &#226;&#8364;&#339;intact&#226;&#8364;?                           effect balancing selection acting individual PAV <br /> distribution (D =  2.05 1.39, respectively). signifi-                          genes general trend balancing selection PAV genes <br /> cantly higher proportion E-PAV CDS-PAV genes                                  does readily apply. excess negative D values compared intact set positive end                               PAV genes coupled higher levels nucleotide di- distribution, infer existence detectable                            versity significant increases dN/dS ratios  <br /> subset PAV genes undergoing balancing                                    sistent scenario weaker purifying selection selection.  pattern clearly observed&#226;&#8364;&#8221;                             explained positive selection. 2.33% E-PAV CDS-PAV genes exceed                                         examined number parameters threshold value. opposite end distribution,                            previously associated studies gene essentiality <br /> observe overrepresentation proportion E-PAV                                explore functional importance PAV genes, <br /> genes estimates D lower threshold                                 including gene&#226;&#8364;&#8482;s age (Chen, Trachana, et al. 2012)  <br /> 62 <br />  Presence&#226;&#8364;&#8220;Absence Variation  thaliana . doi:10.1093/molbev/mst166                                                 MBE <br /> number paralogs (Hanada et al. 2009; Makino et al.       2003).  used reference accession (Col-0) <br /> 2009), weaker associations expression level     calculated TE density gene intergenic <br /> (Cherry 2010) tissue specificity (Wolf et al. 2006).           sequence 1- 100-kb windows centered gene&#226;&#8364;&#8482;s <br />    Compared newer genes, older genes likely       midpoint counting number bases TE essential (Chen, Trachana, et al. 2012). using        annotations  Materials Methods). E-PAV genes phylogenetic relationships plant genomes create            approximately 2-fold increase proxy gene age, observed 330 genes affected        bases annotated TE compared genes E-PAV likely newer additions              intact accessions (e.g., TE sequence accounts <br /> genome (fig. 2 supplementary table S3, Supplementary           ~30% nongenic sequence 10-kb window <br /> Material online). possible E-PAV genes      surrounding E-PAV gene; fig. 3 supplementary table <br /> greater number paralogous genes compensate           S4, Supplementary Material online). Significant enrichment loss function. Consistent        specific TE superfamilies observed, notably, DNA genes missing exons higher number para-         transposons LTR retrotransposons (supplementary <br /> logs compared genes exons present (fig. 2      table S4, Supplementary Material online). supplementary table S3, Supplementary Material online).            addition, genes missing exons  opposite result observed analyzing            average, shorter distance gene boundary <br /> CDS-PAV genes average 4.2 paralogs           nearest TE genes exons present <br /> compared genes exon losses (fig. 2 supple-        (2.5 kb compared 5.7 kb; randomization test, P = 9.9e 5; <br /> mentary table S3, Supplementary Material online), suggesting       supplementary table S3, Supplementary Material online. function essential. assessed expres-     calculating minimum distance nearest TE, <br /> sion patterns genes affected exon presence&#226;&#8364;&#8220;absence,          classified superfamily, E-PAV genes significantly closer broadly highly expressed genes typically        TE type: rolling circle TEs, DNA transposons, LTR <br /> sociated higher levels selection (Yang 2009). Using      retrotransposons, long interspersed elements (LINEs), randomization test, genes exon losses        short interspersed elements (SINEs) (supplementary table accessions, compared intact genes,           S3, Supplementary Material online). Similar findings lower expression levels higher tissue specificity (sup-    obtained analyzing TE content surrounding <br /> plementary table S3, Supplementary Material online). ad-        regions CDS-PAV genes (supplementary table S3, <br /> dition, observed exons missing        Supplementary Material online). <br /> accession  average, shorter exons present           Certain TE sequence motifs associated accessions (170 bp vs. 284 bp, randomization test P = 9.9e 5;      recombination hotspots drive exon loss supplementary table S3, Supplementary Material online).            promoting ectopic recombination events (Oliver Greene  exons affected polymorphic deletions          2009; Horton et al. 2012). explore genes affected shorter average compared nondeleted exons,             PAV local enrichment hotspot motifs, <br /> E-PAV genes longer unaffected genes (2,360 bp             examined density motifs compared 2,142 bp, respectively; randomization test           genes  Materials Methods).  P = 0.008; supplementary table S3, Supplementary Material          observed significant differences hotspot motif occu- <br /> online). contrast, CDS-PAV genes polymorphic              pancy nongenic regions windows surrounding E- <br /> deletions encompass gene&#226;&#8364;&#8482;s entire coding region           PAV genes compared intact genes  window sizes shorter unaffected genes (640 bp compared         1 100 kb centered gene&#226;&#8364;&#8482;s midpoint; supplementary 2,142 bp, randomization test P = 9.9e 5; supplementary        table S4, Supplementary Material online).  sig- <br /> table S3, Supplementary Material online).                          nificant enrichment hotspot motif occupancy    Overall, findings certain func-        observed genic sequence windows centered tional categories overrepresented genes exon        E-PAV genes compared centered intact genes <br /> loss, generally significant coding region loss prevalent   (fig. 3 supplementary table S4, Supplementary Material novel, lowly expressed poorly functionally               online). comparing CDS-PAV genes intact set, <br /> characterized genes. genes evolved              observed consistent pattern higher hotspot motif recently Arabidopsis genome likely          density genic regions marginally higher reduced selective constraint.                             proportion hotspot motifs nongenic regions                                                                    surround  windows 3 kb size (P &lt; 0.01; sup- <br /> PAV Genes Located Genomic Regions                      plementary table S5, Supplementary Material online). Gene-Poor Transposable Element-Rich                            Taken  results PAV genes characterizing genomic context genes affected          located gene-poor TE-rich regions genome, PAV, genes exon coding         supporting hypothesis PAV associated <br /> region loss separated longer intergenic distances (fig. 3   relaxed selective constraints. Enrichments sequence supplementary table S3, Supplementary Material online).        motifs previously associated recombination hotspots <br /> Transposable element density PAV genes             PAV genes suggest exon <br /> assessed gene-poor areas associated            deletion events resulted recombination <br /> higher transposable element (TE) density (Wright et al.            events involving recombination hotspot motifs. <br />  <br />                                                                                                                                63 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                                              MBE <br />                                                                                              Exon Loss Associated Marginal Reduction                                                                                              Expression Level <br />                                     6000 <br />                                                                                              aforementioned results suggest E-PAV associated <br />                                                                                              reduced selective constraints. assess exon <br />      Intergenic distance (bp) <br />  <br />  <br />  <br />                                                                                              loss likely resulted reduced functionality                                                                                              genes affected, compared expression levels genes                                     4000                                                     missing exons accessions. exon loss <br />                                                                                              causes follows diminished functionality previous <br />                                                                                              mutations, expect expression significantly <br />                                                                                              reduced accessions affected E-PAV. Using <br />                                     2000                                                     RNAseq transcription profiles Arabidopsis accession <br />                                                                                              (Gan et al. 2011), compared expression patterns                                                                                              individual genes accessions affected exon deletions                                                                                              accessions gene remained intact.  <br />                                                                                              transformed expression data accession scores <br />                                                             0 <br />                                                                                              (Cheadle et al. 2003). looked genes <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   exon loss occurred single accession (210 <br />                                                             40                               genes). gene, took 1) expression level                                                                                              gene affected accession 2) mean expres- <br />                                                                                              sion level gene 17 unaffected accessions <br />                                                                                               16 study plus reference genome, Col-0). <br />                                                             30 <br />                                                                                              half genes examined expression <br />                                 % TE bases (density) <br />  <br />  <br />  <br />  <br />                                                                                              level mean 37% expression level equal  <br />                                                                                               average, expression levels affected acces- <br />                                                             20 <br />                                                                                              sion departed little mean expression unaffected <br />                                                                                              accessions (0.15 standard deviations). 27 genes (13%                                                                                              cases), expression level gene affected exon deletion <br />                                                                                              higher mean expression unaffected ac- <br />                                                             10                               cessions 14 cases showing statistically significant dif- <br />                                                                                              ference (fig. 4 supplementary table S6, Supplementary <br />                                                                                              Material online). 27 genes generally poorly charac- <br />                                                                                              terized 12 having functional category annotations. <br />                                                              0 <br />                                                                                              genes affected exon deletions low expression <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   levels begin  exceptions notable, <br />                                                             2.5                              rotamase CYP4 (AT3G62030; involved variety                                                                                              cellular functions related metabolism response  <br />                                                                                              eral types stress), average expression level                                                             2.0                              unaffected accessions 400 rpkm, 1%                                                                                              genes detectable expression Col-0. <br />                                 % hotspot bases (density) <br />  <br />  <br />  <br />  <br />                                                                                                 possible moderate effect exon loss gene <br />                                                             1.5                              expression levels explained overrepresentation                                                                                              alternatively spliced exons set missing exons. <br />                                                                                              allow production viable protein prod- <br />                                                             1.0                              ucts absence. order test  quantified <br />                                                                                              alternative splicing 15,540 Arabidopsis genes, including <br />                                                                                              103 330 E-PAV associated genes using &#226;&#8364;&#339;comparable <br />                                                             0.5                              alternative splicing index&#226;&#8364;?  Materials Methods),                                                                                              corrects distorting effect variation transcript <br />                                                                                              coverage genes (reviewed Chen, Tovar-Corona, <br />                                                             0.0                              et al. [2012]). E-PAV genes significantly <br />                                                                   Intact   E&#226;&#710;&#8217;PAV   CDS&#226;&#710;&#8217;PAV   higher number alternative splicing events compared FIG. 3. Genomic context intact (having exons P variation), E-PAV (having   exons missing                     FIG. 3. Continued accession), CDS-PAV (having entire CDS missing                       gene&#226;&#8364;&#8482;s midpoint, percentage recombinogenic motifs accession) genes. Averaged values genes set                    genic sequence 1-kb window centered gene&#226;&#8364;&#8482;s midpoint. given   intergenic distance, percentage                    supplementary tables 3 4 (Supplementary Material online) TE bases nongenic sequence 10-kb window centered                         values specific TE families window sizes. <br />  <br />  <br /> 64 <br />  Presence&#226;&#8364;&#8220;Absence Variation  thaliana . doi:10.1093/molbev/mst166                                                              MBE <br />                                                                             random expectations Arabidopsis (Tan et al. 2012), <br />                                                                             greatest significant enrichment PAV genes                                                                             reported NB-ARC domains. <br />                                                                                 functional gene family enrichments                                                                             suggestive adaptive role PAV events aiding spe- <br />                                                                             cific ecotypes adapting local environment.                                                                             results&#226;&#8364;&#8221;showing genes associated  example, <br />             &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? <br />                 &#226;&#8212;?&#226;&#8212;? <br />                  &#226;&#8212;?    &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? <br />                   &#226;&#8212;?&#226;&#8212;?&#226;&#8212;?&#226;&#8212;?&#226;&#8212;?                   &#226;&#8212;? <br />                                          &#226;&#8212;?&#226;&#8212;?   &#226;&#8212;?    &#226;&#8212;?&#226;&#8212;? <br />                                                   &#226;&#8212;?&#226;&#8212;?    &#226;&#8212;?   &#226;&#8212;?&#226;&#8212;? &#226;&#8212;?&#226;&#8212;? &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? &#226;&#8212;? <br />                                                             &#226;&#8212;? <br />                                                                             resistance likely affected PAV                                                                              glance, consistent hypothesis. addition,                                                                             able confirm previous report CDS-PAV                                                                             members R gene family single-exon gene <br />                                                                             AT5G05400 multiexon genes AT5G18350                                                                             AT5G49140 (Shen et al. 2006) family known sig- <br />                                                                             natures positive selection  thaliana (Mondragon- <br />                                                                             Palomino et al. 2002).  comprehensive analysis                                                                             evidence selection does support general <br />     &#226;&#710;&#8217;4               &#226;&#710;&#8217;2            0                2                   4 <br />                                                                             interpretation. <br />                                                                                 dN/dS ratios widely used estimates            score gene 1+ missing exons 1 ecotype <br />                                                                             selective pressure acting protein coding genes dN/dS <br /> FIG. 4. Distribution scores standardized transcript abundance      &gt;&gt; 1 indicative definitive signature positive <br /> data affected accession. Data 210 genes      selection (Hurst 2002).  average, missing exons 17  thaliana accessions (relative    higher number substitutions E-PAV genes compared Col-0).                                                                  intact genes, clear signature adaptation <br />                                                                             suggest comparatively relaxed negative, intact genes (3.35 1.13, respectively; randomization test               stronger positive, selection. <br /> P = 0.046).                                                                     PAV genes significantly higher <br />    Overall, findings suggest exon losses             nucleotide diversity silent replacement sites. marginal effect expression profile genes                   observations suggestive weaker purifying selection; <br /> accessions affected. higher levels alternative splicing               expected PAV genes genes affected exon loss raises possibility             higher balancing selection.  evidence <br /> significant proportion lost exons normally alternatively             suggest diversity resistance-associated <br /> spliced, reducing selection pressure exons                 genes maintained balancing selection (Van der Hoorn functional protein product produced                     et al. 2002), overrepresented PAV <br /> absence                                                              genes. Balancing selection proposed stably main- <br />                                                                             tain intact gene absent allele (Tan et al. <br /> Discussion                                                                  2012). <br /> Intraspecies structural variations genes proposed                   balancing selection parsimonious explana- play important role adaptation particular                   tion PAV genes associated higher nucleotide <br /> populations variation environmental conditions (Feuk                  diversity? classic scenario transspecies polymorphism, <br /> et al. 2006). characterized presence&#226;&#8364;&#8220;absence                   associated balancing selection, assessed <br /> coding sequence variation 17 fully sequenced  thaliana                 given limited sequence variation data available genomes, relative reference accession Col-0, affecting                lyrata,  thaliana&#226;&#8364;&#8482;s closest sequenced relative. possible <br /> 411 genes including 81 instances coding region                  &#226;&#8364;&#339;gene/exon present&#226;&#8364;? &#226;&#8364;&#339;gene/exon absent&#226;&#8364;? <br /> letions. significant enrichment genes associ-                 alleles selection maintained different <br /> ated terms protein nucleotide binding                 thaliana populations, allowing better adapt signal transduction. gene family Pfam anno-                local environment. consistent tation enrichment analysis revealed significant enrichments                 increase nucleotide diversity, scenario gene members disease resistance associated NBS-                 distinguished alternative neutral models. Conditional <br /> LRR gene families. Significant deviations random expec-                neutrality PAV loci, functional gene ceased <br /> tations observed previous studies PAV genes                 adaptive environments, plants, similar overrepresentation resistance-asso-              ruled (e.g., case resistance genes cor- <br /> ciated gene families PAV genes. instance, sor-                 responding pathogen absent [Gos Wright 2008]). ghum (Zheng et al. 2011), PAV genes enriched                    case, absent allele selective advantage <br /> Pfam categories, including NB-ARC domain-containing                     point result relaxed constraints asso- <br /> family. soybean (McHale et al. 2012), PAV-affected genes                 ciated PAV genes Arabidopsis populations. enriched members NB-                   model generalized relaxed constraints affecting <br /> ARC family, category &#226;&#8364;&#339;defense                       PAV loci lead increased nucleotide diversity <br /> sponse.&#226;&#8364;? CDS-PAV genes shown deviate                      slight increases dN/dS. <br />  <br />                                                                                                                                             65 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                   MBE <br />     Tajima&#226;&#8364;&#8482;s D, comparison estimators    pop-       tend accumulate near genes lower selective <br /> ulation mutation rate 4Nem) number segregating               pressures located regions efficient purging TE <br /> sites average number pairwise differences        sequence.  set E-PAV genes, higher <br /> sequences (Tajima 1989)&#226;&#8364;&#8221;offers reliable estimate          density TEs vicinity. addition, selective pressures acting gene incorporates infor-      genes undergoing PAV increased proportion mation distribution segregating alleles           motifs associated recombination hotspots species. allows accurate estimations degree        sequence. findings consistent PAV events direction departure sequence evolution             associated genes located genomic regions evolving <br /> neutral expectation  nonselectionist interpretations       reduced selective constraints.  higher TE  D possible, recent population expansion      tent hotspot motifs consistent suggestion <br /> bottlenecking negative positive D, respectively)            unequal recombination homologs (Tajima 1989). Tajima&#226;&#8364;&#8482;s D values provide evidence            major mechanism generating P polymorphisms (Tan E-PAV CDS-PAV genes balancing             et al. 2012).  noted recombino- <br /> selection. Taking dN/dS, nucleotide diversity, D estimates      genic motif necessary sufficient recombina-  PAV genes appear evolving                tion event occur (Johnston Cutler 2012),  <br /> relaxed constraints.                                                connection,  PAV remains speculative. <br />     signature relaxed selection associated PAV genes         features considered suggest al- combined variety features associ-       individual deletions adaptive <br /> ated lower gene essentiality. PAV genes          value, overall coding region loss disproportionally affects lower expression levels higher tissue specificity;    genes reduced selective pressures. features associated higher rates         results reconciled enrichment certain gene families <br /> substitutions reduced gene essentiality (Wolf et al. 2006;      functional terms? enrichment specific func- <br /> Cherry 2010). Older genes considered essen-          tional categories gene families PAV genes (fig. 1) <br /> tial (Chen, Trachana et al. 2012) associated       leads implication adaptive pressures favoring PAV humans, flies Aspergillus) higher expression level       genes related specific biological processes (Tan et al. 2012). stronger purifying selection (Wolf et al. 2009).        shown, PAV genes associated PAV genes  average, newer additions              variety features suggestive lower selective constraints. <br /> genome exons affected PAV              argue enrichment certain categories orthologous exon  lyrata (663/794). note            gene families genes associated particular E-PAV CDS-PAV genes enriched reverse tran-          genomic feature does  itself, allow draw conclu- <br /> scriptase domains (supplementary figs. S4 S5,                   sions adaptive processes genes Supplementary Material online) E-PAV genes trans-           undergoing. Consistent  intact genes <br /> posase domains (supplementary fig. S4, Supplementary                associated gene categories PAV genes Material online), suggesting exonization TEs origin       enriched signatures reduced selection PAV-affected exons.                                         (supplementary table S7, Supplementary Material online). <br />     addition, fact gene expression marginally   notable sets genes involved  example, <br /> reduced accessions affected exon deletion events sug-         signal transduction, nucleic acid binding, NBS-LRR <br /> gests lost exons limited impact        family&#226;&#8364;&#8221;categories enriched PAV genes (fig. 1). gene functionality. possibly explained cases     instance, compare set E-PAV genes set alternative splicing, associated     genes exons present set NBS-LRR genes increased frequency exon loss humans, mice,            set genes belonging families, rats&#226;&#8364;&#8221;alternatively spliced forms likely conserved    E-PAV NBS-LRR genes comparatively newer additions species constitutive exons (Modrek Lee             genome, higher dN/dS ratio, higher number 2003).  thaliana, genes E-PAV           alternative splicing events, higher number paralogs, weaker purifying selection greater number          higher proportion SNPs, closer TEs (sup- alternative splice events compared intact genes.       plementary table S7, Supplementary Material online). observation suggests alternatively spliced exons likely    note proportion polymorphic sites higher reduced selective constraints compared         PAV genes genes functional category. stitutive exons, exon deletions      demonstrate PAV genes bias comparison  detrimental effect loss constitutive exon.    example, set NBS-LRR genes set genes best knowledge, time exon loss    belonging families, repeat analysis restricted <br /> events associated elevated alternative splicing      intact genes observe result (supplemen- <br /> levels species species.                tary table S7, Supplementary Material online). <br />     genomic context genes linked          fact observed fewer PAV genes previ- <br /> patterns sequence evolution features associated         ous study examining 80 fully sequenced Arabidopsis genomes <br /> gene essentiality. recent study  thaliana correlated     (n = 2,741; Tan et al. 2012) likely differences meth- presence TEs adjacent genes sequence variation       odology.  analysis uses 17 genomes assembled using gene (Wang, Weigel, et al. 2013), suggesting TEs        combination read reference genome (Col-0) alignment <br />  <br /> 66 <br />  Presence&#226;&#8364;&#8220;Absence Variation  thaliana . doi:10.1093/molbev/mst166                                                     MBE novo approaches, importantly tran-            Zu-0) obtained Gan et al. (2011). did scriptome data available (Gan et al. 2011),       use data Po-0 unusually high frequency 80 accessions reported Cao et al. (2011). Second,        heterozygosity high similarity Oy-0 (Gan et al. 2011). <br /> use conservative methodology defining significant       genome fully sequenced assembled, using <br /> deletions Tan et al. (2012) define PAV genes using      combination novo assembly read mapping referred &#226;&#8364;&#339;broad <span id='am-1' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>&#226;&#8364;?:     reference accession, Col-0. <br /> particular locus genomes compared &#226;&#8364;? allows gene called PAV gene        Detecting Missing Exons Relative Col-0 copy exists different locus. minimize inclusion   analysis, selected set deletions spanning rearrangement events deletions, Tan et al. (2012) exam-      exon accession relative Col-0 <br /> ined predicted PAV genes using BlastN refer-       reference genome wider set deletion events  <br /> ence accession, excluding &#226;&#8364;&#339;absent&#226;&#8364;? category           scribed Gan et al. (2011). Exons absent Col-0 refer- <br /> gene counterpart matches &gt;50% length.           ence genome present accessions included definition PAV restrictive deemed        analysis. Confirmation deletions described exon gene deleted genome alignments showed          original authors analyzed deletion breakpoints (Gan deletion spanned exon gene      et al. 2011). data set, deletion breakpoints esti- single identifiable base remaining. Finally, Tan    mated ~30 bp, left right consensus se- <br /> et al. (2012) study used genomes assembled according        quences established growing inward TAIR8 annotated positions, data assembled          estimates using read-mapping information. according TAIR10. small risk,  having   deletion, ends overlap. Gan et al. (2011) <br /> incorporated obsolete gene models findings.         confirmed alignments left right consensus <br /> Regardless methodological differences result-       sequences, excluding errors sequencing misassem- <br /> ing variation sample size, worth noting          bly. confirmed presence absence results contradiction previous studies      individual exon 17 accessions relative Col-0 <br /> examining PAV Arabidopsis species,            genome annotation using BlastN default parameters similar deviations random expectations         (Altschul et al. 1990). Sequence alignments obtained <br /> functional annotations genes. analysis sequence          using best hit homolog Smith&#226;&#8364;&#8220;Waterman algo- <br /> evolution genic features PAV genes             rithm (fasta35 parameters  (Pearson 2000). rule possibilities conditional neutrality PAV        confirmed exon missing 1) alignment loci balancing selection acting PAV              2) nucleotide positions genes, allowing adaptation environmental conditions         Col-0 exon mapped nucleotide accession. specific ecotypes. Instead, findings presented PAV events explained nonadaptive interpre-        Functional Category Enrichment Analysis <br /> tation genes reduced constraints                                                                    gene classification schemes obtained. GOslim terms <br /> susceptible spread allele variants containing signifi- <br />                                                                    obtained TAIR (ftp://ftp.arabidopsis.org/home/tair/ deletions. <br />                                                                    Ontologies/Gene_Ontology/ATH_GO_GOSLIM.txt, dated 9 <br />     summary, results suggest significant <br />                                                                    July 2013  accessed October 8, 2013]), excluding terms <br /> enrichment functional categories PAV genes                                                                    unsupported experimental computational analysis, <br /> observed, exon loss events observed newer, poorly <br />                                                                     evidence codes ND, NR, NAS. term annota- <br /> functionally characterized genes associated signatures <br />                                                                    tions obtained Ensembl BioMart (17 July 2013) <br /> linked essential genes evolving lower purifying <br />                                                                    (Smedley et al. 2009). &#226;&#8364;&#339;Pfam&#226;&#8364;? terms obtained balancing selection. reduce potential func- <br />                                                                    Pfam v27.0 (17 July 2013) (Punta et al. 2012). addition, <br /> tional relevance structural variations genes. <br />                                                                    7,119 genes classified 49 distinct families Gan conclude adaptive model PAV <br />                                                                    et al. (2011). Statistical significance enrichment ruled  observed functional enrichments <br />                                                                    GOslim, terms, Pfam class family membership PAV genes increased nucleotide diversity                                                                    E-PAV CDS-PAV-affected genes assessed interpreted invoking selection. <br />                                                                    using Monte Carlo random sampling (1000 randomizations), <br />                                                                    P value enrichment category obtained <br /> Materials Methods                                              using test. significance individual categories Genome Sequence Annotations                                    corrected multiple testing Benjamini&#226;&#8364;&#8220;Hochberg <br /> Exon coordinates  thaliana strain Col-0 obtained        procedure. Arabidopsis Information Resource (TAIR) (ftp://ftp. <br /> arabidopsis.org/home/tair/Genes/TAIR10_genome_release/             Sequence Evolution Analysis <br /> TAIR10_gff3/TAIR10_GFF3_genes.gff, dated 20 March 2012             approximate selective constraint gene, calculated <br />  accessed October 8, 2013]). genomes 17                dN/dS. gene, obtained local alignment  thaliana accessions (Bur-0, 0, Ct-1, Edi-0, Hi-0, Kn-0,     Col-0 CDS  lyrata ortholog, using Ler-0, Mt-0, 0, Oy-0, Rsch-4, Sf-2, Tsu-0, Wil-2, Ws-0, Wu-0,   Smith&#226;&#8364;&#8220;Waterman algorithm (fasta35 parameters  <br />  <br />                                                                                                                                    67 <br />  Bush et al. . doi:10.1093/molbev/mst166                                                                                        MBE <br /> (Pearson 2000). dN/dS calculated using Yang              excluded analysis. EST alignments used <br /> Nielson model, implemented yn00 package phy-            create exon template. Alternative splicing events logenetic analysis maximum likelihood (PAML) (Yang                gene identified comparing alignment coordinates 2009). Using substitution estimates,  SNP data          individual EST exon annotations. low EST cov- (Gan et al. 2011), estimated Tajima&#226;&#8364;&#8482;s D (Tajima         erage increase number falsely positive claims 1989) gene. Nucleotide diversity calculated according      exon constitutive, spliced, excluded genes <br /> Gan et al. (2011).                                                   10 fewer ESTs. ESTs assigned genes using gene <br />                                                                      annotation coordinates. comparable alternative splicing <br /> Paralog Number Gene Age Annotations                              index avoids transcript coverage biases obtained <br /> Ortholog paralog data obtained BioMart                 using transcript normalization method described (Vilella et al. 2009). proxy gene age established          Kim et al. (2007). Briefly, gene 100 random samples <br /> using taxonomic classifications, based phylostrati-           10 ESTs selected. Finally, number alternative <br /> graphic method (Domazet-Los&#203;&#8225;o et al. 2007). candidate        splicing events calculated random sample  ortholog identified  thaliana gene 15       detailed earlier), overall average calculated gene. <br /> plant algal species minimum identity 30%, gene considered old &#226;&#8364;&#339;broadest&#226;&#8364;? taxonomic         Randomization Test <br /> category held common  supplementary table S8,                 randomization test used obtain numerical P values <br /> Supplementary Material online). allowed make              assess statistical significance variation use ortholog data despite divergence times relative            characteristics PAV-affected genes compared intact  thaliana known closest relatives ~5         genes. brief, contrasted genomic feature parameters <br /> million years  lyrata (Kuittinen et al. 2004) 20 million   E-PAV (n = 330) CDS-PAV genes (n = 81) distri- <br /> years Brassica rapa (Yang et al. 1999).                          bution means genomic feature s = 10,000 <br />                                                                      randomly generated subsets equal number genes <br /> Gene Expression <br />                                                                      drawn complete gene set. numerical P value <br /> Expression specificity calculated tissue specificity        calculated follows: let q number times index (tau) (Yanai et al. 2005), using massively parallel        mean value PAV set exceeded mean value signature sequencing (MPSS) database (Brenner et al. 2000;           randomly generated subset. Letting r = s   q, P value <br /> Meyers et al. 2004; Nakano et al. 2006). Expression levels      test r + 1/s + 1. <br /> calculated using RNAseq transcript abundance data, abso- <br /> lute read values corrected sequence length acces-         Supplementary Material <br /> sion (known rpkm values: gene, number reads            Supplementary tables S1&#226;&#8364;&#8220;S8 figures S1&#226;&#8364;&#8220;S5 available kilobase million mapped reads) (Gan et al. 2011).            Molecular Biology Evolution online (http://www.mbe. <br />                                                                      oxfordjournals.org/). <br /> TE Hotspot Motif Density <br /> TE coordinates  thaliana strain Col-0 obtained            Acknowledgments TAIR (file &#226;&#8364;&#339;TAIR10_Transposable_Elements,&#226;&#8364;? dated 20             authors acknowledge valuable comments March 2012). analyses, identified instance       anonymous reviewers. work supported 25 hotspot-associated motifs  5&#226;&#8364;&#8220;9 bp) described            University Bath fee studentship S.J.B., CONACyT schol- <br /> Horton et al. (2012) Col-0 reference genome. TE           arships C.M. J.M.T.C., UK&#226;&#8364;&#8220;China scholarship hotspot motif density gene calculated           excellence University Bath research studentship L.C., <br /> proportion base pairs occupied TE hotspot motif         BBSRC grant (grant number BB/F022697/1) P.X.K., windows size 1 100 kb centered nucleotide        Royal Society Dorothy Hodgkin Research Fellowship gene&#226;&#8364;&#8482;s midpoint. Windows consist coding           (DH071902), Royal Society research grant (grant number <br /> noncoding sequence region length (window size)/          RG0870644), Royal Society research grant fellows <br /> 2  downstream midpoint base. TE               (grant number RG080272) U.O. <br /> hotspot motif density calculated number TE motif bases, respectively, relative number intergenic      References genic bases contained window,          Altschul SF, Gish W, Miller W, Myers EW, Lipman DJ. 1990. Basic local <br /> total number bases window.                                     alignment search tool. J Mol Biol. 215:403&#226;&#8364;&#8220;410. <br />                                                                      Bakker  Toomajian C, Kreitman M, Bergelson J. 2006. genome-wide <br /> Alternative Splicing Events                                              survey R gene polymorphisms Arabidopsis. Plant Cell 18: <br />                                                                          1803&#226;&#8364;&#8220;1818. <br /> Alternative splicing events identified using methods        Boguski MS, Lowe TMJ, Tolstoshev CM. 1993. dbEST&#226;&#8364;&#8221;database described Chen et al. (2011). brief, number                 expressed sequence tags. Nat Genet. 4:332&#226;&#8364;&#8220;333. <br /> alternative splicing events gene identified aligning      Brenner S, Johnson M, Bridgham J, et al. (24 authors). 2000. Gene <br /> expressed sequence tag (EST) data obtained dbEST                    expression analysis massively parallel signature sequencing <br />                                                                          (MPSS) microbead arrays. Nat Biotechnol. 18:630&#226;&#8364;&#8220;634. <br /> (Boguski et al. 1993) genome sequence (ftp://ftp.ncbi.        Cao J, Schneeberger K, Ossowski S, et al. (17 authors). 2011.  <br /> nih.gov/repository/dbEST, accessed 1, 2011).              genome sequencing multiple Arabidopsis thaliana populations. <br /> ESTs aligning regions annotated gene                     Nat Genet. 43:956&#226;&#8364;&#8220;963. <br />  <br /> 68 <br />  Presence&#226;&#8364;&#8220;Absence Variation  thaliana . doi:10.1093/molbev/mst166                                                                    MBE <br /> Cheadle C, Vawter MP, Freed WJ, Becker KG. 2003. Analysis micro-              analyses mRNA small RNA. Nucleic Acids Res. 34: <br />     array data using score transformation. J Mol Diagn. 5:73&#226;&#8364;&#8220;81.               D731&#226;&#8364;&#8220;D735. <br /> Chen L, Tovar-Corona JM, Urrutia AO. 2011. Increased levels noisy         Oliver KR, Greene WK. 2009. Transposable elements: powerful facilita- <br />     splicing cancers, oncogene-derived transcripts. Hum           tors evolution. Bioessays 31:703&#226;&#8364;&#8220;714. <br />     Mol Genet. 20:4422&#226;&#8364;&#8220;4429.                                                 Ossowski S, Schneeberger K, Clark RM, Lanz C, Warthmann N, Weigel D. <br /> Chen L, Tovar-Corona JM, Urrutia AO. 2012. Alternative splicing:               2008. Sequencing natural strains Arabidopsis thaliana     potential source functional innovation eukaryotic                  short reads. Genome Res. 18:2024&#226;&#8364;&#8220;2033. <br />     genome. Int J Evol Biol. 2012:10.                                        Pearson WR. 2000. Flexible sequence similarity searching Chen W-H, Trachana K, Lercher MJ, Bork P. 2012. Younger genes                FASTA3 Program Package. Methods Mol Biol. 132:185&#226;&#8364;&#8220;219. <br />     likely essential older genes, duplicates    Punta M, Coggill PC, Eberhardt RY, et al. (16 authors). 2012. Pfam <br />     likely essential singletons age. Mol Biol Evol.       protein families database. Nucleic Acids Res. 40:D290&#226;&#8364;&#8220;D301. <br />     29:1703&#226;&#8364;&#8220;1706.                                                            Santuari L, Pradervand S, Amiguet-Vercher M, Thomas J, Dorcey E, <br /> Cherry JL. 2010. Expression level, evolutionary rate, cost            Harshman K, Xenarios  Juenger T, Hardtke C. 2010. Substantial <br />     expression. Genome Biol Evol. 2:757&#226;&#8364;&#8220;769.                                     deletion overlap divergent Arabidopsis genomes revealed <br /> Clark RM, Schweikert G, Toomajian C, et al. (18 authors). 2007.               intersection short reads tiling arrays. Genome Biol. 11:R4. <br />     Common sequence polymorphisms shaping genetic diversity               Shen J, Araki H, Chen L, Chen JQ, Tian D. 2006. Unique evolutionary <br />     Arabidopsis thaliana. Science 317:338&#226;&#8364;&#8220;342.                                   mechanism R-genes presence/absence polymorphism <br /> DeYoung BJ, Innes RW. 2006. Plant NBS-LRR proteins pathogen                   Arabidopsis thaliana. Genetics 172:1243&#226;&#8364;&#8220;1250. <br />     sensing host defense. Nat Immunol. 7:1243&#226;&#8364;&#8220;1249.                      Smedley D, Haider S, Ballester B, Holland R, London D, Thorisson G, <br /> Domazet-Los&#203;&#8225;o T, Brajkovic&#194;&#180; J, Tautz D. 2007. phylostratigraphy                Kasprzyk  2009. BioMart&#226;&#8364;&#8221;biological queries easy. BMC <br />     approach uncover genomic history major adaptations              Genomics 10:22. <br />     metazoan lineages. Trends Genet. 23:533&#226;&#8364;&#8220;539.                             Swanson-Wagner RA, Eichten SR, Kumari S, Tiffin P, Stein JC, Ware D, <br /> Feuk L, Carson AR, Scherer SW. 2006. Structural variation human           Springer NM. 2010. Pervasive gene content variation copy <br />     genome. Nat Rev Genet. 7:85&#226;&#8364;&#8220;97.                                              number variation maize undomesticated progenitor. <br /> Gan X, Stegle O, Behr J, et al. (23 authors). 2011. Multiple reference        Genome Res. 20:1689&#226;&#8364;&#8220;1699. <br />     genomes transcriptomes Arabidopsis thaliana. Nature 477:         Tajima F. 1989. Statistical method testing neutral mutation <br />     419&#226;&#8364;&#8220;423.                                                                     hypothesis DNA polymorphism. Genetics 123:585&#226;&#8364;&#8220;595. <br /> Gos G, Wright SI. 2008. Conditional neutrality adjacent NBS-LRR       Tan S, Zhong Y, Hou H, Yang S, Tian D. 2012. Variation     disease resistance loci natural populations Arabidopsis lyrata.        presence/absence genes Arabidopsis populations. BMC <br />     Mol Ecol. 17:4953&#226;&#8364;&#8220;4962.                                                      Evol Biol. 12:86. <br /> Hanada K, Kuromori T, Myouga F, Toyoda T, Li WH, Shinozaki K. 2009.          Van der Hoorn RAL, Wit PJGM, Joosten MHAJ. 2002. Balancing <br />     Evolutionary persistence functional compensation duplicate             selection favors guarding resistance proteins. Trends Plant Sci. 7: <br />     genes Arabidopsis. Genome Biol Evol. 1:409&#226;&#8364;&#8220;414.      <br /> </body></html>