<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                          MBoC&#226;&#8364;&#8218; |&#226;&#8364;&#8218; ARTICLE <br />  <br />  <br />  <br />  <br /> Yeast mRNA cap-binding protein Cbc1/Sto1 necessary rapid reprogramming translation hyperosmotic shock <br /> Elena Garrea, Lorena Romero-Santacreua, Nikki Clercqa, Nati Blasco-Anguloa, Sunnerhagenb, Paula Alepuza  Departamento Bioqu&#195;&#173;mica y Biolog&#195;&#173;a Molecular, Facultad Ciencias Biol&#195;&#179;gicas, Universitat Val&#195;&#168;ncia, E-46100 <br /> Burjassot, Valencia, Spain; bDepartment Cell Molecular Biology, Lundberg Laboratory, University Gothen- <br /> burg, S-405 30 Gothenburg, Sweden <br />  <br />  <br />  <br />  <br /> ABSTRACT&#226;&#8364;&#8218; response osmotic stress, global translation inhibited, mRNAs                                      Monitoring Editor <br /> coding stress-protective proteins selectively translated allow cell survival. <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>,                               Karsten Weis <br />                                                                                                                                 University California, <br /> mechanisms factors involved specific translation osmostress-responsive genes                                                                                                                                 Berkeley <br /> Saccharomyces cerevisiae unknown. mRNA cap-binding protein Cbc1 important yeast survival osmotic stress. results provide new evidence support-                                    Received: 12, 2011 <br /> ing role Cbc1 translation initiation. Cbc1 associates polysomes, deletion                                Revised: Oct 27, 2011 CBC1 gene causes hypersensitivity translation inhibitor cycloheximide yields                                  Accepted: Oct 28, 2011 <br /> synthetic &#226;&#8364;&#339;sickness&#226;&#8364;? cells limiting amounts translation initiator factor eIF4E. cbc1&#206;&#8221; <br /> mutants, translation drops sharply osmotic stress, subsequent reinitiation trans- <br /> lation retarded, &#226;&#8364;&#339;processing bodies&#226;&#8364;? containing untranslating mRNAs remain long <br /> periods. Furthermore, osmostress-responsive mRNAs transcriptionally induced os- <br /> motic stress cbc1&#206;&#8221; cells, rapid association polysomes delayed.  cells containing thermosensitive eIF4E allele, inability grow 37&#194;&#186;C suppressed hyperosmosis, Cbc1 relocalizes nucleus cytoplasm. data support model eIF4E-translation stress-sensitive, Cbc1-mediated translation necessary rapid translation osmostress-protective proteins osmotic stress. <br />  <br />  <br />  <br /> INTRODUCTION important response eukaryotic cells stress change                    2010). Defects translation reprogramming response stress <br /> gene expression patterns general shutdown repro-                           associated disease, including cancer, diabetes, gramming protein synthesis. Stress signals inhibit translation                    inflammatory processes (Keene, 2007). specific mRNAs  &#226;&#8364;&#339;housekeeping&#226;&#8364;? proteins promote translation stress-                        come translated stress, general translation  <br /> protective proteins (Yamasaki Anderson, 2008; Spriggs et al.,                        regulated, important issue understanding mechanisms <br />                                                                                          regulating gene expression. <br />                                                                                              Eukaryotic translation complex process controlled wide article published online ahead print MBoC Press (http://www            range regulatory factors. Stress signals usually inhibit process <br /> .molbiolcell.org/cgi/doi/10.1091/mbc.E11-05-0419) November 9, 2011. <br /> Address correspondence  Paula Alepuz (paula.alepuz@uv.es).                            initiation step. Normal translation takes place cap- <br /> Abbreviations used: CBC, cap-binding complex; CHX, cycloheximide; eIFs,                  dependent initiation mechanism; 40S ribosome subunit  <br /> eukaryotic initiation factors; FM, free monosomal; GFP, green fluorescent            cruited modified base 5&#226;&#8364;&#178; end mRNA  m7G cap). <br /> protein; IRES, internal ribosome entry sites; MAPK, mitogen-activated protein ki- <br />                                                                                          process coordinated complex eukaryotic initiation <br /> nase; MOPS, 3-(N-morpholino)-propanesulfonic acid; mRNP, messenger ribonu- <br /> cleoproteins; NMD, nonsense-mediated mRNA decay; P, polysomal; PAB1,                     factors (eIFs) called eIF4F, eIF4G scaffold protein poly -binding protein 1; P-bodies, processing bodies; PI, propidium iodide; RP,        facilitates interactions proteins small ribosomal <br /> ribosomal protein; SC, synthetic complete; SSC, saline sodium citrate; UTR, <br /> untranslated region; wt, wild type; YPD, yeast extract&#226;&#8364;&#8220;peptone&#226;&#8364;&#8220;dextrose.                 subunit, eIF4E recognizes binds cap structure mRNA. <br /> &#194;&#169; 2012 Garre et al. article distributed American Society Cell Biol-   Met-tRNA, 40S subunit, eukaryotic initiation fac- <br /> ogy license author(s). months publication available       tors form complex capped mRNA, start codon recog- public Attribution&#226;&#8364;&#8220;Noncommercial&#226;&#8364;&#8220;Share Alike 3.0 Unported <br /> Creative Commons License (http://creativecommons.org/licenses nc-sa/3.0). <br />                                                                                          nized, large ribosomal subunit attaches. key target &#226;&#8364;&#339;ASCB&#194;&#174;,&#226;&#8364;&#339;  American Society Cell Biology&#194;&#174;,&#226;&#8364;? &#226;&#8364;&#339;Molecular Biology             control translation initiation step eIF4E, Cell&#194;&#174;&#226;&#8364;? registered trademarks American Society Cell Biology.            bound inhibitory proteins subsequently impede mRNA <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                                                                                 137&#226;&#8364;&#402; <br />  binding (Wang Proud, 2007; Lackner B&#195;&#164;hler, 2008; Van Der             translation adaptation (Hoyle et al., 2007; Parker Sheth, <br /> Kelen et al., 2009).                                                         2007; Buchan Parker, 2009; Romero-Santacreu et al., 2009).     global translation inhibited stress, mRNAs         date, precise mechanisms factors modulating changes encoding stress-response proteins able engage ri-            mRNA translation fate osmotic stress remain unknown. <br /> bosomes translated. achieve  mechanisms                   nuclear cap-binding complex (CBC) S. cerevisiae, com- evolved eukaryotic cells.  recruitment          posed Cbc1 Cbc2, recruited cotranscriptionally cap ribosomes regulated elements 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; untranslated          structure mRNAs (G&#195;&#182;rlich et al., 1996; Wong et al., 2007). regions (UTRs) mRNAs, including internal ribosome entry sites         CBC complex mainly nuclear, shuttle nucleus <br /> (IRES) bypass cap-dependent recruitment mRNA,                 cytoplasm using importin &#206;&#177;/&#206;&#178; (G&#195;&#182;rlich et al., <br /> stream open reading frames, microRNA target sites (Yamasaki              1996). Individual CBC subunits essential yeast, dele- Anderson, 2008; Van Der Kelen et al., 2009; Spriggs et al.,              tions corresponding genes diminish cell growth glucose <br /> 2010).  stress conditions translation           media, mutants different phenotypes related different <br /> programming described, selective translation mecha-             aspects RNA metabolism (Das et al., 2000); consequently, CBC <br /> nism stress-specific response mRNAs remains unknown. relo-            proposed participant wide variety mRNA-asso- <br /> calization stress &#226;&#8364;&#339;housekeeping&#226;&#8364;? mRNAs cytoplasmic                ciated processes. Nuclear CBC involved splicing (Colot et al., <br /> foci, stored translationally silenced state,       1996; Fortes et al., 1999; Zhang Rosbash, 1999; G&#195;&#182;rnemann <br /> allow dedication basal translation synthesize protective         et al., 2005; Hossain et al., 2009), formation preinitiation <br /> proteins (Bond, 2006; Buchan Parker, 2009; Lui et al., 2010).            complex active genes (Lahudkar et al., 2011), transcription termi- <br />     experimental models studying regulation            nation (Das et al., 2000; Wong et al., 2007), nuclear mRNA deg- <br /> gene expression stress response Saccharomyces cerevi-           radation (Das et al., 2003; Kuai et al., 2005).  mamma- <br /> siae cells hyperosmotic shock. sudden increase external os-          lian cells, CBC plays role exporting capped RNA, role molarity produces rapid effects yeast cells, rapid      clear yeast (Izaurralde et al., 1995; G&#195;&#182;rlich et al., 1996; Wong <br /> decrease protein synthesis activation HOG mitogen-             et al., 2007). <br /> activated protein kinase (MAPK) pathway. Activation phosphoryla-              nuclear roles Cbc1, mammalian homologue <br /> tion MAPK Hog1, homologous mammalian p38 MAPK,                CBP80 proposed forming pioneer transla- necessary control different cellular processes allow      tion initiation complex. bulk cellular translation initiation cells adapt survive stress condition (reviewed           mediated cytoplasmic cap-binding initiation factor eIF4E, Nadal et al., 2002; Nadal Posas, 2010; Hohmann, 2002).            CBP80-bound mRNA, precursor eIF4E-bound mRNA, main effects Hog1 activation quick, massive change         translated pioneer round translation, gene expression. levels 600 mRNAs (represent-             aberrant mRNAs eliminated nonsense-mediated <br /> ing 10% yeast genes) significantly change minutes         mRNA decay (NMD) pathway (Ishigaki et al., 2001; Lejeune et al., stress, result changes transcription rates stabil-   2002; Chiu et al., 2004). Supporting model, CBP80 ity (Posas et al., 2000; Rep et al., 2000; Molin et al., 2009; Romero-       associated subpolysomal fractions, translation <br /> Santacreu et al., 2009; Miller et al., 2011). quick transient        initiation factors eIF4GI, eIF3, eIF4I, eIF2A poly -bind- <br /> general inhibition translation observed osmotic stress oc-          ing protein (PAB)1, coimmunopurify CBP80 curs initiation step unidentified mechanism appar-      eIF4E, suggests factor functions transla- <br /> ently independent Hog1.  MAPK involved             tion modes. <br /> later activation translation, possibly Rck2,             yeast, Cbc1 proposed mediator pio- <br /> yeast kinase resembles mammalian MAPK-activated protein ki-             neer round translation; nonetheless, contradictory data provide <br /> nases (Bilsland-Marchesan et al., 2000; Teige et al., 2001). Rck2      unclear view Cbc1&#226;&#8364;&#8482;s role translation. hand, Gao <br /> affect global translation directly regulating elongation factor EF-2,     et al. (2005) showed NMD occurs Cbc1-bound mRNAs effect initiation factors, occurs homologous          eIF4E-mRNAs yeast.  Kuperwasser et al. (2004) <br /> mammalian p38 MAPK pathway, ruled  important             argued Cbc1/Cbc2-mediated pioneer round transla- note recent studies shown Hog1-dependent,                tion, NMD occurs minimally Cbc2-bound <br /> short-term regulation mRNA translation rates takes place hy-        mRNA.  remarkably, deletion CBC1 resulted syn- <br /> perosmotic stress.  global inhibition transla-       thetic lethality eIF4GI version mutated sites interac- <br /> tion, important genes cell survival stress condition      tion eIF4E Pab1 (Fortes et al., 2000). authors&#226;&#8364;&#8482; work <br /> translationally regulated, protein synthesis increases          showed eIF4GI simultaneously interact competitively (Melamed et al., 2008; Halbeisen Gerber, 2009; Warringer et al.,         Cbc1 eIF4E Cbc1 stimulates 2.5-fold translation 2010). scenario, connections translation, mRNA             extracts yeast strains harboring mutated eIF4GI interacts <br /> degradation formation RNA containing granules,                weakly eIF4E.  later results revealed <br /> example, stress granules &#226;&#8364;&#339;processing bodies&#226;&#8364;? (P-bodies),            cells expressing eIF4GI, point mutation abolishes important,  parallel regulation translation          binding Cbc1, fail manifest detectable defect rates, stability functionally related groups mRNAs selec-       growth composition transcriptome (Baron-Benhamou <br /> tively modified P-bodies seen temporarily increase           et al., 2003). Collectively, results suggest connection  <br /> hyperosmotic shock (Greatrix van Vuuren, 2006; Hilgers et al.,           tween yeast Cbc1 translation, explain func- <br /> 2006; Molin et al., 2009; Romero-Santacreu et al., 2009; Miller et al.,      tional significance connection yeast cells. <br /> 2011). Translationally repressed mRNAs degraded P-bodies               importance mammalian CBP80-dependent pioneer main mRNA decay pathway, includes decapping                 round translation response stress conditions  <br /> enzyme Dcp1/Dcp2 5&#226;&#8364;&#178;&#226;&#8364;&#8220;3&#226;&#8364;&#178; exonuclease Xrn1. stress,               cently documented. mammalian cells prolonged hypoxia  mRNAs stored translation initia-          serum starvation, steady-state translation dependent tion factors, eIF4E, P-bodies, allows return       eIF4E dramatically inhibited;  NMD, mammalian <br />  <br />  <br /> 138&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                                 Molecular Biology Cell <br />  FIGURE 1:&#226;&#8364;&#8218; cbc1&#206;&#8221; mutant shows growth defects loss viability osmotic stress.   NaCl <br /> concentration required inhibit 50% growth (IC50) measure osmotic stress sensitivity indicated <br /> strains. data mean independent experiments SE indicated. (B) Sensitivity different osmotic stress conditions cbc1&#206;&#8221; hog1&#206;&#8221; mutant cells. Exponentially growing cells spotted YPD plates supplemented NaCl sorbitol. Plates incubated 30&#194;&#176;C. (C) Viability analysis cbc1&#206;&#8221; <br /> mutant osmotic stress. Exponentially growing wt cbc1&#206;&#8221; cells YPD subjected indicated osmotic <br /> stress conditions kept YPD stress. Samples taken indicated times dead cells determined flow cytometry using PI. percentage living cells condition compared cells equivalent aliquot strain incubated YPD. data mean SE independent <br /> experiments. <br />  <br /> cells restricts pioneer round translation CBP80, occurs    tolerance 182 mutants  Euroscarf mutant collec- <br /> efficiently (Oh et al., 2007a, 2007b).  CBP80 translation         tion, http://web.uni-frankfurt fb15/mikro/euroscarf) assayed <br /> differs steady-state translation, specific translation      determining NaCl concentration causes growth reduc- <br /> initiation factor, CTIF, preferentially involved CBP80-transla-      tion 50% (IC50 index; Supplemental Table S1). tion (Lee et al., 2008; Kim et al., 2009).                                 mutants showed reduced tolerance NaCl contained dele- <br />     purpose identifying proteins involved          tion CBC1 gene (Figure 1A), encodes nuclear cap- <br /> posttranscriptional regulation gene expression S. cerevisiae         binding protein Cbc1, homologous mammalian CBP80 pro- <br /> cells osmotic stress, performed osmotic stress sensitiv-       tein (Lewis et al., 1996). sensitivity cbc1&#206;&#8221; mutant NaCl <br /> ity screening using 182 mutants yeast genes encoding RNA-           reduced tolerance osmotic stress, cbc1&#206;&#8221; <br /> binding proteins. sensitive strains carried deletion     mutant displayed hypersensitivity high concentrations CBC1. results Cbc1 directly involved transla-       sorbitol (Figure 1B).  cbc1&#206;&#8221; cells showed strong reduc- <br /> tion normal conditions Cbc1-mediated translation            tion viability treatments 0.6 M KCl 1.2 M NaCl, ensures rapid synthesis osmostress-responsive genes            compared wild-type (wt) cells (Figure 1C; 40% hyperosmotic shock.                                                        0.6 M KCl 30% 1.2 M NaCl). <br />  <br /> RESULTS                                                                    Osmotic stress&#226;&#8364;&#8220;activated mRNAs induced <br /> Deletion CBC1 gene results growth reduction                  cbc1&#206;&#8221; mutants hyperosmotic shock <br /> loss viability osmotic stress                                     Gene expression patterns massive, rapid, transiently purpose discovering yeast proteins involved          changed response hyperosmotic shock. changes regulation gene expression response osmotic stress         place transcriptional posttranscriptional levels yield,  posttranscriptional levels, checked tolerance            things, variation mRNA amounts 10% NaCl mutants genes encoding proteins            yeast genes, controlled Hog1 MAPK (reviewed scribed involved RNA metabolism. select mutants,            Nadal Posas, 2010). Cbc1 protein cotranscriptionally <br /> searched Saccharomyces Genome Database 182                bound mRNAs cap structures, cells genes deletions viable, products             CBC1 gene deleted hypersensitive osmotic <br /> described involved mRNA export, translation, degradation,         stress, reasoned regulation gene expression splicing; considered genes unknown functions           defective cbc1&#206;&#8221; mutant, case corresponding proteins contained RNA-binding domains. NaCl             osmostress-hypersensitive mutants, including mutations HOG <br />  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                        Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 139&#226;&#8364;&#402; <br />  FIGURE 2:&#226;&#8364;&#8218; Increase levels osmostress-responsive mRNAs delayed, inhibited, cbc1&#206;&#8221; mutants. wt, hog1&#206;&#8221;, cbc1&#206;&#8221; strains grown exponentially YPD, cells subjected osmotic stress addition 0.6 M KCl. Samples taken indicated times expression indicated genes analyzed Northern blotting  . (B) Quantifications mRNA levels normalized rRNA. data mean SE independent experiments. <br />  <br />  <br />  <br />  <br /> pathway components (O&#226;&#8364;&#8482;Rourke Herskowitz, 2004). test              osmotic stress (Posas et al., 2000; Romero-Santacreu et al., 2009). possibility, investigated induction total       results reveal variation mRNA levels mRNA levels STL1, GPD1, HSP12, involved                 genes osmotic stress treatment 0.6 M KCl similar glycerol transport, glycerol production, maintenance           cbc1&#206;&#8221; wt cells, delay recovery basal <br /> membrane organization, respectively, corresponding             level RPS31 mRNA observed cbc1&#206;&#8221; mutant cells <br /> mRNAs strongly transiently induced osmotic stress             (Figure 2). summary, results indicate growth defect <br /> (Posas et al., 2000; Romero-Santacreu et al., 2009). previously       loss viability cbc1&#206;&#8221; mutant hyperosmotic stress <br /> reported (Rep et al., 1999; Posas et al., 2000), osmostress       correlate inability induce mRNA levels duction STL1 fully dependent Hog1, induc-            osmotic stress-responsive genes. contrary, results indi- <br /> tions GPD1 HSP12 partially dependent MAPK            cate induction cbc1&#206;&#8221; mutant stronger lasts <br /> (Figure 2). cbc1&#206;&#8221; mutant, genes induced af-        longer wt strain. <br /> ter treatment 0.6 M KCl. increase levels STL1, <br /> GPD1, HSP12 time course       cbc1&#206;&#8221; mutant shows defect reinitiation <br /> 15 min STL1; 30 min GPD1 HSP12) similar        translation hyperosmotic shock cbc1&#206;&#8221; wt cells, slight delay noted mutant           Gene expression changed translation level os- <br /> (Figure 2B).  kinetics mRNA induction showed dif-         motic shock. yeast, increased extracellular osmolarity leads ferences cbc1&#206;&#8221; wt maximal levels reached          transient inhibition translation initiation step. ad- length response. cbc1&#206;&#8221; mutant, increase      aptation, protein synthesis restored unknown Hog1- <br /> STL1, GPD1, HSP12 mRNA amounts approximately                 dependent mechanism (Uesono Toh-e, 2002). stop fold higher peaked 10&#226;&#8364;&#8220;20 min later wt (Figure 2).        global translation occurs parallel transient increase P- mRNA encoding ribosomal proteins (RPs),                bodies incorporation mRNAs polysomes, RPL17A RPS31, described reduced               delayed lasts longer hog1&#206;&#8221; mutant cells (Teixeira <br />  <br />  <br /> 140&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                            Molecular Biology Cell <br />                                                                                                        obtaining polysome profiles fol- <br />                                                                                                       lowing P-bodies&#226;&#8364;&#8482; status. non- <br />                                                                                                       stress conditions, cbc1&#206;&#8221; hog1&#206;&#8221; <br />                                                                                                       mutants showed similar translation pro- <br />                                                                                                       file wt strain (Figure 3A). Af- <br />                                                                                                       ter 10 min osmotic stress treatment                                                                                                       0.6 M KCl, global translation dropped                                                                                                       strains, expected. cbc1&#206;&#8221; <br />                                                                                                       mutant displayed marked decrease <br />                                                                                                       wt hog1&#206;&#8221; mutants <br />                                                                                                       (Figure 3, B). Global translation  <br />                                                                                                       covered 30 min stress wt cells, <br />                                                                                                       polysome levels did recover <br />                                                                                                       60 min cbc1&#206;&#8221; mutant.                                                                                                       time, possible global <br />                                                                                                       translation hog1&#206;&#8221; mutant                                                                                                       completely recovered (Figure 3B).                                                                                                       P-bodies&#226;&#8364;&#8482; status osmotic stress                                                                                                       followed visualizing green fluorescent <br />                                                                                                       protein (GFP)-tagged version  <br />                                                                                                       capping enzyme Dcp2 (Sheth Parker, <br />                                                                                                       2003). previously described (Teixeira <br />                                                                                                       et al., 2005; Romero-Santacreu et al., <br />                                                                                                       2009), treatment 1 M KCl, <br />                                                                                                       P-bodies visualized wt cells af- <br />                                                                                                       ter 30 min stress remained                                                                                                       60 min;  hog1&#206;&#8221; cells, assem- <br />                                                                                                       bly P-bodies observed 60 min, <br />                                                                                                       bright foci visible 2 h <br />                                                                                                       stress (Figure 3, C D). cbc1&#206;&#8221; <br />                                                                                                       mutant, P-body assembly/disassembly ki- <br />                                                                                                       netics similar hog1&#206;&#8221; <br />                                                                                                       mutant: P-bodies clearly visible                                                                                                       60 min stress observed <br />                                                                                                       similar brightness 2 h treat- <br />                                                                                                       ment (Figure 3C). Mild osmotic stress  <br />                                                                                                       ditions, treatment 0.6 M KCl <br />                                                                                                       0.4 M NaCl, cause quicker                                                                                                       transitory formation P-bodies (Romero- <br />                                                                                                       Santacreu et al., 2009; Figure 3C).                                                                                                       conditions, P-bodies visu- <br />                                                                                                       alized longer periods cbc1&#206;&#8221; cells <br /> FIGURE 3:&#226;&#8364;&#8218; cbc1&#206;&#8221; mutant shows defect translation recovery hyperosmotic stress.         compared wt cells (Figure 3, C D). <br />   Reinitiation translation adaptation osmotic stress delayed cbc1&#206;&#8221; mutant.    results indicate Cbc1 es- <br /> Polysome profiles translation inhibition recovery osmotic stress wt,           sential P-body formation.   <br /> hog1&#206;&#8221;, cbc1&#206;&#8221; strains. Cells grown exponentially YPD treated 0.6 M KCl         lay recovery global translation <br /> indicated times. Polysomal profiles obtained described Materials Methods.          prolonged permanence P-bod- <br /> relative rRNA fraction, measured online UV detection 260 nm, shows            ies cbc1&#206;&#8221; mutant suggest role ribosomal light subunits (40S), heavy subunits (60S), monosomes (80S), polysomes (2n, <br />                                                                                                       cap-binding protein P-body disas- <br /> 3n, 4n, &#226;&#8364;&#166;). Representative absorbance curves  repetitions) displayed. (B) <br />                                                                                                       sembly, return mRNA polysomes, <br /> Percentage polysomal area osmotic stress (0.6 M KCl) wt, hog1&#206;&#8221;, cbc1&#206;&#8221;. polysome percentage relative polysomal area strain time 0. data reinitiation translation. similar <br /> given mean SE independent experiments. (C) P-body assembly                    role control mRNA cytoplasmic <br /> disassembly osmotic shock delayed cbc1&#206;&#8221; mutant. wt, hog1&#206;&#8221;, cbc1&#206;&#8221;             fate response osmotic stress strains expressing GFP-tagged version decapping enzyme Dcp2 grown             suggested MAPK Hog1 (Romero- <br /> exponential phase, treated 1 M KCl, 0.6 M KCl, 0.4 M NaCl. Dcp2-GFP             Santacreu et al., 2009; Warringer et al., <br /> allowed P-body visualization fluorescence microscopy. (D) quantification                2010). Altogether, results indicate percentage cells P-bodies indicated graphs. 100 cells quantified      cbc1&#206;&#8221; mutant defective condition.                                                                                   induction osmostress mRNAs  <br />                                                                                                       fective reinitiation translation dur- <br /> et al., 2005; Romero-Santacreu et al., 2009).  second                ing adaptation. cbc1&#206;&#8221; mutant resembles step study gene expression osmotic stress               defects regulation gene expression caused HOG1 <br /> cbc1&#206;&#8221; mutant, analyzed changes global translation                     deletion. <br />  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                          Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 141&#226;&#8364;&#402; <br />                                                                             eIF4E function compromised. test  intro- <br />                                                                            duced CBC1 deletion yeast cells carrying thermosensitive <br />                                                                            mutation eIF4E (cdc33ts; Schwartz Parker, 1999). Yeast cells <br />                                                                            carrying cdc33ts allele (cdc33-42) previously shown                                                                            perform 80% wt translation 24&#194;&#176;C, ratio <br />                                                                            lowered 19% 38&#194;&#176;C (Schwartz Parker, 1999). Accordingly, <br />                                                                            cdc33ts cells did important growth defects yeast <br />                                                                            extract&#226;&#8364;&#8220;peptone&#226;&#8364;&#8220;dextrose (YPD) plates incubated 25&#194;&#176;C 33&#194;&#176;C <br />                                                                            unable grow 37&#194;&#176;C (Figure 4B). interesting <br />                                                                            note cdc33ts cbc1&#206;&#8221; double mutant clearly grew                                                                            cdc33ts mutant 33&#194;&#176;C (Figure 4B). Additionally, tested <br />                                                                            sensitivity CHX plates incubated longer period allow <br />                                                                            growth synthetic sick cdc33ts cbc1&#206;&#8221; strain. observed <br />                                                                            hypersensitive defect cdc33ts cbc1&#206;&#8221; single mu- <br />                                                                            tant strains exacerbated double mutant (Supplemental <br />                                                                            Figure S1A). Collectively, results suggest Cbc1, FIGURE 4:&#226;&#8364;&#8218; Cbc1 involved translation nonstress             eIF4E, functions initiation translation optimal <br /> osmotic stress conditions.   Sensitivity CHX cbc1&#206;&#8221; mutant.     conditions, consistent hypothesis Cbc1&#226;&#8364;&#8482;s involve- wt, cbc1&#206;&#8221; mutant, strains carrying deletions UPF1       ment cytoplasmic pioneer round translation. <br /> NPL3 genes grown exponentially YPD. aliquot                 mentioned earlier, results indicate cbc1&#206;&#8221; mutant <br /> culture diluted 8 &#206;&#188;l fivefold serial dilutions spotted     hypersensitive osmotic stress (Figure 1). proteins eIF4E plates containing YPD YPD supplemented 0.05, 0.075,         Cbc1 involved translation normal conditions, 0.1 &#206;&#188;g/ml CHX. Plates grown 2 d 30&#194;&#176;C. results      Cbc1 required reinitiation translation global <br /> figure representative independent experiments.         translation inhibition caused osmotic stress (Figure 2), (B) Synthetic sickness double mutant cdc33tscbc1&#206;&#8221;                                                                            reason investigate osmotic stress sensitivity resistance osmotic stress single mutant cdc33ts. Cultures                                                                            double mutant cdc33ts cbc1&#206;&#8221; single mutant cdc33ts. wt, cbc1&#206;&#8221;, cdc33ts, cdc33tscbc1&#206;&#8221; strains treated  , growth tested YPD YPD supplemented 0.8 M          Figure 4B depicts, cdc33ts cbc1&#206;&#8221; strain unable grow NaCl. Plates left grow 4 d indicated temperatures.      media 0.8 M NaCl temperature. hand, results figure representative              growth cdc33ts cells permissive temperature, independent experiments.                                                   eIF4E Cbc1 functional, considerably reduced                                                                            0.8 M NaCl added. suggests NaCl retains ability                                                                            affect growth independently eIF4E Cbc1. Unexpect- <br /> cbc1&#206;&#8221; cells hypersensitive translation inhibitor                edly, thermosensitive mutant cdc33ts grew better wt <br /> cycloheximide genetic interaction                          strain hyperosmotic media, nonpermissive tem- <br /> translation initiation factor eIF4E                                        perature (Figures 4B S1B). viability cdc33ts 37&#194;&#176;C Previously, proposed Cbc1 mediate translation            hyperosmotic media explained recovery wt <br /> (Fortes et al., 2000; Gao et al., 2005), results argued     levels protein eIF4E (Figure S1C). summary, results sug- role (Baron-Benhamou et al., 2003; Kuperwasser et al.,        gest Cbc1 eIF4E play redundant roles translation, 2004). results, revealed defects cbc1&#206;&#8221; mutants         roles eIF4E nonstress Cbc1 osmotic recovery translation P-body disassembly hyperos-          stress conditions prevalent. Depletion eIF4E pro- <br /> motic shock, prompted investigate involvement         tein deleterious yeast cells growing osmotic stress, <br /> Cbc1 translation. aim mind, examined sensitiv-      instead making yeast cells osmotolerant, suppression <br /> ity cbc1 mutant inhibitor translation elongation cy-      phenotype lost CBC1 gene deleted, suggests cloheximide (CHX; McKeehan Hardesty, 1969).             potentiation Cbc1&#226;&#8364;&#8482;s role translation pro- <br /> ported yeast strains translational defects hypersensitive    tein eIF4E limited. behavior result stable growth inhibitory effect CHX (Gross et al., 2007).   interaction Cbc1 eIF4G condition (Fortes et al., <br /> case mutants ribonucleoprotein Npl3, promotes transla-        2000). <br /> tion termination accuracy, Upf1, component NMD <br /> RNA surveillance pathway role translation (Leeds et al.,         Cbc1 located nucleus nonstress osmotic <br /> 1991; Estrella et al., 2009). Figure 4A depicts, cbc1&#206;&#8221; strain    stress, redistributes cytoplasm cells lacking <br /> hypersensitive CHX; fact, sensitive upf1&#206;&#8221;        translation initiation factor eIF4E exonuclease Xrn1 npl3&#206;&#8221; mutants (Figure 4A).                                             shown reinitiation translation P-body disas- <br />     Translation initiation factor eIF4E replaces Cbc1 cap struc-    sembly adaptation osmotic stress depend Hog1 ture cytoplasmic mRNAs.  proposed         Cbc1 (Romero-Santacreu et al., 2009; Figure 3). possibility eIF4E Cbc1 conduct pioneer rounds translation          Cbc1 expression function regulated os- <br /> yeast (Gao et al., 2005).  eIF4E essential cell       motic stress; regulation   Hog1-dependent. <br /> survival, deletion CBC1 yields slight growth defect        observed change Cbc1 protein optimal conditions (Figure 4, B), indicating bulk transla-      modification protein change electrophoretic <br /> tion initiated eIF4E (Gao et al., 2005). results hypersen-    mobility osmotic stress (Figure S2). Previous studies sitivity CHX support hypothesis role Cbc1 transla-      demonstrated Cbc1 accumulates nucleus optimal <br /> tion nonstress conditions. hypothesized Cbc1&#226;&#8364;&#8482;s role          growing conditions (G&#195;&#182;rlich et al. 1996, Shen et al., 2000). translation relevant survival conditions      checked Cbc1 protein changed cellular localization <br />  <br />  <br /> 142&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                              Molecular Biology Cell <br />                                                                                                 hyperosmotic media. test possibil- <br />                                                                                                ity, followed distribution Cbc1- <br />                                                                                                GFP fusion treatment 1 M <br />                                                                                                KCl times global translation <br />                                                                                                stopped P-bodies appeared (Figure 5A; <br />                                                                                                Teixeira et al., 2005; Romero-Santacreu <br />                                                                                                et al., 2009). observed Cbc1-GFP <br />                                                                                                enriched nucleus, regardless                                                                                                osmotic stress treatment (Figure 5A).                                                                                                results provide indication                                                                                                Cbc1 regulation osmotic stress. <br />                                                                                                     tested Cbc1 protein <br />                                                                                                changed localization cdc33ts strain, <br />                                                                                                results preceding section(s) sug- <br />                                                                                                gest Cbc1&#226;&#8364;&#8482;s role initiation                                                                                                translation relevant strain, <br />                                                                                                given limitation eIF4E-driven initi- <br />                                                                                                ation translation (Schwartz Parker, <br />                                                                                                1999). previously described, P-bodies <br />                                                                                                observed cdc33ts cells tem- <br />                                                                                                perature (Figures 5B S3; Teixeira et al., <br />                                                                                                2005; Brengues Parker, 2007). note- <br />                                                                                                worthy Cbc1 distributed  <br />                                                                                                cell cdc33ts mutant strain  <br />                                                                                                der nonstress osmotic stress <br />                                                                                                conditions cells incubated                                                                                                25&#194;&#176;C (Figure 5C). restrictive tempera- <br />                                                                                                ture (37&#194;&#176;C), Cbc1 distributed                                                                                                cell, cdc33ts cells                                                                                                additionally stressed 1M KCl 30                                                                                                60 min, Cbc1 protein                                                                                                cytoplasmic foci, partially colocal- <br />                                                                                                ized P-bodies (Figure 5, B C). <br />                                                                                                     inhibition mRNA degradation                                                                                                mutation 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; exonuclease Xrn1 <br />                                                                                                described increasing num- <br />                                                                                                ber P-bodies yeast cells normal <br />                                                                                                conditions (Sheth Parker, 2003). role <br />                                                                                                eIF4E targeting messenger ribonu- <br />                                                                                                cleoproteins (mRNPs) mammalian P- <br />                                                                                                bodies suggested (Andrei et al., <br />                                                                                                2005; Ferraiuolo et al., 2005), eIF4E <br />                                                                                                detected P-bodies yeast cells <br />                                                                                                (Brengues Parker, 2007). hypothe- <br />                                                                                                size eIF4E protein dedi- <br />                                                                                                cated active translation limited <br />                                                                                                xrn1&#206;&#8221; cells, possible stall FIGURE 5:&#226;&#8364;&#8218; Cbc1 accumulates nucleus wt cells nonstress osmotic stress <br />                                                                                                degradation eIF4E-containing <br /> conditions, distributed cell mutants translation initiation <br /> factor eIF4E mRNA exonuclease Xrn1.  D) wt xrn1&#206;&#8221; strains expressing           mRNPs P-bodies, Cbc1-GFP Dcp2-GFP (P-body marker) exponentially grown YPD. Cultures            case, localization Cbc1 xrn1&#206;&#8221; incubated 1 M KCl 30&#194;&#176;C. Cbc1-GFP localization P-bodies (Dcp2-GFP)        cells change. Figure 5D illustrates, <br /> visualized 0, 30, 60 min osmotic stress treatment fluorescence microscopy         Cbc1 lost nuclear accumulation pattern <br /> (left panels). Cells photographed Nomarski optics (right panels). (B) wt      xrn1&#206;&#8221; cells localized cdc33ts mutant strains expressing Cbc1-GFP Dcp2-GFP exponentially grown         cell. distribution Cbc1 cell <br /> YPD 25&#194;&#176;C. aliquot culture treated 1 M KCl incubated           observed xrn1&#206;&#8221; cells os- <br /> permissive temperature (25&#194;&#176;C). aliquot incubated restrictive temperature      motic stress (Figure 5D). <br /> (37&#194;&#176;C) 30 min treated 1M KCl. Cbc1-GFP localization, P-bodies <br />                                                                                                     summary, results indicate (Dcp2-GFP), cells visualized described  D). (C) Cbc1 partially <br />                                                                                                change Cbc1 nuclear localization colocalizes P-bodies. cdc33ts cells coexpressing Cbc1-GFP Dcp2-RFP <br /> (P-body marker) exponentially grown YPD 25&#194;&#176;C. Afterward culture             osmotic stress treatment, global trans- <br /> incubated restrictive temperature (37&#194;&#176;C) 30 min treated             lation transiently inhibited P-bodies <br /> 1 M KCl 60 min. Cbc1-GFP Dcp2-RFP visualized fluorescence                      appear incorporation bulk <br /> microscopy.                                                                                    mRNAs (Uesono Toh, 2002; Teixeira <br />  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                    Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 143&#226;&#8364;&#402; <br />                                                                                                       A260, indicated position ribonucleo- <br />                                                                                                      proteins, ribosomal subunits, monosomes, <br />                                                                                                      polysomes (Figure 6). Western <br />                                                                                                      blot analysis fractions revealed                                                                                                      significant amounts Cbc1 protein   <br />                                                                                                      tween 15 20% total Cbc1)                                                                                                      detectable polysome-containing frac- <br />                                                                                                      tions wt cells grown optimal <br />                                                                                                      conditions (Figure 6A). association                                                                                                      Cbc1 polysomes dependent                                                                                                      blocking translation elongation CHX <br />                                                                                                      (Figures 6B S4), indicating Cbc1 <br />                                                                                                      associates translationally active ribo- <br />                                                                                                      somes (Moldave, 1985; Ram&#195;&#173;rez et al., <br />                                                                                                      1991; Harel-Sharvit et al., 2010). Similar                                                                                                      CBP80 mammalian cells, Cbc1  <br />                                                                                                      tected polysomal fractions,                                                                                                      concentrated subpolysomal <br /> FIGURE 6:&#226;&#8364;&#8218; Cbc1p associates polysomes nonstress osmotic stress conditions.            fractions (Figure 6), eIF4E mainly <br />   wt culture growing exponentially YPD treated 0.6M KCl polysomal profiles detected higher polysomal fractions determined indicated times. mRNP, FM, P (P1&#226;&#8364;&#8220;P5) fractions collected         (Chiu et al., 2004; Hoyle et al., 2007). <br /> analyzed Western blotting anti-Cbc1 antibody. (B) wt cells treated 0.6 M       results suggest Cbc1-bound <br /> KCl 15 min. Polysomal profiles obtained aliquot culture (Control),       mRNAs efficiently translated described Materials Methods. aliquot culture treated NaN3         eIF4E-bound mRNAs. <br /> 30 min 30&#194;&#176;C, CHX MgCl2 omitted cell extract sucrose gradient               Association polysomes disrupt polysomes (&#226;&#710;&#8217;CHX/&#226;&#710;&#8217;MgCl2). analysis Cbc1 protein fractions                described yeast nucleus&#226;&#8364;&#8220;cy- <br /> performed  . Detection RP Rpl5 specific antibody used control                                                                                                      toplasm shuttling mRNA-bound proteins, <br /> polysome disassembly. polysome profiles Western blot analyses shown figure <br />                                                                                                      Npl3, Gbp2, Rpb4. correspond representative experiment. percentage Cbc1  B) Rpl5 (B) nonpolysomal (RNP + F + M) polysomal fractions (P1&#226;&#8364;&#8220;P5) indicated. data             proteins, Cbc1, fusions GFP <br /> provided mean SE independent experiments.                                 visualized nucleus,                                                                                                      higher concentration proteins                                                                                                      compartment (Windgassen et al., 2004; <br /> et al., 2005). Conversely, cells eIF4E limited,   Harel-Sharvit et al., 2010). mammalian cells, CBP80 associated <br /> cdc33ts xrn1&#206;&#8221; strains, Cbc1 relocalizes cytoplasm          polysomes highly enriched nucleus (Kim nonstress osmotic stress conditions. simple             et al., 2009). <br /> interpretation result recycling Cbc1 nu-         examined association Cbc1 polysomes  <br /> cleus needs replacement Cbc1 eIF4E cap structure        der osmotic stress. proportion polysome-associated Cbc1 cytoplasmic mRNA.  alternative nonexclusive           did significantly change 15 30 min addition 0.6 M <br /> terpretation Cbc1 needed cytoplasm replace       KCl (Figure 6A).  similar nuclear accumulation Cbc1, eIF4E functions translation eIF4E lim-        regardless osmotic stress, proportion Cbc1 associ- <br /> ited.  resistance cdc33ts osmotic stress,        ated polysomes remained unchanged stress.  strictive temperature (Figures 4B S1B), suggests increase      noted proportion Cbc1 associated Cbc1 cytoplasm offers cells advantage osmotic stress       polysomes constant, independent reduction poly- <br /> conditions, possibly Cbc1&#226;&#8364;&#8482;s competence translation. Fi-         somes translation inhibition provoked osmotic stress <br /> nally, localization Cbc1 protein cytoplasmic foci    (Figure 6A); Cbc1 constituted larger fraction polysome- cdc33ts cells simultaneously treated high tempera-          associated proteins. <br /> ture osmotic stress resembles visualization eIF4E ini-          examined association Cbc1 polysomes tiation factor P-bodies global translation inhibited            cdc33ts xrn1&#206;&#8221; mutants. results indicate cytoplasmic <br /> (Brengues Parker, 2007), indicative        localization Cbc1 protein cdc33ts xrn1&#206;&#8221; (Figure 5) does <br /> placement eIF4E function translation Cbc1        lead increased association protein polysomes <br /> condition.                                                                 (Figure S5, B). <br />                                                                                 data suggest  1) Association Cbc1 Cbc1 associates polysomes                                             polysomes independent osmotic stress;  propor- <br /> Cbc1&#226;&#8364;&#8482;s role translation initiation performed         tion Cbc1-dependent translation increases osmotic stress, <br /> direct association Cbc1 active translation machinery.       bulk translation inhibited condition, mammalian cells, CBP80 associates polysomal fractions              proportion Cbc1 protein associated polysomes  <br /> carry pioneer round translation (Chiu et al., 2004; Kim         mains constant. 2) Association Cbc1 polysomes remains  <br /> et al., 2009). study Npl3 function translation         stant despite abrogation Xrn1 eIF4E; result sug- <br /> yeast, Cbc1 detected polysomal fractions (Wind-             gests proportion Cbc1-bound mRNA relation gassen et al., 2004).  decided depth anal-          total mRNA active translation increases eIF4E transla- <br /> ysis Cbc1&#226;&#8364;&#8482;s ability cosediment polysomes             tion initiation factor limited cdc33ts putatively optimal osmotic stress conditions. rRNA profiles, measured          xrn1&#206;&#8221; mutants. <br />  <br />  <br /> 144&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                                Molecular Biology Cell <br />                                                                                                          tation. Recently reported                                                                                                         specific functional groups mRNAs  <br />                                                                                                         crease association polysomes <br />                                                                                                         osmotic stress conditions                                                                                                         global translation inhibited (Melamed <br />                                                                                                         et al., 2008; Halbeisen Gerber, 2009; <br />                                                                                                         Warringer et al., 2010). genes <br />                                                                                                         translationally regulated                                                                                                         regulated transcriptional level,  <br />                                                                                                         cluding genes essential survival os- <br />                                                                                                         motic stress, GPD1 (Melamed et al., <br />                                                                                                         2008; Warringer et al., 2010). predic- <br />                                                                                                         tion model Cbc1&#226;&#8364;&#8482;s role transla- <br />                                                                                                         tion osmotic stress associa- <br />                                                                                                         tion osmostress-regulated mRNAs <br />                                                                                                         polysomes affected                                                                                                         yeast cells lacking CBC1 gene. test <br />                                                                                                         possibility, examined association <br />                                                                                                         osmostress-regulated mRNAs <br />                                                                                                         polysomes osmotic stress&#226;&#8364;&#8220; <br />                                                                                                         treatment time course (Figure 7). GPD1, <br />                                                                                                         STL1, HSP12, mRNA amounts increased <br />                                                                                                         15 min treatment 0.6 M KCl <br />                                                                                                         (Figures 2 7A). time, impor- <br />                                                                                                         tant proportion mRNAs  <br />                                                                                                         tected polysomal fraction wt cells <br />                                                                                                         (Warringer et al., 2010), poly- <br />                                                                                                         somal association osmostress-regu- <br />                                                                                                         lated mRNAs clearly lower cbc1&#206;&#8221; <br />                                                                                                         cells (Figure 7A). noteworthy sig- <br />                                                                                                         nificant proportion mRNAs remained <br />                                                                                                         associated monosomal free <br />                                                                                                         fraction cbc1&#206;&#8221; cells 30 min stress, <br />                                                                                                         mRNAs associated                                                                                                         polysomes wt cells (Figure 7A).  <br />                                                                                                         scribed earlier (Figure 2), STL1, GPD1, <br />                                                                                                         HSP12 mRNA levels high                                                                                                         cbc1&#206;&#8221; strain times osmotic stress <br />                                                                                                         levels dropped wt strain <br /> FIGURE 7:&#226;&#8364;&#8218; Rapid translation reprogramming hyperosmotic stress delayed cbc1&#206;&#8221;            (Figure 7A). analysis polysomal (P) <br /> mutant.   Association osmostress response mRNAs RP mRNAs polysomes wt <br />                                                                                                         free monosomal (FM) ratios indi- cbc1&#206;&#8221; osmotic stress. wt cbc1&#206;&#8221; cultures growing exponentially YPD                                                                                                         cates STL1, GPD1, HSP12 <br /> treated 0.6 M KCl, polysomal profiles determined indicated times. fractions corresponding FM P collected analyzed Northern blotting using mRNAs actively translated 60 min specific probes recognize indicated mRNAs. RNP fraction excluded            stress cbc1&#206;&#8221; strain (Figure 7B). gave signal mRNAs tested (unpublished data). (B) quantified signals            analyzed translational status independent replicates used determine average change P/FM                 mRNAs, RPL17A RPS31, encode <br /> ratios indicated genes. (C) Translation efficiency STL1-LacZ fusion wt cbc1&#206;&#8221;      RPs. reported decrease osmotic stress. wt cells expressing fusion STL1-LacZ plasmid  STL1                 levels mRNAs encoding RPs <br /> promoter fused LacZ, LEU2+ [2 &#194;&#181;m]) grown selective media 30&#194;&#176;C                   occurs quickly osmotic stress (Posas <br /> exponential phase, transferred YPD incubated 30&#194;&#176;C 4 h. Cells           et al., 2000; Molin et al., 2009; Romero- subjected osmotic stress adding 0.6 M KCl, samples taken                                                                                                         Santacreu et al., 2009) takes place <br /> indicated times. &#206;&#178;-galactosidase activity measured described Materials Methods, <br />                                                                                                         parallel translational regulation LacZ mRNA level determined Northern blotting. Translation efficiency calculated &#206;&#178;-galactosidase activity/LacZ mRNA ratio, results normalized          RP-encoding mRNAs <br /> efficiency time 0. data shown mean SE independent experiments.              (Melamed et al., 2008; Halbeisen                                                                                                         Gerber, 2009; Warringer et al., 2010). Ac- <br />                                                                                                         cordingly, levels P/FM ratios Cbc1 required rapid translation osmotic-induced                   RPL17 RPS31 mRNAs lower treatment 0.6 M KCl <br /> mRNAs osmotic stress                                                    wt strain (Figure 7, B). Similar results obtained results enabled hypothesize Cbc1 directly         cbc1&#206;&#8221; mutant strain 30 min stress;  <br /> involved translation initiation osmotic stress, bulk            recovery polysomal association RP-encoding <br /> translation inhibited. Given stress condition, Cbc1-dependent         mRNAs delayed cbc1&#206;&#8221;, compared wt cells (Figure 7, translation work efficiently relevant cell adap-       B; differences P/FM ratios 60 min). <br />  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                           Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 145&#226;&#8364;&#402; <br />      independently confirm Cbc1&#226;&#8364;&#8482;s role rapid translation      recovery. Fourth, osmostress-responsive mRNAs induced transcripts osmotic stress, monitored translation efficiency     cbc1&#206;&#8221; mutant response osmotic stress (Figure 2);  cbc1&#206;&#8221; cells checking expression osmotic stress         rapid association polysomes, observed wt cells, ducible reporter gene. used fusion gene STL1             significantly delayed mutant (Figure 7A).   <br /> promoter LacZ reporter gene (STL1-LacZ), measured             crease translation efficiency osmotic stress reporter &#206;&#178;-galactosidase activity corresponding protein relation     gene STL1-LacZ subsequent decline delayed increase LacZ mRNA level osmotic stress treat-       cbc1&#206;&#8221; mutant, compared wt cells (Figure 7B). Fifth, growth <br /> ment cells 0.6 M KCl. Figure 7C shows, efficiency       inability cdc33ts mutants 37&#194;&#176;C suppressed hyperosmosis translation STL1-LacZ mRNA (proportion reporter protein          cells limiting amounts eIF4E, Cbc1 relocalizes <br /> activity relation mRNA  increased significantly faster   cytoplasm. Sixth, Cbc1 protein fraction associated wt cbc1&#206;&#8221; cells. According previous data regarding       polysomes remains constant osmotic stress, global association osmostress-induced mRNAs polysomes                translation inhibited; indicates proportion Cbc1- <br /> (Figure 7B), translation efficiency STL1-LacZ needed          bound mRNA active translation increases stress (Figure 6A). <br /> time recover cbc1&#206;&#8221; mutant osmotic shock (Figure 7C).        phenotypes cbc1&#206;&#8221; mutant translation os- importance 3&#226;&#8364;&#178;-UTRs proper regulation mRNA            motic stress merely result Cbc1&#226;&#8364;&#8482;s role mRNA  <br /> translation osmotic stress indicated (Warringer et al.,    tabolism, mutants genes involved splicing, tran- <br /> 2010). result obtained STL1-LacZ indicates impaired           scription, mRNA export similar effects (Figure S6). <br /> reporter-RNA translation cbc1&#206;&#8221; cells independent            suggest model bulk translation driven eIF4E <br /> native 3&#226;&#8364;&#178;UTR STL1 mRNA. likely interpretation STL1-        stress-sensitive inhibited osmotic stress, LacZ translation regulated STL1 5&#226;&#8364;&#178;-UTR.                    Cbc1-mediated translation stress-resistant. newly syn- <br />     Collectively, results indicate Cbc1 necessary   thesized mRNA encoding osmostress-protective proteins rapid translation mRNAs induced osmotic stress,        bound Cbc1 nucleus undergo Cbc1-dependent regulation translation mRNAs encoding            translation cytoplasm. time, pool house- RPs independent Cbc1. summary, results suggest         keeping mRNAs, actively translated cytoplas- Cbc1-dependent translation inhibited osmotic stress,       mic eIF4E stress, kept P-bodies adapta- required rapid translation reprogramming stress     tion. similar scenario proposed CBP80-mediated <br /> condition.                                                                translation mammalian cells stress conditions provoked                                                                           prolonged hypoxia serum starvation (Oh et al., 2007a, 2007b). <br /> DISCUSSION                                                                stresses, steady-state translation mediated eIF4E response osmotic stress, global translation inhibited S.      drastically abolished, CBP80-mediated translation effi- <br /> cerevisiae cells. Nonetheless, osmostress-responsive mRNAs           ciently occurs. According model, level yeast Cbc1 pro- <br /> engage polysomes, corresponding stress-protective          tein remains constant, protein does modi- <br /> proteins efficiently synthesized (Uesono Toh, 2002; Melamed       fied osmotic stress (Figure S2). <br /> et al., 2008; Halbeisen Gerber, 2009; Warringer et al., 2010).            open question Cbc1-mediated translation results presented identify, time, mRNA-       mechanistically differs eIF4E-mediated translation. stated <br /> binding protein, yeast cap-binding protein Cbc1, involved          Cbc1 interact eIF4GI (Fortes et al., 2000), man- <br /> rapid translation reprogramming osmotic stress.                      ner similar interaction mammalian CBP80     involvement yeast Cbc1 translation matter        eIF4GI (McKendrick et al., 2001; Chiu et al., 2004).  debate past years  Introduction details).       mammalian cells, CBP80-eIF4GI interaction little functional <br /> present results favor direct role Cbc1 translation initiation,    relevance, new translation factor, CTIF, specifically involved essential cell viability normal conditions,     CBP80-dependent translation (Chiu et al., 2004; Lee et al., 2008; Cbc1 replaced eIF4E cap transcripts       Kim et al., 2009). yeast, additional studies conducted cytoplasm (Gao et al., 2005).  cbc1 mutants sensitive    determine Cbc1-mediated translation mechanism. <br /> translation inhibitor CHX (Figure 4A). Second, deletion            unresolved issue Cbc1 mediates transla- <br /> CBC1 gene cells carrying thermosensitive allele eIF4E             tion mRNA Cbc1-dependent translation restricted <br /> (cdc33ts) confers synthetic sickness temperature (Figure 4B).    specific transcripts, osmostress-responsive mRNAs. functional interaction eIF4E Cbc1 demonstrated        shown rapid translation newly synthesized endoge- synergistic CHX hypersensibility cdc33ts cbc1&#206;&#8221; double       nous osmoresponsive mRNAs osmotic stress conditions  <br /> mutant (Figure S1A).  Cbc1 cosediments polysomes          pends Cbc1, does translation osmotic stress&#226;&#8364;&#8220;induced <br /> CHX-dependent  e., translation-dependent) manner. Ribosome-             STL1-LacZ fusion (Figure 7). Finally, modulation translation  <br /> associated Cbc1 concentrated subpolysomal fractions          der osmotic stress Cbc1 imply direct role  <br /> volves small fraction Cbc1 protein (15&#226;&#8364;&#8220;20%; Figures 6       ment mRNAs P-bodies polysomes. Consistently, P-bodies S4).                                                                  remain longer fewer polysomes observed cbc1&#206;&#8221; mu- <br />     data highlight important role played Cbc1-depen-           tant cells osmotic stress (Figure 2).  Cbc1 localized <br /> dent translation osmotic stress conditions.  cbc1&#206;&#8221;        cytoplasmic foci partially colocalize P-bodies cdc33ts <br /> mutant shows low viability hyperosmotic media (Figure 1). Sec-         cells translation abolished simultaneous treatment ond, decrease polysome content response osmotic          high temperature osmotic stress (Figure 5, B C). <br /> stress extensive cbc1&#206;&#8221; mutant wt cells,        hypersensitivity loss viability cbc1&#206;&#8221; mutant cells <br /> recovery normal translation profiles delayed cbc1&#206;&#8221; mu-       osmotic stress relate Cbc1&#226;&#8364;&#8482;s role translation  <br /> tant (Figure 3, B).  P-bodies formed global          programming.  rule Cbc1 functions <br /> translation inhibited osmotic shock remain longer peri-       response yeast osmotic stress. Recently cytoplasmic <br /> ods cbc1&#206;&#8221; mutants wt cells (Figure 3C), indicating defect    membrane-anchored version Hog1 unable induce  <br />  <br /> 146&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                               Molecular Biology Cell <br />  transcription osmostress-responsive genes showed growth                  Strain       Relevant genotype                   Source <br /> defects hyperosmotic media, led proposal regulation transcription Hog1 essential yeast             BY4741       MATa his3&#206;&#8221;1 leu2&#206;&#8221;0                  Euroscarf <br /> survival osmotic stress (Westfall et al., 2008).  cyto-                       met15&#206;&#8221;0 ura3&#206;&#8221;0 <br /> plasmic Hog1 able regulate translation;            PAY611a      CBC1-GFP::KanMX6                    study know important regulation cell survival.             6566                                                                                             cbc1&#206;&#8221;::KanMX4                       Euroscarf <br />     translation mediated mammalian CBP80  <br /> sistant stress conditions, example, heat shock, pro-            2724                                                                                             hog1&#206;&#8221;::KanMX4                       Euroscarf <br /> longed hypoxia, serum starvation (Maquat Li, 2001; Mar&#195;&#173;n-               6214a        upf1&#206;&#8221;::KanMX4                       Euroscarf <br /> Vinader et al., 2006; Oh et al., 2007a, 2007b). yeast Cbc1             4268                                                                                             npl3&#206;&#8221;::KanMX4                       Euroscarf involved translation reprogramming stress condi- <br /> tions addressed.                                                     4540a        xrn1&#206;&#8221;::KanMX4                       Euroscarf <br />     fact osmo-mRNA levels exhibit greater prolonged             PAY664                                                                                               xrn1&#206;&#8221; CBC1-GFP::His3MX6             study <br /> increase cbc1&#206;&#8221; wt osmotic stress (Figure 2)          yRP840       MATa his4, leu2, trp1,              Schwartz Parker <br /> suggests defect transcription shutdown mRNA                              ura3, cup1::LEU2/PGK1pG/            (1999) <br /> stabilization occurs initial increase transcription                    MFA2pG <br /> stability (Molin et al., 2009; Romero-Santacreu et al., 2009). Future <br /> studies reveal Cbc1 plays role cytoplasmic mRNA              PAY671b      CBC1-GFP::KanMX6                    study <br /> decay cross-talk translation               yRP1321b     cdc33::LEU2 (cdc33-42/              Schwartz Parker transcription mediated Cbc1. Recently Rpb4/7 subunits                            TRP1)                               (1999) polymerase II postulated mRNA coordinators                PAY673b      cdc33::LEU2 (cdc33-42/              study <br /> couple transcription, mRNA decay, translation (Goler-Baron et                           TRP1) CBC1-GFP::KanMX6 <br /> al., 2008; Harel-Sharvit et al., 2010). involvement nuclear <br />                                                                                PAY678b      cbc1&#206;&#8221;::KanMX6                       study <br /> Cbc1 transcription (Das et al. 2000; Wong et al., 2007; Lahudkar <br />                                                                                         b <br /> et al., 2011), mRNA splicing (Lewis et al., 1996), nuclear degra-          PAY679       cdc33::LEU2 (cdc33-42/              study <br /> dation (Das et al., 2003), cytoplasmic Cbc1 translation                           TRP1) cbc1&#206;&#8221;::KanMX6 <br /> (Gao et al., 2005; work) mRNA decay, suggest                                                                                          Strain derivative EUROSCARF wt BY4741 strain. Cbc1 mRNA coordinators connecting            b <br />                                                                                Strain derivative wt yRP840 strain. fate mRNA cellular circumstances. <br />                                                                            TABLE 1:&#226;&#8364;&#8218; S. cerevisiae strains used work. <br />  <br /> MATERIALS METHODS <br /> Yeast strains growth conditions                                        plates. incubation times temperature indicated genotypes strains used study listed Table 1.     case. strains containing CBC1 gene deletion express- <br /> ing Cbc1 protein C-terminally tagged GFP constructed             Flow cytometry <br /> following PCR-based gene modification method described              Yeast cells grown exponential phase YPD Longtine et al. (1998). Purified PCR products used yeast          subjected osmotic stress adding 0.6 M KCl 1.2 M <br /> transformation lithium acetate method (Gietz et al., 1995),         NaCl, kept YPD control condition. transformants selected plating appropriate selec-           stress incubation, 100 &#206;&#188;l culture harvested centrifu- <br /> tive-medium plates.                                                        gation 13,000 rpm 30 s, cells resuspended 400 <br />     NaCl tolerance screening growing yeast strains          &#206;&#188;l PBS buffer (140 mM NaCl, 40 mM Na2HPO4, 2.7 mM KCl, 1.8 <br /> 220 &#206;&#188;l YPD medium (1% yeast extract, 2% peptone, 2% glucose)            mM KH2PO4). identify dead cells, propidium iodide 15 &#206;&#188;M (PI) 96 microtiter plates 16 h. Cultures diluted          dye added, fluorescence measured flow cytom- <br /> 5000 times cultures 215 &#206;&#188;l YPD plus NaCl 0, 0.15,        eter (EPICS XL-MCL; Beckman Coulter, Brea, CA) 5 min  <br /> 0.25, 0.4, 0.55, 0.75, 1, 1.25 M. Plates incubated 30&#194;&#176;C        cubation room temperature. yeast cells kept YPD 17&#226;&#8364;&#8220;24 h, OD620 (optical density) measured ab-            used 100% viability control. fluorescence cells  <br /> sorbance microplate reader (SpectraFluor; Tecan, M&#195;&#164;nnedorf,                der control conditions PI used baseline. <br /> Switzerland). IC50 (concentration NaCl required inhibit 50% growth relation maximal growth observed          Polysomal fractionation analysis specific RNAs <br /> strain YPD NaCl) calculated using OriginLab             proteins fractions <br /> software (Northampton, MA).                                                Cells grown exponentially YPD media OD600 0.5     Thermosensitive mutants grown 25&#194;&#176;C               subjected 0.6 M KCl stress indicated times. portion transferred permissive (25&#194;&#176;C 33&#194;&#176;C, indicated) restric-     culture (80 ml) recovered chilled 5 min ice tive (37&#194;&#176;C) temperature. Hyperosmotic stress provoked liq-        presence 0.1 mg/ml CHX. Cells harvested centrifugation <br /> uid medium adding KCl NaCl cultures obtain indi-           6000 &#195;&#8212; g 4 min 4&#194;&#176;C resuspended lysis buffer (20 mM <br /> cated final salt concentration. test osmotic stress sensitivity    Tris-HCl, pH 8, 140 mM KCl, 5 mM MgCl2, 0.5 mM dithiothreitol, 1% <br /> solid medium, YPD supplemented NaCl (0.4, 0.8, 1.2 M)          Triton X-100, 0.1 mg/ml CHX, 0.5 mg/ml heparin). sorbitol (1 1.5 M). CHX sensitivity tests performed        washed, cells resuspended 700 &#206;&#188;l lysis buffer, 0.3-ml solid YPD containing CHX indicated concentration.            volume glass beads added, cells disrupted vor- tests growth solid media exponentially           texing times 30 s. Lysates cleared centri- <br /> growing cultures (OD600 = 0.5), 5- 10-fold serial dilutions        fuged 5000 rpm 5 min, supernatant recov-  8 &#206;&#188;l dilution spotted indicated          ered centrifuged 8000 rpm 5 min. Finally, glycerol  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                            Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 147&#226;&#8364;&#402; <br />  added supernatant final concentration 5%        &#206;&#177;-Cbc1 (1:20,000 dilution; kindly provided D. G&#195;&#182;rlich) extracts stored &#226;&#710;&#8217;70&#194;&#176;C. Samples 10&#226;&#8364;&#8220;20 A260 units           &#206;&#177;-Rpl5 (1:5000 dilution; kindly provided J. L. Woolford). sec- <br /> loaded 10&#226;&#8364;&#8220;50% sucrose gradients separated ul-            ondary antibody anti-rabbit (1:25,000 dilutions; Amersham). <br /> tracentrifugation 2 h 40 min 35,000 rpm Beckman           Membranes developed using ECL Advance Typhoon <br /> SW41 rotor 4&#194;&#176;C. Gradients fractionated using isotonic        scanner (GE Healthcare). Images quantified using Image <br /> pumping 60% sucrose  followed            Gauge software (Fujifilm). <br /> recording polysomal profiles online UV detection 260 nm (Density Gradient Fractionation  Teledyne Isco, Lin-        &#206;&#178;-Galactosidase assays <br /> coln, NE). runoff experiments, cultures incubated        wt cbc1&#206;&#8221; strains containing STL1-LacZ fusion multicopy <br /> 20 min 1 mM NaN3 cells collected, CHX            plasmid (STL1 promoter fused LacZ, LEU2+ [2 &#206;&#188;m]; kind gift <br /> MgCl2 omitted lysis buffer gradient solu-        E. Nadal) grown exponential phase selec- <br /> tions. analysis RNA polysomal fractions, 10 mM          tive media. &#206;&#178;-galactosidase assays, 3 ml culture, ei- <br /> Tris-HCl, 10 mM EDTA 0.5% SDS added fractions.            ther untreated subjected indicated conditions, recov- <br /> Afterward, extractions phenol (pH 7.9):chloroform (1:1)          ered centrifugation. Cells washed resuspended prepared, RNA precipitated ethanol-sodium ac-           500 &#206;&#188;l buffer (16.1 g/l Na2HPO4&#194;&#183;7H2O, 5.5 g/l NaH2PO4&#194;&#183;<span id='am-7' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-8' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span>, <br /> etate. Specific RNAs analyzed Northern blotting. analy-       0.75 g/l KCl, 0.246 g/l MgSO4&#194;&#183;7H2O, pH 7.0). permeabilize cells, <br /> sis protein content fractions, proteins precipitated   50 &#206;&#188;l sample kept liquid N2 1 min heated 10% TCA twice washed cold anhydrous acetone. Dry              37&#194;&#176;C 1 min. start reaction, 600 &#206;&#188;l buffer containing <br /> pellets resuspended SB 1X (360 mM Tris-HCl, 18% SDS,              37.4 mM &#206;&#178;-mercaptoethanol 160 &#206;&#188;l 4 mg/ml ortho- <br /> 60% glycerol, 24% &#206;&#178;-mercaptoethanol, 0.12% bromophenol blue).             nitrophenyl-&#206;&#178;-galactoside added permeabilized cell extracted proteins analyzed Western blotting            sample. Samples incubated 30&#194;&#176;C appearance scribed Western blot analysis.                                         yellow color, reaction stopped adding 400 &#206;&#188;l <br />                                                                           1M Na2CO3. absorbance 420 nm measured super- <br /> Total RNA extraction Northern blotting                                natants. &#206;&#178;-galactosidase units calculated previously  analysis cellular mRNA levels, exponentially growing       scribed (Miller, 1972). results shown mean SE cultures incubated indicated conditions times,        independent experiments. cells 20-ml culture collected centrifugation frozen &#226;&#710;&#8217;20&#194;&#176;C. Total RNA extracted cell pellets using      Fluorescence microscopy FastPrep device (FastPrep 120, Bio101 ThermoSavant; Qbiogene,           Cells expressing Dcp2-GFP Dcp2-RFP plasmids (Sheth Illkirch, France) 0.5 ml LETS buffer (0.1 M LiCl, 0.01 M EDTA,         Parker, 2003) Cbc1-GFP CBC1 genomic locus pH 8.0, 0.01 M Tris-HCl, pH 7.4, 0.2% SDS [wt/vol]), 0.5 ml           grown overnight YPD synthetic complete (SC) medium sup- <br /> phenol (pH 4.5):chloroform:isoamylic alcohol (25/24/1; vol/vol/vol),      plemented appropriate amino acids, respectively. Cells 0.3 ml glass beads. Supernatants extracted                  transferred YPD grown OD600 0.5&#226;&#8364;&#8220;0.6. <br /> phenol:chloroform:isoamylic alcohol (25/24/1; vol/vol/vol)            Cells washed resuspended SC plus amino acids, chloroform:isoamylic alcohol (24/1; vol/vol). RNA precipitated        salt 1M KCl, indicated. Observations adding volumes 96% ethanol 0.1 volumes 5 M LiCl          indicated times using Axio Imager Z1 fluorescence microscope incubating &#226;&#710;&#8217;20&#194;&#176;C 3 h. Isolated RNA        (Zeiss, Tokyo, Japan), FV1000  FV10-ASW 2.1 <br /> polysomal fractions total RNA separated electrophore-         Viewer software colocalization experiments. <br /> sis denaturing 1% (wt/vol) agarose gel (2.2 M <span id='am-12' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>, 1X <br /> MOPS [20 mM 3-(N-morpholino)-propanesulfonic acid, 8 mM NaAc, <br />                                                                           ACKNOWLEDGMENTS <br /> 1 mM EDTA, pH 7.0]), blotted Hybond N+ membranes <br />                                                                           thank D. G&#195;&#182;rlich anti-Cbc1 antibody, J. L. Woolford (Amersham, GE Healthcare, Waukesha, WI; Sambrook Russell, <br />                                                                           anti-Rpl5 antibody, R. Parker cdc33ts strain Dcp2- <br /> 2001). Ethidium bromide-stained gels read phosphorim- <br />                                                                           GFP plasmid, E. Nadal STL1-LacZ plasmid. thank <br /> ager scanner (FLA-3000; Fujifilm, Tokyo, Japan) quantify sig- <br />                                                                           F. Carrasco lab technical assistance,  Flores Flow Cy- <br /> nal 18S 25S rRNA (rRNA). membrane hybridized <br />                                                                           tometry Service SCSIE help yeast viability experiments, 24 h 65&#194;&#176;C 0.5 M sodium phosphate (pH 7.2), 7% SDS <br />                                                                           J. Warringer helping initial polysome experiments. <br /> (wt/vol), 1 mM EDTA (pH 7.0). DNA probes obtained                                                                           thank J. E. P&#195;&#169;rez-Ort&#195;&#173;n helpful discussions critical <br /> PCR labeled random priming using Ready kit <br />                                                                           reading manuscript. work supported grants <br /> (Amersham) [33P]<span id='am-17' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="dCTP" datatype="xsd:string"></span>dCTP</span>. hybridization, membranes                                                                           BFU2008-02114 (Ministerio Espa&#195;&#177;ol Ciencia cofinanciado washed 10 min 25&#194;&#176;C 1X saline sodium citrate (SSC), <br />                                                                           Fondos FEDER) GV05/051 Prometeo 2011/088 (Generalitat <br /> 0.1% SDS, 15 min 65&#194;&#176;C 0.5X SSC , 0.1% SDS. <br />                                                                           Valenciana&#226;&#8364;&#8220;Regional Valencian Government) P  2010-4645 <br /> Membranes exposed imaging plate (BAS-MP; Fujifilm), <br />                                                                           (Swedish Research Council) UNICELLSYS, LSHG-CT2007-201142 read phosphorimager scanner (FLA-3000; Fujifilm). <br />                                                                           (European Commission FP7) P.S. visit P  P.S.&#226;&#8364;&#8482;s lab                                                                           supported grant European Science Foundation, &#226;&#8364;&#339;Fron- <br /> Western blot analysis <br />                                                                           tiers Functional Genomics&#226;&#8364;? Activity. L.R.-S. supported analyze total cellular content Cbc1 protein non- <br />                                                                           F.P  fellowship Spanish Ministry Science N.D.C. stress osmostress conditions, followed protein extrac- <br />                                                                           Santiago Grisol&#195;&#173;a Fellowship Generalitat Valenciana. <br /> tion protocol detailed Alepuz et al. (2001). total protein ex- <br /> tracts protein extracts polysomal fractions                                                                           REFERENCES <br /> resolved 10% SDS&#226;&#8364;&#8220;PAGE. Gels transferred nitro-          Alepuz PM, Jovanovic  Reiser V, Ammerer G (2001). Stress-induced map <br /> cellulose membrane (Protran; Whatman, Piscataway, NJ)                    kinase Hog1 transcription activation complexes. Mol Cell 7, <br /> blocked 5% nonfat milk overnight 4&#194;&#176;C. antibodies utilized          767&#226;&#8364;&#8220;777. <br />  <br />  <br /> 148&#226;&#8364;&#402; |&#226;&#8364;&#402; E. Garre et al.	                                                                                                Molecular Biology Cell <br />  Andrei MA, Ingelfinger D, Heintzmann R, Achsel T, Rivera-Pomar R,                   Hossain MA, Claggett JM, Nguyen T, Johnson TL (2009). cap binding <br />     Luhrmann R (2005). role eIF4E eIF4E-transporter targeting              complex influences H2B ubiquitination facilitating splicing     mRNPs mammalian processing bodies. RNA 11, 717&#226;&#8364;&#8220;727.                              SUS1 pre-mRNA. RNA 15, 1515&#226;&#8364;&#8220;1527. <br /> Baron-Benhamou J, Fortes P, Inada T, Preiss T, Hentze MW (2003).                Hoyle NP, Castelli LM, Campbell SG, Holmes LE, Ashe MP (2007). Stress- <br />     interaction cap-binding complex (CBC) eIF4G dispensable              dependent relocalization translationally primed mRNPs cytoplas- <br />     translation yeast. RNA 9, 654&#226;&#8364;&#8220;662.                                           mic granules kinetically spatially distinct P-bodies. J <br /> Bilsland-Marchesan E, Arino J, Saito H, Sunnerhagen P, Posas F (2000). Rck2             Cell Biol 179, 65&#226;&#8364;&#8220;74. <br />     kinase substrate osmotic stress-activated mitogen-activated        Ishigaki Y, Li X, Serin G, Maquat LE (2001). Evidence pioneer round <br />     protein kinase Hog1. Mol Cell Biol 20, 3887&#226;&#8364;&#8220;3895.                                   mRNA translation: mRNAs subject nonsense-mediated decay Bond U (2006). Stressed  Effects environmental stress mRNA                    mammalian cells bound CBP80 CBP20. Cell 106, 607&#226;&#8364;&#8220;617. <br />     metabolism. FEMS Yeast Res 6, 160&#226;&#8364;&#8220;170.                                          Izaurralde E, Lewis J, Gamberi C, Jarmolowski  McGuigan C, Mattaj IW <br /> Brengues M, Parker R (2007). Accumulation polyadenylated mRNA,                       (1995). cap-binding protein complex mediating U snRNA export. <br />     Pab1p, eIF4E, eIF4G P-bodies Saccharomyces cerevisiae.                  Nature 376, 709&#226;&#8364;&#8220;712. <br />     Mol Biol Cell 18, 2592&#226;&#8364;&#8220;2602.                                                    Keene JD (2007). RNA regulons: coordination post-transcriptional events. <br /> Buchan JR, Parker R (2009). Eukaryotic stress granules: ins outs             Nat Rev Genet 8, 533&#226;&#8364;&#8220;543. <br />     translation. Mol Cell 36, 932&#226;&#8364;&#8220;941.                                              Kim KM, Cho H, Choi K, Kim J, Kim BW, Ko YG, Jang SK, Kim YK (2009). <br /> Chiu SY, Lejeune F, Ranganathan AC, Maquat LE (2004). pioneer                       new MIF4G domain-containing protein, CTIF, directs nuclear cap- <br />     translation initiation complex functionally distinct structurally       binding protein CBP80/20-dependent translation. Genes Dev 23, <br />     overlaps steady-state translation initiation complex. Genes Dev            2033&#226;&#8364;&#8220;2045. <br />     18, 745&#226;&#8364;&#8220;754.                                                                    Kuai L, Das B, Sherman F (2005). nuclear degradation pathway controls <br /> Colot HV, Stutz F, Rosbash M (1996). yeast splicing factor Mud13p              abundance normal mRNAs Saccharomyces cerevisiae. Proc <br />     commitment complex component corresponds CBP20, small                    Natl Acad Sci USA 102, 13962&#226;&#8364;&#8220;13967. <br />     subunit nuclear cap-binding complex. Genes Dev 10, 1699&#226;&#8364;&#8220;1708.            Kuperwasser N, Brogna S, Dower K, Rosbash M (2004). Nonsense-mediated <br /> Das B, Guo  Russo P, Chartrand P, Sherman F (2000). role nuclear               decay does occur yeast nucleus. RNA 10, 1907&#226;&#8364;&#8220;1915. <br />     cap binding protein Cbc1p yeast mRNA termination degrada-             Lackner DH, B&#195;&#164;hler J (2008). Translational control gene expression     tion. Mol Cell Biol 20, 2827&#226;&#8364;&#8220;2838.                                                  transcripts transcriptomes. Int Rev Cell Mol Biol 271, 199&#226;&#8364;&#8220;251. <br /> Das B, Butler JS, Sherman F (2003). Degradation normal mRNA               Lahudkar S, Shukla  Bajwa P, Durairaj G, Stanojevic N, Bhaumik SR (2011). <br />     nucleus Saccharomyces cerevisiae. Mol Cell Biol 23, 5502&#226;&#8364;&#8220;5515.                   mRNA cap-binding complex stimulates formation pre- Nadal E, Alepuz PM, Posas F (2002). Dealing osmostress                  initiation complex promoter interaction Mot1p vivo. <br />     MAP kinase activation. EMBO Rep 3, 735&#226;&#8364;&#8220;740.                                         Nucleic Acids Res 39, 2188&#226;&#8364;&#8220;2209. Nadal E, Posas F (2010). Multilayered control gene expression              Lee HC, Cho H, Kim YK (2008). Ectopic expression eIF4E-transporter <br />     stress-activated protein kinases. EMBO J 29, 4&#226;&#8364;&#8220;13.                                  triggers movement eIF4E P-bodies, inhibiting steady-state <br /> Estrella LA, Wilkinson MF, Gonzalez CI (2009). shuttling protein Npl3               translation pioneer round translation. Biochem Biophys <br />     promotes translation termination accuracy Saccharomyces cerevisiae.              Res Commun 369, 1160&#226;&#8364;&#8220;1165. <br />     J Mol Biol 394, 410&#226;&#8364;&#8220;422.                                                        Leeds P, Peltz SW, Jacobson  Culbertson MR (1991). product <br /> Ferraiuolo MA, Basak S, Dostie J, Murray EL, Schoenberg DR, Sonenberg                   yeast UPF1 gene required rapid turnover mRNAs <br />     N (2005). role eIF4E-binding protein 4E-T P-body formation             containing premature translational termination codon. Genes Dev <br />     mRNA decay. J Cell Biol 170, 913&#226;&#8364;&#8220;924.                                           5, 2303&#226;&#8364;&#8220;2314. <br /> Fortes P, Inada T, Preiss T, Hentze MW, Mattaj IW, Sachs AB (2000).             Lejeune F, Ishigaki Y, Li X, Maquat LE (2002). exon junction complex     yeast nuclear cap binding complex interact translation factor              detected CBP80-bound eIF4E-bound mRNA mammalian <br />     eIF4G mediate translation initiation. Mol Cell 6, 191&#226;&#8364;&#8220;196.                      cells: dynamics mRNP remodeling. EMBO J 21, 3536&#226;&#8364;&#8220;3545. <br /> Fortes P, Kufel J, Fornerod M, Polycarpou-Schwarz M, Lafontaine D,                  Lewis JD, Gorlich D, Mattaj IW (1996). yeast cap binding protein complex <br />     Tollervey D, Mattaj IW (1999). Genetic physical interactions involving          (yCBC) acts early step pre-mRNA splicing. Nucleic Acids Res 24, <br />     yeast nuclear cap-binding complex. Mol Cell Biol 19, 6543&#226;&#8364;&#8220;6553.                 3332&#226;&#8364;&#8220;3336. <br /> Gao Q, Das B, Sherman F, Maquat LE (2005). Cap-binding protein 1-medi-              Longtine MS, McKenzie  III, Demarini DJ, Shah NG, Wach  Brachat  <br />     ated eukaryotic translation initiation factor 4E-mediated pioneer               Philippsen P, Pringle JR (1998). Additional modules versatile eco- <br />     rounds translation yeast. Proc Natl Acad Sci USA 102, 4258&#226;&#8364;&#8220;4263.              nomical PCR-based gene deletion modification Saccharomyces <br /> Gietz RD, Schiestl RH, Willems AR, Woods RA (1995). Studies                      cerevisiae. Yeast 14, 953&#226;&#8364;&#8220;961. <br />     transformation intact yeast cells LiAc/SS-DNA/PEG procedure.          Lui J, Campbell SG, Ashe MP (2010). Inhibition translation initiation fol- <br />     Yeast 11, 355&#226;&#8364;&#8220;360.                                                                  lowing glucose depletion yeast facilitates rationalization mRNA <br /> Goler-Baron V, Selitrennik M, Barkai O, Haimovich G, Lotan R, Choder M                  content. Biochem Soc Trans 38, 1131&#226;&#8364;&#8220;1136. <br />     (2008). Transcription nucleus mRNA decay cytoplasm            Maquat LE, Li X (2001). Mammalian heat shock p70 histone H4 tran- <br />     coupled processes. Genes Dev 22, 2022&#226;&#8364;&#8220;2027.                                     scripts, derive naturally intronless genes, immune G&#195;&#182;rlich D, Kraft R, Kostka S, Vogel F, Hartmann E, Laskey RA, Mattaj IW,                nonsense-mediated decay. RNA 7, 445&#226;&#8364;&#8220;456. <br />     Izaurralde E (1996). Importin provides link nuclear protein           Mar&#195;&#173;n-Vinader L, van Genesen ST, Lubsen NH (2006). mRNA     import U snRNA export. Cell 87, 21&#226;&#8364;&#8220;32.                                          heat shock enters round translation. Biochim Biophys Acta <br /> G&#195;&#182;rnemann J, Kotovic KM, Hujer K, Neugebauer KM (2005). Cotranscrip-                    1759, 535&#226;&#8364;&#8220;542. <br />     tional spliceosome assembly occurs stepwise fashion requires           McKeehan W, Hardesty B (1969). mechanism cycloheximide inhibi- <br />     cap binding complex. Mol Cell 19, 53&#226;&#8364;&#8220;63.                                        tion protein synthesis rabbit reticulocytes. Biochem Biophys Res <br /> Greatrix BW, van Vuuren HJ (2006). Expression HXT13, HXT15                   Commun 36, 625&#226;&#8364;&#8220;630. <br />     HXT17 genes Saccharomyces cerevisiae stabilization                McKendrick L, Thompson E, Ferreira J, Morley SJ, Lewis JD (2001). Inter- <br />     HXT1 gene transcript sugar-induced osmotic stress. Curr Genet 49,                action eukaryotic translation initiation factor 4G nuclear <br />     205&#226;&#8364;&#8220;217.                                                                            cap-binding complex provides link nuclear cytoplasmic <br /> Gross T, Siepmann  Sturm D, Windgassen M, Scarcelli JJ, Seedorf M,                    functions m(7) <span id='am-2' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="guanosine" datatype="xsd:string"></span>guanosine</span> cap. Mol Cell Biol 21, 3632&#226;&#8364;&#8220;3641. <br />     Cole CN, Krebber H (2007). DEAD-box RNA helicase Dbp5 func-                 Melamed D, Pnueli L, Arava Y (2008). Yeast translational response high <br />     tions translation termination. Science 315, 646&#226;&#8364;&#8220;649.                             salinity: global analysis reveals regulation multiple levels. RNA 14, <br /> Halbeisen  Gerber AP (2009). Stress-dependent coordination tran-                  1337&#226;&#8364;&#8220;1351. <br />     scriptome translatome yeast. PLoS Biol 7, e105.                          Miller C et al. (2011). Dynamic transcriptome analysis measures rates Harel-Sharvit L, Eldad N, Haimovich G, Barkai O, Duek L, Choder M (2010).               mRNA synthesis decay yeast. Mol Syst Biol 7, 458. <br />     RNA polymerase II subunits link transcription mRNA decay trans-          Miller J (1972). Experiments Molecular Genetics, New York: Cold Spring <br />     lation. Cell 143, 552&#226;&#8364;&#8220;563.                                                          Harbor Laboratory. <br /> Hilgers V, Teixeira D, Parker R (2006). Translation-independent inhibition       Moldave K (1985). Eukaryotic protein synthesis. Annu Rev Biochem 54, <br />     mRNA deadenylation stress Saccharomyces cerevisiae. RNA                   1109&#226;&#8364;&#8220;1149. <br />     12, 1835&#226;&#8364;&#8220;1845.                                                                  Molin C, Jauhiainen  Warringer J, Nerman O, Sunnerhagen P (2009). <br /> Hohmann S (2002). Osmotic stress signaling osmoadaptation yeasts.                mRNA stability changes precede changes steady-state mRNA <br />     Microbiol Mol Biol Rev 66, 300&#226;&#8364;&#8220;372.                                                 amounts hyperosmotic stress. RNA 15, 600&#226;&#8364;&#8220;614. <br />  <br />  <br /> Volume 23&#226;&#8364;&#402; January 1, 2012	                                                                                   Cbc1-mediated translation osmostress&#226;&#8364;&#402; |&#226;&#8364;&#402; 149&#226;&#8364;&#402; <br />  O&#226;&#8364;&#8482;Rourke SM, Herskowitz (2004). Unique redundant roles HOG             Sheth U, Parker R (2003). Decapping decay messenger RNA occur    MAPK pathway components revealed genome expression                    cytoplasmic processing bodies. Science 300, 805&#226;&#8364;&#8220;808. <br />    analysis. Mol Biol Cell 15, 532&#226;&#8364;&#8220;542.                                          Spriggs KA, Bushell M, Willis AE (2010). Translational regulation gene <br /> Oh N, Kim KM, Cho H, Choe J, Kim YK (2007a). Pioneer round translation            expression conditions cell stress. Mol Cell 40, 228&#226;&#8364;&#8220;237. <br />    occurs serum starvation. Biochem Biophys Res Commun 362,               Teige M, Scheikl E, Reiser V, Ruis H, Ammerer G (2001). Rck2, member <br />    145&#226;&#8364;&#8220;151.                                                                          calmodulin-protein kinase family, links protein synthesis high <br /> Oh N, Kim KM, Choe J, Kim YK (2007b). Pioneer round translation                   osmolarity MAP kinase signaling budding yeast. Proc Natl Acad Sci <br />    mediated nuclear cap-binding proteins CBP80/20 occurs                   USA 98, 5625&#226;&#8364;&#8220;5630. <br />    prolonged hypoxia. FEBS Lett 581, 5158&#226;&#8364;&#8220;5164.                                  Teixeira D, Sheth U, Valencia-S&#195;&#161;nchez MA, Brengues M, Parker R (2005). <br /> Parker R, Sheth U (2007). P bodies control mRNA translation           Processing bodies require RNA assembly contain nontranslating <br />    degradation. Mol Cell 25, 635&#226;&#8364;&#8220;646.                                                mRNAs. RNA 11, 371&#226;&#8364;&#8220;382. <br /> Posas F, Chambers JR, Heyman JA, Hoeffler JP, Nadal E, Ari&#195;&#177;o J (2000).        Uesono Y, Toh-e (2002). Transient inhibition translation initiation    transcriptional response yeast saline stress. J Biol Chem 275,          osmotic stress. J Biol Chem 277, 13848&#226;&#8364;&#8220;13855. <br />    17249&#226;&#8364;&#8220;17255.                                                                  Van Der Kelen K, Beyaert R, Inze D, VL (2009). Translational control Ram&#195;&#173;rez M, Wek RC, Hinnebusch AG (1991). Ribosome association GCN2                eukaryotic gene expression. Crit Rev Biochem Mol Biol 44, 143&#226;&#8364;&#8220;168. <br />    protein kinase, translational activator GCN4 gene Saccharo-       Wang X, Proud CG (2007). Methods studying signal-dependent regula- <br />    myces cerevisiae. Mol Cell Biol 11, 3027&#226;&#8364;&#8220;3036.                                    tion translation factor activity. Methods Enzymol 431, 113&#226;&#8364;&#8220;142. <br /> Rep M, Albertyn J, Thevelein JM, Prior BA, Hohmann S (1999). Different sig-      Warringer J, Hult M, Regot S, Posas F, Sunnerhagen P (2010). HOG <br />    nalling pathways contribute control GPD1 gene expression             pathway dictates short-term translational response hyperos- <br />    osmotic stress Saccharomyces cerevisiae. Microbiology 145, 715&#226;&#8364;&#8220;727.            motic shock. Mol Biol Cell 21, 3080&#226;&#8364;&#8220;3092. <br /> Rep M, Krantz M, Thevelein JM, Hohmann S (2000). transcriptional             Westfall PJ, Patterson JC, Chen  Thorner J (2008). Stress resistance    response Saccharomyces cerevisiae osmotic shock. Hot1p                  signal fidelity independent nuclear MAPK function. Proc Natl Acad <br />    Msn2p/Msn4p required induction subsets high osmolar-            Sci USA 105, 12212&#226;&#8364;&#8220;12217. <br />    ity glycerol pathway-dependent genes. J Biol Chem 275, 8290&#226;&#8364;&#8220;8300.             Windgassen M, Sturm D, Cajigas IJ, Gonzalez CI, Seedorf M, Bastians H, <br /> Romero-Santacreu L, Moreno J, P&#195;&#169;rez-Ort&#195;&#173;n JE, Alepuz P (2009). Specific              Krebber H (2004). Yeast shuttling SR proteins Npl3p, Gbp2p, Hrb1p <br />    global regulation mRNA stability osmotic stress Sac-             translating mRNPs, Npl3p function transla- <br />    charomyces cerevisiae. RNA 15, 1110&#226;&#8364;&#8220;1120.                                         tional repressor. Mol Cell Biol 24, 10479&#226;&#8364;&#8220;10491. <br /> Sambrook J, Russell D (2001). Molecular Cloning: Laboratory Manual,            Wong CM, Qiu H, Hu C, Dong J, Hinnebusch AG (2007). Yeast cap binding <br />    New York: Cold Spring Harbor Laboratory.                                          complex impedes recruitment cleavage factor IA weak termination <br /> Schwartz DC, Parker R (1999). Mutations translation initiation factors lead       sites. Mol Cell Biol 27, 6520&#226;&#8364;&#8220;6531. <br />    increased rates deadenylation <br /> </body></html>