<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>       ReseaRch papeR <br />  <br /> Nucleus 3:3, 286-289; June 2012; &#194;&#169; 2012 Landes Bioscience <br />  <br />  <br />  <br />       Did modulation expression noise shape <br />        evolution dimensional genome <br />               organizations eukaryotes? <br />                                                                 Kuljeet singh sandhu <br />                                        Genome Technology Biology Group; Genome Institute singapore; singapore <br />  <br />  <br />       Keywords: chromatin interactions, expression noise, evolution, epigenetics, 3D genome organization, nuclear architecture <br />  <br />  <br />           pervasive role distant chromatin interactions transcriptional regulation increasingly evident.           possibility greater diversity chromatin interactions genomic locus contribute stochastic <br />           variation gene expression.  issue addressed.  present lines evidence,           suggest variation trans chromatin interactions occur cost expression noise. Genomic regions <br />           nucleosome depletion, abundant rapid transcription essential gene clusters exhibit relatively fewer <br />           trans chromatin interactions nucleus.  loci greater number interactions tend higher <br />           expression noise. Based observations, hypothesize dimensional organization eukaryotic <br />           genomes evolved selective pressure minimize expression noise essential gene clusters  <br />  <br />                     &#194;&#169; 2012 Landes Bioscience. <br />           nucleus. <br />  <br />  <br />  <br />                             Introduction                                       minimum. way locus experience variation  <br />  <br />                                        distribute. <br />                                                                                expression highly diverse chromatin interactions. Large num- <br /> Accumulating evidence suggests prominent role nonran-                   ber chromatin interactions locus generally occur dom dimensional organization chromatin essential                   time cell spatial constraints nucleus, <br /> genomic functions eukaryotic nucleus.1,2 earliest evi-              represent aggregation diverse interactions dence comes observation random insertion                  population cells. Highly varying interactions respect gene distinct genomic locations modulates transcription               time space allow gene maintain constant <br /> distinctly.3 Long range cis trans chromosomal interactions                 expression level. test hypothesis using genome wide inter- <br /> correlate strongly active repressive epigenetic states             chromosomal (trans) chromatin interaction data yeast.7 gene.1  spatial positioning genes nucleus matters expression. nuclear core associated                                             Results high transcription activity, nuclear lamina interac- <br /> tions chromosomal domains associated stable silenc-                Using trans chromatin interactions biological rep- <br /> ing genes.4,5 Recent reports suggest genes               licates,7 calculated normalized number interactions, e., independent transcriptional units  instead               degree (&#206;&#180;), 3 kb region yeast genome (Material <br /> interact extensive manner.6 Despite                Methods section). following key observations: <br /> advances, clear dimensional organiza-                  (1) apparent Figure 1A&#226;&#8364;&#8220;D, genomic regions tion genome evolved non-random conformation.                     low nucleosome occupancy, abundant expression rapid rate <br /> Transcriptional activity  inactivity) coordination genes             nascent transcription significantly lower degree trans arguably best studied constraints eukaryotic genome                 interactions regions moderate levels nucleosome <br /> organization.  present lines evidence, sug-              occupancy, gene expression transcription rate (p = 3.8e-13, <br /> gest modulation expression noise               1.3e-04, 3.2e-05 respectively), suggesting nucleosome evolutionary constraints, shaped                   depleted domains highly transcribing genes, selected dimensional conformation eukaryotic genome.                           high variation trans chromatin interactions. <br />     Highly varying expression levels genes deleterious             Similarly, regions low gene activity repressed genes happens essential gene loci              exhibit lower degree (p = 8.8e-05), presumably ensure <br /> evolutionary constraint shape 3D conformation genome                 stable silencing delicate regulation precise transcrip- manner expression noise essential loci              tional level. <br />  <br />  Correspondence  Kuljeet Singh Sandhu; Email: sandhuks@gis star.edu.sg kulsinsan@gmail.com <br />  Submitted: 02/04/12; Revised: 04/02/12; Accepted: 04/04/12 <br />  http://dx.doi.org/10.4161/nucl.20263 <br />  <br />  <br /> 286                                                                     Nucleus                                                      Volume 3 Issue 3 <br />        ReseaRch papeR                                                                                                                     ReseaRch papeR <br />  <br />  <br />  <br />  <br />                    &#194;&#169; 2012 Landes Bioscience. <br />                                        distribute. <br />  Figure 1.   Linear view degree trans chromatin interactions, nucleosome occupancy gene expression yeast genome. simplic- <br />  ity, 16 chromosomes concatenated increasing order. Rolling mean 10 consecutive windows 3 kb plotted track. highlighted  orange regions high degree identified using cut shown Figures S6 S7. (B&#226;&#8364;&#8220;e) Degree distributions genomic regions  high, moderate low (B) nucleosome occupancy, (c) gene expression, (D) transcription rate (mol/min) (e) density essential genes. detailed <br />  degree distribution distinct representations essential genes given Figure S1. (F) Distribution expression noise regions high <br />  low degree different cut-offs. Dashed horizontal line represents mean expression noise. p values (B&#226;&#8364;&#8220;F) calculated using Mann- <br />  Whitney U test. Ns, significant. <br />  <br />  <br />  <br />     (2) observations prompted explore essen-                    observations line recent report tial gene clusters, generally located nucleosome                   suggests inter-chromosomal DNA interactions occur depleted domains highly expressed,8 selected                cost increased expression noise.10 expression noise highly varying chromatin interactions. Lower degree                      data available far measured protein level gene <br /> domains higher density essential genes compared                 transcription, possibility noise gene transcription ones moderate density,  suggests essential                 partly compensated post-transcriptional level <br /> gene clusters preferentially located genomic domains                 regulatory mechanisms explored context, lower variability chromatin interactions (Fig. 1E; Fig. S1,               explain relatively weaker significance, observed p = 2.3e-03). Category regions essential gene com-                   Figure 1F.  alterations chromatin organization <br /> prises lowly expressed genes shows lower                   account intrinsic noise, expression <br /> degree.  propose essential gene clusters             noise data includes extrinsic noise contributed factors like evolutionary selected high variability                 fluctuating micro-environments cells. <br /> chromatin interactions, consequently high expression <br /> noise, nucleus.                                                                                        Discussion <br />     (3) test variation chromatin <br /> interactions modulates expression noise, analyzed expres-                 recent boom proximity ligation based techniques sion noise data available literature.9 obtained maximal            combination generation sequencing enabled expression noise 3 kb bin. shown Figure 1F,                  mapping chromatin interactions nucleus.7,11,12 <br /> regions higher degree positively associate high expres-               Interestingly, highly expressed housekeeping genes shown <br /> sion noise (p = 3e-02, 1e-02), supporting hypothesis               converge transcriptionally active multi-gene complexes <br /> diversity chromatin interactions contribute stochastic                efficient coordinated transcription.7 coordina- <br /> noise expression levels.                                                     tion transcription disturbed locus involved <br />  <br />  <br /> www.landesbioscience.com                                                 Nucleus                                                                         287 <br />  stochastically moves active compartment  worse       distinct restriction enzymes (EcoRI HindIII), used <br /> case, mis-fires repressed compartment.           calculate normalized degree chromatin interactions. <br /> loci expected relatively stable transcription         Nucleosome occupancy data taken Kaplan et al.18 foci sites, ones essential toler-      RNA-Seq data set taken Nagalakshmi et al.19 Data <br /> ate noisy expression.  highly varying expression level       nascent transcription rate taken Pelechano et al.20 proposed facilitator adaptive gene expres-          List essential genes taken Saccharomyces Genome <br /> sion evolution certain genes advantageous        Deletion Project (http://www-sequence.stanford.edu/group/ certain context.13  importantly, essential genes tend        yeast_deletion_project/downloads.html). Expression noise data low expression noise noisy expression essential        originally generated Newman et al.9 scaled 0 genes deleterious organism.8  order        1 Li et al.21 DNA double strand break data Dmc1&#206;&#8221; mutant minimize expression noise, essential genes evolved          taken Buhler et al. experimental data sets locate clusters chromosome harbor consis-           taken generated yeast cells grown YPD growth media <br /> tently open state chromatin, shown 8        respective studies. <br /> essential gene clusters maintain lower expression noise            Analysis.  yeast genome divided bins limiting total number interactions, possibly restricting    3 kb (equivalent average size EcoRI/HindIII fragments global mobility loci involved. Essential gene clusters       yeast genome). centers restriction fragments tethered nuclear sites abundant rapid tran-      mapped 3 kb bins. Data sets biological replicates (EcoRI <br /> scription. hypothesis strongly supported observa-         HindIII)   quantile normalized. quality  <br /> tion nucleosome depleted, highly rapidly transcribed           trol plots shown Figure S5A. biological <br /> regions lower degree trans, relatively higher degree        replicates, undirected network trans chromatin interactions cis chromatin interactions (Fig. 1A&#226;&#8364;&#8220;C; Figs. S2 S3).       3 kb bins constructed using igraph library (http:// notable maximal extent cis interactions locus      igraph.wikidot.com/) R-package (http://www.r-project.org/). <br />  <br />                 &#194;&#169; 2012 Landes Bioscience. far lower (3.4-fold) trans interactions, suggesting intra-territorial spatial constraints selectively restricting mobility genomic domains high gene activity. Similarly,                                                                         Number interactions (degree) 3 kb region calcu- <br />                                                                         lated using &#226;&#8364;&#732;degree&#226;&#8364;&#8482; function igraph. Geometric mean                                                                         replicates used analysis. Binning data cal- <br /> regions low   gene activity lower degree           culation geometric mean normalized EcoRI HindIII <br />  <br />  <br />                                     distribute. trans interactions hinting maintenance stable            data sets make analysis robust possible biases <br /> repressive state low delicate transcription limiting         caused length local GC content distal ends mobility locus.  attachment sub-nuclear          restriction fragments individual data sets (Fig. S5B C). <br /> structures like nucleoli nuclear periphery, associated     Rolling mean 10 consecutive windows taken plotting transcriptional repression, significantly limits mobil-        visualization purpose. yeast genes (S288C genome) ity chromatin domains.14 Conversely, regions mod-           assigned 3 kb bins mapping location transcription <br /> erate levels expression relatively greater degree trans      start sites (TSS) bins. Mean nucleosome occupancy, gene <br /> interactions susceptible higher expression noise.       expression nascent transcription rate calculated Higher expression noise moderately expressed genes          3 kb window. cut values degree, nucleosome occu- <br /> observed 15 Furthermore, reports suggest             pancy, gene expression transcription rate elaborated certain chromatin interactions deleterious        Figure S6. liberal stringent cut-offs alter mutagenic.16 extrapolation, region having higher number         conclusions article (Fig. S7). calculate essential gene <br /> trans chromatin interactions, risk experi-        density, neighboring windows 3 kb (12 kb total) encing erroneous interactions. Restricting total number        considered. High moderate density essential genes interactions minimize risk mutagenic          demarcated &gt; 3 1&#226;&#8364;&#8220;3 genes 12 kb region respec- <br /> interactions.  DNA double strand break signal strongly           tively. detailed degree distribution distinct representa- <br /> associates regions higher degree chromatin interac-      tions essential genes given Figure S1. expression <br /> tions (Fig. S4).  propose nonrandom          noise data available yeast genes, considered <br /> dimensional organization eukaryotic genome                maximum noise value available 3 kb bin. key findings <br /> evolved selection constraint reduce expression         study scrutinized possible sequence <br /> noise risk erroneous mutagenic interactions essen-      mapping biases removing telomeric (15 kb) centro- <br /> tial gene loci. Having said  rule evo-      meric (15 kb  regions analysis (Fig. S8A&#226;&#8364;&#8220;D). <br /> lutionary constraints like coordinated expression functionally        yeast chromosomes known tethered cen- <br /> related genes, reported 6,17 data sets       tromeres   abundant trans interactions sites <br /> future help scrutinize hypothesis.                            proximal centromeres (&#194;&#177; 50 kb, Fig. S8E). reassessed                                                                         major observations study excluding regions &#194;&#177; 50 kb <br />                     Material Methods                                centromere. analysis consistently adheres                                                                         hypothesis (Fig. S8F . Similarly, key observations Data sets. Genome wide chromatin interaction data taken             study robust variations replicates, Duan et al.7 Biological replicate data, generated using        measured using Shannon&#226;&#8364;&#8482;s entropy normalized EcoRI  <br />  <br /> 288                                                              Nucleus                                                     Volume 3 Issue 3 <br />  HindIII libraries (Figs. S9 S10). pair-wise comparisons                                                                         Acknowledgments distributions, Mann-Whitney U test used.                                        K.S.S. acknowledges financial support STAR <br /> statistical analysis performed R package (http://www.r-                                      Singapore Joint Council Office (JCO) grant (12302EG016). <br /> project.org/). <br />                                                                                                                                       Supplemental Material <br />                  Disclosure Potential Conflicts                                      Supplemental     materials                  potential conflicts disclosed.                                                  www.landesbioscience.com/journals/nucleus/article/20263 <br />                                                                   9.    Newman JR, Ghaemmaghami S, Ihmels J, Breslow DK,              16. Lin C, Y <br /> </body></html>