<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAUR ANID CELLULAR BIOLOGY, Jan. 1993, p. 341-350                                                                 Vol. 13,  1 <br /> 0270-7306/93/010341-10$02.00/0 <br /> Copyright &#194;&#169; 1993, American Society Microbiology <br />  <br />  <br />         Essential Yeast Gene Homology Exonuclease- <br />                Encoding XRNJ/KEM1 Gene Encodes                    Protein Exoribonuclease Activity <br />          MARGARET KENNA,1t AUDREY STEVENS,2 MARK McCAMMON,3 AD MICHAEL G. DOUGLAS`* <br />           Department Biochemistry Biophysics, University North Carolina Medical School, University                 North Carolina Chapel Hill, Chapel Hill, North Carolina 27599-72601; Biology Division, <br />                   Oak Ridge National Laboratory, Oak Ridge, Tennessee 37831-80772; Department <br />                          Biochemistry, Southwestern Graduate School Biomedical Sciences, <br />                              University Texas Health Science Center, Dallas, Texas 752353 <br />                         Received 2 July 1992/Returned modification 18 August 1992/Accepted 12 October 1992 <br />  <br />               essential gene, designated HKE1/RATI, isolated yeast Saccharomyces cerevisiae            characterized. gene encodes protein 116 kDa (p116) significant homology yeast gene <br />            (XRNI/KEMI) encoding related protein (p175) 5&apos;--&apos;3&apos; exonuclease activity activities involving <br />            chromosomal DNA pairing mechanics. Preliminary analysis hkel) mutant reveals precipitous <br />            decline translation mRNA nonpermissive temperature. Sporulation heterozygous <br />            HKE1/hkel::URA3 diploids reveals gene, unlike highly related XRN1/KEM1 gene, essential            cell viability. Overexpression homologous gene product, p175, failed rescue cells lacking functional <br />            p116. vitro studies demonstrate p116 protein 5&apos;- 3&apos; exoribonuclease activity, major activity <br />            related p175. immunoreactive RNase activity 116 kDa abolished antiserum p116. <br />            level protein RNase activity correlate HKEI gene dosage. RNase activity <br />            purifies coincidentally previously described 116-kDa RNase having 5&apos;- 3&apos; exoribonuclease activity. <br />  <br />    characterized temperature-sensitive (ts) mutant              order understand activities operate Saccharomyces cerevisiae obtained                    ensure correct processing transport RNA, screen mutants reduced synthesis binding             studies initiated yeasts advantage hybrid protein competes transport protein              genetic approaches available (3, 16). precursors mitochondria (8). Analysis mutant           studies, designed uncover RNA trafficking mutants led characterization gene significant primary          Saccharomyces cerevisiae, yielded temperature-sensi- <br /> sequence homology previously identified gene, XRNJ                 tive mutant complemented gene identical (25). XRNJ independently identified KEMI (18),              HKE1 (named RATl RNA trafficking [3]). study, <br /> SEP1 (42), DST2 (10), RARS (19). gene referred            poly  RNA localization methods used screen XRNI/KEMI paper. encodes protein 175                 conditional mutants appear blocked kDa (pl75) appears multitude properties,             transport RNA molecules nucleus restric- <br /> including involvement seemingly diverse functions               tive temperature. Mutants obtained screens (3, recombination, RNA processing turnover, microtu-                 16) appear distinct previously described mutants <br /> bule function, karyogamy, DNA replication  sum-               exhibit defects cytoplasmic mRNA production mary, reference 17). protein 5&apos;--3&apos; exoribonu-              mRNA splicing (26; review, reference 32). <br /> clease DNase activity DNA strand-exchange                  studies presented characteriza- <br /> activity (10, 25, 42). XRN1IKEM1 gene disruptions                  tion HKEJ/RAT1 gene hkeltS mutant shown nonessential gene. related gene                  pl16 protein having 5&apos;-33&apos; exoribonuclease activity, <br /> described report originally designated HKE1             major activity product related XRNJ/KEMI <br /> homology KEMI). Yeast cells lacking functional HKE1                  gene. viable rescued XRN1/KEMJ, high copy. Analysis hkel temperature-sensitive <br /> mutant shows reduction translation RNA                             MATERIALS METHODS <br /> nonpermissive temperature. Furthermore, p116 appears 5&apos;-.3&apos; exoribonuclease activity, does related                 Strains media. following yeast strains used <br /> p175 protein. nuclear immunolocalization p116 sug-               study: SEY2101 (MATa ura3-52 leu2-3,112 suc2-A9 <br /> gests participation utilization mRNA             ade2-1 gal2); SEY2102 (AL4Ta ura3-52 leu2-3,112 suc2-A9 <br /> occur prior translation mRNA cytoplasm.                his4-519 gal2) (11); GS8-44 (AL4Ta ura3-52 leu2-3,112 phenotypes associated hkel-l&apos;S indicate                suc2-A9 his4-519 gal2 hkel-l&apos;&apos;); W303 (ALATa ura3-52 <br /> role protein processing trafficking           leu2-3,112 his3-11,15 trpl-1 ade2-1); YPH500 (MATa <br /> nuclear mRNAs.                                                          ura3-52 lys2-801 ade2-101 trpl-A63 his3-A200 leu-Al);                                                                         YPH500xmnl (MATa ura3-52 lys2-801 ade2-101 trpl-A63 <br />                                                                         his3-A200 leu-Al xrn1ABglII::URA3) (25). <br />   * <br />       Corresponding author.                                               Yeast cells grown YPD rich medium (1% yeast <br />   t Present address: Howard Hughes Medical Institute, Duke Uni-         extract, 2% peptone, 2% glucose) minimal medium <br /> versity Medical Center, Durham, NC 27713.                               (YNBD [0.67% yeast nitrogen base, 2% glucose] YNBG <br />                                                                   341 <br />  342     KENNA ET AL.                                                                                          MOL. CELL. BIOL. <br />  <br /> [0.67% yeast nitrogen base, 2% glycerol]) supplemented        plasmids shared common region 4.1 kb appropriate amino acids (34). LSM used labeling            contained length gene. <br /> experiments (8).                                                      Complementation HKEI centromere plasmid.    Escherichia coli strain used plasmid construction       5.1-kb fragment bearing HKEI gene 5&apos; 3&apos; MC1066 (F- AlacX74 galU galK rpsL hsdR trpC9830                flanking DNA amplified polymerase chain reaction <br /> leuB600 pyrF74::TnS) (29). Bacteria grown LB               (PCR) primers flanking BamHI site YEp13 medium (30). Ampicillin-resistant transformants se-           plasmid pMKHKE1. insert cloned lected LB plates supplemented 50 ,ug ampicillin         BamHI site pRS316, CEN6 plasmid (35), trans- ml (30).                                                       formed GS8-44 derivatives. ability    Plasmids. plasmid, pMKHKE1, 2,um LEU2                gene complement ts mutation analyzed grow- <br /> YEp13-based plasmid containing 5.1-kb insert, including          ing cells containing plasmid 23 37&#194;&#176;C. entire HKEJIRATI gene, BamHI site  .             Integrative mapping. Integrative mapping used plasmid pExo2, YEp24-based plasmid Larimer              demonstrate gene cloned complementation Stevens (25), contains XRNI gene. -8-kb                 ts mutant mapped genetic location allele <br /> BamHI-HindIII fragment containing entire XRNJ gene             responsible temperature-sensitive phenotype. subcloned YEpl3 create YEp13XRN1.                      2.3-kb SalI fragment YEp44-7, comple- <br />    Isolation temperature-sensitive mutant. original     menting plasmids, subcloned yeast integrating <br /> temperature-sensitive strain GS8-44 (hkel-1) identified        vector YIp5. Integration plasmid directed ts mutant screen yeast cells designed detect     HKEJ region genome transforming wild-type strain mislocalization fusion protein vivo. Earlier studies   SEY2101 KpnI-linearized DNA. Correct integration demonstrated mitochondrial association              HKEI locus confirmed Southern analysis ATP2-lacZ-encoded hybrid protein mitochondrial <br /> inner membrane prevents growth nonfermentable carbon            genomic DNA prepared transformant. trans- <br /> sources wild-type host (8, 11). genetic          formed strain mated GS8-44 containing screen devised select conditional mutants        temperature-sensitive hkel-1 mutation. Diploids sporu- <br /> suppress growth phenotype hybrid protein        lated, Ura+ phenotype meiotically mapped harbor plasmid, assayed growth glycerol     relation hkel-1 allele. Tetrad analysis confirmed 13-galactosidase activity.                                     cloned sequence represents wild-type gene    Mutagenesis genetic analysis. Strains SEY2101           mutant allele hkel-1 strain. <br /> SEY2102, carrying plasmid pCIZ1  ATP2-lacZ                   Yeast transformation. Yeast cells competent <br /> plasmid [8]), mutagenized ethyl methane           transformation lithium acetate (14). <br /> sulfonate (34) UV irradiation. UV mutagenesis, cells           Nucleic acid methods. Standard procedures used treated cell density 107/ml H20            restriction digestions plasmid constructions. Restriction <br /> approximately 40% killing. mutagenesis, cells           enzymes T4 DNA ligase used according plated YNBG plus 5-bromo-4-chloro-3-indolyl-o-D-ga-           instructions supplier. DNA fragments purified <br /> lactopyranoside <span id='am-17' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> supplements 0.1%               agarose, DNA transformation E. coli glucose. Colonies growing plates 7 10 days       performed described Maniatis et al. (27). room temperature expressed observable 3-galacto-             HKE1 gene sequenced single-stranded <br /> sidase activity (30) selected study (approx-      double-stranded template. Dideoxy sequencing DNA <br /> imately 10&apos; survivors). Candidates subjected            accomplished using kit New England Biolabs, <br /> multiple rounds screening checked temper-          single-stranded DNA cloning vectors, universal primers. <br /> ature-sensitive growth 30 37&#194;&#176;C glucose-         Double-stranded plasmid DNA sequenced glycerol-containing media. screening eliminated ap-           Sequenase (United States Biochemical Corp.) double- <br /> proximately 85% candidates. confirm             stranded DNA cycling sequencing (Bethesda  <br /> lesion chromosome linked, original pC,Z1               search Laboratories) using Taq DNA polymerase. se- <br /> segregated screening <span id='am-79' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-80' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> prototrophy loss        quencing vector Bluescript (Stratagene) used 0-galactosidase expression. Strains retransformed          subclone certain restriction fragments HKE1, se- fresh pCIZ1 screened growth YNBG.                  quence obtained universal sequencing primers. <br /> Approximately 60% final candidates lost           Oligonucleotides synthesized used primers ability grow YNBG retransformation                 complete regions. DNA data analysis performed <br /> pCIZ1.                                                             using MacVector (IBI) program. Protein sequence <br />    Complementation analysis SEY2101-                   alignment sequences NBRF GenBank data <br /> SEY2102-derived mutants backcrosses followed stan-             bases performed using TFASTA program dard genetic protocols (34), did tetrad analysis            University Wisconsin Genetics Group <br /> random spore analysis select haploids using ade2-1       program (7). DNA sequence protein sequence <br /> allele. GS8-44 (hkel-l&apos;s) strain chosen genetic    alignment identical previously presented (3). molecular analyses described report.                Disruption HKEI. 5.1-kb fragment bearing HKEI <br /> mutants described                                gene 5&apos; 3&apos; flanking DNA amplified PCR     Cloning HKEI/RATI gene. Strain GS8-44 (hkel-l&apos;s)       primers flanking BamHI site YEpl3, plasmid <br /> transformed yeast Sau3A genomic library inserted            pMKHKE1. Additional BamHI sites created novo YEp13. Cells plated media selective trans-        PCR method used convenient subcloning formants room temperature 1 day shifting          entire genomic fragment containing HKE1 single <br /> 37&#194;&#176;C. 2 days 37&#194;&#176;C, fast-growing colonies             BamHI fragment. 5.1-kb BamHI fragment cloned <br /> isolated rescreened ability complement            BglII site polylinker reglon pSP71 <br /> hkel-1. independent plasmids identified        (Promega) create pHKE71. pHKE71 digested  19,000 Leu+ transformants. insert DNA            SalI-SacII remove 430-bp internal fragment near 5&apos; <br />  VOL. 13, 1993                                                                   ESSENTIAL YEAST EXORIBONUCLEASE                  343 <br />  <br />                                  ll.7 kb                                  lows: fusion protein obtained E. coli extracts, <br />                                                                           electrophoresed preparative sodium dodecyl sulfate <br />                                                                           (SDS)-polyacrylamide gel electrophoresis, transferred <br />                                                                           nitrocellulose. nitrocellulose stained Ponceau <br />           RI H S S <br />                      \\\\\\\k\\ <br />                   ~~~~~ si <br />                               ~II                         RI <br />                                                                II         S visualize protein, strip containing fusion <br />                                                                           protein excised. Crude antisera p116 incubated <br />                                                                           strip following standard Western blot procedures. <br />                                                                           bound antibodies eluted nitrocellulose                                                                           treatment 0.2 M <span id='am-54' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-55' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span>-1 mM EGTA [ethylene glycol- <br />                                                                           bis(,-aminoethyl ether)-N,N,N&apos;,N&apos;-tetraacetic acid] (pH <br />                                                                           2.7) neutralized 3 N NaOH. Immunopurified anti- <br />                                                                           sera tested Western analysis immunoreactivity <br />               /      &lt;0          3.8 kb\                                  p116. <br />                           hisG- -URA3- -hisG                               Indirect immunofluorescence. Anti-trpE::p116 antibodies <br />                                                                           used immunofluorescence described Adams <br />                                                                     Pringle (1). visualize nuclear DNA, cells stained <br />        S Rl B/Bg                                   B/Bg    X        Ss2   0.5 ,ug 4,6-diamino-2-phenylindole (DAPI) ml. <br />                                     5-FOA                                 Treated cells mounted glass slides 1 mg                                                                           p-phenylenediamine ml 90% glycerol. Cells                                                                           examined photographed using fluorescence micros- <br />                                  hisG                                     copy Nikon Optiphot Microscope fitted UV, <br />                                                                           fluorescein isothiocyanate, rhodamine filters                   S RI    B/Bg             B/Bg X Ss2 <br />                                                                           Nomarski (DIC) optics. Kodak T-Max 400 35-mm film                                                                           used photography. NOPJ antibody gift    FIG. 1. HKEI/RATI essential gene. DNA constructions           John Aris (4). <br /> disrupt HKEIIRATI. 3.8-kb consists yeast URA3 gene             Protein synthesis. Cells grown overnight log phase <br /> flanked direct repeats Salmonella hisG DNA (2) used          23&#194;&#176;C YPD. Cells harvested, washed twice disrupt HKEI/RATI insertion SalI-SacII sites       water, resuspended 107 cells ml low-sulfate medium <br /> 5&apos; region gene (restriction sites used follows: B/Bg,      (LSM  ), incubated 30 min 23&#194;&#176;C prior BamHI-BglII; H, HindIII; Rl, EcoRI; S, Sall; Ssl, Sacl; Ss2, <br /> SacII; X, XbaI) described Materials Methods.      zero-time labeling point. time zero, cells split linear fragment containing construct used              half culture shifted 37&#194;&#176;C. transform diploid strain W303. Tetrad analysis HKEI/           labeling point, 1 ml culture removed labeled 5 <br /> hkel::URA3 diploid revealed segregation pattern 2 viable             min 23&#194;&#176;C 5 ,uCi Na235SO4. Labeling termi- <br /> spores:2 inviable spores. Treatment Ura+ transformants            nated addition 1 ml 5% trichloroacetic acid plus 1% <br /> 5-FOA selects cells excised URA3 gene        Casamino Acids. Samples collected counted recombination direct repeats hisG gene         using GFA glass filters. ura3 mutants maintain hkel::hisG gene disruption.          Protein manipulations. Yeast cell extracts Tetrad analysis diploid gave rise 2 live:2 dead spores.   prepared glass bead lysis radioimmunoprecipitation <br />                                                                           assay buffer (150 mM NaCl, 1% Nonidet P-40, 0.5% sodium <br />                                                                           deoxycholate, 0.1% SDS, 50 mM Tris-HCl [pH 8.0]) plus <br /> end HKEI. linear -7-kb fragment isolated                   protease inhibitors (13). Total insoluble E. coli proteins <br /> used recipient input DNA described                   prepared method Koerner et al. (20). Proteins <br />   3.8-kb fragment consists yeast URA3 gene                  RNase-containing fractions precipitated tri- <br /> flanked direct repeats Salmonella hisG DNA (2)                  chloroacetic acid 5%. Proteins separated electro- <br /> subcloned polylinker region Bluescript vector                 phoretically 7.5 10% gels method Laemmli <br /> (Stratagene). Sall-SacII fragment isolated                (23) transferred nitrocellulose using Bio- <br /> construct subcloned SalI-SacII site               Rad semidry transfer apparatus (13). immunoblotting HKEI gene pHKE71. resulting plasmid, pHKE71::                      hydroxylapatite column fractions, nitrocellulose membranes <br /> hisGURA43, digested XhoI AatII release                 blocked 1 h 5% instant milk TTBS (20 mM Tris <br /> -8-kb fragment useful integrative transformation             [pH 7.5], 500 mM NaCl, 0.05% Tween 20) washed diploid yeast strain W303 (31). Ura+ transformants                   TIBS. Washed blots incubated 1 2 h selected confirmed Southern analysis. Individual                   appropriately diluted antiserum. Blots washed <br /> transformants subsequently used detect excision                   reacted anti-rabbit secondary antiserum <br /> events lead Ura- phenotype                       (horseradish peroxidase conjugated), immunoreactive <br /> selected 5-fluoro-<span id='am-12' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span>orotic acid</span> (5-FOA) (5). Figure 1                bands detected enhanced chemiluminescence depicts disrupted HKE1 gene.                                          kit Amersham. Immunoblotting heparin-agarose <br />   Antibody production. frame gene fusion                column fractions carried crude p116 antisera E. coli trpE gene HKEI/RATI ligating                   described Larimer Stevens (25) using &quot;2I-protein <br /> internal HindIII 706-bp fragment HindIII site                 (ICN). <br /> pATH1l (20). resulting plasmid coded fusion                        Exoribonuclease analysis. Enzymatic assay exoribonu- <br /> protein consisting amino acids 168 403 p116 fused              clease activity [3H]poly  substrate carried 324 amino acids trpE protein, creating              described previously (36, 41) unit activity, <br /> hybrid protein approximately 66 kDa. Rabbit polyclonal                 previously defined (41), needed release 5 <br /> antibodies generated gel-purified trpE fusion                 nmol l H]<span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>AMP</span> 150 nmol [3H]poly  30 min <br /> protein. immunofluorescence Western immunoblot                    37&#194;&#176;C. Hydroxylapatite column fractions assayed <br /> applications, crude sera immunopurified fol-                 addition 20 mM EDTA reaction mixtures, <br />  344     KENNA ET AL.                                                                                                MOL. CELL. BIOL. <br />  values obtained subtracted plus Mg2+                                1      2       3         1      2       3 <br /> values. Mg2 -independent activity 10% Mg2+-dependent activity cases. Cells grown 1 liter minimal medium A650 1.7 1.9 harvested. Crude extracts prepared yeast            WT <br /> parental strain grown 2pm plasmid <br /> harboring HKEI (PMKHKE1) procedure described Fig. 1 Larimer Stevens (25). parental strain       mutant hkel&apos;S mutant grown similarly. Extracts (1 ml) resolved hydroxylapatite (Bio-Rad) chromatography              mutant + pl 16 <br /> (2.7-ml columns) exactly previously described (25). 40-ml linear gradient (50 500 mM potassium phosphate <br /> buffer [pH 7.6]) used, 1-ml fractions collected, 15      mutant + p175 <br /> pl fraction assayed, results plotted graphically units activity versus fraction num- <br /> ber  Fig. 5). heparin-agarose chromatography carried described legend Fig. 6.                                         230C                  370C <br />                                                                    FIG. 2. XRNJ/KEMl fails rescue temperature sensitivity <br />                            RESULTS                               phenotype hkel-1. hkel temperature sensitive growth <br />                                                                  reversed presence XRN1 gene 2,m <br />    Cloning sequence analysis HKEI gene. genetic     plasmid (YEpl3XRN1). Shown duplicate plates (23 37&#194;&#176;C) screen select temperature-sensitive (ts) mutants ex-     serial dilutions (columns: 1, -103 cells; 2, -102 cells; 3, -101 cells) <br /> hibited altered transport mitochondrial protein precursors    wild-type (WT) mutant strains indicated mutant (hkel&apos;s), <br /> yielded mutants falling different complementa-      mutant + p116 (hkel&apos;s transformed HKEl/RATl 2,um <br /> tion groups, defective mitochondrial        plasmid, pMKHKEl), mutant + p175 (XRNIKEM1 2tm <br /> translocation. studies presented      plasmid, pExo2). course study,  arrived mutant temperature sensitivity phenotype segregated <br /> independently mitochondrial defect. cloned           HKEI/RA4T1 essential gene rescued temperature-sensitive gene showed essen-      multicopy XRNI/KEMI. URA3 gene flanked hisG <br /> tial gene preliminary sequence analysis provoked        repeats  Materials Methods) (Fig. 1) inserted pursuing characterization      HKE1 integrated genome generate Discussion). report, molecular char-     heterozygous HKElIhkel::hisG diploid. correct physi- <br /> acterization gene biochemical analysis    cal insertion integration confirmed Southern <br /> gene product.                                                    analysis (data shown). Sporulation tetrad analysis    characterize temperature-sensitive mutant,     diploid revealed spores harboring hkel::hisG recessive ts allele genetically complemented         disruption viable. Microscopic examination shown allele responsible temperature-          disruptive spores revealed early block cell division. <br /> sensitive growth integrative mapping  Materiats       spores did germinate arrested growth Methods). complementing fragment sequenced           single division. data suggest important role contained open reading frame 1,007 codons (3,021 bp)       HKEJIRATJ gene product cell growth. diploid strain <br /> encoding protein approximately 116 kDa (calculated          bearing disrupted gene wild-type copy gene <br /> molecular weight, 116,773).                                      UR43 plasmid sporulated, tetrads dis- <br />    oldest complete sequence data base        sected. Spores allowed germinate rich medium <br /> revealed significant homology open reading frame      later replicated plates containing 5-FOA (5). sequence yeast gene KEMI (18).       cells carry hisG gene insertion (confirmed independent studies, KEM1 gene identi-        PCR strategies) grow 5-FOA, demonstrating <br /> fied different activities named XRN1, SEP1, RAR5,       disrupted strain requires plasmid-borne copy DST2  summary, reference 17).              gene mitotic growth (data shown). result <br /> purposes report, refer gene XRN1I      confirms Amberg et al. (3) disruption RATIJ. <br /> KEM1, protein encodes (175 kDa) referred         confirm independence cellular activities p175.                                                      ascribed HKEJ/RATJ homologous XRNI/KEM1 <br />    high homology complementing gene        gene products, hkel mutant transformed with XRNI/KEMI potential related function,        XRNJ/KEMI multicopy plasmid (pExo2) determine originally named HKEJ homology         increased levels p175 rescue cells nonper- <br /> karyogamy enhancing gene, KEMI. (Alignment HKE1           missive temperature. transformed cells compared <br /> amino acid sequence KEMI revealed          hkel-l&apos;s mutant cells rescued HKEJ/RATJ <br /> regions significant stretches homology, including 37%      plasmid. shown Fig. 2, thermosensitive growth identity 414 amino acid residues [3; data shown].)      hkel-l&apos; mutant rescued XRNJIKEMJ Concurrent studies, present     conditions HKEJ/RATI rescued growth 37&#194;&#176;C. <br /> data base, sequence molecular characterization      Additionally, XRNI/KEMJ gene multicopy plasmid gene (RATI), know identical        (pExo2) did rescue lethality hkel/ratl ::URA3 HKEJ (3). sequence           disruption strain sporulation tetrad analysis. presented  order consistent literature,   meiotic products disrupted diploid strain bearing use HKEI/RATI referring gene            XRN1IKEMI oii multicopy plasmid segregated 2  p116 referring protein.                          ble:2 inviable spores  shown). Parallel, control experi- <br />  VOL. 13, 1993                                                                  ESSENTIAL YEAST EXORIBONUCLEASE                     345 <br />  <br />                                                                          cells, antibody localization requires expression p116 <br />                                                                          multicopy plasmid. affect normal <br />                                                                          subcellular localization protein. <br />                                                                             Translation defect HKE1&apos;3. study Amberg et al. <br />                                                                        (3), RATl identified screen designed isolate <br />                                                                          temperature-sensitive mutants involved trafficking               DIC                                                        mRNA nucleus. situ localization mRNA                                                                          mutant reveals efficiently released                                                                          nucleus cytoplasm nonpermissive tempera- <br />                                                                          ture. Mutant cells held 37&#194;&#176;C exhibit marked nuclear <br />                                                                          staining mRNA loss cytoplasmic poly  signal <br />                                                                          compared cells 23&#194;&#176;C wild type                                                                          temperature. similar study Kadowaki et al. (16) <br />                                                                          mutant (mtrl) shows similar mRNA nuclear reten- <br />                                                                          tion phenotype described. authors additionally <br />           B                                                              demonstrate protein synthesis decreased                                                                          mutant mRNA levels remain constant. determine               DAPI                                                       hkel-l&apos;s mutant exhibited similar phenotype, exam- <br />                                                                          ined protein synthesis mutant. <br />                                                                             Wild-type hkel-ls cells labeled Na235SO4                                                                          permissive nonpermissive temperatures,                                                                          abilities cells incorporate amino acids assayed <br />                                                                          (Materials Methods). Protein synthesis hkel-1 <br />                                                                          temperature-sensitive mutant significantly lower           c <br />                                                                          wild-type cells following shift 37&#194;&#176;C. shifting <br />                                                                          nonpermissive temperature, protein synthesis               anti trpE::p116                                            mutant begins reduced levels 10%                                                                          wild type (Fig. 4B). decrease protein synthesis                                                                          reflect posttranscriptional event blocks acces- <br />                                                                          sibility RNA translation apparatus. <br />                                                                             Evidence HKEI/RATI gene encodes protein                                                                          exonuclease (RNase) activity. homologous p175 protein <br />                                                                          encoded XRNJIKEMI gene   FIG. 3. Intracellular localization p116. Polyclonal antibodies <br />                                                                          5&apos;--3&apos; exoribonuclease activity 5&apos;-)3&apos; DNase activity  generated trpE fusion protein amino acids 168       catalyze DNA strand exchange  review, 403 p116. Crude sera immunopurified          reference 17). identified studies 5&apos;--*3&apos; <br /> fusion protein obtained E. coli extracts. Yeast cells bearing   exoribonuclease activities S. cerevisiae 10 years ago <br /> hkel/ratl::URA3 disruption HKE1/RATJ high-copy-num-             (36, 41). Yeast exoribonuclease activities elute hydrox- <br /> ber vector (pMKHKEl) fixed prepared immunofluo-             ylapatite columns discrete peaks (36) rescence trpE::pll6 antibody described Materials          predominant activities hydrolyze poly  pH 9.4 Methods.   Cell field viewed DIC optics; (B) cells      require Mg2+. exoribonuclease activities account stained DAPI visualize nuclei; (C) fluorescence       &gt;80% Mg2+-dependent RNase activity cytoplasmic <br /> trpE::pll6 antibody, indicating nuclear location protein   ribosome fractions yeast cells (38). Earlier studies cells. <br />                                                                          demonstrated exonuclease peak eluting hy- <br />                                                                          droxylapatite higher salt concentrations p175, <br />                                                                          XRNI/KEM1 gene product (25, 36). RNase <br /> ments  performed pExo2 disruption mutant                peak, eluting lower salt concentration, YPH500km1 ensure plasmid rescue slow                extensively characterized, purified <br /> growth phenotype XRNI/KEMJ disruption mutant                       (37) degrade 5&apos; terminally labeled rRNA (25).                                                                    [5,_32p, 3H](pA)4 manner described p175 <br />    Antibody localization. Indirect immunofluorescence                 (36, 38)  Fig. 7). referred exo-2  wild-type yeast cells bearing HKEI/RATl 2p,m plasmid                determine exoribonuclease activity lower-salt <br /> (pMKHKE1) using anti-trpE::pll6 antisera revealed                  hydroxylapatite column peak highly <br /> nuclear location p116 approximately 30 40%              conserved HKEJIRATJ gene product, prepared protein <br /> cells given field. staining distributed             extracts hydroxylapatite column chromatography nucleus space defined DAPI detection                   yeast strain harboring HKE1/RATl multicopy plasmid <br /> nuclear DNA (Fig. 3). shown control experiment                  strain plasmid. Quantitation <br /> using NOP1 antibodies. NOP1 nucleolar protein (4),              Mg2 -requiring poly -hydrolyzing activities fractions <br /> staining antibody evident nuclei             eluted hydroxylapatite columns revealed  cells. uniform nuclear staining NOP1 indicates              fivefold increase level nuclease activity cofraction- lower percentage nuclear staining                  ating peak eluting lower salt concentrations anti-p116 antibodies likely artifactual effects           yeast cells bearing multiple copies HKEJIRA T1 (Fig. SA). fixation spheroplasting. possible nuclear         high level peak, p175 activity localization cells reflects cell cycle-specific           shoulder activity. overproduced HKEI/ <br /> regulation localization p116.               RATI activity eluted location peak relatively low abundance protein wild-type                activity shown previously remain yeast cells harboring <br />  346            KENNA ET AL.                                                                                                  MOL. CELL. BIOL. <br />   GROWTH                                                                  <br />  <br />  <br />  <br />         E <br />                                                                                    F- <br />  J)C,D <br />      X  CD <br />                                                                                    C) <br />                                                                                    LU <br />  <br />  <br />  <br />  <br />   0i   O                                                                                 ) <br />                                                                                    L] <br />                                                                                                                                                                                                                                                           m <br />  B.     PROTEIN SYNTHESIS                                                          rr <br />                                                                                    0 <br />                                                                                    x <br />                                                                                    IlJ <br />   CU)                                               WT (37-C) <br />                  3- <br />  <br />  <br />                                                     WT (23-C) <br />   =                                                                                              0              10               20             30 <br />          _                                                                                                 FRAC,TION  <br />                                                     Mutant (23-C) <br />   E <br />  <br />                  1- <br />        0 <br />  <br />  <br />                                                  Mutant (37-C) <br />                                                                           B.                          WT            WT   +   (pMKHKE1) <br />                       0   2       4         6          8            10                                              0 <br />  <br />  <br />   FIG. 4. Protein synthesis hkel-l&apos;s.   Log-phase cells grown       Fraction           14    16    18     x <br />                                                                                                                              14   16   18 23&#194;&#176;C shifted fresh medium indicated tempera- <br /> tures. Aliquots taken times indicated, optical <br /> density (OD) 600 nm determined. Open symbols represent              HKE1p <br /> cultures growing 23&#194;&#176;C filled symbols represent cultures             116 KDa <br />  <br /> growing 37&#194;&#176;C. (B) Protein synthesis assayed incorporation Na235SO4 described Materials Methods. times <br /> indicated, aliquots containing numbers cells               FIG. 5.   Separation exoribonuclease activities removed culture flask labeled 5 min addition     p116 overproducing strain. Crude extracts prepared 5 pCi Na235SO4. Samples treated described plotted      yeast parental strain grown pMKHKE1 (2,um) counts minute versus time. Open symbols represent cultures        chromatographed hydroxylapatite column, fractions growing 23&#194;&#176;C LSM filled symbols represent cultures             assayed exoribonuclease activity  Materials Methods). <br /> growing 37&#194;&#176;C LSM. WT, wild type.                                   values obtained plotted units RNase activity versus <br />                                                                          fraction number. peak exoribonuclease activity higher salt <br />                                                                          concentrations fraction 24 wild-type extract p175 <br />                                                                          (21). peak activity noted shoulder major disruption p175-encoding gene (25). increase                                                                          low-salt peak HKEJ/RATI overproducing strain. (B) Western <br />                                                                          blot p116 exonuclease-containing fractions. Immunopurified <br /> RNase activity comparable observed overpro-               rabbit antibody TrpE::pll6 fusion protein  Materials duction p175 identical conditions earlier studies            Methods) used probe Western blot containing hydroxylap- <br /> (25). hkel-lts mutant parental strain                  atite column fractions (Fig. 5A) increased 5&apos;--3&apos; exoribonu- <br /> grown 23&#194;&#176;C, extracts examined                    clease activity crude extracts. Lanes: extract, equal <br /> manner hydroxylapatite chromatography. extract                 protein amounts crude extracts wild-type parental strain parental strain showed results similar Fig. SA;          (WT) WT + pMKHKE1; 14, 16, 18, equal loadings total  poly -hydrolyzing peak eluting lower salt              protein associated exoribonuclease peak (fractions 14, 16, <br /> concentrations reduced 30% parental                  18, respectively) WT WT + pMKHKE1. <br /> level hkeltS mutant (data shown). hkel-l&apos;s <br /> strain grows slowly, 23&#194;&#176;C (Fig. 4A). <br />    increased level RNase activity observed Fig. SA             reacted RNase activity fractions. Immu- consistent increased synthesis HKEI/                  noaffinity-purified antiserum p116 prepared  Ma- <br /> RATI gene product multicopy HKEI/RATJ plas-                     terials Methods) used probe crude extracts <br /> mid.  increase result regulation             partially purified hydroxylapatite peak fractions (14, synthesis activity RNase HKEI/RATl               16, 18) RNase activity Fig. 5A. studies <br /> gene. eliminate possibility, used p116 antibody            showed protein approximately 116 kDa recognized protein expected size            fractions recognized antiserum p116 <br /> HKEJIRAT1 gene product (116 kDa), protein                  (Fig. 5B). level protein considerably  <br /> overexpressed HKE1/RATI plasmid,                      creased  fivefold) identical extracts cells <br />  VOL. 13, 1993                                                                                   ESSENTIAL YEAST EXORIBONUCLEASE                     347 <br />  <br />          TABLE 1. Immunodepletion RNase activity                                    overexpressed activity eluted peak shown. Figure 6B <br />                      anti-p116 antibodiesa                                                shows Western blot peak fractions column. <br />          Enzyme fraction                                                     % Activity   protein band approximately 116 kDa immunoreactive <br />           serum (,ul)                                                    remaining    case elution profile parallels 16 (WT)                                                                                   activity. difference specific activity, e., <br />   ...........................................                              100       RNase activity immunoreactive protein,   Preimmune (1) .......................            ....................          97       overexpressed exo-2 fractions (approximately 1.5-   Preinmune (2.5) ...........................................                    95       2-fold; compare overexpressed fraction 37 exo-2 frac- <br />   Immune (1) .................             ..........................            12       tion 32 [Fig. 6B]) apparent, quantitation actual <br />   Immune (2.5) .....................            ......................           10       amounts protein loaded present bands (e.g.,                                                                                           staining) possible low levels 16 (WT + pMKHKEl)                                                                         protein  20 ,ug/ml case exo-2 fractions). <br />   ...........................................                              100       possible overexpressed protein differs   Preimmune (1) .....                                                           104 <br />                                     ...................................... <br />  <br />  <br />   Preimmune (2.5) ...........................................                   107       structural modification(s) affecting RNase activity immu- <br />   Immune (1) .................             ..........................            19       noreactivity. <br />   Immune (2.5) .....................            ......................           10          poly  hydrolytic activity columns                                                                                           Mg2e dependent, 98% product Heparin-agarose-purified exo-2                                                            5&apos;-AMP (data shown). studies mode   ...........................................                              100       hydrolysis exo-2, purified manner  <br />   Preimmune (1) .......................            ....................         100       scribed legend Fig. 6,   Preimmune (2.5) ...........................................                    93       [3H]poly  substrate (38). studies showed <br />   Immune (1) .................             ..........................            17       98% hydrolysis product 5&apos;-AMP,   Immune (2.5) .        ...........................................               7 <br />                                                                                           remaining 2% short adenylate oligonucleotides (2 4 <br />   Aliquots hydroxylapatite column fraction 16 wild-type                   nucleotides long). probably derived extract (3 pg) wild-type plus pMKHKE1 extract (3.5 pg) (Fig. 5A, WT               termini chains, exoribonucleases fail WT + pMKHKEl, respectively) 0.9 ig heparin-agarose-purified                  hydrolyze chain completely (36). [5_-32P, <br /> exo-2 (36) mixed antiserum shown reaction mixtures (60 <br /> pl) containing 16 mg albumin ml, 1.6% Nonidet P-40, 50 mM NaCI,                3H]poly  substrate, 32P-labeled product 3.3 mM sodium phosphate buffer (pH 7.2). 4 h 4&apos;C, protein agarose              5&apos;-AMP. Poly  lacking 5&apos;-phosphate end group hydro- <br /> (Pierce) (30 pL1 50% suspension) added, suspensions                 lyzed exo-2 25% rate poly  containing mixed 2.5 h 4&#194;&#176;C. Aliquots supematant solutions following                    terminal phosphate. Similar findings centrifugation assayed described Materials Methods, percent activity remaining supematant shown. crude                p175 (41). hydrolysis RNA substrates rabbit antisera used indicated.                                                        studied  Fig. 7 shows analysis                                                                                           type exoribonuclease activity purified exo-2 fractions, <br />                                                                                           prepared manner shown Fig. 6, using <br />                                                                                           yeast [5&apos;-32P, 3H]rRNA (25S) substrate 0&apos;C. <span id='am-78' about='rdfs:label' typeof='owl:Thing'>Label</span> <br /> harboring multicopy plasmid pMKHKE1. size                                      release incubation RNase measured. protein correlated estimated size (116 kDa)                            results 5,_32p label released considerably DNA sequence analysis HKEI/RATI. protein                                   faster 3H label, showing 5&apos;-&gt;3&apos; mode barely detected cell extracts protein level                              hydrolysis predominant  enzyme fractions overproduced protein readily apparent [Fig. 5B,                              contain phosphatase activity conversion WT versus WT + (pMKHKE1)]. activity increase                                          5&apos;-AMP <span id='am-24' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> [5&apos;-32P]poly  32Pi  <br /> measured (Fig. 5A) represent low estimate                                    analysis substrate reaction specificity actual increase vivo.                                                                  exo-2 progress. <br />    Immunodepletion RNase activity tested addition p116 antiserum peak hydroxylapatite <br /> fractions Fig. 5A, preparation exo-2                                                         DISCUSSION <br /> purified YPH50QnJl yeast strain described Stevens (37). Table 1 shows results immunopre-                                    present study characterized conditional <br /> cipitation RNase activity trpE::pll6 fusion protein                             yeast mutant defective nucleolytic hydrolysis antibody. Approximately 90% RNase activity                                     RNA. essential HKEJ/RATI gene uncovered depleted RNase-containing fractions incu-                               mutation encodes protein significant homology bation immune antiserum incubation                                  yeast protein (p175) exhibits variety bio- preimmune serum.                                                                 <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> activities, strand exchange exoribonu- <br />    purification heparin-agarose column chroma-                                 clease activity (17). addition mRNA trafficking data <br /> tography Western immunoblot analysis overex-                                   pre&quot;iously published RATl (3), present <br /> pressed (HKElIRATl plasmid-dependent) RNase activity                                      sty. J HKEI/RATJ gene product exhibits 5&apos;--3&apos; coincidental purification activity                              exoribonuclease activity like previously demonstrated previously described exo-2 (36, 37)                             related p175. <br /> demonstrated. data presented Fig. 6.                                        p175 purified homogeneity based 5&apos;--3&apos; <br /> overexpressed poly -hydrolyzing activity hydroxy-                                exoribonuclease activity (36, 41), gene encoding lapatite column fractions obtained chromatography                                   designated subsequent studies XRNJ (exoribonu- <br /> crude extracts (Fig. SA) purifies similarly heparin-                                 clease 1 [25]). earlier results analysis poly  <br /> agarose column exo-2 activity purified high-salt                            hydrolytic activity cells lacking active XRNJ gene <br /> ribosome wash yeast cells containing disruption                                 suggested second 5&apos;--3&apos; exoribonuclease XRNJ (37). heparin-agarose chromatography results                                  partially compensate loss p175 xrnl disruption 8- 10-fold purification RNase activity.                              mutants existed (25). provided evidence  348            KENNA ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />                                                                               0 <br />                 14                                                              U) <br />                                                                                                                                           51-32p <br />                                                                                       &gt;--J 30- <br />                 12 <br />                                                                                 az:    &lt; 20- <br />                                                                                                                            0.~~~~~~ <br />                                                                                                      0~~~~~ <br />                                                                                                     o0 <br />                 10 <br />                                                                                      -                                                        3H <br />                                                                                               --,_0                  -~00 <br />       E-                                                                                    I1OI                             u                                                                                                                5                    10 <br />   U) .-. <br />                  8                                                                                         INCUBATION TIME, min <br />                                                                                   FIG. 7. Hydrolysis [5&apos;-32P, 3H]rRNA exo-2. [5 _32p, 3H] <br />                  6 <br />                                                                                 rRNA (25S) prepared described Stevens Maupin (40). <br />                                                                               RNA (1.3 nmol) incubated times shown 0&#194;&#176;C                                                                                 50 U exo-2 (purified described [37]) reaction <br />   0m_s4                                                                         mixtures (50 ,ul) containing Tris-HCl buffer (pH 8.0) (25 mM), <br />            :     4                                                              NH4Cl (50 mM), MgCl2 (3 mM), dithiothreitol (0.5 mM), 30 p,g <br />            C                                                                    albumin. Assay acid-soluble radioactivity previ- <br />                                                                                 ously described (41).   Control values reaction mixtures   lacking <br />                  2                                                              enzyme subtracted. <br />  <br />                  0    ___wi__s_------ 7- r              -r--- -t     --r----&apos;   second enzyme encoded essential gene HKEJ/ <br />                       0            10         20          30          40        RAT1. <br />                                         FRACTION                                gene p175 cloned sequenced                                                                                 laboratories. using assay detects trans- <br />                                                                                 fer strand DNA molecule complementary <br />                                                                                 single-stranded partner (21), gene strand-exchange <br />       B.                      W T + pM K H KE1            E xo -2               protein, designated SEPI, characterized (42)                                                                               independently confirmed designated DST2 (9, <br />  Frction               23 26 29 32 35 37 26 28 30 32 34 36                   10). SEPI shown contain 5&apos;--3&apos; exonuclease <br />  Units, 10 2              0.6 1.7 3.4 3.4 2.3 1,3 0.3 0.8 1.7 2.2 1.8 0.8       activity hydrolysis single- double- <br />                                                                                 stranded DNA (15). activity likely <br />    kDo 205-                                                                     responsible synapsis observed 5&apos;-*3&apos; polarity                                                                                 strand-exchange reaction.  mechanistic <br />                116-            *          u                                     details activity resolved present (17). <br />                                                                                 Kolodner et al. (21) data suggesting p175 plays                                                                                 important role meiotic recombination. Finally,                80 -                                                             studies utilizing vivo mutant screens report                                                                                 fidelity nuclear fusion (18) mitotic stability                                                                                 minichromosomes containing altered origins (19)                49 -                                                             yielded gene, designated KEMI RAR5, respec- <br />                                                                                 tively. <br />                                                                                   seemingly different activities   FIG. 6.   Heparin-agarose chromatography overex- <br />                                                                                  Certainly, exonuclease                                                                                 involved number different hydrolytic reactions pressed RNase activity exo-2.Hydroxylapatite chromatography                 reaction affect distinct processes. Loss extracts yeast strain (0.4 g) containing high-copy- <br /> number HKE. l/RATI plasmid carried described                      exonuclease result confusing array phenotypes. <br /> Materials 1Methods using 9-ml column 80-ml gradient             Events 5&apos;-*3&apos; exoribonucleases involved potassium pphosphate buffer described previously (36). Assay              described previously (41) discussed fractions shiowed elution profile similar Fig. SA.              Recent studies p175 mutants mRNA <br /> Fractions 13 tRo 17, containing 4 x 103 U RNase activity,               turnover affected, resulting differences cellular <br /> combined, diluited equal volume 20 mM Tris-HCl buffer                mRNA protein levels (24). Vreken Raue (pH 7.7)-10% glycerol, applied 4-ml column heparin-                 recently presented results suggesting involvement agarose (Sigmaa) prewashed buffer containing 100 mM               exonuclease mRNA turnover (43). role (NH4)2SO4. Tihe column washed 8 ml buffer elu ted 40-ml linear gradient 100 600 mM                exonuclease process affect (NT V-       1Im;111ltPr iractions <br />          n r(une-mllninlter <br /> kI&apos;n4)2Z5k4.                            xw .-r..nih4tfi-d <br />                               frnrtinne weoreconcult;lcu,  snt1 kiiquoLs <br />                                                           anlu  nliniint        activities cell. assayed using [3H]poly  substrate described Materials Methods. Approximately 5.5 x 103 U RNase <br /> activity recovered. exo-2 chromatography, exo-2 purified described Stevens (37) 5 g YPH5Oxrml                 protein RNase activity elute current heparin-agarose <br /> yeast strain hydroxylapatite chromatography step.                   columns higher salt concentration previously described [37], <br /> Hydroxylapatite fractions (fractions 13 17) combined,               apparently difference heparin-agarose.) (B) Western <br /> 12 x 103 U RNase activity chromatographed heparin-                    blotting carried described Materials Methods, agarose described  Aliquots fractions assayed,             using 500 p.l designated overexpressed RNase fractions 200 9.8 x 103 U activity recovered. elution profile,                 ,ul exo-2 fractions. units RNase activity normalized 50 U activity, plotted column.                contained fraction shown. WT, wild type. <br />  VOL. 13, 1993                                                            ESSENTIAL YEAST EXORIBONUCLEASE                          349 <br />  <br />    HKE1IRATI exoribonuclease activity related pheno-          activities begin better understand temporal types.  array phenotypes condi-      spatial order events involved nuclear cytoplasmic RNA <br /> tional HKEJIRAT1 gene mutants. vivo labeling studies        trafficking relationship transcription, process- <br /> characterize consequences conditional defects asso-        ing events, trafficking. Additional study reaction <br /> ciated loss HKEJ/RATl activity reveal post-         specificity p116 currently way. <br /> transcriptional block RNA utilization. Translation mRNA severely reduced  restrictive temperature.                              ACKNOWLEDGMENTS result expected light findings            thank Eva Hoke excellent technical assistance. Amberg et al. RAT1 (3). situ localization mRNA       acknowledge Ben Hall Terni Aldridge providing unpub- mutant reveals efficiently released   lished sequence data compare HKEI, Chuck Cole nucleus cytoplasm nonpermissive tempera-        David Amberg sharing unpublished data sequence, ture. Mutant cells held 37&#194;&#176;C exhibit marked nuclear          Mark Goebl (Indiana School Medicine) drawing attention <br /> staining mRNA loss cytoplasmic poly  signal         KEM1 homology. <br /> compared cells 23&#194;&#176;C wild-type cells       work supported grants National Institutes temperature. phenotype defects hkell        Health (GM41758) Robert  Welch Foundation (AQ-814) <br /> ratl mutants failure nuclear mRNA       American Heart Association established investigator <br />                                                                   award, M.G.D. S. supported Office Health efficiently clear transcriptional process nucleocy-    Environmental Research, U.S. Department Energy,  <br /> toplasmic translocation apparatus  review, refer-       tract AC05-84 21400 Martin Marietta Energy <br /> ence 28). HKEJ/RAT1 cloned independently          Systems. Direct support S. grant Explor- laboratories (12, 33), uncovered          atory Studies Program, Oak Ridge National Laboratory. M.M. temperature-sensitive suppressor tRNA promoter mu-           supported NRSA National Institutes Health. <br /> tation (12) gene overexpression                                         REFERENCES <br /> restore wild-type growth yeasts bearing overproduced mutated form transcription factor TFIID         1. Adams,  E. M., J. R. Pringle. 1984. Relationship actin <br /> (33). data suggest p116 involved            tubulin distribution bud growth wild-type morpho- <br />                                                                       genetic mutant Saccharomyces cerevisiae. J. Cell Biol. 98:934- <br /> transcriptional processes observed transport             945. <br /> defects secondary effects lack produc-        2. Alani, E., L. Cao, N. Kleckner. 1987. method gene <br /> tion components essential RNA processing traf-              disruption allows repeated use URA3 selection ficking.                                                              construction multiply disrupted yeast strains. Genetics 116: <br />    far transcriptional involvement p116               541-545. <br /> cerned, spectrum nuclease strand-exchange ac-           3. Amberg, D. C.,  L. Goldstein, C. N. Cole. 1992. Isolation <br /> tivities related XRNJIKEM1 protein suggest             characterization RAT1: essential gene Saccharo- <br /> similar potential role HKEJIRATl gene product              myces cerevisiae required efficient nucleocytoplasmic <br /> transcript displacement DNA template  addi-              trafficking mRNA. Genes Dev. 6:1173-1189. <br />                                                                    4. Aris, J. P., G. Blobel. 1988. Identification characteriza- <br /> tional discussion, reference 17).  factors           tion yeast nucleolar protein similar rat liver associated RNA polymerase II,              nucleolar protein. J. Cell Biol. 107:17-31. role p116 unlikely, p116 does      5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. positive <br /> appear abundant protein  revealed                  selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> Western analysis protein levels Northern [RNA]                 ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> analysis message levels) relative polymerase II. Possi-         Genet. 197:345-346. <br /> ble involvement 5&apos;- &apos;3&apos; exonuclease transcription          6. Connelly, S., J. L. Manely. 1988. functional mRNA <br /> termination suggested Connelly Manley (6).            polyadenylation signal required transcription RNA exonuclease actually involved termi-           polymerase II. Genes Dev. 2:440-452. <br />                                                                    7. Devereux, J., P. Haeberli, 0. Smithies. 1984. comprehen- <br /> nation event, affect transcription processing            sive set sequence analysis programs VAX. Nucleic <br /> pathway mRNA removing 3&apos;-terminal fragments                 Acids Res. 12:387-395. <br /> resulting transcript cleavage polyadenylation.            8. Douglas, M., B. Geller, S. Emr. 1984. Intracellular targeting <br />    transcription related function, p116        import Fl 1-subunit-,B-galactosidase hybrid protein <br /> involved RNA processing events nucleus.                 yeast mitochondria. Proc. Natl. Acad. Sci. USA 81:3983- <br /> Amberg et al. (3) reported pre-rRNA process-           3987. <br /> ing defect RAT1. 5.8S rRNA suffer           9. Dykstra, C. C., R. K. Hamatake,  Sugino. 1990. DNA <br /> processing maturation defect, slightly longer           strand protein f yeast mitotic cells differs strand <br /> forms rRNA produced nonpermissive                 transfer meiotic cells. J. Biol. Chem. 265:10968-10973. <br />                                                                   10. Dykstra, C. C., K. Kitada,  B. Clark, R. K. Hamatake,  <br /> temperature. exoribonuclease activity ascribed p116            Sugino. 1991. Cloning characterization DST2, gene responsible trimming 5&apos; ends            DNA strand transfer 1B Saccharomyces cerevisiae. <br /> 5.8S rRNA molecules, suggested authors.            Mol. Cell. Biol. 11:2583-2592. <br /> previously suggested enzymatic properties p175        11. Emr, S.,  Vassarotti, J. Garrett, B. Geller, M. Takeda, make good candidate 5&apos; trimming reactions rRNA            M. G. Douglas. 1986. amino terminus F1-ATPase <br /> maturation (25, 41). Additionally, frag-            1-subunit precursor functions mitochondrial import signal. <br /> ments internal transcribed spacer 1 pre-rRNA                J. Cell Biol. 102:523-533. <br /> accumulate yeast cells lacking p175 (39). suggests      12. Hall, B. D. (University Washington). 1991. Personal commu- <br /> role enzyme scavenger RNase pre-rRNA           nication. <br />                                                                   13. Harlow, E., D. Lane. 1988. Antibodies: laboratory man- <br /> processing. plausible p116 involved rRNA             ual, p. 453. Cold Spring Harbor Laboratories, Cold Spring <br /> processing, trimming scavenger RNase               Harbor, N.Y. coupled translocation nu-          14. Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor- <br /> clear envelope. investigation necessary              mation intact yeast cells treated alkali cations. J. <br /> cover interplay seemingly diverse               Bacteriol. 153:163-168. <br />  350     KENNA ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br /> 15. Johnson,  W., R. D. Kolodner. 1991. Strand exchange                Saccharomyces cerevisiae LEU2 gene Escherichia coli <br />     protein 1 Saccharomyces cerevisiae. J. Biol. Chem. 266:             ,B-galactosidase gene. Mol. Cell. Biol. 3:580-586. <br />     14046-14054.                                                       30.   Miller, J. 1972. Experiments molecular genetics, p. 352-355. <br /> 16. Kadowaki, T., Y. Zhao,  Tartakoff. 1992. conditional             Cold Spring Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     yeast mutant deficient mRNA t <br /> </body></html>