<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 15, 552&#226;&#8364;&#8220;562, February 2004 <br />  <br />  <br /> RNA Interference Inhibition Mus81 Reduces <br /> Mitotic Recombination Human Cells <br /> Veronique Blais,* Hui Gao,* Cherilyn  Elwell,*&#226;&#8364;&#160; Michael N. Boddy,* <br /> Pierre-Henri L. Gaillard,* Paul Russell,&#226;&#8364;&#161; Clare H. McGowan&#226;&#8364;&#161;&#194;&#167; <br />  <br /> *Departments Molecular Biology &#226;&#8364;&#161;Cell Biology, Scripps Research Institute, La Jolla, <br /> California 92037 <br />  <br /> Submitted August 11, 2003; Revised October 15, 2003; Accepted October 15, 2003 <br /> Monitoring Editor: Mark Solomon <br />  <br />  <br />  <br /> Mus81 highly conserved endonuclease homology XPF subunit XPF-ERCC1 complex. yeast Mus81 <br /> associates second subunit, Eme1 Mms4, essential endonuclease activity vitro vivo <br /> function. Human Mus81 binds homolog fission yeast Eme1 vitro vivo. recombinant <br /> Mus81-Eme1 cleaves replication forks, 3&#226;&#180;&#8226; flap substrates, Holliday junctions vitro. use differentially tagged <br /> versions Mus81 Eme1, Mus81 associates Mus81 Eme1 associates Eme1.  <br /> complexes containing Mus81-Eme1 units function coordinate substrate cleavage vivo.  <br /> regulation Mus81 RNA interference reduces mitotic recombination human somatic cells. recombination <br /> defect rescued expression bacterial Holliday junction resolvase. data provide direct evidence role Mus81-Eme1 mitotic recombination higher eukaryotes support hypothesis Mus81-Eme1 resolves <br /> Holliday junctions vivo. <br />  <br />  <br /> INTRODUCTION                                                         duplex free double strand end. double strand end <br />                                                                      repaired homologous recombination, process involv- <br /> Maintaining genomic integrity essential component                                                                      ing resection strand free end, invasion process cell duplication division. Cells espe- <br />                                                                      intact linear duplex, creation new replication fork <br /> cially vulnerable DNA damage process DNA replication. Replication forks encounter relatively         (Haber, 1999). importance homologous recombina- <br /> innocuous forms damage single strand break           tion replication restart established prokaryotes. modified base increase damage generating              Evidence important pathway eu- <br /> double strand break fixing mutation. cope            karyotes accumulating (Cox et al., 2000; Michel, 2000; need complete replication presence DNA              McGlynn Lloyd, 2002). activities resolve Hol- <br /> damage, organisms evolved number mecha-               liday junctions linear duplex identified nisms replicating damaged sites             human somatic cell extracts (Chen et al., 2001; Constantinou <br /> ways cause minimal genome instability. option               et al., 2002).  known Resolvase  introduces sym- <br /> involves use bypass translesion polymerases           metrical related nicks strands like polarity, asso- <br /> act sites template damaged           ciates branch migration activities (Constantinou et al., <br /> way deciphered replicative poly-            2001, 2002). identities proteins comprise merases (Friedberg et al., 2002). second mechanism,             activity known. second human activity volves regression fork annealing nascent           resolves Holliday junctions associated Mus81 (Chen <br /> strands form Holliday junction structure (Higgins et al.,       et al., 2001; Constantinou et al., 2002). Mus81-associated  <br /> 1976). leading strand extended, using            donuclease cleaves Holliday junction structures linear <br /> nascent lagging strand template. Subsequent migration           duplexes does cut symmetrically related sites Holliday junction nascent end              products ligatable vitro (Chen et al., 2001). <br /> stores replication fork, allowing replication continue.       Mus81 component highly conserved endonuclease <br /> Use template-switching mechanism does di-              homology XPF subunit XPF&#226;&#8364;&#8220;ERCC1 com- <br /> rectly repair damage, does allow replication                                                                      plex (Boddy et al., 2000; Interthal Heyer, 2000; Chen et <br /> proceed downstream damaged site circum- <br />                                                                      al., 2001; Mullen et al., 2001). yeast, Mus81 mutants vents problems associated incomplete replication. <br /> Stalled replication forks repaired             hypersensitive DNA-damaging agents impede DNA <br /> recombination repair process Holliday junction            replication, Mus81 mutation lethal combination <br /> formed regressed fork resolved intact              disruption RecQ like helicases Rqh1 Sgs1 <br />                                                                      (Boddy et al., 2000; Mullen et al., 2001). humans, mutations <br />                                                                      RecQ like helicases BLM WRN associated  Article published online ahead print. Mol. Biol. Cell 10.1091/   genomic instability increased incidence cancer (Hick- <br /> mbc.E03&#226;&#8364;&#8220; 08 &#226;&#8364;&#8220; 0580. Article publication <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> www. <br /> molbiolcell.org/cgi/doi/10.1091/mbc.E03&#226;&#8364;&#8220; 08 &#226;&#8364;&#8220; 0580. <br />                                                                      son, 2003). BLM WRN demonstrated  &#226;&#8364;&#160; <br />    Present address: Department Medicine, University Califor-   branch-migrate Holliday junctions vitro (Constantinou et <br />    nia, San Francisco, San Francisco, CA 94143.                      al., 2000; Karow et al., 2000).  genetic interactions <br />  &#194;&#167; <br />    Corresponding author. E-mail address: chmcg@scripps.edu.          enzymatic properties Mus81 RecQ helicases <br />  <br />  <br /> 552                                                                                  &#194;&#169; 2004 American Society Cell Biology <br />                                                                                                                     Mus81 Required Recombination <br />  <br />  <br /> suggest act independently stalled replication                     Expression Recombinant Proteins RNA Interference <br /> forks promote replication restart.                                          (RNAi) <br />    Mus81 associates second protein, known Eme1                       3HaMus81 wild-type endonuclease dead (Chen et al., 2001) cloned fission yeast Mms4 budding yeast. Eme1/Mms4                       pCDNA3 (Invitrogen, Carlsbad, CA) using EcoRI XhoI sites. Eme1 <br /> essential Mus81 endonuclease activity vitro                  FLAG tagged C&#226;&#172;&#732; terminus using oligonucleotide <br />                                                                                5&#226;&#172;&#732;GCCCGCTCGAGTCACTTGTCATCGTCGTCCTTGTAGTCAGCACTATC- <br /> vivo function (Boddy et al., 2001; Kaliraman et al., 2001;                     TAAAGA-3&#226;&#172;&#732; inserted pCDNA3 using EcoRI XhoI. Mus81 Mullen et al., 2001; Doe et al., 2002). Based phenotypes                FLAG tagged C&#226;&#172;&#732; terminus using oligonucleotide fission yeast mutants Mus81 Eme1, e., sensitivity                  5&#226;&#172;&#732;-CTCGAGTCACTTGTCATCGTCGTCCTTGTAGTCGGTCAAGGGGCC- agents impede DNA replication, recombination                        GTAGC-3&#226;&#172;&#732;. 3HaEme1 cloning Eme1 pcDNA-3Ha (Mel- <br />                                                                                chionna et al., 2000) using NdeI XhoI. Human cells transfected <br /> pendent failure complete meiosis, evidence                         using Effectene (QIAGEN, Valenica, CA) FuGENE 6 (Roche Diagnostics, <br /> Mus81 containing complexes resolve Holliday junctions                     Indianapolis, ) according manufacturer&#226;&#8364;&#8482;s instructions. expres- <br /> linear duplex vitro, proposed Mus81 Eme1                        sion Sf9 cells, Gst-Mus81 (Chen et al., 2001) cloned pFastBac subunits nuclear Holliday junction resolvase (Boddy                   (Invitrogen) using EcoRI HindIII, Eme1-FLAG cloned using <br />                                                                                EcoRI XhoI. Bac Bac (Invitrogen) used generate <br /> et al., 2001). similar function proposed human                   recombinant viruses. constructs verified sequencing. 19- <br /> enzyme (Chen et al., 2001). hand, specific                     <span id='am-2' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> regions (corresponding 178 &#226;&#8364;&#8220;197 292&#226;&#8364;&#8220;311 Mus81 open <br /> strains budding yeast, Mus81 Mms4 mutants                        reading frame; Chen et al., 2001) selected cloned pSuper profound meiotic recombination defect, suggesting                       used described Brummelkamp et al., (2002). Polymerase chain reaction <br />                                                                                (PCR) carried HeLa cell cDNA library (BD Biosciences Clonetech, Mus81-Mms4 major meiotic Holliday junction                       Palo Alto, CA) using sequences present forms Eme1 (5&#226;&#172;&#732;- <br /> resolvase budding yeast  Los Santos et al., 2001, 2003).                 CGGAATTCACCATGGCTCTAAAGAAGTCATCACC-3&#226;&#172;&#732; 5&#226;&#172;&#732;-GC- <br /> Furthermore, highly purified recombinant Mus81-Mms4                         CCGCTCGAGTCAGTCAGCACTATCTAAAGAGAG-3&#226;&#172;&#732;). PCR products <br /> Mus81-Eme1 partially purified Mus81-complexes                         cloned pTopo (Invitrogen). Restriction enzyme analysis clones <br />                                                                                gave pattern corresponding Eme1B. Sequencing clones verified <br /> human cells extracts greater activity 3&#226;&#172;&#732; flap                    transcript corresponding Eme1B expressed HeLa cells. NLS- <br /> replication fork structures Holliday junctions (Ka-                  RusA-2Ha wild-type inactive cloned pCDNA3 expression liraman et al., 2001; Constantinou et al., 2002; Whitby et al.,                human cells using pRep1-RusA pRep1-RusA-D70N (Boddy et al., <br /> 2003). preference Mus81 shows substrates led                     2001) starting constructs. suggestion replication forks 3&#226;&#172;&#732; flaps,                                                                                Gene Cloning <br /> Holliday junctions, relevant vivo substrate                                                                                clones corresponding sequence AK0055926 retrieved Mus81 (Kaliraman et al., 2001; Constantinou et al., 2002; Doe                  American Type Culture Collection (Manassas, VA) collection human ex- <br /> et al., 2002; Bastin-Shanower et al., 2003).                                   pressed tagged sequences. ATCC clone 2899969 (coding Eme1A) came <br />    gain insight function Mus81 human                    cervical carcinoma library, ATCC clone 3905809 (coding Eme1B) <br /> cells identified human homologs Eme1 bind                      came form uterine tumor library, ATCC clone 814686 (coding                                                                                Eme1C) came form normal B-cell library. clones sequenced <br /> Mus81 vitro vivo. sequence designated                         directly. difference clones observed multiple reads <br /> Eme1A independently identified shown interact                       directions used confirm specific nucleotides. Mus81 groups (Ciccia et al., 2003; Ogrunc Sancar, 2003). recombinant Mus81-Eme1                         Nuclease Assays Western Analysis <br /> efficiently cleaves replication forks 3&#226;&#172;&#732; flap substrates                Nuclease assays carried described previously (Chen et al., 2001; <br /> vitro. Mus81-Eme1 generates lower, readily detectable,                   Constantinou et al., 2002). Antibody Ha-epitope Babco (Rich- <br /> yield linear duplex product Holliday junction struc-                   mond, CA). Antibody FLAG-epitope (FLAG-M2) Sigma- <br />                                                                                Aldrich (St. Louis, MO). Antibody Mus81 described Chen et al. <br /> tures. Resolution Holliday junctions requires coordinate                    (2001). Cells lysates, immune-precipitates, immune-blots analysis cleavage opposing strands junction. use                        carried Chen et al. (2001). <br /> differentially tagged versions Mus81 Eme1, Mus81 associates Mus81, Eme1 associates                     Immune-Fluorescence Microscopy Eme1.  complexes containing Mus81-                       Conditions growing XPA cells, localized UV-irradiation, develop- <br /> Eme1 units function coordinate substrate cleavage                     ing immune-fluorescence images described Gao et al. (2003). vivo. regulation Mus81 RNA interference <br /> reduces mitotic recombination human somatic cells.                      RESULTS <br /> recombination defect rescued expression bacterial <br /> Holliday junction resolvase. data provide direct evi-                    Identification Human Homolog Eme1 <br /> dence role Mus81-Eme1 mitotic recombination                     Reiterative PSI-BLAST searches (www.ncbi.nlm.nih.gov/ <br /> higher eukaryotes support hypothesis Mus81-                       blast/Blast) used identify mammalian homologs Eme1 resolves Holliday junctions vivo.                                      Schizosaccharomyces pombe Eme1 (SpEme1). itera- <br />                                                                                tion, uncharacterized open reading frame (AL356173) <br /> MATERIALS METHODS                                                          Neurospora crassa identified having significant <br />                                                                                similarity spEme1. open reading frame Cell Culture Mitotic Recombination Assays                                  previously noted homology Mus81 bind- <br /> HeLa cells, 293 (human embryonic kidney cells), GM847L22 (Prince et al.,   ing partner budding yeast Mms4 (Mullen et al., 2001). <br /> 2001) grown DMEM, supplemented 10% enriched calf serum, <br /> 100 &#226;?&#174;g/ml penicillin streptomycin. routine culture, GM847L22                                                                                Reiteration search identified highly related, <br /> maintained presence 400 &#226;?&#174;g/ml G418. Sf9 cells grown Excell-401   overlapping human sequences significant similarity media (JRH Biosciences, Lenexa, KS) 50 &#226;?&#174;g/ml penicillin strepto-      SpEme1 (Figure 1A). clones corresponding se- <br /> mycin. assay mitotic recombination 5 &#226;&#171;&#187; 105 cells plated G418 free   quence AK0055926 retrieved American Type Cul- <br /> medium 16 h transfection. <span id='am-20' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-21' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Thymidine</span> (2 mM) added culture <br /> medium, cells grown 16 h. Cells cultured normal <br />                                                                                ture Collection sequenced confirm iden- <br /> growth medium 24 h. Cells fixed 2% <span id='am-10' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span>     tity. Surprisingly, clones differ <br /> phosphate-buffered saline (PBS) 10 min, washed PBS twice,         slightly designated Eme1A, Eme1B, Eme1C <br /> assayed &#226;?&#164;-galactosidase activity incubation PBS containing 1 mg/ml   (Figure 1B). identification single genomic locus <br /> X-Gal, 4 mM potassium ferrocyanide, 4 mM potassium ferricyanide, 2 mM <br /> MgCl2 37&#194;&#176;C overnight (Herzing Meyn, 1993). number blue cells <br />                                                                                (17q21.33) National Center Biotechnology Infor- scored using 20&#226;&#171;&#187; objective inverted light microscope. Statistical    mation human genome database suggested significances calculated using Student&#226;&#8364;&#8482;s t test.                        variants derive single gene. absence specific <br />  <br />  <br /> Vol. 15, February 2004                                                                                                                                   553 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br /> Figure 1. Identification human homolog SpEme1.   human cDNA encoding open reading frame 571 amino acids similarity spEme1 (Boddy et al., 2001) identified PSI-BLAST algorithm. Alignment Eme1 homologs generated using Clustal W program. Identities shown reverse. (B) sequence variants human Eme1 obtained American Type Culture <br /> Collection. Eme1A (ATCC clone 2899969) encodes protein 583 amino acids, Eme1B (ATCC clone 3905809) encodes protein 571 amino <br /> acids, Eme1C (ATCC clone 814686) encodes protein 542 amino acids. <br />  <br />  <br />  <br /> sequences (corresponding amino acids 303&#226;&#8364;&#8220;331 372&#226;&#8364;&#8220;             FLAG-Eme1A FLAG-Eme1C immune-precipitates  384 Eme1A) suggested different forms            unpublished data);  Eme1B-associated endonucle- incomplete splicing.  absence single         ase activity consistently higher. <br /> codon  position 138) longest sequence,                determine forms Eme1 expressed HeLa <br /> substitution arginine cysteine  position 525)      cells, oligonucleotide primers common variants shortest  compatible interpretation.       used amplify sequences HeLa cell cDNA determine   forms human              library. products PCR cloned restriction <br /> Eme1 binds Mus81, epitope-tagged constructs          enzyme analysis clones gave pattern corresponding <br /> variants tested ability interact Mus81          Eme1B. Sequencing 2 clones verified transcript support Mus81-associated endonuclease activity.               corresponding Eme1B expressed HeLa cells. Al- <br /> HeLa cells transiently transfected 3HaMus81            analysis does exclude possibility FLAG-tagged Eme1. shown Figure 2A, 3HaMus81                C form Eme1 expressed cell types, detected immune-precipitates forms Eme1.          low levels HeLa cells, transcripts corresponding B  3HaMus81 associated FLAG-                form protein readily detectable, Eme1B reproducibly higher FLAG-Eme1A C.                  conducted analysis Eme1 using B form Human Mus81 robust activity 3&#226;&#172;&#732; flap substrate          protein. vitro (Constantinou et al., 2002);  Mus81              asked endonuclease activity detected <br /> Eme1 immune-complexes assayed associated endo-              Eme1 immune-precipitate dependent Mus81. <br /> nuclease activity using substrate (Figure 2B).           shown Figure 2C, FLAG-Eme1 associated activity 3HaMus81 greatly increased samples           wild-type Mus81 mutant version Mus81 cotransfected FLAG-Eme1B,                 lacks associated endonuclease activity (Chen et al., 2001). <br /> FLAG-Eme1A FLAG-Eme1C. Likewise, differ-                 Endonuclease activity detected FLAG-Eme1 im- <br /> ent forms Eme1 immune-precipitated using                 mune-precipitate cells simultaneously <br /> FLAG antibody, B form readily detectable activity.           transfected wild-type 3HaMus81, cells Longer exposures reveal endonuclease activity                transfected FLAG-Eme1 (Figure 2D). <br />  <br />  <br /> 554                                                                                                       Molecular Biology Cell <br />                                                                                               Mus81 Required Recombination <br />  <br />  <br />  <br />  <br /> Figure 2. Eme1 interacts Mus81 Mus81-dependent endonuclease activ- <br /> ity. HeLa cells transiently transfected 3HaMus81 presence absence FLAG-Eme1. hours transfection, lysates immune-precipi- <br /> tates probed presence 3Ha- <br /> Mus81 FLAG-Eme1.   3HaMus81 detected FLAG immune-precipitates cells express FLAG-Eme1A, B, C. (B) Ha FLAG immune-precipitates assayed associated endonuclease <br /> activity using 3&#226;&#172;&#732; flap substrate. Coex- <br /> pression 3HaMus81 FLAG-Eme1B <br /> resulted highest activity. (C) FLAG- <br /> Eme1B detected Ha immune-precip- <br /> itates cells express wild-type 3Ha- <br /> Mus81 (WT) endonuclease dead <br /> version Mus81 (AA). 3HaMus81 (wild- <br /> type endonuclease dead version) detected FLAG immune-precip- <br /> itate cells express FLAG-Eme1B <br /> (D) FLAG-Eme1B immune-precipitates cells cotransfected wild-type endonuclease dead <br /> 3HaMus81 cleave 3&#226;&#172;&#732; flap structures. <br /> 3HaMus81WT immune-precipitates associated endonuclease activity increased cells cotransfected FLAG-Eme1. S, substrate  black dot represents radioactive phos- <br /> phate labeling substrate. <br />  <br />  <br />   Eme1 associated endonuclease activity dependent       nucleoli (Figure 3A). determine Eme1  <br /> coexpression wild-type Mus81. reported previously           cruited regions damaged DNA cell cycle-depen- <br /> (Chen et al., 2001), Ha-immune-precipitates cells   dent manner, cells synchronized G1/S bound- transfected 3HaMus81 detectable endonuclease        ary double-thymidine block. Arrested cells, cells <br /> activity absence transfected Eme1 (Figure 2D).          released 5 h, UV-irradiated <br />                                                                   polycarbonate filters containing 10-&#226;?&#174;m-diameter <br /> Eme1 Localizes Nucleoli Regions Damaged              pores 15 min later, samples fixed stained DNA                                                               regions damage Eme1. prevent repair Recent studies human Mus81 showed localizes         UV-photoproducts NER, simian virus 40 (SV40)-trans- <br /> nucleoli regions damaged DNA S-phase cells           formed fibroblasts defect XPA used (Gao et al., 2003). Eme1 acting complex Mus81,      analysis (Mone et al., 2001). thymidine-arrested distinctive pattern subcellular        cells UV irradiated, regions damage clearly <br /> localization. tested using indirect-immune-fluo-         visualized (Figure 3B) using cyclobutane pyrimi- <br /> rescence cells transfected vectors          dine dimer-specific monoclonal antibody TDM2 (Mizuno <br /> encoding 3HaEme1 FLAG-Mus81. previously                    et al., 1991; Mori et al., 1991). 3HaEme1 seen, Mus81 retained nucleoli cells       nucleoli thymidine-arrested cells, subjected situ extraction fixation (Gao et al.,      evidence Eme1 recruited regions damaged <br /> 2003). 3HaEme1 exactly colocalize               DNA (1 25 cases). contrast, 5 h thymidine <br /> Mus81 regions nucleus (Figure 3A).         release majority cells replicating DNA, <br /> Costaining 3HaEme1 nucleolar marker nucleo-            Eme1 nucleoli regions phosmin/B23 confirmed Eme1 retained            damage (25 30 cases).  pattern Eme1 local- <br />  <br />  <br /> Vol. 15, February 2004                                                                                                   555 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 3. Eme1 colocalizes Mus81                                                                                         nucleoli recruited regions dam- <br />                                                                                         age S-phase cells. HeLa cells                                                                                         transfected vectors encoding <br />                                                                                         3HaEme1 FLAG-Mus81 grown <br />                                                                                         coverslips 24 h subjected                                                                                         situ extraction fixation.   Samples <br />                                                                                         processed immune-fluorescence <br />                                                                                         using antibodies Ha epitope <br />                                                                                         (red), FLAG epitope (green) nu- <br />                                                                                         cleophosmin/B23 (green). Samples                                                                                         costained 4,6-diamidino-2-phenylin- <br />                                                                                         dole reveal DNA. (B) XPA cells                                                                                         transfected 3HaEme1 FLAG- <br />                                                                                         Mus81, grown coverslips, synchro- <br />                                                                                         nized G1/S boundary double- <br />                                                                                         thymidine (Tdr) block. UV-opaque filter <br />                                                                                         containing pores &#226;&#172;&#402;10 &#226;?&#174;m diameter <br />                                                                                         placed cells exposed                                                                                         60 mJm&#226;&#171;&#186;2 254-nm light cultured <br />                                                                                         15 min situ fraction fixa- <br />                                                                                         tion. Cells stained Eme1 (red)                                                                                         TDM-2 (green) detect regions <br />                                                                                         UV-induced cyclobutane pyrimidine <br />                                                                                         dimer.  typical cell thymi- <br />                                                                                         dine time irradiation.  typ- <br />                                                                                         ical cell 5 h release thymi- <br />                                                                                         dine block. <br />  <br />  <br />  <br /> ization damaged undamaged cells identical            implies enzyme exist complexes containing recently reported Mus81 (Gao et al., 2003).              catalytic units. determine Mus81-Eme1 ex- <br />                                                                   ists higher order complexes needed coordinate <br /> Activity Recombinant Mus81-Eme1                                cleavage Holliday junctions, human cells simulta- <br /> Eme1 essential Mus81 activity function fission      neously transfected plasmids encoding different <br /> yeast (Boddy et al., 2001). Likewise, Mms4 essential       forms Mus81, 3HaMus81, FLAG-Mus81. shown Mus81 activity function budding yeast (Kaliraman et        Figure 5A, FLAG-Mus81 detected immune-pre- <br /> al., 2001; Mullen et al., 2001). test Eme1 essen-   cipitate 3HaMus81. Control samples cells tial activity human Mus81, insect cells        transfected construct cross- <br /> fected baculoviruses encoding Gst-Mus81, FLAG-               reactivity immune-precipitating antibody. Like- <br /> Eme1, (Figure 4A). Immune-precipitated Gst-Mus81          wise, 3HaEme1 cotransfected FLAG-Eme1, FLAG-Eme1 assayed using 3&#226;&#172;&#732; flap, replication         3HaEme1 detected immune-precipitates FLAG- <br /> fork, Holliday junction structure (X12). Gst-Mus81     Eme1 (Figure 5B). analysis does distinguish detectable activity substrates. Like-      number Mus81, Eme1, molecules coprecipitate <br /> wise, immune-precipitated FLAG-Eme1B detectable              does demonstrate endonuclease activity (Figure 4B). contrast, Gst-         molecules Mus81, Eme1, associate vivo. <br /> Mus81 FLAG-Eme1 coexpressed immune-precip- <br /> itates Mus81 readily cleaved 3&#226;&#172;&#732; flap replication       RNA Interference Mus81 Expression Reduces <br /> fork structures. lower detectable specific activity     Viability observed using X12 substrate (Figure 4B).       role Mus81 human cells investigated using endonuclease activity Mus81 depends Eme1            RNAi suppress Mus81 expression (Brummelkamp et al., <br /> vice versa. similar pattern activity seen immune-     2002). shown Figure 6A, Mus81 protein levels precipitates FLAG-Eme1 purification recom-        substantially reduced cells transfected binant Mus81-Eme1 GSH-Sepharose chromatography                 pSuper vectors containing 19-nucleotide sequences tar- <br />  unpublished data). data obtained immune-pre-          regions Mus81 mRNA (pSuper-178 pSuper- <br /> cipitation recombinant Mus81-Eme1 shown          292) (Brummelkamp et al., 2002). loss Mus81 seen <br /> allows direct comparison activity immune-pre-         cells transfected control vector (pSuper). effect <br /> cipitated endogenous Mus81. Like endogenous Mus81,             Mus81-RNAi cell viability determined assay- <br /> combinant Mus81-Eme1 lower activity Hol-          ing colony survival. shown Figure 6B, suppression liday junction substrate. previously seen              Mus81 expression associated significant reduc- <br /> endogenous transfected Mus81, reaction product            tion ability cells form viable colonies. Loss linear duplex (Chen et al., 2001). cleavage         viability specifically associated reduced <br /> tected mutant form Mus81 (Mus81AA) used.            Mus81 expression seen transfection                                                                   different constructs cause reduced Mus81 expres- <br /> Association Mus81-Eme1 Catalytic Units                         sion seen transfection control RNAi generation linear duplex products requires coordi-         vectors. Flow cytometric analysis did reveal significant <br /> nate cleavage opposing strands X12 substrate        change cell cycle distribution 48 72 h trans- <br />  <br />  <br /> 556                                                                                                  Molecular Biology Cell <br />                                                                                                 Mus81 Required Recombination <br />  <br />  <br />  <br />  <br /> Figure 4. Endonuclease activity recom- <br /> binant Mus81-Eme1. Gst-Mus81 FLAG- <br /> Eme1B immune precipitated Sf9 <br /> cells infected viruses encoding  <br /> dicated proteins.   Mus81 immune-pre- <br /> cipitates probed presence Gst-Mus81 FLAG-Eme1B. (B) Recom- <br /> binant Mus81-Eme1 cleaves 3&#226;&#172;&#732; flap, repli- <br /> cation fork, Holliday junction (X12) <br /> structures vitro. activity associated Mus81 immune-precipitates HeLa cells shown comparison  . S, <br /> substrate  black dot represents <br /> radioactive phosphate labeling sub- <br /> strates. <br />  <br />  <br />  <br /> fection pSuper-178 pSuper-292. Experiments         ically inactive &#226;?&#164;-galactosidase (LacZ) genes rise <br /> termine specific cause cell death infor-    LacZ&#226;&#171;&#185; cells gene conversion, unequal sister chromatid <br /> mative.                                                          exchange, intrachromosomal recombination (Herzing <br />                                                                  Meyn, 1993; Meyn, 1993; Prince et al., 2001). RNAi Suppression Mus81 Reduces Mitotic                        previously used demonstrate cells Recombination                                                    ataxia telangiectasia patients increased mitotic recom- Mus81 meiotic defect depends recombination         bination rates (Meyn, 1993) loss Werner reported fission yeast budding yeast         syndrome protein (WRN) associated decreased pro- <br /> terthal Heyer, 2000; Boddy et al., 2001; Kaliraman et al.,   ductive mitotic recombination (Prince et al., 2001; Saintigny <br /> 2001). determine Mus81 needed mitotic          et al., 2002). feature reporter &#226;?&#164;-galactosidase <br /> recombination human cells, took advantage            activity scored directly single cells, SV40-transformed human fibroblast line, GM847L22 (Prince         compatible transient regulation use et al., 2001), contains single integrated copy    RNAi. Furthermore, does depend long-term viabil- <br /> mitotic recombination reporter plasmid pLrec (Herzing        ity culture. transfection plasmids Meyn, 1993; Meyn, 1993). schematic Lrec cassette     suppress Mus81 expression (pSuper-178, pSuper-292) shown Figure 7A. contains direct repeats genet-     control plasmids (pSuper pCDNA), GM847L22 cells  <br />  <br /> Figure 5. Mus81 Eme1 self-associate. <br /> 293 cells transfected 3HaMus81, <br /> FLAG-Mus81,  hours transfection, lysates Ha immune-pre- <br /> cipitates probed presence 3HaMus81 FLAGMus81.   FLAG- <br /> Mus81 detected Ha immune-precip- <br /> itates cells express 3HaMus81. (B) <br /> 3Ha-Eme1 detected FLAG immune- <br /> precipitates cells express FLAG- <br /> Eme1. <br />  <br />  <br />  <br /> Vol. 15, February 2004                                                                                                        557 <br />  V. Blais et al. <br />  <br />  <br />  <br />  <br />                                                                                         Figure 6. Suppression Mus81                                                                                         RNAi results reduced viability.   <br />                                                                                         Transfection pSuper-178, pSuper- <br />                                                                                         292, pSuper results  <br />                                                                                         duced Mus81 protein. NonTx, untrans- <br />                                                                                         fected cells. (B) HeLa cells                                                                                         transfected pSuper-178, pSuper-292, <br />                                                                                         pSuper plated low density. Col- <br />                                                                                         onies counted 10 d growth <br />                                                                                         staining Geimsa. Triplicate <br />                                                                                         plates counted point.  <br />                                                                                         sults average independent <br />                                                                                         transfections. <br />  <br />  <br /> grown presence thymidine increase incidence   pSuper-178, pSuper-292 plasmids. Cells processed recombination (Lundin et al., 2002). 24 h   stained &#226;?&#164;-galactosidase activity exactly described <br /> growth normal medium, cells stained &#226;?&#164;-galacto-      similar percentage (84 &#226;&#171;&#190; 2%) &#226;?&#164;-galactosidase pos- <br /> sidase activity frequency recombination         itive cells present cases.  interpret scored. Untransfected cultures generated 1470 &#226;&#171;&#190; 180 recom-     data indicate regulation Mus81 suppresses <br /> binants million cells (Figure 7B). similar number     recombination inactive LacZ alleles LacZ&#226;&#171;&#185; cells seen transfection control vectors.   expression &#226;?&#164;-galactosidase activity se. number recombinants reduced fourfold (p &#226;&#171;&#189; <br /> 0.0003) twofold (p &#226;&#171;&#189; 0.0006) cells transfected Mus81-RNAi plasmids, pSuper-292,          RusA Rescues Mus81-dependent Recombination pSuper-178, respectively. data suggested        RusA rescues meiotic defect hypersensitivity suppression Mus81 reduces mitotic recombination         agents cause replication fork stalling Mus81 mutants indicate Mus81-RNAi interfered &#226;?&#164;-ga-      (Boddy et al., 2001; Doe et al., 2002). reasoned lactosidase expression. determine Mus81-RNAi&#226;&#8364;&#8220;       suppression Mus81 human cells results accumu- <br /> transfected cells capable expressing active &#226;?&#164;-galac-   lation Holliday junctions, reduction recombination <br /> tosidase, cells transfected plasmid carrying     rescued expression active RusA (Bolt single intact copy &#226;?&#164;-galactosidase pSuper,     Lloyd, 2002). shown Figure 7B, expression wild-type <br />  <br />  <br /> 558                                                                                                 Molecular Biology Cell <br />                                                                                               Mus81 Required Recombination <br />  <br />  <br />  <br /> Figure 7. Suppression Mus81  <br /> sults reduced mitotic recombina- <br /> tion rescued expression bacterial Holliday junction resolvase, <br /> RusA.   pLrec contains direct  <br /> peat inactive LacZ genes sep- <br /> arated neomycin resistance <br /> gene (black box). Expression control SV40 promoter <br /> (gray box). Base pairs (693) identi- <br /> cal sequence LacZ alleles indicated arrows. Lx2 inac- <br /> tive insertion site indi- <br /> cated X. &#226;&#338;&#172;Lx1 lacks 727 base pairs 5&#226;&#172;&#732; sequence (Herzing Meyn, <br /> 1993). cell-line GM847L22  <br /> tains single intact copy pLrec <br /> (Prince et al., 2001). (B) Incidence LacZ&#226;&#171;&#185; cells. Cells (5 &#226;&#171;&#187; 105) plated G418-free medium 16 h  <br /> fore transfection indicated <br /> plasmid. DNA trans- <br /> fected kept constant use vector. Thymidine (2 mM) added culture medium cells grown 16 h. Cells cultured normal growth  <br /> dium 24 h, stain- <br /> ing &#226;?&#164;-galactosidase activity. Du- <br /> plicate dishes used monitor <br /> cell number expression Mus81 RusA. Error bars represent data separate experiments. (C) <br /> Ha immune-blots cell lysates indi- <br /> cate active mutant forms RusA expressed compara- <br /> ble levels. (D) Immune-precipitates wild-type RusA active X12 structures 3&#226;&#172;&#732; flaps struc- <br /> tures vitro. 80, 8, 2, 1 <br /> indicate relative proportion enzyme added constant substrate. Mutant versions RusA, RusAN70A, Mus81-Eme1, <br /> Mus81AA, did cleave structure. Wild-type Mus81-Eme1 <br /> converted X12 3&#226;&#172;&#732; flap struc- <br /> tures linear duplex products. <br /> Quantification product yield calculated using Image Quant   <br /> ersham Biosciences, Piscataway, NJ) <br /> software. S, substrate  <br />  <br />  <br />  <br /> Ha-tagged RusA did significantly affect incidence      common substrate vivo. tested ability <br /> recombination cells transfected           RusA Mus81-Eme1 cleave X12 3&#226;&#172;&#732; flap <br /> vector (p &#226;&#171;&#189; 0.65), suggesting level expression,   structure vitro. enzymes recovered RusA does drive increased recombination human              HeLa cells using Ha-antibody. shown Figure <br /> cells. contrast, active, nuclear RusA (RusAWT)        7D, Mus81-Eme1 readily detectable activity cotransfected plasmids encoding Mus81-RNAi             substrates. Mus81-Eme1 showed 10- 12-fold ac- <br /> creased incidence recombination levels seen      tivity 3&#226;&#172;&#732; flap compared X12 untransfected control cultures (Figure 7B). endonuclease       range enzyme used. Likewise, RusA cleaved <br /> dead version RusA (RusAN70) did significantly           X12 structure, 80-fold dilution crease number recombinants Mus81-RNAi trans-            immune-precipitate RusA detectably cleaved X12 <br /> fected cells. Immune blotting showed wild-type       structure. contrast, incubating equal amounts RusA <br /> mutant forms RusA equally expressed (Figure 7C).          3&#226;&#172;&#732; flap structure yielded detectable product. <br /> Ha-immune-precipitates endonuclease activity assays           detector sensitivity set 10-fold higher prod- <br /> confirmed wild-type, mutant RusA, active         uct detected samples RusA Holliday junction (X12) substrate (Figure 7D).           incubated 3&#226;&#172;&#732; flap structure. analysis suggests <br /> observation RusA rescues Mus81-dependent recombi-            RusA cleaves 3&#226;&#172;&#732; flaps vitro, does &#226;&#172;?1000- <br /> nation suggests enzymes acting         fold efficiency acts X12 structure. <br />  <br />  <br /> Vol. 15, February 2004                                                                                                    559 <br />  V. Blais et al. <br />  <br />  <br /> DISCUSSION                                                             hypothesis Mus81 component Holliday <br />                                                                     junction resolvase vivo challenged basis better understand function Mus81 human cells,          ability preferentially cleave branched Y-structures set identify human homolog Mus81                 3&#226;&#172;&#732; flaps vitro. Ciccia et al. (2003) showed recom- <br /> binding partner Eme1 Mms4. human sequence                 binant human Mus81-Eme1 &#226;&#172;&#402;12-fold activity limited homology fission yeast Eme1 identified            assayed using mobile Holliday junction (X26) structure <br /> PSI-BLAST. work progress, groups              compared 3&#226;&#172;&#732; flap replication fork. Cleavage reported identification sequence refer         fixed Holliday junction (X0) &#226;&#172;&#402;75-fold 3&#226;&#172;&#732; flap <br /> Eme1A. Ogrunc Sancar (2003) designated protein             (Ciccia et al., 2003). analysis recombinant Mus81- <br /> HsMms4 showed binds Mus81 vivo           Eme1 agreement previous studies immune complexes transfected Mus81-Mms4 (Eme1A)                  showing Mus81-Eme1 preferentially cleaves 3&#226;&#172;&#732; flaps endonuclease activity. Ciccia et al. (2003) identified    replication forks vitro (Kaliraman et al., 2001;  <br /> Eme1A, use bacterial expression showed          stantinou et al., 2002; Bastin-Shanower et al., 2003; Whitby et proteins associate form endonuclease complex.         al., 2003).  contend ability cleave analysis vitro substrate specificity recombi-      branched structures vitro does disqualify nant Mus81-Eme1 closely parallels Ciccia et al. (2003)      Mus81&#226;&#8364;&#8220;Eme1 complex resolving Holliday junctions indicates Mus81-Eme1 cleaves 3&#226;&#172;&#732; flaps effi-           vivo. data presented presented Ciccia et <br /> ciently Holliday junction structures.                          al. (2003) clearly demonstrate recombinant human <br />    study, variants Eme1 retrieved        Mus81-Eme1 capable resolving Holliday junction human expressed sequence tag collection American             linear duplexes vitro. Mus81-Eme1 does behave like <br /> Type Culture Collection. sequences included             prokaryotic Holliday junction resolvase RuvC, potential polymorphic variations splice variants,          displaying strong sequence selectivity, catalyzing tested ability interact Mus81        symmetrical cleavage junctions (Chen et al., 2001;  support Mus81-associated endonuclease activity.              stantinou et al., 2002).  Mus81-Eme1 capable promoter drove expression, Eme1B               catalyzing biologically critical process converting present slightly higher levels            way DNA junction linear duplexes vitro, <br /> proteins. variants associate       vivo. Partially purified prepara- <br /> activate Mus81 differing degrees. higher endonucle-          tions human Mus81 robust Holliday junction activ- <br /> ase activity Eme1B containing immune-precipitates                ity dependent Mus81 protein (e.g., Figure 2 result mainly increased association          Constantinou et al., 2001). Comparison activity 3HaMus81, possible Eme1B stimulated             relatively weak activity recombinant Mus81-Eme1 <br /> Mus81 activity Eme1A Eme1C. 13-amino                work; Ciccia et al., 2003), suggests cofactors acid sequence insert Eme1A present homolo-           improve Holliday junction resolution activity Mus81- <br /> gous murine sequence (accession  CB182498).              Eme1 extracts (Boddy et al., 2001). identification  presence consensus splice donor <span id='am-15' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acceptor</span>           human Eme1 facilitate characterization ancillary <br /> sites flanking insert suggest construct corre-        factors enhance Mus81-Eme1 function. <br /> sponding Eme1A result incomplete splicing.               Mus81 Eme1 colocalize nucleoli human cells  Eme1A bind activate Mus81 (Ciccia          proteins recruited regions damaged DNA, <br /> et al., 2003; Ogrunc Sancar, 2003). likely source        specifically S phase (Gao et al., 2003; Figure 3).  <br /> Eme1C difficult predict. clone derived           cruitment Mus81 Eme1 localized region UV <br />                                                                     damage, specifically S-phase cells, consistent normal tissue. Relative sequences,                                                                     hypothesis Mus81&#226;&#8364;&#8220;Eme1 complex functions  <br /> lacks residues 303&#226;&#8364;&#8220;331 contains cysteine place                                                                     combination repair human cells. <br /> arginine carboxy terminus protein. substi- <br />                                                                        Sequence structural analysis junction resolving <br /> tution particularly intriguing present block <br />                                                                     enzymes different sources suggests significant homology fission yeast Eme1 (Figure 1A). <br />                                                                     evolved independently different ancestors <br /> Determining variants Eme1 differ-          (Aravind et al., 2000; Lilley White, 2001). Despite entially expressed, single site substitutions        differences, characterized Holliday junction resolving  polymorphic variants human populations,              zymes bind junction DNA dimeric form (Lilley associated human disease           White, 2001). association catalytic units enables                                                            paired cleavage opposed strands way junc- <br />    reported previously, immune-precipi-            tion. use differentially tagged version Mus81 tates 3HaMus81 associated endonuclease activity          Eme1, molecules Mus81 absence transfected Eme1 (Chen et al., 2001).         Eme1 associate. analysis needed deter- <br /> trast, immune-precipitates Eme1 active unless           ability Mus81-Eme1 resolve Holliday <br /> Mus81 cotransfected. Given Mus81 expressed Sf9          junctions linear duplex dependent correct <br /> cells active absence Eme1, assume           coordination active Mus81-Eme1 heterodimers activity Mus81-immune-precipitate reflects ability      vivo. <br /> Mus81 associate endogenous human Eme1. idea               mitotic phenotypes associated loss Mus81- supported substantial increase Mus81-associated        function fission yeast budding yeast similar. <br /> endonuclease activity detected transfection Eme1.         Disruption Mus81 results increased sensitivity geno- <br /> reason transfected Eme1 fails form active complex        toxic agents impede DNA replication Mus81 mu- <br /> endogenous Mus81 clear. Transfection Eme1A C           tation lethal combination disruption RecQ <br /> leads reproducible reduction activity associated        homologs.  yeast viable absence endogenous Mus81 (Figure 2B), suggesting trans-           Mus81 (Boddy et al., 2000; Interthal Heyer, 2000). fected Eme1A C disrupt endogenous Mus81              step directly determining role Mus81-Eme1 taining complex.                                                    human cells, used RNAi suppress expression  <br />  <br /> 560                                                                                                    Molecular Biology Cell <br />                                                                                                         Mus81 Required Recombination <br />  <br />  <br /> Mus81. analysis allowed draw conclu-            nealing (SDSA), strand sister chromatid sions function Mus81 human cells.        used template extension free 3&#226;&#172;&#732; end (Kaliraman regulation Mus81 results significant loss     et al., 2001; Constantinou et al., 2002; Bastin-Shanower et al., cell viability. Suppression Mus81 protein expression         2003). SDSA attractive model double strand break maximal 48 &#226;&#8364;&#8220;72 h transfection;           repair. accomplished forming time distinct morphological cell cycle defect        Holliday junction, account mitotic recombination determined. later time points, number           crossover (Paques Haber, 1999).  <br /> detached, dead cells culture increased line         models SDSA based presence double <br /> reduced number Mus81-RNAi&#226;&#8364;&#8220;treated cells               strand ends generated break. case replication <br /> able form viable colonies. Analysis Mus81-RNAi&#226;&#8364;&#8220;trans-        restart, cleavage regressed fork generate fected cultures failed define clear cause cell death.      double strand end.  replication fork recapture <br /> Mus81-Eme1 required repair problems arising           inevitably results formation Holliday <br /> DNA replication yeast. humans, Mus81-Eme1              junction (McGlynn Lloyd, 2002). Mus81 deficiency <br /> cruited sites DNA damage, specifically cells     resulted failure cleave 3&#226;&#172;&#732; flap generated SDSA, replicating DNA (Gao et al., 2003). possible repli-     lead Holliday junction accumulation, cation problems arise frequently human cells,          expect expression Holliday junction <br /> Mus81 essential absence extraneous replica-          resolvase rescue defect SDSA. ability RusA tional stress. determine         rescue Mus81-dependent recombination suggests Hol- <br /> germ-line disruption Mus81 Eme1 effect          liday junctions accumulate Mus81-deficient cells. Clearly, acute loss somatic cells.                                    interpretation dependent knowing specificity <br />    second finding studies inhibition         RusA. ability RusA act retain <br /> Mus81 results reduced mitotic recombination human            activity expressed fused nuclear localization signals <br /> cells. possibility reduced recombination rate         yeast human cells exploited indirect consequence loss viability associ-       studies (Doe et al., 2000; Boddy et al., 2001; Doe et al., 2002; <br /> ated Mus81-RNAi considered.                 Saintigny et al., 2002; Bastin-Shanower et al., 2003; Odagiri et <br /> Mus81-RNAi&#226;&#8364;&#8220;treated cells expressed active &#226;?&#164;-galactosidase          al., 2003). Extensive analysis RusA shown frequency level control cells       nuclease activity highly specific Holliday junctions single intact copy LacZ open-reading frame           unlikely cleave structures vivo (Bolt present plasmid. rate recombination            Lloyd, 2002; Doe et al., 2002). Consistent reports, <br /> calculated incidence Lac&#226;&#171;&#185; cells million live cells   HeLa cell expressed RusA did detectably time harvest;  Mus81-RNAi cells      cleave 3&#226;&#172;&#732; flap structures, protein robust <br /> lost point included analysis.       activity X12 structure. data suggest criteria trypan blue exclusion, cell cycle profile      expressed heterologous  RusA retains &#226;?&#164;-galactosidase expression remaining cells viable         preference symmetrical structures. capable expressing active &#226;?&#164;-galactosidase time       know certain targets RusA vivo, available analysis. important determine             data strongly support hypothesis acts Mus81-deficient cells die unable process       Holliday junctions.  rescue Mus81-dependent <br /> intermediates arise specifically DNA replication.      recombination Holliday junction resolvase, coupled <br /> Mus81 protein detectable quiescent cells.         biochemical evidence Mus81-Eme1 resolves Hol- <br /> crease abundance seen serum stimulation reentry          liday junction vitro, suggests Mus81-Eme1 resolves replicative cycle suggest Mus81 required specif-      Holliday junctions human cells. <br /> ically cells actively dividing (Gao et al., 2003). <br />    Evidence Mus81-Eme1 active 3&#226;&#172;&#732; flap repli- <br /> cation fork structures Holliday junctions vitro      ACKNOWLEDGMENTS <br /> predict number possible roles Mus81-Eme1            grateful Ray Monnat providing GM847L22 advice use combination repair. role directly cleaving replication        cells, Reuven Agami providing pSuper, members forks proposed (Kaliraman et al., 2001;               Scripps Cell Cycle group encouragement advice. P.-H.L.G.                                                                    M.N.B. Research Special Fellows Leukemia Lymphoma Society. <br /> stantinou et al., 2002; Doe et al., 2002).  recently   work funded National Cancer Institute grant awarded emerged lines evidence suggest Mus81 does act          C.H.M. <br /> directly replication forks vivo.  extreme sen- <br /> sitivity yeast strains lack Mus81 camptothecin, topoisomerase inhibitor thought cause fork col-         REFERENCES <br /> lapse, strongly suggests Mus81 required repli-       Aravind, L., Makarova, K.S., Koonin, E.V. (2000). Survey summary: <br /> cation fork collapse (Doe et al., 2002; Bastin-Shanower et al.,    Holliday junction resolvases related nucleases: identification new <br /> 2003) consistent hypothesis Mus81         families, phyletic distribution evolutionary trajectories. Nucleic Acids <br />                                                                    Res. 28, 3417&#226;&#8364;&#8220;3432. <br /> activity required cleave stalled forks. Second, mutations genes act early recombination (Rad51, Rad52,        Bastin-Shanower, S , Fricke, W.M., Mullen, J.R., Brill, S.J. (2003). Rad54) suppress synthetic lethality Mus81-Sgs1              mechanism mus81-mms4 cleavage site selection distinguishes                                                                    homologous endonuclease rad1-rad10. Mol. Cell Biol. 23, 3487&#226;&#8364;&#8220;3496. <br /> strains (Fabre et al., 2002; Bastin-Shanower et al., 2003). Mus81-Eme1 acts directly replication forks growth           Boddy, M.N., Gaillard, P.H., McDonald, W.H., Shanahan, P., Yates, J.R.,                                                                    Russell, P. (2001). Mus81-eme1 essential components Holliday junc- <br /> defects rescued disruption genes.         tion resolvase. Cell 107, 537&#226;&#8364;&#8220;548. <br />    Does observation expression RusA rescues                                                                    Boddy, M.N., Lopez-Girona, , Shanahan, P., Interthal, H., Heyer, W.D., Mus81-dependent recombination defect differentiate role        Russell, P. (2000). Damage tolerance protein mus81 associates FHA1 Mus81 remaining pathways? observation           domain checkpoint kinase cds1. Mol. Cell Biol. 20, 8758 &#226;&#8364;&#8220; 8766. <br /> Mus81-Eme1 efficiently cleaves 3&#226;&#172;&#732; flaps vitro suggests        Bolt, E.L., Lloyd, R.G. (2002). Substrate specificity RusA resolvase <br /> role trimming flaps arise extension 3&#226;&#172;&#732;    reveals DNA structures targeted RuvAB RecG vivo. Mol. Cell <br /> end process synthesis-dependent strand            10, 187&#226;&#8364;&#8220;198. <br />  <br />  <br />  <br /> Vol. 15, February 2004                                                                                                                       561 <br />  V. Blais et al. <br />  <br /> Brummelkamp, T.R., Bernards, R., Agami, R. (2002). stable          Kaliraman, V., Mullen, J.R., Fricke, W.M., Bastin-Shanower, S , Brill, S.J. <br /> expression short interfering RNAs mammalian cells. Science 296, 550 &#226;&#8364;&#8220;         (2001). Functional overlap Sgs1-Top3 Mms4-Mus81 endo- <br /> 553.                                                                                nuclease. Genes Dev. 15, 2730 &#226;&#8364;&#8220;2740. <br /> Chen, X.B., Melchionna, R., Denis, C.M., Gaillard, P.H., Blasina, , Van        Karow, J.K., Constantinou, , Li, J.L., West, S.C., Hickson, D. (2000). <br /> Weyer, , Boddy, M.N., Russell, P., Vialard, J., McGowan, C.H. (2001).         Bloom&#226;&#8364;&#8482;s syndrome gene product promotes branch migration Holliday <br /> Human Mus81-associated endonuclease cleaves Holliday junctions vitro.            junctions. Proc. Natl. Acad. Sci. USA 97, 6504 &#226;&#8364;&#8220; 6508. <br /> Mol. Cell 8, 1117&#226;&#8364;&#8220;1127. <br />                                                                                     Lilley, D., White, M. (2001). junction-resolving enzymes. Nat. Rev. <br /> Ciccia, , Constantinou, , West, S.C. (2003). Identification charac-     Mol. Cell Biol. 2, 433&#226;&#8364;&#8220; 443. <br /> terization human Mus81/Eme1 endonuclease. J. Biol. Chem. 278, <br /> 25172&#226;&#8364;&#8220;25178.                                                                        Lundin, C., Erixon, K., Arnaudeau, C., Schultz, N., Jenssen, D., Meuth, M., <br />                                                                                     Helleday, T. (2002). Different roles nonhomologous end joining Constantinou, , Chen, X.B., McGowan, C.H., West, S.C. (2002). Holliday        homologous recombination following replication arrest mammalian cells. <br /> junction resolution human cells: junction endonucleases distinct        Mol. Cell Biol. 22, 5869 &#226;&#8364;&#8220;5878. <br /> substrate specificities. EMBO J. 21, 5577&#226;&#8364;&#8220;5585. <br />                                                                                     McGlynn, P., Lloyd, R.G. (2002). Recombinational repair restart Constantinou, , Davies, , West, S.C. (2001). Branch migration         damaged replication forks. Nat. Rev. Mol. Cell Biol. 3, 859 &#226;&#8364;&#8220; 870. <br /> Holliday junction resolution catalyzed activities mammalian cells. <br /> Cell 104, 259 &#226;&#8364;&#8220;268.                                                                 Melchionna, R., Chen, X.B., Blasina, , McGowan, C.H. (2000). Threonine <br />                                                                                     68 required radiation-induced phosphorylation activation cds1. <br /> Constantinou, , Tarsounas, M., Karow, J.K., Brosh, R.M., Bohr, V , Hick- <br />                                                                                     Nat. Cell Biol. 2, 762&#226;&#8364;&#8220;765. <br /> son, D., West, S.C. (2000). Werner&#226;&#8364;&#8482;s syndrome protein (WRN) migrates <br /> Holliday junctions localizes RPA replication arrest. EMBO          Meyn, M.S. (1993). High spontaneous intrachromosomal recombination rates <br /> Rep. 1, 80 &#226;&#8364;&#8220; 84.                                                                    ataxia-telangiectasia. Science 260, 1327&#226;&#8364;&#8220;1330. <br /> Cox, M.M., Goodman, M.F., Kreuzer, K.N., Sherratt, D.J., Sandler, S.J.,         Michel, B. (2000). Replication fork arrest DNA recombination. Trends <br /> Marians, K.J. (2000). importance repairing stalled replication forks.        Biochem. Sci. 25, 173&#226;&#8364;&#8220;178. <br /> Nature 404, 37&#226;&#8364;&#8220; 41. <br />                                                                                     Mizuno, T., Matsunaga, T., Ihara, M., Nikaido, O. (1991). Establishment Los Santos, T., Hu <br /> </body></html>