<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Trypanosoma brucei Tb927.2.6100 Essential Protein Associated Kinetoplast DNA <br /> Kirsten Beck Nathalie Acestor Anjelique Schulfer Atashi Anupama Jason Carnes Aswini K. Panigrahi,b Ken Stuarta Seattle Biomedical Research Institute, Seattle, Washington, USAa; King Abdullah University Science Technology, Thuwal, Saudi Arabiab <br />  mitochondrial DNA trypanosomatid protozoa consists complex, intercatenated network tens maxicircles thousands minicircles. structure, called kinetoplast DNA (kDNA), requires numerous proteins multiprotein com- <br /> plexes replication, segregation, transcription. study, used proteomic approach identify proteins associated kDNA network. identified novel protein encoded Tb927.2.6100 present fraction  <br /> riched kDNA colocalized protein kDNA fluorescence microscopy. RNA interference (RNAi) knockdown expression resulted growth defect changes proportion kinetoplasts nuclei cell population. RNAi resulted shrinkage loss kinetoplasts, loss maxicircle minicircle components kDNA similar rates,  secondarily) loss edited pre-edited mRNA. results indicate Tb927.2.6100 protein essential maintenance kDNA. <br />  <br />  <br />  <br /> T    mitochondrial DNA kinetoplastid protozoa consists      tens maxicircles thousands minicircles, intercatenated single network resides                                                                                      lication, repair, segregation, processing sub- <br />                                                                                      organellar localization. <br />                                                                                          study, proteins kDNA enriched fractions iden- <br /> cell&#226;&#8364;&#8482;s large single mitochondrion flagellar basal                    tified mass spectrometry (MS), localization candidate <br /> body (1&#226;&#8364;&#8220;3). Early cytologists named highly visible region                     proteins determined immunofluorescence. novel pro- <br /> cells stained DNA kinetoplast virtue association                   tein demonstrated colocalize kDNA vivo, base motile flagellum. Subsequently, mito-                      functional role assessed using RNA interference (RNAi). chondrial DNA came called kinetoplast DNA (kDNA) (4).                          protein, Tb927.2.6100, associated kDNA     kDNA maxicircles encode mitochondrial rRNAs                          essential maintenance. <br /> 17 proteins, RNAs undergo posttranscriptional se- <br /> quence remodeling called RNA editing (5&#226;&#8364;&#8220;7). minicircles                       MATERIALS METHODS <br /> code small guide RNAs (gRNAs) specify edited sequences. <br />                                                                                      Trypanosome cell growth. Procyclic-form (PF) T. brucei 164 clone IsTaR <br /> Maxicircle minicircle sizes vary different kinetoplas-                     1.7a grown densities 1&#226;&#171;&#187; 107 2 &#226;&#171;&#187; 107 cells/ml vitro 27&#194;&#176;C <br /> tids, species maxicircle sequences homo-                   SDM-79 medium containing hemin (7.5 mg/ml) 10% (vol/vol) <br /> geneous, minicircle sequences heterogeneous (8,                      fetal bovine serum (FBS). PF T. brucei strain 29.13 (20), contains <br /> 9). specific mRNAs undergo editing extent                        integrated genes T7 polymerase tetracycline repressor, editing vary different kinetoplastids,                    grown presence G418 (15 &#226;?&#174;g/ml) hygromycin (25 &#226;?&#174;g/ml). <br /> reflection minicircle sequence diversity (10).                                cells harvested centrifugation 6,000 &#226;&#171;&#187; g 10 min 4&#194;&#176;C. <br />     Transcription maxicircles minicircles polycis-                        Plasmid constructs, transfections, induction. create  <br /> tronic, primary transcripts processed cleavage (11,                   structs inducible expression c-myc epitope-tagged proteins T. <br /> 12). addition, transcripts undergo editing                      brucei, open reading frames (ORFs) PCR amplified <br /> prior cleavage step (13). maxicircles minicircles                    genomic DNA T. brucei strain Lister 427. detailed list                                                                                      tagged proteins primers used amplify selected ORFs provided <br /> appear transcribed single mitochondrial RNA polymer- <br />                                                                                      supplemental material. PCR products cloned ase, characteristics viral polymerase (14, 15).                pGEM-T Easy vector (Promega), digested BamHI BglII unclear RNA polymerases involved mito-                      HindIII enzymes, ligated pLEW79-MHTAP vector (21, 22). <br /> chondrial transcription.                                                             plasmids linearized NotI enzyme transfected     classes DNA molecules,                   PF T. brucei 29.13 cell line; phleomycin-resistant cell lines selected <br /> different mitochondrial DNA polymerases, specific functions                      checked tetracycline (Tet)-regulated expression. transgenic completely elucidated (16&#226;&#8364;&#8220;18). Replication kDNA network studied  especially Englund laboratory (19). studies revealed somewhat differ-                        Received 20 December 2012 Accepted 2 2012 <br /> ent mechanisms different kinetoplastids. Trypanosoma bru-                         Published ahead print 6 2013 <br /> cei, multiple proteins functionally associate kDNA                         Address correspondence Ken Stuart, ken.stuart@seattlebiomed.org. <br /> ensure proper positioning replication, repair                       K.B. N  contributed equally article. segregation cell division transcription.                      Supplemental material article http://dx.doi.org/10.1128 processing primary transcripts coordinated                       /EC.00352-12. transcription, implied editing preprocessed tran-                     Copyright &#194;&#169; 2013, American Society Microbiology. Rights Reserved. <br /> scripts.  likely set proteins functionally                     doi:10.1128/EC.00352-12 <br /> interact kDNA, directly indirectly, accomplish rep- <br />  <br />  <br /> 970   ec.asm.org                                            Eukaryotic Cell   p. 970 &#226;&#8364;&#8220;978                                                    July 2013 Volume 12 Number 7 <br />                                                                                                                                Novel kDNA-Associated Protein <br />  <br />  <br />  <br /> PF cell lines expressing tandem affinity purification (TAP)-tagged pro-         coupled Texas Red-X-conjugated secondary antibody. 4=,6-di- <br /> tein supplemented 2.5 &#226;?&#174;g/ml phleomycin.                              amidino-2-phenylindole dihydrochloride (DAPI) staining, &#226;&#172;&#402;5 &#226;&#171;&#187; 105 <br />     create construct RNAi Tb927.2.6100 transcript,            noninduced RNAi-induced cells pelleted centrifugation, <br /> fragment 439 bp PCR amplified genomic DNA T. brucei                washed, fixed 4% <span id='am-24' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>, treated 50 &#226;?&#174;l 1-&#226;?&#174;g/ml <br /> strain Lister 427 using sense primer 5=-ATACTCGAGGAGGCTCTA                    DAPI visualize DNA. Phase-contrast images cells fluo- <br /> GCAGCAGAGGA-3= antisense primer 5=-ATAAAGCTTGAGTAAC                           rescence captured Nikon fluorescence microscope equipped <br /> TGGGGCTGCTACG-3=. resulting PCR product cloned                       camera appropriate filters. <br /> pZJM plasmid (23, 24) XhoI HindIII restriction sites. RNAi cell               RNA isolation quantitative real-time PCR. Total RNA har- <br /> lines generated transfection 10 &#226;?&#174;g NotI-linearized pZJM           vested cell lines using TRIzol reagent according man- <br /> construct described previously (25). independent cell lines         ufacturer&#226;&#8364;&#8482;s instructions. micrograms RNA treated DNase <br /> selected, growth curves generated absence presence          using DNA-free kit (Ambion) used templates reverse <br /> 1 &#226;?&#174;g/ml tetracycline, induces expression double-stranded              transcription-PCR (RT-PCR) analysis. Real-time RT-PCR measure RNA (dsRNA). induced noninduced cultures maintained               relative abundance mRNAs performed previously described 2 &#226;&#171;&#187; 106 2 &#226;&#171;&#187; 107 cells/ml maintain log-phase growth,            (35). primer pairs used amplify 18S rRNA &#226;?&#164;-tubulin cDNAs, <br /> cell density monitored daily using particle counter (Beckman).              used reference genes, A6 (pre-edited edited), cy- <br />     kDNA isolation. Mitochondrial (mt) vesicles isolated hypo-            tochrome oxidase (COI), 9S/ND8, RPS12/ND5 described <br /> tonic lysis enriched density gradient flotation 20 35% linear        previously (35, 36). sequences primers maxicircles fol- <br /> Percoll gradients described previously (26). enriched vesicles        lows: forward, GGTTCATAGAGGAGAATGGTTCAAG, reverse, CC <br /> solubilized adding n-dodecyl-&#226;?&#164;-D-maltoside (DDM) corresponding              TCCACTTACACAATCTTTTGCTT. minicircles follows: DDM/protein ratio 2 (g/g) incubated ice 30 min. Following         forward, GGCGTGCAAAAATACACATACAC, reverse, GTACCTC <br /> centrifugation 12,000 &#226;&#171;&#187; g 20 min 4&#194;&#176;C, pellet washed thrice      GGACCTCAAAAATGC. average cycle threshold (CT) values buffer containing 0.5 M sucrose, 20 mM Tris-HCl (pH 7.6), 0.1 mM             target used calculations. relative standard devia- <br /> EDTA, 0.8 mM <span id='am-19' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-20' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span>spermidine</span>, 1 &#226;?&#174;g/ml pepstatin, 2 &#226;?&#174;g/ml leupeptin, 1            tion measured CT values quantitative PCR (qPCR) assays  <br /> mM Pefabloc, 0.5% Nonidet P-40 (NP-40). centrifugation               tween 0.03% 1.77%. Analysis carried using Pfaffl method, 15,000 &#226;&#171;&#187; g 20 min 4&#194;&#176;C, pellet used kDNA          PCR efficiencies calculated linear regression LinRegPCR <br /> fraction.                                                                         software (37, 38). Data normalized 18S rRNA &#226;?&#164;-tubulin,     Tandem affinity purification. Expression TAP-tagged Tb927.2.6100           relative changes mRNA abundance RNAi induction ex- induced tetracycline (100 ng/ml culture) 48 h, 500 ml      pressed fold changes relative noninduced control cells. <br /> cells harvested centrifugation density &#226;&#172;&#402;9 &#226;&#171;&#187; 106 cells/ml. Tagged       quantification minicircles maxicircles, total DNA <br /> complex purified cell lysate prepared step lysis Nonidet     isolated cells collected RNAi, dyskinetoplastic <br />                                                                                   EATRO164 blood form (DK164) cells  lack kDNA) used P-40 detergent described previously (27, 28). Briefly, cells lysed <br />                                                                                   control (39, 40). cells harvested washed phos- 0.25% Nonidet P-40 cleared low-speed centrifugation,                                                                                   phate-buffered saline (PBS) lysed PBS containing 0.5% SDS <br /> supernatant lysed 1.5% Nonidet P-40 cleared high- <br />                                                                                   0.2 mg/ml proteinase K 4 h 56&#194;&#176;C. DNA treated speed centrifugation. tagged complexes isolated sequential IgG <br />                                                                                   0.1 mg/ml RNase 37&#194;&#176;C 30 min extracted phenol- <br /> affinity calmodulin affinity columns (29). <br />                                                                                   chloroform-isoamyl alcohol (25:24:1) precipitated ethanol.     Protein identification LC-MS-MS. Proteins analyzed liq- <br />                                                                                   pellets resuspended water stored &#226;&#171;&#186;20&#194;&#176;C uid chromatography-tandem mass spectrometry (LC-MS-MS)  <br />                                                                                   analyzed. control mtDNA isolated wild-type <br /> scribed previously (30). Briefly, proteins enriched kDNA frac- <br />                                                                                   (WT) PF 29.13 cells. <br /> tion separated 10% SDS-PAGE gels visualized SYPRO <br />                                                                                       quantitative PCR minicircles maxicircles performed <br /> ruby staining. entire gel lane divided 9 pieces, pro- <br />                                                                                   essentially described cDNAs, differences teins digested gel sequencing-grade modified trypsin;                                                                                   use EvaGreen (Bio-Rad) PCR mix experimental deter- <br /> resulting peptides extracted 50% acetonitrile&#226;&#8364;&#8220;5% <span id='am-30' about='obo:IMR_0200110' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00058" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200110" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Formic acid" datatype="xsd:string"></span>formic acid</span>            mination optimal concentration DNA preparation; 7.5 pg dried SpeedVac. proteins isolated TAP-tagged                    DNA used samples Fig. 3B C. reference genes, Tb927.2.6100, 200 &#226;?&#174;l calmodulin eluate digested solution,           &#226;?&#164;-tubulin gene 18S rRNA gene tested. reference <br /> peptides extracted described  peptides fraction-           genes gave similar results,  &#226;?&#164;-tubulin gene used <br /> ated nanoflow C18 liquid chromatography analyzed using LTQ               reference gene. kDNA content quantified relative gene <br /> linear <span id='am-6' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> trap mass spectrometer. collision-induced dissociation             using PCR efficiency Pfaffl method described  <br /> spectra compared Tb927, version 4.0, T. brucei protein <br /> database downloaded GeneDB using TurboSEQUEST software, protein matches determined using PeptideProphet Pro-                     RESULTS <br /> teinProphet software (31, 32).                                                    Protein identification. kDNA isolated Percoll gra- <br />     Sequence analysis identified proteins. probable functions           dient-enriched mitochondria, proteins fraction proteins assigned based GeneDB annotation, proteins           identified mass spectrometry. total 2,573 unique unknown function, possible motifs domains searched            peptides identified LC-MS-MS analyses. InterPro (http://www.ebi.ac.uk/Tools/InterProScan/), Pfam (http://pfam            proteins matched pep- <br /> .sanger.ac.uk/), NCBI CDD (http://www.ncbi.nlm.nih.gov/Structure              tide hits, 338 proteins identified single <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> hit, <br /> /cdd/wrpsb.cgi) databases. Alignment Tb927.2.6100 orthologs            175 (52%) protein identification probability formed using MUSCLE (version 3.8.31) (33). Mitochondrial targeting <br />                                                                                   &#226;&#177;&#8211;0.99, 109 (32%) 0.98, rest 0.91 signal Tb927.2.6100 predicted using MitoProt II (version 1.101) <br /> (34). <br />                                                                                   0.97; 510 (98%) proteins identified pep- <br />     Immunofluorescence microscopy. Subcellular localizations               tide matches protein identification probability &#226;&#177;&#8211;0.99, 7 <br /> expressed tagged proteins cell determined immuno-              0.98, rest 0.93 0.97. identified <br /> fluorescence analysis (IFA) using anti-c-myc antibody (Invitrogen)          peptide sequences associated probability values presented <br /> scribed previously (27). Colocalization analysis performed using              Table S1 supplemental material. 861 proteins <br /> monoclonal antibody (MAb) 78 (anti-mt heat shock protein 70) (27)                 identified study, 44 previously detected  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                 ec.asm.org 971 <br />  Beck et al. <br />  <br />  <br />  <br />                                                                                         43), DNA topoisomerase II (Tb09.160.4090) (44), kDNA-associated <br />                                                                                         proteins (Tb10.6k15.1460 Tb927.8.7260), components                                                                                         HslVU protease complex (Tb11.01.2000, Tb11.01.4050,                                                                                         Tb927.5.1520) (45). <br />                                                                                             Subcellular location proteins. order determine <br />                                                                                         identified proteins colocalize kDNA <br /> FIG 1 Tb927.2.6100 associates kDNA. tagged protein visu-               T. brucei PF cells, protein candidates selected based <br /> alized fluorescence microscopy using polyclonal anti-c myc antiserum cou- <br /> pled fluorescein isothiocyanate (FITC)-conjugated secondary antibody.              following criteria:   identification tryptic <br /> Panels phase-contrast light microscopy PF T. brucei cells, anti-c-myc           peptides, (ii) absence known function (&#226;&#8364;&#339;hypothetical  <br /> antibody coupled FITC-conjugated secondary antibody showing local-                 served&#226;&#8364;? proteins T. brucei genome database), (iii) pI ization target protein, DAPI staining nucleus kDNA, merge.            &#226;&#172;&#381;9.5, (iv) domains motifs, (v) mt localization sequence <br />                                                                                         predicted available software MitoProt (http://ihg.gsf ihg <br />                                                                                         /mitoprot.html) SignalP (http://www.cbs.dtu.dk/services <br /> prior proteomic analyses T. brucei mitochondrion (30, 41, <br />                                                                                         /SignalP/). criteria led set 64 hypothetical proteins, 42)  Table S2 supplemental material). <br />                                                                                         34 assigned mt protein complex <br />     861 different proteins identified kDNA <br />                                                                                         mt membranes (30, 36, 41, 46). remaining 30 pro- <br /> fraction mitochondrial, finding reflects fact large, structurally complex mitochondrion disrupted                         teins, 18 pI &#226;&#172;&#381;9.5 TAP tagged, subcellular reseals vesicles isolation, contributing cross                     localization assessed immunofluorescence analysis (IFA) <br /> tamination. Based available GeneDB annotation, keyword                               induction expression PF T. brucei cells (Fig. 1; searches, literature searches  genes                           Fig. S1 supplemental material).  misannotated), 424 proteins (&#226;&#172;&#402;49%) mitochondrial 105                            shown protein p166, acidic pI 5.32, <br /> proteins (&#226;&#172;&#402;12%) likely contaminants enriched kDNA                            localizes kDNA (47).  second round fraction, assignable non-mt locations, including                      selection, protein candidates selected pI glycosomes, endoplasmic reticulum, nucleus, Golgi, lyso-                        &#226;&#172;?9.5  Fig. S1). <br /> somes  Table S3 supplemental material). remaining                            Immunofluorescence assay using anti-myc MAb (anti-tag) <br /> 332 proteins (&#226;&#172;&#402;39%) annotated localization;                            showed 24 tagged successfully expressed pro- <br /> likely mitochondrial proteins location dem-                    teins (Table 1), Tb927.2.6100 exclusively  <br /> onstrated, nonmitochondrial contaminants,                       localized kDNA (Fig. 1). remaining proteins, observed previous analyses (30). expected, proteins                  IFA showed 15 evenly distributed known associated kDNA identified sample,                        reticulated mitochondrion did intensely <span id='am-29' about='rdfs:label' typeof='owl:Thing'>label</span> kin- <br /> including DNA polymerases (Tb927.5.2780 Tb927.5.2790) (17,                          etoplast area, 3 localized nucleus, 5 appeared <br />  <br />  <br /> TABLE 1 Subcellular localization tagged proteins <br />                                                                                                                              Previous mt <br />                                Theoretical pI/molecular                                                                      assignment        New assignment <br /> Protein namea                  mass (kDa)                         Motif domain                                            confidenceb       (IFA based) <br /> Tb927.2.6100TAP233             10.77/53.28                                                                                   Low               mt-kDNA <br /> Tb927.10.2970TAP245            7.74/13.79                                                                                    Low               mt <br /> Tb927.5.4360TAP246             6.95/36.43                         S-Adenosylmethionine-dependent methyltransferase           Low               mt <br /> Tb927.4.3080TAP247             8.58/18.44                         DUF167                                                     Low               mt <br /> Tb09.211.0530TAP234            10.07/30.41                                                                                   Low               mt <br /> Tb927.3.1810TAP235             9.75/36.1                                                                                     Low               mt <br /> Tb927.10.1860 TAP273           9.61/39.1                                                                                     Low               Nuclear <br /> Tb11.01.3970TAP231             10.24/23.49                                                                                   Moderate          mt <br /> Tb927.5.740 TAP269             10.11/23.91                                                                                   Moderate          mt <br /> Tb11.01.8500 TAP270            9.91/25.87                                                                                    Moderate          mt <br /> Tb11.02.1860TAP268             5.52/51.88                         Putative transcriptional regulator                         Moderate          mt <br /> Tb11.01.2740TAP267             5.99/71.77                         AlkB, alkylated DNA repair protein                         Moderate          mt <br /> Tb09.160.2380TAP241            9.89/9.23                          Uncharacterized protein family (UPF0041)                   Moderate          mt <br /> Tb927.10.1660 TAP271           9.73/33.2                                                                                     Moderate          mt <br /> Tb927.5.1130 TAP272            9.59/34.3                                                                                     Moderate          Non-mt <br /> Tb927.3.1720TAP248             5.67/11.17                                                                                    Moderate          Non-mt <br /> Tb927.7.630 TAP274             10.73/28.19                                                                                                mt <br /> Tb927.4.1370TAP237             9.66/17.64                                                                                                 mt <br /> Tb11.02.5230 TAP275            10.25/23.28                                                                                                Nuclear <br /> Tb09.211.1150 TAP277           10.38/20.69                                                                                                Nuclear <br /> Tb09.211.2620TAP238            10.31/22.75                                                                                                Non-mt <br /> Tb927.8.2490TAP250             8.52/35.23                         SET domain                                                 detected      Non-mt   GeneDB accession number (http://www.genedb.org) superscript indicating TAP-tagged identifier. <br /> b <br />   reference 30. <br />  <br />  <br />  <br /> 972   ec.asm.org                                                                                                                                  Eukaryotic Cell <br />                                                                                                                                    Novel kDNA-Associated Protein <br />  <br />  <br />  <br />  <br /> FIG 2 Effects Tb927.2.6100 RNAi.   Cell growth curves representative RNAi cell line Tb927.2.6100 grown presence (open circles) absence <br /> (filled squares) Tet. cumulative cell numbers calculated multiplying cell densities dilution factor. (B) qPCR analysis total RNA isolated cells, expression Tb927.2.6100 repressed 4 days RNAi. relative change Tb927.2.6100 mRNA abundance determined using &#226;?&#164;-tubulin mRNA (right bar) 18S rRNA (left bar) internal control presented log scale. (C) Effect RNAi kinetoplast size. DAPI <br /> staining shows kinetoplasts cells RNAi 4 days RNAi. n, nucleus; k, kinetoplast. (D) Quantitation different cell cycle stages RNAi Tb927.2.6100 showing proportion 200 randomly chosen cells having kinetoplast nucleus (1K1N) kinetoplasts nucleus <br /> (2K1N) reduced RNAi Tb927.2.6100, proportion cells having kinetoplast nucleus (0K/1N) increased dramatically, proportion having reduced-size kinetoplast nucleus (1ksmall1N). <br />  <br />  nonmitochondrial  Fig. S1 supplemental mate-                      encoded single-copy gene, predicted product 487 <br /> rial).  definitive localization assigned                <span id='am-12' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> residues pI 10.77. Bioinformatic analyses proteins indistinct speckled fluorescence;                Tb927.2.6100 predict N-terminal mitochondrial targeting sig- <br /> haps expressed proteins folded way prevented                    nal known motif domain. Homologous genes pres- tag recognized MAb.                                          ent related Trypanosoma species Leishmania species. <br />     Based IFA results, seven proteins pre-                 sequence alignment orthologs T. brucei gambiense, T. <br /> viously assigned mitochondrion moderate confidence                     cruzi, T. congolense, T. vivax showed conservation puta- low confidence (30) clearly localized mito-                tive mitochondrial import sequence regions near N C <br /> chondrion, protein (Tb927.10.1860) earlier assigned                    termini divergence regions  Fig. S2 mitochondrion low confidence localized nucleus                  supplemental material). evaluate Tb927.2.6100 es- <br />  Fig. S1 supplemental material).                                        sential T. brucei, expression knocked PF cells     Tandem affinity purification tagged Tb927.2.6100.                     RNAi. Growth inhibition apparent day 5 lasted <br /> termine proteins associate Tb927.2.6100, tan-                      10 days compared noninduced cell lines (Fig. <br /> dem affinity purification performed cells expressing                      2A). Induction RNAi expression resulted &#226;&#172;&#402;80% knock- <br /> Tb927.2.6100TAP, purified proteins trypsin digested                       Tb927.2.6100 mRNA abundance 4 days compared analyzed mass spectrometry  Table S4 sup-                        noninduced cells (Fig. 2B). Growth resumed &#226;&#172;&#402;10 <br /> plemental material). proteins identified mass                         days, indicating cells increasingly resistant spectrometry, proteins previously identified associating                   RNAi knockdown, previously reported T. brucei mitochondrial ribosome  Tb10.70.7960,                          (50). Tb927.2.6100 protein levels directly assessed Tb11.01.1840, Tb11.02.2250, Tb11.02.3180, Tb927.1.2990,                            RNAi cells lack antibody specific Tb927.3.5610, Tb927.6.3930, Tb927.7.3430, Tb927.7.3510                         Tb927.2.6100. growth defects resulting repression (42).  Tb11.02.3180 Tb10.70.7960                         Tb927.2.6100 imply essential protein PF T. brucei. previously identified MRB1                            Effect Tb927.2.6100 RNAi mitochondrial DNA. RNAi <br /> complex, involved mitochondrial RNA processing                         proteins closely associated kDNA shown thought mediate exchange gRNAs RNA                             cause kDNA loss (16, 47, 51&#226;&#8364;&#8220;53), maintenance kDNA editing (36, 48, 49).                                                              Tb927.2.6100 RNAi examined. DAPI staining revealed     Effect Tb927.2.6100 RNAi cell growth. Tb927.2.6100                    Tb927.2.6100 RNAi caused shrinking disappearance  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                       ec.asm.org 973 <br />  Beck et al. <br />  <br />  <br />  kDNA network. Fig. 2C, image shows DAPI-stained           maxicircle gave similar results, confirming cells (noninduced) normal nucleus kinetoplast.         measured reduction correct. qPCR minicircle DNA <br /> trast, image (4 days RNAi induction) shows cells        showed similar pattern: 4 days changes lost majority kinetoplast staining. RNAi         small, variation higher maxicircle, <br /> cells retain kDNA, DAPI staining showed obvious            2.4-fold increase day 4 (Fig. 3B). days 5 6, crease size kinetoplasts, point barely   minicircle DNA decreased 30% levels non- <br /> detectable (Fig. 2C; Fig. S3 supplemental material).     induced cells. shows decrease kDNA RNAi <br />     effects cell division kinetoplast segregation        Tb927.2.6100 concurrent loss maxicircles monitored DAPI staining (Fig. 2D). cells G1 S         minicircles. <br /> phases nucleus kinetoplast (1N1K).              additional control, mitochondrial DNA isolated mt division precedes nuclear division, cells G2/M phase          days 2, 4, 6 growth Trypanosoma brucei PF 29.13 cells nucleus kinetoplasts (1N2K), cells undergoing            editing exonuclease KREX2 repressed RNAi <br /> cytokinesis nuclei kinetoplasts (2N2K).              (54). KREX2 essential (55), cells suitable <br /> evaluated proportion cells nuclear mitochondrial         control mitochondrial protein lacking, DNA 200 cells sample randomly chosen.            decrease maxicircle minicircle DNA expected. <br /> control Tb927.2.6100 cell line (noninduced cells), &#226;&#172;&#402;90%           qPCR showed change abundance cells normal-size single nucleus single       mitochondrial DNA, small increase, kinetoplast (1N1K); rest cells (&#226;&#172;&#402;10%) 1N2K        2-fold, minicircle 6 days (Fig. 3C). shows 2N2K.  RNAi-induced cells, reduction              observed decrease mitochondrial DNA RNAi proportion cells G1/S phase (1N1K) proportional           Tb927.2.6100 general reaction cell crease cells lacking kinetoplast (1N0K) observed.        depletion mitochondrial protein specific response. <br /> percentage 1N0K cells increased time: 55% day 2, 60%       Southern blot analysis showed minicircles max- <br /> day 4, 75% day 6 (Fig. 2D).  percentage           icircles diminished course RNAi knockdown 1N1K cells harboring smaller kinetoplast remained constant             Tb927.2.6100 (results shown). time, &#226;&#172;&#402;20%. results suggest Tb927.2.6100               decrease mitochondrial DNA accom- involved controlling replication segregation         panied reduction abundance pre-edited kDNA network.                                                            edited kDNA transcripts. assessed using qPCR RNA <br />     assess decrease total kDNA specific        isolated cells RNAi Tb927.2.6100 induced. <br /> reduction minicircle maxicircle component           transcripts examined pre-edited edited A6, COI mitochondrial DNA, quantitative PCR (qPCR) performed                 does edited, regions 9S/ND8 RPS12/ND5 using primers binding maxicircle minicircle.        span site cleaved processing polycis- <br /> DNA DK164, does contain mitochondrial DNA,                tronic primary transcript. transcripts showed slight used control, testing 7.5 pg DNA reaction;           reduction 2 days dramatic reduction, 90% <br /> mitochondrial DNA tested 3 dilutions 75, 7.5, 0.75         (apart edited A6, reduction 75%), compared <br /> pg reaction. analysis threshold cycle shows          levels noninduced cells day 6 (Fig. 4). results amplification using DK164 DNA tem-            expected serve additional control plate (CT &#226;&#172;&#381; 30) maxicircle primer pairs. shows         depletion Tb927.2.6100 leads massive decrease mito- nonspecific binding primers nuclear DNA,         chondrial DNA subsequently mitochondrial transcripts. <br /> making suitable quantify mitochondrial DNA (Fig.                  results confirmed loss kinetoplast <br /> 3A). qPCR serial dilutions mitochondrial DNA pro-              inhibition Tb927.2.6100 showed maxicircle duced CT values reflecting robust amplification (14 19       minicircle DNA disappear comparable rates concentration range), demonstrating specific amplification        time period result RNAi knockdown.  possible mitochondrial targets. linearity CT value plot-      determine protein involved replication ted DNA concentration (R2 &#226;&#172;&#381; 0.9 primer           maintenance  results point pairs) shows primers used quantify mito-        Tb927.2.6100 having general function regulation chondrial DNA. ensure primers amplify         kinetoplast copy numbers. intended target, qPCR amplicons ligated pGEM-T Easy vector sequenced, confirmed speci-             DISCUSSION <br /> ficity primers.                                                   multitude proteins required replication, segre- <br />     Quantitative PCR performed using total DNA purified              gation, transcription complex trypanosome mito- cells RNAi Tb927.2.6100 noninduced               chondrial DNA, known kDNA, consists intercat- <br /> cells controls time points. DNA isolated          enated network tens maxicircles thousands day time course 6 days, 6 days corresponds       minicircles (19). identified novel protein, Tb927.2.6100, development phenotype (Fig. 2A); point,       MS analysis proteins associated enriched <br /> growth stalls depletion Tb927.2.6100. qPCR             kDNA. sequence analysis did identify known <br /> shows 4 days levels maxicircle DNA changed      motifs, alignment orthologs trypanosomes  slightly  maximum increase 1.5-fold),          vealed regions sequence conservation near N C ter- <br /> days 5 6 RNAi maxicircle DNA decreased dramatically,          mini involved protein function. showed eventually 20% compared noninduced cells (Fig. 3B).            protein localizes exclusively kDNA area mito- <br /> Using different primer pairs targeting different parts        chondrion. Furthermore, demonstrated RNAi  <br />  <br /> 974   ec.asm.org                                                                                                                  Eukaryotic Cell <br />                                                                                                                                Novel kDNA-Associated Protein <br />  <br />  <br />  <br />  <br /> FIG 3 Quantitative PCR kDNA.   Threshold cycles using different amounts DNA DK164 cells  mitochondrial DNA) kDNA PF 29.13 <br /> cells. Primers target maxicircle (max1, A6 pre) minicircle (min2) DNA showed significant amplification mitochondrial DNA (DK164), good amplification obtained using kDNA template. Error bars represent standard deviations measured CT replicates. (B) RNAi Tb927.2.6100 results decreases maxicircle minicircle DNAs, maxicircle decreasing larger extent (&#226;&#172;&#402;20%) minicircle (&#226;&#172;&#402;30%). <br /> (C) Maxicircle minicircle abundance altered KREX2 repression RNAi, control indicates effects Tb927.2.6100 RNAi specific. <br /> &#226;?&#164;-Tubulin used internal reference relative comparisons. <br />  <br />  <br /> presence essential kDNA maintenance rep-                     DAPI staining loss kDNA sequence qPCR.  lication.  association kDNA functional                loss kDNA DAPI staining loss maxicircle tran- physical. results RNAi studies indicate                  scripts qPCR occurred day 2, loss kDNA function protein exclusive max-                 qPCR occurred day 5. difference probably reflects fact <br /> icircle minicircle components, knockdown resulted                   maxicircle DNA abundance measured qPCR, loss molecules similar rates.                                    does require maxicircles remain intact     Loss kDNA following RNAi observed loss                template DNA present.  kDNA transcripts rap- <br />  <br />  <br /> July 2013 Volume 12 Number 7                                                                                                                  ec.asm.org 975 <br />  Beck et al. <br />  <br />  <br />  <br />                                                                                  Tb927.2.6100 itself consisted identified peptides, <br />                                                                                  consequence location abundance                                                                                  lysine arginine residues affected susceptibility <br />                                                                                  protease generation optimally sized peptides mass <br />                                                                                  spectrometry. Coverage individual proteins mitochon- <br />                                                                                  drial ribosome limited, ribosomal proteins <br />                                                                                  observed peptides.  observation                                                                                  distinct ribosomal proteins implies association                                                                                  Tb927.2.6100 mitochondrial ribosomes. pro- <br />                                                                                  teins isolations MRB1 com- <br />                                                                                  plex, involved aspect processing kDNA <br />                                                                                  transcripts. Immunofluorescence studies revealed ribo- <br />                                                                                  somal proteins proteins function RNA editing lo- <br /> FIG 4 Quantitative PCR mtRNA. RNAi Tb927.2.6100 results             cated proximal kDNA (57; L. Simpson, personal communica- <br /> strong decrease examined mt transcripts course 6 days. RNAs <br /> examined included pre-edited edited ATPase subunit 6 (A6 pre        tion). Tb927.2.6100 associated kDNA essential <br /> A6 ed, respectively), edited COI, polycistronic junction regions   maintenance, simple explanation proteins primary transcripts corresponding RPS12 ND5 9S ND8.             function maintenance kDNA, transcription RNA <br />                                                                                  processing translation, generally colocalized region <br />                                                                                  kDNA resides, e., kinetoplast. <br /> idly turned disruption transcription, DAPI                          specific role Tb927.2.6100 uncertain. does ap- <br /> staining affected kDNA structure supercoiling.                         pear function directly replication specific     Immunofluorescence analysis 13 additional TAP-tagged pro-                 maxicircle minicircle DNA, molecules lost teins enriched kDNA fraction showed              similar rates. restriction submitochondrial location <br /> localize mitochondria restricted kDNA               kDNA proteins involved kDNA replication, <br /> gion. study identified proteins localize nu-         transcription, RNA processing translation located cleus. surprising proteins associated         apparent loss kDNA structure transcripts sug- <br /> kDNA, contains tens identical &#226;&#172;&#402;20-kb maxicircles hun-               gest role associated structural organization kDNA <br /> dreds heterogeneous &#226;&#172;&#402;1-kb minicircles intercatenated            integration macromolecular processes. absence <br /> single complex DNA network located mitochon-                  protein Leishmania T. cruzi implies role <br /> drion flagellar basal body. proteins include               assumed protein, histone- <br /> DNA polymerases, DNA helicases, topoisomerases, DNA ligases                  like protein hypothetical protein encoded function replication reassembly network            syntenic position Leishmania. Alternatively, absence kDNA, observed kDNA preparation iso-                   reflect functional differences trypanosomes Leishma- <br /> lated work. include p166, links kDNA              nia associated expression segregation flagellar basal body functions analogously centriole segre-          kDNA (19). <br /> gating kDNA cell division. addition, protease                 proteins associated kDNA reflect functions regulating level kDNA. Furthermore, pro-             complexity unusual mitochondrial genome, differ- <br /> teins function polygenic transcription kDNA             ences kinetoplastid species reflect evolutionary diver- <br /> subsequent RNA processing, includes transcript cleavage, ad-               sification. Differences kDNA replication segregation dition 3= oligo(U) tails gRNAs rRNA, extensive                 evolved species (9, 19, 58) reflect editing mRNAs. apparent functions proteins               genetic information differentially utilized regulated consistent localized kDNA.                                 diverse kinetoplastid species, substantial differences     previous proteomics studies identified total 3,043 T.               host target tissues, vectors, metabolic   <br /> brucei proteins assigned 979 mitochondrion                   processes. progressively diminishing stringent criteria 459 high- <br /> confidence, 218 moderate-confidence, 302 low-confidence                      ACKNOWLEDGMENTS <br /> proteins (30, 41, 42). study identified 44 additional proteins <br />                                                                                  work primarily supported National Institutes Health grant previously detected MS. Analysis TAP- <br />                                                                                  AI065935 received support grant AI014102. <br /> tagged proteins identified 2 additional mitochondrial proteins,                     thank Yuko Ogata mass spectrometry analysis. <br /> enhanced confidence mitochondrial location 12 pro- <br /> teins, demonstrated nuclear localization 3 (Table <br />                                                                                  REFERENCES <br /> 1). results update list 978 proteins assigned mitoproteome, 474 high-confidence, 209 moderate-                      1. Riou G, Delain E. 1969. Electron microscopy circular kinetoplastic <br />                                                                                      DNA Trypanosoma cruzi: occurrence catenated forms. Proc. Natl. <br /> confidence, 295 low-confidence mitochondrial proteins                        Acad. Sci. U. S.  62:210 &#226;&#8364;&#8220;217. <br /> Table S5 supplemental material). probably                     2. Kleisen CM, Weislogel PO, Fonk K, Borst P. 1976. structure represent mitochondrial proteins, dual                         kinetoplast DNA. 2. Characterization novel component high com- <br /> localization  e., isoleucyl-tRNA synthetase) (56).                                 plexity present kinetoplast DNA network Crithidia luciliae. Eur. J. <br />                                                                                      Biochem. 64:153&#226;&#8364;&#8220;160. <br />     Analysis proteins associated TAP-tagged Tb927.2.6100                  3. Shapiro TA, Englund PT. 1995. structure replication kineto- <br /> suggests interact directly indirectly mitochon-                  plast DNA. Annu. Rev. Microbiol. 49:117&#226;&#8364;&#8220;143. <br /> drial ribosomes. mass spectrometry coverage tagged                         4. Vickerman K. 1976. diversity kinetoplastid flagellates, p 1&#226;&#8364;&#8220;34. <br />  <br />  <br />  <br /> 976   ec.asm.org                                                                                                                                   Eukaryotic Cell <br />                                                                                                                                   Novel kDNA-Associated Protein <br />  <br />  <br />  <br />       Lumsden WHR, Evans DA (ed), Biology kinetoplastida. Academic          28. Gavin AC, Aloy P, Grandi P, Krause R, Boesche M, Marzioch M, Rau <br />       Press, London, United Kingdom.                                                  C, Jensen LJ, Bastuck S, Dumpelfeld B, Edelmann  Heurtier MA, <br />  5.   Stuart KD, Schnaufer  Ernst NL, Panigrahi AK. 2005. Complex                   Hoffman V, Hoefert C, Klein K, Hudak M, Michon  Schelder M, <br />       management: RNA editing trypanosomes. Trends Biochem. Sci. 30:97&#226;&#8364;&#8220;            Schirle M, Remor M, Rudi T, Hooper S, Bauer  Bouwmeester T, <br />       105.                                                                            Casari G, Drewes G, Neubauer G, Rick JM, Kuster B, Bork P, Russell <br />  6.   Aphasizhev R, Aphasizheva  2011. Mitochondrial RNA processing              RB, Superti-Furga G. 2006. Proteome survey reveals modularity       trypanosomes. Res. Microbiol. 162:655&#226;&#8364;&#8220; 663.                                     yeast cell machinery. Nature 440:631&#226;&#8364;&#8220; 636. <br />  7.   Hajduk S, Ochsenreiter T. 2010. RNA editing kinetoplastids. RNA          29. Rigaut G, Shevchenko  Rutz B, Wilm M, Mann M, Seraphin B. 1999. <br />       Biol. 7:229 &#226;&#8364;&#8220;236.                                                               generic protein purification method protein complex characteriza- <br />  8.   Kleisen CM, Borst P, Weijers PJ. 1976. structure kinetoplast             tion proteome exploration. Nat. Biotechnol. 17:1030 &#226;&#8364;&#8220;1032. <br />       DNA. 1. mini-circles Crithidia luciliae heterogeneous base    30. Panigrahi AK, Ogata Y, Zikova  Anupama  Dalley RA, Acestor N, <br />       sequence. Eur. J. Biochem. 64:141&#226;&#8364;&#8220;151.                                          Myler PJ, Stuart KD. 2009. comprehensive analysis Trypanosoma <br />  9.   Lukes J, Guilbride DL, Votypka J, Zikova  Benne R, Englund PT.                brucei mitochondrial proteome. Proteomics 9:434 &#226;&#8364;&#8220; 450. <br />       2002. Kinetoplast DNA network: evolution improbable structure.        31. Keller  Nesvizhskii AI, Kolker E, Aebersold R. 2002. Empirical statis- <br />       Eukaryot. Cell 1:495&#226;&#8364;&#8220;502.                                                       tical model estimate accuracy peptide identifications 10.   Simpson L, Maslov DA. 1999. Evolution U-insertion/deletion RNA           MS/MS database search. Anal. Chem. 74:5383&#226;&#8364;&#8220;5392. <br />       editing mitochondria kinetoplastid protozoa. Ann. N. Y. Acad. Sci.    32. Nesvizhskii AI, Keller  Kolker E, Aebersold R. 2003. statistical model <br />       870:190 &#226;&#8364;&#8220;205.                                                                   identifying proteins tandem mass spectrometry. Anal. Chem. 75: <br /> 11.   Grams J, McManus MT, Hajduk SL. 2000. Processing polycistronic               4646 &#226;&#8364;&#8220; 4658. <br />       guide RNAs associated RNA editing complexes Trypanosoma          33. Edgar RC. 2004. MUSCLE: multiple sequence alignment method       brucei. EMBO J. 19:5525&#226;&#8364;&#8220;5532.                                                   reduced time space complexity. BMC Bioinformatics 5:113. doi:10 <br /> 12.   Madina BR, Kuppan G, Vashisht AA, Liang YH, Downey KM,                          .1186/1471-2105-5-113. <br />       Wohlschlegel JA, Ji X, Sze SH, Sacchettini JC, Read LK, Cruz-Reyes J.       34. Claros MG, Vincens P. 1996. Computational method predict mito- <br />       2011. Guide RNA biogenesis involves novel RNase III family endoribo-          chondrially imported proteins targeting sequences. Eur. J. <br />       nuclease Trypanosoma brucei. RNA 17:1821&#226;&#8364;&#8220;1830.                               Biochem. 241:779 &#226;&#8364;&#8220;786. <br /> 13.   Koslowsky DJ, Yahampath G. 1997. Mitochondrial mRNA 3= cleavage/            35. Carnes J, Trotter JR, Ernst NL, Steinberg AG, Stuart K. 2005.       polyadenylation RNA editing Trypanosoma brucei independent           essential RNase III insertion editing endonuclease Trypanosoma brucei. <br />       events. Mol. Biochem. Parasitol. 90:81&#226;&#8364;&#8220;94.                                      Proc. Natl. Acad. Sci. U. S.  102:16614 &#226;&#8364;&#8220;16619. <br /> 14.   Hashimi H, Cicova  Novotna L, Wen YZ, Lukes J. 2009. Kinetoplastid        36. Acestor N, Panigrahi AK, Carnes J, Zikova  Stuart KD. 2009.       guide RNA biogenesis dependent subunits mitochondrial              MRB1 complex functions kinetoplastid RNA processing. RNA 15:277&#226;&#8364;&#8220; <br />       RNA binding complex 1 mitochondrial RNA polymerase. RNA 15:                 286. <br />       588 &#226;&#8364;&#8220;599.                                                                   37. Pfaffl MW. 2001. new mathematical model relative quantification 15.   Grams J, Morris JC, Drew  Wang  Englund PT, Hajduk SL. 2002.               real-time RT-PCR. Nucleic Acids Res. 29:e45. doi:10.1093/nar/29.9.e45. <br />       trypanosome mitochondrial RNA polymerase required transcrip-       3 <br /> </body></html>