<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> silico approach combined vivo <br /> experiments enables identification new <br /> protein overexpression compensate specific respiratory defects Saccharomyces <br /> cerevisiae <br />  <br />   Abstract <br />   Background: mitochondrial inner membrane contains large complexes essential oxidative <br />   phosphorylation. structure catalytic mechanisms respiratory complexes   progressively established, biogenesis far fully understood. complex III assembly factors <br />   identified far. probable factors needed assembly functional complex, <br />   genetic approaches used <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> able identify  developed systems <br />   biology approach identify new factors controlling complex III biogenesis. <br />   Results: collected physical protein-protein interactions (PPI) involving core subunits,   supernumerary subunits assembly factors complex III used Cytoscape 2.6.3 plugins   construct network. divided overlapping highly interconnected sub-graphs clusterONE. <br />   sub-graph contained core supernumerary subunits complex III, contained subunits <br />   complex IV proteins participating assembly complex IV. sub-graph split   algorithm sub-graphs. subtraction sub-graphs previous sub-graph allowed   identify protein unknown function Usb1p/Ylr132p interacts complex III subunits Qcr2p   Cor1p. used genetic cell biology approaches investigate function Usb1p. Preliminary results <br />   indicated Usb1p essential protein dual localization nucleus mitochondria,   expression protein compensate defects biogenesis respiratory complexes. <br />   Conclusions: systems biology approach highlighted multiple associations subunits   assembly factors complexes III IV biogenesis. addition, approach allowed   identification new factor, Usb1p, involved biogenesis respiratory complexes,   using classical genetic screens looking respiratory deficient mutants.  systems biology <br />   approach appears fruitful new way study biogenesis mitochondrial multi-subunit complexes. <br />  <br />  <br /> Background                                                     enable systematic pair wise PPI identification oxidative phosphorylation (OXPHOS)               affinity-purification followed mass spectrometry (AP/ <br /> located inner membrane mitochondria           MS)  examples [25-27]). integration responsible production <span id='am-18' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>, energy cur-         data generates network represented graph <br /> rency cell. composed large complexes,     node corresponds protein complexes respiratory chain (complexes IV)     edge interaction. large network complex ATP synthase, complex V. model organism         used directly, specific tools needed order <br /> Saccharomyces cerevisiae lacks complex  instead      extract relevant information network. single subunit <span id='am-8' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADH</span> dehydrogenases.            algorithms methods protein network analy- <br /> OXPHOS complexes II-V similar mammalian             sis (clustering assessment data quality) complexes. Complex II composed subunits          developed tested various datasets available nuclear origin complexes III, IV V com-      yeast S. cerevisiae. <br /> posed subunits dual genetic origin (10, 11 19           Broh&#195;&#169;e Van Helden [28] tested different algo- <br /> subunits respectively S. cerevisiae). core subunits   rithms clustering PPI networks. Starting encoded mitochondrial genome, translated         PPI data S. cerevisiae showed  mitochondrial ribosomes immediately inserted          220 protein complexes presently documen- membrane, subunits encoded               ted MIPS database [29]. algo- <br /> nuclear genes, translated cytoplasm imported        rithms, enable extraction independent mitochondria TOM/TIM machineries            classes  protein single module), review [1]). assembly subunits appears       enable extraction overlapping classes  coordinated  examples [2-5]) site-specific       protein modules). translation mechanisms, periphery       algorithms identifies sub-networks having specific mitochondria numerous assembly factors         topology: weakly interconnected clusters [30] strongly extrinsic complexes identified.        interconnected clusters [31]. group, <br /> addition, supra-complex associations, dif-      MCODE [32] ClusterONE [33] identify large-sized <br /> ferent mitochondrial complexes, TIM import         clusters. particularly interesting purpose, <br /> machinery reported [6-10].               biochemical analyses suggest assembly complexes, especially complex III,      assembly modules complex III mod- interactions complexes            ules contain numerous strongly interconnected proteins. <br /> assembly poorly understood.                                    report, used systems biology approach    Complex III occupies central position respira-     consists integration data different types tory chain, entry points electrons          experiments, mainly large-scale studies. Using <br /> (NADH dehydrogenases complex II) exit,           APID [34] BioGRID databases [20], collected <br /> complex IV. crystallographic structure enzyme       physical interactions concerning core subunits, sources determined [11-14],        supernumerary subunits assembly factors modular assembly scheme proposed ([15]            complex III. repeated partitioning initial <br /> references .  assembly factors       network sub-graphs identified group pro- <br /> specific complex III reported: Cyt2p, Bcs1p,     teins having interactions main complex III <br /> Cbp3p, Cbp4p, Mzm1p  Aim8p) Bca1p [16-19],              sub-graph. Cell biology genetic methods 25 identified complex        used investigate function proteins <br /> IV  review [5]), enzyme similar complexity,      identified silico approach. <br /> suggesting factors involved complex <br /> III biogenesis remain discovered.                        Results Discussion <br />    order identify proteins strongly connected       Network modelling <br /> complex III look proteins participating      order new proteins involved complex III <br /> assembly process, systems biology approach          biogenesis, used Cytoscape facilities [35] com- <br /> used. high-throughput datasets coming                bine physical interactions concerning complex III <br /> genetic proteomic studies concerning yeast S.          (core proteins, supernumerary proteins assembly <br /> cerevisiae accessible public databases [20].       factors, Table 1) APID [34] BioGRID <br /> interactome represents physical interactions           databases [20]. proteins cell. main techniques           computational approach leading complete used produce high quality complementary            PPI network  section methods) gave following <br /> data: yeast hybrid studies  examples [21-24]),     results: <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                                     Page 3 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> Table 1 List proteins complex III (input list) <br /> Complex III                                                 Ordered Locus                            UniProt <span id='am-29' about='oboInOwl:id' typeof='owl:Thing'>ID</span>                        SGD Catalytic core                                                     YEL024W                                  P08067                              RIP1 <br />                                                                    YOR065W                                  P07143                              CYT1 <br />                                                                      Q0105                                  P00163                              COB <br /> Supernumerary subunits                                              YBL045C                                 P07256                             COR1 <br />                                                                    YPR191W                                  P07257                             QCR2 <br />                                                                     YFR033C                                 P00127                             QCR6 <br />                                                                    YDR529C                                  P00128                             QCR7 <br />                                                                    YJL166W                                  P08525                             QCR8 <br />                                                                    YGR183C                                  P22289                             QCR9 <br />                                                                   YHR001W                                 P37299                             QCR10 <br /> Assembly factors                                                    YKL087C                                 Q00873                              CYT2 <br />                                                                    YDR375C                                  P32839                              BCS1 <br />                                                                    YPL215W                                  P21560                              CBP3 <br />                                                                    YGR174C                                  P37267                              CBP4 <br />                                                                    YLR077W                                  Q08023                             BCA1 <br />                                                                    YDR493W                                  Q03429                             MZM1 <br /> Complex III defined groups proteins: catalytic core supernumerary subunits structure, proteins involved assembly complex. UniProt ID used retrieve data APID database BioGRID  SGD) SGD  <br />  <br />  <br />   1) APID database, obtained list 521                             way network sub-divided 26 sub-graphs <br /> unique PPI involving 125 proteins (list 1 Figure 1).                            (clusters 1 26), p-values 0.05 <br />   2) BioGRID database, 123 unique PPI involving                           good quality scores (Table 2). Note clustering <br /> 91 different proteins collected (list 2 Figure 1).                         method robust, sub-graphs interactions new allowed                               starting analysis core super- <br /> enrich obtained APID.                                                   numerary proteins. <br />   3) new interactions, concern ele-                                  Cluster 1 composed 175 PPI involving 37 proteins; <br /> ven new proteins numerous direct                                cluster 2 composed 157 PPI involving 34 proteins <br /> PPI. example, 2645 physical interactions                             cluster 4 144 PPI involving 33 proteins (Additional Nab2p 1444 Ubi4p, proteins                                  file 1 Figure S1). built non-mitochondrial <br /> independent complex III. discuss later                              proteins having interactions independent com- <br /> consequences PPI network.                                          plex III. example Nab2p nuclear polyAdenylated <br />   4) final list, obtained merging PPI lists                       RNA-binding protein, Ubi4p ubiquitin, Ssa2p eliminating redundancy, composed 719 unique                             stress-seventy subfamily protein, Rpn10p non- <br /> PPI involving 136 proteins. 80 proteins common                              ATPase subunit regulatory particle 26S pro- databases, 45 APID 11 BioGRID.                              teasome, Tef1p translational elongation factor Eliminating redundancy facilitates identification                           Ssb1p ribosome-associated molecular chaperone. <br /> links proteins studied proteins                              Clusters 1, 2 4 contain PPI specific <br /> known involved complexes.                                                 complex III. <br />   5) resulting network large yield                                 Cluster 5 composed 101 PPI involving 23 proteins <br /> interesting information. necessary divide                        cluster 6 composed 63 PPI involving 16 proteins <br /> highly connected sub-graphs represent functional                              (Additional file 2 Figure S2). majority pro- <br /> modules, protein sub-complexes                                teins translocases involved import proteins <br /> sider assembly intermediates.                                                   mitochondria, proteins <br />                                                                                    involved complex III: Bcs1p, Cobp Cbp2p, pro- <br /> Network clustering                                                                 tein complex IV: Cox7p, Atp1p Atp2p, identify new proteins associated assembly                              proteins F1 complex V. clusters intermediates complex III, looked sub-groups                          clearly specific complex III include strongly interconnected proteins. looked                         proteins input list. <br /> lapping sub-graphs account fact                             Cluster 3 (Figure 2) composed 193 PPI involving <br /> protein associated sub-complexes.                              36 proteins, including complex III subunits.  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                     Page 4 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />           List proteins involved complex III S. cerevisiae <br />  <br />                             APID database                                 BioGRID database <br />  <br />               list 1       (521 PPI, 125 Proteins)                            list 2 (123 PPI,        91 Proteins) <br />  <br />  <br />  <br />                                    ID unification (SGD Standard  <br />  <br />  Comparison list 1 2                                        11 Proteins list 2 list 1 (N) <br />  <br />  <br />         BioGRID: proteins functional enrivonment N                                               PPI list 3 <br />                                                                                          (359 PPI, 104 Proteins) <br />  <br />        Merging lists 1, 2 3 elimination redundancy <br />                            (719 PPI, 136 Proteins) <br />   (80 proteins common APID BioGRID, 45 APID, 11 BioGRID) <br />  <br />                  Network modeling clustering Cytoscape <br />                                              (ClusterONE MCODE) <br />  <br />  <br />                      Clusters strongly interconnected proteins <br />  <br />                            Identification new proteins involved <br />                                   complex III biogenesis <br />  Figure 1 Computational workflow. overview network modelling clustering. <br />  <br />  split overlapping sub-graphs,              (CoA1p, Mss51p, Shy1p, Rrg10p/CoA3p). fact sub-graph (cluster 3.1) extracted, 185 PPI invol-            network contains proteins complexes III ving 30 proteins. quality equal 0.97. pro-          IV, input list exclusively composed teins sub-graph members complex III (core              proteins complex III probably related exis- supernumerary subunits) complex IV (Cox1p,                    tence III-IV super-complexes, clearly <br /> Cox2p, Cox3p, Cox4p, Cox5Ap, Cox6p, Cox14p,                           identified blue native gels mitochondrial extracts <br /> Cox15p) participate assembly complex IV                  [6,7]. results associations link  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                         Page 5 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> Table 2 Characteristics complex III PPI sub-                     result obtained silico led study function networks obtained ClusterONE                                       Usb1p using genetic cell biology methods. <br />  cluster    nodes      edges      density     quality     p-value <br />     1         37        175         0.25          0.476   2.25 10-7     Usb1p located nucleus     2         34        157         0.25          0.444   1.01 10  -5   mitochondria <br />     3         36        193         0.29          0.556   1.35 10-4     silico analysis shown Usb1p belongs                                                                         cluster 3.1 contains complex III subunits, <br />     4         33        144         0.25          0.414   1.00 10-3 <br />                                                                         11 subunits complex IV,     5         23        101         0.36          0.419   2.00 10-3 <br />                                                                         assembly factors complex IV. order determine     6         16         63        0.450          0.432    0.011 <br />                                                                         Usb1p involved biogenesis mito- <br />    3.1        30        185        0.409          0.967    0.000        chondrial respiratory complexes III IV,                                                                         analyzed sub-cellular localization. <br />                                                                            Large-scale analysis localization GFP-tagged <br /> complexes. Furthermore, assembly factors                     S. cerevisiae proteins suggested Usb1p exhibits complex IV identified interact subunits               multiple location nucleus, cytoplasm complex III: Cor1p, Qcr2p, Cyt1p, suggesting                    mitochondria [36]. verify localization, Usb1p factors play role assembly                    tagged C-terminus GFP. genetic  <br /> complexes III IV.                                                   struction integrated USB1 chromosomal <br />    longer possible split cluster 3.1,               locus strain carrying CFP-tagged Nic96 protein large, overlapping sub-graphs                revealed pores nuclear membrane [37]. <br /> divided independent clusters: cluster 3.1.1 com-                USB1-GFP gene expressed posed 81 PPI involving 13 proteins belonging com-                 control promoter sequence order avoid <br /> plex IV, proteins complex III (Cor1p, Qcr2p                mislocalization overexpression  <br /> Cyt1p) Aac1p, minor isoform <span id='am-2' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ADP</span>/ATP                        struction carried plasmid. Cells expressing mitochondrial transporter (Figure 2). Cluster 3.1.2                  Nic96-CFP Usb1-GFP fusion proteins composed 27 PPI involving 8 proteins;                   examined fluorescence microscopy using appro- members complex III (Cobp, Qcr6p, Qcr7p,                         priate filters. Mitotracker used stain mito- <br /> Qcr8p, Qcr9p, Qcr10p, Rip1p) protein Pet9p/                     chondrial network. shown Figure 3 andAdditional <br /> Aac2, major isoform ADP/ATP transporter                      file 4 (Figure S3), Usb1-GFP located (Figure 2). cluster 3.1 separated              nucleus (3A) membrane stained parts corresponding broadly complexes III  cluster               Nic96-CFP (3C) fluorescent tubules 3.1.2) IV  cluster 3.1.1).  proteins              stained Mitotracker probe (3B). linked proteins complex                Usb1-GFP protein dual localization nucleus proteins complex. exam-                  mitochondria. <br /> ple, Cor1p Qcr2p sub-graph complex <br /> IV sub-graph complex III                       Overexpression USB1 compensates specific respiratory interactions proteins complex IV                 defects Aac1p subunits complex III.                  Systematic deletion analysis suggested gene strengthens hypothesis existence com-                essential [26].  able obtain mon factors involved assembly process                    viable haploid cell carrying deleted allele USB1 <br /> complexes.                                                              gene heterozygous diploid strain  Methods).    subtract sub-clusters (3.1.1 3.1.2)                investigate role, mitochondria, cluster 3.1, proteins left,                 analyzed effect overexpression USB1 gene <br /> (Usb1p, Smk1p, Yju2p, Pcl9p, Rbs1p) connected                    set respiratory deficient mutants. cloned <br /> Cor1p Qcr2p. form sub-graph,                    USB1 multicopy plasmid transformed interconnected detected MCODE.                          wild type strain collection respiratory deficient <br />    proteins Usb1p                     mutants carry mutations various factors essential <br /> function unknown. association pro-                biogenesis respiratory complexes. Table 3 <br /> teins (Cor1p, Qcr2p Usb1p) corresponds complex                 Figure 4A overexpression USB1 com- <br /> isolated tandem affinity purification Gavin et al. [25].          pensates respiratory deficiency oxa1-E65G- Usb1p connected complex III, par-                 F229S mutant deletion MFT2, bcs1 <br /> ticularly Cor1p [25,26] Qcr2p sub-units [25,27]              mutant, deletions OXA1, BCS1 RMD9.  Additional file 3 Table S1: list pro-          level compensation observed oxa1 <br /> teins interacting Usb1p present network).              mtf2 mutants. previously shown  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                      Page 6 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />  <br />  <br />  Figure 2 Complex III sub-graph 3 sub-graphs. Images taken Cytoscape. Cluster 3 comes partition PPI <br />  network complex III ClusterONE. Cluster 3.1 comes partition cluster 3 ClusterONE. Cluster 3.1.1 3.2.2 come  partition cluster 3.1 MCODE. Complex III proteins represented grey circles; complex IV proteins grey rounded squares; complex <br />  IV assembly factors grey octagons; Usb1p white lozenge proteins white circles. <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                                    Page 7 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />                        Usb1-GFP                                     Mitochondria                                       Nucleus <br />                                                                     (mitotracker)                                    (Nic96-CFP) <br />  <br />  <br />  <br />  <br />   Figure 3 Dual localization Usb1p. Cells expressing Usb1-GFP Nic96-CFP fusion proteins grown complete glucose medium <br />   times excess <span id='am-13' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span>adenine</span> 28&#194;&#176;C, washed incubated Mitotracker red CMX ROS (MT, Molecular Probes) 10 min.   additional washes, cells observed fluorescence microscopy appropriate filters visualize GFP, CFP Mitotracker. <br />  <br />  <br /> respiratory deficiency oxa1-E65G-F229S mutant                            choose gene/protein best candidate correlated strong decrease cytochrome b com-                            analysis complicated like OXPHOS, <br /> plex III cytochromes aa3 complex IV [38]. Cyto-                             composed complexes.  inte- <br /> chrome spectra cells grown respiratory medium                                grative method help orient subsequent research overexpression USB1 oxa1 mutant                            highlight specific interactions restored assembly cytochromes b aa3 (Figure                             complexes. <br /> 4B). overexpression USB1 specifically com-                               approach revealed multiple associations pensates respiratory deficiencies.                                        subunits complexes III IV interactions                                                                                    ADP/ATP transporters. Cluster 3.1 identifies connec- <br /> Conclusions                                                                        tions complex III subunits, complex IV subunits computational approaches combined                                 assembly factors, suggesting interaction experimental methods  examples [39,40])                              assembly process complexes  [41] <br /> recently established comprehensive network                               references review complex IV assem- <br /> mitochondrial proteome, analysis systems                          bly factors). example, results biology study strictly devoted OXPHOS complexes                             assembly factor Shy1p connected proteins yeast. huge amounts interaction data                            cluster 3.1.1, e. complex III subunits, different sources (genetic, proteomics) easy                        Cor1p, Qcr2p, Cyt1p subunits assembly <br />  <br />  <br /> Table 3 Overexpression Ylr132 compensates specific respiratory deficiencies <br />    Genotype            Transformation vector                  Transformation wt gene                 Transformation YLR132 <br />       oxa1                                   -                                                 ++                                            + <br />       &#206;&#8221;oxa1                                  -                                                 ++                                            - <br />       bcs1                                   -                                                 ++                                            - <br />       &#206;&#8221;bcs1                                  -                                                 ++                                            - <br />       &#206;&#8221;mtf2                                  -                                                 ++                                            + <br />      &#206;&#8221;rmd9                                   -                                                 ++                                            - <br /> Respiratory-deficient mutants affecting various steps biogenesis respiratory complexes transformed different high-copy plasmids: (1) control vector, (2) plasmid carrying corresponding wild type gene (OXA1 lines 1 2; BCS1 lines 2 3; MTF2 line 5 RMD9 line 6), (3) plasmid carrying wild type USB1 gene. transformants selected minimal glucose medium lacking <span id='am-30' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> select plasmids growth tested non fermentable medium containing 2% glycerol 2% ethanol 28 36&#194;&#176;C. +: respiratory growth; - growth. Figure 4A. oxa1 mutant carries double amino-acid substitution E65G-F229S Oxa1 protein [38]. bcs1 mutant carries single amino-acid <br /> substitution F342C Bcs1p [53]. mutants carry gene deletions indicated [38,44,53]. <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                                     Page 8 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br />                      Transformation                   B             Cytochrome spectra <br />                                 oxa1 mutant                                             b <br />                                                                                                                                                     c c1 <br />                           OXA1 vector USB1 <br />  <br />  <br />                                                                                                           aa3 <br />                           Fermentable medium                                                                  OXA1 <br />  <br />                                                                0.1                                                USB1 <br />  <br />                                                                                   550        600   Onm <br />                            Respiratory medium                         Growth respiratory medium <br />  Figure 4 Overexpression USB1 compensates respiratory deficiency. Panel  thermosensitive mutant oxa1-E65GF229S [38]  transformed high-copy plasmids carrying wild type OXA1 gene USB1 gene, control vector (vector).  transformants spotted fermentable (glucose) respiratory (glycerol/ethanol) medium. plates incubated days 36&#194;&#176; <br />  C. Panel B: Cytochrome absorption spectra oxa1 cells transformed high copy plasmids carrying OXA1 YLR132, grown  respiratory medium 2 5 days 36&#194;&#176;C. Cytochrome c1 b respiratory complex III cytochromes aa3  complex IV. <br />  <br />  <br /> factors complex IV. minor ADP/ATP transporter,                     translation [38-43]. overexpression USB1 Aac1p cluster. Cluster 3.1.2                  compensate absence Rmd9p restore <br /> composed core (Rip1p, Cobp) supernumerary pro-                     respiratory growth deletion MTF2 (NAM1) teins  Cor1p, Qcr2p), Pet9p/Aac2p,                  affects processing/stability mitochondrial <br /> major ADP/ATP transporter.                                                mRNAs, encoding subunits complex V [44]. Mtf2p <br />    subunits complex II (Sdh3p                    shown interact mitochondrial RNA <br /> Sdh4p) present PPI network did retrieve                 polymerase [45]. Finally, Usb1p detected systema- clusters, suggesting interactions                    tic screen RNA-binding proteins yeast [46]. complex III complex II weak. Cyc1p                     dual location Usb1p, nucleus mito- present PPI network strong                chondria, tempting suggest Usb1p  <br /> nection cytochrome c.  cytochrome c                     trol processing stability nuclear mobile carrier, shuttling electrons complexes III                 mitochondrial RNAs. mitochondrial respiratory IV interactions probably dynamic                   function required yeast viability, essential detected analysis detects strongly                    function Usb1p eventual role sta- <br /> interconnected proteins. connections                   bility/processing nuclear RNA(s). complex III NADH dehydrogenases sug-                          sub-graph analysis developed gesting tight contact                       highlighted strong connections subunits    Finally, approach identified protein Usb1p              assembly factors complexes III IV led <br /> probably escaped previous genetic screens                   identification new factor acting biogen- <br /> essential cell survival. protein dual localiza-            esis respiratory complexes yeast. <br /> tion, nucleus mitochondria. Overexpression USB1 compensates respiratory deficiency                     Methods <br /> oxa1 mutant. Oxa1p mediates translational inser-                   Databases <br /> tion mitochondrially-encoded subunits respiratory               construct network collected data different <br /> complexes inner membrane  review                       sources. Information protein-protein interactions <br /> [42]). previously shown increase                   (PPI) APID (Agile Protein Interaction DataAna- <br /> steady state level mitochondrial mRNAs encoding                        lyzer) integrates unifies PPI different pub- <br /> respiratory complex subunits compensate                  lic repositories [34]. database includes data high <br /> respiratory deficiency oxa1 mutant overex-                throughput proteomic techniques small-scale <br /> pression RMD9. Rmd9p controls processing/stability                 experimental data; data quality critical indicates mitochondrial mRNAs encoding respiratory sub-                  reliability given interaction. S. cerevisiae, 5902 <br /> units control initiation mitochondrial              proteins  total 6510) 105728 separate <br />  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                     Page 9 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> interactions listed release (February 2009).      3) consistent APID results APID appropriate purpose qualifies        look functional environment new proteins functional environment given protein. Looking       present BioGRID (list 2) absent APID neighbours protein expands resulting net-        (list 1). compared output lists, list 1 APID work gives reliable map allowing          list 2 BioGRID, proteins absent proteins involved biological     APID list. added physical interactions process. Unfortunately, APID database updated.       &#226;&#8364;&#339;new&#226;&#8364;&#65533; proteins proteins list 1 absent collect data recent experiments used        input list. way obtained list 3. <br /> BioGRID database  Biological General Repository             4) merged lists 1, 2 3, PPI Interaction Datasets), release 3.1.72 (December 25 th ,         identified different experiments 2010), updated monthly archives genetic        taken account list unique PPI. <br /> protein interaction data model organisms                  5) final list imported Cytoscape 2.6.3 [35]. <br /> humans including S. cerevisiae. budding yeast       resulting network weighted, PPI complete coverage literature maintained Bio-      kept  large yield interesting <br /> GRID provides interactions Saccharomyces Gen-            information necessary divide connected sub- <br /> ome Database (SGD). data retrieved,               networks represent functional modules pro- <br /> 163188 non-redundant physical genetic interactions          tein sub-complexes. listed base, corresponding 6049 unique pro-         network weighted, e. score assigned teins [20]. eliminated genetic interactions      edges. protein studied, lots genes complex III proteins submitted        experiments partners interaction high throughput experiments like Synthetic           identified times. number occur- <br /> Genetic Array analysis [47], inclusion          rences interaction criterion reliability, genetic interactions lead lack balance     advantageous attribute higher weight edges <br /> network. example interactions COR1         frequent.  weighting Synthetic Genetic Array analysis           introduce important bias, favour included, interactions       studied proteins. <br /> QCR2. <br />    Complementary information SGD [48].             Network clustering <br />                                                                 new proteins associated assembly intermedi- <br /> Network modelling Cytoscape                                  ates complex III requires identification sub- <br /> Figure 1 shows computational workflow              complexes assembly modules. PPI network, sub- <br /> structural assembly proteins complex III          graphs highly interconnected proteins consid- <br /> identification new proteins implicated complex III        ered protein complexes functional modules. biogenesis. input list proteins known        protein interact sub-complexes involved complex III given Table 1 (core          case assembly factors chaperons) <br /> subunits, supernumerary subunits assembly factors).         possibility overlapping. step approach necessary build com-               partition resulting PPI network overlap- <br /> plete PPI network:                                              ping sub-graphs highly interconnected nodes  <br />    1) using APID2NET plugin Cytoscape [49]            formed plugin ClusterONE (Clustering UniProtID proteins listed Table 1             Overlapping Neighbourhood Expansion) [33] Cytos- <br /> (input list), collected PPI list: list 1. corre-      cape. algorithm works &#226;&#8364;&#339;growing&#226;&#8364;&#65533; dense regions <br /> sponds direct PPI interactions            small seeds guided quality function. qual- proteins having direct interaction pro-          ity group evaluated number internal edges <br /> tein input list.  collected physical     divided number edges involving nodes interactions involving proteins input list          group. Starting single seed  case: BioGRID database. obtained output list,          node) algorithm extends group step step list 2, composed PPI involving proteins named             new edges increase quality group. edge standard SGD                                         removed removal increases quality    2) use interactions APID Bio-         group. process stops possible GRID, need fuse lists merging         increase quality groups adding removing necessary unify names. APID BioGRID         edge. Finally, sub-graphs smaller 3 having use identifiers (UniProt names SGD             density (number edges cluster divided names respectively). used SGD names,         number theoretically possible edges), 0.25 standard names used yeast community.                 p-value 0.05, discarded  Glatigny et al. BMC Systems Biology 2011, 5:173                                                                               Page 10 12 <br /> http://www.biomedcentral.com/1752-0509/5/173 <br />  <br />  <br />  <br />  <br /> overlapping significantly (overlap threshold equal 0.8)   homozygous USB1-GFP heterozygous nic96- <br /> merged form larger sub-graph; chose density      CFP/NIC96. Fluorescence microscopy performed <br /> 0.25 overlapping sub-graphs having good        using Zeiss Axioplan 2 microscope linked Cool <br /> quality.  suppose assembly inter-      Snap camera (Princeton Instruments). <br /> mediates having proteins common required            Cytochrome absorption spectra <br /> make complex III. p-value cluster computed      Cytochrome absorption spectra cells posteriori sided Mann-Whitney U test       recorded liquid nitrogen temperature reduction formed number intra edges external bound-       dithionite using Cary 400 spectrophotometer (Varian, <br /> ary edges. intra edge cluster node      San Fernando, CA). Absorption maxima bands node cluster. external boundary edge         cytochrome c, c1, b aa3 expected 546, 552, cluster node node outside cluster.        558 602 nm, respectively. <br />   large cluster split  <br /> lapping sub-graphs highly interconnected nodes,     Additional material split independent sub-graphs highly intercon- <br /> nected nodes. partition independent sub-graphs           Additional file 1: Figure S1 - Sub-networks 1, 2 4 obtained plugin MCODE (Molecular Complex                 partition complex III PPI network ClusterONE. <br />                                                                   Image taken Cytoscape. <br /> Detection) [32] Cytoscape. algorithm detects den- <br />                                                                   Additional file 2: Figure S2 - Sub-networks 5, 6 obtained sely connected regions. assigns weight         partition complex III PPI network ClusterONE. <br /> node, corresponding local neighbourhood density.           Image taken Cytoscape.  recursively moves outward, including clus-        Additional file 3: Table S1 - List Usb1p interactors present ter nodes weight given threshold.            network [54]. used MCODE plugin default parameters.                 Additional file 4: Figure S3 - Localization Usb1p. Cells expressing <br />                                                                   Usb1-GFP grown complete medium stained                                                                   Mitotracker red CMX ROS. figure shows Usb1-GFP Yeast genetics biochemical analysis                           Mitotracker fluorescent images colored green red respectively, Strains, media, genetic methods, plasmids                         merged image showing localization Mitotracker                                                                   Usb1-GFP signal. <br /> CW252 (mat alpha, ade2, ura3, his3, trp1, leu2) wild type yeast strain. Fermentable media contain 2% glu- <br /> cose respiratory medium contains 2% glycerol 2% ethanol. Yeast cells transformed step    <br /> procedure [50]. high copy vector pFL44 carries        <br /> replication origin yeast 2 micron plasmid      <br /> URA3 marker. YepNB6 contains wild type OXA1 <br />                                                            <br /> gene YepML3 wild type USB1 gene;         <br />                                                              <br /> cloned pFL44 vector.                             <br /> USB1 disruption epitope tagging Usb1p                   USB1 gene inactivated diploid strain        <br /> W303, using PCR method described Wach et al.          <br /> (1994) [51]: entire ORF replaced KanR    <br /> marker gene confers resistance drug G418.         heterozygous USB1/&#206;&#8221;usb1::KanR diploids              <br /> sporulated micro-dissected. G418 sensitive <br />                                                             <br /> spores germinated confirming USB1 gene            <br /> essential. C-terminal epitope tagged proteins  <br /> structed described: Usb1-GFP using plasmid              <br /> pFA6-5GA super bright uses G418 resistance mar-      <br /> ker [52] Nic96-CFP using pBS4, carries            <br /> hygromycin B resistance marker  Yeast Resource              <br /> Center, University Washington). strain expressing       <br /> Usb1-GFP shows growth indistinguishable wild          <br /> type fermentable respiratory medium,            <br /> ing tagged protein fully functional. Nic96-     <br /> CFP strain shows weak growth defect;            microscopic observations strain        <br /> </body></html>