<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>SCFMet30-mediated control transcriptional <br /> activator Met4 required G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />                                pounds (reviewed Thomas Surdin-Kerjan, 1997). <br />                                addition biosynthesis sulfur amino <br />                                   acids methionine cysteine, reduced sulfur atoms                                                                          needed synthesize S-adenosylmethionine (AdoMet),            molecule central general metabolism. AdoMet       <span id='am-6' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> number reactions  <br />            biological compound used: AdoMet main donor <br />    methyl groups methylation nucleic acids, proteins <br />                             lipids, serves precursor biosynthesis                                                   polyamines substrate numerous <br />                                          reactions <span id='am-1' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="vitamin" datatype="xsd:string"></span>vitamin</span> biosynthesis <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> modi- <br />               fication. <br />                                                                             Sulfur flux sulfate assimilation pathway <br /> Progression cell cycle requires coord-                   metabolic arms specific methionine ination basal metabolism cell cycle                      cysteine syntheses controlled intracellular content <br /> growth machinery. Repression sulfur gene net-                     AdoMet. key component regulation MET <br /> work mediated ubiquitin ligase SCFMet30,                       gene transcriptional activator Met4p associated targets transcription factor Met4p                         cofactors Cbf1p, Met28p, Met31p Met32p (reviewed <br /> degradation. Met30p essential protein yeast.                    Thomas Surdin-Kerjan, 1997). Repression met4&#226;&#710;&#8224;met30&#226;&#710;&#8224; double mutant                        Met4p-dependent transcription response increased <br /> viable, suggesting essential function Met30p                 intracellular AdoMet mediated SCFMet30 complex, control Met4p. support hypothesis,                    E3 ubiquitin ligase (Patton et al., 1998; Rouillon et al., <br /> Met4p mutant unable activate transcription does                       2000). Met4p degradation transcriptional activity cause inviability met30&#226;&#710;&#8224; strain.                      governed autoregulatory loop. intracellular <br /> expression unregulated Met4p mutant lethal                      AdoMet low, Met4p-containing transcription activation wild-type cells. non-permissive conditions,                     complexes drive expression MET gene network, <br /> conditional met30&#226;&#710;&#8224; strains arrest large, unbudded                     including MET30 gene. expressed, Met30p <br /> cells 1N DNA content, shortly                       targets Met4p degradation SCFMet30 pheromone arrest point. met30&#226;&#710;&#8224; conditional mutants                       limits expression MET genes response high <br /> fail accumulate CLN1 CLN2, CLN3                           intracellular AdoMet (Rouillon et al., 2000). <br /> mRNAs, CLN1 CLN2 expressed                                SCF ubiquitin ligase complexes target specific substrates strong heterologous promoters.                          degradation, determined F-box component <br /> genes regulation Met4p delay                        (reviewed Craig Tyers, 1999). addition Met4p, <br /> progression G1 S phase specific                        SCFMet30 appears target Swe1p, Cdc28p inhibitory <br /> regulation critical G1 phase mRNAs.                                   kinase necessary morphological checkpoint (Kaiser <br /> Keywords: G1 cyclin/G1&#226;&#8364;&#8220;S transition/methionine/SCF                       et al., 1998). SCFMet30 SCF complexes <br /> ubiquitin ligase                                                         identified yeast. SCFCdc4 targets cell cycle                                                                          transcriptional regulators Sic1p, Far1p, Gcn4p, Ctf13p                                                                          Cdc6p degradation, SCFGrr1 responsible                                                                          degradation G1 cyclins, Cln1p Cln2p, Introduction                                                             Cdc42p effector Gic1p, regulation glucose <br /> Availability nutrients environment essential                repression pathway. Proteins similar Met30p exist cell growth cell cycle progression. budding               fungi, Drosophila mammals, SconB, <br /> yeast Saccharomyces cerevisiae, nutrient deprivation                 Slimb &#206;&#178;-TrCP proteins, respectively (Kaiser et al., <br /> elicit different responses cell cycle arrest G1               1998; Margottin et al., 1998). Recently, &#206;&#178;-TrCP/Slimb <br /> phase, sporulation pseudohyphal growth. particular,                proteins implicated ubiquitin-mediated <br /> sufficient sources carbon, phosphate, nitrogen sulfur             degradation protein targets NF-&#206;&#186;B, Wnt/Wingless required yeast cells pass cell cycle              Hedgehog pathways (reviewed Maniatis, 1999). <br /> commitment point called Start late G1 phase (reviewed                  &#206;&#178;-TrCP shown trigger degradation Pardee, 1989; Hartwell, 1994; Sherr, 1994; Polymenis                  lymphocyte receptor CD4 HIV protein Vpu Schmidt, 1999).                                                      (Margottin et al., 1998). <br />    Budding yeast satisfy elemental sulfur require-                Regulation cell division cycle requires activa- <br /> ment uptake subsequent metabolism                   tion essential cyclin-dependent kinase (CDK) <br /> large number inorganic organic sulfur com-                  Cdc28p G1 cyclins Cln1p, 2p 3p (reviewed  <br /> &#194;&#169; European Molecular Biology Organization                                                                                       1613 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 1.   Simplified view sulfur <span id='am-25' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-26' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> biosynthesis yeast. (B) met4&#226;&#710;&#8224; disruption mutation specifically suppresses lethality <br /> induced loss function mutation met30&#226;&#710;&#8224;. Materials methods details crosses. <br />  <br />  <br /> Nasmyth, 1996). Cyclin-CDK activity rate-limiting                       1A; Materials methods). included cross <br /> passage Start, control cyclin-CDK activity                      met4&#226;&#710;&#8224; mutant strain analysis Met30p reported multiple levels including inhibition                   shown trigger Met4p degradation response cyclin inhibitors, transcription, translation degradation                  increase intracellular AdoMet (Rouillon et al., 2000). <br /> (Hartwell, 1994; Sherr, 1994; Nasmyth, 1996; Polymenis                         Single deletions MET3, MET6 STR4 genes Schmidt, 1999). example, non-degradable forms                          double deletion SAM1 SAM2 genes did G1 cyclins, Cln3-1, cause hyperactivation                       alter met30&#226;&#710;&#8224; lethality phenotype, indicating Start decreased critical cell size (Nash                        met30&#226;&#710;&#8224; lethality accumulation et al., 1988). Repression CLN3 mRNA translation                          sulfur-containing metabolite. contrast, deletion required cell cycle arrest starvation high                          MET4 bypassed lethality met30&#226;&#710;&#8224; strain (Figure <br /> <span id='am-32' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-33' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>iron</span> conditions (Polymenis Schmidt, 1997; Philpott                         1B), suggesting essential function Met30p <br /> et al., 1998).                                                                 regulation Met4p. <br />    MET network essential absence                         conditionally viable met30&#226;&#710;&#8224; mutant sulfur amino acids, MET30 essential gene                        constructing strains express MET4 nutrient conditions (Thomas et al., 1995).                         GAL1 promoter absence MET30. shown essential function Met30p regulation                   Figure 2A, expression GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells Met4p transcriptional activity. Surprisingly, Met4p                         induces lethality minimal complete galactose  <br /> regulation critical progression Start.                      taining medium.  repressive amounts extracellular <br /> results argue misregulation genes                        methionine AdoMet, shown trigger <br /> control Met4p inhibits passage Start pre-                        degradation Met4p, did prevent detrimental <br /> venting accumulation important G1 phase transcripts,                    effect MET4 expression met30&#226;&#710;&#8224; mutants. Similar <br /> including genes encoding G1 cyclins,                          results obtained cells grown sulfur B <br /> Cln1p Cln2p.                                                               medium presence homocysteine methionine <br />                                                                                used sulfur source (data shown). Altering Results                                                                        levels GAL1&#226;&#8364;&#8220;MET4 expression spotting mutants <br />                                                                                galactose concentrations ranging 0.1 2.5% Inactivation MET4 suppresses met30&#226;&#710;&#8224; lethality                               equally effective inducing lethality (data shown). <br /> MET30 essential gene encodes negative                                 functional analyses Met30p carried <br /> regulator sulfur amino acid biosynthesis pathway                        MET30-1 MET30-2 strains, point mutants (Thomas et al., 1995). unregulated synthesis                          MET30 unable fully repress MET gene expres- <br /> particular sulfur-containing metabolite                     sion response high levels extracellular methionine <br /> responsible lethality met30&#226;&#710;&#8224; strain. test                 (Thomas et al., 1995). viable conditional met30&#226;&#710;&#8224; <br /> hypothesis, crossed met30::LEU2 strain harboring                        mutant provided opportunity verify LexA-MET30, HIS3 plasmid battery mutants                        Met30p function absolutely required Met4p regula- specifically block metabolic arm sulfur                        tion. analyzed AdoMet-mediated repression network (met3&#226;&#710;&#8224;, met6&#226;&#710;&#8224;, str4&#226;&#710;&#8224; sam1&#226;&#710;&#8224;sam2&#226;&#710;&#8224;; Figure                            structural genes sulfate assimilation pathway <br />  <br /> 1614 <br />                                                                                                    Role Met4 G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />                                                                         MET genes multiprotein complexes. Depending <br />                                                                         gene, different combinations Cbf1p, Met28p, <br />                                                                         Met31p Met32p factors assembled tether <br />                                                                         Met4p DNA (Kuras et al., 1997; Blaiseau Thomas, <br />                                                                         1998). determine  addition Met4p, com- <br />                                                                         ponents complexes required essential <br />                                                                         function Met30p, crossed met30::LEU2 strain <br />                                                                         harboring LexA-MET30, HIS3 plasmid single <br />                                                                         cbf1&#226;&#710;&#8224; met28&#226;&#710;&#8224; mutants met31&#226;&#710;&#8224;met32&#226;&#710;&#8224; <br />                                                                         double mutant. results genetic tests showed <br />                                                                         met32&#226;&#710;&#8224; met30&#226;&#710;&#8224; spores recovered                                                                         deletion CBF1, MET28 MET31 effect                                                                         lethality met30&#226;&#710;&#8224; mutation (Table . essential <br />                                                                         function Met30p required antagon- <br />                                                                         ize specific Met4p&#226;&#8364;&#8220;Met32p transcription complex. <br />  <br />                                                                         Met4p regulation essential cell viability <br />                                                                         recently reported SCFMet30-dependent <br />                                                                         degradation Met4p response high extracellular <br />                                                                         methionine (Rouillon et al., 2000). genetic data, <br />                                                                         demonstrating essential function Met30p                                                                         bypassed deletion MET4 MET32, suggested <br />                                                                         lethality met30&#226;&#710;&#8224; strain Fig. 2. Expression Met4p GAL1 promoter lethal         unregulated activity Met4p, Met32p lacks transcrip- <br /> met30&#226;&#710;&#8224; cells.   Serial dilutions CC932-6D (met4::GAL1&#226;&#8364;&#8220; <br /> MET4 MET30) CC932-8B (met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2)                  tional activation function (Blaiseau et al., 1997). Detailed <br /> strains plated media containing 2% glucose (Glu), raffinose     analysis functional organization Met4p (Raf) galactose (Gal) carbon source absence presence    identified activation domain (AD), regulatory 1 mM <span id='am-51' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span>L-methionine</span> (Met) 0.2 mM AdoMet. (B) CC932-6D               regions required AdoMet response, called (met4::GAL1&#226;&#8364;&#8220;MET4 MET30) CC932-8B (met4::GAL1&#226;&#8364;&#8220; <br /> MET4 met30::LEU2) cells grown raffinose medium                                                                         inhibitory region   auxiliary domain (AUX), <br /> expression induced transferring cells fresh galactose   domains required tethering Met4p DNA <br /> (2%) medium 90 min. repressing L-Met (1 mM)         protein&#226;&#8364;&#8220;protein interactions Met31p/Met32p added total RNA extracted times indicated.          Met28p/Cbf1p, called INT domain bZIP <br /> Expression MET16, MET25 GAL1&#226;&#8364;&#8220;MET4 determined                domain, respectively (Kuras Thomas, 1995; Blaiseau Northern analysis. actin probe used control RNA loaded.                                                   Thomas, 1998). determine particular aspect <br />                                                                         Met4p function required induce lethality                                                                         absence Met30p, different mutant derivatives Met4p <br /> (MET16 MET25) cells expressing MET4                     expressed GAL1 promoter met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; <br /> GAL1 promoter presence absence MET30.                      cells tested viability. shown Figure 3A, corresponding strains (met4::GAL1&#226;&#8364;&#8220;MET4                      deletion AD (pGalMet4&#226;&#710;&#8224;12) INT (pGalMet4&#226;&#710;&#8224;38) <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;) grown                         domains Met4p rescued GAL1&#226;&#8364;&#8220;MET4-induced leth- <br /> presence raffinose generations, transferred             ality met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells. contrast, deletion medium containing galactose induce GAL1&#226;&#8364;&#8220;MET4                       AdoMet regulatory regions, (pGalMet4&#226;&#710;&#8224;30) <br /> gene expression 90 min repressive concentra-               AUX (pGalMet4&#226;&#710;&#8224;37) domains, did prevent lethal- <br /> tion methionine added, samples extracted                  ity. consistent findings met4&#226;&#710;&#8224; regular time intervals methionine addition. shown              met32&#226;&#710;&#8224;, cbf1&#226;&#710;&#8224; met28&#226;&#710;&#8224;, bypass met30&#226;&#710;&#8224; Figure 2B, Northern blot analysis demonstrated                  arrest, reinforces idea unregulated transcrip- <br /> repression MET16 MET25 transcription response                 tional activity DNA-bound complex containing Met4p high methionine dependent MET30. MET16                        Met32p lethal. notion simplified, MET25 genes repressed wild-type kinetics                   removal Met4p bZIP domain met4::GAL1&#226;&#8364;&#8220;MET4 strains, insensitive high                thought bind Met28p Cbf1p results partial <br /> extracellular methionine met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                      rescue met30&#226;&#710;&#8224; (pGalMet4&#226;&#710;&#8224;146; Figure 3A), cells. Insensitivity MET16 MET25 genes                    simple deletion MET28 CBF1 sufficient high extracellular methionine met4::GAL1&#226;&#8364;&#8220;MET4                        bypass met30&#226;&#710;&#8224; lethality. <br /> met30&#226;&#710;&#8224; cells consistent results showing                  Given expectation Met30p-mediated regulation repressive amounts extracellular methionine                  Met4p requires interaction factors, <br /> AdoMet did suppress lethality induced MET4                   predicted mutations Met4p expression met30&#226;&#710;&#8224; cells. control, transcription             exist impairing Met30p&#226;&#8364;&#8220;Met4p interaction GAL1&#226;&#8364;&#8220;MET4 fusion gene affected                    cause lethality cells expressing MET30. <br /> presence methionine. results confirmed                     previously mapped domain region previous findings Met30p required repression                Met4&#226;&#8364;&#8220;Met30 protein interaction hybrid analysis MET gene network.                                                (Thomas et al., 1995);  deletion mutant <br />    function transcriptional activator, Met4p                 unable induce lethality presence Met30p. <br /> recruited sequence-specific upstream regions                  suggested Met4 domain include <br />  <br />                                                                                                                                 1615 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 3.   Expression Met4p derivatives met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells. Schematic representations modified Met4p derivatives expressed GAL1 promoter region shown. Plasmids encoding fusion genes introduced CC807-1C (met4::TRP1 met30::LEU2) cells. Serial <br /> dilutions resulting transformants plated media containing 2% raffinose galactose carbon source. (B) Expression Met4p <br /> derivatives met4&#226;&#710;&#8224; MET30 cells. Details deletions scaled  Plasmids encoding fusion genes introduced CD106 <br /> (met4::TRP1) cells. Serial dilutions resulting transformants plated media containing 2% raffinose galactose carbon source. <br />  <br />  <br />  <br /> Table  Suppression met30&#226;&#710;&#8224;-induced lethality inactivation MET32 gene <br />  <br /> Genotype             Phenotype         Genotype                 Phenotype        Crosses <br />  <br /> met30&#226;&#710;&#8224;               lethal <br /> met4&#226;&#710;&#8224;                Met&#226;&#8364;&#8220;              met4&#226;&#710;&#8224; met30&#226;&#710;&#8224;             Met&#226;&#8364;&#8220;             Figure 1 <br /> met28&#226;&#710;&#8224;               Met&#226;&#8364;&#8220;              met28&#226;&#710;&#8224; met30&#226;&#710;&#8224;            lethal           CC958 &#226;&#171;&#189; CD130-1A &#226;&#171;&#187; CC850-2B <br /> cbf1&#226;&#710;&#8224;                Met&#226;&#8364;&#8220;              cbf1&#226;&#710;&#8224; met30&#226;&#710;&#8224;             lethal           CC806 &#226;&#171;&#189; CD127-1AT &#226;&#171;&#187; R31-5C <br /> met31&#226;&#710;&#8224;               Met&#226;&#171;&#185;              met31&#226;&#710;&#8224; met30&#226;&#710;&#8224;            lethal           CC867 &#226;&#171;&#189; C847-1D &#226;&#171;&#187; CD179 <br /> met32&#226;&#710;&#8224;               Met&#226;&#171;&#185;              met32&#226;&#710;&#8224; met30&#226;&#710;&#8224;            Met&#226;&#171;&#185;             CC867 &#226;&#171;&#189; C847-1D &#226;&#171;&#187; CD179 <br />  <br />  <br />  <br /> amino acid residues outside 189&#226;&#8364;&#8220;235, consistent                     essential function Met30p required data reported Omura colleagues, isolated                       passage G1 phase <br /> Met4p point mutant (F156S) unresponsive                         Conditional expression MET4 gene GAL1 <br /> high extracellular methionine (Omura et al., 1996).                      promoter met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; double mutants allowed constructed new set deletions                       characterize phenotype met30&#226;&#710;&#8224; mutants. region GAL1&#226;&#8364;&#8220;MET4 expressed met4&#226;&#710;&#8224;                             performed growth curve analysis determine number <br /> MET30 cells. derivatives, Met4&#226;&#710;&#8224;47 (amino acids                          generations arrest, fluorescence-activated cell <br /> 160&#226;&#8364;&#8220;221) Met4&#226;&#710;&#8224;26 (amino acids 160&#226;&#8364;&#8220;186), induced                         sorter (FACS) analysis measure DNA content, lethality met4&#226;&#710;&#8224; MET30 cells (Figures 3B 4E),                         determined budding index. Early log phase cultures <br /> indicating Met30p-mediated regulation Met4p                     met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; grown raffinose, dependent larger domain previously                         split  galactose added culture, cell <br /> reported, supporting notion unregulated Met4p                  number measured time. doubling activity responsible lethality met30&#226;&#710;&#8224;                   GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells arrested, raffinose <br /> mutants.                                                                    culture continued increase cell number (Figure 4A). <br />  <br /> 1616 <br />                                                                                                          Role Met4 G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 4. Conditional met30&#226;&#710;&#8224; mutant cells arrest large unbudded cells 1N DNA.   CC932-6D (met4::GAL1&#226;&#8364;&#8220;MET4 MET30) (j) CC932- <br /> 8B (met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2) (d) cells grown presence 2% raffinose carbon source. time indicated arrow, <br /> 2% galactose added culture growth followed measuring OD650 times indicated. (B) Wild type, met4::GAL1&#226;&#8364;&#8220; <br /> MET4 MET30 met4::GAL1&#226;&#8364;&#8220;MET4 met30::LEU2 grown presence raffinose, cultures divided  filtered transferred raffinose galactose medium. FACS analyses performed times indicated. (C) CC807-1C (met4::TRP1 met30::LEU2) cells transformed pGal316, pGalMet4-1 pGalMet4&#226;&#710;&#8224;12 plasmids. Resulting transformants grown presence 2% raffinose carbon <br /> source. Cultures filtered transferred galactose (2%) medium. Photographs cells taken times indicated. <br /> (D) Budding index cells grown  . (E) CC807-1C (met4::TRP1 met30::LEU2) CD106 (met4::TRP1) transformed plasmids <br /> indicated. Resulting transformants grown raffinose medium early log phase arrested &#206;&#177; factor 3 h 2% galactose. cells released &#206;&#177; factor washing fresh galactose medium analyzed DNA content flow cytometry times indicated. <br />  <br />  examine cells particular cell                          multiple biological pathways cause <br /> cycle arrest phenotype, grew parallel cultures                       unbudded G1 phenotype disrupted (Nasmyth, <br /> met4::GAL1&#226;&#8364;&#8220;MET4 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                                 1996). gain insight mechanism responsible strains raffinose galactose, took samples 0,                 G1 arrest observed GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells, <br /> 3 6 h. shown Figure 4B, expression MET4                       undertook detailed study characteristics absence MET30 causes cells accumulate                     GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; arrest. map arrest position G1 unreplicated DNA. Microscopic observation                       met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutant, arrested showed &#226;&#172;&#381;90% GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells                             asynchronous culture growing raffinose mating <br /> remain unbudded 6 h shift galactose                       pheromone 2 h, split culture  added <br /> medium (Figure 4C D).                                                  raffinose half galactose half    G1 arrest phenotype met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                       20 min, released culture pheromone-free medium, <br /> cells correlated requirement Met4p activa-                took samples 10 min intervals measure DNA <br /> tion domain induce lethality strains. Expression               content budding morphology. Cells raffinose pro- GAL1&#226;&#8364;&#8220;MET4&#226;&#710;&#8224;AD produced little alteration FACS                     ceeded cell cycle similar FACS profile <br /> profile, expression GAL1&#226;&#8364;&#8220;MET4 met4&#226;&#710;&#8224; met30&#226;&#710;&#8224;                     wild type (Figure 5A), demonstrating absence <br /> cells caused G1 arrest (Figure 4E).  expression                  Met4p, deletion MET30 marginal <br /> GAL1&#226;&#8364;&#8220;MET4 160&#226;&#8364;&#8220;221), shown                             effect cell cycle progression. contrast, met4:: <br /> induce lethality presence MET30, arrested                   GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants remained G1 phase cells G1 phase. Expression GAL1&#226;&#8364;&#8220;MET4                       undergoing DNA replication (Figure 5A) initiat- <br /> derivatives, described  showed FACS profiles sim-                    ing budding (Figure 5B). GAL1&#226;&#8364;&#8220;MET4 induced <br /> ilar wild type.  unregulated activity Met4p,                    met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; mutants release lack 160&#226;&#8364;&#8220;221) domain deletion                    pheromone, cells progressed cell cycle MET30, sufficient cause G1 arrest.                              accumulating G1 phase (data shown), <br />  <br />                                                                                                                                          1617 <br />  E.E.Patton et al. <br />  <br />  <br />  <br />  <br /> Fig. 5. essential function MET30 required pheromone arrest position, prior initiation budding DNA <br /> replication.   Wild-type, met4::GAL1&#226;&#8364;&#8220;MET4 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells grown rich raffinose medium early log phase arrested &#206;&#177; factor 2 h. cells split cultures raffinose added  galactose added 20 min. cells released &#206;&#177; factor washing fresh medium time points taken designated intervals. (B) Photographs cells   demonstrate met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; arrest large unbudded cells. tear shape cells effects pheromone. (C) MET30 required cdc7-1 arrest point. Wild-type, cdc7-1 cdc7-1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cultures incubated 37&#194;&#176;C 2 h, galactose added 30 min, shifted 25&#194;&#176;C samples taken time indicated. <br />  <br />  <br />  <br /> suggesting effects GAL1&#226;&#8364;&#8220;MET4 specific                    replication, budding (Yoon et al., 1993). G1 phase cell cycle. Unregulated Met4p                     cdc7-1 mutants arrested 37&#194;&#176;C small buds 1N  function shortly pheromone arrest                DNA content (Figure 5C), progressed S point inhibit passage Start. Consistent                G2 phase shift 25&#194;&#176;C. Similarly, met4::GAL1&#226;&#8364;&#8220; <br /> hypothesis, overexpression MET4, control                 MET4 met30&#226;&#710;&#8224; cdc7-1 mutants arrested G1 phase endogenous levels Met30p, induces 20 min delay                      37&#194;&#176;C, progressed S G2 phase shifted Start (Figure 5A). Met30p originally identified                 25&#194;&#176;C galactose, placing met30&#226;&#710;&#8224; arrest position mutant defective repression MET                        pheromone block Cdc7p require- <br /> gene transcription (Thomas et al., 1995). Given                        ment point cell cycle. <br /> overexpression MET4 delay passage G1 cell cycle 20 min, examined progression                    G1-phase-arrested met30&#226;&#710;&#8224; mutants MET30-1 mutant cells cell cycle FACS                      compromised efficient growth translation <br /> analysis. MET30-1 cells delayed passage S                     Start mutants categorized affecting <br /> phase 10 min (data shown), indicating non-                      growth mutations <span id='am-37' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span>cAMP</span> pathway lethal mutations MET30 partially defective                  translational machinery, affecting division Met4p regulation capable producing G1&#226;&#8364;&#8220;S                       mutations CDC28 G1 cyclins (reviewed phase delay.                                                                Hartwell, 1994). determine component Start <br />    shift pheromone-induced arrest met30&#226;&#710;&#8224;                    altered conditional met30&#226;&#710;&#8224; mutants, mon- <br /> arrest demonstrated met30&#226;&#710;&#8224; arrest phenotype                    itored rate growth small, unbudded met4::GAL1&#226;&#8364;&#8220; pheromone arrest point prior                     MET4 met30&#226;&#710;&#8224; G1 cells collected centrifugal elutriation <br /> DNA replication budding initiation point.                        grown raffinose galactose (Figure 6A). converse experiment, e. arresting met4::GAL1&#226;&#8364;&#8220;MET4                     expected, raffinose medium cells grew steadily met30&#226;&#710;&#8224; cells G1 phase galactose releasing                    time passed Start (25% budded) cell glucose pheromone medium, cells unable                    volume 50 fl. cells progressed S phase recover prolonged GAL1&#226;&#8364;&#8220;MET4 arrest.                       G2&#226;&#8364;&#8220;M phase, indicated increase bud <br /> alternative, constructed met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;                        index DNA content (Figure 6B C). contrast, <br /> cdc7-1 mutant, arrested cells 37&#194;&#176;C released                      cells grown galactose medium did initiate budding galactose-containing medium 25&#194;&#176;C.                        (Figure 6B) maintained 1N DNA content (Figure <br /> Cdc7p kinase required initiation DNA                          6C), did,  accumulate mass time (Figure <br />  <br /> 1618 <br />                                                                                                               Role Met4 G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 6. Conditional G1-arrested met30&#226;&#710;&#8224; mutants slowed cell growth translation. Cultures small G1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells <br /> grown raffinose collected centrifugal elutriation, divided  raffinose added culture galactose  Samples taken times indicated measure   cell volume, (B) budding index (C) DNA content flow cytometry. (D) Polysome profiles wild-type met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells Start position. G1 cells collected centrifugal elutriation rich, raffinose <br /> medium, galactose added time 0, samples taken times indicated polysome profiles, flow cytometry budding analysis. <br />  <br />  <br />  <br /> 6A). Initially, conditional met30&#226;&#710;&#8224; mutants grown                        galactose. Similarly, number discrete polysome <br /> galactose medium accumulated mass rate equal                           peaks UV absorbance polysome peaks raffinose medium, 100 min rate                       slightly reduced met30&#226;&#710;&#8224; mutants 2.5 h (data <br /> growth slowed somewhat.                                                    shown). perform detailed analysis, used <br />    determine decreased rate mass accu-                       synchronized G1 cells isolated centrifugal elutriation <br /> mulation met30&#226;&#710;&#8224; mutants defects                         examined polysome profiles wild-type met30&#226;&#710;&#8224; translation, measured incorporation [35S                       cells function time. comparison wild-type <br /> thionine/cysteine total protein, examined                         cells, extent polyribosome formation reduced <br /> polysome profiles conditional met30&#226;&#710;&#8224; mutants.                            correspondingly, 80S ribosome peak increased <br /> [35S]methionine/cysteine incorporation decreased                        met30&#226;&#710;&#8224; mutants 2.5 h, time wild-type <br /> 20% conditional met30&#226;&#710;&#8224; mutants 2.5 h                           cells passed Start (Figure 6D).  <br />                                                                                                                                                 1619 <br />  E.E.Patton et al. <br />  <br />  <br /> appears met30&#226;&#710;&#8224; cells able reach critical      budding yeast. genetic analysis shows essential <br /> cell size required passage Start gradually    function MET30 bypassed deletion MET4, impaired translation.                               gene encodes main transcriptional activator                                                               sulfur amino acid biosynthesis pathway, Various G1 cyclin RNA transcripts reduced              unregulated Met4p activity lethal. creation met30&#226;&#710;&#8224; mutants                                                met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; conditional mutant conditional met30&#226;&#710;&#8224; strain slightly com-       allowed confirm function Met30p regulating <br /> promised translation, reasoned compon-      Met4p function (Figure 2B) characterize arrest <br /> ents cell division machinery altered       phenotype met30&#226;&#710;&#8224; cells (Figures 3 4). Surprisingly, cells. tested overexpression       met30&#226;&#710;&#8224; conditional mutants arrest large, unbudded cells G1 cyclins rescue met30&#226;&#710;&#8224; phenotype.          unreplicated DNA. Genetic analysis allowed Expression genes encoding G1 cyclins, CLN1,        determine G1 arrest phenotype met30&#226;&#710;&#8224; <br /> CLN2, CLN3, CLN3-1, PCL1 PCL2, S G2            conditional mutant likely result cyclins, CLB2 CLB5, GAL1 promoter              unbalanced synthesis sulfur metabolite met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants unable bypass           mutations structural genes MET pathway lethal phenotype (data shown). Intriguingly,     alter lethal phenotype met30&#226;&#710;&#8224; strains (Figure 1). CLN1 CLN2 genes expressed GAL1           G1 arrest phenotype dependent transcriptional <br /> promoter 3 h, Cln1 Cln2-Cdc28 kinase activity          activity Met4p, Met4p mutants detected met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224;           transcriptional activation domain (Met4&#226;&#710;&#8224;AD), mutant strain. Similarly, Western analysis revealed      incapable binding cofactors Met31p Met32p <br /> Cln1p Cln2p barely detectable (data shown).      (Met4&#226;&#710;&#8224;INT), lethal met30&#226;&#710;&#8224; background ensure decrease protein level      (Figure 3A). Consistent results, met32&#226;&#710;&#8224; repression GAL1 promoter, examined          mutation capable bypassing met30&#226;&#710;&#8224; lethal levels Cln2p expressed endogenous pro-        phenotype (Figure 1B). fact met32&#226;&#710;&#8224; mutation, <br /> moter constitutive ADH1 promoter.         met31&#226;&#710;&#8224; mutation, specifically capable case, Cln2p undetectable met4::GAL1&#226;&#8364;&#8220;MET4           bypassing met30&#226;&#710;&#8224; mutation somewhat unexpected <br /> met30&#226;&#710;&#8224; cells (data shown). performed North-       Met31p Met32p, highly homolo- <br /> ern analysis total RNA extracted met4::GAL1&#226;&#8364;&#8220;        gous zinc finger factors, thought redundant <br /> MET4 met30&#226;&#710;&#8224; cells grown presence galactose          functions (Blaiseau et al., 1997). instance, CLN2 transcripts undetectable,      double met31&#226;&#710;&#8224; met32&#226;&#710;&#8224; mutant methionine auxotroph, contrast, ACT1 mRNA levels clearly detectable,        single met31&#226;&#710;&#8224; met32&#226;&#710;&#8224; mutants (Blaiseau slightly reduced (Figure 7A). protein level   et al., 1997). Likewise, mobility shift assays revealed suggested, CLN1 transcripts undetectable,    difference Met4&#226;&#8364;&#8220;Met28&#226;&#8364;&#8220;Met31 Met4&#226;&#8364;&#8220; PCL2 transcripts, CLN3, WHI3 WHI4              Met28&#226;&#8364;&#8220;Met32 complexes assembled MET3 transcripts appeared normal (data shown).                 MET28 promoter regions (Blaiseau Thomas, 1998). <br />    important determine CLN1 CLN2        results suggest,  precise <br /> mRNAs failed accumulate G1 phase             function Met31p Met32p, met30&#226;&#710;&#8224; mutant cells, lack detectable        proteins required assembly transcripts secondary consequence prolonged      Met4p-containing complexes, differ. <br /> G1 arrest. collected small, unbudded, G1            Regulation Met4p Met30p dependent met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells centrifugal elutri-           Met30p binding region called (Thomas et al., <br /> ation, added raffinose half culture         1995). Accordingly, Met4p derivative lacking galactose  collected samples Northern,   region (amino acids 189&#226;&#8364;&#8220;235) recently shown FACS budding analysis. cells grown raffinose       sensitive SCFMet30-triggered degradation response <br /> medium expressed CLN1, CLN2 CLN3 transcripts              high extracellular methionine (Rouillon et al., 2000). <br /> appropriately, grew cell size, initiated budding        analysis regions Met4p <br /> replicated DNA (Figure 7B C). contrast,      required G1 arrest met30&#226;&#710;&#8224; mutants, cells grown galactose medium failed accumulate          Met30p&#226;&#8364;&#8220;Met4p interaction region previously <br /> CLN1 CLN2 mRNAs did wild-type levels             defined hybrid analysis did entirely correlate CLN3 mRNAs (Figure 7B).  cells did           Met4p region required inhibition Met30p initiate budding DNA replication (Figure 7C).      deletion sufficient induce <br /> inability met30&#226;&#710;&#8224; cells accumulate CLN1/2 PCL1/      G1 arrest cells carrying MET30. Deletion slightly <br /> 2 mRNAs sufficient account arrest Start      greater domain (amino acids 160&#226;&#8364;&#8220;220, Met4&#226;&#710;&#8224;47)  (Espinoza et al., 1994; Measday et al., 1994).        contrast, capable inducing G1 arrest MET30 rule possibility G1 phase        background. suggest previously defined transcripts affected.                                domain required Met30p&#226;&#8364;&#8220;Met4p hybrid <br />                                                               interaction, Met30p repression Met4p requires <br /> Discussion                                                    amino acids 160&#226;&#8364;&#8220;220 Met4p. <br />                                                                  Conditional met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; mutants arrest <br /> Met4p passage G1 phase                 G1 phase cell cycle shift cell cycle                                                    galactose medium (Figures 4A 5). essential study, demonstrate components MET      function Met30p required pheromone <br /> gene network critical passage Start        arrest point, budding DNA replication  <br /> 1620 <br />                                                                                                           Role Met4 G1&#226;&#8364;&#8220;S transition <br />  <br />  <br />  <br />  <br /> Fig. 7. Conditional met30&#226;&#710;&#8224; mutants lack CLN1 CLN2 mRNA transcripts.   Northern analysis CLN mRNA transcripts demonstrates lack CLN2 mRNA expressed endogenous ADH1 promoter galactose-grown met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells. (B) CLN1 CLN2, CLN3 mRNAs, fail accumulate elutriated G1 met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells grown galactose, accumulate normally met4::GAL1&#226;&#8364;&#8220;MET4 met30&#226;&#710;&#8224; cells grown raffinose. Cells grown rich, raffinose medium, culture split  raffinose added culture galactose  Samples taken (B) Northern analysis (C) flow cytometry. <br />  initiated. Overexpression MET4 does                       series genes inhibits passage G1 phase. <br /> cause G1 arrest, sufficient cause slight                   G1 arrest dependent Met4p activation <br /> delay G1 S phase, suggesting overexpression                  domain, transcriptional activity model arise Met4p transiently overcome limiting                      specifically Met4p Met32p act repression SCFMet30.  MET30-1 mutation,                       recruiting Met4p DNA, Met32p does appear partially defective Met4p repression response                transcriptional activity (Blaiseau et al., 1997). high extracellular methionine (Thomas et al., 1995),                     Transcription G1 phase &#226;&#8364;&#732;inhibitor&#226;&#8364;&#8482; gene exhibits delay passage G1 phase.                     depend known Met4p transcription complexes MET30-1 mutation lethal indicate                        assembled upstream classical MET genes excess Met4p activity admissible                       deletion CBF1 MET28 does suppress cell, complete lack regulation, met30&#226;&#710;&#8224;,                  lethality met30&#226;&#710;&#8224; mutation (Figure 1B). Likewise, <br /> intolerable.                                                                &#226;&#8364;&#732;inhibitor&#226;&#8364;&#8482; gene(s) required methionine <br />                                                                             biosynthesis met32&#226;&#710;&#8224; met30&#226;&#710;&#8224; cells methi- <br /> Models Met4p function G1 phase                                   onine prototrophs.  Met4p responsible currently exploring models manifesta-                    regulation multiple gene networks assemble <br /> tion met30&#226;&#710;&#8224; G1 arrest phenotype.                    numerous transcriptional activation complexes. specific transcription activation complex comprising                      alternative model met30&#226;&#710;&#8224; G1 arrest  <br /> Met4p Met32p regulates expression gene                      lack Met4p degradation, increased protein <br />  <br />                                                                                                                                            1621 <br />  E.E.Patton et al. <br />  <br />  <br /> Table II. Yeast strains <br />  <br /> Strain                         Genotype                                                                  Reference <br />  <br /> C180                           MATa,   ade2, his3, leu2, ura3, str4::URA3                                Che&#194;&#180; rest Surdin-Kerjan (1992) <br /> CD127-1AT                      MATa,   ade2, his3, leu2, ura3, met30::LEU2, pLexMet30-4                  Thomas et al. (1995) <br /> CD127-1CT                      MATa,   ade2, his3, leu2, ura3, met30::URA3, pLexMet30-4                  Thomas et al. (1995) <br /> CD106                          MATa,   ade2, his3, leu2, ura3, met4::TRP1                                Thomas et al. (1992) <br /> CD130-1A                       MATa,   ade2, his3, leu2, ura3, met28::LEU2                               Kuras et al. (1996) <br /> CD155                          MATa,   ade2, his3, leu2, ura3, met3::URA3                                study <br /> CD171                          MATa,   ade2, his3, leu2, ura3, met4::URA3, met30::LEU2                   study <br /> CD179                          MATa,   ade2, his3, leu2, ura3, met31::LEU2, met32::TRP1                  Blaiseau et al. (1997) <br /> CC435-6C                       MATa,   ade2, his3, leu2, ura3, sam1::URA3, sam2::URA3                    Thomas et al. (1988) <br /> CC807-1C                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::LEU2                   study <br /> CC807-7D                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::LEU2                   study <br /> CC847-1C                       MATa,   ade2, his3, leu2, ura3, met30::URA3, pLexMet30-4                  study <br /> CC849-8A                       MATa,   ade2, his3, leu2, ura3, met4::TRP1                                study <br /> CC850-2B                       MATa,   ade2, his3, leu2, ura3, met4::TRP1, met30::URA3                   study <br /> CC864-1B                       MATa,   ade2, his3, leu2, ura3, met6                                      study <br /> CC867-1B                       MATa,   ade2, his3, leu2, ura3, met30::URA3, met31::LEU2, met32::TRP1     study <br /> CC867-1D                       MATa,   ade2, his3, leu2, ura3, met30::URA3, met32::TRP1                  study <br /> CC874-18D                      MATa,   ade2, his3, leu2, ura3, str4::URA3                                study <br /> CC906-13C                      MATa,   ade2, his3, leu2, ura3, met3::URA3                                study <br /> CC932-6D                       MATa,   ade2, his3, leu2, ura3, met4::GAL1&#226;&#8364;&#8220;MET4                           study <br /> CC932-8B                       MATa,   ade2, his3, leu2, ura3, met4::GAL1&#226;&#8364;&#8220;MET4, met30::LEU2              study <br /> CC954-10A                      MATa,   leu2, ura3, cdc7-1                                                study <br /> CC954-10C                      MATa,   his3, leu2, ura3, cdc7-1, met4::GAL1&#226;&#8364;&#8220;MET4, met30::LEU2            study <br /> CC970-1A                       MATa,   ade2, his3, leu2, ura3, cdc7-1, met4::GAL1&#226;&#8364;&#8220;MET4                   study <br /> R31-5C                         MATa,   ade2, his3, leu2, lys2, ura3, cbf1::TRP1                          O&#226;&#8364;&#8482;Connell Baker (1992) <br /> W303-1A                        MATa,   ade2, his3, leu2, trp1, ura3,                                     R.Rothstein <br /> W744-1A                        MATa,   ade2, his3, leu2, ura3, sam1::LEU2, sam2::HIS3                    Bailis Rothstein (1990) <br />  <br />  <br />  <br />                                                                             1996; Polymenis Schmidt, 1999). GAL1&#226;&#8364;&#8220; <br /> Table III. Plasmid list <br />                                                                             MET4 met30&#226;&#710;&#8224; mutants unable pass Start, <br /> Plasmid          Relevant characteristics             Source                examined possibilities potential route                                                                             G1 arrest phenotype. used synchronized cells pGalMet4-1       pGal1-MET415&#226;&#8364;&#8220;666, URA3, CEN          study            determine met30&#226;&#710;&#8224; mutants capable pGalMet4&#226;&#710;&#8224;12      pGal1-MET4&#226;&#710;&#8224;79&#226;&#8364;&#8220;180, URA3, CEN         study <br /> pGalMet4&#226;&#710;&#8224;13      pGal1-MET4&#226;&#710;&#8224;189&#226;&#8364;&#8220;221, URA3, CEN        study            growth G1 phase, determine translation <br /> pGalMet4&#226;&#710;&#8224;21      pGal1-MET4&#226;&#710;&#8224;115&#226;&#8364;&#8220;184, URA3, CEN        study            compromised (Figure 6). met30&#226;&#710;&#8224; mutant <br /> pGalMet4&#226;&#710;&#8224;26      pGal1-MET4&#226;&#710;&#8224;160&#226;&#8364;&#8220;186, URA3, CEN        study            did continue accumulate mass G1 phase pGalMet4&#226;&#710;&#8224;30      pGal1-MET4&#226;&#710;&#8224;211&#226;&#8364;&#8220;232, URA3, CEN        study            prolonged growth arrest, translation slightly slowed <br /> pGalMet4&#226;&#710;&#8224;37      pGal1-MET4&#226;&#710;&#8224;352&#226;&#8364;&#8220;366, URA3, CEN        study            comparison MET30 cells, polysome profiles <br /> pGalMet4&#226;&#710;&#8224;38      pGal1-MET4&#226;&#710;&#8224;375&#226;&#8364;&#8220;430, URA3, CEN        study <br /> pGalMet4&#226;&#710;&#8224;47      pGal1-MET4&#226;&#710;&#8224;160&#226;&#8364;&#8220;221, URA3, CEN        study            showed corresponding defect translational apparatus. <br /> pGalMet4&#226;&#710;&#8224;146     pGal1-MET4&#226;&#710;&#8224;616&#226;&#8364;&#8220;666, URA3, CEN        study            delayed kinetics protein synthesis response <br /> pMT291           CLN2HA, LEU2, CEN                    Tyers laboratory      compared cycle G1 arrest suggest pMT1241          CLN1HA, LEU2, CEN                    Tyers laboratory      effect translation secondary consequence pMT485           pGAL1-CLN1HA, LEU2, CEN              G.Tokiwa <br /> pMT634           pGAL1-CLN2HA, LEU2, URA3,            Tyers laboratory      arrest. contrast, unexpectedly CLN1, <br />                  CEN                                                        CLN2 PCL2 transcripts entirely absent pCB1317          Sc-pADH1-CLN2HA, LEU2, CEN           K.Arndt               met30&#226;&#710;&#8224; cells, CLN3, WHI3 WHI4 transcripts <br />                                                                             unaffected. loss G1 cyclin transcripts <br />                                                                             occurs effects protein synthesis, levels Met4p inhibit passage G1 phase                         suggest effect G1 transcripts directly <br /> associating specifically non-specifically tran-               result unregulated Met4p activity. cln1 cln2 <br /> scriptional complexes.  specificity                      pcl1 pcl2 mutants arrest Start, absence met32&#226;&#710;&#8224; mutation bypass G1 arrest,                     transcripts account phenotype met30&#226;&#710;&#8224; cells. <br /> altered CLN1 CLN2 CLN3 transcript levels,                       CLN transcripts unstable wild-type <br /> argue Met4p non-specifically sequestering                     cells, unregulated Met4p activity met30&#226;&#710;&#8224; mutant general transcription machinery away promoters                     hyperactivate G1 cyclin RNA degradation mechan- G1-specific genes.                                                       ism. results indicate pathway <br />                                                                             presumably affect CLN3 mRNA. possible <br /> Potential mechanisms conditional met30&#226;&#710;&#8224;                             comparison, SCF-like complex called VCB&#226;&#8364;&#8220;CUL2 <br /> G1 arrest                                                                   (comprised VHL&#226;&#8364;&#8220;ElonginB/C&#226;&#8364;&#8220;CUL2) thought variety G1 mutants arrest unbudded cells                  regulate, directly indirectly, accumulation unreplicated DNA identified S.cerevisiae,                      hypoxia-inducible mRNAs targeting protein including cell division mutants  cdc28 cln                      degradation hypoxic conditions (reviewed Kaelin <br /> mutants) cell growth translation mutants                    Maher, 1998; Tyers Rottapel, 1999). Nutrient- Ras-cAMP tor&#226;&#8364;&#8220; mutants; Hartwell, 1994; Nasmyth,                     dependent modifications RNA stability  <br /> 1622 <br />                                                                                                              Role Met4 G1&#226;&#8364;&#8220;S transition <br />  <br />  <br /> identified mechanism glucose repression (Lombardo                    loss. Diploids sporulated, progeny cross                                                                              analyzed (20 tetrads  Figure 1B). crosses involving et al., 1992; Cereghino Scheffler, 1996). control                  met3&#226;&#710;&#8224;, met6&#226;&#710;&#8224;, str4&#226;&#710;&#8224; double sam1&#226;&#710;&#8224;sam2&#226;&#710;&#8224; mutants transcription, translation protein stability                    met30&#226;&#710;&#8224; LexA-MET30, HIS3 strain, progeny tetrads contained explored means control G1 cyclin levels,                     viable spores, spores Leu&#226;&#171;&#185;  e. met30&#226;&#710;&#8224;) knowledge, results constitute suggestion                     e. LexA-Met30). progeny cross involving control level G1 cyclin mRNA stability.                       met4&#226;&#710;&#8224; mutation showed high number tetrads giving rise                                                                              viable spores (13 total 20 tested tetrads)  67 viable <br />                                                                              spores, 17 leucine prototrophs  e. met30&#226;&#710;&#8224;) Methionine cell cycle                                                histidine auxotrophs  e. did harbor pLexM30-4 plasmid). <br /> Connections cell cycle methionine                       addition, 17 spores tryptophan prototrophs <br />                                                                              methionine auxotrophs, proving met4&#226;&#710;&#8224;. Conversely, identified Unger Hartwell (1976)                       leucine prototroph spores histidine tryptophan <br /> study met mutants arrested G1 phase                        auxotrophs recovered. results suggested met30&#226;&#710;&#8224; <br /> absence methionine, methionine tRNA synthase                       spores recovered contain met4&#226;&#710;&#8224; disruption mutation. <br /> mutants arrested G1 phase presence                    confirm result, met4&#226;&#710;&#8224; met30&#226;&#710;&#8224; spore backcrossed wild- <br /> methionine. predicted &#226;&#8364;&#732;signal&#226;&#8364;&#8482; allowed                       type strain met4&#226;&#710;&#8224; MET30 strain. cross (CC808), <br />                                                                              25% recovered spores leucine prototroph spores cell arrest specifically G1 phase acted                     spores tryptophan prototrophs, cross <br /> level protein biosynthesis. met30&#226;&#710;&#8224; G1                         (CC809) leucine character segregated perfectly 2&#226;&#171;&#185;/2&#226;&#8364;&#8220;. <br /> arrest similar arrest described Unger                                                                              Plasmids <br /> Hartwell (1976) determined. DNA microarray                     express different Met4p derivatives GAL1 promoter <br /> analysis revealed expression of                    region autonomous replicating plasmid, used pGal316 MET genes specifically activated S phase                       pGal313 plasmids (Rouillon et al., 2000). cleaved (Spellman et al., 1998), suggesting cell cycle                      EcoRI BamHI ligated different EcoRI&#226;&#8364;&#8220;BamHI fragments <br />                                                                              encoding deleted derivatives Met4p set pLexMet4 plasmids <br /> sulfur amino acid metabolism closely linked.                         described Kuras Thomas (1995). New Met4p derivatives study reinforces connection linking regulation                   constructed described Kuras Thomas (1995) main regulator MET network, Met4p,                        ligations directly pGal313 pGal316 plasmids cleaved cell cycle. Important questions arise                     EcoRI BamHI (Table III). <br /> work regulation Met4p                              integrate GAL1&#226;&#8364;&#8220;MET4 fusion gene genome,                                                                              cloned StuI&#226;&#8364;&#8220;EcoRI fragment, blunt ended Klenow fragment <br /> couple methionine passage G1 phase. Cell                        plasmid pM4-1 (Thomas et al., 1992) pGalM4-1, cleaved size experiments rate passage Start                    Asp718, blunt ended Klenow fragment dephosphorylated. synchronized, small G1 cells suggest presence                    correct ligation events restored Asp718 site, resulting plasmid high extracellular methionine, cells reset critical                 contained Asp718&#226;&#8364;&#8220;BamHI fragment corresponding GAL1&#226;&#8364;&#8220; <br />                                                                              MET4 gene fusion flanked 5&#226;&#172;&#732; 3&#226;&#172;&#732; regions MET4 gene. <br /> cell size larger volume delayed passage                     Owing step gene disruption method (Rothstein, 1983), Start (E.E.Patton M.Tyers, unpublished data).                       fragment used replace met4::TRP1 locus strain CC849- <br /> initial finding superficially odds clear                      8A GAL1&#226;&#8364;&#8220;MET4 fusion gene selecting Met&#226;&#171;&#185; transformants <br /> inhibition Start excessive Met4p activity.                     galactose medium. Correct integration events verified Southern met30&#226;&#710;&#8224; met32&#226;&#710;&#8224; mutant methionine prototroph,                      blot analyses. possible specific Met4&#226;&#8364;&#8220;Met32 transcription                      Northern blot analyses <br /> complex responds different signal increase                      Northern blotting performed described Thomas (1980), intracellular AdoMet.                                                     total cellular RNA extracted yeast described Schmitt et al. <br />                                                                              (1990) oligolabeled probes (Hodgson Fisk, 1987). <br />    Finally, sulfur amino acid biosynthesis pathway highly conserved eukaryotes (Griffith, 1987). Connec-                     Flow cytometry <br /> tions methionine metabolism cell cycle                           flow cytometry analyses, cells harvested, fixed 70% ethanol <br /> control noted mammals. vitro                    stained propidium iodide described (Mann et al., 1992). vivo studies demonstrated methionine                         Polysome profiles <br /> dependence numerous transformed cells tumors                          Polysomes isolated described (Zhong Arndt, 1993). Briefly, <br /> (Hoffman, 1997). recently, shown athymic                    cells grown log phase, 60 &#194;&#181;g/ml cycloheximide added <br />                                                                              5 min collecting cells. Cells lysed Tris-based <br /> mice grafted human cancers fed                           buffer containing heparin, extracts loaded sucrose gradients <br /> methionine-free diet greatly reduced tumor                      ranging 7 47% sucrose, spun 35 000 r.p.m. SV41 rotor <br /> burden (Kokkinakis et al., 1997). results reported                       2 h, read A254 continuous drip. emphasize importance future studies coordination nutrients, methionine,                    <br /> cell cycle.                                                                  <br />  <br /> Yeast strains media                                                       <br /> Saccharomyces cerevisiae strains used work listed Table II.    isogenic W303 background. Standard yeast media            prepared described Che&#194;&#180; rest Surdin-Kerjan (1992).            <br /> Saccharomyces cerevisiae transformed acetate chloride treat-       <br /> ment described Gietz et al. (1992). Genetic crosses, sporulation, <br /> dissection scoring nutritional markers described            <br /> Sherman et al. (1979). <br />    crosses shown Figure 1, diploids selected grown        generations presence histidine induce plasmid          <br />  <br /> </body></html>