<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>.-. 1993 Oxford University Press                                     Nucleic Acids Research, 1993, Vol. 21,  23 5391 -5397 <br />  <br />  <br /> SnR31, snR32, snR33: novel, non-essential <br /> snRNAs Saccharomyces cerevisiae <br /> Andrey G.Balakin, Gregory S.Schneider, Max S.Corbett, Jingwei Ni Maurille J.Fournier* <br /> Department Biochemistry Molecular Biology Program Molecular Cellular Biology, <br /> Lederle Graduate Research Center, University Massachusetts, Amherst, MA 01002, USA <br />  <br /> Received August 9, 1993; Revised Accepted October 6, 1993                    GenBank accession nos L22433-L22435 (incl.) <br />  <br />  <br /> ABSTRACT <br /> Genes novel yeast snRNAs                          viability, loss species alters flux identified tested essentiality. Partial sequence              intermediates rRNA processing pathway leads information developed RNA extracted                      cold-sensitive phenotype (12). Effects nearly identical isolated nuclei respective gene sequences                     observed loss essential snRNAs discovered screening DNA sequence                           observed depletion nucleolar proteins, NOP1, G- <br /> database. RNAs contain 222, 188 183                     ARI, NOP3, essential, non-essential <br /> nucleotides designated snR31, snR32                       protein NSR1 (13-17). GAR1, NOPI protein known snR33, respectively. RNA derived single                SSB1 shown associated snoRNAs <br /> copy gene. SNR31 gene adjacent gene                   (14,18,19). Taken  results suggest processing unnamed protein associated cap-binding                    rRNA, like mRNA, mediated assembly <br /> protein elF-4E. SNR32 gene gene                  snRNA particles large &apos;processome&apos; complex (4). <br /> ribosomal protein L41 gene SNR33                       pool uncharacterized snRNAs expected include <br /> chromosome Ill, open reading frames                  additional snRNAs participate ribosome biogenesis known function. Genetic disruption analyses                        stage functions unrelated manufacture ribosome <br /> showed snRNAs required                  possible. present report describes progress growth. new RNAs bring number non-                         ongoing effort characterize novel snRNAs yeast <br /> spliceosomal snRNAs characterized far                         Saccharomyces cerevisiae. approach clone new snRNA <br /> S.cerevisiae 14, essential.                genes determine genes essential growth.                                                                       phenoyWpe observed inactive gene, conditional alleles <br /> INTRODUCTION                                                          developed investigate role new RNA. <br />                                                                       effectiveness strategy established yeast Saccharomyces cerevisiae contains upwards 30 small       spliceosomal snRNAs extended vital non- <br /> nuclear RNAs (1,2). Sixteen RNAs          spliceosomal snRNAs. properties novel snRNAs <br /> partially characterized, including snRNAs involved        described communication. <br /> mRNA splicing 11 RNAs appear associated nucleolus (reviewed ref.3,4). nucleolar small RNAs         MATERIALS METHODS yeast organisms implicated ribosome <br /> biogenesis. species shown            Strains media <br /> important processing ribosomal RNA precursors, based           Yeast nuclear RNA prepared strain BJ2407  ca,   disruption rRNA processing cells extracts depleted    leu2/leu2, trpl/trpl, ura3-52/ura3-52, prbl-1122/prbl-1122, specific snRNAs; ii) purification precursor rRNAs,         prc-407/prc-407, pep4-3/pep4-73, gal2/gal2; 20,21) genomic  iii) cross-linking rRNA (reviewed refs 3-5).        DNA strain GRF167   his3A, ura3-167; G.Fink, 11 non-spliceosomal snRNAs characterized far        unpublished). Disrupted yeast SNR genes characterized S. cerevisiae essential growth, U3, U14 snR30        , Ura- strain MH2  ca, ade2-101/ade2-101, <br /> (6-8). genes dispensable, situation begs         trpl-A901/trpl-A901, ura3-52/ura3-52, leu2-3,112/leu2-3,112, <br /> explanation.                                                          his3/his3; M. Fitzgerald-Hayes, unpublished). Yeast strains    essential nucleolar snRNAs required       grown YPD liquid medium/plates (1% yeast extract, 2% <br /> normal processing rRNA. Loss impairs processing         peptone 2% glucose) selective plates SD medium primary transcript leads shortage mature 18S       (2% glucose 0.67% yeast nitrogen base) containing 20tg/ml <br /> rRNA (9-11). fourth snRNA, snR1O, essential              histidine <span id='am-45' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-46' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> 32&#194;&#176;C unless stated  Alternate <br />  <br /> *   correspondence addressed <br />  5392 Nucleic Acids Research, 1993, Vol. 21,  23 <br /> carbon sources included 2% galactose, 2% glycerol 2%              Northern hybridization characterization terminal <br /> potassium acetate.                                                   nucleotides <br />   Yeast transformed lithium acetate procedure (22).        Total yeast RNA isolated sodium dodecyl sulfate/hot <br /> Tetrad dissections carried essentially described      phenol method (32). Glass beads used break cells. <br /> Sherman Hicks (23). Diploid cells sporulated 1%          RNA separated 8% polyacrylamide gel, blotted potassium acetate, asci treated zymolyase (ICN             Gene Screen membranes UV cross-linked recommended <br /> Biochemicals, ) spores separated YPD plates.          manufacturer. membranes probed   Cloning manipulations carried E. coli DH5aF&apos;         radiolabeled DNA fragments solution containing 5 x SSPE <br /> (F&apos;/endAl, hsdR17(r-m+), supE44, thi-J, gyrA(Nalr), recAl,           buffer, 50% formamide, 0.2% SDS 50,ug/ml heparin (33). <br /> relAl, argF-lacZYA)U169 (480dlacZAM15) strain JM56              20xSSPE contained 3M NaCl, 0.2M Na2HPO4 20mM <br /> (F&apos;lacIQ-proAB, dam3,rpsL; M.J.Fournier, unpublished)                Na2.EDTA. <br /> essentially described Sambrook et al (24).                        Northern blot anti-TMG immunoprecipitated RNAs RNA isolation, 3&apos; end labeling enzymatic sequencing              S. cerevisiae kindly provided R Lempicki. <br />                                                                         Primer extension RNA dideoxy sequencing carried nuclear RNA isolation protocol combination          essentially described (34). oligonucleotides protocols. Spheroplasts prepared lyticase (Sigma; 25),     used primers: (5&apos;-TAGGAACTCATGGTGTAATT-3&apos;), <br /> lysed nuclei purified centrifugation       complementary snR31 position 8 position 27; <br /> 0.5M sucrose cushion (26). RNA extracted washed             (5&apos;-CTGATTCCCAGTGTCTCGTTTC-3&apos;), complementary nuclei phenol 3&apos; end labeled variation method   snR32 positions 153-174;  (5&apos;-GATTGTCCACACAC- <br /> developed Bruce Uhlenbeck (27). RNA incubated             TTC-3&apos;), complementary snR33 positions 167-183. 50 pmols 5&apos;[32P]pCp 36 units RNA ligase 50 I1            RNA 3&apos; end sequences determined variation solution containing: 50 mM potassium HEPES, pH 7.5, 20            T4 RNA ligase/PCR-based approach described Kiss mM MgCl2, 1 mM <span id='am-2' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>, 10mM DTT, 10% glycerol, 10%                      Filipovicz (35). Instead oligoribonucleotide, DMSO, 0.1 mg/ml BSA 40 units RNasin. Individual RNAs             oligodeoxiribonucleotide sequence (35) used purified rounds electrophoresis: 1 mm      ligation total yeast RNA. PCR products obtained gel sequencing gel. Electrophoresis      oligonucleotides specific snR31, snR32 snR33 gel- gel continued RNAs differing             purified sequenced directly snRNA-specific <br /> <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-16' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> resolved. 3&apos; end sequences pCp labeled RNAs       oligonucleotides using dsDNA Cycle Sequencing determined RNA sequencing kit (Pharmacia).               kit BRL Life Technolgies,  Oligonucleotides  <br /> Gene disruption                                                      (5 &apos;-GGAAGCCTTTCTCTTTCACC-3&apos;), corresponding                                                                      positions 161-170 snR31; (5&apos;-GATAGATTGAACGTTGC- <br /> DNA segments containing e new snRNA genes amplified             TGG-3&apos;), corresponding positions 111-130 snR32;  PCR using following primers:                                  (5 &apos;-GGTTTGAGTCGGTTCCTTCG-3&apos;), corresponding SNR31 gene: 5&apos;  GTGCGGAATTCGATATCGGGTGC-                        positions 124- 143 snR33. <br /> AAT 3&apos;  GGTGCGATATCGAATTCAGAAAAGTG, EcoRI sites                                                sequence analysis <br /> SNR32 gene: 5&apos;  GTGCGGAATTCCTCTAGAGAGACGT-                      Analyses RNA DNA sequences performed CGTC 3&apos;  CGTGGGAATTCCCGTGAACG-                              programs designed University Wisconsin Genetics <br /> AGCCGTACA, EcoRI sites                                     Group (36). <br /> SNR33 gene: 5&apos;  GCTTCCTCGAGCACACACTCTCTC- <br /> TTATT 3&apos;  CAAGGGAATTCCTGATCTCTGGCG- <br /> AACGT, XoI EcoRI sites, respectively.                       RESULTS <br />   PCR products cloned shuttle vector pRS314 <br /> (SNR32, SNR33; 28) E.coli vector pACYC 184 (SNR3J;            Isolation 3&apos; sequencing new snRNAs <br /> 29) using indicated restriction enzymes. Unique restriction sites    snRNA genes described identified partial SNR32 SNR33 coding regions used marker           sequencing purified RNA searching GenBank DNA <br /> gene insertions. AseI site witiin pACYC184 sequence           sequence database related sequences. RNA sequences <br /> eliminated AseI site middle SNR31      developed following manner. Nuclei purified <br /> used insert marker gene. BamHI fragments derived           centrifugation 0.5M sucrose cushion extracted <br /> plasmids YDp-U YDp-H (30) containing URA3 HIS3               spheroplast lysis buffer. snRNAs resistant genes used gene disruption analyses.                    treatment isolated phenol extraction. resulting <br />                                                                      RNAs labeled 3&apos; ends [32P]-pCp DNA isolation analysis                                           fractionated gel electrophoresis (Fig. 1). RNAs procedure used prepare yeast genomic DNA described         appeared amenable isolation purified rounds Rose et al (31). Southern analyses carried according     electrophoresis. novel snRNAs number species Sambrook et al (SNR32 SNR33; 24) conditions          characterized identified enzymatic <br /> similar Northern hybridization protocol (SNR31).              sequencing. <br /> Radioactive DNA probes prepared nick translation <br /> (24) using New England Biolab Random Priming               Discovery gene sequences DNA database kit. Southern blot EcoRI-digested genomic DNA          Homologous DNA sequences GenBank different organisms (&apos;Zoo-Blot&apos;) obtained         database novel RNA sequences (Figs. 1 Clontech Labs.                                                   2). work completed learned RNA <br />                                                                                        Nucleic Acids Research, 1993, Vol. 21,  23 5393 <br />                                                                                       SMR31 <br />                                                                                                10           20          30             40                   50 <br />  <br />                                                                                       GAATTCAGAA AAAATTACACC ATCAG                          &apos;ATTAACTCA 50 <br />                                                                                       GCCrCI TACTCATAAT G                      TlTTTG                            100 <br />  <br />                                                                                       CACTGCGTC           TOOT AQATTAATTA TTA              C    TATCTCCAG 150 <br />  <br />                                                                                       CQTAT TAGGGG           AGC7IT=          TCA                              200 <br />  <br />                          snR3                                                         CACCTGATAC          T GA            ACArI-iTAAT TTCTTTATGT              250 <br />                          snR5 ._ <br />                        snRI89,                                                        AACCACGTTG TATATTTTCA ATCTGGAAAG TAACCTAAAA GAGTAAAAAC 300 <br />                        snR190 <br />                        (l90nt)                snR32                                   TGTTTATTAA AAAAAATGTT ATTCAAAATT TATACATAGT ACACGTACAG 350 <br />                          snR9    - <br />  <br />                                                                                       CTGCCTGAAG AAAGCATGAG CGAATGACTT ATGCTTCGTC GTCTTCGTCT 400 <br />                                               snR33 <br />  <br />  <br />                                                                                       SNR32 <br />                                       &apos;                                                      10           20          30            40               50 <br />  <br />                                                                                       TTCGTGCTAG CCCTACAACG GAACCCAAAA AATCAATCAC CAGGGTAAGC                      50 <br />  <br />                                                                                       GGGAAAAAAA AAAAGTGTAG GAACCGCACT ATAAAAAGTC GAAGGTTTGT                     100 <br />  <br />                                                                                       ATTAATTTTC CACACCTCTT TAGTTTTGTT ATAATCAAAT TCCTTTTGGT                     150 <br />  <br />                                                                                       TTTTGTTTTA AAAAGGCCAA CATCATAATT ATATCGCGAA TGTACTACTA                     200 <br />  <br />                                                                                       GTATATGCAG    TTTTC&apos;1AAA ATTGCACA ATGACTAAAA GCCCACGTTT                    250 <br />  <br />                                                                                       TTCCCACTTT TAGTAGTACT GAAGCGTAGA       TAGATT AAC GTTGCTGGGC               300 <br />  <br /> Figure 1. Fractionation nuclear small RNAs. Small RNA    prepared         GCCCTGGTGTT   GATCATTTCT GAAATaAGAT ATTGGGAATC        AGCTGTATCT           350 <br /> isolated nuclei pre-extracted 25mM K HEPES pH7.6, 5mM magnesium                  A2CaTrATT     TATACATATT TATTTTTACT GGTTCTTAGT AACGTACTTT                  400 <br /> acetate, 0.25 mM sucrose 0.1 % NP-40. SnRNAs resistant treatment labeled 3&apos; end [32p] pCp fractionated electrophoresis         AGTAGCAATG CTGTGTTTTA GTCGAACGCC TACGTGTCTT GTTAACTTAT                     450 8% polyacrylamide gel. pattern shown snRNAs size range         CATGGTTGCC AACGCGAACG TTTACAACAG GTTCAAAGGC AGGATGAAAA                     500 158-222 nucleotides. bands corresponding snRNA species snR31, snR32 snR33 purified second cycle electrophoresis subjected direct sequence analysis. identity known RNAs marked left   established direct sequencing.                                              SMR33 <br />                                                                                               10            20          30             40               50 <br />  <br />                                                                                      GCTTGCGCAT CTTTTTTTAC TTTATATACT AGTTTTTTTT TTTTTTTTCT                            50 <br /> estimated contain approximately 220 nucleotides corresponds                       TTTTCCCAAA TTTTTTCATG AAAAATTTGG CGGAACGGTA CATAAGAATA                       100 species named snR31 (3 D.Tollervey, pers. <br />                                                                                      GAAGAGATTC ATTATGAAAA TTTTCTGCTC TCTTCCACAT TTTTTTTTCA <br /> commun.). avoid confusion elected use                                                                                                      150 <br />  <br /> identifying RNA, adopt consecutively                            TAAGAATTAA AAAAAATAAC CTTAATCCCC TCTTTCTACC ATGCTCTCAC                       200 <br /> numbered designators RNAs described                                                                                                        250 e., snR32 snR33. nomenclature                                                                                      GGTTCCTTCG ATCAGTGATT TGGAAAG CAAACT                   GGGACTC            300 used new snRNAs study. <br />    RNA coding sequences database                              ,GTCTGTTTTCT&apos;G            TGGAATTGAT ATAGAAuS           GVGGj2Ac         350 <br /> occurs genomic segments sequenced                              GATTTGCAAA ACTTTTTATT TTATGTATTA CTACTGAGCT TGTTTATCTG                       400 <br /> unrelated reasons. DNA corresponding snR31 segment                        TATACTTTTT TCTTATTATT ATGTAGTTAG AGGTTAGTTC TTATCGTTTT                       450 <br /> (GenBank accession number X15731) specifies unnamed <br /> 20 kDa protein believed associated translation factor                 CTTTAAAAGC GACCTTTCTT CGCACAAAGC AATGTTGACA ATCGAGCCAA                       500 <br />  <br /> eIF-4E (37; Figs.2A 3). DNA segment (GenBank <br /> accession number M62392) corresponding snR32 adjacent sequenced genes ribosomal protein L41 (38;                    Figure 2. Sequences gene regions encoding snR31, snR32 snR33. <br />                                                                                 snRNA genes identified searching GenBank DNA sequence <br /> Figs.2B 4). snR31 snR32 coding sequence                         database related sequences. snRNA sequences highlighted italics <br /> located 99 bp 268 bp respectively downstream protein                 underlines. nearest candidate TATA elements marked overhead <br /> genes, tail tail orientation.                                           asterisks (*). 5&apos; 3&apos; ends coding sequences determined    segment encoding snR33 completely                       primer extension T4 RNA ligase/PCR-based sequencing strategy                                                                                  Materials Methods). <br /> sequenced chromosome mII (GenBank accession number X59720), nucleotide positions 141,410 142,838 (39; Figs.2C 5). snR33 sequence occurs uncharacterized <br /> open reading frames located 1200 200 bp                                                                                  Sequence termini new snRNAs <br />  downstream, respectively; ORF elements snR33                       RNA sequences initially deduced  3&apos; end <br />  coding sequence oriented direction. gene                 partial sequences, sizes RNAs (estimated <br />  <span id='am-21' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>glycine</span> tRNA (GenBank accession number K02654) solo                    electrophoretically), sequences DNA database. <br />  delta element occur upstream, approximately 160 310                   actual boundaries coding sequences (Fig.2) later <br />  nucleotides start snR33 coding sequence.                       determined precise mapping 5&apos; 3&apos; ends RNAs. <br />  5394 Nucleic Acids Research, 1993, Vol. 21,  23 <br />  <br />         \ <br />  <br />  <br />  <br />  <br />        K.B <br />                   L.    U J..:.- ......- <br />  <br />  <br />  <br />  <br /> Flgure 3. Coning disruption SNR3J   gene region.  1.3 kb fragment genomic DNA PCR amplified, cloned disrupted URA3 gene. snR31 coding sequence located 99 bp downstream gene 20 kDa <br /> protein believed associated m7G cap binding protein eIF-4E. UR43 gene fragment inserted AseI site midst SNR31 gene. <br /> B: Southern analysis DNA URA+ transformants. DNA digested EcoRI Clal probed 847 bp EcoRl-ClaI fragment radiolabeled nick-translation. Lane 1, DNA homozygous wild type cells; lanes 2-7, <br /> DNA heterozygotes contaning wild type null alleles SANR3. wild type DNA fragment 860 bp disrupted gene fragment 1.9 kb. <br />                                                                                                          .~         *:$j~~~~~&quot;ii  <br />  <br />  <br />  <br />  <br />                                                                                    derived wild type allele. <br />                                                                                                                           1i: <br />                                                                                                                          :;     s:: <br />                                                                                                                                       ..AMM <br />                                                                                                                                               .         - <br />  <br />  <br />                                                                                                                                                   !J w. <br />                                                                                                                                                          _ S//= <br />  <br />  <br />                                                                                                                                                             ~~~~f*&apos;  F:3 s <br />  <br />  <br />  <br />  <br />                                                                                    Fgure 4. Cloning disruption analysis SNR32 gene locus.  snR32 <br />                                                                                    coding segment located 268 bp downstream gene ribosomal protein <br />                                                                                    L4lb (SaC4Jb), opposite orientation. 1.8 kb DNA fragmnent containing <br />                                                                                    genes cloned PCR amplification SNR32 gene disrupted <br />                                                                                    HIS3 gene fragment inserted BclJ site. cloned region extends <br />                                                                                    nucleotide -378 upstream snRNA segment position 1253 bp <br />                                                                                    snR32 coding sequence; site 211 bp upstream SCL41b. <br />                                                                                    structures wild type snrl85::HIS3 alleles shown. B: Southern <br />                                                                                    blot analysis DNA  transformants wild type cells. Genomic <br />                                                                                    DNA digested Hinda probed 525 bp EcoRI-BcI fragment <br />                                                                                    radiolabeled random priming. Lanes 1,3,5 6, DNAs heterozygous <br />                                                                                    isolates; lanes 2,4 7, DNA cells homozygous SNR32 (wild type). <br />                                                                                    allele disrupted HIS3 yields fragment 900bp smaller fragment <br />  snR32 snR33 sequences verified reverse <br /> transcriptase dideoxy sequencing 5&apos; ends established time. 5&apos; end snR31 mapped primer                         AUl snRNAs encoded single copy, non-essential <br /> extension analysis                                                            genes <br />    3&apos; terminal nucleotides identified clever                         availability genomic sequence information approach described Kiss Filipovicz (35), applied                       possible clone gene PCR amplification strategy modifications. original procedure RNA                               (Figs.3 -5). Cloned DNAs used prepare hybridization <br /> oligonucleotide ligated 3&apos; end RNA cDNA                 probes construct null alleles test essentiality synthesis primed DNA oligonucleotide complementary                       genes. gene copy number determined Southern segment. cDNA amplified PCR using                         analysis genomic DNA digested variety restriction <br /> second DNA oligonucleotide corresponding internal RNA                        enZymes. data showed snRNA encoded single <br /> sequence. PCR product subcloned plasmid                     copy gene. <br /> sequenced. determined readily available DNA                              essentiality snRNA genes tested replacing <br /> oligonucleotides used initial ligation RNA                  wild type SNR genes alleles disrupted marker genes. sequence joint RNA-DNA molecule                             URA3 HIS3 simply inserted midst SNR31 successfully amplified PCR. omitted subcloning                       SNR32 genes respectively (Figs.3A, 4A). null allele SNR33 <br /> sequenced PCR products directly gel-purification,                     prepared replacing 3&apos; half snRNA coding 3&apos; ends RNAs determined                            sequence UR43 (Fig.5A). disrupted alleles homogeneous enzymatic sequencing.                                             transformed Ura-  test strains transformants <br />    SnR31, snR32 snR33 determined 5&apos;                             heterozygous SNR loci identified growth <br /> trimethylguanosine cap (TMG), based hybridization probing                       phenotype Southern blot analysis (Figs.3B, 4B, SB). electrophoretically fractionated snRNA prepared                                Tetrad analysis showed genes snR31, snR32 snR33 <br /> precipitation TMG antibodies (data shown).                                dispensable. Northern assays carried cells                                                                                                                          Nucleic Acids Research, 1993, Vol. 21,  23 5395 <br />                                                                                                                                                B.                   C. <br />                           ORI <br />                                                        1.2 ki, <br />                                                                          SNR33                     ORF <br />                                                                                                                 + + - - LRA                      - + - + HIS1          -        -   + URA 3 <br />                             87l1 t,                                              2(X)p        K,       ip <br />  <br />  <br />                                                                      SNR33       ~        ~            N <br />                       l                      liha          E RV Cla N RsiI)KF <br />                                                                                                                       * ..   .....&quot;, <br />                                                                                                                                        &quot;                        -nR32__                4&lt;- snR33 <br />  <br />               Xhl)                                 kuRI <br />                                                      Rl <br />                                                                                                                               :. _         R3S                                         -   5.8s <br />  <br />                                                                            \ <br />                                                                                                                                                                                                                Sal              Illh                 |N            \             si                                                                                            -&gt;71 <br />  <br />                                                                                                                                        1 2 kb                           9011hbp <br />                                                                                                                   1    2 3 4                      1   2   3 4           1   2   3 4 <br />  <br />  <br />        B.                                                                                                       Figure 6. Northern analysis cells containing disrupted SAR genes.Haploid cells <br />                                                                                                                 containing single disrupted alleles snR31, snR32 snR33 analyzed <br />                                                                                                                 Northern blotting (panels C). Signals corresponding test RNAs                                                                                                                 absent case. series includes RNA isolated spores                                                                                                                 single tetrad (lanes 1-4). (-) (+) corresponds spores containing wild <br />                                                                                                                 type alleles alleles disrupted marker genes (URA3 HIS3). 844 bp <br />                                                                                                                 Hindlll-EcoRl fragment  Fig.4) labeled random priming used                              -W.r                                                                               probe snR32. Probes snR31 snR33 identified Figs.3 5. <br />                                                                                                                 positions 5S 5.8S RNAs marked. <br />  <br />  <br />                                                                                                                    impressive relationship sequenced snRNAs                                                                                                                  search snRNA-conserved elements                                                                                                                 negative, recognizable element species. <br />                                                                                                                 Conservation snR32 snR33 sequences                  1                    2            3             4               5                 6        7   examined experimentally Southern analysis. signals                                                                                                                 detected genomic DNAs human, monkey, rat, mouse, <br />                                                                                                                 dog, cow, rabbit chicken (data shown). <br /> Figure 5. Cloning disruption analysis SNR33 gene region.  1.4                                       expected, segments complementarity exist kb DNA fragment containing snR33 gene cloned PCR amplification                                       snRNAs pre-rRNA, large size 115 bp EcoRV-ClaI gene replaced URA3 marker <br /> gene. structures wild type disrupted alleles (snr33::URA3)                                   rDNA coding unit. relevant shown. cloned region extends nucleotides 141,410 142,838                                         largest regions described highest <br /> chromosome Em. B: DNA URA+ transformants screened Southern                                          potential direct binding; course, important interactions <br /> blot analysis, probing NsIl/SalI digests 500 bp NslI-HhaI fragment                                  occur pairing short regions. longest <br /> radiolabeled random priming. Lanes 1 3, DNA homozygous wild <br /> type strain plasmid containing intact SNR33 (1kb fragment); lane 2, plasmid                                 complementary regions rRNA  1) 12 base segment disrupted allele (0.9 kb 1.2 kb fragments); lanes 4,6, 7, genomic                                5&apos; external transcribed sequence, nucleotides <br /> DNA transformants heterozygous wild type disrupted alleles;                                    567-578 5&apos; end pre-rRNA (snR32); 2) 10 base <br /> lane 5, DNA heterozygote containing disrupted allele URA3                                   segment internal transcribed sequence 1, nucleotides <br /> gene inserted opposite orientation.                                                                      2447-2456 (snR33);  3) 19 base segment region                                                                                                                 includes enhancer element termination signal T2, individual spores confirmed absence test RNAs                                                     nucleotides 6843-6861 (snR31; 40). haploid cells containing disrupted allele (Fig.6).                                                       snRNA-related segments imagined role rRNA <br /> phenotypic difference evident haploid cells 16&#194;&#176;C,                                                synthesis, clear complementarities 25&#194;&#176;C, 32&#194;&#176;C 38&#194;&#176;C. Cells lacking snR33 exhibited similar                                                       statistically significant. similar search using spliceosomal <br /> growth rates media containing 2% glucose, 2% galactose,                                                      snRNA sequences controls yielded complementarities upwards <br /> 2% glycerol 2% potassium acetate; effect carbon source                                                10 contiguous bases. test control tested cells lacking snRNAs. fact                                                 complementarities involve stretches adenines uridines RNAs dispensable precludes possibility                                                  high content yeast genome argues using conditional mutants study function                                                 relationships important statistically. essential snRNAs. <br /> Relatedness snR31, snR32 snR33 RNAs                                                             DISCUSSION searches carried RNA sequence                                                       genes identified bring number S. cerevisiae determine:  snRNAs evolutionarily conserved; ii)                                                snRNA genes analyzed <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> 19, corresponding 5 sequence motifs conserved snRNAs present,                                                  spliceosomal 14 non-spliceosomal RNAs (reviewed 3,4). box D elements nucleolar snRNAs;                                                        novel snRNAs described extraction properties like <br /> iii) new RNAs contain segments complementarity                                                   known nucleolar snRNAs (7,12, data shown), <br /> rRNA suggest roles rRNA synthesis.                                                                 premature conclude RNAs actually <br />  5396 Nucleic Acids Research, 1993, Vol. 21,  23 <br /> nucleolar. Resistance extraction modest salt concentration        nucleolar  number dispensable small RNAs believed property nucleolar snRNAs (12),        associated complex increases 11, corresponding behavior apply RNAs associated stable          80% known nucleolar small RNAs. <br /> complexes. Consistent observed precursor           number non-essential species expands tRNAs resistant extraction low salt  Balakin         increasingly unlikely RNAs simply redundant, M.Fournier, prep.). snRNAs identified          distinguishing function. plausible possibility nucleolar, direct evidence needed        RNAs provide overlapping unique advantages considered established. criteria include:                essential. Dispensable snRNAs  example, <br /> localization nucleolus hybridization probes,             influence kinetics accuracy rRNA processing rate <br /> purification cross-linking precursor rRNA,                  assembly ribosomal RNP intermediates little association known nucleolar proteins.                              effect growth rate. Unless ribosome content limiting <br />    new RNAs lack box C box D elements characteristic         growth phenotypes observed loss non- nucleolar RNAs yeast sources (41 -44),        essential snRNAs. contributions RNAs does argue nucleolar location. number          apparent quantitative biochemical assays discovery snRNAs exhibit nucleolar localization properties,          genetic conditions deleterious effects snRNA loss <br /> lack box C D. include yeast snRNA species snR3,           amplified. <br /> -4, -5, -8, -9 -10 human HeLa snRNAs U17 (El), E2, E3 (12,34,45). yeast snR31, snR32 snR33 species             ACKNOWLEDGEMENTS belong group snRNAs. <br />     snRNAs participating processing mRNA rRNA          thank Richard Lempicki providing blot snRNAs believed function complementary interactions        immuno-precipitated trimethylguanosine antibodies, Molly <br />  substrate molecules (4,46). yeast nucleolar         Fitzgerald-Hayes yeast strain MH2 David Tollervey  snRNAs U3 U14, example, contain rRNA-related                   sharing snR31 gene sequence data prior publication. work <br />  sequences implicated rRNA processing                supported NIH grant GM-19351. <br /> experimentally. Yeast U3 cross-linked rRNA complementary sites (47) mutational analysis shown         REFERENCES <br />  U14 activity depends segments complementary               1. Riedel, N., Wise, J. , Swerdlow, H., Mak,  Guthrie, C. (1986) <br />  18S rRNA (48). point link snRNA            Proc. Natl. Acad. Sci. USA 83, 8097-8101. <br />  species described rRNA synthesis tenuous                2. Thompson, J.R., Zagorski, J., Woolford, J.L. Fournier, M.J. (1988) <br />  nucleolar localization data complementarity               Nucleic Acids Res. 16, 5587-5601. <br />  regions ribosomal DNA. remains seen           3. Mattaj, W., Tollervey, D. Seraphin, B. (1993) FASEB J. 7, 47-53. <br />  new snRNAs role ribosome biogenesis.                         4. Fournier, M.J. Maxwell, E.S. (1993) Trends Biochem. Sci. 18, 131-135. <br />                                                                          5. Sollher-Webb, B., Tyc, K. Steitz, J. (1993) Ribosomal RNA: Stnuture, <br />     finding novel snRNAs derived           Ewvuoion, Gene Etpression Function Protein Synthesis (Zimmermann, <br />  single gene surprising. S. cerevisiae         R  Dahlberg, E., eds), CRC Press  press). <br />  snRNAs characterized far encoded single-copy DNA            6. Hughes, J.M.X., Konings, D M. Cesareni, G. (1987) EMBO J. 6, <br />  (49). exception U3 specified genes,            2145-2155. <br />  containing introns (6,50). snRNA genes                    7. Zagorski, J., Tollervey, D. Foumier, M.J. (1988) Mol. Cell. Biol, 8, <br />                                                                             3282-3290. <br />  S.cerevisiae known contain introns, including          8. Bally, M., Hughes, J. Cesareni, G. (1988) Nucleic Acids Res. 16, <br />  described present study.                                            5291 -5303. <br />     proximity new snRNA genes protein genes              9. Hughes, J.M.X. Ares, Jr., M. (1991) EMBO J. 10, 4231-4239. <br />  ORF elements locus raises possibility          10. Li, H.V., Zagorski, J. Fournier, M. (1990) Mol. Cell. Biol. 10, <br />                                                                             1145-1152. <br />  functional relationship RNA protein genes.         11. Morrissey, J.P. Tollervey, D. (1993) Mol. Cell. Biol. 13, 2469-2477. <br />  involvement snRNAs ribosome biogenesis makes linkage         12. Tollervey, D. (1987) EMBO J. 6, 4169-4175. <br />  snRNA genes genes putative mRNA cap              13. Tollervey, D., Lehtonen, H., Carmo-Fonseca, M. Hurt, E.C. (1991) <br />  binding protein (snR3 1; 37) ribosomal protein (snR32; 38)             EMBO J. 10, 573-583. <br />  especially intriguing. similar situation exists yeast   14. Girard, J.-P., Lehtonen, H., Caizergues-Ferrer, M., Amalric, F., Tollervey, <br />                                                                             D. Lapeyre, B. (1992) EMBO J. 11, 673-682. <br />  snRNA species snR189 located 300 bp gene           15. Russell, D. Tollervey, D. (1992) J. Cell Biol. 119, 737-747. <br />  ribosomal protein rp59 (CRYJ; 2). SNR31 SNR32 genes            16. Lee, W-C., Zabetakis, D. Melese, T. (1992) Mol. Cell. Biol. 12, <br />  tail tail arrangement protein genes,          3865-3871. <br />  SNR189 CRYI tail head arrangement.                 17. Kondo, K., Kowalski, L.R. Inouye, M. (1992) J. Biol. Chem. 267, <br />  closeness genes suggests possible regulatory link                16259-16265. <br />                                                                         18. Schimnang, T., Tollervey, D., Kern, H., Frank, R., Hurt, E.C. (1989) <br />  ribosome biogenesis translation. notion                EMBO J. 8, 4015-4024. <br />  strengthened recent discoveries showing small          19. Clark, M.W., Yip, M.L., Campbell, J. Abelson, J. (1990) J. Cell Biol. <br />  nucleolar RNAs higher eukaryotes encoded introns             111, 1745-1751. <br />  ribosomal protein genes genes heat shock protein          20. Jones, E.W. (1977) Genetics 85, 23-33. <br />                                                                          <br /> </body></html>