<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>     1995 Oxford University Press                                          Nucleic Acids Research, 1995, Vol. 23,  17 3493-3500 <br />  <br />  <br /> HYS2, essential gene required DNA replication Saccharomyces cerevisiae <br /> Katsunori Sugimoto*, Yoshitaka Sakamoto, Osamu Takahashi Kunihiro <br /> Matsumoto <br /> Department Molecular Biology, Faculty Science, Nagoya University, Chikusa-ku, Nagoya 464-01, Japan <br /> Received 12, 1995; Revised Accepted August 8, 1995                                                    DDBJ accession  D50324 <br />  <br />  <br />  <br /> ABSTRACT                                                                mechanism prevents mitosis completion DNA <br />                                                                         replication established (reviewed refs 1,2,7,8). investigate cell cycle regulation S G2 phase                  DNA replication fundamental maintenance growth Saccharomyces cerevisiae, isolated mu-                       eukaryotic cells. Saccharomyces cerevisiae provides tants displaying supersensitivity hydroxyurea (HU),                  excellent model identifying components <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> inhibits DNA replication. mu-                      eukaryotic DNA replication machinery distinctive <br /> tants, named hydroxyurea sensitive                        growth pattern budding yeast allows cell-division-cycle <br /> (hys), defined linkage groups character-                    (Cdc) phenotypes easily distinguished defects ized hys2 mutation study. hys2-1 mutant                 cellular processes (9). example, studies S.cerevisiae cdc <br /> displays temperature sensitive growth constella-                  mutants resulted identification DNA ligase (CDC9) <br /> tion phenotypes indicating defective DNA metab-                      (10,11), DNA polymerase (CDCJ7) (12,13) DNA <br /> olism. restrictive temperature, hys2-1 cells                     polymerase 6 (CDC2) (14,15).  determination arrest large budded cells single nucleus               nature defect cdc mutants led better <br /> neck bud short spindle. hys2-1 mutant                  understanding DNA replication mitosis. <br /> exhibits increased rates chromosome loss                            Enoch et al. (16) developed screen fission yeast <br /> recombination. Additionally, hys2-1 appears accu-                    mutants defective coupling mitosis completion DNA <br /> mulate incompletely replicated DNA                          replication. mutants isolated hydroxyurea <br /> detected pulse field electrophoresis assay. Finally,               (HU) sensitive mutants initiate mitosis DNA repli- <br /> deletion RAD9 hys2-1 strain decreases                       cation blocked HU. different genes (radi, rad3, radl 7, <br /> percentage arrested cells, suggesting intact                 husi, hus2, hus3, hus4 hus5) identified, RAD9-checkpoint required cell cycle arrest                required arrest mitosis response inhibition DNA <br /> hys2-1 cells. HYS2 encodes 55 kDa protein                     replication DNA damage. searched mutants essential viability temperatures. Taken                   supersensitive HU S.cerevisiae identify genes playing <br /> gether, data suggest Hys2 plays role                    role DNA replication itself checkpoint responds <br /> DNA replication.                                                        inhibition DNA replication. isolated different <br />                                                                         hys mutations (hysi, hys2, hys3 hys4). <br /> INTRODUCTION                                                               report characterization HYS2. hys2 mutant cell division cycle eukaryotic cells requires high degree   HU sensitive simultaneously confers temperature sensitive <br /> coordinated control. ability order pathways     growth. hys2 mutation causes cells arrest single <br /> govern carry steps cell division, including DNA              large bud. HYS2 gene essential growth encodes <br /> replication mitosis, fundamental importance       55 kDa protein. hys2 mutant exhibits increased levels eukaryotes. Incomplete DNA replication DNA damage prevents           mitotic chromosome loss recombination, suggesting subsequent mitosis (1,2). fission yeast, dependency      Hys2 plays role DNA replication. <br /> checkpoint control shown involve p34Cdc2 kinase. signal generated active replication ultimately impinges       MATERIALS METHODS phosphorylation state Tyrl5 p34Cdc2 kinase. signal       Strains, media general methods <br /> transduction pathway required prevent mitosis events S phase G2 perturbed (3,4). budding yeast, tyrosine          Saccharomyces cerevisiae strains used study listed phosphorylation p34Cdc28 kinase required arrest mitosis    Table 1. Standard genetic techniques yeast culture media circumstances (5,6).          described previously (17,18). Media used maintain way arresting cell cycle tyrosine phospho-       selection TRPI URA3 plasmids synthetic complete <br /> rylation p34Cdc28. number genes involved               media containing 0.5% casamino acid appropriate <br /> checkpoint control yeasts identified,    supplements. <br />  <br />  * <br />      correspondence addressed <br />   3494 Nucleic Acids Research, 1995, Vol. 23,  17 <br />  <br /> Table 1. List strains used study                              gene. sporulation tetrad dissection ofdiploid, viability <br />                                                                          spores assessed. half spores viable,  Strain           Genotype                                               expected, HU temperatr sensitive,  KSC688          MATa ade2 ural leu2 lys2                                Leu-. <br />  KSH106          MATa adel his2 trpl ura3 leu2 <br />                                                                          Plasmid construction <br />  KSH141          MATa ade2 his3 trpl ura3 <br />  KSH163          MATa adel his3 trpl ura3 leu2                           Sall-HindIl fragment p12 subcloned YCplac22 <br />                                                                          (22) YCplac33 (22) creating YCp22-HYS2  KSH171          MATa adel his2 trpl ura3 leu2                           YCp33-HYS2, respectively. fragment cloned  KSH172          Isogenic KSH106 MATa/MATa                 pSP72 (Promega) cleaved Sall HindHI creating <br />  KSH387          MATa hys3-1 adel his2 trpi ura3 leu2                    pSP-63HS. <span id='am-17' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> sequence Sail HindIH <br />  KSH389          MATa hys4-1 adel his2 trpl ura3 leu2 <br />                                                                          fragment determined strands. construction                                                                          YCpG-HYS2, 1.5 kb fragment containing HYS2 coding <br />  KSH404          MATa hysl-J adel his3 trpl ura3 leu2                    sequence obtained polymerase chain reaction using  KSH541          MATa hys2-1 adel his2 trpl ura3 leu2                    primer GCTCTAGAGGCAATGGACGCATTGT, corresponding <br />  KSH542          MATa hys2-1 adel his2 his3 trpl ura3 leu2 <br />                                                                          nucleotide residues -4 +13 HYS2 gene primer <br />                                                                          GCGTCGACTATTTAAATGTCAATCT, corresponding nu- <br />  KSH543          MATa ade2 his3 trpl ura3                                cleotide residues +1451 +1567 HYS2 gene (DDBJ <br />                  MATa adel his2 trpl ura3 leu2                           accession  D50324). XbaI Sall sites included  KSH544          MA Ta hvs2-1 ade2 his3 trpl ura3                        oligonucleotides underlined. fragment cleaved                  MATa hys2-1 adel his2 trpi ura3 leu2                    XbaI Sall cloned YCpG33 (17), YCplac33 <br />  KSH687          MATa hys2-1 rad9A::LEU2 adel his2 his3 trpl ura3 leu2 <br />                                                                          derivative containing GALl promoter, creating YCpG-HYS2. <br />                                                                          plasmid YCpG-HYS2 complemented disruption HYS2, <br />  KSH689          MATa hys2-1 ade2 his3 trpl ura3                         cells grown YEPD medium. <br />  KSH700          Isogenic KSH172 HYS2/hys2A::LEU2 <br />                                                                          Disruption HYS2 <br />   DNA manipulated standard procedures (19). Sequence              construction hys2A::LEU2, plasmid pDhu2L analysis performed dideoxy chain termination tech-            created replacing internal BamHI fragment HYS2 nique using Sequenase (USB) according manufacturer&apos;s              LEU2 gene. XbaI-PvuII fragment containing <br /> instruction. Yeast tranformation performed alkali             hys2A::LEU2 integrated diploid strain KSH172, <br /> cation method (20). step gene replacement described           creating strain KSH700. sporulation, tetrad analysis Rothstein (21).                                                       used determine phenotype caused deletion. <br />  <br /> Mutant isolation                                                         UV radiation DNA damaging agent sensitivities wild type strain (KSH106) mutagenized 2% ethyl                UV radiation 254 nm delivered use Stratagene <br /> methanesulfonate (EMS) 10% survival. Mutagenized cells           Stratalinker. Cells grown exponentially 30&#194;&#176;C plated grown single colonies YEPD medium 30&#194;&#176;C. Colonies                YEPD irradiated UV according manufacture&apos;s replica plated YEPD YEPD containing 10 mg/ml HU.             manual. incubation 30&#194;&#176;C 3 days, number 3 4 day incubation 30&#194;&#176;C, 19 colonies grew poorly        colonies counted. presence HU. mutants accumulated                  Methyl methanesulfonate (MMS) sensitivity assay  <br /> &gt;80% large budded cells 4 h shift HU containing           scribed previously (23). Cells grown exponentially YEPD <br /> medium.                                                                  medium 30&#194;&#176;C incubated 0-0.8% MMS 40 <br />                                                                          min. incubation terminated adding sodium thiosulfate <br /> Cloning HYS2 gene                                                 final concentration 5%. Aliquots plated YEPD, <br />                                                                          followed incubation 30&#194;&#176;C. 3 days incubation, Strain KSH542 (hys2-1 ura3) transformed genomic               colony number counted. <br /> library constructed YCp5O 30&#194;&#176;C replica plated YEPD medium containing 10 mg/ml HU. single plasmid (p12)               Viability assays identified complement HU temperature <br /> sensitive growth hys2-1 mutation.                                 method Weinert Hartwell (24) used determine <br />   establish complementing plasmid p12 carries            hys2-1 rad9A::LEU2 mutant (KSH687) loses <br /> HYS2 gene, crossed hys2-1 mutant hys2A. :LEU2            viability incubated restrictive temperature. Cells    strain carrying plasmid p12. plasmid p12             grown YEPD medium 250Cto mid log phase cured resulting diploid. diploid heterozy-      shifted restrictive temperature (37&#194;&#176;C). Aliquots gous hys2-1 mutation, disruption wild type gene         culture removed intervals, sonicated plated YEPD unmask HU sensitive temperature sensitive                 plates. incubation 25&#194;&#176;C 2 days, cell viability phenotypes cloned fragment exclusively HYS2 coding             determined microscopy; individual microcolonies plate <br /> sequence. diploid HU temperature sensitive, confrm-          scored inviable (&lt;16 cell bodies) viable (&gt;16 cell <br /> ing mutation resides gene disrupted LEU2      bodies). <br />                                                                                   Nucleic Acids Research, 1995, Vol. 23,  17 3495 <br />  <br /> Immunofluorescence microscopic analysis                                blot analysis sensitive probes derived large <br />                                                                        chromosomes used (28). <br /> Cells processed fluorescence indirect immunofluores- <br /> cence microscopy described previously (25). Cells fixed        Cell synchronization RNA analysis stained DNA 25 jg/ml 4,6-diamidino-2-phenylindole <br /> (DAPI). Microtubule structures observed following             Synchronization factor mating pheromone Northern <br /> maldehyde fixation using anti-tubulin monoclonal antibody          blot analysis performed described previously (29). DNA <br /> TAT-1 FITC-conjugated goat anti-mouse antibody                probes  CLNJ2, 0.9 kb XhoI-HindIII fragment described (26).                                                        CLN2 gene (30); ACT], 1 kb XhoI-HindHI fragment derived <br />                                                                        pYS91 containing ACT1 cDNA (unpublished); HYS2, <br />                                                                        1 kb BamHI fragment YCp33-HYS2. <br /> DNA flow cytometry analysis <br /> Flow cytometry DNA quantitation determined described            RESULTS <br /> previously (26). Cells (1 x 106-1 x 107) collected, washed 1 ml water resuspended 0.3 ml 0.2 M Tris-HCl    Isolation hys mutants <br /> (pH 7.5). Ethanol added final concentration 70%    culture wild type cells presence HU (10 mg/ml) <br /> vigorous agitation cells stored -20&#194;&#176;C overnight.     temporarily accumulates cells large buds eventually <br /> resuspension 0.3 ml 0.2 M Tris-HCl (pH 7.5), cells          recovers HU arrest. mutagenized wild type cells <br /> sonicated briefly preboiled RNase (Sigma) added         (strain KSH106) ethyl methanesulfonate (EMS) screened <br /> final concentration 1 mg/ml. Following 3 h incubation 37&#194;&#176;C,    colonies sensitive HU replica-plating cells propidium iodine (Sigma) added (final concentration, 5 jig/ml)     plates HU identifying clones failed form resulting stained cell suspensions analyzed using       colonies HU. HU blocks DNA synthesis, mutations <br /> Becton-Dickinson FACScan.                                              defective DNA metabolism expected suffer severe <br />                                                                        growth retardation presence HU. incubated <br /> Analysis chromosome IH missegregation                               HU, mutations deficient ability monitor impaired <br />                                                                        DNA synthesis allow cells enter mitosis Quantitative measurement chromosome III loss recom-             defective chromosomes, resulting lethality. -10 000 EMS- <br /> bination performed essentially previously described (27).      mutagenized cells screened, 19 clones showed HU sensitive <br /> Strains tested allowed grow 30&#194;&#176;C YEPD              phenotype. HU sensitive mutants, medium. Mating initiated mixing -1 x 106 cells            accumulated 80% large budded cells 4 h shift HU <br /> tested 1 x 106 MATa haploid tester (KSC688) final            media identified microscopic observation. crosses volume 200 gl. 4 h 30&#194;&#176;C mating mixture plated      wild type strain, mutant recessive HU sensitivity medium selecting prototrophic triploids. Wild type MATat        segregated single gene mutation. Complementation analysis <br /> haploid KSH171 used control determine mating               showed mutants formed different complementation groups, <br /> efficiency. ensure rate MATa homozygosis             calling hysl-hys4 (Table 2). paper, present <br /> chromosome Ill missegregation underestimated          data characterizing hys2 mutation. <br /> mating defect associated hys2-1, hys2-1 haploid strain             HU-teated cultures hys2-1 mutant subjected KSH541 tested. revealed mating deficiency             cytological examination. Cells stained DNA specific <br /> associated hys2-1 30&#194;&#176;C. combined rate MATa              fluorescence dye DAPI analysis cell nuclear morphol- <br /> homozygosis plus chromosome III loss determined           ogy. 90% large budded hys2-1 cells, like wild type <br /> total number cells mated. rate chromosome mI loss       cells, contained single nucleus near neck short calculated subtracting rate mitotic recombination        spindle (data shown). addition HU sensitive phenotype, <br /> determined analysis Leu+ maters combined rate.          hys2-1 mutation cells conferred sensitivity MMS assay performed twice using independent colonies.              temperature sensitive growth defect (Table 2). test hys2 <br />                                                                        growth defect 37&#194;&#176;C HU sensitivity caused single <br />                                                                        mutation, analyzed meiotic products diploid strain <br /> Pulse field gel analysis                                               heterozygous hys2-1. Temperature sensitive growth phenotype <br />                                                                        cosegregated HU sensitive phenotype tetrads, <br /> Cultures grown early log phase 25&#194;&#176;C                indicating hys2-1 mutation determines phenotypes. <br /> incubated 37&#194;&#176;C 4 h. control samples, cultures hys2-1 <br /> cells incubated factor (10 ,ug/ml) 2.5 h, HU (10       hys2-1 mutation defective DNA replication <br /> mg/ml) 4 h nocodazole (20 jg/ml) 4 h 25&#194;&#176;C. Yeast <br /> chromosomal DNA samples prepared previously                 examine tenninal morphology strains carrying scribed (28). Pulse field gel electrophoresis carried 1 %   hys2-1 mutation, growth-arrested populations analyzed <br /> agarose (LE, FMC) pulsaphor electrophoresis unit (Pharma-         microscopically (Fig. 1). exponentially growing cultures cia). Electrophoresis performed 24 h switching          hys2-1 mutant HYS2+ cells grown 25&#194;&#176;C shifted time of70 s 170 V 0.5 x Tris-borate-EDTA (TBE). DNA          37&#194;&#176;C 4 h, 80% hys2-1 cells arrested large buds, transferred nylon membranes probed                 compared 19% HYS2+.  hys2-1 exhibits 32P-labeled TRPJ fragment  1.4 kb EcoRI YRp7                  marked elevation proportion large budded cells plasmid). probe chromosome IV chosen          restrictive temperature. <br />   3496 Nucleic Acids Research, 1995, Vol. 23,  17 <br />  <br />              Table 2. Properties hys mutants <br />  <br />               Genotyp<span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;&apos;              Growthb                                                MMS sensitivityc    UV sensitivityc <br />                                      300C              370C                 10 mg/ml HU     0.4%                90 J/m2                120 J/m2 <br />               HYS                    +                 +                   +                16%                 30%                   7% <br />               hysl                   +                 +                   -               0.09%                0.2%                  NT <br />               hys2                   +                 -                   -               0.06%                17%                   5% <br />               hys3                   +                 +                   -               NT                   NT                    NT <br />               hys4                   +                 +                   -               NT                   NT                    NT <br />              aStrains used KSH106 (HYS), KSH404 (hysl), KSH542 (hys2), KSH387 (hys3) KSH389 (hys4). <br />             bGrowth cells 30 37&#194;&#176;C tested streaking cells YEPD plates incubating corresponding temperature 2 days. <br />             Cells tested growth YEPD plates containing 10 mg/ml HU 300C 4 days. +, growth; -, growth. <br />             CDNA damage sensitivities determined treating log-phase cultures cells 0.4% MMS 40 min irradiating 90             120 J/m2 UV light (254 nm) percent survival measured 3 days incubation 30&#194;&#176;C. NT, tested. <br />  <br />                                                                                      Mutations genes involved DNA replication,                                                                  0               encoding DNA polymerase (CDCJ 7), DNA polymer- <br />                      0                                                             ase 6 (CDC2) DNA ligase (CDC9), exhibit significantly <br />                                                                                    elevated frequencies genetic recombination chromo- <br />                      IC,)  ...1. <br />                   -,IV             .:~                                             loss (32). feature distinguishes mitotic <br />                   fd3 <br />                    (&apos;)4-&quot;                                   .7 <br />                                                                                    defects elevated chromosome loss. assayed                                                                                    recombination chromosome loss frequencies using standard <br />                                                                                    heterozygous markers chromosome III  Materials                                                                                    Methods) (Table 3). Diploids homozygous hys2-1 <br />                                                                                    mutation grown permissive temperature. Frequencies <br />                                                                                    recombination chromosome loss normalized                                                                                    HYS2+1HYS2+ strain. hys2-J/hys2-1 strain exhibited signifi- <br />                                                                                    increases frequency recombination (16-fold <br />                                                                                    increase) chromosome loss (20-fold increase)                                                                                    permissive temperature (Table 3). <br />  <br />                                                                                    Table 3. Rates recombination chromosome III missegregation b                                                                                  hys2-1 mutanta <br />        DAPI <br />                                                                                     Genotype           Recombination rate         Chromosome missegregation rate <br />       TubLI lii                                                                     HYS2/HYS2          8.6 x 10-5 (1)             9.8 x 10-5 (1) <br />                                                                                     hys2-1/hys2-1      1.4 x 10-3 (16)            2.0 x 10-3 (20) <br />                                                                                    aThe rates mating competent diploids arose mitotic recombination <br /> Figure 1. Cell cycle arrest phenotype conferred hys2-1 mutation.          versus chromosome III missegregation HYS2/HYS2 (KSH543) DAPI stained hys2-1 cells. logarithmically growing culture KSH542             hys2-1/hys2-1 (KSH544) diploids analyzed 30&#194;&#176;C. Mating competent di- <br /> (hys2-1) shifted 25 37&#194;&#176;C 4 h. Cells fixed ethanol     ploids scored protrophic triploids incubation leu2 mating <br /> examined phase contrast microscopy (upper panel) DAPI staining (lower       tester (KSC688). fraction diploids mated MATa ho- <br /> panel). (b) Nuclear microtubule structure hys2-1 cells. Cells KSH542 <br /> (hys2-1) grown logarytmiically 250C shifted 370C 4 h.      mozygosis mitotic recombination distinguished Samples fixed <span id='am-25' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> stained DAPI (upper panel)         missegregated chromosome III, replica plating plates lacking leucine. Rates <br /> anti-tubulin antibodies (lower panel) visualize nuclei spindles.            calculated numbers described Materials Methods. <br />                                                                                    Numbers parentheses normalized wild type (HYS2/HYS2)  <br />                                                                                    ground. <br />  <br />   4 h incubation 37&#194;&#176;C, 83% large budded cells                     determine extent DNA replication hys2-1 hys2-1 culture single undivided nucleus                mutant cells undergo arrest, examined DNA <br /> neck mother daughter cells (Fig. la). Following                    content hys2-1 cells restrictive temperature. shift 370C, samples hys2-1 cells prepared                         incubated exponentially growing hys2-1 HYS2+ strains anti-tubulin immunofluorescence microscopy (Fig. lb).                          permissive (25&#194;&#176;C) restrictive temperature (370C), stained hys2-1 cells short mitotic spindle, consisting brightly                 cells propidium iodide determined DNA content stained bar nuclear microtubules faintly staining                     cells FACS analysis. high percentage hys2-1 cells <br /> cytoplasmic microtubules.  phenotype hys2-1 mu-                        arrest 2N content DNA 37&#194;&#176;C, indicating tants restrictive temperature similar observed                majority DNA synthesis completed restrictive <br /> cdc mutants defects DNA replication mitosis (31).                       temperature (Fig. 2). informative result FACS <br />                                                                                                 Nucleic Acids Research, 1995, Vol. 23,  17 3497 <br />  <br />  <br />                                                                              hys2                         hys2             rad9 <br />                                                              4hr <br />                                                                                                                                      <br />  <br />  <br />  <br />                                                     rA Pr-L <br />                                                           - , ,M <br />                                                               ,      , <br />                                                     600     800 <br />  <br />  <br />  <br />  <br />                           0 <br />  <br />  <br /> Figure 2. DNA flow cytometry analysis hys2-1 hys2-1 rad9 mutant cells. Strains KSH106 (HYS2), KSH542 (hys2-1) KSH687 (hys2-1 rad9A::LEU2) grown  early log phase YEPD 25&#194;&#176;C. shifting 37&#194;&#176;C, samples cells taken 2 h intervals fixed, stained propidium iodine examined flow cytometry. Control experiment hys2-1 cells treating 20 tg/ml nocodazole 25&#194;&#176;C 4 h subsequently shifting 37&#194;&#176;C 4 h. experiment confirned hys2-1 cells arrest 2N content DNA 370C. <br />  <br />  <br />  <br /> analysis hys2-1 mutation does disrupt bulk DNA replication restrictive temperature. hys2-1 <br /> temperature sensitive mutation leaky leaky DNA                                                     00 <br /> synthesis mutants (e.g. cdc17) (33) arrest G2/M <br /> content DNA. noted culture hys2-1 mutant <br />                                                                                                        e m C5 X accumulated cells late S G2/M phase DNA content 25&#194;&#176;C. hys2 mutant appears defect S-G2/M phase                                          __         _j&amp;Om      -   execution permissive temperature.                                                                                          chromosome IV <br />  <br />    bulk DNA replicated hys2-1 mutant, small amounts unreplicated DNA sufflcient cause arrest cell cycle. determine DNA completely replicated hys2-1 cells, chromosomes                           Figure 3. Chromosome abnormalities hys2-1 mutant. Exponentially <br /> arrested cells analyzed pulse field gel electrophoresis                     growing cultures hys2-1 (KSH542) HYS2 (KSH106) cells 25&#194;&#176;C (Fig. 3). assay fully replicated DNA enters gel;                   shifted 37&#194;&#176;C incubated 4 h YEPD. controls, hys2-1 cells incompletely replicated DNA (e.g. replication loops single-                      incubated 25&#194;&#176;C YEPD containing 10 jg/ml factor 2.5 h, 10 mg/ml <br />                                                                                     HU 4 h 20 jg/ml nocodazole 4 h. chromosomes sample <br /> stranded regions) causes DNA retarded wells (28). DNA                      separated pulse field gel electrophoresis, blotted probed prepared hys2-1 strain incubated                  DNA fragment ofthe TRPI gene resides chromosome IV. row <br /> restrictive temperature 4 h. controls, hys2-1 strain                 bands indicates samples loaded   contains residual <br /> treated factor mating pheromone (blocked GI phase),                       material able migrate gel. <br /> HU (blocked S phase) nocodazole (blocked M phase) permissive temperature DNA isolation. shown <br /> previously (28), DNA isolated HU-treated culture                           likely explanation phenotype remained wells electrophoresis,                    hys2-1 mutant defective DNA replication, majority DNA mating pheromone nocodazole                            somewhat leaky, DNA replication, complete, <br /> treated cultures migrated gel. Compared DNA                          result accumulation DNA level approaching 2N. <br /> isolated wild type cells, DNA prepared hys2-1 cells                    hys2-1 mutants undergo cell cycle arrest DNA restrictive temperature entered gel greatly reduced                lesions, arrest mediated product RAD9 <br /> efficiency (Fig. 3).  appears DNA incom-                      gene (24,34). test role RAD9 checkpoint cell cycle <br /> pletely replicated hys2-1 cells restrictive temperature.                 arrest hys2-1 mutant, constructed hys2-1 rad9&amp;:^LEU2 <br />   3498 Nucleic Acids Research, 1995, Vol. 23,  17 <br />  <br />                                         b                                         sponding HYS2 gene. sequence predicts protein 487 <br />                                                                                     amino acids, assuming tmnslation begins ATG.                                                                                     search sequence banks failed idenfify                                                                                    known proteins significant sequence similarity Hys2.                                                                                    nucleotide sequence 3&apos;-region HYS2 gene identical <br />                                                                                    upstream sequence SUI2 gene (36), indicating                                                                                    genes reside adjacent  finding consistent <br />                                                                                    result chromosom blotting ofHYS2, SUI2 <br />                                                                                    gene located near centromere chromosome X. genes <br />                                            &quot;-4                                     genetically mapped close SUI2, correspond <br />  ,- <br />                                         a2                                      HYS2 gene. <br />                                           u <br />                                                                                       determine HYS2 essential viability S.cerevisiae,                                                                                    created disruption HYS2 open reading frame. LEU2 <br />                                                                                    fragment replaced 1 kb BamHI fiagment inmtemal                                                                                    HYS2 coding region XbaI-PvuH fragment containing                                                                                    LEU2 insert used transform diploid strain KSH172. <br />                   2        4                                                       Southern blot analysis Leut transfomiants showed                                                             2         4 <br />                   Time (hr)                                 Time (hr)              copies ofthe HYS2 gene disrupted. diploid <br />                                                                                    (KSH700) sporulated, tetrads segregated 2:2 viable:non-vi- <br />                                                                                    able. viable spores Leun, indicating  Figure 4. Viability hys2-1 rad9 double mutant restrictive           wild type copy HYS2 gene. result consistent  temperature. Strains KSH542 (hys2-1), KSH687 (hys2-J rad9As:LEU2)           HYS2 gene product essential growth. Microscopic <br />  wild type strain KSH106 grown exponentially 25&#194;&#176;C shifted         examiaton inviable spores indicated genninated <br />  37&#194;&#176;C. Cell viability determined shifting described Materials <br />  Methods percent viability intervals shown  . Samples     form microcolonies ftree 11 cell bodies.  analyzed cell number (b). Symbols: hys2-1(0), hys2-1 rad9  LEU2        HYS21hys2&amp;:LEU2 diploid (KSH700) transformed  (U), wild type (0).                                                               plasmid YCp33-HYS2 bearing HYS2 URA3 genes                                                                                    sporulation, tetrad analysis caried ascus,                                                                                    spores viable Leu+ spore clone canying                                                                                    hys2&amp;:LEU2 mutation Urat. cultring Leut <br /> double mutant strain. Cultures growig exponentially 25&#194;&#176;C                        Ura+ spore clones generations non-selective YEPD shifted 37&#194;&#176;C cell cycle properties examined.                  medium, Leut Urar segregants selected.  HYS2 hys2-1 RAD9+ strain exhibited G2/M arrest                         gene appears essential cell growth. <br /> incubated 370C 4 h, introduction rad9&amp;:LEU2                             nucleotide sequence HYS2 5&apos;-noncoding region <br /> mutation hys2-1 reduced level arrest G2/M (Fig. 2).                   revealed single MIul site located 107 bp upstam  rad9 mutation appears largely alleviate cell cycle                tranlational initiation site HYS2. sequence &apos;ACGCGT&apos; arrest hys2-1 mutant. failure arrest caused            closely related sequence, called MluI cell cycle box (MCB) <br /> removal checkpoint, expect observe                             element, required coordinate expression group concomitant loss viability dividing double mutant strain                 DNA replication genes including CDCJ7 (DNA polymerase , <br /> (24,35).  viability cells examined                   CDC2 (DNA polymerase 8), POL2 (DNA polymerase e), CDC9 experiment, hys2-1 rad9&amp; :LEU2 strains exhibited rapid                      (DNA ligase) CDC8 (thymidylate kinase), beginning late loss viability failure cease cell division 37WC (Fig.               Gi extending S phase (reviewed ref. 37). analyzed <br /> 4). conclude cell cycle arrest hys2-1 shift                HYS2 gene tanscript Northern blotting using methods restrictive temperature requires intact RAD9 gene.                          described Materials Methods. identified single ranscript <br />                                                                                    -1.7 kb exponentally growing cells (data shown).                                                                                    detemine whetier expression ofHYS2 coordinately reulated <br /> Cloning    HYS2 gene                                                        tughout cell cycle, HYS2 mRNA levels monitred cloned HYS2 gene complementation HU                               culture synchronized G1 phase tment factor mating <br /> sensitive phenotpe hys2-1 mutant. transforming                        pheromone, released arrest incubation fresh medium <br /> genomic library constructed YCp5O strain KSH542                            sampled intervals 20 mi. shown Figure 5, <br /> (hys2-1 ura3), screened tasformants able grow                         accumulation ofHYS2 mRNA oscillated peaking arwnd time YEPD containing HU. single plasmid-linked transfonnant                       bud egence, corresponds late Gl early S.  <br /> identified transfonnant contained plasmid (p12)                   cell cycle regulation modest (activated 2- 3-fold) tie <br /> 10 kb insert. Analysis subclones p12 insert                          triscnpts present cell cycle. <br /> demonstrated 2.5 kb HindLl-SalI fragment complem- <br /> ented HU sensitivity temperature sensitive growth (data                   DISCUSSION shown). HYS2 gene maps chromosome <br /> X using cloned gene probe blot yeast chromosomes                    screened mutants sensitive HU identified <br /> separated pulse field gel electrophoresis (data shown).                 distinct HYS genes. hys mutations expected fall nucleotide sequence 2.5 kb Hind-Sall DNA fragment                        classes. class consists mutatons defective sans detenined (DDBJ accession  D50324).                             coupling mitosis completion DNA replication. single open-reading firme witiin region, corre-                     presence HU, enter mitosis incomplete chromo- <br />                                                                                                               Nucleic Acids Research, 1995, Vol. 23,  17 3499 <br />  <br />                                                                                                   adjacent SUI2. molecular function HYS2                     MI                                                                            unknown, observations suggest Hys2 likely                                                                                                   involved DNA replication. restrictive temperature,                                                                                                   hys2-J mutation causes accumulation cells large bud <br />              7,J                                                                                  single nucleus neck bud.                    4 <br />                                                                                                   average DNA content population hys2-1 cells shifted <br />                                                                                                   G2 value. phenotypes indicate                                                                                                   bulk DNA replication completed hys2-J cells                                                                                                   restrictive temperature, mitosis blocked. hys2-J mutation <br />                                                                                                   causes elevated levels chromosome loss genetic <br />                    8 <br />                                                                                                   recombination causes increase probability death cells <br />                              ~~~~~~~~~~~~~~~~~~~~~~~~~~                                         lacking functional RAD9 checkpoint gene. <br />                        ) <br />                                                                                                      hys2-J mutation causes cells arrest DNA repli- <br />                                                                                                   cation essentially complete level detection <br />                                                                                                   provided flow cytometry. hys2-J mutant arrested                                                                                                   G2 phase restrictive temperature, hys2-J rad9 <br />                                                                                                   double mutant failed arrest died rapidly. consistent <br />                              1   1)   411   61)     81)    1(1(1   12(    140 160 181) 21)0 220   previous observation RAD9 checkpoint                                                           Timie (min)                             required late S/G2 phase arrest mutants defective                                                                                                   DNA replication (24). suggested DNA lesions <br />        HYS2                                   UbN                        e.d           N          activate RAD9 checkpoint arrest cell division G2 prior <br />                                                                                                   mitosis. DNA damage evident hys2-J cells level        CLN2                           4. 4. <br />                                                                                     -         elevated mitotic recombination (32) inability chromo- <br />        ACT]                  ,*Sp           *       au ae                          tpa@ <br />                                                                                                   somes migrate pulse field gel electrophoresis                                                                                                   associated DNA damage.  possible                                                                                                   DNA lesions formed restrictive temperature hys2 <br /> Figure 5. Northern blot analysis HYS2 transcription. micrograms                       mutant activate RAD9 checkpoint arrest G2. Finally, total RNA prepared samples cells taken 20 min intervals                          noted cell cycle arrest cdc8 mutants factorrelease separated 1% formaldehyde-agarose gels analyzed                       restrictive temperature HU treated cells does require Northern blot analysis using HYS2 probe. cell cycle synchronization                    RA4D9 gene (24,42,43). CDC8 gene encodes thymidylate confirmed probing CLN2. levels CLN2 transcript <br /> fluctuate cell cycle, peaking G1/S boundary (29,44).                   kinase required <span id='am-12' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span>dTTP</span> synthesis HU potent inhibitor <br /> blot hybridized ACT] loading control  panel). Samples                          ribonucleotide reductase necessary deoxyribonucleotides analyzed cell-cycle synchrony  panel). Unbudded cells                       production.  limitation precursors DNA synthesis <br /> (U) cells small buds (0) quantitated. intensity HYS2                     does appear result RAD9 checkpoint activation. ACT] bands quantitated densitometric tracing film appropriate exposure levels HYS2 (O) CLN2   mRNAs                               suggest HYS2 plays role step DNA replication <br /> normalized relative toACTI transcript levels (middle panel). peak value                    production precursors DNA synthesis. <br /> transcript levels given value 100 graphical purpose.                                    genes involved DNA replication periodically <br />                                                                                                   expressed cell cycle, peaking GI/S boundary. <br />                                                                                                   periodic expression dependent MCB element <br />                                                                                                   located promoter regions (37). HYS2 gene copy <br />                                                                                                   MCB element appropriate distance upstream somal duplication, resulting lethality. Enoch et al. (16)                                 translation start site.  cell cycle periodicity HYS2 <br /> used screen fission yeast identify genes involved                                   transcription weak (varied 2- 3-fold) checkpoint control prevents mitosis occurring S                                   transcript present cell cycle. Furthermore, cells <br /> phase completed. DNA sequence complementation analyses                                     expressing HYS2 constitutively GALI promoter did <br /> revealed HYS] identical SPK1/MEC21RAD531SADJ,                                        progress cell cycle observable perturba- <br /> gene known involved S phase checkpoint <br />                                                                                                   tion (data shown). results suggest transcrip- <br /> (38-41).  hysi mutation assigned class.                                tional control cell cycle critical roles HYS2. second class consists mutations defective DNA <br /> replication itself. HU blocks DNA synthesis, mutations                                    Additionally, results imply Hys2 protein defective DNA metabolism predicted cause severe                                    present cell cycle roles DNA <br /> growth retardation presence HU. contrast                                metabolism DNA replication, DNA damage <br /> class, mutations advance cell cycle                                       repair. fact, hys2-J mutation conferred sensitivity presence HU. shown evidence indicating                                     DNA damage-inducing agent MMS. <br /> hys2 mutation assigned second class. screen                                         summary, characterization hys2 mutant demonstrated unlikely saturated, identified allele                                 screening hys mutants useful better understanding cell <br /> HYS gene.                                                                                          cycle regulation S G2 phases. addition extensive <br />    single recessive mutation HYS2 responsible HU                                 studies HYSJ/SPKJIMEC21RAD531SADJ HYS2, charac- <br /> sensitive temperature sensitive growth phenotypes.                                    terization hys3 hys4 mutants understanding <br /> shown HYS2 essential gene located chromosome X                                      replication control cell cycle. <br />  3500 Nucleic Acids Research, 1995, Vol. 23,  17 <br />  <br /> ACKNOWLEDGMENTS                                                                  18 Sherman, F., G. R. Fink J. B. Hicks. (1986) Laboratory Course <br />                                                                                     Manualfor Methods Yeast Genetics. Cold Spring Harbor Laboratory, <br />                                                                                     Cold Spring Harbor, NY thank Ted Weinert sending materials communicating                     19 Sambrook, J., E. F. Fritsch T. Maniatis. (1989) Molecular Cloning: unpublished results, Steve Elledge communicating SADI                       Laboratory Manual, Second edition. Cold Spring Harbor Laboratory, Cold <br /> data publication, Hiroyulk Araki Stuart MacNeill                     Spring Harbor, NY <br /> helpful discussion Charlie Brenner, Joe Horecka Rosama-                  20 Ito, H., Y Fukuda, K. Murata  Kimura. (1983) J. Bacteriol., 153, <br />                                                                                      163-168. <br /> na Ruggieri critical reading manuscnpt. work                 21 Rothstein, R. (1983) Methods Enzymol., 101, 202-211. <br /> supported Inamori foundation (K.S.) Grants Aid                    22 Gietz, R. D.  Sugino. (1988) Gene, 74, 527-534. Ministry Education, Science Culture Japan (K.S.                23 Blank, , B. Kim L.  Loeb. (1994) Proc. Natl. Acad. Sci. USA, 91, <br /> K.M.).                                                                              9047-9051. <br />                                                                                  24 Weinert, T.  L. H. Hartwell. (1993) Genetics, 134, 63-80. <br />                                                                                  25 Hagan,  M. J. S. Hyams. (1988) J. Cell Sci., 89, 343-357. <br />                                                                                  26 Hisamoto, N., K. Sugimoto K. Matsumoto. (1994) Mol. Cell. Biol., 14, <br />                                                                                     3158-3165. <br /> REFERENCES                                                                       27 Gerring, S. L., F. Spencer P. Hieter. (1990) EMBO J., 11, 87-96. <br />                                                                                  28 Hennessy, K. M.,  Lee, E. Chen D. Botstein. (1991) Genes Dev., 5, <br />   1 Hartwell, L. H. T.  Weinert. (1989) Science, 246, 629-634.                958-969. <br />  2 Munray,  W. (1992) Nature, 359,599-604.                                     29 Wittenberg, C., K. Sugimoto S.  Reed. (1990) Cell, 62, 225-237. <br />  3 Enoch, T. P. Nurse. (1990) Cell, 60, 665-673.                             30 Hadwiger, J. , C. Wittenberg, M.  Barros Lopes, H. E. Richardson <br />  4 Lundgren, K., N. Walworth, R. Booher., M. Dembski, M. Kirschner D.           S.  Reed. (1989) Proc. Natl. Acad. Sci. USA, 86, 6255-6259. <br />     Beach. (1991) Cell, 64, 1111-1122.                                           31 Hartwell, L. H. (1976) J. Mol. Biol., 104, 803-817. <br />  5 Amon, , U. Surana,  Muroff K. Nasmyth. (1992) Nature, 355,            32 Hartwell, L. H. D. Smith. (1985) Genetics, 110, 381-395. <br />     368-371.             <br /> </body></html>