<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                     Database, Vol. 2013, Article <span id='am-32' about='oboInOwl:id' typeof='owl:Thing'>ID</span> bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />  <br />  <br /> Database update <br /> Version 6 consensus yeast metabolic <br /> network refines biochemical coverage improves model performance <br /> Benjamin D. Heavner1, Kieran Smallbone2, Nathan D. Price1 Larry P. Walker3,* <br /> 1 <br />   Institute Systems Biology, 401 Terry Ave N., Seattle, WA 98109, USA, 2Manchester Centre Integrative Systems Biology, University Manchester, Manchester, M1 7DN, UK 3Biomass Conversion Laboratory, Department Biological Environmental Engineering, <br /> Cornell University, Ithaca, NY 14853, USA <br />  <br /> *Corresponding author: Email: lpw1@cornell.edu <br />  <br /> Submitted 1 March 2013; Revised 14 2013; Accepted 12 July 2013 <br />  <br /> Citation details: Heavner,B.D., Smallbone,K., Price,N.D., et al. Version 6 consensus yeast metabolic network refines biochemical coverage improves model performance. Database (2013) Vol. 2013: article ID bat059; doi:10.1093/database/bat059. <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Updates maintain state art reconstruction yeast metabolic network essential reflect  <br /> standing yeast metabolism functional organization, eliminate inaccuracies identified earlier iterations, improve predictive accuracy continue expand novel subsystems extend comprehensiveness model.  present version 6 consensus yeast metabolic network (Yeast 6) update community <br /> effort computationally reconstruct genome-scale metabolic network Saccharomyces cerevisiae S288c. Yeast 6 <br /> comprises 1458 metabolites participating 1888 reactions, annotated 900 yeast genes encoding catalyzing enzymes. Compared Yeast 5, Yeast 6 demonstrates improved sensitivity, specificity positive nega- <br /> tive predictive values predicting gene essentiality glucose-limited aerobic conditions analyzed flux balance <br /> analysis. Additionally, Yeast 6 improves accuracy predicting likelihood mutation cause auxotrophy. network reconstruction available Systems Biology Markup Language (SBML) file enriched Minimium <br /> Information Requested Annotation Biochemical Models (MIRIAM)-compliant annotations. Small- macromol- <br /> ecules network referenced authoritative databases Uniprot ChEBI. Molecules reactions annotated appropriate publications contain supporting evidence. Yeast 6 freely available http://yeast.sf.net/ separate SBML files: model using SBML level 3 Flux Balance Constraint package, model compatible MATLAB&#195;&#8226; COBRA Toolbox backward compatibility reconstruction containing reactions experimental evidence  non-biological reactions necessary simulating growth). <br /> Database URL: http://yeast.sf.net/ <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />  <br />  <br />                                                                                                                                                   network, Yeast 6. differences Yeast 5 Introduction                                                                                                                                      Yeast 6 described fully detailed 2007, community effort integrate previously pub-                                                                                          supplementary data attached publication. lished genome-scale reconstructions yeast metabolic                                                                                        update maintains emphasis standards compliance, <br /> network (1, 2) produced &#226;&#8364;&#732;consensus&#226;&#8364;&#8482; representation                                                                                           unambiguous metabolite naming readable <br /> yeast metabolism (3), subsequently                                                                                                 annotations available structured document <br /> updated iterative collaborative curation mul-                                                                                          format. Additionally, developed MATLAB&#195;&#8226; scripts <br /> tiple research groups (4, 5).  introduce version 6                                                                                        demonstrate approach comparing Yeast 5 consensus reconstruction yeast metabolic                                                                                            Yeast 6 using flux balance analysis (FBA) methods, <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br /> &#195;&#376; Author(s) 2013. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/ <br /> licenses 3.0/), permits unrestricted reuse, distribution, reproduction medium, provided original work properly <br /> cited.                                                                                                                       Page 1 5 <br />                                                                                                  (page number citation purposes) <br />  Database update                                                                                                                      Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> leveraging COnstraint-Based Reconstruction                                                                                             compatible MATLAB&#195;&#8226; COBRA toolbox (6)  <br /> Analysis (COBRA) Toolbox (6). scripts                                                                                              ward compatibility GENRE containing reactions <br /> included supplementary data.                                                                                                                   experimental evidence. <br />    emphasize distinction established <br /> biochemistry included metabolic genome-scale net- <br /> work reconstruction (GENRE) (7) additional model- <br />                                                                                                                                                   Results <br /> ing assumptions required analysis simulation                                                                                        Overview network characteristics <br /> genome-scale model (GEM) (7), Yeast 6                                                                                                Yeast 6 resulted effort improve predictive <br /> available http://yeast.sf.net/ separate Systems                                                                                       accuracy Yeast 5 manual curation, particu- <br /> Biology Markup Language (SBML) files: GEM using                                                                                             lar focus removing information sup- <br /> SBML level 3 Flux Balance Constraint package, GEM                                                                                               ported published literature adding metabolic <br />                                                                                                                                                   pathway information recently discovered. <br />                                                                                                                                                   result effort, Yeast 6 contains fewer metabolites <br /> Table 1. Summary statistics Yeast 5 Yeast 6 <br />                                                                                                                                                   reactions Yeast 5 (Table 1), accurate Summary statistics                                                                    Yeast 5                             Yeast 6                 predictions gene essentiality (Table 2) auxotroph- <br />                                                                                                                                                   inducing mutations (Table 3). Ninety-seven 1868  <br /> Genes                                                                                    918                                900                   actions shared Yeast 5 Yeast 6 different <br /> Metabolites                                                                           1655                                1458                    constraints, reflecting refinements reaction reversibility <br /> Reactions                                                                             2110                                1888                    yeast metabolic network. <br /> Reactions PMID references                                                        37.6%                               40.4% <br />                                                                                                                                                   Novel features Yeast 6 additional quality curation, Yeast 6 fewer blocked <br />                                                                                                                                                   Metabolites, reactions genes differ Yeast 5 <br /> reactions greater percentage reactions annotated literature evidence; contains fewer genes, metabolites                                                                                Yeast 6. Detailed lists common distinct <br /> reactions Yeast 5. reconstruction includes open read-                                                                                metabolites, reactions, genes, constraints, auxotrophs ing frames annotated &#226;&#8364;&#732;dubious&#226;&#8364;&#8482; Saccharomyces Genome                                                                                     knockout predictions included supplementary data, <br /> Database (7).                                                                                                                                     code used generate comparisons.  <br />  <br />  <br />              Table 2. Comparing gene essentiality predictions Yeast 5 Yeast 6 <br />  <br />  <br />                  Knockout Simulation Results                                                              Yeast 5                    Yeast 6                                              Observed viable                          Observed inviable <br />                                                                                                                                                                                              mutants                                   mutants <br />                                                                                                            683                         672 <br />                                                                                                                                                                   Simulated viable <br />  <br />  <br />  <br />  <br />                  True Positive                                                                            (74%)                      (75%)                                           24                              13         34                                        2 <br />                                                                                                                                                                      mutants <br />  <br />  <br />  <br />  <br />                                                                                                            108                         135 <br />                                                                                                                                                                                                  659                                                 80 <br />                  True Negative                                                                            (12%)                      (15%) <br />                                                                                                            114                         82 <br />                  False Positive                                                                           (12%)                       (9%)                                                  True Positive                                False Positive <br />                                                                                                             13                         11 <br />                                                                                                                                                                   inviable mutants <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                       2                                                                                 27 <br />                  False Negative                                                                            (1%)                       (1%) <br />                                                                                                                                                                       Simulated <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                   11                                               108 <br />                  Sensitivitya                                                                              98.1%                      98.4% <br />                  Specificityb                                                                              48.7%                      62.2% <br />                                                                                                                                                                                            False Negative                                True Negative <br />                                                                            c <br />                  Positive Predictive Value                                                                 85.7%                      89.1%                                          Yeast 5            Yeast 6                   networks <br />                                                                                d <br />                  Negative Predictive Value                                                                 89.3%                      92.5% <br />                                                                                                e <br />                  Matthews Correlation Coefficient                                                            0.59                        0.70 <br />                             TP/(TP + FN) <br />              b <br />                TN/(TN + FP) <br />              c <br />               TP/(TP + FP) <br />              d <br />                TN/(TN + FN) <br />              e              TP TN FP FN <br />                p&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402;&#239;&#172;&#402; <br />                                                                   &#239;&#172;&#402; (8) <br />                    &#195;&#176;TP&#195;&#190;FP &#195;&#382;&#195;&#176;TP&#195;&#190;FN&#195;&#382;&#195;&#176;TN&#195;&#190;FP &#195;&#382;&#195;&#176;TN&#195;&#190;FN&#195;&#382; <br />              Yeast 6 fewer false-positive predictions true-negative predictions gene essentiality, leading improve- <br />              ments sensitivity, specificity, positive predictive value, negative predictive value Matthews correlation coefficient               Discussion information use metric). <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Page 2 5 <br />  Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059                                                                                                                                                                Database update <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> Table 3. Comparing auxotroph mutant predictions Yeast 5                                                                                        biomass produced following deletion  Yeast 6 <br />                                                                                                                                                   essential genes), 27 new true negative predictions  e. FBA <br /> Simulation Results                                                                                 Yeast 5                Yeast 6                 predicts biomass produced essential <br />                                                                                                                                                   genes deleted), 1 new gene predicted induce auxo- <br /> Auxotroph-inducing genes                                                                                92                     93                 trophy following mutation 7 newly correct predictions <br /> Correct auxotrophy predictions                                                                          57                     64                 auxotrophy phenotypes. Yeast 6 introduces new pre- <br /> Mutant incorrectly predicted viable                                                               32                     26                 dictions differ laboratory observations, new <br />  minimal medium                                                                                                                                false-positives (growth predicted despite deletion Mutant incorrectly predicted inviable                                                               3                       3               essential gene), new auxotrophic mutant predicted  <br />  supplemented medium <br />                                                                                                                                                   correctly incapable growth supplemented media <br /> Yeast 6 genes deletion reported                                                                                     new mutants incorrectly predicted viable cause auxotrophy correct predictions                                                                                      minimal media despite reported auxtrophy. note auxotrophy Yeast 5.                                                                                                                          differences model prediction observation <br />                                                                                                                                                   arise regulatory constraints outside                                                                                                                                                   scope metabolic reconstruction, informative <br /> supplementary data include specific rationale reac-                                                                                      opportunities continued network curation. <br /> tion deletions additions, detailed descriptions function gene removed added. sum-                                                                                            Yeast 6 aerobic anaerobic growth <br /> mary, Yeast 5 203 metabolites Yeast 6,                                                                                        Unlike Yeast 5, Yeast 6 does include separate biomass Yeast 6 introduces 6 new metabolites. smaller                                                                                         definitions simulating aerobic anaerobic growth. <br /> number metabolites Yeast 6 arises removal                                                                                          Yeast 6 correctly predicts yeast grow min- metabolites involved dead-end reactions anno-                                                                                         imal media strict anaerobic conditions [Saccharomyces <br /> tated modeling reaction  e. included Yeast 5                                                                                     cerevisiae requires supplementation unsaturated biochemical genomic evidence). dead-end                                                                                             fatty acids sterols (9, 10)]. Simulating anaerobic <br /> reactions annotated gene removed.                                                                                            growth Yeast 6 requires simulated medium Yeast 5 242 reactions Yeast 6,                                                                                        supplemented sterols  e. exchange reactions Yeast 6 20 reactions Yeast 5. detailed                                                                                     episterol, ergosterol, fecosterol, <span id='am-1' about='obo:IMR_0200096' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-2' property="oboInOwl:id" content="IMR:0200096" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C01724" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Lanosterol" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lanosterol</span>, zymosterol supplementary data, reactions removed Yeast 6                                                                                            ergosta-5,7,22,24(28)-tetraen-3beta-ol nonzero <br /> clude incorrect cofactor specificity, invol-                                                                                     lower bounds]. <br /> ving protein modification, supported literature review dead-end modeling reactions. Yeast                                                                                          Data annotation standards <br /> 5 31 genes Yeast 6, Yeast 6                                                                                           network reconstruction provided SBML (11) file <br /> 13 genes Yeast 5. 31 genes removed,                                                                                        enriched MIRIAM-compliant (12) annotations. Small- <br /> 21 encode proteins involved protein modification                                                                                            macromolecules referenced community-stand- <br /> Glycosylphosphatidylinositol (GPI)-anchor assembly (pro-                                                                                          ard databases Uniprot (13) ChEBI (14). <br /> cesses strictly metabolic), 4 annotated                                                                                          Molecules reactions annotated appropri- &#226;&#8364;&#732;putative&#226;&#8364;&#8482; function, 1 removed unclear                                                                                        ate publications contain supporting evidence.  <br /> cofactor specificity, 2 annotated poorly characterized reac-                                                                                      network presented computational framework <br /> tions involved ergosterol biosynthesis, 1 encodes                                                                                          adheres community standards entirely trace- <br /> flippase, require finer level membrane                                                                                           able. facilitate comparison reconstructions compartmentalization currently exists model,                                                                                          models, Yeast 6 metabolite reaction identifiers  metabolic function remaining 2                                                                                             sistent Yeast 5 identifiers (e.g. reaction &#226;&#8364;&#732;r_0123&#226;&#8364;&#8482; established.                                                                                                                                 Yeast 5 reaction &#226;&#8364;&#732;r_0123&#226;&#8364;&#8482; Yeast 6,                                                                                                                                                   metabolite &#226;&#8364;&#732;s_0042&#226;&#8364;&#8482; Yeast 5 metabolite Essentiality auxotrophy predictions Yeast 6                                                                                              &#226;&#8364;&#732;s_0042&#226;&#8364;&#8482; Yeast 6). changes reconstructed metabolic network,                                                                                                Yeast 6 follows modeling conventions Yeast <br /> combined incorporation suggested changes                                                                                           5. used SBML specification encoding reaction biomass <span id='am-12' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> (8), rise altered FBA predic-                                                                                      metabolite annotation COBRA <br /> tions gene essentiality glucose-limited aerobic condi-                                                                                      Toolbox-specific convention using custom &#226;&#8364;&#732;Notes&#226;&#8364;&#8482; <br /> tions altered predictions auxotrophy induced                                                                                            field. sign convention exchange reactions gene deletion. compared list es-                                                                                           positive flux values represent compounds produced sential genes compiled  list included sup-                                                                                      FBA simulation, negative flux values represent com- <br /> plementary data testYeast.m file), Yeast 6 13 new true                                                                                        pounds consumed, include biomass specific <br /> positive predictions gene essentiality  e. FBA predicts                                                                                      species model. <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />                                                                                                                                                                                                                                                       Page 3 5 <br />  Database update                                                                                                                      Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059 <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br /> Assessing metabolic models                                                                                                                           Additional limitations arise appropriately lim- gene picked random likely                                                                                              ited scope metabolic network reconstruction. <br /> inessential essential growth (15), overall accuracy                                                                                      Condition-dependent constraints arise various good metric assessing model predictive ability                                                                                       regulatory mechanisms included Yeast 6.  <br /> (16). statistical issue previously recognized,                                                                                      pathways affected transcriptional regu- <br /> leading use &#226;&#8364;&#732;geometric mean accuracy&#226;&#8364;&#8482;                                                                                          latory events glucose repression incor- <br /> metric evaluating metabolic network models (2).                                                                                               rectly predicted carry fluxes FBA (e.g. geometric mean ignores positive predictive                                                                                            absence additional constraints, malate cycle <br /> value  called precision) (17), report values                                                                                       mitochondrial malate dehydrogenase cyto- contingency matrix (Table 2), summarize pre-                                                                                          plasmic malate dehydrogenase). Integration regulatory <br /> dictive ability model Matthews correlation                                                                                        metabolic networks remains area active  <br /> coefficient (18), metric robust range                                                                                       search (21, 22). <br /> prevalence values incorporates positive negative                                                                                             Like metabolic network models, reactions predictive values.                                                                                                                                Yeast 6 carry flux FBA simulation. Yeast 6 738 <br />    include testYeast.m script supplementary                                                                                             blocked reactions (39%), similar portion blocked reac- <br /> data facilitate evaluation model. script                                                                                          tions Yeast 5 (38%). Blocked reactions indicate know- <br /> compares model phenotype predictions lists ver-                                                                                        ledge limitations  reactions leading dead-end <br /> ified yeast open reading frames, genes consider                                                                                           metabolites metabolic fate origin unknown, <br /> essential genes cause auxotrophy deletion.                                                                                          reactions involved unconnected portions metabol- compiled lists Yeast Deletion Project (15)                                                                                      ism, form unconnected subgraphs network). information Saccharomyces Genome                                                                                                  Like FBA predictions differ observation, Database (19).                                                                                                                                    number blocked reactions affected condition- <br />                                                                                                                                                   specific constraints, particularly constraints reaction  <br />                                                                                                                                                   versibility. observed relaxing reversibility <br /> Discussion                                                                                                                                        reactions involving <span id='am-24' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> cofactors reduces                                                                                                                                                   number blocked reactions. <br /> Yeast 6 current state art reconstruction                                                                                                                                                      Unlike Yeast 5, prediction anaerobic ethanol pro- <br /> S. cerevisiae metabolic network. eliminates inferred <br />                                                                                                                                                   duction FBA simulation requires manual restriction reactions evidence, adds new reac- <br />                                                                                                                                                   reaction catalyzed <span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span>ATP</span> synthase. requirement tions based recent evidence results improved pre- <br />                                                                                                                                                   condition-dependent constraint arise dictions experimental data. maintains distinction <br />                                                                                                                                                   lack regulatory constraints described  GENRE GEM, emphasizing traceable <br />                                                                                                                                                   need physicochemical capacity constraints allow- <br /> annotation included information, differentiates  <br />                                                                                                                                                   able flux. Additionally, reflect  presently <br /> tween established biochemistry hypotheses                                                                                                                                                   uncharacterized limitations. note integrat- generated automated techniques gap-filling <br />                                                                                                                                                   ing regulatory constraints, detailed reconstruction cofac- <br /> algorithms (20). useful addition  <br />                                                                                                                                                   tor proton balancing remains area active <br /> sensus resource large community researchers <br />                                                                                                                                                   research constraint-based modeling community use yeast metabolic model guide experimental <br />                                                                                                                                                   (23); redox conditions clearly critical importance modeling efforts. <br />                                                                                                                                                   function ATP synthase. <br />  <br /> Limitations                                                                                                                                       invitation participate community effort <br /> Improving reconstruction yeast metabolic net-                                                                                          reconstruct yeast metabolic network <br /> work remains ongoing project. addition                                                                                               Computational reconstruction modeling yeast <br /> model predictions differ experimental observa-                                                                                          metabolism ongoing project, invite additional <br /> tions described  e. false-positive, false-negative                                                                                        community participation effort. Suggestions incorrect auxotroph predictions), remains sub-                                                                                          improving yeast consensus reconstruction derived <br /> stantial opportunity improve reconstruction                                                                                             models submitted network.reconstruc <br /> <span id='am-6' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> metabolism. point evident                                                                                             tion@manchester.ac.uk. Metabolites enzymes fact unsaturated fatty acids currently                                                                                               unambiguously identified, using existing model data- <br /> required simulating anaerobic growth,                                                                                                base (ChEBI UniProt) identifiers. New reactions arises continued use generic lipid species                                                                                         supplied primary evidence mechanism <br />  e. compounds using generic residual (-R) groups,                                                                                        catalysis, PubMed identifiers. Reactions precise stoichiometrically balanced definitions                                                                                           evidence clear reasons proposed <br /> fatty acid moieties].                                                                                                                             addition. <br />  <br />  <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br /> Page 4 5 <br />  Database, Vol. 2013, Article ID bat059, doi:10.1093/database/bat059                                                                                                                                                                Database update <br /> ............................................................................................................................................................................................................................................................................................. <br />  <br />  <br />                                                                                                                                                      6. Schellenberger,J., Que,R., Fleming,R.M.T. et al. (2011) Quantitative <br /> Supplementary data                                                                                                                                      prediction cellular metabolism constraint-based models:                                                                                                                                                         COBRA Toolbox v2.0. Nat. Protoc., 6, 1290&#226;&#8364;&#8220;1307. <br /> Supplementary data available Database Online. <br />                                                                                                                                                      7. Price,N.D., Reed,J.L. Palsson,B.O. (2004) Genome-scale models <br />                                                                                                                                                         microbial cells: evaluating consequences constraints. <br />                                                                                                                                                         Nat. Rev. Microbiol., 2, 886&#226;&#8364;&#8220;897. <br /> Acknowledgments                                                                                                                                      8. Zomorrodi R. Maranas,C.D. (2010) Improving iMM 904 S. authors gratefully acknowledge Hnin Aung Kalliopi                                                                                               cerevisiae metabolic model using essentiality synthetic lethality <br />                                                                                                                                                         data. BMC Syst. Biol., 4, 178. <br /> Trachana, Julie Bletz, James Eddy, John Earls                                                                                                                                                      9. Andreasen  Stier,T.J. (1954) Anaerobic nutrition members Price group constructive <br />                                                                                                                                                         Saccharomyces cerevisiae. II. Unsaturated fatty acid requirement <br /> comments manuscript stimulating discussion                                                                                                growth defined medium. J. Cell. Physiol., 43, 271&#226;&#8364;&#8220;281. <br /> metrics evaluating model predictive accuracy.                                                                                                  10. Andreasen  Stier,T.J.B. (1953) Anaerobic nutrition                                                                                                                                                        Saccharomyces cerevisiae.  Ergosterol requirement growth                                                                                                                                                        defined medium. J. Cell. Physiol., 41, 23&#226;&#8364;&#8220;36. <br /> Funding                                                                                                                                            11. Hucka,M., Finney , Sauro,H.M. et al. (2003) systems biology <br />                                                                                                                                                        markup language (SBML): medium representation ex- <br /> Support provided work Cornell Biofuels                                                                                              change biochemical network models. Bioinformatics, 19, <br /> Research Laboratory U. S. Department                                                                                                             524&#226;&#8364;&#8220;531. <br /> Transportation, Federal Grant (#DTOS59-07-G-00052),                                                                                            12. Nove N.L., Finney , Hucka,M. et al. (2005) Minimum informa- U.S. Department Agriculture, Award (#2010-38502-                                                                                                tion requested annotation biochemical models <br />                                                                                                                                                        (MIRIAM). Nat. Biotechnol., 23, 1509&#226;&#8364;&#8220;1515. <br /> 21900). Support provided work Institute <br />                                                                                                                                                    13. UniProt Consortium. (2012) Update activities Systems Biology Center Systems Biology/2P50 <br />                                                                                                                                                        Universal Protein Resource (UniProt) 2013. Nucleic Acids Res., <br /> GM076547, Grand Duchy Luxembourg Systems                                                                                                        41, D43&#226;&#8364;&#8220;D47. <br /> Medicine Consortium, Camille-Dreyfus Teacher                                                                                               14. Hastings,J., Matos,P., Dekker  et al. (2012) ChEBI reference <br /> Scholar Program (NDP)  B.D.H.). Support provided                                                                                                database ontology biologically relevant chemistry: enhance- EU FP7 (KBBE) grant (289434) &#226;&#8364;&#732;BioPreDyn: New                                                                                                    ments 2013. Nucleic Acids Res., 41, D456&#226;&#8364;&#8220;D463. <br /> Bioinformatics Methods Tools Data-Driven                                                                                                   15. Winzeler,E , Shoemaker,D.D., Astromoff  et al. (1999) <br /> Predictive Dynamic Modelling Biotechnological                                                                                                       Functional characterization S. cerevisiae genome gene <br />                                                                                                                                                        deletion parallel analysis. Science, 285, 901&#226;&#8364;&#8220;906. <br /> Applications&#226;&#8364;&#8482;  K.S.). Funding open access charge: <br />                                                                                                                                                    16. Alberg J., Park,J.W., Hager,B.W. et al. (2004) use &#226;&#8364;&#732;&#226;&#8364;&#732;overall <br /> U. S. Department Transportation, Federal Grant <br />                                                                                                                                                        accuracy&#226;&#8364;&#8482;&#226;&#8364;&#8482; evaluate validity screening diagnostic tests. <br /> (#DTOS59-07-G-00052), U.S. Department                                                                                                       J. Gen. Intern. Med., 19, 460&#226;&#8364;&#8220;465. <br /> Agriculture, Award (#2010-38502-21900)                                                                                                             17. Kubat,M., Holte,R.C. Matwin,S. (1998) Machine learning                                                                                                                                                        detection oil spills satellite radar images. Mach. Learn., <br /> Conflict  declared.                                                                                                                   30, 195&#226;&#8364;&#8220;215. <br />                                                                                                                                                    18. Baldi,P., Brunak,S., Chauvin,Y. et al. (2000) Assessing accuracy                                                                                                                                                        prediction algorithms classification: overview. Bioinformatics, <br />  <br /> References                                                                                                                                             16, 412&#226;&#8364;&#8220;424. <br />                                                                                                                                                    19. Cherry,J.M., Hong,E.L., Amundsen,C. et al. (2012) Saccharomyces <br />   1. Mo,M.L., Palsson,B.&#195;&#732;. Herrga&#203;&#353;rd,M.J. (2009) Connecting extra-                                                                                 Genome Database: genomics resource budding yeast. <br />      cellular metabolomic measurements intracellular flux states                                                                                 Nucleic Acids Res., 40, D700&#226;&#8364;&#8220;D705. <br />      yeast. BMC Syst. Biol., 3, 37.                                                                                                                20. Brooks,J.P., Burns,W.P., Fong,S.S. et al. (2012) Gap detection   2. Kuepfer,L., Sauer,U. Blank,L.M. (2005) Metabolic functions                                                                                 genome-scale constraint-based models. Adv. Bioinformatics, 2012, <br />      duplicate genes Saccharomyces cerevisiae. Genome Res., 15,                                                                                     1&#226;&#8364;&#8220;10. <br />      1421&#226;&#8364;&#8220;1430.                                                                                                                                    21. Blazier S. Papin,J  (2012) Integration expression data   3. Herrgard,M.J., Swainston,N., Dobson,P. et a <br /> </body></html>