<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>JOURNAL BACTERIOLOGY, July 2011, p. 3304&#226;&#8364;&#8220;3312                                                                                 Vol. 193,  13 <br /> 0021-9193/11/$12.00 doi:10.1128/JB.00334-11 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />  <br />       Pantethine Rescues Phosphopantothenoylcysteine Synthetase          Phosphopantothenoylcysteine Decarboxylase Deficiency            Escherichia coli Pseudomonas aeruginosa&#228;&#338;&#164;&#226;&#8364;&#160; <br />                            Carl J. Balibar,* Micah F. Hollis-Symynkywicz, Jianshi Tao <br />                Department Infectious Diseases, Novartis Institutes BioMedical Research, 500 Technology Square, <br />                                                   Cambridge, Massachusetts 02139 <br />                                                Received 9 March 2011/Accepted 26 April 2011 <br />  <br />             Coenzyme (<span id='am-66' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span>) plays central essential role living organisms. pathway leading CoA <br />           biosynthesis considered attractive target developing new antimicrobial agents novel <br />           mechanisms action. using arabinose-regulated expression  essentiality coaBC, single <br />           gene encoding bifunctional protein catalyzing consecutive steps CoA pathway converting 4&#226;&#180;&#8226;- <br />           phosphopantothenate 4&#226;&#180;&#8226;-phosphopantetheine, confirmed Escherichia coli. Utilizing regulated <br />           coaBC strain, demonstrated E. coli effectively metabolize pantethine bypass           requirement coaBC. Interestingly, pantethine used Pseudomonas aeruginosa obviate coaBC. <br />           reciprocal complementation studies combination biochemical characterization, dem- <br />           onstrated differential characteristics pantethine utilization microorganisms           different substrate specificities associated endogenous pantothenate kinase, enzyme           CoA biosynthetic pathway encoded coaA E. coli coaX P. aeruginosa. <br />  <br />  <br />    Coenzyme (CoA) ubiquitous essential cofactor                  cell participates key processes, including cell living organisms, functions carrier activated            division virulence (9), biosynthesized primary <br /> acyl groups numerous central metabolic processes.                   precursor <span id='am-95' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-96' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-N-acetylglucosamine. GlmU, final enzyme notably, CoA provides indispensable phospho-                          required convert <span id='am-22' about='obo:IMR_0200474' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00095" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200474" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="D-Fructose" datatype="xsd:string"></span>D-fructose</span>-6-phosphate UDP-N-acetyl- <br /> pantetheine prosthetic group posttranslationally appended                   glucosamine, utilizes <span id='am-90' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-91' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span>acetyl-CoA</span> acetylation gluco- <br /> acyl carrier proteins fatty acids biosynthesized;               samine-1-phosphate (26). Furthermore, UDP-N-acetylgluco-  CoA participates biochemical pro-                       samine precursor teichoic acid(s) Gram-positive <br /> cesses tricarboxylic acid (TCA) cycle <span id='am-71' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-73' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-74' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span>                   bacteria (19) <span id='am-11' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> Gram-negative bacteria (4), degradation (30). step biochemical pathway                    surface-exposed molecules impact permea- <br /> version <span id='am-17' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>vitamin</span> B5 pantothenate CoA conserved                  bility bacterial cell. Finally, majority Gram-posi- taxa begins formation 4&#226;&#172;&#732;-phosphopantothenate,                  tive pathogens, including Staphylococcus aureus, Enterococcus <br /> catalyzed pantothenate kinase (PanK). 4&#226;&#172;&#732;-Phosphopanto-                      faecalis, Streptococcus pneumoniae, utilize mevalonate <br /> thenate subsequently condensed cysteine,                   pathway isoprenoid biosynthesis. mevalonate path- <br /> turn decarboxylated form 4&#226;&#172;&#732;-phosphopantetheine                    way, 3 units acetyl-CoA condensed form isopentenyl <br /> zymes phosphopantothenoylcysteine synthetase (PPCS)                        <span id='am-37' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span>pyrophosphate</span> (IPP), building block cell wall car- <br /> phosphopantothenoylcysteine decarboxylase (PPCDC),                          rier lipid undecaprenyl pyrophosphate (UPP) (7). <br /> spectively. Finally, 4&#226;&#172;&#732;-phosphopantetheine converted  <br />                                                                                   Despite significance bacteria, complete biosyn- <br /> phospho-CoA addition <span id='am-47' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-48' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>AMP</span> moiety catalyzed                                                                                thetic machinery synthesis CoA elucidated 4&#226;&#172;&#732;-phosphopantetheine adenylyltransferase (PPAT)                                                                                relatively recently, culminating identification Esche- 3&#226;&#172;&#732;-hydroxyl AMP ribose phosphorylated  <br />                                                                                richia coli coaBC gene, encodes single bifunctional <br /> phospho-CoA kinase (DPCK) form CoA (8) (Fig. 1). <br />                                                                                PPCS/PPCDC protein (43). existence coaBC single <br />    bacteria, CoA plays vital role biogenesis cell <br />                                                                                gene encoding protein 2 functional domains bac- <br /> envelope. Membrane lipid biogenesis utilizes fatty acid precur- <br />                                                                                teria stark contrast pattern seen eukaryotes <br /> sors biosynthesized using iterative condensation acetyl-CoA precursors (20). <span id='am-78' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-79' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-84' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="PGN" datatype="xsd:string"></span>peptidoglycan</span>, essential                    plants mammals, PPCS PPCDC bacterial structure maintains mechanical integrity                 individual enzymes encoded separate coaB coaC genes, <br />                                                                                respectively (1&#226;&#8364;&#8220;3, 13, 28). Given paucity information <br />                                                                                concerning coaBC bacteria potential target    * Corresponding author. Mailing address: Department Infectious           antimicrobial intervention, sought investigate effect <br /> Diseases, Novartis Institutes BioMedical Research, 500 Technol-            E. coli disruption coaBC. Interestingly, despite ogy Square, Cambridge, MA 02139. Phone: (617) 871-5745. Fax: (617)             validated essentiality coaBC (13, 15) reports pan- <br /> 871-5791. E-mail: carl.balibar@novartis.com. <br />                                                                                tothenate advanced precursor CoA    &#226;&#8364;&#160; Supplemental material article http://jb <br /> .asm.org/.                                                                     characterized actively transported bacteria (8, <br />    &#228;&#338;&#164; <br />      Published ahead print 6 2011.                                   30), <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> complementation pantethine <br />  <br />                                                                         3304 <br />  VOL. 193, 2011                                                                                                PANTETHINE BYPASS coaBC                      3305 <br />  <br />  <br />  <br />  <br />                                                              FIG. 1. CoA biosynthetic pathway. <br />  <br />  <br />  <br /> renders coaBC nonessential E. coli Pseudomonas                        PIR1 chemically competent cells. resulting plasmid, pKD4-PBAD, used <br /> aeruginosa. bypass mechanism dependent pres-                         template subsequent PBAD promoter integrations. <br />                                                                                        PBAD promoter integrated 15 bp upstream coaBC start <br /> ence pantothenate kinase. <br />                                                                                     codon use linear PCR product generated amplification                                                                                     pKD4-PBAD template P5 P6 ultramers, generating                         MATERIALS METHODS                                       BW25113PBADcoaBC strain  strains listed Table S2 supplemen- <br />                                                                                     tal material). PBAD promoter integrated 25 bp upstream coaD <br />    General materials procedures. Standard DNA, molecular cloning,                                                                                     start codon use linear PCR product generated amplification microbiological procedures performed described previously (39). Plas- <br />                                                                                     pKD4-PBAD template P7 P8 ultramers, generating mids, genomic DNA, PCR fragments run agarose gels, restriction digests <br />                                                                                     BW25113PBADcoaD strain. Transformation selection performed purified using QIAprep spin miniprep kit, DNeasy blood tissue kit, <br />                                                                                     described previously (6). Briefly, overnight cultures BW25113 cells harboring <br /> QIAquick gel extraction kit, QIAquick PCR purification kit, respectively <br />                                                                                     pKD46 (6) diluted 100-fold fresh LB containing 100 &#226;?&#174;g/ml ampicillin <br /> (Qiagen). Primers ultramers purchased Integrated DNA Tech- <br />                                                                                     0.2% (wt/vol) arabinose, grown 30&#194;&#176;C optical density 600 nm <br /> nologies. Phusion high-fidelity DNA polymerase, Quick ligase, restriction <br />                                                                                     (OD600) 0.5 reached, washed twice equal volume cold water, <br /> enzymes NEB. Electroporation performed MicroPulser <br />                                                                                     washed times 1 ml 10% glycerol, finally resuspended 10% <br /> Electroporator (Bio-Rad). Shot TOP10 PIR1 E. coli Invi- <br />                                                                                     glycerol using 1/250 initial culture volume. Competent cells (50 &#226;?&#174;l) trogen. DNA sequencing performed Beckman Coulter Genomics. LB, LB <br /> agar, Pseudomonas isolation agar (PIA) Difco. NZYM media          mixed 100 300 ng  exceeding 5 &#226;?&#174;l) gel-purified PCR product sucrose MP Biomedicals. <span id='am-55' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-56' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>, <span id='am-42' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span>NADH</span>, kanamycin, chloramphenicol,            inserted chromosome, electroporated 0.2-cm-gap cuvette using gentamicin, ampicillin, pantethine, CoA, dephospho-CoA, L-arabinose, lactate        EC2 setting (2.5 kV, 5 ms), recovered 1 ml SOC medium (Invitrogen)  <br /> dehydrogenase, pyruvate kinase, phosphoenolpyruvate (PEP)             taining 0.2% (wt/vol) arabinose 37&#194;&#176;C 2 h, finally plated LB agar <br /> Sigma. Pantothenate, IPTG (isopropyl-&#226;?&#164;-D-thiogalactopyranoside), imid-          containing 30 &#226;?&#174;g/ml kanamycin 1 mM arabinose. Colonies grew azole Acros Organics. Carbenicillin Fisher Scientific, dithio-   day replica plated confirm loss pKD46 acquisition arabinose- <br /> threitol (DTT) Promega, TCEP [tris(2-carboxyethyl)phosphine]           dependent growth. integrants confirmed PCR sequencing. Thermo Scientific, Complete protease inhibitor cocktail Roche,           Deletion panF replacement coaA E. coli coaX P. cobalt Talon metal affinity resin Clontech. Fast protein liquid        aeruginosa BW25113PBADcoaBC. panF gene deleted, leaving chromatography (FPLC) purification proteins performed using HiLoad         codons, using linear PCR product generated amplifi- <br /> 16/60 Superdex 200 prep grade column, &#194;&#168; KTA FPLC  Unicorn 5.0       cation pKD3 (12) P9 P10 ultramers, generating strain <br /> software (GE Healthcare).                                                           BW25113PBADcoaBC&#226;&#338;&#172;panF::cat. replacement coaA gene E. coli <br />    Construction coaBC- coaD-regulated E. coli strains. new vector      (EccoaA), splicing overlap extension PCR (21) used fuse chloram- <br /> created allowed step &#226;?&#173; Red recombinase knockout method devel-          phenicol resistance cassette downstream coaX gene P. aeruginosa <br /> oped Datsenko Wanner (12) adapted perform step incor-           (PacoaX) facilitate selection. round PCR, ultramer P11 poration PBAD promoter gene. araC-PBAD region       primer P12 used amplify PacoaX P. aeruginosa genomic DNA, pBAD18 plasmid (17) amplified PCR using primers P1 P2            primer P13 ultramer P14 used amplify cat gene pKD3. primers ultramers listed Table S1 supplemental material),     second round PCR, products round mixed majority pKD4 (12) amplified PCR using primers P3 P4.         amplified using P11 P14. second-round complete PCR product PCR fragments digested ClaI/XhoI, ligated, transformed           used replace EccoaA gene start stop codon, generating <br />  3306        BALIBAR ET AL.                                                                                                                                    J. BACTERIOL. <br />  <br /> strain BW25113PBADcoaBC&#226;&#338;&#172;coaA::PacoaX. Transformation selection                      Cells passed times French press (Thermo) 1,100 lb/in2 performed described  final plating selection, 25             centrifuged 12,000 &#226;&#171;&#187; g 30 min 4&#194;&#176;C, supernatant loaded &#226;?&#174;g/ml chloramphenicol added. integrants confirmed PCR                    10 ml cobalt Talon resin preequilibrated wash buffer (50 <br /> sequencing.                                                                                  mM Tris [pH 8.0], 250 mM NaCl, 2 mM MgCl2, 1 mM TCEP). resin    Deletion coaBC E. coli. EccoaBC gene deleted, leaving              washed 20 column volumes wash buffer, cobalt-bound His6 codons, using linear PCR product generated amplifi-             protein eluted 3 column volumes elution buffer (50 mM Tris [pH 8.0], <br /> cation pKD4 P15 P16 ultramers, generating strain                           250 mM NaCl, 2 mM MgCl2, 1 mM TCEP, 500 mM imidazole, 10% glycerol). <br /> BW25113&#226;&#338;&#172;coaBC::kan. Transformation selection performed                        Eluted fractions containing protein concentrated &#226;&#172;&#402;2 ml using Amicon <br /> scribed  recovery final plating, 1 mM pantethine                Ultra 10,000-molecular-weight cutoff (10,000 MWCO) centrifugal device (Milli- included arabinose omitted media. integrants                     pore) injected FPLC size exclusion chromatography using buffer <br /> confirmed PCR sequencing.                                                             consisting 50 mM Tris [pH 8.0], 250 mM NaCl, 2 mM MgCl2, 1 mM TCEP, <br />    Deletion coaBC P. aeruginosa. Marked deletions PacoaBC gene                  10% glycerol. Fractions (2.5 ml  largest peak UV trace constructed PAO1 hyperpermeable Z61 mutant (5, 27, 51)                       analyzed purity 4 15% Tris-HCl sodium dodecyl sulfate-poly- <br /> (ATCC 35151) use pEX18Tc suicide vector (22), utilizes                 acrylamide gel electrophoresis (SDS-PAGE) gel (Bio-Rad), showing sacB-based selection method resolving cointegrants create double-cross-               single band correct molecular weight pooled. Protein concentrations mutants. PacoaBC gene 10 codons 10 codons                determined 595 nm using Bio-Rad protein reagent bovine serum replaced gentamicin resistance cassette. replacement deletes               albumin (BSA) standard. Purified enzymes stored aliquots majority coaBC minimizing chances hitting neighboring genes.             &#226;&#171;&#186;80&#194;&#176;C. accomplish  splicing overlap extension used fuse flanking                 Biochemical assays pantothenate kinase. Kinase activity measured regions PacoaBC gene introducing midpoint PmlI restriction site.              vitro using pyruvate kinase/lactate dehydrogenase-coupled assay round PCR, 600 bp upstream PacoaBC amplified                 formation <span id='am-5' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span> PanK reaction coupled formation P17 P18 primers 600 bp downstream PacoaBC amplified                  pyruvate pyruvate kinase, turn coupled oxidation P19 P20 primers. second round PCR, products round               NADH lactate dehydrogenase. optimizing PanK enzyme activity mixed amplified using P17 P20. PCR product                 regard buffer, pH, salt, dimethyl sulfoxide (DMSO), magnesium, DTT, deter- <br /> second round digested BamHI/KpnI ligated similarly di-                   gent, BSA, optimal reaction conditions established 100 mM HEPES <br /> gested pEX18Tc vector. new plasmid digested PmlI                      [pH 7.3], 50 mM KCl, 1 mM MgCl2, 0.01% CHAPS {3-[(3-cholamidopropyl)- <br /> ligated gentamicin cassette excised SmaI pUCGM (40).                      dimethylammonio]-1-propanesulfonate}, 20 &#226;?&#174;g/ml BSA, 1 mM DTT, 150 &#226;?&#174;M <br /> PCR used screen plasmids gentamicin cassette                     pantothenate pantethine, 400 &#226;?&#174;M ATP, 500 &#226;?&#174;M PEP, 200 &#226;?&#174;M NADH, 20 <br /> ligated orientation PacoaBC gene. resulting                           U/ml pyruvate kinase, 20 U/ml lactate dehydrogenase, 50 nM PanK. E. <br /> pEX18Tc&#226;&#338;&#172;PacoaBC::gent vector mobilized P. aeruginosa S17-1 E.                  coli PanK Km determinations, ATP fixed 1 mM pantothenate <br /> coli (41). Briefly, 500 &#226;?&#174;l S17-1 E. coli harboring pEX18Tc&#226;&#338;&#172;PacoaBC::gent                  pantethine adjusted 0 1,000 &#226;?&#174;M pantothenate grown overnight 37&#194;&#176;C LB 10 &#226;?&#174;g/ml gentamicin mixed 500 &#226;?&#174;l                 pantethine fixed 1 mM ATP adjusted 0 2,000 &#226;?&#174;M. PAO1 Z61 P. aeruginosa grown LB. Pelleted cells spread                    P. aeruginosa PanK Km determinations, ATP fixed 16 mM nickel-sized plane growth LB agar, incubated 37&#194;&#176;C 18 h,                pantothenate adjusted 0 40 &#226;?&#174;M pantothenate fixed restreaked PIA 100 &#226;?&#174;g/ml  PAO1) 10 &#226;?&#174;g/ml  Z61)                     80 &#226;?&#174;M ATP adjusted 0 16 mM. Reactions mon- <br /> gentamicin select merodiploids. resolve cointegrants create double-             itored measuring decrease fluorescence resulting coupled <br /> crossover knockouts, single colonies restreaked PIA 6% (wt/vol)                 oxidation NADH (excitation 340 nm emission 460 nm) use sucrose, 1 mM pantethine, 100 &#226;?&#174;g/ml  PAO1) 10 &#226;?&#174;g/ml                    SpectraMax Gemini XS microplate spectrofluorometer (Molecular Devices). <br /> Z61) gentamicin select loss sacB-containing vector backbone.                     convert relative fluorescence units (RFU) concentration values, molar <br />    experiments repeated introducing E. coli coaA gene repli-       fluorescence coefficient NADH experimentally determined 14.81 <br /> cative plasmid. EccoaA cloned versions pFlp2 plasmid (22).       RFU/&#226;?&#174;M. <br /> versions, flp gene removed, version sacB gene, renders cells      Growth curves. Overnight cultures desired strains centrifuged nonviable grown sucrose, removed (p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA),             resuspension cells 1 ml fresh LB yielded OD600 1.0. version (p&#226;&#338;&#172;flp2EccoaA). EccoaA amplified genomic DNA               normalized cultures diluted 10,000-fold fresh LB, 100 &#226;?&#174;l using P21 P22 primers. pFlp2 plasmid amplified using primer P23          aliquoted 96 U polystyrene plate (Falcon). Medium compo- primer P24 include sacB gene primer P25 omit sacB gene.          nents (pantothenate, pantethine, dephospho-CoA, CoA, arabinose, antibi- <br /> vector insert digested NcoI/XhoI, ligated, transformed TOP10          otics strain maintenance) added necessary aliquoting <br /> chemically competent cells. plasmids mobilized wild-type             single concentration required assay) column PAO1 membrane-permeable Z61 P. aeruginosa strains S17-1 E. coli use          plate followed 2-fold serial dilutions plate titration 100 &#226;?&#174;g/ml (PAO1) 10 &#226;?&#174;g/ml (Z61) carbenicillin maintenance. Subse-           component necessary. Plates incubated 37&#194;&#176;C shaking, quently, pEX18Tc&#226;&#338;&#172;PacoaBC::gent suicide vector introduced 4           OD600 readings taken minute using SpectraMax Plus 384 plate <br /> grounds cointegrants selected protocol described  Finally,         reader. experiments performed 3 times confirm observed <br /> obtain coaBC knockouts, strains&#226;&#8364;&#8221;PAO1pEX18Tc&#226;&#338;&#172;PacoaBC::gent/                     effects growth. <br /> p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA, PAO1pEX18Tc&#226;&#338;&#172;PacoaBC::gent/p&#226;&#338;&#172;flp2EccoaA, Z61pEX18Tc <br /> &#226;&#338;&#172;PacoaBC::gent/p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA, Z61pEX18Tc&#226;&#338;&#172;PacoaBC::gent/ <br /> p&#226;&#338;&#172;flp2EccoaA streaked media containing gentamicin  select retention <br />                                                                                                                  RESULTS DISCUSSION PacoaBC deletion), 6% sucrose  select loss pEX18Tc sacB containing <br /> backbone p&#226;&#338;&#172;flp2EccoaA), 1 mM pantethine  complement loss PacoaBC).                Construction analysis coaBC-regulated E. coli <br />    Cloning, expression, purification E. coli P. aeruginosa pantothenate <br /> kinase. E. coli coaA amplified using genomic DNA primers P26 P27. <br />                                                                                              strain. order investigate essentiality coaBC P. aeruginosa coaX amplified using genomic DNA primers P28 P29.                  effects depletion E. coli, regulated strain  resulting PCR fragments digested BamHI/SalI ligated                 structed expression coaBC placed similarly digested pTrcHis2B vector (Invitrogen). express C-terminal                  control arabinose-inducible PBAD promoter. growth <br /> His6-tagged protein, overnight cultures TOP10 cells harboring expression <br />                                                                                              resulting strain, BW25113PBADcoaBC, dependent <br /> plasmids diluted 100-fold 3 liters fresh NZYM broth containing 50 <br /> &#226;?&#174;g/ml carbenicillin. Cultures grown 37&#194;&#176;C OD600 reached 0.5                 presence arabinose, consistent (approximately 3 h), point cultures induced 1 mM IPTG,                    reported essentiality coaBC (13, 15). Furthermore, lag <br /> transferred 18&#194;&#176;C, grown additional 48 h.                                       time detectable growth scaled concentration    purification, 3-liter cultures harvested centrifugation resus-            arabinose growth media (Fig. 2A), increased lag <br /> pended 30 ml ice-cold lysis buffer (50 mM Tris [pH 8.0], 250 mM NaCl, 2 <br /> mM MgCl2, 100 &#226;?&#174;g/ml lysozyme, 1 mM TCEP, 0.1 mg/ml DNase  1 Complete <br />                                                                                              times observable 62 &#226;?&#174;M arabinose. fact, midpoint <br /> protease inhibitor cocktail tablet, 10% glycerol). P. aeruginosa PanK purifi-            exponential growth phase (OD600 &#226;&#171;&#189; 0.35) shifted cation, 20 mM imidazole included lysis wash   buffers.              340 min growth presence 125 &#226;?&#174;M arabinose  VOL. 193, 2011                                                                                     PANTETHINE BYPASS coaBC                  3307 <br />  <br />  <br />  <br />  <br />    FIG. 2. Arabinose-dependent growth BW25113PBADcoaBC.   Growth curves BW25113PBADcoaBC grown 37&#194;&#176;C LB 30 &#226;?&#174;g/ml <br /> kanamycin various concentrations arabinose: 125 &#226;?&#174;M (black filled squares), 63 &#226;?&#174;M (dark gray filled diamonds), 16 &#226;?&#174;M (medium gray filled <br /> triangles), 8 &#226;?&#174;M (light gray filled circles), 4 &#226;?&#174;M (black unfilled squares), 2 &#226;?&#174;M (dark gray unfilled diamonds), 1 &#226;?&#174;M (medium gray unfilled <br /> triangles), 0 &#226;?&#174;M  arabinose; light gray unfilled circles). (B) described panel 1 mM pantothenate conditions. <br /> (C) described panel 1 mM pantethine conditions. (D) described panel 1 mM <br /> dephospho-CoA conditions. (E) described panel 1 mM CoA conditions. figure clarity, curves illustrative single replicate, experiments performed 3 times. <br />  <br />  <br />  <br />  <br /> 380, 540, 600, 680, 820, 960 min growth presence             PPCDC defects studies Arabidopsis 63, 16, 8, 4, 2, 1 &#226;?&#174;M arabinose, respectively (Fig. 2A).             thaliana. plant possesses genes encode PPCDC, <br />   Nutrient supplementation. Given CoA biosynthesis                HAL3A HAL3B. plants containing homozygous <br /> primary metabolic process leading formation small-              transfer DNA (T-DNA)-disrupted alleles hal3a-1 molecule cofactor, determine                  hal3b viable, hal3a-1 hal3b double mutant embryoni- <br /> supplementing media substrates, intermediates,                     cally lethal, indicating redundant essential function products pathway rescue deficiencies                 genes. Interestingly, hal3b individuals <br /> coaBC expression. Precedents chemical complementation                   heterozygous HAL3A behave similarly wild-type plants, <br />  3308     BALIBAR ET AL.                                                                                                  J. BACTERIOL. <br />  <br />  <br /> hal3a-1 individuals heterozygous HAL3B severely im-         arabinose-inducible PBAD promoter. coaBC, <br /> paired seedling establishment (38), result supports       coaD confirmed essential, growth notion HAL3A plays dominant role respect          BW25113PBADcoaD dependent coaD expression, HAL3B, suggested differences transcript levels (14).        lag time growth scaled concentration severe defect rescued supplementation          arabinose media  Fig. S1A supplemental <br /> pantethine, restoring normal seedling establishment.                 material). seen Fig. S1B E supplemental <br />    investigate similar phenomenon occur        material, addition 1 mM pantothenate, pantethine, dephos- <br /> E. coli, growth phenotype BW25113PBADcoaBC               pho-CoA, CoA unable rescue arabinose-regu- <br /> function arabinose concentration measured pres-        lated phenotype BW25113PBADcoaD. slight dif- <br /> ence 1 mM pantothenate, pantethine, dephospho-CoA,             ferences overall growth observed CoA. expected, addition pantothenate, primary          exogenous metabolites added, especially low concen- <br /> precursor CoA pathway, unable rescue arabi-        trations arabinose, addition nutrients did nose-dependent growth phenotype BW25113PBADcoaBC                  change fact lag time growth inversely <br /> (Fig. 2B). result consistent fact PPCS/        proportional arabinose present PPCDC acts downstream pantothenate formation CoA               absence arabinose, growth observed. result <br /> biosynthesis implies increasing concentration        confirms idea increasing upstream pre- substrate overcome deficiency coaBC expres-          cursors pantothenate overcome deficiencies sion. noted lag time seen concen-      downstream enzyme activity dephospho-CoA trations arabinose increased adding pantothenate,            CoA entering cells, downstream <br /> suggesting E. coli subject additional stress imposed   PPAT activity. importantly, data demonstrate exogenous addition metabolite. Nonetheless,          pantethine bypass expression coaD, indicat- <br /> apparent change lag time reflects difference          ing pantethine ultimately converted PPAT sub- <br /> growth conditions does alter conclusion pan-        strate phosphopantetheine PPCS/PPCDC-independent <br /> tothenate unable bypass need endogenous PPCS/          mechanism. <br /> PPCDC function.  addition pantethine did deregulate             Despite inability cross cytoplasmic membrane E. <br /> arabinose-dependent growth BW25113PBADcoaBC.               coli, extracellular pantothenate used precursor seen Fig. 2C, presence 1 mM pantethine, lag      CoA biosynthesis active uptake pantothenate <br /> times growth equivalent presence absence       permease encoded panF (24, 45, 47). order determine <br /> arabinose. Finally, addition dephospho-CoA CoA,            uptake pantethine dependent panF, <br /> penultimate ultimate products pathway, did            knockouts constructed BW25113PBADcoaBC  <br /> rescue increased lag time growth underexpres-         ground. seen Fig. S2 S3 supplemental <br /> sion coaBC (Fig. 2D E). dephospho-CoA            material, arabinose-dependent growth deregulated ad- <br /> CoA formation downstream respect PPCS/PPCDC              dition pantethine manner similar seen com- <br /> function, highly charged phosphate-containing molecules        parisons parent BW25113PBADcoaBC mutant unable enter cells. case pantothenate,     BW25113PBADcoaBC&#226;&#338;&#172;panF::cat. deregulation observ- <br /> overall increase lag time bacterial cultures regardless   able decrease lag time different concentrations arabinose concentration observed           arabinose. Furthermore, degrees deregulation seen <br /> phospho-CoA CoA added medium, likely            increasing concentrations pantethine similar change overall growth conditions. Additionally,       strains  Fig. S2B E S3B E supple- slight growth observed 1 mM dephospho-          mental material). data indicate ability CoA absence arabinose (Fig. 2D),          pantethine enter E. coli dependent presence caused breakdown dephospho-CoA pante-              panF. finding consistent studies azido- <br /> theine time presence small             pantetheine analog capable entering cells pantetheine pantethine sample, purchased            processed ultimately <span id='am-89' about='rdfs:label' typeof='owl:Thing'>label</span> endogenous acyl carrier proteins <br /> dephospho-CoA 90% pure. summary, regardless           panF-independent manner (34). general changes growth chemical supplementation             Constructing coaBC knockouts. rule possibility caused, primary phenotype arabinose-depen-           residual PPCS/PPCDC activity BW25113PBADcoaBC <br /> dent growth increased lag time caused regulating          responsible cell viability unknown regulatory <br /> expression coaBC rescued presence              mechanisms triggered addition pantethine, deletion <br /> pantethine.                                                          coaBC attempted.  pantethine-supple- <br />    Pantethine dimeric thiol-oxidized version PPCS/       mented media used, mutants obtained PPCDC product missing 4&#226;&#172;&#732;-phosphate appended           knockout coaBC gene. BW25113&#226;&#338;&#172;coaBC::kan <br /> PanK. results showing complement deficiencies        strain, identity confirmed sequencing, coaBC expression indicates enter cells       viable media lacking exogenous pantethine (Fig. 3). Panto- <br /> bypass need PPCS/PPCDC activity. demonstrate              thenate, dephospho-CoA, CoA used sur- pantethine converted CoA alter-          rogates restoring growth, plating performed multiple <br /> native pathway, chemical complementation experiments            times 1012 cells did yield suppressors repeated strain coaD, encodes PPAT           pantethine auxotrophy. data consistent <br /> enzyme catalyzes step subsequent PPCS/PPCDC              previous studies indicating CoA trans- <br /> formation CoA biosynthesis, placed control       ported bacterial cell envelope (23, 25, 42), alter <br />  VOL. 193, 2011                                                                                      PANTETHINE BYPASS coaBC              3309 <br />  <br />  <br />                                                                                EcPanK active readily phosphorylated pan- <br />                                                                                tothenate pantethine. <br />                                                                                   initial activities established, set Michae- <br />                                                                                lis-Menten parameters kinase reaction measured <br />                                                                                optimized reaction conditions. data Table 1  <br />                                                                                dicate EcPanK Km values ATP 2-fold <br />                                                                                range using pantothenate pantethine substrate <br />                                                                                Km natural pantothenate substrate 2-fold <br />                                                                                greater Km surrogate pantethine. data <br />    FIG. 3. BW25113&#226;&#338;&#172;coaBC::kan auxotrophic pantethine. <br />   BW25113&#226;&#338;&#172;coaBC::kan grown LB agar 30 &#226;?&#174;g/ml kanamy- <br />                                                                                demonstrate EcPanK phosphorylate pantethine cin. (B) BW25113&#226;&#338;&#172;coaBC::kan grown LB agar 30 &#226;?&#174;g/ml ka-                 specific activity conversion 2-fold greater <br /> namycin 1 mM pantethine.                                                   natural pantothenate substrate. <br />                                                                                   promiscuity type II PanKs exploited                                                                                use pantothenamide antimetabolites act notion pantothenate advanced precursor                 substrates PanK downstream enzymes. CoA permeate cells (24, 42, 45).                                      molecules utilized labels carrier proteins    Biochemical experiments using PanK. use pante-                       vivo (32) antimicrobial agents;  thine chemically complement coaBC deficiency coaD-                   debate mechanism action panto- <br /> dependent fashion suggests E. coli capable converting               thenamide class antimetabolites operates direct inhibi- molecule PPCDC product phosphopantetheine.                         tion CoA biosynthetic enzymes (11, 44), inhibition transformation accomplished directly                      fatty acid biosynthesis accumulation nonfunc- reduction disulfide free thiol                       tional acyl carrier proteins (29, 49), inhibition CoA- <br /> phosphorylation 4&#226;&#172;&#732; hydroxyl (Fig. 1).                      acetyl-CoA-utilizing enzymes (42). Given promiscuity, <br /> general reducing environment cytoplasm suffice                    surprising E. coli PanK demonstrated accomplishing disulfide reduction, phosphorylation                     efficiently utilize pantethine substrate biochemical <br /> event likely require specific enzyme.                      assays. known time specific enzyme preparations                        parallel comparison, type III PanK enzyme purified cellular extracts phosphorylate                       isolated cloning expressing coaX gene pantothenate pantethine (1, 31), ultimately                       clinically relevant pathogen Pseudomonas aeruginosa. Possess- <br /> activity attributed protein involved                                                                                ing impermeable outer membrane diverse range <br /> step CoA biosynthesis, PanK. types                                                                                efflux pumps (35, 36), P. aeruginosa inherently recalcitrant <br /> bacterial PanKs differ respect structure, sub- <br />                                                                                antimicrobial intervention. particular given <br /> strate affinity, feedback regulation. type PanKs <br />                                                                                emerged clinic major cause nosocomial type modulated feedback inhibition                                                                                infections immunocompromised patients known <br /> CoA exemplified prototypical member  <br /> coded coaA E. coli (46). Type II PanKs related                    cause declining lung function cystic fibrosis pa- eukaryotic isoforms PanK exemplified                     tients (16, 33). <br /> product coaA gene Staphylococcus aureus (29).                           P. aeruginosa PanK (PaPanK) catalyzed robust <br /> Finally, type III PanKs divergent                    phosphorylation pantothenate Michaelis-Menten pa- high Km values substrates, exemplified                  rameters similar type III PanK H. product coaX gene Helicobacter pylori (10).                   pylori (10) (Table 1), unable utilize pantethine    coaA gene, encoding type E. coli PanK (EcPanK),                   substrate. result held true despite exploration cloned C-terminal His6-tagged vector pTrcHis2B.                   sets conditions high concentrations enzyme <br /> Expression TOP10 E. coli 18&#194;&#176;C 1 mM IPTG induc-                      substrate used large amounts reducing <br /> tion yielded 30 40 mg/liter purification homogeneity               agent supplied form free thiol pantetheine. using cobalt affinity gel filtration chromatography tan-                inability PaPanK utilize pantethine consistent dem. Initial activity titration assays performed using pyruvate              inability H. pylori PanK accept N-pentylpantothena- <br /> kinase/lactate dehydrogenase-coupled demonstrated                       mide alternative substrate (10). <br />  <br />  <br />                                                            TABLE 1. Kinetic properties PanKa <br /> Enzyme                Substrate adjusted             Substrate held constant        Km (mM)                kcat (s&#226;&#171;&#186;1)          kcat/Km (mM&#226;&#171;&#186;1 s&#226;&#171;&#186;1) <br />  <br /> EcPanK                  ATP                              Pantothenate            0.093 &#226;&#171;&#190; 0.008           0.77 &#226;&#171;&#190; 0.02              8.33 &#226;&#171;&#190; 0.75 <br /> EcPanK                  ATP                              Pantethine               0.15 &#226;&#171;&#190; 0.02            1.48 &#226;&#171;&#190; 0.06              9.79 &#226;&#171;&#190; 1.36 <br /> EcPanK                  Pantothenate                     ATP                      0.66 &#226;&#171;&#190; 0.08            1.11 &#226;&#171;&#190; 0.07              1.68 &#226;&#171;&#190; 0.23 <br /> EcPanK                  Pantethine                       ATP                      0.30 &#226;&#171;&#190; 0.04            1.05 &#226;&#171;&#190; 0.06              3.46 &#226;&#171;&#190; 0.50 <br /> PaPanK                  ATP                              Pantothenate               7.3 &#226;&#171;&#190; 0.9           0.090 &#226;&#171;&#190; 0.005            0.012 &#226;&#171;&#190; 0.002 <br /> PaPanK                  Pantothenate                     ATP                     0.005 &#226;&#171;&#190; 0.001           0.22 &#226;&#171;&#190; 0.01              49.6 &#226;&#171;&#190; 10.2 <br />       &#226;&#171;&#190; values represent standard errors calculated using GraFit 5 software. <br />  3310     BALIBAR ET AL.                                                                                                        J. BACTERIOL. <br />  <br />  <br />    Pantethine complementation dependence PanK vivo. inability PaPanK phosphorylate pantethine fur- <br /> ther explored vivo P. aeruginosa attempt create coaBC knockout. Initially, deletion performed wild-type PAO1 background;  single integrants resolved double-crossover knockout mutants. Pseudomonas known possessing especially imperme- <br /> able outer membrane (5, 18), possible pantethine unable penetrate PAO1.  knockout experi- <br /> ments repeated hyperpermeable Pseudomonas <br /> aeruginosa strain Z61 (ATCC 35151) (5, 27, 51). Despite lower permeability threshold, coaBC knockout mutants obtained. <br />    result showing PaPanK unable phosphorylate <br /> pantethine unable form PPAT substrate <br /> phosphopantetheine suggested inability delete coaBC P. aeruginosa inability generate pantethine auxotroph lack cellular <br /> penetration pantethine inability metabolize molecule. case, introducing E. coli coaA gene (EccoaA) P. aeruginosa allow utilize pantethine bypass PPCS/ <br />                                                                           FIG. 4. EccoaA required pantethine bypass coaBC P. <br /> PPCDC function, rendering coaBC deletion viable.                aeruginosa.   Z61pEX18Tc&#226;&#338;&#172;PacoaBC::gent/p&#226;&#338;&#172;flp2EccoaA versely, coaA gene function E. coli paramount        Z61pEX18Tc&#226;&#338;&#172;PacoaBC::gent/p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA plated PIA incorporating pantethine metabolism CoA, replacing              10 &#226;?&#174;g/ml gentamicin, 10 &#226;?&#174;g/ml carbenicillin, 1 mM pantethine, 6% coaX gene P. aeruginosa (PacoaX) render            (wt/vol) sucrose. (B) Z61&#226;&#338;&#172;coaBC::gent/p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA grown                                                                         LB agar 10 &#226;?&#174;g/ml gentamicin. (C) described panel B pantethine bypass coaBC ineffective. <br />                                                                         plates contained 1 mM pantethine. <br />    generate P. aeruginosa pantethine auxotroph, E. <br /> coli coaA introduced wild-type PAO1 mem- <br /> brane-permeable Z61 P. aeruginosa strains replicative plasmids. p&#226;&#338;&#172;flp2EccoaA plasmid contains                 plementation coaBC deficiency, P. aeruginosa, sacB gene, renders cells nonviable grown              endogenous PanK encoded coaX unable catalyze phos- <br /> sucrose, p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA plasmid does                 phorylation. Interestingly, single integrants PAO1 strain  <br /> contain sacB gene. use parallel plasmids               taining p&#226;&#338;&#172;flp2EccoaA p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA plasmid demonstrate essentiality EccoaA pante-                    resolved double-crossover knockout mutants, thine complementation coaBC deficiency.                   pantethine supplied concentrations high 5 mM process generating knockouts, single integrants               media. indicates permeability barrier PAO1 pEX18Tc&#226;&#338;&#172;PacoaBC::gent suicide vector forced                   high allow pantethine enter. <br /> solve double-crossover knockouts use gen-                confirm results, reciprocal pantethine com- <br /> tamicin selects retention resistance cassette incor-         plementation experiment conducted using E. coli. Using porated target insert use sucrose selects           BW25113PBADcoaBC background, replaced coaA coaX loss plasmid backbone contains sacB gene.              gene P. aeruginosa. growth resulting strain, <br /> Concurrently, use sucrose selection selects         BW25113PBADcoaBC&#226;&#338;&#172;coaA::PacoaX, assayed using various loss p&#226;&#338;&#172;flp2EccoaA plasmid select               concentrations arabinose, addition loss p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA plasmid. EcPanK               1 mM pantethine. shown Fig. 5, unlike parental <br /> responsible ability utilize pantethine,                 BW25113PBADcoaBC strain, addition pantethine  <br /> bypassing PPCD/PPCDC activity, cells p&#226;&#338;&#172;flp2&#226;&#338;&#172;sac               passes arabinose dependence (Fig. 2C), BW25113PBADcoaBC <br /> BEccoaA viable selection sucrose,               &#226;&#338;&#172;coaA::PacoaX, addition pantethine effect  retain copy EccoaA.                                creased lag time observed decreasing concentrations arabi- <br />    evidenced Fig. 4A, Z61pEX18Tc&#226;&#338;&#172;PacoaBC::                   nose. results indicate  EccoaA replaced gent/p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA bacteria viable selection          PacoaX, E. coli longer metabolize pantethine phospho- <br /> knockouts gentamicin, sucrose, pantethine performed.         pantetheine bypass deficiencies coaBC. suggests Z61pEX18Tc&#226;&#338;&#172;PacoaBC::gent/p&#226;&#338;&#172;flp2EccoaA strain failed              PanK enzyme E. coli phosphorylate pantethine. <br /> grow, indicating EccoaA required growth pante-              results study differ somewhat findings dem- <br /> thine. Colonies selection isolated sequenced         onstrating pantethine rescue pantothenate-kinase-associ- confirm double-crossover knockouts                 ated neurodegeneration (PKAN) Drosophila studies. coaBC. seen Fig. 4B C, resulting                   neurodegenerative hereditary disease Hallervorden-Spatz syn- <br /> Z61&#226;&#338;&#172;coaBC::gent/p&#226;&#338;&#172;flp2&#226;&#338;&#172;sacBEccoaA strain viable         drome, caused mutations human PANK2 (50), presence pantethine. results indicate PanK phosphor-      successfully recapitulated Drosophila analogous <br /> ylation pantethine necessary sufficient pantethine com-   mutations dPANK2/fbl (37, 48). mutant flies sig- <br />  VOL. 193, 2011                                                                                        PANTETHINE BYPASS coaBC                         3311 <br />  <br />                                                                                                       ACKNOWLEDGMENTS <br />                                                                               thank K. Poole (Queen&#226;&#8364;&#8482;s University) H. Schweizer (Uni- <br />                                                                             versity Colorado) strains plasmids. thank  Jones, X. <br />                                                                             Shen, C. Dean (Novartis) helpful discussions. <br />  <br />                                                                                                              REFERENCES <br />                                                                              1. Abiko, Y. 1967. Investigations pantothenic acid related com- <br />                                                                                 pounds. IX. Biochemical studies. 4. Separation substrate specificity                                                                                 pantothenate kinase phosphopantothenoylcysteine synthetase. <br />                                                                                 J. Biochem. 61:290&#226;&#8364;&#8220;299. <br />                                                                              2. Abiko, Y. 1967. Investigations pantothenic acid related com- <br />                                                                                 pounds. X. Biochemical studies. 5. Purification substrate specificity                                                                                 phosphopantothenoylcysteine decarboxylase rat liver. J. Biochem. 61: <br />                                                                                 300&#226;&#8364;&#8220;308. <br />                                                                              3. Abiko, Y., M. Tomikawa, M. Shimizu. 1968. studies phos- <br />                                                                                 phopantothenoylcysteine synthetase. J. Biochem. 64:115&#226;&#8364;&#8220;117. <br />                                                                              4. Anderson, M. S., C. E. Bulawa, C. R. Raetz. 1985. biosynthesis                                                                                 gram-negative endotoxin. Formation lipid precursors UDP- <br />                                                                                 GlcNAc extracts Escherichia coli. J. Biol. Chem. 260:15536&#226;&#8364;&#8220;15541. <br />                                                                              5. Angus, B. L.,  M. Carey, D.  Caron,  M. Kropinski, R. E. Hancock. <br />                                                                                 1982. Outer membrane permeability Pseudomonas aeruginosa: compari- <br />                                                                                 son wild-type antibiotic-supersusceptible mutant. Antimicrob. <br />                                                                                 Agents Chemother. 21:299&#226;&#8364;&#8220;309. <br />                                                                              6. Baba, T., et al. 2006. Construction Escherichia coli K-12 frame, single- <br />                                                                                 gene knockout mutants: Keio collection. Mol. Syst. Biol. 2:1&#226;&#8364;&#8220;11. <br />                                                                              7. Balibar, C. J., X. Shen, J. Tao. 2009. mevalonate pathway                                                                                 Staphylococcus aureus. J. Bacteriol. 191:851&#226;&#8364;&#8220;861. <br />                                                                              8. Begley, T. P., C. Kinsland, E. Strauss. 2001. biosynthesis coen- <br />                                                                                 zyme bacteria. Vitam. Horm. 61:157&#226;&#8364;&#8220;171. <br />                                                                              9. Beveridge, T. J. 1981. Ultrastructure, chemistry, function bacterial <br />                                                                                 wall. Int. Rev. Cytol. 72:229&#226;&#8364;&#8220;317. <br />                                                                             10. Brand, L. , E. Strauss. 2005. Characterization new pantothenate <br />                                                                                 kinase isoform Helicobacter pylori. J. Biol. Chem. 280:20185&#226;&#8364;&#8220;20188. <br />    FIG. 5. Arabinose-dependent growth BW25113PBADcoaB                    11. Choudhry,  E., et al. 2003. Inhibitors pantothenate kinase: novel anti- <br />                                                                                 biotics staphylococcal infections. Antimicrob. Agents Chemother. 47: <br /> C&#226;&#338;&#172;coaA::PacoaX bypassed pantethine.   Growth                   2051&#226;&#8364;&#8220;2055. <br /> curves BW25113PBADcoaBC&#226;&#338;&#172;coaA::PacoaX grown 37&#194;&#176;C LB                 12. Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo- 30 &#226;?&#174;g/ml kanamycin, 25 &#226;?&#174;g/ml chloramphenicol, various                  somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> concentrations arabinose: 500 &#226;?&#174;M (black filled squares), 63 &#226;?&#174;M               Sci. U. S.  97:6640&#226;&#8364;&#8220;6645. <br /> (dark gray filled diamonds), 31 &#226;?&#174;M (medium gray filled triangles), 16       13. Daugherty, M., et al. 2002. Complete reconstitution human coenzyme <br /> &#226;?&#174;M (light gray filled circles), 8 &#226;?&#174;M (black unfilled squares), 4 &#226;?&#174;M             biosynthetic pathway comparative genomics. J. Biol. Chem. 277:21431&#226;&#8364;&#8220; <br /> (dark gray unfilled diamonds), 2 &#226;?&#174;M (medium gray unfilled triangles),           21439. <br /> 1 &#226;?&#174;M (light gray unfilled circles), 0 &#226;?&#174;M  arabinose; dark gray       14. Espinosa-Ruiz, , J. M. Belles, R. Serrano, F.  Culianez-MacIa. 1999. <br />                                                                                 Arabidopsis thaliana AtHAL3: flavoprotein related salt osmotic <br /> black stars). (B) described panel 1 mM                   tolerance plant growth. Plant J. 20:529&#226;&#8364;&#8220;539. <br /> pantethine conditions. figure clarity, curves illustra-   15. Gerdes, S. Y., et al. 2002. genetic footprinting antimicrobial drug <br /> tive single replicate, experiments performed          targets: examples cofactor biosynthetic pathways. J. Bacteriol. 184:4555&#226;&#8364;&#8220; <br /> 3 times.                                                                        4572. <br />                                                                             16. Govan, J. R., V. Deretic. 1996. Microbial pathogenesis cystic fibrosis: <br />                                                                                 mucoid Pseudomonas aeruginosa Burkholderia cepacia. Microbiol. Rev. <br />                                                                                 60:539&#226;&#8364;&#8220;574. <br />                                                                             17. Guzman, L. M., D. Belin, M. J. Carson, J. Beckwith. 1995. Tight regu- <br />                                                                                 lation, modulation, high-level expression vectors containing nificantly reduced levels CoA, resulting impaired mito-                   arabinose PBAD promoter. J. Bacteriol. 177:4121&#226;&#8364;&#8220;4130. <br /> chondrial function, increased oxidative damage proteins,                 18. Hancock, R. E., H. Nikaido. 1978. Outer membranes gram-negative <br /> defective locomotor abilities, decreased life span.                  bacteria. XIX. Isolation Pseudomonas aeruginosa PAO1 use                                                                                 reconstitution <span id='am-54' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> permeability barrier. J. Bacteriol. 136: phenotypes rescued degree supple-                     381&#226;&#8364;&#8220;390. <br /> mentation food pantethine (37), indicating                     19. Harrington, C. R., J. Baddiley. 1985. Biosynthesis wall teichoic acids <br />                                                                                 Staphylococcus aureus H, Micrococcus varians Bacillus subtilis W23. <br /> pantethine utilized substrate generating CoA.                 Involvement lipid intermediates containing disaccharide N-acetylman- <br /> Interestingly, results suggest Drosophila,                        nosaminyl N-acetylglucosamine. Eur. J. Biochem. 153:639&#226;&#8364;&#8220;645. <br /> pantethine utilized PanK-independent manner,                    20. Heath, R. J., S. W. White, C. O. Rock. 2001. Lipid biosynthesis target <br />                                                                                 antibacterial agents. Prog. Lipid Res. 40:467&#226;&#8364;&#8220;497. contrast findings study,                   21. Ho, S. N., H. D. Hunt, R. M. Horton, J. K. Pullen, L. R. Pease. 1989. <br /> pantethine utilization absolutely dependent                  Site-directed mutagenesis overlap extension using polymerase chain <br />                                                                                 reaction. Gene 77:51&#226;&#8364;&#8220;59. <br /> PanK activity. alternate kinase pathway                  22. Hoang, T. T., R. R. Karkhoff-Schweizer,  J. Kutchma, H. P. Schweizer. <br /> pantethine metabolism Drosophila  ul-                    1998. broad-host-range Flp-FRT recombination site-specific <br /> timately, pantethine likely converted inter-               excision chromosomally-located DNA sequences: application isolation <br />                                                                                 unmarked Pseudomonas aeruginosa mutants. Gene 212:77&#226;&#8364;&#8220;86. <br /> mediate 4&#226;&#172;&#732;-phosphopantetheine, pantethine rescue                23. Jackowski, S. 1996. Biosynthesis pantothenic acid coenzyme  p. <br /> decreased cell counts dPPCS-depleted cells                   687&#226;&#8364;&#8220;694. F. C. Neidhardt, R. Curtiss, C.  Gross, J. L. Ingraham, E. C. C. <br />                                                                                 Lin, K. B. Low, B. Magasanik, W. S. Reznikoff, M. Riley, M. Schaechter, dPPAT-depleted cells (37). consistent results                  H. E. Umbarger (ed.), Escherichia coli Salmonella typhimurium: cellular study, demonstrate pantethine supplemen-                     molecular biology, vol. 1. American Society Microbiology, Washing- <br /> tation bypass arabinose-regulated growth                        ton, DC. <br />                                                                             24. Jackowski, S., J. H. Alix. 1990. Cloning, sequence, expression coaBC strain (BW25113PBADcoaBC) coaD strain                           pantothenate permease (panF) gene Escherichia coli. J. Bacteriol. 172: <br /> (BW25113PBADcoaD).                                                              3842&#226;&#8364;&#8220;3848. <br />  3312       BALIBAR ET AL.                                                                                                                              J. BACTERIOL. <br />  <br /> 25. Jackowski, S., C. O. Rock. 1984. Metabolism 4&#226;&#172;&#732;-phosphopantetheine              manual, 3rd ed. Cold Spring Harbor Laboratory Press, Cold Spring Har- <br />     Escherichia coli. J. Bacteriol. 158:115&#226;&#8364;&#8220;120.                                       bor, NY. <br /> 26. Kotnik, M., P. S. Anderluh,  Prezelj. 2007. Development novel          40.   Schweizer, H. D. 1993. Small broad-host-range gentamycin resistance gene <br />     inhibitors targeting intracellular steps peptidoglycan biosynthesis. Curr.         cassettes site-specific insertion deletion mutagenesis. Biotechniques <br />     Pharm. Des. 13:2283&#226;&#8364;&#8220;2309.                                                             15:831&#226;&#8364;&#8220;834. <br /> 27. Kropinski,  M., J. Kuzio, B. L. Angus, R. E. Hancock. 1982. Chemical      41.   Simon, R., U. Priefer,  Puehler. 1983. broad host range mobilization <br />     chromatographic analysis <span id='am-27' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-28' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="LPS" datatype="xsd:string"></span>lipopolysaccharide</span> antibiotic-su-             vivo genetic engineering: transposon mutagenesis Gram- <br />     persusceptible mutant Pseudomonas aeruginosa. Antimicrob. Agents Che-              negative bacteria. Nat. Biotechnol. 1:784&#226;&#8364;&#8220;791. <br />     mother. 21:310&#226;&#8364;&#8220;319.                                                             42.   Strauss, E., T. P. Begley. 2002. antibiotic activity N-pentylpan- <br /> 28. Kupke, T., P. Hernandez-Acosta, F.  Culianez-Macia. 2003. 4&#226;&#172;&#732;-Phos-              tothenamide results conversion ethyldethia-coenzyme  coen- <br />     phopantetheine coenzyme biosynthesis plants. J. Biol. Chem. 278:             zyme antimetabolite. J. Biol. Chem. 277:48205&#226;&#8364;&#8220;48209. <br />     38229&#226;&#8364;&#8220;38237.                                                                    43.   Strauss, E., C. Kinsland, Y. Ge, F. W. McLafferty, T. P. Begley. 2001. <br /> 29. Leonardi, R., et al. 2005. pantothenate kinase Staphylococcus aureus           Phosphopantothenoylcysteine synthetase Escherichia coli. Identifica- <br />     refractory feedback regulation coenzyme  J. Biol. Chem. 280:3314&#226;&#8364;&#8220;             tion characterization unidentified coenzyme biosynthetic <br />     3322.                                                                                 enzyme bacteria. J. Biol. Chem. 276:13513&#226;&#8364;&#8220;13516. <br /> 30. Leonardi, R., Y. M. Zhang, C. O. Rock, S. Jackowski. 2005. Coenzyme         44.   Thomas, J., J. E. Cronan. 2010. Antibacterial activity N-pentylpan- <br />      action. Prog. Lipid Res. 44:125&#226;&#8364;&#8220;153.                                       tothenamide inhibition coenzyme synthesis. Antimicrob. <br /> 31. Levintow, L., G. Novelli. 1954. synthesis coenzyme                  Agents Chemother. 54:1374&#226;&#8364;&#8220;1377. <br />     pantetheine: preparation properties pantetheine kinase. J. Biol. <br />                                                                                     45.   Vallari, D. S., C. O. Rock. 1985. Isolation characterization     Chem. 207:761&#226;&#8364;&#8220;765. <br />                                                                                           Escherichia coli pantothenate permease (panF) mutants. J. Bacteriol. 164: <br /> 32. Meier, J. L.,  C. Mercer, H. J. Rivera, M. D. Burkart. 2006. Synthesis <br />                                                                                           136&#226;&#8364;&#8220;142. <br />     evaluation bioorthogonal pantetheine analogues vivo protein <br />     modification. J.  Chem. Soc.  <br /> </body></html>