<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body><span id='am-54' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>Uracil</span>-ContainingDNAin Drosophila:Stability,Stage-Specific <br /> Accumulation, Developmental Involvement <br /> Villo&#203;? Muha1., Andra&#194;&#180;s Horva&#194;&#180;th1., Ange&#194;&#180;la Be&#194;&#180;ke&#194;&#180;si1, Ma&#194;&#180;ria Puka&#194;&#180;ncsik1, Barbara Hodoscsek1, <br /> Ga&#194;&#180;bor Mere&#194;&#180;nyi1, Gergely Ro&#194;&#180;na1, Ju&#194;&#180;lia Batki1, Istva&#194;&#180;n Kiss2, Ferenc Jankovics2, Pe&#194;&#180;ter Vilmos2, <br /> Miklo&#194;&#180;s Erde&#194;&#180;lyi2, Bea&#194;&#180;ta G. Ve&#194;&#180;rtessy1,3* <br /> 1 Institute Enzymology, Research Center Natural Sciences, Hungarian Academy Science, Budapest, Hungary, 2 Institute Genetics, Biological Research Centre <br /> (BRC), Hungarian Academy Science, Szeged, Hungary, 3 Department Applied Biotechnology Food Sciences, University Technology Economics, Budapest, <br /> Hungary <br />  <br />  <br />  <br />      Abstract <br />      Base-excision repair control <span id='am-26' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span> pools safe-guard permanent uracil accumulation DNA relying      key enzymes: uracil&#226;&#8364;&#8220;DNA glycosylase dUTPase. Lack major uracil&#226;&#8364;&#8220;DNA glycosylase UNG gene fruit <br />      fly genome dUTPase fruit fly larvae prompted hypotheses  uracil accumulate Drosophila genomic <br />      DNA tolerated, ii) accumulation affect development.  Drosophila <br />      melanogaster tolerates high levels uracil DNA; ii) DNA correctly interpreted cell culture embryo; iii) <br />      physiological spatio-temporal control, DNA fruit fly larvae, pupae, imago contain greatly elevated levels      uracil (200&#226;&#8364;&#8220;2,000 uracil/million bases, quantified using novel real-time PCR&#226;&#8364;&#8220;based assay). Uracil accumulated genomic <br />      DNA larval tissues larval development, DNA imaginal tissues contains uracil.      pupation metamorphosis, uracil content DNA significantly decreased. propose observed <br />      developmental pattern uracil&#226;&#8364;&#8220;DNA lack key repair enzyme UNG Drosophila genome <br />      regulation dUTPase larval tissues. agreement, dUTPase silencing increases      uracil content DNA imaginal tissues induces strong lethality early pupal stages, indicating tolerance      highly uracil-substituted DNA stage-specific. Silencing dUTPase perturbs physiological pattern uracil&#226;&#8364;&#8220;DNA <br />      accumulation Drosophila leads strongly lethal phenotype early pupal stages. findings suggest novel <br />      role uracil-containing DNA Drosophila development metamorphosis present novel example      developmental effects dUTPase silencing multicellular eukaryotes. Importantly, lack UNG gene      available genomes Holometabola insects, indicating potentially general tolerance developmental role      uracil&#226;&#8364;&#8220;DNA evolutionary clade. <br />  <br />   Citation: Muha V, Horva&#194;&#180;th  Be&#194;&#180;ke&#194;&#180;si  Puka&#194;&#180;ncsik M, Hodoscsek B, et al. (2012) Uracil-Containing DNA Drosophila: Stability, Stage-Specific Accumulation,   Developmental Involvement. PLoS Genet 8(6): e1002738. doi:10.1371/journal.pgen.1002738 <br />   Editor: R. Scott Hawley, Stowers Institute Medical Research, United States America <br />   Received August 22, 2011; Accepted April 13, 2012; Published June 7, 2012 <br />   Copyright: &#195;&#376; 2012 Muha et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Hungarian Scientific Research Fund (OTKA K68229, NK84008, CK78646, K75774, NKTH-OTKA H07-BEL74200),   National Innovation Office, New Hungary Development Plan (TA&#194;&#180;MOP-4.2.1/B-09/1/KMR-2010-0002), Howard Hughes Medical Institutes #55005628   #55000342, Alexander von Humboldt-Stiftung, Germany. funders role study design, data collection analysis, decision publish   preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: vertessy@enzim.hu (BGV) <br />   . authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                           DNA, formed absence dUTPase, intolerable cells. Uracil&#226;&#8364;&#8220; <br />                                                                                        DNA glycosylases represent superfamily involves enzymes <br />    wild-type organisms, notably excepting rare bacterio-                       specialized functions. superfamily members, <br /> phages thymineless DNA genomes [1,2], uracil DNA                            UNG reported abundant possesses <br /> thought occur transiently low frequency (,20/                      highest activity removing uracils context million bases) damage product [3,4]. Efficient base-excision                      single-stranded double-stranded DNA [5]. SMUG similar <br /> DNA repair fine-tuned control nucleotide pools                        attributes UNG prefers uracil-containing ssDNA safe-guard uracil DNA relying key enzymes:                           substrate [15]. MBD4 TDG recognize mismatched uracil uracil&#226;&#8364;&#8220;DNA glycosylase (UDG) [5] dUTPase [6].                                      <span id='am-34' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span>thymine</span> bases base-pair <span id='am-49' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>guanine</span> [16]. enzyme <br />    UDG deficiency, combination thymidylate synthase                            known function CpG islands, thymine formed <br /> inhibition depleted dUTPase activity, reported lead                     spontaneous methyl-cytosine deamination [17]. <br /> notable uracil accumulation DNA.  deficiency                                hydrolyzing <span id='am-21' about='obo:IMR_0200056' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0200056" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00460" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="dUTP" datatype="xsd:string"></span>dUTP</span> <span id='am-44' about='obo:IMR_0200008' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:id" content="IMR:0200008" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00365" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="dUMP" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dUMP</span> <span id='am-11' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span>pyrophosphate</span>, <br /> uracil&#226;&#8364;&#8220;DNA glycosylase contributes slightly                     dUTPase serves dual role cell physiology: produces genomic uracil content [7&#226;&#8364;&#8220;10]. dUTPase deficiency                             precursor thymidylate synthesis, removes dUTP silencing rescued depleted activity UDG [11&#226;&#8364;&#8220;14],                       deoxynucleotide pool. Eukaryotic bacterial DNA polymerases argued UDG major factor renders uracil&#226;&#8364;&#8220;                          incorporate deoxyuridine DNA rate depends  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                        1                                June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                            Role dUTPase Drosophila Development <br />  <br />  <br />  <br />   Author Summary                                                               deficient organisms. drug 59-fluorodeoxyuridine (FdUR), <br />                                                                                frequently used inhibitor thymidylate biosynthesis [23,24], <br />   usual paradigm confines &#226;&#8364;&#732;&#226;&#8364;&#732;normal&#226;&#8364;&#8482;&#226;&#8364;&#8482; DNA living cells                   leads perturbation nucleotide levels cell death. Loss   defined restricted <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> space populated                   uracil&#226;&#8364;&#8220;DNA glycosylase activity lead fluoropyr- <br />   bases (<span id='am-16' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>, thymine, guanine, cytosine)                    imidine resistance E. coli [25], yeast [26] C. elegans [27]. <br />   methylated derivatives (e.g. 59-methyl-                    Deficient uracil&#226;&#8364;&#8220;DNA glycosylase activity reported   cytosine). Uracil considered &#226;&#8364;&#732;&#226;&#8364;&#732;normal&#226;&#8364;&#8482;&#226;&#8364;&#8482; DNA                   required increased genomic uracil content FdUR <br />   base, bacteriophages. contrary,                     exposure E. coli mammalian cells [7&#226;&#8364;&#8220;10].  <br />   uracil generally considered error DNA.                     response FdUR treatment indicator testing <br />   Drosophila cells interpret uracil-substituted DNA                  cellular uracil&#226;&#8364;&#8220;DNA glycosylase activity. observed   normal DNA, lack repair enzymes.                            Drosophila S2 cell line shows small decrease viability   Importantly, unusual trait developmental <br />                                                                                presence 1 mM FdUR, HeLa cells, possessing ung <br />   control applies animals pupation. <br />                                                                                gene, strong lethality drug concentration (Figure 1A). <br />   Metamorphosis drastically perturbed silencing   dUTPase, responsible keeping uracil DNA.                      dose-dependent response increasing concentrations   results argue Drosophila,                       FdUR, uracil accumulation genomic DNA Drosophila S2 cells <br />   Holometabola insects  uracil&#226;&#8364;&#8220;DNA plays dedicat-                    highly elevated  approx. 450 uracil/million bases) <br />   ed physiological role.                                                       (Figure 1B). observed relatively high resistance FdUR <br />                                                                                cellular response genomic uracil incorporation                                                                                explained fact Drosophila cells lack significant uracil&#226;&#8364;&#8220; cellular dUTP concentration [6,10]. dUTPase                DNA glycosylase activity. <br /> significant indirect impact prevention uracil incorporation                 asked uracil-containing chemically unusual DNA DNA. Lack dUTPase supposed induce thymine                     tolerated interpreted Drosophila cells. Uracil&#226;&#8364;&#8220;DNA <br /> uracil substitution does alter genetic code;              specific cell response provoked transfecting cells UDG removes kind base modification supposedly                    exogenous plasmid uracil&#226;&#8364;&#8220;DNA. expression signal resulting genome instability. process serve                  fluorescent reporters (eYFP dsRedMonomer) encoded explanation overall lethality dUTPase deficiency [6].                   plasmids produced wild type (normal plasmid) dut-1, ung-1 <br />     propose condition similar simultaneous                   (uracil-plasmid, approx. 5500 uracil/million bases [8,10]) E. coli <br /> deficiency dUTPase UDG present Drosophila                    followed. gene span eYFP DSRed-monomer <br /> larvae. Drosophila genome does encode major uracil&#226;&#8364;&#8220;                reporters included promoter (CMV immediate early DNA glycosylase UNG [18], uracil&#226;&#8364;&#8220;DNA tolerance                   metallothionein), reporter protein gene (DSRed-monomer expected. Drosophila larvae contain proliferating primordial           eYFP), SV40 polyadenylation signal. total length tissues imago differentiated tissues degrade                gene spans comprises 1515 1299 nucleotides. metamorphosis. previous study reported Drosophila dUTPase <br /> [19,20] expression kinds tissues <br /> [21,22]. wished investigate stage- tissue-specific <br /> expression dUTPase translated differences uracil <br /> content genomic DNA putative pattern developmental significance. <br />    report high resistance Drosophila cell lines fluorodeoxyuridine (FdUR) induced high uracil&#226;&#8364;&#8220;DNA levels. addition tolerance uracil-substituted DNA, <br /> Drosophila cells correctly interpret unusual DNA vitro vivo.  wild type Drosophila observe stage- tissue- <br /> specific changes uracil-content DNA inversely <br /> correlated dUTPase expression cellular fate. propose observed pattern lack ung gene Drosophila genome paralleled developmental regulation dUTPase larval tissues. check absence dUTPase causative uracil accumulation DNA, silencing dUTPase larvae expands uracil&#226;&#8364;&#8220;DNA <br /> pattern imaginal tissues,  Interestingly, dUTPase silencing <br /> results abnormal DNA strand breaks, cell death develop- <br /> mental arrest early pupal stage. indicate tolerance highly uracil-containing DNA stage-specific,              Figure 1. Tolerance stability uracil-containing DNA D. <br /> tolerance factors, addition lack UNG, appear          melanogaster.   Dose-response curve FdUR treatment followed <br /> specific developmental phase. knowledge, study               Alamar Blue assay. (B) FdUR leads uracil accumulation DNA reports uracil&#226;&#8364;&#8220;DNA appearance absence                     Drosophila S2 cells. Data indicate increased level uracil  <br /> dUTPase wild type complex eukaryotic organism                         tolerated Drosophila, human cells. Data presented                                                                                mean 6 s.e.m. (C) Drosophila S2  panels) human HeLa cells <br /> demonstrates developmental pattern tolerance stability.          panels) transfected normal plasmid (left panels)                                                                                uracil-containing plasmid (right panels). Expression YFP Drosophila <br /> Results/Discussion                                                             S2 cells dsRedMonomer HeLa cells indicates stability DNA. <br />                                                                                (D) Microinjection uracil-plasmid Drosophila embryo. Non- <br />   Drosophila genome lacks UNG, wished test                     injected embryos served control sample. Drosophila cells similar characteristics UNG                   doi:10.1371/journal.pgen.1002738.g001 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                2                             June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                           Role dUTPase Drosophila Development <br />  <br />  <br /> synthesized double mutant dut-1, ung-1 E.coli strain,               physiological circumstances larval development, dUTPase average number uracil substitutions single strand               predominantly expressed imaginal discs, central nervous gene spans corresponds 8.3 7.1, respectively. Human               testis (Figure 2) larval tissues, like <br /> (HeLa) cells, possessing UNG, showed appreciable fluorescent                   salivary gland gut. ventriculus salivary gland, <br /> signal transformed normal plasmid, indicating              cells associated dUTPase expression &#226;&#8364;&#8220; uracil-containing plasmid interpreted probably              correspond imaginal cells (Figure 2B). imago, <br /> degradation. contrary, Drosophila S2 cells express                dUTPase present ovary females. Cellular localization reporter genes encoded normal uracil containing                   dUTPase usually nuclear [21,28&#226;&#8364;&#8220;31], cytoplasmic <br /> plasmid comparable level (Figure 1C Figure S1).                         occurrence evident nurse cells mature follicles <br />    Having established Drosophila cells vitro culture            ovaries, described previously [21]. present results <br /> tolerate correctly interpret uracil-containing DNA,                  agreement limited earlier observations wished assess fruit fly physiological response uracil&#226;&#8364;&#8220;DNA            protein level [21]. quantified dUTPase mRNA level organism level. similarly cell culture                 real-time PCR protein mRNA levels reporter-assay, uracil&#226;&#8364;&#8220;DNA plasmid introduced Drosophila                similar tendencies development (Figure 3A 3B). <br /> embryo. microinjection pP{Gal4VP16} expression vector                     interested learn coincidence uracil&#226;&#8364;&#8220;DNA <br />  1000 nt segment includes P-element promoter,                tolerance dUTPase regulation results uracil accu- <br /> Gal4-VP16 fusion gene hsp70 terminator                   mulation genome affected developmental stages average number 5.5 uracil substitutions produced                   larval tissues. Determination genomic uracil level carried <br /> double mutant E. coli) P{mw+UASeGFP} transgenic animals,                  applying recent quantitative real-time PCR-based method strong eGFP signal indicated stability exogenous DNA                [10]. Data presented Figure S2 Figure 3C argue DNA <br /> (Figure 1D). pre-hatching embryos eGFP signal detected                  purified embryo relatively uracil-free, larval normal uracil-containing DNA commeasur-                     stages, uracil accumulated. presence uracil&#226;&#8364;&#8220; <br /> able intensities. cases expression pattern reporter         DNA larvae confirmed using independent <br /> construct similar eGFP pronounced gut.                   method, UNG-ARP assay (Figure S3) [8]. Uracil content results explained assuming genetic               varies according tissue type larvae: salivary gland, <br /> information stored uracil-containing DNA serves cognate                representative tissue pupal degradation, accumulates high <br /> code transcription Drosophila cells. ability fruitfly       levels uracil; imaginal discs, undergo <br /> cells probably lack UNG condition                abundant metamorphosis-coupled cell death, contain uracil&#226;&#8364;&#8220;DNA does rapidly degraded.                                      uracil (Figure 3D). Uracil level larval tissues comparable    key factor responsible keeping uracil DNA              previously reported double mutant dut1-1, ung-1 E. coli enzyme dUTPase, importance                    strain [8,10], amounting thousands uracils million <br /> substantiated D. melanogaster lacks UNG. develop-               bases. discussed pattern uracil accumulation ment D. melanogaster, high dUTPase protein levels                    different stages tissues shows clear negative correlation observed embryonic stages (Figure 2A). shown                     expression pattern dUTPase (compare respective develop- <br /> Western blotting immunohistochemistry, starting late                  mental stages Figure 2 Figure 3). <br /> embryonic phase, dUTPase expression level decreased drasti-                    order investigate perturbation wild-type pattern cally remains low postembryonic stages. normal                uracil levels DNA interfere normal development,  <br />  <br />  <br />  <br /> Figure 2. Stage- tissue-specific distribution dUTPase protein levels D. melanogaster. Western blotting   immunohistochemistry (B) performed selected developmental stages tissues. Embryo 0&#226;&#8364;&#8220;6 h (E1), embryo 0&#226;&#8364;&#8220;24 h (E2), 1st larvae (1L), <br /> 2nd larvae (2L), early 3rd larvae (3L1), wandering 3rd larvae (3L2), pupae head eversion (P1), pupae head eversion (P2) pupae 50&#226;&#8364;&#8220;60 h puparium formation (P3). Western blotting, actin used loading control. Note dUTPase protein levels regulated larval stages expression confined specific tissues. <br /> doi:10.1371/journal.pgen.1002738.g002 <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                3                            June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                                Role dUTPase Drosophila Development <br />  <br />  <br />  <br />  <br /> Figure 3. D. melanogaster genomic DNA uracil content inversely                     Figure 4. Silencing dUTPase Drosophila larvae pupae. <br /> correlates dUTPase expression.   Changes dUTPase                         Western blots   protein level dUTPase mRNA level fruitfly development: embryo (E), 1st larvae (L1),           detection limit silenced animals. Actin served loading control. (B) <br /> 2nd larvae (L2), late 3rd larvae (L3) pupae (P). Note dUTPase          Curves relative number silenced non-silenced animals regulated larvae. (B) Comparison dUTPase RNA level               undergone puparium formation given time point larval tissues salivary gland imaginal tissue. Data presented           egg deposition. Inflection points curves represent mean time <br /> mean triplicates 6 s.e.m. mRNA level measured RT-qPCR                puparium formation characteristic given population. dUTPase <br /> dUTPase mRNA level normalized Rp49 mRNA level. (C) Uracil                   silencing did perturb time interval required puparium <br /> content D. melanogaster genome different developmental stages:               formation. (C) Graph shows number counted dead animals <br /> embryo (E), 1st larvae (L1), 2nd larvae (L2), late 3rd larvae (L3) pupae       relative number hatched curly winged control flies. (P). Embryonic sample used reference shown                  dead animals, groups distinct morphological traits charac- <br /> contain undetectable levels uracil DNA (cf. Figure S2). (D)                  teristic wandering larvae (w3L), prepupae (preP), pupal stage P5 <br /> Comparison genomic uracil content wild type imaginal disc                (P5) identified counted. (D) Genomic uracil content salivary gland 3rd larvae. Data presented mean 6 s.e.m.                  dUTPase silenced control tissues 3rd larvae. <br /> doi:10.1371/journal.pgen.1002738.g003                                              doi:10.1371/journal.pgen.1002738.g004 <br />  <br /> aimed silence dUTPase transgenic D. melanogaster strains                     dUTPase level (Table S2). Rescued animals underwent develop- <br /> (Table S1). Efficient silencing successfully achieved                ment metamorphosis just like wild type animals. setup resulting distinguishable F1 phenotypes: non-silenced                larval, prepupal imago stages, rescued animals carefully <br /> animals characterized GFP expression curly wings,                      investigated morphology distinct traits observed. silenced animals markers [32] (Figure S4). Overall                     detailed phenotype analysis dUTPase-silenced animals, <br /> silencing resulted efficiently decreased dUTPase protein levels                 morphologic observations indicated adverse effects larvae pupae (Figure 4A). RNAi silencing operate                    developmental arrest pupal stage P5 [35] <br /> appropriately early embryo maternal effects,                    (Figure 5B, Figures S5 S6). removing puparium <br /> stage scope present experiment transgenic                silenced animal, defects identified inner texture <br /> strains. observed dUTPase silencing did perturb                        everted discs, head sack, development normal life development larvae. silenced versus non-                    adult eye. defects permanent, simply silenced larvae F1 generation selected based GFP                        slow normal development.  3&#226;&#8364;&#8220;4 days expression, time interval egg laying                       puparium formation, tissues showing morphological traits associ- <br /> puparium formation did significant alteration                         ated larval stage identified dissected samples <br /> (Figure 4B). Importantly, effective silencing dUTPase                        (Figure S7 versus Figure S8). Noteworthy, typical organ <br /> imaginal discs larval brain did cause observable                       structures wild type adult (legs, wings, Malpighian tubules, <br /> morphological changes tissue morphology (Figure 5A). pupal                   Yellow Body, eyes ) appeared silenced animals. <br /> stage,  dUTPase silencing induced 100% lethality, e.                  Darkened tissues result cell death, necrosis silenced animals develop imago (observation based                 histolysis observed e.g. prothoracic region counting 2350 curly winged control imagos resulting                 organs (Figure 5B Figure S8) [36]. result argued generation crossings) (Figure 4C).                                              dUTPase dispensable larval tissues [21],    possible proof RNAi specificity rescue                      presence enzyme developmental stage early <br /> corresponding complementary DNA [33,34].  test                        puparium formation essential normal development specificity dUTPase RNAi, RNAi construct                         metamorphosis. <br /> expressed dUTPase transgene led expression                      Tissue specific silencing dorsal wing surface imaginal <br /> dUTPase protein, detected rescued animals (cf. method                    discs developed described Figure S9 (cf. [37]). described Text S1 shown Figure S4C S4D).                          great majority silenced animals developed curly wings, observed rescue lethality caused RNAi                         significant portion showed blisters wings. <br /> confirming RNAi phenotype reduction                       Curling blistering wings suggested result  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                    4                            June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                          Role dUTPase Drosophila Development <br />  <br />  <br />                                                                                   analyze effects dUTPase silencing lead DNA <br />                                                                                damage cell death, applied TUNEL phospho-Histone <br />                                                                                H2Av assays [40,41]. observed imaginal discs isolated <br />                                                                                3rd stage wandering larva dUTPase-silenced animals                                                                                strong enrichment TUNEL positive cells (Figure 6A 6B). <br />                                                                                TUNEL staining tissues indicate primarily cell death                                                                                phase DNA fragmentation. address question                                                                                dUTPase depletion violates genome integrity stained nuclei                                                                                phospho-Histone H2Av. H2Av histone modification ATR/ <br />                                                                                ATM kinases indicates DNA double strand breaks (DSBs)                                                                                proximity foci [41]. observed numerous phospho-H2Av <br />                                                                                foci dUTPase silenced wing imaginal discs, foci <br />                                                                                visible wild type (Figure 6C). results suggest                                                                                potential correlation dUTPase activity DNA <br />                                                                                integrity. increased level DNA damage observed                                                                                experiments dUTPase-silenced animals result                                                                                excessive processing uracil-containing DNA concludes                                                                                DNA fragmentation. <br />                                                                                   Uracil&#226;&#8364;&#8220;DNA measurements provided direct evidence larval <br />                                                                                tissues D. melanogaster undergo developmental degradation <br />                                                                                accumulate uracil genomic DNA. <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span>, presence highly <br />                                                                                uracil-substituted genomic DNA wild type organisms                                                                                reported viruses; e.g. bacteriophages [1,2]                                                                                recently HIV. knowledge, present study present <br />                                                                                example naturally occurring permanent existence                                                                                uracil&#226;&#8364;&#8220;DNA free-living complex eukaryotic organism (cf. <br />                                                                                Figure 3C 3D). condition requires developmental  <br />                                                                                regulation dUTPase absence UNG case D. <br />                                                                                melanogaster. Involvement dUTPase determining genomic <br />                                                                                uracil content regulating dUTP levels confirmed                                                                                dUTPase silencing resulted appearance uracil&#226;&#8364;&#8220;DNA Figure 5. Morphological consequences dUTPase silencing.                  imaginal disc tissues (cf. Figure 4D). Silencing dUTPase resulted <br /> larvae   pupae (B).   Immunohistochemistry wing eye             developmental defects DNA strand breaks (cf. Figure 5 discs, brain non-silenced (control) silenced larvae dUTPase (red) DAPI staining DNA (blue) demonstrate              Figure 6). reported uracil&#226;&#8364;&#8220;DNA tolerated hand highly effective silencing; hand observable           interpreted embryonic 3rd larval stages. <br /> morphological changes tissues. (B) Wild type pupae                   extraordinary situation tolerance stability uracil&#226;&#8364;&#8220; <br /> (control) stage P6 (cf. Figure S5) dUTPase silenced pupae            DNA exclusively present D. melanogaster absence <br /> corresponding time puparium formation dorsal ventral view         ung ubiquitous Holometabola (Table 1). uracil&#226;&#8364;&#8220; shown, puparium removal. Wild type traits, Malpighian tubules <br />                                                                                DNA naturally occurs larval tissues sentenced death, <br /> (white arrows), Yellow Body (white asterix), developing adult eye (white <br /> arrowheads) observable silenced animals. Instead, darkened          consider uracil&#226;&#8364;&#8220;DNA linked metamorphosis <br /> (apoptotic/necrotic melanized) tissues (red arrowheads)              tissue degradation. investigations taken visualized pupae. Note basically different inner texture       mechanism, impact putative role. everted discs (white boxes) head sack (white circles).                    Pupal lethality observed flies affected mutations doi:10.1371/journal.pgen.1002738.g005                                          silencing purine biosynthesis enzymes [42,43]. animals <br />                                                                                showed apoptosis developing imaginal primordium abnormal cell death dorsal wing surface [38,39]. results              metamorphosis observable darkened tissues. suggest dUTPase silencing defined tissue                    studies, deficient deoxynucleotide biosynthesis segment specifically exclusively associated                    resulted imbalanced dNTP levels increased ratio phenotypic effects tissue segment.                            mismatches DNA [44]. Overrepresented DNA modifications <br />    analyze dUTPase silencing changed distribution               like cytosine methylation result pupal lethality [45,46]. uracil&#226;&#8364;&#8220;DNA different larval tissues, assayed genomic              case increased methylation pattern DNA inhibits transcrip- <br /> uracil content imaginal discs non-silenced silenced           tional activity cell cycle progression. <br /> larvae F1 generation. Figure 4D shows, imaginal discs                  suggest hypotheses explain pupal intolerance dUTPase silenced larvae accumulated high uracil                 uracil&#226;&#8364;&#8220;DNA.  similarly case hypermethylated DNA, genome (1097+/255 uracil/million bases, compared                   uracil&#226;&#8364;&#8220;DNA decreased response interaction with 131+/269 uracil/million bases non-silenced animals)                transcriptional regulators, activators morphogenetic approximated genomic uracil content salivary gland.                factors required specifically pupal metamorphosis. Second, increase uracil&#226;&#8364;&#8220;DNA content salivary gland DNA               factor(s), functional pupal stage, observed dUTPase silenced larvae, indicating silencing             process uracil&#226;&#8364;&#8220;DNA resulting genome instability defects effective tissues normally express dUTPase              cell cycle progression cell death [47,48]. UNG effect uracil&#226;&#8364;&#8220;DNA tissues normally express                   missing Drosophila, uracil&#226;&#8364;&#8220;DNA glycosylases dUTPase. result confirms uracil appearance DNA                    suspect role. agreement suggestion, expression <br /> depends dUTPase expression, dUTPase activity causative              patterns uracil&#226;&#8364;&#8220;DNA glycosylases (SMUG Thd1) maintaining DNA low levels uracil.                                  AP endonucleases (Rrp1, RpS3 RpLP0) indicate relative <br />  <br />  <br />        PLoS Genetics | www.plosgenetics.org                                5                          June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                               Role dUTPase Drosophila Development <br />  <br />  <br />  <br />  <br /> Figure 6. dUTPase silencing results cell death DNA strand breaks larval imaginal discs.   Imaginal discs isolated wild type dUTPase silenced wandering 3rd larvae stained TUNEL assay (shown red dots). Discs silenced animals showed highly <br /> increased TUNEL staining. (B) TUNEL positive cell counts imaginal discs wild type dUTPase silenced wandering 3rd larvae. Error bars <br /> represent standard error mean. (C) Imaginal discs wild type dUTPase silenced 3rd wandering larvae stained phospho-H2Av <br /> foci (white dots, appointed white arrowheads). dUTPase depleted discs showed nuclei phospho-H2Av foci <br /> indicating DNA damage. Scale bar represents 50 mm. <br /> doi:10.1371/journal.pgen.1002738.g006 <br />  <br /> upregulation metamorphosis lowest expression                    stages. fruit flies, Holometabola larval stages according microarray data (Table S3) [49].                     insects, unique possessing developmental period Developmental downregulation base-excision DNA repair                     uracil&#226;&#8364;&#8220;DNA tolerated, preservation transmission pathway (BER) enzymes contribute uracil&#226;&#8364;&#8220;DNA appear-                         genetic information generations requires dUTPase. <br /> ance tolerance larval stages, upregulation initiate uracil&#226;&#8364;&#8220;DNA processing, destabilization degradation.                    Materials Methods pupal uracil&#226;&#8364;&#8220;DNA intolerance pathways hypothesized intended analyzed experiments.                                Uracil&#226;&#8364;&#8220;DNA stability assays <br />     Taken  suggest different factors                       uracil&#226;&#8364;&#8220;DNA stability assays, pRm-eYFP, pDsRedMono- <br /> responsible stage-specific elevated level uracil larval              mer-N1, pP{Gal4VP16} (kind gift La&#194;&#180;szlo&#194;&#180; Sipos) amplified <br /> DNA: 1) lack ung gene, 2) absence dUTPase 3) decreased                   E.coli K12 XL1Blue strain CJ236 dut-1, ung-1 strain (NEB). <br /> levels enzymes involved uracil removal. conclude                     Plasmids purified Plasmid Midi Kit (QIAGEN). Uracil dUTPase essential completion Drosophila                content plasmids checked UDG AP endonuclease <br /> life cycle, absence tolerated larval                    treatment followed standard agarose gel electrophoresis [48]. <br />  <br />  <br />  Table 1. Occurrence genes encoding dUTPase UNG different insects. <br />  <br />  <br />  Insecta        Holometabola               Diptera                                                                            dUTPase            UNG <br />  <br />  3              3                          3                  Drosophila melanogaster (fruit fly)                             +                  2 <br />  3              3                          3                  Culexpipiens(mosquito)                                          +                  2 <br />  3              3                          3                  Aedesaegypti(mosquito, yellow fever)                            +                  2* <br />  3              3                          3                  Anopheles gambiae(mosquito, malaria)                            +                  2 <br />  3              3                                             Bombyxmori(silkmoth)                                            +                  2 <br />  3              3                                             Triboliumcastaneum(red flour beetle)                            +                  2 <br />  3              3                                             Apismellifera(honey bee)                                        +                  2 <br />  3              3                                             Nasoniavitripennis(parasitoid wasp)                             +                  2 <br />  3                                                            Acyrthosiphonpisum(pea aphid)                                   +                  2 <br />  3                                                            Pediculushumanuscorporis(body louse)                            +                  + <br />  <br />  gene dUTPase ubiquitous, gene major uracil&#226;&#8364;&#8220;DNA glycosylase, ung encoded genome Holometabola species. <br />   genome Aedes aegypti strain Liverpool, unexpected ung sequence  showing high (87%&#226;&#8364;&#8220;94%) similarity ung gene  Comamonadaceae, family Proteobacteria, arguing bacterial origin. <br />  doi:10.1371/journal.pgen.1002738.t001 <br />  <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                                     6                            June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                      Role dUTPase Drosophila Development <br />  <br />  <br />    Uracil-containing plasmid stability cell culture. Trans-             pH = 6.9) 30 minutes, vigorous shacking room <br /> fection pRm-eYFP-N-C* Drosophila S2 cells carried          temperature (RT). Samples washed inactivating buffer presence Cellfectine (Invitrogen) following manufac-         (50 mM TRIS, 150 mM NaCl, 0.5% Tween-20, pH = 7.4). <br /> turer&#226;&#8364;&#8482;s instructions expression induced metallo-          Blocking performed following: 5% goat serum, 1.5% <br /> thionein promoter 25uC addition 700 mM CuSO4                  BSA, 0.1% Tween-20, 1% Triton-X 100, 0,001% NaN3, PBS, <br /> overnight incubation. pDsRedMonomer-N1 transfected                pH = 7.4 4 hours RT. Tissues incubated primary <br /> HeLa cells using Lipofectamine (Invitrogen). Samples               antibody diluted blocking buffer (1:10000), 4uC 16 h. <br /> visualized Leica DMLS fluorescence microscope 48 hours              Samples washed blocking buffer 8 h RT. transfection.                                                        Secondary antibody applied visualize dUTPase staining <br />   Uracil-containing plasmid stability Drosophila                        (Alexa 543 conjugated anti rabbit IgG, Molecular Probes) embryos. pP{Gal4VP16} plasmids injected 0&#226;&#8364;&#8220;                       blocking buffer (1:1000) 2 h, RT. DAPI used DNA <br /> 30 min pP{mw+UASeGFP} transgenic Drosophila embryos (kind                  staining. washing steps, samples mounted gift La&#194;&#180;szlo&#194;&#180; Sipos). experiment, app. 40 dechorionated        FluorSave (Calbiochem). Images acquired Zeiss <br /> embryos microinjected. aligned glass                   LSCM 710 Leica DM LED Fluo microscope equipped <br /> coverslip, dried 30 min, covered 10S Voltalef oil            Leica DFC345 FX monochrome camera. injection. Plasmid concentration adjusted 1 mg/ml, dilution standard injection buffer. GFP signal detected       TUNEL assay <br /> pre-hatching embryos, 22 h injection. Embryos                   TUNEL assay carried described [40]. careful <br /> injection served negative control.                                    dissection PBS, imaginal discs fixed (0.1M PIPES, <br />                                                                            pH = 6.9, 1 mM EDTA, 1% Triton X-100, 2 mM MgSO4, 1% <br /> Cell viability assay                                                       <span id='am-39' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span>formaldehyde</span>) 30 min RT, washed times PBS <br />    Drosophila Schneider 2 (S2) cells human HeLa cells             buffer containing 0.1% Triton X-100 (PBT) times <br /> cultured 96 plates 56104cells 26103 cells          PBT 5X (PBS, 0.5% Triton X-100) (10 min , respectively. culture media used Serum Free Insect                transferred permeabilization solution (0.1M sodium citrate Medium (Sigma, S3777) supplemented 1% penicillin&#226;&#8364;&#8220;                     PBT) 30 min 65uC. Discs washed twice PBT 5X, <br /> streptomycin solution S2 cells; DMEM-F12 (Sigma,                   times PBT incubated reaction buffer (30 mM Tris- <br /> D8437) supplemented 10% FBS 1% penicillin&#226;&#8364;&#8220;strepto-                HCl, pH = 7.2, 140 mM sodium cacodylate, 1 mM cobalt <br /> mycin HeLa cells. FdUR (Sigma) added final                    chloride) 30 min RT. Reaction carried reaction <br /> concentration range 0.1&#226;&#8364;&#8220;1000 mM. 96 hours culture,             buffer containing 0.2 unit/microL terminal deoxynucleotidyl <br /> cell viability quantified Alamar Blue assay (BioSource).        transferase (NEB) 5 microM Cy5-dUTP (GE Healthcare) <br /> experiment repeated triplicates.                                    1 hour RT. reaction stopped PBT 5X                                                                            washed times PBT finally PBS. Nuclei Quantitative measurement uracil content DNA                      stained Hoechst. Discs imaged laser scanning confocal <br />                                                                            microscopy (Zeiss LSCM 710). number TUNEL positive <br /> samples                                                                    cells determined ImageJ (Rasband, W.S., ImageJ, National <br />    order quantify uracil content DNA, real-time                 Institutes Health, Bethesda, Maryland, USA, http://rsb.info. <br /> quantitative PCR-based assay used [10]. Genomic DNA                nih.gov/ij/, 1997&#226;&#8364;&#8220;2004.) <br /> isolated digested NheI. DNA fragments 4&#226;&#8364;&#8220;5 kb purified gel. Real-time PCR performed Mx3000P <br /> qPCR (Agilent Technologies) using EvaGreen dye <br />                                                                            Supporting Information <br /> (Biotium) PfuTurbo Hotstart DNA polymerase PfuTurbo                Figure S1 Percentage fluorescent cells transfection Cx Hotstart DNA polymerase (Stratagene). segment 963                normal (T pl.) uracil-substituted plasmids (U pl.).   Drosophila <br /> base length defined primers (puBSd-Fw 59-TCGGGAT-                   S2 cells, (B) HeLa cells. number observed fluorescent cells GACTTTTGGGTTCTG-39 puBSd985R 59-CGCGGTT-                               presented bars total number TAACACAGCGTCGG-39) amplified PCR reaction.                   scored cells (shown brackets). fold dilution series prepared DNA samples.             (PDF) parallel measurements performed. <br />                                                                            Figure S2 Genomic uracil content embryo detection <br />                                                                            limit. Uracil content Drosophila embryonic genome compared dUTPase silencing RNA interference rescue RNAi                   DNA plasmid purified wild-type E. coli.   UAS stocks obtained Vienna Drosophila RNAi                  samples showed value detection limit. <br /> Centre (VDRC) [32]. Strain #21883 #21884 used                  (PDF) <br /> dUTPase silencing. Rescue performed expression RNAi construct dUTPase transgene. details Text              Figure S3 Ung-ARP assay. UNG-ARP assay shows presence S1, Tables S1 S2, Figures S4 S9.                                   uracil&#226;&#8364;&#8220;DNA Drosophila larvae. negative positive <br />                                                                            controls, genomic DNA samples XL1 Blue CJ236 ung- <br /> Immunohistochemistry Western blot analysis                             1, dut-1 E.coli strains used respectively. CJ236 ung-1, dut-1 <br />                                                                            E.coli strain produces DNA high uracil content (approx. 5500 <br />    Western blot analysis larval organs stage specific <br />                                                                            uracil/million bases [8,10]). <br /> dUTPase expression performed described previously [21]. immunohistochemistry, applied primary antibodies anti-            (PDF) <br /> dUTPase (1:10000) mouse anti-phospho-Histone H2A.X                      Figure S4 Scheme crossing silencing dUTPase (Ser139) (Millipore) (1:250). shown recognize            Drosophila larvae pupae rescue dUTPase RNAi. <br /> Drosophila phospho-Histone H2Av [50]. Tissues fixed                Crossing schemes shown panel C: Act-Gal4 means <br /> 50% n-Heptane 50% PEM-formaldehyde (100 mM PIPES,                      Gal4 gene coupled actin 5C promoter result 1 mM MgCl2, 1 mM EGTA, 2.5% Tween-20, 4% PFA,                              ubiquitous constitutive expression yeast transcription <br />  <br />  <br />       PLoS Genetics | www.plosgenetics.org                             7                           June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                                    Role dUTPase Drosophila Development <br />  <br />  <br /> factor, Gal4 transgenic D. melanogaster driving transcription                enhancer trap line expressing Gal4 preferentially dorsal <br /> silencing element   following UAS promoter. F1 generation                   compartment wing carrying UAS-Dicer2 homozy- genotypes: Act-Gal4/UAS animals express dsRNA                       gous form second chromosome (Bloomington stock <br /> dUTPase silencing, markers; UAS CyO, GFP                          25706) crossed males carrying Gal4 inducible <br /> animals, silencing element activate, curly wing (CyO)                   silencing element (UAS  homozygous form second GFP markers expressed. Silenced non-silenced animals                       chromosome. silencing element activated MS1096 distinguishable larvae/pupae imago stages basis                  driver [37] female progenies F1 males served GFP (panel B) CyO markers, respectively. Crossing scheme                    internal negative control silencing occurred. Silenced silencing shown panel C: UAS-dUTPase-FLAG stands                         females exhibited dorsally curled wing phenotype (panel B) rescue construct. relevant categories F1                        blisters. penetrance phenotype 85%. <br /> generation unambiguously distinguished based                         35% silenced female progeny showed blistering <br /> phenotype marker mutations CyO, SM6b, TM3                        wings (panel C). Male progenies wing phenotype. Panel D <br /> balancer chromosomes. TM3 phenotype marks gene                             shows expression pattern MS1096 driver dorsal <br /> silenced progenies, rescued animals noTM3                           compartment wing disc visualized crossing MS1096 <br /> phenotype. Panel D shows Western blot dUTPase silenced                      females UAS-MoesinCherry [51] males (panel D) (red <br /> versus rescued animals. Note absence dUTPase protein                     fluorescent staining wing disc). MoesinCherry overexpress- <br /> silenced animals (silencing alleles 21883 21884),                  ing female progeny wing phenotype. <br /> presence dUTPase proteins rescued animals (rescuing                      (PDF) <br /> alleles DMDUT20 DMDUT29). Equivalent total protein <br /> loading verified developing blot tubulin                   Table S1 Genomic position UAS constructs dUTPase <br /> using anti-Tubulin (E7, provided M. Klymkowsky; Develop-                        RNAi stocks. <br /> mental Studies Hybridoma Bank, University Iowa, Iowa city,                      (PDF) <br /> IA).                                                                               Table S2 dUTPase transgene rescues dUTPase RNAi <br /> (PDF)                                                                              phenotype. Table shows results rescue crosses. UAS- <br /> Figure S5 Summary pupal developmental processes. Red                            SM6b; UAS-dUTPase-FLAG/TM3 males crossed arrow shows stage P5  12&#226;&#8364;&#8220;14 h puparium                            Act-Gal4/CyO females (Figure S4). UAS (21883 formation) lethality dUTPase silencing appear.                        21884) transgenic rescue lines (DMDUT20 (PDF)                                                                              DMDUT29) combined. Number progenies relevant <br />                                                                                    F1 categories shown. Gene silencing complete Figure S6 Developmental arrest caused dUTPase silencing                                                                                    UAS Act-GAl4; TM3/+ adult progeny observed.  <br /> Drosophila pupae. Wild type (upper panels) dUTPase silenced <br />                                                                                     dUTPase transgene present, rescued animals <br />  panels) pupae compared stages P4, P5&#226;&#8364;&#8220;6, P6&#226;&#8364;&#8220;7, <br />                                                                                    survived adulthood. P9. panel shows views pupa: dorsal <br />                                                                                    (PDF) <br /> (upper  ventral   puparium. Specific differences appear P5: Malpighian                  Table S3 Uracil&#226;&#8364;&#8220;DNA repair perturbed Drosophila. Micro- <br /> tubules (arrows) Yellow Body (asterices) appears                      array data available FlyBase used. Table shows mRNA <br /> dUTPase silenced pupae.                                                            level genes involved different DNA repair pathways, <br /> (PDF)                                                                              elements uracil&#226;&#8364;&#8220;DNA repair highlighted grey  <br /> Figure S7 Wild type structures pharate adults 3 days                      ground. Q indicates mRNA level decrease, q mRNA level <br /> puparium formation. Wild type pupa dissected stage P11                      increase, &lt; stage-specific change. Note overall base adult organs developed  . Dissected                           excision repair regulated larval development, Malpighian tubules (arrows B) Yellow Body (asterices                     DNA repair processes  <br /> B) wild type pupa organs identified                     (PDF) <br /> dUTPase silenced pupae.                                                            Text S1 Supplementary information. Includes Supplementary <br /> (PDF)                                                                              Materials Methods Supplementary References. <br /> Figure S8 Larval traits dissected silenced pupae 3 days                   (DOC) <br /> puparium formation. days puparium formation, <br /> dissected tissues silenced pupae preserve larval traits: testis           Acknowledgments oval  , foregut gastric caeca larval characteristics (B,                                             &#194;&#180; gnes Rego&#203;?s kind help using                                                                                    thank Prof. Tibor Vellai D, asterices), Malpighian tubules (B, arrows) characteristic              Zeiss stereomicroscope La&#194;&#180;szlo&#194;&#180; Sipos kind help embryo larval ones, brain (C, white arrowhead) preserves                 microinjections. <br /> basic structure larval  Darkened tissues resulted necrosis, apoptosis melanisation [36].                                     Author Contributions <br /> (PDF) <br />                                                                                    Conceived designed experiments: VM AH AB MP GR JB PV Figure S9    Scheme crossing silencing dUTPase                    BGV. Performed experiments: VM AH AB MP BH JB IK GM GR FJ <br /> dorsal compartment Drosophila wing imaginal discs. Crossing                     PV  Analyzed data: VM AH AB JB MP GM GR JB IK FJ PV scheme shown panel  : virgin females MS1096 Gal4                    BGV. Wrote paper: VM AH AB MP GR PV BGV. <br />  <br /> References <br /> 1.   Duncan BK, Warner HR (1977) Metabolism uracil-containing DNA:              2.   Kiljunen S, Hakala K, Pinta E, Huttunen S, Pluta P, et al. (2005) Yersiniophage <br />      degradation bacteriophage PBS2 DNA Bacillus subtilis. J Virol 22:            phiR1-37 tailed bacteriophage having 270 kb DNA genome      835&#226;&#8364;&#8220;838.                                                                           <span id='am-59' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span>thymidine</span> replaced deoxyuridine. Microbiology 151: 4093&#226;&#8364;&#8220;4102. <br />  <br />  <br />  <br />  <br />         PLoS Genetics | www.plosgenetics.org                                   8                                June 2012 | Volume 8 | Issue 6 | e1002738 <br />                                                                                                                                 Role dUTPase Drosophila Development <br />  <br />  <br /> 3.    Lindahl T (1993) Instability decay primary structure DNA               28. Tinkelenberg BA, Fazzone W, Lynch FJ, Ladner RD (2003) Identification       comments]. Nature 362: 709&#226;&#8364;&#8220;715.                                                              sequence determinants human nuclear dUTPase isoform localization. Exp <br /> 4.    Krokan  Drablos F, Slupphaug G (2002) Uracil DNA&#226;&#8364;&#8220;occurrence,                           Cell Res 287: 39&#226;&#8364;&#8220;46. <br />       consequences repair. Oncogene 21: 8935&#226;&#8364;&#8220;8948.                                         29. Bozoky  Rona G, Klement E, Medzihradszky KF, Merenyi G, et al. (2011) <br /> 5.    Visnes T, Doseth B, Pettersen HS, Hagen L, Sousa MM, et al. (2008) Review.                   Calpain-catalyzed proteolysis human dUTPase specifically removes       Uracil DNA processing different DNA glycosylases. Philos Trans R               nuclear localization signal <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>. PLoS 6: e19546. doi:10.1371/ <br />       Soc Lond B Biol Sci.                                                                         journal.pone.0019546. <br /> 6.    Vertessy BG, Toth J (2009) Keeping uracil DNA: physiological role,                30. Merenyi G, Konya E, Vertessy BG (2010) Drosophila proteins involved       structure catalytic mechanism dUTPases. Acc Chem Res 42: 97&#226;&#8364;&#8220;106.                      metabolism uracil&#226;&#8364;&#8220;DNA possess different types nuclear localization signals. <br /> 7.    Andersen S, Heine T, Sneve R, Konig  Krokan  et al. (2005) Incorporation                FEBS J 277: 2142&#226;&#8364;&#8220;2156. <br />       dUMP DNA major source spontaneous DNA damage,                      31. Muha V, Zagyva  Venkei  Szabad J, Vertessy BG (2009) Nuclear localization <br />       excision uracil required cytotoxicity fluoropyrimidines mouse            signal-dependent -independent movements Drosophila melanogaster <br />       embryonic fibroblasts. Carcinogenesis 26: 547&#226;&#8364;&#8220;555.                                           dUTPase isoforms nuclear cleavage. Biochem Biophys Res Commun <br /> 8.    Lari SU, Chen CY, Vertessy BG, Morre J, Bennett SE (2006) Quantitative                       381: 271&#226;&#8364;&#8220;275. <br />       determination uracil residues Escherichia coli DNA: Contribution ung,           32. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) genome- <br />       dug, dut genes uracil avoidance. DNA Repair (Amst) 5: 1407&#226;&#8364;&#8220;1420.                      wide transgenic RNAi library conditional gene inactivation Drosophila. <br /> 9.    Luo Y, Walla M, Wyatt MD (2008) Uracil incorporation genomic DNA                        Nature 448: 151&#226;&#8364;&#8220;156. <br />       does predict toxicity caused chemotherapeutic inhibition thymidylate           33. Hsu YC, Chern JJ, Cai Y, Liu M, Choi KW (2007) Drosophila TCTP       synthase. DNA Repair (Amst) 7: 162&#226;&#8364;&#8220;169.                                                      essential growth proliferation regulation dRheb GTPase. <br /> 10.   Horvath  Vertessy BG (2010) step method quantitative                             Nature 445: 785&#226;&#8364;&#8220;788. <br />       determination uracil DNA real-time PCR. Nucleic Acids Res 38: e196.             34. Lopez TV, Lappin TR, Maxwell P, Shi  Lopez-Marure R, et al. (2011) <br /> 11.   el-Hajj HH, Wang L, Weiss B (1992) Multiple mutant Escherichia coli                       Autocrine/paracrine erythropoietin signalling promotes JAK/STAT-dependent <br />       synthesizing virtually thymineless DNA limited growth. J Bacteriol 174:               proliferation human cervical cancer cells. Int J Cancer 129: 2566&#226;&#8364;&#8220;2576. <br />       4450&#226;&#8364;&#8220;4456.                                                                               35. Bainbridge SP, Bownes M (1981) Staging metamorphosis Drosophila <br /> 12.   Gadsden MH, McIntosh EM, Game JC, Wilson PJ, Haynes RH (1993) dUtp                           melanogaster. J Embryol Exp Morphol 66: 57&#226;&#8364;&#8220;80. <br />       pyrophosphatase essential enzyme Saccharomyces cerevisiae. Embo J               36. Minakhina S, Steward R (2006) Melanotic mutants Drosophila: pathways       12: 4425&#226;&#8364;&#8220;4431.                                                                               phenotypes. Genetics 174: 253&#226;&#8364;&#8220;263. <br /> 13.   Dengg M, Garcia-Muse T, Gill SG, Ashcroft N, Boulton SJ, et al. (2006)                   37. Capdevila J, Guerrero (1994) Targeted expression signaling molecule <br />       Abrogation CLK-2 checkpoint leads tolerance base-excision repair                decapentaplegic induces pattern duplications growth alterations       intermediates. EMBO Rep 7: 1046&#226;&#8364;&#8220;1051.                                                        Drosophila wings. EMBO J 13: 4459&#226;&#8364;&#8220;4468. <br /> 14.   Guillet M, Van Der Kemp PA, Boiteux S (2006) dUTPase activity critical                                                                                                38. Morris EJ, Michaud WA, Ji JY, Moon NS, Rocco JW, et al. (2006) Functional <br />       maintain genetic stability Saccharomyces cerevisiae. Nucleic Acids Res 34: <br />                                                                                                    identificati <br /> </body></html>