<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                            International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199 <br />  <br />  <br />  <br />                                                          Contents lists available SciVerse ScienceDirect <br />  <br />                                              International Journal Parasitology: <br />                                                    Drugs Drug Resistance <br />                                                 journal homepage: www.elsevier.com/locate/ijpddr <br />  <br />  <br />  <br />  silico prediction antimalarial drug target candidates <br /> Philipp Ludin b, Ben Woodcroft c,1, Stuart  Ralph c, Pascal M&#195;&#164;ser b,&#226;&#8225;&#8216;   Swiss Tropical Public Health Institute, 4002 Basel, Switzerland <br /> b <br />   University Basel, 4000 Basel, Switzerland <br /> c <br />   Department Biochemistry Molecular Biology, Bio21 Molecular Science Biotechnology Institute, University Melbourne, Victoria 3010, Australia <br />  <br />  <br />  r t c l e        n f o                           b s t r c t <br />  <br /> Article history:                                       need new antimalarials persistent emergence drug resistant parasites. aim <br /> Received 19 April 2012                                 identify new drug targets Plasmodium falciparum phylogenomics Plasmodium spp. Received revised form 28 June 2012                  comparative genomics Homo sapiens. proposed target discovery pipeline largely independent Accepted 3 July 2012 <br />                                                        experimental data based assumption P. falciparum proteins likely essential   <br /> Available online 17 July 2012 <br />                                                        similar proteins proteome (ii) highly conserved malaria <br />                                                        parasites mammals. hypothesis tested using sequenced Saccharomycetaceae species Keywords: <br />                                                        touchstone. Consecutive &#239;&#172;?lters narrowed potential target space P. falciparum proteins Plasmodium falciparum <br /> Phylogenomics <br />                                                        likely essential, matchless human proteome, expressed blood stages parasite, <br /> Gene essentiality                                      amenable small molecule inhibition. &#239;&#172;?nal set 40 candidate drug targets signi&#239;&#172;?cantly <br /> Drug target                                            enriched essential proteins comprised proven targets (e.g. dihydropteroate synthetase enzymes <br />                                                        non-mevalonate pathway), targets currently investigation (e.g. calcium-dependent protein <br />                                                        kinases), new candidates potential phosphomannose isomerase, phosphoenolpyr- <br />                                                        uvate carboxylase, signaling components, transporters. targets prioritized based drugga- <br />                                                        bility indices availability vitro assays. Potential inhibitors inferred similarity                                                        known targets disease systems. identi&#239;&#172;?ed candidates P. falciparum provide insight                                                        biochemical peculiarities vulnerable points malaria parasite serve starting points <br />                                                        rational drug discovery. <br />                                                                         &#195;&#8220; 2012 Australian Society Parasitology Published Elsevier  rights reserved. <br />  <br />  <br />  <br />  <br /> 1. Introduction                                                                                Rational post-genomic drug discovery based screening <br />                                                                                            large <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> libraries &#226;&#8364;&#8220; virtually high-throughput <br />    Drug discovery programs launched Medicines Malaria                           format &#226;&#8364;&#8220; given target enzyme parasite. persistent <br /> Venture product-development partnerships culmi-                             bottleneck target-based approaches identi&#239;&#172;?cation nated development promising new antimalarial com-                                suitable drug target &#239;&#172;?rst place. enzyme essen- <br /> pounds synthetic peroxide OZ439 (Charman et al.,                               tial survival parasite suf&#239;&#172;?ciently different 2011) spiroindolone NITD 609 (Rottmann et al., 2010),                              closest counterpart human host inhibited selectively. currently undergoing clinical trials. spite                          Experimental tools validate candidate drug targets limited <br /> recent successes, pivotal maintain early phase drug discov-                       malaria parasites. Gene silencing RNAi does ery prevent antimalarial drug development pipeline                             feasible (Baum et al., 2009). Gene replacement select- <br /> draining. propensity parasite                                  able markers (Triglia et al., 1998), inherently problem- <br /> drug-resistant (Muller Hyde, 2010; Sa et al., 2011), need                          atic gene essential failing knock  Inducible new antimalarial chemotypes persist human-                              degradation Plasmodium falciparum proteins pathogenic Plasmodium spp. eventually eradicated.                                      fused FKBP-destabilization domain (Armstrong Goldberg, <br />                                                                                            2007) currently conclusive method antimalarial <br />                                                                                            target validation.  reverse genetic methods <br />   &#226;&#8225;&#8216; Corresponding author  Swiss Tropical Public Health Institute, Socinst- <br /> rasse 57, 4002 Basel, Switzerland. Tel.: +41 61 284 8338; fax: +41 61 284 8101. <br />                                                                                            practicable genome-wide scale. Adding chal- <br />     E-mail addresses: philipp.ludin@unibas.ch (P. Ludin), b.woodcroft@uq.edu.au            lenges Plasmodium molecular biology lack phyloge- <br /> (B. Woodcroft), saralph@unimelb.edu.au (S  Ralph), pascal.maeser@unibas.ch               netically close model organism serve point (P. M&#195;&#164;ser).                                                                                reference &#226;&#8364;&#8220; case parasitic nematodes, essenti- <br />   1 <br />     Current address: School Chemistry Molecular Biosciences, Australian             ality genes estimated based RNAi phenotypes <br /> Centre Ecogenomics, University Queensland, St. Lucia, Queensland 4072, <br /> Australia. <br />                                                                                            (Schindelman et al., 2011) orthologues Caenorhabditis elegans. <br />  <br /> 2211-3207/$ - matter &#195;&#8220; 2012 Australian Society Parasitology Published Elsevier  rights reserved. <br /> http://dx.doi.org/10.1016/j.ijpddr.2012.07.002 <br />  192                             P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199 <br />  <br />  <br />     bioinformatic approaches previously em-                        2.3. Algorithms <br /> ployed help identify prioritize drug targets Plasmodium <br /> parasites. include techniques based automated identi&#239;&#172;?ca-                       pipeline comparative genomics target prediction <br /> tion important steps metabolic pathways (Yeh et al., 2004;                     run semi-automated way, combining existing pro- <br /> Fatumo et al., 2009; Huthmacher et al., 2010; Plata et al., 2010),                   grams  Section 2.2) self Perl scripts parsing, <br /> techniques combine chemical starting points protein-                        formatting, analysis outputs. Druggability scores based queries (Joubert et al., 2009), use TDR-                 extracted TDRtargets website (Magarinos et al., 2012). <br /> targets web-resource (http://www.tdrtargets.org) (Magarinos                          druggability predictions numerical values 0 <br /> et al., 2012) prioritize drug targets combination                  1, 1 corresponding druggable targets. Scores  <br /> multiple data types relevant drug development (Crowther                           &#239;&#172;&#8218;ect combination factors including degree similarity et al., 2010).                                                                       large library targets ChEMBL database (Hopkins <br />     try predict antimalarial drug targets silico, build-               et al., 2011; Gaulton et al., 2012), empirically determined <br /> ing previous approaches labs predicting essentiality                 interactions drug like compounds, possession physiochem- proteins based phylogeny (Doyle et al., 2010; Waterhouse                       ical features enriched known drug targets, drug-like <br /> et al., 2010). de&#239;&#172;?ne protein candidate antimalarial                        quality inhibitors known interact homologues. <br /> drug target   conserved orthologues mamma-                 scores outputs combination multiple discrimina- <br /> lian-pathogenic Plasmodium spp.; (ii) match                          tors based empirically-characterized drug targets, P. falciparum; (iii) does good match human prote-                  untested Plasmodium, concordance high- <br /> ome  (iv) expressed trophozoite gametocyte                      scoring proteins known antimalarial drug tar- <br /> stages; (v) predicted function enzyme, receptor,                  gets. Proteins higher druggability scores likely transporter. rationale conserved genes ful&#239;&#172;?ll                      amenable inhibition drug-like molecules proteins <br /> essential functions &#226;&#8364;&#8220; genes ful&#239;&#172;?lling essential func-                 low druggability scores, used tions negative selection. conserved single copy gene                     aspect ranking  lack parasite-speci&#239;&#172;?c lacks close matches genome likely                          direct data algorithms assign cutoff indispensable. Starting complete predicted proteome                      designating acceptable score. <br /> P. falciparum (Gardner et al., 2002), applied consecutive &#239;&#172;?lters extract candidate drug targets meet criteria. <br />                                                                                      3. Results discussion <br />  <br /> 2. Material methods                                                              3.1. Using yeast touchstone essentiality prediction <br />  <br /> 2.1. Datasets                                                                           central hypothesis implemented present target dis- <br />                                                                                      covery pipeline  protein likely essential    predicted Plasmodium spp. proteomes downloaded                           similar proteins proteome conserved PlasmoDB (http://www.plasmodb.org/common/downloads)                             orthologues related species. applying malaria <br /> (Aurrecoechea et al., 2009), Saccharomyces cerevisiae proteome                   parasites, tested hypothesis yeast S. cerevisiae SGD (Saccharomyces genome database; http://www                             essentiality proteins tested systematic knock loads.yeastgenome.org/) (Engel et al., 2010), Homo sapiens pro-                  respective genes (Winzeler et al., 1999). current null mu- <br /> teome EBI (ftp://www.ftp.ebi.ac.uk/pub/databases/integr8/                       tant dataset Saccharomyces Genome Database comprises <br /> fasta/proteomes) (Mulder et al., 2008), UniProt                  5477 genes, 1109 essential. test <br /> (http://www.uniprot.org/taxonomy) (Magrane Consortium,                           given S. cerevisiae protein unique, best match <br /> 2011). P. falciparum 3D7 cell cycle expression data (Le Roch et al.,                 (excluding itself) proteome identi&#239;&#172;?ed Blastp <br /> 2003) obtained PlasmoDB, using threshold                          (Altschul et al., 1990), Needleman-Wunsch global align- <br /> expression E &gt; 10 probe signal distribution log P &lt;  0.5 pro-                 ment (Needleman Wunsch, 1970) performed posed authors (Le Roch et al., 2003). S. cerevisiae deletion                  proteins. median global identity S. cerevisiae <br /> phenotype data SGD (http://www.downloads.yeastge-                          protein best match 15.7% (red bar Fig. 1). 14% nome.org/curation/literature/phenotype_data.tab). Proteins                      proteins match median essential, com- <br /> termed essential phenotype knock (mutant                           pared 24% median (Fig. 1). difference type = &#226;&#8364;&#732;null&#226;&#8364;&#8482;) corresponding gene &#226;&#8364;&#732;inviable&#226;&#8364;&#8482;. TDRtar- <br /> gets web resource (http://www.tdrtargets.org) (Magarinos et al., <br /> 2012), BRENDA database (http://www.brenda-enzy-                           698 (24%)                   407 (14%) essential proteins <br /> mes.org) (Scheer et al., 2011) used identify proteins precedence interaction small molecule chemical inhibi- <br />                                                                                            200 <br /> tors. Essentiality Plasmodium berghei orthologues                                                                            S. cerevisiae RMgmDB (Rodent Malaria genetically modi&#239;&#172;?ed Parasites;                              150                                            n = 5,887 <br />                                                                                                                                                , <br /> http://www.pberghei.eu/) (Janse et al., 2011).                                             100 <br />  <br />                                                                                              50 <br /> 2.2. Programs <br />                                                                                                         10     20    30     40    50    60     70    80     90    100 <br />    InParanoid obtained http://www.inparanoid.sbc.su.se QuickParanoid http://www.pl.postech.ac.kr/QuickPara-                        Fig. 1. S. cerevisiae non-redundant proteins. Frequency distribution similar- <br /> noid. BLAST 2.2.17 (Altschul et al., 1990) downloaded                       ities predicted S. cerevisiae proteins best match NCBI (ftp.ncbi.nlm.nih.gov), Needle  EMBOSS suite (Rice                     proteome. 1109 essential proteins, 698 median identity                                                                                      15.7% (red bar) 407 (4 proteins exactly 15.7% identity et al., 2000)) http://www.emboss.sourceforge.net. pro-                      best match). black area represents proteins threshold 615% <br /> grams run Basel Biocomputing Linux cluster (http://                      global identity.  interpretation references colour &#239;&#172;?gure legend, <br /> www.bc2.unibas.ch).                                                                  reader referred web version article.) <br />                                              P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199                                                      193 <br />  <br />  <br /> signi&#239;&#172; (p &lt; 0.0001, sided Chi-square test) supported                                 Table 1 hypothesis &#226;&#8364;&#732;unmatched&#226;&#8364;&#8482; proteins likely                                   Proteomes used study, sizes sources. <br />  <br /> essential (Hannay et al., 2008). Setting cut 615% global                                     Species                          Proteins                  Source                   Version <br /> identity 2648 S. cerevisiae proteins passed (black area Fig. 1),                                     P. falciparum                    5410                      PlasmoDB                 8.1 <br /> 26% essential (Fig. 2).                                                                   P. vivax                         5396                      PlasmoDB                 8.1 <br />    determine S. cerevisiae proteins conserved                                       P. berghei                       4861                      PlasmoDB                 8.1 Saccharomycetaceae, set related (Fitzpatrick et al.,                                    P. chabaudi                      5119                      PlasmoDB                 8.1 <br />                                                                                                         P. knowlesi                      5194                      PlasmoDB                 8.1 <br /> 2006) fully sequenced yeasts served reference: Candida                                       P. yoelii                        7724                      PlasmoDB                 8.1 <br /> glabrata, Ashbya gossypii, Kluyveromyces lactis, Candida albicans,                                      H. sapiens                       67,172                    Integr8                  08/2011 <br /> Debaryomyces hansenii, Yarrowia lipolytica. predicted pro-                                      S. cerevisiae                    5887                      SGD                      R64-1-1 <br /> teomes (Table 1) fed InParanoid QuickParanoid                                              gossypii                      4761                      UniProt                  08/2011 <br />                                                                                                         C. albicans                      5727                      UniProt                  08/2011 <br /> (Ostlund et al., 2010), programs utilize reciprocal, proteome- <br />                                                                                                         C. glabrata                      5197                      UniProt                  08/2011 <br /> wide Blastp searches build groups orthologous proteins                                        D. hansenii                      6242                      UniProt                  08/2011 <br /> rank  total 3546 S. cerevisiae proteins possessed ortho-                                      K. lactis                        5074                      UniProt                  08/2011 <br /> logues included yeasts. highly conserved orthology                                       Y. lipolytica                    6414                      UniProt                  08/2011 <br /> clusters, e. median similarity, signi&#239;&#172;?cantly  <br /> riched essential proteins (Fig. 2): 29% S. cerevisiae pro- <br /> teins belonged conserved cluster essential <br />                                                                                                                                                                             P. falciparum <br /> (p &lt; 0.0001, sided Chi-square test). Finally intersect  <br />  <br />  <br />  <br />                                                                                                    Number proteins <br />                                                                                                                         200                                                 n = 5410 sets, unmatched conserved, contained 632 proteins <br />                                                                                                                         150 40% essential (Fig. 2). yeast results supporting working hypothesis, went apply malaria                                                               100 <br /> parasites.                                                                                                              50 <br />  <br /> 3.2. Essentiality prediction P. falciparum <br />                                                                                                                                  10      20       30   40     50      60      70    80      90   100 <br />  <br />     predicted P. falciparum proteome exhibits strikingly low <br />                                                                                                                           % Global identity best match proteome <br /> degree redundancy, large majority proteins exhibit- <br /> ing low level similarity best match                                Fig. 3. P. falciparum non-redundant proteins. Frequency distribution simi- proteome (Fig. 3). described (Section 3.1) S. cere-                            larities predicted P. falciparum proteins best match visiae, closest hit P. falciparum protein determined                              proteome. median shown red, black area represents proteins Blastp. Similarity measured global alignment ex-                                  passing threshold 615% global identity, assumed enriched <br />                                                                                                  essential ones. <br /> pressed percent identical amino acids. median global iden- <br /> tity P. falciparum protein best match 13% (red bar Fig. 3), 3078 5410 proteins passed                               species 26,377, determined InParanoid Quick- threshold 615% global identity best match                                   Paranoid (Ostlund et al., 2010). amounts 78% proteins, <br /> (black area Fig. 3).                                                                          re&#239;&#172;&#8218;ecting close relationship malaria parasites. num- <br />     conserved Plasmodium proteins identi&#239;&#172;?ed compar-                                  ber proteins common clusters conserved ing P. falciparum available genome sequences mam- <br /> malian malaria parasites, P. berghei, Plasmodium chabaudi Plasmodium yoelii (mouse), Plasmodium knowlesi (primate), Plasmodium vivax (human tertian malaria). genomes                                                                                          0                    4000                 8000 <br /> encode total 33,704 predicted proteins (Table 1). common <br /> intersect proteins orthology clusters represented                                                                                                                                       P. falciparum            3610 <br />  <br />                                                                                                                                       P. vivax <br />                            Saccharomyces cerevisiae <br />                       ae <br />                  etacea <br />  <br />  <br />  <br />  <br />                                    Candida glabrata <br />                                    C                                                                                                  P. knowlesi <br />                                     Ashbya gossypii <br />               myce <br />  <br />  <br />  <br />  <br />                                    y     y <br />                                 Kluyveromyces lactis                                                                                  P. berghei <br />           harom <br />  <br />  <br />  <br />  <br />                                    Candida albicans <br />                                                                                                                                       P. yoelii <br />       Sacch <br />  <br />  <br />  <br />  <br />                              Debaryomyces hansenii <br />       S <br />  <br />  <br />  <br />  <br />                                   Yarrowia lipolytica                                                                                 P. chabaudi <br />  <br />  <br />                                                                                                                                                             Species-specif ic proteins <br />                                                         Total    Essential                                                                                  Plasmodium spp. rodents <br />                                                                                                                                                             Plasmodium spp. primates <br />       S cerevisiae proteome <br />       S.                                                5,477 <br />                                                         5 477      1,109 <br />                                                                    1 109       20%                                                                          Plasmodium spp. analyzed <br />       Unmatched &gt;15% global <br />                      g      identity <br />                                    y                    2,648 <br />                                                          ,          677        26%                                                                          Irregular distribution orthologues <br />  <br />       Conserved Saccharomycetaceae                   1,773       514        29% <br />                                                                                                  Fig. 4. conserved malaria proteome. cladogram depicts phylogeny       Unmatched <span id='am-45' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-46' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; conserved                             632         253        40%               fully sequenced malaria parasites mammals (Martinsen et al., 2008). bars <br />                                                                                                  divide predicted proteome proteins present species itself, <br /> Fig. 2. Testing essentiality yeasts. cladogram depicts phylogeny             proteins orthologues Plasmodium spp. sharing intermediate hosts, fully sequenced Saccharomycetaceae (Fitzpatrick et al., 2006). Eliminating S.            proteins orthologues analyzed species (3610 P. falciparum). <br /> cerevisiae proteins possess matches proteome (gray area                     white sections represent proteins irregular distribution insofar Fig. 1), conserved analyzed yeasts, enriches              presence orthologues did correlate host speci&#239;&#172;?city  essential proteins.                                                                          phylogeny). <br />  194                                                        P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199 <br />  <br />  <br /> analyzed Plasmodium spp., size proteomes corre- <br />                                                                                                                    Total proteome                                                             5,410 <br /> lated number species-speci&#239;&#172;?c proteins (Fig. 4). 5410 predicted P. falciparum proteins, 3610 (67%) possessed                                                     Unique P. falciparum                                                    3,078 orthologue Plasmodium species <br /> (Fig. 4). 630 P. falciparum proteins did orthologue                                                 Conserved Plasmodium spp.                                               944 species 139 orthologue P. vivax P. <br /> knowlesi malaria parasites rodents (Fig. 4). dif-                                            good match human proteome                                        426 <br /> ferent protein subsets outlined Fig. 4 parasite avail- <br /> able Supplementary excel &#239;&#172;?le (supplement1.xls). intersect                                                 Expressed asexual erythrocytic stages                               288 subsets, P. falciparum unique (n = 3078) P. falcipa- <br /> rum highly conserved  e. median, n = 1805), contained                                                      Predicted function enzyme, receptor, transporter                        40 <br /> 944 proteins posit &#226;&#8364;&#8220; based results S. cerevi- <br /> siae &#226;&#8364;&#8220; enriched essential ones.                                                                       Fig. 6. genome target. Overview silico pipeline narrow                                                                                                                 potential target space P. falciparum. 40 identi&#239;&#172;?ed candidates listed                                                                                                                 Table 2. <br /> 3.3. Comparative genomics parasite host <br />                                                                                                                 complete list available Supplementary excel &#239;&#172;?le (supple- <br />     predicted protein P. falciparum best match                                           ment2.xls). half (n = 178) assigned human proteome identi&#239;&#172;?ed Blastp, ranking obtained                                                  putative function. potential conserved <br /> hits ascending expectancy (E-values). similarity                                               malaria parasites absent humans, proteins P. falciparum protein closest match H. sapiens                                                 time  removed candidate target set. quanti&#239;&#172;?ed Needleman-Wunsch global alignment                                                         proteins removed manually likely  local alignment provided Blast misleading                                               function susceptible inhibition, e.g. structural pro- similarity unrelated proteins                                                  teins surface antigens. Focusing proteins clear, &#239;&#172;?ned smaller common domain). frequency distribution                                                    putative, functional annotation PlasmoDB  global identity human proteins P. falciparum proteome                                              zyme, receptor transporter resulted &#239;&#172;?nal set 40 P. falcipa- shown Fig. 5. conserved proteins parasite                                                rum candidate drug targets (Fig. 6). host ubiquitins, histones, calmodulin, tubulin. median global identity P. falciparum protein best <br />                                                                                                                 3.5. Assessing essentiality 40 identi&#239;&#172;?ed candidate targets <br /> match H. sapiens 10% (red bar Fig. 5). Proteins exhibiting <br /> &gt;10% identity host (gray area Fig. 5) removed                                                                                                                     predicted essential proteins attempts potential target space, trimmed                                                                                                                 disrupt corresponding gene P. falciparum (Table 3). <br /> 426 proteins (Fig. 6). highly signi&#239;&#172;                                                                                                                 cases disruption attempted deletion Blastp E-values human protein, e.g. P. falciparum kinases <br />                                                                                                                 recombination gene deletion obtained (Santiago <br /> listed Table 2, short region high similarity                                                                                                                 et al., 2004; Wang et al., 2004; Abdi et al., 2010). fourth disparate proteins return high local alignment <br />                                                                                                                 gene ispH (HMBPP reductase; PFA0225w), introduction ham- <br /> score  case P. falciparum kinases, region cat- <br />                                                                                                                 merhead ribozyme targeting transcript lethal (Vinayak <br /> alytic domain).  selective targeting possible <br />                                                                                                                 Sharma, 2007).  genes experimentally domains. <br />                                                                                                                 examined appeared essential. Similarly, small number                                                                                                                 proteins&#226;&#8364;&#8482; orthologues P. berghei Toxoplasma gondii 3.4. Blood-stage expression putative targets                                                                 subject directed gene knock mutagenesis <br />                                                                                                                 experiments aimed assaying essentiality. orthologues <br />    majority genes P. falciparum expressed stage-                                             kinases PF13_0166 CDPK7 (PF11_0242) sug- <br /> dependent manner (Llinas DeRisi, 2004), asex-                                              gested essential P. berghei (Tewari et al., 2010). T. gondii, <br /> ual gametocyte stages attempt &#239;&#172;?nd targets given                                                  orthologue apurinic/apyrimidinic endonuclease roles pathology transmission. relied published                                             (PF13_0176) shown essential conditional knockout <br /> microarray dataset (Le Roch et al., 2003) P. falciparum cultures                                             (Onyango et al., 2011) GAP40 (PFE0785c) essen- <br /> synchronized sorbitol treatment identify potential targets                                            tial glideosome complex, protein itself present blood stages. Retaining products                                            veri&#239;&#172;?ed essential (Frenal et al., 2010). gene genes expressed ring stages, trophozoites,                                                  non-essential. P. berghei CDPK6 (PF11_0239) orthologue <br /> schizonts, reduced target space 288 proteins (Fig. 6).                                               successfully deleted double crossover (Coppi et al., <br />                           400 <br />                                                                                                                 2007). experimental assays essentiality indicate <br />                                                                                                                 predicted proteins essential,                                                                                                                 non-essential, supporting method enrich essentiality. <br />                           300 <br />  P. falciparum proteins <br />  <br />  <br />  <br />  <br />                           200                                                                                   3.6. Prediction druggability <br />  <br />                           100                                                                                       order recognize important role chemical accessibil- <br />                                                                                                                 ity ranking putative drug targets looked druggability <br />                                                                                                                 information predicted essential targets. Druggability  <br />                                      10     20     30     40     50     60     70     80      90    100         scribes properties protein make able interact <br />                                                                                                                 drug-like molecule. proteins physiochemical                                 % Global identity best match H. sapiens <br />                                                                                                                 structural properties make unlikely bind com- <br /> Fig. 5. Parasite vs. host. Frequency distribution percent global identity H. <br />                                                                                                                 pound necessary characteristic drug, sapiens P. falciparum proteome. red bar marks median, gray areas                             genome analyses estimated 10% genome P. falciparum proteins excluded potential target space.                                  represent druggable proteins (Hopkins Groom, 2002).                                          P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199                                     195 <br />  <br />  <br /> Table 2 candidate drug targets P. falciparum identi&#239;&#172;?ed outlined Fig. 6, putative function, Blastp E-value best hit human proteome, druggability index <br /> (D ). categories (italics) arbitrary mainly meant illustrate functional diversity identi&#239;&#172;?ed targets. <br />  <br />   Accession                              Putative function                                                        %<span id='am-103' about='oboInOwl:id' typeof='owl:Thing'>Id</span> Hsa                        E Hsa                         D  <br />   Metabolism <br />   PF08_0095                              Dihydropteroate synthetase (2.5.1.15)                                     0.8                           8.5                           0.8 <br />   MAL8P1.156                             Mannose-6-P isomerase (5.3.1.8)                                           6.9                           1E-8                          0.6 <br />   PF08_0068                              FAD-dependent monooxygenase                                               6.0                           0.07                          0.5 <br />   PF14_0334                              Ornithine aminotransferase (2.6.1.13)                                     4.3                           1E-14                         0.3 <br />   PF13_0234                              PEP carboxykinase (4.1.1.49)                                              4.0                           3.6                           0.2 <br />   PF14_0246                              PEP carboxylase (4.1.1.31)                                                3.8                           4.5                           0.1 <br />   PFI1180w                               Patatin-like phospholipase                                                6.3                           7.4                           0.1 <br />   PF10_0221                              HMB-PP synthase (GcpE, 1.17.7.1)                                          6.5                           2.2 <br />   PFA0225w                               HMB-PP reductase (1.17.1.2)                                               7.3                           5.3 <br />   MAL7P1.88                              Thioredoxin-like protein                                                  3.1                           4.9 <br />   PFI1340w                               Fumarate hydratase (Fumarase, 4.2.1.2)                                   10.0                           2.7 <br />   PF08_0132                              Glutamate dehydrogenase C (1.4.1.2)                                       8.1                           7.2 <br />   PFD0285c                               Lysine decarboxylase (4.1.1.18)                                           5.6                           7.4 <br />   PFL0620c                               Glycerol-3-P acyltransferase (2.3.1.15)                                   9.8                           1.0 <br />   PF14_0250                              Lipase                                                                    3.3                           8.1 <br />   Nucleic acids <br />   PF14_0273                              RNA methyltransferase                                                     9.8                           7E-30                         0.7 <br />   PF13_0080                              Telomerase reverse transcriptase (2.7.7.49)                               0.3                           0.04                          0.5 <br />   PF13_0176                              Apur./apyr. endonuclease (4.2.99.18)                                      2.6                           2.6                           0.2 <br />   MAL13P1.311                            Exonuclease                                                               5.9                           7E-4                          0.1 <br />   MAL13P1.328                            DNA topoisomerase VI, b subunit                                           4.9                           7.1 <br />   PFE0675c                               DNA photolyase (4.1.99.3)                                                 3.7                           0.06 <br />   PF13_0048                              NUDIX hydrolase (3.6.1.17)                                                8.4                           8E-35 <br />   PF08_0111                              RNA helicase                                                              7.8                           8E-15 <br />   PFC0980c                               RNA triphosphatase                                                        8.5                           8.6 <br />   Signaling <br />   PF11_0239                              Ca2+-dependent protein kinase 6                                           7.8                           9E-34                         0.5 <br />   PF11_0242                              Ca2+-dependent protein kinase 7                                           9.0                           6E-41                         0.5 <br />   PF13_0258                              Serine/threonine protein kinase TKL3                                      4.8                           3E-21                         0.5 <br />   PFA0515w                               <span id='am-21' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="MPI" datatype="xsd:string"></span>Phosphatidylinositol</span>-4-P-5-kinase (2.7.1.68)                              7.0                           1E-24                         0.5 <br />   PF13_0166                              Protein kinase                                                            8.9                           7E-4                          0.3 <br />   PF07_0024                              Inositol phosphatase                                                      7.3                           1E-31                         0.3 <br />   MAL7P1.18                              Serine/threonine protein kinase                                           4.7                           5E-13                         0.1 <br />   MAL7P1.64                              G protein-coupled receptor                                                2.5                           93                            0.1 <br />   PF14_0143                              Atypical protein kinase, ABC-1 family                                     3.8                           4E-26 <br />   PF14_0614                              Phosphatase                                                               6.1                           4E-5 <br />   Protease/chaperone <br />   PF14_0382                              Stromal-processing peptidase                                              4.4                           0.001                         0.5 <br />   PF10_0032                              DnaJ protein                                                              2.4                           7E-4 <br />   Transporter <br />   PF11_0092                              Mechanosensitive <span id='am-39' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-40' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> channel                                              5.0                           11                            0.1 <br />   PF13_0019                              Na+:H+ antiporter                                                         6.6                           0.01                          0.1 <br />   PFI0785c                               Sugar transporter                                                         2.8                           0.09                          0.1 <br />   PFE0785c                               Metabolite transporter/glideosome component                               0.7                           10 <br />  <br />  TDRtargets project druggability score assigned                             inhibitors.  dihydropteroate synthetase (DHPS; <br /> zymes P. falciparum proteome (Aguero et al., 2008; Mag-                              PF08_0095) validated targets clinically <br /> arinos et al., 2012). score predicts suitability protein                      used antimalarials, consistent value approach binding drug like molecule, aggregate fac-                         identifying essential targets. additional 13 proteins ortho- <br /> tors including physiochemical structural features pro-                             logues identi&#239;&#172;?ed small molecule inhibitors. Identi&#239;&#172;?cation tein sequence similarity validated druggable proteins                             inhibitors provides useful starting points identi- organisms (Al-Lazikani et al., 2007; Gaulton et al., 2012).                          fying small molecules interrogate Plasmodium 40 targets predicted essential, 23 positive druggability                          targets, indicates likely potentially <br /> scores (Table 2). highest scores predicted                              druggable targets. Compounds curated exclude facile druggable structures prioritized                             inhibitors natural amino acids, <span id='am-57' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-58' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> NaCl, potential targets low zero druggability scores.                              systematic attempt rate drug-like quality higher druggability scores likely indicative prom-                          predicted inhibitors. kinases list slightly mod- <br /> ising drug target-like proteins, positive score indicates                             i&#239;&#172;?ed approach taken, kinase inhibitors structural physiochemical similarity proteins pre-                            known cross reactive multiple kinases. empirical <br /> cedence interaction drug-like molecule.                                            cut 30% local sequence identity kinase domains     search proteins speci&#239;&#172;?c compounds starting                                 used identify homologues potential inhibitors points inhibitor development searched proteins                           TDR/ChEMBL data (Table 3). identi&#239;&#172;?ed targets inhibitors, ortho- <br /> logues identi&#239;&#172;?ed inhibitors. Searches conducted                              3.7. Experimental amenability predicted targets <br /> TDRTargets database, ChEMBL database BRENDA data- <br /> base identify targets. 40 predicted essential P. falci-                           Tractability target developing assay inhibi- <br /> parum genes, 2 directly validated targets                                 tion major factor prioritizing promising drug targets  196                                    P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199 <br />  <br />  <br /> Table 3 <br /> Identi&#239;&#172;?ed targets, evidence essentiality apicomplexan parasites (Pfa, P. falciparum; Pbe, P. berghei; Tgo, T. gondii), known inferred inhibitors, published vitro <br /> assays. <br />  <br />   Accession       Enzyme                Essentiality   Inhibitors       Assay    References <br />   PF08_0095       DHPS                  Pfa            Sulfonamides     Yes      Wang et al. (2010) <br />   PF14_0273       RNA                                  8 inferred       Yes      Riguet et al. (2005), Leber et al. (2009) <br />                   methyltransferase <br />   MAL8P1.156      M6P isomerase                        8                         Wells et al. (1995), Salvati et al. (2001) Foret et al. (2009) <br />   PF11_0242       CDPK7                 Pbe            72 inferred               12 Publications <br />   PF11_0239       CDPK6                 Pbe     2488                      189 Publications <br />                                                        inferred <br />   PFA0515w        PI-4-P-5-kinase                      1                         Kobayashi et al. (2005) Santiago et al. (2004) <br />   PF13_0080       Telomerase RT                        Berberine                 Sriwilaijareon et al. (2002) <br />   PF13_0258       TKL3                  Pfa                             Yes      Abdi et al. (2010) <br />   PF13_0166       Protein kinase        Pbe <br />   PF14_0334       OAT                                  &gt;10              Yes      Strecker (1965), Levillain et al. (2000), Wang et al. (2007), Stranska et al. (2008) Gafan <br />                                                                                  et al. (2001) <br />   PF13_0176       AP endonuclease       Tgo            &gt;20              Yes      Onyango et al. (2011), Zawahir et al. (2009), Bapat et al. (2010), Adhikari et al. (2012)                                                                                  Haltiwanger et al. (2000) <br />   PF13_0234       PEP                                                   Yes      Hayward (2000) <br />                   carboxykinase <br />   PF14_0246       PEP carboxylase                      9                Yes      Pairoba et al. (1996), McDaniel Siu (1972) Jenkins (1989) <br />   PFD0285c        Lysine                               4                         Yamamoto et al. (1991) Takatsuka et al. (1999) <br />                   decarboxylase <br />   PFI1340w        Fumarase                             2                Yes      Flint (1994), Beeckmans Van Driessche (1998) Bulusu et al. (2011) <br />   PFL0620c        G3P                   Pfa            &gt;40              Yes      Santiago et al. (2004), Yamashita Numa (1981), Coleman (1988) Wydysh et al. <br />                   acyltransferase                                                (2009) <br />   PF13_0048       NUDIX hydrolase                      &gt;20                       Guranowski et al. (2003, 2009) Branson et al. (2009) <br />   PF08_0132       GDH C                                &gt;20              Yes      Li et al. (2007, 2009), Rodriguez-Acosta et al. (1999) <br />   PFA0225w        HMB-PP                Pfa                             Yes      Vinayak Sharma (2007) Rohrich et al. (2005) <br />                   reductase <br />  <br />  <br />  <br />  <br /> infectious diseases smaller research communities,                               candidate drug targets P. falciparum shown Table <br /> compared working human proteins, means fewer pro-                               2. con&#239;&#172;?dent set contains promising leads tein reagents available. exacerbated malaria research                        includes known targets dihydropteroate synthetase P. falciparum genome extremely + T rich,                          (PF08_0095), target sulfonamides (dihydrofolate synthetase <br /> proteins long low-complexity insertions, making recombi-                               included 20% identical human mitochondrial <br /> nant protein production challenging. High throughput attempts                               tetrahydrofolylpolyglutamate synthase, EC 6.3.2.17), enzymes generate recombinant Plasmodium enzymes relatively high                             non-mevalonate pathway isoprenoid synthesis. failure rates (Abdi et al., 2010), presence existing assay                     potential target apicoplast stromal processing particularly relevant criteria target prioritization.                     peptidase (PF14_0382), characterized fore prioritized proteins researchers developed                           P. falciparum (van Dooren et al., 2002). Investigational targets iden- direct vitro assay P. falciparum enzyme. 40                      ti&#239;&#172;?ed include glutamate dehydrogenase C (PF08_0132) (Aparicio <br /> predicted essential enzymes vitro assays described                           et al., 2010; Storm et al., 2011) Ca2+-dependent protein kinases <br /> P. falciparum, cases readily                              (PF11_0239, PF11_0242) (Kato et al., 2008; Ojo et al., 2010; Lim <br /> amenable high throughput inhibitor screening.                                            et al., 2012). addition, enzymes tar- <br />                                                                                             geted systems, e.g. apurinic/apyrimidinic endonuclease <br /> 3.8. Candidate antimalarial drug targets                                                    (PF13_0176) (Tell et al., 2010) telomerase reverse transcriptase <br />                                                                                             (PF13_0080) (Chen et al., 2009) antitumor agents, phospho- <br />     approach presented  starting com-                            mannose isomerase (MAL8P1.156) antifungals. Phosphoman- <br /> plete proteome, means embracing sense                           nose key enzyme yeast cell wall synthesis (Mastrolorenzo discover potential drug target. essential pro-                             et al., 2000; Wills et al., 2001); knowledge, P. falciparum <br /> teins known drug targets missed, P. falciparum                             orthologue far studied. surprising &#239;&#172;?nd list <br /> enzymes possess human orthologues exhibit exploitable                             candidate targets phosphoenolpyruvate carboxylase <br /> differences  differences sequence,                        (PF14_0246), enzyme responsible carbon &#239;&#172;?xation C4 case glyceraldehyde-3-phosphate dehydrogenase (Satc-                            plants. P. falciparum orthologue characterized pro- <br /> hell et al., 2005) regulation, demonstrated dihydrofolate                      posed &#239;&#172;?x atmospheric carbon (McDaniel Siu, <br /> reductase (Zhang Rathod, 2002). proven targets                               1972). process essential, PEP carboxylase eliminated possess paralogues, e.g. falci-                         attractive target inhibitors identi&#239;&#172;?ed C4 plant <br /> pains (Teixeira et al., 2011). Using rigorous &#239;&#172;?lters, tried max-                      herbicides (Madhusudana et al., 1980; Jenkins, 1989; Pairoba <br /> imize speci&#239;&#172;?city target prediction cost sensitivity,                       et al., 1996). candidate targets taking account large number false negatives. aim                              identi&#239;&#172;?ed signaling enzymes nutrient transporters (Table 2). identify potential antimalarial drug targets come sizeable list promising targets                            4. Conclusion <br />   serve starting points rational drug discovery (ii) provide new insights peculiarities vulnerable points                             Translating pathogen&#226;&#8364;&#8482;s genome sequence new drugs malaria parasites. applied selection criteria rigor-                        key challenge post-genomic biology. strategies ous (Fig. 6), list potential targets expanded                         prediction drug targets essential unique par- <br /> using stringent &#239;&#172;?lters. present approach identi&#239;&#172;?ed 40                              asite proposed (Payne et al., 2001; Joyce Palsson, <br />                                           P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199                                          197 <br />  <br />  <br /> 2008; Holman et al., 2009; Crowther et al., 2010; Deng et al., 2010;                          Beeckmans, S., Van Driessche, E., 1998. Pig heart fumarase contains distinct <br />                                                                                                   substrate-binding sites differing af&#239;&#172;?nity. J. Biol. Chem. 273, 31661&#226;&#8364;&#8220;31669. <br /> Doyle et al., 2010; Magarinos et al., 2012). implement <br />                                                                                               Branson, K.M., Mertens, H.D., Swarbrick, J.D., Fletcher, J , Kedzierski, L., Gayler, K.R., <br /> phylogenomic comparison clade related parasites                                 Gooley, P.R., Smith, B.J., 2009. Discovery inhibitors lupin diadenosine 50 ,5000 - <br /> fer protein essentiality, referring model organ-                              P(1), P(4)-tetraphosphate hydrolase virtual screening. Biochemistry 48, <br /> ism understood phylogenetically                                        7614&#226;&#8364;&#8220;7620. <br />                                                                                               Bulusu, V., Jayaraman, V., Balaram, H., 2011. Metabolic fate fumarate, distant parasites. addition, estimate redundancy                                     product purine salvage pathway intraerythrocytic stages parasite&#226;&#8364;&#8482;s proteome based global alignments quan-                                Plasmodium falciparum. J. Biol. Chem. 286, 9236&#226;&#8364;&#8220;9245. <br /> tify protein similarity. proposed target identi&#239;&#172;?cation                               Charman, S , Arbe-Barnes, S., Bathurst, C., Brun, R., Campbell, M., Charman, W.N., <br />                                                                                                   Chiu, F.C., Chollet, J., Craft, J.C., Creek, D.J., Dong, Y., Matile, H., Maurer, M., <br /> pipeline does rely experimental data predict essentiality,                              Morizzi, J., Nguyen, T., Papastogiannidis, P., Scheurer, C., Shackleford, D.M., generally applicable. identi&#239;&#172;?ed candidate tar-                               Sriraghavan, K., Stingelin, L., Tang, Y., Urwyler, H., Wang, X., White, K.L., Wittlin, <br /> gets P. falciparum  de&#239;&#172;?nition, present Plasmo-                             S., Zhou, L., Vennerstrom, J.L., 2011. Synthetic ozonide drug candidate OZ439 <br />                                                                                                   offers new hope single-dose cure uncomplicated malaria. Proc. Natl. <br /> dium spp. analyzed. Conservation targets P. falciparum                                 Acad. Sci. USA 108, 4400&#226;&#8364;&#8220;4405. P. vivax important eventual new drugs need ac-                             Chen, H., Li, Y., Tollefsbol, T.O., 2009. Strategies targeting telomerase inhibition. Mol. <br /> tive parasites. Conservation candidate targets                                Biotechnol. 41, 194&#226;&#8364;&#8220;199. <br />                                                                                               Coleman, R , 1988. Hepatic sn-glycerol-3-phosphate acyltransferases: effect human- rodent-pathogenic Plasmodium spp. desir-                                    monoacylglycerol analogs mitochondrial microsomal activities. <br /> able  mouse models indispensable drug                                    Biochim. Biophys. Acta 963, 367&#226;&#8364;&#8220;374. <br /> development process. presented target discovery pipeline                               Coppi, , Tewari, R., Bishop, J.R., Bennett, B.L., Lawrence, R., Esko, J.D., Billker, O., <br />                                                                                                   Sinnis, P., 2007. Heparan sulfate proteoglycans provide signal Plasmodium <br /> corroborated presence known antimalarial targets                                                                                                   sporozoites stop migrating productively invade host cells. Cell Host output list (Table 2). hope newly identi&#239;&#172;?ed candi-                                Microbe 2, 316&#226;&#8364;&#8220;327. <br /> <span id='am-14' about='oboInOwl:date' typeof='owl:Thing'>date</span> targets support development novel antimalarials.                             Crowther, G.J., Shanmugam, D., Carmona, S.J., Doyle, M , Hertz-Fowler, C., <br />                                                                                                   Berriman, M., Nwaka, S., Ralph, S , Roos, D.S., Van Voorhis, W.C., Aguero, F., <br />                                                                                                   2010. Identi&#239;&#172;?cation attractive drug targets neglected-disease pathogens <br /> Acknowledgments                                                                                   using silico approach. PLoS Negl. Trop. Dis. 4, e804. <br />                                                                                               Deng, J., Deng, L., Su, S., Zhang, M., Lin, X., Wei, L., Minai, , Hassett, D.J., Lu, L.J., <br />                                                                                                   2010. Investigating predictability essential genes distantly related <br />    wish thank Basel Computational Biology Center                                    organisms using integrative approach. Nucleic Acids Res. 39, 795&#226;&#8364;&#8220;807. <br /> runtime Linux cluster. SAR funded Australian                             Doyle, M , Gasser, R.B., Woodcroft, B.J., Hall, R.S., Ralph, S , 2010. Drug target <br /> search Council Future Fellowship (FT0990350). BJW supported                                   prediction prioritization: using orthology predict essentiality parasite <br />                                                                                                   genomes. BMC Genomics 11, 222. University Melbourne Research scholarship.                                            Engel, S.R., Balakrishnan, R., Binkley, G., Christie, K.R., Costanzo, M.C., Dwight, S.S., <br />                                                                                                   Fisk, D.G., Hirschman, J.E., Hitz, B.C., Hong, E.L., Krieger, C.J., Livstone, M.S., <br />                                                                                                   Miyasato, S.R., Nash, R., Oughtred, R., Park, J., Skrzypek, M.S., Weng, S., Wong, <br /> Appendix  Supplementary data                                                                    E.D., Dolinski, K., Botstein, D., Cherry, J.M., 2010. Saccharomyces Genome <br />                                                                                                   Database provides mutant phenotype data. Nucleic Acids Res. 38, D433&#226;&#8364;&#8220;436. <br />                                                                                               Fatumo, S., Plaimas, K., Mallm, J.P., Schramm, G., Adebiyi, E., Oswald, M., Eils, R., <br />    Supplementary data associated article                                Konig, R., 2009. Estimating novel potential drug targets Plasmodium online version, http://dx.doi.org/10.1016/j.ijpddr.2012.07.                                falciparum analysing metabolic network knock strains silico. <br /> 002.                                                                                              Infect. Genet. Evol. 9, 351&#226;&#8364;&#8220;358. <br />                                                                                               Fitzpatrick, D , Logue, M.E., Stajich, J.E., Butler, G., 2006. fungal phylogeny based <br />                                                                                                   42 complete genomes derived supertree combined gene analysis. <br /> References                                                                                        BMC Evol. Biol. 6, 99. <br />                                                                                               Flint, D.H., 1994. Initial kinetic mechanistic characterization Escherichia coli <br />                                                                                                   fumarase  Arch. Biochem. Biophys. 311, 509&#226;&#8364;&#8220;516. <br /> Abdi, , Eschenlauer, S., Reininger, L., Doerig, C., 2010. SAM domain-dependent <br />                                                                                               Foret, J., Courcy, B., Gresh, N., Piquemal, J.P., Salmon, L., 2009. Synthesis     activity PfTKL3, essential tyrosine kinase-like kinase human malaria <br />                                                                                                   evaluation non-hydrolyzable <span id='am-68' about='obo:IMR_0200053' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00275" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="D-Mannose 6-phosphate" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200053" datatype="xsd:string"></span>D-mannose 6-phosphate</span> surrogates reveal 6- <br />     parasite Plasmodium falciparum. Cell. Mol. Life Sci. 67, 3355&#226;&#8364;&#8220;3369. <br />                                                                                                   deoxy-6-dicarboxymethyl-<span id='am-104' about='obo:IMR_0200230' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-105' property="oboInOwl:id" content="IMR:0200230" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="D-Mannose" datatype="xsd:string"></span><span id='am-107' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="KEGG:C00159" datatype="xsd:string"></span>D-mannose</span> new strong inhibitor Adhikari, S., Manthena, P.V., Kota, K.K., Karmahapatra, S.K., Roy, G., Saxena, R., Uren, <br />                                                                                                   phosphomannose isomerases. Bioorg. Med. Chem. 17, 7100&#226;&#8364;&#8220;7107. <br />     , Roy, R., 2012. comparative study recombinant mouse human <br />                                                                                               Frenal, K., Polonais, V., Marq, J.B., Stratmann, R., Limenitakis, J., Soldati-Favre, D., <br />     apurinic/apyrimidinic endonuclease. Mol. Cell. Biochem. 362, 195&#226;&#8364;&#8220;201. <br />                                                                                                   2010. Functional dissection apicomplexan glideosome molecular <br /> Aguero, F., Al-Lazikani, B., Aslett, M., Berriman, M., Buckner, F.S., Campbell, R.K., <br />                                                                                                   architecture. Cell Host Microbe 8, 343&#226;&#8364;&#8220;357. <br />     Carmona, S., Carruthers, M., Chan, W., Chen, F., Crowther, G.J., Doyle, M , <br />                                                                                               Gafan, C., Wilson, J., Berger, L.C., Berger, B.J., 2001. Characterization ornithine <br />     Hertz-Fowler, C., Hopkins, L., McAllister, G., Nwaka, S., Overington, J.P., Pain, <br />                                                                                                   aminotransferase Plasmodium falciparum. Mol. Biochem. Parasitol. 118, 1&#226;&#8364;&#8220; <br />     , Paolini, G.V., Pieper, U., Ralph, S , Riechers, , Roos, D.S., Sali, , <br />                                                                                                   10. <br />     Shanmugam, D., Suzuki, T., Van Voorhis, W.C., Verlinde, C.L., 2008. Genomic- <br />                                                                                               Gardner, M.J., Hall, N., Fung, E., White, O., Berriman, M., Hyman, R.W., Carlton, J.M., <br />     scale prioritization drug targets: TDR Targets database. Nat. Rev. Drug <br />                                                                                                   Pain, , Nelson, K.E., Bowman, S., Paulsen, T., James, K., Eisen, J , Rutherford, <br />     Discov. 7, 900&#226;&#8364;&#8220;907. <br />                                                                                                   K., Salzberg, S.L., Craig, , Kyes, S., Chan, M.S., Nene, V., Shallom, S.J., Suh, B., <br /> Al-Lazikani, B., Gaulton, , Paolini, G., Lanfear, J., Overington, J., Hopkins, L. <br />                                                                                                   Peterson, J., Angiuoli, S., Pertea, M., Allen, J., Selengut, J., Haft, D., Mather, M.W., <br />     (2007). molecular basis predicting druggability.  Bioinformatics &#226;&#8364;&#8220; <br />                                                                                                   Vaidya, B., Martin, D.M., Fairlamb, H., Fraunholz, M.J., Roos, D.S., Ralph, S , <br />     Genomes Therapies.  Holy Grail: Molecular Function, vol. 3, <br />                                                                                                   McFadden, G , Cummings, L.M., Subramanian, G.M., Mungall, C., Venter, J.C., <br />     Wiley, T.L. Weinheim, pp. 1315&#226;&#8364;&#8220;1334. <br />                                                                                                   Carucci, D.J., Hoffman, S.L., Newbold, C., Davis, R.W., Fraser, C.M., Barrell, B., <br /> Altschul, S.F., Gish, W., Miller, W., Myers, E.W., Lipman, D.J., 1990. Basic local <br />                                                                                                   2002. Genome sequence human malaria parasite Plasmodium falciparum. <br />     alignment search tool. J. Mol. Biol. 215, 403&#226;&#8364;&#8220;410. <br />                                                                                                   Nature 419, 498&#226;&#8364;&#8220;511. <br /> Aparicio, M., Marin-Menendez, , Bell, , Engel, P.C., 2010. Susceptibility                                                                                               Gaulton, , Bellis, L.J., Bento, P., Chambers, J., Davies, M., Hersey, , Light, Y., <br />     Plasmodium falciparum glutamate dehydrogenase inhibitors possible new <br />                                                                                                   McGlinchey, S., Michalovich, D., Al-Lazikani, B., Overington, J.P., 2012. ChEMBL: <br />     antimalarial target. Mol. Biochem. Parasitol. 172, 152&#226;&#8364;&#8220;155. <br />                                                                                                   large-scale bioactivity database drug discovery. Nucleic Acids Res. 40, <br /> Armstrong, C.M., Goldberg, D.E., 2007. FKBP destabilization domain modulates <br />                                                                                                   D1100&#226;&#8364;&#8220;1107. <br />     protein levels Plasmodium falciparum. Nat. Methods 4, 1007&#226;&#8364;&#8220;1009. <br />                                                                                               Guranowski, , Starzynska, E., McLennan, G., Baraniak, J., Stec, W.J., 2003. <br /> Aurrecoechea, C., Brestelli, J., Brunk, B.P., Dommer, J., Fischer, S., Gajria, B., Gao, X., <br />                                                                                                   <span id='am-73' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-74' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-78' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span>Adenosine</span>-50 -O-phosphorylated <span id='am-88' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-89' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-96' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span>adenosine</span>-50 -O-phosphorothioylated <br />     Gingle, , Grant, G., Harb, O.S., Heiges, M., Innamorato, F., Iodice, J., Kissinger, <br />                                                                                                   polyols strong inhibitors (symmetrical) (asymmetrical) dinucleoside <br />     J.C., Kraemer, E., Li, W., Miller, J , Nayak, V., Pennington, C., Pinney, D.F., Roos, <br />                                                                                                   tetraphosphatases. Biochem. J. 373, 635&#226;&#8364;&#8220;640. <br />     D.S., Ross, C., Stoeckert Jr., C.J., Treatman, C., Wang, H., 2009. PlasmoDB:                                                                                               Guranowski, , Starzynska, E., Pietrowska-Borek, M., Rejman, D., Blackburn, G.M., <br />     functional genomic database malaria parasites. Nucleic Acids Res. 37, D539&#226;&#8364;&#8220; <br />                                                                                                   2009. Novel diadenosine polyphosphate analogs oxymethylene bridges <br />     543. <br />                                                                                                   replacing oxygen polyphosphate chain: potential substrates Bapat, , Glass, L.S., Luo, M., Fishel, M.L., Long, E.C., Georgiadis, M.M., Kelley, M.R., <br />                                                                                                   inhibitors Ap4A hydrolases. FEBS J. 276, 1546&#226;&#8364;&#8220;1553. <br />     2010. Novel small-molecule inhibitor apurinic/apyrimidinic endonuclease 1 <br />                                                                                               Haltiwanger, B.M., Karpinich, N.O., Taraschi, T.F., 2000. Characterization class II <br />     blocks proliferation reduces viability glioblastoma cells. J. Pharmacol. <br />                                                                                                   apurinic/apyrimidinic endonuclease activities human malaria parasite, <br />     Exp. Ther. 334, 988&#226;&#8364;&#8220;998. <br />                                                                                                   Plasmodium falciparum. Biochem. J. 345 (Pt 1), 85&#226;&#8364;&#8220;89. <br /> Baum, J., Papenfuss, T., Mair, G.R., Janse, C.J., Vlachou, D., Waters, P., Cowman, <br />                                                                                               Hannay, K., Marcotte, E.M., Vogel, C., 2008. Buffering gene duplicates:     F., Crabb, B.S., Koning-Ward, T.F., 2009. Molecular genetics                                                                                                   analysis molecular correlates evolutionary conservation. BMC Genomics <br />     comparative genomics reveal RNAi functional malaria parasites. <br />                                                                                                   9, 609. <br />     Nucleic Acids Res. 37, 3788&#226;&#8364;&#8220;3798. <br />  198                                       P. Ludin et al. / International Journal Parasitology: Drugs Drug Resistance 2 (2012) 191&#226;&#8364;&#8220;199 <br />  <br />  <br /> Hayward, R.E., 2000. Plasmodium falciparum phosphoenolpyruvate carboxykinase                      endonucleases apicomplexan parasite Toxoplasma gondii. DNA Repair <br />     developmentally regulated gametocytes. Mol. Biochem. Parasitol. 107, 227&#226;&#8364;&#8220;                     (Amst.) 10, 466&#226;&#8364;&#8220;475. <br />     240.                                                                                        Ostlund, G., Schmitt, T., Forslund, K., Kostler, T., Messina, D.N., Roopra, S., Frings, O., <br /> Holman, G., Davis, P.J., Foster, J.M., Carlow, C.K., Kumar, S., 2009. Computational                Sonnhammer, E.L., 2010. InParanoid 7: new algorithms tools eukaryotic <br />     prediction essential genes unculturable endosymbiotic bacterium,                        orthology analysis. Nucleic Acids Res. 38, D196&#226;&#8364;&#8220;203. <br />     Wolbachia Brugia malayi. BMC Microbiol. 9, 243.                                          Pairoba, C.F., Colombo, S.L., Andreo, C.S., 1996. Flavonoids inhibitors NADP- <br /> Hopkins, L., Groom, C.R., 2002. druggable genome. Nat. Rev. Drug Discov. 1,                    malic enzyme PEP carboxylase C4 plants. Biosci. Biotechnol. Biochem. <br />     727&#226;&#8364;&#8220;730.                                                                                         60, 779&#226;&#8364;&#8220;783. <br /> Hopkins, L., Bickerton, G.R., Carruthers, M., Boyer, S.K., Rubin, H., Overington, J.P.,     Payne, D.J., Holmes, D.J., Rosenberg, M., 2001. Delivering novel targets     2011. Rapid analysis pharmacology infectious diseases. Curr.  Med.                    antibiotics genomics. Curr. Opin. Investig. Drugs 2, 1028&#226;&#8364;&#8220;1034. <br />     Chem. 11, 1292&#226;&#8364;&#8220;1300.                                                                        Plata, G., Hsiao, T.L., Olszewski, K.L., Llinas, M., Vitkup, D., 2010. Reconstruction Huthmacher, C., Hoppe, , Bulik, S., Holzhutter, H.G., 2010. Antimalarial drug                      &#239;&#172;&#8218;ux-balance analysis Plasmodium falciparum metabolic network. Mol. <br />     targets Plasmodium falciparum predicted stage-speci&#239;&#172;?c metabolic network                    Syst. Biol. 6, 408. <br />     analysis. BMC Syst. Biol. 4, 120.                                                           Rice, P., Longden, , Bleasby, , 2000. EMBOSS: European molecular biology <br /> Janse, C.J., Kroeze, H., van Wigcheren, , Mededovic, S., Fonager, J., Franke-Fayard,               open software suite. Trends Genet. 16, 276&#226;&#8364;&#8220;277. <br />     B., Waters, P., Khan, S.M., 2011. genotype phenotype database                    Riguet, E., Desire, J., Boden, O., Ludwig, V., Gobel, M., Bailly, C., Decout, J.L., 2005. <br />     genetically modi&#239;&#172;?ed malaria-parasites. Trends Parasitol. 27, 31&#226;&#8364;&#8220;39.                              Neamine dimers targeting HIV-1 TAR RNA. Bioorg. Med. Chem. Lett. 15, <br /> Jenkins, C.L., 1989. Effects phosphoenolpyruvate carboxylase inhibitor 3,3-                   4651&#226;&#8364;&#8220;4655. <br />     dichloro-2-(dihydroxyphosphinoylmethyl)propenoate photosynthesis: C(4)                   Rodriguez-Acosta, , Dominguez, N., Aguilar, , Giron, M.E., 1999. Detection     selectivity studies C(4) photosynthesis. Plant Physiol. 89, 1231&#226;&#8364;&#8220;1237.                    glutamate dehydrogenase enzyme activity Plasmodium falciparum infection. <br /> Joubert, F., Harrison, C.M., Koegelenberg, R.J., Odendaal, C.J., Beer, T , 2009.                Indian J. Med. Res. 109, 152&#226;&#8364;&#8220;156. <br />     Discovery: interactive resource rational selection comparison             Rohrich, R.C., Englert, N., Troschke, K., Reichenberg, , Hintz, M., Seeber, F., Balconi, <br />     putative drug target proteins malaria. Malar. J. 8, 178.                                      E., Aliverti, , Zanetti, G., Kohler, U., Pfeiffer, M., Beck, E., Jomaa, H., Wiesner, J., <br /> Joyce, R., Palsson, B.O., 2008. Predicting gene essentiality using genome-scale                 2005. Reconstitution apicoplast-localised electron transfer pathway <br />     silico models. Methods Mol. Biol. 416, 433&#226;&#8364;&#8220;457.                                                  involved isoprenoid biosynthesis Plasmodium falciparum. FEBS Lett. <br /> Kato, N., Sakata, T., Breton, G., Le Roch, K.G., Nagle, , Andersen, C., Bursulaya, B.,             579, 6433&#226;&#8364;&#8220;6438. <br />     Henson, K., Johnson, J., Kumar, K , Marr, F., Mason, D., McNamara, C., Plouffe,           Rottmann, M., McNamara, C., Yeung, B.K., Lee, M.C., Zou, B., Russell, B., Seitz, P., <br />     D., Ramachandran, V., Spooner, M., Tuntland, T., Zhou, Y., Peters, E.C., Chatterjee,             Plouffe, D.M., Dharia, N.V., Tan, J., Cohen, S.B., Spencer, K.R., Gonzalez-Paez, G.E., <br />     , Schultz, P.G., Ward, G.E., Gray, N., Harper, J., Winzeler, E , 2008. Gene                  Lakshminarayana, S.B., Goh, , Suwanarusk, R., Jegla, T., Schmitt, E.K., Beck, H.P., <br />     expression signatures small-molecule compounds link protein kinase                      Brun, R., Nosten, F., Renia, L., Dartois, V., Keller, T.H., Fidock, D , Winzeler, E , <br />     Plasmodium falciparum motility. Nat. Chem. Biol. 4, 347&#226;&#8364;&#8220;356.                                     Diagana, T.T., 2010. Spiroindolones, potent compound class treatment <br /> Kobayashi, T., Takematsu, H., Yamaji, T., Hiramoto, S., Kozutsumi, Y., 2005.                         malaria. Science 329, 1175&#226;&#8364;&#8220;1180. <br />     Disturbance <span id='am-9' about='obo:IMR_0001359' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-10' property="rdfs:label" content="sphingolipid" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001359" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:26739" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>sphingolipid</span> biosynthesis abrogates signaling Mss4,                   Sa, J.M., Chong, J.L., Wellems, T.E., 2011. Malaria drug resistance: new observations <br />     phosphatidylinositol-4-phosphate 5-kinase, yeast. J. Biol. Chem. 280, 18087&#226;&#8364;&#8220;                  developments. Essays Biochem. 51, 137&#226;&#8364;&#8220;160. <br />     18094.                                                                                      Salvati, L., Mattu, M., Tiberi, F., Polticelli, F., Ascenzi, P., 2001. Inhibition Le Roch, K.G., Zhou, Y., Blair, P.L., Grainger, M., Moch, J.K., Haynes, J.D., La Vega, P.,        Saccharomyces cerevisiae phosphomannose isomerase donor S- <br />     Holder, , Batalov, S., Carucci, D.J., Winzeler, E , 2003. Discovery gene                nitroso-acetyl-penicillamine. J. Enzym. Inhib. 16, 287&#226;&#8364;&#8220;292. <br />     function expression pro&#239;&#172;?ling malaria parasite life cycle. Science 301,            Santiago, T.C., Zufferey, R., Mehra, R.S., Coleman, R , Mamoun, C.B., 2004.     1503&#226;&#8364;&#8220;1508.                                                                                       Plasmodium falciparum PfGatp endoplasmic reticulum membrane protein <br /> Leber, W., Skippen, , Fivelman, Q.L., Bowyer, P.W., Cockcroft, S., Baker, D , 2009.              important initial step malarial <span id='am-5' about='obo:IMR_0001382' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-6' property="oboInOwl:id" content="IMR:0001382" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="glycerolipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycerolipid</span> synthesis. J. Biol. Chem. <br />     unique phosphatidylinositol 4-phosphate 5-kinase activated <span id='am-15' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>-                          279, 9222&#226;&#8364;&#8220;9232. <br />     ribosylation factor Plasmodium falciparum. Int. J. Parasitol. 39, 645&#226;&#8364;&#8220;653.               Satchell, J.F., Malby, R.L., Luo, C.S., Adisa, , Alpyurek, E., Klonis, N., Smith, B.J., <br /> Levillain, O., Diaz, J.J., Reymond, , Soulet, D., 2000. Ornithine metabolism             Tilley, L., Colman, P.M., 2005. Structure glyceraldehyde-3-phosphate <br />     female mouse nephron: localization ornithine decarboxylase ornithine                      dehydrogenase Plasmodium falciparum. Acta Crystallogr. D: Biol. <br />     aminotransferase. P&#239;&#172;&#8218;ugers Arch. 440, 761&#226;&#8364;&#8220;769.                                                    Crystallogr. 61, 1213&#226;&#8364;&#8220;1221. <br /> Li, M., Allen, , Smith, T.J., 2007. High throughput screening reveals new             Scheer, M., Grote, , Chang, , Schomburg, , Munaretto, C., Rother, M., Sohngen, C., <br />     classes glutamate dehydrogenase inhibitors. Biochemistry 46, 15089&#226;&#8364;&#8220;15102.                     Stelzer, M., Thiele, J., Schomburg, D., 2011. BRENDA, enzyme information <br /> Li, M., Smith, C.J., Walker, M.T., Smith, T.J., 2009. Novel inhibitors complexed                2011. Nucleic Acids Res. 39, D670&#226;&#8364;&#8220;676. <br />     glutamate dehydrogenase: allosteric regulation control protein dynamics.              Schindelman, G., Fernandes, J.S., Bastiani, C , Yook, K., Sternberg, P.W., 2011. <br />     J. Biol. Chem. 284, 22988&#226;&#8364;&#8220;23000.                                                                 Worm Phenotype Ontology: integrating phenotype data Lim, D.C., Cooke, B.M., Doerig, C., Saeij, J.P., 2012. Toxoplasma Plasmodium                     C. elegans community. BMC Bioinform. 12, 32. <br />     protein kinases: roles invasion host cell remodelling. Int. J. Parasitol. 42,        Sriwilaijareon, N., Petmitr, S., Mutirangura, , Ponglikitmongkol, M., Wilairat, P., <br />     21&#226;&#8364;&#8220;32.                                                                                           2002. Stage speci&#239;&#172;?city Plasmodium falciparum telomerase inhibition <br /> Llinas, M., DeRisi, J.L., 2004. Pernicious plans revealed: Plasmodium falciparum                     berberine. Parasitol. Int. 51, 99&#226;&#8364;&#8220;103. <br />     genome wide expression analysis. Curr. Opin. Microbiol. 7, 382&#226;&#8364;&#8220;387.                         Storm, J., Perner, J., Aparicio, , Patzewitz, E.M., Olszewski, K., Llinas, M., Engel, P.C., <br /> Madhusudana, Rao , Swamy, P.M., Das, V.S.R., 1980. Herbricide (sic) inhibited                      Muller, S., 2011. Plasmodium falciparum glutamate dehydrogenase     phosphoenolpyruvate carboxylase leaves nonsucculent scrub species.                     dispensable drug target erythrocytic development. Malar. J. <br />      P&#239;&#172;&#8218;anzenphysiol 99, 69&#226;&#8364;&#8220;74.                                                                     10, 193. <br /> Magarinos, M.P., Carmona, S.J., Crowther, G.J., Ralph, S , Roos, D.S., Shanmugam, D.,         Stranska, J., Kopecny, D., Tylichova, M., Snegaroff, J., Sebela, M., 2008. Ornithine <br />     Van Voorhis, W.C., Aguero, F., 2012. TDR Targets: chemogenomics resource                   delta-aminotransferase: enzyme implicated salt tolerance higher <br />     neglected diseases. Nucleic Acids Res. 40, D1118&#226;&#8364;&#8220;1127.                                           plants. Plant Signal. Behav. 3, 929&#226;&#8364;&#8220;935. <br /> Magrane, M., Consortium, U., 2011. UniProt Knowledgebase: Hub Integrated                   Strecker, H.J., 1965. Puri&#239;&#172;?cation properties rat liver ornithine delta- <br />     Protein Data. Database, Oxford, bar009.                                                          transaminase. J. Biol. Chem. 240, 1225&#226;&#8364;&#8220;1230. <br /> Martinsen, E.S., Perkins, S.L., Schall, J.J., 2008. genome phylogeny malaria         Takatsuka, Y., Onoda, M., Sugiyama, T., Muramoto, K., Tomita, T., Kamio, Y., 1999. <br />     parasites (Plasmodium closely related genera): evolution life-history                     Novel characteristics Selenomonas ruminantium lysine decarboxylase <br />     traits host switches. Mol. Phylogenet. Evol. 47, 261&#226;&#8364;&#8220;273.                                    capable decarboxylating <span id='am-34' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span>L-lysine</span> <span id='am-52' about='obo:IMR_0200342' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-53' property="oboInOwl:id" content="IMR:0200342" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="L-Ornithine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00077" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-ornithine</span>. Biosci. Biotechnol. <br /> Mastrolorenzo, , Scozzafava, , Supuran, C.T., 2000. Antifungal activity Ag                  Biochem. 63, 1063&#226;&#8364;&#8220;1069. <br />     Zn(II) complexes aminobenzolamide (5-sulfanilylamido-1,3,4-                          Teixeira, C., Gomes, J.R., Gomes, P., 2011. Falcipains, Plasmodium falciparum <br />     thiadiazole-2-sulfonamide) derivatives. J. Enzym. Inhib. 15, 517&#226;&#8364;&#8220;531.                            cysteine proteases key drug targets malari <br /> </body></html>