<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 23 October 2012                                                          Nucleic Acids Research, 2013, Vol. 41,  1 e33 <br />                                                                                                                      doi:10.1093/nar/gks972 <br />  <br />  <br /> Rationally designed bidirectional promoter <br /> improves evolutionary stability synthetic genetic circuits <br /> Song Yang1,*, Sean C. Sleight2 Herbert M. Sauro2 <br /> 1 <br />  Genomics Division, Lawrence Berkeley National Laboratory, 1 Cyclotron Road, Mail Stop 84-363, Berkeley, <br /> CA 94720, USA 2Department Bioengineering, University Washington, Seattle, WA 98195, USA <br />  <br /> Received July 16, 2012; Revised September 24, 2012; Accepted September 25, 2012 <br />  <br />  <br />  <br /> ABSTRACT                                                                          population faster growing non-functional <br />                                                                                   mutants inactivated expression function problem synthetic genes genetically                                heterologous genes. experimentally gineered organisms foreign DNAs                                theoretically con&#239;&#172;?rmed genes selective <br /> eventually lose functions evolutionary                                 pressure eventually lose functions (11&#226;&#8364;&#8220;16), time absence selective pressures.                                      lack robustness remains problem synthetic <br /> general limitation restrain long-term study                               biology approaches. industrial application synthetic genetic                                      synthetic genetic circuit bacteria host typically <br /> circuits. Previous studies shown                                composed promoters, ribosomal RNA- crucial regulatory functions, prokaryotic                                binding sites, terminators target genes promoters synthetic genetic circuits espe-                                 (14). various parts genetic circuits, pro- <br /> cially vulnerable mutations. study, ra-                             moters vulnerable loss function <br />                                                                                   mutations (e.g. insertions, deletions entire <br /> tionally designed robust bidirectional promoters <br />                                                                                   promoter, deletions repeated sequences pro- <br /> (BDPs), self-protected com-                                 moter point mutations) (17).  protecting plementarity overlapping forward                                     promoter mutation crucial enhance evolu- <br /> backward promoter sequences DNA duplex.                                        tionary stability synthetic genetic circuits. work, transcription target non-essential                                  report novel design robust genetic circuit <br /> gene (e.g. green fluorescent protein) coupled                                 controlled bidirectional promoter (BDP). transcription essential gene (e.g. antibi-                              Typically strand DNA encodes functional units, <br /> otic resistance gene) BDP, evolu-                                 coding regions regulatory regions; comple- <br /> tionary half-time gene increases                               mentary strand does necessarily contain informa- <br /> 4&#226;&#8364;&#8220;10 times, depending strain experimen-                                tion. sequence promoter region designed <br /> tal conditions used. design using BDPs                                 complementary strand carries function <br />                                                                                   essential survival host, promoter <br /> increase mutational stability genetic circuits <br />                                                                                   coupled protected comple- potentially applied synthetic biology appli-                            mentary sequence. complementary strand cations general.                                                               promoter designed encode promoter, engineered <br />                                                                                   bidirectional promoter regulate gene expression                                                                                   directions: target gene forward direction <br /> INTRODUCTION                                                                      essential gene, e.g. antibiotic gene, reverse recent advances synthetic biology, scientists                         direction. case, mutations occur BDP able manipulate biological organisms perform                            region  principle, affect  turn  expression <br /> speci&#239;&#172;?c functions exist nature                                  target gene essential gene concurrently. <br /> introducing engineered functional genetic circuits                            viability non-functional mutant popu- host genome (1&#226;&#8364;&#8220;9). general, engineered genetic                             lation decrease selected  words, <br /> circuits essential survival host cell,                      target gene protected essential gene expression demands extra metabolic load                              shared BDP. (10).  positive se-                             overlapping functional units encoded com- <br /> lective pressure, synthetic organisms outcompeted                         plementary strands exist nature (18). genomes <br />  <br />  <br />  correspondence addressed. Tel: +1 510 486 7442; Fax: +1 510 486 5614; Email: songyang@lbl.gov <br />  <br /> &#195;&#376; Author(s) 2012. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses nc/3.0/), permits non-commercial reuse, distribution, reproduction medium, provided original work properly cited. commercial use, contact <br /> journals.permissions@oup.com. <br />  e33 Nucleic Acids Research, 2013, Vol. 41,  1                                                                PAGE 2 7 <br />  <br />  organisms, adjacent promoters opposite              freezer stocks streaked LB+100 mg/ml <br /> strands close overlap        ampicillin plates grown overnight 37  C. way, trans-regulatory element             colonies chosen transformant  total regulate expression multiple genes operons        colonies) inoculated 1.5 ml LB+100 mg/ml opposite direction, usually       ampicillin media Eppendorf deep plates sealed <br /> pathway functionally related (18). regulation       Thermo Scienti&#239;&#172;?c <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="gas" datatype="xsd:string"></span>gas</span>-permeable membrane function natural BDPs achieved proximity       allow maximum oxygen diffusion. appropriate, promoters different strands     medium supplemented 50 mg/ml kanamycin. <br /> controlled regulatory element, e.g. repressor.    LacI-regulated circuits inducible expression example shows point mutation natural           MG1655 Z1 cells supplemented 1   10 3 M <br /> BDP region cause reduced expression genes           IPTG. cultures grown 24 h 37  C, sides (19).                                            shaking 250 r.p.m., cell density (OD600) &#239;&#172;&#8218;uor- <br />    Previous studies tried use regulation        escence (excitation wavelength: 485/15, emission wave- <br /> function naturally occurring BDPs synthetic              length: 516/20) measured Biotek Synergy HT <br /> circuits, reviewed (20), attempt aimed        plate reader.  evolved populations increase evolutionary stability genetic circuit       propagated serial dilution scheme using 1:1000 <br /> using BDP. design, engineered gene detrimen-     (log2 1000 = 9.97) dilution, corresponds tal host, function engineered BDP     growth E. coli  10 generations day. pro- prevent mutation.  proximity         cedure repeats cultures lost &#239;&#172;&#8218;uor- overlap forward backward regulatory            escence &#239;&#172;&#8218;uorescence decreased arrived regions required robust self-protective BDPs.       plateau. <br />  <br />                                                                Sequencing results mutant plasmids <br /> MATERIALS METHODS <br />                                                                determine loss function mutations replicate- <br /> Circuit engineering use strains                         evolved MG1655/MDS42 populations, plasmids circuits obtained Registry         extracted individual clones low green &#239;&#172;&#8218;uores- <br /> Standard Biological Parts engineered using              cent protein (GFP) &#239;&#172;&#8218;uorescence. Plasmids polymer- <br /> Clontech Fusion PCR Cloning Kit described             ase chain reaction ampli&#239;&#172;?ed using vector-speci&#239;&#172;?c previously (21). circuits encoded        primers, VF2 VR, gene-speci&#239;&#172;?c primers, <br /> pSB1A2 plasmid, high-copy number plasmid (100&#226;&#8364;&#8220;300            GFP200R KAN200R, 25-bp primers plasmids/cell), ampicillin resistance gene. Plasmids   reverse complements 200&#226;&#8364;&#8220;225 regions GFP transformed strains <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> transformation      kanamycin-resistant gene, respectively.  electroporation. Escherichia coli MG1655, multiple          struct, plasmids duplicates sequenced. <br /> deletion strain (MDS42) &#226;&#8364;&#732;Clean Genome&#226;&#8364;&#8482; Scarab <br /> Genomics MG1655 Z1 (15) used inducible ex-          RESULTS <br /> pression LacI-regulated promoters, strain <br /> overexpress LacI chromosome.                          BPD design characterization <br />    BDPs derived LacI-regulated                Prokaryotic promoters characterized consensus <br /> promoter (R0011) gradual point mutation. Obtained           sequence, conserved sequence motif promoter Registry Standard Biological Parts (14,22),        region. hexamers,           35 region (T82T84G78 <br /> R0011 arti&#239;&#172;?cial promoter consisting 35            A65C54A45) 10 region (T80A95T45A60A50T96), 10 regions phage lambda cI-binding           distance (16&#226;&#8364;&#8220;18 bp), sites replaced identical 17-bp lacO1 operator         conserved characters (25)  subscripts denote sites, 35 10 region            percentage occurrence frequently bases <br /> upstream 35 region. hybrid design makes            E. coli). design rationale identify promoter <br /> R0011 strong promoter, tightly repressed LacI,        forward 35 10 regions happen inducible IPTG strains express LacI. known    reverse complementary sequences backward 10 genetic circuits regulated R0011 quickly lose          35 regions. ideal case, forward function (&lt;50 generations) evolutionary half-life      backward consensus sequences exactly overlap  20 generations, owing chromosome misalignment            mutation (point mutation, insertion homologous sequences replication (termed        deletion) promoter region DNA strand <br /> &#226;&#8364;&#732;replication slippage&#226;&#8364;&#8482;) (23), owing spontaneous direct   inevitably cause mutation reverse promoter. <br /> repeat deletions (24). fast de&#239;&#172;?ned     result, theory, expression engineered <br /> mutation promoter, BDPs study         gene protected expression antibiotic resist- <br /> constructed R0011.                                        ance gene crucial genes encoded                                                                strand shared BDP. <br /> Evolutionary stability measurements <br />                                                                   base consensus sequence statis- individual transformants circuit tested    tically conserved, exact consensus sequence grown overnight 37  C, shaking 250 r.p.m.        rarely nature. Individual promoters usually <br /> LB+100 mg/ml ampicillin, stored glycerol stock.         differ consensus sequence  PAGE 3 7                                                                           Nucleic Acids Research, 2013, Vol. 41,  1 e33 <br />  <br />  <br />  <br />  <br /> Figure 1. Design bidirectional promoter gradually modifying consensus sequence.   sequences shown. Sequence 1 35 10 regions consensus sequence; Sequence 2 reverse complement Sequence 1. Sequence 3 promoter sequence identical <br /> Sequence 1 opposite strand reading backwards. Sequence 4 partial overlapping promoter shifted bases. Bases green represent identical bases backward (Sequences 3 4) reverse complement sequences (Sequence 2), red color indicates positions introduce point mutations create BDP. (B) Illustration genetic constructs, regular promoter (R0011) bidirectional promoter (BDP-01) biological parts strands. Curved arrows represent promoters, ovals represent ribosome <br /> binding sites, hexagons represent terminators horizontal arrows represent coding sequences. <br />  <br />  <br /> positions  5-bp mismatch (25)].                  sequences carry minimum total number mutations <br /> general, similar promoter sequence                     consensus sequence  mutations) equally <br /> consensus sequence, higher transcription ef&#239;&#172;?ciency                 distribute mutations forward backward promoter. rationale &#239;&#172;?nd 35 10                     strands  . sequences forward sequence pairs sequence complementary                            backward promoters identical. sequence close consensus sequence                         totally 14 promoter sequences meet criteria <br /> possible, ensuring promoter ef&#239;&#172;?ciency directions.                  (Supplementary Data). tested BDP total 12 bp position                     function constructing red &#239;&#172;&#8218;uorescent protein (RFP)  <br /> chosen bases, number possible com-                    stream GFP downstream promoter. <br /> binations enormous  Supplementary Information).                     Unfortunately, &#239;&#172;&#8218;uorescence measurement showed design strategy start consensus                         constructs able transcribe bidirectionally <br /> sequence strand gradually make point muta-                      produce red green &#239;&#172;&#8218;uorescence <br /> tions promoter achieve BDP functionality.                        (Supplementary Data). <br />    Figure 1A shows design BDP, Sequence                      ideal design complete overlapping 1 represents forward 35 10 regions                      forward backward promoters, practical <br /> sensus sequence Sequence 2 reverse complement                   partial overlapping design tested. stringent Sequence 1. promoter sequence                     robust point mutations complete overlap- <br /> Sequence 1 opposite strand, consensus sequence                   ping design, easier implement, partial overlapp- <br /> reversed create Sequence 3. words, Sequences 1                 ing design theory maintain promoter function 3 identical sequences, read opposite direc-                 minimizing mutation recombination. Sequence 4 illus- <br /> tions (TTGACA_17bp_TATAAT).  obtain                             trates partial overlapping design shifting entire BDP, Sequences 2 3 match                           backward sequence (Sequence 3) positions Comparison Sequences 2 3 indicates                       left. design, positions need mutated 12 positions consensus sequence, 6                  (instead . designs forward-shifted identical (labeled green), point mutations                  reverse-shifted promoters described Supplementary required make match completely (labeled                     Table S1 Supplementary Information. sequence <br /> red). variable sites allow 46 possible combinations,               shifted 2 bp, increase chance enormous search space.                                            inactivating mutation opposite strand. partial <br />    prove principle work, impossible enumer-              overlapping design, BDP identi&#239;&#172;?ed experimen- <br /> ate possible sequence combinations, experiment                  tally con&#239;&#172;?rmed gradual point mutation proced- <br /> began candidates, designed                          ure. sequence 10 35 regions BDP-01  e33 Nucleic Acids Research, 2013, Vol. 41,  1                                                                                  PAGE 4 7 <br />  <br />  <br />  <br />  <br /> Figure 2. Performance stability genetic circuits E. coli MG1655 strain   clean genome MDS strain (B). MG1655 strain, circuits studied: regular promoter R0011 control  , circuit BDP-01 kanamycin resistance gene encoded backward <br /> direction BDP-01 culture kanamycin (b) kanamycin (c). Construct b negative control c, absence kanamycin media puts evolutionary pressure KanR, bidirectional promoter free evolutionary changes acts regular promoter. MDS strain, addition circuits tested MG1655 strain, GFP&#226;&#8364;&#8220;kanR fusion protein (d,e, R0011 K+G) polycistronic design (f,g, R0011 K-G) compared. <br />  <br />  <br />  follows: atTTGACA_15bp_tgTCAAAT (lowercase                             stability dynamics circuits BDP (BDP-01) <br /> denotes 2 bp required backward promoter                     versus unidirectional promoter (R0011) initially strand).                                                        measured compared. circuits encoded                                                                           high-copy number plasmid conferring ampicillin <br />                                                                           resistance use GFP target gene, Evolutionary stability dynamics loss function                      circuit BDP-01 promoter controls expression mutations genetic circuits E. coli MG1655                           kanamycin resistance gene reverse direction, examine designed BDP does improve                      R0011 does  R0011 contains identical <br /> mutational robustness genetic circuits, evolutionary               17-bp lac operator sites. Derived R0011, BDP01 <br />  PAGE 5 7                                                                             Nucleic Acids Research, 2013, Vol. 41,  1 e33 <br />  <br />  R0011 differ 35 10 region,                          original R0011 circuit BDP circuit propagated <br /> share double operator sites.                                            kanamycin, replication slippage took place    Figure 2A shows evolutionary half-life                   samples, resulting deletion lacO sequence original R0011 circuit    20 generations,                       35 region. contrast, deletions GFP &#239;&#172;&#8218;uorescence/OD level drops half initial                       BDP circuit propagated kanamycin. mutations value. promoter replaced BDP-01,                        BDP kanamycin-evolved populations caused kanamycin growth media (b), circuit                     insertion sequence   elements (26,27) insert evolutionary half-life  45 generations.                        downstream BDP-01 region, disrupting expres- <br /> BDP-01 does protective function                      sion GFP leaving KanR expression intact. <br /> antibiotic, possible reasons                 Interestingly, 13 identi&#239;&#172;?ed insertions, 6 improved life span. reason lowered ex-                     IS2 elements insert right 2 bp pression gene, average predicted                      forward 10 region, right upstream improve evolutionary half-life (15). second reason                   backward 35 region. IS2 elements insert  2-bp shorter repeated lacO regions BDP-01 relative                   rich region (AA-IS2 , hotspot IS2 R0011, presumably lower replication slippage                   insertion.  insertions took place <br /> rate. kanamycin added media (c),                        BDP region, disrupting forward promoter half-life BDP-01 circuit increased  80 gen-                  reverse promoter, expression kanamycin <br /> erations. comparison BDP-01 circuit                     resistance gene affected. kanamycin shows rationally designed <br /> BDP-01 promoter does provide protective function                          Evolutionary stability dynamics loss function increases evolutionary stability genetic circuit.               mutations genetic circuits E. coli MDS42 (&#226;&#8364;&#732;Clean <br />    Sequence analysis loss function plasmids                        Genome&#226;&#8364;&#8482; strain) <br /> evolved populations demonstrates reason pro-                   insertions major cause mutations tective function (Table 1 Figure 3).                        BDP-01 circuits,  obtained cell lines devoid  <br />  <br />  <br />  <br /> Figure 3. Mutation map loss function mutations evolved MG1655 populations shown Table 1.   (B), deletion Lac operator 35 region homologous recombination (replication slippage) original R0011 circuit BDP01 circuit kanamycin growth media. (C), replication slippage  Instead, 13 recognizable insertions, 4 cases located forward GFP coding region ribosomal biding site, 9 cases promoter region 6 cases located exactly boundary start position reverse promoter 35 region. BDP colored red (complete overlapping forward reverse strain) yellow <br /> (partial overlapping). <br />  e33 Nucleic Acids Research, 2013, Vol. 41,  1                                                                                          PAGE 6 7 <br />  <br />  <br /> Table 1. sequencing results loss function mutations evolved MG1655 populations <br />  <br />      Promoter      Reverse gene     Forward gene       Strain       Media           1       2      3       4        5      6        7       8      9 <br />     R0011         N              GFP                MG1655       <span id='am-11' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Amp</span> <br /> b    BDP-1         KanR             GFP                MG1655       Amp <br /> c    BDP-1         KanR             GFP                MG1655       Amp+Kan         IS2     IS2    IS2     N      IS2    N      IS5     IS2    IS2 <br />  replicate-evolved population, cells highlighted red indicate mutations deletion replication slippage; orange cells <br /> indicate mutations insertion elements. Cells highlight indicate sequencing results vague indiscernible. <br /> N = applicable. <br />  <br />  elements test strain increases evolution-                  circuits lower circuits. <br /> ary stability BDP-01 circuits. &#226;&#8364;&#732;Clean Genome&#226;&#8364;&#8482;                      generally lower expression differ- <br /> E. coli MDS42 strain (multiple deletion strain) (28)                       ences strain, improper folding GFP fusion <br /> Scarab Genomics (http://www.scarabgenomics.com/)                         protein circuit differences translation rate <br /> synthetic strain 15% genome deleted                      GFP KanR polycistronic circuit. <br /> E. coli K12, including non-essential genes, recombinogenic                    Despite low expression level, circuits quickly mobile DNA (including elements) cryptic                             lose function, kanamycin, virulent genes. Figure 2B shows evolutionary                              evolutionary half-life  50 generations. result <br /> dynamics various circuits propagated MDS42.                             shows target gene (GFP) protected Overall, expression level GFP lower                         directly connecting essential gene fusion <br /> strain relative MG1655. R0011 circuit evo-                      polycistronic link, accordance previous work. <br /> lutionary half-life  18 generations, propagated MG1655. BDP-01 circuit grown kanamycin media evolutionary                            DISCUSSION <br /> half-life  105 generations, double                       study, designed, constructed MG1655. BDP-01 circuit grown kanamycin                            characterized BDP expresses GFP forward <br /> media evolutionary half-life  185 generations,                      direction kanamycin resistance gene reverse double compared MG1655.                               direction. utility BDP-01 demonstrated kanamycin-evolved BDP-01 circuit 10 times                           increased evolutionary stability circuit life span original R0011 circuit MDS42                      propagated media kanamycin. Furthermore, strain.                                                                       loss function mutations kanamycin-evolved <br />    GFP &#239;&#172;&#8218;uorescence level BDP-01 circuits,                              populations occur GFP-expressing component <br /> propagated kanamycin, &#239;&#172;?rst dropped                           circuit, indicating BDP-01 promoter ex- half maximum magnitude, rose                        pressing kanamycin resistance essential survival. <br /> highest level dropped near zero.                           design BDP sequence genetic circuit controlled <br /> dynamics observed independently evolved                       BDP independent target gene  populations kanamycin. phenom-                           design generally applied synthetic genetic <br /> enon probably occurs physiological                         circuit. <br /> adaptation MDS42 strain required maintain                              clear BDP-01 circuit improved evo- circuit high-copy number plasmid.                           lutionary stability propagated kanamycin versus <br /> strain 15% smaller normal genome,                           kanamycin, compared control time maximize growth rate, given                         KanR-expressing circuits difference <br /> extra metabolic load high-copy number                              stability. previous study (15), plasmid. Sequence analysis loss function                            GFP&#226;&#8364;&#8220;KanR fusion protein polycistronic <br /> plasmids showed insertion observed                          circuits lost function rate, evolved populations, indicating use MDS                           propagated kanamycin  consistent strain prevents prevalent insertion MG1655                          study.  difference KanR- <br /> strains.  various types insertion                        expressing control circuits BDP-01 circuit recombination-based mutations happened.                               different promoters use, R0011 despite rare MG1655, point mutations                       study, R0010 promoter  mutationally robust MDS42 strains, possibly indicating                      absence repeated operator sequences) truncated genome, replication repairing machin-                    previous study. major difference design ery clean genome ef&#239;&#172;?cient                     circuit. metabolic load BDP circuit <br /> complete genome (Supplementary Information).                                  control circuits similar, GFP&#226;&#8364;&#8220;KanR fusion <br />    addition, control circuits tested                 protein polycistronic circuits produce long RNA <br /> MDS42 strain. circuit expresses GFP&#226;&#8364;&#8220;KanR fusion                         products  long proteins fusion version), <br /> protein <span id='am-18' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-19' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span>&#226;&#8364;&#8220;serine linker circuit                      affect transcription translation rate expresses GFP polycistronically linked KanR.                        biological function &#239;&#172;?nal protein product. <br /> circuits originally engineered previous study (15).                 contrast, BDP design keeps original RNA shown Figure 2B, initial expression levels                       protein product target gene (GFP)  PAGE 7 7                                                                         Nucleic Acids Research, 2013, Vol. 41,  1 e33 <br />  <br />  <br /> protection gene (kanamycin resistance), normal                   7. Weber,W. Fussenegger,M. (2009) impact synthetic <br /> biological function maintained.                                           biology drug discovery. Drug Discov. Today, 14, 956&#226;&#8364;&#8220;963. <br />                                                                           8. Gibson,D.G., Glass,J , Lartigue,C., Noskov,V.N., Chuang,R.Y., <br />    interesting result GFP expression                     Algire,M , Benders,G , Montague,M.G., Ma,L., <br /> levels MDS42  40% lower MG1655                             Moodie,M.M. et al. (2010) Creation bacterial cell controlled <br /> average circuits. Previous work demonstrated                     chemically synthesized genome. Science, 329, 52&#226;&#8364;&#8220;56. expression level evolutionary half-life              9. Wang,H.H., Isaacs,F.J., Carr,P , Sun Z., Xu,G., Forest,C.R. <br />                                                                              Church,G.M. (2009) Programming cells multiplex genome <br /> different genetic circuits negatively correlated;                     engineering accelerated evolution. Nature, 460, 894&#226;&#8364;&#8220;898. <br /> average, higher expression level, shorter                10. Glick,B.R. (1995) Metabolic load heterologous gene <br /> half-life (15).  observation based                        expression. Biotechnol. Adv., 13, 247&#226;&#8364;&#8220;261. <br /> comparing different circuits strain.                     11. Cooper,V.S. Lenski,R.E. (2000) population genetics increase 80 185 generations MG1655                        ecological specialization evolving Escherichia coli populations. <br />                                                                              Nature, 407, 736&#226;&#8364;&#8220;739. <br /> MDS42 strains, respectively, kanamycin-evolved                   12. L., Cox,R.S. 3rd, Weiss,R. Arnold,F.H. (2004) <br /> BDP-01 circuit, likely decrease                     Programmed population control cell-cell communication expression level, evolutionary half-life                       regulated killing. Nature, 428, 868&#226;&#8364;&#8220;871. <br /> original R0011 circuit did change ( 20 generations                   13. Elena,S.F. Lenski,R.E. (2003) Evolution experiments MG1655 MDS42 strains).                                           microorganisms: dynamics genetic bases adaptation. <br />                                                                              Nat. Rev. Genet., 4, 457&#226;&#8364;&#8220;469. <br />    learned clean genome evolution ex-                 14. Canton,B., Labno  Endy,D. (2008) Re&#239;&#172;?nement periment, evolutionary half-life designed syn-                    standardization synthetic biological parts devices. Nat. <br /> thetic circuit increases elements eliminated,                    Biotechnol., 26, 787&#226;&#8364;&#8220;793. eventually mutations, point mutations,                 15. Sleight,S.C., Bartley,B , Lieviant,J  Sauro,H.M. (2010) <br />                                                                              Designing engineering evolutionary robust genetic circuits. <br /> insertions/deletions recombination, happen,                     J. Biol. Eng., 4, 12. designed circuit outcompeted. Step step,               16. Arkin P. Fletcher,D  (2006) Fast, cheap somewhat <br /> try eliminate probable mutations, evolu-                     control. Genome Biol., 7, 114. <br /> tion pick feasible mutations eventually                17. Sleight,S.C., Orlic,C., Schneider,D. Lenski,R.E. (2008) Genetic  everlasting procedure synthetic                      basis evolutionary adaptation Escherichia coli stressful <br />                                                                              cycles freezing, thawing growth. Genetics, 180, 431&#226;&#8364;&#8220;443. <br /> biology elongate life span synthetic circuits.                 18. Beck,C.F. Warren,R  (1988) Divergent promoters,                                                                              common form gene organization. Microbiol. Rev., 52, 318&#226;&#8364;&#8220;326. <br />                                                                          19. Yan,G., Fu,Y. Faustman,D.L. (1997) Reduced expression SUPPLEMENTARY DATA                                                           Tap1 Lmp2 antigen-processing genes nonobese diabetic <br />                                                                              (NOD) mouse mutation shared bidirectional <br /> Supplementary Data available NAR Online:                              promoter. J. Immunol., 159, 3068&#226;&#8364;&#8220;3080. <br /> Supplementary Tables 1&#226;&#8364;&#8220;5.                                                20. Venter,M. (2007) Synthetic promoters: genetic control cis <br />                                                                              engineering. Trends Plant Sci., 12, 118&#226;&#8364;&#8220;124. <br />                                                                          21. Sleight,S.C., Bartley,B , Lieviant,J  Sauro,H.M. (2010) <br /> FUNDING                                                                      fusion biobrick assembly engineering. Nucleic Acids <br />                                                                              Res., 38, 2624&#226;&#8364;&#8220;2636. <br /> Funding open access charge: Waived Oxford                         22. Endy,D. (2005) Foundations engineering biology. Nature, 438, <br /> University Press.                                                            449&#226;&#8364;&#8220;453. <br />                                                                          23. Lovett,S.T. (2004) Encoded errors: mutations rearrangements <br /> Con&#239;&#172;&#8218;ict statement. declared.                                mediated misalignment repetitive DNA sequences. Mol. <br />                                                                              Microb <br /> </body></html>