<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The EMBO Journal vol. 14  1 3 pp.3064-3072, 1995 <br />  <br />  calnexin homologue cnx1+ Schizosaccharomyces pombe, essential gene complemented soluble ER <br /> domain <br /> Francesco Parlati1l3, Daniel Dignard3,                              GlcNAc2 Mang GlcI intermediate (Ou et al., 1993; <br /> John J.M.Bergeron2                                              Hammond et al., 1994; Le et al., 1994). Recently, David Y.Thomas&quot;&apos;2&apos;3&apos;4                                               demonstrated GlcNAc2 Mang GlcI <br />                                                                     oligosaccharide interacts directly calnexin (Ware <br /> &apos;Department Biology 2Department Anatomy Cell          et al., 1995). addition, tunicamycin (Ou et al., 1993) <br /> Biology, McGill University, Montreal, Quebec 3Genetics Group,   glucosidases II inhibitors, deoxynojirimycin <br /> Biotechnology Research Institute, National Research Council      castanospermine, (Hammond et al., 1994) inhibit <br /> Canada, 6100 Royalmount Avenue, Montreal, Quebec H4P 2R2, <br /> Canada                                                              association incompletely folded glycoproteins 4Corresponding author <br />                                                                     calnexin, supporting observation GlcNAc2 Mang <br />                                                                     GlcI intermediate important calnexin recognition. <br /> Secretory proteins folded action                    Calnexin associated proteins number molecular chaperones soon enter                folded action ER molecular chaperones endoplasmic reticulum (ER). mammalian cells,                 chaperonins. time association secretory protein ER membrane protein calnexin shown                  calnexin reflects time required folding (Ou molecular chaperone involved folding                 et al., 1993). Secretory proteins assembled <br /> secretory proteins assembly cell surface              correctly complexes fold correctly receptor complexes. used PCR strategy <br />                                                                     mutations incorporation <span id='am-21' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-26' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> analogues, identify Schizosaccharomyces pombe calnexin <br />                                                                     retained calnexin ER (reviewed Bergeron <br />                                                                     et al., 1994). calnexin function homologue, cnxl +. cnxl + encoded protein, Cnxl,                constituent ER quality control apparatus. shown calcium binding type integral                      common molecular chaperones, calnexin <br /> membrane glycoprotein. 5&apos; end, cnxl+ gene                genes identified wide variety eukaryotes consensus heat shock transcriptional control ele-               including mammals, nematodes plants (Wada et al., <br /> ments inducible heat shock                    1991; Sulston et al., 1992; Huang et al., 1993). share <br /> calcium ionophore A23187. Unlike sequence-related               sequence motifs ER lumenal protein calreticulin, <br /> Saccharomyces cerevisiae CNEI gene, S.pombe                     similarly number eukaryotes. <br /> cnxl + gene essential cell viability.              appear form gene family possible <br /> length Cnxl protein able complement cnxl +               similarity function (Smith Koch, 1989; Wada et al., <br /> gene disruption length mammalian calnexin              1991). proteins perform function  ER lumenal domain Cnxl,                     topic considerable  secreted cells, capable complementing                  recently cloned CNEJ gene Saccharo- cnxl::ura4 lethal phenotype. equivalent                     myces cerevisiae shares sequence similarity region mammalian calnexin shown                      mammalian calnexin calreticulin. CNEJ gene <br /> possess molecular chaperone activity. possible                codes integral membrane ER glycoprotein, Cnelp. lethal phenotype caused absence               Unlike mammalian calnexin, Cnelp does bind calcium chaperone activity S.pombe cnxl + gene                  vitro assay, does cytosolically <br /> disruption.                                                         directed domain carboxy terminus (Parlati et al., <br /> Key words: calnexin/ER/essential gene/molecular chaperone/           1995). demonstrated CNEJ does effect <br /> Schizosaccharomyces pombe                                           retention mutant proteins ER (Parlati <br />                                                                     et al., 1995) essential viability                                                                     S.cerevisiae.  properties correspond <br /> Introduction                                                        expected bona fide S.cerevisiae calnexin eukaryotes, endoplasmic reticulum (ER) site           homologue.  Schizosaccharomyces pombe folding secretory proteins assembly            cnxl + gene characterized properties <br /> multimeric cell surface receptors. processes              common mammalian calnexin likely <br /> mediated molecular chaperones (reviewed Bergeron              S.pombe calnexin homologue. <br /> et al., 1994). molecular chaperones identified appear present wide variety          Results eukaryotic species. molecular chaperone, ER membrane protein calnexin, interacts secretory              Cloning S.pombe member calnexini <br /> glycoproteins soon enter ER (Ou et al.,              calreticulin family, cnx1+ <br /> 1993). Newly synthesized glycoproteins specifically                 previously used sequence motifs shared <br /> associate calnexin monomeric, incom-            mammalian calnexin ER lumenal protein <br /> pletely folded oligosaccharide modification            calreticulin identify calnexin related gene S.cere- <br />  <br />   3 <br /> 3064 4 Oxford University Press <br />                                                                     Schizosaccharomyces pombe calnexin homologue cnx1+ <br />  <br /> visiae, CNEJ (Parlati et al., 1995). used similar         heat shock (Figure 3A B). heat shock treatment <br /> strategy cloning S.pombe member calnexin/     carried extended time, level transcrip- <br /> calreticulin family. Degenerate oligonucleotides coding        tion diminished basal level (Figure 3A B). regions amino acid similarity conserved     agents known cause stress mammalian calnexin calreticulin, YKGK,FWKP             tested effect cnxl&#194;&#177; transcription. Treatment repeat motif KPEDWDE (Figure IA) used              glycosylation inhibitor tunicamycin (Figure 3C <br /> prime PCR using S.pombe genomic DNA template            D) 2-deoxyglucose (data shown) (Figure iB). organization motifs          effect, treatment calcium ionophore A23187 <br /> calnexin calreticulin, amplified fragment 350 bp     gave marked increase cnxl + transcription (Figure 3C expected (Figure 1A) double-stranded (ds) DNA          D). <br /> products approximately size cloned plasmid pTZ19R. DNA sequence               cnx1+ gene product, Cnxl, calcium- <br /> clones derived amino acid sequence revealed            binding protein PCR product shared high amino acid sequence          Calnexin mammalian cells originally described similarity mammalian calnexin calreticulin.      major calcium-binding proteins Using fragment probe, entire gene cloned     ER membrane (Wada et al., 1991). calnexins sequenced (Figure IC, Materials methods).          calreticulins tested far bind calcium, cnxl+ genomic sequence identified consensus            S.cerevisiae Cnelp (Parlati et al., 1995). order heat shock elements 5&apos; end (Figure 1C). coding      determine Cnxl calcium-binding protein, used <br /> sequence does contain introns predicts type        GST fusion Cnxl expressed Escherichia coli. integral membrane protein 560 amino acids        fusion comprised amino acids 23-492 Cnxl,  similar overall arrangement mammalian calnexin.       excludes predicted signal sequence C-terminal predicted N-terminal cleavable signal sequence,        transmembrane cytoplasmic domains Cnxl. Tested <br /> N-glycosylation motif residue 418, membrane-          calcium overlay assay, GST::Cnxl fusion protein <br /> spanning domain proximal cytoplasmic domain (Argos        bound calcium (Figure 4, lane 2) expected, GST <br /> et al., 1982; Kyte Doolittle, 1982; von Heijne, 1986)       phosphorylase B GST::Cnelp (S.cerevisiae) <br /> (Figure 2). S.pombe cnxl + encodes repeats related        (Figure 4, lanes 1, 6 7 respectively) did bind motif KPEDWDE, calnexin              calcium assay. positive controls, calmodu- <br /> suggested high-affinity, low-capacity calcium binding    lin (Figure 4, lane 4), parvalbumin (Figure 4, lane 5) sites (Tjoelker et al., 1994). amino acid repeats        bands corresponding mammalian calnexin pgp present Arabidopsis thaliana, mammalian         35 (Figure 4, lane 3) obtained Triton X-1 14-extracted <br /> calnexins mammalian calreticulins. Overall,             canine pancreatic stripped rough microsomes, bound <br /> S.pombe cnxl+ 38% identical thaliana calnexin          calcium.  like mammalian calnexin calreticulin, 34% identical canine calnexin 25% identical      S.pombe Cnxl calcium-binding protein. <br /> mouse calreticulin, 22% identical S.cerevisiae <br /> CNEJ.  S.pombe cnxl + gene likely       Cnxl integral membrane glycoprotein calnexin homologue higher amino acid            Polyclonal antibodies raised E.coli-expressed <br /> identity calnexin calreticulin. Additionally,     GST::Cnxl fusion protein. antibodies recognize encodes C-terminal domain predicted transmem-        protein S.pombe 91 kDa associated brane domain, present calnexin genes.       membrane fraction. determine Cnxl integral looked calnexin calreticulin homologues     membrane protein, S.pombe membrane preparations S.pombe. Hybridization low stringency cnxl +        treated regimen sodium carbonate pH 11.5 probe entire genome, presented form         0.5 M NaCl 2.5 M <span id='am-41' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span> did release bacteriophage P1 clones, did identify related    membranes; 0.1% SDS 1% Triton X-100 <br /> sequences. determined cnxl + gene maps        led solubilization 91 kDa protein (Figure <br /> region chromosome  probes 57bJ2 20h4          SA).  properties S.pombe Cnx 1 correspond location P1 phage clone 3GJOp (data shown)      expected integral membrane protein. Membrane <br /> (Hoheisel et al., 1993). tentatively conclude cnxl +   association expected calnexin homologue S.pombe calnexin gene.                             calreticulin homologue soluble ER <br />                                                                lumenal protein. cnxl+ transcript inducible heat shock                   apparent molecular mass 91 kDa S.pombe calcium ionophore                                        Cnx1 higher predicted Cnxl sequence. eukaryotes, consensus sequence heat shock       confirm predicted site N-glycosylation element characterized repeats       residue 418 used  Figure 2B), S.pombe membrane <br /> sequence nGAAn alternating orientation (Pelham           preparations digested endoglycosidase H Bienz, 1982; <span id='am-29' about='obo:IMR_0001693' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001693" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="amin" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:32952" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="A group of compounds derived from ammonia by substituting organic radicals for the hydrogens. (From Grant &amp; Hackh&apos;s Chemical Dictionary, 5th ed)" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000588" datatype="xsd:string"></span>Amin</span> et al., 1988). Inspection         (endo-H). Immunoblotting antibodies Cnx 1 protein <br /> upstream sequences cnxl+ identified heat shock          revealed increase mobility SDS-PAGE -88 kDa <br /> elements  Figure IC), correct position    (Figure 5B). mobility change corresponds act elements controlling transcription cnxl +.        expected single potential site glycosylation (-3 kDa test  exponentially growing S.pombe cells         site) modified addition core sugar <br /> subjected transient heat shock 39&#194;&#176;C RNA        residues (Herscovics Orlean, 1993). primary <br /> transcripts analysed Northern blots. cnxl +        sequence, predicted molecular mass non- <br /> transcript induced 1.6-fold transient <br />                                     - <br />                                                                glycosylated protein 63 kDa (Figure 1 C).  <br />                                                                                                                      3065 <br />   F.Parlati et aL <br />  <br />  <br />  <br />  <br />                                                           CALNEXIN                             ...   SREIEDPEDQKPEDWDERPKIPDPDAVKPDDWNEDAPAKIPDEEATKPDGWLDDEPEYVPDPDAEKP <br />                              CALRETICUSLIN                        FLPPKKIKDPDAAKPEDWDERAKIDDPTDSKPEDWDK ............EHIPDPDAKKP <br />  <br />  <br />  <br />  <br />                              CALNEXIN <br />                              CALRETICULIN                         EDWDEEMDGEWEPP...........VIQNPEYKGEWKPRQIDNPDYKGTWIHPEIDNPE.. <br />  <br />  <br />                              B                                               PCR S                                                       PCR  <br />                                                               K         F ED         N D     E                                   V   K G             N       K       F <br />  <br />  <br />                                                          5- AARCCNGARGAYTGGGAYGA 3-                                     3, ATRTTYCCNYTYACCTTYGG 5. <br />  <br />  <br />                                   C <br />  <br />                                                                                              GtTTYGATACCTGACAGTTCGTTATGCGTCAGACTAAAGCATATTCTAATGAT <br />  <br />                                   TGAATAAATCATTgCC                                   CAGTTTACATAATTTACTTTTACCTGTGA                                                                361 <br />  <br />  <br />  <br />  <br />                                   TACTCTrrNIrIACTTAATGGGGnGAACAATTAAAGCCAAATACTTAGTATATTAAAATGCAAACGTTT-ACTGTAAACATGTTACTGC                                                      -271 <br />  <br />  <br />  <br />  <br />                                   AACC7TCATTATAGCAAGACA&apos;rTCGCTCCGTAATTCACAAAGTAAGAAGTAAAATAAATATAAAGATCTAATGCAAATACAACTT1ACCAG                                                    181 <br />  <br />  <br />  <br />  <br />                                   ATGTTTAACATACAG                                ATCATG&apos;=GTCGACTTCTArC=GTAGTGT                                                                    91 <br />  <br />  <br />  <br />  <br />                                   ATCCATCTCATCGATCAAAGCCGTAGTACTACCAAAACACACGCAAAACCGCGCACCGTAGTTATCAACTACACCACCCAACACGTGGAA                                                      1 <br />  <br />  <br />  <br />  <br />                                   ATGAAGTACGGAAAGGTATC&#194;&#182;ITrCTAGCGCTTTSTGTGCCTC=ATATGTTCGGGGATCAC=&apos;GCTGATCCTGAATCCGAACAAGAACCT                                                     90 <br />                                    M     K Y     G   K    V S       F    L     L   L   C s     L    Y    VSR     GES       L AD        P   E S         E Q             E   P   30 <br />  <br />  <br />                                   TTGGTTTTTAACCCAACTGAAGTGAAAGCGCCTCTTGTTGAGCAGTTTrCAAGGTGCATGGTCTGAACG&#194;&#182;0-GGAT-rCCTTCACATGCTAAA <br />                                    L V      F    N   P T      E     V    K     P   L V      E Q     F    Q   G      W    S E     R   W     P S             H         K   60 <br />  <br />  <br />                                   CGATITTGTAAATGGTATTGAAGAAATGAGCTATGTAGGTGAATGGACTGT7rGAGGAAAGTTCTGGTCCCGGTGCCCTT-AAGGGTGAAGCC                                                  270 <br />                                    S F      V    N   GlIREE              M S     Y VG        S       NTV       E   E S       S   G   P   G AL            K G             E     90 <br />  <br />  <br />                                   GGTCTCGTTATGAAAGACGAAGCCGCTCATCATGCTATTTCTCTATGAArGATG.AACCCATAAATGAACCAGAAAAGGA&#194;&#182;TTTGGTAG&#194;&#182;Irr                                                  360 <br />                                    G    L V      M   K    D   E        AH H MASIS              Y E       P D     E    P NE           F E     K       D LV                V   120 <br />  <br />  <br />                                   CAATATGAGGTTAACCCCGAAGAGGGTCTGAACTrGTGGTGGTGCATATITTAAGCTCTrAGCAGAACCCACTCATGGTGAAATGTCAAAC                                                    450 <br />                                    Q Y E VHF ESE G L N C G G AY L K L L E P T H G E MS N                                                                                       150 <br />  <br />  <br />                                   TCCATCGATTACAGAATCATGTTGGACCCGACAAATGTGGTGTT~AATGACCGTGTrrCA7TTTATCTTCAAGCATAAGAATCCCCTrACG                                                    54 0 <br />                                    ES <span id='am-28' about='oboInOwl:id' typeof='owl:Thing'>ID</span> Y R IM F G P D K C G V N D R V N F F KMH K N PL T                                                                                     180 <br />  <br />  <br />                                   GGTGAGTATTCCGAGAAGCATCTCGACTCTAGACCCGCCAGCCTTTGAAACCTGGTATTACAAACCTTTACACTTTrAATTGTGAAGCCT                                                     630 <br />                                    IG    Y        K     L  D  S     P   S  L  L  K  P  G     T  N  L     T  L   V  K  P                                                      210 <br />  <br />  <br />  <br />                                                                                                                                                                                  720 <br />                                    D Q      T    F   E VES               N   G D     V   V    E Q     G    S   L   P    Y    DP      F       F   V       L       F       F   V   240 <br />  <br />  <br />                                   GAAATACGATtCCGAAGATATCAAGCCTGCTGATTGGGTTrGACGAACCTGAAATTCCTGACCCCAATGCGGTTAAGCCTGATGATTGG                                                      810 <br />                                    ESI      Y    D       F EDSI          K   P AD        W V     DSEP          ESI      P D      F   NA      V   K       P D             D   W   270 <br />  <br />  <br />                                   GATGAAGACGCTCCCCGTATGATTCCTGATCCTGACGCAGTGAAACCTGAGGATTGG7TAGAGGATGAACCTCTTTACATTCCTGATCCT                                                     900 <br />                                    DRE      D      PENSM               PD      P D     AV      K        F ED     W    L ED         E   P LY            SFP D               F   300 <br />  <br />  <br />                                   GAAGCTCAGAAGCCTGAGGATTGGGATGATGAAGAAGATGGTGATTGGATTCCAAGTGAAATCATCAATCCTAAATGCATTGAAGGTGCT                                                     990 <br />                                    S      Q K          FED       WND       D E ED          G D          N PS        ES       SIN     P   K   C           S              330 <br />  <br />  <br />                                   GGCTGTGGTGAATGGAAGCCACCGATGATTCGTAACCCTAACTATCGTGGCCCTTGGTCTCCTCCTATGATCCCTAATCCTGAACTCATT-                                                    1080 <br />                                    G    C   G E      N    K   P PM           SERN        P   N   YER       G   P       NEFFP         MSI     P   N       PEPF                  360 <br />  <br />  <br />                                                                                                                                                                                  1170 <br />                                    G    E       NY   P    0RK          SFP   N   P   D   Y       D    D    D       P    SN       PFG         LY          G       V       G       3 90 <br />  <br />  <br />                                   G~AACTTTGGACTATGCAGCCCAACATCCGCTTTrAGCAATATCTATGTCGGCCATTCAATTGAGGACGCTGAACGTCTTGGAA.ATGAACT                                                   1260 <br />                                    S    L   N    T   N    Q   P    N       S   F   S   N     Y    V    G   N   S    S    N   D     E   E   L       G       N       N   T   420 <br />  <br />                                   TTCCTACCTAAATTGAAAGCTGAAAGAGAATTACSCCTAAGCAAGAGTCCATGGAAAAGCAATCAATGCATGTITGACGAAGAGTCTAAT                                                     1355 <br />                                    F    L   P    K   L    K      S     R   S   L   L   S   K   Q    N    S   M   E    K    Q   S   N   N   V   D       N       S       S   N   450 <br />                                   CAAATTCTTGAGAAGT&apos;=rCTTGACGTATATGATATTVATTAAAGCTAAACTTCCTCCAAATGTTGCCGAAAAAGTTGACTACTATGTTGAA                                                   1440 <br />                                    QIS      L    S   K    F   L    D     V   Y   D   S     K      K    L   P   F    N    V     E   K   V   D       Y       Y       V   E   485 <br />                                   ACACTGGCCTAATGATCATTGCTCTGCCTACGTTATTATGTC                                                                                                     1530 <br />                                    T      S    S   T    P   E        0   5       V     V    L    G   S   L    T      V   S   L   T   C       Y       F       Y   F   515 <br />                                   TTGTCTTCCGCTTTTTCGACAGAGCGGAGGACAAAG7AGAGGCGGC                                                                                                 1620 <br />                                    F      S    S   S    P      S     L   S   T   G   T   T   S       S   K   E    Q      E   K   P   K   Q       S       T       N   T   540 <br />                                   GAGAAGATAGACGTTTC7rACGCTCCCGAAACTGAATCACCAACTGCGAAGAATGAAGACTAATGAAACTGCTAAAATGTCAAA.AGm                                                   =   1720 <br />                                    S KI D            V    EVYA           F E TES             P TA          K   NE D                                                              560 <br />  <br />  <br />                                        TCTTA=AAGATCGATAGAG7TCAATAACT=CCTTTCCTAAATAATAATCATATAATACCACTATACGCAT1810 <br />                                     A7&apos;&apos;rTGTACAAAGTTGATTGTGGAC7TTGACTGAAGCTCAAAGTAGCGACTCTGTTAAGC                         1889 <br />  <br /> Fig. 1. DNA sequence cnxl + gene deduced amino acid sequence.   central domains dog calnexin mouse calreticulin aligned amino acids 254-389 185-281 respectively (Wada et al., 199 1; Smith Koch, 1989). Amino acid sequences used design forward reverse oligonucleotide primers indicated bold type. (B) Amino acid sequences corresponding <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequences (PCR S, sense; PCR  <br /> antisense) used primers PCR amplification yeast (Y = C T; R = G; N =  C, G, T). (C) Spombe cnxJ + gene codes 560 amino acid <br /> protein. predicted protein contains signal sequence amino terminus (italics), single N-glycosylation site N4 18 (CHO), transmembrane <br /> domain (bold type) 48 amino acid predicted cytosolically oriented sequence. Putative heat shock elements indicated (underlined). Nucleotides <br /> matching heat shock element (HSE) consensus sequences marked asterisks. Sequences encoding partial HSE elements denoted asterisks, underlined. putative TATA box shaded. sequence available GenBank #M98799. <br />  <br /> 3066 <br />                                                                                            :chizosaccnaromyces pombe cainexin ihomologue cnxl+ <br />                                                                                           106   - <br />  <br />                  3-                                                                          80-                    .. <br />  <br />  <br />  <br />  <br />                  0-                                             .   #                    49.5 <br />                  . <br />                       f   Tr         7 r <br />                                                             1VVi&apos; <br />                                                                                                                          _A. <br />                  .3- <br />                                              II <br />                                100     200          300       400         500              27.5 - <br />                                         Residue Number <br />                                                                                            18.5 <br />  <br />              B                                                                                            1     2        3     4     5     6     7 <br />                                                 DS6 <br />                                                                                       Fig. 4. Identification Spombe Cnxl protein calcium binding <br />                                                                                       protein. Lane 1, GST (20 ,ug protein); lane 2, GST::Cnxl <br />                                                 L                                     (S.pombe) fusion (20 jg protein); lane 3, Triton X-1 14 extracted <br />                                                                                       canine pancreas ER (100 ig protein) showing polypeptides                                                                                       90 kDa (calnexin) 35 kDa (pgp 35) known bind Ca2+ (Wada <br />                                                                                       et al. 1991); lane 4, calmodulin (2 jig protein); lane 5, parvalbumin <br />                                      &gt;- N 411                              ER Lumen   (10 jg protein); lane 6, phosphorylase B (10 ig protein); lane 7, <br />                                                                                       GST::Cnelp (S.cerevisiae) fusion protein (20 jg protein)                                                                                       electrophoresed 10% SDS-PAGE, transferred nitrocellulose <br />                                                                                       membrane 45Ca binding perforned described previously (Wada <br />                                                                                       et al, 1991). Molecular mass markers indicated left. <br />  <br />                                                                                       anomalous migration SDS-PAGE observed                                                 D 23 <br />                                                                                       non-glycosylated canine calnexin predicted <br /> Fig. 2. Hydrophobicity plot topology Cnxl.   Hydrophobicity                  molecular mass 67 kDa mobility SDS-PAGE <br /> plot. (B) Predicted topology Cnxl showing single predicted                     corresponding 90 kDa (Wada et al., 1991). N-linked glycosylation site, single transmembrane domain proximal                 mammalian calnexin, low pl Cnxl (calculated carboxyl terminus cytoplasmic tail. predicted signal <br /> sequence cleavage residue aspartate 23 (D 23).                                  pI 4.13) likely responsible discrepancy. <br />                                                                          cnx1+ gene essential <br />                                                      C                Gene disruption performed order determine                    Heat Shock                   CThemical Stress        cnxl+ essential gene S.pombe. Approximately <br />             200. <br />                                                                         80% coding sequence replaced S.pombe <br />           -6150-                                                      ura4+ gene. sequences flanking cnxl +,  500 bp <br />           0                                0 <br />                                                                         5&apos; ORF 1.0 kb 3&apos; retained <br />           L)100-                                          u <br />                                                                         order promote good frequency homologous <br />              50-11                                                      recombination (Figure 6A). diploid S.pombe strain                                                                          Materials methods) transformed linear <br />               0-                                                        construct <span id='am-36' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> prototrophs selected. DNA <br />                    0     15      30 <br />                       Time (min.)                  .R,^9of transformants analysed PCR Southern <br />                                                  &gt; &apos;&gt; ss                blots confirm recombination occurred <br />                                                                         cnxr + locus  . Diploid transformants <br />                                                                         heterozygous cnxl + gene disruption grew normally, <br />       B                                   D                             sporulated 17 tetrads dissected. tetrad <br />                                                                         gave segregation viable spores grew <br />       28 S <br />                                                                       s form visible colonies apparently non-viable <br />                                                                         spores (Figure 6C). spores grew uracil <br />       18 -        -    _                       -~             - 18 s auxotrophs. cnxr + gene S.pombe essential <br />                                                                         viability. <br />                                                                            Southern blot analysis DNA parental <br />                                                                         diploid strain, ura4+ heterozygous diploid ura- <br />                     1    2     3                    1 2 3               spores probed following probes:   entire <br />                                                                         ura4+ gene; (ii) probe corresponding nucleotides -490 <br /> Fig. 3. Induction cnxl + heat shock calIcium ionophore        -1, 5&apos; cnxl + ORF; (iii) NsiIlEcoRV <br /> A23187. Northern blot analysis total S.pombe RNA using probe complete ORF cnxl + gene Spon                                                      cnxc <br />                                                                         fragment +  Figure 6A). probes used <br /> grown 23&#194;&#176;C shifted 39&apos;C 0 min.  B, lane 1);         distinguish wild-type recombinant <br /> 15 min  B, lane 2); 30 min  B, lane 3) grown      allele. DNA parental diploid strain (Figure 6B, <br /> 30&#194;&#176;C (C D, lane 1) treated tunicamy,cin (C D, lane 2)   lanes 1, 5, 9 13), ura4+ diploid transformant (Figure calcium ionophore, A23187 (C D, lan e 3). results     6B, lanes 2, 6, 10 14), ura- spores (Figure 6B, independent experiments evaluated densuitonetry            lanes 3, 4, 7, 8, 11, 12, 15 16) digested illustrated panels C (&#194;&#177;SD). Equal amoi <br /> applied lane gels stained eethidium bromide    EcoRI (Figure 6B, lanes 1-4 9-12) EcoRV (Figure <br /> order verify quantities 28S 18S RNIA.                     6B, lanes 5-8 13-16). Hybridization probe <br />  <br />                                                                                                                                                      3067 <br />  F.Parlati et aL <br />  <br />                                                                           <br />  <br />                        -           P- <br />                        S       s           S   -&apos;s <br />                                                 P     P     s P <br />  <br />  <br />            106 - <br />  <br />             80             - <br />  <br />  <br />  <br />  <br />            49.5 -- <br />                                                                                   B <br />  <br />        B <br />                                                                                        .. <br />                                                                                                                                                                                                                   <span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;F <br />  <br />                      ENDO-H        -       +         M.W.                                         -k <br />                                                                                                                           4.: <br />  <br />  <br />                                                 - 106 kDa <br />                                                 - 80 kDa <br />  <br />  <br />                                                 - 49.5 kDa <br />  <br />  <br />                                                                               *? x               mm <br />                                        1   2 <br /> Fig. 5. Cnx integral membrane glycoprotein.   Membranes              1.   .9 kD_) <br />                                                                                         _,O <br />                                                                                          _ _    _6a_a  prepared extracted 0.1% SDS, 1% Triton X-100, 0.1 M <br /> sodium carbonate, pH 11.5, 0.5 M NaCl (high salt), 2.5 M urea Tris-buffered saline, pH 7.5 (mock), followed centrifugation <br /> (30 min X 100 000 g) pellet (P) supernatant (S) fractions analysed immunoblotting anti-Cnxl antisera. <br /> Molecular mass markers indicated left. (B) total <br /> particulate fraction homogenized spheroplasts digested endo-H. Analysis immunoblotting revealed change mobility Cnxl 91 kDa 88 kDa. endo-H treatment, minor band 74 kDa  likely originating light 78 kDa <br /> band lane 1. significance band unknown, speculate represent minor Cnxl degradation product. <br /> Molecular mass markers indicated right. <br />  entire ura4+ gene resulted 4.7 kb band restricted EcoRI (lane 2) bands 2.3 kb                                                          ..   .N   4;   5   6   7 <br />  <br />  <br />  6.0 kb restricted EcoRV (lane 6) heterozygous diploid. bands present parental ura- spores. Hybridization probe                   Fig. 6. Gene disruption cnxl+.   Schematic representation                                                                         plasmid pFPP3 containing entire cnxl+ gene pFPP4 <br /> derived PCR fragment 5&apos; cnxl + gene                       containing cnxl::ura4. indicated restriction sites used (nucleotides -490 -1) revealed 2.9 kb band                 evaluate Southern blot analysis presence absence cnxl+ wild-type allele 4.7 kb band present                    parental heterozygous diploids, ura- spores. <br /> heterozygous diploid (lane 10). Hybridization                  (B) Analysis cnxl+ disruption Southern blot. DNA                                                                         parental diploid strain (lanes 1, 5, 9 13), heterozygous diploid <br /> probe encompassing NsiV/EcoRV fragment cnxl +                    strain (lanes 2, 6, 10 14) ura- spores (lanes 3, 4, 7, 8, 11, 12, <br /> resulted 1.7 kb band corresponding wild-type                15 16) restricted EcoRI (lanes 1-4 9-12) allele 2.3 kb band present heterozygous               EcoRV (lanes 5-8 13-16) probed    entire ura4+ <br /> diploid (lane 14) corresponding cnxlJ disrupted allele.              gene (lanes 1-8); (ii) PCR fragment nucleotides -490 -1, 5&apos; confirmed cnxl + gene disrupted                cnxl+ ORF (lanes 9-12); (iii) NsilIEcoRV fragment cnxl+ <br />                                                                         (lanes 13-16). ura- spores tetrads, e. lanes 3, 7, homologous recombination yield cnxl+/cnxl::                     11 15 spore 1, lanes 4, 8, 12 16 spore 2. <br /> ura4 heterozygous diploid. spores grew                (C) Results tetrad analysis showing viable non- <br /> ura4- contained wild-type cnxl + allele.                     viable    spores   tetrad. <br />  <br /> 3068 <br />    W <br />       F.X <br />         <br />  <br />  <br />  <br />         1-7F <br />       r.X              x <br />  <br />  <br />  <br />  <br />              /fF7 <br />                  &quot;ilziziiE <br />  <br />  <br />  <br />  <br />   -%c&apos; x ?:--27172A7/?. <br />  <br />   ;, FL         &apos;cccc2-.YN .xK_ <br />                                                                            L .c                 Corn o. <br />                                                                                                                Schizosaccharomvces nombe calnexin ihomologue cnxl+ <br />  <br />                                                                                                            disruption haploid cells. length Cnx 1 protein <br />                                                                                                            detectable intracellularly membrane fractions <br />                                                                                                            (Figure 7B, lanes 4, 11 12). Plasmid pCNX 524 <br />                                                                                                            expressed Cnxl lacking cytosolic tail produced                                                                                                            protein lower molecular weight, detect- <br />                                                                                                            able intracellularly membrane fraction (Figure 7B, <br />                                                                                                            lanes 1, 2 3). truncated protein able                                                                                                            complement cnxl::ura4 gene disruption haploids. <br />                                                                                                            Plasmid pCNX 484 pCNX 474 expressed Cnxl <br />                                                                                                            protein lacking C-terminal cytosolic domain                                                                                                            putative transmembrane domain. Remarkably                                                                                                            complemented lethal phenotype gene disruption. <br />                                                                                                            constructs, Cnxl protein detectable <br />       B                                                                                                    intracellularly (Figure 7B, lanes 5-10) truncated <br />                                                                                                            Cnxl proteins detected medium                                                                                                             shown), retained ER                     CNX 524          560     CNX 474            ,NX 484        CNX 560         FL 90       secreted (Figure 7A). attempted complement <br />                                                            T-                  1-                        gene disruption strain length canine <br />       106-                                                                                                 calnexin expressed plasmid. mammalian <br />        80 - <br />                                                                                                            protein (FL 90) detected diploid cells heterozygous <br />                                                                                                            cnxl disruption (Figure 7B, lanes 13 14). <br />                                                                                                             obtain transformants   49.5-                                                                                                    mammalian calnexin complemented lethal pheno- <br />                                                                                                            type cnxl::ura4 gene disruption haploid cells. <br />                    1         2   3    .4     -         7             9    10      11   12     13 14 <br />                                                                                                            Discussion <br /> Fig. 7. Complementation cnxl::ura4 disruption haploid cells. <br />   Constructs used complement cnxl::ura4 disruption                                            established calnexin acts molecular <br /> depicted lumenal (dashed), transmembrane (filled) cytosolic                                       chaperone recognizes secretory glycoproteins tails (open) rectangles. ability complement (Comp)                                            unassembled multimeric cell surface receptors  cnxl::ura4 disruption intracellular   extracellular                                 Bergeron et al., 1994). mammalian cells, function <br /> (E) location (Loc) noted. pCNX560 encodes entire Cnxl <br /> protein; pCNX 524, pCNX 484 pCNX 474 encodes Cnxl                                                      molecular chaperone largely elucidated <br /> truncated amino acid 524, 484 474 respectively. pFL90 encodes                                       vivo studies showing transient association length canine calnexin gene. (B) Immunoblot detection                                          incompletely folded secretory proteins calnexin. Cnxl canine calnexin S.pombe cells. Protein (10 tg)                                          molecular mechanism recognition proteins membrane fraction different cnxl::ura4 haploids <br /> complemented pCNX 560 (lanes 4, 11 12), pCNX 524                                           calnexin remains uncertain. suggested (lanes 1, 2 3), pCNX 484 (lanes 8, 9 10) pCNX 474                                              multiple mechanisms including N-linked glycosylation <br /> (lanes 5, 6 7) electrophoresed 8% SDS-PAGE,                                                 direct binding <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> motifs incompletely <br /> transferred nitrocellulose blotted anti-Cnxl antisera.                                         folded proteins involved (Bergeron et al., 1994; Ware <br /> Canine calnexin detected membrane fraction diploid cells                                       et al., 1995). certain incompletely folded proteins <br /> heterozygous cnxl deletion transformed pFL90 (lanes 13 14). Membrane proteins (10 ,ug) treated                                                  recognized. example, transferrin expressed (lanes 1-12), immunoblotted anti-canine calnexin antisera.                                        human HepG2 cells released calnexin Molecular mass markers (kDa) indicated left.                                                    disulphide bonds form (Ou et al., 1993).                                                                                                             known calnexin recognize free oligosaccharide <br /> conclude                          cnxl <br />                                                   + <br />                                                       gene            essential        viability.   GlcNAc2 Man9 Glc1 (Ware et al., 1995) affinity microscopic examination apparently non-                                                        oligosaccharide polypeptide probably <br /> viable spores observed did germinate                                                      higher (Arunachalam Cresswell, 1995). known <br /> divided produce microcolonies 20-50 cells.                                                     features calnexin necessary fore, cnxl + essential spore germination                                                function. apparent ubiquity mammalian cells makes <br /> results suggest cell divisions, Cnx                                                     experimental alteration function difficult, diluted cells eventually stop dividing.                                                        characterization yeast calnexin homologue                                                                                                            advantage. previously cloned CNEJ Complementation                                cnxl::ura4 gene                                      S.cerevisiae similar strategy described  <br /> disruption haploid cells                                                                                shown CNEJ sequence similarities cnxl::ura4 gene disruption lethal haploids,                                                calnexin/calreticulin family shares determine region molecule                                                        properties calnexins does bind calcium essential growth. cnxl::ura4 gene disruption strain                                                C-terminal cytoplasmic domain (Parlati et al., 1995). transformed series deletion plasmids lacking                                                 S.pombe cnxl + gene described sequence <br /> fragments cnxl + C-terminus. ability com-                                                  similarity properties common calreticulin/ <br /> plement lethal cnxl::ura4 gene disruption,                                                  calnexin, characteristics argue location Cnx protein determined (Figure                                                     S.pombe cnxl + codes homologue calnexin 7A). Plasmid pCNX 560, expressing length Cnx 1,                                                       identity S.cerevisiae CNEJ certain. <br /> complemented lethal phenotype cnxl::ura4 gene                                                      complete sequence, Cnxl 38% identical  <br />                                                                                                                                                                   3069 <br />  F.Parlati et aL <br />  <br />                                       S.poznbe Cnxl               249   KPADWVD         266   KPDDWDE         285   KPEDWLE         304   KPEDWDD <br />                     thaliana calnexin        230   KPEDWDE         247   KPEDWDE         266   KPEGWLD         285   KPEDWDD <br />                    Canine calnexin             284   KPEDWDE         301   KPDDWNE         320   KPDGWLD         339   KPEDWDE <br />                    S.cerevisiae Cnelp          255   KPHDWDD         272   KLSDRDE         291   EPPEWNS         310   KPSWWKE <br />                    Mouse calreticulin          215   KPEDWDE         232   KPEDWDK                               249   KPEDWDE <br />  <br />  <br />  <br />  <br />                    B <br />                    S.pombe Cnxl                132 CGGAYLKLL               156 IMFGPDKCG              324 PKCIEGAGCG <br />                     thaliana calnexin        108 CGGAYLKYL               136 IMFGPDKCG              305 PKCEAAPGCG <br />                    Canine calnexin             161 CGGAYVKLL               188 IMFGPDKCG              358 PKCESAPGCG <br />                    S. cerevisiae Cnelp         124 CGGAFIKLM               154 LVFGPDYCA              330 PLCTAERGCG <br />                    Mouse calreticulin          105 CGGGYVKLF               130 IMFGPDICG <br />  <br /> Fig. 8. Comparison predicted high affinity calcium binding motifs   cysteine motifs (B) S.pombe Cnxl, thaliana calnexin, canine <br /> calnexin, S.cerevisiae Cnelp mouse calreticulin proteins. Amino acid sequences aligned using GeneWorks program. numbers <br /> denote place original sequence residue motif. <br />  <br /> canine calnexin 25% identical mouse calreticulin,                     suggested N-linked glycosylation specificity compared overall identity Cnelp 24%                       calnexin (Ou et al., 1993),  inhibition canine calnexin 21% mouse calreticulin.                      oligosaccharide glycosylation precursor sensed mammalian plant calnexins, S.pombe Cnxl                          calnexin. addition, result suggest Cnxl repeats related sequence KPEDWDE,                        protein role Ca2+ regulation sequestration. S.cerevisiae Cnelp single related motif KPHDWDD                        heat shock consensus sequence identified conserved variations motif                    S.cerevisiae CNEJ gene, unable equivalent positions (Figure 8A). observation                        experimental conditions altered transcription <br /> noteworthy repeat motif suggested                     (Parlati et al., 1995). <br /> previous studies represent high-affinity calcium                         essential nature S.pombe cnxl+ gene binding domain calnexin calreticulin                           contrast non-essential nature S.cerevisiae <br /> (Michalak et al., 1992; Tjoelker et al., 1994). shown                     CNEJ gene (Parlati et al., 1995). S.cerevisiae cnel Figure 4, Cnelp does bind calcium, Cnxl                       deleted strain small effect observed does. Mammalian calnexins S.pombe Cnxl possess                           function temperature-sensitive mutant  cysteine residues remarkably conserved                         pheromone receptor (ste2-3) non-permissive calnexins including Cnelp,                      temperature. Furthermore, increase secretion conserved calreticulin (Figure 8B).                                  heterologously expressed mammalian ctl-antitrypsin    diagnostic difference calnexins cal-                      observed (Parlati et al., 1995). explain small <br /> reticulins type ER membrane                         effects speculated systems proteins, calreticulins ER lumenal proteins                   protein folding ER S.cerevisiae. Despite search <br /> carboxy terminal KDEL retrieval signals (Pelham, 1990;                       low stringency hybridization S.cerevisiae <br /> Michalak et al., 1992). S.cerevisiae shown                       S.pombe genomes, able demon- Cnelp integral membrane glycoprotein                       strate presence related calnexins calreticulins <br /> localized ER membrane (Parlati et al., 1995).                      (Parlati et al., 1995). shown S.pombe Cnxl integral mem-                           Complementation lethal cnxl::ura4 disruption <br /> brane glycoprotein. demon-                          strain shown C-terminal molecule <br /> strated ER localization, tight banding protein                essential viability. deletion SDS-PAGE consistent ER, Golgi                             transmembrane predicted cytosolic domain <br /> glycosylation. expect Cnxl localized                       led soluble truncated form Cnxl protein ER. obvious ER retention motif present                       secreted complemented. able, Cnxl cytosolic tail ER retention/retrieval                     detect truncated Cnxl protein intracellularly. <br /> motif mammalian calnexin  extreme carboxy-                            previous studies pointed lumenal domain <br /> terminal RKPRRE motif), present Cnx 1                              calnexin important function molecular <br /> (Rajogopalan et al., 1994).                                                  chaperone mammalian cells (Ou et al., 1993),    S.pombe cnxl+ inducible heat shock                           speculate function carried ER <br /> treatment calcium ionophore A23187.                     lumenal soluble Cnx protein ER. feature described mammalian calnexin                        similar result observed S.cerevisiae (Bergeron et al., 1994) known                  ER lumenal molecular chaperone BiP. ER chaperones BiP, GRP 94 PDI                            carboxy-terminal tetrapeptide ER retrieval motif, HDEL, <br /> mammalian cells yeasts (Normington et al., 1989;                         deleted essential gene BiP, protein Rose et al., 1989; Mori et al., 1992; Pidoux Armstrong,                  secreted.  small BiP remains <br /> 1992). difference mRNA abundance cnxl+                          intracellular cells viable (Hardwick et al., <br /> response A23187 tunicamycin consistent                     1990). analogy, propose sufficient levels  <br /> 3070 <br />                                                                                   Schizosaccharomyces pombe calnexin homologue cnx1+ <br />  <br /> truncated soluble Cnx 1 remain ER order                         Membrane extraction endo-H digestion <br /> perform essential function. unable prove                       S.pombe membranes (ML fraction) prepared treated essentially <br />                                                                              described (Parlati et al., 1995). Membranes mixed 1 vol <br /> definitively lethal phenotype cnxl::ura4                     M NaCl, 0.2 M sodium carbonate, pH 11.5, 5 M urea, 2% <br /> disruption lack molecular chaperone                         Triton X-l00 0.2% SDS subsequently analysed described <br /> function.  studies lumenal domain                        (Feldheim et al., 1992). Cnxl antiserum used 1:4000 mammalian calnexin shown interact                       dilution. Endo-H digestions performed incubating 20 jg ML <br />                                                                              fraction proteins 100 mM sodium acetate, pH 4.9, 150 mM NaCl, secretory glycoproteins (W.-J.Ou et al., unpublished                    10 mM DTT, 1% Triton X-100, 0.1% SDS + inhibitors (1 mM PMSF, <br /> results).  likely domain molecular                  1 mg/ml pepstatin, 1 mg/ml leupeptin 1 mg/ml aprotinin) chaperone properties functions.                   incubating 2 ,ug endo-H 16 h 37&#194;&#176;C. <br />    lack complementation cnxl::ura4 dis- <br /> ruptant mammalian (canine) calnexin surprising.                     45Ca overlay <br />                                                                              Spombe GST::Cnx 1 fusion protein (residues 23-492) expressed shown protein S.pombe                        Ecoli purified (Smith Johnson, 1988). Samples electro- <br /> membrane-localized. ER lumenal region cnxl+                           phoresed SDS-PAGE prepared 45Ca overlay exactly (amino acids 1-474) complement cnxl:: ura4                      previously described. Control experiments carried canine <br /> disruption 50% sequence similarity equivalent                   pancreatic ER membranes obtained extraction Triton X- 1 14 (Wada <br /> region mammalian calnexin. observation                       et al., 1991), Cnelp previously described (Parlati et al., 1995). <br /> lumenal domain Cnx 1 complement disrup-                     Gene disruption <br /> tion secreted, argues mammalian                    4.3 kb PstI DNA fragment containing complete cnxl+ gene calnexin ER retained, present                      cloned pTZ19R creating plasmid pFPP3. PCR fragment containing <br /> perform ER function. expect Cnx 1                       nucleotides -490 -1 (5&apos; coding sequence) amplified, cut mammalian calnexin recognize secretory                      SpeI cloned SpeI/SmaI site pBluescript KS+ creating <br />                                                                              plasmid pFPP3. 1. NsilVEcoRV fragment (3&apos; coding sequence) <br /> glycoprotein substrates, sequence differences                        pFPP3 cut, purified inserted EcoRV site proteins value order map regions                      pFPP3. 1 creating plasmid pFPP3.2. 1.8 kb HindIII fragment containing <br /> essential molecular chaperone function.                                  ura4+ gene (Grimm et al., 1988) blunt-ended inserted                                                                              similarly treated EcoRI site pFPP3.2 creating plasmid pFPP4, <br />                                                                              essentially disrupting cnxl+ gene. Plasmid pFPP4 digested Materials methods                                                        XbaI, HindlIl  cleave insert) XmnI <br />                                                                               cuts vector) -5 jg linear fragment containing <br /> Strains media                                                            disrupted gene isolated purified GeneClean method. <br /> S.pombe strains Q358 (h- leul-32 ura4-D18 ade6-M210) Q359 (h+            fragment used disrupt diploid strain h- leul-32 ura4- <br /> leul-32 ura4-DJ8 ade6-M216) used. strains grown             D18 ade6-M2JO/h+ leul-32 ura4-D18 ade6-M216 uracil prototrophs <br /> 30&#194;&#176;C YPD medium EMM media supplemented nutrient                   selected. Colonies grouped pools 10 chromosomal <br /> requirements previously described (Moreno et al., 1991). Ecoli strain     DNA prepared (Moreno et al., 1991). PCR used screen MC1061 (Maniatis et al., 1982) used.                                     homologous recombinants using sense oligonucleotide coding                                                                              nucleotides -510 -495 (5&apos; cnxJl ORF) antisense 20 bp <br /> Cloning S.pombe cnx1+                                                     oligonucleotide ura4+ gene. 20 pools screened, <br />                                                                              15 positive expected 550 bp PCR product. pools amplify calnexin/calreticulin gene equivalent Spombe,        chosen PCR reaction repeated constituent colony. <br /> degenerate oligonucleotides, PCR primer S (5&apos; AARCCNGARGAYT-                 colonies positive chromosomal DNA GGGAYGA 3&apos;) PCR primer (3&apos; ATRTTYCCNYTYACCTTYGG                        prepared  Southem blots performed digesting 5&apos;) used amplify genomic DNA. Amplification reactions           DNAs EcoRI EcoRV, followed electrophoresis 1% <br /> performed previously described (Parlati et al., 1995). products       agarose gels transfer nylon membranes. 32P-labelled probes PCR reaction electrophoresed 2% agarose gel             used    entire ura4+ gene hybridized genomic EcoRI stained ethidium bromide. band -350 bp purified            EcoRV digests; (ii) PCR fragment -490 -1 hybridized cloned SmaI site pTZ-19R (Pharmacia). fragment           genomic EcoRI digest; (iii) Nsil/EcoRV fragment cnxl + hybridized <br /> sequenced encode peptide high amino acid             genomic EcoRV digest. <br /> sequence similarity canine calnexin. PCR fragment 32P- <br /> radiolabelled using QuickPrime method (Pharmacia) sub- <br /> sequently used probe clone length calnexin Spombe        Complementation cnxl gene disruption <br /> genomic library plasmid pWH5 (Wright et al., 1986). gene         Sequences encoding length cnxl +, cnxl + terminated amino <br /> sequenced standard procedures sequence cnxl +              acids 524, 484 474 amplified using PC <br /> </body></html>