<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Billerbeck Panke Microbial Cell Factories 2012, 11:110 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  TECHNICAL NOTES                                                                                                                               Open Access <br />  genetic replacement selection-based <br /> engineering essential proteins <br /> Sonja Billerbeck Sven Panke* <br />  <br />  <br />   Abstract <br />   Background: Essential genes represent core biological functions required viability. Molecular <br />   understanding essentiality design synthetic cellular systems includes engineering essential <br />   proteins. impediment effort lack growth-based selection systems suitable directed evolution <br />   approaches. <br />   Results: established simple strategy genetic replacement essential gene (library  variant(s) <br />   transformation. <br />   validated using different essential genes plasmid combinations reproducibly <br />   shows transformation efficiencies order 107 transformants microgram DNA identifiable <br />   false positives. allowed reliable recovery functional variants 105-fold excess   non-functional variants. outperformed selection conventional bleach strains orders   magnitude, recombination functional non-functional variants interfered reliable recovery <br />   recA negative strains. <br />   Conclusions: propose selection extremely suitable evaluating large libraries   engineered essential proteins resulting reliable isolation functional variants clean strain background <br />   readily used vivo applications expression purification use vitro studies. <br />  <br />  <br /> Background                                                                           proteins complicated knock-outs cause le- percent E. coli genes essential                           thality. means phenotypes engineered proteins <br /> cell [1]. Essential genes particular scientific                      easily evaluated vivo suitable clean strain encode proteins required important biological                            backgrounds available. <br /> functions, building minimal core cellular                                decade approaches investi- <br /> viability tends conserved species.                                gated identify essential genes study func- <br /> Knowledge essential genes protein pro-                               tion vivo conditional elimination protein ducts important drug design [2,3], biotechnological                           cell. achieved triggering interference applications [4], minimal genome approaches [5-8]                             synthesis target protein transcrip- <br /> general, crucial understanding engineering                               tional translational level [10-15].  basic cellular functions required life [9].                            &#226;&#8364;&#339;bleach-out&#226;&#8364;? methods rely conditional protein elimin- <br /> construction Keio-collection, collection single                          ation elimination target gene itself gene knock-outs E. coli [1], enormously facilitated                           retain wild-type copy essential gene systematic investigation physiology E. coli                        cell. sets limitations utility systems protein strain-engineering approaches,                            ready use selection systems directed evolution <br /> restricted non-essential genes protein pro-                             experiments recombination library members ducts. Engineering approaches involving essential genes                              chromosomal wild-type gene mutations sys- <br />                                                                                      tem regulating expression wild-type protein                                                                                      lead selection false positive variants. par- <br /> * Correspondence: sven.panke@bsse.ethz.ch <br /> ETH Z&#195;&#188;rich, Department Biosystems Science Engineering (D-BSSE), <br />                                                                                      ticularly true using library small <br /> Mattenstrasse 26, 4058 Basel, Switzerland                                            fraction variants expected functional.                                        &#194;&#169; 2012 Billerbeck Panke; licensee BioMed Central  Open Access article distributed terms                                        Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, <br />                                        distribution, reproduction medium, provided original work properly cited. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                      Page 2 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> case recombination events preferentially selected     contains 18 bp SceI cleavage site allows <br /> functional library members, results selec-      conditional elimination pKOCOMP tion effort turning laborious screen bona fide      presence SceI. SceI expressed helper <br /> functional library members.                                    plasmid pI-SceI control rhamnose- <br />    evaluation large libraries, desire       inducible RhaSR/PrhaBAD [18]. variant protein engineering essential genes         helper plasmid, pParaI-SceI, carries arabinose- <br /> place wild-type gene single engineered heter-      inducible regulatory AraC/ParaBAD [19] instead ologous variant vivo functional studies      rhamnose-based  testing various set- <br /> construction specialized strains used       ups, promoters chosen exhibited high <br /> purify mutant protein free wild-type protein.           expression level presence rhamnose arabin- <br /> Phage P1-mediated transduction chromosomal                ose efficiently switched presence <br /> knock strain expressing variant essen-     glucose catabolite repression [19,20]. tial gene plasmid current            noted specific set complementation <br /> method choice achieve genetic replacement          vector flexible long carries SceI site, <br /> essential target gene variant (e.g [16]). knock-      easily introduced PCR, compatible created complementing                pKD46, required &#206;&#187; red-mediated <br /> chromosomal gene loss plasmid-encoded version          knock target gene cured growth essential gene.                                            42&#194;&#176;C. completed subse- <br />    P1-transduction widely used, protocol       quent transformation pI-SceI pParaI-SceI. time-consuming restricted variants time efficiencies successful transductions low,      Validation characterization using requiring empirical testing proper phage         essential gene product adenylate kinase (Adk) <br /> concentration followed plating PCR-screening         target gene chosen validation correct genotypes.  aim         characterization adk, encoding E. coli&#226;&#8364;&#8482;s <br /> velop general genetic set turns working        Adk, essential gene product required biosyn- <br /> essential genes engineering gene pro-         thesis purine ribonucleotides regulation ducts straight-forward approach facile work-      intracellular <span id='am-20' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> availability [1,21,22]. comple- <br /> ing non-essential genes.  present simple        mentation, adk control Ptac inserted transformation-based  Establishment sys-         pKOCOMP, giving rise pKOCOMP-adk. induc- <br /> tem begins chromosomal replacement             tion pKOCOMP-adk IPTG, chromosomal <br /> essential gene PCR-derived selection          copy adk replaced PCR-generated kanamycin <br /> marker [17], conjunction complementary               resistance cassette [17]. genotype resulting <br /> vector-encoded version target. central element        strain E. coli adk::kan [pKOCOMP-adk] confirmed method complementation vector car-          PCR using primers binding chromosomal regions <br /> ries SceI nuclease recognition site        downstream adk-locus (Additional file 1: <br /> rapidly conditionally eliminated presence        Table S1 Additional file 2 Figure S1). complete SceI nuclease-expressing helper plasmid.           selection  strain adk::kan [pKOCOMP-adk] <br /> elimination complementation vector, cells           transformed helper plasmid pI-SceI. result- electrocompetent transformed vector-           ing strain adk::kan [pKOCOMP-adk; pI-SceI] grown <br /> encoded library specific variant essential         presence glucose maintenance repress <br /> gene.  actual step gene exchange reduced        SceI nuclease production.  cells single transformation plus preparation suitable      transferred glucose-free LB medium, SceI produc- <br /> knock strain.                                              tion apparently efficiently induced addition <br />                                                                rhamnose, colony-forming units recov- <br /> Results                                                        ered induction-8period 180 minutes <br /> Overview replacement                             (Figure 2A). Importantly, copy adk selection flexibly assembled vari-        gene, cells remained viable generations, <br /> ous user-defined ways, discussed sim-       presumably mRNA protein depleted. <br /> plest version  relies vectors: knock   expected linear DNA rapidly degraded complementation vector pKOCOMP helper              cellular exonucleases half-life range plasmid pI-SceI (Figure 1A Table 1). Plasmid pKO-          minutes [23] proteins relatively stable <br /> COMP derivative pKD46 [17] encodes             half-lives range hours [24]. <br /> arabinose-inducible &#206;&#187; red recombination (genes            intermediate period cells elec- <br /> &#206;&#178;, &#206;&#179; exo) essential gene     trocompetent washing <span id='am-15' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O</span> glycerol  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 3 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 1 Overview transformation-based replacement essential genes.  Plasmids required establishment selection <br />  B: General procedure: essential target gene adk replaced antibiotic resistance cassette pKOCOMP-adk complements  chromosomal loss. Plasmid pKOCOMP conditionally eliminated expression SceI nuclease helper plasmid pI-SceI.  elimination, cells electrocompetent transformed variant library investigation evaluate functionality  select functional library members complement loss pKOCOMP. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                  Page 4 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> Table 1 Bacterial strains plasmids used study <br /> Bacterial strain plasmid               Characteristics                                                             Source <br /> Strains <br /> W3110                                     F- &#206;&#187;- rph-1 INV(rrnD, rrnE)                                                 [33], internal <br />                                                                                                                       strain collection <br /> DH10B                                     F- endA1 recA1 galE15 galK16 nupG rpsL &#206;&#8221;lacX74                              [34], internal <br />                                           &#206;&#166;80lacZ&#206;&#8221;M15 araD139 &#206;&#8221;(ara,leu)7697 mcrA &#206;&#8221;(mrr-hsdRMS-mcrBC) &#206;&#187;-              strain collection <br /> W3110 adk::kan                            Chromosomal adk replaced KmR cassette.                                 work <br />                                           strain viable adk deletion complemented. <br /> W3110 groE::kan                           Chromosomal groS groL replaced KmR cassette.                       work <br />                                           strain viable groE deletion complemented. <br /> W3110                                     Chromosomal secB gpsA replaced KmR cassette.                       work <br /> secB-gpsA::kan                            strain viable gpsA deletion complemented. <br /> SB&#206;&#8221;recA                                   lacIq rrnB3 &#206;&#8221;lacZ4787 hsdR514 &#206;&#8221; (araBAD) 567 &#206;&#8221; (rhaBAD)                     work <br />                                           568 rph-1 groE::ParaBAD-groE-KmR recA::FRT <br /> Plasmids <br /> pKD46                                     ori pSC101ts, ApR. Encodes &#206;&#187; red recombination genes                        [1] <br />                                           &#206;&#179;, &#206;&#178; exo control arabinose-responsive promoter ParaBAD. <br /> pKOCOMP-adk                               pKOCOMP-derived vector adk control                         work <br />                                           IPTG-responsive tac promoter <br /> pSEVA432                                  ori pBBR1, SpecR, multiple cloning site                                     provided                                                                                                                       Victor Lorenzo <br /> pCOMP-adk                                 pSEVA432 encoding adk control IPTG-responsive               work <br />                                           tac promoter, contains SceI cleavage site <br /> pCOMP-ESL                                 pSEVA432 encoding groS groL control                           work <br />                                           IPTG-responsive tac promoter, contains SceI cleavage site <br /> pCOMP-secB-gpsA                           pSEVA432 encoding natural secB-gpsA transcriptional unit,           work <br />                                           contains SceI cleavage site <br /> pSEVA132                                  ori pBBR1, ApR, multiple cloning site                                       provided                                                                                                                       Victor Lorenzo <br /> pSEVA132-adk                              pSEVA132 encoding adk control natural promoter             work <br /> pSEVA132-adkStop                          pSEVA132-adk internal stop codon                                    work <br /> pSEVA132-adkwatermark                     pSEVA132-adk <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span> insertion position P140                  work <br /> pSEVA132-groE                             pSEVA132 coding groS groL control                    work <br />                                           natural promoter <br /> pSEVA132-groEStop                         pSEVA132-groE internal stop codon                                   work <br />                  watermark <br /> pSEVA132-groE                             pSEVA132-groE peptide insertion site I301                     work <br /> pSEVA132-secBgpsA                         pSEVA132 encoding secB gpsA control                  work <br />                                           natural promoter <br /> pSEVA132- secBgpsAStop                    pSEVA132- secBgpsA internal stop codon                              work <br /> pSEVA671                                  ori p15A, GmR, multiple cloning site                                        provided                                                                                                                       Victor Lorenzo <br /> pI-SceI                                   pSEVA671, SceI nuclease control rhamnose                work <br />                                           inducible promoter PRha response regulators <br />                                           RhaS RhaR, derived rhammnose metabolizing <br />                                           transcriptional unit E. coli. <br /> pParaI-SceI                               pI-SceI PrhaBAD regulators RhaS RhaR exchanged                 work <br />                                           arabinose promoter ParaBAD regulator araC <br />  <br />  <br /> transformed various test plasmids (Table 1) order                   transformation efficiencies. Transformation efficiencies determine transformation efficiencies fre-                       107 &#206;&#188;g-1 DNA routinely achieved. quency false positive variants. Vector pSEVA132-adk                      intensively characterized regarding frequency encoding wild-type adenylate kinase control                       false positive variants arise recom- natural promoter used determination                       bination library plasmid residual linearized <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                          Page 5 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />  Figure 2 Characterization pKOCOMP-adk/pI-SceI pCOMP-adk/pParaI-SceI systems.  Elimination pKOCOMP-adk  adk::kan rhamnose-induced expression SceI nuclease helper plasmid pI-SceI. Red arrow: Time point harvest competent <br />  cell preparation. inlet gives better resolution elimination dynamics. chromosomal adk-replacement established using <br />  pKOCOMP-adk. B: Elimination pCOMP-adk adk::kan arabinose-induced expression SceI nuclease helper plasmid <br />  pParaI-SceI. chromosomal adk-replacement established using pKD46 pCOMP-adk. <br />  <br />  <br />  <br />  <br /> pKOCOMP-adk uninduced subpopula-                             transformed cell (Table 2). demonstrates tion cells maintained pKOCOMP (Table 2).                          importance careful management plasmid <br /> Recombination-based false positives tested                        elimination step. <br /> transformation pSEVA132-adkstop, harboring stop <br /> codon adk gene repaired recom-                    Flexibility replacement bination allowing growth corresponding carrier                     verify selection set al- <br /> cell. False positives incomplete induction                    ternative combinations, constructed pCOMP-adk. <br /> examined transformation vector                          vector based pBBR1 ori expected <br /> pSEVA132. cases false positive colonies                   copy number 10-20 cell [25], carrying Ptac <br /> detected SceI expression induced pKO-                      promoter-controlled adk gene, lac-repressor LacI, <br /> COMP eliminated competent cell prepar-                         SceI cleavage site. previous comple- <br /> ation (Table 2), corresponding frequency                     mentation vector pKOCOMP-adk used temperature- 2 x 10-4 false positives transformed cell.                   sensitive pSC101 ori 2-3 copies cell  detectable frequency recombination                   grown 37&#194;&#176;C [26], increased copy number events pKOCOMP eliminated                             pCOMP-adk allowed examination plas- <br /> transformation test plasmids tried                    mid elimination sufficiently efficient higher intra- eliminate pKOCOMP solely growth non-                         cellular plasmid concentration. used pCOMP-adk permissive temperature. case, SceI nucle-                 combination pKD46 replace adk kanamycin <br /> ase expression induced transformation                   cassette. curing cells pKD46 growth 43&#194;&#176;C, <br /> plating arabinose-supplemented agar plates,                  resulting strain adk::kan [pCOMP-adk] trans- plasmids simultaneously present                      formed pParaI-SceI used gene replacement <br /> cells short period time.                        described using arabinose induce SceI pro- <br /> legitimate events frequency 3 x 10-3                      duction. Despite higher copy number pCOMP,  <br />  <br /> Table 2 Transformation efficiencies frequency false positive variants Adk-specific selection systems <br />                   Strain                          SceI induction                            Number transformants <br />                                                 competent cell preparation                      (frequency CFU transformant) <br />                                                                                    pSEVA132-adk         pSEVA132-adkstop           pSEVA132 <br /> W3110 adk::kan [pKOCOMP-adk, pI-SceI]                                            20 200 &#194;&#177; 2900        66 &#194;&#177; 21 (~3x10-3)       28 &#194;&#177; 13 (~1x10-3) <br />                                                              Yes                    21 000 &#194;&#177; 6000           0 (&lt;2x10-4)            0 (&lt;2 x10-4) <br />                                                                                                                           -3 <br /> W3110 adk::kan [pCOMP-adk, pParaI-SceI]                                          32 100 &#194;&#177; 3700         61 &#194;&#177; 17 (2x10 )        35 &#194;&#177; 11 (1x10-3) <br />                                                              Yes                    28 000 &#194;&#177; 2200           0 (&lt;2x10-4)             0 (&lt;2x10-4) <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                     Page 6 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> plasmid rapidly eliminated cells       pCOMP-plasmid carrying adk (Figure 3).  presence arabinose (Figure 2B). Probably       systems routinely showed transformation efficien- faster set possibly higher expression         cies 106-107 colonies &#206;&#188;g-1 DNA transformed <br /> levels arabinose responsive promoter ParaBAD          positive control vectors pSEVA132-groE [19,20], elimination-dynamics pCOMP/pParaI-SceI       pSEVA132-secBgpsA. false positive variants faster pKOCOMP/pI-SceI         detected transformation test plasmids  elimination data sys-        pSEVA132-groEstop pSEVA132-secBgpsAstop,  <br /> tems difficult compare differences          structed analogy pSEVA132-adkstop  Import- <br /> copy number promoters, results indicate       antly, general feature cells stay described approach set multiple ways,       viable - judged doubling time comparison <br /> making easy adapt plasmid strategies spe-       non-induced control culture did produce <br /> cific purposes. Transformation efficiencies           SceI - generations, depending pCOMP/pParaI-SceI (~107 colonies &#206;&#188;g-1 DNA)            target gene, loss complementing plasmid. comparable pKOCOMP/pI-SceI                important characteristic cells <br /> (Table 1) identify false positive           competent transformation library <br /> transformants.                                               variant essential gene period                                                              complementing plasmid lost Generality                                     longer contribute recombination. confirm utility limited adk easily extended essential       Performance selection <br /> genes, constructed vivo selection systems    test regarding performance reliably <br /> essential gene products: chaperonin GroEL        identifying functional variants large library chaperonin GroES (encoded groE operon          contamination false positive variants challenged taining genes groL groS), glycerol-3-            GroEL- Adk-specific selection mock li- <br /> phosphate dehydrogenase (GpsA encoded gpsA).          braries various excesses non-functional variants. establishment GroEL-specific intro-         conventional bleach available duced SceI cleavage site vector pSEVA431-      GroEL  E. coli MGM100 strain [10]), compared <br /> groE PCR giving rise vector pCOMP-groE. Plasmid        performance replacement developed pSEVA431-groE encodes groE-operon control          bleach  strain MGM100, groE pro- IPTG-inducible Ptac promoter. harbors     moter replaced araBAD promoter lacI gene, spectinomycin resistance cassette repli-    regulatory gene araC. strain main- <br /> cates pBBR1 ori. Plasmid pCOMP-groE used          tained presence arabinose GroEL production combination pKD46 replace chromosomal         chromosomal locus fully repressed groE-operon kanamycin resistance cassette (Add-         presence glucose. way, functional variants itional file 2 Figure S1). resulting strain groE::kan    selected library presence glucose.  <br /> [pCOMP-groE] cured pKD46 43&#194;&#176;C           copy wild-type gene present selec- <br /> transformed helper plasmid pParaI-SceI.             tion, recombination lead selection false positive <br />    GpsA-specific constructed cloning       variants, dominate stringent selection natural gpsA transcriptional unit (consisting genes   conditions. improve strain regarding expected <br /> secB gpsA control natural promoter)       recombination frequency deleted recA leading vector pSEVA431 using primers encoding         strain SB&#206;&#8221;recA (Additional file 2: Figure S1). <br /> SceI restriction site, resulting vector pCOMP-               created mock libraries contained decreasing ratios <br /> secBgpsA. Chromosomally encoded secB gpsA           functional non-functional gene variants 1:102 replaced kanamycin resistance cassette using       1:105 (functional:non-functional) mimicking libraries <br /> pKD46 (Additional file 2 Figure S1). complete         decreasing number variants expected set  strain secBgpsA::kan [pCOMP-secBgpsA]            functional large pool non-functional variants, <br /> cured pKD46 transformed helper plasmid           typical e.g. libraries created error-prone <br /> pParaI-SceI. systems characterized regarding       PCR different error rates. non-functional var- <br /> elimination dynamics complementing plasmids           iants used pSEVA132-groEstop pSEVA132-adkstop. <br /> pCOMP-groE pCOMP-secBgpsA SceI induc-            functional variants used pSEVA132-groEwatermark <br /> tion, regarding transformation efficiencies    pSEVA132-adkwatermark. plasmids encode electrocompetent cells prepared pCOMP-                GroEL Adk-variant frame insertion elimination frequency false positive variants.   short peptide previously identified permissive sites. <br /> pCOMP-type plasmids lost comparable rate        variants fully functional identified <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                                          Page 7 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />     Figure 3 Characterization pCOMP-groE/pParaI-SceI pCOMP-secBgpsA/pParaI-SceI  Elimination   pCOMP-groE <br />     groE::kan (B) pCOMP-secBgpsA secBgpsA::kan presence arabinose helper plasmid pParaI-SceI. Elimination induced <br />     arabinose-induced expression SceI nuclease helper plasmid pParaI-SceI. Red arrow: Time point harvest competent cell <br />     preparation. <br />  <br />  PCR using watermark-specific primers. compare                                functional non-functional variants. GroEL-specific selection GroEL                                 plating incubation overnight LB-agar plates <br /> bleach SB&#206;&#8221;recA, electrocompetent W3110                                  supplemented arabinose, corresponding anti- <br /> groE::kan [pCOMP-groE, pParaI-SceI] cells electro-                              biotics,  case W3110 groE::kan [pCOMP- <br /> competent SB&#206;&#8221;recA cells transformed dif-                             groE, pParaI-SceI], glucose, colonies scored ferent mock libraries separate experiments.                                  subset colonies genotyped watermark- <br /> positive negative controls, cells transformed                             specific primers (Table 3). <br />  <br />  <br /> Table 3 Recovery functional GroEL-variants Adk-variants mock libraries increasing excess non-functional variants using established selection comparison conventional bleach Strain                         Library (functional: non-functional)        Number colonies         Number genotyped           Number detected <br />                                                                               transformationa                    variants                false positive variants <br /> W3110 groE::kan                            functional                           ~ 27 000                           20                           0 <br /> [pCOMP-groE, pParaI &#226;&#8364;&#8220;SceI] <br />                                                  1:102                                  231                             20                           0 <br />                                                  1:103                                   33                             20                           0 <br />                                                  1:104                                   4                              4                            0 <br />                                                  1:105                                   1a                             1                            0 <br />                                          non-functional                             0                              0                            0 <br /> SB&#206;&#8221;recA                                    functional                           ~ 25 000                           20                           0 <br />                                                  1:102                                  253                             20                           0 <br />                                                  1:103                                   25                             20                           1 <br />                                                  1:104                                   7                              7                            5 <br />                                                  1:105                                   6                              6                            5 <br />                                          non-functional                             7                              7                            7 <br /> W3110 adk::kan                             functional                           ~ 46 000                           20                           0 <br /> [pCOMP-adk, pParaI-SceI] <br />                                                  1:102                                  459                             20                           0 <br />                                                  1:103                                   36                             20                           0 <br />                                                  1:104                                   3                              3                            0 <br />                                                  1:105                                   1                              1                            0 <br />                                          non-functional                             0                              0                            0            9 <br />  Typically, 10 cells used transformation, cases allowed isolate positive colony high-stringency case. cases, number transformed cells increased 3*109 cells. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                        Page 8 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />    GroEL-specific  numbers colonies         transformation-competent. Efficient plasmid elimination <br /> correlated numbers expected trans-            essential prerequisite high performance formation efficiencies decreasing number              evaluation large libraries functional variants library.          stringent conditions elimination wild-type <br /> stringent library (functional variants frequency         gene cells prior introduction variants pre- <br /> 10-5) functional variants carrying watermark            vents recombination-based background growth. recovered transformation ~ 81,000 cells.        major shortcoming conventionally used bleach colony detected transformation approxi-             systems recombination silenced <br /> mately number cells non-                   wild-type gene selection. recA strain, <br /> functional variants. contrast using SB&#206;&#8221;recA          RecA-independent recombination homologous experiment, false positive variants                regions occurs frequencies 10-3-10-4 [27]. <br /> detected transformation negative control          soon number expected functional variants <br /> (transformation non-functional variants)          library drops 10-3 selection primarily frequency ~ 2 x 10-4. addition, identified false      recombination events instead functional library <br /> positive variants genotyping subset col-          members. clearly seen selection ex- <br /> onies selected different mock            periment using GroEL bleach strain SB&#206;&#8221;recA <br /> libraries (Table 3). Selection stringency positively corre-      mock libraries decreasing ratio functional <br /> lated false positive rate, requiring laborious     variants. strain deficient recA, <br /> orthogonal assays differentiate true false posi-         frequently isolated false positive variants. <br /> tives. directed evolution experiment            recombination-free, advantage sult requirement intensive post-screening          presented set bleach systems selected variants true functional library members.           selected variants directly expressed clean strain- <br /> Plasmids false positive variants,          background circumventing laborious post-transformation isolated transformation non-            work P1-transductions. allows subsequent <br /> functional variants, analyzed sequencing.        purification engineered proteins vitro stop codon reverted              characterization user-defined vivo applications wild-type codon, probably recombination               running risk wild-type contamination. chromosomal groL copy. analyzed                 elimination com- <br /> variant stop codon groL indicating          plementing vector knock strain, cells  ability grow arisen muta-           main viable generations - depending tion araBAD promoter preventing repression.          gene - electrocompetent cells prepared phenomenon described strain             elimination yield 107 transformants &#206;&#188;g-1 DNA. <br /> MGM100 [16].                                                       set essential target genes    experiment performed Adk-               expressed inducible promoter competent <br /> specific selection using mock libraries decreas-     cell preparation, appropriate bleach times ing ratios functional Adk-variants non-functional          adjusted individual gene products tuning  <br /> Adk-variants. set did identify       ducer concentration. <br /> false positive variant transformation           validated different essential E. stringent library reliably identify functional   coli proteins: Adk, glyceraldehyde-3-phosphate dehydro- <br /> variant containing watermark transformation            genase chaperonin GroEL. Transformation efficien- ~ 138,000 cells (Table 3).                                    cies absence detectable recombination events <br />                                                                  proved independent essential target gene. <br /> Discussion study present facile efficient set  tested boundaries) background-free selection         Conclusions functional, engineered, essential proteins.     current work present straightforward, <br /> based conditional elimination complemen-             transformation-based enables genetic <br /> tary plasmid-based copy essential gene knock-         replacement wild-type essential gene strain order replace essential gene genes       library variant. directly results isola- library  example).                                    tion functional variants clean strain background <br />   SceI nuclease-based cleavage proves          considerably reduced effort. suitable strategy fast efficient elimin-           Furthermore, substantially facilitates working ation complementing plasmid knock                engineering essential genes protein products <br /> strain occur strain            making experimentally easy, fast scalable task. <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                  Page 9 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br />   Finally, possible adapt intro-    operon. groS amplified W3110 PCR duced replacement strategy hosts &#226;&#8364;&#8220; like e.g.        primers groS-fw groS-rv groL PCR- <br /> yeast, Bacillus subtilis Clustridium spec. -    amplified primers groL-fw groL-rv se- <br /> homologous recombination-based knock strategies          quentially cloned KpnI HindIII sites available [28-30].                                       pACT3. Plasmid pSEVA132-groE constructed                                                              amplification natural groE operon E. coli <br /> Materials methods                                        genomic DNA using primers groE-forward groE- <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> enzymes                                         reverse cloning unique sites XmaI Restriction enzymes ligase obtained New        XbaI. Plasmid pCOMP-secBgpsA constructed clon- <br /> England Biolabs (Ipswich, MA, USA) used according        ing natural secB-gpsA transcriptional unit, amplified manufacturers&#226;&#8364;&#8482; instructions. Chemicals purchased     primers secBgpsA-forward gpsA_I-SceI-reverse, highest purity available Sigma-Aldrich, Fluka    pSEVA431 restriction sites XmaI XbaI. <br /> (Buchs, Switzerland) Roth (Lauterbourg, France). Tryp-    Plasmids pSEVA132-adkstop, pSEVA132-adkwatermark , <br /> ton yeast extract BD Bioscience (Basel,        pSEVA132-groEstop,       pSEVA132-groEwatermark                                                                                        stop <br /> Switzerland). Oligonucleotides Sanger-sequencing         pSEVA132-secBgpsA        constructed amplifica- <br /> service purchased Microsynth (Balgach,             tion ligation pSEVA132-adk, pSEVA132-groE <br /> Switzerland).                                                pSEVA132-secBgpsA using primers adk-stop-fw/ <br />                                                              adk-stop-rv, adk-watermark-fw/adk-watermark-rv, groE- <br /> Strains plasmids                                         watermark-fw/groE-watermark-rv, groE-stop-fw/groE- <br /> E. coli DH10B used general cloning procedures.       stop-rv secBgpsA-stop-fw/secBgpsA-stop-rv. <br /> E. coli W3110 used chassis chromo-          Helper plasmids pI-SceI pParaI-SceI derivatives <br /> somal deletions  Table 1 overview strains      pSEVA671 (ori p15A, GmR). gene SceI nu- <br /> used study). SB&#206;&#8221;recA derivative strain       clease amplified plasmid pSTKST [32] using <br /> MGM100 [10]. constructed BW25113 recA::          primers SceI-fw SceI-rv cloned FRT [1] P1-phage transduction using lysate         pSEVA671 PacI EcoRI restriction sites. MGM100 selecting KmR. final clone            RhaR-RhaS/PrhaBAD regulatory amplified <br /> confirmed PCR analysis groE recA::FRT          E. coli genomic DNA using primers Rha-forward <br /> locus inability grow glucose          Rha-reverse cloned SceI restric- <br /> GroEL-bleach  Plasmid pKOCOMP-adk deriva-           tion sites NsiI SpeI. AraC/ParaBAD regulatory <br /> tive pKD46 [17] constructed cloning      amplified E. coli genomic DNA using adk gene multiple cloning site expression    primers ParaBAD-fw ParaBAD-rv used ex- <br /> vector pACT3 [31] restriction sites BamHI Hin-       change RhaR-RhaS/Prha using sites NsiI SpeI. <br /> dIII using primers pKOCOMP-adk-fw pKOCOMP- <br /> adk-rv  Additional file 1 Table S1 primer            Preparation competent cells transformation <br /> sequences). resulting vector pACT-adk used        E. coli adk::kan [pKOCOMP-adk, pI-SceI] [pCOMP- <br /> template amplify Ptac promoter-controlled adk         adk, pParaI-SceI] grown overnight LB liquid <br /> gene primers pACT-forward pACT-reverse,          broth supplemented 50 &#206;&#188;g mL-1 kanamycin, 10 &#206;&#188;g <br /> coding SceI recognition site, cloned        mL-1 gentamicin, 100 &#206;&#188;M IPTG 0.5% (wt/vol) glu- <br /> pKD46 unique NcoI-site. construction          cose  efficiently repress SceI production) 30&#194;&#176;C pCOMP-adk Ptac promoter-controlled adk gene          37&#194;&#176;C. Cells pelleted, washed LB amplified primers pACT-SceI-Spe pACT-Pac            diluted 1:100 fresh LB broth, supplemented PCR product cloned unique SpeI          antibiotics glucose. Cells PacI sites pSEVA432 (ori pBBR1, SpecR resist-         grown 37&#194;&#176;C. OD600 0.2, 10 mM rham- <br /> ance). Plasmid pSEVA132-adk derivative               nose 0.2% arabinose (wt/vol) added induce  <br /> pSEVA132 (ori pBBR1, ApR) encodes adk controlled         SceI nuclease production. reaching OD600 natural promoter fused C-terminal            0.4-0.5, cells chilled ice 30 min, harvested <br /> 6xHis-tag. amplified genomic E. coli DNA         washed twice chilled water 10% <br /> using primers adk-forward adk-reverse cloned         glycerol described [26]. transformation pSEVA132 restriction sites XmaI SacI. Plas-     test plasmids mock libraries, 50 &#206;&#188;L cells (OD600 <br /> mid pCOMP-groE constructed amplification           100) mixed 1.5 ng DNA, exposed Ptac-controlled groE-operon pACT-ESL using          electrical pulse 1.3 kV recovered 1 mL LB <br /> primers pACT-SceI-Spe pACT-Pac cloned           broth supplemented 10 mM rhamnose 0.2% ara- unique SpeI PacI sites pSEVA431. pACT-ESL         binose 1 h 37&#194;&#176;C. Selection overnight pACT3 derived encodes Ptac controlled groE        37&#194;&#176;C LB agar plates containing 50 &#206;&#188;g mL-1 <br />  Billerbeck Panke Microbial Cell Factories 2012, 11:110                                                                            Page 10 11 <br /> http://www.microbialcellfactories.com/content/11/1/110 <br />  <br />  <br />  <br />  <br /> kanamycin, 10 &#206;&#188;g mL-1 gentamicin, 100 &#206;&#188;g mL-1 ampi-                  Authors&#226;&#8364;&#8482; contributions <br /> cillin 10 mM rhamnose 0.2% arabinose.                     SB performed experiments, SB SP designed experimental <br />                                                                      approach wrote manuscript, SP supervised research. authors <br /> GroEL-specific selection based W3110 groE::                read approved final manuscript. <br /> kan [pCOMP-groE, pParaI-SceI] GpsA-specific <br /> selection based W3110 secBgpsA::kan                        Acknowledgment <br /> [pCOMP-secBgpsA, pParaI-SceI] treated                  work funded EU FP6 projects Eurobiosyn NANOMOT <br /> way. preparation competent SB&#206;&#8221;recA, cells                ESF project Nanocell. authors indebted Tania Roberts                                                                      proofreading manuscript. <br /> grown overnight LB supplemented 50 &#206;&#188;g mL-1 <br /> kanamycin 0.2% arabinose. Cells washed twice                Received: 13 June 2012 Accepted: 8 August 2012 water diluted 1:100 fresh LB medium.                Published: 16 August 2012 <br /> cells reached OD600 0.1 0.5% glucose added repress chromosomal GroEL production. Cellular GroEL                 References bleached generations cells har-              1. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, <br />                                                                          Tomita M, Wanner BL, Mori H: Construction Escherichia coli K-12 <br /> vested competent cell preparation  OD600 0.4).                 frame, single-gene knockout mutants: Keio collection. Mol Syst Biol <br />                                                                          2006, 2:2006&#226;&#8364;&#8220;0008. <br />                                                                      2. Rosamond J, Allsop  Harnessing power genome                                                                          search new antibiotics. Science 2000, 287:1973&#226;&#8364;&#8220;1976. <br /> Determination plasmid loss                                        3. Haselbeck R, Wall D, Jiang B, Ketela T, Zyskind J, Bussey H, Foulkes JG, <br /> Loss pKOCOMP-adk pCOMP-adk, pCOMP-groE                             Roemer T: Comprehensive essential gene identification platform pCOMP-secBgpsA determined follows:                        novel anti-infective drug discovery. Curr Pharm Des 2002, 8:1155&#226;&#8364;&#8220;1172. <br />                                                                      4. Johnson DBF, Xu JF, Shen ZX, Takimoto JK, Schultz MD, Schmitz RJ, Xiang  <br /> corresponding knock strains adk::kan, groE::kan                  Ecker JR, Briggs SP, Wang L: RF1 knockout allows ribosomal incorporation <br /> secBgpsA::kan containing helper plasmid pI-SceI                       unnatural amino acids multiple sites. Nat Chem Biol 2011, 7:779&#226;&#8364;&#8220;786. <br /> pParaI-SceI grown LB liquid broth supplemented               5. Gil R, Silva FJ, Pereto J, Moya  Determination core minimal <br />                                                                          bacterial gene set. Microbiol Mol Biol Rev 2004, 68:518&#226;&#8364;&#8220;537. 50 &#206;&#188;g mL-1 kanamycin 10 &#206;&#188;g mL-1 gentamicin                  6. Posfai G, Plunkett G 3rd, Feher T, Frisch D, Keil GM, Umenhoffer K, exponential growth reached.  SceI nu-                   Kolisnychenko V, Stahl B, Sharma SS, Arruda M, et al: Emergent properties <br /> clease production induced 10 mM rhamnose                        reduced-genome Escherichia coli. Science 2006, 312:1044&#226;&#8364;&#8220;1046. <br />                                                                      7. Trinh CT, Unrean P, Srienc F: Minimal Escherichia coli cell 0.2% arabinose. induction aliquots taken                   efficient production ethanol hexoses pentoses. Appl Environ 0, 10, 20, 40, 60, 120 180 min normalized                  Microbiol 2008, 74:3634&#226;&#8364;&#8220;3643. OD600. Serial dilutions subsequently plated               8. Lee JH, Sung BH, Kim MS, Blattner FR, Yoon BH, Kim JH, Kim SC: Metabolic <br />                                                                          engineering reduced-genome strain Escherichia coli LB agar supplemented 0.5% glucose. day                     <span id='am-10' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span>L-threonine</span> production. Microb Cell Fact 2009, 8:2. number colony forming units (CFU) counted.                9. Danchin  Natural selection immortality. Biogerontology 2009, 10:503&#226;&#8364;&#8220;516. number colonies resulting aliquots                 10. McLennan N, Masters M: GroE vital cell-wall synthesis. Nature 1998, <br />                                                                          392:139&#226;&#8364;&#8220;139. taken control culture grown ab-               11. Herring CD: Introduction conditional lethal amber mutations sence rhamnose arabinose set 100% CFU.                     Escherichia coli. Methods Mol Biol 2008, 416:323&#226;&#8364;&#8220;334. <br />                                                                      12. Jin Y, Watt RM, Danchin  Huang JD: Use riboswitch-controlled <br />                                                                          conditional hypomorphic mutation uncover role essential csrA <br />                                                                          gene bacterial autoaggregation. J Biol Chem 2009, 284:28738&#226;&#8364;&#8220;28745. <br /> Chromosomal knock essential genes                             13. Ji Y, Zhang B, Van SF, Horn, Warren P, Woodnutt G, Burnham MK, <br /> Knock-outs &#206;&#187; red-based recombination                        Rosenberg M: Identification critical staphylococcal genes using <br />                                                                          conditional phenotypes generated antisense RNA. Science 2001, PCR-encoded kanamycin resistance cassette                                                                          293:2266&#226;&#8364;&#8220;2269. <br /> described earlier [17]. kanamycin-cassettes                 14. Tang YC, Chang HC, Chakraborty K, Hartl FU, Hayer-Hartl M: Essential role generated pKD13 template primers adk-                      chaperonin folding compartment vivo. EMBO J 2008, 27:1458&#226;&#8364;&#8220;1468. <br />                                                                      15. Herring CD, Blattner FR: Conditional lethal amber mutations essential <br /> H1 adk-H2, groE-H1 groE-H2 secBgpsA-H1 <br />                                                                          Escherichia coli genes. J Bacteriol 2004, 186:2673&#226;&#8364;&#8220;2681. secBgpsA-H2. Knock-outs PCR-verified                   16. van der Vies SM, Lund PA: Determination chaperonin activity vivo. <br /> primers P1-P6 indicated Additional file 2 Figure S1                Methods Mol Biol 2000, 140:75&#226;&#8364;&#8220;96. <br />                                                                      17. Datsenko KA, Wanner BL: step inactivation chromosomal genes Additional file 1 Table S1. <br />                                                                          Escherichia coli K-12 using PCR products. Proc Natl Acad Sci U S 2000, <br />                                                                          97:6640&#226;&#8364;&#8220;6645. <br />                                                                      18. Tobin JF, Schleif RF: Transcription rha operon psr promoter. J Mol <br /> Additional files                                                         Biol 1990, 211:1&#226;&#8364;&#8220;4. <br />                                                                      19. Guzman LM, Belin D, Carson MJ, Beckwith J: Tight regulation, modulation, <br />                                                                          high-level expression vectors containing arabinose PBAD <br />  Additional files file 1: Table S1. Primers used study.          promoter. J Bacteriol 1995, 177:4121&#226;&#8364;&#8220;4130. <br />  Additional files file 2: Figure S1. PCR verification knock   20. Egan SM, Schleif RF: regulatory cascade induction rhaBAD. <br />  strains.                                                                J Mol Biol 1993, 234:87&#226;&#8364;&#8220;98. <br />                                                <br /> </body></html>