<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 27 November 2013                                              Nucleic Acids Research, 2014, Vol. 42,  4 2405&#226;&#8364;&#8220;2414 <br />                                                                                                                 doi:10.1093/nar/gkt1200 <br />  <br />  <br /> Genetic robustness functional evolution gene duplicates <br /> Germa&#194;&#180;n Plata1,2 Dennis Vitkup1,3,* <br /> 1 <br />  Department Systems Biology, Center Computational Biology Bioinformatics, Columbia University, <br /> New York City, NY 10032, USA, 2Integrated Program Cellular, Molecular, Structural, Genetic Studies, <br /> Columbia University, New York City, NY 10032, USA 3Department Biomedical Informatics, Columbia <br /> University, New York City, NY 10032, USA <br />  <br /> Received September 5, 2013; Revised November 3, 2013; Accepted November 4, 2013 <br />  <br />  <br />  <br /> ABSTRACT                                                                         duplicates (1,6). Gene duplications frequent evolu- <br />                                                                                  tion range size small-scale (SSD)  <br /> Gene duplications major source evolution-                               genome events (WGD) (7,8).  90% cases <br /> ary innovations. Understanding functional diver-                             duplicate eventually lost evolution (6), duplicated <br /> gence duplicates role genetic                                    genes remain genome  partially, <br /> robustness important challenge biology.                                 backup s functions. Importantly, functional <br /> Previously, analyses genetic robustness                                  compensation duplicates plays signi&#239;&#172; role buf- <br /> primarily focused duplicates essentiality                                 fering deleterious human mutations (9). <br /> epistasis laboratory conditions.                                 Genetic robustness gene duplicates inherently <br /> study, use quantitative data sets                                  tied functional divergence. Duplicates acquire <br /> understand compensatory interactions                                     distinct molecular functions (MFs) naturally unable Saccharomyces cerevisiae duplicates                                     compensate  addition, MF                                                                                  conserved, incomplete compensation duplicates likely relevant natural biological popula- <br />                                                                                  possible owing different expression patterns dosage <br /> tions.  owing high functional                              effects. Gene duplications major source new <br /> load, close duplicates unlikely provide sub-                              genes (10) conceptual models duplicates&#226;&#8364;&#8482; <br /> stantial backup context large natural popu-                            evolution proposed (11,12). neofunctio- <br /> lations. Interestingly, duplicates diverge                               nalization model duplicate gains new functions, e.  overall functional load reduced.                            functions associated ancestral gene, intermediate divergence distances quantita-                               duplicate retains ancestral functions <br /> tive decrease fitness removal dupli-                            (10,13,14). contrast, subfunctionalization model <br /> cate smaller. distances, yeast                                  duplicates indispensable retained duplicates display balanced functional loads                                evolution partitioning ancestral gene functions transcriptional control signifi-                               (15,16). models imply eventual loss                                                                                  ability duplicates fully substitute  cantly complex. yeast duplicates diverge <br />                                                                                  likely signi&#239;&#172; fraction duplicates &#239;&#172;?xed 70% sequence identity, ability                                   retained genomes owing selective advantages, <br /> compensate similar                                     dosage effects condition-speci&#239;&#172;?c expression random pairs singletons.                                              patterns, present moment duplication <br />                                                                                  (17,18). cases &#239;&#172;?xation selective advantage, <br />                                                                                  compensation duplicates unlikely. <br /> INTRODUCTION                                                                        compensation duplicates Survival biological systems crucially depends robust-                      expected long term, ability duplicates <br /> ness harmful genetic mutations, e. genetic robustness,                      buffer deleterious mutations paralogs changes environmental conditions (1&#226;&#8364;&#8220;3).                            demonstrated independent observa- <br /> distinct mechanisms genetic robustness                              tions. include lower expected fraction previously discussed.  alternative signaling meta-                     essential genes close duplicates (1), paucity bolic pathways provide important mechanism                                pairwise epistatic interactions involving duplicated genes <br /> rerouting molecular networks (4,5). Second,                            (19), excess aggravating genetic interactions <br /> major role genetic robustness attributed gene                           paralogs (20,21). contribution duplicates <br />  <br />  <br />  correspondence addressed. Tel: +1 212 851 5152; Fax: +1 212 851 5149; Email: dv2121@columbia.edu <br />  <br /> &#195;&#376; Author(s) 2013. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/3.0/), permits non-commercial use, distribution, reproduction medium, provided original work properly cited. commercial use, contact journals.permissions@oup.com <br />  2406 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  robustness primarily considered context      DeLuna et al. (34) Costanzo et al. (35). Gene qualitative quantitative growth phenotypes      essentiality data obtained study Giaever <br /> nutrient rich small number laboratory condi-         et al. (36). <br /> tions (1,22,23). popular experiments,            functionally characterize duplicated genes, Gene <br /> conditions unlikely approximate natural           Ontology   (37) annotations collected &#226;&#8364;&#732;milieu&#226;&#8364;&#8482; living systems, constantly bombarded      SGD Enzyme Commission (EC) annotations diverse array environmental stresses stimuli.       Comprehensive Yeast Genome Database (CYGD) importantly, strong com-        (38). Transcription factor binding motifs used pensatory interaction pair duplicates, evo-     work compiled Kafri et al. (39) high- <br /> lutionary relevant decrease &#239;&#172;?tness persist     con&#239;&#172;?dence predictions Kellis et al. (28). used incomplete buffering damaging mutation         protein localization data Huh et al. (40), Codon duplicates (24). context long-term         Adaptation Index (CAI) calculations based data <br /> evolution, difference mu-         set Lu et al. (41) annotation protein <br /> tations leading lethal phenotype mutations           complexes CYGD. <br /> associated &#239;&#172;?tness decrease substantially larger inverse effective population size (25,26). <br /> Given typical population sizes free-living microbial    RESULTS <br /> species large (&gt;106&#226;&#8364;&#8220;108) (27), small &#239;&#172;?tness <br />                                                                 Hillenmeyer et al. (33) quanti&#239;&#172;?ed growth phenotypes decrease effectively lethal organisms. <br />                                                                 single-gene yeast deletion strains large collection Consequently, quantitative analyses growth pheno- <br />                                                                 environmental conditions. assembled data set <br /> types, preferably multiple environmental conditions, <br />                                                                 contains  5.5 million phenotypes heterozygous necessary understand extent compen- <br />                                                                 homozygous mutants  400 conditions. sampled <br /> sation duplicates plays important role                                                                 conditions represent 27 different environmental stresses <br /> natural biological populations. perform                                                                 hundreds perturbations diverse <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> com- <br /> analysis context natural popula- <br />                                                                 pounds. Environmental stresses comprised different <br /> tions, genetic buffering mediated duplicates likely                                                                 growth media, media lacking speci&#239;&#172;?c vitamins amino rare  surprisingly, monotonic function                                                                 acids, different pH temperature regimes. <br /> duplicates&#226;&#8364;&#8482; divergence. <br />                                                                 comprehensive collection phenotypes allowed                                                                 investigate diversi&#239;&#172;?cation duplicates&#226;&#8364;&#8482; <br />                                                                 functions contribution genetic robustness MATERIALS METHODS                                           multiple conditions. <br /> Gene protein sequences Saccharomyces cerevisiae,           &#239;&#172;?rst investigated average number sensi- <br /> Saccharomyces paradoxus, Saccharomyces bayanus,                 tive conditions, e. conditions signi&#239;&#172; growth <br /> Saccharomyces      castelli,    Saccharomyces       mikatae,    decrease deletion duplicate, depends Saccharomyces kudriavzevii Saccharomyces kluyveri           sequence divergence (Ka) duplicated genes obtained Saccharomyces Genome                     (Figure 1A B). considered fraction different <br /> Database (SGD; http://downloads.yeastgenome.org/)               conditions growth phenotype quantitative study Kellis et al. (28). Pairs gene duplicates   measure compensation capacity duplicates identi&#239;&#172;?ed sequence homology proteins            various divergence distances. close duplicates genome using BLASTP (29). duplicates           average number sensitive conditions signi&#239;&#172;?cantly bidirectional best hits aligned       different random pair yeast singletons <br /> &gt;80% open reading frame&#226;&#8364;&#8482;s sequence length          (Figure 1B, horizontal line). Importantly, result does <br /> considered analysis (30). Following previous studies     imply random gene pairs close duplicates (1), excluded ribosomal genes analysis owing        equivalent terms similarity MF. high expression, dominant impact growth         demonstrate  observed pattern likely strong codon adaptation bias. Evolutionary distances            higher overall functional load close duplicates. duplicated genes estimated using               article use term &#226;&#8364;&#732;functional <br /> method Yang Nielsen (31) implemented              load&#226;&#8364;&#8482; gene characterize average &#239;&#172;?tness  <br /> PAML package (32); use methods,            crease considered conditions gene <br /> maximum likelihood, estimate Ka Ks did sig-          deletion; note  based de&#239;&#172;?nition  ni&#239;&#172;?cantly change observed patterns (Supplementary           functional load measure total number Figure S1A).                                                    MFs gene  re&#239;&#172;&#8218;ects gene&#226;&#8364;&#8482;s overall &#239;&#172;?tness <br />    used data obtained Hillenmeyer et al. (33)      contribution. <br /> measure &#239;&#172;?tness contribution duplicates               Interestingly, number sensitive conditions initially <br /> multiple environmental conditions chemical perturb-         drops duplicates diverge, decreasing 30% ations. Using P-value cutoff 0.01, obtained         distances corresponding Ka <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; 0.1 (Ks &amp; 1, number experimental conditions growth            Supplementary Figure S2A B). duplicates <br /> defect observed single gene deletion              diverge  average number sensitive conditions <br /> mutant. analyzed quantitative growth measure-           increases  reaching average random pair <br /> ments double single deletion yeast strains obtained     yeast singletons Ka &amp; 0.25. trend shown                                                                                                                                                              Nucleic Acids Research, 2014, Vol. 42,  4 2407 <br />  <br />  <br />       Fraction conditions significant     0.30                                                        B <br />                                                                                                                                                           0.12 <br />  <br />  <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />                                                  0.25 <br />                                                                                                                                                           0.11 <br />                                                  0.20 <br />               growth decrease <br />  <br />  <br />  <br />  <br />                                                                                                                          growth decrease <br />                                                                                                                                                           0.10 <br />                                                  0.15 <br />                                                                                                                                                           0.09 <br />                                                  0.10 <br />                                                                                                                                                           0.08 <br />  <br />                                                  0.05                                                                                                     0.07 <br />                                                                                                                                                                   p=0.01 <br />                                                  0.00                                                                                                     0.06 <br />  <br />                                                          0.0   0.1   0.2     0.3    0.4   0.5    0.6                                                             0.0   0.1   0.2    0.3   0.4    0.5   0.6 <br />                                                                              Ka                                                                                                     Ka <br />  <br />  <br />           C 0.25                                                                                             D                                                                                         SSD <br />                                                                                                                                                          0.14 <br />                                                                                                                                                                                                        WGD <br />  <br />  <br />                                                                                                              Fraction conditions significant <br />             Fraction essential duplicates <br />  <br />  <br />  <br />  <br />                                                0.20 <br />                                                                                                                                                          0.12 <br />  <br />                                                0.15                                                                      growth decrease <br />                                                                                                                                                          0.10 <br />  <br />                                                0.10 <br />                                                                                                                                                          0.08 <br />                                                0.05 <br />                                                                                                                                                          0.06 <br />                                                0.00 <br />                                                                                                                                                          0.04 <br />                                                         0.0    0.2     0.4         0.6     0.8         1.0                                                       0.0   0.1   0. 2   0.3   0. 4   0.5   0.6 <br />                                                                            Ka <br />                                                                                                                                                                                     Ka <br /> Figure 1. Compensation patterns yeast duplicates function evolutionary divergence, Ka, number nonsynonymous <br /> substitutions site.   Scatterplot fraction sensitive conditions, e. conditions detectable growth phenotypes resulting duplicate gene deletion, versus Ka. dot &#239;&#172;?gure represents pair yeast duplicates. (B) average fraction sensitive conditions duplicate pair. P-value calculated using Mann&#226;&#8364;&#8220;Whitney U test. horizontal lines B indicate average fraction sensitive conditions random pair yeast singletons. (C) average fraction essential duplicates, e. duplicates lethal phenotype deletion, function Ka. horizontal line indicates fraction essential yeast singletons. Gene essentiality data obtained Saccharomyces Genome Deletion Project (36). (D) Fraction conditions signi&#239;&#172; growth decrease deletion yeast duplicates arising small-scale (SSD) genome duplications (WGD). duplicates classi&#239;&#172;?ed SSD WGD based study Kellis et al. <br /> (8) horizontal line shows average fraction sensitive conditions random pair yeast singletons. &#239;&#172;?gures, error bars represent standard error mean (SEM). <br />  <br />  <br />  <br /> Figure 1B sensitive P-value cutoff used                                                     compensation heterozygous homozygous <br /> determine signi&#239;&#172;?cance growth decrease observed                                                    mutants similar multiple-drug resistance genes, mutant strains (Supplementary Figure S3). similar                                                       de&#239;&#172;?ned Hillenmeyer et al. (33), considered <br /> trend observed average growth decrease                                                      (Supplementary Figure S4C). trends remain <br /> (functional load), measured log ratios                                                          similar environmental perturbations scores tested conditions (Supplementary                                                         analyzed homozygous experiments (Supplementary <br /> Figure S4A B). Bin-free analyses data                                                             Figure S4D). checked observed compen- <br /> (Supplementary Figures S1B C S2B)                                                               sation patterns closest duplicates signi&#239;&#172;?- <br /> revealed smaller &#239;&#172;?tness cost loss duplicates                                                 cantly in&#239;&#172;&#8218;uenced additional, e. diverged, intermediate distances (Ka &amp; 0.1).                                                                        paralogs (Supplementary Figure S4E). lack signi&#239;&#172;?- <br />    actively growing wild-type yeast popula-                                                     compensation diverged duplicates results tions diploid (42), mainly focused analysis                                                    approximately linear relationship number heterozygous mutant strains. patterns functional                                                      sensitive conditions yeast protein family  2408 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br /> family size (Supplementary Figure S5). Finally,              pairs decreases function sequence divergence <br /> observed compensation patterns affected              (Figure 2E F)  (21)]. <br /> removal gene pairs high CAI (Supplementary                smaller overall functional load, possible <br /> Figure S6A), suggesting observed trend           duplicates intermediate distances explained expression-based constraints rate         properties favor genetic robustness. explore duplicate sequence evolution (Ka) (43) high expres-        possibility, duplicate pair, looked gene <br /> sion levels certain duplicates.                               largest gene smallest number    interesting compare ability duplicates     sensitive conditions (Figure 3A). Notably, dupli- <br /> buffer mutations leading detectable growth                cate conditions (Figure 3A, sensitive <br /> decrease given &#239;&#172;?tness threshold (Figure 1B)             duplicate) follows average trend duplicates role protecting growth pheno-        (Figure 1B), duplicate fewer conditions (Figure <br /> type, e. likelihood observe essential genes dupli-   3A, sensitive duplicate) shows steady gain cate pairs. Figure 1C, using data study           number conditions function Ka. Consequently, <br /> Giaever et al. (36), fraction essential du-       functional load close duplicates, measured plicates function divergence. agreement         number sensitive conditions, different, differ- previous studies (1,22,23)                ence signi&#239;&#172;?cantly smaller genes diverge <br /> fraction essential genes remains low approximately        (Figure 3B. Pearson&#226;&#8364;&#8482;s r = 0.64, P = 7   10 4, constant close duplicates, increases substantially       Supplementary Figure S2C). Close duplicates divergence distances corresponding Ka &gt; 0.4.          larger number sensitive conditions higher <br /> Notably, pattern qualitatively different        evolutionary constraint, evaluated normalized ratio <br /> compensation quantitative growth phenotypes                  nonsynonymous synonymous substitutions nucleo- <br /> (Figure 1B), demonstrating aforementioned impact             tide site, Ka/Ks (Wilcoxon Signed Rank test P = 7   10 3, using quantitative phenotypes assess evolution-        Figure 2C). result agrees previous reports arily relevant consequences mutations. contrast       asymmetric evolution duplicates context ex- patterns obtained studies based essential genes         pression, genetic interaction protein&#226;&#8364;&#8220;protein interaction <br /> (22), observed similar compensation pro&#239;&#172;?les gene          networks (19,44,45). observed asymmetry func- <br /> pairs originating small-scale genome-wide dupli-        tional load close duplicates make buffering <br /> cations (Figure 1D). WGD duplicates         dif&#239;&#172;?cult. example, sensitive duplicate                                                                  expressed speci&#239;&#172;?c environmental conditions. age, result suggests ability duplicates <br />                                                                     explore mechanism observed buffer s function multiple conditions <br />                                                                  backup patterns, analyzed functional diversi&#239;&#172;?cation <br /> depends strongly sequence divergence                                                                  yeast duplicates function sequence diver- time duplication. <br />                                                                  gence (Ka).  genes encoding metabolic enzymes <br />    likely observed decrease number                                                                  calculated fraction gene pairs conserved EC <br /> sensitive conditions intermediate divergence distances <br />                                                                  numbers (Figure 4A); conservation EC numbers <br /> (Ka &amp; 0.1) decrease functional load           indicates corresponding proteins catalyze identical <br /> carried distances union duplicate             biochemical reactions. Second, calculated fraction <br /> genes. explore possibility, considered quan-      shared terms describing protein MF dupli- <br /> titative &#239;&#172;?tness data DeLuna et al. (34) syn-        cates (Figure 4B). measures showed MF thetic genetic array (SGA) data Costanzo et al. (35).       yeast duplicates typically starts substantially diverge studies, authors performed quantitative             Ka &gt; 0.4. timing divergence ap- <br /> growth measurements yeast strains individual             proximately coincides signi&#239;&#172; increase simultaneous deletions duplicates. Using              fraction essential duplicates (Figure 1C). single deletion phenotypes DeLuna et al.                hand, signi&#239;&#172; changes number quantitative <br /> (Figure 2A) Costanzo et al. studies (Figure 2B),          growth phenotypes observed MF dupli- <br /> observed &#239;&#172;?tness pro&#239;&#172;?les similar obtained based        cates usually conserved. data Hillenmeyer et al. (Figure 1B)                complementary analysis transcription factor <br /> function Ka, smaller phenotypic effects inter-        binding sites suggests gene regulation plays import- <br /> mediate distances. Interestingly, overall functional         ant role establishing observed compensation <br /> load duplicate pairs, measured phenotype            patterns. previously demonstrated duplicated <br /> double deletions, substantially decreases            yeast genes  average, higher number cis-regu- divergence (Figure 2C D). result suggests         latory motifs singleton genes (46). Using compre- close duplicates likely similar      hensive data set  150 known predicted DNA <br /> functions, higher functional load makes complete           binding motifs yeast (28,39), average <br /> compensation likely. overall functional         number different motifs regulating duplicate pair  <br /> load duplicates remains approximately constant            creases signi&#239;&#172;?cantly Ka &amp; 0.1 (Figure 4D, dashed line, <br /> Ka &gt; 0.15, higher fraction detectable growth pheno-       Mann&#226;&#8364;&#8220;Whitney U test, P = 0.06). divergence <br /> types distances likely decreased ability    distance, average number different motifs dupli- functional compensation duplicates diverge.               cate pair twice number motifs pair <br /> Compensation duplicates quanti&#239;&#172;?ed                 yeast singletons (Figure 4D, dashed horizontal line). <br /> presence aggravating interactions duplicate           number regulatory motifs increases                                                                                                                        Nucleic Acids Research, 2014, Vol. 42,  4 2409 <br />  <br />  <br />                                                0.7 <br />                                                                 p=0.01 <br />                                                                                  B                                  0.20 <br />                                                                                                                                       p=0.08 <br />  <br />  <br />               Fraction single deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction single deletions <br />                                                  0.6 <br />  <br />  <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                                                                                     0.15 <br />                                                  0.5 <br />  <br />                                                  0.4 <br />                                                                                                                     0.10 <br />                                                  0.3 <br />  <br />                                                  0.2 <br />                                                                                                                     0.05 <br />                                                  0.1 <br />  <br />                                                  0.0                                                                0.00 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               C                                                                  D                                   1.1 <br />                                                  1.0 <br />               Fraction double deletions <br />  <br />  <br />  <br />  <br />                                                                                  Fraction double deletions <br />                                                                                                                      1.0 <br />                growth phenotype <br />  <br />  <br />  <br />  <br />                                                                                   growth phenotype <br />                                                  0.8                                                                 0.9 <br />  <br />                                                                                                                      0.8 <br />                                                  0.6 <br />                                                                                                                      0.7 <br />                                                  0.4                                                                 0.6 <br />  <br />                                                  0.2 <br />                                                                                                                      0.5 <br />  <br />                                                                                                                      0.4 <br />                                                     0.0   0.1        0.2   0.3                                          0.0     0.1            0.2   0.3 <br />                                                                     Ka                                                                     Ka <br />               E                                                                  F <br />                                                                                  Fraction buffering duplicates <br />               Fraction buffering duplicates <br />  <br />  <br />  <br />  <br />                                                  1.0                                                                1.0 <br />  <br />                                                  0.8                                                                0.8 <br />  <br />                                                  0.6 <br />                                                                                                                     0.6 <br />  <br />                                                  0.4 <br />                                                                                                                     0.4 <br />                                                  0.2 <br />                                                                                                                     0.2 <br />                                                  0.0 <br />                                                     0.0   0.1        0.2   0.3                                        0.0      0.1         0.2       0.3 <br />                                                                     Ka                                                                    Ka <br /> Figure 2. Growth phenotypes individual simultaneous deletion duplicates function sequence divergence (Ka). results &#239;&#172;?rst column   C, E) based competition experiments DeLuna et al. (34), second column (B, D, F) synthetic <br /> genetic arrays (SGA) Costanzo et al. (35).   B) Fractions single duplicate deletions signi&#239;&#172; growth decrease. (C, D) Fractions simultaneous (double) duplicate deletions signi&#239;&#172; growth decrease. different measurement sensitivities studies, different <br /> cutoffs used determine signi&#239;&#172; growth decrease: 1% DeLuna et al.   C) 10% Costanzo et al. (B, D); presented results sensitive exact cutoff values  Supplementary Figure S7). P-values obtained using Fisher&#226;&#8364;&#8482;s exact test. (E, F) Fraction paralogs signi&#239;&#172; negative epistatic interaction studies DeLuna et al. Costanzo et al., respectively. &#239;&#172;?gures error bars <br /> represent SEM. <br />  <br />  <br />  <br /> duplicate highest duplicate                            shared motifs [Supplementary Figure S9A, (48)], <br /> smallest number sensitive conditions (Supplementary                           overlap terms describing biological processes <br /> Figure S8A B). increase complexity du-                         (Figure 4C) overlap cellular localization <br /> plicates regulation Ka &amp; 0.1 con&#239;&#172;?rmed sig-                       observed &#239;&#172;&#8218;uorescence-tagging experiments (40) <br /> ni&#239;&#172; increase (Mann&#226;&#8364;&#8220;Whitney U test, P = 1   10 3)                          decrease (Supplementary Figure S8B). pattern distances number transcription factor                            suggests increase regulatory complexity <br /> mutants (47) affecting duplicate gene expression (Figure                         allows duplicates specialize different biological <br /> 4D, solid line).                                                                 processes preserving common MFs.    total number DNA motifs regulating du-                           ability duplicates partially diverged regulatory <br /> plicates initially increases divergence, fraction                    regions compensate expression <br />  2410 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br />                                                        0.10                                                                       B <br />                                                                                               sensitive duplicate <br />              Fraction conditions significant <br />  <br />  <br />  <br />  <br />                                                                                                                                     Relative difference number                                                                                               sensitive duplicate                                                     1.0 <br />                                                          0.09 <br />  <br />                                                          0.08                                                                                                              0.8 <br />  <br />  <br />  <br />  <br />                                                                                                                                             sensitive conditions <br />                          growth decrease <br />  <br />  <br />                                                          0.07 <br />                                                                                                                                                                            0.6 <br />                                                          0.06 <br />                                                                                                                                                                            0.4 <br />                                                          0.05 <br />  <br />                                                          0.04                            p=2E-4                                                                            0.2 <br />  <br />                                                          0.03 <br />                                                                                                                                                                            0.0 <br />                                                          0.02 <br />                                                                      0.0            0.1        0.2          0.3   0.4   0.5   0.6                                                0.00   0.02   0.04   0.06   0.08   0.10 <br />                                                                                                       Ka                                                                                          Ka <br />              C                                                                      p=0.007 <br />                                                          0.07 <br />  <br />                                                          0.06 <br />  <br />                                                          0.05 <br />  <br />                                                          0.04 <br />                                              Ka/Ks <br />  <br />  <br />  <br />  <br />                                                          0.03 <br />  <br />                                                          0.02 <br />  <br />                                                          0.01 <br />  <br />                                                          0.00 <br />                                                                                te                      te <br />                                                                        plica                    pli <br />                                                                                                      ca <br />                                                                 e   du                    e   du <br />                                                            itiv                       itiv <br />                                                     e    ns                    e    ns <br />                                            r     es                        sss <br />                    Mo                                                Le <br /> Figure 3. Differences number sensitive conditions duplicates.   average fraction sensitive conditions duplicates higher lower number sensitive conditions pair; Ka values represent sequence divergence duplicates. P-value Mann&#226;&#8364;&#8220;Whitney U test. (B) relative difference number sensitive conditions duplicates function initial <br /> divergence; Ka values represent sequence divergence duplicates. relative difference calculated absolute difference number sensitive conditions duplicates normalized total number sensitive conditions pair (Spearman&#226;&#8364;&#8482;s r = 0.60, <br /> P = 2   10 3; Pearson&#226;&#8364;&#8482;s r = 0.64, P = 7   10 4). (C) average Ka/Ks ratio paralogs largest  sensitive) smallest <br />  sensitive) number conditions signi&#239;&#172; growth decrease. Ka/Ks ratios calculated relative orthologous sequences S. bayanus. duplicates Ka &lt; 0.15 considered. P-value Wilcoxon signed rank test. <br />  <br />  <br />  <br />  <br /> changes intact gene previously described                                                                             Finally, patterns diversi&#239;&#172;?cation functional <br /> Kafri et al. (39,49).  recent study DeLuna                                                                          compensation described correlate et al. (50) showed deletion duplicate, ex-                                                                       process duplicate loss evolution. investigated pression changes remaining paralog need-                                                                       retention yeast duplicates using complete genomic <br /> based, e. happen primarily corresponding                                                                        sequences seven species: S. cerevisiae, S. paradoxus, <br /> function required. regulatory backup circuits                                                                           S. bayanus, S. castelli, S. mikatae, S. kudriavzevii  cases, enable functional compen-                                                                       S. kluyveri. calculated number remaining dupli- <br /> sation homologs different expression                                                                               cates function sequence divergence (Figure 5, <br /> patterns wild type. Notably, based data                                                                          Supplementary Figure S10A correspond- <br /> recent study Springer et al. (51), measured                                                                          ing relationships individual yeast species). analysis <br /> expression changes yeast genes                                                                               suggests relatively brief initial period high dupli- <br /> genomic copies deleted diploid cells, observed                                                                        cate loss (6) followed long evolutionary period signi&#239;&#172; dosage response genes forming                                                                             (Ka &gt; 0.1) average loss rate decreases <br /> recently duplicated pairs (Ka &lt; 0.15, Figure 4E).                                                                          &gt;10-fold (red Figure 5). Interestingly, loss rate sig- <br /> suggests genes close duplicates respon-                                                                      ni&#239;&#172;?cantly decreases approximately divergence <br /> sive dosage effects.                                                                                                         distance duplicates similar terms <br />                                                                                                                                                                                        Nucleic Acids Research, 2014, Vol. 42,  4 2411 <br />  <br />  <br />                                                                                                                                       B 1.0 <br />                    1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                  Fraction shared molecular functions <br />                  Fraction pairs sharing EC numbers <br />                                                                0.8                                                                                                           0.9 <br />  <br />  <br />                                                                0.6                                                                                                           0.8 <br />  <br />                                                                0.4 <br />                                                                                                                                                                              0.7 <br />  <br />                                                                0.2 <br />                                                                                                                                                                              0.6 <br />  <br />                                                                0.0 <br />                                                                                                                                                                              0.5 <br />                                                                  0.0         0.2    0.4         0.6         0.8    1.0                                                          0.0       0.2         0.4          0.6         0.8        1.0 <br />                    C 0.8                                                                  Ka                                                                                                                Ka <br />                   Fraction shared biological processes <br />  <br />  <br />  <br />  <br />                                                                                                                          D                                                                 TF binding motifs <br />                                                                0.7                                                                                                                         TF mutations affecting expression <br />                                                                                                                                                                                        p=0.06 <br />  <br />                                                                0.6                                                                                                                    p=0.001 <br />  <br />                                                                                                                                                                               10 <br />                                                                0.5 <br />                                                                                                                          Number unique TF associations <br />  <br />  <br />  <br />                                                                                                                                                                                9 <br />  <br />                                                                0.4                                                                                                             8 <br />                                                                                                                                 duplicate pair <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                7 <br />                                                                0.3 <br />                                                                                                                                                                                6 <br />                                                                  0.0         0.2    0.4        0.6          0.8    1.0 <br />                                                                                                                                                                                5 <br />                                                                                           Ka <br />                                                                                                                                                                                4 <br />             E                                                                                          p=0.003 <br />                                                                0.25 <br />                                                                                                                                                                                3 <br />            Expression responsiveness <br />  <br />  <br />  <br />  <br />                                                                0.20                                                                                                            2 <br />                                                                                                                                                                                 0.0             0.1         0.2          0.3         &gt;   0.4 <br />                                                                0.15                                                                                                                                           Ka <br />  <br />  <br />                                                                0.10 <br />  <br />                                                                0.05 <br />  <br />                                                                0.00 <br />  <br />                                                            -0.05 <br />                                                                        0.0    0.1   0.2    0.3        0.4    0.5   0.6 <br />                                                                                            Ka <br /> Figure 4. Diversi&#239;&#172;?cation duplicates function regulation.   Fraction metabolic duplicates sharing EC numbers; conservation EC numbers indicates catalysis identical biochemical reactions. (B) Fraction MF terms shared duplicates. (C) Fraction Biological Process (BP) terms shared duplicates. panels B C considered terms distance corresponding root hierarchy term. (D) Dashed line, average number different transcription factor binding motifs duplicate pair. <br /> Transcription factor (TF) binding motifs compiled studies Kafri et al. (39) Kellis et al. (28). Solid line, average number transcription factor deletions S. cerevisiae signi&#239;&#172;?cantly affect expression duplicate genes. data obtained study Hu <br /> et al. (47). comparison average number motifs TF mutants affecting expression random pairs yeast singletons <br /> (horizontal dashed solid lines); P-values calculated using Mann&#226;&#8364;&#8220;Whitney U test. (E) average dosage compensation (respon- <br /> siveness) duplicates function sequence divergence (Ka). data average expression responsiveness obtained work Springer et al. (51). study, responsiveness measured diploid yeast strains Log2 ratio (perturbed versus normal) expression <br /> changes remaining gene copy following deletion equivalent gene copy sister chromosome. P-value calculated using Mann&#226;&#8364;&#8220;Whitney U test. &#239;&#172;?gures error bars represent SEM. <br />  2412 Nucleic Acids Research, 2014, Vol. 42,  4 <br />  <br />  <br />                                        100 <br />                                                                        Ka &lt; 0.1         et al. (35), duplicates annotated redundant                                         80 <br />                                                                                         signi&#239;&#172;?cantly different yeast duplicates <br />       Number remaining duplicates <br />  <br />                                                                        Ka &gt; 0.1 <br />        averaged 7 yeast species <br />  <br />                                        60                                               (Mann&#226;&#8364;&#8220;Whitney U, P = 0.13 0.35, respectively, <br />                                                                                         Supplementary Figure S11). demonstrates  <br />                                        40 <br />                                                                                         yeast duplicates func- <br />                                                                                         tional overlap laboratory conditions, com- <br />                                                                                         pensation properties probably signi&#239;&#172;?cantly                                        20 <br />                                                                                         important large natural populations ability <br />                                                                                         purifying selection ef&#239;&#172;?ciently prune mutations <br />                                                                                         causing small &#239;&#172;?tness decrease. <br />                                        10 <br />                                                                                            likely different factors contribute                                         8 <br />                                                                                         relative paucity functional compensation                                         6                                               paralogs small divergence distances. signi&#239;&#172;                                          0.0   0.1   0.2   0.3   0.4     0.5      0.6   fraction duplications likely &#239;&#172;?xed owing                                                                                         dosage effects (17), functional compensation <br />                                                            Ka <br />                                                                                         duplicates context natural popula- <br /> Figure 5. average number duplicates retained genomes                   tions unlikely. example, lack signi&#239;&#172; com- <br /> yeast species function duplicates divergence Ka, number                 pensation histone pairs, HTA1-HTA2 nonsynonymous substitutions site. number remaining <br /> duplicates averaged genomes seven yeast species: S.                     HHT1-HHT2, likely consequence role <br /> cerevisiae, S. paradoxus, S. bayanus, S. castelli, S. mikatae, S.                       maintaining proper histone levels yeast cells. Gene <br /> kudriavzevii S. kluyveri. Supplementary Figure S10                     dosage explain inability duplicates number remaining duplicates individual species                    backup  unlikely explan- number remaining duplicates function Ks. rate duplicate loss evolution &gt;10 times lower distances <br />                                                                                         ation. showed duplicate pairs corresponding Ka &gt; 0.1 compared distances Ka &lt; 0.1.                      high CAI (Supplementary Figure S6A) pairs forming &#239;&#172;?gure, error bars represent SEM.                                             known protein complexes (Supplementary Figure S6B)                                                                                         removed analysis, patterns functional com- <br />                                                                                         pensation remain similar. Notably, genes high CAI functional load (Figure 2B) regu-                               associated higher frequencies latory complexity signi&#239;&#172;?cantly increases (Figure 5D).                             interlocus gene conversion (IGC) (53,54). IGC <br /> likely duplicates surviving initial loss stage                             slow rate duplicates sequence divergence <br /> develop independent functionalities preserved                                   (55), analyses based WGDs evidence long times genomes yeast species.                                         IGC [using data recently reported Casola et al. (56)] <br />                                                                                         revealed essentially identical compensation patterns <br />                                                                                         (Supplementary Figure S12). <br /> DISCUSSION                                                                                 Close duplicates likely compensate                                                                                         probably owing aforementioned dichot- present study, analyzed genetic robustness                                omy functional loads (Figure 3A B). duplicates context quantitative growth pheno-                              close duplicates classi&#239;&#172;?ed, based activity <br /> types sensitivities gene deletions multiple envir-                            breadth expression, major minor func- <br /> onmental conditions. robustness important                                   tional isoforms. example, glyceraldehyde-3-phos- <br /> understanding buffering deleterious mutations                                 phate dehydrogenase TDH1 active various stress <br /> large natural biological populations. results demon-                                conditions, isoenzyme TDH2 used primarily <br /> strate  contrary commonly held view, close gene                                 exponential growth (57). Similarly, ubiquitin <br /> duplicates unlikely provide high level backup                            conjugating enzyme UBC4 expressed exponen- context large natural populations. Consequently,                              tial growth, duplicate UBC5 active sta- unlikely duplicates &#239;&#172;?xed natural popu-                              tionary phase (58). difference functional load <br /> lations speci&#239;&#172;?cally selection robustness.                                    close yeast duplicates consistent    analysis suggests duplicate redundancies                               asymmetric partition functions, interactions gene <br /> described genomics databases, frequently observed                                expression, observed close duplicates  laboratory experiments, considered                                    ganisms, example, Arabidopsis Human (45,59,60). <br /> caution, respect functions                                    suggests duplicate-dependent compensation natural biological populations. investigate point                               context natural populations limited  analyzed set, compiled Kafri et al. (52),                              species  112 yeast duplicates reported partially                                  analysis suggests typical lifecycle gene <br /> redundant research publications. duplicates                                    duplicates yeast consists distinct evolutionary described redundant based func-                                   stages (11,12). &#239;&#172;?rst stage  duplicate distances cor- <br /> tional overlap compensatory interactions observed                                   responding Ka &lt; 0.05), duplicates tend high small-scale experimental studies. Interestingly, based                               overall functional loads signi&#239;&#172; asymmetry number conditions quantitative growth                                    number sensitive conditions; factors make <br /> phenotypes study Hillenmeyer et al. (33),                                   complete compensation unlikely. high functional load quantitative growth measurements Costanzo                                    close duplicates suggests adaptive selection plays <br />                                                                                   Nucleic Acids Research, 2014, Vol. 42,  4 2413 <br />  <br />  important role &#239;&#172;?xation. second stage                 10. Ohno,S. (1970) Evolution Gene Duplication. Springer-Verlag, <br /> (0.05 &lt; Ka &lt; 0.25), duplicates diverge                     Berlin, New York. <br />                                                                         11. Conant,G.C. Wolfe,K.H. (2008) Turning hobby job: <br /> overall functional load usually decreases.                         duplicated genes &#239;&#172;?nd new functions. Nat. Rev. Genet., 9, <br /> happen, example, relaxation environmen-                   938&#226;&#8364;&#8220;950. <br /> tal conditions, facilitated original duplicate                12. Innan,H. Kondrashov,F. (2010) evolution gene <br /> &#239;&#172;?xation. vast majority duplicates, likely                        duplications: classifying distinguishing models. Nat. <br />                                                                             Rev. Genet., 11, 97&#226;&#8364;&#8220;108. <br /> paralogs relatively smaller functional loads                       13. Nadeau,J.H. Sankoff,D. (1997) Comparable rates gene <br /> (Figure 3C), lost stage (Figure 5). Gene pairs                  loss functional divergence genome duplications early survive period high duplicate loss display                 vertebrate evolution. Genetics, 147, 1259&#226;&#8364;&#8220;1266. <br /> balanced functional loads complex regulation;                 14. Sidow  (1996) Genome duplications evolution early <br /> gene pairs usually retained long evolutionary times                 vertebrates. Curr. Opin. Genet. Dev., 6, 715&#226;&#8364;&#8220;722. <br />                                                                         15. Force , Lynch,M., Pickett,F.B., Amores , Yan,Y.L. yeast genomes (Figure 5). Surviving duplicates                       Postlethwait,J. (1999) Preservation duplicate genes provide partial compensation intermediate di-                   complementary, degenerative mutations. Genetics, 151, 1531&#226;&#8364;&#8220;1545. <br /> vergence distances serve important source                16. Lynch,M. Force  (2000) probability duplicate gene new protein functions. stage (Ka &gt; 0.3                   preservation subfunctionalization. Genetics, 154, 459&#226;&#8364;&#8220;473. <br />                                                                         17. Kondrashov,F , Rogozin B., Wolf,Y  Koonin,E.V. <br />  70% sequence identity), lifecycle duplicates                     (2002) Selection evolution gene duplications. Genome <br /> completed functional roles diverge,                    Biol., 3, RESEARCH0008. <br /> quantitative compensation properties indistin-                   18. Bergthorsson,U., Andersson,D  Roth,J.R. (2007) Ohno&#226;&#8364;&#8482;s <br /> guishable random pairs yeast singletons.                   dilemma: evolution new genes continuous selection. <br />                                                                             Proc. Natl Acad. Sci. USA, 104, 17004&#226;&#8364;&#8220;17009. <br />                                                                         19. VanderSluis,B., Bellay,J., Musso,G., Costanzo,M., Papp,B., <br />                                                                             Vizeacoumar,F.J., Baryshnikova , Andrews,B., Boone,C. SUPPLEMENTARY DATA                                                          Myers,C.L. (2010) Genetic interactions reveal evolutionary <br /> Supplementary Data available NAR Online.                             trajectories duplicate genes. Mol. Syst. Biol., 6, 429. <br />                                                                         20. Ihmels,J., Collins,S.R., Schuldiner,M., Krogan,N.J.                                                                             Weissman,J.S. (2007) Backup redundancy: genetic <br />                                                                             interactions reveal cost duplicate gene loss. Mol. Syst. <br /> ACKNOWLEDGEMENTS                                                            Biol., 3, 86. <br />                                                                         21. Li,J., Yuan  Zhang  (2010) cellular robustness thank Fyodor Kondrashov (Centre Genomic                              genetic redundancy budding yeast. PLoS Genet., 6, <br /> Regulation) members Vitup laboratory                         e1001187. <br /> illuminating discussions regarding manuscript.                      22. Guan,Y., Dunham,M.J. Troyanskaya,O.G. (2007) Functional <br />                                                                             analysis gene duplications Saccharomyces cerevisiae. <br />                                                                             Genetics, 175, 933&#226;&#8364;&#8220;943. <br /> FUNDING                                                                 23. Conant,G.C. Wagner  (2004) Duplicate genes                                                                             robustness transient gene knock-downs Caenorhabditis <br /> National Institutes Health (NIH) [GM079759]                      elegans. Proc. Biol. Sci., 271, 89&#226;&#8364;&#8220;96. <br /> National    Centers        Biomedical  Computing                    24. Thatcher,J.W., Shaw,J.M. Dickinson,W.J. (1998) Marginal <br />                                                                             &#239;&#172;?tness contributions nonessential genes yeast. Proc. Natl <br /> [U54CA121852]. Funding open access charge: NIH                          Acad. Sci. USA, 95, 253&#226;&#8364;&#8220;257. <br /> [GM079759].                                                             25. Gillespie,J.H. (1998) Population Genetics, Concise Guide. Johns <br />                                                                             Hopkins Univ. Press, Baltimore. <br /> Con&#239;&#172;&#8218;ict statement. declared.                           26. Hartl,D. Clark  (1997) Principles Population Genetics, <br />                                                                             3 edn. Sinauer Associates, Sunderland. <br />                                                                         27. Lynch,M. (2006) Streamlining simpli&#239;&#172;?cation microbial <br /> REFERENCES                                                                  genome architecture. Annu. Rev. Microbiol., 60, 327&#226;&#8364;&#8220;349. <br />                                                                         28. Kellis,M., Patterson,N., Endrizzi,M., Birren,B. Lander,E.S. <br />  1. Gu , Steinmetz,L.M., Gu,X., Scharfe,C., Davis,R.W.                (2003) Sequencing comparison yeast species identify <br />     Li,W.H. (2003) Role duplicate genes genetic robustness            genes regulatory elements. Nature, 423, 241&#226;&#8364;&#8220;254. <br />     null mutations. Nature, 421, 63&#226;&#8364;&#8220;66.                         29. Altschul,S.F., Madden,T.L., Schaffer , Zhang,J., Zhang , <br />  2. Stelling,J., Sauer,U., Szallasi , Doyle,F.J. III Doyle,J.         Miller,W. Lipman,D.J. (1997) Gapped BLAST PSI- <br />     (2004) Robustness cellular functions. Cell, 118, 675&#226;&#8364;&#8220;685.            BLAST: new generation protein database search programs. <br />  3. Wagner  (2005) Robustness Evolvability Living Systems.         Nucleic Acids Res., 25, 3389&#226;&#8364;&#8220;3402. <br />     Princeton University Press, Princeton.                              30. Gu , Cavalcanti , Chen,F.C., Bouman,P. Li,W.H. (2002) <br />  4. Wagner  (2000) Robustness mutations genetic                Extent gene duplication genomes Drosophila, <br />     networks yeast. Nat. Genet., 24, 355&#226;&#8364;&#8220;361.                            nematode, yeast. Mol. Biol. Evol., 19, 256&#226;&#8364;&#8220;262. <br />  5. Papp,B., Pal,C. Hurst,L.D. (2004) Metabolic network             31. Yang  Nielsen,R. (2000) Estimating synonymous     analysis causes evolution enzyme dispensability        nonsynonymous substitution rates realistic evolutionary <br />     yeast. Nature, 429, 661&#226;&#8364;&#8220;664.                                            models. Mol. Biol. Evol., 17, 32&#226;&#8364;&#8220;43. <br />  6. Lynch,M. Conery,J.S. (2000) evolutio <br /> </body></html>