<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>A novel SMC protein complex Schizosaccharomyces pombe contains Rad18 <br /> DNA repair protein <br />  <br />  <br />                            arresting cell cycle following damaging treatments <br />                                                                   review, Caspari Carr, 1999). <br />      previous work used rad18-X mutant  <br />                                                                  Rad18 protein involved repair ionizing <br />                                          radiation damage (Lehmann et al., 1995). Double-strand <br />  <br />                                                                  breaks principal lesions responsible killing <br />                                                                  cells ionizing radiation. Verkade et al. (1999) Schizosaccharomyces pombe, rad18 essential              (unpublished observations) shown rad18-X <br /> gene involved repair DNA damage produced               cells deficient repair radiation-induced double- ionizing radiation tolerance UV-induced             strand breaks. <br /> DNA damage. Rad18 protein member                        Epistasis analysis response UV-irradiation suggests SMC (structural maintenance chromosomes)                  Rad18 involved NER, does play <br /> superfamily,  like SMC                role second excision repair pathway UV <br /> proteins condensin cohesin, Rad18 compon- <br />                                                                  damage. step pathway mediated UV <br /> ent high-molecular-weight complex. complex <br />                                                                  damage-endonuclease (UVDE) (Yonemasu et al., 1997), <br /> contains proteins, largest                                                                  nicks UV-irradiated DNA 5&#226;&#172;&#732; UV Spr18, novel SMC family member closely related <br />                                                                  photoproducts (Avery et al., 1999; Yoon et al., 1999). Rad18, likely heterodimeric partner. <br />                                                                  uvde background, rad18-X mutation sensitized <br /> SMC proteins <span id='am-11' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-binding domains                                                                  cells UV-irradiation (Yonemasu et al., 1997), N- C-termini, extended coiled-coil domains <br />                                                                  epistasis analysis suggested separated hinge middle.                                                                  role Rad18 DNA damage tolerance pathway <br /> N-terminal ATP-binding domain Rad18 essential <br />                                                                  (Murray et al., 1997). Furthermore, deletion rad18 functions, overexpression N-terminal <br />                                                                  gene demonstrated essential cell prolifera- <br /> mutant dominant-negative effect. identi- <br />                                                                  tion, data led propose tentatively fied important mutation (S1045A) near C-terminus Rad18 separates repair                essential role involvement DNA replication <br /> essential roles. Potential models role            (Lehmann et al., 1995). Figure 1 summarizes pathways <br /> Rad18&#226;&#8364;&#8220;Spr18 complex DNA repair dis-                   Rad18 involved, based genetic analysis. <br /> cussed.                                                             Sequence analysis Rad18 (Lehmann et al., 1995) <br /> Keywords: ATPase/coiled coils/DNA repair/fission yeast/          showed 131 kDa Rad18 protein member <br /> SMC proteins                                                     SMC (structural maintenance chromosomes) <br />                                                                  superfamily. SMC proteins globular N-                                                                  C-terminal domains, involved ATP Mg2&#226;&#171;&#185; <br />                                                                  binding, extended &#206;&#177;-helical coiled-coil domains <br />                                                                  involved protein&#226;&#8364;&#8220;protein interactions, separated Introduction                                                     hinge region  reviews, Hirano, 1998, 1999; genes involved response DNA damage           Jessberger et al., 1998). Recent work using Saccharomyces identified fission yeast Schizosaccharomyces         cerevisiae, S.pombe Xenopus laevis identified pombe  Lehmann, 1996). Epistasis analysis enabled        protein complexes containing heterodimeric pairs SMC genes assigned different DNA repair               proteins. Smc2 Smc4  orthologues) form <br /> pathways. Following treatment ionizing radiation,           core condensin, subunit complex involved repair DNA double-strand breaks requires genes known          chromosome condensation (Hirano et al., 1997; Sutani involved homologous recombination, suggesting           et al., 1999). Smc1 Smc3  orthologues) mechanism likely major means           core subunit cohesin complex, repairing DNA damage fission yeast. Photoprod-           required sister chromatid cohesion (Michaelis et al., <br /> ucts induced DNA UV-irradiation repaired            1997; Losada et al., 1998). evidence classical <span id='am-22' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> excision repair (NER)           bovine cells suggest SMC1 SMC3 proteins  conserved eukaryotes,           form complex DNA ligase III DNA <br /> second excision repair process, specific             polymerase &#206;&#181;, DNA strand exchange activity <br /> S.pombe organisms (Yonemasu et al.,              (Jessberger et al., 1996). <br /> 1997; Yasui McCready, 1998). addition               paper purification high- <br /> pathways repair DNA damage directly, cells able         molecular-weight complex DNA-dependent ATPase tolerate UV damage DNA replication (Murray             activity, contains Rad18 et al., 1997) utilize cell cycle checkpoint mechan-       polypeptides. largest novel SMC protein <br /> isms ameliorate deleterious effects DNA damage         (Spr18), likely heterodimeric partner <br />  <br /> &#194;&#169; European Molecular Biology Organization                                                                              1691 <br />  M Fousteri R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 1. Involvement Rad18 different cellular responses DNA <br /> damage. <br />  <br />  Rad18. site-directed mutagenesis N-terminal ATP-binding domain Rad18 required functions, C-terminal mutation separate repair essential functions. <br />  <br /> Results <br /> Rad18 high-molecular-weight complex determine Rad18, like SMC proteins, member protein complex, fractionated extracts S.pombe gel filtration columns. experiments constructed cell strain myc-Rad18, rad18 <br /> gene myc-tagged genome 5&#226;&#172;&#732; end, <br /> remaining control promoter. extracts myc-Rad18 cells passed Superdex <br /> 200 column, Rad18 protein high- <br /> molecular-weight peak (Mr &#226;&#172;&#381;700 kDa). obtain accurate estimate size complex, peak               Fig. 2. Rad18 high-molecular-weight complex. Extracts fractions column pooled analysed                 myc-Rad18   myc-Rad18-X cells (C) precipitated Superose 6 column (Figure 2A). results                ammonium sulfate, fractionated Superdex 200 high- <br />                                                                       molecular-weight peak analysed gel filtration Superose 6 <br /> Rad18 complex apparent Mr ~1.6 MDa.             column. Fractions subjected immunoblotting anti-myc complex stable presence 0.7 M                antibody. high-molecular-weight band gel non-specific <br /> NaCl  shown). Using polyclonal antibody raised                  cross-reacting material. (B) Wild-type strain 501 subjected Rad18 <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span>, showed gel                  treatment  analysed anti-Rad18 antibody. <br /> filtration profile identical Rad18 <br /> protein N-terminally myc-tagged (Figure 2A B). change gel filtration profile detected             ive washing, bound proteins eluted peptide <br /> extracts &#206;&#179;-irradiated cells used (data shown).            corresponding c-myc epitope, analysed     discussed  mutation rad18-X located            immunoblotting colloidal Coomassie staining. hinge region, immediately adjacent second               Immunoblotting showed bound eluted proteins <br /> coiled-coil region  Figure 5A). raised possibil-         contained myc-tagged Rad18 (Figure 3B, lane 2). ity mutation disrupt coiled-coil structure         extracts untagged control cells (strain 501) protein interfere complex formation.              subjected identical purification procedures, corres- <br /> Rad18 profiles gel filtration columns using normal                 ponding protein bound beads (Figure 3B, rad18-X extracts   indistinguishable                 lane 4). Coomassie staining revealed  apart (compare Figure 2C , salt concentrations                Rad18, specific bands eluate 0.7 M NaCl. rad18-X mutation does                       tagged cells (Figure 3C, lane 3)  interfere complex formation.                          similar experiments using untagged control cells <br />                                                                       (Figure 3C, lanes 1 2). specific bands, <br /> Purification Rad18                                                 p124, molecular weight close Rad18 developed purification procedure Rad18-             similar intensity. <br /> containing complex, using extracts myc-Rad18 cells <br /> (Figure 3A). ammonium sulfate precipitation,                ATPase activity <br /> extracts subjected gel filtration Superdex 200.            Mutagenesis studies described implicate high-molecular-weight fractions containing Rad18                  ATP-binding activity Rad18 functions, sug- pooled fractionated heparin&#226;&#8364;&#8220;Sepharose                   gesting protein ATPase activity. column. 0.6 M KCl fractions, contained                  measured ATPase activity Rad18 <br /> Rad18 protein, incubated protein G beads                    complex. assayed peptide eluate immuno- <br /> covalently cross-linked anti-myc antibody. extens-           affinity beads ATPase activities presence  <br /> 1692 <br />                                                                                                Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 3. Rad18 complex.   Scheme purification Rad18 complex myc-Rad18 cell extracts (WCE, cell extracts; FT, flow- . (B) Samples proteins loaded (Load) eluted (B&#226;&#8364;&#8482;nd) anti-myc immuno-affinity beads immunoblotted anti- <br /> myc anti-Spr18 antibodies. (C) Proteins bound beads eluted myc peptide analysed SDS&#226;&#8364;&#8220;PAGE. gel fixed stained colloidal Coomassie Blue. Lanes 1 2, extracts strain 501; lane 3, myc-Rad18. (D) ATPase activity Rad18 complex. <br /> Experiments carried times, typical results shown. <br />  <br />  <br /> absence DNA. Eluate myc-Rad18 cell extracts                    candidate open reading frame (ORF) cosmid significant ATPase activity, stimulated                   C14C4 chromosome 1 (DDBJ/EMBL/GenBank <br /> single-stranded, greater extent double-stranded                accession  Z98596) orthologue DNA (Figure 3D). contrast, eluate untagged                         S.cerevisiae database (ORF SCYOL034W; DDBJ/EMBL/ <br /> control strain 501 containing equal non-                  GenBank accession  Z74776). predicted 123 kDa <br /> specific 73 kDa protein  Figure 3C) minimal                        protein related SMC family, activity, unchanged addition DNA.                      closely related Rad18  Figure 4A). designated complex associated specific DNA-                     Spr18 (SMC partner rad18). Alignments Rad18 <br /> stimulated ATPase activity.                                                Spr18 S.cerevisiae orthologue shown                                                                            Figure 4B. 28% identity S.pombe <br /> Spr18, heterodimeric partner Rad18                                    S.cerevisiae orthologues, 20% identity SMC family, Smc2 Smc4 form heterodimeric                      Rad18 Spr18. Like rad18, spr18 small intron <br /> component condensin complex (Hirano et al., 1997),                  (49 bases) close 5&#226;&#172;&#732; end gene  nt 187 involved chromosome condensation. Smc1                         188). Smc3 similarly cohesin, involved sister                    order determine Spr18 interacts chromatid cohesion (Michaelis et al., 1997).                  Rad18 vivo, carried immunoprecipitations using <br /> anticipated Rad18 heterodimeric partner,                 myc-Rad18 strain (Figure 4C). Extracts immuno- considered possibility p124 polypeptide                precipitated antibody directed peptide complex novel SMC protein partner                     Spr18 (Figure 4C, lanes 2, 4, 6 8)  control, Rad18. searched S.pombe sequence database                       anti-myc antibody (lanes 1, 3, 5 7). immuno- <br />  <br />                                                                                                                                          1693 <br />  M Fousteri R.Lehmann <br />  <br />  <br />  <br />  <br /> 1694 <br />                                                                                                 Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> precipitates electrophoresed SDS&#226;&#8364;&#8220;polyacryl-                      Viability Ura4&#226;&#171;&#185; phenotype segregated 2:0 <br /> amide gel immunoblotted anti-myc antibody.                     manner, confirming spr18, like rad18 SMC <br /> Figure 4C shows myc-tagged Rad18 immunopre-                         genes, essential cell proliferation. order cipitated Spr18 experiments presence                   determine terminal phenotype spr18 deletion salt concentrations 0.75 M (lanes 1&#226;&#8364;&#8220;6). Similar                    strain, spores germinated liquid medium results obtained blots probed                       selection Ura4&#226;&#171;&#185;, cell number morphology <br /> anti-Spr18 (data shown). results affected                  determined. Germinating spores underwent treatment cell extracts DNase prior                     divisions ceased growth variety abnormal <br /> immunoprecipitation, demonstrating interaction                     morphologies. examples shown Figure 4D. mediated binding DNA. immuno-                        cells elongated, cases excessively precipitation experiments carried control                     (Figure 4D, examples 1 5), septa <br /> untagged strain 501, bands detected                          (examples 3 5), nuclear material barely <br /> immunoblot (Figure 4C, lanes 7 8). results                        detectable, present single nucleus <br /> demonstrate Rad18 Spr18 interact,                        (example 4) spread amorphously consistent idea proteins form                       cell (example 1). proportion cells (10&#226;&#8364;&#8220;15%) showed <br /> heterodimer S.pombe cells,                          cut phenotype (Figure 4D, example 2). distribution <br /> formally eliminated possibility interaction                    different phenotypes shown Figure 4E. mediated protein. results                      similarity terminal phenotype spr18 deletion <br /> presented  shown similar interaction                    mutant, rad18 deletion (Lehmann et al., 1995) wild- human orthologues proteins                             type cells overexpressing dominant-negative mutants (E.M.Taylor, J.Moghraby R.Lehmann, unpublished                        rad18  , provides evidence Rad18 <br /> observations).                                                              Spr18 perform similar functions cell. <br />  <br /> Spr18 p124                                                           Mutagenesis rad18 defines important structural peptide elution fractions immuno-affinity                      domains <br /> column obtained using myc-Rad18 501                            characteristic SMC structure Rad18, globular <br /> extracts (Figure 3) analysed immunoblotting                    head tail regions coiled-coil domains separated anti-Spr antibody. Spr18, like Rad18, detected                   hinge, depicted schematically Figure 5A. bound eluted proteins myc-Rad18 extract                    gain insight roles different domains, (Figure 3B, lane 6), untagged control                        used site-directed mutagenesis introduce series <br /> extracts (lane 8). band detected using anti-Spr18                   mutations rad18 cDNA. cDNA cloned <br /> antibody corresponded p124 protein                      control attenuated thiamine-repressible Coomassie-stained gel. results confirm p124                  nmt1 promoter vectors pRep41MH N (Leu&#226;&#171;&#185;) SMC protein Spr18 consistent                        pRep42MH N (Ura&#226;&#171;&#185;) (Maundrell, 1993; Craven et al., <br /> suggestion Rad18&#226;&#8364;&#8220;Spr18 forms heterodimeric core                      1998), Rad18 protein N-terminally myc- large complex similar SMC complexes,                      tagged. effects mutations examined cohesin condensin. investigate Rad18&#226;&#8364;&#8220;                    different ways, follows:   ability mutant <br /> Spr18 complex contains known DNA damage                               plasmids rescue lethal effect rad18 deletion <br /> response genes, carried immunoblotting                        examined transformation rad18::ura4&#226;&#171;&#185;/ <br /> proteins eluted anti-myc affinity column using                   rad18&#226;&#171;&#185; diploid strain; (ii) mutated plasmids series antibodies. unable detect presence                 overexpressed wild-type cells look dominant- DNA damage response proteins Rad9, Rad26,                            negative effects mutant gene cell growth Cds1, Rqh1 Rad21 Rad18&#226;&#8364;&#8220;Spr18 complex (data                        viability, sensitivity UV- &#206;&#179;-irradiation; shown).                                                                 (iii) mutated genes introduced rad18-X <br />                                                                             cells impact sensitivity mutant Spr18 essential gene                                                  ionizing UV-irradiation examined following Spr18 partner protein Rad18, anticipated                    transient overexpression.  like Rad18, essential gene. deleted                    mutations effects summarized entire ORF spr18 gene genome                       Figure 5 Table  mutant constructs <br /> diploid strain replacing copy gene                   dominant-negative effect cell viability <br /> ura4&#226;&#171;&#185; marker. Following sporulation plating                      (Figure 5A). proportion population lost <br /> absence <span id='am-42' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>, haploid colonies obtained,                       viability depended mutation, phenotype suggesting spr18 gene essential cell                       dying cells quite similar cases, resembled <br /> proliferation. order confirm  tetrads dis-                  terminal phenotype deletion mutants rad18 <br /> sected spores plated yeast extract plates.                  (Lehmann et al., 1995) spr18  . simplicity <br />  <br />  <br /> Fig. 4. Spr18, SMC partner protein Rad18.   Phylogenetic tree SMC family members S.cerevisiae S.pombe. (B) Sequence <br /> alignments Rad18 Spr18 S.cerevisiae orthologue (designated scspr). Identical amino acids highlighted black, conserved <br /> residues grey. (C) Extracts myc-Rad18 cells immunoprecipitated anti-myc anti-Spr18 antibody presence salt concentrations indicated. immunoprecipitates immunoblotted probed anti-myc antibody. right hand lanes (7 8) results untagged strain 501. (D) Terminal phenotypes spr18 deletion mutant. Diploid spr18&#226;&#171;&#185;/spr18::ura4&#226;&#171;&#185;cells sporulated grown absence uracil 36 h. Cells fixed stained DAPI calcofluor. (E) Frequency different phenotypes 24 h <br /> growth. <br />  <br />  <br />                                                                                                                                           1695 <br />  M Fousteri R.Lehmann <br />  <br />  <br />  <br />  <br /> Fig. 5. Effects site-directed mutations Rad18.   Schematic Rad18 protein positions mutations, ability rescue lethal <br /> phenotype rad18 deletion mutant effect overexpression viability wild-type strain (&#226;&#171;&#185;&#226;&#171;&#185;&#226;&#171;&#185;, cells dead; <br /> &#226;&#171;&#185;&#226;&#171;&#185;, cells dead retain viability; &#226;&#171;&#185;, 30&#226;&#8364;&#8220;40% cells retain viability). (B C) Survivals &#206;&#179;- UV-irradiation wild-type (B) rad18-X cells (C) overexpressing different mutant plasmids. Experiments carried times, typical results shown (Mrep42, pREP42 MH vector). <br />  <br />  <br />  refer Rad18 phenotype,                         small, septated prior mitosis, resulting intended implication phenotype                        long cells nucleus unique rad18 mutants. phenotype consisted                           nucleus multiple septa. Examples shown <br /> cells variety abnormal morphologies                         Figure 6A (compare overexpressed wild-type <br /> nuclear segregation clearly aberrant.                         rad18 Figure 6B). <br />  <br /> 1696 <br />                                                                                                    Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br />  <br />  <br /> Fig. 6. Dominant-negative effects mutations rad18 gene.  B) Effects overexpressing   rad18K129Q, rad18L843P rad18R706A (B) wild-type rad18 cell growth morphology. (C) Wild-type cells overexpressing wild-type rad18, rad18K129E <br /> mutant plasmid,  incubated absence thiamine cell numbers counted 24 h time period. (D) Western blots cells <br /> overexpressing myc-tagged rad18K129E (lane 1) myc-tagged rad1 (lane 2), rad2 (lane 3), rad3 (lane 4), pcn1 (lane 5), rad17 (lane 7) HA-tagged cdc6 (lane 6). <br />  <br />  <br />  <br /> Table  Summary effects overexpressing mutant constructs <br />  <br />                       K129E          K129Q          R706C         R706A          L843P          L843F             S1045A   S1045T       D1072A <br />  <br /> WT viabilitya         0              35             &#226;&#171;&#185;             42             20             &#226;&#171;&#185;                 &#226;&#171;&#185;        &#226;&#171;&#185;            &#226;&#171;&#185; <br /> WT UVb                dead           S              N             S              dead           N                 S        S            N <br /> WT &#206;&#179; b                dead           S              N             S              dead           N                 S        N            N <br /> Rad18-X UVc           dead           SS             SS            SS             SS             R                 SS       SS           SS <br /> Rad18-X &#206;&#179; c           dead           SSS            SS            SSS            SS             R                 SSS      SSS          SS <br /> Rad18&#226;&#710;&#8224;d                                       yes           partial                    yes               yes                aNumbers  denote percentage viable cells. &#226;&#171;&#185;, &#226;&#172;&#381;70% viability. <br /> bN, normal; S, sensitive normal. <br /> cR,restores wild-type resistance; SS, sensitivity comparable rad18-X; SSS, sensitivity greater rad18-X. <br /> dYes, does;  does rescue lethality deletion strain. <br />  <br />  <br />  <br />  <br />    N-terminal domain Rad18 contains highly                         liquid culture, cell growth prevented (Figure 6C) conserved Walker Type box ATPases.                            cells died Rad18 phenotype. mutated conserved Lys129 residue GXGKS                             generated milder mutation Lys129 motif <span id='am-30' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-31' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span>glutamic acid</span> (rad18K129E). rad18K129E                          glutamine (K129Q). plasmids containing  overexpressed wild-type cells, dramatic                       struct overexpressed wild-type cells, resulting <br /> dominant-negative effect viability, completely abol-                       colonies small, liquid culture, significant <br /> ishing colony-forming ability cells. similar                     proportion cells (65%) showed Rad18 phenotype <br /> result recently obtained different mutation                    reduced viability (Figure 6A). viable cells ATP-binding site Verkade et al. (1999).                        sensitized UV- ionizing radiation rad18K129E overexpressed cells growing                          (Figure 5B). mutant plasmid failed rescue  <br />                                                                                                                                              1697 <br />  M Fousteri R.Lehmann <br />  <br />  <br /> viability deletion mutant slightly increased     mutations rad18. Mutations ATP-binding site sensitivity rad18-X cells &#206;&#179;-irradiation (Figure 5A    N-terminus disruption coiled-coil domain C).                                                         dramatic effects, suggesting ATP- <br />    C-terminal globular region contains motifs LSGG          binding site coiled-coil structures essential  conserved SMC proteins, probably      functions. Effects mutations conserved C-terminal <br /> representing Walker Type B motif involved        domain severe. S1045A mutation ATP&#226;&#8364;&#8220;Mg2&#226;&#171;&#185; binding. mutated Ser1045 LSGG              conserved LSGG motif enabled separate repair sequence alanine threonine (rad18S1045A         essential functions protein, affected <br /> rad18S1045T), Asp1072 alanine (rad18D1072A).              <br /> Overexpression mutated genes significant <br /> effect viability wild-type cells.         Overexpression damage response genes <br /> rad18S1045A mutant plasmid sensitized cells         rescues dominant-negative phenotype <br /> &#206;&#179;- UV-irradiation damage, rad18S1045T               looked genetic interactions rad18 effect rad18D1072A minimal                     genes examining ability suppress effect (Figure 5B). S1045A S1045T plasmids              dominant-negative effects K129E mutation slightly increased &#206;&#179;-irradiation sensitivity         ATP-binding site. normal mutant cDNAs rad18-X mutant (Figure 5C). Importantly,            overexpressed separate plasmids rad18S1045A mutant plasmid,           nmt1 promoter  pRep41 42), normal plasmid able rescue lethality rad18 deletion          able rescue growth-inhibitory effects strain (Figure 5A). significant result shows          mutant plasmid (Figure 6C). used expression S1045A mutation able separate repair         investigate genes (rad1, essential phenotype cells. possible   rad2, rad3, rad17, pcn1 cdc6) rescue rad18 mutant plasmid rescued rad18 deletion         adverse effects mutant plasmid. mutant gene overexpressed.  results          cell cycle checkpoint genes rad1 rad3, presented  eliminated         rad2 gene encoding structure-specific nuclease possibility. integrated S1045A mutation        cdc6 gene encoding DNA polymerase &#206;&#180; overexpressed genomic rad18 gene, expressed        rad18K129E, growth inhibition normal levels. strain viable sensitive           overcome, pcn1 encoding PCNA check- <br /> radiation.                                                      point gene rad17 effect. order ensure    Coiled-coil motifs amphipathic &#206;&#177;-helices character-      merely result second plasmid <br /> ized heptad repeats hydrophobic amino acids          reducing expression mutant gene, used fourth residues heptad (Lupas et al.,     Western blotting determine levels expression <br /> 1991). mutated Leu843, middle long heptad        14 h absence thiamine. genes <br /> repeat, proline (L843P), expected      myc-tagged  cases), able destroy coiled-coil structure, phenylalanine       detect Rad18 expressing proteins using <br /> (L843F), hydrophobic like leucine,           antibody directed myc epitope tag. completely different structure. mutation behaved     expression mutant rad18K129E affected <br /> like wild type respects (Figure 5B): restored         expression second plasmid (Figure 6D). <br /> viability rad18 deletion strain rescued          data provide evidence genetic inter- <br /> sensitivity DNA damage rad18-X cells                  action rad18 rad1, rad2, rad3 cdc6. <br /> (Figure 5C). contrast, L843P dramatic effect.          interactions studied future work. wild-type cells overexpressing plasmid able form colonies, small,          Discussion liquid culture, cells (80%) Rad18 <br /> phenotype. Viability deletion strain restored    Rad18&#226;&#8364;&#8220;Spr18 SMC protein complexes mutant plasmid.                                       understanding DNA repair pro- <br />    mutation rad18-X, sensitive UV-       cesses increased dramatically years, <br /> &#206;&#179;-irradiation elevated temperatures, results         functions DNA repair proteins poorly <br /> change R706C. considered possibility           understood. shown scheme Figure 1, Rad18 <br /> change exert effects mutant protein       plays important role repair processes, contains cysteine residue, form aberrant          exact function unknown. SMC protein family <br /> disulfide bridges. mutated Arg706 alanine       members essential chromosome condensation <br /> (R706A) compared effects R706C.           (SMC2 4) cohesion (SMC1 3)  Hirano, <br /> Overexpression R706C effect wild-           1998; Jessberger et al., 1998), role type rad18-X cells restored viability         proteins play molecular level poorly understood. <br /> deletion strain. effects R706A         shown paper Rad18 resembles severe, phenotype quite similar            SMC proteins forms heterodimeric complex <br /> rad18K129Q  Figure 5B C Table .             closely related partner protein, Spr18, Rad18&#226;&#8364;&#8220; <br /> findings ruled possibility phenotype        Spr18 higher molecular weight complex. rad18-X resulted aberrant disulfide bridges,           complex molecular weight &#226;&#172;&#381;1 MDa, stable implicated R706 important residue              high salt, contains proteins Rad18 function.                                                 DNA-stimulated ATPase activity. Similarly, condensins <br />    summary, gradation severity         Xenopus S.pombe contain multiple subunits <br />  <br /> 1698 <br />                                                                                     Schizosaccharomyces pombe Rad18 protein <br />  <br />  <br /> including, Xenopus, XCAP-C XCAP-E (Hirano <br /> et al., 1997) S.pombe, Cut3 Cut14 (Tanaka <br /> et al., 1999), orthologues SMC2 SMC4. <br /> Likewise cohesins, include SMC1 SMC3 <br /> (Losada et al., 1998), dosage compensation <br /> complex Caenorhabditis elegans, contains SMC protein DPY27 (Chuang et al., 1996), contain <br /> multiple subunits. complex designated RC-1, able carry strand exchange reactions, composed bovine SMC1 SMC3 DNA polymerase &#206;&#181; DNA ligase III (Jessberger et al., 1996). Rad18 Spr18 orthologues S.cerevisiae C.elegans databases isolated human ortholog- <br /> ues (E.M.Taylor, J.Moghraby R.Lehmann,             Fig. 7. Models involvement Rad18&#226;&#8364;&#8220;Spr18 complex DNA <br /> unpublished results). view similarity structural    repair. diagram shows roles SMC proteins chromosome <br />                                                                  condensation cohesion proposed Hirano (1999), composition Rad18&#226;&#8364;&#8220;Spr18 complex,                 possible roles Rad18&#226;&#8364;&#8220;Spr18 repair double-strand breaks. <br /> densin cohesin, consider Rad18&#226;&#8364;&#8220;Spr18            Black lines represent double-stranded DNA, grey symbols conserved SMC complex eukaryotes.                       represent SMC proteins globular N- C-termini hinge <br />                                                                  regions, extended coiled coils  <br /> Structure&#226;&#8364;&#8220;function relationships Rad18 <br /> Structural studies bacterial SMC-like proteins           1990; Gregory et al., 1991). mutation provides carried recently Melby coworkers (Melby          opportunity investigation repair functions <br /> et al., 1998). MukB SMC-like protein              Rad18 homologues. <br /> Escherichia coli antiparallel homodimeric <br /> structure, globular domains attached rigid           Function Rad18&#226;&#8364;&#8220;Spr18 complex <br /> coiled-coil domains hinge. coiled-coil regions        SMC protein complexes function chromosome- <br /> form basis dimerization. hinge appeared          associated processes, condensation, cohesion quite flexible, permitting scissoring movement.          dosage compensation. thought putative <br /> probable Rad18 complex, Rad18 Spr18              scissoring action, possible flexibil- <br /> form antiparallel heterodimers.                                  ity SMC dimer (Melby et al., 1998), enables    Site-directed mutagenesis studies (Figures 5 6)      complexes DNA molecules relative shown globular N-terminal domain coiled-        cellular structures. condensin complex <br /> coil regions required repair essential          Xenopus able supercoil DNA presence <br /> functions Rad18. consistent structure        topoisomerase (Kimura Hirano, 1997) suggested  assume Rad18 forms similar         effect topological reconfigurations DNA molecules <br /> structure heterodimer Spr18, provide        (Kimura et al., 1999). S.cerevisiae SMC1p ready explanation dominant-negative effects         SMC2p proteins able bind DNA, DNA- <br /> overexpressed mutant Rad18 (Figure 6 Verkade et al.,         binding activity resides C-terminus molecule <br /> 1999). majority mutants coiled-       (Akhmedov et al., 1998). Rad18 binds <br /> coil domains remain intact, overexpressed mutant             tightly DNA-cellulose columns  unpublished data), <br /> Rad18 protein dimerize sequester            suggesting Rad18 likely DNA-binding <br /> cellular Spr18, preventing dimerizing           protein, consistent DNA-stimu- <br /> endogenous protein. L843P mutant probably causes             lated ATPase activity associated local disruption &#206;&#177;-helical structure          Rad18&#226;&#8364;&#8220;Spr18 complex. previously suggested C-terminal coiled coil, N-terminal coiled-          Rad18 involved recombinational repair processes <br /> coil domain likely unaffected capable      (Lehmann et al., 1995). current work interacting Spr18.                                          published data functions SMC proteins lead    Interestingly, mutations conserved C-terminal          speculate function Rad18&#226;&#8364;&#8220;Spr18 complex <br /> globular domain relatively mild effects. Particularly        bring hold broken DNA molecules <br /> interesting S1045A mutation (Figure 5),             vicinity double-strand damage allow <br /> destroyed repair function protein, whilst retaining   repair recombination place  Figure 7). essential function. quite surprising view        damage double-strand break produced fact motif LSGG, encompassing Ser1045,           directly ionizing radiation, UV photoproduct conserved SMC proteins  exception          opposite gap stalled replication fork generated Spr18, contains sequence QSGG).             replication UV damage. envisage Rad18&#226;&#8364;&#8220; large family ATP transporter proteins (Ames        Spr18 complex holds ends break et al., 1992), proposed LSGG       alternatively holds broken chromosome register <br /> motif forms important linker region different         sister homologue allow Rhp51 RecA <br /> domains. protein family serine residue      homologue initiate strand exchange. context essential transporter function S.cerevisiae       Rad18&#226;&#8364;&#8220;Spr18 complex presumably plays complementary <br /> Ste6p protein involved transport factor phero-       role cohesins, known hold sister <br /> mone (Browne et al., 1996), mutated CFTR        chromatids  required normal <br /> protein cystic fibrosis patients (Kerem et al.,       response radiation, mutants S.pombe rad21,  <br />                                                                                                                             1699 <br />  M Fousteri R.Lehmann <br />  <br />  <br /> non-SMC component cohesin, sensitive radiation.                      Immunoprecipitation <br />                                                                                experiments constructed S.pombe strain myc-tagged Recent data suggest cohesin complex present                        genome (myc-Rad18). fragment DNA containing 950 bp specific sites chromosome arms                        rad18 upstream sequence coding sequence NcoI site <br /> coating lengths chromosomes (Blat                               nt 1760 cloned integrating vector pSta18 (Carr et al., <br /> Kleckner, 1999; Tanaka et al., 1999). context                        1989), contains sup3.5 tRNASerUGA insert (Hottinger, 1982). <br /> recombinational repair double-strand DNA damage,                         His6-myc2 coding sequence inserted NdeI fragment                                                                                pGEMMH (Craven et al., 1998) NdeI site engineered start cohesins necessary maintain overall                     rad18 ORF. plasmid transformed S.pombe strain <br /> cohesion sister chromatids, Rad18&#226;&#8364;&#8220;                          501 (leu1.32 ura4.D18 ade6.704) selection applied growth <br /> Spr18 complex specific role sites damage.                     absence <span id='am-6' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span>.  ade6.704 mutation 501 strain suggestion role Rad18 recombination                         suppressed sup3.5.) Ade&#226;&#171;&#185; colonies grown analysed supported 4-fold reduction intrachromosomal                          integration tagged sequence Southern blotting immuno- <br />                                                                                blotting. similar procedure followed S.pombe rad18-X <br /> recombination recently reported Rad18 mutant                           myc-tagged strain (myc-Rad18-X). <br /> Arabidopsis thaliana (Mengiste et al., 1999).                              immunoprecipitations, crude extract containing 2 mg total <br /> complex role Rad18 response DNA damage                           protein incubated overnight anti-Spr18 antibody suggested work Verkade et al. (1999),                       9E10 anti-myc antibody ab binding buffer [40 mM HEPES pH 7.8, <br />                                                                                2 mM EDTA, 2 mM EGTA, 60 mM &#206;&#178;-glycerophosphate, 1 mM NaF, <br /> provided evidence suggest circum-                           0.5 mM phenylmethylsulfonyl fluoride (PMSF), 0.2 mM dithiothreitol <br /> stances Rad18 required maintain checkpoint                            (DTT), 0.1 mM orthovanadate, 10% glycerol protease inhibitor <br /> following DNA damage, establishment                            cocktail, described  presence different concentrations <br /> checkpoint clearly normal. finding genetic                        NaCl. immune complexes captured protein G&#226;&#8364;&#8220;agarose <br />                                                                                beads, washed times, boiled gel loading buffer elec- <br /> interaction checkpoint genes                           trophoresed 10% SDS&#226;&#8364;&#8220;PAGE gels. blotted PVDF <br /> rad18 consistent observations. obtained                       membrane filters probed anti-myc antibody. evidence,  physical association Rad18&#226;&#8364;&#8220;Spr18 complex checkpoint proteins                             Purification Rad18 complex <br /> tested. relationship Rad18 checkpoint                            purification, 30 l exponentially growing myc-Rad18 cells                                                                                harvested, washed cell pellet resuspended equal volume <br /> machinery likely,  complex indirect.                    lysis buffer (40 mM HEPES pH 7.8, 0.4 M KCl, 80 mM <br />    Rad18 Spr18 essential cell proliferation.                  &#206;&#178;-glycerophosphate, 12 mM NaF, 5 mM EGTA, 5 mM EDTA, 1 mM results Figure 4D Figure 6                         DTT, 1 mM PMSF, 0.1 mM orthovanadate, 10% glycerol, protease <br /> absence functional Spr18, dominant-negative                         inhibitor cocktail consisting 5 &#194;&#181;g/ml trypsin inhibitor, <br /> mutant rad18 constructs overexpressed, cells appear                        pepstatin, leupeptin, aprotinin, 10 &#194;&#181;g/ml bestatin E-64,                                                                                50 &#194;&#181;g/ml chymotrypsin inhibitor). cells snap-frozen liquid unable replicate DNA,                     nitrogen lysed using blender, following addition copious <br /> continue grow, lay septa develop grossly                           quantities dry ice. lysate clarified centrifugation abnormal morphologies. indicative role                       subjected ammonium sulfate precipitation. 15&#226;&#8364;&#8220;45% cut containing DNA replication mitotic control. Occasional                          Rad18 protein dissolved lysis buffer dialysed overnight <br />                                                                                buffer. extract containing ~300 mg protein double-strand breaks occur DNA replication                          applied multiple injections Superdex 200, 10/30 FPLC column thought repaired recombination. Cells                       (Pharmacia Biotech). Fractions 0.25 ml collected analysed <br /> deficient recombination-repair genes rhp51 rhp54                     Western blotting using anti-myc monoclonal antibody. high- <br /> grow poorly (Muris et al., 1993, 1996), presumably                             molecular-weight fractions containing Rad18 protein pooled difficulty dealing damage arising                    fractionated heparin&#226;&#8364;&#8220;Sepharose Cl-6B (Pharmacia Biotech) <br />                                                                                equilibrated buffer B/0.1 M KCl (25 mM HEPES pH 7.8, 2 mM replication. Double mutants rhp51 rhp54                      EGTA, 3 mM MgCl2, 0.1 mM DTT, 0.5 mM PMSF, 12% glycerol). DNA damage checkpoint genes inviable (Muris                            extensive washing, proteins eluted buffer B/0.2, 0.4, <br /> et al., 1996), showing checkpoint-mediated mitotic                        0.6 1 M KCl. 0.6 M KCl eluate contained Rad18 <br /> delay necessary cell deal kind                      protein incubated overnight 4&#194;&#176;C protein G&#226;&#8364;&#8220;agarose beads <br />                                                                                (Pharmacia Biotech) crosslinked anti-myc antibody buffer B/ <br /> replicative stress. propose inviability rad18                   0.3 M KCl, containing 0.4 mM EDTA protease inhibitor cocktail. spr18 mutants arises involvement linking                     beads washed buffer, times 10 vol. <br /> replication, repair mitotic control. work                     buffer B/0.5 M KCl, 0.1% NP-40, twice buffer B/0.1 M KCl, necessary unravel origin complex                            0.01% NP-40. Bound proteins eluted twice 30&#194;&#176;C 1 h phenotypes.                                                                    peptide corresponding myc epitope (AEEQKLISEEDL)                                                                                3 mg/ml buffer B/0.1 M KCl, 0.2 mg/ml insulin 1 &#194;&#181;g/ml aprotinin <br />                                                                                E64. eluted fractions dialysed twice buffer B containing <br />                                                                                0.1 M KCl 15% glycerol eliminate peptide. proteins Materials methods                                                          separated 10% SDS&#226;&#8364;&#8220;PAGE stained colloidal Coomassie <br />                                                                                Brilliant Blue (Brilliant Blue G, colloidal concentrate; Sigma). Alternat- <br /> Antibodies                                                                     ively analysed immunoblotting antibodies specific 9E10 anti-myc monoclonal antibody used immunoblotting              different DNA damage response proteins. <br /> obtained Santa Cruz. immuno-affinity chromatography                   experiments estimate size complex, pooled Superdex <br /> used supernatants CRL-1729 hybridoma cells obtained              200 fractions contained high-molecular-weight peak, derived American Type Culture Collection, cultured RPMI 10%                 subjected gel filtration Superose 6 FPLC <br /> fetal calf serum. supernatant bound protein G&#226;&#8364;&#8220;agarose,              column (Pharmacia Biotech). case 0.5 ml fractions collected <br /> extensively washed covalently crosslinked using dimethylpimel-             analysed Western blotting. column calibrated imidate (Harlow Lane, 1988). Polyclonal antibodies Rad18                HMW Gel Filtration Calibration Markers (Pharmacia Biotech). Spr18 protein raised sheep peptides <br /> VYSLAKKEEEYKLLWEQSRE RKREREILQNKNQGQSTLNS-                                 ATPase assay <br /> LKDR, respectively. antibodies identified corresponding              assaying ATPase activity, elution affinity column vitro translated proteins. detected proteins appropriate   myc peptide carried 30&#194;&#176;C 20 min  eluate <br /> size cell extracts, detected corresponding pre-       dialysed buffer B containing 0.05 M KCl. Protein eluates <br /> immune sera. Antibodies damage response proteins            incubated 2 h 30&#194;&#176;C presence 0.1 &#194;&#181;Ci [&#206;&#179;-32P]ATP <br /> generously supplied Drs T.Caspari, J.Murray M.Carr.                   (7000 Ci/mmol; ICN) 10 &#194;&#181;l reaction volume containing 25 mM <br />  <br />  <br />  <br />  <br />  <br />  <br /> Tris&#226;&#8364;&#8220;HCl pH 7.5, 4 mM MgCl2, 20 &#194;&#181;M ATP, 1 mM DTT                          <br /> 50 &#194;&#181;g/ml BSA. reaction stopped adding 5 &#194;&#181;l 0.5 M                 <br /> EDTA pH 8.0. microlitre reaction mixture spotted          PEI-cellulose layer chromatography (TLC) plate (Sigma)             developed 0.75 M KH2PO4. Radiolabelled ATP inorganic              <br /> phosphate quantitated Storm 840 PhosphorImager (Molecular             <br /> Dynamics). DNA-dependent ATPase activity assayed presence          absence double- single-stranded DNA final concentration          15 ng/&#194;&#181;l.                                                                   <br /> Site-directed mutagenesis                                                    rad18 cDNA clone constructed PCR subcloning                <br /> total S.pombe cDNA. internal NdeI site destroyed site-directed <br />                                                                             <br /> mutagenesis, removing PCR errors mutagenesis,                                                                              <br /> cDNA cloned NdeI&#226;&#8364;&#8220;SalI fragment pRep41 MH,                                                                               <br /> pRep42 MH epitope-tagging expression vectors (Craven et al., 1998). <br /> Mutations cDNA introduced specific sites using           <br /> procedure Kunkel et al. (1987), mutant clones             <br /> sequenced completely ensure desired mutations      <br /> introduced. regularly subcloning rad18 constructs        <br /> unsuccessful ligated DNA transformed variety          <br /> commonly used E.coli strains. transformed ligation           <br /> mixes directly S.pombe successfully obtained desired          <br /> products.                                                                 <br />                                                                             <br /> Cell survival complementation essential                           <br /> phenotype                                                                      <br /> Mutant constructs introduced wild-type rad18-X cells     presence thiamine maintain nmt1 promoter plasmid     repressed state. survival experiments, cells grown     <br /> absence thiamine 16 h induce steady levels expression       nmt1 promoter. exposed UV- &#206;&#179;-irradiation       <br /> 1000 cells plated described previously (Lehmann et al.,         <br /> 1995). Wild-type cells plated minimal medium, rad18-X            YEP. effects cell viability, wild-type cells grown      <br /> 12 h absence thiamine. Samples plated          <br /> presence absence thiamine. samples taken hour        14 h cells counted, fixed methanol             <br /> stained DAPI calcofluor.                                             <br />    investigate ability mutant constructs rescue         <br /> inviability rad18 deletion, rad18 mutants pRep41MH (Leu&#226;&#171;&#185;) <br />                                                                                <br /> (Craven et al., 1998) transformed rad18&#226;&#171;&#185;/rad18::ura4 <br />                                                                             <br /> diploid strain presence thiamine. Colonies picked induced sporulate growth low nitrogen plates. spores       <br /> digested helicase, plated  thiamine) absence       <br /> leucine, absence uracil leucine. Rescue            <br /> deletion strain indicated growth absence uracil           leucine.                                                                  <br />   <br /> </body></html>