<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Depletion Trypanosome CTR9 Leads Gene <br /> Expression Defects <br /> Benard  Ouna, Benson Nyambega  Theresa Manful&#194;&#164;b, Claudia Helbig, Matilda Males, Abeer Fadda, <br /> Christine Clayton* <br /> Zentrum fu&#194;&#168;r Molekulare Biologie der Universita&#194;&#168;t Heidelberg, DKFZ/ZMBH Alliance, Heidelberg, Germany <br />  <br />  <br />  <br />      Abstract <br />      Paf complex Opisthokonts plants contains subunits: Paf1, Cdc73, Rtf1, Ctr9, Leo1. Mutations  <br />      loss Paf complex subunits shown cause defects histone modification, mRNA polyadenylation,      transcription RNA polymerase RNA polymerase II. investigated trypanosome CTR9, essential      trypanosome survival. results tandem affinity purification suggested trypanosome CTR9 associates      homologues Leo1 Cdc73; genes encoding homologues Rtf1 Paf1  RNAi targeting CTR9 <br />      resulted fold decreases 131 essential mRNAs: included required gene expression <br />      control, encoding subunits RNA polymerases, exoribonucleases target mRNA, RNA helicases <br />      RNA-binding proteins. Simultaneously, genes regions subject chromatin silencing derepressed, <br />      possibly secondary effect loss proteins required silencing, ISWI NLP1. <br />  <br />   Citation: Ouna BA, Nyambega B, Manful T, Helbig C, Males M, et al. (2012) Depletion Trypanosome CTR9 Leads Gene Expression Defects. PLoS 7(4): <br />   e34256. doi:10.1371/journal.pone.0034256 <br />   Editor: Ziyin Li, University Texas-Houston Medical School, United States America <br />   Received January 5, 2012; Accepted February 24, 2012; Published April 20, 2012 <br />   Copyright: &#195;&#376; 2012 Ouna et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Deutsche Forschungsgemeinschaft (German Research Foundation), SFB544 Cl112/9; Bundesministerium fu&#194;&#168;r Bildung <br />   und Forschung (BMBF, Federal Ministry Education Research), Sysmo project &#226;&#8364;&#732; silicon trypanosome&#226;&#8364;&#8482;&#226;&#8364;&#8482;; Deutsches Akademisches Austauschdienst <br />   (German Academic Exchange Service). funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: cclayon@zmbh.uni-heidelberg    Current address: Department Medical Biochemistry, School Medicine, Maseno University, Maseno, Kenya <br />   &#194;&#164;b Current address: Department Biochemistry, Cell Molecular Biology, University Ghana, Legon, Accra, Ghana <br />  <br />  <br />  <br /> Introduction                                                                          IP, complex polymerase-II-transcribed <br />                                                                                       regions yeast genome, association stopped upstream    Paf complex implicated different aspects                   polyadenylation site [9]. human cells yeast, gene expression nuclei yeast mammalian cells                     interaction Paf1 subunit MAP kinase blocked <br /> (Opisthokonts) plants: defects shown influence                       transcription termination specific genes [10]. <br /> transcriptional elongation termination, mRNA processing                          Epigenetic changes associated Paf complex include <br /> export, result epigenetic changes [1]. Saccharomyces                     histone methylation, ubiquitylation acetylation. Rtf1 <br /> cerevisiae, Paf complex consists subunits: Paf1, Cdc73,                   component implicated histone ubiquitylation [11,12], histone <br /> Rtf1, Ctr9, Leo1; human complex contains Ski8 [1].                       [13] DNA methylation [14]; association Rtf1 complex essential yeast: individual                        rest complex required activity [13]. <br /> deletions subunit viable, deletions PAF1                     human Cdc73 homologue parathrombin CTR9 caused decreases 300, increases 900                      shown required specific types histone H3 <br /> mRNAs; half affected similar ways                       methylation [15]; complex shown affect histone <br /> deletions [2]. contrast, mutation human homologues                       acetylation vitro [8]. <br /> Cdc73 (HRPT2/CDC73 parathrombin), loss Ctr9,                              Defects transcription elongation affect choices expression Paf1 associated cancer [1],                    alternative polyadenylation splice sites [16,17,18,19]. parathrombin missense mutant impaired nucleolar localiza-                        possible reason effects pre-RNA secondary <br /> tion promoted growth survival NIH3T3 cells [3].                         structure; second possibility polymerase pauses zebrafish, mutation Rtf1 Leo1 led developmental defects                      synthesis  inefficient, processing site, [4,5], Arabidopisis, complex involved transcrip-                   time site used alternative, efficient <br /> tional control flowering [6].                                                      downstream site synthesised. mutations    complex shown bind RNA polymerase II;                           Paf complex result changes mRNA processing, mutations Cdc73 Rtf1 abolished binding                           clear effects direct. example, yeast <br /> resulted reduced phosphorylation C-terminal domain                       Paf deletion mutants changes poly  site selection (e.g. largest polymerase II subunit [7]. complex stimulates                    [2]); complex directly interacts polyadenylation <br /> elongation chromatin substrates, effect                           factors [7,20]. addition, Leo1 interacts directly RNA vitro <br /> secondary changes histone modification [8]. chromatin                        [21] necessary, sufficient association complex <br />  <br />  <br />        PLoS | www.plosone.org                                                 1                                   April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                             Trypanosome CTR9 <br />  <br />  RNA: association complex chromatin             LEO1, observed multiple effects transcriptome RNA-mediated [21].                                                      CTR9 depletion. <br />    studies examined possible roles Paf1 <br /> complex RNA polymerase transcription.                  Methods <br /> Paf1 required histone K4 methylation rDNA <br /> [22], usually associated silencing. contrast,             Trypanosomes Plasmids group Paf1 complex associated                 RNAi, TbCTR9-specific DNA fragment cloned rDNA, stimulates polymerase elongation [23]: deletions            vector p2T7TA blue [40] using primers 59- <br /> Cdc73, Ctr9 Paf1 resulted polymerase transcription                 GGCACCCTGCAGAATATTCT-39 59-CGGAGGTAA- <br /> defects [24].                                                              ATTTGGCCGAG-39. resulting plasmid linearized    African trypanosomes multiply blood tissue fluids            transfected bloodstream-form trypanosomes mammals  &#226;&#8364;&#732;&#226;&#8364;&#732;bloodstream form&#226;&#8364;&#8482;&#226;&#8364;&#8482;) Tsetse flies,              expressing tet repressor (pHD 1313) T7 RNA polymerase form midgut called &#226;&#8364;&#732;&#226;&#8364;&#732;procyclic               (pHD514). Transfectants selected 10 mg/ml hygromycin <br /> form&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Trypanosomes offer interesting model study           cloned limiting dilution. growth curve shown, <br /> RNA polymerase II transcription. Genes arranged                     RNAi induced adding tetracycline medium, polycistronic transcription units, transcribed              concentration 1 mg/ml. Samples taken start points [25] located regions marked            24 hours RNAi induction days. later experiments, <br /> specific histone modifications [26], obvious                100 ng/ml tetracycline used, similar effects cell <br /> sequence conservation. initiation occurred, elongation            growth. continue 150 kb [26]. Termination sites                      expression C-terminally myc-tagged CTR9, gene                                                                            amplified standard PCR using following oligonucleotides: <br /> enriched specific histone variants followed                                                                            59- GGGCCCATGCAATATATATCGGAACCC&#226;&#8364;&#8220;39 59- <br /> tRNA genes [26]. Individual mRNAs generated                                                                            GTTAACTATATTCCCATCATTAGCGGTGT&#226;&#8364;&#8220;39. frag- <br /> cotranscriptional processing &#226;&#8364;&#8220; addition capped 39nt leader <br />                                                                            ment cloned pHD1700 [41] Apa Hpa sites 59 end trans splicing, 39 polyadenylation [27].                                                                            (pHD2244), expression induced using 100 ng/ml tetracycline. <br /> spliced leader genes, encode capped leader plus                                                                            Expression C-terminally TAP-tagged version (IgG binding <br /> intron, trypanosome polymerase II transcription <br />                                                                            domain calmodulin binding <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>) achieved similarly units monocistronic unique start sites.                                                                            cloning pHD 918 pHD2245 [42,43]. <br /> mRNAs produced, abundances regulated differential degradation [28,29,30]. <br />    Proteins involved trypanosome polymerase II transcription <br />                                                                            Protein Analysis <br />                                                                               determine subcellular location CTR9, approximately <br /> apparatus differ ways Opisthokonts.                                                                            56108 bloodstream cells expressing TAP myc-tagged CTR9 <br /> largest subunit RNA polymerase II lacks characteristic <br />                                                                            collected centrifugation analysed using NP-40 lysis <br /> heptapeptide repeats C-terminus, region                                                                            centrifugation [44]). Proteins analysed Western serine-rich, phosphorylated, required                                                                            blotting, detected enhanced chemiluminescence (Amer- <br /> transcription [31]. Various transcription factors                                                                            sham, Braunschweig, Germany) according manufacturer&#226;&#8364;&#8482;s <br /> identified [32]. identifiable  extremely <br />                                                                            instructions. TAP- myc-tagged CTR9 gave similar results. <br /> diverged) homologues known transcription factors                                                                               tandem affinity purification, expression TAP-tagged <br /> eukaryotes,  example, spliced leader <br />                                                                            protein induced 24 h 161010 cells collected. <br /> promoter-binding complex Kinetoplastid-specific subunit <br />                                                                            TAP purification described [45] modified <br /> [33]; ENF, protein interacts TfIIB, various                [42]. Purified proteins concentrated overnight TCA, <br /> motifs transcription factors, clear           separated 10% uniform SDS-PAGE gel dye <br /> Opisthokont homologue [34]; trypanosome Mediator                   entered gel. gel stained using Colloidal <br /> complex resembles yeast overall shape  judged             Coomassie. section containing proteins sliced equal <br /> electron microscopy) protein primary structures             pieces analysed LC-MS (ZMBH, DKFZ alliance). <br /> [35,36]. <br />    Trans splicing enables trypanosomes produce protein-coding <br />                                                                            RNA Analysis <br /> mRNAs using RNA polymerase  genes encoding                                                                               RNASeq analysis using poly + mRNA extracted <br /> Variant Surface Glycoprotein (VSG) bloodstream forms, <br />                                                                            24 h tetracycline addition; culture used, growth associated genes, transcribed                                                                            inhibition apparent. Methods used polymerase telomeric &#226;&#8364;&#732;&#226;&#8364;&#732;expression sites&#226;&#8364;&#8482;&#226;&#8364;&#8482;, EP             exactly previously described [30]. GPEET procyclins procyclic form,                        RNASeq data CTR9 RNAi mutant compared downstream &#226;&#8364;&#732;&#226;&#8364;&#732;procyclin associated genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; (PAG) transcribed            2 replicates poly + mRNA wild-type cells [30], chromosome-internal locations [37]. cases,                using DESeq [46]. Differentially expressed genes polymerase transcription subject developmental regulation,         p-adjusted-value ,0.05. Comparisons datasets suppresses inappropriate expression; lest partially       follows. Transcription start stop sites annotated <br /> chromatin-mediated [38], specific factors involved         manually, scrutiny histone modifications shown [36]. Transcription EP GPEET procyclin genes           TritrypDB published [26]. ORFs times active procyclic forms bloodstream forms         nearest start stop sites designated Excel <br /> [36,39]; addition, procyclin RNAs rapidly degraded          tables, proximal ORF annotated &#226;&#8364;&#732;&#226;&#8364;&#732;&#226;&#8364;&#8220;1&#226;&#8364;&#8482;&#226;&#8364;&#8482; distal <br /> bloodstream forms stable procyclic forms (reviewed [29]).        &#226;&#8364;&#732;&#226;&#8364;&#732;&#226;&#8364;&#8220;5&#226;&#8364;&#8482;&#226;&#8364;&#8482;. Data developmental regulation extracted <br />    paper, investigated function CTR9                  [47]. determine genes essential growth, <br /> trypanosomes. evidence interaction                    used published RNA interference sequencing data [48]. trypanosome CTR9, CDC73, putative homologue                       determine fold effect RNAi published screen,  <br />  <br />       PLoS | www.plosone.org                                       2                            April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                               Trypanosome CTR9 <br />  <br />  <br /> reads day 3, day 6 tetracycline induction RNAi        BLASTp settings filters, matches poor coverage <br /> [48] divided number reads day 0. ORFs showing             checked reciprocal BLAST. results negative, decrease 4-fold, coupled statistical significance        increased sensitivity searching individual genomes (&#226;&#8364;&#732;&#226;&#8364;&#732;TRUE&#226;&#8364;&#8482;&#226;&#8364;&#8482; judgement Table provided [48]) judged              Ctr9, Cdc73 Leo1, Leo1 domain (Figure 1B). essential normal growth. Northern blots using           Results (Table 2) suggest pentameric complex standard methods; reverse transcription PCRs RNA                    present plants,   unikonts (Opisthokonts <br /> detection described [30] using primers listed        Amoebozoa). Cdc73, Leo1 Ctr9 conserved <br /> Table S2.                                                                   widely eukaryotic evolution, sequences resembling <br />                                                                             subunits Trichomonas Giardia. <br /> Results                                                                     Evidently poor conservation compromised                                                                             searches, does Cdc73, Ctr9 Leo1 Trypanosome CTR9 Essential Protein                           widely distributed Paf1 Rtf1. <br /> purifies Homologues Leo1 Cdc73 <br />    sequence encoded Tb927.3.3220 identified                 Depletion Ctr9 Results Increased mRNA Ctr9 homologue reciprocal BLASTp analysis. Like Ctr9s,             Procyclin Loci contains multiple TPR repeats (Figure 1A); identities                  high-throughput RNAi screen CTR9, LEO1 CDC73 <br /> human, yeast plant Ctr9s 21%, 18% 19%                           essential normal growth respectively, similarities spread sequence              bloodstream procyclic trypanosomes [48]. (Figure S1). identify interaction partners, expressed               RNAi targeting CTR9 bloodstream forms resulted version bearing C-terminal tandem affinity purification (TAP)             growth arrest 1 day tetracycline induction (Figure 1D). tag bloodstream T. brucei. tagged CTR9 subjected              examine reason growth arrest, analysed steps affinity purification proteins present          poly +transcriptome cells 24 h tetracycline addition. <br /> entire preparation identified mass spectrometry.                  Trypanosome polymerase II transcription units contain <br /> potential interaction partners revealed (Table 1).                multiple open reading frames (ORFs). CTR9 required <br /> notably, potential homologues Cdc73                           transcription processivity, expect depletion <br /> (Tb927.11.10230) Leo1 (Tb927.9.12900).                  result preferential depletion mRNAs proteins unknown function (Tb927.7.4030                          ends transcription units. seen - <br /> Tb927.3.5070) recognised functional domains                   increases decreases scattered randomly homologues non-kinetoplastid eukaryotes;               transcription units (Table S1 Figure 2A). Table 3 lists investigated. did check                 47 ORFs greatest increases RNA <br /> tagged CTR9 functional, tandem affinity purification           representation CTR9 RNAi. dramatic change <br /> relatively stringent procedure, additional interaction partners          increase expression ORFs procyclin missed.                                                     transcription units (Table 3). EP GPEET <br />    Trypanosome CDC73 shows 22%, 21%, 25% identity                  procyclin genes, downstream PAGs affected human, yeast plant proteins, conservation                 (Table 3 Figure 2B). remaining ORFs, C-terminus (Figure S2). peptides              half located near rRNA loci, transcriptional detected, likely genuine interaction partner.       stop sites (Table 3 Table S1). phenomenon  contrast, overall identities LEO1 low: 12%,              restricted relatively chromosomal locations. 17% 15% respectively; identification relies exclusively          RNA generally increased near stop sites, presence Leo1 domain (E value 1.6 e-5) positions           examined 150 ORFs immediately upstream points 217 257 (Figure 1B Figure S3). searched                      polymerase II transcription units converge.  trypanosome genome homologues remaining subunits                 median change transcript abundance 2-fold increase: 9 unable  look association            RNAs decreased 3-fold, 48 increased CTR9 RNA polymerase II, precipitated CTR9-TAP                    3 fold. 10 kb convergence regions, effects examined precipitate presence largest                individual mRNAs   means uniform: subunit: association seen (Figure S4).                               mRNA neighbour went  Figure 2A <br />    Extrapolating systems, expected CTR9            shows effect RNAi transcription units: nucleus. examined protein cell                       clear increase end,  <br /> fractionation. control, XRND, retained nucleus                obvious difference organization trypanothione reductase cytoplasmic                  termination points showed increases fraction (Figure 1C). contrast, 100 kDa C-terminally myc-            didn&#226;&#8364;&#8482;t. <br /> tagged CTR9 protein cytoplasm nucleus                    looked CTR9 depletion changed <br /> (Figure 1C). repeat experiments myc- TAP                  RNA processing patterns. Predominant splice <span id='am-27' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span> sites <br /> tagged proteins, using half tetracycline, signals           mapped [30] 4135 genes 2 reads strong protein                    site CTR9 RNAi strain. 37% sites <br /> compartments, slightly nucleus  shown).             database predominant sites used possible localization tagged protein         determined previously basis collection WT <br /> influenced presence tag.                                      RNASeq data [30]. similar ratio obtained                                                                             wild-type sample compared database, <br /> Conservation PAF Complex Subunits Evolution                           concluded CTR9 knockdown does significantly affect <br />   investigate evolutionary conservation PAF                   use splice acceptor sites. <br /> complex, searched genomes representing major                     verify RNASeq results, probed Northern blots. eukaryotic branches  Rhizaria), using yeast, human           increases EP (Figure 2B), GPEET PAG2 mRNAs (Figure 2C) trypanosome sequences queries. used standard NCBI                  confirmed, increase locus Tb11.01.4120,  <br />  <br />       PLoS | www.plosone.org                                        3                             April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                                                Trypanosome CTR9 <br />  <br />  <br />  <br />                                                                                                                                                      901 Tb <br />                                                                                                                                                      1076 Sc <br />                                                                                                                                                          1173 Hs <br />                                                                                                                                                        1091                                                                                                                                                SM00028 <br />  <br />   B <br />       233    L   F   V   S   N     R     V   R   W   S   D   G   S   T      T                     L   R    V   G   S   D   S   F   P   L   TbLEO1 <br />       230    F   K   E   S   N   T   Q     V   Q   W   S   D   G   T   Y      S                     L   K    V   G   E   E   C   T   D     ScLeo1 <br />       236      K   E   S   N   T   R   L   V   T   W   S   D   G   S   S      H                     L   Y      G   N   E   V   L   E     DdLeo1 <br />       433      K   E   S   N     R     V   K   W   S   D   G   S   M      S                     L   H    L   G   N   E   V   F   D   V   HsLeo1 <br />       211    L   R   E   S   N   S   K   L   V   Q   W   E   D   G   T   F      S                     M   F    V   G   K   E     L   T   L   PiLeo1 <br />       273      R   E   S   N   T   R   L   V   K   W   S   D   G   S   Y      T                     L   H    V   G   Q   E   V   L   E   V   TpLeo1 <br />       87     -   L   E   S   N     K       E   W   D   D   G   S   Y      H                     L   M    V   G   D   Q   F   F   K   L   TtLeo1 <br />       156    E     E   S   N   T   R   V   V   Q   Y   E   D   G   S   Y      Q                     V   F    V   G   N   E   V     L   N   NgLeo1 <br />  <br />  <br />  <br />  <br />   C                                                                  D                     10000 <br />                                                                              Cumulative cell number <br />  <br />  <br />  <br />  <br />                  T N         C WT                                                                                      -tet <br />                                              CTR9-myc                                                 1000 <br />                                                                                                                        +tet <br />                                              XRND <br />                                              TRY                                                       100 <br />  <br />  <br />                                                                                                           10 <br />  <br />  <br />                                                                                                           1 <br />                                                                                                                0               1               2            3 <br />                                                                                                                                time [days] <br />  <br /> Figure 1. Properties CTR9 complex components.  locations TPR repeats (SMART SM00028) trypanosome (Tb), Saccharomyces <br /> cerevisiae (Sc), human (Hs) Arabidopsis thaliana   Ctr9 homologues. maps taken Interpro scan web site. Domains shown thickened bars. B. conserved portion Leo1 domain. alignment using MegAlign portion DNAStar package; details Figure S3. Identical amino acids underlain black functionally conserved amino acids underlain grey. Dd: Dictyostelium <br /> disoideum; Pi: Phytophthora infestans &#226;&#8364;&#8220; Tp: Thalassiosira pseudonana; Tt: Tetrahymena thermophila; Ng: Naegleria gruberi. C. Myc-tagged CTR9 <br /> fractionates nucleus cytoplasm. Cells expressing CTR9-myc lysed NP40, centrifuged, extracts subjected Western blotting using antibodies myc (Santa Cruz), XRND (nuclear marker) trypanothione reductase (cytoplasmic marker). T: total; N: nuclear <br /> fraction; C: cytoplasmic fraction. D. Effect RNAi CTR9. separate bloodstream trypanosome lines (represented different symbols) incubated 100 ng/ml tetracycline. starting cell density 26105/ml cultures diluted level daily required. <br /> doi:10.1371/journal.pone.0034256.g001 <br />  <br />  <br /> ORF near end transcription unit showed                                                       assessed degradation rate EP procyclin <br /> 5-fold increase expression RNASeq (Figure 2C). Attempts                                                       mRNA. Results shown Figure 2D. wild type cells confirm regulation mRNAs polymerase                                                          mRNA half-life than10 minutes [49]. CTR9 <br /> II transcription units unsuccessful,                                                       RNAi, onset EP mRNA degradation reproducibly <br /> low abundance (asterisks, Table 3). CTR9-RNAi-specific                                                            delayed (Figure 2D), apparent 2-3-fold increase half-life <br /> reads concentrated segments non-coding regions covered                                                         itself, sufficient explain 9-fold increase parts ORFs; investigated  Probing                                                    mRNA. concluded CTR9 RNAi probably Northern blot spliced leader probe, detect mature                                                   causes failure stage-specific transcriptional suppression mRNAs, did reveal global changes mRNA abundance                                                             procyclin loci. <br />  shown).                                                                                                              trypanosomes, polyadenylation trans splicing coupled <br />    increase EP GPEET procyclin mRNAs                                                            [50,51]. effects CTR9 depletion RNA processing, likely transcriptional effect, according RNASeq                                                   looked changes use splice acceptor sites, results, entire transcription units including PAGs                                                        significant difference  Methods). <br /> affected: effects RNA processing degradation unlikely affect gene similar extents. check  <br />  <br />  <br />        PLoS | www.plosone.org                                                                                  4                                    April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                              Trypanosome CTR9 <br />  <br />  <br />  <br />  Table 1. Proteins purified CTR9-TAP. <br />  <br />  <br />  Identified Proteins                                                                                  Accession Number                      peptides <br />  <br />  Paf complex subunits <br />  CTR9                                                                                                 Tb927.3.3220                          48 <br />  LEO1                                                                                                 Tb927.9.12900                         6 <br />  CDC73                                                                                                Tb927.11.10230                        2 <br />  Novel proteins <br />  hypothetical protein, conserved                                                                      Tb927.7.4030                          4 <br />  hypothetical protein, conserved                                                                      Tb927.3.5070                          4 <br />  Possible contaminants <br />  calmodulin                                                                                           Tb927.11.13020                        1 <br />  60S ribosomal protein L17                                                                            Tb927.11.4820                         2 <br />  microtubule-associated protein 2                                                                     Tb10.v4.0052                          4 <br />  polyubiquitin                                                                                        Tb927.11.9920                         2 <br />  hypothetical protein, conserved                                                                      Tb11.0840                             2 <br />  ribosomal protein L3                                                                                 Tb927.4.1790                          2 <br />  60S ribosomal protein L12                                                                            Tb927.9.14000                         4 <br />  glyceraldehyde 3-phosphate dehydrogenase, glycosomal                                                 Tb927.6.4280                          6 <br />  glycerol kinase, glycosomal                                                                          Tb927.9.12550                         5 <br />  retrotransposon hot spot protein                                                                     Tb11.0050                             2 <br />  elongation factor 1-alpha,EF-1-alpha                                                                 Tb927.10.2100                         6 <br />  alpha tubulin                                                                                        Tb927.1.2340                          20 <br />  beta tubulin                                                                                         Tb927.1.2330                          23 <br />  heat shock protein 70                                                                                Tb927.11.11330                        3 <br />  <br />  Proteins denatured, separated SDS-PAGE gel section containing proteins analysed mass spectrometry. numbers different peptides <br />  detected indicated. Possible contaminants judged appearance  unrelated purifications. <br />  doi:10.1371/journal.pone.0034256.t001 <br />  <br />  <br /> Depletion Ctr9 Reduces Expression Genes                                 quick glance,  sufficient explain CTR9 Essential Control Gene Expression                                  RNAi lethal: fewer 131 reduced mRNAs    examined mRNAs reduced CTR9                             essential normal growth. examples essential genes <br /> RNAi. 349 decreased fold (Table S1).                       20-fold RNA decreases shown Table 4. correlation effect CTR9 depletion                        Multiple genes encode proteins involved gene expression developmental regulation half-life  shown).                   copy mRNA cell: largest RNA <br />                                                                                    polymerase II subunit second largest polymerase  <br />  <br />  Table 2. Putative homologues Paf complex subunits various organisms. <br />  <br />  <br />  Group/Kingdom                 Species                      Ctr9                   Cdc73              Leo1                 Paf1                  Rtf1 <br />  <br />  Plantae                       Arabidopsis thaliana         B5X0I6_ARATH B5X0I At3g22590              At5g61156            At1g79730             At1g61040 <br />  Opisthokonts                  Saccharomyces cerevsiae      NP_014496.2            NP_013522.1        NP_014766.1          NP_009838.1           NP_011270.1 <br />  Opisthokonts                  Schizosaccharomyces pombe    NP_594620.1            NP_595891.1        NP_596263.1          NP_593451.1           NP_595507.1 <br />  Opisthokonts                  Homo sapiens                 NP_055448.1|           NP_078805.3        NP_620147.1          NP_061961.2           NP_055953.3 <br />  Amoebozoa                     Dictyostelium discoideum     XP_642310.1            XP_644100.3        XP_637916.1          XP_645948.1           XP_637691.1 <br />  Amoebozoa                     Entamoeba histolytica        XP_650475.1            XP_648719.1 <br />  Alveolata                     Tetrahymena thermophila      EAR82060.1         EAR84002.1         EAR89919.2 <br />                                                              <br />  Alveolata                     Plasmodium falciparum        &#226;&#8364;&#8220;                      XP_001351765.1     &#226;&#8364;&#8220; <br />  Stramenopila                  Thalassiosira pseudonana     XP_002287483.1         XP_002287950.1     XP_002289477.1 <br />  Heterolobosea                 Naegleria gruberi            XP_002679971.1         XP_002683436.1                          XP_002670362.1 <br />  Euglenozoa                    Trypanosoma brucei           Tb927.3.3220           Tb927.11.10230     Tb927.9.12900 <br />  <br />  homologues Trichomonas Giardia. details Table S3. <br />  doi:10.1371/journal.pone.0034256.t002 <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                             5                                   April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                          Trypanosome CTR9 <br />  <br />  <br />  <br />                                     10 <br />  <br />  <br />  <br />                 rpkm CTR9 RNAi / WT       1 <br />  <br />  <br />  <br />  <br />                                       0.1 <br />  <br />  <br />                                               11.01.5320                                   927.10.4980 <br />                              0.01 <br />  <br /> B <br />      RNAi      -                      + <br />                                                 C <br />     CTR9 <br />                                                     WT -      + <br />                                                                   WT -     +                     WT        -   +      WT    -      + <br />         EP <br />                                                                                                                                        mRNA <br />     HISH4 <br />       TUB <br />                                                                                                                                        rRNA <br />  <br />      rRNA <br />                                                        GPEET            PAG2                      11.01.4120               ISWI <br />  <br />  <br />  <br /> D <br />                                                                                                 100 <br />                                            WT               RNAi +tet <br />                                                                                                                                         -tet <br />                                                                                %RNA remaining <br />  <br />  <br />  <br />  <br /> min. Act D             0                  0 10 20          0 0 10 20                             80 <br />                                                                                                                                         +tet <br />          EP                                                                                      60 <br />                                                                                                  40 <br />                                                                                                           &lt;8 min <br />        SRP                                                                                       20                         15-20 min <br />  <br />                                                                                                   0 <br />                                                                                                       0            10         20               30 <br />                                                                                                                      time [min] <br />  <br /> Figure 2. Effects CTR9 RNAi transcriptome.  Effect CTR9 RNAi RNAseq representation, poly + RNA, randomly- <br /> selected transcription units starting loci 11.01.5320 927.10.4980. units divergent start convergent stop sites; Figure <br /> transcription left right. values Y-axis, rpkm ORF 24 h RNAi divided rpkm wild-type cells [30]. bar represents annotated ORF. B. Northern blot confirmation RNASeq results EP, Histone H4 (HISH4), tubulin (TUB) ORFs, using <br /> new RNA preparations. rRNA control scan methylene blue-stained blot. RNAi induced 24 h. C. Northern blot confirmation RNASeq results GPEET, PAG2, Tb11.01.4120 ISWI ORFs, using new RNA preparations. rRNA control scan ethidium-bromide <br /> stained gel. D. CTR9 RNAi effect EP procyclin mRNA abundance degradation. left hand panel shows Northern blot graph <br /> includes results different experiments, time points 0, 10 20 min time points 0, 10 30 min. Results normalised 7SL RNA signal recognition particle (SRP). half-life EP mRNA 8 min, previously observed. <br /> doi:10.1371/journal.pone.0034256.g002 <br />  <br /> subunit; TFIIB-interacting protein ENF [34]; compo-                                           components cytoskeleton reduced, nents mRNA degradation machinery, NOT1                                                     involved vesicular transport involved DNA <br /> XRNA; poly  binding protein 2 translation factors.                                         replication chromosome maintenance. Correspondingly, targets potentially involved gene                                            analysis term enrichment showed highly significant <br /> expression: 7 RNA helicases, including DHH1; zinc finger                                              representation (P values 10&#226;&#8364;&#8220;10) gene <br /> proteins ZC3H22, ZC3H28, ZC3H32, ZC3H41, RNA-                                                     products categories related <span id='am-13' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span> <span id='am-21' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleoside</span> <br /> binding domain proteins DRBD12 DRBD17, nearly                                             binding hydrolysis, motor activity; cytoskeletal pumilio domain proteins. mRNAs encoding 40                                            categories enriched P values 10&#226;&#8364;&#8220;5   <br />  <br />        PLoS | www.plosone.org                                                                6                                April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                                  Trypanosome CTR9 <br />  <br />  <br /> Table 3. Upregulation transcript abundance CTR9 RNAi. <br />  <br />  <br /> geneID                                                                                              rpkm                CTR9/WT                    features <br />  <br /> Procyclin locus <br /> Tb927.6.520                             EP3-2 procyclin                                             36.4                17                         pol Tb927.6.510                             GPEET2 procyclin                                            31.3                14                         pol Tb927.10.10240                          PAG1                                                        2.5                 48                         pol Tb927.10.10220                          PAG2                                                        1.9                 57                         pol Tb927.6.530                             PAG3                                                        12.2                11                         pol Tb927.10.10210                          PAG4                                                        2.6                 31                         pol Tb927.10.10230                          PAG5                                                        2.3                 37                         pol Near rRNA locus <br /> Tb09.v2.0450                            Unlikely ORF                                                2.0                 17.0                       rRNA locus <br /> Tb927.2.1470                            hypothetical conserved                                      3.6                 10.0                       0 kb, rRNA <br /> Tb927.1.3750                            hypothetical conserved                                      8.7                 7.7                        0 kb, rRNA <br /> Tb927.3.3430                            hypothetical conserved                                      7.4                 7.6                        0 kb, rRNA Tb927.5.4020                            hypothetical                                                4.0                 13**                       3 kb <br /> Tb927.8.6570                            hypothetical conserved                                      3.4                 10                         0 kb <br /> Tb927.8.510                             hypothetical                                                1.1                 21*                        6 kb <br /> Tb927.10.4390                           hypothetical                                                3.6                 7.8                        3 kb <br /> Tb927.10.4380                           hypothetical                                                0.8                 39*                        2 kb, ESAG <br /> Tb927.10.12270                          hypothetical conserved                                      4.1                 14*                        1 kb <br /> Tb927.10.1760                           hypothetical                                                1.7                 20*                        0 kb <br /> Tb927.11.9380                           hypothetical                                                1.1                 14*                        0 kb <br /> Tb927.1.2020                            hypothetical                                                3.3                 11                         0 kb <br /> Tb927.11.11990                          hypothetical                                                1.3                 11                         0 kb <br /> Tb927.8.480                             phosphatase                                                 26.6                9.8                        0 kb <br /> Tb927.2.3330                            hypothetical                                                5.3                 8.7                        0 kb <br /> Tb927.7.310                             hypothetical conserved                                      8.9                 8.1                        0 kb <br /> Tb927.10.9550                           hypothetical                                                0.6                 8.4                        6 kb <br /> Tb927.8.5930                            hypothetical                                                0.3                 9.4                        stop/start <br /> Tb927.3.650                             hypothetical conserved                                      25.8                8.1                        11 kb <br /> Tb927.7.2430                            Long poly  tracts                                        0.2                 16.9*                      repetitive <br /> Tb927.11.4600                           hypothetical                                                1.0                 18.5                       Tb927.10.10040                          hypothetical                                                4.1                 13.3                       Tb927.3.2880                            hypothetical conserved                                      7.6                 11.7                       Tb927.10.9030                           hypothetical conserved                                      10.7                9.7                        Tb927.3.3570                            hypothetical conserved                                      10.8                9.1                        Tb927.11.7260                           hypothetical conserved                                      7.8                 8.7                        Tb927.6.4790                            hypothetical conserved                                      33.9                8.2                        Tb11.v4.0003                            hypothetical                                                2.3                 8.2                        Tb07.26A24.210                          hypothetical                                                4.7                 7.8                        Tb927.3.2370                            hypothetical conserved                                      36.0                7.8                        Tb927.10.260                            hypothetical conserved                                      29.4                7.7                        Tb927.10.7720                           hypothetical conserved                                      35.7                7.6                        Tb927.11.12330                          hypothetical conserved                                      24.5                7.5                        Tb927.2.2070                            hypothetical                                                3.5                 8.6 <br /> Tb927.6.4570                            hypothetical conserved                                      13.8                8.6                        start <br />  representative procyclin locus ORFs shown, 36 ORFs showing greatest increases RNA CTR9 RNAi. &#226;&#8364;&#732;&#226;&#8364;&#732;rpkm&#226;&#8364;&#8482;&#226;&#8364;&#8482; reads kilobase million reads seen poly +RNA normal (wild-type) cells, taken [30]; median ORFs 30.6. &#226;&#8364;&#732;&#226;&#8364;&#732;CTR9/WT&#226;&#8364;&#8482;&#226;&#8364;&#8482; shows fold <br /> change rpkm RNAi (2 significant figures); * signal seen Northern blot using ORF probe;  RNA detected reverse transcription PCR <br /> targeting ORF. &#226;&#8364;&#732;&#226;&#8364;&#732;Features&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates transcription polymerase (pol , distance 39 end locus mRNA convergent transcription stop area <br /> &#226;&#8364;&#8220; &#226;&#8364;&#732;&#226;&#8364;&#732;0 kb&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates ORF transcription unit. &#226;&#8364;&#732;&#226;&#8364;&#732;0 kb, rRNA&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicats ORF pol II transcription unit, rRNA locus read opposite direction. &#226;&#8364;&#732;&#226;&#8364;&#732;2 kb, ESAG&#226;&#8364;&#8482;&#226;&#8364;&#8482; gene upstream ESAG. rpkm 20 corresponds 1 mRNA cell, difficult detect Northern blotting. <br /> doi:10.1371/journal.pone.0034256.t003 <br />  <br />  <br />       PLoS | www.plosone.org                                                         7                                      April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                         Trypanosome CTR9 <br />  <br />  <br />  <br /> Table 4. Essential genes expression repressed CTR9 depletion. <br />  <br />  <br /> geneID                                  gene description                                                  rpkm            RNAi                 BS3             BS6 <br />  <br /> Cytoskeleton <br /> Tb927.10.14470                          IFT140                                                            47.9            0.03                 0.02            0.00 <br /> Tb927.10.2640                           IFT81                                                             114             0.04                 0.00            0.00 <br /> Tb927.10.1170                           IFT172                                                            63.8            0.01                 0.13            0.23 <br /> Tb927.5.3030                            IFT122B                                                           27.7            0.04                 0.06            0.25 <br /> Tb927.6.3150                            Hydin,flagellar component                                         19.2            0.04                 0.05            0.01 <br /> Tb927.3.5310                            paraflagellar rod protein                                         72.6            0.04                 0.10            0.08 <br /> Tb927.11.15100                          Paraflagellar rod Tb5.20                                          142             0.03                 0.00            0.00 <br /> Tb927.11.3290                           p166                                                              23.9            0.04                 0.45*           0.17 <br /> Tb927.4.870                             dynein heavy chain                                                19.3            0.02                 0.01            0.01 <br /> Tb927.11.3250                           dynein heavy chain                                                64.5            0.01                 0.01            0.02 <br /> Tb927.11.11220                          dynein heavy chain                                                16.0            0.04                 0.03            0.04 <br /> Tb927.11.8160                           dynein heavy chain                                                24.9            0.02                 0.06            0.04 <br /> Tb927.3.930                             dynein heavy chain                                                63.3            0.03                 0.03            0.05 <br /> Tb927.2.5270                            dynein heavy chain                                                20.4            0.02                 0.07            0.08 <br /> Tb927.11.2430                           dynein heavy chain (DHC1b)                                        36.1            0.03                 0.05            0.08 <br /> Tb927.10.5350                           dynein heavy chain                                                26.9            0.03                 0.27*           0.13 <br /> Tb927.8.3250                            dynein heavy chain                                                28.1            0.02                 0.06            0.18 <br /> Tb927.7.920                             dynein heavy chain                                                27.3            0.04                 0.18            0.21 <br /> Tb927.5.2090                            kinesin                                                           41.7            0.04                 0.16            0.20 <br /> Gene expression <span id='am-6' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; DNA replication <br /> Tb927.10.1510                           NOT1                                                              36.0            0.01                 0.18            0.08 <br /> Tb927.7.4900                            5&#226;&#8364;&#8482;-3&#226;&#8364;&#8482; exonuclease XRNA                                            14.8            0.04                 0.15            0.17 <br /> Tb927.10.12660                          RNA binding protein PUF2                                          7.0             0.04                 0.15            0.00 <br /> Tb927.9.12510                           DEAD/H RNA helicase                                               58.2            0.05                 0.14            0.05 <br /> Tb927.3.2600                            DEAD/H RNA helicase                                               53.8            0.04                 0.12            0.16 <br /> Tb927.10.10340                          SMC2                                                              34.7            0.03                 0.66*           0.14 <br /> Intracellular protein transport Tb927.10.6050                           clathrin heavy chain (CHC)                                        135             0.01                 0.01            0.00 <br /> Tb927.10.2900                           importin beta-1 subunit                                           38.6            0.03                 0.10            0.02 <br /> Tb927.2.4230                            NUP-1, nuclear pore                                               72.6            0.01                 0.17            0.07 <br /> Tb927.10.2880                           calcium channel protein                                           35.7            0.02                 0.08            0.04 <br /> Tb927.11.1100                           Clan CA cysteine peptidase                                        48.6            0.01                 0.07            0.16 <br /> Tb927.7.2170                            hypothetical protein                                              46.1            0.04                 0.10            0.00 <br /> Tb927.7.3550                            hypothetical protein                                              91.3            0.01                 0.01            0.00 <br /> Tb927.8.4780                            hypothetical protein                                              33.6            0.02                 0.01            0.00 <br /> Tb927.8.2820                            hypothetical protein                                              1801            0.01                 0.31*           0.00 <br /> Tb927.9.7690                            hypothetical protein                                              9.3             0.04                 0.07            0.01 <br /> Tb927.10.870                            hypothetical protein                                              32.4            0.02                 0.08            0.02 <br /> Tb927.9.1750                            hypothetical protein                                              11.0            0.02                 0.05            0.02 <br /> Tb927.3.2050                            hypothetical protein                                              20.3            0.03                 0.92*           0.04 <br /> Tb927.10.15750                          hypothetical protein                                              114             0.01                 0.17            0.05 <br /> Tb927.1.4310                            hypothetical protein                                              23.6            0.01                 0.04            0.12 <br /> Tb927.9.13440                           hypothetical protein                                              25.9            0.02                 0.20            0.18 <br /> Tb927.7.3560                            hypothetical protein                                              29.1            0.03                 0.02            0.23* <br />  <br /> Genes selected following criteria:   mRNA decreased 20-fold CTR9 RNAi (b) significant decrease 5-fold representation RNAi library, using bloodstream forms 3 6 days tetracycline addition [48]. &#226;&#8364;&#732;&#226;&#8364;&#732;rpkm&#226;&#8364;&#8482;&#226;&#8364;&#8482; value wild-type cells RNASeq: rpkm value 20 equivalent mRNA cell [30]. &#226;&#8364;&#732;&#226;&#8364;&#732;RNAi&#226;&#8364;&#8482;&#226;&#8364;&#8482; rpkm CTR9 RNAi divided wild-type. &#226;&#8364;&#732;&#226;&#8364;&#732;BS3&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates representation gene RITSeq analysis bloodstream forms, day 3 RNAi induction, divided representation tetracycline;  significant [48]. &#226;&#8364;&#732;&#226;&#8364;&#732;BS6&#226;&#8364;&#8482;&#226;&#8364;&#8482; 6 days tetracycline. <br /> doi:10.1371/journal.pone.0034256.t004 <br />  <br />  <br />  <br />  <br />       PLoS | www.plosone.org                                                     8                                    April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                  Trypanosome CTR9 <br />  <br />  <br /> shown). Scrutiny list suggested explanation             8 phloretin treatment. CTR9 RNAi results deregulation procyclin loci: mRNAs encoding ISWI                resemble seen lethal knock ZC3H11 (D. <br /> (Tb927.2.1810) NLP1 (Tb927.10.5450), repressed 8-                 Droll, ZMBH, unpublished). suggests transcriptome <br /> fold 13-fold, respectively (Table S1 Figure 2B). RNAi              changes CTR9 RNAi simply caused growth <br /> targeting genes deregulates silent expression sites,       inhibition. procyclin promoters silent regions [52,53].                     studying time course transcriptome changes Paf1 <br />                                                                            depletion yeast, Penheiter et al. [2] 16 RNAs <br /> Discussion                                                                 decreased included 11 essential genes; half                                                                            involved nucleolar structure processing rRNA [2].    shown trypanosome CTR9 purifies                    possible subsequent transcriptome <br /> homologues Leo1 Cdc73 affinity purifica-                changes secondary rRNA processing defects. trypano- <br /> tion steps. possible trypanosomes possess           somes, did study time course effects, certainly <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;PAF&#226;&#8364;&#8482;&#226;&#8364;&#8482; complex 3 subunits. subunits,            24 h, CTR9 depletion decreased mRNAs Paf1 Rtf1, appear missing, limited phylogenetic            disentangling primary secondary effects possible. analysis suggested restricted mainly plants          particular, changes amounts mRNAs unikonts. possible explanations. Paf1             encoding proteins required mRNA synthesis Rtf1 equivalents absent eukaryotes, proteins           control.  mRNAs encoding components RNA polymerases evident sequence similarity able substitute                  II III decreased considerably copy cell: <br /> functionally Rtf1 Paf1. support                   precludes vivo analysis specific roles CTR9 supposition, yeast survive just Ctr9, Leo1 Cdc73:             polymerase II elongation.  expression NOT1 - core <br /> deletion Rtf1 Leo1 led obvious defects,            component major deadenylation complex - XRNA, deletion Paf1 severe effects,               59-39 exoribonuclease important degradation partially abrogated deletion Rtf1 Leo1 [54].           relatively unstable mRNAs, abolished. essential alternative, noted Introduction, functional                  proteins required mRNA degradation. Transcripts <br /> equivalence absence sequence similarity               encoding RNA-binding proteins decreased; <br /> known proteins involved trypanosome transcription.             knock effects occurred proteins regulate did identify additional candidate CTR9 binding                      mRNA degradation lost mRNA degradation pathways <br /> partners, Tb927.7.4030 Tb927.3.5070,                      active. Finally, increase observed verified association. published RNAi screen,                    polymerase transcribed mRNAs likely Tb927.7.4030 essential bloodstream                  reduction chromatin silencing, consequent disappearance <br /> differentiating forms, depletion Tb927.3.5070 inhibited             ISWI NLP1. <br /> growth [48].                                                                  C-terminally tagged CTR9 cytoplasm <br />    affinity purification tagged CTR9, did          nucleus. tag influenced purification subunits RNA polymerase II, proteins             result, localisation accordance PSORTII <br /> involved chromatin modification RNA processing.                 prediction: localization percentages neighbours 22% <br /> interactions survived purification procedure,        nuclear 61% cytoplasmic trypanosome CTR9. contrast, impaired presence tag.                  yeast mammalian proteins yielded 90%, Drosophila 48%,  possible                nuclear scores. possible trypanosome CTR9 <br /> interactions seen systems occur          functions compartments, regulated nucleo- <br /> trypanosomes. example, yeast [54,55] human [8] RNA                 cytoplasmic shuttling. <br /> polymerase II shown coimmunoprecipitate                      Overall, results suggest trypanosome CTR9 various Paf complex components, copurify                   function complex CDC73, LEO1, complex, association human Paf complex RNA             proteins. complex components clearly essential func- <br /> polymerase II depends presence PAF1 subunit [8].             tions, likely gene expression cytoplasm, Trypanosomes lack obvious Paf1 homologue,                    nucleus,  precise natures functions remain <br /> human  interaction seen polymerase II            unclear. <br /> purified CTR9, CDC73 LEO1 [8]. probably, interaction trypanosome CTR9/CDC73/LEO1                     Supporting Information expected   evidence trypanosome CTR9/CDC73/LEO1 complex                   Figure S1 Alignment Ctr9 homologues, using <br /> involved polymerase II elongation.                                      MegAlign (DNAStar). Residues identical    Depletion CTR9 caused changes transcriptome.             majority consensus highlighted black, residues simply consequence slowed              consensus group (consensus #1) grey. Groups                                                                            f-hydrophobic; p-polar, acidic, b-basic. Organisms  Tb- T. <br /> growth, compared regulated genes                                                                            brucei,  Arabidopsis thaliana, NP_178674.6; Dd- Dictyostelium previously detected, microarray, growth- <br />                                                                            discoideum XP_642310; Eh- Entamoeba histolytica EAL45089; Hs- <br /> inhibitory treatments: inhibition glucose transport using <br />                                                                            Homo sapiens NP_055448; Ng- Naegleria gruberi XP_002679971; Sc- <br /> phloretin [56], RNAi targeting RBP10 [57]. 256 <br />                                                                            Saccharomyces cerevisiae Ctr9; Sp- Schizosaccharomyces pombe <br /> ORFs showed greater 3-fold decreases CTR9 <br />                                                                            CAA15833. <br /> RNAi, 4 decreased phloretin  10                                                                            (PDF) <br /> 1.5-fold increased). Similarly, just 19 theses 256 ORFs 1.5-2-fold decreased RBP10 RNAi increased             Figure S2 Alignment Cdc73 homologues, using <br /> 1.6-fold. 866 CTR9 RNAi regulated genes, 10                MegAlign (DNAStar). Details Figure S1. Organisms  <br /> (including EP GPEET procyclin genes, importantly,              Tb- T. brucei, Sc- Saccharomyces cerevisiae; Sp- Schizosaccharomyces pombe <br /> PAGs) 1.5-fold increased RBP10 RNAi,          NP_595891.1; Hs- Homo sapiens NP_078805.3; Dd- Dictyostelium <br />  <br />  <br />       PLoS | www.plosone.org                                       9                               April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                          Trypanosome CTR9 <br />  <br />  <br /> discoideum XP_644100.3; Ng- Naegleria gruberi XP_002683436.1; Pf-                        poly +RNA cells 1 day RNAi targeting CTR9. <br /> Plasmodium falciparum XP_001351765.1; Tt- Tetrahymena thermophila                        reads kilobase million reads ORF, divided EAR84002.1; Pi- Phytophthora infestans XP_002903756.1; Eh-                               ORF copy number  Manful et al., 2011 copy <br /> Entamoeba histolytica XP_648719.1; Tv- Trichomonas vaginalis                             number calculated). E. RNASeq results poly + RNA <br /> XP_001310227.1; Tp- Thalassosira pseudonana XP_002287950.1.                              normal cells  Manful et al., 2011). F. RNASeq results (D) <br /> (PDF)                                                                                    divided (E). G. Half-life total RNA (taken Manful et <br /> Figure S3 Alignment Leo1 homologues, using                                       al., 2011). H. Developmental regulation, taken Siegel et al., <br /> MegAlign (DNAStar). Details Figure S1. Organisms                              2010.  Position ORF near start stop site, judged Tb- T. brucei,  Arabidopsis thaliana; Dd- Dictyostelium discoideum                    enrichment modified histones described Siegel et al., <br /> XP_637916.1; Tp- Thalassiosira pseudonana XP_002289477.1; Hs-                            2009, displayed TritrypDB web site. J. Details Homo sapiens NP_620147.1; Sc- Saccharomyces cerevisiae Leo1; Ng-                         relative position ORF near start stop site: number <br /> Naegleria gruberi XP_002677260.1; Tt- Tetrahymena thermophila                            indicates ORF  second,  relative <br /> EAR92336.1; Pi-Phytophthora infestans XP_002903832.1                                     transcription break. K. Nature start/stop - convergent <br /> (PDF)                                                                                    stops, divergent starts, stops immediately followed                                                                                          start. L. notes - gene position near telomeres, pol Figure      S4 TbCTR9-RPB1            immunoprecipitation.                            transcription,  M. Neighbouring locus, transcribed pol Method: Bloodstream-form trypanosomes cell lines expressing                              pol III. strongly-regulated genes annotated. N. P-value <br /> TAP-tagged TbCTR9 used (lanes 5&#226;&#8364;&#8220;8), cells expressing                           regulation  Methods). Centromeres Genome Biol. <br /> TAP-tag control (lanes 1&#226;&#8364;&#8220;4). Protein expression                             2007; 8(3): R37. <br /> induced 24 hrs. 1.56108 cells collected centrifugation                       (XLSX) washed 16PBS. Cells lysed 500 ml breakage <br /> buffer previously described [42,45]. concentration NaCl                        Table S2Oligonucleotides used amplify cDNAs, adjusted 150 mM cell debris removed                               comments. <br /> centrifugation (4uC) 13000 rpm microfuge 30 min-                           (XLS) <br /> utes. IgG beads washed previously described clarified                       Table S3 Details BLAST search results. Standard <br /> cell lysates added prior incubation 2 hrs 4uC                         settings used. Culumns C D: Arabidopsis thaliana, <br /> microfuge tube rotating 15 rpm. unbound fraction                              single genome searched. genomes, search <br /> collected centrifugation. beads washed 46                               using genomes selected NCBI IPP150 buffer [42,45] protein recovered boiling                                                                                          genomes database (http://www.ncbi.nlm.nih.gov/sutils/genom_ <br /> beads 16 Laemmli lysis buffer. Proteins separated                                                                                          table.cgi). E values comparable 8% SDS-PAGE gel transferred overnight nitrocellulose <br />                                                                                          Arabidopsis larger database searched. <br /> membranes 30 V. Protein detection ECL kit (GE <br />                                                                                          Columns E F: Individual genes blasted Healthcare) using (1:1000) anti-RPB1 (unpublished,                                                                                          S. cerevisiae H. sapiens databases  E values given kind gift Gu&#194;&#168;nzl laboratory); (1:2000) anti- PAP (peroxidase&#226;&#8364;&#8220; <br />                                                                                          relevant protein sequence best match. <br /> antiperoxidase complex; Protein detection (Sigma)) TAP. <br />                                                                                          (XLSX) loading equivalent 56106 cells lysate (lanes 1,8) unbound (Lanes 2,7) fractions; wash 4 (TCA-concentrated, <br /> lanes 3 6) bead-bound fraction (lanes 4 5),                          Acknowledgments preparation loaded. Results: good signal bound                        mass spectrometry Dr. T. Ruppert team CTR9-TAP protein detected (lane 5)                                  ZMBH. thank Arthur Gu&#194;&#168;nzl anti-RBP1 antiserum (University <br /> detectable RBP1 signal (lane 5). Similar results obtained                        Connecticut Health Center) Luise Krauth-Siegel (Biochemie- <br /> immunoprecipiation CTR9-myc  shown).                                              Zentrum Heidelberg) anti-trypanothione reductase. <br /> (PDF) <br /> Table S1 Effects CTR9 transcriptome.                                 Author Contributions <br /> Table genes, ESAGs, VSGs pseudogenes excluded.                           Conceived designed experiments: BO BN TM CH MM CC. second Table (sheet 2) includes &#226;&#8364;&#732;&#226;&#8364;&#732;unique genes&#226;&#8364;&#8482;&#226;&#8364;&#8482;                         Performed experiments: BO BN TM CH MM. Analyzed data: BO <br /> Siegel et al; single copy genes  repeated genes,                      BN TM MM AF CC. Contributed reagents/materials/analysis tools: AF. <br /> single representative gene family. Columns follows:                      Wrote paper: BO BN TM AF CC. Did initial RNAi, prepared                                                                                          RNA RNASeq: BN. Did TAP purification, RBPI  <br /> TritrypDB accession number, v5; B. TritrypDB accession number,                           immunoprecipitation, initial Northern blots, v5, additional zeros gene <span id='am-26' about='oboInOwl:id' typeof='owl:Thing'>ID</span>                              completed CH MM  attempted RT-PCRs): BO. chromosome number digits. enables simple sort                         plasmids: BO BN. Did Cell fractionation: BN. Repeated Cell arrange genes order appear                            fractionation: TM BO. Aligned Analysed RNASeq data: TM AF. <br /> chromosome. C. Annotation TritrypDB; D. RNASeq results                            investigated functional implications results: BO CC. <br />  <br /> References <br /> 1.   Jaehning J (2010) Paf1 complex: Platform player RNA polymerase II         4.   Langenbacher  Nguyen C, Cavanaugh  Huang J, Lu F, et al. (2011)      transcription? Biochim Biophysica Acta 1799: 379&#226;&#8364;&#8220;388.                                    PAF1 complex differentially regulates cardiomyocyte specification. Dev Biol <br /> 2.   Penheiter K, Washburn T, Porter S, Hoffman M, Jaehning J (2005)                        353: 19&#226;&#8364;&#8220;28. <br />      Posttranscriptional Role Yeast Paf1-RNA Polymerase II Complex            5.   Nguyen C, Langenbacher  Hsieh M, Chen J (2010) PAF1 complex <br />      Revealed Identification Primary Targets. Molecular Cell 20: 213&#226;&#8364;&#8220;223.               component Leo1 essential cardiac neural crest development 3.   Panicker L, Zhang J, Dagur P, Gastinger M, Simonds W (2010) Defective                    zebrafish. Dev Biol 34: 167&#226;&#8364;&#8220;175. <br />      nucleolar localization dominant interfering properties parafibromin        6.   Oh S, Zhang H, Ludwig P, van Nocker S (2004) mechanism related      L95P missense mutant causing hyperparathyroidism-jaw tumor syndrome.                 yeast transcriptional regulator Paf1c required expression Arabidopsis <br />      Endocr Relat Cancer 17: 513&#226;&#8364;&#8220;524.                                                         FLC/MAF MADS box gene family. Plant Cell 16: 2940&#226;&#8364;&#8220;2953. <br />  <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                                  10                                    April 2012 | Volume 7 | Issue 4 | e34256 <br />                                                                                                                                                              Trypanosome CTR9 <br />  <br />  <br /> 7.    Nordick K, Hoffman M, Betz J, JA J (2008) Direct interactions Paf1           34. Solnoki K, Sing  Sofa C, Miller R, Ogorzalek P, et al. (2011) TbENF       complex cleavage polyadenylation factor revealed dissociation               essential TbTFIIB-interacting trypanosomatid-specific factor. Mol Biochem <br />       Paf1 RNA polymerase II. Eukaryot Cell 7: 1158&#226;&#8364;&#8220;1167.                                  Parasitol press. <br /> 8.    Kim J, Guermah M, Roeder R (2010) human PAF1 complex acts                         35. Lee JH, Cai G, Panigrahi AK, Dunham-Ems S, Nguyen TN, et al. (2010)       chromatin transcription elongation independently cooperatively                 TFIIH-associated mediator head basal factor small nuclear spliced leader <br />       SII/TFIIS. Cell 140: 491&#226;&#8364;&#8220;503.                                                                RNA gene transcription early-diverged trypanosomes. Mol Cell Biol 30: <br /> 9.    Mayer  Lidschreiber M, Siebert M, Leike K, ding J, et al. (2010) Uniform                5502&#226;&#8364;&#8220;5513. <br />       transitions general RNA polymerase II transcription complex. Nat Struct           36. Park SH, Nguyen TN, Gunzl (2012) Development efficient vitro <br />       Mol Biol 17: 1272&#226;&#8364;&#8220;1278.                                                                      transcription bloodstream form Trypanosoma brucei reveals life <br /> 10.   Kim J, Roeder R (2011) Nucleosomal H2B ubiquitylation purified factors.                 cycle-independent functionality class transcription factor  Mol Biochem <br />       Methods 54: 331&#226;&#8364;&#8220;338.                                                                         Parasitol 181: 29&#226;&#8364;&#8220;36. <br /> 11.   Tomson B, Davis C, Warner M, Arndt K (2011) Identification role                 37. Gunzl  Bruderer T, Laufer G, Schimanski B, Tu LC, et al. (2003) RNA <br />       histone H2B ubiquitylation noncoding RNA 39-end formation                         polymerase transcribes procyclin genes variant surface glycoprotein gene <br />       mutational analysis Rtf1 Saccharomyces cerevisiae. Genetics 188: 273&#226;&#8364;&#8220;289.              expression sites Trypanosoma brucei. Eukaryot Cell 2: 542&#226;&#8364;&#8220;551. <br /> 12.   Crisucci E, Arndt K (2011) Paf1 complex represses ARG1 transcription                                                                                                38. Rudenko G (2010) Epigenetics transcriptional control African <br />       Saccharomyces cerevisiae promoting histone modifications. Eukaryot Cell 10: <br />                                                                                                    trypanosomes. Essays Biochem 48: 201&#226;&#8364;&#8220;219. <br />       712&#226;&#8364;&#8220;723. <br />                                                                                                39. Biebinger S, Rettenmaier S, Flaspohler J, Hartmann C, Pen Diaz J, et al. <br /> 13.   Warner M, Roinick K, Arndt K (2007) Rtf1 multifunctional component       Paf1 complex regulates gene expression directing cotranscriptional               (1996) PARP promoter Trypanosoma brucei developmentally regulated       histone modification. Mol Cell Biol 27: 6103&#226;&#8364;&#8220;6115.                                           chromosomal context. Nucleic Acids Res 24: 1202&#226;&#8364;&#8220;1211. <br /> 14.   Rigbolt K, Prokhorova T, Akimov V, Henningsen J, Johansen P, et al. (2011)               40. Alibu VP, Storm L, Haile S, Clayton C, Horn D (2004) doubly inducible <br />       wide temporal characterization proteome phosphoproteome                    RNA interference rapid RNAi plasmid construction       human embryonic stem cell differentiation. Sci Signal 4: rs3.                             Trypanosoma brucei. Mol Biochem Parasitol 139: 75&#226;&#8364;&#8220;82. <br /> 15.   Yang Y, Han J, Youn H, Cho E (2010) tumor suppressor, parafibromin,                  41. Colasante C, Alibu VP, Kirchberger S, Tjaden J, Clayton C, et al. (2006) <br />       mediates histone H3 K9 methylation cyclin D1 repression. Nucleic Acids Res               Characterisation developmentally regulated localisation mitochon- <br />       38: 382&#226;&#8364;&#8220;390.                                                                                 drial carrier protein homologue MCP6 Trypanosoma brucei. Eukaryot Cell 5: <br /> 16.   Peterson M (2010) Immunoglobulin heavy chain gene regulation                         1194&#226;&#8364;&#8220;1205. <br />       polyadenylation splicing competition. WIREs RNA 2: 92&#226;&#8364;&#8220;105.                           42. Este&#194;&#180;vez  Kempf T, Clayton CE (2001) exosome Trypanosoma brucei. <br /> 17.   Pinto P, Henriques T, Freitas M, Martins T, Domingues R, et al. (2011) RNA                   EMBO J 20: 3831&#226;&#8364;&#8220;3839. <br />       polymerase II kinetics polo polyadenylation signal selection. EMBO J 30:              43. Este&#194;&#180;vez  Lehner B, Sanderson CM, Ruppert T, Clayton C (2003) roles <br />       2431&#226;&#8364;&#8220;2444.                                                                                   inter-subunit interactions exosome stability. J Biol Chem 278: <br /> 18.   Oesterreich F, Bieberstein N, Neugebauer K (2011) Pause locally, splice globally.            34943&#226;&#8364;&#8220;34951. <br />       Trends Cell Biol 21: 328&#226;&#8364;&#8220;335.                                                            44. Zeiner GM, Sturm NR, Campbell DA (2003) Exportin 1 mediates nuclear <br /> 19.   Ip J, Schmidt D, Pan Q, Ramani  AG F, et al. (2011) Global impact RNA                   export Kinetoplastid spliced leader RNA. Eucaryot Cell 2: 222&#226;&#8364;&#8220;230. <br />       polymerase II elongation inhibition alternative splicing regulation. Genome           45. Puig O, Caspary F, Rigaut G, Rutz B, Bouveret E, et al. (2001) tandem <br />       Res 21: 390&#226;&#8364;&#8220;401.                                                                             affinity purification (TAP) method: general procedure protein complex <br /> 20.   Rozenblatt-Rosen O, Nagaike T, Francis J, Kaneko S, Glatt K, et al. (2009)               purification. Methods 24: 218&#226;&#8364;&#8220;229. <br />       tumor suppressor Cdc73 functionally associates CPSF CstF 39 mRNA                46. Anders S, Huber W (2010) Differential expression analysis sequence count <br />       processing factors. Proc Natl Acad Sci USA 106: 755&#226;&#8364;&#8220;760.                                     data. Genome Biol 11: R106. <br /> 21.   Dermody J, Buratowski S (2010) Leo1 subunit yeast paf1 complex binds              47. Siegel T, Hekstra D, Wang X, Dewell S, Cross G (2010) Genome-wide analysis <br />       RNA contributes complex recruitment. J Biol Chem 285: 33671&#226;&#8364;&#8220;33679.                    mRNA abundance life-cycle stages Trypanosoma brucei 22.   Mueller J, Canze M, Bryk M (2006) requirements COMPASS Paf1                      identification splicing polyadenylation sites. Nucleic Acids Res 38: <br />       transcriptional silencing methylation histone H3 Saccharomyces                  4946&#226;&#8364;&#8220;4957. <br />       cerevisiae. Genetics 173: 557&#226;&#8364;&#8220;567.                                                       48. Alsford S, Turner D, Obado S, Sanchez-Flores  Glover L, et al. (2011) High <br /> 23.   Zhang Y, Smith  Renfrow M, Schneider D (2010) RNA polymerase- <br />                                                                                                    throughput phenotyping using parallel sequencing RNA interference targets <br />       associated factor 1 complex (Paf1C) directly increases elongation rate                                                                                                    African trypanosome. Genome Res 21: 915&#226;&#8364;&#8220;924. <br />       RNA polymerase required efficient regulation rRNA synthesis. <br />                                                                                                49. Schwede  Manful T, Jha B, Helbig C, Bercovich N, et al. (2009) role       J Biol Chem 285: 14152&#226;&#8364;&#8220;14159. <br />                                                                                                    deadenylation degradation unstable mRNAs trypanosomes. Nucleic <br /> 24.   Zhang Y, Sikes M, Beyer  Schneider D (2009) Paf1 complex required <br />       efficient transcription elongation RNA polymerase  Proc Natl Acad Sci               Acids Res 37: 5511&#226;&#8364;&#8220;5528. <br />       U S 106: 2153&#226;&#8364;&#8220;2158.                                                                    50. Ullu E, Matthews KR, Tschudi C (1993) Temporal order RNA-processing <br /> 25.   Kolev N, Franklin J, Carmi S, Shi H, Michaeli S, et al. (2010) transcriptome             reactions trypanosomes: rapid trans splicing precedes polyadenylation newly <br />       human pathogen Trypanosoma brucei single-nucleotide resolution. PLoS               synthesized tubulin transcripts. Mol Cell Biol 13: 720&#226;&#8364;&#8220;725. <br />       Pathog 6: e1001090.                                                                      51. Matthews KR, Tschudi C, Ullu E (1994) common pyrimidine-rich motif <br /> 26.   Siegel T, Hekstra D, Kemp L, Figueiredo L, Lowell J, et al. (2009) histone              governs trans-splicing polyadenylation tubulin polycistronic pre-mRNA       variants mark boundaries polycistronic transcription units Trypanosoma             trypanosomes. Genes Dev 8: 491&#226;&#8364;&#8220;501. <br />       brucei. Genes Dev 23: 1063&#226;&#8364;&#8220;1076.                                                         52. Stanne T, Kushwaha M, Wand M, Taylor J, Rudenko G (2011) TbISWI <br /> 27.   Michaeli S (2011) Trans-splicing trypanosomes: machinery impact                regulates multiple Pol transcribed loci present Pol II transcription <br />       parasite transcriptome. Future Microbiol 6: 459&#226;&#8364;&#8220;474.                                     boundaries. Eukaryotic Cell 10: 964&#226;&#8364;&#8220;976. <br /> 28.   Kramer S, Carrington M (2011) Trans-acting proteins regulating mRNA                      53. Narayanan M, Kushwaha M, Ersfeld K, Fullbrook  Stanne T, et al. (2011) <br />       maturation, stability translation trypanosomatids. Trends Parasitol 27:               NLP novel transcription regulator involved VSG expression site control       23&#226;&#8364;&#8220;30.                                                                                       Trypanosoma brucei. Nucleic Acids Res 39: 2018&#226;&#8364;&#8220;2031. <br /> 29.   Ferna&#194;&#180;ndez-Moya S, Este&#194;&#180;vez (2010) Posttranscriptional control role         54. Mueller C, Jaehning J (2002) Ctr9, Rtf1, Leo1 components Paf1/ <br />       RNA-binding proteins gene regulation trypanosomatid protozoan                          RNA polymerase  <br /> </body></html>