<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Decoupling catalytic activity biological function ATPase powers <span id='am-33' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-34' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-37' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span>lipopolysaccharide</span> transport <br /> David J. Shermana, Michael B. Lazarusa,b, Lea Murphyc, Charles Liua, Suzanne Walkerb, Natividad Ruizc,1, Daniel Kahnea,d,1  Department Chemistry <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Chemical</span> Biology, Harvard University, Cambridge, MA 02138; Departments bMicrobiology Immunobiology d <br />  Biological Chemistry Molecular Pharmacology, Harvard Medical School, Boston, MA 02115; cDepartment Microbiology, Ohio State University, <br /> Columbus, OH 43210 <br />  <br /> Edited M. J. Osborn, University Connecticut Health Center, Farmington, CT, approved February 10, 2014 (received review December 17, 2013) <br />  cell surface Gram-negative bacteria contains lipopolysac-             spans compartments cell, cytoplasm charides (<span id='am-23' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-24' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span>LPS</span>), provide barrier entry            OM (15, 16) (Fig. 1). E. coli seven different Lpt proteins antibiotics. LPS assembly involves multiprotein LPS transport              essential LPS transport cell viability (17, 18). (Lpt) complex spans cytoplasm outer mem-                Lpt proteins form ABC composed het- <br /> brane. complex, unusual <span id='am-54' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-55' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>-binding cassette trans-               eromeric TMD complex (LptF LptG) homodimeric <br /> porter thought power extraction LPS outer              NBD complex (LptB). LptB, complex leaflet cytoplasmic membrane transport             LptF LptG, ATPase activity vitro (19&#226;&#8364;&#8220;21). addition, <br /> cell envelope. introduce changes <span id='am-70' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-71' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-77' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span>-binding              LptB2FG complex closely associated bitopic IM <br /> domain, LptB, inactivate transporter function vivo.               protein LptC, binds LPS (20&#226;&#8364;&#8220;23) trans- <br /> characterize residues using biochemical experiments com-               envelope bridge (15, 16). Based findings, current <br /> bined high-resolution crystal structures LptB pre- post-         model LptB2FG extracts LPS outer leaflet ATP hydrolysis suggest role active site residue              IM sole energy input responsible entire <br /> phosphate exit. identify conserved residue             process transport assembly LPS cell surface <br /> required ATPase activity essential interaction           concentration gradient (22). Coupling ABC trans- transmembrane components. studies establish essen-               porter Lpt periplasmic bridge OM translocon enables <br /> tiality ATP hydrolysis LptB power LPS transport cells        cytoplasmic ATP drive periplasmic transit. hetero- <br /> suggest strategies inhibit transporter function away             meric architecture separate NBDs TMDs resembles LptB active site.                                                            bacterial importers, LptB2FG perform unique function <br />                                                                              places class distinct traditional importers ex- <br /> ABC transporter   | antibiotic target | membrane biogenesis                  porters; LptB, cytoplasmic ATPase, power extrac- <br />                                                                              tion glycolipid periplasmic face IM. <br />     TP-binding cassette (ABC) systems represent      largest protein superfamilies domains life (1, 2). systems transporters share common <br />                                                                                 understand LptB2FG transporter works, changed <br />                                                                              residues implicated important activities LptB                                                                              showed essential vivo function. residue architecture transmembrane domains (TMDs)                     proposed essential active site glutamate, cytoplasmic nucleotide-binding domains (NBDs), bind                    residues essential catalytic activity purified hydrolyze ATP. couple energy ATP binding                 LptB. residues affects ATPase <br /> hydrolysis transport variety substrates concentration gradient. eukaryotic ABC transporters                    Significance <br /> involved human diseases received attention (3), canonical ABC systems extensively studied                     Gram-negative bacteria contain unusual outer membrane Gram-negative bacterial importers (2).                                        prevents entry currently available antibiotics. <br />    Gram-negative bacteria, Escherichia coli, unique                membrane contains complex glycolipid, LPS, ex- <br /> double-membrane architecture allows colonize harsh                   terior. understood large molecule, environments. inner membrane (IM) contains phospholip-                        contain hundreds sugars fatty acyl chains, ids, outer membrane (OM) asymmetric <span id='am-17' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-19' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span>                       transported cell envelope site synthesis bilayer composed phospholipids inner leaflet LPS                    cytoplasmic membrane cell surface. Using combi- outer leaflet (4). membranes separated                       nation genetics, biochemistry, structural biology, aqueous periplasmic compartment. LPS complex                           characterized residues protein powers LPS transport <br /> glycolipid composed hundreds sugars attached core                      gain mechanistic insight ATP hydrolysis coupled <br /> containing fatty acyl chains (Fig. 1). Millions LPS molecules                  biological function transporter. tools help properly assembled division cycle (4)                         understand design antibiotics targeting essen- cell surface establish permeability barrier pre-                 tial pathway. <br /> vents entry hydrophobic molecules, including antibiotics <br />                                                                              Author contributions: D.J.S., M.B.L., N.R., D.K. designed research; D.J.S., M.B.L., L.M., <br /> (5).  understanding LPS assembled OM <br />                                                                              C.L., N.R. performed research; D.J.S., M.B.L., L.M., S.W., N.R., D.K. analyzed data; lead development better strategies target                 D.J.S., S.W., N.R., D.K. wrote paper. <br /> Gram-negative infections. 1972, established               authors declare conflict  <br /> biosynthesis LPS completed outer leaflet IM.             article PNAS Direct Submission. LPS passively aqueous peri- <br />                                                                              Data deposition: atomic coordinates structure factors deposited plasm OM, recognized               Protein Data Bank, www.pdb.org (PDB <span id='am-95' about='oboInOwl:id' typeof='owl:Thing'>ID</span> codes 4P31, 4P32, 4P33). <br /> machinery transport LPS cell envelope (6, 7).                  1 <br />                                                                                  correspondence addressed. E-mail: ruiz.82@osu.edu kahne@ <br />    past decade, components essential                   chemistry.harvard.edu. <br /> transport assembly LPS identified (8&#226;&#8364;&#8220;14).                   article contains supporting information online www.pnas.org/lookup/suppl/doi:10. LPS transport (Lpt) proteins comprise complex                   1073/pnas.1323516111/-/DCSupplemental. <br />  <br />  <br />  <br /> 4982&#226;&#8364;&#8220;4987 | PNAS | April 1, 2014 | vol. 111 |  13                                                               www.pnas.org/cgi/doi/10.1073/pnas.1323516111 <br />                                                                                             source wild-type LptB encoded pRC7KanLptB, <br />                                                                   O-Antigen                unstable, segregation-defective plasmid rapidly lost                                                           Glu         Hep <br />                                                                                            population, causing cell death (details strain  <br /> LPS                                                                                        struction provided SI Materials Methods Tables <br />                           D                               Glu                              S1&#226;&#8364;&#8220;S3). plasmid encoding functional LptB LptB- <br />       OM <br />                                                                                            His8 variant introduced strain, cells lost <br />                                                           Glu         Gal <br />  PL <br />                                                                                    Core <br />                                                                                            pRC7KanLptB viable; contrast, plasmid encod- <br />                       E <br />                                                      P <br />                                                           Hep         Hep                  ing nonfunctional LptB LptB-His8 variant introduced <br />                                                                                            strain, cells lost pRC7KanLptB died.                                                           Hep <br />                                                 PEtN                                       approach, unlike wild-type LptB-His8 LptB, <br />       Periplasm <br />                                                         Kdo         Kdo                  E163Q, H195A, F90A variants nonfunctional <br />                                                           O <br />                                                                                            (Fig. 2). loss function result reduced protein <br />                                                  O <br />                                             HO- P             O        O                   levels (Fig. S1A). tested F90Y variant                                C               O <br />                                                      OO <br />                                                  O                NH HO O           O      aromatic residue typically position  PL                                                       O        O            NH P OH <br />                                            O <br />                                                 O <br />                                                           O     HO <br />                                                                             O     O   O-   NBDs. LptB LptB-His8 F90Y variants <br />       IM          F       G                          O                     HO              functional vivo. <br />  PL                                                                              Lipid      introduction plasmid-encoded LptB- <br />       Cytoplasm                                                                            E163Q-His8 LptB-H195A-His8 variants wild-type <br />                   B       B                                                                strain resulted increased OM permeability (Fig. S1B).                                                                                            simplest explanation dominant-negative effect               ATP         <span id='am-11' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span> + Pi                                                         nonfunctional variants substitute wild-type LptB Lpt <br />                                                                                            IM complexes, reducing number viable trans- <br /> Fig. 1. LPS transported cell surface E. coli essential Lpt pro- <br />                                                                                            porters compromising LPS transport. contrast, mer- <br /> teins. (Right) E. coli LPS structure composed lipid (endotoxin),               odiploid strain harboring LptB-F90A-His8 variant did core region, O-antigen. (Left) Lpt proteins form transenvelope                 display increased OM permeability (Fig. S1B), suggesting complex transports LPS site synthesis (IM), aqueous            replace wild-type LptB IM complex. <br /> periplasm, cell surface. LPS transport assembly believed powered ABC transporter LptB2FG IM (yellow),                           Residue F90 Essential Proper Formation Lpt IM Complex. <br /> cause ATP periplasm.  LptA; B, LptB; C, LptC; D, LptD; E,             assessed LptB variants produced LptE; EtN, ethanolamine; F, LptF; G, LptG; Gal, <span id='am-49' about='obo:IMR_0200123' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="D-Galactose" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:id" content="IMR:0200123" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00124" datatype="xsd:string"></span>D-galactose</span>; Glu, <span id='am-90' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-91' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span>D-glucose</span>;               merodiploid strains interact Lpt IM components. <br /> Hep, L-glycero-D-mannoheptose; Kdo, 3-deoxy-D-manno-oct-2-ulosonic acid;                   Wild-type LptB-His8 interacts Lpt IM complex cells <br /> P, phosphate; PL, phospholipids. <br />                                                                                            copurifies endogenous LptF LptC (Fig. 3A). lack <br />                                                                                            LptG antiserum prevented monitoring LptG, activity intact Lpt IM complex,                             likely LptF LptG required formation essential site binding TMDs. Based high-resolution                              stable complex LptB LptC (20). E163Q crystal structures LptB pre- post-ATP hydrolysis,                                H195A variants pull Lpt components, propose residue facilitates phosphate exit                            F90A variant does  results consistent active site. studies highlight importance                                 interpretation dominant-negative phenotypes exerted <br /> combining structural studies genetics biochemistry                             E163Q H195A variants. F90A complexes, individual components, understand                            variant form stable complex Lpt IM components, unique ABC transporter functions. <br />  <br /> Results                                                                                                   vivo function LptB LptB-His8 variants <br /> Identification LptB Residues Required Cell Viability. Using <br />                                                                                                                               Complementation lptB <br /> sequence homology NBDs, identified conserved <br /> residues located different regions LptB. E163                                          <span id='am-83' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-84' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>Amino acid</span> <br />                                                                                                                                   LptB         LptB-His8 <br /> end Walker B motif essential catalysis                                             change <br />                                                                                                                                +                + <br /> ABC transporters. proposed general base deprot- <br />                                                                                                               E163Q                 -                - <br /> onates nucleophilic water molecule attacks &#206;&#179;-phos-                                               H195A                 -                - <br /> phate ATP (2, 24, 25). H195 located conserved switch                                              F90A                 -                - <br /> region adjacent active site NBDs impli-                                                 F90Y                 +                + <br /> cated catalytic activity, specific function debated <br /> (2, 25&#226;&#8364;&#8220;27). Lastly, F90 located Q-loop, pro- <br />                                                                                                         WT E163Q H195A       -          WT   F90A        -   WT    F90Y <br /> posed link ATP-binding site structurally diverse                             LptB-His8 <br />                                                                                                                                                                           MICROBIOLOGY <br />  <br />  <br />  <br />  <br /> region interacts coupling helices TMDs ABC transporters (2, 25, 28, 29). LptB shown                                 LptB <br /> quired LPS transport  consequently, cell viability E. coli <br />                                                                                              OmpA <br /> (11, 12).  mutated lptB determine importance residues LPS biogenesis. defects LPS assembly result phenotypes range increased OM                                                    lptB+                      lptB+                lptB <br /> permeability cell death (10, 17, 30), assessed suscep- <br />                                                                                            Fig. 2. Genetic studies indicate residues essential LptB functionality. <br /> tibility OM-impermeant antibiotics cell viability. <br />                                                                                            Functionality untagged His8-tagged LptB variants alterations    Plasmid-encoded LptB-His8 (used purification,                              active site Q-loop determined ability complement <br /> untagged LptB  assess effect His8 tag) variants                              chromosomal &#206;&#8221;lptB allele. Anti-LptB Western blots residue classified nonfunctional support cell                            changes (shown lanes) alter LptB-His8 levels. WT refers viability absence wild-type lptB allele. assess                        wild-type LptB-His8. Chromosomal lptB alleles strains shown type complementation, constructed strain                            LptB-His8 bands. Cross-reacting OmpA shown loading control. <br />  <br />  <br /> Sherman et al.                                                                                                      PNAS | April 1, 2014 | vol. 111 |  13 | 4983 <br />                                                                                  clear electron density intact ATP (Fig. 4C). Additionally, unlike <br />                                                                                 LptB-ADP structure, prehydrolysis form crystallized                                                                                 canonical nucleotide-sandwich dimer (Fig. S2D). Secondary <br />                                                                                 structure matching alignments (35) LptB-ATP sand- <br />                                                                                 wich dimer ATP-bound E. coli MalK, NBD <br />                                                                                 maltose/maltodextrin importer [Protein Data Bank (PDB) <br />                                                                                 ID code 1q12], Methanocaldococcus jannaschii MJ0796                                                                                  homolog LolD, NBD lipoprotein exporter, PDB ID <br />                                                                                 code 1l2t) rmsd values 1.75 &#195;&#8230;. cases, ATP                                                                                 sandwiched Walker motif subunit                                                                                 signature motif opposing unit (Fig. S2E). <br /> Fig. 3. Biochemical studies indicate residues LptB essential catalysis <br />                                                                                    LptB possesses overall fold resembling NBD proper coupling Lpt components.   Western blots           structures (Fig. 4A). contains canonical L-shaped archi- <br /> affinity purifications LptB variants expressed merodipoloid strains       tecture (36) composed RecA-like &#206;&#177;/&#206;&#178;-ATPase domain harboring plasmids expressing LptB variants. plasmids express LptB-His8,    structurally diverse &#206;&#177;-helical domain. RecA-like domain untagged variant (labeled  tag&#226;&#8364;?). Levels LptF LptC     contains Walker Walker B motifs present copurify LptB shown. Levels BamA, OM protein           <span id='am-78' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-79' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> <span id='am-65' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span>triphosphate</span>-binding proteins (37) (Fig. 4A). nonspecifically interacts purification resin, shown sample    domain furnishes Mg2+- nucleotide-binding motifs <br /> preparation loading control. (B) LptB-His8 variants overexpressed <br />                                                                                 specific ABC proteins,  Q-loop, links purified, ATPase activity measured. Data represent average SD experiments. WT, amino acid change LptB-His8.     highly conserved &#206;&#177;/&#206;&#178;-ATPase domain &#206;&#177;-helical  <br />                                                                                 main, switch region, contains conserved H195. <br />                                                                                 observed NBD structures, LSSG(E/Q) signature does confer dominant phenotype mero-                        motif helical domain (25, 38) (Fig. 4A). Addi- <br /> diploid strain.                                                                 tionally, LptB structures reveal interface   tagged F90Y variant pulls lesser                     predict LptB interacts TMDs LptF/G, based compar- <br /> IM Lpt components compared wild-type, E163Q,                       isons structures NBDs ABC <br /> H195A variants (Fig. 3A), showing does associate                 transporters (Fig. S2F). interface contains grooves (Fig. 5) IM components wild-type variant.                      accommodate coupling helices TMDs, defective interaction functional consequences                     involved communication NBDs TMDs haploid strain producing LptB-F90Y-His8 shows increased OM                      ABC transporters (39, 40). <br /> permeability compared wild-type strain (Fig. S1C). Taken  results implicate F90 association LptB                ATP Hydrolysis Induces Conformational Changes. Comparison LptF/G/C. essential role E163                   pre- posthydrolysis complexes LptB demonstrates  H195 LptB function unrelated Lpt complex formation.                siderable movement number regions associated                                                                                 binding hydrolysis &#206;&#179;-phosphate (Fig. 4B). Confor- <br /> Nonfunctional LptB Variants Variable Catalytic Activity.                mational changes switch region, containing H195, tested ATPase activity different LptB variants               region surrounding Walker B motif, containing E163, gain insight nature functional defects.               apparently driven reorganization active site following ATP <br /> overexpressed purified variants stable proteins.                  hydrolysis. reorganization coupled changes signa- <br /> expected (19), LptB-E163Q-His8 catalytically inactive (Fig.                  ture motif Q-loop, contains F90 (Fig. 4 B). <br /> 3B). variant forms stable IM complex, inability                 closer view LptB-ATP active site reveals key support cell viability attributed defective cata-             residues orient stabilize &#206;&#179;-phosphate  <br /> lytic activity.  ATP hydrolysis LptB required               organize catalytic cycle maintain contact ADP <br /> LPS transport cell. LptB-F90A-His8                     following phosphate-bond cleavage (Fig. 4 C D). Residue <br /> LptB-F90Y-His8 wild-type levels ATPase activity.                   163 (Q163 LptB-ATP E163 native LptB-ADP substitutions affect protein folding cat-                structure), located end Walker B motif, H195, <br /> alytic activity, affect LptB function vivo. Finally,                located switch region, make contacts bridging water <br /> LptB-H195A-His8 variant shows reduction &#226;&#710;&#188;60% ATPase                     molecule &#206;&#179;-phosphate, respectively, LptB-ATP <br /> activity does support cell viability. activity          structure (Fig. 4E). Glutamine-85, start somewhat surprising changing conserved histidine                Q-loop (giving , forms octahedral  ABC transporter systems results nearly complete loss                ordination sphere surrounding metal <span id='am-43' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-44' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-48' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> associated ATPase activity (26, 31&#226;&#8364;&#8220;34).                                                 nucleotide (Fig. 4E). <br />                                                                                    E163 essential catalysis (Fig. 3B). closer look Crystal Structures LptB Bound ADP ATP. understand              carbonyl oxygen chain residue 163 reveals roles F90 H195 better, obtained crystal structures                 &#226;&#710;&#188;2.2 &#195;&#8230; closer nucleotide posthydrolysis struc- <br /> LptB ATP hydrolysis. overexpressed                      ture based structural alignment Walker motifs <br /> purified functional LptB-His8 obtained crystals               (36-GPNGAGKT-43). bridging water molecule, native protein selenomethionine derivative following                   glutamate contacts &#206;&#178;-phosphate nucleotide (Fig. 4E). <br /> cubation ATP/MgCl2 (Fig. S2A Tables S4 S5).                        LptB-ATP, Q163 oriented slightly farther nucle- unambiguous density ADP-Mg2+, indicating ATP                      otide separated bridging water molecule <br /> hydrolysis occurred crystallization (Fig. S2B).                 &#206;&#179;-phosphate (Fig. 4E). Based structural alignments <br /> 1.55-&#195;&#8230; structure referred LptB-ADP.                     E. coli MalK bound transition-state mimics (ADP-vana- <br />    obtain prehydrolysis LptB/ATP complex, purified                  <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> ADP-aluminum fluoride) (41), comparison catalytically inactive variant LptB-E163Q (Fig. 3B) obtained                ATP-bound MJ0796 structure (42), believe water crystals preincubation ATP. resulting 1.65-&#195;&#8230; LptB-               positioned nucleophilic water hydrolysis <br /> E163Q-ATP structure referred LptB-ATP                   reaction. Consistent hypothesis, water molecule (Fig. 4A Fig. S2C). Examination active site demonstrated             present LptB-ADP structure (Fig. 4E). <br />  <br /> 4984 | www.pnas.org/cgi/doi/10.1073/pnas.1323516111                                                                                     Sherman et al. <br />  Fig. 4. Conformational changes ATP hydrolysis reorganization active site causes changes region LptB believed interact LptF/G.   Cartoon rendering LptB-ATP (prehydrolysis), conserved ATPase ABC motifs indicated (Walker  yellow; Walker B, orange; <br /> signature motif, blue; Q-loop, cyan; switch region, red). assigned active site metal LptB-ATP structure Na+ ion. distances metal coordinating water molecules correspond characteristic Na+ (&#226;&#710;&#188;2.4 &#195;&#8230;) characteristic Mg2+ (&#226;&#8240;&#164;2.1 &#195;&#8230;), observed LptB-ADP structure. addition, LptB-ATP crystallized magnesium-free buffer condition containing sodium chloride. (B) Structural <br /> overlay LptB-ATP (red) LptB-ADP (blue) structures shows conformational changes ATP hydrolysis. product (ADP) shown active <br /> site. (C) Close view ATP-bound active site, chains conserved residues Walker B, switch, Q-loop regions shown. Fo-Fc <br /> omit map contoured 3&#207;&#402; shows clear electron density &#206;&#179;-phosphate. (D) Structural overlay, B, pre- posthydrolysis structures, arrows indicating movement chains hydrolysis. Coloring conserved regions  (E) Rearrangement active site residues ATP hydrolysis stabilize ADP shown. octahedral coordination sphere active site cation shown, putative hydrolytic water indicated ATP-bound active site. <br />  <br />  <br />  <br />    Comparison pre- posthydrolysis structures                       C&#206;&#177; shifts &#226;&#710;&#188;4 &#195;&#8230; ADP-bound structure. shows H195 switch region undergoes major                        does switch region  chain H195 <br /> formational change (Fig. 4D). chain H195 makes                        flips. important unanswered question inorganic <br /> direct contact &#206;&#179;-phosphate ATP-bound structure.                   phosphate (Pi) exits active sites NBDs. clear <br />                                                                                   electron density Pi group posthydrolysis crystal <br />                                                                                   structure (Fig. S2B), consistent claims Pi leaves <br />                                                                                   active site ADP released (42&#226;&#8364;&#8220;44). Based                  Groove                               Groove                      electrostatic potential surface LptB, H195 chain flips <br />                                    ATP                                            face negatively charged protein (Fig. S3).                                                                                   possible &#206;&#179;-phosphate remains bound H195      F90 <br />        0 <br />                                                                                                                                                                    MICROBIOLOGY <br />  <br />  <br />  <br />  <br />                                                                                   movement switch region forces Pi electro- <br />                                                                                   static repulsion negatively charged area. possible <br />                                                    F90 <br />                                                    F                              dramatic movement switch region observed dur- <br />           Q163                                                                    ing ATP hydrolysis plays critical role communicating <br />                                              R91 <br />                                                                     ATP           changes active site changes TMDs. <br />                   H195                                                               H195 essential vivo LptB-ATP structure <br />                                                                                   shows imidazole chain directly interacts                                                     View <br />                                                                                   &#206;&#179;-phosphate ATP, suspected ATPase activity ob- <br />                                                                                   served isolated NBD (Fig. 3B) reported inaccurately <br /> Fig. 5. Structural observations LptB implicate key binding site                                                                                   activity complex (21).  overex- <br /> TMDs. Surface renderings monomer LptB-ATP (frontal facing         pressed purified LptB2FGC detergent measured views) residues Q163 H195 near ATP, residue F90 facing            ATPase activity wild-type complex complexes  <br /> terior groove, residue R91 facing away groove.                taining LptB-E163Q LptB-H195A (Fig. S4A). complex <br />  <br /> Sherman et al.                                                                                              PNAS | April 1, 2014 | vol. 111 |  13 | 4985 <br />  containing LptB-E163Q catalytically inactive, complex       ATP hydrolysis induces conformational changes LptB. containing LptB-H195A &#226;&#710;&#188;10% ATPase activity               dramatic conformational change structural snapshots wild-type complex. results confirm hypothesis       switch region. does switch loop  H195 sole catalytic residue, positioning       imidazole chain conserved H195 flips away important step catalytic cycle, suggested        active site negatively charged channel.  crystal structures.                                             ment, combined observation H195 coordinates                                                                        &#206;&#179;-phosphate prehydrolysis complex, leads speculate <br /> Groove Region LptB Essential Interaction IM Partners.   residue helps &#206;&#179;-phosphate active site <br /> Taken  ATP hydrolysis initiates global movement            acidic channel, ejected electrostatic <br /> LptB couples changes regions surrounding active site       repulsion (Fig. S3). shift conserved histidine near <br />  switch Walker B domains) changes struc-         negatively charged surface observed structure turally diverse helical domain  signature motif Q-loop).   NBD MJ0796 (42). role consistent low-level changes near helical domain result shift         ATPase activity observed LptB-H195A complex groove located adjacent Q-loop (Fig. S4B Fig. 5).      Lpt IM components. <br /> coupling ATP hydrolysis movement groove region             crystal structures reveal groove LptB undergoes important communicating TMDs,                                                                        movement ATP hydrolysis. hypothesize groove affecting LPS transport.  interested                                                                        involved interaction TMD partners, LptF/G, positioning F90 Q-loop, interpreted                                                                        movement essential connecting ATP hydrolysis vivo biochemical results described mean F90                                                                        LptB LPS extraction LptF/G. genetic bio- <br /> important binding putative coupling helices TMDs <br />                                                                        chemical studies implicate groove residue F90 important ABC  <br />    chain F90 Q-loop faces interior           binding site LptF/G. crystal structures suggest groove opening ATP- ADP-bound struc-               residue critical, aromatic chain faces inside <br /> tures (Fig. 5), suggesting residue functionally im-      groove likely interacts coupling helices portant interacting TMDs        TMDs. Structural alignments NBDs, isolation <br /> catalytic cycle.  role residue F90 explain   complex TMDs, reveal  despite lptB-F90A allele nonfunctional. test positions       structurally diverse region, aromatic residue chains facing away groove interior im-       location F90 LptB. example, F90 LptB <br /> portant residues pointing groove, assessed im-      aligns F429 Salmonella typhimurium MsbA (PDB ID <br /> portance LptB-R91A, chain facing away         code 3b60), Y87 MalK (PDB ID code 2r6g), interior groove crystallographic snapshots          Y94 Sulfolobus solfataricus GlcV (PDB ID code 1oxx). (Fig. 5). LptB-R91A variant functionally          structure Sav1866 bound nonhydrolyzable ATP ana- <br /> similar wild-type LptB (Fig. S1 C).                           logue (PDB ID code 2onj), guanidinium group R206    Taken  results support hypothesis           TMD close proximity make &#207;&#8364;-cation  <br /> residues Q-loop LptB face interior           teraction aromatic ring F427 NBD, groove important assembly complex, func-        aligns F90 LptB (Fig. S4D), suggesting residue <br /> tion,  high conservation F90 LptB orthologs      critical interaction TMDs. LptB, (Fig. S4C), combined fact lptB-F90Y allele           established importance F90 genetic analysis affinity <br /> confers mild OM permeability defects haploid strains (Fig. S1C),    purifications: F90A variant fails complement results <br /> suggests conservative change phenylalanine ty-      destabilization ABC transporter. aromatic char- <br /> rosine tolerated, optimal, LptB evolved        acter residue critical mediating interaction phenylalanine position. experiments lead       LptB LptF/G, F90Y substitution results clude F90 forms critical binding site LptF/G.   partial loss function vivo, suggesting interaction                                                                        specific. studies underway examine key <br /> Discussion                                                             interaction TMDs important linking ATP crystal structures catalytically inactive         hydrolysis extraction LPS opposite variant LptB bound ATP wild-type LptB bound ADP.          membrane conformational changes groove region. combining biochemistry genetic analysis LptB var-            comprehensive analysis selected residues iants, identified residues LptB important      ATPase component Lpt IM complex combines <br /> ATP hydrolysis (E163), assembly LptF/G (F90),             genetic studies essentiality, biochemical studies catalytic ac- <br /> function complex (H195). Structural studies reveal         tivity protein&#226;&#8364;&#8220;protein interaction, crystallographic analysis <br /> residues involved binding ATP catalyzing &#206;&#179;-phosphate            pre- posthydrolysis complexes. results highlight hydrolysis reorganize stabilize product, ADP. crys-         importance combining multiple techniques understand tallographic snapshots suggest movement residues        roles individual residues process transporting LPS LptB active site coupled movement regions          IM cell surface. catalytic activity cell <br /> interface LptF/G essential LPS transport.       viability illuminate functions certain residues. addition, using catalytically inactive LptB-E163Q variant,     Clearly, interfering catalytic activity coupling demonstrated ATP hydrolysis LptB essential                                                                        Lpt components viable strategies development new <br /> cell viability; contrast, ATP hydrolysis required                                                                        antibiotics targeting LPS biogenesis. <br /> formation Lpt IM complex. <br />    respect mode catalysis, observations         Materials Methods <br /> consistent prevailing general base catalysis model,        Vivo Experiments. Strains, growth conditions, construction plasmids carboxylate chain end Walker B motif        &#206;&#8221;lptB allele described SI Materials Methods Tables S1&#226;&#8364;&#8220; <br />  residue E163) deprotonates water molecule         S3. Functionality mutant lptB alleles assessed complementary <br /> serve nucleophile hydrolysis reaction. locations    methods described SI Materials Methods. necessary, general base potential nucleophile consistent         chromosomal &#206;&#8221;lptB allele transferred P1 transduction (45). structures transition state E. coli maltose im-        OM permeability tests performed using disk diffusion assays (46). <br /> porter (41). pre- postcatalysis structures reveal         LptB protein levels monitored immunoblotting using anti-LptB <br />  <br />  <br /> 4986 | www.pnas.org/cgi/doi/10.1073/pnas.1323516111                                                                                   Sherman et al. <br />  antisera raised rabbit using LptB <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> DDLSAEQREDRANE                      Protein Crystallization, Data Collection, Structure Determination.  <br /> immunogen (ProSci, ).                                                                     tailed description structure determination provided SI Materials <br />                                                                                               Methods. Briefly, protein crystals grown vapor diffusion Affinity Purifications ATPase Assays. Modifications affinity purifications             hanging drops. Experimental phasing obtained multiple isomorphous <br /> (15) explained SI Materials Methods. ATPase assays                        replacement anomalous scattering using selenomethionine derivative <br /> ducted described 5 mM ATP/MgCl2 LptB LptB2FGC                            crystal tantalum derivative crystal. X-ray diffraction data col- <br /> variants (21).                                                                                lected National Synchrotron Light Source beamline X29 Brook- <br />                                                                                               haven National Laboratory, selenomethionine dataset, Protein Overexpression Purification. LptB-His8 catalytically inactive             collected 24-ID-E Advanced Photon Light Source Argonne <br /> E163Q variant overexpressed E. coli purified nickel-affinity                   National Laboratory. <br /> chromatography gel filtration chromatography described (19), notable changes described SI Materials Methods. Overexpression                      ACKNOWLEDGMENTS. thank Rebecca M. Davis strain construction selenomethionine-containing derivative LptB (SeMet-LptB-His8) described               Dr. Seong-Joo Lee collecting selenomethionine dataset.                                                                                               work supported National Science Foundation Graduate Research SI Materials Methods. <br />                                                                                               Fellowship  D.J.S.); Harvard Blavatnik Biomedical Accelerator Fund; <br />    LptB2FGC complexes (wild-type, E163Q, H195A variants) containing                       National Institutes Health Grants AI081059  D.K.), GM066174  D.K.), <br /> His6-LptB overexpressed E. coli purified described SI                       GM094263  S.W.); startup funds Ohio State University <br /> Materials Methods.                                                                         N.R.). <br />  <br />  <br />  1. Dassa E, Bouige P (2001) ABC ABCS: phylogenetic functional classifica-       24. Moody JE, Millen L, Binns D, Hunt JF, Thomas PJ (2002) Cooperative, ATP-dependent <br />     tion ABC systems living organisms. Res Microbiol 152(3-4):211&#226;&#8364;&#8220;229.                      association nucleotide binding cassettes catalytic cycle ATP- <br />  2. Davidson AL, Dassa E, Orelle C, Chen J (2008) Structure, function, evolution           binding cassette transporters. J Biol Chem 277(24):21111&#226;&#8364;&#8220;21114. <br />     bacterial ATP-binding cassette systems. Microbiol Mol Biol Rev 72(2):317&#226;&#8364;&#8220;364.             25. Oswald C, Holland IB, Schmitt L (2006) motor domains ABC-transporters.  3. Dean M, Rzhetsky  Allikmets R (2001) human ATP-binding cassette (ABC)                   structures tell  Naunyn Schmiedebergs Arch Pharmacol 372(6):385&#226;&#8364;&#8220;399. <br />     transporter superfamily. Genome Res 11(7):1156&#226;&#8364;&#8220;1166.                                      26. Zaitseva J, Jenewein S, Jumpertz T, Holland IB, Schmitt L (2005) H662 linchpin  4. Raetz CR, Whitfield C (2002) Lipopolysaccharide endotoxins. Annu Rev Biochem 71:              ATP hydrolysis nucleotide-binding domain ABC transporter HlyB. EMBO J <br />     635&#226;&#8364;&#8220;700.                                                                                      24(11):1901&#226;&#8364;&#8220;1910. <br />  5. Nikaido H (2003) Molecular basis bacterial outer membrane permeability revisited.      27. Ernst R, et al. (2008) mutation H-loop selectively affects rhodamine trans- <br />     Microbiol Mol Biol Rev 67(4):593&#226;&#8364;&#8220;656.                                                         port yeast multidrug ABC transporter Pdr5. Proc Natl Acad Sci USA 105(13): <br />  6. Osborn MJ, Gander JE, Parisi E (1972) Mechanism assembly outer membrane             5069&#226;&#8364;&#8220;5074. <br />     Salmonella typhimurium. Site synthesis lipopolysaccharide. J Biol Chem           28. Karpowich N, et al. (2001) Crystal structures MJ1267 ATP binding cassette  <br />     247(12):3973&#226;&#8364;&#8220;3986.                                                                            veal induced-fit effect ATPase active site ABC transporter. Structure <br />  7. Osborn MJ, Gander JE, Parisi E, Carson J (1972) Mechanism assembly outer            9(7):571&#226;&#8364;&#8220;586. <br />     membrane Salmonella typhimurium. Isolation characterization cytoplasmic         29. Schmitt L, Benabdelhak H, Blight MA, Holland IB, Stubbs MT (2003) Crystal structure <br />     outer membrane. J Biol Chem 247(12):3962&#226;&#8364;&#8220;3972.                                            nucleotide-binding domain ABC-transporter haemolysin B: Identifica- <br />  8. Bos MP, Tefsen B, Geurtsen J, Tommassen J (2004) Identification outer mem-              tion variable region ABC helical domains. J Mol Biol 330(2):333&#226;&#8364;&#8220;342. <br />     brane protein required transport lipopolysaccharide bacterial cell      30. Ruiz N, Wu T, Kahne D, Silhavy TJ (2006) Probing barrier function outer <br />     surface. Proc Natl Acad Sci USA 101(25):9417&#226;&#8364;&#8220;9422.                                            membrane chemical conditionality. ACS Chem Biol 1(6):385&#226;&#8364;&#8220;395. <br />  9. Braun M, Silhavy TJ (2002) <span id='am-96' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="IMP" datatype="xsd:string"></span>Imp</span>/OstA required cell envelope biogenesis Es-       31. Davidson AL, Sharma S (1997) Mutation single MalK subunit severely impairs <br />     cherichia coli. Mol Microbiol 45(5):1289&#226;&#8364;&#8220;1302.                                                maltose transport activity Escherichia coli. J Bacteriol 179(17):5458&#226;&#8364;&#8220;5464. <br /> 10. Ruiz N, Gronenberg LS, Kahne D, Silhavy TJ (2008) Identification inner-mem-        32. Nikaido K, Ames GF (1999) intact ATP-binding subunit sufficient support ATP <br />     brane proteins required transport lipopolysaccharide outer mem-             hydrolysis translocation ABC transporter, histidine permease. J Biol <br />                                                                                                   Chem 274(38):26727&#226;&#8364;&#8220;26735. <br />     brane Escherichia coli. Proc Natl Acad Sci USA 105(14):5537&#226;&#8364;&#8220;5542. <br />                                                                                               33. Schneider E, Hunke S (1998) ATP-binding-cassette (ABC) transport systems: Functional <br /> 11. Sperandeo P, et al. (2007) Characterization lptA lptB, essential genes <br />                                                                                                   structural aspects ATP-hydrolyzing subunits/domains. FEMS Microbiol Rev <br />     implicated lipopolysaccharide transport outer membrane Escherichia coli. <br />                                                                                                   22(1):1&#226;&#8364;&#8220;20. <br />     J Bacteriol 189(1):244&#226;&#8364;&#8220;253. <br />                                                                                               34. Shyamala V, Baichwal V, Beall E, Ames GF (1991) Structure-function analysis 12. Sperandeo P, et al. (2008) Functional analysis protein machinery required                                                                                                   histidine permease comparison cystic fibrosis mutations. J Biol Chem <br />     transport lipopolysaccharide outer membrane Escherichia coli. J Bacteriol <br />                                                                                                   266(28):18714&#226;&#8364;&#8220;18719. <br />     190(13):4460&#226;&#8364;&#8220;4469. <br />                                                                                               35. Krissinel E, Henrick K (2004) Secondary-structure matching (SSM), new tool fast <br /> 13. Sperandeo P, Pozzi C, Deh&#195;&#178; G, Polissi (2006) Non-essential KDO biosynthesis                                                                                                   protein structure alignment dimensions. Acta Crystallogr D Biol Crystallogr <br />     new essential cell envelope biogenesis genes Escherichia coli yrbG-yhbG locus. <br />                                                                                                   60(Pt 12 Pt 1):2256&#226;&#8364;&#8220;2268. <br />     Res Microbiol 157(6):547&#226;&#8364;&#8220;558. <br />                                                                                               36. Hung LW, et al. (1998) Crystal structure ATP-binding subunit ABC <br /> 14. Wu T, et al. (2006) Identification protein complex assembles lipopolysac- <br />                                                                                                   transporter. Nature 396(6712):703&#226;&#8364;&#8220;707. <br />     charide outer membrane Escherichia coli. Proc Natl Acad Sci USA 103(31): <br />                                                                                               37. Walker JE, Saraste M, Runswick MJ, Gay NJ (1982) Distantly related sequences     11754&#226;&#8364;&#8220;11759. <br />                                                                                                   alpha- beta-subunits ATP synthase, myosin, kinases ATP-requiring <br /> 15. Chng SS, Gronenberg LS, Kahne D (2010) Proteins required lipopolysaccharide <br />                                                                                                   enzymes common nucleotide binding fold. EMBO J 1(8):945&#226;&#8364;&#8220;951. <br />     assembly Escherichia coli form transenvelope complex. Biochemistry 49(22):           38. Hopfner KP, et al. (2000) Structural biology Rad50 ATPase: ATP-driven conforma- <br />     4565&#226;&#8364;&#8220;4567.                                                                                    tional control DNA double-strand break repair ABC-ATPase superfamily. <br /> 16. Freinkman E, Okuda S, Ruiz N, Kahne D (2012) Regulated assembly trans-                 Cell 101(7):789&#226;&#8364;&#8220;800. <br />     envelope protein complex required lipopolysaccharide export. Biochemistry             39. Hollenstein K, Frei DC, Locher KP (2007) Structure ABC transporter complex <br />     51(24):4800&#226;&#8364;&#8220;4806.                                                                             binding protein. Nature 446(7132):213&#226;&#8364;&#8220;216. <br /> 17. Ruiz N, Kahne D, Silhavy TJ (2009) Transport lipopolysaccharide cell        40. Mourez M, Hofnung M, Dassa E (1997) Subunit interactions ABC transporters:     envelope: long road discovery. Nat Rev Microbiol 7(9):677&#226;&#8364;&#8220;683.                         conserved sequence hydrophobic membrane proteins periplasmic permeases <br /> 18. Sperandeo P, Deh&#195;&#178; G, Polissi (2009) lipopolysaccharide transport              defines important site interaction ATPase subunits. EMBO J 16(11): <br />     Gram-negative bacteria. Biochim Biophys Acta 1791(7):594&#226;&#8364;&#8220;602.                                 3066&#226;&#8364;&#8220;3077. <br />                                                                                                                                                                    <br /> </body></html>