<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <span id='am-26' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ADP</span> Ribosylation Factor Essential Protein Saccharomyces <br />             cerevisiae Encoded Genes <br />  <br />             ADP ribosylation factor (ARF) ubiquitous 21-kDa <span id='am-12' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GTP</span>-binding protein eucaryotes. ARF          identified animal cells protein factor required efficient ADP-ribosylation mammalian G <br />          protein G. cholera toxin vitro. gene (ARFI) encoding protein homologous mammalian ARF          recently cloned Saccharomyces cerevisiae (Sewell Kahn, Proc. Natl. Acad. Sci. USA, 85:4620-4624, <br />          1988). second gene encoding ARF S. cerevisiae, ARF2. ARF genes 28 <br />          centimorgans chromosome IV, proteins encoded 96% identical. <br />          Disruption ARFI causes slow growth, cold sensitivity, sensitivity normally sublethal concentrations <br />          fluoride <span id='am-32' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-33' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> medium. Disruption ARF2 causes detectable phenotype. Disruption genes <br />          lethal;  ARF essential mitotic growth. ARF1 ARF2 proteins functionally homologous, <br />          phenotypic differences mutations genes accounted level          expression; ARFI produces approximately 90% total ARF. revertants fluoride sensitivity          arfi null mutation ARFI-ARF2 fusion genes created gene conversion event deleted <br />          ARFI sequences repaired recombination ARF2. <br />  <br />    ADP ribosylation factor (ARF) originally identified          protein (39). level sequence homology ARF protein cofactor required efficient vitro ADP               proteins divergent organisms suggests ARF <br /> ribosylation stimulatory regulatory component                fundamental role cellular physiology. inter- <br /> adenylate cyclase, <span id='am-6' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-7' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span>Gas</span>, cholera toxin (17, 36). ARF              ested using yeast genetics determine role. Studies <br /> itself 21-kDa GTP-binding protein active               yeasts function low-molecular-weight, <br /> cholera toxin reaction GTP-liganded state (18).            GTP-binding proteins, e.g. RAS], RAS2 (45), YPTI (38), recent reviews role ARF cholera toxin             SEC4 (32), allowed initial description mutant <br /> reaction, reference 16 R. Kahn, Methods Enzymol.,              phenotypes identification functionally press. Based vitro ADP-ribosylation assay (36),           related gene products. role ARF protein ARF purified number tissues, including              protein secretory pathway recently described, <br /> bovine brain, ARF represents 1 2%            deduced phenotype yeast ARF mutants (43). <br /> total cell protein.                                                      paper report presence second ARF gene <br />    ARF protein encoded genes cows           Saccharomyces cerevisiae encodes protein func- <br /> (26, 39) humans (3). bovine ARFI gene used              tionally homologous encoded ARFI demon- probe isolate homologous yeast gene, ARFI (39),             strate ARF gene required viability. <br /> antibodies conserved region ARF protein               results evidence ARF essential used detect ARF organisms (19),                   function normal cell growth. <br /> suggesting ARF ubiquitous protein eucaryotes. cases, antibodies specifically recognize proteins                           MATERIALS METHODS apparent molecular weight 20,000 24,000.                                                                              Strains plasmids. yeast strains plasmids used <br /> samples, including purified bovine brain ARF, anti-ARF-                listed Table 1. Strains bearing disruptions ARF <br /> reactive species ran sodium dodecyl sulfate (SDS) gels               genes constructed direct replacement wild- closely spaced doublet (17). doublet              type chromosomal loci disrupted loci constructed differences migration ARF proteins encoded                vitro (31). ARFI disrupted replacing BglII-XbaI different genes, covalent modifi-               fragment ARF1  Fig. 2) BamHI-SpeI <br /> cation population ARF proteins (e.g., myristyla-               fragment yeast plasmid YEp24, making arfi:: URA3 <br /> tion [191). functional significance ascribed            (pRB1292),  making ends flush T4 polymer- <br /> different forms ARF.                                                    ase, EcoRV-HincII fragment yeast plasmid <br />    yeast gene ARFI encodes protein highly homolo-                    YEp6, making arf7::HIS3 (pRB1293). ARF2 disrupted <br /> gous (74% <span id='am-51' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence identity) bovine ARF                  replacing XbaI-XbaI fragment ARF2                                                                            NheI-SpeI fragment YEp24, making arJ2:: URA3 <br />                                                                            (pRB1295). GALI-ARFJ plasmids constructed   * <br />     Corresponding author.                                                  cloning NlaIV HaeIII fragment ARF1   t Present address: Department Biochemistry Biophysics,            pUC19 HincIl site. plasmid cut BamHI <br /> University California, San Francisco, CA 94143.                         HindIII ARFI fragment ligated   t Present address: Department Genetics, Stanford University, <br /> Stanford, CA 94305.                                                        BamHI HindIII sites pBM272 (15) pRB1301. <br />   &#194;&#167; Present address: Department Biology, Johns Hopkins              AatII-HindIII fragment pRB1301 inserted University, Baltimore, MD 21218.                                           YCp402 (21) pRB1302. <br />                                                                     6690 <br />  VOL. 10, 1990                                                     ADP RIBOSYLATION FACTOR S. CEREVISIAE                     6691 <br />  <br />                                                       TABLE 1. Strain list <br />   Strain                                                             Genotype <br />   plasmid <br /> Strain <br />   DBY4974            MATot his3-A200 leu2-3,112 Iys2-801 ura3-52 Gal&apos; <br />   DBY4975            MATa ade2-101 his3-A200 leu2-3,112 lys2-801 ura3-52 Gal&apos; <br />   DBY5305            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::URA3 <br />   DBY5306            Ura- fluoride-resistant revertant 5305 <br />   DBY5307            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::URA3 <br />   DBY5308            Ura- fluoride-resistant revertant 5307 <br />   DBY5403            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 arfl::his3 arJ2::ura3 (pRB1302) <br />   DBY5406            MATa his3-A200 leu2-3,112 lys2-801 ura3-52 (pRB1301) <br />   DBY5457            MATa/MATa ade2-1OJ/ADE2 his3-A2001his3-A200 leu2-3,11211eu2-3,112 ura3-521ura3-52 arfi:: URA31ARFI <br />   DBY5458            MATa/MATTa ade2-101/ADE2 his3-A2001his3-A200 leu2-3,1121leu2-3,112 lys2-801/LYS2 trpl-l/trpl-l ura3-521ura3-52 <br />                        arf7::HIS31ARFI <br />   DBY5459            MATa/MATTa ade2-JO1/ADE2 his3-A2001his3-A200 leu2-3,1121leu2-3,112 lys2-801/LYS2 trpl-l/trpl-l ura3-52/ura3-52 <br />                        arfl::HIS31ARFJ arJ2::URA31ARF2 <br /> Plasmid <br />   pRB1292            arfl:: URA3 <br />   pRB1293            arfl::HIS3 <br />   pRB1295            arf2:: URA3 <br />   pRB1297            ARFI YCp5O <br />   pRB1301            gall::ARFI URA3 CEN <br />   pRB1302            gall::ARFI LEU2 CEN <br />   pRB1306            ARF2 YCp50 <br />   pRB1344            ARFI EcoRI-BgIII fragment <br />   pRB1345            Plasmid recovered DBY5308 <br />   pRB1346            Plasmid recovered DBY5306 <br />  <br />  <br />   Media, genetic techniques, transformation. Media         ments relative amounts ARF1 ARF2 <br /> yeast growth sporulation described Sherman          proteins estimated tested known amounts et al. (40), 5-fluoro-<span id='am-38' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span>orotic acid</span> plates, described     bacterially expressed ARF1. Boeke et al. (5), YEPGal, identical YEPD          DNA hybridizations sequencing. ARF2 gene 2% galactose substituted glucose. Yeast     cloned hybridization radiolabeled ARFI probe growth, mating, sporulation, tetrad analysis          yeast genomic DNA library. 100,000 colonies formed described Sherman et al. (40). Growth              YCp5O-based yeast DNA library (29) probed plates scored spotting suspensions cells sterile      EcoRI-PstI fragment ARF labeled 106 cpm/ml nick <br /> water plates 32- 48-point inoculator. Yeast         translation (27). Hybridization 37&#194;&#176;C 6x SSC (lx <br /> cells transformed DNA lithium acetate             SSC 0.15 M NaCl plus 0.015 M sodium citrate)-5x <br /> method Ito et al. (14), modified described (42).      Denhardt solution-0.5% SDS-50% formamide-50 mM KPi- <br /> Transformants plated synthetic complete (SC)            100 ,ug carrier DNA ml. filters washed dium lacking appropriate nutrient select cells carrying     370C 0.5x SSC-0.1% SDS. positive clones transformation marker.                                         analyzed restriction digestion DNA hybridiza- <br />    Preparation antibodies immunodetection ARF.         tion.  24 ARF remaining 2 <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> corresponding amino acid residues 159 173         ARF2. strands NarI fragment containing ARF ARF2 purchased Peptide Technologies,              entire coding region ARF2 sequenced dide- <br /> Washington, D.C. peptide contains sole cysteine        oxy-chain termination method (33). ARF sequence, allowing conjugation carrier keyhole             Total yeast genomic DNA prepared described <br /> limpet hemocyanin described Green et al. (12). Rabbits       previously (9). Restricted DNA transferred agar- immunized multiple intradermal injections             ose gels described Southern (41). Hybridizations conjugated peptide complete Freund adjuvant            32P-labeled probes performed conditions  <br /> boosted 2 weeks described Mumby et al. (24).           scribed Wahl et al. (46); washes carried 2x serum rabbit R-16 used experiments.            SSPE (lx SSPE 0.18 M NaCl, 10 mM sodium phosphate, <br />    Total yeast protein extracts prepared harvesting        1 mM EDTA [pH 7.7])-0.1% SDS 650C. 32P-labeled <br /> approximately 2 x 107 cells centrifugation, resuspending        probes prepared random hexamer labeling  20 uil Laemmli sample buffer, immediately boiling        striction fragments low-melting-point agarose (11). 3 min. Glass beads added samples                Cloning afl::URA3 intragenic revertant loci. ARFE vortexed high speed 2 min. Fifty microliters        loci independent Ura- arf7::URA3 revertants <br /> sample buffer added, supernatant fraction              (DBY5306 DBY5308) cloned gene eviction (28) <br /> removed boiled  aliquot loaded          follows. Plasmid pRB1344 inserting SDS-polyacrylamide (15%) gel. Proteins electrophoret-         EcoRI-BgII fragment ARF) yeast-integrating plas- <br /> ically transferred nitrocellulose membrane, ARF           mid YIp5. plasmid cut NheI trans- detected incubation membrane anti-ARF           formed Ura- revertants, selecting URA3 <br /> primary serum followed &apos;25I-labeled protein (6).         marker plasmid. DNA isolated antibody R-16 functioning linear range experi-       transformants revertants. DNA  6692     STEARNS ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />                   1 GCCTGCAGGT ACGCCCCTTT ATTTGATCAG GAAGCCGTAT TGATTATCTA ATAGGGCCTA <br />                  61 GTTATCCTAA TTGTGGGGAG TCGAGCAGTA CGGCTCTGAT GTTTTTCGAA CGAAGATAAG <br />                121 GAGTTGACAT ACAAAGTCAA CAGAAGTTCT TCTTGTTAGC GTCTCTGTGC TCAATATCTC <br />                181 TCTTTTTTTC TTTAAGTAGT AATTACTAAC ATCAGCCAAC CAATAGAGAT AAAAAAAAAA <br />                241 GGAATTAAGA TTTCATAGAG AAAAGATGGG TCTATACGCT TCTAAGTTAT TCAGCAATCT <br />                  1                            M G    L Y     S K L F      S N L <br />                                                        (F) <br />                301 TTTTGGCAAC AAAGAGATGC GTATACTTAT GGTTGGTCTA GATGGTGCCG GTAAGACCAC <br />                 13 F G N      K E M R      L M     V G L     D G G      K T T <br />  <br />                361 CGTTTTGTAC AAGTTGAAGT TGGGCGAAGT TATCACTACC ATTCCAACCA TTGGTTTCAA <br />                 33 V L Y      K L K L      G E V     T T     P T      G F N <br />  <br />                421 CGTTGAGACT GTCCAATATA AGAACATTTC CTTCACTGTC TGGGACGTCG GTGGACAAGA <br />                 53 V E T      V Q Y K      N S     F T V     W D V G      G Q D <br />  <br />                481 CAGGATTAGA TCTTTATGGA GACACTACTA CAGAAACACC GAAGGTGTTA TTTTTGTCAT <br />                 73 R R      S L W R      H Y Y     R N T     E G V      F V                                                                                                                          (V) <br />                541 CGATTCCAAC GATAGATCGC GTATTGGTGA AGCCAGAGAA GTCATGCAGA GAATGCTGAA <br />                 93 D S N      D R S R      G E     R E     V M Q R      M L N <br />  <br />                601 TGAAGATGAA TTGAGAAATG CTGTCTGGTT AGTCTTCGCT AACAAACAAG ATTTGCCAGA <br />                113 E D E      L R N      V W L     V F     N K Q D      L P E <br />                                             <br />                661 AGCCATGTCT GCTGCTGAAA TCACCGAAAA ATTAGGTTTA CATTCTATTA GAAACCGTCC <br />                133 M S      E      T E K     L G L     H S R      N R P <br />  <br />                721 ATGGTTTATC CAGTCTACTT GTGCAACCTC GGGTGAAGGT                              CTGTACGAAG GTCTGGAGTG <br />                153 W F       Q S T C     T S     G E G                                  L Y E G      L E W <br />                                     <br />                781 GTTAAGCAAC AACTTGAAGA ATCAATCCTA ATCTAAATCT                              GTATAGAACG TTTAGTCATG <br />                173 L S N       N L K N     Q S <br />                               (S)         (S) (T) <br />                841 CGCACCTTGT GTGTTTTGTT TCTAGATTGT TTTATTTTTA                              TGATTGTTGA AGATATAAAC <br />                 901 CACTGTATAG TTGTATAAGA TAGGATAATG ATGGTGCACT GAAAATAAAC TTACTAGCTC <br />                 961 TTTAATATTG CAACGGCTTG TAACGGGCGA CTCTAGAGGA TCCCCGGGTA CC <br />   FIG. 1. DNA sequence ARF2 gene predicted protein sequence. differences predicted ARFI ARF2 <br /> proteins indicated parentheses ARF2 sequence. residues, predicted total protein length, identical ARFI. GenBank accession number ARF2 sequence M 35158. <br />  <br />  <br /> cut SphI, diluted, ligated, used transform Esch-             ARF1 locus  . ARF+ phenotype erichia coli, selecting ampicillin resistance. cases          intragenic revertants linked ARF1 locus, recovered plasmids (pRB1346 DBY5306                         revertants phenotypically Ura-. results <br /> pRB1345 DBY5308) expected structure.                    suggested reversion event gene conversion <br /> EcoRI-BgIII fragment plasmid cloned                     arfi:: URA3 mutation homologous sequences M13tgl3O sequenced dideoxy-chain termination                  yeast genome, possibly second ARF gene, resulting loss <br /> method (33).                                                             URA3 insertion concomitant gain ARF func- <br />                                                                          tion. isolate homologous gene, used ARFI <br />                          RESULTS                                         gene probe yeast genomic DNA. labeled DNA <br />                                                                          fragment containing entire ARFI coding sequence   genes encode ARF S. cerevisiae. yeast gene                  used screen yeast genomic DNA library plasmid <br /> ARFI cloned virtue homology bovine                  YCp5O (29). 26 clones isolated way, 24 ARF gene encodes protein 74% identical                  contain inserts ARF1 locus; 2 appeared mammalian ARF (39). lines evidence indicated                  different ARFI related  sequence <br /> presence second ARF gene yeast cells.  DNA                 ARFI-homologous locus determined hybridization experiments showed second locus hy-                 contain open reading frame 91% identical bridized ARFI probe stringent hybridization                ARFJ <span id='am-43' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-44' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> level called ARF2 (Fig. <br /> conditions  shown). Second, characterizing null mu-               1). significant homology genes <br /> tations ARFI, revertants arfi::URA3                    outside coding sequences. worth noting deletion-insertion mutation appeared occurred               homology ARF1 ARF2 genes  VOL. 10, 1990                                                         ADP RIBOSYLATION FACTOR S. CEREVISIAE                   6693 <br />  <br />          TABLE 2. Genetic mapping ARFI ARF2a                                                        IV <br />   Interval             PD:              NPD:                     arfl-arJ2               51                0                      65 <br /> arfl-cdc2               12                1                      17 <br /> arf2-cdc2               41                0                      16                           ARF1 <br /> arf)-cdc9               44                0                      21 <br /> arf2-cdc9               16                0                       7                           cdc36    - <br />  <br />  <br /> arfl-cdc36              36                0                      12                            cdc9 <br /> arJ2-cdc36              16                0                       6                            ste7 &lt; <br /> cdc2-cdc36              11                0                      11 <br /> cdc9-cdc36              22                0                       1 <br />  PD, Parental ditype; NPD, nonparental ditype;   Tr, tetrad.                                ARF2 <br /> certain restriction sites coding sequences                                      cdc2 <br /> conserved, possibly explaining failure identify ARF2 experiments ARFI originally cloned (39). <br />    predicted ARF2 protein identical length ARF1 protein 96% identical protein sequence, dif- <br /> fering 7 181 residues (Fig. 1). seven <br /> differences conservative amino acid changes, seven C terminus protein, region <br /> relatively divergent low-molecular-weight, <br /> GTP-binding proteins. GTP-binding consensus regions putative myristylation site present ARF                  FIG. 2. Genetic map location ARF ARF2 chromo- <br /> conserved ARF2.                                                    IV. point crosses used establish order    Genetic mapping ARF ARF2. DNA blot yeast               ARF, cdc9, ARF2; ARF, cdc9, cdc36; ARF2, cdc2, <br /> chromosomes separated CHEF electrophoresis                     cdc36. <br /> probed fragment ARFI gene determine chromosomal locations ARF genes. condi-                strain arf mutations trans: tions used allowed hybridization probe ARFE            ar]fl::HIS3 mutation copy chromosome IV ARF2. hybridization signal observed             diploid, arJf2::URA3  DBY5459 sporulated <br /> experiments chromosome IV, indicating                tetrads dissected. 50 tetrads scored, 24 genes chromosome. Strains bearing geneti-             viable spores, arfl::HIS3 arJ2:: URA3 muta- <br /> cally marked ARF loci crossed             tions parental configuration, 26 strains bearing chromosome IV genetic markers. Segre-              viable spores, markers tetratype configura- <br /> gants appropriate genotype crosses             tion. cases missing spore tetratype tetrads crossed order linked genes             inferred double mutant, indicating arf] arJ2 <br /> point crosses. tetrad data presented Table 2         double null mutant inviable. ratio parental ARF genes separated approximately 28                   tetratype tetrads consistent genetic distance <br /> centimorgans left arm chromosome IV,              ARF ARF2 determined mapping crosses. ste7-cdc9-cdc36 gene cluster. ARF2 centromere          strain disruption mutations cis <br /> proximal, cdc2 cdc36, ARF centromere               configuration produced parental tetratype tetrads distal. arrangement depicted Fig. 2. ARF           expected ratios (data shown); case genes occupy map positions distinct mapped             parental tetrads viable spores. parental ditype <br /> yeast genes (23) criterion represent newly                tetrad DBY5459 subjected DNA blotting anal- <br /> identified genes.                                                     ysis, probes ARF ARF2 (Fig. 4). <br />    functional ARF gene required viability.        disruption mutation segregated 2+:2-, step determining function ARF            spores ARF genes wild type genes, constructed deletion-insertion mutations            disrupted. genes. case large portion coding                double-disruption experiment shows spore <br /> sequence replaced yeast marker gene (Fig. 3).              lacking functional ARF gene unable form colony, arfl disruption constructions (arf7:: URA3                    suggesting ARF required growth. sought arfl::HIS3), differing inserted marker,              demonstrate requirement ARF independent spore <br /> transformed separately wild-type diploid strain                  germination. strain constructed DBY1830 replaced chromosomal ARFE                  complete ARF gene control inducible GAL) <br /> locus (31). resulting strains (DBY5457 DBY5458)               promoter. level transcription promoter sporulated tetrads dissected. spores           high galactose carbon source low viable ar)fl:: URA3 arfl::HIS3              glucose carbon source;  growth carbon <br /> structions, disruption markers segregated 2+:2-          source dependent ARF required mitotic growth. <br /> total 39 tetrads examined. DNA hybridization experi-               yeast centromere plasmid pRB1302 contains ARFE <br /> ments showed Ura+  segregants lacked               coding sequence 3&apos; noncoding region fused GAL) <br /> wild-type copy ARF locus  .                          promoter, LEU2 gene. pRB1302 trans- <br />    arf2:: URA3 disruption construction transformed            formed diploid strain phenotypically Gal&apos; diploid ARFlarfl::HIS3 strain, replaced           ARF genotype ARF)/arfl: :HIS3 ARF21arJ2:: URA3. wild-type ARF2 loci, resulting strain DBY5459.          transformant sporulated tetrads dissected <br />  <br />   FIG. 3. Restriction maps ARFI ARF2 disruption constructions. insert fragments (containing URA3 HIS3) scale; sizes indicated separately. <br />  <br />  <br />                                                                         rich medium galactose carbon source (YEP- <br />                                                                        Gal). conditions allow growth haploid <br />                                                                        segregant lacking chromosomal copies ARF,                                        b c d                         carrying plasmid galactose-inducible ARFI <br />                                                                        gene. segregant phenotypically , Ura+,                                                                        Leu+ (DBY5403) checked DNA hybridization                                                                        confirm plasmid-borne ARFI gene                                                                        complete ARF gene (data shown). ability                          ARF2-         *                               strain grow different carbon sources tested. <br />                                                                        Growth galactose medium appeared normal,                                                                        slower wild type. Growth glucose medium <br />                                                                        completely inhibited (Fig. 5). examined growth <br />                                                                        strain liquid medium  shifted <br />                                                                        YEPGal YEPD medium, growth continued 7 h <br />                                                                        leveled  presumably reflecting depletion ARF <br />                                                                        protein. morphological changes observed cells <br />                          ARF1-4 0                                      cultures. cells DBY5403 culture                                                                        inviable 8 h shift glucose; wild-type cells contin- <br />                                                                        ued grow exponentially conditions.                  arfi ::HIS3-                                       incorporating 0.2% glucose 2% galactose <br />                                                                        medium improved growth rate strain DBY5403. Pre- <br />                                                                        sumably improvement reduction level <br />                                                                        expression GAL] promoter, overexpression                                                                        ARF1 protein promoter wild-type cells                                                                        deleterious  . <br />                                                                          Phenotypes arf mutants.   Growth defect. Dissection                                                                        tetrads derived diploids bearing arf7 arJ2 null <br />                                                                        mutations revealed obvious growth defect arfi null <br />                                                                        segregants. Spore colonies strains carrying arfl dis- <br />                 arf2::URA3-         IW                                 ruption marker smaller ARFI siblings. <br />                                                                        contrast, arJ2 null spore colonies indistinguish- <br />                                                                        able size wild-type spore colonies. growth rates <br />   FIG. 4. DNA blot arf disruption strains. Genomic DNA         wild-type, arfi null, arJ2 null strains liquid medium <br /> strains parental ditype tetrad DBY5459 cut PstI   confirmed observations; wild-type arJ2 null strains <br /> probed mixture ARFI ARF2 probes.                         generation time approximately 85 min 30&#194;&#176;C, <br />  VOL. 10, 1990                                                        ADP RIBOSYLATION FACTOR S. CEREVISIAE                   6695 <br />  <br />  <br />                                                    Glucose                             Galactose <br />  <br />  <br />                    DBY5403                7 <br />  <br />                    DBY5406 <br />  <br />  <br />   FIG. 5. Phenotype ARFI depletion overexpression. DBY5403 arfl::HIS3 arf2::URA3 (pRB1302) DBY5406 ARFI ARF2 <br /> (pRB1302) inoculated SC-ura medium containing indicated carbon source incubated 3 days 30&#194;&#176;C. <br />  arfl null strain generation time 125 min.       wells plates seeded lawns wild-type arfi arfl null phenotypic defect strikingly accentuated           null mutant cells. zones inhibition caused incubation lower temperatures. 14&#194;&#176;C, wild-type          EDTA dependent concentration EDTA <br /> arJ2 null strains grow generation time 8 h,        used genotype cells lawn.  arfi null strain doubled 17 h did double         lowering extracellular magnesium concentration 72 h incubation. Microscopic               EDTA does mimic effect fluoride. Chelation examination arfl null cells 18 h 14&#194;&#176;C did        aluminum formation aluminum fluoride complexes reveal cell cycle arrest phenotype. Examination           hours days possibility effect age morphologies nucleus DNA-specific dye                fluoride plates. <br /> 4&apos;,6-diamidino-2-phenylindole microtubule cytoskel-             (iii) Overexpression lethality. Low-copy-number (CEN) <br /> eton antitubulin immunofluorescence did reveal            high-copy-number (2,um) plasmids contained significant structural defects 30 14&#194;&#176;C  shown).     ARFI ARF2 coding sequence constructed <br />    (ii) Sensitivity fluoride ion. observed   used transform wild-type strain. ARFI <br /> trimeric GTP-binding proteins activated vitro         ARF2 centromere plasmids effect, 2,im- <br /> combination aluminum fluoride ions,               based plasmids severely inhibited growth rate wild activation reported monomeric GTP-                 type. particular, ARFI-2,um plasmid (pRB1300) <br /> binding proteins, including ARF. Interestingly,             yielded transformants lower frequency arfl null mutants sensitive normally           vector insert, transformants did grow <br /> sublethal amounts potassium fluoride medium.               colonies grew slowly. ARF2-2,um plasmid <br /> Wild-type haploid yeasts typically resistant approxi-         (pRB1307) similar, severe, effect. mately 60 mM KF YEPD, arfi null mutants              similar result obtained centromere plasmid <br /> inhibited growth little 15 mM. result          (pRB1301) carrying GALI-ARFJ construction described obtained sodium fluoride, potassium chloride           wild-type strain bearing plasmid grew normally sodium chloride effect. 40 mM F-            medium containing glucose unable grow reliable concentration easily distinguishing arfi          galactose sole carbon source, GAL] pro- <br /> null mutants wild type. cell cycle-specific arrest           moter maximally induced (Fig. 5). growth control detected wild-type culture treated         strain bearing GAL vector insert 100 mM KF arfl null mutant culture treated 40             affected change carbon source. <br /> mM KF. Addition AlCl3 range 0 10 mM                  Antibodies ARF gene products. Antibodies <br /> plates containing various KF concentrations effect            yeast ARF generated immunizing rabbits <br />  increasing precipitate higher            synthetic peptide coupled keyhole limpet hemocy- <br /> concentrations). aluminum ions required              anin. sequence used immunization identical effect,  likely sufficient    ARFI ARF2;  antibodies generated present media. arJ2 null mutants indistinguish-          recognize proteins equally  polyclonal antise- <br /> able wild type terms sensitivity fluoride ions.        rum derived rabbit R-16 used experiments <br />    YEPD plates containing 40 mM KF NaF lost ability           recognized -21-kDa protein yeast extract, support growth wild-type strains 5 7 days        protein higher molecular weight. Peptide preparation. ability support growth wild         competition experiments showed 21-kDa signal <br /> type, arfl null strains, restored adding         diminished peptide. ARF protein expressed level <br /> Mg2+ plate prior use (typically 0.1 ml 1 M MgCl2        0.03 0.1% total cell protein determined 35-ml plate). probable cause effect          quantitation immunoblots purified recombinant <br /> complexing precipitation Mg2+ ions F-.                ARFI standard. R-16 polyclonal antiserum raises possibility arfl null defect actually       used examine relative contributions ARFI sensitivity reduced Mg2+ levels media. test        ARF2 total ARF cell. ARFI <br /> possibility, effect EDTA growth examined.              ARF2 identical length nearly identical se- <br /> EDTA solutions different concentrations placed            quence, slightly different mobilities appro- <br />  6696     STEARNS ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                                                                           Suppression fluoride supersensitivity arfi null <br />                            ARFi ARF2                                    mutants. strong fluoride supersensitivity arfi null <br />                                                                         mutants provided genetic selection increased ARF <br />                                                                         function. Spontaneous revertants able grow plates <br />                          &apos;ARFl arf2                                     containing 40 mM KF selected arfl::URA3 <br />                                                                         mutants DBY5305 DBY5307. revertants arose                                                                         frequency 1 107 cells plated.                            arfl1 ARF2                                   revertants isolated, slow growth cold sensitivity <br />                                                                         phenotypes arfl:: URA3 mutant suppressed. <br />                                                                         Surprisingly, 29 102 independent pseudorevertants iso- <br />                                                 yARF2                   lated <span id='am-63' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> auxotrophs, phenotypically losing <br />                                                                         URA3 marker inserted ARF.                                                                         Ura- revertants backcrossed wild-type strain,                            arflt arf2-        p ARF1                    parental ditype asci recovered; spores showed <br />                                                                         wild-type resistance fluoride. indicated    FIG. 6. ARF protein levels wild-type mutant strains.          mutation causing suppression tightly linked ARF protein extracts strains indicated genotypes     ARF gene itself. addition, suppression subjected electrophoresis, blotted, probed anti-ARF         dominant; Ura- pseudorevertants crossed antibody described Materials Methods. antibody used       arfi:: URA3 strain, resulting diploids fluoride resis- directed peptide epitope ARF identical   tant. simplest explanation results ARFI ARF2 proteins.                                                 disrupted arfl gene repaired recombination                                                                         ARF2 sequences. unlikely,                                                                          arfi:: URA3 construction deleted sequences past 3&apos; end <br /> priate electrophoretic conditions. viable spores              ARFJ coding sequence lacked homology tetratype tetrad DBY5459 examined                     ARF2 locus 3&apos; end. <br /> immunoblotting (Fig. 6). arfl: :HIS3 spore clearly lacked              determine event occurred Ura- rever- major band ARF reactivity, retained minor band             tants, genomic DNA wild type, arfl:: URA3, migrates slightly faster ARF. minor band                 Ura- revertants subjected DNA blot analysis. reproducibly appeared missing arj2:: URA3                  probe experiments EcoRI-PstI fragment strain, proximity major band              ARF containing entire coding sequence  difficult resolve. minor band overproduced                   hybridization conditions used, hybridizes ARFI severalfold strain carrying ARF2 centromere                   ARF2. internal BgIH site present homologous <br /> plasmid (presumably present single copy,                   location ARF ARF2 (Fig. 3) destroyed tested), demonstrating minor band              arfl:: URA3 allele ligation BamHI end product ARF2 gene. Assuming R-16                    fragment carrying URA3. BamHIIBglll junction formed <br /> antibody recognizes forms ARF equally, minor                cut enzyme. pattern hybridization <br /> band estimated constitute approximately 10%               confirmed ARF BglII site missing total ARF protein. did appear increase            arfi:: URA3 strain surprisingly reappeared rever- expression ARF2 protein arfi disruption mu-               tants  shown). Hybridization ARF probe tants. findings confirmed ARF-                 number genomic digests revealed restored BglII <br /> ARF2-specific antibodies (R. Kahn, unpublished observa-                 site revertants derived homologous DNA tion).                                                                  ARF2 locus. resulting genes encode <br />    ARF ARF2 proteins functionally homologous.               fusion proteins amino-terminal portion derived near identity ARF ARF2 proteins suggested                ARF remainder derived ARF2. digests, perform identical functions                ARF2 gene remained unchanged revertants. genetic difference genes accounted             results suggested repair arfi:: URA3 loss lower level expression ARF2 gene.             URA3 marker accomplished gene conver- case, simply increasing copy number ARF2                sion event: nonreciprocal transfer information gene suppress phenotypic defects arfi null             ARF2 arfl:: URA3. <br /> mutant. hypothesis tested transforming arfl null               crossover points define extent <br /> mutant strains plasmids bearing ARF2 gene.             replacement arfi:: URA3 DNA ARF2 DNA, <br /> yeast-integrating plasmid bearing ARF2 integrated            nonhomologous URA3 insert. DNA <br /> chromosomal ARF2 locus arf) null strain homolo-                blot analysis revealed converted arfi loci strains <br /> gous recombination, resulting duplication ARF2.                 DBY5306 DBY5308 XbaI site located <br /> Transformants single copy plasmid                near 5&apos; end ARF2 reading frame (Fig. 3), integrated identified DNA blotting. Second,               BglII site ARF2  . site yeast centromere plasmid bearing ARF2 (pRB1306)                     5&apos; crossover 151-bp region introduced arfl null strain; centromere plasmids            restriction sites. site 3&apos; crossover typically maintained 1 2 copies cell (8).             determined lie XbaI site <br /> cases, increased ARF2 gene dosage sufficient                 located 52 bp 3&apos; end ARF2 reading frame completely suppress arfi null phenotypes slow              site acquired revertant loci. <br /> growth, cold sensitivity, fluoride supersensitivity.              determine directly structure revertants, ARF2 copy number little doubled,             arfi loci revertant strains cloned integrat- <br /> sufficient ARF2 protein perform ARF functions             ing plasmid carrying ARF homology immediately adjacent <br /> cell, indicating ARFE ARF2 proteins func-              converted arfl, isolating genomic DNA transfor- <br /> tional homologs.                                                        mants, cutting ligating DNA  <br />  <br />    FIG. 7. Structure ARFJ/ARF2 fusions. genomic DNA ARFI locus intragenic arfl:: URA3 revertants (DBY5306 DBY5308) undergone recombination event arfl::URA3 ARF2 cloned sequenced. locations 5&apos; crossover events revertants shown X&apos;s ARF2 arf7:: URA3 sequences  DBY5306 site 5&apos; . boxed bases ARF2 sequence shown differ ARF1. <br />  <br /> verted loci recovered E. coli plasmid         copies ARF2 phenotypically wild type Materials Methods). DNA sequence loci               ARF protein ARF ARF2 cell (Fig. 6). revealed  predicted, ARFJ/ARF2 fusion               alternative explanation results ARF2 <br /> genes, crossover points expected region (Fig.        protein active way ARFI protein 7). actual site crossover approxi-          able perform necessary functions lower protein <br /> mated high degree sequence identity            levels. Second, function ARF2 obscure, arJ2 null <br /> ARFI ARF2 region. strain DBY5306,                mutants phenotypically indistinguishable wild type, <br /> crossover point narrowed region 39 61 bp 5&apos;         ARFI ARF2 proteins appear functionally BglII site, strain DBY5308,               homologous. known examples yeast cells narrowed region 6 22 bp 5&apos; BglII site (Fig. 7).       duplicated genes genes                                                                      regulated differently, cytochrome c (20),                         DISCUSSION                                   proteins directed different cellular compart- <br />    paper demonstrate ARF encoded           ments, citrate synthase (30). presently <br /> genes S. cerevisiae, genes        lack information regulation ARF genes functional viability, ARF proteins            localization ARF proteins S. cerevisiae, arfi <br /> coded genes functionally homologous.                suppression results indicate difference regulation results evidence essential role        localization overcome simple duplication ARF cellular physiology. begun analyze            ARF2 gene. duplicated yeast ax-tubulin genes, TUB] phenotypes arf mutations, gaining information           TUB3, closely resemble ARF genes number function ARF available previous studies          respects. tubulin proteins functionally homol- vitro properties protein.                          ogous, disruption TUB], makes majority    ARF ARF2 genes encode proteins 96%               tubulin, lethal, disruption TUB3 identical, mutations genes resulted different      minor phenotypic consequences (35). Strikingly, TUB) phenotypes. likely genetic difference      TUB3, like ARF ARF2, genetically linked difference level expression ARF                separated approximately 38 centimorgans chro- <br /> proteins. approximately 90% total ARF               mosome XIII (35). <br /> protein produced ARF) gene, remainder              ARF genes located chromosome IV, produced ARF2; single extra copy ARF2                  28 centimorgans apart, recombine  <br /> suppress phenotypic defects caused deletion                clearly demonstrated recovery ARFI/ARF2 <br /> ARFI. raises number points concerning              fusion proteins created vivo recombination strains <br /> ARF genes.  likely excess ARF yeast          selected suppression arpf:: URA3 null phenotype. <br /> cells grown standard laboratory conditions, arf7            independent recombinants sequenced <br /> arf2 double null mutant strains plasmid-borne              shared common features. recombination events  6698     STEARNS ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br /> nonreciprocal; ARF2 locus remained unchanged,             micromolar range.  exogenous Al3&quot; need added fusion protein created arfi locus. 5&apos; ends       common contaminant aqueous solutions recombination events (relative coding strand       (44). contrast trimeric G proteins (13), tests conducted <br /> ARFI) region DNA homology                  purified bovine ARF effect fluoride ARF <br /> ARFJ ARF2 coding sequences, exact location dif-             activity intrinsic fluorescence, using conditions fering recombinants.  3&apos; ends           activating <span id='am-58' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="Guanine" datatype="xsd:string"></span>guanine</span> nucleotides dramatically increased recombination events determined              (17).  direct effect fluoride complex ARF end ARFI coding sequence. 3&apos;            appears unlikely. plausible model fluoride sensitiv- <br /> crossover interesting detectable ho-         ity arf mutants fluoride activates previously <br /> mology sequenced 3&apos; untranslated regions          uncharacterized trimeric G protein S. cerevisiae ARF genes. possible 3&apos; crossover occurred         distinct containing GPAI GPA2 sub- <br /> repeated DNA, Ty, d, elements (4).                     units. ARF normally acted inhibit G protein,    nature essential function ARF S.        absence ARF, fluoride overactivate  Alterna- <br /> cerevisiae? separate experiments (44)         tively, millimolar concentrations fluoride known <br /> ARF involved protein secretion yeast cells        affect protein phosphatases, phosphorylases, ATPases, <br /> ARF protein localized Golgi mammalian cells.           free-metal concentration (34). interesting phenotypes report arfi arJ2 double null        light ability magnesium restore growth mutants consistent  specific cell         wild-type cells agar plates containing fluoride cycle defect associated loss ARF function cells         reported effects yeast cells yptl mutations Ca2&quot; <br /> depleted ARF repression GALl-ARFI construc-              uptake rescue growth arrest increased extracel- <br /> tion arrest growth points cell cycle.             lular calcium (37). YPTI 21-kDa GTP-binding <br /> phenotypes arfi null mutant, slow growth, cold               protein subcellular distribution apparent role sensitivity, fluoride sensitivity, genetically useful   protein secretion (38) similar recently described revealing function, light role ARF      ARF (43). Ultimately, knowledge basis  <br /> secretion  .                                              creased fluoride sensitivity arfi null mutants provide <br />    ARF originally identified cofactor            insight function ARF secretory pathway. <br /> covalent modification G protein G,S animal cells, interesting note phenotypes                                 observed yeast arf mutants. known <br /> yeast G,, proteins; GPAI involved mating pheromone               <br /> signal transduction (10, 22), function GPA2      <br /> unknown, mutations GPA2 result discernible        <br /> phenotype (25). ARF required GPAI function,           sterility constitutive activation mating path-    <br /> way expected, phenotypes         <br /> observed arfi null mutants conditions      <br /> ARF depletion  Results;  unpublished observation).       caveat interpretation known secretion <br /> defect arf mutants prevent observation                          <br /> morphological changes associated activation         <br /> mating pathway. ARF required solely GPA2            <br /> function, ARF expected dispensable,        shown ARF essential. ruled      possibility ARF involved GPA2 function,          sole function ARF,           addressed vitro GOS ADP-ribosylation         <br /> assay actually relevant vivo function.              <br /> cholera toxin reported effect yeast            <br /> adenylate cyclase protein <span id='am-68' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span> ADP-ribose             identified yeast extracts (7). results clearly      <br /> demonstrate level ARF protein critical          <br /> proper function. underexpression overexpression             ARF protein results impaired growth complete          <br /> cessation growth, depending degree alteration.        <br />    fluoride sensitivity arf mutants intriguing. Al-        model mechanism activation                 <br /> subunits trimeric G proteins fluoride complexes       proposed (2), evidence fluoride                <br /> aluminum fluoride complexes bind activate          <br /> low-molecular-weight GTP-binding proteins. trimer-         <br /> ic G proteins, believed AlF4- active species      activating effect (44), resulting resemblance       compound gamma phosphate GTP.                    <br /> liganded <span id='am-20' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GDP</span> plus A1F4- equivalent liganded GTP            <br /> (1). F- concentrations millimolar range               <br /> required vitro effects, Al3&quot; required         <br /> </body></html>