<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>From Hub Proteins Hub Modules: Relationship Essentiality Centrality Yeast <br /> Interactome Different Scales Organization <br /> Jimin Song, Mona Singh* <br /> Department Science Lewis-Sigler Institute Integrative Genomics, Princeton University, Princeton, New Jersey, United States America <br />  <br />  <br />  <br />      Abstract <br />      Numerous studies suggested hub proteins S. cerevisiae physical interaction network likely      essential proteins. proposed reasons underlying observed relationship topology      functioning subject controversy, recent work suggesting arises participation      hub proteins essential complexes processes.  essential modules distinct network <br />      characteristics, essential proteins differ topological properties non-essential proteins? <br />      aimed advance understanding protein essentiality analyzing proteins, complexes processes      broader functional context considering physical interactions complexes biological <br />      processes. agreement view essentiality modular property, number intracomplex      intraprocess interactions protein better indicator essentiality overall number interactions. <br />       essential complex, essential proteins average interactions, especially <br />      intracomplex interactions, non-essential proteins. Finally, built module-level interaction network      essential complexes processes tend higher interaction degrees network non-essential <br />      complexes processes;  exhibit larger functional cross-talk non-essential counterparts. <br />  <br />   Citation: Song J, Singh M (2013) Hub Proteins Hub Modules: Relationship Essentiality Centrality Yeast Interactome Different <br />   Scales Organization. PLoS Comput Biol 9(2): e1002910. doi:10.1371/journal.pcbi.1002910 <br />   Editor: Frederick P. Roth, Harvard Medical School, United States America <br />   Received August 27, 2012; Accepted December 21, 2012; Published February 21, 2013 <br />   Copyright: &#195;&#376; 2013 Song, Singh. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported NSF ABI-0850063, NIH GM076275, NIH Center Excellence grant P50 GM071508. JS partially <br />   supported Samsung Scholarship Samsung Foundation Culture. funders role study design, data collection analysis, decision   publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: msingh@cs.princeton.edu <br />  <br />  <br />  <br /> Introduction                                                                         proteins essential;  deletion proteins results <br />                                                                                      cell death, optimal growth conditions [11,12]. Early <br />    High-throughput experimental approaches determining                           computational analysis yeast S. cerevisiae protein-protein <br /> protein interactions resulted large-scale cellular networks                  physical interaction network revealed scale-free topology, numerous organisms. Graph-theoretic analyses                            &#226;&#8364;&#732;&#226;&#8364;&#732;hub&#226;&#8364;&#8482;&#226;&#8364;&#8482; proteins interactions, showed <br /> networks great aid advancing understanding                        hub proteins likely essential proteins cellular functioning organization (review, [1]).                   [13]. Numerous subsequent studies confirmed centrality- fundamental discoveries strong relationship                  lethality relationship, yeast [14&#226;&#8364;&#8220;19] topological characteristics cellular networks                     organisms [20]. hand, relationship underlying functioning. example, cellular networks                         observed weak networks consisting interactions <br /> consist tightly clustered groups interacting proteins,                     determined high-throughput yeast hybrid experiments proteins work protein complexes biological                      stronger types networks [16,18,19], processes achieve specific biological functions [2&#226;&#8364;&#8220;7].                         proposed  yeast hybrid networks, observed <br /> orthogonal decomposition reveals recurring                        relationship study bias favoring determination represented topological functional patterns larger                   interactions essential proteins [21].  positive <br /> cellular networks, network motifs [8,9] network                        correlation protein interaction degree essentiality schemas [10] associated dynamic regulatory properties                    generally accepted, numerous reasons proposed shared mechanisms functioning.  revisit                       literature explain relationship. basic structure function relationship                          Initial work suggested high-degree proteins essential <br /> proposed cellular networks number                            role interaction network connectivity [13];  <br /> interactions protein overall functional importance.                    unlikely case subsequently shown    importance gene cell organism                          non-essential hubs just important essential hubs quantitatively measured considering phenotypic effects                     maintaining connectivity, essentiality better correlated <br /> gene deletion disruption. Experimental studies baker&#226;&#8364;&#8482;s                     local, global, measures connectivity protein- <br /> yeast S. cerevisiae demonstrated approximately 19%                  protein interaction networks [17,18]. alternatively proposed <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                1                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                         Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />   Author Summary                                                               cerevisiae protein-protein interaction networks considered <br />                                                                                functional modules derived protein complexes   Network analyses large-scale interactomes                     Gene Ontology   biological process annotations [25]   great aid advancing understanding cellular                         different levels resolution.  address issue study <br />   functioning organization.  examine                    bias, performed analysis additional networks   basic intensely-studied structure function                       removed interactions determined small-scale experiments. <br />   relationships observed cellular networks:                        began examining relationship protein <br />   number interactions protein tendency                    essentiality network modularity. hypothesized   essential. develop new computational                              essentiality modular property, proposed previously <br />   framework systematically analyze essential proteins                       [18], protein&#226;&#8364;&#8482;s intramodular physical interaction degree <br />   cellular context explicitly incorporating <br />                                                                                better predictor protein&#226;&#8364;&#8482;s essentiality   functional information. apply framework                                                                                intermodular physical interaction degree. test  utilized <br />   yeast interactome demonstrate previously <br />   observed positive relationship interaction degree                    biological process functional annotations proteins classified <br />   essentiality largely intramodular interactions.                physical interactions intraprocess interactions processes <br />    essentiality appears modular property                    interprocess interactions processes.   protein complexes broadly biological                         essential proteins tend interactions proteins <br />   processes. essential complex, essential                        functional modules intraprocess <br />   proteins tend interactions, especially intra-                   interaction degree correlated essentiality overall <br />   complex interactions, non-essential proteins.                       degree.  relationship overall <br />   Finally, computationally inferred module-level inter-                   degree essentiality significantly weakened controlling <br />   action network, essential complexes processes tend                    intramodular degree, affected controlling   higher interaction degrees non-essential                     intermodular degree.  direct simple <br />   counterparts. summary, relationship                             manner  essential proteins, essentiality likely <br />   connectivity essentiality                        consequence participation essential modules <br />   protein level, modules module                     consisting functionally similar proteins. <br />   level, complexes processes essential                           ascertain modularity essential <br />   tending interact functional groups.                             proteins roles essential protein complexes <br />                                                                                generally essential biological processes, essentiality property interactions [22].        repeated analysis exclusively focusing proteins <br /> essential protein interactions, organism               protein complexes focusing proteins survive, randomly distributed network;                known protein complexes. hubs tend essential likely participate          essential proteins intraprocess interactions fact essential interactions.  model implies                participate essential protein complexes essential biological <br /> probabilities non-interacting proteins essential              processes include protein complexes;  independent  case [18]. Instead,             modularity protein essentiality appears consequence Zotenko et al. [18] argued correlation degree             protein complexes, broadly biological processes. <br /> essentiality participation essential proteins                   examined complexes contain essential proteins, <br /> essential functional modules consisting groups densely                   compared essential non-essential proteins. clustered functionally related proteins. showed               reasoned relationship essentiality essentiality hubs computationally            interaction degree proteins complexes entirely <br /> extracted modules weakly correlated degree [18].                 consequence complexes essential,  previously essential proteins tended            essential non-essential proteins complex densely connected [15] concentrated                 differ respect degree. contrary, complexes [23,24], suggesting essentiality modular                   essential proteins tend interactions, particularly <br /> property property individual proteins. Building               intracomplex interactions, non-essential counterparts  argued essential complexes tend              protein complexes.  essentiality appears large, proteins larger number                   modular property, degree protein associated interactions, explains hubs tend essential             essentiality essential complexes; hypothesize [19].                                                                          essential proteins play important role maintaining <br />    substantial evidence essentiality modular          structure function complexes. <br /> property protein-protein interaction networks, clear                 Finally, analyzed modules containing essential proteins complexes processes consist entirely essential           context functional modules. inferred <br /> non-essential proteins. essential proteins essential              significant &#226;&#8364;&#732;&#226;&#8364;&#732;cross-talks&#226;&#8364;&#8482;&#226;&#8364;&#8482; protein complexes biological <br /> complex process differ non-essential ones?             processes used build module-level networks, complexes processes contain essential proteins.                complexes processes linked enriched <br /> essential modules distinctive roles cellular networks?         number physical interactions  Using paper, aimed discover  modules,                     module-level networks, uncovered functional modules <br /> essential non-essential proteins differ interaction               essential proteins tend average higher degree; properties, global scale, essential non-              degree module-level network positively correlated essential modules differ network-level properties.                 module essentiality. <br /> accomplish  developed computational framework                       Overall, considering proteins functional context incorporates information functional modules                   yeast interactome, evidence relationship <br /> context network analysis techniques. uncover general                 essentiality network topology different levels robust principles, performed analysis types S.              cellular organization: protein level, protein <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          2                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> complexes, globally module level,                     intraprocess interactions, interprocess interactions, total anno- <br /> complexes processes essential tending interact             tated interactions. considered intraprocess, functional groups.                                                         interprocess total annotated interaction degrees turn,                                                                                 ordered proteins high low degrees respect  Results                                                                         varied threshold number proteins considered,                                                                                 computed fraction essential proteins &#226;&#8364;&#732;&#226;&#8364;&#732;high degree&#226;&#8364;&#8482;&#226;&#8364;&#8482;    analyzed 5640 proteins tested essentiality [12]          &#226;&#8364;&#732;&#226;&#8364;&#732;hub&#226;&#8364;&#8482;&#226;&#8364;&#8482; set. large range thresholds, high degree context large-scale S. cerevisiae protein physical               proteins, ranked intraprocess degree, higher fraction interaction datasets; networks captures different                 essential proteins high degree proteins ranked features biological interactions. network Direct              total annotated degree interprocess degree (Figures 1  , S1   <br /> interaction network, interaction proteins                  S2  ). Pull networks, fraction corresponds direct physical contact; network includes                 essential proteins tends decrease threshold intraprocess, <br /> interactions determined yeast hybrid method                    interprocess, total degree lowered. Direct network types approaches  Materials Methods).                  (Figure 1  ), trend true intraprocess interaction <br /> considered Pull network, interaction                degree notably true total degree; consistent proteins corresponds members multi-                  previous work showing relationship essentiality <br /> protein complex.  considered network consisting               overall interaction degree weak networks consisting of physical interactions BioGRID [26]; case,                 interactions determined yeast hybrid [16,18,21]. <br /> interactions represent direct indirect interactions.              quantify correlation essentiality main body paper, primarily report results              degree, used Spearman&#226;&#8364;&#8482;s rho rank correlation coefficient <br /> Direct interaction network, contains 4031 proteins (898                (SRCC) [29] (Figures 1 (b), S1 (b) S2 (b)), essential) 15,073 interactions. analysis                  SRCC highest intraprocess degree (0.25 Direct, 0.35 described performed Pull                     networks), lower interprocess degree (0 <br /> networks  Table S1) reported Supplementary               Direct, 0.22 Pull 0.21 . note Material. considered additional networks interac-                 protein essentiality binary value tied <br /> tions determined small-scale experiments removed;                  values, possible SRCC achieve value 1. analysis outlined section high-throughput networks                 example, SRCC essentiality annotated degree  detailed figures given Supplementary Material.               Direct network reach maximum value                                                                                 0.7680  e., case essentiality values proteins Categorizing interactions intramodular                                    swapped essential proteins higher degrees intermodular                                                                    non-essential proteins).  disentangle contributions    given interaction network, labeled protein interactions             intraprocess interprocess degree observed correlations, &#226;&#8364;&#732;&#226;&#8364;&#732;intramodular,&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;intermodular&#226;&#8364;&#8482;&#226;&#8364;&#8482; using               computed partial correlations essentiality sources functional data. particular, utilized yeast protein            annotated interactions, controlling intraprocess complex data compiled [27] Gene Ontology   Biological                 interprocess degree. networks Process (BP) annotations [25].  intramodular interactions              controlling intraprocess degree, SRCC total degree <br /> arise intracomplex intraprocess interactions,                essentiality notably diminished, controlling intermodular interactions arise intercomplex interpro-             interprocess degree, SRCC remained high (Figures 1 (b), S1 (b) <br /> cess interactions; separately consider types                    S2 (b)), increased Direct network. <br /> intramodular intermodular interactions. protein complex                    way looking difference intrapro- <br /> data, &#226;&#8364;&#732;&#226;&#8364;&#732;intracomplex&#226;&#8364;&#8482;&#226;&#8364;&#8482; interactions pairs proteins           cess interprocess interaction degree, compared degree participate shared complex &#226;&#8364;&#732;&#226;&#8364;&#732;intercomplex&#226;&#8364;&#8482;&#226;&#8364;&#8482; interac-              distributions essential proteins non-essential proteins <br /> tions pairs proteins         (Figures 1, S1 S2 (c)&#226;&#8364;&#8220;(e)) using Wilcoxon rank sum test. <br /> complex complex.                                comparing degree distributions, included proteins    complicated characterize interactions intramodular          annotated interaction; proteins zero intermodular using BP terms, terms hierarchically              intraprocess interprocess interactions. number <br /> related annotate different numbers proteins,              proteins considered comparing total, intraprocess, general terms. informative specific terms,                   interprocess degree (Figures 1, S1 S2 (c)&#226;&#8364;&#8220;(e)), p-values <br /> considered BP terms annotate 50 proteins                 given comparable. difference number yeast proteome. interaction unannotated unless proteins              interactions essential non-essential proteins annotated specific BP terms.                    significant intraprocess interactions consid- <br /> interaction &#226;&#8364;&#732;&#226;&#8364;&#732;intraprocess&#226;&#8364;&#8482;&#226;&#8364;&#8482; proteins sharing       ered (Figures 1 (d), S1 (d) S2 (d)), compared case specific BP terms. proteins interaction             annotated interactions considered (Figures 1 (c), S1 (c) <br /> annotated specific BP terms share                S2 (c)) interprocess interactions considered interaction &#226;&#8364;&#732;&#226;&#8364;&#732;interprocess.&#226;&#8364;&#8482;&#226;&#8364;&#8482; note physical                (Figures 1 (e), S1 (e) S2 (e)). <br /> interactions largely thought &#226;&#8364;&#732; process,&#226;&#8364;&#8482;&#226;&#8364;&#8482; especially           alternative categorizing annotated interactions compared types interactions [28], significant fraction         interprocess intraprocess, considered case <br /> physical interactions interprocess (Table S2); true         interactions weighted according semantic threshold choosing specific terms increased.                         similarity [30] functional terms annotating                                                                                 proteins. weight range 0 1 proteins <br /> Intraprocess interactions main factor relationship                 sharing highly specific functional terms getting higher scores  protein essentiality interaction degree                             Materials Methods details).  semantic <br />  step relating protein essentiality network               similarity interacting proteins continuous <br /> modularity, protein, computed number                         measure &#226;&#8364;&#732;&#226;&#8364;&#732;intramodularity&#226;&#8364;&#8482;&#226;&#8364;&#8482; interaction.   <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           3                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                         Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 1. intraprocess interaction degree correlated protein essentiality overall interaction degree proteins Direct network, interactions categorized specific BP terms, annotates 50 <br /> proteins.   fraction essential proteins hub proteins proteins considered hub proteins; proteins added groups 50 non-increasing order interaction degree. fraction highest intraprocess degree (green), followed annotated degree (blue) interprocess degree (red). (b) correlations measured SRCCs essentiality intraprocess, annotated interprocess degree. SRCC highest essentiality intraprocess degree. partial correlation computed annotated degree essentiality controlling intraprocess interprocess degree. Starred p-values indicate values w0:05. <br /> (c)&#226;&#8364;&#8220;(e) degree distribution non-essential proteins compared essential proteins (c) annotated, (d) intraprocess, (e) <br /> interprocess degree, respectively. box plot, horizontal bar box corresponds median distribution; ends box indicate quartiles; small circles outliers 2&#226;&#8364;&#8220;98th percentile range. significance difference degree distributions measured Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g001 <br />  <br />  <br /> semantic similarity degree protein defined sum              contribution intracomplex interactions. particular, previously <br /> semantic similarity interactions. Direct, Pull         observed essential proteins tend clustered networks, stronger correlation           essential protein complexes [18,24].  essentiality degree interactions weighted               hypothesized having intracomplex physical interactions semantic similarity just counted (Figure S3).           protein important predicting essentiality having words, proteins having interactions similar               types physical interactions.  defined <br /> functional context likely essential proteins               functional modules comprised protein <br /> having interactions. Altogether, range computational                complexes biological processes corresponding BP terms. <br /> analyses shows large portion observed relationship              previous section, utilized modules derived BP essentiality interaction degree explained             terms. focus modules derived protein complexes, <br /> considering just intraprocess interactions.                                   compiled [27]. begin observing complexes                                                                               enriched essential proteins. particular, correlation intramodular degree protein                       18.60%  1049=5640) proteins essential yeast <br /> essentiality largely complexes, processes                       genome, 37.54%  598=1593) essential considering <br />    Having shown strong correlation intraprocess                   proteins involved set complexes considering. interaction degree essentiality, sought characterize            fact, 57.01%  598=1049) essential proteins involved  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                         4                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> protein complexes, 28.24%  1593=5640)                   dataset, clear positive correlation complex size <br /> proteins set complexes.  conclusions              fraction essential proteins complex (SRCC: <br /> arising analysis protein complexes based                 0.24, p-value: 2e-6).  strong correlation <br /> interaction properties significant fraction essential proteins.        protein essentiality size largest complex    network, derived subnetwork nodes                       belongs, SRCCs 0.25, 0.24 0.24 Direct, <br /> represent proteins involved protein complex edges                   Pull networks, respectively (Figures 2 (b), S4 (b) represent interactions original interactions            S5 (b) ).   relationship strong proteins. Direct network, 35.66%  1364=3825)                      essentiality intracomplex degree networks <br /> interactions intracomplex (Table S2). Repeating analysis               (black vs. green curve Figures 2  , S4   S5  ). performed intraprocess vs. interprocess interactions,                   computed partial correlations essentiality intracomplex physical interactions correlated              annotated interactions, controlling intracomplex protein essentiality annotated physical interactions             degree, intercomplex degree, complex size. (Figures 2  , S4   S5  ).                                            controlling intracomplex degree, SRCC    previously observed strong correlation          total degree essentiality notably diminished  0.17, 0.32 complex size essentiality [19], argued                    0.32 20.01, 0.13 0.14 Direct, Pull essential complexes tend large, proteins                 networks, respectively), controlling intercomplex <br /> tend interactions, driving force             degree complex size, SRCC greatly diminished <br /> relationship essentiality interaction degree.               (Figures 2 (b), S4 (b) S5 (b)).  difference degree <br />  <br />  <br />  <br />  <br /> Figure 2. intracomplex interaction degree correlated protein essentiality overall interaction degree proteins Direct network, interactions categorized using protein complexes.   fraction essential proteins hub proteins proteins considered hub proteins; proteins added groups 50 non-increasing order interaction degree size largest complex protein belongs. relationship protein essentiality interaction degree shown intracomplex interactions (green), annotated interactions (blue) intercomplex interactions (red). relationship protein essentiality complex size shown (black). (b) correlations measured SRCCs essentiality intracomplex degree, annotated <br /> degree, intercomplex degree, complex size. SRCC highest essentiality intracomplex degree. partial correlation computed annotated degree essentiality controlling intracomplex degree, intercomplex degree, size largest complex protein belongs. Starred p-values indicate values w0:05. (c)&#226;&#8364;&#8220;(f) degree distribution non-essential <br /> proteins compared essential proteins complexes (c) annotated degree, (d) intracomplex degree, (e) intercomplex degree, (f) complex size. box plot, horizontal bar box corresponds median distribution; ends box indicate quartiles; small circles outliers 2&#226;&#8364;&#8220;98th percentile range. significance difference degree distributions measured Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g002 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          5                         February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 3. networks (  Direct, (b) Pull  (c) network), set biological processes filtered remove effects complexes, intraprocess interaction degree correlated protein essentiality overall <br /> interaction degree. fraction essential proteins hub proteins function increasing number proteins considered hub <br /> proteins; adding proteins groups 50 non-increasing order interaction degree. relationship protein <br /> essentiality interaction degree shown intraprocess interactions (green), annotated interactions (blue) interprocess interactions (red). <br /> Correlations measured SRCCs essentiality type interaction degree given. <br /> doi:10.1371/journal.pcbi.1002910.g003 <br />  <br /> distribution essential non-essential proteins                       degree (Figure 3  ,(b),(c)). correlations especially weak (Figuress 2, S4 S5 (c)&#226;&#8364;&#8220;(f)) significant considering            Direct network.  Pull networks, <br /> intracomplex degree significant considering                      correlation essentiality interprocess filtered <br /> intercomplex degree. note correlation                interaction degree somewhat higher protein&#226;&#8364;&#8482;s intracomplex degree size complex                  essentiality intraprocess filtered degree. belongs (SRCC: 0.3790, 0.7319 0.7809 Direct, <br /> Pull networks, respectively); stronger                   Essential proteins central essential <br /> correlations Pull networks compared             protein complexes <br /> Direct network expected networks include                     Having shown global analysis proteins complexes indirect  e., complex) interactions.                                  essential proteins tend intracomplex interactions <br />     far, stronger correlation                      non-essential proteins, considered complex <br /> essentiality intramodular degree essentiality                  analysis. hypothesized  essential protein complex, annotated degree focus biological                     essential proteins central higher <br /> process protein complex derived modules. Instead using                    intracomplex degree non-essential proteins. tested biological process protein complex annotations categorize                 hypothesis subset protein complexes member <br /> interactions intramodular intermodular,                    proteins intracomplex interactions. particular, included <br /> considered modules derived network clustering approaches.                  complex test essential proteins particular, applied state art SPICi network                    non-essential proteins, intracomplex <br /> clustering algorithm [31], categorized interactions                  interactions. Table 1 shows large fraction complexes, <br /> clusters intramodular interactions clusters                   essential proteins tend higher average intracomplex <br /> intermodular. note clusters uncovered purely                   degree non-essential proteins. particular, Direct <br /> topological manner correspond protein                         network, 76% complexes, essential proteins complexes functional modules. Direct, Pull                   higher      average     intracomplex       degree     (empirical    p- networks, essentiality correlated intramodular                value~7|10{4 ). Pull network, interaction degree, defined network clustering,             fraction complexes higher average degree essential intermodular total degree (Figure S7).                                proteins lower Direct network (58.8% 61.5%, <br />     happens consider intraprocess interactions                  respectively); expected networks include <br /> excluding intracomplex?  biological                 indirect intracomplex interactions. fact, Pull processes consist single protein complex                    networks, seven &#226;&#8364;&#732;&#226;&#8364;&#732;clique&#226;&#8364;&#8482;&#226;&#8364;&#8482; complexes protein complexes; cases observed intrapro-                protein intracomplex interactions proteins <br /> cess interactions specifically intracomplex interactions               complex, complexes complexes process. focus                     Direct network. clique complexes, percent interactions intracomplex, filtered biological                  complexes higher average intracomplex degree essential <br /> processes remove interactions  Materials Methods               proteins goes 68.2% 71.1% Pull details). proteins annotated               networks, respectively. <br /> filtered biological process, 16.52%  424=2567) proteins                     considering complex individually, analysis better <br /> essential, slightly considering                handles proteins involved multiple complexes. proteins genome. subnetwork set filtered                 removed highly overlapping complexes  Materials biological processes interaction networks,             Methods), 14%  223=1593) proteins belong weaker correlation interaction degree essentiality             complexes.  proteins tend essential; compared correlation complexes, intraprocess              proteins complex, 53.81%  120=223) degree correlated essentiality annotated              essential  opposed 37.54% proteins complexes). <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           6                        February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                             Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  Table 1. essential protein complex, essential                         networks (Figures S8   S9  ). correlation <br />  proteins tend higher average intracomplex degree.                       complex size fraction essential proteins                                                                                    complex [19], complex size correlated                                                                                    intracomplex degree member proteins, possible              Num                 Num Complexes                                     observed relationship intracomplex degree essential- <br />              Complexes           Higher                    Empirical          ity correlation complex size              Tested              Avg Essential Degree           p-value            essentiality. address  normalized interaction <br />  Direct      38                  29 (76.32%)                    7e-4               degree complex size;  normalized intracomplex <br />                                                                                    degree protein computed number intracomplex <br />  Pull   51                  30 (58.82%)                    3e-3 <br />                                                                                    interactions divided complex size.         52                  32 (61.54%)                    1e-3 <br />                                                                                    normalized degree essential proteins tends significantly <br />  Num Complexes Tested gives number complexes considered             greater non-essential proteins (Figures 4 (b), S8 (b)  networks; complex required        S9 (b)). <br />  essential proteins non-essential proteins,  intracomplex interactions. Num Complexes Higher Avg Essential <br />  Degree gives number complexes tested complexes             Essential complexes processes tend higher <br />  essential proteins higher intracomplex degree average        cross-talk degree module-level network <br />  non-essential proteins. determine number significant,          just shown, essential proteins tend  randomly permuted essentiality assignments computed number  complexes higher average intracomplex degrees essential proteins. <br />                                                                                    intramodular interactions, complexes essential <br />  Empirical p-value gives fraction random permutations             proteins, essential proteins tend intracomplex <br />  number complexes greater equal actual number,           interactions non-essential proteins. contrast,  computed 10,000 permutations.                                                intermodular interaction degree protein weaker <br />  doi:10.1371/journal.pcbi.1002910.t001 <br />                                                                                    relationship essentiality.  noted earlier, <br />                                                                                    significant number intermodular physical interac-  possible reason essential proteins tend               tions  Table S2), presumably physical interactions higher intracomplex degree (Figure 2) essentiality               connecting different functional modules network enriched proteins belonging multiple complexes,                   important module accomplish task. <br /> intracomplex degree essential protein computed                      hypothesized essentiality protein complex complexes belongs   looking complex                  functional module related topological prominence time alleviate problem.                                           module-level network. test  built &#226;&#8364;&#732;&#226;&#8364;&#732;module <br />    way addressing possible bias proteins                  network&#226;&#8364;&#8482;&#226;&#8364;&#8482; nodes modules edges participating multiple complexes, protein,                          modules enriched number intermodular cross-talk <br /> computed intracomplex degree using interactions                    interactions. particular, constructed module network largest complex belongs.  compared                     protein complexes filtered biological processes proteins complexes, significant                   physical interaction network computing number physical <br /> difference degree distribution essential non-                       interactions modules comparing essential proteins (Figure 4  ), true          average number randomized networks  Materials  <br />  <br />  <br />  <br /> Figure 4. essential complexes, essential proteins tend higher intracomplex degree Direct network non- <br /> essential proteins.   intracomplex degree (b) normalized intracomplex degree essential proteins significantly larger non-essential proteins. protein complexes essential proteins non-essential proteins, intracomplex interactions, considered. Outliers 2&#226;&#8364;&#8220;98th percentiles shown. significance difference degree distributions determined Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g004 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                              7                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                   Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  Table 2. Module-level networks protein complexes.                                 Table 3. Module-level networks filtered biological <br />                                                                                        processes. <br />  <br />                                                          Fraction  Network       Num Cross-talks       Num Modules         Essential Modules                           Num                  Num                Fraction Essential <br />                                                                                        Network       Cross-talks          Modules            Modules <br />  Direct        194                   143                 0.68 <br />  Pull     535                   242                 0.60                          Direct        1149                 307                0.79 <br />  <br />           727                   279                 0.56                          Pull     1409                 321                0.77 <br />                                                                                                 2306                 371                0.74 <br />  module-level network built protein complexes using  networks. Num Cross-talks gives number inferred cross-talks. Num               module-level network built filtered biological processes using  Modules gives number modules inferred cross-talk.            networks. Num Cross-talks gives number inferred cross-talks. <br />  Fraction Essential Modules gives fraction modules having           Num Modules gives number modules inferred cross- <br />  essential protein, modules cross-talk.                  talk. Fraction Essential Modules gives fraction modules having  doi:10.1371/journal.pcbi.1002910.t002                                                 essential protein, modules cross-talk. <br />                                                                                        doi:10.1371/journal.pcbi.1002910.t003 <br /> Methods). network, number cross-talks <br /> uncovered using modules derived protein complexes                         complexes 0.23&#226;&#8364;&#8220;0.28 considering filtered biological filtered biological processes Tables 2 3, respectively.                  processes.  significantly positive correlation <br /> note number cross-talks processes higher                      normalized cross-talk degree module, defined complexes relatively higher number                         cross-talk degree divided module size, module <br /> interactions processes intermodular intramod-                     essentiality (Table S3). compared cross-talk degree <br /> ular (86.98% vs. 64.34%, Table S2).                                                   distribution essential non-essential modules using    modules, defined complexes filtered biological                    Wilcoxon rank sum test. networks, considering <br /> processes, decrease threshold number cross-talks                 protein complexes biological processes, essential modules <br /> required module considered hub module,                   significantly higher cross-talk degree non-essential fraction modules contain essential protein tends                    modules (Figure 6). Finally, modules larger number <br /> decline (Figure 5).  significant positive correlation              proteins greater chance containing essential protein, module contains essential protein                        considered fraction proteins module cross-talk degree, SRCCs networks w0:33                        essential. cross-talk degree module considering complexes w0:34 considering filtered                             positively correlated fraction proteins module <br /> biological processes. modules proteins                       essential (Table S4), values high larger cross-talk degree, computed partial                      binary essentiality (SRCCs networks w0:27 correlation cross-talk degree module essentiality                    complexes w0:17 filtered biological processes). <br /> controlling number proteins module (Table S3);                    observed cross-talks occur functional <br /> varies networks 0.22&#226;&#8364;&#8220;0.29 considering                          modules functionally related  e.,  <br />  <br />  <br />  <br /> Figure 5. Cross-talk degree module-level network correlated module essentiality.   protein complexes (b) <br /> filtered biological processes, fraction modules containing essential protein &#226;&#8364;&#732;&#226;&#8364;&#732;hub modules&#226;&#8364;&#8482;&#226;&#8364;&#8482; tends decrease network modules considered hubs. data shown, modules added groups 20 non-increasing order cross-talk degree Direct (blue), Pull (cyan) (purple) networks. Correlations computed using SRCC shown network binary essentiality module inferred cross-talk degree. binary essentiality module 1 module essential <br /> protein, 0  <br /> doi:10.1371/journal.pcbi.1002910.g005 <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                                 8                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                     Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 6. cross-talk degree distribution non-essential modules compared essential modules   Direct, (b) <br /> Pull (c) networks protein complexes (d) Direct, (e) Pull (f) networks filtered biological <br /> processes. case, modules essential proteins significantly higher cross-talk degree modules essential proteins, determined Wilcoxon rank sum test. <br /> doi:10.1371/journal.pcbi.1002910.g006 <br />  <br />  general, shared biological process). types cross-            Analysis high-throughput networks <br /> talks interpreted intraprocess interactions broader            low-throughput studies biased studying <br /> level functional similarity. example, Ndc80p                 essential proteins, essential proteins appear complex high cross-talk degree networks studied.           interactions existing network databases.  high  Direct network, uncover seven cross-talks (Figure 7).               studies utilize select set &#226;&#8364;&#732;&#226;&#8364;&#732;bait&#226;&#8364;&#8482;&#226;&#8364;&#8482; proteins <br /> Ndc80p component kinetochore, central              bias degree distribution interaction networks. chromosome segregation couples chromosomes microtu-                 address potential concerns, performed additional <br /> bule polymers. uncovered cross-talks               network analyses. <br /> DASH MIND complexes, kinetochore                    repeated analysis Y2H-union network [21] <br /> associated; cross-talks interpreted                recently built BinaryHQHT network [33], &#226;&#8364;&#732;&#226;&#8364;&#732;intramodular&#226;&#8364;&#8482;&#226;&#8364;&#8482; interactions higher level organization.           derived high-throughput yeast hybrid data. hand, Ndc80 cross-talks                   networks, interactions experiments involving bait <br /> complexes range distinct biological                 proteins removed, high-quality yeast hybrid <br /> processes, including nucleosome remodeling complex SWI/                interactions  experiments) retained. SNF, dynactin microtubule associated complex, MRX                  networks notably smaller size focusing annotated <br /> complex involved DNA damage repair, nuclear condensin               interactions (Tables S9, S10 S11);  outline <br /> complex nuclear cohesion complex.                                   repeating analysis yields similar results Direct <br />    essential functional modules cross-talks            network. functional modules representing truly different biological               Y2H-union BinaryHQHT networks, intraprocess <br /> processes, considered set expert-selected BP terms           interaction degree protein weak statistically <br /> yeast [32], focused annotate 500                 significant correlation essentiality overall degree <br /> proteins. considered functional module annotated           protein correlated essentiality Y2H-union terms &#194;&#167;70% proteins annotated           network weakly correlated essentiality ignored cross-talks functional modules              BinaryHQHT network (Figures S10 S11,   (b)).  annotated shared term; case,          protein essentiality reflected intraprocess degree essential functional modules correlated          networks, overall degree.  intraprocess degree cross-talk degree (Table S5).  range analyses reveals       significantly higher essential proteins non- relationship topological importance            essential proteins (Figure S10 (d) S11 (d)), true <br /> functional module tendency contain essential proteins.          overall degree interprocess degree (Figures S10  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                      9                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  <br />  <br /> Figure 7. Cross-talk network centered Ndc80 complex. Cross-talk analysis Direct network uncovers seven cross-talks Ndc80 complex annotated complexes. Ndc80 complex kinetochore, cross-talks subcomplexes kinetochore (shown dark green box), remaining cross-talks non-kinetochore complexes. individual proteins colored red essential, blue  <br /> doi:10.1371/journal.pcbi.1002910.g007 <br />  <br /> S11, (c) (e)). 84.62% 78.57% essential protein                    overall interaction degree essentiality (Figure S16   <br /> complexes Y2H-union BinaryHQHT networks, essential                    (b)).  relationship overall interaction <br /> proteins tend higher average intracomplex degree                  degree essentiality longer significant controlling non-essential proteins (Table S12), essential proteins                  intraprocess degree (Figure S16 (b)).  intraprocess <br /> higher intracomplex degree normalized intracomplex degree                    degree bait proteins higher essential proteins non-essential proteins (Figures S12 S13).                   non-essential proteins (Figure S16), complexes module-level analysis, cross-talk degree                   essential bait non-essential bait proteins, essential proteins <br /> complex-level network significantly correlated complex                   higher intracomplex degree (Figure S17). <br /> essentiality BinaryHQHT network (Figure S14 ),                     Overall, relationships intraprocess degree cross-talk degree process-level network significantly                    protein essentiality, degree complexes essentiality, correlated process essentiality networks (Figures                   module essentiality cross-talk degree largely recapitulated <br /> S14 (b) S15 (c) (d)). Y2H-union network,               smaller networks, Y2H-union BinaryHQHT networks, <br /> complexes cross-talks, relationship                    low-throughput experiments experiments biased cross-talk degree complex essentiality weak (Figures              essential proteins specifically excluded.  <br /> S14   S15  ); small number                     comparisons bait proteins, enriched intercomplex interactions network (Table S10).                           interactions, confirms relationship number    second analysis, removed interactions uncovered                  (intraprocess) interactions protein essential. <br /> low-throughput experiments  50 interactions determined) Pull network, restricted analysis              Discussion interaction properties proteins labelled bait proteins. Bait <br /> proteins higher fraction essential proteins network              long line previous research studied relationship <br /> (Table S14). considering just bait proteins, stronger               network topology protein essentiality. Recent work <br /> relationships intraprocess degree essentiality                  argued hubs densely connected essential <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           10                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> complexes processes [18], essential complexes tend                Materials Methods large [19].  argued essentiality modular property, essential proteins essential modules               Physical interaction datasets <br /> tend interactions modules tend large.                   performed analysis physical interaction datasets. initial analysis, revealing protein&#226;&#8364;&#8482;s intramodular                   network, physical interactions gathered interaction degree predictive essentiality overall          BioGRID [26], release 3.1.78, using evidence codes indicative <br /> degree, largely supports argument.                physical interactions &#226;&#8364;&#732;&#226;&#8364;&#732;Affinity Capture-RNA&#226;&#8364;&#8482;&#226;&#8364;&#8482; focus proteins belong complexes, intraprocess             &#226;&#8364;&#732;&#226;&#8364;&#732;Protein-RNA.&#226;&#8364;&#8482;&#226;&#8364;&#8482; early yeast hybrid paper Ito et <br /> interaction degree does correlate essentiality better              al. [42], included core data. remove artifacts overall interaction degree; suggests observed                &#226;&#8364;&#732;&#226;&#8364;&#732;sticky proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482; certain experiments, protein network modularity essential proteins largely                      30 interactions single experimental data source, complexes, general feature biological                      removed interactions. second network, extracted <br /> processes.                                                                      direct physical interactions initial network utilizing <br />    observed positive correlation protein essentiality               interactions determined following intramodular degree attributed module-                    experimental systems: Biochemical activity, crystal structure, <br /> level complex essentiality. particular, essential protein             Far western, FRET, Protein-<span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>, Reconstituted complex, complexes, essential proteins tend                  hybrid. network, extracted initial <br /> higher intracomplex degrees non-essential counterparts.              network interactions determined Affinity  essential complexes, topological prominence              capture-Western Affinity capture-MS. refer constitutent proteins related essentiality;        networks  Direct Pull  respectively, sizes importance proteins maintaining structural                  given Table S1. <br /> integrity complexes. view consistent                      considered additional networks, comprised relative enrichment essentiality proteins                  interactions determined small-scale experimental <br /> structural interfaces opposed just structural                  assays; manner, attempt minimize effect study <br /> interfaces [34].                                                                bias. networks, refer Y2H-union, <br />    intermodular interactions                      built earlier study [21]; included interactions <br /> important intramodular interactions explaining protein                  determined large-scale high-quality yeast hybrid studies, <br /> essentiality, observed significant number intermodular             excluded experiment using specific set &#226;&#8364;&#732;&#226;&#8364;&#732;bait&#226;&#8364;&#8482;&#226;&#8364;&#8482; proteins <br /> interactions physical interaction networks. considered              enriched essential proteins [43]. used interactions modular level, demonstrated essential                recently built high-throughput yeast hybrid network functional modules tend cross-talks                     [33], refer BinaryHQHT. Finally, built high- <br /> functional modules.  analysis showed                  throughput network Pull network keeping correlation network topology essentiality               interactions experiments uncovering protein level modular level.  observed             50 interactions 10 &#226;&#8364;&#732;&#226;&#8364;&#732;bait&#226;&#8364;&#8482;&#226;&#8364;&#8482; functionally related modules likely interconnect            proteins. refer second network Pull-downf network,  revealing hierarchical structure physical                 use compare network properties bait proteins interaction networks.                                                           respect  <br />    Overall, work advanced understanding relationship essentiality network topology.                 Protein complexes biological processes <br /> shown importance intramodular interactions, especially                      used set 430 protein complexes compiled [27], <br /> intracomplex interactions, demonstrated essential mod-                 includes SGD Macromolecular Complex standard <br /> ules tend higher cross-talk degree non-essential                 [44], CYC2008 protein complex catalog [45] set modules. findings likely yield improvements                 manually curated complexes. initial set, removed <br /> ability predict protein essentiality.  integrative machine            highly overlapping complexes follows.  proteins <br /> learning approaches use range network sequence                    comprising complex subset proteins comprising <br /> features previously applied predict protein essenti-               complex, smaller complex removed.  ality (e.g., [35&#226;&#8364;&#8220;41]); based work, information                 complexes, Jaccard index proteins making functional modules protein complexes, especially respect                e., number overlapping proteins divided size intramodular cross-talk degree, incorporated              union protein sets) &#194;&#167;0:5, removed smaller frameworks.                                                        complexes. Additionally, previous work [19], removed    future, interesting characterize network           complexes corresponding subunits ribosome, properties essential proteins central protein                contain large number proteins;  physical interaction networks. Based current findings,                complexes disproportionately affect protein analysis. speculate proteins important                 filters, left 390 complexes.  functioning specific essential modules, reflected            Supplement Figure S6, Text S1 intraprocess interprocess interactions proteins modules,             results including ribosomal complexes). relationships better represented types                   functional analysis, worked subset specific <br /> interactions (e.g., regulatory, metabolic genetic). frame-               Gene Ontology   Biological Process (BP) terms [25] work incorporating functional information network                      derived entire (version 1.1.2130) follows.  analysis likely useful establishing             extracted 1418 BP terms, annotates 5 yeast case. Finally, performed analysis S.                proteins 50.  hone contribution cerevisiae, approach applied study essential proteins          specific biological process  opposed effects arising annotated organisms large-scale interaction                     proteins annotated process networks genome-scale gene deletion disruption data.                     protein complexes), pruned set proteins  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          11                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                          Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> associated functional terms. specifically, size          tSS b)~0.  terms  intersection biological process                 tSS b)~1. <br /> original set 430 protein complexes &#194;&#167;2, proteins                   measure naturally extended functional relationships <br /> intersection longer associated process. left           proteins multiple annotations. protein p, <br /> fewer 2 proteins associated process,            let p) set terms p annotated. term <br /> half number proteins known annotate,             annotates p, parent terms naturally included term removed consideration. Finally, highly overlapping                p).  proteins p q, protein semantic <br /> processes removed manner described                    similarity (pSS) defined follows [7]: complexes. procedure resulted 391 &#226;&#8364;&#732;&#226;&#8364;&#732;filtered&#226;&#8364;&#8482;&#226;&#8364;&#8482; processes, 2567 proteins associated processes.                                        2: maxa p) q) s  <br />                                                                                            pSS(p,q)~                                   : <br />                                                                                                         maxa p) s maxa q) s  <br /> Detecting cross-talk complexes processes <br />    given network, exhaustively determined pairs functional modules enriched number interactions [10]. considered modules arising                     Supporting Information <br /> complexes processes turn  e., functional modules consist                                                                                 Figure S1 intraprocess interaction degree proteins complex, shared <br />                                                                                 correlated protein essentiality overall <br /> process annotation 391 filtered processes considered).                                                                                 interaction degree proteins Pull net- <br /> considered proteins network associated                                                                                 work, interactions categorized specific BP modules considering, edges <br />                                                                                 terms, annotates 50 proteins.   correspond intermodular interactions                                                                                 fraction essential proteins hub proteins proteins.  modules c1 c2 counted                                                                                 proteins considered hub proteins; proteins added groups <br /> number &#226;&#8364;&#732;&#226;&#8364;&#732;cross-talk&#226;&#8364;&#8482;&#226;&#8364;&#8482; interactions proteins compris-             50 non-increasing order interaction degree. ing modules. Note interactions           fraction highest intraprocess degree (green), followed proteins annotated c1 c2 included            annotated degree (blue) interprocess degree (red). (b) intramodular interactions. determine number               correlations measured SRCCs essentiality observed cross-talk interactions pair            intraprocess, annotated interprocess degree. expected chance, randomized intermodular interactions                 SRCC highest essentiality intraprocess degree. network 100 times using stub-rewiring  [8]),           partial correlation computed annotated degree <br /> preserving degree distribution, module annotation, overall              essentiality controlling intraprocess number cross-talk interactions.  countfc1 ,c2 g number        interprocess degree. Starred p-values indicate values cross-talk interactions c1 c2 real network,           w0:05. (c)&#226;&#8364;&#8220;(e) degree distribution non-essential proteins avgfc1 ,c2 g average number corresponding cross-talk                  compared essential proteins (c) annotated, (d) <br /> interactions randomized networks, odds-score module               intraprocess, (e) interprocess degree, respectively. box <br /> pair defined                                                              plot, horizontal bar box corresponds median                                                                                 distribution; ends box indicate                           (countfc1 ,c2 g z1)                                   quartiles; small circles outliers 2&#226;&#8364;&#8220;98th <br />                                               :                                 percentile range. significance difference                            (avgfc1 ,c2 g z1)                                    degree distributions measured Wilcoxon rank sum <br />                                                                                 test. <br />                                                                                 (TIFF) <br />   addition pseudocount 1 downweighs contribution rare cross-talks obtain               Figure S2 intraprocess interaction degree high scores simply small  zero) average counts            correlated protein essentiality overall <br /> randomized graphs. order module pair considered                interaction degree proteins network, cross-talk, required                       interactions categorized specific BP terms, <br /> independent  e., non-overlapping) cross-talk interactions,                annotates 50 proteins.   odds-score 2. observed relationship             fraction essential proteins hub proteins proteins module essentiality cross-talk degree persists                considered hub proteins; proteins added groups 50 range odds-scores  Tables S6&#226;&#8364;&#8220;S8).                                        non-increasing order interaction degree. fraction                                                                                 highest intraprocess degree (green), followed annotated <br />                                                                                 degree (blue) interprocess degree (red). (b) Semantic similarity                                                             correlations measured SRCCs essentiality    semantic similarity terms                 intraprocess, annotated interprocess degree. SRCC ontology estimate functional similarity                highest essentiality intraprocess degree. partial <br /> terms. use semantic similarity measure introduced [30].               correlation computed annotated degree particular, let f   fraction proteins yeast annotated         essentiality controlling intraprocess interprocess term total number proteins.                            degree. Starred p-values indicate values w0:05. (c)&#226;&#8364;&#8220;(e) <br /> s ~{log(f  ) measure specific term                    degree distribution non-essential proteins compared compute term semantic similarity b, tSS b)                    essential proteins (c) annotated, (d) intraprocess,             2:s(LCA b))                                                       (e) interprocess degree, respectively. box plot, tSS b)~                     , LCA b) common <br />                s zs(b)                                                        horizontal bar box corresponds median ancestor b ontology. Note LCA                 distribution; ends box indicate terms root term (e.g., 0008150 &#226;&#8364;&#732;biological process&#226;&#8364;&#8482;),               quartiles; small circles outliers 2&#226;&#8364;&#8220;98th <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                          12                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                       Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> percentile range. significance difference             essentiality intracomplex degree. partial degree distributions measured Wilcoxon rank sum                correlation computed annotated degree test.                                                                        essentiality controlling intracomplex degree, <br /> (TIFF)                                                                       intercomplex degree, size largest complex                                                                              protein belongs. Starred p-values indicate values <br /> Figure S3 networks, semantic similarity <br />                                                                              w0:05. (c)&#226;&#8364;&#8220;(f) degree distribution non-essential proteins degree correlated protein essentiality                                                                              compared essential proteins complexes (c) overall interaction degree.  &#226;&#8364;&#8220;(c) fraction                                                                              annotated degree, (d) intracomplex degree, (e) intercomplex <br /> essential proteins hub proteins proteins considered hub proteins Direct, Pull networks;         degree, (f) complex size. box plot, horizontal <br /> proteins added groups 50 non-increasing order          bar box corresponds median distribution; semantic similarity degree. network, SRCC                    ends box indicate quartiles; computed protein essentiality semantic                    small circles outliers 2&#226;&#8364;&#8220;98th percentile range. similarity annotated degree; values boxed           significance difference degree distributions <br /> panel. (d)&#226;&#8364;&#8220;(f) semantic similarity weighted degree distribution          measured Wilcoxon rank sum test. non-essential proteins compared essential proteins          (TIFF) Direct, Pull networks.                                 Figure S6 correlations interaction degree <br /> (TIFF)                                                                       essentiality proteins complexes, including <br /> Figure S4 intracomplex interaction degree                        ribosomal complexes,   Direct, (b) Pull correlated protein essentiality overall                        (c) networks. Interactions categorized using protein <br /> interaction degree proteins Pull net-                        complexes including ribosomal complexes. fraction work, interactions categorized using protein                        essential proteins hub proteins proteins complexes.   fraction essential proteins hub                  considered hub proteins; proteins added groups 50 proteins proteins considered hub proteins; proteins          non-increasing order interaction degree size added groups 50 non-increasing order interaction           largest complex protein belongs. relationship <br /> degree size largest complex protein               protein essentiality interaction degree shown belongs. relationship protein essentiality                   intracomplex interactions (green), annotated interactions (blue) <br /> interaction degree shown intracomplex interactions (green),           intercomplex interactions (red). relationship annotated interactions (blue) intercomplex interactions              protein essentiality complex size shown (black). <br /> (red). relationship protein essentiality complex             (TIFF) <br /> size shown (black). (b) correlations measured                 Figure S7 Essentiality correlated intra- <br /> SRCCs essentiality intracomplex degree,               modular interaction degree intermodular <br /> annotated degree, intercomplex degree, complex size.                  total degree, modules determined SRCC highest essentiality intracomplex degree.                network (  Direct, (b) Pull (c)  partial correlation computed annotated               network clustering approaches. obtain clusters, used <br /> degree essentiality controlling intracomplex             SPICi clustering algorithm, local clustering approach, degree, intercomplex degree, size largest complex           density threshold 0.5 minimum increment ratio 0.3. protein belongs. Starred p-values indicate              fraction essential proteins hub proteins values w0:05. (c)&#226;&#8364;&#8220;(f) degree distribution non-essential               proteins considered hub proteins; proteins added groups <br /> proteins compared essential proteins complexes          50 non-increasing order interaction degree. (c) annotated degree, (d) intracomplex degree, (e)                   relationship protein essentiality interaction degree intercomplex degree, (f) complex size. box plot,             shown intramodular interactions (green), total interactions <br /> horizontal bar box corresponds median                 (blue) intermodular interactions (red). <br /> distribution; ends box indicate           (TIFF) <br /> quartiles; small circles outliers 2&#226;&#8364;&#8220;98th <br /> percentile range. significance difference             Figure S8 essential complexes, essential proteins degree distributions measured Wilcoxon rank sum                tend higher intracomplex degree Pull- <br /> test.                                                                        network non-essential proteins.   intra- <br /> (TIFF)                                                                       complex degree (b) normalized intracomplex degree                                                                              essential proteins significantly larger non-essential <br /> Figure S5 intracomplex interaction degree                        proteins. protein complexes essential <br /> correlated protein essentiality overall                        proteins non-essential proteins, interaction degree proteins network,                    intracomplex interactions considered. Outliers 2&#226;&#8364;&#8220; <br /> interactions categorized using protein complexes.                      98th percentiles shown. significance difference fraction essential proteins hub proteins                degree distributions determined Wilcoxon rank sum test. <br /> proteins considered hub proteins; proteins added groups <br />                                                                              (TIFF) 50 non-increasing order interaction degree size largest complex protein belongs.                     Figure S9 essential complexes, essential proteins <br /> relationship protein essentiality interaction degree             tend higher intracomplex degree shown intracomplex interactions (green), annotated                network non-essential proteins.   intracomplex <br /> interactions (blue) intercomplex interactions (red).                 degree (b) normalized intracomplex degree essential <br /> relationship protein essentiality complex size           proteins significantly larger non-essential proteins. <br /> shown (black). (b) correlations measured SRCCs                protein complexes essential proteins <br /> essentiality intracomplex degree, annotated degree,           non-essential proteins, intracomplex <br /> intercomplex degree, complex size. SRCC highest                    interactions considered. Outliers 2&#226;&#8364;&#8220;98th percentiles <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                       13                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                           Essentiality Hub Proteins Hub Modules <br />  <br />  shown. significance difference degree                  interprocess degree.  essential proteins higher <br /> distributions determined Wilcoxon rank sum test.                       average intraprocess degree non-essential proteins (TIFF)                                                                           network (panel (d)), significant difference                                                                                  interprocess degree (panel (e)). <br /> Figure S10 intraprocess interaction degree                                                                                  (TIFF) <br /> correlated protein essentiality overall <br /> interaction degree proteins Y2H-union network,                        Figure S12 Essential proteins tend higher interactions categorized specific BP                            intracomplex degree non-essential proteins terms, annotates 50 proteins.                        protein complexes Y2H-union network.   fraction essential proteins hub proteins proteins           intracomplex degree (b) normalized intracomplex degree considered hub proteins; proteins added groups size 20              essential proteins significantly greater non-essential <br /> larger proteins degree group).            proteins. protein complexes essential fraction highest intraprocess degree (green), followed        proteins non-essential proteins intracomplex <br /> annotated degree (blue) interprocess degree (red). (b)           interactions tested. Outliers 2&#226;&#8364;&#8220;98% shown. correlations measured SRCCs essentiality                   significance difference degree distributions intraprocess, annotated interprocess degree. SRCC                  determined Wilcoxon rank sum test. <br /> highest essentiality intraprocess degree. partial                (TIFF) <br /> correlation computed annotated degree                    Figure S13 Essential proteins tend higher <br /> essentiality controlling intraprocess interprocess            intracomplex degree non-essential proteins degree. Starred p-values indicate values w0:05. (c)&#226;&#8364;&#8220;(e)               protein complexes BinaryHQHT network.   degree distribution non-essential proteins compared            intracomplex degree (b) normalized intracomplex degree of essential proteins (c) annotated, (d) intraprocess, (e)           essential proteins significantly greater non-essential <br /> interprocess degree, respectively. box plot, horizontal bar          proteins. protein complexes essential box corresponds median distribution;              proteins non-essential proteins intracomplex <br /> ends box indicate quartiles; small            interactions tested. Outliers 2&#226;&#8364;&#8220;98% shown. circles outliers 2&#226;&#8364;&#8220;98th percentile range.                    significance difference degree distributions significance difference degree distributions              determined Wilcoxon rank sum test. measured Wilcoxon rank sum test. Y2H-union                     (TIFF) <br /> network, essentiality intraprocess degree small statistically significant correlation. Essentiality significantly         Figure S14 Cross-talk degree module-level network <br /> correlated overall degree interprocess degree.                  correlated module essentiality.   protein <br /> essential proteins higher average intraprocess degree non-             complexes (b) filtered biological processes, fraction essential proteins network (panel (d)),             modules containing essential protein &#226;&#8364;&#732;&#226;&#8364;&#732;hub <br /> significant difference annotated degree interprocess degree            modules&#226;&#8364;&#8482;&#226;&#8364;&#8482; tends decrease network modules (panels (c) (e)).                                                            considered hubs. data shown, modules added non- <br /> (TIFF)                                                                           increasing order cross-talk degree Y2H-union (blue)                                                                                  BinaryHQHT (cyan) networks. Correlations binary <br /> Figure S11      intraprocess interaction degree                      essentiality module inferred cross-talk degree correlated protein essentiality overall                            computed using SRCC shown network. interaction degree proteins BinaryHQHT                                binary essentiality module 1 module network, interactions categorized specific                         essential protein, 0  Y2H-union module BP terms, annotates 50                                  network comprised complexes (panel  ), correlation proteins.   fraction essential proteins hub                       significant uncover complexes crosstalks proteins proteins considered hub proteins; proteins              small network (549 edges, Table S10). <br /> added groups size 20  larger proteins             (TIFF) degree group). fraction highest intraprocess degree (green), followed annotated degree (blue)             Figure S15 cross-talk degree distribution non- interprocess degree (red). (b) correlations                      essential modules compared essential <br /> measured SRCCs essentiality intraprocess,                  modules   Y2H-union (b) BinaryHQHT annotated interprocess degree. SRCC highest                        networks protein complexes (c) Y2H-union essentiality intraprocess degree. partial correla-               (d) BinaryHQHT networks filtered biological pro- <br /> tion computed annotated degree essentiality              cesses. Y2H-union BinaryHQHT networks, modules controlling intraprocess interprocess degree.                 (derived biological processes) essential proteins Starred p-values indicate values w0:05. (c)&#226;&#8364;&#8220;(e)                   significantly higher cross-talk degree modules degree distribution non-essential proteins compared             essential proteins, determined Wilcoxon rank sum test. <br />                                                                                  modules derived complexes Y2H-union network, <br /> essential proteins (c) annotated, (d) intraprocess, (e) <br />                                                                                  differences essential non-essential modules interprocess degree, respectively. box plot, horizontal <br />                                                                                  significant, modules bar box corresponds median distribution;                                                                                  uncover cross-talks. ends box indicate quartiles;                                                                                  (TIFF) <br /> small circles outliers 2&#226;&#8364;&#8220;98th percentile range. significance difference degree distributions              Figure S16 intraprocess interaction degree measured Wilcoxon rank sum test. BinaryHQHT                    correlated protein essentiality network, essentiality intraprocess degree small                   overall interaction degree bait proteins statistically significant correlation. Essentiality smaller                Pull-downf network excluding small-scale experi- <br /> correlation overall degree correlated                       ments, interactions categorized spe- <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                           14                       February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                Essentiality Hub Proteins Hub Modules <br />  <br />  <br /> cific BP terms, annotates 50                              Table S6 significant correlation cross-talk <br /> proteins. tests bait proteins Pull-                  degree binary module essentiality persists downf network.   fraction essential proteins hub                    range odd-scores Direct network. <br /> proteins proteins considered hub proteins; proteins                    (PDF) added non-increasing order interaction degree.                     Table S7 significant correlation cross-talk fraction highest intraprocess degree (green),                          degree binary module essentiality persists followed annotated degree (blue)                                range odd-scores Pull network. <br /> interprocess degree (red). (b) correlations measured                        (PDF) <br /> SRCCs essentiality intraprocess, annotated interprocess degree. SRCC highest                              Table S8 significant correlation cross-talk essentiality intraprocess degree. partial                          degree binary module essentiality persists correlation computed annotated degree                          range odd-scores network. essentiality controlling intraprocess                       (PDF) <br /> interprocess degree. Starred p-values indicate values                   Table S9   number proteins, number w0:05. (c)&#226;&#8364;&#8220;(e) degree distribution non-essential                            interactions fraction essential proteins proteins compared essential proteins (c)                     Y2H-union BinaryHQHT physical interaction <br /> annotated, (d) intraprocess, (e) interprocess degree,                          networks. <br /> respectively. box plot, horizontal bar                        (PDF) <br /> box corresponds median distribution; ends box indicate quartiles; small                   Table S10 substantial fraction physical interactions <br /> circles outliers 2&#226;&#8364;&#8220;98th percentile range.                      Y2H-union network intermodular. <br /> significance difference degree                              (PDF) <br /> distributions measured Wilcoxon rank sum test.                           Table S11 substantial fraction physical interactions <br /> (TIFF)                                                                             BinaryHQHT network intermodular. <br /> Figure S17     essential complexes, essential bait                          (PDF) <br /> proteins tend higher intracomplex degree                              Table S12   essential protein complex, Pull-downf network non-essential bait proteins.                           essential proteins tend higher average <br />   intracomplex degree (b) normalized intracomplex                     intracomplex degree Y2H-union BinaryHQHT <br /> degree essential bait proteins significantly larger             networks. <br /> non-essential bait proteins. protein complexes                   (PDF) essential bait proteins non-essential bait <br />                                                                                    Table S13Module-level networks Y2H-union proteins, intracomplex interactions, considered. <br />                                                                                    BinaryHQHT networks. <br /> Outliers 2&#226;&#8364;&#8220;98th percentiles shown. significance <br />                                                                                    (PDF) difference degree distributions determined Wilcoxon rank sum test.                                          Table S14       Numbers bait proteins Pull-downf <br /> (TIFF)                                                                             network. <br />                                                                                    (PDF) <br /> Table S1   number proteins, number interactions fraction essential proteins                            Table S15   essential protein complex, physical interaction networks consid-                            essential bait proteins tend higher average <br /> ered.                                                                              intracomplex degree non-essential bait proteins (PDF)                                                                              Pull-downf network. <br />                                                                                    (PDF) <br /> Table S2 substantial fraction physical interactions Direct network intermodular.                                            Text S1 Analysis correlations interac- <br /> (PDF)                                                                              tion degree essentiality proteins complex- <br />                                                                                    es, including ribosomal complexes, net- <br /> Table S3 Correlation cross-talk (CT) degree                                works. binary module essentiality.                                                    (PDF) <br /> (PDF) <br /> Table S4 Correlation cross-talk (CT) degree                                Acknowledgments fraction essential proteins module.                              authors thank members Singh group helpful discussions <br /> (PDF)                                                                              comments manuscript. <br /> Table S5 Correlation cross-talk (CT) degree binary module essentiality removing function-                            Author Contributions <br /> ally similar cross-talks.                                                          Conceived designed experiments: JS MS. Performed (PDF)                                                                              experiments: JS. Analyzed data: JS MS. Wrote paper: JS MS. <br />  <br /> References <br />  1. Aittokallio T, Schwikowski B (2006) Graph-based methods analysing           3. Rives AW, Galitski T (2003) Modular organization cellular networks. Proc <br />     networks cell biology. Briefings Bioinformatics 7: 243&#226;&#8364;&#8220;255.                  Natl Acad Sci USA 100: 1128&#226;&#8364;&#8220;1133. <br />  2. Hartwell LH, Hopfield JJ, Leibler S, Murray AW (1999) molecular         4. Spirin V, Mirny LA (2003) Protein complexes functional modules     modular cell biology. Nature 402: C47&#226;&#8364;&#8220;C52.                                         molecular networks. Proc Natl Acad Sci USA 100: 12123. <br />  <br />  <br />  <br />  <br /> PLOS Computational Biology | www.ploscompbiol.org                             15                          February 2013 | Volume 9 | Issue 2 | e1002910 <br />                                                                                                                              Essentiality Hub Proteins Hub Modules <br />  <br />  <br />  5. Bader GD, Hogue CW (2003) automated method finding molecular                       26. Stark C, Breitkreutz BJ, Reguly T, Boucher L, Breitkreutz  et al. (2006) <br />     complexes large protein interaction networks. BMC Bioinformatics 4: 2.                     BioGRID: general repository interaction datasets. Nucleic Acids Research <br />  6. Pereira-Leal J, Enright AJ, Ouzounis CA (2004) Detection functional modules                34(Database Issue): D535&#226;&#8364;&#8220;D539. <br />     protein interaction networks. Proteins 54: 49&#226;&#8364;&#8220;57.                                    27. Baryshnikova  Costanzo M, Kim Y, Ding H, Koh J, et al. (2010) Quantitative <br />  7. Song J, Singh M (2009) interactome-derived clusters used               analysis fitness genetic interactions yeast genome scale. Nature <br />     predict functional modules protein function? Bioinformatics 25: 3143&#226;&#8364;&#8220;3150.             Methods 7: 1017&#226;&#8364;&#8220;1025. <br />  8. Milo R, Shen-Orr S, Itzkovitz S, Kashtan N, Chklovskii D, et al. (2002) Network           28. Schwikowski B, Uetz P, Fields S (2000) network protein-protein interactions <br />     motifs: simple building blocks complex networks. Science 298: 824&#226;&#8364;&#8220;827.                     yeast. Nature Biotechnology 18: 1257&#226;&#8364;&#8220;1261. <br />  9. Luscombe NM, Babu MM, Yu H, Snyder M, Teichmann SA, et al. (2004)                         29. Spearman C (1904) proof measurement association     Genomic analysis regulato <br /> </body></html>