<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 17 January 2014                                               Nucleic Acids Research, 2014, Vol. 42,  6 3623&#226;&#8364;&#8220;3637 <br />                                                                                                                 doi:10.1093/nar/gkt1386 <br />  <br />  <br /> Comparative ribosome profiling reveals extensive <br /> translational complexity different Trypanosoma <br /> brucei life cycle stages <br /> Juan-Jose&#194;&#180; Vasquez1, Chung-Chau Hon2,3, Jens T. Vanselow4, Andreas Schlosser4 T. Nicolai Siegel1,* <br /> 1 <br />  Research Center Infectious Diseases, University Wuerzburg, Wuerzburg 97080, Germany, <br /> 2 <br />  partement Biologie cellulaire et infection, Institut Pasteur, Unite&#194;&#180; Biologie Cellulaire du Parasitisme, <br /> Paris 75015, France, 3INSERM U786, Paris 75015, France 4Rudolf Virchow Center, University Wuerzburg, Wuerzburg 97080, Germany <br />  <br /> Received November 7, 2013; Revised December 17, 2013; Accepted December 18, 2013 <br />  <br />  <br />  <br /> ABSTRACT                                                                        INTRODUCTION gene expression fundamental tightly                              Eukaryotic gene expression regulated multiple levels, <br /> controlled cellular process regulated                                including extensive elaborate post-transcriptional <br /> multiple steps, exact contribution step                             regulation affects RNA stability, protein translation <br /> remains unknown organism. absence                                 protein turnover (1). individual <br /> transcription initiation regulation RNA polymer-                            levels contribute &#239;&#172;?nal outcome, steady state <br />                                                                                 level protein, known organism. ase II protozoan parasite Trypanosoma brucei <br />                                                                                 need regulation level transcription greatly simplifies task elucidating contri- <br />                                                                                 RNA stability, example level translation bution translation global gene expression.                                protein stability, evident transcriptome  sequenced ribosome-protected                                 proteome studies revealed weak <br /> mRNA fragments T. brucei, permitting                                     correlations mRNA protein levels genome-wide analysis RNA translation trans-                              various organisms, including yeast, humans proto- <br /> lational efficiency. varies                             zoan parasite Leishmania major (2&#226;&#8364;&#8220;6). Furthermore, <br /> greatly life cycle stages parasite                               mRNA translation recently  100-fold genes, contributing                               extensively regulated step mammals (7). <br /> gene expression similar extent RNA stability.                              L. major related kinetoplastids Trypanosoma ability map ribosome positions sub-codon                              cruzi Trypanosoma brucei causative agents resolution revealed extensive translation                                  leishmaniasis, Chagas disease sleeping sickness, upstream open reading frames located 50                                  signi&#239;&#172; human diseases leading death UTRs enabled identification hundreds                              tens thousands people worldwide year (8,9). <br />                                                                                 Interestingly, genome organization kinetoplastids <br /> previously annotated putative coding sequences <br />                                                                                 differs eukaryotes (CDSs). Evaluation existing proteomics                                                                                 protein-coding genes transcribed long <br /> genome-wide RNAi data confirmed translation                                 polycistronic transcription units (PTUs). PTUs previously annotated CDSs suggested                                   encompass &gt;100 functionally unrelated genes important role &gt;200 CDSs                                     (10&#226;&#8364;&#8220;13). Maturation polycistronic RNA precursors <br /> parasite survival, especially form                               functional mRNA transcripts involves trans-splicing <br /> infective mammals. Overall data                                39 nt leader sequence 50 end polyadenyla- <br /> translational control plays prevalent important                           tion 30 end (14&#226;&#8364;&#8220;16). absence promoter <br /> role different parasite life cycle stages                                 sequence motifs, organization genes PTUs T. brucei.                                                                      apparently conserved open chromatin structure <br />  <br />  <br />  <br />  <br />  correspondence addressed. Tel: +49 931 31 88 499; Fax: +49 931 318 25 78; Email: nicolai.siegel@uni-wuerzburg  <br /> &#195;&#376; Author(s) 2014. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution License (http://creativecommons.org/licenses 3.0/), permits unrestricted reuse, distribution, reproduction medium, provided original work properly cited. <br />  3624 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  <br /> surrounding RNA polymerase II (pol II) transcription            approach successfully used demon- <br /> start sites (12,17) strongly indicate lack regulation      strate differential translation RNA T. brucei (43,44), gene expression level pol II transcription         adapted higher resolution approach termed <br /> initiation. lack transcriptional regulation      ribosome pro&#239;&#172;?ling recently developed (45). greatly facilitate effort quantify extent         approach based high-throughput sequencing different post-transcriptional mechanisms contrib-        ribosome-protected RNA fragments, called ribosome <br /> ute gene expression.                                         &#226;&#8364;&#732;footprints&#226;&#8364;&#8482;. RNA protected ribosomes    Despite lack transcriptional control, L. major,       removed nuclease digestion; ribosome footprints T. cruzi T. brucei capable tightly regulating        converted libraries DNA molecules suitable gene expression complex life cycles,           high-throughput sequencing, abundance indi- requires adaptation parasites           vidual footprints determined deep sequencing. large environmental differences insect        yeast, mice Escherichia coli, measurements vectors mammalian hosts (18,19). T. brucei,        average ribosome occupancy (ribosome density pro&#239;&#172;?les) <br /> differences include,  change tempera-         successfully used estimate rates trans- <br /> ture 37  C mammalian host 27  C           lation (rates protein synthesis) (45&#226;&#8364;&#8220;47). Ribosome <br /> insect vector change availability glucose,      density, perfect, shown preferred energy source parasite (20,21).            better predictor protein levels measurements Consequently, signi&#239;&#172; effort invested            mRNA levels (45). <br /> elucidating post-transcriptional mechanisms gene            Importantly, approach provides quantita- <br /> control parasites. Work laboratories      tive information number RNA molecules led identi&#239;&#172;?cation sequence motifs capable      associated ribosomes, yields position-speci&#239;&#172;?c <br /> modulating RNA stability translational ef&#239;&#172;?ciency        information regarding location ribosomes life cycle-speci&#239;&#172;?c manner (22&#226;&#8364;&#8220;27), bias codon usage        mRNA transcripts. information critical, associ- <br /> suggested affect translational ef&#239;&#172;?ciency (28),         ation RNA transcript ribosome does genome-wide analysis mRNA stability half-          necessarily mean transcript translated. <br /> lives transcripts range orders mag-        ribosome associated 50 UTR nitude T. brucei (29).                                       stalled, failing produce functional protein.    comparable study evaluate degree regulation      addition, ribosome positions used accurately map level protein translation performed      correct CDSs help identify novel CDSs multiple observations suggest translational         missed previous genome annotations (48). <br /> control important regulator gene expression.              report adaptation ribosome pro&#239;&#172;?ling example, kinetoplastid genomes encode                  approach T. brucei. Comparative ribosome pro&#239;&#172;?ling <br /> proteins involved translation initiation control        analyses bloodstream procyclic parasite forms yeast metazoans, including              allowed generate genome-wide picture transla- homologues poly  binding proteins        tion. observed 100-fold range translational isoforms translation initiation factor (eIF4E)      ef&#239;&#172;?ciency genes life cycle-speci&#239;&#172;?c differences &#239;&#172;?ve eIF4G isoforms (30&#226;&#8364;&#8220;34).  RNA-                  translational ef&#239;&#172;?ciency large number genes. sequencing analyses performed T. brucei revealed wide-       addition, ribosome pro&#239;&#172;?ling data enabled identi- <br /> spread alternative trans-splicing, resulting multiple        &#239;&#172;?cation hundreds previously annotated CDSs <br /> transcripts, varying 50 untranslated region (UTR)          incorrectly annotated translation initiation sites, lengths particular gene (13,35,36). &#239;&#172;?ndings         suggest regulatory role uORFs. <br /> suggest variations 50 UTR lengths lead  exclusion regulatory elements in&#239;&#172;&#8218;uence <br /> translational ef&#239;&#172;?ciency (37). regulatory elements          MATERIALS METHODS small, called upstream open reading frames <br />                                                                 T. brucei culture cell harvest <br /> (uORFs) micro ORFs, ORFs located 50 UTR mRNA. eukaryotes, translation initiation           procyclic form (PF) T. brucei strain Lister 427 involves recruitment pre-initiation complex         cultured 27  C SDM-79 supplemented 10% fetal 50 end mRNA, followed scanning UTR             bovine serum hemin (7.5 mg/l) medium (21) locate &#239;&#172;?rst AUG start codon (38). uORFs affect       density 107 cells/ml. Wild-type bloodstream form (BF) scanning process shown in&#239;&#172;&#8218;uence            Lister 427 (MITat 1.2 clone 221) cultured 37  C <br /> translation downstream coding sequence (CDS)             HMI-11 cell density 1.5   106 cells/ml. affecting rate site translation initi-   minutes harvest, cycloheximide added &#239;&#172;?nal <br /> ation (39,40). Based RNA-sequencing data, T. brucei       concentration 100 mg/ml. Cells collected centri- <br />  20% 50 UTRs contain uORF (35,36).           fugation 3000   g, 4  C 5 min, washed polysome <br /> addition, shown luciferase reporter           lysis buffer (10 mM Tris-HCl pH 7.4, 300 mM KCl removal upstream start-codon lead         10 mM MgCl2), transferred 1.5-ml microcentrifuge 7-fold increase protein levels (41).                       tube pelleted. lyse cells, 360 ml polysome lysis <br />    past, approaches identify quantify             buffer, 40 ml 10% n-octylglycoside 20 units translated RNA transcripts focused isolation        TURBO DNaseI (Ambion) added 109 cells RNA transcripts polyribosome fractions (42).            incubated 30 min ice. lysate centrifuged                                                                          Nucleic Acids Research, 2014, Vol. 42,  6 3625 <br />  <br />  <br /> 16 000  g, 4  C 10 min, supernatant transferred    RNA library life cycle stage. investigate <br /> new microcentrifuge tube OD260 determined           distinct 3-nt periodicity ribosome pro&#239;&#172;?ling <br /> using Nanodrop 2000.                                          libraries, 50 ends mapped reads piled                                                                 piled coverage position gene <br /> Preparation RNA footprint mRNA sequencing                (using start stop codon reference point) <br /> libraries                                                       summed meta-gene analyses. Genes BF PF, 200 ml aliquots lysate               10 reads plotted region excluded. <br /> (OD260 = 40) digested RNase (Ambion)                                                                 De&#239;&#172;?nition upstream ORFs novel CDSs <br /> RT (1200 units) ice (1600 units). 1 h, diges- <br /> tions stopped adding 100 units SUPERase           uORF de&#239;&#172;?ned ORF  9 nt located <br /> RNase inhibitor (Ambion) RNase-treated samples.          annotated 50 UTR strand parallel, 100 units SUPERase RNase inhibitor           annotated ORF. Novel CDSs de&#239;&#172;?ned added 200 ml aliquot lysate containing         translated ORF   30 nt located 20 nt away RNase (undigested control). Monosomes puri&#239;&#172;?ed            existing annotations. cases multiple <br /> using sucrose gradients described previously (45).           overlapping novel ORFs different frames <br />    Total RNA undigested lysates footprints         strand, longest retained. collected monosome fractions puri&#239;&#172;?ed using          ORF de&#239;&#172;?ned translated   70% CDS hot acid Phenol&#226;&#8364;&#8220;Chloroform&#226;&#8364;&#8220;Isoamyl alcohol (v/v/v               covered pooled ribosome pro&#239;&#172;?ling reads  e. <br /> 25:24:1) 65  C (49). generate mRNA sequencing             libraries)  2 reads <span id='am-20' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>. <br /> libraries, undigested RNA polyA-enriched using Dynabeads&#195;&#8226; mRNA Puri&#239;&#172;?cation Kit (Ambion) according              Calculation ribosome release score manufacturer&#226;&#8364;&#8482;s instructions. polyA-enriched             Ribosome release scores (RRSs) ORFs, including <br /> RNA fragmented incubation RNA                    annotated CDSs, putative CDSs uORFs, Fragmentation Reagent (Ambion) 70  C 30 min.             calculated described slight modi&#239;&#172;?cations (53). <br /> Successful fragmentation monitored 15%                 Brie&#239;&#172;&#8218;y, pseudo 30 UTR region (pUTR3) &#239;&#172;?rst <br /> denaturing-PAGE gel.  ribosome footprints              assigned ORFs. pUTR3 ORF de&#239;&#172;?ned fragmented mRNA (26&#226;&#8364;&#8220;34 nt) size-selected elec-          region stop codon downstream start <br /> trophoresis using 15% denaturing-PAGE gel             codon frames strand. <br /> custom (IDT) synthetic RNA markers [50 -AUGUA              Reads mapped CDSs pUTR3 CACGGAGUCGAGCUCAACCCGCAACGCGA-                                  kilobase  e. CDSrpk pUTR3rpk) ribosome <br /> (Phos)-30 ] [50 -AUGUACACGGAGUCGACCCAAC                     pro&#239;&#172;?ling RNA-sequencing control libraries GCGA-(Phos)-30 ].                                               calculated ORF. ratio CDSrpk    Sequencing libraries prepared described (50)         pUTR3rpk ribosome pro&#239;&#172;?ling RNA- omission rRNA depletion step               sequencing control libraries  e. riboratio RNAratio) footprint library preparation use         calculated. RRS ORF de&#239;&#172;?ned KAPA HiFi polymerase (Kapa Biosystems) &#239;&#172;?nal             ratio riboratio RNAratio. <br /> ampli&#239;&#172;?cation step. <br />                                                                 Identi&#239;&#172;?cation novel putative CDSs important Pre-processing mapping read                              parasite&#226;&#8364;&#8482;s &#239;&#172;?tness <br /> Reads libraries processed mapped using        reanalysed data published genome-wide RNA parameters. Adapter sequence  e. CTGTAGGC            interference &#239;&#172;?tness&#226;&#8364;&#8220;costs association study (54) ACCATCAAT) trimmed 30 end reads                 newly de&#239;&#172;?ned CDSs. Brie&#239;&#172;&#8218;y, short read data  <br /> using cutadapt (http://code.google.com/p/cutadapt/)         loaded European Nucleotide Archive acces- <br /> reads shorter 20 nt trimming removed.           sion number ERP000431. Reads mapped Trimmed reads mapped reference genome               genome using bowtie2 default &#226;&#8364;&#732;local-sensitive&#226;&#8364;&#8482; <br /> using bowtie-2 default &#226;&#8364;&#732;local-sensitive&#226;&#8364;&#8482; mode (51).        mode. number reads mapping annotated <br /> Genome, gene annotations, strain 927              putative CDSs counted. identify version 4.2 downloaded EuPathDB (52)              putative CDSs signi&#239;&#172;?cantly reads mapped <br /> used reference analyses.                          RNA interference libraries, potentially  <br />                                                                 tribute parasite&#226;&#8364;&#8482;s &#239;&#172;?tness, performed differential <br /> Calculation abundance, translational ef&#239;&#172;?ciency read <br />                                                                 read count analyses pair-wise library comparisons <br /> 50 -end periodicity <br />                                                                 (listed  using softwares, including DESeq1 read considered map region midpoint       (55), DESeq2 (http://www.bioconductor.org/packages/ <br /> falls annotated range. Abundance region      release/bioc/html/DESeq2.html) EdgeR (56), library de&#239;&#172;?ned reads mapped kilobase           default settings. CDSs identi&#239;&#172;?ed signi&#239;&#172;?- <br /> million non-structural RNA reads (rpkm). Non-structural         cantly reads RNA interference libraries RNA refers reads mapped genome excluding          algorithms (P &lt; 0.05 case) linear fold change <br /> annotated rRNA tRNA regions. Translational ef&#239;&#172;?-               5 considered potentially related loss ciency region de&#239;&#172;?ned ratio abundance         &#239;&#172;?tness. pair-wise library comparisons include BF ribosome pro&#239;&#172;?ling library control         uninduced versus BF induced 6 days, BF uninduced <br />  3626 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  <br /> versus differentiated cells induced BF uninduced         prepared different conditions demonstrated ro- <br /> versus PF induced.                                           bustness ribosome pro&#239;&#172;?ling approach despite                                                              elaborate library preparation protocol (Supplementary <br /> Analysis proteomics data                                  Figure S2, BF digestion 4  C versus digestion room mass spectrometric raw data &#239;&#172;?les Butter et al.      temperature, R2 = 0.9755). <br /> (57) processed described data      ensure nuclease-resistant RNA fragments rep- <br /> searched additionally previously annotated       resent ribosome footprints short RNA fragments <br /> CDSs listed Supplementary Tables S4 S5. Brie&#239;&#172;&#8218;y,       protected RNA-binding proteins, compared <br /> MaxQuant 1.4.1.2 including Andromeda search engine       genome-wide distribution sequence reads obtained <br /> (58,59) used processing raw data database        ribosome pro&#239;&#172;?ling RNA-sequencing libraries. <br /> searching. experimental design template used      Alignment ribosome pro&#239;&#172;?ling reads revealed described (57). search performed    nuclease-protected reads predominantly mapped combination databases: T. brucei database     CDSs reads aligning intergenic regions <br /> taining 19 103 annotated protein entries strains 427     (Figure 1A). 50 nucleotide ribosome pro&#239;&#172;?ling 927 (version 5.0, Tb427 Tb927, downloaded            reads aligned 12 nt upstream ATG codon http://tritrypdb.org), list uORFs putative      18 nt upstream stop codon (Figure 1B). Given CDSs identi&#239;&#172;?ed study (Supplementary Tables S4        footprint length 28&#226;&#8364;&#8220;30 nt, ribosome protects mRNA S5) database containing typical contaminants.      12 nt upstream start codon 9&#226;&#8364;&#8220;11 nt  <br /> Enzyme search speci&#239;&#172;?city Trypsin/P            stream stop codon. contrast ribosome foot- <br /> miscleavages tryptic digests LysC           print reads, RNA-sequencing reads aligned miscleavages digests lysyl endopeptidase.           CDSs UTRs, expected (Figure 1A). addition <br /> Carbamidomethylation cysteines set &#239;&#172;?xed modi-      enrichment CDSs, alignment &#239;&#172;?cation; methionine oxidation protein N-terminal         ribosome pro&#239;&#172;?ling reads revealed distinct 3-nt period- <br /> acetylation considered variable modi&#239;&#172;?cations.        icity, 68% BF ribosome pro&#239;&#172;?ling reads starting Mass accuracy tolerances  recalibration)          &#239;&#172;?rst nucleotide codon (Figure 1B C 6 ppm precursor ions, 0.5 Da CID MS/MS           Supplementary Figure S3). 3-nt periodicity 20 ppm HCD MS/MS spectra, respectively.              observed mRNA sequence reads. <br /> False discovery rate &#239;&#172;?xed 1% <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>peptide</span>              T. brucei genes introns, encoding protein level, posterior error probability (PEP)     poly  polymerase encoding DNA/RNA <br /> sorting criterion. Identi&#239;&#172;?cations matched       helicase (61,62). Surprisingly, decrease <br /> runs 2-min window. Protein groups identi&#239;&#172;?ed      sequence reads detectable RNA-sequencing potential CDS database unique           introns, ribosome pro&#239;&#172;?ling data clearly peptide PEP &lt; 0.0005.                                  intronic RNA translated (Figure 2A).                                                              data indicate process cis-splicing  <br />                                                              usually inef&#239;&#172;?cient trypanosomes, tight <br /> RESULTS                                                      control mechanism ensures mature mRNAs <br /> Ribosome footprints reveal ribosome position sub-codon    enter translational machinery. <br /> resolution                                                      strong enrichment ribosome pro&#239;&#172;?ling reads <br />                                                              CDSs, lack ribosome pro&#239;&#172;?ling reads obtain genome-wide information protein synthesis,      introns observed 3-nt periodicity characteristic adapted ribosome pro&#239;&#172;?ling protocol published         ribosome-protected RNA footprints argue yeast mice (45,46) use T. brucei. sequenced    unspeci&#239;&#172;?c RNA protection (45).  ribosome pro&#239;&#172;?ling <br /> ribosome footprints fragmented mRNA         reads represent ribosome footprints provide <br /> trypanosome stages adapted vitro        means measure protein synthesis  addition, <br /> culture, PF tsetse &#239;&#172;&#8218;y midgut              identify previously unannotated CDSs proliferating BF lives blood mammalian    non-translated transcripts previously annotated coding <br /> host. Cultured cells treated cycloheximide       example Figure 1A, Tb927.10.2360). arrest translating ribosomes harvested.          incorrectly annotated translated cycloheximide shown stabilize RNA tran-          life cycle stages examined. <br /> scripts T. brucei (60), treatment limited 2 min applied  cells used ribosome <br />                                                              Identi&#239;&#172;?cation translation initiation sites <br /> pro&#239;&#172;?lilng cells used RNA-sequencing analyses. maximize accuracy ribosome footprints,         Genome-wide mapping spliced leader <span id='am-48' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acceptor</span> sites optimized digestion ef&#239;&#172;?ciency testing different    (SASs), e. 50 end 50 UTR nuclease concentrations reaction temperatures            spliced leader RNA trans-spliced, revealed hundreds <br /> (&#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section Supplementary           instances SASs mapped annotated <br /> Figure S1).  map translated regions RNA       CDSs far upstream annotated CDSs (13,35,36). exclude RNase-protected regions scanned       &#239;&#172;?ndings indicated genes true 43S pre-initiation complex, used sucrose gradients    CDS shorter annotated CDS  speci&#239;&#172;?cally enrich monosomes (Supplementary            allowed possibility translation initiation <br /> Figure S1). high reproducibility libraries       AUG upstream annotated translation <br />                                                                                                                                                  Nucleic Acids Research, 2014, Vol. 42,  6 3627 <br />  <br />                                                                                                           <br />  <br />  <br />  <br />                                                                                                           footprint (RPM) mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                  footprint (RPM) mRNA (RPM) <br />                                 mRNA density PF                                                                                               Tb927.3.3160                                                          Tb927.8.1510 <br /> mRNA (RPM) <br />  <br />  <br />  <br />                       4                                                                                                                2.0 <br />                                                                                                                                                                                                                4 <br />                       2 <br />                                                                                                                                        1.0                                                                     2 <br />                                                                                                                                        0.0                                                                     0 <br />                       0 <br /> footprint (RPM) <br />  <br />  <br />  <br />  <br />                       6         ribosome footprint density PF                                                                       0.3 <br />                                                                                                                                                                                                               2.0 <br />                       4                                                                                                                0.2 <br />                                                                                                                                        0.1                                                                    1.0 <br />                       2 <br />                                                                                                                                        0.0                                                                    0.0 <br />                       0 <br />                                                                                                           Chr III                                                                                  Chr VIII 00 <br />                                                                                                                                                 00      00      00      00                                          00      00 <br />                                 mRNA denstiy BF                                                                                           3,0     4,0     5,0     6,0                                   0,0   2,0     4,0 <br /> mRNA (RPM) <br />  <br />  <br />  <br />  <br />                       4                                                                                                                    81      81      81      81                                   49     49      49 <br />  <br />                       2 <br />                                                                                                           B <br />                       0 <br />  <br />  <br />  <br />  <br />                                                                                                           footprint (RPM) mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                              footprint (RPM) mRNA (RPM) <br />                                                                                                                                                 Tb927.5.1990                                                           Tb927.10.2720 <br />                                                                                                                                        4 <br /> footprint (RPM) <br />  <br />  <br />  <br />  <br />                       3         ribosome footprint density BF                                                                                                                                              2.0 <br />                       2                                                                                                                2                                                                      1.0 <br />                       1                                                                                                                0                                                                      0.0 <br />                       0 <br />                                                                                                                                        4                                                                      0.4 <br />                                      20        30          40          50        60          70 <br />                                   .23       .23         .23         .23       .23         .23 <br />                               7.10      7.10        7.10        7.10      7.10        7.10                                             2                                                                      0.2 <br />                             92        92          92          92        92          92 <br />                           Tb       Tb           Tb          Tb        Tb          Tb                                                   0                                                                      0.0 <br />      Chr X 600,000                           602,000          604,000     606,000         608,000 <br />                                                                                                           Chr V    00   00   00   00                                                                           Chr X     00        00        00 <br />                                                                                                                 5,5 16,0 16,5 17,0                                                                                    9,0       0,0       1,0 <br />                                                                                                               61     6    6    6                                                                                    69        70        70 <br /> B                                          28 nt - ribosome footprint <br />                                                                                                           Figure 2. Ribosome footprints reveal translated regions.   Ribosome <br /> relative enrichment <br />  <br />  <br />  <br />  <br />                                                    AUG                                        UAA <br />   read 5&#194;&#180; ends <br />  <br />  <br />  <br />  <br />                                                                                                           pro&#239;&#172;?les intron-containing genes. Introns represented                                                      P site <br />  <br />  <br />  <br />  <br />                                                                                           P site <br />                                                      site <br />  <br />  <br />  <br />  <br />                                                                                           site <br />  <br />  <br />  <br />  <br />                           0.4                                                                             dashed line. (B) Ribosome pro&#239;&#172;?les possibly mis-annotated <br />                                                                                                           CDSs. Black bars mark annotated CDSs. Grey bars mark CDSs pre- <br />                           0.2                                                                             dicted based ribosome pro&#239;&#172;?les. Green boxes represent ATG codons <br />                                                                                                           red boxes represent stop codons. <br />                            0 <br />                                   -24 -18 -12 -6   0     6 12 18 24        -24 -18 -12 -6     0    6 12 <br />                                       CDS position (nt start)          CDS position (nt stop) <br />  <br /> C                                                                                                            contrast RNA-sequencing, ribosome pro&#239;&#172;?ling <br />                                                                                                           allows determination translational landscape. <br /> percentage reads <br />  <br />  <br />  <br />  <br />                            80 <br />                                          ribosome footprint                                                allows identi&#239;&#172;?cation CDS mis-annotations <br />                                          mRNA <br />                            60                                                                             determination probable translation <br />                            40 <br />                                                                                                           initiation sites. Exemplary numerous genes,                                                                                                           Tb927.5.1990 detected translation                            20                                                                             &#239;&#172;?rst second AUG (green bars) indicating                             0                                                                             region translated BF PF                                  012 012                                                                  translation initiates second AUG (Figure 2B, <br />                                    frame <br />                                                                                                           left panel). Tb927.10.2720 observed strong trans- <br /> Figure 1. Ribosome footprints reveal coding sequences sub-codon                                        lation upstream annotated CDS. fact resolution.   Ribosome footprints strongly enriched                                          translation starts upstream AUG, CDSs. mRNA densities ribosome densities shown reads nucleotide million reads (RPM) normalize differences                                       frame annotated translation initiation site, library size. (B) Alignment 50 nucleotide ribosome foot-                                   suggests CDS Tb927.10.2720 longer print reads map close translation start translation termin-                                    annotated (Figure 2B, right panel). examples <br /> ation sites. Blue boxes mark approximate size ribosome                                         ribosome pro&#239;&#172;?ling data used footprint. (C) Percentage position sequence reads relative reading frame. <br />                                                                                                           annotation incorrectly annotated CDSs. <br />  <br />                                                                                                           Ribosome pro&#239;&#172;?ling reveals extensive translational control <br />                                                                                                           role RNA stability investigated initiation site.  RNA-sequencing allowed                                                    genome-wide scale T. brucei (29), global regulation unambiguous identi&#239;&#172;?cation mis-annotation                                                       levels protein translation analysed. genes, translation initiation does occur                                                   decided apply ribosome pro&#239;&#172;?ling &#239;&#172;?rst AUG downstream SAS.  use-                                                     approach evaluate degree regulation occurring <br /> fulness RNA-sequencing identify true translation                                                level translation trypanosomes. <br /> initiation sites limited depends genomic                                                      estimate rate protein synthesis trypano- <br /> context.                                                                                                  somes, determined ribosome footprint density <br />  3628 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  <br />                                                                CDSs [measured reads million reads kb                                                         BF                                                                   PF <br />  <br />  <br />  <br />  <br />                                                                number genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                           number genes <br />                                                                                                      3000                             mRNA                                  3000                              mRNA <br /> (rpkm), &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section]. 77 million                                                                           footprints                                                              footprints <br /> PF 44 million BF ribosome footprint sequence reads,                                              2000                                                                   2000 <br /> 28% (PF) 14% (BF) aligned annotated <br />                                                                                                      1000                                                                   1000 <br /> CDSs, 70% (PF) 85% (BF) mapped known <br /> structural non-coding RNA (ncRNA). high percent-                                                      0                                                                     0 <br /> age structural ncRNA footprint fraction  <br />  <br />  <br />  <br />                                                                                                                                                                                                            15 <br />                                                                                                               -4 <br />                                                                                                               -2 <br />                                                                                                                0 <br />                                                                                                                2 <br />                                                                                                                4 <br />                                                                                                                6 <br />                                                                                                                8 <br />                                                                                                               10 <br />                                                                                                               12 <br />                                                                                                               14 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     -6 <br />                                                                                                                                                                                          -3 <br />                                                                                                                                                                                               0 <br />                                                                                                                                                                                                    3 <br />                                                                                                                                                                                                        6 <br />                                                                                                                                                                                                             9 <br />                                                                                                                                                                                                            12 <br /> reported previously (45) results large                                                                    log2 rpkm                                                           log2 rpkm rRNA monosome fraction. lower percentage <br />                                                                B CDS reads BF sample compared PF                                                   1500       BF                                                          1500     PF <br />  <br />  <br />  <br />  <br />                                                                number genes <br />  <br />  <br />  <br />  <br />                                                                                                                                                           number genes <br /> sample libraries generated wild-type 427 strain reads aligned better-                                         1000                                                                   1000 <br /> annotated genome TREU 927 strain. BF, <br />                                                                                                       500                                                                    500 highly expressed genes encoding variant surface glycoproteins differ sequence                                                  0                                                                     0 427 927 strains, footprints genes <br />  <br />  <br />  <br />  <br />                                                                                                               .0 <br />                                                                                                                     .0 <br />                                                                                                                           .0 <br />                                                                                                                                  .0 <br />                                                                                                                                   0 <br />                                                                                                                                   0 <br />                                                                                                                                   0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      .0 <br />                                                                                                                                                                                            .0 <br />                                                                                                                                                                                             .0 <br />                                                                                                                                                                                             .0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                                                                              0 <br />                                                                                                                                 0. <br />                                                                                                                                 2. <br />                                                                                                                                 4. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           0. <br />                                                                                                                                                                                           2. <br />                                                                                                                                                                                           4. <br />                                                                                                           -8 <br />                                                                                                                    -6 <br />                                                                                                                          -4 <br />                                                                                                                                -2 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                     -8 <br />                                                                                                                                                                                          -6 <br />                                                                                                                                                                                          -4 <br />                                                                                                                                                                                          -2 aligned analysis.                                                                        translational efficiency (log2)                                        translational efficiency (log2) <br />     8072 genes (82% annotated CDSs), able detect   10 ribosome footprint reads,        C <br />  <br />  <br />  <br />                                                                BF rank translational efficiency <br /> threshold de&#239;&#172;?ned translation (Supplementary <br /> Table S1). rate translation differ                                                                                              genes, N = 7782 <br />                                                                                                       low <br /> greatly proteins. PF, observed                                                                                                   PUF (Pumillio), N = 10 <br /> 11 448-fold difference ribosome density (3548-fold dif-                                                                                         cytochrome oxidases, N = 8 <br /> ference mRNA levels) 1% highly                                                                                             enzymes required 1% weakly translated proteins, BF                                                                                            glycolysis, N = 10 observed 1623-fold difference ribosome density                                                                                             alternative oxidase 262-fold difference mRNA levels (Figure 3A). <br />                                                                                                       high                                low &#239;&#172;?ndings suggest translational ef&#239;&#172;?ciency signi&#239;&#172;?- <br />                                                                                                        PF rank translational efficiency <br /> cantly contribute regulation gene expression. <br />    determine translational ef&#239;&#172;?ciency individual <br />                                                                D <br /> genes, normalized differences mRNA abundance,                                                       BF mRNA density                                                   PF mRNA density <br />                                                                mRNA (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                                                                    mRNA (RPM) e. divided ribosome footprint density levels                                          2                                                                     2 mRNA abundance. ribosome pro&#239;&#172;?ling data                                                         1                                                                     1 <br /> indicate relatively unbiased distribution ribosome                                               0                                                                     0 <br /> density CDSs BF minor increase <br />                                                                                                               BF footprint density                                              PF footprint density density 50 end CDSs PF <br />                                                                footprint (RPM) <br />  <br />  <br />  <br />  <br />                                                                                                       4                                                                     4 <br />                                                                                                                                                    footprint (RPM) <br />  <br />  <br />  <br />  <br /> (Supplementary Figure S3).                                                      2                                                                     2 <br /> determining translational ef&#239;&#172;?ciency excluded <br /> mRNA footprint reads mapped &#239;&#172;?rst 40 nt                                                0                                                                     0 CDS avoid possible artifacts ribosome stalling                                                         Tb09.211.1070                                                    Tb09.211.1070 translation initiation sites. 1% ef&#239;&#172;?-          Chr IX                                     0                  0            0                         Chr IX          0         0         0 <br />                                                                                                            ,00                ,00          ,00                                       ,00       ,00       ,00 <br /> ciently 1% ef&#239;&#172;?ciently translated proteins,                                              59                 61           63                                        59        61        63 <br />                                                                                                       1,4                1,4          1,4                                       1,4       1,4       1,4 <br /> observed PF 117-fold BF 64-fold range <br />                                                                Figure 3. Translational ef&#239;&#172;?ciency regulated.   Histograms translational ef&#239;&#172;?ciency, e. proteins     mRNA abundance ribosome density (rate protein synthesis) <br /> produced transcript (Figure 3B Supplementary           BF (left panel) PF parasites (right panel). (B) Histogram Table S2). range translational ef&#239;&#172;?ciency similar    translational ef&#239;&#172;?ciency (ratio ribosome footprint density mRNA observed yeast roughly 10-fold         abundance) BF (left panel) PF parasites (right panel). (C) Pair- <br />                                                                wise comparisons translational ef&#239;&#172;?ciency PF BF. CDSs higher described mice (45,46).          ranked based translational ef&#239;&#172;?ciency (1 = highest translational ef&#239;&#172;?- <br /> Interestingly, observed correlation RNA          ciency, 7782 = lowest translational ef&#239;&#172;?ciency) BF PF. Ranks <br /> abundance translational ef&#239;&#172;?ciency (R2 = 0.03), sug-        life cycle stages correlation R = 0.7428. Gene <br /> gesting translational ef&#239;&#172;?ciency regulated independ-    families developmentally regulated translational ef&#239;&#172;?ciencies ently RNA stability.                                        colour coded: Pumillio genes (red), cytochromes oxidase (blue) genes <br />                                                                required glycolysis (63) (green) alternative oxidase <br />                                                                (Tb927.10.7090, black). (D) Ribosome footprint pro&#239;&#172;?le gene Life cycle-speci&#239;&#172;?c regulation translational ef&#239;&#172;?ciency       developmentally regulated translational ef&#239;&#172;?ciency. Green arrow indi- <br />                                                                cates direction transcription. addressed question translational ef&#239;&#172;?ciency regulated life cycle-speci&#239;&#172;?c manner. Ribosome <br /> density measurements allow meaningful comparisons         cells (46,64);  speed translation rate protein synthesis speed transla-    identical life cycle stages live tion assumed remain constant. assumption         different temperatures. addition, current protocol <br /> supported measurements mouse embryonic stem              does allow absolute quanti&#239;&#172;?cation mRNA                                                                                       Nucleic Acids Research, 2014, Vol. 42,  6 3629 <br />  <br />  <br /> ribosome footprint levels different life cycle                   changes described glucose metabolism <br /> stages.  decided compare overall                        (Figure 3C). BF entirely dependent changes translational ef&#239;&#172;?ciency life cycle                   glycolytic pathway generate energy, PF living stage. end ranked genes life cycle                   insect midgut, glucose availability uncommon stage based relative translational ef&#239;&#172;?ciency                   absent, shown contain elaborate <br /> compared rank life cycle stages.                      energy metabolism (66). contrast, ef&#239;&#172;?cient translation pair-wise comparison indicated generally positive cor-                  group PUF (Pumilio FBF) proteins PF relation translational ef&#239;&#172;?ciency stages                  unexpected (Figure 3C). PUF proteins regulate translation <br /> (Figure 3C, Pearson&#226;&#8364;&#8482;s correlation coef&#239;&#172;?cient = 0.7428,                      mRNA stability binding sequences target <br /> P &lt; 0.0001, 95% CI 0.7327&#226;&#8364;&#8220;0.7526), revealed                     RNAs shown affect gene expression distinct life cycle-speci&#239;&#172;?c differences subsets genes.               organisms divergent Plasmodium falciparum, example, 58 genes lowest 25% terms                   T. brucei, humans yeast (67&#226;&#8364;&#8220;69). Interestingly, translational ef&#239;&#172;?ciency PF 25%                    poly  binding proteins, isoforms ef&#239;&#172;?ciently translated proteins BF.                        eIF4Es &#239;&#172;?ve eIF4G isoforms, proteins <br /> contain various RNA binding proteins, numerous phos-                        involved control translation initiation, ef&#239;&#172;?- <br /> phatases expression site-associated genes (Table 1                  ciently translated PF BF (Supplementary <br /> Supplementary Table S3). example gene                           Table S2). ef&#239;&#172;?ciently translated BF PF                        Given large dynamic range life cycle-speci&#239;&#172;?c <br /> Figure 3D.                                                                  regulation, propose differences translational <br />    Based mRNA protein level measurements, differ-                    ef&#239;&#172;?ciency contribute substantially control gene <br /> ential translational ef&#239;&#172;?ciency predicted                       expression T. brucei. <br /> aconitase, cytochrome oxidases V VI procyclins <br /> (22,25,65). genes, data indicate                                                                             uORFs frequently translated regulate <br /> increased translational ef&#239;&#172;?ciency PF (Table 1),                                                                             translational ef&#239;&#172;?ciency <br /> con&#239;&#172;?rming previous observations validating meas- <br /> urements. selected groups proteins, e.g. annotated                broad  100-fold range translational ef&#239;&#172;?ciency <br /> cytochrome oxidase required glycolysis,                     genes differences translational <br /> noticed translational ef&#239;&#172;?ciency correspond                    ef&#239;&#172;?ciencies life cycle stages raise question  <br />  <br /> Table 1. Developmentally regulated translation <br />  <br /> Gene <span id='am-47' about='oboInOwl:id' typeof='owl:Thing'>ID</span>                    Description                                                            BF rank         PF rank    Change                                                                                                                              rank (PF&#226;&#8364;&#8220;BF) <br />  <br /> Translation regulated   BF <br />   Tb927.7.3250             Expression site-associated gene 6 (ESAG6) protein, putative            1372            7405         6033 <br />   Tb927.4.3980             Chaperone protein DNAj, putative                                       1301            7248         5947 <br />   Tb927.10.4780            GPI inositol deacylase precursor (GPIdeAc)                              390            5994         5604 <br />   Tb927.8.8140             Small <span id='am-6' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GTP</span>-binding rab protein, putative                                1217            6804         5587 <br />   Tb927.6.3480             RNA-binding protein, putative (DRBD5)                                  1856            7262         5406 <br />   Tb11.01.4701             Membrane-bound acid phosphatase 1 precursor (MBAP1)                     689            6065         5376 <br />   Tb927.1.4650             Cyclin-like F-box protein (CFB2)                                         10            5289         5279 <br />   Tb927.3.5660             <span id='am-53' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UDP</span>-Gal UDP-GlcNAc-dependent glycosyltransferase, putative           797            6041         5244 <br />   Tb927.2.6000             Glycosylphosphatidylinositol-speci&#239;&#172;?c phospholipase C (GPI-PLC)         1634            6793         5159 <br />   Tb927.4.5310             Serine/threonine-protein kinase  putative                            1656            6776         5120 <br /> Translation regulated   PF <br />   Tb927.5.440              trans-Sialidase, putative                                              6785             418        6367 <br />   Tb927.7.6850             trans-Sialidase (TbTS)                                                 7513            1445        6068 <br />   Tb927.7.7470             Receptor-type adenylate cyclase GRESAG 4, putative                     7001            1268        5733 <br />   Tb927.1.2120             Calpain, putative                                                      6258             739        5519 <br />   Tb927.4.360              1,2-Dihydroxy-3-keto-5-methylthiopentene dioxygenase, putative         6587            1089        5498 <br />   Tb09.160.5550            Calpain-like cysteine peptidase, putative                              6936            1868        5068 <br />   Tb927.8.7690             <span id='am-28' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Amino acid</span> transporter (pseudogene), putative                          6693            1731        4962 <br />   Tb927.8.1610             MSP-B, putative                                                        7199            2272        4927 <br />   Tb11.01.6650             Serine/threonine-protein kinase, putative                              5280             363        4917 <br />   Tb927.7.7110             Leucine-rich repeat protein (LRRP), putative                           6501            1601        4900 <br /> Genes translational regulation PF previously shown <br />  Tb927.10.280            Cytochrome oxidase subunit VI (COXVI)                                    2820             810        2010 <br />  Tb09.160.1820           Cytochrome oxidase subunit V (COXV)                                       935             291         644 <br />  Tb927.10.14000          Aconitase (ACO)                                                           657              31         626 <br />  Tb927.6.510             GPEET2 procyclin precursor                                               3848             365        3483 <br />  Tb927.10.10260          EP1 procyclin (EP1)                                                      4682              34        4648 <br />  Tb927.10.10220          Procyclin-associated gene 2 (PAG2) protein (PAG2)                        6903            5375        1528 <br />  Tb927.5.330             Receptor-type adenylate cyclase GRESAG 4, putative                       3779             809        2970 <br />  <br /> List genes highest change translational ef&#239;&#172;?ciency PF BF. List does include hypothetical genes. <br />  3630 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  <br />                                                                                                                                                B protein translation regulated. small number                               uORF <br />  <br />  <br />  <br />  <br />                                                                                                                                                  percentage reads <br />                                                                                                                                                                        80 <br />  <br />  <br />  <br />  <br />                                                                  relative enrichment genes T. brucei, sequence motifs 30 UTR                                                AUG                       UAA <br />  <br />  <br />  <br />  <br />                                                                    read 5&#194;&#180; ends linked developmentally regulated changes trans-                                                                                                             60 <br />                                                                                                                                                                                 uORF <br /> lational ef&#239;&#172;?ciency, large majority genes                                                                                                             40 motifs identi&#239;&#172;?ed (70). Numerous reports eukaryotes demonstrated uORFs                                                                                                                      20 <br />                                                                                         -24 -18 -12 -6 0 6 -24 -18 -12 -6 0 6 12 <br /> function important regulators gene expression (71).                                                                                                              0 <br />                                                                                       CDS position (nt start) CDS position (nt stop) example, analysis mRNA protein levels                                                                                                     1 <br />                                                                                                                                                         frame <br />                                                                                                                                                                2            0 <br /> &gt;10 000 mammalian genes revealed presence                                                                                          ribosome footprint uORF correlates signi&#239;&#172; reduction expres-                                                                                      mRNA <br /> sion downstream CDS. Furthermore, using                    C <br />  <br />  <br />  <br />  <br />                                                                     footprint (RPM) <br /> reporter constructs, estimated presence                            0.8 BF                                      BF <br />                                                                                                                           0.8 uORF results decrease gene expression 30&#226;&#8364;&#8220;                               0.4                                 0.4 <br /> 80% minor reduction mRNA levels (72). analyses performed T. brucei,                            0.0                                 0.0 shown removal uATG leads 7-                                     PF                                    PF <br />  <br />  <br />  <br />  <br />                                                                     footprint (RPM) <br /> fold increase protein levels luciferase reporter (41).                        0.4                                   4 manuscript, use term ORF uORF refer                              0.2                                   2 regions DNA sequences beginning ATG ending termination codon                                   0.0                                   0 <br /> encode proteins. term coding sequence (CDS)                       Chr                                                                                                         Tb927.1.1610 <br />                                                                                                                           Chr                                                                                                                                                Tb927.1.1670 <br /> used refer ORFs encode proteins. Minimally,                                      411,000   411,500 412,000                  420,000 420,200 420,400 <br /> uORFs consist 9 nt, containing upstream start codon        Figure 4. Ribosome footprints reveal translation uORFs. <br /> (uATG), additional sense codon termination                Alignment 50 nucleotides ribosome footprint reads <br /> codon, termination codon located              map close translation start translation termination sites                                                                  uORFs. (B) Percentage position sequence reads relative reading <br /> downstream start codon main CDS (73).              frame. (C) Ribosome pro&#239;&#172;?les genes uORF (left panel)    50 UTRs assigned 4909          uORF (right panel). Narrow grey boxes represent 50 UTRs, <br /> genes (927 version 4.2), applying criteria listed      green box represents AUG-codon, red box represents termination identi&#239;&#172;?ed 8310 uORFs 1092 (22%) 50               codon. <br /> UTRs contain uORF (Supplementary <br /> Table S4).  uORF common T. brucei mammals 40&#226;&#8364;&#8220;50% genes                  observed higher mRNA level genes <br /> contain uORFs (74,75).  uORFs                 uORFs compared genes uORFs explain translational regulation tran-        (P &lt; 0.0001), differences mRNA levels lower <br /> scripts, represent factors import-       differences ribosome density (Table 2).  <br /> ant regulation gene expression T. brucei.          data indicate median translational ef&#239;&#172;?ciency <br />    determine degree uORFs translated,         higher genes uORF genes prerequisite affecting translational ef&#239;&#172;?ciency      uORF, suggesting presence uORFs nega- <br /> downstream CDS, calculated ribosome density               tively impact translation downstream CDSs. uORFs identi&#239;&#172;?ed 1834 uORFs (22%)                    Noteworthy, median ribosome density CDSs <br />   2  read-coverage  70% ORF covered.                  higher BF PF (Table 2), observed average length uORFs 22 aa. Just like           opposite 50 UTRs (P &lt; 0.0001). 50 UTRs annotated CDSs, alignment ribosome pro&#239;&#172;?ling           higher ribosome density PF (median rpkm: <br /> reads uORFs revealed distinct 3-nt period-        7.73) BF (median rpkm: 2.89, Supplementary <br /> icity, 63% BF ribosome pro&#239;&#172;?ling reads starting        Table S4, example Figure 4C). addition, &#239;&#172;?rst nucleotide codon (Figure 4A B).          BF 50 UTRs uORF (4.08 rpkm) periodicity ribosome footprints uORFs          higher ribosome density 50 UTRs slightly pronounced annotated CDS,             uORFs (2.54 rpkm; P &lt; 0.0001). Unexpectedly, data suggest large proportion uORFs                 contrary true PF. PF 50 UTRs <br /> translated.                                                      uORF (8.34 rpkm) higher ribosome <br />    evaluate regulatory potential uORFs,             density 50 UTRs uORF (5.01 rpkm; <br /> compared ribosome densities transcripts          P &lt; 0.0001). example non-AUG uORF taining uORF (N = 1092)           Figure 4C, right panel. non-AUG uORFs <br /> uORFs (N = 3817). BF, median ribosome density             observed yeast mice embryonic stem cells CDSs transcripts uORFs                    (mESCs). Interestingly, increase 50 UTR translation, <br /> 30.31 rpkm compared 44.86 rpkm genes            similar observed PF compared uORFs (Table 2). PF, median ribosome densities            BF, observed yeast starvation decrease 20.01 rpkm (genes uORF) 36.50 rpkm                 50 UTR translation observed differentiation (genes uORF).  life cycle stages         pluripotent mESCs embryoid bodies (45,46). <br /> observed higher ribosome density genes                Genome-wide analyses prove disprove direct <br /> uORFs genes uORFs (P &lt; 0.0001).              correlation translation uORFs                                                                                      Nucleic Acids Research, 2014, Vol. 42,  6 3631 <br />  <br />  <br /> Table 2. Translational ef&#239;&#172;?ciency genes uORF <br />  <br />                                                                    Transcripts            Transcripts          Mann&#226;&#8364;&#8220;Whitney test <br />                                                                    uORF (N = 3817)             uORF (N = 1092) <br />  <br /> Bloodstream form <br />   Ribosome density (median rpkm)                                   30.31                       44.66                        P &lt; 0.0001 <br />   mRNA levels (median rpkm)                                        30.83                       36.13                        P &lt; 0.0001 <br />   Translational ef&#239;&#172;?ciency (ribosome density/mRNA levels)            1.00                        1.27                        P &lt; 0.0001 <br /> Procyclic form <br />   Ribosome density (median rpkm)                                   20.01                       36.50                        P &lt; 0.0001 <br />   mRNA levels (median rpkm)                                        24.04                       30.38                        P &lt; 0.0001 <br />   Translational ef&#239;&#172;?ciency (ribosome density/mRNA levels)            0.87                        1.26                        P &lt; 0.0001 <br />  <br /> Reads mapping &#239;&#172;?rst 40 nt CDS considered measurements translational ef&#239;&#172;?ciency. Number genes annotated <br /> 50 UTR, N = 4909. <br />  <br />  <br />  <br />  <br /> translational ef&#239;&#172;?ciency complicated existence                 chromosome 10 assembly genomic coordinates <br /> multiple 50 UTR isoforms, resulting widespread                        incompatible.). <br /> differential trans-splicing (35,36,76). time                      determine set candidate CDSs heterogeneity UTR lengths raises intriguing                     translated, analysed published proteomics datasets <br /> possibility inclusion exclusion uORF                         (57) search protein products. 24 2021 alternative trans-splicing represent regula-                  candidate CDSs (average size 117 aa, 13 kDa), protein <br /> tory mechanism modulate translational ef&#239;&#172;?ciency.                        products identi&#239;&#172;?ed;                                                                             identi&#239;&#172;?ed peptides matched annotated CDSs. Ribosome footprints reveal presence hundreds                     addition, identi&#239;&#172;?ed protein products 31 uORFs <br /> previously annotated CDSs                                               (Supplementary Table S4). suspect low <br />                                                                            number identi&#239;&#172;?ed protein products relative small proteins (&lt;200 aa) shown                    large number candidate CDSs caused,   play major roles plant animal development (77,78),                  experimental set original proteomics complexity short proteome remains largely                       analysis proteins separated 1D-SDS- <br /> unexplored algorithms reliably predict short                    PAGE prior gel protease digest mass spectro- <br /> CDSs lacking (79). observed                       metric analysis proteins size smaller ribosome footprints aligned annotated features,                  5&#226;&#8364;&#8220;10 kDa typically run gel, (ii) limited BF 101 197 reads (0.23%) PF 1 116 030                       sensitivity mass spectrometric analyses compared reads (1.44%) did  noteworthy  just                   DNA sequencing-based techniques (iii) fact <br /> observed 50 UTRs, percentage reads aligning                 ribosome-protected RNAs translated. CDSs higher PF BF. suspect                        &#239;&#172;?rst point supported fact 67 reads originate annotated                       (3%) candidate CDSs  90 aa ( 10 kDa) UTRs stem previously annotated                        28% large candidate CDSs identify <br /> CDSs. previous RNA-sequencing-based analysis                             peptides. Evidence supporting possibility detected 1114 new, annotated transcripts PF,                     ribosome-protected RNAs translated exists mice <br /> 1011 potential encode                      known long ncRNAs peptides (  25 aa). Using available proteomics data,                   associated ribosomes (46). evaluate coding <br /> authors able con&#239;&#172;?rm translation 19 1114                 potential transcript separate long ncRNA <br /> transcripts (13). Given ability determine ribosome                  small coding genes, new metric established, <br /> positions nucleotide resolution, looked new, pre-                called RRS (53). RRS takes advantage viously annotated CDSs.                                                 fact translating ribosomes released encounter- <br />    mapped ribosome footprint reads BF                      ing stop codon. release results sharp decrease PF determined ribosome footprint density                   ribosome occupancy protein-coding regions potential CDSs 10 aa length. avoid overlap                  subsequent 30 UTR. Consequently, RRS annotated CDSs, considered ORFs located                       de&#239;&#172;?ned ratio footprint reads putative 20 nt away annotated features.                          CDS footprint reads corresponding 30 UTR <br /> approach enabled identify set 2021 candidate                    divided ratio RNA reads putative CDS <br /> CDSs  2  read-coverage  70% ORF                            RNA reads corresponding 30 UTR  covered. candidate CDSs ranged length 10                       &#226;&#8364;&#732;Materials Methods&#226;&#8364;&#8482; section). 378 aa (average 30 aa, Supplementary Table S5).                        Similarly annotated CDSs uORFs, averaged <br /> 797 1114 previously identi&#239;&#172;?ed transcripts,                 ribosome footprint data indicated de&#239;&#172;?ned 3-nt peri- average ribosome footprint read-coverage  2                      odicity sequence reads occurring near translation <br /> (Supplementary Table S6. Previously identi&#239;&#172;?ed transcripts                  initiation sites.  observed de&#239;&#172;?ned chromosome 10 considered                         drop ribosome densities translation termination <br /> sequencing reads mapped newer version                     sites (Figure 5A B), indicate lack  3632 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />                                                                                   B <br />                       putative CDS <br />                                                                                                          80 <br />                                                                                                                             1445 genes annotated T. brucei (Supplementary <br /> relative enrichment <br />  <br />  <br />  <br />  <br />                                         AUG                            UAA <br />  <br />  <br />  <br />  <br />                                                                                    percentage reads <br />                                                                                                                             Table S7). 1445 genes median RRS   read 5&#194;&#180; ends <br />  <br />  <br />  <br />  <br />                                                                                                          60                 83.9 compared RRS 0.19 subset                                                                                                          40 <br />                                                                                                                             genes annotated hypothetical unlikely (N = 90).  <br />                                                                                                                             similar observation mice, RRS appears                                                                                                          20                 good predictor productive translation.                       -24 -18 -12 -6   0    6 12      -24 -18 -12 -6   0    6 <br />                                                                                                          0                  determined RRS previously annotated  <br />                       CDS position (nt start) CDS position (nt stop)                                0     1 <br />                                                                                                                   frame <br />                                                                                                                         2   didate CDSs ful&#239;&#172;?lled criteria RRS calculation <br />                                                                                                                             (N = 265). median RRS candidate CDSs                                                                                 ribosome footprint <br />                                                                                 mRNA                                        3.99 lower median RRS annotated <br /> C                                                                                                                           genes (83.9). Thirteen candidate CDSs RRS larger <br />  (RPM) <br />                                                                                                                             83.9 98 candidate CDSs RRS higher                           BF mRNA density                        BF mRNA density                                            described intron-containing poly  polymerase <br />                       4 <br />                       3                                      8                                                              (RRS = 9.63). RRS calculated   mRNA <br />  <br />  <br />  <br />  <br />                       2                                      4                                                              candidate CDSs identi&#239;&#172;?ed <br />                       1                                                                                                     protein products (RRS = 57.44). <br />                       0                                      0 <br />                                                                                                                                Finally, attempt learn function                2.0 BF footprint density                      8   BF footprint density                                       putative CDSs, searched known protein motifs <br />   footprint <br />  <br />  <br />  <br />  <br />                1.5                                           6 <br />                1.0                                           4 <br />                                                                                                                             candidate CDSs identi&#239;&#172;?ed <br />                0.5                                           2                                                              protein products RRS 10. <br />                0.0                                           0                                                               candidate CDS located                20 BF RNAi induction                       8   BF RNAi induction                                       histone H2B gene array contained   RNAi reads <br />  <br />  <br />  <br />  <br />                15                                            6                                                              H2B signature motif (Supplementary Table S5),                10                                            4                                                              motifs identi&#239;&#172;?ed. <br />                 5                                            2 <br />                 0                                            0 <br />                                                                                                                                Taken  proteomics data de&#239;&#172;?ned <br />                                                                                                                             ribosome release (high RRS) suggest extensive translation <br />                           BF 6 days RNAi induction        4   BF 6 days RNAi induction                                annotated CDSs T. brucei. <br />   RNAi reads <br />  <br />  <br />  <br />  <br />                       8                                      3 <br />                       4                                      2 <br />                                                              1                                                              Newly identi&#239;&#172;?ed CDSs important parasite &#239;&#172;?tness <br />                       0                                      0 <br />                                                                                                                             combination ribosome pro&#239;&#172;?ling data previ- <br />                                        putative CDS                        putative CDS <br />                  Chr III                                     Chr X                  0           0                           ously published mass spectrometric data allowed                                 00          00          00                       ,80         ,20                            identify large number new CDSs. Nonetheless, bio- <br />                              2,0         2,4         2,8                      77           78 <br />                            62          62          62                      2,5          2,5 <br />                                                                                                                             logical signi&#239;&#172;?cance CDSs remains determined. <br /> Figure 5. Ribosome footprints reveal previously annotated CDSs.                                                          addition, ORFs high ribosome footprint <br />   Alignment 50 nucleotides ribosome footprint reads map close translation start translation termination sites candi- <br />                                                                                                                             density, low RRS mass spectrometric <br /> <span id='am-14' about='oboInOwl:date' typeof='owl:Thing'>date</span> CDSs. (B) Percentage position sequence reads relative                                                         evidence translated functional proteins, <br /> reading frame. (C) Ribosome, mRNA RIT-seq (RNAi target                                                                    play important regulatory roles. <br /> sequencing) pro&#239;&#172;?les previously annotated putative CDSs.                                                               decided evaluate importance                                                                                                                             candidate CDSs, independent RRS mass spectro- <br />                                                                                                                             metric evidence, parasite survival. previously pub- <br /> productive translation. observed abrupt                                                                       lished high-throughput phenotyping approach, termed <br /> drop footprint density downstream stop codon                                                                      RIT-seq, measured &#239;&#172;?tness&#226;&#8364;&#8220;cost associations using annotated genes T. brucei (Figure 1A B),                                                                       RNA interference (RNAi). RIT-seq data revealed decided apply parameters evaluate like-                                                                  signi&#239;&#172; loss &#239;&#172;?tness RNAi induction 2724 CDSs <br /> lihood ORFs high footprint density                                                                     BF, 1972 CDSs PF 2677 CDSs induced translated functional proteins.  candidate                                                                 differentiated parasites (54). approach based CDSs de&#239;&#172;?ned ORFs minimum length                                                                           integration RNAi libraries trypanosomes 10 aa average ribosome footprint coverage 2                                                                       comparison recovery RNAi targets tryp- 70% ORF. de&#239;&#172;?ne correspond-                                                                      anosome populations RNAi induction. <br /> ing putative 30 UTR, applied parameters used                                                                    RNAi library generated using genomic DNA, mice (53), e. 30 UTR de&#239;&#172;?ned region                                                                      effect parasite &#239;&#172;?tness determined beginning immediately downstream ORF                                                                             annotated genes. analysing datasets, &#239;&#172;?nd <br /> ending &#239;&#172;?rst subsequent start codon                                                                            signi&#239;&#172; loss &#239;&#172;?tness RNAi induction 214 <br /> reading frame). limitation RRS                                                                      putative CDSs BF (6 days RNAi induction), calculated ORF entire foot-                                                                  16 CDSs PF 227 CDSs differentiation <br /> print read RNA read map putative CDS                                                                         (Supplementary Table S8). examples candidate corresponding 30 UTR. 30 UTRs                                                                          CDSs potentially essential viability BF short ful&#239;&#172;?ll criterion simply did                                                                   Figure 5C. important note RNAi library <br /> contain footprint reads, sign ef&#239;&#172;?cient translation                                                                used RIT-seq experiments contains median insert <br /> termination, able determine RRS                                                                     size 600 bp (average: 1 kb), somewhat limiting                                                                            Nucleic Acids Research, 2014, Vol. 42,  6 3633 <br />  <br />  <br /> resolution RIT-seq data (80,81).        regulation translation. Importantly, regulation &#239;&#172;?ndings suggest important biological role subset         observe agrees reported previous studies genes missed previous genome anno-           small subset genes includes proteins <br /> tations. Intriguingly, RNAi data reveal large develop-        previously unknown developmentally regulated. mental differences &#239;&#172;?tness costs associated        addition, ef&#239;&#172;?cient translation proteins annotated regulation putative CDSs, suggesting set         cytochrome oxidases PF enzymes important candidate CDSs important                glycolysis BF consistent differences mammalian stage.                                                  energy metabolism BF entirely dependent <span id='am-35' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-36' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>gly</span>- <br />                                                                   colysis generate energy. <br />                                                                      translational control achieved T. brucei <br /> DISCUSSION                                                        remains seen, analysis suggests uORF <br />                                                                   important contributor. Work organ- study, report &#239;&#172;?rst genome-wide analysis         isms shown generally presence uORFs cor- <br /> protein synthesis strand-speci&#239;&#172;?c analysis RNA            relates reduced gene expression (71).  transcript levels T. brucei. &#239;&#172;?rst analysis     genes, like transcription factor GCN4 yeast eukaryotic pathogen organism tran-           activating transcription factor ATF4 mammals, <br /> scriptional control. data reveal large differences         correlation reversed stress (83,84).  uORF <br /> translational ef&#239;&#172;?ciency transcripts life        exert positive negative effects translation cycle stage transcript different          downstream CDSs. Based previous 50 UTR annota- <br /> stages. addition, sequencing ribosome-protected         tions, identi&#239;&#172;?ed uORFs 22% T. brucei genes, <br /> RNA enabled identify transcripts likely         interesting learn regulatory effects translated. Hundreds putative, previously unidenti-         uORFs supplemented translation non- <br /> &#239;&#172;?ed CDSs appear essential parasite &#239;&#172;?tness,          AUG uORFs. ribosome pro&#239;&#172;?ling data indicate analysis available proteomics data con&#239;&#172;?rmed            increase translation 50 UTRs ubiqui- <br /> existence 20 previously unknown CDSs.        tous translation initiation non-AUG sites    Eukaryotic gene expression regulated multiple            PF compared BF. Similarly, increase levels, translation mRNA proteins              translation 50 UTRs observed yeast important (1,7). ability determine         starvation (45). highly conserved stress- translatome transcriptome enabled                  induced mechanisms regulate translation involves evaluate ef&#239;&#172;?ciency individual transcripts          inactivation translation initiation factor eIF2a translated proteins suggests translational           phosphorylation conserved serine (85,86). <br /> control important regulatory mechanism                Phosphorylation T. brucei L. major orthologs <br /> T. brucei. single condition, observed transla-         occurs corresponding Thr residues (6,87), tional ef&#239;&#172;?ciency vary orders magnitude;            generation T. brucei cell line exclusively expressing  importance equals RNA stability             mutant form eIF2a demonstrated phosphorylation similar range measured (82).             eIF2a required heat-induced translational observed correlation RNA abundance              arrest formation heat shock stress granules <br /> translational ef&#239;&#172;?ciency, suggesting translational ef&#239;&#172;?-       T. brucei (88). Given &#239;&#172;?nding eIF2a ciency regulated independently RNA stability.               important determining stringency initiator codon <br />    use ribosome density estimate rate protein        selection (89), interesting eIF2a <br /> synthesis, speed translation constant.        phosphorylation affects selection initiator codons <br /> assumption supported measurements mouse em-              plays role &#239;&#172;?ne-tuning gene <br /> bryonic stem cells demonstrate rate translation       expression. consistent different classes mRNA,              valuable tool study translational <br /> kinetics elongation independent length            regulation, ribosome pro&#239;&#172;?ling invaluable protein abundance speed translation           discovery small peptides.  2% dependent codon usage (46,64).  speed         human genome contains annotated CDSs (90), transcrip- translation different         tome analyses revealed genome-wide tran- <br /> life cycle stages parasite live 37  C 27  C.   scription RNA, considered non- <br /> Furthermore, approach does permit measurement             protein-coding. lower organisms percentage absolute mRNA footprint abundance              genome encoding proteins higher (62,91,92), individual life cycle stages, making direct comparisons        pervasive transcription non-protein coding regions <br /> translation ef&#239;&#172;?ciencies individual transcripts unreli-        essentially organisms (93). vast <br /> able.  did compare translational ef&#239;&#172;?-            ncRNA triggered great ciency directly, determined &#226;&#8364;&#732;rank&#226;&#8364;&#8482;              elucidating biological signi&#239;&#172;?cance transcripts <br /> translational ef&#239;&#172;?ciency genes life cycle          led identi&#239;&#172;?cation numerous regulatory <br /> stage, assuming &#226;&#8364;&#732;change rank&#226;&#8364;&#8482; indicated life            mechanisms controlled short long ncRNA. <br /> cycle-dependent translational control.             Interestingly, majority transcription occurs general positive correlation translational ef&#239;&#172;?ciency           long ncRNA humans (94), contain life cycle stages (Pearson r = 0.7428),           ORFs making dif&#239;&#172;?cult unambiguously classify tran- large number genes, observed developmental            script non-protein-coding. distinguish long ncRNA <br />  3634 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  mRNA, ORF length served                  SUPPLEMENTARY DATA <br /> commonly applied criterion, select-      Supplementary Data available NAR Online. <br /> ive pressure, short ORFs occur frequently chance probability long ORFs occur chance low.  recent &#239;&#172;?ndings reveal eukaryotic      ACKNOWLEDGEMENTS <br /> genomes contain large number small CDSs (95&#226;&#8364;&#8220;97) small peptides play important biological roles       thank Stan Gorski, Susanne Kramer, Christian Janzen <br /> (77,78,98,99). addition, numerous long ncRNAs          Jose-Juan Lopez-Rubio valuable discussions taining ORFs longer 100 aa         critical reading manuscript Yanjie Chao sociate ribosomes (46,53). &#239;&#172;?ndings           Jan Medenbach appreciated advice complicate annotation CDSs demand new          generating ribosome pro&#239;&#172;?ling libraries. <br /> approaches annotate genes, raise exciting pos- <br /> sibility large number unknown small CDSs exists     FUNDING awaits discovery. <br />    study, RNA obtained 427 strain,        Young Investigator Program Research Center common laboratory strain, sequence reads         Infectious Diseases (ZINF) University aligned complete genome 927           Wuerzburg, Germany; German Research Foundation <br /> strain;  RNA-sequencing analysis indicates       DFG [SI 1610/2-1]; Human Frontier Science Program <br /> widespread transcription.  seen organ-        T.N.S.); French National Research Agency [ANR- <br /> isms, active transcription necessarily good pre-     2010-GENM-011-01, GENAMIBE C.C.H.]. Funding <br /> dictor CDSs. contrast, ribosome                open access charge: German Research Foundation <br /> footprint density highly enriched annotated      (DFG) University Wuerzburg funding <br /> CDSs, excluding introns. serve reliable   programme Open Access Publishing. <br /> tool identify novel genes. search new CDSs                                                               Con&#239;&#172;&#8218;ict statement. declared. evaluate biological function, combined <br /> ribosome pro&#239;&#172;?ling, proteomics genome-wide RNAi <br /> data. proteomics data allowed identify            REFERENCES <br /> protein products 20 relatively large candidate <br /> CDSs, experimental set proteomic analysis        1. Moore,M.J. (2005) birth death: complex lives                                                                   eukaryotic mRNAs. Science, 309, 1514&#226;&#8364;&#8220;1518. involved removal small peptides            2. Nagaraj,N., Wisniewski,J.R., Geiger,T., Cox,J., Kircher,M., <br /> impossible verify existence small CDSs.            Kelso,J., Paabo,S. Mann,M. (2011) Deep proteome  important veri&#239;&#172;?cation putative             transcriptome mapping human cancer cell line. Mol. Syst. <br /> CDSs proteomics data &#239;&#172;?nding                  Biol., 7, 548. <br /> &gt;200 putative CDSs appear essential           3. Lu,P., Vogel,C., Wang,R., Yao,X. Marcotte,E.M. (2007) <br />                                                                   Absolute protein expression pro&#239;&#172;?ling estimates relative <br /> parasite &#239;&#172;?tness. Furthermore, RIT-seq data suggest           contributions transcriptional translational regulation. <br /> candidate CDSs 13-fold likely essential            Nat. Biotechnol., 25, 117&#226;&#8364;&#8220;124. viability PF BF differen-     4. Godoy,L.M., Olsen,J.V., Cox,J., Nielsen,M.L., Hubner,N.C., <br /> tiation cells BF PF. &#239;&#172;?ndings point          Frohlich,F., Walther,T.C. Mann,M. (2008) Comprehensive <br />                                                                   mass-spectrometry-based proteome quanti&#239;&#172;?cation haploid <br /> possible role small peptides adaptation tryp-        versus diploid yeast. Nature, 455, 1251&#226;&#8364;&#8220;1254. <br /> anosomes life mammalian host. Generally,         5. McNicoll,F., Drummelsmith,J., Muller,M., Madore,E., Boilard,N., <br /> data suggest existence large explored          Ouellette,M. Papadopoulou,B. (2006) combined <br /> small proteome, characterizations              proteomic transcriptomic approach study                                                                   stage differentiation Leishmania infantum. Proteomics, 6, <br /> small proteome include targeted               3567&#226;&#8364;&#8220;3581. <br /> proteomics analysis.                                           6. Lahav,T., Sivam,D., Volpin,H., Ronen,M., Tsigankov,P., <br />    analysis, focused measuring translational        Green , Holland,N., Kuzyk,M., Borchers,C., Zilberstein,D. <br /> control identi&#239;&#172;?cation new CDSs.                et al. (2011) Multiple levels gene regulation mediate <br /> ribosome pro&#239;&#172;?ling data aid identi&#239;&#172;?ca-          differentiation intracellular pathogen Leishmania. <br />                                                                   FASEB J., 25, 515&#226;&#8364;&#8220;525. <br /> tion bona &#239;&#172; ncRNAs. annotated CDSs            7. Schwanhausser,B., Busse,D., Li,N., Dittmar,G., Schuchhardt,J., <br /> observed RNA transcripts ribosome footprints,             Wolf,J., Chen,W. Selbach,M. (2011) Global quanti&#239;&#172;?cation <br /> suggesting presence ncRNA.  given              mammalian gene expression control. Nature, 473, 337&#226;&#8364;&#8220;342. analysed life cycle stages,             8. Fevre,E.M., Wissmann,B.V., Welburn,S.C. Lutumba,P. (2008) <br />                                                                   burden human African trypanosomiasis. PLoS Negl. Trop. <br /> putative ncRNAs translated stages             Dis., 2, e333. parasite&#226;&#8364;&#8482;s life cycle.  comprehensive search     9. Lozano,R., Naghavi,M., Foreman,K., Lim,S., Shibuya,K., ncRNA include ribosome pro&#239;&#172;?ling analyses            Aboyans,V., Abraham,J., Adair,T., Aggarwal,R., Ahn,S.Y. et al. life cycle stages.                                            (2012) Global regional mortality 235 causes                                                                   death 20 age groups 1990 2010: systematic analysis <br />                                                                   Global Burden Disease Study 2010. Lancet, 380, <br />                                                                   2095&#226;&#8364;&#8220;2128. <br /> ACCESSION NUMBERS                                             10. Martinez-Calvillo,S., Yan,S., Nguyen,D., Fox,M., Stuart,K.                                                                   Myler,P.J. (2003) Transcription Leishmania major Friedlin sequencing data deposited European           chromosome 1 initiates directions single region. <br /> Nucleotide Archive: PRJEB4801.                                    Mol. Cell., 11, 1291&#226;&#8364;&#8220;1299. <br />                                                                                     Nucleic Acids Research, 2014, Vol. 42,  6 3635 <br />  <br />  <br /> 11. Martinez-Calvillo,S., Nguyen,D., Stuart,K. Myler,P.J. (2004)     31. Yoffe,Y., Zuberek,J., Lerer , Lewdorowicz,M., Stepinski,J., <br />     Transcription initiation termination Leishmania major             Altmann,M., Darzynkiewicz,E. Shapira,M. (2006) Binding <br />     chromosome 3. Eukaryot. Cell, 3, 506&#226;&#8364;&#8220;517.                                speci&#239;&#172;?cities potential roles isoforms eukaryotic initiation <br /> 12. Siegel,T.N., Hekstra,D.R., Kemp,L.E., Figueiredo,L.M.,                   factor 4E Leishmania. Eukaryot. Cell, 5, 1969&#226;&#8364;&#8220;1979. <br />     Lowell,J.E., Fenyo,D., Wang,X., Dewell,S. Cross,G M.           32. Gaudenzi,J., Frasch C. Clayton,C. (2005) RNA-binding <br />     (2009) histone variants mark boundaries polycistronic        domain proteins Kinetoplastids: comparative analysis. <br />     transcription units Trypanosoma brucei. Genes Dev., 23,               Eukaryot. Cell, 4, 2106&#226;&#8364;&#8220;2114. <br />     1063&#226;&#8364;&#8220;1076.                                                           33. Dhalia,R., Reis,C.R., Freire,E.R., Rocha,P.O., Katz,R., <br /> 13. Kolev,N.G., Franklin,J.B., Carmi,S., Shi,H., Michaeli,S.             Muniz,J.R., Standart,N. Melo Neto,O.P. (2005) <br />     Tschudi,C. (2010) transcriptome human pathogen                Translation initiation Leishmania major: characterisation     Trypanosoma brucei single-nucleotide resolution.                      multiple eIF4F subunit homologues. Mol. Biochem. Parasitol., <br />     PLoS Pathog., 6, e1001090.                                               140, 23&#226;&#8364;&#8220;41. <br /> 14. LeBowitz,J.H., Smith,H.Q., Rusche,L. Beverley,S.M. (1993)        34. Zinoviev  Shapira,M. (2012) Evolutionary conservation     Coupling poly  site selection trans-splicing                 diversi&#239;&#172;?cation translation initiation apparatus     Leishmania. Genes Dev., 7, 996&#226;&#8364;&#8220;1007.                                     trypanosomatids. Comp. Funct. Genomics, 2012, 813718. <br /> 15. Ullu,E., Matthews,K.R. Tschudi,C. (1993) Temporal order       35. Siegel,T.N., Hekstra,D.R., Wang,X., Dewell,S. Cross,G M. <br />     RNA-processing reactions trypanosomes: rapid trans splicing           (2010) Genome-wide analysis mRNA abundance life- <br />     precedes polyadenylation newly synthesized tubulin transcripts.       cycle stages Trypanosoma brucei identi&#239;&#172;?cation splicing <br />     Mol. Cell. Biol., 13, 720&#226;&#8364;&#8220;725.                                           polyadenylation sites. Nucleic Acids Res., 38, 4946&#226;&#8364;&#8220;4957. <br /> 16. Matthews,K.R., Tschudi,C. Ullu,E. (1994) common                36. Nilsson,D., Gunasekera,K., Mani,J., Osteras,M., Farinelli,L., <br />     pyrimidine-rich motif governs trans-splicing polyadenylation         Baerlocher,L., Roditi  Ochsenreiter,T. (2010) Spliced leader <br />     tubulin polycistronic pre-mRNA trypanosomes. Genes Dev.,           trapping reveals widespread alternative splicing patterns     8, 491&#226;&#8364;&#8220;501.                                                              highly dynamic transcriptome Trypanosoma brucei. PLoS <br /> 17. Wright,J.R., Siegel,T.N. Cross,G M. (2010) Histone H3              Pathog., 6, e1001037. <br />     trimethylated lysine 4 enriched probable transcription      37. Siegel,T.N., Gunasekera,K., Cross,G  Ochsenreiter,T. <br />     start sites Trypanosoma brucei. Mol. Biochem. Parasitol., 136,        (2011) Gene expression Trypanosoma brucei: lessons     434&#226;&#8364;&#8220;450.                                                                 high-throughput RNA sequencing. Trends Parasitol., 27, 434&#226;&#8364;&#8220;441. <br /> 18. Clayton,C. Shapira,M. (2007) Post-transcriptional regulation     38. Jackson,R.J., Hellen,C.U. Pestova,T.V. (2010)     gene expression trypanosomes leishmanias. Mol.                 mechanism eukaryotic translation initiation principles     Biochem. Parasitol., 156, 93&#226;&#8364;&#8220;101.                                        regulation. Nat. Rev. Mol. Cell Biol., 11, 113&#226;&#8364;&#8220;127. <br /> 19. Matthews,K.R. (2011) Controlling coordinating development        39. Kozak,M. (1984) Selection initiation sites eucaryotic <br />     vector-transmitted parasites. Science, 331, 1149&#226;&#8364;&#8220;1153.                ribosomes: effect inserting AUG triplets upstream 20. Cross,G , Klein,R  Linstead,D.J. (1975) Utilization           coding sequence preproinsulin. Nucleic Acids Res., 12, <br />     amino acids Trypanosoma brucei culture: <span id='am-15' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-threonine</span>           3873&#226;&#8364;&#8220;3893. <br />     precursor acetate. Parasitology, 71, 311&#226;&#8364;&#8220;326.                    40. Somers,J., Poyry,T. Willis E. (2013) perspective 21. Brun,R. Schonenberger,M. (1979) Cultivation vitro             mammalian upstream open reading frame function. Int. J. <br />     cloning procyclic culture forms Trypanosoma brucei            Biochem. Cell Biol., 45, 1690&#226;&#8364;&#8220;1700. <br />     semi-de&#239;&#172;?ned medium. Acta Trop., 36, 289&#226;&#8364;&#8220;292.                         41. Siegel,T.N., Tan,K.S. Cross,G M. (2005) Systematic study <br /> 22. Furger , Schurch,N., Kurath,U. Roditi  (1997) Elements           sequence motifs RNA trans splicing Trypanosoma <br />     30 untranslated region procyclin mRNA regulate                 brucei. Mol. Cell. Biol., 25, 9586&#226;&#8364;&#8220;9594. <br />     expression insect forms Trypanosoma brucei                  42. Arava,Y., Wang,Y., Storey,J.D., Liu,C.L., Brown,P.O.     modulating RNA stability translation. Mol. Cell. Biol., 17,          Herschlag,D. (2003) Genome-wide analysis mRNA translation <br />     4372&#226;&#8364;&#8220;4380.                                                               pro&#239;&#172;?les Saccharomyces cerevisiae. Proc. Natl Acad. Sci. USA, <br /> 23. Hehl , Vassella,E., Braun,R. Roditi  (1994) conserved          100, 3889&#226;&#8364;&#8220;3894. <br />     stem-loop structure 30 untranslated region procyclin       43. Capewell,P., Monk,S., Ivens , Macgregor,P., Fenn,K., <br />     mRNAs regulates expression Trypanosoma brucei. Proc. Natl             Walrad,P., Bringaud,F., Smith,T.K. Matthews,K.R. (2013) <br />     Acad. Sci. USA, 91, 370&#226;&#8364;&#8220;374.                                             Regulation trypanosoma brucei total polysomal mRNA <br /> 24. Hotz,H.R., Hartmann,C., Huober,K., Hug,M. Clayton,C.                 development mammalian host. PLoS  8, <br />     (1997) Mechanisms developmental regulation Trypanosoma             e67069. <br />     brucei: polypyrimidine tract 30 -untranslated region   44. Brecht,M. Parsons,M. (1998) Changes polysome pro&#239;&#172;?les <br />     surface protein mRNA affects RNA abundance translation.              accompany trypanosome development. Mol. Biochem. Parasitol., <br />     Nucleic Acids Res., 25, 3017&#226;&#8364;&#8220;3026.                                       97, 189&#226;&#8364;&#8220;198. <br /> 25. Mayho,M., Fenn,K., Craddy,P., Crosthwaite,S. Matthews,K.         45. Ingolia,N.T., Ghaemmaghami,S., Newman,J.R. Weissman,J.S. <br />     (2006) Post-transcriptional control nuclear-encoded cytochrome        (2009) Genome-wide analysis vivo translation     oxidase subunits Trypanosoma brucei: evidence genome-             nucleotide resolution using ribosome pro&#239;&#172;?ling. Science, 324, <br />     wide conservation life-cycle stage-speci&#239;&#172;?c regulatory elements.       218&#226;&#8364;&#8220;223. <br />     Nucleic Acids Res., 34, 5312&#226;&#8364;&#8220;5324.                                   46. Ingolia,N.T., Lareau,L.F. Weissman,J.S. (2011) Ribosome <br /> 26. Walrad,P., Paterou , Acosta-Serrano  Matthews,K.R.               pro&#239;&#172;?ling mouse embryonic stem cells reveals complexity <br />     (2009) Differential trypanosome surface coat regulation             dynamics mammalian proteomes. Cell, 147, 789&#226;&#8364;&#8220;802. <br />     CCCH protein associates procyclin mRNA cis-             47. Oh,E., Becker H., Sandikci , Huber,D., Chaba,R., Gloge,F., <br />     elements. PLoS Pathog., 5, e1000317.                                     Nichols,R.J., Typas , Gross,C , Kramer,G. et al. (2011) <br /> 27. Helm,J.R., Wilson,M.E. Donelson,J.E. (2009) Differential             Selective ribosome pro&#239;&#172;?ling reveals cotranslational chaperone <br />     expression protease gene family African trypanosomes.            action trigger factor vivo. Cell, 147, 1295&#226;&#8364;&#8220;1308. <br />     Mol. Biochem. Parasitol., 163, 8&#226;&#8364;&#8220;18.                                 48. Michel M. Baranov,P.V. (2013) Ribosome pro&#239;&#172;?ling: 28. Horn,D. (2008) Codon usage suggests translational selection         Hi-Def monitor protein synthesis genome-wide scale. <br />     major impact protein expression trypanosomatids.             Wiley Interdiscip. Rev. RNA, 349, 4184&#226;&#8364;&#8220;4188. <br />     BMC Genomics, 9, 2.                                                  49. Ingolia,N.T. (2010) Genome-wide translational pro&#239;&#172;?ling 29. Manful,T., Fadda  Clayton,C. (2011) role 50 -30         ribosome footprinting. Methods Enzymol., 470, 119&#226;&#8364;&#8220;142. <br />     exoribonuclease XRNA transcriptome-wide mRNA                      50. Ingolia,N.T., Brar,G , Rouskin,S., McGeachy M.     degradation. RNA, 17, 2039&#226;&#8364;&#8220;2047.                                         Weissman,J.S. (2012) ribosome pro&#239;&#172;?ling strategy 30. Yoffe,Y., Zuberek,J., Lewdorowicz,M., Zeira , Keasar,C.,               monitoring translation vivo deep sequencing     Orr-Dahan , Jankowska-Anyszka,M., Stepinski,J.,                        ribosome-protected mRNA fragments. Nat. Protoc., 7, 1534&#226;&#8364;&#8220;1550. <br />     Darzynkiewicz,E. Shapira,M. (2004) Cap-binding activity       51. Langmead,B. Salzberg,S.L. (2012) Fast gapped-read <br />     eIF4E homolog Leishmania. RNA, 10, 1764&#226;&#8364;&#8220;1775.                    alignment Bowtie 2. Nat. Methods, 9, 357&#226;&#8364;&#8220;359. <br />  3636 Nucleic Acids Research, 2014, Vol. 42,  6 <br />  <br />  <br /> 52. Aurrecoechea,C., Barreto , Brestelli,J., Brunk,B.P., Cade,S.,            database eukaryotic uORF biology. Nucleic Acids Res.,, 42, <br />     Doherty,R., Fischer,S., Gajria,B., Gao,X., Gingle  et al., (2013)        D60&#226;&#8364;&#8220;D67. <br />     EuPathDB: eukaryotic pathogen database. Nucleic Acids Res.,        72. Calvo,S.E., Pagliarini,D.J. Mootha,V.K. (2009) Upstream <br />     41, D684&#226;&#8364;&#8220;D691.                                                             open reading frames cause widespread reduction protein <br /> 53. Guttman,M., Russell,P., Ingolia,N.T., Weissman,J.S.                    expression polymorphic humans. Proc. Natl Acad. <br />     Lander,E.S. (2013) Ribosome Pro&#239;&#172;?ling Provides Evidence                Sci. USA, 106, 7507&#226;&#8364;&#8220;7512. <br />     Large Noncoding RNAs Encode Proteins. Cell, 154,                73. Hood,H.M., Neafsey,D.E., Galagan,J. Sachs,M.S. (2009) <br />     240&#226;&#8364;&#8220;251.                                                                   Evolutionary roles upstream open reading frames mediating <br /> 54. Alsford,S. Horn,D. (2008) Single-locus targeting constructs            gene regulation fungi. Annu. Rev. Microbiol., 63, 385&#226;&#8364;&#8220;409. <br />     reliable regulated RNAi transgene expression                74. Matsui,M., Yachie,N., Okada,Y., Saito,R. Tomita,M. (2007) <br />     Trypanosoma brucei. Mol. Biochem. Parasitol., 161, 76&#226;&#8364;&#8220;79.                  Bioinformatic analysis post-transcriptional regulation uORF <br /> 55. Anders,S. Huber,W. (2010) Differential expression analysis             human mouse. FEBS Lett., 581, 4184&#226;&#8364;&#8220;4188. <br />     sequence count data. Genome Biol., 11, R106.                       75. Iacono,M., Mignone,F. Pesole,G. (2005) uAUG uORFs <br /> 56. Robinson,M.D., McCarthy,D.J. Smyth,G.K. (2010) edgeR:                human rodent 50 untranslated mRNAs. Gene, 349, 97&#226;&#8364;&#8220;105. <br />     Bioconductor package differential expression analysis           76. Helm,J.R., Wilson,M.E. Donelson,J.E. (2008) Different trans <br />     digital gene expression data. Bioinformatics, 26, 139&#226;&#8364;&#8220;140.                 RNA splicing events bloodstream procyclic Trypanosoma <br /> 57. Butter,F., Bucerius,F., Michel,M., Cicova , Mann,M.                  brucei. Mol. Biochem. Parasitol., 159, 134&#226;&#8364;&#8220;137. <br />     Janzen,C.J. (2013) Comparative proteomics life cycle            77. Camby , Le Mercier,M., Lefranc,F. Kiss,R. (2006) <br />     stages stable isotope-labeled Trypanosoma brucei reveals novel          Galectin-1: small protein major functions. Glycobiology, <br />     components parasite&#226;&#8364;&#8482;s host adaptation machinery. Mol.               16, 137R&#226;&#8364;&#8220;157R. <br />     Cell. Proteomics, 12, 172&#226;&#8364;&#8220;179.                                         78. Fletcher,J.C., Brand,U., Running,M.P., Simon,R. 58. Cox,J., Neuhauser,N., Michalski , Scheltema,R , Olsen,J.V.             Meyerowitz,E.M. (1999) Signaling cell fate decisions     Mann,M. (2011) Andromeda: peptide search engine                      CLAVATA3 Arabidopsis shoot meristems. Science, 283, <br />     integrated MaxQuant environment. J. Proteome Res., 10,            1911&#226;&#8364;&#8220;1914. <br />     1794&#226;&#8364;&#8220;1805.                                                             79. Dinger,M.E., Pang,K.C., Mercer,T.R. Mattick,J.S. (2008) <br /> 59. Cox,J. Mann,M. (2008) MaxQuant enables high peptide                    Differentiating protein-coding noncoding RNA: challenges <br />     identi&#239;&#172;?cation rates, individualized p.p.b.-range mass accuracies           ambiguities. PLoS Comput. Biol., 4, e1000176. <br />     proteome-wide protein quanti&#239;&#172;?cation. Nat. Biotechnol., 26,         80. Morris,J.C., Wang , Drew,M.E. Englund,P.T. (2002) <br />     1367&#226;&#8364;&#8220;1372.                                                                 Glycolysis modulates trypanosome glycoprotein expression 60. Webb,H., Burns,R., Ellis,L., Kimblin,N. Carrington,M.                  revealed RNAi library. EMBO J., 21, 4429&#226;&#8364;&#8220;4438. <br />     (2005) Developmentally regulated instability GPI-PLC            81. Alsford,S., Turner,D.J., Obado,S.O., Sanchez-Flores , <br />     mRNA dependent short-lived protein factor. Nucleic Acids           Glover,L., Berriman,M., Hertz-Fowler,C. Horn,D. (2011) <br />     Res., 33, 1503&#226;&#8364;&#8220;1512.                                                       High-throughput phenotyping using parallel sequencing RNA <br /> 61. Mair,G., Shi,H., Li,H., Djikeng , Aviles,H.O., Bishop,J.R.,              interference targets African trypanosome. Genome Res., 21, <br />     Falcone,F.H., Gavrilescu,C., Montgomery,J.L., Santori,M  et al.          915&#226;&#8364;&#8220;924. <br />     (2000) new twist trypanosome RNA metabolism: cis-splicing         82. Manful,T., Cristodero,M. Clayton,C. (2009) DRBD1     pre-mRNA. RNA, 6, 163&#226;&#8364;&#8220;169.                                              Trypanosoma brucei homologue spliceosome-associated <br /> 62. Berriman,M., Ghedin,E., Hertz-Fowler,C., Blandin,G.,                       protein 49. Mol. Biochem. Parasitol., 166, 186&#226;&#8364;&#8220;189. <br />     Renauld,H., Bartholomeu,D.C., Lennard,N.J., Caler,E.,                  83. Vattem,K.M. Wek,R.C. (2004) Reinitiation involving <br />     Hamlin,N.E., Haas,B. et al. (2005) genome African               upstream ORFs regulates ATF4 mRNA translation     trypanosome Trypanosoma brucei. Science, 309, 416&#226;&#8364;&#8220;422.                     mammalian cells. Proc. Natl Acad. Sci. USA, 101, 11269&#226;&#8364;&#8220;11274. <br /> 63. Bakker,B.M., Michels,P , Opperdoes,F.R. Westerhoff,H.V.          84. Hinnebusch G. (1993) Gene-speci&#239;&#172;?c translational control     (1997) Glycolysis bloodstream form Trypanosoma brucei               yeast GCN4 gene phosphorylation eukaryotic initiation <br />     understood terms kinetics glycolytic enzymes.          factor 2. Mol. Microbiol., 10, 215&#226;&#8364;&#8220;223. <br />     J. Biol. Chem., 272, 3207&#226;&#8364;&#8220;3215.                                        85. Dever,T.E., Feng,L., Wek,R.C., Cigan M., Donahue,T.F. 64. Dana  Tuller,T. (2012) Determinants translation                   Hinnebusch G. (1992) Phosphorylation initiation factor <br />     elongation speed ribosomal pro&#239;&#172;?ling biases mouse                    2 alpha protein kinase GCN2 mediates gene-speci&#239;&#172;?c <br />     embryonic stem cells. PLoS Comput. Biol., 8, e1002755.                     translational control GCN4 yeast. Cell, 68, 585&#226;&#8364;&#8220;596. <br /> 65. Saas,J., Ziegelbauer,K., von Haeseler , Fast,B. Boshart,M.       86. Kong,J. Lasko,P. (2012) Trans <br /> </body></html>