<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                  Letters <br />  <br />                                                                                                                     pubs.acs.org/acschemicalbiology <br />  <br />  <br />  <br />  <br /> Release Factor Nonessential Escherichia coli <br /> David B. F. Johnson,&#226;&#8364;&#160;,&#226;&#352;&#165; Chong Wang,&#226;&#8364;&#160;,&#226;&#352;&#165; Jianfeng Xu,&#226;&#8364;&#160; Matthew D. Schultz,&#226;&#8364;&#161;,&#194;&#167; Robert J. Schmitz,&#226;&#8364;&#161; <br /> Joseph R. Ecker,&#226;&#8364;&#161;,&#226;&#710;&#165; Lei Wang*,&#226;&#8364;&#160; <br /> &#226;&#8364;&#160; <br />  Jack H. Skirball Center <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> Biology Proteomics &#226;&#8364;&#161;Genomic Analysis Laboratory, Salk Institute  Biological Studies, 10010 N. Torrey Pines Road, La Jolla, California 92037, United States <br /> &#194;&#167; <br />  Bioinformatics Program, University California San Diego, La Jolla, California 92093, United States <br /> &#226;&#710;&#165; <br />   Howard Hughes Medical Institute, 4000 Jones Bridge Road, Chevy Chase, Maryland 20815, United States <br />     * <br />     S Supporting Information <br />  <br />  <br />  <br />     ABSTRACT: Recoding stop codon <span id='am-30' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span>     a&#239;&#172;&#8364;ord orthogonal genetic systems biosynthesizing new <br />     protein organism properties. reassignment     stop codons extant organisms, model <br />     organism lacking investigate reassignment process     direct code evolution. Complete reassignment stop <br />     codon precluded release factors (RFs), recognize <br />     stop codons terminate translation. discovered     RF1 unconditionally knocked various <br />     Escherichia coli stains, demonstrating reportedly <br />     essential RF1 generally dispensable E. coli species. <br />     apparent essentiality RF1 caused     ine&#239;&#172;&#402;ciency mutant RF2 terminating UAA stop codons; wild type RF2 su&#239;&#172;&#402;cient RF1 knockout. RF1- <br />     knockout strains autonomous unambiguously reassigned UAG encode natural unnatural amino acids (Uaas)     multiple sites, a&#239;&#172;&#8364;ording previously unavailable model studying code evolution unique host exploiting Uaas     evolve new biological functions. <br />  <br />  <br />      lthough canonical genetic code preserved       organisms, small deviations discovered, <br /> including reassignment sense codons amino <br />                                                                            ambiguity. attractive route reassign UAG <br />                                                                            stop codon Uaa bacteria. Orthogonal tRNA/synthetase <br />                                                                            pairs engineered incorporate Uaas proteins acid reassignment stop sense                response UAG,8&#226;&#710;&#8217;12 presence RF1 makes codons.1,2 Stop codons decoded class release factors              meaning UAG ambiguous, stop signal Uaa <br /> (RFs).3 eukaryotes archaea use single RF                  simultaneously. RF1 competition limits incorporation recognize stop codons,4,5 bacteria use  RF1               Uaas single UAG site low e&#239;&#172;&#402;ciency; addition speci&#239;&#172;?c UAA/UAG, RF2 speci&#239;&#172;?c UAA/UGA.6                   second UAG codon decreases protein yields unknown class RFs bacteria               precipitously.13 Uaas incorporated single class RF su&#239;&#172;&#402;cient organisms             site protein,14,15 low UAG-encoding <br /> domains. process stop codon reassignment                   e&#239;&#172;&#402;ciency prevents e&#239;&#172;&#8364;ective use Uaas multiple sites potential association RF evolution unclear.                  explore novel protein organismal properties Natural code evolution occurs millions years. Extant               directed evolution. addition, ambiguity UAG <br /> organisms harboring altered genetic codes end-point             codon hinder eventual &#239;&#172;?xation altered genetic code evolution. records initial causes          code organism, protein products truncated responses altered genetic codes;                     UAG sites interfere normal protein functions adaptations details eventual &#239;&#172;?xation completely              detrimental e&#239;&#172;&#8364;ects host cell, preventing advanta- <br /> unknown. enable depth investigation code change               geous coding inherited selected directed concurrent cellular adaptations real time, necessary          evolution. exclude Uaa incorporation legitimate generate model organism able undergo                                                                            termination sites speci&#239;&#172;?ed UAG, endogenous UAG codons <br /> evolutionary processes laboratory. <br />                                                                            E. coli genome replaced synonymous UAA <br />    Synthetically recoding genome a&#239;&#172;&#8364;ord new properties <br />                                                                            stop codon genome engineering.16  organism encoding unnatural amino acids <br /> (Uaas) preventing cross-contamination wild type life              Received: 9, 2012 <br /> forms.7 successful genome recoding, target codon              Accepted: June 4, 2012 reassigned new meaning high e&#239;&#172;&#402;ciency                      Published: June 5, 2012 <br />  <br />                              &#194;&#169; 2012 American Chemical Society       1337                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                 Letters <br />  <br /> complete reassignment UAG sense codon, necessary critical step knock RF1 E. coli genome. <br />    eukaryotes ciliates green algae, reassignment stop codon sense codon accompanied convergent changes eRF1.2 instance, eRF1 Tetrahymena restricts recognition UGA, UAA/UAG reassigned Gln; eRF1 Euplotes recognizes UAA/UAG stop codons, UGA used encode <br /> Cys.17,18 bacteria, Mycoplasma species lost RF2 <br /> gene, UGA codon encodes Trp instead.19  <br /> Mycoplasma obligatory pathogens highly reduced <br /> genomes. <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span>, free-living bacterium lacking RF1 RF2.19 E. coli, RF1 reported essential,20 conditionally lethal knockouts described.21,22 Recently managed knock RF1 special E. coli strain reduced genome mutated RF2 gene. 13 discovered dispensability RF1 general property wild type E. <br /> coli, arguing paradigm RF1 essential. revealed underlying mechanism preventing RF1 <br /> knockout, generated autonomous RF1 deletion strains <br /> valuable studying code evolution genome recoding. <br />    attempted replace RF1-encoding gene, prfA, chloramphenicol acetyltransferase gene variety E. coli <br /> strains (Figure 1) using established &#206;&#187; red recombinase- <br /> based homologous recombination method.23 E. coli K-12 B progenitors E. coli strains derived.24 Reported attempts knockout RF1 used E. coli K-12 strains unsuccessful.20,25 initially <br /> attempted delete RF1 common K-12 strains                    Figure 1. RF1 knocked E. coli strains containing wild <br /> MG1655, DH10&#206;&#178;, HT115. Knockout RF1 assayed                       type RF2.   RF1-encoding gene prfA replaced genomic PCR amplifying prfA locus (Figure 1A).                       chloramphenicol acetyltransferase (cat) gene homologous <br /> Consistent previous reports, knock RF1                recombination using phage &#206;&#187; red recombinase expressed K-12 strains (Figure 1B).                                   plasmid pKD46.23 Chloramphenicol-resistant colonies screened <br />                                                                             using PCR cat replacement con&#239;&#172;?rmed sequencing.    attempted RF1 knockout K-12                   agarose gels PCR results parental knockout strains strains contain alterations related translational                   labeled. (B) list E. coli strains tested RF1 knockout. K-12 termination. BP5&#206;&#177; strain harbors glutaminyl amber                     B strains widely used E. coli subjects suppressor tRNA, making UAG coding ambiguous                        classical experiments. genomic di&#239;&#172;&#8364;erences K-12 stop signal Gln. strain test stop codon           B strains,24 A246T RF2 peculiar mutation K-12 strains <br /> ambiguity factor prime RF1 removal. MDS42 strain                reduces release activity RF2 stop codon UAA. nearly 700 nonessential genes deleted26 used                 mutation reverted wild type Ala DH10&#206;&#178;f. glnV44 gene <br /> assess reduced genome size termination load                  encodes glutaminyl amber suppressor tRNA. RF1 successfully <br />                                                                             knocked B strains DH10&#206;&#178;f strain. <br /> assist RF1 removal.  RF1 knockouts generated strains  suggesting amber <br /> suppression minimal genome insu&#239;&#172;&#402;cient RF1                     results indicate RF1 essential E. coli, contrast removal K-12 derivatives.                                                previous conclusions.20,25 <br />    K-12 strains contain peculiar A246T mutation                   genomic sequencing performed RF1 knockout <br /> RF2-encoding gene prf B, lowering release activity RF2               strains JX1.0, CW1.0, CW2.0 compared UAA 5-fold.27,28 UAA codon accounts                         respective parental strains.30,31 RF1 deletion veri&#239;&#172;?ed termination &#226;&#710;&#188;64% E. coli genes.29 reasoned                cases. CW1.0 CW2.0, mutations A246T mutation severely impair ability RF2                  genome. JX1.0, seven additional <br /> recognize UAA stop codons removal RF1                  single <span id='am-22' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span>nucleotide</span> polymorphisms (SNPs) (Support- <br /> prevent RF1 knockout. tested common                  ing Table S1). SNPs silent mutations E. coli B strains B strains encode wild type RF2               genes phage origin. SNPs correspond known <br /> contains Ala246.28  RF1 knockout successful                 mutations complement RF1 de&#239;&#172;?ciency.32&#226;&#710;&#8217;35 B strains tested; REL606, BL21, BL21(DE3)                results indicate RF1 knocked parental E. <br /> used generate knockout strains CW1.0, CW2.0,                     coli strains incurring compensatory mutations. <br /> JX1.0, respectively (Figure 1A). addition, determine              happen UAG site protein <br /> A246T mutation RF2 prevents RF1 removal K-12 strains,                 translation absence RF1? mutated reverted A246T mutation Ala K-12 strain                    tyrosine codons TAG enhanced green <br /> DH10&#206;&#178; generate DH10&#206;&#178;f strain. strain                       &#239;&#172;&#8218;uorescent protein (EGFP) gene36 generate 1-TAG <br /> permitted direct knockout RF1 (Figure 1B).                     3-TAG EGFP reporters, respectively, tested                                                                      1338                       dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                  Letters <br />  <br /> expression BL21(DE3) RF1 knockout strain JX1.0                      anticodons tRNAs single base <br /> (Figure 2A). expected, BL21(DE3) cells expressing                    mispairing UAG, tRNAs weakly misread                                                                                UAG codon.37 Misreading UAG tRNAs                                                                                obvious presence RF1 BL21(DE3)                                                                                marked JX1.0 RF1 removal. <br />                                                                                   unconditional knockout RF1 opens possibility <br />                                                                                reassigning meaning UAG codon completely,                                                                                UAG codon does ambiguously encode stop signal <br />                                                                                amino acid simultaneously. attempted recruit <br />                                                                                UAG amino acid JX1.0, mimic possible code <br />                                                                                evolution pathway implicated RF constriction stop codon <br />                                                                                recognition Tetrahymena, Euplotes, eukaryotes.17,18 <br />                                                                                orthogonal tRNA/synthetase pair, tRNATyr     CUA/LW1RS <br />                                                                                pair, introduced JX1.0. pair does crosstalk <br />                                                                                endogenous E. coli tRNA/synthetase pairs functionally <br />                                                                                couples E. coli&#226;&#8364;&#8482;s protein translational machinery.8,38                                                                                tRNATyr CUA decodes UAG codon speci&#239;&#172;?cally                                                                                anticodon CUA, LW1RS engineered charge                                                                                                                                           39 <br />                                                                                tRNATyr CUA Uaa p-acetyl-<span id='am-17' about='obo:IMR_0200274' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00079" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0200274" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="L-Phenylalanine" datatype="xsd:string"></span>L-phenylalanine</span> (pActF).                                                                                   EGFP gene containing 1-, 2-, 3-, 10-TAG codons  <br />                                                                                expressed tRNATyr CUA/LW1RS single plasmid pAIO- <br />                                                                                EGFP(n-TAG) (Figure 3A). <br />                                                                                   EGFP expression assayed Western blotting (Figure <br />                                                                                3B) cell &#239;&#172;&#8218;uorescence (Figure 3C). pActF                                                                                added growth media, BL21(DE3) cells expressed small <br />                                                                                length EGFP 1-TAG reporter  <br />                                                                                suggesting tRNATyr  CUA/LW1RS pair incorporates                                                                                natural amino acid low e&#239;&#172;&#402;ciency absence                                                                                cognate pActF. length EGFP detected 2-, 3-, <br />                                                                                10-TAG reporters. contrast, JX1.0 expressed length <br /> Figure 2. UAG codon RF1 knockout strain JX1.0 suppressed             EGFP 1-, 2-, 3-, 10-TAG mutants, endogenous tRNAs mispairing anticodons.   EGFP gene               e&#239;&#172;&#402;ciency decreased number UAG codons. TAG codon permissive sites driven arabinose             puri&#239;&#172;?ed EGFP protein JX1.0 expressing pAIO- <br /> promoter plasmid pBAD-EGFP(n-TAG) test EGFP                          EGFP(1-TAG) absence pActF (Figure 3D) expression. (B) cell &#239;&#172;&#8218;uorescence intensity measured                    analyzed mass spectrometry. Consistently, Tyr, Gln, &#239;&#172;&#8218;uorometry BL21(DE3) JX1.0 cells transformed                      Trp UAG site observed Figure 2C, <br /> pBAD-EGFP(1-TAG) pBAD-EGFP(3-TAG). Samples                             con&#239;&#172;?rming misreading endogenous tRNAs JX1.0. <br /> normalized cell numbers. (C) Fourier transform mass spectro- <br />                                                                                   pActF supplied growth medium, BL21- <br /> metric analysis EGFP protein puri&#239;&#172;?ed JX1.0 expressing pBAD- <br /> EGFP(1-TAG). spectrum shows intact precursor ions (charge              (DE3) cells showed e&#239;&#172;&#402;cient expression 1-TAG EGFP <br /> deconvoluted) tryptic fragment EGFP (HNIEDGSVQ-                     mutant, EGFP expression decreased precipitously LADHXQQNTPIGDGPVLLPDNHYLSTQSALSK, X representing                               addition UAG codon competition RF1 UAG encoded site). Monoisotopic masses measured high                  termination. use 3 UAG codons EGFP virtually <br /> accuracy indicate Gln, Tyr, Trp (expected [M + H]+ =                  abolished protein expression, protein detected <br /> 4437.17, 4472.17, 4495.19 Da; measured 4437.16, 4472.16,               10-TAG mutant. stark contrast, RF1-deletion <br /> 4494.16 Da, respectively) incorporated UAG site.          strain JX1.0 showed high expression EGFP mutants amino acids detected UAG site.                                     presence pActF, indicated Western (Figure 3B)                                                                                cell &#239;&#172;&#8218;uorescence (Figure 3C). EGFP proteins puri&#239;&#172;?ed 1- 3-TAG reporter showed EGFP &#239;&#172;&#8218;uorescence,                           yields 8.5 (&#194;&#177;0.4), 7.1 (&#194;&#177;0.4), 9.7 (&#194;&#177;0.3), 1.2 <br /> indicating RF1 terminates EGFP translation UAG                         (&#194;&#177;0.1) mg/L 1-, 2-, 3-, 10-TAG EGFP samples, <br /> normally. Surprisingly, JX1.0 showed high level                           respectively (Figure 3D). decrease &#239;&#172;&#8218;uorescence 1-TAG EGFP mutant small                         incorporation e&#239;&#172;&#402;ciency UAG codon increased <br /> reproducible level &#239;&#172;&#8218;uorescence 3-TAG EGFP                          1 2 3, indicating UAG changed sense <br /> mutant (Figure 2B). data suggest endogenous                    codon JX1.0. drop o&#239;&#172;&#8364; yield 10-TAG EGFP <br /> tRNAs suppress UAG codon JX1.0.                                     sample likely 10 pActF interferes EGFP folding <br />    reveal identity tRNAs, puri&#239;&#172;?ed EGFP                      stability, corroborated lack protein expressed 1-TAG reporter JX1.0                         &#239;&#172;&#8218;uorescence 10-pActF EGFP (Figure 3C). <br /> identi&#239;&#172;?ed amino acid incorporated UAG site using                       JX1.0 RF1 terminate protein translation <br /> mass spectrometry (Figure 2C). Tyr, Gln, Trp                    UAG codon, small protein detected UAG site Fourier transform mass spectrometric                   Western blot detected mass <br /> analysis tryptic fragments EGFP. ensure              spectrometry truncated UAG site suppressor mutation endogenous tRNAs,                        (Figure 3B). think decoding UAG codon chromosomal tRNATyr, tRNAGln, tRNATrp rese-                           Uaa pActF e&#239;&#172;&#402;cient decoding quenced, con&#239;&#172;?rmed wild type JX1.0.                   canonical sense codons natural amino acids.  pActF- <br />                                                                         1339                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                 Letters <br />  <br />  <br />  <br />  <br /> Figure 3. UAG-decoding orthogonal tRNA/synthetase pair reassigns UAG codon amino acid JX1.0.   plasmid pAIO-EGFP(n- <br /> TAG) encodes orthogonal tRNATyr    CUA/LW1RS pair EGFP reporter various numbers TAG codons. Structure pActF shown right. (B) Western blot analysis EGFP expression pAIO-EGFP(n-TAG) BL21(DE3) JX1.0 cells presence absence pActF. penta antibody used detect Hisx6 tag fused N-terminus EGFP. (C) cell &#239;&#172;&#8218;uorescence intensity EGFP expressed pAIO-EGFP(n-TAG) BL21(DE3) JX1.0 cells presence absence pActF. Samples normalized cell numbers lane. <br /> Measurement performed independent batches cells, error bars represent SEM. (D) SDS-PAGE analysis EGFP proteins <br /> puri&#239;&#172;?ed JX1.0 cells expressing pAIO-EGFP(n-TAG) absence presence pActF. (E) Fourier transform mass spectrometric analysis EGFP expressed pAIO-EGFP(3-TAG) JX1.0 pActF growth medium. spectra intact precursor ions (charge <br /> deconvoluted) tryptic fragments EGFP. Monoisotopic masses measured high accuracy indicate pActF incorporated UAG sites. <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>Peptide</span> sequences shown, X represents UAG encoded site.  site 39, expected [M + H]+ =1529.65 Da, measured <br /> 1529.66 Da. Middle: site 151, expected [M + H]+ =1999.89 Da, measured 1999.90 Da; 2015.89 peak corresponds Met oxidation pActF-containing peptide.  site 182, expected [M + H]+ =4498.18 Da, measured 4498.14 Da. peaks corresponding amino <br /> acids UAG positions detected. signal noise ratio &gt;1000, translating &#239;&#172;?delity pActF incorporation &gt;99.9%. Tandem <br /> MS spectra peptides shown Supporting Figure S1, unambiguously con&#239;&#172;?rm peptide sequences pActF position X. <br />  <br /> speci&#239;&#172;?c LW1RS active wild type synthetases39,40             TAG mutant. Nonetheless, underlying mechanism generate fewer aminoacylated orthogonal tRNAs.                  generating truncated products absence RF1 Second, orthogonal tRNATyr  CUA evolutionarily              intriguing warrants studies. <br /> optimized UAG decoding, natural tRNAs                      identify amino acid incorporated response UAG <br /> posttranscriptionally modi&#239;&#172;?ed evolution e&#239;&#172;&#402;cient                  JX1.0, puri&#239;&#172;?ed EGFP expressed JX1.0 using pAIO- <br /> codon recognition.41  natural aminoacyl-tRNAs                EGFP(3-TAG) presence pActF analyzed &#239;&#172;?ne-tuned binding elongation factor Tu               mass spectrometry (Figure 3E). monoisotopic masses ribosome achieve e&#239;&#172;&#402;cient translation,42,43                    tryptic peptides clearly showed pActF incorporated <br /> pActF-charged orthogonal tRNA optimized                     UAG sites. peaks corresponding binding  factors lead e&#239;&#172;&#402;cient             incorporation amino acids UAG sites decoding UAG pActF ribosome drop-o&#239;&#172;&#8364;                         detected. precursor ions peptides containing translation, resulting truncated protein products.               UAG sites individually fragmented <span id='am-11' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> trap mass <br /> e&#239;&#172;&#402;cient decoding UAG pActF accumulative                  spectrometer. fragment ion masses unambiguously <br /> e&#239;&#172;&#8364;ect account decrease protein yield 10-               assigned, con&#239;&#172;?rming pActF incorporated UAG <br />                                                                       1340                      dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                               Letters <br />  <br /> sites (Supporting Figure S1). results indicate                   variety genes removed. studying various <br /> misreading UAG JX1.0 endogenous tRNAs                          nonengineered E. coli strains, discovered wild <br /> outcompeted tRNATyr    CUA/LW1RS pair, speci&#239;&#172;?cally             type RF2 su&#239;&#172;&#402;cient necessary RF1 removal decodes UAG pActF.                                                       incurring compensatory mutations. mutations    evaluate initial response E. coli RF1 deletion          arti&#239;&#172;?cially introduced strains, results clearly <br /> subsequent UAG reassignment, assessed health JX1.0                demonstrate RF1 nonessential wild type E. coli. using growth assay. JX1.0 healthy, cloneable, stable           successful RF1 knockout multiple strains indicates RF1 <br /> culture; changes phenotype genotype observed                  dispensability general property E. coli peculiar growing 200 generations. Compared parental                    requirements. <br /> BL21(DE3), JX1.0 showed slower doubling rate (Figure 4).                     Unconditional RF1 removal explained usage                                                                             UAG E. coli genome. UAG used stop codon E. <br />                                                                             coli, terminating &#226;&#710;&#188;7% total genes.29 302 <br />                                                                             essential genes E. coli,44 7 genes ended                                                                             UAG codon, 7 genes UGA UAA <br />                                                                             stop codon downstream short distances (Supporting Figure <br />                                                                             S2). UAG readthrough, second di&#239;&#172;&#8364;erent stop <br />                                                                             codon ensure termination proteins                                                                             small number amino acids appended. additional amino <br />                                                                             acids completely disable function proteins, <br />                                                                             allowing E. coli survive observed study.                                                                             UAG-ending essential genes presence double stop <br />                                                                             codons account nonessentiality RF1. <br />                                                                             Consistently, wild type RF2                                                                             weakened A246T mutant permitted RF1 knockout, suggesting <br />                                                                             e&#239;&#172;&#402;cient termination dominant UAA UGA stop <br /> Figure 4. RF1 knockout UAG reassignment slow                   codons required E. coli survival. <br /> doubling JX1.0. growth curves RF1 knockout strain               Knockout RF1 provides novel insights evolution JX1.0 represented closed symbols parental              RFs genetic code. major distinction bacteria BL21(DE3) strain open symbols. Squares represent cells  <br /> circles represent cells transformed pAIO plasmid,                                                                             eukaryotes archaea protein translation bacteria use <br /> expresses orthogonal tRNACUA   Tyr <br />                                        /LW1RS pair; triangles           di&#239;&#172;&#8364;erent RFs recognize stop codons, represent cells transformed pAIO plasmid grown          single RF su&#239;&#172;&#402;cient organisms domains. presence pActF. Error bars represent s.e.m., n = 3.                  di&#239;&#172;&#8364;erence served evidence evolution RF                                                                             translation termination bacteria eukaryotes/ doubling time JX1.0 91 min compared 26 min                   archaea nonconserved.45,46 results BL21(DE3). di&#239;&#172;&#8364;erence suggests RF1 knockout                    autonomous E. coli strain single class RF creates burdens cells lack proper termination             generated. viability bacterium blurs apparent relatively weak missuppression UAGs near-cognate                  vs RF distinction prokaryotes eukar- <br /> tRNAs, e&#239;&#172;&#8364;ect lethal perceived                      yotes/archaea, suggesting RF evolution Introduction pAIO plasmid expressing orthogonal                  domains life similar current di&#239;&#172;&#8364;erences <br /> tRNATyr <br />       CUA/LW1RS pair absence pActF increased                  suggest. addition, certain eukaryotes doubling JX1.0 BL21(DE3) 135 41 min,                          reassign stop codon sense codon restrict respectively, possibly general toxicity expressing              recognition eRF1 rest stop codons.2,17,18 unacylated tRNAs. addition pActF growth                       unclear stop codon reassignment caused eRF1 <br /> medium expression tRNATyr     CUA/LW1RS                restriction vice versa code evolution. increased doubling time 253 min JX1.0,                   bacteria, addition eukaryotes nature, BL21(DE3) a&#239;&#172;&#8364;ected. dramatic doubling                   undergo stop codon reassignment accompanied <br /> reduction JX1.0 likely e&#239;&#172;&#402;cient                          RF changes.  results demonstrate RF <br /> incorporation pActF UAG positions                      changes synthetically drive reassignment stop <br /> proteome, re&#239;&#172;&#8218;ecting pressure generated UAG reassign-                 codon amino acid, providing experimental evidence ment sense codon. contrast, BL21(DE3) expresses RF1,                hypothetic evolution pathway. competes UAG positions termination,                       autonomous RF1 knockout bacterium enable new <br /> mitigating pressure.                                                   research understanding code evolution. major challenge <br />    RF1 nonessential E. coli species,           studying evolution genetic code contrast previous conclusion RF1 essential.            questions reach direct experimentation.47 reverting Thr246 Ala removing autoregulation                  organisms exist containing primitive intermediate genetic <br /> RF2 expression, recently knocked RF1                         code comparison, natural code evolution takes millions <br /> engineered E. coli strain MDS42 reduced genome.13                years. RF1 knockout strain reported a&#239;&#172;&#8364;ord &#226;&#710;&#188;700 nonessential genes deleted MDS42                    previously unavailable model organism study multiple mutations introduced RF2 gene,                  intractable questions codon reassignment process real <br /> di&#239;&#172;&#402;cult conclude factors speci&#239;&#172;?cally contribute RF1              time. RF1 knockout introduction knockout RF1 dispensability general property               orthogonal tRNA/synthetase pair completely reassigned E. coli.  MDS42 wild type E. coli              UAG stop signal amino acid, setting initial stage <br /> suitable investigation code evolution adaptation             bacterium adopt altered genetic code. stage <br />                                                                      1341                     dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                       Letters <br />  buried recess evolution experimentally                     electroporated DH10&#206;&#178; harboring pKD46 plasmid. KanR <br /> inaccessible. Using RF1 knockout strain                         clones screened using PCR sequence veri&#239;&#172;?ed mutation <br /> able address questions altered code                    position 246 Ala. <br /> eventually &#239;&#172;?xed, long process                         Plasmid Construction. plasmids assembled standard <br />                                                                                   cloning methods con&#239;&#172;?rmed DNA sequencing. pAIO plasmids <br /> physiological changes necessary adaptations.                         containing EGFP gene di&#239;&#172;&#8364;erent TAG codons synthesized <br /> Answers questions help understand                      follows: EGFP cassettes N-terminal Hisx6 tag TAG fundamentals code evolution provide rare                          codons various positions created using overlapping PCRs. empirical data guide code optimization speci&#239;&#172;?c synthetic                   following sites used: Y182 1-TAG; Y39 Y182 2-TAG; <br /> purposes. addition, hypothesized genetic code                   Y39, Y182, Y151 3-TAG; Y39, K101, D102, E132, D133, <br /> started set primitive amino acids                  K140, E172, D173, D190, V193 10-TAG. cassettes added total 20 reached.48,49 remains mysterious                  &#239;&#172;?rst cloned pBP-Blunt (Biopioneer, San Diego, CA) addition new amino acids repertoire                         digested ligated pBK-AIO vectors containing orthogonal <br />                                                                                                           39 <br />                                                                                   tRNATyrCUA LW1RS       using Spe BglII restriction sites. pBAD <br /> a&#239;&#172;&#8364;ords evolutionary advantage drive code expansion.                       vectors containing 1- 3-TAG EGFP genes constructed challenging fundamental question investigated                          inserting EGFP cassettes pBAD vector (Invitrogen <br /> experimentally strains reported                                         life technologies) using Nco Hind III restriction sites. <br />    &#239;&#172;?nding guide genome recoding reassign stop                             Cell Fluorescence. cell &#239;&#172;&#8218;uorescence intensity deter- <br /> codons successfully. promising generate E. coli                       mined using FluoroLog-3 (Horiba Jobin Yvon). E. coli colonies genome engineering endogenous UAG                                picked grown 2xYT medium 16 h Uaas. <br /> codons replaced synonymous UAA stop codon.16                             Cells washed twice PBS bu&#239;&#172;&#8364;er diluted PBS  reach &#239;&#172;?nal goal setting aside UAG                          OD600 0.1. emission spectrum EGFP recorded 503 <br /> codon encode Uaa unambiguously, RF1 removed                          560 nm using excitation wavelength 488 nm. &#239;&#172;&#8218;uorescence <br />                                                                                   intensity sample compared using intensity 511 host strain. MG1655, K-12 derivative strain,                       nm emission peak. used current genome engineering e&#239;&#172;&#8364;ort,16                           Growth Assay. colony picked E. coli strain permit RF1 knockout basis &#239;&#172;?ndings                     grown overnight 2xYT medium appropriate antibiotics. results provide solution impasse: reversion                 Cells normalized OD600 1 diluted 1:50 fresh 2xYT <br /> Thr246 Ala RF2 gene enable RF1 knockout                          medium antibiotics 1 mM Uaa  applicable). clean recoding, conveniently applied                      BL21(DE3) strains, OD600 measured 30 min 10 h. <br /> genome engineering stage &#239;&#172;?nal RF1 gene removal.                        JX1.0 strains, OD600 measured 60 min 48 h. <br />    autonomous RF1 knockout bacterium a&#239;&#172;&#8364;ord unique                       Doubling times calculated exponential growth phase                                                                                   strain. <br />  <br />                                                                                   &#226;&#8211;&#160; <br /> host synthesizing evolving new protein functions biological properties Uaa exploitation. date,                                ASSOCIATED CONTENT <br /> directed laboratory evolution new biosynthesis ability using <br /> genetically encoded Uaas feasible                         * <br />                                                                                   S Supporting Information <br />  <br /> demanding requirements: simultaneous Uaa incorporation                         Supporting methods genome sequencing, Western analyses, <br /> multiple sites autonomous host. Multisite Uaa                              protein puri&#239;&#172;?cation, mass spectrometry, supporting &#239;&#172;?gures <br /> incorporation enables synergy maximizes exploration                       table. material available free charge                                                                                   Internet http://pubs.acs.org. <br />  <br />                                                                                   &#226;&#8211;&#160; protein sequence space, self-sustaining host ability necessary experimental evolution. RF1 <br /> deletion strains generated ful&#239;&#172;?ll requirements                             AUTHOR INFORMATION open &#239;&#172;?eld Uaa-based directed                              Corresponding Author <br /> evolution. variety new functionalities                          *E-mail: lwang@salk.edu. <br /> introduced proteins Uaas, organisms containing                       Author Contributions <br />                                                                                   &#226;&#352;&#165; expanded genetic repertoire greater potential novel                       authors contributed equally work. <br /> biosynthesis directed evolution, biorenewable                         Notes <br /> production chemicals fuels.                                                authors declare competing &#239;&#172;?nancial  <br />  <br /> &#226;&#8211;&#160;   METHODS <br />    Strain Construction. Knockout prfA gene attempted <br />                                                                                   &#226;&#8211;&#160;    ACKNOWLEDGMENTS <br />                                                                                   thank W. Fischer mass spectrometric analyses. using chloramphenicol acetyltransferase (cat) cassette established          work supported,  Howard Hughes Medical <br /> procedures.23 Brie&#239;&#172;&#8218;y, 51 nucleotide overhangs homologous                   Institute grant Gordon Betty Moore <br /> regions immediately 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; prfA appended cat gene.              foundation J.R.E. L.W. acknowledges support microgram cassette electroporated various strains            Searle Scholar Program (06-L-119), Beckman Young Inves- <br /> harboring pKD46 plasmid, expresses phage &#206;&#187; red                      tigator Program, March Dimes Foundation (#5-FY08-110), <br /> recombinase. Chloramphenicol-resistant clones screened                                                                                   California Institute Regenerative Medicine (RN1-00577-1) <br /> knockout genomic PCR using primers 5&#226;&#8364;&#178;-GGA TAA CGA ACG <br />                                                                                   National Institutes Health (1DP2OD004744-01). <br />  <br />                                                                                   &#226;&#8211;&#160; <br /> CCT GAA TA-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- TCC AGC AGG ATT TCA GCA TC-3&#226;&#8364;&#178;. <br /> Positive clones veri&#239;&#172;?ed DNA sequencing genomic <br /> sequencing.                                                                             REFERENCES <br />    DH10&#206;&#178;f constructed DH10&#206;&#178; follows revert                      (1) Osawa, S., Jukes, T., Watanabe, K., Muto,  (1992) Recent <br /> Thr246 prf B Ala. knockin cassette &#239;&#172;?rst generated                     evidence evolution genetic code. Microbiol. Rev. 56, 229&#226;&#710;&#8217; <br /> containing prf B gene BL21(DE3) transcriptionally coupled                264. kanamycin resistant (KanR) cassette. KanR cassette                    (2) Knight, R. D., Freeland, S. J., Landweber, L. F. (2001) <br /> &#239;&#172;&#8218;anked 3&#226;&#8364;&#178; end 51-nucleotide region homologous 3&#226;&#8364;&#178;               Rewiring keyboard: evolvability genetic code. Nat. Rev. <br /> end endogenous prf B gene. microgram cassette              Genet. 2, 49&#226;&#710;&#8217;58. <br />  <br />                                                                            1342                       dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                                           Letters <br />  <br />  (3) Youngman, E. M., McDonald, M. E., Green, R. (2008)                            (22) Mukai, T., Hayashi, , Iraha, F., Sato, , Ohtake, K., Yokoyama, <br /> Peptide release ribosome: mechanism implications                      S., Sakamoto, K. (2010) Codon reassignment Escherichia <br /> translational control. Annu. Rev. Microbiol. 62, 353&#226;&#710;&#8217;373.                            coli genetic code. Nucleic Acids Res. 38, 8188&#226;&#710;&#8217;8195. <br />  (4) Frolova, L., Le Go&#239;&#172;&#8364;, X., Rasmussen, H. H., Cheperegin, S.,                        (23) Datsenko, K., Wanner, B. (2000) step inactivation Drugeon, G., Kress, M., Arman, , Haenni,  L., Celis, J. E., Philippe,            chromosomal genes Escherichia coli K-12 using PCR products. Proc. <br /> M., Justesen, J., Kisselev, L. (1994) highly conserved eukaryotic              Natl. Acad. Sci. U.S  97, 6640&#226;&#710;&#8217;6645. <br /> protein family possessing properties polypeptide chain release                      (24) Studier, F. W., Daegelen, P., Lenski, R. E., Maslov, S., Kim, <br /> factor. Nature 372, 701&#226;&#710;&#8217;703.                                                         J. F. (2009) Understanding di&#239;&#172;&#8364;erences genome sequences <br />  (5) Dontsova, M., Frolova, L., Vassilieva, J., Piendl, W., Kisselev, L.,            Escherichia coli B strains REL606 BL21(DE3) comparison Garber, M. (2000) Translation termination factor aRF1                   E. coli B K-12 genomes. J. Mol. Biol. 394, 653&#226;&#710;&#8217;680. <br /> archaeon Methanococcus jannaschii active eukaryotic ribosomes.                 (25) Kang, Y., Durfee, T., Glasner, J. D., Qiu, Y., Frisch, D., <br /> FEBS Lett. 472, 213&#226;&#710;&#8217;216.                                                             Winterberg, K. M., Blattner, F. R. (2004) Systematic mutagenesis <br />  (6) Scolnick, E., Tompkins, R., Caskey, T., Nirenberg, M. (1968)                Escherichia coli genome. J. Bacteriol. 186, 4921&#226;&#710;&#8217;4930. <br /> Release factors di&#239;&#172;&#8364;ering speci&#239;&#172;?city terminator codons. Proc. Natl.              (26) Posfai, G., Plunkett, G., 3rd, Feher, T., Frisch, D., Keil, G. M., <br /> Acad. Sci. U.S  61, 768&#226;&#710;&#8217;774.                                                       Umenho&#239;&#172;&#8364;er, K., Kolisnychenko, V., Stahl, B., Sharma, S. S., Arruda, <br />  (7) Gibson, D. G., Glass, J. , Lartigue, C., Noskov, V. N., Chuang, R.            M., Burland, V., Harcum, S. W., Blattner, F. R. (2006) Emergent <br /> Y., Algire, M. , Benders, G. , Montague, M. G., Ma, L., Moodie, M.               properties reduced-genome Escherichia coli. Science 312, 1044&#226;&#710;&#8217;1046. <br /> M., Merryman, C., Vashee, S., Krishnakumar, R., Assad-Garcia, N.,                      (27) Uno, M., Ito, K., Nakamura, Y. (1996) Functional speci&#239;&#172;?city <br /> Andrews-Pfannkoch, C., Denisova, E. , Young, L., Qi,  Q., Segall-                amino acid position 246 tRNA mimicry domain bacterial <br /> Shapiro, T. H., Calvey, C. H., Parmar, P. P., Hutchison, C. , 3rd,                 release factor 2. Biochimie 78, 935&#226;&#710;&#8217;943. <br /> Smith, H. O., Venter, J. C. (2010) Creation bacterial cell                    (28) Dincbas-Renqvist, V., Engstrom, , Mora, L., Heurgue-Hamard, <br /> controlled chemically synthesized genome. Science 329, 52&#226;&#710;&#8217;56.                   V., Buckingham, R., Ehrenberg, M. (2000) post-translational <br />  (8) Wang, L., Brock, , Herberich, B., Schultz, P. G. (2001)                   modi&#239;&#172;?cation GGQ motif RF2 Escherichia coli stimulates <br /> Expanding genetic code Escherichia coli. Science 292, 498&#226;&#710;&#8217;500.                termination translation. EMBO J. 19, 6900&#226;&#710;&#8217;6907. <br />  (9) Wang, L., Schultz, P. G. (2005) Expanding genetic code.                   (29) Nakamura, Y., Gojobori, T., Ikemura, T. (2000) Codon <br /> Angew. Chem., Int. Ed. 44, 34&#226;&#710;&#8217;66.                                                    usage tabulated international DNA sequence databases: status  (10) Wang, Q., Parrish,  R., Wang, L. (2009) Expanding                    year 2000. Nucleic Acids Res. 28, 292. <br /> genetic code biological studies. Chem. Biol. 16, 323&#226;&#710;&#8217;336.                          (30) Jeong, H., Barbe, V., Lee, C. H., Vallenet, D., Yu, D. S., Choi, S. <br />  (11) Wu, X., Schultz, P. G. (2009) Synthesis interface                H., Couloux, , Lee, S. W., Yoon, S. H., Cattolico, L., Hur, C. G., Park, <br /> chemistry biology. J.  Chem. Soc. 131, 12497&#226;&#710;&#8217;12515.                           H. S., Segurens, B., Kim, S. C., Oh, T. K., Lenski, R. E., Studier, F. W., <br />  (12) Liu, C. C., Schultz, P. G. (2010) Adding new chemistries                Daegelen, P., Kim, J. F. (2009) Genome sequences Escherichia genetic code. Annu. Rev. Biochem. 79, 413&#226;&#710;&#8217;444.                                   coli B strains REL606 BL21(DE3). J. Mol. Biol. 394, 644&#226;&#710;&#8217;652. <br />  (13) Johnson, D. B., Xu, J., Shen, , Takimoto, J. K., Schultz, M. D.,               (31) Durfee, T., Nelson, R., Baldwin, S., Plunkett, G., 3rd, Burland, <br /> Schmitz, R. J., Xiang, , Ecker, J. R., Briggs, S. P., Wang, L. (2011)          V., Mau, B., Petrosino, J. F., Qin, X., Muzny, D. M., Ayele, M., Gibbs, <br /> RF1 knockout allows ribosomal incorporation unnatural amino                       R. , Csorgo, B., Posfai, G., Weinstock, G. M., Blattner, F. R. <br /> acids multiple sites. Nat. Chem. Biol. 7, 779&#226;&#710;&#8217;786.                                (2008) complete genome sequence Escherichia coli DH10B: <br />  (14) Anderson, J. C., Wu, N., Santoro, S. W., Lakshman, V., King, D.                insights biology laboratory workhorse. J. Bacteriol. 190, <br /> S., Schultz, P. G. (2004) expanded genetic code                        2597&#226;&#710;&#8217;2606. <br /> functional quadruplet codon. Proc. Natl. Acad. Sci. U.S  101, 7566&#226;&#710;&#8217;                  (32) Zhang, S., Ryden-Aulin, M., Kirsebom, L. , Isaksson, L.  <br /> 7571.                                                                                (1994) Genetic implication interaction release factor <br />  (15) Huang, Y., Russell, W. K., Wan, W., Pai, P. J., Russell, D. H.,            ribosomal protein L7/L12 vivo. J. Mol. Biol. 242, 614&#226;&#710;&#8217;618. <br /> Liu, W. (2010) convenient method genetic incorporation                        (33) Ito, K., Uno, M., Nakamura, Y. (1998) Single amino acid <br /> multiple noncanonical amino acids protein Escherichia coli.              substitution prokaryote polypeptide release factor 2 permits Mol. Biosyst. 6, 683&#226;&#710;&#8217;686.                                                            terminate translation stop codons. Proc. Natl. Acad. Sci. <br />  (16) Isaacs, F. J., Carr, P. , Wang, H. H., Lajoie, M. J., Sterling, B.,          U.S  95, 8165&#226;&#710;&#8217;8169. <br /> Kraal, L., Tolonen,  C., Gianoulis, T. , Goodman, D. B., Reppas, N.                (34) Dahlgren, , Ryden-Aulin, M. (2000) novel mutation B., Emig, C. J., Bang, D., Hwang, S. J., Jewett, M. C., Jacobson, J. M.,             ribosomal protein S4 a&#239;&#172;&#8364;ects function mutated RF1. Church, G. M. (2011) Precise manipulation chromosomes                      Biochimie 82, 683&#226;&#710;&#8217;691. <br /> vivo enables genome-wide codon replacement. Science 333, 348&#226;&#710;&#8217;353.                      (35) Kaczanowska, M., Ryden-Aulin, M. (2004) Temperature <br />  (17) Lozupone, C., Knight, R., Landweber, L. (2001)                         sensitivity caused mutant release factor 1 suppressed molecular basis nuclear genetic code change ciliates. Curr. Biol.              mutations a&#239;&#172;&#8364;ect 16S rRNA maturation. J. Bacteriol. 186, 3046&#226;&#710;&#8217; <br /> 11, 65&#226;&#710;&#8217;74.                                                                           3055. <br />  (18) Inagaki, Y., Doolittle, W. F. (2001) Class release factors              (36) Yang, T. T., Cheng, L., Kain, S. R. (1996) Optimized codon <br /> ciliates variant genetic codes. Nucleic Acids Res. 29, 921&#226;&#710;&#8217;927.                 usage chromophore mutations provide enhanced sensitivity  (19) Inagaki, Y., Bessho, Y., Osawa, S. (1993) Lack peptide-                 green &#239;&#172;&#8218;uorescent protein. Nucleic Acids Res. 24, 4592&#226;&#710;&#8217;4593. <br /> release activity responding codon UGA Mycoplasma capricolum.                     (37) Murgola, E. J. (2005) Translational suppression: Nucleic Acids Res. 21, 1335&#226;&#710;&#8217;1338.                                                    wrongs make right, tRNA: Structure, Biosynthesis, Function <br />  (20) Gerdes, S. Y., Scholle, M. D., Campbell, J. W., Balazsi, G.,                   (Soll, D., RajBhandary, U. L., Eds.), pp 491&#226;&#710;&#8217;509, ASM Press, <br /> Ravasz, E., Daugherty, M. D., Somera,  L., Kyrpides, N. C.,                        Washington, DC. <br /> Anderson, , Gelfand, M. S., Bhattacharya, , Kapatral, V., D&#226;&#8364;&#8482;Souza,                 (38) Wang, L., Schultz, P. G. (2001) general approach M., Baev, M. V., Grechkin, Y., Mseeh, F., Fonstein, M. Y., Overbeek,                 generation orthogonal tRNAs. Chem. Biol. 8, 883&#226;&#710;&#8217;890. <br /> R., Barabasi,  L., Oltvai,  N., Osterman,  L. (2003)                         (39) Wang, L., Zhang, , Brock, , Schultz, P. G. (2003) <br /> Experimental determination level analysis essential                    Addition keto functional group genetic code genes Escherichia coli MG1655. J. Bacteriol. 185, 5673&#226;&#710;&#8217;5684.                      Escherichia coli. Proc. Natl. Acad. Sci. U.S  100, 56&#226;&#710;&#8217;61. <br />  (21) Ryde&#204;?n, S., Isaksson, L. (1984) temperature-sensitive                      (40) Takimoto, J. K., Adams, K. L., Xiang, , Wang, L. (2009) <br /> mutant Escherichia coli shows enhanced misreading UAG                   Improving orthogonal tRNA-synthetase recognition e&#239;&#172;&#402;cient increased e&#239;&#172;&#402;ciency tRNA nonsense suppressors. Mol.                      unnatural amino acid incorporation application mammalian <br /> Gen. Genet. 193, 38&#226;&#710;&#8217;45.                                                              cells. Mol. Biosyst. 5, 931&#226;&#710;&#8217;934. <br />  <br />                                                                               1343                        dx.doi.org/10.1021/cb300229q | ACS Chem. Biol. 2012, 7, 1337&#226;&#710;&#8217;1344 <br />  ACS Chemical Biology                                                                                                                   Letters <br />  <br />  (41) Yokoyama, S., Nishimura, S. (1995) Modi&#239;&#172;?ed nucleosides codon recognition, tRNA: Structure, Biosynthesis, Function (Soll, <br /> D., RajBhandary, U. L., Eds.), pp 207&#226;&#710;&#8217;223, ASM Press, <br /> Washington, DC. <br />  (42) Schrader, J. M., Chapman, S. J., Uhlenbeck, O. C. (2011) <br /> Tuning a&#239;&#172;&#402;nity aminoacyl-tRNA elongation factor Tu optimal decoding. Proc. Natl. Acad. Sci. U.S. . 108, 5215&#226;&#710;&#8217;5220. <br />  (43) Ledoux, S., Uhlenbeck, O. C. (2008) Di&#239;&#172;&#8364;erent aa-tRNAs selected uniformly ribosome. Mol. Cell 31, 114&#226;&#710;&#8217;123. <br />  (44) Kato, J., Hashimoto, M. (2007) Construction consecutive <br /> deletions Escherichia coli chromosome. Mol. Syst. Biol. 3, 13 <br /> </body></html>