<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Microbiol. 2012 June ; 84(5): 816&#226;&#8364;&#8220;831. doi:10.1111/j.1365-2958.2012.08060.x. <br />  <br />  <br />  <br />                            95 kDa protein Plasmodium vivax P. cynomolgi <br />                            visualized 3-D tomography caveola-vesicle complexes <br />                            (Sch&#195;&#188;ffner&#226;&#8364;&#8482;s dots) infected erythrocytes member                            PHIST family <br />                            Sheila Akinyi1,&#226;&#8364;&#160;,&#226;&#8364;&#160;, Eric Hanssen2,3,&#226;&#8364;&#160;, Esmeralda V.S. Meyer1, Jianlin Jiang1, Cindy C. <br />                            Korir1,&#194;&#167;, Balwan Singh1, Stacey Lapp1, John W. Barnwell4, Leann Tilley3,5,&#194;&#165;, Mary R. <br />                            Galinski1,6,*,&#194;&#165; <br />                            1Emory Vaccine Center, Yerkes National Primate Research Center, Emory University, Atlanta, <br />  <br />                            GA, USA <br />                            2ElectronMicroscopy Unit, Bio21 Molecular Science Biotechnology Institute, University <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Melbourne, Melbourne, VIC 3010, Australia <br />                            3ARC    Centre Excellence Coherent X-ray Science <br />                            4MalariaBranch, Division Parasitic Diseases Malaria, Centers Disease Control                            Prevention, Atlanta, GA, USA <br />                            5Department   Biochemistry Molecular Biology, Bio21 Molecular Science Biotechnology <br />                            Institute, University Melbourne, Melbourne, VIC 3010, Australia <br />                            6Department      Medicine, Division Infectious Diseases, Emory University, Atlanta, GA, USA <br />  <br />                            Summary <br />                                  Plasmodium vivax P. cynomolgi produce numerous caveolae-vesicle complex (CVC) <br />                                  structures surface infected erythrocyte membrane. contrast                                  electron-dense knob protrusions expressed surface P. falciparum infected erythrocytes. <br />                                  investigate 3-dimensional structure CVCs identity predominantly <br />                                  expressed 95 kDa CVC protein. Liquid chromatography - tandem mass spectrometry analysis                                  immunoprecipitates monoclonal antibodies P. cynomolgi extracts identified protein <br />                                  member Plasmodium helical interspersed sub-telomeric (PHIST) superfamily NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  calculated mass 81 kDa. named orthologous proteins PvPHIST/CVC-8195                                  PcyPHIST/CVC-8195, analyzed structural features, including PEXEL motif, repeated <br />                                  sequences C-terminal PHIST domain, PHIST/CVC-8195 highly <br />                                  expressed trophozoites. generated images CVCs 3-D using electron tomography,                                  used immuno-Electron Tomography (ET) PHIST/CVC-8195 localizes cytoplasmic <br />                                  CVC tubular extensions. Targeted gene disruptions attempted vivo.                                  pcyphist/cvc-8195 gene disrupted, parasites containing episomes tgdhfr <br />                                  selection cassette retrieved selection pyrimethamine. suggests PHIST/ <br />                                  CVC-8195 essential survival malaria parasites. <br />  <br />  <br />  <br />  <br />                            * <br />                             Correspondence. mary.galinski@emory.edu; Tel: 404-727-7214; Fax: 404-727-8199. <br />                             individuals contributed equally research authors. <br />                             individuals contributed equally development research senior authors. <br />                            &#226;&#8364;&#160;Department Parasitic Diseases Malaria, Centers Disease Control Malaria, Atlanta GA 30083; <br />                            &#194;&#167;Kimberly-Clark Corporate Research Engineering, Roswell GA 30076. <br />                             Akinyi et al.                                                                                                Page 2 <br />  <br />  <br />  <br />                               Keywords <br />                                     Malaria; Plasmodium; Erythrocytes; Tomography; PHIST; Caveolae-Vesicle Complex <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Introduction <br />                                              Malaria, infectious disease caused predominantly Plasmodium vivax P. <br />                                              falciparum, results illness million people year                                              countries   2010). parasite undergoes cyclical process invading, <br />                                              thriving  rupturing host erythrocytes. blood-stage disease,                                              merozoite form parasite efficiently enters host red blood cells (RBCs), grows                                              multiplies create new asexual progeny, differentiates generate sexual stage <br />                                              parasites. Pathogenic processes ensue host, resulting organ damage, anemia                                              associated medical complications (reviewed Price et al., 2009; Mueller et al., 2009; Rowe <br />                                              et al., 2009). biological mechanisms used P. vivax P. falciparum invade, grow <br />                                              survive inside RBCs important species-specific differences, largely <br />                                              unexplored. Research P. vivax trailed far P. falciparum, comparative <br />                                              studies  <br />  <br />                                              merozoite invading target host RBC creates parasitophorous vacuole (PV) <br />                                              (reviewed Galinski et al., 2005). parasite grows PV, surrounded PV <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              membrane, host RBC cytoplasm host RBC membrane. uninfected, <br />                                              enucleated RBC lacks protein synthesis trafficking machinery, parasite                                              capacity produce transport proteins PV membrane onwards using <br />                                              specialized structures develops RBC cytoplasm surface iRBC <br />                                              membrane. Investigations area malaria biology advanced P. <br />                                              falciparum, resulting breakthroughs scientific understanding transport <br />                                              biology species (reviewed Tilley et al., 2008; Maier et al., 2009; Haase et al., 2010; <br />                                              Tilley et al., 2011). Research P. falciparum blood-stage biology aided                                              1976 availability robust vitro culture systems (Trager et al., 1976),                                               decade P. falciparum genome database (Gardner et al., 2002) variety                                              functional genome technologies (reviewed Di Girolamo et al., 2005; Aurrecoechea et al., <br />                                              2009). growing body knowledge exists relating structure composition                                              P. falciparum-induced cytoplasmic membranous structures develop PV                                              surface iRBC. Notably, P. falciparum iRBC membrane surface                                              studded electron-dense structures known knobs, contain erythrocyte <br />                                              membrane protein-1 (PfEMP-1), antigenically variant virulence protein encoded <br />                                              large var multigene family (reviewed Scherf et al., 2008). Recently, electron <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              tomography (ET) employed imaging P. falciparum iRBCs, technology <br />                                              enabled 3-dimensional (3-D) imaging iRBC, Maurer&#226;&#8364;&#8482;s cleft organelles                                              discovery tethers bridging iRBC membrane (Hanssen et al., 2008; <br />                                              Hanssen et al., 2011). <br />  <br />                                              contrast developing protruding knob structures, peppered surface iRBC <br />                                              host membrane, P. vivax induces pitted membrane surface formation numerous <br />                                              flask-shaped indentations iRBC membrane, called caveolae, association                                              vesicles. unique intricate structures, termed caveola-vesicle complexes (CVCs),                                              observed transmission electron microscopy (TEM) 1975 (Aikawa et al., 1975)                                              described 90&#226;&#8364;&#8220;100 nm wide caveolae 40&#226;&#8364;&#8220;50 nm wide electron-dense vesicles <br />                                              extending caveolar base. vesicles appear alveolar shape,                                              elongated tubular (Aikawa et al., 1975). studies, focused iRBC <br />                                              membranes P. vivax simian malaria parasite P. cynomolgi, species  <br />  <br />                                                   Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                             Page 3 <br />  <br />  <br />                                            genetically close P. vivax (Waters et al., 1993), established caveolae <br />                                            structures open exterior host cell physically accumulate Giemsa stain <br />                                            resulting abundant reddish speckled appearance P. vivax iRBCs treated NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Romanowsky-based stains known Sch&#195;&#188;ffner&#226;&#8364;&#8482;s stippling (Schuffner, 1899). CVCs                                            confirmed TEM present human malaria species P. ovale <br />                                            (Matsumoto et al., 1986; Aikawa et al., 1977),  based similar Giemsa-stained <br />                                            patterns, presumed present non-human primate species <br />                                            Plasmodium (P. fieldi, P. simiovale P. gonderi); iRBCs species <br />                                            similarly characterized profuse Sch&#195;&#188;ffner&#226;&#8364;&#8482;s like stippling Giemsa-stained iRBCs                                            blood smears (Coatney, 1971). <br />  <br />                                            CVC form functional purpose abundant membrane structures                                            known, suggested function nutrient transport release <br />                                            parasite metabolites infected erythrocytes (Aikawa et al., 1975; Udagama et al., <br />                                            1988; Matsumoto et al., 1988). Limited exploratory biochemical studies confirmed                                            provide means metabolic exchange parasitized cell host plasma <br />                                            (Aikawa et al., 1975). Lactoperoxidase catalyzed 125I surface examination, metabolic <br />                                            radiolabeling experiments immunobiochemical studies verified CVCs                                            comprised 95 kDa specific parasite-derived proteins (Barnwell et al., 1990; <br />                                            Udagama et al., 1988; Barnwell, 1986).  composition, developmental origin, <br />                                            function(s) CVCs remain largely unexplored. simian malaria <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            species P. knowlesi, related P. vivax, contrast, produces                                            caveolae surface iRBCs, associated alveolar tubular vesicles <br />                                            (Aikawa et al., 1975). <br />  <br />                                            Investigations relating unique species-specific modifications P. vivax iRBCs                                            corresponding transport biology progressed parallel similar lines                                            research P. falciparum iRBCs. largely continuous vitro blood-stage <br />                                            culture systems routine functional genetic technologies established                                            studies P. vivax P. cynomolgi, serves effective reliable <br />                                            experimental model P. vivax (reviewed Galinski et al., 2008; Galinski et al., press). <br />                                             P. vivax genome sequence associated database published                                            recently (Carlton et al., 2008), P. cynomolgi genome sequence                                            published. functional genomics species experimental infancy, initial <br />                                            successful transient transfections P. vivax (Pfahler et al., 2006) P. cynomolgi <br />                                            (Kocken et al., 1999) iRBCs reported experimental infections non-human <br />                                            primates. <br />  <br />                                            present study, aimed investigate 95 kDa protein shown NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            predominant parasite-derived component exclusively associated CVCs P. vivax <br />                                            iRBCs (Barnwell et al., 1990), examine structure CVCs time <br />                                            3-D. Liquid chromatography - tandem mass spectrometry (LC-MS/MS) analysis                                            protein immunoprecipitated detergent extracts P. cynomolgi iRBC using <br />                                            monoclonal antibodies (mAbs) revealed member Plasmodium Helical <br />                                            Interspersed Sub-Telomeric (PHIST) superfamily proteins (Sargeant et al., 2006)                                            calculated mass 81 kDa. Electron tomograms presented showing elaborate multi- <br />                                            vesicular/tubular nature parasite&#226;&#8364;&#8482;s CVCs confirming presence 95 kDa <br />                                            protein, termed PHIST/CVC-8195, specifically cytoplasmic tubules. <br />                                            Finally, demonstrate PHIST/CVC-8195 protein appears required growth <br />                                            parasite. Transfected P. cynomolgi parasites retrieved vivo rhesus <br />                                            monkey blood-stage infections applying pyrimethamine drug pressure, episomes <br />                                            containing pcyphist/cvc-8195 knock (KO) construct tgdhfr drug selection <br />                                            cassette, able achieve disruption pcyphist/cvc-8195 gene. <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                         Page 4 <br />  <br />  <br />                               Results <br />                               Proteomic identification predominant P. vivax 95 kDa CVC protein member NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               PHIST superfamily detection analysis homolog P. cynomolgi iRBCs <br />                                            subset mAbs developed mature P. vivax iRBCs shown previously target <br />                                            specifically CVCs P. vivax iRBC membranes immunoprecipitate SDS <br />                                            extracts iRBCs predominant P. vivax antigen migrated 95 kDa SDS-PAGE <br />                                            (Barnwell et al., 1990; Barnwell, 1986). mAbs reported                                            time cross-react indirect immunofluorescence assays (IFA) P. cynomolgi iRBC. P. <br />                                            cynomolgi iRBCs easily generated large quantities rhesus monkey infections                                            amenable depth study P. vivax attained small New World <br />                                            monkey infections clinical isolates. set current studies use                                            mAb reagents proteomic experiments identify associated gene P. cynomolgi                                            investigate structure, location function predominant protein                                            context CVCs. proceed, reconfirmed crossreactivity mAbs <br />                                            2H12.B4, 2H8.E10, 4C12.G4, 1H4.B6 P. cynomolgi trophozoite iRBCs (Fig. 1A). <br />                                            typical fluorescence pattern representative CVCs dense pattern speckling <br />                                            reliably produced. heavily dotted patterns mimic classic spread pink                                            dark red dots Sch&#195;&#188;ffner&#226;&#8364;&#8482;s stippling observed P. vivax P. cynomolgi iRBCs <br />                                            Giemsa-stained blood smears. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            confirmed mAbs (2H12.B4, 2H8.E10, 4C12.G4, 1H4.B6) recognized <br />                                            expected 95 kDa antigen, SDS extracts P. cynomolgi iRBCs immunoblot <br />                                            analysis (Fig. 1B), immunoprecipitated corresponding protein associated <br />                                            breakdown products. P. cynomolgi 95 kDa protein immunoprecipitated                                            mAb, excised processed SDS-PAGE gel slices analyzed LC-MS/MS. <br />                                            resulting <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> sequences searched P. vivax genome database                                            gene <span id='am-76' about='oboInOwl:id' typeof='owl:Thing'>ID</span>: PVX_093680 (annotated member PHIST superfamily; Sargeant et al., <br />                                            2006) identified antibody reagent. PHIST protein, like known <br />                                            members superfamily, characterized presence PHIST domain, <br />                                            containing multiple predicted alpha helical domains conserved tryptophan <br />                                            residues. representative immunoprecipitation result using mAb 1H4.B6 shown Fig. <br />                                            1C, detection 95 kDa PHIST protein breakdown product 37 kDa;                                            major protein bands verified LC-MS/MS IgG. MASCOT tool                                            used search P. vivax proteome database gene ID: PVX_093680 identified <br />                                            22 spectral counts, 10 unique shared peptides. PVX_093680 2,133 <br />                                            nucleotides (nt) coding sequence contained exons. 710 <span id='am-45' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-46' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> <br />                                            protein calculated mass 80.73 kDa, lower 95 kDa extrapolated                                            relative electrophoretic mobility SDS-PAGE gels. differences unusual NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Plasmodium antigens, depending amino acid composition imposed <br />                                            secondary structure   migrate quite different                                            calculated molecular masses. <br />  <br />                                            maintain original designation protein 95 kDa antigen determined SDS- <br />                                            PAGE (Barnwell et al., 1990), recognize calculated molecular mass 81 kDa,                                            refer protein P. vivax P. cynomolgi respectively PvPHIST/ <br />                                            CVC-8195 PcyPHIST/CVC-8195. <br />  <br />                               PvPHIST/CVC-8195 PcyPHIST/CVC-8195 lack species-specific repeated motifs present <br />                               species conservation PHIST domain retained species <br />                                            Fifty amino acids downstream N-terminus, PvPHIST/CVC-8195 22 amino <br />                                            acid hydrophobic region. According bioinformatic analyses using MalSig algorithm <br />  <br />  <br />                                                Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                           Page 5 <br />  <br />  <br />                                            (http://131.172.128.59/cgi-bin/pfsigseq_refs.py), region serve recessed signal <br />                                            sequence signal anchor. predicted Plasmodium export element (PEXEL; RxLxE/Q/D; <br />                                            Marti et al., 2004; Hiller et al., 2004) follows approximately 12 amino acids downstream NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            hydrophobic domain; sequence shown important trafficking <br />                                            necessarily proteins PV onwards surface iRBC <br />                                            (Spielmann et al., 2010). protein&#226;&#8364;&#8482;s PHIST domain C-terminal portion                                            protein, predicted alpha helical domains positionally conserved <br />                                            tryptophan residues (Sargeant et al., 2006). PvPHIST/CVC-8195 classified                                            aspartate (11.7%) <span id='am-52' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> (11%) rich (http://ca.expasy.org/tools/protparam.html)                                            presence remnants degenerated repeated <span id='am-37' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-38' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-41' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence motifs                                            central region gene (discussed  encode comparatively high <br />                                            number amino acids. Arginine proline residues relatively high 11.4% <br />                                            11%, respectively. <br />  <br />                                            support continued depth investigations PvPHIST/CVC-8195 biological                                            functional analysis CVCs, proceeded identify characterize orthologous gene <br />                                            experimental model P. cynomolgi. absence P. cynomolgi genome database                                            time work necessitated traditional scheme identify characterize                                            pcyphist/cvc-8195 gene using polymerase chain reaction (PCR) standard DNA <br />                                            sequencing methods.  pvphist/cvc-8195 homologs detected P. falciparum                                            simian malaria species P. knowlesi BLAST (Basic Linear Alignment Search <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Tool) nucleotide searches PlasmoDB database (http://plasmodb.org/plasmo/).                                            closely related sequences identified respective gene IDs: Pf08_0137 <br />                                            (annotated Plasmodium exported protein (PHISTc)) PkH_011720, calculated <br />                                            molecular masses 147 kDa 105 kDa, respectively; referred                                            comparable PfPHIST-147 PkPHIST-105 nomenclature. annotated                                            PlasmoDB database proteins unknown function. BLAST analyses NCBI <br />                                            database (http://www.ncbi.nlm.nih.gov/protein/) performed using PvPHIST/ <br />                                            CVC-8195 amino acid sequence reveal distant homologs rodent malaria parasite <br />                                            species. P. yoelii (gene ID: Py01786) P. berghei (duplicated gene IDs: PB108348.00.0 <br />                                            PB000848.03.0 (Sargeant et al., 2006)) homologs identified; unlike primate <br />                                            malaria parasite species, family paralogous genes present rodent parasite <br />                                            species. <br />  <br />                                            Degenerate gene-specific 3&#226;&#8364;&#178;UTR primer pairs subsequently designed based                                            pvphist/cvc-8195 coding non-coding regions  experimental procedures)                                            closely related corresponding gene sequence P. knowlesi (PkH_011720; pkphist-105). <br />                                            Subsequently, PCR amplification cDNA sequencing experiments performed                                            successfully identify ortholog P. cynomolgi. analyses verified pcyphist/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cvc-8195 gene 2,433 nucleotides, encoding 720 amino acid protein exons, <br />                                            similar basic structure exists P. vivax, P. falciparum P. knowlesi <br />                                            homologs (Fig. 2A). Notably, hydrophobic sequence end exon,                                            putative PEXEL motif (Marti et al., 2004) beginning exon II, central region <br />                                            predominated variety degenerated repeated sequences, conserved C-terminal <br />                                            PHIST domain characteristic alpha-helical domains tryptophan residues <br />                                            (Fig. 2A). <br />  <br />                                            analysis shows proteins classified members PHISTc <br />                                            subfamily, based positioning conserved tryptophans PHIST <br />                                            domain (Sargeant et al., 2006). P. cynomolgi gene, 270 nucleotide intron separates <br />                                            short initial exon 213 nucleotides second exon 1,950 nucleotides. size <br />                                            exons introns differs species, conserved sequence motifs features <br />                                            define homologs (Fig. 2B; P. knowlesi phist-8195 intron-exon boundaries based <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                             Page 6 <br />  <br />  <br />                                            RNA sequencing information, Rayner Galinski unpublished data). Table 1 shows                                            percentage identities nucleotide amino acid levels entire P. <br />                                            vivax P. cynomolgi PHIST/CVC-8195 sequences aligned, compared NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            separate regions comprising molecules DNA (5&#226;&#8364;&#178; coding, central, 3&#226;&#8364;&#178; coding <br />                                            regions) protein (N-terminal, central C-terminal regions) levels. N-terminal 155 <br />                                            amino acid regions (92%), especially C-terminal 155 amino acid regions (100%) <br />                                            encoding PHIST domain, highly conserved compared central region                                            protein. Clustal alignments highlighting relationship N-terminal C-terminal <br />                                            regions PvPHIST/CVC-8195, PcyPHIST/CVC-8195, PkPHIST-105 PfPHIST-147 <br />                                            proteins shown Fig. 2B Fig. 2C. Supplementary Information includes                                            sequences alignments proteins, including P. berghei P. yoelii rodent <br />                                            parasite orthologs (Fig. S1 Fig. S2). C-terminal PHIST domains compared <br />                                            species alpha helical domains (helix &#226;&#710;&#8217;1, &#226;&#710;&#8217;2 &#226;&#710;&#8217;3) align,                                            tryptophan residues conserved number species, including rodent parasites <br />                                            (Fig. S1). degenerate repeated motifs readily apparent PfPHIST-147                                            PkPHIST-105 proteins, rodent parasite PHIST proteins, PvPHIST/ <br />                                            CVC-8195 PcyPHIST/CVC-8195. small 4-amino acid motifs (HDGX)                                            present P. vivax sequence number doublets (e.g. DD, PP GG)                                            repeated times central regions PvPHIST/CVC-8195 PcyPHIST/ <br />                                            CVC-8195. suggests central regions genes undergoing evolutionary <br />                                            changes constrained requirement strict conservation central <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            domain. contrary, hypothesize central regions PvPHIST/CVC-8195 <br />                                            PcyPHIST/CVC-8195 proteins evolved accommodate specific roles                                            proteins P. vivax P. cynomolgi CVCs. <br />  <br />                               PcyPHIST/CVC-8195 expressed erythrocytic stages P. cynomolgi, <br />                               predominantly trophozoites <br />                                            pvphist/cvc-8195 pcyphist/cvc-8195 genes predicted proteins                                            identified analyzed, recombinant protein representing 710 amino acids encoded                                            pvphist/cvc-8195 (rPvPHIST/CVC-8195) expressed Escherichia coli used                                            produce polyclonal rabbit antiserum (anti-rPvPHIST/CVC-8195). antiserum                                            evaluated immunofluorescence microscopy blood smears containing P. cynomolgi <br />                                            iRBCS different stages ~48 hour cycle. immunofluorescence pattern small <br />                                            dots spotted iRBC evident beginning moderate density ring-stage <br />                                            development. speckling greatly increased abundance intensity                                            trophozoite schizont stages (Fig. 3A). Rabbit pre-immune serum negative                                            stages (data shown). PcyPHIST/CVC-8195 detected western immunoblot <br />                                            assays developmental stages maximal expression apparent NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mature trophozoite stage (Fig. 3B). Interestingly, ring stage development                                            fluorescent labeled dots sparser number, imperfect hexagonal plate called <br />                                            &#226;&#8364;&#339;soccer ball&#226;&#8364;? pattern fluorescence frequently apparent, fluorescent areas <br />                                            intermixed dark areas lacking fluorescence (Fig. 3A, panel 1, arrows). <br />  <br />                               Immunoelectron tomography reveals elaborate 3-dimensional signatures CVCs                               P. cynomolgi-iRBCs localizes PcyPHIST/CVC-8195 cytoplasmic                               tubular extensions <br />                                            Transmission EM CVCs cross-sectional analysis P. vivax P. cynomolgi iRBCs <br />                                            provide incomplete picture physical nature organelles (Aikawa et al., <br />                                            1975; Udagama et al., 1988, Matsumoto et al., 1988, Barnwell et al., 1990). Particularly                                            <span id='am-36' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> specific CVC proteins antisera hand, interested                                            exploring CVC structures visually 3-D space determining localization                                            predominant PcyPHIST/CVC-8195 protein using immuno-ET methods. Electron <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                            Page 7 <br />  <br />  <br />                                            tomography enables generation 3-D images previously shown                                            powerful technique visualizing internal membranous structure organelles P. <br />                                            falciparum iRBCs (Hanssen et al., 2008; Hanssen et al., 2010).  P. cynomolgi <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            trophozoite iRBC samples processed similar manner, using equinatoxin II (EqtII) <br />                                            (Anderluh et al., 1996; Jackson et al., 2004) treatment release hemoglobin allow entry <br />                                            antibodies. Transmission electron micrographs 3-D electron tomograms intact <br />                                            EqII-permeabilised cells generated (Fig. 4, 5). images revealed elaborate <br />                                            structural nature CVCs. caveolar &#226;&#8364;&#339;cup&#226;&#8364;? electron-dense coat remains open                                            surface iRBC provides open connection variable number coated <br />                                            vesicles tubules. average width tubular structures 40 nm, vesicles <br />                                            range average diameter 25 45 nm. Additional vesicles similar diameters <br />                                            appear located close CVCs. structures similar intact EqtII- <br />                                            permeabilised iRBCs indicating procedure does compromise CVC <br />                                            membrane structure, permitting improved contrast. membranous structures                                            observed including Maurer&#226;&#8364;&#8482;s cleft-like structures (Fig. 4C) host cell mitochondria (Fig. <br />                                            4A) consistent preference P. cynomolgi, like P. vivax, invading reticulocytes <br />                                            (reviewed Galinski et al., 2008; Galinski et al., press). <br />  <br />                                            demonstrate PcyPHIST-8195 associated CVCs P. cynomolgi <br />                                            trophozoite iRBCs. Equinatoxin II-permeabilised iRBCs labeled mAb 4C12.G4 <br />                                            (Fig. 4D, 6 D) anti-rPvPHIST/CVC-8195 serum (Fig. 4E, 6 E&#226;&#8364;&#8220;H). 4C12.G4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            gold-labeled mAb localized CVCs, previously observed immuno-TEM experiments <br />                                            P. vivax-iRBCs (Barnwell et al., 1990). labeling indicates epitope                                            present cytoplasmic tubular extensions CVCs (Fig. 6D 6H). <br />  <br />                                            used IMOD stitch serial 3-D reconstructed tomograms region P. <br />                                            cynomolgi-iRBC containing CVCs. interconnected nature caveolar opening                                            CVC, body caveola connected multiple tubular extensions                                            appreciated examining individual sections electron tomograms (Fig. 5, 6)                                            translating sections (Supplementary Videos S1, S3). CVC unique <br />                                            &#226;&#8364;&#732;morphological signature&#226;&#8364;&#8482;, differing number sizes vesicles tubules. IMOD <br />                                            used render surface CVCs 3-D rotations presented                                            Supplementary Videos S2 S4. caveolar opening rendered blue tubule- <br />                                            vesicular extensions orange. <br />  <br />                               Genetic manipulation P. cynomolgi pcy&#206;&#8221;phist/cvc8195 vector <br />                                            Given predominance PvPHIST/CVC-8195 PcyPHIST/CVC-8195 proteins                                            iRBC membrane component CVCs (Barnwell et al., 1990),                                            hypothesized protein prove essential parasite&#226;&#8364;&#8482;s survival. test <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            hypothesis, carried transfection experiments aiming target knock                                            (KO) pcyphist/cvc-8195 gene. pcy&#206;&#8221;phist/cvc-8195 KO cassette designed                                            target 600 bp 5&#226;&#8364;&#178;-end 554 bp 3&#226;&#8364;&#178; end pcyphist-8195 gene (Fig. 7A). <br />                                            construct contains mutated tgdhfr-ts gene flanked P. berghei dhfr 5&#226;&#8364;&#178; <br />                                            promoter 3&#226;&#8364;&#178; UTR regions drive expression DHFR protein confer <br />                                            pyrimethamine resistance. pcy&#206;&#8221;phist/cvc-8195 vector digested BamHI                                            EcoRI release KO cassette pUC19 backbone vector. released, linearized <br />                                            constructs visualized agarose gel electrophoresis (Fig 7B). electroporation                                            performed using suspension P. cynomolgi schizont-iRBCs enriched blood <br />                                            collected infected donor rhesus monkey. Transfected iRBCs immediately <br />                                            reconstituted incomplete RPMI-1640 inoculated recipient rhesus monkey. <br />  <br />                                            Parasitemia recipient monkeys sequential experiments monitored daily                                            microscopic analysis blood smears. experiment,  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                           Page 8 <br />  <br />  <br />                                            parasitemia increased 0.63% day 3 dose pyrimethamine                                            administered. drug administered days 4 10. Parasites                                            subsequently observed day 14. Additional pyrimethamine doses administered NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            days 17 18, continue select parasites harboring mutated tgdhfr-ts gene.                                            parasitemia increased 3.1% day 24, infected blood collected, stored. <br />                                            second experiment parasitemia increased 1.2% day 6 dose                                            pyrimethamine administered. drug administered days 7, 8, 13, 14, 19, <br />                                            20, 24 25. Selected parasites observed day 18. parasitemia increased                                            2.6% day 25, infected blood collected, stored day 26. Graphs                                            parasitemias shown Fig. S3. <br />  <br />                                            Genomic DNA recovered parasites extracted used template PCR <br />                                            amplification assess integration status pcy&#206;&#8221;phist/cvc-8195 cassette, using <br />                                            pcyphist/cvc-8195 forward reverse primers (Fig. 7B). Despite retrieval                                            pyrimethamine resistant parasites, PCR tests negative, indicating lack                                            genomic integration pcy&#206;&#8221;phist/cvc-8195 cassette. Southern blots experiments                                            consistent lack integration  shown). questioned                                            episomes, possibly resulting low copy number undigested plasmids                                            electroporation mixture, conferring resistance drug. address question, <br />                                            DNA transfected parasites extracted using QIAgen miniprep kit plasmid <br />                                            purification. Plasmid DNA retrieved iRBCs used transform TOP10 E. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            coli cells (invitrogen). Plasmid DNA purified digested EcoRI EcoRV.                                            digested products recovered (5.1 kb 3.6 kb) corresponded pUC19/pcy&#206;&#8221; <br />                                            phist8195 vector used transfect P. cynomolgi iRBCs (Fig. 7C). expected, IFA                                            western immunoblot experiments showed expression PHIST protein (Fig. 7D Fig <br />                                            7E). experiment performed parallel, hand, served positive control, <br />                                            showing integration expression red fluorescent protein (rfp) gene (Fig. S4,                                            unpublished data). data serve confirm time integration                                            transgene possible P. cynomolgi. <br />  <br />                               Discussion <br />                                            Advances post-genomic tools microscopic methods greatly expanded                                            potential investigations cellular biology Plasmodium iRBCs, particularly P. <br />                                            falciparum. recent publication P. vivax P. knowlesi genome databases <br />                                            (Carlton et al., 2008; Pain et al., 2008), focused efforts sequencing P. cynomolgi <br />                                            genomes (unpublished), raise prospects comparable discoveries species. <br />                                            especially important light increasing data showing P. vivax, addition                                            causing enormous morbidity worldwide, cause severe disease infections NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            lethal (reviewed Mueller et al., 2009; Price et al., 2009). <br />  <br />                                            P. cynomolgi iRBCs main focus exploratory studies reported                                             unlike P. vivax accessible monkey (rhesus) infections significant numbers <br />                                            experimental manipulations depth investigations. Based knowledge regarding <br />                                            close phylogenetic relationship P. cynomolgi P. vivax common <br />                                            overarching morphological biological features, protein composition functional <br />                                            biology specific organelles CVC expected extremely similar.                                            current experiments prelude using P. cynomolgi surrogate expanded cell <br />                                            biological studies relating CVC ultrastructure, molecular composition, function,  <br />                                            advance specific questions directly P. vivax iRBCs attained limited quantities <br />                                            small New World monkey infections. <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                             Page 9 <br />  <br />  <br />                                            study 1975 investigate structure molecular make- <br />                                            Plasmodium CVCs (Aikawa et al., 1975; Barnwell et al., 1990; Barnwell, 1986; <br />                                            Aikawa et al., 1977; Atkinson et al., 1990; Bracho et al., 2006; Udagama et al., 1988; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Matsumoto et al., 1988; Matsumoto et al., 1986) present structures 3- <br />                                            D. ET imaging results presented corroborate previous TEM reports showing <br />                                            numerous CVCs interspersed surface P. vivax P. cynomolgi iRBCs                                            caveolae open surface (Aikawa et al., 1975). iRBC plasmalemma                                            highly reorganized course approximate 48-hour life span growing <br />                                            parasites accommodate intricate structures. interesting questions remain                                            answered including 1) proteins make structures, 2) formed, 3) <br />                                            purpose(s), 4) targets intervention? <br />  <br />                                            Proteomic experiments critical identify phist/cvc8195 gene encoding 95 kDa <br />                                            CVC antigen designate member PHISTc sub-family. protein <br />                                            shown previously predominant protein localized CVCs P. vivax <br />                                            P. cynomolgi, extractable primarily SDS, predicted associated                                            cytoskeleton underlying RBC membrane (Barnwell et al., 1990), gene encoded <br />                                            protein characteristics remained unknown. discovery fact marks                                            designation physical location member PHIST superfamily (Sargeant et al., <br />                                            2006). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            tomograms CVC opening (narrowing diameter ~35 nm)                                            thickened limiting membrane region connection iRBC membrane; e.                                            electron dense caveola cup. 5&#226;&#8364;&#8220;7 tubules (~40 nm diameter) emanating                                            CVC opening; tubules vary length considerably elongated                                             400 nm). Vesicles ranging average diameter 25 45 nm integral                                            CVCs, appear isolation nearby. Large Maurer&#226;&#8364;&#8482;s cleft-like structures                                            observed host cell cytoplasm. cytoplasmic face tubules clearly <br />                                            labeled anti-PHIST/CVC-8195 mAb polyclonal antiserum immuno-ET <br />                                            experiments, CVC openings, CVC-associated vesicles cleft-like <br />                                            structures labeled. previous immuno-TEM analyses, using post embedding <br />                                            immunolabeling procedures, mAbs, including 2H12.G4 1H4.B6, stained                                            caveola P. vivax CVCs addition vesicles tubules (Barnwell et al., 1990; <br />                                            Matsumoto et al., 1988). Future analysis iRBCs different stages development <br />                                            prove informative regards definitive static dynamic localization <br />                                            protein. certainly possible, excluded time, protein <br />                                            localized functions cytoplasmic interface CVCs,                                            caveolae. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Different numbers types PHIST family members (PHISTa, PHISTb, PHISTc) <br />                                            presumptive common ancestral lineages revealed P. falciparum, P. vivax, <br />                                            P. knowlesi, P. gallinaceum, P. yoelii, P. berghei, P. chabaudi definitive <br />                                            localizations functions ascribed members. Family members                                            differ considerably, example size, structure transcription pattern,                                            called PHIST domain common characteristic multiple alpha helices                                            conserved tryptophan residues (Sargeant et al., 2006). presence domain defines <br />                                            family members, specific functions remain unknown (Sargeant et al., 2006).                                            71 phist paralogs annotated P. falciparum, 39 P. vivax (18 paralogs reported                                            (Carlton et al., 2008)) 27 P. knowlesi (Sargeant et al., 2006); PHIST proteins <br />                                            identified P. cynomolgi proteomic studies (Akinyi, Korir Galinski, <br />                                            unpublished data). remains determined genes representing members                                            PHIST family expressed species, stage(s),                                            different family members quite different similar functions. BLAST searches <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                           Page 10 <br />  <br />  <br />                                            identified phist gene family member Plasmodium species closely <br />                                            related pvphist/cvc-8195 pcyphist/cvc-8195, comparative strong homology                                            selected gene set 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; regions encoding N- C-terminal sequences <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            supports proposed relationship orthologs. analyses                                            genes encodes central domain degenerate amino acid repeated motifs, remnants <br />                                            thereof (Fig. S3). repeated motifs evident P. falciparum, P. knowlesi, P yoelii <br />                                            P. berghei PHIST homologs, apparent ancestral remnants exist                                            PvPHIST/CVC-8195 PcyPHIST/CVC-8195 sequences, raising possibility                                            repeat motifs se functionally important proteins. contrary, <br />                                            PvPHIST/CVC-8195 PcyPHIST/CVC-8195 evolved harness amino acid <br />                                            sequences critical structure function proteins. Repeated <br />                                            motifs previously noted main characteristic PHIST family members, <br />                                            aside mentioned brief report (Chookajorn et al., 2006). variable                                            degenerate repeated motifs unique PHIST, recognized typical <br />                                            features various Plasmodium genes proteins (reviewed Anders et al., 1993)                                            intra-species sequence diversity high midst conserved <br />                                            sequences (e.g. Galinski et al., 1987; Cowman et al., 1985). <br />  <br />                                            P. falciparum, P. knowlesi, P. yoelii P. berghei produce CVCs host iRBC <br />                                            membranes, respective PHIST homologs reasonably expected                                            perform function(s) quite different PvPHIST/CVC-8195 PcyPHIST/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            CVC-8195. worth noting regard, PfPHIST-147 (gene ID: Pf08_0137),                                            associated P. falciparum translocon exported proteins (PTEX) <br />                                            complex  Koning-Ward et al., 2009). PTEX complex predicted transport <br />                                            proteins parasite PV membrane erythrocyte cytosol. possible <br />                                            homologous proteins, PHIST/CVC-8195, conserved PHIST domains,                                            different localizations course development species share common <br />                                            primary functions, example, serving protein chaperones. <br />  <br />                                            P. cynomolgi experimental model lends itself exploration open <br />                                            questions relating biogenesis function CVCs biology P. vivax.                                            example, caveolae tubules CVCs created result invagination                                            iRBC membrane numerous points?  CVCs develop novo <br />                                            biogenesis composite proteins, lipids vesicles process progressive <br />                                            pre-formed docking inner surface infected erythrocyte membrane? ring <br />                                            stage IFA patterns representing PHIST intriguing distinctive compared later <br />                                            stage parasites (Fig. 3A). &#226;&#8364;&#339;soccer ball&#226;&#8364;? pattern PHIST reactivity ring-stage <br />                                            IFAs suggestive semi-organized initial placement CVCs surface                                            iRBCs. pattern biologically relevant, parasite requires NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            organized scaffold selectively placed CVCs brace iRBC progressively <br />                                            continuing process destroying rebuilding RBCs natural cytoskeletal <br />                                            architecture accommodate increasing number CVCs. <br />  <br />                                            weak limited vitro cytoadherent characteristics noted P. <br />                                            vivax iRBCs (Carvalho et al., 2010), <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> evidence dominant cytoadhesion <br />                                            phenotype P. vivax P. cynomolgi, akin observed P. falciparum                                            attributed adhesion parasite&#226;&#8364;&#8482;s iRBC knobs variety host endothelial <br />                                            receptors. P. vivax paralogs var genes P. falciparum, known encode                                            highly cytoadherant variant antigens (Scherf et al., 2008), related SICAvar genes                                            P. knowlesi (al-Khedery et al., 1999; Korir et al., 2006), CVC conduit                                            implanting members variable P. vivax vir gene family (Fernandez-Becerra et al., <br />                                            2005) functional proteins membrane iRBC. CVC participate                                            regulate necessary metabolic physiological functions likely  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                              Page 11 <br />  <br />  <br />                                            primary purpose organelles; making strong potential targets <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> <br />                                            immune intervention. comparison, Hanssen colleagues reported 25 nm 80 nm <br />                                            diameter vesicles P. falciparum iRBC (Hanssen et al., 2008; Hanssen et al., 2010). 25 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            nm vesicles limited ordinary <span id='am-11' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipid</span> bilayer ET appeared free                                            cytoplasm associated Maurer&#226;&#8364;&#8482;s clefts iRBC membrane. contrast, 80 <br />                                            nm vesicles electron-dense coat covering 25 nm bilayer circumscribed aperture                                            typically number. electron dense &#226;&#8364;&#732;knob&#226;&#8364;&#8482; protrusions                                            predominant feature surface P. falciparum iRBCs, rare instances 80 nm <br />                                            vesicles appear fused iRBC membrane form caveolae, similar <br />                                            electron dense coat observed caveolae component CVCs. Sparse caveolae <br />                                            observed TEM surface P. knowlesi iRBCs (Aikawa et al., 1975). <br />                                            envision caveolae function distinct numerous <br />                                            CVCs P. vivax P. cynomolgi. <br />  <br />                                            Importantly, replica vivo transfection experiments support hypothesis                                            predominant 95 kDa CVC protein P. vivax P. cynomolgi, known PvPHIST/ <br />                                            CVC-8195 PcyPHIST/CVC-8195, essential parasite survival. P. cynomolgi <br />                                            parasites transfected construct targeting disruption pcyphist/ <br />                                            cvc-8195 gene selected vivo rhesus monkeys, parasites containing <br />                                            episomes mutated tgdhfr gene conferring pyrimethamine resistance survived.                                            parallel experiment showing transgene fact integrated P. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            cynomolgi genome particularly important, evidence                                            possible species. concluded stage, PHIST/CVC-8195 protein <br />                                            essential parasite. precise roles protein plays remains defined,                                            abundance prominent location suggests critical role. <br />  <br />                                            Aside PHIST/CVC-8195, proteins defined association                                            CVCs IFA IEM. include large (&gt;200 kDa) transmembrane protein <br />                                            (Barnwell Galinski, unpublished data), 70 kDa protein (Barnwell et al 1990) 86 <br />                                            kDa protein (Udagama et al., 1988). Caveolins flotillin-2 reported                                            present CVCs antibody reactivity (Bracho et al 2006). future investigations <br />                                            important determine composite proteins associated CVCs, <br />                                            define CVC architecture develops growth intra-erythrocytic <br />                                            parasite, identify biological biochemical processes serve parasitized <br />                                            cell. Given potential medicinal chemistry design novel drug compounds based                                            structure-associated relationships, exploration direction highly <br />                                            warranted develop new drugs active P. vivax. <br />  <br />                               Experimental Procedures <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Parasite materials <br />                                            Using standard procedures, cryopreserved reconstituted P. cynomolgi (Berok strain) <br />                                            ring iRBCs inoculated required splenectomized Macaca mulatta (rhesus) <br />                                            monkeys following approved protocols Institutional Animal Care Use <br />                                            Committee Emory University. approximate 5% target parasitemia, blood                                            collected passed glass beads cellulose CF11 columns remove platelets <br />                                            white blood cells, respectively, enriched trophozoite schizont-iRBCs                                            centrifugation 52% 48% Percoll cushion, respectively, described (Barnwell et <br />                                            al., 1990; Galinski et al., 1992). <br />  <br />                                            P. cynomolgi genomic DNA (gDNA) prepared schizont-stage parasites using                                            QIAamp DNA Blood Extraction kit (Qiagen) following manufacturer&#226;&#8364;&#8482;s instructions.                                            schizonts obtained fresh infected blood, described  reconstituted <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                         Page 12 <br />  <br />  <br />                                            ring-stage iRBCs matured schizont stage short-term culture described <br />                                            (Barnwell et al., 1999). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Monoclonal antibodies <br />                                            Monoclonal antibodies 2H8.E10, 2H12.B4, 4C12.G4 1H4.B6 battery                                            mAbs raised P. vivax schizont merozoite proteins characterized                                            previously described (Barnwell et al., 1990; Matsumoto et al., 1988; Barnwell, 1986). Total <br />                                            IgG purified using Protein MAPS affinity isolation (BioRad) <br />                                            following manufacturer&#226;&#8364;&#8482;s protocol. <br />  <br />                               Production recombinant protein antisera <br />                                            Pvphist/cvc-8195 gene-specific primers PvPHIST/CVC-8195F (5&#226;&#8364;&#178;tat gga tcc ATG AGT <br />                                            CCC TGC AAC ATC) PvPHIST/CVC-8195R (5&#226;&#8364;&#178;ata ctc gag TTA GAG TTT GCT GTG <br />                                            TTT CT) used amplify length gene standard PCR conditions <br />                                            following manufacturer&#226;&#8364;&#8482;s protocol (Calbiochem). amplicon cloned                                            expression vector pGEX 4T-2 (GE Healthcare) using BamHI XhoI restriction sites. <br />                                            Positive clones confirmed ABI 3100 DNA sequencer. clones  <br />                                            transformed E. coli BL21 StarTM (DE3) cells (Invitrogen) protein expression. <br />                                            Soluble protein purified using <span id='am-24' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-25' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span>Glutathione</span> SepharoseTM 4B (GE Healthcare) slurry <br />                                            according manufacturer&#226;&#8364;&#8482;s protocol. Recombinant PvPHIST-8195 (rPvPHIST/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            CVC-8195) inoculated New Zealand White Rabbit (Covance) production                                            polyclonal antiserum, rabbit anti-rPvPHIST/CVC-8195. <br />  <br />                               Indirect Immunofluorescence Assays (IFA) western immunoblots <br />                                            cross-reactivity mAbs raised P. vivax tested IFA air-dried, cold <br />                                            acetone-fixed films RBCs infected P. cynomolgi ring, trophozoite schizont <br />                                            stage parasites. Following incubation primary antibodies, expression detected <br />                                            using affinity-purified goat IgG anti-mouse conjugated Alexa Fluor 488 (Invitrogen)                                            secondary antibodies. mAbs tested 1:100, 1:200, 1:400 1:800 dilutions                                            phosphate-buffered saline (PBS; Lonza) containing 0.2% bovine serum albumin (BSA; <br />                                            Sigma). rabbit anti-rPvPHIST/CVC-8195 IFA reactivity tested dilutions 1:100, <br />                                            1:200, 1:400 1:800, followed anti-rabbit antibodies conjugated Alexa Fluor 488 <br />                                            (Invitrogen) 1:200 dilution. parasite nuclei visualized DAPI, contained                                            ProLong Gold Antifade Reagent (Invitrogen). slides examined Zeiss <br />                                            Imager.Z1 Axioskope 2 microscope filters appropriate fluorescent dyes,                                            images merged. <br />  <br />                                            western immunoblot analyses, P. cynomolgi trophozoite extracts NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            electrophoretically separated SDS-polyacrylamide gels transferred 0.2 &#206;&#188;m <br />                                            nitrocellulose membrane (Schleicher <span id='am-17' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; Schuell) probed mAbs diluted 1:1000. <br />                                            Membranes incubated corresponding alkaline phosphatase-conjugate                                            secondary antibody (Promega) immunoreactivity detected incubating NBT/ <br />                                            BCIP substrate (Promega). <br />  <br />                               Immunoprecipitation P. cynomolgi extracts monoclonal antibodies <br />                                            P. cynomolgi-iRBC extracts prepared follows: Ice-cold 1X NET/1% NP-40 <br />                                            containing protease inhibitors (10 mM EDTA-Na2; 1 mM PMSF; 0.1 mM TPCK, <br />                                            TLCK, Leupeptin, Chymostatin, Antipain 3,4-DCI; 10 &#206;&#188;M EP-64 1 &#206;&#188;M Pepstatin <br />                                             Sigma) added P. cynomolgi-infected erythrocyte pellet, extracted 30 min <br />                                            ice occasional vortexing. samples transferred pre-chilled <br />                                            microcentrifuge tubes spun 20,817 &#195;&#8212; g 20 min 4&#194;&#176;C. resulting NP-40 <br />  <br />  <br />                                                Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                         Page 13 <br />  <br />  <br />                                            extracts transferred stored, pellet extracted 1% SDS occasional <br />                                            vortexing 10 min room temperature. sample centrifuged 20,817 &#195;&#8212; g <br />                                            20 min room temperature. SDS extracts combined NP-40 extracts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            use immunoprecipitations. <br />  <br />                                            &#206;&#188;l rProtein G agarose suspension (Invitrogen) incubated                                            200 &#206;&#188;g P. vivax mAb overnight 4&#194;&#176;C, rotating.  extracts                                            preclarified rProtein G agarose overnight 4&#194;&#176;C, rotational shaker. extracts <br />                                            separated preclarifying beads. Dimethyl pimelimidate (Sigma)                                            added mAb/rProtein G mix final concentration 20 mM incubated 6 h,                                            4&#194;&#176;C, rotating. uncoupled mAb removed pulse spin, mixture washed <br />                                            NETT (150 mM NaCl/5 mM EDTA/50 mM Tris/0.5% Triton X-100). milliliter                                            combined extract added coupled mAb/rProtein G mix incubated <br />                                            overnight. beads washed twice NETT, twice NETT/0.5 M NaCl <br />                                            NETT/0.05% SDS. samples resuspended 2X SDS-PAGE loading <br />                                            buffer resolved 4&#226;&#8364;&#8220;20% gradient SDS-PAGE gels (Bio-Rad). <br />  <br />                               Mass spectrometric analysis immunoprecipitated proteins <br />                                            resolving extracts membrane samples 4&#226;&#8364;&#8220;15% SDS-PAGE gradient gels,                                            gels stained colloidal Coomassie blue (Imperial Protein Stain; Thermo Scientific). <br />                                            Gel slices excised, destained, dried, processed described (Korir et al., <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            2006). Briefly, gel pieces digested trypsin (Sigma) resulting peptides <br />                                            extracted trifluoroacetic acid (Sigma). samples desalted                                            concentrated using ZipTip pipette tips (Millipore). Cleaned peptides analyzed                                            reverse-phase LC-MS/MS (Peng et al., 2001) using LTQ-Orbitrap mass spectrometer <br />                                            (Thermo Finnigan). reverse database strategy using SEQUEST algorithm                                            implemented evaluate false discovery rate; matched peptides filtered according <br />                                            matching scores remove false matches reverse database (Peng et al., 2003). <br />                                            proteins matched peptides accepted improve <br />                                            confidence identification. peptides searched NCBI <br />                                            database, searches limited Plasmodium results. <br />  <br />                               Bioinformatics peptide gene analyses <br />                                            Protein matches acquired LC-MS/MS searched Plasmodium (PlasmoDB; <br />                                            www.plasmodb.org) general (National Center Biotechnology Information; <br />                                            http://www.ncbi.nlm.nih.gov/sites/entrez?db=Protein&amp;itool=toolbar) databases determine <br />                                            homologous genes. Signal peptide cleavage sites transmembrane domains                                            predicted SignalP V3.0 (http://www.cbs.dtu.dk/services/SignalP/) TMpred <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            (www.ch.embnet.org/software/TMPRED_form.html) software, respectively. Multiple <br />                                            alignments PHIST protein sequences generated using ClustalW2 <br />                                            (http://www.ebi.ac.uk/Tools/clustalw2/index.html) MacVector v7.2.3 software.                                            GOR4 program (http://npsa-pbil.ibcp.fr/cgi-bin/npsa_automat.pl?page=npsa_gor4.html; <br />                                            Combet et al., 2000) used predict secondary structure using amino acid sequences <br />                                            PHIST family members. <br />  <br />                               Polymerase Chain Reaction amplification cloning pcyphist/cvc-8195 gene <br />                                            Polymerase Chain Reaction (PCR) amplification pcyphist/cvc-8195 gene P. <br />                                            cynomolgi gDNA performed using combination pvphist-8195 related <br />                                            pkphist-105 (Fig. 2, S1, S2) gene-specific primers Expand High Fidelity                                            (Roche) kit manufacturer&#226;&#8364;&#8482;s protocol. PCR products purified using                                            Qiaquick purification (Qiagen), cloned pCR2.1 vector (Invitrogen)                                            sequenced using ABI Prism BigDye Terminator v3.0 cycle sequencing kit (Applied <br />  <br />                                                Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                          Page 14 <br />  <br />  <br />                                            Biosystems). gene walking, subsequent design pcyphist/cvc-8195-specific <br />                                            primers, 2,200 nucleotides pcyphist/cvc-8195 gene sequenced                                            verified. sequence 3&#226;&#8364;&#178; end pcyphist/cvc-8195 stop codon, degenerate <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            reverse primers designed based 3&#226;&#8364;&#178; UTR sequences pvphist/cvc-8195                                            pkphist-105. primer pair PcyPHIST/CVC.2023.F (5&#226;&#8364;&#178;GAT GCA AGA GTA CAT TAT <br />                                            GC) PcyPHIST/CVC.3&#226;&#8364;&#178;UTR.R (5&#226;&#8364;&#178;CAA AA C) GTT CTC CTA TGA CG) amplified <br />                                            sequence stop codon. result confirmed sequencing entire gene <br />                                            using pcyphis/cvct8195-specific primers: <br />                                                PcyPHIST/CVC.1.F (5&#226;&#8364;&#178;ATG AGT CCC TGC AAC ATC), <br />                                                PcyPHIST/CVC.263.R (5&#226;&#8364;&#178;CTC AGA GAG ATA TGC TCA AA), <br />                                                PcyPHIST/CVC.566.R (5&#226;&#8364;&#178;CAT CTC CTC CTC TTG CCA), <br />                                                PcyPHIST/CVC.762R (5&#226;&#8364;&#178;TCA GAG GGA TCG GTA TCG), <br />                                                PcyPHIST/CVC.1229.R (5&#226;&#8364;&#178;CAC CTC TTC CGT GGT ATT), <br />                                                PcyPHIST/CVC.2023.F (5&#226;&#8364;&#178;GAT GCA AGA GTA CAT TAT GC), <br />                                                PcyPHIST/CVC.2160.R (5&#226;&#8364;&#178;GAG TAT TGC ATA ATG TAC TC),                                                PcyPHIST/CVC.2433.R (5&#226;&#8364;&#178;TAC AAT TTA CTG TGT TTC TTC). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Transfection P. cynomolgi iRBCs <br />                                            Selection Cassette vector pBlueScript backbone containing pyrimethamine <br />                                            selection cassette inserted NdeI restriction site multiple cloning site                                            kindly provided Alan Thomas (van der Wel et al., 1997). vector includes                                            following gene fragments positioned head tail: 5&#226;&#8364;&#178; UTR sequence pbdhfr-ts, mutated <br />                                            tgdhfr-ts coding sequence, 3&#226;&#8364;&#178; UTR sequence pbdhfr-ts. total size 4,996 bp. <br />                                            mutations tgdhfr-ts coding sequence (Ser36Arg, Thr83Asn Phe245Ser) <br />                                            required confer resistance pyrimethamine confirmed end sequencing. <br />  <br />                                            P. cynomolgi phist/cvc-8195 knockout construct (pcy&#206;&#8221;phist/cvc8195 vector)&#226;&#8364;&#8221; <br />                                            following primer pairs designed amplify 600 bp 5&#226;&#8364;&#178; region pyphist/ <br />                                            cvc-8195  &#226;&#710;&#8217;555 45) 554 bp 3&#226;&#8364;&#178; region pcyphist/cvc-8195                                             693 1246): Pcyphist/cvc-8195 3FP, ATTT CCCGGG AGAATGTATGATGAAGAATA; <br />                                            Pcyphist/cvc-8195 3RP, TCCATATCAAGTCTTCCACCTCGT; Pcyphist/cvc-8195 5FP, <br />                                            TTAAGAGAGCCATCGATGCC; Pcyphist/cvc-8195 5RP, ATTT CCCGGG <br />                                            ATCATAGTAGTCATGGTTAC. Sma site required subsequent reactions                                            underlined. amplicons generated using KOD polymerase (Novagen). 5&#226;&#8364;&#178; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            3&#226;&#8364;&#178; fragments ligated using SmaI restriction site added reverse primer                                            5&#226;&#8364;&#178; region forward primer 3&#226;&#8364;&#178; region, amplified forward <br />                                            primer recognizing 5&#226;&#8364;&#178; region reverse primer recognizing 3&#226;&#8364;&#178; region.                                            amplicon representing combined 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; fragments inserted Sma site                                            pUC19 vector (New England Biolabs) make pUCpcyphist5&#226;&#8364;&#178;/3&#226;&#8364;&#178; construct. Ligation <br />                                            selection cassette pUCpcyphist/cvc5&#226;&#8364;&#178;/3&#226;&#8364;&#178; vector required addition                                            blunt restriction enzyme site selection cassette. Briefly, 24 bp adaptor carrying                                            PmeI/NdeI/PmeI restriction site sequence inserted EcoRV restriction site                                            pCR2.1. selection cassette released pBS vector using NdeI                                            subcloned Nde site pCR2.1 PmeI/NdeI/PmeI sequences cassette <br />                                            released digestion PmeI. final vector generated blunt end <br />                                            ligation PmeI released selection cassette pUC19pcyphist/cvc5&#226;&#8364;&#178;/3&#226;&#8364;&#178; plasmid <br />                                            digested SmaI. vivo selection. Sequential transfection experiments carried  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                          Page 15 <br />  <br />  <br />                                            animal experimental protocols presented approved Emory University&#226;&#8364;&#8482;s <br />                                            Institutional Animal Care Use Committee. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            experiment, donor rhesus macaque inoculated ~1.5 x108 P. cynomolgi <br />                                            (Berok) ring-stage iRBCs. blood sample drawn parasitemia reached &gt;4% <br />                                            majority iRBCs schizont stage. infected blood processed using <br />                                            standard procedures (Barnwell et al., 1999) mature schizonts separated using <br />                                            Percoll (Amersham) gradient 50%, washed RPMI resuspended cytomix <br />                                            (Kocken et al., 1999) 1.5 &#195;&#8212; 109 parasites ml&#226;&#710;&#8217;1. Electroporation performed                                            BioRad Gene pulser II (25 &#206;&#188;F, 200 Ohms 2.5 kV (Kocken et al., 1999; van der Wel et <br />                                            al., 1997) 0.4 ml iRBC suspension mixed 0.4 ml linearized plasmid DNA <br />                                            (0.1 mg) 0.4 cm cuvette room temperature. electroporated iRBCs                                            resuspended RPMI inoculated immediately recipient rhesus macaque. <br />                                            Parasitemias checked daily microscopic examination blood smears.                                            parasitemia rising, doses 1 mg kg&#226;&#710;&#8217;1 pyrimethamine (Sigma) administered, <br />                                            dosing times noted results section. pyrimethamine resistant <br />                                            parasitemia rising &gt;3% infected blood collected cryopreserved using <br />                                            standard procedures. animals treated standard curative regimen                                            chloroquine (15mg kg&#226;&#710;&#8217;1 IM &#195;&#8212; 3 days) terminate infections. <br />  <br />                               Transmission electron microscopy immuno-labeling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            P. cynomolgi trophozoite-iRBCs fixed incomplete RPMI-1640 containing 2% <br />                                            paraformaldehyde (PFA) permeabilized EqtII (Anderluh et al., 1996)                                            previously described (Jackson et al., 2007, Hanssen et al., 2008). samples refixed <br />                                            PBS containing 2% paraformaldehyde, blocked 3% BSA PBS incubated <br />                                            rabbit anti-rPvPHIST-8195 primary antibody PBS/3% BSA. washing,                                            cells incubated 6 nm gold-conjugated Protein (Aurion) according                                            manufacturer&#226;&#8364;&#8482;s instructions, washed  fixed 1% glutaraldehyde/0.5% <br />                                            PFA/0.1 M cacodylate buffer overnight. samples embedded 3% agarose <br />                                            rinsed 0.175 M cacodylate buffer. post-fixation 1% osmium tetroxide, <br />                                            cells stained  bloc&#226;&#8364;&#8482; 1% uranyl acetate, serially dehydrated                                            embedded LR White resin. samples sectioned 70 nm thickness                                            staining lead citrate uranyl acetate, observed 120kV 2010HC (Jeol, Japan) <br />                                            transmission electron microscope  La Trobe University EM Facility, Melbourne). <br />  <br />                               Electron tomography <br />                                            Tomography performed described previously (Hanssen et al., 2008). 200&#226;&#8364;&#8220;300 nm <br />                                            sections cut collected grid incubated fiducial gold particles. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            sections contrasted lead citrate uranyl acetate observed tilt <br />                                            series &#226;&#710;&#8217;69 degrees 69 degrees 1.5 degrees captured images                                            axis 3 degrees captured images second axis. Data                                            acquired accelerating voltage 200 kV using Tecnai G2 TF30 transmission electron <br />                                            microscope (FEI, Netherlands) Bio21 Institute electron microscopy facility <br />                                            (Melbourne). tilt images aligned tomograms generated rendered using                                            IMOD package (Kremer et al., 1996). <br />  <br />                               Nucleotide sequence accession number <br />                                            genomic sequence pcyphist-8195 submitted GenBank database                                            accession number JN636815. <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                                         Page 16 <br />  <br />  <br />                               Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Acknowledgments <br />                                            research supported National Institutes Health grants AI24710, AI096067, HL0788626                                            Yerkes National Primate Research Center Base grant RR-00165 awarded National Center Research <br />                                            Resources National Institutes Health. SA supported funds Australia Research <br />                                            Council Australian National Health Medical Research Council. <br />  <br />  <br />                               References <br />                                            Aikawa M, Hsieh CL, Miller LH. Ultrastructural changes erythrocyte membrane ovale-type <br />                                               malarial parasites. J Parasitol. 1977; 63:152&#226;&#8364;&#8220;154. [PubMed: 403266] <br />                                            Aikawa M, Miller LH, Rabbege J. Caveola--vesicle complexes plasmalemma erythrocytes <br />                                               infected Plasmodium vivax P cynomolgi. Unique structures related Schuffner&#226;&#8364;&#8482;s dots.                                               J Pathol. 1975; 79:285&#226;&#8364;&#8220;300. [PubMed: 50017] <br />                                            al-Khedery B, Barnwell JW, Galinski MR. Antigenic variation malaria: 3&#226;&#8364;&#178; genomic alteration <br />                                               associated expression P. knowlesi variant antigen. Mol Cell. 1999; 3:131&#226;&#8364;&#8220;141. <br />                                               [PubMed: 10078196] <br />                                            Anderluh G, Pungercar J, Strukelj B, Macek P, Gubensek F. Cloning, sequencing, expression NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               equinatoxin II. Biochem Biophys Res Commun. 1996; 220:437&#226;&#8364;&#8220;442. [PubMed: 8645323] <br />                                            Anders RF, McColl DJ, Coppel RL. Molecular variation Plasmodium falciparum: polymorphic <br />                                               antigens asexual erythrocytic stages. Acta Trop. 1993; 53:239&#226;&#8364;&#8220;253. [PubMed: 8100673] <br />                                            Atkinson CT, Aikawa M. Ultrastructure malaria-infected erythrocytes. Blood Cells. 1990; 16:351&#226;&#8364;&#8220; <br />                                               368. [PubMed: 2257318] <br />                                            Aurrecoechea C, Brestelli J, Brunk BP, Dommer J, Fischer S, Gajria B, et al. PlasmoDB: functional <br />                                               genomic database malaria parasites. Nucleic Acids Res. 2009; 37:D539&#226;&#8364;&#8220;543. [PubMed: <br />                                               18957442] <br />                                            Barnwell JW. Antigens Plasmodium vivax blood stage parasites identified monoclonal <br />                                               antibodies. Memorias Instituto Oswaldo Crus. 1986; 81:59&#226;&#8364;&#8220;61. <br />                                            Barnwell JW, Galinski MR, DeSimone SG, Perler F, Ingravallo P. Plasmodium vivax, P. cynomolgi, <br />                                               P. knowlesi: identification homologue proteins associated surface merozoites. <br />                                               Exp Parasitol. 1999; 91:238&#226;&#8364;&#8220;249. [PubMed: 10072326] <br />                                            Barnwell JW, Ingravallo P, Galinski MR, Matsumoto Y, Aikawa M. Plasmodium vivax: malarial <br />                                                 proteins associated membrane-bound caveola-vesicle complexes cytoplasmic cleft <br />                                                 structures infected erythrocytes. Exp Parasitol. 1990; 70:85&#226;&#8364;&#8220;99. [PubMed: 2403934] <br />                                            Bracho C, Dunia  Romano M, Raposo G, La Rosa M, Benedetti EL, Perez HA. Caveolins                                                 flotillin-2 present blood stages Plasmodium vivax. Parasitol Res. 2006; 99:153&#226;&#8364;&#8220;159. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 [PubMed: 16521037] <br />                                            Carlton JM, Adams JH, Silva JC, Bidwell SL, Lorenzi H, Caler E, et al. Comparative genomics                                                 neglected human malaria parasite Plasmodium vivax. Nature. 2008; 455:757&#226;&#8364;&#8220;763. [PubMed: <br />                                                 18843361] <br />                                            Carvalho BO, Lopes SC, Nogueira PA, Orlandi PP, Bargieri DY, Blanco YC, et al.                                                 cytoadhesion Plasmodium vivax-infected erythrocytes. J Infect Dis. 2010; 202:638&#226;&#8364;&#8220;647. <br />                                                 [PubMed: 20617923] <br />                                            Chookajorn T, Hartl DL. Position-specific polymorphism Plasmodium falciparum Stuttering motif <br />                                                 PHISTc PFI1780w. Exp Parasitol. 2006; 114:126&#226;&#8364;&#8220;128. [PubMed: 16674945] <br />                                            Coatney GR. simian malarias: zoonoses, anthroponoses,  J Trop Med Hyg. 1971; <br />                                                 20:795&#226;&#8364;&#8220;803. [PubMed: 5002245] <br />                                            Combet C, Blanchet C, Geourjon C, Deleage G. NPS@: network protein sequence analysis. Trends <br />                                                 Biochem Sci. 2000; 25:147&#226;&#8364;&#8220;150. [PubMed: 10694887] <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                                Page 17 <br />  <br />  <br />                                            Cowman AF, Saint RB, Coppel RL, Brown GV, Anders RF, Kemp DJ. Conserved sequences flank <br />                                                variable tandem repeats S-antigen genes Plasmodium falciparum. Cell. 1985; 40:775&#226;&#8364;&#8220; <br />                                                783. [PubMed: 3886159] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Koning-Ward TF, Gilson PR, Boddey JA, Rug M, Smith BJ, Papenfuss  et al. newly <br />                                                discovered protein export machine malaria parasites. Nature. 2009; 459:945&#226;&#8364;&#8220;949. [PubMed: <br />                                                19536257] <br />                                            Di Girolamo F, Raggi C, Bultrini E, Lanfrancotti  Silvestrini F, Sargiacomo M, et al. Functional <br />                                                genomics, new tools malaria research. Ann Ist Super Sanita. 2005; 41:469&#226;&#8364;&#8220;477. [PubMed: <br />                                                16569915] <br />                                            Fernandez-Becerra C, Pein O, Oliveira TR, Yamamoto MM, Cassola AC, Rocha C, et al. Variant <br />                                                proteins Plasmodium vivax clonally expressed natural infections. Mol Microbiol. <br />                                                2005; 58:648&#226;&#8364;&#8220;658. [PubMed: 16238616] <br />                                            Galinski, M.; Barnwell, J. Malaria Infections Non-Human Primates Model Systems                                                Research.  Abee, C.; Mansfield, K.; Tardif, S.; Morris, T., editors. Nonhuman Primates                                                Biomedical Disease. press <br />                                            Galinski MR, Arnot  Cochrane AH, Barnwell JW, Nussenzweig RS, Enea V.                                                circumsporozoite gene Plasmodium cynomolgi complex. Cell. 1987; 48:311&#226;&#8364;&#8220;319. [PubMed: <br />                                                3802196] <br />                                            Galinski MR, Barnwell JW. Plasmodium vivax: cares? Malar J. 2008; 7(Suppl 1):S9. [PubMed: <br />                                                19091043] <br />                                            Galinski, MR.; Dluzewski, AR.; Barnwell, JW. Malaria Merozoites Invasion Erythrocytes.  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                Sherman, IW., editor. Molecular Approaches Malaria. New York: ASM Press; 2005. p. <br />                                                113-168. <br />                                            Galinski MR, Medina CC, Ingravallo P, Barnwell JW. reticulocyte-binding protein complex                                                Plasmodium vivax merozoites. Cell. 1992; 69:1213&#226;&#8364;&#8220;1226. [PubMed: 1617731] <br />                                            Gardner MJ, Hall N, Fung E, White O, Berriman M, Hyman RW, et al. Genome sequence                                                human malaria parasite Plasmodium falciparum. Nature. 2002; 419:498&#226;&#8364;&#8220;511. [PubMed: <br />                                                12368864] <br />                                            Haase S, Koning-Ward TF. New insights protein export malaria parasites. Cell Microbiol. <br />                                                2010; 12:580&#226;&#8364;&#8220;587. [PubMed: 20180801] <br />                                            Hanssen E, Goldie KN, Tilley L. Ultrastructure asexual blood stages Plasmodium falciparum. <br />                                                Methods Cell Biol. 2010; 96:93&#226;&#8364;&#8220;116. [PubMed: 20869520] <br />                                            Hanssen E, Knoechel C, Klonis N, Abu-Bakar N, Deed S, LeGros M, et al. Cryo transmission X-ray <br />                                                imaging malaria parasite, P. falciparum. J Struct Biol. 2011; 173:161&#226;&#8364;&#8220;168. [PubMed: <br />                                                20826218] <br />                                            Hanssen E, Sougrat R, Frankland S, Deed S, Klonis N, Lippincott-Schwartz J, Tilley L. Electron <br />                                                tomography Maurer&#226;&#8364;&#8482;s cleft organelles Plasmodium falciparum-infected erythrocytes <br />                                                reveals novel structural features. Mol Microbiol. 2008; 67:703&#226;&#8364;&#8220;718. [PubMed: 18067543] <br />                                            Hiller NL, Bhattacharjee S, van Ooij C, Liolios K, Harrison T, Lopez-Estrano C, Haldar K. host- <br />                                                targeting signal virulence proteins reveals secretome malarial infection. Science. 2004; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                306:1934&#226;&#8364;&#8220;1937. [PubMed: 15591203] <br />                                            Jackson KE, Klonis N, Ferguson DJ, Adisa  Dogovski C, Tilley L. Food vacuole-associated lipid <br />                                                bodies heterogeneous lipid environments malaria parasite, Plasmodium falciparum. Mol <br />                                                Microbiol. 2004; 54:109&#226;&#8364;&#8220;122. [PubMed: 15458409] <br />                                            Kocken CH, van der Wel  Thomas AW. Plasmodium cynomolgi: transfection blood-stage <br />                                                parasites using heterologous DNA constructs. Exp Parasitol. 1999; 93:58&#226;&#8364;&#8220;60. [PubMed: <br />                                                10464040] <br />                                            Korir CC, Galinski MR. Proteomic studies Plasmodium knowlesi SICA variant antigens <br />                                                demonstrate relationship P. falciparum EMP1. Infect Genet Evol. 2006; 6:75&#226;&#8364;&#8220;79. <br />                                                [PubMed: 16376842] <br />                                            Kremer JR, Mastronarde DN, McIntosh JR. visualization dimensional image data <br />                                                using IMOD. J Struct Biol. 1996; 116:71&#226;&#8364;&#8220;76. [PubMed: 8742726] <br />                                            Maier AG, Cooke BM, Cowman AF, Tilley L. Malaria parasite proteins remodel host <br />                                                erythrocyte. Nat Rev Microbiol. 2009; 7:341&#226;&#8364;&#8220;354. [PubMed: 19369950] <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                               Page 18 <br />  <br />  <br />                                            Marti M, Good RT, Rug M, Knuepfer E, Cowman AF. Targeting malaria virulence remodeling <br />                                                proteins host erythrocyte. Science. 2004; 306:1930&#226;&#8364;&#8220;1933. [PubMed: 15591202] <br />                                            Matsumoto Y, Aikawa M, Barnwell JW. Immunoelectron microscopic localization vivax malaria <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                antigens clefts caveola-vesicle complexes infected erythrocytes. J Trop Med <br />                                                Hyg. 1988; 39:317&#226;&#8364;&#8220;322. [PubMed: 3056054] <br />                                            Matsumoto Y, Matsuda S, Yoshida Y. Ultrastructure human erythrocytes infected Plasmodium <br />                                                ovale. J Trop Med Hyg. 1986; 35:697&#226;&#8364;&#8220;703. [PubMed: 3728792] <br />                                            Mueller  Galinski MR, Baird JK, Carlton JM, Kochar DK, Alonso PL, del Portillo HA. Key gaps                                                knowledge Plasmodium vivax, neglected human malaria parasite. Lancet Infect Dis. 2009; <br />                                                9:555&#226;&#8364;&#8220;566. [PubMed: 19695492] <br />                                            Pain  Bohme U, Berry AE, Mungall K, Finn RD, Jackson AP, et al. genome simian                                                human malaria parasite Plasmodium knowlesi. Nature. 2008; 455:799&#226;&#8364;&#8220;803. [PubMed: 18843368] <br />                                            Peng J, Elias JE, Thoreen CC, Licklider LJ, Gygi SP. Evaluation multidimensional chromatography <br />                                                coupled tandem mass spectrometry (LC/LC-MS/MS) large-scale protein analysis:                                                yeast proteome. J Proteome Res. 2003; 2:43&#226;&#8364;&#8220;50. [PubMed: 12643542] <br />                                            Peng J, Gygi SP. Proteomics: mixtures. J Mass Spectrom. 2001; 36:1083&#226;&#8364;&#8220;1091. [PubMed: <br />                                                11747101] <br />                                            Pfahler JM, Galinski MR, Barnwell JW, Lanzer M. Transient transfection Plasmodium vivax blood <br />                                                stage parasites. Mol Biochem Parasitol. 2006; 149:99&#226;&#8364;&#8220;101. [PubMed: 16716418] <br />                                            Price RN, Douglas NM, Anstey NM. New developments Plasmodium vivax malaria: severe disease <br />                                                rise chloroquine resistance. Curr Opin Infect Dis. 2009 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Rowe JA, Opi DH, Williams TN. Blood groups malaria: fresh insights pathogenesis                                                identification targets intervention. Curr Opin Hematol. 2009; 16:480&#226;&#8364;&#8220;487. [PubMed: <br />                                                19812491] <br />                                            Sargeant TJ, Marti M, Caler E, Carlton JM, Simpson K, Speed TP, Cowman AF. Lineage-specific <br />                                                expansion proteins exported erythrocytes malaria parasites. Genome Biol. 2006; 7:R12. <br />                                                [PubMed: 16507167] <br />                                            Scherf  Lopez-Rubio JJ, Riviere L. Antigenic variation Plasmodium falciparum. Annu Rev <br />                                                Microbiol. 2008; 62:445&#226;&#8364;&#8220;470. [PubMed: 18785843] <br />                                            Schuffner W. Beitrag zur kenntniss der malaria. Deutsches Archiv fur Klinische Medizin. 1899; <br />                                                64:428&#226;&#8364;&#8220;449. <br />                                            Spielmann T, Gilberger TW. Protein export malaria parasites: multiple export motifs add                                                multiple export pathways? Trends Parasitol. 2010; 26:6&#226;&#8364;&#8220;10. [PubMed: 19879191] <br />                                            Tilley L, Dixon MW, Kirk K. Plasmodium falciparum-infected red blood cell. Int J Biochem Cell <br />                                                Biol. 2011; 43:839&#226;&#8364;&#8220;842. [PubMed: 21458590] <br />                                            Tilley L, Sougrat R, Lithgow T, Hanssen E. twists turns Maurer&#226;&#8364;&#8482;s cleft trafficking P. <br />                                                falciparum-infected erythrocytes. Traffic. 2008; 9:187&#226;&#8364;&#8220;197. [PubMed: 18088325] <br />                                            Trager W, Jensen JB. Human malaria parasites continuous culture. Science. 1976; 193:673&#226;&#8364;&#8220;675. <br />                                                [PubMed: 781840] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Udagama PV, Atkinson CT, Peiris JS, David PH, Mendis KN, Aikawa M. Immunoelectron <br />                                                microscopy Schuffner&#226;&#8364;&#8482;s dots Plasmodium vivax-infected human erythrocytes. J Pathol. <br />                                                1988; 131:48&#226;&#8364;&#8220;52. [PubMed: 3281470] <br />                                            van der Wel  Tomas  Kocken CH, Malhotra P, Janse CJ, Waters AP, Thomas AW. <br />                                                Transfection primate malaria parasite Plasmodium knowlesi using entirely heterologous <br />                                                constructs. J Exp Med. 1997; 185:1499&#226;&#8364;&#8220;1503. [PubMed: 9126931] <br />                                            Waters AP, Higgins DG, McCutchan TF. Phylogeny malaria: useful study. Parasitol Today. <br />                                                1993; 9:246&#226;&#8364;&#8220;250. [PubMed: 15463768] <br />                                             World Malaria Report. Geneva: World Health Organization; 2010. p. xii <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                         Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 1. <br />                                            predominant 95 kDa CVC protein member PHIST protein superfamily.  <br />                                            Air-dried, acetone-fixed smears P. cynomolgi trophozoite-iRBCs incubated NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            nuclear DAPI stain (blue) P. vivax mAbs 4C12.G4 (1), 2H8.E10 (2), 2H12.B4 (3)                                            1H4.B6 (4) followed Alexafluor 488-conjugated goat anti-mouse IgG (Invitrogen) <br />                                            (green). Immunofluorescence detected using Zeiss Imager.Z1 microscope (panels 1&#226;&#8364;&#8220; <br />                                            3), Zeiss Axioskop 2 (panel 4). Scale bar: 4&#206;&#188;m. B. P. cynomolgi trophozoite-iRBC <br />                                            extracts separated SDS-PAGE reducing conditions 4&#226;&#8364;&#8220;20% gradient gel, <br />                                            transferred nitrocellulose membrane, probed P. vivax mAbs 2H12.B4, <br />                                            4C12.G4, 2H8.E10 1H4.B6. characteristic band 95 kD (Barnwell et al., 1990)                                            consistently obtained mAbs (lanes 1&#226;&#8364;&#8220;4), predominant breakdown product <br />                                            75 kDa extracts (lanes 3 4). Lanes 1 2, 3 4 represent different <br />                                            extracts. C. LC-MS/MS performed bands excised 4&#226;&#8364;&#8220;20% SDS-PAGE gels <br />                                            containing proteins P. cynomolgi trophozoite-iRBC extracts immunoprecipitated                                            mAbs 2H12.B4, 4C12.G4, 2H8.E10 1H4.B6; representative experiment using mAb <br />                                            1H4.B6 shown. proteins identified LC-MS/MS noted PHIST/CVC-8195                                            IgG. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                      Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 2. <br />                                            Protein structure sequence identity PcyPHIST/CVC-8195 homologs.                                             schematic represents PvPHIST/CVC-8195, PcyPHIST/CVC-8195, PkPHIST-105                                            PfPHIST-147 proteins, showing number amino acids main features indicated. <br />                                            B C. N- C-terminal amino acid sequence alignments generated using ClustalW. <br />                                            Positions identity species shown dark gray; hydrophobic <br />                                            sequence marked brown line; putative PEXEL motif boxed red;                                            conserved tryptophans denoted green triangle. regions predicted using <br />                                            GOR4 algorithm (Combet et al., 2000) form alpha helices shown blue <br />                                            boxes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                        Page 21 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 3. <br />                                            PcyPHIST/CVC-8195 expressed ring, trophozoite schizont stages                                            development. <br />                                             IFA tests. Air dried acetone-fixed smears ring, trophozoite schizont-stage P. <br />                                            cynomolgi-iRBCs incubated rabbit anti-rPvPHIST/CVC-8195 serum mounted <br />                                            Prolong Gold Antifade Reagent containing nuclear DAPI stain (Invitrogen; blue). <br />                                            Positive antibody reactivity detected using Alexa Fluor 488-conjugated anti-rabbit IgG <br />                                            (Invitrogen, green). ring-stage panel illustrates &#226;&#8364;&#339;soccer ball&#226;&#8364;? pattern PHIST <br />                                            reactivity; white arrows point blackened areas lacking fluorescence. Immunofluorescence <br />                                            detected using Zeiss Imager.Z1 microscope. Scale bar: 0.4&#206;&#188;m. B. Western <br />                                            immunoblot. SDS detergent extracts 1 &#195;&#8212; 108 P. cynomolgi ring, trophozoite                                            schizont stage iRBC pellets diluted 1:100 sample buffer subjected SDS- <br />                                            PAGE non-reducing conditions 4&#226;&#8364;&#8220;20% gradient gel. Rabbit anti-rPvPHIST/ <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            CVC-8195 used 1:1000 dilution. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                         Page 22 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig 4. <br />                                            Transmission EM intact permeabilized P. cynomolgi iRBCs.  intact iRBC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            showing CVC surface (arrow); note presence mitochondrion (m). B.                                            EqtII permeabilized iRBC section showing CVC long extensions. C. Low <br />                                            magnification image showing section EqtII permeabilized iRBC showing                                            presence multiple CVCs  indicated arrows), clefts (C) parasite (P). D-F. <br />                                            Immunolabeling (denoted arrowheads) CVCs EqtII permeabilized iRBCs using <br />                                            mAb 4C12.G4 (D), rabbit anti-rPvPHIST/CVC-8195 (E) negative control (F). Scale <br />                                            bars: 200 nm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                            Page 23 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig 5. <br />                                            Electron tomography CVCs P. cynomolgi trophozoite-iRBC. Virtual sections                                            approximately 200 nm tomogram presented. image represents thickness <br />                                            2.7 nm spacing image 18 nm. P. cynomolgi trophozoite-iRBCs <br />                                            permeabilized EqtII prepared electron tomography. dual tilt series                                            collected 1.5&#194;&#176; axis 3&#194;&#176; second axis, used generate                                            3-D volume. Caveolar openings indicated arrowheads. RBC membrane                                            indicated open arrow, isolated vesicles diameters ranging 25 &#226;&#8364;&#8220; 45 nm <br />                                            noted black arrows. sequence images right left,                                            indicated directional arrow right panel. Scale bar 200 nm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                     Page 24 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 6. <br />                                            Immuno-electron tomography CVC P. cynomolgi trophozoite-iRBC. H. <br />                                            Equinatoxin II permeabilised P. cynomolgi trophozoite-iRBCs labeled                                            different antibodies  D, mAb 4C12.G4 E&#226;&#8364;&#8220;H, rabbit anti-rPvPHIST/CVC-8195). C <br />                                            E&#226;&#8364;&#8220;G. Selected virtual sections (20 nm) showing CVCs. D H. Segmentation models <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            created IMOD software. RBC membrane rendered blue, CVC                                            orange, gold particles yellow. CVC opening surface iRBC                                            indicated arrows  D,F,H). long tubular extension highlighted                                            arrowhead B extension shown arrowhead D. section                                            short tubular extensions highlighted arrowhead (G)                                            extension shown arrowhead H. gold particles concentrated                                            tubular extensions denoted fine arrows (D,H). Bar: 100 nm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                           Page 25 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Fig. 7. <br />                                             P. cynomolgi phist/cvc-8195 transfection experiments result retrieval episomes <br />                                            conferring resistance pyrimethamine, integration genome. Results                                            shown transfection experiments.  Schematic pcyphist/cvc-8195 target <br />                                            gene   related gene disruption vector construct pUC-19 backbone <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             . Red arrows represent location phist/cvc forward (FP) reverse (RP) <br />                                            oligonucleotide primers used diagnostic PCR amplifications. hatched boxes represent <br />                                            5&#226;&#8364;&#178; 3&#226;&#8364;&#178; ends pcyphist/cvc-8195 gene. pcyphist/cvc-8195 sequences flank                                            selection cassette, comprised P. berghei 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; UTRs mutated tgdhfr-ts <br />                                            gene confer resistance pyrimethamine. EcoR1 (R1) BamHI (HI) restriction enzyme <br />                                            sites used linearize construct release selection cassette transfection <br />                                            experiments. B. Using phist/cvc-FP phist/cvc-RP, pcyphist/cvc-8195 gene                                            amplified PCR genomic DNA purified wild type (Wt) transfected <br />                                            (Tx) parasites. C. Plasmid DNA bands 5.1 kb 3.6 kb corresponding gene <br />                                            disruption vector retrieved vivo selection rhesus monkeys, purification                                            DNA selected parasites, subsequent transformation E. coli DNA, plasmid <br />                                            purification E. coli, digestion R1 EcoRV (RV) restriction enzymes. <br />                                            D. IFA experiments normal speckled expression pattern PHIST/CVC Tx <br />                                            parasites. P. cynomolgi trophozoite-iRBCs shown rabbit anti-PHIST/CVC-8195 <br />                                            reactivity (green) DAPI nuclear stain (blue). Immunofluorescence detected using                                            Zeiss Imager.Z1 microscope. Scale bar: 4&#206;&#188;m. E. Western immunoblot showing rabbit anti- <br />                                            PHIST/CVC-8195 reactivity (95 kDa, arrow, associated breakdown products)                                            Wt Tx trophozoite-iRBCs. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Microbiol. Author manuscript; available PMC 2013 June 01. <br />                             Akinyi et al.                                                                                Page 26 <br />  <br />  <br />  <br />  <br />                                                                                       Table 1 <br />                            Identity PvPHIST/CVC-8195 PcyPHIST/CVC-8195 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Nucleotide   Amino acid <br />                             Global            83%           75% <br />  <br />                             5&#226;&#8364;&#178;/N-terminal     92%           93% <br />  <br />                             Central           74%           56% <br />  <br />                             3&#226;&#8364;&#178;/C-terminal     100%         100% <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Mol Microbiol. Author man <br /> </body></html>