<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, OCt. 1992, p. 4314-4326                                                                   Vol. 12,  10 <br /> 0270-7306/92/104314-13$02.00/0 <br /> Copyright &#194;&#169; 1992, American Society Microbiology <br />  <br />  <br />         RPC53 Encodes Subunit Saccharomyces cerevisiae RNA <br />             Polymerase C (III) Inactivation Leads                         Predominantly G1 Arrest <br />           CARL MANN,* JEAN-YVES MICOUIN, NUCHANARD CHIANNILKULCHAI, ISABELLE TREICH, <br />                                  JEAN-MARIE BUHLER, ANDRE SENTENAC <br />                 Service Biochimie et Genetique Moleculaire, Batiment 142, Centre d&apos;Etudes Saclay, <br />                                              F-91191 Gif-sur-Yvette, France <br />                               Received 4 1992/Returned modification 12 June 1992/Accepted 30 June 1992 <br />  <br />              RPC53 shown essential gene encoding C53 subunit specifically associated yeast RNA <br />            polymerase C (M). Temperature-sensitive rpc53 mutants generated showed rapid inhibition tRNA <br />            synthesis transfer restrictive temperature. Unexpectedly, rpc53 mutants preferentially arrested <br />            cell division G1 phase large, round, unbudded cells. RPC53 DNA sequence predicted code <br />            hydrophiic M,-46,916 protein enriched charged <span id='am-22' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> residues. carboxy-terminal 136 amino <br />            acids C53 significantly similar (25% identical amino acid residues) region human BN51 <br />            protein. BN51 cDNA originally isolated ability complement temperature-sensitive hamster cell <br />            mutant undergoes G1 cell division arrest, true rpc53 mutants. <br />  <br />    eukaryotic RNA polymerases complex enzymes                                     MATERIALS METHODS <br /> composed multiple, distinct subunits. Saccharomyces <br /> cerevisiae RNA polymerase C activity associated                    Strains media. yeast strains used study complex 13 different polypeptides ranging 10             described Table 1. CMY356 haploid meiotic segregant 160 kDa (4, 11, 13, 49, 56). subset yeast RNA                 obtained sporulation CMY242 transformed polymerase C subunits homologous eubacterial                    plasmid pEMBLYc32-RPC53. CMY396 derived RNA polymerase core enzyme. C160 C128 homolo- <br />                                                                           growing log-phase CMY356 cells yeast extract-peptone- <br />                                                                           dextrose (2%) (YPD) presence 4 ,ug acriflavine <br /> gous Escherichia coli 1 13&apos; subunits, respectively             ml generations. [rhoo] segregant identi- <br /> (1, 24). molecules subunit E. coli              fied inability use glycerol carbon source RNA polymerase. AC40 AC19 domain                          absence mitochondrial DNA determined fluores- <br /> similar functionally important domain subunit               cent staining 4&apos;,6&apos;-diamidino-2-phenylindole (DAPI)  (8). proposed copy AC40                  Fig. 7E). CMY397 mitotic segregant CMY396 AC19 RNA polymerases C functionally homol-                    pEMBLYc32-RPC53 lost. <br /> ogous ao homodimer E. coli RNA polymerase.                   Yeasts cultivated YPD synthetic complete <br /> homodimer B44.5 subunit likely represent               medium (50). <br /> homolog B enzyme (29, 30). largest subunits                   DNA sequencing RPC53. sequence  Fig. 2) al homologs probably perform                   2,414-bp RPC53 HindIII-NmI fragment (Fig. 1C) functions polymerase assembly basic catal-                determined subcloning portions fragment ysis transcription homologs studied            M13 Bluescript (Stratagene) vectors orien- <br /> eubacterial enzyme. contrast core subunits,              tations. DNA sequence fragments deter- function assigned remaining                mined dideoxynucleotide method modified T7 <br /> small subunits. subunits (ABC27, ABC23,                     DNA polymerase (Sequenase; United States Biochemical <br /> ABC14.5, ABC10a, ABC10,B) shared                      Corp.). described Results, unexpectedly ob- nuclear polymerases probably contribute               served 2.4-kb RPC53 HindIII-NruI fragment cloned <br /> common eukaryotic  possibly archaebacterial) core                  AB320 yeast strain (38) complement zyme (4, 56). small subunits specifically associated             rpc53::HIS3-2 lethal disruption cloned pUN20 RNA polymerase C (C82, C53, C34, C31)                       (TRPI SUP11 ARSI CEN4) vector (10), cloned <br /> destined perform functions specific transcription           YCp5O (URA3 ARS1 CEN4) (46). eventually ascribed <br /> polymerase. C82 (6), C34 (52), C31 (37) proteins                  result inhibitory plasmid context effect YCp5O shown necessary yeast cell viability              expression RPC53.  pUN20 carries synthesis tRNA RNA polymerase C.                suppressor tRNA gene, YCp5O does  report, C53 performs essential cellular              initially wanted exclude possibility RPC53 <br /> function required tRNA synthesis. Furthermore,                     gene obtained AB320 strain contained nonsense <br /> report sequence similarity C53 BN51                     mutation. cloned directly corresponding 2.4-kb <br /> protein encode human homolog C53.                         HindIII-NruI RPC53 fragment strain LL20 (39)                                                                           vector pEMBLYc32 (37). RPC53 DNA sequences                                                                           strains LL20 AB320 determined. Shown                                                                           Fig. 2 DNA sequence strain LL20.                                                                           sequence RPC53 gene strain AB320 did contain <br />   * <br />       Corresponding author.                                               nonsense mutations, did number sequence <br />                                                                    4314 <br />                                    J <br /> VOL. 12, 1992                                                         YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST                               4315 <br />  <br />                                                                       TABLE 1. Yeast strains used <br />   Strain                                                                                                      Genotype <br /> CMY214 .....               trpl Al/trpl Al ura3-52/ura3-52 his3A200Ihis3A200 lys2-801/lys2-801 ade2-101/ade2-101 canl/+ <br /> CMY242 .....               CMY214 rpcS3::HIS3-1/+ <br /> CMY274 .....               trplAl his3A200 lys2-801 ade2-101 canl RPC53::pEMBLYi32(URA3) <br /> CMY292 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3::HIS3-l/pUN20-RPC53 <br /> CMY356 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3:JHIS3-l/pEMBLYc32-RPC53 <br /> CMY370 ......              CMY214 rpcS3:HIS3-2/+ <br /> CMY381 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3:1HIS3-2/pEMBLYc32-rpcS3-ts28 <br /> CMY383 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpc53::HIS3-2/pUN20-RPC53 <br /> CMY384 .....               trplAl his3 A200 lys2-801 ade2-101 canl rpcS3::HIS3-2/pEMBLYc32-RPC53 <br /> CMY323a ..... (RY262) rpbl-l ura3-52 <br /> CMY388b ..... cdc9 his7 leu2 canl ura3 hom3 sap3 <br /> CMY396 .....               trplAl his3A200 lys2-801 ade2-101 canl rpcS3::HIS3-1 [rho&#194;&#176;]/pEMBLYc32-RPC53 <br /> CMY397 .....               trplAl his3A200 lys2-801 ade2-101 canl rpc53::HIS3-1 [rho&apos;] <br /> SC12 .....                 leu2Al trplAl his3A200 lys2-801 ade2-101 canl rpc53::HIS3-2IpEMBLYc32-RPCS3 <br />  gift Rick Young. <br />   b   gift Lee Hartwell. <br />  <br />  <br /> differences relative RPC53 sequence LL20.                                      ARS fragmentation vector transformants carried chro- substitutions codon positions hav-                                    mosome 4 derivative 200- 250-kb deletion, ing effect predicted C53 amino acid sequence:                                        opposite orientation, CEN vector transformants <br /> C303T, A570G, C633T, A885G, A984G  numbering                                           displayed chromosome fragment 200 250 kb. Southern <br /> scheme refers shown Fig. 2). substitutions                                      blotting analysis gel showed HO gene leading conservative amino acid changes                                       hybridized 200- 250-kb chromosome fragment. predicted C53 proteins: G417C leading substitution                                  results indicated RPC53 gene 200 <br /> E139D, C833T leading substitution A276V.                                        250 kb left end chromosome 4. sequenc- consequential sequence differences                                           ing RPC53 gene, noticed presence rare NotI <br /> deletion 6 bp, 793 798, leading deletion                                     sequence coding region. NotI-SfiI map <br /> amino acids (GlyLeu, 255 256), inversion 4 bp,                                   S. cerevisiae genome available (32), 858 861, leading change residues, LysArg                                   attempted identify RPC53 NotI site. (286 287) AsnAla. variations represent                                        NotI sites 250 kb left arm chromo- <br /> sequence polymorphisms effect function                                       4. probed membrane containing yeast chromo- C53 protein RPCS3 genes LL20 AB320                                         somal DNA separately digested NotI Sfil sepa- equally capable complementing rpc53::HIS3-2                                         rated orthogonal-field-alternation gel electrophoresis mutation cloned pUN20 pEMBLYc32 vectors.                                            localize RPC53 NotI site  Fig. 4A). blot    Physical mapping RPCS3. initially mapped                                          generous gift Phil James, digested yeast chromo- <br /> RPCS3 gene left arm chromosome 4                                              somal DNA situ blocks agarose (24). hybridiza- <br /> chromosome fragmentation method (62). RPCS3 frag-                                           tion pattern obtained probing RPC53 PvuII- <br /> ment cloned orientations ARS fragmen-                                       XbaI 1-kb fragment spanned NotI site conclusively <br /> tation vector YCF3 CEN fragmentation vector                                            identified RPC53 NotI site sites <br /> YCF4. linearization plasmids junction                                      left end chromosome. places RPC53 RPCS3 vector sequences, stable transfor-                                       gene 210 kb chromosome 4 left telomere, good <br /> mants diploid yeast strain obtained. Orthogonal-                                     agreement estimation chromosome frag- <br /> field-alternation gel electrophoresis (OFAGE) chromo-                                       mentation method. <br /> somes transformants showed orientation,                                         Genetic mapping RPC53. knowledge  <br />                                                                          X gtll-RPC53 <br />                                                                        IacZ             p- <br />  <br />  <br />                                                                                 R                             R       R <br />  <br />                                          B&apos; R             Sn   RH P       VRG G <br />                                                                                        X INr                R   G <br />                                                                                                                                                                                                                                             SnRv                  B&apos; <br />                                                                                                  L~~~~                                                                                                                                               1 kb <br />                                                                                                                           i_             <br />  <br />                                                 H   PNc          ATG V          RG   G    NIh N Ns X     Nr <br />                                                                                                                                0.5 kb <br />                                   r-                                                                                                                              r         .        .           1 ---I1                                                                                                     S                TAA <br />    FIG. 1. Restriction maps cloned RPCS3 fragments.   Agtll-RPC53 clone obtained immunoscreening anti-C53 immunoglob- <br /> ulins (45). solid portion represents RPCS3 coding sequences fused carboxy terminus p-galactosidase. (B) Genomic fragment <br /> containing complete RPC53 gene obtained screening YRp7-Sau3A yeast genomic DNA bank (38) EcoRI DNA insert Agtll-RPC53. (C) 2,414-bp RPCS3 HindIII-NnI fragment DNA sequence determined  Fig. 2). B&apos;, BamHI-Sau3A <br /> junction; G, BglII; H, HindIII; N, NaeI; Nc, NcoI; Nh, NMeI; Ns, NsiI; P, PstI; R, EcoRI; Rv, EcoRV; S, Sall; Sn, SnaBI; Ss, SstI; V, <br /> PvuII; X, XbaI. <br />  4316     MANN ET AL.                                                                                                                                                                                                                                                                                                                                                                            MOL. CELL. BIOL. <br />  <br />            aagcttccttttctatttccttagcagcttcgtcatcatctaaatcatcagccccataatattctccttttgtggacccccaggcattctcgttgtctaa -670 <br />            cataccattctcttcatcttcctcaggaagctggtcagtttctaagtttctaccaaatattttcttataggctgcagcaccatctagttcctcttcctct -570 <br />  <br />            tcttcttcctcatcttctct&apos;ttcatctattgactcatcatcttcatccatggctaagacttcttcctcatcttcgtcttctaacagagaatggtcatcat                                                                                                                                                                                                                                                                                            -470 <br />  <br />            ctttattactgtcgccaaaggtggattgtccaagtaaaaccttttcccttttagaagcaaaatcatcaacttcgtttaaaccatacggattaatctcatc -370 <br />                                     *                                   *                                       *                                       *.&lt;M                                    *                       RPG                                                                                                     RPC160 <br />            cccaacttcagaggtcttagttctgtttgagcctttgcgtacCATattattctatj= <br />                                                                  ggA tatgttttctcttcctccaaaactttttg= -270 <br />                                     *           PAC                                         PAC                 * <br />           AaaagcatcatgtagatgaQcag<span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>;t_                                                                                    cttcattattgaattttttttcattg cagaagcg cagtttgaaaatttttcatccgccctgcaaa -170 <br />  <br />            gtccaattttttgatcagcagaagataatgaccagagaattatggtattcaagagaacaattcaaaagaggacagctagaaagatattgaggtattt&amp;ata                                                                                                                                                                                                                                                                                            -70 <br />  <br />           Atttgctagactaaccgaaagtcgaagcgtaatcttgttttttattaagcgggatag cattcattaagtATGAGCAGTAATAAAGGAAATGGGCGCTTGC                                                                                                                                                                                                                                                                                              31 <br />                                                                                                                                                                                                                                                                                 M           S           S           N           K           G           N           G           R           L           P <br />  <br />            CATCATTAAAAGATTCCTCCTCCAATGGAGGAGGATCTGCCAAGCCCTCATTAAAGTTTAAACCAAAAGCAGTTGCAAGAAAGTCCAAGGAAAAAAGAGA 131 <br />              S L K D S S S N G G G S K                  S L K F K P K V R K S K E K R E <br />  <br />            AGCAGCTGCGTCCAAAGTAAAGCTAGAGGAGGAATCTAAGAGAGGTAATGACAAGAAACATTTCAATAACAAGAATAAAAGAGTAACCGGCGCTGGCGGC 231 <br />                                   S       K       V               K           L       E               E       E               S           K           R       G           N               D           K           K           H           F       N               N           K           N           K           R           V           T           G                     G           G <br />  <br />            CAGCAAAGGCGAATGGCCAAATACTTAAATAACACACACGTTATCTCTAGCGGTCCATTGGCGGCTGGGAACTTTGTAAGTGAGAAGGGTGATTTGAGAA 331 <br />            Q Q R R M K Y L N N T H V S S G P L G N F V S E K G D L R R <br />  <br />            GAGGATTCATCAAATCAGAAGGAAGCGGGTCATCTCTTGTGCAAAAGGGCCTAGAAACTATTGACAATGGTGCTGAGAGCTCTGAGAATGAGGCAGAAGA 431 <br />                 G       F                 K       S           E           G           S           G           S           S           L           V           Q           K           G           L           E           T                     D           N           G                     E           S           S           E           N           E                     E           D <br />  <br />            CGATGATAATGAAGGTGTAGCGTCCAAATCTAAGAAGAAGTTCAATATGGGAAAAGAATTCGAGGCACGCAATCTCATAGAGGACGAAGATGACGGCGAA 531 <br />             D       D       N           E       G           V                     S           K           S           K           K           K           F           N           M           G           K           E           F           E                     R           N           L                     E           D           E           D           D           G           E <br />  <br />            AGTGAAAAGAGCAGTGACGTCGACATGGATGACGAAGAATGGAGATCTAAACGGATTGAACAGTTATTCCCTGTGAGACCTGTCCGCGTAAGACACGAAG 631 <br />            S E K S S D V D M D D E E W R S K R E Q L F P V R P V R V R H E D <br />  <br />            ACGTTGAAACTGTCAAAAGAGAAATACAAGAAGCTCTTTCAGAAAAGCCAACTCGTGAACCAACCCCCAGTGTGAAGACAGAGCCTGTCGGAACCGGATT 731 <br />              V E T V K R E Q E L S E K P T R E P T P S V K T E P V G T G L <br />  <br />            ACAATCTTATTTGGAAGAAAGAGAAAGGCAAGTCAATGAGAAACTGGCAGATCTTGGACTTGGACTTGAAAAGGAGTTTCAATCGGTTGATGGGAAAGAA 831 <br />             Q S Y L E E R E R Q V N E K L D L G L G L E K E F Q S V D G K E <br />  <br />            GCGGCCGCTGAGTTGGAATTATTAAAACGTGATCATCAGCATATATTACGAAAACTAAAGAAAATGAATAATAAACCAGAAAGATTCATGGTATTCCAGT 931 <br />            E L E L L K R D H Q H L R K L K K M N N K P E R F M V F Q L <br />  <br />            TACCTACTAGGTTACCAGCTTTTGAAAGACCCGCTGTGAAAGAAGAAAAAGAAGACATGGAAACCCAGGCTAGCGACCCTTCAAAGAAGAAGAAGAATAT 1031 <br />                 P       T       R           L       P                     F           E           R           P                     V           K           E           E           K           E           D           M           E           T           Q                     S           D           P           S           K           K           K           K           N            <br />            TAAAAAGAAGGACACGAAGGATGCTTTGTCTACTAGAGAACTTGCCGGCAAGGTTGGGTCTATACGGGTTCACAAATCTGGAAAACTTTCCGTGAAAATT 1131 <br />             K K K D T K D L S T R E L G K V G S R V H K S G K L S V K  <br />            GGAAATGTGGTGATGGATATTGGCAAAGGTGCCGAAACCACATTTTTACATGATGTTATAGCATTAAGTATCGCTGATGATGCATCCTCAGCGGAACTTC 1231 <br />            G        N       V       V           M       D                     G           K           G                     E           T           T           F           L           H           D           V                               L           S                               D           D                     S           S                     E           L           L <br />  <br />            TAGGCCGTGTGGACGGTAA&apos;AATAGTAGTCACACCTCAAATCTAAtcgc <br />              G R V D G K V V T P Q               * <br />                                                              actcgc                                                                                                                                                                     atctgtcgagtatataaatgaatatacacagtcataaatacttcta 1331 <br />  <br />            gaacaaattacactaattaagatgcttagattcccattcaaaaggtactattgacgtcttgtcttctacaatttctcatcctctttgaacattggtaatg 1431 <br />            ttattctagttattattacccagtttcggaactttaggggagccacactttaaacaaaaagaagcgccataaccaccgtgaagtaatgagtatctcactt                                                                                                                                                                                                                                                                                             1531 <br />  <br />            tgataagaagacatttaagaacagcctgaaatatcaaaatcacggaattattaggttatggagagagatgaataccagttacccaactctcatgggaaga 1631 <br />            atactttcttatcg <br />    FIG. 2. Sequence 2,414-bp RPC53 HindIII-NruI (Fig. 1C) DNA fragment predicted C53 amino acid sequence. sequence assigned EMBL Data Library accession number X63501. asterisks -87 -89 major sites transcription <br /> initiation, asterisk 1446 major site transcription termination determined S1 nuclease mapping (35). underlined <br /> sequence KPSLK (amino acids 25 29) represents possible nuclear localization signal (16). RPG box [consensus sequence <br /> R(C AYCCRYNCAYY (61)] -314, PAC boxes [consensus sequence TG C)GATGAG (8)] -258 -246, 16-bp sequence <br /> (-290) similar sequence -45 RPC160 promoter (1), TATAA sequences -272 -73 RPC53 <br /> promoter region. sequences contribute transcriptional regulation RPC53 gene. Note divergently oriented open <br /> reading frame encoding 148 amino acids observed starting -325 (indicated &lt;M initiating codon figure). predicted protein sequence open reading frame composed 50% acidic amino acid residues. <br />  <br />  <br /> RPC53 200 kb left end chromosome 4,                                                                                                                                                           plasmid. 800-bp EcoRI-XbaI RPC53 fragment cloned sought genetic linkage mutations                                                                                                                                                              pEMBLYi32 (2), resulting plasmid cut <br /> region. RPC53 chromosomal locus marked                                                                                                                                                                 RPC53 sequences BglII target integration URA3 targeted integration pEMBLYi32-RPC53                                                                                                                                                                   homologous site lithium acetate transformation (20) <br />  VOL. 12, 1992                                        YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST                         4317 <br />  CMY215. integration pEMBLYi32(URA3)                  tained retransformation CMY383 tester strain <br /> RPC53 locus confirmed Southern blot analysis           purified mutant plasmid. <br /> transformant CMY274 used meiotic mapping experi-                  Continuous radioactive labeling protein RNA. <br /> ments. CMY274 (RPC53::URA3) crossed rpbl                       CMY397 rpc53::HIS3-1 [rho&apos; isogenic wild-type <br /> (CMY323) cdc9 (CMY388) temperature-sensitive mutant               CMY396 rpc53::HIS3-1 [rho ]/pEMBLYc32-RPC53 cells <br /> strains. sporulation resulting diploid strains, tetrad   grown 23&#194;&#176;C synthetic complete medium contain- <br /> analysis showed RPC53::URA3 lie 1.5 centimorgans              ing 10 ,ug <span id='am-34' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span>uracil</span> ml optical density 660 nm rpbl 4 centimorgans cdc9  Fig. 4B).                      0.1. [3H]uracil (50 Ci/mmol) [35S]methionine (500 Ci/ <br />    Disruption RPC53. pBR322-rpc53::HIS3-1 plasmid                mmol) added cultures concentration created subcloning 2.4-kb HindIII-NruI               1 0.5 p,Ci/ml, respectively. cultures transferred <br /> RPC53 fragment pBR322 substituting internal           38&#194;&#176;C 1 h addition radioactive precursors. <br /> 207-bp BglII RPC53 fragment 1.7-kb BamHI HIS3                Incorporation radioactivity macromolecules  Fig. <br /> fragment (54). Digestion plasmid NcoI NruI           6D) followed trichloroacetic acid precipitation: 4 ml <br /> liberates 3.6-kb rpc53::HIS3-1 fragment  Fig. 5)           cold 5% trichloroacetic acid added 1 ml culture, used gene disruption (47) copy RPC53            precipitate collected filtration glass fiber <br /> gene diploid yeast strain CMY214.                              filters extensive washing 5% trichloroacetic <br />    essentially complete deletion RPC53 gene             acid. filters washed ethanol, dried, subcloning 2.4-kb HindIII-NruI RPC53                placed liquid scintillation fluid simultaneous count- <br /> fragment YCp5O (46). digestion plasmid          ing tritium 35S. <br /> PvuII, short treatment nuclease Bal 31 performed              increase cell number optical density delete RPC53 coding sequences. followed                percentage unbudded cells monitored parallel <br />                                                                       unlabeled cultures  Fig. 6A, B, C). <br /> digestion XbaI, filling protruding ends          Labeling analysis small, stable RNAs. CMY396, <br /> Klenow DNA polymerase, recircularization plas-             CMY397, CMY381, CMY384 cells grown 23&#194;&#176;C <br /> mid T4 DNA ligase presence BamHI linker                synthetic complete medium containing 5 ,ug uracil oligonucleotides. recovery E. coli, plasmid            ml optical density (660 nm) 0.4. labeling, 20 containing deletion approximately 60 bp         ml cells transferred 37 38&#194;&#176;C 30 <br /> RPC53 coding sequence BamHI linker site              min, 3 h, 6 h  temperature time incubation deletion. size deletion estimated           different strains indicated Fig. 9). RNA gel migration restriction fragments. 1.7-kb BamHI              labeled adding culture [3H]uracil (50 Ci/mmol) HIS3 fragment inserted plasmid yield          concentration 15 ,uCi/ml 1 h. Labeled cells YCp5O-rpc53::HIS3-2 construction  Fig. 5). Digestion           centrifuged resuspended 5 ml extraction buffer (100 plasmid NcoI NruI liberates 2.6-kb                    mM Tris-HCl [pH 7.5], 250 mM NaCl, 5 mM EDTA, 0.5% <br /> rpc53::HIS3-2 fragment used delete             sodium dodecyl sulfate [SDS]) followed immediately chromosomal RPC53 genes CMY214 diploid yeast                   addition equal volume phenol. Small RNAs strain.                                                               extracted vigorous agitation emulsion 65&#194;&#176;C 1 <br />    vitro mutagenesis RPC53. pEMBLYc32-RPC53                 h. RNA aqueous phase precipitated plasmid converted single-stranded form superin-             addition 2 volumes ethanol. RNA pellet fecting E. coli JM107 cells containing plasmid          resolubilized 0.1 ml diethyl pyrocarbonate-treated <br /> R408 Fl helper bacteriophage (48). gapped heteroduplex              water, radioactivity incorporated determined <br /> (42) formed denaturation double-stranded            yield RNA estimated sample A260  pEMBLYc32 vector (digested sites used subclon-             A260 1 taken represent 40 ,ug RNA ml). RNA <br /> ing RPC53 fragment) presence single-stranded            (20 p,g) sample denatured 90% formamide <br /> pEMBLYc32-RPC53 DNA; followed period                 loading buffer electrophoresis 6% polyacryl- <br /> renaturation. resulting heteroduplex molecules com-           amide-7 M <span id='am-44' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Urea" datatype="xsd:string"></span>urea</span> x TBE (Tris-borate-EDTA) gel sepa- <br /> posed double-stranded vector sequences               rate small RNAs. gel briefly stained 0.5 <br /> RPC53 fragment remaining single stranded. Treatment                mg ethidium bromide ml visualize RNA heteroduplexes sodium bisulfite specifically            photographed fixed 7% <span id='am-39' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span>, treated <br /> stricts GC transition mutations RPC53                Amplify (Amersham) fluorography, dried. Flu- <br /> fragment. Gapped heteroduplexes incubated 4 M               orography -70&#194;&#176;C preflashed Kodak XAR <br /> sodium bisulfite 37&#194;&#176;C 10, 20, 40 min,               film. <br /> reactions stopped DNA purified previously                Flow cytometiry yeast cells. sample described (33). mutagenized heteroduplexes              analyzed, approximately 107 cells fixed 70% ethanol <br /> used transform directly E. coli ung strain BD1528 (42)         1 h. Cells washed 0.2 M repair single-stranded gap fix C U               Tris-HCl (pH 7.5) resuspended 0.1 ml transition mutations. Conditional rpc53 mutants             buffer containing 100 ,ug RNase  RNA digested screened plasmid-shuffle test. Mutagenized DNA               incubating 37&#194;&#176;C 2 h. Cells washed banks used transform yeast strain CMY383                  phosphate-buffered saline (PBS) resuspended 0.1 ml <br /> rpc53::HIS3-2/pUN20-RPC53, conditional mutants                    50 ,ug propidium iodide ml PBS 20 min. (cold heat sensitive) sought transformants            washed PBS, cells resuspended 5 ,ug unable grow restrictive temperature counters-         propidium iodide ml PBS concentration electing pUN20(SUP11)-RPC53 plasmid replica plat-              5 x 106 cells ml. Cells lightly sonicated ing transformants YPD plus 2.5 M ethylene glycol plates            observed fluorescence microscopy ensure specific <br /> (10). Mutations shown plasmid linked recov-             nuclear staining quantifying fluorescent emission ering pEMBLYc32-RPC53 E. coli showing                 OrthoDiagnostics Systems 2150 fluorescence-activated conditional mutant phenotype uniformly ob-               cell sorter (FACS). <br />  4318     MANN ET AL.                                                                                                                        MOL. CELL. BIOL. <br />  <br />    Expression functional testing BN51 protein yeast      R        Acidic Basic Residues C53 <br /> cells. galactose-inducible BN51 genes                                                                                                                                 100 <br />                                                                                                                                                                                                                     200 <br />                                                                                                                                                                                                                                                                                                             400 <br />                                                                                                                                                                      structed, entire coding sequence                           11I1 d III II III III                                                                                          AI portion sequence encoding 167 amino           B                       I1                0I 0                     I10 4I0 <br />                                                                                                                                              1 <br /> acids BN51 protein (starting internal methi- <br />                                                                                               100               200               300                            400 <br /> onine position 228 amino acid sequence [22]). BN51 sequences placed control                B <br /> GALIO promoter Fusionator multicopy plasmid                                                                                                                       2 <br /> generous gift Stephen Johnson, Department Genet-                                                                                                                    1 <br />                                                                                                                                                                            10 <br /> ics, University Washington). Fusionator 2 ,um-                                                                                                                 -1 <br /> based plasmid containing LEU2 selectable marker (26).                                                                                                                  -2 <br /> Oligonucleotides used amplify polymerase chain                                                                                                                  -3 <br />                                                                                                                                                                            -4 <br /> reaction BN51-coding sequences SalI-BamHI DNA <br /> fragments starting human BN51 cDNA clone (19). following primers used amplification       c <br /> entire BN51-coding region: HBN3M = TTCCGTCGACCAT <br /> GTCGGAAGGAAACGC (5&apos; oligonucleotide) HBN1V <br />                                                                         0 <br /> =  GGGGGGATCCTTACCGGTG&apos;TTGTGATCC (3&apos; oligo-                            -1 <br /> <span id='am-14' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span>). SalI-BamHI fragment encoding 167               -2 <br /> BN51 amino acids amplified using primers                    -3 <br />                                                                        -4- <br /> HBN2M (GGGGGTCGACCATGAAGGCTCCTCCCAAA <br /> GC [5&apos; oligonucleotide]) 3&apos; oligonucleotide <br /> (HBN1V)  Cloning SalI-BamHI fragments             FIG. 3.   Graphic display positions acidic   sites Fusionator polylinker placed       basic (B) residues C53 protein sequence. bars transcription coding sequences control      (acidic) row represent glutamate residues, lower bars GAL10 promoter. Polymerase chain reaction-derived             represent aspartate residues. B (basic) row, bars <br /> clones plasmids introduced            represent arginine, intermediate bars represent lysine, yeast strain SC12 lithium acetate transformation           small bars represent histidine residues. (B C) Kyte-Doolittle <br /> (20). SC12 [rpc53::HIS3-2IpEMBLYc32(URA3)-RPC53]              hydropathy plots C53 BN51 proteins, respectively. <br />                                                                   Positive values represent hydrophobic regions, negative values <br /> used test ability BN51 proteins complement       represent hydrophilic regions, window 11 amino acids complete deletion RPC53 gene. SC12 transformants         protein. Notice overall similarity profiles expressing BN51 BN51 carboxy-terminal sequences           proteins. graphics figure executed galactose-containing medium tested ability      DNA Strider program (34). <br /> grow absence pEMBLYc32(URA3)-RPC53 <br /> plasmid. Cells unable lose plasmid sensitive drug 5-fluoro-<span id='am-9' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orotic acid</span> (3). <br />    EMBL data library submission. RPC53 DNA se-                (Fig. 2). reading frame fused lacZ <br /> quence (Fig. 2) assigned accession number            original Xgtll clone determined sequencing X63501.                                                           fusion site phage DNA. difference                                                                   predicted molecular weight C53 observed                           RESULTS                                 SDS-polyacrylamide gel electrophoresis inherent <br />                                                                   property sequence posttranslational <br />    Cloning sequencing RPC53 gene. Xgt11            modifications yeast cells. factors clones potentially encoding RPC53 gene         contribute anomalous electrophoretic migration <br /> obtained screening expression library antibodies       RNA polymerase subunits (56). TACTAAC <br /> prepared C53 subunit (45). Temperature-induc-         splicing signal (64) DNA sequence, ible lysogens clones produced fusion protein      assume gene does contain intron. Northern <br /> 30 kDa larger 0-galactosidase reacted         (RNA) blot nuclease S1 protection analysis (35) indi- <br /> anti-C53 anti-,-galactosidase antibodies. Extracts         cated RPC53 expressed mRNA 1,500 <br /> bacteria producing fusion protein able bind          nucleotides initiation termination sites shown antibodies anti-C53 immunoglobulin preparation           Fig. 2. capable specifically inhibiting RNA polymerase          deduced amino acid sequence C53 subunit C transcription vitro (35), indicating        hydrophilic length, excepting carboxy- <br /> likely isolated C53-coding sequences. Restriction enzyme          terminal 50-amino-acid region shown hydropathy <br /> analysis showed clones contained yeast DNA         plot Fig. 3B. hydrophilicity accounted insert common EcoRI site ligated 3&apos; end      striking distribution charged residues (Fig. 3A). ami- <br /> lacZ gene Xgtll (Fig. 1A). fusion protein            terminal 125-amino-acid domain highly basic analysis indicated portion RPC53 coding        followed alternating clusters acidic basic residues. <br /> sequence isolated, complete gene sought          structural features suggest C53 subunit lies YRp7 yeast genomic DNA bank (38) colony                 surface RNA polymerase C extended conforma- <br /> hybridization Xgt1l yeast DNA insert. 7.7-kb           tion. suggestion consistent extreme suscep- <br /> genomic DNA fragment obtained contained              tibility C53 subunit proteolytic degradation (19, 49) <br /> complete gene (Fig. 1B). DNA sequence 2,414-bp         relative ease dissociated <br /> HindIII-NnuI fragment shown Fig. 1C determined.            polymerase (5, 63). C53 calculated isoelectric open reading frame 1,272 nucleotides corresponding       point 9.2. protein 424 amino acids (predicted Mr, 46916)         Sequence motifs potentially involved nuclear transport <br />  VOL. 12, 1992                                                               YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST                                  4319 <br />  <br />           Chrs Noti Sfi1                                                                             participate transport C53, asso- <br />         chr4- &lt;                      B                                                                 ciated RNA polymerase C subunits, nucleus. <br />                                       Genetic Interval PD NPD T SM <br />                                                                                                           Putative regulatory signals seen sequences <br />                                                                                                        flanking RPCS3 coding sequence (Fig. 2). 5&apos; non- <br />                                       RPC531URA31-rpb 1 66             0          2   1.5 <br />                                       (CMY274&apos;CMY323)                                                  coding region contains candidate TATA box sequences, <br />         620_ J <br />          kb       . -s <br />                   _.4 <br />                          _ <br />                                       RPC531URA3]-cdc9           58    0          5   4 <br />                                                                                                        16-bp sequence similar sequence                                       (CMY274 CMY388 <br />                                                    8                                                   RPC160 promoter, RPG box sequence, tandem <br />                              *:., <br />                                       cdc9-rpb                 37     0         4   5                repetition PAC box. RPG box upstream <br />                                                                                                        activating sequence genes (61), including        40kb-                                                                                           encoding proteins involved protein stable RNA syn- <br />                                                                                                        thesis. RPG sequence bound GRF1/RAPITUF <br />               C                                                                                        protein (61). PAC box sequence                              BA      CDC9- -:: RPCbS-           :-t RPB                              promoter region 11 genes examined <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> encoding <br />                                                                                                    subunits specific RNA polymerases   C (III)                                                                                                        exception RPCJ60 (8). <br />    FIG. 4. Physical genetic mapping RPC53.   S. cerevi-                                          Physical genetic mapping RPC53 gene. siae chromosomal DNA, (Chrs lane) digestion                                      RPC53 gene estimated lie 200 250 kb <br /> NotI SfiI  indicated), separated pulsed-field gel                                        left telomere chromosome 4 S. cerevisiae electrophoresis, DNA transferred nylon                                           chromosome fragmentation method mapping  membrane probed radioactively labeled 1-kb PvuII-XbaI                                       Materials Methods). presence rare NotI site RPC53 fragment (Fig. 1C). fragment contains sequences                                          RPCS3 gene allowed precise determination NotI site RPCS3 gene. size                                  physical map position. RPC53 DNA hybridized NotI hybridizing fragments identifies NotI site NotI sites left end chromosome 4 (32). (B) Table                                           bands 40 620 kb Sfi band 635 kb <br /> summarizing tetrad analysis indicating genetic linkage                                   (Fig. 4A). pattern identifies NotI site RPC53 URA3 plasmid integrated RPCS3 chromosomal locus                                         NotI sites left telomere chromo- <br /> cdc9 rpbl mutations. Shown number tetrads                                       4 Link Olson NotI-SfiI map yeast <br /> case markers segregated parental ditypes (PD),                                     genome (32) places 210 kb left end nonparental ditypes (NPD), tetratypes (T). genetic distance                                     chromosome. Genetic mapping URA3 insertion centimorgans (cM) calculated according equation cM                                       RPC53 locus showed RPC53 locus tightly linked = 50(T + 6 NPD)/(PD + NPD + T). (C) Diagram summarizing                                            CDC9 RPB1 (Fig. 4B). <br /> physical genetic mapping RPC53 gene.                                                          RPC53 encodes essential gene. different disruptions <br />                                                                                                        RPC53 gene constructed determine                                                                                                        gene essential. rpc53::HIS3-2 disruption, observed C53. sequence KPSLK (amino acids                                                  approximately 60 nucleotides RPC53 coding sequence <br /> 25 29) form KPXXK, serves class                                              substituted HIS3 DNA fragment (Fig. 5A). nuclear targeting signal S. cerevisiae (16).                                         diploid strain heterozygous rpc53::HIS3-2 substitu- <br /> basic clusters protein sequences resembling                                           tion gave rise sporulation tetrads containing nuclear targeting signals (9, 12). sequences                                           viable spores invariably auxotrophic  <br />  <br />                                      R                      H <br />                                                                             770 <br />                                                                                             ATG 130 <br />                                                                                                  v <br />                                                                                                         440 <br />                                                                                                               G AA_ <br />                                                                                                                    207 <br />                                                                                                                           G <br />                                                                                                                                495 <br />                                                                                                                                      TAA X   300 <br />                                                                                                                                                    N <br />                                                                                                                            1                                                                  RPC53 <br />                                                             II <br />  <br />                                                             ___I_ <br />                                                                                                 160            H <br />                                     rpc53::HIS3-2                        ____HIS3I <br />                                                                       _I__                                                                  II <br />  <br />  <br />  <br />  <br />                                                                                           ATG                 GB          GB         TAAI <br />                                     rpc53::HIS3-1           L                                                 H           H <br />                                                                                                                    HIS3 <br />  <br />  <br />  <br />                                                             B                                     E <br />  <br />  <br />  <br />    FIG. 5. Effect disrupting RPC53 gene.   Shown different RPC53 disruption constructions. rpcS3::HIS3-2 disruption <br /> deletes 60 bp RPC53 substitutes 1.7-kb BamHI HIS3 fragment (54). rpc53::HIS3-1 disruption replaces internal 207-bp RPCS3 BglII fragment 1.7-kb BamHI HIS3 fragment. Restriction sites defined legend Fig. 1. orientation HIS3 gene indicated. (B) Tetrad dissection asci obtained sporulation CMY370  rpcS3::HIS3-21+). viable spores obtained, auxotrophic histidine. absence viable rpcS3:  spore colonies <br /> suggests deletion RPCS3 creates recessive lethal mutation. (C) Tetrad dissection asci obtained sporulation CMY242  ax <br /> rpcS3::HIS3-1I+). percent spores gave rise colonies growing considerably slowly sister-spore colonies 10 days growth 23&#194;&#176;C. slow-growing cells proved genotype rpcS3: :HIS3-1 [rho-] temperature sensitive growth. <br />  4320     MANN ET AL.                                                                                           MOL. CELL. BIOL. <br />  <br /> histidine (Fig. 5B). Microscopic examination rpc53::       having achieved cell volumes larger normal (Fig. <br /> HIS3-2 spores showed germinated           7). <br /> gone cell division arresting unbudded cells.           generated rpc53 point mutants determine absence  haploid spores suggested             G1 arrest general property associated inactivation RPC53 deletion lethal. possibility tested          RPC53 subunit specific pheno- <br /> cloning HindIII-NruI fragment containing RPC53            type rpc53::HIS3-1 [rho&apos;] disruption. RPC53 gene <br /> gene (Fig. 1C) centromere vectors pUN20 (TRP1            pEMBLYc32 mutagenized vitro sodium <br /> SUP11) YCp50 (URA3) separately introducing           bisulfite, conditional mutants screened plasmid rpc53::HIS3-21+ diploid strain.       plasmid shuffling strain CMY383 (rpc53::HIS3-2 [rho+]/ <br /> sporulation, haploid rpc53::HIS3-2IpUN20-RPC53 segre-             pUN20-RPC53)  Materials Methods). cold- <br /> gants readily obtained. cells incapable        sensitive heat-sensitive rpc53 [rho&apos;] mutants losing RPC53 plasmid determined plasmid-              obtained. case, mutant cells arrested sectoring assay based colony color         restrictive temperature cell divisions inability grow YPD containing 2.5 M ethylene glycol,        65 85% unbudded cells.  preferential arrest kills cells carrying SUPJ1 gene pUN20 (10).          G1 phase general consequence Rpc53  RPC53 essential gene. Unexpectedly,        inhibition. RPC53 fragment complemented rpc53::HIS3-2              Synthesis tRNA inhibited rpc53 mutants. Incorpo- <br /> substitution present pUN20 unable            ration [3H]uracil total RNA (Fig. 6D) greatly present YCp50. investigate             affected transfer temperature-sensitive strain <br /> transferred HindIII-NruI RPC53 fragment YCp50            CMY397 (rpc53::HIS3-1 [rhoo]) 38&#194;&#176;C. centromeric vector pEMBLYc32(URA3) (37).               synthesis represented large rRNAs YCp5O-RPC53 pEMBLYc32-RPC53 plasmids                  continuous labeling conditions, concluded troduced strain CMY383 (rpc53::HIS3-2IpUN20-                 RNA polymerase   functional rpc53::HIS3-1 <br /> RPCS3), transformants tested ability        [rhoo] strain. C53 protein specifically associated lose resident pUN20-RPCS3. plasmid-shuffle            RNA polymerase C, anti-C53 antibodies inhibit test, pEMBLYc32-RPC53 capable complementing                vitro transcription tRNA gene (19). examined rpcS3::HIS3-2 mutation, YCp5O-RPCS3             RNA polymerase C transcription affected  2.4-kb HindIII-NruI fragment, containing 760 bp          rpc53 mutants labeling RNA 60 min [3H]uracil <br /> upstream 360 bp downstream RPCS3 coding                extracting electrophoretically separating sequence, appears contain promoter termina-        small RNAs polyacrylamide gels. Transfer RNA synthe- <br /> tion signals necessary RPCS3 expression (Fig. 2).        sis severely inhibited CMY397 (rpc53::HIS3-1 [rhoo]) believe RPCS3 expression inhibited          cells 6 h incubation 38&#194;&#176;C comparison unknown manner presented context          CMY396 (RPC53+ [rhoo]) congenic control strain (Fig. 9A inserted YCp50. Plasmid context effects gene            B). contrast, 5S RNA, transcribed RNA <br /> expression observed yeast TRPI         polymerase C, polymerase derived 5.8S RNA <br /> HIS3 genes (53).                                                  little affected rpc53::HIS3-1 strain    disruption RPC53 gene constructed         period. <br /> replacing 207-bp BglII fragment encoding amino acids 193            examined synthesis small RNAs 259 HIS3 DNA. Sporulation diploids heterozygous        strain CMY381 (rpc53::HIS3-2 [rho+])IpEMBLYc32-rpc53- rpc53::HIS3-1 disruption occasionally gave rise       ts28. specific inhibition tRNA synthesis seen slow-growing  colonies germination 23&#194;&#176;C (Fig.        strain 30 min transfer 37&#194;&#176;C (Fig. 9C).  <br /> 5C). Southern blot analysis confirmed  cells         temperature-sensitive inactivation RPC53 caused contained disrupted form RPC53 gene (data         insertional disruption point mutations leads shown). Surprisingly, rpc53::HIS3-1 haploids       inhibition tRNA gene transcription RNA polymerase [rho-] (lacking functional mitochondrial genome)         C. similar preferential inhibition tRNA compared temperature sensitive growth. detailed anal-      5S RNA synthesis observed yeast RNA <br /> ysis [rho phenotype rpc53::HIS3-1 disruption      polymerase C mutants examined date. include described separate communication (5).       mutants 5 13 subunits compose C cell division arrest associated temper-     enzyme  Discussion). <br /> ature-sensitive phenotype.                                           Sequence similarity yeast RPC53 human <br />    rpc53 mutants preferentially arrest G1 phase.          BN5J gene products. protein significant similarity <br /> CMY397 (rpc53::HIS3-1 [rhoo]) cells growing 23&#194;&#176;C          C53 searching EMBL data library <br /> transferred 38&#194;&#176;C, cell number increased threefold     (release 29, December 1991) TFASTA program (40, division arrest (Fig. 6A). contrast, optical        41). carboxy-terminal 136 amino acids human <br /> density culture (Fig. 6C) incorporation [3H]        BN51 protein (22) yeast C53 share 25% identity 78% <br /> uracil total RNA [35S]methionine total protein      similarity conservative substitutions dictated increased steadily 13 h incubation (Fig. 6D).    PAM250 matrix (40) allowed. FASTA similarity congenic control strain, CMY396 rpc53::HIS3-1 [rho0]!           score 166 carboxy-terminal region pEMBLYc32-RPC53, grew divided continuously                 proteins statistically significant; match 8 standard <br /> 38&#194;&#176;C. mutant cells arrested division         deviations mean comparison C53 <br /> large, unbudded cells single nucleus (Fig. 6B 7).      shuffled BN51 sequences RDF2 program (40, 41). morphology cells suggested              Weaker similarities seen amino-terminal regions <br /> arrested G1 phase cell cycle, analysis       proteins (Fig. 10). Furthermore, overall size <br /> flow cytometry confirmed cells G1 content      (395 amino acids BN51, 424 amino acids C53) DNA (Fig. 8). mutant cells retained good viability (85%)      overall hydrophilicity (compare Fig. 3B C) 12 h restrictive temperature despite   proteins similar. direct test functional homology <br />  VOL. 12, 1992                                                  YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST                               4321 <br />  <br />  <br />  <br />  <br />                              44                 //-m           ut                  X         |                               Ur-wt <br />  <br />                          o                                                             o <br />                                                                                        100                                   Met-mut <br />                                                                                                                                                  Ur-mut <br />  <br />  <br />                              0                                                         ~~~~~~~~~~~~~~~~~0 <br />                               0    2   4   6    8    10   12   14   16   18   20             0   2   4   6     8   10   12   14   16   18   20 <br />                                                Hours 38&#194;&#176;C                                                 Hours 38&#194;&#176;C <br />   FIG. 6. Characterization temperature-sensitive phenotype associated rpc53::HIS3-1 disruption mutation. CMY397 <br /> rpc53::HIS3-1 [rho&apos;] (designated mut figure) isogenic CMY396 rpc53::HIS3-1 [rho0]/pEMBLYc32-RPC53 (designated wt) cells grown synthetic complete medium 23&#194;&#176;C early log phase transferred 38&#194;&#176;C times indicated figure. <br />   Relative increase cell number. Cultures wild-type mutant cells shifted 38&#194;&#176;C initial value 2 x 106 cells ml. <br /> (B) Percentage unbudded cells. initial accumulation unbudded cells hours transfer 38&#194;&#176;C transient <br /> heat shock inhibition cell cycle G1 phase (28). Note wild-type mutant cells recovered heat shock effect mutant cells rearrested unbudded cells end division. (C) Relative increase optical density (OD) cultures. Mutant wild-type cell cultures transferred 38&#194;&#176;C A6. 0.15. (D) Continuous labeling total cellular protein RNA. [35S]methionine [3H]uracil added cell cultures 1 h transfer 38&#194;&#176;C. Incorporation radioactive precursors followed trichloroacetic acid (TCA) precipitation. Note mutant cells continued accumulate protein RNA arrest <br /> period. <br />  proteins attempted placing BN51                             C53 mutant proteins temperature sensitive human cDNAs control yeast GAL1O pro-                              function stability associate moter multicopy plasmid  Materials Methods).                          RNA polymerase C subunits. Mutants uniquely complementation lethal rpc53::HIS3-2 disruption                          temperature sensitive assembly RNA polymerase observed entire BN51-coding sequence                           C, rpc4O-ts mutants (33), exhibit decreased levels portion encoding 167 amino acids                           transcription generations growth protein  region exhibiting similarity C53                        restrictive temperature. genetic data  corresponding functionally essential portion                         sistent antibody inhibition studies suggesting C53 [5]) expressed GAL promoter.                             direct role C53 subunit tRNA synthesis. Analysis <br /> preliminary results indicated BN51 protein                           vitro transcription RNA polymerase C purified does accumulate yeast strains, work                       C53 mutants permit precise delineation necessary definitive conclusion drawn                      role transcriptional process (5). <br /> regarding capacity BN51 functionally replace C53.                          preferential inhibition tRNA compared 5S RNA <br />                                                                                    synthesis vivo property RNA polymerase C <br />                                   DISCUSSION                                       mutants examined date. Aside C53 (Fig. 9),                                                                                    includes mutants C160 (15), C82 (6), C34 (52), C31 <br />   RPC53 encodes RNA polymerase C-specific subunit                               (37) subunits. C31 subunit, nonsense mutation <br /> required tRNA gene transcription. role C53 RNA                        weakly suppressed tRNA inserting homologous <br /> polymerase C function previously exper-                          amino acid led specific inhibition tRNA synthesis. <br /> iments anti-C53 antibodies seen inhibit                            simple reduction wild-type <br /> tRNA gene transcription vitro (19). report,                          RNA polymerase C preferentially affects tRNA compared functional C53 protein required yeast cell                      5S RNA synthesis. hand, viability inactivation C53 temperature-sensitive                       rpc160 rpc53 mutant phenotypes mutants rapidly leads inhibition tRNA gene tran-                          enhanced thermolability mutant polymerase. scription vivo. rapidity inhibition                           likely possibilities explain heightened <br /> rpc53::HIS3-1 rpc53-ts28 mutants (Fig. 9) indicates                       resistance 5S RNA synthesis compared tRNA syn- <br />  4322     MANN ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />  <br />        R) 230                                                                B) 370- 6h <br />  <br />  <br />  <br />  <br />        C) 370- 12 h                                                          D) 370- 24 h <br />  <br />  <br />  <br />  <br />        E) DFPI; 370- 10 h                                                    F) 380- 24 h <br />  <br />  <br />  <br />  <br />   FIG. 7. Morphology CMY397 rpc53::HIS3-1 temperature-arrested cells.  D) Log-phase CMY397 cells YPD 23&#194;&#176;C  0 = 0.17; <br /> 2 x 106 cells ml; 47% unbudded cells) transferred 37&#194;&#176;C periods indicated figure. time point, aliquot cells fixed 3.7% <span id='am-29' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> examined differential interference contrast microscopy. (E) DAPI (4&apos;,6&apos;-diamidino-2- <br /> phenylindole) staining CMY397 DNA cells incubated 10 h 37&#194;&#176;C. Cells fixed 70% ethanol stained 0.5 ,ug DAPI ml. arrested cells exhibited single nucleus. punctate cytoplasmic staining seen [rho&apos;] cells lack <br /> mitochondrial DNA. (F) Terminal morphology CMY397 cells 24 h incubation YPD 38&#194;&#176;C. mutant cells arrest division lyse rapidly 38&#194;&#176;C 37&#194;&#176;C. cells figure shown magnification (approximately x880). <br />  1. <br /> E S <br /> = <br />      VOL. 12, 1992 <br />  <br />  <br />  <br />  <br />       7 <br />      X=, <br />  <br />  <br />  <br />                        <br />               <br />  <br />  <br />  <br />               B <br />  <br />  <br />  <br />               w <br />  <br />  <br />  <br />  <br />              1flS. <br />                   Gl <br />                   56% <br />  <br />  <br />  <br />  <br />                       ... .... .... <br />  <br />  <br />  <br />  <br />                      Gl <br />                       75% <br />  <br />  <br />  <br />                      .^ <br />  <br />  <br />  <br />  <br />                      40 <br />                                       Mut 23- 54%1UB <br />  <br />  <br />  <br />  <br />                                       - <br />                                         G2 <br />                                         44% <br />  <br />  <br />  <br />  <br />                                       mut 38 90% UB <br />  <br />  <br />  <br />  <br />                                       T,$ <br />  <br />                              Fluorescence <br />                                             G2 <br />                                             c <br />                                                -lOh <br />  <br />  <br />  <br />  <br />                                             ~~25 <br />                                                    1 22 17i <br />  <br />  <br />  <br />  <br />                                                    120 <br />                                                                <br />  <br />  <br />  <br />                                                               160 <br />                                                                   0 <br />  <br />  <br />  <br />  <br />         FIG. 8. Flow cytometry quantify DNA content yeast <br />      cells. CMY397 rpc53::HIS3-1 [rho&apos;] cells (mut figure) YPD <br />      23&#194;&#176;C   10 h incubation 38&#194;&#176;C (B) fixed 70% <br />      ethanol stained propidium iodide FACS analysis.      shown CMY396 rpc53::HIS3-1 [rho0]/pEMBLYc32-RPC53  <br />      trol cells (wt figure) log-phase growth 38&#194;&#176;C YPD. (C) <br />      Plot number cells (vertical axis) emitting given arbitrary <br />      unit fluorescence (horizontal axis) proportion DNA <br />      content cell. peaks labeled G1 G2 correspond      peaks haploid diploid content DNA. S-phase cells      poorly resolved G1 G2 peaks analysis,      attempt define subpopulation cells.      clear CMY397 mutants arrest cell division <br />      preferentially G1 content DNA incubation 38&#194;&#176;C.      accumulation G1 cells seen CMY396 RPC53+ cells <br />      grown 38&#194;&#176;C. consistently observed slightly lower percentage <br />      Gl-phase cells, determined FACS analysis, unbud- <br />      ded (UB) cells, seen light microscopy, CMY397 tempera- <br />                                                                         17j <br />  <br />  <br />  <br />  <br />                                                                       280 <br />  <br />  <br />  <br />  <br />      ture-arrested mutants. mean fraction unbud- <br />                                                                               -    C <br />  <br />  <br />  <br />  <br />                                                                                   II.. <br />                                                                                          Gl <br />                                                                                          46% <br />  <br />  <br />  <br />  <br />                                                                                          40 <br />                                                                                                        YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST <br />  <br />  <br />  <br />  <br />                                                                                                  Wt 38c 36%UB <br />  <br />  <br />  <br />  <br />                                                                                                  54% <br />  <br />  <br />  <br />  <br />                                                                                                   es <br />  <br />  <br />  <br />  <br />                                                                                               Fluorescence <br />  <br />                                                                                                                    genes <br />                                                                                                                              23&apos; 3h <br />  <br />  <br />  <br />  <br />                                                                                                                              _ <br />                                                                                                                                  R <br />  <br />  <br />  <br />                                                                                                                                  S <br />                                                                                                                                      RPC53+ <br />                                                                                                                                       389 389 <br />  <br />  <br />  <br />  <br />                                                                                                                                        ._ <br />                                                                                                                                             6h <br />                                                                                                                                             w     w@ . . =-S1B <br />                                                                                                                                                  io <br />  <br />                                                                                                                                                 --_ <br />                                                                                                                                                       B <br />                                                                                                                                                           rpc53::HIS3-1 <br />                                                                                                                                                    23&apos; 3..23&apos; 39&apos; <br />                                                                                                                                                           23&apos; <br />  <br />  <br />                                                                                                                                                           , <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                           38&#194;&#176; 38&apos; <br />                                                                                                                                                                 3h <br />  <br />  <br />  <br />  <br />                                                                                                                                                                 * <br />                                                                                                                                                                  U <br />  <br />  <br />  <br />  <br />                                                                                                                    CMY396 rpc53::HIS3-1/pEMBLYc-RPC53   CMY397 <br />                                                                                                                                                                      6h <br />  <br />                                                                                                                                                                       &apos; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                      -m <br />                                                                                                                                                                      __ <br />                                                                                                                                                                           c <br />  <br />  <br />  <br />  <br />                                                                                                                       FIG. 9. Synthesis small RNA molecules rpc53 mutants. <br />                                                                                                                    rpc53::HIS3-1 (B) cells grown synthetic complete medium <br />                                                                                                                    labeled [3H]uracil 60 min 23 38&#194;&#176;C 3 6 h                                                                                                                    incubation. Small RNAs extracted, 20 p1g RNA                                                                                                                    sample analyzed electrophoresis 6% polyacryl- <br />                                                                                                                    amide gel fluorography. (C) Small RNA synthesis                                                                                                                    examined CMY rpcS3-ts28 point mutants [3H]uracil <br />                                                                                                                    labeling 60 min 23&#194;&#176;C 30 min incubation 37&#194;&#176;C.                                                                                                                    instances, specific inhibition tRNA accumulation                                                                                                                    observed rpcS3 mutants restrictive temperature. <br />  <br />  <br />                                                                                                                    preferentially transcribed relative nucleolar 5S RNA <br />                                                                                                                           (59). interesting determine <br />                                                                                                                    preference remained conditions RNA <br />                                                                                                                    polymerase C limitation. model predicts tRNA <br />                                                                                                                    5S RNA synthesis equally affected RNA <br />                                                                                                                    polymerase C mutations eukaryotes 5S RNA <br />                                                                                                                    genes localized nucleolus. <br />                                                                                                                       Preferential G1 arrest rpc53 mutants. unexpected <br />                                                                                                                    phenotype rpc53 mutants rapid prefer- <br />                                                                                                                                                                               RPC53. ts28 <br />  <br />  <br />  <br />  <br />                                                                                                                    ential arrest G1 phase cell cycle large, round, <br />                                                                                                                    unbudded cells (Fig. 6 8). arrested cells maintain high <br />                                                                                                                    levels protein synthesis viability restrictive <br />                                                                                                                    temperature exhibit shmoo morphology <br />                                                                                                                    typical cdc28 mutants arrested G1 phase Start <br />                                                                                                                    (18).  phenotype closely resembles certain <br />                                                                                                                    translational machinery mutants (prtl/cdc63; translation <br />                                                                                                                    initiation factor [17], mesl; methionyl-tRNA synthetase [57, <br />                                                                                                                                                                                , <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                &quot; <br />                                                                                                                                                                                    370      370 <br />  <br />  <br />                                                                                                                                                                                               <br />  <br />  <br />  <br />                                                                                                                                                                                          ,,,M. <br />                                                                                                                                                                                                   25619$8 <br />                                                                                                                                                                                                   5.8S <br />                                                                                                                                                                                                   5.9 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                    ItRNs <br />                                                                                                                                                                                                             4323 <br />  <br />  <br />  <br />  <br />      ded cells arrest S/G2 content DNA,                                             58]) G1 arrest associated partial <br />      experimental error classification budded versus unbudded                                            inhibition protein synthesis. believed, having <br />      G1 versus G2 cells.                                                                                        demonstrated, partial inhibition protein syn- <br />                                                                                                                    thesis lead specific arrest G1 preferentially <br />                                                                                                                    affecting accumulation unstable polypeptides                                                                                                                     G1 cyclins) accumulation necessary      thesis conditions RNA polymerase C limitation.                                                       passage Start G1 (44). Similarly, G1 arrest      Transcription 5S RNA genes requires protein TFIIIA                                                     rpc53 mutants result partial inhibition      factors TFIIIB TFIIIC, tRNA                                                        translation secondary consequence inhibition      gene transcription requires TFIIIB TFIIIC                                                         stable RNA transcription RNA polymerase C. alter- <br />      TFIIIA (11, 13). RNA polymerase C higher                                                        native hypothesis C53 subunit required      affinity thermostability associated TFIIIA,                                                     transcription RNA polymerase C nonabundant RNA <br />      -B, -C compared TFIIIB -C                                                       species required passage G1 phase. <br />      hypothesis, expected purified compo-                                                    Based drug inhibition studies, previously proposed <br />      nents, greater affinity thermoresistance                                                       transcription RNA species necessary <br />      5S RNA compared tRNA gene transcription                                                        performance Start yeast cells (27, 60). U6 <br />      vitro. second possibility suggested exceptional                                                small nuclear RNA nuclear RNase P RNA (RPR1) <br />      localization 5S RNA genes nucleolus S.                                                       genes genes apart tRNA 5S <br />      cerevisiae (43). RNA polymerase C higher                                                           genes presently known transcribed RNA <br />      affinity 5S RNA genes nucleolus                                                        polymerase C S. cerevisiae (31, 36). transcription      stable nucleoplasm.                                                      genes inhibited rpc160-41 mutant (31, 36). <br />      transfer marked 5S RNA gene autonomously                                                           Interestingly, reexamination rpcl6O-41 mutant      replicating plasmid yeast cells showed plasmid-                                                   shown accumulates large, round, unbudded cells      borne 5S RNA genes (presumably nucleoplasmic)                                                            37&#194;&#176;C rpc53 mutants (55). particular tRNA,  4324          MANN ET AL.                                                                                                                 MOL. CELL. BIOL. <br />  <br />   R                                                                                    hamster cells, tRNA 5S RNA syntheses <br />                                                                                        (products transcription RNA polymerase C) inhibited <br />                                                                                        15 h restrictive temperature,                                                                                        synthesis 5.8S RNA. RNA derived                                                                                        nucleolytic cleavage 45S rRNA precursor transcribed <br />                                                                                      RNA polymerase  . remains possible RNA <br />                                                                                        polymerase C inhibition primary defect BN51 <br />                                                                                        mutant secondarily maturation 35S RNA <br />                                                                                        affected depletion 5S RNA RNAs <br />                                                                                        necessary rRNA processing.  observed                                                                                        accumulation stable RNA inhibited 12 h                                                                                        incubation restrictive temperature rpcS3:: <br />                                            C53                                         HIS3-1 mutant (data shown), tRNA synthe- <br />   B                                                                                    sis preferentially inhibited 6 h incubation (Fig. <br />        25.0%    identity 136      aa    overlap; init:     102, opt:    166         9). Finally, unable demonstrate complementation <br />                  300          310             320           330           340    350   rpcS3 mutants expression BN51 gene yeast <br />        C53:    LKKMN-NKPERFMVFQLPTRLPAFER-PAVKEEKEDMETQASDP--SKKKKNIKKKDTKD <br />                                                                                        cells. Likewise, expression RPC53 gene mutant <br />        BN51   :LRELSLTKEEELLFLQLPDTLPGQPPTQDIKPIKTEVQGEDGQWLIKQEKDREA}LAEN             BN51 hamster cells unable complement temper- <br />                        260          270             280       290         300    310 <br />  <br />                         360          370             380          390      400 <br />                                                                                        ature-sensitive defect (21). yeast mammalian <br />                ALSTRELA-GKVGSIRVHKSGKLSVKIGNVVMIGKGAETTFLHDVIALSIADD-ASSAE <br />                : .   :. X.: : .. .:::.. <br />                                    .   .   .: . :... <br />                                                   . . .. <br />                                                        : :: .. .. ... X.X <br />                                                                                        homologs interchangeable,                ACTLADLTEGQVGKLLIRKSGRVQLLLGKVTLDVTMGTACSFLQELVSVGLGDSRTGEMT <br />                        320          330             340       350         360    370 <br />                                                                                        case. example, human fly TFIID unable <br />                                                                                        supply essential function homologous S. cerevi- <br />           410          420 <br />                LLGRVDGKIVVTPQI                                                         siae protein (7, 14). Functional incapacity               VLGHVKHKLVCSPDFE <br />                                                                                        small number amino acid differences. substitution <br />                        380                                                             single amino acid human fibrillarin sequence <br />   FIG. 10. Sequence similarity yeast C53                           exists yeast fibrillarin sufficient correct human BN51 proteins.   PLFASTA analysis showing regions                           temperature-sensitive defect ability human <br /> similarity proteins. numbers                          protein complement yeast mutant (25). Polypeptides <br /> diagonal line indicate optimized FASTA score different                     approximately molecular weight determined local regions similarity. (B) Shown FASTA alignment (ktup                    BN51 protein associated highly <br /> = 1) carboxy-terminal regions C53 BN51 proteins. <br />  <br /> Evolutionarily conservative amino acid substitutions (PAM250 <br />                                                                                        purified preparations RNA polymerase C mouse <br /> scores zero greater) indicated single dot, contrast <br />                                                                                        plasmacytoma (51) human KB (23) cells. remains double dots identical amino acids. Xs indicate                      seen correspond BN51 protein. <br /> boundaries region highest local similarity C53 BN51 proteins. analyses performed                                            ACKNOWLEDGMENTS <br /> FASTA program package (40, 41). <br />                                                                                          study C.M. Burroughs- <br />                                                                                        Wellcome postdoctoral fellow Life Sciences Research Foun- <br />                                                                                        dation. work supported research funding grant SCI- <br /> U6 RPR1 RNAs, unidentified RNA species                                 CT91-0702 European Economic Community. <br />                                                                                          C.M. thanks Lee Hartwell hospitality. thank Michael <br /> transcribed C enzyme preferentially required                             Ittmann collaboration Pierre Thuriaux initially G1 phase.                                                                       bringing attention BN51 similarity, thank    Homology yeast C53 mammalian BN51                                   Ben Hall critical reading manuscript. <br /> proteins. search protein sequence data <br /> banks C53 sequence revealed significant degree                                                        REFERENCES <br /> sequence similarity protein, human BN51                               1. Allison, L. , M. Moyle, M. Shales, C. J. Ingles. 1985. <br /> protein (21, 22). proteins 25% amino acid iden-                                 Extensive homology largest subunits eukaryotic <br /> tity 78% similarity (allowing conservative amino acid                               prokaryotic RNA polymerases. Cell 42:599-610. <br /> substitutions PAM250 [40] score zero greater)                              2. Baldari, C., G. Cesareni. 1985. Plasmids pEMBLY: new 136 carboxy-terminal amino acids proteins                                   single-stranded shuttle vectors recovery analysis (Fig. 10). points argue sequence similarities                             yeast DNA sequences. Gene 35:27-32. proteins represent functional homology.                               3. Boeke, J. D., F. Lacroute, G. R. Fink. 1984. positive proteins similar size hydrophilicity,                               selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br />                                                                                            ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. localized nucleus. carboxy-terminal                                    Genet. 197:345-346. <br /> region proteins highest level sequence                            4. Caries, C.,  Treich, F. Bouet, M. Riva,  Sentenac. 1991. <br /> similarity, region corresponds function-                              additional common subunits, ABC10a ABC10p, ally essential portion C53 subunit (5). BN51                                    shared yeast RNA polymerases. J. Biol. Chem. 266:24092- <br /> protein originally identified ability cDNA                            24096. <br /> complement temperature-sensitive G1-phase arrest                               5. Chiannilkulchai, N.,  Moenne,  Sentenac, C. Mann. <br /> BHK (baby hamster kidney) cell mutant. Similarly, rpcS3                                    Biochemical genetic dissection S. cerevisiae RNA <br /> mutations yeast cells lead preferential G1 arrest.                                 polymerase C53 subunit analysis mitochondri- <br />                                                                                            ally mis-sorted mutant construct. J. Biol. Chem., press. <br /> Finally, tRNA synthesis thermolabile vivo                             6. Chiannilkulchai, N., R. Stalder, M. Riva, C. Caries, M. Werner, <br /> vitro temperature-sensitive rpc53 (Fig. 9) (5)                                  Sentenac. RPC82 encodes highly conserved  <br /> BN51 cell mutants (21).  possible                                 largest subunit RNA polymerase C (III) Saccharomyces <br /> conclude BN51 protein mammalian homolog                                    cerevisiae. Mol. Cell. Biol. 12:4433-4440. C53 subunit. temperature-sensitive BN51 mutant                                7. Cormack, B. P., M. Strubin,  S. Ponticelli, K. Struhl. <br />  VOL. 12, 1992                                           YEAST RNA POLYMERASE C53, HUMAN BN51, G1 ARREST                               4325 <br />  <br />       1991. Functional differences yeast human TFIID              tion apparatus. Mol. Cell. Biol. 9:5387-5394. <br />       localized highly conserved region. Cell 65:341-348.      30. Kolodziej, P., R.  Young. 1991. Mutations  8.   Dequard-Chablat, M., M. Riva, C. Caries,  Sentenac.               largest subunits yeast RNA polymerase II affect enzyme <br />       1991. RPC19, gene subunit common yeast RNA                 assembly. Mol. Cell. Biol. 11:4669-4678. <br />       polymerase   C (III). J. Biol. Chem. 266:15300-15307.       31. Lee, J.-Y., C. F. Evans, D. R. Engelke. 1991. Expression  9.   Dingwall, C., R.  Laskey. 1991. Nuclear targeting se-             RNase P RNA Saccharomyces cerevisiae controlled       quences consensus? Trends Biochem. Sci. 16:478-481.                   unusual RNA polymerase III promoter. Proc. Natl. Acad. Sci. <br /> 10.   Elledge, S. J., R. W. Davis. 1988. family versatile            USA 88:6986-6990. <br />       centromeric vectors designed use sectoring-shuffle       32. Link,  J., M. V. Olson. 1991. Physical map       mutagenesis assay Saccharomyces cerevisiae. Gene 70:303-             Saccharomyces cerevisiae genome 110-kilobase resolution. <br />       312.                                                                    Genetics 127:681-698. <br /> 11.   Gabrielsen, 0. S.,  Sentenac. 1991. RNA polymerase C          33. Mann, C., J.-M. Buhler,  Treich,  Sentenac. 1987. <br />       (III) transcription factors. Trends Biochem. Sci. 16:412-       RPC40, unique gene subunit shared yeast RNA <br />       416.                                                                    polymerases C. Cell 48:627-637. <br /> 12.   Garcia-Bustos, J., J. Heitman, M. N. Hall. 1991. Nuclear        34. Marck, C. 1988. &quot;DNA strider&quot;: &quot;C&quot; program fast <br />       protein localization. Biochim. Biophys. Acta 1071:83-101.               analysis DNA protein sequences Apple Macintosh <br /> 13.   Geiduschek, P. E., G. P. Tocchini-Valentini. 1988. Tran-            family computers. Nucleic Acids Res. 16:1829-1836. <br />       scription RNA polymerase III. Annu. Rev. Biochem. 57:            35. Micouin, J.-Y. 1988. Clonage et etude du gene RPC53, codant <br />       879-914.                                                                pour la sous-unite &quot;C53&quot; l&apos;ARN polym6rase C la levure <br /> 14.   Gill, G., R. Tjian. 1991. highly conserved domain              Saccharomyces cerevisiae. Ph.D. thesis. Universite Paris- <br />       TFIID displays species specificity vivo. Cell 65:333-340.            Sud, Centre d&apos;Orsay, France. <br /> 15.   Gudenus, R., S. Mariotte,  Moenne,  Ruet, S. Memet, J. M.       36. Moenne, , S. Camier, G. Anderson, F. Margottin, J. Beggs, <br />       Buhler,  Sentenac, P. Thuriaux. 1988. Conditional mu-              Sentenac. 1990. U6 gene Saccharomyces cerevi- <br />       tants RPC160, gene encoding largest subunit RNA           siae transcribed RNA polymerase C (III) vivo       polymerase C Saccharomyces cerevisiae. Genetics 119:517-             vitro. EMBO J. 9:271-277. <br />       526.                                                                37. Mosrin, C., M. Riva, M. Beltrame, E. Cassar,  Sentenac, 16.   Hall, M. N., L. Hereford,  Herskowitz. 1984. Targeting         P. Thuriaux. 1990. RPC31 gene Saccharomyces cerevi- <br />       P-galactosidase nucleus yeast. Cell 36:1057-1065.             siae encodes subunit RNA polymerase C (III) acidic <br /> 17.   Hanic-Joyce, P. J., G. C. Johnston, R.  Singer. 1987.             tail. Mol. Cell. Biol. 10:4737-4743. <br />       Regulated arrest cell proliferation mediated yeast prtl       38. Nasmyth, K. , S.  Reed. 1980. Isolation genes       mutations. Exp. Cell Res. 172:134-145.                                  complementation yeast: molecular cloning cell-cycle <br /> 18.   Hartwell, L. H., R. K. Mortimer, J. Culotti, M. Culotti.            gene. Proc. Natl. Acad. Sci. USA 77:2119-2123. <br />       1973. Genetic control cell division cycle yeast. V.       39. Orr-Weaver, T. L., J. W. Szostak, R. J. Rothstein. 1981. <br />       Genetic analysis cdc mutants. Genetics 74:267-286.                   Yeast transformation: model study recom- <br /> 19.   Huet, J., M. Riva,  Sentenac, P. Fromageot. 1985. Yeast           bination. Proc. Natl. Acad. Sci. USA 78:6354-6358. <br />       RNA polymerase C subunits. Specific antibodies           40. Pearson, W. R. 1990. Rapid sensitive sequence comparison <br />       structural functional probes. J. Biol. Chem. 260:15304-             FASTP FASTA. Methods Enzymol. 183:63-98. <br />       15310.                                                              41. Pearson, W. R., D. J. Lipman. 1988. Improved tools 20.   Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor-           biological sequence comparison. Proc. Natl. Acad. Sci. USA <br />       mation intact yeast cells treated alkali cations. J.            85:2444-2448. <br />       Bacteriol. 153:163-168.                                             42. Peden, K. W. C., D. Nathans. 1982. Local mutagenesis <br /> 21.   Ittmann, M., J. Ali,  Greco, C. Basilico. gene                deletion loops DNA heteroduplexes. Proc. Natl. <br />       complementing temperature-sensitive cell cycle mutant              Acad. Sci. USA 79:7214-7217. <br />       BHK cells human homolog yeast RPC53 gene,         43. Philippsen, P., M. Thomas, R.  Kramer, R. W. Davis. <br />       encodes subunit RNA polymerase III. Submitted            1978. Unique arrangement coding sequence 5S, 5.8S, 18S <br />       publication.                                                            25S ribosomal RNA Saccharomyces cerevisiae deter- <br /> 22. Ittmann, M.,  Greco, C. Basilico. 1987. Isolation            mined R-loop hybridization analysis. J. Mol. Biol. <br />     human gene complements temperature-sensitive cell                  123:387-404. <br />     cycle mutation BHK cells. Mol. Cell. Biol. 7:3386-3393.            44. Reed, S.  1991. Gl-specific cyclins: search S-phase- <br /> 23. Jaehning, J. , P. S. Woods, R. G. Roeder. 1977. Purifica-           promoting factor. Trends Genet. 3:95-99. <br />     tion, properties, subunit structure deoxyribonucleic           45. Riva, M., S. Memet, J.-Y. Micouin, J. Huet,  Treich, J. Dassa, <br />     acid-dependent ribonucleic acid polymerase III uninfected            R. Young, J.-M. Buhler,  Sentenac, P. Fromageot. 1986. <br />     adenovirus 2-infected KB cells. J. Biol. Chem. 252:8762-              Isolation structural genes yeast RNA polymerases     8771.                                                                     immunological screening. Proc. Natl. Acad. Sci. USA 83:1554- <br /> 24. James, P., S. Whelen, B. D. Hall. 1991. RETI gene              1558. <br />     yeast encodes second-largest subunit RNA polymerase            46. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. <br />     III. Structural analysis wild-type retl-l mutant               Fink 1987. Saccharomyces cerevisiae genomic plasmid bank <br />     alleles. J. Biol. Chem. 266:5616-5624.                                    based centromere-containing shuttle vector. Gene 60:237- <br /> 25. Jansen, R. P., E. C. Hurt, H. Kern, H. Lehtonen, M. Carmo-                243. <br />     Fonseca, B. Lapeyre, D. Tollervey. 1991. Evolutionary             47. Rothstein, R. J. 1983. step gene disruption yeast. Meth- <br />     conservation human nucleolar protein fibrillarin           ods Enzymol. 101:202-211. <br />     functional expression yeast. J. Cell. Biol. 113:715-729.           48. Russel, M., S. Kidd, M. R. Kelley. 1986. improved <br /> 26. Johnson, S. L. 1991. Structure function analysis yeast         filamentous helper phage generating single-stranded plasmid <br />     CDC4 gene product. Ph.D. thesis. University Washington,                DNA. Gene 45:333-338. <br />     Seattle.                                                              49. Sentenac,  1985. Eukaryotic RNA polymerases. Crit. Rev. <br /> 27. Johnston, G. C., R.  Singer. 1978. RNA synthesis                Biochem. 18:31-90. <br />     control cell division yeast S. cerevisiae. Cell 14:951-     50. Sherman, F. 1991. Getting started yeast. Methods  <br />     958.                                                                      zymol. 194:3-21. <br /> 28. Johnston, G. C., R.  Singer. 1980. Ribosomal precursor          51. Sklar, V. E. F., R. G. Roeder. 1976. Purification subunit <br />     RNA metabolism cell division yeast Saccharomyces               structure DNA-dependent RNA polymerase III     cerevisiae. Mol. Gen. Genet. 178:357-360.                                 mouse plasmacytoma, MOPC 315. J. Biol. Chem. 251:1064- <br /> 29. Kolodziej, P., R.  Young. 1989. RNA polymerase II                   1073. <br />     subunit RPB3 essential component mRNA transcrip-         52. Stettler, S., S. Mariotte, M. Riva,  Sentenac, P. Thuriaux. <br />  4326       MANN ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />       RPC34 encodes essential specific subunit yeast RNA        59. Van Ryk, D. , Y. Lee, R. N. Nazar. 1990. Efficient <br />       polymerase C (III). J. Biol. Chem., press.                           expression utilization mutant 5S rRNA Saccharomy- <br /> 53.   Struhl, K. 1981. Position effects Saccharomyces cerevisiae. J.       ces cerevisiae. J. Biol. Chem. 265:8377-8381. <br />       Mol. Biol. 152:569-575.                                             60. Veinot-Drebot, L. M., R.  Singer, G. C. Johnston. 1989. <br /> 54.   Struhl, K. 1985. Nucleotide sequence transcriptional map-           rRNA transcription initiation decreased inhibitors       ping yeast pet56-his3-dedl gene region. Nucleic Acids            yeast cell cycle control step &quot;start.&quot; J. Biol. Chem. 264:19528- <br />       Res. 13:8587-8601.                                                      19534. <br /> 55.   Thuriaux, P. Unpublished da <br /> </body></html>