<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Sroka et al. BMC Bioinformatics 2011, 12:196 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  SOFTWARE                                                                                                                                      Open Access <br />  <br /> Acorn: grid computing constraint <br /> based modeling visualization genome <br /> scale metabolic reaction networks web <br /> interface <br /> Jacek Sroka1, &#197;?ukasz Bieniasz-Krzywiec1, Szymon Gw&#195;&#179;&#197;&#186;d&#197;&#186;1, Dariusz Leniowski1, Jakub &#197;?&#196;&#8230;cki1, Mateusz Markowski1, <br /> Claudio Avignone-Rossa2, Michael E Bushell2, Johnjoe McFadden2 Andrzej M Kierzek2* <br />  <br />  <br />   Abstract <br />   Background: Constraint-based approaches facilitate prediction cellular metabolic capabilities, based, turn <br />   predictions repertoire enzymes encoded genome. Recently, genome annotations   used reconstruct genome scale metabolic reaction networks numerous species, including Homo sapiens, <br />   allow simulations provide valuable insights topics, including predictions gene essentiality   pathogens, interpretation genetic polymorphism metabolic disease syndromes suggestions novel <br />   approaches microbial metabolic engineering. constraint-based simulations integrated   functional genomics portals, activity requires efficient implementation constraint-based simulations <br />   web-based environment. <br />   Results:  present Acorn, open source (GNU GPL) grid computing constraint-based <br />   simulations genome scale metabolic reaction networks interactive web environment. grid- <br />   based architecture allows efficient execution computationally intensive, iterative protocols Flux <br />   Variability Analysis, readily scaled numbers models  users) increase. web <br />   interface uses AJAX, facilitates efficient model browsing search functions, intuitive <br />   implementation appropriate simulation conditions. Research groups install Acorn locally create user <br />   accounts. Users import models familiar SBML format link reaction formulas major <br />   functional genomics portals choice. Selected models simulation results shared different <br />   users publically available. Users construct pathway map layouts import server <br />   using desktop editor integrated  Pathway maps used visualise numerical results <br />   web environment. illustrate features deployed Acorn created web server <br />   allowing constraint based simulations genome scale metabolic reaction networks E. coli, S. cerevisiae <br />   M. tuberculosis. <br />   Conclusions: Acorn free software package, installed research groups create web based <br />   environment simulations genome scale metabolic reaction networks. facilitates shared access   models creation publicly available constraint based modelling resources. <br />  <br />  <br />  <br />  <br /> * Correspondence: kierzek@surrey.ac.uk <br /> 2 <br />  Faculty Health Medical Sciences, University Surrey, Guildford, GU2 <br /> 7XH, UK list author information available end article <br />  <br />                                         &#194;&#169; 2011 Sroka et al; licensee BioMed Central  Open Access article distributed terms Creative Commons <br />                                         Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction                                         medium, provided original work properly cited. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                Page 2 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> Background                                                   conditions, determined terms set nutrients <br /> Analysis biochemical reaction networks, enable         present medium. <br /> predictions phenotype catalogue            concept gene essentiality provides good illus- <br /> molecular parts encoded genome, fulfils major       tration new capabilities. Currently, functional <br /> objective Systems Biology. scarcity         genomics portals gene terms com- <br /> detailed quantitative information individual           parison similar sequences organisms, direct <br /> enzyme rate constants intracellular concentrations,      experimental evidence function particu- <br /> genome scale models  far, constructed      lar experimental conditions high throughput data- special case metabolic reaction networks      sets expression gene operating steady state conditions. models        observed. constraint-based analysis appropriate <br /> capitalize fact metabolic conversions occur      GSMN, integrated  allow cell high rates       content dependent experimental conditions, <br /> given set genes active experimental conditions      specified user. user specify compo-  network metabolic reactions       sition growth medium compute considered quasi-steady state  metabo-      function gene expressed set metabolic cap- <br /> lite concentrations change).       abilities gene essential user- <br /> modelled set linear relationships        defined experimental conditions. <br /> calculated reaction fluxes.  linear        far, following attempts corresponding genome scale models underdeter-          creating web-based resources, allowing computation <br /> mined exploited Linear Programming (LP)         GSMN models subsequent linking models <br /> based optimization objective function repre-     results functional genomics resources. sents metabolic capability  objective    GSMN-TB [6] reconstruction GSMN <br /> function defined flux specific reac-     Mycobacterium tuberculosis, causative agent tion flux specific metabolite. LP opti-     Tuberculosis. case, web interface allows mization results unique value objective         specification growth medium composition, function, corresponding flux distribution     using information, provides calculation unique. range individual reaction flux    maximal theoretical flux biomass flux distributions consistent maximal      gene essentiality prediction. implements reac- <br /> objective function value exploited iterative   tion essentiality scan Flux Variability Analysis <br /> LP protocols, described following sections.      (FVA) [14], iterative protocols performing <br /> approach referred constraint based [1],     respectively linear programming optimiza- evaluates metabolic capabilities set     tions reaction model. gene <br /> straints expressed balance equations, known quanti-       names model linked Tuberculist [15], tative values certain fluxes (determined vitro      functional genomics portal dedicated Mycobacteria. <br /> experimentation) information  literature)       CycSim [16] allows computation maximal capacity reversibility reac-        maximal theoretical flux selected metabolites <br /> tions. acquisition information initi-     does implement iterative protocols <br /> ared analysis genome sequence      described   implements sophisticated approach applied reconstruction      user interface, enabling specification experimental <br /> genome scale metabolic reaction networks (GSMNs)         conditions perturbations. allows visualisation <br /> major model organisms [2-5], pathogens [6-8]             numerical data pathway maps imported microorganisms used industrial bioprocesses [9,10].       KEGG [17] PathwayTools [18] software. WEbcoli reconstruction human metabolism         [19] server allows modification constraint based <br /> available [11] currently used investigate    simulation Escherichia coli iJR904 model. ser- <br /> mechanisms metabolic diseases [12,13].                    ver displays metabolic pathway maps overlays simu- <br />    availability genome sequences moti-       lation results diagrams. single <br /> vated development functional genomics internet        evaluation objective function provided portals, multiplicity sequence-derived, high-     iterative simulation approaches available. throughput traditional experimental data inte-       note web servers described grated appropriate comparative analysis proto-       available software downloaded used <br /> cols. integration constraint-based simulations     create new web servers users institution. <br /> GSMNs portals equip users,     Recently, BioMet Toolbox [20] server time, tools enable predictions meta-   set constraint-based modelling tools developed bolic phenotype cells defined experimental          Nielsen&#226;&#8364;&#8482;s group available web interface.  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                  Page 3 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> advantage BioMet Toolbox wide range meth- <br /> ods run.  comparison servers mentioned integrated func- <br /> tional genomics portals visualisation tools. user expected create text file format particular <br /> tool download file repository models subsequently upload file form web <br /> interface particular tool. servers inte- <br /> grated databases models format models presentation web interface. Reactions linked gene names functional genomics portals, facilitating interpretation models context genome annotation. results formatted <br />                                                                Figure 1 Components Acorn  web presentation numerical data visualised network maps. <br />    work present Acorn, grid computing sys- <br /> tem, deployed users institution         2. worker console application performing used create web based resources constraint based       computations requested users web interface, <br /> simulations GSMN models. Research groups                 3. graphical editor desktop application defining <br /> install Acorn, set user accounts import models         parts model visualized  SBML format [21]. Users define silico        user interface main application web <br /> experiments specifying nutritional conditions run      based, designed ergonomics mind optimised <br /> virtual experiments order calculate maximum         handling large models. example lat- <br /> theoretical value objective function, identify         ter feature, user quickly filter long lists essential reactions, predict gene essentiality perform    AJAX (Asynchronous JavaScript XML) based <br /> Flux Variability Analysis (FVA) [14]. Models experi-      mechanism, e., delays ments shared users. grid-based             incurred transferring data browser Acorn architecture facilitates implementation          web server. example ergonomic approach <br /> computationally intensive, iterative protocols        positioning key control buttons FVA interactive web environments            visible place screen using <br /> easily scaled number users models            specially defined CSS (Cascading Style Sheets). grow. web server allows            facilitates working long lists. useful outcome visualization numerical results pathway maps        approach user does scroll defined user. Acorn includes integrated        start simulation accepting choice optimi- <br /> graphic editor fast creation metabolic map layouts     sation target reaction list. command line tool, used connec-           computations Acorn time intensive, <br /> tion interfaces. demonstrate features     chosen architecture (Figure 2) allows used Acorn implement web resource pro-           easy distribution workload viding constraint-based simulations GSMNs           machines, enabling rapid easy creation Saccharomyces cerevisiae, E. coli M. tuberculosis :       computational cluster. Computations commissioned    http://sysbio3.fhms.surrey.ac.uk:8080/acorn/homepage.      users automatically divided subtasks, jsf.                                                          independently computed workers running                                                               different machines. main application <br /> Implementation                                                places subtask specifications queue pulled section implementation details     workers. worker cyclically consumes Acorn. explain justify technological      current oldest subtask queue, accesses cor- <br /> architectural choices  design approaches         responding model data main applications&#226;&#8364;&#8482; data- <br /> applied implementation                   base, performs computation stores results   Acorn consists components               database. addition submission <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span>, (Figure 1):                                                   time consuming tasks given higher priority.    1. main application, includes web based         queue contains subtasks, worker pulls user interface defining executing required         subtasks currently available, <br /> computational task database storing mod-        waits new task appears. use queue <br /> els, experiments experimental results,                    based pull model processing jobs pulled  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                      Page 4 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 2 Interaction worker, main application web client. <br />  <br />  <br /> free workers opposed pushed                  users standalone command line tool, main application results optimal workload distri-             used implement interfaces. <br /> bution. renders administration                  graphical editor standalone Java desktop <br /> easier, content queue moni-               application built NetBeans Platform, tored workers added removed according                 generic framework Swing applications http://plat- needs availability free processing power.                 form.netbeans.org/. runs platforms   implemented Java use             Java runtime environment available. accesses Java Enterprise Edition (Java EE) technology stack.             main application obtain model data store <br /> main application deployed Java EE 5.0 com-              defined visualisations presentation web inter- <br /> patible application server Open Source GlassFish             face. communication performed Web <br /> 2.1 https://glassfish.dev.java.net/. Acorn profits         Service provided main application,  maturity enterprise features Java EE component               user, added advantage blocked <br /> technologies. example, operations queue              firewalls. <br /> database subject transaction control, com- <br /> putations consistently completed situations like          Results <br /> concurrent execution workers, failure                 Distribution Acorn implemented  <br /> active worker restart main application.               loading Google Code repository http://code.google. <br /> GlassFish server equipped necessary tools          com/p c-o-r-n/. main application installed <br /> monitoring tuning. Furthermore, flexibility          platform running Java VM, Glassfish applica- <br /> enterprise programming environment allows deploy-               tion server relational database management sys- <br /> ment Acorn relational database management               tem. binaries linear programming engine providing JDBC drivers. current version              available Linux, MacOS &#195;&#8212; Windows operating tested Java Derby http://db.apache.org/derby/             systems  Java VM, necessary MySQL http://www.mysql.com databases.                           run worker. graphical editor desktop applica- <br />   Linear programming calculations performed using               tion used running Java VM. appropriate functions GLPK library http://               current version Acorn provides following features. <br /> www.gnu.org/software/glpk/. linear programming <br /> engine binary executable file implemented C++.              web-based user interface GSMN models worker starts linear programming engine using               user logs web interface operating interface Java, sends             browse GSMN models tabular format (Figure 3). linear programming problem standard input                rows reaction tables contain reaction  reaction <br /> retrieves results standard output creating             formula lower upper flux bounds. cell temporary files. engine available               reaction entry contains gene-reaction association <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                    Page 5 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 3 web page representingt GSMN. table containing reaction names interactively filtered keyword &#226;&#8364;&#339;alanine&#226;&#8364;?  reaction  Reaction bounds editable. gene names Boolean gene-reaction association formula linked functional <br />  genomics portal chosen user created model. <br />  <br />  <br /> formula representing genes encoding protein subunits                     results calculation flux range consistent <br /> enzymes catalysing reaction. genes encoding                        maximal value objective function. <br /> components multisubunit enzymes linked                          flux ranges unique  unlike flux values boolean statement paralogous genes                              obtained particular single FBA solution, represented statements. Gene names linked                         used reason internal flux distribution gene pages selected functional genomics resources                     nutritional conditions [22]. <br /> integrating GSMN model literature annotations comparative sequence analysis results. Reaction                       run simulation Acorn web interface bounds editable used set initial conditions              user selects methods specifies constraint-based simulations.                                    objective function nutritional conditions.                                                                         objective function specified metabolite web interface constraint-based analysis                       (maximal flux metabolite) reaction protocols                                                               (maximal flux reaction). Nutritional condi- following computational methods available:                      tions defined constraining lower upper <br />                                                                         bounds transport reaction fluxes. Acorn interface <br />      Flux Balance Analysis (FBA) - calculation                 allows specification bounds reactions     maximal theoretical value objective function                  allowing complex analysis scenarios <br />     nutritional conditions specified user;                 incorporation internal metabolic flux measure- <br />     iI) Gene essentiality prediction - fluxes           ments simulations multiple gene knockouts.     reactions require functionality selected             keyword searches used quickly reactions <br />     gene constrained 0 maximal theoretical               metabolites  Simulations scheduled <br />     value objective function calculated.               user wait completion com- <br />     objective function value equals 0, gene declared             putational tasks, log return later <br />     essential particular metabolic function;                  examine results. simulation scenario,     iii) Reaction essentiality scan - reaction, turn,           simulation results saved database     inactivated FBA optimisation run.                 silico experiment. Users share silico experi- <br />     result lists maximal value objective func-               ments visualise results pathway diagrams. <br />     tion single reaction knock  <br />     iv) Flux Variability Analysis (FVA) [14] - flux                 grid-based environment distributing computational <br />     corresponding objective function constrained               load <br />     maximal value flux reac-                   single execution FBA computationally effi- <br />     tion, turn, maximised minimised.                     cient. example, calculation maximal growth <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                     Page 6 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> rate yeast GSMN model iND750 takes 0.73s 2.13                     user management, model silico experiment <br /> GHz Intel Core 2 Duo processor CPU calculated                       sharing linear programming engine Acorn sys-                     new user Acorn register web <br /> tem.  iterative protocols reaction essentiality               interface. user registered, software <br /> scans FVA require (respectively) evalua-                  installation step, given administration privileges. tions objective function reaction                 models created SBML file upload, initi-   execution FVA yeast                        ally used user uploaded GSMN model containing 1265 reactions takes 15                      SBML file. Likewise, silico experiments parti- <br /> minutes CPU time. computational time                      cular model initially belong user created substantial web-based applications, especially mul-               model. user chose share models tiple users simultaneously using server.                     silico experiments users  Acorn server, hand, using grid-based                     allows creation online communities solutions distribute computational load mul-                  users share models results numerical simulations. <br /> tiple computers. worker program installed                      models experiments available cluster. worker connects                     publicly accessible account, enables creation Acorn server receives set FBA objective                     web based resources constraint based simulations <br /> function evaluation tasks execute. Results                   GSMN models. <br /> tasks sent server  Implementation <br /> section details).  FBA calculations                   Desktop metabolic pathway layout editor Acorn server distributed arbi-                   Acorn provides desktop editor metabolic pathway <br /> trary number computers local area network                   diagrams (Figure 5). Diagrams created Petri net internet, rendering scalable large num-                [24] notation reactions substances represented <br /> ber users computational tasks.                                    different symbols. editor connects Acorn <br />                                                                          database, using <span id='am-3' about='oboInOwl:id' typeof='owl:Thing'>ID</span> password Acorn <br /> SBML file import                                                         users prompts user select GSMN user import GSMN models SBML format                       models available user&#226;&#8364;&#8482;s Acorn account. Subse- <br /> (Figure 4). reaction capacity bounds gene-reaction               quently, user creates diagram association formulas, features specific FBA simulations               substance reaction symbol associated implemented Acorn  provided                 substance reaction GSMN. editor annotations following BiGG database conventions [23].                    substances (reactions) associated given reac- user upload file using web interface.                    tion (substance) place map. <br /> Optionally, user input HTTP address <span id='am-2' about='xsd:string' typeof='owl:Thing'>string</span>,                   placing symbol diagram, concatenated gene names create links               user does check subsequent metabolite gene pages functional genomics portal               names quickly build network map simply <br /> user&#226;&#8364;&#8482;s choice. SBML file parsed stored                 following connections GSMN. diagram relational database backend area Acorn                     sent server saved database. <br />  <br />  <br />  <br />  <br />  Figure 4 model import interface. user prompted provide SBML file BiGG database annotations <br />  representing FBA specific information. Subsequently, user specifies model organism. user link gene names  functional genomics portal choice providing HTTP address gene replaced &#226;&#8364;&#339;%s&#226;&#8364;? string. address template  used link individual genes model pages corresponding pages functional genomics portal. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                     Page 7 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 5 Screenshot desktop editor session S. cerevisiae iND750 model. user create network map circles <br />  represent substances reactions represented rectangles. symbols linked substances reactions GSMN.  <br />  map does need represent genome scale network, numerical results, mapped diagram obtained <br />  computations entire model. map layout send Acorn server stored database used generation  visualisations web interface. Numerical results visualised desktop editor. figure shows results FBA <br />  optimisation biomass flux. <br />  <br />  means visualisation numerical results                         script languages used bioinformatics community <br /> metabolic pathway diagrams                                               annotation pipeline. Numerical results FBA FVA calculations                     used create graphics user interfaces visualised pathway diagrams (Figure 6). pathway                   Acorn. Finally, users Acorn web server diagrams created user available                    use command line tool run web associated simulation                         computations local machines resources results. Subsequently, user create image files                   web server limited constrained. numerical simulation results, representing fluxes flux ranges, printed boxes corre-               GSMN web resource <br /> sponding reactions. Numerical results                 demonstrate features Acorn,  <br /> visualised way stand path-                  structed web resource constraint-based simulation <br /> way editor.                                                              GSMNs research group.                                                                          resource contains GSMN-TB model M. tuberculo- stand command line tool                                          sis [6], reconstructed group reconstruc- computational engine Acorn software                        tions published groups: E. coli iAF1260 [2] available stand tool. tool used                 S. cerevisiae iND750 [3]. iAF1260 iND750 evaluate objective function GSMN model                        directly uploaded Acorn SBML files <br /> written simple text file format. command line                     available BiGG database. shows tool interprets boolean gene-reaction association formu-                 Acorn model upload tool compatible SBML <br /> las used perform evaluate                    specification constraint-based specific annotations <br /> objective function single gene knockout version                 used COBRA toolbox software BiGG model. Availability Acorn computational                       database [23]. case GSMN-TB iND750 <br /> engine form command line tool binary                      models, flat files accompanying publications <br /> useful users intend develop                     converted annotated SBML file sub- <br /> interfaces. tool easily integrated                 sequently uploaded server. models  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                        Page 8 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br />  Figure 6 Visualisation numerical results pathway maps web interface Acorn  Network map created  desktop editor stored database used. reaction fluxes FVA flux ranges printed reaction symbols. figure shows <br />  image generated web interface showing reaction fluxes central metabolism map S. cerevisiae iND750 model.  results obtained FBA optimisation biomass flux. <br />  <br />  uploaded tested users sys-                  pathway layouts choice. fluxes tem upload privileges subsequently published,                    obtained calculations involving reactions sharing &#226;&#8364;&#339;guest&#226;&#8364;? user account available                   entire GSMN, reactions public.                                                            users displayed format,    models available web resource                   informative  linked relevant functional genomics portal. gene names GSMN-TB model linked                             Conclusions <br /> Tuberculist database, E. coli model linked                      paper, present Acorn software package, <br /> KEGG database, S. cerevisiae model linked                   deployed user&#226;&#8364;&#8482;s institution used SGD database. implementations demon-                       create web-based resources constraint based <br /> strated model upload mechanism Acorn                      simulations GSMNs. software used used link reaction lists results files GSMN                  make collection GSMNs available community <br /> models major functional genomics resources,                       users hold accounts  Users <br /> facilitating interpretation simulations                upload models run constraint-based simulation <br /> context wealth knowledge comparative                        methods make predictions metabolic cap- <br /> analysis results available knowledge bases.                  abilities organisms investigation. models <br />    user use web resource run simula-                   results linked functional genomic <br /> tion protocols implemented Acorn models                         resources gene names.  mentioned  demonstrate visualisation                         results constraint based calculations examined <br /> functionality Acorn, used desktop editor                   context genome sequences, annotations create layout glycolytic pathway Yeast                     sequence similarity-based predictions available uploaded layout iND750 model. Users                     portals. grid-based architecture Acorn allows visualise fluxes calculated simulation                    implementation computationally expensive <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                 Page 9 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> simulation protocols requiring execution thou-        code distribution, used create collabora- <br /> sands linear programming evaluations entire       tive model development environments public web <br /> model. Users visualise numerical results simula-      resources laboratories. <br /> tion methods pathway maps loaded            summarise, believe Acorn server maps represent tthe users view,      useful research groups reconstructing model specific user concerned,      studying GSMNs. application result establish- <br /> numerical data visualised map calculated       ment online communities sharing models results <br /> constraint-based modelling entire network. Path-      silico experiments. facilitate improvement <br /> way diagrams constructed desktop tool,          metabolic models used identification novel supports rapid assignment graphical sym-        drug targets microbial pathogens metabolic engi- <br /> bols substances reactions present model.       neering industrially valuable microbial strains. <br />    important features Acorn user <br /> management model sharing. Select users upload        Availability Requirements <br /> starting models share         Project  Acorn <br /> users create variations starting models        Project home page: http://code.google.com/p c-o-r- <br /> modifying parameters set simula-         n/ <br /> tions affecting models simulations run         Surrey GSMN server: http://sysbio3.fhms.surrey.ac. users. choose render           uk:8080/acorn/homepage.jsf <br /> models, simulation scenarios results visible      Operating systems(s): Platform independent <br /> users  guest user account,       Programming language: Java, C++ <br /> available research community.  Acorn          requirements: Java 6.0 higher, GlassFish Java used create research community websites            EE Server v2 higher <br /> dedicated annotation investigation meta-         License: GNU GPL v2 <br /> bolic reaction network models.                                 restrictions use non-academics: use    addition software used create    constraint based modelling results obtained using constraint-based modelling server, provide       iAF1260 iND750 models Surrey FBA resource web resource giving access GSMN models S.          subjected BiGG database license. <br /> cerevisiae, E. coli, M. tuberculosis. Users use guest account register gain access                                                              Acknowledgements <br /> models need installing learning mod-      work supported BBSRC EraSysbioPlus grant BB/ <br /> elling environments  example, MATLAB             I00453X/1. <br /> based COBRA toolkits. Users use server                                                              Author details <br /> upload models, represented          1 <br />                                                               Institute Informatics University Warsaw, Poland. 2Faculty Health SBML format, use computational tools implemented         Medical Sciences, University Surrey, Guildford, GU2 7XH, UK. study models  share                                                              Authors&#226;&#8364;&#8482; contributions <br /> results users).                                   AMK JS designed software functionality. JS designed software <br />    believe Acorn provides advantages         architecture, supervised software development participated software alternative solutions compared existing web ser-      implementation. LBK, SG, DL, JL implemented database worker modules. <br />                                                              MM implemented pathway visualisation. JL implemented command line <br /> vers constrained-based modelling metabolic net-       computational engine. CAR developed case studies visualisation yeast <br /> works. provides advantages compared           pathways. MB JMF tested software functionality participated GSMN-TB server respect visualisation             writing manuscript. AMK JS wrote manuscript. authors <br />                                                              read approved final manuscript. <br /> implements convenient interface model spe- <br /> cification. main advantage CycSim WEbcoli       Received: 18 August 2010 Accepted: 24 2011 implementation FVA reaction essenti-       Published: 24 2011 <br /> ality scans. FVA currently method,                                                              References <br /> allows exploration internal flux distribution      1. Price ND, Reed JL, Palsson BO: Genome-scale models microbial cells: <br /> GSMN.s efficiently implemented Acorn,            Evaluating consequences constraints. Nature Reviews Microbiology grid based architecture. Acorn provides alter-       2004, 2(11):886-897. <br />                                                              2. Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, <br /> native mechanism visualising metabolic pathway               Broadbelt LJ, Hatzimanikatis V, Palsson BO: genome-scale metabolic <br /> maps. CyCsim uses standard KEGG BioCyc                 reconstruction Escherichia coli K-12 MG1655 accounts 1260 <br /> maps, Acorn server allows individual user cre-        ORFs thermodynamic information. Molecular Systems Biology 2007, <br />                                                                  3:18. <br /> ate views, linked model.        3. Duarte NC, Herrgard MJ, Palsson BO: Reconstruction validation major advantage Acorn environment                 Saccharomyces cerevisiae iND750, fully compartmentalized genome- CycSim WEbcoli provides source               scale metabolic model. Genome Research 2004, 14(7):1298-1309. <br />  Sroka et al. BMC Bioinformatics 2011, 12:196                                                                                                      Page 10 10 <br /> http://www.biomedcentral.com/1471-2105/12/196 <br />  <br />  <br />  <br />  <br /> 4.    Oh YK, Palsson BO, Park SM, Schilling CH, Mahadevan R: Genome-scale <br />       reconstruction metabolic network Bacillus subtilis based high-        doi:10.1186/1471-2105-12-196 <br />                                                                                      Cite article  Sroka et al.: Acorn: grid computing       throughput phenotyping gene essentiality data. Journal Biological <br />                                                                                      constraint based modeling visualization genome scale <br />       Chemistry 2007, 282(39):28791-28799. <br />                                                                                      metabolic reaction networks web interface. BMC Bioinformatics <br /> 5.    Poolman MG, Miguet L, Sweetlove LJ, Fell DA: Genome-Scale Metabolic          2011 12:196. <br />       Model Arabidopsis Properties. Plant Physiology 2009, <br />       151(3):1570-1581. <br /> 6.    Beste DJV, Hooper T, Stewart G, Bonde B, Avignone-Rossa C, Bushell M, <br />       Wheeler P, Klamt S, Kierzek  McFadden J: GSMN-TB: web-based <br />       genome scale network model Mycobacterium tuberculosis <br />       metabolism. Genome Biology 2007, 8(5). <br /> 7.    Becker SA, Palsson BO: Genome-scale reconstruction metabolic <br />       network Staphylococcus aureus N315: initial draft  <br />       dimensional annotation. Bmc Microbiology 2005, 5:12. <br /> 8.    Oberhardt MA, Puchalka J, Fryer KE, dos Santos V, Papin JA: Genome-scale <br />       metabolic network analysis opportunistic pathogen Pseudomonas <br />       aeruginosa PAO1. Journal Bacteriology 2008, 190(8):2790-2803. <br /> 9.    Borodina  Krabben P, Nielsen J: Genome-scale analysis Streptomyces <br />       coelicolor A3(2) metabolism. Genome Research 2005, 15(6):820-829. <br /> 10.   Chung BKS, Selvarasu S, Andrea C, Ryu J, Lee H, Ahn J, Lee DY: Genome- <br />       scale metabolic reconstruction silico analysis methylotrophic <br />       yeast Pichia pastoris strain improvement. Microbial Cell Factories 2010, <br />       9:15. <br /> 11.   Duarte NC, Becker SA, Jamshidi N, Thiele  Mo ML, Vo TD, Srivas R, <br />       Palsson BO: Global reconstruction human metabolic network <br />       based genomic bibliomic data. Proceedings National <br />       Academy Sciences United States America 2007, 104(6):1777-1782. <br /> 12.   Jerby L, Shlomi T, Ruppin E: Computational reconstruction tissue- <br />       specific metabolic models: application human liver metabolism. Mol <br />       Syst Biol 2010, 6:401. <br /> 13.   Gille C, B&#195;&#182;lling C, Hoppe  Bulik S, Hoffmann S, Ganeshan R, K&#195;&#182;nig M, <br />       Rother K, Weidlich M, Behre J, Holzhutter HG: HepatoNet1:       comprehensive metabolic reconstruction human hepatocyte       analysis liver physiology. Mol Syst Biol 2010, 6:411. <br /> 14.   Mahadevan R, Schilling CH: effects alternate optimal solutions       constraint-based genome-scale metabolic models. Metabolic Engineering <br />       2003, 5(4):264-276. <br /> 15.   Camus JC, Pryor MJ, M&#195;&#169;digue C, Cole ST: annotation genome <br />       sequence Mycobacterium tuberculosis H37Rv. Microbiology 2002, <br />       148:2967-2973. <br /> 16.   Le Fevre F, Smidtas S, Combe C, Durot M, d&#226;&#8364;&#8482;Alche-Buc F, Schachter V: <br />       CycSim online tool exploring experimenting genome- <br />       scale metabolic models. Bioinformatics 2009, 25(15):1987-1988. <br /> 17.   Kanehisa M, Goto S, Furumichi M, Tanabe M, Hirakawa M: KEGG       representation analysis molecular networks involving diseases <br />       drugs. Nucleic Acids Res 2009, 38:D355-D360. <br /> 18.   Karp PD, Paley SM, Krummenacker M, Latendresse M, Dale JM, Lee TJ, <br />       Kaipa P, Gilham F, Spaulding  Popescu L, Altman T, Paulsen  Keseler IM, <br />       Caspi R: Pathway Tools version 13.0: integrated software pathway/ <br />       genome informatics systems biology. Brief Bioinform 2010, 11. <br /> 19.   Jung TS, Yeo HC, Reddy SG, Cho WS, Lee DY: WEbcoli: interactive       asynchronous web application silico design analysis       genome-scale E.coli model. Bioinformatics 2009, 25(21):2850-2852. <br /> 20.   Cvijovic M, Olivares-Hernandez R, Agren R, Dahr N, Vongsangnak W, <br />       Nookaew  Patil KR, Nielsen J: Nucleic Acids Research 2010, 38:W145-W149. <br /> 21.   Hucka M, Finney  Sauro HM, Bolouri H, Doyle JC, Kitano H, Arkin AP, <br />       Bornstein BJ, Bray D, Cornish-Bowden  et al: systems biology <br />       markup language (SBML): medium representation exchange       biochemical network models. Bioinformatics 2003, 19(4):524-531.                   Submit manuscript BioMed Central <br /> 22.   Bushell  Sequeira SIP, Khannapho C, Zhao H, Chater KF, Butler MJ,              advantage  <br />       Kierzek  Avignone-Rossa CA: use genome scale metabolic flux <br />       variability analysis process feed formulation based                                                                                         &#226;&#8364;&#162; Convenient online submission <br />       investigation effects zwf mutation antibiotic production <br />       Streptomyce <br /> </body></html>