<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                   Cell Research (2013) 23:491-507. <br />                                                                    &#194;&#169; 2013 IBCB, SIBS, CAS rights reserved 1001-0602/13 <br />                                                                                                                                   npg <br /> ORIGINAL ARTICLE                                                   www.nature.com/cr <br />  <br />  <br />  <br /> XBP-1u suppresses autophagy promoting degradation FoxO1 cancer cells <br /> Ying Zhao1, Xue Li1, Mu-Yan Cai2, Ke Ma1, Jing Yang1, Jingyi Zhou1, Wan Fu1, Fu-Zheng Wei1, Lina Wang1, <br /> Dan Xie2, Wei-Guo Zhu1, 3 <br /> 1 <br />  Key Laboratory Carcinogenesis Translational Research Ministry Education, Department Biochemistry Molecu- <br /> lar Biology, Peking University Health Science Center, Beijing 100191, China; 2State Key Laboratory Oncology South China, <br /> Sun Yat-Sen University Cancer Center, Guangzhou, Guangdong 510060, China; 3Peking-Tsinghua University Center Life Sci- <br /> ences, Peking University, Beijing 100871, China <br />  <br />    Autophagy activated maintain cellular energy homeostasis response nutrient starvation.  au- <br /> tophagy persistently activated, poorly understood mechanistic level.  report turn- FoxO1 involved dynamic autophagic process caused glutamine starvation. X-box-binding protein- <br /> 1u (XBP-1u) critical role FoxO1 degradation recruiting FoxO1 20S proteasome. addition, phosphorylation XBP-1u extracellular regulated protein kinases1/2 (ERK1/2) Ser61 Ser176 critical increased interaction XBP-1u FoxO1 glutamine starvation. Furthermore, <br /> knockdown XBP-1u caused sustained level FoxO1 persistent activation autophagy, leading significant decrease cell viability. Finally, inverse correlation XBP-1u FoxO1 expression agrees expression profiles observed human cancer tissues.  findings link dynamic process autophagy XBP-1u-induced FoxO1 degradation. <br /> Keywords: FoxO1; XBP-1u; ERK; autophagy; cancer <br /> Cell Research (2013) 23:491-507. doi:10.1038/cr.2013.2; published online 1 January 2013 <br />  <br />  <br />  <br />  <br /> Introduction                                                       death. example, autophagy commonly occurs                                                                    hours serum starvation, provides <br />    Autophagy process intracellular degradation             initial burst free amino acids synthesis delivers cytoplasmic constituents lysosome             essential proteins;  autophagic process maintenance homeostasis bioenergetics            rarely observed 4-8 h serum starvation mammalian cells [1-4]. Autophagy activated              normal rat kidney cells mouse fibroblasts [6, 7]. <br /> number stressful conditions, including serum starva-               regulators reported impor- <br /> tion oxidative stress, conditions           tant induction autophagy, including beclin1, lead increased protein misfolding [1, 5].          mammalian target <span id='am-5' about='obo:IMR_0002818' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:id" content="IMR:0002818" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="rapamycin" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C07909" datatype="xsd:string"></span>rapamycin</span> (mTOR) FoxO induction autophagy, mammalian cells able               family members [8-12]. Mammalian FoxO transcription obtain influx free amino acids essen-           factors, including FoxO1, FoxO3a, FoxO4 FoxO6, <br /> tial nutrients.  autophagy rarely persistently          homologs Caenorhabditis elegans abnormal dauer <br /> activated response stress, instead activated          formation-16 (DAF-16) Drosophila dFoxO [13- dynamic manner avoid autophagy-induced cell                16]. FoxO transcription factors involved                                                                    important biological processes, cell cycle arrest, <br />                                                                    DNA repair, apoptosis, glucose metabolism aging <br /> Correspondence: Wei-Guo Zhua, Ying Zhaob                           [13, 17]. Recent reports raised possibility <br /> Tel: +86-10-82802235                                                                    FoxO family members involved induction <br />  E-mail: zhuweiguo@bjmu.edu.cn                                     autophagy transcription-dependent   <br /> b <br />   E-mail: zhaoying0812@ bjmu.edu.cn <br /> Received 14 November 2012; revised 21 November 2012; accepted 22   dependent pathways [10-12, 18]. Loss FoxO protein <br /> November 2012; published online 1 January 2013                     responsible attenuation autophagy  npg FoxO1 degradation interacting XBP-1u <br /> 492 <br />  <br />       Drosophila, mouse muscle cells cancer cell        promotion tumor development. <br />       lines [10-12, 19]. <br />          FoxO family proteins key regulators multi-       Results <br />       ple important biological processes, factors influence <br />       fate FoxO1 great researchers.    Dynamic induction autophagy associated       reported FoxO proteins degraded         changes FoxO1 levels <br />       proteasomal pathway response insulin           Starvation caused removal amino acids <br />       growth factors. Additionally, oncoprotein S-phase ki-     treatment Earle&#226;&#8364;&#8482;s balanced salt solution (EBSS) <br />       nase-associated protein 2 (Skp2) shown ubiq-      induces rapid autophagy [31], makes dif- <br />       uitinate FoxO1 promote degradation [20]. FoxO1        ficult study dynamics process. induce <br />       degraded ubiquitin-proteasome pathway       long-term autophagic model, used glutamine-free <br />       Fao hepatoma cells, process mediated COP1 &#226;&#8364;&#8221;            medium starve human cancer cells. shown Fig- <br />       RING-finger E3 ubiquitin ligase [21]. Finally, ubiq-      ure 1A, autophagy induced human colon  <br />       uitin E3 ligase MDM2 bind FoxO1 FoxO3            <span id='am-43' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-44' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span>cer</span> HCT116 cell line response glutamine starvation, <br />       promote polyubiquitination degradation       reached peak 24 h, measured p62 degra- <br />       proteins [22, 23]. addition factors mentioned       dation LC3-II accumulation, common autopha- <br />        X-box-binding protein-1s (XBP-1s) recently         gic hallmarks. Interestingly, p62 degradation LC3-II <br />       reported interact FoxO1 cause            accumulation gradually returned normal levels 36 <br />       ubiquitin-dependent proteasomal degradation FoxO1          h post treatment (statistical data shown Supplementary <br />       MEF cells [24], mechanism FoxO1            information, Figure S8). Autophagy directly <br />       degradation induced XBP-1s fully understood.        observed electron microscopy, used          XBP-1s transcriptional activator unfolded      monitor formation autophagosomes,       protein response (UPR) critical factor      fluorescent microscopy, used observe LC- <br />       regulation genes involved cell survival [25, 26].   3-immunopositive punctate signals. Numerous vesicular <br />       Unconventional splicing XBP-1 mRNA result          structures double membranes commonly ob- <br />       alternate proteins, XBP-1s  spliced form)      served HCT116 cells incubated glutamine- <br />       XBP-1u  unspliced form). Following endoplasmic            free medium 12-24 h, 48 h (Figure 1B). <br />       reticulum (ER) stress, XBP-1 pre-mRNA cleaved          treatment conditions, punctate LC3 <br />       activated inositol-requiring enzyme 1 (IRE1)        signals consistently observed HCT116 cells. <br />       form &#226;&#8364;&#339;mature&#226;&#8364;?  e., spliced) XBP-1 mRNA,           staining significantly decreased 36 h gluta- <br />       encodes XBP-1s [27]; XBP-1s bind ER               starvation (Figure 1C 1D), indicating       stress-response elements induce transcription       observed autophagy dynamic process.  <br />       ER chaperones [28]. addition regulating UPR,        sought determine key regulators controlling <br />       XBP-1 implicated regulation           changes. FoxO1 acetylation involved regu- <br />       autophagy murine nervous [29]. XBP-             lating autophagic process [10], measured <br />       1 deficiency resulted increased levels FoxO1        FoxO1 levels treated cells. shown Figure <br />       enhanced macroautophagy cellular animal models         1A (lower panels), FoxO1 levels gradually decreased       Huntington&#226;&#8364;&#8482;s disease, authors did         response glutamine starvation. Surprisingly,       interpret XBP-1 deficiency related autophagy,       FoxO1 acetylation increased 2 h glutamine <br />       FoxO1 level negatively correlated XBP-        starvation reached peak 24 h treatment, <br />       1 level [30].                                                 detected immunoprecipitation assay (Fig- <br />          study, results demonstrate FoxO1 turn-    ure 1A, lower panels, 1D). FoxO1 acetylation       highly associated dynamic autophagy          highly correlated interaction Atg7 (Figure <br />       process. Phosphorylation XBP-1u ERK required         1A, lower panels) autophagic changes induced <br />       XBP-1u binding FoxO1, leads localiza-         glutamine starvation HCT116 cells (Figure <br />       tion FoxO1 20S proteasome degradation.          1D). measure overall rates long-lived protein <br />       Interestingly, identified inverse correlation      degradation, HCT116 cells pre-labeled [3H]- <br />       XBP-1u FoxO1 expression 229 cases              leucine 48 h prior glutamine starvation presence <br />       human colorectal cancer tissues. relationship       absence 3-methyladenine (3-MA), inhibitor       tween XBP-1u FoxO1 expression           autophagy. Figure 1E shows glutamine starvation <br />       highly correlated p62 status cancer tis-     significantly enhanced overall degradation proteins <br />       sues, suggests XBP-1 critical protein     HCT116 cells time-dependent manner.  <br />  <br />  <br />                                                                                               Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                          Ying Zhao et al . npg <br />                                                                                                                                           493 <br />  <br />  <br />  <br />  <br />    Figure 1 Dynamic induction autophagy associated changes FoxO1 levels.   Western blotting performed <br />    determine levels total FoxO1 expression, p62 degradation LC3-II accumulation HCT116 cells response    glutamine starvation 72 h (upper panels). Proteins extracted immunoprecipitation anti-FoxO1 <br />    antibody probed anti-Atg7 anti-acetylated lysine (lower panels). (B) Electron micrographs showing autopha- <br />    gic vesicles HCT116 cells starved 48 h. (C) LC3-immunopositive punctate signals observed    HCT116 cells glutamine starvation 72 h. (D) quantitative analysis LC3-positive punctate cells shown    C (upper panels). criterion counted cell 10 puncta. analysis changes acetylated <br />    FoxO1 shown (lower panels). band scanned relative intensive average density determined. data <br />    presented mean &#194;&#177; SD (n = 3). (E) HCT116 cells labeled [3H]-leucine 48 h, washed, incubated 24 h <br />    subjected glutamine starvation 72 h presence absence 3-MA. relative degradation    long-lived proteins measured counting radioactivity. Data mean &#194;&#177; SD (n = 3). (F, G) Western blotting    performed determine levels total FoxO1 expression, FoxO1 acetylation, p62 degradation LC3-II accumulation    A549 (F), HeLa cells (G) response glutamine starvation 48 h. <br />  <br />  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 494 <br />  <br />       3-MA-sensitive degradation gradually decreased       Figure 2E, degradation endogenous FoxO1       36 h glutamine removal HCT116 cells, indicat-      observed cytoplasmic fraction. Taken  <br />       ing long-lived protein degradation 36 h   data demonstrate glutamine starvation-induced <br />       glutamine starvation maybe depend forms         FoxO1 degradation occurs cytoplasm       autophagy macroautophagy [6].                    nucleus. <br />          determine phenomenon occurs              Finally, determine ubiquitination  <br />       cell lines, assayed FoxO1 protein levels      quired FoxO1 degradation, plasmids expressing flag- <br />       acetylation status A549 HeLa cell lines    tagged FoxO1 HA-ubiquitin transfected       glutamine-starvation conditions (FoxO1 highly ex-       H1299 cells; FoxO1 ubiquitination detected  <br />       pressed A549 HeLa cells, virtually expres-    immunoprecipitation anti-flag antibody followed <br />       sion detected H1299 cells [10]). Similar     immunoblotting anti-HA antibody. Contrary <br />       observed HCT116 cells, FoxO1 levels gradu-        expectations, unable detect polyu- <br />       ally decreased FoxO1 acetylation increased fol-     biquitinated FoxO1 H1299 cells response glu- <br />       lowing treatment, correlated changes        tamine starvation (Figure 2F).  investigated <br />       p62 degradation LC3-II accumulation cell     20S proteasome plays role FoxO1 deg- <br />       lines (Figure 1F 1G). data suggest dy-   radation. assayed immunoprecipitation, flag- <br />       namic autophagic process likely associated    tagged FoxO1 physically associated 20S <br />       FoxO1 acetylation.                                          proteasome (Figure 2G), confirmed                                                                   vitro protein degradation assay (Figure 2H).       FoxO1 degraded 20S proteasome                     data suggest FoxO1 degraded 20S proteasome. <br />          determine reduction FoxO1 protein <br />       levels result protein degradation gene     XBP-1u required glutamine starvation-induced <br />       pression, measured FoxO1 mRNA levels using real-         FoxO1 degradation <br />       time PCR. Glutamine starvation effect FoxO1          possible molecules reported       mRNA levels (Figure 2A), indicating protein degra-     involved degradation FoxO proteins, including <br />       dation likely responsible decrease FoxO1     Skp2 [20], MDM2 [22] XBP-1s [24]. determine <br />       protein levels glutamine starvation.                  proteins associated FoxO1 degra- <br />          determine pathways involved FoxO1 deg-         dation response glutamine starvation, designed <br />       radation, pretreated HCT116 cells panel        siRNA fragments proteins. Using <br />       inhibitors, proteasome inhibitor MG132, calpain     RNAi technique, excluded Skp2 MDM2       inhibitor calpeptin lysosome inhibitor chloro-      molecules required FoxO1 degradation response <br />       quine (CHQ). MG132 significantly blocked glu-           glutamine starvation (data shown). tested <br />       tamine starvation-induced reduction FoxO1 levels         role XBP-1 FoxO1 degradation transfect- <br />       (Figure 2B) calpeptin CHQ effect      ing plasmids encoding XBP-1u XBP-1s HCT116 <br />       FoxO1 degradation (Figure 2C). results suggest        cells. XBP-1s nucleus-localized protein XBP- <br />       FoxO1 degraded proteasomal pathway        1u predominately localized cytoplasm (Supple- <br />       response glutamine starvation.                           mentary information, Figure S1A). shown Figure <br />           determine FoxO1 degraded          3A, FoxO1 protein level markedly reduced       nucleus cytoplasm, ectopically expressed      overexpression XBP-1u, overexpression       FoxO1 H1299 cells, cell line normally    XBP-1s. addition, knockdown XBP-1u significantly <br />       low levels FoxO1 [10]. generated plasmids          abrogated glutamine starvation-induced reduction       express flag-tagged wild-type FoxO1 (WT) mutated         FoxO1 protein HCT116 cells (Figure 3B Supple- <br />       FoxO1 (3A) residues T24, S256 S319         mentary information, Figure S1B). performed       FoxO1 changed alanine. mutant           rescue experiment using plasmid specifically express- <br />       protein FoxO1 (3A) predominantly localized       ing mutated XBP-1u validate XBP-1u&#226;&#8364;&#8482;s activity <br />       nucleus. shown Figure 2D, significant decrease   induction FoxO1 degradation HCT116 cells. <br />       FoxO1 (WT) protein levels observed response       XBP-1u expressing plasmid rescued reduction <br />       glutamine starvation, FoxO1 (3A) protein levels     FoxO1 degradation XBP-1u RNAi absence <br />       unchanged. characterize process,       glutamine (Figure 3C), showing XBP-1u <br />       nuclear cytoplasmic fractions prepared        responsible cytosolic FoxO1 degradation  <br />       glutamine-starved HCT116 cells subjected        sponse glutamine starvation. <br />       immunoblotting FoxO1 antibody. shown              confirm role XBP-1u degradation  <br />  <br />                                                                                             Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                          Ying Zhao et al . npg <br />                                                                                                                                           495 <br />  <br />  <br />  <br />  <br />    Figure 2 FoxO1 degraded 20S proteasome pathway.   Quantitative PCR (qPCR) performed measure <br />    FoxO1 mRNA expression levels HCT116 cells response glutamine starvation 72 h. (B, C) Western blotting    performed determine FoxO1 levels HCT116 cells response glutamine starvation 48 h MG132 <br />    (2 &#206;&#188;M) (B), CHQ (10 &#206;&#188;M) calpeptin (50 &#206;&#188;M) (C). (D) H1299 cells transfected plasmids encoding FoxO1 (WT) <br />    FoxO1 (3A). 24 h transfection, H1299 cells glutamine starved 24 h presence absence MG132, <br />    proteins extracted, western blotting performed detect changes FoxO1 levels. (E) HCT116 cells    incubated glutamine-free medium presence absence MG132 48 h. Nuclear cytoplasmic proteins    extracted, FoxO1 levels analyzed western blotting. HDAC1 used loading control nuclear proteins, <br />    tubulin used loading control cytoplasmic proteins. (F) H1299 cells cotransfected plasmids encod- <br />    ing flag-FoxO1 HA-ubiquitin, followed incubation glutamine-free medium presence MG132 (2 &#206;&#188;M) 24 h. <br />    Cell lysates immunoprecipitated anti-flag antibody blotted anti-ubiquitin antibody. panel <br />    indicates equal amounts FoxO1. Ubiquitination p53 used positive control. (G) H1299 cells transfected <br />    flag-FoxO1. 24 h transfection, H1299 cells glutamine starved 48 h, cell lysates immuno- <br />    precipitated anti-flag antibody blotted anti-20S proteasome antibody. (H) vitro-translated FoxO1 protein <br />    incubated 20S proteasome presence absence MG132 (10 &#206;&#188;M) 90 min. Following incubation, <br />    samples examined western blotting analysis. <br />  <br />  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 496 <br />  <br />  <br />  <br />  <br />          Figure 3 XBP-1u involved induction FoxO1 degradation.   Flag-tagged XBP-1u, XBP-1s plasmid <br />          individually transfected HCT116 cells, endogenous FoxO1 protein levels measured western blotting. (B) <br />          HCT116 cells transfected XBP-1u-specific non-specific siRNA. 48 h transfection, HCT116 cells glu- <br />          tamine starved 48 h, cell lysates extracted quantification XBP-1u FoxO1 levels. (C) rescue ex- <br />          periment using plasmid specifically expressing mutated XBP-1u performed validate XBP-1u&#226;&#8364;&#8482;s activity induction <br />          FoxO1 degradation HCT116 cells presence absence glutamine starvation. (D) HCT116 cells glutamine <br />          starved 48 h presense MG132, cell lysates extracted immunoprecipitation anti-XBP- <br />          1u antibody followed probing anti-FoxO1 antibody. (E) length XBP-1u N- C-terminal fragments <br />          XBP-1u expressed bacteria purified; proteins incubated vitro-translated FoxO1. West- <br />          ern blotting performed detect interaction XBP-1u FoxO1. (F) H1299 cells transfected plasmids <br />          encoding GFP-FoxO1 flag-XBP-1u, immunofluorescence performed staining anti-flag antibody. (G) <br />          vitro-translated FoxO1 protein incubated 20S proteasome supplemented GST (negative control) various <br />          GST-XBP-1u fusion proteins; 1 h incubation, samples examined western blotting analysis. (H) GST          GST-FoxO1 fusion protein incubated 20S proteasome presence absence vitro-translated wild-type          mutant XBP-1u. 20S proteasome GST-FoxO1 fusion protein assayed western blotting analysis. <br />  <br />  <br />  <br />  <br />       FoxO1, tested XBP-1u interacts               blotting performed using anti-FoxO1 anti- <br />       FoxO1 using immunoprecipitation assay. shown               body. shown Figure 3E, C-terminal region       Figure 3D, endogenous FoxO1 interacted XBP-                XBP-1u interacted FoxO1. examine subcel- <br />       1u presence MG132 HCT116 cells              lular localization FoxO1 XBP-1u interaction, <br />       glutamine starvation.  test interac-           transfected H1299 cells plasmids expressing <br />       tion XBP-1u FoxO1 direct, vitro              GFP-FoxO1 flag-XBP-1u, performed immuno- <br />       GST pull assay carried using bacterially              fluorescence assays using anti-flag antibody.       expressed proteins. Wild-type FoxO1 protein               majority wild-type FoxO1 present       translated vitro incubated GST fusion pro-             cytoplasm localized XBP-1u       teins length, C- N-terminal XBP-1u; western             absence presence glutamine (Figure 3F Supple- <br />  <br />                                                                                                        Cell Research | Vol 23 4 | April 2013 <br />                                                                                                            Ying Zhao et al . npg <br />                                                                                                                             497 <br />  <br /> mentary information, Figure S1C). results suggest      tion (Figure 4C), indicating ERK-mediated FoxO1 FoxO1 interacts XBP-1u cytoplasm       degradation does result direct phosphorylation <br /> glutamine starvation.                                        FoxO1 ERK. <br />     FoxO1 degradation 20S proteasome            FoxO1 interacted XBP-1u glutamine enhanced incubation length XBP-1u           starvation, rule possibility ERK <br /> protein. contrast, N-terminal XBP-1u protein,       directly phosphorylate XBP-1u, turn, observed directly interact FoxO1, did        phosphorylated XBP-1u play role FoxO1 affect FoxO1 degradation 20S proteasome           degradation.  glutamine starva- <br /> (Figure 3G Supplementary information, Figure             tion induced phosphorylation endogenous S2). Furthermore, examined effects XBP-1u          overexpressed XBP-1u HCT116 cells, shown FoxO1/20S proteasome interaction. shown            immunoprecipitation assay (Figure 4D Supple- Figure 3H, 20S proteasome fraction able        mentary information, Figure S3B). Using software <br /> interact FoxO1 demonstrated GST-FoxO1           program GPS2.1, identified residues XBP-1u, <br /> pull assay. addition vitro translated    Ser61 Ser176, putative ERK target sites.  <br /> length XBP-1u, N-terminal XBP-1u, facilitated        fore, generated plasmid encoding mutated XBP- FoxO1/20S proteasome interaction. Taken         1u (S61/176A) serine residues data suggest interaction XBP-1u       changed alanine. transfection HCT116 cells, FoxO1 required degradation FoxO1        confirmed glutamine starvation did induce <br /> 20S proteasome.                                              XBP-1u (S61/176A) phosphorylation (Figure 4E                                                              Supplementary information, Figure S3C). purified ERK pathway involved XBP-1u-mediated               XBP-1u ERK1 incubated vitro  <br /> FoxO1 degradation                                            vestigate ERK responsible XBP-1u phos- <br />    Prior degradation, FoxO1 usually post-transla-      phorylation. shown Figure 4F, XBP-1u signifi- <br /> tionally modified phosphorylation [32, 33].         cantly phosphorylated ERK1. data suggest fore, used characterized kinase inhibitors,      ERK activity required XBP-1u modulate <br /> PD098059, inhibitor ERK, SB203580,              degradation FoxO1. <br /> inhibitor p38, investigate cellular signaling <br /> pathway involved XBP-1u-mediated FoxO1           Phosphorylation XBP-1u ERK critical  <br /> degradation. HCT116 cells pre-incubated            creased interaction XBP-1u FoxO1 <br /> PD098059 SB203580 30 min followed incuba-             test phosphorylation XBP-1u tion glutamine presence inhibi-    ERK effect interaction XBP-1u tor 24 h. Blockage ERK PD098059            FoxO1, HCT116 cells treated completely inhibited glutamine starvation-induced            ERK inhibitor PD098059 exposed glutamine <br /> FoxO1 degradation; SB203580 did inhibit degrada-         starvation. Figure 5A Supplementary information, <br /> tion (Figure 4A). confirm role ERK      Figure S4A PD098059 significantly disrupted <br /> FoxO1 degradation, HCT116 cells transfected        interaction FoxO1 XBP-1u. test <br /> non-specific ERK-specific siRNA, FoxO1 degra-         decreased interaction FoxO1 dation assayed. efficiency RNAi       XBP-1u changes XBP-1u localization, <br /> ERK significant, expected, glutamine         collected nuclear cytoplasmic fractions starvation, FoxO1 degraded cells transfected      HCT116 cells treated control siRNA. contrast, FoxO1 remained     PD098059 prior glutamine starvation. Figures 5B, 5C <br /> higher level cells transfected ERK siRNA     Supplementary information, Figure S4B (Figure 4B Supplementary information, Figure 3A),        majority cytoplasmic XBP-1u translocated suggesting ERK plays critical role FoxO1 deg-      nucleus presence PD098059 gluta- <br /> radation.                                                    starvation. investigate role phos- <br />    determine ERK directly phosphorylates          phorylation nuclear export XBP-1u, XBP- <br /> FoxO1 induce FoxO1 degradation, plasmid encoding        1u (WT), XBP-1u (S61A), XBP-1u (S176A) XBP- mutated FoxO1 (9A) putative ERK          1u (S61/176A) plasmids separately transfected <br /> phosphorylation sites FoxO1 mutated alanine,      H1299 cells, followed immunostaining or plasmid encoding wild-type FoxO1 (WT), sep-        nuclear/cytoplasmic extraction. shown Figures <br /> arately transfected H1299 cells. FoxO1 (WT)        5D, 5E Supplementary information, Figure S4C, FoxO1 (9A) degraded glutamine starva-         XBP-1u (S61/176A) localized nucleus <br />  <br />  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 498 <br />  <br />  <br />  <br />  <br />          Figure 4 Activation ERK signaling pathway involved FoxO1 degradation.   Western blotting performed          detect FoxO1 levels HCT116 cells presence PD098059 (10 &#206;&#188;M) SB203580 (10 &#206;&#188;M) response glutamine <br />          starvation 48 h. (B) HCT116 cells transfected ERK1/2-specific non-specific siRNA. 48 h transfection, <br />          HCT116 cells glutamine starved 24 h MG132 (2 &#206;&#188;M), cell lysates extracted          western blotting using anti-ERK1/2 anti-FoxO1 antibodies. (C) H1299 cells transfected flag-tagged FoxO1 (WT) <br />          FoxO1 (9A). 24 h transfection, cells subjected glutamine starvation 24 h. cell lysates extract- <br />          ed immunoblotted anti-flag anti-&#206;&#178;-actin antibodies. (D) HCT116 cells incubated glutamine-free medium <br />          72 h, cell lysates extracted immunoprecipitation anti-XBP-1u antibody probed anti- <br />          phosphoserine antibody. Cell extraction XBP-1u XBP-1s overexpressed used positive  <br />          trol. (E) HCT116 cells transfected plasmids encoding XBP-1u (WT) XBP-1u (S61/176A). 24 h transfection, <br />          cell lysates extracted immunoprecipitation anti-XBP-1u antibody probed anti-phosphoserine <br />          antibody. (F) GST-XBP-1u incubated ERK1 30 &#194;&#176;C 60 min reaction terminated addi- <br />          tion sample loading buffer. mixtures subjected Mn2+-Phos-tag SDS-PAGE, analyzed western <br />          blotting using anti-XBP-1 antibody. <br />  <br />  <br />  <br />  <br />       cytoplasm, XBP-1u (WT) mainly                    Figure S4E), indicating Ser61 Ser176 es- <br />       localized cytoplasm. interaction 20S               sential XBP-1u interaction 20S proteasome. <br />       proteasome XBP-1u disrupted XBP-                   Furthermore, XBP-1u mutants lacking ERK phosphory- <br />       1u (S61/176A) expressed presence absence               lation sites (S61/176A) lacking 20S proteasome bind- <br />       glutamine (Figure 5F Supplementary information,               ing sites (1-204 aa) tested ability  <br />  <br />                                                                                                           Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                           Ying Zhao et al . npg <br />                                                                                                                                            499 <br />  <br />  <br />  <br />  <br />    Figure 5 Phosphorylation XBP-1u ERK critical increased interaction XBP-1u FoxO1.   HCT116 cells <br />    incubated glutamine-free medium presence absence PD098059 (10 &#206;&#188;M) 24 h. cell lysates    extracted immunoprecipitation anti-XBP-1u antibody probed anti-FoxO1 antibody. (B) Cell frac- <br />    tions extracted treatment analyzed determine cellular localization XBP-1u. HDAC1 used <br />    loading control nuclear proteins, tubulin used loading control cytoplasmic protein. (C) Localization    XBP-1u determined transfection plasmid encoding flag-XBP-1u HCT116 cells, incubated glu- <br />    tamine-free medium presence absence PD098059. 24 h treatment, immunostaining performed using <br />    anti-flag antibody. quantification, 100 cells coverslip counted, results shown percentage    cells. values shown C mean &#194;&#177; SD (n = 3). (D) Nuclear cytoplasmic proteins extracted, western <br />    blotting performed analyze cellular localization wild-type mutant XBP-1u. (E) localization wild-type    mutant XBP-1u determined transfection XBP-1u HCT116 cells immunostaining anti-flag antibody (left <br />    panel). quantification, 100 cells coverslip counted, results shown percentage cells.    values shown E mean &#194;&#177; SD (n = 3) (right panel). (F) 24 h plasmids encoding flag-XBP-1u (WT) flag-XBP- <br />    1u (S61/176A) transfected HCT116 cells, cell lysates extracted immunoprecipitation anti- <br />    flag antibody probed anti-20S proteasome antibody. (G) HCT116 cells transfected control plasmid    plasmids encoding flag-XBP-1u (WT), flag-XBP-1u (S61/176A) flag-XBP-1u (1-204) presence absence MG132 <br />    (2 &#206;&#188;M). 24 h treatment, cell lysates extracted immunoblotted anti-flag, anti-FoxO1 anti-&#206;&#178;-actin <br />    antibodies. <br />  <br />  <br />  <br />  <br /> destabilize FoxO1. Overexpression XBP-1u (WT),                    (Figure 5G), induced decrease FoxO1 protein levels. XBP-1u (S61/176A) XBP-1u (1-204 aa)                   addition, single mutated XBP-1u (S61A)  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 500 <br />  <br />       XBP-1u (S176A) induced modest degradation                 Supplementary information, Figure S5B, statistical <br />       FoxO1, WT XBP-1u (Supplementary                  data shown Supplementary information, Figure S8). <br />       information, Figure S4D). data indicate               Additionally, glutamine starvation-induced p62 deg- <br />       phosphorylation XBP-1u promotes XBP-1u transloca-                radation markedly reduced XBP-1-knockdown <br />       tion cytoplasm nucleus attenuates           HCT116 cells compared control cells (Figure 6C, <br />       interaction 20S proteasome, leads               statistical data shown Supplementary information, <br />       decrease FoxO1 degradation.                                      Figure S8). Consistently, 48 h glutamine <br />                                                                           starvation, number autophagic cells did drop       Downregulation XBP-1 enhances FoxO1-dependent                    XBP-1-knockdown cell lines observed fluores- <br />       autophagy                                                           cent electron microscopy (Figure 6D 6E).          FoxO1 essential induction autophagy             performed long-lived protein degradation assay       response serum starvation oxidative stress [10],           XBP-1-knockdown HCT116 cells. 3-MA-sensitive <br />       tested XBP-1u suppresses autophagy                   degradation persistently increased glutamine <br />       induction FoxO1 degradation. shown Figure 6A               removal (Figure 6F). contrast, FoxO1/XBP-1 <br />       Supplementary information, Figure S5A, overexpres-              double knockdown HCT116 cells, glutamine starvation <br />       sion XBP-1u significantly decreased FoxO1 protein                did induce p62 degradation LC3-II accumulation <br />       levels, blocked accumulation LC3-II deg-             (Figure 6G). data indicate downregulation       radation p62 induced glutamine starvation. Simi-              XBP-1 promotes autophagy function  <br />       larly, number LC3 puncta significantly decreased             quires FoxO1. <br />       glutamine-starved HCT116 cells transfected <br />       XBP-1u, XBP-1s (Figure 6B). fur-               Persistent activation autophagy XBP-1 knockdown <br />       ther clarify role endogenous XBP-1u plays           associated cell death <br />       induction autophagy, XBP-1 RNAi plasmid                      known autophagic death non-apoptot- <br />       transfected HCT116 cells establish stable XBP-1-            ic form programmed cell death. investigate       knockdown cell lines; non-specific RNAi plasmid               autophagy induced XBP-1 knockdown effect       used negative control. cell lines sepa-             overall survival cancer cells, tested viability <br />       rately cultured glutamine-free medium 48 h.            HCT116 cells XBP-1 knockdown  <br />       Cell lysates extracted western blotting detect          der conditions glutamine starvation. expected, fol- <br />       changes p62 degradation LC3-II accumulation.                 lowing glutamine starvation 72 h, cell viability       LC3-II levels dropped 36 h glutamine starvation            significantly decreased XBP-1-knockdown HCT116 <br />       cells containing normal XBP-1 levels (Figure 1A),                cells compared wild-type HCT116 cells (Figure <br />       XBP-1-knockdown cells, LC3-II levels did                 7A). Interestingly, cell death inhibited HCT116 <br />       drop 72 h glutamine starvation (Figure 6C           cell lines stably transfected XBP- <br />  <br />  <br />  <br />          Figure 6 Downregulation XBP-1u induces autophagy.   HCT116 cells transfected control plasmid plas- <br />          mid encoding flag-XBP-1u flag-XBP-1s. 24 h transfection, cells subjected glutamine starvation 12 h          presence absence E64/pepstatin  cell lysates extracted immunoblotted anti-flag, anti-FoxO1, <br />          anti-LC3, anti-p62 anti-&#206;&#178;-actin antibodies. (B) HCT116 cells transfected control plasmid plasmid encoding <br />          flag-XBP-1u flag-XBP-1s. 24 h transfection, cells subjected glutamine starvation 24 h, forma- <br />          tion LC3 punctate signals observed (left panels). criterion counted cell 10 puncta. <br />          Quantification ratio LC3 punctate cells total HCT116 cells shown right panel. Values shown B          mean &#194;&#177; SD (n = 3). (C) Stable XBP-1-RNAi HCT116 cells incubated glutamine-free medium 72 h. cell <br />          lysates extracted immunoblotted anti-FoxO1, anti-LC3, anti-p62 anti-&#206;&#178;-actin antibodies. (D) Stable XBP-1- <br />          RNAi HCT116 cells incubated glutamine-free medium 72 h formation LC3 punctate signals ob- <br />          served (upper panels). Quantification ratio LC3 punctate cells total HCT116 cells shown lower panel.          criterion counted cell 10 puncta. Values D mean &#194;&#177; SD (n = 3) (lower panel). (E) Electron <br />          micrographs showing autophagic vesicles stable XBP-1-RNAi HCT116 cell line incubated glutamine-free medium <br />          48 h. (F) Stable HCT116 cell line XBP-1-RNAi labeled [3H]-leucine 48 h, washed, incubated 24 h <br />          subjected glutamine starvation 72 h presence absence 3-MA. relative degradation          long-lived proteins measured counting radioactivity. Data mean &#194;&#177; SD (n = 3). (G) HCT116 cells HCT116 cell <br />          lines stably expressing XBP-1-RNAi XBP-1/FoxO1 double RNAi incubated glutamine-free medium 48 h          presence absence E64/pepstatin  cell lysates extracted immunoblotted anti-XBP-1u, anti-FoxO1, <br />          anti-LC3, anti-p62 anti-&#206;&#178;-actin antibodies. <br />  <br />  <br />                                                                                                         Cell Research | Vol 23 4 | April 2013 <br />                                          Ying Zhao et al . npg <br />                                                           501 <br />  <br />  <br />  <br />  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 502 <br />  <br />       1- FoxO1-RNAi plasmids, suggesting XBP-1            tor influences development cancer. <br />       knockdown-induced cell death glutamine starvation <br />       dependent FoxO1 (Figure 7A). determine        Discussion <br />       type cell death occurring XBP-1 knockdown, <br />       autophagy inhibitor 3-MA apoptosis inhibi-           study, provide evidence autophagy  <br />       tor VAD-fmk used. cell death greatly          duced glutamine starvation dynamic process       suppressed 3-MA, verifying role autophagy       highly associated turnover FoxO1 protein. <br />       cell death caused XBP-1 knockdown glutamine          XBP-1u binds FoxO1 recruits FoxO1 20S <br />       starvation (Figure 7B). data suggest persistent   proteasome degradation. interaction       induction autophagy induces cell death, likely      XBP-1u FoxO1 greatly enhanced ERK1/2, <br />       results excessive degradation cell components        specifically catalyzes phosphorylation       essential survival.                                      XBP-1u Ser61 Ser176; suggests cellular <br />                                                                    signaling pathways regulate autophagic process       Significant correlations XBP-1u, FoxO1           influencing FoxO1 levels (Figure 7E). <br />       p62 expressions demonstrated human column rec-           Earlier studies shown macroautophagy       tum cancer (CRC) tissues                                     constitutively activated process [6, 7, 34, 35].          investigate expression profiles XBP-1,          believed macroautophagy activated       FoxO1 p62 CRCs, conducted immunohisto-             hours serum deprivation gradually  <br />       <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> (IHC) staining XBP-1u, FoxO1 p62            turned basal levels. starvation continued       CRC-TMA containing 229 CRC specimens. Using             time, decrease macroautophagy concomitant <br />       criteria described Materials Methods, high        progressive increase chaperone-mediated au- <br />       expression cytoplasmic XBP-1, FoxO1 p62           tophagy (CMA), benefitted cell survival [35].       observed 120 (52.4%), 107 (46.7%) 119 (52.0%)         CMA-deficient cells, high rates sustained protein <br />       229 CRC cases, respectively. Representative IHC       degradation response serum starvation offset <br />       stainings XBP-1u, FoxO1 p62 shown Fig-         upregulation macroautophagy [34]. known <br />       ure 7C 7D. correlation analysis demonstrates      mTOR, important autophagy repressor, reacti- <br />       expression levels XBP-1u CRCs inversely        vated degradation autolysosomal products       correlated expression levels FoxO1 (P &lt; 0.05)    dynamic manner [7]. mTOR reactivation, tubules <br />       (Table 1). addition, expression profiles XBP-      extend autolysosomal membranes rise       1u FoxO1 CRC cohort positively          vesicles ultimately form functional lysosomes [7]. <br />       negatively correlated p62 expression, respectively (P   Consistent previous reports, au- <br />       &lt; 0.05) (Table 1). Importantly, high      tophagy decreased long-term glutamine starvation <br />       levels XBP-1u low levels FoxO1 signifi-        treatment (Figure 1), suggests  <br />       cantly associated survival CRC patients    cer cells autophagy dynamic process. Furthermore, <br />       surgery, tumors lymph node dis-     FoxO1 turnover important factor used  <br />       tant metastasis advanced clinical stage (Sup-     cer cells control autophagic process occurs       plementary information, Figure S7, Tables S1 S2).        response glutamine starvation. <br />       data support hypothesis correct      reported degradation FoxO <br />       balance XBP-1u FoxO1 activities critical fac-    family proteins mediated ubiquitin-proteasome <br />  <br />  <br />  <br />       Table 1 Correlations expressions XBP-1, FoxO1 p62 229 colorectal carcinomas <br />       		                                                  XBP-1 expression	 	                             FoxO1 expression <br />        	                        cases	        Low	           High	         P value*	          Low	         High        P value* <br />        FoxO1 expression				0.016				 <br />          High	                     107	       60 (56.1%)	     47 (43.9%)					 <br />          Low	                      122	       49 (40.2%)	     73 (59.8%)					 <br />        p62 expression				0.005			 0.005 <br />          Low	                      110	       63 (57.3%)	     47 (42.7%)		                    48 (43.6%)	    62 (56.4%)	 <br />          High	                     119	       46 (38.7%)	     73 (61.3%)		                    74 (62.2%)	    45 (37.8%)	 <br />       *Chi-square test. <br />  <br />  <br />                                                                                               Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                         Ying Zhao et al . npg <br />                                                                                                                                          503 <br />  <br />  <br />  <br />  <br />     Figure 7 expression levels XBP-1u FoxO1 associated cell viability inversely correlated human <br />     cancer samples.   HCT116 cells HCT116 cell lines stably expressing XBP-1 RNAi XBP-1/FoxO1 double RNAi     glutamine starved, cells stained PI 72 h treatment. PI-stained (dead) cells counted flow cytometry <br />     percent cell death represented. Values mean &#194;&#177; SD (n = 3). (B) Stable XBP-1 RNAi HCT116 cell lines <br />     glutamine starved 72 h presence absence 3-MA (10 mM), presence absence VAD-fmk (50 <br />     &#206;&#188;M). number cells counted staining PI. Values mean &#194;&#177; SD (n = 3). (C, D) Immunohistochemical <br />     stainings XBP-1, FoxO1 p62 CRC tissues. CRC sample (case 26) showing high expression XBP-1 p62     low expression FoxO1 tumor cells (C). CRC sample (case 139) showing high expression FoxO1 low <br />     expression XBP-1 p62 tumor cells (D). (E) schematic model showing relationship autophagy <br />     XBP-1u-induced FoxO1 degradation response glutamine starvation. <br />  <br />  <br />  <br />  <br /> pathway [32, 33, 36]. demonstrated             believe conflicting results uncovered XBP-1s targets FoxO1 ubiquitin-dependent degrada-                group specific isoform XBP-1 tion MEF cells [24]. study,  dem-              involved degradation FoxO1 explained <br /> onstrate XBP-1u key regulator modulates          different cell lines used respective studies; <br /> FoxO1 degradation response glutamine starvation.               cancer cell lines used study, MEFs  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 504 <br />  <br />       used study. addition, differences treat-   1u alanine (Ser61/176A) diminishes capacity       ment conditions cause cells use alternative     XBP-1u drive FoxO1 degradation (Figure 5G).       responses induce FoxO1 degradation; example,           data support notion ERK-dependent phosphory- <br />       exposed cells glutamine-free medium,        lation XBP-1u Ser61 Ser176 plays significant <br />       group used normal medium.  pres-        role enabling rapid destabilization inactivation <br />       ent important pieces evidence XBP-1u       FoxO1. <br />       true regulator FoxO1 degradation.  glutamine          growing body evidence suggests autoph- <br />       starvation induced significant increase XBP-1u           agy protect cells cell death mediate <br />       mRNA levels slightly increased XBP-1s mRNA           autophagic cell death [40-42]. response starvation, <br />       levels (Supplementary information, Figure S6). Second,        autophagy appears cytoprotective process,       FoxO1 degradation occurred          cells degrade non-essential components order gen- <br />       cytoplasm XBP-1u directly interacted        erate essential nutrients meet cell&#226;&#8364;&#8482;s energetic <br />       FoxO1 (Figures 2D, 2E, 3D 3E);  XBP-              demands maintain biosynthetic reactions [43, 44]. <br />       1s localized nucleus,           autophagy involved cell death,       make difficult directly induce FoxO1 degradation     caused progressive destruction cytoplas- <br />       response glutamine starvation.  demonstrate       mic structures, including mitochondria [45, 46].       XBP-1u recruits FoxO1 20S proteasome        study suggests autophagy protective mechanism <br />       ubiquitin-independent degradation (Figure 3G 3H).         starvation conditions, long autophagic <br />       Consistent data, XBP-1u reported       process persistently activated. autophagy <br />       play important role protein degradation HeLa      constitutively activated, induces cell death, <br />       cells. XBP-1s  ATF6) forms complex XBP-             likely result excessive degradation <br />       1u degraded proteasome; degradation        essential cellular components. Consistently,       domain C-terminus XBP-1u critical        glutamine starvation maintenance FoxO1 <br />       interaction XBP-1u 20S proteasome [37,            activity knocking XBP-1u sufficient       38]. Consistent studies, overex-     decrease cell viability autophagy-dependent man- <br />       pression wild-type XBP-1u, XBP-1u lacking          ner (Figure 7A 7B). Interestingly, recent study       proteasome binding sites (1-204 aa), sufficient        shows XBP-1 deficiency mice led increased <br />       decrease FoxO1 protein levels (Figure 5G).                    expression FoxO1 related enhanced <br />          reported ERK directly phosphorylates      autophagy neurons [29], supporting XBP- <br />       FoxO3 phosphorylated FoxO3 degraded           1 level negatively related FoxO1 level autopha- <br />       MDM2-mediated ubiquitin-proteasome path-              gy. Furthermore, repression XBP-1 expression       way [22].  study, phosphorylation         shown inhibit tumor development animal mod- <br />       FoxO1 key modification ERK1/2-          els [47] human tumor tissues [48]. Consistent       mediated FoxO1 degradation (Figure 4C). example,           XBP-1u highly expressed       FoxO1 degraded 20S proteasome              human colon cancer samples, FoxO1 weakly <br />       vitro, fairly stable vivo, suggesting    expressed (Figure 7C 7D). Taken  data <br />       FoxO1 protein easily accessible 20S         suggest relationship XBP-1u FoxO1 <br />       proteasome cells.  modification     significant relevance clinical research. <br />       FoxO1-interacting proteins, XBP-1u, likely            conclusion, turnover FoxO1 important <br />       contribute FoxO1 destabilization response         mechanism allows cells modulate autophagic <br />       glutamine starvation. possible glutamine      process. response glutamine starvation, phosphory- <br />       starvation, XBP-1u undergoes phosphorylation-induced          lated XBP-1u binds FoxO1 induces FoxO1 deg- <br />       conformational changes facilitate entry FoxO1       radation.  targeting XBP-1u useful <br />       channel 20S proteasome.                       strategy development novel cancer therapies          XBP-1s phosphorylated p38 MAPK                future. <br />       Thr48 Ser61, results nuclear transloca- <br />       tion increased stability XBP-1s [39]. results      Materials Methods <br />       ERK important regulator XBP-1u <br />                                                                     Cell culture <br />       localization. phosphorylation Ser61 Ser176,         A549, HCT116 H1299 cells grown DMEM supple- <br />       XBP-1u prone localize cytoplasm interact     mented 10% fetal bovine serum (heat inactivated 56 &#194;&#176;C       20S proteasome, leading degradation XBP-       45 min) appropriate penicillin/streptomycin       1u FoxO1. Substitution Ser61 Ser176 XBP-        37 &#194;&#176;C incubator humidified, 5% <span id='am-10' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span>CO2</span> atmosphere. <br />  <br />  <br />                                                                                                   Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                              Ying Zhao et al . npg <br />                                                                                                                                               505 <br />  <br /> Plasmids                                                              Equal amounts GST GST fusion proteins incubated <br />    Flag-XBP-1u XBP-1u plasmids kindly provided            <span id='am-15' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-16' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span>glutathione</span>-Sepharose 4B beads (Amersham Biosciences) <br /> Dr Ling Qi (Cornell University, USA). GST-XBP-1, GST-XBP-1            rocking 4 &#194;&#176;C 1 h, beads washed (1-101) GST-XBP-1 (83-261) kindly provided Dr Qi-         times buffer (20 mM Tris-HCl, pH 7.4, 0.1 mM EDTA <br /> nong Ye (Beijing Institute Biotechnology, China). FoxO1 (9A)       100 mM NaCl). Total cell extracts HCT116 cells kindly provided Dr Akiyoshi Fukamizu (University Tsu-       added beads incubated rocking 4 &#194;&#176;C overnight. <br /> kuba, Japan).                                                         beads washed times TENT buffer (0.5% NP- <br />                                                                       40, 20 mM Tris-HCl, pH 7.4, 0.1 mM EDTA 300 mM NaCl), <br /> Immunoblot analysis immunoprecipitation assay                     dissolved 2&#195;&#8212; SDS loading buffer centrifugation <br />     Protein expression detected western blotting previ-     boiled 5 min 100 &#194;&#176;C. centrifuging, supernatant <br /> ously described minor modifications. Equal amounts pro-       extracted analyzed western blotting. <br /> teins (100 150 &#194;&#181;g) separated 9% 15% SDS-PAGE. antibodies used anti-XBP-1 (Santa Cruz), anti-FoxO1           Long-lived protein degradation assay <br /> (#9462, Cell Signaling), anti-20S proteasome (Santa Cruz), anti-         Cells grown 12 plate (1 ml DMEM 10% <br /> LC3 (Cell Signaling), anti-actin (Santa Cruz), anti-Flag (sigma),     FBS  1 day prior experiment. Cells washed, <br /> p62 (MBL) anti-GFP (Santa Cruz).                                  incubated leucine-free medium (leucine-free DMEM + <br />     immunoprecipitation, cells harvested lysed      10% FBS) 1 h exclude endogenous leucine, 5 Ci/ml NP-40 buffer supplemented complete protease inhibi-         [3H]-leucine added medium presence 1 ml <br /> tor cocktail (Roche). cell lysates used immuno-        leucine-free DMEM 10% dialyzed FBS 48 h incubation. <br /> precipitation indicated antibodies. Generally, 1-4 &#194;&#181;g     washing times 1 ml DMEM containing 2 mM <br /> commercial antibody added 1 ml cell lysate,       unlabelled leucine, cells incubated DMEM (containing <br /> incubated 4 &#194;&#176;C 8-12 h. addition protein G agarose   10% FBS 2 mM unlabelled leucine) 24 h degrade short- <br /> beads, incubation continued 1 h. Immunoprecipitates       lived proteins. transfection starvation, cell aliquots (100 extensively washed lysis buffer eluted SDS         &#194;&#181;l) taken detect radioactivity (count minute, cpm) loading buffer boiling 5 min.                                  different time points. medium removed, cells                                                                       lysed placed vial determine total incorporated ra- <br /> RT-PCR                                                                dioactivity. protein degradation rate calculated cpm    Primers human XBP-1 5&apos;-CCTGGTTGCTGAAGAG-                  medium divided total incorporated cpm. <br /> GAGG-3&apos; (forward) 5&apos;-CCATGGGGAGATGTTCTGGAG-3&apos; <br /> (reverse). Primers human XBP-1u real-time PCR 5&apos;-ATG-        20S proteasome degradation assay <br /> GATTCTGGCGGTATT-3&apos; 5&apos;-AAGGGAGGCTGGTAAGG-3&apos;.                          GST-FoxO1 XBP-1u/XBP-1uNT incubated 37 &#194;&#176;C                                                                       HEPES buffer (50 mM HEPES, pH 7.5, 1 mM DTT 0.018% <br /> Cellular fractionation                                                SDS) presence 20S proteasomes 1 h. Samples    Cells suspended buffer (10 mM Hepes, pH 7.9, 10          reaction mixtures subjected immunoblot analysis. <br /> mM KCl, 0.1 mM EDTA, 0.1 mM EGTA, 1 mM DTT, 0.15% NP- <br /> 40 1% cocktail), swollen 10 min ice, centrifuged       vivo phosphorylation analysis XBP-1u phos-tag 12 000&#195;&#8212; g 30 s. Supernatant collected cytoplasmic     acrylamide gel electrophoresis <br /> extract. pellet washed, resuspended buffer B (20 mM            Mn2+-Phos-tag-modified acrylamide gel, phosphory- <br /> Hepes, pH 7.9, 400 mM NaCl, 1 mM EDTA, 1 mM EGTA, 1 mM                lated proteins migrate slower non-phosphorylated proteins DTT, 0.5% NP-40 1% cocktail), rocked 15 min 4 &#194;&#176;C.      interaction phosphate groups Mn2+-Phos-tag (NARD supernatant centrifugation used nuclear extract.     Institute, , Hyogo, Japan). GST-XBP-1u incubated                                                                       GST-ERK (MERCK) kinase buffer (40 mM Tris- <br /> Immunofluorescence analysis                                           HCl, pH 7.5, 80 mM NaCl, 8 mM MgCl2, 0.8 mM DDT 50 <br />     Cells cultured ~80% confluency using Chamber          &#194;&#181;M <span id='am-27' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>) total volume 50 &#194;&#181;l. Reaction mixtures incu- <br /> Slide  stress treatment, cells fixed 4%        bated 30 &#194;&#176;C 60 min terminated addi- <br /> paraformaldehyde permeabilized methanol. Chamber             tion sample loading buffer. mixtures subjected slides incubated blocking solution (2% BSA PBS),         Mn2+-Phos-tag SDS-PAGE (12% polyacrylamide gel including <br /> treated primary antibody (1:100 dilution       50 &#206;&#188;m MnCl2 Phos-tag acrylamide), analyzed western <br /> tibodies) overnight 4 &#194;&#176;C. washed 3 times blocking       blotting using anti-XBP-1 antibody. <br /> buffer, sides treated FITC/TRITC secondary anti- <br /> body. Microscopic observation performed using Olympus          Cell death analysis <br /> BX-51 Research Microscope.                                               Post-treatment cell viability determined staining                                                                       propidium iodide (PI, 2 &#206;&#188;g/ml) flow cytometric analysis Purification GST fusion proteins GST pull ex-             FACScan. <br /> periments <br />    GST GST fusion proteins expressed E. coli BL21          Patients tissue specimens <br /> (Tiangen Biotechnology, Beijing, China), induced 0.1 mM             study, paraffin-embedded pathologic specimens <br /> isopropyl-1-thio-&#206;&#178;-D-galactopyranoside 12 h 16 &#194;&#176;C,         229 patients CRC obtained archives purified following protocol Amersham Biosciences.            Department Pathology, Cancer Center, Sun Yat-Sen University <br />  <br />  <br /> www.cell-research.com | Cell Research <br />  npg FoxO1 degradation interacting XBP-1u <br /> 506 <br />       Guangdong Provincial People&#226;&#8364;&#8482;s Hospital, Guangzhou, China,         identify independent prognostic factors. P &lt; 0.05  <br />       January 2000 November 2006. cases selected            sidered statistically significant. <br />       based distinctive pathologic diagnosis CRC, undergo- <br />       ing primary curative resection CRC, availability resec-    Acknowledgments <br />       tion tissue followup data, received preoperative <br />       anticancer treatment. CRC cases included 142 (62.0%) men <br />                                                                                study supported Ministry Science Tech- <br />       87 (38.0%) women, mean age 57.3 years. Average fol- <br />                                                                             nology China (2011CB910100 YZ; 2011CB504200 WZ), <br />       lowup time 55.42 months (median, 60.0 months; range, 0.5- <br />                                                                             Program New Century Excellent Talents University, Na- <br />       98 months). Patients cause death remained unknown <br />                                                                             tional Natural Science Foundation China (81222028, 30900722, <br />       excluded study. Clinicopathologic characteristics <br />                                                                             31070691 30921062), &#226;&#8364;&#339;111 project&#226;&#8364;? Minister       patients summarized Table 1. study ap- <br />                                                                             Education China. <br />       proved medical ethics committee institutes. <br />  <br />       Tissue microarray (TMA) IHC                                       References <br />           TMAs constructed accordance previously  <br />       scribed method [49]. Triplicate 0.6 mm diameter cylinders        1	 Mizushima N. Autophagy: process function. Genes Dev <br />       punched representative areas individual donor tissue           2007; 21:2861-2873. <br />       block, embedded recipient paraffin block defined   2	 Levine B, Klionsky DJ. Development self-digestion: mo- <br />       position, using tissue arraying instrument (Beecher Instruments,        lecular mechanisms biological functions autophagy. <br />       Silver Spring, MD, USA).                                                  Dev Cell 2004; 6:463-477. <br />           TMA blocks cut 5-&#206;&#188;m sections processed          3	 Klionsky DJ, Emr SD. Autophagy regulated pathway       IHC. TMA slides incubated anti-XBP-1 (1:100),               cellular degradation. Science 2000; 290:1717-1721. <br />       anti-FoxO1 (1:50) anti-p62 (1:200), respectively, stored      4	 Mizushima N, Komatsu M. Autophagy: renovation cells       overnight 4 &#194;&#176;C. Immunostaining performed using Envi-           tissues. Cell 2011; 147:728-741. <br />       sion diaminobenzidine (Dako, Glostrup, Denmark).          5	 Kroemer G, Marino G, Levine B. Autophagy integrat- <br />       negative control obtained replacing primary anti-            ed stress response. Mol Cell 2010; 40:280-293. <br />       body normal rabbit IgG. case non-informative        6	 Massey AC, Follenzi  Kiffin R, Zhang C, Cuervo  Early <br />       TMA samples  e., samples &lt; 500 tumor cells case            cellular changes blockage chaperone-mediated au- <br />       lost samples), IHC staining performed using tissue           tophagy. Autophagy 2008; 4:442-456. <br />       slides.                                                               7	 Yu L, McPhee CK, Zheng L, et al. Termination autophagy <br />           Positive expressions XBP-1, FoxO1 p62 CRC tissues           reformation lysosomes regulated mTOR. Nature <br />       primarily cytoplasmic patterns cells showing nu-          2010; 465:942-946. <br />       clear staining (Figure 7C 7D).  study, assessed   8	 Ravikumar B, Vacher C, Berger  et al. Inhibition mTOR <br />       cytoplasmic expressions 3 proteins cohort 229        induces autophagy reduces toxicity polyglutamine ex- <br />       CRC tissues. semi-quantitative scoring criterion IHC             pansions fly mouse models Huntington disease. Nat <br />       XBP-1, FoxO1 p62 used, staining intensity           Genet 2004; 36:585-595. <br />       positive areas recorded. staining index (values 0-12),     9	 Liang XH, Jackson S, Seaman M, et al. Induction au- <br />       obtained intensity XBP-1, FoxO1 p62 positive stain-         tophagy inhibition tumorigenesis beclin 1. Nature <br />       ing (negative = 0, weak = 1, moderate = 2 strong = 3 scores)       1999; 402:672-676. <br />       proportion immunostaining positive cells (&lt; 25%    10	 Zhao Y, Yang J, Liao W, et al. Cytosolic FoxO1 essential       = 1, 25%-50% = 2, &gt; 50% &#226;&#8364;&#8220; &lt; 75% = 3, &#226;&#8240;&#165; 75% = 4 scores)               induction autophagy tumour suppressor activity. Nat <br />       calculated. mean staining index XBP-1, FoxO1 p62               Cell Biol 2010; 12:665-675. <br />       CRC cohort 5.2, 2.7 6.3.  categories high     11	 Zhao J, Brault JJ, Schild  et al. FoxO3 coordinately activates <br />       expressions XBP-1, FoxO1 p62 defined CRC cases             protein degradation autophagic/lysosomal protea- <br />       staining index 5.2, 2.7 6.3, respectively.          somal pathways atrophying muscle cells. Cell Metab 2007; <br />       independent pathologists (M-Y Cai D Xie) blinded           6:472-483. <br />       clinic pathologic information performed scorings. inter-      12	 Mammucari C, Milan G, Romanello V, et al. FoxO3 controls <br />       observer disagreements reviewed second time, followed           autophagy skeletal muscle vivo. Cell Metab 2007; 6:458- <br />       conclusive judgment pathologists.                               471. <br />                                                                             13	 Greer EL, Brunet  FOXO transcription factors  <br />       Statistical analysis                                                      terface longevity tumor suppression. Oncogene <br />          Statistical analysis performed using SPSS statisti-         2005; 24:7410-7425. <br />       cal software package (standard version 13.0; SPSS , Chicago,      14	 Kenyon C, Chang J, Gensch E, Rudner  Tabtiang R. C. el- <br />        USA). correlations molecular features detected            egans mutant lives twice long wild type. Nature 1993; <br />       associations expressions XBP-          366:461-464. <br />       1, FoxO1 p62, CRC patients&#226;&#8364;&#8482; clinic pathological features      15	 Lee SS, Kennedy S, Tolonen AC, Ruvkun G. DAF-16 target <br />       assessed Chi-square test. univariate survival             genes control C. elegans life-span metabolism. Sci- <br />       analysis, survival curves obtained Kaplan-Meier             ence 2003; 300:644-647. <br />       method. Cox proportional hazards regression model used        16	 Lin K, Dorman JB, Rodan  Kenyon C. daf-16: HNF-3/ <br />  <br />  <br />                                                                                                            Cell Research | Vol 23 4 | April 2013 <br />                                                                                                                            Ying Zhao et al . npg <br />                                                                                                                                             507 <br />     forkhead family member function double life-          FoxO1 transcriptional regulator cells transformed     span Caenorhabditis elegans. Science 1997; 278:1319-               P3k Akt oncoproteins. Proc Natl Acad Sci USA 2004; <br />     1322.                                                                 101:13613-13617. <br /> 17	 Fu  Tindall DJ. FOXOs, cancer regulation apoptosis.      34	 Massey AC, Kaushik S, Sovak G, Kiffin R, Cuervo   <br />     Oncogene 2008; 27:2312-2319.                                          sequences selective blockage chaperone-mediated <br /> 18	 Zhou J, Liao W, Yang J, et al. FOXO3 induces FOXO1                 autophagy. Proc Natl Acad Sci USA 2006; 103:5805-5810. <br />     pendent autophagy activating AKT1 signaling pathway.       35	 Bejarano E, Cuervo  Chaperone-mediated autophagy. <br />     Autophagy 2012; 8:1712-1723.                                          Proc Thorac Soc 2010; 7:29-39. <br /> 19	 Juhasz G, Pusk&#195;&#161;s LG, Komonyi O, et al. Gene expression pro-       36	 Plas DR, Thompson CB. Akt activation promotes degradation <br />     filing identifies FKBP39 inhibitor autophagy larval       tuberin FOXO3a proteasome. J Biol Chem 2003; <br />     Drosophila fat body. Cell Death Differ 2007; 14:1181-1190.            278:12361-12366. <br /> 20	 Huang H, Regan KM, Wang F, et al. Skp2 inhibits FOXO1          37	 Yoshida H, Uemura  Mori K. pXBP1(U), negative regula- <br />     tumor suppression ubiquitin-mediated degradation.             tor unfolded protein response activator pXBP1(S), tar- <br />     Proc Natl Acad Sci USA 2005; 102:1649-1654.                           gets ATF6 ATF4 proteasome-mediated degradation. <br /> 21	 Kato S, Ding J, Pisck E, Jhala  Du K. COP1 functions          Cell Struct Funct 2009; 34:1-10. <br />     FoxO1 ubiquitin E3 ligase regulate FoxO1-mediated gene         38	 Yoshida H, Oku M, Suzuki M, Mori K. pXBP1(U) encoded <br />     expression. J Biol Chem 2008; 283:35464-35473.                        XBP1 pre-mRNA negatively regulates unfolded protein  <br /> 22	 Yang JY, Zong CS, Xia W, et al. ERK promotes tumorigenesis            sponse activator pXBP1(S) mammalian ER stress response. <br />     inhibiting FOXO3a MDM2-mediated degradation. Nat               J Cell Biol 2006; 172:565-575. <br />     Cell Biol 2008; 10:138-148.                                       39	 Lee J, Sun C, Zhou Y, et al. p38 MAPK-mediated regulation <br /> 23	 Fu W, Ma Q, Chen L, et al. MDM2 acts downstream p53             Xbp1s crucial glucose homeostasis. Nat Med 2011; <br />     E3 ligase promote FOXO ubiquitination degradation.          17:1251-1260. <br />     J Biol Chem 2009; 284:13987-14000.                                40	 Shintani T, Klionsky DJ. Autophagy health disease: 24	 Zhou Y, Lee J, Reno CM, et al. Regulation glucose homeo-           double-edged sword. Science 2004; 306:990-995. <br />     stasis XBP-1-FoxO1 interaction. Nat Med 2011;           41	 Baehrecke EH. Autophagy: dual roles life death? Nat <br />     17:356-365.                                                           Rev Mol Cell Biol 2005; 6:505-510. <br /> 25	 Shajahan  Riggins RB, Clarke R. role X-box bind-        42	 Levine B, Yuan J. Autophagy cell death: innocent  <br />     ing protein-1 tumorigenicity. Drug News Perspect 2009;             vict? J Clin Invest 2005; 115:2679-2688. <br />     22:241-246.                                                       43	 Edinger AL, Thompson CB. Death design: apoptosis, ne- <br /> 26	 Yamamoto K, Yoshida H <br /> </body></html>