<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Biology Cell <br /> Vol. 5, 899-905, August 1994 <br />  <br />  <br /> Mitochondrial Morphological Functional Defects Yeast Caused ymel Suppressed Mutation 26S Protease Subunit Homologue <br /> Corey L. Campbell, Noriko Tanaka, Karen H. White, Peter E. Thorsness <br /> Department Molecular Biology, University Wyoming, Laramie, Wyoming 82071-3944 <br /> Submitted 25, 1994; Accepted June 30, 1994 <br /> Monitoring Editor: Gottfried Schatz <br />  <br />  <br />          absence functional Ymelp, putative <span id='am-13' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-14' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> zinc-dependent protease localized <br />          mitochondria yeast, results abnormal mitochondrial function morphology. <br />          Yeast lacking Ymelp lose DNA mitochondria accelerated rate, fail grow          nonfermentable carbon sources 37&#194;&#176;C, severely deficient growth mitochondrial <br />          DNA suffers large deletions completely lost. place normal reticulated mi- <br />          tochondrial network, strains lacking Ymelp punctate mitochondria grossly <br />          swollen compartments. growth phenotypes morphological alterations evident          mutant yeast compensated mutation YNT1, essential gene yeast. <br />          sequence YNT1 gene product indicates number related <br />          regulatory subunits 26S protease. proteolytic activity necessary progression <br />          cell cycle implicated regulation transcription. Yntlp          distantly related Ymelp. <br />  <br />  <br /> INTRODUCTION                                                   phenotypes yeast: increased rate DNA escape <br />                                                                mitochondria subsequent migration budding yeast Saccharomyces cerevisiae, mito-           nucleus, inability utilize nonfermentable carbon <br /> chondrial morphology function vary according               sources ethanol glycerol 37&#194;&#176;C, slow growth carbon source cell-cycle stage (Stevens, 1977,          complete glucose media 14&#194;&#176;C, severely com- <br /> 1981). Using genetic analyses, number genes            promised growth mitochondrial DNA deleted <br /> volved dynamic processes isolated           lost entirely (Thorsness Fox, 1993; Thorsness et <br /> (McConnell et al., 1990; McConnell Yaffe, 1992;            al., 1993) (Figure 1). <br /> Thorsness Fox, 1993). gene YME1,                  characterize Ymelp, used suppressor encodes protein necessary maintaining              analysis isolate possible interacting gene products. integrity mitochondrial compartment                 Suppressing mutations bypassed requirement <br /> (Thorsness et al., 1993).                                      Ymelp isolated ymel-Al:: URA3 strain   nuclear-encoded YME1 gene encodes mito-                selecting revertants grew nonfermentable <br /> chondrially localized protein predicted size 82      carbon sources 37&#194;&#176;C. reverting mutation, <br /> kDa (Thorsness et al., 1993). According sequence            yntl-1, suppressed phenotypes associated analysis, Ymelp contains sequence elements suggestive          ymel-Al::URA3. addition, mitochondrial mor- ATPase zinc protease activities. gene            phological abnormalities ymel strain partially <br /> coding Ymelp originally isolated genetic screen       reversed mutation.  iso- identified mutations cause increased rate         lation characterization YNT1 yeast. YNT1 DNA escape mitochondria nucleus             identified virtue mutant allele yntl-1, reces- yeast (Thorsness Fox, 1993).                            sive bypass suppressor intrinsic cold-sensitive <br />    absence functional Ymelp,            growth phenotype nonfermentable carbon sources. original mutant allele, ymel-l deletion allele   Cloning sequencing revealed YNT1 identical <br /> ymel-A1 ::URA3, (Thorsness et al., 1993), results      previously sequenced gene, YTA2, exhibits se- <br />  <br /> &#194;&#169;) 1994 American Society Cell Biology                                                                     899 <br />  C.L. Campbell et al. <br />  <br /> quence homology genes involved cell-                   ethanol/glycerol 30&#194;&#176;C fixed according methods described <br /> cycle regulation.                                                        Byers Goetch (1990). <br />  <br /> MATERIALS METHODS                                                    Identification Cloning YNT1 <br />                                                                          Approximately 108 cells yeast strain PTY52 (ymel-M ::URA3) <br /> Strains                                                                  plated YPEG cultured 37&#194;&#176;C. Revertant cells capable Escherichia coli strain used preparation manipulation     respiratory growth 37&#194;&#176;C picked 1 wk incubation <br /> DNA DH5a [F-endAl hsdRl 7(rk - mk+) supE44 thi-1 X recA gyrA96       recultured restrictive conditions. Multiple rounds relAl argF-laczya) U169 +80 lac ZAM15]. genotypes S.       subculturing eventually produced independent revertants ca- <br /> cerevisiae strains used work listed Table 1. Standard     pable growth nonfermentable carbon sources restrictive <br /> genetic techniques used construct analyze various        temperature,  NTY1, characterized work. NTY1 <br /> yeast strains (Sherman et al., 1986).                                    backcrossed PTY33 (wild-type) PTY60 (ymel-Al :: URA3), <br />                                                                          diploids sporulated, tetrads dissected. colony arising <br />                                                                          spore scored nutritional markers, mating type, Media                                                                    ability grow nonfermentable carbon sources 14, 30, <br /> E. coli-containing plasmids grown LB (10 g bactotryptone,        37&#194;&#176;C. <br /> 10 g NaCl, 5 g yeast extract/l) plus 125 ,ug/ml ampicillin. Yeast      Yeast strains bearing yntl-l allele grow poorly nonfer- <br /> grown YPD (20 g glucose, 20 g bactopeptone, 10 g yeast extract,       mentable carbon sources 14&#194;&#176;C. phenotype used basis 40 mg tryptophan), YPEG (30 ml glycerol, 30 ml ethanol,        cloning YNT1. NTY4 (ura3-52 yntl-1) transformed 100 <br /> 20 g bactopeptone, 10 g yeast extract, 40 mg tryptophan),   jig DNA prepared multicopy, 2 ti-based S. cerevisiae genomic SD + nutrients (6.7 g yeast nitrogen base amino acids,     DNA bank  gift Jasper Rine). Approximately 24 000 transfor- <br /> 20 g glucose, appropriate nutrients). Nutrients included      mants screened complementation cold-sensitive  <br /> <span id='am-40' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> 40 mg/l, <span id='am-8' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> 40 mg/l, tryptophan 40 mg/l, lysine     spiratory deficient (Pet-cs) phenotype. transformants comple- 60 mg/l, leucine 100 mg/l. agar plates, bactoagar      mented Pet-cs phenotype. Plasmid DNA prepared added 20 g/l. Ethidium bromide containing medium SD + nu-          transformants used transform E. coli. Restriction mapping <br /> trients supplemented 25 Ag/ml ethidium bromide. liter        rescued plasmids showed plasmids sporulation media contained 10 g potassium acetate, 1 g yeast extract,   identical. insert DNA multicopy plasmids <br /> 0.5 g glucose, 20 g bactoagar. Bactoagar, bactopeptone, bacto-       (pNT18) subcloned CEN vector (pRS316) (Sikorski tryptone, yeast extract, yeast nitrogen base amino acids     Hieter 1989), ability low copy plasmid, pNT30, obtained Difco (Eietroit, MI). Ampicillin nutrients   rescue Pet-cs phenotype assayed. <br /> obtained Sigma (St. Louis, MO). <br />                                                                          Nucleic Acid Manipulations <br /> Growth Assays                                                            Restriction enzymes T4 DNA ligase purchased Bethesda escape DNA mitochondria assayed described             Research Labs (Gaithersburg, MD) New England Biolabs (Beverly, <br /> (Thorsness Fox 1993). Briefly, yeast containing copy TRPI       MA). Standard techniques generating recombinant DNAs integrated mitochondrial genome grown rich media         performing DNA blot hybridizations used (Maniatis et al., 1982). <br /> (YPD YPEG) agar plates replica plated minimal media       DNA sequence determined double-stranded templates using <br /> (SD + nutrients) lacking tryptophan. cells DNA        <span id='am-24' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-29' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> chain termination method (Sanger et al., 1977). <br /> taining TRP1 sequences escaped mitochondria migrated            Plasmid pNT30 subcloned, minimal complementing insert nucleus rise colonies virtue complementation    low-copy plasmid determined. pNT53 contains 2 kilobase <br /> trpl-A1 allele nucleus. Strains DNA escapes mito-        (kb) Stu Xba insert yeast genomic DNA pRS315, CEN plasmid <br /> chondria higher rate wild-type yeast increased      (Sikorski Hieter 1989), able rescue pet-cs phenotype papillation tryptophan prototrophy minimal media     yntl-1 mutant yeast strains. integrating plasmid containing plates incubating 72 h 30&#194;&#176;C.                                minimal complementing region constructed cloning ge- <br />   ability yeast strains grow absence mitochondrial   nomic fragment pNT53 pRS306, creating pNT54. pNT54 <br /> DNA assayed culturing cells minimal media agar plates      targeted integrate chromosomal locus corresponding containing 25 ug/ml ethidium bromide. Cells streaked single    insert digestion <span id='am-45' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-46' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span>Sph</span>  cuts unique site just 5&apos; <br /> colonies plates incubated 72 h 30&#194;&#176;C. single     YNT1 open reading frame. <br /> colony restreaked second time media. plates          yntl-A1::URA3 mutation constructed follows. 2-kb scored second 72-h incubation 30&#194;&#176;C.          Stu Xba fragment containing YNT1 pNT53 inserted single colonies arising ethidium bromide         Sma Xho sites pBluescript KS&apos; (Stratagene, La Jolla, CA). <br /> taining plates lack mitochondrial DNA.                                   construct digested HincII, removing 60%   cold-sensitive phenotype yeast strains scored      YNT1 coding region  30 codons YNT1 remain 5&apos; point <br /> cubation cells 140 10-14 d. temperature-sensitive phe-     insertion), 1.5-kb Sma fragment containing URA3  <br /> notype yeast strains scored incubation cells 37&#194;&#176;C     serted. plasmid, pNT55, containing yntl-A1l::URA3 allele 48 h.                                                                digested HindIII Sph  linear DNA used                                                                          transform yeast diploid PTY33 X PTY44 uracil prototrophy.                                                                          Ura+ transformant sporulated dissected. <br /> Microscopy confocal microscopy, strains cultured YPEG enriched <br /> galactose 30 37&#194;&#176;C harvested log phase. DiOC6 (3,3&apos;-         RESULTS <br /> dihexyloxacarbocyanine iodide) used 1 ,g/ml concentrations, mitochondrially specific concentration. Methods equipment     Identification Isolation YNT1, Bypass <br /> used Koning et al. (1993). Strains observed using       Suppressor ymel <br /> DAPSMI, 2-(4-dimethylaminostyryl)-N-methylpyridinium iodide, Nikon epifluorescence microscope (Garden City, NY) similar        YME1 encodes mitochondrial protein putative <br /> results. transmission electron microscopy strains cultured   ATPase protease activity (Figure 3A). Yeast cells <br />  <br /> 900                                                                                                              Molecular Biology Cell <br />                                                                                        . _. <br />                                                                                                         Suppression Mitochondrial Defects <br />  <br />  <br /> Table 1. Yeast strains <br />      Strain              Genotype                                                                              Source <br />      NTY1                MATa ura3-52 lys2 leu2-3,112 trpl-Ml ymel-A1::URA3 yntl-1 [p+, TRP1]                  study <br />      NTY4                MATa ura3-52 ade2-101 leu2-3,112 trpl-A1 yntl-1 [p&apos;, TRP1]                            study <br />      PTY33               MATa ura 3-52 ade2 leu2-3,112 trpl-Ml [p+, TRP1]                                      (Thorsness Fox, 1993) <br />      PTY44               MATa ura3-52 lys2 leu2-3,112 trpl-A1 [p+, TRP1]                                       (Thorsness Fox, 1993) <br />      PTY52               MATa ura3-52 lys2 leu2-3,112 trpl-Ml ymel-A1::URA3 [p+, TRP1]                         (Thorsness et al., 1993) <br />      PTY60               MATa ura3-52 ade2 leu2-3,112 trpl-Al ymel-A1l::URA3 [p+, TRPI]                        (Thorsness et al., 1993) <br />      PTY61               MATa ura3-52 ade2-101 leu2-3,112 trpl-A1 ymel-1 [p+, TRP1]                            (Thorsness Fox, 1993) <br />      PTY106              MATa ura 3-52 Iys2 leu2-3,112 trpl-Al ymel-1 yntl-l [p+, TRP1]                        study <br />  <br />  <br />  <br /> lacking Ymelp number phenotypes                     linked ymel-Al::URA3. crosses, Pet- <br /> dicate abnormal mitochondrial activity (Figures 1                 cs phenotype segregated 2cs:: 2wt. inability 2) (Thorsness et al., 1993). learn                  utilize nonfermentable carbon sources 14&#194;&#176;C cose- <br /> function protein, genetic approach taken                gregated suppressor locus (Figure 1A). identify proteins processes com-               genetic locus designated YNT1 mutant <br /> pensate mutations YME1. Initially, attempts               allele yntl-1. second-site revertants point mutation YME1, tightly linked (presumably intragenic) <br /> revertants   spontaneous bypass                  revertants ymel-Al::URA3 strain, PTY52,                         YPEG 370        YPD 14&#194;&#176;           YPEG 140 <br /> isolated culturing restrictive growth  <br /> dition, 37&#194;&#176;C ethanol/glycerol medium (Thorsness                                                                          wildt gpe Fox, 1993; Thorsness et al., 1993). isolate, des-                             r                 1         1 <br /> ignated NTY1 containing nuclearly encoded                                     _                                     ymel <br /> suppressor yntl-l  , suppressed                                    X         ]       L_S                 ymeI, yntI <br /> growth phenotypes associated ymel (Figure 1). Al- reverting mutation suppressed temper-                                                                        yntl <br /> ature-sensitive growth phenotype (Pet-ts), growth                                                                       ymel, yntl/pYNT1 wild-type condition <br /> (Figure 1A).  cold sensitivity enriched media <br /> containing glucose galactose completely reversed.              B                      S D -Trp              S <span id='am-39' about='oboInOwl:id' typeof='owl:Thing'>iD</span>+EtBr addition, suppressing mutation NTY1 delayed                                     wildt!ype                         Idtyjpe <br />                                                                                                                        lb <br /> onset mitochondrial DNA (mtDNA) escape (Figure <br /> 1B), resulting reduced papillations colonies Trp- Trp+. suppressor reversed growth                                ymeJ                                      yme1 <br /> deficiency ymel (Figure 1C) presence partially completely lost mitochondrial genome, induced growth media containing ethidium                                ymel, yntl                                      ymel, yntl <br /> bromide. <br />    backcrosses NTY1 parental strain op- <br /> posite mating type (PTY60, ymel-A1 ::URA3),                       Figure 1. yntl-1 suppresses functional defects mitochondria ymel <br /> suppression Pet-ts phenotype segregated 2ts::                  yeast strains.   Yeast indicated mutations inoculated <br /> 2wt tetrads dissected, indicating single nu-             equivalent concentrations YPEG YPD incubated clear locus responsible suppressor activity.              indicated temperatures (Thorsness Fox, 1993). plasmid bearing suppressor determined recessive,                  YNT1 (pYNT1) restored growth ymel, yntl yeast strain YPEG evidenced inability NTY1 X PTY60                     14&#194;&#176;C. (B) Escape DNA containing TRP1 gene mito- <br />                                                                       chondria subsequent migration nucleus allows comple- <br /> diploid grow nonfermentable carbon sources                   mentation nuclear trpl-A1 mutation. increased rate DNA <br /> 37&#194;&#176;C. yntl-l suppressed phenotypes strains                escape mitochondria ymel strain suppressed yntl- <br /> bearing ymel-1, point mutation allele YME1                       1. (C) Complete loss mitochondrial DNA induced culturing <br /> (Thorsness et al., 1993). Backcrossing NTY1                     yeast presence 25 gg/ml ethidium bromide (Thorsness et <br /> PTY33 (wild-type) sporulation allowed recov-                  al., 1993). severe growth deficiency ymel strains mi- <br />                                                                       tochondrial DNA suppressed yntl-1. Strains used figure <br /> ery Pet-ts spores (tetratype nonparental ditype                follows: wild-type PTY44, ymel PTY52, ymel yntl NTY1, yntl <br /> tetrads), indicated reverting mutation                  NTY4, ymel yntl/pYNT1 PTY106 containing pNT53. <br />  <br /> Vol. 5, August 1994                                                                                                                        901 <br />  C.L. Campbell et al. <br />  <br />   <br />  <br />  <br />  <br />  &apos;&apos;&lt; <br />  .,\,, <br />       \st.*, ., . _s <br />       .w.&apos; <br />  <br />  <br />       _s.- <br />                        ;.\; <br />  <br />                                w <br />  <br />                               5S <br />                                X <br />                               _tAK <br />                                                  G <br />  <br />  <br />  <br />  <br />                                                                        Figure 2. Morphological defects mitochondria                                                                        ymel mutant yeast strain suppressed yntl-l <br />                                                                        mutation. Transmission electron <br />                                                                        focal fluorescence images wildtype  C), ymel (D- <br />                                                                        F),     ymel <br />                                                                                                               micrographs <br />  <br />                                                                                           yntl-l (G  yeast strains revealed altered <br />                                                                        mitochondrial profiles. electron micrographs   <br />                                                                        D, G) yeast grown YPEG 30&#194;&#176;C. Mito- <br />                                                                        chondria indicated arrows. Bar, 1 ,um. Vacuoles <br />                                                                        containing osmium-dense material (D) occasionally <br />                                                                        detected wild-type mutant yeast.                                                                        confocal fluorescence images (B, C, E, F, H, , yeast <br />                                                                        grown YPEG 30&#194;&#176;C (B, E, H)                                                                        rich galactose media 37&#194;&#176;C (C, F, , mito- <br />                                                                                                                                <br />  <br />  <br />  <br />  <br />                                                                        chondria stained DiOC6 (3,3&apos;-dihexyloxacar- <br />                                                                        bocyanine iodide). Confocal images constructed                                                                        optical sections (0.5 ,um  series <br />                                                                        scan. Bar, 5  Strains used figure follows: <br />                                                                        wild-type PTY44, ymel PTY52, ymel yntl-l NTY1. <br />  <br />  <br /> Morphology Mitochondrial Compartments                     observed 12% (chi square test, p &lt; 0.001) Wild-Type, ymel, ymel/yntl-l Yeast                   PTY52 cells observed, compared &lt;3% (p &lt; 0.001) Analysis mitochondrial compartment super-structure       revertant strain NTY1. yntl-1, present using confocal fluorescence electron microscopy         wild-type YME1 background, apparent affect revealed significant morphological alterations yeast     mitochondrial superstructure evidenced epi- <br /> lacking Ymelp (Figure 2). nonrepressing           fluorescence microscopy. <br /> ditions log phase, wild-type yeast complex <br /> reticulated network mitochondrial compartments           Cloning Sequencing YNT1 <br /> (Figure 2, C). contrast, yeast bearing ymel-Al::       cold-sensitive phenotype yntl-l strains URA3 abbreviated globular mitochondria          used isolate wild-type gene. YNT1 origi- <br /> subpopulation cells containing grossly swollen       nally cloned 2-,u (multicopy) plasmid bank compartments (Figure 2, D-F). Altered mitochondrial         complementing cold-sensitive growth nonfer- <br /> morphology necessarily consequence muta-        mentable carbon sources associated yntl tions inhibiting oxidative phosphorylation,         plasmid-dependent fashion.  gene strains containing unrelated mutations affecting    complemented phenotype low-copy process retain normal mitochondrial profiles.     CEN plasmid (Figure 1A). confirm isolated abnormal morphology merely characteristic       genomic DNA corresponded yntl-l locus, growth conditions requiring respiratory            minimal complementing genomic fragment cloned <br /> growth, punctate mitochondrial forms ev-        integrating URA3 plasmid. plasmid ident fermentable medium galactose        targeted integrate yeast genome digesting <br /> (Figure 2, C, F, . morphological abnormalities    restriction endonuclease Sph  cuts ymel -Al:: URA3 strain partially reversed   just 5&apos; YNT1 open reading frame (described  <br /> yntl-l suppressor gene (Figure 2, G .          low). linear DNA transformed PTY61, vertants display shorter branching mitochondrial seg-       ymel strain. resulting transformant ments wild-type yeast.                              mated PTY106 (ymel-1, yntl-l). resulting diploid <br />    distended organelles caused ymel             sporulated, tetrads dissected. inte- <br /> counted using DAPSMI staining epifluorescence           grated URA3 marker segregated opposition microscopy. Grossly swollen mitochondrial profiles          suppressor activity 19 20 tetrads examined, <br />  <br />  <br /> 902                                                                                                   Molecular Biology Cell <br />                                                                                                                 Suppression Mitochondrial Defects <br />  <br />                                                                               Ymelp   FrVaSKILVSP   TVTTN       LPQIGAS LI                 LRF(CFYR}SE KNSGNMPPKX 4 pDS1              75 <br />                                        Yntlp     -------     ----   2I3 ----VEKA -                 K- ----j- --ASLL----                      41 <br />  <br />                                                                        VAV8                        TS <br />                                                                                                            ! i1 <br />                                        rmelp   SIDNSQPPPP            N <br />                                                                           _ <br /> Figure 3. Yntlp belongs ex- <br />                                        Yntlp   NVN --------                   KLlErKCE!       L*--- -&quot;~   IK- -----                          93 <br />  <br /> tended putative ATPase family.   !melp ECKLr L QRIGRNSEAD IiQNLLTAS SAGr4P4                        SNQSG!HlNGNCPiGS1lGSRKNPLHV VVS4 225 <br /> Alignment Ymelp Yntlp Yntlp LVIGQFIJI DQ--NTG                           ----STT GXSIVR- 1$LDRELLK -- S!V<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;RNSNALVDI LPP1              154 <br /> <span id='am-32' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-33' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-35' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> sequences. Identical <br /> amino acids boxed, nucleotide Ymelp TFTVVSRWVK WLLVFGILTY SFSEGFIIT NNTTLLKSSE                KS      NVKFDlG   CD    9 IVDFiq 300 <br /> binding motifs (Chin et al., 1988) Yntlp DSSI------ -----SVMGE NEKPDV- --------AD                 -         --       -   ---- <br />                                                                                                                                             192 <br /> underlined, putative neutral Ymelp X <br /> zinc protease motif marked *.                  DPTUK <br />                                                        SAKA <br />                                                     TfDTPGG TAIt <br />                                                                                                                                             375 Yntlp Ymelp sequences rntlp <br />                                              K                                                           SEP            K                   267 aligned aid !melp jlIiP YAKQTLNQIS--GII4+ Tp                                                                IVVNV 445 program GeneWorks 2.2 Yntlp D1          D                                    3fR <br />                                                                                         _TINVKVF                           R          I3PP 341 IntelliGenetics (Mountain <br /> View, CA); zinc protease motif Ymelp         MMNI,DT              T           TII ARGTP4SiAI ILVN             VS V D DKILM 520 identified Blocks sequence Yntlp                      GTIA&quot; Z1F DSL IIRNDGjAiNQd G L                      !V         ATQVK 416 <br /> analysis program (Henikoff                                        Ymelp GABRKTMVLT DAARKATAFH EAGNAIKAK! TNGATPL!KA TILPRGRALG ITFQLPENDK VDITKRBCQA RLDVC 595 <br /> Henikoff, 1991). (B) Yntlp belongs Yntlp     TDNTVDKFDF YK subfamily ATPase proteins regulatory subunits Ymelp MGGRIA1ELI YGKDNTTSGC GSDLQSATGT ARANMTQYGN SDDVGPVNLS ENWKSWSNKI RDIADNEVIE LLKDS 674) 26S protease. degree sim- Ymelp EERARRLLTK KNVELHRLAQ GLIEYETLDA HEIEQVCKGK KLDKLKTSTN TVVEGPDSDE RKDIGDDKPK IPTML 745 <br /> ilarity displayed using grayscale Ymelp NA                                                                                             747 <br /> (Fr6lich, 1994). Percentage amino <br /> acid identity Yntlp calculated subsets seven amino acids, subset displaced residue, producing B                 0% 25% 50% 75% 100% identity Yntl p <br /> overlapping values percentage <br /> identity sequence. Values <br />                                            B~~                        . <br />  <br />  <br />  <br />  <br /> &lt;20% identity displayed                    0                         100                 200r-1         n   300                   400 aa <br /> white, values equal 100% Yntl p <br /> black, thresholds,               I . 51 <br /> gray values equivalent pro- S4 <br /> portion    identity used.   Mts2p       P. FIT,.1 , ATPase sequence elements  Cim5p             U.- ID&apos;       1. .     -, s ==..                                   H-M mm.. M. <br /> dicated brackets.                   Sug1p             Ai -v&apos;m              F K, NM                                    . f W. 71, <br />  <br />  confirmed isolated DNA did fact contain                            pletion mitosis Schizosaccharomyces pombe (Gordon YNT1 locus. Presumably, tetrad showed                           et al., 1993); Cim5p Suglp, 26S protease sub- <br /> 3ts:: lwt undergone gene conversion.                                         units necessary degradation ubiquitinated sub- <br />    Sequence analysis YNT1 revealed identical                       strates anaphase chromosome separation S. previously identified yeast gene unassigned                              cerevisiae (Ghislain et al., 1993). <br /> function, YTA2, GenBank accession number X73570 <br /> (Schnall et al., 1994). YNT1 nucleic acid sequence                           YNT1 Essential Gene assigned GenBank accession number U06229. <br />    Yntlp homologous extended family pro-                             chromosomal copy YNT1 deleted teins, including Ymelp (Figure 3A), related                          diploid strain PTY33 X PTY44. deleted locus 200-amino acid region containing consensus ATPase                            marked URA3 DNA  MATERIALS element. Proteins family implicated                            METHODS) verified DNA-blot hybridization. divergent processes, ranging vesicle fusion (Diaz                        diploid sporulated dissected, et al., 1989) transcription (Ohana et al., 1993).                       viable spores recovered 20 <br /> mains unclear sequence homologies                                  tetrads dissected. viable spores Ura+, family members correspond functional simi-                              indicating functional copy YNT1 necessary <br /> larities.                                                                        cell viability. <br />    Yntlp high degree sequence homology genes identified regulatory                            DISCUSSION <br /> subunits 26S protease (Figure 3B): subunit 4 26S protease humans (Dubiel et al., 1992);                              demonstrated Ymelp necessary Mts2p, 26S protease subunit necessary com-                             proper mitochondrial function structure yeast. <br />  <br /> Vol. 5, August 1994                                                                                                                            903 <br />  C.L. Campbell et al. <br />  <br /> Ymelp sequence elements suggest possible ATP                1991). role ubiquitination proteins organelle <br /> pendent neutral zinc protease activity (Thorsness et al.,      formation proposed based obser- <br />  1993) (Figure 3). Ymelp mitochondrial protein            vation PAS2 S. cerevisiae, gene necessary (Thorsness, 1993, #595). recent work lab-          peroxisome biogenesis proliferation, encodes oratory demonstrated Ymelp tightly asso-           member ubiquitin-conjugating protein family <br /> ciated inner mitochondrial membrane (Weber            (Wiebel Kunau, 1992). 26S protease known Thorsness, unpublished data). closely related            act ubiquinated nonubiquinated sub- <br /> protein bacteria containing sequence             strates (Hershko Ciechanover, 1992; Murakami et <br /> elements, FtsH protein, identified screen         al., 1992). Yntlp, concert 26S protease com- cell division mutants E. coli (Begg et al., 1992;       plex, acts subset mitochondrial outer membrane <br /> Tomoyasu et al., 1993). homology, considered         proteins proteins destined interior mito- mitochondrial morphology abnormalities              chondria, yntl-l mutation alters sub- <br /> ficient mitochondrial function ymel-Al ::URA3             strate specificity complex include substrates <br /> strain, suggests Ymelp plays role rearranging          normally acted Ymelp.  pleiotropic <br /> maintaining proper mitochondrial superstructure.           growth phenotypes associated ymel merely Ymelp functions protease, phenotypic           demonstrate results lack maturation mul- <br /> sequences loss Ymelp activity result         tiple substrates. mutated Yntlp partially com- general reduction mitochondrial protein turn-        pensate range ymel phenotypes tar-  punctate distended mitochondrial com-            geting substrates lowered efficiency compared partments ymel mutant strain result directly          Ymelp. Alternatively, mutant Yntlp excess abnormal mitochondrial proteins. Al-         recognize subset Ymelp substrates. <br /> ternatively, Ymelp maturase activity             work provided intriguing genetic mi- <br /> subset mitochondrial proteins. unprocessed             croscopic evidence possible association proteins lead poorly assembled protein com-             26S protease mitochondrial function mor- <br /> plexes, resulting structurally functionally im-         phology. Localization Yntlp yeast cell paired mitochondrial compartments.                             required elucidation function. <br />    altered form Yntlp partially compensates absence Ymelp. yntl-l mediate suppression ymel-Al::URA3 different ways.            ACKNOWLEDGMENTS  Yntlp altered way         like thank Tom Fox, Teru Ogura, Michael Reith, Eric <br /> gain Ymelp-like function. Typically gain func-         Weber, Mary Thorsness helpful discussions. thank <br /> tion mutations dominant;  yntl-l displays          Robin Wright assistance microscopy. work sup- recessive phenotype. second model, Ymelp            ported United States Public Health Service grant (GM-47390) <br />                                                                Junior Faculty Research Award American Cancer  <br /> Yntlp function competing pathwa <br /> </body></html>