<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>  <br /> Chlamydia pneumoniae CdsL regulates CdsN ATPase activity, disruption <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> mimetic prevents bacterial <br /> invasion <br />  <br />  <br />  <br />                                   Chlamydiae obligate intracellular pathogens likely require type III secretion (T3S)  <br />                                             invade cells replicate intracellularly cytoplasmic vacuole called inclusion body. <br />  <br />                                             Chlamydia pneumoniae possess aYscL ortholog, CdsL, shown interact  <br />    T3S ATPase (CdsN). report demonstrate CdsL regulates CdsN enzymatic <br />                  activity dose-dependent manner. Using Pepscan epitope mapping identified separate <br />                          binding domains CdsL binds viz. CdsN221&#226;&#8364;&#8220;229 CdsN265&#226;&#8364;&#8220;270. confirmed binding <br />                           domains using pull assay showed GST&#226;&#8364;&#8220;CdsN221&#226;&#8364;&#8220;270, encompasses  <br />                                             peptides, purified CdsL.  used orthology modeling based crystal <br />  <br />             structure T3S ATPase ortholog Escherichia coli, EscN, map binding domains <br />          predicted 3D structure CdsN. CdsL binding domains mapped catalytic <br />                           domain ATPase, central channel ATPase hexamer outer <br />  <br />                                             face. peptide mimetics used disrupt essential protein interactions                                             chlamydial T3S inhibit T3S-mediated invasion HeLa cells, hypothesized                                             CdsL&#226;&#8364;&#8220;CdsN binding essential regulating T3S CdsN peptide mimetic                                             used potentially block T3S chlamydial invasion. Treatment elementary body                                             CdsN peptide mimetic inhibited C. pneumoniae invasion HeLa cells dose-dependent <br />                                             fashion report represents use Pepscan technology identify binding domains                                             specific T3S proteins viz. CdsL ATPase, CdsN, demonstrates peptide mimetics <br />                                             used anti-virulence factors block bacterial invasion. <br />                                             Keywords: type III secretion, peptide mimetic, Chlamydia, ATPase, Pepscan <br />  <br /> IntroductIon                                                                         close association inclusion membrane, suggesting need <br /> Chlamydia pneumoniae obligate, intracellular Gram-negative                     active T3S RBs. interaction inclusion membrane <br /> bacteria associated community acquired pneu-                      allows RB communicate host cell T3S, allowing <br /> monia (Clifton et al., 2004), atherosclerosis (Grayston, 2000),                      Chlamydia commandeer host cell pathways obtain lipids, <br /> arthritis (Ardeniz et al., 2005), Alzheimer&#226;&#8364;&#8482;s disease (Balin                     <span id='am-74' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-75' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cholesterol</span>, nutrients crucial growth repli- <br /> et al., 2008). members genus Chlamydia share                        cation, prevent phagosome endosome fusion (Wylie <br /> unique, biphasic life-cycle initiated attachment                   et al., 1997; Hoare et al., 2008; Scidmore Hackstadt, 2008). <br /> metabolically quiescent elementary body (EB) host cell.                   Following inclusion body growth accommodate replicating RBs, <br /> association EB host cell membrane poorly                      unknown signal (possibly quorum sensing) triggers detach- <br /> understood, glycosaminoglycans involved (Chen                         ment RBs membrane subsequent assortment <br /> Stephens, 1997). attached host cell, type III secretion                  infectious EBs. EBs exit host cell cell <br /> (T3S) utilized inject translocated actin recruitment pro-                  lysis extrusion, packaged release mechanism tein (TARP) facilitate bacterial internalization plasma-                   leaves host cell intact (Hybiske Stephens, 2007). <br /> membrane derived vacuole, termed inclusion (Clifton et al.,                           Type III secretion virulence mechanism commonly used <br /> 2004; Lane et al., 2008). MEK&#226;&#8364;&#8220;ERK PI 3-kinase pathways                       Gram-negative bacteria directly translocate effector proteins host cell involved bacterial uptake, possibly             bacterial cytoplasm host cell cytoplasm sin- <br /> targets T3S effectors translocated cytoplasmic                   gle step, use syringe-like apparatus termed membrane EB contact binding host cells (Coombes                          injectisome (Galan Collmer, 1999; Ghosh, 2004; Galan Mahony, 2002; Carabeo et al., 2004; Subtil et al., 2004).                   Wolf-Watz, 2006). injectisome constructed 20&#226;&#8364;&#8220;25 proteins <br /> inside host cell remainder life-cycle occurs inside                   spanning inner membrane, periplasm, outer membrane, inclusion body EBs differentiate metabolically                    extending extracellular milieu allow host cell sensing <br /> active, non-infectious reticulate bodies (RB). RB remains                     contact. host cell contact, T3S injectisome apparatus <br />  <br />  <br />  <br /> injects translocator proteins host cell membrane        CdsN (Bernard et al., 2004; Teeling et al., 2006; Timmerman et al., <br /> form translocon, molecular pore secreted pro-       2007). CdsN possesses distinct binding domains <br /> teins enter host cell (Goure et al., 2004). Despite fact    CdsL, potentially mediate tethering function Chlamydia possesses repertoire T3S genes scattered       mediate regulation function. genome 10 distinct operons, systematic     CdsN peptide acts peptide mimetic preventing bacterial <br /> study injectisome undertaken chlamydial spe-    invasion. report identify binding domains cies genetically intractable (Hefty Stephens, 2007). Recent     T3S ATPase regulating protein use peptide mimetic reports identified T3S structural effector proteins,     novel antimicrobial agent potentially target T3S. <br /> providing understanding Chlamydia T3SS. labo- <br /> ratory shown CdsD, unique protein orthologs YscD          MaterIals Methods contains fork-head associated domains, interacts      expressIon plasMIds <br /> predicted C. pneumoniae ATPase tethering protein, CdsL,         Chlamydia pneumoniae CWL029 (VR1310:ATCC; GenBank acces- CdsQ, putative multi-cargo transport protein (Johnson et al.,      sion  AE001363) strain used isolate genomic DNA <br /> 2008; Spaeth et al., 2009). extended findings     cloning protein expression. length cdsL cdsN CdsN binds complex, interacts independently       amplified CWL029 using AttB-containing primers (Gateway; <br /> CdsD, CdsL, CdsQ (Stone et al., 2008). Betts et al. (2008)     Invitrogen). amplified products cloned pDONR201 recently identified C. trachomatis CT666 gene encoding         (Gateway; Invitrogen) generate pENT vectors. pENT vectors <br /> CdsF, needle filament protein injectisome. role     used LR reactions (Gateway; Invitrogen) produce effector proteins C. pneumoniae recently discov-       pEX vectors containing genes  used pEX17 <br /> ered. Cpn0585 inclusion membrane interacts          (N-terminal tag) pEX15 (N-terminal GST-tag) vectors Rab 1, 10, 11, possibly recruiting ATPases        protein expression. constructs confirmed sequencing inclusion membrane (Cortes et al., 2007). Cpn0827 second           Molecular Biology Facility McMaster University. <br /> effector protein detectable inclusion membrane 20 hpi (Hermann et al., 2006). Chlamydia protein associating        proteIn expressIon death domains (CADD) conserved effector known              constructs expressed E. coli Rosetta pLysS. Expression <br /> interact TNF family receptors (Stenner-Liewen et al., 2002).       plasmids used transform E. coli Rosetta pLysS Recent biophysical evidence presented Markham          plated LB plates containing 100 &#194;&#181;g/mL ampicillin. LB broth <br /> et al. (2009) suggesting C. trachomatis protein CT584    (750 mL), containing antibiotics, inoculated 12 mL <br /> function needle tip protein, imperative host cell   overnight culture grown 37&#194;&#176;C reached sensing translocator insertion host cell membrane.         optical density (OD)600 approximately 0.8. Cultures     Type III secretion ATPases believed assembled        cooled ice 20&#194;&#176;C induced 0.2 mM isopropyl <br /> hexameric ring basal body injectisome play role     &#206;&#178;-d-thiogalactopyranoside (IPTG). Cultures incubated delivery effector proteins injectisome (Zarivach      23&#194;&#176;C 2 h bacteria harvested centrifugation et al., 2007). T3S ATPases partially characterized,   6500&#195;&#8212;g 10 min Sorvall RC-5B centrifuge washed including EscN Escherichia coli (Andrade et al., 2007), YscN       ice-cold phosphate-buffered saline (PBS). Bacteria containing  Yersinia (Blaylock et al., 2006), InvC Salmonella         tagged protein resuspended Binding Buffer (50 mM <br /> (Akeda Galan, 2004), ability T3S ATPase      potassium phosphate pH 7.2, 150 mM KCl, 1 mM MgCl2) coordinate numerous protein interactions explored.         used GST pull-downs, Nickel buffer (20 mM Tris pH 7.0, ATPases shown significant sequence              0.02% &#206;&#178;-mercaptoethanol, 400 mM KCl, 1% Triton X-100) orthology &#206;&#178; subunit F0F1 ATPase, hydrolyze         used ATPase activity assays, bacteria containing GST- <br /> <span id='am-41' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-42' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span>. ATPases important providing energy      tagged protein resuspended PBS stored &#226;&#710;&#8217;20&#194;&#176;C protein transport, believed play role unfolding    use. effector proteins translocation, accom- <br /> plished releasing chaperone cognate effector protein    purIfIcatIon recoMbInant proteIns <br /> (Akeda Galan, 2005). Yersinia, ATPase tethering protein      Escherichia coli pellets containing expressed  GST-tagged <br /> YscL localizes ATPase inner membrane, potentially         proteins thawed ice sonicated using Fischer Scientific <br /> performing function similar gamma-stalk         Sonic Dismembrator Model 100, followed centrifugation F0F1 ATPase (Pallen et al., 2006). YscL shown play     20,000&#195;&#8212;g 40 min remove insoluble material. Supernatants role regulation ATPase activity regulating enzy-    containing tagged protein use GST pull assays matic activity (Blaylock et al., 2006). quaternary structure     stored 4&#194;&#176;C. GST-tagged protein use GST pull assays <br /> ATPase, viz. hexamer, associated enhanced enzy-       bound 300 &#194;&#181;L <span id='am-17' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-18' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span>glutathione</span> beads overnight 4&#194;&#176;C, matic activity likely reflects native conformation      blocked overnight Tris Buffered Saline 0.1% Tween-20 protein.                                                           4% BSA stored 4&#194;&#176;C use.  GST-tagged protein <br />     report CdsL regulates enzymatic       supernatants use ATPase activity assays filtered activity CdsN. utilized Pepscan epitope mapping deter-         0.45 &#194;&#181;m acrodisc filters (Pall Corporation) purified CdsL binds CdsN, used orthology modeling           1-mL GSTrap FF column (GE Healthcare) 1-mL HisTrap <br /> approach map domain predicted 3D structure          HP column (GE Healthcare). GST-tagged protein, columns <br />  <br />  <br /> Frontiers Microbiology | Cellular Infection Microbiology                                           February 2011 | Volume 2 | Article 21 | 2 <br />  Stone et al.                                                                                       CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  washed PBS + 0.1% Tween flow             order reconstruct conformational epitopes, using Chemically OD280 0.005. GST-tagged protein eluted          Linked Peptides Scaffolds (CLIPS) technology (Timmerman beads using 1.5 &#194;&#181;g/&#194;&#181;L <span id='am-29' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-30' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span>reduced glutathione</span> (Sigma)            et al., 2007). example, single looped peptides synthesized <br /> dialyzed activity buffer (50 mM Tris&#226;&#8364;&#8220;HCl pH 7.0, 5 mM            containing dicysteine, cyclized treating alpha, <br /> MgCl2, 10 mM KCl). Purity confirmed using SDS&#226;&#8364;&#8220;PAGE               &#206;&#177;&#226;&#8364;&#178;-dibromoxylene, size loop varied introduc- <br /> Coomassie blue staining. tagged proteins, columns           ing cysteine residues variable spacing. cysteines washed increasing imidazole concentrations eluted          newly introduced cysteines present, replaced 300 mM imidazole dialyzed activity buffer. Purity          alanine. chains multiple cysteines peptides <br /> confirmed using SDS&#226;&#8364;&#8220;PAGE Coomassie blue staining.                    coupled CLIPS templates reacting credit-card  <br />                                                                          mat polypropylene Pepscan cards (455 peptide formats/card) <br /> cdsn enzyMatIc actIvIty regulatIon experIMents                      0.5-mM solution CLIPS template 1,3-bis (bro- <br /> ATP hydrolysis GST&#226;&#8364;&#8220;CdsN purified GSTrap columns              momethyl) benzene ammonium bicarbonate (20 mM, pH 7.9)/ <br /> measured using malachite green assay (R<span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;D Systems)               acetonitrile [1:1(v/v)]. cards gently shaken solution <br /> measures released inorganic phosphate ATP. experi-          30&#226;&#8364;&#8220;60 min completely covered solution. Finally, ments, specific activity determined using equation      cards washed extensively excess <span id='am-52' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span> sonicated standard line generated using phosphate standard (R&amp;D Systems).          disrupt-buffer containing 1% SDS/0.1% &#206;&#178;-mercaptoethanol PBS <br /> Reaction mixtures contained 150 ng GST&#226;&#8364;&#8220;CdsN, 40 &#194;&#181;M ATP,               (pH 7.2) 70&#194;&#176;C 30 min, followed sonication H2O 50 mM Tris&#226;&#8364;&#8220;HCl pH 7.0, 5 mM MgCl2, 10 mM KCl. reac-              45 min. binding antibody labeled protein/peptide <br /> tion mixture (1 mL) incubated 37&#194;&#176;C 1 h 50 &#194;&#181;L           peptide tested Pepscan-based ELISA. 455 mixture taken inorganic phosphate determination           credit-card format polypropylene cards containing covalently <br /> various time points. reaction stopped addition         linked peptides incubated peptide solution example <br /> 10 &#194;&#181;L Malachite Green Reagent followed 10 &#194;&#181;L Malachite        consisting 1 &#194;&#181;g/mL diluted blocking solution, example 4% <br /> Green Reagent B incubated room temperature 1 min              horse serum, 5% ovalbumin (w/v) PBS/1% Tween. washing, OD610 reading taken, according manufacturer&#226;&#8364;&#8482;s       peptides incubated monoclonal mouse anti  <br /> instructions. negative control, purified CdsN digested       tag antibody (1/1000, Novagen, 70796-3) subsequently 10 min 37&#194;&#176;C using Proteinase K (Invitrogen).  nega-     washing rabbit-anti-mouse antibody peroxidase conjugate <br /> tive control, GST-tagged protein (CopN) lacks ATPase        (1/1000, Southern Biotech, 6175-05) 1 h 25&#194;&#176;C. washing, <br /> activity purified manner tested activity.        peroxidase substrate 2,2&#226;&#8364;&#178;-azino-di-3-ethylbenzthiazoline sul- <br /> ATPase activity expressed nmol phosphate released min&#226;&#710;&#8217;1,          fonate (ABTS) 2 &#194;&#181;L 3% <span id='am-69' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-70' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>H2O2</span> added. 1 h, color experiments performed triplicate. CdsL           development measured. color development quantified purified Ni-NTA agarose beads tested contami-           charge coupled device (CCD)-camera image processing <br /> nating ATPase activity using Malachite Green Assay.       raw-data optical values obtained CCD-camera. <br /> added varying amounts reaction mixture CdsN             values range 0 3000, log scale similar 1&#226;&#8364;&#8220;3 <br /> prior addition ATP incubated 20 min ice.            standard 96 plate ELISA-reader. CCD-camera <br /> activity assay performed described                     makes picture card peroxidase coloring                                                                          picture peroxidase coloring. pictures <br /> gst pull assays                                                     subtracted results data examine interaction CdsN221&#226;&#8364;&#8220;270 CdsL, GST pull-           called raw-data. copied Peplab&#226;&#8222;&#162; database. assays performed described previously Johnson            values copied excel file labeled raw-data file. et al. (2008), following modifications. Briefly, <span id='am-64' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span>glutathione</span>&#226;&#8364;&#8220;   follow manipulation allowed. contains agarose beads (30 &#194;&#181;L) bound 50 ng GST-tagged CdsN221&#226;&#8364;&#8220;270           air-bubble resulting false-positive value, cards manu- <br /> protein used assay. beads incubated overnight        ally inspected, values caused air-bubble scored 4&#194;&#176;C E. coli lysate expressing tagged proteins.       0. 3D structure prediction performed using 3D-JIGSAW beads collected centrifugation washed 0.1%          (http://www.bmm.icnet.uk/&#226;&#710;&#188;3djigsaw/). Modeling  <br /> Triton X-100 increasing concentrations NaCl elimi-             formed using PYMOL software. <br /> nate spurious protein interactions. presence GST- tagged proteins confirmed Coomassie blue            peptIde desIgn entry ebs <br /> Western blot. proteins eluted glutathione beads        Based Pepscan epitope mapping CdsN, domains electrophoresed 11% SDS&#226;&#8364;&#8220;PAGE gel probed           mediate interaction CdsN CdsL. tagged protein. GST bound glutathione beads         smaller domains (residues 265&#226;&#8364;&#8220;279), corresponding <br /> used negative control pull                             TRFARA, chosen create peptide mimetic compound. <br />                                                                          synthesized TRFARA peptide N-terminal mem- <br /> peptIde lIbrary synthesIs, screenIng assays, 3d ModelIng             brane transport sequence (MTS), YGRKKRRQRRR (Efthymiadis linear CLIPS peptides synthesized based            et al., 1998). extended CdsN sequence flanking <span id='am-57' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-58' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> sequence target protein using standard Fmoc-           TRFARA peptide ensure entire binding domain chemistry protected using trifluoric acid scavengers.        included, flanked peptide cysteine residues assist constrained peptides synthesized <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> scaffolds        disulfide bonding required binding. <br />  <br />  <br /> www.frontiersin.org                                                                                         February 2011 | Volume 2 | Article 21 | 3 <br />  Stone et al.                                                                                    CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  final peptide utilized 28-amino acid sequence              using cell monolayers glass cover slips shell vials, fixed <br /> YGRKKRRQRRRCVVLMMDSVTRFARALC used                       methanol, stained using Pathfinder kit, uses <br /> linear form inhibition experiments. Peptides synthesized      FITC-conjugated major outer membrane protein (MOMP)- Peptide 2.0.                                                        monoclonal antibody specific Chlamydia, manu- <br />                                                                        facturer&#226;&#8364;&#8482;s instructions. <br /> gst&#226;&#8364;&#8220;Mts uptake ebs <br /> Chlamydial EBs purified using discontinuous gastrografin        results <br /> gradient, resuspended PBS. GST GST&#226;&#8364;&#8220;MTS            regulatIon cdsn cdsl <br /> incubated varying concentrations (0, 100, 500 &#194;&#181;m) vary-      Type III secretion ATPases known enzymatically  <br /> ing times (5, 30, 60 min) purified EBs 4&#194;&#176;C. EBs     regulated YscL orthologs (Blaylock et al., 2006). C. pneumoniae pelleted washed times PBS, followed        encodes YscL ortholog, CdsL (Cpn0826), led explore <br /> trypsinization 30 min ensure protein bound       CdsL regulates CdsN C. pneumoniae. outside EB. EBs boiled loading dye         previously confirmed CdsN interacted CdsL using GST <br /> probed presence GST using Western blot.                     pull assay, showing CdsL purified CdsN                                                                        500 mM NaCl (Stone et al., 2008). showed previously peptIde cytotoxIcIty                                                   CdsN-GST fusion protein possessed enzymatic activity consist- cytotoxicity performed described Johnson et al.          ent T3S ATPases (Stone et al., 2008). (2009). effect CdsN TRFARA peptide HeLa cell         CdsN hydrolyzed ATP rate 0.0734 &#194;&#177; 0.0053 nmol/min bility determined. Briefly, 50 100 &#194;&#181;M CdsN peptide,      concentration used assay (150 ng total). Prior test- <br /> control peptides, positive control (1% SDS plus 0.1% Triton     ing ability CdsL regulate CdsN activity, confirmed <br /> X-100) added sub-confluent HeLa cells plates.   CdsL lacked ATPase activity assaying 1 h supernatants harvested tested presence ade-    malachite green assay control possible contamination nylate kinase using cytotoxicity assay (Lonza ToxiLight&#194;&#174; BioAssay,   CdsL possessed ATPase activity. test regu- <br /> Rockland). cytotoxicity assay performed manu-       lation CdsN CdsL added increasing amounts CdsL <br /> facturer&#226;&#8364;&#8482;s protocol. Briefly, supernatants HeLa cell cultures     CdsN reaction mixture. addition CdsL <br /> incubated CdsN TRFARA peptide  MEM containing              reduced CdsN enzymatic activity 0.3 CdsL:1 CdsN molar ratio, <br /> cycloheximide) tested evidence eukaryotic cell cytotox-    effect dominant 2.1 CdsL:1 CdsN molar <br /> icity. Aliquots (5 &#194;&#181;L) supernatant mixed 25 &#194;&#181;L    ratio CdsL: CdsN, enzymatic activity reduced Adenylate Kinase Detection Reagent samples incubated          82% (Figure 1). increasing CdsL concentration room temperature 5 min. Relative light units (RLUs)        marginally reduced activity. addition control tagged <br /> measured using 20/20-n Single Tube Luminometer Turner           protein  CopN, T3S structural component) effect <br /> BioSystems (Sunnyvale). Assays conducted triplicate        CdsN activity. control, CdsL effect condition. Cell monolayers washed warm PBS.             enzymatic activity second C. pneumoniae ATPase, GspE (data <br /> 0.75 mL trypsin added  0.75 mL MEM      shown). GspE type II secretion ATPase possesses <br /> added complete trypsinization (trypsinization monitored      enzymatic activity. light microscopy). sample thoroughly resuspended aliquoted plastic cuvette cell number immediately       pepscan analysIs cdsn&#226;&#8364;&#8220;cdsl InteractIon <br /> quantitated determining optical density 800 nM using       shown CdsN enzymatically regulated spectrophotometer.                                                     CdsL, mechanism occurs unknown.                                                                        attempt expand understanding CdsL function electron MIcroscopy IMMunofluorescence                             regulate CdsN, utilized Pepscan epitope mapping HeLa cells grown 90% confluency 6 plates infected      identify domains CdsN responsible CdsL inter- C. pneumoniae, pre-incubated linear CdsN peptide             action. Purified CdsL added lapping peptide 30 min linear random peptide (MTS&#226;&#8364;&#8220;MFAVNAQ)                 library consisting 4000 lapping linear conformational 30 min various concentrations, centrifuged 45 min        peptides CdsN protein search peptide domain <br /> room temperature, followed incubation 37&#194;&#176;C 1 h.          mediating protein binding (Timmerman et al., 2007). CdsN  <br /> inoculum peptide removed medium replaced              tained binding regions mediated interaction CdsL. MEM containing 1 &#194;&#181;g/mL cycloheximide inhibit host              CdsL binding sequences spanned residues 221&#226;&#8364;&#8220;229, cell protein synthesis. Cells incubated 37&#194;&#176;C 72 h,         residues 265&#226;&#8364;&#8220;270 corresponding amino acids &#226;&#8364;&#339;RSVIVVSTS&#226;&#8364;&#65533; <br /> collected trypsinization centrifugation 1000 rpm         &#226;&#8364;&#339;TRFARA,&#226;&#8364;&#65533; respectively, present 10 min fixed overnight 4&#194;&#176;C 2% glutaraldehyde (v/v)        catalytic domain (Figure 2). corroborate Pepscan data, cacodylate buffer, 7.2. cell pellets post-fixed 2%      cloned expressed fragment CdsN spanning residues <br /> osmium tetroxide (v/v), embedded Spurs medium,             221&#226;&#8364;&#8220;270 containing CdsL predicted binding domains <br /> sections cut Reichert Ultracut E microtome. Sections         test CdsN fragment bound CdsL. examined Jeol 1200 electron microscope 80 kV          CdsN221&#226;&#8364;&#8220;270 purified CdsL presence pictures captured AMT digital camera. immun-          500 mM NaCl GST did purify CdsL ofluorescent staining inclusions, HeLa cells infected       salt conditions (Figure 3). <br />  <br />  <br /> Frontiers Microbiology | Cellular Infection Microbiology                                          February 2011 | Volume 2 | Article 21 | 4 <br />  Stone et al.                                                                                                      CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  <br />   FIgure 1 | CdsL regulations CdsN enzymatic activity dose-               Increasing amounts CdsL resulted decrease CdsN enzymatic activity. <br />   dependent fashion. Aliquots (150 ng) GST&#226;&#8364;&#8220;CdsN assayed                 maximum reduction activity seen 2.1: CdsL:1 CdsN molar ratio <br />   enzymatic activity using malachite green assay detect released inorganic   (82% reduction CdsN enzymatic activity). experiments performed   phosphate ATP. CdsL added reaction mixture molar ratios    triplicate data presented mean error bars representing 1 SD   0.3 2.1, ATPase activity GST&#226;&#8364;&#8220;CdsN measured.                mean. <br />  <br />  <br />  <br /> 3d ModelIng cdsn&#226;&#8364;&#8220;cdsl bIndIng doMaIn                                             MTS, YGRKKRRQRRR (Efthymiadis et al., 1998). extended crystal structure E. coli T3S ATPase, EscN, recently                 CdsN sequence flanking TRFARA peptide ensure elucidated, allowing use orthology modeling approach                  entire binding domain included, flanked peptide obtain predicted 3D structure CdsN visualize CdsL                cysteine residues assist disulfide bonding binding domains (Zarivach et al., 2007). CdsN EscN possess                    required binding. final peptide utilized 28-amino <br /> large sequence orthology catalytic domain,                     acid sequence YGRKKRRQRRRCVVLMMDSVTRFARALC. CdsL binding domains located (Stone et al., 2008).                    examine peptide-MTS entering EBs, incu- <br /> Using 3D-JIGSAW, 3D structure CdsN generated used                      bated GST GST&#226;&#8364;&#8220;MTS 50, 100, 500 &#194;&#181;M model binding domains CdsL. Modeling binding                      gastrografin purified EBs. saw GST&#226;&#8364;&#8220;MTS <br /> regions CdsL predicted CdsN monomer revealed                    entering EBs increased dose-dependent manner (Figure 5A). binding domains CdsL present central region                   GST control (500 &#194;&#181;m) taken poorly Chlamydia. CdsN (Figure 4A). revealing mapping binding                        added GST GST&#226;&#8364;&#8220;MTS increasing time periods <br /> domains shown hexameric CdsN modeled EscN                       (5, 30, 60 min) showed increasing uptake time, start- <br /> crystal structure (Zarivach et al., 2007). clarity, shown               ing early 5 min (Figure 5B).  GST control showed half predicted hexameric structure CdsN  CdsN                  poor uptake Chlamydia, suggesting MTS facili- <br /> trimeric complex). Residues 221&#226;&#8364;&#8220;229 exposed outer                    tates peptide entry time- dose-dependent manner. HeLa <br /> face hexamer (Figure 4B, blue), residues 265&#226;&#8364;&#8220;270                   cells infected C. pneumoniae MOI 2 exposed central channel hexameric ring (Figure 4C,                    stained inclusions 72 h using FITC-conjugated anti-MOMP <br /> red). location binding domains consistent pro-                 monoclonal antibody. CdsN peptide added EBs 0, <br /> posed mechanism CdsL regulation CdsN sterically hin-                     20, 50, 100-&#194;&#181;M concentrations 30 min prior infection <br /> dering access ATP P-loop (Figures 4B,C, magenta).                         saw infectivity reduced dose&#226;&#8364;&#8220;response fashion <br />                                                                                     (Figure 6A). absence peptide, inclusions clearly <br /> InhIbItIon C. pneumoniae growth replIcatIon cdsn                          visible 72 h (Figure 7). Pre-treatment cells 50 &#194;&#181;M <br /> cognate peptIde                                                                     peptide resulted drastic reduction number inclusions Pepscan binding data revealed distinct domains CdsN                      (&gt;95%), 100 &#194;&#181;M toxicity HeLa cells. rule mediate interaction CdsN CdsL. Using                      possibility inhibitory activity 50 &#194;&#181;M information sought design utilize peptide mimetic                       toxicity tested effect peptide 50 &#194;&#181;M HeLa <br /> targeting interaction CdsN CdsL examine                         cells using adenylate kinase release assay little effect chlamydial replication. discussed  CdsN pos-                 toxicity (Figure 6B). examined infected cells sessed different binding domains CdsL; selected                      CdsN random peptide transmission electron microscopy. amino acid sequence TRFARA create potential peptide                        infected cells exposed random peptide large inclusion <br /> mimetic. synthesized TRFARA peptide N-terminal                        bodies containing mixture RBs EBs readily visible <br />  <br />  <br /> www.frontiersin.org                                                                                                         February 2011 | Volume 2 | Article 21 | 5 <br />  Stone et al.                                                                                                 CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  <br />                                                                                     interacts putative type III secretion protein, CdsL (Stone <br />                                                                                     et al., 2008). extended observations                                                                                     CdsL functions regulate CdsN enzymatic activity.                                                                                     used novel approach map exact region                                                                                     CdsN mediates CdsN&#226;&#8364;&#8220;CdsL interaction, used                                                                                     information design peptide mimetic disrupts chlamy- <br />                                                                                     dial replication cycle. Combined, data provides new insights <br />                                                                                     regulation T3S importance                                                                                     chlamydial life-cycle. <br />                                                                                         YscL orthologs shown regulate T3S ATPase <br />                                                                                     enzymatic activity Yersinia, C. pneumoniae encodes YscL <br />                                                                                     ortholog (CdsL; Blaylock et al., 2006; Stone et al., 2008).                                                                                     CdsN treated increasing concentrations CdsL reduced <br />                                                                                     enzymatic activity 82%. regulation CdsL <br />                                                                                     specific tested ability CdsL regulate type <br />                                                                                     II secretion ATPase Chlamydia, GspE,                                                                                     effect enzymatic activity. suggests CdsL specific <br />                                                                                     regulation CdsN. CdsL previously shown <br />                                                                                     expressed time points replication cycle <br />                                                                                     apparent accumulation final 48 h (Slepenkin <br />                                                                                     et al., 2003). consistent role CdsL regulating T3S <br />                                                                                     invasion intracellular replication phases life-cycle <br />                                                                                     EBs RBs, respectively. <br />                                                                                         Type III secretion ATPases known mediate numerous pro- <br />   FIgure 2 | Pepscan analysis CdsL binding domain CdsN.                                                                                     tein interactions structural components chap- <br />   lapping peptide library length CdsN consisting 4000 linear   erone/effector complexes, play critical role construction <br />   looped peptides constructed Pepscan Presto  Netherlands).              injectisome secretion effectors (Gauthier Finlay, <br />   CdsL screened peptide library identify CdsN peptides       2003; Akeda Galan, 2005). used Pepscan Analysis deter- <br />   interact CdsL.   Raw Pepscan data obtained screening                                                                                     CdsN domain mediates interaction CdsL. <br />   length CdsL lapping peptide library 4000 linear   looped CdsN peptides (x-axis). distinct CdsL binding domains                                                                                     distinct domains CdsN   highlighted black vertical boxes, signal strength    interacted CdsL; exposed central channel   interaction visualized y-axis. peaks correspond          hexameric structure CdsN exposed outer <br />   sequences RSVIVVSTS TRFARA. (B) amino acid sequences               face. Using GST pull-downs, corroborated Pepscan data   CdsL binds (RSVIVVSTS TRFARA) bolded length <br />                                                                                     demonstrating 49-amino acid fragment CdsN containing <br />   CdsN amino acid sequence. <br />                                                                                     predicted CdsL binding domains able purify <br />                                                                                     length CdsL. shown previously CdsL forms <br />                                                                                     dimers, likely CdsL molecules interact                                                                                     different regions CdsN (Pallen et al., 2006; Stone et al., 2008). <br />                                                                                     consistent binding domains CdsN. <br />                                                                                     regulating ATPase activity, CdsL believed tether                                                                                     ATPase inner membrane, functioning similar manner                                                                                     gamma-stalk F0F1 ATPase runs central <br />   FIgure 3 | CdsN221&#226;&#8364;&#8220;270 peptide, containing predicted                 channel F0F1 hexamer interact membrane <br />   CdsN&#226;&#8364;&#8220;CdsL binding domains, purifies CdsL. GST&#226;&#8364;&#8220;CdsN221&#226;&#8364;&#8220;270             (Pallen et al., 2006). location CdsL binding domain   bound glutathione beads used pull CdsL Escherichia        central channel CdsN supports concept CdsL func- <br />   coli lysates. Beads collected centrifugation, washed 0, 200,                                                                                     tions similar manner gamma-stalk domain CdsN. <br />   500 mM NaCl probed tagged protein Western blot using anti   antibody. GST&#226;&#8364;&#8220;CdsN221&#226;&#8364;&#8220;270 purified CdsL GST did           binding domains close proximity P-loop <br />                                                                                     catalytic domain coordinates ATP binding                                                                                     cleavage (Stone et al., 2008). possible CdsL bound infected cells incubated 50 &#194;&#181;M CdsN peptide lacked                       domains sterically hinder access ATP detectable inclusions 200 cells examined                     P-loop, reducing enzymatic function CdsN. tethering <br /> showed evidence cytotoxicity (Figure 7).                                      function CdsL crucial T3S function, teth- <br />                                                                                     ering mediated binding domain central chan- <br /> dIscussIon                                                                          nel CdsN hexamer likely interaction maintained C. pneumoniae contains genes coding T3SS,                    entire developmental cycle. exposed binding small number characterized.                       domain outer surface CdsN hexamer,  previously shown CdsN possesses enzymatic activity                         responsible regulation CdsN CdsL. CdsL  <br />  <br /> Frontiers Microbiology | Cellular Infection Microbiology                                                       February 2011 | Volume 2 | Article 21 | 6 <br />  Stone et al.                                                                                                     CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  <br />   FIgure 4 | CdsL binding domains mapped predicted 3D <br />   structure CdsN reveals distinct domains. 3D structure CdsN            FIgure 5 | uptake gST-Membrane transport signal (MTS) fusion <br />   generated based orthology modeling using 3D-JIGSAW based            protein eBs. EBs purified discontinuous gastrografin gradient <br />   structure EscN, used visualize CdsL binding domains identified      collected centrifugation ensure use homogenous EB <br />   &#226;&#8364;&#8220; 30 &#226;&#8364;&#8220; peptide library binding data. CdsL binding domains     mixture. GST GST&#226;&#8364;&#8220;MTS protein expressed Escherichia <br />   CdsN blue (residues 221&#226;&#8364;&#8220;229) red (residues 265&#226;&#8364;&#8220;270), ATP        coli cells purified using glutathione beads. EBs incubated <br />   coordinating P-loop region CdsN highlighted magenta.                   GST GST&#226;&#8364;&#8220;MTS, trypsinized 30 min ensure   Representation predicted CdsN monomeric structure indicating          extracellular GST GST&#226;&#8364;&#8220;MTS degraded, examined &#206;&#177;-GST <br />   CdsL binding domains (RSVIVVSTS blue, TRFARA red). (B) Structural           Western blot presence intracellular GST-tagged protein. CdsL,   model showing location residues 221&#226;&#8364;&#8220;229 (RSVIVVSTS), displayed          intracellular type III secretion protein, used loading control (LC).   <br />   half predicted hexameric structure viewed outer face,     Time-course incubation EBs GST&#226;&#8364;&#8220;MTS 5, 30, 60 min. GST <br />   revealing binding domains exposed outside hexamer.     control 60 min displayed little protein GST&#226;&#8364;&#8220;MTS 5, 30,   P-loop represented magenta. (C) Structural model viewed            60 min increasing amounts GST&#226;&#8364;&#8220;MTS present EBs. (B) <br />   outside looking central channel showing location residues       Dose&#226;&#8364;&#8220;response EBs incubated GST&#226;&#8364;&#8220;MTS 50, 100, 500 &#206;&#188;m. GST <br />   265&#226;&#8364;&#8220;270 (TRFARA), displayed half predicted hexameric structure      control 500 &#206;&#188;m displayed little protein GST&#226;&#8364;&#8220;MTS accumulated <br />   revealing domain exposed central channel. P-loop    EBs concentration increased. CdsL shown LC. <br />   represented magenta. <br />  <br />                                                                                    cellular invasion. confirmed MTS facilitated <br /> bind dissociate outer CdsN&#226;&#8364;&#8220;CdsL binding domain                        uptake peptide EBs time- dose-dependent conformational change CdsN, activating ATPase                    manner. Afterward, treated chlamydial EBs 30 min <br /> activity CdsL release allowing ATP access P-loop.                CdsN peptide reduced chlamydial infection tempting speculate delivery effector/chaperone com-                  dose&#226;&#8364;&#8220;response fashion, concentration 50 &#194;&#181;M reduced <br /> plexes CdsN, possibly multi-cargo transport protein                Chlamydia infectivity &gt;95%, determined staining CdsQ, trigger CdsL release ATPase activation,                 counting inclusions 72 h. alternative explanation allowing dissociation effector/chaperone complex                data peptide bactericidal Chlamydia, <br /> subsequent secretion effector (Spaeth et al., 2009).                        feel unlikely random peptide failed     Using Pepscan epitope mapping shown CdsL                         inhibit chlamydial replication. shown addi- <br /> binds CdsN. Using binding data designed cognate                       tional peptide mimetics targeting essential protein inter- <br /> CdsN peptide containing membrane transport signal allow                       actions inhibit chlamydial replication (Mahony et al., peptide enter Chlamydia showed peptide                        2010). currently investigating CdsN peptide <br /> blocked chlamydial invasion HeLa cells. Membrane transport                      mimetic inhibit chlamydial replication administered signals known interact hydrophobically eukaryotic                      infection cycle, infection occurred, success- <br /> membranes, triggering entry endocytosis mem-                         ful indicate peptide disrupt CdsN&#226;&#8364;&#8220;CdsL com- <br /> brane (Efthymiadis et al., 1998). working hypothesis                   plexes RBs block T3S inclusion membrane. transport signal facilitate entry CdsN peptide                   suggest ATPase activity carefully regulated EBs, potentially disrupt CdsN&#226;&#8364;&#8220;CdsL                    CdsL EBs RBs, CdsN&#226;&#8364;&#8220;CdsL interaction interaction inhibit T3S, invasion,                   critical chlamydial replication. shown inhibition T3S se. achieved                         discovery unique CdsN binding domains CdsL cou- disrupting regulation ATPase interfering                 pled structural modeling binding domains allowed tethering function ATPase, displacing peripheral                    postulate CdsL functions regulate CdsN. association inner membrane. sce-                      allowed create cognate CdsN peptide functions <br /> narios disrupt T3SS, preventing chlamydial infection                     novel antimicrobial agent, preventing C. pneumoniae  <br />  <br /> www.frontiersin.org                                                                                                        February 2011 | Volume 2 | Article 21 | 7 <br />  Stone et al.                                                                                                                    CdsN possesses discreet binding domains CdsL <br />  <br />  <br />  <br />  <br />                                                                                                FIgure 7 | CdsN peptide reduces chlamydial infectivity HeLa cells                                                                                                staining eM. HeLa cells pretreated CdsN control <br />                                                                                                peptide (MTS&#226;&#8364;&#8220;MFAVNAQ), infected Chlamydia pneumoniae                                                                                                MOI 3 examined staining using anti-MOMP monoclonal <br />                                                                                                antibody treatment pathfinder kit   electron microscopy <br />                                                                                                 .   HeLa cells treated control peptide (100 &#206;&#188;M)                                                                                                infected C. pneumoniae. Immunofluorescent staining reveals numerous <br />                                                                                                infected cells   electron microscopy reveals large inclusion body <br />                                                                                                EBs RBs  . (B) HeLa cells treated CdsN peptide <br />                                                                                                (50 &#206;&#188;M) infected C. pneumoniae. Evaluation 200 cells treated <br />                                                                                                CdsN peptide staining EM examination revealed drastic <br />   FIgure 6 | CdsN peptide containing CdsL binding sequence TrFArA                          reduction C. pneumoniae infected cells complete lack detectable <br />   inhibits infection HeLa cells dose-dependent fashion shows                   inclusions EM. <br />   low toxicity 50 &#206;&#188;M. Shell vials grown 85% confluency infected   Chlamydia pneumoniae MOI 3 incubation CdsN control <br />   peptide. Infected cells counted staining Chlamydia specific major <br />   outer membrane protein (MOMP)-monoclonal antibody (Pathfinder kit). Toxicity               invading host cells. major difficulties working   measured using adenylate kinase release assay HeLa cells   presence CdsN control peptide.   CdsN peptide added                                                                                              Chlamydia genetically intractable. Peptide mimet- <br />   increasing concentrations (20, 50, 100 &#206;&#188;M) demonstrated dose&#226;&#8364;&#8220;                    ics CdsN peptide effectively create chemical <br />   response reduction C. pneumoniae infectivity HeLa cells. control              knockouts disrupting protein complexes, allowing mimic <br />   represents random peptide sequence MFAVNAQ&#226;&#8364;&#8220;MTS                      genetic knockouts explore specific functions various pro- <br />   effect C. pneumoniae infectivity. experiments performed triplicate <br />                                                                                              teins T3SS. <br />   data presented mean error bars representing mean + 1 <br />   SD **representing p &lt; 0.01. (B) Toxicity CdsN peptides HeLa <br />   cells evaluated using Lonza adenylate kinase release assay. CdsN              <br />   peptide added HeLa cells 1 h 50 100 &#206;&#188;M, random                 <br />   peptide added 1 h 100 &#206;&#188;M. saw CdsN peptide 50 &#206;&#188;M <br />                                                                                          <br />   toxicity HeLa cells, minor toxicity 100 &#206;&#188;M. <br /> </body></html>