<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Nat Cell Biol. 2011 January ; 13(1): . doi:10.1038/ncb2138. <br />  <br />  <br />  <br />                            Getting rid damaged goods: Loss RhoGAP SRGP-1 <br />                            promotes clearance dead injured cells C. elegans <br />                            Lukas J. Neukomm1,#, Andreas P. Frei1,2,&#206;&#8221;, Juan Cabello3,4, Jason M. Kinchen5, Ronen <br />                            Zaidel-Bar6, Zhong Ma5, Lisa B. Haney5, Jeff Hardin6, Kodi S. Ravichandran5, Sergio <br />                            Moreno3, Michael O. Hengartner1,* <br />  <br />                            1Instituteof Molecular Life Sciences, University Zurich, 8057 Zurich, Switzerland 2Ph.D. <br />                            Program Molecular Life Sciences (MLS), University Zurich/ETH Zurich, 8057 Zurich, <br />                            Switzerland 3Instituto Biolog&#195;&#173;a Molecular y Celular del C&#195;&#161;ncer. CSIC/Universidad                            Salamanca, 37007 Salamanca, Spain 4Center Biomedical Research La Rioja (CIBIR). C/ <br />                            Piqueras 98, 26006 Logro&#195;&#177;o, Spain 5Center Cell Clearance Department                            Microbiology, University Virginia, Charlottesville, Virginia 22902, USA 6Department Zoology, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            University Wisconsin &#226;&#8364;&#8220; Madison, Wisconsin 53706, USA <br />  <br />                            Abstract <br />                                  Multicellular animals rapidly clear dying cells organism. pathways                                  mediate cell removal conserved evolution. identify srgp-1 negative <br />                                  regulator cell clearance C. elegans mammalian cells. Loss srgp-1 function results <br />                                  improved engulfment apoptotic cells srgp-1 expression inhibits apoptotic cell <br />                                  corpse removal. SRGP-1 functions engulfing cells acts GTPase <br />                                  activating protein (GAP) CED-10(Rac1). Interestingly, loss srgp-1 function promotes                                  clearance dead cells, removal cells brought                                  verge death sublethal apoptotic, necrotic cytotoxic insults. contrast, impaired <br />                                  engulfment allows damaged cells escape clearance, results increased long-term <br />                                  survival. propose C. elegans uses engulfment machinery primitive,                                  evolutionarily conserved survey mechanism identifies removes unfit cells tissue. <br />  <br />                                              Multicellular organisms use programmed cell death (apoptosis) remove cells                                              superfluous potentially dangerous1, 2. Apoptotic cells immediately recognized, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              engulfed digested neighboring specialized cells. Compromised clearance cell <br />                                              corpses results persistence unwanted cell debris lead inflammation                                              autoimmune diseases3. development therapeutic approaches increase engulfment <br />                                              activity useful treating diseases. <br />  <br />                                              mechanisms underlying apoptotic cell engulfment evolutionary conserved4. Genetic <br />                                              studies C. elegans led identification &#226;&#8364;&#339;partially redundant&#226;&#8364;? signaling <br />  <br />                            * <br />                             correspondence addressed. michael.hengartner@imls.uzh.ch, Phone: +41 44 635 31 40. <br />                            #Current address: Department Neurobiology, Howard Hughes Medical Institute, University Massachusetts Medical School, <br />                            Worcester, Massachusetts 01605, USA. <br />                            &#206;&#8221;Current address: Institute Molecular Systems Biology, ETH Zurich, 8093 Zurich, Switzerland. <br />                            Author Contributions <br />                            L. J. N.,  P. F. R. -B. contributed generation nematode transgenics fluorescence microscopy studies;  P. F.                            L. J. N. conducted unbiased screen epistasis; J. C. performed 4D microscopic analysis; J. M. K. performed                            mammalian cell culture experiments;  M. L. B. H. performed pull downs hydrolysis assays; L. J. N. performed                            cell killing assay wrote manuscript.  P. F. M. O. H. contributed data analysis, project planning writing                            manuscript. authors contributed editing manuscript. <br />                             Neukomm et al.                                                                                            Page 2 <br />  <br />  <br />                                             pathways mediate engulfment degradation apoptotic cells (Figure S1).                                             pathway uses transmembrane proteins, CED-7(ABCA1 humans)                                             CED-1(MEGF10), function receptors dying cells5, 6. adaptor protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             CED-6(GULP) transduces signal(s) CED-1 downstream CED-10(Rac1)                                             potentially regulates proteins (e.g. DYN-1, RAB-7) recruited phagosome7&#226;&#8364;&#8220;9. <br />                                             second signaling cascade Rho GTPases act serial manner: RhoGEF <br />                                             UNC-73(TRIO) activates MIG-2(RhoG), turn regulates recruits                                             membrane bipartite CED-12(Elmo)/CED-5(Dock180) GEF complex, acts                                             GEF CED-10/Rac110&#226;&#8364;&#8220;12. <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> loading CED-10 facilitated adaptor <br />                                             molecule CED-2(CrkII)13&#226;&#8364;&#8220;15. pathways likely converge level CED-10, <br />                                             promotes extensive cytoskeletal rearrangements required engulfment8.                                             pathway, ABL-1(Abl) kinase opposes cell clearance ABI-1(Abi), possibly                                             modulation CED-10 activity16. <br />  <br />                                             Recently, MOM-5(Frizzled) shown act major receptor recognition                                             early embryonic corpses. Genetic analyses suggested MOM-5 regulates CED-10 <br />                                             activity CED-2, likely atypical Wnt signaling pathway includes <br />                                             GSK-3(GSK3&#206;&#178;) APR-1(APC)17. Additionally, integrins INA-1(Integrin &#206;&#177;)                                             PAT-3(Integrin &#206;&#178;) play redundant role corpse recognition recruit CED-2 <br />                                             phagocytic cup phospho-tyrosine dependent manner SRC-1(Src)18. <br />                                             Taken  observations suggest CED-10 center  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             eventually signaling pathways control cell corpse clearance. <br />  <br />                                             Rho GTPase superfamily members MIG-2 CED-10 cycle GTP-bound <br />                                             ( ) <span id='am-9' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span>GDP</span>-bound ( ) states. GTP loading promoted <span id='am-28' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span>Guanosine</span> exchange <br />                                             factors (GEFs), GTP hydrolysis facilitated GTPase activating proteins <br />                                             (GAPs). GEFs MIG-2 CED-10 identified,  GAPs affecting <br />                                             cell corpse engulfment known  <br />  <br />                                             identify GAPs MIG-2 CED-10, compiled list C. elegans genes <br />                                             predicted contain RhoGAP domain (Table S1)19, 20. hypothesized                                             engulfment deficient animals, knockdown RNA interference (RNAi) RhoGAPs <br />                                             involved cell corpse clearance lead partial regain engulfment activity                                              use Acridine Orange, Figure S2 Materials Methods). <br />  <br />                                             Using approach, identified single gene, srgp-1 (Slit-Robo GAP homolog),                                             knockdown resulted significant improvement activity ced-5 ced-6 mutant <br />                                             backgrounds (Figure 1A&#226;&#8364;&#8220;H Table S1). test srgp-1 mutants, ok300 <br />                                             tm3701, confirm RNAi results (Figure S3 <span id='am-21' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>; S4). mutant alleles NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             reduced number persistent cell corpses head freshly hatched ced-6 L1 larvae. <br />                                             (Figure 1I &amp; S5). effect reversed transgenic expression SRGP-1 <br />                                             driven endogenous srgp-1 promoter (Figure 1J). confirmed phenotype <br />                                             observed srgp-1 mutant worms loss srgp-1 function. <br />  <br />                                             reduction persistent apoptotic cell corpses srgp-1 mutants arisen                                             reduction apoptosis increased engulfment activity. took advantage                                             characterized fixed cell lineage nematodes directly test hypotheses21. 16 cells <br />                                             undergo programmed cell death anterior pharynx wild-type animals,                                             scored extra cell nuclei (&#226;&#8364;&#339;undead cells&#226;&#8364;?) apoptosis-defective ced-3(lf) mutants. <br />                                              srgp-1 mutants extra surviving nuclei identified, suggesting                                             developmental apoptosis affected srgp-1 (Figure 2A). used 4-dimensional <br />                                             microscopy follow 13 embryonic cell deaths21. observed overall <br />                                             development developmental apoptosis aberrant delayed srgp-1 mutant <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                            Page 3 <br />  <br />  <br />                                             embryos (Figure S6). contrast, striking decrease corpse persistence                                             different ced-6; srgp-1 embryos compared ced-6 single mutants: 10 cell corpses <br />                                             failed engulfed ced-6 embryos, 1 persistent corpse left NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             ced-6; srgp-1 embryos (Figure 2C Table S2). Cell corpse persistence reduced <br />                                             srgp-1 single mutant compared wild type (p &lt; 0.005). observations suggest <br />                                             loss SRGP-1 function results increased engulfment activity reduced <br />                                             apoptosis. <br />  <br />                                             determine SRGP-1 expression pattern, analyzed rescuing transgene <br />                                             opIs228[Psrgp-1::srgp-1::gfp] (Figure S7). Consistent neuronal function srGAP <br />                                             proteins vertebrates, expression nerve ring projecting sensory <br />                                             neurons tip head. larvae adults, SRGP-1 abundantly expressed                                             hypodermal tissues (Figure S7C, D &amp; E). SRGP-1::GFP localizes cortex cells                                             early embryos, particularly abundant highly condensed  e. late) apoptotic cell <br />                                             corpses (Figure S7A &amp; B). similar enrichment corpses observed                                             number genes involved corpse removal5, 22, suggesting SRGP-1 function                                             engulfing cells. directly test hypothesis, expressed srgp-1::gfp &#226;&#8364;&#339;tissue- <br />                                             specific&#226;&#8364;? manner. Expression dying cell using egl-1 promoter23 did affect cell <br />                                             corpse numbers, expression engulfing cell using ced-1 promoter provided <br />                                             significant rescue (Figure 2B). results demonstrate SRGP-1 functions                                             engulfing dying cells. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             loss srgp-1 function engulfing cells results increased engulfment activity,                                             asked expression srgp-1 engulfing cells cause opposite effect, <br />                                             inhibit cell corpse engulfment. rescuing transgene <br />                                             opEx1424[Psrgp-1::srgp-1::mcherry], extra-chromosomal high-copy array likely <br />                                             results srgp-1::mcherry expression, significantly enhanced persistent cell corpse <br />                                             phenotype engulfment mutants tested (Figure 2D). results support                                             hypothesis SRGP-1 direct negative regulator engulfment activity C. elegans. <br />  <br />                                             determine engulfment signaling cascade srgp-1 acts, generated <br />                                             double mutants srgp-1 various engulfment genes scored numbers                                             apoptotic cell corpses freshly hatched larval L1 heads.  addition                                             ced-6, ced-1 ced-7 mutant alleles partially suppressed                                             srgp-1(ok300) (Table 1A). scored directly engulfment activity visualizing F- <br />                                             actin formation apoptotic corpses internalization process8. genotype <br />                                             tested, loss srgp-1 led significant increase cell corpses covered actin halos <br />                                             (indicative active engulfment signaling) (Figure S8 Table S3). srgp-1 likely acts <br />                                             downstream parallel ced-1, -7, -6 signaling cascade. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Genes second signaling pathway (e.g. ced-2, -5, -12) mediate cell corpse <br />                                             clearance, number cell migration processes. example, mutants                                             pathway severe distal tip cell (DTC) migration defect. Interestingly,                                             srgp-1(ok300) mutation partially suppress persistent cell corpses DTC <br />                                             migration defects analyzed double mutants, suggesting SRGP-1 acts                                             negative regulator processes (Table 1B, Figure S8 Table S3). Importantly, <br />                                             srgp-1 suppress null allele RhoG homologue mig-2(mu28), suggesting                                             MIG-2 major target SRGP-1. <br />  <br />                                             hypomorphic ced-10 alleles, n1993 n3246 (membrane targeting GTP-binding <br />                                             defective, respectively), partially suppressed srgp-1(lf). Unfortunately,                                             ced-10(t1875) null allele scored L1 corpse assay embryonic <br />                                             lethality24. alternative, used AO staining measure ability srgp-1(ok300)  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                          Page 4 <br />  <br />  <br />                                             promote uptake/internalization germ cell corpses ced-5, ced-6 maternally <br />                                             rescued ced-10(null) mutants. loss srgp-1 leads increased numbers                                             AO+ corpses ced-5 ced-6, ced-10(null) mutants (Figure S9). Similarly, DTC <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             migration defects maternally rescued homozygote ced-10(null) hermaphrodites                                             suppressed srgp-1 mutants (Table 1B). conclude srgp-1(ok300)                                             compensate loss CED-10(Rac1) activity. Taken  results suggest                                             SRGP-1 acts downstream engulfment genes, possibly functions                                              parallel CED-10(Rac1). <br />  <br />                                             Given genetic epistasis data, tested possibility SRGP-1 acts GAP                                             CED-10. performed vitro binding assays SRGP-1 CED-10 variants                                             mimic GTP- GDP-bound states (Q61L T17N, respectively).                                             SRGP-1 GAP domain specifically interacted GTP-bound, GDP-bound <br />                                             CED-10 protein (Figure 3A); consistent property GAP proteins                                             associate GTP-bound version GTPases. SRGP-1 GAP failed bind GTP- <br />                                             bound MIG-2 RHO-1, Rho family members, demonstrating specificity                                             SRGP-1/CED-10 interaction (Figure 3B). Importantly, SRGP-1 GAP <br />                                             domain enhance intrinsic GTPase activity mammalian Rac1, albeit relatively <br />                                             high molar concentrations respect mammalian Rac1 (Figure 3C). confirm <br />                                             result, mutated conserved arginine finger SRGP-1 GAP domain                                             previously shown necessary GAP domains25. mutant <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             SRGP-1(R563A), bind GTPase, failed enhance GTPase activity <br />                                             (Figure 3C). <br />  <br />                                             ability C. elegans SRGP-1 act mammalian Rac1 suggested function                                             srGAP regulation cell corpse clearance evolutionary conserved. tested <br />                                             hypothesis looking engulfment apoptotic Jurkat cells Chinese hamster <br />                                             ovary LR73 cells. cells basal phagocytic activity, stimulated                                             expression BAI1, receptor upstream Rac1 involved cell corpse clearance                                             mammalian models26. inhibit engulfment activity LR73 cells  <br />                                             expression dominant negative Rac1(N17), mammalian srGAP1 protein <br />                                             (Figure 3D). contrast, siRNA-mediated knockdown individual srGAP <br />                                             family members minor effect engulfment activity, possibly                                             compensatory increases gene expression family members (Figure S10).  <br />                                             simultaneous knock srGAP family members resulted significant <br />                                             increase engulfment activity (Figure 3E). Taken  observations clearly <br />                                             suggest srGAP family members, particular srGAP1, function                                             inhibit cell corpse clearance mammals. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             support vitro results, performed vivo structure/function <br />                                             analysis. created variety rescue constructs encoding SRGP-1 derivatives lacking <br />                                             various domains SRGP-1 protein. constructs expressed levels similar                                             length protein  unpublished observations). SRGP-1 modifications lead                                             SRGP-1 mislocalization (&#206;&#8221; BAR), abrogated GAP activity (&#206;&#8221; GAP, GAPinactive BAR <br />                                              resulted complete loss rescuing activity (Figure 3F). &#206;&#8221; C-terminus <br />                                             construct failed rescue; reason currently unknown, construct <br />                                             showed nearly wild-type expression pattern.  modification tyrosines <br />                                             (Y724;732F) predicted regulated phosphorylation27 did alter rescuing <br />                                             ability, suggesting phosphorylation residues essential SRGP-1 <br />                                             function. <br />  <br />                                             wide range BAR superfamily members regulate invaginations protrusions                                             plasma membranes28. example, F-BAR domain-containing FCHo1/2 required  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                             Page 5 <br />  <br />  <br />                                             plasma membrane clathrin-coated vesicle formation29. Contrary, BAR domain                                             human srGAP2 necessary sufficient membrane localization induction                                             filopodia-like membrane protrusions30. Interestingly, SRGP-1 F-BAR domain failed NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             rescue corpse clearance phenotype assays, suggesting distinct modes action                                             srGAP family members BAR domains processes. <br />  <br />                                             Taken  vitro vivo results strongly suggest evolutionary conserved <br />                                             function SRGP-1 GAP CED-10(Rac1) cell corpse clearance. vivo, loss                                             srgp-1 function results enhanced engulfment signaling, partially suppress                                             cell corpse clearance defects engulfment mutants. suspect interventions                                             increase engulfment signaling effective treating human diseases <br />                                             characterized defective cell clearance. <br />  <br />                                             negative regulators C. elegans cell corpse engulfment recently                                             described16, 31, 32. Interestingly, suggested act  parallel  <br />                                             CED-10(Rac1), CED-5/CED-12 GEF complex. double mutant analysis <br />                                             suggests srgp-1 acts parallel 2  mtm-1 abl-1 (Figure S11).                                             multitude regulatory molecules acting CED-10(Rac1) emphasizes key position                                             GTPase engulfment signaling pathways.  likely additional <br />                                             regulators exist act points pathways. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             previously shown conditions limiting caspase activity (e.g. <br />                                             weak reduction function (rf) ced-3 mutants), loss engulfment activity leads                                             corpse persistence, promotes survival: ced-3(rf) backgrounds, significantly <br />                                             cells survive engulfment-defective mutants control strains33, 34.                                             test activated engulfment signaling  srgp-1 mutants) cause <br />                                             opposite phenotype, drive death cells survived.                                             looked ventral cord development, Pn.aap cells (P1, P2                                             P9&#226;&#8364;&#8220;P12) undergo programmed cell death. reporter transgenes nIs96 nIs106 <br />                                             express GFP differentiated Pn.aap cells [Plin-11::gfp], used score                                             survival cells (Figure 4A). 50% surviving Pn.aap cells                                             ced-3(rf) mutants did survive srgp-1 ced-3(rf) double mutants (Figure 4A&#226;&#8364;&#178;).                                             &#226;&#8364;&#339;killer&#226;&#8364;?-phenotype srgp-1(lf) mutants dependent ced-10, increased cell death <br />                                             observed ced-10(0) srgp-1 double mutants (Figure 4A&#226;&#8364;&#179;). similar effect                                             observed pharynx, srgp-1(ok300) reduced number surviving cells                                             ~50% distinct ced-3(rf) alleles (Figure S12). Importantly, case did observe <br />                                             loss cells normally fated die, failed undergo programmed cell <br />                                             death complete loss apoptotic pathway (e.g. strong ced-3(lf) mutants, Figure <br />                                             4A&#226;&#8364;&#178; S12). Taken  results demonstrate increased engulfment activity, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             srgp-1 mutants, promote removal cells viable close death                                             conditions limiting caspase activity. <br />  <br />                                             C. elegans engulfment pathway reported mediate removal                                             apoptotic cells, cells die necrosis  non-canonical pathways35, 36. <br />                                             example, necrosis-like death induced C. elegans rare gain function (gf) <br />                                             mutations various <span id='am-15' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> channels coding genes (DEG/ENaC family members mec-4, mec-8 <br />                                             mec-10). result, channels leaky, causeing osmotic imbalance, cell <br />                                             swelling, ultimately cell lysis37. used transgene Ismec-10(d) expressing <br />                                             MEC-10(A673V) GFP mechanosensory touch cell neurons, induces                                             incompletely penetrant, temperature sensitive neurotoxic cell death38. result,                                             touch cells undergo necrotic cell death, recover insult survive.                                             apoptotic cell death, hyperactive engulfment activity (srgp-1 mutants) <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                                            Page 6 <br />  <br />  <br />                                             promotes cell removal, impaired engulfment (ced-5 mutants) leads increased cell <br />                                             survival (Figure 4B&#226;&#8364;&#8220;B&#226;&#8364;&#179;). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Horvitz colleagues recently reported new type non-apoptotic, cytotoxic cell death <br />                                             induced lin-24(gf) lin-33(gf) C. elegans36. fraction affected Pn.p <br />                                             hypodermal blast cells undergoes morphological changes (distinct apoptotic necrotic <br />                                             cell death); die cleared, survive eventually recover                                             insult. Interestingly, modulation engulfment machinery influences cell survival                                             model cytotoxic cell death: engulfment-defective mutants showed increased Pn.p <br />                                             cell survival (Figure 4C, previously observed Galvin et. al.36), hyperactive <br />                                             engulfment led decreased Pn.p cell survival. <br />  <br />                                             Based observations, tempting speculate engulfment pathway                                             used generally C. elegans identify eliminate sick damaged cells (Figure 4D). <br />                                             propose sick cells tissue signal unhealthy status neighbor. <br />                                             Depending strength signal, sick cell tolerated (light blue),                                             removed (red, &#226;&#8364;&#339;eaten alive&#226;&#8364;?) phagocytosis. animals hyperactive engulfment <br />                                             pathway, weak signals lead cell removal. Conversely, engulfment-defective <br />                                             mutants, highly unhealthy cells tolerated allowing possibly recover                                             survive. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             interesting speculate type tissue quality control, cells sense                                             eliminate unfit neighbors, provide basis sophisticated regulatory <br />                                             mechanisms, cell competition.  Li Baker recently reported                                             involvement engulfment genes cell competition Drosophila <br />                                             development39. Given conservation engulfment pathway metazoa,                                             possibility recognize remove sick viable cells present mammals. <br />  <br />                               Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                             like thank Hengartner lab members comments discussions manuscript, M. Weiss <br />                                             statistical advice M. Jovanovic vivo pull supports. strains supplied                                             Caenorhabditis Genetic Center (CGC), C. elegans knock consortium (Oklahoma, USA) National <br />                                             Bioresource Project (Japan). work supported grants American Heart Association                                             American Cancer Society  J. M. K.), NIGMS/NIH  K. S. R., William Benter Senior Fellow                                             American Asthma Foundation), Junta Castilla y Le&#195;&#179;n (grant CSI03A08) Riojasalud Foundation (J. <br />                                             C.), Junta Castilla y Le&#195;&#179;n (Grupo Excelencia GR265) Spanish Ministry Science Innovation <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             (grants BFU2008-01808 Consolider CSD2007-00015) (S. M.), NIH postdoctoral training grant GM078747 <br />                                             fellowship Machiah Foundation (R. -B.), Swiss National Science Foundation, Ernst <br />                                             Hadorn Foundation European Union (FP5 project APOCLEAR) (M. O. H). <br />  <br />  <br />                               References <br />                                             1. Vaux DL, Korsmeyer SJ. Cell death development. Cell. 1999; 96:245&#226;&#8364;&#8220;254. [PubMed: 9988219] <br />                                             2. Debatin KM. Apoptosis pathways cancer cancer therapy. Cancer Immunology, <br />                                                Immunotherapy. 2004; 53:153&#226;&#8364;&#8220;159. [PubMed: 14749900] <br />                                             3. Savill J, Gregory C, Haslett C. Science. 2003; 302:1516&#226;&#8364;&#8220;1517. [PubMed: 14645835] <br />                                             4. Fullard JF, Kale  Baker N. Apoptosis. 2009 <br />                                             5. Zhou  Hartwieg E, Horvitz HR. CED-1 transmembrane receptor mediates cell corpse <br />                                                engulfment C. elegans. Cell. 2001; 104:43&#226;&#8364;&#8220;56. [PubMed: 11163239] <br />  <br />  <br />  <br />  <br />                                                  Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                                    Page 7 <br />  <br />  <br />                                             6. Wu YC, Horvitz HR. C. elegans cell corpse engulfment gene ced-7 encodes protein similar                                                ABC transporters. Cell. 1998; 93:951&#226;&#8364;&#8220;960. [PubMed: 9635425] <br />                                             7. Yu X, Odera S, Chuang CH, Lu N, Zhou  Developmental Cell. 2006; 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             8. Kinchen JM, et al. pathways converge CED-10 mediate actin rearrangement corpse <br />                                                removal C. elegans. Nature. 2005; 434:93&#226;&#8364;&#8220;99. [PubMed: 15744306] <br />                                             9. Liu QA, Hengartner MO. Candidate adaptor protein CED-6 promotes engulfment apoptotic <br />                                                cells C. elegans. Cell. 1998; 93:961&#226;&#8364;&#8220;972. [PubMed: 9635426] <br />                                             10. Gumienny TL, et al. CED-12/ELMO, novel member CrkII/Dock180/Rac pathway,                                                  required phagocytosis cell migration. Cell. 2001; 107:27&#226;&#8364;&#8220;41. [PubMed: 11595183] <br />                                             11. Wu YC, Horvitz HR. Nature. 1998; 392:501&#226;&#8364;&#8220;504. [PubMed: 9548255] <br />                                             12. deBakker CD, et al. Phagocytosis apoptotic cells regulated UNC-73/TRIO-MIG-2/RhoG <br />                                                  signaling module armadillo repeats CED-12/ELMO. Curr Biol. 2004; 14:2208&#226;&#8364;&#8220;2216. <br />                                                  [PubMed: 15620647] <br />                                             13. Reddien PW, Horvitz HR. CED-2/CrkII CED-10/Rac control phagocytosis cell migration <br />                                                  Caenorhabditis elegans. Nat Cell Biol. 2000; 2:131&#226;&#8364;&#8220;136. [PubMed: 10707082] <br />                                             14. Akakura S, et al. C-terminal SH3 domain CrkII regulates assembly function                                                  DOCK180/ELMO Rac-GEF. J Cell Physiol. 2005; 204:344&#226;&#8364;&#8220;351. [PubMed: 15700267] <br />                                             15. Tosello-Trampont AC, et al. Identification signaling submodules CrkII/ELMO/ <br />                                                  Dock180 pathway regulating engulfment apoptotic cells. Cell Death Differ. 2007; 14:963&#226;&#8364;&#8220;972. <br />                                                  [PubMed: 17304244] <br />                                             16. Hurwitz  et al. Abl kinase inhibits engulfment apoptotic [corrected] cells NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  Caenorhabditis elegans. PLoS Biol. 2009; 7:e99. [PubMed: 19402756] <br />                                             17. Cabello J, et al. Wnt pathway controls cell death engulfment, spindle orientation,                                                  migration CED-10/Rac. PLoS Biol. 2010; 8:e1000297. [PubMed: 20126385] <br />                                             18. Hsu TY, Wu YC. Engulfment apoptotic cells C. elegans mediated integrin alpha/SRC <br />                                                  signaling. Curr Biol. 2010; 20:477&#226;&#8364;&#8220;486. [PubMed: 20226672] <br />                                             19. Rual JF, et al. improving Caenorhabditis elegans phenome mapping ORFeome- <br />                                                  based RNAi library. Genome Res. 2004; 14:2162&#226;&#8364;&#8220;2168. [PubMed: 15489339] <br />                                             20. Kamath RS, Ahringer J. Genome-wide RNAi screening Caenorhabditis elegans. Methods. 2003; <br />                                                  30:313&#226;&#8364;&#8220;321. [PubMed: 12828945] <br />                                             21. Sulston JE, Schierenberg E, White JG, Thomson JN. embryonic cell lineage nematode <br />                                                  Caenorhabditis elegans. Dev Biol. 1983; 100:64&#226;&#8364;&#8220;119. [PubMed: 6684600] <br />                                             22. Kinchen J, Ravichandran K. Journal Cell Science. 2007; 120:2143&#226;&#8364;&#8220;2149. [PubMed: 17591687] <br />                                             23. Conradt B, Horvitz HR. C. elegans protein EGL-1 required programmed cell death                                                  interacts Bcl-2-like protein CED-9. Cell. 1998; 93:519&#226;&#8364;&#8220;529. [PubMed: 9604928] <br />                                             24. Patel FB, et al. WAVE/SCAR complex promotes polarized cell movements actin <br />                                                  enrichment epithelia C. elegans embryogenesis. Dev Biol. 2008; 324:297&#226;&#8364;&#8220;309. <br />                                                  [PubMed: 18938151] <br />                                             25. Barrett T, et al. Nature. 1997; 385:458&#226;&#8364;&#8220;461. [PubMed: 9009196] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             26. Park D, et al. Nature. 2007; 450:430&#226;&#8364;&#8220;434. [PubMed: 17960134] <br />                                             27. Blom N, Gammeltoft S, Brunak S. Sequence structure-based prediction eukaryotic protein <br />                                                  phosphorylation sites. J Mol Biol. 1999; 294:1351&#226;&#8364;&#8220;1362. [PubMed: 10600390] <br />                                             28. Suetsugu S, Toyooka K, Senju Y. Subcellular membrane curvature mediated BAR domain <br />                                                  superfamily proteins. Semin Cell Dev Biol. 2010; 21:340&#226;&#8364;&#8220;349. [PubMed: 19963073] <br />                                             29. Henne WM, et al. FCHo proteins nucleators clathrin-mediated endocytosis. Science. <br />                                                  328:1281&#226;&#8364;&#8220;1284. [PubMed: 20448150] <br />                                             30. Guerrier S, et al. F-BAR domain srGAP2 induces membrane protrusions required                                                  neuronal migration morphogenesis. Cell. 2009; 138:990&#226;&#8364;&#8220;1004. [PubMed: 19737524] <br />                                             31. Yang Y, Lu J, Rovnak J, Quackenbush SL, Lundquist EA. SWAN-1, Caenorhabditis elegans WD <br />                                                  repeat protein AN11 family, negative regulator Rac GTPase function. Genetics. 2006; <br />                                                  174:1917&#226;&#8364;&#8220;1932. [PubMed: 16980389] <br />                                             32. Zou W, et al. Caenorhabditis elegans myotubularin MTM-1 negatively regulates engulfment                                                  apoptotic cells. PLoS Genet. 2009; 5:e1000679. [PubMed: 19816564] <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                                     Page 8 <br />  <br />  <br />                                             33. Reddien PW, Cameron S, Horvitz HR. Phagocytosis promotes programmed cell death C. <br />                                                 elegans. Nature. 2001; 412:198&#226;&#8364;&#8220;202. [PubMed: 11449278] <br />                                             34. Hoeppner DJ, Hengartner MO, Schnabel R. Engulfment genes cooperate ced-3 promote <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 cell death Caenorhabditis elegans. Nature. 2001; 412:202&#226;&#8364;&#8220;206. [PubMed: 11449279] <br />                                             35. Chung S, Gumienny TL, Hengartner MO, Driscoll M. Nat Cell Biol. 2000; 2:931&#226;&#8364;&#8220;937. [PubMed: <br />                                                 11146658] <br />                                             36. Galvin BD, Kim S, Horvitz HR. Caenorhabditis elegans genes required engulfment                                                 apoptotic corpses function cytotoxic cell deaths induced mutations lin-24 lin-33. <br />                                                 Genetics. 2008; 179:403&#226;&#8364;&#8220;417. [PubMed: 18458102] <br />                                             37. Mano  Driscoll M. DEG/ENaC channels: touchy superfamily watches salt. Bioessays. <br />                                                 1999; 21:568&#226;&#8364;&#8220;578. [PubMed: 10472184] <br />                                             38. Zhang W, et al. Intersubunit interactions mutant DEG/ENaCs induce synthetic <br />                                                 neurotoxicity. Cell Death Differ. 2008; 15:1794&#226;&#8364;&#8220;1803. [PubMed: 18670436] <br />                                             39. Li W, Baker NE. Cell. 2007; 129:1215&#226;&#8364;&#8220;1225. [PubMed: 17574031] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                                 Page 9 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. Loss srgp-1 activity reduces numbers persistent apoptotic cell corpses C. <br />                                             elegans <br />                                              H) srgp-1(RNAi) allows Acridine Orange (AO) straining (indicative engulfment) <br />                                             germ cell corpses ced-6 mutants. Adult hermaphrodites indicated genotypes <br />                                             stained AO described Materials Methods observed DIC   <br />                                             D) fluorescence microscopy (E&#226;&#8364;&#8220;H). strains carry gla-1 mutation order                                             increase overall number apoptotic germ cells. Arrows point AO-positive NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             arrowheads AO-negative germ cell corpses. pictures, anterior left, dorsal                                              Scale bar, 10 &#206;&#188;m. <br />                                               Loss srgp-1 reduces numbers persistent apoptotic cell corpses (arrowheads)                                             head ced-6 L1 larvae. Scale bar, 10 &#206;&#188;m. <br />                                             (J) Expression srgp-1::gfp srgp-1::mcherry driven endogenous srgp-1 promoter <br />                                             rescues srgp-1(ok300) phenotype. Persistent cell corpses scored head region <br />                                             freshly hatched L1 larvae indicated genotypes. Data shown average &#194;&#177; <br />                                             standard deviation (avg. &#194;&#177; s.d.); n, number scored individuals. <br />                                             Alleles used: gla-1(op234), ced-6(n1813), srgp-1(ok300), <br />                                             opIs224[Psrgp-1::srgp-1(cDNA)::gfp], opIs228[Psrgp-1::srgp-1(genomic)::gfp]                                             opEx1424-6[Psrgp-1::srgp-1(genomic)::mcherry]. <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                              Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. SRGP-1 acts engulfing cell antagonizes engulfment activity <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Developmental apoptosis defective srgp-1 mutants. Extra cell nuclei                                             scored pro- metacorpus L3/L4 larval heads indicated genotypes. Alleles <br />                                             used: ced-3(n2433), ced-6(n1813) srgp-1(ok300). <br />                                             (B) Focus action analysis: expression srgp-1::gfp engulfing cell (opEx1373) <br />                                             rescues srgp-1(ok300) phenotype; expression apoptotic cells (opIs391) does  <br />                                             Alleles used: ced-6(n1813), srgp-1(ok300), opEx1373[Pced-1::srgp-1(cDNA)::gfp]                                             opIs391[Pegl-1::srgp-1(cDNA)::gfp]. <br />                                             (C) Loss srgp-1 increases engulfment kinetics. 13 apoptotic cell deaths                                             AB lineage followed 4D microscopy cell corpse persistence measured <br />                                             described Materials Methods. circle represents single cell. Analysis                                             performed independent individuals genetic background (n = 3 &#195;&#8212; 13 cells). <br />                                             Cells indicated engulfed floated tissue egg-shell cavity                                             lineage followed anymore beginning muscle contraction  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                        Page 11 <br />  <br />  <br />                                             1.5-fold stage (320 min post fertilization &#226;&#8240;&#710; 120 min post onset cell death). Alleles <br />                                             used: ced-6(tm1826) srgp-1(ok300). * p &lt; 0.005 (Kolmogorov-Smirnov test). <br />                                             (D) expression srgp-1::mcherry enhances corpse persistence. Persistent cell corpses <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             scored head region freshly hatched L1 larvae indicated genotypes. <br />                                             Alleles used: ced-1(e1735), ced-2(n1994), ced-5(n1812), ced-6(n1813), ced-7(n1996), <br />                                             ced-10(n1993) ced-12(k149). Indicated strains carried unc-119(ed3)                                             opEx1424[Psrgp-1::srgp-1::mcherry] transgene. * p &#226;&#8240;&#164; 0.02, ** p &lt; 10&#226;&#710;&#8217;6 *** p &lt; 10&#226;&#710;&#8217;10 (t- <br />                                             test, 1-tailed, unequal variances). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                        Page 12 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 3. SRGP-1(srGAP1) binds modulates CED-10(Rac1) GTPase activity <br />                                               GAP domain SRGP-1 specifically binds GTP-loaded CED-10 vitro. Q61 <br />                                             N17 isoforms (resembling GTP- GDP-bound, respectively) tagged CED-10 <br />                                             used GST-fusion pull using GAP domain SRGP-1, Pak1 p21 binding <br />                                             domain (PBD, positive control) GST (negative control). <br />                                             (B) SRGP-1 GAP domain preferentially binds CED-10 vitro. GTP-bound <br />                                             isoforms tagged CED-10, MIG-2 RHO-1 (Q61, Q65 Q63, respectively)                                             used GST-fusion pull using GAP domain SRGP-1, GTP hydrolysis- <br />                                             deficient GAP(R563A) GST  <br />                                             (C) SRGP-1 GAP domain promotes Rac1 GTP-hydrolysis vitro. GTP-hydrolysis                                             human Rac1 (loaded [&#206;&#179;-32P]GTP) measured loss radioactivity Rac1                                             result cleavage &#206;&#179;-32P GTP. controls, loaded Rac1 treated                                             buffer GTP hydrolysis-deficient GAP(R563A). Data shown average &#194;&#177; <br />                                             standard deviation (avg. &#194;&#177; s.d., n = 4 independent experiments). <br />                                             (D) srGAP1 function corpse removal evolutionarily conserved. LR73 cells NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             transiently transfected triplicate indicated plasmids (srGAP1, FLAG::Rac1(N17), <br />                                             BAI1::GFP GFP  respectively). Phagocytosis GFP-positive cells assessed <br />                                             color flow cytometry. percentage GFP-positive cells ingested <br />                                             apoptotic SCI cells indicated condition. expression transfected <br />                                             proteins confirmed immunoblotting total lysates indicated. <br />                                             (E) srGAP family members inhibit cell corpse clearance. Knock 3 srGAP(1-3) <br />                                             family members increases phagocytotic activity LR73 cells. Knockdown individual <br />                                             family members corresponding phagocytic activities shown Figure S11. <br />                                             (F) vivo structure/function analysis SRGP-1. GFP fusions srgp-1(cDNA) length, <br />                                             &#206;&#8221; BAR, &#206;&#8221; GAP, GAP inactive (&#206;&#8221; FRV (&#206;&#8221; aa 562-4)), &#206;&#8221; C-terminus, BAR                                             Y(724;732)F tested rescue ability ced-6; srgp-1 mutants. Persistent cell corpses <br />                                             scored head region freshly hatched L1 larvae indicated genotypes. Data <br />                                             shown average &#194;&#177; standard deviation. Alleles used: ced-6(n1813) srgp-1(ok300). <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                               Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 4. Enhanced engulfment promotes removal viable sick cells C. elegans <br />                                              ) Loss srgp-1 promotes killing cells verge apoptotis larval <br />                                             ventral cord. 12 blast cells (P1&#226;&#8364;&#8220;P12) divide post-embryonically generate Pn.aap <br />                                             motor neurons ventral cord, labeled nIs96[Plin-11::gfp] reporter33. P3.aap <br />                                             &#226;&#8364;&#8220; P8.aap generate VC motor neurons (white arrowheads) innervade vulval <br />                                             muscles. P1.aap, P2.aap P9.aap - P12.aap undergo programmed cell death wild <br />                                             type, survive apoptosis defective ced-3(lf) animals differentiate VC-like <br />                                             neurons (green arrowheads).   Representative wild type, ced-3(lf) (loss function), <br />                                             ced-3(rf) (reduction function) srgp-1 ced-3(rf) L4 larvae shown. Alleles used: <br />                                             ced-3(lf) = n717, ced-3(rf) = op149 srgp-1(ok300), nIs96 present backgrounds. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Scale bar, 100 &#206;&#188;m  ) Quantification Pn.aap extra cell survival (P1, P2 &amp; P9&#226;&#8364;&#8220;P12, <br />                                             labeled green arrowheads) monitored indicated genotypes using nIs96 <br />                                             transgene. srgp-1(lf) sufficient kill cells normally survive (P3.aap &#226;&#8364;&#8220; P8.aap). <br />                                             Alleles used: ced-3(n717lf), ced-3(op149rf), ced-3(n2438rf), srgp-1(ok300). Data <br />                                             shown avg. &#194;&#177; s.d. 3 experiments, n &#226;&#8240;&#165; 40 animals/exp. * p &lt; 10&#226;&#710;&#8217;10 (t-test, 1-tailed, <br />                                             unequal variances).  ): Enhanced cell killing srgp-1 mutants ced-10-dependent. Extra <br />                                             cells ventral cord wild type, ced-10(t1875) ced-10(t1875) srgp-1(ok300)                                             scored using nIs106[Plin-11::gfp] transgene33 (n &#226;&#8240;&#165; 100 genotype). ced-10 animals <br />                                             generated heterozygote mothers. <br />                                             (B&#226;&#8364;&#8220;B&#226;&#8364;&#179;) Engulfment activity modulates removal viable cells subjected neurotoxic <br />                                             insult. (B) Ismec-10(d) transgene induces weak temperature-sensitive neurotoxic cell <br />                                             death expression MEC-10(A673V), codes leaky DEG/ENaC <br />                                             channel touch neurons38. Survival PLM touch neurons (labeled GFP) <br />                                             quantified L4 larvae grown 20&#194;&#176;C (B&#226;&#8364;&#178;) 15&#194;&#176;C (B&#226;&#8364;&#179;) indicated <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                          Page 14 <br />  <br />  <br />                                             genotype. Alleles used: ced-5(tm1949) srgp-1(ok300), Ismec-10(d) present                                             genetic backgrounds. Scale bar, 20 &#206;&#188;m. <br />                                             (C) Engulfment activity modulates removal viable cells subjected non-classical <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             toxic insult. animals carrying gain function mutations lin-24 lin-33,                                             Pn.p hypodermal blast cells increase refractivity form non-circular (oval) <br />                                             shaped bodies persist minutes 3 hours L1 larvae. Subsequently, <br />                                             cells removed survive  recover) 36. number surviving <br />                                             Pn.p cells scored early L3 larvae based DIC optics. Data shown avg. &#194;&#177; s.d., n <br />                                             = 25. * p &lt; 10&#226;&#710;&#8217;4 (t-test, 1-tailed, unequal variances). Alleles used: srgp-1(ok300)                                             ced-2(e1752). <br />                                             (D) Simplified model sick cell tolerance vs. removal  text). <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Nat Cell Biol. Author manuscript; available PMC 2013 October 28. <br />                             Neukomm et al.                                                                                                               Page 15 <br />  <br />  <br />  <br />  <br />                                                                                                Table 1 <br />                                               srgp-1 acts downstream parallel engulfment signaling pathways <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Engulfment: Persistent cell corpses scored head L1 larvae indicated genotypes. Asterisk, <br />                            corpse numbers scored early fold embryos dyn-1(n4039) embryos fail hatch. Data shown <br />                            average &#194;&#177; standard deviation. <br />                            Cell migration: Adult worms gonads deviated wild type U-shaped tube scored                            DTC-defective, described materials methods. n.d.,  <br />  <br />                             Loss srgp-1 reduces ced-7, ced-1, ced-6 dyn-1 mutant persistent cell corpses. <br />  <br />                                                 Genotype    Corpses/L1 head (DIC, n = 20)       % mismigrated DTC (n &#226;&#8240;&#165; 100) <br />                                                 wild type                          0.1 &#194;&#177; 0.2                             0.4 <br />                                             srgp-1(ok300)                          0.0 &#194;&#177; 0.0                             8.5 <br />     <br /> </body></html>