<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Characterization Novel Class Transcription Factor (CITFA) <br /> Subunit Indispensable Transcription Multifunctional RNA Polymerase Trypanosoma brucei <br /> Tu N. Nguyen Bao N. Nguyen Ju Huck Lee  Aswini K. Panigrahi,c* Arthur G&#195;&#188;nzla,b <br /> Department Genetics Developmental Biology, University Connecticut Health Center, Farmington, Connecticut, USAa; Department Molecular, Microbial Structural Biology, University Connecticut Health Center, Farmington, Connecticut, USAb; Seattle Biomedical Research Institute, Seattle, Washington, USAc <br />  <br /> Trypanosoma brucei organism known evolved multifunctional RNA polymerase (pol  used express parasite&#226;&#8364;&#8482;s ribosomal RNAs, major cell surface antigens,  variant surface glycoprotein <br /> (VSG) procyclin, vital establishing successful infections mammalian host tsetse vector, respec- <br /> tively. far, biochemical analyses T. brucei RNA pol transcription machinery elucidated subunit structure enzyme identified class transcription factor (CITFA). CITFA binds RNA pol promoters, CITFA-2 <br /> subunit shown absolutely essential RNA pol transcription parasite. Tandem affinity purification (TAP) CITFA revealed subunits CITFA-1 -6, conserved kinetoplastid organisms, plus dynein light <br /> chain DYNLL1.  tagging CITFA-6 instead CITFA-2, complex purified contained known CITFA subunits, novel proline-rich protein. Functional studies carried vivo vitro, colocalization study,  <br /> equivocally demonstrated protein bona fide CITFA subunit, essential parasite viability indispensable RNA pol transcription ribosomal gene units active VSG expression site mammalian-infective life cycle stage parasite. Interestingly, CITFA-7 function appears species specific, expression RNA interference (RNAi)- <br /> resistant CITFA-7 transgene Trypanosoma cruzi rescue lethal phenotype silencing endogenous CITFA-7. <br />  <br />  <br />   n eukaryotes, RNA polymerase (pol  transcribes exclusively <br />    large ribosomal gene units nucleolus. unicellular <br /> parasite Trypanosoma brucei exceptional regard                                                                                    RNA pol allows parasite express VSG                                                                                    single gene. mammals RNA pol unable synthesize <br />                                                                                    functional mRNA (5), deviating gene expression mechanisms organism known evolved multifunctional                    trypanosomatids enables T. brucei use RNA pol RNA pol used rRNA synthesis                       mRNA synthesis. trypanosomatids, protein-coding genes expression proteins crucial parasite&#226;&#8364;&#8482;s successful              arranged long tandem arrays transcribed polycis- <br /> interaction hosts. T. brucei tsetse-borne parasite                tronically, RNA precursors resolved individual <br /> sub-Saharan Africa causes lethal diseases humans live-                 mRNAs spliced leader (SL) trans splicing polyadenylation <br /> stock animals (2). lives freely mammalian bloodstream                 (reviewed reference 6). eukaryotes, mRNA cap- <br /> virtue dense coat variant surface glycoprotein (VSG)                 ping occurs cotranscriptionally direct interaction cap- <br /> shields invariant membrane proteins immune recognition                        ping enzymes RNA pol II, SL trans splicing, (32) antigenic variation enables parasite evade                   capped, 5=-terminal SL RNA fused 5= end host&#226;&#8364;&#8482;s immune  &#226;&#172;&#402;10 million VSG copies                      mRNA, uncouples capping RNA pol II transcription, surface bloodstream-form (BF) trypanosome,                   enabling T. brucei RNA pol express functional mRNA expressed single gene drawn repertoire                  (26, 37). <br /> 2,000 VSG genes (16). accommodate dense coat, active                        multifunctional RNA pol T. brucei versatile. <br /> VSG gene, resides 15 telomeric expression sites                    eukaryotes RNA pol confined nucleolus, <br /> (ESs) (11), needs transcribed extremely high rates;                transcribes rRNA gene units (RRNAs), active VSG ES <br /> estimated VSG RNA synthesis active ES exceeds                   transcribed outside nucleolus (3) extranucleolar ex- single &#226;?&#164;-tubulin gene &#226;&#172;&#402;50-fold (4). high VSG expres-                  pression site body (ESB), DNase-resistant compartment sion required antigenic variation essential BF              appears limit productive transcription single site (17). viability itself, VSG silencing led rapid block trypano- proliferation culture clearance parasites  <br /> fected mice (33).                                                                     Received 11 September 2012 Accepted 22 October 2012 <br />     eukaryotic cells, RNA pol transcription typically accounts                   Published ahead print 26 October 2012 50% total transcriptional activity,                     Address correspondence Arthur G&#195;&#188;nzl, gunzl@uchc.edu. number ribosomal gene units far lower number                       * Present address: Ju Huck Lee, Harvard Medical School, Boston, Massachusetts, protein-coding genes. efficiency RNA pol                      USA; Aswini K. Panigrahi, KAUST, Thuwal, Saudi Arabia. <br /> appears result high transcription initiation rates,                      Supplemental material article http://ec.asm.org/. impressively documented transmission elec-                         Copyright &#194;&#169; 2012, American Society Microbiology. Rights Reserved. <br /> tron microscopy called &#226;&#8364;&#339;Miller spreads&#226;&#8364;? (reviewed refer-                     doi:10.1128/EC.00250-12 <br /> ence 28). likely high efficiency  <br />  <br /> December 2012 Volume 11 Number 12                                             Eukaryotic Cell   p. 1573&#226;&#8364;&#8220;1581                                            ec.asm.org     1573 <br />  Nguyen et al. <br />  <br />  <br />  <br /> addition VSG expression, parasite utilizes RNA pol           untranslated region (UTR) (positions 409 941 relative translation <br /> insect-stage, procyclic form (PF) transcription gene loci         initiation codon) vector pT7-stl (1). Allele deletions (25) encode types cell surface antigen procyclin.           achieved transfecting PCR products 100 bp 5= 3= gene <br /> Procylins important parasite establish successful             flanks fused hygromycin phosphotransferase coding region. <br /> infections tsetse vector (27). RRNA, VSG ES, pro-                 following DNA oligonucleotides used reverse transcription <br />                                                                              (RT)-PCR: CITFA-7, 5=-TAGGGCCCAGCAACCTTGCCTGCTATATTA <br /> cyclin gene promoters structurally different, suggesting                                                                              CTTGATACC-3=/5=-TAGGTACCTTGCACTCCTGCGTCGGCCGTGA recruit different transcription factors (9).                                                                              GC-3=; CITFA-2, 5=-ATAACTCGGAGACTGACTCG-3=/5=-AACTGGT <br /> promoters absent trypanosomatid organisms Trypano-                GTATTGAACAACG-3=; CITFA-6, 5=-AGAAGCTTACGCGTATGGCT <br /> soma cruzi Leishmania spp., expect T. bru-             GTACAACCGGGAAAGGTG-3=/5=-AGTCTAGACTCGAGGTTATGT <br /> cei-specific proteins VSG ES procyclin gene transcription.           ACCTGAATCCAGCTGTC-3=; Tb927.7.2590, 5=-TCGAGCAACTTGAA  proteins involved T. brucei RNA pol transcrip-             GAGTTACG-3=/ 5=-TCCTGTGGCTTCTGTTCGTTGG-3=; TFIIB, 5=-C <br /> tion far conserved trypanosomatids, suggesting              ACCTCGACTACCACAGATTT TGTTGGGC-3=/5=-TGGTGATCAACG fulfill general RNA pol functions. Alternatively,          TCCTCATACGTGG-3=; TcCITFA-7, 5=-CCTCCCTTTAGCTGCGATG <br /> possible common transcription factors gained specific             CCTAATGTGCGCCGTTGTCGTTGG-3=/5=-TAGCGGCCGCTCCGA <br /> functions protein-coding gene transcription T. brucei evo-            TGACAACTCATTTCCCTCTTG-3=; VSG221, 5=-AGCTTTTTGGCAAC <br /> lution.                                                                      CTCTTTGCCAGG-3=/5=-TGGTAACGCCTGTTTTGCCAGCTATTCC-3=. <br />     Bioinformatic biochemical analysis T. brucei RNA pol                Cells. BFs cultured HMI-9 medium described previously <br />                                                                              (22). stable BF transfection, 3 &#226;&#171;&#187; 107 cells, grown exponential phase, <br /> revealed 10 12 core subunits present nuclear <br />                                                                              washed 2 ml Cytomix (120 mM KCl, 0.15 mM CaCl2, 10 mM <br /> RNA Pol (13, 19, 35). notion missing ortholog                                                                              K2HPO4, 25 mM HEPES-KOH, pH 7.6, 2 mM EDTA, 5 mM MgCl2), <br /> essential yeast RPA43 subunit replaced T. brucei          resuspended 0.4 ml Cytomix, mixed 10 &#226;?&#174;g linearized plas- <br /> RNA pol II paralog RPB7 (13) substantiated (22).                mid DNA 5 &#226;?&#174;g PCR product. Cells electroporated 1.2 kV, 186 <br /> Instead, essential RNA pol subunit, termed T. brucei RPA31              &#226;?&#8364;, 50 &#226;?&#174;F BTX ECM600 electroporator, immediately resuspended (TbRPA31), novel subunit extremely diver-            50 ml 37&#194;&#176;C prewarmed medium, dispensed 1-ml aliquots gent ortholog yeast RPA43, (18). Recently, class          24 plates. incubation overnight, 1 ml antibiotic-contain- <br /> transcription factor (CITFA) identified; purification led          ing medium added  Cells selected using G418, identification novel subunits, termed CITFA-1 -6,           phleomycin, hygromycin, blasticidin final concentrations 2.5, 1, 1, <br /> plus dynein light chain DYNLL1  known LC8) (1).                  2 &#226;?&#174;g/ml, respectively. PF culturing transfection carried Tandem affinity-purified CITFA bound 67-bp-long VSG ES                described previously (8, 14). <br /> promoter electrophoretic mobility shift assay, requiring                   vitro transcription. Preparation transcriptionally active extracts <br />                                                                              cotranscription reactions BES promoter template VSG-trm upstream promoter elements efficient binding. gel <br />                                                                              RNA pol II-recruiting control template SLins19 carried shift competed RRNA procyclin gene promot- <br />                                                                              described previously (14). immunodepletion CITFA-2 ers, indicating CITFA general factor RNA pol tran-           CITFA-7 transcription reactions, extracts prepared cells <br /> scription T. brucei. Accordingly, silencing CITFA-2 expression            exclusively expressing PTP&#226;&#8364;&#8220;CITFA-2 (1) CITFA-7&#226;&#8364;&#8220;PTP depleted lethal specifically reduced rRNA VSG mRNA abun-                  IgG beads (GE Healthcare). parallel, extracts mock treated <br /> dances BFs.  vitro transcription assays unequivo-             beads. Transcription signals obtained primer extension <br /> cally demonstrated CITFA-2 absolutely required                  32P-end-labeled oligonucleotides Tag-PE SLtag, hybrid- <br /> transcription types RNA pol promoters (1).               ize unrelated oligonucleotide insertions VSG-trm SLins19 <br />     identified novel, proline-rich protein            RNAs, respectively. Primer extension products separated  <br /> purified CITFA-6 extract. protein known                 naturing 50% <span id='am-59' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-60' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span>&#226;&#8364;&#8220; 6% polyacrylamide gels detected autora- <br /> protein domains conserved trypanosomatids.                 diography. data protein bona fide CITFA subunit                 RNA analysis. Total RNA preparation hot phenol method                                                                              Northern blot primer extension analyses carried previ- <br /> essential RRNA VSG transcription BFs. Interestingly, <br />                                                                              ously described (1, 18). Reverse transcription total RNA using <br /> CITFA-7 silencing rescued RNA interference (RNAi)-                 SuperScript II reverse transcriptase (Invitrogen), following manufac- <br /> resistant T. brucei CITFA-7 (TbCITFA-7) transgene             turer&#226;&#8364;&#8482;s recommendations. <br /> heterologous T. cruzi ortholog (TcCITFA-7). data sug-                  Antibodies. generation immune sera, length CITFA-6 <br /> gest TcCITFA-7 generally defective T. brucei, ex-           CITFA-7 recombinantly expressed Escherichia coli periment confirmed previous observations mammals es-                 N-terminal <span id='am-6' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glutathione</span> S-transferase (GST) tag purified glutathi- <br /> sential functions RNA pol transcription diverge rapidly            affinity chromatography. GST fusion proteins used im- <br /> (10).                                                                        munizing rats published previously (29). addition immune <br />                                                                              sera, immunoblot detections carried polyclonal antibodies <br /> MATERIALS METHODS                                                        directed CITFA-2 (1), TFIIB (29), spliceosomal protein <br /> DNAs. pCITFA-6-PTP-NEO described previously (1). pCITFA-7-               U2A=  known U2-40K [20]). PTP tag detected PTP-NEO pCITFA-7-HA-BLA generated inserting 353 bp            monoclonal anti-protein C epitope antibody (Roche)  CITFA-7 coding region (position 56 position 408) ApaI    oxidase antiperoxidase reagent (Sigma). hemagglutinin (HA) tag  <br /> NotI sites vectors pC-PTP-NEO (31) pC-HA-BLA (18), respec-            tection, rat monoclonal anti-HA antibody used (Roche). <br /> tively.  pTcCITFA-7-HA-bla, 543 bp T. brucei CITFA-7 5=            Protein analysis. Extract preparation CITFA-6-P CITFA-7- <br /> gene flank fused T. cruzi CITFA-7 open reading frame (ORF)     PTP tandem affinity purifications conducted according stan- <br /> overlap PCR. product cloned vector pC-HA-BLA                dard protocol (31). Purified proteins separated SDS&#226;&#8364;&#8220;10 20% ApaI NotI sites. construct linearized MscI site   polyacrylamide gradient gels stained Coomassie blue (Gelcode integration T. brucei CITFA-7 locus. RNAi vector            Coomassie stain; Thermo Fisher Scientific). protein identification, <br /> pCITFA-7-T7-stl generated inserting 533 bp CITFA-7 3=          protein bands excised gels, digested trypsin, subjected <br />  <br />  <br /> 1574   ec.asm.org                                                                                                                          Eukaryotic Cell <br />                                                                                                                                Novel CITFA Subunit T. brucei <br />  <br />  <br />  <br />  <br /> FIG 1 Identification CITFA-7.   Tandem affinity purification CITFA-6 &#226;&#8364;&#8220;PTP. Purified proteins separated SDS-PAGE stained Coomassie blue. comparison, 0.002% crude extract (Inp) 5% TEV protease eluate coanalyzed. right, mass spectrometric protein <br /> identifications CITFA-1 -7, DYNLL1, CITFA-2 degradation products (2-deg) indicated. (B) Reciprocal IP CITFA-7&#226;&#8364;&#8220;HA PTP&#226;&#8364;&#8220;CITFA-2 PF BF extracts. immunoblot analysis includes equal amounts starting material (Inp) supernatant (S) 5-fold enrichment precipitate (P). Detection RNA pol II transcription factor TFIIB served negative control. (C) Multiple sequence alignment trypanosomatid CITFA-7 <br /> orthologs T. brucei (accession numbers listed Table S1 supplemental material), T. congolense (Tco), T. vivax (Tv), T. cruzi (Tc), Leishmania <br /> major (Lm), L. infantum (Li), L. braziliensis (Lb). Positions 50% identity similarity shaded black gray, respectively. Overall <br /> identity/similarity values specified end sequence determined pairwise comparison T. brucei sequence using EMBOSS software <br /> program (http://www.ebi.ac.uk/emboss/align/) default settings. Dashes indicate sequence gaps. Numbers parentheses specify number residues significant sequence similarity. Asterisks mark 13 invariant proline positions. <br />  <br />  liquid chromatography-tandem mass spectrometry. resulting data            RESULTS analyzed using TurboSEQUEST program BioWorks 3.1                 Identification novel CITFA subunit. CITFA originally <br /> software package (Thermo Fisher Scientific). Coimmunoprecipitations              characterized tandem affinity purification tagged CITFA-2. <br />  IPs) carried previously published (18), BF extract <br />                                                                                  confirm nature complex, tandem affinity prepared recently described (21). Sucrose sedimentation  <br /> formed exactly previously procyclic extract (1). <br />                                                                                  purified conserved subunit, CITFA-6,      Immunofluorescence microscopy. BFs grown exponential phase               C-terminally fused composite PTP tag consisting tan- centrifuged 800 &#226;&#171;&#187; g 5 min, washed ice-cold phosphate-             dem protein (ProtA) domain, TEV protease cleavage site, buffered saline (PBS), fixed adding <span id='am-49' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span> final           protein C epitope (ProtC [31]). CITFA-1 -6 DYNLL1 <br /> centration 2% 10 min room temperature. Fixation termi-             recovered final eluate addition fainter bands <br /> nated adding <span id='am-25' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-26' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span> final concentration 125 mM 5 min.            35 kDa band &#226;&#172;&#402;17.5 kDa (Fig. 1A). Liquid <br /> Cells washed times PBS resuspended PBS          chromatography-tandem mass spectrometry revealed density 3 &#226;&#171;&#187; 107 cells/ml. Cells (5 &#226;&#171;&#187; 106) spotted silanized         larger bands specific degradation products CITFA-2, <br /> coverslips (Thermo Fisher Scientific) allowed settle 30 min,          17.5-kDa band identified new protein coverslips washed twice PBS 5 min                                                                                   termed CITFA-7 (accession number Tb927.7.2600). <br /> reduce spherical aberrations cells, coverslips placed 24 plate centrifuged 2 min 800 &#226;&#171;&#187; g (12). 10-min rehydra-             apparent size CITFA-7 range tion PBS, fixed cells permeabilized 0.1% NP-40-containing PBS         predicted mass 15.1 kDa, identification based exactly 5 min, washed twice PBS, blocked PBG (PBS containing           single, partially tryptic <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> match (data shown).  <br /> 0.5% bovine serum albumin [BSA] 0.2% cold water fish gelatin;                firm correct protein identified, generated PF Sigma), incubated consecutively primary secondary anti-             BF cell lines CITFA-2 -7 PTP HA tagged, <br /> bodies diluted PBG. Coverslips extensively washed PBS         respectively. Reciprocal coimmunoprecipitation  IP) assays antibody probing finally mounted slide VectaShield          extracts prepared cells unequivocally demon- <br /> (Vector lab). PTP-tagged proteins detected polyclonal anti-          strated interaction proteins (Fig. 1B). protein immune serum (Sigma) followed Alexa 594-conjugated               C terminus, CITFA-7&#226;&#8364;&#8482;s <span id='am-18' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> sequence highly  <br /> anti-rabbit secondary antibody (Invitrogen), HA-tagged CITFA-7 <br />                                                                                  served members Trypanosomatidae se- detected specified anti-HA antibody Alexa <br /> 488-conjugated anti-rat secondary antibody (Invitrogen). Immunofluo-             quence similarity proteins outside phylogenetic family. <br /> rescence assays procyclics described previously (15). Im-        Furthermore, CITFA-7 does contain sequence motifs ages acquired Zeiss AxioVert 200 microscope, colocaliza-           known function does contain 13 proline residues tions quantified using Zeiss Axiovision 4.6.3.0 colocalization          absolutely conserved trypanosomatid orthologs (Fig. 1C). <br /> software program.                                                                    CITFA-7 bona fide subunit CITFA. CITFA-7  <br />  <br /> December 2012 Volume 11 Number 12                                                                                                             ec.asm.org 1575 <br />  Nguyen et al. <br />  <br />  <br />  <br />  <br /> FIG 2 CITFA-7 bona fide subunit CITFA.   Schematic drawing CITFA-7 wild-type (WT) allele allele modifications cell line TbC7ee, exclusively expresses CITFA-7&#226;&#8364;&#8220;PTP untagged CITFA-7. CITFA-7 coding region, HYGR NEOR genes, PTP tag sequence depicted open boxes, striped boxes, black box, respectively. Gene flanks RNA processing signals indicated smaller gray boxes. (B) Immunoblot <br /> analysis cell lysates wild-type procyclic cells TbC7ee cells using newly generated polyclonal anti-CITFA-7 immune serum (&#226;?&#163; CITFA-7) commercially available (Roche), monoclonal anti-ProtC antibody (&#226;?&#163; ProtC). (C) Immunoblot monitoring CITFA-7&#226;&#8364;&#8220;PTP purification (note TEV <br /> protease digest reduced CITFA-7&#226;&#8364;&#8220;PTP CITFA-7&#226;&#8364;&#8220;P). Aliquots crude extract (Inp), flowthrough IgG affinity chromatography (FT-IgG), TEV <br /> protease elution (TEV), flowthrough anti-ProtC affinity chromatography (FT-ProtC), final eluate (Elu) separated 10% SDS- <br /> polyacrylamide gel, blotted, probed anti-ProtC antibody. blot reprobed immune sera directed CITFA-2 CITFA-6. Arrows <br /> point FT-IgG signals indicating codepletion subunits. (D) Protein analysis CITFA-7 TAP, analogous CITFA-6 TAP Fig. 1A. (E) Anti-ProtC <br /> immunoblot final eluate CITFA-7&#226;&#8364;&#8220;PTP purification. purified complex left untreated incubated alkaline phosphatase (AP)  calf intestine) AP phosphatase inhibitor cocktail (ph-inh). <br />  <br />  <br />  detected CITFA-2 tandem affinity purification (1)                   faint band just main band resolved, <br /> purified CITFA-6 substoichiometric amounts (Fig. 1A),                 suggesting CITFA-7 carries modification possible represented distinct factor interacted               phosphates. CITFA complex. explore possibility, tagged                      Monitoring PTP tandem affinity purification immuno- tandem affinity purified CITFA-7. generated PF cell               blotting showed CITFA-7&#226;&#8364;&#8220;PTP, reduced line, termed TbC7ee, exclusively expressed CITFA-7                  CITFA-7&#226;&#8364;&#8220;P TEV protease elution, efficiently purified C-terminal PTP tag. achieved integrating plasmid                 chromatography steps (Fig. 2C, upper panel). blot <br /> pCITFA-7-PTP-NEO CITFA-7 allele knocking                         shown Fig. 2C reprobed polyclonal anti-CITFA-2 remaining CITFA-7 wild-type allele hygromycin                   antibody (1), apparent CITFA-2 efficiently <br /> phosphotransferase (HYGR) marker (Fig. 2A). molecular                       codepleted extract (Fig. 2C, middle panel, arrow). tool, raised specific anti-CITFA-7 immune serum rats                 analyze finding extended CITFA subunits, <br /> confirmed TbC7ee cells did express wild-type, untagged                  raised specific polyclonal anti-CITFA-6 immune serum CITFA-7 (Fig. 2B). CITFA-7 encoded essential gene                 rats  Fig. S1 supplemental material). Reprobing blot <br />  , concluded tag did interfere                    antiserum showed codepletion CITFA-6 (Fig. <br /> CITFA-7 function. anti-ProtC anti-CITFA-7 antibodies                    2C, lower panel). follows, CITFA complex quantitatively <br /> detected wild-type tagged CITFA-7 distinct, nar-                associated tagged CITFA-7. Furthermore, protein profile <br /> rowly spaced bands. pattern primarily result multiple             CITFA-7 purification revealed CITFA sub- <br /> CITFA-7 phosphorylations, treatment alkaline phos-                    units copurified apparently stoichiometric amounts, phatase resolved larger bands lower band (Fig. 2E).                 major protein bands detectable (Fig. 2D). concluded <br />  <br />  <br /> 1576   ec.asm.org                                                                                                                                 Eukaryotic Cell <br />                                                                                                                                    Novel CITFA Subunit T. brucei <br />  <br />  <br />  <br />  <br /> FIG 3 CITFA-7 silencing lethal, leading specific loss RNA pol synthesized transcripts.   Growth curves clonal BF cell line absence (-Dox, <br /> blue) presence (&#226;&#171;&#185;Dox, red) doxycyline,  line, induces synthesis CITFA-7 dsRNA targets 3= UTR corresponding mRNA. lethal phenotype presence doxycycline rescued (Rescue &#226;&#171;&#185;Dox, green) fusing HA tag sequence RPA1 3= UTR endogenous <br /> CITFA-7 allele. schematic wild-type modified CITFA-7 mRNAs  scale), coding region depicted gray box wild-type UTRs black lines, capped spliced leader drawn cyan, poly  tail indicated AAA. modified mRNA, HA coding sequence modified 3= UTR drawn green. (B) Semiquantitative RT-PCR analysis CITFA-7, CITFA-2, CITFA-6 mRNA levels total RNA prepared cells CITFA-7 silenced specified periods. addition, mRNA abundance gene Tb927.7.2590, downstream, tandemly linked gene CITFA-7, TFIIB, independent control, determined. post ind., postinduction. (C) Anti-HA immunoblot analysis cell lysates prepared rescued <br /> cells induced doxycycline specified periods expressed CITFA-7&#226;&#8364;&#8220;HA RNAi-resistant mRNA. U2A= detection blot <br /> served loading control. (D) Analysis total RNA prepared CITFA-7-silenced cells. rRNA detected ethidium bromide staining, VSG mRNA mRNA spliceosomal protein SmD1 Northern blotting, SL RNA U2 snRNA primer extension analysis. <br />  <br />  <br />  CITFA-7 bona fide subunit CITFA complex.                       gene modification achieved targeted integration plas- <br /> appears tagging CITFA-2 lesser extent tagging                  mid CITFA-7 locus fused HA tag sequence CITFA-6 destabilized CITFA-7 binding CITFA complex,                         3= flank gene encoding largest RNA pol subunit RPA1 <br /> leading partial losses CITFA-7 corresponding purifi-                  CITFA-7 allele. shown Fig. 3C, abundance cations.                                                                           HA-tagged CITFA-7 did decrease doxycycline induc- <br />     CITFA-7 essential trypanosome viability RNA pol                     tion. concluded CITFA-7 essential mediated gene expression. address CITFA-7                       viability BFs. essentiality CITFA-7 appears extend essential component CITFA, silenced CITFA-7 expression                       PFs  corresponding knockdown life cycle BFs conditional, doxycycline-induced expression                           stage halted culture growth 3 days  Fig. S2 supple- <br /> CITFA-7 double-stranded RNA (dsRNA) (36), targeting 3=                         mental material). <br /> UTR CITFA-7 mRNA RNAi pathway. obtained                              analysis total RNA prepared CITFA-7-silenced single clonal cell line proliferation ceased 1 day            BFs showed RNA pol transcripts rRNA VSG mRNA doxycycline induction, noninduced cells prolifer-                   abundant doxycycline induction (Fig. 3D). ated logarithmically 3-day period experi-                    samples containing equal amounts RNA, compensa- <br /> ment (Fig. 3A). lethal phenotype appeared                         tory increases relative abundance observed RNA <br /> CITFA-7 silencing CITFA-7 mRNA level strongly                      pol II-synthesized SmD1 mRNA, encoding RNA splicing pro- <br /> reduced 24 h doxycycline induction mRNA abun-                        tein, SL RNA, RNA pol III-synthesized U2 <br /> dances gene downstream adjacent CITFA-7                       snRNA. corresponding results obtained loss CITFA-2, CITFA-6, TFIIB control affected (Fig.                    CITFA-2 (1) RNA pol subunit RPA31 (18), profile 3B). unequivocally demonstrate lethal effect spe-                  RNA abundances strongly indicated CITFA-7 essential cific CITFA-7 silencing opposed nonspecific target                  RNA pol transcription. <br /> siRNA effect, manipulated CITFA-7 allele &#226;&#8364;&#339;RNAi&#226;&#8364;?                          CITFA-7 colocalizes CITFA-2 RNA pol subunit <br /> cell line create RNAi-resistant CITFA-7 mRNA fully                     RPB6z nucleolus. far, localization CITFA sub- <br /> restored cell growth presence doxycycline (Fig. 3A).                 units investigated. facilitate analysis,  <br />  <br /> December 2012 Volume 11 Number 12                                                                                                                 ec.asm.org 1577 <br />  Nguyen et al. <br />  <br />  <br />  <br />  <br /> FIG 4 CITFA-7 colocalizations.   Colocalization PTP-RPB6z (red) CITFA-7&#226;&#8364;&#8220;HA (green). (B) Colocalization PTP&#226;&#8364;&#8220;CITFA-2 (red) CITFA- <br /> 7&#226;&#8364;&#8220;HA (green). White bars represent 10 &#226;?&#174;m. <br />                                                                             FIG 5 CITFA-7 essential RNA pol transcription. Cotranscriptions                                                                             VSG ES promoter template VSG-trm RNA pol II control template SLins19 <br />                                                                             extracts mock depleted (mock), immunodepleted (depl)                                                                             CITFA-2 CITFA-7, reconstituted buffer final eluates (elu) generated BF cell lines CITFA-7 C-ter-             PTP&#226;&#8364;&#8220;CITFA-2 CITFA-7&#226;&#8364;&#8220;PTP TAPs  panel). Anti-CITFA-2 (middle <br /> minally HA tagged RPB6z CITFA-2 N-termi-                  panel) anti-CITFA-7  panel) immunoblots depleted recon- <br /> nally PTP tagged. tags functional, exclusive           stituted extracts detecting endogenous, PTP-tagged, P-tagged  TEV <br /> expression tagged versions essential proteins did           protease digest) proteins shown. impair proliferation cultured trypanosomes (1, 18; study). 4=,6-diamidino-2-phenylindole (DAPI) stains nucleus, nucleolus recognized spherical structure        concentrated nucleolus ESB (Fig. 4B, second panel <br /> low DNA density (Fig. 4). expected, RNA pol subunit                set; data shown). <br /> RPB6z localized consistently nucleolus, cases               CITFA-7 depletion extract abolishes RNA Pol tran- accumulation nucleolar periphery (Fig. 4A),           scription vitro. verify essential transcription function observed (23). tagged CITFA-7 subunit exhibited                CITFA-7, depleted CITFA-7 extracts prepared similar distribution, colocalization CITFA-7 RPB6z               TbC7ee cells. Cotranscription VSG ES promoter template calculated approximately 50% protein  Fig.           VSG-trm RNA pol II-recruiting template SLins19, har- <br /> S3 supplemental material). partial overlap consistent        boring SL RNA gene promoter, showed depletion CITFA promoter-binding transcription initiation                CITFA-7 extract virtually abolished VSG-trm transcription <br /> factor associated transcribing RNA pol  ex-               affecting SLins19 transcription (Fig. 5, compare lanes 5 <br /> pected, colocalization CITFA-7 CITFA-2 (Fig. 4B) showed              6). result mirrored depletion CITFA-2 extract greater overlap. colocalization signal pro-         prepared cells exclusively expressed tagged CITFA-2 <br /> teins comprised 80% CITFA-2 signal 63%                    (lanes 1 2). Previously, reported RNA pol transcrip- <br /> CITFA-7 signal  Fig. S4), supporting conclusion                tion CITFA-2-depleted extract partially restored <br /> CITFA-7 bona fide subunit CITFA complex.                    addition CITFA complex purified tagged <br /> greater, noncolocalized CITFA-7 signal indicate            CITFA-2 (1);  result reproduced CITFA-2- <br /> CITFA complexes carry CITFA-2 subunit  .                      CITFA-7-depleted extracts (lanes 3 7). contrast, add- <br />     hands, detection ESB difficult, ob-            ing CITFA purified tagged CITFA-7 resulted served extranucleolar spots 10% cells (n &#226;&#171;&#189; 174).            restoration VSG-trm transcription (lanes 4 8). Detec- <br /> Interestingly, spots observed RPB6z                tion CITFA-2 CITFA-7 proteins depleted  <br /> CITFA subunit localizations. rare cases, CITFA-7 colocal-          stored extracts showed amounts proteins added ized RPB6z CITFA-2 outside nucleolus, raising            extract comparable respective protein levels mock- possibility transcription initiation factor CITFA           depleted extracts (Fig. 5, middle panel).  <br />  <br />  <br /> 1578   ec.asm.org                                                                                                                        Eukaryotic Cell <br />                                                                                                                      Novel CITFA Subunit T. brucei <br />  <br />  <br />  immunoblots revealed CITFA-2 depletion did codeplete CITFA-7 (lane 2) CITFA-7 depletion led efficient codepletion CITFA-2 (lane 6). major differ- <br /> ence CITFA-2 CITFA-7 purifications pres- <br /> ence CITFA-7 complex, results confirmed CITFA-7 essential component CITFA. <br />     increased functionality CITFA-7-purified complex apparent electrophoretic mobility shift assay. PTP&#226;&#8364;&#8220;CITFA-2-purified complex, harboring little <br /> CITFA-7, bound VSG ES promoter efficiently CITFA-7&#226;&#8364;&#8220;PTP-purified complex  Fig. S5 supplemental <br /> material). result indicate CITFA-7 function binding CITFA VSG ES promoter  Discussion). <br />     transcriptional function T. brucei CITFA-7 species <br /> specific. Despite fact multifunctional RNA pol sys- <br /> tem present trypanosomatids, T. cruzi, RNA pol exclusively transcribes RRNA copies eukaryotes, RNA pol CITFA subunits T. brucei conserved trypanosomatids. lack T. brucei- <br /> specific factor raises possibility T. <br /> brucei factors gained function evolution specific VSG ES procyclin gene expression. test possibility CITFA-7, introduced C-terminally HA-tagged T. cruzi <br /> CITFA-7 transgene T. brucei CITFA-7 RNAi cell line conditionally expressed dsRNA CITFA-7 3= UTR (Fig. 6A). contrast equivalently tagged, RNAi-resistant T. brucei <br /> CITFA-7&#226;&#8364;&#8220;HA (Fig. 3A), T. cruzi CITFA-7&#226;&#8364;&#8220;HA rescue <br /> trypanosome proliferation (Fig. 6A), expression heterologous transgene affected mRNA (Fig.            FIG 6 Expression T. cruzi CITFA-7-HA transgene did rescue trypano- <br /> 6B) protein (Fig. 6C) levels. RNA analysis BF cell        proliferation T. brucei CITFA-7 knockdown.   Culture growth <br />                                                                          BFs noninduced (-Dox) TbCITFA-7-silenced (&#226;&#171;&#185;Dox) cells ex- <br /> line showed abundances RNA pol tran-             press TcCITFA-7-HA. (B) Ethidium bromide staining rRNA semiquan- <br /> scripts, rRNA VSG mRNA, strongly reduced                  titative RT-PCR analysis listed mRNAs 0, 24, 72 h doxycycline <br /> TbCITFA-7 silencing (Fig. 6B), indicating TcCITFA-7&#226;&#8364;&#8220;HA              induction. Note TbCITFA-7 mRNA targeted knockdown generally unable function T. brucei background. Al-        (KD). (C) Immunoblot analysis cell lysates prepared nonin- surprising conserved transcription factor     duced cells cells induced 24 h. TbCITFA-7 TFIIB                                                                          detected polyclonal antibodies endogenous proteins, related organism genus dysfunctional, species        TcCITFA-7&#226;&#8364;&#8220;HA detected monoclonal anti-HA antibody. (D) Sed- <br /> specificity function hallmark RNA pol typi-      imentation extract ultracentrifugation 10 40% linear sucrose <br /> cally RNA pol II III systems. previously     gradient. Fractions 4 20, taken  analyzed im- <br /> shown closely related human               munoblotting. comparison, sedimentations TEV protease (29 kDa), <br />                                                                          Taq DNA polymerase (95 kDa), IgG (150 kDa, 6.6S), TRF4-SNAPc-TFIIA <br /> murine systems, factor binds ribosomal core pro-            transcription factor complex (TST) (&#226;&#172;&#402;230 kDa) (30), apoferritin (AP) (444 <br /> moter, termed selectivity factor 1 (SL1) humans TIF-IB         kDa, 17S), thyroglobin (TG) (660 kDa, 19S) analyzed parallel mouse, dysfunctional s (10).            gradients (arrowheads). <br />     final step, wanted specific defect TcCITFA-7 T. brucei. Localization HA-tagged protein indirect immunofluorescence   microscopy           CITFA-2. notion consistent finding centrated nucleolus, expected CITFA subunit          substantial CITFA-7 signal did colocalize CITFA-2 Fig. S6 supplemental material). possibility      analysis (Fig. 4B; Fig. S4). contrast T. brucei <br /> TcCITFA-7 did assemble CITFA complex. test            CITFA subunits, TcCITFA-7&#226;&#8364;&#8220;HA spread  sedimented transcription extract noninduced BF             gradient exhibiting strong sedimentation peak <br /> cells linear sucrose gradient. TbCITFA-7 detected        fractions 13 14. sedimentation pattern showed gradient unbound protein             TcCITFA-7&#226;&#8364;&#8220;HA efficiently incorporated CITFA complex, concentrated peak fraction 13 (Fig.         CITFA complex. Formation partial CITFA complexes  <br /> 6D, middle panel). sedimentation profile TbCITFA-2             cruitment heat shock proteins incomplete protein  gradient closely resembled previous sedimentation             semblies account signal spread. alternative expla- <br /> analysis (1), unbound protein peaking fraction 8             nation finding self-aggregation TcCITFA-7&#226;&#8364;&#8220;HA CITFA-associated protein peaking fraction 14 (Fig. 6D,            extract.  likely, self-aggregation typ- <br /> panel). Interestingly, finding revealed shift sedimen-   ically observed protein overexpression, avoided tation fraction gradient CITFA-7 peak,          integrating TcCITFA-7-HA transgene endogenous suggested distinct CITFA complexes           CITFA-7 locus. Furthermore, formation partial CITFA extract: inactive complex CITFA-2 active           complexes explain correct nucleolar localization  <br />  <br /> December 2012 Volume 11 Number 12                                                                                                   ec.asm.org 1579 <br />  Nguyen et al. <br />  <br />  <br />  <br /> TcCITFA-7&#226;&#8364;&#8220;HA. sum, sedimentation analysis suggested               equivalently generated CITFA-7 antibody, raised inefficient formation fully assembled CITFA complex            recombinant length protein  Materials reason TcCITFA-7&#226;&#8364;&#8220;HA rescue silencing            Methods; data shown). exclude endogenous CITFA-7. addition, TcCITFA-7 appeared                possibility polyclonal antibody did recognize  <br /> carry fewer posttranslational modifications TbCITFA-7 (Fig.         teraction domain CITFA-7, likely interpretation result <br /> 2B C, 3C, 6C D), contributed           CITFA-7&#226;&#8364;&#8482;s essential interactions occur complex <br /> failure heterologous protein function T. brucei        inaccessible antibody blockage. true, CITFA-7&#226;&#8364;&#8482;s                                                                   main function correct assembly CITFA, notion <br />                                                                          supported finding CITFA-7, contrast DISCUSSION                                                               CITFA-2, CITFA complexes extract. characterized seventh nondynein subunit                    Finally, confirmed trypanosomes RNA pol CITFA termed  accordingly, CITFA-7. CITFA complex             prone rapid divergence closely related spe- essential transcription ribosomal, VSG ES,           cies. Expression heterologous CITFA-7 gene T. cruzi <br /> procyclin gene promoters  shown VSG ES promoter,          rescue knockdown endogenous CITFA-7 <br /> binds DNA sequence-dependent fashion (1). CITFA-7            gene. data strongly indicated TcCITFA-7 unable discovered previously tagging tandem affinity            assemble functional CITFA complex showed  purification CITFA-2 copurified minute amounts                comparison TbCITFA-7, carried fewer posttranslational <br /> CITFA-7 (1).  data clearly demonstrated            modifications. Interestingly, defect murine CITFA-7 bona fide CITFA subunit essential proper        human RNA pol systems attributed rapid divergence <br /> function protein complex. showed CITFA-7             rDNA promoter sequence DNA-protein interaction tagged depleted extract, trackable             core factors (10), results suggest Tc- <br /> CITFA subunits, 2 6, codepleted extracts, indicat-         CITFA-7 unable undergo correct complex assembly ing subunits quantitatively bound CITFA-7.        obtain potentially important posttranslational modifications. <br /> Consistent CITFA-7 CITFA subunit, knockdown             dramatic defects trypanosomes likely  <br /> specifically affected abundance RNA pol transcripts         sequence changes amino acid sequences trypano- <br /> rRNA VSG mRNA.  vitro assays unequivo-               somatid transcription factors diverged rapidly <br /> cally demonstrated importance CITFA-7 CITFA func-              example, trypanosomatid RNA processing factors (6, <br /> tion: CITFA-2-purified complex, comprised CITFA                7).  approach reconstituting CITFA function <br /> subunits CITFA-7, efficient CITFA-7-           vivo heterologous subunits subunit domains purified complex restoring RNA pol transcription CITFA-          potential uncover VSG procyclin gene-specific functions depleted extracts binding VSG ES promoter gel shifts.      essential RNA pol transcription factor complex. <br />     specific function CITFA-7? know. <br /> CITFA inherently difficult analyze sub-        ACKNOWLEDGMENTS <br /> units, lack sequence similarity mammalian yeast RNA        grateful Barbara Burleigh (Harvard University) providing T. <br /> pol transcription factors, lack known sequence motifs.      cruzi genomic DNA Sung Hee Park help generating Mechanistically, RNA pol transcription initiation conserved         anti-CITFA-6 immune serum. yeast mammals (28). involves formation            project funded NIH grant R01 AI059377  G.). <br /> RNA pol transcription preinitiation complex (PIC),                                                                          REFERENCES <br /> core promoter-binding factor recruits RNA pol transcrip- <br />                                                                           1. Brandenburg J, et al. 2007. Multifunctional class transcription tion initiation site interaction enzyme-bound               Trypanosoma brucei depends novel protein complex. EMBO J. 26: <br /> factor RRN3. yeast, factor termed core factor,            4856 &#226;&#8364;&#8220; 4866. <br /> humans termed SL1. addition, distinct factor            2. Brun R, Blum J. 2012. Human African trypanosomiasis. Infect. Dis. Clin. <br /> binds upstream promoter element activates ribosomal                North  26:261&#226;&#8364;&#8220;273. <br />                                                                           3. Chaves  et al. 1998. Subnuclear localization active variant surface <br /> DNA (rDNA) transcription stabilizing PIC formation.                                                                              glycoprotein gene expression site Trypanosoma brucei. Proc. Natl. Acad. <br /> VSG ES promoter extends 67 bp upstream transcrip-                Sci. U. S.  95:12328 &#226;&#8364;&#8220;12333. <br /> tion initiation site consists short sequence elements          4. Ehlers B, Czichos J, Overath P. 1987. RNA turnover Trypanosoma <br /> (24, 34). elements required efficient binding          brucei. Mol. Cell. Biol. 7:1242&#226;&#8364;&#8220;1249. <br /> purified CITFA (1) close initiation site,    5. Grummt  Skinner JA. 1985. Efficient transcription protein-coding <br />                                                                              gene RNA polymerase promoter transfected cells. Proc. Natl. <br /> CITFA appears functional equivalent yeast core              Acad. Sci. U. S.  82:722&#226;&#8364;&#8220;726. <br /> factor human SL1. According known functions           6. G&#195;&#188;nzl  2010. pre-mRNA splicing machinery trypanosomes: <br /> factors, CITFA-7 directly involved DNA binding                  complex simplified? Eukaryot. Cell 9:1159 &#226;&#8364;&#8220;1170. <br /> interact RRN3-like factor.  think          7. G&#195;&#188;nzl  2012. RNA polymerases transcription factors trypano- <br />                                                                              somes, p 1&#226;&#8364;&#8220;27. Bindereif (ed), RNA metabolism trypanosomes, vol case. Previously, able inhibit transcrip- <br />                                                                              28. Springer Press, Berlin, Germany. <br /> tional function RNA pol II transcription factor TFIIB           8. G&#195;&#188;nzl  Bindereif  Ullu E, Tschudi C. 2000. Determinants cap <br /> adding polyclonal anti-TFIIB antibody extract prior            trimethylation U2 small nuclear RNA conserved vitro transcription reaction (30). RNA Pol II PIC formation            Trypanosoma brucei higher eukaryotic organisms. Nucleic Acids Res. <br /> requires TFIIB bind promoter, TATA-binding pro-                28:3702&#226;&#8364;&#8220;3709. <br />                                                                           9. G&#195;&#188;nzl  Vanhamme L, Myler PJ. 2007. Transcription trypanosomes: <br /> tein, polymerase, appears antibody able           different means end, p 177&#226;&#8364;&#8220;208. Barry JD, Mottram JC, Mc- <br /> interfere interactions. function CITFA sim-            Culloch R, Acosta-Serrano (ed), Trypanosomes genome. <br /> ilarly inhibited antibody CITFA-2 (1)           Horizon Press, Pittsburgh, PA. <br />  <br />  <br />  <br /> 1580   ec.asm.org                                                                                                                         Eukaryotic Cell <br />                                                                                                                                   Novel CITFA Subunit T. brucei <br />  <br />  <br />  <br /> 10. Heix J, Zomerdijk JC, Ravanpay  Tjian R, Grummt  1997. Cloning              24. Pham VP, Qi CC, Gottesdiener KM. 1996. detailed mutational analysis <br />     murine RNA polymerase specific TAF factors: conserved interactions             VSG gene expression site promoter. Mol. Biochem. Parasitol. 75: <br />     subunits species-specific transcription initiation factor        241&#226;&#8364;&#8220;254. <br />     TIF-IB/SL1. Proc. Natl. Acad. Sci. U. S.  94:1733&#226;&#8364;&#8220;1738.                       25. Roditi  Clayton C. 1999. unambiguous nomenclature major <br /> 11. Hertz-Fowler C, et al. 2008. Telomeric expression sites highly              surface glycoproteins procyclic form Trypanosoma brucei. Mol. <br />     served Trypanosoma brucei. PLoS 3:e3527. doi:10.1371/                        Biochem. Parasitol. 103:99 &#226;&#8364;&#8220;100. <br />     journal.pone.0003527.                                                           26. Rudenko G, Chung HM, Pham VP, Van der Ploeg LH. 1991. RNA <br /> 12. Kawahara T, et al. 2008. essential MYST-family proteins display                 polymerase mediate expression CAT neo protein-coding <br />     distinct roles histone H4K10 acetylation telomeric silencing try-         genes Trypanosoma brucei. EMBO J. 10:3387&#226;&#8364;&#8220;3397. <br />     panosomes. Mol. Microbiol. 69:1054 &#226;&#8364;&#8220;1068.                                       27. Ruepp S, et al. 1997. Survival Trypanosoma brucei tsetse fly 13. Kelly S, Wickstead B, Gull K. 2005. silico analysis trypanoso-             enhanced expression specific forms procyclin. J. Cell Biol. <br />     matid RNA polymerases: insights unusual transcription.                   137:1369 &#226;&#8364;&#8220;1379. <br />     Biochem. Soc. Trans. 33:1435&#226;&#8364;&#8220;1437.                                              28. Russell J, Zomerdijk JC. 2005. RNA-polymerase directed rDNA tran- <br /> 14. Laufer G, Schaaf G, Bollg&#195;&#182;nn S, G&#195;&#188;nzl  1999. vitro analysis                 scription, life works. Trends Biochem. Sci. 30:87&#226;&#8364;&#8220;96. <br />     alpha-amanitin-resistant transcription rRNA, procyclic acidic          29. Schimanski B, Brandenburg J, Nguyen TN, Caimano MJ, G&#195;&#188;nzl  <br />     repetitive protein, variant surface glycoprotein gene promoters              2006. TFIIB-like protein indispensable spliced leader RNA gene <br />     Trypanosoma brucei. Mol. Cell. Biol. 19:5466 &#226;&#8364;&#8220;5473.                                 transcription Trypanosoma brucei. Nucleic Acids Res. 34:1676 &#226;&#8364;&#8220;1684. <br /> 15. Luz Ambrosio D, et al. 2009. Spliceosomal proteomics Trypanosoma             30. Schimanski B, Nguyen TN, G&#195;&#188;nzl  2005. Characterization multi- <br />     brucei reveal new RNA splicing factors. Eukaryot. Cell 8:990 &#226;&#8364;&#8220;1000.                 subunit transcription factor complex essential spliced-leader RNA <br /> 16. Marcello L, Barry JD. 2007. Analysis VSG gene silent archive                                                                                         gene transcription Trypanosoma brucei. Mol. Cell. Biol. 25:7303&#226;&#8364;&#8220;7313. <br />     Trypanosoma brucei reveals mosaic gene expression prominent                                                                                     31. Schimanski B, Nguyen TN, G&#195;&#188;nzl  2005. Highly efficient tandem <br />     antigenic variation favored archive substructure. Genome Res. <br />                                                                                         affinity purification trypanosome protein complexes based nove <br /> </body></html>