<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                               INVESTIGATION <br />  <br />  <br />  <br />  <br />                             Integration New Genes Cellular Networks, <br />                                              Structural Maturation <br />                                                                                                                                  Gy&#195;&#182;rgy Abrus&#195;&#161;n1 <br />                        Synthetic Systems Biology Unit, Institute Biochemistry, Biological Research Centre Hungarian Academy Sciences, <br />                                                                                                                               Szeged H-6701, Hungary <br />  <br />  <br />  <br />  <br /> ABSTRACT recently discovered new genes originate novo noncoding DNA, biological traits <br /> including expression sequence composition form continuum noncoding sequences conserved genes. article, using <br /> yeast genes test integration new genes cellular networks structural maturation shows continuum analyzing changes gene age. 1) number regulatory, protein&#226;&#8364;&#8220;protein, genetic interactions increases <br /> continuously gene age, different rates. New regulatory interactions emerge rapidly million years, number protein&#226;&#8364;&#8220;protein genetic interactions increases slowly, rate 2&#226;&#8364;&#8220;2.25 3 1028/year 4.8 3 1028/year, <br /> respectively. 2) Gene essentiality evolves relatively quickly: youngest essential genes appear proto-genes  14 old. 3) contrast interactions, secondary structure proteins robustness mutations indicate new genes face bottle- <br /> neck evolution: proto-genes characterized high b-strand content, high aggregation propensity, low robustness mutations, conserved genes characterized lower strand content higher stability, likely higher <br /> probability gene loss young genes accumulation neutral mutations. <br />  <br />  <br />  <br /> T   established view new genes evolve primarily <br />     duplications recombination, e., reorganiza- <br /> tion existing domains genes (Long et al. 2003; <br />                                                                                       close relatives sequenced.  fact                                                                                       detected model organisms indi- <br />                                                                                       cates emergence exceptionally rare, Kaessmann 2010). Recent studies,  highlighted                            ongoing process   genomes. <br /> genes emerge noncoding DNA: novo emer-                                  novo origination genes questioned <br /> gence genes demonstrated number cases                             anymore, little known frequency sub- Drosophila (Levine et al. 2006; Begun et al. 2007; Zhou                            sequent fate genes genome, e., et al. 2008), humans (Toll-Riera et al. 2009; Knowles                             subject turnover, rapidly new protein&#226;&#8364;&#8220;protein <br /> Mclysaght 2009; Wu et al. 2011; Xie et al. 2012), rodents                             interactions formed (Lynch 2012) integrated (Heinen et al. 2009; Murphy Mclysaght 2012), yeast                                regulatory networks (Ding et al. 2010; Chen et al. 2012), <br /> (Cai et al. 2008; Carvunis et al. 2012), viruses (Rancurel                        structural changes affect novel pro- <br /> et al. 2009; Sabath et al. 2012). orphan genes&#226;&#8364;&#8221;genes                            teins. Real proteins random structures; e., homologs common genomes (Tautz                                     characterized distinct biophysical properties like <br /> Domazet-Lo 2011), proving gene emerged novo                              stability aggregation propensity (DePristo et al. 2005; straightforward, presence nonfunctional                            Monsellier Chiti 2007),  quite surprisingly, <br /> parent sequence demonstrated, presence                           basic features, like presence secondary convincingly demonstrated species                           structural elements  helices b-strands),                                                                                       formed random sequences (Schaefer et al. 2010). Ge- <br /> Copyright &#194;&#169; 2013 Genetics Society America <br /> doi: 10.1534/genetics.113.152256                                                      nome-wide studies transcription species indi- <br /> Manuscript received April 15, 2013; accepted publication August 27, 2013;         cate large fraction genome, including published Early Online September 20, 2013. <br /> Supporting information available online http://www.genetics.org/lookup/suppl/   noncoding  transcribed species (Kapranov <br /> doi:10.1534/genetics.113.152256/-/DC1. <br /> 1 <br />                                                                                       et al. 2007; Nagalakshmi et al. 2008; Xu et al. 2009),  Address correspondence: Synthetic Systems Biology Unit, Institute  Biochemistry, Biological Research Centre Hungarian Academy Sciences, <br />                                                                                       noncoding transcripts associated ribo- <br />  Temesv&#195;&#161;ry krt. 62, Szeged, H-6701, Hungary. E-mail: abrusan@brc.hu                   somes occasionally translated (Wilson Masel 2011). <br />  <br />  <br />  <br />                                                                                                 Genetics, Vol. 195, 1407&#226;&#8364;&#8220;1417 December 2013     1407 <br />  recent large-scale study yeast demonstrated exis- <br /> tence continuous change level expression, <br /> selective constraints, codon adaptation index recently <br /> emerged &#226;&#8364;&#339;proto-genes&#226;&#8364;? highly conserved, ancient genes <br /> (Carvunis et al. 2012), suggested genes placed continuum nongenic sequences conserved genes, novo emergence genes common emer- <br /> gence classic duplication&#226;&#8364;&#8220;divergence mechanism. supported phylogenetic pattern gene emer- <br /> gence vertebrates (Neme Tautz 2013), indicating young genes typically short length, number exons, domains change gradually time. <br />    study goals. &#239;&#172;?rst examine broadly possible fate new translatable <br /> open reading frames (ORFs) emergence, e., test rapidly integrated regulatory, protein&#226;&#8364;&#8220; <br /> protein, genetic interaction networks, stable genome, characteristic <br /> sequence structural changes process func-         Figure 1 schematic phylogenetic tree fungal species conserva- <br /> tionalization. second test observed            tion yeast genes (modi&#239;&#172;?ed Carvunis et al. 2012). main bi- <br /> patterns support theory proposed Carvunis                furcation events Saccharomyces lineage numbered, 0 et al. (2012), nongenic sequences               10, age estimates obtained using TimeTree (Hedges et al. <br />                                                                      2006), S. castellii, split Saccharomyces lineage <br /> characterized genes form continuum sharp <br />                                                                      100&#226;&#8364;&#8220;150 MYA, genome duplication yeasts (Cliften <br /> boundaries exist                                        et al. 2006). Yeast genes classi&#239;&#172;?ed according conservation <br />                                                                      level, corresponds phylogenetic spread orthologs;                                                                      example, yeast gene conservation level 5 means ortho- <br /> Materials Methods                                                logs S. castellii fungal species split earlier                                                                      Saccharomyces lineage, conservation level 7 means yeast <br /> Categorization yeast genes age groups                        gene orthologs Debaryomyces hansenii C. albicans.  <br />                                                                      servation level 0 marks putative ORFs identi&#239;&#172;?ed Carvunis et al. <br /> Similarly Carvunis et al. (2012), genes Saccharomyces          (2012), excluding sequences shorter 50 aa, conservation level <br /> cerevisiae divided 11 groups according            1 indicates genes annotated Saccharomyces Genome Consortium, <br /> evolutionary conservation fungi. Conservation level           having orthologs species. <br /> 0 corresponds unannotated genes identi&#239;&#172;?ed Carvunis et al. (2012) longer 50 amino acids.           org/regev/orthogroups). orthology data set Genes conservation level 1 genes anno-            used determine gene losses S. paradoxus S. mika- <br /> tated genes Saccharomyces Genome Database                  tae  Results). <br /> (Cherry et al. 2012) orthologs species,            age estimates main branching events higher conservation levels indicate presence              phylogeny fungi (Figure 1) obtained using TimeTree <br /> orthologs fungi  Figure 1); e., conservation          (Hedges et al. 2006), S. castellii, split level N means orthologs gene present             Saccharomyces lineage 100&#226;&#8364;&#8220;150 MYA, species branch rooted Nth bifurcation           genome duplication yeasts (Cliften et al. 2006). nodes <br /> (Figure 1), species split earlier           age estimates estimates unavailable <br /> Saccharomyces lineage. Following Carvunis et al. (2012),           con&#239;&#172;&#8218;ict phylogeny species. <br /> use term proto-genes genes present                                                                      Data sources <br /> S. cerevisiae closest relatives (conservation levels 0&#226;&#8364;&#8220;4). <br />     classi&#239;&#172;?cation genes proto-genes (conservation        yeast genome protein sequences downloaded <br /> levels 0&#226;&#8364;&#8220;4) taken directly Carvunis et al. (2012);          Saccharomyces Genome Database (Cherry et al. sequences proto-genes conservation level 0               2012); recent duplications removed protein <br />  absent Saccharomyces Genome Database)              data set clustering 80% <span id='am-1' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span> similarity obtained using genomic coordinates provided              uclust (Edgar 2010). study, using R56 (20070406) release                 &#239;&#172;?tness effect yeast genes  e., effect single- <br /> genome. sequences minimum length 50 amino             gene deletion growth) obtained Deutschbauer <br /> acids used analysis. Conserved genes clas-          et al. (2005). Yeast protein&#226;&#8364;&#8220;protein interactions genetic <br /> si&#239;&#172;?ed conservation groups (Figure 1) Carvunis             interactions downloaded BioGRID (v. 3.1.91) <br /> et al. (2012) using fungal gene orthology data set pro-          database (Stark et al. 2011); regulatory interactions vided Wapinski et al. (2007; http://www.broadinstitute.           downloaded YEASTRACT database (Abdulrehman <br />  <br />  <br />  <br /> 1408     G. Abrus&#195;&#161;n <br />                                                                               et al. 2011). strength genetic interactions (genetic <br />                                                                              interaction score, e) obtained Costanzo et al. <br />                                                                              (2010); largest interaction data set &#226;&#8364;&#339;lenient <br />                                                                              cutoff&#226;&#8364;? (P , 0.05) used analysis,                                                                              includes weak interactions. gene charac- <br />                                                                              terized average |e| values signi&#239;&#172;                                                                              genetic interactions, calculated separately                                                                              proto-genes, conserved genes. number strength <br />                                                                              protein&#226;&#8364;&#8220;protein, genetic, regulatory interactions                                                                              gene determined Perl scripts, developed                                                                              house.  supporting information, Table S1.)                                                                              low expression low conservation, BioGRID database <br />                                                                              probably signi&#239;&#172;?cantly biased proto-genes                                                                              included genes analysis interac- <br />                                                                              tions databases, majority genes                                                                              conservation group reported interactions                                                                               case protein&#226;&#8364;&#8220;protein genetic interactions  <br />                                                                              servation levels 1&#226;&#8364;&#8220;3). <br />                                                                                  Saccharomyces cerevisiae RNAi absent, &#239;&#172;?t- <br />                                                                              ness effect individual genes, genetic interactions,                                                                              epistasis genes determined deletion                                                                              genes  Deutschbauer et al. 2005) opposed RNAi <br />                                                                              knockdown used mutlicellular organisms (Dixon <br />                                                                              et al. 2009).  means case  <br />                                                                              lapping genes, deletions target gene,                                                                              &#239;&#172;?tness measurements estimates strength                                                                              genetic interactions accurate. consequence, anal- <br />                                                                              yses involving &#239;&#172;?tness, genetic interactions, epistasis, <br />                                                                              overlapping genes excluded data set. <br />                                                                              Determination secondary structure mutagenesis <br />                                                                              proteins <br />                                                                              structural analysis, addition sequences                                                                              conservation levels 0&#226;&#8364;&#8220;10 used amino acid sequences                                                                              generated randomly, length amino <br />                                                                              acid frequencies youngest proto-genes conserva- <br />                                                                              tion levels 0&#226;&#8364;&#8220;1. secondary structures sequences <br />                                                                              determined PSSpred (http://zhanglab.ccmb.med. <br />                                                                              umich.edu/PSSpred/), aggregation propensity Tango <br />                                                                              (Fernandez-Escamilla et al. 2004)  Table S1). pro- <br />                                                                              tein sequences different conservation levels                                                                              different numbers protein homologs, secondary struc- <br />                                                                              ture prediction includes homology search, avoid biases, <br />                                                                              PSSpred run use homology search <br />                                                                              step (psi-blast), reduces accuracy secondary <br />                                                                              structure prediction 81 69%. <br />                                                                                 mutagenesis yeast proteins based                                                                              Schaefer et al. (2010) performed follows                                                                               Figure 6). sequence mutated gradually 70 <br />                                                                              steps, step 1% residues changed                                                                              sequence. protein sequences evolve entirely <br /> Figure 2 Integration new genes regulatory networks. Proto-genes      randomly, simulate realistic mutations, new residues acquire regulatory interactions rapidly; genes conservation <br /> level 1 regulated thousands genes  , (B) regulated transcription factors (B), rapidly gain regulatory motifs <br /> (feed-forward loops) (C). difference proto-genes         million-year-old genes (P . 0.05 conservation levels 5,6,8,10, <br /> served genes largely disappears conservation level 4, representing  14    ANOVA, Bonferroni post hoc tests). <br />  <br />  <br />  <br />                                                                                                 Dynamics Protogene Functionalization   1409 <br />  introduced context speci&#239;&#172;?c manner, using csbuild <br /> tool CS-BLAST suite (Biegert S&#195;&#182;ding 2009). step, secondary structure mutated sequence determined, fraction residues similar <br /> secondary structure original secondary structure (Q3 <br /> value) calculated. yeast protein sequence, mutagenesis repeated &#239;&#172;?ve times independently, average &#239;&#172;?ve replicates used analyses. save computing time, genes conservation level 10, 10% sequences mutated  cho- <br /> sen randomly). <br />  <br /> Results <br /> Rapid emergence regulatory interactions functionalization new gene means  addition evolving translatable open reading frame, needs        Figure 3 percentage essential genes genes different transcriptionally active, e., acquire promoter     conservation level; proportions indicated bars. youn- <br />                                                                   gest essential genes overlap older genes (YEL035C, regulated genes interacts                                                                   YPL124W) appear conservation level 4. (Note proto-genes genes expressed life stages         overlap conserved genes, essentiality present  <br /> environmental conditions. Studies novel retrogenes indi-       servation level 1;  cases &#239;&#172;?tness effect  <br /> cate regulatory elements new genes acquired        dependent overlapping conserved gene.) nearby genes distant locations <br /> (Kaessmann et al. 2009; Kaessmann 2010). Using YEAS-          type interactions components, FFLs <br /> TRACT database (Abdulrehman et al. 2011) analyzed           speed delay response target genes (Mangan <br /> integration proto-genes yeast regulatory network      Alon 2003). identi&#239;&#172;?ed FFL-like motifs net- quantifying number genes share            work YEASTRACT TFs tested number transcription factors (TFs), regulated, num-      FFLs regulating gene changes gene age. ber TFs regulating genes, emergence sim-      high connectivity network overlaps ple network motif, feed-forward loop. results        motifs, large number FFLs identi&#239;&#172;?ed YEASTRACT. integration genes regulatory network          similar, increasing pattern happens gradually, quickly; young proto-           number TFs; e., conserved genes regulated genes conservation levels 1&#226;&#8364;&#8220;3 regulated          proportionally FFLs proto-genes (P , 0.05, signi&#239;&#172;?cantly smaller number genes genes            ANOVA, Bonferroni post hoc tests, Figure 2C), indicating <br /> conservation level 5&#226;&#8364;&#8220;10 (P , 0.05 comparisons,            gradual rapid acquisition regulatory motifs ANOVA, Bonferroni post hoc tests, Figure 2A), genes        enable complex regulatory behaviors. youngest group (conservation level 1)  <br />                                                                   Essential genes present proto-genes <br /> regulated thousands genes, oldest <br /> proto-genes conservation level 4 ( 14 milion years           Recent &#239;&#172;?ndings Drosophila indicate essential genes <br /> old) signi&#239;&#172;?cantly different older groups         e., genes deletion mutants knockouts (conservation levels 5,6, 8, 10, P . 0.05, ANOVA, Bonferroni      viable) evolve rapidly (Chen et al. 2010) post hoc tests). comparable pattern visible number   common ancient genes. Using &#239;&#172;?t- transcription factors regulating genes (Figure 2B)     ness data set provided Deutschbauer et al. (2005) tested <br /> proto-genes conservation level 1&#226;&#8364;&#8220;3 regulated sig-     time necessary emergence essential <br /> ni&#239;&#172;?cantly transcription factors conserved genes,        genes yeast. results youngest essential proto-genes conservation level 4 (P , 0.05,          genes (YEL035C, YPL124W) present proto- <br /> ANOVA, Bonferroni post hoc tests);            genes conservation level 4, homologs S. baya- <br /> youngest group (conservation level 1) regulated     nus (Figure 3), split S. cerevisiae lineage  14 <br /> transcription factors.                                            MYA.  essentiality evolve faster:  <br />    Regulatory networks characterized signi&#239;&#172;?cantly        letion proto-genes  7 genes overrepresented patterns TFs, termed network motifs,           conservation level 1) lethal phenotypes,   capable performing distinct regulatory func-          likely overlap conserved genes. <br /> tions (Lee et al. 2002; Odom et al. 2006).        Determining independent &#239;&#172;?tness effect require <br /> important motifs feed-forward loop (FFL),       &#239;&#172;?tness estimates independent effect consists genes: TFs, regulates      genes overlap  e.g., obtained RNAi introduced  regulating target gene. Depending        yeasts (Drinnenberg et al. 2009). <br />  <br />  <br />  <br /> 1410      G. Abrus&#195;&#161;n <br />                                                                             Protein secondary structure aggregation propensity <br />                                                                            analysis secondary structure motifs genes                                                                            different conservation level shows surprising pattern.                                                                            abundance helices in&#239;&#172;&#8218;uenced age                                                                            proteins; helices  40% sequence, regardless <br />                                                                            conservation level, random amino acid sequences <br />                                                                            (Figure 4A, P . 0.05 comparisons proto-genes <br />                                                                            conserved genes, Bonferroni post hoc tests, ANOVA).                                                                            contrast, predicted b-sheets highest                                                                            proto-genes random sequences ( 20%) shows  <br />                                                                            cline  10% increasing level conservation (Figure <br />                                                                            4B, P ,, 0.001 comparisons proto-genes <br />                                                                            conserved genes, Bonferroni post hoc tests, ANOVA). <br />                                                                            suggests b-strands evolutionarily unsta- <br />                                                                            ble, gradually disappear protein sequence accu- <br />                                                                            mulates neutral mutations, declining abundances <br />                                                                            caused natural selection. selective force                                                                            aggregation propensity (Monsellier Chiti 2007),                                                                            depends largely presence b-strands (Jahn et al. <br />                                                                            2010). Predicted aggregation propensity shows                                                                            stronger pattern b-strands; average aggregation <br />                                                                            propensity random sequences proto-genes                                                                            times higher conserved genes (Figure 4C, <br />                                                                            P ,, 0.001, comparisons proto-genes                                                                            conserved genes, ANOVA). <br />                                                                            Turnover proto-genes <br />  <br />                                                                            Functional noncoding sequences short lived subject <br />                                                                            rapid turnover mammalian genomes (Ponting et al. <br />                                                                            2011). natural selection cause b-strand loss,                                                                            manifest itself gradual loss strands muta- <br />                                                                            tions, loss entire genes high strand content.                                                                            examined importance mechanism testing <br />                                                                            probability gene loss depends age                                                                            gene. used genes conservation levels 4&#226;&#8364;&#8220;10, using <br />                                                                            orthogroup data set provided Wapinski et al. (2007)                                                                            quanti&#239;&#172;?ed gene losses youngest Saccharomyces spe- <br />                                                                            cies, S. paradoxus S. mikatae (conservation levels 2&#226;&#8364;&#8220;3). <br />                                                                            results relatively young genes conserva- <br />                                                                            tion levels 4&#226;&#8364;&#8220;5 lost signi&#239;&#172;?cantly higher frequencies                                                                            species conserved genes (40% lost                                                                            opposed 7&#226;&#8364;&#8220;10%, P , 0.05 comparisons, x-square <br />                                                                            tests, Figure 5), corresponds pattern                                                                            b-strand loss indicates turnover proto-genes <br />                                                                            likely contribute observed structural change. <br />                                                                            Robustness secondary structures mutations <br />                                                                            test effect mutation accumulation second- <br />                                                                            ary structure robustness yeast proteins, performed <br />                                                                            silico mutagenesis studies. followed procedure                                                                            based Schaefer et al. (2010); mutations introduced <br /> Figure 4 Changes secondary structure aggregation propensity <br />                                                                            gradually protein sequences, examined gene age. helices does depend pro- <br /> tein age  , b-strands declines signi&#239;&#172;?cantly  <br /> servation levels 4 6 (B). Aggregation propensity, partly      b-strands, random amino acid sequences proto-genes caused presence b-strands, shows stronger trend     prone aggregation conserved genes (C). <br />  <br />  <br />  <br />                                                                                               Dynamics Protogene Functionalization   1411 <br />                                                                              structure (Zhang et al. 2012), demonstrated <br />                                                                             interacting surfaces frequently characterized <br />                                                                             similar arrangement secondary structure elements <br />                                                                             (Tuncbag et al. 2008; Zhang et al. 2010; Gao Skolnick <br />                                                                             2010). size age protein complexes highly <br />                                                                             variable, currently consensus rate <br />                                                                             evolution (Qian et al. 2011; Lewis et al. 2012). pre- <br />                                                                             vious study Capra et al. (2010) demonstrated                                                                             yeast genes younger genome duplication <br />                                                                             integrated protein&#226;&#8364;&#8220;protein interaction networks <br />                                                                             older genes. tested age (conservation <br />                                                                             level) yeast proteins in&#239;&#172;&#8218;uences number phys- <br />                                                                             ical interactions mirrors changes sec- <br />                                                                             ondary structure proteins. Unlike secondary structure <br />                                                                             elements, protein&#226;&#8364;&#8220;protein interactions increase gradually <br />                                                                             age proteins (Figure 8A) appears Figure 5 probability gene loss S. paradoxus S. mikatae.   monotonous process; using conservation levels 6 proteins emerged S. cerevisiae&#226;&#8364;&#8220;S. mikatae split                                                                             10 estimate, median number interactions <br /> examined. Genes conservation levels 4&#226;&#8364;&#8220;5 lost signi&#239;&#172;?cantly <br /> higher frequencies conserved genes (P , 0.05          increases rate 2&#226;&#8364;&#8220;2.25 3 1028/year (7/311 comparisons conservation levels 4&#226;&#8364;&#8220;5 vs. 6&#226;&#8364;&#8220;10, x-square tests).      15/760 . <br />                                                                                Genetic interactions mean synergistic effect                                                                             genes j &#239;&#172;?tness (fij) different expected rapidly original secondary structure degrades                  multiplicative effects: fij = fi fj + e, e 6&#194;&#188; 0, fj <br /> change amino acid composition sequence                    denotes effect gene j &#239;&#172;?tness, e magnitude <br /> Figure 6 Materials Methods details). results                epistasis genes (Dixon et al. 2009; proto-genes sensitive mutations                  Costanzo et al. 2010). extreme example synthetic older genes; e., secondary structure changes faster             lethal phenotype, independent deletion sequence change secondary structure                    genes cases results viable phenotypes, served genes (Figure 7A, P ,, 0.001, Bonferroni post hoc                    double deletion mutant lethal. tested num- <br /> tests, ANCOVA conservation level categorical predic-                ber genetic interactions depends gene age tor sequence divergence continuous predictor).                   positive relationship (Figure 8B):  structural composition different           number genetic interactions increases rate of conserved genes (Figure 4), product                4.8 3 1028/year (15/311 MYA);  appears compositional differences. comparison muta-                   level conservation level 6 (311 MYA). Similarly tional robustness helices b-sheets indicates                  number genetic interactions, strength changes <br /> b-sheets decay considerably faster random mutations                    age genes: magnitude epistasis (|e|) helices (Figure 7B, P ,, 0.001, ANCOVA),                     interactions proto-genes signi&#239;&#172;?cantly lower robustness proteins mutations negatively correlated               conserved genes (P ,, 0.001, Mann&#226;&#8364;&#8220;Whitney b-strand content (Figure 7C, P ,, 0.001, R =                     U-test, Figure 9). <br /> 20.54). suggests b-strands dif&#239;&#172;?cult maintain longer evolutionary periods helices,                   Discussion reduced conserved genes result accumulation neutral mutations. Repeat-                Overall, integration new genes cellular networks <br /> ing stability analysis nonstrand regions            supports hypothesis Carvunis et al. (2012)  <br /> proteins indicates proto-genes remain sensitive               tinuum exists conserved genes proto-genes. <br /> mutations (Figure 7D, P ,, 0.001, Bonferroni post hoc tests,                fastest process integration proto-genes ANCOVA), exclusion b-strands                     regulatory networks (Figure 2): youngest <br /> analysis; higher robustness old genes compared                  proto-genes regulated TFs, regulated proto-genes merely product lower                    large number genes, regulated network <br /> b-strand content.                                                           motifs (FFLs). oldest proto-genes appeared  14 <br />                                                                             MYA integrated regulatory net- number strength genetic protein&#226;&#8364;&#8220;protein <br />                                                                             work ancient fraction yeast pro- <br /> interactions increases age genes <br />                                                                             teome, appeared S. cerevisiae&#226;&#8364;&#8220;S. pombe <br /> Proteins typically perform function association               split. analysis focused topological fea- proteins form protein complexes. Physical inter-                  tures entire regulatory network  e., ignored actions proteins largely determined                    different environmental conditions life stages  <br />  <br />  <br /> 1412     G. Abrus&#195;&#161;n <br />                                                                                 subsets genes expressed), agreement <br />                                                                                observations reporting acquisition regula- <br />                                                                                tory sequences new (retro)genes happen rapidly <br />                                                                                (Kaessmann et al. 2009) horizontally transferred genes <br />                                                                                bacteria acquire regulatory interactions mil- <br />                                                                                lion years (Lercher P&#195;&#161;l 2008) &#239;&#172;?ndings <br />                                                                                proportion proto-genes proximity known TF <br />                                                                                binding sites changes little conservation level (Carvunis <br />                                                                                et al. 2012). fact youngest essential genes                                                                                present proto-genes (Figure 3),                                                                                agreement experimental &#239;&#172;?ndings reporting differences <br />                                                                                sets essential genes yeast strains <br />                                                                                (Dowell et al. 2010), indicates proto-genes gain <br />                                                                                functions rapidly. Unlike regulatory evolution, acquisi- <br />                                                                                tion protein&#226;&#8364;&#8220;protein interactions (PPI) dramatically <br />                                                                                slower process, does appear level pro- <br />                                                                                tein age, range examined conser- <br />                                                                                vation levels (Figure 8). observed rate PPI gain                                                                                order magnitude faster loss interac- <br />                                                                                tions, estimated occur rate 1029 (Lewis <br />                                                                                et al. 2012) 1.6&#226;&#8364;&#8220;2.6 3 10210 (Qian et al. 2011). addi- <br />                                                                                tion, interaction databases incomplete, rate                                                                                PPI gain likely underestimated. case genetic <br />                                                                                interactions increase number new interactions <br />                                                                                pronounced conservation level 6 (311 MYA, <br />                                                                                Figure 8), oldest group (conservation level <br />                                                                                10) characterized signi&#239;&#172;?cantly higher number <br />                                                                                interactions genes conservation levels 6&#226;&#8364;&#8220;9. <br />                                                                                Unlike PPIs, genetic interactions allow measurement <br />                                                                                strength interactions (e), indicates stronger <br />                                                                                epistasis conserved genes proto- <br />                                                                                genes; change quantitative qual- <br />                                                                                itative (Figure 9). <br />                                                                                    contrast, structural maturation new genes shows <br />                                                                                different pattern: secondary structures structural <br />                                                                                stability proteins indicate proto-genes conserved <br />                                                                                genes form relatively distinct groups: proto-genes                                                                                high b-strand content, high aggregation propensity, low <br />                                                                                robustness mutations conserved genes                                                                                lower strand content higher stability. Proto-genes                                                                                conservation levels 0&#226;&#8364;&#8220;3 different random <br />                                                                                sequences  similar amino acid frequency),                                                                                transition states falls conservation <br />                                                                                levels 4 ( 14 MYA) 6 ( 311 MYA), requiring <br />                                                                                considerably time regulatory evolution (Figures <br />                                                                                4 7). mechanisms cause <br />                                                                                shift; selection high b-strand content neutral <br />                                                                                process. selection favoring mutations  <br /> Figure 6 overview analysis protein structural robustness,      duce aggregation propensity proto-genes example yeast ORF YDR103W.   tertiary structure protein (PDB <span id='am-8' about='oboInOwl:id' typeof='owl:Thing'>id</span>: 4F2H). helices highlighted blue b-strands yellow. (B) sequence protein gradually mutated 70 <br /> steps; step 1% residues changed, step       original protein, particularly b-strands. (C) protein secondary structure determined. change location heli-        mutagenesis repeated &#239;&#172;?ve times, Q3 value percentage <br /> ces sheets occurs mutagenesis indicted           residues secondary structure original structure&#226;&#8364;&#8221; <br /> respective colors. sequence similarity original sequence declines,   calculated step. line represents mutagenesis path <br /> fewer fewer residues secondary structure        (replicate); analyses average &#239;&#172;?ve replicates used. <br />  <br />  <br />  <br />                                                                                                      Dynamics Protogene Functionalization        1413 <br />  Figure 7 Structural robustness proteins.   robustness secondary structures mutations depends conservation level. Proto-genes ancient genes highly signi&#239;&#172; difference (ANCOVA, P ,, 0.001 comparisons proto-genes conserved genes, Bonferroni <br /> post hoc tests; whiskers represent 95% con&#239;&#172;?dence intervals); secondary structure ancient genes sensitive mutations  e., Q3 value higher). (B) b-strands decay faster random mutations helices (P ,, 0.001, ANCOVA). (C) b-strands proteins correlates <br /> negatively structural stability protein. Q3 values calculated 50% sequence similarity original sequence. (D) structural <br /> stability proteins, excluding regions b-strands. difference proto-genes ancient genes highly signi&#239;&#172; (P ,, 0.001, <br /> ANCOVA), indicating merely product compositional differences ancient proto-genes  Figure 4). <br />  <br />  <br />  attractive explanation reduction seen                             est number founder genes typically form peak b-strands, &#239;&#172;?tness cost misfolding-induced aggre-                     youngest evolutionary strata (Tautz Domazet-Lo gation demonstrated experimentally yeast                            2011), implying proto-genes subject form <br /> (Geiler-Samerotte et al. 2011), tests selection                       turnover, similarly recently shown Carvunis et al. (2012) reported weak purifying selection                     functional noncoding sequence mammals (Meader et al. <br /> proto-genes, strong adaptive evolution. Alternatively,                  2010). Gene deletion inactivation studies reduction strand content aggregation propensity                      80&#226;&#8364;&#8220;90% genes eukaryotes prokaryotes lost re&#239;&#172;&#8218;ect  despite partly integrated                   individually signi&#239;&#172; &#239;&#172;?tness effect (Korona <br /> regulatory networks, considerable fraction proto-genes                     2011), laboratory conditions, does make higher conservation levels lost                     suggested genes lost easily evolution <br /> relatively quickly appearance. suppor-                 important, e., lower expression levels, fewer <br /> ted phylostratigraphic studies, high-                    protein&#226;&#8364;&#8220;protein interactions (Krylov et al. 2003), higher <br />  <br />  <br />  <br /> 1414      G. Abrus&#195;&#161;n <br />                                                                                Figure 9 Genetic interactions proto-genes weaker epistasis                                                                               conserved genes (P ,, 0.001, Mann&#226;&#8364;&#8220;Whitney U-test). proto- <br />                                                                               conserved gene mean absolute genetic interaction scores <br />                                                                               (|e|) calculated; histograms represent positive                                                                               negative epistatic interactions. <br />  <br />  <br />  <br />  <br />                                                                               consideration higher b-strand content (Figure 7).                                                                               mutations proteins accumulate                                                                               change structure key determinants <br />                                                                               capacity produce evolutionary innovations,  <br />                                                                               cause robust proteins better tolerate innovative <br />                                                                               destabilizing mutations (Bloom et al. 2006b; Ferrada <br />                                                                               Wagner 2008). pattern observed yeast pro- <br />                                                                               teome line experimental &#239;&#172;?ndings individual <br />                                                                               proteins, suggest robustness mutations <br />                                                                               promotes evolvability (Bloom et al. 2006b) evo- <br />                                                                               lution favors mutational robustness (Bloom et al. 2007; <br />                                                                               Bershtein et al. 2008). Additionally, large-scale analyses <br />                                                                               folding energies (DG) protein structures indicate <br />                                                                               ancient, eukaryote-wide proteins thermodynami- <br /> Figure 8 dependence protein&#226;&#8364;&#8220;protein genetic interactions gene age. Note y-axis logarithmic        cally stable robust mutations younger <br /> genes included interactions, correct research biases.   ones present metazoans vertebrates (Toll-Riera <br />   number protein&#226;&#8364;&#8220;protein interactions increases continuously         et al. 2012).  lower tolerance b-strands conservation level; new protein&#226;&#8364;&#8220;protein interactions emerge rate    mutations predicts strands, conserved <br /> 2&#226;&#8364;&#8220;2.25 3 1028/year. (B) number genetic interactions increases                                                                               proteins, evolve lower rates helices coils, rate  4.8 3 1028;  rate change slows conservation level 6.                                                         yeast (Bloom et al. 2006a), al- <br />                                                                               effect weaker effects struc- <br />                                                                               tural traits like solvent accessibility. <br /> evolutionary rate (Zhang 2005). analysis gene                      conclude, results somewhat contradictory <br /> loss study indicates proto-gene turnover exists;                 picture evolution new genes yeast: integra- <br /> young genes lower conservation levels (4&#226;&#8364;&#8220;5) lost                     tion proto-genes cellular networks shows contin- <br /> considerably easily S. paradoxus S. mikatae                       uum, rates regulatory evolution gain ancient genes (Figure 5), integrated                      protein genetic interactions different.  cellular networks. alternative explanation                    trast, structural point view, new genes decay b-strands neutral process,                 face bottleneck: proto-genes conserved genes form helices maintained selection, loss                  relatively distinct groups, different b-strand content, strands neutral mutations compensated.                       aggregation propensity, robustness mutations.  <br />    robustness secondary structure mutations                         &#239;&#172;?nding young genes lost easily <br /> shows similar shift conservation levels 4 6                   conserved ones, indicates  b-strand content aggregation propensity: proto-genes                      functionality, young genes unstable robust conserved genes, takes                  genome. <br />  <br />  <br />  <br />                                                                                                     Dynamics Protogene Functionalization       1415 <br />  Acknowledgments                                                             esis regulating Y-linked male fertility genes. PLoS <br />                                                                             Genet. 6: e1001255. thank Bal&#195;&#161;zs Papp referees useful comments                 Dixon, S. J., M. Costanzo,  Baryshnikova, B. Andrews, C. suggestions. study supported Hungarian                  Boone, 2009 Systematic mapping genetic interaction net- <br /> Scienti&#239;&#172;?c Research Fund (OTKA) grant PD83571 G                         works. Annu. Rev. Genet. 43: 601&#226;&#8364;&#8220;625. <br />                                                                          Dowell, R. D., O. Ryan,  Jansen, D. Cheung, S. Agarwala et al., author declares competing interests. <br />                                                                             2010 Genotype phenotype: complex problem. Science <br />                                                                             328: 469. <br />                                                                          Drinnenberg,  , D. E. Weinberg, K. T. Xie, J. P. Mower, K. H. <br /> Literature Cited                                                            Wolfe et al., 2009 RNAi budding yeast. Science 326: 544&#226;&#8364;&#8220; <br />                                                                             550. <br /> Abdulrehman, D., P. T. Monteiro, M. C. Teixeira, N. P. Mira,  B.       Edgar, R. C., 2010 Search clustering orders magnitude <br />     Louren&#195;&#167;o et al., 2011 YEASTRACT: providing programmatic               faster BLAST. Bioinformatics 26: 2460&#226;&#8364;&#8220;2461. <br />     access curated transcriptional regulatory associations Sac-    Fernandez-Escamilla, -M., F. Rousseau, J. Schymkowitz, L. <br />     charomyces cerevisiae web services interface. Nucleic         Serrano, 2004 Prediction sequence-dependent muta- <br />     Acids Res. 39: D136&#226;&#8364;&#8220;D140.                                               tional effects aggregation peptides proteins. Nat. <br /> Begun, D. J., H.  Lindfors,  D. Kern, C. D. Jones,                  Biotechnol. 22: 1302&#226;&#8364;&#8220;1306. <br />     2007 Evidence novo evolution testis-expressed genes        Ferrada, E.,  Wagner, 2008 Protein robustness promotes <br />     Drosophila yakuba/Drosophila erecta clade. Genetics 176:         evolutionary innovations large evolutionary time-scales. <br />     1131&#226;&#8364;&#8220;1137.                                                              Proc. Biol. Sci. 275: 1595&#226;&#8364;&#8220;1602. <br /> Bershtein, S., K. Goldin, D. S. Taw&#239;&#172;?k, 2008 Intense neutral          Gao, M., J. Skolnick, 2010 Structural space protein-protein <br />     drifts yield robust evolvable consensus proteins. J. Mol.           interfaces degenerate, close complete, highly  <br />     Biol. 379: 1029&#226;&#8364;&#8220;1044.                                                   nected. Proc. Natl. Acad. Sci. USA 107: 22517&#226;&#8364;&#8220;22522. <br /> Biegert, , J. S&#195;&#182;ding, 2009 Sequence context-speci&#239;&#172;?c pro&#239;&#172;?les        Geiler-Samerotte, K. , M. F. Dion, B.  Budnik, S. M. Wang, D. L. <br />     homology searching. Proc. Natl. Acad. Sci. USA 106: 3770&#226;&#8364;&#8220; <br />                                                                             Hartl et al., 2011 Misfolded proteins impose dosage-dependent <br />     3775. <br />                                                                             &#239;&#172;?tness cost trigger cytosolic unfolded protein response Bloom, J. D., D.  Drummond, F. H. Arnold, C. O. Wilke, <br />                                                                             yeast. Proc. Natl. Acad. Sci. USA 108: 680&#226;&#8364;&#8220;685. <br />     2006a Structural determinants rate protein evolution <br />                                                                          Hedges, S. B., J. Dudley, S. Kumar, 2006 TimeTree: public <br />     yeast. Mol. Biol. Evol. 23: 1751&#226;&#8364;&#8220;1761. <br />                                                                             knowledge-base divergence times organisms. Bioin- <br /> Bloom, J. D., S. T. Labthavikul, C. R. Otey, F. H. Arnold, <br />                                                                             formatics 22: 2971&#226;&#8364;&#8220;2972. <br />     2006b Protein stability promotes evolvability. Proc. Natl. Acad. <br />                                                                          Heinen, T. J.  J., F. Staubach, D. H&#195;&#164;ming, D. Tautz, <br />     Sci. USA 103: 5869&#226;&#8364;&#8220;5874. <br />                                                                             2009 Emergence new gene intergenic region. <br /> Bloom, J. D.,  Lu, D. Chen,  Raval, O. S. Venturelli et al., <br />                                                                             Curr. Biol. 19: 1527&#226;&#8364;&#8220;1531. <br />     2007 Evolution favors protein mutational robustness suf&#239;&#172;?- <br />                                                                          Jahn, T. R., O. S. Makin, K. L. Morris, K. E. Marshall, P. Tian et al., <br />     ciently large populations. BMC Biol. 5: 29. <br /> Cai, J., R. Zhao, H. Jiang, W. Wang, 2008 novo origination           2010 common architecture cross-beta amyloid. J. Mol. <br />     new protein-coding gene Saccharomyces cerevisiae. Ge-           Biol. 395: 717&#226;&#8364;&#8220;727. <br />     netics 179: 487&#226;&#8364;&#8220;496.                                                 Kaessmann, H., 2010 Origins, evolution, phenotypic impact <br /> Capra, J. , K. S. Pollard, M. Singh, 2010 Novel genes exhibit         new genes. Genome Res. 20: 1313&#226;&#8364;&#8220;1326. <br />     distinct patterns function acquisition network integra-       Kaessmann, H., N. Vinckenbosch, M. Long, 2009 RNA-based <br />     tion. Genome Biol. 11: R127.                                            gene duplication: mechanistic evolutionary insights. Nat. <br /> Carvunis, -R., T. Rolland,  Wapinski, M.  Calderwood, M.            Rev. Genet. 10: 19&#226;&#8364;&#8220;31. <br />     Yildirim et al., 2012 Proto-genes novo gene birth. Na-        Kapranov, P.,  T. Willingham, T. R. Gingeras, <br />     ture 487: 370&#226;&#8364;&#8220;374.                                                      2007 Genome-wide transcription implications ge- <br /> Chen, S., Y. E. Zhang, M. Long, 2010 New genes Drosoph-              nomic organization. Nat. Rev. Genet. 8: 413&#226;&#8364;&#8220;423. <br />     ila quickly essential. Science 330: 1682&#226;&#8364;&#8220;1685.                Knowles, D. G.,  McLysaght, 2009 Recent novo origin Chen, S., X. Ni, B. H. Krinsky, Y. E. Zhang, M. D. Vibranovski et al.,      human protein-coding genes. Genome Res. 19: 1752&#226;&#8364;&#8220;1759. <br />     2012 Reshaping global gene expression networks sex-           Korona, R., 2011 Gene dispensability. Curr. Opin. Biotechnol. 22: <br />     biased gene expression integration young gene. EMBO J.          547&#226;&#8364;&#8220;551. <br />     31: 2798&#226;&#8364;&#8220;2809.                                                       Krylov, D. M., Y.  Wolf,  B. Rogozin, E. V. Koonin, <br /> Cherry, J. M., E. L. Hong, C. Amundsen, R. Balakrishnan, G. Binkley         2003 Gene loss, protein sequence divergence, gene dispens- <br />     et al., 2012 Saccharomyces Genome Database: genomics                ability, expression level, interactivity correlated eu- <br />     resource budding yeast. Nucleic Acids Res. 40: D700&#226;&#8364;&#8220;D705.            karyotic evolution. Genome Res. 13: 2229&#226;&#8364;&#8220;2235. <br /> Cliften, P. F., R. S. Fulton, R. K. Wilson, M. Johnston,             Lee, T. , N. J. Rinaldi, F. Robert, D. T. Odom,  Bar-Joseph et al., <br />     2006 duplication: gene loss adaptation Sac-            2002 Transcriptional regulatory networks Saccharomyces <br />     charomyces genomes. Genetics 172: 863&#226;&#8364;&#8220;872.                              cerevisiae. Science 298: 799&#226;&#8364;&#8220;804. <br /> Costanzo, M.,  Baryshnikova, J. Bellay, Y. Kim, E. D. Spear et al.,    Lercher, M. J., C. P&#195;&#161;l, 2008 Integration horizontally trans- <br />     2010 genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431.             ferred genes regulatory interaction networks takes DePristo, M. , D. M. Weinreich, D. L. Hartl, 2005 Missense            million years. Mol. Biol. Evol. 25: 559&#226;&#8364;&#8220;567. <br />     meanderings sequence space: biophysical view protein         Levine, M. T., C. D. Jones,  D. Kern, H.  Lindfors, D. J. <br />     evolution. Nat. Rev. Genet. 6: 678&#226;&#8364;&#8220;687.                                 Begun, 2006  <br /> </body></html>