<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>          Inactivation YME1, Member ftsH-SEC18-PAS1- <br />           CDC48 Family Putative ATPase-Encoding Genes, Causes <br />                Increased Escape DNA Mitochondria                            Saccharomyces cerevisiae <br />  <br />              yeast nuclear gene YMEJ genes recently identified screen mutations elevate <br />            rate DNA escapes mitochondria migrates nucleus. ymel mutations, including            deletion, cause known recessive phenotypes: elevation rate copies TRPI ARSI, <br />            integrated mitochondrial genome, escape nucleus; heat-sensitive respiratory-growth defect;            cold-sensitive growth defect rich glucose medium; synthetic lethality rho- (cytoplasmic petite) cells. <br />            cloned YMEI gene complements phenotypes. gene product, Ymelp, immunologically <br />            detectable 82-kDa protein present mitochondria. Ymelp member family homologous <br />            putative ATPases, including Secl8p, Paslp, Cdc48p, TBP-1, FtsH protein. Ymelp similar            Escherichia coli FtsH protein, essential protein involved septum formation cell division.            observation suggests hypothesis Ymelp play role mitochondrial fusion division. <br />  <br />    intracellular movement DNA mitochondria                  generating respiration-competent  temperatures 300 nucleus experimentally detected strains                    petite-negative strains S. cerevisiae. Saccharomyces cerevisiae contain nuclear genetic                      isolation <span id='am-43' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-46' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> sequence YME1 <br /> markers mitochondrial genomes (43, 44). phe-                 detection product, Ymelp, mitochondria. <br /> nomenon probably played role              Ymelp significant homology family putative transfer genetic information mitochondrial                  ATPases members involved processes ranging <br /> nuclear genomes occurred eucaryotic evolu-                 organelle biogenesis gene expression cell division. <br /> tion (10, 13, 19, 25, 31, 47). Equally importantly,            Ymelp appears related Escherichia coli examination mechanism DNA escapes                       FtsH protein, essential protein involved septum  mitochondria provide novel avenue iden-                  mation cell division (46). <br /> tifying studying factors involved maintaining integrity mitochondria dynamic processes                               MATERIALS METHODS <br /> cell division fusion. <br />    study genetically mechanism DNA es-                     Strains, strain constructions, genetic methods. E. <br /> capes mitochondria, devised screen mutations              coli strain used preparation manipulation DNA <br /> elevate rate escape occurs (44). fragment             DH5a [F- endA41 hsdR17(r - iK+) supE44 thi-l X recA <br /> yeast nuclear DNA bearing TRPJ gene associated              gyrA96 relAI argF-lacZYA) U169 +80 lacZAM15]. <br /> origin nuclear DNA replication, ARS1, integrated                    genotypes S. cerevisiae strains used fully functional mitochondrial chromosome. Strains               work listed Table 1. Standard genetic techniques containing mitochondrially located TRP1 gene                  used construct analyze various yeast strains (38). <br /> nuclear trpl deletion phenotypically tryptophan auxo-                 Media. E. coli containing plasmids grown LB (10 g <br /> trophs (Trp-).  strains produced Trp+ deriv-               Bacto Tryptone, 10 g NaCl, 5 g yeast extract atives contained TRPJ-ARSJ fragment por-                  liter) plus 125 ,ug ampicillin ml. Yeast strains tions mitochondrial DNA (mtDNA) replicating                       grown complete glucose medium (YPD medium), com- <br /> plasmids nuclei. identify mutations affecting           plete ethanol glycerol medium (YPEG), minimal glu- <br /> rate escape, screened elevated production Trp+             cose medium plus indicated nutrients (SD medium) (44). <br /> derivatives. recessive nuclear mutations                Ampicillin nutrients obtained Sigma. <br /> fell complementation groups, designated YME1                      Isolation ymel-complementing plasmids. tempera- YME6  yeast mitochondrial escape), ob-                 ture-sensitive respiratory-growth phenotype caused tained.                                                                 ymel mutation used basis cloning YMEL. <br />    YMEJ gene particular ymel muta-             PTY62 transformed (18) 70 ,ug DNA prepared <br /> tions cause heat-sensitive defect respiratory growth           YCp50-based S. cerevisiae genomic DNA bank (34). cold-sensitive defect growth fermentable carbon                 thousand Ura+ colonies grew minimal SD  <br /> sources (44). addition, reported  ymel mutations             dium incubation 4 days 30&#194;&#176;C. transformants <br /> cause lethality rho- (cytoplasmic petite) cells,             prewarmed 37&#194;&#176;C 1 h replica plated                                                                         YPEG plates incubated 37&#194;&#176;C. transformants <br />                                                                         able grow restrictive condition plasmid- <br />   * <br />       Corresponding author.                                             dependent fashion. Total yeast DNA prepared                                                                     5418 <br />  VOL. 13, 1993                                       ymel MUTATIONS INCREASE DNA ESCAPE MITOCHONDRIA                            5419 <br />  <br />                                                   TABLE 1. Yeast strains used study <br />     Strain                                                       Genotype                                            Source reference <br /> PTY33                            MATa ade2 ura3-52 leu2-3,112 trpl-Al [rho&apos; TRP1]                                      44 <br /> PTY33 xPTY44                     MATaad2 LYS2 ura3-52 leu2-3,112 til-Al [rho&apos; TRP1]                                    study <br />                                  M4TaADE2 lys2 ura3-52 leu2-3,112 trpl-Al <br /> PTY44                            MATa Iys2 ura3-52 leu2-3,112 trpl-Al [rho&apos; TRP1]                                      44 <br /> PTY51                            ALTa ade2 ura3-52 leu2-3,112 trpl-AJ YMEI::URA3 [rho&apos; TRP1]                           study <br /> PTY52                            AMTa Iys2 ura3-52 leu2-3,112 trpl-Al ymel-Al::URA3 [rho&apos; TRPI]                        study <br /> PTY62                            MATa, lys2 ura3-52 leu2-3,112 trpl-Al ymel-l [rho&apos; TRPI]                              44 <br /> PTH78                            AMTa ade2-101 ura3-52 petl23::URA3                                                    16 <br /> LSF236                           MATa ade2 ura3-52 trpl-Al pet494-41                                                   L. S. Folley <br /> TF197                            ATcTa, leu2-3,112 ura3-52ymel-l [rho&apos; TRPI]                                           study <br /> A236-57B                         MATa, leu2-3 trpl met4 aro7 his3 lysll SUC2 MAL3 canl                                 12 <br /> X12-6B                           MATa, radl-l ade2-1 gal2                                                              YGSCa <br />  YGSC, Yeast Genetic Stock Center, Berkeley, Calif. <br />  <br />  transformants used transform E. coli.            generate chimeric gene encoding <span id='am-15' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-16' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span>glutathione</span> S-trans- rescued plasmids identical (pPT31 Fig. 2),           ferase-Ymelp fusion protein. resulting plasmid, pPT44, contained smaller overlapping insert (pPT32             transformed E. coli DH5a, fusion protein Fig. 2).                                                             induced isolated described previously (41).    Nucleic acid techniques plasmid constructions. Restric-          fusion protein used immunize rabbits previously <br /> tion enzymes T4 DNA ligase purchased                      described (5). <br /> Bethesda Research Laboratories New England Biolabs.                   Total protein extracts prepared YPEG-grown <br /> Standard techniques generating recombinant DNAs                 yeast cells. Cells collected washed 1 M <br /> performing DNA blot hybridizations used (26).                      sorbitol-100 mM EDTA, pH 7.5. Cells resuspended    Plasmids pPT31AX pPT31AB  Fig. 2)                   0.05 times original volume buffer 160 ,ug <br /> structed digestion pPT31 XbaI BamHI,                     Zymolyase 20T (ICN Immunobiologicals, ) ml respectively, followed gel purification large restric-        incubated 3 h 37&#194;&#176;C. spheroplasted cells tion fragment subsequent recircularization T4 li- <br /> gase. pPT31AX pPT31AB YCp50-based CEN                      pelleted 10 pellet volumes 50 mM Tris-20 mM <br /> plasmids. Plasmids pPT34 pPT35  respectively, Sall              EDTA-1% sodium dodecyl sulfate (SDS), pH 7.4, BamHI fragments pPT32 gel purified                 heated 65&#194;&#176;C 30 min. protein extracts ligated corresponding sites polylinker         frozen -80&#194;&#176;C needed. crude cell extract, purified <br /> URA3 CEN vector pRS316 (40). Plasmid pPT49                      mitochondria, postmitochondrial supernatant pre- <br /> structed digesting pPT32 MluI EcoRI                pared YPEG-grown PTY44 yeast cells essentially purifying 3-kb insert fragment ligating vector        described Yaffe (50). Briefly, cells spheroplasted fragment created digesting pPT34 AMluI EcoRI.               treatment Zymolyase broken Dounce homog- kilobase pairs yeast DNA plus 378 bp vector                  enizer. Unbroken cells debris removed centrif- <br /> sequence contained plasmid excised digestion             ugation 3,000 x g. crude mitochondrial pellet XhoI EcoRI ligated sites                  generated centrifuging extract 9,500 x g pRS315, LEU2 CEN vector (40), create pPT49.                        collecting pellet. supernatant spin consti- <br />    ymel-Al::URA3 mutation constructed fol-                   tuted postmitochondrial supernatant. Mitochondria lows. pPT42 DNA digested HpaI, removing 1,370                  washed twice resuspending mitochondrial pellet nucleotides internal YME1 sequence. pPT42               pelleting mitochondria 9,500 x g. Mitochondria <br /> insert DNA pPT49 corresponding sites                   purified banding Percoll density <br /> polylinker pBluescript KS&apos;. place nucleotides            gradient. ligated 1.5-kb SmaI fragment containing URA3                     Protein fractions subjected SDS-polyacrylamide <br /> gene, generating pPT45  Fig. 2). Sixteen micrograms              gel electrophoresis described (23). Protein- <br /> pPT45 DNA digested BglII HindIII,                 containing fractions separated 10% gel elec- used transform yeast diploid PTY33 xPTY44 <span id='am-88' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-89' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> prototrophy. Ura+ transformant sporulated,                 troblotted 15 V 45 min 0.2-,um nitrocellulose tetrads dissected. spore                   (Bio-Rad) using Bio-Rad Trans-Blot SD cell. tetrads PTY52.                                                       transfer buffer 48 mM Tris-39 mM <span id='am-58' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span>, pH 9.2. <br />    DNA sequence determined double-stranded                   transfer, filter stained Ponceau S 5% <br /> templates nucleotide chain termination method (37).              <span id='am-93' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acetic acid</span> subsequently destained distilled water. <br /> Templates generated pPT34 pPT35 sub-                   filter blocked 20 mM Tris-500 mM NaCl-3% <br /> cloning restriction enzymes exonuclease III-gener-             gelatin, pH 7.5. filter washed 20 mM <br /> ated deletions. gaps sequence closed             Tris-500 mM NaCl-0.05% Tween 20, pH 7.5. rabbit <br /> having oligonucleotides synthesized using prim-             antisera diluted 1:200 wash solution containing <br /> ers.                                                                    1% gelatin incubated filter 1 h 37&#194;&#176;C    Detection Ymelp mitochondrial fractionation.                shaking. Immune complexes identified reaction 1.5-kb HpaI-EcoRI fragment DNA, encoding car-                    secondary antibody, purified goat anti-rabbit immunoglob- <br /> boxy-terminal 105 residues Ymelp, ligated               ulin G heavy-plus-light-chain gold conjugate (Bio-Rad), <span id='am-83' about='obo:IMR_0200318' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="Glutathione" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:id" content="IMR:0200318" datatype="xsd:string"></span>glutathione</span> S-transferase vector pGEX-3X (Pharmacia)                 16 h 37&#194;&#176;C shaking. <br />  5420     THORSNESS ET AL.                                                                                               MOL. CELL. BIOL. <br />  <br />   Nucleotide sequence accession number. sequence                 Vme YM <br />                                                                                   :IIRA YME1 assigned GenBank accession number                                      l .1  L14616. <br />                                                                   ymc 1 -Al::URA3                                                  YMF1 <br />                            RESULTS <br />                                                                          lVec^torl|                                                YMII <br />    Phenotypes associated YME] mutations.  genes identified previous screen mutations <br />  affecting rate mtDNA escapes nucleus, <br /> YME1 particular ymel mutations                                  ymc I                Evmc ] <br /> caused phenotypes. addition having                                YMI 1                      IVectorl <br /> creased rate mtDNA escape (Fig. 1A), ymel strains  unable grow nonfermentable carbon sources 37&#194;&#176;C showed significantly decreased growth rates rich medium containing glucose galactose 14&#194;&#176;C (Fig.            B                                  S&apos;&apos;ITAI N <br />  1B).                                                                     PLASMID)                YM <br />                                                                                                    N v1; <br />                                                                                                        n                                                                                                          mv      yrnC -    preliminary phenotypic characterization                                                                :(1RA ,3 <br /> ymel mutants (44), elevation                 Ve(tor frequency cytoplasmic petite, rho-, mutants.  careful examination suggested ymel-1 mutation actually prevent accumulation                      YMFI <br /> rho- mutants: 14,000 clones ymel-1 mutant <br /> strain, rho- colonies  contrast isogenic wild-type strain, approximately 1%                Vector <br /> cells rho-. <br />    incompatibility ymel-1 mutation rho- <br /> mutations confirmed ways.  ymel-1 strains                  YMI grow medium containing 25-,ug/ml concentra- <br /> tion ethidium bromide (Fig. 1C), drug effi- <br /> ciently induces rho- mutations (14). Second, ymel-1 <br /> allele caused inviability strains carried null <br /> petl23 mutation, known destabilize rho&apos; mtDNA           C ymel-AI::11l/A43 <br /> (16). ymel mutant strain PTY62 crossed                                                                     YMUIJ <br /> petl23::URA3 strain PTH78, resulting diploids                       IYMliII         Ad                        Vec tor <br /> sporulated. tetrads dissected YPD medium scored growth, respiratory ability, uracil pro- <br /> totrophy  follow segregation petl23::URA3).  <br /> quarter spores did yield viable haploids,     ymclI-Al::1IRA.4 germinated formed microcolonies. cases               lVectorl                                           YMFI <br /> viable spores wild type contained ymel-1 <br /> orpetl23::URA3 mutation,  <br />    ymel mutation simply incompatible unconditional respiratory defects. demonstrated                           IVic/-                     PI <br />  <br />                                                                                                                    viln - 1 <br /> crossing ymel-1 strain (PTY62) strain (LSF236)                                 YmJ]:                      Ves tI orI <br /> carrying pet494 mutation, prevents respiration         FIG. 1. Complementation ymel phenotypes cloned yeast <br /> does destabilize rho&apos; mtDNA (8, 29). viability       genomic DNA. YMEI strain (PTY44), ymel-l mutant (PTY62), <br /> spores cross normal, quarter       ymel-Al ::URA3 mutant (PTY52) transformed progeny double mutants.  ymel-1 appears         YMEl-carying plasmid pPT49 vector pRS315. trans- <br /> specifically incompatible rho- (cytoplasmic petite) mu-    formants, indicated plasmid brackets appropriate, <br /> tations effectively produces respiration-competent       analyzed follows.   Detection DNA escape mito- <br /> temperatures 30&#194;&#176;C  petite-negative strains S.    chondria nucleus. Confluent sectors grown SD medium <br /> cerevisiae.                                                     supplemented <span id='am-27' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>, uracil, lysine, tryptophan replica <br />    Isolation yeast genomic DNA complements ymel         plated SD medium supplemented adenine, uracil, lysine <br />                                                                  plate shown) detect Trp+ colonies resulting escape mutations. yeast strain PTY62, bearing ymel       TRP1 mitochondria. plate photographed 5 days ura3-52 mutations, transformed YCp5O-based         incubation 30&#194;&#176;C. (B) Respiratory growth 37&#194;&#176;C growth rich <br /> genomic library Rose et al. (34). 12,000 transfor-     glucose medium 14&#194;&#176;C. Cells patched YPD medium printed <br /> mants obtained, replica plated YPEG, incubated      YPEG incubated 2 days 37C (upper panel) printed 37&#194;&#176;C. transformants capable growth          YPD medium incubated 7 days 14&#194;&#176;C (lower panel). (C) <br /> YPEG 37&#194;&#176;C. Total DNA prepared trans-          Growth glucose presence ethidium bromide. cells formants, plasmids recovered transforma-      streaked SD medium containing 25 Fg ethidium bromide ml, <br /> tion E. coli. Reintroduction plasmids PTY62    adenine, uracil, lysine, leucine, tryptophan incubated 30&apos;C <br /> demonstrated complemented heat-sensitive          3 days. Cells picked isolated colonies  strains <br />                                                                 containing YMEJ) area heaviest growth  ymel <br /> respiration phenotype ymel-1. Restriction analysis    strains), streaked medium, incubated 30&#194;&#176;C 4 days <br /> plasmids revealed recovered plasmids       growth shown. necessary reculture <br /> identical (pPT31 Fig. 2). plasmid, pPT32,          strains times presence ethidium bromide ensure contained somewhat smaller insert region     generation rho- rhoo mitochondrial genomes. <br />                         <br />    FIG. 2. Restriction map, complementation analysis, gene disruption YMEI region. lines represent vector sequences, thinner lines represent yeast genomic sequences. genomic sequences various subclones aligned. position orientation YME1 open reading frame indicated plasmid pPT31 box arrow, respectively. Plasmids pPT31, pPT32, <br /> pPT31AX, pPT31AB YCp5O-based plasmids. Plasmids pPT34 pPT35 pRS316 based, pPT49 pRS315 based. insert plasmid pPT45 pBluescript KS+. pPT45 used step gene replacement YMEI locus. resulting locus placed trans ymel-l diploid, did complement, plasmid indicated noncomplementing. Restriction <br /> endonuclease cleavage sites: B, BamHI; C, ClaI; E, EcoRI; G, BglII; H, HindIII; Hp, HpaI; M, MluI; S, SatI; X, XbaI; Xh, XhoI. <br />  <br />  genome. Subsequent subcloning transformation                    chain termination reactions used determine PTY62 defined complementing region DNA                         sequence 2,839 nucleotides strands (Fig. 3). accurately (Fig. 2). subcloned segment DNA                       sequence single open reading frame encodes plasmid pPT49 complemented phenotypes associ-                     protein 747 amino acids predicted relative molec- <br /> ated ymel mutation (Fig. 1).                                    ular weight 81,679. sequence predicted protein, <br />    Genetic mapping YMEI locus. 3.5-kb Clal frag-                  Ymelp, used search sequence data bases. ment pPT32, containing YMEJ gene                      significant sequence homology family proteins 3&apos; sequence, introduced integrating URA3 <br /> vector. plasmid, pPT37, linearized restric- <br /> tion endonuclease MluI used transform wild-type <br /> strain PTY33. Ura+ transformant, PTY51, mated                     TABLE 2. Tetrad analysis linkage markers ymel-1 strain TF197, resulting diploid                                  YMEI region chromosome 16a <br /> sporulated. recombinants 18 tetrads                                                  Ascus type                 Segregation <br /> scored uracil prototrophy heat-sensitive respiratory                                               asci)                asci)&apos;        Distance <br /> growth, indicating cloned chromosomal DNA cor-                          Interval                                                                  (cM) <br /> responded YMEI locus.                                                                        PD      NPD            T        FD           SD <br />    plasmid pPT35 used probe chromosome blot                   ymel-aroT                33        2            45                               35 S. cerevisiae DNA. probe hybridize                  ymel-CEN16C                                              53           24         16 filter position chromosome 16 (data                 ymel-radld            21       5       37                                        53 <br /> shown). Meiotic-mapping data YME1, tetrad anal-                     PD, parental ditype; NPD, nonparental ditype; T, tetratype; FD, ysis, summarized Table 2. YMEJ located right               division; SD, second division; cM, centimorgans. <br /> arm chromosome 16 16 centimorgans                          b Scored relative centromere-linked marker trpl. <br /> centromere 35 centimorgans aro7.                              c cross TF197 x A236-57B. <br />    Primary sequence analysis YMEI. Dideoxynucleotide                      d   cross TF197 x X12-6B. <br />  5422    THORSNESS ET AL.                                                                                      MOL. CELL. BIOL. <br />  <br />          TGGTACCTCTTTCTTCGAGGAGTGG TAGCTCCTGTACGCTGTAGCGGTAG TGAAAAGGCAATATATCGGAGTTGG TGAAGTGAGTTAAATCTCTACCTTG   100 <br />           CGITITTGATAAGTAAAAAGCAAGC ACAGCTTAAAGGATAAACAGGATTT TATTGAAGCAATITMAATTATAAT ACATTGTGGATAGAACGAAAACAGA   200 <br />           GACGTGATAGATGAACGTlICAAAA ATACTTGTGTCGCCCACGGTCACGA CAAATGTIITACGCATATTTGCTCC CAGGCTACCTCAAATCGGTGCIICT  300 <br />                      M N V S K L V S P T V T T N V L R F P R L P Q G S <br />          TTGTTAGTTCAAAAAAAATGGGCCT TAAGATCAAAGAAGITCTACCGTTT TTATTCTGAAAAGAATAGCGGTGAA ATGCCTCCTAAGAAGGAAGCTGATA   400 <br />           L L V Q K K W L R S K K F Y R F Y S E K N S G E M P P K K E D S <br />           GCTCTGGAAAGGCATCCAATAAATC CACGATATCTTCAATCGACAATTCG CAACCACCACCTCCATCGAACACTA ATGATAAAACCAAACAAGCGAACGT  500 <br />             S G K S N K S T S S D N S Q P P P P S N T N D K T K Q N V <br />          AGCTGTGTCACATGCTATGCTAGCA ACTAGAGAACAAGAGGCCAATAAAG ACTTAACGAGTCCTGATGCACAAGC AGCCTTTTACAAAC1TCTCTTACAA   600 <br />            V S H M L T R E Q E N K D L T S P D Q F Y K L L L Q <br />           TCAAACTACCCGCAATACGTGGTCT CTAGGTTTGAGACCCCCGGTATTGC GTCATCGCCTGAATGCATGGAACTG TACATGGAGGCCCTGCAGAGGATAG  700 <br />           S N Y P Q Y V V S R F E T P G S S P E C M E L Y M E L Q R G <br />           GCAGACACTCGGAGGCTGATGCCGT TAGACAAAACCTACTGACAGCCAGC TCTGCTGGCGCAGTTAACCCATCAT TGGCGTCATCTTCATCAAACCAGTC  800 <br />             R H S E D V R Q N L L T S S G V N P S L S S S S N Q S <br />          AGGTTATCATGGTAACTTTCCATCG ATGTATTCTCCGCTTTATGGATCTC GCAAAGAGCCACTACATGTTGTCGT ATCTGAATCAACTIIIACTGTTGTA   900 <br />            G Y H G N F P S M Y S P L Y G S R K E P L H V V V S E S T F T V V <br />           TCGAGATGGGTAAAGTGGCTGCTTG TTTTCGGTATCTTAACCTACTCTTT TTCTGAAGGTTITAAATACATCACA GAAAATACAACGCTACTAAAGTCGT 1000 <br />           S R W V K W L L V F G L T Y S F S E G F K Y T E N T T L L K S S <br />           CAGAAGTAGCCGACAAATCAGTTGA TGTAGCTAAGACAAATGTTAAATTT GATGATGTCTGCGGTTGTGATGAGG CCCGTGCGGAATTGGAAGAAATTGT 1100 <br />             E V D K S V D V K T N V K F D D V C G C D E R E L E E V <br />           TGATTTCCTGAAAGATCCAACTAAG TACGAGTCCIIGGGTGGTAAACTAC CAAAGGGTGTACTGTTGACTGGACC TCCTGGTACAGGTAAAACTTTGTTG 1200 <br />            D F L K D P T K Y E S L G G K L P K G V L L T G P P G T G K T L L <br />           GCTAGGGCCACTGCCGGAGAGGCTG GTGTAGATFITTTCTTTATGTCAGG TTCTGAATTTGATGAAGTTTACGTC GGTGTTGGTGCTAAACGTATCCGTG 1300 <br />          R T G E G V D F F F M S G S E F D E V Y V G V G K R R D <br />          ATTTGTTTGCTCAAGCGCGTTCTCG TGCACCAGCTATTAIITFMATTGAT GAATTAGATGCCATCGGTGGTAAGC GTAATCCAAAGGACCAAGCTTACGC 1400 <br />             L F Q R S R P I F D E L D G G K R N P K D Q Y           CAAACAAACGTTGAATCAGTTATTG GTCGAATTAGATGGTTTCTCACAAA CAAGTGGAATTATTATTATTGGTGC CACAAATTTCCCTGAGGCTTTAGAT 1500 <br />            K Q T L N Q L L V E L D G F S Q T S G I I G T N F P E L D <br />           AAGGCGTTAACTAGACCAGGTAGAT TTGATAAGGTTGTGAATGTGGAITT GCCAGATGTTCGTGGTCGTGCTGAT ATTITAAAGCATCACATGAAGAAGA 1600 <br />           K L T R P G R F D K V V N V D L P D V R G R D L K H H M K K          TTACATTGGCAGACAATGTGGATCC AACTATTATTGCGCGTGGTACCCCC GGTTTATCAGGCGCTGAGCTGGCAA ATTTAGTCAACCAAGCAGCAGTTTA 1700 <br />             T L D N V D P T I R G T P G L S G E L N L V N Q V Y <br />           CGCGTGTCAAAAAAATGCTGITTCC GTTGATATGTCCCACTTCGAGTGGG CTAAGGATAAGATATTGATGGGTGC TGAGAGAAAGACTATGGTCTTAACA 1800 <br />            C Q K N V S V D M S H F E W K D K L M G E R K T M V L T <br />           GATGCAGCCAGAAAGGCCACTGCTT TCCACGAGGCTGGACATGCCATTAT GGCCAAATACACCAATGGTGCTACC CCGCTATACAAGGCCACGATATTGC 1900 <br />           D R K T F H E G H M K Y T N G T P L Y K T L P <br />           CTAGAGGTAGGGCATTGGGTATTAC TTTTCAATTGCCAGAAATGGATAAG GTCGACATCACCAAAAGGGAGTGTC AAGCCAGACTGGACGTGTGCATGGG 2000 <br />             R G R L G T F Q L P E M D K V D T K R E C Q R L D V C M G <br />           GGGCAAAATTGCAGAAGAATTAATT TATGGTAAAGATAACACCACAAGTG GTTGTGGGTCTGACTTGCAAAGCGC CACCGGCACAGCAAGGGCTATGGTT 2100 <br />            G K E E L Y G K D N T T S G C G S D L Q S T G T R M V <br />           ACTCAATATGGTATGAGTGATGATG TAGGTCCCGTTAACTTGTCAGAAAA TTGGGAATCTTGGTCTAATAAGATT CGCGATATTGCTGATAATGAAGTGA 2200 <br />           T Q Y G M S D D V G P V N L S E N W E S W S N K R D D N E V           TTGAACTTITGAAGGACTCCGAGGA AAGAGCAAGAAGACTATTAACTAAG AAAAATGTTGAGCTACATAGACTTG CGCAAGGTCIIATTGAATATGAAAC 2300 <br />             E L L K D S E E R R R L L T K K N V E L H R L Q G L E Y E T <br />           TCTAGATGCCCACGAAATCGAACAA GTTTGTAAAGGTGAAAAACTGGACA AACTGAAAACITCCACCAATACAGT CGTAGAAGGACCAGACAGTGATGAA 2400 <br />            L D H E E Q V C K G E K L D K L K T S T N T V V E G P D S D E <br />           CGTAAAGATATAGGCGATGATAAAC CCAAAATTCCTACAATGTTAAATGC ATGA1TTTATTCTAAGAAG1IAAAC GTATGAAGGAACCTACCTCAAGACC 2500 <br />             R K D G D D K P K P T M L N           GAGAATGCTTTGTTCTT17CTGGTC GGTTTCGCTTTTGCTTCATAACACC AGCAATGAAGCATGATCGCGGGAAA CITTCAGAATCTACATCTTCTCCTG 2600 <br />           GTCAGCTTGAAACTTCAGATTATTC GTAAGGAGAAATTATAATATTATTT CTGTGATAAGAAAAGAGGCATTAAG CTTGGGTATTTGCGACAAAATACTT 2700 <br />           CAGGATAACCGATGATCGCAAGTGG GTTATTAATTTCTTTCATCCCTTAA CCAACCGTCGCTT7CCCCCAGTGAC AAGCCTTAAAAAAGTCACACCATGG 2800 <br />           AACGGAGTATCGTTGATGATTAAAC CAATCATTAATCTTTCATAAAAGTT AAGATAAAAGCACTTGTTGATATAT ATAGGTACATGTGTATCAATACGCC 2900 <br />           ATACATTGTAAATAGAATATTCAAT GGATAAACGTGTGTAGCAAAAATAA TCACATTGTAGCGCTAGTTAG                               2971 <br />   FIG. 3. Nucleotide sequence YME1 predicted <span id='am-51' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-52' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span> sequence Ymelp. sequence 2,971 bp determined sequencing coding anticoding strands. <br />  <br />  putative ATPases, (32) biochem-         shorter proteins: Suglp (42) Afglp (24) yeast cells <br /> ically demonstrated ATPase activity. Ymelp        HIV-Tat-associated proteins TBP-1 (30) Mssl <br /> closely related FtsH protein E. coli (46).        (39) mammalian cells. <br /> FtsH protein apparently involved posttranslational        comparison sequences Ymelp FtsH <br /> processing PBP 3, protein necessary septation     protein presented Fig. 4. lengths, Ymelp <br /> cell division E. coli (46). Ymelp related       FtsH protein share 30% amino acid identity, 84-kDa yeast protein Secl8p, required        additional 3% residues conserved substitutions. <br /> membrane fusion events involving endoplasmic reticu-          amino-terminal portions proteins lum, vesicles, Golgi apparatus secretory           significant homology. amino terminus FtsH <br /> pathway (7). mammalian homolog Secl8p NSF,          protein contains hydrophobic stretches span N-ethylmaleimide-sensitive fusion protein (49).         cytoplasmic membrane twice, leaving bulk protein <br /> large yeast proteins family Paslp,           cytoplasm (45). obvious sequence <br /> involved peroxisome assembly (9), Cdc48p,            elements Ymelp suggest association appears involved nuclear division (11). Cdc48p       protein membranes. residue 272 Ymelp homologs, VCP (21) p97-ATPase (32),           residue 145 FtsH protein respective C termini, higher eucaryotes. Ymelp homologous         proteins 46% identical. sequences include <br />  VOL. 13, 1993                                        ymel MUTATIONS INCREASE DNA ESCAPE MITOCHONDRIA                             5423 <br />  <br />   Ymelp      MNVSRILVSP TVTTNVLRIF APRLPQIGAS LLVQKRWALR SRRFYRFYSE RNSGEMPPRK                             EADSSGKASN RSTIS          75 <br />  <br />   Ymelp      SIDNSQPPPP SNTNDRTRQA NVAVSHAMLA TREQEANKDL TSPDAQAAFY RLLLQSNYPQ YVVSRFETPG IASSP 150 <br />   FtsHp                                                                                      MALILVIA                SV FSFG         23 <br />  <br />   Ymelp      ECMELYMEAL QRIGRHSEAD AVRQNLLTAS SAGAVNPSLA SSSSNQSGYH GNFPSMYSPL YGSRREPLHV VVSES 225 <br />   FtsHp      PSESNGRKVD YSTFLQEVNN DQVREARING REINVTRRDS NRYTTYIPVQ DPRLLDNLLT RNVRVVGEPP        98                       EBEU <br />   Ymelp      TFTVVSRWVK WLLVFGILTY SFSEGFRYIT ENTTLLKSSE VADKS                                             CDE                     300 <br />   FtsHp      LASIFISWFP MLLLIGVWIF FMRQNQGGGG RGAMSFGKSR ARMLT                                             CD                      171 <br />  <br />   Ymelp      K      KYE G G        K        T    P        L                                                               S        375 <br />   FtsHp      R eRFQ1G GP                             TE                     F           F          GV                              246 <br />  <br />   Ymelp         t                   P---R DQAY QTLN         LV           QT I447 <br />   FtsHp             <span id='am-82' about='oboInOwl:id' typeof='owl:Thing'>ID</span>                AGLGG GHDE QTLN          V           GN GI 321 <br />  <br />   Ymelp         DVHRI   IAR <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> SGDKN5V <br />                                   Y Q V F D22 <br />   FtsHp         |DVRG!D    D <br />                            V     DNR <br />                                   F     F DK 396 <br />  <br />   Ymelp                                      GHA YTN GAR L                                              VD ITRRECQARL DVCi597 <br />   FtsHp                        KE t         EAGA RVPH   PRGRAL                                 P         ASRQKLESQI STL          471 <br />  <br />   Ymelp                  IYG     DNTT*4GS FQ                               D        LI ----WES                  ERD-----          660 <br />   FtSHp                  IYG     EHVS SN HR                                       LY3 EGEVFLG             jKHMSDET                546 <br />  <br />    Ymelp         KEI4RDS E              LRNVELAQ                   YE            QV--CR     GEKLD---RL RTS   PDSDE 730 <br />    FtsHp                 ERN Y          LNM   RD                   YE            DLMARR     DVRPPAGWEE PGA tSGDN GSPKA 621 <br />  <br />    Ymelp     RRDIGDDRPRK                -       NA <br />                                                  N                                                                                  747 <br />    FtsHp        PRPVDEPRTP     143E1TEDR                                                                                            644 <br />   FIG. 4. Amino acid sequence comparison Ymelp FtsH protein (FtsHp). sequences aligned aid alignment program GeneWorks 2.2 IntelliGenetics. Identical amino acids boxed, nucleotide-binding motifs (3) indicated lines sequence elements, membrane-spanning sequences FtsH protein (45, 46) underlined. significant level homology proteins evident amino-terminal portions. Consequently, boxed identities gaps introduced alignment program prior residue 272 Ymelp residue 145 FtsH protein eliminated alignment. <br />  <br />  region homology shared members                 analysis  shown) verify Ura+ spores  <br /> family putative ATPases. previously noted,               tained expected mutation, termed ymel-Al::URA3. region contains elements believed important              case, Ura+ spores exhibited phenotypes <span id='am-32' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-33' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> binding (underlined sequences Fig. 4) (3),            caused ymel mutation (Fig. 1): high rate DNA contains homologous sequence elements addition               escape mitochondria nucleus, temperature- assigned ATP-binding ATPase activity.                   sensitive respiratory growth 37&#194;&#176;C, slow growth rich <br /> duplication conserved nucleotide-binding region noted            glucose medium 14&#194;&#176;C, lethality combined Paslp, Secl8p, Cdc48p (9, 11) evident          rho- mtDNA. <br /> Ymelp FtsH protein.                                                ymel-Al::URA3 mutation deleted 60%    Generation phenotypic consequences ymel null                structural gene, including highly conserved se- <br /> mutation. destroy function YMEI, 1,370 nucle-                 quence associated ATP binding hydrolysis, sug- <br /> otides internal reading frame deleted                gesting null mutation. case, placed URA3 gene. resulting DNA fragment                     Ymelp dispensable growth respiration 30&#194;&#176;C, <br /> (pPT45; Fig. 2) carried URA3 flanked approximately 2 kb              absence results high rates DNA escape 5&apos; YMEI sequence, including presumptive YMEJ                     mitochondria. <br /> promoter 185 codons, approximately 1.5 kb                independent alleles ymel isolated 3&apos; YMEJ sequence included C-terminal 105                   original mutant screen (44), phenotypes caused codons (Fig. 2). DNA, linear form, used                  essentially caused ymel -Al:: <br /> transform diploid strain PTY33xPTY44 Ura+, delet-                URA3 mutation. ymel-l mutation appears ing copy YMEJ (36). Ura+ transformant                      missense mutation length Ymelp detected <br /> purified sporulated. resulting tetrads contained           aymel-l strain antibodies   directed viable spores, Ura+                   Ymelp. interesting thatymel-l exhibits weak intragenic Ura-. Total DNA prepared spore                 complementation ymel-3 allele, suggesting tetrad subjected DNA blot hybridization                   Ymelp function multimeric complex. <br />  <br />    FIG. 5. Immunological detection Ymelp yeast extracts anti-Ymelp antisera. Lanes: P, preimmune serum probing wild- <br /> type cell extract; 1, PTY52 (ymel-A1::URA3) cell <br /> extract; 2, PTY44 (wild type) cell extract; 3, PTY52/pRS315 <br /> (vector) cell extract; 4, PTY52/pPT49 (YME1 plasmid)  <br /> cell extract; 5, PTY44/pRS202 (multicopy vector) cell ex- <br />  <br />                                                                                                      mitochondria. <br />                                                                                                                                                  MOL. CELL. BIOL. <br />  <br />                                                                                                      (44).  shown phenotype,                                                                                                      associated withymel mutations, caused loss <br />                                                                                                      protein product, Ymelp, normally located                                                                                                         addition increasing rate DNA escape                                                                                                      mitochondria, Ymelp deficiency causes pheno- <br />                                                                                                      types clearly related mitochondrial function. <br />                                                                                                       ymel mutants unable grow nonfermentable <br />                                                                                                      carbon sources 37&#194;&#176;C. second phenotype lethality                                                                                                      cells carrying ymel nuclear mutation rho- <br />                                                                                                      (cytoplasmic petite) deletion mutations affecting mito- <br />                                                                                                      chondrial genome. phenotypes, caused                                                                                                      loss Ymelp, result decreased integrity <br />                                                                                                      (increased leakiness) mitochondrial compartment. <br />                                                                                                         rho- cells carry mitochondrial protein <br />                                                                                                      synthesis.  mitochondrial inner membranes <br />                                                                                                      lack major protein components (reviewed refer- <br />                                                                                                      ences 6 33), condition likely result decreased <br /> tract; 6, PTY44/pPT48 (YME1 multicopy vector) cell                                          mitochondrial integrity.  observed extract; 7, crude cell extract; 8, postmitochondrial supernatant; 9,                                 DNA escapes rho- mitochondria 30-fold-higher <br /> purified mitochondrial pellet; 10, Percoll gradient-purified mitochon-                               rate rho&apos; organelles (44). Intact mitochondrial <br /> dria. crude cell extract lane 7, postmitochondrial super-                                 compartments required viability S. cerevisiae (1). <br /> natant lane 8, mitochondrial pellet lane 9 prepared                                interpretation synthetic lethality rho- concentrations soluble protein protein                                     ymel double mutants combined absence associated mitochondria remained constant, equal volumes                                    mitochondrial translation products Ymelp reduces loaded polyacrylamide gel.                                                               mitochondrial integrity threshold necessary                                                                                                      viability. <br />                                                                                                         Ymelp exhibits significant homology family pro- <br />   Ymelp associated mitochondria. generate                                              teins contain sequence elements implicated nucle- <br /> antibody Ymelp, DNA encoding C-terminal 105                                              otide binding. member protein family, mamma- <br /> amino acids Ymelp fused frame 3&apos; end                                         lian p97-ATPase, shown biochemically glutathione S-transferase-encoding gene (41), result-                                        ATPase activity (32). Ymelp homologous ing fusion protein isolated injected rabbits.                                       protein E. coli, encoded ftsH gene, antisera used probe immunoblots total                                               involved formation septum cell division (46). <br /> protein extracts wild-type yeast strain PTY44                                           homologous protein Ymelp yeast ymel-Al::URA3 strain PTY52 elec-                                              protein Secl8p (19% identical Ymelp). Secl8p trophoresed 10% polyacrylamide-SDS gel. protein                                            corresponding mammalian protein, NSF, required expected size 82 kDa detected immunogold                                               membrane fusion events involving endoplasmic reticu- <br /> labeling wild-type extract absent                                              lum, vesicles, Golgi apparatus secretory <br /> ymel-Al::URA3 strain (Fig. 5). 82-kDa protein                                                pathway (7, 49). NSF assembled membrane-associ- <br /> restored ymel-Al::URA3 strain introducing                                                ated &quot;fusion machine&quot; containing components <br /> plasmid expressing YME1 gene.  antisera raised                                          (4; reviewed references 35 48). fusion protein detect Ymelp crude extracts                                            large yeast proteins family, Paslp <br /> S. cerevisiae. interesting YMEI gene                                          Cdc48p, closely related Ymelp. carried multicopy plasmid, apparently                                               functions defined Secl8p, increase Ymelp present cell.                                                 appear involved organellar assembly    ymel mutations cause phenotypes associated                                                rearrangement. Paslp required assembly perox- mitochondria, likely Ymelp                                            isomes (9). Cold-sensitive mutations affecting Cdc48p block <br /> mitochondrial protein. test idea, crude extracts                                          spindle pole body duplication nuclear division (11). <br /> wild-type yeast cells fractionated mitochondrial                                              homology Ymelp FtsH protein, pellets postmitochondrial supernatants. fractions                                          lesser extent Secl8p, suggests hypothesis electrophoresed blotted probed                                           Ymelp involved promoting efficient mitochon- antisera. Ymelp depleted                                           drial fusion division. hypothesis easily <br /> postmitochondrial fraction enriched mitochondrial                                         account mitochondrion-related phenotypes fraction (Fig. 5). confirm mitochondrial location                                          ymel mutants described  absence Ymelp, <br /> Ymelp, mitochondrial pellet resuspended sub-                                             organellar fusion division structures tightly <br /> jected buoyant density centrifugation Percoll gradi-                                         sealed. example, breaches mitochondrial membranes <br /> ent. gradient-purified mitochondria contained                                            aberrant fusion division events allow DNA <br /> Ymelp (Fig. 5).                                                                                      leak cytoplasm. fourth known phenotype                                                                                                      ymel mutants cold-sensitive growth rich glucose- <br />                                              DISCUSSION                                              containing medium (Fig. 1) medium containing <br />                                                                                                      nonfermentable carbon sources (44).   Mutations gene YME1 isolated screening                                              defect mitochondrial division assumed respiration-competent cells 30&#194;&#176;C exhibited                                            low temperatures division mitochondria lacking Ymelp <br /> increased rate (25 times higher wild-type rate                                          slow pace rapid cell growth glucose <br /> ymel-1) DNA escape mitochondria nucleus                                               checkpoint function held cell division  <br />  <br /> completion mitochondrial division (17). total block                <br /> mitochondrial division expected produce lethal               <br /> phenotype (27, 28, 51).                                                    <br />    role Ymelp play mitochondrial function,         polypeptide appears multimeric complex.        <br />                                                                     observed intragenic complementation ap-               <br /> parent missense allele ymel-1 uncharacterized               <br /> ymel-3 mutation, suggesting functional unit contains              copy Ymelp. addition, elevated gene         <br /> dosage YME1 did cause overaccumulation               <br /> protein, suggesting level complex containing       <br /> Ymelp limited level component               unassembled Ymelp degraded. previous                   <br /> observation aymel yme2 double mutant synthetic          <br /> nonrespiratory phenotype (44) suggests possibility               YME2 product complex.                           <br />    novel consequence ymel mutations                       <br /> convert petite-positive yeast S. cerevisiae respiring          <br />  permissive temperatures) petite-negative strains pre-             <br /> venting growth rho- cells. property useful              genetic screens selections             <br /> frequent rho- mutations represent background                   easily eliminated. opi (pet9) mutation kills rho-            <br /> cells.  mutation, inactivates major               <br /> mitochondrial <span id='am-9' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span>-ATP translocator, prevents respira-                <br /> tory growth (20, 22). Interestingly, recent evidence                  indicates differences petite-positive                 petite-negative yeasts great. Nuclear                <br /> mutations selected petite-negative yeasts                  <br /> Schizosaccharomyces pombe (15) Kluyveromyces lactis                     <br /> (2) allow tolerate loss mtDNA.                    <br />   <br /> </body></html>