<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Nov. 1990, p. 5707-5720                                                                 Vol. 10,  11 <br /> 0270-7306/90/115707-14$02.00/0 <br /> Copyright C) 1990, American Society Microbiology <br />  <br />  <br />  Phenotype Minichromosome Maintenance Mutant mcm3 <br />      Characteristic Mutants Defective DNA Replication <br />                        SUSAN        GIBSON, RICHARD T. SUROSKY,t BIK-KWOON TYE* <br />                                        Section Biochemistry, Molecular Cell Biology, <br />                                             Cornell University, Ithaca, New York 14853 <br />                                             Received 8 1990/Accepted 31 July 1990 <br />  <br />            MCM3 essential gene involved maintenance minichromosomes yeast cells. encodes          protein 971 amino acids shows striking homology Mcm2 protein. mapped mcm3-1 <br />          mutation left arm chromosome V approximately 3 kb centromere proximal anpl. mcm3-1 <br />          mutant thermosensitive growth. permissive growth conditions, defective          minichromosome maintenance autonomously replicating sequence-specific manner showed          increase chromosome loss recombination. nonpermissive conditions, mcm3-1 exhibited cell cycle <br />          arrest phenotype, arresting large-bud stage undivided nucleus DNA content nearly <br />          2n. phenotypes consistent incomplete replication genome mcm3-1 mutant, possibly <br />          result limited replication initiation selective autonomously replicating sequences leading cell cycle <br />          arrest mitosis. phenotype exhibited mcm3 mutant similar mcm2, suggesting <br />          Mcm2 Mcm3 proteins play interacting roles DNA replication. <br />  <br />  <br />    Eucaryotes contain multiple chromosomes, replica-                  Gel retardation filter binding assays used tion initiating numerous sites chromosome                 identify factors bind ARSs. ABF1, identified (24). yeast Saccharomyces cerevisiae, total                 affinity ARSI, binds subset ARSs number replication origins estimated electron            sequences associated ARSs (8, 15). OBF1 <br /> microscopy fiber autoradiography 160              (19, 20), identified affinity ARS121, DNA- <br /> 400 haploid genome (45, 50, 51). Autonomously replicat-             binding specificity similar ABF1, suggesting ing sequences (ARSs) allow extrachromosomal                    ARS-binding proteins (S. Eisen- <br /> maintenance sequences contain                    berg, personal communications). Analysis OBF1- <br /> isolated (28, 63). Studies using electron microscopy (55)           binding site ARS121 indicates site acts dimensional gel systems (5, 30, 31) indicate ARSs              enhancer DNA replication way stimulation <br /> serve vivo origins replication plasmids              origin function provided synthetic OBF1- <br /> chromosomes.                                                            binding site independent orientation distance 5&apos;    <span id='am-7' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> sequences 20 ARSs                   ARS consensus sequence (71). role ABF1 determined groups (3, 34, 68). Comparison               OBF1 DNA replication unclear, deletion nucleotide sequences ARSs indicates                binding sites number ARSs causes small T rich contain 11-bp consensus                    decrease plasmid stability (35, 59, 72). Little known <br /> sequence (6). Deletion analysis ARSs                   ARS-binding protein, ABF2, binds shows consensus sequence essential                  sites ARSI undefined DNA- <br /> sufficient ARS activity (3, 9, 34). Flanking sequences 3&apos;           binding specificity (15). 5&apos; consensus sequence play auxiliary roles              taken genetic approach identify factors initiation DNA replication ARSs.                interact ARSs (40). method involves screening flanking sequences different ARSs analyzed,             mutants unable stably maintain minichromosomes lack significant secondary structures direct           contain ARS centromere. Minichromosome inverted repeats. Experiments reconstruct functional              maintenance-defective (mcm) mutants representing 18 dif- <br /> ARSs plasmids suggest flanking sequences                   ferent complementation groups isolated facilitate melting bending ARSs important             independent screening procedures  work; 22, 40). initiation DNA replication ARSs (69, 73). Local               focused study characterization mcm <br /> melting DNA replication origins initiator auxil-           mutants, mcml-l, mcm2-1, mcm3-1, comple- <br /> iary proteins demonstrated studies repli-            mentation groups. mutants affect minichromosome <br /> cation origins Escherichia coli (1), bacteriophage lambda            maintenance ARS-specific manner. (17), simian virus 40 (13). obvious key           complementation groups isolated inde- understanding replication initiation ARSs identify          pendent mutant screens, involving minichromosome <br /> initiator auxiliary proteins bind ARSs.                     carrying different ARS, mutants similar <br />                                                                         ARS specificities. words, ARSs confer stability <br />   * <br />     Corresponding author.                                               minichromosome mutant strains   t Present address: Plant Research Laboratory, Michigan State          confer minichromosome stability strains. Simi- <br /> University, East Lansing, MI 48824.                                     larly, ARSs fail confer minichromosome stability   t Present address: Department Molecular Genetics Cell          strains fail strains. Biology, University Chicago, Chicago, 60637.                      ARS specificities mutants suggested                                                                  5707 <br />  5708        GIBSON ET AL.                                                                                                MOL. CELL. BIOL. <br />  <br />                                                               TABLE 1. Yeast strains <br />        Strain                                                    Description genotype&apos;                                       reference <br />  <br /> 8534-8C                  leu2-3,112 ura3-52 his4-A34                                                                        G. Fink <br /> 8534-10A                 leu2-3,112 ura3-52 his4-A34                                                                        G. Fink <br /> A2                       leu2-3,112 his3-11,15                                                                              G. Fink <br /> A3                       leu2-3,112 his3-11,1S                                                                              G. Fink <br /> F808                     leu2-3,112 ura3-52 his4-519 adel-100 GAL]                                                          G. Fink <br /> Y133                     ura3-52 his3-A200 Iys2-801 ade2-101 Atrpl                                                          R. Davis <br /> Y134                     ura3-52 lys2-801 ade2-101 Atrpl tyri                                                               R. Davis <br /> K396-27B                 leul ura3-1  adel lys7 met3 AtrpS                                                              S. Klapholtz <br /> F483                     ura3 his4 Iys2 thrl adel cdc6 <br /> 6697-1                   met4 <br /> B5816                    ura3-1 leul-12 trp2-1 lys2-1 cycl CYC7H3                                                           42 <br /> SUH-H2                   met3 MAL2 trpl URA3::his4                                                                          64 <br /> DBY4962                  ade2 his3-A200 ura3-52 leu2-AJOJ::URA3::leu2-A102 lys2-AJO::HIS3::Iys2-A102                        C. Chan <br /> R61b                     leu2-3,112 ura3-52 his4-A34 mcm3-1                                                                 study <br /> R61-lDc                  leu2-3,112 ura3-52 his4-A34 mcm3-1                                                                 study <br /> R61-2Bd                  leu2-3,112 ura3-52 his3-11,15 mcm3-1                                                               study <br /> tBR61-2B                 R61-2B 1 copy M3LL (-+4100) (LEU2 MCM3) integrated mcm3 locus                         study <br /> R61-3Cd                  leu2-3,112 ura3-52 his4-A34 mcm3-1                                                                 study <br /> tAR61-3C                 R61-3C copies M3LL (-+4100) (LEU2 MCM3) integrated mcm3 locus                 study <br /> tBR61-3C                 R61-3C YIp5/MCM3 (URA3 MCM3) integrated mcm3 locus                                       study <br /> RY2Be                    ura3-52 his3-A200 ade2-101 mcm3-1                                                                  study <br /> RY2Ce                    leu2-3,112 ura3-52 his3-A200 his4-A34 lys2-801 ade2-101 Atrpl mcm3-1                               study <br /> RY6Ae                    ura3-52 his4-A34 ade2-101 Atrpl                                                                    study <br /> RY71Ae                   ura3-52 leu2-3,112 his3-A200 lys2-801 ade2-101 Atrpl                                               study <br /> RY8Ae                    ura3-52 his3-A200 lys2-801 ade2-101 mcm3-1                                                         study <br /> RY2B/Y4                  Diploid formed crossing RY2B Y134                                                            study <br /> RY6A/Y4                  Diploid formed crossing RY6A Y134                                                            study <br /> RY6A/8A                  Diploid formed crossing RY6A RY8A                                                            study <br /> RY2B/2C                  Diploid formed crossing RY2B RY2C                                                            study <br /> RY2B/8A                  Diploid formed crossing RY2B RY8A                                                            study <br /> RS2Af                    ura3-52 met3 trpl his4:: URA3 mcm3-1                                                               study <br /> RSlOlBf                  ura3-52 his4:: URA3 mcm3-1                                                                         study <br /> RS2A/R2                  Diploid formed crossing RS2A R61-2B                                                          study <br /> RS2A/tBR2                Diploid formed crossing RS2A tBR61-2B                                                        study <br /> RS1O1B/R3                Diploid formed crossing RS1O1B R61-3C                                                        study <br /> RS1O1B/tAR3              Diploid formed crossing RS1O1B tAR61-3C                                                      study <br /> 8C/1OA                   Diploid formed crossing 8534-8C 8534-1OA                                                     study <br /> 8C/SU                    Diploid formed crossing 8534-8C SUH-H2                                                       study <br /> 1OA/RY71A                Diploid formed crossing 8534-1OA RY71A                                                       study <br /> tA1OA/RY71A              1OA/RY71A 1 copy MCM3 gene replaced interrupted MCM3 gene M3L <br />                            (-+500) = LEU2::mcm3 <br />  aFor descriptions plasmids yeast strains, Table 2. <br />  b <br />    Original mcm3-1 mutant isolated ethyl methanesulfonate mutagenesis 8534-8C. <br />  c Segregant cross R61 x A3. <br />  d Segregant cross R61-1D x A2. <br />    Segregant cross R61-3C x Y133. <br />  f Segregant cross R61-3C x SUH-H2. <br />  <br />  <br />  <br /> MCM gene products play role minichromosome repli-                                      MATERIALS METHODS <br /> cation segregation functions required stability.                                                                      Strains plasmids. E. coli HB101 (hsdS20 recA13) <br />    Analysis properties functions MCMJ                       used routine bacterial transformations plasmid <br /> gene product indicates multifunctional protein.                  preparations. E. coli JM101  lac-proAB) F&apos; traD36 proAB <br /> Aside involvement ARS function, Mcml acts                     lacIq ZAM15] used phage transformations transcription factor (48, 49). respect, similar            preparation single-stranded DNA (ssDNA). E. coli RR1 transcriptional silencer proteins ABF1 RAP1 (8, <br /> 59), striking sequence homology                      (hsdS20) used producing TrpE-Mcm3 fusion <br /> (16). involved repression yeast                   protein. E. coli RDP146 [recA Spr lac-pro)] NS2114 <br /> silent mating-type loci shown bind                    Sm (recA Smr, lambda cre prophage chromosome) ARSs associated loci. known                      used shuttle mutagenesis (57). S. cerevisiae strains <br /> Mcm2 Mcm3 proteins. report describes char-                       used listed Table 1. Table 2 describes plasmids <br /> acterization mcm3-1 mutant analysis MCM3                    used. <br /> gene. Like mcm2-1 (61), mutant shares                          Media, chemicals, enzymes. T4 DNA polymerase, <br /> properties characteristic mutants defective                    terminal deoxyribonucleotidyltransferase, subclon- aspect DNA replication.                                               ing primer RD22 (International Biotechnologies, , New <br />  VOL. 10, 1990                                                            PROTEIN INVOLVED ARS FUNCTION                 5709 <br />  <br />                      TABLE 2. Plasmids                           lyzed using Ortho Diagnostics 2151 machine <br />                                                                  488-nm excitation. <br />         Plasmid        Relevant sequences        Reference <br />                                                     source <br />                                                                  Fluctuation analysis (chromosome loss recombination <br />                                                                  assay). diploid strain used contains single copy YCpl              ARSI CEN5 LEU2 URA3        40                  URA3 gene, integrated gene replacement his4 locus <br /> YCp131            ARSJ31 CEN5 LEU2           40                  left arm chromosome III. single colony, isolated <br /> YCp121            ARS121 CEN5 LEU2           40                  plate containing complete medium lacking <span id='am-39' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span>, YCp131C           ARS131C CEN5 LEU2 URA3 40 <br /> YCplOl            ARSJ CEN5 LEU2             laboratory     suspended water plated single colonies YCp2B             ARS2B CEN5 LEU2 URA3       study          YEPD plates. Plates incubated 14, 23, 30&#194;&#176;C pYES3             ARS1 CEN5 URA3             41                  approximately 25 generations cell growth  average <br /> YCp5O             ARSI CEN4 URA3             54                  colony contained approximately 5 x 107 cells). Colonies <br /> YCp5O bank        Yeast DNA bank YCp5O    54                  picked, suspended water, spread 5-fluoro- <br /> YIp5              URA3                       63                  <span id='am-2' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-3' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span>orotic acid</span> (FOA) (2) YEPD plates. total number R61-1             ARSI CEN4 URA3 MCM3        study          cells colony measured plating efficiency  R61-4             ARSI CEN4 URA3 MCM3        study          cases, direct counting cells suspension. <br /> YIp5/MCM3         URA3 MCM3                  study          number Ura- cells determined number <br /> pAB83             URA3 ANPI                  42 <br />                                                                  colonies grew FOA plate. observed <br /> pATH3             Amino terminus E. coli   Tzagoloff <br />                     trpE                                         recombination frequencies multiplied 2 p3MCM3HH          Expresses TrpE-Mcm3 fusion study          half, average, recombination events led loss                     protein E. coli                           URA3 marker daughter cells. rates <br /> p3MCM3BC          Expresses TrpE-Mcm3 fusion study          chromosome loss mitotic recombination cell <br />                     protein E. coli                           division determined method median pCGS286           GAL] UAS fused lacZ,    23                  described previously (39). Final values rates                     2p.m ARS URA3                                chromosomal loss recombination results 18 B620              GAL] UAS 2,um ARS URA3       23                20 independent assays. <br /> B620/MCM3         GAL] UAS fused MCM3,      study           Isolation mcm3 mutant. mcm3 mutant                     21Lm ARS URA3 <br /> pLB101            Cmr, Tn3 transposase         57                isolated using slightly modified version mutant <br /> pOX38::m-Tn3      Apr, mini-Tn3                57                screen used Maine et al. (40). minichromosome <br /> pHSS6             Kmr ori                      57                YCp2B (47) used instead minichromosome YCpl <br /> pHSS6/MCM3        MCM3                         study        YCp131C, mutant screen conducted 26 <br /> M3LL(- +4100)     pHSS6/MCM3 minitrans-   study        instead 30&#194;&#176;C allow isolation mutants                     poson m-Tn3(LEU2 lacZ)                       temperature sensitive growth. <br />                     inserted 3&apos; MCM3 ORF                         DNA RNA hybridizations radioactively labeled DNA <br /> M3L(-+500)        pHSS6/MCM3 minitrans-   study        probes. DNA transferred agarose gels nitrocel- <br />                     poson m-Tn3(LEU2)                         lulose filters described Southern (62) Biotrans <br />                     serted MCM3 <br />                     ORF                                          (Biodyne) filters suggested Pall Ultrafine Filtration <br />                                                                  Corp., East Hills, N.Y. RNA denatured glyoxal, <br />                                                                  run agarose gels, transferred Biotrans filters (65). <br />                                                                  Hybridizations filters carrying RNA performed Haven, Conn.)   used make deletions DNA cloned        described manufacturer. 32P-labeled double-stranded M13-based vectors.                                            DNA probes prepared nick translation (53). 32p_ <br />    Genetic techniques DNA transformation. Meiotic tet-       labeled ssDNA probes prepared using hybrid- <br /> rad analysis described (58). Transforma-      ization probe primer protocol (29). <br /> tion yeast cells spheroplasting method           Sequencing construction clones sequencing. (27).                                                            strands DNA fragment contains MCM3 gene <br />    Mitotic plasmid stability assay. Stability assays measure     plus base pairs flanking DNA cumulative loss minichromosome yeast             sequenced  Overlapping DNA fragments culture generations growth nonselec-      MCM3 gene subcloned M13 vectors mpl8 <br /> tive media. Single colonies, containing i107 cells ml,       mp19 (46), T4 DNA polymerase used picked plates containing selective medium used      construct series deletions MCM3 DNA inoculate 5-ml samples nonselective medium (YEPD).         fragments (11). constructs sequenced cultures grown saturation (approximately       dideoxynucleotide chain termination method (56). <br /> generations growth) diluted water, samples          Isolation TrpE-Mcm3 hybrid protein preparation plated YEPD plates. plates             anti-Mcm3 antibodies. E. coli RR1 cells transform- <br /> replica plated selective medium. Minichromosome sta-        ed trpE-MCM3 fusion gene p3MCM3HH bility expressed percentage cells YEPD       p3MCM3BC. Induction partial purification TrpE <br /> plates grew selective plates (40).                       hybrid proteins described (14, 36).    minichromosome loss rate determined 1 -             generate antibodies, TrpE-Mcm3 hybrid protein  <br /> (FI)l/N, initial percentage plasmid-contain-   tains 37 kDa amino terminus TrpE protein <br /> ing cells F percentage plasmid-containing cells    fused -60 kDa Mcm3 protein purified E. N generations.                                             coli extracts used immunize rabbits. giant <br />    DAPI staining. Cell staining DNA specific dye        Flemish/chinchilla rabbits immunized 0.4 1.1 <br /> 4,6-diamidino-2-phenylindole (DAPI) carried ac-          mg TrpE-Mcm3 hybrid protein Freund complete <br /> cording published procedures (74).                            adjuvant (70). rabbits boosted twice, 1 6 <br />   Flow cytometry. Cells stained propidium iodide ac-        months initial injection, Freund incomplete <br /> cording method Hutter Eipel (32) ana-         adjuvant 0.3 1.0 mg TrpE-Mcm3 hybrid protein. <br />  5710       GIBSON ET AL.                                                                                                                   MOL. CELL. BIOL. <br />  <br />         TABLE 3. Stabilities minichromosomes carrying different ARSs mutant wild-type strains various temperatures <br />                                                                                      Stability given YCp minichromosomeb <br />         Strainsa                 Genotype            Temp (&#194;&#176;C) <br />                                                                                           1               131              121            131C               2B <br /> R61-2B, R61-3C                      mcm3                             14                  0.3              0.3              33               31 <br />                                                                      23                  0.6              0.2              21               17             &lt;0.2 <br />                                                                      30                  0.1            &lt;0.2               10                2 <br /> 8534-8C, A2                         MCM3                             14                 90               66                91               97 <br />                                                                      23                 89               76                98               96              94 <br />                                                                      30                 80               62                90               93               - <br /> RY2B/2C, RY2B/8A                    mcm3/mcm3                        23                  0.2                               -                34 <br /> RY3BIY4, RY6A/8A                    mcm3lMCM3                        23                 52                                                  98 <br /> RY6A/Y4, 8C/10A                     MCM31MCM3                        23                 88                                                  87 <br />   &apos; Results pair strains genotype averaged. <br />   b Expressed percentage cells carrying tester plasmid generations nonselective growth. YCp2B assays carried 26&#194;&#176;C, <br /> using strains R61-3C 8534-8C. -, determined. <br />  <br />  <br /> Serum isolated blood samples taken 11 days                          &lt;0.8% mutant strain. minichromosomes, pYES3 second boost.                                                                  YCp101, carry ARS centromere   Immunoblot analyses. Immunoblots performed                               different selectable markers (URA3 LEU2, respectively) <br /> using published procedures (67). Protein preparations                         similar stabilities &lt;0.8% mutant strain. <br /> run 7% sodium dodecyl sulfate polyacrylamide gels                           ability mutant strain differentiate transferred Tris-<span id='am-15' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span> buffer  methanol) nitrocel-                      minichromosomes depending ARS carry <br /> lulose filters pore size 0.1 ,um. Transfers                         implies defect acts near ARS. <br /> typically carried 0.5 2.5 3 h. Western                                approximate loss rates minichromosomes YCpl <br /> immunoblotting-grade goat anti-rabbit immunoglobulin G-                            YCp131C determined mutant (R61-3C) horseradish peroxidase conjugate (Bio-Rad Laboratories,                            wild-type (A2) strains growing 14&#194;&#176;C. YCpl lost Richmond, Calif.) normally used second anti-                            rate 1% generation wild-type strain body.                                                                              34% generation mutant strain. YCpl instability <br />   Yeast protein preparations. Small-scale (75) large-                          mutant strain solely failure replicate, scale (33) yeast protein preparations carried                          34% loss rate correspond 68% YCpl- <br /> using published procedures. Protein concentrations                            containing cells culture failing replicate determined method Bradford (4).                                          minichromosome generation  opposed 2%   Overexpression Mcm3 protein. Induction GAL]                            wild-type cells failing replicate YCpl minichro- <br /> upstream activator sequence (UAS) described                               mosome generation). comparison, mcm3-1 (75). brief, yeast strain F808 containing                         mutation small effect YCp131C, plasmid B620/MCM3 (URA3 2,im ARS GALI-MCM3)                                    loss rate 4% generation mutant strain;  <br /> grown early log phase complete medium lacking uracil                         significant increase 0.6% containing 3% glycerol 2% ethanol. Galactose                           generation loss rate wild-type strain. added final concentration 2%, cells                             effect mcm3 mutation minichromosome <br /> grown additional 10 24 h harvesting.                              maintenance temperature dependent: higher growth <br />   Nucleotide sequence accession number. GenBank ac-                            temperature enhanced mutant phenotype. Studies cession number MCM3 gene X53540.                                        MCM3Imcm3 heterozygous diploid strains indicated                                                                                    mcm3 mutation caused slight, ARS-dependent semidomi- <br />                             RESULTS                                                nant phenotype. Homozygous mutant wild-type diploid <br />    mcm3-1 mutation different effects function-                      strains maintained minichromosomes rates similar ing ARSs. mcm3 mutant originally identified                          corresponding haploid strains (Table 3). inability stably maintain minichromosome YCp2B,                            mcm3-1 mutant cells contain fully fully replicated contains ARS located 3&apos; histone H2B gene                          DNA arrest 37&#194;&#176;C. addition minichromo- <br /> (47). examine minichromosome maintenance                            maintenance defect, strains carrying mcm3-1 mu- <br /> defect mcm3 mutant, stabilities minichromo-                          tation temperature sensitive growth. grew somes carrying different ARSs (Table 3), centromeres,                          approximately wild-type rates 22&#194;&#176;C stopped growing <br /> selectable markers measured mcm3 mutant                                soon shift 38&#194;&#176;C (Fig. 1). arrested cells wild-type strains.                                                                 cell division cycle (cdc) phenotype; -95% cells <br />    minichromosomes showed decreased stabil-                           arrested large-bud shape. mcm3 mutant cells showed <br /> ity mcm3 mutant strain, affected                        imperfect cell-cycle arrest, approximately half  instability appeared determined                          cells taking cycles arrest. arrest, ARS centromere selectable marker                         occurred, irreversible (Fig. 1). Staining arrested cells <br /> contained minichromosome. instance,                                 DNA-specific dye DAPI revealed majority <br /> YCp121 YCp131 minichromosomes differ                               arrested cells contained single nucleus ARS contain, 14&#194;&#176;C, YCp121 33% stable                          generally located neck bud (Fig. 2). mutant strain YCp131 0.3% stable (Table                           determine mutant cells replicate 3). contrast, YCpSO pYES3, contain                           DNA cell cycle arrest 37&#194;&#176;C, flow cytometry exper- <br /> ARS yeast selectable marker different centromeres                          iments performed. Wild-type mutant cells growing <br /> (CEN4 CENS, respectively), similar stabilities                          rich medium (YEPD) 21&#194;&#176;C shifted 37&#194;&#176;C. <br />  VOL. 10, 1990                                                                      PROTEIN INVOLVED ARS FUNCTION                        5711 <br />  <br />  <br />                                                                                                                B (-) <br />               E~100 <br />                <br />                                                                             4                                   Ohrs <br />  <br />               -J <br />               -J                                                           C.) <br />               0 <br />                     107                                                    0 <br />                                                                                            6hrs                 6hrs <br />                                                                            0) <br />                                                                            .c <br />                                                                                                             HOURS                                      <br />                                                                            c             Increasing DNA content                     w <br />                                                                              FIG. 3. DNA content mcm3 mutant cells arrested growth                                                      380C                 37&#194;&#176;C. mcm3 mutant wild-type cells grown early log phase <br />                                                                           rich medium (YEPD) 21&#194;&#176;C shifted growth 37&#194;&#176;C. <br />                                                                           Samples removed cultures 0 6 h                                                                           temperature shift. cell densities different cultures ranged <br />                                                                           1.5 x 106 9 x 106 cells ml. Cells stained         C,n <br />         -J <br />                                                                           nucleic acid-specific fluorescent dye propidium iodide  treating <br />                   wL                                                               RNaseA degrade RNA) analyzed using flow <br />                                                                           cytometer. results plotted relative DNA                                                                           cell versus number cells. vertical dotted lines correspond <br />                                                                           -ln 2n amounts DNA. following strains used: <br />                                                                           RY8A (mcm3)  , R61-3C (mcm3) (B), DBY4962 (MCM3) (C). <br />                                                         30 <br />                                HOURS                                      Samples culture removed 0 6 h                                                                           shift 37&#194;&#176;C, treated RNaseA, stained nucleic <br />   FIG. 1. mcm3 mutant wild-type growth curves. mcm3 mu-               acid-specific fluorescent dye propidium iodide, run <br /> tant (R61-3C) wild-type (8534-8C) cultures grown log          flow cytometer measure DNA content phase rich medium (YEPD) 22&#194;&#176;C diluted -5.0 x 106        individual cells. wild-type strain maintained bimodal <br /> cells ml fresh YEPD. Half culture grown additional 22.5 h 22&#194;&#176;C, half grown         DNA content profiles ln 2n 0 6 h shift <br /> 38&#194;&#176;C. Cell densities measured periodically direct counting.       37&#194;&#176;C. contrast, 6 h shift 37&#194;&#176;C, majority number viable cells culture determined plating      cells mutant cultures arrested fully efficiency. Symbols: 0, R61-3C, total cell density; 0, R61-3C,            fully replicated DNA (Fig. 3), suggesting cells <br /> viable cells;  8534-8C, total cell density;  8534-8C, viable cells.   unable complete S phase enter mitosis. <br />  <br />  <br />  <br />  <br />   FIG. 2. Nuclear staining arrested growing mcm3 cultures. mcm3 strain R61-3C used. log-phase culture growing rich <br /> medium (YEPD) 23&#194;&#176;C diluted rich medium divided portions. grown additional 7 h rich medium, 23&#194;&#176;C   38&#194;&#176;C (B), reached cell densities approximately 1.8 x 10&apos; cells ml. cells growing 38&#194;&#176;C stopped dividing 5 h shifted 38&#194;&#176;C. cells stained DNA-specific dye DAPI visualized using fluorescence microscope. <br />  5712    GIBSON ET AL.                                                                                             MOL. CELL. BIOL. <br />  <br />  <br />                          Chromosome loss                                   Recombination <br />                                                                                            ML~~~~~~~~~  <br />            Xj    0~~~~~~~~~~~~~~~~~        30 <br />                                            2: 140C 24&#194;&#176;C 300 <br />                  0)~~~~~~~~~~~~~~~~ <br />                                                                                                                    -5 <br />  <br />             C: <br />             0)   16                                                                                                     C <br />             0) <br />  <br />  <br />  <br />             0 <br />                  0 <br />                                                                                                                         ~~~~~~~~~~~~~~~~~ <br />                                                                                                                         U) <br />  <br />  <br />  <br />  <br />   FIG. 4. Chromosome loss recombination mcm3 mutant wild-type strains. rates chromosome III loss recombination his4 CEN3 measured wild-type, heterozygous, homozygous mcm3 diploid strains different growth <br /> temperatures. results heterozygous strains averaged graphed, results mcm3 <br /> homozygous strains. Symbols: 0M, 8C/SU (MCM31MCM3); C], RS2A/tBR2 RS1O1B/tAR3 (mcm3/mcm3 strains transformed MCM3 gene); *, RS2A/R2 RS1O1B/R3 (mcm3lmcm3). <br />  <br />    mcm3 mutant cells increased rate chromosome           lethal effects UV light (S. Gibson, Ph.D. thesis, <br /> loss recombination. mcm3-1 mutation affects           Cornell University, Ithaca, N.Y., 1989). stability minichromosomes, possible             Isolation MCM3 gene. Characterization stabilities normal chromosomes affected mcm3         mcm3-1 mutant suggests defect affects aspect <br /> mutant strains. test chromosome stability, number           DNA replication. gain insight function <br /> wild-type, heterozygous, mcm3 homozygous diploid                Mcm3 protein, necessary clone MCM3 <br /> strains constructed. strains homozy-          gene. scheme used clone MCM3 gene involves <br /> gous ura3-52 mutation. addition, strain             simultaneously selecting complementation contains copy URA3 gene integrated           temperature-sensitive growth Mcm- phenotypes copies chromosome III, gene replacement his4           mcm3 mutant strain transformed locus left arm (64). ura3 cells grow       wild-type yeast DNA bank. mcm3 mutant strain, R61- <br /> media containing FOA, integrated copy URA3              3C, transformed uracil prototrophy yeast DNA <br /> gene, FOA, provides means monitoring             bank cloned vector YCp5O (54) leucine pro- <br /> chromosome loss recombination events.                           totrophy second tester plasmid, YCp1O1.    strains grown approximately 25 genera-            YCp5O YCplO1 plasmids carry ARS1, poorly <br /> tions complete medium, allowing cells lose URA3           utilized mcm3 mutant strain (Table 3).  <br /> gene continuing grow divide. cells            mutant cells contain copy MCM3 gene plated YEPD medium containing FOA             identified virtue ability grow 35&#194;&#176;C determine percentage cells lost URA3            maintaining plasmids. <br /> marker. mating type strains checked           using foregoing strategy, overlapping yeast DNA <br /> distinguish loss URA3 marker recom-           fragments complement Mcm- tempera- <br /> bination loss entire chromosome. Cells lose          ture-sensitive growth phenotypes mcm3 mutant strain copy URA3 gene result chromosome loss          isolated. confirm cloned yeast DNA inserts lose MATax allele right arm, giving rise         did contain MCM3 gene, DNA fragment  <br /> diploids mate. contrast, cells lose URA3 gene      taining complementing activity integrated normal recombination URA3 CEN3 retain             chromosomal location mcm3 strain. crossing MAT alleles unable mate.                            strain wild-type strain, segregation Mcm <br />    mcm3 homozygous strains tested showed                 phenoiype analyzed. 14 tetrads examined yielded <br /> creased levels chromosome III loss recombina-           spores wild-type Mcm phenotypes, indicating <br /> tion. effect temperature dependent, rates         recombination events took place puta- chromosome loss recombination approximately            tive MCM3 gene mcm3 locus (Gibson, Ph.D. thesis, <br /> wild type 14&#194;&#176;C, slightly elevated 24&#194;&#176;C, significantly     1989). result indicated DNA fragment elevated 30&#194;&#176;C (Fig. 4). 30&#194;&#176;C, mitotic recombination           complementing activity contained MCM3 gene. <br /> occurred 80-fold frequently chromosome loss                   MCM3 gene maps position 3 kb centromere <br /> occurred 14-fold frequently mutant          proximal ANPI left arm chromosome V. wild-type heterozygous strains. phenotype             MCM3 gene lie near ANPI locus heterozygous strains indicated effect mutation      left arm chromosome V probing yeast chromo- chromosome loss recombination recessive.             blot (kindly supplied H. Klein) MCM3 gene <br /> hyperrecombination chromosome loss phenotype             examining segregation mcm3, URA3 inte- <br /> mcm3 mutant probably result defect DNA          grated MCM3, respect , ura3, anpl, adel, <br /> repair, mcm3 mutant cells wild-type response        met3, leu2 markers (Table 4). results  <br />  VOL. 10, 1990                                                                         PROTEIN INVOLVED ARS FUNCTION                   5713 <br />  <br />  TABLE 4. Mapping position MCM3 yeast genome                       translation begins ATG large ORF, <br />                                    ofV:         Distance markers <br />                                                                               MCM3 gene encodes protein 971 amino acids length <br />     Cross                                                                     (Fig. 6) molecular weight 107 kDa. protein                        PD           NPD      T-          (centimorgans)       predicted negatively charged, cysteine <br /> mcm3b x hisl            8            8       45                76             residues. hydrophilicity plot protein shows mcm3 x ura3             5            0        1                 8             slightly hydrophilic, particularly hydrophilic regions <br /> mcm3b x anpl           10            0        0                 0             located near amino carboxy termini. <br />                                                                                  Mcm3 protein sequence searched similarity  PD, parental ditype; NPD, nonparental ditype; TT, tetratype. <br />  b Determined using URA3 integrated MCM3.                               protein sequences contained data bases compiled                                                                               laboratory R. Doolittle (18) National Biomed- <br />                                                                               ical Research Foundation, Protein Identification Resource, <br /> firmed plasmid, pAB83, containing ANPI gene                        Genentech gene banks. Interestingly, protein flanking DNA (42) cross-hybridize                   significantly similar Mcm3 protein MCM3 gene (Gibson, Ph.D. thesis, 1989). way,                       Mcm2 protein. proteins, defects lead able map MCM3 gene location 3 kb                   remarkably similar phenotypes (61), contain 30% identical <br /> centromere proximal ANPI.                                                  amino acids entire lengths 53% identity   MCM3 gene necessary cell viability. determine                 200-amino-acid region  positions 359 558) MCM3 gene necessary cell growth,                         Mcm3 protein (H. Yan, S. Gibson, B.-K. Tye, unpub- <br /> diploid strain (tA1OA/RY71A) contains wild-type                      lished results). <br /> MCM3 gene interrupted MCM3 gene                                  Located 175 bp 5&apos; ORF, MCM3 gene contains <br /> structed. interrupted gene contains 3.5-kb transposon                   sequence CCTAAATGGG, similar insertion (57) located 0.5 kb downstream                         consensus sequence (CCTAAT T]NGG) DNA binding <br /> putative MCM3 start codon. diploid containing                         MCM gene product, Mcml protein (48). <br /> interrupted MCM3 gene isogenic diploid (10A/                            site apparently essential Mcm3 <br /> RY71A) homozygous wild-type MCM3 gene                            expression, determined transposon insertion mutation <br /> sporulated, resultant tetrads dissected check                 deletion analyses. cloned MCM3 gene containing spore viability. tetrads heterozy-                  transposon insertion mutation position - 126 fails gous MCM3 strain analyzed. 26 tetrads, 6                        complement temperature-sensitive growth phenotype yielded viable spore, 20 yielded viable spores,                   mcm3-1 mutation, insertion mutation yielded viable spores. viable spores                   position - 155 complements mutation (Gibson, Leu- contained uninterrupted copy                      Ph.D. thesis, 1989). sequences 3&apos; bp -156 MCM3 gene. spores failed rise                 confirmed sufficient MCM3 expression high colonies appeared divided twice, indicat-                  level complement mcm3-1 mutation  <br /> ing MCM3 gene necessary cell growth                    structing plasmids contain yeast DNA 5&apos; spore germination. contrast, 29 tetrads                    -106, -126, -155, -197 insertion events. control strain, 23 yielded viable spores, 4                     plasmids integrated direct transformation yielded viable spores, 2 yielded viable spores.                 ura3 locus mcm3 mutant strain R61-3C. integrated <br />   MCM3 gene encodes single large ORF. MCM3                          MCM3 constructs containing deletions - 197 -155, gene localized 3.4-kb DNA fragment testing                        -126 -106, complemented temperature- <br /> various DNA fragments original MCM3 clone                        sensitive growth phenotype caused mcm3-1 mutation. ability complement mcm3-1 mutation.                           results suggest region located 126 <br /> MCM3-containing DNA fragment sequenced                          155 bp upstream ATG (+ 1) long ORF contain single large open reading frame (ORF)                       important MCM3 gene expression. 29-bp se- <br /> extends approximately 3 kb  Fig. 5 restriction                 quence strikingly T-rich region 16 17 <br /> map Fig. 6 complete DNA sequence). Insertions                     nucleotides T (Fig. 6). obvious place E. coli lacZ gene frame MCM3                       TATA element located -155 ATG ORF result production chimeric proteins                        ORF. <br /> P-galactosidase activity, indicating MCM3 ORF                        MCM3 gene produces transcript 3.3 kb. translated yeast cells (Gibson, Ph.D. thesis, 1989). Inser-                MCM3 gene used probe yeast poly  RNA tions MCM3 ORF destroy ability                     Northern (RNA) blot experiment (Fig. 7). probes used <br /> gene complement mcm3-1 mutation (Gibson, Ph.D.                         experiment 32P-labeled MCM3 ssDNA thesis, 1989).                                                                32P-labeled nick-translated YIp5 DNA. single transcript                                                                               approximately 3.3 kb detected MCM3 ssDNA <br />                                                                               probe (Fig. 7). orientation transcript determined <br />                              H3       H3     H3                               ssDNA consistent translation                                                                               large ORF contained MCM3 gene (Fig. 6). Com- <br />                                                                               parison MCM3 transcript levels mcm3 mutant <br />       ML                   ||       I51I I                                wild-type strains suggested mutant (R61-3C) <br />        .4    1.8       1.4         .8 .1 1.6 .3 .4 1.0   1.9   &apos;.    kb       contained twice MCM3 transcript, relative                                                                               level ura3-52 transcript, did wild-type strain <br />   FIG. 5. MCM3 restriction map. R61-1 consists yeast DNA                 (8534-8C). <br /> fragment partially digested Sau3A, cloned BamHI site YCp5O. selected restriction sites shown vector               S1 mapping experiments conducted determine sequences plasmid. Symbols:  vector sequence; _,                    transcript start sites. Poly  RNA isolated mutant <br /> MCM3 ORF; +-, transcript orientation. Restriction sites: B,                   (R61-3C) wild-type (8534-8C) strains. major start <br /> BamHI; (B), BamHI site destroyed plasmid construction;                 sites transcripts produced strains Bg, BglII; C, Clal; H, Hindlll; M, MluI; R, EcoRI; S, Sall; St, StuI.         located 50 + 6 bp 5&apos; ATG long ORF. <br />  5714     GIBSON ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />                           1                                                                                        60 <br />                           61   CATCTATATAGCAATAATATACTTTGTTTTTATTCGTATTTCACACTTTTCTTTTTCCTT                        120 <br />                         121    ATGCAGGCAGTGTAATTCATTGGGGAGGATGATTTTCATGTGCGCATATCTACCGGCTGC                        180 <br />                         181    AAGCAGCCGGTCGGTGGCAAATCCGGCGCTTCCCCCTCAAAAAAAAAAAAAAAAAAAAA                         240 <br />  <br />                         241    AAAGGGAACTCTCAGAACGGGGGAGGTTGAAGAGCAGGCCAAGGGAAATATTAGTTTTGA                        300 <br />                         301    CCTATGTGGGAAACAGAATTTTCAATGAGTTATGGCAACTTGGCCGAGTGGTTAAGGCGA                        360 <br />                         361    AAGATTAGAAATCTTTTGGGCTTTGCCCGCGCAGGTTCGAGTCCTGCAGTTGTCGTTATT                        420 <br />                         421    TTTCTCTTTTTTTTCAATTTCCCTTGTTCGTCAGATCGAGGCGGTAGAAGAAACAATTAC    480 <br />                                                          -155(+)       .               -126(-) <br />                         481    TTTTCAAAAACTCGTGTTTTA^&quot;&apos;GGAAAAAAAAGAAAAATTTGGTCAAAACTA 540 <br />                         541    CGAAAGATAGGTTCTTAATCTTCTTTCAAGTTGAAAAGGCCTACGCTCTTTTCCTTGAAG                        600 <br />                         601    CATTTTCATCCTACTGCTCGTATTGAACTCCACTATAAGCGCACCAAAAAGATACAAACG                        660 <br />                                        (+1)) <br />                         661    TCAATTATGGAAGGCTCAACGGGATTTGATGGAGACGCTACTACTTTTTTCGCTCCAGAC                        720 <br />                                      M E G S T G F D G D T T F F P D <br />                         721    GCTGTGTTTGGTGACAGAGTGCGCAGATTTCAAGAGTTTTTAGATACTTTCACCTCATAC                        780 <br />                                V F G D R V R R F Q E F L D T F T S Y <br />                         781    AGAGACTCTGTAAGGTCCATACAAGTTTACAACAGCAATAACGCGGCCAACTACAACGAT                        840 <br />                                R D S V R S Q V Y N S N N N Y N D <br />  <br />                         841                 EDAGACGLAACGAATTTGCGTAGG <br />                                GATCAAGATGACDC                                      DTDATGAC GDC EATGATCGAA <br />                                                                                                   D                900 <br />                                D   Q D D D               E   R   D   L   L   G   D   D   D   G   D   D   L   E <br />                          901   AAGGAAAAGAAAGCAGCATCGTCCACCTCATTGAATATACTCCCTCACAGGATTATCATC                        960 <br />                                K E K K S S T S L N L P H R I                          961   TCGCTTGATGACTTGAGAGAATTCGACAGGTCGTTCTGGTCGGGCATTTTAGTCGAACCA                        1020 <br />                                S L D D L R E F D R S F W S G L V E P <br />                        1021    GCATACTTCATCCCGCCTGCCGAAAAGGCGCTTACTGACCTAGCAGATTCCATGGACGAT                        1080 <br />                                  Y     F       P   P     E   K     L   T   D   L     D   S   M   D   D <br />  <br />                        1081    GTTCCACATCCCAATGCCTCTGCAGTATCGTCTCGCCATCCTTGGAAGCTTTCGTTCAAA                        1140 <br />                                V P H P N S V S S R H P W K L S F K <br />                        1141    GGCTCATTTGGTGCACACGCATTGTCTCCTCGTACTCTAACGGCACAACATTTAAACAAA                        1200 <br />                                G S F G H L S P R T L T Q H L N K <br />                        1201    CTGGTCTCTGTTGAGGGTATCGTAACTAAGACTTCGTTGGTCAGGCCAAAGCTTATCAGA                        1260 <br />                                L V S V E G V T K T S L V R P K L R <br />                        1261    TCTGTCCACTACGCGGCAAAGACTGGTAGATTCCATTACAGAGATTATACAGATGCTACT                        1320 <br />                                S V H Y K T G R F H Y R D Y T D T <br />                        1321    ACAACTCTCACCACCCGCATCCCAACGCTTPACATCTATCCAACGEAGGACACTGAAGGT                        1380 <br />                                T T L T T R P T P Y P T E D T E G <br />    FIG. 6. MCM3 DNA sequence predicted Mcm3 protein sequence. Underscored sequence indicates potential Mcml protein-binding <br /> site. Symbols:  transposon insertion; ( ), position insertion does (+) does (-) destroy ability MCM3 gene complement temperature-sensitive growth phenotype mcm3-1 mutant; ****, approximate location major transcription start sites. <br />  <br /> Additional minor start sites located 4 85                     Judging immunoblot experiments using bp upstream ATG (Fig. 6). MCM3 transcript                       Mcm3-overproducing strain  , protein corre- <br /> probably spliced, necessary splicing consensus                   sponds Mcm3 protein (Fig. 8). <br /> sequence, TACTAAC (38), contained                              Yeast strain F808 transformed GAL] UAS <br /> MCM3 open reading frame 600 bp 5&apos; 100 bp                       expression vector B620 plasmid B620/MCM3, <br /> 3&apos; MCM3 ORF.                                                            contains MCM3 gene control    Mcm3 protein apparent molecular size 125                      GAL] UAS. Transformed untransformed cells kDa. Anti-Mcm3 antibodies generated using                            grown early log phase medium 2% ethanol 3% <br /> TrpE-Mcm3 hybrid protein purified E. coli extracts.                       glycerol. Galactose added culture final <br /> Immune preimmune sera used probe total yeast                       concentration 2%. different times addition cell extracts immunoblot experiments. immune                        galactose, samples removed culture preimmune sera recognized protein appar-                     analyzed immunoblot experiment. single protein <br /> ent molecular size 125 kDa (Gibson, Ph.D. thesis, 1989).                    band 125 kDa cross-reacted anti-Mcm3 antibody <br />  VOL. 10,1990                                                                      PROTEIN INVOLVED ARS FUNCTION               5715 <br />  <br />  <br />                      1381   AACAAACTAACCACCGAATATGGGTATAGTACGTTCATAGACCATCAGCGTATCACTGTG                      1440 <br />                             N K L T T E Y G Y S T F D H Q R T V <br />                      1441   CAAGAAATGCCCGAAATGGCCCCCGCTGGCCAACTTCCCAGGTCCATTGACGTCATTCTC                      1500 <br />                             Q E M P E M P G Q L P R S D V L <br />                      1501   GATGACGACCTTGTGGACAAGACCAAGCCAGGTGACAGAGTTAACGTTGTCGGGGTATTC                      1560 <br />                             D D D L V D K T K P G D R V N V V G V F <br />                      1561   AAGTCGCTTGGTGCTGGTGGCATGAACCAGTCCAACTCTAATACATTGATCGGGTTCAAA                      1620 <br />                             K S L G G G M N Q S N S N T L G F K <br />                      1621   ACTCTGATCCTAGGTAATACGGTGTATCCTCTCCACGCCAGATCCACGGGTGTCGCTGCG                      1680 <br />                             T L L G N T V Y P L H R S T G V                      1681   AGACAAATGTTGACAGATTTCGATATAAGAAATATCAATAAACTATCCAAAAAAAAGGAC                      17 40 <br />                             R Q M L T D F D R N N K L S K K K D <br />                      1741   ATTTTCGATATCTTGTCTCAATCTTTAGCGCCTTCTATTTATGGACATGACCATATAAAG                      1800 <br />                               F   D     L   S   Q   S    L     P    S     Y   G   H   D   H     K <br />  <br />                      1801   AAAGCCATTTTATTGATGCTCATGGGAGGTGTGGAGAAAAATTTAGAAAATGGCTCGCAT                      1860 <br />                             K L L M L M G G V E K N L E N G S H <br />                      1861   TTAAGAGGTGACATCAATATCCTAATGGTGGGTGATCCATCCACTGCCAAGTCCCAATTG                      1920 <br />                             L R G D N L M V G D P S T K S Q L <br />                      1921   CTAAGGTTTGTGTTGAATACAGCATCACTGGCAATTGCTACTACTGGTAGAGGTTCTTCC                      1980 <br />                             L R F V L N T S L T T G R G S S <br />                      1981   GGTGTCGGTTTGACCGCAGCGGTCACTACTGATAGGGAAACAGGTGAAAGAAGACTAGAG                      2040 <br />                             G V G L T V T T D R E T G E R R L E <br />                      2041                                                                                     2100 <br />                               G     M   V   L     D    R   G   V    V   C     D   E   F   D   K   M <br />  <br />                      2101   ACAGATGTGGATAGAGTCGCCATTCATGAAGTAATGGAACAACAAACGGTGACGATTGCC                      2160 <br />                             T D V D R V H E V M E Q Q T V T  <br />                      2161   AAAGCAGGTATTCACACAACATTAAATGCTCGTTGTAGTGTTATTGCTGCCGCAAATCCC                      2220 <br />                             K G H T T L N R C S V N P <br />  <br />                      2221   GTTTTTGGGCAGTACGATGTCAATAGAGATCCACACCAAAACATTGCCCTACCGGACTCG                      2280 <br />                             V  F G Q Y D V N R D P H Q N L P D S <br />  <br />                      2281   CTGTTGTCTCGTTTTGATTTACTATTTGTTGTGACAGACGATATCAATGAAATCAGAGAT                      2340 <br />                             L L S R F D L L F V V T D D N E R D <br />  <br />                      2341   AGATCCATTAGTGAGCATGTCTTAAGAACACACAGATATTTGCCTCCAGGTTATTTAGAG                      2400 <br />                             R S S E H V L R T H R Y L P P G Y L E <br />  <br />                      2401   GGTGAACCTGTGAGAGAGCGTTTGAATTTATCATTAGCCGTTGGGGAGGATGCAGATATA                      2460 <br />                             G E P V R E R L N L S L V G E D D  <br />                      2461   AATCCTGAAGAGCATTCCAACTCCGGGGCTGGTGTAGAAAATGAAGGAGAAGATGATGAA                      2520 <br />                             N  P E E H S N S G G V E N E G E D D E <br />  <br />                                                             FIG 6-Continued. <br />  <br /> (Fig. 8). cause discrepancy predicted             plasmid B620/MCM3 control strains (Table 5). <br /> (107 kDa) apparent (125 kDa) molecular sizes Mcm3                 control strains untransformed F808, F808 trans- unknown. approximately 50-fold overproduc-                formed GAL] expression vector, F808 trans- <br /> tion Mcm3 protein Mcm3-overproducing strain                formed GALI-lacZ fusion gene, produces induction galactose. level overproduction            ,-galactosidase control GAL] UAS. Mcm3 noticeable effect cell growth, cell             pointed high dosage effect doubling times overproducing control strains               Mcm3 stability YCplO1 YCp131 subtle galactose medium similar (Gibson, Ph.D. thesis,                 assay, effect underestimated B620/ <br /> 1989).                                                                   MCM3 plasmid itself unstable; 31% cells <br />    Overproduction Mcm3 protein causes decrease               actually contain plasmid overproducing Mcm3.  stability minichromosomes containing certain ARSs.                ingly, minichromosomes (YCplO1 YCp131) overproduction Mcm3 protein                    decreased stabilities Mcm3-overproducing strain <br /> ticeable effect cell growth, did effect                 minichromosomes particularly <br /> minichromosome maintenance. stabilities                    unstable mcm3 mutant strains. Similarly, mini- minichromosomes tested slightly reproduc-                 chromosome (YCp121) tested wild-type stability ibly lower Mcm3-overproducing strain containing                   Mcm3-overproducing strain relatively stable  5716   GIBSON ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />                    2521   GACCATGTCTTCGAAAAGTTCAACCCCTTATTACAAGCAGGTGCTAAGTTAGCAAAAAAC                        2580 <br />                           D H V F E K F N P L L Q G K L K N <br />  <br />                    2581   AAAGGTAACTATAACGGTACAGAAATTCCAAAGCTAGTCACCATCCCATTCTTAAGAAAG                        2640 <br />                           K   G   N   Y   N   G   T   E     P   K   L   V     T     P    F    L   R   K <br />                    2641   TACGTTCAATATGCCAAGGAAAGGGTTATTCCACAGTTAACACAAGAAGCCATCAATGTT                        2700 <br />                           Y V Q Y K E R V P Q L T Q E N V <br />                    2701   ATTGTGAAAAATTATACTGATTTAAGAAACGATGATAATACCAAAAAATCGCCCATTACT                        2760 <br />                             V   K   N   Y   T   D   L   R   N   D   D   N     T   K   K    S    P     T <br />  <br />                    2761   GCAAGAACTTTGGAGACTTTGATCAGATTAGCCACAGCTCACGCCAAAGTCAGGTTATCC                        2820 <br />                           R T L E T L R L T H K V R L S <br />                    2821   AAAACAGTCAACAAGGTGGATGCTAAAGTGGCTGCCAATCTACTAAGGTTTGCACTATTG                        2880 <br />                           K   T   V   N   K   V   D     K   V       N     L   L   R    F      L   L <br />                    2881   GGTGAGGATATCGGCAATGATATCGATGAAGAGGAAAGTGAATACGAAGAAGCTTTGTCG                        2940 <br />                           G   E   D     G   N   D     D   E   E   E   S     E   Y   E    E      L   S <br />                    2941   AAGAGGTCTCCACAGAAATCACCGAAAAAAAGACAAAGAGTCAGACAACCAGCAAGCAAC                        3000 <br />                           K R S P Q K S P K K R Q R V R Q P S N <br />                    3001   TCTGGATCCCCAATCAAATCTACTCCAAGAAGGTCAACGGCZiTCTTCCGTTAATGCCACG                       3060 <br />                           S G S P K S T P R R S T S S V N T <br />                    3061   CCATCGTCAGCACGCAGAATATTACGTTTTCAAGATGACGAACAGAACGCTGGTGAAGAC                        3120 <br />                           P S S R R L R F Q D D E Q N G E D <br />                    3121   GATAACGATATAATGTCACCGCTTCCTGCGGATGAGGAAGCTGAATTACAAAGAAGGCTT                        3180 <br />                           D N D M S P L P D E E E L Q R R L <br />                    3181   CAACTGGGGTTGAGAGTGTCTCCAAGACGTAGAGAACATCTTCACGCACCTGAGGAAGGT                        3240 <br />                           Q L G L R V S P R R R E H L H P E E G <br />                    3241   TCGTCGGGACCTCTTACCGAGGTCGGTACTCCAAGATTACCTAACGTATCTTCTGCAGGT                        3300 <br />                           S   S   G   P   L   T   E   V   G   T   P   R   L     P   N   V    S    S     G <br />  <br />                    3301   CAGGATGATGAGCAACAACAGTCAGTTATTTCTTTTGACAATGTGGAGCCTGGTACCATT                        3360 <br />                           Q D D E Q Q Q S V S F D N V E P G T                    3361   TCTACTGGTAGATTGTCTTTAATCTCAGGTATTATTGCGCGTCTGATGCAAACAGAAATA                        3420 <br />                           S T G R L S L S G I R L M Q T E                    3421   TTTGAAGAAGAATCCTATCCTGTGGCCTCTTTGTTCGAAAGAATCAACGAAGAACTACCG                        3480 <br />                           F E E E S Y P V S L F E R N E E L P <br />                    3481   GAGGAGGAAAAATTCTCCGCTCAAGAATATTTAGCAGGTTTGAAGATCATGTCGGACAGA                        3540 <br />                           E E E K F S Q E Y L G L K M S D R <br />                    3541   AATAACTTAATGGTTGCTGACGATAAAGTTTGGAGAGTCTGATGTCACAGGAATGTTTAC                        3600 <br />                           N N L M V D D K V W R V <br />                    3601   TAAAAGACTTCCATTCTTGGCTTTTTTTCTTTTTGTCATTCATAATTTTCAGTTCATTTC                        3660 <br />                    3661   CAGATTATTTAGTTCATTAATTGTTATTCGTGTATTAAAGAATTATGTATTAAAACGCTT                        3720 <br />  <br />                    3721   ATTTCCTTCTTTTGTTTGAATGTGTTGTAAGCAGTTATGGGGCACTATATCTTCTGTTAA                        3780 <br />                    3781   TGAATATGTGTGATACTTTGGCTTATCTTACTTGCTCATTCGTGGCTTCTTGTCACGCCA                        3840 <br />                    3841   TACGTTACGCGTTTCTCTTCCTCTTCGCGTGTTCTGAATAAAGCCAAGTGGCAATGTCTC                        3900 <br />                    3901   TTCCACCATATAACATGAAGCTTT 3924 <br />                                                  FIG. 6-Continued. <br />  <br /> mcm3 mutant strains.  overproduction                 particularly interesting stability minichromo- <br /> Mcm3 protein caused Mcm- phenotype quali-                 mcm3 mutant depends ARS tatively similar seen mcm3 mutant cells.                minichromosome contains (Table 3). results         suggests <br />                                                                       mcm3 mutation affects ARS function. Mcm- <br />                                                                       phenotype mcm3 strains temperature dependent,                      DISCUSSION                                       greater   effects       minichromosome stability occurring   new minichromosome maintenance mutant, mcm3,                  higher temperatures. addition having Mcm- phe- characterized. Mcm- phenotype mutant              notype, mcm3 cells high rates recombination  VOL. 10, 1990                                                                      PROTEIN INVOLVED ARS FUNCTION                      5717 <br />  <br />                                 B                                        necessary normal cell growth minichro- <br />                                                                            mosome maintenance. <br />                                                                               Analysis MCM3 gene indicates encodes                                                                            3.3-kb transcript containing single ORF striking <br />                                                                            homology Mcm2 protein. Located approximately 125 <br />                                                                            bp upstream major transcriptional start site                      MCM3      mm   - -3.3 kb                              potential binding site transcriptional factor Mcml. <br />                                                                            site apparently essential expression                                                                            Mcm3 play role modulating expression.                                                                            T-rich region 76 bp upstream transcriptional start <br />                                                                            site identified insertion analysis (Gibson, Ph.D. <br />                                                                            thesis, 1989) important MCM3 expression. <br />                                                                               plausible model function Mcm3 consistent <br />                                                                            data presented Mcm3 plays role <br />                    URA3-52              - 0.5 kb                           initiation chromosomal DNA replication inter- <br />                                                                            acting directly ARSs  replication origins).  <br />                                                                            rule possibility Mcm3 plays                                                                            indirect role DNA replication regulating expression <br />                                                                            gene(s) modulating function protein(s) <br />    FIG. 7. MCM3 transcript wild-type mcm3 mutant                  involved DNA replication. ARS specificity strains. 20-,ug sample poly  RNA, isolated mcm3            mcm3-1 mutant temperature dependence mutant strain R61-3C (lane  wild-type strain 8534-8C         Mcm- phenotype explained terms direct- <br /> (lane B) grown complete medium (YEPD), separated gel           binding model. model, depending affinity transferred Biodyne filter. filter probed      ARS Mcm3, suboptimal amounts Mcm3 altered <br /> 32P-labeled ssDNA BglII-BamHI DNA fragment contained              Mcm3 structure affect function different ARSs MCM3 gene. ssDNA complementary                   different extents. explanation consistent predicted coding strand MCM3 ORF. confirm equal             finding mcm3-1 mutation semidominant <br /> amounts poly  RNA present lane filter, filter probed 32P-labeled nick-translated DNA       effect certain ARSs recessive effect  URA3-containing plasmid YIp5.                                              heterozygous strains contain fully active, wild- <br />                                                                            type Mcm3 protein homozygous wild-type strains. <br />                                                                            nonpermissive growth conditions, DNA replication chromosome loss grown 30&#194;&#176;C. mcm3 cells                         mcm3 mutant proceed cell DNA <br /> temperature sensitive growth, arresting 37&#194;&#176;C large             content reaches 2n close 2n mutant cells arrest <br /> buds, undivided nuclei, fully fully replicated               large-bud stage mitosis. phenotype DNA (Fig. 2 3). mutant cells imperfect                      incongruent defect DNA replication. cell-cycle arrest, approximately half cells               ARSs similarly affected taking cell cycles arrest. arrest irreversible.             mcm3 mutation, replication initiation selective characteristics indicate MCM3 gene product                  ARSs allow DNA synthesis proceed  <br />  <br />                                        B620/MCM3                        B620 untrans. <br />                                       0 0.51.5 4.5 75 20 S              0 4.0 20 0 4.0 20 <br />  <br />  <br />                                                                                                       -200 kd <br />  <br />  <br />                                                                                                       -97 kd <br />  <br />                                                                                                        -68 kd <br />  <br />    FIG. 8. Overproduction Mcm3 protein. Mcm3-overproducing control strains grown medium containing ethanol glycerol carbon sources shifted growth galactose. Western blot protein extracts prepared samples removed culture different time points shift growth galactose shown. Numbers refer time, hours, addition galactose cultures removal different samples. Lanes: S, prestained molecular size standards; B620/MCM3, F808 <br /> cells transformed plasmid B620/MCM3; B620, F808 cells transformed plasmid B620; untrans., untransformed F808 cells. arrow <br /> marks Mcm3 protein band. Positions size standards kilodaltons (kd) shown right. <br />  5718      GIBSON ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />                                     TABLE 5. Minichromosome stabilities Mcm3-overproducing straina <br />                                                                                                                     Minichromosome stabilityb <br />    Carbon source                      Plasmid                       GALl construct <br />                                                                                                        YCplO1                YCpl31                YCpl21 <br /> Glycerol + ethanol                                                                                        96                    91                   85 <br />                                    B620                        GAL] vector (80 + 8)C                      97                    80                   94 <br />                                    pCGS286                     GALJ-lacZ (59 + 5)                         97                    93                   96 <br />                                    B620/MCM3                   GALI-MCM3 (65 &#194;&#177; 3)                         96                    86                   97 <br /> Glucose                                                                                                   72                    75                   90 <br />                                    B620                        GAL) vector (83 &#194;&#177; 0.5)                     85                    87                   81 <br />                                    pCGS286                     GALI-lacZ (74 &#194;&#177; 5)                         83                    79                   77 <br />                                    B620/MCM3                   GALI-MCM3 (76 &#194;&#177; 1)                         78                    67                   82 <br /> Galactose                                                                                                 82                    80                   87 <br />                                    B620                        GALI vector (75 &#194;&#177; 4)                       86                    83                   93 <br />                                    pCGS286                     GALI-lacZ (36 &#194;&#177; 9)                         90                    76                   85 <br />                                    B620/MCM3                   GALI-MCM3 (31 &#194;&#177; 5)                         32                    15                   88 <br />       Mcm3-overproducing strain B620/MCM3 contains MCM3 gene control GAL] UAS. Yeast strains carrying minichromosome  contains LEU2)  cases, GALI UAS construct  contains URA3) grown plates selecting plasmids. Single colonies used inoculate samples selective medium 2% ethanol 3% glycerol carbon source. hours growth, cultures supplemented leucine 2% glucose 2% galactose. cultures grown saturation 30&#194;&#176;C plated nonselective medium (YEPD). plates replica plated plates containing complete medium lacking leucine plates containing complete medium lacking uracil appropriate. <br />   b Expressed percentage colonies growing nonselective medium grew selective medium. Value obtained averaging results independent experiments. <br />     Values parentheses indicate stability GAL) construct averages numbers measured strain containing YCplOl, YCpl31, YCp121, respectively. <br />  <br />  allow complete replication chromosomes                    growth similar ARS specificities (40, 49; H. Yan <br /> cells enter mitosis (12). imperfect cell cycle arrest                   B. Tye, unpublished results). addition, mcm2 explained cells initiated DNA                       (40, 61) mcm3 mutants high rates chromosome <br /> synthesis temperature shift. cells able                  loss recombination temperature-dependent cell cycle, arresting second cell cycle                Mcm phenotypes.  cells containing certain mcml alle- mitosis. terminal phenotype mcm3 mutant                           les mcm3-1 mutation identical cell cycle arrest <br /> cells similar mutants, cdc2 cdc17,                       phenotypes grown nonpermissive conditions: <br /> known defective DNA replication (25, 52; M. J.                         arrest large buds DNA content 2n <br /> Carson, Ph.D. thesis, University Washington, Seattle,                         work; R. Elble, unpublished results). nearly iden- <br /> 1987). CDC2 encodes DNA polymerase III (60), CDCJ7                          tical phenotypes mutants suggest encodes catalytic subunit DNA polymerase (Carson,                      affect step DNA replication. possi- <br /> Ph.D. thesis, 1987). arrest, cdc2 mutant cells repli-                    bility supported findings Mcm2 cate approximately thirds genome (10) cdc17                      Mcm3 proteins significant sequence homology mutant cells replicate entire genome (25). recent                       5&apos; region MCM3 gene contains sequence report Dasso Newport (12) initiation mitosis                  close match consensus sequence DNA binding <br /> feedback inhibited unreplicated DNA cycling Xenopus                       Mcml protein (48). Experiments progress extracts explain failure mutants defective DNA                    laboratory help elucidate roles different <br /> replication enter mitosis.                                                   MCM genes gene products. <br />    increased rates chromosome loss mitotic <br /> recombination mcm3 mutant suggest                                             ACKNOWLEDGMENTS <br /> gene involved DNA replication. Mutations genes <br /> involved DNA replication bacteriophage T4 E.                         work supported Public Health Service grant <br /> coli result induced recombination (7, 37). yeast                        GM34190 National Institutes Health. R.T.S. sup- <br /> mutants known believed deficient DNA                       ported Public Health Service predoctoral training grant <br />                                                                                 2T32GM07273 National Institutes Health. <br /> replication exhibit increased rates chromosome loss                       thank Clarence Chan help performing flow cytom- recombination (21, 26). Presumably, mutations genes                      etry experiment pointing homology encoding replication factors lead increases time                      Mcm2 Mcm3. <br /> necessary replication defects replicated DNA, recombinogenic.                                                                LITERATURE CITED <br />    Mcm3 involved initiation DNA replication                   1. Baker, T. , K. Sekimizu, B. Funnel,  Kornberg. 1986. <br /> ARSs, feel likely functions                         Extensive unwinding plasmid template staged <br /> complex. effect Mcm3 overproduction minichro-                             enzymatic initiation DNA replication origin mosome stability explained overproduction                          replication E. coli chromosome. Cell 45:53-64. <br /> causing stoichiometric imbalance Mcm3-containing                         2. Boeke, J. D., F. LaCroute, G. Fink. 1984. positive <br /> multisubunit complex. view possibility,                           selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> markable similarities phenotypes                           ylase yeast: 5-fluoro-orotic acid resistance. Mol. Gen. Genet. <br />                                                                                     197:345-346. <br /> mcm mutants studied <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> especially interesting.                       3. Bouton,  H., M. Smith. 1986. Fine-structure analysis contrast mutants affect plasmid stability,                        DNA sequence requirements autonomous replication tupl (68) ssn6 (J. Schultz M. Carlson, personal                             Saccharomyces cerevisiae plasmids. Mol. Cell. Biol. 6:2354- <br /> communications), MCM genes essential                              2363. <br />  VOL. 10, 1990                                                                     PROTEIN INVOLVED ARS FUNCTION                       5719 <br />  <br />  4. Bradford, M. M. 1976. rapid sensitive method              23. Goff, C. G., D. T. Moir, T. Kohno, T. C. Gravnis, R.  Smith, <br />     quantitation microgram quantities protein utilizing           E. Yamasaki,  Taunton-Rigby. 1984. Expression calf <br />     principle protein-dye binding. Anal. Biochem. 72:248-254.            prochymosin Saccharomyces cerevisiae. Gene 27:35-46. <br />  5. Brewer, B. J., W. L. Fangman. 1987. localization         24. Hand, R. 1978. Eukaryotic DNA: organization genome <br />     replication origins ARS plasmids S. cerevisiae. Cell              replication. Cell 15:317-325. <br />     51:463-471.                                                         25. Hartwell, L. H. 1973. additional genes required DNA <br />  6. Broach, J. R., Y.-Y. Li, J. Feldman, M. Jayaram, K.                   synthesis S. cerevisiae. J. Bacteriol. 115:966-974. <br />     Nasmyth, J. B. Hicks. 1983. Localization sequence           26. Hartwell, L. H., D. Smith. 1985. Altered fidelity mitotic <br />     analysis yeast origins DNA replication. Cold Spring               chromosome transmission cell cycle mutants S. cerevisiae. <br />     Harbor Symp. Quant. Biol. 47:1165-1173.                                 Genetics 110:381-395. <br />  7. Broker, T.,  H. Doermann. 1975. Molecular genetic         27. Hinnen, , J. B. Hicks, G. R. Fink. 1978. Transformation     recombination bacteriophage T4. Annu. Rev. Genet. 9:213-             yeast. Proc. Natl. Acad. Sci. USA 75:1929-1933. <br />     244.                                                                28. Hsiao, C.-L., J. Carbon. 1979. High-frequency transforma- <br />  8. Buchman,  R., W. J. Kimmerly, J. Rine, R. D. Kornberg.            tion yeast plasmids containing cloned yeast gene. <br />     1988. DNA-binding factors recognize specific sequences           Proc. Natl. Acad. Sci. USA 76:3829-3833. <br />     silencers, upstream activating sequences, autonomously repli-       29. Hu, N., J. Messing. 1982. making strand-specific M13 <br />     cating sequences, telomeres Saccharomyces cerevisiae.            probes. Gene 17:271-277. <br />     Mol. Cell. Biol. 8:210-225.                                         30. Huberman, J. , L. D. Spotila, K.  Nawotka, S. M. El- <br />  9. Celniker, S. E., K. Sweder, F. Srienc, J. E. Bailey, J. L.          Assouli, L. R. Davis. 1987. vivo replication origin     Campbell. 1984. Deletion mutations affecting autonomously               yeast 2p.m plasmid. Cell 51:473-481. <br />     replicating sequence ARSI Saccharomyces cerevisiae. Mol.         31. Huberman, J. , J. Zhu, L. R. Davis, C. S. Newlon. 1988. <br />     Cell. Biol. 4:2455-2466.                                                Close association DNA replication origin ARS element <br /> 10. Conrad, M. N., C. S. Newlon. 1983. Saccharomyces cere-              chromosome III yeast, Saccharomyces cerevisiae. <br />     visiae cdc2 mutants fail replicate approximately        Nucleic Acids Res. 16:6373-6384. <br />     nuclear genome. Mol. Cell. Biol. 3:1000-1012.                 32. Hutter, K. J., H. E. Eipel. 1978. Flow cytometric determi- <br /> 11. Dale, R., B. McClure, J. Houchins. 1985. rapid single-            nations cellular substances algae, bacteria, molds     stranded cloning strategy producing sequential series          yeasts. Antonie van Leeuwenhoek J. Microbiol. Serol. 44:269- <br />     overlapping clones use DNA sequencing: application            282. <br />     sequencing corn mitochondrial 18S rDNA. Plasmid 13:31-          33. Johnson,  D.,  Herskowitz. 1985. repressor (MATa2 <br />     40.                                                                     product) operator control expression set cell type <br /> 12. Dasso, M., J. W. Newport. 1990. Completion DNA                   specific genes yeast. Cell 42:237-247. <br />     replication monitored feedback controls     34. Kearsey, S. 1984. Structural requirements function     initiation mitosis vitro: studies Xenopus. Cell 61:811-        yeast chromosomal replicator. Cell 37:299-307. <br />     823.                                                                35. Kimmerly, W.,  Buchman, R. Kornberg, J. Rine. 1988. <br /> 13. Dean, F. B., P. Bullock, Y. Murakami, C. R. Wobbe, L.                   Roles DNA-binding factors replication, segregation <br />     Weissbach, J. Hurwitz. 1987. SV40 DNA replication: SV40             transcriptional repression mediated yeast silencer. <br />     large T antigen unwinds DNA containing SV40 origin               EMBO J. 7:2241-2253. <br />     replication. Proc. Natl. Acad. Sci. USA 84:16-20.                   36. Kleid, D. G., D. Yansura, B. Small, D. Dowbenko, D. M. Moore, <br /> 14. Dieckmann, C. L.,  Tzagoloff. 1985. Assembly               M. J. Grubman, P. D. McKercher, D. 0. Morgan, B. H. <br />     mitochondrial membrane  CBP6, yeast nuclear gene               Robertson, H. L. Bachrach. 1981. Cloned viral protein <br />     necessary synthesis cytochrome b. J. Biol. Chem. 260:            vaccine foot mouth disease: responses cattle     1513-1520.                                                              swine. Science 214:1125-1129. <br /> 15. Diffley, J. F. X., B. Stillman. 1988. Purification yeast   37. Konrad, E. G. 1977. Method isolation Escherichia coli <br />     protein binds origins DNA replication transcrip-       mutants enhanced recombination chromosomal <br />     tional silencer. Proc. Natl. Acad. Sci. USA 85:2120-2124.               duplications. J. Bacteriol. 130:167-172. <br /> 16. Diffley, J. F. X., B. Stillman. 1989. Similarity    38. Langford, C. J., D. Gallwitz. 1983. Evidence intron- <br />     transcriptional silencer proteins ABF1 RAP1. Science 246:           contained sequence required splicing yeast RNA <br />     1034-1037.                                                              polymerase 2 transcripts. Cell 33:519-527. <br /> 17. Dodson, M., H. Echols, S. Wickner, C. Alfano, K. Mensa-             39. Lea, D. E., C.  Coulson. 1948. distribution     Wilmot, B. Gomes, J. LeBowitz, J. D. Roberts, R. Mac-               numbers mutants bacterial populations. J. Genet. 49:264- <br />     macken. 1986. Specialized nucleoprotein structures origin        284. <br />     replication bacteriophage lambda: localized unwinding      40. Maine, G. T., P. Sinha, B.-K. Tye. 1984. Mutants S. <br />     duplex DNA protein reaction. Proc. Natl. Acad. Sci.            cerevisiae defective maintenance minichromosomes. <br />     USA 83:7638-7642.                                                       Genetics 106:365-385. <br /> 18. Doolittle, R. F. 1986. URFs ORFs: primer         41. Maine, G. T., R. T. Surosky, B. K. Tye. 1984. Isolation     analyze derived amino acids sequences. University Science               characterization centromere chromosome V (CEN5) <br />     Books, Valley, Calif.                                              Saccharomyces cerevisiae. Mol. Cell. Biol. 4:86-91. <br /> 19. Eisenberg, S., C. Civalier, B.-K. Tye. 1988. Specific inter-    42. McKnight, G. L., T. S. Cardillo, F. Sherman. 1981.     action Saccharomyces cerevisiae protein DNA             extensive deletion causing overproduction yeast iso-2-cy- <br />     element associated certain autonomously replicating se-            tochrome c. Cell 25:409-419. <br />     quences. Proc. Natl. Acad. Sci. USA 85:743-746.                     43. Morrison, D.  1977. Transformation Escherichia coli: <br /> 20. Francesconi, S. C., S. Eisenberg. 1989. Purification            cryogenic preservation competent cells. J. Bacteriol. 132: <br />     characterization OBF1: Saccharomyces cerevisiae protein            349-351. <br />     binds autonomously replicating sequences. Mol. Cell.        44. Mortimer, R. K., D. Schild. 1980. Genetic map Saccha- <br />     Biol. 9:2906-2913.                                                      romyces cerevisiae. Microbiol. Rev. 44:519-571. <br /> 21. Game, J. C., L. H. Johnston, R. C. von Borstel. 1979.           45. Newlon, C. S., W. G. Burke. 1980. Replication small <br />     Enhanced mitotic recombination ligase defective mutant          chromosomal DNAs yeast. ICN-UCLA Symp. Mol. Cell. <br />     yeast Saccharomyces cerevisiae. Proc. Natl. Acad. Sci.              Biol. 19:399-409. <br />     USA 76:4589-4592.                                                   46. Norrander, J., T. Kempe, J. Messing. 1983. Construction 22. Gibson, S., R. Surosky, P. Sinha, G. Maine, B.-K. Tye. 1987.        improved M13 vectors using oligodeoxynucleotide-directed mu- <br />     Complexity enzyme initiation DNA               tagenesis. Gene 26:101-106. <br />     replication yeast. ICN-UCLA Symp. Mol. Cell. Biol. 47:341-       47. Osley, M. , L. Hereford. 1982. Identification se- <br />     354.                                                                    quence responsible periodic synthesis yeast histone 2A <br />  5720     GIBSON ET AL.                                                                                                   MOL. CELL. BIOL. <br />  <br />     mRNA. Proc. Natl. Acad. Sci. USA 79:7689-7693.                       62. Southern, E. 1975. Detection specific sequences DNA <br /> 48. Passmore, S., R. Elble, B.-K. Tye. 1989. protein involved          separated gel electrophoresis. J. Mol. Biol. 98:503-517. <br />     minichromosome maintenance yeast binds transcriptional       63. Struhl, K., D. T. Stinchcomb, S. Scherer, R. W. Davis. 1979. <br />     enhancer conserved eukaryotes. Genes Dev. 3:921-935.                  High-frequency transformation yeast: autonomous replica- <br /> 49. Passmore, S., G. Maine, R. Elble, C. Christ, B.-K. Tye.              tion hybrid DNA molecules. Proc. Natl. Acad. Sci. USA <br />     1988. Saccharomyces cerevisiae protein involved plasmid               76:1035-1039. <br />     maintenance necessary mating MATa cells. J. Mol.           64. Surosky, R. T., B.-K. Tye. 1985. Construction telocentric <br />     Biol. 204:593-606.                                                       chromosomes Saccharomyces cerevisiae. Proc. Natl. Acad. <br /> 50. Petes, T. D., C. S. Newlon. 1974. Structure DNA DNA            Sci. USA 82:2106-2110. <br />     replication mutants yeast. Nature (London) 251:637-639.           65. Thomas, P. S. 1980. Hybridization denatured RNA small <br /> 51. Petes, T. D., D. H. Williamson. 1975. Fiber autoradiography          DNA fragments transferred nitrocellulose. Proc. Natl. Acad. <br />     replicating yeast DNA. Exp. Cell Res. 95:103-110.                     Sci. USA 77:5201-5205. <br /> 52. Pringle, J. R., L. H. Hartwell. 1981. Saccharomyces          66. Thrash-Bingham, C., W. L. Fangman. 1989. yeast muta- <br />     cerevisiae cell cycle, p. 79-142. J. N. Strathern, E. W. Jones,       tion stabilizes plasmid bearing mutated ARSI element. <br />     J. R. Broach (ed.), molecular biology yeast               Mol. Cell. Biol. 9:809-816. <br />     Saccharomyces. Cold Spring Harbor Laboratory, Cold Spring            67. Towbin, H., T. Staehelin, J. Gordon. 1979. Electrophoretic <br />     Harbor, N.Y.                                                             transfer proteins polyacrylamide gels nitrocellulose <br /> 53. Rigby, P. W. J., M. Dieckmann, C. Rhodes, P. Berg. 1977.             sheets: procedure applications. Proc. Natl. Acad. Sci. <br />     Labeling deoxyribonucleic acid high specific activity vitro        USA 76:4350-4354. <br />     nick translation DNA polymerase 1. J. Mol. Biol. <br />     113:237-251.                                                         68. Tschumper, G., J. Carbon. 1980. Sequence yeast <br /> 54. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R.             fragment containing chromosomal replicator TRP1 <br />     Fink. 1987. Saccharomyces cerevisiae genomic plasmid based             gene. Gene 10:157-166. <br />     centromere-containing shuttle vector. Gene 60:237-243.          69. Umek, R. M., D. Kowalski. 1988. ease DNA <br /> 55. Saffer, L. D., 0. L. Miller, Jr. 1986. Electron microscopic          unwinding determinant initiation yeast replication <br />     study Saccharomyces cerevisiae rDNA chromatin replica-                origins. Cell 52:559-567. <br />     tion. Mol. Cell. Biol. 6:1148-1157.                                  70. Vaitukaitis, J. L. 1981. Production antisera small doses <br /> 56. Sanger, F., S. Nicklen,  R. Coulson. 1977. DNA sequenc-            immunogen: multiple intradermal injections. Methods  <br />     ing chain-terminating inhibitors. Proc. Natl. Acad. Sci.            zymol. 73:46-52. <br />     USA 74:5463-5467.                                                    71. Walker, S. S., S. C. Francesconi, S. Eisenberg. 1990. DNA <br /> 57. Seifert, H. S., E. Y. Chen, M.  F. Heffron. 1986. Shuttle         replication enhancer Saccharomyces cerevisiae. Proc. Natl. <br />     mutagenesis: method transposon mutagenesis Saccha-              Acad. Sci. USA 87:4665-4669. <br />     romyces cerevisiae. Proc. Natl. Acad. Sci. USA 83:735-739.           72. Walker, S. S., S. C. Francesconi, B.-K. Tye, S. Eisenberg. <br /> 58. Sherman, F., G. R. Fink, C. W. Lawrence. 1974. Methods            1989. OBF1 protein DNA-binding site important <br />     yeast genetics. Cold Spring Harbor Laboratory, Cold Spring               function autonomously replicating sequence     Harbor, N.Y.                                                             Saccharomyces cerevisiae. Mol. Cell. Biol. 9:2914-2921. <br /> 59. Shore, D., D. J. Stillman,  H. Brand, K.  Nasmyth.           73. Williams, J. S., T. T. Eckdahl, J. N. Anderson. 1988. Bent <br />     1987. Identification silencer binding proteins yeast:            DNA functions replication enhancer Saccharomyces <br />     possible roles SIR control DNA replication. EMBO J.               cerevisiae. Mol. Cell. Biol. 8:2763-2769. <br />     6:461-467.                      <br /> </body></html>