<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                 INVESTIGATION <br />  <br />  <br />  <br />  <br />                                 Jaw Worm: GTPase-activating Protein <br />                                          EAT-17 Regulates Grinder Formation                                                         Caenorhabditis elegans <br />                                                           Sarah Straud,*,1,2 Inhwan Lee,&#226;&#8364;&#160;,1 Bomi Song,* Leon Avery,&#226;&#8364;&#161; Young-Jai ,3 <br />                                      *Department Molecular Biology, University Texas Southwestern Medical Center, Dallas, Texas 75390-9148, <br />                          &#226;&#8364;&#160;Department Biochemistry Molecular Biology, Virginia Commonwealth University, Richmond, Virginia 23298-0614,                                       &#226;&#8364;&#161;Department Physiology Biophysics, Virginia Commonwealth University, Richmond, Virginia 23298-0551 <br />  <br />  <br />  <br />  <br /> ABSTRACT Constitutive transport cellular materials essential cell survival. multiple small GTPase Rab proteins required process, regulators Rabs known. report EAT-17, novel GTPase-activating protein (GAP), <br /> regulates RAB-6.2 function grinder formation Caenorhabditis elegans. identi&#239;&#172;?ed EAT-17 novel RabGAP interacts RAB-6.2, protein presumably regulates vesicle traf&#239;&#172;?cking Golgi, endoplasmic reticulum, plasma membrane form functional grinder. EAT-17 canonical GAP domain critical function. RNA interference 25 con&#239;&#172;?rmed predicted RABs C. elegans shows RNAi rab-6.2 produces phenotype identical eat-17. directed yeast  <br /> hybrid screen using EAT-17 bait 25 RAB proteins prey identi&#239;&#172;?es RAB-6.2 interacting partner EAT-17, <br /> con&#239;&#172;?rming RAB-6.2 speci&#239;&#172;?c substrate EAT-17. Additionally, deletion mutants rab-6.2 grinder defects identical eat-17 loss function mutants, RAB-6.2 EAT-17 expressed terminal bulb pharynx grinder located. Collectively, results suggest EAT-17 speci&#239;&#172;?c GTPase-activating protein RAB-6.2. Based conserved function Rab6 vesicular transport, propose EAT-17 regulates turnover rate RAB-6.2 activity cargo <br /> traf&#239;&#172;?cking grinder formation. <br />  <br />  <br />  <br /> C   ELLS constitutively transport newly synthesized proteins, <br />     lipids, molecules periphery vesicle traf&#239;&#172;?cking. process conserved yeast                                                                                        et al. 1992; Martinez et al. 1994, 1997; Girod et al. 1999; <br />                                                                                        Opdam et al. 2000; Del Nery et al. 2006). fast turnover, <br />                                                                                        Rabs require <span id='am-70' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-71' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>guanine</span> <span id='am-50' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-51' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-52' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> exchange factors (Rab GEFs) <br /> humans, essential function survival cells.                        activation GTPase-activating proteins (Rab GAPs) Rab GTPases small <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="GTP" datatype="xsd:string"></span>GTP</span> binding proteins regulate                               turn activity (Grosshans et al. 2006). role Rab6 transport vesicles different compartments                            membrane traf&#239;&#172;?cking established, GAPs cell (Zerial McBride 2001; Jordens et al. 2005;                                GEFs Rab6 identi&#239;&#172;?ed; physiologi- <br /> Grosshans et al. 2006). Rab6&#226;&#8364;&#8482;s localized Golgi mem-                             cal importance largely unknown. physiological roles branes mark target anterograde cargos                                 RabGAPs addressed recently &#239;&#172;&#8218;ies (Houalla et al. <br /> Golgi post-Golgi compartments  plasma                                   2010; Uytterhoeven et al. 2011) worms (Chotard et al. <br /> membrane) retrograde cargos early/recycling                                2010). cases, removing function RabGAPs <br /> dosomes Golgi endoplasmic reticulum (Jasmin                                 produced phenotypes identical removing                                                                                        relevant Rabs, showing fundamental roles RabGAPs                                                                                        modulating Rab functions. Currently, identi&#239;&#172;?ed GAP <br /> Copyright &#194;&#169; 2013 Genetics Society America <br /> doi: 10.1534/genetics.113.152538 <br />                                                                                        Rab6 GAPCenA humans, associated Manuscript received April 25, 2013; accepted publication June 12, 2013             centrosomes regulates Golgi dynamics dividing cells <br /> Supporting information available online http://www.genetics.org/lookup/suppl/    (Cuif et al. 1999). Nonetheless, misregulation GAPs doi:10.1534/genetics.113.152538/-/DC1. <br /> 1 <br />   authors contributed equally work.                                      small G proteins, Ras, directly relates diseases <br /> 2 <br />   Present address: Department Biological Physical Sciences, Saint Augustine&#226;&#8364;&#8482;s   cancer, implicating essential roles GAPs  <br />   University, Raleigh, NC 27610-2298. <br /> 3 <br />  Corresponding author: P. O. Box 980614, 1220 E. Broad St., MMRB Bldg, Room <br />                                                                                        trolling kinetics G protein activity (Tanabe et al. 2006; <br />  2046, Richmond, VA 23298. E-mail: yjyou@vcu.edu                                       Durkin et al. 2007; Pamonsinlapatham et al. 2009). <br />  <br />  <br />  <br />                                                                                                      Genetics, Vol. 195, 115&#226;&#8364;&#8220;125 September 2013     115 <br />      Caenorhabditis elegans grinder complex structure     SNP mapping <br /> required proper grinding food (bacteria)                                                                   CB4856 males crossed DA773 hermaphrodites. <br /> passed intestine. food abundant, worms feed <br />                                                                   unc-93; lin-15 eat-17 sup-10/CB4856(+) worms iso- average rate 200 pumps minute (ppm) based                                                                   lated F2 generation ability produce 25% <br /> counts pharyngeal pumping grinding motions (Avery <br />                                                                   Muv non-Unc self-progeny near absence non-Muv 2012; Raizen et al. 2012). high frequency                                                                   non-Unc self-progeny. Sup non-Muv recombinants iso- <br /> grinding worm&#226;&#8364;&#8482;s life suggests transport                                                                   lated F3 generation. scored eat-17 <br /> grinder components build functional grinder essential <br />                                                                   feeding defect, progeny analyzed PCR worms&#226;&#8364;&#8482; rapid growth normal development. Consid- <br />                                                                   subsequent restriction digestion SNPs located ering location structure C. elegans pharynx, <br />                                                                   lin-15 sup-10. SNPs examined primers used grinder components likely produced terminal bulb <br />                                                                   ampli&#239;&#172;?cation shown Supporting Information, <br /> muscles pm6 pm7 traf&#239;&#172;?cked apical surfaces <br />                                                                   Table S1. <br /> (Albertson Thomson 1976). exact components                                                                       Deletion mutants ser-1(ok345), hda-4(ok518), grinder molecular mechanisms regulating trans- <br />                                                                   T27A8.2(ok570) mapped scoring F4 progeny port,  remain unknown. <br />                                                                   deletion detected PCR. Primers used detecting     report C. elegans gene eat-17 encodes Rab <br />                                                                   deletions shown Table S2. <br /> GTPase-activating protein (Rab GAP) speci&#239;&#172;?c RAB-6.2, Rab6 homolog C. elegans. eat-17 loss function         Analysis eat-17 gene structure identi&#239;&#172;?cation <br /> rab-6.2 deletion mutants similar phenotypes,          splice variants <br /> disorganized, poorly formed grinders. RNAi rab-6.2        determine intron exon structure eat-17, pri- <br /> produces similar defects. directed yeast hybrid screen      mers designed based Gene&#239;&#172;?nder predictions shown <br /> using EAT-17 bait identi&#239;&#172;?ed RAB-6.2 interacting         WormBase amplify partial length transcripts partner. Furthermore, transgenes mutations        RT&#226;&#8364;&#8220;PCR. strand poly + cDNA derived mixed <br /> served catalytic arginine residue required GAP activ-     stage N2 hermaphrodites used template reac- <br /> ity EAT-17 failed rescue grinder defects injected      tions. Trizol (Invitrogen) used isolate RNA eat-17 mutants. Based results           Strand cDNA Synthesis kit (Roche) used gener- <br /> served function Rab6 vesicle transport, propose     ate cDNA. Primers used amplify PCR fragments shown <br /> RAB-6.2 EAT-17 cooperate transport grinder          Table S3. PCR reactions yielded single product, components apical surfaces terminal bulb            exons &#239;&#172;?rst appeared correct. assess <br /> muscles form functional grinder.                             number abundance different splice variants,  <br />                                                                   length eat-17 cDNAs cloned pGEM-T Easy <br /> Materials Methods                                             vector (Promega) sequenced. <br /> Worm culture                                                      SL1 trans-splicing <br /> Worms cultured handled described (Sulston             identify &#239;&#172;?rst exon eat-17, forward primer recog- Hodgkin 1988) following modi&#239;&#172;?cations.          nizing 22 nucleotide-SL1-splice leader sequence (59- routinely grown NGMSR plates (Avery 1993).           ggtttaattacccaagtttgag -39) nested reverse primers <br /> NGMSR differs NGM containing 200 pg/ml strepto-           recognizing sequences exon 2 [1050: 59-tgttcagctgctc <br /> mycin sulfate, 10 pg/ml nystatin, 2% agar instead          catcttg-39(outside primer)/1051: 59-cgacttcatttacgcatactg-39 <br /> 1.7%. (Streptomycin nystatin reduce bacterial fun-        (inside primer)] used. Products puri&#239;&#172;?ed (Qiaquick <br /> gal contamination. higher agar concentration delays       Gel Extraction kit), cloned pGEM-T Easy vectors se- <br /> burrowing worms agar.) addition, worms     quenced. resulting sequences BLASTed (blastn) maintained 20  Escherichia coli HB101 unless        C. elegans ESTs genomic DNA using WormBase. <br /> dicated  wild-type strains Bristol N2                                                                   Cosmid rescue <br /> Hawaiian CB4856. eat-17(ad707) rab-6.2(ok2254) used study outcrossed twice        Cosmids used study obtained Alan Coulson <br /> N2. strains used study DA2120 adEx2120          (Wellcome Trust, UK). T24D11 F01G12 isolated <br /> [eat-17p::GFP rol-6(sd)], DA773 unc-93(e1500sd); lin-15(n309)     using Qiagen Plasmid Mini kit injected eat-17(ad707) sup-10(n183), DA1814 ser-1(ok345), RB758            eat-17 mutants (50 ng/ml). Plasmids pPD118.20 myo-3:: <br /> hda-4(ok518), RB787 T27A8.2(ok570), DA707 eat-17(ad707),          GFP  Andrew  Stanford University) pRAK3 <br /> NL2099 rrf-3(pk1426), DA1983 eat-17(ad707); adEx1983              rol-6(d) used injection markers concentration <br /> [F01G12 T24D11 rol-6(d)], DA1982 rrf-3(pk1426); eat-17            10 ng/ml. Worms fed Comamonas injection (ad707), DA2035 adEx2035[rab-6.2p::GFP rol-6(d)], DA2033          allow good growth (Avery Shtonda 2003). Transgenic <br /> eat-5(ad1402); adEx2033[RAB-6.2::GFP unc-122::RFP eat-5           worms fed DA837 enhance feeding defect. <br /> (+)], YJ89 rab-6.2(ok2254), MT993 lin-10(e1439); 5               Growth rates transgenic nontransgenic progeny <br /> (e1467ts).                                                        compared assess rescue. Eggs transgenic mothers <br />  <br />  <br />  <br /> 116    S. Straud et al. <br />  placed individually plates seeded DA837            &#194;&#183;100 objective lens. Images acquired using Zeiss checked 12 hr stage worms. Growth           Axiovision software. <br /> rate de&#239;&#172;?ned inverse time required worms <br />                                                                   RNAi reach adulthood produce progeny. Worms scored adults laid egg.                      RNAi performed described minor modi&#239;&#172;?cations <br />                                                                   (Kamath Ahringer 2003). cDNA sequences eat-17 Generating eat-17 rescue construct <br />                                                                   25 C. elegans Rabs ampli&#239;&#172;?ed PCR using primers <br /> Rescuing fragments containing length eat-17 cDNA         shown Table S5. HiScribe RNAi Transcription kit generated overlap extension PCR (Ho et al. 1989).         (New England Biolabs) used generate dsRNAs. &#239;&#172;?rst round PCR reactions, 59 39 eat-17 cDNA fragments <br />                                                                   GFP fusions EAT-17 RAB-6.2 ampli&#239;&#172;?ed using primers: 59-ttgtcaccgccgatggcagccactg <br /> cagcgctac&#226;&#8364;&#8220;39 59-tagggatgttgaagagtaattggacctagtggctatcc        eat-17p::GFP: 5.6-kb eat-17 promoter ampli&#239;&#172;?ed gacagtt&#226;&#8364;&#8220;39. resulting PCR fragments puri&#239;&#172;?ed          N2 genomic DNA using following primers: 59-taggt <br /> subjected second round PCR using overlap exten-       tacggtagttggtacg-39 59-gaaaagttcttctcctttactcatcggcggt <br /> sion method fuse eat-17 promoter unc-54 39-UTR         gacaattgg-39. addition 59 regulatory sequence, end. primers used amplify promoter region      DNA contains &#239;&#172;?rst codons eat-17. GFP ampli-  59-taggttacggtagttggtacg&#226;&#8364;&#8220;39 59-gtagcgctgcagtggctgc       &#239;&#172;?ed pPD95.75 plasmid DNA (gift   Stanford <br /> catcggcggtgacaa&#226;&#8364;&#8220;39. primers used amplify 500 bp unc-54     University) using primers: 59-ccaattgtcaccgccgatgagtaaagga <br /> 39 UTR 59-aactgtcggatagccactaggtccaattactcttcaacatcccta&#226;&#8364;&#8220;39    gaagaacttttc-39 59-tttggtatattgggaatggtattctg-39. DNAs 59-tttggtatattgggaatgtattctg&#226;&#8364;&#8220;39. resulting PCR prod-      fused overlap extension PCR using following <br /> ucts puri&#239;&#172;?ed injected eat-17 worms (25 ng/ml).      nested primers: 59-acggtagtgttttatcagtagtg-39 59-caaacc &#239;&#172;?nal injected product contained 5.6 kb sequence            caaaccttcttccgatc-39. <br /> located directly upstream eat-17 transcriptional start        resulting product puri&#239;&#172;?ed injected N2 <br /> site. 2.5-kb length eat-17 cDNA construct fused        adults pRAK3 rol-6(d) injection marker. &#226;&#8364;&#339;promoter region&#226;&#8364;? heterologous unc-54 39-         established transgenic lines observed using Zeiss <br /> UTR added promote stability. construct lacked         Axio A2 Imager &#194;&#183;630 &#194;&#183;1000 magni&#239;&#172;?cation. <br /> intron elements, including large 4-kb &#239;&#172;?rst intron. let-       Images acquired using Zeiss Axiovision software. <br /> 858::GFP plasmid DNA (18 ng/ml) used injection <br /> marker.                                                           rab-6.2p::GFP: 2.8-kb rab-6.2 promoter fragment    transgenic lines isolated examined. F2            ampli&#239;&#172;?ed N2 genomic DNA using following pri- <br /> transgenic animals identi&#239;&#172;?ed using SZX12GFP dis-         mers: 59-aatcgcacagcaggcctcc-39 59-gtgaaaagttcttctcctt <br /> secting scope (Olympus). transgenic embryos          tactcggattaccaaagtccgacat-39. addition 59 regulatory <br /> die, animals reached adulthood scored          sequence, DNA fragment contains &#239;&#172;?rst seven codons grinder phenotype. transgenic lines, 100%         rab-6.2. GFP ampli&#239;&#172;?ed plasmid pPD95.75 using adult transgenic animals rescued defects          following primers: 59-atgtcggactttggtaatccgagtaaaggagaa <br /> grinder formation. Grinder morphology observed using          gaacttttcac-39 59-tttggtatattgggaatggtattctg-39. Products Zeiss Axio A2 Imager &#194;&#183;630 &#194;&#183;1000 magni&#239;&#172;?ca-          fused overlap extension PCR using following <br /> tion. Images acquired using Zeiss Axiovision software.       primers: 59-tttgccgaacggaagagcc-39 59-caaacccaaaccttcttcc <br />                                                                   gatc-39. &#239;&#172;?nal product puri&#239;&#172;?ed injected N2 <br /> Assay EAT-17 GAP activity <br />                                                                   adults rol-6(d) injection marker. GFP expression determine functional importance GAP activity        observed described  EAT-17, rescuing fragments containing catalytically <br />                                                                   Yeast hybrid assay <br /> inactive R116/119K R116/119A mutations gener- <br /> ated using overlap extension method         generate &#239;&#172;?rst-strand cDNA, RNA isolated Primers used generate R116/119K mutation               fed, mixed stage N2 population using Trizol. Poly + <br /> shown Table S4. resulting PCR products gel            &#239;&#172;?rst-strand cDNA prepared using Strand cDNA <br /> puri&#239;&#172;?ed injected eat-17 worms concentration         Synthesis kit RT&#226;&#8364;&#8220;PCR (Roche). 25 ng/ml. let-858::GFP plasmid DNA used               Inserts used preys ampli&#239;&#172;?ed PCR using injection marker concentration 100 ng/ml. Wild-type,      Expand Long Template PCR kit (Roche). Primers amplify <br /> R116/119K, R116/119A injections performed             preys shown Table S6. PCR products cloned <br /> parallel. score rescue grinder defects, F1 gravid   pGEM-T Easy vector sequenced. Constructs adult progeny injected mothers picked 4% agar          wild-type sequence subcloned pACT2 vector (gift <br /> pads containing 10 mM sodium azide (Sigma, St. Louis).            R. Lin, University Texas Southwestern Medical Center <br /> Defects grinder formation recorded           Dallas), frame GAL4 activation domain. <br /> presence absence GFP expression determined.                truncated versions eat-17 used baits Worms observed using Zeiss Axio A2 Imager             ampli&#239;&#172;?cation, sequencing cloning pVJL11 vector <br />  <br />  <br />  <br />                                                                                             EAT-17 RabGAP RAB-6.2   117 <br />                                                                                                     Figure 1 eat-17 mutants                                                                                                    defects grinder formation.   <br />                                                                                                    DIC image wild-type pharynx. <br />                                                                                                    C. elegans grinder com- <br />                                                                                                    posed plates arranged <br />                                                                                                    approximate triradial symme- <br />                                                                                                    try. fully formed plates                                                                                                    visible lateral sections (arrow). <br />                                                                                                    (B) DIC image eat-17(ad707) <br />                                                                                                    mutant pharynx. ma- <br />                                                                                                    terial comprising grinder plates <br />                                                                                                    missing eat-17 mutants. (C) <br />                                                                                                    intestine wild-type worms <br />                                                                                                    fed mCherry-expressing E. coli <br />                                                                                                    shows smooth &#239;&#172;&#8218;uorescence                                                                                                    &#239;&#172;&#8218;uorescence released                                                                                                    ground bacteria. (D) intestine <br />                                                                                                    eat-17 mutants                                                                                                    unground bacteria indicated                                                                                                    arrow. (E F) Correspond- <br />                                                                                                    ing DIC images wild type (E) <br />                                                                                                    eat-17 mutant (F), respec- <br />                                                                                                    tively. F, arrow indicates <br />                                                                                                    DIC image unground bacteria. <br />                                                                                                    images, anterior shown <br />                                                                                                    left. Bars, 10 mm. <br />  <br />  <br />  <br /> (gift M. Cobb, University Texas Southwestern Medical     bacteria slightly asynchronous terminal bulb  <br /> Center Dallas), frame LexA DNA binding         tractions feeding (Avery 1993). striking <br /> main. Primers shown Table S7. Mutant versions      phenotype,  defect grinders; wild-type <br /> baits generated using vitro mutagenesis kit (Invi-    grinders consist highly refractile cuticle ridges trogen). Primers containing mutations shown Table S4.     observed DIC optics (Figure 1A, arrow). ridges    test interactions, combinations baits preys     completely disorganized eat-17 mutants (Figure 1B). transformed yeast strain L40 (Hama et al. 1999).   wild-type worms, grinder composed separate <br /> X-gal &#239;&#172;?lter assays quantitative ortho-nitrophenyl-b-D-       plates arranged approximate triradial symmetry <br /> galactopyranoside (ONPG) assays performed             (Albertson Thomson 1976; von Lieven 2003). scribed (Yeast Protocols Handbook, Clontech). average         plates secreted apical surfaces pharyngeal colonies tested  growth assays,          muscle cells pm6 pm7 terminal bulb (Albertson <br /> colonies streaked plates lacking histidine         Thomson 1976). feeding, plates rotate <br /> supplemented 5 mM 3-amino-1,2,4-triazole (3 .             grinding material comes  <br /> Miller units calculated OD420/t &#194;&#183; V &#194;&#183; OD600.            tween  defects grinder, eat-17 mutants <br />                                                                  grind bacteria ef&#239;&#172;?ciently. result, unground bac- <br /> Chitin staining <br />                                                                  teria pass intestine (Figure 1, C&#226;&#8364;&#8220;F) contributing Chitin stained described (Ruvkun Finney 2005).        Eat phenotype. <br /> Fluorescein-conjugated chitin-binding probe (P5211S, NEB) used 1:100 dilution.                                    eat-17 encodes GTPase activating protein                                                                  coiled-coil domains C terminus <br /> Grinder function assay <br />                                                                  Previous work placed eat-17 chromosome X, Worms fed mCherry-expressing HB101 described             lin-15 sup-10 (Avery 1993). mapped eat-17 <br />  et al. 2008). Images acquired using Zeiss Axiovi-      47-kb region SNP markers, pkP6096 sion software.                                                   snp-F38E9.2 (Figure 2A), using multipoint SNP mapping <br />                                                                  strategy (Wicks et al. 2001). genes predicted Results                                                          interval (Figure 2B): F01G12.1 encodes putative copper <br />                                                                  transporter, T24D11.1 (currently designated tbc-4) pre- <br /> eat-17 mutants Eat defects <br />                                                                  dicted encode Rab GAP containing TBC (Tre2/Bub/ grinder formation <br />                                                                  Cdc16 homology) domain canonical GAP domain eat-17 mutants isolated genetic screen           F01G12.6 predicted encode protein defects feeding behavior. defective trapping      coiled-coil domains. Interestingly, T24D11.1 F01G12.6 <br />  <br />  <br />  <br /> 118    S. Straud et al. <br />                                                                                                        Figure 2 eat-17 gene structure. <br />                                                                                                         Region C. elegans genome <br />                                                                                                       eat-17 maps. eat-17                                                                                                       mapped 47 kb                                                                                                       SNP markers pkP6096 snp- <br />                                                                                                       F38E9.2 using multipoint SNP <br />                                                                                                       mapping strategy. (B) Genes pre- <br />                                                                                                       dicted located inter- <br />                                                                                                       val gene structure                                                                                                       eat-17. Red arrows indicate                                                                                                       exons targeted amplify                                                                                                       validate eat-17 transcripts. (C) <br />                                                                                                       Exon usages possible <br />                                                                                                       splice forms deduced SL1 <br />                                                                                                       RT&#226;&#8364;&#8220;PCR analysis. red <br />                                                                                                       box indicates exon 4B.                                                                                                       different splice forms eat-17a <br />                                                                                                       eat-17b encode proteins <br />                                                                                                       differing amino acids. <br />                                                                                                       (D) Predicted protein structure <br />                                                                                                       EAT-17 location                                                                                                       mutation ad707 allele. <br />                                                                                                       N-terminal Rab GAP domain <br />                                                                                                       followed C-terminal <br />                                                                                                       predicted coiled-coil domains. <br />                                                                                                       arrow indicates position <br />                                                                                                       mutated <span id='am-58' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-64' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>amino acid</span>                                                                                                       ad707. <br />  <br />  <br />  <br />  <br /> share homology human protein Evi5 (Figure S1),             predicted WormBase (www.wormbase.org) failed. Rab11 GAP, mutation associated neuroblas-           identi&#239;&#172;?ed individual intron/exon boundaries toma (Liao et al. 1997; Dabbeekeh et al. 2007).             &#239;&#172;?rst exon incorrectly predicted. iden- <br /> T24D11.1 F01G12.6 transcribed direc-            tify 59 end transcript, used SL1 trans-splicing <br /> tion (Figure 2B) unable retrieve             analysis. C. elegans,  60% transcripts cDNAs coding individual genes RT&#226;&#8364;&#8220;PCR, hy-                SL1 SL2 trans-spliced 59 ends (Blumenthal <br /> pothesized predicted genes comprised single        1995; Conrad et al. 1995). results &#239;&#172;?rst exon <br /> gene. RT&#226;&#8364;&#8220;PCR using primers predicted exon          eat-17 located  4 kb upstream exon 2 (indicated * T24D11.1 &#239;&#172;?fth predicted exon F01G12.6 iden-           Figure 2B). learned transcript contains <br /> ti&#239;&#172;?ed single transcript (indicated arrows Figure 2B),      40 bp 59-UTR. Using primer targeting newly iden- <br /> con&#239;&#172;?rming T24D11.1 F01G12.6 parts               ti&#239;&#172;?ed &#239;&#172;?rst exon targeting predicted exon gene. Furthermore, injecting cosmids, T24D11             eat-17, able amplify single 2.5-kb fragment. F01G12, completely cover region, rescued          con&#239;&#172;?rm isolated length transcript, growth, feeding, grinder defects eat-17 mutants. Inject-     generated rescuing transgene 2.5-kb sequence <br /> ing did rescue phenotype (data             attached  5.6-kb region sequence upstream shown).                                                             predicted eat-17 transcriptional start site (eat-17 pro- <br />     determine gene structure eat-17, &#239;&#172;?rst tried        moter) 59 end 500-bp unc-54 39-UTR. isolate length transcripts intron/exon structure   construct injected eat-17 mutants,  <br />  <br />  <br />                                                                                               EAT-17 RabGAP RAB-6.2       119 <br />                                                                                                                         Figure 3 EAT-17 interacts                                                                                                                        RAB-6.2.   eat-17 rescue. Ex- <br />                                                                                                                        pressing wild-type eat-17 cDNA <br />                                                                                                                        construct mutant worms res- <br />                                                                                                                        cues defects grinder formation. <br />                                                                                                                        Grinder plates normal size <br />                                                                                                                        (yellow arrow)                                                                                                                        unground bacteria intes- <br />                                                                                                                        tine (red arrow). (B) Fluorescence <br />                                                                                                                        image  Smooth &#239;&#172;&#8218;uorescence <br />                                                                                                                        intestine indicates bac- <br />                                                                                                                        teria ground (red arrow). (C) <br />                                                                                                                        rab-6.2(ok2254) mutants                                                                                                                        grinder defect. Grinder  <br />                                                                                                                        med size small. (D) <br />                                                                                                                        eat-17(ad707) grown 15   <br />                                                                                                                        tainslarge refractile body                                                                                                                        terminal bulb (arrow). (E) Yeast <br />                                                                                                                        hybrid screen using EAT-17 <br />                                                                                                                        bait identi&#239;&#172;?ed RAB-6.2                                                                                                                        interacting Rab protein. (F)                                                                                                                        constitutively GTP bound form                                                                                                                        RAB-6.1 enhances binding                                                                                                                        EAT-17 RAB-6.1.                                                                                                                        hand, constitutively <span id='am-9' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-10' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span>GDP</span> bound <br />                                                                                                                        form RAB-6.2 (6.2T) abolishes <br />                                                                                                                        binding EAT-17 RAB-6.2. <br />                                                                                                                        EAT-17 construct used                                                                                                                        truncated protein (amino acids <br /> 1&#226;&#8364;&#8220;460) containing &#239;&#172;?rst coiled coil domain. truncated protein binds RAB-6.2 ef&#239;&#172;?ciently, suggesting &#239;&#172;?rst coiled-coil domain suf&#239;&#172;?cient binding. Abbreviations follows: 6.1, RAB-6.1 length wild type 6.1Q, RAB-6.1 Q70L mutation; 6.1T, RAB-6.1 T27N mutation; <br /> 6.2T, RAB-6.2 T27N mutation; 6.2, RAB-6.2 length wild type 6.2Q, RAB-6.2 Q69L mutation. *P , 0.05; NS, statistically signi&#239;&#172; Student t-test. Miller units calculated OD420/t &#194;&#183; V &#194;&#183; OD600. Bars, 10 mm. <br />  <br />  100% surviving F2 transgenic progeny rescued                      GAP) domain coiled-coil domains (CC1&#226;&#8364;&#8220;3, Figure defects grinders (Figure 3A).                                     2D). Rab GAP domain implicates EAT-17 vesicle traf- <br />    determine exact splicing pattern(s) eat-17,                    &#239;&#172;?cking, role consistent grinder defects eat-17 <br /> cloned ampli&#239;&#172;?ed PCR fragments vectors se-                         mutants. identify speci&#239;&#172;?c mutation eat-17(ad707), <br /> quenced inserts  Materials Methods). named                      sequenced predicted eat-17 coding region (based possible splice products EAT-17  b, c, d (Figure                     RT&#226;&#8364;&#8220;PCR results) C T base-pair transition <br /> 2C). EAT-17a intron/exon boundaries consistent                        nucleotide 607 exon 5 ad707 mutant allele, <br /> WormBase/GeneFinder predictions. EAT-17b uses alter-                        creating stop codon amino acid 203 (Figure 2D). native splice <span id='am-75' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-76' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span>acceptor</span> site exon 3/4 boundary (exon                     mutation generate protein truncated Rab- <br /> 4A, labeled red, Figure 2, B C).                      GAP domain, likely rendering nonfunctional. <br /> predicted exon 5 T24D11.1 truncated 180 nucleo-                         determine EAT-17 contains GAP activity, tides spliced directly predicted exon 2 F01G12.6                  mutated catalytic arginine residue (R116) WormBase. splice form contained predicted &#239;&#172;?rst                     lysine (K) alanine   abolish GAP activity exon F01G12.6. proteins encoded EAT-17a                          expressed mutated constructs eat-17 mutants  EAT-17b differ 3 amino acids. designed                         Materials Methods). total 21% F1&#226;&#8364;&#8482;s carrying set primers amplify sequences exon 1                       wild-type copy eat-17 rescued defects exon 6 (corresponding predicted &#239;&#172;?rst exon F01G12.6),                 grinder formation (Figure 3, B), 4.1 isolate transcripts included exon 6. named                    1.6% mutants expressing R116K R116A predicted transcripts include exon 6 EAT-17c                      mutations rescued (Table 1). Collectively, data <br /> EAT-17d (Figure 2C). GeneFinder predictions suggested                        catalytic activity EAT-17 critical splicing pattern create stop codon junc-                   function EAT-17 GAP protein. <br /> tion exon 5 exon 6. transcript  <br />                                                                                EAT-17 RAB-6.2 interact regulate <br /> code truncated protein lacking C terminus.                                                                                grinder formation likely b splice forms encoding  <br /> length proteins functional ones.                                       identify substrates GAP activity EAT-17,    Analysis predicted protein sequence revealed                     performed RNA interference 25 29 predicted <br /> conserved domains EAT-17: Rab GTPase activating (Rab                      Rab genes C. elegans genome (Audhya et al. 2007) <br />  <br />  <br />  <br /> 120     S. Straud et al. <br />  Table 1 GAP activity required EAT-17 function                             Table 2 RNAi rab-6.2 phenocopies eat-17 mutants <br />  <br />                         rescued        Total  examined           % rescue       dsRNA injected                                 Phenotype <br /> Wild-type eat-17             16                      76                    21                                     Wild type <br />   construct                                                                           rab-1                            Embryonic lethal <br /> R116K                          2                     49                    4.1        rab-2                            Slight growth delay <br /> R116A                          1                     64                    1.6        rab-3                            Wild type <br /> eat-17 mutants injected DNA constructs encoding wild-type                                                                                       rab-5                            Embryonic lethal mutant versions (R116K R116A) EAT-17 protein  Materials      rab-6.1                          Wild type <br /> Methods). F1 transgenic progeny blindly assayed rescue grinder defects.   rab-6.2                          Growth delay abnormal grinder <br /> GFP used identify transgenic progeny afterward.                                rab-7                            Wild type <br />                                                                                       rab-8                            Loopy movement, exaggerated body bends <br />                                                                                       rab-10                           NDa looked grinder phenotype similar eat-17                          rab-11.1                         Embryonic lethal <br />                                                                                       rab-11.2                         Arrested L1s <br /> mutants. RNAi performed worms mutant rrf-3, <br />                                                                                       rab-14                           growth delay, Dpyb hypersensitive background RNAi (Simmer et al. 2002).                            rab-18                           Growth delay, infrequent pumping 25 Rab genes tested, RNAi rab-6.2                                                                 normal grinder <br /> produced phenotype similar eat-17 (Table 2),                             rab-19                           Wild type <br /> rrf-3 mutants obvious defects grinder formation                            rab-21                           Sick, starved, strong Dpy <br />                                                                                       rab-27                           Wild type <br /> (data shown). examined rab-6.2 deletion <br />                                                                                       rab-28                           Wild type <br /> mutants, defects size organization                              rab-30                           Wild type <br /> grinder plates identical eat-17 mutants                              rab-33                           Slightly Uncc, wild type <br /> (Figures 1B 3C), con&#239;&#172;?rming RNAi results.                                      rab-35                           Wild type <br />     examine EAT-17 RAB-6.2 directly bind                            rab-37                           Loopy movement, wild type <br />                                                                                       rab-39                           Wild type binding speci&#239;&#172;?c,  <br />                                                                                       4R79.2                           Loopy movement, Unc, wild type <br /> formed directed yeast hybrid assays EAT-17                                   K02E10.1                         ND 25 RAB proteins. RABs,                                  F11A5.4                          Wild type RAB-6.2 interacted EAT-17 (Figure 3E, data                              F11A5.3                          Wild type <br /> shown). examine guanylate nucleotide binding                                C56E6.2                          Loopy movement, Unc <br /> status RAB-6.2 critical, introduce types muta-                        25 Rabs tested, RNAi rab-6.2 shows identical grinder phenotype                                                                                       eat-17. length dsRNAs Rab listed injected rrf-3 RNAi hyper- <br /> tions RAB-6.2: (1) glutamine (Q69) leucine (L)                                                                                       sensitive mutants. Worms grown 15  enhance phenotype. <br /> produce constitutively GTP bound form (2) threonine                                                                                                                     ND, determined. <br />                                                                                       b <br /> (T27) asparagine (N) produce constitutively GDP                                 Dpy, dumpy phenotype, worms shorter wild type. <br />                                                                                       c <br />                                                                                         Unc, uncoordinated movement phenotype, worms abnormally. <br /> bound form (Martinez et al. 1994, 1997). interaction EAT-17 RAB-6.2 completely abolished T27N mutation introduced (Figure 3E, 6.2T),                              mammalian RabGAP, interacts Rab6 coiled- enhanced Q69L mutation                                coil domain (Cuif et al. 1999). tested trun- <br /> (Figure 3C, 6.2Q), compared wild-type RAB-6.2 (Figure                           cated EAT-17 constructs, EAT-17 (aa 1&#226;&#8364;&#8220;316), EAT-17 (aa <br /> 3F, 6.2). suggests reaction reached maximum                             1&#226;&#8364;&#8220;364), EAT-17 (aa 1&#226;&#8364;&#8220;405), did interact binding wild-type RAB-6.2                               RAB-6.2, showing GAP domain suf&#239;&#172;?- <br /> hance binding conditions                            cient interaction (data shown). construct performed assay.                                                                  EAT-17 containing &#239;&#172;?rst coiled-coil domain [EAT-17 (aa <br />     RAB-6.1, Rab6 C. elegans genome,                            1&#226;&#8364;&#8220;460)],  shows signi&#239;&#172; interaction RAB-6.2 <br /> shares 81% amino acid sequence identity RAB-6.2.                              (Figure 3E). Collectively, results GAP <br /> cause high similarity Rab6&#226;&#8364;&#8482;s,                               domain EAT-17 mediates catalytic activity tested EAT-17 bind RAB-6.1  Al-                              &#239;&#172;?rst coiled-coil domain mediates interaction did detect binding wild-type                                RAB-6.2. Presumably, coiled-coil domains impor- <br /> RAB-6.1 EAT-17 assay, RAB-6.1(Q70L) (equiva-                               tant provide EAT-17 speci&#239;&#172;?city RAB-6.2. <br /> lent Q69L RAB-6.2) showed enhanced binding EAT-                              Altogether, data suggest conserved structural <br /> 17, suggesting EAT-17 binds RAB-6&#226;&#8364;&#8482;s                               mechanism interaction Rab6&#226;&#8364;&#8482;s Rab <br /> conserved guanylate nucleotide binding site. Taken                           GAP proteins coiled-coil domain (Cuif et al. data strongly support hypothesis RAB-6.2                            1999). speci&#239;&#172;?c substrate EAT-17 EAT-17 promotes <br />                                                                                       EAT-17 RAB-6.2 expressed terminal bulb <br /> hydrolysis GTP binding active (GTP bound) form <br />                                                                                       muscle, site grinder secretion RAB-6. <br />     studies identi&#239;&#172;?ed EAT-17 GAP protein                          determine EAT-17 RAB-6.2 function  <br /> coiled-coil domains. Coiled-coil domains generally                             gether terminal bulb, examined expression <br /> volved protein&#226;&#8364;&#8220;protein interactions, GAPCenA,                                patterns transgenic lines carry promoters  <br />  <br />  <br />                                                                                                                          EAT-17 RabGAP RAB-6.2             121 <br />                                                                                                     Figure 4 EAT-17 expressed                                                                                                    pharyngeal muscle, intestine,                                                                                                    vulva.  B) Corresponding <br />                                                                                                    GFP DIC images EAT-17:: <br />                                                                                                    GFP. (C D) Corresponding GFP <br />                                                                                                    DIC images RAB-6.2::GFP. <br />                                                                                                    EAT-17 RAB-6.2                                                                                                    highly expressed pharyn- <br />                                                                                                    geal muscle. RAB-6-2                                                                                                    highly expressed neurons. (E <br />                                                                                                    F) Corresponding GFP                                                                                                    DIC images EAT-17::GFP                                                                                                    pharynx (red arrows), intes- <br />                                                                                                    tine (white arrows), vulva <br />                                                                                                    (yellow arrows). Figure 4, D, <br />                                                                                                    bars, 10 mm. Figure 4, E F, <br />                                                                                                    bars, 30 mm. <br />  <br />  <br />  <br />  <br /> gene fused GFP  Materials Methods).            bodies share similar refractile characteristics EAT-17 RAB-6.2 strongly expressed terminal        grinder. results suggest materials traf- <br /> bulb muscles secrete components grinder          &#239;&#172;?cked RAB-6.2 terminal bulb compo- <br /> (Figure 4, D, Materials Methods). construct        nents grinder refractile bodies eat-17 shows EAT-17 expression outside pharynx,        components, accumulating muscle  intestine vulva (Figure 4, E F).                   cause traf&#239;&#172;?cking defect. <br />    tissue expression pattern EAT-17 overlaps largely previously reported RAB-6.2 (Zhang et al.          Discussion <br /> 2012), supporting inference EAT-17 speci&#239;&#172;?c <br /> GAP regulate RAB-6.2 activity tissues.               study mapped characterized eat-17, new <br />                                                                  GTPase-activating protein gene C. elegans. Using dif- <br /> Chitin deposition super&#239;&#172;?cially intact eat-17 <br />                                                                  ferent approaches, RNA interference yeast hybrid, rab-6.2 mutants <br />                                                                  showed EAT-17 interacts RAB-6.2, conserved <br /> Chitin provides mechanical support exoskeletons       small G protein regulating membrane traf&#239;&#172;?cking Golgi <br /> invertebrate animals. yeast, proper chitin deposition      (Del Nery et al. 2006; Girod et al. 1999). fact essential strictly regulated controlling traf&#239;&#172;?cking   approaches identify RAB-6.2 substrate EAT-17 synthase, chs3p (Schorr et al. 2001). chitin   25 tested Rabs strongly suggests EAT-17 speci&#239;&#172;?c components grinder synthase         GAP RAB-6.2. eat-17 rab-6.2 mutants regulated traf&#239;&#172;?cking, stained grinders eat-17       identical defects grinders, feeding rab-6.2 mutants chitin examine chitin deposi-      organ crushes food,   essential <br /> tion misregulated mutants. Despite global        rapid growth normal development. new grinder <br /> disorganization grinder, overall chitin deposition    replaces old molt, implying indistinguishable eat-17, rab-6.2, wild-type       grinder components needed transported (Figure 5, F).  grown low tempera-           molt build new  observations mutation ture, eat-17 mutants occasionally large refractory bod-     rab-6.2, functions cargo traf&#239;&#172;?cking, mutation <br /> ies terminal bulbs secrete components     GAP protein EAT-17 striking phenotypes <br /> grinder (Figure 3D). Interestingly, appearance,      grinder, suggest precise regulation traf&#239;&#172;?cking <br />  <br />  <br />  <br /> 122    S. Straud et al. <br />                                                                                                    Figure 5 Chitin deposition                                                                                                   grinders eat-17 rab-6.2 <br />                                                                                                   mutants normal.  B) <br />                                                                                                   Chitin staining grinder                                                                                                   wild type corresponding DIC <br />                                                                                                   image. (C D) eat-17 <br />                                                                                                   mutants corresponding DIC <br />                                                                                                   image. (E F) rab-6.2 <br />                                                                                                   mutants corresponding DIC <br />                                                                                                   image. Arrowheads indicate                                                                                                   position grinder. Bars, 10 mm. <br />  <br />  <br />  <br />  grinder components critical build functional         worms super&#239;&#172;?cially normal, rab-6.2(RNAi) worms <br /> grinder.                                                        defects grinder. Knockout genes, <br />    know identities cargos       produces gross defects molting growth. RAB-6.2 targeted vesicles, eat-17 mutants         fact EAT-17 does interact RAB-6.1 regulate <br /> grown 15  refractory bodies terminal bulbs.      grinder formation similarity eat-17 phenotype bodies appear similar  e., highly refractile)      rab-6.2 single mutant rab-6.2(RNAi) consistent <br /> grinder. suggests bodies grinder com-    proposed EAT-17 speci&#239;&#172;?city RAB-6.2. <br /> ponents accumulating muscle slow traf-           Recently Zhang et al. (2012) identi&#239;&#172;?ed LIN-10 RAB- <br /> &#239;&#172;?cking low temperature. fact, Chotard et al. (2010)       6.2 effector regulate glutamate receptor recycling C. <br /> showed mutant tbc-2, encodes          elegans neurons.  did grinder defect <br /> RabGAP C. elegans, accumulates enormous RAB-7-               lin-10 mutants (data shown). addition, positive late endosome intestine containing refractile   detect EAT-17 expression neurons LIN-10 <br /> materials. results strongly suggest refractile   interacts RAB-6.2. does exclude bodies common phenotype endosomal traf&#239;&#172;?cking        possibility LIN-10 functions effector RAB-6.2 <br /> mutants EAT-17 regulates endosomal traf&#239;&#172;?cking.         EAT-17 pathway places, LIN-10 does obvi- <br />    Small G proteins substrates GAPs fast       ously function grinder formation. suggests LIN-10 <br /> turnover. importance GAPs demonstrated        affects RAB-6.2 function probably speci&#239;&#172;?cally neurons, <br /> examples oncogene Evi5, homologous         supporting notion differential expression speci&#239;&#172;?c EAT-17 (Liao et al. 1997; Dabbeekeh et al. 2007).       GEFs GAPs different places different times GAP Rab6 identi&#239;&#172;?ed far, GAPCenA,      provides spatial temporal speci&#239;&#172;?city G proteins interacts Rab6 coiled-coil domain.    (Segev Kahn 2012). requirement coiled-coil domain interaction          mutations Rab GAPs cause appears unique Rab6 GAPs. Studies shown      identical phenotype demonstrates critical precise class Rab6 effector, Golgins, interacts Rab6&#226;&#8364;&#8482;s   kinetics Rab provided GAPs  Furthermore, coiled-coil domains direct vesicle capture     implies identify Rab interactors sorting Golgi. possible Rab6&#226;&#8364;&#8482;s bind   GAPs looking mutants exhibiting phenotypes GAPs effectors similar mechanism         similar Rab mutants. 20 TBC GAP domain- <br /> mediated coiled-coil domains (Sinka et al. 2008).            containing proteins, characterized <br />    discovery EAT-17 speci&#239;&#172;?city RAB-       (Mukhopadhyay et al. 2007; Chotard et al. 2010). &#239;&#172;?nd- <br /> 6.2 strongly suggest speci&#239;&#172;?c GAPs      ing provides insight useful discovering <br /> discovered RAB-6.1, Rab6 C. elegans,      GAPs Rabs. addition, using Eat phe- <br /> sharing 81% amino acid sequence identity RAB-6.2.          notype caused grinder defect eat-17 mutants, rab-6.1 rab-6.2 partially redundant; rab-6.1(RNAi)      identify interactors RAB-6.2 effectors <br />  <br />  <br />  <br />                                                                                            EAT-17 RabGAP RAB-6.2         123 <br />  genetic screens isolate suppressors enhancers              Ho, S. N., H. D. Hunt, R. M. Horton, J. K. Pullen, L. R. Pease, eat-17 mutants.                                                          1989 Site-directed mutagenesis overlap extension using                                                                             polymerase chain reaction. Gene 77: 51&#226;&#8364;&#8220;59. <br />                                                                          Houalla, T., L. Shi, D. J. van Meyel, Y. Rao, 2010 Rab-mediated <br /> Acknowledgments                                                             vesicular transport required neuronal positioning  <br />                                                                             veloping Drosophila visual  Mol. Brain 3: 19. strains used studies provided               Jasmin, B. J., B. Goud, G. Camus, J. Cartaud, 1992 low <br /> Caenorhabditis Genetics Center, funded National                 molecular weight <span id='am-40' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-41' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>guanosine</span> <span id='am-45' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-46' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span>-binding protein <br /> Institutes Health (NIH) Of&#239;&#172;?ce Research Infrastructure                 Rab6p associates distinct post-Golgi vesicles Torpedo <br />                                                                             marmorata electrocytes. Neuroscience 49: 849&#226;&#8364;&#8220;855. <br /> Programs (P40 OD010440). thank Melanie Cobb                                                                          Jordens, , M. Marsman, C. Kuijl, J. Neefjes, 2005 Rab pro- <br /> Rueyling Lin DNA constructs reagents David                      teins, connecting transport vesicle fusion. Traf&#239;&#172;?c 6: 1070&#226;&#8364;&#8220; <br /> Raizen, Jim McKay, Scott Cameron, Pamela Marshall                       1077. invaluable discussions. work supported NIH               Kamath, R. S., J. Ahringer, 2003 Genome-wide RNAi screen- <br /> grant HL46154 (L ) 09SDG2150070 Ameri-                       ing Caenorhabditis elegans. Methods 30: 313&#226;&#8364;&#8220;321. <br />                                                                          Liao, X., Y. Du, H. C. Morse, 3rd, N.  Jenkins, N. G. Copeland, Heart Association (Y.-J.Y.). <br />                                                                             1997 Proviral integrations Evi5 locus disrupt novel 90 <br />                                                                             kDa protein homology Tre2 oncogene cell-cycle <br />                                                                             regulatory proteins. Oncogene 14: 1023&#226;&#8364;&#8220;1029. <br /> Literature Cited                                                         Martinez, O.,  Schmidt, J. Salamero, B. Ho&#239;&#172;&#8218;ack, M. Roa et al., <br />                                                                             1994 small GTP-binding protein rab6 functions intra- <br /> Albertson, D. G., J. N. Thomson, 1976 pharynx Cae-               Golgi transport. J. Cell Biol. 127: 1575&#226;&#8364;&#8220;1588. <br />    norhabditis elegans. Philos. Trans. R. Soc. Lond. B Biol. Sci. 275:   Martinez, O., C. Antony, G. Pehau-Arnaudet, E. G. Berger, J. Salamero <br />    299&#226;&#8364;&#8220;325.                                                                 et al., 1997 GTP-bound forms rab6 induce redistribution Audhya, ,  Desai, K. Oegema, 2007 role Rab5                Golgi proteins endoplasmic reticulum. Proc. Natl. Acad. <br />    structuring endoplasmic reticulum. J. Cell Biol. 178: 43&#226;&#8364;&#8220;56.         Sci. USA 94: 1828&#226;&#8364;&#8220;1833. <br /> Avery, L., 1993 genetics feeding Caenorhabditis elegans.       Mukhopadhyay, , X. Pan, D. G. Lambright, H.  Tissenbaum, <br />    Genetics 133: 897&#226;&#8364;&#8220;917.                                                   2007 endocytic pathway target tubby regulation <br /> Avery, L., B. B. Shtonda, 2003 Food transport C. ele-            fat storage. EMBO Rep. 8: 931&#226;&#8364;&#8220;938. <br />    gans pharynx. J. Exp. Biol. 206: 2441&#226;&#8364;&#8220;2457.                           Opdam, F. J.,  Echard, H. J. Croes, J.  van den Hurk, R.  van Avery, L.  Y.J. C. elegans feeding  21, 2012),                  Vorstenbosch et al., 2000 small GTPase Rab6B, novel <br />    WormBook, ed. C. elegans Research Community, WormBook,               Rab6 subfamily member, cell-type speci&#239;&#172;?cally expressed    doi/10.1895/wormbook.1.150.1, http://www.wormbook.org.                   localised Golgi apparatus. J. Cell Sci. 113(Pt 15): 2725&#226;&#8364;&#8220; <br /> Blumenthal, T., 1995 Trans-splicing polycistronic transcrip-            2735. <br />    tion Caenorhabditis elegans. Trends Genet. 11: 132&#226;&#8364;&#8220;136.            Pamonsinlapatham, P., R. Hadj-Slimane, Y. Lepelletier, B. Allain, M. <br /> Chotard, L.,  K. Mishra, M.  Sylvain, S. Tuck, D. G. Lambright          Toccafondi et al., 2009 P120-Ras GTPase activating protein <br />    et al., 2010 TBC-2 regulates RAB-5/RAB-7-mediated endoso-                (RasGAP): multi-interacting protein downstream signaling. <br />    mal traf&#239;&#172;?cking Caenorhabditis elegans. Mol. Biol. Cell 21:            Biochimie 91: 320&#226;&#8364;&#8220;328. <br />    2285&#226;&#8364;&#8220;2296.                                                            Raizen, D., B. M. Song, N. Trojanowski, Y. J.  2012 Wormbook, <br /> Conrad, R., K. Lea, T. Blumenthal, 1995 SL1 trans-splicing              Methods measuring pharyngeal behaviors. Available http:// <br />    speci&#239;&#172;?ed AU-rich synthetic RNA inserted 59 end              www.ncbi.nlm.nih.gov/pubmed/23255345. <br />    Caenorhabditis elegans pre-mRNA. RNA 1: 164&#226;&#8364;&#8220;170.                      Ruvkun, G., M. Finney, 2005 Antibody staining <span id='am-65' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>- <br /> Cuif, M. H., F. Possmayer, H. Zander, N. Bordes, F. Jollivet et al.,        &#239;&#172;?xed animals. Available http://www.wormatlas.org/ <br />    1999 Characterization GAPCenA, GTPase activating pro-               EMmethods/Antibodystaining.htm Wormatlas, edited <br />    tein Rab6, associates centrosome.             D. Hall. WormBook. <br />    EMBO J. 18: 1772&#226;&#8364;&#8220;1782.                                                Schorr, M.,   S. Tahirovic, N. Hug, P. Mayinger, <br /> Dabbeekeh, J. T., S. L. Faitar, C. P. Dufresne, J. K. Cowell,           2001 <span id='am-21' about='obo:IMR_0001650' typeof='owl:Thing obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001650" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="inositol phospholipid" datatype="xsd:string"></span>phosphoinositide</span> phosphatase Sac1p controls traf- <br />    2007 EVI5 TBC domain provides GTPase-activating                  &#239;&#172;?cking yeast Chs3p chitin synthase. Curr. Biol. 11: 1421&#226;&#8364;&#8220; <br />    protein motif RAB11. Oncogene 26: 2804&#226;&#8364;&#8220;2808.                         1426. <br /> Del Nery, E., S. Miserey-Lenkei, T. Falguieres, C. Nizak, L. Johannes    Segev, N., R.  Kahn, 2012 Kinetic cell-based analyses    et al., 2006 Rab6A Rab6A&#226;&#8364;&#8482; GTPases play non-overlapping               GTPase regulators. Cell. Logist. 2: 138&#226;&#8364;&#8220;139. <br />    roles membrane traf&#239;&#172;?cking. Traf&#239;&#172;?c 7: 394&#226;&#8364;&#8220;407.                      Simmer, F., M. Tijsterman, S. Parrish, S. P. Koushika, M. L. Nonet <br /> Durkin, M. E., B.  Yuan, X. Zhou, D. B. Zimonjic, D. R. Lowy et al.,      et al., 2002 Loss putative RNA-directed RNA polymerase <br />    2007 DLC-1 Rho GTPase-activating protein tumour sup-               RRF-3 makes C. elegans hypersensitive RNAi. Curr. Biol. 12: <br />    pressor. J. Cell. Mol. Med. 11: 1185&#226;&#8364;&#8220;1207.                               1317&#226;&#8364;&#8220;1319. <br /> Girod, , B. Storrie, J. C. Simpson, L. Johannes, B. Goud et al.,       Sinka, R.,  K. Gillingham, V. Kondylis, S. Munro, 2008 Golgi <br />    1999 Evidence COP independent transport route               coiled-coil proteins contain multiple binding sites Rab family <br />    Golgi complex endoplasmic reticulum. Nat. Cell Biol.          G proteins. J. Cell Biol. 183: 607&#226;&#8364;&#8220;615. <br />    1: 423&#226;&#8364;&#8220;430.                                                           Sulston, J. E., J. G. Hodgkin, 1988 Methods, pp. 587&#226;&#8364;&#8220;606 <br /> Grosshans, B. L., D. Ortiz, P. Novick, 2006 Rabs              Nematode Caenorhabditis elegans, edited W. B. <br />    effectors: achieving speci&#239;&#172;?city membrane traf&#239;&#172;?c. Proc. Natl.          Wood Cold Spring Harbor Laboratory Press, Cold Spring <br />    Acad. Sci. USA 103: 11821&#226;&#8364;&#8220;11827.                                         Harbor, NY. <br /> Hama, H., G. G. Tall, B. F. Horazdovsky, 1999 Vps9p               Tanabe, K., S. Kon, W. Natsume, T. Torii, T. Watanabe et al., <br />    guanine nucleotide exchange factor involved vesicle-                2006 Involvement novel <span id='am-15' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span>ADP</span>-ribosylation factor GTPase- <br />    mediated vacuolar protein transport. J. Biol. Chem. 274: 15284&#226;&#8364;&#8220;          activating protein, SMAP, membrane traf&#239;&#172;?cking: implications <br />    15291.                                                                   cancer cell biology. Cancer Sci. 97: 801&#226;&#8364;&#8220;806. <br />  <br />  <br />  <br /> 124     S. Straud et al. <br />  Uytterhoeven, V., S. Kuenen, J. Kasprowicz, K. Miskiewicz, P.    Y. J., J. Kim, D. M. Raizen, L. Avery, 2008 Insulin, <span id='am-80' about='obo:IMR_0100025' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-81' property="rdfs:label" content="Guanosine 3&apos;,5&apos;-cyclic phosphate" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Cyclic GMP" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0100025" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="3&apos;,5&apos;-Cyclic GMP" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="cGMP" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Guanosine 3&apos;,5&apos;-cyclic monophosphate" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasAlternativeId" content="IMR:0200372" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="Guanosine 3&apos;,5&apos;-cyclic phosphate" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Cyclic GMP" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic GMP" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Guanosine 3&apos;,5&apos;-cyclic monophosphate" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="KEGG:C00942" datatype="xsd:string"></span>cGMP</span>, <br />    Verstreken, 2011 Loss skywalker reveals synaptic endo-           TGF-beta signals regulate food intake quiescence    somes sorting stations synaptic vesicle proteins. Cell       C. elegans: model satiety. Cell Metab. 7: 249&#226;&#8364;&#8220;257. <br />    145: 117&#226;&#8364;&#8220;132.                                                    Zerial, M., H. McBride, 2001 Rab proteins membrane  <br /> von Lieven,  F., 2003 Functional morphology evolutionary         ganizers. Nat. Rev. Mol. Cell Biol. 2: 107&#226;&#8364;&#8220;117. <br />    origin pharynx nematodes. Zoology (Jena)    Zhang, D., N. R. Isack, D. R. Glodowski, J. Liu, C. C. Chen et al., <br />    106: 183&#226;&#8364;&#8220;201.                                                       2012 RAB-6.2 retromer regulate glutamate receptor <br /> Wicks, S. R., R. T. Yeh, W. R. Gish, R. H. Waterston, R. H.        recycling retrograde pathway. J. Cell Biol. 196: <br />    Plasterk, 2001 Rapid gene mapping Caenorhabditis elegans         85&#226;&#8364;&#8220;101. <br />    using high density polymorphism map. Nat. Genet. 28: 160&#226;&#8364;&#8220; <br />    164.                                                                                         Communicating editor: D.  Greenstein <br />  <br />  <br />  <br />  <br />                                                                                                 EAT-17 RabGAP RAB-6.2     125 <br />              GENETICS <br />                                                               Supporting Information <br />            http://www.genetics.org/lookup/suppl/doi:10.1534/genetics.113.152538/-/DC1 <br />  <br />  <br />  <br />  Jaw Worm: GTPase-activating Protein <br />         EAT-17 Regulates Grinder Formation                        Caenorhabditis elegans <br />                  Sarah Straud, Inhwan Lee, Bomi Song, Leon Avery, Young-Jai  <br />  <br />  <br />  <br />                                                Copyright &#194;&#169; 2013 Genetics Society America <br />                                                                DOI: 10.1534/genetics.113.152538 <br />    Dm CG11727-PA    -----------------------------------------------MTLTTTTTASSAE   13 <br />   Dm CG11727-PB    -----------------------------------------------MTLTTTTTASSAE   13 <br />   EAT-17a          -----------------------------------------------MAATAALRRSSDY   13 <br />   EAT-17b          -----------------------------------------------MAATAALRRSSDY   13 <br />   Hs Evi5/NB4S     MVTNKMTAAFRNPSGKQVATDKVAEKLSSTLSWVKNTVSHTVSQMASQVASPSTSLHTTS   60 <br />   Mm Evi5          MVTTKMTAAFRNPNRRQVATDKVAEKLSSTLSWVKNTVSHTVSQMASQVASPSASLHTTS   60 <br />                                                                    . :.:    : <br />  <br />    Dm CG11727-PA   SQAKMDVKGGALPGEENLPTSEMDLLAKLEAANKLIESDAKSLNSLHSTHSRKNSDTSQI   73 <br />    Dm CG11727-PB   SQAKMDVKGGALPGEENLPTSEMDLLAKLEAANKLIESDAKSLNSLHSTHSRKNSDTSQI   73 <br />    EAT-17a         GSADTECGEPCETG-APVSLNEVDLLAKMEQLNKSNEEDSRSVASKKTGSS--ESRKGAR   70 <br />    EAT-17b         GSADTECGEPCETG-APVSLNEVDLLAKMEQLNKSNEEDSRSVASKKTGSS--ESRKGAR   70 <br />    Hs Evi5/NB4S    SSTTLSTPALSPSSPSQLSPDDLELLAKLEEQNRLLETDSKSLRSVNGSRRNSGSSLVSS   120 <br />    Mm Evi5         SSTTLSTPTQSPSSPSKLSPDDLELLAKLEEQNRLIETDSKSLRSVNGSRRNSGSSLVSS   120 <br />                    ..: .     . ..   :. .:::****:* *: * *::*: * :         * <br />  <br />    Dm CG11727-PA   SLTSSGNSVAEEDIWTTWATILNDWEGALKRKNPCVSELVRRGIPHHFRAIVWQQLSGAS   133 <br />    Dm CG11727-PB   SLTSSGNSVAEEDIWTTWATILNDWEGALKRKNPCVSELVRRGIPHHFRAIVWQQLSGAS   133 <br />    EAT-17a         EHSPEEDE---EDLWSVWGELILNWEIEVKKRPNYIKDLVKRGIPQHFRMIAWQNLSNAS   127 <br />    EAT-17b         EHSPEEDEYFQEDLWSVWGELILNWEIEVKKRPNYIKDLVKRGIPQHFRMIAWQNLSNAS   130 <br />    Hs Evi5/NB4S    SSASSNLSHLEEDSWILWGRIVNEWEDVRKKKEKQVKELVHKGIPHHFRAIVWQLLCSAQ   180 <br />    Mm Evi5         SSASSNLSHLEEDSWILWGRIVNEWDDVRKKKEKQVKELVRKGIPHHFRAIVWQLLCNAQ   180 <br />                    . :.. .    ** * *. :: :*:    *::   :.:**::***:*** *.** *..*. <br />  <br />    Dm CG11727-PA   DGDKK-QYAEYIKATSACEKVIRRDIARTYPEVEFFKEKDGPGQEALFNVIKAYSLHDRE   192 <br />    Dm CG11727-PB   DGDKK-QYAEYIKATSACEKVIRRDIARTYPEVEFFKEKDGPGQEALFNVIKAYSLHDRE   192 <br />    EAT-17a         VSSVHDLYSDYMRQSSVYEKVIQRDIPRTYPELDFFKDGER-GQSLLFNVIKAYSVHDKE   186 <br />    EAT-17b         VSSVHDLYSDYMRQSSVYEKVIQRDIPRTYPELDFFKDGER-GQSLLFNVIKAYSVHDKE   189 <br />    Hs Evi5/NB4S    SMPIKDQYSELLKMTSPCEKLIRRDIARTYPEHNFFKEKDSLGQEVLFNVMKAYSLVDRE   240 <br />    Mm Evi5         SMTIKDQYSELLKMTSPCEKLIRRDIARTYPEHNFFKEKDSLGQEVLFNVMKAYSLVDRE   240 <br />                        : *:: :: :* **:*:***.***** :***: : **. ****:****: *:* <br />  <br />    Dm CG11727-PA   VGYCQGSGFIVGLLLMQMPEEEAFAVLVQIMQQHRMRHMFKPSMSELGLCMYQLENLVQE   252 <br />    Dm CG11727-PB   VGYCQGSGFIVGLLLMQMPEEEAFAVLVQIMQQHRMRHMFKPSMSELGLCMYQLENLVQE   252 <br />    EAT-17a         VGYCQGSAFIVGLLLLQMPEEEAFAVLVSLMENYRLRELYKPTMTDLGLCMFQLECLVQD   246 <br />    EAT-17b         VGYCQGSAFIVGLLLLQMPEEEAFAVLVSLMENYRLRELYKPTMTDLGLCMFQLECLVQD   249 <br />    Hs Evi5/NB4S    VGYCQGSAFIVGLLLMQMPEEEAFCVFVKLMQDYRLRELFKPSMAELGLCMYQFECMIQE   300 <br />    Mm Evi5         LVTVRAVLSSLDCCCMQMPEEEAFCVFVKLMQDYRLRELFKPSMAELGLCMYQFECMIQE   300 <br />                    :   :.    :.   :********.*:*.:*:::*:*.::**:*::*****:*:* ::*: <br />  <br />    Dm CG11727-PA   QIPDMHIHFQQQGFQTTMYASSWFLTLYTTTLNVNLSCRIMDVFLSEGMEFIFKVALALL   312 <br />    Dm CG11727-PB   QIPDMHIHFQQQGFQTTMYASSWFLTLYTTTLNVNLSCRIMDVFLSEGMEFIFKVALALL   312 <br />    EAT-17a         QMPDLYTHFNNMGFDTSMYASSWFLTLFTTTMPLDIANRIMDCFLVEGMDFIFCISIAIL   306 <br />    EAT-17b         QMPDLYTHFNNMGFDTSMYASSWFLTLFTTTMPLDIANRIMDCFLVEGMDFIFCISIAIL   309 <br />    Hs Evi5/NB4S    HLPELFVHFQSQSFHTSMYASSWFLTIFLTTFPLPVATRIFDIFMSEGLEIVFRVGLALL   360 <br />    Mm Evi5         YLPELFVHFQSQSFHTSMYASSWFLTIFLTTFPLPIATRIFDIFMSEGLEIVFRVGLALL   360 <br />                     :*::. **:. .*.*:*********:: **: : :: **:* *: **::::* :.:*:* <br />  <br />    Dm CG11727-PA   LTGKDTLLCLDMEAMLKFFQKELPGRVEADVEGFFNLAYSIKLNTKRMKKMEKEYQDLKK   372 <br />    Dm CG11727-PB   LTGKDTLLCLDMEAMLKFFQKELPGRVEADVEGFFNLAYSIKLNTKRMKKMEKEYQDLKK   372 <br />    EAT-17a         QQARIELLRLDMEGMLKYFQREVRERYEFDADLLFTVANQVQLNAKRMKRLEKDYLTKRT   366 <br />    EAT-17b         QQARIELLRLDMEGMLKYFQREVRERYEFDADLLFTVANQVQLNAKRMKRLEKDYLTKRT   369 <br />    Hs Evi5/NB4S    QMNQAELMQLDMEGMLQHFQKVIPHQFDGVPDKLIQAAYQVKYNSKKMKKLEKEYTTIKT   420 <br />    Mm Evi5         QMNQAELMQLDMEGMLQHFQKVIPHQFDGGPEKLIQSAYQVKYNSKKMKKLEKEYTTIKT   420 <br />                       : *: ****.**:.**: : : :     : :: * .:: *:*:**::**:*    :. <br />  <br />    Dm CG11727-PA   KEQEEMAELRRLRRENCLLKQRNELLEAESAELADRLVRGQVSRAEEEETSYAIQTELMQ   432 <br />    Dm CG11727-PB   KEQEEMAELRRLRRENCLLKQRNELLEAESAELADRLVRGQVSRAEEEETSYAIQTELMQ   432 <br />    EAT-17a         KEQEEAVELRRLRTENRLLRQRIDYLEAESSALADRLVKGQVNLAQEAENYINIAHELNK   426 <br />    EAT-17b         KEQEEAVELRRLRTENRLLRQRIDYLEAESSALADRLVKGQVNLAQEAENYINIAHELNK   429 <br />    Hs Evi5/NB4S    KEMEEQVEIKRLRTENRLLKQRIETLEKH-----------KCSSNYNEDFVLQLEKELVQ   469 <br />    Mm Evi5         KEMEEQGEIKRLRTENRLLKQRIETLEKH-----------KCSSTYNEDFVLQLEKELVQ   469 <br />                    ** ** *::*** ** **:** : ** .            : .   : :    : ** : <br />  <br />    Dm CG11727-PA   LRRSYLEVSHQLENANEEVRGLSLRLQENNNSRQSSIDELCMKEEALKQRDEMVSCLLEE   492 <br />    Dm CG11727-PB   LRRSYLEVSHQLENANEEVRGLSLRLQENNNSRQSSIDELCMKEEALKQRDEMVSCLLEE   492 <br />    EAT-17a         LRDMNSDVHRKLEGAYETIRELSSARRDNIMDTGTQVDDTSMIEHIHS--------LQQE   478 <br />    EAT-17b         LRDMNSDVHRKLEGAYETIRELSSARRDNIMDTGTQVDDTSMIEHIHS--------LQQE   481 <br />    Hs NB4S/Evi5    ARLSEAESQCALKEMQDKV--LDIEKRNNSLPDENNIAR-----------------LQEE   510 <br />    Mm Evi5         ARLSEAESQCALKEMQDKV--LDIEKKNNSFPDENNIAR-----------------LQEE   510 <br />                     *    :    *:   : : *.    ::*     ..:                   * :* <br />  <br />  <br />  <br />  <br /> 2 SI                                        S. Straud et al. <br />   Dm CG11727-PA            LVKVRQGLAESEDQIRNLKAKVEELEEDKKTLRETTP--------DNSVAHLQDELIASK                      544 <br />  Dm CG11727-PB            LVKVRQGLAESEDQIRNLKAKVEELEEDKKTLRETTP--------DNSVAHLQDELIASK                      544 <br />  EAT-17a                  LIEAHTRQADSENTLRDAKLRVSELEMANKRLLENEP--------SEDVAGLQEELISVK                      530 <br />  EAT-17b                  LIEAHTRQADSENTLRDAKLRVSELEMANKRLLENEP--------SEDVAGLQEELISVK                      533 <br />  Hs NB4S/Evi5             LIAVKLREAEAIMGLKELRQQVKDLEEHWQRHLARTTGRWKDPPKKNAMNELQDELMTIR                      570 <br />  Mm Evi5                  LIAVKLREAEAIMGLKELRQQVRTLEEHWQRHLARTSGRWKDPPKKNAVNELQDELMSIR                      570 <br />                           *: .:   *::   ::: : :* **    :      .        .: : **:**:: : <br />  <br />  Dm CG11727-PA            LREAEASLSLKDLKQRVQELSSQWQRQLAE----------NQRSESERTTNAVDSTP---                      591 <br />  Dm CG11727-PB            LREAEASLSLKDLKQRVQELSSQWQRQLAE----------NQRSESERTTNAVDSTP---                      591 <br />  EAT-17a                  MREAESSLALKEMRQRLAELEQHWAKYVHVRAFDPSSASIEKESTSEAHSTQQQPSPPLT                      590 <br />  EAT-17b                  MREAESSLALKEMRQRLAELEQHWAKYVHVRAFDPSSASIEKESTSEAHSTQQQPSPPLT                      593 <br />  Hs NB4S/Evi5             LREAETQAEIREIKQRMMEMETQNQINSNH----------LRRAEQEVISLQEKVQYLSA                      620 <br />  Mm Evi5                  LREAETQAEIREMKQRMMEMETQNQINSNQ----------LRRAEQEVNSLQEKVCSLSV                      620 <br />                           :****:. :::::**: *:. :                   :.: .* :    . <br />  <br />  Dm CG11727-PA            ------KKLLTNFFD-SSKSSE---HTQKLEEELMTTRIREMETLTELKELRLKVMELET                      641 <br />  Dm CG11727-PB            ------KKLLTNFFD-SSKSSE---HTQKLEEELMTTRIREMETLTELKELRLKVMELET                      641 <br />  EAT-17a                  SARARLAKITASLIGGSTEETDNCISVRELEDQLMGVRIKEADTLAELKEMRQKVMELET                      650 <br />  EAT-17b                  SARARLAKITASLIGGSTEETDNCISVRELEDQLMGVRIKEADTLAELKEMRQKVMELET                      653 <br />  Hs NB4S/Evi5             QN----KGLLTQLSEAKRKQAE---IECKNKEEVMAVRLREADSIAAVAELRQHIAELEI                      673 <br />  Mm Evi5                  KN----KGLLAQLSEAKRRQAE---IECKNKEEVMAVRLREADSIAAVAELQQHIAELKI                      673 <br />                                   : :.:   . ..::      : ::::* .*::* :::: : *:: :: **: <br />  <br />  Dm CG11727-PA            QVQVSTNQLRRQDEEHKKLKEELEMAVTREKDMSNKAREQQHRYSDLESRMKDELMNVKI                      701 <br />  Dm CG11727-PB            QVQVSTNQLRRQDEEHKKLKEELEMAVTREKDMSNKAREQQHRYSDLESRMKDELMNVKI                      701 <br />  EAT-17a                  QNHVCTNQLKRQDEEMKRVREDSEVLVKKRKELEDQLKDEKEKLDNKESEFNEGRINDRL                      710 <br />  EAT-17b                  QNHVCTNQLKRQDEEMKRVREDSEVLVKKRKELEDQLKDEKEKLDNKESEFNEGRINDRL                      713 <br />  Hs NB4S/Evi5             QKEEGKLQGQLN-------KSDSNQYIG---ELKDQIAELNHELRCLKGQKG---FSGQP                      720 <br />  Mm Evi5                  QKEEGKLQGQLN-------RSDSNQYIR---ELKDQIAELTHELRCLKGQRD---FSSRP                      720 <br />                           * . . * : :        :.: : :     ::.:: : ..      :..     :. : <br />  <br />  Dm CG11727-PA            KFTEQSQTVAELKQEISRLETKNSEMLAEGELR----ANLDDSDKVR-------------                      744 <br />  Dm CG11727-PB            KFTEQSQTVAELKQEISRLETKNSEMLAEGELR----ANLDDSDKVR-------------                      744 <br />  EAT-17a                  KYSEAMQTIQDLQSSISQLELKKAEKWTQNQLRGSSVCDLDEESNSHGSICSNVDHLSLA                      770 <br />  EAT-17b                  KYSEAMQTIQDLQSSISQLELKKAEKWTQNQLRGSSVCDLDEESNSHGSICSNVDHLSLA                      773 <br />  Hs NB4S/Evi5             PFDG-IHIVNHLIGDDESFHSSDEDFIDNSLQETGVGFPLHGKSGSMS------------                      767 <br />  Mm Evi5                  PFDG-IHIVSHLIGDDELFHSSDEDFIDSSLQESAIGFPLHRKSGPMS------------                      767 <br />                            :    : : .* . . :. .. :    .. .       *. .. <br />  <br />  Dm CG11727-PA            --DLQDRLADMKAELTALKSRGKFPGAKLRSSSIQSIESTEIDFNDLNMVRRGSTELST-                      801 <br />  Dm CG11727-PB            --DLQDRLADMKAEYPTP---------------ITSPDTEPWKWIS--------------                      773 <br />  EAT-17a                  SDEMNALLADMTVRIPTLDDLAE------EGSATETDELRPKELNDGNDTTDSGVQLSDS                      824 <br />  EAT-17b                  SDEMNALLADMTVRIPTLDDLAE------EGSATETDELRPKELNDGNDTTDSGVQLSDS                      827 <br />  Hs NB4S/Evi5             ---LDPAVADGSESETEDSVLET----------RESNQVVQKERPPRRRESYSTTV----                      810 <br />  Mm Evi5                  ---LNPALADGSESEAEDGMLGP----------QESDPEAPQKQPPQR-ESYSTTV----                      809 <br />                              :: :** .    .                   :      . <br />  <br />  Dm CG11727-PA            - Dm <br />  CG1172               - <br />  EAT-17a                  H 825 <br />  EAT-17b                  H 828 <br />  Hs NB4S/Evi5             - <br />  <br />  <br /> Figure S1 Sequence alignments EAT-17 Evi5 homologs <br /> EAT-17 shows greatest sequence similarity EVI5/NB4S proteins implicated human neuroblastoma mouse tumorigenesis. EAT-17 contains types conserved domains. putative Rab GAP domain (aa107-316) (shown yellow) located N-terminal half protein. predicted coiled-coil domains located C terminal half protein (aa 364-405, aa 475-552, aa 628-736 coiled-coil (CC) domains 1, 2, 3, respectively) (shown blue). <br /> Shown red indicated arrow invariant arginine residue crucial GAP function. sequences obtained GenBank. Accession numbers follows: NM-005665.3 (human Evi5/NB4S), NM-007964.1 (mouse Evi5), NM- <br /> 167285.1 (Drosophila CG11727-PA), NM-132488.2 (Drosophila CG11727-PB). Dm=Drosophila melanogaster, Hs= Homo <br /> sapiens, Mm= Mus musculus. EAT-17a EAT-17b spliced form EAT-17 C. elegans. <br />  <br />  <br />  <br />  <br />                                                              S. Straud et al.                                                         3 SI <br />  Table S1 Primers used SNP mapping <br />  <br /> SNP                 Primer Number       Forward Primer (5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;)         Reverse Primer (5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;) <br />                      Rev) <br /> pkP6169             742/743             cctcaggatttaccagtgacac          ttagtcttgcgcccctagag <br /> pkP6093             744/745             tagatatcgtggaacccc              cctgggaatccgttttctcc <br /> pkP6096             746/747             gattgaacatagctcacagc            tttcgatcgttttggacgcc <br /> pkP6171             748/749             cgatgcggtttcctagcttac           attgcccatttcaagccc <br /> pkP6170             785/786             cgctgtcacaatctctaaaatg          aaaccctccccactttgttgtc <br /> snp-T21F2           889/890             ttgacgcgctcttctactga            ttggggtggttgtttgatct <br /> snp-C06G1           891/892             gagcataccgtttggcagat            atggctcagcactctcgaat <br /> snp-F59C12          893/894             gaaaaagtggccaaaggtga            aaaaactgacaagcggcaat <br /> snp-T25D1           895/896             catttcagaaaccatgaacctg          ttgaagaactactcctctggca <br /> snp-F38E9           897/898             ctcacgctgacctctttcct            tttgcatctttggagaattgg <br /> snp-T24D11          899/900             gcgggaatgtgcactaaaat            gcgtgtatatttggcagcaa <br /> snp-W09B12          901/902             cctcaggatttaccagtgacac          ttagtcttgcgcccctagag <br /> snp-C11G6           903/904             tgctggccagactacaaaaa            ctcgaaaagtcctatcagcca <br /> snp-F23A7           905/906             tttaaagttcccagctgtgct           gaaagggattctgccacaaa <br />  <br />  <br />  <br />  <br /> 4 SI                                                 S. Straud et al. <br />  Table S2 Deletion mapping primers <br /> Deletion strain       Primer        Primers outside deletion               Primers inside deletion <br /> DA1814                forward       5&#226;&#8364;&#8482;- aagcatctttgagcgcattt &#226;&#8364;&#8220;3&#226;&#8364;&#8482;           5&#226;&#8364;&#8482;- catagcgagtgtttggagca &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br /> ser-1(ok345)          Reverse       5&#226;&#8364;&#8482;- aatttcaggggtgtggacat &#226;&#8364;&#8220;3&#226;&#8364;&#8482;           5&#226;&#8364;&#8482;- aatcatttttgaaaccgaccc &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br /> RB758                 forward       791: 5&#226;&#8364;&#8482;- tcacagctcaccaaagatcg &#226;&#8364;&#8220;3&#226;&#8364;&#8482;      793: 5&#226;&#8364;&#8482;- ttgccaacaggagtaaaggg &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br /> hda-4(ok518)          Reverse       792: 5&#226;&#8364;&#8482;- gttgttgctgctgcatttgt &#226;&#8364;&#8220;3&#226;&#8364;&#8482;      794: 5&#226;&#8364;&#8482;- ccaatgagtgcctggaattt &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br /> RB787                 forward       787: 5&#226;&#8364;&#8482;- atcgaatacatccgtccagc &#226;&#8364;&#8220;3&#226;&#8364;&#8482;      789: 5&#226;&#8364;&#8482;- ggcaacataccatttccacc &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br /> T27A8.2(ok570)        Reverse       788: 5&#226;&#8364;&#8482;- tcttgacccagaaacgaacc &#226;&#8364;&#8220;3&#226;&#8364;&#8482;      790: 5&#226;&#8364;&#8482;- tgacccagaaacgtacccat &#226;&#8364;&#8220;3&#226;&#8364;&#8482; <br />  <br />  <br />  <br />  <br />                                                         S. Straud et al.                                       5 SI <br />  Table S3 Primers used gene structure analysis <br /> Gene                     Forward Primer (5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;)          Reverse Primer (5&#226;&#8364;&#8482;- 3&#226;&#8364;&#8482;)           Amplified Region <br /> T24D11.1                 atgaatgtcctcccactctttttc         tgcaatcattcgaaagtgctg             5&#226;&#8364;&#8482; R116/119 codon <br /> T24D11.1                 atgaatgtcctcccactctttttc         acggggaatgtcgcgttgaat             5&#226;&#8364;&#8482;                                                                                             D152/155 codon <br /> T24D11.1                 cagcactttcgaatgattgca            ctagtggctatccgacagtt              3&#226;&#8364;&#8482; R116/119 codon <br /> T24D11.1                 attcaacgcgacattccccgt            ctagtggctatccgacagtt              3&#226;&#8364;&#8482; D152/155 codon <br /> T24D11.1                 atgaatgtcctcccactctttttc         gagacggagaagctcgatg               3&#226;&#8364;&#8482; Exons 1-5 <br /> T24D11.1                 aatgtggaatgagagaggcg             gagacggagaagctcgatg               Exons 1-5 <br /> T24D11.1                 gatccgcagacaccgagtg              gagacggagaagctcgatg               Exons 2-5 <br /> T24D11.1                 gaagtgttgcctccaagaag             gagacggagaagctcgatg               Exons 3-5 <br /> T24D11.1                 cgtgtggggagagcttattc             gagacggagaagctcgatg               Exons 4-5 <br /> T24D11.1                 aggcattcgcgttcaaactc             gagacggagaagctcgatg               Exons 1*-5 <br /> T24D11.1/F01G12.6        gaagtcaagaagcgtcccaa             tcgttgaactcgctctcctt              Exon 3 T24D11.1                                                                                             exon 7 F01G12.6 <br /> T24D11.1/F01G12.6        ccgtctcgatatggaaggaa             tcgttgaactcgctctcctt              Exon 5 T24D11.1                                                                                             exon 7 F01G12.6 <br /> T24D11.1/F01G12.6        gacgtcagcctcatggaaaact           gccaatgtgtcagcttcctt              Exon 5 T24D11.1                                                                                             exon 5 F01G12.6 <br /> T24D11.1/F01G12.6        aggcattcgcgttcaaactc             ctagtggctatccgacagtt              Exon 1** T24D11.1 (eat-17 cDNA)                                                        <br /> </body></html>