<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                              Discovery Antagonist Peptides Bacterial Helicase- <br />                                               Primase Interaction B. stearothermophilus Reverse Yeast <br />                                               Hybrid <br />   <br />                                               Summary <br />                                                    Developing small-molecule antagonists protein-protein interactions provide powerful <br />                                                    tools mechanistic/functional studies discovery new antibacterials.                                                    developed reverse yeast hybrid approach allows high-throughput screening                                                    antagonist peptides essential protein-protein interactions. applied                                                    methodology essential bacterial helicase-primase interaction Bacillus stearothermophilus <br />                                                    isolated unique antagonist <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span>. peptide binds primase, excluding                                                    helicase inhibiting essential interaction bacterial DNA replication. provide proof                                                    principle reverse yeast hybrid method powerful  step&#226;&#8364;&#65533; screen tool direct <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                    high-throughput antagonist peptide selection protein-protein interaction detectable                                                    traditional yeast hybrid systems. peptides provide useful &#226;&#8364;&#339;leads&#226;&#8364;&#65533;                                                    development new antibacterials. <br />  <br />  <br />                                               Introduction <br />                                                                 functions biology involve large macromolecular assemblies. ability                                                                 individual protein components interact subtle highly specific <br />                                                                 manner defines mechanistic regulatory aspects functional requirements                                                                 assemblies. Characterization molecular details underpin specific protein- <br />                                                                 protein interactions considerable challenge. Traditional structural biology, X-ray <br />                                                                 crystallography, nuclear magnetic resonance (NMR) spectroscopy, microscopy, <br />                                                                 provides powerful tools studying interactions, technological experimental <br />                                                                 limits difficult apply approaches generic, high-throughput manner. <br />                                                                 Instead, biochemical techniques, phage-display, yeast hybrid (Y2H), mass <br />                                                                 spectrometry-based technologies, developed generic methods [1-4].                                                                 methods used extensively identify protein-protein interaction interfaces, <br />                                                                 interacting domains, small ligands mediate interactions. <br />  <br />  <br />  <br />                                                           discovery agonist antagonist peptides regulate protein-protein interactions <br />                                                           great importance, understanding structural, regulatory, mechanistic <br />                                                           aspects macromolecular assemblies, designing new drugs employing <br />                                                           combinatorial bioinformatics approaches [5, 6]. use peptide antagonists                                                           disrupt protein-protein interactions demonstrated variety systems. Examples <br />                                                           include peptides block 1a binding type 1 receptor [7], insulin-like growth <br />                                                           factor 1 binding regulatory binding protein, IGFBP-1 [8], angiogenesis factor <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           VEFGF binding cell surface receptor, KDR [9, 10], IgG Fc binding streptococcal <br />                                                           protein [11], HIV-1 gp120 binding CD4 [12]. comparison, use                                                           peptides DNA replication documented. <br />  <br />                                                           overcome mechanistic topological problems associated DNA replication, <br />                                                           evolution created large, complex multiprotein assemblies, like primosome                                                           replisome. functions assemblies dependent delicate collaborations                                                           component proteins [13-15]. collaborations based allosteric protein- <br />                                                           protein protein-DNA interactions ensure effective progression entire <br />                                                           assembly various functional stages order complete effective faithful <br />                                                           replication genetic information. Interference essential interactions <br />                                                           inevitably result stalling DNA replication  ultimately, cell death. Small <br />                                                           molecules peptides act antagonists block interactions,                                                           represent new avenue explore develop new antibacterial drugs.                                                            recent examples demonstrated peptide bind geminin suppress DNA <br />                                                           replication HCT116 colon cancer cells disrupting geminin-Cdt1 interaction [16], <br />                                                           small molecule mimic growth-inhibitory effect phage polypeptides                                                           disrupting interaction prototypic pair, ORF104 DnaI  putative helicase <br />                                                           loader) Staphylococcus aureus [17] small molecules inhibit                                                           interaction herpes simplex virus DNA polymerase processivity subunit UL42 <br />                                                           inhibiting viral replication [18]. <br />  <br />                                                           article, report discovery, using reverse yeast hybrid (Y3H) approach, <br />                                                           antagonist peptide essential interaction main replicative <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           hexameric helicase, DnaB, primase, DnaG, Gram-positive Bacillus <br />                                                           stearothermophilus. peptide competes DnaB binding DnaG, disrupting <br />                                                           essential bacterial protein-protein interaction DNA replication making                                                           powerful &#226;&#8364;&#339;lead compound&#226;&#8364;&#65533; development synthetic peptidomimetics novel <br />                                                           broad-spectrum antibiotics bacterial DNA replication. antagonist peptide                                                           useful studying molecular details helicase-primase interaction. <br />  <br />                                                           Furthermore, potential Y3H analysis small molecule-protein <br />                                                           interactions, applications drug discovery, emphasized [19], <br />                                                           data presented provide proof principle use Y3H technology                                                           reverse manner screen random peptide libraries valuable advance existing <br />                                                           methods. Apart direct screen isolating antagonist peptides                                                           protein-protein interaction, offers unique flexibility, allowing targeting                                                           partner binary protein complex. used investigate, rapidly,                                                           effects mutations activity antagonist peptide, easily adapted                                                           high-throughput screen manner similar Y2H high-throughput screens amplify <br />                                                           potential industrial method direct antagonist peptide screening. offers new <br />                                                           opportunities field peptide-based drug discovery protein-protein <br />                                                           interaction detectable traditional Y2H  <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 3 <br />  <br />  <br />                                            Results Discussion <br />                                            Antagonist Peptides Helicase-Primase Interaction <br />                                                           helicase (DnaB)-primase (DnaG) interaction B. stearothermophilus stable                                                           complex reconstituted detected purified proteins [20]. C-terminal <br />                                                           domain DnaG, known P16, substitute length DnaG complex                                                           interacts linker region connects N- C-terminal domains DnaB Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           mediate interaction structurally functionally [21].  precise <br />                                                           structural details interaction remain unresolved. stoichiometry                                                           complex unresolved issue, DnaB6-DnaG3 model likely [20],                                                           DnaB6-DnaG2 DnaB6-DnaG1 complexes detectable [21]. <br />  <br />                                                           interactions DnaG P16 DnaB detected traditional Y2H. <br />                                                           devised novel use Y3H technology reverse manner screen commercial <br />                                                           MATCHMAKER random peptide library Clontech order discover antagonist <br />                                                           peptides essential bacterial protein-protein interaction. peptides                                                           provide potential leads development drugs target bacterial DNA <br />                                                           replication. <br />  <br />                                                           Using pBridgeBP16 plasmid carries GAL4BD-DnaB GAL4AD-P16 <br />                                                           fusions, proceeded test detect DnaB-P16 interaction. GAL4BD- <br />                                                           DnaB control constitutive PADH promoter, GAL4AD-P16                                                           placed control PMET25 inducible promoter (Figures 1A&#226;&#8364;&#8220;1D). absence <br />                                                           methionine, PMET25 promoter switched GAL4AD-P16 expressed.                                                           DnaB-P16 interaction detectable trp&#226;&#710;&#8217;/met&#226;&#710;&#8217; media using pBridge- <br />                                                           BP16 plasmid (data shown), indicating proceed use                                                           pBridgeBP16 plasmid reverse Y3H experiment screen antagonist peptides using                                                           random peptide library, shown Figure 1E. <br />  <br />                                                           screened approximately 6000 clones commercial MATCHMAKER random <br />                                                           peptide library Clontech, cloned pGADGH plasmid (Figure 1D). Selection                                                           antagonist peptides carried trp&#226;&#710;&#8217;/met&#226;&#710;&#8217;/leu&#226;&#710;&#8217; media. Potential clones isolated <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           antagonist functions primary Y3H screen verified binding                                                           primase DnaG  P16 helicase-interacting domain) secondary Y2H screen (data                                                           shown). clone isolated screens sequenced. code                                                           unique 16-mer peptide named lpg1. sequence lpg1 did resemble                                                           <span id='am-43' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-44' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-47' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> sequence motifs  amino acid sequence parts) DnaB sequence                                                           presented Figure 1F. proceeded test antagonist function,                                                           DnaB-P16 interaction, length DnaB-DnaG interaction applying <br />                                                           reverse Y3H approach, confirmed   inhibited interaction <br />                                                           (Figures 2A&#226;&#8364;&#8220;2C). data lpg1 clearly inhibits DnaB-DnaG DnaB-P16 <br />                                                           interactions reverse Y3H  <br />  <br />                                                           configuration reverse Y3H ensured GAL4AD-peptide                                                           interact exclusively GAL4AD-P16 antagonist activity apparent                                                           screens (Figure 1E). interaction GAL4AD-peptide GAL4BD-DnaB                                                           simply result &#206;&#178;-galactosidase expression. offers flexibility                                                           selecting antagonist peptides interact exclusively partner                                                           complex. example, GAL4BD-peptide fusion result negative signal                                                           interacts GAL4BD-DnaB,  case, interaction GAL4AD-P16                                                           result positive signal.  antagonist activity isolated lpg1 peptide                                                           specific interactions P16  DnaG) exclude DnaB complex.                                                           verify confirmed lpg1-DnaG interaction conventional Y2H  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                            Page 4 <br />  <br />  <br />                                                           using pASG pGADGH-lpg1 plasmids tpr&#226;&#710;&#8217;/leu&#226;&#710;&#8217; media. expected, lpg1                                                           interact DnaG (Figures 2D 2E). contrast, lpg1 did interact                                                           DnaB tested similar manner using pASB instead pASG plasmid (data <br />                                                           shown). <br />  <br />                                            Antagonist Activity Vitro <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Confirmation antagonist activity lpg1 DnaB-DnaG complex formation <br />                                                           verified examining effect DnaG-mediated stimulation ATPase <br />                                                           activity DnaB vitro. Purified B. stearothermophilus DnaB DnaG proteins form                                                           stable complex isolated gel filtration [20, 21]. ATPase activity DnaB                                                           stimulated significantly complex DnaG [20, 21].  DnaB-DnaG <br />                                                           particularly complex, (1) stoichiometry complex absolute <br />                                                           (2) ATPase kinetic behavior DnaB unresolved. majority                                                           complexes exist DnaB6-DnaG3 stoichiometry, significant percentage complexes <br />                                                           DnaB6-DnaG2 DnaB6-DnaG1 stoichiometry exist [20, 21]. complexity                                                           amplified ATPase kinetic behavior DnaB. does obey <br />                                                           order Michaelis Menten kinetics [20, 21] relative kinetic behavior                                                           heterogeneous complexes far certain. <br />  <br />                                                           Despite complexity  argued lpg1 reversibly competes                                                           DnaB binding DnaG, stimulatory effect DnaB ATPase activity DnaG <br />                                                           inhibited presence lpg1, lpg1 bind DnaG prevent                                                           DnaB-DnaG complex formation. Obviously, certain DnaG <br />                                                           molecules need disengaged complex or DnaG molecules <br />                                                           need disengaged inhibition stimulatory effect observed.                                                           DnaB6-DnaG2 DnaB6-DnaG1 complexes exhibit stimulated ATPase activity. <br />                                                           Despite complications, anticipate removing DnaG molecules <br />                                                           complex lead inhibition DnaG-mediated stimulatory effect DnaB <br />                                                           ATPase activity. <br />  <br />                                                           tested hypothesis employing different ATPase assays: continuous <span id='am-22' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>NADH</span>- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           linked assay indirectly measures rate <span id='am-27' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span> hydrolysis monitoring oxidation <br />                                                           NADH NAD+ end-point TLC-based assay directly measures conversion <br />                                                           ATP <span id='am-10' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span> inorganic phosphate (Pi). assays indicated lpg1 does,  <br />                                                           inhibit DnaG-mediated stimulation DnaB ATPase activity. fact, TLC- <br />                                                           based assay, lpg1 exhibited good inhibitory activity 100- 3.12-fold molar excess <br />                                                           DnaG appeared lose inhibitory effect diluted 1.5-fold molar <br />                                                           excess half concentration DnaG (Figure 3A). emphasize                                                           TLC-based assay provides qualitative absolute quantitative assay. fact                                                           assay competitive inhibitory effect lpg1 likely magnified. assay <br />                                                           measuring percentage ATP hydrolysed 10 min. ATP                                                           hydrolysed concentration reduced time affecting complex kinetic <br />                                                           behavior DnaB.  measuring continuous ATPase rate lpg1 <br />                                                           dose-response experiment appear better robust end-point assay. <br />                                                           Lpg1 actually reduced stimulatory effect DnaG DnaB ATPase activity                                                           30%&#226;&#8364;&#8220;50% determined TLC-based end-point ATPase assay. Interestingly,                                                           truncated version lpg1 missing amino acid residues ([des-Ser1-Trp2] lpg1) N <br />                                                           terminus exhibited significantly reduced antagonist activity TLC-based assay <br />                                                           compared length lpg1 (Figure 3B). inhibited marginally stimulatory effect                                                           DnaG ~10% (compare Figures 3A 3B). <br />  <br />                                                           antagonist effect lpg1 evident continuous NADH-linked assay                                                           measures steady-state rate ATP hydrolysis. assay measures real-time continuous <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 5 <br />  <br />  <br />                                                           rate ATP continuously regenerated. ATPase profile DnaB                                                           assay presence absence DnaG complicated, initial phase                                                           resembles order kinetics 0 2.0 mM ATP, followed increase 2.0 <br />                                                           3.5 mM, sharp inhibition higher concentrations ATP [20, 21, 24].                                                           complex kinetic behavior unresolved, attributed negative <br />                                                           cooperativity monomers hexameric ring [20].  suggestion                                                           purely speculative absolute kinetic mechanism complex behavior Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           mystery. Initially, examined effect lpg1 ATPase activity fixed 2.0 <br />                                                           mM ATP ATPase rate DnaB-DnaG mixture reduced                                                           approximately 15 &#194;&#183; s&#226;&#710;&#8217;1 4 &#194;&#183; s&#226;&#710;&#8217;1, comparable rate DnaB  <br />                                                           approximately 2 &#194;&#183; s&#226;&#710;&#8217;1 (Figure 3C). <br />  <br />                                                           order obtain reliable quantitative information &#226;&#8364;&#339;dose response&#226;&#8364;&#65533; lpg1 <br />                                                           peptide, investigated ATPase profile DnaB DnaB-DnaG mixtures                                                           presence absence different concentrations lpg1 varying concentrations ATP <br />                                                           2 mM, corresponding initial phase complicated ATPase profile DnaB <br />                                                           (Figure 4). presence DnaG reaction mixture absence lpg1 results                                                           marked stimulation DnaB ATPase activity (Figure 4A). Significantly, inclusion                                                           large excess truncated ([des-Ser1-Trp2] lpg1) mutant ([Ala2]lpg1) peptides resulted <br />                                                           marginal differences ATPase profile obvious suppression                                                           DnaG-mediated stimulatory effect (Figure 4B). contrast, inclusion lpg1                                                           reaction mixture resulted marked suppression DnaG-mediated stimulatory effect, <br />                                                           suggesting inhibits DnaB-DnaG complex formation (Figure 4C). <br />  <br />                                                           complexity actual stoichiometry DnaB-DnaG mixtures                                                           presence heterogeneous complexes   DnaG molecule                                                           DnaB hexamer ATPase assays, used excess DnaG (55.2 nM)                                                           DnaB hexamer (9.2 nM), ensure absolutely complexes                                                           DnaB6-DnaG3 stoichiometry.  excess free DnaG likely sequester lpg1 <br />                                                           solution. complexity makes impossible engage                                                           simple protein-ligand type analysis determination meaningful IC50 dose <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           response lpg1. <br />  <br />                                                           Despite problems, carried semiquantitative analysis lpg1-mediated <br />                                                           inhibition, established  experimental conditions, effect <br />                                                           observed 50 molar excess lpg1 (2.76 &#206;&#188;M) DnaG (55.2 nM) reaction <br />                                                           mixture, inhibitory effect gradually increased raised concentration                                                           lpg1 5.52, 11, 27.6 &#206;&#188;M, corresponding 100, 200, 500 molar excess lpg1 <br />                                                           DnaG reaction mixture, respectively (Figure 4C). <br />  <br />                                                           verified direct binding lpg1 DnaG isothermal titration calorimetry (ITC), <br />                                                           interaction strong accurate KB value calculated                                                            Supplemental Data available article online). tight binding (KB &gt; 109 M&#226;&#710;&#8217;1), <br />                                                           ITC titrations lose curvature KB measured accurately [25]. <br />  <br />                                            N Terminus lpg1 Critical Antagonist Activity <br />                                                           search minimal sequence lpg1 critical antagonist <br />                                                           activity, discovered truncated lpg1 missing N-terminal serine                                                           tryptophan residues lost antagonist activity. fact, suppresses DnaG- <br />                                                           stimulatory effect DnaB ATPase activity marginally TLC-based (Figure <br />                                                           3B) NADH-linked ATPase assays (Figure 4B). Interestingly, substituted                                                           tryptophan residue close N terminus alanine mutant <br />                                                           ([Ala2]lpg1) lost antagonist activity, weak activity <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                           Page 6 <br />  <br />  <br />                                                           exhibited mutant lpg1 NADH-linked assay (Figure 4B). fact, (Ala2)lpg1 <br />                                                           marginally worse truncated (des-Ser1-Trp2) lpg1 (compare curves                                                           Figure 4B). fact (Ala2)lpg1 (des-Ser1-Trp2)lpg1 peptides exhibit loss                                                           antagonist activity indicates antagonist activity lpg1 specific N- <br />                                                           terminal segment lpg1 critical. subsequent search protein database (PDB) <br />                                                           revealed lpg1 sequence homology helices mammalian mevalonate <br />                                                           kinase (pdb: 1KVK) viral polymerase (pdb: 1MUK) (Figures 5A 5B) [26, 27]. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Interestingly, cases homology confined N-terminal half lpg1 <br />                                                           peptide (Figure 5C), indicating region lpg1 peptide propensity                                                           adopt helical secondary structure. homologous helices proteins                                                           functional significance case mammalian mevalonate <br />                                                           kinase homologous helix &#206;&#177;8 close classic helix bundle, composed helices <br />                                                           &#206;&#177;9 &#206;&#177;10, constitutes dimerization interface protein [26]. Detailed <br />                                                           structural analysis required establish lpg1 does adopt stable <br />                                                           secondary structure integrity structure essential antagonist <br />                                                           activity. <br />  <br />                                            Lpg1 Inhibits Stimulatory Effect DnaG DnaB Helicase Activity <br />                                                           Having established lpg1 inhibits stimulatory effect DnaG ATPase activity <br />                                                           DnaB, examined effect helicase activity DnaB. Using  <br />                                                           established vitro helicase assay, showed lpg1 does inhibit stimulatory <br />                                                           effect DnaG helicase activity DnaB (Figure 6). DnaB relatively low <br />                                                           helicase activity level stimulated significantly DnaG (Figure 6A). presence <br />                                                           excess lpg1, stimulatory effect DnaG inhibited significantly,                                                           significant effect activity DnaB observed (Figure 6B). data                                                           consistent ATPase data   provide strong evidence lpg1 does <br />                                                           act antagonist inhibitor DnaG-meidated stimulation DnaB activity. <br />  <br />                                            Lpg1 Disrupts DnaB-DnaG Complex Formation Vitro <br />                                                           B. stearothermophilus DnaB-DnaG complex, contrast Escherichia coli, Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           stable detected simply mixing purified DnaB DnaG proteins                                                           resolving mixture gel filtration column (Figure 7A) [20, 21].                                                           DnaG molecules believed interact single DnaB hexamer, minor <br />                                                           species DnaB6-DnaG2 DnaB6-DnaG1 complexes exist [20, 21].                                                           investigated direct effect lpg1 complex gel filtration. complex elutes <br />                                                           column earlier (10.21 ml elution volume) DnaG (13.8 ml elution <br />                                                           volume). resolving power Superdex S-200 column insufficient resolve <br />                                                           clearly complex (MW, 504 kDa) DnaB hexamer (303 kDa). discovered                                                           adding excess lpg1 (175 &#206;&#188;M) DnaB (0.5 &#206;&#188;M hexamer)-DnaG (0.67 &#206;&#188;M <br />                                                           monomer) mixture resulted release substantial DnaG                                                           complex, confirming lpg1 competes DnaB binding DnaG. antagonist <br />                                                           activity lpg1 apparent Figure 7B, increase DnaG eluted free                                                           complex comparative decrease DnaG remaining complex <br />                                                           (compare relative amounts DnaB DnaG complex Figure 7A, lanes 1&#226;&#8364;&#8220;3                                                           Figure 7B, lanes 1&#226;&#8364;&#8220;4). Lpg1 compete DnaB helicase displace                                                           preformed DnaB-DnaG complex. affinity lpg1 DnaG                                                           nanomolar region (Supplemental Data), apparently DnaB binds DnaG high <br />                                                           affinity, DnaB displaced DnaG lpg1.                                                            proteins mixture loaded column, begin separate                                                           immediately. small size, excess lpg1 starts separating rest                                                           proteins immediately. remaining DnaB-DnaG complex moves column <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                            Page 7 <br />  <br />  <br />                                                           quicker, DnaB remain bound DnaG.  partial,                                                           significant, displacement DnaB complex lpg1 observed. <br />  <br />                                                           Y3H, Y2H, vitro biochemical data lpg1 peptide indicate                                                           competitively binds DnaG disengages DnaB-DnaG complex.                                                           complexity DnaB-DnaG interaction unresolved ATPase kinetics DnaB, <br />                                                           possible obtain reliable quantitative information dose response lpg1 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           manner analogous traditional simple peptide-ligand interaction. <br />                                                            combined data provide proof principle novel use reverse Y3H <br />                                                           technology, screening random peptide libraries antagonist peptides protein- <br />                                                           protein interactions detectable traditional Y2H, viable method. <br />  <br />                                            Significance <br />                                                           Overall, data provide proof principle reverse Y3H approach                                                           directly select antagonist peptides protein-protein interaction detectable                                                           traditional Y2H  targeted bacterial helicase-primase interaction                                                           isolated antagonist peptide binds primase disrupts helicase-primase <br />                                                           complex. Despite complexity DnaB-DnaG  verified antagonist <br />                                                           activity using range vitro experiments. shown truncated                                                           point mutant version peptide inactive. approach adapted rapid, <br />                                                           high-throughput direct screening assay isolation antagonist peptides                                                           partner binary protein complex, similar manner traditional Y2H high- <br />                                                           throughput screen [28, 29], applied protein-protein interaction                                                           detected traditional Y2H method. significant advance existing <br />                                                           methods  (1) selects antagonist peptides bind directly protein-protein <br />                                                           interaction interface (direct antagonists) peptides act allosterically distant <br />                                                           binding site (allosteric antagonists); (2) allows specificity targeting exclusively <br />                                                           partner binary protein complex; (3) adapted high-throughput <br />                                                           method; (4) primary antagonist peptide isolated, used                                                           rapidly investigate large number mutant antagonist peptides improved activity.                                                           provide powerful tool development peptide-based drugs target essential <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           protein-protein interactions variety systems. Isolated antagonist peptides used <br />                                                           &#226;&#8364;&#339;<span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> seeds&#226;&#8364;&#65533; development chemical compounds mimic                                                           antagonist action probe directly structural/functional molecular aspects                                                           protein-protein interactions. <br />  <br />                                            Experimental Procedures <br />                                            Plasmid Constructions Y2H Y3H <br />                                                           Y2H 32H methodologies based MATCH-MAKER Hybrid <br />                                                           2 Clontech. dnaB, dnaG genes, p16 C-terminal fragment                                                           dnaG coding 16 kDa polypeptide (P16) interacts DnaB, subcloned <br />                                                           pET expression vectors [20] pACT2 pAS2-1 vectors, shown                                                           Figures 1A 1B. <br />  <br />                                                           dnaB gene cloned NcoI-PstI sites pAS2-1 pASB (Figure 1B)                                                           codes GAL4 DNA binding domain (BD)-DnaB fusion protein, control                                                           PADH1 constitutive promoter dnaG p16 genes subcloned                                                           NcoI-XhoI sites pACT2 produce pACTG pACTP16 plasmids, respectively <br />                                                           (Figure 1A). pACTG pACTP16 code GAL4 activation domain (AD)-DnaG <br />                                                           AD-P16 fusion proteins, respectively, control PADH1 promoter. <br />                                                           dnaG gene subcloned NcoI-XhoI fragment pET21d plasmid <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 8 <br />  <br />  <br />                                                           [20] NcoI-SalI sites pAS2-1 (XhoI SalI sites compatible), produce                                                           pASG plasmid coding GAL4BD-DnaG fusion protein control PADH1 <br />                                                           promoter (Figure 1B). <br />  <br />                                                           dnaB gene subcloned NdeI(blunt)-PstI fragment pASB                                                           EcoRI(blunt)-PstI sites pBridge produce pBridgeB plasmid, coding GALBD- <br />                                                           DnaB fusion protein placing control constitutive PADH1 promoter <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           (Figure 1C). p16 gene subcloned HindIII(blunt)-HindIII(blunt) fragment                                                           NotI(blunt) site pBridgeB placing GAL4AD-P16 fusion protein control                                                           inducible PMET25 promoter, pBridgeBP16 plasmid (Figure 1C). correct <br />                                                           orientation verified appropriate restriction endonuclease mapping. dnaG gene <br />                                                           subcloned BglII-BglII fragment pACTG sites pBridgeBP16, <br />                                                           replacing p16 gene form pBridgeBG plasmid, placing GAL4AD-DnaG <br />                                                           fusion protein control PMET25 promoter. Correct orientation verified                                                           appropriate restriction endonuclease mapping. <br />  <br />                                                           random peptide library purchased Clontech cloned                                                           BamHI-EcoRI sites pGADGH (Figure 1D), control constitutive PADH1 <br />                                                           promoter, stop codons frames immediately EcoRI site. library <br />                                                           contains 106 independent random peptide clones coding random peptides 16 amino <br />                                                           acids maximum length fused GAL4-AD. <br />  <br />                                            Growth Media Yeast <br />                                                           AH109 Saccharomyces cerevisiae yeast strain used study. components <br />                                                           yeast complete medium (YPD) different synthetic dropout (SD) media,                                                           various supplements, purchased Becton Dickinson  Sigma <br />                                                           prepared according manufacturer&#226;&#8364;&#8482;s instructions. Nutritionally selective media <br />                                                           prepared omitting leucine, tryptophan, methionine  combinations  <br />                                                           growth media. E. coli XL1-Blue bacteria cultured Luria-Bertani (LB) <br />                                                           media presence tetracycline. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Yeast Transformations <br />                                                           Yeast cells electrocompetent chemically-competent transformations <br />                                                           DNA. electrocompetent yeast preparations, cells cultured YPD plate <br />                                                           30&#194;&#176;C 3 days. Single colonies used inoculate liquid YPD cultures                                                           incubated 30&#194;&#176;C overnight vigorous shaking. Yeast cells collected                                                           centrifugation washed repeatedly filtered-sterilized, ice-clod 1 M sorbitol. Finally, <br />                                                           yeast cells resuspended 200 &#206;&#188;l 1 M sorbitol used transformation                                                           electroporation MicroPulser (Bio-Rad), according manufacturer&#226;&#8364;&#8482;s instructions. <br />  <br />                                                           chemically-competent yeast, single colonies isolated, described  used <br />                                                           inoculate liquid YPD cultures incubated 30&#194;&#176;C overnight vigorous <br />                                                           shaking. OD600 cultures measured volume containing approximately <br />                                                           2.5 &#195;&#8212; 108 cells collected brought 50 ml prewarmed YPD incubated <br />                                                           4 hr 30&#194;&#176;C shaking. Yeast cells collected centrifugation, <br />                                                           washed times double distilled (dd) <span id='am-38' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span>, resuspended 1 ml 100 mM LiOAc,                                                           incubated 10 min 30&#194;&#176;C. Aliquots 100 &#206;&#188;l used, aliquot, cells <br />                                                           collected centrifugation. supernatant removed 240 &#206;&#188;l 50 w/v PEG, 36 <br />                                                           &#206;&#188;l 1 M LiOAc, 50 &#206;&#188;l salmon sperm DNA, 2 &#206;&#188;l plamid DNA, 32 &#206;&#188;l dd H2O <br />                                                           added, order. cells vortexed vigorously incubated 30 min                                                           30&#194;&#176;C. heat shocked 42&#194;&#176;C 30 min collected  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                             Page 9 <br />  <br />  <br />                                                           centrifugation. supernatant removed cells resuspended 1 ml dd H2O <br />                                                           loaded plates appropriate SD media. <br />  <br />                                            Y2H Y3H Screening <br />                                                           Y2H Y3H screening carried agarose overlay [22] filter-lift <br />                                                           methods according yeast protocols handbook Clontech. pBridgeBP16 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           plasmid cotransformed random peptide library cloned pGADGH plasmid. <br />                                                           Following transformation, double selective plates incubated 30&#194;&#176;C 36 hr.                                                           typical transformation yielded approximately 2000 colonies. agarose overlay assay                                                           performed, described previously [22]. allowed rapid identification potential <br />                                                           positive white colonies protein-protein interaction appeared blocked,                                                           preventing &#206;&#178;-galactosidase activity. colonies typically fail develop blue <br />                                                           coloration overnight. colonies patched double selective plates, <br />                                                           cultured 36 hr 30&#194;&#176;C, selected absent &#206;&#178;-galactosidase activity. Colonies <br />                                                           exhibited &#206;&#178;-galactosidase activity cultured plasmid DNA coding                                                           interfering peptide prepared. control, yeast strain containing                                                           pASG plasmid transformed plasmid pGADGH coding potentially <br />                                                           interfering peptide order confirm interaction peptide target protein, <br />                                                           DnaG. <br />  <br />                                            Peptide Synthesis <br />                                                           Peptide synthesis carried using standard fluoren-9-ylmethoxycarbonyl (Fmoc) <br />                                                           solid-phase approach [23] 4-[2&#226;&#8364;&#178;, 4&#226;&#8364;&#178;- <br />                                                           dimethoxyphenyl(amino)methyl]phenoxyacetamido methyl Novagel (Rink Amide Novagel, <br />                                                           100&#226;&#8364;&#8220;200 mesh, 1% crosslinking, 0.61 mmol &#194;&#183; g&#226;&#710;&#8217;1) resin. peptide assembly achieved <br />                                                           stepwise manner, amino acid residues incorporated activation using <br />                                                           N-[(dimethylamino)-1H-1,2,3-triazolo[4,5-b]pyridine-1 ylmethylene]-N-methylmethan- <br />                                                           aminium hexafluorophosphate N-oxide:7-aza-1-hydroxybenzotriazole:N,N- <br />                                                           diisopropylethylamine (HATU:HOAt:DIEA, 1.0:0.5:2.0) N,N-dimethylformamide <br />                                                           (DMF). Exceptionally, Cys residue installed activation using N,N- <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           diisopropylcarbodiimide:HOAt (1:1) DMF. 4-fold molar excess activated Fmoc- <br />                                                           amino acids used synthesis double-couplings (6 hr followed 18 hr <br />                                                           reaction times) typically required. solution 20% (v/v) piperidine DMF                                                           applied sequential Fmoc-deprotection. <br />  <br />                                                           Following solid-phase peptide assembly, resin-supported product collected                                                           dried. material treated mixture trifluoroacetic acid (TFA): <br />                                                           H2O:triisopropylsilane: ethyl methyl sulfide (90:5:3:2 v/v, 10 ml) 4 hr ambient <br />                                                           temperature. reaction mixture filtered filtrate evaporated dryness                                                           vacuo yield, following trituration diethyl ether, peptide amide beige solid, <br />                                                           dissolved water (5 ml) lyophilized. desired peptide amide                                                           purified reversed-phase HPLC Kromasil C8 column (150 &#195;&#8212; 10 mm) 4.5 ml &#194;&#183; <br />                                                           min&#226;&#710;&#8217;1. eluents   0.06% TFA water, B, 0.06% TFA 90% aqueous <br />                                                           acetonitrile. elution profile 20%&#226;&#8364;&#8220;40% B 12 min period,                                                           required peptide amide eluted 9.7 min. <br />  <br />                                                           Electrospray-high resolution mass spectrometry (ES-HRMS) analysis (Micromass LCT)                                                           purified peptide showed presence expected molecular <span id='am-16' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-20' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> (calculated C78 <br />                                                           H122N23O19S [MH+], 1718.0103;  1718.7845). Lpg1 dissolved 10 mg/ml <br />                                                           stock solution neat DMSO brought 1 mg/ml 10% v/v DMSO                                                           dropwise addition dH2O. Aliquots frozen liquid N2, stored &#226;&#710;&#8217;80&#194;&#176;C, used                                                           required. <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 10 <br />  <br />  <br />                                                           truncated peptide, (des-Ser1,Trp2)lpg1, mutant peptide, (Ala2)lpg1,                                                           synthesized purified using approach outlined  analyzed ES- <br />                                                           HRMS, purified 14-mer peptide showed presence required molecular ion <br />                                                           (calculated MH+, 1443.7158;  1443.9214 [des-Ser1-Trp2]lpg1; calculated <br />                                                           MH+, 1602.8784; 1602.8606 [Ala2]lpg1). peptides readily soluble 1 <br />                                                           mg/ml stock solutions dH2O. Aliquots frozen liquid N2, stored &#226;&#710;&#8217;80&#194;&#176;C,                                                           used required. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Protein Purifications <br />                                                           DnaB DnaG proteins purified quantitated described previously [21]. <br />  <br />                                            ATPase Assays Analytical Gel Filtration <br />                                                           DNA-independent ATPase activity DnaB assayed using TLC-based                                                           NADH-linked assays. <br />  <br />                                                           end-point TLC-based ATP hydrolysis assay carried 20 mM Tris (pH 7.5), 50 <br />                                                           mM NaCl, 10 mM MgCl2, 2.5 mM EDTA, 5 mM DTT, 2 mM ATP, 5 pM 32P-labeled <br />                                                           ATP, 0.6 &#206;&#188;M DnaB (referring hexamers), presence absence 1.6 &#206;&#188;M <br />                                                           DnaG, 37&#194;&#176;C 10 min. Reactions carried presence absence                                                           lpg1 various concentrations indicated figure legends. reactions                                                           terminated addition 50 mM EDTA run TLC plates 1 M formamide <br />                                                           0.5 M LiCl 1 hr. TLC plates visualized analyzed using phosphorimager. <br />                                                           reactions carried triplicate average values plotted. <br />  <br />                                                           continuous NADH-linked ATPase assay carried described previously                                                           DnaB-DnaG complex [21] presence absence varying concentrations lpg1                                                           DnaG. reactions carried triplicate average values plotted.                                                           ATPase rates defined molecules ATP hydrolyzed second hexamer                                                           DnaB. emphasize unlikely monomers hexameric <br />                                                           complex hydrolyzing ATP rate.  stock lpg1 peptide                                                           dissolved 10% v/v DMSO, investigated effect increasing concentrations Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           DMSO ATPase activity assays  50% v/v,                                                           detectable effect ATPase activity (data shown). DnaG does exhibit ATPase <br />                                                           activity ([20] Figure 3B) synthesizes primers presence appropriate <br />                                                           ssDNA substrate  review, article Frick Richardson [30]). <br />  <br />                                                           Analytical gel filtration experiments carried using Superdex S-200 gel filtration <br />                                                           column (Amersham Pharmacia Biotech) described previously [20, 21], using 0.5 &#206;&#188;M <br />                                                           DnaB hexamer, 0.67 &#206;&#188;M DnaG, 175 &#206;&#188;M lpg1. brief, DnaB DnaG mixed <br />                                                           left 10 min ice enable complex formation, point lpg1 added,                                                           mixture proteins left 5 min loaded sizing <br />                                                           column. <br />  <br />                                            ITC <br />                                                           ITC experiments carried VP-ITC Microcalorimeter data analyzed <br />                                                           associated Origin software. Typically, appropriate cell filled 1.4 ml <br />                                                           degassed solution DnaG (16.4 &#206;&#188;M) 50 mM Tris (pH 7.4), 100 mM NaCl, 10% v/v <br />                                                           DMSO. Samples (5 &#206;&#188;l) 300 &#206;&#188;l degassed solution lpg1 (0.45 mM 10% v/v <br />                                                           DMSO) peptide buffer injected DnaG solution 240 s intervals <br />                                                           data collected injection. comparative reference control experiment <br />                                                           10% v/v DMSO carried  reference data subtracted                                                           experimental data. <br />  <br />  <br />  <br />                                            Helicase Assays <br />                                                           Helicase assays carried using established vitro assay [20, 21, 24, <br />                                                           31]. radiolabeled oligonucleotide (5&#226;&#8364;&#178;-GTT <br />                                                           ATTGCATGAAAGCCCGGCTGACTCTAGAGGATCCCCGGGTACGTTATTGCATGA <br />                                                           AAGCCCGGCTG-3&#226;&#8364;&#178;) annealed ssM13mp18 DNA produce 3&#226;&#8364;&#178;-5&#226;&#8364;&#178;-tailed DNA <br />                                                           substrate. molecule DNA substrate defined molecule ssM13mp18 DNA <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           molecule oligonucleotide annealed  DnaB (25 nM) incubated                                                           helicase reaction mixture (20 mM Tris [pH 7.5], 20 mM MgCl2, 10% v/v glycerol, 5 mM <br />                                                           DTT, 0.5 nM DNA substrate) 37&#194;&#176;C 5 min presence absence 150 nM <br />                                                           DnaG 12.5 &#206;&#188;M lpg1, appropriate. Reactions initiated addition 3 <br />                                                           mM ATP terminated 5 min intervals addition helicase stop-buffer (0.4% w/v <br />                                                           SDS, 40 mM EDTA, 8% v/v glycerol, 0.1% w/v bromophenol blue). Displaced <br />                                                           oligonucleotide separated annealed oligonucleotide 10% nondenaturing <br />                                                           polyacrylamide gel constant voltage (100 V). Gels dried quantitative analysis <br />                                                           performed using phosporimager associated software (Molecular Imager FX, Bio- <br />                                                           Rad). Helicase activity defined percentage radiolabeled oligonucleotide <br />                                                           displaced M13mp18 DNA. <br />  <br />  <br />  <br />  <br />                                                           Figure 1. <br />                                                           Construction Plasmids Y2H Y3H Experiments <br />                                                             Construction pACT-based plasmids. insertion sites dnaG p16 genes                                                           indicated. <br />                                                           (B) Construction pAS-based plasmids. insertion sites dnaG dnaG genes <br />                                                           indicated. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           (C) Construction pBridge-based plasmids. insertion sites dnaB p16 genes <br />                                                           indicated. <br />                                                           (D) pGADGH plasmid carrying random peptide library BamHI-EcoRI,                                                           indicated. cases direction inserted genes indicated arrows. <br />                                                           (E) Schematic representation Y2H proteins (P1, P2)                                                           pBridge plasmid (left panel). plasmid selected trp&#226;&#710;&#8217; media. P1 expressed <br />                                                           constitutively, P2 expressed met&#226;&#710;&#8217; media. schematic representation                                                           reverse Y3H presented right panel. pGADGH plasmid (carrying                                                           peptides, P3) selected leu&#226;&#710;&#8217; media P3 expressed constitutively. P2-P3 <br />                                                           interaction negative signal, P1-P3 interaction positive <br />                                                           signal.  P3 interacts P2 disrupts P1-P2 interaction,                                                           detected negative signal reverse Y3H  <br />                                                           (F) sequence lpg1 peptide. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 15 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 2. <br />                                                           Lpg1 Antagonist Activity Yeast Hybrid Systems <br />                                                           Verification antagonist activity lpg1 DnaB-DnaG   DnaB-P16 <br />                                                           (B) interactions Y3H  panels, positive control experiments                                                           DnaB-DnaG DnaB-P16 interactions indicated blue coloration, indicated. <br />                                                           Additional controls traditional positive interaction SV40 large T antigen <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           p53, negative control SV40 large T antigen DnaB, shown                                                           (C). Verification lpg1-DnaG interaction filter-lift method traditional Y2H <br />                                                           shown (D) (yeast patches) (E) (filter lifts). Lpg1 exhibits clear DnaG <br />                                                           binding activity. positive control indicates DnaB-DnaG interaction negative <br />                                                           control indicates interaction DnaB SV40 large T antigen. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                         Page 16 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 3. <br />                                                           Layer Chromatography-Based ATPase Assays <br />                                                             TLC-based end-point ATPase assay showing inhibitory effect lpg1 different <br />                                                           concentrations: 100-, 50-, 25-, 12.5-, 6.12-, 3.12-, 1.5-, 0.5-fold molar excess lpg1 <br />                                                           DnaG indicated bars 3&#226;&#8364;&#8220;10, respectively. Bars 1 2 indicate ATPase <br />                                                           activities DnaB-DnaG mixture DnaB protein  respectively. Triplicate <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           experiments carried  representative TLC plate shown. standard <br />                                                           deviation indicated error bars. <br />                                                           (B) results similar assay, using 100-fold molar excess N-terminally <br />                                                           truncated version (des-Ser1Trp2)lpg1 lpg1 missing N-terminal residues (bar <br />                                                           3). Bar 1 2 indicate ATPase activities DnaB-DnaG mixture DnaB <br />                                                           protein  respectively. Bars 4 5 indicate controls DnaG ATP protein, <br />                                                           respectively. Experiments carried triplicate error bars indicate standard <br />                                                           deviation. <br />                                                           (C) synopsis steady-state ATPase data collected using continuous NADH- <br />                                                           linked ATPase assay. Bar 1 indicates ATPase rate DnaB (9.2 nM hexamer)-DnaG <br />                                                           (55.2 nM monomer) mixture, bar 2 rate DnaB (9.2 nM hexamer)  bar 3 <br />                                                           rate DnaB (9.2 nM hexamer)-DnaG (55.2 nM monomer) presence 5.52 <br />                                                           &#206;&#188;M lpg1. reactions carried 2.0 mM (ATP). <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                        Page 17 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 4. <br />                                                           NADH-Linked ATPase Assays <br />                                                             comparison ATPase profile DnaB hexamer (9.2 nM) presence <br />                                                           (closed triangles) absence (open squares) DnaG (55.2 nM) 0 2.0 mM (ATP). <br />                                                           (B) ATPase profile DnaB hexamer (9.2 nM) presence DnaG (55.2 nM) <br />                                                           27.6 &#206;&#188;M truncated lpg1 (open squares) mutant lpg1 (closed <br />                                                           triangles). <br />                                                           (C) ATPase profile DnaB hexamer (9.2 nM) presence DnaG (55.2 nM) <br />                                                           varying concentrations lpg1, 2.76 &#206;&#188;M (closed circles), 5.52 &#206;&#188;M (open triangles), 11 <br />                                                           &#206;&#188;M (closed squares), 27.6 &#206;&#188;M (open squares) corresponding 50, 100, 200, 500 <br />                                                           molar excess lpg1 DnaG, respectively. graphs scale axes                                                           direct comparison, error bars indicate standard deviation duplicate <br />                                                           experiments. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                         Page 18 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 5. <br />                                                           Sequence Similarities Structural Database <br />                                                           N-terminal half lpg1 peptide homologous helices mammalian <br />                                                           mevalonate kinase   reovirus RNA polymerase lambda3 (B). cases                                                           relevant helices indicated black. actual sequences homology lpg1 <br />                                                           sequence shown (C). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                          Page 19 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 6. <br />                                                           Effects Helicase Activity <br />                                                           DnaG stimulates helicase activity DnaB  , lpg1 inhibits stimulatory <br />                                                           effect DnaG (B). Lanes 1&#226;&#8364;&#8220;6 represent helicase reaction time points 5, 10, 15, 20, 25,                                                           30 min, lanes b represent annealed boiled controls, respectively.                                                           actual gels labeled appropriately shown left quantitative data shown                                                           graphs right.  , data represent helicase reactions DnaB presence <br />                                                           (circles) absence (squares) DnaG. (B), data represent helicase reactions DnaB + <br />                                                           lpg1 presence (circles) absence (squares) DnaG. Annealed displaced <br />                                                           oligonucleotide indicated labeled arrows clarity. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />                                          Gardiner et al.                                                                                       Page 20 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           Figure 7. <br />                                                           Lpg1 Competes Helicase Binding Primase <br />                                                           Analytical gel filtration experiments DnaB (0.5 &#206;&#188;M hexamers)-DnaG (0.67 &#206;&#188;M <br />                                                           monomers) mixtures absence   presence (B) lpg1 (175 &#206;&#188;M). Samples                                                           peak fractions analyzed SDS-PAGE. numbers lanes                                                           polyacrylamide gels correspond numbers peak fractions, indicated. Lane M <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                           shows molecular weight markers, indicated. lpg1 causes dissociation DnaG (67 <br />                                                           kDa) DnaB (50.5 kDa) competitive manner. <br />  <br />  <br />  <br />  <br />                                                               Chem Biol. Author manuscript; available PMC 2011 February 08. <br />   <br /> </body></html>