<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Mol Biol. 2011 November 4; 413(4): 751&#226;&#8364;&#8220;761. doi:10.1016/j.jmb.2011.09.004. <br />  <br />  <br />  <br />                            Systematic chromosomal deletion bacterial ribosomal protein <br />                            genes <br />                            Shinichiro Shoji&#226;&#8364;&#160;,&#226;&#8364;&#161;, Corey M. Dambacher&#226;&#8364;&#160;,&#226;&#8364;&#161;, Zahra Shajani&#226;&#8364;&#160;,&#194;&#167;, James R. Williamson&#226;&#8364;&#160;,&#194;&#167;,                            Peter G. Schultz&#226;&#8364;&#160;,* <br />                            &#226;&#8364;&#160;Departments Chemistry, Scripps Research Institute, 10550 North Torrey Pines Road, La <br />  <br />                            Jolla, CA 92037, USA <br />                            &#194;&#167;Molecular   Biology Skaggs Institute <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> Biology, Scripps Research <br />                            Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA <br />  <br />                            Abstract <br />                                 Detailed studies ribosomal proteins, essential components protein biosynthetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 machinery, hampered lack readily accessible chromosomal deletions                                 corresponding genes.  report systematic genomic deletion 41 individual ribosomal <br />                                 protein genes Escherichia coli, included Keio collection. Chromosomal <br />                                 copies genes replaced antibiotic resistance gene presence                                 inducible, easy exchange plasmid-born allele. Using knockout collection, 9 ribosomal <br />                                 proteins (L15, L21, L24, L27, L29, L30, L34, S9 S17) nonessential survival <br />                                 induction conditions various temperatures. Taken previous results,                                 analysis revealed 22 54 E. coli ribosomal protein genes individually deleted <br />                                 genome. strains allow expression truncated protein variants probe                                 importance RNA-protein interactions functional sites ribosome. set strains <br />                                 enhance vivo studies ribosome assembly/function, ultimately allow <br />                                 systematic substitution ribosomal proteins RNA. <br />  <br />  <br />                            Keywords <br />                                 ribosome; translation; recombineering; ribonucleoprotein complex; protein engineering <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            &#194;&#169; 2011 Elsevier  rights reserved. <br />                            * <br />                              Corresponding author: Tel: 858-784-9273. Fax: 858-784-9440. schultz@scripps.edu Address: 10550 N. Torrey Pines Road, La <br />                            Jolla, CA 92037, USA . <br />                             authors contributed equally <br />                            Author email addresses: <br />                            Shinichiro Shoji: ichishoji@gmail.com <br />                            Corey M. Dambacher: cdambach@scripps.edu <br />                            Zahra Shajani: zsyi@scripps.edu <br />                            James R. Williamson: jrwill@scripps.edu <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                            Supporting Information Available. <br />                            Propensity complementary vector removal/exchange, genomic deletion strategies, verification deletion strains PCR                            supplementary material online. <br />                             Shoji et al.                                                                                                  Page 2 <br />  <br />  <br />                                Introduction <br />                                            Ribosomes large ribonucleoprotein particles iteratively translate triplet codons NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            mRNA corresponding amino acids growing polypeptide chain.                                            round translational elongation, mRNA codon decoding center                                            (Aminoacyl) site decoded cognate aminoacyl-tRNA. peptidyl moiety attached                                            tRNA P (Peptidyl) site transferred &#206;&#177;-amino group site aminoacyl- <br />                                            tRNA nucleophilic attack ester bond linkage peptidyl-tRNA, tRNAs                                            P sites translocate adjacent P E (Exit) sites, respectively.                                            functional ribosome (70S prokaryotes 80S eukaryotes) contains 2 subunits:                                            large subunit small subunit. E. coli, large (50S) subunit consists 2 <br />                                            ribosomal RNAs (rRNA; 23S 5S) 33 different ribosomal proteins (L1-L7, L9-L36), <br />                                            small (30S) subunit consists 1 rRNA 21 ribosomal proteins (S1-S21). <br />  <br />                                            Ribosomal proteins highly specialized, RNA-interacting proteins&#226;&#8364;&#8221;approximately half                                            conserved domains life, domain-specific. <br />                                             composition ribosomal proteins highly conserved domain 1. <br />                                            Crystal structures revealed intricate protein-RNA interactions peripheral <br />                                            internal regions ribosome 2; 3. addition, ribosomal proteins                                            suggested involved essential ribosomal functions 4; 5; 6; 7; 8; 9; 10; 11; 12; 13; 14.                                            decades, significant efforts elucidate roles NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ribosomal proteins translation, numerous mutations confer diverse phenotypes <br />                                            antibiotic resistance E. coli isolated 15; 16; 17; 18. instance, restrictive <br />                                            mutations S12, originally identified streptomycin-resistant/dependent strains, confer                                            hyperaccurate phenotype  frequency miscoding stop-codon readthrough                                            decreased) ribosome 19; 20, ram mutations S4 S5, identified revertants <br />                                            streptomycin-dependence, rise error-prone phenotype  frequency                                            miscoding stop-codon readthrough increased) 21; 22. Additionally, restrictive <br />                                            mutations S12 (R86S P91Q), known slow rate ribosome <br />                                            translation, rise increased folding expression aggregation-prone <br />                                            eukaryotic proteins E. coli 23.  functions ribosomal proteins                                            fully deciphered, primarily lack readily accessible mutant lines containing <br />                                            genetic deletions individual essential ribosomal proteins. <br />  <br />                                            &#206;&#187;-Red-mediated recombination powerful tool precisely alter bacterial genomes, <br />                                            providing simple method study protein function, genetic networks, genome <br />                                            evolution living cells 24. Previous efforts generate complete set gene deletion <br />                                            strains  non-essential genes) E. coli resulted collection 3985 individual <br />                                            mutants covering ~90% known E. coli genes (Keio collection 25).  13 NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            54 ribosomal protein genes (encoding L1, L9, L11, L25, L31, L32, L33, L35, L36, S6, <br />                                            S15, S20, S21) deleted way.  employed step E. coli <br />                                            genome recombineering technique 26, create series genetic deletion mutants covering <br />                                            41 ribosomal protein genes (excluding covered                                            publically-available Keio collection). Deleterious effects deletions minimized                                            presence complementing plasmid carrying gene  precise <br />                                            genomic replacement using antibiotic marker. begun use deletion <br />                                            strains probe functions individual ribosomal proteins translation. <br />  <br />                                Results <br />                                Deletion ribosomal protein genes E. coli chromosome <br />                                            ribosomal proteins thought essential, deleted chromosomal <br />                                            gene presence plasmid-born, inducible copy gene  Supplemental <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                             Page 3 <br />  <br />  <br />                                            Experimental Procedures). individually cloned 41 targeted ribosomal <br />                                            protein (RP) genes (amplified E. coli DH10B genomic DNA) target plasmid <br />                                            (pCDSSara) arabinose-inducible araBAD promoter (Fig. 1). E. coli strain DH10B <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            used parental strain ability stably maintain complementing <br />                                            plasmids. selective toxic marker sacB, encoding levansucrase Bacillus subtilis,                                            included target plasmid facilitate subsequent exchange plasmid                                            vector  toxic effects conferred sacB successful exchange                                            pCDSSara-RP RP expression plasmid confirmed presence 5% <br />                                            sucrose (Fig. S1). pCDSSara-RP plasmids transformed E. coli DH10B,                                            corresponding genes chromosome replaced chloramphenicol <br />                                            acetyltransferase (CAT) gene &#206;&#187; Red-mediated recombination presence                                            arabinose 26. CAT gene amplified polymerase chain reaction                                            included 40-100 bp extensions homologous genomic regions flanking target <br />                                            gene. PCR fragments purified transformed cells expressing &#206;&#187;-Red <br />                                            recombination presence complementing plasmid. <br />  <br />                                            Roughly 70% targeted genes (29 genes) precisely replaced CAT gene; <br />                                            remainder required partial replacement initial attempts failed (Tables 1; Fig. <br />                                            S2). rplD (encoding L4), rplO (L15), rpmD (L30), rpsE (S5) contain                                            internal Shin-Dalgarno (SD) sequence 3&#226;&#8364;&#178;-downstream gene, stop codon                                            CAT gene introduced immediately 5&#226;&#8364;&#178; SD sequence, preserving translation NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            downstream gene. rplE (encoding L5), rplM (L13), rplW (L23), rpmB (L28), rpsB (S2), <br />                                            rpsC (S3), 5&#226;&#8364;&#178; region gene left intact, 3&#226;&#8364;&#178; portion replaced                                            frame stop codon followed SD sequence CAT gene. Likewise, rpsD <br />                                            (encoding S4) rpsJ (S10), 3&#226;&#8364;&#178; region gene left intact, 5&#226;&#8364;&#178; portion <br />                                            replaced CAT gene, followed SD sequence start codon minimize <br />                                            possible polar effects downstream genes operon. Probable reasons                                            difficulties associated complete replacement RP genes                                            presence important regulatory element polycistronic mRNAs stable <br />                                            secondary structures recombination sites. Unexpectedly, replacement rplL                                            rpsR genes, encoding L7/L12 S18, respectively, occurred plasmid copy <br />                                            gene expressed control IPTG (isopropyl &#206;&#178;-D-1- <br />                                            thiogalactopyranoside)-inducible trc promoter, requirement higher <br />                                            expression levels (Fig. 1). recombination events verified genomic PCR                                            primer pairs targeting CAT gene flanking regions genome (Fig. S3). <br />  <br />                                Essentiality ribosomal protein genes <br />                                            deletion mutants, length, wild-type protein conditionally expressed                                            inducible copy targeted gene encoded complementing plasmid. gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            essential survival cell given condition, absence arabinose IPTG <br />                                            induce gene expression result cell death. gene nonessential, cells                                            survive absence inducer. 5 mutants  chromosomal <br />                                            deletion gene encoding L15, L21, L29, L30, S9) showed obvious growth                                            absence inducer (Fig. 2a). test essentiality genes thoroughly,                                            mutants grown liquid media absence inducer 30 &#194;&#176;C 37 &#194;&#176;C, <br />                                            plated media containing 5% sucrose select cells lost                                            complementing plasmid. Surviving colonies analyzed absence target gene <br />                                            presence desired chromosomal recombination event (Fig. S4). <br />  <br />                                            Mutants lost complementing plasmid included extensive set non- <br />                                            essential genes, encoding L15, L21, L24, L27, L29, L30, L34, S9, S17 <br />                                            (Fig. 2b). Growth mutant lacking L21 inhibited lower temperature (30 &#194;&#176;C), <br />                                            growth mutants lacking L24 L27 inhibited higher temperature (42 &#194;&#176;C) <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                               Page 4 <br />  <br />  <br />                                            (Fig. 2b). knowledge, mutants lacking L21 L34 reported.                                            absence L21 confirmed mass spectroscopy analysis 14N-labeled <br />                                            ribosomes isolated L21 deletion strain, spiked 15N-labeled ribosomes isolated <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            parent DH10B strain 27. results displayed complete absence protein L21 <br />                                            intact 50S 70S subunits isolated L21 deletion strain, confirmed                                            stoichiometric presence essential protein components ribosomes (Fig. 3). <br />                                             &#226;&#8364;&#339;protein level&#226;&#8364;? refers concentration  abundance) individual ribosomal <br />                                            proteins present 14N labeled ribosomes isolated &#206;&#8221;L21 line, normalized                                            15N labeled ribosomes isolated parent DH10B strain. L35 signal                                            missing data, L35 detected wild-type sample  general, <br />                                            peptides shorter ribosomal proteins L31-L36 difficult resolve                                            technique 27. similar analysis performed L34 deletion strain,                                            reason, L34 peptides identified 14N-ribosomes deletion strain <br />                                            15N-labeled wild-type ribosomes (data shown). Mutants lacking L15, L24, L27, <br />                                            L29, L30, S9, S17 isolated previously various groups diverse <br />                                            contexts 28; 29; 30; 31; 32. Mutants lacking L28 S13 30; 31; 33 insertion mutant <br />                                            S8 gene 34 isolated previously, unable remove                                            complementing plasmid corresponding strains created, possibly                                            differences nature mutation strain background. <br />  <br />                                Probing importance protein-RNA contacts ribosomal 30S 50S P sites <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Crystal structures ribosomal complexes revealed intricate interactions                                            ribosomal proteins RNAs, including rRNA, tRNA, mRNA,                                            ribosome 4; 35; 36. importance interactions poorly understood,                                            lack facile experimental create analyze mutations ribosomal <br />                                            proteins. using mutant strains obtained study, able create series <br />                                            truncations ribosomal proteins L5, L27, S19, positioned proximity                                            functional ribosomal sites, evaluate significance interactions formed                                            proteins (Fig. 4a). Mutant versions genes placed control trc promoter <br />                                            pTrc99A-derived plasmid compatible pCDSSara carrying wild-type copy,                                            introduced corresponding deletion mutants. absence genomic <br />                                            allele, homogeneous population mutant wild-type protein expressed                                            cell presence IPTG arabinose, respectively; presence sucrose, <br />                                            surviving cells lost sacB-containing vector, resulting production                                            mutant protein cell. <br />  <br />                                             C-terminal tail S19, protrudes                                            decoding center 30S site, truncated 7 amino acids (S19C&#206;&#8221;7), cell <br />                                            growth substantially perturbed (Fig. 4b). Furthermore, removal 3 consecutive Lys <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            residues C terminus (S19C&#206;&#8221;3), unresolved crystal structures 4, <br />                                            did significantly affect growth. 50S P site, removal L5 loop interacting                                            tRNA (L5&#206;&#8221;70-83) caused severe growth defect, consistent recent study suggesting <br />                                            yeast counterpart loop (&#226;&#8364;&#339;P-site loop&#226;&#8364;?) important cell viability tRNA <br />                                            binding 37. Finally, truncation 9 amino acids N-terminal tail L27 (L27N&#206;&#8221;9), <br />                                            remove interactions site P-site tRNAs, minimally affected cell <br />                                            growth expressed  surprising fact                                            presence L27 non-essential cell viability. <br />  <br />                                            Mutated truncated ribosomal protein variants altered affinities ribosome. <br />                                             high level truncated L27 (L27&#206;&#8221;N16) expression required restore growth <br />                                            L27 deletion (Fig. 4b), imply reduced interaction ribosome.                                            assess ribosomal incorporation truncated protein variants used study <br />                                            (L5&#206;&#8221;70-83, L27N&#206;&#8221;16, L27N&#206;&#8221;20, S19C&#206;&#8221;16; able grow &#206;&#8221;S19 cells <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                Page 5 <br />  <br />  <br />                                           expressing S19C&#206;&#8221;23 truncation, suggesting truncation lethal) used <br />                                           quantitative mass spectrometry. 14N-labeled proteins expressed deletion lines <br />                                           harboring various truncations assessed comparison 15N-labeled wild-type <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ribosomes. truncated variants analyzed retained ribosome.                                           &#206;&#8221;L27 mutants expressing L27N&#206;&#8221;16 L27N&#206;&#8221;20 truncations, observed elevated <br />                                           levels truncated L27 protein 14N-labeled samples compared wild-type 15N- <br />                                           labeled samples. observed retention S19C&#206;&#8221;16 protein &#206;&#8221;S19 cells,                                           able detect changes relative abundance truncated S19,                                           ribosomal proteins &#206;&#8221;S19 cells expressing S19C&#206;&#8221;16. Ribosomes &#206;&#8221;L5 cells <br />                                           expressing L5 truncation mutant (L5&#206;&#8221;70-83) contain slightly elevated <br />                                           levels L5 protein compared wild-type ribosomes, proteins detected                                           ribosomes wild-type abundance. Additionally, exception ribosomes <br />                                           isolated &#206;&#8221;S19 cells expressing S19C&#206;&#8221;16, ribosomes containing truncated <br />                                           variants (L5&#206;&#8221;70-83, L27N&#206;&#8221;16, L27N&#206;&#8221;20) slightly elevated levels                                           ribosomal protein L31 (Fig. S5). <br />  <br />                                Discussion <br />                                           deletion lines generated accelerate studies various activities                                           ribosome individual ribosomal proteins, shed light mechanism                                           translation assembly ribonucleoprotein complexes 38; 39; 40. example, studies NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           ribosome assembly pathways limited small number available deletion <br />                                           mutants difficulties associated vitro reconstitution ribosome assembly 41. <br />                                           Analysis vivo assembly intermediates depletion various ribosomal proteins <br />                                           provide insights regarding involvement proteins assembly                                           prokaryotic ribosomes.  deletion collection generated allows                                           extensive analysis point deletion mutations specific ribosomal protein genes.                                           data suggest E. coli DH10B survive deletion 9 ribosomal protein <br />                                           genes targeted, including 2 genes (encoding L21 L34) considered <br />                                           essential. L21 encoded rplU gene, transcribed                                           downstream gene, rpmA, encoding L27. Previous attempts generate viable L21 deletion <br />                                           lines failed unintentional alteration L27 expression. L21 assembled <br />                                           ribosome near 5&#226;&#8364;&#178; end 23S rRNA forms extensive interactions L20. <br />                                           previous vitro study showed omitting L21 affects binding L30 assembly <br />                                           intermediates 42.  unable detect substantial alteration relative <br />                                           abundance ribosomal proteins large subunit mass spectroscopy L21 <br />                                           absent vivo. results suggest vivo ribosome assembly pathways                                           differ aspects vitro 43. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           illustrate utility strain collection, carried preliminary study                                           generating truncation mutants S19, L27, L5. <span id='am-26' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-30' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> triplet mRNA                                           decoded specific aminoacyl-tRNA molecule, contingent specific base <br />                                           pairing decoding center ribosomal 30S site. incoming aminoacyl-tRNA <br />                                           forms interactions anticodon stem loop proteins, S12 S19. S12                                           extensively investigated mutations gene affect translational fidelity                                           antibiotic resistance (e.g., &#226;&#8364;&#339;restrictive&#226;&#8364;? mutations causing hyperaccuracy) 14.  S19 <br />                                           poorly studied recent crystal structures revealed C-terminal interactions <br />                                           tRNA possible involvement decoding 4. terminal 3 amino acids (4 amino <br />                                           acids Thermus thermophilus) unresolved crystal structures,                                           nature interactions tRNA remains unclear. observed nearly wild-type <br />                                           growth C-terminal tail S19 truncated 3 residues, suggesting <br />                                           minimal participation residues protein translation.  4 residues <br />                                           preceding play important role S19 function, truncation analysis <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                            Page 6 <br />  <br />  <br />                                           revealed poor growth C-terminal 7-13 residues protein deleted. <br />                                           data indicate importance S19 30S site, studies                                           necessary elucidate precise role S19 translation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           peptidyltransferase center catalytic center <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> bond formation large <br />                                           subunit mainly comprised rRNA, supports hypothesis peptide bond <br />                                           formation catalyzed solely RNA prebiotic environment 44; 45.  <br />                                           crystal structures bacterial ribosomes, N-terminal tail L27 protrudes                                           peptidyltransferase center contacts backbone P-site site tRNAs 4; 36. <br />                                           L27 non-essential, complete depletion protein strongly perturbs growth                                           ribosomal functions (Fig. 2) 10; 28. previously shown truncation                                           3 amino acids N-terminus L27 decreases peptidyltransferase <br />                                           activity 10, catalysis peptide bond formation directly involve L27 46.                                           increased expression levels truncated L27  observed expressing <br />                                           N-terminal deletions 3-16 residues, observed expressing N-terminal <br />                                           deletion 20 residues) alleviate deleterious effects growth, suggesting L27 <br />                                           N-terminal tail play role assembly 50S subunit. <br />  <br />                                           L5 lies interface 30S 50S subunits, generating inter-subunit bridge B1b,                                           makes extensive contacts 5S rRNA. L5 forms interactions tRNA                                           50S P site, specifically elbow region tRNA. postulated removal NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           tRNA-interacting loop L5 perturb precise positioning P-site tRNA                                           peptidyltransferase center affect catalytic efficiency peptide bond <br />                                           formation.  cells expressing L5 mutant loop (L5&#206;&#8221;70-83) showed <br />                                           severely impaired growth. plausible removal loop caused <br />                                           unfolding substantial decrease affinity L5 ribosome, yeast L11 (eukaryotic <br />                                           homolog L5) similar truncation retained ribosome affected tRNA <br />                                           binding 37. investigation precise role L5-tRNA interaction P site <br />                                           remains performed future. Mutated truncated ribosomal protein variants                                           altered affinities ribosome. methods described allow <br />                                           investigators determine ribosomal retention relative abundance proteins                                           sucrose gradient-isolated ribosomes. <br />  <br />                                           Engineering cellular translational machinery allowed biosynthesis proteins <br />                                           containing non-native amino acids diverse chemical properties 47. amino acids <br />                                           genetically incorporated cellular proteins orthogonal aminoacyl-tRNA <br />                                           synthetase (AARS)/tRNA pairs response nonsense codons frame-shift (4- <br />                                           base) codons 48; 49. Notably, recently shown 16S rRNA engineered                                           enhance decoding frame-shift codons non-natural aminoacyl-tRNA E. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           coli 50; 51. Just rRNA, ribosomal proteins involved conformational changes                                           ribosome interactions translation components. likely engineering <br />                                           functionally critical ribosomal proteins, directed evolution, improve                                           translational efficiency non-native amino acids incorporated response <br />                                           nonsense frame-shift condons. Fine-tuning ribosomal protein function                                           provide differential modulation ribosomal activities, including decoding, peptidyltransfer, <br />                                           intermolecular interactions. ribosomal proteins considered later <br />                                           additions primitive RNA ribosome core 52, examine                                           ribosomal protein components functionally replaced entirety                                           RNA, cis-acting insertion libraries placed near site protein deletion,                                           trans-acting vivo RNA aptamers. <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                         Page 7 <br />  <br />  <br />                                Materials Methods <br />                                Plasmid design <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           pCDSSara pCDSStrc constructed combining PCR-amplified fragments <br />                                           containing pCloDF13 origin spectinomycin/streptomycin-resistance marker <br />                                           derived pCDF-1b (Novagen), sacB gene B. subtilis, araBAD trc <br />                                           promoter pBAD (Invitrogen) pTrc99A (GenBank: A13038.1), respectively.                                           ribosomal protein genes PCR amplified genomic DNA E. coli DH10B (F&#226;&#710;&#8217; <br />                                           mcrA &#206;&#8221;(mrr-hsdRMS-mcrBC) &#206;&#166;80dlacZ&#206;&#8221;M15 &#206;&#8221;lacX74 endA1 recA1 deoR <br />                                           &#206;&#8221;(ara,leu)7697 araD139 galU galK nupG rpsL(StrR) &#206;&#187;&#226;&#710;&#8217;) alteration native stop <br />                                           codon TAA   cloned XhoI SpeI sites pCDSSara                                           pCDSStrc  rplL rpsR). <br />  <br />                                Knockout experiments <br />                                           plasmids (pCDSSara pCDSStrc rplL rpsR) transformed DH10B <br />                                           carrying pKD46, expression plasmid &#206;&#187;-Red recombinase 26. cells grown <br />                                           30 &#194;&#176;C 2YT media containing 50 &#206;&#188;g/ml spectinomycin 100 &#206;&#188;g/ml ampicillin. 0.2% <br />                                           arabinose (pCDSSara) 0.2% arabinose 1 mM IPTG (pCDSStrc) added                                           media optical density 600 nm (OD600) reached 0.15-0.2; cells                                           harvested OD600 approximately 0.5; electrocompetent cells prepared NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           washing 10% glycerol 3 times 4 &#194;&#176;C. Freshly-prepared electrocompetent cells                                           transformed 300-1000 ng DNA containing CAT gene sequence PCR-amplified <br />                                           pEVOL 53. Primers containing 40-70 nt 3&#226;&#8364;&#178; overhangs homologous target <br />                                           genomic regions used amplify CAT gene. spacer sequence containing Shine- <br />                                           Dalgarno sequence included template necessary (Fig. S2). rplL, rpsB, <br />                                           rpsR deletions, DNA fragments containing CAT gene ~100-bp extension                                           homology flanking genomic regions constructed overlap extension PCR                                           increase propensity recombination. Transformants recovered 2YT containing <br />                                           0.2% arabinose (pCDSSara) 1 mM IPTG (pCDSStrc), plated LB agar containing 50 <br />                                           &#206;&#188;g/ml spectinomycin, 7-15 &#206;&#188;g/ml chloramphenicol, arabinose IPTG, incubated <br />                                           37 &#194;&#176;C 2-4 days. Surviving colonies individually analyzed genomic PCR using <br />                                           taq polymerase (Invitrogen) primers annealing CAT gene (5&#226;&#8364;&#178;-CAT Fwd, 5&#226;&#8364;&#178;- <br />                                           AATTAACTGCAGTTACTGCCACTCATCGCAGTACTGTTG-3&#226;&#8364;&#178;; 3&#226;&#8364;&#178;-CAT Rev, 5&#226;&#8364;&#178;- <br />                                           AATTAACATATGGAGAAAAAAATCACTGGATATACC-3&#226;&#8364;&#178;) flanking <br />                                           genomic regions ~100-bp upstream downstream target gene (Fig. S3). Loss                                           pKD46 strains confirmed loss ampicillin resistance (data shown). <br />  <br />                                Vector removal/exchange <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           remove pCDSSara pCDSStrc plasmids deletion mutants, cells grown <br />                                           overnight LB media containing 7 &#206;&#188;g/ml chloramphenicol. 100 &#206;&#188;l culture                                           spread LB agar containing 7 &#206;&#188;g/ml chloramphenicol 5% sucrose incubated 30 <br />                                           &#194;&#176;C 37 &#194;&#176;C 2-4 days. Loss plasmid confirmed LB agar containing 50 &#206;&#188;g/ <br />                                           ml spectinomycin appropriate inducer. test facility vector exchange,                                           IPTG-inducible pTrc-S16 plasmid used electro-transformation S16 deletion <br />                                           strain. cells recovered overnight  8 h) 2YT + IPTG 37 &#194;&#176;C,                                           plated LB agar plates containing 100ug/ml ampicillin 1mM IPTG 5% sucrose. <br />                                           Single colonies surviving sucrose counter-selection picked streaked LB <br />                                           agar plates containing 100ug/ml ampicillin + 1mM IPTG, 100ug/ml ampicillin, 50ug/ml <br />                                           spectinomycin + 0.2% arabinose (Fig. S1). <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                              Page 8 <br />  <br />  <br />                                Truncation analysis <br />                                            expression truncated ribosomal proteins, pTrc99A vector PCR amplified <br />                                            using primers SpeI-pTrc-Fwd 5&#226;&#8364;&#178;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            AATTAAACTAGTGGCGGATGAGAGAAGATTTTCAGCC-3&#226;&#8364;&#178; XhoI-pTrc-Rev 5&#226;&#8364;&#178;- <br />                                            AATTAACTCGAGTTCCTCCTGTGTGAAATTGTTATCCGCTCACAATTCC-3&#226;&#8364;&#178;                                            digested XhoI SpeI. truncated L5 &#206;&#8221;70-83 gene amplified overlap <br />                                            extension PCR, truncated L27 S19 genes amplified PCR.                                            fragments digested cloned XhoI SpeI sites pTrc99A vector <br />                                            created  resulting plasmids contain genes control trc promoter.                                            &#206;&#8221;rplE pL5 &#206;&#8221;rpmA pL27, &#206;&#8221;rpsS pS19 strains transformed plasmids, <br />                                            transformants recovered 2YT containing 0.2% arabinose 2 h selected                                            LB agar supplemented 50 &#206;&#188;g/ml spectinomycin, 100 &#206;&#188;g/ml ampicillin, 0.2% <br />                                            arabinose overnight 37 &#194;&#176;C. <br />  <br />                                Sample preparation mass spectrometry <br />                                            Cells grown 37 &#194;&#176;C minimal media  liter: 12.08 g Na2HPO4, 3 g KH2PO4, 0.5 <br />                                            g NaCl, 2 g glucose, 20.3 mg MgCl2, 24.6 mg MgSO4, 30 mg Na2EDTA, 7.5 mg CaCl2, <br />                                            25.05 mg FeCl3, 0.27 mg ZnSO4, 0.27 mg CuSO4, 0.27 mg CoCl2, 0.18 mg MnSO4, 20 mg <br />                                            biotin, 20 mg <span id='am-21' about='obo:IMR_0200356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="rdfs:label" content="Folic acid" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0200356" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00504" datatype="xsd:string"></span>folic acid</span>, 50 mg thiamine, 50 mg niacinamide, 50 mg riboflavin, 50 mg Ca- <br />                                            pantothenate, 0.001 mg <span id='am-10' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>vitamin</span> B12, 1 g (14NH4)2SO4 1 g (15NH4)2SO4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            14N- 15N-labeling mutant DH10B ribosomal proteins, respectively), harvested <br />                                            OD600 approximately 0.5 centrifugation 7,000 &#195;&#8212; g 10 min, stored                                            &#226;&#710;&#8217;80 &#194;&#176;C subsequent analysis. Frozen cell pellets L21 L34 mutants                                            parent DH10B strain thawed suspended high magnesium lysis buffer (20 mM <br />                                            Tris HCl pH 7.5, 100 mM NH4Cl, 10mM MgCl2, 0.5 mM EDTA, 6 mM &#206;&#178;- <br />                                            mercaptoethanol) lysed bead-beater (BioSpec Products,  Bartesville, OK) <br />                                            using 0.1 mm zirconia/silica beads. Insoluble debris removed centrifugation <br />                                            steps: initial spin 13000 &#195;&#8212; g 10 min followed second spin 13000 &#195;&#8212; g 90 <br />                                            min. 1mL aliquot supernatant layered 35 mL 10-40% sucrose gradient (50 <br />                                            mM Tris HCl pH 7.8, 10 mM MgCl2, 100 mM NH4Cl) SW 32 rotor (Beckman Coulter, <br />                                            Fullerton, CA) centrifuged 26,000 rpm 4 &#194;&#176;C 16 h order separate                                            subunits. gradients fractionated Brandel fractionator. Subunit concentration <br />                                            fractions estimated measuring optical density 260 nm. quantitative <br />                                            mass spectrometry experiments performed fractions sucrose gradient <br />                                            peak corresponding 50S subunit fractions 70S subunit                                            &#206;&#8221;L21 &#206;&#8221;L34 mutants. Eights fractions run total. fractions, 50 pmol <br />                                            14N sample used 50 pmol 15N 70S added standard. Samples                                            prepared electrospray ionization time flight (ESI-TOF) analysis described <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            previously 43. <br />  <br />                                ESI-TOF mass spectrometry <br />                                            peptides analyzed Agilent 1100 Series high performance liquid <br />                                            chromatography (HPLC) instrument coupled Agilent ESI-TOF instrument                                            capillary flow electrospray. Peptides separated ACN gradient 0.1% formic <br />                                            acid flow rate 7 &#206;&#188;L/min. steps gradient 5-15% ACN 10 min, <br />                                            15-50% ACN 70 min 50-95% ACN 4 min. Data collected m                                            range 100-1300. <br />  <br />                                Quantitative analysis protein levels using mass spectrometry <br />                                            Briefly, feature list generated using software Agilent (Mass Hunter Mass <br />                                            Profiler), feature entire isotopic envelope single <span id='am-15' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> contains  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                             Page 9 <br />  <br />  <br />                                            monoisotopic peak isotopomers. Features defined m value                                            monoisotopic peak, charge ion retention time column. Feature lists <br />                                            compared theoretical digest ribosomal proteins E. coli 70S <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            ribosome. Experimental features assigned possible identities theoretical ion&#226;&#8364;&#8482;s <br />                                            charge identical mass 50 ppm experimental feature&#226;&#8364;&#8482;s <br />                                            mass. 14N/15N pairs extracted possible identities features                                            matched retention time 0.1 min  cases multiple <br />                                            identities feature peptide matched multiple <br />                                            proteins, features discarded. remaining feature pairs, segments                                            complete mass spectrum extracted theoretical isotope distributions fit                                            extracted spectra using program Isodist 54. distributions fit, unlabeled <br />                                            (natural abundance deletion strains) fully labeled (15N fixed 99.3%, <br />                                            corresponding external standard, case wild-type ribosomes). fits                                            extracted spectra evaluated visually, spectra poor fitting distributions                                            rejected. amplitudes given Isodist yield relative amounts unlabeled 15N <br />                                            labeled peptide sample used calculate abundance protein 44. <br />                                            addition samples run searching extracting 15N standard <br />                                            peaks. searching presence 15N 14N verify absent L21 <br />                                            protein L21 deletion strain truly absent deletion strain standard <br />                                            (Fig. 3b 3c). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                            thank Drs. Huiwang Ai, Tsotne Javahishvili, Charles E. Melancon helpful comments discussions. <br />                                            grateful Lyn&#226;&#8364;&#8482;Al L. Nosaka Han Xiao technical assistance. thank Dr. Eli Chapman                                            providing pTrc99A plasmid (supplied pTrc99A-GroEL). work supported Uehara <br />                                            Memorial Foundation (fellowship S.S.), National Institutes Health (NIH) grants R37-GM-053757                                             J.R.W) grant 5R01 GM062159  P.G.S.). <br />  <br />  <br />                                Abbreviations used <br />  <br />                                           CAT          chloramphenicol acetyltransferase <br />                                           IPTG         isopropyl &#206;&#178;-D-1-thiogalactopyranoside <br />                                           RP           ribosomal protein <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           SD           Shine-Dalgarno <br />  <br />                                References <br />                                            1. Roberts E, Sethi  Montoya J, Woese CR, Luthey-Schulten  Molecular signatures ribosomal <br />                                               evolution. Proc. Natl. Acad. Sci. USA. 2008; 105:13953&#226;&#8364;&#8220;13958. [PubMed: 18768810] <br />                                            2. Ban N, Nissen P, Hansen J, Moore PB, Steitz TA. complete atomic structure large <br />                                               ribosomal subunit 2.4 &#195;&#8230; resolution. Science. 2000; 289:905&#226;&#8364;&#8220;920. [PubMed: 10937989] <br />                                            3. Wimberly BT, Brodersen  Clemons WM Jr. Morgan-Warren RJ, Carter AP, Vonrhein C, <br />                                               Hartsch T, Ramakrishnan V. Structure 30S ribosomal subunit. Nature. 2000; 407:327&#226;&#8364;&#8220;339. <br />                                               [PubMed: 11014182] <br />                                            4. Jenner L, Demeshkina N, Yusupova G, Yusupov M. Structural rearrangements ribosome                                               tRNA proofreading step. Nat. Struct. Mol. Biol. 2010; 17:1072&#226;&#8364;&#8220;1078. [PubMed: 20694005] <br />                                            5. Hoang L, Fredrick K, Noller HF. Creating ribosomes RNA 30S subunit. Proc. Natl. <br />                                               Acad. Sci. USA. 2004; 101:12439&#226;&#8364;&#8220;12443. [PubMed: 15308780] <br />  <br />  <br />                                                   J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                      Page 10 <br />  <br />  <br />                                           6. Takyar S, Hickerson RP, Noller HF. mRNA helicase activity ribosome. Cell. 2005; 120:49&#226;&#8364;&#8220; <br />                                              58. [PubMed: 15652481] <br />                                           7. Schmeing TM, Voorhees RM, Kelley AC, Gao YG, Murphy F. V. t. Weir JR, Ramakrishnan V. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              crystal structure ribosome bound EF-Tu aminoacyl-tRNA. Science. 2009; 326:688&#226;&#8364;&#8220; <br />                                              694. [PubMed: 19833920] <br />                                           8. Gao YG, Selmer M, Dunham CM, Weixlbaumer  Kelley AC, Ramakrishnan V. structure                                              ribosome elongation factor G trapped posttranslocational state. Science. 2009; <br />                                              326:694&#226;&#8364;&#8220;699. [PubMed: 19833919] <br />                                           9. Ogle JM, Ramakrishnan V. Structural insights translational fidelity. Annu. Rev. Biochem. <br />                                              2005; 74:129&#226;&#8364;&#8220;177. [PubMed: 15952884] <br />                                           10. Maguire BA, Beniaminov AD, Ramu H, Mankin  Zimmermann RA. protein component                                                heart RNA machine: importance protein L27 function bacterial <br />                                                ribosome. Mol. Cell. 2005; 20:427&#226;&#8364;&#8220;435. [PubMed: 16285924] <br />                                           11. Cukras AR, Southworth DR, Brunelle JL, Culver GM, Green R. Ribosomal proteins S12 S13 <br />                                                function control elements translocation mRNA:tRNA complex. Mol. Cell. 2003; <br />                                                12:321&#226;&#8364;&#8220;328. [PubMed: 14536072] <br />                                           12. Diaconu M, Kothe U, Schlunzen F, Fischer N, Harms JM, Tonevitsky AG, Stark H, Rodnina MV, <br />                                                Wahl MC. Structural basis function ribosomal L7/12 stalk factor binding                                                GTPase activation. Cell. 2005; 121:991&#226;&#8364;&#8220;1004. [PubMed: 15989950] <br />                                           13. Fei J, Kosuri P, MacDougall DD, Gonzalez RL. Coupling ribosomal L1 stalk tRNA <br />                                                dynamics translation elongation. Mol. Cell. 2008; 30:348&#226;&#8364;&#8220;359. [PubMed: 18471980] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           14. Zaher HS, Green R. Hyperaccurate error-prone ribosomes exploit distinct mechanisms                                                tRNA selection. Mol. Cell. 2010; 39:110&#226;&#8364;&#8220;120. [PubMed: 20603079] <br />                                           15. Dabbs ER. Mutational alterations 50 proteins Escherichia coli ribosome. Mol. Gen. Genet. <br />                                                1978; 165:73&#226;&#8364;&#8220;78. [PubMed: 362167] <br />                                           16. Isono S, Isono K. Mutations affecting structural genes genes coding modifying <br />                                                enzymes ribosomal proteins Escherichia coli. Mol. Gen. Genet. 1978; 165:15&#226;&#8364;&#8220;20. [PubMed: <br />                                                362162] <br />                                           17. Zengel JM, Young R, Dennis PP, Nomura M. Role ribosomal protein S12 peptide chain <br />                                                elongation: analysis pleiotropic, streptomycin-resistant mutants Escherichia coli. J. Bacteriol. <br />                                                1977; 129:1320&#226;&#8364;&#8220;1329. [PubMed: 321423] <br />                                           18. Zaman S, Fitzpatrick M, Lindahl L, Zengel J. Novel mutations ribosomal proteins L4 L22 <br />                                                confer erythromycin resistance Escherichia coli. Mol. Microbiol. 2007; 66:1039&#226;&#8364;&#8220;1050. <br />                                                [PubMed: 17956547] <br />                                           19. Gorini L, Kataja E. Phenotypic Repair Streptomycin Defective Genotypes E. Coli. Proc. <br />                                                Natl. Acad. Sci. USA. 1964; 51:487&#226;&#8364;&#8220;493. [PubMed: 14171463] <br />                                           20. Ozaki M, Mizushima S, Nomura M. Identification functional characterization protein <br />                                                controlled streptomycin-resistant locus E. coli. Nature. 1969; 222:333&#226;&#8364;&#8220;339. [PubMed: <br />                                                4181187] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           21. Birge EA, Kurland CG. Reversion streptomycin-dependent strain Escherichia coli. Mol. <br />                                                Gen. Genet. 1970; 109:356&#226;&#8364;&#8220;369. [PubMed: 4925047] <br />                                           22. Brownstein BL, Lewandowski LJ. mutation suppressing streptomycin dependence.  effect <br />                                                ribosome function. J. Mol. Biol. 1967; 25:99&#226;&#8364;&#8220;109. [PubMed: 5340532] <br />                                           23. Siller E, DeZwaan DC, Anderson JF, Freeman BC, Barral JM. Slowing bacterial translation speed <br />                                                enhances eukaryotic protein folding efficiency. J Mol. Biol. 2010; 396s:1310&#226;&#8364;&#8220;1318. [PubMed: <br />                                                20043920] <br />                                           24. Sharan SK, Thomason LC, Kuznetsov SG, Court DL. Recombineering: homologous <br />                                                recombination-based method genetic engineering. Nat. Protoc. 2009; 4:206&#226;&#8364;&#8220;223. [PubMed: <br />                                                19180090] <br />                                           25. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner <br />                                                BL, Mori H. Construction Escherichia coli K-12 frame, single-gene knockout mutants:                                                Keio collection. Mol. Syst. Biol. 2006; 2 2006.0008. <br />                                           26. Datsenko KA, Wanner BL. step inactivation chromosomal genes Escherichia coli K-12 <br />                                                using PCR products. Proc. Natl. Acad. Sci. USA. 2000; 97:6640&#226;&#8364;&#8220;6645. [PubMed: 10829079] <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                 Page 11 <br />  <br />  <br />                                           27. Sharpe Elles LM, Sykes MT, Williamson JR, Uhlenbeck OC. dominant negative mutant                                               E. coli RNA helicase DbpA blocks assembly 50S ribosomal subunit. Nucleic Acids Res. <br />                                               2009; 37:6503&#226;&#8364;&#8220;6514. [PubMed: 19734347] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           28. Wower IK, Wower J, Zimmermann RA. Ribosomal protein L27 participates 50 S subunit <br />                                               assembly peptidyl transferase reaction. J. Biol. Chem. 1998; 273:19847&#226;&#8364;&#8220;19852. [PubMed: <br />                                               9677420] <br />                                           29. Stoffler-Meilicke M, Dabbs ER, Ehrlich R, Stoffler G. mutant Escherichia coli lacks <br />                                               ribosomal proteins S17 L29 used localize proteins ribosomal surface. Eur. <br />                                               J. Biochem. 1985; 150:485&#226;&#8364;&#8220;490. [PubMed: 3926498] <br />                                           30. Dabbs ER. Selection Escherichia coli mutants proteins missing ribosome. J. <br />                                               Bacteriol. 1979; 140:734&#226;&#8364;&#8220;737. [PubMed: 387748] <br />                                           31. Bubunenko M, Baker T, Court DL. Essentiality ribosomal transcription antitermination <br />                                               proteins analyzed systematic gene replacement Escherichia coli. J. Bacteriol. 2007; <br />                                               189:2844&#226;&#8364;&#8220;2853. [PubMed: 17277072] <br />                                           32. Franceschi FJ, Nierhaus KH. Ribosomal protein L20 replace assembly-initiator protein L24 <br />                                               low temperatures. Biochemistry. 1988; 27:7056&#226;&#8364;&#8220;7059. [PubMed: 2461735] <br />                                           33. Cukras AR, Green R. Multiple effects S13 modulating strength intersubunit <br />                                               interactions ribosome translation. J. Mol. Biol. 2005; 349:47&#226;&#8364;&#8220;59. [PubMed: <br />                                               15876367] <br />                                           34. Kang Y, Durfee T, Glasner JD, Qiu Y, Frisch D, Winterberg KM, Blattner FR. Systematic <br />                                               mutagenesis Escherichia coli genome. J. Bacteriol. 2004; 186:4921&#226;&#8364;&#8220;4930. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               15262929] <br />                                           35. Korostelev  Trakhanov S, Laurberg M, Noller HF. Crystal structure 70S ribosome-tRNA <br />                                               complex reveals functional interactions rearrangements. Cell. 2006; 126:1065&#226;&#8364;&#8220;1077. <br />                                               [PubMed: 16962654] <br />                                           36. Selmer M, Dunham CM, Murphy FV, Weixlbaumer  Petry S, Kelley AC, Weir JR, <br />                                               Ramakrishnan V. Structure 70S ribosome complexed mRNA tRNA. Science. <br />                                               2006; 313:1935&#226;&#8364;&#8220;1942. [PubMed: 16959973] <br />                                           37. Rhodin MH, Dinman JD. flexible loop yeast ribosomal protein L11 coordinates P-site tRNA <br />                                               binding. Nucleic Acids Res. 2010; 38:8377&#226;&#8364;&#8220;8389. [PubMed: 20705654] <br />                                           38. Schmeing TM, Ramakrishnan V. recent ribosome structures revealed                                               mechanism translation. Nature. 2009; 461:1234&#226;&#8364;&#8220;1242. [PubMed: 19838167] <br />                                           39. Kaczanowska M, Ryden-Aulin M. Ribosome biogenesis translation process Escherichia <br />                                               coli. Microbiol. Mol. Biol. Rev. 2007; 71:477&#226;&#8364;&#8220;494. [PubMed: 17804668] <br />                                           40. Cech TR. Crawling RNA world. Cell. 2009; 136:599&#226;&#8364;&#8220;602. [PubMed: 19239881] <br />                                           41. Sykes MT, Williamson JR. complex assembly landscape 30S ribosomal subunit. Annu <br />                                               Rev Biophys. 2009; 38:197&#226;&#8364;&#8220;215. [PubMed: 19416066] <br />                                           42. Herold M, Nierhaus KH. Incorporation additional proteins complete assembly map                                               50 S subunit Escherichia coli ribosomes. J. Biol. Chem. 1987; 262:8826&#226;&#8364;&#8220;8833. [PubMed: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               3298242] <br />                                           43. Sykes MT, Shajani  Sperling E, Beck AH, Williamson JR. Quantitative proteomic analysis                                               ribosome assembly turnover vivo. J. Mol. Biol. 2010; 403:331&#226;&#8364;&#8220;345. [PubMed: 20709079] <br />                                           44. Nissen P, Hansen J, Ban N, Moore PB, Steitz TA. structural basis ribosome activity                                               peptide bond synthesis. Science. 2000; 289:920&#226;&#8364;&#8220;930. [PubMed: 10937990] <br />                                           45. Noller HF. Evolution Protein Synthesis RNA World. Cold Spring Harb. Perspect Biol. <br />                                               2010:a003681. <br />                                           46. Rodnina MV, Beringer M, Wintermeyer W. ribosomes make peptide bonds. Trends Biochem. <br />                                               Sci. 2007; 32:20&#226;&#8364;&#8220;26. [PubMed: 17157507] <br />                                           47. Liu CC, Schultz PG. Adding new chemistries genetic code. Annu. Rev. Biochem. 2010; <br />                                               79:413&#226;&#8364;&#8220;444. [PubMed: 20307192] <br />                                           48. Wang L, Brock  Herberich B, Schultz PG. Expanding genetic code Escherichia coli. <br />                                               Science. 2001; 292:498&#226;&#8364;&#8220;500. [PubMed: 11313494] <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                                 Page 12 <br />  <br />  <br />                                           49. Anderson JC, Wu N, Santoro SW, Lakshman V, King DS, Schultz PG. expanded genetic code <br />                                               functional quadruplet codon. Proc. Natl. Acad. Sci. USA. 2004; 101:7566&#226;&#8364;&#8220;7571. [PubMed: <br />                                               15138302] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           50. Wang K, Neumann H, Peak-Chew SY, Chin JW. Evolved orthogonal ribosomes enhance                                               efficiency synthetic genetic code expansion. Nat. Biotechnol. 2007; 25:770&#226;&#8364;&#8220;777. [PubMed: <br />                                               17592474] <br />                                           51. Neumann H, Wang K, Davis L, Garcia-Alai M, Chin JW. Encoding multiple unnatural amino acids <br />                                               evolution quadruplet-decoding ribosome. Nature. 2010; 464:441&#226;&#8364;&#8220;444. [PubMed: <br />                                               20154731] <br />                                           52. Bokov K, Steinberg SV. hierarchical model evolution 23S ribosomal RNA. Nature. 2009; <br />                                               457:977&#226;&#8364;&#8220;980. [PubMed: 19225518] <br />                                           53. Young TS, Ahmad  Yin JA, Schultz PG. enhanced unnatural <span id='am-34' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-39' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>amino acid</span> <br />                                               mutagenesis E. coli. J. Mol. Biol. 2010; 395:361&#226;&#8364;&#8220;374. [PubMed: 19852970] <br />                                           54. Sperling E, Bunner AE, Sykes MT, Williamson JR. Quantitative analysis isotope distributions                                               proteomic mass spectrometry using squares Fourier transform convolution. Anal. Chem. <br />                                               2008; 80:4906&#226;&#8364;&#8220;4917. [PubMed: 18522437] <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                          Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 1. Complementing plasmids rescue genomic deletions ribosomal protein genes <br />                                           genes encoding 41 targeted ribosomal proteins E. coli genome                                           disrupted presence complementing plasmid.  39 genes disrupted                                           presence pCDSSara-RP (RP corresponds protein encoded gene) <br />                                           expresses ribosomal protein presence arabinose induction. rplL rpsR <br />                                           (encoding L12 S18, respectively) disrupted presence pCDSStrc-L12                                           S18, respectively, expresses protein presence IPTG induction. Spc, <br />                                           specitinomycin; Sm, streptomycin. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                           Page 14 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 2. Growth analysis deletion strains <br />                                             Growth dependence deletion mutants complementing genes. strains <br />                                           contained genomic deletion (&#206;&#8221;) plasmid (p) copy ribosomal protein gene <br />                                           indicated. Overnight cultures mutants grown presence 0.2% arabinose                                            &#206;&#8221;rplL pL12 &#206;&#8221;rpsR pS18) 1 mM IPTG  &#206;&#8221;rplL pL12 &#206;&#8221;rpsR pS18) <br />                                           diluted 100-fold, spotted LB agar supplemented antibiotics NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           inducer (+) 0.2% glucose (&#226;&#710;&#8217;), incubated 15-36 h 37 &#194;&#176;C. (b) Temperature <br />                                           dependence knockout strains lacking complementing plasmid. pCDSSara plasmids <br />                                           containing nonessential ribosomal protein genes removed sacB-based <br />                                           counterselection, overnight cultures resulting cells diluted 100-fold, spotted, <br />                                           grown LB agar 48 h various temperatures. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                          Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 3. Protein Level 70S subunit &#206;&#8221;L21 mutant ribosomes <br />                                             Protein level single fraction 70S subunit peak. Error bars represent                                           standard deviation multiple measurements different peptides/ions                                           protein. case single measurement obtained, open circle used. (b)                                           peptide L5 (MWEFFER) identified present L21 mutant strain <br />                                           (blue) 15N ribosomal standard (red). (c) peptide L21 (MYAVFQSGGK)                                           identified present 15N ribosomal standard (red) absent L21 mutant <br />                                           strain. position 14N peptide appear shown blue arrow. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                             Shoji et al.                                                                                         Page 16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Fig. 4. Mutational analysis L27, L5, S19 function <br />                                             schematic view protein-tRNA interactions catalytic centers ribosome. <br />                                           N-terminal 16 residues L27, residues 70-83 L5, C-terminal 16 residues S19 <br />                                           shown red. tRNA nucleotides comprising sites potential interaction                                           proteins shown stick representation (Protein Data Bank entries 318H 318I). (b) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Growth analysis mutants containing homogeneous populations various truncated <br />                                           ribosomal proteins. Overnight cultures mutants carrying pCDSSara pTrc <br />                                           plasmids containing length truncated version protein, respectively,                                           diluted 100-fold spotted LB agar supplemented antibiotics inducer <br />                                           indicated. pTrc plasmids containing wild-type gene green fluorescent protein (GFP) <br />                                           gene used controls. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                                                                  NIH-PA Author Manuscript                                     NIH-PA Author Manuscript        NIH-PA Author Manuscript <br />  <br />  <br />                                                                                                                                                    Table 1 <br />                                                                                                                                                                                               Summary deletion strains ribosomal protein genes . <br />  <br />                                                                                            Replaced   Remaining <br />                                                                                                                                                                                         Shoji et al. <br />  <br />  <br />  <br />  <br />                                                                                                 b          c             d                     e <br />                                                                  Protein   Gene   Operon   (bp)       (bp)        Plasmid         Essentiality <br />                                                                  L2        rplB   S10                    pCDSSara-L2     Essential <br />                                                                  L3        rplC   S10                    pCDSSara-L3     Essential <br />                                                                  L4        rplD   S10      1-585      [586-606]   pCDSSara-L4     Essential <br />                                                                  L5        rplE   spc      130-540    1-129       pCDSSara-L5     Essential <br />                                                                  L6        rplF   spc                    pCDSSara-L6     Essential <br />                                                                  L10       rplJ   rif                    pCDSSara-L10    Essential <br />                                                                  L7/L12    rplL   rif                    pCDSStrc-L12    Essential <br />                                                                  L13       rplM   L13      31-429     1-30        pCDSSara-L13    Essential <br />                                                                  L14       rplN   spc                    pCDSSara-L14    Essential <br />                                                                  L15       rplO   spc      1-424      [425-435]   pCDSSara-L15    Nonessential <br />                                                                  L16       rplP   S10                    pCDSSara-L16    Essential <br />                                                                  L17       rplQ   alpha                  pCDSSara-L17    Essential <br />                                                                  L18       rplR   spc                    pCDSSara-L18    Essential <br />                                                                  L19       rplS   S16                    pCDSSara-L19    Essential <br />                                                                  L20       rplT   L20                    pCDSSara-L20    Essential <br />                                                                  L21       rplU   L21                    pCDSSara-L21    Nonessential <br />                                                                  L22       rplV   S10                    pCDSSara-L22    Essential <br />                                                                  L23       rplW   S10      31-303     1-30        pCDSSara-L23    Essential <br />  <br />  <br />  <br />  <br /> J Mol Biol. Author manuscript; available PMC 2013 June 27. <br />                                                                  L24       rplX   spc                    pCDSSara-L24    Nonessential <br />                                                                  L27       rpmA   L21                    pCDSSara-L27    Nonessential <br />                                                                                                                                            f <br />                                                                  L28       rpmB   L28      31-237     1-30        pCDSSara-L28    Essential <br />                                                                  L29       rpmC   S10                    pCDSSara-L29    Nonessential <br />                                                                  L30       rpmD   spc      1-171      [172-180]   pCDSSara-L30    Nonessential <br />                                                                  L34       rpmH   L34                    pCDSSara-L34    Nonessential <br />                                                                  S1        rpsA   S1                     pCDSSara-S1     Ess <br /> </body></html>