<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Mutations CG8878, Novel Putative Protein Kinase, <br /> Enhance P Element Dependent Silencing (PDS) Position Effect Variegation (PEV) Drosophila <br /> melanogaster <br /> Allen McCracken, John Locke* <br /> Department Biological Sciences, University Alberta, Edmonton, Alberta, Canada <br />  <br />  <br />  <br />      Abstract <br />      Genes multicellular organisms expressed developmental program largely dependent self- <br />      perpetuating higher-order chromatin states. mechanism establishing maintaining epigenetic events      studied Drosophila. known example epigenetic effect (PEV) Drosophila,      shown gene silencing heterochromatin formation. investigating process similar Position Effect <br />      Variegation (PEV) using mini-w transgene, called Pci, inserted upstream regulatory region ci. mini-white+ <br />      transgene Pci expressed adult eye;  P KP elements present, variegated eye <br />      phenotype results indicating random w+ silencing development. P element dependent silencing (PDS)      modified haplo-suppressors/triplo-enhancers, Su(var)205 Su(var)3&#226;&#8364;&#8220;7, indicating heterochromatic <br />      modifiers act dose dependently PDS. use spontaneous derivative mutation Pci called PciE1 (E1)      variegates like PDS absence P elements, presumably adjacent gypsy element insertion, screen      second-site modifier mutations enhance variable silencing white+ E1. isolated 7 mutations CG8878,      essential gene, enhance E1 variegated phenotype. CG8878, previously uncharacterized gene, potentially encodes <br />      serine/threonine kinase closest Drosophila paralogue, ballchen (nhk-1), phosphorylates histones. mutant <br />      alleles enhance PDS E1 Position Effect Variegation (PEV) wm4, indicating previously unknown common <br />      silencing mechanism  <br />  <br />   Citation: McCracken  Locke J (2014) Mutations CG8878, Novel Putative Protein Kinase, Enhance P Element Dependent Silencing (PDS) Position Effect <br />   Variegation (PEV) Drosophila melanogaster. PLoS 9(3): e71695. doi:10.1371/journal.pone.0071695 <br />   Editor: Barbara Jennings, University College London, United Kingdom <br />   Received April 17, 2013; Accepted July 4, 2013; Published March 10, 2014 <br />   Copyright: &#195;&#376; 2014 McCracken, Locke. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported funding Discovery Grant Natural Sciences Engineering Research Council (NESRC) Canada (http:// <br />   www.nserc-crsng.gc.ca). funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: john.locke@ualberta.ca <br />  <br />  <br />  <br /> Introduction                                                                          characteristics P strains modifying P- repressor <br />                                                                                       sensitive alleles, enabling P element transposition) <br />    Drosophila melanogaster, expression white+ gene (w+) cell             variegated expression occurs resulting mosaic expression autonomous, necessary import pigment precursors                        white ommatidia red background eye [3]. PDS normal colour adult eye. white2 mutants, absence                    occurs w+ transgenes inserted near location pigment results white-eyed phenotype rescued                      [2]. w+ containing P-element transgenes.                                 random silencing w+ minigene Pci insertion locations, expression w+ sensitive local                       development indicates phenomenon similar heterochromatin <br /> chromatin environment, adjacent heterochromatin.                          spreading hPEV. supported Pci responding dose <br /> example, P{lacW}ciDplac  Pci), transgene inserted <br />                                                                                       sensitive manner Su(var) 205 3&#226;&#8364;&#8220;7, like wm4 proximally chromosome 4 Ribosomal protein S3A <br />                                                                                       centromeric PEV lines [4,5]. PEV, position important (RpS3A) cubitus interruptus (ci), originally isolated                                                                                       insertions transgene locations display <br /> enhancer trap ci [1]. w+ minigene sensitive changes                                                                                       PDS.  chromosomal translocations Pci away gene dosage heterochromatin proteins HP1 SU(VAR)3&#226;&#8364;&#8220;7 <br />                                                                                       centromere-proximal location reduces PDS (Bushey <span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; Locke <br /> [2]. <br />                                                                                       2004)2. position dependence variegated phenotype <br />    Pci transgene sensitive presence P elements <br />                                                                                       suggest PDS occurs similar mechanism heterochromatic genome phenomenon called P element dependent <br />                                                                                       PEV. <br /> silencing (PDS), phenotypically similar heterochro- <br /> matic position effect variegation (hPEV). flies lacking P elements                    investigation PDS Pci spontaneous mutants, <br /> (M strains) w+ transgene expressed uniform manner                         P{lacW}ciDplacE1 P{lacW}ciDplacE2  E1 E2),  red eye phenotype).  flies containing P elements                     recovered [2] showed variegated eye phenotype (P strains) KP s (derivative elements capable mimicking                    absence P elements complete white eye phenotype <br />  <br />  <br /> PLOS | www.plosone.org                                                       1                              March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                    Mutations Drosophila Putative Protein Kinase <br />  <br />  combined P elements, P{ry+ SalI}89D. PEV                    complementation analysis deficiencies region. modifier loci, Su(var)205 Su(var)3&#226;&#8364;&#8220;7, suppressed                  100 progeny scored heterozygous mutant/ <br /> variegation E1 E2. Molecular analysis E1 E2                     deficiency combination did occur combination revealed novel gypsy element insertion approximately               considered lethal. <br /> 1 kb distal Pci insert, opposite orientations 547 bp <br /> apart. Testing E1 E2 mutants su(Hw)                     DNA sequencing <br /> mod(mdg4) showed variegation E1 E2 result                  series overlapping CG8878 gene segments amplified gypsy insulator function se [2];  possible               PCR product sequenced. Point mutations interaction gypsy insulator wari element           identified double peaks chromatogram. polymor- <br /> 39 end w+ transgene ruled [6]. E1              phisms mutations confirmed sequencing strands. <br /> E2 trans-silence w+ expression Pci paired homolog, present translocations. position pairing                 Eye pigment assays <br /> contributes trans-silencing similar dominant trans-                    w+ gene activity assayed measuring inactivation wild-type homologue bw+/bwD heterozygotes              brown eye pigment using modification method <br /> [2,7,8,9].                                                                    [12]. Heads 5&#226;&#8364;&#8220;9 day old adult flies stored 220u <br />    previous screen genetic modifiers PDS [2] required                extracted. genotype, replicate samples 10 <br /> tracking chromosomes. work described                  heads extracted 200 mL acidified ethyl alcohol (1% HCl <br /> taken advantage E1 variegating PDS                     30% ethanol) shaking 48 hours. Absorbance 470 nm <br /> simpler, components chromosome.                        measured using 96 Costar flat plate Furthermore, intermediate variegated eye phenotype                    Bio-Tek PowerWave XS spectrophotometer. Photographs visibly reveal second site genetic enhancers (white eye phenotype).           representative adult flies eyes mineral oil taken using isolation characterization seven                  Zeiss stereo microscope Nikon Coolpix 995 digital camera. <br /> mutants, enhance PDS hPEV wm4. gene,                    eye pigment assay adult eye photographs, CG8878, recovered mutations trithorax ash1              balancer chromosome CyO used control. appears encode novel type kinase similar Vaccinia-Related-Kinase(VRK), Casein-Kinase (CK), Tau-                    Results <br /> Tubulin-Kinase (TTK). <br />                                                                               Screen second site enhancers w+ variegation E1 <br /> Materials Methods                                                            screened ,44,000 progeny EMS treated fathers                                                                               dominant enhancement w+ silencing E1 recovered 58 <br /> Drosophila stocks mutations                                               confirmed mutations. Inter se recessive lethal complementation <br />    Unless cited, D. melanogaster mutations described           analysis showed fell simple complex <br /> previously [10]. P{lacW}ciDplac (Pci) allele (Eaton &amp; Kornberg            recessive lethal complementation groups, singles. <br /> 1990)1 P{lacZP\T.Ww+mC ampR ori = lacW} construct inserted               mutations trithorax ash1 (manuscript ,3 kbp upstream (distal) ci locus chromosome 4.                   preparation), simple group, 7 alleles (1a27a, <br /> P{lacW}ci DplacE1(E1) gypsy element insertion ,1 kb             3a22a, 3a52a, 3a66a, 3a90a, 3a97a, 4a7a), described  <br /> upstream previously described [2]. y1 w* P{lacW}3-76a lacW            seven examined dominant enhancer transgene inserted X-chromosome originally                     E1/Pci phenotype allele 4a7a genetically mapped 2&#226;&#8364;&#8220;65.4 <br /> isolated Y. N. Jan provided Bloomington stock                   (n = 490) recombination relative wgSp L Bc Pin markers. <br /> center, 1)wm4,wm4 obtained K.D. Tartof. Fly                 Linkage dominant enhancer E1/Pci recessive <br /> stocks maintained room temperature standard yeast/                 lethal phenotypes demonstrated described Materials <br /> cornmeal medium.                                                              Methods. Deficiency mapping lethal phenotype    Mutagenesis used w2; dp2; e2; E1 males treated 25 mM                  group refined position 48E2;48E4, Df(2R)BSC199 <br /> EMS [11] mated y2 w2; +/+ virgin females screened               (7,779,605: 8,059,989) Df(2R)BSC879 (7,779,605: dominant enhanced eye colour phenotype progeny.                  8,029,867), includes predicted CG8878 gene. Putative mutants mated w2; dp2; e2; Pci flies confirm              candidate gene region predicted influence gene <br /> transmission segregation determine chromosomal                     expression heritable manner, expressed correct tissue location. Mutations crossed inter se establish recessive lethal       correct time, size likely result 7 independent <br /> complementation groups. Mutant CG8878 alleles kept                    mutations mutagenesis size. note Hen1 <br /> balanced stocks CyO.                                                     Prp8 influence gene expression post-translationally, <br />                                                                               lead silencing clonal manner (variegation) <br /> Genetic Mapping                                                               seen  <br />    dominant enhancer phenotype E1/Pci background used genetic recombination mapping gave                  DNA sequencing mutants <br /> fuller red eye phenotype, provided room                           DNA sequencing spanning entire predicted coding region enhancement allowed reliable visual assessment                CG8878, heterozygotes CyO balancer chromosome, enhancement. Mutants mapped relative wgSp L Bc                 showed alleles (1a27a, 3a22a, 3a52a, 3a66a, 3a97a) Pin markers. Recombinants left right enhancer                 base pair change CG8878 altered predicted amino collected tested retention enhancer phenotype             acid coding sequence (Table 1, Figure 1). alleles crossing males w2; dp2; e2; E1 virgin females,                  (3a27a, 3a52a 3a97a) G/C T transitions retention recessive lethal phenotype crossing              resulted premature stop codons; 3a52a amino <br /> members complementation group. establishing                 terminal end predicted STKc domain absolute linkage recessive lethal dominant                    likely null allele. Allele 3a66a single <span id='am-8' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-9' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> <br /> enhancer phenotypes, position lethal locus refined             deletion caused frame-shift leading multiple premature <br />  <br />  <br /> PLOS | www.plosone.org                                                2                           March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                  Mutations Drosophila Putative Protein Kinase <br />  <br />  <br /> stop codons 1a27a G/C T transition                    method pigment determination accurate predicts loss intron donor splice site, frame-shift         precise. <br /> multiple premature stop codons. alleles (3a90a                    asked mutants effect classical <br /> 4a7a) identical nineteen base pair deletions 59 upstream         hPEV crossing y2 w2; dp2 CG8878*/CyO, Cy dp2 males promoter region included 4 base pairs proximal                  virgin 1)wm4; dp2; e2 females. Variegation wm4 visibly <br /> predicted E box presumptive transcriptional                    enhanced mutants tested (Figure 5  quantita- <br /> regulatory mutants.                                                         tively (95% 99% confidence limits) enhanced male                                                                             female flies respectively (Figure 5b). <br /> Phenotypic characterization mutants <br />     Visual pigment assessment dominant enhancement               <span id='am-16' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-21' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>Amino acid</span> sequence comparisons <br /> white-eyed variegation E1/+ heterozygotes indicated mutant              Analysis CG8878&#226;&#8364;&#8482;s predicted polypeptide sequence using <br /> alleles enhanced relative CyO control sexes,        SMART (University Heidelberg) predicts domains related <br /> frequently produced flies indistinguishable w2. Representa-            protein kinase separated 194 amino acids (Figure 1). tive photographs mutant eyes given Figure 2a. extent          comparison CG8878&#226;&#8364;&#8482;s predicted amino acid sequence enhancement quantified pigment assays. male              Drosophila species reveals homologs present <br /> female mutants half pigment non-mutant                 highly conserved species studied; supports <br /> internal control flies (CyO balancer) cross,         CG8878 essential gene (Figure 6, Figure S1). significant (95% confidence limit - Fig. 2b). mutant              cladogram parallels determined species [13]. <br /> alleles enhance silencing E1/+.                                             amino acid sequence CG8878 shows similarity <br />     dominant enhancement (w+ silencing) limited           D. melanogaster ballchen, human orthologs, Vaccinia Related E1 allele, quantitatively assayed effect mutants       Kinases (VRK1, VRK2), encode nucleosomal histone 2a Pci/+ flies, lack gypsy element present E1.             kinase (Figure 7, Figure S2). Amino acid sequence comparisons <br /> mutants reduced pigment compared non-mutant                suggest CG8878 ballchen derived common VRK <br /> internal control flies (CyO balancer) cross                   like precursor.  CG8878, VRK domain appears (Figure 3b).  putative null allele 3a52a showed            split ,194 amino acid insertion (Figures 1, <br /> significance sexes. Visually, variegation visible       8). CG8878 shows 36% identity 56% positive correlation cross, eyes displaying pattern weak silencing           BALLCHEN parts kinase domain (http://blast. <br /> starting posterior edge, rapidly decaying anterior              ncbi.nlm.nih.gov/Blast.cgi) indicating functional conservation. <br /> progression (Figure 3a).                                                    Note, mutation 3a22a (R546Opal) recessive lethal     address possibility mutations acting             enhances variegation E1, Pci, wm4 (Figures 2 b, 3 b, 5 b), directly w+ transgene, dependence                 appears second CG8878&#226;&#8364;&#8482;s split VRK-like kinase domain <br /> insertion position, assayed pigment y1 w* P{lacW}3-76a/+              (Fig. 8) essential CG8878&#226;&#8364;&#8482;s function. <br /> flies mutants. transgene P{lacW}3-76a Pci located 18A1 (60.7 cM) X               Structure CG8878 <br /> chromosome presents red eye. alleles tested,           amino acid sequence similarity human 4a7a, putative regulatory mutant, males showed barely              Casein Kinases human Tau-Tubulin Kinases. corre- <br /> significant difference (p = .03) non-mutant control flies (CyO         sponds PcK kinase conserved domain (called PHA02882). <br /> balancer) cross indicating dominant                     dot plot comparison CG8878 ballchen enhancement (silencing) dependent chromosomal location              hVRK1 (Figure 9) shows single, conserved PcK domain w+ transgene Pci construct itself (Figure 4).            split regions CG8878. CG8878 <br /> consistency results mutants tested indicates         sequence separated 5 regions (1&#226;&#8364;&#8220;96, 97&#226;&#8364;&#8220;291, 292&#226;&#8364;&#8220;516, <br />  <br />  <br />  <br />  <br /> Figure 1. Schematic representation CG8878 polypeptide sequence showing domains predicted SMART (University Heidelberg) location lesions described study. Mutant designations polypeptide backbone nature corresponding mutation  Regions sequence similarity ballchen (nhk-1) shown mauve bars CG8878 sequence. regions identified dot plots shown polypeptide diagram. <br /> doi:10.1371/journal.pone.0071695.g001 <br />  <br />  <br /> PLOS | www.plosone.org                                              3                           March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                             Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  Table 1. List EMS induced mutations CG8878, DNA sequence changes, predicted changes amino acid <br />  sequence. <br />  <br />  <br />  Mutant            Mutation                                Type                                                      Predicted amino acid change <br />  <br />  1a27a             GRA 8037095                             Loss intron donor splice site, frameshift.             Insert 130&#226;&#8364;&#8220;180Opal <br />  3a22a             CRT m1942                               Point, transition.                                        R546Opal <br />  3a52a             GRA m675                                Point, transition.                                        W123Opal <br />  3a66a             1D C m1665                              Frameshift.                                               454&#226;&#8364;&#8220;468Amber <br />  3a90a             19 bp D T 8,038,801-19                  59 regulatory deletion.                                   4 bp D E box <br />  3a97a             CR T m1942                              Point, transition.                                        R546Opal <br />  4a7a              19 bp D T 8,038,801-19                  59 regulatory deletion.                                   4 bp D E box <br />  <br />  Table showing mutagen used, coding sequence change, type mutation, resulting effective amino acid alteration mutant. <br />  doi:10.1371/journal.pone.0071695.t001 <br />  <br />  <br />  <br />  <br /> Figure 2. Enhancement E1 CG8878 mutations.  Representative photographs eyes class progeny heterozygous <br /> CG8878 mutants crossed parental E1 stock used mutagenesis. Cy versus Cy+ flies compared sex. Flies posed facing <br /> right. (y2 w2; dp2 3a52a/CyO, Cy dp2 = X w2; dp2; e2; E1 ). b) Pigment assays heterozygous CG8878 mutants crossed parental E1 stock <br /> used mutagenesis. Cy versus Cy+ flies compared sex. (y2 w2; dp2 CG8878/CyO, Cy dp2= X w2; dp2; e2; E1 ) Statistical <br /> significance pairs (T-Test = 1-tailed, independent (unpaired, unequal variance)) given mutant pair tested, * = p,0.05, <br /> ** = p,0.01, NS = significant. <br /> doi:10.1371/journal.pone.0071695.g002 <br />  <br />  <br /> PLOS | www.plosone.org                                                      4                              March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                   Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 3. Enhancement Pci CG8878 mutations.  Photographs representative examples class progeny heterozygous <br /> CG8878 mutants crossed Pci, parental stock E1 derived. Cy versus Cy+ flies compared sex. Flies posed <br /> facing right. (y2 w2; dp2 3a52a/CyO, Cy dp2 = X w2; dp2; e2; Pci ). b) Pigment assays heterozygous CG8878 mutants crossed parental <br /> Pci stock E1 derived  Cy versus Cy+ flies compared sex. (y2 w2; dp2 CG8878/CyO, Cy dp2 = X w2; dp2; e2; Pci ) Statistical <br /> significance pairs (T-Test = 1-tailed, independent (unpaired, unequal variance)) given mutant pair tested, * = p,0.05, <br /> ** = p,0.01, NS = significant. <br /> doi:10.1371/journal.pone.0071695.g003 <br />  <br /> 517&#226;&#8364;&#8220;662, 663&#226;&#8364;&#8220;1004), 2 4 corresponding PcK               genes identified; PcK containing genes single, <br /> conserved domain.                                                           uninterrupted domains. <br />    ClustalW sequence alignment D. melanogaster CG8878                Additional dot plot analysis CG8878 hVRK1, hCK1, Drosophila orthologs similar split PcK              hTTK1 (Figure 12) shows hVRK1 hCK1 single PcK <br /> domain 5 region organization (Figure S1). Comparison                 domains lack long C-terminal region. hTTK1 sequence variation 5 regions shows regions 2            single PcK domain, contains acid-rich repeat region, like 4 conserved orthologs, suggesting                  CG8878. D. melanogaster ortholog hTTK1 asator [14], evolutionary sequence conservation (Figure 10).                             single PcK domain N-terminal quarter    Dot plot analysis, comparing D. melanogaster D. virilis             polypeptide, lacks acid-rich repeat regions C- <br /> (Figure 11), shows regions repeated sequence         terminal quarters end. <br /> rich glutamic aspartic acid. Repeat 1 (position 300&#226;&#8364;&#8220;353)            examining 3D hVRK1 structure determined 42%, repeat 2 (421&#226;&#8364;&#8220;449) 46%, repeat 3 (790&#226;&#8364;&#8220;854)                 Nuclear Magnetic Resonance [15] linear alignment 50% glutamic aspartic acid content. Similarly positioned                CG8878 hVRK1 sequences, splitting PcK domain repeats mosquito genes   aegypti,  gambiae,       CG8878 correspond insertion Region 3 C. quinquefasciatus) supporting contention            catalytic loop (just putative Ser/Thr kinase active <br /> orthologs. Bombyx mori, moth, similar gene                          site) activation loop. location surface (BGIBMGA000118-PA_www.silkdb.org) split PcK domain,                  structure corresponds position 187&#226;&#8364;&#8220;191 (YKNPD) lacks acid-rich regions. split PcK domain         hVRK1 (Figure 13). <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                              5                             March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                  Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 4. Enhancement P{lacW}3-76a CG8878 mutations.  Photographs representative examples class progeny heterozygous CG8878 mutants crossed P{lacW}3-76a, insertion transgene Pci, 18A1 X chromosome. Cy versus <br /> Cy+ flies compared sex. Flies posed facing right. (y2 w2; dp2 3a52a/CyO Cy dp2 = X y1 w2 P{lacW}3-76a ). b) Pigment assays heterozygous CG8878 mutants crossed P{lacW}3-76a. Cy versus Cy+ flies compared sex. (y2 w2; dp2 CG8878/CyO Cy dp2 = X y1 w2 <br /> P{lacW}3-76a ) Statistical deviation pairs insignificant (T-Test = 1-tailed, independent (unpaired, unequal variance)) 4a7a <br /> males significant p,0.05 (* = p,0.05). <br /> doi:10.1371/journal.pone.0071695.g004 <br />  <br />    NucPred [16] predicts, score 1.0, nuclear localization       mutations resulting stop codons; amino terminal end <br /> signal (NLS) CG8878 position 17&#226;&#8364;&#8220;21 (RKRKR).                      CG8878&#226;&#8364;&#8482;s amino proximal predicted STKc domain likely <br /> sequence present similar position 12 Drosophila              represent null alleles, CG8878&#226;&#8364;&#8482;s predicted kinase <br /> orthologs (Figure S1), suggesting conserved. Similar NLS              domains, amino end CG8878&#226;&#8364;&#8482;s carboxy proximal <br /> sequences present, different locations,             predicted kinase domain. Taken  shows loss potential mosquito orthologs (score 0.95&#226;&#8364;&#8220;1.0). Discrete NLS                 CG8878 gene function responsible dominant <br /> sequences appear absent B. mori gene (Score 0.87).                 enhanced silencing w+ E1 recessive lethal phenotype. <br />    conclude hybrid nature (kinase sequence similar           Bioinformatic analysis CG8878 indicates likely protein <br /> ballchen/hVRK1 presence acid rich repeats like         kinase, putative functional domain split  <br /> hTTK1) split PcK domain CG8878 defines novel kinase                Furthermore, split form appears limited Dipterans. <br /> type added VRK, CK, TTK groups. <br />                                                                             CG8878 Hen1 <br /> Discussion                                                                     CG8878 transcription unit located entirely   induced, recovered, characterized seven mutations             large (5.4 kb) second intron gene, Hen1  Pimet), <br /> dominantly enhance variable silencing (variegation) E1,              antisense orientation. Hen1 shown mediate 29-O- expression similar P element dependent silencing                methylation 39 end Piwi interacting RNAs Drosophila <br /> (PDS). dominant enhancement genetically maps near                 [17,18]. Piwi interacting RNAs germ-line specific 24&#226;&#8364;&#8220;30 nt CG8878 locus separated lethal              RNAs couple PIWI proteins silence invading <br /> phenotype crossing  lethal phenotype deficiency maps            transposable elements (reviewed [19]). Given Pci P fine region includes CG8878. alleles contain            element terminal repeats  59 end, P element <br />  <br />  <br /> PLOS | www.plosone.org                                              6                            March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                   Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 5. Enhancement wm4 CG8878 mutations.  Photographs representative examples class progeny heterozygous CG8878 mutants crossed wm4. Cy versus Cy+ flies compared sex. Flies posed facing right. (y2 w2; dp2 3a52a/CyO, Cy <br /> dp2 = X wm4; dp2; e2 ). b) Pigment assays heterozygous CG8878 mutants crossed wm4. Cy versus Cy+ flies compared sex. (y2 w2; <br /> dp2 CG8878/CyO Cy dp2 = X wm4; dp2; e2 ) Statistical significance pairs (T-Test = 1-tailed, independent (unpaired, unequal variance)) given mutant pair tested, * = p,0.05, ** = p,0.01, NS = significant. <br /> doi:10.1371/journal.pone.0071695.g005 <br />  <br />  <br />  <br />  <br /> Figure 6. Comparison D. melanogaster CG8878 amino acid sequences Drosophila species. <br /> Accession numbers given Table S1 alignment Figure S1.  Score table showing degree similarity CG8878 homologues. b) <br /> Cladogram showing relative evolutionary distances CG8878 homologues Drosophila species (http://www.ebi.ac.uk/Tools/msa/ <br /> clustalw2/). Names abbreviated using capitalized letter genus followed letters species. <br /> doi:10.1371/journal.pone.0071695.g006 <br />  <br />  <br /> PLOS | www.plosone.org                                              7                             March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                     Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 7. Phylogram showing evolutionary distances CG8878, ballchen mammalian VRK1 VRK2. Abbreviations follows: Drosophila melanogaster (Dmel), Homo sapiens (Homo), Mus musculus (Mus) (http://www.ebi.ac.uk/Tools/msa/clustalw2/) Accession numbers <br /> given Table S1. <br /> doi:10.1371/journal.pone.0071695.g007 <br />  <br />  <br />  <br />  <br /> Figure 8. Pairwise alignment CG8878, BALLCHEN, closest Drosophila paralogue. Comparison symbols: * = identity, : = groups strongly similar properties, . = groups weakly similar properties. Amino acid color code: red = small hydrophobic, blue = acidic, <br /> magenta = basic, green = hydroxyl, sulfhydryl, amine, G. <br /> doi:10.1371/journal.pone.0071695.g008 <br />                                                          N    Accession numbers given Table S1. <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                                8                             March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                   Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 9. JDotter dotmatrix comparison [25] amino acid sequences D. melanogaster CG8878 (NP_610733.1) ballchen (NP_651508.1) human VRK1 (NP_003375.1). single PcK domain ballchen hVRK1 split CG8878. Horizontal, red <br /> dotted lines mark regions, 2 4 corresponding conserved PcK functional domain. vertical, green line separates ballchen hVRL1 sequences. Setting JDotter  Window size: 50; Zoom factor: 1 base/pixel; Pixel factor: 48; Scoring matrix: BLOSUM62; GreyMap Tool: <br /> 0, 13; Maximum Plot size: 700 bases/pixel; Sliding window size new plots: 50. <br /> doi:10.1371/journal.pone.0071695.g009 <br />  <br /> transposase lacZ fusion, considered Hen1, CG8878,            CG8878 essential gene mutated dominant potentially enhancer identified screen,             var) phenotype. points argue  1) seven mutants lesions CG8878 coding regulatory sequences; 2) lesions           Potential molecular function CG8878 <br /> entirely inside Hen1&#226;&#8364;&#8482;s second intron, predict effect Hen1             unable split kinase domain <br /> expression; 3) Hen1 essential gene PBac(WH)-               CG8878 homologues outside order Diptera, CG8878 Hen1[f00810] null Hen1 [18] recessive lethal; 4)         highly conserved Drosophila species (Figures 6, Figure S1). <br /> P{lacW}3-76a appears unaffected var)s despite                 conservation VRK like kinase domains construct different location; 5) wm4,           indicates likely encode kinase unknown, P element derived, significantly affected             essential, specificity. closest Drosophila melanogaster paralog var)s. parsimonious explanation                    CG8878 ballchen  NHK-1 homolog), regions maximum <br /> mutations lesions CG8878, Hen1,                   similarity coinciding CG8878&#226;&#8364;&#8482;s putative kinase domains  <br />  <br /> PLOS | www.plosone.org                                               9                            March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                       Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 10. Amino acid sequence similarity regions CG8878 Drosophila species. orthologs D. <br /> melanogaster CG8878  Figure S1) aligned using ClustalW sequence similarities counted identical, conserved, similar, match regions shown percent total region. Regions 2 4, correspond PcK domain <br /> conserved ballchen hVRK1, conserved orthologs regions. <br /> doi:10.1371/journal.pone.0071695.g010 <br />  <br /> shown Figure 1b). ballchen high affinity chromatin               arrested aberrant mitotic spindles polar bodies. shown phosphorylate Threonine 119 carboxy                    lack Histone H4K5 H3K14 acetylation terminus nucleosomal, free, H2A Drosophila embryos.               karyosomes nhk-1 mutant control oocytes, implying H2A T119 phosphorylated mitosis S phase                    Histone H2A threonine 119 phosphorylation required coincides NHK-1&#226;&#8364;&#8482;s chromatin association shown                  meiotic acetylation residues. Lancaster et al. [22] immunostaining component histone code                       phosphorylation barrier autointegration factor protein (BAF) <br /> related cell cycle progression [20]. Ivanovska et al. [21]                   NHK-1 necessary karyosome formation. Loss NHK- <br /> described point mutation, Z3-0437, kinase domain                    1 expression nonphosphorylatable BAF resulted ectopic <br /> NHK-1 led female sterility defects formation              chromosome-nuclear envelope association oocytes leading karyosome. led metaphase arrest result                 authors propose tethering chromosomes nuclear <br /> failure synaptonemal complex disassemble load                  envelope disrupted NHK-1 mediated BAF phosphorylation, <br /> condensin chromosomes mutant. Mitosis                      allowing karyosome formation oocytes. <br /> shown affected, embryos laid nhk-12/2 mutant females <br />  <br />  <br />  <br />  <br /> Figure 11. JDotter dotmatrix comparison [25] amino acid sequences D. melanogaster CG8878 (NP_610733.1) orthologs D. virilis (XP_002050972.1),  aegypti (EAT48618.1), B. mori (BGIBMGA000118-PA_www.silkdb.org). repeated, aspartic <span id='am-23' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span>glutamic acid</span> rich regions present Diptera absent B. mori, moth. Setting JDotter Figure 9, Maximum Plot size: 1200 bases/pixel. <br /> doi:10.1371/journal.pone.0071695.g011 <br />  <br />  <br /> PLOS | www.plosone.org                                                 10                             March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                     Mutations Drosophila Putative Protein Kinase <br />  <br />  <br />  <br />  <br /> Figure 12. JDotter dotmatrix comparison [25] amino acid sequences D. melanogaster CG8878 (NP_610733.1) orthologs D. virilis (XP_002050972.1), human VRK1 (NP_003375.1), human CK1 (CaseinKinase 1; NP_620693.1), human <br /> TTK1 (Tau-Tubulin Kinase 1; NP_115927.1). repeated, aspartic glutamic acid rich regions present hTTK1 hVRK1 hCK1. Setting JDotter Figure 11. <br /> doi:10.1371/journal.pone.0071695.g012 <br />  <br />    CG8878&#226;&#8364;&#8482;s exact target mode action                     domain, argue predicted kinase domain determined, sequence similarities suggest Histone                    essential CG8878 function. <br /> phosphorylation CG8878 readily explain action                    putative Kinase coding region CG8878 similar var). example, JIL1 phosphorylation H3S10 blocks                 hVRK1, split segments (Regions 2 4). methylation H3K9 allowing hyperacetylation Histone 3                conserved NLS sequence supports nuclear localization promoting transcriptionally active chromatin state [23].                    possible role chromatin modification. conserved presence <br /> CG8878&#226;&#8364;&#8482;s expression profile consistent genome              aspartic glutamic acid rich repeats suggest possible <br /> wide inhibitor heterochromatin spread expressed            interaction sites. lacking hCK1, cytosolic protein, <br /> tissues, stages development, maxima times peak           present hTTK1, absent D. melanogaster asator <br /> developmental change, early embryogenesis pre-                    (TTK1 ortholog).  suggests CG8878 encodes pupariation [24].                                                             protein Kinase modifies chromatin structure. <br />    mutants suggest predicted kinase domains essential function. enhancer  E1 wm4) phenotypes                     CG8878 acts ci locus <br /> recessive lethal phenotypes 3a66a, results premature                Pci isolated enhancer trap ci locus stop codon CG8878&#226;&#8364;&#8482;s predicted kinase domains,                 enhancer-trap reporter accurately mimicked ci RNA 3a22a, 3a97a, result premature stop codon               expressed specifically anterior compartment cells amino end CG8878&#226;&#8364;&#8482;s carboxy proximal predicted kinase                       imaginal discs [1]. w+ transgene Pci  E1 gypsy <br />                                                                               element) inserted ci distal regulatory region. Pci                                                                               recessive allele ci exhibits ci wing phenotype                                                                               heterozygous ci57g  deletion upstream Pci regulatory <br />                                                                               region) ci1  gypsy insert upstream Pci). mutant <br />                                                                               CG8878 alleles enhance variegation (reduce w+ expression                                                                               transgene) E1 E1/Pci (unpublished observation),                                                                               little effect P{lacW}3-76a, construct different <br />                                                                               location. silencing location dependent                                                                               likely direct interaction white promoter,                                                                               ci regulatory region itself. Pci reporter expression                                                                               approximately halved 3a52a present, does depend <br />                                                                               presence E1, infer CG8878 normally acts ci <br />                                                                               regulatory region impede spread heterochromatin                                                                               region, likely dose sensitive manner. <br />  <br />                                                                               Supporting Information <br /> Figure 13. dimensional structure diagrams, deter- <br /> mined Nuclear Magnetic Resonance, human hVRK1 (file                     Figure S1 Pairwise alignment CG8878 12 Dro- <br /> 3OP5) displayed using Cn3D4.3 software [26]. Regions hVRK1                 sophila homologues. Species names abbreviated using similar D. melanogaster CG8878 highlighted yellow.         capitalized letter genus followed letters <br /> Regions 2 (left) 4 (centre) correspond conserved sequence <br /> regions. location regions, corresponding         species. Comparison symbols: * = identity, : = groups <br /> insertion Region 3 (right) surface polypeptide        strongly similar properties, . = groups weakly similar <br /> highlighted yellow.                                                        properties. Amino acid color code: red = small hydrophobic, <br /> doi:10.1371/journal.pone.0071695.g013                                         blue = acidic, magenta = basic, green = hydroxyl, sulfhydryl, <br />  <br />  <br /> PLOS | www.plosone.org                                               11                             March 2014 | Volume 9 | Issue 3 | e71695 <br />                                                                                                                       Mutations Drosophila Putative Protein Kinase <br />  <br />  <br /> amine, G (http://www.ebi.ac.uk/Tools/msa/clustalw2/. Note:                                   Table S1 Polypeptide accession numbers used. Note: D. persimilis nucleotide removed  s s &#226;&#8364;&#8220;                         persimilis sequence modified removal nucleotide <br /> presumed sequencing error) facilitate amino acid alignment.                               DNA sequence alter amino acid reading frame Accession numbers given Table S1.                                                         facilitate alignment. <br /> (TIF)                                                                                        (PDF) <br /> Figure S2 Pairwise alignment BALLCHEN VRK1 mouse humans. Symbols Figure S1. <br />                                                                                              Acknowledgments <br /> Comparison symbols: * = identity, : = groups strongly                              thank Scott Hanna Tanya McCracken technical <br /> similar properties, . = groups weakly similar properties.                          assistance. <br /> Amino acid color code: red = small hydrophobic, blue = acidic, <br /> magenta = basic, green = hydroxyl, sulfhydryl, amine, G (http://                             Author Contributions <br /> www.ebi.ac.uk/Tools/msa/clustalw2/.) Accession numbers given                                 Conceived designed experiments: JL. Performed Table S1.                                                                                 experiments: JL. Analyzed data: JL. Contributed reagents/ <br /> (TIF)                           <br /> </body></html>