<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>512                                                                              Biophysical Journal   Volume 105    July 2013     512&#226;&#8364;&#8220;522 <br />  <br />  <br /> Quantitative Assessment Thermodynamic Constraints Solution <br /> Space Genome-Scale Metabolic Models <br />  <br /> Joshua J. Hamilton, Vivek Dwivedi, Jennifer L. Reed* <br /> Department <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Biological Engineering, University Wisconsin-Madison, Madison, Wisconsin <br />  <br />  <br /> ABSTRACT Constraint-based methods provide powerful computational techniques allow understanding prediction cellular behavior. methods rely physiochemical constraints eliminate infeasible behaviors space available behaviors. constraint thermodynamic feasibility, requirement intracellular flux distributions <br /> obey laws thermodynamics. past decade seen constraint-based methods interpret constraint different ways, including limited small networks, rely predefined reaction directions, neglect relationship reaction free energies metabolite concentrations. work, utilize approach, <br /> thermodynamics-based metabolic flux analysis (TMFA), make genome-scale, quantitative predictions metabolite  <br /> centrations reaction free energies absence prior knowledge reaction directions, accounting uncertainties thermodynamic estimates. applied TMFA genome-scale network reconstruction Escherichia coli examined effect thermodynamic constraints flux space. assessed predictive performance TMFA gene <br /> essentiality quantitative metabolomics data, aerobic anaerobic, optimal suboptimal growth condi- <br /> tions. Based results, propose TMFA useful tool validating phenotypes generating hypotheses, additional types data constraints improve predictions metabolite concentrations. <br />  <br />  <br /> INTRODUCTION <br /> Genome-scale network reconstructions provide concise                    (7), loops (8). constraints nonlinear, <br /> mathematical representations organism&#226;&#8364;&#8482;s biochemical               EBA restricted small models, scalable algo- <br /> capabilities, serve platform constraint-based              rithm recently proposed (9). scientists <br /> techniques used understanding predicting            developed method compute eliminate closed <br /> cellular behavior (1,2). predictive accuracy                     cycles flux-balance models (10,11) based solely constraint-based methods depends degree                 stoichiometry, technique remains computa- <br /> constraints eliminate physiochemically biologically                 tionally demanding genome-scale networks. Recently, <br /> infeasible behaviors.                                                   scalable, mixed-integer approach eliminating closed <br />    Flux-balance analysis (FBA) (3) commonly employed                 cycles developed (8). predict state network identifying steady-               limited availability free energy data state flux distribution maximizing cellular growth,               reactions metabolites (12,13), satisfying mass-balance enzyme capacity                    approaches directly account relationships <br /> straints. Reaction directionality typically assigned based           Dr G, metabolite concentrations, free energies enzyme assays biological considerations (e.g.,                 formation (Df G). Fortunately, group contribution <br /> free <span id='am-41' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-42' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> synthesis), consideration given thermo-             methods (GCMs) (14&#226;&#8364;&#8220;16) developed estimate <br /> dynamic feasibility. second law thermodynamics                   free energies metabolites reactions data states negative Gibbs energy reaction (Dr G)               unavailable. recent model Escherichia coli used needed drive forward reaction flux, v, leading              GCM assign reaction directionalities based thermodynamic feasibility constraint v $ Dr G &gt; 0                   thermodynamic feasibility (16,17). approaches, <br /> nonzero v.                                                              experimentally measured thermodynamic data    attempt enforce thermodynamic feasibility            combined heuristics group contribution data FBA energy balance analysis (EBA) (4,5), incor-               define feasible reaction directions E. coli (18,19).  <br /> porated nonlinear constraints chemical potentials                approaches (17&#226;&#8364;&#8220;19) neglect thermodynamic <br /> FBA eliminate closed cycles. Closed cycles sets               interactions reactions network arise <br /> reactions overall thermodynamic driving force             shared metabolites. result, directionality zero, net flux occur (e.g.,               reaction assigned independently reaction <br /> B / C / . closed cycles variously                     directions network. example, reactions <br /> referred Type III pathways (6), internal flux cycles              feasible forward reverse directions, <br />                                                                         shared metabolite, pair reactions                                                                         proceed direction. approaches fail Submitted October 5, 2012, accepted publication June 5, 2013. <br />                                                                         capture type thermodynamic coupling *Correspondence: reed@engr.wisc.edu <br />                                                                         reactions. <br /> Editor: Daniel Beard. <br />   2013 Biophysical Society <br /> 0006-3495/13/07/0512/11 $2.00                                                                     http://dx.doi.org/10.1016/j.bpj.2013.06.011 <br />  Quantitative Assessment TMFA                                                                                                                                  513 <br />  <br />    GCMs used approaches capture                                                 vmin %vj %vmax cj;Jirrev :                                (4) <br /> thermodynamic interactions including metabolite  <br /> centrations variables. EBA extended predict                      network steady state, net production metabolites intracellular metabolite concentrations small network                      zero. reaction constrained flux appropriate <br /> (20), mixed-integer approaches                           range given enzyme capacities, reactions assumed                                                                                 irreversible (j &#203;&#8250; Jirrev). limits reactions set vmax &#194;&#188; <br /> developed, thermodynamic constraints                               1000 mmol/gDW/h vmin &#194;&#188;  1000 mmol/gDW/h, measured <br /> imposed predefined reaction directions. NET                           fluxes (e.g., carbon uptake rates). <br /> analysis (21) integrates quantitative metabolomics data                            Previous implementations TMFA (7,23) imposed thermody- thermodynamic constraints predict feasible free                         namic constraints FBA, allowing constraints energy ranges reactions network.                         restrict reaction directions;  reaction deemed irreversible <br />                                                                                 FBA reversible indicated thermodynamic <br /> method, thermodynamics-based metabolic flux analysis                            constraints. contrast, implementation makes assumptions (TMFA) (7), extends FBA thermodynamic constraints,                         reaction  reversibility, allowing thermodynamic constraints decide <br /> enabling quantitative prediction feasible ranges                      directions reactions. replaced Eqs. 3 4 metabolite concentrations reaction free energies, <br />                                                                                                           vmin %vj %vmax cj&#203;&#8250;J                                    (5) relying metabolomic data.  methods  <span id='am-11' about='oboInOwl:date' typeof='owl:Thing'>date</span>, relied prior knowledge                                                                                 rely thermodynamic constraints determine reaction reversibility directionality reactions (7,21&#226;&#8364;&#8220;23),                     directions. restricting predictive capabilities. <br />    work, examine extent thermody- <br /> namics-based flux-balance methods make genome-                              Calculating free energies reaction <br /> scale, quantitative predictions, absence outside                      Enforcing thermodynamic constraints requires knowledge standard <br />                                                                                                                                 0 <br /> information flux directions, considering presence                   transformed Gibbs energy reaction (Dr G 0) reactions absence uncertainty thermodynamic estimates.                       model. paucity experimental data, group contribution methods <br />                                                                                 (14,15) used provide estimates uncertainties standard trans- end, applied TMFA iJR904 model E. coli                                                                  0 <br />                                                                                 formed Gibbs energy formation (Df G 0) metabolites reaction <br /> (24). model used group contribution esti-                            0 <br />                                                                                         0 <br />                                                                                 (Dr G ) reactions. <br />                                                                                                                  0 <br /> mates available higher fraction metabolites                       GCMs assume Df G 0 metabolite linear combination iJR904 model newer models (16). assessed                      formation energies constituent molecular substructures predictive performance TMFA number                           groups), k, <br /> large-scale datasets (22,25&#226;&#8364;&#8220;27), encompassing metabolite                                                        0 <br />                                                                                                                         X                      0 <br />                                                                                                        Df G 0 &#194;&#188;                ni;k Dgr Gk0 ;                  (6) <br /> concentrations gene deletion phenotypes,                                                                  k <br /> aerobic anaerobic, optimal suboptimal growth <br />                                                                                               0                                                                   0 <br /> conditions. analysis, highlight impor-                      Dgr Gk 0 estimated contribution group k overall Df G 0, <br /> tance quantitative concentration measurements                            ni,k number groups k molecular structure compound  <br />                                                                                 used software implementation GCM Jankowski et al. (16), <br /> thermodynamic coupling achieving physiologically real-                       Henry et al. (28), Finley et al. (29) obtain estimates uncertainties <br /> istic predictions growth rates flux distributions.                             0           0 <br />                                                                                 Df G 0 Dr G 0 metabolites reactions iJR904 network able generate hypotheses regarding metabolite                       Supporting Methods Table S1 Table S2 Supporting <br /> concentrations thermodynamic bottlenecks,                            Material). <br />                                                                                                                                    0 <br />  <br /> discuss additional types data constraints                          GCM method returns estimates Df G 0 predominant ionic <br />                                                                                 species biochemical standard state: pH 7, zero ionic strength, temper- <br /> improve predictions metabolite concentrations.                               ature 298 K. Simulations performed conditions 298 K zero <br />                                                                                 ionic strength, pH 7 (7.4) intra- (extra)cellular metabolites, using <br />                                                                                 ionic species represented iJR904 model stoichiometric matrix. <br /> MATERIALS METHODS                                                           differences pH major ionic species required adjusting                                                                                                          0 <br />                                                                                 GCM estimates Df G 0  Supporting Material). <br />                                                                                                                       0                       0 <br /> Overview relationship previous                                              Using new estimates Df G 0, calculated Dr G 0from stoichi- <br /> implementations                                                                 ometry reaction, <br />                                                                                                                     0 <br />                                                                                                                         X                  0 <br /> Given stoichiometric matrix S set reactions J, flux-balance analysis                           Dr G j 0 &#194;&#188;               Si;j Df Gi0 ;                   (7) <br /> (FBA) seeks steady-state flux distribution (v) maximizing flux                                             biomass reaction (vBM), satisfying mass-balance enzyme <br /> capacity constraints individual reactions, j:                               Si, j stoichiometric coefficient metabolite reaction j.                                                                                                                                                         0 <br />                                                                                 calculated transformed Gibbs energy reaction (Dr G )                                 max vBM ;                                (1)    function metabolite concentration, xi, <br />                                                                                                   0         0 <br />                                                                                                                          X                             0 <br />                                                                                            Dr Gj &#194;&#188; Dr Gj0 &#195;&#190; RT                     Si;j ln&#195;&#176;xi &#195;&#382; &#195;&#190; Dt Gj0 ;       (8) <br />                              s:t: S$v &#194;&#188; 0;                               (2)                                                                                                                                                                                                              0 <br />                                                                                 R <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-7' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>gas</span> constant, T temperature (298 K), Dt G 0 reflects <br />                         0%vj %vmax cj&#203;&#8250;Jirrev ;                           (3)                              0 <br />                                                                                 contribution Dr G transport metabolites  <br />                                                                                                                         Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  514                                                                                                                                                    Hamilton et al. <br />  <br />                                                                               0 <br /> membrane. Following established derivation (7), calculated Dt G 0                                             vj R0 cj&#203;&#8250;Jfor ;                              (15) function electrochemical potential (DJ) pH gradient (DpH) membrane, <br />                        0 <br />                                                                                                                      dj &#194;&#188; 0 cj&#203;&#8250;Jrev ;                               (16) <br />                  Dt Gj0 &#194;&#188; cj FDJ   2:3 hj RTDpH;                                  (9) <br />                                                                                                                       vj %0 cj&#203;&#8250;Jrev ;                               (17) F Faraday&#226;&#8364;&#8482;s constant, cj net charge transported outside inside cell, hj number protons transported membrane  Table S3 details). constraints intra- extracel-                                          vj &#194;&#188; 0 cj&#203;&#8250;Jbl :                               (18) <br /> lular proton concentrations resulted values  130 mV DJ, 0.4 DpH (7,30). <br />                                                                                         reactions, flux maximization using RTMFA computationally <br />    performed simulations uncertainties esti- <br />                  0                                                      0               intensive, FVA simulations performed time limit 5 min. <br /> mates Dgr G 0 group. TMFA, fixed values Dgr G 0                 result, possibility true flux ranges larger reported <br />                                    0 <br />                                      0 estimated values (Dgr Gest ) given group contribution <br />                                                                                          Results). <br /> method, <br />                                       0           0 <br />                                Dgr Gk0 &#194;&#188; Dgr Gk;est <br />                                                0 <br />                                                     :                         (10) <br />                                                                                         Differences phenotype: CONGA <br />                                                                   0 relaxed TMFA (RTMFA), allowed Dgr G 0 group                    used algorithm previously developed, CONGA (33), identify <br /> vary 95% confidence interval, determined standard error            single, double, triple gene deletions predicted lethal FBA (SE) reported GCM software  e., SEDgr G0 0 ):                                  TMFA  RTMFA). CONGA employs bilevel optimization problem <br />                                                           k;est <br />                                                                                         identify gene deletion strategies maximizing difference       00                                  0           0 <br /> Dgr Gk;est  2SEDgr G0 0 %Dgr Gk0 %Dgr Gk;est <br />                                         0 <br />                                              &#195;&#190;2SEDgr G0 0 : (11)                        maximum biomass flux different models, FBA TMFA <br />                            k;est                                      k;est              RTMFA). maximum possible difference biomass flux <br />                                                                                         occurs model zero biomass flux, CONGA identifies gene Supporting Methods Supporting Material additional                    deletions lethal model. Additional details details.                                                                                Supporting Methods Supporting Material                                                                                         original work (33). <br />  <br /> Enforcing thermodynamic consistency <br />                                                                                         Synthetic lethals phenotype correction <br /> Thermodynamic consistency requires reaction fluxes  <br />                                      0 <br /> sistent predicted values Dr G  e., v $ Dr G &lt; 0). employed                 identified gene deletions lethal FBA model, mixed-integer approach enforce requirement, binary                 employed synthetic lethals (SL) Finder (34) identify reaction(s) <br /> variable d indicates reaction operating forward (d &#194;&#188; 1)                  responsible difference predicted phenotype. SL Finder employs reverse (d &#194;&#188; 0) direction. added following equations                  bilevel optimization problem identify reaction deletion strategies model:                                                                              minimize maximum biomass flux network.                                                                                         lowest maximum biomass zero, SL Finder finds lethal reaction <br />                            &#195;&#176;1   d&#195;&#382;vmin %v%dvmax ;                             (12)      deletions. Additional details Supporting Methods                                                                                         Supporting Material original work (34). <br />                                               0                                            gene deletions predicted lethal FBA, TMFA pre- <br />                    Md &#195;&#190; &#206;&#181;%Dr G %M&#195;&#176;1   d&#195;&#382;   &#206;&#181;:                                 (13)      diction disagreed experimental observation. suggested                                                                                     0 <br />                                                                                         SL reaction did operate vivo direction predicted FBA. Eq. 13 big-M constraint (31) M upper limit Dr G             instances, developed constraint metabolite concentrations <br /> (300 kcal/mol) &#206;&#181; small nonzero number (10 6). Equations 1, 2, 5&#226;&#8364;&#8220;9,             prevented SL reaction operating TMFA-predicted 12&#226;&#8364;&#8220;13 used TMFA equations 1, 2, 5&#226;&#8364;&#8220;8, 10&#226;&#8364;&#8220;13                     direction, correcting phenotype. refer constraints <br /> used RTMFA.                                                                          phenotype-correction constraints. Additional details                                                                                         Supporting Methods Supporting Material. <br />  <br /> Flux thermodynamic variability analysis <br />                                                                                         Simulation conditions performed flux thermodynamic variability analysis variety conditions identify thermodynamically feasible flux                   simulations performed using CPLEX 12 (IBM, Armonk, NY) <br /> metabolite concentration ranges. flux variability analysis (FVA) (32),               accessed General Algebraic Modeling  Ver. 23.3.3 (GAMS; flux v reaction minimized maximized subject                  GAMS Development Corporation, Washington, DC). Simulations  constraints TMFA. thermodynamic variability analysis (TVA)                    formed Red Hat Enterprise Linux server 2.66 GHz Intel Xeon pro- <br />                                                           0 <br /> (7), concentration metabolite  Dr G reaction)                 cessors 8 GB RAM. TMFA formulation solves seconds, <br /> minimized maximized subject constraints TMFA. Additional                  RTMFA formulation takes ~1 h prove global optimality. <br /> details Supporting Methods Supporting <br /> Material. <br />    Using FVA results, defined sets reactions operate            Sources experimental data experimental forward (Jfor) reverse (Jrev) directions, blocked (Jbl) entirely.      methods imposing following constraints model, able reduce <br /> simulation times order magnitude:                                         Details experimental datasets methods used validate                                                                                         TMFA predictions given Supporting Methods Supporting <br />                                    dj &#194;&#188; 1 cj&#203;&#8250;Jfor ;                           (14)      Material. <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                          515 <br />  <br /> RESULTS                                                                                            asp      2H+ <br />  <br /> Optimization aerobic growth glucose using <br /> TMFA                                                                                    ABC-type              proton <br />                                                                                        transporter          symporter <br />                                                                                                                             Membrane used thermodynamics-based metabolic flux anal- <br />                                                                                                                                Cytosol <br /> ysis (TMFA) determine maximum growth rate                               H+ <br /> E. coli glucose aerobic conditions, using iJR904                                          asp       2H+ <br />                                                                                     <span id='am-12' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>           ATP <br /> genome-scale metabolic model (24). Building previous <br /> work (7), constrained concentration intracellular                              [x]min    [x]max         [x]meas <br />                                                                  B              RTFMA    (mM)      (mM)           (mM) <br /> metabolites range 0.001&#226;&#8364;&#8220;20 mM. constrained <br />                                                                                   ADP     0.89       20       0.437 0.704 concentrations extracellular nutrients                                                                                    ATP    0.01     0.22        8.13 11.4 <br /> 0.4% glucose MOPS medium (35). concentration intracellular H&#195;&#190; fixed pH 7, extracellular     FIGURE 1 Examples thermodynamically feasible physiologically <br /> H&#195;&#190; pH 7.4. concentration extracellular gases    implausible behavior.   RTMFA predicts ATP synthesized (<span id='am-81' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-82' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="O2" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span>O2</span> <span id='am-30' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-31' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span>CO2</span>) based experimental measurement (36),       cycling small molecules membrane. cycle involves <br />                                                                asp. (B) Metabolite concentration ranges predicted RTMFA requirement intracellular concentration     cycle shown panel thermodynamically feasible (columns 1 extracellular concentration.     2), experimentally measured metabolite concentrations (column <br /> ditions, TMFA predicted maximum growth rate zero,         3). (Highlighted type) Constraining concentration model renders <br /> suggesting growth possible thermo-     cycle thermodynamically infeasible. <br /> dynamic infeasibility essential reactions. expanded concentration range seven metabolites enable thermodynamic feasibility essential reac-      ATP (Fig. 1 B). additionally constraining concentra- <br /> tions  Supporting Results Supporting Material).     tion ATP experimentally measured range (22), simulations performed glucose limiting    ATP present higher concentration ADP, <br />                                                                force RTMFA model use ATP synthase substrate, maximum uptake rate 10 mmol/gDW/h. <br />                                                                synthesize ATP. constraint ATP concentration set metabolite concentration exchange flux <br />                                                                added model, predicted growth rate exceeds <br /> constraints Table S6. <br />                                                                FBA 3%. constraint ATP concentration <br />    performed simulations neglecting uncertainty <br />                                                                used subsequent RTMFA simulations discussed standard Gibbs energy formation group <br />         0                                                      work. <br /> (Dgr G 0) (Eq. 10). maximum growth rate predicted TMFA exceeded FBA 2%, despite increased network flexibility possible lack     Flux variability analysis: thermodynamically <br /> predefined reaction directions TMFA.                        feasible reaction directions <br />                                               0 <br />    introduced uncertainty Dgr G 0 (Eq. 11)                                                                used flux variability analysis (FVA) determine <br /> observed maximum growth rate exceeded         thermodynamically feasible directions reactions FBA ~12%. elevated growth rate highlights          network glucose aerobic conditions (Fig. 2  <br /> additional network flexibility possible uncer-     Table S7 Table S8). classified reactions tainty free energy estimates arising GCM.       fully bidirectional, constrained ways: <br /> particular, growth rate difference mecha-       blocked entirely; capable operating forward direc- <br /> nisms relaxed TMFA model (RTMFA) enable            tion  capable operating reverse direction <br /> ATP synthesized lower energetic cost occurs       performed FVA fully reversible version <br /> physiologically  reflected FBA). example,        iJR904 model thermodynamic constraints <br /> RTMFA predict ATP synthesis ATP synthase,           (FBAr), reactions  biomass) using energy released transport protons       allowed reversible. allowed identify direc- plasma membrane. Instead, RTMFA model           tionally constrained reactions based stoichiometry identifies numerous cycles synthesize ATP using energy    external environment. <br /> released transport fewer numbers protons,       observed FBAr, majority reactions shuttling metabolites forth plasma mem-    bidirectional (60%), remainder constrained <br /> brane (Fig. 1 . metabolite aspartate (asp),      way. hand, FBA model large enables ATP synthesized using energy released      set irreversible reactions, causing number bidirec- transport protons plasma       tional reactions decrease significantly, mere 5% membrane.  shuttling relies aspartate      network. majority reactions FBA model <br /> ABC-type transporter pumping aspartate,           operate forward direction   quires ADP present concentration higher        neglect priori reaction directionality assignments  <br />                                                                                                      Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  516                                                                                                                                                                                         Hamilton et al. <br />  <br />                                                         reactions         thermodynamic   uncertainties <br />                                                                                                                   TMFA                         reaction directionalities FVA analysis                                                                                                         &#194;&#180;0      directions <br />                                                           reversible             constraints     &#226;&#710;&#8224;fGest <br />                                                     FBA                  FBAr              TMFA           RTMFA           TMFA-LP <br />                                                                                                                                             TMFA model used constrain fluxes                                                0 <br />                                                                                                                                             FBAr (TMFA-LP), predicted growth rate 150%                                          100 <br />                                                                                                                                             predicted FBA (Fig. 2 C). increase growth <br />                                                                                                                                             rate indicates ATP- energy-generating cycles   Number Reactions <br />  <br />  <br />  <br />  <br />                                          200                                                                                                present network present thermody- <br />                                          300 <br />                                                                                                                                             namic constraints imposed directly. cycle <br />                                                                                                                                             involves shuttling sodium ions forth                                          400                                                                                                membrane, resulting proton gradient used synthe- <br />                                          500 <br />                                                                Blocked                                                                      size ATP  Fig. S1 Supporting Material). cycle <br />                                                                Forward <br />                                                                Reverse                                                                      infeasible TMFA, despite reaction operating                                                                Bidirectional <br />                                          600 <br />                                                                 LP    MILP <br />                                                                                                                                             thermodynamically feasible direction. emphasizes <br />                                                                                                                                             need actively impose thermodynamic constraints <br /> B                                                                                                                                           account thermodynamic interactions reac- <br />                                                                                                                                             tions (7,8,21), opposed methods impose thermody- <br /> Cumulative Distribution Function (CDF) <br />  <br />  <br />  <br />  <br />                                          0.8                                                                                                namically feasible reaction directions accounting <br />                                                                                                                                             thermodynamic coupling reactions (17&#226;&#8364;&#8220;19). <br />                                          0.6                         C               Growth Rate (Fraction FBA Growth Rate) <br />                                                                                                                                                Finally, used FVA results formulation                                                                                  0   0.2    0.4   0.6     0.8    1       1.2   1.4    1.6 <br />                                                                           FBA <br />                                                                                                                                             cumulative distribution function (CDF) flux <br />                                          0.4                             TMFA                                                               span, difference maximum minimum <br />                                                                      RTMFA                                           *                      flux given reaction (Fig. 2 B Table S9). <br />                                          0.2        FBA <br />                                                     TMFA <br />                                                             TMFA-LP <br />                                                                                                           * [ATP] constraint <br />                                                                                                                                             CDF FBA sharpest, &gt;90% reac- <br />                                                     RTMFA <br />                                                     TMFA-LP <br />                                                                                                                                             tions having span &lt;300 mmol/gDW/h. CDF                                           0                                                                                                 TMFA similar FBA, despite increase                                                0   200      400       600         800   1000 1200 1400            1600       1800    2000 <br />                                                                                Flux Span (mmol / gDW / hr)                                  network flexibility. CDF shallow                                                                                                                                             RTMFA  reactions larger spans), reaction&#226;&#8364;&#8482;s <br /> FIGURE 2 Comparison thermodynamic formulations glucose <br /> aerobic conditions.   Reactions classified blocked, forward, <br />                                                                                                                                             span increase directionally unconstrained. <br /> reverse, bidirectional based thermodynamic feasibility. Differ-                                                                 CDF TMFA-LP shallow: jumps ences formulation given chart. (B) Cumulative                                                                    CDF value flux spans 1000 2000 mmol/gDW/h <br /> distribution function (CDF) flux span formulation. (C) Plot                                                                point existence thermodynamically infea- growth rate formulation, normalized FBA growth rate.                                                                     sible closed cycles (e.g.,  B / C /  (Formulations) FBAr: FBA reaction directions fully reversible. <br /> TMFA: FBAr thermodynamic constraints, global metabolite bounds, <br />                                                                                                                                             participating reactions operate maximum flux. media constraints. RTMFA: cTMFA uncertainties Df G 0est <br />                                                                         0 <br />                                                                                                                                             results highlight important role thermodynamic interac- constraint ATP concentration. TMFA-LP: LP FBAr reaction                                                                       tions play shaping feasible flux space. observed <br /> directions consistent TMF.                                                                                                             sets bidirectional constrained reactions vary <br />                                                                                                                                             slightly media conditions, overall CDFs <br />                                                                                                                                             remain qualitatively similar (data shown). <br /> instead allow reaction directions determined solely thermodynamic constraints (TMFA), fraction direc- <br />                                                                                                                                             Gene deletion studies: comparison FBA <br /> tionally constrained reactions 85%, compared                                                                                                                                             TMFA <br /> 95% FBA, 40% FBAr. reveals thermody- <br /> namic constraints play major role eliminating                                                                                  glucose aerobic conditions, CONGA identified 22 network flexibility resulting eliminating predefined                                                                               single gene deletions TMFA FBA reaction directions (FBAr).  relative FBA                                                                               different growth phenotype predictions (Table 1). 19 <br />  predefined reaction directions), TMFA enabled previ-                                                                                  cases, TMFA predicted nonlethal phenotype FBA pre- <br /> ously forward- reverse reactions bidirec-                                                                                 dicted lethal  remaining cases <br /> tional, previously blocked reactions feasible                                                                                 TMFA predicted lethal phenotype FBA predicted reverse direction.                                                                                                                   nonlethal  Relative experimental data (25), corre- <br />    Just moving FBA TMFA led decrease                                                                                     sponded better prediction TMFA seven cases, number constrained reactions, does moving                                                                                     worse prediction remaining 15 cases. Using RTMFA TMFA RTMFA. number constrained reac-                                                                                         instead TMFA introduces worse predictions <br /> tions decreases 85% TMFA 77% RTMFA.                                                                                           better prediction (Table 1). number bidirectional reactions increases 50%,                                                                                        TMFA predict growth FBA predicts previously forward reactions feasible                                                                                   growth, TMFA enable reaction proceed direc- reverse direction.                                                                                                                   tion allowed FBA. used variant SL Finder <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                                      517 <br />  <br /> TABLE 1 Single-gene deletions FBA (R)TMFA predict different growth phenotypes glucose aerobic <br /> conditions <br />                       Gene locus     Gene       Phenotype        FBA          TMFA        TMFA (corrected)        RTMFA        RTMFA (corrected) <br /> Better TMFA          b0907           serC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b0928           aspC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b2913           serA         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b3359           argD         Nonlethal      Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3429           glgA         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b3430           glgC         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br />                         b4388           serB         Nonlethal      Lethal       Nonlethal          Nonlethal         Nonlethal          Nonlethal <br /> Worse           b0474            adk          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b0720           gltA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b1207           dnaR          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b2615           nadK          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b2818           argA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3607           cysE          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3608           gpsA          Lethal        Lethal       Nonlethal           Lethal           Nonlethal          Nonlethal <br />                         b3729           glmS          Lethal        Lethal       Nonlethal           Lethal           Nonlethal          Nonlethal <br />                         b3957           argE          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3958           argC          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3959           argB          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b4226            ppa          Lethal        Lethal       Nonlethal           Lethal           Nonlethal           Lethal <br />                         b3919           tpiA         Nonlethal     Nonlethal      Lethal             Lethal            Lethal             Lethal <br /> Worse TMFA           b1849           purT         Nonlethal     Nonlethal      Lethal             Lethal           Nonlethal          Nonlethal <br />                         b2500           purN         Nonlethal     Nonlethal      Lethal             Lethal           Nonlethal          Nonlethal <br /> Better RTMFA         b0888           trxB          Lethal        Lethal        Lethal             Lethal           Nonlethal          Nonlethal <br /> Worse RTMFA          b1136            icd          Lethal        Lethal        Lethal             Lethal           Nonlethal           Lethal <br />                         b2780           pyrG          Lethal        Lethal        Lethal             Lethal           Nonlethal           Lethal <br /> Columns labeled &#226;&#8364;&#732;&#226;&#8364;&#732;corrected&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicate RTMFA predictions additional phenotype-correction constraints (described Table S10 Supporting <br /> Material) included. <br />  <br />  <br />  <br /> (34) identify reactions knockout mutant.                  prediction. suggests ornithine transaminase cases TMFA makes better prediction, hypothe-                        rescue DargD.  argD::kan DastC double <br /> sized SL reaction active mutant pheno-                   mutant proved viable  Fig. S4), suggesting type. hypothesis tested knocking SL                     reaction isozyme rescues DargD mutant. <br /> reaction single mutant: resulting mutant                      cases TMFA worse prediction FBA, incapable growth. Conversely, TMFA makes                          developed phenotype-correction constraint pre- <br /> worse prediction, SL reaction operate pre-                  vented SL reaction operating rescuing direc- <br /> dicted direction vivo.                                                      tion  Table S10). instances, SL Finder predicted <br />    seven deletions TMFA better                         multiple reactions acted rescue pheno- <br /> prediction FBA, selected experimental vali-                    type, reaction rescued multiple phenotypes. <br /> dation: DaspC DargD. mutants exhibit robust                           example, SL Finder predicted reactions ornithine <br /> growth, removing SL reactions requires                          transaminase n-acetylornithine deacetylase acted single gene deletion. DaspC mutant, SL Finder                        rescue phenotypes DargA, DargB, identified aspartase (encoded aspA) SL reaction.                     DargC. told, 12 genes TMFA aspA::kan aspC double mutant proved viable                            incorrect nonlethal prediction associated 10 SL <br />  Fig. S2), identified studies reporting tyrB                     reactions. able identify metabolite concentration isozyme aspC (37,38). tyrB::kan DaspC double                          constraints 10 reactions  Table S10), <br /> mutant proved nonviable  Fig. S3), confirming                          implemented TMFA, resulted correct <br /> tyrB aspA rescues DaspC mutant.                        predictions 12 genes (Table 1). RTMFA, correct phenotype adding tyrB isozyme                         19 SL reactions associated 14 genes aspC model, imposing constraint pre-                      RTMFA incorrect nonlethal predictions. vents aspartase operating reverse direction. SL                    able identify concentration constraints 17 Finder predicted DargD rescued ornithine                      reactions  Table S10),  implemented transaminase, reaction argD astC                           RTMFA, resulted correct predictions 12 14 <br /> reported activity (39,40).  allowing orni-                     genes (Table 1). exception DargD noted  <br /> thine transaminase reversible results TMFA making                     correct TMFA RTMFA predictions unaffected worse predictions correcting single DargD                       additional phenotype-correction constraints. <br />  <br />                                                                                                                  Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  518                                                                                                                                                         Hamilton et al. <br />  <br />  <br />                       &#206;&#8221;b0720, citrate synthase (2.3.3.1)                                                    Taken single-gene deletion data,                                                .6                                                             suggests additional concentration measurements  <br />  <br />  <br />  <br />                                        4.1.3 <br />                                                 ac                                                            needed accurately define metabolic flux space <br />                                                oaa <br />                                                      X <br />                                                      2.3 <br />                                                         .3.1     cit <br />  <br />                                                                accoa                   icit <br />                                                                                                               predict growth phenotypes. <br />                                                                                                                  identified gene deletions TMFA <br />                                                                                                               falsely predicted lethal phenotype (Db1949, Db2500,                                                                                                               Db3919) FBA predicted nonlethal (Table 1).                                       mal                  TCA Cycle                          akg             cases, hypothesized SL reaction active                                                                                                               FBA thermodynamically infeasible TMFA.  <br />                                                                                                               cases, uncertainty free energies <br />                         biomass                fum                                succ-<span id='am-52' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span>                    included, RTMFA correct prediction                         precursors                                                              biomass        exception Db3919). RTMFA picked                                                                 succ                          precursors      additional gene deletion (Db0888) TMFA        B                    Constraining citrate lyase (4.1.3.6)                                              FBA incorrectly predicted lethal phenotype (Table 1). <br />                                                                                                                cases thermodynamic <br />                                                                                                               assignment reaction directions led inaccurate <br />                        &#226;&#710;&#8217;4 <br />                        &#226;&#710;&#8217;5 <br />                                                                                                               growth predictions, cases thermody- <br />                        &#226;&#710;&#8217;6                                                                                     namic constraints needed explain observed growth <br />         ln[cit] (mM) <br />  <br />  <br />  <br />  <br />                        &#226;&#710;&#8217;7                                                                                     phenotypes. <br />                        &#226;&#710;&#8217;8 <br />                        &#226;&#710;&#8217;9 <br />                                                                                                                  evaluated TMFA aerobic growth glycerol <br />                   &#226;&#710;&#8217;10                                                                                         anaerobic growth glucose  Table S5).                   &#226;&#710;&#8217;11                                                                                         enable aerobic growth glycerol M9 medium TMFA <br />                     &#226;&#710;&#8217;4 <br />                             &#226;&#710;&#8217;5 <br />                                  &#226;&#710;&#8217;6                                                                 &#226;&#710;&#8217;5 <br />                                                                                                          &#226;&#710;&#8217;4   required concentration constraints glucose <br />                                       &#226;&#710;&#8217;7                                                      &#226;&#710;&#8217;6 <br />              ln[oaa] (mM)                  &#226;&#710;&#8217;8 <br />                                                 &#226;&#710;&#8217;9                                &#226;&#710;&#8217;8 <br />                                                                                          &#226;&#710;&#8217;7                   case;  anaerobic growth glucose, TMFA <br />                         TMFA                         &#226;&#710;&#8217;10                     &#226;&#710;&#8217;9          ln[ac] (mM) <br />                                                                        &#226;&#710;&#8217;10 <br />                         RTMFA <br />                                                         &#226;&#710;&#8217;11     &#226;&#710;&#8217;11                                           required expanded concentration ranges slightly <br />                                                                                                               different set metabolites. performed CONGA <br /> FIGURE 3 Example reduced concentration spaces imposed pheno-                                            single-gene deletions conditions. glyc- <br /> type-correction constraints.   TMFA incorrectly predicts deletion <br />                                                                                                               erol aerobic conditions, 20 gene deletions gltA (citrate synthase, 2.3.3.1) nonlethal. gltA deleted <br /> (large X), TMFA predicts citrate lyase (4.1.3.6) synthesizes citrate                                     FBA TMFA different predictions, (cit) operating reverse. (B) correct phenotype, citrate lyase                                    better 14 worse predictions  Table S12). constrained operate forward direction. requires                                   glucose anaerobic conditions, 26 gene deletions ac, oaa, cit concentrations lie beneath shaded surface.                                        FBA TMFA different predictions.                                                                                                               performed growth phenotype screens 20 dele- <br />    example phenotype-correction constraint illus-                                                    tion strains available Keio collection  Fig. S5). <br /> trated Fig. 3. TMFA predicts deletion citrate                                                  Assuming deletion strains unavailable (cit) synthase (gltA, EC 2.3.3.1) nonlethal, citrate                                                  Keio collection involve essential genes (25), lyase (EC 4.1.3.6) rescuing growth. case citrate                                                   screens reveal TMFA makes better prediction synthesized oxaloacetate (oaa) acetate (ac),                                                         FBA cases worse prediction 18 cases <br /> instead oaa <span id='am-76' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-77' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span>acetyl-CoA</span> (accoa) (Fig. 3 .                                                     Table S13). phenotype-correction constraint        DgltA ln&#195;&#176;ac&#195;&#382; &#195;&#190; ln&#195;&#176;oaa&#195;&#382;   ln&#195;&#176;cit&#195;&#382;&lt;   2:76                                                         Thermodynamic variability analysis: ranges                                                                                                               metabolite concentration TMFA,                                                                                                 Thermodynamic variability analysis (TVA) used                                                                                                               study ranges metabolite concentrations allow <br />                              ln&#195;&#176;ac&#195;&#382; &#195;&#190; ln&#195;&#176;oaa&#195;&#382;   ln&#195;&#176;cit&#195;&#382;&lt;   7:77                                               maximal growth glucose minimal media absence <br />                                                                                                               uncertainty standard Gibbs energy formation                                                                                                                                   0 RTMFA. Fig. 3 B, constraints indicated                                                    group (Dgr G 0). Using TMFA, identified total shaded planes, concentrations half-space                                                      124  618) intracellular metabolites feasible plane satisfying constraint.                                                                    concentration range default global bounds <br />    CONGA identified total 20 double- triple-                                                    (0.001 20 mM, Table 2, Table S14 Fig. S5), <br /> deletions TMFA predicted nonlethal phenotype                                                      indicating thermodynamic constraints impose FBA lethal  Table S11). able                                                    restrictions metabolite concentrations. <br /> experimental phenotypes 14 multi-gene dele-                                                         study Bennett et al. (22) examined exponential <br /> tions, TMFA worse prediction cases.                                                         growth E. coli, reported measurements 107 <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                                                                                519 <br />  <br /> TABLE 2 Comparison model-predicted experimentally                                                                      tion range restricted thermodynamic constraints <br /> measured metabolite concentration ranges glucose aerobic                                                                  (Fig. 4 B), 57 did  e., ranges conditions maximal growth <br />                                                                                                                               global bounds). results suggest TMFA                                                                    Predicted TMFA                                          suitable framework incorporating mea- <br />                                                              Unconstrained Constrained                                  sured concentration data constraint-based models, <br />                                                              thermodynamics thermodynamics Total                              tool predict experimental concentration <br /> Experimentally  Overlap                                            57                              16            73           measurements. <br /> Measured       overlap                                          11                              23            34              second study, Ishii et al. (27), examined E. coli <br />                 data                                            426                             85            511          grown continuous culture 0.2 h 1, reported mea- <br /> Total                                                              494                             124           618 <br />                                                                                                                               surements 88 metabolites iJR904 model.  <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;Constrained&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates concentration ranges tighter global                                                formed TVA growth rate examine effect bounds (0.001&#226;&#8364;&#8220;20 mM).                                                                                                         suboptimal growth predicted metabolite concentrations. <br />                                                                                                                               general, fewer metabolites predicted metabolites iJR904 model.  predicted                                                                      constrained ranges suboptimal growth (97 <br /> centration ranges overlapped measured values                                                                       compared 124, Table S15 Fig. S7), likely 73 instances, failed overlap 34 (Table 2,                                                                        increase network flexibility associated Fig. 4, Fig. S6). 34 metabolites (Fig. 4                                                                     suboptimal growth. compared model predic- , 12 measurements did overlap predicted                                                                      tions measurements taken dataset Ishii et al. <br /> values measurements fell outside global                                                                       (27) predictions agree experi- <br /> bounds defined model. addition, 11                                                                     mental measurements (74 88 measurements, Table <br /> 12 metabolites, concentration range predicted                                                                          S16, Fig. S8, Fig. S9, Fig. S10). 68 metabo- <br /> TMFA spanned range allowed global bounds,                                                                     lites measured studies, 42 predictions agreed indicating changing global bounds likely                                                                       experimental measurements studies. <br /> resolve conflicts.  doing unlikely result tighter ranges predicted metabolite concentra- <br />                                                                                                                               Examination thermodynamic bottlenecks <br /> tions. remaining 22  34) conflicting metabo- <br /> lites measurements did fall global                                                                           Previous studies utilized thermodynamic constraints bounds, thermodynamic consistency  e., predicted concen-                                                                    identify candidate reactions regulation  trans- <br />                                                                                                                                                                           0 <br /> trations consistent experimental measurement)                                                                      formed Gibbs energy reaction (Dr G ) far zero) achieved 22 metabolites allowing uncer-                                                                     (21,23), identify thermodynamic bottlenecks tainty using RTMFA.  predicted concentration                                                                      cellular metabolism (7,23). Thermodynamic bottlenecks <br /> range metabolites RTMFA spanned                                                                       defined reactions render metabolic path- <br /> global range, indicating RTMFA unable predict                                                                      ways infeasible given known concen- metabolite concentrations great precision.                                                                      trations (41,42). term later used (7,23)                                                                                                                                                                   0 73 instances overlap measured TMFA                                                                         reactions Dr G close equilibrium. <br /> predicted values, 16 metabolites feasible concentra-                                                                    reactions feasible narrow concentration <br />  <br />                 adn                                                        B              nadp <br />                 dgsn <br />                  cytd <br />                   gsn                                                                      fum <br />                  ade <br />               dad-2                                                                        trp-L <br />              adpglc <br />                                                                                                                                                    FIGURE 4 Comparison model-predicted                                                                                            <span id='am-18' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>nadh</span> <br />                  anth <br />               ppcoa <br />                                                                                                                                                    experimentally observed metabolite concentrations <br />                 inost <br />                   aps <br />                                                                                            prpp                                                    maximal growth. dataset Bennett et al. <br />                  glcn                                                                      pep                                                     (22) contains metabolite concentrations 107 <br />              dhor-S <br />                 histd                                                                     arg-L                                                    metabolites iJR904.   Plot 34 metabo- <br />                   coa <br />                                                                                                                                                    lites experimental theoretical  <br /> Metabolite <br />  <br />  <br />  <br />  <br />                                                                              Metabolite <br />  <br />  <br />  <br />  <br />                    fad <br />                                                                                            r5p <br />               <span id='am-36' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>nadph</span> <br />              succoa                                                                                                                                centration measurements overlap. (B) Plot <br />                                                                                           ru5p-D <br />                   <span id='am-86' about='obo:IMR_0200221' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="IMP" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:id" content="IMR:0200221" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00130" datatype="xsd:string"></span>imp</span> <br />                  <span id='am-91' about='obo:IMR_0200494' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00055" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="CMP" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:id" content="IMR:0200494" datatype="xsd:string"></span>cmp</span> <br />                                                                                                                                                    16 metabolites experimental                  <span id='am-23' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-24' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>                                                                      xu5p-D <br />                   akg <br />                                                                                                                                                    theoretical concentration measurements overlap, <br />                                                                                           mal-L <br />                 succ <br />               cbasp <br />                                                                                                                                                    metabolite concentration ranges <br />                pro-L                                                                       <span id='am-96' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-97' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ctp</span>                                                     constrained thermodynamics.  B) <br />                  adp <br />               accoa                                                                        nad                                                     Circles (diamonds) Mean predicted (measured) <br />                  actp <br />                     cit                                                                    6pgc                                                    metabolite concentration . (Horizontal bars)                asp-L <br />                  g6p                                                                       atp <br />                                                                                                                                                    concentration range. Metabolite abbreviations                  g1p         Global Default Bounds <br />                    f6p       Measured Concentration                                                                                                Supporting Material. <br />                                                                                            fdp <br />                 glu-L        Predicted Concentration <br />                     0.0001     0.001       0.01        0.1     1     10                       0.0001     0.001   0.01   0.1      1     10    100 <br />                                            Concentration (mM)                                                    Concentration (mM) <br />  <br />  <br />  <br />                                                                                                                                                              Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  520                                                                                                                                       Hamilton et al. <br />  <br />                                                                                                                                                  0 <br /> range.  term &#226;&#8364;&#732;&#226;&#8364;&#732;thermodynamic bottleneck&#226;&#8364;&#8482;&#226;&#8364;&#8482; refers                              prediction. Reactions GLUPRT (Dr G &lt; 0) bottleneck space potential metabolite concentra-                              remain feasible concentration ranges spanning tions. Thermodynamic models provide quantitative                                      orders magnitude. Finally, PRPP synthetase, primary <br /> values feasible concentration range metabolites                                regulatory control point purine biosynthesis (30), associated bottleneck. TVA used study                                   free energy range far zero. <br />                                                0 <br /> ranges reaction Gibbs energies (Dr G ) allow <br /> maximal growth glucose minimal media absence <br />                          0                                                                DISCUSSION uncertainty Dgr G 0. Using TVA, identified total 168 reactions operate single direction                                  TMFA modifies flux balance analysis thermodynamic free energy range includes equilibrium                                      constraints, allowing expanded predictive capability Table S17).                                                                               constraint-based methods. TMFA ensures reactions <br />    reaction operating close equilibrium                               operate thermodynamically feasible directions, elimi- <br /> phosphoribosylaminoimidazole synthase (PRAIS), inter-                                  nates thermodynamically infeasible closed cycles, mediate step purine biosynthesis. used TVA iden-                                 accounts thermodynamic coupling reactions <br />                            0 <br /> tify range Dr G metabolite concentrations                                      network. work, demonstrate thermody- <br /> consistent cellular growth reaction metab-                              namic constraints provide guide predicting reac- <br /> olite pathway (Fig. 5). TVA shows driving                                 tion directions absence prior knowledge. force PRAIS large concentration gradient                                 systematically evaluated impacts thermody- <br /> 2-formamido-n(1)-(5-phospho-D-ribosyl)acetamidine                                         namic constraints metabolic flux distributions (fpram) 5-amino-1-(5-phospho-D-ribosyl)imidazole-4-                                   cellular growth rates, highlight importance carboxamide (air), PRAIS feasible narrow                                explicitly accounting thermodynamic coupling range fpram air concentrations. Conversely, reactions                              reactions. used TMFA make qualitative                                        0 large positive negative Dr G relatively                               quantitative predictions, validated predic- <br /> insensitive metabolite concentration, TMFA confirms                                tions variety genome-scale datasets.                                                                                           predictions generate hypotheses regarding <br />                                                                                           reaction directions thermodynamic bottlenecks.                                                     &#206;&#8221;rG&#226;&#8364;&#178; (kcal / mol) <br />                                            -30    -20        -10        0            10   instances predictions metabolite <br />                       atp <br />                           r5p                                                             concentrations lack precision accuracy,                             PRPPS <br />                     amp <br />                           prpp <br />                                                                                           case additional types data constraints incorpo- <br />                     gln-L <br />                     glu-L <br />                             GLUPRT                                                        rated TMFA. Identifying additional data                 atp + <span id='am-64' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-65' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span>gly</span> <br />                           pram <br />                             PRAGS <br />                                                                                           useful important question addressed                      adp <br />                           gar <br />                                                                                           years ahead. <br />                    10fthf <br />                       thf <br />                             GARFT                                                            work, TMFA able achieve <br />               atp + gln-L <br />                           fgam <br />                             PRFGS <br />                                                                                           physiologically realistic predictions growth rates               adp + glu-L <br />                           fpram <br />                                                                                           flux distributions absence uncertainty esti- <br />                       atp                                                                                                          0 <br />                      adp <br />                             PRAIS <br />                                                                                           mated contributions groups (Dgr G 0k,est) formation <br />                                                                                                            0 <br />               hco3 + atp air <br />                                                                                                              0 <br />                             AIRC2 <br />                                                                                           energies (Df G ), concentration measurements                      adp <br />                           5caiz                                                           ATP required presence uncertainty.                             AIRC3* <br />                                                                                           encouraging TMFA reproduce wild-type growth <br />               asp-L + atp 5aizc <br />                             PRASCS <br />                                                                                           rates minimum experimental data,                     adp <br />                           25aics                                                          additional concentration measurements necessary <br />                                 ADSL2r <br />                        fum                                                                refine growth rate predictions conditions (e.g., <br />                               aicar <br />                      10fthf <br />                                 AICART                                                    knockout mutants). observed slightly <br />                         thf <br />                               fprica                                                      different global concentration bounds necessary                                 IMPC* <br />                                                                                           support growth aerobic versus anaerobic conditions, sug- <br />                               imp <br />        Reaction         0 <br />                    v&#194;&#183;&#206;&#8221;rG&#226;&#8364;&#178; &gt; 0 <br />                   Feasible &#206;&#8221;rG&#226;&#8364;&#178; Range 0.01       0.1             1              10        gesting concentration predictions media-dependent. <br />                                                    Concentration (mM) <br />                   Feasible Concentration Range <br />                                                                                              TMFA used generate hypotheses regarding <br /> FIGURE 5 Model-predicted concentrations Dr G ranges                                                                             0             reaction directions thermodynamic bottlenecks. metabolites reactions purine biosynthesis pathway. (Circles                    example, cofactor pairs ATP/ADP appear                                                       0 diamonds) Mean predicted concentrations (Dr G ). (Horizontal bars) <br />                          0 <br />                                                                                           numerous reactions, resulting constraints  concentration (Dr G ) range. Underlined reaction abbreviations signify               centration ratio metabolites (7).            0          0 Dr G Dr G 0 opposite signs. Starred reaction abbreviations <br />                                                                                           constraining ATP concentration indicate reaction direction iJR904 model opposite                               0 <br /> shown figure, Dr G values sign reversed agree                necessary achieve physiologically realistic behavior                                                                                                                                 0 direction shown. Metabolite reaction abbreviations                    presence uncertainty Dgr G 0k,est estimates.  Supporting Material                                                          constraints metabolite ratios serve drive <br />  <br /> Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  Quantitative Assessment TMFA                                                                                                              521 <br />  <br /> reactions physiological directions. Physiological       improve solver performance different formu- <br /> action directions assigned based vitro charac-      lations (50). observed priori thermodynamic <br /> terization enzymes conditions vary                constraints reaction directions (Eqs. 14&#226;&#8364;&#8220;18) reduced <br /> significantly vivo.  envision           solution times order magnitude. Alternatively, <br /> TMFA complementing approaches (17&#226;&#8364;&#8220;19,43)                 enforcing strict thermodynamic requirements, used calculate thermodynamically feasible reaction           use thermodynamic metabolomic data directions new genome-scale models. TMFA prom-             guide, seek solution maximizes consistency <br /> ises useful tool metabolic engineering applica-         available data (e.g., allowing thermodynamic <br /> tions, identifying thermodynamic bottlenecks                  concentration constraints violated, employing <br /> engineered pathways (44) pinpointing reactions          penalty objective). study suggests metabo- reversible operation enables new routes chemical          lite concentrations remain stable response perturba- <br /> synthesis (45).                                                     tions, implying single set metabolomics data (27) <br />    observed TMFA model limited           used model variety conditions. <br /> ability predict metabolite concentrations. majority reactions iJR904 network thermodynamically favorable, relatively insen-         SUPPORTING MATERIAL <br /> sitive metabolite concentrations. Obtaining tighter              Methods, results, figures, references (51&#226;&#8364;&#8220;54) available http:// <br /> bounds predicted metabolite concentrations require           www.biophysj.org/biophysj/supplemental/S0006-3495(13)00685-1. use penalty function (46), thermodynamic objec- <br />                                                                     authors acknowledge Mink Arunrattanamook John Friel tive (20), use kinetic constraints. Recent studies        experimental contributions. group contribution method software used identified correlations metabolite concentra-          free energy calculations kindly provided Vassily Hatzimanikatis. <br /> tions physiochemical properties (47,48), including          work funded National Science Foundation KM metabolic enzymes (22). Alternatively, incorporation          CAREER grant J.L.R. (NSF 1053712) Graduate Research known metabolite concentrations enable TMFA               Fellowship J.J.H. (grant  DGE-0718123). V.D. supported predict concentrations metabolites data         Khorana Program University Wisconsin-Madison. <br /> available. light results, suggest  additional constraints, TMFA better suited validating <br /> phenotypes generating hypotheses quantitative          REFERENCES <br /> prediction metabolite concentrations.                             1. Oberhardt, M. , B. &#195;&#732;. Palsson, J.  Papin. 2009. Applications    Furthermore, results suggest additional types            genome-scale metabolic reconstructions. Mol. Syst. Biol. 5:320. <br /> data constraints needed improve TMFA&#226;&#8364;&#8482;s pre-           2. Lewis, N. E., H. Nagarajan, B. &#195;&#732;. Palsson. 2012. Constraining dictions growth phenotypes metabolite concentra-                 metabolic genotype-phenotype relationship using phylogeny                                      0                                  silico methods. Nat. Rev. Microbiol. 10:291&#226;&#8364;&#8220;305. <br /> tions uncertainties Df G 0est. recently published <br />                                              0                       3. Orth, J. D.,  Thiele, B. &#195;&#732;. Palsson. 2010. flux balance <br /> GCM provides tighter estimates Df G 0est (49),              analysis? Nat. Biotechnol. 28:245&#226;&#8364;&#8220;248. approaches successfully combined group contri-            4. Beard, D. , S. D. Liang, H. Qian. 2002. Energy balance                                                                0 <br /> bution estimates experimentally measured Df G 0                    analysis complex metabolic networks. Biophys. J. 83:79&#226;&#8364;&#8220;86. <br /> values equilibrium constants (18,21). observed            5. Qian, H., D.  Beard, S. D. Liang. 2003. Stoichiometric network inclusion additional constraints cofactor            theory nonequilibrium biochemical systems. Eur. J. Biochem. <br />                                                                         270:415&#226;&#8364;&#8220;421. <br /> centrations formation energies (ATP, NAD, NADP, ) constrains flux space (data shown).             6. Schilling, C. H., D. Letscher, B. &#195;&#732;. Palsson. 2000. Theory                                         0                               systemic <span id='am-35' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> metabolic pathways use interpreting <br /> experimental measurements Df G 0 cofactors                metabolic function pathway-oriented perspective. J. Theor. Biol. promising way improve accuracy thermodynamic                203:229&#226;&#8364;&#8220;248. <br /> models. Finally, note approach underestimates            7. Henry, C. S., L. J. Broadbelt, V. Hatzimanikatis. 2007. Thermody- <br />                         0 <br /> uncertainty Dr G 0 neglecting error associated                namics-based metabolic flux analysis. Biophys. J. 92:1792&#226;&#8364;&#8220;1805. structural groups unchanged reaction, approach          8. Schellenberger, J., N. E. Lewis, B. &#195;&#732;. Palsson. 2011. Elimination                                                 0         0             thermodynamically infeasible loops steady-state metabolic models. valid contributions Dgr G 0 Df G 0        Biophys. J. 100:544&#226;&#8364;&#8220;553. fact, linearly additive. Additional types data likely     9. Heuett, W. J., H. Qian. 2006. Combining flux energy balance necessary considering error associated                 analysis model large-scale biochemical networks. J. Bioinform. unchanged groups.                                                 Comput. Biol. 4:1227&#226;&#8364;&#8220;1243. <br />                                                              0 <br />    Associated need reduce uncertainty Df G 0est        10. Beard, D. , E. Babson, ., H. Qian. 2004. Thermodynamic  <br />                                                                         straints biochemical networks. J. Theor. Biol. 228:327&#226;&#8364;&#8220;333. need improve model run-time performance, large <br /> mixed-integer programs TMFA quite                    11. Yang, F., H. Qian, D.  Beard. 2005. Ab initio prediction                                                                         thermodynamically feasible reaction directions biochemical <br /> cumbersome. recent Master&#226;&#8364;&#8482;s thesis examines number                  network stoichiometry. Metab. Eng. 7:251&#226;&#8364;&#8220;259. thermodynamic approaches (EBA, TMFA, )                12. Alberty, R.  2003. Thermodynamics Biochemical Reactions. John <br /> theoretical practical perspective, provides insights            Wiley &amp; Sons, Hoboken, NJ. <br />  <br />                                                                                                         Biophysical Journal 105(2) 512&#226;&#8364;&#8220;522 <br />  522                                                                                                                                           Hamilton et al. <br />  <br /> 13. Goldberg, R. N., Y. B. Tewari, T. N. Bhat. 2004. Thermodynamics           34. Suthers, P. F.,  R. Zomorrodi, C. D. Maranas. 2009. Genome- <br />     enzyme-catalyzed reactions database quantitative biochem-                scale gene/reaction essentiality synthetic lethality analysis. Mol. <br />     istry. Bioinformatics. 20:2874&#226;&#8364;&#8220;2877.                                              Syst. Biol. 5:301. <br /> 14. Mavrovouniotis, M. L. 1990. Group contributions estimating stan-          35. Neidhardt, F. C., P. L. Bloch, D. F. Smith. 1974. Culture medium <br />     dard Gibbs energies formation biochemical compounds                      enterobacteria. J. Bacteriol. 119:736&#226;&#8364;&#8220;747. <br />     aqueous solution. Biotechnol. Bioeng. 36:1070&#226;&#8364;&#8220;1082.                           36. Yang, C., Q. Hua, ., K. Shimizu. 2003. Analysis Escherichia coli <br /> 15. Mavrovouniotis, M. L. 1991. Estimation standard Gibbs energy                   anaplerotic metabolism regulation mechanisms meta- <br />     changes biotransformations. J. Biol. Chem. 266:14440&#226;&#8364;&#8220;14445.                    bolic responses altered dilution rates phosphoenolpyruvate <br />                                                                                       carboxykinase knockout. Biotechnol. Bioeng. 84:129&#226;&#8364;&#8220;144. <br /> 16. Jankowski, M. D., C. S. Henry, ., V. Hatzimanikatis. 2008. Group <br />     contribution method thermodynamic analysis complex metabolic           37. Gelfand, D. H., R.  Steinberg. 1977. Escherichia coli mutants <br />     networks. Biophys. J. 95:1487&#226;&#8364;&#8220;1499.                                               deficient aspartate aromatic <span id='am-57' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-58' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-59' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> aminotransferases. <br />                                                                                       J. Bacteriol. 130:429&#226;&#8364;&#8220;440. <br /> 17. Feist,  M., C. S. Henry, ., B. &#195;&#732;. Palsson. 2007. genome-scale <br />     metabolic reconstruction Escherichia coli K-12 MG1655                38. Berg, C. M., M. D. Wang, ., L. Liu. 1988. Acquisition new meta- <br />     accounts 1260 ORFs thermodynamic information. Mol. Syst.                  bolic capabilities: multicopy suppression cloned transaminase genes <br />     Biol. 3:121.                                                                      Escherichia coli K-12. Gene. 65:195&#226;&#8364;&#8220;202. <br /> 18. Fleming, R. M. T.,  Thiele, H. P. Nasheuer. 2009. Quantitative          39. Billheimer, J. T., H. N. Carnevale, ., E. E. Jones. 1976. Ornithine <br />     assignment reaction directionality constraint-based models                  d-transaminase activity Escherichia coli: identity acetylorni- <br />     metabolism: application Escherichia coli. Biophys. Chem. 145:               thine d-transaminase. J. Bacteriol. 127:1315&#226;&#8364;&#8220;1323. <br />     47&#226;&#8364;&#8220;56.         <br /> </body></html>