<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cell. Author manuscript; available PMC 2012 January 7. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell. 2011 January 7; 144(1): 143&#226;&#8364;&#8220;156. doi:10.1016/j.cell.2010.11.052. <br />  <br />  <br />  <br />                            Phenotypic Landscape Bacterial Cell <br />  <br />                            Robert J. Nichols1,2, Saunak Sen3, Yoe Jin Choo2, Pedro Beltrao4, Matylda Zietek2, <br />                            Rachna Chaba2, Sueyoung Lee2, Krystyna M. Kazmierczak5, Karis J. Lee2,&#194;&#182;, Angela <br />                            Wong2,&#194;&#167;, Michael Shales4, Susan Lovett6, Malcolm E. Winkler5, Nevan J. Krogan4, <br />                            Athanasios Typas2,*, Carol  Gross2,7,* <br />                            1 Oral Craniofacial Sciences Graduate Program, University California San Francisco, <br />  <br />                            513 Parnassus Ave, CA-94143 <br />                            2Department Microbiology Immunology, University California San Francisco, San <br />                            Francisco, 600 16th Street, CA-94158 <br />                            3Department Epidemiology Biostatistics, University California, San Francisco, <br />                            CA-94107 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            4Department Cellular Molecular Pharmacology <span id='am-42' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-43' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; California Institute Quantitative <br />                            Biomedical Research, University California San Francisco, San Francisco, 1700 4th Street, <br />                            CA-94158 <br />                            5   Department Biology, Indiana University, Bloomington, Indiana 47405 <br />                            6Department Biology Rosenstiel Basic Medical Sciences Research Center, Brandeis <br />                            University, Waltham, Massachusetts 02454-9110 <br />                            7Department Cell Tissue Biology, University California San Francisco, San <br />                            Francisco, California 94158 <br />  <br />                            Summary <br />                                  explosion sequence information bacteria makes developing high-throughput, cost- <br />                                  effective approaches matching genes phenotypes imperative. Using E. coli proof                                  principle, combining large-scale <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> genomics quantitative fitness <br />                                  measurements provides high-quality data set rich discovery. Probing growth profiles                                  mutant library hundreds conditions parallel yielded &gt; 10,000 phenotypes allowed                                  study gene essentiality, discover leads gene function drug action, understand higher- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  order organization bacterial chromosome. highlight new information derived                                  study, including insights gene involved multiple antibiotic resistance synergy <br />                                  broadly used combinatory antibiotic therapy, trimethoprim sulfonamides. data <br />  <br />  <br />                            &#194;&#169; 2010 Elsevier  rights reserved. <br />                            * <br />                              Corresponding authors: Athanasios Typas: Phone: 415-476-1493; Fax: 415-514-4080; athanasios.typas@ucsf.edu Carol  Gross: <br />                            Phone: 415-476-1461; Fax: 415-514-4080; cgrossucsf@gmail.com . <br />                            &#194;&#182;new address: UCLA School Dentistry, 10833 Le Conte Avenue, Los Angeles, CA 90095-1762 <br />                            &#194;&#167;new address: Massachusetts General Hospital, 50 Staniford Street, Boston, MA 02114 <br />                            Contributions  RJN, NJK CAG conceived study; RJN designed research;  RJN, YJC, SL, JL AW <br />                            performed screen; RJN did data processing; RJN, SS, PB analyzed data;  RJN, CAG SL interpreted                            data; RJN, MZ performed follow work anti-folate drug story, work Streptococcus pneumoniae <br />                            conducted KMK; RJN, RC, MZ performed follow work mar story; RJN, CAG wrote                            manuscript; RJN MS prepared figures; PB, RC, MEW, KMK, SL, SS NJK edited manuscript; CAG <br />                            supervised aspects project. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Nichols et al.                                                                                               Page 2 <br />  <br />  <br />                                     set, publicly available http://ecoliwiki.net/tools/chemgen/, valuable resource                                     microbiological bioinformatic communities, provides high-confidence associations <br />                                     hundreds annotated uncharacterized genes inferences mode NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     action poorly understood drugs. <br />  <br />  <br />                               Introduction <br />                                               physical basis genes understood, associating phenotypes heritable <br />                                               unit laid foundation modern genetics. Following discovery genetic code, linking <br />                                               phenotype responsible gene remained expeditious way unravel gene <br />                                               function. explosion sequence information, balance shifted.                                               genes unknown function. capitalize burgeoning sequence bank,                                               imperative develop high-throughput technologies link genes phenotypes                                               expedite discovery gene function. particularly true prokaryotes,                                               represent major fraction sequenced genomes forefront                                               metagenomic efforts (Qin et al., 2010). <br />  <br />                                               Chemical environmental perturbations traditionally linked phenotypes                                               genotypes forward genetic screens, reverse genetic approaches                                               increasingly utilized (Barker et al., 2010). Phenotype microarrays utilize high-resolution <br />                                               readout cellular respiration evaluate fitness strain hundreds conditions <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               (Bochner, 2009). approach appropriate studying strains, difficult                                               expand genome-scale screens. pooled growth competitions, thousands strains                                               assayed single culture environment. Fitness values derived measuring strain <br />                                               abundance test relative control condition (Giaever et al., 2004; Girgis et al., 2009; <br />                                               Hillenmeyer et al., 2008; Hoon et al., 2008; Lee et al., 2005; Pan et al., 2004; Parsons et al., <br />                                               2006; Warner et al., 2010; Xu et al., 2007). approaches efficient,                                               competition strains condition makes difficult determine relative strain <br />                                               growth conditions (Girgis et al., 2009), especially strains grow slowly                                               absence perturbation (Lee et al., 2005). Arraying mutant strains solid media allows <br />                                               independent evaluation strain fitness, used low-resolution <br />                                               measurements entire libraries (Liu et al., 2010; Tamae et al., 2008) essential genes <br />                                               (Pathania et al., 2009). High-throughput genetic interaction studies, pioneered yeast <br />                                               (Schuldiner et al., 2005; Tong et al., 2001), complementary chemical genomics <br />                                               approaches. analyses quantitatively measure colony growth double mutants high- <br />                                               density format agar surfaces, led numerous successes identifying gene <br />                                               function network organization (Beltrao et al., 2010). Similar methodology                                               developed E. coli (Butland et al., 2008; Typas et al., 2008). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               use E. coli illustrate power applying high-resolution quantitative fitness <br />                                               measurements genetic interaction analysis high-throughput phenotypic analysis                                               culturable microbes. &#226;&#8364;&#339;Phenomic profiling&#226;&#8364;? provides quantitative description                                               response single gene deletions physiologically relevant stresses drug challenges. <br />                                               profiling ~4000 genes &gt;300 perturbations, identified thousands phenotypes                                               diverse suite conditionally essential genes. approach provides new insights                                               chromosome organization, functional landscape evolutionary trajectory E. coli.                                               facilitates high confidence association genes unknown function known <br />                                               function, highlighted discovery role gene involved multiple antibiotic <br />                                               resistance manuscript identification novel lipoproteins essential                                               <span id='am-62' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-63' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span>peptidoglycan</span> synthesis (Typas et al. Cell, 2010). Finally, degree various gene <br />                                               deletions alter toxic drug effects lead powerful insights regarding drug mode  <br />  <br />  <br />  <br />  <br />                                                        Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                            Page 3 <br />  <br />  <br />                                             action (Kohanski et al., 2008) demonstrate analysis generates numerous <br />                                             leads concerning drug function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Results Discussion <br />                               Phenomic Profiling E. coli K12 yields robust, high-quality dataset <br />                                             determined quantitative growth scores Keio single-gene deletion library (Baba et <br />                                             al., 2006); essential gene hypomorphs [C-terminally tandem-affinity tagged (Butland et al., <br />                                             2005) specific alleles]; small RNA/small protein knockout library (Hobbs et al., <br />                                             2010) conditions representing range stresses E. coli encounters. Mutant strains <br />                                             arrayed high-density agar plates (1536 colonies/plate) grown 324 conditions <br />                                             covering 114 unique stresses (Figs. 1A &amp; S1A, Table S1). Colony sizes analyzed                                             converted drug-gene score using approach developed quantifying genetic <br />                                             interactions  Experimental Procedures). half conditions antibiotic/ <br />                                             antimicrobial treatments (Fig. 1A). using sub-inhibitory concentration series                                             maximally inhibited growth wildtype (wt) strain &#226;&#8240;&#164;50%, able search                                             specific drug-gene interactions (Fig. S1A), reduce ability spontaneous suppressor <br />                                             mutations overtake colony. independently derived clones mutant strain <br />                                             analyzed  sRNA mutants, single isolate arrayed twice) screens                                             performed twice, enabling scores based 4-6 independent measurements. <br />                                             Correlation replicate colony size measurements high (r=0.77, Fig. 1B). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             final dataset (Table S2) comprised scores 3979 mutant strains passing <br />                                             quality control (e.g. proper normalized colony size distribution replicate reproducibility; <br />                                             Experimental Procedures). entire dataset available interactive, searchable <br />                                             format flat file E. coli wiki website  http://ecoliwiki.net/tools/chemgen/). <br />  <br />                                             entire matrix (3979 mutants &#195;&#8212; 324 conditions) subjected 2-D hierarchical <br />                                             clustering (Fig. 1C). Drugs similar effects cluster X-axis; mutants behaved <br />                                             similarly cluster Y-axis. Notably, concentrations drug, drugs                                             family similar conditions clustered did mutants genes known                                             operon, biological pathway protein complex. Zoomed insets                                             clustergram illustrate examples. Genes rfa operon (rfaG, rfaP, rfaQ, rfaB rfaI), <br />                                             encodes enzymes synthesize inner outer <span id='am-27' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-36' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span>lipopolysaccharide</span> (<span id='am-17' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span>LPS</span>) core <br />                                             strongly cluster 3/4 genes responsible synthesis sugar <br />                                             building blocks, <span id='am-11' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span>-L glycero-&#206;&#178;-D-manno heptose (rfaD, rfaE, lpcA). Importantly, <br />                                             clustering reflects shared sensitivities concentration series compounds known                                             perturb envelope integrity cell, consistent role LPS. dsbA dsbB, <br />                                             encoded different operons, cluster. DsbA/DsbB complex generates disulfide <br />                                             bridges periplasm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             response mutant strain conditions denoted &#226;&#8364;&#339;phenotypic <br />                                             signature&#226;&#8364;?. High correlation phenotypic signatures highly predictive                                             known indicators functional connection genes. Gene pairs correlation <br />                                             coefficients (r) 0.6 0.8 (p-value &lt;10&#226;&#710;&#8217;34) 100-fold enriched                                             genes sharing common operon membership, 150-fold enriched genes known <br />                                             protein interactions determined low-throughput experiments (www.ecocyc.org, Fig. <br />                                             1D). benchmarking analysis indicates phenomics dataset biologically <br />                                             meaningful. Correlated phenotypic signatures reproduce connections curated <br />                                             biological pathways (Fig. S1B). example, electron transfer components cluster tightly <br />                                             (e.g. nuo genes encoding <span id='am-37' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>NADH</span> dehydrogenase complex; Fig. 1C). clustering <br />                                             reflects high sensitivity membrane-perturbing stresses including detergents, dyes                                             metals, increased resistance aminoglycosides, agreement early studies                                             illustrate decreased aminoglycoside uptake absence fully functional electron <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                             Page 4 <br />  <br />  <br />                                             transport chain (Girgis et al., 2009;Taber et al., 1987). examples described Fig. <br />                                             1C consistent expectation highly correlated phenotypic signatures                                             biologically meaningful (r&#226;&#8240;&#165; 0.6 -0.8). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Phenomic profiling defines Responsive Conditionally-Essential Genes <br />                                             central goal study systematically evaluate impact gene deletion <br />                                             E. coli fitness diverse environments, gene deletions E. coli robust <br />                                             reported growth phenotypes 8% genes essential rich media (Baba et <br />                                             al., 2006; Yamamoto et al., 2009). used statistical method define reliable <br />                                             phenotype. Briefly, standardized interquartile range distribution scores                                             screen determined probability condition-gene interaction <br />                                             represented true phenotype using normal cumulative distribution function                                              Experimental Procedures). Using 5% probability phenotypes arose chance                                             cut (false discovery rate (FDR) &#226;&#8240;&#164;5%), 49% strains tested (1957/3979 strains; Fig. <br />                                             2A) phenotypes. refer genes &#226;&#8364;&#339;responsive genome&#226;&#8364;?.                                             &#226;&#8364;&#339;responsive genome&#226;&#8364;? work progress, limited genes removal causes <br />                                             growth phenotypes response stresses tested. Expanding stresses tested                                             readout (e.g. motility) certainly increase number (Girgis et al., 2007).                                             cumulative plot number individual phenotypes strain shows genes <br />                                             phenotypes. Multi-Stress Responsive (MSR) strains (&#226;&#8240;&#165;30 phenotypes; Table S3) <br />                                             participate cellular processes, suggesting stresses encompassed diverse <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             cellular challenges (Fig. 2B). stringent cut 5% FDR, maximum number                                             phenotypes single screen 173 (~4% strains; Fig. S2A), total number                                             phenotypes (13497) represent ~1% condition-gene pairs tested. Overall, 80%                                             phenotypes negative (gene deletion sensitive) 20% positive (gene deletion <br />                                             resistant), consistent recent genetic interaction analyses S. cerevisiae (Fiedler <br />                                             et al., 2009) S. pombe (Roguev et al., 2008). suggests removal gene <br />                                             product likely decrease enhance resistance stress (Fig. S2B). summary, <br />                                             analysis captured numerous highly specific condition-gene responses. Clearly,                                             dataset used assign phenotypes lower confidence level.  recent <br />                                             chemical genomics dataset S. cerevisiae reported phenotypes 95% gene <br />                                             deletions tested, stemming handful severe stresses (Hillenmeyer et al., <br />                                             2008). <br />  <br />                                             Conditionally essential (CE) genes essential growth particular condition. <br />                                             Deletions genes small colony sizes high confidence negative scores <br />                                             particular conditions  Experimental Procedures). identified 197 CE genes, <br />                                             comprised auxotrophs, exhibit growth minimal media, rich-media CE <br />                                             genes, exhibit growth rich media-based stress (Table S4). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Importantly, dataset 70% overlap previous study Keio Collection <br />                                             auxotrophs (Fig 2C, (Joyce et al., 2006) despite significant experimental differences (e.g. <br />                                             growth liquid vs. solid media). remaining 30% extremely sick,                                             stringent threshold used define growth. identified 23 additional <br />                                             auxotrophs specific alternative carbon/nitrogen sources tested Joyce et al. study. <br />  <br />                                             Genes essential survival natural environments likely extend                                             required laboratory growth, targets new antimicrobials (D&#226;&#8364;&#8482;Elia et al., <br />                                             2009). 116 rich-media CE genes identified (Fig. 2C) result physiologically <br />                                             relevant stresses, increase current number essential genes roughly 30%. <br />                                             Interestingly, gene products located outer cell envelope (Fig 2D),                                             selective permeability barrier gram-negative bacteria severely underrepresented <br />                                             known essential genes (Fig. 2D). stresses generating CE phenotypes                                             natural environment E. coli, e.g. bile salts (Table S5), indicating genes <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                            Page 5 <br />  <br />  <br />                                             likely indispensable E. coli survive vivo. Similarly, using largest <br />                                             metagenomic dataset <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span>, Qin et al. reported envelope-specific functions,                                             adhesion, commonly required life gut (Qin et al., 2010). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Phenomic profiling helps assign function uncharacterized genes <br />                                             key motivation study provide phenotypes mutants genes                                             functional annotation. Using recently assembled compendium &#226;&#8364;&#339;orphan genes&#226;&#8364;?                                             E. coli (Hu et al., 2009), fraction mutant orphan genes phenotypes                                             close annotated genes (Fig. 3A), tend fewer phenotypes, <br />                                             indicating power phenomic analysis identifying phenotypes. Importantly,                                             phenotypic profiles &gt;25% orphan genes correlate strongly annotated <br />                                             genes (r &#226;&#8240;&#165; 0.5; Fig. 3B &amp; Table S6), providing high confidence leads (p-value &lt;10&#226;&#710;&#8217;22)                                             discovery function. orphans tied wide variety cellular processes <br />                                             (Fig. 3C), dataset broad utility. <br />  <br />                                             small fraction orphan gene knockouts phenotypes. annotated genes <br />                                             responsible phenotypes broadly distributed bacteria,                                             responsive orphans tend narrowly distributed (Fig. 3D). result suggests                                             evolutionary conservation reliable indicator importance orphan gene                                             organism, annotating solely homology limitations. orphans <br />                                             evolved fulfill important specialized function required niche NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             organism. support idea, multi-responsive orphan identified study <br />                                             (lpoB) restricted enterobacteria regulates peptidoglycan synthesis, conserved <br />                                             process ubiquitous bacteria (Typas et al.; Cell, 2010). <br />  <br />                               Using phenotypic signatures identify gene function <br />                                             correlated phenotypic signatures (Fig. 1C&amp;D; Typas et al.; Cell, 2010)                                             anticorrelated phenotypic signatures functional significance. example,                                             phenotypic signatures deletions transcriptional repressor important target genes <br />                                             likely anticorrelated. marR&#226;&#710;&#8217; marB&#226;&#710;&#8217; highly anticorrelated <br />                                             acrB&#226;&#710;&#8217;, marR&#226;&#710;&#8217; marB&#226;&#710;&#8217; highly correlated (Fig. 4A). marB gene                                             unknown function multiple antibiotic resistance operon (marRAB),                                             includes operon repressor, marR, activator, marA. MarA activates genes <br />                                             involved antibiotic resistance, importantly acrAB, encoding major antibiotic <br />                                             efflux pump E. coli (Fig. 4B; (Martin Rosner, 2002). explored MarB, <br />                                             like MarR, repressed MarA. inherent problems high-throughput <br />                                             collections (suppressors, gene duplications, cross-contamination), apply stringent <br />                                             quality control procedures follow investigations including PCR-validation Keio <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             isolates, verification retransduced strains maintain phenotype. mar                                             hotspot adaptive mutations, sequenced entire operon promoter region                                             single deletions double mutants constructed. <br />  <br />                                             Deletion marB marR resulted higher MarA levels, double marRB <br />                                             mutant showed additive effects MarA transcript level (Fig. 4C) protein level (data <br />                                             shown). effects observed Kan-marked clean deletions (Kan <br />                                             cassette excised, leaving 82nt scar). &#206;&#8221;marR strain exhibits ~2X increase                                             MarA transcript levels marR::kan (data shown), arguing small polar effect                                             cassette. marB::kan &#206;&#8221;marB exhibit 2-fold increase MarA levels <br />                                             (data shown). data suggest MarB represses MarA independently MarR. <br />                                             MarB does signature DNA-binding protein, suggesting acts post- <br />                                             transcriptionally. MarA level controlled Lon protease (Griffith et al., 2004),                                             lon&#226;&#710;&#8217; marB&#226;&#710;&#8217; effects additive, indicating MarB does function Lon <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                            Page 6 <br />  <br />  <br />                                             (data shown). MarA proposed scan activation sites bound RNA <br />                                             polymerase; direct binding partner, MarB disrupt complex formation. <br />                                             Aternatively, MarB function periplasm. MarB predicted periplasmic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             signal sequence, titrate activating ligand mar (e.g. salicylate). <br />  <br />                                             mar highly studied (~200 primary publications; Pubmed), screen provided <br />                                             lead MarB function. MarA targets approximately 40 genes,                                             regulated SoxS Rob activators, similar DNA-binding motifs                                             MarA (Martin et al., 2008; Martin Rosner, 2002). rules engagement poorly <br />                                             understood, activator responds different environmental cues overexpression <br />                                             leads distinct phenotypes (Warner Levy, 2010). likely tight control <br />                                             activator impacts final gene expression output, crucial cellular <br />                                             proliferation. MarB important player fine-tuning expression MarA, <br />                                             especially conserved member mar operon, recently <br />                                             emerged selected enterobacteria. Strong evidence importance mar operon <br />                                             regulation organisms mar hotspot mutations conferring higher drug <br />                                             resistance E. coli (Nicoloff et al., 2007; Nicoloff et al., 2006). <br />  <br />                               Phenomic profiling reveals metabolic network behaviors anti-folate drug stress <br />                                             Tetrahydrofolate (THF) methyl/formylated derivatives key molecules                                             kingdoms life carbon metabolism. THF used synthesize <span id='am-73' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-74' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>glycine</span>, methionine, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             purines <span id='am-49' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="rdfs:label" content="dTTP" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span>dTTP</span>, process leads recycling THF species THF                                             dihydrofolate (DHF) (Fig. 5A). bacterial THF biosynthesis pathway targeted                                             drugs: Sulfonamides (Sulfa) target FolP, Trimethoprim (TMP) targets FolA (Fig. 5A). <br />                                             Dual inhibition Sulfa TMP strongly synergistic, drugs                                             exclusively administered combination treatment ear, urinary tract                                             bronchial infections. Despite extensive clinical use years laboratory investigation,                                             lack complete mechanistic understanding drugs strongly synergistic.                                             network feature identified phenomic profiling contribute synergy. <br />  <br />                                             drug classes major phenotypic differences. Sulfa TMP <br />                                             treatments highly correlated class (r=0.57 Sulfa; 0.67 TMP),                                             poorly correlated (r=0.15 +/&#226;&#710;&#8217; 0.04), just slightly correlation <br />                                             observed screens (r=0.025 +/&#226;&#710;&#8217; 0.12).  subinhibitory TMP Sulfa <br />                                             treatments fundamentally different effects cell, partially block <br />                                             THF biosynthesis. Importantly, removing enzymes acting directly downstream THF <br />                                             production resulted opposite drug sensitivities: serine hydroxymethyltransferase <br />                                             mutant (glyA::kan) sensitive TMP; conversely, <span id='am-85' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-86' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span> cleavage (GCV) mutants <br />                                             (gcvP::kan, gcvH::kan, gcvT::kan) sensitive Sulfa (Fig 5B). mutant results <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             reproduced liquid culture (Fig. 5D), glyA&#226;&#710;&#8217; TMP sensitivity manifested                                             growth rate phenotype (left panel), gcvP&#226;&#710;&#8217; sulfamethizole (SMT) sensitivity registered <br />                                             low stationary phase density (right panel). <br />  <br />                                             GlyA GCV lie opposite sides branched pathway converts THF 5,10- <br />                                             methylene THF (5,10-mTHF; Fig 5A). glyA gcv mutants exhibit synthetic lethality, <br />                                             routes production essential metabolite (Fig. 5C). simple <br />                                             explanation differential responses glyA&#226;&#710;&#8217; gcvP&#226;&#710;&#8217; 5,10-mTHF                                             predominantly produced different branches drug treatment. corollary                                             combination drug treatment inhibits branches, resulting synergistic limitation                                             5,10-mTHF, pools THF depleted. support idea, despite                                             increased sensitivity glyA&#226;&#710;&#8217; gcvP&#226;&#710;&#8217; single drugs, strains grew poorly <br />                                             wt drug combination (Fig. 5D).  genetically eliminating branch                                             pathway reduced did eliminate synergy. downstream biosynthetic reactions <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                               Page 7 <br />  <br />  <br />                                             differentially affected TMP Sulfa (Fig. S3A), currently testing <br />                                             partially account residual synergy. Streptococcus pneumoniae lacks                                             GCV exhibits significantly drug synergy E. coli different growth <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             conditions (Fig. 5E &amp; data shown). performed comparison using concentrations <br />                                             TMP SMT caused relative growth defect species (Fig. 5E). <br />                                             data support hypothesis simultaneous inhibition branched <br />                                             pathway production 5,10-mTHF contributes observed anti-folate synergy E. <br />                                             coli. <br />  <br />                                             data indicate differential effects TMP Sulfa GlyA GCV <br />                                             result differential inhibition expression activity, intrinsic properties                                             enzyme. favor idea differential metabolite accumulation subsequent feed- <br />                                             forward enzymatic regulation make contribution distinct cellular responses                                             drugs. Recent metabolomic flux analyses indicate high doses TMP lead                                             accumulation depletion select metabolites, protein-level regulation                                             portions network (Kwon et al., 2010; Kwon et al., 2008). comparable <br />                                             analysis performed Sulfa drugs, deletion predicted 5-formyl-THF <br />                                             cycloligase, ygfA, likely degrades 5-formyl-THF (Jeanguenin et al., 2010), clusters <br />                                             tightly gcv mutants, exhibits sensitivity Sulfa drugs (Fig. S3A). 5- <br />                                             formyl-THF degradation critical Sulfa stress suggests differential <br />                                             accumulation  requirement) THF species Sulfa TMP treatments. 5-formyl- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             THF known inhibitor enzymes THF network organisms (Field <br />                                             et al., 2006; Stover Schirch, 1993), act effective protein-level regulator. <br />                                             Similarly, strain lacking predicted alanine racemase, yggS, sensitive Sulfa; D- <br />                                             alanine known inactivate GlyA (Schirch et al., 1985), yggS&#226;&#710;&#8217; glyA&#226;&#710;&#8217; form                                             synthetic lethal pair (Fig. S3B).  different cellular responses drugs                                             metabolite-based enzymatic regulation. extension synergy                                             combination therapy rest primarily complementary inhibition different  <br />                                             carbon biosynthesis reactions, recycling THFs. model allow                                             synergy expected additive limitation THF production. <br />  <br />                                             summary, results illustrate power phenomic profiling yield insights drug <br />                                             action ability networks view provide new paradigms analysis drug <br />                                             interaction mechanisms, facilitate hypothesis-driven research drug interactions <br />                                             (Bollenbach et al., 2009). type analysis generally useful predicting drug <br />                                             synergies, explaining variable drug-drug interactions species. <br />  <br />                               Phenomic profiling gives insights genomic organization <br />                                             E. coli genome encoded single, circular chromosome, single origin NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             replication, oriC. Essential genes biased plus (+) strand, transcription <br />                                             proceeds direction DNA replication. avert head collisions <br />                                             RNA DNA polymerases result aborted transcripts, truncated,                                             frame-shifted proteins (Rocha Danchin, 2003). responsive CE <br />                                             genes, important optimal growth organism, + strand bias (Fig. <br />                                             6A).  weighted responsive genome (responsive genes weighted number                                             phenotypes identified) heavily biased + strand, indicating great selective pressure                                             place genes important rapid growth + strand. Conversely, non-responsive <br />                                             genome biased minus (&#226;&#710;&#8217;) strand. approaches expand incorporate <br />                                             additional phenotypic readouts important cells reduced division DNA <br />                                             replication (e.g. biofilm formation), + strand bias responsive genes presumably                                             reduced. <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                             Page 8 <br />  <br />  <br />                                             chromosome massively compacted cell create nucleoid, thought <br />                                             contribute significantly organization gene expression (Travers                                             Muskhelishvili, 2005; Vora et al., 2009). Chromosomal loci spatial addresses NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             cell, corresponding closely chromosomal position (Toro Shapiro, 2010). <br />                                             Additionally, highly expressed genes associated transcription translation                                             located near origin replication (oriC), presumably benefit &#226;&#8364;&#339;gene dosage&#226;&#8364;? <br />                                             effect created rapidly growing cells initiate multiple rounds DNA replication                                             division (Couturier Rocha, 2006). Projecting spatial distribution responsive <br />                                             genes circular chromosome (Fig. 6B, black trace) provides snapshot                                             E. coli genome functional perspective. projection based 100kb sliding <br />                                             window captures organization operon level (Fig. S4A; Experimental <br />                                             Procedures). pattern alternating peaks valleys clearly evident, indicating                                             responsive genes cluster spatially large chromosomal regions separated regions <br />                                             generally devoid responsive genes. &#226;&#8364;&#339;Valleys&#226;&#8364;? comprised spatially separated <br />                                             operons transcribed different strands, indicating low responsiveness                                             regional characteristic artifact large non-responsive operons.                                             finding clustering operon level accord studies showing                                             gene expression broadly correlated certain regions chromosome (Carpentier <br />                                             et al., 2005; Jeong et al., 2004). <br />  <br />                                             responsive genome enriched oriC, highest concentration <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             responsive genes (Fig. 6B, black trace). area enriched responsive <br />                                             genes (Fig. S4B), conditionally essential (CE) genes (Fig. 6B, red trace), providing <br />                                             strong support idea E. coli chromosome tends store genes high <br />                                             functional importance near oriC. contrast, terminus region relatively devoid                                             responsive genes (Fig 6B, black trace), paucity broadly conserved genes (Fig 6C, <br />                                             red trace) corresponding enrichment genes restricted &#206;&#179;-proteobacteria (Fig 6C, <br />                                             blue trace). postulate terminus region contains newly acquired genes                                             fully integrate cellular network, tend lack phenotypes.                                             enable cells avoid unnecessarily high expression genes consequence                                             gene-dosage effect. result prove true bacterial species, point                                             general organizing principle circular chromosomes. <br />  <br />                               Phenomic Profiling Describes Drug Action <br />                                             &#226;&#8364;&#339;Drug-centric&#226;&#8364;? analyses complex &#226;&#8364;&#339;gene-centric&#226;&#8364;? analyses. genes <br />                                             participate single biological process, parameters required                                             drug action: uptake, primary/secondary targets, efflux.  pairwise relationships <br />                                             drugs complex genes. example, drugs                                             cluster based drug uptake, primary targets differ. addition, drug <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             signatures order magnitude larger gene signatures (3979 vs. 324). reduce <br />                                             complexity drug signatures, calculated Drug-Gene Ontology   scores,                                             represent probability given group specifically interacts given drug  e. <br />                                             number phenotypes associated genes group vs. entire dataset). <br />                                             used Drug scores explore drug mode action network-based <br />                                             clustering strategy  Experimental Procedures). position drugs network (Fig. <br />                                             7A) based magnitude Drug-Gene Ontology   scores (gray)                                             Drug-Drug correlations (yellow). 719 significant Drug interactions (p-value &#226;&#8240;&#164; <br />                                             10&#226;&#710;&#8217;3), include 64 drugs 218 groups, 657 negative 62                                             positive.  disrupting linked biological process likely increase drug <br />                                             sensitivity (Table S7). Drug-Drug correlations increased resolution network                                             captured drug similarities escaped Drug analysis. <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                             Page 9 <br />  <br />  <br />                                             drugs cellular target tend cluster. example, drugs <br />                                             targeting DNA (orange) fall lower right, targeting THF-biosynthesis (light <br />                                             green) fall edge targeting peptidoglycan (PG; purple) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             predominantly cluster upper left. Interestingly, &#206;&#178;-lactams cover center PG <br />                                             cluster, drugs targeting steps PG synthesis located periphery.                                             correlation coefficients &#206;&#178;-lactams reveal similarity phenotypic <br />                                             signatures related respective primary target Penicillin Binding Proteins (Fig. S5A). <br />                                             Interestingly, known synergistic double drug combinations (TMP/sulfonamides                                             mecillinam/cefsulodin) occupy spaces distinct individual drug, arguing                                             combination elicits different cellular response individual drugs.                                             interesting determine holds true antagonistic neutral interactions                                             combinations elicit responses closer drugs. <br />  <br />                                             Importantly, specific Drug interactions suggest hypotheses mechanism action <br />                                             studied drugs. Quinolones inhibit DNA gyrase trapping quinolone <br />                                             adduct, mechanism resolution poorly understood (Drlica et al., 2008).                                             category, &#226;&#8364;&#339;cellular DNA catabolic process&#226;&#8364;? (xseAB) selectively specifically (p- <br />                                             value=10&#226;&#710;&#8217;6) interacted negatively quinolones screened (Fig 7B &amp; S5B), <br />                                             expanding previous report xseAB mutant sensitivity fluoroquinolones (Tamae et <br />                                             al., 2008). suggest XseAB (exonuclease VII) enzyme cleaves quinolone- <br />                                             bound DNA gyrase DNA allow repair proceed, possibility currently <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             exploring. <br />  <br />                                             drug network provides clues mode action poorly described drugs,  <br />                                             conversely suggests additional factors required explain action drugs. <br />                                             Nitrofurantoin (NTF; Fig. 7B) reported multi-faceted impact cells (McOsker <br />                                             Fitzpatrick, 1994;Tu McCalla, 1975), data suggests cytotoxic effects <br />                                             reflect DNA damage, causes lesions requiring <span id='am-54' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-55' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-57' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span>-excision repair (NER)                                             activates SOS response. NTF DNA-targeting drug requiring NER,                                             double-strand break repair, suggesting primary toxic lesion associated                                             replication fork (Fig. S5B). Additionally, network analysis validates idea                                             indolicidin, neutrophil antimicrobial <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span>, mediates effects compromising                                             inner membrane permeability E. coli manner similar proton motive force <br />                                             uncoupler, CCCP (Falla et al., 1996). Finally, phleomycin bleomycin cluster                                             DNA response drugs, suggesting broader cellular impact (Hecht, 2000;Yeh et al., <br />                                             2006), inducing DNA scissions (Giloni et al., 1981). insights suggest                                             phenomic dataset rich source discovery drug function interrelationships. <br />  <br />                               Perspectives <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             pace exploding sequence information, cost-effective, high-throughput <br />                                             phenotyping technologies developed. phenomic profiling E. <br />                                             coli fulfills goal. dataset great utility identifying function orphan <br />                                             genes. cases (marB, lpoA, lpoB) investigated study based                                             dataset (Typas et al., Cell, 2010), actively pursuing functional discovery                                             numerous (&gt;20) orphan genes, annotated genes previously unsuspected roles <br />                                             collaboration  &gt;25% orphan genes highly correlated                                             annotated gene (r&#226;&#8240;&#165; 0.5), dataset provides rapid method function discovery. <br />  <br />                                             important finding responsive orphan genes tend narrowly <br />                                             distributed bacteria. Interestingly, results mirror initial observations human <br />                                             microbiome studies. studies   roughly half functions encoded                                             minimal gut metagenome (ubiquitously present 124 individuals screened)  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                                               Page 10 <br />  <br />  <br />                                             unknown limited evolutionary conservation (Qin et al., 2010); b) 4 pan- <br />                                             genome species analyzed, vast majority non-common genes unknown <br />                                             function (~70%) unique family members functions core gene set <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             (Nelson et al., 2010). probably species-specific additions conserved <br />                                             biological processes pan-genome. studies argue                                             computational methods based gene conservation fail, large-scale phenomic analyses                                             second tier assigning function. make approach reality, low cost methods <br />                                             developing deletion libraries developed (Goodman et al., 2009). Single-gene <br />                                             deletion ordered libraries currently available handful organisms [(Cameron <br />                                             et al., 2008; Berardinis et al., 2008; Gallagher et al., 2007; Goodman et al., 2009; Kim et <br />                                             al., 2010; Liu et al., 2008; Noble et al., 2010) references (Barker et al., 2010)],                                             advances transposon mutagenesis make feasible create ordered mutant libraries                                             organisms. E. coli, expansion work rest ability assess additional <br />                                             phenotypes deeper exploration phenotypic space. greatest potential resides                                             intersection screening diverse stresses incorporating additional cellular <br />                                             readouts. Colorimetric readouts enable measurement transcriptional activity                                             biofilm formation solid agar surfaces, represent immediate potential advance                                             phenomic profiling. High-throughput microscopy provide new avenue                                             approaches (Werner et al., 2009). <br />  <br />                                             dataset provides information substantial collection antibiotics/antimicrobial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             compounds cover broad spectrum drug targets, structural classes drug <br />                                             generations, providing platform future studies focused natural products                                             antimicrobials unknown targets. dataset provide platform studying <br />                                             mechanism drug interactions (Yeh et al., 2009), shown case                                             sulfonamides TMP. Understanding mechanism underlying known drug interactions <br />                                             help predict novel interactions manipulate existing drug combinations                                             increase effectiveness clinic. <br />  <br />                                             summary, generated valuable resource microbiologists studying wide <br />                                             range biology, demonstrated numerous diverse applications dataset                                             infer information gene drug function. comprehensive prokaryotic <br />                                             chemical genomic study date (3979 strains &#195;&#8212; 324 conditions), dataset serve                                             base future studies aim increase information resolution gene <br />                                             drug fronts. hope usefulness resource trigger analogous studies <br />                                             organisms, bringing step nearer closing gene sequence-function gap. <br />  <br />                               Experimental procedures <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Experimental procedures partially elaborated text figure legends, fully <br />                                             explicated supplementary material. <br />  <br />                               Supplementary Material <br />                                             Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                             thank R. Kishony, J. Weissman,  Hochschild RG Martin critically reading manuscript; J. Hu                                             P. Thomas hosting data E. coli Wiki; C. Raetz CHIR-090 M. Gottesman Bicyclomycin; H. <br />                                             Mori Keio Collection; G. Storz sharing sRNA deletion library prior publication; J.Greenblatt                                              Emili SPA-tagged alleles; W. Margolin, R. Misra, T. Silhavy B. Palsson mutants; T. Baker R. <br />                                             Sauer controllable degradation plasmids. work supported NIH R01 GM085697 &amp; ARRA <br />                                             GM085697-01S1 CAG NJK; NIH R01 GM036278 CAG, NIH K99GM092984  NIH AI060744                                             MEW, NIH GM078338 SS; NIH F31 DE020206-01 NIH T32 DE007306 (RJN support); European <br />  <br />  <br />                                                       Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                                      Page 11 <br />  <br />                                             Molecular Biology Organization long-term fellowship  ; Human Frontier Science Program Long-Term <br />                                             Postdoctoral Fellowship  PB). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               References <br />                                             Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, Baba M, Datsenko KA, Tomita M, Wanner BL, <br />                                                Mori H. Construction Escherichia coli K-12 frame, single-gene knockout mutants: Keio <br />                                                collection. Mol Syst Biol. 2006; 2:0008. 2006. [PubMed: 16738554] <br />                                             Barker CA, Farha MA, Brown ED. Chemical genomic approaches study model microbes. Chem <br />                                                Biol. 2010; 17:624&#226;&#8364;&#8220;632. [PubMed: 20609412] <br />                                             Beltrao P, Cagney G, Krogan NJ. Quantitative genetic interactions reveal biological modularity. Cell. <br />                                                2010; 141:739&#226;&#8364;&#8220;745. [PubMed: 20510918] <br />                                             Bochner BR. Global phenotypic characterization bacteria. FEMS Microbiol Rev. 2009; 33:191&#226;&#8364;&#8220;205. <br />                                                [PubMed: 19054113] <br />                                             Bollenbach T, Quan S, Chait R, Kishony R. Nonoptimal microbial response antibiotics underlies <br />                                                suppressive drug interactions. Cell. 2009; 139:707&#226;&#8364;&#8220;718. [PubMed: 19914165] <br />                                             Butland G, Babu M, Diaz-Mejia JJ, Bohdana F, Phanse S, Gold B, Yang W, Li J, Gagarinova AG, <br />                                                Pogoutse O, et al. eSGA: E. coli synthetic genetic array analysis. Nat Methods. 2008; 5:789&#226;&#8364;&#8220;795. <br />                                                [PubMed: 18677321] <br />                                             Butland G, Peregrin-Alvarez JM, Li J, Yang W, Yang X, Canadien V, Starostine  Richards D, <br />                                                Beattie B, Krogan N, et al. Interaction network containing conserved essential protein <br />                                                complexes Escherichia coli. Nature. 2005; 433:531&#226;&#8364;&#8220;537. [PubMed: 15690043] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Cameron  Urbach JM, Mekalanos JJ. defined transposon mutant library use identifying <br />                                                motility genes Vibrio cholerae. Proc Natl Acad Sci U S  2008; 105:8736&#226;&#8364;&#8220;8741. [PubMed: <br />                                                18574146] <br />                                             Carpentier  Torresani B, Grossmann  Henaut  Decoding nucleoid organisation Bacillus <br />                                                subtilis Escherichia coli gene expression data. BMC Genomics. 2005; 6:84. [PubMed: <br />                                                15938745] <br />                                             Couturier E, Rocha EP. Replication-associated gene dosage effects shape genomes fast-growing <br />                                                 bacteria transcription translation genes. Mol Microbiol. 2006; 59:1506&#226;&#8364;&#8220;1518. <br />                                                 [PubMed: 16468991] <br />                                             D&#226;&#8364;&#8482;Elia MA, Pereira MP, Brown ED. essential genes really essential? Trends Microbiol. 2009; <br />                                                 17:433&#226;&#8364;&#8220;438. [PubMed: 19765999] <br />                                             Berardinis V, Vallenet D, Castelli V, Besnard M, Pinet  Cruaud C, Samair S, Lechaplais C, <br />                                                 Gyapay G, Richez C, et al. complete collection single-gene deletion mutants Acinetobacter <br />                                                 baylyi ADP1. Mol Syst Biol. 2008; 4:174. [PubMed: 18319726] <br />                                             Drlica K, Malik M, Kerns RJ, Zhao X. Quinolone-mediated bacterial death. Antimicrob Agents <br />                                                 Chemother. 2008; 52:385&#226;&#8364;&#8220;392. [PubMed: 17724149] <br />                                             Falla TJ, Karunaratne DN, Hancock  Mode action antimicrobial peptide indolicidin. J <br />                                                 Biol Chem. 1996; 271:19298&#226;&#8364;&#8220;19303. [PubMed: 8702613] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Fiedler D, Braberg H, Mehta M, Chechik G, Cagney G, Mukherjee P, Silva AC, Shales M, Collins SR, <br />                                                 van Wageningen S, et al. Functional organization S. cerevisiae phosphorylation network. <br />                                                 Cell. 2009; 136:952&#226;&#8364;&#8220;963. [PubMed: 19269370] <br />                                             Field MS, Szebenyi DM, Stover PJ. Regulation novo purine biosynthesis                                                 methenyltetrahydrofolate synthetase neuroblastoma. J Biol Chem. 2006; 281:4215&#226;&#8364;&#8220;4221. <br />                                                 [PubMed: 16365037] <br />                                             Gallagher LA, Ramage E, Jacobs MA, Kaul R, Brittnacher M, Manoil C. comprehensive transposon <br />                                                 mutant library Francisella novicida, bioweapon surrogate. Proc Natl Acad Sci U S  2007; <br />                                                 104:1009&#226;&#8364;&#8220;1014. [PubMed: 17215359] <br />                                             Giaever G, Flaherty P, Kumm J, Proctor M, Nislow C, Jaramillo DF, Chu  Jordan MI, Arkin AP, <br />                                                 Davis RW. Chemogenomic profiling: identifying functional interactions small molecules                                                 yeast. Proc Natl Acad Sci U S  2004; 101:793&#226;&#8364;&#8220;798. [PubMed: 14718668] <br />                                             Giloni L, Takeshita M, Johnson F, Iden C, Grollman AP. Bleomycin-induced strand-scission DNA. <br />                                                 Mechanism deoxyribose cleavage. J Biol Chem. 1981; 256:8608&#226;&#8364;&#8220;8615. [PubMed: 6167580] <br />  <br />  <br />  <br />                                                      Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                                   Page 12 <br />  <br />  <br />                                             Girgis HS, Hottes AK, Tavazoie S. Genetic architecture intrinsic antibiotic susceptibility. PLoS <br />                                                  2009; 4:e5629. [PubMed: 19462005] <br />                                             Girgis HS, Liu Y, Ryu WS, Tavazoie S. comprehensive genetic characterization bacterial <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 motility. PLoS Genet. 2007; 3:1644&#226;&#8364;&#8220;1660. [PubMed: 17941710] <br />                                             Goodman AL, McNulty NP, Zhao Y, Leip D, Mitra RD, Lozupone CA, Knight R, Gordon JI. <br />                                                 Identifying genetic determinants needed establish human gut symbiont habitat. Cell Host <br />                                                 Microbe. 2009; 6:279&#226;&#8364;&#8220;289. [PubMed: 19748469] <br />                                             Griffith KL, Shah IM, Wolf Jr. Proteolytic degradation Escherichia coli transcription activators <br />                                                 SoxS MarA mechanism reversing induction superoxide (SoxRS)                                                 multiple antibiotic resistance (Mar) regulons. Mol Microbiol. 2004; 51:1801&#226;&#8364;&#8220;1816. [PubMed: <br />                                                 15009903] <br />                                             Hecht SM. Bleomycin: new perspectives mechanism action. J Nat Prod. 2000; 63:158&#226;&#8364;&#8220;168. <br />                                                 [PubMed: 10650103] <br />                                             Hillenmeyer  Fung E, Wildenhain J, Pierce SE, Hoon S, Lee W, Proctor M, St Onge RP, Tyers M, <br />                                                 Koller D, et al. chemical genomic portrait yeast: uncovering phenotype genes. <br />                                                 Science. 2008; 320:362&#226;&#8364;&#8220;365. [PubMed: 18420932] <br />                                             Hobbs EC, Astarita JL, Storz G. Small RNAs small proteins involved resistance cell <br />                                                 envelope stress acid shock Escherichia coli: analysis bar-coded mutant collection. J <br />                                                 Bacteriol. 2010; 192:59&#226;&#8364;&#8220;67. [PubMed: 19734312] <br />                                             Hoon S, Smith  Wallace IM, Suresh S, Miranda M, Fung E, Proctor M, Shokat KM, Zhang C, <br />                                                 Davis RW, et al. integrated platform genomic assays reveals small-molecule bioactivities. <br />                                                 Nat Chem Biol. 2008; 4:498&#226;&#8364;&#8220;506. [PubMed: 18622389] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Hu P, Janga SC, Babu M, Diaz-Mejia JJ, Butland G, Yang W, Pogoutse O, Guo X, Phanse S, Wong P, <br />                                                 et al. Global functional atlas Escherichia coli encompassing previously uncharacterized <br />                                                 proteins. PLoS Biol. 2009; 7:e96. [PubMed: 19402753] <br />                                             Jeanguenin L, Lara-Nunez  Pribat  Mageroy M. Hamner, Gregory JF 3rd, Rice KC, Crecy- <br />                                                 Lagard V, Hanson AD. Moonlighting glutamate formiminotransferases functionally replace 5- <br />                                                 formyltetrahydrofolate cycloligase. J Biol Chem. 2010 <br />                                             Jeong KS, Ahn J, Khodursky AB. Spatial patterns transcriptional activity chromosome                                                 Escherichia coli. Genome Biol. 2004; 5:R86. [PubMed: 15535862] <br />                                             Joyce AR, Reed JL, White  Edwards R, Osterman  Baba T, Mori H, Lesely SA, Palsson BO, <br />                                                 Agarwalla S. Experimental computational assessment conditionally essential genes                                                 Escherichia coli. J Bacteriol. 2006; 188:8259&#226;&#8364;&#8220;8271. [PubMed: 17012394] <br />                                             Kim DU, Hayles J, Kim D, Wood V, Park HO, Won M, Yoo HS, Duhig T, Nam M, Palmer G, et al. <br />                                                 Analysis genome-wide set gene deletions fission yeast Schizosaccharomyces pombe. <br />                                                 Nat Biotechnol. 2010; 28:617&#226;&#8364;&#8220;623. [PubMed: 20473289] <br />                                             Kohanski MA, Dwyer DJ, Wierzbowski J, Cottarel G, Collins JJ. Mistranslation membrane proteins <br />                                                 component activation trigger antibiotic-mediated cell death. Cell. 2008; 135:679&#226;&#8364;&#8220; <br />                                                 690. [PubMed: 19013277] <br />                                             Kwon YK, Higgins MB, Rabinowitz JD. Antifolate-induced depletion intracellular glycine NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 purines inhibits thymineless death E. coli. ACS Chem Biol. 2010; 5:787&#226;&#8364;&#8220;795. [PubMed: <br />                                                 20553049] <br />                                             Kwon YK, Lu W, Melamud E, Khanam N, Bognar  Rabinowitz JD. domino effect antifolate <br />                                                 drug action Escherichia coli. Nat Chem Biol. 2008; 4:602&#226;&#8364;&#8220;608. [PubMed: 18724364] <br />                                             Lee W, St Onge RP, Proctor M, Flaherty P, Jordan MI, Arkin AP, Davis RW, Nislow C, Giaever G. <br />                                                 Genome-wide requirements resistance functionally distinct DNA-damaging agents. PLoS <br />                                                 Genet. 2005; 1:e24. [PubMed: 16121259] <br />                                             Liu  Tran L, Becket E, Lee K, Chinn L, Park E, Tran K, Miller JH. Antibiotic sensitivity profiles <br />                                                 determined Escherichia coli gene knockout collection: generating antibiotic bar code. <br />                                                 Antimicrob Agents Chemother. 2010; 54:1393&#226;&#8364;&#8220;1403. [PubMed: 20065048] <br />                                             Liu OW, Chun CD, Chow ED, Chen C, Madhani HD, Noble SM. Systematic genetic analysis                                                 virulence human fungal pathogen Cryptococcus neoformans. Cell. 2008; 135:174&#226;&#8364;&#8220;188. <br />                                                 [PubMed: 18854164] <br />  <br />  <br />  <br />  <br />                                                      Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                                  Page 13 <br />  <br />  <br />                                             Martin RG, Bartlett ES, Rosner JL, Wall  Activation Escherichia coli marA/soxS/rob <br />                                                 regulon response transcriptional activator concentration. J Mol Biol. 2008; 380:278&#226;&#8364;&#8220;284. <br />                                                 [PubMed: 18514222] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Martin RG, Rosner JL. Genomics marA/soxS/rob regulon Escherichia coli: identification                                                 directly activated promoters application molecular genetics informatics microarray <br />                                                 data. Mol Microbiol. 2002; 44:1611&#226;&#8364;&#8220;1624. [PubMed: 12067348] <br />                                             McOsker CC, Fitzpatrick PM. Nitrofurantoin: mechanism action implications resistance <br />                                                 development common uropathogens. J Antimicrob Chemother. 1994; 33(Suppl :23&#226;&#8364;&#8220;30. <br />                                                 [PubMed: 7928834] <br />                                             Nelson KE, Weinstock GM, Highlander SK, Worley KC, Creasy HH, Wortman JR, Rusch DB, <br />                                                 Mitreva M, Sodergren E, Chinwalla  et al. catalog reference genomes human <br />                                                 microbiome. Science. 2010; 328:994&#226;&#8364;&#8220;999. [PubMed: 20489017] <br />                                             Nicoloff H, Perreten V, Levy SB. Increased genome instability Escherichia coli lon mutants: <br />                                                 relation emergence multiple-antibiotic-resistant (Mar) mutants caused insertion sequence <br />                                                 elements large tandem genomic amplifications. Antimicrob Agents Chemother. 2007; <br />                                                 51:1293&#226;&#8364;&#8220;1303. [PubMed: 17220404] <br />                                             Nicoloff H, Perreten V, McMurry LM, Levy SB. Role tandem duplication lon protease                                                 AcrAB-TolC- dependent multiple antibiotic resistance (Mar) Escherichia coli mutant <br />                                                 mutations marRAB acrRAB. J Bacteriol. 2006; 188:4413&#226;&#8364;&#8220;4423. [PubMed: <br />                                                 16740948] <br />                                             Noble SM, French S, Kohn LA, Chen V, Johnson AD. Systematic screens Candida albicans <br />                                                 homozygous deletion library decouple morphogenetic switching pathogenicity. Nat Genet. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 2010; 42:590&#226;&#8364;&#8220;598. [PubMed: 20543849] <br />                                             Pan X, Yuan DS, Xiang D, Wang X, Sookhai-Mahadeo S, Bader JS, Hieter P, Spencer F, Boeke JD.                                                 robust toolkit functional profiling yeast genome. Mol Cell. 2004; 16:487&#226;&#8364;&#8220;496. [PubMed: <br />                                                 15525520] <br />                                             Parsons AB, Lopez  Givoni  Williams  Gray CA, Porter J, Chua G, Sopko R, Brost RL, Ho <br />                                                 CH, et al. Exploring mode action bioactive compounds chemical-genetic profiling                                                 yeast. Cell. 2006; 126:611&#226;&#8364;&#8220;625. [PubMed: 16901791] <br />                                             Pathania R, Zlitni S, Barker C, Das R, Gerritsma DA, Lebert J, Awuah E, Melacini G, Capretta FA, <br />                                                 Brown ED. Chemical genomics Escherichia coli identifies inhibitor bacterial lipoprotein <br />                                                 targeting. Nat Chem Biol. 2009; 5:849&#226;&#8364;&#8220;856. [PubMed: 19783991] <br />                                             Qin J, Li R, Raes J, Arumugam M, Burgdorf KS, Manichanh C, Nielsen T, Pons N, Levenez F, <br />                                                 Yamada T, et al. human gut microbial gene catalogue established metagenomic sequencing. <br />                                                 Nature. 2010; 464:59&#226;&#8364;&#8220;65. [PubMed: 20203603] <br />                                             Rocha EP, Danchin  Essentiality, expressiveness, drives gene-strand bias bacteria. Nat Genet. <br />                                                 2003; 34:377&#226;&#8364;&#8220;378. [PubMed: 12847524] <br />                                             Roguev  Bandyopadhyay S, Zofall M, Zhang K, Fischer T, Collins SR, Qu H, Shales M, Park HO, <br />                                                 Hayles J, et al. Conservation rewiring functional modules revealed epistasis map                                                 fission yeast. Science. 2008; 322:405&#226;&#8364;&#8220;410. [PubMed: 18818364] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Schirch V, Hopkins S, Villar E, Angelaccio S. Serine hydroxymethyltransferase Escherichia coli: <br />                                                 purification properties. J Bacteriol. 1985; 163:1&#226;&#8364;&#8220;7. [PubMed: 3891721] <br />                                             Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati  Punna T, Ihmels J, Andrews B, <br />                                                 Boone C, Greenblatt JF, et al. Exploration function organization yeast early <br />                                                 secretory pathway epistatic miniarray profile. Cell. 2005; 123:507&#226;&#8364;&#8220;519. [PubMed: <br />                                                 16269340] <br />                                             Stover P, Schirch V. metabolic role leucovorin. Trends Biochem Sci. 1993; 18:102&#226;&#8364;&#8220;106. <br />                                                 [PubMed: 8480361] <br />                                             Taber HW, Mueller JP, Miller PF, Arrow  Bacterial uptake aminoglycoside antibiotics. <br />                                                 Microbiol Rev. 1987; 51:439&#226;&#8364;&#8220;457. [PubMed: 3325794] <br />                                             Tamae C, Liu  Kim K, Sitz D, Hong J, Becket E, Bui  Solaimani P, Tran KP, Yang H, et al. <br />                                                 Determination antibiotic hypersensitivity 4,000 single-gene-knockout mutants                                                 Escherichia coli. J Bacteriol. 2008; 190:5981&#226;&#8364;&#8220;5988. [PubMed: 18621901] <br />  <br />  <br />  <br />  <br />                                                      Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                                  Page 14 <br />  <br />  <br />                                             Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, Page N, Robinson M, Raghibizadeh S, <br />                                                 Hogue CW, Bussey H, et al. Systematic genetic analysis ordered arrays yeast deletion <br />                                                 mutants. Science. 2001; 294:2364&#226;&#8364;&#8220;2368. [PubMed: 11743205] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Toro E, Shapiro L. Bacterial chromosome organization segregation. Cold Spring Harb Perspect <br />                                                 Biol. 2010; 2:a000349. [PubMed: 20182613] <br />                                             Travers  Muskhelishvili G. DNA supercoiling - global transcriptional regulator enterobacterial <br />                                                 growth? Nat Rev Microbiol. 2005; 3:157&#226;&#8364;&#8220;169. [PubMed: 15685225] <br />                                             Tu Y, McCalla DR. Effect activated nitrofurans DNA. Biochim Biophys Acta. 1975; 402:142&#226;&#8364;&#8220; <br />                                                 149. [PubMed: 1100114] <br />                                             Typas  Banzhaf M, van den Berg van Saparoea B, Verheul J, Biboy J, Nichols RJ, Zietek M, <br />                                                 Beilharz K, Kannenberg K, von Rechenberg M, Breukink E, den Blaauwen T, Gross CA, Vollmer <br />                                                 W. Regulation peptidoglycan synthesis outer-membrane proteins. Cell. Dec 23; 2010 143(7): <br />                                                 1097&#226;&#8364;&#8220;109. PMID: 21183073. [PubMed: 21183073] <br />                                             Typas  Nichols RJ, Siegele DA, Shales M, Collins SR, Lim B, Braberg H, Yamamoto N, Takeuchi <br />                                                 R, Wanner BL, et al. High-throughput, quantitative analyses genetic interactions E. coli. Nat <br />                                                 Methods. 2008; 5:781&#226;&#8364;&#8220;787. [PubMed: 19160513] <br />                                             Vora T, Hottes AK, Tavazoie S. Protein occupancy landscape bacterial genome. Mol Cell. 2009; <br />                                                 35:247&#226;&#8364;&#8220;253. [PubMed: 19647521] <br />                                             Warner DM, Levy SB. Different effects transcriptional regulators MarA, SoxS Rob                                                 susceptibility Escherichia coli cationic antimicrobial peptides (CAMPs): Rob-dependent <br />                                                 <span id='am-97' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-98' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-108' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span>CAMP</span> induction marRAB operon. Microbiology. 2010; 156:570&#226;&#8364;&#8220;578. [PubMed: <br />                                                 19926649] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Warner JR, Reeder PJ, Karimpour-Fard  Woodruff LB, Gill RT. Rapid profiling microbial <br />                                                 genome using mixtures barcoded oligonucleotides. Nat Biotechnol. 2010; 28:856&#226;&#8364;&#8220;862. <br />                                                 [PubMed: 20639866] <br />                                             Werner JN, Chen EY, Guberman JM, Zippilli AR, Irgon JJ, Gitai  Quantitative genome-scale <br />                                                 analysis protein localization asymmetric bacterium. Proc Natl Acad Sci U S  2009; <br />                                                 106:7858&#226;&#8364;&#8220;7863. [PubMed: 19416866] <br />                                             Xu D, Jiang B, Ketela T, Lemieux S, Veillette K, Martel N, Davison J, Sillaots S, Trosok S, <br />                                                 Bachewich C, et al. Genome-wide fitness test mechanism action studies inhibitory <br />                                                 compounds Candida albicans. PLoS Pathog. 2007; 3:e92. [PubMed: 17604452] <br />                                             Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, Touda Y, Furubayashi  Kinjyo S, <br />                                                 Dose H, Hasegawa M, et al. Update Keio collection Escherichia coli single-gene deletion <br />                                                 mutants. Mol Syst Biol. 2009; 5:335. [PubMed: 20029369] <br />                                             Yeh P, Tschumi AI, Kishony R. Functional classification drugs properties pairwise <br />                                                 interactions. Nat Genet. 2006; 38:489&#226;&#8364;&#8220;494. [PubMed: 16550172] <br />                                             Yeh PJ, Hegreness MJ, Aiden AP, Kishony R. Drug interactions evolution antibiotic <br />                                                 resistance. Nat Rev Microbiol. 2009; 7:460&#226;&#8364;&#8220;466. [PubMed: 19444248] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                             Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 1. Phenomic Profiling enhanced Keio Collection yields robust rich dataset <br />                                               Classification 324 stresses screened (left), cellular targets antibiotic/ <br />                                             antimicrobial/drug classes (right). (B) Heat map representation scatter plot comparing <br />                                             normalized colony sizes pixels plate replicates 1 2 entire dataset. Bins <br />                                             indicate square root number replicate pairs 10 &#195;&#8212; 10 pixel window                                             depicted color scale. Note vast majority replicates highly correlated <br />                                             colony sizes. (C) Clustergram fitness scores 3979 mutant strains response 324 <br />                                             conditions. Zoomed insets demonstrate clustering conditions (x-axis) genes (y- <br />                                             axis) common pathway (rfa cluster), protein complexes encoded                                             operon (nuo) different operons (dsbA dsbB). Gray boxes indicate missing data. (D) <br />                                             High correlation pair phenotypic signatures predictive shared protein <br />                                             interaction operon membership. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                         Page 16 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 2. Identification responsive conditionally-essential genes <br />                                               Using 5% false discovery rate (FDR), 49% strains tested phenotype <br />                                             (open circle red line). FDR relaxed, phenotypes identified (red <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             line). 5% FDR, strains phenotypes (black) (2.3%; 94 <br />                                             strains) 30 phenotypes (green, Multi-Stress Responsive (MSR) genes). (B) <br />                                             MSR genes participate wide variety cellular processes, particularly related                                             metabolism cell envelope. Genes manually curated COG-based functions; <br />                                             gene allowed belong single function. (C) 196 genes conditionally- <br />                                             essential (CE) study.  roughly half previously described CE                                             auxotrophy. Note auxotrophic genes display growth phenotype                                             rich medium condition classified jointly auxotroph rich media CE. <br />                                             (D) Rich media CE gene products enriched outer cell envelope (periplasm                                             outer membrane) relative Keio essential genes (p=0.00026), highlighting importance <br />                                             compartment tolerating stress. cytoplasmic gene category displayed <br />                                              enriched CE gene products. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                      Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                          Page 17 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 3. Phenomic profiling identifies phenotypes orphan gene mutants <br />                                               Cumulative distribution phenotypes indicating fraction gene mutants                                             class having number phenotypes shown X-axis. plot reveals                                             orphan gene mutants phenotypes, tend fewer phenotypes annotated <br />                                             gene mutants. insert quantifies phenotype deficit orphan mutants. (B) Cumulative <br />                                             distribution highly correlated pairs identifies orphan genes correlate highly                                             annotated gene, providing high confidence clues function orphan gene. <br />                                             Values shown pair bars p-values associated pairwise correlation <br />                                             strains indicated correlation coefficients. (C) High confidence correlations <br />                                             orphans annotated genes (r&#226;&#8240;&#165; 0.5) provide leads related different <br />                                             cellular functions. Procedure functional assignment described Fig. 2. Note                                             &#226;&#8364;&#339;annotated genes&#226;&#8364;? classified genes &#226;&#8364;&#339;unknown function&#226;&#8364;? &#226;&#8364;&#339;general <br />                                             function prediction ? manual curation. (D) Annotated genes responsible                                             phenotypes tend broadly conserved, responsive orphan genes tend                                             restricted &#206;&#179;-proteobacteria. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                          Page 18 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 4. function marB <br />                                               marR&#226;&#710;&#8217; marB&#226;&#710;&#8217; phenotypic signatures highly correlated                                              highly anticorrelated acrB&#226;&#710;&#8217;  . graph positions correlations <br />                                             histogram showing pair-wise correlation coefficients 3979 mutants. (B) <br />                                             Schematic E. coli multiple antibiotic resistance (mar) operon. marB gene                                             unknown function, results suggest encodes protein inhibits MarA. (C) RT- <br />                                             PCR analysis shows marA transcription derepressed marB&#226;&#710;&#8217; cells. Derepression                                             independent additive marR&#226;&#710;&#8217;. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 January 7. <br />                             Nichols et al.                                                                                          Page 19 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Figure 5. new network feature contributing anti-folate drug synergy <br />                                               Schematic E. coli tetrahydrofolate (THF) biosynthesis pathway enzymatic <br />                                             steps inhibited Sulfa TMP. (B) Clustergram genes respond Sulfa, TMP,                                             combination. Zoomed image indicates gcv mutants sensitive Sulfa, glyA&#226;&#710;&#8217;                                             sensitive TMP, mutants exhibit essentially wildtype growth                                             response drug combination. (C) glyA&#226; <br /> </body></html>