<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                      NIH Public Access <br />                       Author Manuscript <br />                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                   Published final edited form  <br />                    Chem Res Toxicol. 2012 November 19; 25(11): 2542&#226;&#8364;&#8220;2552. doi:10.1021/tx300337j. <br />  <br />  <br />  <br />                   Novel Oxidatively Activated Agents Modify DNA                   Enhanced Ercc1 Silencing <br />                   Amy R. Jones*, Tiffany R. Bell-Horwath*, Guorui Li*, Stephanie M. Rollmann**,&#194;&#167;, Edward <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                   J. Merino*,&#194;&#167; <br />                   **Department Biological Sciences, University Cincinnati, Cincinnati, Ohio 45221 <br />  <br />                   *Department     Chemistry, University Cincinnati, Cincinnati, Ohio 45221-0172 <br />  <br />                   Abstract <br />                         Agents chemically modify DNA form backbone cancer treatments. key problem <br />                         DNA modifying agents lack specificity. address issue, designed novel <br />                         molecular scaffolds, termed Hq Hq2, activated hallmark                         cancers: elevated concentrations reactive oxygen species. Elevated reactive oxygen species                         linked oncogenesis increase aggressive cancers. agents                         quinones  oxidation, form highly electrophilic species. vitro studies identified $watermark-text <br />  <br />  <br />  <br />  <br />                         mode addition DNA. aniline portion Hq serves enhance nucleophilic addition <br />                         ethyl phenyl ether instead forming common Michael additions. Structural <br />                         characterization showed agents add 2&#226;&#8364;&#178;-deoxyguanosine N2,N3-positions. product <br />                         formed bulky hydroxy-N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct. addition,                         oxidatively activated agents added 2&#226;&#8364;&#178;-deoxyadenosine 2&#226;&#8364;&#178;-deoxycytidine, <span id='am-66' about='obo:IMR_0200384' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:hasDbXref" content="KEGG:C00214" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Thymidine" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0200384" datatype="xsd:string"></span>thymidine</span> <br />                         2&#226;&#8364;&#178;-deoxyinosine. findings confirmed primer extension analysis 392 base pair <br />                         DNA. length primer extension product reduced 69.0 &#194;&#177; 0.6% oxidative <br />                         activation Hq2 compared controls. Little sequence dependence observed 76%                         <span id='am-51' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span>guanine</span>, <span id='am-23' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenine</span>, cytosine residues showing increase extension stops                         fold controls. Benzetheno-nucleobase addition double stranded DNA confirmed <br />                         LC/MS self-complementary oligonucletide. Experiments carried confirm                         vivo DNA damage. lesion identified vitro, reasoned <span id='am-33' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-40' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span>nucleotide</span> excision <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                         repair involved reversing effects oxidatively activated agents                         enhance toxicity Drosophila melanogaster. Using RNAi based approach, Ercc1 silenced <br />                         survival monitored injection agent. expected, bulky cross-linking DNA <br />                         modifying agents, cisplatin chlorambucil, showed statistically significant enhanced toxicity                         Drosophila silenced Ercc1. addition, 5-fluorouracil, does produce bulky lesions, <br />                         showed selective toxicity. Hq Hq2 showed statistically significant toxicity                         Drosophila silenced Ercc1. Examination cytotoxicity shows renal carcinoma cell lines                         target agents median IC50 1.8 &#206;&#188;M. Taken  data shows                         designed oxidatively-activated agents form distinct, bulky DNA modifications prove difficult <br />                         cancer cells possessing elevated reactive oxygen species phenotype overcome.                         modification produced relatively unique anticancer agents. <br />  <br />  <br />  <br />  <br />                   &#194;&#167; <br />                    Correspondence sent  Dr. Edward J. Merino, Department Chemistry, University Cincinnati, Cincinnati, OH <br />                   45221-0172, Phone number: 513-556-9200, Fax Number: 513-556-9239, merinoed@uc.edu. Dr. Stephanie M. Rollmann, Department <br />                   Biological Sciences, University Cincinnati, Cincinnati, OH 45221-0006, USA, Phone number: 513-556-9729, Fax number: <br />                   513-556-5299, stephanie.rollmann@uc.edu. <br />                    Jones et al.                                                                                                 Page 2 <br />  <br />  <br />  <br />                       Keywords <br />                             Reactive oxygen species; ROS; DNA-modifying agent; 2&#226;&#8364;&#178;-deoxyguanosine; anti-cancer agent; <br />                             Ercc1; Renal carcinoma <br />  <br />  <br />                       INTRODUCTION <br />                                      harsh effects current chemotherapy treatments motivating factors <br />                                      increasing selectivity novel anticancer agents.1 Agents chemically modify DNA <br />                                      form backbone cancer treatments. typical DNA-modifying agent cisplatin, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      used treat fifty percent cancer patients.2 agents represent large portion <br />                                      current phased anticancer agents. design DNA-modifying agents                                      slowed drastically perception agents high levels effects <br />                                      modification DNA, subsequent cytotoxicity, non-target cells. lack <br />                                      specificity termed target reactivity, limits tolerated doses   <br />                                      decreases efficacy.3 Instead, agents attached transport active scaffolds <br />                                      enhance uptake target cancer cells.4,5 effects,  problematic, <br />                                      improving selectivity major goal development novel anticancer agents.                                      developing novel agents address problems. <br />  <br />                                      potential path forward cancer drug development advantage hallmark                                      cancer cells, elevated reactive oxygen species (ROS).6,7 Persistent elevation ROS <br />                                      numerous cancers, including renal cell carcinoma, melanoma, $watermark-text <br />  <br />  <br />  <br />  <br />                                      leukemia.8&#226;&#8364;&#8220;10 ROS occurs major endogenous forms cell: superoxide, <br />                                      hydrogen peroxide, singlet oxygen, hydroxyl radical.11&#226;&#8364;&#8220;13 Mitochondrial dysfunction, <br />                                      common cancer, known increase ROS.14 Increased ROS damages DNA, leading                                      mutation.15 Consequently, surprise levels ROS-induced DNA damage <br />                                      correlates cancer prognosis.16 turn, mutations cause enhanced activation                                      oncogenes.17 critical role ROS oncogene transformed cancer cells shown <br />                                      investigating c-Myc induced damage.18 Furthermore, Ras oncogene locked <br />                                      active state concentration cellular antioxidant enzymes lowered causing <br />                                      elevated ROS.19,20 result, <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> research teams actively searching                                      means utilize elevated ROS order generate selective anticancer agents.21&#226;&#8364;&#8220;23 lab <br />                                      designed oxidatively activated DNA modifying agents selectivity target                                      elevated ROS phenotype cancer cells.24 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      new oxidatively activated agents possess attractive anticancer properties.24                                      agents, Hq Hq2, utilize novel mechanism activation highly <br />                                      selective (Figure 1A). agents unreactive stable water, strongly <br />                                      activated ROS forms, excluding hydrogen peroxide. Addition oxidative <br />                                      equivalents leads rate enhancement greater 1700-fold compared                                      oxidant present. reactivity examined, nitrogen <br />                                      mustard, agents actually oxidatively activated quinones. specifically,                                      demonstrated agents capable adding guanine. product                                      hydroxy-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct, guanine possessed added <br />                                      phenol. <br />  <br />                                      possible formation benzethenoguanine left important questions                                      addressed. Specifically, questioned new agents actually DNA modifying <br />                                      agents. set structurally characterize lesion produced nucleosides                                      double stranded DNA, examine reaction mechanism vitro (Figure 1A).                                      determine vivo DNA adduct formation, studies using powerful genetic model 25 <br />  <br />  <br />                                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 3 <br />  <br />  <br />                                  Drosophila melanogaster, accomplished (Figure 1B). D. melanogaster shown <br />                                  effective, animal model cancer research drug discovery.26&#226;&#8364;&#8220;29                                  key features DNA repair cell death conserved Drosophila                                  mammals.30 Approximately 75% human disease related genes functional ortholog <br />                                  flies.31,32  studies Drosophila instrumental elucidating Ras <br />                                  signaling, major components conserved mammals.33 <br />  <br />                                   investigated vitro DNA lesions produced novel oxidatively <br />                                  activated agents NMR. agents required <span id='am-46' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span>nucleoside</span> base amine                                  initiate reactivity. primer extension assay used explore lesion produced                                  392-nucleotide PCR product. discovered nucleosides possessing amines <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  modified double stranded DNA, producing bulky lesions. agents product <br />                                  adduct small double stranded DNA. lesions led predict                                  nucleotide excision repair important mechanism repairing adducts.34  <br />                                  used targeted RNAi knockdown approach based GAL4/UAS (Figure 1B) <br />                                  evaluate bulky DNA modification occurring vivo.35 silenced Ercc1 gene, <br />                                  protein forms complex XPF key step nucleotide excision double <br />                                  strand break repair pathway.36 Drosophila lacking Ercc1 expression showed <br />                                  elevated toxicity known DNA modifying agents, cisplatin chlorambucil.                                  elevated Ercc1-dependent sensitivity did occur Drosophila treated 5- <br />                                  fluorouracil, does produce bulky lesion. results confirmed D. <br />                                  melanogaster effective model organism ascertaining genetic mechanisms                                  DNA modifying agents. examined novel agents, Hq Hq2.                                  demonstrate flies Ercc1 expression knocked enhanced <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  sensitivity novel agents, particularly Hq, supporting role                                  oxidatively activated agents modifying DNA vivo. <br />  <br />                       MATERIAL METHODS <br />                       Synthesis Hq Hq2 <br />                                  synthesis Hq Hq2 accomplished.24 NMR (1H 13C) spectra                                  recorded Bruker AMX 400MHz spectrometer. Chemical resonances reported &#206;&#180; <br />                                  (ppm) units using 13C residual 1H signals deuterated solvents references. High- <br />                                  resolution mass spectra (ESI) recorded Micromass Q-TOF 2 (Waters). Analytical <br />                                  layer chromatography (TLC) performed silica gel 60 GF254 (Merck). <br />                                  Compounds greater 99% pure according HPLC. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       Reaction Hq Nucleoside Analogs <br />                                  nucleoside reactivity studies, Hq (22 &#206;&#188;l dissolved DMSO, 1mmol) added <br />                                  phosphate buffer (2 mL, 25mM NaH2PO4 5% acetonitrile 95% <span id='am-28' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span>, pH 8.0) <br />                                  contained 0.4 mmol listed nucleoside. Na2Ir2Cl6 (22 &#206;&#188;l dissolved water, 1 <br />                                  mmol) added, reaction mixed. Product monitored using Beckman <br />                                  Coulter Gold HPLC equipped diode array detector (260 nm detection)                                  lesion formation twice day 7 days. HPLC conditions follows: Cosmosil <br />                                  5C18-PAQ Waters column used (4.6ID, 150mm length). gradient (solvent <br />                                  95% water, 5% acetonitrile solvent B=5% water, 95% acetonitrile) linear: 0% B <br />                                  5 min, 100% B 20 min, 100% B 4 min, 0% B 2 min held 4 min.                                  mass spectrometry (MS), peaks collected, dried remove solvent  stable,                                  directly injected MS), analyzed MS. yields 0.5                                  3%, depending nucleoside adduct. <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                             Page 4 <br />  <br />  <br />                                   MS performed follows: isolated products resuspended 100 &#206;&#188;L 0.25% <br />                                   <span id='am-71' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-72' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span>acetic acid</span> 15% acetonitrile. Infusion instrument occurred rate 5 &#206;&#188;L/min. <br />                                   mass spectrometry performed Thermo Fisher Scientific LTQ-FT, hybrid <br />                                   instrument consisting linear <span id='am-5' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-10' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span>ion</span> trap Fourier transform ion cyclotron resonance <br />                                   mass spectrometer. entire elutant introduced LTQ-FT, using standard <br />                                   electrospray ionization source instrument spray voltage 5 kV capillary <br />                                   temperature 275&#194;&#176;C. Autogain control used set 500,000 maximum injection <br />                                   time 1250 ms FT-ICR scans. Collision induced dissociation, MS/MS,                                   executed linear trap AGC setting 10,000 maximum injection time                                   500 ms. FT-ICR scans acquired positive ion mode 100,000 resolving <br />                                   power m 400. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       2&#226;&#8364;&#178;-Deoxyguanosine Hq Adduct NMR Analysis <br />                                   1H NMR experiments carried Bruker DGX-501, 500 MHz instrument. <br />                                   Chemical resonances reported &#206;&#180; (ppm) units, using residual 1H signals                                   deuterated solvents references. reaction listed scaled 100-fold (total volume <br />                                   200 mL). reaction incubated dark 25&#194;&#176;C days. Solvent                                   evaporated adduct purified stages.  Biotage SP1 Flash <br />                                    outfitted 5.5g RediSep Rf Gold C18 column, used purify reaction <br />                                   mixture, using 97% water 3% methanol buffer methanol buffer B.                                   gradient 0% methanol 10% 10 column volumes, 10% 100% 3 column <br />                                   volumes held 5 column volumes. sample (~75% pure HPLC)                                   collected 13 column volumes lyophilized remove solvent. second stage <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   employed HPLC purification, using Grace Alltima HP C8 semi-preparative column (3&#206;&#188;m, <br />                                   7 &#195;&#8212; 53mm) 2.5ml/min. Solvent 95% water 5% acetonitrile solvent B                                   95% acetonitrile 5% water. gradient 0% B 5 min, 30% B 8 min, 100% <br />                                   B 3 min held 4 min. Absorbance monitored 260nM.                                   collected sample eluted 8 min. HPLC purification, product dried                                   dissolved 500 &#206;&#188;L d6-DMSO evaluated NMR 15 hrs determine 1H- <br />                                   NMR. <br />  <br />                       Solvolysis Hq Methanol Using Na2IrCl6 Oxidant <br />                                   Hq (10 mg, 0.039 mmol) dissolved methanol (2 mL). Na2IrCl6&#194;&#183;6H2O (43 mg, 0.078 <br />                                   mmol) N,N-diisopropylethylamine (4.5 mg, 0.039 mmol) added mixture.                                   mixture reacted rt 0.5 hr. reaction filtered silica gel washed <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   acetonitrile. filtrate concentrated residue purified flash <br />                                   chromatography silica gel provide product (9 mg, 0.017 mmol, 90% yield)                                   yellow oil. 1H NMR (CDCl3, 400 MHz) &#206;&#180; 7.23 (t, J=8.2 Hz, 2 H), 6.80 (d, J=10.4 Hz, 2 H), <br />                                   6.69 (m, 3 H), 6.26 (d, J=10.0 Hz, 2 H), 3.76 (t, J=6.2 Hz, 2 H), 3.54 (t, J=6.2 Hz, 2 H), 3.41 <br />                                   (q, J=7.1 Hz, 2 H), 3.36 (s, 3 H), 0.87 (t, J=7.1 Hz, 3 H); HRMS (ESI, positive) m calcd. <br />                                   C17H22NO3 [M+H]+: 288.1600,  288.1614. <br />  <br />                       DNA Damage Visualization 392-Nucleotide PCR Product <br />                                   Primer extension experiments performed using 392-nucleotide dsDNA, synthesized <br />                                   pUC19 plasmid vector (New England Biolabs). 30 cycle PCR amplification (55&#194;&#176;C <br />                                   30 s, 75&#194;&#176;C 45 s, 95&#194;&#176;C 30 s) performed, using 19-nucleotide forward <br />                                   primer, GGCCTCTTCGCTATTACGC, starting nucleotide position 287                                   vector, 19-nucleotide reverse primer, ATACGCAAACCGCCTCTC, starting                                   position 672.37 oligonucleotide purified using Cycle Pure Kit (Omega BioTek), <br />                                   yielding final concentration 40 &#206;&#188;g/&#206;&#188;L. DNA reacted agent, Hq2,                                   Na2IrCl6 5 hrs 37&#194;&#176;C (720 &#206;&#188;g DNA, 0.2 mM sodium phosphate, pH=8, 0.5 mM  <br />                                   Hq2, 2 mM iridium). pre-quenched control, DNA added 24 hrs  <br />                                    Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 5 <br />  <br />  <br />                                   agent oxidant mixed. end products benzoquinone N- <br />                                   phenyldiethanolamine. sample reacted DNA added primer extension <br />                                   mix (1X vent buffer, 100 &#206;&#188;M dNTP, 200 nM primer, 0.05 U/&#206;&#188;L vent(exo-) DNA <br />                                   polymerase). primer fluorescently-labeled IRDye700. cycles                                   accomplished (55&#194;&#176;C 15 s, 72&#194;&#176;C 1 min, 95&#194;&#176;C 30 s). Denaturing load dye                                   added samples, 12% denaturing PAGE gel performed. gel                                   visualized using Odyssey Infrared Imaging (LiCor) 169 &#206;&#188;m resolution                                   700-channel. Sequencing performed standard methods, using manual sequencing <br />                                   392-nucleotide PCR product acyclo-terminators, fluorescently labeled <br />                                   M13 primer substituted. Experiments performed triplicate, standard errors <br />                                   calculated. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       LC/MS Product Analysis Reaction Hq2 Oligonucleotide <br />                                   High purity oligonucleotides purchased Eurofins MWG operon. Electrophoresis <br />                                   showed oligonucletide greater 95% pure. DNA sequence 5&#226;&#8364;&#178;- <br />                                   GCGCAATTGCGC-3&#226;&#8364;&#178;. DNA annealed 25 mM sodium phosphate. Prior                                   reaction, DNA desalted placed 5mM Ammonium Acetate buffer, pH 8.0.                                   100 &#206;&#188;L reaction contained 2 mM DNA, 10 mM Hq2, 10 mM                                   (diacetoxyiodo)benzene oxidant. reaction left room temperature dark                                   days. Analysis mass spectroscopy utilized Thermo Scientific LTQ-FT, hybrid <br />                                   instrument consisting linear ion trap Fourier transform ion cyclotron resonance <br />                                   mass spectrometer. injection volume 10 &#206;&#188;L. Liquid chromatography                                   accomplished using Waters Symmetry C18 5 &#206;&#188;m, 2.1 &#195;&#8212; 150 m column, Finnigan Surveyor <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   MS pump, Finnigan Micro autosampler. flow rate 200 &#206;&#188;L/min                                   gradient ranged 2% (v/v) acetonitrile 5mM ammonium formate 15% 35 min. <br />                                   Autogain control used set 500,000 maximum injection time 1250 ms                                   FT-ICR scans. FT-ICR scans acquired negative ion mode 100,000 <br />                                   resolving power m 400. Mass accuracy errors 500 ppb scan. <br />  <br />                       Drosophila Stocks Husbandry <br />                                   Drosophila maintained standard cornmeal, agar, molasses media 25&#194;&#176;C                                   12:12 hr light:dark cycle. induce targeted gene silencing, following RNAi line                                   obtained Vienna Drosophila RNAi Center:35 UAS-Ercc1RNAi (v12622VDRC).                                   line crossed daughterless-GAL4 (da-GAL4) driver line order ubiquitously <br />                                   inactivate gene expression. isogenic host strain, w1118 (60100VDRC), crossed da- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   GAL4 control genetic background effects. da-GAL4 line kindly provided <br />                                   Dr. Mike Grotewiel (Virginia Commonwealth University.38 <br />  <br />                       RNAi Mediated Suppression Targeted Genes <br />                                   confirm knockdown efficiency transgenic RNAi approach, RT-PCR assays                                   conducted. sample, adult male Drosophila homogenized, total <br />                                   RNA isolated using Trizol (Invitrogen, Carlsbad, CA). Total RNA DNase <br />                                   treated using DNA-free (Ambion, Austin, TX), according manufacturer&#226;&#8364;&#8482;s instructions. <br />                                   Total RNA (0.5 &#206;&#188;g) reverse transcribed using Accuscript High Fidelity Strand <br />                                   cDNA synthesis kit (Agilent Technologies, Santa Clara, CA), resulting cDNA                                   used RT-PCR assays. Primers follows: Ercc1-F 5&#226;&#8364;&#178;- <br />                                   CGTGCTGTACCTCTCGC-3&#226;&#8364;&#178; Ercc1-R 5&#226;&#8364;&#178;-CTGAGGAACGGTTCCTG-3&#226;&#8364;&#178;. <br />                                   QuantumRNA &#206;&#178;-actin Internal Standards (Ambion, Austin, TX) used amplify &#206;&#178;- <br />                                   actin (control) according manufacturer&#226;&#8364;&#8482;s instructions, following optimization &#206;&#178;- <br />                                   actin:competimer ratio. PCR products separated electrophoretically 2% agarose <br />                                   gel visualized ethidium bromide. RT-PCR assays independent RNA <br />                                   isolations genotype performed. <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                               Page 6 <br />  <br />  <br />                       Microinjection Drosophila <br />                                   Microinjections performed individual adult male Drosophila                                   seven days old. Individuals held position microinjection using gentle vacuum. <br />                                   pulled glass micropipette attached Picospritzer III (Parker Hannifin, Cleveland, <br />                                   OH) used deliver anticancer agents fly abdomen. 0.5 &#206;&#188;L volume agent <br />                                   injected ~40 psi compressed air. Agents dissolved 100% DMSO,                                   solutions fresh daily. Blue food dye (0.125 mg/mL) added solutions                                   confirm delivery agent. Agents tested included Hq, Hq2, chlorambucil, <br />                                   cisplatin, 5-fluorouracil. Microinjections vehicle (control)                                   performed. agent, individuals genotype injected 10 mM <br />                                   concentration. concentration 10 mM used preliminary analysis showed <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   effective eliciting response. Post injection, flies placed fresh standard <br />                                   Drosophila media 25&#194;&#176;C 12:12 hr light:dark cycle. Individuals scored                                   survival day followed total 7 days examine longer-term effects                                   agents. analyze significance data, chi-squared calculated. <br />  <br />                       IC50 Determination Hq2 <br />                                   Hq2 submitted NCI Developmental Therapeutics Program screening.                                   method uses total protein content developed Monks et. al.39 Briefly, cells                                   treated Hq2 fixed plate surface. number cells proportional <br />                                   relative protein measured sulforhodamine B. <br />  <br />                       RESULTS <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       vitro Reactions Nucleosides Oligonucleotides <br />                                   data established Hq Hq2 highly selective cytotoxic agents                                   certain cancers. new convention,40 numbering 2&#226;&#8364;&#178;-deoxyguanosine (dG)                                   retained manuscript. numbering Hq phenol derivatives                                   denoted double prime, discussing phenol addition dG.                                   designed agents release hydroquinone moiety oxidation leave                                   activated nitrogen mustard. Instead, hydroquinone active portion <br />                                   molecule. Initially, Hq reacted dG, using Na2IrCl6 oxidant                                   determine addition occurred. mass product corresponded                                   formation benzetheno-2&#226;&#8364;&#178;-deoxyguanosine. Previous literature demonstrates                                   benzoquinone add 2&#226;&#8364;&#178;-deoxyguanosine produce 3&#226;&#8364;&#179;-hydroxy-1,N2- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   benzetheno-2&#226;&#8364;&#178;-deoxyguanosine.41 step reaction mechanism Michael <br />                                   addition.  probed ability Hq react form benzetheno-2&#226;&#8364;&#178;- <br />                                   deoxyguanosine adduct different. <br />  <br />                                   order determine agents modify DNA benzoquinone, Hq oxidized                                   presence methanol product determined. Na2IrCl6 used induce oxidation <br />                                   methanol (Figure 2A). Solvolysis methanol serves simple nucleophile                                   means trap site nucleophilic addition. oxidation, Hq 90% oxidized                                   new compound 15 min. 1H-NMR used analyze isolated product; region <br />                                   3 8 ppm shown Figure 2A. eighteen protons region, <br />                                   indicating product additional methoxy group. singlet peak 3.4 ppm                                   integration 2.9 corresponds added methoxy group. aromatic region                                   symmetric, indicating addition did occur C3&#226;&#8364;&#179; C2&#226;&#8364;&#179; phenol                                   Michael addition. used mass spectrometry (MS) ensure product consistent                                   single methanol addition.  Hq does behave like benzoquinone. Instead, <br />                                   oxidative activation leads potent electrophile, addition C4&#226;&#8364;&#179;. infers  <br />  <br />  <br />                                    Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                            Page 7 <br />  <br />  <br />                                  DNA lesion produced different benzoquinone, agents simply <br />                                  releasing benzoquinone mechanism action. <br />  <br />                                  difference reactivity oxidative activation, investigated lesion <br />                                  produced NMR analysis (Figure 2B). consistency, NMR labeled starting <br />                                  products. DNA lesion formed incubation Hq compared <br />                                  formed benzoquinone. product Hq, Na2IrCl6, dG water- <br />                                  phosphate buffer used isolate dG-adduct formed. oxidant used Na2IrCl6 <br />                                  possesses correct potential oxidize agents dG.42 reaction                                  benzoquinone dG established Jowa et al. Chenna                                  Singer.41,43 followed procedure, entails incubation dG $watermark-text <br />  <br />  <br />  <br />  <br />                                  dimethylformamide (DMF) potassium carbonate. non-aqueous formed                                  benzetheno-adduct high yield. reaction, purified products greater                                  95% purity levels  Figure S1). Analysis positive ion MS showed products m/ <br />                                  value 358.1146 elemental composition C16H16N5O5+ (Figure 3). MS/MS <br />                                  shows deglycosylation, indicating ribose portion nucleoside modified <br />                                   Figure S2). <br />  <br />                                  Reaction benzoquinone revealed addition N1,N2 previously observed <br />                                  (Figure 2B, . resonances 4&#226;&#8364;&#8220;10 ppm shown. B shift                                  singlet 9.4 ppm ascribed terminal phenol hydrogen C1&#226;&#8364;&#179;. strong <br />                                  singlets, C D, observed 8.0. singlets C8-hydrogen C2&#226;&#8364;&#179;- <br />                                  hydrogens, C2&#226;&#8364;&#179; deshielded. Importantly, C8-hydrogen indicated <br />                                  membered ring guanine modified. Accordingly, N7 $watermark-text <br />  <br />  <br />  <br />  <br />                                  addition site. Resonances E F remaining C5&#226;&#8364;&#179; C6&#226;&#8364;&#179; singlet phenol <br />                                  hydrogens. Taken  phenol resonances indicate dG positions                                  added phenol. triplet 6.2 ppm, G, C3&#226;&#8364;&#178;-hydrogen. resonances H,  <br />                                  J ribose protons, hydroxyl ribose. Finally, shift K hydrogen                                  N2. hydrogen easily exchangeable   integration low. <br />                                  added drop D2O differentiate exchangeable non-exchangeable protons <br />                                  (Figure S3). resonances B, H,  K exchangeable confirming                                  NMR assignments. <br />  <br />                                  Analysis Hq-dG adduct accomplished similar manner (Figure 2B, . <br />                                   resonances observed key differences. resonances                                  identical 3&#226;&#8364;&#179;-hydroxy-1,N2-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct. Shift D, C8- <br />                                  hydrogen, confirmed Hq does modify N7 membered ring. Likewise, <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  resonances B, G, H,  J directly overlap (compare Figure 2B  indicating <br />                                  modification ribose. NMR identified distinct hydrogens. new <br />                                  hydrogen peaks seen.  shift 9.9 ppm strong singlet. strongly <br />                                  deshielded hydrogen come amido-hydrogen N1. addition, K-shift <br />                                  N2 hydrogen observed 4.3 ppm. Importantly, shift (bolded Figure 2B) <br />                                  indicated N1 position longer phenol-dG adduct.                                  interesting  taken appearance C8-hydrogen, NMR reveals                                  phenol added N2 N3. support data Hq producing                                  different hydroxy-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adduct, determined stability acid.                                  observed N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine adducts prone                                  degradation incubation acid weaker glycosidic bond present.44 Incubation                                  pH 1 elevated temperatures led degradation adduct compared 3&#226;&#8364;&#179;- <br />                                  hydroxy-1,N2-benzetheno adducts  Figure S4). noted tautomers <br />                                  drawn assign correctly NMR data  Figure S5). addition proposed <br />                                  mechanism form guanine adduct listed Figure S5. Importantly, data showed <br />  <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                         Page 8 <br />  <br />  <br />                                  Hq generates distinct lesion benzoquinone. distinct lesion stemmed                                  oxidative activation mechanism Hq. <br />  <br />                                  performed series HPLC analyses determine DNA bases modified                                  Hq (Figure 3). Nucleosides incubated Hq Na2IrCl6.                                  incubation period, reactions analyzed HPLC UV detection set 260 nm.                                  nucleoside, reaction chromatogram (black) control chromatograms (grey)                                  shown. controls excludes single reaction component: Hq, dG,                                  Na2IrCl6. reaction dG Hq shown left Figure 3. clearly <br />                                  observable product seen 13 min comparing controls reaction traces.                                  product characteristic band absorbance spectrum (inset). MS analysis showed $watermark-text <br />  <br />  <br />  <br />  <br />                                  predominate ion m 358, elemental composition C16H16N5O5+                                  300 ppb error. noted small product 18 min                                  possessed m value equal deglycosylation product (data shown).                                  right Figure 3, examined 7-deaza-dG reactivity. NMR revealed N7                                  involved reaction. Based data, substitution dG 7-deaza-dG                                  interfere formation reaction product. performing reaction 7- <br />                                  deaza-dG, new series products observed 15 min.  products                                  similar UV absorbance. MS spectra mass change 1 amu C17H17N4O5+, <br />                                  correct 7-deaza-dG adduct added phenol. data proved N7                                  involved reaction. validate NMR results, reacted 2&#226;&#8364;&#178;- <br />                                  deoxyinosine Hq (Figure 3B, left). 2&#226;&#8364;&#178;-deoxyinosine lacks N2  <br />                                  according NMR results, react. reaction prepared identically                                  reaction dG. saw product formation, confirming NMR results N2 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  required product formation. <br />  <br />                                  examined Hq adduct formation nucleoside. Analysis reaction <br />                                  2&#226;&#8364;&#178;-deoxyadenosine Hq revealed new product 15 min (Figure 3, middle <br />                                  left). absorbance similar dG adduct formed. product formed                                  low yield. yield ~0.4%. MS analysis adduct gave mass indicative                                  phenols added (addition 188 amu). Addition phenols observed                                  literature involving quinone addition nucleosides.43 reaction 2&#226;&#8364;&#178;- <br />                                  deoxycytidine (Figure 3, middle right) gave product, limited <br />                                  characterization HPLC. Finally, did observe adduct formation incubation <br />                                  thymidine (Figure 3, right). key difference thymidine lacks                                  nucleoside aryl amine. Altogether, data demonstrates oxidative activation $watermark-text <br />  <br />  <br />  <br />  <br />                                  Hq derivatives generates electrophile add exocyclic amino groups <br />                                  adenine, guanine, cytosine. yield reactions double stranded DNA                                  differ nucleosides aryl amines adenine cytosine accessible                                  major groove, N2 guanine accessible minor groove. <br />  <br />                                  ability modify nucleosides different parts                                  helix, set investigate agents&#226;&#8364;&#8482; reactivity double stranded DNA <br />                                  (dsDNA), biologically relevant substrates. Reaction Hq2 Na2IrCl6 large <br />                                  DNA strand allow identify specific DNA base sequences susceptible damage <br />                                  agents (Figure 4). 392-nucleotide section pUC19 plasmid, positions 287&#226;&#8364;&#8220; <br />                                  678, amplified PCR. fluorescently labeled primer used studies overlaps                                  positions 370&#226;&#8364;&#8220;389. Vent(exo-) polymerase used extension manual <br />                                  sequencing. nucleotides listed Figure 4 sequence lanes correspond template <br />                                  strand. example, bands G lane equivalent G nucleotides 392- <br />                                  nucleotide template. Agent oxidant required stop primer extension.                                  unmodified DNA extended, length PCR product 301 bp observed. <br />                                  Comparison unreacted primer length product allowed calculation percent <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                              Page 9 <br />  <br />  <br />                                  yield primer extension (Figure 4A). controls utilized identify unique <br />                                  damage produced activated Hq2. controls included minus Na2IrCl6 control <br />                                  (&#226;&#710;&#8217;OX), end products produced Hq2 oxidation (EP), pre-quenched reaction <br />                                  control (PQ), negative control containing DMSO (NR). control yielded 3.1&#226;&#8364;&#8220; <br />                                  3.2% length product demonstrated oxidative activation required. end <br />                                  products gave Hq2 oxidation produced 3.1 &#194;&#177; 0.4% extension yield, similar                                  minus oxidant control. control established end products cause extension <br />                                  stops high yield. 392-nucleotide DNA incubated oxidized  <br />                                  Hq2, similar yield 3.2 &#194;&#177; 0.1% length extension product observed.                                  reagent control yielded 3.1 &#194;&#177; 0.1% extension yield. positive control, incubated                                  392-nucleotide DNA cisplatin. Cisplatin known DNA modifying agent  <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   induce replication stops DNA damage sites. expected, observed                                  length extension product cisplatin positive control.  Hq2 tested                                  investigate DNA modifying capabilities. Treatment 392-nucleotide DNA <br />                                  Hq2 followed oxidative activation led reduction length product <br />                                  3.2% 1.0 &#194;&#177; 0.2%, decrease 69 &#194;&#177; 0.6%. decline product formation <br />                                  shows Hq2 derivatives modify dsDNA stop progression DNA <br />                                  polymerase vitro. <br />  <br />                                  primer extension assay provides evidence damage induced Hq2                                  derivatives guanine specific (Figure 4B). positive control, cisplatin, reacts <br />                                  guanine repeats. seen CIS lane gel, largest replication <br />                                  stop position 413. corresponding sequence 5&#226;&#8364;&#178;-GGGG. negative controls <br />                                  (&#226;&#710;&#8217;OX, EP, PQ, NR) limited replication stops. attributed stops $watermark-text <br />  <br />  <br />  <br />  <br />                                  polymerase&#226;&#8364;&#8482;s inability pass certain structural features hard replicate cloning <br />                                  region pUC19. contrast, activated Hq2 RXN lane capable modifying <br />                                  bases positions. looking positions 410&#226;&#8364;&#8220;470, nucleotides showed                                  greater fold enhancement early terminations. particular, guanine                                  positions 412&#226;&#8364;&#8220;414, 422&#226;&#8364;&#8220;423, 430&#226;&#8364;&#8220;436 exhibited replication stops. Stops adenine <br />                                  occur. example, replication stops seen 410 430. Similar extension <br />                                  stops cytosine positions 415&#226;&#8364;&#8220;419 440&#226;&#8364;&#8220;441 observed.                                  predominate stops extension, 76% base types showing                                  increase extension stops fold. contrast, observed little <br />                                  modification <span id='am-41' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span>thymine</span>. thymine positions 392-nucleotide sequence denoted <br />                                  asterisks Figure 4C. sequence 5&#226;&#8364;&#178;-TT position 421 422 displayed limited <br />                                  extension stops compared pre-quench control. thymine residues <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  showed change extension stops. fact, thymine nucleotides nineteen <br />                                  difference PQ RXN (Figure 4C, grey asterisks). data <br />                                  illustrates Hq2 derivatives modify DNA vitro, modification <br />                                  occurs types DNA bases. data line nucleoside reactivity <br />                                  studies Figure 3. <br />  <br />                                  Reactivity DNA explored vitro using 12-nucleotide oligonucleotide. <br />                                  sequence chosen self-complementary sequence 5&#226;&#8364;&#178;-GCGCAATTGCGC-3&#226;&#8364;&#178;, <br />                                  molecular weight 3646 g/mol elemental composition                                  C116H147N46O70P11. Reaction Hq2 produce addition benzetheno-group <br />                                  oligonucleotide increase molecular weight 3736 g/mol                                  elemental composition C122H149N46O71P11. difference quite large easily <br />                                  detected FT-MS. oligonucleotide annealed form dsDNA, reacted  <br />                                  Hq2 oxidant, incubated days. reaction mixture analyzed <br />                                  LC/MS determine Hq2 formed benzetheno-adduct double <br />                                  stranded DNA vitro (Figure 5). total ion chromatogram seen                                  Figure 5A, predominate ion peak 12.0 min ion peak 13.5 min.  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                           Page 10 <br />  <br />  <br />                                  MS spectrum 12.0 min peak shown Figure 5B. 12.0 min spectrum                                  large ion current m 1214, triply negatively charged unmodified <br />                                  oligonucleotide (Figure 5B). observed doubly negatively charged ion                                  unmodified oligonucleotide m value 1821. MS spectrum 13.5 min <br />                                  peak shown Figure 5C. major ion observed total ion current 13.5 min <br />                                  corresponds m value 1244, expected m triply negatively <br />                                  charged benzetheno-oligonucleotide. observed doubly negatively charged <br />                                  modified oligonucleotide m 1866. elemental composition determined <br />                                  C122H149N46O71P11 error 447 ppb (Figure S6). major ion <br />                                  observed 13.5 min FT-MS ion pair triply negatively charged <br />                                  oligonucleotide trifluoroacetic acid (TFA) m value 1282; TFA <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  ion pair unmodified oligonucleotide seen 12.0 min FT-MS m                                  1252. note modification nucleotide positions                                  dsDNA, modification location mass. result                                  significant proves Hq2 add dsDNA form lesion mass <br />                                  shown nucleoside studies. <br />  <br />                       Genetic Knockdown Nucleotide Excision Repair <br />                                  Hq derivatives modify double stranded DNA, known                                  DNA modification occurs cellular context.  turned D. <br />                                  melanogaster, powerful animal genetic model. DNA lesion <br />                                  identified vitro, reasoned nucleotide excision repair important means <br />                                  repair adduct occurring vivo.  prediction knockdown <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Ercc1, gene nucleotide excision repair pathway, sensitize Drosophila                                  oxidatively-activated agents. sensitization result limited avenues repair <br />                                  remain, low basal levels Hq activation. Furthermore, predicted                                  expedient response DNA modifying agents. DNA damaging agents,                                  reactive nature, rapidly metabolized cells plasma. example, cisplatin                                  hydrolyzed blood minutes hours active forms.45 active agents quickly <br />                                  react DNA, <span id='am-11' about='obo:IMR_0100173' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001700'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00051" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Reduced glutathione" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasExactSynonym" content="L-Glutathione" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="glutathione" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="A tripeptide with many roles in cells. It conjugates to drugs to make them more soluble for excretion, is a cofactor for some enzymes, is involved in protein disulfide bond rearrangement and reduces peroxides." datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="5-L-Glutamyl-L-cysteinylglycine" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="L-Glutathione" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0100173" datatype="xsd:string"></span>glutathione</span>, thioredoxin, protein produce irreversible adducts.46,47 <br />                                  Similar profiles observed chlorambucil, rapidly produces DNA lesions                                  minutes.48 <br />  <br />                                  step testing hypotheses, began validating Drosophila suitable <br />                                  model study agents known mechanisms action. effects agents, <br />                                  cisplatin, chlorambucil, 5-fluorouracil, examined. Specifically, conducted $watermark-text <br />  <br />  <br />  <br />  <br />                                  targeted transgenic RNAi knockdown experiment using GAL4/UAS                                  expression Ercc1 silenced. confirmed silencing Ercc1 expression da- <br />                                  GAL4 driver using RT-PCR (Figure S7). rapid induction agents&#226;&#8364;&#8482; <br />                                  mechanisms action, monitored tolerated doses agents Drosophila                                  course 7 days.  recorded differences survival relative control, <br />                                  following delivery DNA modifying agent vehicle control.35 majority <br />                                  injections healthy animals just vehicle  death noted                                  approximately 20%. attributed stress injection process                                  expected. <br />  <br />                                   examined effects cisplatin, cross-links proximal guanines create <br />                                  helix distorting adducts. cross-linking occurs, nucleotide excision repair required                                  correct adduct. fact, ERCC1 activity used prognostic marker cisplatin- <br />                                  based cancer treatments.49,50 predicted, Drosophila lacking Ercc1 expression (da- <br />                                  GAL4/UAS-Ercc1RNAi) injected cisplatin, rapid reduction survival                                  observed relative control line (da-GAL4/+) Day 1 (X2 = 5.0, P &lt; 0.025; Figure <br />                                  6A). Day 1, 40% flies expressing Ercc1RNAi died comparison  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                            Page 11 <br />  <br />  <br />                                  control flies. 40% difference Drosophila lacking Ercc1 significant compared <br />                                  injection vehicle Day 1 (X2 = 5.0, P &lt; 0.025). significant difference survival <br />                                  observed vehicle cisplatin injected Drosophila Ercc1                                  silenced. Finally, seven day period, observed reduced survival genotypes <br />                                  agent delivery consistent highly toxic nature cisplatin.  Ercc1 <br />                                  results D. melanogaster accurately replicate observations cisplatin Ercc1                                  cancer. <br />  <br />                                  confirm Drosophila-based assay sensitive DNA modifying agents <br />                                  producing bulky lesions, examined response Drosophila lacking Ercc1 expression <br />                                  chlorambucil. Chlorambucil nitrogen mustard produce bulky DNA- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  DNA cross-links, require Ercc1 repair. case chlorambucil, significant <br />                                  difference response agent observed Day 1 Drosophila lacking <br />                                  Ercc1 expression control line(X2 = 3.8, P &lt; 0.05; Figure 6B). suggests flies <br />                                  wild-type Ercc1 expression levels tolerate formation cross-links                                  Drosophila lacking Ercc1 higher sensitivity, 50% death Day 1.                                  Drosophila lacking Ercc1 expression received chlorambucil died days <br />                                  compared 80% survival wild-type Drosophila. Additionally, significant difference                                  survival flies lacking Ercc1 expression observed flies injected <br />                                  chlorambucil vehicle (X2 = 6.7, P &lt; 0.001).  significant <br />                                  survival difference observed agent vehicle wild-type flies.                                  supports model demonstrating Ercc1 vital repair bulky lesions <br />                                  induced chlorambucil. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  investigated second agent provide additional validation model                                   second agent, 5-fluorouracil, mainly repaired base excision repair, signals                                  incorrect incorporation <span id='am-61' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-62' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span>. Consequently, did expect downregulation Ercc1 <br />                                  affect mechanism action,   considered negative control. <br />                                  similar experiments conducted 5-fluorouracil, significant <br />                                  discrimination flies Ercc1 silencing controls (Figure 6C). Day 1, <br />                                  injected control Drosophila died. Similarly, Ercc1 knockdown                                  20% died 5-fluorouracil injection, Drosophila perished. <br />                                  Survival agent delivery did significantly differ injection vehicle                                  genotype.  little change mortality observed treatment 5- <br />                                  fluorouracil indicative DNA modification occurring repaired.  <br />                                  validated Drosophila, model organism, used determine $watermark-text <br />  <br />  <br />  <br />  <br />                                  agent inducing DNA damage vivo. <br />  <br />                                  Finally, investigated modification DNA oxidatively activated agents                                  repaired Drosophila model pathway involving Ercc1. Experiments similar <br />                                  using DNA-modifying agents accomplished, using Hq Hq2. <br />                                  bulky lesion produced DNA modification, unlikely Drosophila <br />                                  lacking Ercc1 expression able repair damage.  expected                                  flies exhibit greater sensitivity Hq Hq2. explored <br />                                  Hq. injection Hq, observed 10% Drosophila died                                  Day 1 controls (Figure 7A). contrast, Drosophila lacking Ercc1 expression showed <br />                                  60% death Day 1. result significant P &lt; 0.002 X2 = 5.5. Drosophila <br />                                  lacking Ercc1 expression significant reduction survival agent delivery <br />                                  relative injection vehicle (X2 = 8.5 P &lt; 0.003). results unique                                  Hq; Hq2 showed similar trends flies lacking Ercc1 showing significant <br />                                  reduction survival (50% death) Day 1 compared control line (X2 = 3.8, P &lt; 0.05; <br />                                  Figure 7B) compared delivery vehicle (X2 = 6.7, P &lt; 0.01). Wild-type <br />                                  Drosophila did differ significantly treated Hq2 versus vehicle Day 1. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 12 <br />  <br />  <br />                                  data emphasizes novel oxidatively activated agents induce DNA modification <br />                                  vivo strongly supports role DNA damaging agents highly selective <br />                                  cytotoxicity. <br />  <br />                       Potency Renal Carcinoma Cells <br />                                  sought elucidate target cell type novel agents. previous work, <br />                                  MTT assay used quantify potency types cancer cell lines. <br />                                  Interestingly, cancer cells tested displayed weak potency treated <br />                                  Hq2.24 renal cancer cells displayed low IC50 value.   assessed <br />                                  renal cells sensitive Hq2 treatment. Hq2 evaluated effects                                  viability using sulforhodamine B total protein content assay NCI Developmental <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Therapeutics Program. Seven renal cancer cell lines examined: 786-0, A498, ACHN, <br />                                  CAKI-1, RXF 393, SN12C, UO-31. Data NCI fit sigmoid                                  IC50 fitting error determined. renal cancer cells tested sensitive Hq2 <br />                                  median IC50 value 1.8 &#206;&#188;M. Importantly, cell lines display high sensitivity <br />                                  IC50 values ACHN CAKI-1 cells 360 &#194;&#177; 90 370 &#194;&#177; 40 nM, <br />                                  respectively. cell line, SN12C, displayed low potency IC50 value 21 &#194;&#177; 1.2 <br />                                  &#206;&#188;M. seven renal carcinoma cell lines IC50 values 5 &#206;&#188;M, infer <br />                                  cancer targeted oxidatively-activated agents. <br />  <br />                       CONCLUSION <br />                                  DNA modifying agents highly used anticancer treatments, non- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  selective. designed agents activation occurs elevated reactive <br />                                  oxygen species present cancers. Previously, demonstrated                                  designed agents selective certain cancer cell types. Expansion studies                                  renal cell carcinoma cell lines shows agents potency mid- <br />                                  nanomolar range. important finding research teams utilizing <br />                                  ROS-activated agents.7,51 example, boronic esters limit target effects nitrogen <br />                                  mustards protease inhibitors.52&#226;&#8364;&#8220;55 work shows renal cancer cells natural target <br />                                  approaches. manuscript, explored mechanism action                                  agents evaluation DNA lesion produced. novel oxidatively activated <br />                                  agents behave differently current DNA modifying agents. nitrogen <br />                                  mustards, simple quinone. Instead, occurs aniline ring tethered                                  quinone induces nucleophilic addition 4&#226;&#8364;&#179;-carbon DNA. DNA addition occurs <br />                                  exocylic amine bearing residue, means capable modifying $watermark-text <br />  <br />  <br />  <br />  <br />                                  DNA bases. primer extension experiment revealed modification                                  occurring, likely minor groove guanine major groove adenine                                  cytosine. end product bulky hydroxy-N2,3-benzetheno-2&#226;&#8364;&#178;-deoxyguanosine addition <br />                                  apt strong replication stop. making array lesions,                                  oxidatively activated agents expected impart strong need DNA repair.  <br />                                  DNA repair, altered cancers, essential component cytotoxic <br />                                  mechanism action novel anticancer agents. <br />  <br />                                  Drosophila model shown loss Ercc1 function plays major role repair <br />                                  bulky DNA lesions. noted assess cellular DNA modifying <br />                                  ability novel agents, validated rapid assay. fact, detection DNA <br />                                  lesions formed anticancer agents challenging process. difficulty                                  fact lesions need form impart cytotoxicity. RNAi-mediated <br />                                  silencing Ercc1 Drosophila serve simple means determine therapeutic                                  DNA damage agent. study illustrates DNA modifying agents selective, <br />                                  form highly disruptive lesions. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                             Page 13 <br />  <br />  <br />                       Supplementary Material <br />                                    Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                       Acknowledgments <br />                                    FUNDING SUPPORT <br />  <br />                                    Funding came University Cincinnati form startup funding faculty research award                                    E.J.M. graduate student research fellowship T.R.B.H. Furthermore, acknowledge NIH (Grant <br />                                    NIHRR19900) acquisition LTQ-MS used studies. Drosophila research supported grant <br />                                    National Institute Health S.M.R. (GM080592). <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                    like acknowledge Jim Deddens, John Layne, Larry Sallans technical assistance helpful <br />                                    discussions. thank Vienna Drosophila RNAi center providing Drosophila stocks. <br />  <br />  <br />                       ABBREVIATIONS <br />  <br />                                  ROS               reactive oxygen species <br />                                  dG                2&#226;&#8364;&#178;-deoxyguanosine <br />                                  Hq             4-(2-(ethyl(phenyl)amino)ethoxy)phenol <br />                                  Hq2            4,4&#226;&#8364;&#178;-(2,2&#226;&#8364;&#178;-(phenylazanediyl)bis(ethane-2,1-diyl)bis(oxy))diphenol <br />                                  NMR               nuclear magnetic resonance <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                       References <br />                                    1. Chabner B, Roberts T. Timeline - Chemotherapy war cancer. Nat Rev Cancer. 2005; <br />                                       5:65&#226;&#8364;&#8220;72. [PubMed: 15630416] <br />                                    2. Dhar S, Kolishetti N, Lippard S, Farokhzad O. Targeted delivery cisplatin prodrug safer                                       effective prostate cancer therapy vivo. Proc Natl Acad Sci USA. 2011; 108:1850&#226;&#8364;&#8220;1855. <br />                                       [PubMed: 21233423] <br />                                    3. Lovejoy, K.; Lippard, S. Dalton Trans. 2009. Non-traditional platinum compounds improved <br />                                       accumulation, oral bioavailability, tumor targeting; p. 10651-10659. <br />                                    4. Puckett C, Barton J. Targeting ruthenium complex nucleus short peptides. Bioorg Med <br />                                       Chem. 2010; 18:3564&#226;&#8364;&#8220;3569. [PubMed: 20430627] <br />                                    5. Zhilina  Ziemba  Nielsen P, Ebbinghaus S. PNA-nitrogen mustard conjugates effective <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                       suppressors 2/neu biological tools recognition PNA/DNA interactions. Bioconj <br />                                       Chem. 2006; 17:214&#226;&#8364;&#8220;222. <br />                                    6. Trachootham D, Alexandre J, Huang P. Targeting cancer cells ROS-mediated mechanisms:                                       radical therapeutic approach? Nat Rev Drug Discov. 2009; 8:579&#226;&#8364;&#8220;591. [PubMed: 19478820] <br />                                    7. Pelicano H, Carney D, Huang P. ROS stress cancer cells therapeutic implications. Drug <br />                                       Resist Updat. 2004; 7:97&#226;&#8364;&#8220;110. [PubMed: 15158766] <br />                                    8. Sudarshan S, Sourbier C, Kong HS, Block K, Romero VAV, Yang Y, Galindo C, Mollapour M, <br />                                       Scroggins B, Goode N, Lee MJ, Gourlay CW, Trepel J, Linehan WM, Neckers L. Fumarate <br />                                       hydratase deficiency renal cancer induces glycolytic addiction hypoxia-inducible <br />                                       transcription factor 1 stabilization glucose-dependent generation reactive oxygen species. Mol <br />                                       Cell Biol. 2009; 29:4080&#226;&#8364;&#8220;4090. [PubMed: 19470762] <br />                                    9. Wittgen H, van Kempen L. Reactive oxygen species melanoma therapeutic implications. <br />                                       Melanoma Res. 2007; 17:400&#226;&#8364;&#8220;409. [PubMed: 17992124] <br />                                    10. Hole P, Darley R, Tonks  reactive oxygen species play role myeloid leukemias? Blood. <br />                                         2011; 117:5816&#226;&#8364;&#8220;5826. [PubMed: 21398578] <br />                                    11. Cadet J, Douki T, Ravanat J. Oxidatively generated base damage cellular DNA. Free Radic Biol <br />                                         Med. 2010; 49:9&#226;&#8364;&#8220;21. [PubMed: 20363317] <br />  <br />  <br />  <br />                                       Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                 Page 14 <br />  <br />  <br />                                  12. Solivio MJ, Nemera DB, Sallans L, Merino EJ. Biologically relevant oxidants cause bound <br />                                      proteins readily oxidatively cross-link guanine. Chem Res Toxicol. 2012; 25:326&#226;&#8364;&#8220;336. <br />                                      [PubMed: 22216745] <br />                                  13. Alp O, Zhang Y, Merino EJ, Caruso JA. Selenium effects arsenic cytotoxicity protein <br />                                      phosphorylation human kidney cells using chip-based nanoLC-MS/MS. Metallomics. 2011; <br />                                      3:482&#226;&#8364;&#8220;490. [PubMed: 21479294] <br />                                  14. Valko M, Rhodes C, Moncol J, Izakovic M, Mazur M. Free radicals, metals antioxidants                                      oxidative stress-induced cancer. Chem Biol Interact. 2006; 160:1&#226;&#8364;&#8220;40. [PubMed: 16430879] <br />                                  15. Adhikary  Becker D, Collins S, Koppen J, Sevilla M. C5&#226;&#8364;&#178; - C3&#226;&#8364;&#178;-sugar radicals produced                                      photo-excitation electron oxidized adenine 2&#226;&#8364;&#178;-deoxyadenosine derivatives. <br />                                      Nucleic Acids Res. 2006; 34:1501&#226;&#8364;&#8220;1511. [PubMed: 16537838] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  16. Murtas D, Piras F, Minerba L, Ugalde J, Floris C, Maxia C, Demurtas P, Perra M, Sirigu P. <br />                                      Nuclear 8-hydroxy-2&#226;&#8364;&#178;-deoxyguanosine survival biomarker patients cutaneous <br />                                      melanoma. Oncol Rep. 2010; 23:329&#226;&#8364;&#8220;335. [PubMed: 20043092] <br />                                  17. Loft S, Poulsen H. Cancer risk oxidative DNA damage man. J Mol Med. 1996; 74:297&#226;&#8364;&#8220;312. <br />                                      [PubMed: 8862511] <br />                                  18. Vafa O, Wade M, Kern S, Beeche M, Pandita T, Hampton G, Wahl G. c-Myc induce DNA <br />                                      damage, increase reactive oxygen species, mitigate p53 function: mechanism oncogene- <br />                                      induced genetic instability. Mol Cell. 2002; 9:1031&#226;&#8364;&#8220;1044. [PubMed: 12049739] <br />                                  19. Hu Y, Lu W, Chen G, Wang P, Chen  Zhou Y, Ogasawara M, Trachootham D, Feng L, Pelicano <br />                                      H, Chiao PJ, Keating MJ, Garcia-Manero G, Huang P. K-ras(G12V) transformation leads                                      mitochondrial dysfunction metabolic switch oxidative phosphorylation glycolysis. <br />                                      Cell Res. 2012; 22:399&#226;&#8364;&#8220;412. [PubMed: 21876558] <br />                                  20. Hole PS, Pearn L, Tonks AJ, James PE, Burnett AK, Darley RL, Tonks  Ras-induced reactive <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      oxygen species promote growth factor-independent proliferation human CD34(+) hematopoietic <br />                                      progenitor cells. Blood. 2010; 115:1238&#226;&#8364;&#8220;1246. [PubMed: 20007804] <br />                                  21. Jourden JLM, Cohen SM. Hydrogen peroxide activated matrix metalloproteinase inhibitors:                                      Prodrug Approach. Angew Chem Int Edit. 2010; 49:6795&#226;&#8364;&#8220;6797. <br />                                  22. Kuang Y, Balakrishnan K, Gandhi V, Peng X. Hydrogen peroxide inducible DNA cross-linking <br />                                      agents: targeted anticancer prodrugs. J Chem Soc. 2011; 133:19278&#226;&#8364;&#8220;19281. [PubMed: <br />                                      22035519] <br />                                  23. Hagen H, Marzenell P, Jentzsch E, Wenz F, Veldwijk MR, Mokhir  Aminoferrocene-based <br />                                      prodrugs activated reactive oxygen species. J Med Chem. 2012; 55:924&#226;&#8364;&#8220;934. [PubMed: <br />                                      22185340] <br />                                  24. Li G, Bell T, Merino EJ. Oxidatively activated DNA-modifying agents selective cytotoxicity. <br />                                      ChemMedChem. 2011; 6:869&#226;&#8364;&#8220;875. [PubMed: 21374823] <br />                                  25. Richgels PK, Rollmann SM. Genetic variation odorant receptors contributes variation $watermark-text <br />  <br />  <br />  <br />  <br />                                      olfactory behavior natural population Drosophila melanogaster. Chem Senses. 2012; <br />                                      37:229&#226;&#8364;&#8220;240. [PubMed: 22038943] <br />                                  26. Bier E. Drosophila, golden bug, emerges tool human genetics. Nat Rev Gen. 2005; 6:9&#226;&#8364;&#8220; <br />                                      23. <br />                                  27. Bell  McBride S, Dockendorff T. Flies ointment Drosophila modeling enhance drug <br />                                      discovery. Fly. 2009; 3:39&#226;&#8364;&#8220;49. [PubMed: 19164936] <br />                                  28. Das T, Cagan R. Drosophila novel therapeutic discovery tool thyroid cancer. Thyroid. <br />                                      2010; 20:689&#226;&#8364;&#8220;695. [PubMed: 20578898] <br />                                  29. Kislukhin G, Murphy ML, Jafari M, Long AD. Chemotherapy-induced toxicity highly heritable <br />                                      Drosophila melanogaster. Pharmacogenet Genom. 2012; 22:285&#226;&#8364;&#8220;289. <br />                                  30. Sekelsky J, Brodsky M, Burtis K. DNA repair Drosophila: Insights Drosophila genome <br />                                      sequence. J Cell Biol. 2000; 150:F31&#226;&#8364;&#8220;F36. [PubMed: 10908583] <br />                                  31. Pandey UB, Nichols CD. Human disease models Drosophila melanogaster role fly <br />                                      therapeutic drug discovery. Pharmacol Rev. 2011; 63:411&#226;&#8364;&#8220;436. [PubMed: 21415126] <br />                                  32. Reiter L, Potocki L, Chien S, Gribskov M, Bier E. systematic analysis human disease- <br />                                      associated gene sequences Drosophila melanogaster. Genome Res. 2001; 11:1114&#226;&#8364;&#8220;1125. <br />                                      [PubMed: 11381037] <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                                   Page 15 <br />  <br />  <br />                                  33. Pagliarini R, Quinones  Xu T. Analyzing function tumor suppressor genes using                                      Drosophila model. Meth Mol Biol. 2003; 223:349&#226;&#8364;&#8220;382. <br />                                  34. Laat WL, Jaspers NG, Hoeijmakers JH. Molecular mechanism nucleotide excision repair. <br />                                      Genes Dev. 1999; 13:768&#226;&#8364;&#8220;785. [PubMed: 10197977] <br />                                  35. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, Fellner M, Gasser B, Kinsey K, Oppel S, <br />                                      Scheiblauer S, Couto  Marra V, Keleman K, Dickson BJ. genome-wide transgenic RNAi <br />                                      library conditional gene inactivation Drosophila. Nature. 2007; 448:151&#226;&#8364;&#8220;156. [PubMed: <br />                                      17625558] <br />                                  36. Aboussekhra  Biggerstaff M, Shivji MK, Vilpo JA, Moncollin V, Podust VN, Proti&#196;&#8225; M, <br />                                      H&#195;&#188;bscher U, Egly JM, Wood RD. Mammalian DNA nucleotide excision repair reconstituted                                      purified protein components. Cell. 1995; 80:859&#226;&#8364;&#8220;868. [PubMed: 7697716] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  37. Solivio MJ, Joy TJ, Sallans L, Merino EJ. Copper generated reactive oxygen leads formation                                      lysine-DNA adducts. J Inorg Biochem. 2010; 104:1000&#226;&#8364;&#8220;1005. [PubMed: 20684045] <br />                                  38. Martin  Jones MA, Grotewiel M. Manipulation Sod1 expression ubiquitously,                                      nervous muscle, impacts age-related parameters Drosophila. FEBS Lett. 2009; <br />                                      583:2308&#226;&#8364;&#8220;2314. [PubMed: 19540235] <br />                                  39. Monks  Scudiero D, Skehan P, Shoemaker R, Paull K, Vistica D, Hose C, Langley J, Cronise P, <br />                                      Vaigro-Wolff  Feasibility high-flux anticancer drug screen using diverse panel cultured <br />                                      human tumor cell lines. J Natl Cancer Inst. 1991; 83:757&#226;&#8364;&#8220;766. [PubMed: 2041050] <br />                                  40. Cooke MS, Loft S, Olinski R, Evans MD, Bialkowski K, Wagner JR, Dedon PC, M&#195;&#184;ller P, <br />                                      Greenberg MM, Cadet J. Recommendations standardized description nomenclature <br />                                      concerning oxidatively damaged nucleobases DNA. Chem Res Toxicol. 2010; 23:705&#226;&#8364;&#8220;707. <br />                                      [PubMed: 20235554] <br />                                  41. Chenna  Singer B. Synthesis benzene metabolite adduct, 3&#226;&#8364;&#179;-Hydroxy-1, N2-benzetheno-2&#226;&#8364;&#178;- <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                      deoxyguanosine, site-specific incorporation DNA oligonucleotides. Chem Res <br />                                      Toxicol. 1997; 10:165&#226;&#8364;&#8220;171. [PubMed: 9049427] <br />                                  42. Munk BH, Burrows CJ, Schlegel HB. Exploration mechanisms transformation 8- <br />                                      hydroxy guanine radical FAPyG density functional theory. Chem Res Toxicol. 2007; <br />                                      20:432&#226;&#8364;&#8220;444. [PubMed: 17316026] <br />                                  43. Jowa L, Witz G, Snyder R, Winkle S, Kalf GF. Synthesis characterization deoxyguanosine- <br />                                      benzoquinone adducts. J Appl Toxicol. 1990; 10:47&#226;&#8364;&#8220;54. [PubMed: 2335711] <br />                                  44. Khazanchi R, Yu P, Johnson F. N-2,3-Etheno-2&#226;&#8364;&#179;-deoxyguanosine [8,9-dihydro-9-oxo-2-&#226;&#8364;?deoxy-3- <br />                                      beta-D-ribofuranosylimidazo[2,1-B]purine] - practical synthesis characterization. J Org <br />                                      Chem. 1993; 58:2552&#226;&#8364;&#8220;2556. <br />                                  45. Verschraagen M, van der Born K, Zwiers T, van der Vijgh W. Simultaneous determination                                      intact cisplatin metabolite monohydrated cisplatin human plasma. J Chromatog B-Analyt <br />                                      Technol Biomed Life Sci. 2002; 772:273&#226;&#8364;&#8220;281. [PubMed: 12007772] <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  46. Vermorken JB, van der Vijgh WJ, Klein  Hart AA, Gall  Pinedo HM. Pharmacokinetics                                      free total platinum species short-term infusion cisplatin. Cancer Treat Rep. 1984; <br />                                      68:505&#226;&#8364;&#8220;513. [PubMed: 6538459] <br />                                  47. Paz MM, Zhang X, Lu J, Holmgren  new mechanism action anticancer drug <br />                                      mitomycin C: mechanism-based inhibition thioredoxin reductase. Chem Res Toxicol. 2012; <br />                                      25:1502&#226;&#8364;&#8220;1511. [PubMed: 22694104] <br />                                  48. Bank BB, Kanganis D, Liebes LF, Silber R. Chlorambucil pharmacokinetics DNA binding                                      chronic lymphocytic leukemia lymphocytes. Cancer Res. 1989; 49:554&#226;&#8364;&#8220;559. [PubMed: 2910477] <br />                                  49. Besse B, Massard C, Haddad V, Andre F, Dunant  Pirker R, Olaussen KA, Brambilla E, Fouret <br />                                      P, Soria JC. ERCC1 influence incidence brain metastases patients non-squamous <br />                                      NSCLC treated adjuvant cisplatin-based chemotherapy. Ann Oncol. 2011; 22:575&#226;&#8364;&#8220;581. <br />                                      [PubMed: 20801905] <br />                                  50. Arora S, Kothandapani  Tillison K, Kalman-Maltese V, Patrick SM. Downregulation XPF- <br />                                      ERCC1 enhances cisplatin efficacy cancer cells. DNA Repair (Amst). 2010; 9:745&#226;&#8364;&#8220;753. <br />                                      [PubMed: 20418188] <br />                                  51. Peng X, Gandhi V. ROS-activated anticancer prodrugs: new strategy tumor-specific damage. <br />                                      Ther Del. 2012; 3:823&#226;&#8364;&#8220;833. <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                             Page 16 <br />  <br />  <br />                                  52. Kuang Y, Balakrishnan K, Gandhi V, Peng X. Hydrogen Peroxide Inducible DNA Cross-Linking <br />                                      Agents: Targeted Anticancer Prodrugs. J Chem Soc. 2011; 133:19278&#226;&#8364;&#8220;19281. [PubMed: <br />                                      22035519] <br />                                  53. Major Jourden JL, Cohen SM. Hydrogen Peroxide Activated Matrix Metalloproteinase Inhibitors: <br />                                      Prodrug Approach. Angew Chem Int Ed Engl. 2010; 49:6795&#226;&#8364;&#8220;6797. [PubMed: 20715043] <br />                                  54. Hagen H, Marzenell P, Jentzsch E, Wenz F, Veldwijk MR, Mokhir  Aminoferrocene-Based <br />                                      Prodrugs Activated Reactive Oxygen Species. J Med Chem. 2012; 55:924&#226;&#8364;&#8220;934. [PubMed: <br />                                      22185340] <br />                                  55. Cao S, Wang Y, Peng X. ROS-Inducible DNA Cross-Linking Agent New Anticancer Prodrug <br />                                      Building Block. Chem Eur J. 2012; 18:3850&#226;&#8364;&#8220;3854. [PubMed: 22378607] <br /> $watermark-text <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                              Page 17 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Figure 1. Identification Determinants Selective Activation Oxidatively-Activated <br />                                  Agent <br />                                    Structure oxidatively activated agents Hq Hq2. ROS present, <br />                                  Hq activated, leading addition phenol guanine. seek determine <br />                                  structure dependence damage. (B) Genes responsible producing proteins involved <br />                                  DNA repair targeted using RNAi-mediated silencing. Delivery known DNA <br />                                  modifying agents lead selective death D. melanogaster model. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                   Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                          Page 18 <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Figure 2. Hq Derivatives React Guanine Differently Benzoquinone <br />                                    Quinones react paths: C4&#226;&#8364;&#179; addition, black, C2&#226;&#8364;&#179;-Michael addition, grey. <br />                                  product path shown. determine reaction path, product                                  oxidation methanol isolated. NMR MS shown. Oxidative activation <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Hq methanol proceeds C4&#226;&#8364;&#179; addition based grey NMR assignments                                  integrals. (B)  NMR reaction Hq, 2&#226;&#8364;&#178;-deoxyguanosine, iridium <br />                                  oxidant phosphate buffer. left reaction assigned NMR resonances <br />                                  superimposed product. right H1-NMR 4&#226;&#8364;&#8220;10 ppm.                                  assignments grey. N1-proton signal observed.  reaction 2&#226;&#8364;&#178;- <br />                                  deoxyguanosine, base, benzoquinone according literature. left reaction <br />                                  assignments. right H1-NMR. Importantly, N1 peak 10ppm <br />                                  missing benzoquinone reaction product NMR. different reaction path  <br />                                  Hq leads different guanine lesion. <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Chem Res Toxicol. Author manuscript; available PMC 2013 November 19. <br />                    Jones et al.                                                                                       Page 19 <br /> $watermark-text <br /> $watermark-text <br />  <br />  <br />  <br />  <br />                                  Fi <br /> </body></html>