<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>           REB1 Site Essential Component Terminator <br />               RNA Polymerase Saccharomyces cerevisiae <br />  <br />               identified terminator transcription RNA polymerase genes coding rRNA            yeast Saccharomyces cerevisiae. terminator located 108 bp downstream 3&apos; end mature 25S <br />            rRNA shares characteristics previously studied polymerase terminators vertebrates. <br />            example, yeast terminator orientation dependent, inhibited sequence, forms RNA <br />            3&apos; ends 17 2 bp upstream essential protein binding site. recognition sequence binding            previously cloned REB1 protein (Q. Ju, B. E. Morrow, J. R. Warner, Mol. Cell. Biol. 10:5226-5234, 1990) <br />            essential component terminator. addition, efficiency termination depends sequence <br />            context extending 12 bp upstream REB1 site. <br />  <br />    Studies mouse frog systems shown                  includes binding site REB1, essential yeast protein termination transcription RNA polymerase                   previously cloned (11, 23, 24). <br /> requires binding termination protein specific <br /> sequence DNA template. mice, protein called TTF1, specific binding site 18-bp-long                            MATERIALS METHODS <br /> element called Sal box (10)  element fortuitously                  Plasmids. yeast rRNA genes (rDNA) used contains SalI restriction site). frogs, binding               study derived originally SmaI repeat unit protein christened Rib2 (21) binds site          cloned pBD4 (1). <br /> GACTTGCNC (19). systems, terminator se-                        termination test vector constructed arranging quence cause termination situated                 serial order ribosomal gene promoter, polylinker  normal orientation ceases function reversed.                insertion sequences tested termination activity), <br /> Partially purified TllF1 stimulates termination added              series unique restriction sites linearizing crude extract, termination abolished                template. construct diagrammed Fig. 1A excess oligonucleotide containing Sal box sequence            follows. 1,078-bp fragment rDNA (extending added reaction. Likewise, partially purified Rib2             SmaI site 220 bp upstream transcription initiation <br /> causes specific DNase footprints terminator                  XbaI site 858 bp downstream initiation) inserted <br /> element, footprinting termination abolished            polylinker pBluescript SK+ (Stratagene). competition oligonucleotides containing termi-              XbaI site, NdeI site rDNA portion, nator sequence. progress understanding                   destroyed cutting, filling  religation. mechanism termination hampered inabil-               rDNA fragment excised cutting SmaI ity far completely purify Tl F1 Rib2              cutting SstI site pBluescript polylinker, clone relevant cDNAs coding proteins.                     fragment inserted pUC18.  pUC18 recipient <br />    hope circumvent problems             cut EcoRI, end filled blunt  vertebrate systems, begun study termination               recut SstI; insertion rDNA fragment <br /> RNA polymerase yeast Saccharomyces cerevisiae.                 destroyed SmaI site upstream rDNA promoter.) yeasts offer potential advantages             shown Fig. 1, final construct polylinker <br /> vertebrates:  easier obtain large amounts         located 900 bp downstream transcription initi- <br /> extract needed biochemical purification, second,            ation site. <br /> possibility using developed genetics yeasts              Fragments intergenic spacer tested 3&apos;-end- <br /> supplement replace biochemical approaches exists.                forming ability insertion various restriction sites <br /> recent development yeast extracts capable supporting              polylinker. StuI-HpaI (positions -245 accurate initiation RNA polymerase vitro (20, 27, 28)           +309), AccI-HpaI (positions -36 +309), EcoRI-HpaI offers opportunity develop vitro assays         (positions +2 +309), EcoRI-PvuII (positions +2 +871), <br /> DNA protein elements involved termination.                       EcoRV-SmaI (positions +1955 +2162) fragments <br />    paper, yeast cell extract             blunt ended inserted orientation <br /> developed Schultz et al. (28) capable supporting               SmaI site polylinker (Fig. 1A, position +897 <br /> RNA 3&apos;-end formation RNA polymerase transcripts.                   termination test vector). Nucleotides numbered sites features similar previously               according used Fig. 1B 2A, characterized vertebrate polymerase terminators.              reflects actual sequences inserted test vector.  fine-scale mutagenesis indicates terminator <br />                                                                         Position +1 designates G residue EcoRI site <br />                                                                         adjacent REB1 protein binding site. <br />                                                                            Transcription assays. Transcription extracts pre- <br />                                                                         pared  multiply protease-deficient strain <br />   * <br />       Corresponding author.                                             BJ2168 strain containing rpo4l mutation)                                                                   649 <br />  650        LANG REEDER                                                                                                                                                                                                                 MOL. CELL. BIOL. <br />  <br />                                   Test Site                                             B. <br />                                 (polylirnker) <br />                          Noat Asp718 SmiaI Barnl-11 XbZa                                      .::d <br />                                                                                         -&apos;e n:l                                             LO -&apos;                                                                                                       8E9 893      897 S02       S08 <br />                                                                                              :GT T   T -T    r7 <br />                                                                                                                                               C ._ <br />                                                                                                                     T C -T T C TAAG T 2;GG T Ar.;:   &apos; <br />                                                                                                                                                          ` : -G.r- :-&quot;:.R::&apos;&apos;- .: -.&apos;:   ::;&apos;.    ............... <br />                                                                                                                                                                                                              .^ <br />                                                                                                                                                                                                                                 &apos;-4 &apos;:&apos; -&apos;:&apos; <br />                                                                                                                                                                                                                         ,................ <br />  <br />       (Sm<span id='am-10' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>;nI)                                   NdeI              S-pI    ScaI <br />        -220                                    11503            1I06    1890          EC    RT <br />  <br />  <br />                                                                                                                        REB: <br />                                        _   -1 <br />                                             -   -1.      _.- <br />               o -n3t                                                             -.-1 <br />                                                                                                                                                                                                                                ~~~~~~~~~~c2c -(x;L~~~ <br />                                                                                                                                                                                                                                                          .. ^    5A <br />          p.rorr,. oe-r     5&apos;end 35S <br />  <br />  <br />  <br />  <br />                                                                                         TOC CC        zA      TT:    :TG CT     T C.- <br />                                                                                                                               TCA                                                                                                                                    G-    -t <br />                                                                                                                                       ..CCAA <br />  <br />  <br />                                                                                                       Hpa T <br />                                                                                         c:&quot;TA TAG :LTAA&apos;; <br />  <br />  <br />  <br />                                                 wt           rpo 4                                                                FIG. 1. Specific initiation RNA polymerase linear <br />                                                                                                                                template.   Structure termination assay template. (B) <br />                                            12        3445 6                                                                    Sequence intergenic spacer 3&apos; end 25S rRNA                                                                                                                                HpaI site. designate G residue EcoRI site                                                                                                                                + 1. binding site REB2 consensus site REB1                                                                                                                                underlined. (C) Transcription linear templates showing specific <br />                                                                                                                                initiation. Templates linearized Ndel (lanes 1                                                                           .: <br />                                                                            r <br />                                                                                                                                4), SspI (lanes 2 5), ScaI (lanes 3 6). Extracts                                                                                                                                wild-type strain (BJ2168) strain deficient                                                                                                                                mitochondrial polymerase (rpo4l). Sizes runoff transcripts <br />                                                                                                                                estimated using coelectrophoresed DNA markers. <br />  <br />  <br />                                                 -)       r <br />                                                          - <br />  <br />  <br />  <br />                                                                                                                                Tris-HCl (pH 8.0), 50 mM KCI, 1 mM EDTA, 0.5 pug                                                                                                                                poly(dI-dC), 5% glycerol, 10 ng (8,000 cpm) oligonucle- <br />                                                                                                                                otide probe, 1 ,ul recombinant REB1 protein final <br />                                                                                                                                volume 10 ,ul. Samples incubated ice 20 min, <br />                                                      U) <br />                                                                                                                                loaded, electrophoresed 5% polyacrylamide gels <br />                                                                           l,&gt;lfjr <br />                                                                                                                                (80:1, acrylamide bis) 20 mM Tris-HCl (pH 8.0) mM <br />                                                                                                                                EDTA. buffer changed prior loading                                                                                                                                1 h. <br />                                                      HH        Hi    Hi <br />  <br />  <br />                                             } tn           D)      tJ <br />  <br />  <br />                                                                                                                                                                                                      RESULTS <br />                                                                                                                                  Establishment vitro termination assay RNA <br />                                                                                                                                polymerase  template containing yeast ribosomal gene <br /> described previously (28), cells ground                                                                   promoter convenient unique restriction sites located <br /> minimum achieve 90% breakage. extensive                                                                         downstream constructed (Fig. 1A). Aliquots grinding resulted higher background runoff assay.                                                                    template cut NdeI, SspI, ScaI site Protein concentrations extracts range 15                                                                 transcribed vitro, labeled runoff RNA subjected 25 mg/ml assayed Bradford reaction (2).                                                                           gel electrophoresis. shown Fig. 1C, RNA   Templates runoff transcription linearized                                                                    generated various reactions correct length NdeI site unless stated  Typical reactions                                                                      extend initiation site gene promoter run room temperature volume 40 ,ul containing 20                                                                    unique restriction site. previously shown mM HEPES (N-2-hydroxyethylpiperazine-N&apos;-2-ethanesul-                                                                           extract supports specific initiation RNA polymerase fonic acid)-KOH (pH 7.9), 50 mM KCl, 5 mM EGTA                                                                                 circular templates (5, 28). data Fig. 1C support [ethylene glycol-bis(,B-aminoethyl ether)-N,N,N&apos;,N&apos;-tetra-                                                                     conclusion specific initiation ribosomal gene <br /> <span id='am-41' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span>acetic acid</span>], 0.05 mM EDTA, 2.5 mM dithiothreitol, 10 ,ug                                                                   promoter occurring linear templates. amanitin ml, 10% glycerol, 0.5 mM   <span id='am-17' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>, <span id='am-1' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span>GTP</span>,                                                                         transcription reactions described paper <span id='am-36' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-37' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span>UTP</span>, 0.1 mM <span id='am-46' about='obo:IMR_0200202' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00063" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200202" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="CTP" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CTP</span>, 10 ,uCi 32P-CTP (800 Ci/mmol),                                                                       contained oa-amanitin eliminate transcription 10 pl extract, 25 ng template. Reactions                                                                        RNA polymerase II. Previous analysis DNA fragment <br /> stopped adding 250 ptl 7 M <span id='am-51' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-52' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span>urea</span>-100 mM LiCl-0.5%                                                                            used promoter cassette (5, 28) showed contains sodium dodecyl sulfate-10 mM EDTA-10 mM Tris-HCl (pH                                                                           promoters RNA polymerase III. Furthermore, sig- <br /> 8.0) extracted twice phenol-chloroform.                                                                          nals insensitive Tagetin (Epicentre Technologies, <br /> tRNA carrier (10 p,g) added reaction,                                                                          Madison, Wis.), specific inhibitor RNA polymerase III <br /> nucleic acids precipitated 2.5 volumes ethanol.                                                                   (data shown). possible interfering RNA <br /> Pellets resuspended mixture 5 pul 98%                                                                          polymerase activity mitochondrial polymerase. cryptic <br /> formamide, 20 mM EDTA, 0.025% (wt/vol)                                                                                promoter mitochondrial RNA polymerase located xylene cyanole bromphenol blue, boiled 3 min,                                                                      position -75 relative initiation site RNA <br /> loaded 3% polyacrylamide gels containing 7 M urea.                                                                        polymerase (5, 27), level resolution gel <br />    Gel retardation assays. reaction contained 20 mM                                                                       shown Fig. 1C, RNA initiated position -75  <br />   FIG. 2. Survey intergenic spacer RNA 3&apos;-end-forming sites.   Map intergenic spacer showing location fragments  tested. (B) Survey 3&apos;-end-forming sites StuI site (position -248) HpaI site (position +307). lane 1, StuI-HpaI fragment inserted SmaI site test vector transcribed linearization downstream NdeI site. lane 2, AccI-HpaI fragment inserted test site. lane 3, EcoRI-HpaI fragment tested. locations runoff <br /> RNA bands attributable 3&apos; end 25S rRNA, 3&apos;-end formation sites  II, III, length runoff indicated. (C) <br /> Orientation dependence 3&apos;-end-forming sites. EcoRI-HpaI fragmnent inserted test site forward (lane 1) reverse (lane 2) orientation tested runoff transcription. Likewise, EcoRV-SmaI fragment tested forward (lane 3) reverse (lane 4) orientation, EcoRI-PvuII fragment tested forward (lane 5) reverse (lane 6) orientation. locations runoff RNA bands attributable 3&apos;-end formation sites  II, III, IV indicated. <br />  <br />  <br /> possibly confused RNA initiated position + 1.                                            tional, shorter runoff products indicative RNA 3&apos;-end  mitochondrial polymerase makes little,                                              formation. <br /> contribution runoff signal seen assay                                            Survey ribosomal gene spacer RNA 3&apos;-end- <br /> Figure 1C, lanes 1 3, shows runoff signal                                               forming sites. began inserting test site generated cell extract wild-type yeast                                         550-bp StuI-HpaI fragment extends 25S <br /> cells. Lanes 4 6 transcripts generated                                        rRNA coding region point downstream ribosomal truncated templates utilizing extract                                           gene enhancer.  sequence relevant portion rpo4l strain lacks mitochondrial RNA polymerase                                               fragment shown Fig. 1B; location fragment <br /> activity (9). relative size intensity runoff                                         spacer shown Fig. 2A.) shown Fig. 2B, <br /> RNAs essentially identical cases. conclude,                                          lane 1, StuI-HpaI fragment caused appearance  RNA signals generated assay                                             shorter RNAs runoff assay. shortest RNA attributable specific initiation RNA poly-                                         length suggesting 3&apos; end corresponds 3&apos; <br /> merase                                                                                            end 25S rRNA itself. workers presented <br />    assay RNA 3&apos;-end-forming elements, polylinker                                           evidence 3&apos; end formed processing (26, 32), inserted template, locating unique SmaI                                         experiments agree conclusion (Fig. 3). <br /> site 897 bp downstream position + 1 initiation site.                                       3&apos; ends RNAs formed sites  II, <br /> Various DNA fragments inserted SmaI                                             III, numbered order increasing <br /> site capable generating addi-                                           distance ribosomal gene promoter. <br />  652     LANG REEDER                                                                                                  MOL. CELL. BIOL. <br />  <br />                         1 2 3 4 5 6                                      survey shown Fig. 2C. Comparing lanes 1 2, <br />                                                                          observed sites II III function orienta- <br />                                                                          tion  site II function decreases somewhat            run -                                                     orientation reversed).  3&apos;-end-forming ability <br />                                                                          site eliminated reversing sequence. Lanes 5                              V            -. run                   6 essentially result longer test <br />                                                                       fragment used.   lane 6, site II clearly <br />                                                                        resolved band attributable site III.) Comparison <br />             3&apos;25S-               3;     4 ,-                           lanes 3 4 shows site IV functions                                                                          orientation. site shows complete orientation <br />                                                                          dependence, site prime candidate                                                                          terminator RNA polymerase transcription. <br />                                                                             RNA processing activity detected site  II, III. <br />                                                                          Figure 3, lane 2, shows typical experiment                                                                          StuI-HpaI fragment inserted test site tran- <br />    FIG. 3. Lack behavior sites  II, III RNA process-      scription allowed proceed 20 min. 3&apos; ends attrib- <br /> ing sites. lanes 1 3, test vector contained StuI-HpaI      utable end 25S rRNA sites  II, fragment. lanes 4 6, EcoRI-HpaI fragment tested.       III clearly visible. Figure 3, lane 3, shows parallel <br /> lanes 2 5, transcription ran 20 min isolation RNA.     reaction transcription allowed proceed lanes 3 6, transcription ran 20 min, actinomycin D added 20 ,ug/ml stop transcription, incubation contin-     20 min, time actinomycin D added stop <br /> ued additional 20 min RNA isolated. lanes 1        transcription incubation continued additional 4, actinomycin D added 20 pLg/ml transcription         20-min chase. chase, formation 25S 3&apos; end initiated triphosphates. Note accumulation site     continued occur (band intensity increased 60%, mea-  II, III bands runoff bands require continued transcription,   sured densitometry). contrast, intensity of accumulation 3&apos; end 25S continues tran-        bands decreased 50%. Yip Holland (32) <br /> scription stopped.                                                    previously shown RNA transcribed SP6 <br />                                                                          polymerase processed vitro yield 3&apos; end                                                                          25S rRNA, providing strong evidence    processing site 3&apos; end 25S rRNA               location harbors processing site terminator active  vivo vitro) easier study            transcription. chase experiment shown Fig. 3, lanes 2 3&apos;-end-forming sites spacer 3&apos; end            3, good agreement conclusion. <br /> 25S eliminated. accomplished inserting                 examine behavior sites  II, III AccI-HpaI fragment test site. shown Fig.              closely, inserted EcoRI-HpaI fragment test <br /> 2B, lane 2, 3&apos; end 25S eliminated, site             site performed actinomycin D chase experi- predominant 3&apos;-end-forming site, sites II                ment. 20 min transcription, 3&apos; ends attributable III clearly detected. typical vitro assay         sites  II, III clearly visible (Fig. 3, lane 5), conditions, 70% total transcripts end site             20 min chase absence transcription, AccI-HpaI fragment assayed. Figure 2B, lane 3,               intensities decreased 20% (compare lanes 5 <br /> shows result obtained EcoRI-HpaI fragment                6). conclude sites  II, III require continued <br /> inserted test site. Shorter RNAs attributable sites          RNA transcription order form 3&apos; ends. respect,  II, III appear seen previously AccI-HpaI              behavior expected termination sites, fragment (compare lane 2 lane 3), case            contrast 3&apos; end 25S rRNA, shorter fragment, 25% total transcripts            behavior expected processing site. <br /> attributable site                                                      Site sensitive competition DNA sequence. <br />    Fig. 2C, survey extended parts              distinguishing characteristic vertebrate poly- <br /> intergenic spacer. Lane 1 shows repeated experiment                 merase terminators require binding EcoRI-HpaI fragment inserted test                 sequence-specific protein order cause termination (10, <br /> site. Lane 5 shows results inserting larger EcoRI-              21).  termination mouse frog <br /> PvuII fragment.  RNAs attributable sites  II,            systems highly sensitive competition caused adding III appear. Finally, lane 3 shows result inserting            excess DNA fragment containing terminator EcoRV-SmaI fragment test site. frag-               sequence. tested yeast sites  II, III ment, new truncated RNA appears, indicating                    similar sensitivity competition. RNA 3&apos;-end-forming location, numbered site                 shown Fig. 4A, addition increasing amounts IV.                                                                      EcoRI-HpaI fragment  contains sites  II, III) <br />    relative locations of 3&apos;-end-forming sites          causes rapid loss 3&apos;-end formation site  Site II identified survey summarized              sensitive competition, site III completely resistant Fig. 2A. survey omits 1,100 bp intergenic                 range competitor. <br /> spacer  PvuII site EcoRV site). Apparently                  sensitivity site competition examined element region unstable vectors            closely Fig. 4B. experiment, competitor using, recovered rearranged inserts               polymerized 30-bp double-stranded oligonucleotide  repeated attempts subclone region.                            positions 7 30 EcoRI-HpaI fragment shown    site orientation dependent. RNA polymerase                Fig. 1B). 3&apos;-end formation site specifically rapidly <br /> terminators frogs mice function                    inhibited oligonucleotide, sites II III positioned forward orientation (10, 18).             unaffected. <br /> surveyed yeast sites  II, III, IV              conclude  RNA 3&apos;-end-forming sites similar orientation dependence. results               identified intergenic spacer, site  <br />  <br />  <br />   FIG. 4. Sensitivity sites  II, III competition       FIG. 5. 5&apos; 3&apos; deletion mapping sites  II, III.   sequence. Test vector (25 ng) containing EcoRI-HpaI                Assay deletions EcoRI-HpaI fragment. lane 1, NotI <br /> fragment added transcription reaction.   Competition         adaptors added EcoRI-HpaI fragment,   EcoRI-HpaI fragment. following amounts gel-              inserted NotI site position 869 test vector <br /> purified restriction fragment added reaction           polylinker. procedure lanes 2, 3, 4 transcription began: (lane 1), 50 ng (lane 2), 100 ng (lane 3), 200   lane 1 novel XbaI site introduced positions 87, 144, <br /> ng (lane 4), 400 ng (lane 5). (B) Competition oligonu-         185, respectively. lanes 5 10, XbaI sites used cleotide containing REBi site. Oligonucleotide 7/30 (sequence          create test various deletions EcoRI-HpaI fragment. <br /> shown Fig. 6B) polymerized, aliquot added             Coordinates sequences tested follows: 92 309 (lane reaction transcription began, follows: competitor        5), 147 309 (lane 6), 192 309 (lane 7), 1 87 (lane 8), 1 143 <br /> (lane 1), 0.5 ng (lane 2), 2 ng (lane 3), 10 ng (lane 4).              (lane 9), 1 185 (lane 10). (B) Assay finer deletions                                                                            ExoIII digestion. Lanes 1 4 tests 5&apos; deletions                                                                            EcoRI-HpaI fragment. Coordinates sequences tested                                                                            follows: 1 87 (lane 1), 1 137 (lane 2), 1 23 (lane 3), 1 16 (lane passes tests applied                4), 2 309 (lane 5), 2 288 (lane 6), 2 266 (lane 7), 2 194 (lane <br /> terminator. requires continued transcription 3&apos;-end                 8), 15 309 (lane 9). <br /> formation, orientation dependent, sensitive competition. precise molecular events occur- <br /> ring site remain determined, refer terminator rest paper.                                      assayed finer set deletions EcoRI-HpaI <br />    5&apos; 3&apos; deletion mapping 3&apos;-end-forming sites           fragment, shown Fig. SB. Deleting positions 24 309 <br /> EcoRI-HpaI fragment. RNA 3&apos; ends formed sites                     allows 3&apos;-end formation site (lane 3), II, III map approximate positions shown                  removal positions 17 309 (lane 4) positions 1 14 Fig. 2A. wished know location                  (lane 9) abolishes  Deleting positions 289 309 allows DNA sequences responsible forming 3&apos; ends.                   3&apos;-end formation site III (lane 6), removal Accordingly, series mutants                  sequence position 267 abolishes (lane 7). Site II novel XbaI sites introduced intervals                   affected deletion positions 195 309 (lane 8). <br /> EcoRI-HpaI fragment  novel XbaI sites designated                      conclusions experiments se- <br /> ELS-1, -2, -3, sequence locations shown               quences essential site map positions 1 23, <br /> Fig. iB). shown Fig. 5A, lanes 2 4, introduction               sequences site II map positions 106 194, novel Xbal sites did cause alteration RNA                 sequences site III map positions 192 288. <br /> 3&apos;-end formation. XbaI sites used end                        level resolution, RNA 3&apos; ends formed <br /> points produce small series 5&apos;- 3&apos;-terminal                    sites  II, III maps general location, deletions EcoRI-Hpal fragment.                                      DNA sequences cause 3&apos; ends. Site II 3&apos; <br />    Results assay 5&apos; deletion mutants shown              end map near region sequence pure  <br /> Fig. 5A, lanes 5 7. Removing sequences                        Site III maps HindIlI site HpaI site. Site <br /> positions 1 91 eliminates 3&apos;-end formation site 1 (lane               putative terminator, maps close EcoRI site 5). deletion position 146 eliminates 3&apos;-end forma-              5&apos; end EcoRI-HpaI fragment. examine <br /> tion site II (lane 6). Site III remains 5&apos; deletion          region  position 191 (lane 7).                                                     terminator (site  contains binding site REB1. <br />    T corresponding set 3&apos; deletions shown Fig. 5,              DNase footprinting gel shift analyses identified <br /> lanes 8 10. deletion positions 186 309                 binding sites proteins, called REB1 REB2 <br /> eliminates site III greatly reduces site IIL Site                 sites, just downstream EcoRI restriction site (23, 24) <br /> unaffected 3&apos; deletions sequences                 (Fig. 1B). REB1 site consists 8 bp positions 13 <br /> positions 88 309 (lanes 8 9).                                      20, REB2 site located positions 42 56.    map boundaries sites  II, III                REB1 protein cloned (11) likely  654     LANG REEDER                                                                                                                            MOL. CELL. BIOL. <br />  <br />                   2 3 4 5 6 7 8 9 10          B. <br />                                                 Wi ldtypef:               TATG rSC G G: AA  TAP <br />                                                                 G ATT(rlC                                     Ar                                                                                                                     J &quot; 1 T ATAe^ <br />                                                                                                                                       k TAC G GA &apos;  &apos;&apos;-r|r <br />                                                                                                                                                       <br />  <br />  <br />                                                                                            REBI                                        REE2 <br />                                                 07 /30 Oligo:   t t aaTATGATCCGGG-&apos;AAAAA.&lt;: <br />                                                                                            AG?iA <br />                                                                                                    c AgA <br />              .1--   illw         !              35/62 Oligo: <br />                                                 10/38 Alice:               gGAfrCCGGGTAAA     1 <br />                                                                                                        7:TA <br />                                                                                                         TA  TAT &apos;:AAATACO <br />                                                                                                     TT&apos;TATATAq . cCC- <br />                                                                                                                            T .CA:ATiA <br />                                                                                                                                                                                                                                                                     c* <br />  <br />                                                 10/23 Olig.:               gGATCCGGGTAAAA..^^&gt;Aggatce <br />  <br />    FIG. 6. Termination activity oligonucleotides containing REBM REB2 binding site. Oligonucleotides 7/30 35/62 inserted XbaI site test vector polylinker; oligonucleotides 10/38 10/23 inserted BamHI site polylinker. sequence oligonucleotides shown Fig. 7C.   Termination activity oligonucleotides. Lanes oligonucleotides <br />  orientations) follows: 1 10, assay vector insert; 2, oligonucleotide 7/30 (forward); 3, oligonucleotide 7/30 (reverse); <br /> 4, oligonucleotide 35/62 (forward); 5, oligonucleotide 35/62 (reverse); 6, oligonucleotide 10/38 (forward); 7, oligonucleotide 10/38 (reverse); 8, <br /> oligonucleotide 10/23 (forward); 9, oligonucleotide 10/23 (reverse). (B) Sequences oligonucleotides used panel  Uppercase <br /> letters represent natural rDNA sequence; lowercase letters indicate linker sequence added cloning purposes. REBM REB2 binding <br /> sites underlined wild-type sequence. <br />  <br />  <br />  <br /> identical protein independently purified                    -216 -225 relative transcription initiation [24]). called GRF2 (4). REB2 site subset                           site does match consensus binding site exactly; sequence binds protein ABF1 (3). test                       purine residue position 23 mutated C. <br /> possible involvement termination, prepared oligonu-                        REBM mutants tested termination <br /> cleotides contained REB1 REB2 site                    activity Fig. 7A tested ability bind cloned test site. Results termination                    REB1 protein gel shift assay (Fig. 7B). correlation <br /> assay shown Fig. 6A.                                                   termination activity ability bind REB1 <br />    oligonucleotide containing REB2 site showed                     protein complete. behavior mutant 21/22/23 detectable termination (oligonucleotide 35/62) (Fig. 6A, lane <br /> 4).  oligonucleotide containing REB1 site <br /> did exhibit termination activity (oligonucleotide 7/30, lane 2). oligonucleotide contains 6 bp sequence upstream                                                         234 5678910 CCGGGTAA core sequence identified DNase footprinting (23, 24). oligonucleotides  <br /> taining core sequence 3 bp upstream sequence failed cause termination (oligonucle- <br /> otides 10/38 10/23, lanes 6 8). Lane 3 shows REB1 oligonucleotide did cause termination shows orientation dependence characteristic RNA polymerase terminators. <br />    experiments shown Fig. 5 6 suggest                                                 B. <br /> nucleotides positions 7 23 important termination activity, region includes REB1                                                         00               46*    bouno DNA <br /> consensus binding site. examine region  series mutants just nucleotides time mutated stepwise fashion <br /> starting consensus REB1 binding site proceeding  sequences                                                    ILA                    - free DNA <br /> mutants shown Fig. 7C, ability cause <br /> termination inserted test site shown Fig. <br /> 7A. Beginning upstream REB1 site, pairwise                          C. <br /> mutants positions 7 8, 9 10, 11 12 (designated <br /> 7/8, 9/10, 11/12-lanes 2, 3, 4, respectively)                  .4.: . !.!.y e <br />  <br /> effect termination. mutations outside REB1 binding site. Mutation 13/14, changes <span id='am-28' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-31' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-32' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span>nucleotide</span> binding site, eliminates termination                           <br /> (lane 5). true mutations site                        .. <br />  <br />  <br />  <br />  <br /> (mutations 15/16, 17/18, 19/20-lanes 6, 7, 8, respec- <br /> tively). far, correlation mutations                        FIG. 7. REBM site essential polymerase affect REB1 site mutations eliminate termina-                   terminator.   Assay REBM binding site mutants termination <br /> tion perfect. apparent anomaly cluster                  activity. Lanes: 1 10, termination wild-type sequences 1 mutations (21/22/23), residues                      23; 2, mutant 7/8; 3, mutant 9/10; 4, mutant 11/12; 5, mutant 13/14; <br /> changed C&apos;s. changes REB1 consen-                     6, mutant 15/16; 7, mutant 17/18; 8, mutant 19/20; 9, mutant <br /> sus binding site (4), eliminate termination                   21/22/23. (B) Assay REB1 binding site mutants binding                                                                               REBM protein. Lanes: 1, mutant 7/8; 2, mutant 9/10; 3, mutant 11/12; <br /> (lane 9). making cluster mutations,                  4, mutant 13/14; 5, mutant 15/16; 6, mutant 17/18; 7, mutant 19/20; <br /> realized changes inadvertently altered                     8, mutant 21/22/23. (C) Sequences REBM site mutants. sequence second REB1 binding site known                  sequence consensus REB1 site underlined. Term, termina- exist upstream ribosomal gene promoter (positions                   tion activity. -, absent; +, present; + +, strongly present. <br />  <br />  <br />    FIG. 8. Map 3&apos; end formed site 17 &#194;&#177; 2 bp upstream REB1 binding site. length transcript specified site  compared length runoff RNA products generated transcription vectors truncated known sites. Lane 1 shows <br /> 5&apos;-end-labeled DNA size standards (HpaI digest pBR322). DNA size standards run precisely RNA standards sequence. DNA standards included illustrate degree resolution obtained region gel. Lane 2 shows RNA <br /> size standards generated assay vector  insert) transcribed linearization BglIl HindIII site. RNA standards lengths 123 256 nucleotides, corresponding map positions -69 +64, respectively. Lane 3, RNA <br /> size standard, generated deleting assay vector  shorten  linearizing Asp-718 site. produces RNA <br /> 191 nucleotides length 3&apos; end corresponds map position -2. Lane 4, assay template containing EcoRI-HpaI fragment test site, modified removing 706 bp vector sequence polymerase promoter test site. transcription template, RNA generated site migrates slightly faster position -2 marker RNA. places 3&apos; end formed site <br /> 1 17 &#194;&#177; 2 bp upstream REB1 consensus binding site. <br />  <br />  <br /> especially interesting. shown particular                     dependent sequences upstream REBM site. REB1 binding sequence higher level affinity                       illustrated Fig. 9. Figure 9, lane 1, shows result REB1 protein (4) wild-type sequence does. data                            inserting AccI-HpaI fragment test site. agree observation 21/22/23 mutant                            fragment contains 49 bp normal sequence upstream reproducibly shifts larger fraction probe does                       REBM consensus binding site efficient wild-type sequence gel shift assays (Fig. 7B, compare                       terminator fragment assayed  70% lane 3 lane 8). assays, mutant 21/22/23                       transcripts terminate site . lane 2, shorter EcoRI- efficient terminator wild-type sequence                         HpaI fragment blunt ended, NotI linkers attached, effect clearly seen Fig. 7B.                   result inserted test site. manip- <br /> set mutants, correlation sequences required                         ulations allowed retention G EcoRI REB1 binding sequences required termination                         recognition site (13 bp upstream REB1 site). Termi- <br /> RNA polymerase perfect.                                                       nation occurs fragment, clearly    conclude REBM site essential component                        efficient AccI-HpaI fragment shown lane 1. yeast polymerase terminator. implies                     lane 3, EcoRI-HpaI fragment assayed  REBM itself unknown protein identical DNA                             time blunt ended inserted directly binding specificity essential termination.                                  SmaI site test site. manipulations result    3&apos; ends formed site map 17 bp upstream                        loss G EcoRI recognition sequence, REB1 site. additional characteristic mouse frog polymerase terminators 3&apos; ends map 11 15 bp upstream essential terminator <br /> protein binding site (15, 17). Finer mapping 3&apos; end                                                            1 2 345 <br /> specified site shows true yeast terminator (Fig. 8). experiment, marker RNA <br /> molecules precisely known length generated                                                    run  <br /> RNA polymerase runoff transcription assay <br /> template truncated appropriate sites. marker                                                             U <br /> RNAs used lengths correspond 3&apos;-end <br /> formation positions -69 +64 (lane 2) -2 (lane 3). lane 4, RNA produced 3&apos;-end formation site migrates slightly faster position -2 marker. places 3&apos; end specified site 17 + 2 bp <br /> upstream boundary REBM site essential formation.  yeast terminator characteristics similar vertebrate counterparts. Similar                      FIG. 9. Dependence efficiency termination site experiments resolution located 3&apos; ends formed                        sequences upstream REBM site. DNA fragments various site II site III positions 155 250, respec-                      amounts natural sequence upstream REBM consensus <br /> tively (data shown).                                                           binding site assayed relative efficiency termination. <br />                                                                                    Lanes follows: 1, assay AccI-HpaI fragment assayed <br />    Efficiency termination site influenced sequences                  (49 bp upstream sequence); 2, EcoRI-HpaI fragment NotI <br /> upstream REB1 site. mutagenesis shown Fig. 7                         adaptors (13 bp upstream); 3, EcoRI-HpaI fragment blunt ended demonstrates REB1 site essential component                          SmaI site (12 bp upstream); 4, oligonucleotide 7/30 (7 bp terminator.  efficiency termination                       upstream); 5, oligonucleotide 10/23 (4 bp upstream). <br />  656    LANG REEDER                                                                                          MOL. CELL. BIOL. <br />  significant reduction termination effi-     chinery RNA polymerase  possible ciency. lane 4, oligonucleotide 7 bp upstream     REB1 sites genome terminators REB1 site tested drop termination     RNA polymerase II? REB1 sites efficiency observed. Finally, lane 5 shows       identified located known genes, just <br /> oligonucleotide tested 4 bp upstream          upstream activating sequences nearby promoter <br /> REBM site, termination detectable. require         (4).  conceivably function prevent tran- work determine region upstream         scription gene running dislodging REB1 site functions binding site additional     activator proteins control region downstream <br /> protein influences efficiency termination      gene. REB1 sites located upstream polymerase II activa- mechanism.                                                  tor regions reverse orientation <br />                                                                   compared REB1 site studied site                          DISCUSSION                                indicates REB1 actually involved polymerase II <br />                                                                   termination, orientation element polymerase <br />    function REB1 binding sites    specific. <br /> yeast genome? experiments reported paper            vivo searches terminator RNA polymerase protein DNA binding characteristics indistin-         transcription. 3&apos; end yeast rRNA coding region <br /> guishable REBM involved termination        previously studied vivo attempts locate <br /> RNA polymerase transcription yeast. require          possible terminators polymerase transcription. experiments definitively determine      example, Planta&apos;s group studied transcription riboso- <br /> REBM terminator protein.  reasonable            mal minigenes introduced multicopy plasmids reports <br /> assumption terminator protein actually REB1,          identification sites 3&apos;-end formation (12, 30). known protein worth reviewing.      site,  corresponds 3&apos; end 25S rRNA    REB1 probably identical GRF2, protein         concluded RNA processing site basis <br /> independently identified purified Kornberg group <br /> (4, 8). search revealed REB1 consensus             3&apos; end largely disappears yeast cells carrying binding sites number locations yeast       ma 82.1 mutation, mutation inactivates endonucle- <br /> genome, including near UAS elements polymerase           ase involved 3&apos;-end processing 5S RNA (26). Yip II promoters, telomeres, near centromere (4).            Holland (32) shown artificial transcript rDNA, REB1 sites,           processed vitro site yield 3&apos; end studied paper site  downstream 3&apos; end   25S. vitro transcription assays, observed 25S rRNA, second located 220 bp                3&apos; end maps terminus 25S rRNA (Fig. 2B, <br /> upstream gene promoter opposite orienta-        lane 1; corresponds position -96 sequence <br /> tion (23). Considering orientation dependence site    shown Fig. 1B). results actinomycin D chase unlikely REBM site near gene promoter          experiment shown Fig. 3 lend support involved polymerase termination.                             conclusion RNA processing site    REB1 essential gene yeasts (11, 24).    termination site. numerous binding sites genome (4, 31),            vivo sites 3&apos;-end formation identified possibility arises involved just      Planta&apos;s group include Ti, T2, T3, Tp. Ti 3&apos; ends termination transcription RNA polymerase             located downstream 3&apos; end 25S rRNA (Fig. 1B, <br /> possibility REB1 sites function complex        positions -81 -50) appear activation regions stimulate transcription certain        attributable processing (30). observed promoters. example, Fedor et al. (8) shown          termini correspond Ti vitro assays. <br /> REB1 binding site promoters GALl            T2 maps beginning T-rich stretch GALIO genes excludes nucleosomes 230 bp            EcoRI HindIII sites  position 100 Fig. 1B) DNA phases nucleosome positioning           concluded likely candidate REB1 site combined T-rich activator           authentic terminator (30). observe 3&apos; end control region DEDI gene, striking syner-       precisely location vitro.  possible gism activating transcription nearby TATA box           T2 calling site II. Site II maps observed. REBi site itself essentially             T-rich stretch compared T2 <br /> inactive assay. ability REBi site            position 155 Fig. 1B). Si nuclease protection <br /> synergize DEDI element strongly distance             assays  employed Planta group), T-rich hybrid <br /> dependent completely eliminated spacing            lost, yielding apparent map position coinciding elements 121 bp apart. Chasman et al. (4) point        T2. shown Fig. 2C, 3&apos;-end formation results suggest mechanism REBi binding            site II occurs sequence orientation. <br /> creates nucleosome free space site,           concluded site II  presumably <br /> making nearby activator elements accessible           T2  likely simple roadblock caused binding activator proteins. alternate suggestion    tight binding proteins. unlikely,  <br /> REB1 repressor RNA polymerase II transcription.           authentic terminator polymerase transcription. <br /> Wang et al. (31) placed REB1 site CYCi                T3 refers locations, T3A T3B, upstream activator cognate TATA element noted         map positions 550 890 Fig. 2A. caused repression CYCi promoter.         EcoRI-PvuII fragment sur- possible activating repressive         veyed Fig. 2C, lane 5. vitro effects REBi binding incidental fact      corresponds 3&apos; ends. Tp reported  REB1 strong DNA binder likely displace       -300 bp upstream site transcription initiation  nearby DNA binding proteins.                                     ribosomal gene promoter. approximate mapping <br />    REB1 site actually termination ma-        suggests site IV near SmaI site upstream  VOL. 13, 1993                                                                     YEAST POLYMERASE TERMINATOR                       657 <br />  <br /> promoter position -216;  absence finer          3&apos; end 25S formed processing  known <br /> mapping, possible Tp site IV             exception rule Ascaris lumbricoides,    Previous vivo searches 3&apos; ends             terminator 3&apos; end largest mature rRNA correspond site observed vitro (12, 30).        coincide [25]). yeast terminator requires specific think possible reasons discrepancy            sequence DNA binding site specific <br /> exists.  multicopy rDNA plasmids yeast               protein (REB1). Termination caused simple <br /> subject transcription type poly-              roadblock,  evidenced fact merase. transcription dual effect             terminator functions orientation. Finally, 3&apos; <br /> obscuring termini attributable RNA polymerase           ends formed site map 17 bp upstream creating additional termini having             protein binding site. <br /> polymerase transcription. second problem 3&apos;             interesting characteristic yeast polymerase end formed site located nucleotides      terminator influence sequences upstream EcoRI site. Si protection probes used          REB1 protein binding site efficiency 3&apos;-end forma- <br /> vivo studies (12, 30) EcoRI-HpaI                  tion (Fig. 9). related effects context sequences fragment detected particular 3&apos; end.         noted vertebrate systems. example,    Direct evidence REB1 site  site  involved       mice, sequences upstream downstream SalI terminator function vivo vitro comes        box influence termination efficiency precise location work Holland&apos;s group (22). assayed RNA 3&apos;-end            3&apos; ends (16). frog  saw context <br /> formation ribosomal minigenes utilizing low-copy-               effect vitro (21), efficiency termination appeared <br /> number CEN plasmid. EcoRI-HpaI fragment                 context dependent oocyte injection experiments <br /> placed downstream ribosomal gene promoter, gener-            (17). <br /> ated single 3&apos; terminus. showed                REB1 terminator protein? believe deletion sequences corresponding positions 8 46             shown paper REB1 binding site essential <br /> (Fig. 1B), including REB1 site, 3&apos;-end formation             element RNA polymerase terminator yeast. eliminated longer read products observed.           obvious question REB1 protein interesting vivo low-copy-number plasmid,         binding site causes termination. summarize 3&apos; terminus specified REB1 binding element           results number frustrating experiments, (site  observed 3&apos; termini corresponding       know answer question. J. Warner sites II III. leads think termination      coworkers generously supplied recombinant <br /> site efficient vivo transcription ordinarily   REB1 protein strains yeast carrying tempera- reaches sites II III.                                      ture-sensitive mutations REB1 protein. <span id='am-9' about='oboInOwl:date' typeof='owl:Thing'>date</span>,    causes 3&apos;-end formation sites II III?      unsuccessful obtaining significant stimula- locations, 3&apos;-end formation independent orienta-         tion termination adding excess REB1 protein. tion. suggests locations, tightly bound            added recombinant protein bacteria proteins act roadblocks RNA polymerase                  native protein partially purified yeast extracts. <br /> approaching direction. Lac repressor protein         adding extracts reported block elongating RNA polymerase II            partially depleted REB1 binding activity compe- <br /> (6, 14), likely tightly bound proteins    tition specific oligonucleotides heat shock similar effects. question                 temperature-sensitive extracts. possibil- <br /> proteins likely bound sites II III? 3&apos; ends     ities,   REB1 really relevant <br /> formed site II map closely appears             protein correct protocol upstream boundary rDNA enhancer element. vivo              reconstitution; (ii) real terminator protein work (7, 22, 29) indicates enhancer element extends         REB1, happens bind essentially position 150 slightly HindIII site.         identical REB1 binding site; (iii) additional factor shown enhancer element                 involved termination, limiting extracts. highly active yeast cell extract (27a),        require work, including fractionation expect proteins bound enhancer                 extract, decide possibilities. <br /> element termination assays  conclude possible 3&apos; ends observed site II attributable           roadblock imposed enhancer binding <br /> proteins.                                                   <br />    explanation site III speculative.          <br /> reports fail detect transcription enhancer activity      region (7, 22).  work Stewart Roeder (29) clearly showed                        <br /> region strongly affected recombination activity.        <br /> note recent paper showing terminator replication          <br /> maps general region (13). detecting             transcription roadblock caused proteins binding           elements vitro.                                      <br />    Comparison site known terminators                <br /> polymerase transcription. yeast polymerase termina-               <br />                                                                          <br /> tor characteristics similar terminators          previously studied mice frogs.             <br /> begin  yeast terminator located distance               <br /> (108 bp) downstream 3&apos; end mature 25S,              <br /> </body></html>