<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>rst Transcriptional Activity Influences kirre mRNA <br /> Concentration Drosophila Pupal Retina Final Steps Ommatidial Patterning <br /> Maiaro Cabral Rosa Machado, Shirlei Octacilio-Silva&#194;&#164;, Mara Silvia  Costa, Ricardo Guelerman P. Ramos* <br /> Departmento Biologia Celular e Molecular e Bioagentes Patoge&#203;&#8224;nicos, Faculdade Medicina Ribeira&#203;&#339;o Preto, Universidade Sa&#203;&#339;o Paulo, Ribeira&#203;&#339;o Preto, Sa&#203;&#339;o Paulo, <br /> Brazil <br />  <br />  <br />  <br />      Abstract <br />      Background: Drosophila retinal architecture laid 24&#226;&#8364;&#8220;48 hours puparium formation,      uncommitted interommatidial cells (IOCs) recruited secondary tertiary pigment cells      remaining ones undergo apoptosis. choice survival death requires product roughest (rst) gene, <br />      immunoglobulin superfamily transmembrane glycoprotein involved wide range developmental processes.      temporal misexpression Rst truncation protein intracytoplasmic domain, lead severe defects IOCs <br />      remain undifferentiated die late erratically  instead, differentiate extra pigment cells. <br />      Intriguingly, mutants expressing wild type protein normal mild rough eyes. <br />  <br />      Methodology/Principal Findings: using quantitative real time PCR examine rst transcriptional dynamics pupal <br />      retina, wild type mutant alleles showed tightly regulated temporal changes rst transcriptional rate <br />      underlie proper function final steps eye patterning. Furthermore demonstrated unexpected <br />      wild type eye phenotype mutants low rst expression correlates upregulation mRNA levels      rst paralogue kin irre (kirre), able substitute rst function process, similarly role      myoblast fusion. compensatory upregulation kirre mRNA levels directly induced wild type pupa      RNAi-mediated silencing rst, indicating expression genes coordinately regulated physiological <br />      conditions. <br />  <br />      Conclusions/Significance: findings suggest general mechanism rst kirre expression fine <br />      tuned optimize redundant roles development provide clearer picture specification      survival apoptotic fates differential cell adhesion final steps retinal morphogenesis insects      controlled transcriptional level. <br />  <br />   Citation: Machado MCR, Octacilio-Silva S, Costa MSA, Ramos RGP (2011) rst Transcriptional Activity Influences kirre mRNA Concentration Drosophila Pupal <br />   Retina Final Steps Ommatidial Patterning. PLoS 6(8): e22536. doi:10.1371/journal.pone.0022536 <br />   Editor: Madhuri Kango-Singh, University Dayton, United States America <br />   Received March 26, 2011; Accepted June 23, 2011; Published August 8, 2011 <br />   Copyright: &#195;&#376; 2011 Machado et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Sa&#203;&#339;o Paulo State Agency Research Support FAPESP, grant # 2007/55984-4. MCRM recipient MSc fellowship <br />   Brazilian Capes (Coordenadoria aperfeic&#194;&#184;oamento Pessoal Ensino Superior). funders role study design, data collection analysis, <br />   decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: rgpramos@fmrp.usp.br <br />   &#194;&#164; Current address: Departamento Morfologia, Universidade Federal Sergipe, Aracaju, Sergipe, Brazil <br />  <br />  <br />  <br /> Introduction                                                                           specificity mechanical strength interactions. Addi- <br />                                                                                        tionally significant number CAM families composed    Differential adhesion plays central role morphogenesis,                        transmembrane glycoproteins intracellular domains allowing cells sort initially unpatterned                       interact cell cytoskeleton, able mediate <br /> populations committed specific fates [1,2],                      complex changes cell shape binding dissociating helping wide range cell types, neurons, lymphocytes                       ligands.  important, albeit comparatively neural crest cells navigate interacting                          little studied group Irre Recognition Module IRM [8], small <br /> partners exquisite precision [3,4].  central                 set evolutionarily conserved transmembrane glycoproteins <br /> molecular mechanisms mediating changes tissue organization                          belonging immunoglobulin superfamily. IRM proteins shape, physiological, pathological contexts [5&#226;&#8364;&#8220;7].                     divided classes according structural    Adhesive properties cells critically dependent highly                   interacting properties: group shorter extracellular <br /> diverse group cell surface adhesion molecules, CAMs,                        domains containing immunoglobulin-like (Igl) repeats, specifically bind similar different CAMs present                   prototypical members products roughest (rst) kin- cells extracellular matrix. given time                     irre (kirre) paralogue genes Drosophila [9&#226;&#8364;&#8220;11], mammalian <br /> cell expresses characteristic subset CAMs define                    Nephs [12,13] C. elegans SYG-1 [14];  having <br />  <br />  <br />         PLoS | www.plosone.org                                                 1                                August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                          rst kirre Transcription Pupal Retina <br />  <br />  <br /> longer ectodomains Igl repeats                      precise temporal regulation rst transcription fibronectin type III domain, best studied members                 key requirements correct functioning final steps eye Nephrin humans [15], Stick Stones (Sns) Hibris                  morphogenesis [32]. study investigated (Hbs) Drosophila [16&#226;&#8364;&#8220;18] C. elegans SYG-2 [19]. Recent                  aspect rst function analysing transcriptional dynamics work laboratories shown tightly regulated                  developing retina quantitative real-time PCR, wild <br /> interactions members classes appear                      type mutant alleles, IOC sorting stages <br /> underlie involvement IRM proteins wide range                    immediately preceding following  results support developmental processes phylogenetic tree [19&#226;&#8364;&#8220;                  model previously forward Araujo et al [32], proposed <br /> 24]. Drosophila, processes include axonal pathfinding                 Rst protein redistribution consequence selective <br /> [9,25,26] myoblast cell fusion [10,11,16&#226;&#8364;&#8220;18] sensory organ spacing             stability border IOC pc1s tightly regulated <br /> [8,27], embryonic midline integrity [28; Moda LMR Ramos                    temporal changes locus transcriptional rate. Additionally RGP unpublished observations], salivary gland autophagy [29]                rst paralogue kirre appears function redundantly surplus cell elimination cell fate specification            process, similarly previously shown pupal retina [30&#226;&#8364;&#8220;32]. remarkable aspect IRM protein                       genes myoblast fusion embryogenesis, function insect ability partially substitute       providing explanation weak phenotype observed important morphogenetic events.                       loss expression rst alleles. Finally provide evidence particularly demonstrated generation larval                 genes regulated, having complementary transcription- <br /> body wall musculature embryogenesis, null muta-                   al dynamics RNAi mediated silencing rst leads tions affecting rst kirre subtle muscle               dramatic increase kirre mRNA levels time IOC <br /> defects, embryos deficient genes              redistribution. Taken  results refine provide completely deficient myoblast fusion [11]. Partially redundant              clearer picture specification survival apoptotic <br /> roles process recently demonstrated hbs sns          fates differential cell adhesion final steps [33].                                                                          compound eye morphogenesis insects controlled    final steps Drosophila compound eye morphogenesis                    transcriptional level. <br /> constitute excellent model investigate functional interplay IRM molecules context cell fate specification.            Results <br /> D. melanogaster adult eye 750 repetitive <br /> hexagonal units called ommatidia, containing photo-                 Rst protein redistribution IOCs correlates changes <br /> receptors, lens-secreting cone cells, morphologically           rst mRNA levels <br /> functionally distinct groups pigment cells (pc1, pc2 pc3),                 rstD flies differ previously described rst mutant alleles mechanosensory bristles.                   having &#226;&#8364;&#732;&#226;&#8364;&#732;glassy&#226;&#8364;&#8482;&#226;&#8364;&#8482;- looking compound eyes brownish-red <br /> specification cell types previously unpatterned                color (Fig. 1A) instead usual bright red, rough-like <br /> epithelial monolayer - eye imaginal disc - assembly              phenotype, capable spontaneously stably revert ommatidial units proceeds sequentially, starting mid-              wild type-like severe rough eye phenotypes larval instar, visualized wave cell                   moderate frequency (Octacilio-Silva S, Machado M Ramos <br /> differentiation, morphogenetic furrow, sweeps disc                RGP, unpublished observations) displaying delay posterior anterior. process concludes            IOCs intracellular redistribution Rst protein, happens <br /> half pupal development, unspecified                  8&#226;&#8364;&#8220;10% later development compared wild type (Fig. 1B). <br /> interommatidial cells (IOCs) recruited pc2 pc3               delay correlates late cell sorting, late erratic <br /> cells remaining ones undergo apoptosis. choice                   apoptotic elimination surplus IOCs incomplete pc2 life death specific context rely           pc3 differentiation (32). Identification genomic rearrange- <br /> relative position IOCs, sort maximize             ment associated rstD allele mapping rst apical contacts border specified pc1             regulatory region [28,32] raised question cells [30]. turn, occurs concomitantly  appears            mutation directly affect Rst protein redistribution IOCs. <br /> depend  reorganization Rst protein localization IOC               relatively simple explanation, based combination membranes, initial homogeneous                      differential stability Rst pc1/IOC borders tightly <br /> distribution progressively restricted IOC/pc1                   regulated changes rst transcription cell sorting stage <br /> borders. inability correctly promote Rst protein                   proposed [32]. order test model compared <br /> redistribution, shown block IOC sorting result             relative changes rst mRNA levels retinae wild <br /> adult eyes containing surplus cells differentiate extra              type controls homozygote rstD mutants isolated 0 <br /> pigment cells, distorting ommatidial organization                 44% pupal development using real time quantitative PCR <br /> causing strong, characteristic &#226;&#8364;&#732;&#226;&#8364;&#732;rough&#226;&#8364;&#8482;&#226;&#8364;&#8482; phenotype gives            (Fig 1C). wild type retinae, rst mRNA initially detected gene [30,31,34].  rst mutants                  relatively low levels 0 10% development puparium <br /> appear completely lack Rst protein compound eyes                formation (AFP). 19% AFP  4-fold increase normal mild roughness [9,25]                 basal levels observed, reaching 10-fold 24% <br /> indicating certain conditions IOCs able sort                 APF. peak 34% AFP rst mRNA levels differentiate normally presence              steadily decreased 3-fold reaching basal levels low levels rst activity .                                              43% APF. temporal dynamics rst mRNA <br />    Previous work group resulted phenotypic                concentration retinas rstD pupae, hand, molecular characterization rstD, relatively unstable, semi-           followed similar pattern 24% AFP, initial rise <br /> dominant regulatory mutation rst gene, showed                  tended steeper peak levels consistently higher right timing Rst redistribution IOCs appears critical          wild-type.   rst mRNA concentration correct elimination surplus IOCs              remained high 34% APF, slowly started <br /> process pc2 pc3 differentiation, strongly indicated            falling eventually reach basal levels wild type <br />  <br />  <br />        PLoS | www.plosone.org                                          2                            August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                          rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> Figure 1. Adult eye phenotypes protein expression pattern plus mRNA level quantification rst 43% pupal <br /> retina development.   wild type control (Canton S) rstD adult compound eyes. Note glassy brownish appearance mutant. (B) <br /> Comparative pattern Rst protein localization wild type control rstD mutant retinae, visualized anti&#226;&#8364;&#8220;rst Mab 24A.5. mutant IOC <br /> sorting does start 32% APF Rst continues detected IOC/pc1 border 43%APF (C) Quantification relative mRNA levels rst wild type rstD mutant retinae half pupal development RT-qPCR. mutant, starting 20% APF,rst mRNA <br /> concentration rises sharper falls significantly slower wild type, express similar basal levels beginning end half pupal development. text details. <br /> doi:10.1371/journal.pone.0022536.g001 <br />  <br />  <br /> (Fig. 1C). results correlate directly temporal                  end investigated retinal rst mRNA concentration <br /> course Rst protein redistribution IOCs shown figure 1B,               dynamics independently isolated rstD wild type-like mutant wild type, corroborating idea              revertants, RWT6 RWT8, conditions time phenomenon directly regulated transcriptional level             windows previously described rstD. Additionally performed providing simple explanation delay IOC sorting              analyses rstCT mutants, carry small deletion observed rstD mutants.                                                      coding region normal mRNA, causing                                                                                production Rst protein truncated intracellular domain. <br />                                                                                correlates phenotypically block cell sorting, lack absence wild type-like temporal dynamics rst                       elimination surplus IOCs severe rough eye phenotype <br /> mRNA concentration does necessarily prevent                            [9,31]  Fig. 2, left). truncated RstCT polypeptide <br /> normal organization adult compound eye                                  shows timing expression indistinguishable wild type     previous experiment imply, rst transcriptional activity          fails accumulate pc1/IOCs borders, collecting instead directly impacts correct timing Rst protein redistribution           intracellular endocytic vesicles [31], allowing test IOCs, promoting correct IOC sorting, rstD                    destabilization Rst protein membrane <br /> mutants reverting wild type-like eye phenotypes                 way modulate rst mRNA levels cell sorting <br /> revert wild type-like temporal profile rst mRNA expression.            stage. expected temporal profile Rst mRNA concentra- <br />  <br />  <br />        PLoS | www.plosone.org                                          3                            August 2011 | Volume 6 | Issue 8 | e22536 <br />                                   rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> PLoS | www.plosone.org   4   August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                         rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br /> Figure 2. RT-qPCR quantification relative rst mRNA levels rstCT mutant rstD wild type-like revertants corresponding eye phenotypes. Despite severe rough eye phenotype, rstCT  expresses truncated version Rst protein) shows essentially wild type rst mRNA temporal profile. rstD wild type-like revertants  defective rst mRNA expression, completely <br /> absent RTW6 displays striking departure wild type temporal profile RTW8, showing requirement rst wild type function bypassed mutants pupal eye development. <br /> doi:10.1371/journal.pone.0022536.g002 <br />  <br />  <br /> tion rstCT mutants closely follows wild type pattern,         kirre substitute rst function compound eye decrease 24% APF peak appears somewhat                    development <br /> slower mutant (Fig. 2).                                                   apparent paradox posed rstD revertants having normal <br />    analyses rstD revertants, hand, gave           compound eyes despite lacking rst expression pupal retina <br /> surprising results. Despite having essentially normal adult              easily resolved assuming rst functions <br /> compound eyes, mRNA concentration dynamics differed                     eye development equally performed strikingly wild type IOC sorting stage                 paralogue kirre, similarly demonstrated developmental time interval immediately following (Fig. 2).                myoblast fusion embryo [11]. investigate possibility <br /> RWT8, instance, showed actually decrease rst mRNA                    repeated RT-qPCR analyses presented Figures 1 2, <br /> concentration 24% APF, compared 19%, peaked                      time using kirre specific primers. comparison kirre 34% APF, reaching levels wild type time              mRNA temporal profiles wild type rstD shown Figure 3. <br /> point, RWT6 unable detect retinal rst mRNA                wild type retinae, absolute levels kirre mRNA expression time point 0 44% APF.                   lower rst remained relatively constant finding, implying complete lack rst function eye, does          time period analysed. significant departures appear artifact  instance, absence specific        basal level did occur, lowest levels coinciding sequences recognized primers used RT-qPCR                      peak rst mRNA expression, 24% APF highest <br /> experiments, rst mRNA detected RWT6 embryos                  34% APF rst mRNA concentration approached post embryonic tissues salivary glands submitted            basal levels. qualitative profile displayed experimental conditions.  immunoreactivity                rstD retinae  peak levels higher wild seen RWT6 pupal retinae treated anti-rst monoclonal                type. contrast, quantification kirre mRNA retinae antibody Mab 25A.4 (data shown). Taken data                rstD revertants (Fig. 4) showed completely different <br /> lead intriguing implication                      pattern, closer shown rst mRNA wild type, <br /> circumstances, rst expression dispensable correct                sharp increase time cell sorting followed ommatidial patterning.                                                        return near basal levels 34% AFP. particularly clear <br />  <br />  <br />  <br />  <br /> Figure 3. Comparative temporal quantification relative kirre mRNA levels wild type rstD. symbols representing genotype indicated graph. qualitatively similar wild type, kirre mRNA levels rstD retinae reach significantly higher <br /> concentrations 34% APF expression peak compared basal levels. <br /> doi:10.1371/journal.pone.0022536.g003 <br />  <br />  <br />        PLoS | www.plosone.org                                         5                            August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                        rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> Figure 4. Comparative temporal quantification relative kirre mRNA levels wild type, rstCT rstD wild type-like <br /> revertants. symbols representing genotype indicated. mutants &#226;&#8364;&#732;&#226;&#8364;&#732;rst-like&#226;&#8364;&#8482;&#226;&#8364;&#8482; temporal profile kirre mRNA expression, case rstCT, changes concentration relative wild type small. <br /> doi:10.1371/journal.pone.0022536.g004 <br />  <br />  RTW6 retinae temporal dynamics kirre mRNA                     remaining genomic copy kirre, indication <br /> shown Figure 4 indistinguishable wild-              eye phenotype rescue seen RTW6 revertants type rst mRNA (Fig. 2). interesting note rstCT          kirre redundant function. test possibility retinae, despite wild type-like levels low quantitative               used UAS-kirre RNAi construct driven GMR Gal4 knock <br /> variations, kirre mRNA concentration tended follow rst-like              kirre function RTW6 retinae. shown Figure 5B, temporal distribution, peaking 24% instead 34% AFP (Fig. 4).            clear increase severity phenotype observed, <br />    drastic quantitative qualitative changes kirre mRNA             particularly irregular spacing number ommatidial <br /> levels seen rstD revertants, particularly RTW6,                    fusions. Taken results high levels kirre <br /> compared wild type controls, compelling,                   expression time IOC sorting bypass directly demonstrate ability kirre rescue rst function       requirement rst function process provide strong <br /> IOC sorting. end genetically lowered kirre dosage               support idea genes act redundantly RWT6 mutants crossing flies carrying Df(1)w67k30               eye development. <br /> deletion removes rst kirre genes [11]. Figure 5A <br /> shows scanning electron micrographs compound eyes                 Post transcriptional silencing rst pupal retina <br /> parental strains relevant progeny. Homozygote individuals             leads upregulation kirre mRNA cell deficiency die embryogenesis severe defects           sorting stage <br /> muscle development heterozygotes, rst+ kirre+/rst2 kirre2             quantitative qualitative differences kirre mRNA viable normal compound eyes, rst2 kirre+/              concentration profile compared wild type, rstD <br /> rst2 kirre+ homozygote RTW6 flies.  trans-heterozygotes               revertants absence normal rst mRNA levels Df(1)w67k30/RTW6, genotipically rst2 kirre2/rst2 kirre+ display mild          19&#226;&#8364;&#8220;34% APF time window, perfectly characteristic ommatidial defects present rst mutants,           complementary timing expression, leads naturally including irregularly spaced bristles fused ommatidia (inset).            suggestion transcription genes regulated relatively weak phenotype observed consequence                  sensitive s RNA protein levels. high levels kirre mRNA produced retina                rst kirre map genome <br />  <br />  <br />        PLoS | www.plosone.org                                         6                          August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                       rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> Figure 5. Genetically lowering kirre dosage RTW6 revertant causes defects adult ommatidial organization.   Scanning <br /> electron micrographs. Compound eye phenotype RTW6/Df(1)w67k30 heterozygote female, compared parent genotypes, <br /> RTW6/Y Df((1)w67k30)/+. Leaving kirre copy present RTW6 chromosome functional copies rst present eye lead mild clear disruption ommatidial patterning showed inset: lack mispositioning mechanosensory bristles (arrowheads), bristle <br /> duplications (arrow) fused ommatidia (asterisk). (B) Dissecting stereomicroscope images. Head RTW6 male kirre function completely knocked expressing copy UAS-kirre RNAi 3111 transgene control GMR-Gal4 eye driver  methods). Eyes male carrying driver RWT6/Y shown. Note clear increase severity phenotype, <br /> particularly higher number fused ommatidia mispositioned bristles, compared RTW6/Df(1)w67k30eye  . <br /> doi:10.1371/journal.pone.0022536.g005 <br />  <br />  transcribed opposite directions [11], making possible          changes rst mRNA levels elicited kirre silenced  principle, share common regulatory                  retinae. results presented Figure 7 support elements.  highly coordinated compensatory                 assumption. Despite reduction retinal kirre mRNA levels changes mRNA levels genes shown previous               25% average parental levels did detect <br /> experiments clearly demonstrated mutants               significant changes rst mRNA, 24% APF peak previously gone reversion events involving genomic               34% APF, coincides kirre mRNA highest levels, <br /> rearrangements rst regulatory region, conceivable         indicating lowering kirre expression effect rst reflect &#226;&#8364;&#732;&#226;&#8364;&#732;lucky accident&#226;&#8364;&#8482;&#226;&#8364;&#8482; caused favorable                mRNA concentration tissue. <br /> reorganization regulatory sequences DNA RTW6 RTW8 true physiological response.                 Discussion <br /> asked deliberately lowering rst mRNA peak levels wild type retinae bring corresponding increase              ability interommatidial cells reorganize apical <br /> kirre mRNA. end quantified levels rst kirre            contacts maximize membrane interactions mRNA 24% APF wild-type retinas expressed               primary pigment cells, 24% AFP, key step independently generated rst RNAi constructs                 generating highly precise geometrical pattern adult <br /> control GMR-Gal4 driver (Fig. 6). Activation                   compound eye, needed create differences constructs wild type retinae lead reduction rst mRNA                intercellular adhesion signaling allow final cell fate <br /> concentration levels 20 35% average parental               specification decision pupal retina - survival <br /> levels quantified conditions (Fig. 6A)             programmed cell death &#226;&#8364;&#8220; correctly place. central mild defects apparent adult individuals (data               involvement Rst cell adhesion molecule IOC reorgani- <br /> shown). Quantification kirre mRNA showed                zation event inferred observation reduction rst mRNA levels correlated 5 10 fold                  process paralleled Rst protein redistribution IOC <br /> increase relative concentration kirre mRNA compared             membranes independent sets evidence:  <br /> average parental levels (Fig. 6B), providing strong evidence            complete absence truncation rst protein product genes coordinately regulated.                           blocked IOC sorting lead &#226;&#8364;&#732; &#226;&#8364;&#8482; switch cell <br />    able reduction rst mRNA                      fate, surplus pigment cells appearing adult eye concentration induce compensatory increase mRNA kirre               consequence lack cell death [30,31]; second, Rst <br /> levels, asked converse true compensatory              redistribution delayed, IOC sorting, produced  <br />  <br />        PLoS | www.plosone.org                                        7                           August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                          rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> Figure 6. rst RNAi silencing leads increase kirre mRNA levels time cell sorting.   Relative quantification rst mRNA <br /> levels 24% APF retinae expressing copy UAS-rstRNAi 951 UAS-rstRNAi 27223 transgenes, driven GMR-Gal4, compared expressing driver transgenes  (B) Relative quantification kirre mRNA levels 24% APF retina flies genotypes  . clear increase kirre mRNA levels seen rst-silenced retinae. <br /> doi:10.1371/journal.pone.0022536.g006 <br />  <br />  <br /> &#226;&#8364;&#732; &#226;&#8364;&#8482; response survival versus death choice          previously present membrane, main factor responsible correctly properly implemented, leading              critical changes cell adhesion specificity allow IOC <br /> surplus cells dying later erratically remaining ones             sorting place. findings help shed light little <br /> failed differentiate proper pigment cells [32]. data             studied  far, underestimated aspect eye development, <br /> emphasized need rst function right place           adding new dimension complex process ommatidial right time, implied precise spatial                   patterning differentiation.  results demonstrate <br /> temporal controls expression. Unraveling details           functionally ability kirre rescue rst function IOC sorting. <br /> control mechanisms essential prerequisite                possibility redundant roles rst kirre fuller understanding nature dynamics signals              retinal development previously suggested, supporting cells respond die differentiate final steps         evidence mainly indirect obtained protein  ommatidial patterning. quantitative analysis presented                  localization overexpression experiments (reviewed [8]). showed striking correlation rst mRNA concentra-                 recently, Bao et al [35] provided convincing evidence, based tion time qualitative dynamics Rst protein                     analysis retinal patterning time IOC sorting, localization IOC membrane, wild type                     Rst Kirre function redundantly maintain spacing <br /> regulatory rstD mutant implicating temporal control rst            developing ommatidial groups. extended transcription, reshuffling Rst protein molecules              observations directly showing ability high levels kirre <br />  <br />  <br />        PLoS | www.plosone.org                                          8                            August 2011 | Volume 6 | Issue 8 | e22536 <br />                                   rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br />  <br /> PLoS | www.plosone.org   9   August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                           rst kirre Transcription Pupal Retina <br />  <br />  <br />  <br /> Figure 7. kirre silencing does appear influence rst mRNA concentration retina.   Relative quantification kirre mRNA <br /> levels 24% APF retinae expressing copy UAS-kirreRNAi 3111 UAS-kirreRNAi 27277 transgenes, driven GMR-Gal4, compared expressing driver transgenes  (B) Relative quantification rst mRNA levels 24% APF retinae flies genotypes  . (C) Relative quantification kirre mRNA levels 34% APF retinae expressing copy UAS-kirreRNAi 27277, driven GMR-Gal4, compared expressing driver transgene  (D) Relative quantification rst mRNA levels 34% APF retinae genotypes (C). time point significant differences rst mRNA levels kirre-silenced retinas controls detected. <br /> doi:10.1371/journal.pone.0022536.g007 <br />  <br /> expression mutants little rst mRNA bypass                 findings suggest general mechanism need rst function, including influence cell fate choice.         expression genes fine tuned optimize molecular mechanism underlying redundancy,                     redundant roles development. <br /> remains elucidated. Rst Kirre highly conserved <br /> extracellular domains [11], localize IOCs appear              Materials Methods <br /> bind Hbs border pc1s [8,35], &#226;&#8364;&#8220; case Kirre direct evidence assertion missing &#226;&#8364;&#8220;              Fly stocks genetic crosses imply proteins fully interchangeable                  Fly manipulations performed vials containing standard interactions extracellular intracellular                   Drosophila food 25uC temperature-controlled room. RNAi <br /> molecules retinal development. intracellular               transgenic stocks UAS-rstRNAi27223, UAS-rstRNAi951, UAS-kirreR- <br /> domains little similarity [11] suggesting             NAi3111 UAS-kirreRNAi27277 obtained Vienna   common cytoplasmic binding partners. Genetic                  Drosophila RNAi stock center [41]. rstD revertant stocks RTW6/ biochemical studies aiming identifying possible intracellular            FM7i RTW8 isolated laboratory spontaneous <br /> interactors Kirre Rst performed different                   reversions wild-type eye phenotype homozygous rstD stock <br /> tissues, uncovering sets directly binding proteins potential             Canton S background additionally rstD dominant <br /> signaling pathways distinct gene product [36&#226;&#8364;&#8220;38].             suppressors. Genomic organization RTW6 region intracellular domain Rst required function              rstD rearrangement mapped [32] indistinguishable pupal retina [31], probably interacting actin                    wild type, assessed Southern Blot analysis, RTW8 <br /> cytoskeleton [29,39], important ascertain                appears resulted 1 kb internal deletion rst/kirre functional redundancy context results              rearranged area. mutants homozygous viable ability act intracellular pathways               RWT6/RWT6 females sterile. wild type mutant <br /> consequence interaction different ensembles                    stocks used study previously described [9,11,32]. <br /> intracellular targets lead end                Df(1)67k30/RTW6 flies generated selecting non-Bar <br /> result.                                                                         female progeny cross RTW6/Y males     interesting finding presented                  Df(1)67k30/FM7 females. Flies specifically expressing rst kirre <br /> evidence compensatory regulation rst kirre mRNA                   RNAi pupal retinae resulted progeny crosses <br /> concentrations asymmetrical nature. possible,                UAS-RNAi stocks described GMR-Gal 4 flies <br /> unlikely explanation asymmetry reside                [42]. Flies genotypically RTW6/Y; GMR-Gal4/UAS-kirre-RNA3111 <br /> higher absolute levels rst mRNA present temporal                  obtained constructing RTW6/FM7i; CyO/UAS-kirre- <br /> period examined, making small changes relative concentration                 RNA3111 females, crossing GMR-Gal4 homozygote males <br /> easier detect confidence. investigated             carrying transgene second chromosome selecting effect overexpression gene concentration            non-Bar, non Curly males. Flies examined Leica  similar coordination mRNA expression                   MZFLIII dissecting stereomicroscope equipped Leica DFC <br /> taking places developmental contexts complemen-                  500 digital camera acquired images processed using tary redundant functioning rst kirre required,             Leica Application Suite V3.1.0 software. myoblast fusion [11], salivary gland [29] optic lobe <br /> development [40]. context worth mentioning RTW6, RTW8 rstD, axonal pathfinding <br />                                                                                 Histological Immunohistochemical methods <br /> defects optic lobe (Machado M, Octac&#196;&#177;&#194;&#180;lio-Silva S                       lmmunohistochemistry staged pupal retinae monoclonal <br /> Ramos, R unpublished observations) suggesting                  antibody 24A5 [26] performed described previously [32,43] developmental contexts rst kirre fully                    analysed Leica SP5 Confocal microscope. Fly heads redundant.                                                                      prepared scanning electron microscopy ethanol substitution <br />     Finally, intriguing aspect compensatory response                  followed critical point drying gold coating. Image <br /> reported implies kind sensor mechanism                  acquisition Jeol JSM5200 scanning electron <br /> capable post-transcriptionally read rst expression levels                microscope. <br /> adjust kirre accordingly. mRNA, protein concentration, play main role, rstCT low               Quantitative Real Time Polimerase Chain Reaction <br /> levels rst activity caused truncated intracellular domain                retinae single staged pupa dissected PBS destabilization protein membrane leads              retina processed immunohistochemistry minimal,  increase kirre mRNA levels 24% APF.               anti-Rst control, immediately transferred conceivable upregulating kirre transcription,          Trizol reagent (Invitrogen) RNA extraction according decrease degradation taking place, leading                    manufacturer&#226;&#8364;&#8482;s specification. Genomic DNA removed accumulation kirre mRNA molecules. Experiments designed                   treatment DNase (Promega). Concentration purity test possibility map rst mRNA sequences              total RNA samples determined NanoDrop spectropho- relevant putative regulatory feedback loop               tometer (Thermo Scientific, USA). Samples optical rst kirre currently underway. case,                density absorption ratio A260/A280 1.8 2.2  <br />  <br />        PLoS | www.plosone.org                                          10                            August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                                                rst kirre Transcription Pupal Retina <br />  <br />  <br /> taken cDNA synthesis. High-Capacity Reverse Tran-                                       performed. Reactions template run parallel scription Kit (Applied Biosystems) used reverse-                                    plates internal controls. melting curve analysis transcription reaction concentration 200&#226;&#8364;&#8220;400 ng RNA                                 run confirm specificity amplification. DDCq <br /> sample accordance manufacturer&#226;&#8364;&#8482;s protocol. RT-                                      method [44] used compare rst expression wild <br /> qPCR reactions performed biological                                        type mutants. rst transcript levels normalized technical replicates 96 plate (Applied Biosystems)                                RpL32A transcript levels cDNA templates used using Gene AmpH 7500 Sequence Detection (PE                                          indicators expression levels. Data points represent Applied Biosystems) Sybr Green Rox Plus (LGC Biotech-                                      average independent biological samples  general) nology) Sybr Green PCR Master Mix (Applied Biosystems),                                     error bars corresponding standard deviations. experiments involving RNAi silencing. primers <br /> sequences  QPCR-RST-Forward CG4125:3 Type exon 59&#226;&#8364;&#8220;                                         Acknowledgments <br /> TGCCACCGAGGATCGCAAAG; QPCR-RST-Reverse CG4125: <br /> 4 Type exon 59&#226;&#8364;&#8220; GCAGCGGTATCACCGTGTAC; QPCR-                                                     like thank Vienna Drosophila RNAi stock center KIRRE-Forward CG3653:6 59- CGGTGCACAAAATGCAGT-                                                  Bloomington Drosophila stock center fly stocks members TAC QPCR-KIRRE-Reverse CG3653:6/7 59- GCTAACT-                                              Ramos lab critically reading manuscript. <br /> CAGGAAGCGAAAT. reactions run 95uC 10 <br /> minutes, 40&#226;&#8364;&#8220;45 cycles 95uC 15 seconds 60uC 1                                Author Contributions <br /> minute. RpL32A used reference amplified using                                        Conceived designed experiments: MCRM S RGPR. <br /> primers QPCR-RPL32A-Forward CG7939 59&#226;&#8364;&#8220; TCGACAACA-                                               Performed experiments: MCRM S MSAC. Analyzed data: <br /> GAGTGCGTCGC QPCR-RPL32A-Reverse CG7939 59&#226;&#8364;&#8220;                                                  MCRM S MSAC RGPR. Contributed reagents/materials/analysis <br /> CTTGAATCCGGTGGGCAGCA. check amplification effi-                                              tools: GPR MCRM. Wrote paper: GPR MCRM. <br /> ciency, calibration curves target reference genes  <br /> References <br />  1. Townes PL, Holtfreter J (1955) Directed movements selective adhesion                 20. Barletta GM, Kovari IA, Verma RK, Kerjaschki D, Holzman LB (2003) <br />     embryonic amphibian cells. J Exp Zoolog  Comp Exp Biol 128: 53&#226;&#8364;&#8220;120.                           Nephrin Neph1 localize podocyte foot process intercellular <br />  2. Steinberg MS (2007) Differential adhesion morphogenesis: modern view.                      junction form cis hetero-oligomers&#226;&#8364;&#8482;&#226;&#8364;&#8482;. J Biol Chem 278: 19266&#226;&#8364;&#8220;19271. <br />     Curr Opin Genet Dev 17: 281&#226;&#8364;&#8220;286.                                                            21. Khoshnoodi J, Sigmundsson K, Ofverstedt LG, Skoglund U, Obrink B, et al. <br />  3. Hattori D, Millard SS, Wojtowicz WM, Zipursky SL (2009) Dscam-mediated                          (2003) Nephrin promotes cell-cell adhesion homophilic interactions. <br />     cell recognition regulates neural circuit formation. Annu Rev Cell Dev Biol 24:                 J Pathol 163: 2337&#226;&#8364;&#8220;2346. <br />     597&#226;&#8364;&#8220;620.                                                                                    22. Bao S, Cagan RL (2005) Preferential adhesion mediated Hibris Roughest <br />  4. Duband JL (2006) Neural crest delamination migration: integrating                           regulates morphogenesis patterning Drosophila eye. Dev Cell 8: <br />     regulations cell interactions locomotion, survival fate. Adv Exp Med                     925&#226;&#8364;&#8220;35. <br />     Biol 589: 45&#226;&#8364;&#8220;77.                                                                            23. Gerke P, Sellin L, Kretz O, Petraschka D, Zentgraf H, et al. (2005) NEPH2  5. Carthew RW (2005) Adhesion proteins control cell shape. Curr Opin                    located glomerular slit diaphragm, interacts nephrin cleaved <br />     Genet Dev 15: 358&#226;&#8364;&#8220;363.                                                                          podocytes metaloproteases. J Soc Nephrol 16: 1693&#226;&#8364;&#8220;1702. <br />  6. Lee JM, Dedhar S, Kalluri R, Thompson EW (2006) epithelial&#226;&#8364;&#8220;                             24. Zhuang S, Shao H, Guo F, Trimble R, Pearce E, et al. (2009) Sns Kirre,     mesenchymal transition: New insights signaling, development, disease.                    Drosophila orthologs Nephrin Neph1, direct adhesion, fusion     J Cell Biol 172: 973&#226;&#8364;&#8220;981.                                                                       formation slit diaphragm-like structure insect nephrocytes. Development <br />  7. Perez-Pomares JM, Foty RA (2006) Tissue fusion cell sorting embryonic                    136: 2335&#226;&#8364;&#8220;2334. <br />     development disease: Biomedical implications. Bioessays 28: 809&#226;&#8364;&#8220;821.                    25. Boschert U, Ramos RGP, Tix S, Technau GM, Fischbach KF (1990) Genetic <br />  8. Fischbach KF, Linneweber GA, Andlauer TFM, Hertenstein  Bonengel B,                           developmental analysis irreC, genetic function required optic chiasm <br />     et al. (2009) Irre Cell Recognition Module (IRM) Proteins. J Neurogenetics                  formation Drosophila. J Neurogenetics 6: 153&#226;&#8364;&#8220;171. <br />     23: 48&#226;&#8364;&#8220;67.                                                                                  26. Schneider T, Reiter C, Eule E, Bader B, Lichte B, et al. (1995) Restricted <br />  9. Ramos RGP, Igloi GL, Baumann U, Maier D, Lichte B, et al. (1993)                            expression cell adhesion molecule IrreC-rst required normal axonal <br />     irregular chiasm C-roughest locus Drosophila affects axonal projections            projections columnar visual neurons. Neuron 15: 259&#226;&#8364;&#8220;271. <br />     programmed cell death encodes novel immunoglobulin-like protein. Genes                    27. Reddy GV, Reiter C, Shanbhag S, Fischbach KF, Rodrigues V (1999) Irregular <br />     Dev 7: 2533&#226;&#8364;&#8220;2547.                                                                               chiasm C-roughest, member immunoglobulin superfamily, affects sense- <br /> 10. Ruiz Gomez M, Coutts N, Price  Taylor M, Bate M (2000) Drosophila                             organ spacing Drosophila antenna influencing positioning     Dumfounded: myoblast attractant essential fusion. Cell 102: 189&#226;&#8364;&#8220;198.                      founder cells disc ectoderm. Dev Genes Evol 209: 581&#226;&#8364;&#8220;591. <br /> 11. Stru&#194;&#168;nkelnberg M, Bonengel B, Moda LMR, Hertenstein  Gert Couet H,                     28. Apitz H, Kambacheld M, Hohne M, Ramos RGP, Straube  Fischbach KF <br />     et al. (2001) rst paralogue kirre act redundantly embryonic muscle               (2004) Identification regulatory modules mediating specific expression     development Drosophila. Development 128: 4229&#226;&#8364;&#8220;4239.                                          roughest gene Drosophila melanogaster. Dev Genes Evol 214: 453&#226;&#8364;&#8220;459. <br /> 12. Donoviel DB, Freed DD, Vogel H, Potter DG, Hawkins E, et al. (2001)                         29. Simon CR, Moda LMR, Octacilio-Silva S, Anhezini L, Machado LCH, et al. <br />     Proteinuria perinatal lethality mice lacking NEPH1, novel protein                 (2009) roughest locus interferes autophagic cell death Drosophila <br />     homology NEPHRIN. Mol Cell Biol 21: 4829&#226;&#8364;&#8220;4836.                                               salivary glands. Genesis 47: 492&#226;&#8364;&#8220;504. <br /> 13. Sellin L, Huber TB, Gerke P, Quack  Pavenstadt H, et al. (2003) NEPH1                     30. Wolff T, Ready DF (1991) Cell death normal rough eye mutants     defines novel family podocin interacting proteins. FASEB J 17: 115&#226;&#8364;&#8220;117.                    Drosophila. Development 113: 825&#226;&#8364;&#8220;839. <br /> 14. Shen K, Bargmann CI (2003) immunoglobulin superfamily protein SYG-1                     31. Reiter C, Schimansky T, Nie  Fischbach KF (1996) Reorganization     determines location specific synapses C. elegans. Cell 112: 619&#226;&#8364;&#8220;630.                  membrane contacts prior apoptosis Drosophila retina: role 15. Kestila&#194;&#168; M, Lenkkeri U, Ma&#194;&#168;nnikko&#194;&#168; M, Lamerdin J, McCready P, et al. (1998)                     IrreC-Rst protein. Development 122: 1931&#226;&#8364;&#8220;1940. <br />     Positionally cloned gene novel glomerular protein&#226;&#8364;&#8220;nephrin mutated               32. Araujo H, Machado LCH, Octacilio-Silva S, Mizutani CM, Silva MJF, <br />     congenital nephrotic syndrome. Mol Cell 1: 575&#226;&#8364;&#8220;582.                                             Ramos RGP (2003) Requirement roughest-irregular chiasmC gene cell fate <br /> 16. Bour BA, Chakravarti M, West JM, Abmayr SM (2000) Drosophila SNS,                             specification final stages Drosophila compound eye development. Mech <br />     member immunoglobulin superfamily essential myoblast fusion.                 Dev 120: 537&#226;&#8364;&#8220;547. <br />     Genes Dev 14: 1498&#226;&#8364;&#8220;1511.                                                                    33. Shelton C, Kocherlakota KS, Zhuang S, Abmayr S (2009) immunoglobulin <br /> 17. Artero RD, Castanon  Baylies MK (2001) immunoglobulin-like protein                        superfamily member Hbs functions redundantly Sns interactions <br />     Hibris functions dose-dependent regulator myoblast fusion                        founder fusion-competent myoblasts. Development 136: <br />     differentially controlled Ras Notch signaling. Development 128:                          1159&#226;&#8364;&#8220;1168. <br />     4251&#226;&#8364;&#8220;4264.                                                                                  34. Grzeschik NA, Knust E (2005) IrreC/rst-mediated cell sorting 18. Dworak HA, Charles MA, Pellerano LB, Sink H (2001) Characterization                          Drosophila pupal eye development depends proper localisation  <br />     Drosophila hibris, gene related human nephrin. Development 128:                            cadherin. Development 132: 2035&#226;&#8364;&#8220;2045. <br />     4265&#226;&#8364;&#8220;4276.                                                                                  35. Bao S, Fischbach K-F, Corbin V, Cagan RL (2010) Preferential adhesion <br /> 19. Shen K, Fetter RD, Bargmann CI (2004) Synaptic specificity generated                  maintains separation ommatidia Drosophila eye. Dev Biol 344: <br />     synaptic guidepost protein SYG-2 receptor, Syg-1. Cell 116: 869&#226;&#8364;&#8220;881.                    948&#226;&#8364;&#8220;956. <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                                         11                                August 2011 | Volume 6 | Issue 8 | e22536 <br />                                                                                                                           rst kirre Transcription Pupal Retina <br />  <br />  <br /> 36. Vishnu S, Hertenstein H, Betschinger J, Knoblich JA, Gert Couet H, et al.            40. Sugie  Umetsu D, Yasugi T, Fischbach K-F, Tabata T (2010) Recognition     (2006) adaptor protein X11La/Dmint1 interacts PDZ-binding                      pre- postsynaptic neurons nephrin/NEPH1 homologs basis     domain cell recognition protein Rst Drosophila. Dev Biol 289:                     formation Drosophila retinotopic map. Development 137: 3303&#226;&#8364;&#8220;3313. <br />     296&#226;&#8364;&#8220;307.                                                                                41. Dietzl G, Chen D, Schnorrer F, Su KC, Barinova Y, et al. (2007) genome- <br /> 37. Bulchand S, Menon SD, George SE, Chia W (2010) Intracellular Domain                     wide transgenic RNAi library conditional gene inactivation Drosophila. <br />     Dumbfounded Affects Myoblast Fusion Efficiency Interacts Rolling                Nature 448: 151&#226;&#8364;&#8220;156. <br />     Pebbles Loner. PLoS 5: e9374.                                                   42. Freeman M (1996) Reiterative use EGF receptor tri <br /> </body></html>