<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                 Volume 15 Number 9           September 2013          pp. 1075&#226;&#8364;&#8220;1085        1075 <br /> www.neoplasia.com <br />  <br />  <br />  <br />  <br />                                                                                     Jing Shao*,&#226;&#8364;&#160;, Yong Teng&#226;&#8364;&#161;, Ravi Padia&#226;&#8364;&#160;, <br />           COP1 GSK3&#206;&#178; Cooperate                                                  Sungguan Hong&#226;&#8364;&#160;, Hyangsoon Noh&#226;&#8364;&#160;, Xiayang Xie&#194;&#167;, <br />           Promote c-Jun Degradation                                              Jeff S. Mumm&#194;&#167;, Zheng Dong&#194;&#167;, Han-Fei Ding&#226;&#8364;&#161;, <br />                                                                                     John Cowell&#226;&#8364;&#161;, Jaejik Kim&#226;&#8364;&#161;,&#194;&#182;, Jiahuai Han* <br />           Inhibit Breast Cancer                                                 Shuang Huang&#226;&#8364;&#160;,# <br />           Cell Tumorigenesis1,2                                                     *State Key Laboratory Cellular Stress Biology, School <br />                                                                                     Life Sciences, Xiamen University, Xiamen, Fujian, <br />                                                                                     China; &#226;&#8364;&#160;Department Biochemistry Molecular Biology, <br />                                                                                     Medical College Georgia, Georgia Regents University, <br />                                                                                     Augusta, GA; &#226;&#8364;&#161;Cancer Center, Georgia Regents University, <br />                                                                                     Augusta, GA; &#194;&#167;Department Cellular Biology Anatomy, <br />                                                                                     Medical College Georgia, Georgia Regents University, <br />                                                                                     Augusta, GA; &#194;&#182;Department Biostatistics, Georgia <br />                                                                                     Regents University, Augusta, GA; #E-Institute Shanghai <br />                                                                                     Municipal Education Committee, Shanghai University <br />                                                                                     Traditional Chinese Medicine, Shanghai, China <br />  <br />  <br />  <br />  <br />           Abstract <br />           High abundance c-Jun detected invasive breast cancer cells aggressive breast tumor malignancies.  <br />           demonstrate major cause high c-Jun abundance invasive breast cancer cells prolonged c-Jun <br />           protein stability owing poor poly-ubiquitination c-Jun. known c-Jun&#226;&#8364;&#8220;targeting E3 ligases, iden- <br />           tified constitutive photomorphogenesis protein 1 (COP1) E3 ligase responsible c-Jun degradation           invasive breast cancer cells depletion COP1 reduced c-Jun poly-ubiquitination leading stabilization <br />           c-Jun protein. panel breast cancer cell lines, observed inverse association levels COP1 <br />           c-Jun.  overexpressing COP1 unable decrease c-Jun level invasive breast cancer cells, <br />           indicating efficient c-Jun protein degradation necessitates additional event.  glycogen <br />           synthase kinase 3 (GSK3) inhibitors elevated c-Jun abundance invasive breast cancer cells GSK3&#206;&#178; <br />           nonphosphorylable c-Jun&#226;&#8364;&#8220;T239A mutant displayed greater protein stability poorer poly-ubiquitination compared <br />           wild-type c-Jun. ability simultaneously enforced expression COP1 constitutively active GSK3&#206;&#178;           decrease c-Jun abundance invasive breast cancer cells allowed conclude c-Jun negatively regulated <br />           coordinated action COP1 GSK3&#206;&#178;. Importantly, expressing COP1 active GSK3&#206;&#178; blocked <br />           vitro cell growth/migration vivo metastasis invasive breast cancer cells. Gene expression profiling breast <br />           tumor specimens revealed higher COP1 expression correlated better recurrence-free survival.           study supports notion COP1 suppressor breast cancer progression. <br />  <br />           Neoplasia (2013) 15, 1075&#226;&#8364;&#8220;1085 <br />  <br />  <br />  <br />  <br /> Address correspondence  Shuang Huang, PhD, Department Biochemistry Molecular Biology, Medical College Georgia, Georgia Regents University, Augusta, <br /> GA 30912. E-mail: shuang@gru.edu <br /> 1 <br />   work supported funding National Institutes Health HL083335, Shanghai Eastern Scholar Fund, E-Institutes Shanghai Municipal Education <br /> Commission Project E03008. <br /> 2 <br />  article refers supplementary materials, designated Figures W1 W5 Table W1 available online www.neoplasia.com. <br /> Received 11 2013; Revised 27 June 2013; Accepted 2 July 2013 <br /> Copyright &#194;&#169; 2013 Neoplasia Press,  rights reserved 1522-8002/13/$25.00 <br /> DOI 10.1593/neo.13966 <br />  1076         Regulation c-Jun COP1 GSK3&#206;&#178;            Shao et al.                                             Neoplasia Vol. 15,  9, 2013 <br />  <br /> Introduction                                                                         objective study answer protein level c-Jun member activating protein 1 family transcription             c-Jun regulated breast cancer cells. performing cycloheximide- <br /> factors [1]. forming heterodimers members activat-               chasing experiment, c-Jun protein ing protein 1 family, c-Jun regulates expression variety              stable invasive breast cancer cells invasive ones. Pro- <br /> genes important diverse cellular functions including cell growth,             longed c-Jun protein stability c-Jun cell migration, invasion [2]. identified proto-           poly-ubiquitinated invasive breast cancer cells. characterize oncogenes, extensive studies exerted characterize                mechanism mediating c-Jun protein degradation, showed role c-Jun cancer development including breast cancer. early             depleting COP1 inhibiting GSK3 activity prevented poly- <br /> study showed forced c-Jun expression able convert non-                ubiquitination c-Jun increased c-Jun abundance invasive <br /> invasive/hormone-dependent breast cancer MCF7 line inva-                    breast cancer cells.  simultaneous enforcement sive hormone-resistant line [3]. later study using clinical breast          COP1 constitutively active GSK3&#206;&#178; (GSK3&#206;&#178;S9A) expression tumor specimens revealed c-Jun detected invasive            able effectively downregulate c-Jun invasive breast cancer cells. breast tumors level correlated increased angiogenesis             results indicate high COP1 expression GSK3&#206;&#178;- <br /> [4]. role c-Jun promote breast tumor progression metas-             mediated phosphorylation necessary c-Jun protein turnover. <br /> tasis supported recent vivo studies: 1) depletion c-Jun           notion supported observation non&#226;&#8364;&#8220;GSK3&#206;&#178;- <br /> reduced cell migration invasion ErbB2-induced mammary                      phosphorylable c-Jun significantly stable wild-type <br /> tumors ErbB2 mammary tumor transgenic mouse model [5]                      c-Jun. Importantly, showed expressing COP1 2) overexpression c-Jun sufficient confer nonmetastatic                 GSK3&#206;&#178;S9A suppressed vitro cell growth/migration vivo <br /> breast cancer cells capability metastasize [6]. active             metastasis invasive breast cancer cells. Gene expression analysis role c-Jun breast tumorigenesis lie ability promote           breast tumor specimens revealed COP1 mRNA expres- <br /> cell proliferation [7] migration/invasion [8].                                sion independent positive prognostic factor recurrence-free <br />     High abundance c-Jun detected various aggressive tumor               survival patients breast cancer. <br /> types [9,10]. Immunohistochemistry revealed c-Jun level low present cells normal benign breast <br /> tissues; contrast, immunoreactivity c-Jun detected high              Materials Methods <br /> intensity usually detected significant percentage cells breast <br /> carcinoma specimens [11]. limited number human breast                 Cell Culture, Inhibitors, siRNAs <br /> cancer cell lines, previously showed c-Jun             cell lines used study purchased American Tissue <br /> higher invasive lines invasive ones [8]. abun-          Culture Collection (ATCC, Manassas, VA) cultured Dulbecco&#226;&#8364;&#8482;s <br /> dance c-Jun governed transcriptional, posttranscriptional,           modified Eagle&#226;&#8364;&#8482;s medium supplemented 10% FBS humidi- <br /> translational, posttranslational levels [2], mechanism             fied atmosphere containing 5% <span id='am-5' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CO2</span> 37&#194;&#176;C. Inhibitors used elevated c-Jun level invasive breast cancer cells understood.      study PYR-41 Calbiochem (San Diego, CA); MG132     normal circumstances, c-Jun protein known                     Chloroquine Tocris (Bristol, United Kingdom); SP600125, highly unstable [12] level regulated                  SB216763, 4,5,6,7-tetrabromobenzotriazole (TBB) Sigma <br /> ubiquitination/proteasome-dependent mechanism [13]. Ubiquitin E3 li-              (St Louis, MO). E3 ligase&#226;&#8364;&#8220;targeting siRNAs control siRNA <br /> gases add poly-ubiquitin chain c-Jun include constitutive             purchased Thermo Scientific (West Palm Beach, FL). <br /> photomorphogenesis protein 1 (COP1) [14], cullin 4 (CUL4) [15], <br /> F-box WD repeat domain containing 7 E3 ubiquitin protein ligase               Determination c-Jun Protein Stability <br /> (FBW7) [16], Itchy E3 ubiquitin protein ligase (ITCH) [17], mitogen-                 Overnight cultured MCF7, MDA-MB-231, MDA-MB-436, <br /> activated protein kinase kinase kinase 1 (MEKK1) [18], sensitive           T47D cells transfected plasmids encoding human <br /> apoptosis, zinc ring finger protein (SAG) [19]. COP1 unique               influenza hemagglutinin (HA)-tagged c-Jun green fluorescent dual ability act E3 ligase adaptor   protein (GFP) 1 day. Cells trypsinized reseeded recruit substrate etiolated homolog 1/damage-specific DNA                12 hours followed addition 10 &#206;&#188;g/ml cycloheximide. vary- <br /> binding protein 1/cullin 4/ring-box 1, E3 ubiquitin protein ligase                ing time points, cells harvested cell lysates subjected <br /> (DET1/DDB1/CUL4/RBX1) ubiquitin ligase complex [15]. ex-                      immunoblot analysis detect HA-tagged c-Jun GFP using <br /> pression E3 ligases displays tissue specificity varies various    respective antibodies. intensities bands quantified types stages cancers;  great investigate    Odyssey Image (LI-COR Biosciences, Lincoln, NE). c-Jun&#226;&#8364;&#8220;targeting E3 ligases involved regulation c-Jun protein abundance breast cancer cells.              Immunoprecipitation <br />     Function c-Jun shown regulated phosphoryla-                 determine c-Jun poly-ubiquitination breast cancer cells, <br /> tion. Casein kinase II phosphorylates c-Jun, leading suppres-              HA-tagged c-Jun Flag-tagged ubiquitin transfected sion c-Jun activity [20]. contrast, c-Jun N-terminal kinases                MDA-MB-231, MDA-MB-436, MCF7, T47D cells 1 day <br /> (JNKs) activate c-Jun directly phosphorylating c-Jun [21].            followed treatment MG132 8 hours. Cells lysed ingly, JNKs activate c-Jun&#226;&#8364;&#8220;targeting E3 ligase ITCH, estab-          buffer containing 1% sodium dodecyl sulfate, 150 mM NaCl, lishing negative feedback loop [17]. Recognition c-Jun E3 ligase          10 mM Tris-HCl (pH 8.0) followed 10-minute boiling. Samples <br /> FBW7 subsequent degradation necessitate c-Jun phos-               sheared sonication device diluted buffer contain- <br /> phorylated glycogen synthase kinase 3 beta (GSK3&#206;&#178;) [22].               ing 10 mM Tris-HCl (pH 8.0), 150 mM NaCl, 1 mM EDTA, unknown c-Jun phosphorylation prerequisite deg-        1% Triton X-100. 30-minute centrifugation 20,000g, radation COP1                                                       resulting supernatants immunoprecipitated HA monoclonal <br />  Neoplasia Vol. 15,  9, 2013                                             Regulation c-Jun COP1 GSK3&#206;&#178;         Shao et al.     1077 <br />  <br /> antibody (mAb) immunoprecipitates subjected immuno-         using 24 formatted collagen-coated transwell previously <br /> blot analysis detect Flag-ubiquitin using Flag mAb.                  described [8]. Briefly, 1 &#195;&#8212; 105 cells 100 &#206;&#188;l serum-free medium <br />                                                                         added upper chamber transwell allowed Cell Fractionation                                                      migrate 4 hours. Nonmigrating cells upper chambers    Cell fractionation performed previously described          removed cotton swabs, migrating cells lower surface [23]. Briefly, cells lysed Buffer [10 mM Hepes (pH 7.9),      chamber stained using crystal violet. migrating <br /> 1.5 mM MgCl2, 10 mM KCl, 0.5 mM PMSF, 0.5 mM DTT]                   cells determined counting stained cells phase- <br /> followed centrifugation 2000g 10 minutes obtain crude    contrast microscope. <br /> nuclear pellet cytoplasmic supernatant (cytoplasmic fraction). crude nuclear pellets purified resuspension      Zebrafish Metastatic Model <br /> Buffer B [20 mM Hepes (pH 7.9), 25% (vol/vol) glycerol, 0.45 M              effect combined expression COP1 GSK3&#206;&#178;S9A NaCl, 1.5 mM MgCl 2 , 1 mM EDTA, 0.5 mM PMSF,                       evaluated recently developed zebrafish metastatic model [24]. <br /> 0.5 mM DTT]. lysates subjected high-speed centrifuga-       experimental procedures approved Institutional <br /> tion 1 hour obtain nuclear fraction.                             Animal Care Use Committee Georgia Regents University. Briefly, <br />                                                                         control COP1/GSK3&#206;&#178;S9A-expressing MDA-MB-231 cells Lentiviral Constructs Packaging                                     labeled fluorescent dye CM-Dil (Life Technologies, Carlsbad, <br />    Coding sequences COP1 GSK3&#206;&#178;S9A mutant sub-               CA), approximately 200 labeled cells microinjected <br /> cloned pCDH Lentivector  Biosciences, Mountain View,        perivitelline space 48-hour post fertilization (hpf) zebrafish <br /> CA). Lentiviruses packaged 293FT cells according         embryos using pressure microinjector. embryos kept manufacturer&#226;&#8364;&#8482;s instruction.                                             34&#194;&#176;C imaged anaesthetic confocal microscopy                                                                         48-hour post injection. <br /> Nuclear Proteasome Activity Assay <br />    Nuclear 20S proteasome activities determined using overnight    Statistical Analysis <br /> cultured cells aid 20S Proteasome Assay Kit (Cayman            Statistical analyses laboratory studies performed <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span>, Ann Arbor, MI) measures ability fractions     Student&#226;&#8364;&#8482;s t test analysis variance. determine potential cleave proteasome-specific substrate SUC-LLVY-AMC. Briefly, cells    correlation COP1 expression clinicopathologic param- resuspended Buffer [10 mM Hepes, 10 mM KCl, 2 mM              eters, obtained data COP1 expression microarray analysis <br /> MgCl2, 0.1 mM EDTA, 1 mM PMSF (pH7.9)] incubated                clinical features 126 patients breast cancer  ice 25 minutes followed adding 10% NP-40 final         ceived tamoxifen monotherapy, chemotherapy, radiation therapy <br /> concentration 1%. Cells vortexed 15 seconds        Gene Expression Omnibus database (91 samples microcentrifuged 5000g 5 minutes. collected pellets (cell    GSE18229 35 samples GSE40954). cutoff value nucleus) analyzed 20S proteasome activity.                     high low COP1 expression groups based median <br />                                                                         value. Associations COP1 expression clinicopathologic <br /> Immunofluorescence Staining                                             parameters determined using Pearson &#207;&#8225;2 test. univariate sur- <br />    Cells culture collagen-coated glass coverslips overnight,    vival analysis, Kaplan-Meier method log-rank test used fixed 4% paraformaldehyde permeabilize 1%            compare recurrence-free survival curves high low <br /> Triton X-100 15 minutes. visualize c-Jun proteasome          COP1 expression groups. multivariate survival analysis, Cox <br /> S5A subunit, coverslips incubated c-Jun mAb S5A           proportional hazard regression model used identify important <br /> polyclonal antibody followed incubation fluorescein isothio-    factors disease-free survival. identify factors, initial <br /> cyanate rhodamine-labeled secondary antibodies. fluorescence     model COP1 expression considered particular <br /> staining visualized fluorescence microscope (Axiovert       clinicopathologic parameter added model time. 200M; Zeiss, Munich, Germany). 4&#226;&#8364;&#178;,6-Diamidino-2-phenylindole            repeating procedure, final regression model important <br /> (DAPI) used stain nucleus breast cancer cells.                factors identified likelihood ratio test. P &lt; .05 considered <br />                                                                         statistically significant. <br /> Cell Proliferation Study <br />    determine ability enforced COP1 GSK3&#206;&#178;S9A <br /> expression affect cell proliferation, MDA-MB-231 MDA-            Results <br /> MB-436 cells transduced lentiviral vector vector <br /> containing COP1 GSK3&#206;&#178;S9A 3 days. Transduced cells           Ubiquitin/Proteasome-Dependent Mechanism Controls <br /> subjected 3-(4,5-dimethylthiazol-2-yl)-2,5-diphenyltetrazolium       c-Jun Abundance Breast Cancer Cells <br /> bromide (MTT) assay evaluate cell proliferation previously           limited human breast cancer cell lines, previously showed <br /> described [23]. Briefly, 5 &#195;&#8212; 104 cells seeded 24 cul-    abundance c-Jun higher invasive breast  <br /> ture plates allowed grow 1 3 days addition     <span id='am-25' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-26' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cer</span> cells invasive ones [8]. extend early finding, MTT. Cell proliferation determined reading plates          performed immunoblot analysis examine c-Jun abundance <br /> 595 nm dissolving MTT formazan DMSO.                         10 established breast cancer cell lines included 5 invasive                                                                         5 invasive lines (Figure 1B). High level abundance c-Jun <br /> Cell Migration Assay                                                    seen invasive lines, low barely detectable <br />   ability enforced COP1 GSK3&#206;&#178;S9A expression affect cell   invasive cell lines (Figure 1, B). determine migration MDA-MB-231 MDA-MB-436 cells determined             difference c-Jun protein stability invasive <br />  1078        Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                           Neoplasia Vol. 15,  9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 1. Abundance c-Jun breast cancer cells determined protein stability.   Immunoblot analysis c-Jun breast cancer <br /> cell lines. (B) vitro invasion breast cancer cell lines. Data means &#194;&#177; SE (n = 3). (C) Cells transfected HA-tagged <br /> c-Jun GFP 24 hours followed addition 20 &#206;&#188;g/ml cycloheximide varying times. Cells lysed, cell lysates subjected immunoblot analysis detect HA-tagged c-Jun GFP. relative HA-tagged c-Jun standardized GFP. <br /> (D) Cells treated vehicle (DMSO), 50 &#206;&#188;M PYR-41, 10 &#206;&#188;M MG-132 8 hours lysed immunoblot analysis detect <br /> c-Jun &#206;&#178;-actin. (E) T47D cells treated 10 &#206;&#188;M MG-132 varying times lysed immunoblot analysis detect <br /> c-Jun &#206;&#178;-actin. (F) Cells treated 50 &#206;&#188;M chloroquine, 10 mM NH4Cl, 10 &#206;&#188;M MG-132 8 hours lysed immunoblot <br /> analysis detect c-Jun &#206;&#178;-actin. <br />  <br />  <br />  invasive lines, transfected HA-tagged c-Jun GFP         ure 1C ), results indicate greater c-Jun protein stability invasive (MDA-MB-231 MDA-MB-436)                 contributing factor high c-Jun abundance invasive breast <br /> invasive lines (MCF7 T47D) followed performing cyclo-               cancer cells. <br /> heximide-chasing experiment measure stability HA-tagged             c-Jun protein degraded ubiquitin/proteasome- <br /> c-Jun protein lines. HA-tagged c-Jun relatively stable        dependent mechanism various cell types, including melanoma detected 40% reduction                lung cancer cells [25,26]. determine ubiquitin/ <br /> HA-tagged c-Jun MDA-MB-231 MDA-MB-436 cells                   proteasome plays role regulating c-Jun abundance breast <br /> 2 hours cycloheximide addition (Figure 1C ). contrast,          cancer cells, treated MCF7, MDA-MB-231, MDA-MB-436, 80% HA-tagged c-Jun degraded time                  T47D cells ubiquitin E1 ligase inhibitor PYR-41 [27] <br /> period T47D MCF7 cells (Figure 1C). GFP exhibited           proteasome inhibitor MG132 8 hours. Immunoblot analysis <br /> similar protein stability invasive invasive lines (Fig-   showed treatment inhibitor markedly increased  Neoplasia Vol. 15,  9, 2013                                                 Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.    1079 <br />  c-Jun invasive cells (MCF7 T47D), did        Status c-Jun Poly-ubiquitination Determines c-Jun significantly alter c-Jun abundance invasive cells (MDA-MB-231       Abundance Breast Cancer Cells MDA-MB-436; Figure 1D). MG132-induced accumulation                      determine difference c-Jun protein stability c-Jun protein rapid; 1 hour MG132 treatment elevated            linked variation proteasome activity invasive fold c-Jun level T47D cells (Figure 1E).        invasive cells, examined subcellular localization results suggest ubiquitin/proteasome responsible         c-Jun MCF7 MDA-MB-436 cells. Immunoblot analysis efficient c-Jun protein degradation invasive breast cancer cells.   cytoplasmic nuclear fractions showed c-Jun simi- lysosome regulate protein stability, treated          larly present nucleus lines, nuclear localization <br /> MCF7 T47D cells lysosome inhibitors chloroquine                c-Jun affected MG132 treatment (Figure 2A). NH4Cl 8 hours. Immunoblot analysis showed signifi-         measured proteasome activities nucleus MCF7, <br /> cantly altered c-Jun level (Figure 1F), excluding participation    MDA-MB-231, MDA-MB-436, T47D cells. 1-hour lysosome c-Jun protein degradation.                                   reaction period, did observe significant difference nuclear <br />  <br />  <br />  <br />  <br /> Figure 2. extent poly-ubiquitination c-Jun different invasive invasive breast cancer cells.   Cells treated 10 &#206;&#188;M MG132 left untreated 8 hours. Cells lysed preparation cytoplasmic (Cy) nuclear (Nu) frac- <br /> tions, fractions analyzed immunoblot analysis detect c-Jun histone deacetylase 1 (HDAC1; indicator nuclear fraction). <br /> (B) Proteasome activities breast cancer cell lines. Data means &#194;&#177; SE (n = 3). (C) Immunofluorescence staining c-Jun PMSA5 MDA-MB-231 10 &#206;&#188;M MG-132&#226;&#8364;&#8220;treated T47D cells. DAPI used nuclear staining. (D) Cells transfected HA-tagged <br /> c-Jun Flag-tagged ubiquitin 1 day followed 10 &#206;&#188;M MG-132 treatment 8 hours. Cells lysed, cell lysates immuno- <br /> precipitated HA mAb, immunoprecipitates subjected immunoblot analysis detect Flag-tagged ubiquitin (Ub). <br />  1080        Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                         Neoplasia Vol. 15,  9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 3. COP1 E3 ligase mediating c-Jun turnover breast cancer cells.   T47D MCF7 cells transfected 50 nM <br /> siRNA pool CUL4, FBW7, ITCH, MEKK1, COP1, SAG 3 days. Cells lysed, cell lysates analyzed immunoblot analysis detect c-Jun, COP1, &#206;&#178;-actin. (B) T47D MCF7 transfected sequence-distinct COP1 siRNAs <br /> (50 nM) lysed immunoblot analysis detect c-Jun, COP1, &#206;&#178;-actin. (C) T47D cells transfected 50 nM control COP1 siRNA 2 days transfected HA-tagged c-Jun GFP 1 day. adding 20 &#206;&#188;g/ml cycloheximide varying times, cells harvested cell lysates analyzed immunoblot analysis detect HA-tagged c-Jun, GFP, COP1. relative HA-tagged c-Jun standardized GFP. (D) T47D cells treated 50 nM control COP1 siRNA 2 days transfected HA-tagged c-Jun Flag-tagged ubiquitin 1 day. Cells treated 10 &#206;&#188;M MG-132 treatment 8 hours subjected immunoprecipitation HA mAb. Immunoprecipitates analyzed immuno- <br /> blot analysis detect Flag-tagged ubiquitin (Ub). (E) Immunoblot analysis c-Jun COP1 breast cancer cell lines. <br />  <br />  <br /> proteasome activity lines (Figure 2B). Immunofluores-          precipitates anti&#226;&#8364;&#8220;Flag mAb showed HA-tagged c-Jun cence staining showed similar pattern c-Jun protea-       robustly poly-ubiquitinated MCF-7 T47D cells, somal PMSA5 localization MDA-MB-231 MG132-treated                poly-ubiquitination HA-tagged c-Jun weak MDA- <br /> T47D cells (Figure 2C ). Identical pattern observed            MB-231 MDA-MB-436 cells (Figure 2D). results suggest <br /> MDA-MB-436 MG132-treated MCF7 cells (Figure W1).                 effectiveness c-Jun poly-ubiquitinated determines <br /> results indicate difference c-Jun protein stability    abundance c-Jun breast cancer cells. <br /> invasive invasive breast cancer cells caused variation nuclear proteasome activity localization c-Jun       COP1 E3 Ligase Required c-Jun Degradation proteasome.                                                       Breast Cancer Cells <br />    Subsequently, investigated status c-Jun poly-ubiquitination     c-Jun substrate CUL4, FBW7, ITCH, MEKK1, COP1, MG132-treated MCF7, MDA-MB-231, MDA-MB-436,                         SAG E3 ligases. identify E3 ligase regulates c-Jun <br /> T47D cells transfected HA-tagged c-Jun               abundance breast cancer cells, treated MCF7 T47D cells Flag-tagged ubiquitin. Immunoblot analysis HA immuno-               predesigned siRNA pool E3 ligases.  Neoplasia Vol. 15,  9, 2013                                              Regulation c-Jun COP1 GSK3&#206;&#178;     Shao et al.   1081 <br />  effectively knocked respective targets (Fig-      knockdown T47D cells control T47D cells (Figure 3C ). <br /> ure W2), COP1 siRNA pool markedly elevated               Immunoprecipitation experiment revealed extent c-Jun cell lines (Figure 3A). confirm results gener-   c-Jun poly-ubiquitination significantly reduced COP1- <br /> ated siRNA pool, MCF7 T47D cells treated          knockdown cells compared control cells (Figure 3D). <br /> sequence distinct COP1 siRNAs 3 days. Immunoblot analysis               substantiate findings, investigated potential func- <br /> showed siRNAs increased c-Jun abundance (Figure 3B).        tional link c-Jun COP1 breast cancer cell examined effect COP1 knockdown c-Jun pro-              lines. Immunoblot analysis revealed lines relatively high <br /> tein stability introducing HA-tagged c-Jun control          c-Jun abundance (BT549, Hs578t, MDA-MB-231, MDA- COP1-knockdown T47D cells. Cycloheximide-chasing experi-             436) displayed COP1 relatively low <br /> ment showed HA-tagged c-Jun stable COP1-                c-Jun abundance (BT474, MCF7 T47D; Figure 3E ).  <br />  <br />  <br />  <br /> Figure 4. Activity GSK3&#206;&#178; required efficient c-Jun protein turnover breast cancer cells.   COP1 lentivirally overexpressed MDA-MB-231 MDA-MB-436 cells. Populations cells lysed immunoblot analysis detect COP1, c-Jun, &#206;&#178;-actin. <br /> (B) T47D MCF7 cells treated 10 &#206;&#188;M SP600125, 10 mM LiCl, 10 &#206;&#188;M SB216763, 50 &#206;&#188;M TBB 12 hours lysed immunoblot analysis detect c-Jun &#206;&#178;-actin. (C) HA-tagged c-Jun c-Jun&#226;&#8364;&#8220;T239A transfected T47D cells GFP 1 day followed addition 20 &#206;&#188;g/ml cycloheximide varying times. Cells harvested, cell lysates analyzed immunoblot analysis detect HA-tagged c-Jun GFP. relative HA-tagged c-Jun standardized GFP. <br /> (D) T47D cells transfected HA-tagged c-Jun Flag-tagged ubiquitin 1 day treated 10 &#206;&#188;M MG-132 8 hours. Cells lysed, cell lysates subjected immunoprecipitation HA mAb. Immunoprecipitates analyzed immunoblot analysis detect Flag-tagged ubiquitin (Ub). (E) MDA-MB-231 MDA-MB-436 cells lentivirally transduced COP1 constitutively active GSK3&#206;&#178; 3 days. Cells lysed, cell lysates subjected immunoblot <br /> analysis detect COP1, GSK3, c-Jun, &#206;&#178;-actin. <br />  1082       Regulation c-Jun COP1 GSK3&#206;&#178;   Shao et al.                                          Neoplasia Vol. 15,  9, 2013 <br />  <br />  <br />  <br />  <br /> Figure 5. Simultaneously expressing COP1 constitutively active GSK3&#206;&#178; suppress breast tumorigenesis.   MDA-MB-231 MDA-MB-436 cells lentivirally transduced COP1 constitutively active GSK3&#206;&#178; 3 days. Popula- <br /> tions transduced cells subjected MTT assay measure cell proliferation. Data means &#194;&#177; SE (n = 3). #P &lt; .01 <br /> versus vector (control). (B) Populations transduced cells subjected transwell assay measure cell migration. Data means &#194;&#177; SE (n = 3). *P &lt; .001 versus vector (control). (C) Control COP1/GSK3&#206;&#178;S9A-transduced MDA-MB-231 cells labeled fluorescent dye CM-Dil microinjected perivitelline space 48-hpf zebrafish embryos. Embryos imaged confocal microscope. <br />  <br />  <br /> inverse correlation c-Jun COP1 abundance implicates     cells, populations transduced cells analyzed c-Jun COP1 negatively regulates abundance c-Jun breast     abundance. Immunoblot analysis showed COP1 overexpression <br /> cancer cells.                                                       unable decrease level c-Jun (Figure 4A), suggesting                                                                     event additional high COP1 expression required rapid <br />                                                                     c-Jun protein degradation. <br /> c-Jun Protein Degradation Sensitive Inhibition            activities CK2, JNK, GSK3 shown regu- <br /> GSK3 Activity                                                       late c-Jun activity stability [17,20&#226;&#8364;&#8220;22]. Especially, recognition    inverse correlation c-Jun COP1 abundance         c-Jun E3 ligase FBW7 necessitates GSK3-mediated phosphoryla- <br /> prompted test enforcing COP1 expression invasive   tion c-Jun [22]. test phosphorylation CK2, JNK, breast cancer cells downregulate c-Jun.      GSK3 required c-Jun turnover, MCF7 T47D cells lentivirally introduced COP1 MDA-MB-231 MDA-MB-436         treated specific inhibitors JNK (SP600125), GSK3 (LiCl  Neoplasia Vol. 15,  9, 2013                                                 Regulation c-Jun COP1 GSK3&#206;&#178;          Shao et al.     1083 <br />  <br /> SB216763), CK2 (TBB). Immunoblot analysis showed               Simultaneously Enhancing COP1 Expression Elevating <br /> GSK inhibitors JNK CK2 inhibitor led significant         GSK3&#206;&#178; Activity Suppress Breast Tumorigenesis <br /> increase c-Jun (Figure 4B), indicating necessity       previously showed presence c-Jun essential GSK3 activity instability c-Jun protein. generated       cell migration metastasis invasive breast cancer cells [8]. HA-tagged non&#226;&#8364;&#8220;GSK3&#206;&#178;-phosphorylable c-Jun (c-Jun&#226;&#8364;&#8220;T239A)                   reported c-Jun promotes breast cancer cell prolifera- <br /> performing Thr&#226;&#8224;&#8217;Ala mutagenesis <span id='am-11' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> residue 239                 tion controlling expression cyclins E2F [7]. c-Jun, GSK phosphorylation site. Cycloheximide-chasing experi-          hypothesized reduced c-Jun abundance combined expres- <br /> ment showed HA-tagged c-Jun&#226;&#8364;&#8220;T239A significantly               sion COP1 constitutively active GSK3&#206;&#178; lead stable HA-tagged c-Jun (Figure 4A), depletion COP1              suppression breast tumorigenesis. test hypothesis, len- <br /> did affect stability HA-tagged c-Jun&#226;&#8364;&#8220;T239A (Figure W3).          tivirally introduced COP1 GSK3&#206;&#178;S9A com- <br /> Compared HA-tagged c-Jun, HA-tagged c-Jun&#226;&#8364;&#8220;T239A                      bination MDA-MB-231 MDA-MB-436 cells. MTT assay <br /> poorly poly-ubiquitinated (Figure 4D). results suggest           showed raising COP1 level enforcing GSK3&#206;&#178;S9A expression <br /> GSK3&#206;&#178;-mediated phosphorylation prerequisite efficient poly-        moderately reduced cell proliferation, expression <br /> ubiquitination c-Jun subsequent degradation.                     led greater degree inhibition cell proliferation (Figure 5A). <br />    enforced expression COP1 constitutive active           Transwell migration assay showed forced expression COP1 <br /> GSK3&#206;&#178; (GSK3&#206;&#178;S9A) MDA-MB-231                 GSK3&#206;&#178;S9A blocked 90% cell migration, MDA-MB-436 cells. did reduce                  expressing inhibited 20% 40% cell migra- c-Jun, combination diminished 90%             tion compared control (Figures 5B W4). 70% c-Jun MDA-MB-231 MDA-MB-436 cells, respec-                    analyzed effect combined COP1 GSK3&#206;&#178;S9A expression <br /> tively (Figure 4E ). results suggest COP1 GSK3&#206;&#178;              metastasis MDA-MB-231 MDA-MB-436 cells using work facilitate effective c-Jun protein degradation breast   established zebrafish metastasis model [24,28,29]. Control cells <br /> cancer cells.                                                               cells expression COP1 GSK3&#206;&#178;S9A                                                                             microinjected perivitelline space 48-hpf zebrafish embryos, <br />                                                                             embryos imaged 2 days. Metastasis detected <br />                                                                             44 48 46 56 embryos injected control MDA-MB- <br />                                                                             231 MDA-MB-436 cells, respectively (Figures 5C W5).                                                                             contrast, metastasis seen 10 54 4 42 embryos <br />                                                                             injected MDA-MB-231 MDA-MB-436 cells expressing <br />                                                                             COP1 GSK3&#206;&#178;S9A, respectively, remaining embryos dis- <br />                                                                             played cells site injection (Figures 5C W5).                                                                             results demonstrate simultaneously enhancing COP1 expression <br />                                                                             elevating GSK3&#206;&#178; activity effectively suppress breast cancer <br />                                                                             cell tumorigenesis. <br />  <br />                                                                             COP1 Expression Correlates Recurrence-Free Survival <br />                                                                             Patients Breast Cancer <br />                                                                                laboratory study recapitulate clinical breast <br />                                                                             malignancy, explored potential clinical relevance COP1 <br />                                                                             expression human breast cancer.  assessed                                                                             relationship COP1 mRNA expression clinical features <br />                                                                             breast tumors using microarray data 126 patients breast  <br />                                                                             cer obtained Gene Expression Omnibus.                                                                             significant associations COP1 expression (divided low <br />                                                                             high expression groups median) pathologic parameters, <br />                                                                             including histologic grade, tumor diameter, estrogen proges- <br />                                                                             terone receptor expression, lymph node status (Table W1).  <br />                                                                              univariate survival analysis (Kaplan-Meier method, log-rank <br />                                                                             test) showed higher COP1 expression significantly associated <br />                                                                             high recurrence-free survival patients (P = .006; Figure 6A). <br />                                                                             multivariate analysis (proportional hazard method) encom- <br />                                                                             passing basic prognostic factors tumor diameter, histologic <br /> Figure 6. COP1 expression positive prognostic factor               grade, lymph node status addition COP1 expression (initial <br /> recurrence-free survival patients breast cancer.   Univariate     regression model), COP1 expression status remained indepen- <br /> survival method (Kaplan-Meier method) patients breast               dent positive prognostic factor final multivariate model (P = <br /> cancer indicates strong positive prognostic value COP1 expres-         .041, likelihood ratio test) tumor diameter lymph <br /> sion (P = .006, log-rank test). (B) Multivariate survival analysis <br /> (proportional hazard method) shows positive, independent prog- <br />                                                                             node status, hazard ratio 1.69 patients high <br /> nostic importance COP1 expression (P &lt; .05, likelihood ratio             low COP1 expression  e., patients low COP1 expression <br /> test), addition independent prognostic impact tumor            1.69 times higher hazard recurrence breast cancer <br /> diameter lymph node status. HR, hazard ratio.                           high COP1 expression; Figure 6B). results <br />  1084         Regulation c-Jun COP1 GSK3&#206;&#178;        Shao et al.                                                      Neoplasia Vol. 15,  9, 2013 <br />  support notion COP1 negative regulator breast          COP1 promotes tumorigenesis consequence tumor  <br /> tumor progression.                                                           mation;  mice partial loss COP1 function tissue- <br />                                                                              specific COP1 ablation strongly suggest COP1 tumor <br />                                                                              suppressor [14,35]. Using publicly available breast tumor gene expres- <br /> Discussion                                                                   sion microarray database, reduced COP1 expression <br /> High c-Jun abundance detected invasive breast cancer cell         correlated poorer disease-free survival COP1 expression lines specimens aggressive breast tumors [8,11].              independent positive prognostic factor recurrence-free survival c-Jun abundance regulated breast cancer cells          (Figure 6). study adds strong evidences COP1 suppressor <br /> understood. study, showed c-Jun protein         breast tumor progression. <br /> stable invasive breast cancer cells invasive ones (Figure 1),      mechanistic study performed established <br /> indicating prolonged protein stability major cause high         breast cancer cell lines, observation high COP1 expression <br /> c-Jun abundance invasive breast cancer cells aggressive breast         correlated better disease-free survival patients breast <br /> tumor specimens [8,11]. stability c-Jun protein breast             cancer supports results generated experimental studies. <br /> cancer cells appeared depend c-Jun efficiently             conclude, study suggests COP1 GSK3&#206;&#178; cooperate <br /> poly-ubiquitinated nuclear proteasome activity            promote c-Jun degradation   subsequent inhibition ability c-Jun localize proteasomes (Figure 2).        breast tumor progression metastasis. <br /> findings led hypothesize lack c-Jun&#226;&#8364;&#8220;targeting E3 <br /> ubiquitin ligase invasive breast cancer cells. aid target- <br /> specific siRNAs, identified COP1 E3 ligase responsible          References <br /> c-Jun turnover breast cancer cells COP1 depletion enhanced              [1] Karin M, Liu  Zandi E (1997). AP-1 function regulation. Curr <br />                                                                                   Opin Cell Biol 9, 240&#226;&#8364;&#8220;246. <br /> c-Jun protein stability diminished c-Jun poly-ubiquitination                                                                               [2] Shaulian E Karin M (2002). AP-1 regulator cell life death. Nat invasive breast cancer cells (Figure 3). human primary T lym-             Cell Biol 4, E131&#226;&#8364;&#8220;E136. <br /> phocytes, proto-oncogene c-Abl phosphorylates c-Jun, turn            [3] Smith LM, Wise SC, Hendricks DT, Sabichi AL, Bos T, Reddy P, Brown PH, <br /> prevents c-Jun ITCH interaction stabilizes c-Jun [30].               Birrer MJ (1999). cJun overexpression MCF-7 breast cancer cells pro- <br /> Receptor activated C-kinase 1 stabilizes c-Jun releasing           duces tumorigenic, invasive hormone resistant phenotype. Oncogene 18, <br /> FBW7 complex [31]. study, showed levels                    6063&#226;&#8364;&#8220;6070. <br />                                                                               [4] Vleugel MM, Greijer AE, Bos R, van der Wall E, van Diest PJ (2006). <br /> c-Jun COP1 inversely correlated breast cancer cells (Fig-              c-Jun activation associated proliferation angiogenesis invasive <br /> ure 3), implicating low abundance COP1 principal cause                breast cancer. Hum Pathol 37, 668&#226;&#8364;&#8220;674. prolonged c-Jun protein stability invasive breast cancer cells.         [5] Jiao X, Katiyar S, Willmarth NE, Liu M, Ma X, Flomenberg N, Lisanti MP, <br />    Phosphorylation specific site prerequisite ubiquitina-          Pestell RG (2010). c-Jun induces mammary epithelial cellular invasion tion E3 ubiquitin ligases. example, Skp2 ubiquitinates               breast cancer stem cell expansion. J Biol Chem 285, 8218&#226;&#8364;&#8220;8226. <br />                                                                               [6] Zhang Y, Pu X, Shi M, Chen L, Song Y, Qian L, Yuan G, Zhang H, Yu M, promotes degradation forkhead box protein O1 (FOXO1).                  Hu M, et al. (2007). Critical role c-Jun overexpression liver metastasis  Skp2-mediated event requires Akt-specific phosphoryla-              human breast cancer xenograft model. BMC Cancer 7, 145. <br /> tion FOXO1 Ser-256 [32]. c-Jun ubiquitinated FBW7              [7] Shen Q, Uray IP, Li Y, Krisko TI, Strecker TE, Kim HT, Brown PH <br /> various cell types, GSK3&#206;&#178;-mediated phosphorylation Thr-239                 (2008). AP-1 transcription factor regulates breast cancer cell growth                                                                                   cyclins E2F factors. Oncogene 27, 366&#226;&#8364;&#8220;377. necessary c-Jun ubiquitination place [22]. study, <br />                                                                               [8] Chen H, Zhu G, Li Y, Padia RN, Dong  Pan ZK, Liu K, Huang S (2009). showed blocking GSK3 activity increased c-Jun abundance                   Extracellular signal&#226;&#8364;&#8220;regulated kinase signaling pathway regulates breast cancer cell invasive breast cancer cells (Figure 4). Specifically, mutating           migration maintaining slug expression. Cancer Res 69, 9228&#226;&#8364;&#8220;9235. <br /> Thr-239 Ala stabilized c-Jun protein prevented c-Jun poly-             [9] Okutomi Y, Shino Y, Komoda F, Hirano T, Ishihara T, Yamaguchi T, Saisho <br /> ubiquitination (Figure 4). data suggest GSK3&#206;&#178;-mediated              H, Shirasawa H (2003). Survival regulation pancreatic cancer cells phosphorylation Thr-239 event required COP1-                c-Jun. Int J Oncol 23, 1127&#226;&#8364;&#8220;1134. <br />                                                                              [10] Neyns B, Katesuwanasing, Vermeij J, Bourgain C, Vandamme B, Amfo K, <br /> FBW7-mediated c-Jun protein degradation. possibility              Lissens W, DeSutter P, Hooghe-Peters E, DeGr&#195;&#168;ve J (1996). Expression <br /> supported observation simultaneously overexpressing COP1              jun family genes human ovarian cancer normal ovarian surface elevating GSK3&#206;&#178; activity effectively diminish c-Jun abundance             epithelium. Oncogene 12, 1247&#226;&#8364;&#8220;1257. invasive breast cancer cells (Figure 4).                                  [11] Tiniakos DG, Scott LE, Corbett IP, Piggott NH, Horne CH (1994). Studies <br />                                                                                   c-jun oncogene expression human breast using new monoclonal antibody, <br />    vitro vivo studies demonstrated convincing role <br />                                                                                   NCL-DK4. J Pathol 172, 19&#226;&#8364;&#8220;26. c-Jun breast tumorigenesis. example, enforced expression        [12] Jariel-Encontre  Salvat C, Steff  Pariat M, Acquaviva C, Furstoss O, c-Jun confers noninvasive hormone-dependent MCF7 cells                   Piechaczyk M (1997). Complex mechanisms c-fos c-jun degradation. invasive hormone-resistant phenotype [3]. MMTV-ErbB2                    Mol Biol Rep 24, 51&#226;&#8364;&#8220;56. <br /> transgenic breast cancer model, c-Jun deficiency led reduced cell         [13] Westermarck J (2010). Regulation transcription factor function targeted <br /> migration, invasion, mammosphere formation erbB2-induced                   protein degradation: overview focusing p53, c-Myc, c-Jun. Methods <br />                                                                                   Mol Biol 647, 31&#226;&#8364;&#8220;36. <br /> mammary tumors [5]. study, showed simultaneously             [14] Migliorini D, Bogaerts S, Defever D, Vyas R, Denecker G, Radaelli E, Zwolinska <br /> raising COP1 level elevating GSK3&#206;&#178; activity suppressed vitro                Depaepe V, Hochepied T, Skarnes WC, et al. (2011). Cop1 constitutively <br /> cell growth/migration vivo metastasis invasive breast cancer            regulates c-Jun protein stability functions tumor suppressor mice. <br /> cells (Figure 5). Taken  study presents strong evidence              J Clin Invest 121, 1329&#226;&#8364;&#8220;1343. high COP1 expression GSK3&#206;&#178; activity negatively impact          [15] Wertz  O&#226;&#8364;&#8482;Rourke KM, Zhang  Dornan D, Arnott D, Deshaies RJ,                                                                                   Dixit VM (2004). Human etiolated-1 regulates c-Jun assembling CUL4A <br /> breast tumor progression metastasis.                                          ubiquitin ligase. Science 303, 1371&#226;&#8364;&#8220;1374. <br />    evidences regulation loss COP1            [16] Nateri  Riera-Sans L, Da Costa C, Behrens (2004). ubiquitin <br /> cancers [14,33,34]. remains determined overexpression              ligase SCFFbw7 antagonizes apoptotic JNK signaling. Science 303, 1374&#226;&#8364;&#8220;1378. <br />  Neoplasia Vol. 15,  9, 2013                                                          Regulation c-Jun COP1 GSK3&#206;&#178;                 Shao et al.        1085 <br />  <br /> [17] Gao M, Labuda T, Xia Y, Gallagher E, Fang D, Liu YC, Karin M (2004).       [27] Yang Y, Kitagaki J, Dai RM, Tsai YC, Lorick KL, Ludwig RL, Pierre SA, Jensen <br />      Jun turnover controlled JNK-dependent phosphorylation E3          JP, Davydov IV, Oberoi P, et al. (2007). Inhibitors ubiquitin-activating enzyme <br />      ligase Itch. Science 306, 271&#226;&#8364;&#8220;275.                                                  (E1), new class potential cancer therapeutics. Cancer Res 67, 9472&#226;&#8364;&#8220;9481. <br /> [18] Xia Y, Wang J, Xu S, Johnson GL, Hunter T, Lu (2007). MEKK1              [28] Zhang L, Zhou F, Drabsch Y, Gao R, Snaar-Jagalska  Mickanin C, Huang H, <br />      mediates ubiquitination degradation c-Jun response osmotic         Sheppard KA, Porter JA, Lu CX, et al. (2012). USP4 regulated AKT phos- <br />      stress. Mol Cell Biol 27, 510&#226;&#8364;&#8220;517.                                                  phorylation directly deubiquitylates TGF-&#206;&#178; type receptor. Nat Cell Biol 14, <br /> [19] Gu Q, Bowden GT, Normolle D, Sun Y (2007). SAG/ROC2 E3 ligase                   717&#226;&#8364;&#8220;726. <br />      regulates skin carcinogenesis stage-dependent targeting c-Jun/AP1    [29] Thomas C, Rajapaksa G, Nikolos F, Hao R, Katchy  McCollum CW, <br />      I&#206;&#186;B-&#206;&#177;/NF-&#206;&#186;B. J Cell Biol 178, 1009&#226;&#8364;&#8220;1023.                                            Bondesson M, Quinlan P, Thompson  Krishnamurthy S, et al. (2012). ER&#206;&#178;1 <br /> [20] Lin  Frost J, Deng T, Smeal T, al-Alawi N, Kikkawa U, Hunter T, Brenner           represses basal-like breast cancer epithelial mesenchymal transition      D, Karin M (1992). Casein kinase II negative regulator c-Jun DNA        destabilizing EGFR. Breast Cancer Res 14, R148. <br />      binding AP-1 activity. Cell 70, 777&#226;&#8364;&#8220;789.                                   [30] Gao B, Lee SM, Fang D (2006). tyrosine kinase c-Abl protects <br /> [21] Derijard B, Hibi M, Wu IH, Barrett T, Su B, Deng T, Karin M, Davis RJ           c-Jun ubiquitination-mediated degradation T cells. J Biol Chem 281, <br />      (1994). JNK1: protein kinase stimulated UV light Ha-Ras binds         29711&#226;&#8364;&#8220;29718. <br />      phosphorylates c-Jun activation domain. Cell 76, 1025&#226;&#8364;&#8220;1037.            [31] Zhang J, Zhu F, Li X, Dong  Xu Y, Peng C, Li S, Cho YY, Yao K, Zykova <br /> [22] Wei W, Jin J, Schlisio S, Harper JW, Kaelin Jr (2005). v-Jun             TA, et al. (2012). Rack1 protects N-terminal phosphorylated c-Jun      point mutation allows c-Jun escape GSK3-dependent recognition                Fbw7-mediated degradation. Oncogene 31, 1835&#226;&#8364;&#8220;1844. <br />      destruction Fbw7 ubiquitin ligase. Cancer Cell 8, 25&#226;&#8364;&#8220;33.                [32] Huang H, Regan KM, Wang F, Wang D, Smith DI, van Deursen JM, [23] Hong S, Noh H, Chen H, Padia R, Pan ZK, Su SB, Jing Q, Ding HF,                 Tindall DJ (2005). Skp2 inhibits FOXO1 tumor suppression      Huang S (2013). Signaling p38 MAPK stimulates nuclear localization        ubiquitin-mediated degradation. Proc Natl Acad Sci USA 102, 1649&#226;&#8364;&#8220;1654. <br />      microprocessor component p68 processing selected primary microRNAs.     [33] Dornan D, Bheddah S, Newton K, Ince W, Frantz GD, Dowd P, Koeppen <br />      Sci Signal 6, ra16.                                                                 H, Dixit VM, French DM (2004). COP1, negative regulator [24] Rouhi P, Jensen LD, Cao  Hosaka K, Lanne T, Wahlberg E, Steffensen JF,            p53, overexpressed breast ovarian adenocarcinomas. Cancer Res 64, <br />      Cao Y (2010). Hypoxia-induced metastasis model embryonic zebrafish.          7226&#226;&#8364;&#8220;7230. <br />      Nat Protoc 5, 1911&#226;&#8364;&#8220;1918.                                                       [34] Lee YH, Andersen JB, Song HT, Judge AD, Seo D, Ishikawa T, Marquardt <br /> [25] Lopez-Bergami P, Huang C, Goydos JS, Yip D, Bar-Eli M, Herlyn M, Smalley            JU, Kitade M, Durkin  Raggi C, et al. (2010). <span id='am-10' about='obo:IAO_0000115' typeof='owl:Thing'>Definition</span> ubiquitination <br />      KS, Mahale  Eroshkin  Aaronson S, et al. (2007). Rewired ERK-JNK sig-           modulator COP1 novel therapeutic target human hepatocellular carcinoma. <br />      naling pathways melanoma. Cancer Cell 11, 447&#226;&#8364;&#8220;460.                               Cancer Res 70, 8264&#226;&#8364;&#8220;8269. <br /> [26] H, Gu Q, Zheng M, Normolle D, Sun Y (2008). SAG/ROC2/RBX2               [35] Vitari AC, Leong KG, Newton K, Yee C, O&#226;&#8364;&#8482;Rourke K, Liu J, Phu L, Vij R, <br />      E3 ligase promotes UVB-induced skin hyperplasia, skin tumors,            Ferrando R, Couto SS, et al. (2011). COP1 tumour suppressor causes <br />      simultaneously targeting c-Jun/AP-1 p27. Carcinogenesis 29, 858&#226;&#8364;&#8220;865.            degradation ETS transcription factors. Nature 474, 403&#226;&#8364;&#8220;406. <br />  Supplemental Information                                              FBW7 F      AATGGCCAAGGGCAACAA <br />  <br /> Antibody Information                                                  FBW7 R      CATTCCTGGAGGCCTGTAGGT <br />  <br />     c-Jun rabbit mAb: CS-9165, Cell Signaling Technology (Danvers,    ITCH F      GATGGCTCCAGATCCAAGGA <br />     MA); titer 1:1000 (immunoblot analysis) <br />                                                                       ITCH R      TGCATCTTCAGGGTCATCTGAT <br />     COP1 rabbit polyclonal antibody: A300-894A, Bethyl Laboratories <br />     (Montgomery, TX); titer 1:1000 (immunoblot analysis)              CUL4A F     GAATCAGAGTGACTCAGGCCCTATA <br />     GSK3 rabbit mAb: SC-5676, Cell Signaling Technology; <br />     titer 1:1000 (immunoblot analysis)                                CUL4A R     GGCCAGTAGCCCATTGTGA <br />     HA rabbit mAb: CS-5017, Cell Signaling Technology;                SAG F       AACAATCGCTGCCCTCTCT <br />     titer 1:1000 (immunoblot analysis) <br />     &#206;&#178;-Actin mouse mAb: SC-47778, Santa Cruz Biotechnology (Santa      SAG R       CTCATTTGCCGATTCTTTGGA <br />     Cruz, CA); titer 1:500 (immunoblot analysis) <br />                                                                       MEKK1 F     CAAACCGCCGTGTTAACAAA <br />     Flag mouse mAb: F 3165, Sigma; titer 1:1000 (immuno- <br />     blot analysis)                                                    MEKK1 R     GGCCCTATCTGCTGCAGTAAGT <br />     c-Jun mouse mAb: BD-610327, BD Biosciences (San Jose, CA); <br />     titer 1:100 (immunofluorescence staining)                         &#206;&#178;-Actin F   CCAGCTCACCATGGATGATG <br />     PSMA5 (proteasome S5A subunit) rabbit polyclonal antibody:        &#206;&#178;-Actin R   ATGCCGGAGCCGTTGTC <br />     CS-2457, Cell Signaling Technology; titer 1:50 (immuno- <br />     fluorescence staining) <br />                                                                       Sequences COP1 siRNAs <br /> Information Primers Used Quantitative Reverse <br /> Transcription&#226;&#8364;&#8220;Polymerase Chain Reaction                                   siCOP1 #1, AAAAGAAAUGACCUGCAAUUCGATTGGA- <br />                                                                           TCCAATCGAATTGCAGGTCATTTC <br /> COP1 F        CCACAGGTGGCAGATATTTCAA <br />                                                                           siCOP1 #2, AAAAAAACAGCCUUGGUAUAAUATTGGA- <br /> COP1 R        CATTGGCCAAATCAAGGTTATCTT                                    TCCAATATTATACCAAGGCTGTTT <br />  Figure W1. Colocalization c-Jun PMSA5. Immunofluorescence staining c-Jun PMSA5 MDA-MB-436 10 &#206;&#188;M MG132- <br /> treated MCF7 cells. DAPI used stain nucleus. <br />  <br />  <br />  <br />  <br />                                   Figure W2. siRNA-mediated knockdown effect c-Jun&#226;&#8364;&#8220;targeting <br />                                   E3 ligases. MCF7 T47D cells transfected 50 nM  <br />                                   trol siRNA pool CUL4, FBW7, ITCH, MEKK1, COP1,                                   SAG 3 days. Total RNA extracted cells                                   subjected quantitative reverse transcription&#226;&#8364;&#8220;polymerase chain <br />                                   reaction respective primers. &#206;&#178;-Actin mRNA used                                   internal control standardization. <br />  <br />  <br />  <br />  <br /> Figure W3. Depletion COP1 does alter protein stability c-Jun&#226;&#8364;&#8220;T239A. T47D cells treated control COP1 siRNA 2 days. HA-tagged c-Jun&#226;&#8364;&#8220;T239A transfected cells GFP 1 day followed addition 20 &#206;&#188;g/ml cyclo- <br /> heximide varying times. Cells harvested, cell lysates analyzed immunoblot analysis detect HA-tagged c-Jun GFP. relative HA-tagged c-Jun standardized GFP. <br />  Figure W4. Simultaneously expressing COP1 constitutively                     Figure W5. Simultaneously expressing COP1 constitutively <br /> active GSK3&#206;&#178; block cell migration invasive breast cancer cells.               active GSK3&#206;&#178; suppress breast cancer cell metastasis. MDA-MB-436 <br /> COP1 GSK3&#206;&#178;(CA) lentivirally transduced MDA-MB-231                  cells lentivirally transduced COP1 constitutively active MDA-MB-436 cells  Populations trans-             GSK3&#206;&#178; 3 days. Control COP1/GSK3&#206;&#178;S9A-transduced <br /> duced cells added transwells allowed migrate                MDA-MB-436 cells labeled fluorescent dye CM-Dil <br /> 4 hours. Images crystal violet&#226;&#8364;&#8220;stained migratory cells                microinjected perivitelline space 48-hpf zebrafish <br /> undersurface transwells.                                                      embryos. Embryos imaged confocal microscope. <br />  <br />  <br />  <br />                                     Table W1. Clinicopathologic Correlates COP1 Expression Human Breast Cancer. <br />  <br />  <br />                                     Clinical Features               COP1 Expression*                                P Value&#226;&#8364;&#160; <br />  <br />                                                                     Low (%)                 High (%) <br />  <br />                                     Histologic grade                                                                .28 <br />                                       1                              5                       6 <br />                                       2-3                           26                      77 <br />                                     Tumor diameter                                                                  .72 <br />                                       &lt;2 cm                          6                      22 <br />                                       &#226;&#8240;&#165;2 cm                         26                      70 <br />                                     Lymph node status                                                               .71 <br />                                       Negative                      14                      48 <br />                                       Positive                      15                      40 <br />                                     ER receptor                                                                     .46 <br />                                       Negative                      13                      44 <br />                                       Positive                      20                      46 <br />                                     PR receptor                                                                     .89 <br />                                       Negative                      15                      42 <br />                                       Positive                      12                      35 <br />  <br />                                     PR indicates <span id='am-18' about='obo:IMR_0100034' typeof='owl:Thing obo:IMR_0100033 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000966 obo:IMR_0000001'><span id='am-19' property="oboInOwl:id" content="IMR:0100034" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="4-Pregnene-3,20-dione" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="progesterone" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00410" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="4-Pregnene-3,20-dione" datatype="xsd:string"></span>progesterone</span> receptor. <br />                                     *Low/high median (low, COP1 &lt; 0; high, COP1 &#226;&#8240;&#165; 0 expression value). <br />                                     &#226;&#8364;&#160; <br />                                   <br /> </body></html>