<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br /> Changes transcriptional orientation associated increases evolutionary <br /> rates enterobacterial genes <br />  <br />   Abstract <br />   Background: Changes transcriptional orientation (&#226;&#8364;&#339;CTOs&#226;&#8364;&#65533;) occur frequently prokaryotic genomes.   changes usually result genomic inversions, cause conflict directions replication <br />   transcription increase mutation rate.  CTOs lead replication- <br />   transcription confrontation. Furthermore, CTOs cause deleterious disruptions operon structure gene <br />   regulations. currently existing CTOs indicate relaxation selection pressure.    investigate CTOs independent effect evolutionary rates affected genes,   genes subject type selection pressure prokaryotes. <br />   Methods: closely related enterbacteria, Escherichia coli, Klebsiella pneumoniae Salmonella enterica serovar <br />   Typhimurium, selected comparisons synonymous (dS) nonsynonymous (dN) substitution rate <br />   genes experienced changes transcriptional orientation (changed-orientation genes, <br />   &#226;&#8364;&#339;COGs&#226;&#8364;&#65533;)  orientation genes, &#226;&#8364;&#339;SOGs&#226;&#8364;&#65533;). dN/dS ratio derived evaluate   selection pressure analyzed genes. Confounding factors estimation evolutionary rates,   gene essentiality, gene expression level, replication-transcription confrontation, decreased dS gene terminals <br />   controlled COG-SOG comparisons. <br />   Results: demonstrate COGs significantly higher dN dS SOGs series confounding <br />   factors controlled.  dN/dS ratios similar gene groups, suggesting   increase dS sufficiently explain increase dN COGs.  increases evolutionary rates   COGs mainly mutation-driven. <br />   Conclusions: CTOs increase evolutionary rates affected genes. effect   independent replication-transcription confrontation, suggested major cause inversion- <br />   associated evolutionary rate increases. real cause evolutionary rate increases remains unclear   worth explorations. <br />  <br />  <br /> Introduction                                                 major driver changes evolutionary <br /> Genome rearrangements occur frequently evolu-         rates, dN/dS ratios expected differ signifi- <br /> tion prokaryotes. rearrangement events,         cantly gene groups. Clarifying inversions usually occur symmetrically origin     molecular mechanisms prokaryotic genomes <br /> (designated &#226;&#8364;&#339;Ori&#226;&#8364;&#65533;) terminus (&#226;&#8364;&#339;Ter&#226;&#8364;&#65533;) replication     evolve help understand prokaryotes develop closely related bacterial genomes [1].         novel functions, turn relevant ecological <br /> rearrangement events lead changes transcrip-     biomedical studies. <br /> tional orientation (designated &#226;&#8364;&#339;CTOs&#226;&#8364;&#65533;) [2]              Accordingly, compared genomes clo- <br /> increases mutation pressure affected genes         sely related enterobacteria [5,6]: Escherichia coli (ECO), conflict directions transcrip-   Klebsiella pneumonia (KPN) Salmonella enterica <br /> tion replication [1,3,4].  inver-       serovar Typhimurium (STM). series analyses sion events result flipping Ori Ter,      performed control potential confounding factors, <br /> replication-transcription conflict occur [1].        including gene essentiality, expression level, background <br /> Furthermore, CTOs cause changes homology-             mutation rate, pattern replication-transcription <br /> based recombination impediments DNA replica-           confrontation, codon usage bias. potential <br /> tion  altering, example, DNA protein binding          confounding factors reported associated <br /> sites secondary structure), increase muta-      evolutionary rates. example, highly expressed <br /> tion rate.  CTOs lead disruptions        genes, essential genes, genes large codon usage <br /> operon structure transcriptional regulations,      bias tend evolve slowly [7-9].  higher potentially deleterious.  currently existing   mutation rate observed genes near Ter[10] <br /> CTOs signify relaxation selection pressure         genes subject orientation conflicts affected genes.  remains unknown         DNA replication transcription [3]. selection actually plays important role        results suggest COGs significantly higher dN <br /> maintaining CTOs. view potential influences     dS SOGs. increases independent CTOs gene evolution, interested investigat-     analyzed confounding factors, mainly muta- <br /> ing CTO se effect evolution-      tion-driven. <br /> ary rates prokaryotic genes. <br />   Note evolution prokaryotic genes driven     Methods major forces: mutation natural selection.      Data sources <br /> increased mutation rate accelerate short-term            complete genomic sequences gene annotations <br /> <span id='am-1' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> substitutions, retained         Escherichia coli K-12 MG 1655 (GenBank accession <br /> eliminated natural selection according fitness   number U00096), Klebsiella pneumonia MGH 78578 <br /> effects. evolutionary rates prokaryotic genes         (CP000647) Salmonella enterica serovar Typhimur- <br /> mainly reflect combinatorial effects        ium LT2 (AE006468) retrieved GenBank <br /> forces. general, substitutions cause protein         [11]. species selected genomes <br /> sequence changes (nonsynonymous substitutions)           completely sequenced carefully annotated, <br /> subject stronger selection pressure    genome-scale gene expression data (synonymous substitutions). comparing nonsy-          available species. orthologous gene pairs <br /> nonymous (dN) synonymous substitution rate (dS),         identified according reciprocal BLASTP best evaluate mutation selection plays        matches following parameters: (1) E value &lt; 1.0 important role evolution prokaryotic          x 10 -5 ; (2) &gt; 50% <span id='am-9' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> sequence identity. <br /> genes.                                                       orthologous genes   attempt examine (1) effects CTOs         compared species retained. total 2,574 prokaryotic gene evolution; (2) relative      orthologous gene groups tributions abovementioned driving forces       obtained subsequent analyses. ana- <br /> dS dN genes experienced CTOs.           lyses measured evolutionary rates separately end, compared dS dN           terminal central regions individual genes, genes experienced CTOs (changed-orientation        analyzed genes sufficiently long reduce varia- <br /> genes, &#226;&#8364;&#339;COGs&#226;&#8364;&#65533;)  orientation       tions evolutionary rate estimates. previous study <br /> genes, &#226;&#8364;&#339;SOGs&#226;&#8364;&#65533;) closed related prokaryotic genomes.        suggested 50 20 codons usually case mutation-driven evolution, COGs           evolved slowly rest [12,13].  SOGs significantly different dS approxi-     took twice length 70 (2*(50+20) = 140) include <br /> mately dN/dS ratios. Alternatively, selection    central region minimal length 70 codons. <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                           Page 3 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> Accordingly, genes shorter 420 bp (140*3)       expression log2-transformed normalized discarded. Note practice lead dif-     using quantile normalization. cross-species compari- <br /> ferences number analyzed gene pairs         sons, gene expression levels species different pair-wise species comparisons. numbers       standardized median value 0 variance 1. <br /> analyzed gene pairs listed Additional file 1. reciprocal-BLASTP approach optimal          Functional enrichment analysis <br /> solution finding orthologous genes, retrieved     Gene Ontology   enrichment analysis  <br /> orthologous genes analyzed species           formed using DAVID Bioinformatics Resources <br /> OMA database comparison [14]. fact, 98%        [20,21]. genes ECO extensively orthologous gene pairs retrieved OMA            studied species, identical identified using reciprocal BLASTP      COGs ECO-STM ECO-KPN comparisons matches  Additional file 2). actually observed         included analysis. gene clustering similar results 98% orthologous genes           term enrichment assessed reference analyzed  e. COGs significantly higher dS         enrichment score p-values modified Fish- dN similar dN/dS compared SOGs;             er&#226;&#8364;&#8482;s Exact test. p&lt;0.05, Additional file 3). <br />   locations Ori Ter determined             Results discussion <br /> using program Oriloc [15].                                COGs higher dN dS similar dN/dS ratio <br />                                                               compared SOGs <br /> Estimation evolutionary rates statistical analyses     pair-wise species comparisons (ECO-KPN, amino acid sequences orthologous gene pairs            STM-KPN, ECO-STM), COGs significantly aligned using ClustalW 2.0 [16] trans-       higher dS dN SOGs  p&lt;0.05).  lated nucleotide sequences. dS dN esti-           dN/dS ratio does significant difference <br /> mated using CODEML module PAML 4.1                  COGs SOGs pair-wise com- <br /> [17]. orthologous gene pairs dS value &lt; 3      parisons (Figure 1). used way (ECO- considered [18]. Note lead dif-   STM-KPN) comparison infer lineage-specific COGs, <br /> ferences number analyzed genes different         &#226;&#8364;&#339;completely conserved SOGs&#226;&#8364;&#65533;  e. genes pair-wise species comparisons. end, 1,784, 1,809       changed transcriptional orientation 2,069 orthologous gene pairs obtained, respec-       compared species). shown Additional file 5, line- <br /> tively, ECO-KPN, STM-KPN, ECO-STM                 age-specific (ECO STM  COGs sig- <br /> comparison (Additional file 1). dS &gt; 3 indi-        nificantly higher dS similar dN/dS ratios cate extremely high mutation rate loss func-        compared completely conserved SOGs. tion, investigated COGs tend          observations suggest increases evolutionary <br /> dS &gt; 3.  proportion COGs dS &gt; 3         rates COGs mainly mutation-driven.  higher of analyzed genes  Addi-      number confounding factors need excluded tional file 4). observation fact consistent   confirm results. confounding factors results COGs tend evolve faster SOGs.        controlled include gene essentiality, gene expres- <br />   Spearman&#226;&#8364;&#8482;s rank correlations partial correlations       sion level, background mutation rate, transcription- evolutionary rates rate-determining fac-      replication confrontation, codon usage bias. <br /> tors performed using R program (http:// <br /> www.r-project.org). statistical significance       Confounding factor 1: gene essentiality <br /> evolutionary rate differences evaluated using      Essential genes known evolutionarily <br /> Wilcoxon Rank Sum test study.                  conserved nonessential genes [8].                                                               necessary clarify COGs tend non- <br /> Analysis gene expression data                              essential, evolve faster SOGs. genome-scale gene expression data ECO                  retrieved gene essentiality information E.coli[22], <br /> (GSE15534) STM (GSE11486) downloaded                 divided orthologous genes essential Gene Expression Omnibus (GEO) database               nonessential genes ECO-KPN ECO-STM <br /> [19]. gene expression data KPN generated          comparisons. results indicate dN dS using custom NimbleGen tiling array (pro-           significantly higher COGs SOGs nones- <br /> vided Dr. Bernhard O. Palsson University Cali-       sential genes (Table 1). observed similar trends fornia, San Diego). data sets, gene   essential genes, differences dN dS <br /> expression levels measured separately log         statistically insignificant possibly small stationary phase. signal intensity gene            sample sizes reduced variations evolutionary <br />  <br />   Figure 1 evolutionary rates COGs SOGs ECO-KPN STM-KPN comparisons. *: p-value &lt; 0.05; **: p-value &lt; 0.01; ***: <br />   p-value &lt; 0.0001 <br />  <br />  <br />  <br /> rates gene group.  dN/dS ratios                                       evolve slowly lowly expressed genes [23,24]. similar COGs SOGs, consistent                                          question SOGs tend highly previous observations.  difference                                     expressed, evolve slowly COGs. level gene essentiality does affect                                  classified analyzed genes highly  result. expected COG-SOG differences                                      20%) non-highly  80%) expressed genes. evolutionary rates mainly mutation-driven.                                              ECO-KPN STM-KPN comparisons, dS                                                                                             COGs significantly larger SOGs highly <br /> Confounding factor 2: gene expression level                                                 non-highly expressed genes growth phases <br /> Highly expressed genes strong selection pres-                                     (stationary log phase, Table 1). dN values sure maintain functional stability, usually                                     similar trends.  dN/dS ratios  <br /> Table 1 median evolutionary rates COGs SOGs different confounding factors controlled. <br /> confounding factors                        comparison                     #gene (COG/                dN                                dS                       dN/dS <br />                                                                              SOG) <br />                                                                                              COGs SOGs       p-     COGs              SOGs      p-      COGs    SOGs      p- <br />                                                                                                             value                              value                     value <br /> nonessential gene <br />                                              ECO-KPN                       121/1426          0.076 0.067      *     1.6811 1.466                ***    0.0481 0.0463       - <br />                                              ECO-STM                       240/1582          0.057 0.045     ***    1.5921 1.1926               ***    0.0395 0.0381       - <br /> essential gene <br />                                              ECO-KPN                        8/224            0.045 0.041      -     1.374             0.978      -      0.042   0.046      - <br />                                              ECO-STM                        17/225           0.025 0.025      -     0.785             0.825      -      0.029   0.032      - <br /> non-highly expressed <br /> gene <br />                                       ECO-KPN (log phase)                  106/1240          0.082 0.074      -     1.721             1.561     **      0.049   0,047      - <br />                                   ECO-KPN (stationary phase)               103/1342          0.077 0.066      -     1.686             1.457     ***     0.048   0.046      - <br />                                       STM-KPN (log phase)                   88/1416          0.077 0.068      -     1.845             1,507     ***     0.042   0.045      - <br />                                   STM-KPN (stationary phase)                80/1475          0.077 0.065      -      1.77             1.458     ***     0.042   0.046      - <br /> highly expressed <br />                                       ECO-KPN (log phase)                   23/415           0.045 0.037      -     1.352             0.896     **      0.042   0.044      - <br />                                   ECO-KPN (stationary phase)                26/313           0.064 0.047      -     1.459             1.142      *      0.047   0.045      - <br />                                     STM-KPN (log phase)                     18/287           0.06 0.037       -     1.384             0.895      *      0.046   0.046      - <br />                                   STM-KPN (stationary phase)                26/228           0.075 0.053      *     1.845             1.273     ***     0.042   0.044      - <br /> * : p &lt; 0.05: **: p &lt; 0.01: ***: p &lt; 0.001; -: insignificant <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                Page 5 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> approximately COGs SOGs                             location-independent effect dS.  (Table 1). Notably, previously reported                 increase dN COGs likely affected chro- <br /> mutation rate increase gene expression level                    mosomal locations genes.  [25-27].  investigate             COGs close Ter far outnumber increases evolutionary rate COGs asso-                      evidence needed clarify correla- <br /> ciated increased expression levels genes.                  tion dN COG/SOG gene type  results indicate case.                 context background mutation rate. <br /> Highly expressed COGs actually lower dN dS non-highly expressed COGs (Additional file 6).                      Confounding factor 4: replication&#226;&#8364;&#8220;transcription  increases evolutionary rates (particu-                 confrontation <br /> larly dS) COGs result increased expres-                   head collision directions tran- <br /> sion levels genes.                                              scription replication reported increase <br />                                                                          mutation rate affected genes [3,10,29]. Confounding factor 3: background mutation rate                           gene involved inversion event, transcrip- gene inversions usually occur vicinity                tional orientation likely change, causing replica- Ori Ter[1,2], actually                       tion-transcription confrontation. investigated <br /> COGs analyzed study located near Ter                        COGs replication-transcription confronta- <br /> (Figure 2). known genes located near Ter                  tion SOGs. Interestingly, increased mutation rate [28]. ask                        COGs maintain status terms replication-tran- uneven chromosomal distribution COGs                      scription confrontation. example, using KPN contributed COG-SOG differences evolu-                     outgroup, 157 162 COGs ECO- <br /> tionary rates.  performed partial correla-                  STM comparison maintain status head colli- <br /> tion analyses examine relationship                        sion orientation replication transcrip- <br /> evolutionary rates gene type (COG SOG).                       tion occurrences inversion events. <br /> results indicate dS significantly correlated                 ratio head collision orienta- COG/SOG gene type distance effect                      tion genes COGs SOGs 1.1 0.64, respec- controlled (Spearman&#226;&#8364;&#8482;s r &gt; 0.05 p &lt; 0.05) (Addi-                  tively ECO&#226;&#8364;&#8220;STM comparison.  COGs <br /> tional file 7). positive correlation indicates                  fact likely head collision genes <br /> CTOs location-independent effect                           SOGs. clarify difference increase dS.  distance-controlled cor-                  degree head collision major determinant <br /> relations statistically insignificant dN                 COG-SOG differences evolutionary rate, dN/dS) COG/SOG gene type.                          compared evolutionary rates COGs SOGs results indicate CTOs actually                         controlling replication-transcription confrontation <br />  <br />  <br />  Figure 2 Dot plot analyzed orthologous genes   ECO-KPN; (b) STM-KPN comparison. Y X axis represents, <br />  respectively, chromosomal positions KPN genes  base pair) orthologous genes ECO (left panel) STM (right <br />  panel). black red dots represent SOGs COGs, respectively. COGs located close Ter. positions Ter  1,892,000, 1,549,000 1,635,000 bp KPN, ECO STM, respectively. <br />  Lin et al. BMC Bioinformatics 2011, 12(Suppl 9):S19                                                                                        Page 6 8 <br /> http://www.biomedcentral.com/1471-2105/12/S9/S19 <br />  <br />  <br />  <br />  <br /> pattern. ECO-STM comparison, rates dN,                         Functional enrichment analysis COGs <br /> dS dN/dS head collision COGs differ                         ask rapidly evolving COGs significantly orientation COGs (Additional File                      enriched certain functional categories. Additional file <br /> 8). contrast, head collision SOGs signifi-                        10 indicates protein products COGs tend cantly higher evolutionary rates orientation                         located membranes cell walls, SOGs. results suggest higher evolutionary                     involved variety metabolic reactions. obser- <br /> rates COGs result higher proportion                      vation biologically sensible (1) proteins head collision genes.  CTO itself                        located cell surface tend evolve faster; major reason increased evolutionary rate.                        (2) different bacterial species different meta- <br />                                                                              bolic capacities.  relationship Confounding factor 5: decreased dS gene terminals                         CTOs functional preferences genes <br /> (codon usage bias)                                                           remains unknown. reported synonymous substitution <br /> rate decreases near 50-100 codons [12]                     Conclusions 20 codons gene [13], possibly codon                        knowledge, study demonstrate <br /> usage biased regions. ask                           changes transcriptional orientation increase SOGs tend lowered dS ends                         evolutionary rates (dN dS) prokaryotic genes. <br /> compared COGs, overall dS                                evolutionary effects CTOs inde- <br /> decreased. Accordingly, divided analyzed                      pendent gene essentiality, gene expression level, replica- <br /> genes terminal  50 20 codons)                        tion-transcriptional confrontation, decrease dS middle region  rest codons). ran-                         gene terminals.  increase dN par- <br /> domly sampled 70 codons middle region                           tially related gene locations. Furthermore, results <br /> comparison. differences evolutionary rates (&#206;&#8221;dS,                      suggest increases evolutionary rates COGs <br /> &#206;&#8221;dN, &#206;&#8221;dN/dS) terminal codons                             mainly mutation-driven, dN/dS ratios similar randomly sampled codons calculated com-                         COGs SOGs. real cause increases <br /> pared COGs SOGs. results                           evolutionary rate COGs (particularly dS) remains <br /> &#206;&#8221;dS, &#206;&#8221;dN, &#206;&#8221;dN/dS similar COGs                            unclear worth explorations. speculated <br /> SOGs (Figure 3 ECO-KPN comparison Addi-                              CTOs result impediments DNA <br /> tional file 9 STM-KPN comparison; pair-                       replication  altering, example, DNA protein binding <br /> wise differences statistically insignificant).                 sites secondary structure), turn lead difference codon usage bias evolutionary rate                      recruitment error-prone DNA repair polymerases <br /> does appear affect overall results.                               increase mutation rate [30,31]. <br />  <br />                  genes COGs      SOGs                         genes COGs        SOGs                              genes COGs    SOGs <br />  Figure 3 Difference evolutionary rates terminal middle region ECO-KPN orthologous genes. Y axis shows <br />  difference (terminal minus middle) dN, dS, dN/dS ratio, respectively, left, middle, right panel. differences dN, dS  dN/dS ratio statistically insignificant pair-wise gene group comparisons panel. <br />  <br />  <br /> Additional material                                                             Authors&#226;&#8364;&#8482; contributions <br />                                                                                 CHL conceived study carried data analyses. CHL, FCC                                                                                 CAH designed conceptual framework study, interpreted     Additional file 1: numbers detailed information              results, drafted manuscript. CYL pre-processed gene expression data <br />     orthologous gene pairs analyzed study.                              assisted statistical analyses. <br />     Additional file 2: numbers ECO-KPN, ECO-STM, STM-KPN <br />     orthologous gene pairs identified reciprocal BLASTP             <br />     OMA database                                                                <br />     Additional file 3: evolutionary rates COGs SOGs                                                                              <br />     ECO-KPN STM-KPN comparisons. genes included     analysis identified reciprocal BLASTP OMA <br />     database orthologous genes. *: p-value &lt; 0.05; **: p-value &lt;             <br />     0.01; ***: p-value &lt; 0.0001                                           <br />     Additional file 4: numbers COGs dS &gt; 3 ECO-KPN, <br />                                                                            <br />     ECO-STM, STM-KPN comparisons. <br />                                                                      <br />     Additional file 5: evolutionary rates   ECO-KPN comparison;         <br />     (b) STM-KPN comparison. compare evolutionary                    <br />     rates genes changed transcriptional orientation      <br />     ECO, STM, KPN, (&#226;&#8364;&#339;ESK&#226;&#8364;&#65533;), genes changed orientation               <br />     ECO-STM lineage KPN lineage (&#226;&#8364;&#339;ES/K&#226;&#8364;&#65533;),           <br />     genes changed orientation species (&#226;&#8364;&#339;ECO           <br />     &#226;&#8364;&#339;STM ).                                                             <br />     Additional file 6: Comparison evolutionary rates highly           <br />     non-highly expressed COGs.   ECO-KPN comparison         <br />     log phase; (b) ECO-KPN comparison stationary phase; (c)                 <br />     STM-KPN comparison log phase; (d) STM-KPN comparison            <br />     stationary phase. *: p-value &lt; 0.05; **: p-value &lt; 0.01             <br />     Additional file 7: Partial correlations evolutionary rates <br />                                                                                  <br />     COG/SOG gene type physical distance Ter                                                                             <br />     controlled. <br />                                                                                     <br />     Additional file 8: evolutionary rates COGs SOGs          <br />     subject head collision orientation DNA       <br />     replication transcription ECO-STM comparison. *                   <br />     represents p value &lt; 0.05.                                                <br />     Additional file 9: Comparison &#206;&#8221;dN, &#206;&#8221;dS &#206;&#8221;dN/dS                <br />     COGs SOGs STM-KPN comparison. &#206;&#8221;dN, &#206;&#8221;dS &#206;&#8221;dN/dS                    <br />     calculated subtracting evolutionary rates middle        <br />     region terminal regions gene. Note          <br />     pair-wise comparisons panels              <br />     statistically significant.                                                 <br />     Additional file 10: Functional enrichment analyses COGs       ECO-KPN (b) ECO-STM comparisons. Note                                                                                    <br />     functional assignments based gene annotations                                                                                     <br />     ECO. <br /> </body></html>