<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Dev Biol. 2012 March 1; 363(1): 128&#226;&#8364;&#8220;137. doi:10.1016/j.ydbio.2011.12.025. <br />  <br />  <br />  <br />                            Mutation zebrafish Snapc4 associated loss                            intrahepatic biliary network <br />                            Madeline Schaub1, Justin Nussbaum1, Heather Verkade2,3, Elke  Ober2,4, Didier Y.R. <br />                            Stainier2,5, Takuya F. Sakaguchi1,2,5 <br />                            1Department Stem Cell Biology Regenerative Medicine, Cleveland Clinic, Cleveland, Ohio <br />  <br />                            44195, USA <br />                            2Department Biochemistry Biophysics, Programs Developmental Stem Cell Biology, <br />                            Genetics, Human Genetics, Liver Center, University California, San Francisco, San <br />                            Francisco, CA 94158 <br />  <br />                            Abstract <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  Biliary epithelial cells line intrahepatic biliary network, complex dimensional network <br />                                  conduits. loss differentiated biliary epithelial cells primary cause                                  congenital liver diseases. identified zebrafish snapc4 (small nuclear RNA-activating <br />                                  complex polypeptide 4) mutant biliary epithelial cells initially differentiate                                  subsequently disappear. snapc4 mutant larvae, biliary epithelial cells undergo <br />                                  apoptosis, leading degeneration intrahepatic biliary network. Consequently, snapc4 <br />                                  mutant larvae, biliary transport ingested fluorescent lipids gallbladder blocked. Snapc4 <br />                                  largest subunit protein complex regulates small nuclear RNA (snRNA) <br />                                  transcription. snapc4s445 mutation causes truncation C-terminus deleting                                  domain responsible specific interaction Snapc2, vertebrate specific subunit                                  SNAP complex. mutation leads hypomorphic phenotype, subset snRNA <br />                                  transcripts quantitatively altered snapc4s445 mutant larvae. snapc2 knockdown disrupts <br />                                  intrahepatic biliary network similar fashion snapc4s445 mutant larvae. data <br />                                  indicate physical interaction Snapc2 Snapc4 important expression <br />                                  subset snRNAs biliary epithelial cell survival zebrafish. <br />  <br />  <br />                            Keywords <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  Liver; snRNA; SNAP190; SNAPC2; biliary epithelial cells; vanishing bile duct <br />  <br />  <br />                            Introduction <br />                                              liver major digestive organ responsible important physiological functions, <br />                                              including bile synthesis secretion. Bile synthesized hepatocytes transported  <br />  <br />  <br />                            &#194;&#169; 2012 Elsevier  rights reserved. <br />                            5 <br />                             Authors correspondence (didier.stainier@ucsf.edu; sakagut2@ccf.org). <br />                            3Current Address: School Biological Sciences, Monash University, Clayton, VIC 3800, Australia <br />                            4Current Address: National Institute Medical Research, Division Developmental Biology, Ridgeway, Hill, London <br />                            NW7 1AA, UK <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Schaub et al.                                                                                             Page 2 <br />  <br />  <br />                                            intestine biliary network. biliary network complex  <br />                                            dimensional conduits lined specialized cells called biliary epithelial cells.                                            congenital liver diseases associated biliary network malformation (Alvaro, 1999; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Lazaridis et al., 2004; Lemaigre Zaret, 2004; Strazzabosco et al., 2005; Raynaud et al., <br />                                            2011). Despite etiological heterogeneity difference degree inflammatory <br />                                            response, end result biliary disorders loss biliary cells (Strazzabosco et <br />                                            al., 2000; Lazaridis et al., 2004).  understanding molecular mechanisms underlying <br />                                            loss biliary epithelial cells lead novel insights pathogenesis                                            various biliary disorders, provide basis novel therapeutic strategies. <br />  <br />                                            small nuclear RNA (snRNA)-activating protein complex (SNAPc) transcription <br />                                            factor essential transcription snRNA genes, including U1, U2, U4, U5, U6 <br />                                            spliceosomal snRNAs (Jawdekar Henry 2008, Ma Hernandez, 2001; Yoon et al., <br />                                            1995). SNAPc composed proteins; Snapc1  know SNAP43                                            PTF&#206;&#179;) (Yoon et al., 1996), Snapc2 (SNAP45 PTF&#206;&#180;) (Sadowski et al., 1996), Snapc3 <br />                                            (SNAP50 PTF&#206;&#178;) (Henry et al., 1996), Snapc4 (SNAP190 PTF&#206;&#177;) (Wong et al., 1998), <br />                                            Snapc5 (SNAP19) (Henry et al., 1998). Snapc4 largest subunit, serves                                            backbone assemble SNAPc. subunits Snapc2 assemble N-terminus <br />                                            Snapc4. Snapc2 instead binds domain called Snapc2 binding domain C- <br />                                            terminus Snapc4 (Ma Hernandez, 2001). Interestingly, partial SNAPc containing <br />                                            Snapc1, Snapc3 N-terminal Snapc4 fully sufficient target DNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            sequence binding snRNA gene transcription vitro (Mittal et al., 1999).                                             C-terminal thirds Snapc4 Snapc2 required vitro basal snRNA <br />                                            transcription. subunits required partial SNAPc widely conserved <br />                                            Drosophila, trypanosomes plants (Li et al., 2004; Das Bellofatto, 2003; <br />                                            Schimanski et al., 2005). contrast, Snapc2 Snapc5 subunits conserved                                            invertebrates plants, subunits  vitro, dispensable vertebrate <br />                                            snRNA transcription (Mittal et al., 1999). Snapc2 binding changes SNAPc target <br />                                            recognition, vivo function vertebrate specific subunit known. <br />  <br />                                            Zebrafish express genes encoding 5 subunits SNAPc contain vertebrate specific <br />                                            organs liver, making suitable genetic model study functions                                            vertebrate specific subunits SNAPc. zebrafish, formation function biliary <br />                                            network characterized (Lorent et al., 2004; Matthews et al., 2004; Sakaguchi et al., <br />                                            2008). small size larval liver enables visualization dimensional <br />                                            branching structure intrahepatic biliary network (Sakaguchi et al., 2008). Imaging <br />                                            studies intrahepatic biliary network facilitated recent development                                            notch activity reporter transgenic line GFP specifically expressed biliary <br />                                            network (Parsons et al., 2009; Lorent et al., 2010). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            study, identified mutation zebrafish snapc4 gene truncates C- <br />                                            terminal protein deletes Snapc2 binding domain. mutant form                                            Snapc4 similar invertebrate Snapc4 (Hernandez Jr et al., 2006; Kim et al., 2010) <br />                                            mini-Snapc4 used vitro studies (Ma Hernandez, 2001). expected                                            vitro studies, truncated mutant form Snapc4 regulate expression <br />                                            target snRNAs, considered hypomorphic allele.                                            snapc4 hypomorph unexpectedly causes loss differentiated <br />                                            biliary epithelial cells zebrafish. <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                                Page 3 <br />  <br />  <br />                               Results <br />                               Zebrafish s445 mutants exhibit intrahepatic biliary network defects <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            identified recessive lethal mutation, s445, forward genetic screen genes                                            regulate liver organogenesis zebrafish, Danio rerio (Ober et al., 2006). 120 hours post- <br />                                            fertilization (hpf), homozygous s445 mutant larvae morphologically similar wild-type <br />                                            siblings (Fig. 1A B). liver, visualized Tg(gutGFP)s854 <br />                                            expression (Field et al., 2003), noticeably smaller (Fig. 1B; Supplementary fig. 1).                                            compared size regions marked hepatocyte specific Tg(fabp1a:DsRed)gz15 <br />                                            expression, clearly smaller s445 mutant larvae compared wild-type 120 <br />                                            hpf (Fig. 1C D), supporting observation liver smaller s445 mutant <br />                                            larvae. <br />  <br />                                            use PED6 fluorescence assay test vivo function                                            digestive organs s445 mutants. assay, larvae fed quenched fluorescent <br />                                            <span id='am-6' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-7' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span>phospholipid</span> PED6, cleaved intestine, activating fluorescent signal <br />                                            (Farber et. al., 2001). fluorescent metabolites transported biliary <br />                                            gallbladder (Fig. 1E). PED6 fluorescence observed intestines                                            s445 mutant larvae (Fig. 1F, asterisk).  s445 mutant larvae, PED6 signal                                            observed gallbladder, despite presence morphologically identifiable <br />                                            gallbladder extrahepatic bile duct (Fig. 1H). Based data, hypothesized NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            defect PED6 metabolite transportation impairment intrahepatic <br />                                            biliary network. <br />  <br />                                            test hypothesis, undertook time course observations intrahepatic biliary <br />                                            network formation s445 mutants. Tg(Tp1-MmHbb:EGFP)um14 expressed                                            intrahepatic biliary epithelial cells zebrafish liver (Parsons et al., 2009, Lorent et al., <br />                                            2010). 80 hpf, did observe difference Tg(Tp1-MmHbb:EGFP)um14 <br />                                            expression s445 mutant liver (Fig. 2A D), suggesting intrahepatic biliary <br />                                            network initially formed. 100 hpf, Tg(Tp1-MmHbb:EGFP)um14 positive intrahepatic <br />                                            biliary network highly branched wild-type larvae. contrast, 100 hpf                                            s445 mutant larvae, Tg(Tp1-MmHbb:EGFP)um14 expression largely disappears                                            liver, weak fluorescence detected central liver (Fig. 2E). 120 <br />                                            hpf, Tg(Tp1-MmHbb:EGFP)um14 fluorescence detected s445 mutant liver <br />                                            (Fig. 2F). data suggest intrahepatic biliary network forms initially                                            disappears 120 hpf s445 mutant larvae. <br />  <br />                                            better understand defects caused s445 mutation, visualized  <br />                                            dimensional structure intrahepatic biliary network. Immunofluorescence staining NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Alcam, intrahepatic biliary network marker (Sakaguchi et al., 2008), used                                            visualize intrahepatic biliary network (Fig. 3A). Consistent time course <br />                                            observations, did observe difference Alcam Tg(Tp1-MmHbb:EGFP)um14 <br />                                            expression s445 mutant larvae 80 hpf (Fig. 3A B; Supplementary Fig. 2), <br />                                            supporting idea biliary epithelial cells initially differentiate s445 mutant larvae. <br />                                             s445 mutant larvae 100 hpf, Tg(Tp1-MmHbb:EGFP)um14 expression                                            liver significantly different wild-type siblings (Fig. 3D). wild-type livers, <br />                                            Tg(Tp1-MmHbb:EGFP)um14 expressing intrahepatic biliary network added new <br />                                            branches (Fig. 3C). contrast, intrahepatic biliary network appears degenerating <br />                                            s445 mutant livers (Fig. 3D). <br />  <br />                                            s445 mutant larvae, detected fluorescent TUNEL-positive/Tg(Tp1- <br />                                            MmHbb:EGFP)um14-positive cells liver (Average 6.6 cells liver, SD= 4.9, n=5, p <br />                                            &lt; 0.05) 100 hpf (Fig. 3F G), Tg(Tp1-MmHbb:EGFP)um14 expressing cells  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                            Page 4 <br />  <br />  <br />                                            wild-type liver (n=5) showed TUNEL signal stage (Fig. 3E G).                                            data suggest biliary epithelial cells initially present undergo cell death s445 <br />                                            mutant larvae. contrast, significance difference number TUNEL- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            positive/Tg(Tp1-MmHbb:EGFP)um14-negative cells liver wild-type (average <br />                                            3.8 cells liver; SD = 2.16; n = 5) s445 mutant (average 2.6 cells liver; SD = 1.81; <br />                                            n = 5; p &gt; 0.05) larvae, suggesting cell death significantly increased biliary <br />                                            epithelial cells s445 mutant liver. Consistent observation, 120 hpf,                                            Tg(Tp1-MmHbb:EGFP)um14 expressing biliary epithelial cells s445 <br />                                            mutant livers (Fig. 3I), Tg(fabp1a:DsRed)gz15 expressing hepatocytes                                            sustained (Fig. 3H . Tg(kdrl:EGFP)s843 (Jin et al., 2005) expressing intrahepatic <br />                                            vascular network highly branched conduit (Fig. 3J), intertwined                                            intrahepatic biliary network zebrafish (Fig. 3J; Sakaguchi et al., 2008). s445 mutant <br />                                            larvae, intrahepatic vascular network affected, maintaining branched <br />                                            structures (Fig. 3K&#226;&#8364;?), Alcam-expressing intrahepatic biliary network largely <br />                                            degenerated (Fig. 3K&#226;&#8364;&#8482;). Differentiated biliary epithelial cells lumens evident                                            electron micrographs wild-type liver 120 hpf (Fig. 3L). stage, <br />                                            biliary epithelial cells morphologically confirmed electron micrographs                                            s445 mutant (Fig. 3M). Overall, data suggest s445 mutation preferentially <br />                                            affects intrahepatic biliary network. <br />  <br />                               s445 mutation truncates C-terminal zebrafish Snapc4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            identify s445 gene, employed positional cloning strategy. Bulk segregant <br />                                            analyses using markers zebrafish genetic map placed s445 locus zebrafish <br />                                            chromosome 5 (Fig. 4A). subsequent steps, defined critical region surrounding                                            s445 locus, flanked markers 0.2 cM (z26603) 0.07 cM (CA-YE2)                                            s445 locus (Fig. 4A). partially overlapping BACs, DKEY-20G15 DKEY-1O2, <br />                                            spanned critical region (Fig. 4A). Injecting DKEY-102, DKEY-20G15 BAC <br />                                            s445 homozygous mutants partially rescued phenotype (data shown), indicating <br />                                            DKEY-102 BAC contains s445 locus. <br />  <br />                                            predicted genes CA-YE2 end DKEY-102. Sequence <br />                                            analysis candidate cDNAs s445 mutant wild-type larvae identified                                            single point mutation zebrafish snapc4 (snRNA-activating protein complex <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="peptide" datatype="xsd:string"></span>peptide</span> <br />                                            4) gene (Fig. 4B). C T transition generated premature stop codon zebrafish <br />                                            snapc4 cDNA (Fig. 4C), indicating lesion responsible s445 phenotype. <br />  <br />                                            Snapc4 largest subunit transcription factor SNAPc (snRNA activating protein <br />                                            complex; Wong et al., 1998), responsible snRNA gene transcription. Zebrafish <br />                                            Snapc4 contains conserved Myb DNA binding Snapc2 binding domains (Fig. 4C). NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            s445 mutation deletes Snapc2 binding domain Myb DNA binding domain <br />                                            remains intact (Fig. 4C). confirm snapc4 gene responsible                                            s445 mutant phenotype, injected antisense morpholino oligonucleotides (MOs) knock <br />                                            snapc4 gene function. splice blocking MO  designed truncate C-terminal <br />                                            Snapc4 protein phenocopied s445 mutant phenotype (n=35, Supplementary <br />                                            fig. 3C D). data support hypothesis truncation C-terminal <br />                                            Snapc4 protein cause s445 mutant phenotype. Injecting                                            splice blocking MO(B), designed block splicing Myb DNA <br />                                            binding domain, generated severe morphological phenotype s445 mutants (n <br />                                            &gt; 50, Supplementary fig. 3E F), suggesting s445 hypomorphic mutation                                            s445 mutant phenotype milder complete loss function phenotype. <br />                                            Injecting snapc4 MO(C) targets translation initiation codon induced <br />                                            developmental arrest 18 24 hpf (n=38, data shown). splice blocking MOs <br />                                            target zygotic snpac4 mRNA translation initiation blocking MO targets  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                          Page 5 <br />  <br />  <br />                                            maternal zygotic snapc4 mRNA, complete loss snapc4 function severely affects <br />                                            embryonic development. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            region surrounding zebrafish snapc4 syntenic human chromosome 9q34.3 <br />                                            human SNAPC4 located (Fig. 4D), zebrafish snapc4 likely orthologue                                            human SNAPC4. <br />  <br />                                            Consistent s445 mutant phenotypes, snapc4 gene expressed liver 80 <br />                                            hpf (Fig. 4E). Strong snapc4 expression detected brain, eyes branchial <br />                                            arches 80 hpf (Fig. 4E). <br />  <br />                               snRNA expression s445 mutant larvae <br />                                            SNAPc required transcription snRNA genes vitro (Jawdekar Henry, <br />                                            2008). examined snRNA gene expression s445 mutant larvae real-time <br />                                            quantitative PCR Northern blotting. Unexpectedly, significant differences <br />                                            U1, U2 U6 snRNA expression levels s445 mutant larvae 100 hpf (Fig. 5                                            B).  s445 mutant larvae, U4 U5 snRNA expression levels                                            downregulated approximately 50 75 percent, respectively (Fig. 5A B). data <br />                                            support hypothesis s445 mutation hypomorphic, affecting subset                                            downstream target snRNA genes. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Snapc2 required intrahepatic biliary network formation <br />                                            Snapc2 binding domain deleted Snapc4 s445 mutants, hypothesized <br />                                            absence Snapc2/Snapc4 interaction responsible downregulation U4 <br />                                            U5 snRNA transcription degenerate intrahepatic biliary network <br />                                            phenotype s445 mutant larvae. test hypothesis, knocked zebrafish <br />                                            snapc2 gene. designed non-overlapping splice blocking snapc2 MOs (MO                                             MO(B)). Injecting snapc2 MOs generated phenotype (Fig. 6B), suggesting <br />                                            MOs working target-specific manner. snapc2 MOs-injected larvae <br />                                            smaller livers slight pericardial edema 100 hpf (Fig. 6B), body length <br />                                            normal. snapc2 gene expressed liver 80 hpf (Fig. 6C), expression <br />                                            pattern similar snapc4 gene (Compare Fig, 6C Fig. 4E). Similar s445 <br />                                            mutant larvae, snapc2 MO -injected larvae 100 hpf, Tg(Tp1-MmHbb:EGFP)um14 <br />                                            expression liver missing (Fig. 6E), hepatocyte marker <br />                                            Tg(fabp1a:DsRed)gz15 expression sustained (Fig. 6G). data suggest  similar <br />                                            s445 mutant larvae, intrahepatic biliary network preferentially affected snapc2 <br />                                            MO  injected larvae. snapc2 MO  injected larvae, 52 bp deletion snacp2 <br />                                            cDNA detected (Fig. 6H), confirming snapc2 MO  works target <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            specific manner. snapc2 MO injected larvae, U4 U5 snRNA <br />                                            expression levels downregulated (Fig. 6I) similar manner s445 mutant larvae.                                            snapc2 MO injected larvae, U2 snRNA expression slightly regulated (Fig. <br />                                            6I). data suggest Snapc2 required intrahepatic biliary network maintenance, <br />                                            snRNA transcription. <br />  <br />                                            snapc2 binding domain Snapc4 known interact physically Snapc2,                                            hypothesized expressing Snapc2 binding domain inhibit Snapc2 <br />                                            binding endogenous Snapc4. test hypothesis, cloned Snapc2 binding <br />                                            domain Snapc4, attached translation initiation stop codons beginning                                            end  respectively (Figure 6J&#226;&#8364;&#8482; materials methods). injected <br />                                            synthesized mRNA encoding Snapc2 binding domain, injected larvae showed similar <br />                                            phenotype (26/33) larvae injected Snapc2 MOs 100 hpf (compare Figure <br />                                            6B 6J). Tg(Tp1bglob:EGFP)um14 expression liver missing Snapc2 <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                                Page 6 <br />  <br />  <br />                                            binding domain mRNA injected larvae 100 hp (compare Figure 6D 6K). data <br />                                            support hypothesis physical interaction Snapc4 Snapc2                                            important maintenance intrahepatic biliary network. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Discussion <br />                                            study, identified zebrafish snapc4 mutant differentiated biliary <br />                                            epithelial cells initially develop later degenerate. knowledge,                                            vivo evidence showing Snapc4 required specific biological process,                                            maintenance intrahepatic biliary network. <br />  <br />                               C-terminal region Snapc4 inhibitory function <br />                                            snRNA-activating protein complex, SNAPc, multi-subunit transcription factor                                            required transcription snRNA genes (Jawdekar Henry, 2008). Snapc4,                                            largest subunit SNAPc, forms backbone complex (Wong et al., 1998).                                            vitro studies demonstrated C-terminal region Snapc4 possesses inhibitory <br />                                            effect target DNA binding transcriptional abilities (Mittal et al., 1999). Binding                                            Snapc2 C-terminal portion Snapc4 masks inhibitory effect, enhances                                            DNA binding Snapc4 target DNA vitro (Mittal et al., 1999). fact, removing <br />                                            C-terminal thirds Snapc4 improves DNA binding transcriptional abilities. <br />                                             vitro, omission Snapc2, releases inhibitory effect C-terminal <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            portion Snapc4, impairs Snapc4 function greater degree deletion C- <br />                                            terminal thirds Snapc4 (Mittal et al., 1999; Ma Hernandez, 2001). vitro <br />                                            data fully consistent zebrafish vivo data. depletion Snapc2 larvae <br />                                            injected snapc2 MO results severe morphological phenotype                                            s445 mutant larvae, C-terminal Snapc4 deleted (Compare Fig. 1B <br />                                            Fig. 6B). <br />  <br />                                            vitro, N-terminal Snapc4 sufficient regulate basal levels U1 U6 <br />                                            snRNA transcription (Mittal et al., 1999). Consistent data, s445 mutant larvae, <br />                                            detected significant change U1, U2 U6 snRNA expression levels (Fig. 5). <br />                                            Blockage U2 snRNA function known induce early developmental arrest zebrafish <br />                                            (K&#195;&#182;nig et al., 2007), suggesting snRNA genes indispensable zebrafish <br />                                            development. Consistently, detected developmental arrest translational blockage                                            snapc4 MO(C) injected larvae, Snapc4 protein depleted.                                            result suggests length Snapc4 required early developmental progress.  <br />                                            conclude s445 mutation hypomorphic, affecting subset target snRNA <br />                                            transcription. knowledge, vivo genetic evidence  consistent <br />                                            vitro data, C-terminal Snapc4 required U1and U6 snRNA <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            transcription. <br />  <br />                               Zebrafish human Snapc4 <br />                                            synteny conserved (Fig. 4D), zebrafish snapc4, affected s445 <br />                                            mutation, likely orthologue human SNAPC4. Zebrafish Snapc4 contains <br />                                            conserved Myb DNA binding Snapc2 binding domains.  Oct-1 binding <br />                                            domain, conserved mammalian Snapc4 genes, zebrafish <br />                                            Snapc4. lack recognizable Oct-1 binding domain does exclude possibility <br />                                            different sequence zebrafish Snapc4 responsible Oct-1 binding.                                            surprise, zebrafish snapc4 shows tissue specific expression 80 hpf (Fig. 4E), time <br />                                            U1 snRNA expressed ubiquitously (data shown). assume highly <br />                                            proliferating cells require additional snRNA transcriptional regulators maintain constant <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                           Page 7 <br />  <br />  <br />                                            snRNA levels.  exclude possibility SNAPc possesses <br />                                            additional roles organogenesis snRNA transcriptional regulation. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Human SNAPC4 located chromosome 9q34.3. Interestingly, deletion 9q34.3 <br />                                            results complicated clinical syndromes (Iwakoshi et al., 2004; Harada et al., 2006; Neas et <br />                                            al., 2005; Pointon et al., 2010). fact hypomorphic snapc4 mutation cause <br />                                            tissue cell type specific phenotypes zebrafish help identify responsible genes <br />                                            human diseases. <br />  <br />                                            Vanishing bile duct syndrome irreversible human liver disorder caused biliary duct <br />                                            destruction (Pass et al., 2008). vanishing bile duct syndrome, differentiated biliary <br />                                            epithelial cells undergo apoptotic cell death disappear (Strazzabosco et al., 2000).                                            noteworthy phenotype observe s445 mutant larvae - biliary epithelial cells <br />                                            initially differentiate subsequently disappear apoptosis - similar                                            observed vanishing bile duct syndrome. <br />  <br />                               Degenerating biliary epithelial cells s445 mutant larvae <br />                                            livers s445 mutant larvae indistinguishable wild-type siblings 80 <br />                                            hpf (Fig. 2A D; Fig. 3A B).  100 hpf, intrahepatic biliary network <br />                                            shows drastic differences s445 mutant larvae (Fig. 3C D). s445 mutant larvae, <br />                                            biliary branches contracted, displaying prominent cell death Tg(Tp1- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            MmHbb:EGFP)um14 positive biliary epithelial cells (Fig. 3F). 100 hpf, biliary <br />                                            epithelial cells vulnerable s445 mutation cell types liver <br />                                            (Fig. 3I K).  noted  120 hpf, hepatocytes decrease                                            number (Fig. 1D) appear morphological differences (Fig. 3M) s445 mutant <br />                                            larvae. s445 mutant larvae generally die 8 dpf. <br />  <br />                                            snapc2 knock leads similar biliary phenotype, posit absence                                            Snapc2-Snapc4 interaction responsible loss biliary epithelial cells s445 <br />                                            mutant larvae.  blocking Snapc2-Snapc4 interaction overexpression                                            Snapc2 binding domain impairs intrahepatic biliary network formation (Fig. 6J K). <br />                                            demonstrated novel functions Snapc2-Snapc4 <br />                                            interaction: proper transcription U4 U5 snRNA genes survival biliary <br />                                            epithelial cells. predict downregulation snRNA genes affects biliary epithelial <br />                                            cell survival.  Snapc2 plays roles cell, subunit                                            SNAPc factor mitotic regulation (Shanmugam Hernandez, 2008), <br />                                            exclude possibility Snapc2 regulates biliary epithelial cell survival <br />                                            independent SNAPc snRNA gene transcription. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Materials Methods <br />                               Zebrafish husbandry <br />                                            Zebrafish (Danio rerio) larvae obtained natural crosses wild-type                                            heterozygous mutant fish, raised 28.5&#194;&#176;C staged according age (hours post- <br />                                            fertilization (hpf) 28.5 &#194;&#176;C) previously described. following transgenic mutant <br />                                            lines used study: Tg(fabp1a:DsRed; ela3l:EGFP)gz15 (Korzh et al., 2008); <br />                                            Tg(Tp1-MmHbb:EGFP)um14 (Parsons et al., 2009); Tg(kdrl:EGFP)s843 (Jin et al., 2005); <br />                                            Tg(gutGFP)s854  known Tg(XlEef1a1:GFP)s854 ) (Field et al., 2003). <br />  <br />                               Immunohistochemistry, situ cell detection situ hybridization <br />                                            Immunohistochemistry performed previously described (Sakaguchi et al., 2008). <br />                                            Confocal images obtained series TCS SP5 confocal microscope (Leica <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                          Page 8 <br />  <br />  <br />                                            Microsystems). confocal images processed using Volocity software (PerkinElmer). <br />                                            situ hybridization performed previously described (Sakaguchi et al., 2008). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Bulk segregant linkage analysis <br />                                            heterozygous s445 fish carrying Tg(gutGFP)s854 transgene crossed SJD <br />                                            strain create map cross. s445 mutant larvae collected offspring map <br />                                            cross fish based Tg(gutGFP)s854 expression. Genomic DNA isolated                                            individual mutant larvae Proteinase K digestion. Using bulk segregant analysis,                                            initially mapped s445 locus linkage group 5  z21461 z4299).                                            collected 2000 mutants performed linkage recombinant analysis.                                            tested specific potential polymorphic markers linkage group 5 PCR using <br />                                            genomic DNA individual mutant larvae. identified polymorphic marker, CA-YE2 <br />                                            (5&#226;&#8364;&#8482;- tcaaaggggagctaattattctgac-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;-ggtgcactctttgcaaatcata-3&#226;&#8364;&#8482;), tightly <br />                                            linked marker s445 locus. zebrafish snapc4 cDNA amplified PCR                                            cloned using PCRIITOPO (Invitrogen). <br />  <br />                               Genotyping s445 mutants <br />                                            order genotype s445 mutants, used dCAPS primers (Neff et al., 1998) introduce <br />                                            s445 mutant allele specific restriction site PCR products. PCR performed                                            s445 dCAPS primers (5&#226;&#8364;&#8482;-tcaacaacaatctctaccaccttctcat-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;-cactaaatgttgcacttccc-3&#226;&#8364;&#8482;) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            PCR product digested MseI. <br />  <br />                               Transmission electron micrograph (TEM) <br />                                            Wild-type s445 mutant larvae 120 hpf fixed 4% glutaraldehyde 0.2M <br />                                            sodium cacodylate (pH7.4) overnight 4 &#194;&#176;C. Samples post-fixed 1% osmium <br />                                            tetroxide 1 h incubated 1% uranyl acetate Maleate buffer 1 h. Samples                                            dehydrated ethanol series, embedded. Ultrathin sections 85 nm cut <br />                                            diamond knife, stained uranyl acetate lead citrate. Images obtained <br />                                            Philips CM12 electron microscope operated 60 kV. <br />  <br />                               Real-time quantitative reverse transcriptase-PCR <br />                                            Total RNA isolated RNeasy kit (Qiagen) cDNA synthesized miScript <br />                                            (Qiagen) according manufacturer&#226;&#8364;&#8482;s instructions. SYBR green qPCR mix (SA <br />                                            Bioscience) used monitor real-time PCR 7500 real-time PCR (Applied <br />                                            Biosystems). used following primers: U1 (5&#226;&#8364;&#8482;-TTACCTGGCAGGGGAGACACCA-3&#226;&#8364;&#8482; <br />                                            5&#226;&#8364;&#8482;-CAGTCGAGATTCCCACATTTGG-3&#226;&#8364;&#8482;); U2 (5&#226;&#8364;&#8482;- <br />                                            TATCTGATACGTGCCCTACCCG-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;-GGGTGCACCGTTCCCGGAAGTA-3&#226;&#8364;&#8482;); <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            U4 (5&#226;&#8364;&#8482;-AGCTTTGCGCAGTGGCAGTATCG-3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;- <br />                                            TATCAGTCTCTGTCGAGACCGTC-3&#226;&#8364;&#8482;); U6 (5&#226;&#8364;&#8482;-CGCTACGGTGGCACATATAC-3&#226;&#8364;&#8482;                                            5&#226;&#8364;&#8482;- GAGCGCTTCACGGATTTG-3&#226;&#8364;&#8482;); Large subunit ribosomal RNA (EF417169: 5&#226;&#8364;&#8482;- <br />                                            TTGGGAATGCAGCCCAAAGCGG &#226;&#710;&#8217;3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;- ACCTCTCAACGGTTTCACGCCC <br />                                            &#226;&#710;&#8217;3&#226;&#8364;&#8482;). &#206;&#8221;&#206;&#8221;Ct method used relative quantification. <br />  <br />                               Northern blotting <br />                                            Northern blotting performed using NorthernMax kit (Applied Biosystems) according <br />                                            manufacturer&#226;&#8364;&#8482;s instruction. <br />  <br />                               Morpholino injection <br />                                            Morpholinos prepared injected previously described (Sakaguchi et al., 2006). <br />                                            used following morpholinos study: 2 ng/embryo snapc4 MO  5&#226;&#8364;&#8482;- <br />                                            AATCAACGCTATCAGGCTTACTTTG-3&#226;&#8364;&#8482;; 2 ng/embryo snapc4 MO(B) 5&#226;&#8364;&#8482;- <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                                              Page 9 <br />  <br />  <br />                                            GCATGCTGTCTTAATACGTACATCT-3&#226;&#8364;&#8482;; 8 ng/embryo snapc4 MO(C) 5&#226;&#8364;&#8482;- <br />                                            CGCTTGTAAGTCGTCAGATGCCATT- 3&#226;&#8364;&#8482;; 1 ng/embryo snapc2 MO  5&#226;&#8364;&#8482;- <br />                                            TTCTCATACACACCTGGGAAAAGGC-3&#226;&#8364;&#8482;; 1ng/embryo snapc2 MO(B) 5&#226;&#8364;&#8482;- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            TGTGAGCTTACCTGTTTGAGAGAAC-3&#226;&#8364;&#8482;. <br />  <br />                               Morpholino specificity test <br />                                            RT-PCR used test snapc2 MO  snapc4 MO(B) specificity. Wild-type, snapc2 <br />                                            MO  snapc4 MO(B) injected larvae collected 100 hpf total RNA                                            isolated using RNeasy kit (Qiagen). RT reactions performed using SuperScriptII <br />                                            RT-PCR kit (Gibco BRL) according manufacturer&#226;&#8364;&#8482;s recommendations. PCR                                            performed using following primer pairs: snapc2 (5&#226;&#8364;&#8482;-AAGTTCCCATTGAAACCTGGG-3&#226;&#8364;&#8482; <br />                                            5&#226;&#8364;&#8482;-TTAGGGGTGTGAGCAGGGTAAAGA-3&#226;&#8364;&#8482;) snapc4 (5&#226;&#8364;&#8482;- <br />                                            CAGGCAACTCTATCTTCTCA &#226;&#710;&#8217;3&#226;&#8364;&#8482; 5&#226;&#8364;&#8482;-TTGATCGTCTTTCAGGGTTC-3&#226;&#8364;&#8482;)                                            amplify snapc2 snpac4 cDNA fragments, respectively. <br />  <br />                               Cloning overexpression Snapc2 binding domain Snapc4 <br />                                            cDNA corresponding snapc2 binding domain snapc4 amplified using <br />                                            following primers: snapc2BDF, GAATTCACCATGGAGTGGATGGATGGGAAAGGTG; <br />                                            snapc2BDR, CTCTCGAGCTAACTCAATAACTGACGTTCCT. amplified PCR <br />                                            product subcloned EcoRI-XhoI sites pCS2+. Corresponding mRNA NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            synthesized previously described (Sakaguchi et al., 2006). 100 pg embryo snapc2 <br />                                            binding domain mRNA injected 1-cell stage. <br />  <br />                               Supplementary Material <br />                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                            thank Laura Nagy, Michele Pritchard, Allyson McCledon Arthur McCullough critical reading                                            manuscript. thank Mei Yin Cleveland Clinic Imaging Core TEM. work supported                                            APS postdoctoral fellowship, NIH K99/R00 award (DK078138), startup package                                            Cleveland Clinic Foundation T.F.S. work supported grants NIH (NIDDK) Packard <br />                                            foundation D.Y.R.S. snapc4s445 mutation identified positionally cloned UCSF T.F.S.                                            postdoctoral fellow D.Y.R.S.&#226;&#8364;&#8482;s lab; phenotypic analysis characterization snapc4s445 <br />                                            mutant carried UCSF Cleveland Clinic T.F.S. independent investigator. <br />  <br />  <br />                               References <br />                                            Alvaro D. Biliary epithelium: new chapter cell biology. Ital J Gastroenterol Hepatol. 1999; 31:78&#226;&#8364;&#8220; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               83. [PubMed: 10091109] <br />                                            Das  Bellofatto V. RNA polymerase II-dependent transcription trypanosomes associated                                               SNAP complex-like transcription factor. Proc Natl Acad Sci U S  2003; 100:80&#226;&#8364;&#8220;85. [PubMed: <br />                                               12486231] <br />                                            Farber SA, Pack M, Ho SY, Johnson <span id='am-19' about='oboInOwl:id' typeof='owl:Thing'>ID</span>, Wagner DS, Dosch R, Mullins MC, Hendrickson HS, <br />                                               Hendrickson EK, Halpern  Genetic analysis digestive physiology using fluorescent <br />                                               phospholipid reporters. Science. 2001; 292:1385&#226;&#8364;&#8220;1388. [PubMed: 11359013] <br />                                            Field HA, Ober EA, Roeser T, Stainier DYR. Formation digestive zebrafish.  Liver <br />                                               morphogenesis. Developmental Biology. 2003; 253:279&#226;&#8364;&#8220;290. [PubMed: 12645931] <br />                                            Harada N, Visser R, Dawson  Fukamachi M, Iwakoshi M, Okamoto N, Kishino T, Niikawa N, <br />                                               Matsumoto N. 1-Mb critical region patients 9q34.3 terminal deletion syndrome. J <br />                                               Hum Genet. 2004; 49:440&#226;&#8364;&#8220;444. [PubMed: 15258833] <br />  <br />  <br />  <br />  <br />                                                     Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                                  Page 10 <br />  <br />  <br />                                            Henry RW, Ma B, Sadowski CL, Kobayashi R, Hernandez N. Cloning characterization                                               SNAP50, subunit snRNA-activating protein complex SNAPc. EMBO J. 1996; 15:7129&#226;&#8364;&#8220; <br />                                               7136. [PubMed: 9003788] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Henry RW, Mittal V, Ma B, Kobayashi R, Hernandez N. SNAP19 mediates assembly                                               functional core promoter complex (SNAPc) shared RNA polymerases II III. Genes Dev. <br />                                               1998; 12:2664&#226;&#8364;&#8220;2672. [PubMed: 9732265] <br />                                            Hernandez G Jr, Valafar F, Stumph  Insect small nuclear RNA gene promoters evolve rapidly                                               retain conserved features involved determining promoter activity RNA polymerase <br />                                               specificity. Nucleic Acids Res. 2007; 35:21&#226;&#8364;&#8220;34. [PubMed: 17148477] <br />                                            Iwakoshi M, Okamoto N, Harada N, Nakamura T, Yamamori S, Fujita H, Niikawa N, Matsumoto N. <br />                                               9q34.3 deletion syndrome unrelated children. J Med Genet  2004; 126A:278&#226;&#8364;&#8220;283. <br />                                               [PubMed: 15054842] <br />                                            Jawdekar GW, Henry RW. Transcriptional regulation human small nuclear RNA genes. Biochim <br />                                                 Biophys Acta. 2008; 1779:295&#226;&#8364;&#8220;305. [PubMed: 18442490] <br />                                            Jin SW, Beis D, Mitchell T, Chen JN, Stainier DY. Cellular molecular analyses vascular tube <br />                                                 lumen formation zebrafish. Development. 2005; 132:5199&#226;&#8364;&#8220;5209. [PubMed: 16251212] <br />                                            Kim MK, Kang YS, Lai HT, Barakat NH, Magante D, Stumph  Identification SNAPc subunit <br />                                                 domains interact specific <span id='am-11' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-15' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> positions U1 U6 gene promoters. Mol <br />                                                 Cell Biol. 2010; 30:2411&#226;&#8364;&#8220;2423. [PubMed: 20212087] <br />                                            Konig H, Matter N, Bader R, Thiele W, Muller F. Splicing segregation: minor spliceosome acts <br />                                                 outside nucleus controls cell proliferation. Cell. 2007; 131:718&#226;&#8364;&#8220;729. [PubMed: 18022366] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Korzh S, Pan X, Garcia-Lecea M, Winata CL, Wohland T, Korzh V, Gong  Requirement                                                 vasculogenesis blood circulation late stages liver growth zebrafish. BMC Dev Biol. <br />                                                 2008; 8:84. [PubMed: 18796162] <br />                                            Lazaridis KN, Strazzabosco M, Larusso NF. cholangiopathies: disorders biliary epithelia. <br />                                                 Gastroenterology. 2004; 127:1565&#226;&#8364;&#8220;1577. [PubMed: 15521023] <br />                                            Lemaigre F, Zaret KS. Liver development update: new embryo models, cell lineage control,                                                 morphogenesis. Curr Opin Genet Dev. 2004; 14:582&#226;&#8364;&#8220;590. [PubMed: 15380251] <br />                                            Li C, Harding GA, Parise J, McNamara-Schroeder KJ, Stumph  Architectural arrangement                                                 cloned proximal sequence element-binding protein subunits Drosophila U1 U6 snRNA <br />                                                 gene promoters. Mol Cell Biol. 2004; 24:1897&#226;&#8364;&#8220;1906. [PubMed: 14966271] <br />                                            Lorent K, Moore JC, Siekmann AF, Lawson N, Pack M. Reiterative Use Notch Signal                                                 Zebrafish Intrahepatic Biliary Development. Developmental Dynamics. 2010; 239:855&#226;&#8364;&#8220;864. <br />                                                 [PubMed: 20108354] <br />                                            Ma B, Hernandez N. map protein-protein contacts small nuclear RNA-activating <br />                                                 protein complex SNAPc. J Biol Chem. 2001; 276:5027&#226;&#8364;&#8220;5035. [PubMed: 11056176] <br />                                            Matthews RP, Lorent K, Russo P, Pack M. zebrafish onecut gene hnf-6 functions                                                 evolutionarily conserved genetic pathway regulates vertebrate biliary development. Dev Biol. <br />                                                 2004; 274:245&#226;&#8364;&#8220;259. [PubMed: 15385156] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Mittal V, Ma B, Hernandez N. SNAP(c): core promoter factor built DNA-binding damper <br />                                                 deactivated Oct-1 POU domain. Genes Dev. 1999; 13:1807&#226;&#8364;&#8220;1821. [PubMed: <br />                                                 10421633] <br />                                            Neas KR, Smith JM, Chia N, Huseyin S, St Heaps L, Peters G, Sholler G, Tzioumi D, Sillence  <br />                                                 Mowat D. patients terminal deletions subtelomeric region chromosome <br />                                                 9q. J Med Genet  2005; 132:425&#226;&#8364;&#8220;430. [PubMed: 15633179] <br />                                            Neff MM, Neff JD, Chory J, Pepper AE. dCAPS, simple technique genetic analysis single <br />                                                 nucleotide polymorphisms: experimental applications Arabidopsis thaliana genetics. Plant J. <br />                                                 1998; 14:387&#226;&#8364;&#8220;392. [PubMed: 9628033] <br />                                            Ober EA, Verkade H, Field HA, Stainier DY. Mesodermal Wnt2b signalling positively regulates liver <br />                                                 specification. Nature. 2006; 442:688&#226;&#8364;&#8220;691. [PubMed: 16799568] <br />                                            Parsons MJ, Pisharath H, Yusuff S, Moore JC, Siekmann AF, Lawson N, Leach SD. Notch-responsive <br />                                                 cells initiate secondary transition larval zebrafish pancreas. Mechanisms Development. <br />                                                 2009; 126:898&#226;&#8364;&#8220;912. [PubMed: 19595765] <br />  <br />  <br />  <br />                                                    Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                                Page 11 <br />  <br />  <br />                                            Pass AK, McLin VA, Rushton JR, Kearney DL, Hastings CA, Margolin JF. Vanishing bile duct <br />                                                syndrome Hodgkin disease: case series review literature. J Pediatr Hematol <br />                                                Oncol. 2008; 30:976&#226;&#8364;&#8220;980. [PubMed: 19131796] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Pointon JJ, Harvey D, Karaderi T, Appleton LH, Farrar C, Stone MA, Sturrock RD, Brown MA, <br />                                                Wordsworth BP. Elucidating chromosome 9 association  CARD9 candidate gene. <br />                                                Genes Immun. 2010; 11:490&#226;&#8364;&#8220;496. [PubMed: 20463747] <br />                                            Raynaud P, Tate J, Callens C, Cordi S, Vandersmissen P, Carpentier R, Sempoux C, Devuyst O, <br />                                                Pierreux CE, Courtoy P, et al. classification ductal plate malformations based distinct <br />                                                pathogenic mechanisms biliary dysmorphogenesis. Hepatology. 2011; 53:1959&#226;&#8364;&#8220;1966. <br />                                                [PubMed: 21391226] <br />                                            Sadowski CL, Henry RW, Kobayashi R, Hernandez N. SNAP45 subunit small nuclear <br />                                                RNA (snRNA) activating protein complex required RNA polymerase II III snRNA gene <br />                                                transcription interacts TATA box binding protein. Proc Natl Acad Sci U S  1996; <br />                                                93:4289&#226;&#8364;&#8220;4293. [PubMed: 8633057] <br />                                            Sakaguchi T, Kikuchi Y, Kuroiwa  Takeda H, Stainier DY. yolk syncytial layer regulates <br />                                                myocardial migration influencing extracellular matrix assembly zebrafish. Development. <br />                                                2006; 133:4063&#226;&#8364;&#8220;4072. [PubMed: 17008449] <br />                                            Sakaguchi TF, Sadler KC, Crosnier C, Stainier DY. Endothelial signals modulate hepatocyte <br />                                                apicobasal polarization zebrafish. Curr Biol. 2008; 18:1565&#226;&#8364;&#8220;1571. [PubMed: 18951027] <br />                                            Schimanski B, Nguyen TN, Gunzl  Characterization multisubunit transcription factor complex <br />                                                essential spliced-leader RNA gene transcription Trypanosoma brucei. Mol Cell Biol. 2005; <br />                                                25:7303&#226;&#8364;&#8220;7313. [PubMed: 16055738] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Shanmugam M, Hernandez N. Mitotic functions SNAP45, subunit small nuclear RNA- <br />                                                activating protein complex SNAPc. J Biol Chem. 2008; 283:14845&#226;&#8364;&#8220;14856. [PubMed: 18356157] <br />                                            Strazzabosco M, Spirli C, Okolicsanyi L. Pathophysiology intrahepatic biliary epithelium. J <br />                                                Gastroenterol Hepatol. 2000; 15:244&#226;&#8364;&#8220;253. [PubMed: 10764023] <br />                                            Strazzabosco M, Fabris L, Spirli C. Pathophysiology cholangiopathies. J Clin Gastroenterol. 2005; <br />                                                39:S90&#226;&#8364;&#8220;S102. [PubMed: 15758666] <br />                                            Wong MW, Henry RW, Ma B, Kobayashi R, Klages N, Matthias P, Strubin M, Hernandez N.                                                large subunit basal transcription factor SNAPc Myb domain protein interacts                                                Oct-1. Mol Cell Biol. 1998; 18:368&#226;&#8364;&#8220;377. [PubMed: 9418884] <br />                                            Yoon JB, Murphy S, Bai L, Wang  Roeder RG. Proximal sequence element-binding transcription <br />                                                factor (PTF) multisubunit complex required transcription RNA polymerase II-                                                RNA polymerase III-dependent small nuclear RNA genes. Mol Cell Biol. 1995; 15:2019&#226;&#8364;&#8220;2027. <br />                                                [PubMed: 7891697] <br />                                            Yoon JB, Roeder RG. Cloning proximal sequence element-binding transcription factor subunits <br />                                                (gamma delta) required transcription small nuclear RNA genes RNA <br />                                                polymerases II III interact TATA-binding protein. Mol Cell Biol. 1996; 16:1&#226;&#8364;&#8220;9. <br />                                                [PubMed: 8524284] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                   Page 12 <br />  <br />  <br />                                                                            Research Highlights <br />                                            identified novel zebrafish mutation, snapc4s445. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            intrahepatic biliary network degenerates snapc4s445 mutant larvae. <br />                                            snapc4s445 mutation deletes Snapc2 interaction domain. <br />                                            Snapc4-Snapc2 interaction important maintenance intrahepatic <br />                                            biliary network. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                              Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                            Page 13 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 1. <br />                                            Morphological physiological phenotype s445 mutant larvae 120 hpf.  B) <br />                                            Bright-field images wild-type   s445 mutant (B) larvae 120 hpf overlaid                                            Tg(gutGFP)s854 expression (green). physiological appearance s445 mutant larvae                                            normal, liver size smaller 120 hpf. Lateral views, anterior left. <br />                                            White arrowheads point liver. (C D) Fluorescent image wild-type (C) s445 <br />                                            mutant (D) Tg(fabp1a:DsRed)gz15 larvae 120 hpf. Liver size visualized                                            Tg(fabp1a:DsRed)gz15 expression smaller s445 mutant larvae. Lateral views, anterior                                            left. (E F) Fluorescent images wild-type (E) s445 mutant (F) larvae soaked                                            PED6. PED6 processed phospholipase fluorescence detected intestine                                            wild-type s445 mutant larvae.  PED6 fluorescence missing                                            s445 mutant gallbladder. Lateral views, anterior right. Asterisks indicate intestine. <br />                                            (G H) Projected confocal image gallbladders wild-type (G) s445 mutant <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                        Page 14 <br />  <br />  <br />                                            (H) Tg(gutGFP)s854 larvae. gallbladder extrahepatic bile duct visualized                                            Tg(gutGFP)s854 expression schematically presented (G&#226;&#8364;&#8482; H&#226;&#8364;&#8482;). Ventral views, <br />                                            anterior  <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                          Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 2. <br />                                            Time course observation intrahepatic biliary network development.  C) Tg(Tp1- <br />                                            MmHbb:EGFP)um14 expression wild-type intrahepatic biliary network 80  , 100 <br />                                            (B) 120 (C) hpf; larva shown stages. (D&#226;&#8364;&#8220;F) Tg(Tp1- <br />                                            MmHbb:EGFP)um14expression s445 mutant intrahepatic biliary network 80  , <br />                                            100 (B) 120 (C) hpf; larva shown stages. 80 hpf, Tg(Tp1- <br />                                            MmHbb:EGFP)um14 expression s445 mutant intrahepatic biliary network                                            indistinguishable wild-type, disappears 100 hpf. Lateral views, <br />                                            anterior left. White arrowheads point liver image. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                       Page 16 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 3. <br />                                            intrahepatic biliary network disappears s445 mutant larvae. Projected  D J&#226;&#8364;&#8220;K) <br />                                            plane (E&#226;&#8364;&#8220;F H  confocal images liver 80  B), 100 (C&#226;&#8364;&#8220;F) 120 <br />                                            (H&#226;&#8364;&#8220;K) hpf. Ventral views, anterior   B) Wild-type   s445 mutant (B) <br />                                            larvae visualized Alcam expression. Alcam-positive intrahepatic biliary network                                            s445 mutant larvae indistinguishable wild-type 80 hpf. (C D) Wild- <br />                                            type (C) s445 mutant (D) Tg(Tp1-MmHbb:EGFP)um14 larvae visualized GFP <br />                                            expression. Tg(Tp1-MmHbb:EGFP)um14 expression liver restricted                                            intrahepatic biliary network 100 hpf. intrahepatic biliary network s445 mutant <br />                                            appears degenerating stage. (E F) Wild-type (E) s445 mutant (F) <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                        Page 17 <br />  <br />  <br />                                            Tg(Tp1-MmHbb:EGFP)um14 larvae stained red fluorescent TUNEL. GFP expression                                            TUNEL signals outlined area magnified shown separately F&#226;&#8364;&#8482; F&#226;&#8364;?, <br />                                            respectively. (G) Number TUNEL positive Tg(Tp1-MmHbb:EGFP)um14 expressing cells <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            wild-type (open square) s445 mutant (closed circle) liver. *, p &lt; 0.05. (H  <br />                                            Wild-type (H) s445   mutant Tg(Tp1-MmHbb:EGFP)um14; Tg(fabp1a:DsRed)gz15 <br />                                            larvae visualized GFP DsRed expression. Tg(Tp1-MmHbb:EGFP)um14 expressing <br />                                            biliary epithelial cells largely disappeared s445 mutant livers 120 hpf,                                            Tg(fabp1a:DsRed)gz15expressing hepatocytes remain. GFP expression shown separately <br />                                            H&#226;&#8364;&#8482; . (J K) Wild-type (J) s445 mutant (K) Tg(kdrl:EGFP)s843 larvae <br />                                            visualized GFP (green) Alcam (red) expression. Tg(kdrl:EGFP)s843-expressing <br />                                            intrahepatic vascular network relatively unaffected Alcam-expressing <br />                                            intrahepatic biliary network severely affected s445 mutant larvae. Alcam GFP <br />                                            expressions shown separately (J&#226;&#8364;&#8482; K&#226;&#8364;&#8482;) (J&#226;&#8364;? K&#226;&#8364;?), respectively. (L M) <br />                                            Electron micrographs wild-type (L) s445 mutant (M) liver 120 hpf. h, <br />                                            hepatocyte; c, cholangiocyte; l, lumen cholangiocyte. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                         Page 18 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 4. <br />                                            Positional cloning s445 mutant locus.   Schematic 445 locus. s445 located <br />                                            chromosome 5 CA-repeat markers, z26603 (0.2 cM; 7 recombinants                                            3072 meioses) CA-YE2 (0.07 cM; 2 recombinants 3056 meioses). critical <br />                                            region s445 locus spanned partially overlapping BAC clones, <br />                                            DKEY-20G15 DKEY-1O2. (B) single nucleotide mutation (C T) results                                            premature stop snapc4 gene identified s445 mutants (0 recombinants &gt; <br />                                            3000 meioses). (C) Schematic zebrafish Snapc4. Zebrafish Snapc4 contains conserved <br />                                            Myb DNA binding domain Snapc2 binding domain. s445 mutation generates                                            premature stop codon zebrafish snapc4 cDNA. truncated mutant Snapc4 protein <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                             Page 19 <br />  <br />  <br />                                            approximately thirds length wild-type Snapc4 lacks Snapc2 binding <br />                                            domain. (D) s445 located zebrafish chromosome 5 region syntenic region                                            human chromosome 9q34.3. (E) snapc4 expression wild-type larvae 80 hpf. snapc4 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            expression detected liver stage. outlined area magnified shown                                            E&#226;&#8364;&#8482;. Lateral views, anterior left. Cross section stained larva region                                            liver shown E&#226;&#8364;?. liver outlined E&#226;&#8364;&#8482; E&#226;&#8364;?. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                         Page 20 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 5. <br />                                            snRNA expression s445 mutant larvae.   Real-time PCR compare snRNA gene <br />                                            expression wild-type s445 mutant larvae 100 hpf. Data represented                                            fold change relative expression levels wild-type larvae. expression levels U1, <br />                                            U2, U6 snRNAs significantly changed, U4 (average 0.59 fold; <br />                                            s.d. = 0.22; p = 0.037) U5 (average 0.36 fold; s.d. = 0.29; p = 0.024) snRNAs  <br />                                            regulated s445 mutant larvae. *, p &lt; 0.05; N.S., significant (p &gt; 0.05). (B) Northern <br />                                            blotting snRNAs. Total RNA wild-type s445 mutant larvae 100 hpf                                            blotted snRNA probes indicated genes. probe 28S ribosomal RNA                                            used loading control. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                         Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            Figure 6. <br />                                            Zebrafish Snapc2 essential intrahepatic biliary network formation.  B) Bright- <br />                                            field images wild-type   snapc2 morpholino injected (B) larvae 100 hpf                                            overlaid Tg(gutGFP)s854 fluorescence (green). snapc2 morpholino injected larvae                                            smaller livers mild pericardial edema. (C) snapc2 expression wild-type larvae 80 <br />                                            hpf. snapc2 expression detected liver stage. outlined area magnified <br />                                            shown E&#226;&#8364;&#8482;. liver outlined E&#226;&#8364;&#8482;. (D E) Tg(Tp1bglob:EGFP)um14 expression <br />                                            intrahepatic biliary network wild-type (D) snapc2 morpholino injected (E) <br />                                            larvae 100 hpf. Tg(Tp1-MmHbb:EGFP)um14 expression absent snapc2 <br />                                            morpholino injected liver. (F G) Tg(fabp1a:DsRed)gz15 expression hepatocytes  <br />                                                   Dev Biol. Author manuscript; available PMC 2013 July 15. <br />                             Schaub et al.                                                                                            Page 22 <br />  <br />  <br />                                            wild-type (F) snapc2 morpholino injected (G) larvae 100 hpf. Lateral views, anterior <br />                                            left. White arrowheads point liver image. (H) Reverse transcriptase- <br />                                            PCR analysis shows 52 bp deleted snapc2 mRNA snapc2 MO  injected <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            larvae.   Realtime PCR compare snRNA gene expression wild-type snapc2 <br />                                            MO  injected larvae 60 hpf. Data represented fold change relative expression <br />                                            levels wild-type larvae. expression levels U1 U6 snRNAs significantly <br />                                            changed, U2 (average 0.75 fold; s.d. = 0.153), U4 (average 0.657 fold; s.d. = <br />                                            0.02) U5 (average 0.368 fold; s.d. = 0.218) snRNAs regulated s445 mutant <br />                                            larvae. *, p &lt; 0.05; N.S., significant (p &gt; 0.05). (J) Bright-field image snapc2 binding <br />                 <br /> </body></html>