<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Dec. 1994, p. 8037-8050                                                                        Vol. 14,  12 <br /> 0270-7306/94/$04.00+0 <br /> Copyright &#194;&#169; 1994, American Society Microbiology <br />  <br />  <br />                 Mutations Saccharomyces cerevisiae CDC1 Gene <br />                         Affect Double-Strand-Break-Induced <br />                          Intrachromosomal Recombination <br />                                          JAMES HALBROOK* MERL F. HOEKSTRA <br />                                              ICOS Corporation, Bothell, Washington 98021 <br />                          Received 8 June 1994/Returned modification 19 July 1994/Accepted 21 September 1994 <br />  <br />              isolate Saccharomyces cerevisiae mutants defective recombinational DNA repair, constructed strain <br />           contains duplicated ura3 alleles flank LEU2 ADES genes ura3 locus chromosome V. <br />           HO endonuclease cleavage site located ura3 alleles, Ura+ recombination           increased 100-fold wild-type strains following HO induction GAL1,10 promoter. strain <br />           used screen mutants exhibited reduced levels HO-induced intrachromosomal recombination <br />           significantly affecting spontaneous frequency Ura+ recombination. mutations isolated <br />           screen affect essential gene CDC]. mutation, cdcl-100, completely <br />           eliminated HO-induced Ura+ recombination maintained spontaneous preinduced recombination <br />           levels cell viability. cdcl-100 mutants moderately sensitive killing methyl methanesulfonate           -y irradiation. elect cdcl-100 mutation recombinational double-strand break repair indicates           recombinationally silent mechanism sister chromatid exchange responsible efficient <br />           repair DNA double-strand breaks. <br />  <br />    Genomic integrity maintained generations                 cycle progression arrested hydroxyurea (10) cyclohex- <br /> coordinated control DNA replication chromosome                         imide (53) wild-type cells. data suggest abnor- <br /> segregation cell growth division. important aspect                mally processed DSBs responsible cell death rad52 process recognition repair DNA lesions                  mutant cells. arise course normal cellular metabolism                    Experiments HO-induced DSBs introduced exposure genotoxic agents environment.                  ribosomal DNA cluster 18-fold repeated CUP1 <br /> yeast Saccharomyces cerevisiae, toxicity DNA double-strand                 gene locus demonstrate efficient repair occur breaks (DSBs) dependent factors, including                    RAD52-independent mechanism (43). CUPJ repeats <br /> ploidy (9, 15, 48), cell cycle position (6, 39, 49), genetic          reduced 18 3,  RAD52 dependency <br /> constitution cell (15, 36). Prior DNA replication, <br /> haploid cells G1 phase particularly sensitive killing           DSB repair restored. RAD52-independent DSB repair <br />                                                                               tandemly repeated sequences thought result ionizing radiation, indicating DSBs repaired DNA homology-dependent mechanism. Mutant strains defec-                       extensive single-stranded exonucleolytic DNA diges- <br /> tive recombinational DNA repair processes, notably                    tion likely causes lethal lesions DSBs introduced mediated RAD52 epistasis group, reduced                   unique DNA sequences. Single-stranded DNA degra- <br /> threshold level DSB tolerance (26, 49). Mutations                 dation adjacent repeats provide complementary <br /> genes confer sensitivity killing physical <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span>                  single-strand DNA chains heteroduplex formation agents generate DSBs, ionizing radiation                     restoration chromosome continuity (14, 43, 56). rad52 <br /> methyl methanesulfonate (MMS). collection genes                  mutant cells proficient repair damage affect genetic recombination interpreted                     multiply repeated elements, likely DSBs indicate role mitotic recombination repair DSBs               intrinsically responsible cell death, DNA breaks <br /> (15, 23, 36, 48).                                                             lethally processed absence redundant intrachro- <br />    DSBs inhibit cell division cycle unless repaired                   mosomal homology. <br /> represent lethal lesions. single DSB generated enzymatically                   observations led consider repair processes HO endonuclease MA4T (40, 59) HO cut site                   identified mutations generate lethal <br /> introduced genome (42, 53) causes death                   lesions following introduction genomic DSBs. rad52 cells. Estimates physical analysis genomic DNA                  regard particular recombinational DNA repair model, X-irradiated yeast cells indicated approximately                interested isolating mutants affected HO-induced <br /> DSB responsible lethality radS2 mutants (27, 49).                  DSB recombination directly repeated genes Unrepaired DSBs lethal strains                  grossly altering level spontaneous recombination. Al- <br /> lethal trans, zygotes formed X-irradiated untreated rad52 rad54 mutant strains inviable (27).                    spontaneous mitotic recombination thought result <br /> Extensive exonucleolytic DNA degradation occurs                      repair DNA lesions incurred vegetative <br /> site HO cut rad52 mutant strains (61). contrast,                   growth (44), repair necessarily proceed HO-generated DSBs stable hours cell                      result DSB. reasoned requiring basal-level <br />                                                                               background recombination proficiency mutants <br />                                                                               uncover set reactions unique processing   * <br />     Corresponding author. Mailing address: ICOS Corporation, 22021            repair DSBs. report DSB-stimulated <br /> 20th Ave. SE, Bothell, WA 98021.                                              recombination mutants capacity repair                                                                        8037 <br />  8038      HALBROOK HOEKSTRA                                                                                      MOL. CELL. BIOL. <br />  <br /> HO-induced DSB duplicated ura3 gene                    Strains. K1864 (AL4 Ta his4-260 ade2-1 ade5 ura3-52 leu2- <br /> stimulating frequency Ura&apos; recombination.                     3,112 trp1-HIII canl 1ys2-ABX::CAN1 LYS2) gift R. <br />                                                                      Keil (38). ura3-52 allele replaced (52) ura3 gene <br />                METHODS MATERIALS                                 containing NotI recognition site StuI (ura3-SNot).                                                                      strain transformed <span id='am-42' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> prototrophy linearized <br />    Plasmids. pREGIA-1 disruption plasmid donated             hisG-URA3-hisG (2) disrupted allele REG1 gene. Ura+ N. Dhillon. Briefly, REG1 gene (provided M.                transformant colonies streaked media containing <br /> Johnston) inserted ClaI site pBluescript II        5-fluoro-<span id='am-12' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orotic acid</span> (5); 5-fluoro-orotic acid-resistant papillae <br /> (Stratagene, La Jolla, Calif.) derivative lacked BamHI        tested uracil auxotrophy, deletion REG1 <br /> site. Digestion BamHI released entire REG1 coding           gene confirmed DNA hybridization. Strain 543 (Table <br /> sequence, replaced BglII-BamHI fragment             1) constructed targeted integration pALUM pNKY51 (2) containing URA3 gene flanked                  ura3-SNot allele. yielded ura3 duplication, ura3-SNot:: <br /> Salmonella hisG DNA.                                                 ADE5 LEU2 ura3-HOcs (Fig. 1). <br />    ura3-HOcs plasmid, pALUM, constructed insert-             Strain 543 transformed histidine prototrophy ing ura3-CIR HindIII fragment YIplacl28 (17). ura3       targeted integration (52) pHGH12 generate partial <br /> allele ClaI site changed EcoRI site (42).           duplication his4 locus, his4-260A::GAL1,10-HO HIS4. <br /> approximately 5-kb ADE5 fragment pL1067 (provided            structure locus strain 619 confirmed DNA <br /> R. Keil) introduced polylinker              hybridization. <br /> YIplacl28 backbone create pALU. 117-bp HO site                Targeted integration YIpCDCl strains 622 MATa (34) flanked EcoRI linkers inserted yield             601-7D (Table 1) carried transforming plasmid <br /> pALUM. Integration pALUM targeted ura3                 linearized unique XhoI site (52). resulting trypto- <br /> locus digestion NsiI.                                        phan prototrophic strains, 619-1 (CDCl::CDCl TRP1)    Plasmid pRSgalHO (TRP1) contains 3.7-kb EcoRI-SalI              601-7D (cdcl-100::CDC1 TRPJ), confirmed Southern <br /> GAL1,10 promoter-HO endonuclease DNA fragment                   analysis. strain showed identical shift 7 kb pGALHO (31) pRS314 (54).                                          unique CDC1 DNA band analyzed BglII    GAL1,10:HO integration plasmids, pHGH12                   ClaI restriction enzyme digestion genomic DNA. <br /> pHGH15, contain 1.6-kb Sall fragment 470 bp 5&apos;            Mutant genes introduced strain 619 genetic <br /> HIS4 coding sequence pR82 (gift R. Keil) SalI           background backcrossing mutant segregants 619 XhoI sites pBluescript 11-SK (Stratagene). resulting      derivatives times. initial cdcl-100 mutation <br /> plasmid digested EcoRI EcoRV ligated           backcrossed (seven times) strain 619. 2.5-kb EcoRI-EcoRV GAL1,10:HO fragment                      rad52A::TRPJ mutation strain W838-1OB gift R. <br /> pGALHO (31). Plasmid pHGH12 GAL1,10:HO his4A                 Rothstein. Strains 369 (cdcl-1) 342 (cdcl-2) (Table 1) <br /> oriented transcribed opposite directions,        provided L. Hartwell. oriented transcribed           Yeast culture conditions. Yeast methods media direction pHGH15. Targeted integration          essentially described Rose et al. (51). YEPD 1% plasmids strain 543 following linearization      (wt/vol) yeast extract, 2% (wt/vol) Bacto Peptone, 2% <br /> XhoI resulted truncated 5&apos; his4 allele contained      (wt/vol) glucose. YEPD(0.5) contained 0.5% (wt/vol) glucose. his4-260 mutation HIS4 gene flanking                   YEP lacks carbon source. YEPGal supplemented GAL1,10:HO gene. HO expression affected               2% galactose instead glucose. YEPDGal contained relative orientation HIS4 GAL1,10:HO genes.                   glucose galactose 1% (wt/vol)  SC synthetic <br />    Plasmid YIpCDC1 inserting 3.8-kb BglII              complete medium (2% glucose), SC-H lacked histidine, <br /> fragment containing CDC1 BglII site YIplac2O4            SC-LH lacked histidine leucine, SC-U lacked uracil, SC-T <br /> (17).                                                                lacked tryptophan, SC-LT lacked leucine tryptophan. <br />  <br />                                            TABLE 1. S. cerevisiae strains used study <br />  Strain                                                                                        Genotype <br /> 543                        ..M.                        LYS2::CAN1 lys2-ABX reglA <br />                                                          ATa ho ura3-SNot:ADE5 LEU2 ura3-HOcs <br />            his4-260 ade2-1 ade5  leu2-3,112 trpl-HIII <br /> 619   ...   4Tat ho ura3-SNot::ADE5 LEU2 ura3-HOcs LYS2::CAN1 lys2-ABX reglA <br />            his4A-260::GALJ 10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII <br /> 622 ...M4Ta ho ura3-SNot::ADE5 LEU2 ura3-HOcsslys2-ABX reglA his4A-260::GALl <br />            10-HO ade2-1 adeS canl leu2-3,112 <br /> 601-1B    ATa ho ura3-SNot::ADE5 LEU2 ura3-HOcs LYS2::CANJ lys2-ABX reglA <br />            his4A-260::GAL1 10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII cdcl-100 <br /> 601-7D    MAT <br />           ....      ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA his4A-260::GALJ <br />                                                                    o <br />  <br />            10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-100 <br /> 295-4A     4Ta ho ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA his4A-260::GALJ <br />                                                         <br />  <br />  <br />  <br />            10-HO ade2-1 ade5 canl leu2-3,112 trpl-HIII radS2::TRPJ <br /> 648-lA                                                  MATa ho ura3-SNot:ADE5 LEU2 ura3-HOcs lys2-ABX reglA      his4A-260::GALl <br />                                                           10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-1 <br /> 631-4A    ...                                                           his4A-260::GALl <br />                                                         MATa ho ura3-SNot::ADE5 LEU2 ura3-HOcs lys2-ABX regl              10-HO ade2-1 adeS canl leu2-3,112 trpl-HIII cdcl-2 <br /> 369 ....MATa adel ade2 ural his7 tyrl lys2 gall cdcl-l <br /> 342        MATa adel ade2        his7 tyrl lys2 gall cdc]-2              ural <br />  <br /> 488   .....4.             o .       -H <br />               leu2-3.112   t          rl                  II rA canl ura3-SNot ade2-1 ade5 <br />            MAToa leu2-3,112 his4-260 lys2-ABX ho trpl-HIII reglA canl ura3-SNot ade2-1 adeS <br />  VOL. 14, 1994                                                               CDCI GENE S. CEREVISIAE   8039 <br />  <br />  <br />  <br />  <br />                                          X <br />                            CS <br />                 QO <br />                  CD <br />                 c. <br />                        CD            - <br />  <br />                                                    D <br />                            0.   n                                                                                        CN <br />  <br />  <br />  <br />  <br />                                               Lo <br />  <br />                                               0 <br />  <br />  <br />  <br />  <br />                        CD       f-~~~~D <br />  <br />                                                   0.                                                                                   mA                   0 <br />  <br />  <br />                                               DI                                   CD <br />                                                                                     y <br />  <br />                                                                X <br />                                                    CoI <br />                                                    CD                                    Xf, <br />  <br />                                                          N              0 <br />  <br />  <br />  <br />  <br />                                                                     <br />                                                                    SI                         0 <br />  <br />  <br />                                                                    0u <br />  <br />                                                          oft <br />                                                          ro <br />                                          c <br />  <br />  <br />  <br />                                                                Ut1 <br />                                                                0b <br />                                                                                                                                4., o <br />                                                                aD <br />  <br />  <br />  <br />  <br />                                                                                          Qj <br />                                 N <br />  <br />  <br />  <br />  <br />                       C: <br />  <br />  <br />  <br />  <br />                                                                                      N <br />  <br />  <br />  <br />  <br />                        w <br />  <br />  <br />  <br />                            0a <br />  8040    HALBROOK HOEKSTRA                                                                                    MOL. CELL. BiOL. <br />  suffix DG indicates glucose galactose (1%       19% efficiency produced 5% spored asci. supplied (e.g., SC-HDG).                                     132 dissected tetrads incubated 240C 3 days. Plates <br />    Sporulation medium 1% potassium acetate, 0.1% Bacto        replicated SC medium (incubated 24 370C) Yeast Extract, 0.05% glucose, 1.5% agar complete       SC-T medium (incubated 240C). Viability dissected <br /> supplement amino acids (51).                                   tetrads 88%. 119 viable spored tetrads exhibited <br />    Solid medium containing methyl methanesulfonate (MMS;          parental ditype segregation pattern, confirming identity <br /> Kodak Chemical , Rochester, N.Y.) prepared         cloned (cdcl-100-suppressing) gene CDC1. 24 h advance use. MMS (100%) added           Recombination assays. SC-LH medium containing 0.5% <br /> YEPD medium allowed cool prior pouring             glucose (5 ml) inoculated single colony incu- <br /> plates.                                                           bated overnight vigorous aeration. Cells diluted    incubations carried 30&apos;C noted.    5 ml YEPD(0.5) 106 cells ml incubated cdcl-1 cdcl-2 mutant cells incubated 240C         2 4 h. HO endonuclease induced addition 20% <br /> missive temperature).                                             galactose 2% final concentration. yields final ratio <br />    Mutant isolation. single colony strain 543 (pRSGAL-       galactose glucose 4 1. degree induction HO) inoculated 5 ml SC-LT medium grown            GAL1,10 promoter regl strain dependent overnight 30&apos;C. culture diluted 10 ml fresh   absolute galactose concentration galactose  <br /> SC-LT medium grown hours concentration        glucose ratio (18, 29). Aliquots removed 1 X 106 2 X 106 cells ml addition 200 ,ul     galactose addition plated YEPD SC medium ethyl methanesulfonate (100%; Kodak). culture di-         determine viable cell concentration SC-U medium vided equal portions incubated additional    select Ura&apos; recombinants. Recombination frequencies 90 min 30&apos;C. Sodium thiosulfate (50%) added 5%        calculated ratio Ura&apos; colonies viable cells plated. <br /> final concentration, cells pool plated    Red, white, half-sectored red/white colonies scored <br /> SC-LT medium density 1,000 3,000 cells plate   SC (unselected) SC-U (selected) media. <br /> incubated 30&apos;C. conditions, viability typi-       Recombination measured serially replica plating <br /> cally reduced approximately 10%, resulting 100 300       patches master plate ultimately SC-U medium <br /> colonies plate.                                               growth SC-H SC-HDG media  strains derived <br />    Colonies mutagenized cells replicated HO-         619). number Ura&apos; papillae formed uracil <br /> repressing medium (SC-T) HO-inducing medium (SC-           omission medium pregrown glucose plus galactose relative <br /> TDG), incubated overnight 30&apos;C, replicated SC-U.        pregrown glucose indication dissection microscope, differences frequency     level HO-stimulated recombination. <br /> Ura&apos; recombination estimated comparing level          Mating-type switching assayed filtering 106 log-phase <br /> Ura&apos; papillae derived colonies previously grown SC-T      cells (prepared single colonies described  SC-TDG. Potential isolates, demonstrating HO-stimu-        nitrocellulose discs. Filters washed YEP  <br /> lated Ura&apos; recombination, rescreened eliminate false-     dium, placed YEPD YPGal medium, incu- <br /> positive, petite, galactose utilization-defective strains.    bated 300C. Filters removed plates    CDC1 gene cloning molecular analysis. recom-        indicated times, suspended YEP medium, serially diluted, <br /> bination defect cdcl-100 mutation recessive,        plated SC medium 30&#194;&#176;C. 2 3 days, colonies <br /> sought identify molecular DNA clone genetic comple-       replica plated mating-type tester lawns scored <br /> mentation (50). Strain 601-7D (cdcl-100) transformed          2 days. ALTa strains, mating-type interconversion <br /> (13) yeast genomic DNA centromere library (ATCC            expressed (number AL4To colonies + number 77164 [provided P. Hieter]). Tryptophan prototrophs       nonmating colonies)/total number colonies.  <br /> isolated screened HO-stimulated recombination pro-        ground frequency ALT switching cells approxi- <br /> ficiency. 900 colonies screened, recombination-        mately 1 2%. high level presumably leaky <br /> proficient transformants obtained. Structural analysis    expression HO endonuclease. <br /> plasmids isolated strains indicated repre-      DNA damage induction. Single colonies used inoc- <br /> sented genomic segment. TnJOLUK mutagenesis              ulate 5 10 ml SC-LH medium grown overnight. <br /> (30) yielded transposon insertions near pair closely     Cultures diluted 1 X 106 2 X 106 cells ml juxtaposed BamHI sites eliminated HO-induced recombi-        YEPD(0.5) grown 2 3 h. radiation-induced <br /> nation. region complementing 3.5-kb BglII frag-     damage, cells counted serially diluted, 5 ,ul ment subcloned pRS200 (ATCC 77165 [54]) revealed           placed YEPD medium. Aliquots 10-fold serial <br /> single open reading frame 1,473 nucleotides (491 amino         dilutions contained average 3 X 104 30 cells acids). plasmid designated p220. Deletion 390-bp    spot. Plates immediately irradiated -y UV <br /> BamHI fragment resulted frame deletion 130 amino      light, using Mark 68A 137Cs _y irradiator (J. L. <br /> acid residues (p221). Insertion 3.8-kb BglII-BamHI frag-     Shepard <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Associates, San Fernando, Calif.) used 414 <br /> ment containing URA3 gene flanked Escherichia coli         krads/min 15-W germicidal lamp (General Electric) hisG repeats pNKY51 (2) remaining BamHI site        flux 3.3 J/m2/s. Irradiated cells incubated 30&#194;&#176;C  p221 yielded deletion/insertion mutation CDC1            scored 16 48 h. <br />  (cdc1::URA3) p222.                                               Treatment MMS carried adding drug <br />    cdcl::URA3 allele used replace       directly log-phase cultures prepared irradiation <br /> wild-type copies CDC1 diploid strain 488 transforming    experiments. Cultures grown aeration 30&#194;&#176;C 1 p222 linearized digestion BglII NotI.          h. Sodium thiosulfate (50%) added 5% final concen- <br />  candidate uracil prototroph sporulated, tetrads     tration, cells serially diluted placed YEPD <br />  dissected.                                                       medium described  Cell growth scored 16     demonstrate cloned gene fact CDC1,         48 h. <br />  strain 619-1 mated strain 648-1A, diploids          Cell viability continuous exposure MMS  sporulated 5 days 30&#194;&#176;C. diploid strain sporulated     determined transferring serially diluted cells log-phase <br />  VOL. 14, 1994                                                                               CDCJ GENE S. CEREVISAE             8041 <br />  <br /> cultures medium containing MMS. Cells incubated               measure recombination. Viability determined micro- <br /> 30&apos;C scored 16 48 h.                                    scopic analysis plated cells. Percent viability number <br />    Cell viability function level HO expression     microcolonies divided total number cells plated. <br /> assayed transferring serially diluted log-phase cells         Ura&apos; recombination frequency number uracil <br /> YEPD, YEPDG, YEPGal media. Cells incubated 30&apos;C                prototrophs divided total number cells plated. scored 16 48 h.                                         experiment carried times. <br />    cdcl-100 strains grow slowly solid media,              <span id='am-22' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> sequence accession number. nucleotide se- <br /> examined cells microscopically scored viable cell growth       quence data reported paper appear EMBL, <br /> terms microcolony formation. Viable microcolonies             GenBank, DDBJ nucleotide sequence databases usually characterized uniform radial growth producing              accession number X81813. <br /> round colonies composed normally <br /> shaped (round) cells. Inviable terminal microcolonies                                      RESULTS <br /> characterized jagged unevenly edged colonies composed 2 20 aberrantly shaped, enlarged elongated cells.          DSB-stimulated genetic recombination assay. design- colonies, composed cells capable          ing genetic recombination assay, goal create limited number cell divisions, did support continued           allow identify characterize mitotic <br /> growth did increase size time. Viable cell            recombination-defective mutants. primary elements frequencies calculated number viable microcolo-        assay   regulated initiation recombination <br /> nies divided total number cells microcolonies           events introduction specific DSB (ii) counted.                                                              duplicated chromosomal gene DSB targeted. <br />    method reproducible cdcl-100             nontandem duplication ura3 locus chromosome V <br /> cells examined media containing MMS galactose           generated integration plasmid containing LEU2, sole carbon source. Colony growth rate                ADES, ura3 gene resides 117-bp HO <br /> conditions qualitatively reduced relative growth            endonuclease recognition sequence (Fig. 1). Recombination <br /> YEPD YEPDGal media. combina-                 events occurred duplicated ura3 genes tion effects including expanded cell division cycle time        inferred colonies papillae formed uracil increased rate killing. cdel-100 cells grown       omission medium. parental yeast strain (Table 1) ade2 conditions scored 2 days 1 day.          adeS mutations formed white colonies. ADES perturbation cell growth partly reflected          provided ura3 duplication, <span id='am-17' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> biosynthetic <br /> large standard deviation independent determina-           pathway blocked ade2 step, cells accumulate red <br /> tions. Contributing variation microcolony formation         metabolic intermediate. Colony color used distinguish YEPGal MMS media difficult                   recombination events resulted deletion distinction viable nonviable microcolonies.            DNA flanked ura3 alleles (Fig. 1) gene conversion <br /> cause colonies combinations cells normal            events associated deletions (Fig. 1). abnormal morphology, subjective decision            yielded white Ura- (Leu-) Ura&apos; (Leu-) recombinants, basis relative number cell types             yielded red Ura&apos; colonies. Ura&apos; recombi- <br /> microcolony. Despite obstacle describing cell growth       nants arise unequal sister chromatid exchange media, clear cdcl-100 cells capable         (SCE) events. <br /> completing repeated cell division cycles conditions          potential recombinant products obtained completely inhibited radS2A cell growth  Tables 3 4).         repair reactions sister chromatids. examples    Recombination rate determinations. Mitotic recombination           represented Fig. 1: unaltered ura3 locus generated frequencies determined fluctuation analysis. YEPD             equal SCE event triplicated ura3 gene resulting medium (3 ml) inoculated approximately 50 cells,             gene conversion associated crossover mis- grown 30&apos;C late log phase  45 h). Cells        aligned sister chromatids. S(NotI) plated SC medium determine viable cell number,             RV(HO) mutations converted unequal SCE SC-U medium determine uracil prototrophy,                event generate functional URA3 gene  ura3 DNA <br /> canavanine medium measure deletion formation lys2           configuration illustrated Fig. 1), expected class locus. Colonies scored 2 days strain 622 4         recombinants small. Alternatively, allele need <br /> days strain 601-1B. method median (37)             converted unequal SCE reaction associated used analyze data seven independent cultures         reciprocal exchange generate URA3 gene. <br /> determination. Recombination rates calculated divid-             Intrachromosomal recombination repeated genomic <br /> ing frequency ln n - ln (62).                               DNA sequences occurs rates order 10&apos; 10-6 <br />    DSB induction factor-arrested cells. log-phase             (1, 56, 62). Introduction HO cut site gene <br /> cultures (2 ml 1 X 106 2 x 106 cells ml) grown           duplication stimulates recombination fold <br /> YEPD(0.5) added factor (Sigma Chemical , St.                 HO endonuclease expressed (42). effec- <br /> Louis, Mo.) 5 ,uM final concentration incubated           tively use GALl,10 promoter-regulated HO means 30&apos;C. 90 min, 10 [LI 1 mM stock solution        stimulating double-strand DNA cleavage, mutation factor added incubation continued. 3 h             galactose regulatory network utilized (29, 41). Deletion medium containing factor, 95 90% wild-             REG1 gene mitigated effects catabolite repression, <br /> type cdcl-100 cells, respectively, unbudded. Cultures        allowing galactose serve gratuitous inducer HO divided half, culture strain         strains remained growing glucose. strategy provided added galactose (20%) 2% final concentration (galactose          rapid efficient HO induction shifting carbon glucose ratio, 4 1) final hour factor treatment.   sources. Furthermore, previous mutant screens designed Cells incubated galactose washed             identify mutations viable faced remove factor galactose serially diluted, 5-,ul         persistent expression GALJ,10::HO (28), aliquots plated SC score viability SC-U             predominant class mutants Gal-. reglA <br />  8042           HALBROOK HOEKSTRA                                                                                           MOL. CELL. BIOL. <br />  <br />        TABLE 2. Spontaneous DSB-induced recombination                       bination wild-type cells differences            frequencies wild-type cdcl-100 strains                       mutant wild-type colonies distinguished. <br />                                    Recombinants/viable cell (10-4                Ethyl methanesulfonate-mutagenized strains 543 619 <br />       Strain     Galac-                                                         screened colonies exhibited reduced levels                   tose      Ura+          Ura+         Ura+ white      Ura-     Ura&apos; papillae overnight growth media containing <br />                                            red                         white <br />                                                                                 glucose galactose. detailed analysis approach 622                -       7.1 &#194;&#177; 2.8    3.8 &#194;&#177; 1.8      3.2 &#194;&#177; 0.9       &lt;2.5&quot;    mutants isolated described   (wild type)                                                                   (18). Briefly, anticipated broad classes mutations <br />                    +      1,200 &#194;&#177; 280 360 &#194;&#177; 180        930 &#194;&#177; 200 250 &#194;&#177; 210      screen: strains   showed reduced viability                                                                                 galactose presumably affect reactions essential 601-1B             -        38 &#194;&#177; 11     33 &#194;&#177; 13        4.8 &#194;&#177; 1.3       &lt;3.8b    repair DNA DSBs, (ii) exhibited reduced spontaneous   (cdcl-100)                                                                    DSB-stimulated recombination represent mutations <br />                    +        56&#194;&#177; 13      49&#194;&#177; 16         6.6&#194;&#177;3.9        18&#194;&#177;8.7    affect wide range DNA metabolic reactions   Data averages &#194;&#177; standard deviations independent experiments.   influence recombination DSB repair  (iii) demon- <br />   b white colonies observed 4,000 (&lt;2.5) 2,600 (&lt;3.8)         strated spontaneous Ura&apos; recombination proficiency unselected colonies.                                                            defective DSB-stimulated recombination.                                                                                 rad52A mutant showed spontaneous HO-stimulated <br />                                                                                 Ura+ recombination reduced viability, mutants mutation reduced number Gal- false-positive results                      appear second categories. specifically analysis.                                                             focused mutations class potentially <br />    Addition galactose media containing glucose stimu-                     represent mutations play role strand break repair lated frequency Ura+ recombination 100-fold                         reported previously S. cerevisiae. <br /> (Table 2). frequency white colonies increased                        Examination 90,000 colonies formed mutagenized cells <br /> 500-fold following HO induction. spontaneous Ura+                         yielded 300 mutants. Preliminary analysis collec- <br /> recombinant colonies, 50% white,                       tion suggests complementation groups exist (18). <br /> 75% HO-induced Ura+ recombinants white.                           Partial characterization mutant strains indicated values agree reported Nickoloff et al.,                       mutant clearly satisfied criteria imparting reported use similar ura3 alleles flanked HIS3                    DSB recombination defect studied  shown <br /> marker (42). stimulation Ura+ recombination fre-                          mutation identified allele quency reglA strains directly proportional galac-                 previously described CDC1 gene, refer mutant <br /> tose concentration ratio galactose glucose (data                  allele cdcl-100. shown) (29, 41).                                                               cdcl mutant defective recombinational repair <br />    Identification recombination-defective mutants.                   DSBs. recombination defect conferred cdcl-100 <br /> DSBs recombinogenic lethal, depending                      mutation backcrossed wild-type strains 619 622 <br /> repair capacity particular strain examined,                     reexamined using regimen used initial uncertain extent DSB recombination-defective                       identification (Fig. 2A). Strain 601-7D (cdcl-100) exhibited mutants sensitive HO-induced killing. deter-                     increase Ura&apos; papillae following galactose treatment, differential levels recombination proficiency                   wild-type strain 622 showed increased frequency inhibiting cell division potentially broad spectrum DSB                 Ura&apos; papillae identical conditions. spontane- <br /> repair-defective mutants, simultaneously minimized                       ous galactose-induced recombination frequencies strain <br /> killing impact DSBs maximized recombinogenic                       601-7D appear decreased;  result potential attenuated expression HO galactose                     reduced growth rate strain solid media (Fig. 2B). <br /> promoter. DSBs lethal rad52-1 mutants (15, 40,                        cdcl-100 mutation initially isolated strain 543, <br /> 59), examined plating efficiency GAL::HO radS2A                     harbored GALJ,10::HO gene pRS314. pos- <br /> (Table 3) mutant strain media containing galactose.                       sibility mutations HO endonuclease responsi- <br /> expected, radS2A strain inviable grown                          ble galactose-uninducible recombination phenotype <br /> YEPGal medium arrested large-budded cells (18)                      eliminated ways.  mutant derivative result inability repair HO-induced DSBs ura3                       strain 543 cured pRS314 (GALl,10::HO), retrans- MA4T (40, 59). YEPDGal, rad52A mutant strain                      formed, retain cdcl-100 phenotype. Second, <br /> exhibited 70% viability  Table 3) produced                    cdcl-100 mutation, introduced genetic crosses Ura+ recombinants. contrast, wild-type strains 543                    strain 619 possessed integrated copy GALJ,10::HO, <br /> 619 showed 100% viability increased Ura+ recombina-                      retained DSB-stimulated recombination-defective <br /> tion frequency incubated YEPDGal  .                           phenotype. Finally, HO endonuclease activity assayed  rad52A mutation served control viability                measuring DSB formation duplicated ura3 locus. Mo- challenged HO-induced DSBs assay                        lecular analysis genomic DNA isolated cdcl-100 plating efficiency rad52A mutant strain YEPD-                     mutant cells revealed galactose-dependent DSB Gal suggests 30% cells experienced DSB                         ura3 duplication (18). <br /> completing cell division. finding consistent                    determine frequency Ura+ recombinants <br /> reduced growth rate rad52A colonies YEPDGal (data                         arise cdcl-100 strain, cells grown liquid culture shown). results suggest GALJ,10 promoter                     assayed. absence galactose, number Ura+ partially active reglA cells grown                      recombinants viable cell cdcl-100 cultures presence glucose galactose fraction                    fivefold greater wild-type cultures (Table 2).  <br /> cells population experience HO-induced DSB.                    wild-type cells exhibited 170-fold increase Ura+ recombi- <br /> continual generation DSBs likely growth inhibitory                  nants following galactose treatment. contrast, HO-stimu- recombinational repair mutants, chose investigate                      lated Ura+ recombination eliminated cdcl-100 <br /> attenuated GALJ,10 promoter activity, stimulated recom-                   mutation, Ura+ recombination frequency following <br />  VOL. 14, 1994                                                                                      CDC1 GENE S. CEREVISIAE           8043 <br />  <br />                                                                       B. <br />  <br />                                   Spontaneous Induced                                       24tC       37tC <br />  <br />                            622    -                                              622- <br />                                                         p200                                                   p200 <br />                        cdcl-100                         p220(CDCl)           cdcl-100                          p220(CDC1) <br />                                                         p221 (Cdc1A)                                           p221 (cdclA) <br />  <br />                            622    -                                              622    - <br />                                                          p200                                                  p200 <br />                          cdcl-1 [                        p220(CDCI)            cdcl-1   [                      p220(CDCl) <br />                                                          p221(cdclA)                                           p221 (cdc 1A) <br />  <br />  <br />                             622   -                                              622    - <br />                                                          p200                                                  p200 <br />                          cdcl-2 [                        p220(CDC1)           cdc 1-2 [                        p220(CDC1) <br />                                                          p221 (cdclA)                                          p221 (cdclf) <br />  <br />    FIG. 2.   HO-induced Ura+ recombination assayed solid media. Cells patched SC-LH medium replica plated SC-H <br /> (2% glucose) SC-HDG (1% glucose galactose) media grown overnight. Cells replicated uracil omission medium, Ura+ papillae scored photographed following overnight growth. Cells panels (cdcl-100) incubated 2 days better demonstrate partial complementation p221 cdcl-100 recombination defect. (B) Temperature-dependent cell growth. Cells <br /> patched SC-LH medium replica plated SC medium incubated 24&#194;&#176;C (permissive cdcl) 37&#194;&#176;C (nonpermissive cdcl). Strains: 622, wild type; 601-7D, cdcl-100; 648-1A, cdcl-1; 631-4A, cdcl-2. Plasmids: p200, control plasmid, TRPI; p220, BglII fragment <br /> containing CDC1; p221, BglII fragment BamHI fragment deleted, removing 130 internal amino acids Cdcl. <br />  <br />  <br /> galactose treatment (5.6 X 10-&apos;) significantly different         measured. Wild-type cells exhibited hyperbolic kinetics, uninduced value.                                                reaching maximum 50% mating-type interconversion <br />    spontaneous Ura+ recombination rates wild-type                1 h galactose treatment (Fig. 3B). contrast, cdcl-100 strains roughly equivalent,             cdcl-100 mutation adversely affected rate mating-type <br /> 5.8 X 10-6 1.5 X 10-5 Ura+ cells viable cell                 switching. number switched cells increased linearly <br /> generation, respectively. represents twofold increase             5 h continuous incubation galactose reached <br /> relative wild-type value. spontaneous rate                 level 15%. mating-type interconversion recombination nontandem repeated alleles LYS2                  inhibited cdcl-100 mutation, eliminated, gene (lacking HO cut site) flanking CANM inferred               switching increased approximately sevenfold. MAT <br /> identification canavanine-resistant colonies (38). rates          switching, involves gene conversion associated canavanine resistance viable cell generation              deletion formation (25), permissive cdcl-100 mutant <br /> wild-type cdcl-100 strains 6.8 X 10-6 2.1 X 10-6,           background consistent data presented Table 2 <br /> respectively. represents threefold decrease relative           showing gene conversion events ura3 locus occurred wild-type value. data indicate cdcl-100               frequently deletion reactions. <br /> mutation did adversely affect spontaneous recombination                 Recombination measured following treatment repeated genes, contrast dramatic affect            MMS determine nonenzymatically generated DSBs <br /> DSB-initiated recombination.                                             induced recombination differentially CDC1 cdcl-100 <br />    examine cdcl-100 mutation affected              cells. frequency Ura+ recombination wild-type rate extent Ura+ recombination, kinetics            cultures induced approximately 20-fold following 40-min <br /> HO-induced recombination measured (Fig. 3). Wild-type               treatment 0.01% MMS. Similar treatment cdcl-100 <br /> cultures displayed rapid, hyperbolic response galac-           cells failed induce Ura+ recombination (data shown). <br /> tose treatment, reaching nearly 70% potential maximum            finding indicates recombinational repair defect 15 min. HO-generated DSB repair occurs 1              cdcl-100 mutation specific HO-generated DSBs 2 h (10, 42, 53, 61), recombinogenic DSBs           extends notion cdcl-100 primarily affects non- <br /> brief galactose treatment likely repaired plating.           spontaneous recombination. <br /> cdcl-100 cultures showed negligible increase Ura+ cells               cdcl-100 mutation affected quality HO- 2 h. cdcl-100 mutation,  defective           stimulated recombinants cleavage site recombination blocked events normally occur                substrate. wild-type cells grown absence combinational repair HO-induced DSBs              galactose, roughly equal numbers Ura+ recombinants rate extent Ura+ recombination affected.                white red. level white Ura+ recombinants increased <br />   determine CDC1 function specific HO-gener-               times level red Ura+ recombinants ated breaks ura3, HO-induced mating-type interconversion             cells grown liquid medium containing glucose  8044      HALBROOK HOEKSTRA                                                                                            MOL. CELL. BIOL. <br />  <br />                                                                          repaired SCE relative homologous recombination. Chro- <br />                                                                            mosomal repair equal SCE mechanism                                                                            chromatids received DSB recombina- <br />                                                                            tionally silent assay (Fig. 1).  gene conversion <br />                                                                            associated reciprocal exchange misaligned <br />                                                                            sister chromatids result triplication ura3 locus <br />          0 <br />  <br />  <br />          CD                                                                URA3 gene restored (Fig. 1). Analysis                                                                            URA3 locus Southern blotting 20 selected 20 <br />                                                                            unselected red Ura&apos; CDC1 cdcl-100 recombinants,               D <br />                                                                            HO induced uninduced, failed identify triplication                      20 <br />                                                                            ura3 gene indicative unequal SCE (data shown). <br />                                                                             SCE occurred 5% duplication. <br />                                                                               Evidence unequal SCE sought scoring <br />                                                                            red/white half-sectored colonies. Half-sectored red/white col- <br />                                         Time (min) <br />                                                                            onies generated G2 unequal SCE                                                                            intrachromosomal recombination reactions occur inde- <br />                                                                            pendently homolog. frequencies half-sectored <br />                                                                            colonies wild-type cdcl-100 strains following galac- <br />                                                                            tose treatment approximately 2 0.5%, <br />                                                                            respectively. data indicate unequal SCE ac- <br /> B                                                                          count minority recombinants, assay <br />                     100                                                    conditions, DNA exchange properly aligned sister <br />                                                                            chromatids responsible efficient recombinationally <br />                                                                            silent repair. <br />                                                                               address issue equal SCE, examined               C) <br />                                                                            behavior (x-factor-arrested cdcl-100 mutant cells               C) <br />                                                                            challenged galactose. experiments, wild-type <br />               CD <br />                                                                            ao-factor-arrested cells maintained greater 95% viability               01)                                                          absence galactose. presence galactose               0-                                                           (x-factor-induced arrest, viability varied 64 90%. cdcl- <br />                                                                            100 cells somewhat sensitive -factor arrest, exhibiting <br />                                                                            57 88% viability.  presence factor <br />                                                                            galactose, viability remained high (66 88%) cdcl-100 <br />                           0   1     2        3       4      5      6       cultures. conditions, wild-type Ura&apos; recombina- <br />                                         Time (hrs)                         tion induced average 75-fold. Recombination                                                                            cdcl-100 strain uninduced galactose treatment    FIG. 3. Kinetics HO-induced recombinant formation wild-           remained threefold lower wild-type cells, type cdcl-100 strains. Galactose (2% final concentration) added log-phase cultures strains 622 601-1B (cdcl-100)           10-3 Ura&apos; recombinants cell.  ax-factor-arrested <br /> grown YEPD(0.5). Aliquots removed indicated times, <br />                                                                            cdcl-100 cells able survive 1-h incubation galac- <br /> diluted, plated uracil omission medium score Ura&apos; colonies       tose, indicating equal SCE occurring significant SC medium determine Ura&apos; recombination frequency  .            extent DSB repair cdcl-100 wild-type cells Colonies developed SC medium tested mating           ura3 duplication locus. consistent observation <br /> type (B). fraction cells switched represented number   log-phase cdcl-100 cultures maintain high viability ax colonies plus number nonmating colonies divided total     treated galactose, G1 cells killed SCE <br /> number colonies (ax + + nonmating). Open circles, strain 622;         responsible high-viability DSB repair. <br /> closed circles, strain 601-1B (cdcl-100).                                     Efficient repair recombinogenic lesions cdcl-100 mu- <br />                                                                            tants. identified level HO endonuclease expression <br />                                                                            threshold efficient killing known <br />                                                                            DSB repair-deficient strain, radS2A  , galactose (Table 2). data similar Nick-              capable providing recombinogenic lesions oloff et al. (42) differ results Ray et al. (45),          wild-type population enable visual detection large conversions associated deletions occurred              stimulation recombination (Fig. 2A). cdcl-100 <br /> infrequently. contrast, ratio red white Ura&apos; cells            mutation differentially affected DSB recombination repair, independent galactose treatment remained                    cell survival following treatment galactose  induce stant 7 1 strain 601-1A (cdcl-100) cultures.            expression HO) DSB-inducing agents exam- ratio Ura+ gene conversion deletion events affected           ined. cdcl-100 mutation altered following HO-                     Plating efficiencies determined wild-type, cdcl-100, <br /> induced DSB repair.                                                        rad52A strains grown media containing glucose, <br />   SCE events responsible efficient dsb repair                   galactose, (Table 3). rad52 mutants unable <br /> cdcl-100 mutants? SCE reactions damaged intact                 repair DSB, included rad52A strain analysis chromatids repair mechanism account               control DSB induction. Wild-type cells grew readily high viability absence increased Ura&apos; recombination              medium containing glucose galactose (YEPDGal). <br /> frequency observed cdcl-100 mutant. Repair ionizing              Viability YEPGal 92%. Viability cdcl-100 cells <br /> DNA damage G2 diploid cells efficient (6, 7). Kadyk             plated YEPDG medium comparable wild- Hartwell (33) demonstrated recombination-                    type cells (97%). cdcl-100 rad52A mutant cells inducing lesions incurred G2 diploids preferentially                capable colony formation YEPGal medium (61 versus <br />  VOL. 14, 1994                                                                                           CDC1 GENE S. CEREVISIE              8045 <br />  <br />        TABLE 3. Viability solid media containing glucose                        observed DSBs induced galactose, cdcl-100 <br />                         galactose                                           mutant cells grew slowly increased frequency                                                                                    aberrant cellular morphology grown continuously        Strain             Carbon source         nb       Avg colony formation      medium containing MMS;  sensitivity HO                                                                &#194;&#177;SD   (%)C <br />                                                                                    MMS approached rad52A level. <br /> 622 (wild type)          Glc                    920               100                 repair capacity cdcl-100 mutants examined <br />                          Glc + Gal              904             98 &#194;&#177; 2             following single exposures DNA-damaging agents (Fig. 4). <br />                          Gal                    999             92 &#194;&#177; 7             Survival following y irradiation marginally 601-1B (cdcl-100)         Glc                  1,331              100              cdcl-100 wild-type cells. Roughly 30% cdcl-100 <br />                           Glc + Gal            1,136            97 &#194;&#177; 2             cells resistant doses greater 20 kilorads. Wild-type <br />                           Gal                  1,040            61 &#194;&#177; 22            cultures exhibited 40% survival 20 kilorads. plateau                                                                                    resistance ionizing radiation results G2-phase cells 295-4A (radS2A)           Glc                  1,026              100              culture time irradiation (15, 16). wild- <br />                           Glc + Gal            1,109            70 &#194;&#177; 15            type cdcl-100 survival curves similar, slightly <br />                           Gal                  1,093            &lt;0.004d            increased sensitivity cdcl-100 cells represent lower <br />   Media: Glc, YEPD; GIc + Gal, YEPDG; Gal, YEPGal.                               level G2 cells time irradiation. <br />   b Total number cells examined experiments.                             cdcl-100 mutant showed intermediate sensitivity   c Normalized growth YEPD medium. Average plating efficiencies         killing single exposures MMS relative wild-type experiments 99% &#194;&#177; 1% strain 622, 89% &#194;&#177; 3% strain 601-1B;         rad52A strains. cdcl-100 mutation 85% &#194;&#177; 4% strain 295-4A. <br />   d colonies approximately 250 cells examined exper-        sensitive MMS rad52A, similar dose <br /> iment. Examination larger numbers cells spotted plate revealed   response rad52A exponential killing occurred colony formation approximately 2.5 x 104 cells spotted            elimination low-dose-resistant shoulder observed experiment.                                                                        wild-type strains. Irradiation UV light slight <br />                                                                                    effect cdcl-100. noted  radS2A mutant strains <br />                                                                                    marginally sensitive killing UV irradiation <br /> 0%). radS2A mutant cells completed                         (15, 23, 47). <br /> cell division cycles galactose media usually arrested                       Molecular cloning characterization CDC] gene. large budded cells result inability repair                   wild-type CDC1 gene isolated complementation DSBs generated ura3 AMT loci. results                             mutant recombination defect. DNA sequencing revealed indicate cdcl-100 mutant capable efficient DSB                      single open reading frame encoding predicted polypeptide <br /> repair conditions induce complete inviability                    product 491 amino acids (Fig. 5). significant similarities DSB repair-defective rad52A mutant.                                         existing proteins SwissProt, PIR, Gen- <br />    Cells similarly analyzed colony formation                        Pept, GPUpdate databases, using BLAST search <br /> dium containing MMS (Table 4). Wild-type cells exhibited                           program (3). noted Cdcl primary sequence did <br /> high-level plating efficiency concentration                         terminally located regions  unknown function) <br /> range tested (0.005 0.02%). expected, rad52A strain                      rich serine basic <span id='am-30' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-33' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> residues  lysine; Fig. <br /> experienced extensive killing, lowest drug dose.                    5). open reading frame examined M. Goebl plates 0.005% MMS, cdcl-100 cells formed colonies                        determine showed similarities proteins contained <br /> nearly wild-type efficiency (95%). intermediate drug                        database. analysis revealed gene concentration (0.01%), colony formation reduced                          identical open reading frame identified M. Paidhungat <br /> 50% decreased 5% 0.02% MMS.                               S. Garrett complement temperature-sensitive <br />                                                                                    mutation obtained cdcl strain. Using genomic grid <br />                                                                                    plate provided Linda Riles, CDC1 gene localized          TABLE 4. Viability solid media containing MMS                          chromosome IV near RAD9. Northern blot (RNA) analysis <br />                                                                                    identified RNA transcript approximately 1.4 kb, large <br />          Strain <br />        Strain              Conen (%) <br />                            Concn              ~                                                n         Avg colony formation <br />                                                                 SD (%)b            encode putative protein (data shown). <br />                                                                                       determine CDC1 mutants showed pheno- <br /> 622 (wild type)                 0            1,000                100              typic similarity cdcl-100, examined recombination <br />                                 0.005          928             100 &#194;&#177;0              phenotype mutant alleles CDC1 (21). Strains 369 (cdcl-l) <br />                                 0.01           964              99 &#194;&#177; 0             342 (cdcl-2) backcrossed strain 622 create <br />                                 0.02           876              98 &#194;&#177; 1             strains 648-1A (cdcl-l) 631-4A (cdcl-2).  <br /> 601-1B (cdcl-100)               0            1,080                100              ditions assay, cdcl-2 allele cdcl-1 allele <br />                                 0.005        1,152              95 &#194;&#177; 5             exhibited galactose-inducible recombination (Fig. 2A). cdel-] <br />                                 0.01         1,040              56 &#194;&#177; 20            exhibited 5-fold increase HO-induced Ura+ recom- <br />                                 0.02         1,441               5&#194;&#177;2               bination frequency  2 h), strain cdcl-2 showed                                                                                    50-fold increase (18). cdcl-2 mutation 295-4A (rad52A)                 0            1,046                100              proficient stimulation recombination following                                 0.005        1,400            0.02 &#194;&#177; 0.02          DSB, extent fold increase obtained <br />                                 0.01         1,400              &lt;0.005c            wild-type cells (Fig. 2A). <br />                                 0.02         1,400              &lt;0.005c               cloned CDC1 gene capable restoring HO- <br />       Total number cells examined experiments.                          induced Ura+ recombination proficiency cdcl-100   b <br />     Normalized drug control. average plating efficiencies      cdcl-1 strains (Fig. 2A). plasmid copy CDC1 gene <br /> experiments 100% &#194;&#177; 0% strain 622, 87% &#194;&#177; 3% strain 601-1B,         reversed slow-growth defect cdcl-100 84% &#194;&#177; 2% strain 295-4A. <br />   c example, 1,400 cells counted, positive.        cdcl-1 cells grown 24&#194;&#176;C solid media temperature-  examination larger numbers cells spotted plate          sensitive growth (37&#194;&#176;C) phenotype cdcl-1 cdcl-2 (Fig. <br /> revealed colony formation average frequency approximately 1 2 X 104.   2B). Surprisingly, frame deletion allele CDC1 (initially <br />  8046       HALBROOK HOEKSTRA                                                                                                 MOL. CELL. BIOL. <br />  <br />                              y - irradiation                                  MMS                                      UV - irradiation <br />           100 <br />  <br />  <br />  <br />  <br />            101 <br />     9 <br />  <br />  <br />    U)2. <br />    ux <br />    (L <br />    0. <br />  <br />  <br />            1 <br />  <br />  <br />  <br />  <br />                                  Krad                                     percent (v / v)                                   J / m2 <br />   FIG. 4. DNA damage dose-response curves. Aliquots log-phase cultures irradiated -y radiation UV light plating solid medium treated directly MMS liquid culture 1 h plating drug-free medium. case, percent survival calculated following microscopic determination viable nonviable cells. Circles, strain 622; squares, strain 601-1B (cdcl-100); triangles, <br /> strain 295-4A (rad52::TRPI). <br />  <br />  <br /> obtained intermediate molecule construction               100 cdcl-100/cdcl-1  cdcl-2) diploids exhibited poor <br /> deletion/insertion mutation; Materials Methods) par-                sporulation frequencies spore viabilities 3% <br /> tially restored HO-stimulated recombination cdcl-100                 allele combination. cdcl-100 mutation identified <br /> cdcl-1 strains (despite missing 130 internal amino acid resi-               impact mitotic DSB recombinational DNA repair dues) capable complementing temperature                  adversely affects meiosis, phenotype dominant sensitivity cdcl-1 cdcl-2 strains. finding                cdcl-1 cdcl-2 mutations. <br /> suggests amino carboxy termini Cdcl                    investigate properties severe mutations CDC1, <br /> form domain influences DSB-induced recombination                     inserted URA3 gene flanked E. coli hisG DNA (2) temperature-sensitive mutations reside                  130-amino-acid deletion allele used replace <br /> internal domain.                                                            copy CDC1 diploid strain 488. Tetrad analysis    demonstrate cloned gene CDC1, mated                   Ura+ transformants showed 2+:2- segregation viability. cdcl-l mutant strain tagged CDC1 strain constructed                41 44 tetrads, exhibited viable Ura+ segregants (82 integration plasmid containing cloned CDC1 gene                 82 Ura-), demonstrating CDC1 essential gene  TRP1 gene. Tetrad analysis sporulated                        aberrant tetrads, showed viable spores, <br /> CDC1::CDC1 TRPJ/cdcl-1 diploid revealed 119                     Ura- Ura+). <br /> viable spored asci  132 total tetrads dissected)                Microscopic examination dissected spores 3 days <br /> demonstrated parental ditype segregation pattern.               showed unbudded small-budded single cells. complete coincidence 2+:2- growth 37&#194;&#176;C 2+:2- tryptophan prototrophy 24&#194;&#176;C, indicating cloned gene CDC1 tightly linked. Unfor-                                             DISCUSSION <br /> tunately, poor sporulation heteroallelic diploid combi- <br /> nations cdcl-100 mutation impossible                       Recombinational DSB repair defect cdcl-100 muta- <br /> observe segregation cdcl-100 cdcl-l cdcl-2               tion. intramolecular recombination assay used alleles. cloned gene integrated                    isolate DSB-stimulated recombination-defective mutants S. <br /> cdcl-100 strain complemented mutant defect, strain                 cerevisiae.  cdcl-100, characterized (cdcl-100::CDCJ TRP1) sporulated poorly mated                     deficient DSB-stimulated recombination cdcl mutant strains, suggesting dosage effect             duplicated ura3 alleles. defect associated CDC1 sporulation.                                                     increased cell lethality, indicating DSBs effectively <br />    Sporulation spore viability affected ho-                repaired. absence DSB-induced killing cdcl-1 <br /> mozygous cdcl-100 diploids heteroallelic cdcl-1  cdcl-               cdcl-100 mutations suggests manner 2)/cdcl-100 diploids (Table 5). cdcl-l/cdcl-l ho-                   recombinational DNA DSB repair normally carried mozygous diploids sporulated high efficiency (55)                  altered eliminating repair occurs RADS2A <br />  exhibited spore viability greater 75%, cdcl-100/cdcl-              mutants. level repair proficiency suggests  VOL. 14, 1994                                                                                                                      CDCI GENE S. CEREVISME                                 8047 <br />  <br />                                                                                  =   AC w -GC TCAAAACTCGCGGG&apos; AGA TT <br /> -24 0                                                   GGr-cnTi-AATo-TGCTAC7M- <br />          TIAAAAATACCCA TTG- AACTl^CTCATATScTTTkAArcTAGTAUC  <br /> -1 20     TT=A7-rAAAATAcTAcAcAcGATT7,GAACAACkAATAAC CAAAGkGAAACAG-TACCGCTCTACTA ATIC7&apos;lr;ATA CATTAGAC AATAAGCI1CTAGZACGT <br />     1    K        V   Y   R   U   R   SL&amp;     S   V   L   S   T   H   S    K       K   S   D    D   K       H   Y    K   S      S     K       K   K    S   K    S   R   S       K    R    L <br />     1     GTATAMGCAACACAAAGAGCGTTATCTACACACAGCAAGAAGAGCGACTATA.AGGCCCACTATAAA                                                   &apos;ACG                 TCAAAAAGCAGGACkAGAAGATA <br />   41      R  Y W R Y S V W L W L 0 L S Y Y E S V V V K R M K K C 0 W S T W E D <br />  12 1     AGAATCTAC7GAGATACA TATCTATTSTATGA7ATITAGGATTAATAT&apos;-ATrA7&apos;AATC AG1TTGTlrAGAG AGCAAr-AAGAA-v,CCAAI&apos; H,;WC ACTAGGArT <br />   81      W       P   E   0     K   S   H   R   V   0   L   F     D    P       Q     N    D   E    Y    S    Y P G R P Q V H Y F T R V r v D <br />  24 1     TGCCTAAGGGTAAAGMATAGGroG-ATGC^AGACC CAC AAATAAIACGAATAC                                                 ATA TCC7GAGGCCTCAAATAGTTAArTAr=lAC AAGAGTIArT7,AC <br />  <br />  121      H       Y   H     R   U   W   K   Y   V   Q   Y   Y   L   D    P       D   S   W    F   F    L    C    D   L   F   D    0     0       R   U    W   D    D   K   Q   W      K    E <br />  3 61     CATTAMACAGAGAM7ATAMTCCAGTATTAC~rGATCCT                                       ATAGCAIrTCCTGGGA TCTATr rAGAAAGAC                                         TA TAA GCAAlr,,ATNAGAC <br />  161      Y       T   R   F   S   Q     F   P   K   K   P   L   R   R    T       V   X   8    L   P    0    S   H    D     0    F     0       D   T    V   V    E   8   8   L   Q    R    F <br />  4 81     TACACACGA~rAACCAAATATCCCCACAGCCACNrAGAAGAACGG rA&apos;LALN-L -ll-ACCA0G TAAT1 ACGATA7_Ii&apos;-GAGACACAGIrTCrAGAGT-ACAGAGATT <br />  201      8 8 Y F 0 E T S 8 S L D 0 W H T F V L L D S 8 L S D K S U P U V S R V P R Q <br />  6 01     TCTAGTTAC &apos;GAGAAACCMCTCCTCArrAGACGCTC-AATCACTACCTTMTTCAGATACAAT-rC&apos;MTGATAAAAC AAAWCGAATIGTCTCGAGGGICCCTAGACAG <br />  241      r  L D X F K 0 S H P L P R L L T H V P L W R D P K Q Q T C 0 Q L R E S K E P <br />  &apos;7 21    IiGrATAAlolCAAAl oGTACATCCGC7CCTAGAACTTTC&apos;&apos;&apos;-AACTCACGnCC CAIrATGGAGG;GATlCCAGAACAG&apos;CAAACGlIGCGG TCAGCTACGGATIC CAAGAC CT <br />  281      T       P     Q   K   C   H   Q   Y   Q   T   V     E   Y    D         S    Q   E      L    T    K     0   P    K           L   r    S   0    D   D   H   D   H    C    Q <br />  84 1     lTMCTATTAGAAAGGGCACCAGTACCAGACAGTTATAGAAACGATATATCACAAGAAATnTAACGAAAATTCAACCAGAAATATTATICGGGGATGATCAMACCAWGCAG <br />  321      S H 8 Y P F Q 0 K T K                             U     Q    K         T    V   K    S   C       M   S   K   0        S       R   P         0   L   L   S   L    Y    H <br />  961l     ATA7CACACTATACGCrC GGGAACTAAAA                      rCCAAGAAATTACAGTAAAGTCAM7TGAATr.ACA7WCTCAGTACGCCIGTATrkAlrCTCTICCTTATATAA T <br />  361          P   8   D   L   T   U   V   U     0   0       Y     S    K       T   Y    Q   T    E   L     C Y     K   P   D    P     Y       K        R    X   Y   L   W   0    L    L <br /> 10 81     CCTTCGA7rACAAGTITAAGCCGGCGGAGAGTATACTT                          -AAAACCAAACAGAGCTAlE;              CTATA     TGCCAGATICCCTATAAGGCAATCAGAATIGTACCTAlE;GGGATTATTA <br />  401          F S F Y U H F F            P K S F S Y R V T K U R V F T R P D 0 U T S D L L <br /> 12 01         7 XIX:CLGC                          CCAAAAlI~lWAATkATICGCGTAGCAACCATAA~tAACAGAGTATTTAC ACGACC TGA~r.;GTAA CACC^C AGA TIrGCATTA <br />                          CATCGCATACAlrf;CA71r&apos;l-ss- s <br />  441          P T 8 BK S K S K BJL       K     L T H S K Y V U D T R s iK Q r L V N V L F G S V <br /> 1 3 21        CCTACAAGTACCAAATCGAAATCCAAAAAATCA7&apos;1GACACATAGTA.AATACGCM7GTAAlfATACGCGTTCGATCAAACAA TTETAGTTAA l;C AAT=TCC7&apos;r-r7GlMC m;G <br />  481          U   P     F     Y   F   Y   T   V   V   - <br /> 14 41         ASGCCCATMrrATTbA~mAATAC7&apos;GTAGTICTAGACAGAATTACTS;CTAT7TCTC7TCCAGGATCAA.AAT&lt;;CTIGrAAGGGAAACG&apos;CT&apos;iC;TATTAATATATIGCGAATACACCTTAT <br /> 1 5 61        ATACAGACAAsxACAAGAGATATAAATTATACCGOGG7GGT~ATTACTATGCTICACCAAACAGI&apos;rrl.CATAAC C&apos;-TACTGGArTC ATATAGl TllE;GTCCkAGC TAAAAC 1GC l168B1        GGACIX;GCGTAACACCAATTACT~rCACTATACoGGCGACAAAAAAGAACACCTAGGTAAGlEIlwloGTTCTTTAG -MrrllATA7TCAAATTTAGCA.AGTCATIGACGGCAAAlC7TAST <br />  <br />  <br />  <br />  <br />     B <br />                                                                                                                                                                    N-   _               N- <br />          (.                                                                            t3                                                                               N-O r <br />                   LnCo <br />                                                                                                              , <br />  <br />  <br />  <br />                                                                                        __ <br />  <br />                                                                                 <br />  <br />  <br />                                                                             B <br />  <br />  <br />  <br />                                               hisG                        URA3                                hisG <br />    FIG. 5.   Nucleotide inferred amino acid sequences CDC1 gene. Primary sequences rich serine, lysine, arginine underlined. (B) CDCI gene structure. drawing represents wild-type gene subcloned p220. middle drawing shows frame <br /> deletion constructed removing 390-nucleotide internal BamHI fragment (p221). drawing shows cdcl::URA3 null allele used replace CDCI gene (p222). Triangles, sites TnlOLUK insertions used locate CDC1 open reading frame; dashed lines, DNA <br /> deletions; unbroken lines, DNA insertions. Restriction enzymes: B, BamHI; E, EcoRV; M, MluI; P, PstI; R, EcoRI; S, Sail; X, XbaI;  XhoI. <br />  <br />  recombinational repair defect cdel mutants does                                        mutations DSB repair partial loss function lead formation unrepairable reaction intermedi-                                       differs significantly dramatic affect rad52 null <br /> ates products likely occurs radS2 cells (56, 61).                                    mutation.  finding cdcl-100 mutation <br /> CDCJ essential gene, likely affect cdcl                                     alters spectrum recombinants affecting cell <br />  8048       HALBROOK HOEKSTRA                                                                                                 MOL. CELL. BIOL. <br />  <br />            TABLE 5. Sporulation defect cdcl mutations                      rad52-like sensitivity cdcl-100 cells DSBs                                                                               absence homologous sequences provide additional <br />   Relevant genotype                                                           evidence support homology-dependent recombinational <br />    parental strain        spored           Total            Spore     repair DNA DSBs. <br />                                  ascia          sporulation&apos;      viability      Repair damage induced MMS -y irradiation. CDC1 x CDC1                       45                63               96       cdcl-100 mutation somewhat unusual DNA repair <br /> CDC1 X cdcl-100                   49                55               90       characteristics, particularly sensitive large single- <br /> CDC1 x cdcl-1                     22                53               80       dose exposures -y irradiation, continuous expression CDC1 X cdcl-2                     26                51               83       HO endonuclease, sensitive MMS treatment. <br /> cdcl-100 X cdcl-100                8                18                2        phenotype unprecedented, cdcl-100 X cdcl-1                  0                 2              NDc       genes mediate repair alkylation damage S. <br /> cdcl-100 X cdcl-2                  3                17                3       cerevisiae (32, 58) DSBs Schizosaccharomyces pombe <br /> cdcl-1 X cdcl-1                   30                52               75       (12) MMS sensitivity refractory killing cdcl-1 X cdcl-2                   53                60               98 <br /> cdcl-2 X cdcl-2                   64                79               88       ionizing radiation. differential -y-irradiation MMS <br />                                                                               sensitivity nature DSBs resulting   &quot; Calculated following examination 400 cells. Sporulation   treatments. DSBs induced MMS thought carried 24&#194;&#176;C.                                                          indirect consequence incomplete excision repair alkylated <br />   &quot; Calculated analysis 22 dissected tetrads. <br />   c ND, determined.                                                       nucleotide base residues close proximity opposite <br />                                                                               DNA strands (9). DNA breaks caused ionizing radiation <br />                                                                               generate variety DNA double-strand termini (24)                                                                               result nucleophilic attack activated oxygen radi- <br /> viability indicates mutation profound effect               cals DNA direct excitation ionization DNA manner DSBs repaired.                                        components (46).  repair MMS-generated DSBs <br />    number possibilities exist explain apparent                    efficiently repaired y-ray-induced breaks dissociation DSB repair measurable recombina-                     cdcl-100 strain. Alternatively, cdcl-100 mutant tion observed cdcl-100 mutant strain. possibility                  suffer defect alkylation excision repair. ruled SCE events responsible                 absence MMS-resistant shoulder mutant strain <br /> high-viability DSB repair strains. SCE              suggests regardless lesion responsible inhibition <br /> efficient DNA repair pathway (33), appears play                   cell growth, repair capacity saturated doses major role repair DSBs ura3 gene duplication                 affect wild-type cells. <br /> substrate wild-type cdcl-100 mutant cells. Instead,                 intermediate levels HO expression continuous cdcl-100 mutation slow inhibit recombinational                 exposure low levels MMS (&lt;0.01%), DSBs probably <br /> repair process reactions normally occur               generated rate repair capacity cdcl-100 cells low rates divert substrates reaction interme-               repeated cell division cycles completed. <br /> diates competing repair pathways increased effi-                    Protracted exposure intermediate concentrations MMS <br /> ciency. Similarly, rad52 mutations slow recombi-                 (&gt;0.01%) maximal HO endonuclease expression  YEP- <br /> nation rate step reactions deleterious               Gal) result DNA damage rate approaches effects (e.g., extensive exonucleolytic DNA degradation)                  saturation level repair. High-level HO expression growth <br /> occur. Alternatively, exist point recombi-                 0.01% MMS reduces cdcl-100 plating efficiency nearly <br /> national DNA repair process blocked,                         50%;  treatment result  <br /> preventing completion recombination inhib-                 peated cutting sites: T ura3  repair <br /> iting, stimulating, reverse reactions ligation            restores HO cut site). Presumably, growth MMS initial partially processed DNA DSB. cdcl-100                medium continually creates DNA lesions, sites distrib- <br /> mutation causes inhibition entire DSB recombinational                   uted genome. Interestingly, 30% log-phase <br /> repair pathway, cells utilize alternative, hitherto               cdcl-100 mutant culture (compared 40% wild-type cell <br /> unidentified pathway. related possibility major                   cultures) withstand 40-krad exposure y rays. repair pathway blocked simultaneously creating                 data Resnick Martin (49), estimate expo- alternative repair pathway peculiar cdcl mutants               sure induce 6 12 DSBs haploid genome, normally observed wild-type cells. Finally,                  indicating radiation-induced lesions single treat- <br /> exist nonrecombinational mechanisms normally compete                     ment repaired efficiently cdcl-100 mutant.  recombinational repair pathways DNA DSB sub-                         repeated generation DSB AVT  possibly ura3) <br /> strates.                                                                      represents special situation severely affects cell <br />    Substantial evidence suggests DSB repair occurs                  growth cdcl-100 growth wild-type strains. <br /> recombinational mechanism (15, 36). large measure                           cell division cycle recombinational DNA repair. <br /> support comes observation homologous DNA                     introduced recombination assay strain, required efficient DSB repair (9, 15, 39, 48). observa-               cdcl-1 allele exhibited slow-growth HO-stimulated  <br /> tion mutations affect induced mitotic recombina-                 combination defect  permissive temperature) sim- <br /> tion cell survival following DNA damage enhances                  ilar cdcl-100 allele. cdcl-100 mutation, notion (15, 36). properties cdcl-100                 defects recessive suppressed plasmid-borne <br /> mutation imply existence nonrecombinational DNA                      copy cloned CDC1 gene. contrast, cdcl-2 allele repair process  increased recovery Ura&apos; recombinants                 HO-stimulated recombination proficient.  associated DSB repair), tested                   mutations CDC1 gene disrupt DSB recombina- mutation repair competency absence DNA                     tional DNA repair. <br /> homology. Recently, Kramer et al. (35) demonstrated                         cdcl mutants arrest nonpermissive temperature HO-induced DSBs repaired R4D52-                     heterogeneous population respect cell spindle <br /> DNA homology-independent manner, albeit low efficiency.                    pole body morphology (8, 19, 21). cdcl mutant cells incubated <br />  VOL. 14, 1994                                                                                       CDC1 GENE S. CEREVISIAE                   8049 <br />  nonpermissive temperature exhibit spectrum cell                 3. Altschul, S. F., W. Gish, W. Miller, E. W. Myers, D. J. Lipman. <br /> morphologies ranging cells single duplicated                      1990. Basic local alignment search tool. J. Mol. Biol. 215:403-410. <br /> spindle pole bodies partially elongated nuclear               4. Atkinson, D. E. 1977. Cellular energy metabolism regula- <br /> spindles. DNA replication affected, nuclear                     tion. Academic Press, New York. <br />                                                                             5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. positive <br /> division inhibited. Bud development inhibited                     selection mutants lacking orotidine-5&apos;-phosphate decarboxyl- <br /> cultures arrest unbudded small-budded cells.                   ase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br /> participation CDC1 gene product discontinuous                     Genet. 197:345-346. <br /> events cell cycle questioned, cdcl-1               6. Brunborg, G., M.  Resnick, D. H. Williamson. 1980. <br /> allele suffers temperature-sensitive defect RNA                  Cell-cycle-specific repair DNA double-strand breaks Saccha- <br /> synthesis (19, 20).                                                             romyces cerevisiae. Radiat. Res. 82:547-548. <br />    CDC1 mutations, cdcl-J, cdcl-2, cdcl-100,                   7. Brunborg, G., D. H. Williamson. 1978. relevance studied unique respect combination                    nuclear division cycle radiosensitivity yeast. Mol. Gen. Genet. mutant defects exhibit. cdcl-1 cdcl-2                  162:277-286. <br /> exhibit temperature-sensitive cell cycle arrest (cdcl-100 does               8. Byers, B., L. Goetsch. 1974. Duplication spindle plaques <br />                                                                                 integration yeast cell cycle. Cold Spring Harbor Symp. , cdcl-J cdcl-100 recombinational repair                       Quant. Biol. 38:123-131. <br /> defective. deletion allele CDC1, lacking 130 internal                  9. Chlebowicz, E., W. J. Jachymczyk. 1979. Repair MMS- <br /> amino acids, partially suppress recombination defect                 induced DNA double-strand breaks haploid cells Saccharo- cdcl-100 cdcl-J mutations temperature-                      myces cerevisiae, requires presence duplicate <br /> sensitive growth arrest phenotype cdcl-J cdcl-2 muta-                    genome. Mol. Gen. Genet. 167:279-286. <br /> tions. finding suggests elements amino             10. Connolly, B., C.  White, J. E. Haber. 1988. Physical moni- <br /> carboxy termini responsible recombinational DSB                     toring mating type switching Saccharomyces cerevisiae. Mol. <br /> repair property CDC1 gene product cdcl-1                    Cell. Biol. 8:2342-2349. <br /> cdcl-100 mutations reside regions. terminal               11. Davies, P. J., W. E. Evans, J. M. Parry. 1975. Mitotic <br />                                                                                 recombination induced chemical physical agents amino acid sequences rich basic residues                       yeast Saccharomyces cerevisiae. Mutat. Res. 29:301-334. <br /> support interactions nucleic acids. cdcl-1 mu-              12. Dhillon, N., M. F. Hoekstra. 1994. Characterization tation temperature-sensitive cell cycle defect                 protein kinases Schizosaccharomyces pombe involved recombinational DNA repair defect, analysis additional                       regulation DNA repair. EMBO J. 13:2777-2788. <br /> cdel mutants required conclusions                    13. Elble, R. 1992. simple efficient procedure transformation <br /> reached regarding possible roles CDC1 functional                         yeasts. BioTechniques 13:18-20. <br /> domains.                                                                   14. Fishman-Lobell, J., N. Rudin, J. E. Haber. 1992.    Initial characterization cdcl mutants did indicate                  alternative pathways double-strand break repair kinet- <br /> role CDC1 gene events associated DNA                            ically separable independently modulated. Mol. Cell. Biol. <br />                                                                                 12:1292-1303. <br /> recombination.  behavior cdcl-1 allele               15. Game, J. C. 1983. Radiation-sensitive mutants repair yeast, recombination assay strain indicates CDC1 partici-                     p. 109-137. J. F. T. Spencer et al. (ed.), Yeast genetics: <br /> pates aspect DNA metabolism, recombina-                      fundamental applied aspects. Springer-Verlag, Berlin. <br /> tional repair DSBs. Byers Goetsch (8) observed                 16. Game, J. C., R. K. Mortimer. 1974. genetic study X-ray <br /> cdcl mutants exhibit heterogeneous arrest respect                     sensitive mutants yeast. Mutat. Res. 24:281-292. <br /> spindle pole body spindle morphology. CDC1 partici-                 17. Gietz, R. D.,  Sugino. 1988. New yeast-Eschenichia coli <br /> pates aspect microtubule metabolism nuclear                       shuttle vectors constructed vitro mutagenized yeast genes <br /> division, influence DSB-induced recombination                    lacking base pair restriction sites. Gene 74:527-534. <br /> repeated sequences direct indirect conse-               18. Halbrook, J., M. Hoekstra. Unpublished data. <br />                                                                            19. Hartwell, L. H. 1971. Genetic control cell division cycle quence disrupting DNA metabolism. study                     yeast. IV. Genes controlling bud emergence cytokinesis. Exp. <br /> mutations addition cdcl (20) DSB-stimulated                     Cell Res. 69:265-276. <br /> recombination-defective mutations isolated (18)               20. Hartwell, L. H., J. Culotti, J. R. Pringle, B. J. Reid. 1974. provide better understanding recom-                     Genetic control cell division cycle yeast. Science 183:46- <br /> binational DNA repair processes integrated                         51. <br /> execution cell division cycle.                                      21. Hartwell, L. H., J. Culotti, B. Reid. 1970. Genetic control                                                                                 cell-division cycle yeast.  Detection mutants. Proc. Natl. <br />                       ACKNOWLEDGMENTS                                           Acad. Sci. USA 66:352-359. <br />                                                                            22. Hartwell, L. H., R. K Mortimer, J. Culotti, M. Culotti. 1973. <br />    appreciate critical discussions Namrita Dhillon              Genetic control cell division cycle yeast. V. Genetic <br /> help constructing plasmids used study. appreciate            analysis cdc mutants. Genetics 74:267-286. <br /> helpful comments provided Jac Nickoloff preparation       23. Haynes, R. H., B.  Kunz. 1981. DNA repair mutagenesis <br /> manuscript. thank Christi Wood providing DNA sequenc-                yeast, p. 371-414. J. N. Strathern, E. W. Jones, J. R. <br /> ing acumen Mark Goebl analyzing cloned DNA sequence                 Broach (ed.), molecular biology yeast Saccharomyces: database. appreciate S. Garrett sharing             life cycle inheritance. Cold Spring Harbor Laboratory, Cold <br /> results.                                                                        Spring Harbor, N.Y. <br />    work partly supported Damon Runyon-Walter Winchell          24. Henner, W. D., S. M. Grunberger, W.  Haseltine. 1982. Sites <br /> Cancer Research Fund fellowship DRG-1077 J.H.                                structure y radiation-induced DNA strand breaks. J. Biol. <br />                                                                                 Chem. 257:11750-15754. <br />                              REFERENCES                                    25. Herskowitz, , J. Rine, J. N. Strathern. 1992. Mating-type <br />  1. Aguilera, , H. L. Klein. 1988. Genetic control intrachro-          determination mating-type interconversion Saccharomyces <br />     mosomal recombination Saccharomyces cerevisiae.  Isolation             cerevisiae, p. 583-656. E. W. Jones, J. R. Pringle, J. R. <br />     genetic characterization hyper-recombination mutations.              Broach (ed.), molecular cellular biology yeast <br />     Genetics 119:779-790.                                                       Saccharomyces. Cold Spring Harbor Laboratory Press, Cold <br />  2. Alani, E., L Cao, N. Kleckner. 1987. method gene                  Spring Harbor, N.Y. <br />     disruption allows repeated use URA3 selection       26. Ho, K. S. Y. 1975. Induction DNA double-strand breaks     struction multiply disrupted yeast strains. Genetics 116:541-545.        X-rays radiosensitive strain yeast Saccharomyces <br />  8050      HALBROOK HOEKSTRA                                                                                                     MOL. CELL. BIOL. <br />  <br />     cerevisiae. Mutat. Res. 33:157-164.                                           nation yeast, p. 407-522. J. R. Broach, E. W. Jones, J. R. <br /> 27. Ho, K. S. Y., R. K. Mortimer. 1973. Induction dominant                 Pringle (ed.), molecular cellular biology yeast <br />     lethality X-rays radiosensitive strain yeast. Mutat. Res.          Saccharomyces. Cold Spring Harbor Laboratory Press, Cold <br />     20:45-51.                                                                     Spring Harbor, N.Y. <br /> 28. Hoekstra, M. Unpublished data.                                          45.   Ray, ,  Siddiqi,  L. Kolodkin, F. W. Stahl. 1988. <br /> 29. Hovland, P., J. Flick, M. Johnston, R.  Sclafani. 1989.                 Intra-chromosomal gene conversion induced DNA double- <br />     Galactose gratuitous inducer GAL gene expression                   strand break Saccharomyces cerevisiae. J. Mol. Biol. 201:247- <br />     yeasts growing glucose. Gene 83:57-64.                                     260. <br /> 30. Huisman, O., W. Raymond, K. U. Froehlich, P. Errada, N.                 46.   Repine, J. E., 0. W. Pfenninger, D. W. Talmage, E. M. Berger,     Kleckner, D. Botstein, M.  Hoyt. 1987. TnlO-lacZ-                     D. E. Pettijohn. 1981. Dimethyl sulfoxide prevents DNA nicking <br />     kanR-URA3 gene fusion transposon insertion mutagenesis                    mediated ionizing radiation <span id='am-37' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span>iron</span>/hydrogen peroxide-gener- <br />     fusion analysis yeas <br /> </body></html>