<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Molecular Strategy Survival Critical High <br /> Temperature Eschierichia coli <br /> Masayuki Murata1., Hiroko Fujimoto1., Kaori Nishimura1, Kannikar Charoensuk1, Hiroshi Nagamitsu1, <br /> Satish Raina2, Tomoyuki Kosaka3, Taku Oshima4, Naotake Ogasawara4, Mamoru Yamada1,3* <br /> 1 Applied Molecular Bioscience, Graduate School Medicine, Yamaguchi University, Ube, Japan, 2 Division Medical Biochemical Microbiology, Research Center <br /> Borstel, Borstel, Germany, 3 Department Biological Chemistry, Faculty Agriculture, Yamaguchi University, Yamaguchi, Japan, 4 Graduate School Information <br /> Science, Nara Institute Science Technology, Ikoma, Nara, Japan <br />  <br />  <br />  <br />      Abstract <br />      molecular mechanism supporting survival critical high temperature (CHT) Escherichia coli investigated. <br />      Genome-wide screening single-gene knockout library provided list genes indispensable growth 47uC, <br />      called thermotolerant genes. Genes expression affected exposure CHT identified DNA chip <br />      analysis. Unexpectedly, contents did overlap dnaJ dnaK, indicating      specific set non-heat shock genes required organism survive severe condition. half      mutants thermotolerant genes sensitive <span id='am-140' about='obo:IMR_0200487' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-141' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="H2O2" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="KEGG:C00027" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:id" content="IMR:0200487" datatype="xsd:string"></span>H2O2</span> 30uC, suggesting mechanism      thermotolerance partially overlaps oxidative stress resistance. encoded enzymes proteins related      outer membrane organization, DNA double-strand break repair, tRNA modification, protein quality control, translation <br />      control cell division. DNA chip analyses essential genes suggest genes encoding ribosomal proteins <br />      regulated CHT. Bioinformatics analysis comparison genomic information microbes <br />      suggest E. coli possesses systems survival CHT. analysis allows speculate      <span id='am-28' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-29' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipopolysaccharide</span> biosynthesis outer membrane organization sulfur-relay tRNA modification <br />      acquired horizontal gene transfer. <br />  <br />   Citation: Murata M, Fujimoto H, Nishimura K, Charoensuk K, Nagamitsu H, et al. (2011) Molecular Strategy Survival Critical High Temperature   Eschierichia coli. PLoS 6(6): e20063. doi:10.1371/journal.pone.0020063 <br />   Editor: Mick F. Tuite, University Kent, United Kingdom <br />   Received January 6, 2011; Accepted April 12, 2011; Published June 10, 2011 <br />   open-access article, free copyright, freely reproduced, distributed, transmitted, modified, built  used   lawful purpose. work available Creative Commons CC0 public domain dedication. <br />   Funding: work supported Program Promotion Basic Research Activities Innovative Biosciences Special Coordination Funds   Promoting Science Technology, Ministry Education, Culture, Sports, Science Technology. work performed collaborative research   Asian Core Program Yamaguchi University Khon Kaen University, supported Scientific Cooperation Program agreed   Japan Society Promotion Science (JSPS) National Research Council Thailand (NRCT). funders role study design, data collection <br />   analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: m-yamada@yamaguchi-u.ac.jp <br />   . authors contributed equally work. <br />  <br />  <br />  <br /> Introduction                                                                            folding, refolding, quality control degradation [6]. HSPs                                                                                         complex regulations divided regulatory <br />    Responses Escherichia coli high temperatures                         groups major stimulons [7]. control extensively investigated, previous studies mainly                           expression,  highly variable organisms focused response temperature shift 42uC,                        various bacteria [8]. <br /> response known heat shock response (HSR) induce                                control elements Gram-negative bacteria expression set proteins, heat-shock proteins (HSPs) [1].                    heat shock sigma factor regulates transcription major <br /> fact HSPs conserved species indicates                          HSPs. HSR E. coli generally mediated alternative sigma actions HSRs fundamentally physiologically                           factors, sigma 32 sigma 24 [4,7,8]. Transcription rpoH <br /> important mechanisms living organisms [2,3]. HSPs play crucial                       gene sigma 32 induced elevated temperature action <br /> roles rescue removal proteins damaged                          sigma 24 [7]. Sigma 24, inactive non-stress <br /> environmental stresses, including heat stress salt stress,                      conditions interaction anti-sigma factor, activated intrinsic folding proteins normal growth                           misfolding outer membrane periplasmic proteins conditions [4].                                                                         stresses including heat shock [9]. sigma factors    shown 384 genes regulated short-time                      regulated translation level posttranslational <br /> exposure temperature 43uC heat shock E. coli [5],                       level. factor sigma 24 regulated cognate small genes directly indirectly induced                            RNA, sigma 32 synthesis regulated structural change treatment. directly induced genes encode HSPs, including                        mRNA molecules serving cellular thermometer main cellular chaperone machineries GroEL DnaK, <span id='am-55' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-59' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-                             activity modulated phosphorylation [10,11]. microor- <br /> dependent proteases Lon, HslUV, Clp FtsH (HflB),                                 ganisms, hand, appear possess diverged regulatory <br /> periplasmic protease DegP, proteins involved protein                       mechanisms [12]. <br />  <br />  <br />        PLoS | www.plosone.org                                                   1                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                        E. coli Survival Critical High Temperature <br />  <br />  <br />    information molecular mechanisms                     expressional alteration 4 genes nearly response survival critical high temperature (CHT)              temperatures, growth corresponding mutant strains 37uC <br /> organisms, probably limited experimental procedures.               significantly changed parental strain. <br /> Developments single-gene knockout library DNA chip                   Taken  results suggest thermotolerant <br /> analysis encouraged perform genome-wide investi-                 phenotype 51 mutants disruption targeted <br /> gation responses organisms extreme conditions.              gene polar effect downstream genes. mesophilic bacteria including E. coli grow survive            51 thermotolerant genes, 8 genes, cydB, degP, dnaJ, dnaK, dnaQ, high temperatures compared mesophilic bacteria,              nlpI, rfaD rfaC, reported genes supporting growth assumed acquired potential thermotolerance                high temperature [16&#226;&#8364;&#8220;22], newly identified 43 evolution. study, utilized new procedures             thermotolerant genes organism. time obtain information molecular <br /> mechanisms related thermotolerance E. coli CHT.                      Effects supplements oxidative stress growth Screening thermosensitive mutants CHT informatics                   thermosensitive mutant strains <br /> analysis corresponding genes revealed pathways factors                 LB utilized medium screening indispensable survival CHT. essential genes,                 thermosensitive mutants, limitation carbon source cause <br /> possible involvement response CHT examined                   sensitiveness CHT. examined effect glucose DNA chip analysis. Based results, propose novel                     supplement growth thermosensitive mutant strains <br /> molecular mechanisms survival CHT E. coli.                          (Table 1, Table 2 Figure S4). tested effect                                                                               MgCl2 Mg2+ protects cell damage Results Discussion                                                        stress conditions [23,24]. growth 20 37 mutants                                                                               improved CHT addition 0.5% glucose 20 mM <br /> Thermosensitive mutants thermotolerant genes <br />                                                                               MgCl2, respectively. growth sixteen mutant strains    order identify genes required survival CHT E. coli, <br />                                                                               improved supplementation glucose MgCl2. screened thermosensitive mutants single-gene <br />                                                                                   effect exogenous oxidative stress thermo- <br /> knockout library [13], constructed according                                                                               sensitive mutant strains tested higher temperature step gene disruption method aph cassette [14]                                                                               causes oxidative stress (Noor et al, 2009; unpublished data). construct confirmed extensively [15].                                                                               exposed thermosensitive mutant strains 0.5 mM H2O2 disrupted gene mutant strain, region                                                                               LB liquid medium 30uC. mutants 1st codon 6 codons displaced aph cassette, <br />                                                                               sensitive H2O2 (Table 1, Table 2 Figure S4), coding region gene deleted.                                                                               corresponding approximately 60% thermosensitive <br /> experiments indicated parental strain used construc- <br />                                                                               mutants.  10 thermosensitive mutants tion disrupted library able grow temperatures                                                                               glucose MgCl2 supplementation effect, 9 <br /> 47uC, temperature CHT. <br />                                                                               mutants showed sensitivity H2O2. results suggest    successive screening steps library, including <br />                                                                               mechanism thermotolerance CHT partially overlaps 3,908 disrupted-mutant strains, 51 strains                                                                               oxidative stress resistance. <br /> sensitive CHT. growth curves 37uC, 45uC 46uC compared parental strain (Figure S1). growth profiles suggest mutants selected                    Bioinformatics analysis classification significantly sensitive 46uC 45uC.                 thermotolerant genes <br /> disrupted gene responsible thermosensitive phenotype                 understand molecular mechanism E. coli survival designated thermotolerant gene (Table 1 Table 2).                    CHT, bioinformatics analysis various public databases <br />    gene organization generated construction                     including KEGG PATHWAY database performed. disrupted mutants rise polar effect inserted           51 thermotolerant genes, 29 genes successfully mapped <br /> aph gene transcription downstream genes                        E. coli pathways KEGG PATHWAY database. <br /> intrinsically transcribed read promoter                Interestingly, genes involved region upstream disrupted gene. organization               metabolic pathway, suggesting organism possesses 42 51 mutants. Sensitivity               indispensable pathways CHT. remaining 19 genes polar effect 29 42 mutants disruption genes            3 unknown genes extensively analyzed using DDBJ <br /> just downstream disrupted gene method                    GenBank database. basis results analyses caused thermosensitive phenotype. remaining 13 mutants                 effects supplements, 51 thermotolerant genes essential gene thermotolerant gene                  classified 7 groups (Table 1 Table 2). <br /> immediate downstream gene (Figure S2). possible polar                      Group consists genes concerned energy metabolism <br /> effects tested RT-PCR total RNA prepared                    production ATP. gene products aceE, aceF, lpd cells exposed temperature 37uC 47uC (Figure S3).              ackA mapped pyruvate metabolism pathway results suggest transcription level immediate             pyruvate acetyl <span id='am-66' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> [25&#226;&#8364;&#8220;28] rpe located downstream gene mutant              pentose phosphate pathway. cydB yhcB encode subunits parent cases cases mutants aceF, tolQ, dnaK       cytochrome d terminal oxidase, generates membrane rpsF. downstream genes                potential responsible ATP synthesis [16,29]. lipA, promoters transcription level read                encodes LipA produce lipoate required pyruvate dehydro- nearly aph promoter.                   genase reaction, contributes pyruvate metabolism [30]. <br /> transcription levels lpd, tolR dnaJ located downstream              Based results showing disrupted mutations aceF, tolQ dnaK, respectively, increased level            genes caused thermosensitive phenotype, assumed rpsR located downstream rpsF decreased compared               cells require ATP higher temperature. assumption parental strain temperatures.                     supported finding phenotype mutants <br />  <br />  <br />        PLoS | www.plosone.org                                         2                            June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                         E. coli Survival Critical High Temperature <br />  <br />  <br />  <br />  Table 1. Thermotolerant genes identified study. <br />  <br />  <br />  Classification        Sub-classification                       Gene     Function                                                                     Glca Mg2+b H2O2c <br />  <br />  Energy metabolism Pyruvate metabolism                          aceE     pyruvate dehydrogenase, decarboxylase component E1                           ++              S <br />  (Group              Pyruvate metabolism                      aceF     pyruvate dehydrogenase, dihydrolipoyltransacetylase component E2             ++     ++ <br />                        Pyruvate metabolism                      lpd      <span id='am-71' about='obo:IMR_0200456' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-72' property="rdfs:label" content="Lipoamide" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00248" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:id" content="IMR:0200456" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipoamide</span> dehydrogenase, E3 component enzyme                ++ <br />                                                                          complexes <br />                        Pyruvate metabolism                      lipA     lipoate synthase                                                             ++     + <br />                        Propanate metabolism                     ackA     acetate kinase propionate kinase 2                                            ++       S <br />                        Pentose phosphate pathway                rpe      D-ribulose-5-phosphate 3-epimerase                                           ++              S <br />                        Respiratory chain                        cydB     cytochrome d terminal oxidase, subunit II                                    +      ++ <br />                        Respiratory chain                        yhcB     cytochrome d terminal oxidase, subunit III                                          ++       S <br />  Outer membrane        Lipopolysaccharide biosynthesis          gmhB     D,D-heptose 1,7-bisphosphate phosphatase                                            + <br />  stabilization         Lipopolysaccharide biosynthesis          lpcA     <span id='am-13' about='obo:IMR_0200141' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200141" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="D-Sedoheptulose 7-phosphate" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C05382" datatype="xsd:string"></span>D-sedoheptulose 7-phosphate</span> isomerase                                        +      ++ <br />  (Group B)             Lipopolysaccharide biosynthesis          rfaC     <span id='am-1' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span>-heptose:<span id='am-18' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-19' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="LPS" datatype="xsd:string"></span>LPS</span> heptosyl transferase                                              ++ <br />                        Lipopolysaccharide biosynthesis          rfaD     ADP-L-glycero-D-mannoheptose-6-epimerase, NAD(P)-binding                            ++ <br />                        Lipopolysaccharide biosynthesis          rfaE     fused heptose 7-phosphate kinase heptose 1-phosphate                            + <br />                                                                          adenyltransferase <br />                        Lipopolysaccharide biosynthesis          rfaF     ADP-heptose:LPS heptosyltransferase II                                              +        S <br />                        Lipopolysaccharide biosynthesis          rfaG     glucosyltransferase                                                               + <br />                        <span id='am-106' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-107' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-109' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="PGN" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-113' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span>Peptidoglycan</span>-associated lipoprotein     ydcL     predicted lipoprotein                                                        +      +        S <br />                        Peptidoglycan-associated lipoprotein     yfgL     protein assembly complex, lipoprotein component                                     ++       S <br />                        Peptidoglycan-associated lipoprotein     ynbE     predicted lipoprotein                                                        +      + <br />                        Peptidoglycan-associated lipoprotein     nlpI     conserved protein                                                            +      + <br />                        Peptidoglycan-associated lipoprotein     ycdO     conserved protein                                                            +      + <br />                        Outer membrane integrity                 pal      peptidoglycan-associated outer membrane lipoprotein                                 ++ <br />                        Outer membrane integrity                 tolQ     membrane spanning protein TolA-TolQ-TolR complex                                 ++ <br />                        Outer membrane integrity                 tolR     membrane spanning protein TolA-TolQ-TolR complex                                 ++       S <br />                        Outer membrane integrity                 yciM     conserved hypothetical protein                                                      + <br />  <br />     According data Figure S4, ratios growth presence glucose absence glucose 46uC wer estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;++&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;+&#226;&#8364;&#8482;&#226;&#8364;&#8482; represent    2.0 1.5&#226;&#8364;&#8220;2.0, respectively. <br />  b <br />    According data Figure S4, ratios growth presence MgCl2 absence MgCl2 46uC estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;++&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;+&#226;&#8364;&#8482;&#226;&#8364;&#8482; represent    2.0 1.5&#226;&#8364;&#8220;2.0, respectively. <br />  c <br />   According data Figure S4, ratios growth presence H2O2 absence H2O2 30uC estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;S&#226;&#8364;&#8482;&#226;&#8364;&#8482; represents 0.5. <br />  doi:10.1371/journal.pone.0020063.t001 <br />  <br />  group partially suppressed addition glucose                        maintain outer membrane integrity, especially high temper- <br /> (Table 1 Figure S4).                                                                 ature. Similarly, data suggest Mg2+ able stabilize    Group B consists genes related biosynthesis cell wall                    outer membrane structure LPS biosynthesis pathway organization outer membrane. products gmhB,                             defective. <br /> lpcA(gmhA), rfaC (waaC), rfaD (waaD/htrM), rfaE (gmhC), rfaF (waaF)                         Group C consists dnaQ, holC, priA, ruvA ruvC DNA rfaG (waaG) mapped lipopolysaccharide (LPS)                            double-strand break repair (DSBR) [37]. DnaQ HolC biosynthesis pathway [17,18,31,32]. products genes                          epsilon chi subunits, respectively, DNA polymerase III involved synthesis heptose unit ADP-L-glycero-D-                        [38,39], required homologous recombination manno-heptose sedoheptulose-7phosphate encode early                              DSBR [19]. RuvA RuvC act DNA helicase heptosyl transferases KDO-<span id='am-7' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span>lipid</span> (rfaC rfaF)                             endonuclease, respectively [19,40], replication restart extend inner core LPS glcosyltransferase (rfaG).                     DSBR process, PriA functions DNA helicase ydcL, yfgL (bamB), ynbE, nlpI ycdO encode peptidoglycan-                             replication restart [41]. requirement DSBR survival associated outer membrane lipoproteins, products pal,                         CHT suggests DNA molecules subjected double- <br /> tolQ tolR components complex structure forming                           strand breaks higher temperature. Interestingly, mutants biopolymer transporter [33,34]. yciM encodes protein possibly                          members group exhibited sensitivity oxidative stress required integrity outer membrane [35].                                   30uC.  thought strong connection <br /> thermosensitive phenotype caused disrupted mutants                          oxidative stress DNA double-strand breaks. genes significantly suppressed addition Mg2+                            Group D includes genes tRNA modification. Products (Table 1 Figure S4). Mg2+ known stabilize outer                      iscS, yheL (tusB), yheM (tusC), yheN (tusD) yhhP (tusA) membrane structure binding extracellularly [36], assumed                        demonstrated compose sulfur-relay [42&#226;&#8364;&#8220;44]. IscS YdcL, YfgL, YnbE, NlpI, YcdO, Pal, TolQ, TolR YciM                              widely distributed cysteine desulfurase catalyzes desulfuration <br /> act components scaffold proteins membrane                                <span id='am-50' about='obo:IMR_0200467' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasDbXref" content="KEGG:C00097" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="L-Cysteine" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0200467" datatype="xsd:string"></span>L-cysteine</span> transfer sulfur active-site cysteine  <br />  <br />        PLoS | www.plosone.org                                                    3                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                         E. coli Survival Critical High Temperature <br />  <br />  <br />  <br />  Table 2. Thermotolerant genes identified study. <br />  <br />  <br />  Classification          Sub-classification                  Gene Function                                                                            Glca Mg2+b H2O2c <br />  <br />  DNA repair              DNA replication <span id='am-38' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="AMP" datatype="xsd:string"></span>amp</span>; repair, DSBR      dnaQ DNA polymerase III subunit, epsilon                                                        +        S <br />  (Group C)               DNA replication &amp; repair, DSBR      holC   DNA polymerase III subunit, chi                                                          ++       S <br />                          DNA replication &amp; repair, DSBR      priA   primosome factor n&#226;&#8364;&#8482;                                                               ++     +        S <br />                          DNA repair, DSBR                    ruvA   component RuvABC resolvasome, endonuclease                                     ++     ++       S <br />                          DNA repair, DSBR                    ruvC   conserved protein required cell growth                                        +      +        S <br />  tRNA modification       tRNA modification                   iscS   sulfer relay  cysteine desulfurase                                                         S <br />  (Group D)               tRNA modification                   yheL   sulfer relay  predicted intracellular sulfur oxidation protein                             S <br />                          tRNA modification                   yheM sulfer relay  predicted intracellular sulfur oxidation protein                               S <br />                          tRNA modification                   yheN   sulfer relay  predicted intracellular sulfur oxidation protein                             S <br />                          tRNA modification                   yhhP   conserved protein required cell growth                                                        S <br />                          tRNA modification                   miaA   delta(2)-isopentenylpyrophosphate tRNA-<span id='am-76' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-77' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-82' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-90' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> transferase                             +        S <br />                          tRNA modification                   trmU   tRNA (5-methylaminomethyl-2-thiouridylate)- methyltransferase                                     S <br />                          tRNA modification                   truA   pseudouridylate synthase                                                        + <br />  Chaperone/protease      Chaperon                     dnaJ   chaperone Hsp40, chaperone DnaK                                                  ++       S <br />  (Group E)               Chaperon                     dnaK   chaperone Hsp70, chaperone DnaJ                                                           S <br />                          Chaperon                     degP   chaperone/serine endoprotease <br />                          Chaperon regulator                  rseA   anti-sigma factor                                                                        ++       S <br />  Translation control     Translation control                 rpmJ   50S ribosomal subunit L36, related secY expression                                             S <br />  (Group F)               Translation control                 rpsF   30S ribosomal subunit S6, supecifically modified <span id='am-117' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-118' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-122' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span>glutamic acid</span>                            S <br />                                                                     phosphate <br />                          Translation control                 dksA   DNA-binding transcriptional regulator rRNA transcription DnaK suppressor +            +        S <br />                          Translation control                 smpB component trans-translation process                                              +      +        S <br />  Cell division           Related cell division            xerC   site-specific tyrosine recombinse involved chromosome dimer resolution                ++       S <br />  (Group G)               Related cell division            dedD   membrane-anchored periplasmic protein involved separation                             ++ <br />                          Related cell division            envC   regulator cell wall hydrolases responsible cell separation                        + <br />                   Membrane transport                  zntA   zinc/cadmium/mercury/lead-exporting ATPase                                               +        + <br />                          Membrane transport                  ybgH   predicted proton-dependent oligopeptide Transporter, POT family                   +      ++ <br />                          Membrane transpor                   ybhH   conserved hypothetical protein                                                    ++     ++       S <br />  <br />     According data Figure S4, ratios growth presence glucose absence glucose 46uC wer estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;++&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;+&#226;&#8364;&#8482;&#226;&#8364;&#8482; represent    2.0 1.5&#226;&#8364;&#8220;2.0, respectively. <br />  b <br />    According data Figure S4, ratios growth presence MgCl2 absence MgCl2 46uC estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;++&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;+&#226;&#8364;&#8482;&#226;&#8364;&#8482; represent    2.0 1.5&#226;&#8364;&#8220;2.0, respectively. <br />  c <br />   According data Figure S4, ratios growth presence H2O2 absence H2O2 30uC estimated. &#226;&#8364;&#732;&#226;&#8364;&#732;S&#226;&#8364;&#8482;&#226;&#8364;&#8482; represents 0.5. <br />  doi:10.1371/journal.pone.0020063.t002 <br />  <br />  <br /> form persulfide group (-SSH), responsible                          dnaK dnaJ encode chaperone chaperone, respectively, <br /> YheL, YheM, YheN YhhP biosynthesis 2-thio                                 maturation protein folding refolding unfolded proteins <br /> modification 5-methylaminomethyl-2-thiouridine (mnm5s2U)                              [20,50], degP encodes chaperone/serine protease located [43] different thio modifications bacterial tRNAs [45].                      periplasm [21]. indispensability genes CHT <br /> IscS works general sulfur donor various metabolic                           suggests DnaK/DnaJ play crucial role dealing pathways [46] including biosynthesis <span id='am-129' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-130' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>iron</span>-sulfur (Fe-S) cluster                       unfolded proteins caused CHT DegP plays [47], thiamine [48], <span id='am-135' about='obo:IMR_0200415' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-136' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="KEGG:C00253" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:id" content="IMR:0200415" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="Nicotinic acid" datatype="xsd:string"></span>nicotinic acid</span> branched-chain amino                             important role removal damaged proteins acids [49]. Additionally, miaA, trmU truA group                          accumulated temperature. rseA group encodes involved tRNA modification. mutations genes related                         anti-sigma factor sigma 24 inactive non-stress <br /> sulfer modification cause phenotype sensitivity anti-                          conditions. thermosensitivity caused rseA disrupted <br /> oxidation stress [44]. Consistently, study provided evidence                         mutation suggests fine tuning intracellular level mutants group exhibited hypersensitivity oxidative                       active sigma 24 regulates expression chaperone protease <br /> stress. YheL, YheM, YheN YhhP, mainly function                              genes crucial adaptation CHT condition. <br /> tRNA modification [45], conserved thermotolerant bacteria                         Alternatively, defective mutant rseA increased sigma 24 mesophiles  Table S3), iscS, general sulfur donor,                      activity, turn decreased production outer widely conserved mesophiles.                                                       membrane proteins MicA RybB sigma 24 regulon <br />    findings suggest tRNA modifications presented                         gene [51], resulting membrane unstability thermosensi- indispensable growth CHT.                                                     tiveness CHT. <br />    Group E genes encode chaperones protease                                  Genes group F belong translation control apparatus. <br /> contribute cellular process regulating heat shock response:                    S6 encoded rpsF interacts central domain 16S rRNA <br />  <br />  <br />        PLoS | www.plosone.org                                                    4                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                            E. coli Survival Critical High Temperature <br />  <br />  demonstrated play regulatory                  Expressional change caused heat shock CHT <br /> structural role ribosome [52]. L36 encoded rpmJ                   thermotolerant genes identified study component 50S subunit ribosome, disruption              encode HSPs previously identified E. coli decreases expression secY [53], encodes protein-               dnaJ, dnaK, degP dnaQ. examine thermotolerant <br /> conducting channel cytoplasmic membrane. DksA encoded                  genes regulated CHT  examined transient dksA functions negative regulator rRNA genes [54].                change expression genomic genes CHT DNA chip <br /> Overexpression DksA shown suppressor                analysis. results showed 42 genes 111 genes dnaK deletion mutation [55] ensures replication completion             significantly regulated regulated, respectively (Table <br /> removing transcription roadblocks [56]. SmpB encoded smpB               S4). regulated genes classified mainly genes component trans-translation process performs rescue              involved cellular process, regulated genes stalled ribosomes binding partner, transfer-messenger             classified genes involved energy metabolism, <br /> RNA [57]. lines evidence suggest constituents           transport/binding protein translation.  translation pathways crucial survival CHT.                      thermotolerant genes including degP dnaQ heat-shock gene <br />    Finally, genes group G related cell division. xerC-            identified regulated genes dnaJ dnaK. <br /> encoded protein site-specific recombinase [58] essential          Taken data shown  possible conversion chromosome dimers monomers cell                   chaperone systems DnaJ/DnaK GroEL/GroES <br /> division. envC encodes component cell division machinery             neccesarily involved thermotolerant mechanisms direct regulator cell wall hydrolase responsible         acquired CHT.  likely products cell separation required cell division [59]. DedD                 thermotolerant genes HSPs organism <br /> encoded dedD membrane-anchored periplasmic protein                    possesses specific set genes required survival CHT. <br /> involved septation [60] shown participate                  possible essential genes crucial cytokinesis [61].                                                             growth CHT. genes,  examined    functions remaining genes, ybgH, yciM yhhH,             analysis disrupted mutants genes unknown. Notably, thermosensitiveness mutations              available conditional mutants. listed <br /> partially suppressed addition Mg2+. likely          essential genes significant fluctuation expression CHT gene products related cellular activities similar       (Table 3). groEL (groL) encoding HSP regulated, indicating group B, C G.                                                           possibility gene product contributes survival                                                                               CHT. Consistently, reported GroEL appears Possible acquisition thermotolerant genes                          mediator evolution extremely heat-resistant E. coli cells [63]. <br /> horizontal gene transfer                                                      hand, 90% regulated genes    groups outer membrane integrity tRNA                           mapped translation pathway (Figure S5), encoding modification completely conserved limited bacterial             components ribosomal proteins. possible  <br /> species optimal growth relatively high temperature                  regulation ribosomal genes strategies survival (Table S1). group members, genes LPS                         CHT E. coli. Noteworthily, Alix et al. reported ribosome <br /> biosynthesis pathway, lipoproteins sulfur-relay                  biogenesis E. coli high temperature-sensitive DnaK- distributed limited bacterial species including            dependant predicted high temperature causes severe <br /> Enterobacteriaceae (Tables S2 Table S3). sulfur-relay          limitation DnaK/DnaJ hamper ribosome assembly classified tRNA modification demonstrated modify nucleotides tRNA molecules, contributing stabilization structure, required survival extremely <br />                                                                                Table 3. Essential genes significantly regulated high temperature Thermous thermophilus [62]                                                                                regulated CHT. <br /> conserved Thermoanaerobacter tengcongensis (Table S3). mature <br /> LPS biosynthesis pathway assembly outer membrane <br /> consists enzyme reactions,                       Classificationa       Pathwaya                      Gene <br /> dispensable lower temperature. Interestingly, pathway conserved Thermodesulfovibrio yellowstonii Thermanaer-           regulated <br /> ovibrio acidaminovorans (Table S2). Enzymes LPS biosynthesis            Cpn60 chaperonine     RNA degradation               groEL sulfur-relay E. coli share 40% sequence                    tRNA-Leu              Transfer RNA                  leuU <br /> identity 50% sequence similarity corresponding                regulated <br /> enzymes thermophilic bacteria. E. coli closely related              Lipid metabolism      Fatty acid biosynthesis       fabG <br /> bacteria acquired genes groups <br />                                                                                Transcription         RNA polymerase                rpoA <br /> presumably horizontal gene transfer evolution. groups widely conserved                   Translation           16S rRNA processing protein   rim <br /> thermotolerant mesophilic bacteria mesophilic                Translation           Translation factors           fusA <br /> bacteria, intrinsically present E. coli.              Translation           Ribosome                      rpsP, rplQ, rpsD, rpsK, <br /> consistent conserved nature essentiality lipid                                  rpsM, rpmC, rplP, rpsC, LPS essentiality synthesis lipid IVA                                                                                                      rplV, rpsS, rplB, rplW, <br /> dispensability enzymes involved extension Kdo2-lipid                                                                                                      rplD, rplC, rpsJ, rplJ, various glycosyltransferases. draws support (rfaC) <br /> mutants tetraacylated lipid exhibiting narrow                                     rplL, rpsR <br /> growth range ability grow slow growth                                                                                                       Classification Pathway accoding KEGG PATHWAY shown. (http:// <br /> conditions minimal medium 23uC [18], suggesting                       www.genome.jp/kegg). <br /> overall importance outer membrane integrity CHT.                         doi:10.1371/journal.pone.0020063.t003 <br />  <br />  <br />  <br />        PLoS | www.plosone.org                                         5                                 June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                       E. coli Survival Critical High Temperature <br />  <br />  <br /> heat-induced misfolded proteins titrate free               Screening thermosensitive mutants <br /> DnaK/DnaJ [64,65].                                                              Keio collection consisting 3,908 mutant strains used <br />                                                                              screening. 1st screening, mutant strains grown consideration mechanisms survival CHT                      LB plates 30uC overnight. colony strain patched <br />    groups DNA double-strand repair chaperone/                     LB plates incubated 47uC 48 h sensitive <br /> proteinase genes contribute endurance oxidative               strains. sensitive strains subjected 2nd screening stress addition CHT. Evidence higher temperature                spotting tests plates. Cells cultured LB medium 18 h <br /> results accumulation oxidative stress [24]                diluted LB medium adjust turbidity OD600 finding mutants members groups exhibited                 0.5, 0.05 0.005. diluted samples (10 ml) spotted sensitivity oxidative stress allow speculate oxidative         LB plates incubated 47uC 48 h. thermosensitive <br /> stress main cause DNA double-strand breaks damage             strains selected 2nd screening subjected 3rd proteins CHT. Interestingly, oxidative stress involved           screening liquid culture. 8-h preculture, cells diluted <br /> heat-induced cell death Saccharomyces cerevisiae [66],           turbidity corresponding OD600 0.1 inoculated supported findings overexpression catalase                LB medium final OD600 0.001. Samples superoxide dismutase genes increase degree thermo-              incubated 47uC 18 h shaking condition. Thermo- <br /> tolerance thermotolerance increased                    sensitivity determined measuring OD600. Thermosensitive <br /> anaerobic conditions. assume CHT causes                 strains defined ,0.1 OD600. experiments intracellular oxidative stress elicit harmful effects cells       performed times, results confirmed secondary stress.                                                            reproducible. <br />    Significant suppression thermosensitive phenotype defect group energy metabolism (Group                     Effects glucose MgCl2 sensitivity H2O2 <br /> addition glucose suggests limitation energy level CHT              examine effects supplements, glucose (0.5% (w/v)) organism. limitation resolved alternative          MgCl2 (20 mM) added LB liquid culture. 8-h <br /> pathways generate ATP glucose assimilation.                  preculture, cells diluted turbidity corresponding requirement ATP CHT consistent expression               OD600 0.1 inoculated LB medium ribosomal genes. genes ribosomal proteins             supplement final OD600 0.001. Samples regulated exposure CHT, disrupted mutant               incubated 47uC 18 h shaking condition. 18 h, dksA encodes negative regulator rRNA genes              turbidity OD600 measured. test sensitivity thermosensitive CHT. findings evidence                     oxidative stress, H2O2 added culture medium translation ribosomal activity utilizes energy,              final concentration 0.5 mM. 8-h preculture, cells 90% energy consumed cells [67], suggest cells               diluted turbidity corresponding OD600 0.1 manage reduce energy consumption severe condition              inoculated LB medium H2O2 final <br /> CHT. saved energy utilized crucial                   OD600 0.001. Samples incubated 30uC 8 h <br /> activities repair degradation damaged DNA                   shaking condition. experiments performed protein molecules. smooth translational process CHT               times, results confirmed reproducible. save energy, S6 L36 ribosomal proteins addition SmpB important functions. <br />    strategies E. coli survive CHT discovered.         RT-PCR analysis responsible stresses                 Cultures grown LB medium 37uC conserved mesophilic bacteria. Early glycosyltransferases            exponential phase, temperature shifted LPS core biosynthesis proper outer membrane assembly                 47uC incubation continued 8 min. Total RNA permeability barrier function sulfur-relay            immediately prepared heat-stressed cells hot <br /> tRNA modification acquired organism               phenol method [68]. RT-PCR analysis performed using perform main task survive CHT. Considering genetic               mRNA-selective RT-PCR kit (TAKARA BIO  Otsu, Japan) conversion non-thermotolerant thermotolerant bacteria,             examine expression immediate downstream genes strategies applicable.                                          disrupted genes described previously [69]. primer set used <br />                                                                              gene shown Table S5. RT reaction carried <br />                                                                              42uC 15 min, 85uC 1 min, 45uC 1 min Materials Methods <br />                                                                              extension 72uC 2 min using specific primers Materials                                                                    gene. completion 15, 20, 25 30 cycles, PCR <br />   Oligonucleotide primers polymerase chain reaction (PCR)                products analyzed 0.9% agarose gel electrophoresis purchased FASMAC  (Atsugi, Japan).                    stained ethidium bromide. relative amounts RT-PCR <br /> chemicals analytical grade.                                      products gel compared measuring band density <br />                                                                              color image taken reversed using model <br /> Bacterial strains growth conditions                                      GS-700 Imaging Densitometer (Bio-Rad Laboratories,  <br />    Strains used study derivatives E. coli K-12.              Tokyo, Japan) [70]. <br /> W3110  (rrnD-rrnE), rph-1), BW25113 (rrnB3, D(lacZ)4787, <br /> hsdR514, D(araBAD)567, D(rhaBAD)568, rph-1) [14] mutants              Bioinformatics phylogenetic analyses <br /> BW25113 Keio collection single-gene knockout library                Bioinformatics analysis mainly performed according [13] grown plates liquid modified Luria-Bertani             instructions KEGG site (http://www.genome.jp/kegg/). <br /> (LB) medium (1% Bactotryptone, 0.5% yeast extract, 0.5%                  Databases DDBJ (http://www.ddbj.nig.ac.jp) GenBank <br /> NaCl) 37uC, 45uC 47uC appropriate times.                           (http://www.ncbi.nlm.nih.gov/genbank/) used. <br />  <br />  <br />  <br />        PLoS | www.plosone.org                                        6                             June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                            E. coli Survival Critical High Temperature <br />  <br />  <br />  <br /> DNA chip analysis                                                                           RT-PCR performed primers specific just <br />    W3110 cultures grown LB medium 37uC                                 downstream gene thermotolerant gene amplify exponential phase, temperature shifted                               500-bp DNA fragments.  B) RT reaction, PCR 47uC incubation continued 8 min. control culture                              performed 15, 20, 25 30 cycles products incubated parallel 37uC 8 min. Total RNA                                  analyzed. (C D) control, total RNA (10 mg) immediately prepared heat-stressed cells hot                                submitted 1.2% agarose gel electrophoresis staining phenol method [69]. Preparation cDNA, fragmentation                              ethidium bromide. <br /> end-labeling DNA fragments performed according                               (TIF) <br /> instruction manual Affymetrix. ENZO Bioarray <br />                                                                                             Figure S4 Effects addition glucose MgCl2 terminal labeling kit (Enzo Life Sciences,  New York, USA) <br />                                                                                             sensitivity H2O2. Thermosensitive mutant strains shown used end-<span id='am-134' about='rdfs:label' typeof='owl:Thing'>label</span> DNA fragments. DNA hybridization, data <br />                                                                                             gene names. Growth conditions described Materials capture analyses performed described protocol <br />                                                                                             Methods. Black white columns represent turbidity supplied Affymetrix GCOS software (Affymetrix,  <br /> California, USA). independent experiments performed                                conditions supplements (0.5% glucose   data sets  data sets 37uC: 37uC-1 37uC-2,                           20 mM MgCl2 (B)) 0.5 mM H2O2 (C). <br /> data sets 47uC: 47uC-1 47uC-2) gene obtained.                               (TIF) expression ratio used indicates average ratios                          Figure S5 regulated genes ribosomal pro- <br /> obtained independent experiments. Spots                                   teins. Systematic analysis gene function performed significantly lower (,0.50; e., negative fold difference) higher                     database KEGG PATHWAY. regulated genes (2.; e., positive fold difference) fluorescence ratio heated                     ribosomal proteins mapped 6 operons. <br /> sample control sample considered represent real                            (TIF) <br /> significant difference. Physiological function functional <br /> classification genes derived Genobase                                  Table S1 Distribution thermotolerant genes database (http://ecoli.aist-nara.ac.jp/). Array data accessible                         various bacteria. ArrayExpress accession number E-MEXP-3191.                                          (DOC) <br />                                                                                             Table S2 Distribution thermotolerant genes group <br /> Supporting Information                                                                      B various bacteria. <br /> Figure S1 Growth thermosensitive mutants LB                                           (DOC) <br /> liquid culture different temperatures. 51 thermo-                                   Table S3 Distribution thermotolerant genes group <br /> sensitive mutant strain (opened symbols) parental strain,                           D various bacteria. <br /> BW25113 (closed symbols), grown 30 ml LB medium                                  (DOC) <br /> 37uC (circles), 45uC (squares), 46uC (triangles). times <br /> indicated, turbidity OD600 measured.  group  B, group                            Table S4 Genes significantly regulated  <br /> B; C, group C; D, group D; E, group E; F, group F; G, group G;                              regulated CHT. <br /> H,                                                                                   (DOC) <br /> (TIF)                                                                                       Table S5 RT-PCR primers used study. <br /> Figure S2 Gene organizations genes having                                            (DOC) essential gene thermotolerant gene just downstream gene. Gene organizations 13                                          Acknowledgments <br /> thermotolerant genes essential gene                                                                                             thank K. Matsushita T. Yakushi helpful discussion, J. <br /> thermotolerant gene just downstream gene depicted. Black                           Makino  Ishii assisting E. coli growth experiments. thank <br /> boxes represent identified 13 thermotolerant genes. Grey boxes                              National BioResource Project (NIG, Japan) providing 3908 mutants <br /> represent essential thermotolerant genes. direction boxes                         KEIO Collection. <br /> shows direction transcription. <br /> (TIF)                                                                                       Author Contributions <br /> Figure S3 Testing possible polar effects aph                                      Conceived designed experiments: MM HF SR  <br /> insertion. Total RNA prepared cells cultured 37uC                               Performed experiments: MM HF KN KC HN. Analyzed data: <br />   C) 47uC (B, D) described Materials Methods.                               MM HF TK  Wrote paper: MM HF SR  <br />  <br /> References <br />  1. Morimoto RI, Kroeger PE, Cotto JJ (1996) transcriptional regulation               6. Narberhaus F, Kaser R, Nocker  Hennecke H (1998) novel DNA element <br />     heat shock genes. plethora heat shock factors regulatory conditions.            controls bacterial heat-shock gene expression. Mol Microbiol 28: 315&#226;&#8364;&#8220;323. <br />     Feige U, Morimoto RI, Yahara  Polla BS, eds. transcriptional regulation            7. Erickson JW, Gross CA (1989) Identification sigma E subunit Escherichia <br />     heat shock genes: plethora heat shock factors regulatory conditions.            coli RNA polymerase: second alternate sigma factor involved high- <br />     Basle: Birkha&#194;&#168;user-Verlag. pp 139&#226;&#8364;&#8220;163.                                                      temperature gene expression. Genes Dev 3: 1462&#226;&#8364;&#8220;1471. <br />  2. Boorstein WR, Ziegelhoffer T, Craig EA (1994) Molecular evolution                 8. Raina S, Missiakas D, Georgopoulos C (1995) rpoE gene encoding sigma <br />     HSP70 multigene family. J Mol Evol 38: 1&#226;&#8364;&#8220;17.                                                E (sigma 24) heat-shock sigma factor Escherichia coli. EMBO J 14: 1043&#226;&#8364;&#8220;1055. <br />  3. Gupta RS (1995) Evolution chaperonin families (Hsp60, Hsp10, Tcp-             9. Alba BM, Leeds JA, Onufryk C, Lu CZ, Gross CA (2002) DegS YaeL <br />     1) proteins origin eukaryotic cells. Mol Microbiol 15: 1&#226;&#8364;&#8220;11.                  participate sequentially cleavage RseA activate sigma(E)- <br />  4. Yura T, Kanemori M, Morita MT (2000) heat shock response: regulation                    dependent extracytoplasmic stress response. Genes Dev 16: 2156&#226;&#8364;&#8220;2168. <br />     function.  Storz G, Hennecke H, eds. Bacterial stress responses.                 10. Morita M, Kanemori M, Yanagi H, Yura T (1999) Heat-induced synthesis     Washington, D.C.: ASM Press. pp 3&#226;&#8364;&#8220;18.                                                       sigma32 Escherichia coli: structural functional dissection rpoH mRNA <br />  5. Gunasekera TS, Gsonka LN, Pally O (2008) Genome-wide transcriptional                        secondary structure. J Bacteriol 181: 401&#226;&#8364;&#8220;410. <br />     responses Escherichia coli K-12 continuous osmotic heat stresses.             11. Klein G, Dartigalongue C, Raina S (2003) Phosphorylation-mediated regulation <br />     J Bacteriol 190: 3712&#226;&#8364;&#8220;3720.                                                                 heat shock response Escherichia coli. Mol Microbiol 48: 269&#226;&#8364;&#8220;285. <br />  <br />  <br />  <br />         PLoS | www.plosone.org                                                      7                                   June 2011 | Volume 6 | Issue 6 | e20063 <br />                                                                                                                                    E. coli Survival Critical High Temperature <br />  <br />  <br /> 12. Helmann JD, Chamberlin MJ (1988) Structure function bacterial sigma                     40. Bennett RJ, West SC (1996) Resolution Holliday junctions genetic <br />     factors. Annu Rev Biochem 57: 839&#226;&#8364;&#8220;872.                                                             recombination: RuvC protein nicks DNA point strand exchange. Proc <br /> 13. Baba T, Ara T, Hasegawa M, Takai Y, Okumura Y, et al. (2006) Construction                          Natl Acad Sci USA 93: 12217&#226;&#8364;&#8220;12222. <br />     Escherichia coli K-12 frame, single-gene knockout mutants: Keio                      41. Al-Deib AA, Mahdi AA, Lloyd RG (1996) Modulation recombination     collection. Mol Syst Biol 2: 1&#226;&#8364;&#8220;8.                                                                  DNA repair RecG PriA helicases Escherichia coli K-12. J Bacteriol <br /> 14. Datsenko KA, Wanner BL (2000) step inactivation chromosomal genes                           178: 6782&#226;&#8364;&#8220;6789. <br />     Escherichia coli K-12 using PCR products. Proc Natl Acad Sci USA 97:                        42. Rojas DM, Va&#194;&#180;squez CC (2005) Sensitivity potassium tellurite Escherichia coli <br />     6640&#226;&#8364;&#8220;6645.                                                                                         cells deficient CSD, CsdB IscS cysteine desulfurases. Res Microbiol 156: <br /> 15. Yamamoto N, Nakahigashi K, Nakamichi T, Yoshino M, Takai Y, et al. (2009)                          465&#226;&#8364;&#8220;471. <br />     Update Keio collection Escherichia coli single-gene deletion mutants.                43. Ikeuchi Y, Shigi N, Kato J, Nishimura  Suzuki T (2006) Mechanistic insights <br />     Mol Syst Biol 5: 335&#226;&#8364;&#8220;337.                                                                          sulfur relay multiple sulfur mediators involved thiouridine biosynthesis <br /> 16. Wall D, Delaney JM, Fayet O, Lipinska B, Yamamoto T, et al. (1992) arc-                            tRNA wobble positions. Mol Cell 21: 97&#226;&#8364;&#8220;108. <br />     dependent thermal regulation extragenic suppression Escherichia coli                44. Dahl C, Schulte  Stockdreher Y, Hong C, Grimm F, et al. (2008) Structural <br />     cytochromed operon. J Bacteriol 174: 6554&#226;&#8364;&#8220;6562.                                                    molecular genetic insight widespread sulfur oxidation pathway. J Mol <br /> 17. Raina S, Georgopoulos C (1991) htrM gene, product essential                       Biol 384: 1287&#226;&#8364;&#8220;1300. <br />     Escherichia coli viability elevated temperatures, identical rfaD gene.       45. Lauhon CT (2002) Requirement IscS biosynthesis thio-nucleosides     Nucleic Acids Res 19: 3811&#226;&#8364;&#8220;3819.                                                                   Escherichia coli. J Bacteriol 184: 6820&#226;&#8364;&#8220;6829. <br /> 18. Klein G, Lindner B, Brabetz W, Brade H, Raina S (2009) Escherichia coli K-12                   46. Mihara H, Esaki N (2002) Bacterial cysteine desulfurases: function     Suppressor-free Mutants Lacking Early Glycosyltransferases Late Acyl-                          mechanisms. Appl Microbiol Biotechnol 60: 12&#226;&#8364;&#8220;30. <br />     transferases: minimal lipopolysaccharide structure induction envelope                   47. Frazzon J, Dean DR (2003) Formation iron-sulfur clusters bacteria:     stress response. J Biol Chem 284: 15369&#226;&#8364;&#8220;15389.                                                     emerging field bioinorganic chemistry. Curr Opin Chem Biol 7: 166&#226;&#8364;&#8220;173. <br /> 19. Adams  West SC (1996) Bypass DNA heterologies RuvAB-                              48. Taylor SV, Kelleher NL, Kinsland C, Chiu HJ, Costello CA, et al. (1998) <br />     mediated  strand branch migration. J Mol Biol 263: 582&#226;&#8364;&#8220;596.                         Thiamin biosynthesis Escherichia coli. Identification thiocarboxylate 20. Tomoyasu T, Ogura T, Tatsuta T, Bukau B (1998) Levels DnaK DnaJ                             immediate sulfur donor thiazole formation. J Biol Chem 273: <br />     provide tight control heat shock gene expression protein repair                          16555&#226;&#8364;&#8220;16560. <br />     Escherichia coli. Mol Microbiol 30: 567&#226;&#8364;&#8220;581.                                                   49. Lauhon CT, Kambampati R (2000) iscS gene Escherichia coli required <br /> 21. Lipinska B, Zylicz M, Georgopoulos C (1990) HtrA (DegP) protein,                               biothynthesis 4-thiouridine, thiamine, NAD. J Biol Chem 280: <br />     essential Escherichia coli survival high temperatures, endopeptidase.                 26760&#226;&#8364;&#8220;26769. <br />     J Bacteriol 172: 1791&#226;&#8364;&#8220;1797.                                                                    50. Ben-Zvi AP, Goloubinoff P (2001) Review: mechanisms disaggregation 22. Ohara M, Wu HC, Sankaran K, Rick PD (1999) Identification                                      refolding stable protein aggregates molecular chaperones. J Struct Biol 135: <br />     characterization new lipoprotein, NlpI, Escherichia coli K-12. J Bacteriol                 84&#226;&#8364;&#8220;93. <br />     181: 4318&#226;&#8364;&#8220;4325.                                                                                51. Valentin-Hansen P, Johansen J, Rasmussen AA (2007) Small RNAs controlling <br /> 23. Kabir MS, Yamashita D, Koyama S, Oshima T, Kurokawa K, et al. (2005) Cell                          outer membrane porins. Curr Opin Microbiol 10: 152&#226;&#8364;&#8220;155. <br />     lysis directed sigmaE early stationary phase effect induction              52. Britton RA, Lupski JR (1997) Isolation characterization suppressors     rpoE gene global gene expression Escherichia coli. Microbiology 151: <br />                                                                                                        Escherichia coli dnaG mutations, dnaG2903 parB. Genetics 145: 867&#226;&#8364;&#8220;875. <br />     2721&#226;&#8364;&#8220;2735. <br />                                                                                                    53. Ikegami  Nishiyama K, Matsuyama S, Tokuda H (2005) Disruption rpmJ <br /> 24. Noor R, Murata M, Yamada M (2009) Oxidative stress trigger growth <br />                                                                                                        encoding ribosomal protein L36 decreases expression secY upstream     phase-specific sigmaE-dependent cell lysis Escherichia coli. J Mol Microbiol <br />                                                                                                        spc operon inhibits protein translocation Escherichia coli. Biosci Biotechnol <br />     Biotechnol 17: 177&#226;&#8364;&#8220;187. <br />                                                                                                        Biochem 69: 1595&#226;&#8364;&#8220;1602. <br /> 25. Haydon DJ, Quail MA, Guest JR (1993) mutation causing constitutive <br />                                                                                                    54. Perron K, Comte R, van Delden C (2005) DksA represses ribosomal gene <br />     synthesis pyruvate dehydrogenase complex Escherichia coli located <br />                                                                                                        transcription Pseudomonas aeruginosa interacting RNA polymerase     pdhR gene. FEBS Lett 336: 43&#226;&#8364;&#8220;47. <br />                                                                                                        ribosomal promoters. Mol Microbiol 56: 1087&#226;&#8364;&#8220;1102. <br /> 26. Lyngstadaas  L&#195;&#184;bner-Olesen  Boye E (1995) Characterization genes <br />                                                                                                    55. Kang PJ, Craig EA (1990) Identification characterization new Escherichia <br />     dam-containing operon Escherichia coli. Mol Gen Genet 247: 546&#226;&#8364;&#8220;554. <br />                                                                                                        coli gene dosage-dependent suppressor dnaK deletion mutation. <br /> 27. Cassey B, Guest JR, Attwood MM (1988) Environmental control pyruvate <br />                                                                                                        J Bacteriol 172: 2055&#226;&#8364;&#8220;2064. <br />     dehydrogenase complex expression Escherichia coli. FEMS Microbiol Lett <br />     1159: 325&#226;&#8364;&#8220;329.                                                                                 56. Tehranchi AK, Blankschien MD, Zhang Y, Halliday JA, Srivatsan  et al. <br />                                                                                                        (2010) transcription factor DksA prevents conflicts DNA <br /> 28. Barak R, Abouhamad WN, Eisenbach M (1998) acetate kinase acetyl <br />                                                                                                        replication transcription machinery. Cell 141: 595&#226;&#8364;&#8220;605. <br />     coenzyme synthetase involved acetate-stimulated change     direction flagellar rotation Escherichia coli. J Bacteriol 180: 985&#226;&#8364;&#8220;988.                 57. Watts T, Cazier D, Healey D, Buskirk (2009) SmpB contributes reading <br /> 29. Mogi T, Mizuochi-Asai E, Endou S, Akimoto S, Nakamura H (2006) Role                           frame selection translation transfer-messenger RNA. J Mol Biol 391: <br />     putative subunit YhcB assembly function cytochrome bd-                         275&#226;&#8364;&#8220;281. <br />     type <span id='am-45' about='obo:IMR_0200466' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Ubiquinol" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00390" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0200466" datatype="xsd:string"></span>ubiquinol</span> oxidase Escherichia coli. Biochim Biophys Acta 1757:                       58. Bloor AE, Cranenburgh RM (2006) efficient method selectable marker <br />     860&#226;&#8364;&#8220;864.                                                                                           gene excision Xer recombination gene replacement bacterial <br /> 30. Reed KE, Cronan JE, Jr. (1993) Lipoic acid metabolism Escherichia coli:                         chromosomes. Appl Environ Microbiol 72: 2520&#226;&#8364;&#8220;2525. <br />     sequencing functional characterization lipA lipB genes. J Bacteriol             59. Uehara T, Parzych KR, Dinh T, Bernhardt TG (2010) Daughter cell separation <br />     175: 1325&#226;&#8364;&#8220;1336.                                                                                    controlled cytokinetic ring-activated cell wall hydrolysis. EMBO J 29: <br /> 31. Kneidinger B, Marolda C, Graninger M, Zamyatina  McArthur F, et al.                              1412&#226;&#8364;&#8220;1422. <br />     (2002) Biosynthesis pathway ADP-L-glycero-beta-D-manno-heptose                           60. Gerding MA, Liu B, Bendezu&#194;&#180; FO, Hale CA, Bernhardt TG, et al. (2009) Self- <br />     Escherichia coli. J Bacteriol 184: 363&#226;&#8364;&#8220;369.                                                        enhanced accumulation FtsN Division Sites Roles Proteins <br /> 32. Roncero C, Casadaban MJ (1992) Genetic analysis genes involved                           SPOR domain (DamX, DedD, RlpA) Escherichia coli cell <br />     synthesis lipopolysaccharide core Escherichia coli K-12: operons                constriction. J Bacteriol 191: 7383&#226;&#8364;&#8220;7401. <br />     rfa locus. J Bacteriol 174: 3250&#226;&#8364;&#8220;3260.                                                     61. Arends SJ, Williams K, Scott RJ, Rolong S, Popham DL, et al. (2010) Discovery <br /> 33. Gerding MA, Ogata Y, Pecora ND, Niki H, Boer PA (2007) trans-                               characterization new Escherichia coli septal ring proteins contain <br />     envelope Tol-Pal complex cell division machinery required                   SPOR domain: DamX, DedD, RlpA. J Bacteriol 192: 242&#226;&#8364;&#8220;255. <br />     proper outer-membrane invagination cell constriction E. coli. Mol                    62. Shigi N, Sakaguchi Y, Asai S, Suzuki T, Watanabe K (2008) Common thiolation <br />     Microbiol 63: 1008&#226;&#8364;&#8220;1025.                                                                           mechanism biosynthesis tRNA thiouridine sulphur-containing <br /> 34. Kampfenkel K, Braun V (1993) Membrane topologies TolQ TolR                              cofactors. EMBO J 7: 3267&#226;&#8364;&#8220;3278. <br />     proteins Escherichia coli: inactivation TolQ missense mutation               63. Rudolph B, Gebendorfer KM, Buchner J, Winter J (2010) Evolution     proposed transmembrane segment. J Bacteriol 75: 4485&#226;&#8364;&#8220;4491.                                   Escherichia coli growth high temperatures. J Biol Chem 285: 19029&#226;&#8364;&#8220;19034. <br /> 35. Niba ET, Naka Y, Nagase M, Mori H, Kitakawa M (2007) genome-wide                             64. Al Refaii  Alix JH (2009) Ribosome biogenesis temperature-dependent     approach identify genes involved biofilm formation E. coli. DNA Res                   delayed Escherichia coli lacking chaperones DnaK DnaJ. Mol Microbiol <br />     14: 237&#226;&#8364;&#8220;246.                                                                                       71: 748&#226;&#8364;&#8220;762. <br /> 36. Nikaido H (2003) Molecular basis bacterial outer membrane permeability                      65. Rene&#194;&#180; O, Alix JH (2010) Late steps ribosome assembly E. coli sensitive     revisited. Microbiol Mol Biol Rev 67: 593&#226;&#8364;&#8220;656.                                                     severe heat stress assisted HSP70 chaperone machine. Nucleic <br /> 37. Motamedi MR, Szigety SK, Rosenberg SM (1999) Double-strand-break repair                            Acids Res 39: 1855&#226;&#8364;&#8220;1867. <br />    <br /> </body></html>