<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                                                                   Article <br />  <br />                                                                                                                             pubs.acs.org/biochemistry <br />  <br />  <br />  <br />  <br /> Expression Translationally Fused TAP-Tagged Plasma Membrane <br /> Proton Pump Arabidopsis thaliana <br /> Rachel B. Rodrigues, Gregorz Sabat, Benjamin B. Minko&#239;&#172;&#8364;, Heather L. Burch, Thao T. Nguyen, Michael R. Sussman* <br /> Department Biochemistry, Biotechnology Center, University Wisconsin, 425 Henry Mall, Madison, Wisconsin 53706, United <br /> States <br />   * <br />   S Supporting Information <br />  <br />  <br />  <br />   ABSTRACT: Arabidopsis thaliana plasma membrane proton ATPase genes, AHA1 AHA2, <br />   highly expressed isoforms 11 gene family collectively essential   embryo development. report translational fusion tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation tag   5&#226;&#8364;&#178; end AHA1 open reading frame genomic clone. Stable expression TAP-tagged AHA1 <br />   Arabidopsis rescues embryonic lethal phenotype endogenous double aha1/aha2 <br />   knockdowns. Western blots SDS-PAGE Blue Native gels enrichment AHA1   plasma membrane fractions indicate hexameric quaternary structure. TAP-tagged AHA1 rescue <br />   lines exhibited reduced vertical root growth. Analysis plasma membrane soluble <br />   proteomes identi&#239;&#172;?ed plasma membrane-localized proteins alterred abundance TAP- <br />   tagged AHA1 rescue lines compared wild type. Using a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry,   uniquely identi&#239;&#172;?ed additional AHA isoforms, AHA9 AHA11, copuri&#239;&#172;?ed TAP- <br />   tagged AHA1. conclusion, generated transgenic Arabidopsis lines TAP-tagged <br />   AHA1 transgene complemented essential endogenous AHA1 AHA2 functions   shown plants used purify AHA1 protein identify planta interacting <br />   proteins mass spectrometry. <br />  <br />  <br />  <br /> T      Arabidopsis thaliana H+-ATPase genes, AHA1        AHA2, belong protein family 100 kDa, <br /> plasma membrane-localized P-type ATPases. AHA proteins <br />                                                                             dant, nature AHA1 AHA2 complicated reverse- <br />                                                                             genetic approaches elucidate speci&#239;&#172;?c cellular functions                                                                             proteins.20 create facile belong electrogenic class P-type ATPases responsible              investigating AHA1 speci&#239;&#172;?c functions, developed transgenic generating primary active membrane potentials,               endogenous double aha1/aha2 knockdown lines rescued includes Ca2+-ATPase sarcoplasmic reticulum                  tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation (TAP)-tagged AHA1, Na+/K+-ATPase mammalian cells.1 Speci&#239;&#172;?cally, AHAs                    replacing &#226;&#710;&#188;70% AHA protein crude seedling link <span id='am-34' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-35' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span> catalysis active transport protons            vegetative extracts fusion protein capable facilitating <br /> cell, establishing transmembrane proton motive force              isoform-speci&#239;&#172;?c methods including puri&#239;&#172;?cation, biochemical <br /> (PMF) drives secondary transport processes.2&#226;&#710;&#8217;4                analysis, mass spectrometry. <br /> PMF composed pH gradient electrical                        increasing de&#239;&#172;?ning protein-interaction net- <br /> membrane potential reach excess &#226;&#710;&#8217;250 mV                   works greatly facilitated emerging proteomic, <br /> roots.5 AHAs highly expressed cell types specialized             genomic, imaging technologies.21&#226;&#710;&#8217;23 especially <br /> active transport, including phloem tissue, guard cells,                 signi&#239;&#172; contribution identi&#239;&#172;?cation protein <br /> epidermal root hairs, major consumers cellular                   interactions growing access modern mass spectrometry <br /> ATP.4,6&#226;&#710;&#8217;8                                                                   (MS) resources ability perform quantitative <br />    11 genes encoding functional proteins, AHA1                   experiments.24 Recently, large Arabidopsis protein-inter- <br /> AHA2 highly expressed isoforms,                        action networks published including yeast  <br /> abundant tissues developmental stages based                   hybrid approach identi&#239;&#172;?ed 6200 interactions 2700 <br /> EST expression data, estimated make 80%                 proteins25 split ubiquitin approach focused membrane AHA activity.5,9,10 plasma membrane proton pumps                proteins revealed 541 interactions 239 proteins.26 <br /> hypothesized play roles control tip growth                  AHA1 AHA2 protein interactions identi&#239;&#172;?ed generating localized proton &#239;&#172;&#8218;uxes11,12 highly regulated             yeast hybrid interactome database, high- control guard cell opening closure response biotic             con&#239;&#172;?dence candidates, annotated leucine-rich repeat abiotic stimuli.13&#226;&#710;&#8217;16 Using reverse-genetic analysis, roles AHA3 pollen development, AHA4 salt stress, AHA10                  Received: August 12, 2013 proanthocyanidin vacuole biosynthesis                      Revised: January 2, 2014 <br /> reported.17&#226;&#710;&#8217;19  essential, genetically redun-               Published: January 3, 2014 <br />  <br />                               &#194;&#169; 2014 American <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>Chemical</span> Society        566                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                    Article <br />  <br /> kinases, generated split ubiquitin approach.                    Forward: GATTTCTTCTGGGTGAAGATGTCG- <br /> Furthermore, proteins previously published interact              ACTCTCGAAGATATCAAGAACGAGA plasma membrane proton pumps identi&#239;&#172;?ed                      Reverse: TCTCGTTCTTGATATCTTCGAGAGT- interactome studies.27&#226;&#710;&#8217;29,14,30,31                                 CGACATCTTCACCCAGAAGAAATC <br />    A&#239;&#172;&#402;nity puri&#239;&#172;?cation epitope-tagged proteins combined                   gAHA1 fragment subcloned Agrobacterium mass spectrometry-based proteomic analysis             transformation vector pGreenII0179 SacI XhoI <br /> widely established method characterization protein            sites.39 N-terminal TAPi tag ampli&#239;&#172;?ed high- <br /> complexes, tandem a&#239;&#172;&#402;nity-puri&#239;&#172;?cation tags                  &#239;&#172;?delity polymerase gateway cloning vector nta- <br /> successfully used investigate protein complexes plants         pi.289.gw32 using following primers designed append a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry (AP&#226;&#710;&#8217;MS) ap-                   SalI restriction overhangs: <br /> proaches.32&#226;&#710;&#8217;36,23 A&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry meth-                Forward: AAGTCGACCGTGGTGGACAACAAGTTCAA <br /> ods combined chemical cross-linking                 Reverse: TTGTCGACGAAGAAGATCCTCCTCCTCCC- puri&#239;&#172;?cation low-abundance plasma membrane protein                 AGTGCGCCGCTG <br /> complexes plants.36                                                    <span id='am-52' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-53' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-66' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span>Adenosine</span> overhangs added Taq polymerase,     report functional rescue aha1/aha2                  fragment cloned pCR2.1. Ntapi SalI restriction <br /> double-knockout plants TAP-tagged AHA1 genomic                    fragments cloned unique AHA1 SalI site. <br /> transgene (Ntapi:gAHA1). observed reduced vertical root               Constructs con&#239;&#172;?rmed sequencing used growth Ntapi:gAHA1 transgenic rescue lines,              transform aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4 plants Agro- <br /> phenotypic consequences overall plant growth apparent             bacterium-mediated &#239;&#172;&#8218;oral dip, strain GV3101 carrying standard laboratory growth conditions. investigate              pSOUP.40 Transgenic seeds selected hygromycin basis short-root phenotype assess validity         media (half-strength M<span id='am-27' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;S, 1% sucrose, 0.7% agar, 25 &#206;&#188;g/ using plants future AHA1 functional studies,             mL hygromycin). Hygromycin-resistant (HygR) aha1/ <br /> used metabolically labeled mass spectrometry approach               aha1;aha2/aha2 plants complemented presence characterize proteins showing increased decreased abun-               N-terminal Ntapi:gAHA1 construct isolated T2 <br /> dance Ntapi:gAHA1 compared wild-type plants. used               population PCR genotyping. <br /> a&#239;&#172;&#402;nity puri&#239;&#172;?cation followed tandem mass spectrometry                  Plasma Membrane Preparation. Arabidopsis seedlings <br /> identify AHA1-interacting protein candidates Ntapi:AHA1               grown 9 days half-strength M&amp;S liquid media copuri&#239;&#172;?cation investigated e&#239;&#172;&#8364;ect vivo elicitation          described previously.41 day old seedlings extracted copurifying proteins. Ntapi:gAHA1 transgenic plants               2&#195;&#8212; (v/w) homogenization bu&#239;&#172;&#8364;er (230 mM sorbitol, 50 mM <br /> reported provide important tool AHA1 isoform-           Tris-HCl, pH 7.5, 10 mM KCl, 1 mM potassium metabisul&#239;&#172;?te, <br /> speci&#239;&#172;?c biochemical analysis, identi&#239;&#172;?cation new protein               1 mM PMSF, 10 mM sodium &#239;&#172;&#8218;uoride, 2 mM sodium <br /> interactions, veri&#239;&#172;?cation interactions previously identi-         <span id='am-16' about='obo:IMR_0200280' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:id" content="IMR:0200280" datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="Pyrophosphate" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00013" datatype="xsd:string"></span>pyrophosphate</span>, 1 mM ammonium molybdate, complete <br /> &#239;&#172;?ed using heterologous techniques.                                       EDTA-free protease inhibitor cocktail (Roche)). Microsomes <br />  <br /> &#226;&#8211;&#160;    EXPERIMENTAL PROCEDURES <br />    Plant Materials Growth Conditions. Mutants <br />                                                                          collected previously described.20 microsomal pellet <br />                                                                          resuspended resuspension bu&#239;&#172;&#8364;er (330 mM sorbitol, 5 <br />                                                                          mM KCl, 5 mM K2HPO4, pH 7.8) homogenized (ecotype Columbia) carrying homozygous T-DNA insertions                  Te&#239;&#172;&#8218;on homogenizer, plasma membranes enriched AHA1 (aha1&#226;&#710;&#8217;6, SALK016325) heterozygous insertions                 phase partition described previously.42 Protein concen- AHA2 (aha2&#226;&#710;&#8217;4, SALK082786) created crossing                 trations determined Bradford assays. <br /> described previously.20 Seeds germinated plates                     Protein Puri&#239;&#172;?cation. Plasma membrane protein (100&#226;&#710;&#8217;500 <br /> containing half-strength Murashige &amp; Skoog (M&amp;S) salts, 1%               &#206;&#188;g) solubilized 10:1 (w/w) n-dodecyl-&#206;&#178;-D-maltoside <br /> (w/v) sucrose, 0.7% (w/v) agar. Seedlings transferred           resuspension bu&#239;&#172;&#8364;er 1 h 4 &#194;&#176;C. Unsolubilized material soil/perlite mixture grown constant light 21 &#194;&#176;C.         pelleted benchtop microcentrifuge speed 30 <br />    Root Growth Assays. Root growth asssays conducted                min 4 &#194;&#176;C. Solubilized upper-phase protein (0.1 mg, 100&#226;&#710;&#8217;200 described previously.20,37                                            &#206;&#188;L total volume) incubated 1 h 4 &#194;&#176;C 15 &#206;&#188;L    DNA Extraction Plant Genotyping. Plant genomic                    rabbit IgG agarose resin (Sigma, A9294) 10 &#206;&#188;L IgG <br /> DNA extracted using method Krysan et al.             Sepharose 6 Fast Flow resin (GE Life Sciences, 17-0969-01) <br /> elimination phenol/chloroform step.38 presence             equilibrated resuspension bu&#239;&#172;&#8364;er 0.1% (w/v) DDM. <br /> aha1&#226;&#710;&#8217;6 aha2&#226;&#710;&#8217;4 T-DNA insertion wild-type alleles               Resin pelleted 0.8 rcf 1 min, supernatant determined PCR using allele-speci&#239;&#172;?c primers spanning              discarded. Resin washed times 4 &#194;&#176;C 10 min T-DNA AHA1 AHA2 genomic junctions (T-DNA LB:                      100&#195;&#8212; volume resuspension bu&#239;&#172;&#8364;er containing 1% NP40 <br /> T C AA C G G T T T T C G C C T G CT ; S0 1 6 3 2 5 L P :           followed &#239;&#172;?nal wash 100&#195;&#8212; volume resuspension <br /> CGTCTCAACAAAAGTCTCTTTCA; S016325 RP:                                     bu&#239;&#172;&#8364;er containing 1% NP40 0.01% SDS. Resin pelleted <br /> CGAAAGATCAACCTCGTGAGT; S082786 LP: ATGTTC-                               1 min 0.8 rcf wash, supernatant ATTGCAAAGGTGGTG; S082786 RP: CCCAT-                                  discarded. Protein eluted overnight 4 &#194;&#176;C 30 units <br /> TAGCTCGTGGTTATTG).20                                                     AcTEV protease (Invitrogen). <br />    Cloning Transformation TAP-Tagged AHA1.                           SDS-PAGE. SDS-PAGE analysis performed Novex <br /> Cloning 9.9 kb genomic region containing AHA1                  10% Tris-<span id='am-82' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-83' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-88' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span> gels (Invitrogen) using 1&#195;&#8212; Novex Tris- <br /> 3377 bp upstream 1100 bp downstream DNA                        <span id='am-94' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-95' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>Glycine</span> SDS running bu&#239;&#172;&#8364;er NuPAGE 4&#226;&#710;&#8217;12% Bis-Tris <br /> (gAHA1) described previously.20 unique SalI restriction-           gels using 1&#195;&#8212; MOPS running bu&#239;&#172;&#8364;er 80 V 3 4 h. <br /> enzyme site created 5&#226;&#8364;&#178; end AHA1 open                   Protein samples diluted 1:1 Laemmli sample bu&#239;&#172;&#8364;er reading frame using QuikChange II XL kit (Stratagene)            5% &#206;&#178;-mercaptoethanol (BioRad). western blot, gels following mutagenic primers:                                         incubated room temperature 20 min 100 mL                                                                    567                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                   Article <br />  <br /> equilibration bu&#239;&#172;&#8364;er (2&#195;&#8212; NuPAGE transfer bu&#239;&#172;&#8364;er, 10%                      100 000 resolving power. &#239;&#172;?ve ions, excluding <br /> methanol, 1:1000 NuPAGE antioxidant). Proteins                 unassigned +1 charge states, repeatedly isolated <br /> transferred PVDF using iBlot dry blotting                 2.5 Da window CID analysis linear <span id='am-21' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-22' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-23' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> trap <br /> (Invitrogen) using program P3 12 min. Immunoblotting                using normalized collision energy 35.0 activation Q performed according standard protocols using HRP-                   time 0.25 30 ms, respectively. Dynamic exclusion <br /> conjugated rabbit anti-goat IgG colorometric visualization          used 40 s repeat count 1 rabbit anti-CBP (Calmodulin-binding protein, Genscript)              duration analysis. TAIR9 proteome database followed IRDye goat anti-rabbit (LI-COR) membranes               searched using Mascot v2.2, allowing missed tryptic <br /> visualized LI-COR Odyssey infrared scanner.                      cleavage, +2 +3 charge states, <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span>peptide</span> tolerance 20 <br />    Blue Native PAGE. micrograms plasma membrane                ppm, MS/MS ion tolerance 0.6 Da. Carbamidomethy- <br /> protein combined 1% digitonin membrane-                     lation set &#239;&#172;?xed modi&#239;&#172;?cation, phosphorylated S/T/ <br /> solubilization bu&#239;&#172;&#8364;er (50 mM Tris-HCl, 150 mM NaCl, 1                Y, deamidated N/Q, oxidated M set variable <br /> mM EDTA, pH 7.5) previously described.43 NativePAGE                  modi&#239;&#172;?cations. reverse database appended  <br /> sample bu&#239;&#172;&#8364;er (Invitrogen) added supernatant             currently searched. Data &#239;&#172;?ltered 1% false discovery rate <br /> &#239;&#172;?nal concentration 1&#195;&#8212;, NativePAGE 5% G-250 sample                using house developed software (available http://www. <br /> additive added &#239;&#172;?nal concentration 0.06%. Samples            biotech.wisc.edu/sussmanlab/home), ratio calculations run NativePAGE Novex 3&#226;&#710;&#8217;12% Bis-Tris gels 150 V               performed using Census.45 sample manually 20 min dark blue cathode bu&#239;&#172;&#8364;er followed 45 min              median-normalized prior analysis. light blue cathode bu&#239;&#172;&#8364;er. Gels washed 20 min              Quantitative Analysis Metabolically Labeled <br /> equilibration bu&#239;&#172;&#8364;er (80 mL ddH20, 10 mL 20&#195;&#8212; transfer              Proteomes. Peptides Mascot scores 30 peak <br /> bu&#239;&#172;&#8364;er, 10 mL methanol, 100 &#206;&#188;L NuPage antioxidant),            correlations 0.8 discarded analysis. Peptide <br /> transferred Immobilon-FL membranes (Millipore) using              ratios averaged achieve single ratio protein <br /> iBlot dry blotting (Invitrogen). Membrane allowed            identi&#239;&#172;?ed. Proteins showing 14N/15N reciprocal ratios dry 30 min. Immunoblotting performed according              &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8 identi&#239;&#172;?ed experimental pair standard protocol using rabbit anti-CBP IRDye goat anti-            subjected manual veri&#239;&#172;?cation. Peptides blasted rabbit (LI-COR). Membranes visualized LI-COR                TAIR10 protein database (www.arabidopsis.org) Odyssey infrared-imaging                                         determine uniqueness, nonunique peptides discarded. <br />    Metabolic Labeling Mass Spectrometry. Seedlings                  Reciprocal protein ratios proteins identi&#239;&#172;?ed metabolically labeled mass spectrometry experiments            peptides 14N/15N 15N/14N tissue pairs tested <br /> grown 1&#195;&#8212; M&amp;S micronutrient solution supplemented                statistical signi&#239;&#172;?cance using sample, tailed <br /> 1.5 mM CaCl2, 0.75 mM MgSO4, 0.625 mM KH2PO4, 2.3 mM                    student&#226;&#8364;&#8482;s t test p &#226;&#8240;&#164; 0.05. Protein-abundance changes <br /> MES salts, 29.2 mM sucrose. Natural abundance (14N)              strengthened protein identi&#239;&#172;?ed heavy (15N) NH4NO3 ( 0.825 g) 14N 15N KNO3 (0.96                 biological replicates direction change g) added 1 L media, pH adjusted 5.7           replicates agreed. Proteins showing opposite abundance KOH. Fourteen milligrams seeds grown 75 mL               changes reciprocal pairs discarded. sterile media supplemented ampicillin. Seeds                  Mass Spectrometry IgG-Puri&#239;&#172;?ed Proteins. IgG- <br /> strati&#239;&#172;?ed dark 2 nights 4 &#194;&#176;C grown         puri&#239;&#172;?ed, TEV-eluted protein samples TCA/acetone- <br /> constant light gentle shaking 9 days. Seedlings           precipitated, pellets suspended 7.4 &#206;&#188;L 8 M <br /> spun-dry weighed, equal fresh weights heavy              <span id='am-133' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-134' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span><span id='am-135' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>urea</span>, 50 mM NH4HCO3 (pH 8.5), 1 mM Tris-HCl 5 <br /> light seedlings combined. Microsome plasma                     min subsequently diluted 30 &#206;&#188;L 1.25 &#206;&#188;L 25 mM <br /> membrane puri&#239;&#172;?cation proceeded  Methanol/chloro-               DTT, 2.5 &#206;&#188;L MeOH, 18.75 &#206;&#188;L 25 mM NH4HCO3 (pH 8.5) <br /> form extraction performed plasma membrane                    reduction alkylation. Samples incubated 50 &#194;&#176;C <br /> soluble fractions previously described.44 Protein                15 min cooled ice room temperature. IAA (1.5 &#206;&#188;L <br /> suspended 8 M urea/50 mM NH4HCO3 gentle                         55 mM) added, samples incubated room <br /> sonication diluted 1 M urea using 50 mM NH4HCO3.               temperature dark 15 min. reaction quenched <br /> BCA protein assay performed determine protein                    4 &#206;&#188;L 25 mM DTT. microliters trypsin (10 ng/ <br /> concentration. Aliquots samples reduced 5 mM               &#206;&#188;L Trypsin Gold, Promega Corp., 25 mM NH4HCO3) DTT &#226;&#710;&#188;50 &#194;&#176;C 45 min, alkylated 15 mM                          6.5 &#206;&#188;L 25 mM NH4HCO3 (pH 8.5) added &#239;&#172;?nal <br /> iodoacetamide room temperature 45 min, digested              volume 50 &#206;&#188;L. Digestion proceeded 2 h 42 &#194;&#176;C <br /> overnight 37 &#194;&#176;C 50:50 mixture trypsin/lys-C (total         followed additional 5 &#206;&#188;L trypsin &#239;&#172;?nal enzyme/ <br /> protein/enzyme ratio 100:1). Samples acidi&#239;&#172;?ed 0.3%            substrate ratio 1:40. Digestion proceeded overnight 37 &#194;&#176;C. <br /> TFA desalted using Waters tC18 sep-pak columns                      Digestions quenched 2.5% TFA 0.3% &#239;&#172;?nal <br /> according manufacturer&#226;&#8364;&#8482;s instructions. drying          concentration, 8 &#206;&#188;L (&#226;&#710;&#188;700 ng) loaded nanoLC&#226;&#710;&#8217; <br /> samples, peptides solubilized LC&#226;&#710;&#8217;MS quality 0.1%                MS/MS analysis. Microsomal phase plasma mem- <br /> <span id='am-128' about='obo:IMR_0200110' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-129' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-130' property="oboInOwl:hasDbXref" content="KEGG:C00058" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:id" content="IMR:0200110" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="Formic acid" datatype="xsd:string"></span>formic acid</span>. Approximately 5 &#206;&#188;g samples loaded               brane-enriched fractions delipidated methanol/ <br /> analysis using Agilent 1100 LC inline LTQ             chloroform separation prior digestion. micro- <br /> Orbitrap XL. Peptides separated house packed            grams microsomal protein 10 &#206;&#188;g plasma membrane- <br /> reversed-phase analytical column (Magic-C18, 200 &#195;&#8230;, 3 &#206;&#188;m,               enriched fractions precipitated methanol/chloroform/ <br /> Michrom Bioresources, ) postloading mobile phase             water (4:1:1), protein pellet organic aqueous <br /> elution gradient 0&#226;&#710;&#8217;40% B 3 h   0.1% formic acid; B,           interphase washed neat methanol. protein dried <br /> 95% CH3CN 0.1% formic acid) 200 nL/min. Data-                    digested 100 &#206;&#188;L total volume 1:40 <br /> dependent analysis performed LTQ Orbitrap XL                 ratio trypsin/substrate. Peptides analyzed nanoLC&#226;&#710;&#8217; <br /> using FTMS preview mode MS1 scan acquisition                MS/MS using Agilent 1100 nano&#239;&#172;&#8218;ow (Agilent <br />                                                                   568                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                    Article <br />  <br /> Technologies) connected linear hybrid ion trap-orbitrap            test AHA1 transgene fused tandem a&#239;&#172;&#402;nity- <br /> mass spectrometer (LTQ-Orbitrap XL, Thermo Fisher                       puri&#239;&#172;?cation (TAP) tag rescue embryonic lethality. Scienti&#239;&#172;?c) equipped nanoelectrospray ion source.                 TAP-tag construct optimized use plants containing HPLC performed house fabricated 15 cm C18                  tandem protein domains separated calmodulin- <br /> column packed MAGIC C18AQ 3 &#206;&#188;m particles (Michrom                  binding protein domain (CBP) TEV protease site BioResources, ) laser-pulled tip (P-2000, Sutter               ligated 5&#226;&#8364;&#178; end genomic AHA1 open reading frame32 <br /> Instrument) using 360 &#206;&#188;m &#195;&#8212; 75 &#206;&#188;m fused silica tubing. Sample            (Supporting Information Figure S1). <br /> loading desalting 15 &#206;&#188;L/min using trapping              Arabidopsis (Col-0, aha1/aha1;AHA2/aha2) trans- <br /> column line autosampler (Zorbax 300SB-C18, 5                formed Ntapi:gAHA1 clone Agrobacterium- <br /> &#206;&#188;M, 5 &#195;&#8212; 0.3 mm, Agilent Technologies). Peptide elution                  mediated &#239;&#172;&#8218;oral dip, successful transformation events solvent 0.1% formic acid water solvent B 0.1%          selected hygromycin media.40,47 Plants allowed self, <br /> formic acid 95% acetonitrile water. &#239;&#172;&#8218;ow rate             PCR genotyping T2 population identi&#239;&#172;?ed aha1/ <br /> 200 nL/min. column loaded, desalted,                        aha1;aha2/aha2 double-knockdown plants carrying equilibrated 20 min 1% solvent B followed                  Ntapi:gAHA1 transgene. recovered independent rescue <br /> increase 40% solvent B 195 min, ramped 60% solvent           lines expressing Ntapi:gAHA1 (Ntapi:gAHA1a Ntapi:gA- <br /> B 20 min, increased 100% solvent B 5 min, held           HA1b; TAPa TAPb). band approximately 125 kDa 100% solvent B additional 3 min. column            detected western blots denaturing SDS protein gels <br /> equilibrated 1% B 30 min. MS/MS data acquisition             Ntapi:gAHA1 microsomal fractions probed protein  <br /> gathered data-dependent mode follows: MS survey scans           Following phase partitioning, majority protein m 300 2000 collected centroid mode               staining enriches plasma membrane phase (Figure 1). resolving power 100 000. MS/MS spectra collected &#239;&#172;?ve abundant signals survey scan. Dynamic <br /> exclusion 40 s precursors 0.55 m 1.05 <br /> m previously selected precursors employed increase dynamic range maximize peptide <br /> identi&#239;&#172;?cations. Singly charged ions ions charge state assigned rejected consideration MS/MS. <br />    Quantitative Analysis. <span id='am-106' about='rdfs:label' typeof='owl:Thing'>Label</span>-free raw data &#239;&#172;?les converted Mascot generic format (mgf) initial open- <br /> source mzXML format conversion using Trans-Proteomic <br /> Pipeline (<span id='am-107' about='obo:IMR_0200378' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="TPP" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00068" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:id" content="IMR:0200378" datatype="xsd:string"></span>TPP</span>) software suite version 4.4. Mgf &#239;&#172;?les searched Arabidopsis thaliana TAIR9 database decoy reverse entries common contaminants included total 27 279 Arabidopsis protein-coding genes 62 <br /> 523 total entries using house Mascot search engine 2.2.07 <br /> (Matrix Science) variable methionine oxidation asparagine glutamine deamidation. Peptide mass tolerance set 20 ppm fragment mass 0.8 Da. Protein <br /> annotations, identi&#239;&#172;?cation signi&#239;&#172;?cance, spectral-based <br /> quanti&#239;&#172;?cation using Sca&#239;&#172;&#8364;old software (version <br /> 3.6.3, Proteome Software , Portland, ) Protein                   Figure 1. Detection Ntapi:AHA1 total microsomes (Mic), <br /> identi&#239;&#172;?cations accepted established              plasma membrane (PM), intracellular membranes (non-PM). Size <br /> greater 95% probability contained                 markers kDa. Upper panel coomassie stained, lower panel <br /> peptides using 1% false discovery rate. Protein probabilities         western blot probed HRP-conjugated rabbit anti-goat IgG. assigned using Protein Prophet.46 Proteins containing <br /> nonunique peptides assigned MS/MS <br /> analysis grouped satisfy principles                addition major band 110 130 kDa <br /> parsimony. Metabolically labeled raw data acquired,                 markers, commonly observe slightly higher smaller <br /> converted, searched  Mascot search results            &#226;&#710;&#188;70&#226;&#710;&#8217;80 kDa bands western blots. Smaller bands transformed pepXML format using TPP software              partial proteolysis C-terminal regulatory domain, <br /> house-generated false-discovery-rate &#239;&#172;?lter. FDR peptide             larger bands alterred migration resulting <br /> identi&#239;&#172;?cation set 10% low-quality MS/MS spectra          bound <span id='am-10' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-13' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span>lipid</span> detergent molecules. allowed available manual inspection desired.              Previous studies Nicotiana plumbaginifolia plasma <br /> Relative quanti&#239;&#172;?cation 14N/15N peptide ratios performed          membrane proton pump, PMA2, indicate protein using TPP tools Census software (Scripps Research                   present membrane hexameric complexes, especially <br /> Institute). <br />  <br /> &#226;&#8211;&#160; <br />                                                                         activation fusicoccin subsequent 14-3-3 protein <br />                                                                         binding.48,49 determine molecular weight non- <br />    RESULTS                                                              denatured Ntapi:AHA1, puri&#239;&#172;?ed plasma membranes run <br />   Complementation TAP-Tagged AHA1 Trans-                         blue native gels probed rabbit anti-CBP antibody. <br /> gene. lab previously shown wild-type AHA1               observed bands migrating close 242 720 <br /> AHA2 transgenes successfully rescue aha1/aha1;aha2/aha2                 kDa markers. Proteins migrate higher expected double-knockdown embryo lethality.20 wished             blue native gels based detergents dyes used bound <br />                                                                   569                         dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  <br /> lipids;  bands likely correspond dimeric                   Ntapi:gAHA1b progeny performed hexameric complexes observed previously (Figure 2).48                        genotype transformed seedling aha1&#226;&#710;&#8217;6;aha1&#226;&#710;&#8217;6/ <br />                                                                              aha2&#226;&#710;&#8217;4;aha2&#226;&#710;&#8217;4. <br />                                                                                  Ntapi:gAHA1 Plants Exhibit Reduced Root Length. <br />                                                                              grown vertically standard half-strength M&amp;S 1% <br />                                                                              (w/v) sucrose nutrient media, observed approximately 20% <br />                                                                              reduced root length Ntapi:gAHA1 plants compared wild- <br />                                                                              type aha1/aha2 double knockdown mutants rescued                                                                              wild-type AHA1 transgene (gAHA1) (Figure 3A). looked <br />                                                                              additional growth changes time &#239;&#172;?rst bolting <br />                                                                              (Figure 3B) plant growth soil (Figure 3C), did <br />                                                                              observe readily apparent di&#239;&#172;&#8364;erences. Work lab                                                                              previously shown aha2 single-knockdown mutants exhibit <br />                                                                              root-growth phenotypes seedlings grown stress <br />                                                                              conditions (high external pH, high external potassium,                                                                              toxic cations) alter rely proton motive force.20,37 <br />                                                                              determine e&#239;&#172;&#8364;ects TAP-tagged AHA1 protein                                                                              proton motive force, grew Ntapi:gAHA1 plants                                                                              presence gentamicin lithium chloride.                                                                              conditions, aha2 knockdowns enhanced root growth <br />                                                                              compared wild-type plants reduced proton <br />                                                                              motive force energizes reduced uptake toxic <br />                                                                              compounds. Ntapi:gAHA1 lines similar root-growth <br />                                                                              phenotypes aha2 single knockdowns rescued Figure 2. micrograms Ntapi:gAHA1 wild-type plasma                gAHA1 transgene (Figure 3D). <br /> membrane protein run blue native gels. Proteins                      Proteome Analysis Ntapi:gAHA1 Plants. transferred membranes probed rabbit anti-CBP antibodies.         ultimate goal using Ntapi:gAHA1 plants planta <br />                                                                              physiological investigation AHA1, wished establish <br />    e&#239;&#172;&#402;ciency aha1/aha2 double-knockdown rescue                      closely transgenic lines resembled wild-type plants Ntapi:gAHA1 assessed comparing genotypes                       merely visible phenotypes. end, compared aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1a progeny                            plasma membrane soluble proteomes Ntapi:gAHA1 ratios expected basis Mendelian segregation                plants wild-type gAHA1 rescue lines using single unlinked insertional events (Table 1).                metabolically labeled mass spectrometry approach determine <br />                                                                              expressing Ntapi:AHA1 proteomic consequences. <br /> Table 1. Progeny aha1&#226;&#710;&#8217;6/aha1&#226;&#710;&#8217;6;AHA2/aha2&#226;&#710;&#8217;4                                Equal wet weights seedlings grown natural-abundance <br /> Parental Plant Carrying Ntapi:gAHA1 Transgene                            (14N) isotopically heavy nitrogen (15N) liquid media                                   plants    experiment    experiment         mixed prior protein extraction. Subsequent downstream <br />             genotype               obsa         Ab            Bc             sample processing mass spectrometry analysis identical <br />     AHA2/aha2&#226;&#710;&#8217;4                    8        18.33         4.58               paired tissues, relative abundance peptide     AHA2/AHA2                      5        9.17          2.29               original tissue determined calculating ratio     aha2&#226;&#710;&#8217;4/aha2&#226;&#710;&#8217;4                  0        0             0                  14 <br />                                                                                 N/15N peptide intensity.52 ratio 1.0 indicates     AHA2/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1        55       41.25         51.56              di&#239;&#172;&#8364;erence peptide abundance metabolically labeled <br />     AHA2/AHA2;Ntapi:gAHA1          31       20.625        25.78              tissue pairs; ratio greater 1.0 indicates increased peptide <br />     aha2&#226;&#710;&#8217;4/aha2&#226;&#710;&#8217;4;Ntapi:gAHA1      11       20.625        25.78              abundance 14N-grown tissue compared 15N-grown <br />     total                          110      110           110                tissue; ratio 1.0 indicates decreased peptide <br />                                             p = 0.0002d   p = 0.004          abundance 14N-grown tissue   Number plants observed PCR genotyping. bNumber plants                generated 14N/15N reciprocal 15N/14N tissue pairs expected Mendelian segregation single transgene insertion.         liquid-grown, 9 day old wild-type/wild-type, wild-type/gAHA1, <br /> c <br />   Number plants expected Mendelian segregation unliked          wild-type/Ntapi:gAHA1a plants performed biological <br /> transgene insertional events. dResult chi-squared goodness &#239;&#172;?t          duplicates experimental pair. Proteomic analysis tests observed expected genotypes using 4 degrees freedom.                                                                     soluble plasma membrane proteomes performed. <br />                                                                              number unique peptides proteins identi&#239;&#172;?ed                                                                              tandem mass spectrometry metabolically labeled presence multiple insertions complex transgenic loci              tissue pair summarized Supporting Information Tables S1 <br /> following Agrobacterium-mediated transformation com-                      S2. Approximately half protein identi&#239;&#172;?cations mon.50,51 Progeny ratios di&#239;&#172;&#8364;ered signi&#239;&#172;?cantly                      samples based single peptide discarded <br /> expected scenarios based chi-squared analysis                    analysis (Supporting Information Table S3). closely resembled scenario involving unlinked transgene                proteins identi&#239;&#172;?ed peptide, peptide ratios <br /> insertional events. reduction number rescued                     averaged achieve single 14N/15N protein ratio. Protein <br /> progeny expected number observed                   ratios plasma membrane (Figure 4) soluble (Figure 5) altered expression, translation, tra&#239;&#172;&#402;cking, biochemical            fractions plotted ratios 14N/15N reciprocal <br />                                                                              15 <br /> activity fusion protein. similar segregation analysis               N/14N genotype pairs. <br />                                                                        570                        dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                           Article <br />  <br />  <br />  <br />  <br /> Figure 3. Growth Ntapi:gAHA1 plants.   Vertical root growth (mm) plants half-strength M&amp;S 1% (w/v) sucrose media measured 7 10 days. Bars indicate standard error, tailed, sample Student t tests used determine statistical di&#239;&#172;&#8364;erence (p &lt; <br /> 0.005, n &#226;&#8240;&#165; 12). (B) Sixteen seedling genotype transferred soil monitored visually bloting. (C) Representative experiment <br /> shown. Vegetative growth representative plants genotype 32 days. (D) Vertical root growth (mm) 4 day old wild-type, <br /> Ntapi:gAHA1a, Ntapi:gAHA1b seedlings transferred stress media 5 days. Bars indicate standard error, Student t tests performed (p &lt; 0.05, n = 7). (E) Vertical root growth (mm) 4 day old wild-type, Ntapi:gAHA1a, gAHA1 seedlings transferred stress media 5 days. <br />  <br />    Following manual veri&#239;&#172;?cation proteins showing 14N/15N                   rescue lines observable phenotypes standard reciprocal ratios &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8, proteins met             laboratory growth conditions. wild-type/Ntapi:gAHA1 tissue <br /> required criteria  Experimental Procedures) 14Nwild-                   pairs, identi&#239;&#172;?ed plasma membrane proteins type/15Nwild-type reciprocal plasma membrane soluble                   di&#239;&#172;&#8364;erentially abundant reciprocal experiments protein fractions. wild-type/gAHA1 tissue pairs, AHA1                 20% abundance change experiment (Table 3). <br /> itself showed statistically signi&#239;&#172; reciprocal abundance               majority proteins identi&#239;&#172;?ed decreased abundance <br /> changes biological replicates (Table 2). Despite                  Ntapi:gAHA1 seedlings compared wild-type,  reduced levels AHA1 protein membrane, gAHA1-                        phosphate transporter ATPT2 AHA1 itself, increased <br />                                                                         571                          dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  <br />  <br />  <br />  <br /> Figure 4. Reciprocal plasma membrane protein ratios metabolically labeled tissue pairs. Wild-type (WT), wild-type AHA1-rescued (gAHA1), Ntapi:gAHA1-rescued plants grown natural-abudance (14N) heavy isotope (15N) nitrogen media. Equal fresh weights   wild-type/ <br /> wild-type, (B) wild-type/gAHA1, (C) wild-type/Ntapi:gAHA1 genotype pairs biologically replicated experiments (upper lower <br /> plots) mixed prior homogenization, sample processing, MS analysis. Protein ratios (14N/15N) calculated averaging median- <br /> normalized peptide ratios determined Census, minimum MASCOT score 30. Calculated protein ratios reciprocally labeled tissue <br /> pairs plotted proteins identi&#239;&#172;?ed peptides samples. number proteins showing reciprocal ratios &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8 sample indicated. <br />  <br />  <br />  <br />  <br /> Figure 5. Reciprocal soluble protein ratios metabolically labeled tissue pairs. Wild-type (WT), wild-type AHA1-rescued (gAHA1), Ntapi:gAHA1-rescued plants grown natural-abudance (14N) heavy isotope (15N) nitrogen media. Equal fresh weights   wild-type/ <br /> wild-type, (B) wild-type/gAHA1, (C) wild-type/Ntapi:gAHA1 genotype pairs biologically replicated experiments (upper lower <br /> plots) mixed prior homogenization, sample processing, MS analysis. Protein ratios (14N/15N) calculated averaging median- <br /> normalized peptide ratios determined Census, minimum MASCOT score 30. Protein ratios reciprocally labeled tissue pairs plotted proteins identi&#239;&#172;?ed peptides samples. number proteins showing reciprcal ratios &#226;&#8240;&#165;1.2 &#226;&#8240;&#164;0.8 sample indicated. <br />  <br />  <br />                                                                       572                         dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                                          Article <br />  <br /> Table 2. Manually Veri&#239;&#172;?ed Protein Abundance Changes gAHA1 vs Wild-Type Plasma Membrane Fractions <br />                                                                             experiment 1                                                                 experiment 2 <br />                                                    14 <br />         accession              annotation               NgAHA1/ 15NWT                  14 <br />                                                                                            NWT/ 15NgAHA1                      14 <br />                                                                                                                                    NgAHA1/ 15NWT                     14 <br />                                                                                                                                                                         NWT/ 15NgAHA1 <br />       AT2G18960                  AHA1                     0.20 (5                         5.64 (6)                                     0.15 (3)                         4.67 (8) <br />                                                                                             p = 0.0003b                                                                   p = 0.001 <br />       AT5G44610                  MAP18                    1.33 (3)                          0.74 (2)  number unique peptides ratios averaged calculate total protein ratio indicated parentheses. bTwo sample, tailed <br /> Student t tests used determine statistical signi&#239;&#172;?cance reciprocal protein ratios unique peptides detected sample. <br />  <br /> Table 3. Manually Veri&#239;&#172;?ed Protein Abundance Changes Ntapi:gAHA1 vs Wild-Type Plasma Membrane Fractions <br />                                                                experiment 1                                     experiment 2                                        experiment 3 <br />                                             14            15          15        14              14         15          15           14              14         15          15 <br />      accession             annotation            NTAPa/ NWT            NWT/ NTAPa                    NTAPa/ NWT             NWT/ NTAPa               NTAPb/ NWT                 NWT/14NTAPb <br />                                                                    AT2G18960       AHA1                          1.06 (6)                 0.79 (9)                   1.93 (6)              0.62 (7)                     1.34 (9)               0.70 (5) <br />                                                                            p = 0.01b                                        p = 1 &#195;&#8212; 10&#226;&#710;&#8217;5                                        p = 1 &#195;&#8212; 10&#226;&#710;&#8217;4 <br />     AT2G38940       ATPT2                         1.69 (4)                 0.88 (3)                                         0.76 (2)                     1.08 (4)               0.69 <br />                                                                            p = 0.05 <br />     AT3G01290       band 7                        1.53 (6)                 0.69 (8)                   1.17 (4)              1.05 (6)                     1.74 (7)               0.47 (4) <br />                                                                            p = 6 &#195;&#8212; 10&#226;&#710;&#8217;9                                                                                         p = 2 &#195;&#8212; 10&#226;&#710;&#8217;6 <br />     AT1G66970       SVL2                          0.84 (7)                 1.37 (10)                  0.66 (8)              1.57 (8)                     0.80 (8)               0.95 (8) <br />                                                                            p = 2 &#195;&#8212; 10&#226;&#710;&#8217;5                                     p = 4 &#195;&#8212; 10&#226;&#710;&#8217;7 <br />     AT1G72150       PATL1                         0.94 (7)                 1.55 (7)                   0.72 (6)              2.47 (8)                     0.78 (10)              1.07 (11) <br />                                                                            p = 1 &#195;&#8212; 10&#226;&#710;&#8217;5                                     p = 6 &#195;&#8212; 10&#226;&#710;&#8217;6                                        p = 0.03 <br />     AT3G16450       jacalin family                0.96                     2.20 (4)                                         0.64                         0.75 (4)               1.38 (4) <br />                                                                                                                                                                                 p = 1 &#195;&#8212; 10&#226;&#710;&#8217;4 <br />     AT3G16460       jacalin family                0.65 (5)                 1.36 (8)                   0.91 (6)              1.35 (7)                     0.80 (11)              1.45 (16) <br />                                                                            p = 5 &#195;&#8212; 10&#226;&#710;&#8217;6                                     p = 8 &#195;&#8212; 10&#226;&#710;&#8217;4                                        p = 2 &#195;&#8212; 10&#226;&#710;&#8217;11 <br />     AT3G05900       neuro&#239;&#172;?lament protein-         0.64 (10)                1.21 (11)                  0.91 (4)              1.29 (4)                     0.70 (8)               1.44 (10) <br />                       related <br />     AT4G20260       DREPP family                  0.95 (6)                 1.24 (8)                   0.80 (3)              1.71 (6)                     0.93 (7)               1.15 (5) <br />                                                                            p = 0.009                                        p = 0.008                                           p = 0.006  number unique peptides ratios averaged calculate total protein ratio indicated parentheses. bTwo sample, tailed <br /> Student t tests used determine statistical signi&#239;&#172;?cance reciprocal protein ratios unique peptides detected sample. experiment performed comparing Ntapi:gAHA1b wildtype plants distinguish e&#239;&#172;&#8364;ects speci&#239;&#172;?c Ntapi:AHA1 <br /> expression versus transformation artifacts. <br />  <br /> abundance. known cellular roles identi&#239;&#172;?ed proteins                                        use washing conditions containing 0.1% SDS additional <br /> include membrane transport (ATPT2), cell wall structure                                          salts reducing background protein contamination (SVL2), vesicle tra&#239;&#172;&#402;cking (PATL1), microtubule structure                                     retaining speci&#239;&#172;?cally bound Ntapi:AHA1. organization (DREPP).53&#226;&#710;&#8217;58                                                                       Identi&#239;&#172;?cation Ntapi:AHA1-Copurifying Proteins. <br />    TAP-Tag-Mediated Puri&#239;&#172;?cation AHA1. goal                                           establishing speci&#239;&#172;?c IgG puri&#239;&#172;?cation Ntapi:AHA1, using Ntapi:gAHA1 plants facilitate e&#239;&#172;&#402;cient isoform-                                       pursued identi&#239;&#172;?cation Ntapi:AHA1-copurifying proteins <br /> speci&#239;&#172;?c puri&#239;&#172;?cation AHA1 Arabidopsis tissue,                                          using a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry approach. <br /> investigated ability speci&#239;&#172;?cally purify Ntapi:AHA1                                   Ntapi:gAHA1 wild-type plants grown 14N IgG puri&#239;&#172;?cation resin. developed modi&#239;&#172;?ed tandem                                             15 <br />                                                                                                     N liquid media combined prior homogenization a&#239;&#172;&#402;nity-puri&#239;&#172;?cation protocol purify Ntapi:AHA1 using                                      described previously. technical puri&#239;&#172;?cation &#239;&#172;?rst a&#239;&#172;&#402;nity-puri&#239;&#172;?cation step limit possibility losing <br />                                                                                                  replicates performed 14NNtapi:gAHA1/15Nwild-type <br /> protein interactors increased sample handling.60 Puri&#239;&#172;?ca- <br />                                                                                                  reciprocal 14Nwild-type/15NNtapi:gAHA1 tissue pairs. <br /> tion Ntapi:AHA1 plasma membrane fractions IgG <br />                                                                                                  Eluted proteins subjected tandem mass spectrometry <br /> resin demonstrated western blot Figure 6. Including <br /> 0.1% SDS IgG resin washing bu&#239;&#172;&#8364;er decreases                                                analysis, proteins showing reciprocal 14N/15N ratios <br /> nonspeci&#239;&#172;?cally bound proteins retaining high Ntapi                                       deviating 1.0 wild-type Ntapi:gAHA1 <br /> HA1 antibody reactivity eluted fractions. Using normalized                                    seedlings subjected manual analysis. proteins <br /> spectral counts relative quanti&#239;&#172;?cation abundance                                  showed increased puri&#239;&#172;?cation Ntapi:gAHA1 plants <br /> Ntapi:AHA1 membrane puri&#239;&#172;?ed samples, protein                                          compared wild-type including additional AHA isoforms <br /> enrichment upper-phase fractions IgG resin                                            (Table 5). enrichment AHA9 AHA11 approximately 3.5-fold mild washing conditions (Table                                      Ntapi:gAHA1 versus wild-type protein extracts validated 4). Addition 150 mM NaCl reduces MS identi&#239;&#172;?cation                                     searching raw data peaks corresponding unique contaminating proteins results loss                                            peptides isoform (Figure 7). high <br /> Ntapi:AHA1 staining western blots TEV-eluted fractions                                     degree sequence identity isoforms, (Supporting Information Figure S2). routinely                                       AHA peptides discarded analysis;  list <br />                                                                                           573                                      dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                       Article <br />  <br />                                                                                          proteins,61 wished investigate treatment vivo <br />                                                                                          compounds known hypothesized cause AHA <br />                                                                                          activation e&#239;&#172;&#8364;ect identi&#239;&#172;?cation NTapi:AHA1-copurify- <br />                                                                                          ing proteins. fungal phytotoxin fusicoccin (FC) binds                                                                                          stabilizes AHA phosphorylated C-terminus 14-3-3 <br />                                                                                          protein complex, causing permanent activation pump.62 <br />                                                                                          hypothesized treating liquid-grown seedlings FC <br />                                                                                          increase mass spectrometry identi&#239;&#172;?cation 14-3-3 <br />                                                                                          proteins IgG-puri&#239;&#172;?ed eluates, failed identify                                                                                          nonelicited copuri&#239;&#172;?cation experiments (Supporting Informa- <br />                                                                                          tion Table S5). day old Ntapi:gAHA1 liquid-grown 14N + <br />                                                                                          FC/15N &#226;&#710;&#8217; FC reciprocally treated pairs subjected                                                                                          combined homogenization, puri&#239;&#172;?cation, MS analysis                                                                                          previously described. Treatment seedlings 1 &#206;&#188;M FC                                                                                          30 min increased puri&#239;&#172;?cation AHA11 14-3-3 <br /> Figure 6. Puri&#239;&#172;?cation Ntapi:AHA1 visulized western blot.                       protein isoforms including GRF1, GRF2, GRF3 (Table 6). micrograms solubilized plasma membrane protein                            Auxin regulates plasma membrane proton pump activity incubated IgG puri&#239;&#172;?cation resin, washed times 0.1%                       phosphorylation, protein kinases phos- <br /> SDS resuspension bu&#239;&#172;&#8364;er followed times AcTEV,                        photases responsible identi&#239;&#172;?ed.63 Additionally, <br /> eluted overnight AcTEV protease. Eluted proteins acetone-                      inhibitory e&#239;&#172;&#8364;ect auxin root growth established <br /> precipitated resuspended directly Laemmli sample bu&#239;&#172;&#8364;er                       likely involves regulation proton pump activity.64 reducing agent. microgram nonplasma membrane (non-PM) <br />                                                                                          seedlings treated 1 &#206;&#188;M indole-3-acetic-acid 15 plasma membrane (PM) wild-type Ntapi:gAHA1 protein half TEV-eluted proteins loaded 4&#226;&#710;&#8217;12% NuPAGE gels.                            min, observed decreased copuri&#239;&#172;?cation protein kinase left panel silver stained total protein, right panel             PHOT1 (Table 6). <br />  <br />                                                                                          &#226;&#8211;&#160; <br /> western blot probed rabbit anti-CBP followed LI-COR IRDye <br /> goat anti-rabbit antibody. Size-marker values kDa.                                     DISCUSSION <br /> Table 4. Successive Puri&#239;&#172;?cation Ntapi:AHA1 Liquid-                                functional rescue aha1/aha2 <br /> Grown Seedlings                                                                          knockdown embryonic lethality Arabidopsis stable <br />                                                                                          transformation N-terminal TAP-tagged genomic <br />                                           spectral                  fold                 AHA1 transgene, used a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass <br /> step                  fraction             Ctsa      NSAFb      enrichmentc <br />                                                                                          spectrometry approach copurify candidate Ntapi:AHA1- <br />      1   microsomes                         75       0.0012                              interacting proteins. ability identify proteins interacting <br />      2   non-PM membranes                   63       0.0010                              AHA1 planta particular strength approach <br />      3   plasma membranes                   273      0.0051        4.25 <br />                                                                                          feasible heterologous methods used identify <br />      4   IgG bound, TEV-eluted              99       0.0175        14.6 <br />            (mild washd)                                                                  protein interactions, including yeast hybrid split <br />      5   IgG bound, TEV-eluted              100      0.5639        470                   ubiquitin systems. <br />            (stringent washe)                                                                observed slight reduction number   Spectral counts representative sample determined &#226;&#8364;&#339;number                    endogenous double knockdown, transgene-rescued progeny assigned spectra&#226;&#8364;? Sca&#239;&#172;&#8364;old. bNormalized spectral abundance                          expected numbers based Mendelian segregation, <br /> factor calculated according method Zhang et al.59                              result non-WT transgene expression c <br />   Enrichment Ntapi:AHA1 microsomal samples determined                         altered translation, tra&#239;&#172;&#402;cking, activity tagged protein, increasing NSAF values. dTwice 100&#195;&#8212; volume 1% NP40,                         resulting embryonic lethality. Western blots 100&#195;&#8212; volume 1% NP40, 0.01% SDS. eThree times 100&#195;&#8212;                              phase separated membranes showed Ntapi:AHA1 <br /> volume 0.1% SDS, 150 mM NaCl. <br />                                                                                          enriched plasma membrane fractions compared                                                                                          nonplasma membrane fractions, expected. result identi&#239;&#172;?ed AHA peptides 14N/15N ratios                               alleviated concerns TAP tag deleteriously <br /> included Supporting Information (Table S4).                                       a&#239;&#172;&#8364;ecting proper maturation tra&#239;&#172;&#402;cking Ntapi:AHA1 <br />    Response Ntapi:AHA1-Copurifying Proteins                                     protein, rule possibility Vivo Treatment. autoinhibited state AHA                                portion protein mistra&#239;&#172;&#402;cked misfolded. <br />  <br /> Table 5. Ntapi:AHA1-Copurifying Proteins Reciprocal Metabolically Labeled Tissue Pairs <br />           accession                   annotation                    unique peptidesa                14 <br />                                                                                                          NTAP/15NWT ratiob                   14 <br />                                                                                                                                                   NWT/15NTAP ratioc <br />          AT2G18960               AHA1                                          7                              14.10                                    0.03 <br />          AT1G80660               AHA9                                          1                              24.34                                    0.04 <br />          AT5G62670               AHA11                                         2                              1.73                                     NDd <br />          AT4G10480               unknown/&#206;&#177;-NAC                                 2                              1.35                                     0.71 <br />          AT1G62480               unknown/Ca2+-binding                          2                              1.32                                     0.72 <br />   Peptides BLASTed agains TAIR10 database determine uniqueness. bPeptide ratios IgG-puri&#239;&#172;?ed, TEV-eluted proteins 14 <br />   NNtapi:gAHA1/15NCol-0 WT metabolically labeled, liquid-grown tissue pairs. cPeptide ratios IgG-puri&#239;&#172;?ed, TEV-eluted proteins reciprocally labeled tissue pairs. dPeptide intensity 14N-grown tissue low detection raw data Mascot set 14N search mode. <br />  <br />                                                                                    574                           dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                Article <br />  <br />  <br />  <br />  <br /> Figure 7. Peptides unique AHA isoforms enrich Ntapi:AHA1 TAP-tag-mediated puri&#239;&#172;?cation. Metabolically labeled <br /> 14 <br />   NNtapi:gAHA1/15Nwild-type (red) reciprocal 14Nwild-type/15NNtapi:gAHA1 (blue) plasma membrane pairs incubated IgG resin, <br /> washed, eluted overnight TEV protease. raw data inspected manually peaks corresponding indicated unique peptides AHA1, AHA9, AHA11. <br />  <br /> Table 6. Response Ntapi:AHA1-Copurifying Proteins Vivo Treatments <br />          accession                 annotation               treatment                 14 <br />                                                                                            NTAP+/15NTAP                            14 <br />                                                                                                                                           NTAP&#226;&#710;&#8217;/15NTAP+b <br />                                                                    c <br />        AT2G18960                   AHA1                       FC                              0.93 (8)                                       0.98 (6) <br />                                                                                               p = 0.74 <br />                                                               IAAd                            1.00 (5)                                       1.14 (7) <br />                                                                                               p = 0.52 <br />        AT5G62670                   AHA11                      FC                              1.10 (3)e                                      0.67 (3) <br />                                                                                               p = 0.03f <br />        AT4G09000                   GRF1                       FC                              1.41(4)                                        0.95 (4) <br />                                                                                               p = 0.006 <br />        AT1G78300                   GRF2                       FC                              1.13 (3)                                       0.80 (3) <br />                                                                                               p = 0.003 <br />        AT5G38480                   GRF3                       FC                              1.25 (4)                                       0.76 (5) <br />                                                                                               p = 0.003 <br />        AT3G45780                   PHOT1                      IAA                             0.87 (2)                                       1.17 (5) <br />        AT2G26080                   ATGLDP2                    IAA                             1.23 (3)                                       0.94 (3) <br />                                                                                               p = 0.02 <br />        AT3G23990                   HSP60                      IAA                             1.21 (4)                                       0.81 (2)  Liquid-grown Ntapi:gAHA1 seedlings: 14N seedlings treated, 15N seedlings mock treated. bReciprocal experiment: 14N seedlings mock treated, 15N <br /> seedlings treated. cFusicoccin treatment: 1 &#206;&#188;M 30 min. dIndole-3-<span id='am-112' about='obo:IMR_0200208' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-113' property="oboInOwl:id" content="IMR:0200208" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-115' property="rdfs:label" content="Acetic acid" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasDbXref" content="KEGG:C00033" datatype="xsd:string"></span>acetic acid</span> treatment: 1 &#206;&#188;M 15 min. eNumber unique peptides included protein ratio average indicated parentheses. fStatistical signi&#239;&#172;?cance based sample, tailed Student&#226;&#8364;&#8482;s t test. <br />  <br />    Ntapi:gAHA1-rescue lines exhibited minimal observable                      e&#239;&#172;&#8364;ect.56 DREPP family protein identi&#239;&#172;?ed study, <br /> phenotypic consequences &#226;&#710;&#188;20% reduced vertical                      PcaP1, microtubule destabilizing activity results root growth standard media.  mass spectrometry                     negative regulation hypotcotyl elongation.58 closely related <br /> revealed subtle plasma membrane proteomic phenotype                      gene, PcaP2, negatively regulates root hair tip growth.67 Ntapi:gAHA1 plants. proteins showing abundance                        proteins bind Ca2+/calmodulin complexes;  <br /> di&#239;&#172;&#8364;erences Ntapi:gAHA1 versus wild-type plants                        Ntapi:AHA1 construct interefereing normal <br /> reported roles cell wall structure, vesicle tra&#239;&#172;&#402;cking,                 regulation activity protein respect cell elongation, proteins contributing factors             calmodulin binding.  altered abundance observed Ntapi:gAHA1 short-root phenotype. SVL2                     proteins Ntapi:gAHA1 seedlings contributing related plasma membrane anchored, glycerophosphoryl                    reduced vertical root growth. possible tip <br /> diester phosphodiesterase protein SHV3. SHV3 plays role                  growing systems a&#239;&#172;&#8364;ected. example, pollen tube <br /> cell wall rigidity, knockout mutants exhibit ruptured root                growth endogenous knockout pollen carrying hairs swollen guard cells.57 PATL1 expressed root tip               Ntapi:gAHA1 transgene negatively a&#239;&#172;&#8364;ected, cells plays important role membrane tra&#239;&#172;&#402;cking                   explain reduced numbers Ntapi:gAHA1 rescue progeny <br /> cell plate maturation, important processes cell                      observed. <br /> elongation.54,65,66 AtPrx34 known cell wall associated                     essential nature AHA1 AHA2 peroxidase expressed root cells plays role cell                  number important plant processes elongation. Knockout lines closely related gene, AtPrx33,                hypothesized function, hypothesize interacting short roots, double knockouts exhibit additive                    proteins, including protein kinases important regulation <br />                                                                         575                               dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                        Article <br />  AHA phosphorylation multiple sites C-terminal            material available free charge Internet http:// <br /> regulatory region,68 identi&#239;&#172;?ed. Initial exper-             pubs.acs.org. <br />  <br />                                                                          &#226;&#8211;&#160; <br /> imentation identify Ntapi:AHA1-interacting proteins identi- <br /> &#239;&#172;?ed additional copurifying AHA isoforms, AHA9                         AUTHOR INFORMATION <br /> AHA11. shown AHA proteins form <br />                                                                          Corresponding Author <br /> multimeric complexes plasma membranes, complexes homomultimeric heteromultimeric                    *E-mail: msussman@wisc.edu. Phone: 608-262-8608. Fax: 608- <br /> respect isoform composition shown.                   262-6748. <br /> copuri&#239;&#172;?cation results suggest complexes mixed          Funding isoform composition opens possibility         U.S. Department Energy O&#239;&#172;&#402;ce Basic Energy Sciences isoform composition native AHA complexes play               grant DEFG02-88ER13938 Michael R. Sussman. Molecular <br /> important regulatory signaling roles.69 results          Biosciences Training Grant, NIH National Research Service <br /> contrast recent publication Bobik et al., showed           Award T32 GM07215 Rachel Rodrigues (Nee Nelson). Nicotiana plumbaginifolia H+-ATPase isoforms PMA2               Notes <br /> PMA4 did oligomerize 70 di&#239;&#172;&#8364;erent                 authors declare competing &#239;&#172;?nancial  <br />  <br />                                                                          &#226;&#8211;&#160; <br /> results di&#239;&#172;&#8364;erences experimental design, <br /> including use natural versus overexpression promoters              ABBREVIATIONS USED <br /> di&#239;&#172;&#8364;erences biological activity H+-ATPase isoforms Arabidopsis versus Nicotiana. observed increased            AHA, Arabidopsis H+-ATPase; PMF, proton motive force; TAP, <br /> copuri&#239;&#172;?cation AHA11 treatment fungal toxin             tandem a&#239;&#172;&#402;nity puri&#239;&#172;?cation; M&amp;S, Murashige &amp; Skoog; AP&#226;&#710;&#8217; <br /> FC, suggesting isoform composition quaternary                MS, a&#239;&#172;&#402;nity-puri&#239;&#172;?cation mass spectrometry; CBP, calmodulin- <br /> AHA complexes constantly &#239;&#172;&#8218;ux responsive             binding protein <br /> environment. obvious limitation rescue plants express AHA2,  high expression <br /> levels shared sequence identity AHA1,                                                                          &#226;&#8211;&#160;    REFERENCES <br />                                                                            (1) Ku&#204;&#710;hlbrandt, W. (2004) Biology, structure mechanism P- <br /> predict common isoform complex                    type ATPases. Nat. Rev. Mol. Cell Biol. 5, 282&#226;&#710;&#8217;295. <br /> AHA1. absence AHA2, isoforms                        (2) Buckhout, T. J. (1989) Surcose transport isolated plasma- <br /> artifactually incorporating multimeric AHA complexes.               membrane vesicles sugar beet (Beta vulgaris L.) Evidence    autoinhibited state AHA1 reduce ability             electrogenic sucrose-proton symport. Planta 178, 393&#226;&#710;&#8217;399. <br /> identify interacting proteins. tested             (3) Bush, D. R. (1993) Proton-coupled sugar <span id='am-45' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-46' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> <br />                                                                          transporters plants. Annu. Rev. Plant Physiol. Plant Mol. Biol. 44, <br /> expand identi&#239;&#172;?cation copurifying proteins combining            513&#226;&#710;&#8217;542. vivo treatments hypothesized known regulate              (4) Palmgren, M. G. (2001) Plant plasma membrane H+-ATPases: <br /> AHA activity metabolic labeling, a&#239;&#172;&#402;nity puri&#239;&#172;?cation,            Powerhouses nutrient uptake. Annu. Rev. Plant Physiol. Plant Mol. <br /> mass spectrometry. dynamic copuri&#239;&#172;?cation PHOT1                Biol. 52, 817&#226;&#710;&#8217;845. <br /> 14-3-3 response exogenous auxin fusicoccin suggest                (5) Sussman, M. H. R. (2012) effect genetically reduced experiments reveal similar, potentially            plasma membrane protonmotive force vegetative growth novel, AHA1-interacting proteins response speci&#239;&#172;?c                  Arabidopsis. Plant Physiol. 158, 1158&#226;&#710;&#8217;1171. <br /> conditions. current limitation inability discern               (6) Sussman, M. R. (1994) Molecular analysis proteins plant proteins di&#239;&#172;&#8364;erential abundance puri&#239;&#172;?ed                   plasma membrane. Annu. Rev. Plant Physiol. Plant Mol. Biol. 45, 211&#226;&#710;&#8217; <br /> samples vivo elicitation re&#239;&#172;&#8218;ect true Ntapi:AHA1-                234. <br />                                                                            (7) DeWitt, N. D., Harper, J. F., Sussman, M. R. (1991) <br /> interacting proteins indicate changes abundance             Evidence plasma membrane proton pump phloem cells proteins plasma membrane fractions prior                     higher plants. Plant J. Cell Mol. Biol. 1, 121&#226;&#710;&#8217;128. <br /> puri&#239;&#172;?cation IgG resin. addressed problem                   (8) DeWitt, N. D., Sussman, M. R. (1995) Immunocytological analyzing aliquot unpuri&#239;&#172;?ed plasma membrane                   localization epitope-tagged plasma membrane proton pump (H+- <br /> samples tandem mass spectormetry;                       ATPase) phloem companion cells. Plant Cell 7, 2053&#226;&#710;&#8217;2067. <br /> increased sample complexity nonpuri&#239;&#172;?ed samples result               (9) Schmid, M., Davison, T. S., Henz, S. R., Pape, U. J., Demar, M., decreased peptide identi&#239;&#172;?cation inconclusive results,             Vingron, M., Scho&#204;&#710;lkopf, B., Weigel, D., Lohmann, J. U. (2005) especially abundant proteins. Introducing cross-linking         gene expression map Arabidopsis thaliana development. Nat. Genet. <br /> reagents maintain protein interactions increasing           37, 501&#226;&#710;&#8217;506. <br /> denaturing quality washing steps remove nonspeci&#239;&#172;?cally               (10) Winter, D., Vinegar, B., Nahal, H., Ammar, R., Wilson, G. V., <br />                                                                          Provart, N. J. (2007) &#226;&#8364;&#339;electronic fluorescent pictograph&#226;&#8364;? <br /> bound proteins puri&#239;&#172;?cation resin facilitate                                                                          browser exploring analyzing large-scale biological data sets. <br /> identi&#239;&#172;?cation interacting proteins future. <br />  <br /> &#226;&#8211;&#160; <br />                                                                          PLoS 2, e718-1&#226;&#710;&#8217;e718-12. <br />                                                                            (11) Certal,  C., Almeida, R. B., Carvalho, L. M., Wong, E., <br />      ASSOCIATED CONTENT                                                  Moreno, N., Michard, E., Carneiro, J., Rodrigue&#204; Le&#204;  J., Wu, H.-M., <br /> * Supporting Information <br />  S                                                                       Cheung,  Y., Feijo&#204;?, J.  (2008) Exclusion proton ATPase predicted protein sequence Ntapi:AHA1 expressed              apical membrane associated cell polarity tip <br /> protein; information metabolically labeled             growth Nicotiana tabacum pollen tubes. Plant Cell 20, 614&#226;&#710;&#8217;634. <br />                                                                            (12) Feijo&#204;?, J. , Sainhas, J., Hackett, G. R., Kunkel, J. G., Hepler, <br /> proteome experiments including total protein peptide                 P. K. (1999) Growing pollen tubes possess constitutive alkaline band <br /> identi&#239;&#172;?cations; list AHA peptides identi&#239;&#172;?ed                    clear zone growth-dependent acidic tip. J. Cell Biol. 144, <br /> metabolically labeled, nonelicited Ntapi:gAHA1 copuri&#239;&#172;?cation             483&#226;&#710;&#8217;496. <br /> experiments; list 14-3-3 general regulatory factor proteins             (13) Merlot, S., Leonhardt, N., Fenzi, F., Valon, C., Costa, M., Piette, <br /> identi&#239;&#172;?ed metabolically labeled, nonelicited Ntapi:gAHA1              L., Vavasseur, , Genty, B., Boivin, K., Muller, , Giraudat, J., copuri&#239;&#172;?cation experiments 14N/15N ratios.                 Leung, J. (2007) Constitutive activation plasma membrane H <br />  <br />                                                                    576                            dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                 Article <br />  <br /> +-ATPase prevents abscisic acid-mediated stomatal closure. EMBO J.                 profiling tandem affinity purified 14-3-3 protein complexes 26, 3216&#226;&#710;&#8217;3226.                                                                     Arabidopsis thaliana. Proteomics 9, 2967&#226;&#710;&#8217;2985. <br />   (14) Liu, J., Elmore, J. M., Fuglsang,  T., Palmgren, M. G.,                    (31) Wittho&#204;&#710;ft, J., Caesar, K., Elgass, K., Huppenberger, P., Kilian, J., <br /> Staskawicz, B. J., Coaker, G. (2009) RIN4 functions plasma                Schleifenbaum, F., Oecking, C., Harter, K. (2011) activation <br /> membrane H+-ATPases regulate stomatal apertures                          Arabidopsis P-ATPase 1 brassinosteroid receptor BRI1 pathogen attack. PLoS Biol. 7, e1000139-1&#226;&#710;&#8217;e1000139-16.                             independent threonine 948 phosphorylation. Plant Signaling Behav. <br />   (15) Hayashi, Y., Nakamura, S., Takemiya, , Takahashi, Y.,                     6, 1063&#226;&#710;&#8217;1066. <br /> Shimazaki, K. , Kinoshita, T. (2010) Biochemical character-                   (32) Rohila, J. S., Chen, M., Cerny, R., Fromm, M. E. (2004) <br /> ization vitro phosphorylation dephosphorylation                   Improved tandem affinity purification tag methods isolation plasma membrane H+-ATPase. Plant Cell Physiol. 51, 1186&#226;&#710;&#8217;1196.                      protein heterocomplexes plants. Plant J. Cell Mol. Biol. 38, 172&#226;&#710;&#8217; <br />   (16) Kinoshita, T., Hayashi, Y. (2011) New insights                 181. <br /> regulation stomatal opening blue light plasma membrane                    (33) Rubio, V., Shen, Y., Saijo, Y., Liu, Y., Gusmaroli, G., Dinesh- <br /> H(+)-ATPase. Int. Rev. Cell Mol. Biol. 289, 89&#226;&#710;&#8217;115.                                Kumar, S. P., Deng, X. W. (2005) alternative tandem affinity <br />   (17) Robertson, W. R., Clark, K., Young, J. C., Sussman, M. R.               purification strategy applied Arabidopsis protein complex isolation. <br /> (2004) Arabidopsis thaliana plasma membrane proton pump                      Plant J. Cell Mol. Biol. 41, 767&#226;&#710;&#8217;778. <br /> essential pollen development. Genetics 168, 1677&#226;&#710;&#8217;1687.                          (34) Chang, -F. (2006) Mass spectrometry-based proteomic <br />   (18) Gevaudant, F., Duby, G., von Stedingk, E., Zhao, R.,                        analysis epitope-tag affinity purified protein complexes Morsomme, P., Boutry, M. (2007) Expression constitutively                 eukaryotes. Proteomics 6, 6158&#226;&#710;&#8217;6166. <br /> activated plasma membrane H+-ATPase alters plant development                    (35) Leene, J. V., Stals, H., Eeckhout, D., Persiau, G., Slijke, E. V. D., <br /> increases salt tolerance. Plant Physiol. 144, 1763&#226;&#710;&#8217;1776.                           Isterdael, G. V., Clercq,  D., Bonnet, E., Laukens, K., Remmerie, N., <br />   (19) Baxter,  R., Young, J. C., Armstrong, G., Foster, N.,                     Henderickx, K., Vijlder, T. D., Abdelkrim, , Pharazyn, , Onckelen, <br /> Bogenschutz, N., Cordova, T., Peer, W. , Hazen, S. P., Murphy,                 H. V., Inze&#204;?, D., Witters, E., Jaeger, G. D. (2007) tandem affinity <br /> S., Harper, J. F. (2005) plasma membrane H+-ATPase                        purification-based technology platform study cell cycle <br /> required formation proanthocyanidins seed coat                   interactome Arabidopsis thaliana. Mol. Cell. Proteomics 6, 1226&#226;&#710;&#8217; <br /> endothelium Arabidopsis thaliana. Proc. Natl. Acad. Sci. U.S  102,            1238. <br /> 2649&#226;&#710;&#8217;2654.                                                                          (36) Qi, Y., Katagiri, F. (2009) Purification low-abundance <br />   (20) Haruta, M., Burch, H. L., Nelson, R. B., Barrett-Wilt, G., Kline,           Arabidopsis plasma-membrane protein complexes identification K. G., Mohsin, S. B., Young, J. C., Otegui, M. S., Sussman, M. R.              candidate components. Plant J. Cell Mol. Biol. 57, 932&#226;&#710;&#8217;944. <br /> (2010) Molecular characterization mutant Arabidopsis plants                 (37) Haruta, M., Sussman, M. R. (2012) effect reduced plasma membrane proton pump activity. J. Biol. Chem. 285,                  genetically reduced plasma membrane protonmotive force 17918&#226;&#710;&#8217;17929.                                                                       vegetative growth Arabidopsis thaliana. Plant Physiol. 158, 1158&#226;&#710;&#8217; <br />   (21) Lalonde, S., Ehrhardt, D. W., Loque&#204;?, D., Chen, J., Rhee, S. Y.,            1171. Frommer, W. B. (2008) Molecular cellular approaches                 (38) Krysan, P. J., Young, J. C., Tax, F., Sussman, M. R. (1996) <br /> detection protein-protein interactions: Latest techniques                   Identification transferred DNA insertions Arabidopsis genes <br /> current limitations. Plant J. 53, 610&#226;&#710;&#8217;635.                                         involved signal transduction ion transport. Proc. Natl. Acad. Sci. <br />   (22) Miernyk, J. , Thelen, J. J. (2008) Biochemical approaches             U.S  93, 8145&#226;&#710;&#8217;8150. discovering protein-protein interactions. Plant J. Cell Mol. Biol. 53,          (39) Hellens, R. P., Edwards, E. , Leyland, N. R., Bean, S., 597&#226;&#710;&#8217;609.                                                                           Mullineaux, P. M. (2000) pGreen: versatile flexible binary Ti <br />   (23) Fukao, Y. (2012) Protein-protein interactions plants. Plant              vector Agrobacterium-mediated plant transformation. Plant Mol. <br /> Cell Physiol. 53, 617&#226;&#710;&#8217;625.                                                         Biol. 42, 819&#226;&#710;&#8217;832. <br />   (24) Gingras, -C., Gstaiger, M., Raught, B., Aebersold, R.                  (40) Clough, S. J., Bent,  F. (1998) Floral dip: simplified <br /> (2007) Analysis protein complexes using mass spectrometry. Nat.                 method Agrobacterium-mediated transformation Arabidopsis <br /> Rev. Mol. Cell Biol. 8, 645&#226;&#710;&#8217;654.                                                   thaliana. Plant J. Cell Mol. Biol. 16, 735&#226;&#710;&#8217;743. <br />   (25) Arabidopsis Interactome Mapping Consortium (2011) Evidence                   (41) Nelson, C. J., Huttlin, E. L., Hegeman,  D., Harms,  C., network evolution Arabidopsis interactome map. Science 333,              Sussman, M. R. (2007) Implications 15N-metabolic labeling 601&#226;&#710;&#8217;607.                                                                           automated peptide identification Arabidopsis thaliana. Proteomics 7, <br />   (26) Chen, J., Lalonde, S., Obrdlik, P., Noorani Vatani, , Parsa, S.           1279&#226;&#710;&#8217;1292. , Vilarino, C., Revuelta, J. L., Frommer, W. B., Rhee, S. Y. (2012)           (42) Larsson, C., Widell, S. Isolation plant plasma membranes <br /> Uncovering Arabidopsis membrane protein interactome enriched                    production inside vesicles, Aqueous Phase Systems: <br /> transporters using mating-based split ubiquitin assays classification          Methods Protocols (Hatti-Kaul, R., Ed.) pp 159&#226;&#710;&#8217;166, Humana <br /> models.  Plant Sci. 3, 124.                                                  Press, Totowa, NJ. <br />   (27) Jahn, T., Fuglsang,  T., Olsson, , Bru&#204;&#710;ntrup,  M., Collinge,           (43) Lee, M. C. S., Hamamoto, S., Schekman, R. (2002) <br /> D. B., Volkmann, D., Sommarin, M., Palmgren, M. G., Larsson, C.                <span id='am-117' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-118' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-122' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-123' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-125' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Ceramide</span> biosynthesis required formation oligomeric <br /> (1997) 14-3-3 protein interacts directly C-terminal                   H+-ATPase Pma1p yeast endoplasmic reticulum. J. Biol. Chem. <br /> region plant plasma membrane H+-ATPase. Plant Cell 9, 1805&#226;&#710;&#8217;                 277, 22395&#226;&#710;&#8217;22401. <br /> 1814.                                                                               (44) Wessel, D., Flu&#204;&#710;gge, U.  (1984) method   (28) Morandini, P., Valera, M., Albumi, C., Bonza, M. C., Giacometti,            quantitative recovery protein dilute solution presence S., Ravera, G., Murgia, , Soave, C., Michelis, M.  (2002)             detergents lipids. Anal. Biochem. 138, 141&#226;&#710;&#8217;143. <br /> novel interaction partner C-terminus Arabidopsis thaliana                (45) Park, S. K., Venable, J. D., Xu, T., Yates, J. R., 3rd (2008) plasma membrane H+-ATPase (AHA1 isoform): Site mechanism                       quantitative analysis software tool mass spectrometry-based action H+-ATPase activity differ 14-3-3 proteins.              proteomics. Nat. Methods 5, 319&#226;&#710;&#8217;322. <br /> Plant J. Cell Mol. Biol. 31, 487&#226;&#710;&#8217;497.                                               (46) Nesvizhskii,  , Keller, , Kolker, E., Aebersold, R. <br />   (29) Fuglsang,  T., Guo, Y., Cuin, T. , Qiu, Q., Song, C.,                   (2003) statistical model identifying proteins tandem mass <br /> Kristiansen, K. , Bych, K., Schulz, , Shabala, S., Schumaker, K. S.,           spectrometry. Anal. Chem. 75, 4646&#226;&#710;&#8217;4658. <br /> Palmgren, M. G., Zhu, J.-K. (2007) Arabidopsis protein kinase                   (47) Zhang, X., Henriques, R., Lin, S.-S., Niu, Q.-W., Chua, N.- <br /> PKS5 inhibits plasma membrane H+-ATPase preventing                          H. (2006) Agrobacterium-mediated transformation Arabidopsis <br /> interaction 14-3-3 protein. Plant Cell 19, 1617&#226;&#710;&#8217;1634.                         thaliana using floral dip method. Nat. Protoc. 1, 641&#226;&#710;&#8217;646. <br />   (30) Chang, -F., Curran, , Woolsey, R., Quilici, D., Cushman, J.              (48) Kanczewska, J., Marco, S., Vandermeeren, C., Maudoux, O., <br /> C., Mittler, R., Harmon, , Harper, J. F. (2009) Proteomic                    Rigaud, J.-L., Boutry, M. (2005) Activation plant plasma <br />  <br />                                                                              577                           dx.doi.org/10.1021/bi401096m | Biochemistry 2014, 53, 566&#226;&#710;&#8217;578 <br />  Biochemistry                                                                                                                                                 Article <br />  <br /> membrane H+-ATPase phosphorylation binding 14-3-3                        sec14-related protein, zucchini (Cucurbita pepo). J. Plant Physiol. <br /> proteins converts dimer hexamer. Proc. Natl. Acad. Sci. U.S             163, 1150&#226;&#710;&#8217;1158. <br /> 102, 11675&#226;&#710;&#8217;11680.                                                                   (67) Kato, M., Aoyama, T., Maeshima, M. (2013) Ca2+- <br />   (49) Ottmann, C., Marco, S., Jaspert, N., Marcon, C., Schauer, N.,               binding protein PCaP2 located plasma membrane involved Weyand, M., Vandermeeren, C., Duby, G., Boutry, M., Wittinghofer,                  root hair development possible signal transducer. Plant J. Cell Mol. , Rigaud, J.-L., Oecking, C. (2007) Structure 14-3-3                    Biol. 74, 690&#226;&#710;&#8217;700. <br /> coordinated hexamer plant plasma membrane H+-ATPase                       (68) Rudashevskaya, E. L., Ye, <br /> </body></html>