<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, June 1989. p. 2682-2694                                                                    Vol. 9,  6 <br /> 0270-7306/89/062682-13$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  <br />         Yeast Pheromone Response Pathway: Characterization          Suppressor Restores Mating Receptorless Mutants <br />                                    KAREN L. CLARK GEORGE F. SPRAGUE, JR.* <br />          Institiute Molecillar Biology (Hid Depar-tmnent Biology, University egoni, Eiugene, Oregonl 97403 <br />                                         Received 28 December 1988/Accepted 14 March 1989 <br />  <br />            Saccharomyces cerevisiae haploid cells, ax  mate appropriately stimulated pheromone <br />          secreted opposite cell type  factor factor, respectively). binding pheromone          receptor signal initiates series intracellular changes lead specific physiological <br />          alterations required mating. identify components signal transduction pathway, sought <br />          pseudorevertants restored mating competence receptor mutants (MATTa ste3::LEU2). suppressor <br />          srml-l isolated recessive mutation conferred temperature-sensitive growth strains          mating ability MAToa ste3::LEU2 strains nonpermissive temperature. addition, srml-l <br />          mutants shifted nonpermissive temperature, exhibited phenotypes characteristic          pheromone response, induction FUSI transcription accumulation cells G, phase cell <br />          cycle. srml-l mutation suppressed deletion factor-receptor gene cells.           phenotypes suggest wild-type SRMI product component pheromone response pathway. <br />          Deletion STE4 STE5, required haploid cell types mating response pheromone, <br />          suppressed srml-1, suggesting SRMI product function STE4 STE5 <br />          products. SRMI essential gene expressed haploid cell types product          mating, diploids. Homozygous srml-l diploids temperature sensitive did arrest <br />          G,.  SRMI product role vegetative life cycle cells. <br />  <br />    yeast Saccharomvces cerevisiae haploid cell               transduction interaction heterotrimeric <span id='am-10' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span>GTP</span>-binding <br /> types, (x, reproduce mitosis.                  proteins (G proteins)  recent review, reference 64). cocultured, cell types exit mitotic cell                   like understand processes cycle undergo mating process form cell type,            binding pheromone receptor causes physiolog- ax diploid cell. Interaction cells                 ical events discussed  requires identification requires secretion response <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> pheromones                 components pheromone response pathway, path <br />  recent reviews, references 10, 21, 59).  (x cells          propagates signal cell surface intracel- <br /> secrete ax-factor cells respond, cells secrete          lular targets. cellular component pathway factor (x cells respond. responses phero-              pheromone receptor. Specificity pheromone recog- <br /> mone include increased transcription genes prod-                nition provided receptor; rest pathway <br /> ucts involved mating (15, 18, 38, 65), increased                  common ax cells (2, 19, 36, 45). capacity agglutinate cells opposite mating type           analogy mammalian receptors discussed  (3, 13), preparation nucleus mating (52), arrest          expect G protein involved pheromone G1 phase cell cycle (7, 68). member            signaling. fact, homolog subunit G proteins mating pair elongates partner (projection forma-              encoded GPAI (SCG1) gene (11, 41, 43). GPAI <br /> tion) (34; J. Rine, Ph.D. thesis, University Oregon,                  product undoubtedly component pheromone <br /> Eugene, 1979); cell fusion followed nuclear fusion               response pathway, mutations GPAI suppress creates zygote.                                                   mating defect caused deletion factor- ac- <br />    Pheromones interact target cells specific              factor-receptor genes (23, 41). support notion <br /> receptors. ax-factor receptor encoded STE2 (4, 25),            G protein component pheromone response gene expressed cells (44). STE3 gene,              pathway comes finding STE4 STE18 expressed cells (61), thought encode             products, required response phero- factor receptor necessary sufficient             mones, homologous , y subunits, respec- <br /> response factor (2, 14, 45). hydropathic profiles            tively, G proteins (67). possible elements receptors, sequence homology,                    pheromone response pathway include STE5, STE7, <br /> predict similar structure, seven potential              STE  STE12 gene products. roles transmembrane segments followed cytoplasmic car-                    proteins defined precisely, boxyl-terminal domain hydrophilic (8, 14, 44).            required cells pheromone response features proposed structure veri-              mating (19, 36). <br /> fied, STE3 product (9a). structure                 identify elements involved pheromone <br /> similar receptors, including                signaling, screened mutants suppress mating <br /> mammalian rhodopsins (47), 3-adrenergic receptors (12),              defect caused deletion factor-receptor gene. muscarinic <span id='am-82' about='obo:IMR_0100111' typeof='owl:Thing obo:IMR_0001693 obo:IMR_0100121 obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0002121'><span id='am-83' property="oboInOwl:hasExactSynonym" content="O-Acetylcholine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0100111" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="O-Acetylcholine" datatype="xsd:string"></span><span id='am-86' property="rdfs:label" content="acetylcholine" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="KEGG:C01996" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acetylcholine</span> receptors (32). mam-              cells lack factor receptors, malian receptors believed initiate intracellular signal           respond factor unable mate. Cells                                                                          acquire new mutation activates pheromone  <br />   * Corresponding author.                                                sponse pathway regain ability mate.                                                                    2682 <br />  VOL. 9, 1989                                                                       SUPPRESSOR RECEPTOR MUTATIONS                   2683 <br />  <br />                                                                  TABLE 1. Strains <br />      Strain                                                  Genotype                                                      Source <br /> DC5                       MATa leO2-3. -112 Ii.s3 gail2 (                                         J. Strathern <br /> SY1229                    Isogenic DC5 ilro3                                             work <br /> SY1263                    Isogenic SY1229 MATot                                          work <br /> 227                       MATa lNsl/ Jr!J. Hicks 1. Herskowitz <br /> EG123                     MATa tirpl leit2 ira3 his4-519 (canI                                       K. Tatchell <br /> SY1346                    Isogenic EG123 lYs2                                            work <br /> SM1229                    Isogenic EG123 tnfal::LEU2 tnfa2::URA3                         S. Michaelis l. Herskowitz <br /> SY1347                    Isogenic SM1229 lvs2 lYs5                                   work <br /> X2180-1A-L1-U1            MATa gal2 Ivs2 mr-3 mal inel                                               J. Rothman T. Stevens <br /> HR125-5d                  MATa leii2-3. -112 lr(3-52 his3 liis4 trpl gal2                            R. Jensen 1. Herskowitz <br /> SY816                     Isogenic HR125-5d MATo.                                        R. Jensen  Herskowitz <br /> SY817                     Isogenic HR125-5d MATa/MATAt                                   R. Jensen  Herskowitz <br /> YY508                     Isogenic SY816 ste3::LEU2                                      D. Hagen <br /> 320                       MATa leil ura3 ade2 (cail (cv1h2-21 r-olel                              Y. Kassir <br /> YY601&quot;                    MATa ste3::LEU2 lei2-3, -112 LEU2 mra3 ade2                             work <br /> YY750                     MATot ste3::LEU2 itr-3 leO2 trpl his4-519  l                            work <br /> XR28-50c                  MATa leu2 trpl airo7  lys2                                              J. Rine <br /> SY1077                    320 x mutagenized YY508: harbors srm11l                                  work <br /> SY1115&quot;                   MAToi srIl-l lee2-3, -112 ura3 tirpl lis4                                  Segregant SY1077 <br /> YY811&quot;                    MATot ste3::LEU2 srmil Ile2-3. -112 imra3 his3 lhis4 trpl                Segregant SY1077 <br /> SY1154                    MATa ura3 leu2 mnetl4 ade2-loc Ode! his6 urplan sup4 SUP4-3              Bender <br /> SY1160                    lsogenic SY1154 baHrl::LEU2                                     Bender <br /> SY1325&quot;                   MATot sste3::LEU2 srml-l itrl3 Ien2 trpl atilel his4 Ihis3 HIS3 his6 Segregant YY811 x SY1154 <br />                             HIS6 mnetl4 MET14 <br /> SY1327&quot;                   MATot siIIl-1 uir(a3 leu2 trpl tide! ade2 his4 his3 HIS3 his6 HJS6   Segregant YY811 x SY1154 <br /> SY1355                    Isogenic SY1327 ste3::URA3                                     work <br /> SY1357                    Isogenic SY1327 ste4::LEU2                                     work <br /> SY1359                    Isogenic SY1327 ste5::LEU2                                     work <br /> SY1383                    Isogenic SY1327 MATa ste2::LEU2                            work <br /> SY1333&quot;                   MATot ste3::LEU2 leit2 itra3 trpl (idel ade2 ADE2 metl4 METJ4        Segregant YY811 x SY1154 <br /> SY1334&quot;                   MATot leu2 ira3 trpl adel ade2 liis4 hiis3 HIS3 lhis6 HIS6 metl4  Segregant YY811 x SY1154 <br />                             MET14 <br /> SY1392&quot;                   MATot srmnl-l ara3 lea2 idel trpl liis3 HIS3 his4 HIS4 lhis6      Segregant YY811 x SY1154 <br />                             HIS6 <br /> SY1393                    Isogenic SY139&apos; MATa                                           work <br /> SY1394                    Isogenic SY1392 MATa ste2::LEU2                            work <br /> SY1411&quot;                   MATa ste3::LEU2 leu2 wra3 tide! ade2 trpl /his3 HJS3 his6 HIS6       Segregant YY811 x SY1154 <br />                             metI4 MET14 <br /> SY1225&quot;                   MA Ta/MA Tot srmI-1/srinl- itra3lwra3 +1/eil aide2/+ (/liS3 HIS3)1      Diploid cosegregants                             (his3 HIS3) ( s4 HIS4)/(his4 HIS4)                               SY1077 <br />    strains ((111l-ll CANI. (clz2-21 CYH2. mmil RMEI. <br />  <br /> report isolation characterization suppressor                     transforming SY1392 ste2::LEU2 DNA (srml-1) deletion factor receptor gene.                           AB506 (kindly provided  Burkholder L. Hartwell) <br />                                                                               pGAL-HO plasmid simultaneously. Similarly, strain <br />                MATERIALS METHODS                                          SY1383 generated simultaneous transformation   Strains media. strains used experiments                   SY1327 ste2::LEU2 sequences AB506 presented Table 1. Details construction                 pGAL-HO plasmid. <br /> strains presented  step gene replacement                       Strains SY1346 SY1347 created selecting method Rothstein (53) used create isogenic                  h&apos;s mutants EG123  K. Tatchell  Mitchell) srml-l mutant strains. Strain SY1357 generated                         SM1229 (kindly provided S. Michaelis), respectively, transforming SY1327  relevant genotype MAToh srml-l                    ox-amino adipic acid medium (9). itra3 mutant DC5  STE3 STE4 STE5) ste4::LEU2 DNA plasmid p121,                        J. Strathern) selected 5-fluoro-<span id='am-49' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>orotic acid</span> medium <br /> kindly provided V. MacKay. Similarly, SY1327                            D. Hagen; 5),creating SY1229 switched <br /> transformed ste5::LEU2 DNA plasmid pSL567                        MAThs pGAL-HO   Bender) create SY1263. <br /> create strain SY1359 ste3::URA3  complete dele-                        media used YEPD  YPD; 56), SD (56), <br /> tion STE3) create SY1355. genotypes                        SD-URA SD-LYS (SD supplemented <span id='am-54' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span>, L- <br /> transformants confirmed Southern analysis (58),                       leucine, <span id='am-39' about='obo:IMR_0200137' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-40' property="oboInOwl:id" content="IMR:0200137" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="L-Histidine" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00135" datatype="xsd:string"></span>L-histidine</span>, <span id='am-77' about='obo:IMR_0200428' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="L-Arginine" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00062" datatype="xsd:string"></span><span id='am-81' property="oboInOwl:id" content="IMR:0200428" datatype="xsd:string"></span>L-arginine</span>, <span id='am-109' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-110' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span><span id='am-112' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span>L-methionine</span>, <span id='am-104' about='obo:IMR_0200240' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-105' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00082" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="L-Tyrosine" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:id" content="IMR:0200240" datatype="xsd:string"></span>L-tyrosine</span>, <br /> using nick-translated pSL8  STE3 URA3), p121                      <span id='am-94' about='obo:IMR_0200307' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-95' property="oboInOwl:id" content="IMR:0200307" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="KEGG:C00078" datatype="xsd:string"></span><span id='am-97' property="rdfs:label" content="L-Tryptophan" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-tryptophan</span>, <span id='am-44' about='obo:IMR_0200274' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00079" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200274" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="rdfs:label" content="L-Phenylalanine" datatype="xsd:string"></span>L-phenylalanine</span>, <span id='am-59' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-60' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span>L-threonine</span>, <span id='am-29' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-30' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span>L-lysine</span> STE4), SF9  STE5; provided V. MacKay)                            <span id='am-99' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-100' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-101' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span>; 56), SD supplemented individual nutrients <br /> probes. Strain SY1393 MATa derivative SY1392                      (e.g., SD plus URA). <br /> relevant genotype MATot sr,nIl STE3 STE2)                           Mutant isolation. different cultures MATot ste3:: <br /> HO-promoted mating-type interconversion using construc-                     LEU2 strains mutagenized ethyl methane sul- <br /> tion places HO endonuclease gene control                   fonate described previously (42), surviving cells (4 GALI promoter (pGAL-HO; 27). Strain SY1394                         25% mutagenized population) screened  2684     CLARK SPRAGUE                                                                                    MOL. CEt-L. BIOL. <br />  <br /> ability mate. Immediately inactivation ethyl           4-kb EcoRI fragment pSL737 yielded plasmid <br /> methane sulfonate, cells diluted YEPD broth        (pSL739) able complement sr-n/l plated YEPD plates density 0.7 x 10s 1.5 x 105       mutation. BgIII-SalI fragment plasmid viable cells plate. 3 days alt room temperature.     able complement s,til-1 cloned YCp5O <br /> cells replica plated lawns MATa cells plates         (creating pSL749). contrast, plasmid pSL742, selective growth diploids. selections   obtained deletion 0.8-kb BaimtiHI fragment diploid growth room temperature.         pSL739, lacked complementing activity  Fig. 3A). <br /> anticipated bypass suppressor mutations lethal          second method used locate SRMI gene mutations, expected selection yield             insert transposon-mediated mutagenesis (22; mutants partially inactive gene product         instructions generously provided M. Hoyt D. <br /> room temperature severe phenotype elevated           Botstein). Plasmid pSL739 used mutagenesis. <br /> temperatures. fourth selection duplicate        ability various mutated plasmids complement <br /> room temperature 30&#194;&#176;C obtain larger group              sitIl determined described  location bypass suppressors. diploids sporulated dis-          transposon noncomplementing sected. meiotic products analyzed             complementing plasmids determined restriction  <br /> presence suppressor ste3::LEU2.                            donuclease digestion  Fig. 3A). <br />    selections examined 5 x 10&quot; muta-            Presumptive null alleles SRMI created cloning <br /> genized cells strain YY508. culture replica       2.9-kb BglII fragment YEp13 (16) containing LEU2 <br /> plated lawns strain 320 cells (MA Ta STE3) SD plus        gene 1.2-kb Bta,nHI fragment pSL641 containing uracil (20 mg/liter). second cultures           URA3 gene unique BainHI site pSL742 create <br /> replica plated lawns strain YY601 (MATa stie3::LEU2)         pSL786 pSL828. respectively. pSL641 plasmid SD plus uracil. total 11 diploids appeared        G. McCaffrey yeast URA3 gene cloned <br /> analyzed. diploid apparently product rare         polylinker flanked BatnHI sites. <br /> mating event c ste3::LEU2 segregants              plasmids constructed RNA analysis. <br /> able mate, diploids showed evidence suppressor          pSL757 pSL760 2.3-kb BamHHI-Sall 0.8-kb <br /> mutations intractable genetically,           BimHI fragments, respectively, pSL749 cloned harbored readily identifiable suppressor ste3::LEU2.          pSPT18 (Pharmacia), used make SP6- fourth selection examined 2 x 106 mutagenized cells         T7-generated RNA probes SRMJ mRNA. pSL743, strain YY750, replica plated lawns strain         0.8-kb BaniHI fragment cloned YIp5 (pBR322 <br /> XR28-50c (MA Ta STE3) SD plus tryptophan (20 mg/liter).         containing yeast URA3 gene; 63), nick translated 22TC, 121 diploids formed, 274 diploids ap-            used DNA probe SRMI URA3 mRNA. <br /> peared 30&#194;&#176;C. diploids analyzed;             sets nested deletions created sequencing <br /> product rare mating events, seven            SRMI  Fig. 3B). 2.4-kb XlioI-PstI fragment difficult analyze genetically. suppressor identified     pSL749 cloned pBluescriptR-KS+ pBlue- selection involving strains YY508 320 chosen       script&apos;-SK+ (Stratagene, ) strands intensive examination. suppressor called sirInil        sequenced. Nested deletions plasmid suppressor receptor mutations.                              exonuclease III mung bean nuclease digestion (20; <br />    Cloning sequencing. clone SRMI gene               Stratagene), using Exo/Mung deletion kit according complementation temperature-sensitive lethality          instructions manufacturer (Stratagene). Single- <br /> sriml-l, transformed strain YY811  relevant genotype        stranded DNA prepared plasmids infect- MATot srmnl-l ste3::LEU2) YCp5O library yeast         ing plasmid-bearing strains helper bacteriophage <br /> genomic sequences, 10 15 kilobase pairs (kb) length (51;     M13K07 (66) essentially described Stratagene. 8 <br /> gift J. Thorner T. Stevens). YCp5O yeast shuttle       18 h incubation, single-stranded DNA prepared vector harboring yeast URA3 gene yeast cen-              usual. Sequencing dideoxy-chain termina- <br /> tromere (51). Spheroplasts strain YY811 prepared,          tion method (54), using 7-deaza-<span id='am-34' about='obo:IMR_0200286' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00286" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="dGTP" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0200286" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dGTP</span> <span id='am-69' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-75' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-76' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>nucleotide</span> samples transformed equal amounts             mixes (40). <br /> DNA. transformations treated follows.             plasmid constructed test complementing transformation, transformants selected room            activity open reading frame (ORF) revealed se- <br /> temperature screened ability grow           quencing. XlioI linkers added HindIll site 370C. 1,180 transformants  corresponds sam-          plasmid p126H (28). EcoRI-XhoI fragment new <br /> pling genomic equivalent), able grow        plasmid, containing UAS TATA sequences 37&#194;&#176;C. transformation, transformants          GAL] gene, cloned E(oRI XhoI sites selected 37&#194;&#176;C initial period growth (16 h)       pSL749.  GAL] UAS fused upstream allow regeneration cell walls. selection yielded      presumptive SRMI ORF. transformant able grow 37&#194;&#176;C. DNA prepared               DNA manipulations. Bacterial transformations, bacterial transformants used transform bacteria          DNA preparations, plasmid constructions  <br /> ampicillin resistance. Examination plasmids revealed        formed standard methods (37). Yeast transformations contained presumably identical 10.6-kb inserts       performed essentially described Beggs (1). Yeast <br /> Fig. 3); later analyses plasmid obtain-         DNA (plasmid chromosomal) prepared proto- <br /> ed directly 37&#194;&#176;C. plasmid (pSL737) complemented            col Nasmyth Reed (46). Southern analyses (58) su nl-l retransformed YY811            performed, hybridizations described pre- sr-inzl-l mutants.                                           viously RNA analysis (61), using nick-translated plas- <br />    SRMI gene mapped insert              mids kit Bethesda Research Laboratories. <br /> methods. subclone fragments YCp5O               RNA analysis. RNA prepared yeast cells determine complement                described previously (61). Poly + RNA isolated temperature sensitivity strain YY811 SY1115. Deletion        Hybond-mAP paper Amersham Corp. suggested  VOL. 9, 1989                                                           SUPPRESSOR RECEPTOR MUTATIONS                   2685 <br />  manufacturer. Hybridization analysis performed                                  RESULTS <br /> described (38, 61), using SP6-generated RNA probes FUSI URA3. RNA probes opposite polarities               Isolation srml-l. mate, haploid cells identify SRMJ mRNA pSL757                 interact specific receptors secreted pheromones. <br /> pSL760, using SP6 T7 RNA polymerases Ribo-             Mutants make functional receptors phero- <br /> probe transcription Promega Biotec. DNA probes          mone unable mate  recent reviews, 10, 21, 59). SRMI URA3 generated nick translation           Binding pheromone receptor presumably generates <br /> pSL743, using kit Bethesda Research Laboratories.        internal signal causes cell respond Biomed Instruments scanning densitometer used              characteristic fashion prepare mate. identify <br /> quantitate bands visible autoradiogram. URA3        components internal signaling mechanism, began <br /> mRNA internal standard lane.                     strain internal substitution LEU2 <br />     Arrest assays. Cell cycle arrest induced shifting expo-    gene gene encoding factor receptor (ste3:: <br />                                                                   LEU2) isolated mutants regained ability nentially growing cultures room temperature (22           mate. <br /> 24&#194;&#176;C) 33&#194;&#176;C 4 h assayed described previously            Strain YY508 (MA Ta ste3::LEU2) mutagenized (15). 200 cells sample counted      ethyl methane sulfonate detailed Materials Meth- <br /> knowing identity sample, experiment       ods. Approximately 5 x 106 cells examined, times. Unbudded cells cells           diploid possessing readily identifiable suppressor ste3:: undergone projection formation, called shmoos,           LEU2 isolated. temperatures used assay <br /> considered G1 phase cell cycle. morphol-      phenotypes conferred suppressor; 37&#194;&#176;C used <br /> ogy pheromone-arrested cells examined exposing          assess viability, 33&#194;&#176;C used assess mating <br /> strain SY1160 (MATa barl::LEU2; step gene             ability. following genetic evidence indicates replacement BARI strain SY1154) 625 ng factor       diploid SY1077 harbors suppressor ste3: :LEU2. Diploid <br /> (Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> ) ml 4 h 30&#194;&#176;C.                      SY1077 grow 37&#194;&#176;C segregated fully viable <br />     Mating assays. Quantitative mating assays haploid          temperature-sensitive spores tetrad. strains performed filter mating assay described      addition, MATa ste3::LEU2 segregants, previously (19, 24) modifications:        temperature sensitive mate, threefold excess tester cells used; (ii) mating    mate 33&#194;&#176;C room temperature qualitative <br /> allowed proceed 6 h room temperature 33&#194;&#176;C;       mating assays. MATa ste3::LEU2 segregants (iii) mating cultures titered selective medium     grow 37&#194;&#176;C unable mate temperatures <br /> 30&#194;&#176;C determine number diploids room              tested. segregation pattern observed temperature different selective medium determine      outcross ste3 segregants mate <br /> number diploids plus haploids mating efficiency          33&#194;&#176;C (YY811 = MATa ste3::LEU2 srml-J) crossed determined  experimental haploids).                                                                   SY1154 (MATa SRMI STE3). viable tempera- <br />                                                                   ture-sensitive spores tetrad observed, mating efficiency defined number diploids ob-        fourths segregants mate 33&#194;&#176;C, served selective medium divided number        half mate room temperature.  diploids plus experimental haploids observed            MATot ste3: :LEU2 segregants temperature <br /> second selective medium. Strain 227 usual tester      sensitive mate, 33&#194;&#176;C.  tempera- strains, isogenic strains SY1346 SY1347        ture-sensitive growth phenotype ability restore <br /> tester strains experiments determined           mating ax strains harboring ste3::LEU2 allele cosegre- <br /> factor required mating exhibited MATot             gated crosses. cosegregation indicates single <br /> ste3::LEU2 srml cells. Strain SY1263 tester       mutation, designated srml suppressor receptor <br /> strains.                                                          mutations, responsible phenotypes. <br />    mating efficiencies plasmid-bearing strains           MATaIMATot SRMIIsrml-J diploids able grow assayed essentially described  Cultures grown       37&#194;&#176;C, MATa/MATa srml-llsrml-1 diploids temper- <br /> overnight SD-URA. cells pelleted              ature sensitive growth.  srml-l recessive centrifugation, suspended YEPD, used inoculate          respect temperature-sensitive growth phenotype. YEPD 6 x 106 cells ml. cultures           determine mating suppression phenotype grown 1.5 x 107 cells ml room temperature,        srmI recessive, analyzed mating efficiencies mating efficiencies determined described        potentially mating-competent diploids. MATalmata2 dip- ensure plasmid-bearing cells assayed,        loids functional STE3 allele mate efficiently used ura3 mutant strains (SY1263 test MATa strains         cells (62).  assayed ability MATotlmata <br /> SY1229 test MATot strains) tester strains. Diploids     ste3::LEU21ste3::LEU2 srml-JISRMI strains mate. identified plating mating mix SD plus <br />                                                                   diploids low mating effi- <br />                                                                   ciencies MATa ste3::LEU2 SRMI strains did (data histidine (20 mg/liter), diploids plus plasmid-bearing        shown), indicating mating suppression phenotype haploids identified plating SD-URA.              srrml recessive. <br /> scribed  mating efficiency calculated           analysis described relied qualitative mating <br /> titer cells able grow SD plus histidine divided   tests, noted ste3::LEU2 srml-J strains <br /> titer cells able grow SD-URA.                            mated wild-type STE3 strains. determine <br />    Qualitative mating tests (patch matings) using strain 227   degree srml-J restored mating receptor tester described previously (60).                mutants, performed quantitative mating tests (Table 2). <br />    Microscopy. Micrographs fixed cells taken       mating efficiencies 33&#194;&#176;C different ste3::LEU2 <br /> Zeiss Axioplan photomicroscope Nomarski optics,              srmnl-l mutants varied 10-4 10-2 constant <br /> using x 100 objective.                                          given strain. suppression observed  2686      CLARK SPRAGUE                                                                                                              MOL. CELL. BIOL. <br />  <br />                                                 TABLE 2. Suppression stc&apos;3::LEU2 srmll                                                                                           MaIting efficiency strain  <br />                           Genotype <br />  Strain                                                         M&apos;ATa MFal MFa2&quot;                          MATa lFa/ MFa2&quot;                  MATa mfal ,nfh2 <br />                    STE3              SRMI                22&apos;C                      33WC                           (33WC)                         (33&#194;&#176;) <br /> SY1334               +                +                  0.46                 0.34 0.69                                                       &lt;3.2 x 10     6 <br />  <br /> SY1327               +                -                 0.70       0.92       0.73                                                              1.6 x 10    4 <br />  <br /> SY1325               -                -              1.5 x 10   3.1 x 102-5. x 10                                                               1.0 x 10 <br /> SY1333               -                               2.8 x 10 &apos;.6 x1(      &lt;9.5x10 s                                                          &lt;9.5 x 10 <br />     Results average duplicate triplicate experiments. Strain 227 tester. <br />   &apos; Results atverage duplicate experiments. MATa M,Fal MFa2 strain SY1346. MATa        mzfal mif.2 strain SY1347. Genotypes presented Table 1. <br />  permissive temperature, suggesting .sr,nIl prod-              isolates (including SY1383) assayed room tempera- <br /> uct altered activity room temperature.                   ture 33&#194;&#176;C.  33&#194;&#176;C, quite good mating expected given mutant isolation scheme used.                             observed; roughly 6% mutant population mated. <br />    Lack requirement factor. results suggest                   Remarkably, 0.8% MATa ste2::LEU2 srmi-l srmnl activate signal transduction pathway                 mutants mated room temperature. value pheromone-independent fashion.  ex-                      higher mating efficiency observed MATa ste2 <br /> planations possible. truncated, partially                  SRMI strains (1.6 x 10 SY1383 harboring pSL749 <br /> functional receptor synthesized ste3::                     [Table 3]) suggests, argued experiments <br /> LEU2 allele  internal substitution                 involving Matct ste3 srmEn l strains, sr ml-l mutant <br /> complete deletion [14]) altered receptor                 product fully functional room temperature. Conse- <br /> interact factor create weak signal enhanced              quently, low temperatures, srml-l mutation sirt l-l mutant product. Alternatively, srmIl-l                 creates signal pheromone response pathway allow cryptic factor receptor function. case,               allows mating, albeit inefficiently. <br /> mating occur mating partner secreting                   Induction FUSI transcription. s,nIl-l mutation factor. examined ability srmn l mutants mate                identified basis suppression mating defect strain (SY1347) unable make factor                   ox ste3::LEU2 mutants. mating culmination genes encoding factor deleted (39).                   number physiological changes response phero- <br /> srinl mutants, SRMI strains, able mate                     mone, examined particular physiological changes, pheromone-deficient strain (Table 2).                     transcriptional induction FUSI (38, 65) cell cycle MAToC ste3::LEU2 srml mutant mating                       arrest  , determine srmnl-l mutants <br /> efficiency pheromone-producing phero-                      exhibit aspects normal pheromone  <br /> mone-deficient strains. results indicate factor              sponse. exponentially growing cultures sr,nl-1 required (x srml-l mutants mate                      mutants shifted room temperature 33&#194;&#176;C, observed mating presence cryptic                   increased amounts FUSI transcript (Fig. 1). truncated factor receptor. addition, srinl-l suppresses                 contrast, SRMI strains increase FUSI transcrip- mating defect mutant allele STE3                    tion shifted 33&#194;&#176;C (Fig. 1C). increase FUSI <br /> entire STE3 promoter coding sequences deleted (sste3::                   transcript levels srInl-l mutants ranged 7- 17-fold <br /> URA3; data shown).  .si-nl-l appears circumvent                  comparable regardless cell type  ax) receptor-mediated functions.                                             status receptor gene (Fig. 1 data shown). <br />    Suppression factor-receptor mutation. Genetic                     induction seen result temperature shift <br /> physiological experiments indicate pheromone                    comparable seen MATot STE3 srmnl-l <br /> receptors activate common response pathway (2, 19, 36,                     mutants incubated permissive temperature treated <br /> 45). determine SRMI pathway                         factor (Fig. 1).  srtml-1 mutants exhibit phe- <br /> common cx cells defines CL-specific                notype characteristic pheromone response  examined ability srn)1-l suppress                       incubated nonpermissive temperature absence <br /> mutation cL-factor-receptor gene (ste2::LEU2).                      pheromone. <br /> MATot STE2 sriinl-l strain (SY1327) changed MATa                        magnitude FUS1 induction ot sr I1 ste2::LEU2 srinl-1 (strain SY1383) gene replacement STE2 locus HO-mediated mating-type intercon-                                   TABLE 3. srninl-l suppresses ste2::LEU2 version MAT locus  Materials Methods).                                           ste4::LEU2 ste5::LEU2 <br /> create isogenic srmIl-l SRMI strains, transformed <br /> SY1383 YCp5O pSL749, YCp5O derivative                                                                                Mating efficiency&quot; carrying SRMI gene   determined                          Strain                  Genotype                       strain harboring: <br /> mating efficiencies transformants 33&#194;&#176;C (Table 3).                                                                       pSL749            YCp5O mating efficiency MATa ste2::LEU2 srml                                         MATa ste2::LEU2 srmnl-J <br /> mutants harboring SRMI plasmid, pSL749, low <br />                                                                              SY1383                                              1.6 x 10-s      4.4 x 10--- <br />                                                                              SY1357         MATot ste4::LEU2 srml-/            &lt;1.9 x 10-7      &lt;6.4 x 10- 6 <br /> (10-5 [Table 3]).  mating efficiency MATa                     SY1359         MATot ste5::LEU2 sriml-1           &lt;1.1 X 10- 7     &lt;1.8 x 10-6 <br /> ste2::LEU2 srinl-l mutants harboring YCp5O 4 x <br /> 10-2.  srm I suppresses deletion STE2 STE3                     &quot; mating efficiencies strains SY1383. SY1357. SY1359 harboring extent (compare Tables 2 3).                              pSL749 (SRMI YCp5O) YCp5O determined 33&#194;&#176;C  Materials <br />                                                                              Methods). transformants assayed, results    second way compare mating efficiency                     averaged. diploids observed SY1357 SY1359. data SRMI, MATa ste2::LEU2 srmnl                        presented represent limit detection assay. <br />  VOL. 9. 1989                                                                 SUPPRESSOR RECEPTOR MUTATIONS                                2687 <br />  <br />                                                                                   TABLE 4. Accumulation G1 srtnIl mutants <br />                                        B        1   2     3                                                   %   GI&quot;          % Shmoo&apos; <br />                  1    2    3                                             Strain     MAT cep-      SRMI <br />                                                                                             tor                22&#194;&#176;C        33&#194;&#176;C      22&#194;&#176;C    33&#194;&#176;C <br />                                                                        SY1333             -       +     41       5      43      4   0       0 <br />      FUS1                                FUS1                          SY1411             +       +     35       1      49      7   0       0 <br />                                                                        SY1325             -       -     52       7      73      7 2 &#194;&#177; 2 35 &#194;&#177; 9 <br />      URA3                                URA3                          SY1327&apos;            +       -     51       4      70      5 3 &#194;&#177; 3 15 &#194;&#177; 9 <br />                                                                        SY1355&apos;            -       -     62       3      86      3 15&#194;&#177; 1 64 &#194;&#177; 1 <br />                                                                        SY1383&apos;            -       -     70   &#194;&#177;   9      69      9 27&#194;&#177; 12 38 &#194;&#177; 15 <br />                                                                        SY1392             +       -     51   &#194;&#177;   3      68      5 3 &#194;&#177; 2 20 &#194;&#177; 7 <br />       Temp.      22 22 33             Temp. 22 22 33                   SY1393             +       -     52   &#194;&#177;   11     68      6 9 &#194;&#177; 6 19 &#194;&#177; 7 <br />                                                                        SY1394             -       -     59   &#194;&#177;   7      63      5 17&#194;&#177; 7 13 &#194;&#177; 5 <br />         aF       -+-                         aF   -   +                SY1225        aa     -&apos;      -     36   &#194;&#177;   4      29      8   0       0 <br />                                                                           &quot; percentage cells GI (mean &#194;&#177; standard deviation) presented. <br />                                                                        strain tested times. <br />                                                                            percentage cells total population projection <br />                       C          1    2      3    4 <br />                                                                        formations (shmoos) (mean &#194;&#177; standard deviation) presented. <br />                                                                          ` <br />                                                                             SY1327 SY1355 isogenic STE3 locus. SY1327                                                                        SY1383 isogenic MAT STE2 loci. <br />                                                                           d SY1392 SY1393 isogenic MAT locus, SY1393 <br />                                      1234                              SY1394 isogenic STE2 locus. <br />                                                                           &quot; SY1225 la diploid. STE2 STE3 expressed. SY1425 <br />                       FUSi                                             homozygous srml mutation. <br />  <br />                       URA 3                                                                                 tions. called shmoos (Table 4 Fig. 2). contrast,                                                                        morphology SRMI cells affected shift <br />                                                                        (Table 4 Fig. 2). shmoos induced 33&#194;&#176;C srml-J <br />                       Temp. 22 33 22 33                                mutants similar induced wild-type cells                       SRM1                                             pheromone (Fig. 2).  altered activity srml                                                                        protein caused temperature shift mimics cell cycle <br />    FIG. 1. Induction FUSI transcription.   Total RNA          arrest response pheromone response pathway. <br /> harvested exponentially growing cultures strain SY1115            Interestingly, extent arrest proportion (MATa srml-l). cases, cells grown 22&#194;&#176;C         cells showing projections varied receptor status (-) (+) induction factor (aF) 90 min,    srtnl mutants (Table 4). STE3 replaced cases, cells shifted 22 33&#194;&#176;C 1 h. RNA fractionated 1.5% agarose-6% <span id='am-89' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-90' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> gel, blotted         ste3::URA3  complete deletion STE3), strain nitrocellulose, hybridized SP6 probes FUSI URA3       lacked receptors exhibited arrest mRNAs described Materials Methods. Similar results      shmoos (compare SY1327 SY1355 [Table 4]). result <br /> obtained cultures incubated 33&#194;&#176;C 1 4 h. (B)   suggests wild-type receptor dampening <br /> Total RNA harvested cultures strain YY811 (MATa            effect signal generated shifting srml-l mutants ste3::LEU2 srmnl-l). cultures grown described panel    330C. incubated 33&#194;&#176;C 4 h  1 h)       examined effect srmnl-l cell cycle harvested. RNAs treated described         morphology cells. compare srml pheno- <br /> panel  (C) Total RNA harvested cultures strain           type cells, constructed isogenic strains. MATa <br /> YY811 harboring pSL749 (SRMI YCp5O; lanes 1 2) YCpSO <br /> (lanes 3 4). cultures grown described panel B,     srmnl-l strains switched MATa using HO endo- RNAs treated described panel                     nuclease gene control GALI promoter                                                                         Materials Methods). cells recovered <br />                                                                        regimen; cells behaved like original mutants, caused temperature shift      phero-          strain, indicating transformation switching pro- <br /> mone,   observed (38, 65).            cedures deleterious. srmnl-l mutants exhib- reduced induction seen probably high              ited phenotype srrml mutants cell <br /> basal level FUSI transcription srmnl-l mutants,           accumulation GI projection formation (compare reduction quantity FUSI transcript                 SY1327 SY1383, SY1392 SY1393 [Table 4]). <br /> present induction, basal level FUSI RNA             srml-l mutants exhibit characteristic features greatly reduced plasmid pSL749 (SRMI cloned                  pheromone response shifted nonpermissive <br /> YCp5O;  present srml strain (Fig. 1C).            temperature. high basal level indication                  differences srml-l srml-l mutants <br /> mutant product fully functional permissive          worth noting.  contrast findings temperature.                                                           srmnl-l mutants, presence absence receptor did   Accumulation G1 phase cell cycle.              influence phenotype srmnl-l mutants (SY1393 response  pheromone arrest cells G, phase          SY1394 [Table 4]). Second, srml-l mutants cell cycle (7, 68). examine cell cycle arrest             showed dramatic change proportion cells G1 <br /> srml-l mutants, shifted exponentially growing cultures              projections cultures shifted  mutants nonpermissive temperature 4 h,               missive nonpermissive temperature, srml-l mutants fixed cells formaldehyde, determined cell             essentially proportion cells G, cycle profile population. Shifting srmnl-l mutants           projections temperatures (Table 4). 33&#194;&#176;C caused accumulate GI form projec-               noted, believe mutant protein present srmnl-l <br />  2688          CLARK SPRAGUE                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />                       N~~~~~~O <br />  <br />  <br />  <br />        .40.    N <br />               04 <br />  <br />  <br />  <br />  <br />                                                 Ei   <br />                                                    X X               46                        F                     n <br />  <br />         01% <br />                        wov <br />                                                              ,tW*. <br />                                                                        .. <br />                                                                             &apos;* <br />  <br />  <br />  <br />                                                                                                  ll,-N <br />                                                                                                           4., <br />                                                                                                                  e 4p <br />                                                                                                                       fd <br />                            fA*.       <br />   FIG. 2. Cellular morphology srml-J mutants. Related wild-type  D) srml-l mutant (B E) cells growing exponentially 22&#194;&#176;C  B) shifted 33&#194;&#176;C 4 h (D E). comparison, exponentially growing MATa barl :LEU2 cells (C) exposed factor (625 ng/ml) 4 h (F). strains SY1333  D; MATa ste3::LEU2 SRMI), SY1325 (B E; MATa ste3::LEU2 srml , related strain SY1160 (MATa SRMI barl::LEU2). Photographs fixed cells taken described Materials Methods. <br />  <br /> strains fully functional temperatures permissive        determine increased projection formation growth cell division. subtle differences               33&#194;&#176;C dominant altered function srml-l <br /> phenotypes srml-J srml mutants suggest              protein, examined morphology MATalmata2ste3 :: cell types react differently differences            LEU21ste3::LEU2 srml-JISRMJ diploids. diploids did <br /> quantity quality mutant srml product.                     arrest exhibit projections 33 37&#194;&#176;C (data   noted  cells homozygous srml-J                 shown), indicating srml-l does dominant <br /> temperature sensitive growth. addition,          phenotype 33&#194;&#176;C. did exhibit cell cycle arrest projection formation          Cloning expression SRM1. SRMJ cloned (strain SY1225 [Table 4]). phenotype reminiscent           complementation temperature-sensitive phenotype cdc36 cdc39 mutants, arrest G1 haploid.             srml-J described Materials Methods, using  diploids homozygous cdc36                library yeast genomic fragments cloned YCp5O (51). <br /> cdc39 arrest uniform position cell cycle           apparently identical clones 10.6-kb insert temperature sensitive (49, 57).            obtained.  named pSL737 (Fig. 3), shown determine srml-J allelic cdc36, cdc39,             complement temperature sensitivity mating mutant genes confer G1 arrest,               phenotypes ste3::LEU2 srml-J mutants; plasmid- <br /> cdc28 cdc37 (49), srml-llcdc diploids                  bearing cells grow restrictive temperature mating. cdc mutations complemented srml  anal-              mate temperature. <br /> ysis meiotic products heterozygous diploids              determine pSL737 really harbors SRMJ  revealed srml-1 allelic cdc genes      gene fortuitously complements temper- <br />  (data shown).                                                    ature sensitivity srml-J, examined plas- <br />    experiments described performed 33&#194;&#176;C.            mid integrates homologous recombination SRM1 <br />  examined phenotype srml-J mutants 37&#194;&#176;C.            locus. transformed SY1115  relevant genotype  Shifting srml-l mutants 37&#194;&#176;C caused accumulate            MATa srml-1) BglII-cut pSL744 (EcoRI SphI  G1 essentially extent occurred              pSL737 YIp5) identified transformants  incubation 33&#194;&#176;C.  fewer projections formed           plasmid integrated locus corresponding  37&#194;&#176;C formed 33&#194;&#176;C (data shown).                cloned DNA. transformant crossed  VOL. 9, 1989 <br />  <br />                           <br />  <br />  <br />                        B/.Sa <br />  <br />  <br />                          -L <br />  <br />  <br />  <br />                          B <br />                                   ~                                    E B <br />                                           1 kb <br />  <br />  <br />  <br />  <br />                               -~~~~~~~~~~~~ ----- <br />  <br />  <br />  <br />  <br />                                             x_ <br />  <br />  <br />  <br />                                             -.40 <br />                                                    -~~ <br />                                                          P PE S <br />  <br />  <br />  <br />  <br />                                                          -- <br />                                                      -b0- -4-~ <br />                                                          -~ <br />                                                               - <br />  <br />  <br />  <br />  <br />                                                               - <br />                                                                   ~-0 <br />  <br />  <br />  <br />  <br />                                                                   ~-00-1 <br />                                                                         x <br />  <br />  <br />  <br />                                                                         t <br />  <br />  <br />  <br />    FIG. 3. SRMJ gene.   yeast genomic fragment (E1) cloned YCp5O ( <br />                                                                             Bg <br />                                                                             1 <br />                                                                                  SUPPRESSOR RECEPTOR MUTATIONS <br />  <br />  <br />  <br />  <br />                                                                                     t <br />                                                                                     p <br />                                                                                                                                                                               B <br />                                                                                          + <br />                                                                                              Bs 4; <br />  <br />  <br />  <br />  <br />                                                                                               0.5 kb <br />                                                                                                          P <br />                                                                                                              ACTIVITY <br />  <br />  <br />                                                                                                              _ <br />  <br />  <br />  <br />  <br />                                                                                                              Sa/B <br />                                                                                                                  |S <br />                                                                                                                     +T1 <br />                                                                                                                                          2689 <br />  <br />  <br />  <br />  <br />                                                                                       ) complements temperature-sensitive phenotype srml-J mutant  plasmid pSL737). arrows box ( ) indicate sites transposon insertion disrupt <br /> complementation, arrows box ( T ) indicate insertion sites disrupt complementation. subclones pSL737 <br /> (E) (+) (-) complement srmi-J ORF SRMI (0) shown. Abbreviations: E, EcoRI; B, BamHI; Sa, Sau13A; <br /> P, Pstl; S, Sall; X, Xhol; Bg, BgIII. entire clone tested Xhol BgIII sites. (B) box represents fragment pSL737 subcloned sequencing. Sets deletions created Xhol PstI sites, arrows indicate deletion subclones sequenced direction sequencing  Materials Methods). ORF (0) direction transcription (-) shown. <br />  <br /> wild-type strain (X2180-1A-L1-U1), meiotic prod-                    147 include 11 acidic amino acids 19 total residues, ucts analyzed. segregants diploid                  residues 470 482 include 8 acidic 2 basic amino acids <br /> (15 tetrads examined) temperature sensitive, indicating                13 total residues (Fig. 4). Comparison sequence pSL744 integrated srml-J locus.                   ORF National Biomedical Research Foundation cloned DNA linked SRMI gene.                                  protein data base using program FASTP (35) did    determine multiple copies SRMI gene                    reveal strong homologies. searched sequences present yeast genome, performed high-                        similar consensus sequences known functional <br /> stringency hybridizations yeast genome                  groups proteins, e.g., zinc fingers (29), calcium-binding <br /> cloned SRMJ gene. DNA fragments observed                        EF hands (31), protein kinase sequence motifs (17), Southern blots predicted restriction map                sequence motifs common subunit G proteins (64). SRMJ gene, indicating copy                      closest identity residues 60 66 (GLGPLAK) SRMJ gene present yeast genome. experi-                     consensus nucleotide, particularly GTP, binding site <br /> ments revealed sequences pSL739                     (GXGXXGK [16]). <br /> srml-l-complementing plasmid created deletion                        transcript approximately 1.6 kb detected 4-kb EcoRI fragment pSL737 [Fig. 3A]) colinear                       cell types    ala) (Fig. 5). quantity genome (data shown).                                         transcript unaffected addition pheromone (Fig. 5). <br />    SRMJ gene mapped 10.6-kb insert                    Hybridization poly + RNA single-stranded probes <br /> subcloning transposon-mediated mutagenesis (22)                      revealed SRMI transcript orientation <br /> (Fig. 3). procedures defined small region clone              ORF (data shown).  sequenced 1.8 kb region (Fig.                SRMI essential gene. presumptive null alleles 3). ORF encoding 482 amino acids  ATG                      SRMI created replacing 0.8-kb BamHI fragment <br /> codon serves start codon protein) ob-                    SRMI LEU2 URA3 (Fig. 3 Materials <br /> served (Fig. 4). ORF disrupted deletions                 Methods). heterozygous srml-lIsrml::LEU2 diploid transposons destroy complementing activity,                      created step gene replacement. Tetrad analysis <br /> presume corresponds SRMI product.                        room temperature revealed diploid segregated fusion GAL] promoter upstream ORF                     viable inviable spores. viable spores <br /> XhoI site) produced hybrid gene showed galactose-                    Leu+, indicating SRMJ essential gene. <br /> dependent complementation srml-1.                                        srml-J mutants tended arrest G1 unbudded <br />   hydropathic profile (33) deduced protein se-                   cells cells projections, expected srml:: <br /> quence predicts hydrophilic protein. regions                   LEU2 cells display similar phenotype.  protein stretches charged residues: residues 129                 inviable (srml::LEU2) spores germinated arrested  2690     CLARK SPRAGUE                                                                                         MOL. CELL. BIOt. <br />  <br />                                   -88   CTCGAGAATATTAAAAGTCATCAAGACAT1CCTCTTTAAGTCAAATCCTGGATCCA(,AAACTTAAAAAG1TrGTCATATAAAGTeTTAAA <br />    1   ATGGTCAAAAGAACAGTCGCCACCAATGGAGACGCGTCTGGCGCTCATAGAGCAAAGAAAATGTCTAAAACTCATGCTTCTCATATCATAAACGCTCAAGAAGACTACAAGCATATGTAC <br />    1   MetValLysArgThrValAlaThrAsnGlyAspAlaSerGlyAlaHisArgAlaLysLysMetSerLysThrHisAlaSerHisIlelleAsnAlaGlnGluAspTyrLysiHisMetTyr <br />  121   TTAAGTGTCCAACCATTGGATATATTTTGCTGGGGTACTGGTTCTATGTGTGAGCTTGGGTTGGGTCCACTAGCCAAAAATAAGGAAGTTAAAAGACCAAGGTTAAACCCATTTTTACCT <br />   41   LeuSerValGlnProLeuAspIlePheCysTrpGlyThrGlySerMetCysGluLe*lyLeuGlyProLeuAlaLyssnLysGIuValLysArgProArgLeuAsnProPheLeuPro <br /> 241    CGCGATGAGGCCAAGATCATTTCTTTTGCAGTCGGTGGTATGCACACCTTAGCCCTTGATGAAGAAAGCAATGTATGGTCCTGGGGTTGCAATGACGTGGGTGCGTTGGGCAGAGATACT <br />  81    ArgAspGluAlaLysIleIleSerPheAlaValGlyGlyMetHisThrLeuAlaLeuAspGluGluSerAsnValTrpSerTrpGlyCysAsnAspValGlyAlaLeuGlyArgAspThr <br /> 361    TCCAACGCCAAGGAACAACTCAAGGACATGGACGCCGATGACTCAAGTGATGACGAAGATGGTGATTTGAACGAATTGGAATCTACGCCCGCTAAAATCCCAAGAGAGTCATTTCCTCCC <br />  121   SerAsnAlaLysGluGlnLeuLysAspMetAspAlaAspAspSerSerAspAspGluAspGlyAspLeuAsnGluLeuGluSerThrProAlaLysIleProArgGluSerPheProPro <br />  481   TTAGCGGAGGGACACAAGGTCGTGCAATTGGCAGCTACAGACkATATGAGTTGCGCCCTGTTCAGCAATGGCGAGGTATACGCATGGGGTACTTTCCGTTGCAATGAAGGTATCCTAGGG <br />  161   LeuAlaGluGlyHisLysValValGlnLeuAlaAlaThrAspAsnMetSerCysAlaLeuPheSerAsnGlyGluValTyrAlaTrpGlyThrPheArgCysAsnGluGlyIleLeuGly <br /> 601    TTTTACCAAGATAAAATTAAAATTCAAAAAACTCCATGGAAAGTGCCTACATTCTCTAAGTACAATATTGTCCAATTAGCTCCCGGTAAGGATCACATACTTTTCTTAGATGAAGAGGGT <br /> 201    PheTyrGlnAspLysIleLysIleGlnLysThrProTrpLysValProThrPheSerLysTyrAsnIleValGlnLeuAlaProGlyLysAspHisIeLeuPheLeuAspGluGluGly <br /> 721    ATGGTGTTCGCCTGGGGTAACGGTCAGCAAAACCAATTAGGAAGAAAAGTTATGGAAAGGTTCCGTTTAAAGACTTTGGATCCCAGACCATTTGGGTTAAGACATGTAAALATACATCGCA <br /> 241    MetValPheAlaTrpGlyAsnGlyGlnGlnAsnGlnLeuGlyArgLysValMetGluArgPheArgLeuLysThrLeuAspProArgProPheGlyLeuArgHisValLysTyrIleAla <br />  841   TCTGGCGAAAACCATTGCTTTGCTTTAACAAAAGACAACAAGTTAGTAAGCTGGGGGTTGAATCAGTTTGGTCAATGTGGTGTTTCAGAGGATGTGGAAGACGGTGCGCTTGTTACAAAG <br />  281   SerGlyGluAsnHisCysPheAlaLeuThrLysAspAsnLysLeuValSerTrpGlyLeuAsnGlnPheGlyGlnCysGlyValSerGluAspValGluAspGlyAlaLeuValThrLys <br />  961   CCAA.AAAGACTAGCGCTACCTGATAACGTTGTTATCAGAAGCATTGCTGCTGGTGAACATCACTCATTGATCCTATCCCAAGATGGCGATTTGTATTCGTGTGGTAGACTAGACATGTTT <br />  321   ProLysArgLeuAlaLeuProAspAsnValValIleArgSerIleAlaAlaGlyGluHisHisSerLeuIleLeuSerGlnAspGlyAspLeuTyrSerCysGlyArgLeuAspMetPhe <br /> 1081   GAAGTTGGTATCCCAAAAGACAATTTACCCGAATATACTTACAAAGATGTTCATGGAAAAGCACGTGCCGTGCCACTTCCAACTAAGTTGAATAATGTTCCGAAGTTCAAAAGCGTTGCG <br />  361   GluValGlyIleProLysAspAsnLeuProGluTyrThrTyrLysAspValHisGlyLysAlaArgAlaValProLeuProThrLysLeuAsnAsnValProLysPheLysSerValAla <br /> 1201   GCGGGGTCTCACCATTCAGTCGCGGTAGCTCAAAATGGCATTGCCTATTCTTGGGGTTTTGGTGAAACCTACGCCGTTGGATTAGGCCCTTTTGAGGACGACACTGAAGTTCCAACCAGA <br />  401   AlaGlySerHisHisSerValAlaValAlaGInAsnGlyIleAlaTyrSerTrpGlyPheGlyGluThrTyrAlaValGlyLeuGlyProPheGluAspAspThrGluValProThrArg <br /> 1321   ATCAAGAATACTGCTACACAGGATCATAACATCATTTTAGTCGGGTGTGGAGGCCAATTTTCCGTGTCTGGTGGTGTTAAGCTGTCTGATGAGGACGCAGAAAAGAGAGCGGATGAAATG <br />  441   IeLy5AsnThrAlaThrGInAspHisAsnllelleLeuValGlyCysGlyGlyGlnPheSerValSerGlyGlyValLysLeuSerAspGluAspAlaGluLysArgAlaAspGluMet <br /> 1441   GATGATTAAGGTAGTAGTACAAAGATAAATAAAAGAAAATCGACATAATGCACCTCAGTATAAATTGTAAAATATAACATAAATCAAAGAAAAACGAGAAAACGATATGATTATCCGATA <br />  481   AspAsp   482 <br />  <br /> 1561 <br />  <br /> 1681   ATGAAATCAACACTGAGTAAGTATTATGTAATCCATCCGCTCGTCCAGCCCTTCAGGTTCTTCCCTCTTCTGCACGT      1757 <br />   FIG. 4. Sequence SRMI translation product. region 1.835 kb Xliol site Pstl site SRMI sequenced described Materials Methods. translation ORF shown nucleotide sequence. frame <br /> methionine residues near amino terminus underlined twice. boxed amino acids weak match nucleotide binding site <br /> (GXGXXGK [16]). stretches charged (predominantly acidic) residues underlined. standard letter abbreviations amino acids used. <br />  large-budded cell physically separated       advantage. Consistent interpretation obser- unbudded (G1) cells 12% time. rest          vation continued propagation dip- large-budded cells represent cells arrested G1       loids yielded tetrads viable spores remain physically joined, G1 initiates end     resulted producing tetrads viable <br /> mitosis cytokinesis cell separation (48). viable      spores (data shown). alternate hypothesis based spores produced, inferred inviable         analysis TUBI gene, encodes x-tubulin. <br /> spores cells, indicating null allele     Aneuploidy observed deletions TUBI phenotype cell types. total         created (55), SRMI product, like TUBI <br /> independent srml srmm::LEU2 transformants  dif-            product. required fidelity chromosome segregation. <br /> ferent diploids) analyzed results.                   Lack suppression ste4 steS mutations.    Given results, properties additional trans-    genes identified products required formants unexpected suggested diploid cells        pheromone response mating. include STE4 quantity mutant srinl product gene         STE5, required ox cells (19, 36). insufficient optimal growth. transformants           determine order action respect SRMI srml-l/srml ::LEU2 srm1-//srml:: URA3) yielded tet-             pheromone response pathway, double <br /> rads viable spores. spores         mutants MATot ste4::LEU2 s}srml MATo ste5::LEU2 <br /> tetrads Leu- Ura&apos;, indicating sr-inl null          srl-t71/- gene disruption STE4 STE5 strain <br /> allele present. Outcrossing Ura1 haploids          SY1327 (MATot si/ 1). ste4::LEU2 srmll-1 ste5:: wild-type strain revealed haploid Ura strains         LEU2 smnl-l mutants (SY1357 SY1359, respectively) aneuploid harbored sirIIIl-l srnzl::URA3          did arrest exhibit projections distinct chromosomes segregated meiosis (data        shifted 33&#194;&#176;C (data shown), starting strain <br /> shown).  set diploid transformants            ste3:: URA3 derivative starting strain did arrest undergone nondisjunction event mitosis          exhibit projection formation (SY1327 SY1355 [Table <br /> caused particular transformants growth              4]). SY1357 SY1359 temperature sensitive  VOL. 9. 1989                                                                  SUPPRESSOR RECEPTOR MUTATIONS                      2691 <br />  <br />                              1     2       3       4    5                   srinl-l mutants exhibited phenotype mimicked <br />                                                                          response wild-type cells pheromone. particular,                                                                          elevated temperatures, srInl mutants induced expression <br />                  SRM1                                                    FUSI, accumulated G1 phase cell cycle,                                                                          formed projections. extent accumulation G1                                                                          great occurs pheromone induction. <br />                  URA3                                                    explanation failure cells arrest G,                                                                          partial adaptation signal generated permissive <br />                                                                          temperature srinIl-l mutation. high basal level                 MAT                              &quot;               FUS1 RNA ability receptorless srin l mutants                                                                          mate inefficiently, room temperature, suggest             Pheromone        -     +               +                     low temperatures srinl-l mutation creates feeble signal <br />     FIG. 5. SRMI transcription. Total RNA harvested             pheromone response pathway. cells exponentially growing cultures MA Ta  , MATot (oa), MA Ta/a_     adapted presence signal low temperature,   cells untreated (-) treated appropriate      respond increased signal high temperature <br /> pheromone 90 min (-+ indicates factor cells, factor    arrest G1. result  cells). strains isogenic series. HR125-5d. SY816.                                                                          nonhomogeneous response. <br /> SY817 (Table 1). RNA fractionated 1.5% agarose-6%              Surprisingly, strength GI accumulation pheno- <br /> formaldehyde gel. blotted nitrocellulose. hybridized         type sensitive presence absence phero- <br /> nick-translated pSL743, detects SRMI URA3 transcripts.   described Materials Methods.                                 mone receptor, especially  cells; extent accumu- <br />                                                                          lation GI projection formation STE3 <br />                                                                          srmI2-l cells ste3 srtnIl-J cells. finding suggests <br /> growth, unexpected ax sriml-1/srmil               receptor quench srinI generated signal. diploids, respond pheromone,                    concert possibility, temperature sensitive. second assay suppression              carboxyterminus factor receptor component ste mutants srinl-, asked double                 signal attenuation enables cells recover <br /> mutants mate. create isogenic SRMI srmn                   factor-induced cell cycle arrest (30, 50). strains, transformed SY1357 SY1359 pSL749                    SRMl product constituent pheromone <br /> (SRMI cloned YCp5O) YCp5O determined                   response pathway target receptor-mediated signal mating efficiencies resulting transformants.           attenuation. <br /> Strains bearing plasmid undetectable levels                   Contrary simple expectation model, did <br /> mating, indicating srinIl mutation did suppress           observe increase G1 accumulation projection mating defect ste4 steS mutant strains (Table           formation STE2  (x-factor-receptor gene)  <br /> 3).  sriml-J requires functional STE4 STE5 prod-                leted ssrmIl mutants (Table 4).  srm l ucts activate pheromone response pathway.                         sri-Itl- mutants differ regards  exam- <br />                                                                          ple, compared sr,nl-l cells, srInl cells exhibit <br />                            DISCUSSION                                    features pheromone response temperatures  <br />                                                                          missive growth; room temperature, srmIl-l ste2 cells <br />    Identification srml-1. sought identify components            appreciable mating efficiency high proportion pheromone response pathway yeast.                    cells G1 phase cell cycle study initiated, defined components            formed projections. basis subtle difference pathway pheromones receptors.                      phenotype srm-l-1 cells unknown, identify intracellular components pathway,                    discrepancies sort noted study <br /> screened pseudorevertants deletion factor-             genes involved pheromone response. exam- <br /> receptor gene (ste3::LEU2). MATax ste3::LEU2 strains                  ple, mutation CDC39 sufficient suppress mating unable mate, searched mating-proficient                    defect ste4-3 cells, cells, mutation strains, reasoning acquisition mating ability           CDC39 SST2 required suppression (26, 57). mutation downstream elements                  Role SRMI. phenotypes sr,n      I mutants, induc- <br /> pheromone response pathway. mutation initi-                 tion FUSI. accumulation Gl, mating absence <br /> ate signaling autonomously, absence factor.              pheromone receptor, suggest SRMI product allow receptor-deficient strain mate.                       component pheromone response pathway.    Using strategy, isolated suppressor ste3::               sini l recessive mutation allows receptorless <br /> LEU2, called srinil-1. similar scheme, screening              strains mate, inactivation SRM1 probably step suppressors (x-factor-receptor deletion, Jahng et al.          pheromone response pathway.  absence (23) isolated mutations genes: GPAI (SCGI). yeast             pheromone. wild-type SRM1 function maintain <br /> homolog subunit mammalian G proteins (11. 41.               expression pheromone-responsive genes low basal <br /> 43), CDC72, CDC73. expected cell cycle arrest               level allow entry S phase cell cycle. response pheromone. suppressors                  results presented suggest possible locations temperature sensitive growth, suppress              SRM1 pathway. <br /> mating defect high temperature. addition, previously                imagine signal transduction pathway identified cell-division-cycle gene. CDC36, similar                linear, branches receptor various <br /> phenotype mutant (23). SRMI allelic              targets signal. words, terminal physio- suppressors (unpublished observations).  muta-                logical change monitor, e.g., transcription induction tions genes cause temperature-sensitive            GI arrest, activated chain events single <br /> cell cycle arrest G1 restore mating ability recep-            path. Genetic evidence indicates early step torless mutants.                                                         pathway involves heterotrimeric G protein (11, 23, 41, 67). <br />  2692     CLARK SPRAGUE                                                                                           MOL. CELL. BIOL. <br />  analogy mammalian systems  reference 64),        conjunction undiscovered G proteins thought absence pheromone,  1, -y         essential viability. <br /> subunits G protein complexed inactive,            summary, mutation SRMI confers mating ability subunit bound <span id='am-23' about='obo:IMR_0001357' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00035" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="GDP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0001357" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasAlternativeId" content="IMR:0200191" datatype="xsd:string"></span>GDP</span>. pheromone binds           strains lack pheromone receptors. phenotypes receptor, GTP exchanged GDP 13y subunits        srml-l mutants suggest SRM1 involved pher- released complex. contrast usual              omone response pathway normal vegetative growth. <br /> situation mammalian systems, S. cerevisiae thought     complete understanding nature role 13y complex, ot subunit bound        SRM1 pheromone response awaits biochemical <br /> GTP, propagates signal pheromone bound         genetic analysis. <br /> receptor interacting effector unidentified. STE4 likely encodes 13 subunit yeast G                               ACKNOWLEDGMENTS <br /> protein (67), finding srml-J suppress deletion        thank members laboratory discussions; Vivian STE4 implies SRM1 functions 13y complex.        MacKay, Anne Burkholder, Lee Hartwell plasmids gene possibility SRM1 component signal          disruptions; Jeremy Thorner Tom Stevens yeast genomic <br /> transduction pathway operates receptor            library; Susan Michaelis Ira Herskowitz strain SM1229; M. G protein. second possibility SRM1               Andrew Hoyt David Botstein transposon mutagenesis <br /> interacts G-protein subunits affects activ-      Nick Davis, David Hagen, Karen Sprague ity. instance, SRM1 promote reassociation         comments manuscript. <br />                                                                       work supported National Research Service award <br /> subunit 13y complex, catalyzing GTP            1F32 GM10711  K.L.C.) Public Health Service research grant <br /> hydrolysis ax subunit. possibility SRM1     GM30027  G.F.S.), National Institutes Health. required expression ax subunit structural gene.   G.F.S. recipient Faculty Research Award FRA-282 scenario, inactivation SRM1 cause depletion       American Cancer Society. G protein subunit free 13y subunits interact effector initiate signaling. Similarly,                              LITERATURE CITED <br /> SRM1 involved repressing expression 1           1. Beggs, J. D. 1978. Transformation yeast replicating <br /> y subunit structural gene. Inactivation SRM1                  hybrid plasmid. Nature (London) 275:104-109. cause increase free 13y complex.                    2. Bender,  D., G. F. Sprague, Jr. 1986. Yeast peptide <br />    inability srml-1 suppress ste4 muta-             pheromones, factor ao-factor, activate common response <br />                                                                        mechanism target cells. Cell 47:929-937. <br /> tions implies SRM1 acts G protein signal         3. Betz, R., W. Duntze, T. R. Manney. 1978. Mating-factor- <br /> transduction, second interpretation offered.             mediated sexual agglutination Saccharomyces cerevisiae. interpretation supposes absence               FEMS Microbiol. Lett. 4:107-110. <br /> pheromone, low level signal propagated        4. Blumer, K. J., J. E. Reneke, J. Thorner. 1988. STE2 pathway. example, free                gene product ligand binding component factor <br /> 13y complex. support view, basal level tran-            receptor Saccharomnyc es cerevisiae. J. Biol. Chem. 263: <br /> scription pheromone-inducible genes FUSI                 10836-10842. <br />                                                                     5. Boeke, J. D., F. LaCroute, G. R. Fink. 1984. positive <br /> reduced 40-fold ste4 mutants (38). extrapola-            selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br /> tion, imagine suppression srml-l              ylase activity yeast: 5-fluoro-orotic acid resistance. Mol Gen. <br /> mating defect receptor mutants dependent low            Genet. 197:345-346. <br /> level signal exists cells stimulated phero-     6. Broach, J. R., J. N. Strathern, J. B. Hicks. 1979. Transfor- <br /> mone. view, SRM1 functions G protein,                mation yeast: development hybrid cloning vector                                                  isolation CANI gene. Gene 8:121-133. <br />    SRM1 role ih vegetative life cycle            7. Bucking-Throm, E., W. Duntze, L. H. Hartwell, T. R. <br /> addition role mating. diploids contain        Manney. 1973. Reversible arrest haploid yeast cells                                                                        initiation DNA synthesis diffusible sex factor. Exp. Cell <br /> signal transduction G protein transcription             Res. 76:99-110. <br /> structural genes repressed (11, 41, 67). Nonetheless,            8. Burkholder,  C., L. H. Hartwell. 1985. yeast factor <br /> diploids homozygous srml-l temperature sensitive               receptor: structural properties deduced sequence growth   arrest growth               STE2 gene. Nucleic Acids Res. 13:8463-8475. <br /> particular position cell cycle. Additional evidence     9. Chatoo, B. B., F. Sherman, D.  Azubalis, T.  Fjellstedt, D. <br /> SRM1 role vegetative life cycle comes                Mehnert, M. Ogur. 1979. Selection lys2 mutants analysis srml null phenotype haploid cells. Spores           yeast Saccharomyces cerevisiae utilization amino- <br /> harboring srml::LEU2 inviable form projec-              adipate. Genetics 93:51-65. <br /> tions; arrest G1 phase cell cycle     9a.Clark, K. L., N. G. Davis, D. K. Wiest, J.-J. Hwang-Shum,                                                                        G. F. Sprague, Jr. 1988. Response yeast cells factor <br /> uncertain. model explain difference phe-             pheromone: topology receptor identification notype srml::LEU2 srml-l mutants srml                 component response pathway. Cold Spring Harbor Symp. <br /> mutants substantially reduced residual SRM1 ac-               Quant. Biol. 53:611-620. <br /> tivity nonpermissive temperature, condition          10. Cross, F., L. H. Hartwell, C. Jackson, J. B. Konopka. 1988. <br /> activates pheromone response pathway allows                Conjugation Saccharoimvces c erevisiae. Annu. Rev. Cell <br /> vegetative functions SRM1 fulfilled.           Biol. 4:429-457. <br /> complete absence SRM1 product, case          11. Dietzel, C., J. Kurjan. 1987. yeast SCGI gene: srml::LEU2 strains, inability carry functions           Ga-like protein implicated  factor response <br /> required vegetative growth dictates phenotype.            pathway. Cell 50:1001-1010. <br />                                                                    12. Dixon, R.  F., B. K. Kibilka, D. J. Strader, J. L. Benovic, possible roles SRM1 mating discussed            H. G. Dohlman, T. Frielle, M.  Bolanowski, C. D. Bennet, E. extended include similar role                 Rands, R. E. Diehl, R.  Mumford, E. E. Slater,  S. Sigal, <br /> vegetative life cycle. example, SRM1 involved           M. S. Caron, R. J. Lefkowitz, C. D. Strader. 1986. Cloning <br /> GTP hydrolysis subunit, SRM1 work                   gene cDNA mammalian 3-adrenergic receptor  VOL. 9, 1989                                                                 SUPPRESSOR RECEPTOR MUTATIONS                          2693 <br />  <br />     homology rhodopsin. Nature (London) 321:75-79.                     177:753-754. <br /> 13. Fehrenbacher, G., K. Perry, J. Thorner. 1978. Cell-cell         35. Lipman, D. J., W. R. Pearson. 1985. Rapid sensitive <br />     recognition Saccharomyces cerev&apos;isiae: regulation mating-         protein similarity searches. Science 227:1435-1441. <br />     specific adhesion. J. Bacteriol. 134:893-901.                       36. MacKay, V., T. R. Manney. 1974. Mutations affecting <br /> 14. Hagen, D. C., G. McCaffrey, G. F. Sprague, Jr. 1986.                sexual conjugation related processes Saccharomvces <br />     Evidence yeast STE3 gene encodes receptor                 cerevisiae. II. Genetic analysis nonmating mutants. Genetics <br />     peptide pheromone factor: gene sequence implications          76:272-288. <br />     structure presumed receptor. Proc. Natl. Acad. Sci.      37. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     USA 83:1418-1422.                                                       cloning: laboratory manual. Cold Spring Harbor Laboratory, <br /> 15. Hagen, D. C., G. F. Sprague, Jr. 1984. Induction             Cold Spring Harbor, N.Y. <br />     yeast specific STE3 gene peptide pheromone factor.       38. McCaffrey, G., F. J. Clay, K. Kelsay, G. F. Sprague, Jr. <br />     J. Mol. Biol. 178:835-852.                                              1987. Identification regulation gene required cell <br /> 16. Halliday, K. 1984. Regional homology GTP-binding proto-              fusion mating yeast Saccharomvces cerevisiae. <br />     oncogene products elongation factors. J. <span id='am-18' about='obo:IMR_0001355' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-19' property="oboInOwl:id" content="IMR:0001355" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="cyclic nucleotide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:23447" datatype="xsd:string"></span>Cyclic Nucleotide</span>          Mol. Cell. Biol. 7:2680-2690. <br />     Res. 9:435-448.                                                     39. Michaelis, S.,  Herskowitz. 1988. factor pheromone <br /> 17. Hanks, S. K.,  M. Quinn, T. Hunter. 1988. protein             Saccharomyces cerev&apos;isiae essential mating. Mol. Cell. <br />     kinase family: conserved features deduced phylogeny          Biol. 8:1309-1318. <br />     catalytic domains. Science 241:42-52.                               40. Misuzawa, S., S. Nishimura, F. Seela. 1986. Improvement 18. Hartig, , J. Holly, G. Saari, V. L. MacKay. 1986. Multiple        dideoxy chain termination method DNA sequencing     regulation STE2, mating-type-specific gene Saccharonvy-         use deoxy-7-deazaguanosine <span id='am-64' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-65' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span> place dGTP. <br />     ces cerevisiae. Mol. Cell. Biol. 6:2106-2114.                           Nucleic Acids Res. 14:1319-1324. <br /> 19. Hartwell, L. 1980. Mutants Saccharomvces cerei&apos;isiae unre-       41. Miyajima, , M. Nakafuku, N. Nakayama, C. Brenner,  <br />     sponsive cell division control polypeptide mating hor-            Miyajima, K. Kaibuchi, K.  Arai, Y. Kaziro, K. Matsu- <br />     mone. J. Cell Biol. 85:811-822.                                         moto. 1987. GPAI, haploid-specific essential gene, encodes 20. Henikoff, S. 1984. Unidirectional digestion exonuclease III        yeast homolog mammalian G protein involved <br />     creates targeted breakpoints DNA sequencing. Gene 28:               mating factor signal transduction. Cell 50:1011-1019. <br />     351-359.                                                            42. Moir, D., S. E. Stewart, B. C. Osmond, D. Botstein. 1982. <br /> 21. Herskowitz,  1988. Life cycle budding yeast Saccharo-          Cold-sensitive cell-division-cycle mutants yeast: isolation, <br />     myces cerevisiae. Microbiol. Rev. 52:536-553.                           properties, pseudoreversion studies. Genetics 100:547-563. <br /> 22. Huisman, O., W. Raymond, K.-U. Froehlich, P. Errada, N.             43. Nakafuku, M., H. Itoh, S. Nakamura, Y. Kaziro. 1987. <br />     Kleckner, D. Botstein, M.  Hoyt. 1987. TnIO-lacZ-               Occurrence Saccharomnvces cerevisiae gene homologous <br />     kanR-URA3 gene fusion transposon insertion mutagenesis              cDNA coding subunit mammalian G proteins. <br />     fusion analysis yeast bacterial genes. Genetics              Proc. Natl. Acad. Sci. USA 84:2140-2144. <br />     116:191-199.                                                        44. Nakayama, N.,  Miyajima, K. Arai. 1985. Nucleotide <br /> 23. Jahng, K.-Y., J. Ferguson, S.  Reed. 1988. Mutations         sequence STE2 STE3, cell type-specific sterile genes <br />     gene encoding subunit Saccharoinvces cereWisiae G            Saccharomvces cerevisiae. EMBO J. 4:2643-2648. <br />     protein indicate role mating pheromone signaling. Mol. Cell.   45. Nakayama, N.,  Miyajima, K. Arai. 1987. Common signal <br />     Biol. 8:2484-2493.                                                      transduction shared STE2 STE3 haploid cells <br /> 24. Jarvis, E. E., D. C. Hagen, G. F. Sprague, Jr. 1988.                Saccharomvces cerei&apos;isiae: autocrine cell-cycle arrest     Identification DNA segment necessary sufficient        forced expression STE2. EMBO J. 6:249-254. <br />     specific gene control Saccharomvces cereWisiae: impli-     46. Nasmyth, K. , S.  Reed. 1980. Isolation genes     cations regulation specific specific genes. Mol.         complementation yeast: molecular cloning cell-cycle <br />     Cell. Biol. 8:309-320.                                                  gene. Proc. Natl. Acad. Sci. USA 77:2119-2123. <br /> 25. Jenness, D. D.,  C. Burkholder, L. H. Hartwell. 1983.         47. Nathans, J., D. S. Hogness. 1984. Isolation nucleotide <br />     Binding factor pheromone yeast cells: chemical            sequence gene encoding human rhodopsin. Proc. Natl. <br />     genetic evidence factor receptor. Cell 35:521-529.             Acad. Sci. USA 81:4851-4855. <br /> 26. Jenness, D. D., B. S. Goldman, L. H. Hartwell. 1987.            48. Pringle, J., L. H. Hartwell. 1981. Saccharomyces <br />     Saccharomrvces cere0isiae mutants unresponsive factor              cerevisiae cell cycle, p. 97-142. J. N. Strathern, E. W. Jones. <br />     pheromone: factor binding extragenic suppression. Mol.            J. R. Broach (ed.), molecular biology yeast <br />     Cell. Biol. 7:1311-1319.                                                Saccharoinvces: life cycle inheritance. Cold Spring Harbor <br /> 27. Jensen, R. E.,  Herskowitz. 1984. Directionality              Laboratory, Cold Spring Harbor, N.Y. <br />     regulation cassette substitution yeast. Cold Spring Harbor    49. Reed, S.  1980. selection S. cerei&apos;isiae mutants defec- <br />     Symp. Quant. Biol. 49:97-104.                                           tive start event cell division. Genetics 95:561-577. <br /> 28. Johnston, M., R. W. Davis. 1984. Sequences regulate        50. Reneke, J. E., K. J. Blumer, W. E. Courchesne, J. Thorner. <br />     divergent GALI-GALIO promoter Saccharoinyces cere-               1988. carboxy-terminal segment yeast factor recep- <br />     *isiae. Mol. Cell. Biol. 4:1440-1448.                                   tor regulatory domain. Cell 55:221-234. <br /> 29. Klug, , D. Rhodes. 1987. &apos;Zinc fingers&apos;: novel protein      51. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. R. <br />     motif nucleic acid recognition. Trends Biochem. Sci. 12:            Fink. 1987. Saccharomvces cerevisiae genomic plasmid <br />     464-469.                                                                bank based centromere-containing shuttle vector. Gene <br /> 30. Konopka, J. B., D. D. Jenness, L. H. Hartwell. 1988.            60:237-243. <br />     C-terminus S. cerei&apos;isiae pheromone receptor mediates      52. Rose, M., B. Price, G. R. Fink. 1986. Saccharomyces <br />     adaptive response pheromone. Cell 54:609-620.                     cerevisiae nuclear fusion requires prior activation f <br /> </body></html>