<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>       COMMENTARY                                                                                                                  COMMENTARY <br /> Worm 2:2, e22188; April June 2013; &#194;&#169; 2013 Landes Bioscience <br />  <br />  <br />  <br />  <br /> Catalytic-independent roles UTX-1 C. elegans development <br /> Julien Vandamme Anna Elisabetta Salcini* <br /> Biotech Research Innovation Centre (BRIC); University Copenhagen; Copenhagen, Denmark <br />  <br />  <br />  <br />  <br />                                                      W       e recently analyzed functional <br />                                                              roles UTX-1 develop- <br />                                                      ment. utx-1 essential gene required <br />                                                                                                      post-translational modifications,                                                                                                      methylation lysines arginines, phos- <br />                                                                                                      phorylation serines threonines,                                                      correct embryonic post-             ubiquitylation sumoylation lysines. <br />                                                      embryonic development C. elegans,            modifications proposed                                                      displays H3K27me3 demeth-             represent combinatorial &#226;&#8364;&#339;histone code&#226;&#8364;?, <br />                                                      ylase activity. Rescue experiments dem-         specifies function affected <br />                                                      onstrated enzymatic activity        genomic regions terms chromosome <br />                                                      UTX-1 relevant role devel-    segregation, cell cycle progression, DNA <br />                                                      opment. phenotypes associated          replication/repair transcriptional activ- <br />                                                      loss UTX-1  instead, result       ity.3,4  modifications <br />                                                      compromised functions UTX-1-           appear play important roles dividing <br />                                                      containing complex. discuss         genome transcriptionally active/ <br />                                                      possible mechanisms UTX-1              relaxed inactive/compacted regions. <br />                                                      contributes normal development.              addition regulating chromatin com- <br />                                                                                                      paction, modified nucleosomes provide <br />                                                                    Introduction                      binding sites regulatory proteins  <br />                                                                                                      turn, stabilize chromatin sig- <br />                                                      cells multicellular          nature provide platform recruit <br />                                                      organism contain genes,       additional factors.5 <br /> Keywords: development, epigenetics,                  used way cell.         Methylation histone tails occurs <br /> chromatin, histone demethylase               development, genes differen-         arginine lysine residues tone methyltransferase, histone marks,               tially expressed repressed needed,      regulated histone methyltransfer- <br /> transcription                                        specifying features unique         ases (HMTs) histone demethylases <br />                                                      cell type. activity transcription fac-   (HDMs). Methylation lysine 27 Abbreviations: H3K27me3, trimethyl                   tors regulation gene expression       histone 3 (H3K27) considered repres- <br /> lysine 27 histone H3; H3K4me3,                    patterns known, recent years     sive marker established trimethyl lysine 4 histone H3; KDM,               role epigenetics regulation       EZH2-containing Polycomb Repressive <br /> lysine demethylase; Jmj-C, jumonji-C                 transcription increasingly        Complex 2 (PRC),6 mes complex domain                                               clear. term &#226;&#8364;&#339;epigenetics&#226;&#8364;? refers heri-   C. elegans, removed members <br /> Submitted: 08/21/12                                  table changes gene expression phe-        KDM6 family, UTX (ubiquitously- <br />                                                      notypic alterations related changes      transcribed TPR protein X chro- <br /> Accepted: 09/11/12 <br />                                                      DNA sequence, resulting         mosome) JMJD3.7-9 protein, <br /> http://dx.doi.org/10.4161/worm.22188                 modifications chromatin structure.2      UTY, located Y chromosome, *Correspondence  Anna Elisabetta Salcini;         Epigenetic mechanisms include DNA               KDM6 family shares high <br /> Email: lisa.salcini@bric.dk                          methylation, histone modifications          homology UTX, unable <br />                                                      non-coding RNAs.                                remove H3K27me3 mark, Commentary  Vandamme J, Lettier G, Sidoli S, <br />                                                          nuclei eukaryotic cells,      vitro.8,9 C. elegans 3 homo- <br /> Di Schiavi E, N&#195;&#184;rregaard Jensen O, Salcini AE. C. elegans H3K27 demethylase UTX-1 essential      genomic DNA tightly wrapped           logs JMJD3 (jmjd-3.1&#226;&#8364;&#8220;3) single normal development, independent           core histones (H2A, H2B, H3 H4)        homolog UTX/UTY, called utx-1.1,7 <br /> enzymatic activity. PLoS Genet 2012; 8:e1002647;     compacted structure called chromatin.         UTX-1 protein contains tetratrico- <br /> PMID:22570628; http://dx.doi.org/10.1371/journal.    Core histones flexible tails 25&#226;&#8364;&#8220;40      <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> repeats (TPR) N-terminal <br /> pgen.1002647. <br />                                                      amino acids, marked vast variety      region, predicted protein <br />  <br />  <br /> www.landesbioscience.com	Worm	e22188-1 <br />   Figure 1. Schematic representation UTX-1 protein. tetratricopeptide repeats (TPRs) Jmj-C domain black red, respectively. <br />  Phosphorylated serines indicated. <br />  <br />  <br /> interaction motifs,10 carboxyl-termi-        female Utx KOs embryonic lethal,                 brevity UTX-1/SET-16 com- <br /> nal Jmj-C domain, identified proteins           hemizygous males carry Uty               plex) includes WDR-5.1&#226;&#8364;&#8220;2, ASH-2, demethylase activity (Fig. 1).11           Y chromosome, born, suggesting              F21H12.1, SET-16 PIS-1 (Fig. 2). RNA interference approaches, utx-1             developmental roles mouse UTX                similar complex identified observed involved vulva          fulfilled UTY, despite lack            mammalian cell.20-22 WDR-5, <br /> mation12 longevity.13,14                    catalytic activity. Taking          ASH-2 F21H12.1  homolog                                                    results, shows UTX                 RBBP5) common com- <br />   Developmental Roles UTX-1                     retained catalytic-independent role dur-           plexes,23-25 SET-16, nematode homolog <br />                                                    ing evolution.                                       MLL3/4 H3K4 methyltransferase biological roles UTX-1                   analysis suggests              activity, PIS-1, homolog PTIP, <br /> development analyzed using utx-1              members KDM6 family serve dif-                specific components. findings null mutant alleles.1 phenotypes asso-         ferent roles nematode life.             loss reduction com- <br /> ciated UTX-1 loss pleiotropic,            utx-1 broadly expressed gene required           ponents complex gives rise suggesting utx-1 essential gene         viability, JMJD3 homologs (jmjd-             phenotypes identified utx-1 null <br /> involved aspects development.        3.1&#226;&#8364;&#8220;3) specific sub-           mutant, analyses Homozygous utx-1 animals maternal             tle roles. indicated restricted       genetic interactions members <br /> contribution develop adulthood           cells/tissue localization proteins          complex, indicates UTX-1/ somatic gonadal defects steril-           fact single triple mutant         SET-16 complex plays relevant role dur- <br /> ity. eggs produced major-         animals lacking jmjd-3-like genes          ing development UTX-1 acts <br /> ity failing hatch. escapers, lacking        viable fertile (Vandamme et al.,             complex. mass spectrom- maternal zygotic contribution,            unpublished).7 Genetic analyses            etry, identified UTX-1 associa- <br /> develop deformed animals die             jmjd-3 genes act redun-             tion chromatin factors, L1 stage, prominent defects body           dantly utx-1, supporting           HDA-1, LIN-53 MIX-1, key com- <br /> morphology. mammalian counter-              members KDM6 family play                 ponents NuRD complex 26  UTX-1 H3K27me3 demethyl-               distinct roles nematode.             Dosage Compensation Complex.27 ase, loss results global increase     agreement studies mammals                 finding suggests UTX-1 present H3K27 trimethylation, indicating           demonstrating specific roles JMJD3               chromatin-associated machineries <br /> UTX-1 actively removes large portion          response environmental cues.17-            supports notion chromatin <br /> H3K27me3 animal.                            19 <br />                                                       fact nematode, JMJD3              factors components large complexes, <br />     major findings               represented 3 homolog genes                containing enzymatic functions. <br /> manuscript catalytic activity       reflect necessity animal           Strikingly, histone demethylases UTX-1 required described            adapt external stimuli. Notably,          association histone meth- <br /> phenotypes.  reintroduction            previously reported loss jmjd-            yltransferases7,21,22,28-31 suggesting catalytically dead UTX-1 (UTX-1DD)                 3.1 results aberrant gonadal migration            coordinated removal histone modifica- able, similar wild-type form,        misshaped germline mutant               tions HDMs addition rescue sterility, embryonic lethality      animals grown high temperature.7            modifications HMTs required morphological defects null          detailed analysis JMJD-3              timely establishment chro- <br /> mutants. implies UTX-1              homologs, particular stress             matin environment required precise functions demeth-            ditions, bring information spe-          transcriptional output. Future studies <br /> ylase activity. catalytic-independent            cific role proteins aspect       directed identification <br /> function members KDM6                nematode biology.                                 UTX-1-containing complexes spe- <br /> family previously hypothesized,15                                                              cific tissues specific developmental demonstrated vivo.                UTX-1 Chromatin Complexes                       stages.  recent results obtained mice                                                                    major questions cell <br /> suggest Utx catalytic-inde-        Mass spectrometry analyses indicate                  type specifications development <br /> pendent role organism.16             UTX-1 complex                 extracellular signals control  <br />  <br /> e22188-2	Worm	 <br /> 2	       Worm	                                                                                                                          Volume 2 Issue 2 <br />   Figure 2. Schematic representation possible UTX-1 functions. wild-type conditions (central box), UTX-1/SET-16 complex recruited  promoter developmental genes. combined action UTX-1  red) SET-16  green) results loss H3K27me3 acquisition H3K- <br />  4me3, leading chromatin relaxation proper transcription activation (green arrow). utx-1 null mutant (upper drawing), complex <br />  assembled   properly targeted promoter regions (B). cases transcription remains repressed (red line), leading  utx-1 phenotypes. lower drawing, possible functions catalytic inactive UTX-1 (UTX-1DD, indicated white portion UTX- <br />  1 molecule) represented. UTX-1DD recruited complex hide H3K27me3 site (C), recruit chromatin-remodeling <br />  factors  gray) (D). cases, transcription activated, presence H3K27me3, leading correct development. Core components <br />  (WDR-5, ASH-2 F21H12.1/RBBP5) complex shown white. PIS-1 represented yellow. <br />  <br />  <br /> transcriptional programs              direction provide infor-               phenotypes strongly suggests cat- activity epigenetic enzymes.           mation UTX-1 modulation,                alytic function UTX-1 absolutely <br /> posttranslational modifications           provides hints concerning epigen-           required, development, required correct protein localization,          etic machineries integrate signaling           functionality complex.  <br /> stability activity, took advantage           networks.                                           taking account results, briefly mass spectrometry analysis iden-                                                              summarize possible models UTX-1 <br /> tify modifications associated UTX-                   Mechanisms UTX-1 Action                      action. <br /> 1. Interestingly, UTX-1                                                                      fact specific members phosphorylated serines 830                evidences UTX-1, demethylase             complex downregulated utx-1 <br /> 842, just upstream catalytic JmjC            removing repressive mark H3K27me3,              mutants suggests UTX-1 domain.  far              associates SET-16, methyltrans-              important establish complex able detect role phosphory-           ferase deposits activating mark            regulating level expression/avail- <br /> lation sites C. elegans development          H3K4me3, suggest UTX-1/                    ability components (Fig. 2A). <br /> (Vandamme J et al., unpublished),             SET-16 complex promotes transcriptional             Furthermore, presence UTX-1 presence suggests UTX-1 subject             activation modulating chromatin              stabilize complex protein-protein regulation kinases  possibly,             environment (Fig. 2).  ability          interactions, mediated tetratrico- <br /> phosphatases. studies               inactive UTX-1 restore wild-type          peptide repeats, located N-terminal <br />  <br />  <br /> www.landesbioscience.com	Worm	e22188-3 <br />  region. Unfortunately, deletions          mechanisms underlin-            directly assess relevance regions destabilize mislocalize UTX-1       ing ability UTX-1DD fulfil         enzymatic activities studied mol- <br /> (Vandamme, et al., unpublished),          developmental functions wild-type         ecules. relevant develop- <br /> preventing analysis direc-     protein?                                         ment, UTX-1 enzymatic activity tion. Point mutations TPRs,              recent article suggests demethylase-       important functions vitro binding assays reconstitution        independent roles members              nematode life.  recently, cata- vitro complex help iden-     JMJD6 family interaction          lytic activity UTX-1 shown tify direct partners UTX-1       Brg1-containing Swi/Snf complexes            modulate longevity.13,14 analyses <br /> clarify dynamics binding       T-box proteins.15 Despite presence           testing response stresses cell repro- <br /> members complex. Alternatively,         T-box genes C. elegans         gramming, identification addition, UTX-1 contribute           homologs Swi/Snf complex com-             target genes, reveal catalytic- localization complex spe-     ponents, mass spectrometry analysis          dependent roles UTX-1. <br /> cific regions genome. recruit-      failed recover proteins UTX-1 <br /> ment UTX-1 genome, illustrated       partners, suggesting mecha-            Disclosure Potential Conflicts Figure 2B, mediated         nism relevant C. elegans.          potential conflicts H3K27me3 mark, recognized UTX-1,            Interestingly, reported         disclosed. unknown DNA-binding proteins             phosphorylation H3S28, concomi- transcription factors  shown).      tance H3K27me3, results tran-                           Acknowledgments <br /> test hypothesis, chromatin immu-          scriptional activation &#226;&#8364;&#339;hiding&#226;&#8364;?       thank Kristian Helin (BRIC, <br /> noprecipitation performed         repressive mark.33 light        University Copenhagen) useful dis- utx-1 null background, experiment         results, considering point          cussion. studies UTX-1 sup- moment unfeasible        mutations UTX-1DD impair               ported Danish National Research lethality mutants. pos-     recognition substrate,34   Council, Program Excellence sible explanation rely ability      conceivable UTX-1DD, failing          University Copenhagen UTX-1 counteract recruitment         remove H3K27me3, able           Lundbeck foundation. PRC2/MES complex32 responsible          bind hide mark just SET-16 H3K27me3 deposition, creat-           methylating H3K4, resulting activa-                               References <br /> ing unbalanced ratio H3K27me3/           tion gene transcription rescue         1.	 Vandamme J, Lettier G, Sidoli S, Di Schiavi E, <br />                                                                                                     N&#195;&#184;rregaard Jensen O, Salcini AE. C. elegans <br /> H3K4me3 marks favor higher             utx-1 phenotypes. model shown             H3K27 demethylase UTX-1 essential normal <br /> level H3K4me3, leading transcrip-        Figure 2C. Alternatively, UTX-1DD                development, independent enzymatic activity. <br />                                                                                                     PLoS Genet 2012; 8:e1002647; PMID:22570628; <br /> tional activation.  synthetic       able recruit, context          http://dx.doi.org/10.1371/journal.pgen.1002647. <br /> interactions components        complex, unknown chroma-            2.	 Berger SL, Kouzarides T, Shiekhattar R, Shilatifard <br /> MES complex identified, indicating        tin-remodeling factors allow tran-               operational <span id='am-16' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> epigenetics. Genes <br />                                                                                                     Dev 2009; 23:781-3; PMID:19339683; http:// scenario occur        scriptional activation, presence          dx.doi.org/10.1101/gad.1787609. <br /> nematode.                                      H3K27me3 (represented Figure 2D).             3.	 Berger SL. complex language chromatin regu- <br />                                                                                                     lation transcription. Nature 2007; 447:407- <br />     results showing enzymati-     test possibility, specific binding           12; PMID:17522673; http://dx.doi.org/10.1038/ <br /> cally inactive UTX-1 able rescue     partners UTX-1DD identi-                nature05915. <br /> developmental defects associated      fied mass spectrometry analysis.              4.	 Kouzarides T. Chromatin modifications                                                                                                     function. Cell 2007; 128:693-705; PMID:17320507; <br /> loss utx-1, provide additional frame-                                                         http://dx.doi.org/10.1016/j.cell.2007.02.005. <br /> work elaborate possible              Concluding Remarks                       5.	 Gardner KE, Allis CD, Strahl BD. Operating mechanisms function UTX-1.                                                                    chromatin, colorful language context mat- <br />                                                                                                     ters. J Mol Biol 2011; 409:36-46; PMID:21272588; <br />     Mutagenesis analyses Jmj-C          general, study provides indications           http://dx.doi.org/10.1016/j.jmb.2011.01.040. <br /> domain indicate critical role    increased global levels H3K27me3         6.	 Margueron R, Reinberg D. Polycomb complex <br />                                                                                                     PRC2 mark life. Nature 2011; 469:343- folding localization pro-     deleterious.  utx-1 animals           9; PMID:21248841; http://dx.doi.org/10.1038/ <br /> tein.  deletion 20 amino acids    rescued UTX-1DD animals                     nature09784. Jmj-C domain UTX-1 mislocalizes         lacking 3 JMJD-3 homologs             7.	 Agger K, Cloos PA, Christensen J, Pasini D, Rose <br />                                                                                                     S, Rappsilber J, et al. UTX JMJD3  <br /> UTX-1 cytoplasm (Vandamme, et           high level global H3K27me3                tone H3K27 demethylases involved HOX gene <br /> al., unpublished).  cata-     viable fertile. results,            regulation development. Nature 2007; 449:731- <br /> lytically inactive mutant UTX-1,       indicate UTX-1 possesses functions              4; PMID:17713478; http://dx.doi.org/10.1038/ <br />                                                                                                     nature06145. <br /> carries <span id='am-17' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-18' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-22' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> substitutions      catalytic activity. Similar      8.	 Lan F, Bayliss PE, Rinn JL, Whetstine JR, Wang <br /> sufficient impair binding <span id='am-24' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Iron" datatype="xsd:string"></span>iron</span>       ings emerging chromatin           JK, Chen S, et al. histone H3 lysine 27 demethyl- <br />                                                                                                     ase regulates animal posterior development. Nature <br /> ions required enzymatic activity,   factors model systems.15,18,35-39       2007; 449:689-94; PMID:17851529; http://dx.doi. <br /> nuclear large complex      outcomes increase cau-              org/10.1038/nature06192. <br /> includes WDR-5, suggesting       tion researchers establishing <br /> catalytic activity required     roles chromatin factors empha- <br /> complex formation.                             sizing need specific experiments <br />  <br />  <br /> e22188-4	Worm	 <br /> 4	       Worm	                                                                                                                      Volume 2 Issue 2 <br />  9.	 Hong S, Cho YW, Yu LR, Yu H, Veenstra TD, Ge                                20.	Issaeva  Zonis Y, Rozovskaia T, Orlovsky K,             31.	Fang R, Barbera AJ, Xu Y, Rutenberg M, Leonor <br />      K. Identification JmjC domain-containing UTX                                 Croce CM, Nakamura T, et al. Knockdown                     T, Bi Q, et al. Human LSD2/KDM1b/AOF1 regu- <br />      JMJD3 histone H3 lysine 27 demethyl-                                     ALR (MLL2) reveals ALR target genes leads              lates gene transcription modulating intragenic <br />      ases. Proc Natl Acad Sci U S 2007; 104:18439-                                 alterations cell adhesion growth. Mol Cell Biol        H3K4me2 methylation. Mol Cell 2010; 39:222-33; <br />      44; PMID:18003914; http://dx.doi.org/10.1073/                                   2007; 27:1889-903; PMID:17178841; http://dx.doi.              PMID:20670891; http://dx.doi.org/10.1016/j.mol- <br />      pnas.0707292104.                                                                org/10.1128/MCB.01506-06.                                     cel.2010.07.008. <br /> 10.	 Blatch GL, L&#195;&#164;ssle M. tetratricopeptide repeat:                       21.	Cho YW, Hong T, Hong S, Guo H, Yu H, Kim                  32.	Lee ER, Murdoch FE, Fritsch MK. High histone <br />      structural motif mediating protein-protein interac-                             D, et al. PTIP associates MLL3- MLL4-                acetylation decreased polycomb repressive com- <br />      tions. Bioessays 1999; 21:932-9; PMID:10517866;                                 containing histone H3 lysine 4 methyltransfer-                plex 2 member levels regulate gene specific tran- <br />      h t t p : / / d x . d o . o r g / 1 0 . 1 0 0 2 / ( S C ) 15 2 1 -        ase complex. J Biol Chem 2007; 282:20395-406;                 scriptional changes early embryonic stem cell <br />      1878(199911)21:11&lt;932::AID-BIES5&gt;3.0 2-N.                                    PMID:17500065; http://dx.doi.org/10.1074/jbc.                 differentiation induced <span id='am-29' about='obo:IMR_0100041' typeof='owl:Thing obo:IMR_0000877 obo:IMR_0001656 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001'><span id='am-30' property="rdfs:label" content="vitamin A acid" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="all-trans-Vitamin A acid" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="Retin A" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0100041" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="vitamin A acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Tretinoin" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="all-trans-Vitamin A acid" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="retinoic acid" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="all-trans-Retinoic Acid" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Tretinoin" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00777" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasExactSynonym" content="Retin A" datatype="xsd:string"></span>retinoic acid</span>. Stem Cells <br /> 11.	 Cloos PA, Christensen J, Agger K, Helin K. Erasing                              M701574200.                                                   2007; 25:2191-9; PMID:17525233; http://dx.doi. <br />      methyl mark: histone demethylases center                        22.	Patel SR, Kim D, Levitan  Dressler GR.                   org/10.1634/stemcells.2007-0203. <br />      cellular differentiation disease. Genes Dev                              BRCT-domain containing protein PTIP links PAX2           33.	 Gehani SS, Agrawal-Singh S, Dietrich N, <br />      2008; 22:1115-40; PMID:18451103; http://dx.doi.                                 histone H3, lysine 4 methyltransferase complex.          Christophersen NS, Helin K, Hansen K. Polycomb <br />      org/10.1101/gad.1652908.                                                        Dev Cell 2007; 13:580-92; PMID:17925232; http://              group protein displacement gene activa- <br /> 12.	Fisher K, Southall SM, Wilson JR, Poulin GB.                                     dx.doi.org/10.1016/j.devcel.2007.09.004.                      tion MSK-dependent H3K27me3S28 <br />      Methylation demethylation activities C.                           23.	Greer EL, Maures TJ, Ucar D, Hauswirth AG,                     phosphorylation. Mol Cell 2010; 39:886-900; <br />      elegans MLL-like complex attenuate RAS signal-                                  Mancini E, Lim JP, et al. Transgenerational epi-              PMID:20864036; http://dx.doi.org/10.1016/j.mol- <br />      ling. Dev Biol 2010; 341:142-53; PMID:20188723;                                 genetic inheritance longevity Caenorhabditis            cel.2010.08.020. <br />      http://dx.doi.org/10.1016/j.ydbio.2010.02.023.                                  elegans. Nature 2011; 479:365-71; PMID:22012258;         34.	Kruidenier L, Chung CW, Cheng  Liddle J, Che <br /> 13.	 Jin C, Li J, Green CD, Yu X, Tang X, Han D, et al.                              http://dx.doi.org/10.1038/nature10572.                        K, Joberty G, et al. selective jumonji H3K27 <br />      Histone demethylase UTX-1 regulates C. elegans life                        24.	Greer EL, Shi Y. Histone methylation: dynamic                demethylase inhibitor modulates proinflamma- <br />      span targeting insulin/IGF-1 signaling path-                             mark health, disease inheritance. Nat Rev              tory macrophage response. Nature 2012; 488:404- <br />      way. Cell Metab 2011; 14:161-72; PMID:21803287;                                 Genet 2012; 13:343-57; PMID:22473383; http://                 8; PMID:22842901; http://dx.doi.org/10.1038/ <br />      http://dx.doi.org/10.1016/j.cmet.2011.07.001.                                   dx.doi.org/10.1038/nrg3173.                                   nature11262. <br /> 14.	Maures TJ, Greer EL, Hauswirth AG, Brunet                                 25.	Shilatifard  Molecular implementation phys-         35.	 Yang  Jiang J, Stewart DM, Qi S, Yamane K, Li J, <br />      H3K27 demethylase UTX-1 regulates C. ele-                                   iological roles histone H3 lysine 4 (H3K4)                et al. AOF1 histone H3K4 demethylase possessing <br />      gans lifespan germline-independent, insulin-                               methylation. Curr Opin Cell Biol 2008; 20:341-                demethylase activity-independent repression func- <br />      dependent manner. Aging Cell 2011; 10:980-90;                                   8; PMID:18508253; http://dx.doi.org/10.1016/j.                tion. Cell Res 2010; 20:276-87; PMID:20101264; <br />      PMID:21834846; http://dx.doi.org/10.1111/j.1474-                                ceb.2008.03.019.                                              http://dx.doi.org/10.1038/cr.2010.12. <br />      9726.2011.00738.x.                                                         26.	Solari F, Ahringer J. NURD-complex genes                  36.	 DiTacchio L, Le HD, Vollmers C, Hatori M, Witcher <br /> 15.	Miller SA, Mohn SE, Weinmann  Jmjd3                                       antagonise Ras-induced vulval development                  M, Secombe J, et al. Histone lysine demethylase <br />      UTX play demethylase-independent role chro-                                Caenorhabditis elegans. Curr Biol 2000; 10:223-6;             JARID1a activates CLOCK-BMAL1 influenc- <br />      matin remodeling regulate T-box family member-                               PMID:10704416; http://dx.doi.org/10.1016/S0960-               es circadian clock. Science 2011; 333:1881-5; <br />      dependent gene expression. Mol Cell 2010; 40:594-                               9822(00)00343-2.                                              PMID:21960634; http://dx.doi.org/10.1126/sci- <br />      605; PMID:21095589; http://dx.doi.org/10.1016/j.                           27.	 Lieb JD, Albrecht MR, Chuang PT, Meyer BJ. MIX-               ence.1206022. <br />      molcel.2010.10.028.                                                             1: essential component C. elegans mitotic      37.	Secombe J, Li L, Carlos L, Eisenman RN. 16.	 Welstead GG, Creyghton MP, Bilodeau S, Cheng AW,                                machinery executes X chromosome dosage com-                   Trithorax group protein Lid trimethyl  <br />      Markoulaki S, Young RA, et al. X-linked H3K27me3                                pensation. Cell 1998; 92:265-77; PMID:9458050;                tone H3K4 demethylase required dMyc- <br />      demethylase Utx required embryonic develop-                              http://dx.doi.org/10.1016/S0092-8674(00)80920-4.              induced cell growth. Genes Dev 2007; 21:537- <br />      ment sex-specific manner. Proc Natl Acad Sci U                        28.	Tahiliani M, Mei P, Fang R, Leonor T, Rutenberg                51; PMID:17311883; http://dx.doi.org/10.1101/ <br />      S 2012; 109:13004-9; PMID:22826230; http://                                   M, Shimizu F, et al. histone H3K4 demeth-                 gad.1523007. <br />      dx.doi.org/10.1073/pnas.1210787109.                                             ylase SMCX links REST target genes X-linked           38.	Li L, Greer C, Eisenman RN, Secombe J. Essential <br /> 17.	Agger K, Cloos PA, Rudkjaer L, Williams K,                                       mental retardation. Nature 2007; 447:601-5;                   functions histone demethylase lid. PLoS Genet <br />      Andersen G, Christensen J, et al. H3K27me3                                  PMID :17468742 ;          http://dx.doi.org/10.1038/          2010; 6:e1001221; PMID:21124823; http://dx.doi. <br />      demethylase JMJD3 contributes activation                                 nature05823.                                                  org/10.1371/journal.pgen.1001221. <br />      INK4A-ARF locus response onco-                                29.	 Pasini D, Hansen KH, Christensen J, Agger K, Cloos       39.	 Klose RJ, Gardner KE, Liang G, Erdjument-Bromage <br />      gene- stress-induced senescence. Genes Dev                                  PA, Helin K. Coordinated regulation transcrip-             H, Tempst P, Zhang Y. Demethylation histone <br />      2009; 23:1171-6; PMID:19451217; http://dx.doi.                                  tional repression RBP2 H3K4 demethylase                H3K36 H3K9 Rph1: vestige H3K9 <br />      org/10.1101/gad.510809.                                                         Polycomb-Repressive Complex 2. Genes Dev                  methylation Saccharomyces cerevisiae? <br /> 18.	 Santa F, Narang V, Yap ZH, Tusi BK, Burgold T,                               2008; 22:1345-55; PMID:18483221; http://dx.doi.               Mol Cell Biol 2007; 27:3951-61; PMID:17371840; <br />      Austenaa L, et al. Jmjd3 contributes control                             org/10.1101/gad.470008.                                       http://dx.doi.org/10.1128/MCB.02180-06. <br />      gene expression <span id='am-6' about='obo:IMR_0001655' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001362'><span id='am-7' property="oboInOwl:hasExactSynonym" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:16412" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="lipopolysaccharide" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0001655" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="LPS" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D008070" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="Lipopolysaccharide (LPS) is a glycosylated lipid on the outer surface of Gram-negative bacteria acts as immunostimulants." datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00338" datatype="xsd:string"></span>LPS</span>-activated macrophages <br /> </body></html>