<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Escherichia coli Deletion Mutants Illuminate Trade-Offs Growth Rate Flux Foreign <br /> Anabolic Pathway <br /> Kelly C. Falls  Aimee L. Williams&#194;&#164;b, Anton V. Bryksin&#194;&#164;c, Ichiro Matsumura* <br /> Department Biochemistry, Emory University School Medicine, Atlanta, Georgia, United States America <br />  <br />  <br />  <br />      Abstract <br />      Metabolic engineers strive improve production yields microbial fermentations, mutating      genomes production strains. mutations detrimental health organism, quantitative      mechanistic understanding trade-offs inform better designs. employed bacterial luciferase operon <br />      (luxABCDE), uses ubiquitous energetic cofactors (<span id='am-22' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span>NADPH</span>, <span id='am-27' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-28' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span>, FMNH2, acetyl-<span id='am-38' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span>CoA</span>) host cell, proxy      novel anabolic pathway. strains Escherichia coli Keio collection, contains single deletion non- <br />      essential gene, represent mutational choices engineer make optimize fermentation yields. Keio strains <br />      parental BW25113 strain transformed luxABCDE expression vector. transformant propagated <br />      defined M9 medium 37uC 48 hours; cell density (optical density 600 nanometers, OD600) luminescence <br />      measured 30 minutes. trade-offs visualized plotting maximum growth rate luminescence/ <br />      OD600 transformant &#226;&#8364;&#732;&#226;&#8364;&#732;production possibility frontier&#226;&#8364;&#8482;&#226;&#8364;&#8482;. results loss function mutations <br />      enhance growth vitro light production, improvement trait generally comes expense  <br />  <br />   Citation: Falls KC, Williams AL, Bryksin AV, Matsumura (2014) Escherichia coli Deletion Mutants Illuminate Trade-Offs Growth Rate Flux   Foreign Anabolic Pathway. PLoS 9(2): e88159. doi:10.1371/journal.pone.0088159 <br />   Editor: Patrick C. Cirino, University Houston, United States America <br />   Received August 26, 2013; Accepted January 4, 2014; Published February 4, 2014 <br />   Copyright: &#195;&#376; 2014 Falls et al. open-access article distributed terms Creative Commons Attribution License, permits unrestricted <br />   use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported National Science Foundation. KF IM supported grant number EF-1104988 (NSF3); AB, AW IM   supported MCB-0951076 (NSF2). funders role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: imatsum@emory.edu <br />    Current address: Medical Scientist Training Program, University Iowa, Carver College Medicine, Iowa City, Iowa, United States America <br />   &#194;&#164;b Current address: Department Cellular Biology, University Georgia, Athens, Georgia, United States America <br />   &#194;&#164;c Current address: Parker H. Petit Institute Bioengineering Bioscience, Georgia Institute Technology, Atlanta, Georgia, United States America <br />  <br />  <br />  <br /> Introduction                                                                           used homologous recombination create Keio collection, <br />                                                                                        consists 3985 isogenic Escherichia coli strains, <br />    Metabolic engineers seek optimize environmentally                            missing single open reading frame [6]. Hara et al. <br /> benign biosynthesis pharmaceuticals, materials, fuels                     subsequently propagated strain rich medium 24 hours, <br /> economically valuable compounds [1]. economic impact                            permeated cells osmotic shock, reacted released project,  predicated yield. example,                      ATP firefly luciferase substrate luciferin [7]. pathway produces artemisinin, anti-malarial drug,                       measured optical density (600 nm) culture, light <br /> engineered yeast received deserved attention [2].                         emission, microplate luminometer, identified numerous   cost-effective substantial,                      deletion mutations exhibited increased ATP production  appreciated, strain improvement fermentation process                      cell minute) stationary phase. observation <br /> development [3]. economically valuable pathways                               suggests wild-type E. coli cells produce ATP processes anabolic (energy-consuming) non-essential host                              essential growth rich medium (Lysogeny Broth +2.8% <br /> organism vitro. utilize intermediates produced                              glucose), inactivation processes certain <br /> endogenous pathways, including common energetic cofactors                          deletion mutations increase pool intracellular ATP entered pathways: <span id='am-63' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-64' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-73' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span>adenosine</span>                            artificial anabolic processes. interpretation, <span id='am-43' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>triphosphate</span> (ATP), <span id='am-93' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-94' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span>nicotinamide</span> <span id='am-17' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-18' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> dinucleotide (NADPH),                         &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch hypothesis,&#226;&#8364;&#8482;&#226;&#8364;&#8482; sensible, domestication flavin mononucleotide (reduced form, FMNH2), acetyl- <br />                                                                                        plants animals generally produces breeds econom- <br /> coenzyme (<span id='am-98' about='obo:IMR_0200414' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-99' property="oboInOwl:id" content="IMR:0200414" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="Acetyl-CoA" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasDbXref" content="KEGG:C00024" datatype="xsd:string"></span>acetyl-CoA</span>). metabolic pathways cell <br />                                                                                        ically valuable fit wild. interconnected, remains difficult know priori                                                                                            wondered free lunch hypothesis perturbation pathway affect flux                                                                                        generalized energetic cofactors bacterial life <br /> heterologously expressed pathway health cell. <br />                                                                                        cycle. employed bacterial Lux pathway utilizes <br />    experimental tools functional genomics make possible <br />                                                                                        common exchangeable cofactors (ATP, NADPH, address question. Transposons inserted nearly <br />                                                                                        FMNH2 fatty acids produced acetyl-CoA) functions <br /> random chromosomal positions, creating populations                                                                                        living cells. bacterial lux operon encodes proteins <br /> insertion mutants [4,5]. Alternatively, Mori colleagues <br />  <br />  <br /> PLOS | www.plosone.org                                                         1                             February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                             Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 1. Deletion mutations affect growth Escherichia coli, production light bacterial luciferase. E. coli Keio <br /> strains, parental BW25113 strain, transformed bacterial luciferase expression vector (pIMBB-T5-lux). transformants shaken minimal medium 37uC 48 hours; optical density 600 nm (green blue) luminescence (red) recorded 30 <br /> minutes. Data   parental lux/BW25113 control, representative Keio transformants, (B) lux/DybhC, (C) lux/Dlon (D) lux/Dfiu, presented  growth curves Keio strains differ BW25113 control, luminescence curves similar shape. <br /> doi:10.1371/journal.pone.0088159.g001 <br />  form complexes. LuxCDE complex uses ATP                      <span id='am-48' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-49' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> triphosphate (dNTP) kits Roche Applied <br /> NADPH catalyze reduction fatty acids fatty                     Science (Indianapolis, . <br /> aldehydes. LuxAB luciferase uses FMNH2 oxygen                         polyethylene glycol (MW = 8,000) Fluka  catalyze conversion fatty aldehyde fatty acid,         division Sigma-Aldrich, vide infra). DNA purified with concomitant release photon [8].            QIAprep Gel Extraction Spin Miniprep kits (Qiagen, <br /> expression lux operon Keio strains,             Valencia, CA). Isopropyl-b-D-thio-galactoside (IPTG) measurement light emission cell density 48 hours              Gold Biotechnology (St. Louis, MO). chemicals growth. wild-type E. coli genome nearly                Sigma-Aldrich (St. Louis, MO). microtiter plates optimal growth rate light production, deletion               Nunc (96  clear, flat  Corning (384  white <br /> mutations enhance trait usually diminish                   white clear . construction pIMBB-T5- <br />                                                                               lux expression vector previously described [9]. Briefly, series <br /> Materials Methods                                                         lux ORFs (luxA, luxB, luxC, luxD luxE) PCR amplified <br />                                                                               Photorhabdus luminescens (ATCC 29999) chromosome Bacterial strains, media plasmids                                         separately cloned pIMBB. internal Xba site luxD <br />    Keio knockout collection [6] parent BW25113 E. <br />                                                                               eliminated site-directed mutagenesis. BioBrick coli strain obtained Hirotada Mori (Nara Institute                                                                               combined optimized ribosomal binding site (rbs). rbs- <br /> Science Technology, Nara, Japan) March 2006, immedi- <br />                                                                               ORF BioBricks combined  T5-promoter, <br /> ately available; stocks                                                                               lac operators form pIMBB-T5-lux. <br /> propagated. Lysogeny Broth  Luria-Bertani, LB) EMD, M9 medium bacto-agar Difco. <br /> Restriction enzymes, T4 DNA ligase, Phusion DNA polymerase,                   High Throughput transformation KEIO collection 1 kb ladder New England Biolabs (Ipswich,                   pIMBB-T5-Lux <br /> MA). DNA oligonucleotides IDT (Coralville, IA),                   modified polyethylene glycol/dimethyl sulfoxide (PEG/ <br /> custom DNA sequencing services provided Macrogen                      DMSO) protocol, originally developed Chung et al. [10], (Rockville, MD). Polymerase chain reaction (PCR)-grade deoxy-                 used transform Keio strains pIMBB-T5-Lux <br />  <br />  <br /> PLOS | www.plosone.org                                                2                           February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                           Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />                                                                               (OD600 = 0.4&#226;&#8364;&#8220;0.7). Growth temperature (17 37uC) did affect <br />                                                                               transformation efficiency. <br />                                                                                  Thermo Scientific Multidrop 384 coupled Titertek Titan <br />                                                                               plate stacker used add 20 microliters 2X TSS (2X LB, <br />                                                                               50 mM MgCl2, 50 mM MgSO4, 20% PEG 8000, 10% DMSO) <br />                                                                               containing pIMBB-T5-Lux concentration 1 ng/microliter <br />                                                                               micro-culture. Plates shaken briefly 2 minutes                                                                               600 rpm incubated ice 30&#226;&#8364;&#8220;60 minutes. Multidrop <br />                                                                               384 dispenser used add 200 microliters LB                                                                               microculture. microplates transferred ATR <br />                                                                               Microtitertron, shaken 33uC 1 hr 600 rpm allow <br />                                                                               expression ampicillin (<span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>Amp</span>)-resistance gene. dispenser <br />                                                                               used add 10 microliters ampicillin stock solution <br />                                                                               (3.5 mg/mL) (final concentration 140 micrograms/ <br />                                                                               mL. microcultures replicated using 96-pin microplate <br />                                                                               replicator new plates; contained 200 microliters                                                                               fresh LB supplemented Amp (100 micrograms/mL), <br />                                                                               BW25113 strain, Amp kanamycin (Kan, 50 micro- <br />                                                                               grams/mL), Keio mutants. E. coli cells transformed <br />                                                                               96 microtiter plates, resulting transformants                                                                               arrayed order configuration original <br />                                                                               (untransformed) Keio collection [6]. E. coli micro-cultures <br />                                                                               allowed grow saturation overnight 33uC                                                                               600 rpm. Saturated cultures supplemented glycerol <br />                                                                               (final concentration 10%), shaken 2 minutes 600 rpm, <br />                                                                               frozen stored 280uC. <br />                                                                                  transformants propagated saturation liquid LB <br />                                                                               supplemented ampicillin  kanamycin Keio <br />                                                                               strains), reformatted 384 microtiter plates; lux/ <br />                                                                               BW25113 replicated wells 384 microtiter plate <br />                                                                               3747 lux/Keio strains distributed 126384- <br />                                                                               plates. micro-cultures propagated overnight                                                                               30uC, subsequently frozen -80uC. PCR using primers <br />                                                                               designed recognize kanamycin phosphotransferase gene <br />                                                                               (used knock genes), specific adjacent regions, <br />                                                                               used confirm identities arbitrarily chosen <br />                                                                               transformed Keio strains 12 microtiter plates (data <br />                                                                               shown). <br />  <br />                                                                               Luminescence Growth Assays <br />                                                                                  Frozen, transformed Keio strains stored 384 plates                                                                               thawed diluted 50-fold 384-pin replicator <br />                                                                               new plates; contained 50 microliters M9 <br />                                                                               supplemented 1 mM thiamine, 0.4% glucose, 250 micromo- <br />                                                                               lar isopropyl-b-D-thiogalactoside (IPTG), 100 micrograms/ <br />                                                                               mL ampicillin  kanamycin). kanamycin resistance marker <br />                                                                               Keio strains does affect cell growth absence Figure 2. Light production cell normally distributed         antibiotic, knock-outs single copies multicopy genes result <br /> 384 lux/BW25113 parental control replicates  , 3747       wild-type-like strains (data 42 Keio strains (N = 3) mutant lux/Keio cultures (B). average maximum luminescence <br />                                                                               shown). microtiter plate sampled times different <br /> (Relative Light Units) transformant divided maximum <br /> OD600, resulting values plotted histograms.                   days, recipient plates separately assayed doi:10.1371/journal.pone.0088159.g002                                         BioTek Synergy2 microplate reader. OD600 luminescence <br />                                                                               measured 30 minute intervals 48 hours. Plates plasmid. method, unlike  doesn&#226;&#8364;&#8482;t require prepa-               shaken continuously medium speed, temperature kept <br /> ration competent cells little 5&#226;&#8364;&#8220;             37uC. Absorbance read 600 nm. Luminescence 6 hours batch. Keio strains delivered 96                 recorded following settings: 1.0 sec integration time, plates. seeded 96-pin microplate replicator              4.5 mm read height, 130 gain. <br /> flat 96 plates (Nunc); contained 20 <br /> microliters fresh LB supplemented 10 mM MgSO4                     Data Analysis <br /> 50 mM 2-(N-morpholino)ethanesulfonic buffer (pH 6.1).                        Data BioTek Synergy2 microplate reader acquired <br /> microtiter plates agitated 600 rpm ATR Micro-                   analyzed Gen5 software, exported Excel files <br /> titertron shaker cells exponential phase                (raw data available request). derived values,                                                                               maximum growth rate (mOD600/min), maximum optical density, <br />                                                                               maximum luminescence, integrated OD600 integrated lumi- <br />  <br />  <br /> PLOS | www.plosone.org                                                3                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                         Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 3. Maximum growth rates 384 lux/BW25113 parental control replicates   normally distributed corrected edge effects (B). corrected maximum growth rates 3747 (N = 3) mutant lux/Keio cultures (C) distributed widely control population size. <br /> doi:10.1371/journal.pone.0088159.g003 <br />  <br /> nescence, technical replicates lux/Keio              pin replicator microtiter plates containing fresh minimal <br /> plate manually combined Excel document plate.             medium supplemented ampicillin select non- <br /> Average values standard error calculated Microsoft              transformants isopropyl-b-D-thiogalactoside (IPTG) induce <br /> Excel, resulting parameters derived entire Keio            expression luxABCDE operon. plate shaken collection consolidated single Excel document (Table              37uC 48 hours Synergy2 microplate spectrophotometer/ <br /> S1). Data technical replicates lux/BW25113 plate          luminometer; optical density (600 nm) light emission similarly combined separate document (Table S2).                  micro-culture measured 30 minutes. Kaleidagraph 3.5 (Synergy Software) used create                  technical replicates stock micro-culture, figures.                                                                    parental lux/BW25113 strain propagated monitored <br />    Liquids outermost wells 384 microtiter plates tend        1152 times (36384). evaporate quickly situated interior;                 Light production rose cell density increased, suggesting bacterial cultures edges increase cell density 20%          observed lag times function cell physiology faster middle. edge effects                 sensitivity spectrophotometer. Luminescence <br /> documented [11,12], commonplace difficult avoid.                  peaked cells entered stationary phase (Figure 1a), demonstrate  parental control strain (lux/BW25113)           suggests intracellular concentration propagated 384 microtiter plates lids containing           energetic cofactor decreases cells left log phase, presumably <br /> standard media (50 microliters M9-ampicillin) humidity-                nutrient depletion. mutant cultures exhibit controlled ATR Microtitertron (600 rpm 80% humidity, 33uC                patterns light production, growth patterns vary 23 hours). OD600 manually measured                         significantly regard lag time, maximum growth velocity, <br /> SpectraMax M5 plate reader (Molecular Devices) 5, 8                  cell death rate (Figure 1b&#226;&#8364;&#8220;c). mutants exhibit phase <br /> 23 hours; edge effects similar recorded growth           growth, light production diminishing phase Biotek Synergy2 observed. calculated              (Figure 1d); hypothesize second phase begins growth <br /> average maximum growth rate (mOD600/min) values cultures                 exported acetate commences [13], leading decreased loops wells, outermost (A1-24, P1-         light production. <br /> 24, 1A-P, 24A-P) innermost (H8-17, I8-17) continuous growth Synergy2. values derived                                                                             lux/Keio strains exhibit faster growth, outer loops average 1.35, 1.16 1.05-fold greater, <br /> respectively, inner loops. maximum               produce light <br /> growth rate values cultures (lux/BW25113 lux/Keio)               sought test free lunch hypothesis, understand outer wells corrected multiplying 0.74,           biochemical nature tradeoffs anabolism 0.86 0.95 respectively. mutants probably respond                   cellular health, focused analysis parameters <br /> differently parental control strain reductions culture       derived data. identified maximum <br /> volume, reasoned did                                   luminescence (Relative Light Units) maximum optical density <br />                                                                             (OD600) culture. values recorded                                                                             experiment cells reached end logarithmic growth, Results <br />                                                                             maximum luminescence divided maximum optical density productivity bacterial luciferase pathway                     (lum/OD600) reflects rate cells produce light <br /> growth phase dependent                                                      (Figure 2). derived maximum growth rate <br />    attempted transform 3985 Keio strains  96              (mOD600/minute), reflects rate cells produce <br /> microtiter plates) parental BW25113 E. coli strain         cells (Figure 3). mean standard error values plasmid-borne luxABCDE operon. Approximately 6% (238/3985)                  parameters calculated transformed mutant (N = 3). Keio strains failed transform repeated attempts        maximum growth rate values corrected (Materials high throughput protocol (Table S3). micro-aliquot            Methods) compensate increased evaporation wells <br /> culture, lux/BW25113 lux/Keio, transferred 384-              edges plate. <br />  <br />  <br /> PLOS | www.plosone.org                                              4                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                              Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />                                                                                     corrected maximum growth rate values 384 lux/ <br />                                                                                  BW25113 replicates normally distributed (Shapiro Wilk <br />                                                                                  test statistic = 0.98, P-value = 0.0002); values range 0.55 <br />                                                                                  0.94 mOD600/min, mean 0.68 standard <br />                                                                                  deviation 0.063 (Figure 3b).  mutants exhibited <br />                                                                                  greater variation did parental controls (Kolmogorov&#226;&#8364;&#8220; <br />                                                                                  Smirnov maximum D = 0.230, p,0.0000001). comparable <br />                                                                                  maximum growth rate values lux/Keio strains ranged                                                                                  0.01 1.75, mean 0.71 standard deviation 0.17 <br />                                                                                  (Figure 3c). population 3747 lux/BW25113 replicates                                                                                  exhibit range maximum velocities 0.46 0.90 mOD600/ <br />                                                                                  min (presuming null hypothesis mutants wild- <br />                                                                                  type-like), 148 lux/Keio transformants (3.9%) grew                                                                                  slowly expected 450 (12%) grew quickly. Previous <br />                                                                                  workers did observe E. coli strains enhanced fitness <br />                                                                                  collection 226 transposon insertion mutants [14],                                                                                  growth cells hampered constitutive <br />                                                                                  expression vector. <br />                                                                                     fitness luminescence values associated 3747 lux/ <br />                                                                                  Keio strains plotted dimensional trade curve, <br />                                                                                  maximum growth rate (mOD600/min) X-axis                                                                                  maximum luminescence/maximum optical density Y-axis <br />                                                                                  (Figure 4). Economists use similar plot, &#226;&#8364;&#732;&#226;&#8364;&#732;production <br />                                                                                  possibility frontier&#226;&#8364;&#8482;&#226;&#8364;&#8482; (PPF), depict quantities                                                                                  commodities manufactured organization                                                                                   country) given interval time fixed                                                                                  factors production. PPF plot (Figure 4) shows                                                                                  lux/Keio strains similar parental control (lux/BW25113) <br />                                                                                  growth rate light production. grow                                                                                  slowly produce light control.                                                                                  produce light ancestor strain tend grow <br />                                                                                  slowly, grow quickly produce light.                                                                                  investigated clones margins frontier                                                                                  E. coli offers free lunches. <br />  <br />                                                                                  lux/Keio transformants outshine parental lux/ <br />                                                                                  BW25113 ancestor <br />                                                                                     Metabolic engineers generally concerned fermen- <br />                                                                                  tation yields growth rates. Chromosomal mutations                                                                                  increase productivity foreign anabolic pathway, represented <br />                                                                                  case luminescence/OD600, economically <br />                                                                                  valuable [4]. mutations,  combined                                                                                  improve yields unless neutral net effect                                                                                  cellular health. lux/Keio transformants exhibited statisti- <br /> Figure 4. maximum growth rate luminescence cell 384 <br /> lux/BW25113 control replicates   lux/Keio transformants (B)                                                                                  cally significant improvements lux/BW25113 control shown scatter plots. distances points (green squares,           light production modest effects growth rate: DhyfC, <br /> reflect variations isogenic lux/BW25113 cultures 384              DyccC, DdcuC, DbglG, DyfiR, DyheV, DnusB, DthrL. Colonies plate; error bars represent variations repetitions       (biological replicates, N = 6) separately propagated 300 experiment (N = 3). data (green squares, B) plotted             microliter cultures M9-ampicillin, diluted 1:50 lux/Keio transformant (blue squares, B) facilitate         medium, supplemented 250 micromolar IPTG, 96 direct comparisons. <br /> doi:10.1371/journal.pone.0088159.g004 <br />                                                                                  microtiter plates. plates agitated (medium)                                                                                  spectrophotometer/luminometer (Biotek Synergy 2) 24 hours; <br />    average lum/OD600 values 384 replicates lux/                    optical density (600 nm) luminescence (Relative Light <br /> BW25113 normally distributed (Shapiro Wilks test statistic                   Units) measured 15 minutes. <br /> .0.99, P-value = 0.03). values range 52,400                       clones tested, lux/DthrL, exhibited <br /> 113,000 RLU/OD600, average 79,800 standard                      significant improvement parental lux/BW25113, deviation 10,400 (Figure 2a). comparable lum/OD600 values                 cell growth (Figure 5a) light production (Figure 5b). 3747 lux/Keio transformants distributed broadly                   lux/DyveH, exhibited enhanced light production asymmetrically (Shapiro Wilk test statistic = 0.86), ranging                significant reduction growth (data shown). 2,820 479,000 RLU/OD600, average 82,900                      possible combine &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch&#226;&#8364;&#8482;&#226;&#8364;&#8482; mutations produce standard deviation 39,100 (Figure 2b). normally distributed                 strain evinced better performance, population 3747 lux/BW25113 replicates expected                   laboratory conditions. transformants, lux/DhyfC <br /> vary 43,800 116,000, 357 lux/Keio transformants                      (Figure 5c&#226;&#8364;&#8220;d), lux/DyfiR lux/DnusB produced light (9.5%) fell range 524 (14%)                          expense growth.  lux/DyccC, lux/DdcuC  <br />  <br /> PLOS | www.plosone.org                                                   5                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                           Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />  <br />  <br /> Figure 5. lux/DthrL lux/DhyfC strains exhibit improvements parental lux/BW25113. strain propagated (N = 6) 24 hours M9 medium supplemented ampicillin IPTG Biotek Synergy2 plate reader (37uC, medium shaking); OD600 luminescence recorded culture 15 minutes. lux/DthrL strain (blue squares,  grows quickly higher cell density does lux/BW25113 (green squares), produces light (B). lux/DhyfC transformant grows slowly parental control (C), produces light (D). <br /> doi:10.1371/journal.pone.0088159.g005 <br />  <br /> lux/DblgG, produced light control lux/BW25113                   ATP. Contrary expectation, exhibited relative activity <br /> transformant (data shown), indicative observed                     (maximum luminescence divided maximum OD600) imprecision initial screen (Figure 4a).                                double lux/BW25113 control assay                                                                                112 strains assay exceeded criterion). Similarly, <br /> Different assays produce different results                                    seven 20 dec mutants exhibited half    Hara et al. mixed ATP Keio strains stationary phase            relative activity control (319 strains fell firefly luciferase luciferin substrate. reported        threshold assay); dec mutants actually exhibited <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;cellular ATP synthetic activity,&#226;&#8364;&#8482;&#226;&#8364;&#8482; quantity light relative        higher relative activity control. disparity wild-type value divided cell density  relative wild-       surprising. bacterial luciferase used study utilizes <br /> type value) strain. identified 21 &#226;&#8364;&#732; &#226;&#8364;&#8482; strains           cofactors (ATP, NADPH, FMNH2 fatty acids), exhibited .200% relative activity  cell, compared wild-            active log phase, firefly luciferase employed <br /> type) 20 &#226;&#8364;&#732;&#226;&#8364;&#732;dec&#226;&#8364;&#8482;&#226;&#8364;&#8482; exhibited ,50% relative activity [7].            Hara et al. required ATP  luciferase exogenously <br /> expected priori strains similar phenotypes            added) assayed stationary phase. assay, luxABCDE pathway partially dependent <br />  <br />  <br /> PLOS | www.plosone.org                                                6                         February 2014 | Volume 9 | Issue 2 | e88159 <br />                                                                                                                         Genetic Modifiers Lux Escherichia coli <br />  <br />  <br />    Baptist et al. recently cloned lux operon downstream                        study inform efforts improve biosynthetic <br /> acetyl-CoA synthetase (acs) promoter. transformed Keio                        yields genetically modified microorganisms. Engineers strains lux expression vector used colony-based                       taught formulate quantitative models, generally useful <br /> screen identify E. coli genes normally regulate acs.                       skill lead oversimplify. synthetic biologists long <br /> genes identified naturally quite different                          free-living organism minimal genome [22], strongest effectors (vide supra). lux operon constitutively                    showed deletions non-essential genes impart <br /> expressed experiment, regarded negative                     precipitous decreases biosynthetic yield. Photorhabdus luminescens, <br /> control study like  Alternatively,                        source lux genes, apparently optimized evolution <br /> characterized reporter genes, lacZ gfp, sensitive                  produce light, outshines best deletion mutants (data metabolic state cell                         shown). expresses additional specialized genes, luxG [23], <br /> appropriate genome-wide screens modifiers                               phenocopied loss function mutations. E. coli <br /> particular gene                                                            engineered produce 1,3 propanediol 82% theoretical <br />                                                                                        yield [24], transformable species likely prove Discussion                                                                             better starting points production challenging target <br />    sought test &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch hypothesis,&#226;&#8364;&#8482;&#226;&#8364;&#8482; notion                    compounds. inactivation anabolic processes essential vitro increase energy available competing processes.                         Supporting Information <br /> observations suggest &#226;&#8364;&#732;&#226;&#8364;&#732;free lunch&#226;&#8364;&#8482;&#226;&#8364;&#8482; mutations exist                    Table S1 Lux Keio parameters. lux/Keio strains (Figure 5ab), vast majority mutations enhance                     propagated times 384 microtiter plates, light production exhibit average growth rate (Figure                        luminescence optical density 600 nm measured 5cd). conclude light production competes                              30 minutes, maximum growth rate (max V OD), integrated <br /> production new cells. shape PPF informs economists                       optical density (area growth curve, int OD), integrated opportunity costs producing commodity.                             luminescence (area luminescence curve, int lum), maximum <br /> straight line reflects constant opportunity costs, means                                                                                        optical density 600 nm (max OD), maximum luminescence (max <br /> resources specialized production output. <br />                                                                                        lum) calculated technical replicate. inward curving PPF (Figure 4b) reflects economies <br />                                                                                        (XLSX) scale, means opportunity cost manufacturing output falls produced. plot suggests                    Table S2      Lux BW25113 parameters. parental lux/ mutations largest phenotypic effects tend                      BW25113 control strain used seed wells 384 pleiotropic, means disrupt multiple pathways                      microtiter plate. micro-cultures propagated normally affect growth light production opposite ways.                          times, luminescence optical density 600 nm    ATP flux does determine growth rate E. coli, ATP                     measured 30 minutes, growth light emission <br /> yield does determine cell density stationary phase [15].                    parameters  Table S1 Legend) calculated. <br /> trade-offs fitness light production                      (XLSX) <br /> indirect nature. Genome-scale metabolic models used formulate mechanistic hypotheses. thrL gene                           Table S3 Unsuccessful transformation. Keio strains negative regulator threonine operon [16], deletion                       unable transform pIMBB-T5-lux expression <br /> likely increases production <span id='am-56' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-57' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-59' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> minimal                          vector listed. <br /> medium. YccC, Etk, membrane tyrosine autokinase                           (XLSX) <br /> participates form <span id='am-1' about='obo:IMR_0001790' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="polysaccharide" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001790" datatype="xsd:string"></span>polysaccharide</span> secretion [17] essential laboratory. HyfC component hydrogenase 4,                         Acknowledgments <br /> converts formate <span id='am-12' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span> H2 [18].                                   thank Prof. Susan Gilbert teaching IM Production <br /> comprehensive models [19,20] remain incomplete. primary                            Possibilities Frontier model, Matsumura lab functions quarter genes E. coli K-12, including yveH                  helpful suggestions. yfiR, remain unknown [21]. Fundamental biochemical <br /> parameters (including concentrations, rate constants, Michaelis                    Author Contributions <br /> constants hundreds catalysts metabolites) regulatory <br /> properties annotated genes                                 Conceived designed experiments: IM. Performed experiments: <br />                                                                                        AW KF. Analyzed data: IM KF AB. Contributed reagents/materials/ <br /> experimentally determined. hope data set <br />                                                                                        analysis tools: AB. Wrote paper: IM KF. <br /> (Table S2), like  help refine genome-scale models. <br />  <br /> References <br />  1 <br /> </body></html>