<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Published online 16 August 2012                                           Nucleic Acids Research, 2012, Vol. 40,  20 10567&#226;&#8364;&#8220;10575 <br />                                                                                                                 doi:10.1093/nar/gks756 <br />  <br />  <br /> Assessing biocompatibility click-linked <br /> DNA Escherichia coli  Pia Sanzone1, Afaf H. El-Sagheer1,2, Tom Brown1 Ali Tavassoli1,3,* <br /> 1 <br />  Chemistry, University Southampton, Southampton, SO17 1BJ, UK, 2Department Science Mathematics, Chemistry Branch, Faculty Petroleum Mining Engineering, Suez Canal University, Suez, <br /> 43721, Egypt 3Cancer Research UK Centre, Faculty Medicine, University Southampton, Southampton, <br /> SO16 6YD, UK <br />  <br /> Received 10, 2012; Revised July 14, 2012; Accepted July 16, 2012 <br />  <br />  <br />  <br /> ABSTRACT                                                                        inability incorporate epigenetic information,                                                                                 modi&#239;&#172;?ed bases genes genomes. biocompatibility triazole mimic DNA                             alternative approach DNA assembly envisaged <br /> phosphodiester linkage Escherichia coli                             instead enzymes, uses highly ef&#239;&#172;?cient <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span>  <br /> evaluated. requirement selective pressure                               actions ligation oligonucleotides (5&#226;&#8364;&#8220;7). click-containing gene probed                                   approach eliminate need enzymatic <br /> plasmid containing click DNA backbone linkages                               ligation cloning gene synthesis enable strand gene encoding fluorescent                                automation large-scale gene synthesis, protein mCherry. effect proximity                                 readily allow incorporation modi&#239;&#172;?ed bases click linkers biocompatibility                                large DNA fragments. resulting click-linked DNA <br /> probed placing click DNA linkers 4-bp apart                               contain unnatural triazole linkage region encoding fluorophore fluor-                            backbone sites ligation  place canon- <br />                                                                                 ical phosphodiester linker). enzyme-free approach escent protein. resulting click-containing <br />                                                                                 DNA ligation use unnatural <br /> plasmid encode mCherry E. coli                               backbone linkage does adversely affect rate similar level canonical equivalent. ability                        &#239;&#172;?delity DNA replication transcription, func- cellular machinery read                                       tioning normally biological systems. <br /> click-linked DNA probed using                                   end, recently reported (8) rapid, clean click-linked plasmid express mCherry                                   ef&#239;&#172;?cient chemical ligation DNA oligonucleotides <br /> using vitro transcription/translation                              using copper-catalysed alkyne&#226;&#8364;&#8220;azide cycloaddition similar canonical                             (CuAAC) reaction (Figure 1) (9,10). DNA sequence <br /> DNA. yield fluorescence recombinant                                  linker ampli&#239;&#172;?ed PCR high &#239;&#172;?delity, <br /> mCherry expressed click-linked plasmid                                 TEM-1 b-lactamase gene (BLA) encoding ampicillin compared canonical                                    resistance containing click linkers (83-bp apart,                                                                                 strand), functional E. coli equivalent,  biocom- <br />                                                                                 similar levels equivalent plasmid canonical <br /> patibility click DNA ligation sites close proxim-                         DNA backbone. click DNA backbone linker ity non-essential gene demonstrated E. coli                             shown functional UvrB de&#239;&#172;?cient strain suggests possibility using click DNA ligation                            E. coli, suggesting <span id='am-16' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> excision enzyme-free assembly chemically                                      repair does play key role observed biocom- <br /> modified genes genomes.                                                     patibility (UvrB central component nucleotide <br />                                                                                 excision repair) (8). work shown T7 <br />                                                                                 RNA polymerase reads linker vitro, <br /> INTRODUCTION                                                                    effect click-linker &#239;&#172;?delity <br /> Current methods assembly large DNA mol-                              rate process thoroughly <br /> ecules (e.g. genes) utilize mixture oligonucleotide syn-                   evaluated (11). <br /> thesis, PCR ampli&#239;&#172;?cation enzymatic ligation.(1,2)                              selective pressure exerted organism ampi- extremely powerful (3,4), approach                            cillin previously reported experiment (8) weaknesses, including additional time                           pre-selected colonies correctly read resources required enzymatic steps,                             click-linker, possibly masking true &#239;&#172;?delity  <br />  <br />  correspondence addressed. Tel: +44 2380 592395; Fax: +44 2380 593131; Email: tavassoli@soton.ac.uk <br />  <br /> &#195;&#376; Author(s) 2012. Published Oxford University Press. Open Access article distributed terms Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/ nc/3.0), permits unrestricted non-commercial use, distribution, reproduction medium, provided original work properly cited. <br />  10568 Nucleic Acids Research, 2012, Vol. 40,  20 <br />  <br />  <br />  <br />  <br /> Figure 1.   Click DNA ligation. CuAAC reaction used combination alkyne azide-modi&#239;&#172;?ed DNA bases join oligo- <br /> nucleotides  (B) structure equivalent canonical DNA comparison triazole backbone natural phosphodiester. <br />  <br />  <br />  <br /> process; errors replication transcription                    automated trityl cation conductivity monitoring <br /> click-modi&#239;&#172;?ed BLA gene result non-functional                    facility cases &gt;98.0%. b-cyanoethyl <br /> b-lactamase subsequently prevent survival                    phosphoramidite monomers dissolved anhydrous host organism ampicillin containing media,                     acetonitrile concentration 0.1 M immediately prior observed. account                     use. coupling time normal  G, C T <br /> high rates biocompatibility previously observed                   monomers 25 s. Cleavage oligonucleotides click-containing BLA gene. view using                       solid support deprotection achieved click DNA ligation synthesis wide variety                 exposure concentrated aqueous ammonia solution genes, sought assess biocompatibility click-              60 min room temperature followed heating linked DNA incorporated non-essential                       sealed tube 5 h 55  C. oligonucleotides gene encoding red-&#239;&#172;&#8218;uorescent protein mCherry (12).                    puri&#239;&#172;?ed reversed-phase HPLC Gilson encoded &#239;&#172;&#8218;uorescent protein does exert selective                using XBridgeTM BEH300 Prep C18 10 mM <br /> pressure bacterial host, true &#239;&#172;?delity                       10   250 mm column (Waters) gradient acetoni- <br /> click-DNA replication transcription                           trile ammonium acetate (0% 50% buffer B determined. sought assess effect click                  30 min, &#239;&#172;&#8218;ow rate 4 ml/min), buffer  0.1 M ammonium <br /> DNA linker proximity biocompatibility; click                       acetate, pH 7.0, buffer B: 0.1 M ammonium acetate, pH <br /> linkers previous study 83-bp apart, potentially               7.0, 50% acetonitrile. Elution monitored UV <br /> minimizing distortive effects double helix.                    absorption 295 nm. HPLC puri&#239;&#172;?cation, oligo- <br /> Recent structural data shown joining                    nucleotides desalted using NAP-10 columns <span id='am-24' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-25' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="Thymine" datatype="xsd:string"></span>thymine</span> residues, click DNA backbone linker causes                    analysed gel electrophoresis. <br /> thermodynamic destabilization duplex, spread                     sequences SDM primers shown 4&#226;&#8364;&#8220;5 bp surrounding incorporated triazole (13).                    Figure 2A; unmodi&#239;&#172;?ed primers synthesized  utilize site-directed mutagenesis (SDM)                  Euro&#239;&#172;?ns MWG Operon (Ebersberg, Germany) construction plasmid containing click DNA                        click-linked primers synthesized outlined  <br /> linkers, 4-bp apart, gene encodes &#239;&#172;&#8218;uorescent              template plasmid SDM pRSET-mCherry, <br /> protein mCherry. click-linked plasmid used                     kindly provided Tsien laboratory. Plasmids demonstrate biocompatibility click ligated                    isolated puri&#239;&#172;?ed using QIAprep Spin Minipreps Kit <br /> DNA non-essential gene vitro E. coli.                      (QIAgen, Cat  27 106) following manufacturer&#226;&#8364;&#8482;s <br />                                                                           instructions. DNA sequencing carried MATERIALS METHODS                                                     Euro&#239;&#172;?ns MWG Operon (Ebersberg, Germany). <br />                                                                           Antibodies purchased GE Healthcare. <br /> General method oligonucleotide synthesis puri&#239;&#172;?cation <br />                                                                           Synthesis 30 -alkyne 50 -azide oligonucleotides <br /> Standard DNA phosphoramidites, solid supports add- <br /> itional reagents purchased Link Technologies                      Synthesis 30 -alkyne oligonucleotides ODN-1 Applied Biosystems  oligonucleotides                     ODN-2 <br /> synthesized Applied Biosystems 394 automated                        30 -Alkyne oligos synthesized attachment <br /> DNA/RNA synthesizer using standard 0.2 1.0 mmol                      50 -O-(4,40 -dimethoxytrityl)  30 -O-propargyl-5- <br /> phosphoramidite cycle acid-catalysed detritylation,                    methyldeoxycytidine solid support according coupling, capping iodine oxidation. Stepwise                          previously published method (8) packing resin <br /> coupling ef&#239;&#172;?ciencies overall yields determined                   twist column (Glen Research) assembling                                                                       Nucleic Acids Research, 2012, Vol. 40,  20 10569 <br />  <br />  <br /> required sequence 30 - 50 -direction normal       template gel puri&#239;&#172;?cation 0.8% agarose using standard phosphoramidite synthesis.                            QIAquick Gel Extraction Kit (QIAgen) following                                                                manufacturer&#226;&#8364;&#8482;s instructions. <br /> (ii) Synthesis 50 -azide oligonucleotides ODN-3 ODN-4                                                      Transformation SDM product <br /> Oligonucleotides assembled 1.0 0.2 mmol <br /> scale (trityl  described general           5 ml aliquot puri&#239;&#172;?ed SDM product added method normal 50 -hydroxy group dC.               50 ml aliquot KRX chemical competent cells (Cat.  <br /> convert 50 -hydroxyl 50 -iodo resin, protected   L3002, Promega), incubated 30 min ice. oligonucleotide attached synthesis column           mixture heat-shocked 42  C 30 s placed <br /> treated 0.5 M solution methyltriphenoxy-             ice 2 min. 450 ml SOC added <br /> phosphonium iodide DMF (1.0 ml) period-           mixture cells incubated shaking ically passed column 1 ml syringes         37  C 1 h. 100 ml aliquots cell 15 min room temperature. column           suspension spread LB agar plates containing <br /> washed times dry DMF (14). convert         ampicillin (100 mg/ml) 0.1% rhamnose. plates <br /> 50 -iodo dC 50 -azido dC, sodium azide (50 mg) sus-     incubated overnight 37  C. <br /> pended dry DMF (1 ml), heated 10 min 70  C cooled supernatant taken 1 ml          Assessing biocompatibility click-linked DNA <br /> syringe passed forth column           number visible colonies plate counted left room temperature overnight. column            compared click-SDM plates washed DMF acetonitrile dried            positive negative control plates  experiments passing stream argon <span id='am-5' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-8' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="gas" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>gas</span> (15).              repeated 10 times generate statistically signi&#239;&#172; Resultant 50 -azide oligonucleotides cleaved         data). colonies click-SDM plates solid support, deprotected puri&#239;&#172;?ed HPLC          selected random grow overnight 5 ml LB supple- <br /> explained general method oligonucleotide       mented 100 mg/ml ampicillin, 37  C. plasmid <br /> synthesis puri&#239;&#172;?cation.                                     isolated puri&#239;&#172;?ed mCherry gene <br />                                                                sequenced. random colonies picked Templated click ligation                                       assessed presence BamHI watermark alkyne, azide splint oligonucleotides (30.0 nmol       restriction digestion using BamHI-HF restriction endo-  0.2 M NaCl (400 ml) annealed heating       nuclease (New England Biolabs) according manu- 80  C 5 min cooled slowly room tem-         facturer&#226;&#8364;&#8482;s instructions. <br /> perature. solution CuI click catalyst prepared tris-hydroxypropyltriazole ligand (22) (2.1 mmol       vitro mCherry expression <br /> 0.2 M NaCl, 91.0 ml), sodium ascorbate (3.0 mmol            S30 T7 High Yield Protein Expression 0.2 M NaCl, 6.0 ml) CuSO4.5H2O (0.3 mmol 0.2 M          (Promega, L1110) used vitro protein expression. <br /> NaCl, 3.0 ml). CuI click catalyst solution added    Reactions set following manufacturer&#226;&#8364;&#8482;s  annealed oligonucleotides reaction mixture         structions, 1 mg puri&#239;&#172;?ed SDM product was kept room temperature 2 h. Reagents            used instead plasmid DNA template. template <br /> removed using NAP-10 gel-&#239;&#172;?ltration column              dialysed distilled water 2 h using Millipore <br /> ligated products analysed puri&#239;&#172;?ed HPLC             &#239;&#172;?lter (Merk Millipore &#226;&#8364;&#732;V&#226;&#8364;&#8482; Series Membranes, 0.025 mm) denaturing 20% polyacrylamide gel electrophoresis.         prior use. reaction mixtures incubated Yield HPLC puri&#239;&#172;?cation 49&#226;&#8364;&#8220;50%.                       37  C 1 h, shaking 300 rpm, followed                                                                addition 30 ml slurry Ni&#226;&#8364;&#8220;NTA resin <br /> Site-directed mutagenesis                                      (Ni SepharoseTM 6 Fast Flow, GE Healthcare) <br /> PCR ampli&#239;&#172;?cation carried follows: typical         pre-washed Ni wash buffer [20 mM Tris base, <br /> reaction performed total volume 50 ml,         20 mM Imidazole, 500 mM NaCl, 10%(v/v) Glycerol, <br /> taining 10 ng plasmid DNA, 0.3 mM mutagenic              1% (v/v) Triton X-100]. mixtures incubated primers, 5 ml 10  KOD Hot Start reaction buffer, 4 ml          4  C shaking 1 h. Ni&#226;&#8364;&#8220;NTA resin washed <br /> 1.5 mM MgSO4, 5 ml 0.2 mM dNTPs, 0.02 U/ml KOD                 times 1 ml Ni wash buffer. bound protein <br /> Hot Start Polymerase (Novagen, Cat.  71 086-3).         eluted 25 ml elution buffer [Tris base 20 mM, <br /> reactions hot-started heating 95  C 2 min.      Imidazole 500 mM, NaCl 200 mM, 5%(v/v) Glycerol, reaction subjected cycles 95  C       0.1% (v/v) Triton-X 100]. product analysed 20 s, 54  C 10 s 70  C 2 min. solution     western blot. <br /> allowed cool room temperature 1 h. <br />                                                                Western blot analysis <br /> DpnI digestion puri&#239;&#172;?cation SDM products <br />                                                                Samples resolved SDS&#226;&#8364;&#8220;PAGE (12% polyacryl- <br /> DpnI restriction endonuclease (NEB, Cat.  R0176L)           amide), protein transferred PROTRAN nitro- directly added product ampli&#239;&#172;?ca-       cellulose membrane (Bio-Rad). Membranes blocked <br /> tion reaction, incubated room temperature 6 h.      5% non-fat dry milk PBS-Tween 1 h, followed SDM product separated remaining               incubation 5% milk PBS-Tween 1:1000 <br />  10570 Nucleic Acids Research, 2012, Vol. 40,  20 <br />  <br />  <br />  <br />  <br /> Figure 2.   click-oligonucleotides used site directed mutagenesis contained silent C mutation (shown blue), introduces BamHI restriction site present native mCherry gene. click-linked bases shown red. (B) Assembly click-linked <br /> pRSET-mCherry plasmid site directed mutagenesis, introducing BamHI watermark. (C) Gel electrophoresis (0.8% agarose gel) SDM <br /> products (expected size 3577 bp); Lane 1, 2-log DNA ladder (New England Biolabs); lane 2, pRSET-mCherry SDM normal primers; lane 3, <br /> pRSET-mCherry SDM normal primers followed DpnI digestion; lane 4, pRSET-mCherry SDM using click primers; lane 5, pRSET-mCherry <br /> SDM using click primers followed DpnI digestion; lane 6, negative control (pRSET-mCherry SDM using water instead primers); lane 7, <br /> negative control followed DpnI digestion; lane 8, pRSET-mCherry template plasmid. <br />  <br />  <br />  <br /> dilution anti antibody overnight 4  C.                  RESULTS DISCUSSION <br /> followed incubation 5% milk PBS-Tween                           Construction click-pRSET-mCherry plasmid SDM <br /> containing 1:1000 dilution horseradish peroxid- <br /> ase-conjugated secondary antibody 1 h room tem-                     optimized SDM protocol (16) used incorpor- <br /> perature. ImmunStar Western C (Bio-Rad) used                        ate click linkers strand gene encoding detect horseradish peroxidase-labelled secondary                       &#239;&#172;&#8218;uorescent protein mCherry (12). Forward reverse <br /> antibody, followed imaging analysis                            mutagenic primers, containing pair click-linked <br /> ChemiDoc MP imaging (Bio-Rad), quanti&#239;&#172;?-                         cytosine nucleosides designed overlap cation Image Lab software (Bio-Rad).                                  region encoding &#239;&#172;&#8218;uorophore mCherry (Figure <br />                                                                            2A), formed tripeptide MYG motif   <br />                                                                            thionine 71, tyrosine 72 <span id='am-29' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-30' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>glycine</span> 73) protein. <br /> mCherry expression E. coli experiments                                  primers introduced click linkers, 4-bp apart, product SDM click-linked primers trans-                    region critical &#239;&#172;&#8218;uorescence mCherry. <br /> formed heat-shock KRX competent cells                           transformed E. coli, deletions muta- <br /> detailed  volume recovery mixture                     tions arising presence click linkers <br /> increased 5 ml LB media, grown OD600                     expected result easily identi&#239;&#172;?able, 0.6 reached. Recombinant protein expression                     non-&#239;&#172;&#8218;uorescent phenotype. click SDM primers <br /> induced 18  C 0.1 mM IPTG, shaken 18  C                     designed introduce silent BamHI restric- <br /> overnight. cells pelleted frozen  80  C.                   tion site, allow progeny click-linked plasmids cell pellet suspended 2 ml MHW buffer                     readily distinguishable parent <br /> [20 mM Tris base, 10 mM imidazole, 400 mM NaCl, 1 mM                       pRSET-mCherry plasmid (Figure 2A). Introduction Na3VO4, 10 mM NaF, 1 mM DTT, Protease inhibitor                            watermark SDM (Figure 2B) changes codon <br /> cocktail tablets (Roche), 20% (v/v) Glycerol, 1% (v/v)                     used <span id='am-41' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-42' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Gly" datatype="xsd:string"></span>glycine</span> MYG &#239;&#172;&#8218;uorophore GGC <br /> Triton X-100] lysed sonication. lysate                      GGT, codons similar relative abun- <br /> centrifuged 8000 rpm 20 min supernatant                     dance E. coli (3.0% 2.8%, respectively), applied 100 ml slurry Ni&#226;&#8364;&#8220;NTA resin                              change expected signi&#239;&#172;?cantly affect protein <br /> (Ni SepharoseTM 6 Fast Flow, GE Healthcare)                                expression levels. <br /> pre-washed Ni wash buffer [20 mM Tris base,                              click-linked forward reverse mutagenic primers <br /> 20 mM Imidazole, 500 mM NaCl, 10%(v/v) Glycerol,                           synthesized CuAAC reaction 1% (v/v) Triton X-100]. 1 h incubation 4  C                       oligonucleotide 30 -alkyne modi&#239;&#172;?cation shaking, beads bounded protein washed                        50 -azide group (ODN-1 ODN-3 times 1 ml Ni wash buffer eluted                       ODN-2 ODN-4, Table 1) presence splint <br /> 25 ml elution buffer [Tris base 20 mM, Imidazole                        (ODN-5 ODN-6 respectively, Table 1) CuI <br /> 500 mM, NaCl 200 mM, 5%(v/v) Glycerol, 0.1% (v/v)                          catalyst. presence splint, conversion Triton-X 100]. puri&#239;&#172;?ed protein analysed using                      CuAAC reaction quantitative. 12% polyacrylamide SDS&#226;&#8364;&#8220;PAGE Tecan M200pro                            click-linked oligonucleotides puri&#239;&#172;?ed HPLC <br /> &#239;&#172;&#8218;uorescent plate reader.                                                    remove splint) used subsequent experiments. <br />                                                                                     Nucleic Acids Research, 2012, Vol. 40,  20 10571 <br />  <br />  <br />    plasmid used study pRSET-                                puri&#239;&#172;?ed gel electrophoresis ensure absence mCherry,(12) high copy number plasmid pUC                          template subsequent reactions. <br /> origin encodes mCherry control bac- <br /> teriophage T7 promoter. high copy number                         Biocompatibility pRSET-click-mCherry E. coli <br /> plasmid maximizes number times click-linked <br /> DNA replicated, propagating arising                        puri&#239;&#172;?ed products SDM reactions <br /> errors replication. click-containing primers                      (click, positive control negative control) initially <br /> canonical equivalents  positive control)                            transformed high-ef&#239;&#172;?ciency E. coli DH5a assess incorporated pRSET mCherry using pfu poly-                      yield SDM protocol primer type.                                                                             strain DH5a used lacked chromosomal copy T7 <br /> merase, KOD polymerase. procedure                                                                             RNA polymerase, red-&#239;&#172;&#8218;uorescent phenotype expected <br /> repeated water instead mutagenic primers                                                                             mCherry  expression control negative control. yield SDM product                                                                             bacteriophage T7 promoter) observed. signi&#239;&#172;?cantly higher KOD polymerase <br />                                                                             SDM product using click-linked primers KOD <br /> (Supplementary Data), line previous reports.(17) <br />                                                                             polymerase resulted average 179 colonies LB <br /> KOD polymerase used following ex- <br />                                                                             agar plate supplemented 100 mg/ml ampicillin, periments. products subsequently digested <br />                                                                             positive control SDM canonical DNA resulted DpnI restriction endonuclease remove                                                                             185 colonies, negative control SDM water <br /> methylated template plasmid; note absence                            place mutagenic primers resulted colonies <br /> plasmid template (band  2 kb, reference lane 8)                       (Supplementary Table 1). order utilize red- <br /> lanes 3, 5 7 compared presence lanes 2, 4                     &#239;&#172;&#8218;uorescent phenotype rapidly identify functional 6 Figure 2C. expected, SDM                           copies mCherry gene, SDM products <br /> product present negative control reaction (lanes 6                   transformed KRX strain E. coli 7, Figure 2C). incubation time DpnI                        (Promega), high transformation ef&#239;&#172;?ciency derivative <br /> optimized (Supplementary Data), plasmid                            express T7 RNA polymerase control template fully degraded 6 h. SDM product                      rhamnose promoter. Transformation SDM <br />                                                                             product click-linked DNA primers resulted 78 <br />                                                                             colonies, compared 84 colonies positive <br /> Table 1. Oligonucleotides used synthesis click-linked        control using native mutagenic primers. colonies <br /> mutagenic primers                                                           visibly pink colour displayed red &#239;&#172;&#8218;uorescence <br />                                                                             viewed UV light (Figure 3A). single colony <br /> Code                                  Oligonucleotide sequences (50 &#226;&#8364;&#8220;30 ) <br />                                                                             observed plate containing transformants ODN-1                                 AGTTCATGTACGGATMeCk                   negative control SDM product, indicating number ODN-2                                 CGTAGGCCTTGGATMeCk                    colonies arising background levels parent <br />                                       ODN-3 <br />                                                                              CAAGGCCTACG                          plasmid. experiment repeated 10 times <br /> ODN-4                                  CGTACATGAACT                         (Supplementary Table 2) generate statistically signi&#239;&#172;?- <br /> ODN-5                                 CATGTACGGATCCAAG <br /> ODN-6                                 AGGCCTTGGATCCGTA                      numbers; colonies containing triazole-containing <br />                                                                             plasmids red cases, contained 92.7 &#194;&#177; 8.3% <br /> Ck, alkyne-modi&#239;&#172;?ed <span id='am-11' about='obo:IMR_0200161' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="oboInOwl:id" content="IMR:0200161" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="Cytidine" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasDbXref" content="KEGG:C00475" datatype="xsd:string"></span>cytidine</span>, Cz, azide-modi&#239;&#172;?ed cytidine.                    colonies native plates, negative <br />  <br />  <br />  <br />  <br /> Figure 3. Biocompatability click DNA E. coli.   plate left contains transformants undergone SDM native primers, plate right shows transformants SDM triazole-containing primers. replicates experiment performed, colonies showing expected mCherry &#239;&#172;&#8218;uorescence viewed UV light source. (B) Comparison number colonies water control (W), native (N) triazole (T) plates. triazole plates contained 92.7 &#194;&#177; 8.3% colonies native plates, negative control plate contained &lt;1%. <br />  10572 Nucleic Acids Research, 2012, Vol. 40,  20 <br />  <br />  <br />  <br />  <br /> Figure 4.   (B) Representative sequencing data mCherry gene colonies triazole DNA plates. forward strand shown  , complementary strand shown (B), location triazole backbone linkers shown strand. (C) Lane 1 2-Log <br /> DNA ladder (New England Biolabs), lane 2 BamHI restriction digestion native pRSET mCherry plasmid, lane 3 progeny click-modi&#239;&#172;?ed variant. presence additional BamHI watermark incorporated click-SDM primers results highlighted 229-bp <br /> fragment lane 3. <br />  <br />  <br /> control plates containing &lt; 1% colonies                             transcription/translation used. vitro <br /> (Figure 3B).                                                               contains T7 RNA polymerase S30 Extract <br />    con&#239;&#172;?rm origin plasmid colonies                 components required translation (18&#226;&#8364;&#8220;20),  <br /> transformed product SDM click-linked                      formation inferred effect click-linker <br /> primers, 20 colonies picked pRSET-                          &#239;&#172;?delity rate transcription linker <br /> mCherry plasmid expressed, isolated sequenced.                    T7 RNA polymerase, re&#239;&#172;&#8218;ected quantity cases examined, forward reverse                        &#239;&#172;&#8218;uorescent mCherry produced  protein <br /> strands contained BamHI watermark, errors                      quantity lower canonical DNA <br /> observed sequence mCherry gene                      T7 RNA polymerase slowed click linker, (Figure 4A B). parental origin additional 10                 errors translation skipping colonies probed restriction digestion BamHI.                   click-linked bases polymerase result loss native pRSET-mCherry plasmid contains BamHI                          &#239;&#172;&#8218;uorescent phenotype protein product. restriction site multiple cloning site prior start               noted systems optimized use mCherry; progeny click-containing plasmids                  plasmid DNA, products SDM typ- <br /> contain additional BamHI watermark introduced                       ically sub-cloned expressed E. coli prior use mutagenic click-linked primers region encoding                  templates vitro protein production. case MYG &#239;&#172;&#8218;uorophore. Restriction digestion                          click-pRSET-mCherry  expression plasmids BamHI result bands,                    plasmid sub-cloning result loss 229 bp 3348 bp, digestion                 click DNA-linker, DNA replication E. coli results native plasmid lead single band 3577 bp.                  phosphodiester backbone. sought bands observed colonies examined                      directly use SDM product template click-SDM plate, demonstrating presence                     vitro transcription/translation. quantity BamHI watermark (Figure 4C), directly linking                          puri&#239;&#172;?ed DNA isolated reaction typically heritage click-containing SDM primers.                         0.1 mg, (10-fold lower 1 mg plasmid recom- <br />    &#239;&#172;?delity click-DNA replication                        mended manufacturer), 10 SDM reactions   <br /> probed analysing progeny click-pRSET                         carried parallel click-linked primers, <br /> mCherry plasmids single colony. trans-                        canonical primers (positive control) water (negative <br /> formed click-containing plasmid replicated multiple                     control). products set pooled used times bacterium, errors process                    template vitro transcription/translation observed isolating, transforming assessing                    mCherry. SDM product used directly plasmids single colony. Plasmids 10                            template vitro transcription/translation, treat- <br /> click-transformed colonies isolated trans-                     ment SDM product T4 DNA ligase did formed KRX cells. resulting colonies displayed                    affect yield reaction (Supplementary Figure S5). red-&#239;&#172;&#8218;uorescent phenotype associated correct                       mCherry protein produced reaction readily <br /> replication click-containing mCherry gene                           visualized quanti&#239;&#172;?ed western blot; yield (Supplementary Figure S4).                                                 vitro mCherry production click-SDM <br />                                                                            product comparable canonical-SDM <br />                                                                            product (Figure 5A), &#239;&#172;&#8218;uorescence emission <br /> Protein expression click-pRSET-mCherry <br />                                                                            associated mCherry 610 nm (Figure 5B), probe compatibility click-DNA linker                    product &#239;&#172;&#8218;uorescence observed negative control transcription machinery, vitro coupled                           template. results indicate transcription                                                                                       Nucleic Acids Research, 2012, Vol. 40,  20 10573 <br />  <br />  <br />  <br />  <br /> Figure 5.   Western blot product vitro transcription/translation using SDM products water (lane 1), canonical primers (lane 2) click-linked primers (lane 3), template. intensity band quanti&#239;&#172;?ed blot (average repeats). (B) Fluorescent <br /> emission mCherry  610 nm) vitro transcription/translation pRSET-mCherry constructed SDM click-linked primers (red line), <br /> normal primers (blue line); &#239;&#172;&#8218;uorescence observed negative control (black line). <br />  <br />  <br />  click linker vitro results functional mCherry                      primers (Figure 6B). expected solution protein, similar quantity equivalent                     negative control plasmid &#239;&#172;&#8218;uorescent. rate canonical DNA.                                                               mCherry expression normal click-linked <br />    effect click DNA linker DNA replication                     SDM products assessed monitoring protein transcription E. coli probed                           levels 1, 2, 3, 4 12 h IPTG induction, comparing &#239;&#172;&#8218;uorescence mCherry protein                             comparable (Supplementary Figure S6). produced click-linked DNA canon-                       similarity mCherry quantity quality produced ical equivalent. determine &#239;&#172;&#8218;uorescence                    click-linked plasmid equivalent canonical <br /> mCherry individual colonies,                            DNA, vitro vivo experiments <br /> expected stochastic variation (21), sought                     (Figures 5 6) demonstrates biocompatibility <br /> assess total mCherry levels population trans-                      product click DNA-backbone ligation formants. Typically starter culture single                          con&#239;&#172;?rms potential alternative enzymatic <br /> colony used protein expression  ensure genomic                     DNA ligation. <br /> uniformity); case  transform- <br /> ation recovery mixture used starter culture protein expression. allowed total mCherry expres-                   CONCLUSIONS <br /> sion population transformants assessed, <br /> averaging variations mCherry levels single                        essential prerequisite use chemically <br /> cells. product SDM click primers, native                         modi&#239;&#172;?ed DNA molecular biology modi&#239;&#172;?ca- <br /> primers water transformed KRX competent                          tion benign, modi&#239;&#172;?ed DNA cells, recovery 1 h, volume solution                functional mimic natural counterpart. triazole increased 5 ml LB media, cells grown                    linkage Figure 1a ful&#239;&#172;?ls criterion surprisingly  OD600 0.6. IPTG added induce mCherry                            despite apparent dissimilarity normal phospho- <br /> expression, resulting protein af&#239;&#172;?nity                            diester bridge; triazole linkage accurately ef&#239;&#172;?- <br /> puri&#239;&#172;?ed. transformants placed                          ciently read DNA RNA polymerases <br /> selective pressure maintain plasmid encoding                          (8,11), making click DNA ligation promising alternative <br /> mCherry, &#239;&#172;&#8218;uorescent protein produced                           enzymatic ligation. demonstrated affected replication rate transformed                  biocompatibility click DNA-backbone linker plasmid, informing effect click DNA                            non-essential gene, determining presence select- <br /> linker process. isolated protein product                    ive pressure required correct function click-SDM native plasmids visibly pink                        E. coli. click-linked mCherry gene replicated <br /> colour, negative control solution colour-                    transcribed high &#239;&#172;?delity despite close (Figure 6A, insert). Analysis puri&#239;&#172;?ed protein                    proximity triazole backbone linkers (4-bp apart), SDS&#226;&#8364;&#8220;PAGE showed band 30 kDa, line                            demonstrates robustness click DNA mass expected mCherry (Figure 6A). function-                     backbone linker mimics canonical DNA. &#239;&#172;?ndings <br /> ality &#239;&#172;&#8218;uorescent protein produced                           suggest possibility alternative, fully synthetic <br /> plasmid assessed comparing &#239;&#172;&#8218;uorescence                         approach gene synthesis potential signi&#239;&#172;?- <br /> 2 ml mCherry solution using &#239;&#172;&#8218;uorescence plate                      cance synthetic biology industrial biotechnology. <br /> reader. excited 587 nm, protein                                  approach particularly relevant synthesis <br /> expressed click-pRSET-mCherry showed similar                            large DNA strands containing site-speci&#239;&#172;?c epigenetic <br /> levels &#239;&#172;&#8218;uorescent emission 610 nm                     modi&#239;&#172;?cations, DNA constructs dif&#239;&#172;?cult pRSET-mCherry constructed SDM normal                                 make PCR-based methods. <br />  10574 Nucleic Acids Research, 2012, Vol. 40,  20 <br />  <br />  <br />  <br />  <br /> Figure 6.   SDS&#226;&#8364;&#8220;PAGE Ni2+ af&#239;&#172;?nity puri&#239;&#172;?ed mCherry (30 kDa) colonies transformed SDM product water (negative control, <br /> lane 2), canonical (positive control, lane 3), click-linked primers (lane 4). Lane 1 NEB ColorPlus Prestained Protein Ladder insert shows readily visible mCherry protein product 30 kDa using normal (left) click-linked primers (right) white light. (B) Fluorescent emission recombinant mCherry (610 nm) expressed pRSET-mCherry constructed SDM click-linked primers (red line), normal primers (blue <br /> line) water control (black line). Insert shows puri&#239;&#172;?ed mCherry pRSET-mCherry constructed SDM normal (left) click-linked <br /> (right) primers, UV light 365 nm. <br />  <br />  <br /> SUPPLEMENTARY DATA                                                                Brownley , Thomas,D.W., Algire,M  et al. (2008) Complete <br />                                                                                   chemical synthesis, assembly, cloning Mycoplasma <br /> Supplementary Data available NAR Online:                                   genitalium genome. Science, 319, 1215&#226;&#8364;&#8220;1220. <br /> Supplementary Tables 1&#226;&#8364;&#8220;3, Supplementary Figures 1&#226;&#8364;&#8220;6                            4. Gibson,D.G., Glass,J , Lartigue,C., Noskov,V.N., Chuang,R.Y., Supplementary Methods.                                                        Algire,M , Benders,G , Montague,M.G., Ma,L., <br />                                                                                   Moodie,M.M. et al. (2010) Creation bacterial cell Controlled <br />                                                                                   chemically synthesized genome. Science, 329, 52&#226;&#8364;&#8220;56. <br />                                                                                5. El-Sagheer H. Brown,T. (2009) Synthesis polymerase <br /> ACKNOWLEDGEMENTS                                                                  chain reaction ampli&#239;&#172;?cation DNA strands containing                                                                                   unnatural triazole linkage. J.  Chem. Soc., 131, 3958&#226;&#8364;&#8220;3964. authors thank Professor Roger Tsien providing                          6. Herrlein,M.K., Nelson,J.S. Letsinger,R.L. (1995) covalent <br /> pRSET-mCherry, Dr Patrick Duriez Dr Fedor                                     lock self-assembled oligonucleotide conjugates. J.  Chem. <br />                                                                                   Soc., 117, 10151&#226;&#8364;&#8220;10152. <br /> Forafonov helpful discussions, Dr Paul                                 7. Xu,Y  Kool,E.T. (1998) Chemical enzymatic properties <br /> Duckmanton taking photograph Figure 6B.                                bridging 5&#226;&#8364;&#8482;-S-phosphorothioester linkages DNA. Nucleic <br />                                                                                   Acids Res., 26, 3159&#226;&#8364;&#8220;3164. <br />                                                                                8. El-Sagheer H., Sanzone P., Gao,R., Tavassoli                                                                                    Brown,T. (2011) Biocompatible arti&#239;&#172;?cial DNA linker read <br /> FUNDING                                                                           DNA polymerases functional Escherichia <br />                                                                                   coli. Proc. Natl Acad. Sci. USA, 108, 11338&#226;&#8364;&#8220;11343. <br /> Cancer Research UK [Career Establishment Award 10 263                          9. Rostovtsev,V.V., Green,L.G., Fokin,V.V. Sharpless,K.B. T.]; Engineering Physical Sciences Research                              (2002) stepwise huisgen cycloaddition process: copper - <br /> Council [EP/H04986X/1 T.]; Southampton                                       catalyzed regioselective &#226;&#8364;&#732;&#226;&#8364;&#732;ligation&#226;&#8364;&#8482;&#226;&#8364;&#8482; azides terminal alkynes. <br /> University Life Sciences Interface [studentship P.S.                        Angew. Chem. Int. Ed. Engl., 41, 2596&#226;&#8364;&#8220;2599. T.]; Biotechnology Biological Sciences                           10. Tornoe,C.W., Christensen,C. Meldal,M. (2002) <br />                                                                                   Peptidotriazoles solid phase: [1,2,3]-triazoles regiospeci&#239;&#172;?c <br /> Research Council [sLOLA grant BB/J001694/1                                     copper -catalyzed 1,3-dipolar cycl <br /> </body></html>