<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Feb. 1991, p. 721-730                                                                    Vol. 11,  2 <br /> 0270-7306/91/020721-10$02.00/0 <br /> Copyright C 1991, American Society Microbiology <br />  <br />  <br />       Characterization RPRI, Essential Gene Encoding RNA <br />          Component Saccharomyces cerevisiae Nuclear RNase P <br />            JAE-YONG LEE, CHRISTOPHER E. ROHLMAN, LISA  MOLONY, DAVID R. ENGELKE* <br />              Department Biological Chemistry, University Michigan, Ann Arbor, Michigan 48109-0606 <br />                                         Received 27 August 1990/Accepted 8 November 1990 <br />  <br />               RNA components identified preparations RNase P number eucaryotic sources,            final proof RNAs true RNase P subunits elusive eucaryotic RNAs, unlike <br />            procaryotic RNase P ribozymes, shown catalytic activity absence protein. <br />            previously identified RNA component Saccharomyces cerevisiae nuclear RNase P preparations <br />            characterized corresponding, chromosomal gene, called RPRI (RNase P ribonucleoprotein <br />            1). Gene disruption experiments showed RPR1 single copy essential. Characterization gene <br />            region located RPRI 600 bp downstream URA3 coding region chromosome V. sequenced 400 <br />            bp upstream 550 bp downstream region encoding major 369-<span id='am-14' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-17' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> RPRI RNA.            presence abundant, potential precursor RNAs extra 84 nucleotides 5&apos; leader 30 <br />            nucleotides 3&apos; trailing sequences suggests primary RPRI transcript subjected multiple <br />            processing steps obtain 369-nucleotide form. Complementation RPRI-disrupted haploids            variant RPRI gave slow-growth temperature-sensitive phenotype. strain accumulates tRNA <br />            precursors lack 5&apos; end maturation performed RNase P, providing direct evidence RPRI RNA <br />            essential component enzyme. <br />  <br />   RNase P procaryotic eucaryotic sources                essential gene encoding RNA identified charac- endonuclease cleaves pre-tRNA substrates yield                terized. Disruption gene, termed RPRI (RNase P <br /> mature 5&apos; termini. procaryotes, RNA protein              ribonucleoprotein 1), complementation plasmid- <br /> components ribonucleoprotein enzymes required                 borne variant defective growth leads accumulation vivo activity (15, 27, 29, 33, 43, 49, 57, 58),           tRNA precursors lacking 5&apos; RNase P cleavage. RNA component capable efficiently catalyzing             observation confirms identity S. cerevisiae RPRI <br /> correct reaction conditions vitro (3, 18, 21, 22,          RNA essential component RNase P suggests <br /> 52). Extensive phylogenetic sequence comparisons                similar RNAs isolated eucaryotes RNAs (30), combined folding energy calculations                 bona fide RNase P components. <br /> cleavage sensitivity studies (6, 19, 51, 52), suggest  <br /> served, highly ordered secondary structure. key contacts substrates depend struc-                             MATERIALS METHODS <br /> ture, protein contributing efficiency                  Strains genetic methods. S. cerevisiae W3031A (MA Ta <br /> secondary effects charge shielding RNA                   ade2-1 his3-11,15 leu2-3,112 trpl-J ura3-1 cani -100) chain phosphate backbones. mechanism pre-                   isogenic strain W3031B (MATot ade2-1 his3-11,15 leu2-3,112 <br /> tRNA substrates recognized clearly understood,                trpl-l ura3-1 can1-100) obtained E. Esposito  obvious regions required                  (originally R. Rothstein) used gene disruption, com- <br /> Watson-Crick base pairing enzyme sub-                    plementation analysis, Southern Northern (RNA) <br /> strate RNAs (20, 36).                                                    analyses. Strains BJ411 (MATTa  prbl-1122 prel-126), <br />    Studies eucaryotic RNase Ps suggested               previously referred EJ101 (31; gift E. Jones), contain essential RNA subunits, role            PP1002 (MATTa ade2 leu2-3,112 pep4-3 rna3 rna82) (46; gift <br /> RNA components firmly established. RNAs                     P. Piper) used preparation RNase P end copurify nuclear organelle RNase Ps              analysis RPRI RNA transcript. Strain YPH149 characterized (1, 7, 9, 11, 12, 14, 24, 32, 34, 35, 37, 38,         (MATa ade his7 lys2 ura3-52 tri-di::TRPJ+) obtained <br /> 40, 42) shown sequence structural similar-               P. Hieter used chromosome mapping. Yeast <br /> ities lesser degree procaryotic            transformation performed lithium acetate proce- <br /> RNAs (5, 37). identification eucaryotic RNAs               dure (28). Sporulation tetrad analysis performed essential RNase P subunits rigorously                   using standard genetic methods (59). Cells grown firmed,  reports eucaryotic            selection medium contained 0.67% yeast nitrogen <br /> RNAs having enzyme activity absence protein                base amino acids, 2% dextrose amino acids studies report RNase Ps essential RNA                       selection, YPD medium (2% dextrose, 2% peptone, 1% <br /> component discernible (9, 63).                                           yeast extract). <br />    previously reported sequence 369-nucleotide                 Cloning RPRI gene. yeast genomic DNA library RNA copurifies nuclear RNase P activity                   centromere-containing plasmids (YCp5O library) ob- <br /> Saccharomyces cerevisiae. report, single-copy,                 tained D. Thiele (54; originally D. Botstein G. <br />                                                                          Fink) screened colony hybridization (2).                                                                          probe used screening prepared vitro <br />   * <br />       Corresponding author.                                              transcription antisense RPRJ cDNA clone T7 <br />                                                                    721 <br />  722     LEE ET AL.                                                                                             MOL. CELL. BIOL. <br />  <br /> RNA polymerase presence  32P]<span id='am-22' about='obo:IMR_0200199' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="oboInOwl:id" content="IMR:0200199" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="UTP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00075" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>UTP</span> (10).             tween CUP] promoter CYCI terminator eral clones chosen 23 positive clones           single-copy centromere vector 2,um-based multicopy <br /> subsequently used restriction mapping subcloning.        vector. plasmid constructs used test 1.3-kb BglII-HindIII fragment subcloned pUC19,          complementation disrupted chromosomal RPRI gene strands sequenced dideoxynucleotide chain         transformation diploid strain heterozygous termination method, using modified T7 DNA polymerase               disruption tetrad analysis resulting transfor- <br /> (26) oligonucleotide primers hybridize intervals       mants. RPRI region (37). T7 RNA polymerase (10)              Northern analysis RPRI RNA tRNA. wild-type <br /> modified T7 DNA polymerase (62; gift J. Huibregtse)        haploid strain (W3031A), RPRI-disrupted haploid strain <br /> prepared described previously. Primers sequencing           complemented multiple copies RPRI cDNA  prepared Center Molecular Genetics Core            struct, RPRI-disrupted haploid strain complemented <br /> Facility University Michigan School Medicine.         multiple copies RPRI cDNA dimer construct    Gene disruption Southern analysis. 1.75-kb BamHI          grown YPD medium overnight 30&#194;&#176;C. Fresh YPD <br /> fragment containing HIS3 gene isolated                medium inoculated overnight cultures YCp407 (39) using agarose gel electrophoresis elec-         initial OD650 0.05 grown 30 38&#194;&#176;C. Cell <br /> troelution (41). isolated fragment inserted      growth monitored spectrophotometrically 2 h BamHI site RPRI genomic clone. W303 diploid cells           determine doubling time. milliliter (approximately 1.2 x transformed linearized HIS3-interrupted              108 cells) cultures harvested OD650 1.0 <br /> RPRI genomic DNA clone, transformants selected            used preparation total RNA (23). RNA defined medium lacking histidine (55). resulting            samples separated 6% denaturing acrylamide gel <br /> transformants sporulated dissected tetrad anal-       transferred nylon membrane, using semidry <br /> ysis. DNAs isolated haploids (W3031A             electroblotter (Gelman Sciences, ) conditions W3031B), parental diploid (W303), hetero-          specified manufacturer. RNA cross- <br /> zygous diploid, single RPRJ disruption (59).            linked membrane UV exposure (Fotodyne UV light <br /> DNAs digested StuI, separated 1% agarose            mode 3-3000) 3 min. membrane probed gel, transferred nylon membrane (41). mem-             32P]UTP-labeled antisense RPRI RNA labeled  <br /> brane probed  32P]UTP-labeled antisense          tisense tRNALeU3 (37, 59). vitro digestion RNA <br /> RPRI RNA transcript previously described (37, 59).              nuclear RNase P, highly purified fraction su- <br />    Mapping RPR1 gene. Strain YPH149 grown            crose gradient centrifugation used previously  optical density 650 nm (OD650) 0.4. Cells            scribed (37). <br /> harvested embedded agarose gel make chromo-               5&apos; end analysis. 5&apos; end RPRI transcripts somal DNA plug described Southern et al. (60).               determined using primer extension labeled  <br /> Chromosomes separated 1.0% agarose gel 14&#194;&#176;C           tisense oligonucleotide hybridized nucleotides 33 pulsed-field electrophoresis, using Bio-Rad CHEF-             48 RPRI coding region (Fig. 1). total yeast RNA <br /> DRII apparatus. switch time 70 s 15 h 120 s        purified RNase P holoenzyme RNA tested additional 12 h. Seventeen chromosomes visible         gave essentially results: major product corre- staining ethidium bromide. Chromosomes             sponding mature 5&apos; end minor product -84 <br /> partially digested gel soaking twice 0.25 M HCl       corresponding potential precursor (data shown). 15 min transferred nylon membrane         consistent previous results showing standard capillary method (41). membrane              small longer RPRI RNA copurifies probed labeled antisense RPRJ RNA (37, 59).               major 369-nucleotide RNA RNase P holoenzyme. Total <br /> RPRI probe hybridized chromosome V (data               yeast RNA (10 ,g) combined reverse transcriptase <br /> shown). subsequently Riles Olson          buffer, extensions performed previously  RPRI gene localized restriction fragment            scribed (25). primer annealling extension <br /> containing URA3 gene chromosome V (52a). Com-               reactions carried 44&#194;&#176;C. Stop mix (4 ,ul 2% <br /> parison restriction map RPRI 8-kb genomic            sodium dodecyl sulfate, 0.1 M EDTA, 1 mg proteinase K <br /> DNA clone published URA3 region (53)         ml) added, mixture incubated 10 min <br /> sequencing RPRI clone downstream HindIII             44&#194;&#176;C. precipitation, reaction resuspended site confirmed relative positions genes (Fig. 1).   formamide loading buffer, heated 95 100&#194;&#176;C 2 min, <br />    Plasmid construction. 400-nucleotide 5&apos;-flanking         separated 8% denaturing acrylamide gel. gion 550-nucleotide 3&apos;-flanking region amplified       positions primer extension stops determined polymerase chain reaction using Taq polymerase (56)            direct comparison dideoxynucleotide chain termination inserted BamHI HindlIl sites               reactions linearized RPRJ genomic clone, using single-copy centromere vector YCpSO 2,m-based               labeled primer (26). <br /> multicopy vector YEp24. resulting constructs contained            S1 analysis. probe determination 3&apos; end EcoRI site junction flanking regions.         RPRI transcripts protection S1 nuclease RPRI cDNA, Escherichia coli RNase P Ml cDNA (50),                  generated RPRI-containing RsaI fragment cloned <br /> human RNase P Hi cDNA (4; gift S. Altman), S.                   EcoRI polylinker site pUC19 use  cerevisiae mitochondrial RNase P RNA cDNA, RPRI            EcoRI linkers. DNA linearized unique Styl <br /> cDNA dimer ligated EcoRI linker inserted        site coding region RPRI gene, 5&apos; <br />  EcoRI site. S. cerevisiae mitochondrial RNase P         overhangs end filled [t_-32P]<span id='am-27' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dCTP</span>, using standard <br />  RNA cDNA prepared amplification syn-              techniques (41). 330-bp 3&apos; end probe containing thetic primers Taq polymerase (56) genomic clone          antisense strand labeled StyI site removed <br />  (42; gift C. Wise N. Martin). attempt obtain      pUC19 vector cleavage EcoRI. radio- <br />  conditionally expressed clones, various RNase P RNA               labeled strand isolated electrophoresis  cDNAs inserted adjacent GALIO promoter            denaturing 6% polyacrylamide gel electroelution (41). S1 <br />  VOL. 11, 1991                                            RPRI ENCODES S. CEREVISIAE NUCLEAR RNase P RNA                             723 <br />  <br />  <br />                                                                                                  1.3 kb <br />                                                                         m <br />                                 K                 P BgSt       Bg St B       HSStAPH           H         Bg <br />                                                    l11ii           II -_ W- <br />                                                                                  IIA                                                                       RPR1         URA3 <br />                                     1 kb <br />                    B <br />                          BglIl                                    SpeI <br />                     -407 agatctgcca attgaacata acatggtagt tacatatact agtaatatgg ttcggcacac <br />                     -347 attaaaagta taaaaactat ctgaattacg aattacatat attggtcata aaaatcaatc <br />                     -287 aatcatcgtg tgttttatat gtctcttatc taagtataag aatatccata gttaatattc <br />                                                                                                EpaI <br />                     -227 acttacgcta ccttttaacc tgtaatcatt gtcaacaggt atgttaacga cccacattga <br />                                                                                  NaeI <br />                    -167 taaacgctag tatttctttt tcctcttctt attggccggc tgtctctata ctcccctata <br />                                                                StuI        Box <br />                    -107 gtctgtttct tttcgtttcg attGTTTTAC GTTTGAGGCC TCGTGGCGCA CATGGTACGC <br />                                                                            1 <br />                     -47 TGTGGTGCTC GCGGCTGGGA ACGAAACTCT GGGAGCTGCG ATTGGCAC.*_aC<span id='am-7' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>;MGGT <br />                                                                                BamilI <br />                        14 AATTCCTaC ATTAAaAAAC! CTGTTTACAG                QA=^ACCC         CTACGG    GTTATCAG <br />                        74 ATATTATCAQTGAkTsGTGAC                           CAGTGGA=GC TTG2TCCTCCG GGTTAATGTC <br />                      134 GCTTTTGGC TClCTGCTCCT                      AIAATATA (%-GGAAC AGTCTTTACC <br />                                                                      kA <br />                      194 <br />                      254 ACCIGTTGTTC CGTTACGCGCoCCT ACcGGCrTr.Ar CGTCAfGC GTaGC <br />                                    Aatll                                    B Box <br />                      314 AATCGTAGG CGTCATTAGT GGCGAACCCG ATACCGATTA CTGCTGCTGT TCCAGCCCAT <br />                      374 ATCCAACTTC CAATTTAATC TTTCTTtttt aattttcact tatttgcgat                          acagaaagaa <br />  <br />                      434 aaaagcgata gtaactattg aattttgttt ggatttggtt agattagata tggtttctct <br />                      494 ttatatttac atgctaaaaa tgggctacac cagagataca taattagata tatatacgcc <br />                                                          Styx <br />                      554 agtacacctt atcggcccaa gccttgtccc aaggcagcgt tttgttcttg gaaacgctgc <br />                      614 cctacacgtt cgctatgctt caagaacttt tctgagcact tcatgatgca tgtttgttcc <br />                      674 ttattggtta gctttgatgt tgtgaagtca ttgacacagt ctgtgaaaca tctttctacc <br />                      734 agattagagt acaaacgcat gaaatccitc atttgctttt gttccactac tttttggaac <br />                      794 tcttgttgtt ctttcggagt tcaatgcgtc catctttaca gtcctgtctt attgttcttg <br />                      854 atttgtgccc cgtaaaatac tgttacttgg ttctggcgag gtattggata gttccttttt <br />                                       HindlIl <br />                      914 ataaaggcca tgaagctt <br />    FIG. 1. Location sequence RPRJ gene region.   Location RPR1 gene. RPRI gene located chromosome <br /> V immediately adjacent URA3 described Materials Methods. partial restriction endonuclease map region relative <br /> orientations RPRI URA3 coding regions shown. Restriction sites:  ApaI; B, BamHI; Bg, BglII; H, HindIII; K, KpnI; P, PstI; <br /> S, SmaI; St, StuI. (B) Sequence RPRI region sense strand BglII Hindlll sites. Positions numbered relative nucleotide major RPR1 RNA (+ 1). sequence corresponding major RNA underlined, sequence corresponding longest prevalent longer RNAs (Fig. 6) indicated capital letters (Fig. 2). Sequences corresponding suspected RNA <br /> polymerase III box B box internal promoters indicated overlining  Discussion). Styl site used prepare 3&apos; end S1 <br /> probe indicated. <br />  724    LEE ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br /> nuclease analysis performed previously                         &lt; <br /> scribed (2). Size markers Si-protected fragments                      g unrelated dideoxynucleotide chain termination reac-                    m0:aG.X <br />                                                                             jm <br /> tions adjacent lanes  shown). <br />   Nucleotide sequence accession number. GenBank ac- <br /> cession number RPRI region sequence M27035. <br />                                                                                                    B <br />                                                                                                          T-        X-     0 <br />  <br />                            RESULTS                                                                       Vm <br />                                                                                              nt               m           CL~~~0 <br />    Antisense transcripts cDNA encoding S. cerevisiae                 _-         - -   84 <br /> RNase P-associated RNA (37) used screen S. <br /> cerevisiae genomic library YCp5O filter hybridization <br /> (2). positive clones encompassed cDNA sequence identified. partial restriction endo- <br /> nuclease map 6-kb region surrounding gene shown Fig. 1A. preparation obtaining genetic map                                                                                 nt <br /> location locus, referred subsequently RPRJ                                                 *         ]w               J 385-388 <br /> (RNase P ribonucleoprotein 1), cDNA mapped chromosome V probing chromosomal blots pulsed- <br /> field gel electrophoresis (60; data shown). classi- <br /> cal genetic mapping clone accomplished,                                                 *  localization hybridization chromosomal                                                           *        -3369 <br /> striction digest blots indicated RPRI close URA3 <br /> (52a). Comparison restriction map genomic <br /> RPRJ clones published map URA3 (53) locus <br /> suggested genes adjacent, possibility                                 +1 confirmed extending sequence RPRI region (Fig. 1B;  downstream past HindIII site encountering previously published se- <br /> quence downstream URA3 (53). <br />    sequences strands DNA region <br /> shown Fig. 1B obtained direct analysis pUC19 subclone containing 1.3-kb BglII-HindIII fragment,          FIG. 2. 5&apos; 3&apos; end analysis RPRI RNAs.   Primer <br /> using sequential oligonucleotide primers dideoxynucle-        extension analysis performed described Materials otide chain termination methods double-stranded tem-          Methods 10 ,ug RNA indicated haploid yeast <br /> plates (26). positions corresponding major RPRI        strains radiolabeled primer anneals positions 33 48 RNA underlined. larger RNA copurifies        RPRI sequence. (B) SI nuclease analysis performed RNase P relatively small quantities hybridizes      DNA fragment labeled antisense strand Styl site RPRI probes (37) characterized (Fig. 2),        described Materials Methods. exact 3&apos; terminus extent larger RNA indicated capital letters   major RNA determined previously (37). (Low levels slightly <br /> Fig. 1B. Previous work indicated RPRI RNA            longer 3&apos; termini observed experiments [e.g., Fig. 6] <br /> DNA sequences colinear (37), suggesting             dependent concentration Si nuclease larger RNA longer 5&apos; 3&apos; terminus           experiment.) nt, Nucleotides. <br /> 369-nucleotide major RNA precursor. 5&apos; 3&apos; termini larger RNA mapped primer <br /> extension Si nuclease analysis, respectively, total        sponding additional 28 30 nucleotides past cellular RNA S. cerevisiae strains used      369-nucleotide signal seen experi- previous study (37). Extension performed         ments. data chosen Fig. 2, example, radiolabeled primer anneals 32 nucleotides downstream        obvious longer RNA, data Fig. 6  5&apos; terminus smaller RNA (Fig. 1B).        able demonstrate longer <br /> strains, multiple extension stops corresponding        RNAs primary transcripts RPRI gene slightly heterogeneous 5&apos; end major 369-            precursors major form. conversion <br /> nucleotide RNA significant extension         taking place,  lack stable intermediates product corresponding transcript 84 nucleotides      5&apos; end suggests single step involved, <br /> additional 5&apos; sequence (Fig. 2). S1 nuclease mapping 3&apos;    multiple 3&apos; reactions suggested results <br /> end RNA showed major product corresponding            PP1002 alternate constructs described  unique 3&apos; end 369-nucleotide RNA lesser       Fig. 6). pattern interesting resembles amounts RNAs ranging 16 30 nucleotides longer          pattern pre-tRNA processing, preliminary <br /> (Fig. 2). strains BJ411 W3031A, longer products        evidence RNase P RNAs synthesized RNA <br /> appeared clusters roughly 16 19 nucleotides longer     polymerase III (4; Discussion). Given possibility, major 3&apos; terminus 369-nucleotide RNA.           potential box B box intragenic promoter elements unable tell appearance multiple bands           RNA polymerase III (16) indicated Fig. 1B reflects RNA heterogeneity artifact Si assay.     discussed  80% sequence <br /> Strain PP1002,  harbors rna82 lesion causes       similarity corresponding promoter consensus (16). <br /> aberrant 3&apos; end processing 5S tRNA RNA polymerase             Gene disruption performed strategy outlined III transcripts (46-48). strain, faint signal corre-    Fig. 3 confirm gene single copy, examine <br />  VOL. 11, 1991                                               RPR! ENCODES S. CEREVISIAE NUCLEAR RNase P RNA                                          725 <br />  <br />            RPR1                                                                                DNA Insert <br />  <br />                                                                                  RPR15&apos;F7                 ,RPR 1F <br />                                                                         2 408 bD                                                     Complementation <br />        -i3-                                                                    B           E                E              H <br />                                                                                b |     RPR1 cDNA single &amp; multi-copy vector                 + <br />                                                                40alO <br />                                                                           0 zzzzz4w1bE         coi RNase P MI cDNA single &amp; multi-copy vector   - <br />  <br />  <br />  <br />  <br />                                                                                                                                      0 <br />  <br />                                                                                                                                         ui <br />                                                                            <br />  <br />  <br />  <br />                                                                                                                                                              * .j* <br />  <br />                                                                                    53 <br />  <br />                                                                               ===                                                                                            Human RNase P HI cDNA single &amp; multi-copy vector <br />  <br />                                                                                            S. cerevislae mitochondria RNase P RNA cDNA <br />                                                                                            single &amp; multi-copy vector <br />  <br />                                                                                                     * RPRI cDNA dimer single-copy vector <br />                                                                                                                                          O <br />                                      Sporulation &amp; <br />                                                                                                                                         0 <br />                                                                                   * *        ** RPRI cDNA dimer multi-copy vector             + <br />                                      Tetrad Analysis           00 <br />                                                                             FIG. 5. Complementation disrupted chromosomal RPRI gene <br />                                                                           various RNase P RNA cDNAs. cDNA mature RNase P <br />   FIG. 3. Scheme RPRI gene disruption complementation.            RNA inserted replace 369-nucleotide major RPR1 RNA <br /> Procedural details described Materials Methods.                coding region addition EcoRI linker described Materials <br />                                                                           Methods. plasmids used transform diploid <br />                                                                           heterozygous RPRI gene disruption. Spores obtained gene product essential, prepare rprl             transformants dissected determine complementation. <br />                                                                           +, spores viable result complementation mutant background phenotype defective                     plasmid-borne RNase P RNA gene; -, spores RPRI gene constructs tested. HIS3 insertion                 viable result functional complementation diploids selected growth medium lacking histi-                  plasmid. <br /> dine, followed sporulation tetrad dissection. spores tetrad viable (Fig. 4A), viable <br /> spores   shown), indicating HIS3                    inserted essential locus. Evidence substitutive <br />                                                                           transformation site RPRJ gene provided                                                                           Southern analysis DNA parental diploid,                                                                         diploid heterozygous disruption, viable haploids <br />                                                                           probed RPRI antisense RNA (Fig. 4B). case <br />                                                    1 <br />                                                                           heterozygous disrupted  diploid did StuI diges- <br />                                               - 2                         tion produce RPRJ fragments, normal size                                               -3                          second larger corresponding 1,750-bp <br />                                               -4                          HIS3 fragment. Taken  results indicate                                                                           RPRI single copy essential. <br />              B                                                               experiments conducted test var- <br />                                                                           iants RPRI cDNA sequences heterologous RNase <br />                          b     c    d                                   P RNA component cDNAs complement disrupted <br />                                                                           RPRI gene. experiments conducted                                               -8454                       reasons.  precedent mixed RNase P protein <br />                                               -5686                       RNA components distantly related bacteria (21)                                               - 3675 <br />                                                                           bacteria human cells (17) able reconstitute <br />                                               - 2323                      functional holoenzyme vitro,                                               - 1929                      determine RPRJ product sufficiently related <br />                                                 1371                      analogous RNAs E. coli, human cells, yeast <br />                                                 1264                      mitochondria (42) reconstitute mixed holoenzymes                                                                           vivo. Second, repeated attempts produce partially defec- <br />    FIG. 4. Analyses diploids RPRI gene disruptions. Dip-          tive conditionally lethal mutations RPRI loids selected URA3 plasmid acquisition RPRI gene              unsuccessful, hoped heterologous <br /> disruption scheme (Fig. 3) sporulated, tetrads              altered cDNA constructs defect dissected.   Growth dissected spores rich medium showed           molecular phenotype identifiable RNase P lesion. 2:2 segregation expected disruption essential gene. 1, 2,       heterologous cDNAs mutated RPRI cDNAs 3, 4 refer positions spores       placed control inducible S. cerevisiae RNA tetrads shown. (B) Southern analyses performed         polymerase II promoters plasmids, construct <br /> antisense RPRI RNA probe DNAs wild-type haploid              including bona fide RPRI cDNA complement  strains [lane  W3031A ; lane b, W3031B ], wild-type diploid       shown), suggesting RNase P RNA strain (lane c, W303), diploid strain heterozygous RPRJ      properly synthesized RNA polymerase II. <br /> gene disruption (lane d) cleavage StuI. Sizes RPR1         YCp5O YEp24 plasmid constructs described fragments 1.5 kb wild-type haploid strains (lanes b) wild-type diploid strain (lane c) 1.5 3.25 kb   Materials Methods Fig. 5 devised <br /> strain heterozygous RPRJ gene disruption (lane d). Size            drive expression cDNAs RPRI 5&apos;- markers (indicated nucleotides) lambda phage DNA di-              3&apos;-flanking regions. case, cDNA corresponding <br /> gested BstEII stained ethidium bromide  shown).         indicated mature RNase P RNA inserted  726      LEE ET AL.                                                                                                            MOL. CELL. BIOL. <br />  <br />              TABLE 1. Doubling time YPD medium                                 RNA, little smaller RNA 5&apos; end appeared <br />                                                      Doubling time (h) <br />                                                                                   (Fig. 6B). Longer heterogeneous primer extension products <br />         Strain                                                                    primer annealing second copy                                              300C                        38&#194;&#176;C     RNA dimeric transcripts. Similarly, 3&apos; termini Wild type                                    1.27                        1.60     dimeric transcripts appeared roughly pattern <br /> RPRJ cDNA                                    1.36                        1.70     observed cDNA monomer insert (Fig. 6C) RPRI cDNA dimer                              2.60                        5.80a    3&apos; signal corresponding small <br />                                                                                   RPRJ RNA.  likely interpretation   3.2 h approximately generations temperature shift, followed   pattern dimer clone RNAs Fig. 6A transcripts 5.8 h.                                                                         initiating normal position upstream cDNA <br />                                                                                   copy. heterogeneous pattern long RNAs arises                                                                                   multiple terminations past second copy replace 369-nucleotide major RPRJ RNA coding region                           termination correct site dimer insert addition EcoRI linkers. use plasmids                     followed aberrant processing. case, little RNA transform heterozygous RPRI-disrupted diploid, spores                        normal size growth-defective strain obtained dissected determine complementation.                         30 38&#194;&#176;C. mature RPRI cDNA served control defects                             using RNA samples Fig. 6, steady- linker insertions YCpSO YEp24 showed                         state levels mature precursor tRNA examined <br /> growth defects.  heterologous cDNAs                           probing duplicate Northern blots antisense RNA E. coli, human cells, yeast mitochondria gave                         probe tRNALeU3. tRNA synthesized precursor <br /> detectable complementation. clear time                          10-nucleotide leader, 32-nucleotide intron, 6 finding defects transcription, RNA                      7 nucleotides 3&apos; trailing sequence (44, 61). terminal <br /> processing, assembly transport, function.                          sequences normally removed intron (61), variants RPRJ gene did complement                             vitro removal 3&apos; sequences yeast lysates quite <br /> shown) serendipitous insertion RPRJ cDNA                            slow 5&apos; leader removed RNase P <br /> direct tandem repeat tested. single copy,                         (unpublished observations). partial processing products <br /> struct did complement, diploids carrying                         Fig. 7, assigned direct size comparison sequenced dimer multicopy plasmid gave normal spores                               vitro products  shown), pattern (nondisrupted chromosomal RPRI copies) slowly                             followed parental strain disrupted strain <br /> growing spores sporulated. Growth curves                             carrying single cDNA insert, somewhat pri- <br /> normal defective spores (Table 1) differ-                       mary transcript accumulating higher temperature ence doubling twofold 30&#194;&#176;C increased                         Ordered removal 5&apos; end, 3&apos; end, threefold 38&#194;&#176;C.                                                           intron implied observed intermediates (61). <br />    characterize defect caused direct cDNA                           contrast, strain RPRI complemented repeat RPRI dimer, expression RPRI RNA                                cDNA dimer accumulated near-primary-tran- <br /> tRNA examined 30 38&#194;&#176;C. Figure 6 shows North-                            script-size RNA, little intermediate appearing ern blot analysis 5&apos; 3&apos; end analysis RPRI                       contains intron lacks 5&apos; 3&apos; termini. End <br /> parent strain, disrupted RPRI haploid strain comple-                            processing defective RPRJ-defective strain. <br /> mented multiple copies single RPRI cDNA                                38&#194;&#176;C strain accumulated intermediate construct (Fig. 5), disrupted haploid complemented                           intron appears removed, multiple copies cDNA dimer construct.                              5&apos; leader 3 4 nucleotides 3&apos; trailing sequence. <br /> wild-type RNA displayed normal pattern 369-nucleo-                          Smearing primary transcript apparently <br /> tide RNA, lesser amounts larger RNA containing                         spliced intermediate downward probably reflects partial 3&apos; <br /> longer 5&apos; 3&apos; termini. cDNA monomer clone                                exonucleolytic processing slowed presence 5&apos; <br /> multiple copies accumulated considerably larger                        leader (12). RNA slightly smaller mature-size <br /> RNA somewhat smaller 369-nucleotide                                tRNALeU, appeared 38&#194;&#176;C samples, RNA correct 5&apos; 3&apos; termini. longer RNA                       known lacks portions 3&apos; CCA fact precursor 369-nucleotide form, appears                    deficiency. worth noting spliced prod- correctly processed 5&apos; 3&apos; ends                      ucts containing terminal sequences normally regard exact sequence flanking mature                          accumulate, observed appearance RNA size termini. Accumulation presumptive precursor                           spliced RNase P-cut intermediates RPRI- <br /> indicate step processing unable                     defective strain, indicating normal ordering prob- increased synthesis, overloading                       ably obligatory vivo. occurrence tRNA splicing <br /> assembly processing machinery slowing                           end maturation recently observed rate limiting step(s). 5&apos; termini larger                  (45). <br /> RNAs discrete extend wild type                                  size analyses tRNA suggest pre- <br /> approximately size added linker (Fig. 6B),                          cursors accumulate, purified total RNA defective <br />  multiple longer products seen cDNA Northern analy-                         strain grown 38&#194;&#176;C digested isolated RNase P (37) <br />  ses probably extended 3&apos; ends, consistent                     blotted samples shown Fig. 7. <br /> longer products seen S1 3&apos; analyses. known                      accumulated tRNALeU3 primary transcript  longer products arise inefficient tran-                         verted size intron-containing precursor slight <br />  scription termination inappropriately processed longer                         size heterogeneity consistent variable amounts  <br />  transcripts stable seen                        maining 3&apos; trailer. Identical size reductions observed <br />  wild-type gene.                                                                   lesser amounts primary transcripts accumulated    RPRI cDNA dimer construct transcribed                           wild-type RPRI cDNA monomer strains  large percentage accurate 5&apos; termini longer                       shown). spliced intermediate  <br />  VOL . 1 l, 1991                                                           RPRI ENCODES S. CEREVISIAE NUCLEAR RNase P RNA                                             727 <br />  <br />  <br />                                       30&#194;&#176;C                38&#194;&#176;C             B          300C           380C           C                     30&#194;&#176;C             380C <br />                                            -                                                                                                     1&quot;            <br />                                                                                                  0               0 <br />                                                                                                                                                    0                 0 <br />                                                0)                 0                              E               E                                 E                 E <br />                                              E                    E                                                                                X                 C <br />                                          4c .:               4: .:                          Z        Z <br />  <br />                                     2Za                 @ Za                           &gt;-                                              X, <br />                                                                                                                                        3          <br />                                                                                                                                                     <br />                                                                                                                                                    T <br />                                                                                                                                                                                                                                                                                                                             rr                                                                                                                                                                        T- <br />                                                                                             1- w-   1~ <br />                                                                                                        <br />                                                                                                                 T-. <br />                                                                                                                                         = <br />                                                                                                                                             v           .v    m <br />                                                                                                                                                         = <br />                                                                                         =0. 0.             0.0.M <br />                                     = <br />                                     3   0.            =    0. <br />                                                              ccX                       r   cr              Er   E <br />                                                                                                                       nt <br />                                                                                                                       397-399      [ <br />                                                                   w <br />  <br />                                                                                                                       385388so *4 <br />                         nt               &lt; <br />  <br />                       540    - <br />  <br />                                                                                                                           369 -- <br />                                                                            nt <br />                       450    - <br />  <br />  <br />  <br />                                                                            -84    -.        _              _ <br />  <br />                   369   nt <br />                major -             -     *                  1 <br />              RPR1 RNA <br />  <br />  <br />  <br />  <br />                                                                             .1- <br />                                                                                                        <br />  <br />  <br />  <br />    FIG. 6. RPRI expression monomer tandem dimer cDNA clones. Northern end analyses performed RNA isolated parental haploid strain (&apos;Wild Type&apos;), W3031A, disrupted haploids complemented monomer tandem dimer <br /> RPRI cDNA insert RPRI flanking regions described legend Fig. 4 text. Analyses performed described legend Fig. 2 text total cellular RNA cultures grown 30 38&#194;&#176;C.   Northern analysis using antisense <br /> RPRI RNA. positions known 369-nucleotide (nt) RNA longer markers indicated. (B) 5&apos; end analysis primer extension. cDNA construct termini corresponding 369-nucleotide RNA wild-type cells, RNAs initiating upstream longer 5 bp corresponding added EcoRI linker. (C) 3&apos; end analysis S1 nuclease protection. major 3&apos; termini discussed text. Positions numbered according Fig. 1B. <br />  <br />  <br />  <br /> creased, presumably conversion mature-size                                   14, 24, 32, 34, 35, 37, 38, 40, 42). sequence similarities <br /> tRNA removal 5&apos; terminus.  major                                        nuclear RNase P RNAs potential <br /> accumulated pre-tRNAs RPRI-defective strain                                 structural similarities analogous procaryotic ri- cleaved RNase P,                                 bozymes (5, 37) strongly suggested identification appropriate substrates enzyme.                                                  RNAs RNase P components correct, formal <br />                                                                                         proof lacking. Unlike procaryotic ribozymes, <br />                                  DISCUSSION                                             eucaryotic RNAs shown catalytic <br />                                                                                         activity purified synthesized vitro,   RNAs copurify RNase P activity                                sequence-specific attacks RNase H shown isolated number eucaryotic sources (1, 7, 9, 11, 12,                          destroy RNase P activity. enzyme purified <br />  728     LEE ET AL.                                                                                                               MOL. CELL. BIOL. <br />  <br />              0                                                                    previous biochemical assignment assign- <br />              0 <br />              C&apos;) <br />                                        <br />                                       C&apos;)                                           ment analogous eucaryotic RNase P-associated <br />              jn <br />                                                                                                                                      RNAs  <br />                                                                                        Analysis RPRI transcription unit preliminary. <br />                                                    00                               main RNAs appear wild-type cells, predominant <br />                      EEE                                                            369-nucleotide form (37) longer RNA containing <br />                                                                                     extra 84 nucleotides 5&apos; end 16 30 extra <br />                           z zB                                                residues 3&apos; end. determined <br />                Ct       -               LD                                      RNAs primary transcript                                                                                     precursor-product relationship                               ca:C:                                               longer smaller forms. reasons,  <br />                                                                                     speculate longest observed RNA                                                                                     primary transcript RPRJ 369-nucleotide form <br />        wool                                              o-    primary transcript <br />                                                                (+5&apos; +3.+IVS) <br />                                                                                     results processing steps.  RNAs <br />                                                                                     derive single-copy gene, use <br />                                                                                     probes detectable genes similar <br />                                                         -o*    +3&apos;,+IVS             sequence S. cerevisiae. Second, RNAs                          &quot;_01-1             -111 <br />                                                         -.,-   +IVS                 appear single RNase P holoenzyme.                                                                                     copurify stages <span id='am-1' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-4' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="ion" datatype="xsd:string"></span>ion</span>-exchange chro- <br />                                                                                     matography, suggesting similar protein complements,                                                                                     precisely comigrate velocity sedimentation (37). <br />                                                                                      longer, shorter, sequence  <br />                                        0                --- <br />                                                                +5,+3 <br />                                                                                     tributes RNA polymerase III transcription unit.                                                                                     gene encoding analogous human RNA shown                                                                                     transcribed RNA polymerase III vitro (4),                                                                                     preliminary work laboratory suggests RPRI                                                         -o     mature     tRNALeU <br />                                                                               3     transcribed RNA polymerase III vitro vivo <br />                                                                                     basis amanitin tagetin sensitivities    FIG. 7. Accumulation tRNA precursors RPRI-defective                     effects mutations expression (unpublished data). RPRI <br /> strain. Northern analysis tRNALCU3 RNAs performed                         contains box B box internal promoters typical described Materials Methods, using total cellular RNA                   tRNA-like yeast U6 RNA gene promoters (8). experiment shown Fig. 6. known sizes sequences                    best B box sequences marked Fig. 1 normal precursors (61) indicated right                      80% similarity consensus GGTTCGANTCC (16). <br /> confirmed electrophoresis radiolabeled tRNALCU3 gene tran-                    sequences considerably farther downstream scription processing products prepared vitro  shown, 13,                 normal tRNA promoters, distance 61). +5&apos;, +3&apos;, +IVS (intervening sequence) refer portions                 inconsistent functional B box far downstream primary transcript remaining indicated precursor. far                yeast U6 transcription initiation site (8). best box <br /> right-hand lane shows products obtained sample purified 38&#194;&#176;C RPR1 cDNA dimer construct strain RNA cleaved                      match marked 83% similarity S. cerevisiae nuclear RNase P vitro described Materials               TPuGCNNAGPyNGG consensus (16), perfect matches Methods.                                                                        invariant residues. position sequence 20 <br />                                                                                     residues downstream longer RNA 5&apos; end correct                                                                                     positions boxes relative transcription initiation sites <br />                                                                                     (13). sequences absolute homogeneity, remained possible                        box similarity RPRJ, correctly positioned identified RNAs copurifying contaminants.                               5&apos; end major 369-nucleotide RNA address problem, characterized RPRJ gene                           transcription start. observed 3&apos; ends appear heteroge- yeast nuclear RNase P-associated RNA used                             neous represent multiple processing events, defective variant determine RNase P function                        longest extend T6 region downstream impaired.                                                                           appropriate RNA polymerase III terminator. rea- <br />    Disruption single RPRI gene chromosome V                            sonable hypothesis RPRI RNA production lethal, number RPRI variants tested                         synthesis precursor (indicated underlining Fig. 1) <br /> complementation gave defective growth phenotype short                             followed processing 5&apos; 3&apos; ends complete lethality. variant, tandem cDNA dimer                             369-nucleotide mature RNA. <br /> inserted RPRI flanking regions, little                                disruption chromosomal RPRI gene opens correct-size RPRI RNA accumulated tRNA precursors                                possibility examining effects incorporating variant undergone cleavage RNase P. formally                           RNase P RNAs holoenzyme vivo, circumventing <br /> possible defect RPRI actually RNase                         inactivity RNA inability recon- <br /> P subunit causes accumulation indirectly                             stitute significant activity separated RNA protein undetermined interference stage                           components repeated experiments. studies  <br /> tRNA RNase P RNA maturation. unlikely,                                 scribed  attempted substitute heterolo-  considering   RPRJ RNA cofractionates                           gous RNase P cDNAs mature RPRJ cDNA highly purified S. cerevisiae RNase P (37), (ii) similar RNAs                        investigate function yeast holo- <br /> copurify different methods RNase Ps                               enzyme, demonstrated previously reconsti- eucaryotes, (iii) RNase P cleavage appears                         tuted heterologous components. Unfortunately, early step tRNA maturation. genetic evidence                             attempts lethal recessive, unambig- <br /> presented represents essential validation                       uously determined defect processing, <br />  VOL.   11, 1991                                            RPRI ENCODES S. CEREVISIAE NUCLEAR RNase P RNA                               729 <br />  <br /> assembly, stability, activity, transcription (e.g.,            17. Gold, H. , S. Altman. 1986. Reconstitution RNase P <br /> lack internal promoters). expected,                     activity using inactive subunits E. coli HeLa cells. Cell <br /> reconstitution RNase P holoenzyme vivo, future                    44:243-249. <br /> studies evaluating heterologous RNAs mutations               18. Guemer-Takada, C., S. Altman. 1984. Catalytic activity RPRJ facilitated RPRI-disrupted -defec-                   RNA molecule prepared transcription vitro. Science <br /> tive strains.                                                                223:285-286. <br />                                                                          19. Guerrier-Takada, C., S. Altman. 1984. Structure solution <br />                                                                              Ml RNA, catalytic subunit ribonuclease P                        ACKNOWLEDGMENTS                                       Escherichia coli. Biochemistry 23:6327-6334. <br />                                                                          20. Guerrier-Takada, C., S. Altman. 1986. Ml RNA large <br />    thank M. Hollingsworth, C. Wise, N. Martin, M. Baer, S.                terminal deletions retains catalytic activity. Cell 45:177-183. <br /> Altman, D. Waugh, N. Pace gifts plasmids N. Pace,         21. Guerrier-Takada, C., K. Gardiner, T. Marsh, N. Pace, S.  Forster, S. Altman helpful discussions. thank W.          Altman. 1983. RNA moiety ribonuclease P cata- <br /> Studier T7 RNA polymerase clone, S. Tabor T7 DNA             lytic subunit enzyme. Cell 35:849-857. <br /> polymerase clone, E. Esposito, E. Jones, P. Hieter yeast         22. Guerrier-Takada, C., K. Haydock,  Leland, S. Altman. <br /> strains, D. Thiele plasmids yeast genomic DNA                1986. Metal ion requirement aspects reaction <br /> library. deeply indebted L. Riles M. Olson                 catalyzed Ml RNA, RNA subunit ribonuclease P mapping gene.                                                            Escherichia coli. Biochemistry 25:1509-1515. <br />    work supported National Institutes Health grant        23. Hanning, E. M., D. J. Thiele, M. J. Leibowitz. 1984. <br /> GM34869 National Science Foundation grant DMB-8901559                 Saccharomyces cerevisiae killer virus transcripts contain tem- <br /> D.R.E.                                                                       plate-coded polyadenylate tracts. Mol. Cell. Biol. 4:101-109. <br />                                                                          24. Hollingsworth, M., N. C. Martin. 1986. RNase P activity                             REFERENCES                                       mitochondria Saccharomyces cerevisiae depends  1. Akaboshi, E., C. Guerrier-Takada, S. Altman. 1980. Veal              mitochondrion nucleus-encoded components. Mol. Cell. <br />       heart ribonuclease P essential RNA component. Bio-              Biol. 6:1058-1064. <br />       chem. Biophys. Res. Commun. %:831-837.                             25. Huibregtse, J. M., D. R. Engelke. 1986. Direct identification <br />  2.   Altman, S., C. Guerrier-Takada. 1986. Ml RNA, RNA              small sequences changes chromosomal DNA. Gene 44: <br />       subunit Escherichia coli ribonuclease P, undergo              151-158. <br />       pH-sensitive conformational change. Biochemistry 25:1205-          26. Huibregtse, J. M., D. R. Engelke. 1990. Sequence       1208.                                                                  footprint analysis yeast DNA primer extension. Methods <br />  3.   Ausubel, F. M., R. Brent, R. E. Kington, D. D. Moore, J. G.            Enzymol. 194:550-562. <br />       Seidman, J.  Smith, K. Struhl (ed.). 1987. Current proto-    27. Ikemura, T., Y. Shimura, H. Sakano, H. Ozeki. 1975. <br />       cols molecular biology. John Wiley &amp; Sons, , New York.          Precursor molecules Escherichia coli transfer RNAs accumu- <br />  4.   Baer, M., T. W. Nilsen, C. Costigan, S. Altman. 1990.              lated temperature-sensitive mutant. J. Mol. Biol. 96:69-86. <br />       Structure transcription human gene Hi RNA:        28. Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor- <br />       RNA component human RNase P. Nucleic Acids Res.                     mation intact yeast cells treated alkali cations. J. <br />       18:97-101.                                                             Bacteriol. 153:163-168. <br />  5.   Bartkiewicz, M., H. Gold, S. Altman. 1989. Identification   29. Jain, S. K., M. Gurevitz, D. Apirion. 1982. small RNA <br />       RNA molecule copurifies RNase P activity             complements mutants RNA processing enzyme <br />       HeLa cells. Genes Dev. 3:488-499.                                      ribonuclease P. J. Mol. Biol. 162:515-533. <br />  6.   Boehm, S. 1987. Similarities predicted secondary         30. James, B. D., G. J. Olsen, J. Liu, N. R. Pace. 1988.       structure Ml RNA ribozyme tRNA binding                 secondary structure ribonuclease P RNA, catalytic ele- <br />       center 16S rRNA E. coli. FEBS Lett. 220:283-287.               ment ribonucleoprotein enzyme. Cell 52:19-26. <br />  7.   Bowman, E. J., S. Altman. 1980. Identification ribonu-      31. Jones, E. W. 1977. Proteinase mutants Saccharomyces cere- <br />       clease P activity chick embryos. Biochim. Biophys. Acta           visiae. Genetics 85:23-33. <br />       613:439-447.                                                       32. Kline, R., S. Nishikawa, D. Soll. 1981. Partial purification  8. Brow, D. , C. Guthrie. 1990. Transcription yeast U6           RNase P Schizosaccharomyces pombe. J. Biol. Chem. <br />     snRNA gene requires pol III promoter element novel                256:5058-5063. <br />     position. Genes Dev. 4:1345-1356.                                    33. Kole, R., M. F. Baer, B. C. Stark, S. Altman. 1982. <br />  9. Castano, J. G., R. Ornberg, J. G. Koster, J.  Tobian, M.           Escherichia coli RNase P required RNA component     Zasloff. 1986. Eukaryotic pre-tRNA 5&apos; processing nuclease:               vivo. Cell 19:881-887. <br />     copurification complex cylindrical particle. Cell 46:377-     34. Koski, R.,  Bothwell, S. Altman. 1976. Identification     387.                                                                     ribonuclease P-like activity human KB cell. Cell 9:101- <br /> 10. Davanloo, P.,  H. Rosenberg, J. J. Dunn, W. Studier.               106. <br />     1984. Cloning expression gene bacteriophage T7        35. Krupp, G., B. Cheravil, D. Frendewey, S. Nishikawa, D. <br />     RNA polymerase. Proc. Natl. Acad. Sci. USA 81:2035-2039.                 Soll. 1986. RNA species copurify RNase P 11. Doersen, C., C. Guerrier-Takada, S. Altman, G. Attardi.              fission yeast Schizosaccharomyces pombe. EMBO J. 5:1697- <br />       1985. Characterization RNase P activity HeLa cell           1703. <br />       mitochondria. J. Biol. Chem. 260:5942-5949.                        36. Lawrence, N. P., S. Altman. 1986. Site-directed mutagenesis <br /> 12.   Engelke, D. R., P. Gegenheimer, J. Abelson. 1985. Nucleo-          Ml RNA, RNA subunit Escherichia coli ribonuclease <br />       lytic processing tRNAAr-tRNAASP dimeric precursor            P. J. Mol. Biol. 191:163-175. <br />       homologous component Saccharomyces cerevisiae. J.             37. Lee, J.-Y., D. R. Engelke. 1989. Partial characterization       Biol. Chem. 260:1271-1279.                                             RNA component copurifies Saccharomyces cere- <br /> 13.   Evans, C. F., D. R. Engelke. 1990. Yeast extracts tRNA         visiae RNase P. Mol. Cell. Biol. 9:1058-1064. <br />       gene transcription processing. Methods Enzymol. 181:439-       38. Leontis, N.,  DaLio, M. Strobel, D. Engelke. 1988. Effects <br />       450.                                                                   tRNA-intron structure cleavage precursor tRNAs 14.   Garber, R. L., S. Altman. 1979. vitro processing B.          RNase P Saccharomyces cerevisiae. Nucleic Acids Res. <br />       mori transfer RNA precursor molecules. Cell 17:389-397.                16:2537-2552. <br /> 15.   Gardiner, K., N. R. Pace. 1980. RNase P Bacillus subtilis   39. Ma, H., S. Kunes, P. J. Schatz, D. Botstein. 1987. Plasmid <br />       RNA component. J. Biol. Chem. 255:7507-7509.                    construction homologous recombination yeast. Gene <br /> 16.   Geiduschek, E. P., G. P. Tocchini-Valentini. 1988. Tran-           58:201-216. <br />       scription RNA polymerase III. Annu. Rev. Biochem. 57:           40. Manam, S., G. C. Van Tuyle. 1987. Separation charac- <br />       873-914.                                                               terization 5&apos;- 3&apos;-tRNA processing nuclease rat liver <br />  730     LEE ET AL.                                                                                                         MOL. CELL. BIOL. <br />  <br />     mitochondria. J. Biol. Chem. 262:10272-10279.                        52. Reich, C., K. J. Gardiner, G. J. Olsen, B. Pace, T. L. Marsh, <br /> 41. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular            N. R. Pace. 1986. RNA component Bacillus <br />     cloning: laboratory manual. Cold Spring Harbor Laboratory,             subtilis RNase P. Sequence, activity partial secondary <br />     Cold Spring Harbor, N.Y.                                                 structure. J. Biol. Chem. 261:7888-7893. <br /> 42. Mller, D. L., N. C. Martin. 1983. Characterization          52a.Riles, L., M. Olson. Personal communication. <br />     yeast mitochondrial locus necessary tRNA biosynthesis:           53. Rose, M., P. Grisafi, D. Botstein. 1984. Structure     DNA sequence analysis identification new transcript.            function yeast URA3 gene: expression Escherichia <br />     Cell 34:911-917.                                                         coli. Gene 29:113-124. <br /> 43. Motamedi, H., K. Lee, L. Nichols, F. J. Schmidt. 1982.        54. Rose, M. D., P. Novick, J. H. Thomas, D. Botstein, G. Fink. <br />     RNA species involved Escherichia coli ribonuclease P activ-            1987. Saccharomyces cerevisiae genomic plasmid bank based <br />     ity. J. Mol. Biol. 162:535-550.                                          centromere-containing shuttle vector. Gene 60:237-243. <br /> 44. Newman,  J., R. C. Ogden, J. Abelson. 1983. tRNA gene          55. Rothstein, R. J. 1982. step disruption yeast. Methods <br />     transcription yeast: effects specified base substitutions        Enzymol. 101:202-211. <br />     intragenic promoter. Cell 35:117-125.                            56. Saiki, R. K., D. H. Gelfand, S. Stoffel, S. J. Scharf, R. Higuchi, <br /> 45. O&apos;Connor, J. P., C. L. Peebles. 1991. vivo pre-tRNA               G. T. Horn, K. B. Mullis, H.  Erlich. 1988. Primer- <br />     processing Saccharomyces cerevisiae. Mol. Cell. Biol. 11:             directed enzymatic amplification DNA thermostable <br />     425-439.                                                                  DNA polymerase. Science 239:487-491. <br /> 46. Piper, P. E., J <br /> </body></html>