<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Silva et al. BMC Structural Biology 2011, 11:40 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                             Open Access <br />  <br /> Central domain deletions affect SAXS solution <br /> structure function Yeast Hsp40 proteins <br /> Sis1 Ydj1 <br /> Julio C Silva1,2,6&#226;&#8364;&#160;, Julio C Borges3&#226;&#8364;&#160;, Douglas M Cyr4, Carlos HI Ramos5 Iris L Torriani1,2* <br />  <br />  <br />   Abstract <br />   Background: Ydj1 Sis1 structurally functionally distinct Hsp40 proteins yeast cytosol. Sis1   essential gene ydj1 gene essential growth elevated temperatures complement sis1 <br />   gene deletion. Truncated polypeptides capable complementing sis1 gene deletion comprise J-domain   Sis1 Ydj1 connected G/F region Sis1  Ydj1). Sis1 mutants G/F deleted <br />   G/M maintained capable complementing sis1 gene deletion. <br />   Results: investigate relevance central domains structure function Ydj1 Sis1   prepared Sis1 constructs deleting specific domains. mutants decreased affinity heated luciferase   equally capable stimulating ATPase activity Hsp70. Detailed low resolution structures obtained   overall flexibility Hsp40 mutants assessed using SAXS methods. Deletion G/M   G/M plus CTDI domains little impact quaternary structure Sis1 analyzed SAXS technique. <br />    deletion ZFLR-CTDI changed relative position J-domains Ydj1 way   ended resembling Sis1. results revealed G/F G/M regions flexible <br />   domains. model structures exhibit common clamp-like conformation. <br />   Conclusions: results suggest central domains, previously appointed important features   substrate binding, relevant keeping J-domains specific relative positions. clamp-like <br />   architecture observed favorable interactions Hsp40 Hsp70. <br />  <br />  <br /> Background                                                                          families Heat Shock Proteins (HSPs), family acts <br /> Molecular chaperones proteins involved                              assist protein folding different way. <br /> assisting folding assembly newly synthesized                               important chaperone family 40-kDa Heat <br /> proteins recognizing non-native substrate proteins pre-                             shock protein (Hsp40). Chaperones Hsp40/ <br /> dominantly exposed hydrophobic residues [1].                              DnaJ family play important roles cells working  conditions successful folding                               molecular chaperone Hsp70 promote <br /> vivo favorable. cellular environment                             protein folding, assembly, translocation degradation crowded protein denaturation aggrega-                               [2-5]. Hsp40 proteins interact hydrophobic <br /> tion major problems.  need                            chains non-native polypeptides preventing aggre- <br /> chaperones protect cells elevated tem-                               gation [6,7]. Hsp40 form transient complexes <br /> perature cellular stress situations, achieve                            Hsp70 presenting non-native polypeptides sub- <br /> successful folding proteins vivo.                           sequent protein folding [8-10]. members                                                                                     Hsp40 family typically contain J-domain, regu- <br />                                                                                     lates <span id='am-17' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>-dependent binding peptides Hsp70 <br /> * Correspondence: torriani@ifi.unicamp.br <br />                                                                                     [4,11,12]. <br /> &#226;&#8364;&#160; Contributed equally                                                                 Members Hsp40 family act molecular cha- <br /> 1 <br />  Department Condensed Matter Physics, &#226;&#8364;&#339;Gleb Wataghin&#226;&#8364;? Physics Institute,         perones bind deliver non-native proteins State University Campinas (UNICAMP), Campinas, SP 13083-859, Brazil list author information available end article <br />                                                                                     Hsp70 divided main groups,  <br />                                        &#194;&#169; 2011 Silva et al; licensee BioMed Central  Open Access article distributed terms Creative Commons <br />                                        Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution, reproduction                                        medium, provided original work properly cited. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                      Page 2 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />   Type Type II studied.                 High-resolution structural studies fragments types functionally equivalent [13-15]             Sis1 Type II Hsp40s indicated pro- exhibit major differences chaperone activity [16].         teins function homodimers clamp-like types J-domains connected central          architecture use shallow groove located C-terminal domains G/F-rich (<span id='am-47' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-48' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-54' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>Glycine</span>/pheny-             surface monomers bind non-native proteins <br /> lalanine-rich) linker (Figure 1).                                 [20,21]. addition, G/F-rich regions Ydj1   Type Hsp40s, Escherichia coli DnaJ, yeast             Sis1 lie adjacent putative polypeptide binding <br /> Ydj1 human Hdj2, contain zinc-finger-like linker            domains specify functions region (ZFLR) G/F domain C-ter-               Hsp40s [22-24]. structure/function studies <br /> minus (Figure 1B), Type II Hsp40 proteins             hydrodynamic analysis showed yeast Sis1 human Hdj1 contain G/M-rich (<span id='am-35' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-36' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Glycine" datatype="xsd:string"></span>Gly</span>-                types Hsp40 proteins form dimers solution <br /> cine/Methionine-rich) domain linker region (Fig-           [6,18,19]. Dimer formation plays critical role ure 1A) [14-17]. types substrate-binding site         Hsp40s chaperone activity disruption located C-terminal domain, divided           dimerization motifs results severe defects cha- <br /> subdomains (CTDI) II (CTDII) (Figure 1).                    perone functions [20,25]. <br />   reason Type Type II Hsp40s exhibit                  Previous results showed human yeast Type differences chaperone activity unknown. bio-           Type II Hsp40s, distinct quaternary structure <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> structural studies provided             [18,19]. results raised hypothesis cen- insight question, suggesting              tral domains controlled quaternary structure answer resides structural differences [15,18,19].        types Hsp40s, chimeric mutants,  need comprehensive study                central domains Ydj1 (ZFLR) Sis1 (G/M) <br /> structure considerable importance.                          switched, properties exchanged.  <br />  <br />  <br />  <br />  Figure 1 Domain organization Hsp40s mutants.   Schematic representation Sis1 domains known high-resolution <br />  structures. (B) Schematic representation Ydj1 domains high-resolution structures. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                               Page 3 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> chimeric Ydj1 ZFLR switched                   G/M region deleted, Sis1_&#206;&#8221;121- G/M Sis1 proved functionally struc-                 257, G/M CTDI turally similar Type II Sis1. Correspondingly, chi-               deleted (Figure 1A). Ydj1 yeast Type Hsp40 meric Sis1 G/M switched                   contains 409 residues arranged highly  <br /> ZFLR Ydj1 proved functionally structu-                  served helical N-terminal J-domain, disordered mid- <br /> rally similar Type Ydj1 [15,19].                                  dle region  <span id='am-59' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-60' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-63' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-64' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span>glycine</span>/phenylalanine (G/F)), Zinc finger <br />   increase knowledge role central                 domain (ZFLR Cys-rich domain) C-term- <br /> domains structure/function relationship                      inal sub-domains (CTDI CTDII) shown Figure <br /> defined biophysical functional features Sis1                1B.  understand role regions Ydj1 mutants deleted central domains                     protein structure, studied Ydj1 deletion specific Type Type II Hsp40s. stu-                 mutant named Ydj1_&#206;&#8221; 106-255 , dies entailed functional analysis mutated Hsp40s                  Zinc finger-like region (ZFLR) CTDI coupled biophysical investigation quaternary               deleted (Figure 1B). mutants purified structure dynamic light scattering, analytical ultra-               apparent contamination (&gt;95% pure; Figure 2A) centrifugation small-angle X-ray scattering (SAXS).                maintained 4&#194;&#176;C avoid degradation. folded  refined solution structure Sis1 obtained                 formation proteins investigated circular <br /> using improved SAXS data. outcome stu-                    dichroism (CD) spectroscopy (Figure 2B). previously <br /> dies showed deletion G/M G/M                 shown wild-type proteins Sis1 Ydj1 [19], plus CTDI domain regions minor impact                       mutants CD spectra folded proteins overall quaternary structure Sis1. Consequently,                minima 208 220 nm positive peak <br /> results suggest central domains important                 bellow 200 nm (Figure 2B). shapes spectra substrate binding maintenance J-domains                 indicated large unfolded portion present. specific relative positions.                                  good agreement results hydro- <br />                                                                        dynamic measurements  . <br /> Results Discussion <br /> Protein samples                                                        Hsp40 function <br /> Sis1 yeast member Type II Hsp40s family                Hsp40s act binding unfolded partially unfolded <br /> contains 352 residues arranged highly conserved                 protein (client protein) delivering Hsp70  <br /> helical N-terminal J-domain, disordered middle region                comitantly stimulating Hsp70 ATPase activity. ATP <br /> (divided glycine/phenylalanine (G/F) glycine/                 hydrolysis Hsp70 crucial step protein folding <br /> methionine (G/M) rich regions) C-terminal                      assisted chaperone. effect deletions <br /> sub-domains (CTDI CTDII) shown Figure 1A.                    function Hsp40s assayed testing order understand role central regions              ability bind client protein stimula- structure function Type II Hsp40s, Sis1                 tory effect ATPase activity Hsp70 (Figure 3). <br /> deletion mutants produced: Sis1_&#206;&#8221; 124-174 ,                   ability bind heated luciferase  client <br />  <br />  <br />  <br />  <br />  Figure 2 Results protein expression:   SDS-PAGE 12% showing recombinant mutant proteins &gt; 95% pure. Molecular <br />  masses standard proteins shown left. (B) CD spectra recombinant proteins (10-40 &#206;&#188;M) measured 195 260 nm 25 <br />  mM Tris-HCl (pH 7.5) containing 500 mM NaCl shown molar residual ellipticity ([&#206;&#732;]). <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                       Page 4 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />  Figure 3 Functional tests:   Hsp40s ability bind client proteins 40%. (B) Hsp40s ability stimulate ATPase activity Hsp70 <br />  90% compared Sis1.  text details). <br />  <br />  <br /> protein) tested Sis1 binding set stan-                  s 0 20,w induced external factors (pH changes, salt <br /> dard (100%). Compared Sis1, efficacy                             strength, ligands temperature) related conforma- <br /> Sis1_&#206;&#8221;124-174, 60%, Sis1_&#206;&#8221;121-257                    tional changes [26]. values s 0 20,w D 0 20,w , Ydj1_&#206;&#8221;106-255 40% (Figure 3A). Sec-                      obtained dynamic light scattering (DLS) experi- <br /> ond, mutants assayed regarding ability                   ments, shown Table 1. M values obtained <br /> stimulate ATP hydrolysis Ssa1 (Hsp70) Sis1                      c(M) distribution s/D ratio <br /> stimulatory effect set standard (100%). Sis1_&#206;&#8221;124-                  (Equation 4), weight average factor &#198;&#8217;/&#198;&#8217;0 174 Ydj1_&#206;&#8221;106-255 effect similar Sis1,                  shown (Table 1). results suggested Sis1_&#206;&#8221;124- <br /> inside error, effect Sis1_&#206;&#8221;121-257                   174 , Sis1_&#206;&#8221; 121-257 Ydj1_&#206;&#8221; 106-255 dimers 90% (Figure 3B). comparison, perfor-                         solution asymmetric elongated <br /> mance Ydj1 experiments similar                      shape shown previously Hsp40 proteins [18,19]. Sis1, experimental error (data shown). results deletions decreased                   SAXS results <br /> 50% ability bind client proteins,                      explore impact selective central domains dele- effect ability J-domain interact-                 tion Hsp40s quaternary structure, SAXS analysis ing Hsp70. results suggested J-                        performed. corrected normalized experimental <br /> domains maintained proper conformation                           SAXS curves proteins Sis1, Sis1_&#206;&#8221; 124-174 , <br /> functionality.                                                             Sis1_&#206;&#8221;121-257, Ydj1_&#206;&#8221;106-255 q range 0.01 &lt; q <br />                                                                            &lt; 0.25&#195;&#8230;-1 displayed Figure 5 , Hydrodynamics                                                              respective regularization fitting (solid lines) resulting <br /> Sis1 Ydj1 dimers solution used                      p(r) function calculation using GNOM. analytical ultracentrifugation (AUC) investigate                    corresponding p(r) functions, shown Figure 5(B), indi- <br /> oligomeric status deleted mutants. performed                     cated slightly elongated shape proteins AUC sedimentation velocity (SV) experiments fitted                     solution, confirming AUC results. Kratky plots data using SedFit supplied continuous sedimen-                    presented Figure 5(C) showed proteins tation distribution c(S) (Figure 4A). maximum                     quite flexible solution, gave clue peaks c(S) curve, apparent s,                  difficulty crystallizing  inspection corrected standard conditions (s 20,w ) plotted                     Porod plots Figure 5(D) shows curves protein concentration (Figure 4B). extrapo-                    contain significant plateau region indicating lation s20,w 0 mg/mL gave s020,w intrin-                 defined particle volumes, plots Sis1 sic property protein contains information                       Sis1_&#206;&#8221;124-174 present fairly flat region q4-range molecular mass (M) asymmetry                        0.0012&lt;q4&lt;0.0025&#195;&#8230;-4, suggesting compact  molecule. Normally, variation value                     formation proteins. complete <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                    Page 5 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />                                                                                    deviation Porod regime leads predict                                                                                    flexible structures Sis1_&#206;&#8221;121-257, Ydj1_&#206;&#8221;106- <br />                                                                                    255 proteins. degree flexibility individual <br />                                                                                    protein confirmed plots. inset                                                                                    Figure 5  displays ln q)) vs. q2 plot                                                                                    validity region Guinier approximation (qR g&lt;1) <br />                                                                                    corresponding linear regression                                                                                    protein. linearity plots confirmed                                                                                    monodispersity samples. maximum dimen- <br />                                                                                    sion (Dmax) values, radii gyration (Rg) obtained                                                                                    Guinier approximation p(r) functions,                                                                                    calculated estimated M plus oligo- <br />                                                                                    merization states proteins presented                                                                                    Table 2. Rg values obtained approaches <br />                                                                                    close agreement. reliable Rg value                                                                                    molecule obtained p(r) function, <br />                                                                                    derived complete experimental curve.                                                                                    described Material Methods, bovine serum <br />                                                                                    albumin (BSA) solution used reference sample <br />                                                                                    M estimation using SAXS data. results <br />                                                                                    indicated proteins exist dimeric state                                                                                    solution, values obtained approximately <br />                                                                                    twice values calculated primary sequence <br />                                                                                    (Table 2). <br />  <br />                                                                                    Ab initio rigid body model calculations based                                                                                    SAXS data <br />                                                                                    ab-initio computational routines used cal- <br />                                                                                    culate low resolution models molecular <br />                                                                                    envelopes proteins Sis1, Sis1_&#206;&#8221;124-174, Sis1_&#206;&#8221;121- <br />   Figure 4 Results analytical ultracentrifugation (AUC) <br />                                                                                    257 , Ydj1_&#206;&#8221; 106-255 proteins. Dummy Atoms <br />   sedimentation velocity (SV) experiments.   Figure shows <br />   experiments using 1,000 &#206;&#188;g/mL protein concentration 25                     (DA) Dummy Residues (DR) models derived <br />   mmol/L Tris-HCl (pH 7.5) containing 500 mmol/L NaCl. (B)                     X-ray scattering data introducing 2-point <br />   software Sednterp used correct apparent s s20,w.               symmetry constraint calculations. assump- <br />   Proteins s020,w values calculated plot s20,w versus             tion based solution scattering data,   protein concentration shown Table 1. <br />                                                                                    indicated molecules study dimers <br />                                                                                    solution. crystallographic structure identified <br />  <br />  <br /> Table 1 Summary hydrodynamic data obtained DLS AUC experiments. <br />       Hydrodynamic properties                                                                  Protein <br />                                                      Sis1                   Sis1_&#206;&#8221;124-174                Sis1_&#206;&#8221;121-257       Ydj1_&#206;&#8221;106-255 <br />           Predicted M (kDa)*                         75.2                        66.1                        46.8                56.1 <br />          Experimental M (kDa)                      64 &#194;&#177; 3&#226;&#8218;&#172;                 58 &#194;&#177; 2# 58 &#194;&#177; 1&#226;&#8218;&#172;           45 &#194;&#177; 2# 43 &#194;&#177; 1&#226;&#8218;&#172;        54 &#194;&#177; 1# 55 &#194;&#177; 2&#226;&#8218;&#172; <br />                 s020,w (S)#                       3.5 &#194;&#177; 0.1%                   3.4 &#194;&#177; 0.1                   3.0 &#194;&#177; 0.1           3.6 &#194;&#177; 0.1 <br />                   ssph (S)                            6.5                         5.9                         4.7                 5.2 <br />           D20,w (10-7cm2/seg)                     5.1 &#194;&#177; 0.2                    5.4 &#194;&#177; 0.1                   6.3 &#194;&#177; 0.1           5.9 &#194;&#177; 0.2 <br />            Dsph (10-7cm2/seg)                         7.7                         8.0                         9.0                8.45 <br />                   &#198;&#8217;/&#198;&#8217;0#                           1.6 &#194;&#177; 0.1                    1.6 &#194;&#177; 0.1                   1.5 &#194;&#177; 0.1           1.4 &#194;&#177; 0.1 <br /> * M predicted amino acids sequence proteins dimers. <br /> # <br />   data SV experiments fitted SedFit software  Material methods details); <br /> &#226;&#8218;&#172; <br />   M calculated s/D ratio showed Equation 4; <br /> % <br />   data [19]. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                                              Page 6 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br />   Figure 5 SAXS results proteins Sis1, Sis1_&#206;&#8221;124-174, Sis1_&#206;&#8221;121-257, Ydj1_&#206;&#8221;106-255.   Scattering intensity curves   proteins. inset shows Guinier region showing linear regression fit experimental data proteins. (B) Corresponding p(r) <br />   functions. (C) Kratky plots showing different degrees flexibility protein. (D) Porod plots proteins. small plateau region   curves Sis1 Sis1_&#206;&#8221;124-174 indicate defined molecular volume macromolecules. <br />  <br />  Sis1 <span id='am-5' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> binding fragment homodimer                            generally chosen application program crystal [20] information modeled                         (qmax~ 8/Rg). q-range range (0.01&lt;q&lt;0. 25 &#195;&#8230;-1) <br /> dimer structure Ydj1 peptide binding fragment                           used DR rigid body modeling routines <br /> [25] taken account calculations. q-                        Methods). Keeping mind SAXS ab initio <br /> range used DAMMIN model calculation                             modeling routines produce unique solution, <br />  <br />  <br /> Table 2 Overall parameters derived SAXS results proteins Sis1, Sis1_&#206;&#8221;124-174, Sis1_&#206;&#8221;121-257 Ydj1_&#206;&#8221;106-255 <br />                                                                                              Protein <br />        Structural properties                      Sis1                   Sis1_&#206;&#8221;124-174                 Sis1_&#206;&#8221;121-257                 Ydj1_&#206;&#8221;106-255 <br />              C (mg/mL)                           7.1, 3.1                   6.8, 4.2                      5.2, 3.9                      5.5, 2.75 <br />            Rg (&#195;&#8230;) (Guinier)                      42 &#194;&#177; 1                     42 &#194;&#177; 1                        35 &#194;&#177; 1                         33 &#194;&#177; 1 <br />           Rg (&#195;&#8230;)  p(r))                   43.3 &#194;&#177; 0.5                 42.6 &#194;&#177; 0.3                     35.2 &#194;&#177; 0.3                    34.2 &#194;&#177; 0.3 <br />               Dmax (&#195;&#8230;)                             160                        140                           125                           110 <br />            M (kDa) (SAXS)                         ~71                        ~69                            ~42                           ~47 <br />          Predicted M (kDa)*                       75.2                       66.1                           46.8                          56.1 <br />        Oligomerization state                     Dimer                      Dimer                          Dimer                         Dimer <br /> * M predicted amino acids sequence proteins dimers. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                          Page 7 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  independent runs performed                        identification position individual domains <br /> calculation.                                                        RB models. <br />    resolution DA DR models does                     ab initio rigid body models calculated permit unambiguous determination spatial                  length Sis1 protein deleted-domain <br /> positions secondary structure elements,                 mutants presented Figure 6. DA (filter aver- <br /> portray overall recurring extended shape                 aged), DR RB models proteins Sis1, Sis1_&#206;&#8221;124- frequent conformations adopted mole-                  174 , Sis1_&#206;&#8221; 121-257 , shown panels  B C <br /> cules solution. spherical start volume used                respectively. case DR models, averaging DA modeling calculation order minimize                does add substantially results.  mod- generation specific direction p2                 els displayed presenting lowest normal- <br /> symmetry introduced based fact mole-                ized spatial discrepancy (NSD) values, cules dimers.  dealing                 showed best agreement DA model. NSDs <br /> flexible molecules, important check DR             tend zero nearly similar objects Rigid Body approaches (non initial-volume depen-                exceed 1, objects systematically differ dent) giving similar results. Consequently, rigid               explained Methods section). body (RB) calculations performed attempt                 NSDs calculated considered reasonably good obtain information position J-domain               DA DR modeling approaches account domains available crystallographic data              flexibility conformational changes restore protein structures based SAXS data.               molecules. itemized NSD values DA, DR cases, tried model molecular envelope              RB models Sis1, Sis1_&#206;&#8221;124-174, Sis1_&#206;&#8221;121-257, protein, performing calculations 2-point                listed Table 3. <br /> symmetry constraints proteins dimeric                   filtered average ab initio DA model, structure explained   previously men-              DR model lowest NSD value rigid <br /> tioned, dimerization sites proteins                 body model Ydj1_&#206;&#8221;106-255 protein presented <br /> known priori X-ray crystallographic                    Figure 7. results obtained following structure information used impose                   procedures previously described Sis1 dimerization contact conditions RB calculations.             length protein deleted domain mutants. position linker C-terminal                   NSD values corresponding model calculations <br /> domains J-domain protein known                  (Table 3) values considered <span id='am-28' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-29' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>amino acid</span> sequences. runs                    reasonably good different approach. DR <br /> formed set calculations. multiple runs              model lowest NSD value presents remarkably <br /> gave coincident results allowed                      good resemblance DA rigid body models, <br />  <br />  <br />  <br />  <br />  Figure 6 Results ab initio (DA DR) model calculations rigid body models displayed length <br />  Sis1 protein deleted mutants.   Sis1, (B) Sis1_&#206;&#8221;124-174, (C) Sis1_&#206;&#8221;121-257. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                                        Page 8 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> Table 3 Normalized Spatial Discrepancy (NSD) range obtained pairwise comparison SAXS models group calculations <br />                                                                                    Protein <br />         Modelling method                     Sis1                  Sis1_&#206;&#8221;124-174             Sis1_&#206;&#8221;121-257       Ydj1_&#206;&#8221;106-255 <br />           Dummy atoms                     1.00 - 1.22               1.17 - 1.41               0.75 - 1.01         0.88 - 1.21 <br />          Dummy residues                   1.41 - 1.97               1.84 - 2.68               1.47 - 1.76         1.38 - 1.56 <br />       Ab initio rigid body            1.95 - 2.87               1.95 - 2.51               1.54 - 1.85         1.87 - 2.11 <br />  <br />  considered excellent description                   activity, did decrease polypeptide binding activity. <br /> Ydj1_&#206;&#8221;106-255 protein conformation solution.                         Surprisingly, deletion G/F region did change <br />                                                                         quaternary structure overall flexibility Hsp40s. <br /> Conclusions                                                              deleting ZFLR plus CTDI domain previous studies, Ydj1 Sis1                     Type Ydj1 altered conformation J-domains homodimers, structures differed context                 lying long axis molecule orien- orientation J-domain relation long-                tated crosswise direction Type II Sis1.  <br /> axis respective proteins [18,19]. differences              orientation J-domains Type Hsp40  structure correlate functional differences exhib-               trolled ZFLR-CTDI, consistent ited Ydj1 Sis1, G/F G/M regions                   results domain swap experiments [15,19]. proteins proposed impact J-domain orien-                  domains missing, deleted mutants <br /> tation [15]. test hypothesis carried                     studied Type II Hsp40s, J-domains <br /> hydrodynamic low-resolution structural studies                   oriented crosswise direction. results <br /> Sis1 Ydj1 deletion mutants. deletion G/F                 point interactions ZFLR-CTDI region Sis1 Ydj1 little impact func-                J-domains important factor determining tion J-domain regulating Hsp70 ATPase                         unique quaternary structure Type Hsp40s. <br />                                                                            interesting feature Type Type II Hsp40s <br />                                                                         clear data sub-types                                                                         Hsp40s highly flexible. SAXS intensity data <br />                                                                         obtained wider range reciprocal space                                                                         present experiments allowed comparative analysis                                                                         flexible nature proteins means                                                                         Kratky Porod plots. results showed                                                                         Type Ydj1 Type II Sis1 Hsp40 proteins highly <br />                                                                         flexible solution confirm dimerization                                                                         C-terminal.  flexibility observed  <br />                                                                         structs studied, deleting flexible region (G/ <br />                                                                         M).  flexible regions exist protein <br />                                                                         addition G/M. Prior studies presented <br />                                                                         assumed G/F G/M domains <br />                                                                         flexible regions Hsp40s enabled <br />                                                                         J-domain multiple paths gain access Hsp70 <br />                                                                         ATPase domain.  Kratky Porod analysis                                                                         SAXS data Sis1 Ydj1 deletion mutants                                                                         remain highly flexible absence G/F <br />                                                                         region. Hsp40s bind deliver proteins range <br />                                                                         nascent monomeric polypeptides amyloid-like <br />                                                                         aggregates Hsp70.  overall flexibility Type <br />                                                                         Type II Hsp40s detected important  <br />                                                                         mit delivery proteins different sizes assembly <br />                                                                         states Hsp70 polypeptide binding site allow <br />  Figure 7 Results ab initio (DA DR) model                    simultaneous interaction J-domains  calculations rigid body model obtained                 Hsp70s ATPase domain [5]. results  Ydj1_&#206;&#8221;106-255 protein, using high resolution data available        agreement crystallographic structural studies <br />  domains. <br />                                                                         reported Hu et al. [27], observed  Silva et al. BMC Structural Biology 2011, 11:40                                                             Page 9 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  CTDI human Hsp40 possess significant flex-         favorable interactions Hsp40 ibility. work, authors proposed &#226;&#8364;&#339;anchor-      Hsp70. models built proteins ing docking&#226;&#8364;? model Hsp40                 agree anchoring docking model, proposed <br /> flexibility CTDI important Hsp40         Qian et al. [10], describing Hsp40 facilitates regulate size cleft interaction non-   delivery non-native polypeptides Hsp70. <br /> native polypeptides transfer Hsp70. <br />   Functions G/F G/M regions Type         Methods <br /> II Hsp40 regulation Hsp70 ATPase activity        Protein expression purification <br /> substrate binding clear.  important    recombinant protein Sis1 Ydj1 expressed <br /> question Sis1 fragment containing just J-       purified chromatographic steps previously <br /> domain G/F region capable rescuing lethal-      described [15,19,21]. Sis1 mutants prepared <br /> ity sis1&#206;&#8221; strain [22]. addition, amino acid resi-   DNA constructions expressed Escherichia <br /> dues G/M region appear play role               coli BL21(DE3)pLys: Sis1_&#206;&#8221; 124-174 (pET11aSIS1_&#206;&#8221; 124- <br /> specification Sis1 function [28]. Studies forms    174), deleting residues 124 174, Sis1_&#206;&#8221;121- <br /> Sis1 G/M domain deleted            257 (pET11aSIS1_&#206;&#8221;121-257), deleting residues 121 loss G/M impairs ability Sis1 bind dena-     257. Additionally, DNA construction Ydj1 <br /> tured luciferase, effect regulation         mutant (pET11aYdj1_&#206;&#8221; 106-255 ) deleted residues <br /> Hsp70 ATPase activity. G/M region lies adjacent        106 255, prepared expressed E. hydrophobic grove CTDI implicated polypep-        coli BL21(DE3) strain. Cells grown 37&#194;&#176;C tide binding site methionine hydrophobic       optical density 600 nm 0.7. temperature <br /> chain, help build proper binding site.       reduced 30&#194;&#176;C protein expression  G/M region helps specify Type II Hsp40 func-        induced 0.4 mM isopropyl thio-b-D-galactoside <br /> tion assisting substrate binding.                       (IPTG) 4 hours.  cells har- <br />   effects function likely related   vested centrifugation 10 min 2,600 &#195;&#8212; g. <br /> changes structure Hsp40s, important         pellet ressuspended 50 mM Tris-HCl (pH <br /> objective work obtain low resolution mod-      8.0), 500 mM KCl 1 mM EDTA (15 mL/L LB <br /> els mutants study using latest computa-      medium). cells lysed adding 30 &#206;&#188;g/mL tional methods available spatial representation    lysozyme (Sigma) 5 U DNAse (GIBCO BRL), molecules solution using small angle scattering      kept 30 min ice bath, disrupted soni- <br /> data. <span id='am-16' about='oboInOwl:date' typeof='owl:Thing'>date</span>, crystallographic structures       cation centrifuged (30 min 26,000 &#195;&#8212; g). <br /> isolated domains quaternary structure                 purification proteins performed pre- <br /> Hsp40s mainly known result SAXS data          viously described [16,29]. Summarily, Sis1, Sis1_&#206;&#8221;124-174 proteins solution, combined hydro-        Sis1_&#206;&#8221;121-257 submitted cationic chroma- <br /> dynamic techniques. Previous structural studies          tography Macro-prep (TM) High S Support resin proved Type Ydj1 Type II Sis1 dis-       (BioRad) using &#195;&#8222;KTA FPLC device (Pharmacia Bio- <br /> tinct functions quaternary structures [15,19].    tech). resin equilibrated 20 mM Tris-HCl <br /> work, estimation molecular masses SAXS            buffer (pH 7.5) 20 mM NaCl. Ydj1 Ydj1_&#206;&#8221;106- <br /> AUC data indicated deletion mutants solu-         255 submitted anionic chromatography tion dimerized C-terminal, just like length   Macro-prep(R) High Q Support resin (BioRad) using protein.  clear C-terminal region      &#195;&#8222;KTA FPLC device (Pharmacia Biotech). proteins important dimerization Type            eluted NaCl gradient, dialyzed overnight Type II Hsp40s.                                               buffer 20 mM Phosphate (pH 7.5), purified <br />   Using different low-resolution modeling meth-         chromatography CHTTM Ceramic Hydroxyapa- <br /> ods, obtained molecular envelopes protein.        tite Type II resin (BioRad) &#195;&#8222;KTA FPLC (Pharma- rigid body modeling method          cia Biotech). target proteins eluted appropriate flexible proteins like Hsp40 elu-    phosphate gradient. Ydj1_&#206;&#8221;106-255 purified <br /> cidated domain arrangement important          size exclusion chromatography Superdex <br /> understand possible functions deleted domain           200pg using &#195;&#8222;KTA FPLC (Pharmacia Biotech) pre- <br /> mutants cell. Interestingly, deleting        viously equilibrated 25 mM Tris-HCl buffer (pH <br /> flexible domains Sis1 Ydj1, constructs kept        7.5) 500 mM NaCl. efficacy purification flexibility maintained clamp-like architec-     checked 12% SDS-PAGE. Unless stated  <br /> ture length protein J-domains point-     wise, proteins diluted buffer 25 mM Tris- <br /> ing outwards opposite directions. architecture        HCl (pH 7.5) containing 500 mM NaCl. <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                             Page 10 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> Circular dichroism                                            applying equation 1 frictional ratio (&#198;&#8217;/&#198;&#8217; 0 ) Circular dichroism (CD) measurements performed           used indicate particle asymmetry compared using Jasco J-810 spectropolarimeter coupled Pel-      globular protein M giving information tier-type PFD 425S temperature control         shape proteins [26,31]. <br /> optimized best performance previously described <br /> [30]. proteins suspended buffer 25 mM          Analytical Ultracentrifugation <br /> Tris-HCl (pH 7.5) containing 500 mM NaCl. Proteins        Analytical ultracentrifugation (AUC) experiments centration ranged 10 40 &#206;&#188;M spectra       performed Beckman Optima XL analytical ultra- <br /> collected scan rate 50 nm/min spectral band-   centrifuge. Sedimentation velocity (SV) experiments <br /> width 1 nm using 0.2 mm path length cell.            carried concentrations ranging 150                                                               1,000 &#206;&#188;g/mL. SV experiments performed Chaperone activity                                            20&#194;&#176;C, using 30,000 rpm  60Ti rotor) Sis1_&#206;&#8221;121- <br /> Hsp40s activities tested ability bind        257 Ydj1_&#206;&#8221;106-255, 25,000 rpm Sis1_&#206;&#8221;124-174. <br /> heated denaturated luciferase stimulate Hsp70          SedFit software (Version 9.4) used deconvo- <br /> Ssa1 ATPase activity previously described [15,21].         lute sedimentation diffusion data order ability bind heated luciferase  client protein)      obtain continuous sedimentation distribution c(S) tested Sis1 binding set standard           weight average value frictional ratio. &#198;&#8217;/&#198;&#8217;0 <br /> (100%). respect Hsp40s ability stimulate       value used parameter regularization <br /> ATPase activity Hsp70, mutants assayed            function used estimate molecular mass <br /> regarding ability stimulate ATP hydrolysis        c(M) plots [32,33]. apparent sedimentation <br /> Ssa1 (Hsp70) Sis1 stimulatory effect set       coefficients (s) obtained maximum peak <br /> standard (100%).                                              values c(S) curves. standard sedimentation <br />                                                               coefficients (s20,w) protein concentration Dynamic Light Scattering                                      estimated avoid interferences caused viscosity experimental diffusion coefficient (D) obtained       density increment [26,31]. Sednterp software http:// dynamic light scattering (DLS) using DynaPro-MS/         www.jphilo.mailway.com/download.htm used X device (Protein Solutions). experiments        estimate important hydrodynamic parameters: (1) formed 20&#194;&#176;C, proteins concentration ranged        partial specific volume (V bar ) Sis1 (0.7263 mL/g), <br /> 0.5 2.0 mg/mL. D value corrected stan-          Sis1_&#206;&#8221;124-174 (0.7312 mL/g), Sis1_&#206;&#8221;121-257 (0.7284 mL/g) <br /> dard conditions (D20,w ) extrapolated 0 mg/mL          Ydj1_&#206;&#8221;106-255 (0.7331 mL/g) amino acid <br /> concentration (D020,w) order avoid effects viscos-   sequence; (2) ssph Dsph globular protein ity temperature. D related frictional coeffi-   molecular mass M buffer viscosity (h = 1.0605 <br /> cient (&#198;&#8217;) following equation:                          x10-2 poise) density (r = 1.01938 g/mL); (3)                                                               correct apparent value s s 20,w . standard <br />           RT <br />    D=                                                   (1)   sedimentation coefficient extrapolated 0 mg/mL (s020, <br />           NA f <br />                                                               w) calculated linear regression values s20, <br />  <br />   T absolute temperature, R <span id='am-10' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-11' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-14' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="gas" datatype="xsd:string"></span>gas</span>       w versus protein concentration. molecular mass <br />  <br /> stant NA Avogadro&#226;&#8364;&#8482;s number.                        values obtained ratio sedimentation <br />   protein known Stokes radius (Rs) visc-       diffusion coefficient using following equation. <br /> osity (h), &#198;&#8217; obtained applying Stokes equa-                    sRT <br /> tion:                                                            M=                                                  (4) <br />                                                                       D(1 &#226;&#710;&#8217; Vbar &#207;?) <br />     f = 6&#207;&#8364; &#206;&#183;Rs                                          (2) <br />   comparison, frictional coefficient spheri-    Small-Angle X-ray scattering experiments <br /> cal particle (&#198;&#8217;0) calculated using predicted       SAXS experiments performed D02A-SAXS2 <br /> Stokes radius (R0) smooth compact spherical         beamline Laborat&#195;&#179;rio Nacional Luz S&#195;&#173;ncrotron <br /> protein molecular mass M:                                  (LNLS, Campinas-SP, Brazil). X-ray scattering data <br />                         1/3                                   recorded using dimensional position-sensi- <br />               3MVbar                                          tive MARCCD detector. measurements  <br />    R0 =                                                 (3) <br />               4&#207;&#8364; NA                                           formed monochromatic X-ray beam (wavelength <br />                                                               l = 1.488 &#195;&#8230;) sample detector distance  Vbar partial specific volume, &#198;&#8217;0 used     1374.4 mm, corresponding scattering vector <br /> obtain maximum diffusion coefficient (D <span id='am-71' about='obo:IMR_0100178' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-72' property="oboInOwl:hasExactSynonym" content="sphingenine" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="sphingenine" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Sphingenine" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0100178" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="sphingosine" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="sph" datatype="xsd:string"></span><span id='am-78' property="rdfs:label" content="Sphingenine" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00319" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="sph" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="Sphingoid" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasExactSynonym" content="Sphingoid" datatype="xsd:string"></span>sph</span> )             range 0.01 &lt; q &lt; 0.25&#195;&#8230;-1, q magnitude <br />  Silva et al. BMC Structural Biology 2011, 11:40                                                            Page 11 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  q-vector defined (2&#206;&#184; scattering angle).    qualitative information degree flexibility samples placed 1-mm path length cell          compactness [38,39,41-43]. mica windows [34]. scattering patterns recorded different sample concentrations         Ab initio modeling based SAXS data sample: 7.1 3.1 mg/mL Sis1, 6.8 4.2        Ab initio calculations based SAXS data  <br /> mg/mL Sis1_&#206;&#8221; 124-174 , 5.2 3.9 mg/mL            formed obtain low-resolution models confor- <br /> Sis1_&#206;&#8221;121-257, 5.5 2.75 mg/mL Ydj1_&#206;&#8221;106-255.    mation following proteins: Sis1, Sis1_&#206;&#8221; 124-174 , samples measured buffer 25 mM Tris-         Sis1_&#206;&#8221;121-257, Ydj1_&#206;&#8221;106-255. different ab initio <br /> HCl (pH 7.5) containing 500 mM NaCl. succes-          approaches applied using dummy atoms (DA) <br /> sive frames 300s recorded sample      dummy residues (DR) modeling methods. Given frames buffer. scattering          existence certain flexibility proteins curves individually corrected detector response    types calculations necessary identify scaled incident beam intensity sam-      existence common structural features ples absorption. corrected buffer scattering curve      different models. dummy atoms modeling method subtracted corresponding sample scatter-       provided bead model calculated intensity fitted <br /> ing. resulting curves normalized respec-    experimental SAXS curve  Figure S1 Addi- <br /> tive concentrations carefully inspected check    tional file 1). DA approach implemented using <br /> possible radiation-induced damage concentration         program DAMMIN [44] q-range (qmax &lt;8/Rg) <br /> effects, effects observed. 5.6 mg/      ends slightly different protein. <br /> mL bovine serum albumin (BSA, 66 kDa) solution             unique solution obtained cal- <br /> used standard sample determine molecular        culation, independent calculations  <br /> mass proteins. molecular mass estimated      formed.  models pair wise compared comparison extrapolated value intensity    averaged using programs DAMAVER origin value, 0), samples scattering data   suite [45] SUPCOMB [46]. program aligns reference solution Bovine Serum       models represented ensembles points mini- <br /> Albumin (BSA) described Orthaber et al. [35]      mizing dissimilarity measure called Normalized Spatial <br /> Mylonas et al. [36].                                        Discrepancy (NSD). Generally, NSD values tend zero <br />                                                             increasingly similar objects; significantly <br /> SAXS data analysis                                          exceed 1, objects systematically differ Determination radius gyration (Rg)        [46]. Subsequently, new calculations  <br /> formed using Guinier approximation:                     formed protein using DR approach.                                                             dummy residues modeling method provides    q) = 0) exp(&#226;&#710;&#8217;q2 R2g /3)                        (5)   insights possible dimensional conformation <br />                                                             proteins deleted mutants solution. <br />   valid q-range qRg&lt;1 [37-39]. linear-     implemented using program GASBOR [47] <br /> ity scattering curves validity region     using range q values.  indepen- <br /> firmed monodispersity samples allowed            dent calculations performed NSD values analysis.  Rg values evaluated   evaluated.  average DR represen- pair distance distribution function p(r) calcu-    tations does substantially improve quality lated scattering intensity data means    models flexibility molecules.  quan- <br /> Indirect Fourier Transform package GNOM [40]. p         titative analysis NSD values obtained  <br /> (r) function provided maximum dimension            eral models performed order obtain Dmax molecule, p (r &#226;&#8240;&#165; Dmax) = 0 [38,39].     appropriate molecular conformation  e., model Guinier approximation, calculated p(r)     having lowest average NSD value), function provided values forward scattering         low resolution structure protein. <br /> intensity 0). 0) values used esti- <br /> mation molecular mass. confirmation       SAXS-based Modeling multidomain arrangement <br /> monodispersity dimerization Sis1, Sis1_&#206;&#8221;124-     proteins studied <br /> 174 , Sis1_&#206;&#8221; 121-257 , Ydj1_&#206;&#8221; 106-255 proteins      topology protein deleted-domain <br /> inferred molecular mass obtained         mutant Sis1 examined applying rigid body <br /> proteins. flexibility molecules estimated    modeling method SAXS data. approach scattered intensity analyzing behavior      employs simulated annealing protocol opti- Kratky curves (q2I(q) versus q) [38,39] Porod       mal positions orientations high-resolution struc- <br /> plots (q4I(q) versus q4). representations provided    tures known regions protein.  Silva et al. BMC Structural Biology 2011, 11:40                                                                                                Page 12 13 <br /> http://www.biomedcentral.com/1472-6807/11/40 <br />  <br />  <br />  <br />  <br /> time, conformation obtained unknown                                 hydrodynamics spectroscopic experiments interpreted                                                                                 CHIR. DMC produced deletion mutants performed <br /> regions (flexible linker attached appropriate resi-                      functional experiments. authors organized results, wrote dues domains) calculated finding best fit                        manuscript, read approved final version manuscript. experimental scattering data. rigid body cal- <br />                                                                                 Received: 17 June 2011 Accepted: 19 October 2011 <br /> culations implemented program BUNCH                                 Published: 19 October 2011 <br /> [48] using q-range. order construct model mutants Sis1, used high resolu-                          References <br /> tion structure Sis1 C-terminal peptide-binding                           1. Fink AL: Chaperone-mediated protein folding. Physiol Rev 1999, <br />                                                                                     79:425-449. <br /> domain Sis1 J-domain, Pro-                            2. Hartl FU: Molecular chaperones cellular protein folding. Nature 1996, <br /> tein Data Bank http://www.rcsb.org identified                                381:571-580. <br /> codes 1C3G 2O37 respectively. compose                                3. Hartl FU, Hayer-Hartl M: Protein folding - Molecular chaperones                                                                                     cytosol: nascent chain folded protein. Science 2002, <br /> model Ydj1 deleted domain mutant, used                               295:1852-1858. <br /> high resolution structure Ydj1 dimerization                              4. Bukau B, Horwich AL: Hsp70 Hsp60 chaperone machines. Cell <br /> domain (PDB code 1XAO) J-domain                                   1998, 92:351-366. <br />                                                                                 5. Summers DW, Douglas PM, Ramos CHI, Cyr DM: Polypeptide transfer Sis1 structure (PDB code 2O37). indepen-                                Hsp40 Hsp70 molecular chaperones. Trends Biochem Sci 2009, <br /> dent calculations performed protein.                               34:230-233. <br /> applied 2-point symmetry constraint model cal-                         6. Langer T, Lu C, Echols H, Flanagan J, Hayer MK, Hartl FU: Successive Action <br />                                                                                     Dnak, Dnaj Groel Pathway Chaperone-Mediated <br /> culations, using priori knowledge                            Protein Folding. Nature 1992, 356:683-689. dimers solution. NSD values evaluated                          7. Schmid D, Baici  Gehring H, Christen P: Kinetics Molecular Chaperone order select typical model pro-                             Action. Science 1994, 263:971-973. <br />                                                                                 8. Misselwitz B, Staeck O, Rapoport TA: J proteins catalytically activate hsp70 <br /> tein. fits models (Figure S1) corre-                            molecules trap wide range peptide sequences. Mol Cell 1998, <br /> sponding chi values (Table S1)                                      2:593-603. <br /> additional files 1 2 respectively.                                          9. Laufen T, Mayer MP, Beisel C, Klostermeier D, Mogk  Reinstein J, et al: <br />                                                                                     Mechanism regulation Hsp70 chaperones DnaJ cochaperones. <br />                                                                                     Proc Natl Acad Sci USA 1999, 96:5452-5457. <br /> Additional material                                                             10. Qian XG, Hou WB, Li ZG, Sha BD: Direct interactions molecular <br />                                                                                     chaperones heat-shock protein (Hsp) 70 Hsp40: yeast Hsp70 Ssa1 <br />                                                                                     binds extreme C-terminal region yeast Hsp40 Sis1. Biochem J <br />     Additional file 1: Figure S1. Model fits proteins studied                                                                                     2002, 361:27-34. <br />     work. <br />                                                                                 11. Cyr DM, Langer T, Douglas MG: Dnaj-Like Proteins - Molecular <br />     Additional file 2: Table S2. Chi-values (root mean square c2)                Chaperones Specific Regulators Hsp70. Trends Biochem Sci 1994, <br />     obtained fit theoretical scattering calculated         19:176-181. <br />     models experimental intensity curve.                                 12. Wall D, Zylicz M, Georgopoulos C: Nh2-Terminal 108 Amino-Acids                                                                                     Escherichia-Coli Dnaj Protein Stimulate Atpase Activity Dnak <br />                                                                                     Sufficient Lambda-Replication. J Biol Chem 1994, <br />                                                                                     269:5446-5451. <br /> Acknowledgements Funding                                                    13. Luke MM, Sutton  Arndt KT: Characterization Sis1, Saccharomyces- research supported Brazilian Synchrotron Light Laboratory           Cerevisiae Homolog Bacterial Dnaj Proteins. J Cell Biol 1991, <br /> (LNLS)/Ministry Science Technology (MCT) State S&#195;&#163;o Paulo          114:623-638. <br /> Research Foundation (FAPESP). Research laboratory CHIR             14. Caplan AJ, Douglas MG: Characterization Ydj1 - Yeast Homolog supported grants FAPESP, MCT/National Research Council                  Bacterial Dnaj Protein. J Cell Biol 1991, 114:609-621. <br /> (MCT/CNPq), NIH-R03TW007437 funded Fogarty International             15. Fan CY, Lee S, Ren HY, Cyr DM: Exchangeable chaperone modules <br /> Center (FIRCA-NIH). J.C. Silva thanks FAPESP CNPq financial support         contribute specification type type II Hsp40 cellular function. FIRCA-NIH paying travelling expenses training. J.C. Borges          Mol Biol Cell 2004, 15:761-773. <br /> thanks FIRCA-NIH fellowship, FAPESP (2007/05001-4) financial support    16. Lu  Cyr DM: Protein folding activity Hsp70 modified differentially CNPq Research Fellowship. ILT CNPq (1B) Research Fellow,           Hsp40 chaperones Sis1 Ydj1. J Biol Chem 1998, <br /> CHIR CNPq (1C) Research Fellow. DMC supported NIH RO1-                      273:27824-27830. <br /> GM067785.                                                                       17. Banecki B, Liberek K, Wall D, Wawrzynow  Georgopoulos C, Bertoli E, et al: <br />                                                                                     Structure-function analysis zinc finger region DnaJ <br /> Author details                                                                      molecular chaperone. J Biol Chem 1996, 271:14840-14848. <br /> 1 <br />  Department Condensed Matter Physics, &#226;&#8364;&#339;Gleb Wataghin&#226;&#8364;? Physics Institute,     18. Borges JC, Fischer H, Craievich AF, Ramos CHI: Low resolution structural <br /> State University Campinas (UNICAMP), Campinas, SP 13083-859, Brazil.             study human HSP40 chaperones solution - DjA1 2 <br />  Brazilian Biosciences National Laboratory, Center Research Energy       subfamily DjB4 subfamily B different quaternary <br /> Materials (CNPEM), Campinas, SP 13083-970, Brazil. 3Institute Chemistry       structures. J Biol Chem 2005, 280:13671-13681. <br /> S&#195;&#163;o Carlos, University S&#195;&#163;o Paulo, S&#195;&#163;o Carlos, SP 13.560-970, Brazil.         19. Ramos CHI, Oliveira CLP, Fan CY, Torriani  Cyr DM: Conserved Central <br /> 4 <br />  Department Cell Developmental Biology, University North                  Domains Control Quaternary Structure Type Type II Hsp40 <br /> Carolina, Chapel Hill, NC 27599, USA. 5Department Organic Chemistry,             Molecular Chaperones. J Mol Biol 2008, 383:155-166. <br /> Institute Chemistry, University Campinas UNICAMP, SP 13083-970, Brazil.   20. Sha BD, Lee S, Cyr DM: crystal structure peptide-binding <br /> 6 <br />  European Synchrotron Radiation Facility, Grenoble, France.                         fragment yeast Hsp40 protein Sis1. Structure 2000, 8:799-807. <br />                                                                                 21. Lee S, Fan CY, Younger JM, Ren HY, Cyr DM: Identification essential <br /> Authors&#226;&#8364;&#8482; contributions                                                              residues Type II Hsp40 Sis1 function polypept <br /> </body></html>