<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, July 1989, p. 3081-3087                                                                   Vol. 9.  7 <br /> 0270-7306/89/073081-07$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  Single Essential Gene, PRI2, Encodes Large Subunit DNA <br />                Primase Saccharomyces cerevisiae <br />            MARCO FOIANI, CORRADO SANTOCANALE, PAOLO PLEVANI. GIOVANNA LUCCHINI* <br />       Dipartirnento di Genetica e di Biologia dei Mlicrorgadisini, Univ&apos;ersitdi di Milano, Viai Celoria 26, 20133 Milani, Ital <br />                                           Received 28 December 1988/Accepted 2 March 1989 <br />  <br />              DNA primase activity yeast DNA polymerase-primase complex related polypeptides, p58            p48. reciprocal role protein species clarified, participate            formation active center enzyme. gene encoding p58 subunit cloned screening <br />            Xgtll yeast genomic DNA library, using specific anti-p58 antiserum. Antibodies inhibited DNA <br />            primase activity purified lysates Escherichia coli cells infected recombinant bacteriophage <br />            containing entire gene, designate PRI2. gene transcribed 1.7-kilobase <br />            mRNA level appeared fluctuate mitotic cell cycle. <span id='am-26' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>Nucleotide</span> sequence determination <br />            indicated PRJ2 encodes 528-amino-acid polypeptide calculated molecular weight 62,262.            gene unique haploid yeast genome, product essential cell viability, shown            components yeast DNA polymerase-primase complex. <br />  <br />    DNA polymerase-primase complex highly                   essential cell viability directly related DNA <br /> served polypeptide structure number              primase activity (20). Recently, used highly specific <br /> eucaryotic organisms phylogenetically distant yeasts              affinity-labeling procedure map catalytic site yeast humans. complete understanding functions                DNA primase p48 p58 participate complex achieved, data              formation primase active center, <span id='am-10' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span>ATP</span>- <br /> suggest plays crucial role priming            binding site exclusively located p48 subunit (Sa). <br /> elongation Okazaki fragments initiation             paper, report isolation characterization <br /> step DNA replication origin (reviewed references            yeast PRI2 gene, encoding p58 polypeptide. 4 15). DNA polymerase activity associated               gene unique haploid yeast genome highest-molecular-size polypeptide complex (170               transcribed 1.7-kb mRNA appears fluctuate 180 kilodaltons [kDa]), smallest polypep-               cell cycle. gene encodes protein <br /> tides (-60 -50 kDa, respectively) correlated               calculated molecular size 62,262 Da essential <br /> DNA primase activity. fourth polypeptide, 70 80                 cell viability directly related DNA primase <br /> kDa, function unknown, present            activity, confirming previous biochemical findings. <br /> complex (4, 15). <br />    better understand functions interactions                          MATERIALS METHODS <br /> protein complex essential DNA replication,                    Strains plasmids. E. coli Y1090 (45) used took cloning characterization genes encoding             plaque immunoscreening. E. c oli Y1089 (45) lysogenized different protein species Saccharomnvces cerevisiae.             recombinant phages used protein analysis genes coding DNA polymerase  POLI. (13. 19, 26),              phage DNA preparation. Growth media phage infection small primase polypeptide, PRII (20, 30),             conditions described Maniatis et al. (22). Plasmid <br /> cloned sequenced. Sequence comparison revealed                    pGEM4 Promega Biotech, Cambridge, Mass. <br /> consistent <span id='am-34' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-39' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> sequence homology putative                    Plasmids pCS10 pCS11 obtained subcloning <br /> catalytic domain located carboxy-terminal half                X215 yeast DNA insert orientations EcoRI <br /> yeast DNA polymerase viral DNA polym-                  site pGEM4. Plasmid pCS15 constructed inserting <br /> erases. Interestingly, yeast DNA polymerase human                 HinidIII-ScaI fragment PRI2 5&apos; deletion, obtained <br /> DNA polymerase ox, sequenced polymer-                plasmid pCS11  , HindIII-NruI sites <br /> ases tightly associated DNA primase,              plasmid YIp5 (34). Plasmid pCS1OA67, used synthesize higher amino acid homology, extends                 vitro transcript RNase protection mapping 5&apos; <br /> amino-terminal half proteins (26, 43). Sequencing           endpoint PRI2 transcript, derived exonucle- <br /> yeast poll temperature-sensitive mutant altered stability            ase III-digested pCS10   contains PRI2 polymerase-primase complex located mutation               DNA fragment spanning positions -520 + 141  Fig. 6). conserved amino-terminal region (21, 26, 28),                  Plasmid pMF18 derivative plasmid YEp24 (3) carrying <br /> suggesting polypeptide structure               15-kb yeast DNA insert includes 7.5-kb EcoRI <br /> complex kinds interactions                  genomic PRI2 segment; identified screening protein components conserved yeast human cells.              YEp24 yeast genomic library (5) colony hybridization, <br />    S. cerev&apos;isiae, DNA primase correlates polypep-            using X215 yeast DNA insert probe. <br /> tides, 58 48 kDa (29), roles                    S. cerevisiae S288C (MATot prototroph) DES5 <br /> completely defined. Cloning characterization                 (MATaIMATot ura3-521/ar3-52) previously  <br /> PRII gene demonstrated p48 polypeptide                  scribed (20). Strain CG378 (MATa ura3-52 leiu2-3,112 trpl- <br />                                                                         289) L. H. Johnston (M.R.C., London, United <br />   * <br />       Corresponding author.                                             Kingdom). <br />                                                                  3081 <br />  3082     FOIANI ET AL.                                                                                             MOL. CELL. BIOI- <br />  <br />    Enzyme assays, immunol6gicai reagents, plaque screen-                                         B <br /> ing. Isolation DNA polymerase DNA primase com-                           Mr <br /> plex immunoaffinity chromatography physical sepa-                          (Xt)t 11 AE215       ACB 1 Ji.215; <br /> ration enzyme activities described (29. <br /> 31). DNA primase activity assayed poly(dT)                             116- <br /> template measuring incorporation  -32PIdAMP                            97 <br />  acid-insoluble material presence 200 U E.                        66                      -   _ <br /> coli DNA polymerase ml (29). <br />    Polyvalent rabbit antisera DNA polymerase-                               45 <br />  <br />  <br /> primase complex isolated protein species                         36-  - <br />  <br />  <br />  <br />                                                                                                 m <br /> described (20, 31). Antiserum p58 polypeptide                          29- <br />  affinity purified absorption specific antibodies                     20- p58 immobilized nitrocellulose strips (39). recov- <br /> ered antibodies used confirm identities              FIG. 1. Analysis proteins encoded X215 genome. <br /> positive recombinant phages plaque lysates transferred         Crude extracts E. coli Y1089 cells lysogenized Xgt1l nitrocellulose filters (19). Affinity purification p58 antibod-   X215 grown presence (+) absence (-) 5 mM IPTG <br /> ies plaque proteins phage X215 carried            prepared (45) analyzed 10% sodium dodecyl sulfate- <br />                                                                      polyacrylamide gel.   Coomassie blue-stained gel; (B) Western <br /> described previously (20).                                           blot analogous gel probed anti-p58 immunopurified <br />    Identification immunoreactive polypeptides. Lysates        antibodies. Arrows indicate 58- 50-kDa immunoreactive prepared E. coli Y1089 cells lysogenized              polypeptides. <br /> Xgtl1 X215 described Young Davis (45) analyzed sodium dodecyl sulfate-polyacrylamide gel                                       RESULTS <br /> electrophoresis (17). Western blotting (immunoblotting)             Identification p58 recombinant phages Xgtl1 expres- <br /> detection immunoreactive polypeptides ap-            sion library. previously described production propriate antiserum performed described            specific antisera raised single protein species (19).                                                                DNA polymerase-primase complex use    DNA sequencing. Plasmids pCSI0 pCS11 di-                 mixture anti-p74. anti-p58. anti-p48 antibodies gested SplIl BainHI exonuclease III treat-           screen genomic yeast DNA library Xgtll expression <br /> ment construct set unidirectional deletions 150        vector (20. 27). positive recombinant phages iden- <br /> base pairs (bp) apart procedure Henikoff (11). T7          tified retested specific antiserum. pro- <br /> phage promoter primer used sequence PRI2                  cedure allowed identification classes clones, <br /> region directions pCS10- pCS11-derived            designated p74. p58. p48 basis deletion plasmids dideoxynucleotide chain termina-            immunological properties (20). 10 recombinant phages <br /> tion method (33), using 5&apos;-[cc-35S]thio-<span id='am-5' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span>dATP</span>, denatured plas-        belonging class p58 retested anti-p58 affinity- <br /> mid DNA template (10), Klenow fragment E. coli                purified antibodies. Seven clones positively reacted DNA polymerase  sequences immediately adjacent              second test analyzed. phages, EcoRI linkers X215 yeast DNA insert (45)             X215, shown contain 2.6-kb yeast DNA fragment <br /> confirmed belong PR12 region using appropriate          inserted EcoRI site l(cZ  Fig. 4). fragment <br /> synthetic oligonucleotides primers plasmid pMF18.              purified used probe plaque hybridization test, <br /> Protein data base searches DNA sequence analysis            showed seven clones belonged performed UWGCG program.                                    cross-hybridization group (data shown). Western analy- <br />    Hybridization procedures techniques. Conditions         sis crude extracts prepared E. (coli cells lysogenized Southern blot hybridization described Roeder          seven recombinant phages revealed Fink (32). Northern (RNA) hybridization DNA                 constitutive polypeptides, 50 58 kDa, <br /> probes carried method Maniatis et al. (22)         respectively, specifically reacting anti-p58 antibod- <br />                                                                      ies (Fig. 1). extensive degradation foreign electrophoresis 1% agarose gel containing 2.2 M           proteins E. (coli strains (19, 44, 45), minor 58-kDa <br /> <span id='am-41' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-42' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span>formaldehyde</span> hybridization 42&apos;C presence            protein species native product yeast DNA <br /> 50Y formamide. Northern blot hybridization RNA                  insert. contain entire p58-coding <br /> probes performed 60&#194;&#176;C conditions.              region. transcribed opposite direction <br />    vitro transcription RNase protection mapping          respect la(cZ. confirm recombinant phage <br /> carried described Melton et al. (24). Phage            genome encodes p58 primase subunit, plasmid DNA preparation, DNA restriction analysis, agar-             determined X215 E. coli lysates used ose gel electrophoresis, nick translation DNA probes          purifv anti-p58 antibodies antiserum raised (specific activity, 108 cpm/[Lg) described             DNA polymerase-primase complex (31) previously (22). Total yeast DNA prepared                 reactive polypeptides. X215 affinity-purified <br /> method Winston et al. (42).                                       antibodies reacted p58 protein species    Total yeast RNA (12) enriched poly &apos; RNA            Western blots complex isolated DNA primase step purification oligo(dT)-cellulose. filter car-      (Fig. 2).  antibodies inhibited enzyme <br /> rying DNA separated yeast chromosomal bands                 activity tested isolated DNA primase (Table 1), kind gift P. Philippsen (Universitait Giessen.            confirming identity X215 yeast DNA insert <br /> Giessen, Federal Republic Germany). Synchronization            involvement p58 subunit DNA primase <br /> CG378 yeast cultures obtained oc-factor (Sigma              activity. <br /> <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> . St. Louis, Mo.) procedure White et            Characterization p58 gene (PRI2). X215 yeast <br /> al. (41).                                                            DNA insert used probe Northern blot analysis  VOL. 9, 1989                                                                        LARGE SUBUNIT YEAST DNA PRIMASE                                   3083 <br />  <br />  <br />                        Mr <br />                        Mr                                                                                1        2 3 <br />                                   1      2 3   4   5 <br />                      (xlO0 3 <br />                          180-   bow <br />                          140- <br />  <br />  <br />                           74-         9- <br />                           58- <br />                                                                                                         %W-               -   1.7 kb <br />                           48-            -- <br />                                           4        .&quot; <br />  <br />  <br />  <br />  <br />    FIG. 2. Immunoblot analysis using X215 affinity-purified antibod- <br /> ies. DNA polymerase-primase complex electrophoresed sodium dodecyl sulfate-polyacrylamide gel stained Coomassie blue (lane 1) blotted nitrocellulose filters probed antibodies raised complex (lane 3) X215 affinity-purified antibodies (lane 4). analysis performed isolated DNA primase: lane 2. stained gel: lane 5.                    FIG. 3. Identification determination direction Western blot probed X215 affinity-purified antibodies.                     PRI2 transcript. Poly + RNA (5 ,ug) strain S288C                                                                                 probed Northern blots X215 EcoRI yeast DNA insert <br />                                                                                 (lane 1). riboprobe 1 (lane 2). riboprobe 2 (lane 3). <br /> poly + yeast RNA. probe identified single 1.7-kb <br /> mRNA species large encode 58-kDa polypeptide                        p58 message opposite direction respect (Fig. 3). X215 yeast DNA insert subcloned                       IlaZ, previously observed Xgtll recombinant orientations EcoRI site plasmid pGEM4                         phages (7, 19). Preliminary studies cell cultures synchro- <br /> generate plasmids pCS10 pCS11, transcripts                      nized ox-factor (Fig. 5) indicated steady-state <br /> strand synthesized vitro T7 RNA polymerase                         level 1.7-kb mRNA fluctuated mitotic cell linearization plasmids HpaI NuoI,                         cycle reached peak G1 S-phase transition, respectively (Fig. 4).                                                          previously observed transcripts genes <br />    transcripts used probe sequentially                      involved DNA replication (14). identified gene, poly + RNA blot. direction transcription                      PR12, unique haploid yeast genome. fact, p58 gene BglII N(oI (Fig. 4),                        Southern blots total yeast DNA restricted E(oRI, <br /> riboprobe 2 hybridized 1.7-kb mRNA identified                       XloI, BamflHI, AvaI, KpnzI showed single bands X215 DNA insert, riboprobe 1 did recognize                      probed X215 DNA insert (data shown). Southern mRNA species (Fig. 3). data agreement                          blot analysis intact yeast chromosomes, fractionated observation X215 does express isopro-                     orthogonal-field gel electrophoresis (35) tested py-13-D-thiogalactopyranoside (IPTG)-inducible ,B-galactosi-                    probe, located gene chromosome XI. <br /> dase-p58 fusion protein. fact, orientation yeast <br /> DNA insert phage genome allows transcription                                   1Kb <br />  <br />  <br />  <br />       TABLE 1. Inhibition DNA primase activity X215 <br />                  affinity-purified antibodies&quot;                                                                -        &gt;~ <br />                                                                                                                        X <br />                                                                                                                                  O     cni&gt;40c     Cc <br />                                                                                          O                       C             OM <br />                                                                                          o    0.            .                                    0 <br />                                           % DNA primase activity&quot; <br /> Antiserum dilution                                 Affinity-purified antibody                                                         * --mRNA <br />                                 Ab mix <br />                                                    Xgtll               X215 <br />                                                                                                                                            215 <br />       1:256                      100                100                 100 <br />       1:128                      108                106                  85                                                                RIBOPROBE <br />       1:64                        92                104                  78                                                                RIBOPROBE 2 <br />       1:32                        81                 97                  61 <br />       1:16                        71                 91                  57        FIG. 4. Restriction map PR12 genomic region DNA- <br />       1:8                         60                 85                  47     sequencing strategy. figure depicts 7.5-kb EcoRI yeast ge- <br />       1:4                         41                 78                  30     nomic DNA fragment containing entire PRJ2 coding sequence <br />       1:2                         23                 66                  17     (C). map determined restriction analysis X215 DNA <br />                                                                                 plasmid pMF18  Materials Methods) confirmed <br />      10-ng sample purified DNA primase (29) incubated wPith serial     Southern blot analysis total yeast DNA digested dilutions antiserum raised DNA polymerase-primase com-        appropriate enzymes probed X215 2.6-kb EcoRI insert. <br /> plex (Ab mix) Xgt1l X215 affinity-purified antibodies. Dilutions    size orientation insert indicated ( H-H ) 50 mM Tris hydrochloride (pH 8.0)-l(O mM NaCI-0.1 mg bovine          asterisk denoting position amino-terminal portion serum albumin ml- 10% preimmune serum. saimples incubated       1a(cZ gene. direction PRI2 transcription  exact 3&apos; endpoint <br /> 0&#194;&#176;C 2.5 h microdilution plates precoated 1 mg bovine serum       determined) sizes directions riboprobes <br /> albumin ml. DNA primase activity remaining antiserum tr-eat- <br /> ment assayed described Materials Methods.                         generated vitro transcription X215 yeast DNA insert    &quot; 100% DNA primase activity corresponds 40 pmol [ot-32PldAMP           shown  Arrows indicate extents incorporated 30 min 3TC.                                                  directions sequenced fragments. <br />  3084           FOIANI ET AL.                                                                                                                                             MOL. CELL. BIOL. <br />  <br />                                                                                                              Nucleotide sequence PRI2. pointed                                                  -e       1-:&apos; 11                4-2.5 Kb <br />                                                                                    - <br />                                                                                                            sizes imnmunoreactive polypeptides identified X215 <br />                                                                                4-1.7 Kt;                   lysates suggested phage genome contain                                                                                                            entire p58-coding region.  sequenced X215 <br />                     +                                                                                      yeast DNA insert. set partially overlapping deletions <br />        BC                                                                                                  generated exonuclease III subcloning        -,o                                                                                                 DNA fragment orientations plasmid pGEM4         60                                                                                                  Materials Methods). Serial deletions sides, <br />        50                                                                                                  150 bp apart, sequenced dideoxynucle- <br />            ,, <br />        114 0 <br />        30 <br />                                                                                                -           otide chain termination method (33). restriction sites <br />                                                                                                            deduced corresponding 2,645-nucle- <br />        20 <br />        10                                                                                                  otide sequence (Fig. 6) agreement restriction <br />         0                                                                                                  map established experimentally (Fig. 4). 531-codon open <br />                                                                                                            reading frame terminated TAG stop codon                         30 60 90 120 150 180 210 240 270 300C                                              sequence  longest reading frame  <br />                                Time (miinutes&apos;                                                             coded 86 amino acids). direction transcription <br />                                                                                                            identified previously Northern <br />    FIG. 5. Fluctuation level PR12 transcript                                      hybridization  . ATG longest open <br /> cell cycle. x-Factor added time zero (+) cell cultures                                        reading frame located 239 bp upstream PvuI <br /> strain CG378 logarithmically growing synthetic medium                                           site, second ATG 244 bp downstream <br /> removed filtration 125 min (-). percentage budded                                          site. did invariant signal sequence <br /> cells indicated times determined light microscopy. Northern blot total RNA (5,ug lane) samples                                       TACTAAC, present introns yeast genes <br /> taken times probed X215 yeast DNA insert                                          transcribed RNA polymerase 11 (18). Given transla- EcoRI-SallI 1.6-kb yeast DNA fragment hybridizing                                         tion starts ATG (position +1 Fig. 6) 2.5-kb transcript, previously shown fluctuate                                             gene does contain introns, identified open reading cell cycle (14). <br />  <br />                                                                                                                                        CGGGTGACGGGACACCGCTACGAAT            -496 <br />                ATAGTACGATACACTTATACATATATGGAAGCATTTATAGCGTATATTCATCGTACGGTACATTCTTTAAATAAATGATTGAATTTTTCTGAACCAACA                                                          -397 <br />                GTTATAAGCTGAGATCCAGTGCGTTCATGAAACGGAACATCTTACATCATCAAGAGCCCAGAAGCACGGCGTAGGATTGTTTCCGCTGCTACAGACAGA                                                          -298 <br />                CATGGAAACACTGTTTTTATCAGGCCAAAGGTGATACGACAGGTCTCTTCTTGTTTCAATATTGCCTTACATATCTGCTGCTCACTCTATTTGCTTTTT                                                          -199 <br />                CCTGCTCATTTATAGTTAACTTGATTTGAAATTACGCGTCGCGGGGAATCGCGTAAAACTCGATATTAAATCCTGTGATTTTGCAAAAGTAAATTTTTT                                                          -100 <br />                ACTATCTACATCTCTCCAATTTGCACAACTTTGACAGGACTGATAACTACCCAGGTGGAGCAACCAGTGACAGCAGGTAAAGCACAATAAGGAGAAGAA                                                            -1 <br />                 N F R Q S K R R         S R     K N F S S Y D D V K S E L D V G N T N                                                                                 33 <br />                ATGTTCAGGCAGTCAAAAAGAAGAATTGCCTCAAGGAAGAATTTTAGTTCATACGATGATATTGTCAAAAGCGAATTGGACGTAGGCAACACAAATGCA                                                           +99 <br />                 N Q I L S S S S S E E E K K L Y R L Y E S K L S F Y D L P P                                                                                             66 <br />                GCTAACCAGATCATCTTAAGTTCAAGTTCATCTGAAGAAGAGAAAAAACTTTACGCTAGGCTGTATGAGTCTAAATTATCATTTTACGATTTACCACCA                                                          +198 <br />                 Q G E T L E Q F E W             D R L K L L E E S C L S R                   N K S                                                               99 <br />                CAGGGAGAAATAACGTTGGAACAATTTGAGATCTGGGCAATCGATCGTCTCAAAATTCTGTTAGAGATCGAATCTTGCCTTTCAAGAAACAAATCGATC                                                          +297 <br />                 K E E T I K P Q F Q K L L P F N T E S L E D                         R   K K D Y Y S H F                                                                  132 <br />                AAAGAAATTGAAACCATTATCAAGCCGCAGTTCCAAAAACTACTGCCTTTCAACACAGAGAGTTTAGAAGACAGAAAAAAGGATTACTACTCTCATTTC                                                          +396 <br />                   L   R   L   C F     C R          S      K      E   L   R   E       K   F       V   R     E T     F   L   F   K       R   F   N   N   L   T   S       165 <br />                ATTTTAAGATTGTGCTTTTGTCGTTCAAAAGAACTGCGGGAAAAATTCGTCAGGGCTGAAACGTTTTTGTTCAAAATTAGATTTAACATGCTAACCTCC                                                          +495 <br />                 T   D   Q T K F V Q S L D L P L L Q F S N E E K E L S H Q L Y Q T                                                                                        198 <br />                ACCGATCAAACAAAATTTGTTCAATCATTGGACTTACCCTTACTACAGTTTATCTCAAATGAAGAAAAGGCCGAATTATCTCATCAGTTGTATCAAACA                                                          +594 <br />                 V S S L Q F Q L N L N E E H Q R K Q Y F Q Q E K F K L P F E N                                                                                            231 <br />                GTTTCTGCGTCTCTACAGTTCCAATTGAATTTAAACGAGGAACATCAAAGAAAACAGTATTTCCAACAGGAAAAATTCATAAAATTACCTTTCGAAAAT                                                          +693 <br />                 V     E   L   V G     N   R        L      V      F   L   K   D       G   Y         Y   L   P   Q   F   Q   Q   L     N   L   L   S   N   E   F         264 <br />                GTGATAGAACTGGTAGGAAACCGTTTAGTGTTTTTGAAGGACGGGTACGCATATTTACCACAATTCCAGCAATTGAATTTACTTTCTAATGAGTTTGCT                                                          +792 <br />                 S   K   L   N   Q   E   L          K T           Y   Q   Y   L       P R         L   N   E   D   D   R   L   L   P       L   N   H   L   S   S   G       297 <br />                AGCAAATTGAACCAGGAGTTAATAAAAACGTACCAGTACTTACCAAGACTGAATGAGGATGACCGGTTGTTACCAATTCTAAATCATCTTTCGTCGGGG                                                          +891 <br />                 Y   T       D   F   N   Q        Q      K        N   Q   F       S   E       N   V   D   D   E     N     Q     S   V W     S   E   E     S       330 <br />                TACACTATCGCGGATTTCAACCAGCAAAAGGCAAACCAATTCAGTGAGAACGTAGATGATGAGATTAATGCGCAAAGTGTCTGGTCTGAAGAGATTAGC                                                          +990 <br />                 S   N   Y   P   L   C     K        N       L     N   E G     L       K   K       N   H   H   L R     Y   Y   G R       Q   Q   L   S   L   F   L   K       363 <br />                TCAAACTATCCGCTATGTATCAAAAACCTGATGGAGGGTCTTAAAAAGAACCATCATTTGAGGTATTATGGGAGACAACAACTGAGTCTGTTTTTGAAA                                                         +1089 <br />                 G     G   L   S     D   E              L     K   F W     S       E         F   T   R   N   G   N   N   T   N     E   K   F   N   K   E   Y   R       396 <br />                GGTATCGGCCTGAGCGCTGATGAAGCGTTAAAATTTTGGTCTGAGGCATTTACAAGAAATGGGAACATGACAATGGAGAAGTTCAATAAAGAATACCGT                                                         +1188 <br />                 Y S F R         H   N   Y G   G      L E           P W D C H T L S K P R P G R G <br />                                                                        N R           N   Y       K                                                                         429 <br />                TACAGCTTCAGGCATAATTACGGTCTTGAAGGTAACAGAATCAACTACAAACCATGGGACTGTCACACTATCCTTTCCAAGCCCAGACCTGGGCGCGGA                                                         +1287 <br />                 D Y H G C P F R D W S H E R L S E L R S N K L T Q Q I S V L                                                                                            462 <br />                GATTATCATGGATGCCCTTTCCGTGATTGGAGTCACGAGAGACTATCTGCAGAATTGCGCTCTATGAAGTTGACCCAAGCGCAGATCATCAGTGTTCTG                                                         +1386 <br />                 D   S   C   Q   K   G   E   Y        T             C   T   K       V   F       E M     T   H   N   S     S       D   L   E     G   E   Q T         495 <br />                GATTCCTGCCAGAAAGGTGAATACACAATTGCTTGCACTAAAGTGTTTGAAATGACACACAACTCTGCATCAGCGGATTTGGAAATTGGCGAGCAAACG                                                         +1485 <br />                 H       H   P   N   L   Y        F       E     R   S   R   Q       L   Q       K   K   Q   Q   K   L   E   K   E     K   L   F   N   N G     N   H       528 <br />                CATATCGCGCATCCTAACCTATACTTCGAAAGATCAAGGCAACTGCAAAAGAAACAGCAGAAGCTGGAAAAGGAAAAACTATTCAATAATGGTAATCAT                                                         +1584 <br />                tAGTGCAGAAAGGGAAAAGGAAAAAAGCGAAGAGATAACTAAAAAGGATTAATAGTATACGTAATTTAGTCTACTGTAAGTATGGTCAAAAATTTTCTT +1683 <br />                TAGTGAGAAAAAAAAAAAAAAAGAGAAAGCATAAACGCTGTACAATACCAGTGACTGCACTGCTCCGAAAACGTCCGAGATATGTATATTGTCCGAATT +1782 <br />                TTTTCATACTTTAATAGCTGGCCAGAGAAACCTTATAATTTTGTGGCTGCAAGCTATCGCGCTGTTCC.TCTGTGACAGGCAACACAGAACAACTAACCG                                                        +1881 <br />                CCTGTCTTTTATGCCTGTACACTTCATTAAAATTTACCGTTTTCTTCCATTACAAGCGGGGACACTTTCCGTTTAAACCTGCAATATGTTGCAGGGCCT                                                         +1980 <br />                ATCCACGCCAATTTACTGTGGCAGAAGTTACTGCAGTGGAAGAACTACCTTTTCCAGCATAGCCTCCGCCACCAAAAAATGCAGGTGCTGCGTTTTCTG                                                         +2079 <br />                GATACGTGGCTGGCCTTTAAAACGCATGGGAGGGACAGTATCTCAC <br />    FIG. 6. Nucleotide sequence deduced amino acid sequence PR12. nucleotide sequence numbered ATG initiation <br /> codon PR12 open reading frame (position + 1). amino acid sequence translated DNA sequence; amino acids  single-letter code) numbered methionine. Sequences 5&apos; noncoding region POL1  text) underlined. -*, Putative positions major 5&apos; endpoints PR12 transcript. <br />  VOL. 9, 1989                                                             LARGE SUBUNIT YEAST DNA PRIMASE                        3085 <br />  <br />  <br />                                                                                         TI                        T2 <br />              C TG              1 2 3 <br />  <br />  <br />  <br />  <br />                                                                                                      B <br />  <br />  <br />  <br />  <br />   FIG. 7. RNase protection mapping 5&apos; endpoint mature PR12 mRNA. 5&apos; termini PRI2 transcript analyzed RNase protection assay (24) 32P-labeled RNA <br /> probe complementary PR12 mRNA spanning positions                                                          <br />  <br />  <br />  <br /> -520 +141. Lanes 1. 2. 3 results obtained 10. <br /> 20, 40 jig yeast poly + RNA. respectively. ladder known DNA sequence run parallel used size marker. major signals (*-) correspond positions -69 5. -60 + 5. <br />                                                     &#194;&#177; <br />  <br /> -52 5. -49 5 ATG. <br />                    &#194;&#177;                                                   FIG. 8. Spore lethality morphology disruption PRI2. <br />                                                                      upper shows spore survival tetrads deriving                                                                      sporulation strain DESS heterozygous integration frame codes 528-amino-acid polypeptide calcu-           plasmid pCS15 PR12 (Ti) URA3 (T2) locus. Tetrads lated molecular weight 62,262, good agreement         dissected YPD plates (37) incubated 25&#194;&#176;C. Panels value 58,000 estimated electrophoretic          D pictures. taken light microscope 20 h                                                                      dissection. Ti tetrad. Spore survival scored 3 days. <br /> mobility p58 polypeptide polyacrylamide gel         viable spores tested <span id='am-46' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-47' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span>uracil</span> requirement.  C) <br /> presence sodium dodecyl sulfate (29). AUG context             Viable Ura- spores: (B D) inviable spores. <br /> does correspond consensus highly <br /> expressed yeast genes, 5&apos;-AAAAAAAUGUC-3&apos; (8). Fur- <br /> thermore, codon usage p58 gene reveals little,         similarity POLl striking, codon bias, usually seen poorly expressed yeast            -165 hexanucleotide immediately followed <br /> genes (36). observations agreement              direct repetition 5&apos;-CGCG-3&apos;, finding p58 low abundance (29).                           POLI repeats. <br />    p58 protein 35.8% nonpolar, 35.1% polar, 16.3%               Disruption PRI2 gene lethal yeast cells. positively charged, 12.8% negatively charged amino               showed PR12 gene product related DNA <br /> acids, represents random distribution            primase activity, expect essential cell <br /> types amino acids. hydrophilic profile p58             viability.  carried gene disruption exper- <br /> polypeptide (16) indicates protein rich          iment procedure Shortle et al. (38). <br /> hydrophilic regions. secondary structure predicted               DNA fragment internal p58-coding region (span- assisted analysis (6) shows regions              ning segment positions +141 +829) sub- helical structure 10 amino acids. helix-              cloned yeast integrating vector YIp5 (34), turn-helix motif, DNA-binding proteins              derivative plasmid pCS15 used disrupt PRI2 gene. <br /> (25), located residues 210 239.                             plasmid linearized digestion BglII SinaI <br />    RNase-mapping experiments (24) revealed presence           direct integration PRI2 URA3 locus, respec- major transcription starts, positions -49 5. -52&#194;&#177;       &#194;&#177;    tively, used transform diploid strain DES5 5, -60 5, -69 5 ATG (Fig. 7).          &#194;&#177;                                                           uracil prototrophy. Transformants heterozygous integra- <br /> convincing TATA box upstream major                tion single copy plasmid loci <br /> start sites, weakly expressed yeast genes         identified Southern blot analysis total yeast DNA clearly recognizable TATA homology. 5&apos;-               digested EcoRI probed X215 DNA insert. <br /> flanking region contained short repeated DNA se-             transformant class sporulated, tetrad <br /> quences, relevant control              analysis performed. Integration pCS15 PR12 <br /> transcription PRI2 gene genes            locus lethal spores (Fig. 8). fact, tetrads <br /> involved DNA metabolism. particular, sequence              derived diploid heterozygous integration <br /> 5&apos;-ACGCGT-3&apos;, present twice POLI (26)            showed 2:2 lethality linked URA3 gene CDC21 (40) 5&apos; noncoding regions CDC9 (1)             plasmid, viable spores Ura-. CDC8 (2), PRI2 position -165, similar          plasmid did contain sequences harmful se sequence, 5&apos;-TCGCGT-3&apos;, located 15 bp downstream.                cells, spore lethality caused integration  3086    FOIANI ET AL.                                                                                              MOL. CELL. BIOL. <br />  URA3 locus.  spore lethality         PRII, similar kinetics fluctuation tran- absence functional p58 polypeptide.           scripts cell cycle (reviewed reference 14). <br /> inviable germinated spores appeared single, elongated             520-bp sequence upstream PRI2 coding region <br /> cells, eventually buds (Fig. 8).         contains hexanucleotide 5&apos;-ACGCGT-3&apos;, pres- <br /> morphology, typical mutants impaired DNA           ent 5&apos; noncoding regions genes <br /> replication (9), observed germinated spores        listed (23, 26). Analysis 5&apos; noncoding region containing disrupted copy PRII gene (20).                CDC21, contains 5&apos;-ACGCGT-3&apos; repeats, indi- <br />                                                                   cates 37-bp deletion, removing region                         DISCUSSION                                reconstituting single hexamer, severely affects                                                                   level CDC21 transcript possibly impairs fluctuation <br />    Eucaryotic DNA replication highly regulated process       cell cycle (23). investigating requiring precise coordination protein-protein     6-bp sequence role mediating transcriptional protein-DNA interactions (4, 15). essential compo-        control POLl, PRII, PRI2. <br /> nent replicative apparatus DNA polymerase-             PRI2 gene present single copy haploid <br /> primase complex, generally considered         yeast genome, localized chromosome XI. enzymatic machinery involved synthesis          Disruption gene lethal cell;  p58 <br /> Okazaki fragments lagging strand replication        polypeptide performs essential function yeast cells, forks (4, 15). prerequisite understanding physical       expected protein species integral component <br /> structure protein complex examining mecha-        yeast DNA primase. production conditional lethal <br /> nisms controlling expression structural genes   mutants vitro mutagenesis PRII cloning genes encoding DNA polymerase-            PRI2 genes eventually establish roles DNA <br /> primase polypeptides. Cloning sequencing yeast         primase vivo helpful identifying structural <br /> genes POLI (DNA polymerase  PRII (48-kDa subunit            domains important functional interactions DNA primase) described (13, 19, 20, 26, 30).      different protein components polymerase-primase report, provide information genes       complex.  possibility obtaining large <br /> coding components yeast DNA polymerase-            amounts proteins overexpression cloned <br /> primase complex describing isolation partial           genes homologous heterologous systems characterization gene encoding large subunit        allow integration genetic data detailed <br /> DNA primase (58 kDa), called PRI2.                       biochemical characterization  <br />    PRI2 gene identified using specific <br /> anti-p58 antibodies immunological screening yeast <br /> genomic DNA library Xgtll (45). 2.6-kb yeast DNA                                ACKNOWLEDGMENTS <br /> insert positive recombinant phage X215 appears         thank Marco Muzi Falconi assistance analysis contain entire coding region gene,         Simonetta Piatti help synchronization yeast cell <br /> transcribed opposite direction respect lcZ.     cultures. <br /> fact, crude extracts prepared E. coli cells lysogenized         M.F. supported fellowship Fondazione  Buzzati X215 contain 58-kDa immunoreactive polypeptide.            Traverso. rapidly converted 50-kDa protein species; concentrations proteins increased IPTG-                              LITERATURE CITED <br /> treated cells. Lysates X215-infected cells affinity         1. Barker, D. G., J. H. M. White, L. H. Johnston. 1985. purify p58 antibodies specifically inhibit DNA primase           nLucleotide sequence DNA ligase gene (CDC9) Saccha- <br /> activity, confirming identity gene       roniv es cereviisiae: gene cell cycle regulated correlation p58 polypeptide DNA pri-             induced response DNA damage. Nucleic Acids Res. <br /> mase.                                                                 13:8323-8337. <br />                                                                    2. Birkenmeyer, L. G., J. C. Hill, L. B. Dumas. 1984. Saccha- <br />    X215 yeast DNA insert identifies single 1.7-kb               romyiv(cs cerei&apos;isiae CDC8 gene product. Mol. Cell. Biol. <br /> mRNA species used probe Northern blots                  4:583-590. <br /> yeast poly + RNA. size sufficient encode            3. Botstein, D., S. C. Falco, S. E. Steward, M. Brennen, S. Scherer, <br /> 58-kDa polypeptide, direction transcription             D. T. Stinchcomb, K. Struhl, R. W. Davis. 1979. Sterile host BglII NcoI, expected results A215              yeast (SHY): eukaryotic biological containment protein analysis.                                                     recombinant DNA experiments. Gene 8:17-24. <br />   Nucleotide sequence determination X215 DNA                4. Campbell, J. L. 1986. Eukaryotic DNA replication. Annu. Rev. <br /> insert confirmed contains entire PRI2 coding              Biochem. 55:733-771. <br />                                                                    5. Carlson, M., D. Botstein. 1982. differentially regulated <br /> region, translated 528-amino-acid polypeptide           mRNAs different 5&apos; ends encode secreted intracellular calculated molecular weight 62,262. protein          forms yeast invertase. Cell 28:145-154. rich hydrophilic regions, predicted secondary      Sa.Foiani, M.,  J. Lindner, G. R. Hartmann, G. Lucchini, P. <br /> structure shows helix-turn-helix motif,         Plevani. 1989. Affinity labeling active center ribonu- <br /> DNA-binding proteins (6). assisted analysis did              cleoside <span id='am-21' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-22' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span>triphosphate</span> binding site yeast DNA primase. J. significant homology p58 known             Biol. Chem. 264:2189-2194. <br /> proteins. procaryotic DNA primases com-              6. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis pared, PRI2 eucaryotic gene encoding           accuracy implication simple methods predicting <br /> large primase subunit cloned sequenced.             secondary structure globular proteins. J. Mol. Biol. <br />                                                                       120:97-120. <br />   Preliminary analysis steady-state level 1.7-kb     7. Goto, T., J. C. Wang. 1984. Yeast DNA topoisomerase II mRNA synchronized yeast cells indicates level          encoded single-copy, essential gene. Cell 36:1073-1080. <br /> PRI2 transcript varies cell cycle, reaching peak      8. Hamilton, R., C. K. Watanabe, H.  Boer. 1987. G1 S-phase transition. Interestingly, DNA             Compilation comparison sequence context synthesis genes, CDC8. CDC9, CDC21, POLI,                 AUG start codons SaccharoinNces cerev&apos;isiae mRNAs. <br />  VOL. 9, 1989                                                                  LARGE SUBUNIT YEAST DNA PRIMASE                           3087 <br />  <br />     Nucleic Acids Res. 15:3581-3593.                                          polymerase-DNA primase complex. p. 341-346. T. Kelly  9. Hartwell, L. H. 1971. Genetic control cell division cycle       B. Stillman (ed.). Cancer cells, vol. 6. Cold Spring Harbor <br />     yeast. J. Mol. Biol. 59:183-194.                                          Laboratory. Cold Spring Harbor, N.Y. <br /> 10. Hattori, M., Y. Sasaki. 1986. Dideoxy sequencing method           28. Plevani, P., M. Foiani, M. Muzi Falconi,  Pizzagalli, C. <br />     using denatured plasmid templates. Anal. Biochem. 152:232-                Santocanale, S. Francesconi, P. Valsasnini,  Comedini, S. <br />     238.                                                                      Piatti, G. Lucchini. 1988. yeast DNA polymerase- <br /> 11. Henikoff, S. 1984. Unidirectional digestion exonuclease III          primase complex: genes proteins. Biochim. Biophys. Acta <br />     creates targeted break points DNA sequencing. Gene 28:                951:268-273. <br />     351-359.                                                              29. Plevani, P., M. Foiani, P. Valsasnini, G. Badaracco, E. Cheria- <br /> 12. Hinnebusch,  G., G. R. Fink. 1983. Repeated DNA                     thundam, L. M. S. Chang. 1985. Polypeptide structure     sequences upstream HISI occur                  DNA primase yeast DNA polymerase-primase complex. <br />     coregulated genes Saccharomvces cerei isiae. J. Biol. Chem.            J. Biol. Chem. 260:7102-7106. <br />     258:5238-5247.                                                        30. Plevani, P., S. Francesconi, G. Lucchini. 1987. nucleo- <br /> 13. Johnson, L. M., M. Snyder, L. M. S. Chang, R. W. Davis,               tide sequence PRII gene related DNA primase     J. L. Campbell. 1985. Identification gene encoding yeast           Saccharonixvces cerei&apos;isiae. Nucleic Acids Res. 15:7975-7989. <br />     DNA polymerase  Cell 43:369-377.                                    31. Plevani, P., G. Lucchini, M. Foiani, P. Valsasnini,  Brandazza, <br /> 14. Johnston, L. H., J. H. M. White,  L. Johnson, G. Lucchini,              M. Bianchi, G. Magni, G. Badaracco. 1987. Structure     P. Plevani. 1987. yeast DNA polymerase transcript            function yeast DNA polymerase DNA primase com- <br />     regulated mitotic cell cycle meiosis       plex. Mol. Genet. 6:53-60. <br />     induced DNA damage. Nucleic Acids Res. 15:5017-5030.            32. Roeder, G. S., G. R. Fink. 1980. DNA rearrangements <br /> 15. Kaguni, L. S.,  R. Lehman. 1988. Eukaryotic DNA                     associated transposable element yeast. Cell 21:239- <br />     polymerase-primase: structure, mechanism function. Bio-               249. <br />     chim. Biophys. Acta 950:87-101.                                       33. Sanger, F., S. Nicklen,  R. Coulson. 1977. DNA sequenc- <br /> 16. Kyte, J., R. F. Doolittle. 1982. simple method                  ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br />     displaying hydropathic character protein. J. Mol. Biol.          USA 74:5463-5467. <br />     157:105-132.                                                          34. Scherer, S., R. W. Davis. 1979. Replacement chromo- <br /> 17. Laemmli, U. K. 1970. Cleavage structural proteins           segments altered DNA sequences constructed     assembly head bacteriophage T4. Nature (London)                 *itro. Proc. Natl. Acad. Sci. USA 76:4951-4955. <br />     227:680-685.                                                          35. Schwartz, D. C., C. R. Cantor. 1984. Separation yeast <br /> 18. Langford, C. J., F. J. Klinz, C. Donath, D. Gallwitz. 1984.           chromosome-sized DNAs pulsed field gradient gel electro- <br />     Point mutations identify conserved. intron-contained TAC              phoresis. Cell 37:67-75. <br />     TAAC box essential splicing sequence yeast. Cell             36. Sharp, P. M., T. M. F. Tuohy, K. R. Mosurski. 1986. Codon <br />     36:645-653.                                                               usage yeast: cluster analysis clearly differentiates highly 19. Lucchini, G.,  Brandazza, G. Badaracco, M. Bianchi, P.              lowly expressed genes. Nucleic Acids Res. 14:5125-5143. <br />     Plevani. 1985. Identification yeast DNA polymerase           37. Sherman, F., G. R. Fink, J. B. Hicks. 1986. Methods     gene antibody probes. Curr. Genet. 10:245-252.                       yeast genetics: laboratory course manual. Cold Spring Harbor <br /> 20. Lucchini, G., S. Francesconi, M. Foiani, G. Badaracco, P.             Laboratory, Cold Spring Harbor, N.Y. <br />     Plevani. 1987. yeast DNA polymerase-primase complex:              38. Shortle, D., J. E. Haber, D. Botstein. 1982. Lethal disruption <br />     cloning PRII, single essential gene related DNA primase           yeast actine gene integrative DNA transformation. <br />     activity. EMBO J. 6:737-742.                                              Science 217:371-373. <br /> 21. Lucchini, G., C. Mazza, E. Scacheri, P. Plevani. 1988.            39. Smith, D. B., P.  Fisher. 1984. Identification, develop- <br />     Genetic mapping Saccharonvyces cereviisiae DNA poly-               mental regulation, response heat shock antigeni- <br />     merase gene characterization poll temperature-                 cally related forms major nuclear envelope protein     sensitive mu <br /> </body></html>