<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Mar. 1987, p. 1180-1192                                                                     Vol. 7,  3 <br /> 0270-7306/87/031180-13$02.00/0 <br /> Copyright &#194;&#169; 1987, American Society Microbiology <br />  <br />  <br />   RAD4 Gene Saccharomyces cerevisiae: Molecular Cloning      Partial Characterization Gene Inactivated                           Escherichia coli <br />          REINHARD FLEER, CHARLES M. NICOLET,t GLENN  PURE,t ERROL C. FRIEDBERG* <br />               Department Pathology, Stanford University Medical Center, Stanford, California 94305 <br />                                         Received 11 August 1986/Accepted 24 November 1986 <br />  <br />             contrast Saccharomyces cerevisiae RAD genes involved <span id='am-1' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> excision repair DNA,          RAD4 gene isolated screening yeast genomic library recombinant plasmids          complement UV sensitivity rad4 mutants (Pure et al., J. Mol. Biol. 183:31-42, 1985).          attempted walk RAD4 neighboring SPT2 gene obtained integrating derivative plasmid <br />          isolated Roeder et al. (Mol. Cell. Biol. 5:1543-1553, 1985) contains 4-kilobase fragment yeast <br />          DNA including mutant allele SPT2. integrated different rad4 mutant strains,          plasmid (pR169) complements UV sensitivity frequency -10%.  centromeric plasmid <br />          containing rescued sequences include flanking yeast DNA longer complements phenotype rad4 <br />          mutants. Complementing activity restored vivo repair defined gap centromeric plasmid. <br />          repaired plasmid fully complements UV sensitivity rad4 mutants tested isolated directly <br />          yeast cells, plasmid propagated Escherichia coli complementing activity lost.          mapped physical location RAD4 gene insertional mutagenesis transcript mapping. <br />          gene -2.3 kilobases size located immediately upstream SPT2 gene. genes          transcribed direction. RAD4 essential gene, increased transcription gene          observed cells exposed DNA-damaging agent 4-nitroquinoline-1-oxide. site inactivation RAD4 <br />          particular plasmid propagated E. coli localized 100-base-pair region gene disruption gap <br />          repair experiments. addition, identified approximate locations chromosomal rad4-2, <br />          rad4-3, rad44 mutations. <br />  <br />    excision base damage, causes conforma-                     Prakash S. Prakash, personal communication).  <br /> tional distortion secondary structure DNA                combinant plasmid (pNF4000) partially restores UV produced UV irradiation bulky chemicals),               resistance particular mutant strain designated rad4-3 <br /> complex biochemical process generally referred nucle-                (29).  rad4-3 allele contains ochre mutation, <br /> otide excision repair (8). yeast Saccharomyces cere-                partial complementation observed result visiae 10 genes involved repair process (9,           overexpression normal yeast tRNAGiA gene present 11). Mutations genes render cells abnormally sensi-               multicopy plasmid pNF4000 (29). <br /> tive DNA damage UV radiation defective                           RAD4 gene mapped chromosome V S. <br /> various degrees) ability incise UV-irradiated DNA              cerevisiae genetic analysis tightly linked excise pyrimidine dimers vivo (9, 11).             SPT2 locus  SPM2) (36). mutant allele genes (RADI, RAD2, RAD3, RAD4, RADIO) abso-                        gene (spt2-1) -3 kilobases (kb) lutely required excision repair UV-irradiated DNA                upstream flanking sequences isolated Roeder et al. <br /> yeast, strains mutations genes totally <br /> defective pyrimidine dimer excision detect-                 recombinant plasmid called pR140 (32). study able incision DNA incubation UV irradiation                provide evidence integrating derivative plas- <br /> (9, 30, 35).                                                               mid designated pR169 (G. S. Roeder, personal communica- <br />    Previous studies laboratory resulted                   tion) contains entire RAD4 gene. Integration  homol- <br /> isolation recombinant plasmids containing yeast                     ogous recombination) plasmid genome RADI, RAD2, RAD3, RADIO genes (22, 24, 34, 38).                        various UV-sensitive rad4 mutant strains generated UV- isolated screening yeast genomic library               resistant derivatives significant frequency.  <br /> autonomously replicating multicopy plasmids comple-                  appropriate sequences integrant strain <br /> ment UV sensitivity selected rad mutants. Similar                   rescued subcloned centromeric plasmid, results reported (12-14, 28, 39)                   complementing activity lost. ability complement <br /> include recent isolation RAD7 gene (27).                UV sensitivity rad4 mutants tested restored <br /> extensive screening yeast libraries failed            repair vivo gap generated putative RAD4 <br /> yield plasmids containing functional RAD4 gene (29; L.                   gene centromeric plasmid. gap-repaired <br />                                                                            plasmid isolated directly yeast propagated                                                                            various Escherichia coli strains,  plasmids  <br />   * <br />     Corresponding author.                                                  covered lost complementing activity.   t Present address: 32/93 Chewings Street, Scullin C.T. 2614,           conclude plasmids containing functional RAD4 gene <br /> Australia. <br />    t Present address: Department Physiological Chemistry, Uni-          propagated E. coli suffering inactivation <br /> versity Wisconsin, Madison, WI 53706.                                   gene. <br />                                                                     1180 <br />  VOL. 7, 1987                                                                            R       CENEVIS. <br />                                                                                                        CEREVISIAE               1181 <br />  <br />                  TABLE 1. S. cerevisiae strains                      YCp5O BgllI site BglII PvuI sites, <br /> Strain         Relevant genotype                  Source          respectively, inactivated filling termini E. <br />  mutant                                                              coli DNA polymerase (Klenow fragment) religating YNN218      RAD his3A200 ade2-101          R. Davis, Stanford      blunt-ended plasmids. Tailoring BglII site results               ura3-52 lys2-801                 University            generation new ClaI site does interfere SX46a       RAD his3-532 ade2 ura3-52      laboratory stocks   capacity high-frequency transformation endowed               trpl-289                                               ARS function (4). Tailoring PvuI site yields bacterial <br /> S288C       RAD gal2                       laboratory stocks   colonies Amps Tetr. derivative YCpS0 radl-J      radl-J his3-532 ade2 ura3-     E. Yang, labora-   used gap repair experiments. plasmid (YCpS1) <br />               52 trpl-289                      tory                  contains -0.8-kb deletion Tetr gene YCp5O, <br /> rad2-1      rad2-1 his3-532 ade2 ura3-     L. Naumovski,      removes BamHI, SphI, SalI sites leaves <br />               52 trpl-289                      laboratory            EcoRl HindIll sites intact. Transformation yeast <br /> rad3-1      rad3-1 his3-532 ade2 ura3-     L. Naumovski,               52 trpl-289                      laboratory            strains plasmid DNA carried treating cells <br /> rad4-2      rad4-2 his3-532 ade2 ura3-52   G. Pure, labora-   lithium acetate promote DNA uptake (17). Transfor- <br />               trpl-289                         tory                  mation E. coli carried previously described <br /> rad4-3      rad4-3 his3-532 ade2 ura3-     G. Pure, labora-   (22). <br />               52 trpl-289                      tory                     Plasmids generally extracted E. coli rad44       rad44 his3-532 ade2 ura3-52    G. Pure, labora-   modification (19) alkaline lysis procedure Bimboim <br />                                                tory                  Doly (2). rapid screening plasmid DNA, lysates <br /> rad4-10     rad4-10 his3A 200 ade2-101     R. Fleer, study    prepared procedure Holmes Quigley (15) <br />               lys2-801                                               subjected electrophoresis 0.7% agarose gels <br /> rad4-11     rad4-11 his3-532 ade2 trpl-    R. Fleer, study <br />               289                                                    prior purification. scaled version alka- <br />                                                                      line lysis procedure (19) used isolate chlorampheni- <br />                                                                      col-amplified plasmid DNA followed purification <br />                                                                      cesium chloride-ethidium bromide gradients. Yeast <br />                                                                      genomic DNA prepared spheroplasts described <br />                                                                      previously (34), ethanol-precipitated DNA    mapped physical location RAD4 gene             additionally treated 50 ,ug DNase-free RNase ml immediately flanks STP2 gene           37&#194;&#176;C 30 min. Cesium chloride-ethidium bromide gra- genes transcribed direction. site        dient centrifugation plasmid DNA, restriction enzyme <br /> inactivation RAD4 particular plasmid propagated E.        digestion, gel electrophoresis, size fractionation DNA <br /> coli localized 100-base-pair (bp) region        fragments sucrose gradients, general recombi- coding sequence gene disruption vivo repair         nant DNA methods followed standard protocols (5, 19). 32p plasmids containing defined gaps. Finally,           labeling DNA carried using mixed random <br /> unlike RAD2 (18, 31) enhanced expression RAD4 gene            primers prime DNA synthesis method Feinberg induced exposure cells DNA-damaging           Vogelstein (7). <br /> agent 4-nitroquinoline-1-oxide unlike RAD3 (13, 23)            Isolation purified plasmid DNA yeast cells. Cells essential gene.                                         grown early stationary phase minimal  <br />                                                                      dium. Cells washed, suspended 1 M sorbitol,                 MATERIALS METHODS                                converted spheroplasts 300 ,ug zymolyase ml                                                                      presence 25 mM EDTA, 50 mM phosphate, 1 p.l   Yeast bacterial strains culture conditions. Strains        P-mercaptoethanol ml. washing 1 M sorbitol, transformation plasmids containing URA3 gene            spheroplasts  250-ml culture) suspended 2.5 selectable marker originally constructed            ml 1 M sorbitol, equal volume solution 50 <br /> laboratory crossing rad mutants strain SX46a,                  mM glucose-25 mM Tris hydrochloride (pH 8.0)-10 mM <br /> sporulating diploids, isolating appropriate haploid          EDTA added. Subsequent steps described strains. genotypes yeast strains           large-scale alkaline lysis E. coli preparations (19), used listed Table 1.                                          resulting potassium acetate supematants pre- <br />    Yeasts bacteria grown previously described            cipitated 23&#194;&#176;C 15 min adding 14 ml isopropanol. <br /> (38). Yeast strains transformed plasmids grown          dissolved precipitates treated 50 ,ug RNase <br /> synthetic medium lacking <span id='am-16' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-17' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="Uracil" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>uracil</span> maintain selection            ml 37&#194;&#176;C 20 min 150 ,ug proteinase K  plasmid-bearing cells. YPD medium (1% yeast extract, 2%              0.5 M NaCl, 0.5% Sarkosyl, 25 mM EDTA) ml 60&#194;&#176;C <br /> Bacto peptone [Difco], 2% glucose) used nonselec-            1 h. centrifugation 5 min Eppendorf <br /> tive growth measurement cell viability UV           table centrifuge, supernatants ethanol precipitated <br /> irradiation. selection E. coli transformants cells       -70&#194;&#176;C 10 min, pellets dissolved grown L broth supplemented ampicillin (50 ,ug/ml)         purified cesium chloride-ethidium bromide gradients. Cen- <br /> tetracycline (15 ,ug/ml) appropriate. isolation            trifugation Ti5O fixed-angle rotor 42,000 rpm plasmid DNA E. coli cells grown L broth (19)              48 h, followed rebanding plasmid-containing fractions chlorampenicol (200 p.g/ml) added absor-             VTi65 vertical rotor 55,000 rpm 12 16 h. Yeast <br /> bance 600 nm 0.55. Incubation continued 20 h.          2 ,um plasmid DNA accounted 90%    Cloning vectors transformation procedures. shut-          resulting plasmid preparations, served localize tle vectors YIpS  integrating plasmids) YCpSO            plasmid band UV light. yield single-copy <br /> centromeric plasmids) carrying yeast URA3 gene              centromeric plasmid -600-800 ng DNA liter kindly provided R. W. Davis, Stanford University.             stationary-phase culture. Assuming single copy integrating plasmid pR169 gift G. S. Roeder, Yale         14-kb plasmid cell (-1.5 x 10-11 ,ug DNA), liter University. Plasmids YCp52 YCp53 derivatives              yeast culture grown minimal medium (density, -8 x 107 <br />  1182    FLEER ET AL.MOL.MOL. CELL. BIoL. <br />                                    BIOL. <br />  <br /> cells ml) contains -1.2 ,ug plasmid DNA. Based         ing blunt-ended XbaI site generate plasmid pNF420A calculations recovery centromeric plasmid DNA       (Fig. 1). Linear fragments (HindIII-XmnI, 4.5 kb; HindIII- yeast cells -50 70%.                                 PvuI, 4.6 kb; HindIII-HpaI, 4.85 kb) used transforn <br />    Measurement UV survival yeast strains. Quantitative     strain SX46a. <br /> survival transformed yeast strains UV irradiation          Transcript mapping. RNA isolated Rad+ measured described previously (22). rapid             yeast strain S288C previously described (25) used <br /> screening UV-irradiated transformants small amounts        map RAD2 transcriptional start site modification <br /> single yeast coloties individually suspended 0.5 ml      (6) method Berk Sharp (1). 1.65-kb BglII- sterile water, 5-,ul samples applied YPD agar     XhoI fragment including 5&apos; end RAD4 gene plates regular pattern. plates (containing similarly    end labeled BglII site hybridized 4 h treated samples Rad+ strain SX46a Rad- rad4         100 ,ug total RNA 10 pu 90% recrystallized formam- <br /> mutant positive negative controls, respectively),    ide. optimal temperature hybridization (43&#194;&#176;C) irradiated 40 J/m2 incubated 30&#194;&#176;C 36 h.        optimal S1 nuclease concentration (1,500 U/ml) growth UV-sensitive (Rad-) mutants completely abol-        determined empirically. <br /> ished conditions, repair-proficient <br /> (Rad+) cells visible confluent colonies. semiquan-                                RESULTS <br /> titative survival test performed similar manner, cells different strains applied agar       Complementation UV sensitivity rad4 mutants plates horizontal rows 5-,u samples given suspen-     plasmid pR169. Plasmid pR169 integrating plasmid <br /> sion, vertically aligned spots irradiated 0, 10,   containing 4.0-kb yeast chromosomal insert includes <br /> 20 40 J UV light m2 respectively.                      spt2-1 allele near end (Fig. 1). region    Gap repair yeast. Repair yeast centromeric plas-     leftmost HindIll site BglII site identified Fig. <br /> mids defined gaps carried described           1 sequenced Roeder et al. (32), identified Orr-Weaver et al. (26) modified Naumovski et al.          complete incomplete open reading frames (ORFs) <br /> (21). gap PvuI BamHI sites present      (Fig. 1). complete ORFs contains coding <br /> insert pNF402 (Fig. 1) generated complete diges-       region spt2-1 gene. small overlap- <br /> tion BamHI partial digestion PvuI, followed        ping unassigned ORfs, 257 155 amino acids length. isolation appropriate DNA fragment agarose gel      incomplete ORF present end sequenced <br /> electrophoresis. Stable Ura+ transformants screened         fragment unassigned (32). DNA-RNA hybridiza- UV resistance, presence centromeric plasmids      tion analysis revealed transcripts encoded. se- confirmed demonstrating cosegregation UV resist-       quences pR169. (-1.4 kb length) ance Ura+ phenotype growth nonselective        spt2-1 transcript (32). transcript -2.6 kb size, <br /> medium. Gaps plasmids generated unique      position pR169 mapped (M. Smith, <br /> restriction sites, cells transformed complete      personal communication). detailed restriction map digests prior purification DNA fragments.             region including SPT2 locus -6 kb 5&apos;-flanking <br />    Gene transplacement. constructed series insertion     DNA shown Fig. 1. <br /> mutants using step gene transplacement method            view tight genetic linkage SPT2 Rothstein (33). Mutations generated inserting     RAD4 (36), considered possibility yeast URA3 gene  served selectable marker)       ORFs pR169 represent RAD4 selected restriction sites various plasmids. Linear      coding sequence. investigate integrated plasmid <br /> fragments containing URA3 disruptions                  pR169 rad4 mutant strains (rad4-2, rad4-3, <br /> transplaced genome selected Rad+ strains.            rad44); -10% stable rad4-2 rad4-3 transform- <br />      Disruption BamHI. Plasmid pNF402GR-EC1,           ants generated untargeted integration smaller <br /> rivative pNF402 (Fig. 1), cut unique BamHI site      number transformants generated targeted integra- insert ligated 2.5-kb BamHI fragment         tion showed wild-type levels UV resistance (Table 2). <br /> taining URA3 gene generate plasmid pNF404 (Fig. 1).        rad44 integrants showed enhanced <br /> Linear fragments (EcoRI, 9.1 kb; XmnI, 5.1 kb; HpaI-XmnI,        UV resistance (Table 2). initially thought incom- <br /> 4.75 kb) isolated 0.7% agarose gels used        plete ORF pR169 likely candidate gene <br /> transform strain SX46a (Rad+ ura3-52).                           encoding -2.6-kb transcript described    (ii) Disruption BglII. 4.6-kb HindlIl fragment      expected 5&apos; end located -1 kb upstream <br /> pNF402GR-EC1 cloned plasmid YIpS generate            BamHI site defines right end insert plasmid pNF411 (Fig. 1). 2.5-kb BamHI fragment car-          plasmid (Fig. 1).  interpretation rying URA3 gene   cloned unique       integration experiments incomplete ORF BglIj site plasmid generate plasmid pNF412 (Fig.      represent truncated RAD4 allele. Depending sites 1), linear 4.3-kb HpaI-XhoI fragment containing URA3       homologous recombination plasmid chromo- <br /> inserted BglII site used transform strain SX46a.   somal sequences relative location rad4-2    (iii) Disruption MstII. Plasmid pNF416 (Fig. 1) contains   rad4-3 mutations, integration reconstitute 1.lTkb SmaI fragment carrying URA3 gene cloned        complete wild-type copy RAD4 (Fig. 2A). failure blunt-ended uniique MqtII site pNF411. 3.9-kb           observe UV-resistant rad44 integrants expected SphI-XmnI linear fragment pNF416 (including -530 bp         rad44 mutation located region chromosomal pBR322 DNA end) used transform strains          gene represented pR169, complete wild-type <br /> SX46a YNN218 (Rad+ ura3-52).                                 gene reconstituted conditions (Fig. <br />    (iv) lpisruption XbaI SPT2 gene. Plasmid pNF411     2A). deleted 1.7-kb BglII-XhoI fragment, remov-         Plasmid rescue integrant strain yields noncom- <br /> ing XbaI sites. 2.2-kb HindIII-BamHI frag-        plementing derivatives. results described sug- <br /> ment containing URA3 gene cloned remain-        gested strategy rescuing plasmid containing entire <br />  VOL. 7, 1987                                                                                                                 RAD4 GENE S. CEREVISIAE   1183 <br />  <br />                                          0                        2                       4                              6        kb <br />                                         H                                                          H               H H <br />                                         H                                                          H               H H <br />                                         H                     H           H       H                H           H   H H       H <br />                                         t        H            H       HH H    H   $                            H <br />                                          ~       41           41       V~.                                              0 <br />  <br />  <br />                                                             .1I           f                                  111,1 <br />                                              H <br />                                                                     ~                                                                       H <br />                                                                                                                                                        H <br />                                                                                           IH IH <br />                                                                                           H <br />  <br />  <br />  <br />  <br />                                          H                                G                                                       Vector DNA: <br />                                          H X                  M       VG                   B <br />                        pRl 69                                                                                                           YIp5 <br />                                               4--      4-                 4~~---------- <br />                                              SPT2      ORFs                   ORF <br />                                        ER                             V                    B                                 E <br />                                         11                                                    l                                         YCp50 <br />                        pNF402 <br />                                                  (P)                  VG                  (B) <br />                        pNF417                                            II                                                              YCp53 <br />                                                                                                                                        (ABgIl,APvul) <br />                                                  (P)                  VG X                X            (H) <br />                        pNF418                     L                   II                                                           YCp53 <br />                                                                           B       U1RA3           2 .5 kbB                             (ABgIl,APvul) <br />                                        El <br />                                         II <br />                        pNF404                                                                                                           YCp51 <br />                                                                                                                                        (ABamHI) <br />                                          H X                  x           G                B                                                                                                                                     pNF411                                                                                                           YIp5 <br />                                                         B         UIR&quot;        2 . 5 kb B <br />  <br />                                                                                                  - <br />                        pNF412                                                                                                            YIp5 <br />                                                      UPA3 1. 1 kb <br />                                                     Sm           Sm <br />                                   Sp                                                       B               H <br />  <br />  <br />                        pNF416 46                                                                                                         YIp5 <br />                                (H) URA3 2. 2 kb (B)                      Np <br />  <br />                                                                    NJV G)X X B(O)H <br />                         pNF420A \\x\\                                                                                                    YIp5 <br />                                                                               A1.7 kb    FIG. 1. Physical maps recombinant inserts various plasmids used study. line shows detailed restriction map 6.6-kb region includes SPT2 gene upstream regions. sequenced region pR169 extends HindIII site BgllI <br /> site identified bar (H-G). ORFs identified region shown open boxes relative restriction maps. horizontal arrows represent direction translation ORFs inferred deduced <span id='am-9' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-10' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-11' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> sequences. dashed horizontal <br /> arrow indicates extension incomplete ORF, based assumption ORF encodes unassigned -2.6-kb transcript <br /> detected using fragments pR169 hybridization probes (M. Smith, personal communication). regions various plasmid inserts <br /> deleted gap repair linear transplacement genome identified open hatched boxes respectively. Plasmid vectors indicated right case. following abbreviations used: B, BamHI; E, EcoRI; G, BglII; H, HindIlI; Hp, HpaI; K, KpnI; <br /> M, MstII; N, XmnI; 0, XhoI; P, PstI; S, Sall; Sm, SmaI; Sp, SphI; V, PvuI; X, XbaI. Restriction sites parentheses destroyed blunt-end ligation. <br />  <br /> RAD4 gene. integration putative truncated                                     -15-kb fragment containing entire pR169 yeast insert <br /> RAD4 gene genome rad4-2 rad4-3 mutant,                                    -6 kb yeast chromosomal DNA right <br /> UV-resistant integrants contain complete copy                                  region essentially YIp vector sequences wild-type gene truncated copy containing                                     (Fig. 2A). vector sequences include pBR322- <br /> rad4-2 rad4-3 mutation (Fig. 2A). Restriction mapping                                derived origin replication Ampr gene, <br /> genomic DNA UV-resistant rad4-2 integrant                                       intramolecular ligation SalI fragments vitro followed carried Southern hybridization analysis pR169                               transformation E. coli yield plasmids containing probe indicated digestion Sall yield                             complete RAD4 gene (Fig. 2A). <br />  1184       FLEER ET AL.                                                                                                                                                MOL. CELL. BIOL. <br />  <br />       TABLE 2. Integration plasmids pR169 pNF400                                            TABLE 3. Transformation plasmids pNF401, pNF402GR                       various rad4 mutantsa                                                                     pNF402GR-EC1 various rad4 mutantsa <br />                      UV-resistant integrants plasmid (%):                                                           UV-resistant transformants plasmid (%): <br />                                                                                                        MIutant <br />                             pR169                                       pNF400                                             pNF401                  pNF402GR                    pNF402GR-EC1 <br /> Mutant <br />    Mutant <br />               Untargeted                Targeted           Untargeted <br />                                                                                 ~~~~~~~~~~~~Targeted <br />                                                                                                                                                                                               rad4-2             0 (0/50) <br />                                                                                                                           0 (0/50) <br />                                                                                                                                                    100 (18/18) <br />                                                                                                                                                     95 (21/22) <br />                                                                                                                                                                                     0 (0/75) <br />                                                                                                                                                                                     0 (0/75) <br />                          Bglll                                                                      rad4-3 <br />                     BamH(                                                                              rad4-4             0 (0/50)                 100 (3/3)                        0 (0/75) <br /> rad4-2         12 (7/60) 2 (1/50) 0 (0/169) 0 (0/21)                                                     numbers parentheses represent UV-resistant transformants rad4-3         10 (6/63) 2 (1/50) 0 (0/116) 0 (0/21)                                                   function total number Ura+ clones screened. text details. <br /> rad44           0 (0/92)                0 (0/70)            3 (3/114)            5 (6/114) <br />       numbers parentheses represent UV-resistant integrants function total number Ura+ clones screened. text details.                        plasmid (pNF401) contains 6.6-kb yeast insert                                                                                                        includes 2.6 kb rescued sequence contiguous insert <br />                                                                                                        pR169. Plasmid pNF401 did complement UV <br />    Using strategy isolated single Ampr E. coli                                               sensitivity rad4 mutant strains (Table 3). <br /> transformant containing plasmid expected size                                               Integration plasmid pR169 rad4-2 rescue restriction pattern, designated pNF400 (Fig. 2A). estab-                                            complete RAD4 gene pNF400 predicated lish presence entire RAD4 gene pNF400                                                 hypothesis plasmid missing 5&apos; end integrated plasmid rad4 mutant strains.                                            RAD4 (Fig. 2A).  considered alternative <br /> Cohtrary results obtained pR169, tJV-                                                   possibility pR169 contains complete mutant copy <br /> resistant colonies identified untargeted                                          RAD4, case integration rad4-2 strain <br /> targeted integration pNF400 rad4-2 rad4-3,                                             generate complete genes (Fig. 2B). integrative small fraction rad44 integrants UV                                                  recombination occurred chromosomal plas- <br /> resistant (Table 2). constructed autonomously                                               mid-borne mutations (crossover 2 Fig. 2B), replicating single-copy derivative plasmid pNF400.                                             integrant genes wild type  <br />                     SPT2    ORF          RAD4    ?                                                                        SPT2        RAD     ? <br />   A_                                                                                                    B <br />                   HX            M     VG&#194;&#174;B                 E     S                                                       HX           M <br />                                                                                                                                           *VG~ <br />                                                                                                                                             *q B             E     S <br />  <br />  <br />  <br />  <br />                                  vector      HX        M VG           B          E     S             H X        M VG          BS <br />                                                                                                                                        vector   HX        OVG           B    &#194;&#174;      E   S <br />   H X        M VG          BS <br />  <br />               truncated gene                                   complete gene <br />                                                                                                                  c <br />                                                                                                                  complete g. <br />                                                                                                                       l. gene                              l <br />                                                                                                                                                          complete ge <br />                                                                                                                                                                   gene <br />  <br />                                                   15 kb Sall fragment                                                                                15 kb Sall fragment <br />  <br />  <br />                                                                                                                H X        M VG          B          E      S                                                       HX         MVGM         B           E       S <br />  <br />  <br />  <br />  <br />                                                                         pF400                                                                                          pNF400 <br />  <br />  <br />  <br />    FIG. 2. Cloning chromosomal DNA sequences contiguous yeast insert plasmid pR169. Panels B diagrammatically <br /> illustrate alternative models location RAD4 gene relative SPn2 locus chromosome insert pR169. model panel assutnes plasmid pR169 contains truncated RAD4 allele, model shown panel B based hypothesis entire RAD4 gene present plasmid, mutated. Homologous chromosomal plasmid sequences identified boxed regions RAD4 alleles hatched. circled M represents hypothetical locations sites mutation(s) chromosomal plasmid RAD4 alleles  . Integration plasmid occur homologous recombination mutations (numbered crossovers), yielding UV-resistant UV-sensitive integrants.  products homologous <br /> recombination shown regenerate wild-type RAD4 allele result UV-resistant integrants (crossover  2 case). Digestion chromosomal integrant DNA Sall subsequent intramolecular ligation generates plasmid pNF400. Note inversion relative positions chromosomal plasmid-borne mutations B generate UV-resistant integrant containing wild-type RAD4 allele 15-kb Sall fragment, mutant allele shown figure. case, plasmid pNF400 rescue wild-type RAD4 gene, inactivated subsequent propagation E. coli. abbreviations restriction <br /> sites shown Fig. 1. <br />  VOL. 7, 1987                                                                              RAD4 GENE S. CEREVISIAE                  1185 <br />  <br />                                                                        mutation(s) pNF400 centromeric deriv- <br />                                                                        ative pNF402. <br />                                                                           transformation gapped centromeric plas- <br />                                                                        mid, -10% Ura+ transformants showed wild-type <br />                          PvuI         amIC <br />                                                     UVs                levels UV resistance. growing independent <br />                                                                        transformants nonselective conditions, observed <br />                                                                        strict cosegregation UV resistance Ura+ pheno- <br />                                                                        types case, indicating UV resistance deter- <br />                          (pNF42                                        minant plasmid borne. results demonstrate success- <br />                                                                        ful repair gapped centromeric plasmid pNF402                                                                        support contention plasmid contains muta- <br />                                 7      GAP-                            tion(s) present equivalent region rad44 <br />                                        REPAIR                          allele. results rad44 mutation                                                                        located outside PvuI-BamHI interval yeast <br />                                                                        genome (Fig. 3). observation -10%                                                                        stable rad44 transformants UV resistant indicates                                                                        efficiency gap repair rad44 strain limited. <br />                                                                        result rescue chromosomal rad44 <br />                                                        UVR             mutation extension PvuI-BamHI gap                                                                        exonucleolytic degradation vivo (Fig. 3). Alternatively,  <br />                            pNr                                                     <br />    FIG. 3. Reconstitution wild-type RAD4 gene vivo gap                                                N- <br /> repair mutant rad4 plasmid allele, using rad4 mutant                                                     recipient strain. Chromosomal plasmid RAD4 alleles shown                                                            N&apos; hatched open boxes, respectively, circled Ms <br /> representing hypothetical locations mutations alleles. recipient UV-sensitive (UVs) strain transformed linearized plasmid (pNF402) containing deletion removes plasmid mutation(s). double-strand gap repaired gene <br /> conversion homologous chromosomal sequences, Ura+ <br /> transformants screened UV resistance. UV-resistant (UVR) <br /> transformants arise repaired gap does include                             \ <br /> chromosomal mutation.  vivo extension gap              IV <br /> limit efficiency productive gap repair. efficiency gap   0 <br /> repair limited plasmid-bome mutation situated <br /> outside close gap. event, productive repair occur instances vivo extension gap <br /> removes plasmid-borne mutation.                                                                                                                                        U <br />  <br />  <br />  <br />                                                                        o 10 <br />                                                                        contain mutations (Fig. 2B). Rescue mutant allele account results observed plasmids                   5; <br /> pNF400 (Table 2) pNF401 (Table 3).                                                                      U\ <br />   Gap repair rescued sequences. test idea           ct) putative RAD4 genes plasmids pR169, pNF400, pNF401 represent mutated alleles, replaced portion                   io-4 cloned gene transfer homologous sequences yeast genome using modification (21) gap <br /> repair technique Orr-Weaver et al. (26) (Fig. 3). centromeric derivative plasmid pNF401 containing unique BamHI site yeast insert constructed yield plasmid pNF402 (Fig. 1), 1.6-kb region                      10-5 <br /> tween BamHI PvuI sites plasmid deleted.                                         20        40                       60 carried gap repair pNF402 rad4 mutant                                      UV FLUENCE (J /m2) <br /> UV-sensitive cells screened transformation UV resistance (Fig. 3). detection repaired plasmids               FIG. 4. Survival transformed yeast strains various UV <br /> containing functional RAD4 gene obviously requires              doses. Cells transformed relevant plasmids grown chromosomal sequences homologous deleted plas-                minimal broth, harvested centrifugation, spread mini- <br />                                                                        mal agar plates appropriate dilutions 0.07 M phosphate buffer. <br /> mid sequences free mutations. end used           Plates exposed UV irradiation indicated incubated <br /> rad44 strain, demonstrated ability generate               5 days 30&#194;&#176;C colonies quantitated. Duplicate UV-resistant transformants integration               symbols represent independent experiments. Symbols: 0, RAD4 <br /> plasmid pNF400 strain (Table 2) suggested           (YCp5O); O, rad44 (YCp5O); 0, rad44 (pNF400GR); *, rad44 <br /> rad44 mutation located site distinct putative         (pNF400GR-EC). <br />  1186     FLEER ET AL. <br />  <br />  <br />  <br />  <br />                   b <br />          c <br />                    <br />                   H <br />                   H <br />                   H <br />                   V <br />                   .I4 <br />                   _ <br />                       4 <br />                           SPT2 <br />  <br />                            H <br />                            . <br />  <br />  <br />  <br />                           L                            X <br />                                    -   -4 <br />                                             pR1 69 <br />                                             s <br />                                          ORFs <br />                                               H <br />                                               H <br />                                               4J <br />  <br />  <br />  <br />                                        -JKVK,Ctkqqm <br />                                                     4 <br />                                                                                                           c <br />                                                           .1 <br />  <br />  <br />  <br />  <br />                                                            \:l\K\\\\ I II <br />                                                                 <br />  <br />                                                                U <br />                                                                II <br />                                                                II <br />                                                                    ~ <br />                                                                    H H H <br />                                                                    &gt; <br />                                                                                                                                            pNF402 <br />                                                                         - <br />  <br />  <br />  <br />  <br />                                                                         :U: <br />                                                                               4 <br />                                                                                   O0 <br />                                                                                   ORF <br />                                                                                   H <br />                                                                                   - <br />  <br />  <br />                                                                                                                                                                           H <br />                                                                                         .4 <br />                                                                                         ff <br />                                                                                                                                                                                 CJ <br />                                                                                              1 <br />  <br />                                                                                              H <br />                                                                                              m <br />                                                                                              9 <br />  <br />                                                                                              m <br />  <br />  <br />  <br />  <br />                                                                                          BamHJ1 <br />  <br />  <br />                                                                                                                                                                                                H <br />                                                                                                   0 <br />  <br />  <br />                                                                                                   x m <br />                                                                                                       H <br />                                                                                                       H <br />                                                                                                       H <br />                                                                                                       XI <br />  <br />  <br />                                                                                                        II <br />                                                                                                            H <br />  <br />                                                                                                            X <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                                                                                                                                       <br />  <br />  <br />  <br />                                                                                                                        H <br />                                                                                                                        0 <br />                                                                                                                        U <br />                                                                                                                         C) <br />                                                                                                                                                                                                                                                                UV-sensitive <br />                                                                                                                                         disruption <br />                                                                                                                                          mutants: <br />  <br />  <br />  <br />                                                                                                                                        70% (34/50) <br />                                                                                                                                         3% (3/92) <br />                                                                                                                                         0 (0/200) <br />                                                                                                                                                     MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                           BglII <br />          d                                                                                                                           95% (48/50) <br />                                                      III!!!! <br />                                             mst I: <br />                                                                : <br />           e- <br />           Af                                                                                                                                                                        95% (89/93) <br />          f <br />                       ,   XbaI-,,.01 <br />                                                                                                                                                                         n      (n/inni <br />   FIG. 5. Mapping physical location RAD4 gene gene disruption. various linear fragments  containing URA3 gene <br /> [shaded triangles] inserted restriction sites shown) used omega transformation shown lines f derived plasmid inserts shown Fig. 1. upper figure shows extent yeast inserts plasmids pR169 pNF402 arrows indicate ORFs described Fig. 1. partial restriction map region genome provided. open box connotes location SP12 gene. hatched box shows location RAD4 gene based results gene disruptions transcript <br /> mapping  Fig. 8). percentage UV-sensitive disruption mutants obtained type transplacing fragment shown right case. Numbers parentheses indicate number UV-sensitive transformants function total number clones <br /> screened. <br />  <br />  <br /> additionally, mutation(s) present pNF402                                                 apparent identity pNF402GR, plasmid <br /> located close outside PvuI-BamHI interval;                                            isolates complemented UV sensitivity rad4-2, rad4-3, <br /> successful repair require gap extension vivo.                                                   rad44 mutants (Table 3). <br />    Plasmids complement rad4 mutants prop-                                                Based results conclude autonomously <br /> agated E. coli. gap-repaired centromeric plasmid                                                replicating plasmids containing functional yeast RAD4 complements rad44 designated pNF402GR.                                                     gene inactivated E. coli. Inactivation result isolated plasmid directly yeast transformed                                              mutation(s) specific sequence recognized E. coli, rad4-2, rad4-3, rad44 mutants                                                case independently cloned plasmids ex- <br /> propagation plasmid E. coli. plasmid                                            pected contain mutation(s). Alternatively, propagated E. coli essentially Ura+ trans-                                           given plasmid, site mutation(s) generated E. <br /> formants UV resistant (Table 3 Fig. 4).                                            coli differ independently cloned plasmids. avoid cases tested phenotypes UV resistance Ura+                                           possible complications arising situation cosegregated.  centromeric plasmids                                                used stable derivatives single plasmid (pNF402GR-EC1) <br /> isolated E. coli HB101 able support phenotypic                                             exclusive source transforming DNA complementation rad4 mutants tested, regardless                                              experiments. genetic background E. coli strain. Details                                             Localization RAD4 gene. inability propagate <br /> analysis single-copy multicopy autonomously repli-                                              plasmids functional RAD4 gene E. coli precluded <br /> cating plasmids isolated various E. coli strains gap                                        standard subcloning strategies designed localize gene <br /> repair yeast presented (R. Fleer, W.                                              6.6-kb yeast insert plasmid pNF402. Siede, E. C. Friedberg, manuscript preparation).                                                constructed series linear yeast DNA fragments independent transformants E. coli HB101 contained                                             plasmid pNF402GR-EC1, contains URA3 <br /> plasmids detectable deletions rearrangements                                             gene inserted selected restriction site. Integration determined restriction analysis) compared                                                 linear fragments yeast genome occurs plasmid pNF402GR isolated directly yeast.                                              double-crossover events regions sequence homology, <br /> plasmids stably propagated E. coli HB101                                             resulting exchange plasmid-derived linear fragments <br /> designated collectively pNF402GR-EC. Despite                                                  containing URA3 insertions, homologous regions  VOL. 7, 1987                                                                                                                            RAD4 GENE S. CEREVISIAE              1187 <br />  <br />                                 EcoRI <br />                                                           XmnI                     BglIl                                 initial experiments, Rad+ ura3-52 strain trans- <br />                                II                                                                                 formed linear 5.1-kb XmnI fragment plasmid <br />                                                                                          B                          pNF404 (Fig. 1) carrying URA3 disruption BamHI <br />                                     UVrUV                     UVrUVs            WT     UVrUVs <br />                                                                                                                       site shown Fig. 5. Ura+ transformants exam- <br />                                WT                   WT <br />                                                                                                                       ined response UV irradiation, great majority <br />                                                                                                                       UV resistant (Fig. 5, line b). confirmed                        10-                                                                                            transplacement occurred near SPT2 locus                                                                                                                       Southern hybridization analysis UV-resistant <br />                                                                                                                       UV-sensitive Ura+ transformants (Fig. 6).                                                                                                                       transplacements disruption                        5 <br />                                                                                                                       BamHI site does affect UV resistance Rad+ <br />                        4-                                                                                             strain, conclude site RAD4 <br />                                                                                                                       gene. small fraction (3%) UV-sensitive transformants <br />                        3-                                                                                             population Ura+ transformants containing                                                                                                                       disruption BamHI site (Fig. 5, line b) presumably <br />                                                                                                                       arose recombinational events include site(s)                        2-                                                                                             inactivation mutant rad4 allele plasmid pNF402GR- <br />                                                                                                                       ECL. low frequency events suggests                                                                                                                       site(s) located near end transplacing DNA                                                                                                                       included small fraction crossovers. <br />                         I1- <br />                                                                                                                       support suggestion, transplacement larger <br />                                                                                                                       (9.1-kb) EcoRI fragment plasmid pNF404 yielded                                             PROBE <br />                                                                                                                   larger fraction UV-sensitive Ura+ transformants (Fig. 5, <br />                                                                                                                       line , transplacement slightly smaller <br />                E           G        N            NG                  N       N E               G <br />                                                                                                                       (4.75-kb) HpaI-XmnI fragment resulted UV-sensitive <br />              Iws       S <br />                                     IH                           B <br />                                                                                                    7 <br />                                                                                                    S <br />                                                                                                         - <br />                                                                                                                       transformants (Fig. 5, line c). Furthermore, observation <br />                                                                                                                       transplacement linear fragment terminates                1--                      12.2                           ----                               EcoRI   HpaI site (Fig. 5, line c) results detectable <br />                                      1    2.4       1     -2.7       X                                        XmnI    UV-sensitive transformants, terminates                                     5.2                                    6.2       _       }              Bgll    XmnI site just 350 bp left does (Fig. 5, line b), <br />            L <br />                                                                                                                       indicates sites mutation plasmid <br />                E           G        N            N        G      B N         N E               G                      pNF402GR-EC1 located restriction <br />            -_xv1                                    1         .II                                                   sites. <br />                        S            H <br />                                                                                                                          demonstration disruption BamHI site does <br />                                                                                                    s <br />  B                                                      URA3 (2.5 Kb)                                                 affect function RAD4 gene suggests                I__                         14.7               1IIITIEIIITIII                                  EcoRI   plasmid pR169 contains entire gene, site <br />                                                                                                               XmnI    defines end yeast insert carried plasmid <br />      h.. <br />                            1         &#194;&#167;.2                  1      11111111111111 8.7                         BgIll   (Fig. 5). localize RAD4 gene carried   FIG. 6. Southern hybridization analysis chromosomal yeast                                                        disruptions BglII MstII sites shown Fig. 5. DNA gene disruption BamHI site near SPT2 locus.                                                        situated incomplete ORF, MstII site <br /> UV-resistant UV-sensitive disruption mutants generated                                                       small overlapping ORFs identified Roeder et transforming Rad+ ura3-52 strain linear DNA fragment                                                      al. (32) derivative plasmid pR169 (Fig. 5). carrying URA3 disruption BamHI site defines                                                        distinguish possible mechanisms RAD4 <br /> boundary yeast insert plasmid pR169 (Fig. 5, line b).                                                       inactivation discussed  control transplacements Chromosomal DNA cleaved EcoRI, XmnI, BglII,                                                               carried similar fragments sites electrophoresed 0.7% agarose gel, transferred nylon                                                         disruption known outside RAD4 gene. <br /> membrane, hybridized radiolabeled 4-kb HindIII-BamHI                                                       Transplacement HpaI-XhoI fragment URA3 <br /> fragment pR169 (probe). figure shows restric-                                                    disruption BglII generated 95% UV-sensitive transform- <br /> tion maps relevant region genomic DNA sizes restriction fragments expected hybridize probe                                                  ants (Fig. 5, line d).  demonstrated  untransformed cells   transformed cells (B). case                                                       control transplacement slightly larger HpaI-XmnI <br /> DNA fragments expected sizes observed  , regard-                                                      fragment containing URA3 insert BamHI (situated UV phenotype. Restriction sites described                                                   outside RAD4) (Fig. 5, line c) resulted UV-sensitive <br /> legend Fig. 1.                                                                                                     transformants. Similarly, disruption MstII site  <br />                                                                                                                       sulted 95% UV-sensitive transformants (Fig. 5, line e), <br /> yeast genomic DNA. process referred omega                                                               control disruption XbaI site SP72 gene <br /> transformation (33) provides mechanism generating                                                           (using identical linear DNA fragment) yielded <br /> URA3 disruptions selected sites near chromo-                                                             exclusively UV-resistant colonies (Fig. 5, line f). somal RAD4 allele. Disruption RAD4 gene Rad+                                                                results UV-sensitive transformants obtained <br /> cells result UV-sensitive Ura+ transformants.                                                               insertional mutagenesis BglII MstII sites arose  analysis complicated fact                                                            result gene disruption inclusion transplacing DNA fragments derived plasmid                                                                       sites inactivation plasmid-borne RAD4 gene. pNF402GR-EC1 carry mutations unknown                                                                   conclude RAD4 gene extends locations.  special care taken distinguish                                                               MstII site interval BglII BamHI sites <br /> transplacement events inactivated RAD4 gene                                                              (Fig. 5). <br /> URA3 disruption inactivated gene                                                                  Genetic evidence plasmid pNF400 derivatives <br /> introduction plasmid-borne mutation(s).                                                                        contain RAD4 gene. confirm plasmid pNF400  1188     FLEER ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                            0       10    20      30    40                                0     10    20     30    40 <br />                                UV DOSE         m2l                                               UV DOSE (J/m2) <br />   FIG. 7. UV-sensitive transplacement derivative SX46a (Rad+) (rad4-11) contains mutation(s) allelic rad4-3. Equal <br /> samples various cell suspensions (vertical rows) ordered horizontally agar plates, plates exposed UV doses <br /> shown  text details). Plate  1, SX46a (Rad+); 2, (rad4-11): 3, rad4-IJIradl-l diploid; 4, rad4-JJIrad2-1 diploid; 5, <br /> rad4-JlIrad3-1 diploid; 6, rad4-lJIrad4-3 diploid. Plate b: 1 2,  3, radl-1; 4, rad2-1; 5, rad3-1; 6, rad4-3. <br />  derivatives contain RAD4 gene nonspe-                 unlike RAD3 gene (13, 23), RAD4 essential cific suppressor UV sensitivity rad mutants, dem-                  viability yeast cells absence DNA damage. <br /> onstrated mutants generated gene disruption                     Transcriptional analysis RAD4 gene. Based genetic determinant UV-sensitive phenotype                    localization RAD4 gene described  likely <br /> haves like rad4 allele. indicated (Fig. 5, line d),              3&apos; end small ORF closest SPT2 gene transplacement 4.3-kb HpaI-XhoI fragment containing                  pR169 represents 3&apos; end RAD4-coding region (Fig. <br /> URA3 disruption BglII site RAD4 renders 95%                    5). identify 5&apos; end gene mapped RAD4 transformants UV sensitive. mutants                   transcriptional start site hybridizing total yeast RNA <br /> (designated rad4-11; Rad- Ura+) mated radl-1,                        1.65-kb BglII-XhoI DNA probe end labeled rad2-1, rad3-1 rad4-3 strains, complementation                   BglII site (Fig. 8). S1 nuclease digestion revealed protected <br /> UV-sensitive phenotype observed diploids                  fragment -790 nucleotides long, placing transcrip- <br /> cases (Fig. 7), demonstrating allelism                 tional start site near SphI site shown Fig. 8. site rad4-11 rad4-3 mutations. mated                     550 bp downstream BamHI site defines UV-sensitive mutant generated disruption                      end insert pR169 outside RAD4 <br /> MstII site RAD4 gene (rad4-10) (Fig. 5, line e),              gene (Fig. 5). results provide confirmation rad4-2 strain. resulting UV-sensitive diploid                       entire RAD4 gene present plasmid pR169. <br /> sporulated subjected conventional tetrad analysis.                  transcriptional analysis showed increase complete tetrads examined showed linkage genetic                     RAD4 mRNA treatment cells DNA- <br /> determinants Rad- phenotype, expected                    damaging agent 4-nitroquinoline-1-oxide (Fig. 8). URA3 disruption MstII site inactivated                    contrary, RAD4 messenger actually decreased <br /> RAD4 gene (Table 4). unlinked auxotrophic markers                     30 min treatment showed partial recovery 60 <br /> segregated 2+:2-, demonstrating normal meiotic segrega-                     min levels near observed control cells (Fig. 8). tion tetrads (Table 4).                                              similar response treatment 4-nitroquinoline-1-oxide <br />    Haploid cells containing disruptions RAD4 gene                 observed examination transcripts URA3 <br /> BglII MstII growth characteristics indistinguishable                gene (Fig. 8).  unlike RAD2 gene S. cerevisiae Rad+ strains. conclude                      (18, 31), RAD4 inducible DNA-damaging agents. <br />                                                                                Mapping mutation(s) plasmid pNF402GR-EC1                                                                             various chromosomal alleles. gene disruption experi- <br /> TABLE 4. Genetic analysis UV-sensitive disruption mutanta              ments described localized mutation(s) Phenotype             marker segregation (ratio):                   pNF402GR-EC1 350-bp interval XmnI <br />  selected      4+:0-        3+:1-        2+:2-        1+:3-         0+:4- <br />                                                                             HpaI sites (Fig. 5). localize mutation(s)                                                                             precisely, generated defined gaps plasmid Rad+              0            0            0          0             11     restriction enzyme digestion allow gap repair homol- <br /> Trp+              0            1           10          0              0     ogous chromosomal sequences transformation Lys+              0            0           11          0              0     gapped plasmid various rad4 mutant strains. fre- <br /> Ura+              0            0           10          1              0     quency Ura+ Rad+ colonies recovered rad44 <br />   YNN218 disrupted rad4-10 crossed rad4-2 mutant.           transformants -25% gap created  VOL. 7, 1987                                                                               RAD4 GENE S. CEREVISIAE               1189 <br />  <br />                         0      30 60             0    30 60              degradation vivo (21, 26) result rescue                                                                          chromosomal rad44 mutation. investigate possibility <br />                                                                          used mutant strains rad44 recipients.                                                                          UV-resistant colonies recovered rad4-2        848         -_                                                    rad4-3 mutants transformation plasmids contain- <br />                              M_         p <br />                                                                          ing gaps PvuI BglII sites                                   - <br />                                                                          PvuI XbaI sites (Fig. 9A, lines e-h). results <br />                                                                          demonstrate rad4-2 rad4-3 mutations map <br />                                                                          near 350-bp PvuI-BgIII interval. availability        622                                                               new mutation RAD4 gene (rad4-10) known <br />                                                                          location 450 bp left PvuI site (generated                                                                          disruption RAD4 gene MstII site; Fig. 5, line e), <br />                                                                          afforded opportunity test proximity <br />                                                                          rad44 mutation PvuI site limits efficiency                                                                          productive repair PvuI-BglII gap. observed                                                                          60% Ura+ rad4-10 transformants UV resistant <br />                    RAD4                              URA3                (Fig. 9A, line . represents significant increase                                                                          25% UV-resistant transformants recovered rad44 <br />                                                                          supports view rad44 mutation situated <br />                                       PROBE                              MstII PvuI sites apparently located <br />       5&apos;   *                                               3X            closer yeast chromosome. <br />                             790 bp                                          suggested  additional factor limit <br />                1 <br />                                                                          efficiency productive gap repair possibility                                                                          gap PvuI BglII sites (Fig. 9A)                G        X         K         Sp   XB O                    delete site inactivation plasmid pNF402GR-EC1 <br />                                                                          vivo gap expansion PvuI site <br />                                                                          necessary. absence unique restriction sites <br />                                                                          PvuI MstII sites precluded controlled <br />                                                                          leftward extension PvuI-BglII gap test hypo- <br />    FIG. 8. Si nuclease mapping 5&apos; end RAD4 tran-           thesis. carried omega transforma- <br /> script. RNA isolated Rad+ strain hybridized DNA        tions linear fragments containing URA3 disruptions probe (5&apos; end labeled Bgll site), treated S1 nuclease,   defined sites outside cloned RAD4 gene. indicated <br /> electrophoresed described text. figure shows            transplacement linear 4.5-kb HindIII-XmnI frag- <br /> protected bands observed treatment cells DNA damaging agent 4-nitroquinoline-1-oxide.  width       ment containing 3&apos; region RAD4 disrupts flanking <br /> band 0 min suggests doublet consisting        SPT2 gene, does inactivate RAD4 gene (Fig. 9B, <br /> protected fragments -790 -780 bp size.) Posttreatment times       line . demonstrates absence inactivating muta- 0, 30, 60 min indicated. control, right panel       tions left XmnI site.  right <br /> shows protected fragments URA3 transcript obtained           end fragment extended 100 bp include similar fashion. arrows left indicate size markers    PvuI site (Fig. 9B, line b), 350 bp include HpaI <br /> RAD4 transcript mapping. schematic representation pro-          site (Fig. 9B, line c), progressively increasing numbers tected fragment (solid bar) relative physical map RAD4     UV-sensitive transformants generated. Similarly, locus shown  right end arrow represents      shown  omega transformation carried mapped transcriptional start sites RAD4 gene. restriction sites shown described legend Fig. 1.        fragment containing 5&apos; region RAD4 (extending <br />                                                                          PvuI site rightward XmnI site outside                                                                          gene), results URA3 disruption BamHI,                                                                          UV-sensitive transformants observed (Fig. 9B, line d). PvuI XbaI sites (1.6 kb) (Fig. 9A, line              extension left end fragment 100 bp PvuI BglII sites (0.35 kb) (Fig. 9A, line b).            include XmnI site RAD4 gene (Fig. 9B, line e) deletion BglII XbaI sites                   2.4 kb include EcoRI site vector (Fig. 9B, line <br /> (immediately right PvuI-BglIll deletion), 2%          f) resulted progressively increasing frequency UV- Ura+ transformants UV resistant (Fig. 9A, line               sensitive transformants. results transplace- <br /> c). results localize site inactivation region          ments obtained gap repair plasmids approximate limits defined PvuI BglII sites.             demonstrate mutations generated Parenthetically, noted repair              propagation pNF402GR1 E. coli HB101 localized gap PvuI BgIII sites carried              100-bp interval defined XmnI PvuI sites <br /> plasmid (pNF417) (Fig. 1) containing yeast insert equiva-              highlighted Fig. 9B. <br /> lent present pR169  exclusion spt2-1 allele) (Fig. 9A, line d), frequency UV-resistant Ura+ transformants essentially                                         DISCUSSION just described repair plasmid (pNF418) (Fig. <br /> 1) containing additional upstream sequences (Fig. 9A, line                 Previous attempts isolate RAD4 gene screening <br /> b), providing independent evidence entire                  different yeast genomic libraries recombinant plasmids <br /> RAD4 gene present plasmid pR169.                                   complement UV sensitivity rad4 mutants    explanation limited recovery UV-resistant              unsuccessful. experiments reported transformants repair gapped plasmids strain                  isolation functional RAD4 gene alternative strat- <br /> rad44 expansion defined gaps exonucleolytic                egy provide explanation failure isolate  1190     FLEER ET AL.                                                                                                        MOL. CELL. BIOL. <br />  <br />  <br />  <br />  <br />                                                                                            pR169 -H <br />                                         pNF417 -                                       pNF418                                                        SPT2                    RAD4                                   recipient              UV-resistant <br />                                                    H                H <br />                                                                                       strain:                gap-repair <br />                           H              ~H            Xe HHm       xH          H <br />                      5) <br />                       4 <br />                             H   H         H    H <br />                                                          d <br />                                                                 H <br />                                                                 4 <br />                                                                         m   H t <br />                                                                             O                              transformants: <br />                           m xP            X x            m      x       m   x  <br />  <br />  <br />  <br />                                             II <br />                                dAzl                                                                        25% (6/25) <br />               b                                                                                              25% (6/25) <br />                                                                                         rad44                 2% (1/50) <br />               c <br />               d                                                                                              20% (5/25) <br />               e <br />                                                                                     }rad4-2                   0 (0/25) <br />               f                                          .: .M                                                0 (0/25) <br />               g                                                                     }   rad4-3                0 (0/25) <br />               h                                         .7 -1 ..                                              0 (0/25) <br />                                                                                     )rad4-10                 60% (29/50) <br />  <br />  <br />                                                                                       recipient             UV-sensitive <br />                                                                                        strain:               disruption <br />                                                                                                               mutants: <br />  <br />                                                                                                              0 (0/200) <br />              b                                                                                                 5% (9/200) <br />               c <br />                                                                                         RAD4 <br />                                                                                                               15% (10/70) <br />               d                                                                                                0 (0/200) <br />               e                                                                                                3% (3/92) <br />               f                                                                                               70% (34/50) <br />   FIG. 9. Mapping chromosomal plasmid mutations RAD4 gap repair   gene transplacement (B). case, physical map SPT2 RAD4 loci relevant flanking sequences shown. hatched box represents location RAD4 gene established gene disruption transcript mapping  text details).   Lines indicate yeast inserts centromeric plasmids pNF417 (line d) pNF418  . Double-stranded gaps created restriction digestion vitro represented dotted lines. (B) Lines e indicate linear DNA fragments used gene transplacement assay described text. shaded triangles represent insertions fragments carrying URA3 gene restriction sites indicated  text details). hatched vertical zone XmnI PvuI sites denotes localization site(s) inactivation plasmid pNF402GR-EC1. <br />  VOL. 7, 1987                                                                           RAD4 GENE S. CEREVISIAE                 1191 <br />  <br /> functional gene libraries propagated E. coli.       knowledge RAD4 gene represents documented <br /> known previous studies SPT2 RAD4 genes           example inability propagate yeast gene E. coli tightly linked genetically (36). demonstrated        HB101 suggests presence yeast genes physical linkage loci showing integration      recombinant libraries unrecognized similar rea- 4-kb yeast fragment (containing mutant allele SPT2)         sons. rad4 mutant strains generates UV-resistant integrants,           provided genetic evidence complement- <br /> albeit low frequency. Rescue relevant genomic              ing activity present gap-repaired plasmids sequences integrants yielded plasmid           lost propagation E. coli RAD4 gene, inactivated RAD4 gene. functional gene               physically mapped gene position immedi- <br /> restored deleting (presumably) mutated sequences            ately upstream SPT2 gene. localization demon- <br /> repairing deletions gene transfer normal homol-        strates entire gene present plasmid pR169, <br /> ogous chromosomal sequences vivo. Direct transforma-            originally isolated G. S. Roeder (personal communica- <br /> tion gap-repaired plasmid rad4 mutants            tion).  tight genetic linkage SPT2 restores UV resistance wild-type levels.          RAD4 genes (36) confirmed physical analysis <br /> complementing activity lost plasmid           presented  initial difficulty recognizing propagated E. coli.                                             plasmid pR169 contains entire RAD4 gene stems    Inactivation cloned RAD4 gene occur         inability detect Rad+ transformants integration unique sites gene, reflecting presence       plasmid rad44 mutant. presumably reflects <br /> structural features consistently modified E. coli.     close proximity chromosomal rad44 muta- examples phenomenon noted           tion site mutations rad4 allele eucaryotic genes DNA sequences           plasmid pR169, resulting low frequency propagated certain E. coli strains (3, 20, 37).       recombinational events  <br /> hands, transformation plasmids carrying functional               presence inactivating mutations rad4 allele RAD4 gene different recA recBC sbcB mutants did           pR169 altered translational reading frame prevent inactivation gene (Fleer et al., prepa-      region sequenced Beard Smith (personal commu- <br /> ration). pathway recombination E. coli            nication), leading erroneous identification independent recA, recBC, sbcB genes involves        multiple ORFs (32). shifting reading frame recE gene, encodes exonuclease VIII regulated         positions -450 bp apart, single continuous ORF sbcA gene (16). tested recE mutants         generated, 3&apos; end located 280 bp upstream tolerance RAD4 gene E. coli.               beginning SPT2 coding region. ORF interesting note site-specific deletions sequences 3&apos;   interrupted BglII site defines end adult P globin gene Xenopus laevis occurs range     sequenced region pR169. contend ORF E. coli hosts including recA recBC sbcB recBC          contains coding region RAD4 (Fig. 5). stop codon <br /> sbcA recE strains (10).                                            defines 3&apos; end ORF separated    alternative explanation inability propagate     RAD4 transcriptional start site distance -2.3 kb, functional RAD4 gene E. coli Rad4 protein toxic      value agrees unassigned 2.6-kb organism, plasmid copy number-                transcript detected pR169 hybridization probes (M. <br /> dependent manner. observed                Smith, personal communication). estimated size plasmids isolated E. coli HB101 (designated               RAD4 gene (-2.3 kb) indicates encode protein <br /> pNF402GR-EC class) stable propagation E.         Mr -88,000. <br /> coli arid restriction analysis evidence exten-          Haploid cells containing disruptions RAD4 gene sive deletion rearrangement (Fleer et al., preparation).     different sites normal growth characteristics, plasmids RAD4 gene randomly inacti-           Southern analysis demonstrates single copy vated point mutations small deletions,         RAD4 present yeast genome. conclude <br /> anticipate different plasmids containing functional          unlike RAD3, RAD4 essential viability RAD4 gene suffer mutational inactivation indepen-         haploid yeast cells absence DNA damage. dent sites passaged E. coli. Evidence        addition, RAD4 mRNA increased cells exposed support notion comes observation             DNA-damaging agent known enhance transcriptional <br /> integration plasmid pR169 rescued derivative         activity RAD2 gene. <br /> pNF400 various rad4 mutants yielded different results            molecular cloning RAD4 gene completes (Table 2). suggests pNF400 rescued         search known RAD genes absolutely required UV-resistant rad4-2 integrant contained            nucleotide excision repair yeast S. cerevisiae. wild-type RAD4 allele inactivated propa-          inability propagate plasmids functional RAD4 gene <br /> gation E. coli, yielding stable derivative rad4        E. coli reflects toxic effects native Rad4 protein, <br /> mutation site(s) different plasmid pR169.        controlled reversible inactivation gene plasmid pR169 yielded UV-resistant integrants             facilitate propagation mutant gene E. coli transformation rad44, plasmid pNF400 yielded          expression wild-type gene yeast. studies UV-resistant integrants transformation rad4-2      progress. rad4-3 strains, site(s) inactivation pR169 apparently close rad44 mutation,                                 ACKNOWLEDGMENTS <br /> pNF400 closer rad4-2 rad4-3 mutations. localized site RAD4 inactivation particular         particularly grateful G. Shirleen Roeder gift plasmid designated pNF402GR-EC1 100-bp region          plasmid pR169 helpful discussions Mike Smith <br /> middle gene. Detailed examination lesion(s)       Caroline Beard communicating unpublished data. thank plasmid await sequencing region              Jane Cooper, Louie Naumovski, David Kalainov, Weiss, Linda <br /> comparison sequence wild-type gene.         Couto, Paul Saxon, Roger Schultz, Wolfram Siede helpful <br />  1192       FLEER ET AL.                                                                                                    MOL. CELL. BIOL. <br />  <br /> comments suggestions Jean Oberlindacher typing            19. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982, Molecular <br /> manuscript.                                                                   cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />   studies supported Public Health Service research               Cold Spring Harbor, N.Y. <br /> grant CA 12428 National Institutes Health. R.F.       20. Nader, W. F., T. D. Edlind,  Huettermann, H. W. Sauer. <br /> recipient postdoctoral fellowship Deutscher                     1985. Cloning Physarum actin sequences exonuclease- <br /> Akademischer Austauschdienst, G P. supported                   deficient bacterial host. Proc. Natl. Acad. Sci. USA <br /> Public Health Service postdoctoral fellowship Fogarty                82:2698-2702. <br /> Foundation.                                                               21. Naumovski, L., G. Chu, P. Berg, E. C. Friedberg. 1985. <br />                                                                               RAD3 gene Saccharomyces cerevisiae: nucleotide sequence <br />                         LITERATURE CITED                                      wild-type mutant alleles, transcript mapping, aspects <br />  1. Berk, , P. Sharp. 1977. Size mapping early                   gene regulation. Mol. Cell. Biol. 5:17-26. <br />     adenovirus mRNAs gel electrophoresis S1 endonuclease-           22. Naumovski, L., E. C. Fri <br /> </body></html>