<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                    Drosophila poly suggests novel <br />                                                     role Elongator complex                                                     insulin receptor &#226;&#8364;&#8220; target rsob.royalsocietypublishing.org <br />                                                     <span id='am-6' about='obo:IMR_0002818' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="rapamycin" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0002818" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C07909" datatype="xsd:string"></span>rapamycin</span> signalling <br />                                                     Ekin Bolukbasi&#226;&#8364;&#160;, Sharron Vass, Neville Cobbe&#226;&#8364;&#161;, Bryce Nelson}, <br /> Research <br />                                                     Victor Simossis k , Donald R. Dunbar Margarete M. S. Heck <br /> Cite article: Bolukbasi E, Vass S, Cobbe <br /> N, Nelson B, Simossis V, Dunbar DR, Heck            University Edinburgh, Queen&#226;&#8364;&#8482;s Medical Research Institute, University/BHF Centre MMS. 2011 Drosophila poly suggests novel          Cardiovascular Science, 47 Little France Crescent, Edinburgh EH16 4TJ, UK <br /> role Elongator complex insulin <br /> receptor &#226;&#8364;&#8220; target rapamycin signalling. <br /> Open Biol 2: 110031.                                1. Summary <br /> http://dx.doi.org/10.1098/rsob.110031               Multi-cellular organisms need successfully link cell growth metabolism <br />                                                     environmental cues development. Insulin receptor &#226;&#8364;&#8220;target rapamy- <br /> Received: 23 November 2011                          cin (InR&#226;&#8364;&#8220;TOR) signalling highly conserved pathway mediates link. <br />                                                      poly, essential gene Drosophila mediates InR &#226;&#8364;&#8220;TOR <br /> Accepted: 21 December 2011 <br />                                                     signalling. Loss poly results lethality instar larval stage,                                                     stage extreme larval longevity. Analysis Drosophila demon- <br /> Subject Area: <br />                                                     strates Poly InR interact poly mutants overall <br /> cellular biology/developmental biology/             decrease InR&#226;&#8364;&#8220;TOR signalling, evidenced decreased phosphorylation <br /> genetics/molecular biology                          Akt, S6K 4E-BP. Metabolism altered poly mutants, revealed                                                     microarray expression analysis decreased triglyceride : protein ratio Keywords:                                           mutant animals. Intriguingly, cellular distribution Poly dependent <br /> cell growth, nucleus, signalling, insulin           insulin stimulation Drosophila human cells, moving                                                     nucleus insulin treatment, consistent role InR&#226;&#8364;&#8220;TOR signalling. <br /> Author correspondence:                           data reveal Poly novel, conserved  flies                                                     humans) mediator InR signalling promotes increase cell growth <br /> Margarete M. S. Heck <br />                                                     metabolism. Furthermore, homology small subunits Elongator <br /> e-mail: margarete.heck@ed.ac.uk                     demonstrates novel, unexpected role complex insulin signalling. <br />  <br />  <br /> Present addresses: <br /> &#226;&#8364;&#160; <br />   Institute Healthy Ageing, University College <br />                                                     2. Introduction <br /> London, Darwin Building, Gower Street,          Multi-cellular organisms evolved mechanisms link cellular metabolism <br /> London WCIE 6BT, UK.                                growth external environmental cues, nutrient growth factor <br />                                                     levels, order survive adapt fluctuations availability &#226;&#8364;&#161; <br />  School Biological Sciences, University       factors. InR&#226;&#8364;&#8220;TOR pathway key regulators cellular energy <br /> Liverpool, Biosciences Building, Crown          homeostasis growth [1], signalling pathway evolutionarily  <br /> Street, Liverpool L69 7ZB, UK.                      served metazoa [1,2].  studies carried using Drosophila } <br />  Donnelly Center Cellular Biomolecular      model played major role expanding understanding Research, University Toronto, 160 College        mechanisms, downstream consequences, signalling Street, Toronto, Ontario, Canada, M5S 3E1.          pathway [3&#226;&#8364;&#8220;5]. humans, aberrations InR&#226;&#8364;&#8220;TOR signalling lead various <br />                                                     metabolic syndromes, including diabetes obesity, develop- <br /> k <br />  Institute Molecular Oncology, Biomedical        ment various types cancers [6]. <br /> Sciences Research Center &#226;&#8364;&#732;Alexander Fleming&#226;&#8364;&#8482;,           cascade phosphorylation events mediates signalling InR&#226;&#8364;&#8220; <br /> 16672 Varkiza, Greece.                              TOR pathway. binding insulin InR leads phosphorylation <br />                                                     insulin receptor substrate (IRS) InR. IRS acts recruitment site <br /> Electronic supplementary material available      <span id='am-31' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-32' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-40' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>phosphatidylinositol</span> 3-kinase (PI3K), catalyses conversion http://dx.doi.org/10.1098/rsob.110031            phosphatidylinositol (4,5)-bisphosphate (<span id='am-57' about='obo:IMR_0001350' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001348 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-58' property="rdfs:label" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001350" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C04637" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Phosphatidyl-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="PIP2" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="PI(4,5)P2" datatype="xsd:string"></span><span id='am-67' property="obo:IAO_0000115" content="PI(4,5)P2 is cleaved by phospholipase C-beta or gamma to generate IP3 and diacylglycerol." datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol diphosphate" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-1D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-D-myo-inositol 4,5-bisphosphate" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-74' property="rdfs:label" content="PIP2" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasAlternativeId" content="IMR:0200477" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="phosphatidylinositol bisphosphate" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasExactSynonym" content="Phosphatidylinositol-4,5-bisphosphate" datatype="xsd:string"></span>PIP2</span>) phosphatidylinositol <br />  <br />  <br />                                                      <span id='am-50' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-51' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; 2012 Authors. Published Royal Society terms Creative Commons Attribution <br />                                                      License http://creativecommons.org/licenses 3.0/, permits unrestricted use, provided original <br />                                                      author source credited. <br />  (3,4,5)-trisphosphate (<span id='am-89' about='obo:IMR_0001352' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001348 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-90' property="rdfs:label" content="Phosphatidylinositol-3,4,5-trisphosphate " datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C05981" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasExactSynonym" content="PIP3" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="PIP3" datatype="xsd:string"></span><span id='am-95' property="obo:IAO_0000115" content="PI(3,4,5)P2 is generated by phosphorylation of PI(4,5)P at the 3 position of the inositol ring catalyzed by PI3-kinase. NOT cleaved by phospholipase C." datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasAlternativeId" content="IMR:0200396" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:id" content="IMR:0001352" datatype="xsd:string"></span><span id='am-98' property="rdfs:label" content="1-Phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate" datatype="xsd:string"></span><span id='am-99' property="rdfs:label" content="PI(3,4,5)P3" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasExactSynonym" content="1-Phosphatidyl-1D-myo-inositol 3,4,5-trisphosphate" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasExactSynonym" content="Phosphatidylinositol-3,4,5-trisphosphate " datatype="xsd:string"></span>PIP3</span>) cell membrane. PIP3 turn      verified expression overlapping CG8790 gene              2 <br /> recruits PDK1 Akt membrane, PDK1 phos-              affected P-element insertion poly05137 allele <br />  <br />  <br />  <br />  <br />                                                                                                                                             rsob.royalsocietypublishing.org <br /> phorylates activates Akt. Phosphorylated Akt signals             (figure 1b). independent experiments additionally corro- <br /> inhibit tuberous sclerosis complex (TSC, Tsc1 &#226;&#8364;&#8220;Tsc2) [7 &#226;&#8364;&#8220;        borate lesion CG9829 gene responsible 9]. TSC inhibited, small GTPase Rheb             mutant phenotype:   excision P-element following gen- <br /> active [10]. activates TOR, integrating TOR           etic exposure transposase completely reverted mutant insulin signalling process.                                      phenotype, (ii) successful rescue poly05137 lethality <br />     TOR component different complexes: TORC1         achieved using hs-Gal4 driver direct expression <br /> TORC2. Activation TORC1 various downstream effects            UAS-poly transgene larval development. <br /> contributing increase cell growth proliferation.        Mutation poly results pleiotropic effects manifesting example, TORC1 directly phosphorylates S6K 4E-BP, result-        particularly striking phenotype. poly mutants appear progress <br /> ing increase ribosome biogenesis m7G cap-               normally embryogenesis, larval development pro- <br />  <br />  <br />  <br />  <br />                                                                                                                                             Open Biol 2: 110031 <br /> dependent translation [1]. addition, TORC1 activation inhibits    ceeds slowly. normal generation time autophagy [11]. negative feedback loop signals IRS     10 days 258C, poly mutant larvae exhibit extreme instar S6K, ensuring attenuation TOR signalling <span id='am-129' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-130' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-132' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-135' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-136' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-138' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="ceramide" datatype="xsd:string"></span>cer</span>-     longevity 21 days dying pupation <br /> tain level [12]. TORC2 complex phosphorylates activates      (figure 1d). homozygous mutant larvae examined, <br /> Akt kinase [13], resulting phosphorylation forkhead-   morphology tissues appeared abnormal: brain, <br /> like transcription factor FoxO. Phosphorylated FoxO excluded      ring gland, salivary glands imaginal discs reduced nucleus, precluding transcription FoxO target        size compared heterozygous siblings wild-type ani- <br /> genes [14&#226;&#8364;&#8220;16].                                                       mals, suggesting cell growth proliferation defects <br />     critical consequence activation InR &#226;&#8364;&#8220;TOR sig-        (figure 1e). Mutant larval neuroblasts characterized nalling inhibition autophagy: cellular response       abnormally shaped nuclei, mitotic figures, evident starvation components cytoplasm                  20 day old mutant larvae, normal appearance. engulfed small double-membrane-enclosed vesicles.             lobulated nuclei resembled nuclei mammalian poly- <br /> contents vesicles degraded autophagic            morphonuclear leucocytes, suggesting poly <br /> machinery, breakdown products serve nutrient           (figure 1f ). lengthened instar larval phase, <br /> source cell nutrients available         melanotic masses appeared haemolymph poly environment. Alterations autophagy            mutants, increasing size number time (figure 1g). cancer neurodegenerative diseases [17,18].                    second hypomorphic allele poly recently reported <br />      report identification Poly novel          result abnormal nurse-cell chromosome dispersal mediator InR &#226;&#8364;&#8220;TOR signalling pathway. poly              oocyte polarity Drosophila germline, possibly owing essential gene Drosophila mutated P-element         proposed interaction mRNP complex involved similar <br /> transposon mutagenesis screen [19]. Crucially, gene pro-         processes [20]. <br /> duct conserved higher eukaryotes, including humans,                Expression analysis poly revealed high levels mRNA <br /> showing homology ELP6 subunit Elongator                protein 4 h embryogenesis, suggestive complex. Loss poly function results lethality late      maternal loading mRNA possibly protein larval stage, extreme larval longevity.          oocytes (figure 1h . level Poly decreased <br /> intriguing phenotypic features observed larvae lacking        remainder embryogenesis, level poly, including abnormal nuclear morphology neuroblasts           higher later stages development. Overall, poly development large melanotic masses instar        expressed development (figure 1h . <br /> larvae. combined genetic, biochemical bioinfor- <br /> matic approaches functionally characterize poly. data <br /> reveal poly novel mediator InR&#226;&#8364;&#8220;TOR signalling <br />                                                                      3.2. Poly conserved higher eukaryotes loss poly results downregulation number               resembles Elp6 subunit Elongator components InR &#226;&#8364;&#8220;TOR pathway. pro- <br /> pose wild-type Poly protein positive regulator <br />                                                                           complex cell growth metabolism.                                       Drosophila poly gene encodes 251-amino-acid-long <br />                                                                      protein, lacking motifs suggestive specific function. <br />                                                                      Sequence similarity searches revealed Poly conserved <br />                                                                      higher eukaryotes  importantly, human <br /> 3. Results                                                           homologue (figure 2a). apparent orthologue Poly                                                                      Saccharomyces cerevisiae Elp6 protein, 6-sub- <br /> 3.1. Characterization poly mutant phenotype                   unit Elongator complex  association RNA poly mutation isolated P-element mutagenesis            polymerase II holoenzyme, responsible transcriptional <br /> screen aimed generate large collection single           elongation [21,22]. Elongator holocomplex conserved <br /> P-element-induced mutations Drosophila [19].                  composition yeast humans [23,24], acetylation <br /> P-element insertion led lethal poly05137 allele      activity contributed Elp3 catalytic subunit [25]. Intri- <br /> mapped single intron CG9829 gene, localizing           guingly, acetylation directed different substrates, 87E7-8 chromosome (figure 1a). poly05137         depending cell complex (e.g. tubulin <br /> insertion led absence poly mRNA assessed              cytoplasm, histone nucleus) [26,27]. Elonga- <br /> Northern blotting  shown), reverse transcriptase &#226;&#8364;&#8220;               tor additionally linked translational control <br /> polymerase chain reaction (RT-PCR; figure 1b) Poly               tRNA modification cytoplasm. <span id='am-11' about='oboInOwl:date' typeof='owl:Thing'>date</span>, protein revealed immunoblotting (figure 1c). RT-PCR            Elp3 subunit studied Drosophila,                                                                                                                                                                                          poly/TM6B <br />                                                                                                      (b)                                                     (c)                                                                      3 <br />  <br />  <br />  <br />  <br />                                                                                                                           wild-type <br />  <br />  <br />                                                                                                                                           poly/poly <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            wild-type <br />  <br />  <br />                                                                                                                                                                                                      poly/poly <br />                                                                                                                                                                                                                     Dmel2 <br />                                                     P{PZ} poly 05137 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                       rsob.royalsocietypublishing.org <br />                                                                                         poly <br />  <br />                                                                                    CG9829 <br />                                                                                                              poly <br />  <br />                                                                                    1 kb                                                                        poly <br />                                                                                                        CG8790 <br />                   CG8790 <br />                                                                                                                                                                actin <br />                                                                                                              rp49 <br />  <br />             (d)          day      1                 2                 3       4                    6                                                           10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                       Open Biol 2: 110031 <br />                      wild-type embryo               1*                2*           3*                                    pupation                                                      eclosion <br />                    poly/TM6B embryo                     1*            2*           3*                                    pupation                                                      eclosion <br />                       poly/poly embryo                  1*             2*           3*          melanotic masses visible                                                                            21 days <br />  <br />  <br />             (e) <br />  <br />  <br />  <br />  <br />                                                                             wild-type                                                                                   poly <br />  <br />             (f)                                                                                                          (g) <br />                        wild&#226;&#8364;&#8220;type                                           poly <br />  <br />  <br />  <br />  <br />             (h)                                                                                      <br />                                                                                                                               emb 4-20 <br />                                                                                                                emb 0-4 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                           adult-m <br />                                                2nd  <br />  <br />  <br />  <br />                                                                                                                                                       2nd  <br />  <br />  <br />  <br />                                                                                                                                                                                                                             adult-f <br />                                                              3rd  <br />  <br />  <br />  <br />                                                                                                                                                                3rd                                       1st  <br />  <br />  <br />  <br />                                                                                                                                          1st                                                                                         adult <br />                                emb <br />  <br />  <br />  <br />  <br />                                                                        EP <br />  <br />                                                                               LP <br />  <br />  <br />  <br />  <br />                                                                                                                                                                            EP <br />  <br />                                                                                                                                                                                          LP <br />  <br />  <br />  <br />  <br />                     poly <br />                                                                                                        Poly <br />  <br />  <br />                     rp49                                                                             actin <br />  <br /> Figure 1. Characterization poly gene mutant phenotype.   Gene map showing P-element insertion intron poly. (b) RT-PCR wild- <br /> type homozygous poly instar larvae demonstrating absence Poly poly mutant animals. level CG8790 unaffected poly mutants. RP49 <br /> serves control. (c) Immunoblotting extracts wild-type, heterozygous homozygous poly instar larvae demonstrating absence Poly poly <br /> mutant animals. Poly present Dmel2-cultured Drosophila cells. (d ) Developmental timing wild-type, heterozygous homozygous poly animals. <br /> (e) instar larval brain imaginal discs dissected wild-type poly animals. Note reduced brain size absence imaginal discs homozygous poly mutant. (f ) DAPI staining wild-type poly larval neuroblasts. Note abnormally shaped nuclei homozygous poly mutant. <br /> (g) Homozygous poly larvae develop melanotic masses larval instar, increase number time. (h  Expression analysis poly mRNA protein development. Northern immunoblotting wild-type extracts specific developmental stages showing Poly mRNA protein expression <br /> levels. emb, embryo (0&#226;&#8364;&#8220; 4, 4 &#226;&#8364;&#8220; 20 &#194;&#188; age hours);  instar larvae (1st, 2nd 3rd); EP, early pupae; LP, late pupae; adult-m, adult males; adult-f, adult females. <br />  mutant phenotype recently reported                                                     identified flies humans). Consistent <br /> remarkably similar described [28].                                                        previous identification Elp4 Elp6 proteins     Elp6 budding yeast Elongator subcomplex                                             paralogues [29], resulting phylogeny implies contains Elp4 Elp5 proteins [21,22],                                             different Elongator subunits probably arose share sufficient homology aligned                                                 common gene duplication events eukaryotic evolution <br />                           Hs                                                                                                                                                                                 4 <br />                              Dm <br />                               <br />  <br />  <br />  <br />                                                                                                                                                                                                             rsob.royalsocietypublishing.org <br />                              Sc <br />  <br />                              Hs <br />                              Dm <br />                                                           Sc <br />  <br />                              Hs <br />                              Dm <br />                                                           Sc <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                             Open Biol 2: 110031 <br />                              Hs                                                                                                                                                              conserved <br />                              Dm <br />                                                           Sc <br />                                                                                                                                                                                                  variable <br />  <br />  <br />  <br />                        (b) <br />                                                                                                                                                      Elp 4                                    s <br />                                                                                          Dicty <br />                                                                                                                                                                                             ce <br />  <br />  <br />  <br />  <br />                                                                                                                                    Dictyostelium <br />                                                                       p6                                                                                                                 y <br />                                                                    /El                                                                                                                om <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                                                                                                                                                                                      ar <br />                                                                                             osteli <br />  <br />  <br />                                                            y <br />  <br />  <br />  <br />  <br />                                                                                                                                                                  ps <br />                                                                                                                                                                               ch <br />                                                          ol                                                        h <br />  <br />  <br />  <br />  <br />                                                                                                                                                             o <br />                                                                                                                                                                              c <br />                                                                                                                 zebrafis <br />  <br />  <br />  <br />  <br />                                                                                                                                                          <span id='am-122' about='oboInOwl:id' typeof='owl:Thing'>id</span> <br />                                                                                                                                                                           sa <br />  <br />  <br />  <br />  <br />                                                                                                                                                      rice <br />                                 Sch <br />                                                                                      rice <br />  <br />  <br />  <br />  <br />                                                                                                                                                       ab <br />                                                                                                                                                                         zo <br />                                                                                                   um <br />  <br />  <br />  <br />                                      izo                                                                                                                                                           ces <br />                                                                                                                                                                      hi <br />                                                     P <br />  <br />  <br />  <br />  <br />                                                                                                                                                     Ar <br />                                         sac <br />                                              cha <br />                                                                Ar <br />                                                                    ab                                                                                              Sc             har    omy <br />                                                     rom                  ido <br />                                                                                psi                                                                                        Sacc <br />                                                                                                                            m <br />                                                          yce                      s                                    hu                                                               habditis <br />                                                                s                                                                                                          Caenor <br />                              Caenorha                                                                                                                           Drosoph <br />                                            bditis                                                                                                                          ila <br />                                                                                                                                                                zeb <br />                                                                                                                                                                    rafi <br />                                                                                                                                                           hum           sh <br />                                                                                                                                                             <br />                                             yces <br />                                    arom <br />                              Sacch                                       hil <br />                                                                                                                                                  op <br />                                                            D   ros <br />                                                                                                                                                            human <br />  <br />                                                                 rice                                                                      zeb <br />                                                                                                                                                     rafi <br />                                                                                                                                                           sh <br />                                                                           s <br />                                                                       psi <br />                                                                    ido <br />                                                            ab <br />                                                          Ar <br />                                                                                                          stelium <br />  <br />  <br />  <br />  <br />                                                                          es <br />                                                                  m  yc <br />                                                               ro <br />                                                                                                        Dictyo <br />  <br />  <br />  <br />  <br />                                                           ha <br />                                                       cc <br />                                                                                      s <br />                                                                                   yce <br />  <br />  <br />  <br />  <br />                                                                                                     iz os                                                                                                                         Ca <br />                                                                                                                                                                                                                                                              rom <br />  <br />  <br />  <br />  <br />                                               h <br />                                            Sc                                                                                                                                                                                                                                                                                 Dros <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                        ha <br />                                                                          cha <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                          bd <br />                                                                                                   E lp 5                                                                                         iti <br />                                                                                                                                                                                                     s <br />                                                                                                                                                    ophi <br />                                                                      Sac <br />  <br />  <br />  <br />  <br />                                                                                                                                                     la <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    0.5 <br /> Figure 2. Alignment phylogenetic analysis Poly.   Alignment showing Poly homologues human (Hs; NP_001026873), Drosophila melanogaster (Dm; <br /> AAF40432), Arabidopsis thaliana   NP_567351) Saccharomyces cerevisiae (Sc; NP_014043), shading identical similar amino acids rendered BOXSHADE program. coloured band underneath section alignment indicates sequence conservation according PAM250 matrix, averaged sliding window 10 amino acids. location Walker B motif (ordinarily associated <span id='am-78' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-79' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-82' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span>ATP</span> binding) indicated blue box, Walker (P-loop) motif required ATP hydrolysis similarly conserved. (b) Phylogenetic tree showing relationship Poly, identified Drosophila <br /> orthologue Elp6, homologous sequences. scale bar indicates distance according WAG substitution matrix. <br />  <br />  <br /> (figure 2b; electronic supplementary material, figure S1).                                                                 interacting physically Poly, immunoprecipitation  Poly evolutionarily conserved protein exhibits                                                            antibody generated recombinant Poly  <br /> homology yeast Elp6 Elongator subcomplex component.                                                                 formed using 0 &#226;&#8364;&#8220;5 h Drosophila embryo extracts (figure 3a,b). <br />                                                                                                                            Poly immunoprecipitated immune serum, <br />                                                                                                                            wild-type extracts. Samples analysed <br /> 3.3. Identification protein interactors Poly                                                                         using tandem mass spectrometry [30,31]. Immunoprecipi- poly gene appeared &#226;&#8364;&#732;pioneer&#226;&#8364;&#8482;, adopted                                                                 tates pre-immune serum served control. unbiased approach determine pathways processes                                                                  Poly-binding proteins identified significant Poly involved. order identify proteins                                                             scores, InR scored highest represented numerous <br />                                                                                                                    poly <br />                                                                                       (b)                                                                                5 <br />  <br />  <br />  <br />  <br />                                                            poly <br />                                            <br />  <br />  <br />  <br />  <br />                                                                                                          pre                                                    pre  <br />  <br />  <br />  <br />                                                                                                                                                                        rsob.royalsocietypublishing.org <br />                                                                                             97 <br />                                          97 <br />                                                                                                              6             1 <br />                                                                                             64               7             2 <br />                                          64                                                                  8             3 <br />                                                                                             51 <br />                                          51                                HC <br />                                                                                             39 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        Open Biol 2: 110031 <br />                                          39 <br />                                                                                                              9             4 <br />                                                                                             28 <br />                                          28                                poly                              10            5 <br />                                                                                             19 <br />                                          19                                                 14 <br />  <br />  <br />                        (c)                                      (d)                              (e) <br />                                                                                                                                    poly <br />                                  wt       InR      Akt1                    wt         InR <br />  <br />                          Poly                                       poly <br />  <br />  <br />                        &#194;&#181;-tub                                        rp49 <br />                                                                                                        InR <br />  <br /> Figure 3. Poly insulin receptor physically interact.   antibody generated recombinant Poly immunoprecipitates Poly wild-type Drosophila instar larval extracts. HC represents heavy chain recognized secondary antibody detection. Poly migrates predicted size 28 kDa. (b) Poly immunoprecipitated 0 &#226;&#8364;&#8220; 5 h wild-type embryos using Poly antibodies compared identical sample immunoprecipitated pre-immune <br /> serum. Bands numbered 1 &#226;&#8364;&#8220; 5 immunoprecipitate lane corresponding bands pre-immune lane (numbered 6 &#226;&#8364;&#8220; 10) excised Coomassie-stained gel analysed mass spectrometry. (c) Immunoblotting using Poly antibody carried wild-type, InR05545 Akt104226 larval <br /> extracts. (d ) RT-PCR performed using primers specific poly rp49 RNA extracted wild-type InR05545 larvae. (e) Melanotic masses (arrows) <br /> observed InR05545 poly mutant larvae. <br />  <br />  <br /> peptides distinct bands (table 1). result suggests                        3.5. Poly mutation results decreased signalling Poly InR coexist complex, <br /> possibly playing role InR signalling pathway.                                     activity downstream insulin receptor <br />                                                                                       Based results presented far, hypothesized                                                                                       genetic interactions poly com- <br /> 3.4. Poly decreased insulin receptor mutants                                    ponents InR signalling pathway. Consequences Considering physical interaction Poly InR,                             interactions revealed state  <br /> assessed level Poly various InR signalling mutants.                         stream effectors InR pathway. <br /> Strikingly, level Poly dramatically decreased                               expressed poly adult fly eyes, using InR05545 mutant larvae, level Poly Akt1                             GMR-Gal4 construct drive expression UAS-poly <br /> mutant larvae unaffected (Akt1 situated downstream                             transgene [33]. 278C, led rough eye (disorganized InR; figure 3c). level poly mRNA unchanged                              ommatidia) phenotype (figure 4a). Disruption InR&#226;&#8364;&#8220; TOR InR mutant, suggesting difference protein                          signalling mutation dAkt1 dS6K led level instability decreased protein syn-                           striking suppression poly-induced rough eye pheno- <br /> thesis Poly absence InR (figure 3d).                                     type (figure 4b,c), suggesting intact InR &#226;&#8364;&#8220;TOR <br />     evidence connection Poly                                signalling cascade required Poly exert effects. InR provided phenotypic analysis InR05545                               experiments critically demonstrate cell auton- <br /> mutant larvae (figure 3e). allele reported                             omous importance Poly insulin signalling. <br /> previously embryonic lethal [32], noticed                                 hypothesized dAkt dS6K mutants act small number homozygous InR05545 animals reached                               suppressors poly expression phenotype, instar larval stage. Strikingly, phenotype                               activity kinases altered poly mutant ani- InR mutant larvae resembled observed poly                               mals. probed early instar larval extracts <br /> larvae increased larval lifespan development                              antibodies recognize specifically phos- <br /> melanotic masses (figure 3e). observations demonstrate                          phorylated (active) forms dAkt dS6K.  loss InR leads decrease Poly InR05545                        phosphorylation dAkt dS6K decreased <br /> mutants exhibit phenotypic features similar poly                          mutation poly (figure 4d,e). data reveal mutant larvae.                                                                        activity dAkt dS6K kinases decreased <br />                                            +                  GMR-Gal4&gt;UAS-pol                   (d)                          wt     poly                                    6 <br />                                             +                          +                             64 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             rsob.royalsocietypublishing.org <br />                                                                                                                                                     p-dAkt <br />                                                                                                                       51 <br />                                                                                                                       64 <br />                                                                                                                                                     dAkt <br />  <br />  <br />                                                                                                                       51                            &#194;&#181;-tub <br />  <br />  <br />                                                                                                   (e)                           wt        poly <br />  <br />  <br />  <br />  <br />                                                                                                                                                                             Open Biol 2: 110031 <br />                         (b) GMR-Gal4&gt;UAS-poly                 GMR-Gal4&gt;UAS-poly                                64                                   p-dS6K <br />                                           TM3                             Akt1                                 51 <br />  <br />                                                                                                                       51                            &#194;&#181;-tub <br />  <br />  <br />                                                                                                   (f)                                       ** <br />                                                                                                                        0.8 <br />  <br />  <br />  <br />  <br />                                                                                                    4E-BP expression <br />                                                                                                                        0.6 <br />  <br />                                                                                                                        0.4 <br />  <br />                                                                                                                        0.2 <br />                         (c) GMR-Gal4&gt;UAS-poly                 GMR-Gal4&gt;UAS-poly <br />                                                                                                                            0 <br />                                           TM3                             S6K                                                        wt           poly <br />  <br />  <br />                                                                                                   (g)                           wt         poly <br />  <br />                                                                                                                       19                                 p-4E-BP <br />  <br />  <br />                                                                                                                       51                                 &#194;&#181;-tub <br />  <br /> Figure 4. Phosphorylation levels Akt S6K decreased poly larvae.  &#226;&#8364;&#8220; c) Images showing Drosophila compound eyes.   rough eye phenotype caused UAS-poly overexpression control GMR-Gal4 driver. (b,c) rough eye phenotype caused UAS-poly overexpression control GMR-Gal4 driver suppressed Akt104226 S6K07084 mutations. TM3 represents balancer chromosome carried control sibling progeny. (d,e) <br /> Immunoblotting instar larval extracts using phospho-specific antibodies dAkt dS6K reveals decreased phosphorylation kinases poly larval <br /> extracts. (f ) d4E-BP transcript levels assessed real-time qPCR RNA isolated wild-type poly mutant larvae. d4E-BP levels normalized Actin5C levels. error bars derive reactions performed biological triplicate samples. double asterisk represents significant difference ( p , 0.01) 4E- <br /> BP expression levels wild-type poly larvae. (g) Immunoblotting instar larval extracts showing decreased phosphorylation d4E-BP poly <br /> compared wild-type larvae. <br />  <br />  <br /> Table 1. Mass spectrometry identi&#239;&#172;?es insulin receptor physical interactor Poly. Analysis pre-immune immunoprecipitation samples tandem <br /> mass spectrometry identi&#239;&#172;?ed interacting proteins speci&#239;&#172;?cally immune-precipitate sample. Insulin receptor identi&#239;&#172;?ed bands 2 3, highest scores. &#226;&#8364;&#732;exp score&#226;&#8364;&#8482; quanti&#239;&#172;?es log scale (base 10) expectation hit achieved chance, calculated using program X!TANDEM. <br />  <br />   band                    exp score                     unique/total peptides                             protein <br />  <br />   1                       23                            1/1                                               ATP-dependent RNA helicase (Rm62) <br />   2                       213                           4/7                                               insulin-like receptor <br />   3                       213                           4/7                                               insulin-like receptor <br />   4                       25.3                          3/3                                               cadherin <br />   5                       22.5                          2/2                                               transient receptor potential cation channel subfamily (TrpA1) <br />  <br />  poly mutant animals. addition, level p-Akt                                      crucial downstream effector TOR 4E-BP, tissue level, examined immunofluorescence                                   translation initiation factor eIF-4E-binding protein. Phos- mount instar larval brains, additionally corro-                              phorylation 4E-BP TOR leads dissociation borates decrease (electronic supplementary material,                                m7G-cap-binding protein eIF-4E, allowing acti- <br /> figure S2).                                                                              vation cap-dependent translation consequent <br />  positive effects cell growth [34]. Disruption InR&#226;&#8364;&#8220; TOR       detect mitosis apoptosis, respectively (figure 5c). Mitotic      7 <br /> signalling causes inhibition cap-dependent translation       figures, rare latest stages, normal  <br />  <br />  <br />  <br />                                                                                                                                       rsob.royalsocietypublishing.org decrease 4E-BP phosphorylation results binding         appearance (data shown). generation mosaic eIF-4E [14]. Disruption signalling causes increased     imaginal discs allows side comparison wild- <br /> transcription 4E-BP owing increased FoxO activity.       type versus mutant cells, generated poly loss function <br /> sistent decrease InR &#226;&#8364;&#8220;TOR signalling, level       clones imaginal discs. Staining mosaic discs revealed <br /> d4E-BP transcript threefold greater poly             increased levels cleaved caspase-3 poly mutant clones, <br /> mutant larvae compared control animals (figure 4f ).          discernible lack &#195;&#376;-galactosidase addition, immunoblotting revealed d4E-BP               (figure 5d ). Cleaved caspase-3 detectable hypo-phosphorylated poly mutant extracts compared               adjacent wild-type cells. wild-type extracts (figure 4g). Taken               independent lines evidence demonstrate <br /> results consistent decrease cap-dependent            loss poly leads activation autophagy coupled <br />  <br />  <br />  <br />  <br />                                                                                                                                       Open Biol 2: 110031 <br /> translation loss Poly function.                            increase apoptotic cell death. <br />  <br />  <br /> 3.6. Autophagy constitutively active fat body            3.8. Metabolism affected poly mutant larvae <br />      poly larvae                                                Drosophila frequently used study metabolic regulation                                                                    fruitflies share majority metabolic functions ver- critical consequence activation InR &#226;&#8364;&#8220;TOR signal- <br />                                                                    tebrates [42]. larval fat body main organ ling inhibition autophagy. Drosophila undergoes <br />                                                                    regulation energy homeostasis excess energy stored <br /> developmentally programmed autophagy defined times <br />                                                                    form glycogen triglycerides (TAGs; lipids). Acti- facilitate tissue remodelling metamorphosis <br />                                                                    vation signalling InR pathway promotes [35 &#226;&#8364;&#8220;38]. hand, starvation-induced autophagy <br />                                                                    anabolic metabolism storage nutrients  response nutrient deprivation) takes place specifi- <br />                                                                    TAGs [43]. important development, break- <br /> cally larval fat body [11]. larval fat body <br />                                                                    larval fat body consequent release energy <br /> serves similar function vertebrate liver acting <br />                                                                    facilitates Drosophila metamorphosis. nutrient storage organ, commonly used                                                                        order identify genes differentially <br /> examine autophagy starvation-induced                                                                    expressed poly mutant larvae, carried microarray <br /> developmentally regulated autophagy [11,39,40].                                                                    analysis. list 106 genes downregulated poly <br /> fat body fed wild-type larvae does normally exhibit <br />                                                                    mutants, functional enrichment group analysis identified autophagy, autophagy evident short period                                                                    strong enrichment gene ontology   terms belonging <br /> <span id='am-110' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-111' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-114' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> starvation. <br />                                                                    metabolic processes, suggesting metabolism     TOR directly inhibits starvation-induced autophagy, <br />                                                                    affected poly mutants (table 2). investi- components InR signalling  InR Akt, <br />                                                                    gated decrease InR &#226;&#8364;&#8220;TOR activity acting upstream TOR) behave negative regulators <br />                                                                    manifest metabolic level storage TAGs autophagy. hypothesized poly acts InR signal- <br />                                                                    poly mutants. TAG levels normalized total protein <br /> ling, state autophagy visualized Lysotracker <br />                                                                    accurate quantification lipids unit mass staining altered poly mutant fat body compared <br />                                                                    wild-type poly mutant larval extracts. Consistent wild-type fat body. anticipated, Lysotracker <br />                                                                    decrease InR&#226;&#8364;&#8220;TOR signalling, TAG : protein <br /> puncta observed fed wild-type early instar fat <br />                                                                    ratio half detected wild-type larvae (figure 6a). <br /> body (figure 5a, fed).  Lysotracker puncta                                                                        <span id='am-12' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-13' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>Lipid</span> storage droplet-2 protein (Lsd-2), protector apparent following 4 h amino acid starvation, owing                                                                    lipolysis Drosophila perilipin homologue, localized activation autophagy (figure 5a, starved). Strikingly, <br />                                                                    outer membrane lipid droplets, main storage <br /> Lysotracker puncta abundant fed poly fat <br />                                                                    organelle TAGs [44,45]. level Lsd-2 <br /> bodies, demonstrating autophagy constitutively <br />                                                                    poly larval extracts decreased relative wild-type <br /> active fat body poly mutants (figure 5b, compare <br />                                                                    fat body, consistent decreased TAG : protein ratio <br /> fed starved). <br />                                                                    observed poly mutants (figure 6b). <br />                                                                         reduction TAG storage Lsd-2 protein <br /> 3.7. Poly loss function leads increase                 poly mutant larvae suggest decrease anabolic <br />                                                                    metabolism, consistent expected consequence <br />      apoptotic cell death                                          diminished InR&#226;&#8364;&#8220; TOR signalling. <br /> Autophagy apoptosis closely related, increased <br /> levels autophagy lead apoptosis [39,41]. example, increased autophagy resulting clonal <br />                                                                    3.9. Poly protein relocalizes following insulin expression Atg1 wing disc resulted elevated           stimulation Drosophila haemocytes human <br /> apoptosis, evidenced appearance cleaved <br /> caspase-3 clones [39]. <br />                                                                         cultured cells <br />     loss poly led constitutive activation auto-   Given physical interaction Poly InR func- <br /> phagy fat body, investigated loss         tional links Poly InR&#226;&#8364;&#8220; TOR signalling, poly resulted increased apoptosis.  cell death      interested determine level distribution <br /> increased dramatically instar larval neuroblasts       Poly cell dependent InR activity. <br /> poly larvae aged. readily seen larval neuro-             Strikingly, staining Poly noticeably increased fol- <br /> blasts stained pS10-histone H3 TUNEL               lowing insulin stimulation larval haemocytes (cells                                                   wild-type                            (b)                            poly                                                     8 <br />  <br />                                       fed                             starved                           fed                          starved <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               rsob.royalsocietypublishing.org <br />                  Lysotracker <br />                    DNA Lysotracker <br />  <br />  <br />  <br />  <br />                                                                                                                                                                               Open Biol 2: 110031 <br />                                         (c) 8 <br />                                             7 <br />                                             6 <br />                                             5 <br />                                         % cells <br />  <br />  <br />  <br />  <br />                                             4 <br />                                             3 <br />                                             2 <br />                                             1 <br />                                             0 <br />                                                   6-wt 6-het      6       8     10    12     14        16     18      20      22 <br />                                                                                   age (days) <br />  <br />                               (d)                                     b-gal                      caspase-3                          DAPI <br />  <br />  <br />  <br />  <br /> Figure 5. poly mutations leads constitutive autophagy increased cell death.  b) Lysotracker  acidic component-specific fluorescent dye) used monitor state autophagy early instar larval fat body. Autophagy indicated punctate Lysotracker appearance. Live fat body tissues stained Lysotracker (red) Hoechst 33342 <span id='am-123' about='rdfs:label' typeof='owl:Thing'>label</span> DNA (blue).   wild-type larval fat body, autophagy evident. indicated Lysotracker positive bright spots, <br /> autophagy active 4 h amino acid starvation 20 cent sucrose solution. (b) Lysotracker puncta detected fed poly fat body, starved poly <br /> fat body.  b) Scale bar, 50 mm. (c) Quantitation mitosis apoptosis larval neuroblasts development. mitotic activity decreases, apoptosis increases extended larval stage poly larva. Blue bars, p H3; red bars, TUNEL. (d) Images showing imaginal disc dissected instar larva genotype <br /> eyFLP; FRT82Bb-Gal/FRT82Bpoly05137. Immunostaining cleaved caspase-3 (red), b-galactosidase (green) DNA (blue) performed. poly mutant clones indicated lack b-Gal staining. Scale bar, 30 mm. <br />  <br />  <br />  <br /> innate immune . Prior isolation haemocytes,                                  insulin stimulation (figure 7a). Nearly, 60 cent cells <br /> larvae starved 3 h 20 cent sucrose. Haemo-                                showed relocalization Poly 60 min insulin stimu- <br /> cytes stimulated culture 200 nM insulin                                 lation, level increasing 100 cent 90 min. <br /> (electronic supplementary material, figure S3). increased                                rapid change HsPoly behaviour dependent <br /> staining Poly appears rapid response detect-                          TOR signalling. Incubation HeLa cells rapamycin <br /> able following 15 min insulin stimulation, levels                             overnight starvation period prevented increase Poly remain elevated 75 min.                                                     HsPoly staining following insulin stimulation (figure 7b). <br />     change appearance Poly following insulin                                  rapamycin-treated cells, HsPoly remained evenly dis- <br /> stimulation conserved human cells. Overnight serum-                                tributed cell following 90 min starved HeLa cells stimulated insulin                                 insulin treatment. <br /> 90 min. Drosophila haemocytes, staining HsPoly                                      Taken  observations demonstrate significantly stronger following insulin stimulation,                                 increase nuclear relocalization HsPoly following insu- <br /> appearing maximal 90 min. additionally appeared                                lin treatment occurs TOR-dependent manner. Overall, increase HsPoly concentrated near                                   results identify Poly novel component InR&#226;&#8364;&#8220; TOR <br /> nucleus, relocalization evident 30 min                                 signalling conserved flies humans. <br />                                                           1.2                        (b)                                                                                 9 <br />                                                                                                      wt       poly <br />  <br />  <br />  <br />  <br />                                     triglyceride:protein <br />                                                            1.0 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         rsob.royalsocietypublishing.org <br />                                                                                       51 <br />  <br />  <br />  <br />  <br />                                          normalized <br />                                                            0.8 <br />                                                                        **                                                     Lsd-2 <br />                                                            0.6 <br />                                                                                       39 <br />                                                            0.4 <br />                                                            0.2                                                                &#194;&#181;-tub <br />                                                                                       51 <br />                                                             0 <br />                                                                  wt    poly <br /> Figure 6. poly mutant larvae characterized reduced levels TAGs Lsd-2.   TAG levels assessed wild-type poly mutant instar larvae. total TAG level normalized total protein level. Error bars derive standard deviation independent experiments. unpaired  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         Open Biol 2: 110031 <br /> tailed t-test used compare data wild-type poly larvae. double asterisk represents significant difference ( p , 0.01) TAG : protein ratio control poly mutant larvae. (b) Immunoblotting carried wild-type poly mutant larval extracts antibody specifically recognizing <br /> Lsd2 protein. <br />  <br />  <br />  <br /> Table 2. Microarray analysis revealed metabolic functions genes downregulated poly mutants. DAVID  Database Annotation, Visualization Integrated Discovery) functional enrichment group analysis list differentially expressed genes identi&#239;&#172;?ed enrichment gene ontology   terms belonging metabolic processes genes downregulated poly mutants. data analysed Limma  &#194;&#167;5.10) biological replicates. Genes selected functional/pathway analysis adjusted (corrected multiple testing) p-value 0.05. Log (base 2) fold changes given (log2FC). &#226;&#8364;&#732;rep&#226;&#8364;&#8482; indicates reported gene function, &#226;&#8364;&#732;pred&#226;&#8364;&#8482; indicates predicted gene function according gene-speci&#239;&#172;?c FlyBase report. <br />  <br />   TERM                                                                      affected gene (log2FC)                                   fold-enrichment <br />  <br />   0006739 NADP metabolic process                                            FBgn0004057 // CG12529 (21.42)                                   81.9 <br />                                                                                                                         rep <br />                                                                                glucose-6-phosphate dehydrogenase <br />                                                                                FBgn0023477 // CG2827 (21.27) <br />                                                                                                rep <br />                                                                                transaldolase <br />                                                                                FBgn0004654 // CG3724 (22.15) <br />                                                                                                                   rep <br />                                                                                phosphogluconate dehydrogenase <br />                                                                                FBgn0030239 // CG17333 (21.34) <br />                                                                                6-phosphogluconolactonase pred <br />                                                                                FBgn0037607 // CG8036 (21.39) <br />                                                                                transketolase pred <br />   0006769 <span id='am-117' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-118' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span><span id='am-121' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span>nicotinamide</span> metabolic process                                    &#239;&#172;?ve genes                                                   64.4 <br />   0019362 pyridine <span id='am-102' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-103' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-106' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-107' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> metabolic process                             &#239;&#172;?ve genes                                                   60.1 <br />   0006006 glucose metabolic process                                         &#239;&#172;?ve genes,                                              16.4 <br />                                                                                FBgn0002569 // CG8694 (20.89) <br />                                                                                maltase A2 rep <br />  <br />                                                                                        growth. Phosphorylation Akt substrates contributes 4. Discussion                                                                          range cellular processes including cell growth, proliferation <br /> Poly conserved protein plays novel essential role                       survival [46]. Dysregulation Akt involved various <br /> InR signalling  crucially, promotes effects TOR                         diseases, including type-2 diabetes cancer [47,48]. S6K <br /> kinase cell growth metabolism.                                                  kinase downstream effectors InR&#226;&#8364;&#8220; TOR <br />                                                                                        signalling, subject phosphorylation activation <br />                                                                                        TOR. Activation S6K leads increase translation <br /> 4.1. activity InR signalling decreased                                    phosphorylation activation ribosomal <br />                                                                                        protein S6 [1]. <br />      poly larvae                                                                           Poly acts upstream kinases InR data reveal essential involvement Poly InR sig-                          signalling pathway, levels phosphorylated (active) <br /> nalling, important pathway linking nutritional status                            dAkt dS6K kinases decreased poly larvae. metabolism cell growth [3]. kinases, Akt                               Poly acts activation dAkt dS6K S6K, key effectors InR &#226;&#8364;&#8220;TOR signalling. Akt                                 supported genetic data revealed suppression kinase located upstream TSC1&#226;&#8364;&#8220;TSC2 complex                                 poly-induced rough eye phenotype mutations dAkt pathway critical role promoting cell                                 dS6K. kinases act genetic <br />                                                                     poly                  DNA                    actin                                                 10 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         rsob.royalsocietypublishing.org <br />                                     0 <br />                                     30 min <br />                                     60 min <br />  <br />  <br />  <br />  <br />                                                                                                                                                                         Open Biol 2: 110031 <br />                                     90 min <br />  <br />  <br />  <br />  <br />                                    (b) + rapamycin <br />                                     0 <br />                                     30 min <br />                                     60 min <br />                                     90 min <br />  <br />  <br />  <br />  <br /> Figure 7. Insulin stimulation HeLa cells leads increase HsPoly staining, accumulated nuclear area.   HeLa cells serum-starved overnight stimulated 100 nM insulin 30, 60 90 min prior staining HsPoly (red), DNA (blue) F-actin (green). Staining HsPoly indicated peri- <br /> nuclear accumulation 30 min, reaching maximum following 90 min insulin stimulation. (b) relocalization HsPoly inhibited overnight <br /> incubation cells 20 nM rapamycin prior insulin treatment. Scale bars, 50 mm. <br />  <br /> suppressors situated downstream poly sig-                            data predict decrease InR&#226;&#8364;&#8220;TOR signalling <br /> nalling pathway, resulting overall decrease InR &#226;&#8364;&#8220;TOR                          absence Poly. read TOR signalling, signalling absence Poly.                                                     examined autophagy, multi-step, catabolic process used     biochemical data presented revealed phys-                             nutrient recycling development starvation. TOR <br /> ical interaction Poly InR. Intriguingly, level                          activity responsible inhibiting autophagy cell <br /> Poly reduced InR mutant larval extracts indepen-                                growth-promoting conditions. finding autophagy dently transcription, suggesting fraction Poly                            constitutively active fat body poly mutant larvae unstable subject degradation absence                              accord observation reduced InR&#226;&#8364;&#8220;TOR signalling, InR.  numerous examples instabil-                          consistent inhibition cell growth prolifer- <br /> ity partner protein complex absent [49&#226;&#8364;&#8220;51].                             ation poly mutant animals. Furthermore, increase Poly exists discrete complex InR,                                apoptotic cell death poly mutant animals clones generated complex(es) regulated response                                   imaginal discs result increased levels autophagy <br /> insulin stimulation, currently investigation.                           mutation poly. interaction transient highly dynamic, Poly appears accumulate nucleus following <br /> insulin stimulation. Considering insulin-induced, TOR- <br />                                                                                        4.2. Metabolism disrupted poly mutants <br /> dependent nuclear enrichment Poly HeLa cells,                              major function InR&#226;&#8364;&#8220; TOR pathway modulation <br /> likely insulin-mediated signalling Poly conserved                          metabolism. example, 4E-BP mutant animals flies humans.                                                                  increased sensitivity starvation [16]. Interestingly,  previously shown loss tumour suppressor                 Poly role transcription, phylogenetic       11 <br /> PTEN (responsible dephosphorylation PIP3)            data suggest, explain binding transmem- <br />  <br />  <br />  <br />  <br />                                                                                                                                            rsob.royalsocietypublishing.org <br /> Drosophila nurse cells results accumulation activated    brane receptor? explanation dynamic changes Akt cytoplasm. activated Akt drives formation      cellular localization Poly, occurring TOR-depen- enlarged lipid droplets increase           dent manner. Interestingly, InR phosphorylation IRS-1 <br /> expression Drosophila Lsd-2 [52]. Lsd-2 mutants             IRS-2  human IRS homologues) characterized decreased TAG levels [45]. Interestingly,         leads activation PI3K signalling, associated <br /> autophagy lipid metabolism shown inter-         IRS-1 translocation nucleus, activates <br /> linked processes, suggested decrease autophagy          transcription various genes [59 &#226;&#8364;&#8220;61]. Given Poly resulting increase lipid storage cell [53].    interacts InR, moves nucleus following insu- <br /> sistent decreased InR&#226;&#8364;&#8220; TOR signalling absence          lin stimulation, possible changes localization poly, Lsd-2 TAG : protein ratio reduced       Poly ultimately response InR signalling. <br />  <br />  <br />  <br />  <br />                                                                                                                                            Open Biol 2: 110031 poly mutant animals.  propose Poly            Immunofluorescence Drosophila haemocytes affects metabolism interaction InR&#226;&#8364;&#8220;TOR path-      HeLa cells demonstrated level distribution <br /> way, acting positive regulator anabolic metabolism.        Poly significantly affected following insulin stimulation. <br />                                                                    Importantly, HeLa cells, Poly relocalization nucleus <br /> 4.3. Comparison poly mutants                       occurred rapamycin-sensitive manner.  speculate <br />                                                                    stimulatory effects Poly cell growth metab- <br />      Elongator                                                     olism exerted effects transcription.  <br /> Numerous aspects poly phenotype observed          analysis required assess Poly participates mutations, including InR TOR.              Elongator complex   myriad <br /> extreme larval longevity, remarkable aspects       functions currently ascribed Elongator. poly mutant phenotype, observed Tor             Future research address detailed nature [54] InR mutants  study). appearance melano-       interaction Poly InR. Does Poly interact tic masses frequently seen mutations aberrant           InR absence presence insulin? dynamic <br /> immune responses haematopoietic defects [55&#226;&#8364;&#8220;57].            interaction? level post-translational state poly instar larvae, melanotic masses appearing         Poly modified insulin treatment? <br /> multiple different locations larval body likely           light data presented  propose Poly result alteration immune response         novel mediator InR&#226;&#8364;&#8220;TOR signalling regulation hyper-proliferation blood cells, haemocytes.        cell metabolism growth Drosophila (figure 8). suggest observation InR mutant larvae reach          physical interaction Poly InR followed instar stage develop melanotic masses high-         translocation Poly nucleus  insulin stimu- <br /> lights phenotypic similarities poly InR loss         lation), expression key metabolic genes affected, <br /> function mutations, corroborates functional          contributing promotion cell growth metab- <br /> relationship poly InR.                                 olism. detailed nature regulation Poly&#226;&#8364;&#8482;s <br />     Given apparent similarity Poly yeast Elp6,     interaction InR remain elucidated, highly intriguing recently described mutant phe-       likely  given conservation Poly, crucial interaction <br /> notype Elongator Elp3 catalytic subunit Drosophila      function hold true human cells. <br /> resembles poly [28]. observations suggestive role Elongator holocomplex insulin signalling&#226;&#8364;&#8221; histone tubulin acetylation transla-        5. Material methods <br /> tional control tRNA modification. addition, human mutations Elp3 linked familial            5.1. Drosophila culture <br /> dysautonomia [58], likely future analysis                                                                    fly stocks maintained 258C standard medium <br /> Elongator function model organisms significant <br />                                                                    unless stated. Fly stocks used study  <br /> preclinical value. <br />                                                                    Canton S (wild-type), poly05137/TM6B. following fly <br />                                                                    lines obtained Bloomington Stock Center: <br /> 4.4. Model Poly action                                         InR05545/TM3, Akt104226/TM3, S6K07084/TM3, Cg-Gal4. shown Poly binds InR  transmembrane <br /> receptor) modulates activity various downstream <br /> proteins. lack clearly discernible functional <br />                                                                    5.2. Phylogenetic analysis <br /> motifs hampered prediction Poly&#226;&#8364;&#8482;s molecular function,         Sequences homologous Drosophila Poly identified extensive database searches phylogenetic analyses ident-       multiple iterative searches using PSI-BLAST program [62] <br /> ified Poly member Elp6 subfamily Elongator          HHpred interactive server [63]. Alignments proteins, distant homology various members            corresponding sequences generated using M-COFFEE RecA/Rad51/DCM1 superfamily  KaiC               [64] manually adjusted based predicted secondary <br /> RadB); observations described distinct study        structure according ALI2D [65]. Identical similar laboratory. KaiC Elp6 shown            amino acids representative subset aligned sequences significantly affect gene expression, pointing degree      shaded using BOXSHADE server (http://www.ch. functional conservation.  suggest involvement        embnet.org/software/BOX_form.html). Phylogenetic trees Poly transcription ( relocated      calculated using TREE-PUZZLE [66], MRBAYES [67] nucleus following insulin stimulation).                            PROML program PHYLIP Package [68] clustering <br />                                                                                            (b)                                                                                           12 <br />                                                         insulin                                                               insulin <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           rsob.royalsocietypublishing.org <br />                                                     InR                                                                  InR <br />  <br />                                                                                                              Poly <br />                                    Poly                            Akt                                                               Akt <br />  <br />  <br />                                          positive                           TOR                                                             TOR <br />                                         regulation <br />  <br />                                           lsd2              S6K                                                  lsd2       S6K <br />                                                                                                    cytosol <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                           Open Biol 2: 110031 <br />                                                       autophagy               4E-BP                                  autophagy                4E-BP <br />  <br />  <br />  <br />                                            Poly          transcription                                                    transcription <br />  <br />                                               metabolic genes                                                            disrupted <br />  <br />                                                     wild-type                                                           loss poly <br /> Figure 8. Model function Poly InR&#226;&#8364;&#8220;TOR signalling pathway. interaction Poly   insulin receptor allows fulfil role positive regulator InR&#226;&#8364;&#8220;TOR signalling, overall effect increasing phosphorylation   activity numerous positive regulators InR&#226;&#8364;&#8220;TOR pathway, Akt, S6K 4E-BP, inhibiting negative regulators cell growth autophagy. Furthermore, Poly aids promote anabolic metabolism resulting increase TAG Lsd-2 levels. (b) absence Poly leads decrease activation positive regulators cell growth Akt S6K kinases. 4E-BP protein hypo- <br /> phosphorylated, consistent overall downregulation InR&#226;&#8364;&#8220;TOR signalling. Furthermore, autophagy negative effector InR/TOR signalling constitutively <br /> active. Anabolic metabolism downregulated indicated decrease TAG Lsd-2 levels. <br />  related sequences smaller groups using SPLITSTREE4 [69].                                  MS). high-performance liquid chromatography (HPLC) <br /> Branch lengths calculated application WAG                                          instrument (Dionex, UK) interfaced LTQ <span id='am-44' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-45' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> <br /> substitution matrix [70] using TREE-PUZZLE, modelling rate                                        trap mass spectrometer (ThermoFinnigan, CA). Chromato- <br /> heterogeneity according gamma distribution 16                                           graphy buffer solutions (buffer  5% acetonitrile 0.1% <br /> rate categories, bootstrap confidence intervals ( provided                                    <span id='am-140' about='obo:IMR_0200110' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-141' property="oboInOwl:id" content="IMR:0200110" datatype="xsd:string"></span><span id='am-142' property="oboInOwl:hasDbXref" content="KEGG:C00058" datatype="xsd:string"></span><span id='am-143' property="rdfs:label" content="Formic acid" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formic acid</span>; buffer B: 80% acetonitrile 0.1% formic electronic supplementary material, figure S1) esti-                                       acid) used deliver 60 min gradient (35 min 45% <br /> mated using SEQBOOT program PHYLIP Package [68].                                       buffer B, 10 min 90%, hold 10 min, 3 min 5% hold <br />                                                                                                   15 min). flow rate 2 ml min21 used electro- <br /> 5.3. Immunoprecipitation                                                                          spray source. scan followed 10 MS/MS <br />                                                                                                   events, duty cycle programmed enforce dynamic <br /> Fifty microlitres 0&#226;&#8364;&#8220; 5 h wild-type embryos (raised 188C)                                    exclusion 2 min. house proteomics pipeline software homogenized 1 ml cold lysis buffer (100 mM NaCl,                                       (PROLINE) used process data. Spectra searched <br /> 10 mM EDTA, 50 mM Tris &#226;&#8364;&#8220;HCl pH 7.6, 0.1% Triton-X100,                                             using Sequest algorithm [72] SwissProt.2007.04.19 <br /> 10 mg ml21 chymostatin, leupeptin, antipain pep-                                      database restricted Drosophila melanogaster entries. Proteins <br /> statin, 50 mg ml21 PMSF) briefly sonicated.                                           <span id='am-1' about='obo:IMR_0001700' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0001700" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="peptide" datatype="xsd:string"></span><span id='am-5' property="obo:IAO_0000115" content="The polypeptide that is not a gene product." datatype="xsd:string"></span>peptide</span> prophet probability score greater 0.99 <br /> microlitres rabbit pre-immune anti-poly serum coupled                                       [73] (ii) identified minimum different peptide 40 ml Dynal beads (Invitrogen) added pre-                                           spectra automatically accepted, spectra cleared embryo lysates overnight. cent                                            minority proteins identified single spectra sample subjected immunoblot analysis verify suc-                                        manually checked quality. <br /> cessful immunoprecipitation Poly, remaining <br /> 90 cent sample resolved precast 4&#226;&#8364;&#8220;12 cent <br /> Bis-Tris polyacrylamide gels (Novex) stained                                             5.5. Larval manipulations <br /> Colloidal Coomassie Blue (Invitrogen). Comparable molecu-                                         Prior dissections protein extractions, 10&#226;&#8364;&#8220;15 second instar <br /> lar weight regions excised lane                                        larvae transferred fresh vial food supplemented <br /> ( pre-immune- immune-precipitations) mass spec-                                           fresh yeast paste. Manipulations carried trometry analysis performed (Dr Gerard Cagney, Dublin).                                           larvae reached early instar stage. starvation exper- <br />                                                                                                   iments, early instar larvae starved 20 cent <br /> 5.4. Mass spectrometry                                                                            sucrose solution 3&#226;&#8364;&#8220;4 h prior dissection. proteins slices sodium dodecyl sulphate poly- <br /> acrylamide gel electrophoresis (SDS &#226;&#8364;&#8220;PAGE) gels                                              5.6. RNA extraction quantitative reverse <br /> digested gel trypsin method Shevchenko <br /> et al. [71]. resulting peptide mixtures resuspended <br />                                                                                                        transcriptase &#226;&#8364;&#8220;polymerase chain reaction 1 cent formic acid analysed nano-electrospray                                       Larvae transferred RNase-free Eppendorf tubes <br /> liquid chromatography mass spectrometry (nano-LC MS/                                              (Ambion). RNA extracted Qiagen RNeasy <br />  Plus Kit (Qiagen Hilden, Germany) according manufac-            (Molecular Probes), 1 mM Hoechst 33342 (Molecular              13 <br /> turer&#226;&#8364;&#8482;s instructions. cDNA synthesis carried            Probes) PBS. Fat bodies mounted phosphate- <br />  <br />  <br />  <br />  <br />                                                                                                                                        rsob.royalsocietypublishing.org <br /> using SuperScript III (Invitrogen) following manufac-      buffered saline (PBS) glass slide visualized using <br /> turer&#226;&#8364;&#8482;s instructions. Real-time quantitative RT-PCR analysis       Olympus AX-70 Provis epifluorescence microscope performed LightCycler (Roche) using          Hamamatsu Orca II charge coupled device (CCD) camera. <br /> universal probe library. reactions set following       Images captured using SMARTCAPTURE 3 processed <br /> manufacturer&#226;&#8364;&#8482;s recommendation LightCycler                 using Adobe PHOTOSHOP. <br /> master mix kit. relative cDNA ratio calculated Lightcycler software 480. Actin5C used control normalize equal loading template cDNA. <br />                                                                    5.10. Microarray processing analysis <br />                                                                    second instar wild-type Canton S poly larvae <br />                                                                    transferred vials supplemented fresh yeast paste. <br /> 5.7. Preparation larval protein extracts  <br />  <br />  <br />  <br />                                                                                                                                        Open Biol 2: 110031 <br />                                                                    Early instar larvae transferred TRIzol 24 h later <br />      immunoblotting                                                RNA extraction. microarray processing Flychip <br />                                                                    team University Cambridge (http://www.flychip.org. wandering instar larvae appro- <br />                                                                    uk/). RNA wild-type Canton S poly early instar <br /> priate genotype placed 1.5 ml tubes rinsed                                                                    larvae extracted (medium scale) using TRIzol (http:// <br /> times Ephrussi&#226;&#8364;&#8220;Beadle Ringer&#226;&#8364;&#8482;s solution (130 mM <br />                                                                    www.flychip.org.uk/protocols/gene_expression/standard_ <br /> NaCl, 4.7 mM KCl, 1.9 mM CaCl2, 10 mM HEPES pH <br />                                                                    extraction.php). RNA reverse transcribed labelled using <br /> 6.9).  300ml cold lysis buffer (Ephrussi&#226;&#8364;&#8220;Beadle Ringer&#226;&#8364;&#8482;s <br />                                                                    Klenow labelling: 5 mg total RNA reverse transcribed solution 10 mM EDTA, 10 mM DTT, 1 mg ml21                                                                    cDNA (anchored oligo (dT)23 (Sigma), Superscript III (Invitro- chymostatin, leupeptin, antipain pepstatin [CLAP, <br />                                                                    gen)) second strand synthesis performed (Second <br /> Sigma], 1 mM phenylmethanesulphonyl fluoride [PMSF, <br />                                                                    strand buffer (Invitrogen), DNA Polymerase (Invitrogen), <br /> Sigma], 1 unit Aprotinin [Calbiochem] added                                                                    RNaseH (New England Biolabs), Escherichia coli DNA ligase tubes. larvae homogenized using motor- <br />                                                                    (GE Healthcare)) obtain double-stranded DNA. Random <br /> ized hand pestle starting lowest speed gently <br />                                                                    primers annealed 500 ng denatured DNA template <br /> increasing speed maximum approximately <br />                                                                    extended Klenow fragment using Bioprime <br /> 2 min.  150 ml hot (708C) 3X SDS-PAGE Sample <br />                                                                    DNA Labeling (Invitrogen), fluorescent dyes <br /> Buffer containing 10 mM DTT added homogenate <br />                                                                    Cy3-<span id='am-124' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-125' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-127' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span>dCTP</span> Cy5-dCTP (GE Healthcare) incorpora- tube placed 1008C 10 min. Particulate matter <br />                                                                    ted (http://www.flychip.org.uk/protocols/gene_expression/ pelleted 13 000 r.p.m. 2 min, supernatant <br />                                                                    klenow_v2.php). Hybridization FL002 microarrays: hybridiz- <br /> transferred fresh tube. Samples stored 2208C <br />                                                                    ation amino-modified long oligonucleotide microarrays required. <br />                                                                    using Genomic Solutions hybridization station Bioso- <br />                                                                    lutions hybridization buffer (http://www.flychip.org.uk/ <br /> 5.8. Sodium dodecyl sulphate polyacrylamide gel                    protocols/gene_expression/hyb_oligoMWG.php). Scanning <br />                                                                    Genepix 4000B dual laser scanner 5 mM <br />      electrophoresis immunoblotting                            pixel resolution (http://www.flychip.org.uk/protocols/gene_ <br /> Protein samples resolved 170 V SDS-PAGE              expression/scanning2.php). Spot finding quantitation <br /> precast 4&#226;&#8364;&#8220;12% Bis-Tris polyacrylamide gels (Novex)             DAPPLE package. Raw data normalized transferred nitrocellulose membranes Trans-Blot          vsn method BIOCONDUCTOR (http://www.bioconductor. <br /> apparatus (Biorad). Membranes blocked TBSTw (Tris-         org/packages/2.8/bioc/html/vsn.html) generate log <br /> buffered saline (150 mM NaCl, 20 mM Tris pH 7.5) &#195;&#190; 0.05%           (base 2) fold changes average signals. Differential expres- <br /> Tween 20) 5% (w/v) semi-skimmed milk 1 h room           sion tested using Limma, BIOCONDUCTOR (http:// <br /> temperature incubated 1 h primary antibody       www.bioconductor.org/packages/2.8/bioc/html/limma.html). TBSTw. washing times 5 min TBSTw,      KEGG biological pathway enrichment differen- <br /> membranes incubated appropriate horseradish         tially expressed gene set assessed using DAVID <br /> oxidase-linked secondary antibody (1 : 10 000) 1 h TBSTw    functional annotation bioinformatics microarray analysis room temperature. Finally, membranes washed         tool (http://david.abcc.ncifcrf.gov/). Microarray data Tris-buffered saline plus 0.2% Triton X immune-       deposited gene expression omnibus (GEO) complexes detected enhanced chemiluminescence (ECL;             accession number GSE32637 (http://www.ncbi.nlm.nih.gov/ <br /> Amersham Biosciences).                                             geo/query/acc.cgi?acc=GSE32637). <br />      Primary antibodies dilutions used immunoblotting <br /> experiments follows: DmPoly 504 (1 : 1000), anti-Isd2 <br /> (1 : 2000), p-Akt antibody (1 : 1000), p-S6K antibody (1 : 1000) <br />                                                                    5.11. Triglyceride assay larvae p-4E-BP antibody (1 : 200). anti-Isd2 antibody       TAG assay larvae carried described Gronke <br /> generous gift Ronald Ku&#194;&#168;hnlein (<span id='am-18' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-19' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-23' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span>MPI</span>, ttingen),       et al. [75]. Briefly, larvae corresponding gen- purchased Cell Signaling Technology.      otype collected 500 ml 0.05 cent Tween 20                                                                    homogenized using Polytron apparatus, followed treat- <br />                                                                    ment 708C 5 min. Samples centrifuged 1 min 5.9. Lysotracker staining                                          3500 r.p.m. 350 ml supernatant transferred Lysotracker staining performed described [74]. Fat          new vial centrifuged 3 min 2500 r.p.m. hun- <br /> body fed starved larvae dissected PBS,          dred microlitres Thermo Infinity Trig solution (Thermo <br /> incubated 2 min 100 nM LysoTracker Red DND-99               Electron, 981786) added 75 ml isolated supernatant <br />  absorbance 540 nm measured following incu-                                     5.14. Immunofluorescence HeLa cells                                                 14 <br /> bation 30 min. Similarly, protein content determined <br />                                                                                               HeLa cells cultured coverslips plates  <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      rsob.royalsocietypublishing.org adding 10 ml isolated supernatant Bradford reagent <br /> (Sigma, B6916) reading absorbance 595 nm. TAG                                      night. Cells rinsed PBS 3 min followed 3 min <br /> levels normalized corresponding protein levels.                                       fixation PBS containing 4 cent paraformaldehyde <br />                                                                                               (diluted 16% ampoules). fixative removed                                                                                               2 min wash PBS. Cells permeabilized 5 min <br /> 5.12. Antibody staining haemocytes                                                         incubation PBS &#195;&#190; 0.5% Triton X-100 followed 1 h <br />                                                                                               block PBS containing 3 cent BSA. Subsequently, cells instar larvae bled multispot microscope slides                                  washed PBS &#195;&#190; 0.1% Triton X-100 5 min. Primary <br /> (Hendley-Essex) using pair forceps 25-gauge                                        antibody diluted PBS &#195;&#190; 0.3% BSA &#195;&#190; 0.1% Triton <br /> needle 20 ml PBS. Cells left settle slide                            X-100 incubated cells 1 h room temperature. <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      Open Biol 2: 110031 <br /> 1 h room temperature humidified chamber allow                                      Cells washed twice 5 min followed 1 h second- <br /> adherence slide. Cells fixed 20 ml 3.7                                ary antibody incubation diluted PBS &#195;&#190; 0.3% BSA &#195;&#190; 0.1% <br /> cent paraformaldehyde PBS 5 min. Cells washed                                     Triton X-100. Subsequently, 45 min washes performed PBS 5 min, followed 5 min permeabilization                                   PBS &#195;&#190; 0.1% Triton X-100. DAPI included PBS &#195;&#190; 0.1% Triton X-100. Following additional 5 min wash                                   penultimate wash 0.1 mg ml21 concentration. PBS, blocking performed incubating cells PBS &#195;&#190;                                       Primary rabbit anti-HsPoly serum used immuno- <br /> 3% BSA 1 h. Cells incubated primary anti-                                  fluorescence experiments 1 : 1000 dilution. Texas-Red <br /> body diluted PBS &#195;&#190; 3% BSA overnight 48C.                                            conjugated goat anti-rabbit (1 : 500) secondary antibody <br /> following day cells washed PBS times incu-                                  Alexa Fluor-488-conjugated phalloidin (1 : 500) bated room temperature secondary antibody diluted                                     purchased Molecular Probes. PBS &#195;&#190; 3% BSA. Following secondary antibody, incu- <br /> bation cells washed twice PBS 5 min. Cells incubated DAPI diluted PBS (1 : 5000 dilution)                                    6. Acknowledgements <br /> 5 min followed final 5 min wash PBS. Coverslips mounted mowiol slide.                                                      authors like express gratitude Dr <br />      Primary rabbit anti-DmPoly serum used immuno-                                     Gerard Cagney (Conway Institute, University College <br /> fluorescence experiments 1 : 1000 dilution. Alexa Fluor-488                              Dublin) invaluable assistance mass spectrometry <br /> anti-rabbit (1 : 500) Alexa Fluor-594 conjugated phalloidin                               experiments. thank Kasper Pedersen (1 : 500) purchased Molecular Probes.                                               Conway Institute Mass Spectrometry Resource exper- <br />                                                                                               imental support, Andreas Stefani bioinformatics <br />                                                                                               support. Emanuela Giacometti, Josefin Fernius, Shubha <br />                                                                                               Gururaja Rao, Swati Naidu Mei Xuan Lye valuable <br /> 5.13. HeLa cell culture                                                                       contributions various stages project. thank <br /> HeLa cells maintained Dulbecco&#226;&#8364;&#8482;s modified Eagle&#226;&#8364;&#8482;s                                     William Earnshaw Paul Hartley critical feedback medium (DMEM; Sigma) supplemented 10 cent foetal                                     manuscript. E.B., S.V., N.C. M.M.S.H. sup- <br /> bovine serum (FBS; Gibco). Cells grown 80&#226;&#8364;&#8220;90% conflu-                                 ported Wellcome Trust. B.N. supported ence overnight serum withdrawal. insulin stimulation,                              Natural Sciences Engineering Research Council serum-starved cells treated 100 nM insulin (Sigma).                                 Canada, V.S. MSc student University Cells treated 20 nM rapamycin (Cell Signaling                                       Edinburgh. D.R.D. supported British Heart Foun- <br /> Technology) overnight serum starvation.                                             dation Centre Research Excellence Award. <br />  <br />  <br />  <br /> References <br /> 1.   Wullschleger S, Loewith R, Hall MN. 2006                 5.   Puig O et al. 2003 Control cell number          9.    Potter CJ, Huang H, Xu T. 2001 Drosophila Tsc1 <br />      TOR signaling growth metabolism.                       Drosophila FOXO: downstream feedback                    functions Tsc2 antagonize insulin signaling <br />      Cell 124, 471&#226;&#8364;&#8220;484. (doi:10.1016/j.cell.2006.                  regulation insulin receptor pathway.                 regulating cell growth, cell proliferation,      01.016)                                                       Genes Dev. 17, 2006&#226;&#8364;&#8220;2020. (doi:10.1101/gad.                 organ size. Cell 105, 357&#226;&#8364;&#8220; 368. (doi:10.1016/ <br /> 2.   Grewal SS. 2009 Insulin/TOR signaling growth               1098703)                                                    S0092-8674(01)00333-6) <br />      homeostasis: view fly world.              6.   Manning BD. 2004 Balancing Akt S6K:              10.   Long X et al. 2005 Rheb binds regulates      Int. J. Biochem. Cell Biol. 41, 1006&#226;&#8364;&#8220; 1010. (doi:10.          implications metabolic diseases                mTOR kinase. Curr. Biol. 15, 702&#226;&#8364;&#8220;713. (doi:10. <br />      1016/j.biocel.2008.10.010)                                    tumorigenesis. J. Cell Biol. 167, 399&#226;&#8364;&#8220;403.                  1016/j.cub.2005.02.053) <br /> 3.   Britton JS et al. 2002 Drosophila&#226;&#8364;&#8482;s insulin/PI3-kinase        (doi:10.1083/jcb.200408161)                           11.   Scott RC, Schuldiner O, Neufeld TP. 2004 Role      pathway coordinates cellular metabolism             7.   Cai SL et al. 2006 Activity TSC2 inhibited         regulation starvation-induced autophagy      nutritional conditions. Dev. Cell 2, 239&#226;&#8364;&#8220;249.                 AKT-mediated phosphorylation membrane                   Drosophila fat body. Dev. Cell. 7, 167&#226;&#8364;&#8220; 178. <br />      (doi:10.1016/S1534-5807(02)00117-X)                           partitioning. J. Cell Biol. 173, 279 &#226;&#8364;&#8220;289. (doi:10.         (doi:10.1016/j.devcel.2004.07.009) <br /> 4.   Hennig KM, Colombani J, Neufeld TP. 2006 TOR                  1083/jcb.200507119)                                   12.   Radimerski T et al. 2002 dS6K-regulated cell growth <br />      coordinates bulk targeted endocytosis         8.   Gao X, Pan D. 2001 TSC1 TSC2 tumor                      dPKB/dPI(3)K-independent, requires dPDK1. <br />      Drosophila melanogaster fat body regulate cell             suppressors antagonize insulin signaling cell            Nat. Cell. Biol. 4, 251&#226;&#8364;&#8220;255. (doi:10.1038/ncb763) <br />      growth. J. Cell Biol. 173, 963&#226;&#8364;&#8220;974. (doi:10.1083/             growth. Genes Dev. 15, 1383&#226;&#8364;&#8220;1392. (doi:10.1101/       13.   Sarbassov DD et al. 2008 Phosphorylation      jcb.200511140)                                                gad.901101)                                                 regulation Akt/PKB rictor-mTOR complex. <br />        Science 307, 1098 &#226;&#8364;&#8220;1101. (doi:10.1126/science.               Acids Res. 30, 3643&#226;&#8364;&#8220;3652. (doi:10.1093/nar/                  Dev. 120, 1071&#226;&#8364;&#8220;1081. (doi:10.1016/S0925-                    15 <br />       1106148)                                                     gkf498)                                                      4773(03)00158-8) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             rsob.royalsocietypublishing.org <br /> 14.   Marr MT et al. 2007 IRES-mediated functional           30.   Figeys D, McBroom LD, Moran MF. 2001 Mass              46.   Manning BD, Cantley LC. 2007 AKT/PKB signaling: <br />       coupling transcription translation amplifies          spectrometry study protein&#226;&#8364;&#8220;protein                navigating downstream. Cell 129, 1261&#226;&#8364;&#8220;1274. <br />       insulin receptor feedback. Genes Dev. 21,                    interactions. Methods 24, 230&#226;&#8364;&#8220; 239. (doi:10.1006/            (doi:10.1016/j.cell.2007.06.009) <br />       175&#226;&#8364;&#8220;183. (doi:10.1101/gad.1506407)                           meth.2001.1184)                                        47.   Bellacosa et al. 2005 Activation AKT kinases 15.   Puig O, Tjian R. 2005 Transcriptional feedback         31.   Shiio Y et al. 2002 Quantitative proteomic analysis          cancer: implications therapeutic targeting. Adv. <br />       control insulin receptor dFOXO/FOXO1. Genes            Myc oncoprotein function. Embo J. 21, 5088&#226;&#8364;&#8220;               Cancer Res. 94, 29 &#226;&#8364;&#8220;86. (doi:10.1016/S0065- <br />       Dev. 19, 2435&#226;&#8364;&#8220;2446. (doi:10.1101/gad.1340505)                5096. (doi:10.1093/emboj/cdf525)                             230X(05)94002-5) <br /> 16.   Teleman AA, Chen YW, Cohen SM. 2005 4E-BP              32.   Fernandez R et al. 1995 Drosophila insulin         48.   Engelman JA, Luo J, Cantley LC. 2006 evolution <br />       functions metabolic brake used stress             receptor homolog: gene essential embryonic             phosphatidylinositol 3-kinases regulators       conditions normal growth. Genes               development encodes receptor isoforms               growth metabolism. Nat. Rev. Genet. 7, <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             Open Biol 2: 110031 <br />       Dev. 19, 1844&#226;&#8364;&#8220;1848. (doi:10.1101/gad.341505)                 different signaling potential. Embo J. 14,                   606&#226;&#8364;&#8220;619. (doi:10.1038/nrg1879) <br /> 17.   Mizushima N et al. 2008 Autophagy fights disease             3373 &#226;&#8364;&#8220;3384.                                            49.   Ciapponi L et al. 2004 Drosophila Mre11/Rad50 <br />       cellular self-digestion. Nature 451,           33.   Brand AH, Perrimon N. 1993 Targeted gene                     complex required prevent telomeric <br />       1069&#226;&#8364;&#8220;1075. (doi:10.1038/nature06639)                         expression means altering cell fates             fusion chromosome breakage. Curr. Biol. 14, <br /> 18.   Levine B, Kroemer G. 2008 Autophagy                   generating dominant phenotypes. Development                  1360&#226;&#8364;&#8220; 1366. (doi:10.1016/j.cub.2004.07.019) <br />       pathogenesis disease. Cell 132, 27&#226;&#8364;&#8220; 42. (doi:10.          118, 401 &#226;&#8364;&#8220;415.                                         50.   Savvidou E et al. 2005 Drosophila CAP-D2 required <br />       1016/j.cell.2007.12.018)                               34.   Hay N, Sonenberg N. 2004 Upstream                        condensin complex stability resolution 19.   Karpen GH, Spradling AC. 1992 Analysis                    downstream mTOR. Genes Dev. 18, 1926&#226;&#8364;&#8220;1945.                sister chromatids. J. Cell. Sci. 118, 2529&#226;&#8364;&#8220;2543. <br />       subtelomeric heterochromatin Drosophila               (doi:10.1101/gad.1212704)                                    (doi:10.1242/jcs.02392) <br />       minichromosome Dp1187 single P element              35.   Butterworth FM, Forrest EC. 1984 Ultrastructure        51.   Vass S et al. 2003 Depletion Drad21/Scc1 Drosophila <br />       insertional mutagenesis. Genetics 132, 737 &#226;&#8364;&#8220;753.             preparative phase cell death                cells leads instability cohesin complex 20.   Klusza S, Deng WM. 2010 poly required nurse-          larval fat body Drosophila melanogaster. Tissue           disruption mitotic progression. Curr. Biol. 13, <br />       cell chromosome dispersal oocyte polarity             Cell.     16,      237&#226;&#8364;&#8220;250.      (doi:10.1016/0040-          208&#226;&#8364;&#8220;218. (doi:10.1016/S0960-9822(03)00047-2) <br />       Drosophila. Fly (Austin) 4, 128&#226;&#8364;&#8220;136.                         8166(84)90047-8)                                       52.   Vereshchagina N, Wilson C. 2006 Cytoplasmic <br /> 21.   Krogan NJ, Greenblatt JF. 2001 Characterization        36.   Butterworth FM, Emerson L, Rasch EM. 1988                    activated protein kinase Akt regulates lipid-droplet <br />       subunit holo-elongator complex required             Maturation degeneration fat body           accumulation Drosophila nurse cells. Development <br />       regulated expression group genes             Drosophila larva pupa revealed                     133, 4731&#226;&#8364;&#8220;4735. (doi:10.1242/dev.02659) <br />       Saccharomyces cerevisiae. Mol. Cell Biol.                 morphometric analysis. Tissue Cell 20, 255&#226;&#8364;&#8220;268.        53.   Singh R et al. 2009 Autophagy regulates lipid <br />       21, 8203&#226;&#8364;&#8220;8212. (doi:10.1128/MCB.21.23.8203-                  (doi:10.1016/0040-8166(88)90047-X)                           metabolism. Nature 458, 1131&#226;&#8364;&#8220;1135. (doi:10. <br />       8212.2001)                                             37.   Juhasz G et al. 2003 Drosophila homolog               1038/nature07976) <br /> 22.   Winkler GS et al. 2001 RNA polymerase II elongator           Aut1 essential autophagy development.       54.   Zhang H et al. 2000 Regulation cellular <br />       holoenzyme composed discrete                       FEBS Lett. 543, 154 &#226;&#8364;&#8220;158. (doi:10.1016/S0014-                growth Drosophila target rapamycin dTOR. <br />       subcomplexes. J. Biol. Chem. 276, 32 743&#226;&#8364;&#8220;                    5793(03)00431-9)                                             Genes Dev. 14, 2712&#226;&#8364;&#8220;2724. (doi:10.1101/gad. <br />       32 749. (doi:10.1074/jbc.M105303200)                   38.   Rusten TE et al. 2004 Programmed autophagy                835000) <br /> 23.   Hawkes NA et al. 2002 Purification                       Drosophila fat body induced ecdysone         55.   Braun  Hoffmann JA, Meister M. 1998 Analysis       characterization human elongator complex.             regulation PI3K pathway. Dev. Cell            Drosophila host defense domino mutant <br />       J. Biol. Chem. 277, 3047&#226;&#8364;&#8220; 3052. (doi:10.1074/jbc.            7, 179&#226;&#8364;&#8220; 192. (doi:10.1016/j.devcel.2004.07.005)              larvae, devoid hemocytes. Proc. Natl <br />       M110445200)                                            39.   Scott RC, Juhasz G, Neufeld TP. 2007 Direct                  Acad. Sci. USA. 95, 14 337 &#226;&#8364;&#8220;14 342. (doi:10.1073/ <br /> 24.   Nelissen H et al. 2005 elongata mutants                  induction autophagy Atg1 inhibits cell growth          pnas.95.24.14337) <br />       identify functional Elongator complex plants            induces apoptotic cell death. Curr. Biol. 17,      56.   Dearolf CR. 1998 Fruit fly &#226;&#8364;&#732;leukemia&#226;&#8364;&#8482;. Biochim. <br />       role cell proliferation organ               1 &#226;&#8364;&#8220;11. (doi:10.1016/j.cub.2006.10.053)                       Biophys. Acta 1377, M13&#226;&#8364;&#8220;M23. <br />       growth. Proc. Natl Acad. Sci. USA. 102, 7754 &#226;&#8364;&#8220;         40.   Juhasz G et al. 2007 Gene expression profiling         57.   Minakhina S, Steward R. 2006 Melanotic mutants       7759. (doi:10.1073/pnas.0502600102)                          identifies FKBP39 inhibitor autophagy            Drosophila: pathways phenotypes. Genetics <br /> 25.   Winkler GS et al. 2002 Elongator histone H3             larval Drosophila fat body. Cell Death Differ. 14,           174, 253&#226;&#8364;&#8220;263. (doi:10.1534/genetics.106.061978) <br />       H4 acetyltransferase important normal                1181 &#226;&#8364;&#8220;1190. (doi:10.1038/sj.cdd.4402123)               58.   Nguyen L et al. 2010 Elongator emerging role <br />       histone acetylation levels vivo. Proc. Natl Acad.   41.   Berry DL, Baehrecke EH. 2007 Growth arrest               neurological disorders. Trends Mol. Med. 16, <br />       Sci. USA 99, 3517&#226;&#8364;&#8220; 3522. (doi:10.1073/pnas.                  autophagy required salivary gland cell               1&#226;&#8364;&#8220;6. (doi:10.1016/j.molmed.2009.11.002) <br />       022042899)                                                   degradation Drosophila. Cell 131, 1137&#226;&#8364;&#8220;1148.        59.   Prisco M et al. 2002 Nuclear translocation insulin <br /> 26.   Svejstrup JQ. 2007 Elongator complex:               (doi:10.1016/j.cell.2007.10.048)                             receptor substrate-1 simian virus 40 T <br />       roles does play? Curr. Opin. Cell. Biol. 19,        42.   Baker KD, Thummel CS. 2007 Diabetic larvae               antigen activated type 1 insulin-like <br />       331&#226;&#8364;&#8220;336. (doi:10.1016/j.ceb.2007.04.005)                     obese flies-emerging studies metabolism                growth factor receptor. J. Biol. Chem. 277, <br /> 27.   Versees W, Groeve S, Van Lijsebettens M. 2010             Drosophila. Cell Metab. 6, 257 &#226;&#8364;&#8220;266. (doi:10.1016/           32 078 &#226;&#8364;&#8220;32 085. (doi:10.1074/jbc.M204658200) <br />       Elongator, conserved multitasking complex? Mol.            j.cmet.2007.09.002)                                    60.   Sun H et al. 2003 Insulin-like growth factor       Microbiol. 76, 1065&#226;&#8364;&#8220;1069. (doi:10.1111/j.1365-         43.   Saltiel AR, Kahn CR. 2001 Insulin signalling         receptor signaling nuclear translocation       2958.2010.07162.x)                                           regulation glucose lipid metabolism. Nature           insulin receptor substrates 1 2. Mol. <br /> 28.   Walker J et al. 2011 Role Elongator Subunit Elp3          414, 799 &#226;&#8364;&#8220;806. (doi:10.1038/414799a)                         Endocrinol. 17, 472 &#226;&#8364;&#8220;486. (doi:10.1210  <br />       Drosophila melanogaster larval development      44.   Gronke S et al. 2007 Dual lipolytic control body          2002-0276) <br />       immunity. Genetics 187, 1067 &#226;&#8364;&#8220;1075. (doi:10.1534/            fat storage mobilization Drosophila. PLoS       61.   Wu  Chen J, Baserga R. 2008 Nuclear insulin <br />       genetics.110.123893)                                         Biol. 5, e137. (doi:10.1371/journal.pbio.0050137)            receptor substrate-1 activates promoters cell cycle <br /> 29.   Ponting CP. 2002 Novel domains orthologues      45.   Teixeira L et al. 2003 Drosophila Perilipin/ADRP             progression genes. Oncogene 27, 397&#226;&#8364;&#8220; 403. (doi:10. <br />       eukaryotic transcription elongation factors. Nucleic         homologue Lsd2 regulates lipid metabolism. Mech.             1038/sj.onc.1210636) <br />  62. Altschul SF et al. 1997 Gapped BLAST PSI-           67. Ronquist F, Huelsenbeck JP. 2003 MrBayes 3:                  gels. Anal. Chem. 68, 850&#226;&#8364;&#8220;858. (doi:10.1021/          16 <br />     BLAST: new generation protein database search          Bayesian phylogenetic inference mixed                  ac950914h) <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    rsob.royalsocietypublishing.org <br />     programs. Nucleic Acids Res. 25, 3389&#226;&#8364;&#8220;3402.                 models. Bioinformatics 19, 1572&#226;&#8364;&#8220; 1574. (doi:10.        72.   Yates III JR et al. 1995 Method correlate <br />     (doi:10.1093/nar/25.17.3389)                                1093/bioinformatics/btg180)                                  tandem mass spectra modified peptides 63. ding J, Biegert  Lupas  2005 HHpred         68. Felsenstein J. 2005 PHYLIP (Phylogeny Inference              amino acid sequences protein database. <br />     interactive server protein homology detection           Package) version 3.6. Distributed author.             Anal. Ch <br /> </body></html>