<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Single methyl group determines prion propagation protein degradation activities yeast heat shock <br /> protein (Hsp)-70 chaperones Ssa1p Ssa2p <br />  <br /> Organisms encode multiple homologous heat shock protein (Hsp)-                  ogous Hsp70 protects animals various stresses (5, 6). <br /> 70s, essential protein chaperones play major                  highly homologous isoforms display clearly <br /> role cellular protein &#226;&#8364;&#339;quality control.&#226;&#8364;&#65533; Hsp70s func-           distinct functions (4, 7&#226;&#8364;&#8220;11). ampli&#239;&#172;&#65533;cation Hsp70 <br /> tionally redundant highly homologous, possess distinct                 genes long considered advantageous allowing reg- <br /> functions. regulatory motif underlying distinctions,                 ulation spatiotemporal abundance Hsp70 according  unknown. 98% identical cytoplasmic Hsp70s Ssa1p                cellular need chaperone function, ability Hsp70 Ssa2p function differently regard propagation yeast                  function distinctly implies expansion Hsp70 genes [URE3] prions vacuolar-mediated degradation gluco-                provides advantage source functional diversity. Subtle <br /> neogenesis enzymes, FBPase.                       differences intrinsic cochaperone-regulated activities likely <br /> Hsp70 <span id='am-23' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-24' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> binding domain (NBD) regulates functional                allow certain Hsp70 isoforms perform better speci&#239;&#172;&#65533;cities. &#239;&#172;&#65533;nd little difference ATPase, protein refolding,            speci&#239;&#172;&#65533;c tasks. Despite intensive study,  presence amyloid inhibiting activities puri&#239;&#172;&#65533;ed Ssa1p Ssa2p,               regulatory element Hsp70 confer speciali- interchanging NBD residue alanine 83 (Ssa1p) <span id='am-38' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-39' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-44' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="glycine" datatype="xsd:string"></span>gly</span>-                 zation described. <br /> cine 83 (Ssa2p) switched functions Ssa1p Ssa2p [URE3]                    S. cerevisiae Ssa subfamily cytosolic Hsp70 composed <br /> propagation FBPase degradation. Disrupting degradation                  constitutively expressed Ssa1p Ssa2p stress- <br /> pathway did affect prion propagation,  indicating                   inducible Ssa3p Ssa4p (12). <span id='am-31' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-32' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>Amino acid</span> identity distinct processes Ssa1/2p chaperones func-                 family highest isoforms expressed similar  <br /> tion differently. results suggest primary evolutionary             ditions. Constitutively expressed Ssa1p Ssa2p 98% iden- <br /> pressure Hsp70 functional distinctions specify inter-             tical, inducible Ssa3p Ssa4p 88% identical, Ssa3/4p <br /> actions Hsp70 substrate, specify regulation               80% identical Ssa1/2p. Ssa family Hsp70 function es- activity. data suggest rationale maintaining multiple            sential viability, isoforms support <br /> Hsp70s suggest subtle differences Hsp70s evolved                 cell growth expressed abundantly (13, 14). provide functional speci&#239;&#172;&#65533;city affecting overall enzy-                   Despite near identity, Ssa1p Ssa2p clear dif- <br /> matic activity.                                                                 ferences physiologic activity. &#239;&#172;&#65533;rst shown differ                                                                                 vacuolar import degradation (Vid) pathway,                                                                                 speci&#239;&#172;&#65533;cally requires Ssa2p degradation gluconeogenic <br /> H    eat shock protein (Hsp)-70 molecular chaperones      highly conserved proteins. act <span id='am-12' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span>ATP</span>-regulated cycle binding release hydrophobic <br />                                                                                 enzymes addition glucose starved cells (15). Ssa Hsp70 <br />                                                                                 isoforms function distinctly regard yeast prions. <br /> surfaces incompletely folded proteins. interaction helps                Prions infectious amyloid-like aggregates misfolded cel- proteins adopt maintain native conformation                       lular proteins propagate inducing similar misfolding allows Hsp70 play central role essential processes                 native form protein. Yeast [URE3] prions, propa- <br /> including protein traf&#239;&#172;&#65533;cking, signaling, degradation (1&#226;&#8364;&#8220;3).                 gate amyloid forms Ure2p, unaffected depletion versatile roles, Hsp70 promising therapeutic target             Ssa1p lost frequently cells lacking Ssa2p (16, 17). human diseases cancer amyloid-based                       increasing abundance Ssa1p wild-type cells causes <br /> neurodegenerative diseases diabetes. surprisingly,                lose [URE3], increasing Ssa2p does (18). <br />  <br />  <br />  <br />                                                                                                                                                                            GENETICS broad understanding molecular basis                         used distinction prion phenotype identify <br /> Hsp70 action.                                                                   structural elements Hsp70 regulate functional dif- <br />    Hsp70 activity regulated communication                       ferences Ssa1p Ssa2p. determining amino-terminal ATPase domain (nucleotide binding domain,                        ATPase domain responsible, single amino acid NBD), adjacent substrate binding domain (SBD),                    domain necessary functional distinction <br /> conserved carboxyl-terminal region (CTD), forms lid                     Ssa1p Ssa2p regard propagation [URE3]. binding pocket SBD interacts variety                 Remarkably, residue speci&#239;&#172;&#65533;ed Ssa2p function pro- cochaperones. Binding ATP NBD opens lid                    tein degradation mediated Vid pathway. pathway allows rapid reversible binding substrate. ATP hydrolysis                    important [URE3] propagation,  indicating induces conformational changes close lid, trapping                     minimal change Hsp70 specify functional differences <br /> substrate binding pocket. Release <span id='am-1' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span>ADP</span> res-                    different processes. <br /> toration ATP-bound &#226;&#8364;&#339;open&#226;&#8364;&#65533; state promotes substrate  <br /> lease. Substrate binds high af&#239;&#172;&#65533;nity ADP bound state, ATP binding decreases af&#239;&#172;&#65533;nity approximately                     Author contributions: D.S. D.C.M. designed research, performed research, contrib- orders magnitude.                                                        uted new reagents/analytic tools, analyzed data, wrote paper. <br />    organisms encode multiple Hsp70 isoforms. Humans                        authors declare con&#239;&#172;&#8218;ict  Saccharomyces cerevisiae canonical cytosolic                   Direct Submission article prearranged editor. <br /> Hsp70s. functions overlap extent human Hsp70                  1 <br />                                                                                 correspondence addressed. E-mail: masisond@helix.nih.gov. <br /> (Hsc70) support viability yeast lacking essential Ssa                article contains supporting information online www.pnas.org/lookup/suppl/doi:10. <br /> Hsp70 subfamily (4). Similarly, exogenously expressed heterol-                  1073/pnas.1107421108/-/DCSupplemental. <br />  <br />  <br />  <br /> www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                    PNAS | August 16, 2011 | vol. 108 |  33 | 13665&#226;&#8364;&#8220;13670 <br />  Results                                                                          denaturing 1% activity treatment guani- <br /> Functional Difference Ssa1p Ssa2p Apparent                 dinium chloride. Adding Ydj1p restored 10% original Biochemical Activities. Previously, showed              luciferase activity (Fig. 1C). Coincubating denatured luciferase <br /> [URE3] propagation inhibited increasing Ssa1p                        Ydj1p 10-fold molar excess Ssa1p Ssa2p, <br /> pleting Ssa2p, increasing Ssa2p depleting Ssa1p                     resulted nearly complete restoration luciferase <br /> (16&#226;&#8364;&#8220;19). assessed functions puri&#239;&#172;&#65533;ed Ssa1p Ssa2p                        activity. rates refolding assisted Ssa1p Ssa2p vitro determine differences                  nearly identical.  Hsp70s did differ differences biochemical activities. kinetics binding             functional cooperation Ydj1p refold denatured protein. release substrates Hsp70 depend ATP cycle.                       puri&#239;&#172;&#65533;ed, Ure2p spontaneously assembles amyloid <br />                                                                                  &#239;&#172;&#65533;laments, assembly inhibited addition puri&#239;&#172;&#65533;ed <br /> Intrinsic ATPase activity various Hsp70s low, ranging <br />                                                                                  Ssa1p (23, 24). hands assembly puri&#239;&#172;&#65533;ed Ure2p 0.01 1 ATP hydrolyzed minute (20).                                                                                  amyloid showed sigmoidal curve short lag phase ATPase activity Ssa1p Ssa2p 0.02/ <br />                                                                                  5 min (Fig. 1D). Preincubation Ure2p (18 &#206;&#188;M) min, agrees previous results (Fig. 1A) (21, 22).                      Ssa1p Ssa2p (6 &#206;&#188;M) increased lag time Ure2p <br />    Low intrinsic ATPase activity provides key step regula-                 amyloid formation decreased yield amyloid formed tion &#239;&#172;&#65533;ne-tuning Hsp70 function cochaperones. Ydj1p,                    degree. data suggest differences major yeast cytosolic Hsp40, extensively studied                       way Ssa1p Ssa2p in&#239;&#172;&#8218;uence [URE3] propagation cochaperone stimulates Ssa1p ATPase. Ydj1p stimulated                       differences way in&#239;&#172;&#8218;uence Ure2p amyloid forma- <br /> ATPase activity Ssa1p Ssa2p roughly 10-fold 7-fold,                   tion. data did uncover differences respectively, agrees previous data (Fig. 1A) (21,                enzymatic activities Ssa1p Ssa2p substantial <br /> 22). tested small difference                     explain functional distinction vivo. difference interaction Ydj1p Ssa1p Ssa2p using surface plasmon resonance. Ydj1p consistently interacted                   [URE3] Unstable Cells Expressing Ssa1p. [URE3] prop- Ssa2p slightly better Ssa1p (Fig. 1B), suggesting                 agates insoluble amyloid aggregates Ure2p,  lower stimulation Ssa2p ATPase Ydj1p                      pressor nitrogen catabolic genes allantoate reduced physical interaction Ydj1p Ssa2p.                          transporter Dal5p. [URE3] present, depletion Ure2p <br />    compared ability Ssa1p Ssa2p refold mis-                     prion aggregates relieves repression. strains folded protein measuring reactivation &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase               ADE2 controlled DAL5 promoter, expression Ade2p <br />                                                                                  used monitor presence [URE3] (25, 26).                                                                                  [URE3] present strains, express Ade2p                                                                                  grow <span id='am-7' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Adenine" datatype="xsd:string"></span>adenine</span> white. [ure-o]  e., <br />                                                                                  lack [URE3]) require adenine grow red media <br />                                                                                  containing limiting adenine (Materials Methods). <br />                                                                                     Amino acid identity Ssa1p Ssa2p NBD, SBD,                                                                                  CTD &#226;&#710;&#188;99, 99, 93% (Fig. 2A). Hsp70 amino acids  <br />                                                                                  served species essential Hsp70 function                                                                                  known interact cochaperones substrates present                                                                                  Ssa1p Ssa2p. example, contain R169, <br />                                                                                  crucial interaction Hsp40 cochaperones, linker <br />                                                                                  residues 382&#226;&#8364;&#8220;392 NBD SBD, required                                                                                  interdomain communication, residues I400, A403, L410, F425, <br />                                                                                  A428, V435, I437, form hydrophobic substrate- <br />                                                                                  interacting core, C-terminal GPT V)EEVD motif                                                                                  eukaryotic Hsp70s, interacts tetratricopeptide repeat <br />                                                                                  (TPR) cochaperones. dif&#239;&#172;&#65533;cult predict priori <br />                                                                                  residues critical Hsp70 functional differences. <br />                                                                                     Growth strain 1161, lacks chromosomal SSA <br />                                                                                  family genes (SSA1&#226;&#8364;&#8220;SSA4), supported equally Ssa1p                                                                                  Ssa2p expressed plasmid (17), indicating essential <br />                                                                                  Hsp70 functions Ssa1p Ssa2p completely  <br />                                                                                  terchangeable.  [URE3] propagates stably                                                                                  cells expressing Ssa2p, unstable cells expressing Fig. 1. Ssa1p Ssa2p comparable enzymatic activities.   ATPase <br />                                                                                  Ssa1p (17). instability seen appearance red [ure- <br /> activity Ssa1p Ssa2p (0.4 &#206;&#188;M) absence presence Ydj1p (1.6 <br /> &#206;&#188;M). ATP 50 &#206;&#188;M. Values averages &#194;&#177; SD replicate <br />                                                                                  o] colonies arising population cells spread plate <br /> experiments. (B) Representative surface plasmon resonance experiment          limiting adenine (Fig. 2B, compare 111 222). Pheno- <br /> physical interaction Ssa1p Ssa2p (concentration indicated Right)        typic differences cells expressing Ssa1p immobilized Ydj1p. assay, background resonance control        Ssa2p explained differences abundance Ssa <br /> cells lacking Ydj1p subtracted. Highly reproducible results             protein major Hsp70 cochaperones known in&#239;&#172;&#8218;uence <br /> obtained additional replicate experiments separate machines         [URE3] propagation (Ydj1p, Sse1p, Sis1p; Fig. S1A). ing proteins different preparations &#239;&#172;&#65533;ve concentrations Ssa       used readily distinguishable prion phenotypes determine <br /> protein. (C) Denatured &#239;&#172;&#65533;re&#239;&#172;&#8218;y luciferase (0.5 &#206;&#188;M) incubated Ydj1p        molecular basis functional differences Ssa1p <br /> (0.2 &#206;&#188;M) presence absence Ssa1p Ssa2p (2 &#206;&#188;M) indicated.                                                                                  Ssa2p. <br /> reactions contained 20 &#206;&#188;M ATP. Values averages &#194;&#177; SD replicate <br /> experiments. (D) Puri&#239;&#172;&#65533;ed Ure2p dialyzed buffer containing 1 M <br />                                                                                  ATPase Domain Determines Prion Stability. &#239;&#172;&#65533;rst examined GdmCl buffer GdmCl centrifuged 12,000 g 1 h <br /> immediately adding Ure2p (18 &#206;&#188;M) cuvettetes            regulatory effect Ssa functional region swapping Hsp70 (6 &#206;&#188;M). Reactions stirred 25&#194;&#176; 10-&#206;&#188;L aliquots removed      major Hsp70 structural domains. constructed Ssa1p&#226;&#8364;&#8220; added 490 &#206;&#188;L 100 &#206;&#188;M Thio&#239;&#172;&#8218;avin-T. Fluorescence monitored         Ssa2p hybrid proteins representing possible combina- <br /> 485 nm excitation 450 nm.                                                tions domains described  Names wild-type <br />  <br /> 13666 | www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                                               Sharma Masison <br />                                                                                   determine minimal difference conferred functional <br />                                                                                  distinction, began switching individual residues Ssa2p                                                                                  amino acid Ssa1p. Switching residues 20, 41, 62 <br />                                                                                  did in&#239;&#172;&#8218;uence [URE3] stability (Fig. 2C). Remarkably,  <br />                                                                                   cells expressing Ssa2p G83A substitution <br />                                                                                  (Ssa2G83A) displayed unstable Ssa1p-like [URE3] phenotype <br />                                                                                  (Fig. 2 C E).  Ssa2G83A behaves like Ssa1p                                                                                  identical Ssa2p having methyl group place                                                                                  hydrogen chain residue 83. <br />                                                                                      switched <span id='am-50' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-51' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-53' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> 83 bulkier residues leucine                                                                                  phenylalanine con&#239;&#172;&#65533;rm crucial role <span id='am-62' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> lo- <br />                                                                                  cation. alterations similarly reduced [URE3] stability like <br />                                                                                  G83A (Fig. 2C). determine glycine residue 83                                                                                  promote stable [URE3] propagation,  <br />                                                                                  ciprocal A83G substitution Ssa1p (Ssa1A83G). Ssa1A83G <br />                                                                                  identical Ssa1p residue 83, cells expressing                                                                                  source Ssa protein stable Ssa2p-like [URE3] pheno- <br />                                                                                  type. data demonstrate glycine residue 83 determines                                                                                  Ssa2p-speci&#239;&#172;&#65533;c function important [URE3] prion propagation. <br />                                                                                      Structural information Ssa Hsp70 known. Hsp70 <br />                                                                                  chaperones highly conserved,  structural  <br />                                                                                  formation available Hsp70 proteins species. <br />                                                                                  Using Web-based program (http://swissmodel.expasy.org/),                                                                                  modeled Ssa1p structure. Residue 83 lies short &#206;&#177;-helix                                                                                  ATPase domain (residues 80&#226;&#8364;&#8220;88), conserved                                                                                  Hsp70s structure known (Fig. S2).                                                                                  Hsp70s glycine homologous position. gly- <br />                                                                                  cine unfavorable helix stability, Ssa2p al- <br />                                                                                  ternative structure presence G83. tested <br /> Fig. 2. ATPase domain determines speci&#239;&#172;&#65533;city Hsp70 function.   Linear        making substitutions Q82G P, M85G P diagram Ssa1/2p structural domains indicated locations dif-       Ssa1p, expected similarly disrupt putative <br /> ferences amino acid residues Ssa1p Ssa2p highlighted red       helix, restore [URE3] stability like A83G. tested <br /> bars. (B&#226;&#8364;&#8220;D) Cells taken white colonies grown nonselectively 1/2 YPD      deletion residue 83 Ssa1p Ssa2p (A83&#206;&#8221; diluted water, spread 1/2 YPD plates grown 3 d. (B) <br />                                                                                  G83&#206;&#8221;, respectively) determine importance residue <br /> Leftmost panels wild-type strain 1075 [URE3] [ure-o] colonies (WT+ WT&#226;&#8364;&#8220;, respectively), display characteristic white red color. <br />                                                                                  directly. changes stabilized [URE3] like A83G, <br /> Degree mitotic instability [URE3] strain 1161 expressing Ssa1p (111),   reduced [URE3] stability G83A <br /> Ssa2p (222), Ssa1/2p hybrid proteins  indicated) proportional    (Fig. 2D).  prion phenotypes cells expressing <br /> frequency pigmented colonies. (C D) [URE3] phenotype cells             Ssa1p Ssa2p lacking residue 83 indistinguishable,  <br /> expressing Ssa1p (A1) Ssa2p (A2) indicated single amino acid        dicating nonconserved residues contribute <br /> changes. (E) Quanti&#239;&#172;&#65533;ed frequencies loss [URE3]. Cells taken white     functional speci&#239;&#172;&#65533;city Ssa1p Ssa2p regarding [URE3] <br /> colonies expressing indicated Hsp70 grown 1/2 YPD, diluted        propagation.  G83 provide unique se- <br /> water, spread 1/2 YPD plates. percentage (&#194;&#177;SD) red [ure-o]       quence element important Ssa2p-speci&#239;&#172;&#65533;c Hsp70 activity. <br /> colonies measured total 900&#226;&#8364;&#8220;1,500 colonies shown. <br />                                                                                      Overexpressing Ssa1p, Ssa2p, cures cells [URE3] <br />                                                                                  (18). tested interchanging residue 83                                                                                  switch speci&#239;&#172;&#65533;city did (Fig. S3). result hybrid proteins re&#239;&#172;&#8218;ect domain composition. ex- <br />                                                                                  shows Hsp70 activities important prion propagation <br /> ample, 111 Ssa1p 122 NBD Ssa1p SBD <br />                                                                                  differ important prion elimination. CTD Ssa2p. hybrids  point mutants,  <br />                                                                                      tested minor G/A83 difference dis- <br />  <br />  <br />  <br />                                                                                                                                                            GENETICS <br /> used expressed similar level (Fig. S1 B&#226;&#8364;&#8220;D)                    tinguishes Ssa1p Ssa2p cellular process known <br /> supported growth cells like wild-type Ssa1p Ssa2p (gen-                   depend strictly Ssa2p. Starving cells glucose induces <br /> eration times 104 &#194;&#177; 5 (SD) minutes YPAD liquid medium                      expression gluconeogenic enzymes, fructose-1,6- 30 &#194;&#176;C), indicating function like wild-type proteins             bisphosphatase (FBPase), restoring glucose 3 d causes regard essential Ssa functions.                                          degradation enzymes vacuolar import    Half hybrid proteins (122, 121, 112) produced                   degradation (Vid) pathway (15). FBPase degraded similarly <br /> Ssa1p-like unstable [URE3] phenotype (Fig. 2B). contrast,                     wild-type cells cells expressing Ssa2p, cells expressing remaining hybrids (211, 212, 221)                       stabilized cells expressing Ssa1p (Fig. 3A). results <br /> propagated [URE3] stably, like expressing Ssa2p. Quan-                     con&#239;&#172;&#65533;rmed speci&#239;&#172;&#65533;c requirement Ssa2p Vid pathway <br /> ti&#239;&#172;&#65533;ed frequencies [URE3] loss shown Fig. 2E.                       function. Amazingly, FBPase stabilized cells expressing <br /> spicuously, hybrids impaired [URE3] propagation                     Ssa2G83A degraded cells expressing Ssa1A83G. NBD Ssa1p supported stable [URE3]                       minimal difference structure switched propagation NBD Ssa2p. results                    roles Ssa1p Ssa2p Vid pathway function. <br /> ATPase domains Ssa1p Ssa2p confer                          &#239;&#172;&#65533;ndings suggested link [URE3] <br /> speci&#239;&#172;&#65533;c Hsp70 isoform character regarding prion stability.                       propagation Vid pathway function, tested  <br />                                                                                  leting SEC28 reduced [URE3] stability. Sec28p coatomer <br /> Glycine Residue 83 Determines Ssa2p-Speci&#239;&#172;&#65533;c Hsp70 Function.                  complex subunit regulates Golgi ER traf&#239;&#172;&#65533;cking NBDs Ssa1p Ssa2p differ residues: 20  S),                  facilitates import Vid cargo Vid vesicles (27). Deleting <br /> 41  G), 62 (S , 83  G) (Ssa1p residue listed &#239;&#172;&#65533;rst),                SEC28 stabilized FBPase (Fig. 3B), indicating disrupted relatively minor difference structure (Fig. 2A).                 Vid pathway, did affect [URE3] stability (Fig. 3C). <br />  <br /> Sharma Masison                                                                                  PNAS | August 16, 2011 | vol. 108 |  33 | 13667 <br />                                                                                    function Ssa1p Ssa2p cytoplasmic Hsp70s  <br />                                                                                   dependent cellular processes. Additionally, deleting residue <br />                                                                                   Ssa1p Ssa2p generated proteins functionally <br />                                                                                   equivalent, behaved differently Ssa1p Ssa2p. <br />                                                                                   Hsp70s homology overlap func- <br />                                                                                   tionally, bit surprising nearly identical proteins <br />                                                                                   act differently astonishing small change <br />                                                                                   responsible distinction function. Despite known <br />                                                                                   regulatory activity NBD, &#239;&#172;&#65533;nding unexpected  <br />                                                                                   cause NBDs Ssa1p Ssa2p differ mini- <br />                                                                                   mally different spatially separated residues. anticipated <br />                                                                                   C-terminal region likely provide speci&#239;&#172;&#65533;city <br />                                                                                   divergent region Hsp70s, including <br />                                                                                   Ssa1p Ssa2p, possesses elements interact                                                                                   variety cochaperones regulate Hsp70 function.                                                                                   &#239;&#172;&#65533;nding residue 83 does in&#239;&#172;&#8218;uence speci&#239;&#172;&#65533;city overex- <br /> Fig. 3. Residue 83 speci&#239;&#172;&#65533;es Hsp70 function Vid pathway.   Cultures       pressed Ssa1p Ssa2p curing cells [URE3] indicates strain 1161 [ure-o] cells starved glucose 3 d split                                                                                   amino acid differences  fact, contribute addi- <br /> half. Glucose added half indicated 2 h abundance FBPase monitored Western analysis. Numbers 1 2 <br />                                                                                   tional regulatory speci&#239;&#172;&#65533;city. <br /> designate Ssa1p Ssa2p, respectively; WT wild type (strain 1075). Similar      major human Hsp70s HspA1A HspA1B differ results obtained [URE3] cells. (B) experiment repeated          amino acids, resides NBD, using isogenic sec28&#206;&#8221; strain. (C) [URE3] phenotypes wild-type           proteins believed fully interchangeable (10). &#239;&#172;&#65533;nd- <br /> sec28&#206;&#8221; cells grown 1/2 YPD plates 3 d 30 &#194;&#176;C.                            ings raise alternative possibility differences  <br />                                                                                   termine speci&#239;&#172;&#65533;c functions unidenti&#239;&#172;&#65533;ed tasks.                                                                                   raise concerns commonly performed experiments using  stable [URE3] propagation did require functional                      exogenously expressed Hsp70s. mentioned  98% <br /> Vid pathway, somewhat unexpected sup-                        identical Ssa1p Ssa2p differ ability cure yeast posed minimal alanine/glycine difference unique                  prions overexpressed, similar differences effects  limited effect Hsp70 activity.  residue 83                       ing different Hsp70s expected model systems. <br /> appears differentiated specify functional distinction                   strong allosteric coordination structural important de&#239;&#172;&#65533;ned unrelated Hsp70 tasks cell.               domains Hsp70. substrate binds SBD,                                                                                   kinetics binding release substrate regulated pri- <br /> Discussion                                                                        marily ATP hydrolysis nucleotide exchange identify minimal amino acid change basis                        NBD. ruled possibility intrinsic <br /> functional distinction highly homologous Hsp70 family                     Hsp70 activity altered, large cohort Hsp70 <br /> chaperones, phenomenon conserved evo-                        cochaperones interact NBD regulate pro- <br /> lution. adaptability ATPase domain regulate                        cesses. residue 83 known interact directly speci&#239;&#172;&#65533;city Hsp70 function different cellular processes                      cochaperones, subtle Gly/Ala difference in&#239;&#172;&#8218;uence widens versatility mere structural domain                    ef&#239;&#172;&#65533;ciency regulate ATP reaction cycle. Resi- <br /> zymatically in&#239;&#172;&#8218;uences substrate binding. &#239;&#172;&#65533;ndings demon-                       83 lies NBD lobe IB, near nucleotide-binding <br /> strate exquisitely de&#239;&#172;&#65533;ned variations Hsp70 structure                  cleft (Fig. S2). structural implication Gly/Ala difference <br /> strongly in&#239;&#172;&#8218;uence actions Hsp70 speci&#239;&#172;&#65533;c complex                     affect movement lobe interaction <br /> physiological processes. success-                    nucleotide exchange factors (NEFs) (32), possibly altering <br /> fully switch functional speci&#239;&#172;&#65533;city Ssa1p Ssa2p                     way different NEFs able induce structural transition <br /> coupling protein engineering use yeast prions monitor                  facilitates release phosphate ADP. <br /> Hsp70 activity, shows potential designing Hsp70                         Gly/Ala difference in&#239;&#172;&#8218;uence ef&#239;&#172;&#65533;ciency perform desired ways.                                                          Hsp70 interaction major chaperone machinery components <br />    major Hsp70 activities cell depend                      Hsp104 Hsp90 nonchaperone factors  <br /> ATP-regulated cycle binding release exposed hydro-                      volved speci&#239;&#172;&#65533;c processes require particular Hsp70 phobic surfaces partially folded proteins. necessity                    job. Alternatively, modest change kinetics possessing basic structural information carry                speci&#239;&#172;&#65533;c steps overall rate Hsp70 reaction cycle <br /> important reaction evident conservation sec-                    little affect general functions Hsp70 ondary tertiary structures widely diverged Hsp70s.                    importance certain tasks. <br /> Gross structural information obtained recombining domains                         addition de&#239;&#172;&#65533;ned functions essential housekeeping Hsp70 shown SBD in&#239;&#172;&#8218;uence                          processes, Hsp70 protects large number proteins trinsic activity adjacent NBD (28), SBD, broad                prone aggregation various kinds stresses. substrate speci&#239;&#172;&#65533;city, interchanged destroying                      evolution clear advantage possess <br /> functions associated divergent Hsp70s (4, 7). NBD,                       different Hsp70s allow interaction protein se-  regulates substrate binding SBD,                    quence elements maintaining speci&#239;&#172;&#65533;city performing important determining speci&#239;&#172;&#65533;c activities different                    certain tasks. &#239;&#172;&#65533;ndings suggest NBD evo- <br /> Hsp70s vivo. example, majority Hsp70 alterations                 lutionarily optimized conferring functional speci&#239;&#172;&#65533;city. affect yeast prion propagation NBD, capacity                   likely SBD optimized interacting Hsp70 support yeast viability governed primarily                  proteins collectively high diversity amino acid <br /> NBD (4, 29&#226;&#8364;&#8220;31). observations suggest regulation                    sequence. unique combination functional speci&#239;&#172;&#65533;city Hsp70 activity, substrate binding se, critical              NBD broader range bind various substrates parameter determining speci&#239;&#172;&#65533;city function.                                 SBD allows Hsp70 perform wide variety cellular tasks    line conclusion, minimal Gly/                  known cellular protein. genomic ampli&#239;&#172;&#65533;cation <br /> Ala difference NBD interconvert speci&#239;&#172;&#65533;city                 differentiation Hsp70 genes magni&#239;&#172;&#65533;es diversi&#239;&#172;&#65533;es <br />  <br /> 13668 | www.pnas.org/cgi/doi/10.1073/pnas.1107421108                                                                                Sharma Masison <br />  bene&#239;&#172;&#65533;ts. observation minimal tweaking                                 Plasmids expressing Ssa2p (pPROEX-HTb-SSA2) Ydj1p (pPROEX-HTb- <br /> structure determines major distinctions function points                               YDJ1) obtained replacing SSA1 plasmid pPROEX-HTb-SSA1 using <br /> Hsp70 powerfully malleable object selection.                                        standard molecular biology techniques. Ure2p expression vector (pKT41- <br />                                                                                             Ure2) gift Dr. Reed Wickner (National Institutes Health, <br /> Materials Methods                                                                       Bethesda, MD) (23). vector encodes Ure2p N-terminal hexa Yeast Strains Plasmids. Strain 1075 MATa, kar1-1, SUQ5, PDAL5::ADE2,                 tag. proteins puri&#239;&#172;&#65533;ed using Talon metal af&#239;&#172;&#65533;nity resin. his3&#206;&#8221;202, leu2&#206;&#8221;1, trp1&#206;&#8221;63, ura3-52 (17). Strain 1161 addition ssa1::                 Ure2p, polyhistidine tag removed assaying proteins KanMX, ssa2::HIS3, ssa3::TRP1, ssa4::ura3-2f carries plasmid pJ401                  activity. . Strain 1366 1075 sec28::KanMX. Strain 628&#226;&#8364;&#8220;4B (18), used monitoring curing [URE3], MAT&#206;&#177;, kar1-1, SUQ5, ade2-1, his3&#206;&#8221;202, ura2.                 ATPase Assay. ATP hydrolysis measured essentially described (22). <br /> Plasmid pJ401 URA3-based plasmid SSA2. Plasmid pC210 single-                 Brie&#239;&#172;&#8218;y, ATPase activity 0.4 &#206;&#188;M Ssa1p Ssa2p 1.6 &#206;&#188;M Ydj1 <br /> copy LEU2-based plasmid containing SSA1 NdeI site initiator                  measured 200 &#206;&#188;L assay buffer (25 mM Tris-Cl pH 7.4, 100 mM KCl, 5 <br /> ATG codon, control SSA2 promoter (18). Plasmid p111 pC210                   mM MgCl2, 1 mM DTT) 25 &#194;&#176;C. Reactions incubated 10 min XbaI XhoI sites created altering amino acid coding                      assay started addition [&#206;&#179;-32P] ATP &#239;&#172;&#65533;nal concentration 50 &#206;&#188;M. <br /> formation SSA1 codons 391 540, respectively (4). Plasmid p222 p111                Aliquots removed 10, 20, 30 min ATP hy- coding region SSA1 replaced SSA2. Plasmids                                                                                             drolysis determined described. ATP turnover rates linear SSA1/SSA2 hybrid genes constructed replacing NdeI&#226;&#8364;&#8220;XbaI (NBD), <br />                                                                                             course assay highly reproducible multiple separate <br /> XbaI&#226;&#8364;&#8220;XhoI (SBD), XhoI&#226;&#8364;&#8220;SphI (CTD) fragments p111 PCR-ampli&#239;&#172;&#65533;ed <br />                                                                                             protein preparations. <br /> fragments SSA2. Substitutions single amino acids using Stratagene Quick-Change kit. HSP70 alleles veri&#239;&#172;&#65533;ed DNA se- <br /> quencing. Plasmids pA1-83G pA2-83A pC210 encoding SSA1A83G                      Luciferase Refolding. Promega (catalog  E1701) recombinant &#239;&#172;&#65533;re&#239;&#172;&#8218;y lucif- <br /> SSA2G83A, respectively. alleles cloned pRS313 (single-copy                  erase (0.5 &#206;&#188;M) denatured 50 &#206;&#188;L denaturation buffer (6 M GdmCl, <br /> HIS3 vector) generate pRS313A1-83G pRS313A2-83A monitoring                       30 mM Hepes, 50 mM KCl, 5 mM MgCl2, 5 mM &#206;&#178;-mecaptoethanol, pH 7.3) curing [URE3].                                                                           1 h 30 &#194;&#176;C. Luciferase diluted sequential steps.  75 &#206;&#188;L    assess effects hybrid Hsp70 proteins, plasmid pJ401 strain 1161             refolding buffer (denaturation buffer GdmCl &#206;&#178;-mecaptoethanol) <br /> replaced LEU2-based plasmids carrying hybrid alleles. Ssa                   added reaction incubated room temperature 1 min. Sec- <br /> protein cells plasmid. Ssa protein es-                ond, 1 &#206;&#188;L diluted solution added 50 &#206;&#188;L refolding buffer  <br /> sential, cells retain plasmid grow, rich media.                   taining 0.2 uM Ydj1p 2&#206;&#188;M Ssa1p Ssa2p                                                                                             incubated room temperature 5 min. Luciferase refolding initiated <br /> Media, Growth Conditions, Monitoring Prions. Media growth                       adding 5 mM ATP. Aliquots removed indicated intervals ditions described (29) indicated Results. [URE3] moni-                 luciferase refolding recorded using Zylux FB15 luminometer lucif- <br /> tored using wild-type ADE2 gene regulated DAL5 promoter                       erase assay reagent (Promega; catalog  E1501). Spontaneous refolding (PDAL5::ADE2) (25, 26). standard ammonium-containing media Ure2p                         luciferase added chaperones subtracted background. <br /> represses expression DAL5 promoter Ade2p produced cells Ade&#226;&#8364;&#8220; red. [URE3] present depletion functional <br />                                                                                             Ure2p Amyloid Formation. Puri&#239;&#172;&#65533;ed Ure2p (120 &#206;&#188;L 30 &#206;&#188;M stock) freshly di- <br /> Ure2p insoluble prion aggregates activates DAL5 promoter                                                                                             alyzed PBS added 76 &#206;&#188;L PBS reaction buffer. Final concentrations <br /> ADE2 expression, making cells Ade+ white. Weakened [URE3] propa- <br /> gation seen intermediate colony color (pink) increased fre-                    1 mM ATP, 20 mM MgCl2, 1 mM KCl, 18 &#206;&#188;M Ure2p 6 &#206;&#188;M Ssa1p quency appearance [ure-o] cells population. presence                      Ssa2p. Samples incubated stirring 500-&#206;&#188;L cuvettete.  <br /> [URE3] con&#239;&#172;&#65533;rmed dominant infectious phenotype ability                  dicated times, 10 &#206;&#188;L samples removed added 490 &#206;&#188;L 100 &#206;&#188;M <br /> cured cells grown presence 3 mM guanidine hydrochlo-                     Thio&#239;&#172;&#8218;avin-T. Fluorescence recorded 485 nm excitation 450 nm. <br /> ride, inactivates Hsp104 (33&#226;&#8364;&#8220;37). <br />                                                                                             Western Analysis. Yeast harvested centrifugation suspended Protein Puri&#239;&#172;&#65533;cation. Plasmid pPROEX-HTb-SSA1 expressing His6&#226;&#8364;&#8220;Ssa1 fu-                   lysis buffer (PBS 0.2% Triton X-100 protease inhibitor mixture; <br /> sion protein obtained Addgene (38). plasmid encodes                       Roche). Cells broken vortexing glass beads, cell lysate 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; direction hexa tag, TEV cleavage site, Ssa1p. Protein                  centrifuged 3,000 rpm 2 min. Supernatant boiled loading dye puri&#239;&#172;&#65533;ed Escherichia coli strain Rosetta 2(DE3) (Invitrogen) grown LB                15 min 95 &#194;&#176;C 5 &#206;&#188;g protein separated 4&#226;&#8364;&#8220;12% gradient SDS <br /> medium 300 mM NaCl 30 &#194;&#176;C OD600 = 0.6&#226;&#8364;&#8220;0.8. culture                     gels transferred PVDF membranes. Hsp70 antibodies induced 0.5 mM IPTG grown overnight 18 &#194;&#176;C. Cells har-                      Stressgen (catalog  SPA-822), FBPase antibodies gift vested lysed buffer (20 mM Hepes, 150 mM NaCl, 20 mM MgCl2,                        C. Randell Brown (Pennsylvania State College Medicine, Hershey, PA) (27), <br /> mM KCl 20, complete protease inhibitor mixture) using lysozyme. Cell debris                 Sec28p antibodies gift R. Duden (University L&#195;&#188;beck, cleared centrifugation (12,000 g) supernatant loaded                                                                                             L&#195;&#188;beck, Germany) (39). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                             GENETICS <br /> cobalt-based Talon metal af&#239;&#172;&#65533;nity resin. washing, protein eluted buffer containing 250 mM imidazole. Puri&#239;&#172;&#65533;ed protein incubated <br />                                                                                         TEV (Invitrogen; catalog  12575-023) protease 30&#194;&#176; 1 h.                                                                                              <br /> sample extensively dialyzed 4&#194;&#176; passed Talon metal               <br /> af&#239;&#172;&#65533;nity resin remove cleaved tag TEV protease. Protein                   <br /> purity 99% determined SDS/PAGE Coomassie                        <br /> staining. Protein identity con&#239;&#172;&#65533;rmed using mass spectrometry.                        <br /> </body></html>