<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Cell. Author manuscript; available PMC 2012 April 29. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Cell. 2011 April 29; 145(3): 470&#226;&#8364;&#8220;482. doi:10.1016/j.cell.2011.03.037. <br />  <br />  <br />  <br />                            High-Resolution C. elegans Essential Gene Network Based                            Phenotypic Profiling Complex Tissue <br />                            Rebecca Green, Huey-Ling Kao^,+, Anjon Audhya*,+, Swathi Arur%, Jonathan R. Mayers*, <br />                            Heidi Fridolfsson#, Monty Schulman^, Siegfried Schloissnig&#194;&#176;, Sherry Niessen&#226;&#8364;&#161;, Kimberley <br />                            Laband, Shaohe Wang, Daniel Starr#, Anthony Hyman&#194;&#176;, Tim Schedl&#194;&#167;, Arshad Desai, Fabio <br />                            Piano^, Kristin C. Gunsalus^,&#226;&#8364;&#160;, Karen Oegema&#226;&#8364;&#160; <br />                            Ludwig Institute Cancer Research Department Cellular Molecular Medicine, <br />                            University California, San Diego, CMM-East 3053, 9500 Gilman Drive, La Jolla, CA 92093 <br />                            *Department Biomolecular Chemistry, University Wisconsin-Madison Medical School, 1300 <br />                            University Avenue. Madison, WI 53706 <br />                            ^Center Genomics Systems Biology, Department Biology, New York University, 12 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            Waverly Place, 8th Floor, New York, NY 10003 <br />                            %Department Genetics, University Texas MD Anderson Cancer Center, 1515 <br />                            Holcombe Blvd, Unit 1010, Houston, TX 77030 <br />                            #Dept. Molecular Cellular Biology, College Biological Sciences, University California, <br />                            Davis, Shields Avenue, Davis, CA 95616 <br />                            &#194;&#167;Department Genetics, Washington University School Medicine, Saint Louis, Missouri, <br />                            63110 Washington University, St. Louis, MO <br />                            &#194;&#176;Max Planck Institute Molecular Cell Biology Genetics, Pfotenhauerstrasse 108, 01307 <br />                            Dresden, Germany <br />                            &#226;&#8364;&#161; Skaggs Institute <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Biology Department Chemical Physiology, Center <br />                            Physiological Proteomics, Scripps Research Institute, La Jolla, CA 92037 <br />  <br />                            SUMMARY <br />                                  High-content screening gene profiling generally limited single cells.                                   explore alternative approach&#226;&#8364;&#8221;profiling gene function analyzing effects gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  knockdowns architecture complex tissue multicellular organism. profile 554 <br />                                  essential C. elegans genes imaging gonad architecture scoring 94 phenotypic features.                                  generate reference evaluating methods network construction, genes manually <br />                                  partitioned 102 phenotypic classes, predicting functions uncharacterized genes  <br />  <br />                            &#194;&#169; 2011 Elsevier  rights reserved. <br />                            &#226;&#8364;&#160; <br />                              Corresponding authors: Karen Oegema, Phone: (858) 534-9576, Fax: (858) 534-7750, koegema@ucsd.edu <span id='am-19' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>amp</span>; Kristin Gunsalus, <br />                            Phone: (212) 998-8236, Fax: (212) 992-4015, kcg1@nyu.edu. <br />                             authors contributed equally <br />                            Author Contributions: Conceived designed experiments: KO, AD, RG, AA. Performed experiments: RG, AA, SN, SA, TS, <br />                            SW, HF, DS, JM. Scored data: AA, RG, KO. Manually partitioned genes classes: RG, KO. Reworked Phenobank                            include gonad morphology data: SS, RG, KO, AH. Developed N-Browse: HK, MS, FP, KG. Developed CSI: KO, RG, HK, MS, <br />                            FP, KG. Conceptualized performed network analysis: HK, FP, KG. Contributed reagents/materials/analysis tools: KO, RG, AA, <br />                            SN, KL, SA, TS, AD, HK, MS, KG, FP, HF, DS. <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Green et al.                                                                                                  Page 2 <br />  <br />  <br />                                     diverse cellular processes. Using classification benchmark, developed robust <br />                                     computational method constructing gene networks high-content profiles based                                     network context-dependent measure ranks significance links genes. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                     analysis reveals multi-parametric profiling complex tissue yields functional maps                                     resolution similar genetic interaction-based profiling unicellular eukaryotes&#226;&#8364;&#8221;pinpointing <br />                                     subunits macromolecular complexes components functioning common cellular <br />                                     processes. <br />  <br />  <br />                               INTRODUCTION <br />                                               major challenge post-genomic era translate parts lists generated genome <br />                                               sequencing maps pathways execute cellular processes. Approaches                                               combine systematic gene inhibition functional tests span continuum single <br />                                               readout assays complex assays interrogate broad spectrum cellular processes. <br />                                               single readout assays identify pathways impact specific process (Mathey- <br />                                               Prevot Perrimon, 2006), complex assays used construct functional networks <br />                                               collections genes diverse cellular roles. approaches emerged                                               distilling complex phenotypes phenotypic profiling: genetic interaction profiling                                               high-content screening. methodologies distinct, strategies translate                                               consequences inhibiting gene activity phenotypic profiles compared NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               generate map functional relationships genes (Boone et al., 2007; Collins et <br />                                               al., 2009; Conrad Gerlich, 2010; Piano et al., 2002; S&#195;&#182;nnichsen et al., 2005). <br />  <br />                                               Genetic interaction profiling pioneered budding yeast, using comprehensive <br />                                               deletion library non-essential genes collections hypomorphic alleles essential <br />                                               genes (Boone et al., 2007; Collins et al., 2009). Genetic interaction profiling captures                                               consequences inhibiting gene measuring effect growth rate pairwise <br />                                               inhibitions genes collection. analysis generates <br />                                               quantitative interaction profiles gene clustered reveal functionally <br />                                               significant relationships. genome-scale genetic interaction map recently constructed <br />                                               S. cerevisiae (Costanzo et al., 2010), maps generated subsets                                               gene implicated specific processes RNA processing, chromosome biology, <br />                                               proteasome function, secretory pathway (Breslow et al., 2008; Collins et al., 2007; <br />                                               Schuldiner et al., 2005; Wilmes et al., 2008). <br />  <br />                                               metazoans, genetic interaction profiling difficult implement comprehensive <br />                                               libraries deletion/hypomorphic strains exist developing reproducible high- <br />                                               throughput methods quantify fitness formidable barrier (Gunsalus, 2008). <br />                                               Consequently, high-content screening primary method mapping functional gene <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               networks animal cells. high-content screen, light microscopy used assess <br />                                               phenotypes arising gene inhibition RNA-mediated interference, phenotype <br />                                               captured scoring large parameter set (Conrad Gerlich, 2010). depth                                               phenotypic profile based biological complexity assay nature                                               accuracy parameter scoring. <span id='am-5' about='oboInOwl:date' typeof='owl:Thing'>date</span>, requirement high-resolution imaging                                               generally limited high-content screening single cells early embryos. <br />  <br />                                               C. elegans prototype metazoan functional mapping essential genes <br />                                               (Piano et al., 2006). C. elegans ~20,000 genes, ~2,500 essential embryo <br />                                               production viability (WormBase release WS210; Harris et al., 2010). set                                               pioneering high-content screens, time-lapse Differential Interference Contrast (DIC) <br />                                               microscopy used film early divisions embryos following individual inhibitions <br />                                               specific subsets C. elegans genes (G&#195;&#182;nczy et al., 2000; Piano et al., 2000; Zipperlen et <br />  <br />  <br />  <br />                                                        Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                               Page 3 <br />  <br />  <br />                                           al., 2001). extended genome screen generated high-content <br />                                           phenotypic profiles ~500 essential genes (S&#195;&#182;nnichsen et al., 2005). profiles                                           combined protein-protein interaction expression profiling data create  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           generation integrative map linked 305 essential C. elegans genes &#226;&#8364;&#339;multiple support&#226;&#8364;? <br />                                           network grouped genes modules involved specific processes including spindle <br />                                           assembly, chromosome segregation, nuclear envelope dynamics, cortical dynamics,                                           centrosome function (Gunsalus et al., 2005). Despite success studies, large <br />                                           collection essential genes profiled inhibition results sterility <br />                                           treated worm.  554 genes &#226;&#8364;&#339;sterile&#226;&#8364;? collection, control <br />                                           fundamental cellular processes membrane trafficking, translation, proteasome <br />                                           function, cortical remodeling, largely absent analysis. <br />  <br />                                           gap analysis C. elegans essential gene set, profiled 554 sterile <br />                                           genes imaging syncytial gonad architecture high-resolution following gene knockdown <br />                                           scoring 94 phenotypic parameters. generate reference evaluating computational <br />                                           methods network construction, genes manually partitioned 102 phenotypic <br />                                           classes, predicting functions 106 116 uncharacterized genes collection. <br />                                           Using manual classification benchmark, developed robust computational <br />                                           method constructing gene networks high-content profiles based network <br />                                           context-dependent measure ranks significance functional links genes. <br />                                           method allowed integrate data prior high-content embryo- <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           filming dataset generate network representation 818 essential C. elegans genes                                           viewed multiple levels functional resolution. <br />  <br />                               RESULTS <br />                               Phenotypic Profiling Based Morphology Complex Tissue: C. elegans <br />                               Gonad <br />                                           900 C. elegans genes required embryo production early <br />                                           embryonic cell divisions (S&#195;&#182;nnichsen et al., 2005); collection includes majority                                           genes essential basic processes common cells. inhibition leads                                           sterility, 554 genes profiled embryo filming. 554 sterile <br />                                           genes, 166 unnamed, indicating prior characterization (Fig. 1A). unnamed <br />                                           gene, determined predicted product member KOG (eukaryotic <br />                                           orthologous group; Tatusov et al., 2003) used Ensembl database determine                                           orthologs species. 166 unnamed genes, 50 characterized orthologs                                           predicted function C. elegans protein (Unnamed-Group Table S2).                                           remaining 116 members KOGs unknown function, predicted <br />                                           orthologs, multiple C. elegans paralogs (Unnamed-Group II Table S2); refer NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           116 genes &#226;&#8364;&#339;uncharacterized&#226;&#8364;? (Fig 1A). <br />  <br />                                           profile 554 sterile genes, developed high-content assay based 3D color <br />                                           fluorescence confocal imaging gonad, complex tissue adult C. elegans <br />                                           hermaphrodite (Fig. 1B). syncytial gonad contains ~1000 meiotic nuclei cup-shaped <br />                                           compartments open common cytoplasmic core. Compartments mature oocytes                                           progress distal tip proximal region gonad adjacent                                           spermatheca. Gonad maturation maintenance involves broad spectrum basic cellular <br />                                           processes (Fig. 1C), making tissue attractive substrate high-content profiling. <br />                                           Gonad architecture analyzed strain expressing fluorescent markers target                                           plasma membrane (GFP fusion binds PI4,5P2) chromosomes (mCherry-histone <br />                                           H2B). Hermaphrodites soaked dsRNA target gene 24 hours beginning <br />                                           late L4 larval stage, gonad achieved complement nuclei <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                           Page 4 <br />  <br />  <br />                                           (Kimble Crittenden, 2005). 48 hours recovery, gonad architecture assessed                                           triplicate anesthetizing worms imaging gonad worm. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Binary phenotypic profiles generated scoring set 3 image stacks target <br />                                           gene 94 possible defects. movie set gene inspected 94 <br />                                           defects (Fig. 1D; complete list examples Table S1), assigning &#226;&#8364;&#339;0&#226;&#8364;?                                           defect absent &#226;&#8364;&#339;1&#226;&#8364;? defect present 2 3 movies.                                           image stacks analyzed pair individuals, viewed scored                                            image stacks indexed RNA number, making analysis blind gene <br />                                           identity. 24 cases movies consistent, experiment                                           repeated  Fig. S1, Suppl. Experimental Procedures, Table S6 details screen <br />                                           design scoring methods). <br />  <br />                               Generation Benchmark Constructing Gene Networks Based Gonad Architecture <br />                               Phenotypes <br />                                           Initial attempts using raw parameter dataset automated clustering broadly grouped <br />                                           genes, failed partition resolution similar achieved                                           blinded manual classification experienced investigator. develop better <br />                                           computational method, began manually partitioning genes classes generate <br />                                           reference use evaluate computational methods network construction. <br />                                           Manual partitioning placed 554 sterile genes 102 phenotypic classes (Table S2 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           contains description, sample image, gene list class). organizational <br />                                           purposes, 102 classes grouped 29 broad categories (labeled  AA, AB                                           AC) contain classes sharing prominent defects. Movies class <br />                                           designations accessed Phenobank website <br />                                           (http://worm.<span id='am-6' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-7' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span>mpi</span>-cbg phenobank_gonad). <br />  <br />                                           manually defined phenotypic classes contained characterized genes common <br />                                           annotated molecular functions (Fig. 1E), indicating similar gonad architecture <br />                                           phenotypes reflect similar molecular functions. Manual partitioning placed 106 116 <br />                                           uncharacterized genes phenotypic classes containing characterized genes, leading                                           predictions functions. Prior using manually defined classes benchmark                                           optimize computational methods network analysis, validated functional <br />                                           predictions arising classification following 8 uncharacterized genes 5 <br />                                           classes. <br />  <br />                                           class (E2) predicted role T09E8.1 microtubule cytoskeleton (Fig. 2A). <br />                                           Unlike class E2 genes, partial T09E8.1 inhibition did lead defects early <br />                                           embryonic cell divisions  shown). dividing cells, microtubule cytoskeleton NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           organized centrosomes, gonad dominated non-centrosomal microtubule <br />                                           arrays (Zhou et al., 2009).  hypothesized T09E8.1 specifically required                                           non-centrosomal microtubule array formation. Hypodermal cells, born late                                           embryogenesis, non-centrosomal microtubule arrays function nuclear <br />                                           positioning (Fridolfsson Starr, 2010). Partial T09E8.1 inhibition hypodermis led <br />                                           defects nuclear positioning (Fig. 2B), ~3 fold reduction number                                           increase length GFP-EBP-1 &#226;&#8364;&#339;comets&#226;&#8364;? mark growing microtubule ends <br />                                           (Fig. 2C). conclude T09E8.1, noca-1  non-centrosomal array), <br />                                           plays important role organization non-centrosomal microtubule arrays                                           gonad hypodermis. <br />  <br />                                           second class (G1) predicted role F54D12.5 DAF-21, hsp90-family <br />                                           chaperone, MAPK signaling pathway (Fig. 2D). tested predictions using                                           genetic test. Worms homozygous reduction function allele mpk-1(ga111) exhibit <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                           Page 5 <br />  <br />  <br />                                           normal gonad morphology, phosphorylated MPK-1 levels, readout                                           pathway activity, reduced compared controls (Fig. 2E,F; Lee et al., 2007). Partial <br />                                           knockdown daf-21 F54D12.5, conditions did result morphological <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           phenotype control worms, led strong MAPK knockdown phenotype                                           mpk-1(ga111) background (Fig. 2E,F; Fig. S2, Table S3). RNAi daf-21 reduced <br />                                           phosphorylated MPK-1 levels control let-60 gain function worms (Fig. 2E,F; <br />                                           Table S3), indicating DAF-21 acts level downstream LET-60/Ras                                           MAPK pathway (Fig. 2G). F54D12.5 RNAi did reduce phosphorylated MPK-1 levels, <br />                                           F54D12.5 contains 2 potential MPK-1 docking sites suggesting MPK-1 <br />                                           substrate (Fig. 2G). conclude DAF-21 F54D12.5 function different points                                           MAPK signaling pathway F54D12.5, eom-1, enhancer mpk-1(ga111). <br />  <br />                                           class (S2) predicted roles uncharacterized proteins anaphase- <br />                                           promoting complex/cyclosome (APC/C; Fig. 3A). Immuno-affinity purification                                           uncharacterized gene product, K10D2.4, C. elegans extracts followed mass <br />                                           spectrometry recovered seven APC components (Fig. 3B). product                                           uncharacterized Class S2 gene, C09H10.7, recovered&#226;&#8364;&#8221;indicating                                           K10D2.4 C09H10.7 APC/C subunits. K10D2.4 recently identified                                           metazoan-specific component APC/C (Hubner et al., 2010; Hutchins et al., 2010; <br />                                           Kops et al., 2010) gene named emb-1/apc-16. C09H10.7, apc-17. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           fourth class (F2) predicted role BTB-domain containing protein C08C3.4                                           cortical remodeling/cytokinesis (Fig. 3C). GFP fusion C08C3.4 localized                                           contractile ring tip cleavage furrow dividing embryos (Fig. 3D), embryos <br />                                           partially depleted C08C3.4 exhibited cytokinesis defects (Fig. 3E). conclude                                           C08C3.4 required cortical remodeling gonad cytokinesis embryos                                           gene cyk-7. <br />  <br />                                           fifth class (I2) phenotype included debris labeled plasma membrane probe, <br />                                           suggesting role membrane trafficking (Fig. 3F). tested trafficking function                                           imaging compartment boundaries strain expressing mCherry labeled plasma <br />                                           membrane probe GFP fusion SNB-1, SNARE trafficked                                           endomembrane delivered plasma membrane (Fig. 3G). Compartment <br />                                           boundaries control worms SNB-1-GFP plasma membrane probe                                           yellow. Trafficking defects prevent SNB-1-GFP reaching plasma membrane, <br />                                           leading red compartment boundaries. 8 15 Class I2 genes, including F27C8.6                                           T01B7.6, exhibited defects SNB-1 assay (Fig. 3G). F27C8.6 T01B7.6 <br />                                           trcs-1 trcs-2, respectively, (transport cell surface). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           follow work classes demonstrates gonad architecture sufficient <br />                                           resolution functionally classify genes broad spectrum essential cellular <br />                                           functions. validates manual classification, establishing benchmark                                           evaluating computational methods network construction. <br />  <br />                               Development Network Context-Dependent Method Evaluate Significance                               Functional Links Genes Based High-Content Phenotypic Profiles <br />                                           initial efforts using automated clustering unable partition genes resolution <br />                                           comparable achieved experienced investigator. circumvent                                           limitation, used manually-defined classes tool develop robust computational <br />                                           method constructing gene networks based high-content parameter profiles.                                           compared phenotypic profiles calculating Pearson&#226;&#8364;&#8482;s Correlation Coefficient <br />                                           (PCC) pair genes. resulting network visualized using N-Browse,                                           interactive Java-based tool (Kao Gunsalus, 2008), display connections genes <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                              Page 6 <br />  <br />  <br />                                           profiles correlated PCC greater equal specified threshold <br />                                           (Fig. 4A,B; dark blue lines connecting grey gene nodes). assess effectiveness                                           approach, circled gene clusters corresponded manually-defined classes. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           approach revealed optimal PCC threshold viewing functionally relevant <br />                                           connections (red outlined boxes Fig. 4B) varied substantially different network <br />                                           neighborhoods. variability varying nature profiled phenotypes                                           extent captured parameter set, extent scored <br />                                           parameters related versus independent, fact profiles features                                           exhibit variance. Viewing entire network single PCC threshold possible <br />                                           regions threshold low view cluttered non-specific <br />                                           connections (Fig 4B, images right red boxed images), regions                                           threshold high, yielding network meaningful connections                                           absent (Fig. 4B, images left red boxed images). <br />  <br />                                           circumvent limitations PCC-based analysis, developed measure ranks                                           significance functional links genes based network context. pair                                           genes B, assigned Connection Specificity Index (CSI)  1) calculating                                           PCC connections B genes dataset, 2) <br />                                           counting number genes connected B PCC &#226;&#8240;&#165; PCCAB &#206;&#188; 0.05  e. level <br />                                           comparable better correlation B, correlations PCC                                           0.05 PCCAB considered similar offset determined empirically); 3) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           dividing number total number genes screen (554); 4) subtracting                                           result 1.0 (Fig. 4C). CSI equivalent fraction genes dataset                                           profiles similar B profile profile B.                                           example, CSI 0.97 means similarity B highly specific:                                           ~3% gene knockdowns profiles comparable higher similarity                                           B. CSI scales uniformly functional significance entire network, <br />                                           connections similar level significance simultaneously displayed single <br />                                           CSI threshold (Fig. 4D). <br />  <br />                                           evaluated performance CSI PCC threshold networks comparing ability <br />                                           automated clustering algorithm   Module Identification NEtworks; K. <br />                                           Rhrissorrakrai K.C. Gunsalus, press) identify gene clusters corresponding                                           manually-defined classes. analysis revealed useful range                                           parameters, networks generated using CSI thresholds ~3-fold fewer connections                                           comparable PCC networks; noise reduction translates substantial improvement                                           ability automated algorithm identify functionally relevant gene clusters (Fig. <br />                                           4E,F; Fig. S3). Approximately half manually-defined classes containing 4                                           genes identified automated clustering network constructed using single <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           CSI threshold 0.96, 14% identified comparable PCC network <br />                                           (Fig. 4F). allowed search networks spanning range CSI thresholds <br />                                           (0.90 0.99), clusters corresponding ~90% manually-defined classes                                           identified, compared 65% networks spanning range PCC thresholds (0.5&#226;&#8364;&#8220;1). Heat <br />                                           map dendograms revealed sharply defined clusters using CSI, indicating <br />                                           CSI increases network clarity (Fig. 4G; Table S4). <br />  <br />                                           conclude constructing networks using CSI thresholds circumvents variability                                           significance strict correlation measure different network regions arises                                           complexity high-content screening parameters. Using CSI instead PCC <br />                                           reduces connection noise allows connections similar functional significance                                           viewed entire network single threshold. <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                          Page 7 <br />  <br />  <br />                               Varying CSI Threshold Reveals Functional Modularity Gene Network Different <br />                               Levels Resolution <br />                                           CSI-based network representation allows exploration functional modularity NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           different levels resolution. point illustrated region gene network <br />                                           involved protein production (Fig. 5A): relatively low threshold CSI 0.90 connects                                           entire set genes involved protein translation, mRNA splicing, protein folding                                           dense meshwork. intermediate CSI threshold 0.93 results sparser network                                           specific connections defines smaller gene groups. high CSI threshold 0.97,                                           chaperonins, tRNA synthetases, translation initiation/elongation factors, small ribosome <br />                                           subunits, large ribosome subunits, splicing factors resolved separate clusters. <br />                                            dialing CSI reveals functional relationships different levels                                           resolution. automated clusters identified different CSI thresholds                                           provided supplement (Table S5; note genes multiple clusters). <br />  <br />                                           assessed resolution limit gonad architecture assay using region                                           network representing genes involved protein degradation (Fig. 5B, Fig. S4).                                           high CSI threshold 0.99, connections remained linked genes specific <br />                                           proteasome subcomplexes  Lid, core &#206;&#178;-ring, core &#206;&#177;-ring 19S ATPase base; <br />                                           Fig. 5B). conclude phenotypic profiling based complex tissue architecture, <br />                                           coupled automated construction CSI-based networks parameter profiles,                                           capable correctly assigning fine distinctions protein function. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               CSI-Enabled Integration High-Content Data Sets Generates Global View C. <br />                               elegans Essential Gene Network <br />                                            gonad architecture (554 genes) time-lapse embryo filming (661 genes) <br />                                           screens provide high-content profiles 885 essential genes (330 profiled                                           screens). CSI ideally suited integrating datasets filters low <br />                                           specificity phenotypic links, leading network combines significant <br />                                           relationships identified screens. binarized phenotypic signatures                                           embryo timelapse screen, based scoring 45 possible defects (S&#195;&#182;nnichsen <br />                                           et al., 2005), calculated CSI gene pair, simultaneously displayed data <br />                                           sets N-Browse create integrated network. merged network, network regions <br />                                           centered genes  viewed CSI threshold N-Browse (instructions <br />                                           demo video describing use N-Browse accessed                                           http://worm.mpi-cbg phenobank_gonad/nbrowse). CSI threshold 0.96,                                           integrated network 3382 high significance connections linking 818 genes (Fig. 6A&#226;&#8364;&#8220;C). <br />                                           combined view provides functional information ~90% genes required                                           embryo production embryonic divisions. Despite presence 330 genes <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           common, relationships identified datasets highly orthogonal 23 <br />                                           total 3382 phenotypic links shared (Fig. 6A). lack overlap explained                                           majority 330 common genes failing make high-significance connections                                           embryo time-lapse screen sterility onset (Fig. S5). Overall, CSI-based network <br />                                           integrates information acquired biologically distinct contexts gonad embryo <br />                                           provide multi-layered view function 818 genes required essential cellular <br />                                           processes multicellular organism. <br />  <br />                               DISCUSSION <br />                                           High-content screening primary method mapping functional gene networks                                           animal cells (Collins et al., 2009; Conrad Gerlich, 2010; Gunsalus, 2008). number                                           high-content screens classified genes based cell morphology following gene <br />                                           knockdown (examples include Bakal et al., 2007; Echard et al., 2004; Eggert et al., 2004; <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 8 <br />  <br />  <br />                                           Goshima et al., 2007; Liu et al., 2009; Neumann et al., 2010; Piano et al., 2002; S&#195;&#182;nnichsen <br />                                           et al., 2005). successful identifying genes contribute cell division                                           morphology, ability screens assess protein function breadth NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           cellular processes falls far short attained using genetic interaction profiling <br />                                           fungi. Screen resolution ability discriminate different biological <br />                                           functions&#226;&#8364;&#8221;depends information content phenotypic assay;                                            approach enhance resolution time-lapse imaging (Conrad Gerlich, 2010). <br />                                            explored alternative approach&#226;&#8364;&#8221;analyzing effects gene knockdowns                                           complex tissue architecture single timepoint multicellular organism. monitored <br />                                           C. elegans gonad architecture following gene knockdown scoring 94 phenotypic features. <br />                                           results demonstrate biological complexity gonad translates                                           phenotypes enable profiling wide spectrum essential cellular processes                                           resolution approaching genetic interaction profiling yeasts. <br />  <br />                               Sterile Gene Collection: Prominent Gap Functional Genomic Analysis                               C. elegans Essential Gene Set <br />                                           Sequencing C. elegans genome discovery RNAi catalyzed efforts                                           systematically catalog functions predicted genes (Piano et al., 2006). C. elegans <br />                                           ~900 essential genes required embryo production events                                           embryonic divisions. subset genes previously characterized time-lapse <br />                                           imaging early embryos (Fraser et al., 2000; G&#195;&#182;nczy et al., 2000; Piano et al., 2002; <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           S&#195;&#182;nnichsen et al., 2005).  sterility onset following RNAi major complication <br />                                           precluded depth characterization large number genes. efforts focused                                           554 sterile genes, profiled imaging gonad architecture following RNAi <br />                                           knockdowns generating parameterized description resulting phenotypes. <br />  <br />                                           analysis generated functional predictions 106 116 uncharacterized sterile <br />                                           genes. predictions span variety cellular processes including membrane <br />                                           trafficking, glycosylation, fatty acid synthesis, cortical dynamics, chromosome structure/ <br />                                           segregation, RNA splicing, mitochondrial function, microtubule cytoskeleton, proteasome <br />                                           function, translation, MAPK signaling, transcription, RNA binding, metaphase- <br />                                           anaphase transition. validated predictions 8 genes play important roles                                           microtubule cytoskeleton, MAP kinase signaling, metaphase-anaphase transition, <br />                                           cortical remodeling/cytokinesis, membrane trafficking. gonad used                                           &#226;&#8364;&#339;test tube&#226;&#8364;? functionally profile genes, subsequent work early later stage <br />                                           embryos indicates majority essential genes collection broadly <br />                                           important. 554 sterile genes, ~60% predicted homologs higher organisms,                                           50% 106 uncharacterized genes predicted homolog conserved <br />                                           domain.  addition filling large gap analysis C. elegans essential <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           gene set, findings provide starting point address functions related genes                                           humans. <br />  <br />                               High-Resolution Phenotypic Profiling Based Morphology Complex Tissue <br />                                           original motivation screen profile sterile gene set, fortuitous <br />                                           byproduct realization complex tissue, structure depends                                           dynamic coordination broad spectrum cellular processes, ideal substrate                                           phenotypic profiling. major finding work profiles based complex tissue <br />                                           architecture acquired single time point following gene knockdown greater <br />                                           information depth profiles derived timelapse imaging divisions                                           early embryo  single cell). example, shown Figure 5, gonad data partitions <br />                                           set genes involved protein production 15 different classes (chaperones, <br />                                           ribosome components, tRNA synthetases, proteins involved protein folding, splicing <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 9 <br />  <br />  <br />                                           components ) distinctions based embryo filming <br />                                           data. high level resolution generally holds dataset. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           note means inferring function single point morphology assay distinct <br />                                           used embryo-filming screen. embryo screen, gene function                                           inferred directly phenotype. example, defects spindle assembly, chromosome <br />                                           segregation, polarity establishment led assignment classes implicated                                           corresponding processes (S&#195;&#182;nnichsen et al., 2005). contrast, link cataloged <br />                                           phenotypic features gonad architecture assay gene function direct                                           know knockdown genes specific functions lead specific spectrum                                           phenotypic features. Instead, protein function inferred comparing profile                                           phenotypic features genes dataset, process conceptually analogous <br />                                           function inferred genetic interaction profiles budding yeast. <br />  <br />                                           presence proteasome components dataset offered opportunity compare <br />                                           resolution approach genetic interaction profiling yeast. proteasome                                           2.5-MDa macro-molecular machine contains 30 different subunits organized                                           20S core complex, composed &#206;&#177; &#206;&#178; rings, 19S regulatory particle, containing base <br />                                           lid sub-complexes (Bochtler et al., 1999). yeast, distinct genetic interaction profiles <br />                                           obtained subunits proteasome sub-complexes using sensitive <br />                                           competitive growth assay (Breslow et al., 2008). CSI &#226;&#8240;&#164; 0.99  high-stringency filter), <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           connections remained subunits specific proteasome subcomplexes <br />                                           (&#206;&#177; ring, &#206;&#178; ring, lid base; Fig. 5B). observation indicates phenotypic profiles based <br />                                           gonad morphology provide resolution sufficient group subunits specific proteasome <br />                                           subcomplexes. <br />  <br />                                           core dataset profiles composed parameters visually scored                                           experienced investigators acquired automated image analysis. Given                                           complexity substrate 3 dimensional tissue living organism variably <br />                                           positioned worm large spectrum knockdown phenotypes entirely <br />                                           change properties structure (gonad size, shape, position, compartment nuclei <br />                                           number morphology) automated parameter scoring exceedingly <br />                                           difficult.  exact properties make automated analysis difficult varied <br />                                           dramatic effects gene knockdowns gonad architecture                                           properties assay profiling power. manual parameter scoring                                           introduce bias, minimized performing analysis blinded gene identity <br />                                           fact individual parameters scored investigators largely <br />                                           oblivious larger patterns ultimately emerge. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                               Robust Computational Method Constructing Gene Networks High-Content <br />                               Screening Data <br />                                           evaluating computational methods network construction using validated manual <br />                                           classification rich phenotypic dataset, able devise robust computational <br />                                           method constructing gene networks high-content phenotypic profiles. method <br />                                           overcomes challenges encountered network analysis high-content datasets.                                           challenge level profile correlation significant varies different <br />                                           network regions varying nature profiled phenotypes extent                                           captured parameter set. second challenge commonly encountered <br />                                            informative) phenotypes generate &#226;&#8364;&#339;hairball&#226;&#8364;? connections                                           obscures meaningful functional links. center method developed                                           overcome challenges simple metric CSI, network context- <br />                                           dependent measure ranks significance functional links genes. Compared <br />                                           Pearson&#226;&#8364;&#8482;s Correlation Coefficient, constructing networks based CSI reduces <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 10 <br />  <br />  <br />                                           non-specific connection noise, improves network clarity, allows connections similar <br />                                           functional significance simultaneously viewed entire network single <br />                                           threshold. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Ranking connection significance allows exploration gene network different levels <br />                                           functional resolution integration high-content screening data different <br />                                           sources. demonstrate usefulness CSI using integrate high-content <br />                                           data gonad architecture screen prior embryo-filming screen,                                           generate integrated network provides multi-layered view 818 genes C. <br />                                           elegans essential gene set. phenotypic profiles datasets composed                                           parameters visually scored measured automated image <br />                                           analysis.  constructing networks based phenotypic profiles faces                                           challenges, regardless parameters scored manual automated <br />                                           means. Consequently, anticipate CSI-based method described                                           equal utility analyzing integrating datasets composed parameters acquired                                           automated analysis. <br />  <br />                               EXPERIMENTAL PROCEDURES <br />                               C. elegans Strains <br />                                           Strains listed Table S7. strains OD95, UD299, DP38, OD70, MSN142, NL2098, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           BS3623 previously described (Arur et al., 2009; Essex et al., 2009, Fridolfsson                                           Star, 2010; Kachur et al., 2008; Maduro Pilgrim, 1995; Shi et al., 2010; Sijen et al., <br />                                           2001). OD447 generated using PDS-1000 Biolistic Particle Delivery                                           (Bio-Rad Laboratories; Praitis et al., 2001) bombard construct containing C08C3.4 <br />                                           genomic locus cloned SpeI site pIC26 (Cheeseman et al., 2004) DP38. <br />                                           OD449 generated mating OD447 OD70. <br />  <br />                               RNA Production <br />                                           Templates dsRNA production generated using primers tails containing                                           T3 T7 promoters amplify amplify 500&#226;&#8364;&#8220;1000 bp region corresponding gene <br />                                           genomic DNA. possible, oligo pairs used S&#195;&#182;nnichsen et al. (2005)                                           chosen. New oligos designed genes S&#195;&#182;nnichsen screen                                           S&#195;&#182;nnichsen oligo pairs amplified introns regions smaller larger 500&#226;&#8364;&#8220;1000 bp <br />                                           (oligos listed Table S2). PCR reactions cleaned (PerfectPrep 96 kit; Eppendorf) <br />                                           used templates T3 T7 transcription reactions (Megascript T3 T7 kits, <br />                                           Ambion). Transcription reactions mixed, cleaned (MegaClear96 kit, Ambion),                                           annealed adding 3x Soaking buffer (32.7 mM Na2HPO4, 16.5 mM KH2PO4, 6.3 mM <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           NaCl, 14.1 mM NH4Cl) final concentration 1X incubating reactions 68&#194;&#176;C <br />                                           10 minutes followed 37&#194;&#176;C 30 minutes. <br />  <br />                               RNA Interference <br />                                           Larval (L4 stage) worms strain OD95 rinsed M9 soaked 5 &#206;&#188;l                                           dsRNA (supplemented 0.5 &#206;&#188;l 50/50 mixture 63 mM <span id='am-26' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span>spermidine</span> 1.1 % <br />                                           gelatin) 24 hrs 20 &#194;&#176;C inside humid chamber. Worms transferred NGM plates <br />                                           seeded OP50 E. coli allowed recover 20&#194;&#176;C 48 hrs prior imaging. <br />  <br />                               Live Imaging <br />                                           worms anesthetized fresh mixture 1 mg/ml Tricane (ethyl 3-aminobenzoate <br />                                           methanesulfonate salt) 0.1 mg/ml tetramisole hydrochloride (TMHC) dissolved                                           M9 15&#226;&#8364;&#8220;30 minutes transferring agarose pad coverslip                                           imaging. Gonads imaged collecting 80 x 0.5&#206;&#188;m series including DIC, GFP  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 11 <br />  <br />  <br />                                           RFP images plane. Microscopy performed spinning disk confocal <br />                                           mounted Nikon TE2000-E inverted microscope equipped 60x 1.4 NA <br />                                           PlanApochromat lens, krypton-argon 2.5 W water-cooled laser (Spectra-Physics, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Mountain View, CA) Hamamatsu Orca ER CCD camera. Acquisition parameters, <br />                                           shutters, focus controlled MetaMorph software (Molecular Devices, <br />                                           Downington, PA). <br />  <br />                               Sterility Assessment <br />                                           time worms cohort anesthetized imaged, 3 worms <br />                                           moved fresh plate. 24 hours 20&#194;&#176;C, number embryos L1 progeny <br />                                           plate counted. gene &#226;&#8364;&#339;sterile&#226;&#8364;? embryos L1 larvae observed, <br />                                           &#226;&#8364;&#339;partially sterile&#226;&#8364;? &lt;50 embryos/L1 worms present, &#226;&#8364;&#339;wild-type&#226;&#8364;? &gt;50 embryos/ <br />                                           L1 worms present. additional 24 hours, assessed knockdown <br />                                           resulted embryonic lethality. &gt;50 L1-L2 larvae present, considered <br />                                           &#226;&#8364;&#339;viable&#226;&#8364;?, &lt;50 L1-L2 larvae present &gt;5 dead embryos,                                           considered &#226;&#8364;&#339;embryonic lethal&#226;&#8364;?. <br />  <br />                               Antibody Production <br />                                           Antibodies K10D2.4 generated using oligos <br />                                           cgcgcgggatccgctttgatgtacccattcca gcgcgcgaattctcaaggatttgcaggcatattt amplify region <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           encoding amino acids 2&#226;&#8364;&#8220;81 cDNA library. product digested Bam HI/ <br />                                           EcoRI cloned pGEX6P-1 digested enzymes (GE Healthcare Life <br />                                           Sciences). purified GST fusion protein outsourced injection rabbits <br />                                           (Covance). Antibodies affinity purified binding columns antigen <br />                                           removal GST tag described previously (Desai et al., 2003). <br />  <br />                               Immunoprecipitation Mass Spectrometry <br />                                           K10D2.4 immunoprecipiated extracts prepared frozen worm pellets (Desai et <br />                                           al., 2003) mass spectrometry performed (Cheeseman et al., 2004) described, <br />                                           using recent version predicted C. elegans proteins (Wormprep111). <br />  <br />                               Hypodermal Cell Experiments, MAP Kinase Experiments, Membrane Trafficking Assay <br />                                           Figure 2C, embryos produced adult hermaphrodites strain UD299                                           imaged described (Fridolfson Starr, 2010), 20 hours injection dsRNA                                           T09E8.1 gonad. Nuclear migration scored L1 larvae using DIC optics                                           described (Starr et al., 2001). attenuated RNAi, gonad dissection, immunofluorescence, <br />                                           gonad analysis Figure 2E F (Arur et al., 2009) GFP-SNB-1 trafficking <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           assay (Shi et al., 2010) performed described. <br />  <br />                               Phenobank N-Browse <br />                                           Movies accessed Phenobank website <br />                                           (http://worm.mpi-cbg phenobank_gonad) RNAiDB (http://www.rnai.org). <br />                                           Written instructions demo video explaining search Phenobank                                           http://worm.mpi-cbg phenobank_gonad/project. Written instructions demo video <br />                                           describing visualize gene network N-Browse2                                           http://worm.mpi-cbg phenobank_gonad/nbrowse. described instructions, gonad <br />                                           embryo data viewed N-Browse2 entering URL <br />                                           http://gnetbrowse.org/gonad.jnlp browser providing password <br />                                           &#226;&#8364;&#339;phenotypes&#226;&#8364;? prompt. Additional information N-Browse, including                                           requirements tutorials, http://www.gnetbrowse.org. <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                       Page 12 <br />  <br />  <br />                               Automated clustering using                                           (Module Identification Networks) graph clustering algorithm shows <br />                                           comparable better performance compared similar algorithms identifying <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           functional modules dense interaction networks (K. Rhrissorrakrai K.C. Gunsalus,                                           press).  similar MCODE (Bader Hogue, 2003) uses modified <br />                                           weighting scheme takes account network modularity, available Cytoscape <br />                                           plug http://www.cytoscape.org Perl package  request authors). <br />  <br />                               Supplementary Material <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                           work supported grants American Cancer Society (PF-06-254-01-CCG, RG), Helen Hay <br />                                           Whitney Foundation  AA), NIH (R01 GM074207 KO, R01 HD046236, KCG FP; R01 <br />                                           GM085503, KCG; R01 GM085150 TS; R01 GM088151 AA) funding Ludwig Institute                                           Cancer Research KO AD. thank Kahn Rhrissorrakrai algorithm, Michael Volkmer                                           movie formatting, Caenorhabditis Genetics Center, funded NIH National Center Research <br />                                           Resources, strains. <br />  <br />  <br />                               References <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Arur S, Ohmachi M, Nayak S, Hayes M, Miranda  Hay  Golden  Schedl T. Multiple ERK <br />                                             substrates execute single biological processes Caenorhabditis elegans germ-line development. <br />                                             Proc Natl Acad Sci U S  2009; 106:4776&#226;&#8364;&#8220;4781. [PubMed: 19264959] <br />                                           Bader GD, Hogue CW. automated method finding molecular complexes large protein <br />                                             interaction networks. BMC Bioinformatics. 2003; 4:2. [PubMed: 12525261] <br />                                           Bakal C, Aach J, Church G, Perrimon N. Quantitative morphological signatures define local signaling <br />                                             networks regulating cell morphology. Science. 2007; 316:1753&#226;&#8364;&#8220;1756. [PubMed: 17588932] <br />                                           Bochtler M, Ditzel L, Groll M, Hartmann C, Huber R. proteasome. Annu Rev Biophys Biomol <br />                                             Struct. 1999; 28:295&#226;&#8364;&#8220;317. [PubMed: 10410804] <br />                                           Boone C, Bussey H, Andrews BJ. Exploring genetic interactions networks yeast. Nat Rev <br />                                             Genet. 2007; 8:437&#226;&#8364;&#8220;449. [PubMed: 17510664] <br />                                           Boxem M, Maliga  Klitgord N, Li N, Lemmens  Mana M, Lichtervelde L, Mul JD, van Peut <br />                                             D, Devos M, et al. protein domain-based interactome network C. elegans early <br />                                             embryogenesis. Cell. 2008; 134:534&#226;&#8364;&#8220;545. [PubMed: 18692475] <br />                                           Breslow DK, Cameron DM, Collins SR, Schuldiner M, Stewart-Ornstein J, Newman HW, Braun S, <br />                                             Madhani HD, Krogan NJ, Weissman JS. comprehensive strategy enabling high-resolution <br />                                             functional analysis yeast genome. Nat Methods. 2008; 5:711&#226;&#8364;&#8220;718. [PubMed: 18622397] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Cheeseman IM, Niessen S, Anderson S, Hyndman F, Yates JR 3rd, Oegema K, Desai  conserved <br />                                             protein network controls assembly outer kinetochore ability sustain tension. Genes <br />                                             Dev. 2004; 18:2255&#226;&#8364;&#8220;2268. [PubMed: 15371340] <br />                                           Collins SR, Miller KM, Maas NL, Roguev  Fillingham J, Chu CS, Schuldiner M, Gebbia M, Recht <br />                                             J, Shales M, et al. Functional dissection protein complexes involved yeast chromosome <br />                                             biology using genetic interaction map. Nature. 2007; 446:806&#226;&#8364;&#8220;810. [PubMed: 17314980] <br />                                           Collins SR, Weissman JS, Krogan NJ. information knowledge: new technologies defining <br />                                               gene function. Nat Methods. 2009; 6:721&#226;&#8364;&#8220;723. [PubMed: 19953683] <br />                                           Conrad C, Gerlich DW. Automated microscopy high-content RNAi screening. J Cell Biol. 2010; <br />                                               188:453&#226;&#8364;&#8220;461. [PubMed: 20176920] <br />                                           Costanzo M, Baryshnikova  Bellay J, Kim Y, Spear ED, Sevier CS, Ding H, Koh JL, Toufighi K, <br />                                               Mostafavi S, et al. genetic landscape cell. Science. 2010; 327:425&#226;&#8364;&#8220;431. [PubMed: <br />                                               20093466] <br />  <br />  <br />  <br />  <br />                                                     Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                  Page 13 <br />  <br />  <br />                                           Desai  Rybina S, Muller-Reichert T, Shevchenko  Hyman  Oegema K. KNL-1 directs assembly <br />                                               microtubule-binding interface kinetochore C. elegans. Genes Dev. 2003; 17:2421&#226;&#8364;&#8220; <br />                                               2435. [PubMed: 14522947] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Echard  Hickson GR, Foley E, O&#226;&#8364;&#8482;Farrell PH. Terminal cytokinesis events uncovered RNAi <br />                                               screen. Curr Biol. 2004; 14:1685&#226;&#8364;&#8220;1693. [PubMed: 15380073] <br />                                           Eggert  Kiger AA, Richter C, Perlman ZE, Perrimon N, Mitchison TJ, Field CM. Parallel chemical <br />                                               genetic genome-wide RNAi screens identify cytokinesis inhibitors targets. PLoS Biol. <br />                                               2004; 2:e379. [PubMed: 15547975] <br />                                           Essex  Dammermann  Lewellyn L, Oegema K, Desai  Systematic analysis Caenorhabditis <br />                                               elegans reveals spindle checkpoint composed largely independent branches. Mol. <br />                                               Biol. Cell. 2009; 20:1252&#226;&#8364;&#8220;1267. <br />                                           Fraser AG, Kamath RS, Zipperlen P, Martinez-Campos M, Sohrmann M, Ahringer J. Functional <br />                                               genomic analysis C. elegans chromosome systematic RNA interference. Nature. 2000; <br />                                               408:325&#226;&#8364;&#8220;330. [PubMed: 11099033] <br />                                           G&#195;&#182;nczy P, Echeverri C, Oegema K, Coulson  Jones SJ, Copley RR, Duperon J, Oegema J, Brehm <br />                                               M, Cassin E, et al. Functional genomic analysis cell division C. elegans using RNAi genes <br />                                               chromosome III. Nature. 2000; 408:331&#226;&#8364;&#8220;336. [PubMed: 11099034] <br />                                           Fridolfsson HN, Starr DA. Kinesin-1 dynein nuclear envelope mediate bidirectional <br />                                               migrations nuclei. J Cell Biol. 2010; 191:115&#226;&#8364;&#8220;128. [PubMed: 20921138] <br />                                           Goshima G, Wollman R, Goodwin SS, Zhang N, Scholey JM, Vale RD, Stuurman N. Genes required <br />                                               mitotic spindle assembly Drosophila S2 cells. Science. 2007; 316:417&#226;&#8364;&#8220;421. [PubMed: <br />                                               17412918] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Gunsalus KC. Caenorhabditis elegans genetic-interaction map wiggles view. J Biol. 2008; 7:8. <br />                                               [PubMed: 18341704] <br />                                           Gunsalus KC, Ge H, Schetter AJ, Goldberg DS, Han JD, Hao T, Berriz GF, Bertin N, Huang J, <br />                                               Chuang LS, et al. Predictive models molecular machines involved Caenorhabditis elegans <br />                                               early embryogenesis. Nature. 2005; 436:861&#226;&#8364;&#8220;865. [PubMed: 16094371] <br />                                           Harris TW, Antoshechkin  Bieri T, Blasiar D, Chan J, Chen WJ, La Cruz N, Davis P, Duesbury <br />                                               M, Fang R, et al. WormBase: comprehensive resource nematode research. Nucleic Acids <br />                                               Res. 2010; 38:D463&#226;&#8364;&#8220;467. [PubMed: 19910365] <br />                                           Hubner NC, Bird AW, Cox J, Splettstoesser B, Bandilla P, Poser  Hyman  Mann M. Quantitative <br />                                               proteomics combined BAC TransgeneOmics reveals vivo protein interactions. J Cell Biol. <br />                                               2010; 189:739&#226;&#8364;&#8220;754. [PubMed: 20479470] <br />                                           Hutchins JR, Toyoda Y, Hegemann B, Poser  Heriche JK, Sykora MM, Augsburg M, Hudecz O, <br />                                               Buschhorn BA, Bulkescher J, et al. Systematic Analysis Human Protein Complexes Identifies <br />                                               Chromosome Segregation Proteins. Science. 2010; 328:593&#226;&#8364;&#8220;599. [PubMed: 20360068] <br />                                           Kachur TM, Audhya  Pilgrim DB. UNC-45 required NMY-2 contractile function early <br />                                               embryonic polarity establishment germline cellularization C. elegans. Dev. Biol. 2008; <br />                                               314:287&#226;&#8364;&#8220;299. <br />                                           Kimble, J.; Crittenden, SL. WormBook. Germline proliferation control. C. elegans <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Research Community, WormBook. August 15. 2005 http://www.wormbook.org <br />                                           Kops GJ, Voet MV, Manak MS, van Osch MH, Naini SM, Brear  McLeod IX, Hentschel DM, Yates <br />                                               JR 3rd, van den Heuvel S, et al. APC16 conserved subunit anaphase-promoting <br />                                               complex/cyclosome. J Cell Sci. 2010; 123:1623&#226;&#8364;&#8220;1633. [PubMed: 20392738] <br />                                           Lee MH, Ohmachi M, Arur S, Nayak S, Francis R, Church D, Lambie E, Schedl T. Multiple functions <br />                                               dynamic activation MPK-1 extracellular signal-regulated kinase signaling Caenorhabditis <br />                                               elegans germline development. Genetics. 2007; 177:2039&#226;&#8364;&#8220;2062. [PubMed: 18073423] <br />                                           Li S, Armstrong CM, Bertin N, Ge H, Milstein S, Boxem M, Vidalain PO, Han JD, Chesneau  Hao <br />                                               T, et al. map interactome network metazoan C. elegans. Science. 2004; 303:540&#226;&#8364;&#8220; <br />                                               543. [PubMed: 14704431] <br />                                           Liu T, Sims D, Baum B. Parallel RNAi screens different cell lines identify generic cell <br />                                               type-specific regulators actin organization cell morphology. Genome Biol. 2009; 10:R26. <br />                                               [PubMed: 19265526] <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                                Page 14 <br />  <br />  <br />                                           Maduro M, Pilgrim D. Identification cloning unc-119, gene expressed Caenorhabditis <br />                                               elegans nervous  Genetics. 1995; 141:977&#226;&#8364;&#8220;988. [PubMed: 8582641] <br />                                           Mathey-Prevot B, Perrimon N. Drosophila genome-wide RNAi screens: delivering NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               promise? Cold Spring Harb Symp Quant Biol. 2006; 71:141&#226;&#8364;&#8220;148. [PubMed: 17381290] <br />                                           Neumann B, Walter T, Heriche JK, Bulkescher J, Erfle H, Conrad C, Rogers P, Poser  Held M, <br />                                               Liebel U, et al. Phenotypic profiling human genome time-lapse microscopy reveals cell <br />                                               division genes. Nature. 2010; 464:721&#226;&#8364;&#8220;727. [PubMed: 20360735] <br />                                           Oegema, K.; Hyman, AA. WormBook. Cell division. C. elegans Research Community, <br />                                               WormBook. January 19. 2006 http://www.wormbook.org <br />                                           Piano, F.; Gunsalus, KC.; Hill, ; Vidal, M. WormBook. 2006. C. elegans network biology:                                               beginning; p. 1-20. <br />                                           Piano F, Schetter AJ, Mangone M, Stein L, Kemphues KJ. RNAi analysis genes expressed                                               ovary Caenorhabditis elegans. Curr Biol. 2000; 10:1619&#226;&#8364;&#8220;1622. [PubMed: 11137018] <br />                                           Piano F, Schetter AJ, Morton DG, Gunsalus KC, Reinke V, Kim SK, Kemphues KJ. Gene clustering <br />                                               based RNAi phenotypes ovary-enriched genes C. elegans. Curr Biol. 2002; 12:1959&#226;&#8364;&#8220; <br />                                               1964. [PubMed: 12445391] <br />                                           Praitis V, Casey E, Collar D, Austin J. Creation low-copy integrated transgenic lines                                               Caenorhabditis elegans. Genetics. 2001; 157:1217&#226;&#8364;&#8220;1226. [PubMed: 11238406] <br />                                           Rhrissorrakrai K, Gunsalus KC.  Module Identification NEtworks. BMC Bioinformatics.                                               press. <br />                                           Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati  Punna T, Ihmels J, Andrews B, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                               Boone C, Greenblatt JF, et al. Exploration function organization yeast early <br />                                               secretory pathway epistatic miniarray profile. Cell. 2005; 123:507&#226;&#8364;&#8220;519. [PubMed: <br />                                               16269340] <br />                                           Shi  Chen CC, Banerjee R, Glodowski D, Audhya  Rongo C, Grant BD. EHBP-1 functions                                               RAB-10 endocytic recycling Caenorhabditis elegans. Mol. Biol. Cell. 2010; 21:2930&#226;&#8364;&#8220; <br />                                               2943. <br />                                           Sijen T, Fleenor J, Simmer F, Thijssen KL, Parrish S, Timmons L, Plasterk RH,  role                                               RNA amplification dsRNA-triggered gene silencing. Cell. 2001; 107:465&#226;&#8364;&#8220;476. [PubMed: <br />                                               11719187] <br />                                           Simonis N, Rual JF, Carvunis AR, Tasan M, Lemmens  Hirozane-Kishikawa T, Hao T, Sahalie JM, <br />                                               Venkatesan K, Gebreab F, et al. Empirically controlled mapping Caenorhabditis elegans <br />                                               protein-protein interactome network. Nat Methods. 2009; 6:47&#226;&#8364;&#8220;54. [PubMed: 19123269] <br />                                           S&#195;&#182;nnichsen B, Koski LB, Walsh  Marschall P, Neumann B, Brehm M, Alleaume  Artelt J, <br />                                               Bettencourt P, Cassin E, et al. genome RNAi profiling early embryogenesis                                               Caenorhabditis elegans. Nature. 2005; 434:462&#226;&#8364;&#8220;469. [PubMed: 15791247] <br />                                           Starr DA, Hermann GJ, Malone CJ, Fixsen W, Priess JR, Horvitz HR, Han M. unc-83 encodes novel <br />                                               component nuclear envelope essential proper nuclear migration. Development. <br />                                               2001; 128:5039&#226;&#8364;&#8220;5050. [PubMed: 11748140] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Tatusov RL, Fedorova ND, Jackson JD, Jacobs AR, Kiryutin B, Koonin EV, Krylov DM, Mazumder <br />                                               R, Mekhedov SL, Nikolskaya  et al. COG database: updated version includes <br />                                               eukaryotes. BMC Bioinformatics. 2003; 4:41. [PubMed: 12969510] <br />                                           Wilmes GM, Bergkessel M, Bandyopadhyay S, Shales M, Braberg H, Cagney G, Collins SR, <br />                                               Whitworth GB, Kress TL, Weissman JS, et al. genetic interaction map RNA-processing <br />                                               factors reveals links Sem1/Dss1-containing complexes mRNA export splicing. <br />                                               Mol Cell. 2008; 32:735&#226;&#8364;&#8220;746. [PubMed: 19061648] <br />                                           Zhou K, Rolls MM, Hall DH, Malone CJ, Hanna-Rose W. ZYG-12-dynein interaction nuclear <br />                                               envelope defines cytoskeletal architecture C. elegans gonad. J Cell Biol. 2009; 186:229&#226;&#8364;&#8220; <br />                                               241. [PubMed: 19635841] <br />                                           Zipperlen P, Fraser AG, Kamath RS, Martinez-Campos M, Ahringer J. Roles 147 embryonic lethal <br />                                               genes C. elegans chromosome identified RNA interference video microscopy. EMBO <br />                                               J. 2001; 20:3984&#226;&#8364;&#8220;3992. [PubMed: 11483502] <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                             Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 1. Phenotypic profiling using high-content assay based architecture complex <br />                                           tissue <br />                                            Breakdown 554 genes sterile collection. B. Screen flow chart. C. C. <br />                                           elegans gonad complex dynamic tissue architecture depends broad spectrum <br />                                           interacting cellular processes (blue text schematic). Spinning disc confocal <br />                                           microscopy used collect 80 plane color series gonad. imaged <br />                                           region indicated (red dashed box) sample central section control gonad                                           shown. D. sample defects illustrated pairing numbered boxed regions                                           control gonad C corresponding regions gonads indicated defects. <br />                                           E. Central plane images gene knockdowns 3 phenotypic classes. Knockdowns <br />                                           Class B2 (left column), contains chaperonin complex subunits, led rounded <br />                                           compartments nuclei fell compartments (yellow arrowheads). <br />                                           Knockdowns Class F3 (middle column), contains genes implicated Rho GTPase <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                        Page 16 <br />  <br />  <br />                                           signaling, led &#226;&#8364;&#339;tubulated&#226;&#8364;? gonads clustered nuclei. Knockdowns Class C2 (right <br />                                           column), contains cell cycle regulators, led gonads compartments/ <br />                                           oocytes. Bars, 10 &#206;&#188;m. Figure S1, Table S1 Table S2. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                         Page 17 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 2. Validation manual classification                                           Class E2:  List genes Class E2. Schematic central plane images illustrate                                           class phenotype. B. Nuclei dorsal cords T09E8.1(RNAi) (yellow arrowheads), <br />                                           control, L1 worms (insets 2.4X), reflecting hypodermal cell nuclear migration <br />                                           defect. Table shows quantification nuclear migration defect. effects                                           simultaneous inhibition dynein-regulatory proteins BICD-1 NUD-2 quantified <br />                                           dataset Fridolfsson et al. (2010) shown comparison. C. effect                                           T09E8.1 knockdown microtubule arrays hypodermis (schematic) monitored <br />                                           timelapse imaging EB1-GFP fusion (Fridolfsson Starr, 2010). Bar graphs                                           mean length (left) number (right) EB1-GFP comets. Error bars SE. <br />                                           Class G1: D. Class G1 contains 3 characterized genes implicated MAPK signaling <br />                                           (gray), 1 characterized gene previously implicated MAPK signaling (orange), 1 <br />                                           uncharacterized gene (purple). Schematic central plane images illustrate class <br />                                           phenotype. E. Partial RNAi daf-21, GFP control, performed presence  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                      Page 18 <br />  <br />  <br />                                           absence weak mpk-1 loss function mutation (ga111). Gonads fixed stained <br />                                           chromosomes (DAPI, green), plasma membrane (SYN-4 PTC-1, red),                                           activated MPK-1 (Phospho-MPK-1, right panels). analysis performed NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           rrf-1(pk1417) background RNAi effective gonad,                                           surrounding somatic cells. F. Quantification phenotypes resulting partial RNAi                                           daf-21 F54D12.5 mpk-1(ga111) background  Table S3). * Percentages                                           include 21% germlines showed severe MPK-1 &#226;&#8364;&#339;null&#226;&#8364;? phenotype. G. Schematic <br />                                           places DAF-21 F54D12.5 MAPK signaling pathway. Error bars SE. Bars, <br />                                           10 &#206;&#188;m. Figure S2, Table S2 Table S3 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                   Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                            Page 19 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 3. Validation manual classification II <br />                                           Class S2:  List genes Class S2. Schematic central plane images gonads <br />                                           (left) adjacent embryos (right) illustrate class phenotype. B. List relevant proteins <br />                                           identified mass spectrometry immuno-affinity purification K10D2.4 worm <br />                                           extracts, percent coverage. Proteins encoded uncharacterized Class S2 <br />                                           genes (purple) listed, APC components Class S2 (gray) additional <br />                                           APC components screen (black). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Class F2: C. List genes Class F2. Schematic central plane images illustrate                                           class phenotype. D. Embryo expressing GFP-C08C3.4 (green) mCherry tagged <br />                                           plasma membrane probe (red) cell division. E. Central plane images                                           control C08C3.4(RNAi) embryos expressing mCherry-histone H2B (green) GFP <br />                                           plasma membrane probe (red). Multiple nuclei cell C08C3.4(RNAi) embryo <br />                                           cytokinesis failure. <br />                                           Class I2: F. Schematic central plane images illustrate Class I2 phenotype,                                           includes punctate debris containing plasma membrane probe (arrow schematic). G. <br />                                           Schematic trafficking assay. List Class I2 genes defects SNB-1-GFP <br />                                           trafficking assay. Images assay 1 characterized 2 uncharacterized Class I2 <br />                                           genes. Bars 10&#206;&#188;m F 5 &#206;&#188;m G. Table S2. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                            Page 20 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 4. Constructing gene networks using Connection Specificity Index (CSI) instead                                           PCC reduces connection noise allows connections similar functional significance                                           viewed entire network uniform threshold <br />                                            Flowchart steps used construct gene networks B. B. Gene networks                                           constructed displaying connections (dark blue lines) genes knockdown <br />                                           profiles correlated PCC &#226;&#8240;&#165; specified thresholds (0.46,  0.60, middle; <br />                                           0.70, . column shows network region, labeled based primary function <br />                                           genes region. compare computational network manually-defined <br />                                           phenotypic classes, gene groups manually-assigned classes circled labeled. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           optimal PCC threshold significant connections displayed non- <br />                                           specific connections filtered (red boxes) different different network <br />                                           regions. C. Method used calculate CSI. D. Gene network showing regions                                           B constructed displaying connections (light blue lines) CSI &#226;&#8240;&#165; 0.97. E. Bar graph <br />                                           showing number connections gene networks constructed using PCC CSI <br />                                           thresholds 0.55 0.96, respectively. thresholds chosen                                           highest thresholds retain genes network; genes drop                                           network longer make connections PCC/CSI exceeds                                           specified threshold. F. Bar graph showing percent 49 manually-defined <br />                                           phenotypic classes containing 4 genes identified automated clustering <br />                                           algorithm   networks constructed using PCC CSI thresholds 0.55 0.96, <br />                                           respectively. G. Heatmap dendograms sterile gene set constructed based PCC <br />                                           CSI. Figure S3 Table S4. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                    Cell. Author manuscript; available PMC 2012 April 29. <br />                             Green et al.                                                                                              Page 21 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Figure 5. Varying CSI threshold reveals functional modularity gene network                                           different levels resolution <br />                                            region gene network involved protein production shown using                                           different CSI thresholds filter displaye <br /> </body></html>