<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Orth Palsson BMC Systems Biology 2012, 6:30 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                                 Open Access <br />  <br /> Gap-filling analysis iJO1366 Escherichia coli <br /> metabolic network reconstruction discovery metabolic functions <br /> Jeffrey D Orth Bernhard   Palsson* <br />  <br />  <br />   Abstract <br />   Background: iJO1366 reconstruction metabolic network Escherichia coli complete <br />   accurate metabolic reconstructions available organism.  knowledge  <br />   studied model organisms incomplete, network reconstruction contains gaps possible <br />   errors. total 208 blocked metabolites iJO1366, representing gaps network. <br />   Results: new model improvement workflow developed compare model based phenotypic predictions   experimental data gaps correct errors. Keio Collection based dataset E. coli gene essentiality   obtained literature data compared model predictions. SMILEY algorithm used predict <br />   likely missing reactions reconstructed network, adding reactions KEGG based universal set   metabolic reactions. feasibility putative reactions determined comparing updated versions   model experimental dataset, genes predicted feasible reactions. <br />   Conclusions: Numerous improvements iJO1366 metabolic reconstruction suggested analyses. <br />   Experiments performed verify computational predictions, including new mechanism growth <br />   <span id='am-209' about='obo:IMR_0200177' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-210' property="oboInOwl:hasDbXref" content="KEGG:C00137" datatype="xsd:string"></span><span id='am-211' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-212' property="rdfs:label" content="myo-Inositol" datatype="xsd:string"></span><span id='am-213' property="oboInOwl:id" content="IMR:0200177" datatype="xsd:string"></span>myo-inositol</span>. predictions study experimentally verifiable similar means. <br />   Validating predictions represents substantial important undertaking. <br />   Keywords: Constraint-based modeling, Metabolic network reconstruction, Escherichia coli, Gap-filling, Gene <br />   annotation <br />  <br />  <br /> Background                                                                              model missing essential reaction [5]. pre- <br /> Constraint-based modeling widely used systems biology                              diciton called false negative. current metabolic net- <br /> method particularly suited predicting                               work reconstruction entirely complete realistic <br /> phenotypes microbial organisms gene knockouts                               knowledge metabolism organism grown different substrates [1-3]. variable                                complete. studied model organisms <br /> conditions simply represented additional constraints                             Escherichia coli genes  model, growth predicted flux balance                                 known functions [6,7]. result analysis (FBA) [4]. realistic constraint                           gaps metabolic network reconstructions. gaps represented typical metabolic model, quite possible                          form dead-end metabolites, model predict growth conditions                               producing consuming reactions [8]. <br /> growth does really occur. actual organism                                 different types gaps exist reconstructed <br /> express required gene growth, fluxes limited                            metabolic networks [8,9]. gaps result blocked kinetic thermodynamic constraints, example.                              reactions, unable carry flux steady state, <br /> case called false positive prediction. hand,                          blocked metabolites, exist blocked reac- <br /> false predictions growth taken indications                              tions produced consumed. Root  <br />                                                                                         production gaps metabolites consuming <br /> * Correspondence: palsson@ucsd.edu                                                      reactions blocked producing <br /> Department Bioengineering, University California, San Diego, La Jolla, <br /> CA, USA, 9500 Gilman Drive, Mail Code 0412, La Jolla CA 92093-0412, USA                 reactions. Metabolites produced  <br />                                           &#194;&#169; 2012 Orth Palsson; licensee BioMed Central  Open Access article distributed terms                                           Creative Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, <br />                                           distribution, reproduction medium, provided original work properly cited. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                    Page 2 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> root production metabolites blocked,       organism specific databases BioCyc [17]. program <br /> referred downstream gaps. Likewise, root          fills gaps complete metabolic pathways includes <br /> sumption gaps metabolites producing reactions         hole-filling algorithm assigns genes gap-filling reac- consuming reactions, metabolites          tions [18,19]. recent procedure uses network ex- <br /> blocked gaps called upstream gaps. gaps       pansion determine minimum number reactions metabolic network classified        need added network make compliant <br /> scope gaps knowledge gaps. Scope gaps        experimental data [20]. production metabolites <br /> exist scope metabolic network models       macromolecule degradation products considered, <br /> does include features like macromolecular degradation      genes predicted using hidden Markov models. use charged tRNAs protein synthesis. Know-        strategy applied improve metabolic models ledge gaps, hand, actually result      E. coli [21] Chlamydomonas reinhardtii [22]. incomplete knowledge metabolism              present study builds methods new <br /> ganism [10].                                                   workflow includes use SMILEY algorithm. <br />    comparison model predictions experimental         SMILEY used instead GapFill GrowMatch  <br /> data useful way network gaps dis-         cause modified make predictions cover new genes reactions. possible         wider range experimental data originally <br /> outcomes comparing computationally predicted           applied  Specifically, used make predictions <br /> experimentally measured growth phenotypes: true posi-          using gene essentiality data network gaps addition <br /> tives, model correctly predicts growth; true          data growth different substrates. iJO1366 <br /> negatives, model correctly predicts           metabolic network reconstruction E. coli K-12, lat- <br /> growth possible; false positives, model pre-       est complete genome-scale reconstruction dicts growth condition growth            organism [10], used analysis. begin, observed; false negatives, model fails         large dataset E. coli gene essentiality Keio <br /> predict growth growth experimentally                 Collection [23], combined published datasets <br /> observed. false positive false negative results       [10,23-25], assembled.  model growth predic- useful refining model content,        tions using iJO1366 model compared false negative cases help gaps. meth-    dataset, false positive false negative <br /> ods developed predict correct gap        comparisons analyzed identify potential errors ing reactions based comparisons experimental             model experimental datasets. data.                                                          SMILEY algorithm used predict gap-filling <br />    method published called            reactions reactions correct false negative model <br /> SMILEY [5]. mixed-integer linear programming         predictions. feasibility reactions algorithm identifies minimum number reac-          assessed comparing augmented model predictions tions need added metabolic model        experimental dataset. Finally, genes predicted <br /> universal database reactions order allow mini-      feasible putative reactions. sets mum defined growth rate achieved. SMILEY             gene function predictions presented, provide <br /> algorithm developed used predict reac-        plausible hypotheses experimental validation. tions missing iJR904 E. coli reconstruction [11]      predictions potential improve metabolic caused false negative model growth predictions            reconstruction lead new metabolic gene discover- compared Biolog growth data [12].              ies [8]. Knockout strain growth phenotyping experiments <br /> results experimentally verified new genes        performed identify gene involved myo-inosi- <br /> characterized [5]. SMILEY recently used pre-       tol metabolism, demonstrating types experimental <br /> dict gap-filling reactions Recon 1 human metabolic      analyses validate biological predictions. <br /> reconstruction [13,14]. algorithms GapFind/GapFill [9] GrowMatch [15] later developed, pre-        Results <br /> dict missing reactions connecting model gaps         Comparison model predictions experimental data <br /> comparing model predictions gene essentiality data,      applying developed workflow analyze iJO1366 <br /> spectively. <span id='am-15' about='oboInOwl:date' typeof='owl:Thing'>date</span>, methods used make      model gaps compare model predicted phenotypes predictions E. coli yeast metabolic networks       experimental data, new biological hypotheses gener- <br /> [15,16], predictions experimen-    ated (Figure 1).  experimental datasets tally verified. Non-constraint-based methods recon-        assembled combined. dataset consisted large <br /> structing metabolic networks filling gaps   set E. coli gene knockout strains grown different types <br /> developed. example PathoLogic, component       media. gene knockout strains Pathway Tools software used assemble      Keio Collection E. coli BW25113 single gene knockouts, <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                Page 3 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />     Baba et al.        Joyce et al.          Orth et al.           Ito et al. <br />       Entire Keio        Entire Keio         1075 genes       1440 genes      Collection      Collection       glucose, lactate,     Biolog GN2 <br />        glucose            glycerol           succinate           plates <br />  <br />  <br />  <br />  <br />      JO1366 E.                  Combined                KEGG universal <br />     coli metabolic                phenotypic                 reaction <br />                                    dataset                  database <br />          model                                              8447 reactions <br />                                 13,470 phenotypes <br />  <br />  <br />  <br />  <br />                                         711 false <br />             Compared model              positives <br />               predictions             experimental data                                                                           Computed <br />                                         265 false                                                                             Feasible <br />                                                                                                        feasibility                                         negatives                                                                             solutions <br />                                                                                                      SMILEY solutions <br />                                                           Used SMILEY               1176 <br />                                                           predict gap-filling        SMILEY <br />                                                               reactions              solutions <br />              Used GapFind              208 <br />                                                                                                      Predicted vivo <br />              identify network           network                                                                                vivo <br />                                                                                                       experiments         experiments <br />                    gaps                  gaps <br />                                                                                                      confirm reactions <br />  <br />  <br />  <br />  Figure 1 Workflow predicting FN-correcting gap-filling reactions using SMILEY. Input datasets shown blue, computational <br />  prediction steps orange, analyzed outputs green. <br />  <br />  <br />  <br /> allowing analyzed  dataset,                    259 1440 genes iJO1366, Baba et al. [23], contained phenotypes entire                  genes included. 38 substrates Keio Collection grown glucose MOPS minimal media.                         included iJO1366 model KEGG com- defined media contains buffer MOPS (3-(N-mor-                       pound database, included pholino)propanesulfonic acid), potential sulfur source.                    connected model content using second dataset growth screen entire                   methods presented  phenotypes combined <br /> Keio Collection, glycerol M9 minimal media [25].                      dataset adjusted slightly original publica- dataset screen 1075 Keio Collection                       tions based recent analysis Keio Collection <br /> strains, genes included iAF1260 E. coli meta-                 genotypes [26]. new genes classified essen- <br /> bolic reconstruction [21], grown different media                     tial, added essential genes glucose <br /> conditions [10]. strains grown glucose M9                        glycerol. gene, b0103, removed Bio- <br /> media aerobic anaerobic conditions, lac-                   log screen data based analysis. screen 1075 <br /> tate M9 aerobically, succinate M9 aerobically.                    strains conditions performed Keio <br /> fourth dataset consisted phenotypes 1440 Keio Col-                   Collection update, accounted lection strains grown Biolog GN2 plates [24].                      changes. datasets contained phenotypes wild-type E. coli grow 38 different car-                 substrates. example, Biolog data contained <br /> bon sources Biolog plate, dataset                        strains grown glycerol, succinate, lactate. included 38 substrates.                                                cases, data point included gene <br />    datasets combined large                   knockout strain grown substrate. phenotypic dataset. screens entire Keio                      datasets included &#226;&#8364;&#339;growth&#226;&#8364;? phenotype, phenotype <br /> Collection glucose glycerol, growth phenotype                   set &#226;&#8364;&#339;growth&#226;&#8364;? combined dataset. strain <br /> included 1366 genes iJO1366.                       growth&#226;&#8364;? phenotype datasets classified <br /> screen conditions, phenotypes available                     essential combined dataset. making 1075 1366 genes. screen Biolog plates,                     adjustments, final combined dataset contained 13,470 <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                       Page 4 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> experimental phenotypes. 12,120 &#226;&#8364;&#339;growth&#226;&#8364;? phe-           &#226;&#8364;&#339;Cofactor Prosthetic Group Metabolism,&#226;&#8364;? 7.4 % notypes 1350  growth&#226;&#8364;? phenotypes.                           7.7 % respectively. false negative cases indicate    iJO1366 E. coli metabolic network model            likely presence currently unknown isozymes alterna- <br /> used predict growth phenotypes 13,470 condi-          tive pathways. <br /> tions. model E. coli K-12 MG1655 metabolism modified slightly match genotype E. coli           False positive model predictions <br /> BW25113, parent strain Keio Collection. FBA [4]         set false positive predictions investigated used predict growth rates using iJO1366            determine occurred. gene <br /> core biomass objective gene knockout              false positive prediction sub- substrate experimental dataset. growth rate       strate experimental growth substrate zero classified computational &#226;&#8364;&#339;growth&#226;&#8364;?              tested. possible <br /> phenotype, growth rate zero classified        reasons false positive prediction growth&#226;&#8364;?. silico dataset 13,470 phenotypes        iJO1366 model.  possible model generated, compared vivo dataset.           contain error unrealistic reaction (Table 1). <br /> model prediction classified true positive, true    model, reaction CBPS (<span id='am-295' about='obo:IMR_0200109' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-296' property="oboInOwl:hasDbXref" content="KEGG:C00169" datatype="xsd:string"></span><span id='am-297' property="rdfs:label" content="Carbamoyl phosphate" datatype="xsd:string"></span><span id='am-298' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-299' property="oboInOwl:id" content="IMR:0200109" datatype="xsd:string"></span>carbamoyl phosphate</span> syn- <br /> negative, false positive, false negative. Additional file   thase (glutamine-hydrolyzing)) converts <span id='am-76' about='obo:IMR_0200448' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-77' property="rdfs:label" content="L-Glutamine" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasDbXref" content="KEGG:C00064" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:id" content="IMR:0200448" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-glutamine</span> car- <br /> 1 complete sets computational experimental          bamoyl phosphate, essential precursor <span id='am-187' about='obo:IMR_0200428' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-188' property="oboInOwl:id" content="IMR:0200428" datatype="xsd:string"></span><span id='am-189' property="rdfs:label" content="L-Arginine" datatype="xsd:string"></span><span id='am-190' property="oboInOwl:hasDbXref" content="KEGG:C00062" datatype="xsd:string"></span><span id='am-191' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-arginine</span>. <br /> phenotypes. total 11,855 true positives, 639 true nega-       reaction catalyzed complex carA (b0032) tives, 711 false positives, 265 false negatives identi-   carB (b0033), experimentally essen- <br /> fied (Figure 2 . Matthews Correlation Coefficient            tial glucose, glycerol, succinate, lactate minimal <br /> (MCC) predictions, measure accuracy           media. model, genes non-essential binary classifications, 0.5418. Overall, silico         alternate reaction produces carbamoyl phosphate, <br /> screen predicted growth phenotypes            CBMKr (carbamate kinase), catalyzed products experimental data (93.3 % 90.0 %, respectively).        yahI (b0323), arcC (b0521), yqeA (b2874). putative result largely explained nature          reaction included iJO1366 based physiological <br /> straint-based modeling FBA. iJO1366                data [28], functions genes model does contain regulation, FBA use reac-           characterized. likely CBMKr reaction <br /> tion network produce biomass. E. coli cell,        unrealistic. False positives caused errors different levels regulation make certain enzymes         iJO1366 core biomass reaction. gene pdxH (b1638) <br /> available certain conditions,          catalyzes reactions PDX5POi (<span id='am-71' about='obo:IMR_0200344' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-72' property="rdfs:label" content="Pyridoxine" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:id" content="IMR:0200344" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasDbXref" content="KEGG:C00314" datatype="xsd:string"></span>pyridoxine</span> 5&#226;&#8364;&#8482;-phosphate <br /> allowed growth. real constraints, kinetic        oxidase) PYAM5PO (<span id='am-270' about='obo:IMR_0200382' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-271' property="rdfs:label" content="Pyridoxamine" datatype="xsd:string"></span><span id='am-272' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-273' property="oboInOwl:id" content="IMR:0200382" datatype="xsd:string"></span><span id='am-274' property="oboInOwl:hasDbXref" content="KEGG:C00534" datatype="xsd:string"></span>pyridoxamine</span> 5&#226;&#8364;&#8482;-phosphate oxi- thermodynamic constraints [27], accounted            dase), required synthesis <span id='am-5' about='obo:IMR_0200504' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00250" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:id" content="IMR:0200504" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Pyridoxal" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyridoxal</span> 5&#226;&#8364;&#8482;-phosphate model cause false positive       (<span id='am-22' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-23' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>vitamin</span> B6). vitamin included core bio- <br /> predictions.                                                       mass, reactions essential model. <br />    genes iJO1366 model classified         essentiality gene glucose <span id='am-214' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-215' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-216' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-217' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-218' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-219' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-220' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-221' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-222' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-223' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-224' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-225' property="rdfs:label" content="glycine" datatype="xsd:string"></span>gly</span>- <br /> 11 functional categories, according metabolic func-         cerol minimal media indicates vitamin B6 fact es- <br /> tions serve [10]. different categories      sential E. coli, included model <br /> contain genes varying levels predictive accuracy           biomass reaction. <br /> (Figure 2 b). Genes  ? category, including               false positive predictions likely occurred mainly tRNA charging genes genes             gene incorrectly identified essential placed categories, lead false        experimental screens (Table 2). case positives 23.7 % cases. known tRNA        genes involved energy production. cyto- <br /> charging gaps iJO1366 model [10]. tRNA char-          chrome oxidase gene cydA (b0733) knockout strain does <br /> ging genes essential vivo. false       exist Keio Collection, presumed positives &#226;&#8364;&#339;Energy Production Conversion&#226;&#8364;?             essential. viable knockout strain gene genes (12.4 %). outcome partly caused miss-         produced,  knockouts ing thermodynamic constraints, partly fact         cytochrome oxidases [29]. <span id='am-101' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-102' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-104' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-110' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-111' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span>ATP</span> synthase genes <br /> disruptions cellular energy generation cause E. coli     atpCDGAHFEB (b3731-8) classified essential grow slowly, strains            minimal media, inspection actual growth mea- essential experimental screens             surements experiments [10,23,25] reveals actually slowly growing. computa-             knockout strains did actually grow, albeit slowly. <br /> tional screen classified growing strains non-essential,        false positive cases occurred gene knockout grew slowly. False negatives com-           strains known isozymes alternative pathways <br /> mon genes &#226;&#8364;&#339;<span id='am-140' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-141' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-145' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-146' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span>Amino Acid</span> Metabolism&#226;&#8364;?                 (Table 3). iJO1366 model, knockouts  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                                                                  Page 5 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                      Experimental <br />                                                                                                                  Growth        growth <br />                                                                                                                   12,120          1350 <br />                                                                                                                  (90.0%)        (10.0%) <br />  <br />  <br />                                                                                                12,566             11,855          711 <br />                                                            Computational <br />                                                                                Growth <br />                                                                                               (93.3%)            (88.0%)        (5.3%) <br />  <br />  <br />  <br />  <br />                                                                                growth        904                265                639 <br />                                                                                               (6.7%)             (2.0%)             (4.7%) <br />  <br />  <br />                                                                                                                        13,470 Total <br />  <br />  <br />  <br />                      b <br />                                                                                 True Positives                                                         False Positives <br />                                                  100                                                                           25 <br />                        Percent wtihin category <br />  <br />  <br />  <br />  <br />                                                  80                                                                            20 <br />  <br />                                                  60                                                                            15 <br />  <br />                                                  40                                                                            10 <br />  <br />                                                  20                                                                             5 <br />  <br />                                                    0                                                                            0 <br />                                                          C               W    P   E        L    N        O   TI                     C   W      P     E        L    N    O   TI    <br />  <br />  <br />  <br />                                                                                False Negatives                                                         True Negatives <br />                                                   10                                                                           20 <br />                      Percent wtihin category <br />  <br />  <br />  <br />  <br />                                                    8                                                                           16 <br />  <br />                                                    6                                                                           12 <br />  <br />                                                    4                                                                            8 <br />  <br />                                                    2                                                                            4 <br />  <br />                                                    0                                                                            0 <br />                                                          C               W    P   E         L   N        O   TI                  C      W      P     E       L     N    O   TI    <br />                                                                            Metabolic functional categories                                           Metabolic functional categories <br />  <br />  <br />  Figure 2 Comparison model predicted growth phenotypes experimental data.   overall comparison, indicating numbers true <br />  positives, true negatives, false positives, false negatives. (b) numbers type prediction 11 functional categories  metabolic reactions. categories  amino acid metabolism  , <span id='am-27' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-33' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span>carbohydrate</span> metabolism (C), cell wall/membrane/envelope metabolism (W), <br />  cofactor prosthetic group metabolism (P), energy production conversion (E), inorganic <span id='am-34' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-35' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ion</span> transport metabolism  , <span id='am-16' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-17' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-18' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>lipid</span> metabolism <br />  (L), <span id='am-127' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-128' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-132' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-133' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-134' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> metabolism (N), (O), inner membrane transport (TI), outer membrane transport  . <br />  <br />  <br /> overcome using isozyme alternative pathway                                                                    reactions blocked scope gaps, important reac- <br /> synthesize biomass components. vivo, genes                                                                    tions used model. Finally, false essential isozyme genes expressed                                                                 positives explained model  ex- experimental conditions, capable                                                                 ample, gene spoT (b3650) required synthesize catalyzing reaction sufficient rate growth                                                               signaling molecule <span id='am-81' about='obo:IMR_0200445' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:id" content="IMR:0200445" datatype="xsd:string"></span><span id='am-84' property="rdfs:label" content="guanosine" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00387" datatype="xsd:string"></span>guanosine</span> tetraphosphate (ppGpp). occur. types false positive model predictions                                                                  metabolic model does require signaling, overcome standard FBA using meta-                                                                      gene non-essential. Experimentally, spoT bolic model. model including regulation add-                                                                    essential rich media, likely non- <br /> itional constraints required. false positives                                                                 metabolic function. false negatives occur tRNA charging genes knocked                                                                      explained model ftsI (b0084), adk (b0474), mrdA <br /> model (Table 4). iJO1366 tRNA charging                                                                           (b0635), cydC (b0886), gapA (b1779), ligA (b2411), suhB <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                  Page 6 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 1 False positive model predictions indicate                   indication missing metabolic reactions pathways model errors                                                             model. section, use SMILEY algorithm <br /> Gene           Error                                                     predict likely missing reactions presented. carA (b0032)   alternate pathway (CBMKr) gene functions              possible explanations false negative predic- <br />                confirmed                                                 tions.  possible model biomass reaction <br /> carB (b0033)   alternate pathway (CBMKr) gene functions              used objective incorrect (Table 5).                confirmed                                                 false negative cases occurred knockouts genes <br /> proB (b0242)   alternate pathway (NACODA) gene function              involved molybdenum cofactor synthesis, including mog <br />                confirmed                                                 (b0009), moaA (b0781), moaC (b0783), moaD (b0784), <br /> proA (b0243)   alternate pathway (NACODA) gene function              moaE (b0785), moeA (b0826), moeB (b0827), mobA <br />                confirmed <br />                                                                          (b3857). iJO1366 model, genes essential <br /> folD (b0529)   5fthf[c] methf[c] essential                    required produce bmocogdp[c] (bis- <br /> entD (b0583)   enter[c] essential                                 molybdopterin <span id='am-260' about='obo:IMR_0200319' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-261' property="oboInOwl:id" content="IMR:0200319" datatype="xsd:string"></span><span id='am-262' property="rdfs:label" content="Guanine" datatype="xsd:string"></span><span id='am-263' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-264' property="oboInOwl:hasDbXref" content="KEGG:C00242" datatype="xsd:string"></span>guanine</span> dinucleotide), component pyrD (b0945)   alternate pathway (DHORDfum) orphan reaction        core biomass formulation. gene knockout <br /> pdxH (b1638)   pydx5p[c] essential                                strains experimentally viable conditions,                                                                          likely cofactor essential growth, pgsA (b1912)   pgp120[p] - pgp181[p] essential <br /> nrdA (b2234)   alternate pathway (RNDR1b &#226;&#8364;&#8220; RNDR4b) gene functions <br />                confirmed <br /> nrdB (b2235)   alternate pathway (RNDR1b &#226;&#8364;&#8220; RNDR4b) gene functions <br />                                                                          Table 3 False positive model predictions caused                confirmed                                             isozymes alternate pathways <br /> ptsI (b2416)   alternate pathway (GLCt2pp) glucose transport         Gene           Isozyme alternate pathway reactions <br />                confirmed                                                 thrA (b0002)   metL (b3940) lysC (b4024) <br /> waaK (b3623)   colipa[e] essential                                carA (b0032)   alternate reaction: CBMKr <br /> wzyE (b3793)   eca4colipa[e] essential                            carB (b0033)   alternate reaction: CBMKr <br /> ubiE (b3833)   reactions AMMQLT8 OMBZLM blocked gaps          folA (b0048)   folM (b1606) <br /> ubiB (b3835)   alternate pathway (OPXHH3) orphan reaction          (b0126)    cynT (b0339) <br /> ppa (b4226)    isozymes, ppx (b2502) surE (b2744),            pyrH (b0171)   cmk (b0910) <br />                incorrect <br />                                                                          int (b0657)    lpp (b1677) <br />                                                                          fldA (b0684)   fldB (b2895) <br /> (b2533), eno (b2779), fbaA (b2925), pgk (b2926), dut                     fabA (b0954)   fabZ (b0180) <br /> (b3640), pslB (b4041), alsK (b4084). <br />                                                                          nrdA (b2234)   alternate reactions: RNDR1b, RNDR2b, RNDR3b, RNDR4b <br />  <br /> False negative model predictions                                         nrdB (b2235)   alternate reactions: RNDR1b, RNDR2b, RNDR3b, RNDR4b genes false negative predictions               cysK (b2414)   cysM (b2421) <br /> substrate computationally predicted growth                 ptsI (b2416)   alternate reaction: GLCt2pp <br /> substrate investigated  constraint-             cysA (b2422)   modA (b0763) + modB (b0764) + modC (b0765) <br /> based metabolic model fails predict growth                cysP (b2425)   modA (b0763) + modB (b0764) + modC (b0765) <br /> dition growth observed experimentally,                                                                          guaB (b2508)   alternate reaction: XPPT <br />                                                                          glyA (b2551)   alternate reaction: GLYCL <br /> Table 2 False positive model predictions indicate <br /> incorrectly identified essential genes                                   acpS (b2563)   acpT (b3475) <br /> Gene                   Reason incorrect phenotype                    serA (b2913)   alternate reaction: GHMT2r <br /> cydA (b0733)           knocked successfully Portnoy et al. [29]   metC (b3008)   tnaA (b3708) malY (b1622) <br /> atpC (b3731)           ATP synthase knockout causes low growth rate      aroE (b3281)   ydiB (b1692) <br /> atpD (b3732)           ATP synthase knockout causes low growth rate      ilvA (b3772)   tdcB (b3117) <br /> atpG (b3733)           ATP synthase knockout causes low growth rate      metE (b3829)   metH (b4019) <br /> atpA (b3734)           ATP synthase knockout causes low growth rate      ubiB (b3835)   alternate reaction: OPHHX3 <br /> atpH (b3735)           ATP synthase knockout causes low growth rate      glnA (b3870)   ycjK (b1297) <br /> atpF (b3736)           ATP synthase knockout causes low growth rate      metL (b3940)   thrL (b0002) malY (b1622) <br /> atpE (b3737)           ATP synthase knockout causes low growth rate      ppa (b4226)    ppx (b2502) surE (b2744) <br /> atpB (b3738)           ATP synthase knockout causes low growth rate      serB (b4388)   alternate reaction: GHMT2r <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                              Page 7 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 4 False positive model predictions caused tRNA             reaction essential histidine synthesis <br /> charging reactions                                                  pathway, essential minimal media Gene                                              Amino Acid        model. vivo datasets,  hisH ileS (b0026)                                      <span id='am-55' about='obo:IMR_0200272' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-56' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:hasDbXref" content="KEGG:C00407" datatype="xsd:string"></span><span id='am-58' property="rdfs:label" content="L-Isoleucine" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:id" content="IMR:0200272" datatype="xsd:string"></span>L-isoleucine</span>      essential aerobic conditions. essential <br /> proS (b0194)                                      <span id='am-182' about='obo:IMR_0200183' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-183' property="rdfs:label" content="L-Proline" datatype="xsd:string"></span><span id='am-184' property="oboInOwl:id" content="IMR:0200183" datatype="xsd:string"></span><span id='am-185' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-186' property="oboInOwl:hasDbXref" content="KEGG:C00148" datatype="xsd:string"></span>L-proline</span>         HisH, HisF able catalyze                                                                     reaction, using <span id='am-65' about='obo:IMR_0200075' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-66' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="NH3" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasDbXref" content="KEGG:C00014" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:id" content="IMR:0200075" datatype="xsd:string"></span>NH3</span> instead glutamine N donor <br /> cysS (b0526)                                      <span id='am-45' about='obo:IMR_0200467' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00097" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200467" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="L-Cysteine" datatype="xsd:string"></span>L-cysteine</span> <br />                                                                     [30]. hisH essential compo- <br /> leuS (b0642)                                      <span id='am-290' about='obo:IMR_0200492' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-291' property="oboInOwl:id" content="IMR:0200492" datatype="xsd:string"></span><span id='am-292' property="rdfs:label" content="L-Leucine" datatype="xsd:string"></span><span id='am-293' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-294' property="oboInOwl:hasDbXref" content="KEGG:C00123" datatype="xsd:string"></span>L-leucine</span> <br />                                                                     nent IG3PS GPR. GPR change sug- <br /> glnS (b0680)                                      L-glutamine       gested cyaY (b3807), gene involved serS (b0893)                                      <span id='am-135' about='obo:IMR_0200453' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-136' property="rdfs:label" content="L-Serine" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:id" content="IMR:0200453" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-139' property="oboInOwl:hasDbXref" content="KEGG:C00065" datatype="xsd:string"></span>L-serine</span>          transferring <span id='am-250' about='obo:IMR_0200435' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-251' property="oboInOwl:id" content="IMR:0200435" datatype="xsd:string"></span><span id='am-252' property="oboInOwl:hasDbXref" content="KEGG:C00023" datatype="xsd:string"></span><span id='am-253' property="rdfs:label" content="Iron" datatype="xsd:string"></span><span id='am-254' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>iron</span> [Fe-S] cluster synthesis. asnS (b0930)                                      <span id='am-275' about='obo:IMR_0200113' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-276' property="rdfs:label" content="L-Asparagine" datatype="xsd:string"></span><span id='am-277' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-278' property="oboInOwl:hasDbXref" content="KEGG:C00152" datatype="xsd:string"></span><span id='am-279' property="oboInOwl:id" content="IMR:0200113" datatype="xsd:string"></span>L-asparagine</span>      model, gene essential component reac- <br /> tyrS (b1637)                                      <span id='am-265' about='obo:IMR_0200240' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-266' property="oboInOwl:id" content="IMR:0200240" datatype="xsd:string"></span><span id='am-267' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-268' property="rdfs:label" content="L-Tyrosine" datatype="xsd:string"></span><span id='am-269' property="oboInOwl:hasDbXref" content="KEGG:C00082" datatype="xsd:string"></span>L-tyrosine</span>        tions ISC SUF [Fe-S] cluster synthesis <br />                                                                     pathways, essential conditions. gene <br /> pheT (b1713)                                      <span id='am-60' about='obo:IMR_0200274' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-61' property="oboInOwl:hasDbXref" content="KEGG:C00079" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="L-Phenylalanine" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0200274" datatype="xsd:string"></span>L-phenylalanine</span> <br />                                                                     characterized, experimen- <br /> pheS (b1714)                                      L-phenylalanine <br />                                                                     tally non-essential, likely strictly required thrS (b1719)                                      <span id='am-91' about='obo:IMR_0200154' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-92' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="L-Threonine" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:hasDbXref" content="KEGG:C00188" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:id" content="IMR:0200154" datatype="xsd:string"></span>L-threonine</span>       reactions I2FE2SS, I2FE2SS2, S2FE2SS, 2FE2SS2. <br /> aspS (b1866)                                      L-aspartate       false positive cases likely experimental <br /> argS (b1876)                                      L-arginine        errors (Table 7). genes involved synthesis <br /> metG (b2114)                                      <span id='am-280' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-281' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-282' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-283' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-284' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span>L-methionine</span>      cofactors biotin thiamin experimentally <br />                                                                     classified non-essential. cofactors known hisS (b2514)                                      <span id='am-50' about='obo:IMR_0200137' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-51' property="oboInOwl:id" content="IMR:0200137" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="L-Histidine" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:hasDbXref" content="KEGG:C00135" datatype="xsd:string"></span>L-histidine</span> <br />                                                                     required small quantities [31-33], likely alaS (b2697)                                      <span id='am-96' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span><span id='am-100' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span>L-alanine</span> <br />                                                                     residual biotin thiamin media dur- <br /> fmt (b3288)                                       N-formyl-L-       ing growth experiments. experimental screen                                                   methionine <br />                                                                     different conditions, thorough washing proce- <br /> trpS (b3384)                                      <span id='am-255' about='obo:IMR_0200307' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-256' property="oboInOwl:id" content="IMR:0200307" datatype="xsd:string"></span><span id='am-257' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-258' property="rdfs:label" content="L-Tryptophan" datatype="xsd:string"></span><span id='am-259' property="oboInOwl:hasDbXref" content="KEGG:C00078" datatype="xsd:string"></span>L-tryptophan</span>      dures used prevent carryover preculture <br /> glyS (b3559)                                      <span id='am-226' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-227' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-228' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-229' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-230' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-231' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-232' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-233' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-234' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-235' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-236' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-237' property="rdfs:label" content="glycine" datatype="xsd:string"></span>glycine</span>           media, genes classified essential. Fi- <br /> glyQ (b3560)                                      <span id='am-238' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-239' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-240' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-241' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-242' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-243' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-244' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-245' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-246' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-247' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-248' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-249' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span>           nally, false negatives caused currently unidenti- <br /> valS (b4258)                                      <span id='am-10' about='obo:IMR_0200501' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-11' property="rdfs:label" content="L-Valine" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0200501" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00183" datatype="xsd:string"></span>L-valine</span>          fied isozymes (Table 8). cases false <br />                                                                     negatives explained means, included iJO1366 core biomass             BLASTp used identify possible isozymes E. <br /> reaction.                                                           coli genome. predicted isozyme ex- <br />   false positive cases explained incor-             perimentally verified. prpC (b0333), currently rect gene-protein-reaction associations (GPRs)                   model associated MCITS (2-methylcitrate <br /> iJO1366 (Table 6).  gene hisH (b2023)                 synthase), confirmed isozyme required reaction IG3PS (Imidazole-glycerol-3-              gltA (b0720), catalyzing CS (citrate synthase) [34,35]. <br /> phosphate synthase), hisF (b2025).                                                                     Computational prediction gap-filling reactions <br /> Table 5 False negative model predictions caused                  cause model gaps false negative phenotypic <br /> incorrect core biomass composition                                  predictions realistic reactions miss- <br /> Gene                                          Biomass component     ing iJO1366 model. current version mog (b0009)                                   bmocogdp[c]           iJO1366 contains 48 root production gaps, 63 root <br /> moaA (b0781)                                  bmocogdp[c]           consumption gaps, 52 downstream gaps, 69  <br /> moaC (b0783)                                  bmocogdp[c]           stream gaps. scope gaps, caused                                                                     limited scope metabolic network, moaD (b0784)                                  bmocogdp[c] <br />                                                                     previously identified [8,10]. SMILEY algo- <br /> moaE (b0785)                                  bmocogdp[c] <br />                                                                     rithm used predict likely sets reac- <br /> moeA (b0826)                                  bmocogdp[c]           tions missing model. predict false negative <br /> moeB (b0827)                                  bmocogdp[c] <br /> ubiX (b2311)                                  2ohph[c] <br />                                                                     Table 6 False negative model predictions suggest <br /> iscS (b2530)                                  bmocogdp[c]           changes iJO1366 model GPRs <br /> cysG (b3368)                                  sheme[c]              Gene             GPR correction <br /> mobA (b3857)                                  bmocogdp[c]           hisH (b2023)     essential IG3PS [30] <br /> ubiC (b4039)                                  2ohph[c]              cyaY (b3807)     essential I2FE2SS, I2FE2SS2, S2FE2SS, S2FE2SS2 <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                       Page 8 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 7 False negative model predictions                      Table 8 False negative model predictions caused misidentified experiment phenotypes media                         missing isozymes alternate pathways <br /> compositions                                                         Gene                    Putative Isozyme             E-value <br /> Gene           Explanation                                           purK (b0522)            purT (b1849)                 2.00E-12 <br /> mtn (b0159)    essential according Choi-Rhee et al. [36]          gltA (b0720)            prpC (b0333)                 1.00E-41 <br /> thiI (b0423)   possibly thiamin media incomplete washing   aspC (b0928)            tyrB (b4054)                 4.00E-94 <br /> bioA (b0774)   possibly biotin media incomplete washing    fabH (b1091)            identified <br /> bioB (b0775)   possibly biotin media incomplete washing    pabC (b1096)            ilvE (b3770)                 7.00E-8 <br /> bioF (b0776)   possibly biotin media incomplete washing    icd (b1136)             dmlA (b1800)                 3.00E-19 <br /> bioC (b0777)   possibly biotin media incomplete washing    aldA (b1415)            gabD (b2661)                 1.00E-90 <br /> bioD (b0778)   possibly biotin media incomplete washing                            prr (b1444)                  3.00E-80 <br /> aroD (b1693)   experimental growth condition,                                 feaB (b1385)                 1.00E-69 <br />                possible error <br />                                                                                              aldB (b3588)                 2.00E-66 <br /> thiD (b2103)   essential according Orth et al. [10] <br />                                                                                              betB (b0312)                 4.00E-65 <br /> cysD (b2752)   experimental growth condition, <br />                possible error                                        ubiX (b2311)            identified <br /> argG (b3172)   experimental growth condition,         luxS (b2687)            identified <br />                possible error                                        thyA (b2827)            identified <br /> cysG (b3368)   experimental growth condition,         zupT (b3040)            identified <br />                possible error <br />                                                                      folB (b3058)            folX (b2303)                 1.00E-4 <br /> bioH (b3412)   possibly biotin media incomplete washing <br />                                                                      argG (b3172)            identified <br /> ilvE (b3770)   experimental growth condition, <br />                possible error                                        folP (b3177)            identified <br /> thiH (b3990)   essential according Orth et al. [10]               yrbG (b3196)            identified <br /> thiG (b3991)   essential according Orth et al. [10]               kdsC (b3198)            identified <br /> thiF (b3992)   essential according Orth et al. [10]               argD (b3359)            astC (b1748)                 1.00E-146 <br /> thiE (b3993)   essential according Orth et al. [10]                                       gabT (b2662)                 3.00E-64 <br /> thiC (b3994)   essential according Orth et al. [10]                                       puuE (b1302)                 3.00E-54 <br /> cysQ (b4214)   MOPS possible alternate S source                                         patA (b3073)                 4.00E-52 <br />                                                                                              hemL (b0154)                 3.00E-32 <br /> resolving reactions, model constrained match              cysG (b3368)            identified false negative condition, time, SMILEY             ilvE (b3770)            pabC (b1096)                 8.00E-8 run. gene knockout strains lead false               dapF (b3809)            identified <br /> negative predictions 34 tested substrates                                                                       argC (b3958)            identified , likely set missing <br /> reactions cause incorrect predictions         argB (b3959)            identified <br /> strain. cases, SMILEY run model                  hemE (b3997)            identified glucose  aerobic anaerobic), glycerol,            ubiC (b4039)            identified <br /> lactate, succinate substrates. predict gap-filling         purA (b4177)            identified <br /> reactions, small lower bound placed known <br /> producing consuming reaction knowledge <br /> gap metabolite, SMILEY run. order actu-                incomplete reactions removed set (Add- <br /> ally carry small flux reactions satisfy          itional file 3). model constraints, gap-filling reaction set reac-           total 1176 optimal suboptimal solutions tions need added. SMILEY run 166                  identified SMILEY. Solutions identified 106 false negative cases 49 gap reactions (Additional file           false negative cases 32 gaps. Multiple optimal <br /> 2). model knowledge gaps [8] targeted,                 solutions cases, total <br /> scope gaps. algorithm set 25 alter-            198 different optimal solutions 983 different subopti- <br /> nate solutions condition, time limit 2 h           mal solutions. solutions optimal placed solution. reactions added                  suboptimal solutions different cases, 385 solu- <br /> SMILEY universal set reactions based               tions multiple times. reactions KEGG Release 58.0 [37]. Unrealistic             gene knockout strains grown multiple substrates  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                       Page 9 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> genes required GPRs reaction                                         model used predict growth phenotypes reactions pathway. false negative                                          FBA 13,470 conditions experimental <br /> cases gaps, small number optimal solutions                                            dataset. false positive false negative predictions (Figure 3 . solution 77 cases,                                        identified comparison experimental data- solutions 91 cases.                                           set, number false negatives eliminated cases, 25 solutions optimal. cases                                         new false positives created SMILEY solution iscS (b2530) knockout strain grown different                                        counted. feasible solutions conditions. gene ISC [Fe-S] cluster gen-                                      fixed false negatives introdu- <br /> eration model essential role                                        cing false positives. average, solution cor- molybdenum cofactor synthesis. alternate                                           rected 7.48 false negatives created 7.07 new false <br /> solutions involves import cofactor. average                                         positives. total 144 solutions (11 optimal 133 <br /> number optimal solutions SMILEY run                                                suboptimal) eliminate false negatives <br /> 2.56. optimal solutions included reaction,                                            producing new false positives. GapFind included (Figure 3 b). average                                          run model solution added, determine <br /> number reactions optimal solution 1.41.                                                  model gaps eliminated. 74 solutions   molybdenum cofactor uptake reactions demon-                                                gap  <br /> strate, SMILEY solutions realistic. computa- <br /> tional feasibility check performed identify                                              Predictions genes hypothesized reactions realistic solutions. solutions added                                           feasible SMILEY solutions complete iJO1366 model time, augmented                                               set 1176 solutions investigated  <br />                                                                                                     feasible solutions, BLASTp used try <br />                                                                                                     identify candidate genes E. coli genome <br />                                                                                                     (Table 9). solutions divided categor- <br />                                                                                                             ies. Category solutions optimal solutions                           80 77                                                                     eliminated false negative condition cre- <br />      Number FNs/gaps <br />  <br />  <br />  <br />  <br />                                                                                                     ating new false positives. solutions gave                                     61 <br />                           60                                                                        average MCC 0.5436, slightly better original <br />                                                                                                     model. 11 category solutions, fixed false <br />                           40                                                                        negatives adding deleted model reaction                                      30                                                             universal reaction list. indicates                           20                         17                                             uncharacterized isozymes possible aspC (b0928), <br />                                                           7                          8              pabC (b1096), aldA (b1415), argD (b3359), hemE <br />                                              5                4                                 4 <br />                                                  2                                                  (b3397). solution suggested false negatives <br />                            0 <br />                                 0        2       4        6       8   10 12 14 16 18 20 22 24       &#206;&#8221;aspC strains corrected adding exist- <br />                                                   Number optimal solutions                       ing model reaction ASP1DC (aspartate 1-decarboxylase) <br />                                                                                                     reverse. literature evidence support <br />     b                                                                                               refute reversibility reaction.                           120 <br />                                                                                                     solutions involve addition new reactions                                      109 <br />                                                                                                     model. provide potential production routes <br />    Number solutions <br />  <br />  <br />  <br />  <br />                           100                                                                       aspartate compliment aspC deletion.                           80                                                                        provides new reaction consume glycoaldehyde                                                                                                     &#206;&#8221;aldA strains. reactions asso- <br />                           60 <br />                                                                                                     ciated genes KEGG database, indicating                           40                                                                        global orphan reactions. Candidate genes                                                                                                     reactions identified reference <br />                           20                                             16 <br />                                                               9 <br />                                                                                          4 <br />                                                                                                     sequences available. <br />                            0                                                                          second category SMILEY solutions inves- <br />                                          1                    2           3      4       5 <br />                                                                                                     tigated optimal solutions fixed                                                  Number reactions solution <br />                                                                                                     false negatives number new false positives <br />  Figure 3 Properties 198 optimal SMILEY solutions.   <br />                                                                                                     created. 70 category II solutions   <br />  Number optimal solutions SMILEY run. cases,                                        cluding category solutions, fall  zero optimal solutions  (b) Number                                   <span id='am-70' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>). average MCC solutions <br />  reactions optimal solution. optimal solutions consisted                                0.5531. solutions involve uptake  reaction.                                                                                 molybdenum cofactors precursors. explained <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                            Page 10 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Table 9 Predicted genes feasible FN-                        solutions added feasible new reactions.  slightly <br /> correcting SMILEY solutions                                              different reaction producing <span id='am-147' about='obo:IMR_0200187' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-148' property="oboInOwl:hasDbXref" content="KEGG:C00364" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:id" content="IMR:0200187" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="dTMP" datatype="xsd:string"></span>dTMP</span> added Hypothesized changes directionality                                   compliment thyA (b2827) deletion. currently unchar- <br /> Reaction     Category   Support                                          acterized E. coli gene, ybiU (b0821), identified ASP1DC                                                                 BLASTp candidate gene reaction, providing <br /> ASPT         III        reversible (Karsten Viola [38]) <br />                                                                          testable hypothesis function gene.                                                                          category II feasible solution added new reaction <br /> ICL          IV         reversible (MacKintosh Nimmo [39]) <br />                                                                          convert <span id='am-197' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-198' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-199' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-200' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-201' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-202' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-203' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-204' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-205' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-206' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-207' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-208' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span>L-glutamate</span> &#206;&#177;-ketoglutarate. candi- <br /> AKGDH        IV         reversible (EcoCyc)                          date genes high sequence homology known genes <br /> CITL         IV                                                          organisms, ydbL (b0600) ydcR (b1439), <br /> Hypothesized gap-filling reactions                                        <br /> Reaction     Category   Putative gene                         E-value       category investigated consisted R00352 (R)   IV         sucD (b0729)                          2.00E-20   suboptimal solutions fixed false <br /> R00373 (F)            global orphan                                    negative producing new false positives.                                                                          total 133 category III solutions  having <br /> R00400 (F)            global orphan <br />                                                                          average MCC 0.5433. solutions <br /> R00507 (R)   IV         yhfW (b3380)                          0.47 <br />                                                                          included unrealistic reactions, oxygen <br /> R00529 (F)   IV         cysN (b2751) cysD (b2752) *                  consuming KEGG reaction R00357 reverse, <br /> R00530 (F)   IV         global orphan                                    oxygen producing direction. attempt com- <br /> R00531 (R)   IV         global orphan                                    pensate loss cofactor producing pathways <br /> R00695 (R)            global orphan                                    simply adding new uptake reactions                                                                          cofactors.  realistic reactions suggested <br /> R00709 (F)   IV         dmlA (b1800)                          6.00E-26 <br />                                                                          BLASTp identified candidate genes. solution <br />                         icd (b1136)                           1.00E-26 <br />                                                                          consisted addition current model reac- <br />                         leuB (b0073)                          2.00E-15   tion ASPT (L-aspartase) reverse. Experimental evi- <br /> R00732 (R)   III        aroA (b0908)                          5.00E-32   dence supports reversibility reaction [38], <br />                         murA (b3189)                          7.00E-8    currently listed irreversible iJO1366. <br /> R00733 (R)   III        tyrA (b2600)                          2.80E-2    fourth final category SMILEY solutions                                                                          examined optimal solutions R01393 (R)            global orphan <br />                                                                          categories II. 62 solutions R01618 (R)   IV         glgP (b3428)                          2.10 <br />                                                                          category average MCC R01713 (F)            global orphan                                    0.5304, slightly worse unmodified iJO1366 <br /> R01731 (F)   IV         tyrB (b4054) *                                   model. solutions simply new  <br /> R01785 (R)   III        rhaD (b3902) *                                   reactions blocked essential biomass compo- <br /> R01902 (R)   III        rhaB (b3904) *                                   nents, 14 new realistic reactions suggested, <br />                                                                          current model reactions running R02200 (F)   IV         global orphan <br />                                                                          opposite directions. new reversible <br /> R04209 (R)   IV         purC (b2476)                          7.00E-16 <br />                                                                          reactions, ICL (isocitrate lyase), confirmed R05717 (R)   IV         cysH (b2762)                          3.00E-12   published study [39],  AKGDH (2-Oxo- <br /> R06613 (F)   II         ybiU (b0821)                          1.6        glutarate dehydrogenase), actually reversible <br /> R07164 (R)   III        ydiJ (b1687)                          0.9        according EcoCyc [40]. Additional file 4 R07165 (R)   III        ydiJ (b1687)                          0.9        category IV solutions investigated. <br />                                                                             72 gap-filling SMILEY solutions investigated <br /> R07176 (R)   IV         global orphan <br />                                                                          (Additional file 5), BLASTp used predict genes <br /> R07463 (F)   IV         dadA (b1189)                          2.00E-18 <br />                                                                          realistic reactions (Table 10). total 20 new real- <br /> R07613 (R)   II         ydbL (b0600)                          7.00E-26   istic reactions  candidate genes                         ydcR (b1439)                          6.00E-15   predicted half  global  <br /> R08553 (R)   IV         ysaA (b3573)                          4.00E-5    phan reactions. SMILEY suggested 15 existing <br />                                                                          model reactions reversible gaps. <br />                                                                          According EcoCyc, reactions  likely explanation false nega-                 reversible.  evidence supporting tives molybdenum cofactor strictly                    reversibility model reactions, DKGLCNR1 (2,5- <br /> required growth E. coli. solutions                  diketo-D-gluconate reductase) [41] DKGLCNR2y (2,5- <br /> added deleted reactions network,                   diketo-D-gluconate reductase (<span id='am-86' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-87' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span>)) [42]. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                         Page 11 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> Experimental validation predicted genes                    Table 10 Predicted genes gap-filling SMILEY solutions <br /> SMILEY gap-filling algorithms useful         Hypothesized changes directionality <br /> cause use model existing experimental          Reaction               Support <br /> data generate predictions. performing ex-       DOGULNR                reversible (EcoCyc) <br /> periment verify predictions, hy- <br />                                                               DKGLCNR1               reversible (Habrych et al. [41]) <br /> potheses. iJO1366 model used design simple <br />                                                               DKGLCNR2y              reversible (Yum et al. [42]) <br /> growth phenotype experiments confirm predictions. 1176 solutions added      PGLYCP                 reversible (EcoCyc) <br /> model time, growth simulated          CYSSADS <br /> combinations set 115 carbon sources 62 ni-        HMPK1                  reversible (EcoCyc) <br /> trogen sources aerobic anaerobic condi-        4HTHRS <br /> tions. substrates selected readily <br />                                                               HETZK                  reversible (EcoCyc) <br /> available chemicals use laboratory.                                                               NNDMBRT                reversible (EcoCyc) <br /> substrate combination growth predicted model SMILEY solution added,           ACONMT                 reversible (EcoCyc) unmodified iJO1366 model, vivo experiment           CINNDO                 reversible (EcoCyc) performed determine E. coli actually         MCPST <br /> grow substrates, giving supporting experi-         GPDDAS                 reversible (EcoCyc) <br /> mental evidence predicted reactions.                                                               APCS                   reversible (EcoCyc) <br /> solutions, new growth conditions identified.                                                               SARCOX <br /> realistic solutions testable experimental conditions listed Additional file 6.                              Hypothesized gap-filling reactions <br />   reaction growth experiment pre-         Reaction               Putative gene                          E-value <br /> dicted possible R01184, myo-inositol:oxygen         R01742 (F)             ydiS (b1699)                           0.003 <br /> oxidoreductase. reaction combines myo-inositol           R00893 (F)             ygfM (b0419)                           0.58 oxygen form D-glucuronate water. Myo           R02133 (F)             yhbO (b3153)                           0.069 <br /> ositol root consumption gap iJO1366            R02721 (F)             global orphan <br /> model, reaction fills gap. reaction <br />                                                               R03472 (R)             global orphan <br /> included, E. coli predicted grow myo-inositol substrate. vivo experiment performed,          R01297 (R)             global orphan wild-type E. coli inoculated 2 g/L myo        R01299 (R)             global orphan <br /> ositol minimal media carbon sources.            R02252 (F)             fadH (b3081)                           7.00E-71 replicates performed, 72 h, cul-                            nemA (b1650)                           1.00E-22 <br /> tures reached OD600 0.017 &#194;&#177; 0.006. result      R00895 (R)             aspC (b0928)* <br /> indicates E. coli grow slowly myo  <br />                                                               R03530 (F)             ndk (bb2518)* <br /> ositol carbon source.  candidate <br /> genes predicted reaction using BLASTp.          R00012 (F)             global orphan genes ydeN (b1498), yfdE (b2371), yphC             R01232 (R)             yjhG (b4297)                           0.028 <br /> (b2545), yhiJ (b3488). functions genes                              yagF (b0269)                           0.057 currently unknown, non-essential.        R00838 (F)             chbF (b1734)                           9.00E-42 <br /> Keio Collection knockout strains genes                                melA (b4119)                           2.00E-21 obtained grown glucose myo- <br />                                                               R00655 (R)             global orphan <br /> inositol minimal media. strains grew similar <br /> OD600 wild-type E. coli glucose, myo-inosi-      R07300 (F)             global orphan <br /> tol, knockout strains did grow (Figure 4).    R00683 (F)             global orphan yhiJ knockout strain did grow  indicating      R00367 (F)             global orphan gene likely codes myo-inositol:oxygen oxidore-     R02559 (F)             global orphan <br /> ductase E. coli.                                           R02560 (F)             global orphan <br />                                                               R05623 (F)             yjiN (b4336)                           0.16 <br /> Discussion study, iJO1366 metabolic network model E. <br /> coli used discovery tool, leading predictions     coli metabolism structured format, compar- new metabolic gene functions. date         ing model predictions experimental data, errors model represents current state knowledge E.         gaps knowledge identified. large dataset <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                     Page 12 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  assembled Keio Collection gene knockout phe-                  careful analysis false positive results, <br /> notypes grown 34 different substrates. pheno-              possible model errors identified. types compared model predicted phenotypes,                  cases, model predicted growth pathways model false positive false negative predic-             poorly characterized genes required. genes <br /> tions identified. analyzed, false positive            need investigated  future experi- <br /> predictions indicated possible errors                mental evidence shows encode current model, including pathways thought cata-               enzymes currently believed encode, lyzed poorly studied enzymes, uncertain                  model updated removing reactions. False <br /> requirements E. coli biomass formation. false                negative results, hand, indicate errors <br /> negative cases indicated potential errors           experimental design, model. biomass likely experimental errors.          cases identified E. coli grow min- <br /> Importantly, false negative cases indicated          imal media despite lacking genes synthesis es- model currently incomplete. SMILEY algo-                 sential cofactors. Biotin thiamin known rithm used predict likely missing reac-             essential, substrates growth observed <br /> tions, novel procedure, feasibility           knockout strains able produce <br /> predictions assessed comparisons ex-             cofactors. possible explanation perimental data model gap analysis. Gene                alternate synthesis routes compounds, predictions feasible predicted               cases, likely trace amounts bio- <br /> reactions, experimental evidence generated               tin thiamin present media support predicted function gene yhiJ.                    growth experiments [23,25]. false positive false <br />                                                                     negative results indicate potential errors iJO1366 <br />                                                                     core biomass reaction. False positives indicate gene <br />                                                                     helps produce essential compound missing <br />                                                                   biomass reaction, false negatives indicate <br />                   2                                                 gene produces non-essential compound                                                                     included. biomass components essen- <br />                 1.6 <br />                                                                     tial non-essential certain conditions,  <br />                                                                     necessitating use condition-specific biomass reac- <br />        OD 600 <br />  <br />  <br />  <br />  <br />                 1.2 <br />                                                                     tions specific model applications. <br />                 0.8                                                    SMILEY used predict gap-filling false <br />                                                                     negative correcting reactions added                 0.4 <br />                                                                     metabolic network. reactions                   0 <br />                                                                     existing model reactions opposite direction. <br />                        WT   &#206;&#8221;ydeN   &#206;&#8221;yfdE    &#206;&#8221;yphC     &#206;&#8221;yhiJ        Literature data searched confirm refute pre- <br />                                     Strain                          dictions, supporting evidence                                                                     reactions. SMILEY solutions predicted addition <br />                                                                     completely new reactions network. gap-filing <br />    b <br />                                                                     solutions feasible false negative correcting <br />                 0.03 <br />                                                                     solutions inspected manually, potentially real- <br />             0.025                                                   istic reactions, genes predicted based protein se- <br />              0.02 <br />                                                                     quence homology. feasible predicted reactions <br />    OD 600 <br />  <br />  <br />  <br />  <br />                                                                     cover wide range metabolic functions.             0.015 <br />                                                                     corrected false negative predictions aspC knockout <br />              0.01                                                   strains, model unable produce L-aspar- <br />                                                                     tate. predicted new reactions involving TCA <br />             0.005 <br />                                                                     cycle intermediates. New reactions predicted <br />                  0                                                  metabolism <span id='am-152' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-153' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-157' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-158' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-160' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-161' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-162' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-164' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-165' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-166' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> 5&#226;&#8364;&#8482;-phosphosulfate, <br />                        WT   &#206;&#8221;ydeN   &#206;&#8221;yfdE    &#206;&#8221;yphC     &#206;&#8221;yhiJ <br />                                                                     dehydroglycine, dTMP, glycoaldehyde, L-isoleucine, <br />                                     Strain <br />                                                                     5-phosphoribosyl-5-carboxyaminoimidazole. <br />  Figure 4 Growth Keio Collection gene knockout stains <br />                                                                        new workflow presented theoretically <br />  identify possible myo-inositol:oxygen oxidoreductase.       applied organism metabolic net- <br />  Growth strains WT glucose M9 minimal media.     work reconstruction available. requirement <br />  (b) Growth strains WT myo-inositol M9 minimal   fairly large set experimental phenotypes, <br />  media. replicate measurements performed.                growth different substrates growth <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                         Page 13 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  gene knockouts, available. possible   Methods workflow applied using meta-     Comparison model predictions experimental data <br /> bolic network gaps, case organisms     experimental gene essentiality data obtained <br /> extensive experimental data. similar study utilizing   publications [10,23-25], &#226;&#8364;&#339;essential&#226;&#8364;? gaps phenotypes performed       &#226;&#8364;&#339;non-essential&#226;&#8364;? designations assigned original <br /> human metabolic network [13]. Despite number         studies used. corrections essentiality potentially useful predictions  SMILEY did       assignments based updated analysis solutions nearly half cases       Keio Collection [26]. newly identified essential run. reason    genes added lists essential genes universal set reactions used based         conditions, genes essentiality identi- <br /> KEGG [37]. database contains reactions    fied uncertain changed original known exist organism,     designations. completely undiscovered reactions              iJO1366 E. coli K-12 MG1655 metabolic network <br /> added.  metabolite iJO1366      reconstruction loaded COBRA Toolbox <br /> connected KEGG reactions. 1133 compart-        [44], adjusted match phenotype E. coli <br /> ment-independent metabolites iJO1366, 203      BW25113, missing metabolic genes KEGG compound IDs. larger set reactions        (&#206;&#8221;araBAD, &#206;&#8221;rhaBAD, &#206;&#8221;lacZ). associated reactions <br /> including model metabolites allow         isozymes (ARAI, RBK_L1, RMPA, LYXI, RMI, <br /> additional valid SMILEY solutions       RMK, LACZ) constrained carry zero flux. <br /> gene predictions based sequence hom-        model reactions retained default bounds <br /> ology, reactions, gene pre-     [10]. Minimal media simulated setting lower <br /> dicted reference sequence           bound &#226;&#710;&#8217;1000 (allowing unlimited uptake) ex- <br /> available. global orphan reactions [43],       change reactions Ca2+, Cl-, <span id='am-40' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-41' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span>CO2</span>, Co2+, Cu2+, Fe2+, known gene organism. pro-       Fe3+, H+, <span id='am-112' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-113' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-116' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span>, K+, Mg2+, Mn2+, MoO2-         +     2+ <br />                                                                                                 4 , Na , Ni , NH4, <br />                                                                                                                   - <br />                                                                    2-    2-     2-    2-     2-          2+ <br /> liferation global orphans (estimated 30-40 %    <span id='am-285' about='obo:IMR_0200217' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-286' property="oboInOwl:hasDbXref" content="KEGG:C00007" datatype="xsd:string"></span><span id='am-287' property="oboInOwl:id" content="IMR:0200217" datatype="xsd:string"></span><span id='am-288' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-289' property="rdfs:label" content="O2" datatype="xsd:string"></span>O2</span>, HPO4 , SeO4 , SeO3 , SO4 , WO4 , Zn . lower known enzymatic functions [6]) makes gene         bound &#226;&#710;&#8217;0.01 placed cob alamin exchange <br /> function prediction difficult, account   reaction. knockout strain modeled using fact studied organism E.     deleteModelGenes function constrain correct <br /> coli, uncharacterized genes.        reactions zero. Model growth phenotypes deter- <br />                                                          mined using FBA core biomass reaction Conclusions                                              objective, time condition. Strains study utilized genome-scale metabolic network     growth rates zero classified non-essential, <br /> reconstruction tool analysis high-       strains growth rates zero classified throughput experimental data. ultimate result     essential. Tomlab (Tomlab Optimization , Se- study number valuable predictions      attle, WA) linear programming solver used   predictions        form FBA. <br /> adjustments iJO1366 model, pre- <br /> dicted changes core biomass reaction          Computational prediction gap-filling reactions <br /> changes GPRs reactions IG3PS, I2FE2SS,     COBRA Toolbox 2.0 implementation I2FE2SS2, S2FE2SS, S2FE2SS2. Corroborating lit-      SMILEY algorithm (growthExpMatch) used pre- <br /> erature evidence        dict sets gap-filling reactions false negative <br /> predictions, incorporated fu-     model comparison. universal database reactions <br /> ture updates reconstruction. predic-    obtained KEGG Release 58.0 [37]. reactions <br /> tions study gene functions,    set listed &#226;&#8364;&#339;incomplete reaction&#226;&#8364;? black- isozymes reactions currently        listed, excluded possible SMILEY solutions. known occur E. coli. predictions provide     reaction compound appearing hypotheses experimentally tested.      substrate product blacklisted, example, prediction missing myo-inositol:oxy-   reactions identified initial tests (R00090, <br /> gen oxidoreductase reaction led design       R00113, R00274) forming unrealistic energy gen- <br /> simple experiment previously uncharac-      erating reaction loops existing iJO1366 model reac- <br /> terized gene yhiJ essential growth   tions. minimum growth threshold required myo-inositol. expect        SMILEY algorithm 0.05 h-1. 25 alternate solu- <br /> predictions study likewise serve     tions allowed, single solution time limit hypotheses experimental analysis.                    2 h. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                 Page 14 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br />    SMILEY run gaps instead false nega-           sodium phosphate dibasic, 3.0 g/L potassium phosphate <br /> tive cases, producing consuming reaction            monobasic, 0.5 g/L sodium chloride, 0.24 g/L magnesium gap metabolite identified iJO1366             sulfate, 0.011 g/L calcium chloride), trace elements (0.1 g/L <br /> model. lower bound 0.01 mmol/gDW/h applied             iron (III) chloride, 0.02 g/L zinc sulfate, 0.004 g/L copper reaction, time, SMILEY used         chloride, 0.01 g/L manganese sulfate, 0.006 g/L cobalt <br /> predict gap-filling reactions. gaps demand        chloride, 0.006 g/L disodium EDTA), Wolfe&#226;&#8364;&#8482;s Vitamin <br /> reactions model, demand reactions           Solution. Wild-type E. coli strains strained zero running SMILEY. Tomlab              Keio Collection yphC (JW5842), yfdE (JW2368), ydeN <br /> mixed-integer linear programming solver used.               (JW5243), yhiJ (JW3455) gene knockouts (supplied                                                                 Open Biosystems) grown overnight LB. Computational feasibility analysis predictions           day, 15 mL culture centrifuged 4000 rpm predicting sets false negative correcting          8 min, supernatant discarded, culture gap-filling reactions, sets solution reac-     resuspended M9 salt solution carbon nitro- <br /> tions added iJO1366 model time.             gen sources. culture centrifuged resuspended growth phenotype strains            new M9 times completely wash LB. <br /> 13,470 experimental data conditions predicted           1 &#206;&#188;L washed E. coli cultures used  <br /> using FBA, threshold zero determining             oculate 10 mL aerobic myo-inositol M9 cultures, growth growth. number new false positives          grown 37&#194;&#176;C. optical density 600 nm solution determined number             measured points growth. <br /> conditions true negatives wild-type <br /> model grow new reactions                Additional files <br /> added. number corrected false negatives solution number false negatives           Additional file 1: Experimental computational growth phenotypes <br />                                                                  13,470 conditions. <br /> true positives new reactions added. Gap- <br />                                                                  Additional file 2: false negative conditions gaps run iJO1366 model set          SMILEY run. <br /> solution reactions added  time. set        Additional file 3: KEGG based universal database reactions network gaps compared set gaps ori-         list excluded reactions. <br /> ginal model determine gaps eliminated.            Additional file 4: Category IV SMILEY solutions. <br />   order determine SMILEY solutions             Additional file 5: gap-filling SMILEY solutions. tested simple vivo experiments, FBA used          Additional file 6: Media conditions indicate presence test growth iJO1366 model solution           predicted reactions. <br /> added set 115 carbon sources 62 nitrogen <br /> sources aerobic anaerobic conditions.            Competing interests <br />                                                                 authors declare competing interests. growth unmodified iJO1366 model tested condition using FBA.  model             Acknowledgements solution added, time, tested      like thank Pep Charusanti, Tom Conrad, Adam Feist                                                                 helpful comments insights. work funded 3896 conditions unmodified model pre-              National Institutes Health grant GM057089. <br /> dicted growth. Conditions modified ver- <br /> sions model grow used design               Authors&#226;&#8364;&#8482; contributions <br />                                                                 JDO BOP designed study. JDO performed computational <br /> experiments.                                                    predictions, analyzed results, performed experimental analysis,                                                                 wrote manuscript. JDO BOP revised manuscript. authors <br />                                                                 approved content final manuscript. <br /> Experimental validation predicted genes <br /> Candidate genes SMILEY predicted reaction sets predicted using bi-directional protein BLAST               Received: 16 January 2012 Accepted: 1 2012 <br />                                                                 Published: 1 2012 <br /> (BLASTp) gene organism KEGG E. coli K-12 MG1655 genome. Protein <br />                                                                 References <br /> sequences organisms phylogenically close          1. Edwards JS, Palsson, B. .: Metabolic flux balance analysis E. coli used possible. gene               silico analysis Escherichia coli K-12 gene deletions. BMC bioinformatics <br /> highest BLAST expectation value (E-value)                 2000, 1(1). <br />                                                                 2. Edwards JS, Covert M, Palsson B : Metabolic modeling microbes: reported. multiple genes E-values              flux-balance approach. Environmental Microbiology 2002, 4(3):133&#226;&#8364;&#8220;140. 10-13, reported candidate genes.              3. Price ND, Papin JA, Schilling CH, Palsson B : Genome-scale microbial   test growth E. coli myo-inositol carbon       silico models: constraints-based approach. Trends biotechnology <br />                                                                     2003, 21(4):162&#226;&#8364;&#8220;169. energy source, 2 g/L myo-inositol M9 media         4. Orth JD, Thiele  Palsson B : flux balance analysis? Nature filter sterilized. media contained M9 salts (6.8 g/L       biotechnology 2010, 28(3):245&#226;&#8364;&#8220;248. <br />  Orth Palsson BMC Systems Biology 2012, 6:30                                                                                                      Page 15 15 <br /> http://www.biomedcentral.com/1752-0509/6/30 <br />  <br />  <br />  <br />  <br /> 5.    Reed JL, Patel TR, Chen KH, Joyce AR, Applebee MK, Herring CD, Bui OT,              allantoin nitrogen linked glyoxylate metabolism Escherichia coli. <br />       Knight EM, Fong SS, Palsson B : Systems approach refining genome                 Journal bacteriology 1999, 181(24):7479&#226;&#8364;&#8220;7484. <br />       annotation. Proceedings National Academy Sciences United     29.   Portnoy VA, Herrgard MJ, Palsson B : Aerobic fermentation <span id='am-192' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-193' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-194' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-195' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-196' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span>D-glucose</span> <br />       States America 2006, 103(46):17480&#226;&#8364;&#8220;17484.                                        evolved cytochrome oxidase-deficient Escherichia coli strain. <br /> 6.    Karp PD: enzyme genomics initiative. Genome biology 2004,               Appl Environ Microbiol 2008, 74(24):7561&#226;&#8364;&#8220;7569. <br />       5(8):401.                                                                     30.   Klem TJ, Davisson VJ: Imidazole glycerol phosphate synthase: 7.    Pouliot Y, Karp PD: survey orphan enzyme activities.                           glutamine amidotransferase histidine biosynthesis. Biochemistry 1993, <br />       BMC bioinformatics 2007, 8:244.                                                     32(19):5177&#226;&#8364;&#8220;5186. <br /> 8.    Orth JD, Palsson B : Systematizing generation missing metabolic        31.   Neidhardt FC (ed.): Escherichia coli Salmonella: cellular molecular <br />       knowledge. Biotechnology bioengineering 2010, 107(3):403&#226;&#8364;&#8220;412.                   biology, 2nd edn. Washington, D.C.: ASM Press; 1996. <br /> 9.    Satish Kumar V, Dasika MS, Maranas CD: Optimization based automated           32.   Lin S, Hanson  Cronan JE: Biotin synthesis begins hijacking fatty <br />       curation metabolic reconstructions. BMC bioinformatics 2007, 8:212.              acid synthetic pathway. Nature <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> biology 2010, 6(9):682&#226;&#8364;&#8220;688. <br /> 10.   Orth JD, Conrad TM, Na J, Lerman JA, Nam H, Feist  Palsson B :           33.   Bettendorff L, Wins P: Thiamin diphosphate biological chemistry: new <br />       comprehensive genome-scale reconstruction Escherichia coli                       aspects thiamin metabolism, especially <span id='am-122' about='obo:IMR_0200358' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-123' property="oboInOwl:id" content="IMR:0200358" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasDbXref" content="KEGG:C00536" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-126' property="rdfs:label" content="Triphosphate" datatype="xsd:string"></span>triphosphate</span> derivatives acting <br />       metabolism-2011. Molecular systems biology 2011, 7:535.                             cofactors. FEBS J 2009, 276(11):2917&#226;&#8364;&#8220;2925. <br /> 11.   Reed JL, Vo TD, Schilling CH, Palsson B : expanded genome-scale            34.   Patton AJ, Hough DW, Towner P, Danson MJ: Does Escherichia coli possess <br />       model Escherichia coli K-12 (iJR904 GSM/GPR). Genome biology 2003,               second citrate synthase gene? European journal biochemistry/FEBS <br />       4(9):R54.51&#226;&#8364;&#8220;R54.12.                                                                 1993, 214(1):75&#226;&#8364;&#8220;81. <br /> 12.   Bochner BR, Gadzinski P, Panomitros E: Phenotype microarrays              35.   Gerike U, Hough DW, Russell NJ, Dyall-Smith ML, Danson MJ: Citrate <br />       high-throughput phenotypic testing assay gene function.                      synthase 2-methylcitrate synthase: structural, functional       Genome research 2001, 11(7):1246&#226;&#8364;&#8220;1255.                                              evolutionary relationships. Microbiology (Reading, England) <br /> 13.   Rolfsson O, Palsson B , Thiele  human metabolic reconstruction                1998, 144 (Pt 4):929&#226;&#8364;&#8220;935. <br />       Recon 1 directs hypotheses novel human metabolic functions. BMC            36.   Choi-Rhee E, Cronan JE: nucleosidase required vivo function       systems biology 2011, 5:155.                                                        <span id='am-117' about='obo:IMR_0200054' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-118' property="rdfs:label" content="S-Adenosyl-L-methionine" datatype="xsd:string"></span><span id='am-119' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-120' property="oboInOwl:hasDbXref" content="KEGG:C00019" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:id" content="IMR:0200054" datatype="xsd:string"></span>S-adenosyl-L-methionine</span> radical enzyme, biotin synthase. Chem Biol <br /> 14.   Duarte NC, Becker SA, Jamshidi N, Thiele  Mo ML, Vo TD, Srivas R, Palsson         2005, 12(5):589&#226;&#8364;&#8220;593. <br />       B : Global reconstruction human metabolic network based             37.   Kanehisa M, Goto S, Furumichi M, Tanabe M, Hirakawa M: KEGG       genomic bibliomic data. Proceedings National Academy                  representation analysis molecular networks involving diseases <br />       Sciences United States America 2007, 104(6):1777&#226;&#8364;&#8220;1782.                    drugs. Nucleic acids research 2010, 38(Database issue):D355-360. <br /> 15.   Kumar VS, Maranas CD: GrowMatch: automated method reconciling          38.   Karsten  Viola  Kinetic studies L-aspartase Escherichia coli: <br />       silico vivo growth predictions. PLoS computational biology 2009,              pH-dependent activity changes. Archives biochemistry biophysics <br />       5(3):e1000308.                                                                      1991, 287(1):60&#226;&#8364;&#8220;67. <br /> 16.   Zomorrodi AR: Maranas CD. Improving iMM904 S. cerevisiae metabolic        39.   MacKintosh C, Nimmo HG: Purification regulatory properties       model using essentiality synthetic lethality data. BMC systems biology          isocitrate lyase Escherichia coli ML308. Biochemical journal 1988, <br />       2010, 4:178.                                                                        250(1):25&#226;&#8364;&#8220;31. <br /> 17.   Paley SM, Karp PD: Evaluation computational metabolic-pathway              40.   Keseler IM, Bonavides-Martinez C, Collado-Vides J, Gama-Castro S, Gunsalus <br />       predictions Helicobacter pylori. Bioinformatics (Oxford, England) 2002,         RP, Johnson DA, Krummenacker M, Nolan LM, Paley S, Paulsen et al: <br />       18(5):715&#226;&#8364;&#8220;724.                                                                      EcoCyc: comprehensive view Escherichia coli biology. Nucleic acids <br /> 18.   Green ML, Karp PD: Bayesian method identifying missing enzymes                research 2009, 37(Database issue):D464-470. <br />       predicted metabolic pathway databases. BMC bioinformatics 2004, 5:76.      41.   Habrych M, Rodriguez S, Stewart JD: Purification identification 19.   Green ML, Karp PD: Using genome-context data identify specific types             Escherichia coli beta-keto ester reductase 2,5-diketo-D-gluconate <br />       functional associations pathway/genome databases. Bioinformatics              reductase YqhE. Biotechnol Prog 2002, 18(2):257&#226;&#8364;&#8220;261. <br />       (Oxford, England) 2007, 23(13):i205-211.                                      42.   Yum DY, Lee  Pan JG: Identification yqhE yafB genes <br /> 20.   Christian N, P, Kempa S, Handorf T, Ebenhoh O: integrative                   encoding 2, 5-diketo-D-gluconate reductases Escherichia coli. <br />       approach completing genome-scale metabolic networks.                        Appl Environ Microbiol 1999, 65(8):3341&#226;&#8364;&#8220;3346. <br />       Molecular bioSystems 2009, 5(12):1889&#226;&#8364;&#8220;1903.                                   43.   Osterman  Overbeek R: Missing genes metabolic pathways: 21.   Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, Broadbelt           comparative genomics approach. Curr Opin Chem Biol 2003, 7(2):238&#226;&#8364;&#8220;251. <br />       LJ, Hatzimanikatis V, Palsson B&#195;&#732;: genome-scale metabolic reconstruction     44.   Schellenberger J, Que R, Fleming RM, Thiele  Orth JD, Feist  Zielinski <br />       Escherichia coli K-12 MG1655 accounts 1260 ORFs                    DC, Bordbar  Lewis NE, Rahmanian S, et al: Quantitative prediction       thermodynamic information. Molecular systems biology 2007, 3(121).                  cellular metabolism constraint-based models: COBRA Toolbox <br /> 22.   Boyle NR, Morgan JA: Flux balance analysis primary metabolism                 v2.0. Nature protocols  <br /> </body></html>