<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                        Europe PMC Funders Group <br />                                         Author Manuscript <br />                                         Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                               Published final edited form  <br />                                                Mol Microbiol. 2006 July ; 61(1): 89&#226;&#8364;&#8220;105. doi:10.1111/j.1365-2958.2006.05216.x. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                               glycosylphosphatidylinositol (GPI) biosynthetic pathway                                               bloodstream-form Trypanosoma brucei dependent                                               novo synthesis inositol <br />                                               Kirstee L. Martin Terry K. Smith* <br />                                               Division Biological Chemistry Molecular Microbiology, School Life Sciences, <br />                                               University Dundee, Dundee DD1 5EH, Scotland, UK <br />  <br />                                               Summary <br />                                                    bloodstream-form Trypanosoma brucei  causative agent African sleeping sickness)                                                    glycosylphosphatidylinositol (GPI) anchor biosynthetic pathway validated genetically <br />                                                    chemically drug target. conundrum GPI anchors vivo labelled <br />                                                    [3H]-inositol led hypothesize novo synthesis responsible supplying myo- <br />                                                    inositol <span id='am-17' about='obo:IMR_0001651' typeof='owl:Thing obo:IMR_0001650 obo:IMR_0001356 obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001384 obo:IMR_0001362'><span id='am-18' property="rdfs:label" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-20' property="rdfs:label" content="phosphatidylinositol" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="MPI" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="phosphoinositide" datatype="xsd:string"></span><span id='am-24' property="rdfs:label" content="inositol phosphoglyceride" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="KEGG:C00626" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasExactSynonym" content="phosphoinositide" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasExactSynonym" content="monophosphoinositide" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:id" content="IMR:0001651" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasExactSynonym" content="MPI" datatype="xsd:string"></span>phosphatidylinositol</span> (PI) destined GPI synthesis. rate-limiting step                                                    novo synthesis isomerization glucose 6-phosphate 1-D-<span id='am-95' about='obo:IMR_0200177' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-96' property="rdfs:label" content="myo-Inositol" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="KEGG:C00137" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:id" content="IMR:0200177" datatype="xsd:string"></span>myo-inositol</span>-3-phosphate, <br />                                                    catalysed 1-D-myo-inositol-3-phosphate synthase (INO1). grown non-permissive <br />                                                    conditions, conditional double knockout demonstrated INO1 essential gene                                                    bloodstream-form T. brucei. showed novo synthesized myo-inositol utilized                                                    form PI, preferentially used GPI biosynthesis. time                                                    extracellular myo-inositol fact used GPI formation limited extent. <br />                                                    Despite  extracellular inositol compensate deletion INO1. Supporting                                                    results, change PI levels conditional double knockout cells grown Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                    non-permissive conditions, showing perturbation growth specific lack                                                    novo synthesized myo-inositol general inositol death. results suggest                                                    distinction novo synthesized myo-inositol extracellular <br />                                                    environment. <br />  <br />  <br />                                               Introduction <br />                                                                 Trypanosoma brucei, causative agent African sleeping sickness, expresses dense <br />                                                                 cell-surface coat consisting approximately 5 &#195;&#8212; 106 dimers variant surface glycoprotein <br />                                                                 attached plasma membrane covalent linkage                                                                 glycosylphosphatidylinositol (GPI)1 membrane anchor. coat protects parasite                                                                 alternative complement pathway host antigenic variation,  <br />  <br />                                               &#194;&#169; 2006 Authors Journal compilation &#194;&#169; 2006 Blackwell Publishing                                               * <br />                                                correspondence. T.K.Smith@dundee.ac.uk; Tel. (+44) 1382 388688; Fax (+44) 1382 345764. . <br />                                               Supplementary material <br />                                               following supplementary material available article online: <br />                                               Fig. S1. Southern blot analysis T. brucei genomic DNA, using TbINO1 ORF probe. <br />                                               Fig. S2. Deamination <span id='am-11' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-12' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="lipid" datatype="xsd:string"></span>lipid</span> samples vivo labelling. <br />                                               Fig. S3. sVSG deglycosylation analysis samples vivo labelling. <br />                                               Fig. S4. Analysis exchange reaction GPI-VSG substrate. <br />                                               Table S1. Deamination lipid samples vivo labelling. Cells labelled [3H]-glucose [3H]-mannose, lipids <br />                                               extracted, deaminated subjected butanol water partitioning. [3H]-Levels butanol water phases determined                                               scintillation spectrometry expressed percentage total cpm. <br />                                               material available online article http://www.blackwell-synergy.com <br />                                          Martin Smith                                                                                            Page 2 <br />  <br />  <br />                                                            specific immune responses (Cross, 1996). biosynthesis conserved GPI anchor <br />                                                            previously genetically (Nagamune et al., 2000; Chang et al., 2002) chemically <br />                                                            validated (Smith et al., 2004) drug target bloodstream-form T. brucei. <br />  <br />                                                            structure biosynthesis GPI membrane anchors related molecules                                                            recently reviewed (Ferguson et al., 1999; Kinoshita Inoue, 2000; McConville                                                            Menon, 2000; Morita et al., 2000a; Macedo et al., 2003). basic GPI core structure <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            attached protein comprises NH2CH2CH2PO4H-6Man&#206;&#177;1-2Man&#206;&#177;1-6Man&#206;&#177;1-4GlcN&#206;&#177;1-6D- <br />                                                            myo-inositol-1-HPO4-lipid (EtN-P-Man3GlcN-PI), lipid <span id='am-145' about='obo:IMR_0100027' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001382 obo:IMR_0000001 obo:IMR_0001362'><span id='am-146' property="rdfs:label" content="diacylglyceride" datatype="xsd:string"></span><span id='am-147' property="oboInOwl:id" content="IMR:0100027" datatype="xsd:string"></span><span id='am-148' property="oboInOwl:hasDbXref" content="KEGG:C00165" datatype="xsd:string"></span><span id='am-149' property="rdfs:label" content="DAG" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasExactSynonym" content="DAG" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="diacylglycerol" datatype="xsd:string"></span><span id='am-152' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-153' property="oboInOwl:hasExactSynonym" content="diacylglyceride" datatype="xsd:string"></span><span id='am-154' property="oboInOwl:hasExactSynonym" content="Diglyceride" datatype="xsd:string"></span><span id='am-155' property="rdfs:label" content="Diglyceride" datatype="xsd:string"></span>diacylglycerol</span>, <br />                                                            alkylacylglycerol <span id='am-171' about='obo:IMR_0100177' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0000947 obo:IMR_0001359 obo:IMR_0000001 obo:IMR_0001362'><span id='am-172' property="rdfs:label" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-173' property="rdfs:label" content="ceramide" datatype="xsd:string"></span><span id='am-174' property="rdfs:label" content="cer" datatype="xsd:string"></span><span id='am-175' property="oboInOwl:id" content="IMR:0100177" datatype="xsd:string"></span><span id='am-176' property="rdfs:label" content="C2-ceramide" datatype="xsd:string"></span><span id='am-177' property="oboInOwl:hasDbXref" content="KEGG:C00195" datatype="xsd:string"></span><span id='am-178' property="oboInOwl:hasExactSynonym" content="cer" datatype="xsd:string"></span><span id='am-179' property="oboInOwl:hasExactSynonym" content="C2-ceramide" datatype="xsd:string"></span><span id='am-180' property="oboInOwl:hasExactSynonym" content="N-Acylsphingosine" datatype="xsd:string"></span><span id='am-181' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ceramide</span> (Ferguson et al., 1999). <br />  <br />                                                            sequence events underlying GPI biosynthesis studied T. brucei <br />                                                            (Masterson et al., 1989; Masterson et al., 1990; Menon et al., 1990a,b; G&#195;&#188;ther Ferguson, <br />                                                            1995; Morita et al., 2000b), Trypanosoma cruzi (Heise et al., 1996), Toxoplasma gondii <br />                                                            (Striepen et al., 1999), Plasmodium falciparum (Gerold et al., 1999), Leishmania (Smith et <br />                                                            al., 1997; Ralton McConville, 1998; Ralton et al., 2002), Saccharomyces cerevisiae <br />                                                            (Sutterlin et al., 1998; Flury et al., 2000) mammalian cells (Hirose et al., 1992; Puoti <br />                                                            Conzelmann, 1993; Chen et al., 1998). cases, GPI biosynthesis initiated                                                            addition GlcNAc <span id='am-166' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-167' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-168' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-169' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span><span id='am-170' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span>UDP</span>-GlcNAc phosphatidylinositol (PI) GlcNAc-PI, <br />                                                            N-acetylated form GlcN-PI processed species-specific <br />                                                            manner. <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> eukaryotes synthesize PI exchange <span id='am-161' about='obo:IMR_0200494' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-162' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-163' property="oboInOwl:hasDbXref" content="KEGG:C00055" datatype="xsd:string"></span><span id='am-164' property="rdfs:label" content="CMP" datatype="xsd:string"></span><span id='am-165' property="oboInOwl:id" content="IMR:0200494" datatype="xsd:string"></span>CMP</span> moiety <span id='am-53' about='obo:IMR_0200468' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="CDP" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasDbXref" content="KEGG:C00112" datatype="xsd:string"></span><span id='am-57' property="oboInOwl:id" content="IMR:0200468" datatype="xsd:string"></span>CDP</span>- <br />                                                            diacylglycerol inositol. <br />  <br />                                                            myo-Inositol carbon cyclitol, essential metabolite eukaryotes.                                                            metabolism myo-inositol plays vital role growth regulation, signal transduction, <br />                                                            membrane biogenesis, osmotolerance essential biochemical processes,                                                            formation GPI anchors. prokaryotes inositol sparingly, example                                                            exception mycobacteria, essential biogenesis mycothiol, PI, <br />                                                            phosphatidylinositol mannosides (PIM) GPIs (Besra Chatterjee, 1994; Haites et al., <br />                                                            2005, references containing . <br />  <br />                                                            Cellular myo-inositol arise sources:  myo-inositol taken Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            extracellular environment; second, myo-inositol released dephosphorylation                                                            inositol phosphates produced turnover inositol phospholipids. Finally, myo- <br />                                                            inositol novo synthesized concerted action enzymes: 1-D-myo- <br />                                                            inositol-3-phosphate synthase (INO1) (E.C.5.5.1.4) myo-inositol monophosphatase <br />                                                            (IMPase) (E.C.3.1.3.25). rate-limiting step novo synthesis myo-inositol                                                            NAD+-dependent conversion <span id='am-85' about='obo:IMR_0200430' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-86' property="rdfs:label" content="D-Glucose 6-phosphate" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasDbXref" content="KEGG:C00092" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:id" content="IMR:0200430" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>D-glucose 6-phosphate</span> 1-D-myo-inositol-3-phosphate <br />                                                            INO1 stage reaction involving:   oxidation <span id='am-90' about='obo:IMR_0200042' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0002125'><span id='am-91' property="rdfs:label" content="D-Glucose" datatype="xsd:string"></span><span id='am-92' property="oboInOwl:hasDbXref" content="KEGG:C00031" datatype="xsd:string"></span><span id='am-93' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-94' property="oboInOwl:id" content="IMR:0200042" datatype="xsd:string"></span>D-glucose</span> 6-phosphate 5- <br />                                                            keto-D-glucose 6-phosphate, (ii) cyclization myo-2-inosose1-phosphate (iii) reduction <br />                                                            1-D-myo-inositol-3-phosphate (Majumder et al., 1997). <br />  <br />                                                            INO1s characterized prokaryotic various eukaryotic organisms                                                            Porteresia coarctata (Majee et al., 2004), Arabidopsis thaliana (Johnson Sussex, <br />                                                            1995), Mycobacterium tuberculosis (Norman et al., 2002; Movahedzadeh et al., 2004), <br />                                                            Mycobacterium smegmatis (Haites et al., 2005), Drosophila melanogaster (Park Kim, <br />                                                            2004), S. cerevisiae (Donahue Henry, 1981), Entamoeba histolytica (Lohia et al., 1999), <br />                                                            Leishmania mexicana (Ilg, 2002) Homo sapiens (Guan et al., 2003; Ju et al., 2004). <br />                                                            Genetic studies pathogenic organisms, L. mexicana (Ilg, 2002) M. <br />                                                            tuberculosis (Movahedzadeh et al., 2004), revealed INO1 important                                                            regulating intracellular myo-inositol levels, INO1 potential drug target                                                            organisms. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 3 <br />  <br />  <br />                                                            study, begin investigate novo synthesis metabolism myo-inositol <br />                                                            T. brucei. molecular cloning expression Escherichia coli                                                            T. brucei myo-inositol-3-phosphate synthase characterization recombinant <br />                                                            enzyme. report creation T. brucei conditional null mutant                                                            demonstrates novo synthesis myo-inositol essential bloodstream form <br />                                                            parasite, presence excess myo-inositol. results suggest                                                            novo synthesized myo-inositol preferentially used GPI anchor biosynthesis Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            differentiation novo synthesized myo-inositol myo-inositol taken                                                            extracellular environment. <br />  <br />                                            Results <br />                                            Cloning sequencing T. brucei INO1 <br />                                                            INO1 identified T. brucei genome database (Sanger Centre) using TBLASTN;                                                            putative open reading frame (ORF) PCR-amplified, cloned sequenced.                                                            sequence submitted GenBank, Accession  AJ86670. alignment                                                            predicted translated sequence INO1s organisms shown Fig. 1A                                                            unrooted phylogenetic tree Fig. 1B. predicted TbINO1 perfect copy motif <br />                                                            GWGGNNG (underlined Fig. 1A), involved NAD+ binding, Rossmann <br />                                                            fold typical oxidoreductase (Majumder et al., 2003). contains motifs <br />                                                            previously identified eukaryotic INO1s, LWTANTERY, <br />                                                            NGSPQNTFVPGL SYNHLGNNDG (Majumder et al., 2003). TbINO1 shows <br />                                                            considerable similarity INO1s variety organisms, closely related                                                            kinetoplastids, T. cruzi L. mexicana (Fig. 1B). size                                                            predicted TbINO1 protein 58 kDa, good agreement INO1s                                                            generally fall range 58&#226;&#8364;&#8220;67 kDa (Majumder et al., 1997; 2003). <br />  <br />                                            Overexpression INO1 E. coli <br />                                                            enable biochemical characterization T. brucei INO1 overexpressed E. coli <br />                                                            using pBAD TA vector, encodes C-terminal hexa tag. tagged <br />                                                            recombinant protein purified using Ni2+ eluted using increasing concentration Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            imidazole. purification protocol resulted pure protein preparation, shown Fig. <br />                                                            2A. apparent molecular weight recombinant protein approximately 75 kDa <br />                                                            SDS-PAGE analysis, higher predicted size 64 kDa, includes short <br />                                                            leader sequence hexa tag.  molecular weight purified <br />                                                            recombinant protein shown approximately 64 kDa MALDI analysis (data                                                            shown). <br />  <br />                                                            recombinant protein shown catalytically active inositol-3-phosphate <br />                                                            synthase use coupled assay, product INO1 activity, <br />                                                            inositol-3-phosphate, used substrate IMPase, phosphate released                                                            IMPase measured colorimetrically. study kinetic parameters glucose 6- <br />                                                            phosphate, components standard assay held constant glucose 6- <br />                                                            phosphate concentration varied 0 5 mM. Saturation kinetics shown                                                            enzyme, apparent Km 0.58 mM Vmax 0.25 &#206;&#188;mole h&#226;&#710;&#8217;1 (Fig. 2B)                                                            specific activity recombinant TbINO1 756 U mg&#226;&#710;&#8217;1. Like                                                            studied INO1 proteins, recombinant TbINO1 activity dependent NAD+ activity <br />                                                            observed absence NAD+, NAD+ replaced <span id='am-43' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span>NADH</span>                                                            <span id='am-68' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span>. activity (5% normal) observed NAD+ replaced                                                            NADP+, suggesting NADP+ act hydrogen <span id='am-140' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-141' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-142' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span><span id='am-143' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-144' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>acceptor</span>/donor                                                            reaction limited degree. Interestingly, NAD+ concentrations low <br />                                                            0.1 &#206;&#188;mole, 100% activity observed compared 1 mM, suggesting extremely <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 4 <br />  <br />  <br />                                                            efficient binding NAD+. enzyme activity T. brucei INO1 stimulated                                                            presence NH+4 reaction mixture; little activity observed                                                            absence NH4+ (10 U mg&#226;&#710;&#8217;1), compared activity presence 2 mM <br />                                                            NH4Ac (756 U mg&#226;&#710;&#8217;1), enhancement activity observed                                                            NH4Ac concentration increased 10 mM. <br />  <br />                                            INO1 essential gene T. brucei bloodstream-form cells <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Southern blot analysis T. brucei genomic DNA showed INO1 present single- <br />                                                            copy gene haploid genome (Fig. S1 Supplementary material). schematic                                                            construction conditional double knockout shown Fig. 3A. allele INO1 <br />                                                            replaced puromycin drug resistance gene homologous recombination                                                            selection puromycin, creating &#206;&#8221;INO1::PAC cell line (Fig. 3B, lane 2). Attempts                                                            create null mutant homologous replacement second allele hygromycin <br />                                                            resistance gene unsuccessful, inositol level media increased                                                            40 &#206;&#188;M (normal HMI-9 media) 100 mM.  &#226;&#8364;&#732;wild-type&#226;&#8364;&#8482; cell line used                                                            constitutively expresses T7 RNA polymerase tetracycline repressor protein,                                                            decided introduce tetracycline-inducible (Ti) myc-tagged ectopic copy INO1 <br />                                                            &#206;&#8221;INO1::PAC cell line prior deletion second allele, allowing creation                                                            conditional double knockout control tetracycline. ectopic copy <br />                                                            integrated pLew 100 vector, INO1 inserted downstream <br />                                                            trypanosome procyclin promoter tetracycline operators. Integration                                                            pLew vector known occur rDNA locus, encodes phleomycin <br />                                                            resistance protein (Wirtz et al., 1999). drug selection puromycin phleomycin, <br />                                                            INO1-mycTi &#206;&#8221;INO1::PAC clones obtained, integration pLew <br />                                                            100 ectopic copy confirmed PCR using primers specific pLew 100 vector <br />                                                            (data shown) later Southern analysis (Fig. 3B, lane 3). second allele                                                            replaced hygromycin resistance gene presence tetracycline. INO1- <br />                                                            mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG clone obtained genotype confirmed                                                            Southern blotting (Fig. 3B, lane 4). <br />  <br />                                                            establish INO1 essential gene, growth INO1-mycTi <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG cell line monitored tetracycline containing                                                            tetracycline-free HMI-9 contained 40 &#206;&#188;M myo-inositol. presence                                                            tetracycline INO1-mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG cell line showed normal growth <br />                                                            rates (Fig. 4B) compared wild-type cells (Fig. 4A).  absence                                                            tetracycline cells grew normally 2 days, cell numbers declined                                                            limits detection light microscopy increasing cellular debris observed <br />                                                            suggesting cell death (Fig. 4C). medium supplemented                                                            additional 100 mM myo-inositol, INO1 conditional knockouts unable survive                                                            tetracycline-free media (Fig. 4D).  tetracycline-free HMI-9  40 <br />                                                            &#206;&#188;M 100 mM myo-inositol) c. day 8 live cells visible light <br />                                                            microscopy, day 12 cells resumed normal growth kinetics, shown Fig. 4C. <br />                                                            Northern blot analysis (Fig. 4E) showed transcript level ectopic INO1-mycTi <br />                                                            slightly lower endogenous copy seen bloodstream-form wild- <br />                                                            type cells insect-form procyclic cells. absence tetracycline 48 h                                                            transcript INO1-mycTi undetectable (Fig. 4E, lane 4), cells                                                            spontaneously resumed normal growth absence tetracycline 12 days,                                                            transcript INO1-mycTi clearly visible (Fig. 4E, lane 5). suggests                                                            revertant cells able overcome tetracycline control, phenomena                                                            described essential genes T. brucei (Krieger et al., 2000; Milne et al., 2001; <br />                                                            Chang et al., 2002; Roper et al., 2002; 2005; Martin Smith, 2006). <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 5 <br />  <br />  <br />                                            vivo labelling wild-type cells <br />                                                            Wild-type bloodstream-form T. brucei labelled [3H]-mannose, [3H]-glucose                                                            [3H]-inositol pulse pulse-chase experiments. labelling 1 h (pulse <br />                                                            experiments) 2 h (pulse-chase experiments) samples taken protein lipid <br />                                                            analysis. <br />  <br />                                                            cells pulse labelled [3H]-mannose mature GPI glycolipids C <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            observed (Fig. 5A, lane 3), low levels galactosylated versions                                                            glycolipid  described previously (Menon et al., 1988). expected <br />                                                            pulse chase labelling showed decreased amounts [3H]-labelled glycolipids C <br />                                                            (Fig. 5, lane 4). decrease addition equal volume HMI-9 media                                                            labelling start chase longer labelling time pulse-chase <br />                                                            experiment, allowing labelled glycolipids C used GPI-VSG attachment. <br />                                                            Protein analysis confirmed [3H]-labelled glycolipds C incorporated                                                            mature GPI-anchored VSG equivalent signal observed pulse                                                            pulse-chase experiments (Fig. 5C, lanes 5 6). <br />  <br />                                                            labelled [3H]-glucose presence absence myo-inositol, <br />                                                            wild-type cells produced main lipid species (1, 2, 3) (Fig. 5A, lane 5). Lipid species 1 <br />                                                            identical Rf PI; digests showed sensitive PI-PLC PLD, indicative                                                            PI containing <span id='am-5' about='obo:IMR_0001356' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001362'><span id='am-6' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="phospholipid" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001356" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:26063" datatype="xsd:string"></span>phospholipid</span>. results base treatment, hydrogen fluoride (HF) <br />                                                            treatment deamination consistent [3H]-<span id='am-134' about='rdfs:label' typeof='owl:Thing'>label</span> head group                                                            diacylglycerol portion. confirm identity head group aqueous phase <br />                                                            HF treatment desalted, concentrated analysed high performance layer <br />                                                            chromatography (HPTLC); Rf [3H]-labelled species identical inositol <br />                                                            standard, clearly showing head group inositol (data shown). results <br />                                                            clearly existence functional INO1 bloodstream-form T. brucei, [3H]- <br />                                                            glucose converted [3H]-inositol. [3H]-inositol subsequently used                                                            cell form PI (lipid species 1). <br />  <br />                                                            Lipid species 2 [3H]-glucose labelling identical Rf glycolipid C standard <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glycolipid C obtained [3H]-mannose labellings (lanes 3 4). Lipid species 2&#226;&#8364;&#8482;s <br />                                                            insensitivity PI-PLC sensitivity PLD consistent glycolipid C. <br />                                                            Lipid species 3 identical Rf glycolipid standard glycolipid                                                            [3H]-mannose labelling (lanes 3 4). results identification lipid species 2                                                            glycolipid C suggest lipid species 3 glycolipid  Furthermore, lipids [3H]- <br />                                                            glucose labelling subjected deamination butanol-water partioning. HPTLC <br />                                                            autoradiography butanol phase revealed PI detected, glycolipids C <br />                                                            deaminated, losing glycan head group, confirming identities <br />                                                            GPIs (Fig. S2 Supplementary material).  confirm [3H]-label                                                            myo-inositol head group (like PI) glycan head group [3H]-GlcN                                                            [3H]-Man, relative amounts [3H]-label butanol aqueous phases                                                            deamination measured. shown Table S1 Supplementary material, deamination <br />                                                            [3H]-glucose labelling lipids resulted [3H]-label remaining                                                            butanol phase, showing [3H]-label remained bound lipid moiety. significant <br />                                                            [3H]-label detected aqueous phase indication [3H]- <br />                                                            label released glycan head group deamination. control, <br />                                                            [3H]-mannose-labelled glycolipids C deaminated expected                                                            [3H]-label partitioned aqueous phase [3H]-mannose glycan head <br />                                                            group (Table S1 Supplementary material), indicating deamination process                                                            working efficiently. results confirm majority [3H]-glucose                                                            incorporated GPI precursors [3H]-myo-inositol [3H]-GlcN [3H]- <br />                                                            mannose. results [3H]-glucose converted [3H]-inositol,  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 6 <br />  <br />  <br />                                                            used form [3H]-PI, subsequently utilized GPI biosynthetic pathway                                                            form [3H]-glycolipids C. Similar results observed pulse-chase [3H]- <br />                                                            mannose labelling, [3H]-glucose-labelled glycolipids C decreased <br />                                                            pulse chase, indicative [3H]-glycolipid species utilized GPI-VSG <br />                                                            attachment (Fig. 5A, lane 6). <br />  <br />                                                            Analysis protein samples [3H]-glucose labelling showed [3H]-labelled <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glycolipids C incorporated mature GPI-anchored VSG (Fig. 5C). similar <br />                                                            intensity signal observed pulse pulse-chase experiments (Fig. 5C, lanes <br />                                                            3 4). analysis lipid species 1 2 clearly shown [3H]-label <br />                                                            inositol head group, wanted confirm [3H]-label observed                                                            GPI-anchored VSG incorporation N-linked oligosaccharides present <br />                                                            VSG. confirmed methods.  labelling [3H]-glucose, <br />                                                            sVSG (VSG soluble portion GPI anchor attached) purified aliquot <br />                                                            deglycosylated PNGaseF. clear difference molecular weight sVSG <br />                                                            observed SDS-PAGE analysis, suggesting N-linked oligosaccharides                                                            successfully removed (Fig. S3A Supplementary material, lanes 1 2). <br />                                                            Autoradiography revealed significant decrease intensity [3H] <br />                                                            signal deglycosylation (Fig. S3A Supplementary material, lanes 3 4). Second, <br />                                                            conducted [3H]-glucose [3H]-mannose labellings presence                                                            tunicamycin  inhibitor N-glycosylation) [3H]-label remained GPI-anchored <br />                                                            VSG (Fig. S3B C Supplementary material). results confirm                                                            signal observed sVSG labelling [3H]-glucose N- <br />                                                            glycosylation. checked incorporation [3H]-label mature GPI- <br />                                                            anchored VSG exchange mechanism observed previously                                                            incorporation myristate (Buxbaum et al., 1996). Labelling performed presence <br />                                                            absence cycloheximide incorporation [3H]-label GPI-anchored <br />                                                            VSG determined. detect incorporation [3H]-glucose GPI- <br />                                                            VSG presence cycloheximide showing incorporation exchange <br />                                                            reaction uses GPI-VSG substrate (Fig. S4 Supplementary material).                                                            expected incorporation [3H]-myristate observed presence cycloheximide <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            described previously (Buxbaum et al., 1996). <br />  <br />                                                            surprisingly cells pulse labelled [3H]-inositol, PI lyso-PI                                                            observed (Fig. 5A, lane 1).  cells pulse chase labelled [3H]- <br />                                                            inositol additional lipid species observed identical Rf values glycolipids                                                            C respectively (Fig. 5A, lane 2). identity new lipid species                                                            glycolipids C confirmed digests incorporation [3H]-inositol <br />                                                            GPI-anchored VSG (Fig. 5C D), inositol head <br />                                                            group. appearance increase glycolipids C pulse-chase experiment <br />                                                            [3H]-inositol opposite observed labelling [3H]-glucose                                                            [3H]-mannose, levels glycolipids C decreased. parallel                                                            increase incorporation [3H]-labelled glycolipids C GPI-anchored <br />                                                            VSG pulse-chase experiments compared pulse (compare Fig. 5D <br />                                                            lanes 1 2). fact pulse labelling incorporation [3H]-inositol mature <br />                                                            GPI-anchored VSG 5% incorporation [3H]-glucose                                                            conditions (Fig. 5C D). <br />  <br />                                            Biochemical phenotype INO1 conditional knockout <br />                                                            biochemical phenotype INO1 conditional knockout cells investigated                                                            vivo pulse-labelling experiments [3H]-mannose [3H]-glucose pulse-chase <br />                                                            experiments [3H]-inositol cells grown presence                                                            absence tetracycline 2 days compared undertaken wild-type cells. <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 7 <br />  <br />  <br />                                                            labelling, lipids extracted, desalted, separated HPTLC detected                                                            fluorography. expected wild-type INO1 conditional knockout cells grown                                                            presence tetracycline showed identical incorporation [3H]-lipids  earlier                                                            identification [3H]-lipids). <br />  <br />                                                            Labelling INO1 conditional knockout cells grown absence tetracycline                                                            [3H]-glucose showed decrease [3H]-PI formed (Fig. 6, lane 3), <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            confirming deleted gene functional INO1. concomitant decrease <br />                                                            glycolipid C glycolipid  suggesting direct knock effect <br />                                                            GPI biosynthetic pathway lack PI [3H]-inositol derived <br />                                                            [3H]-glucose. knock effect GPI biosynthetic pathway confirmed                                                            [3H]-mannose labelling (Fig. 6). significant reproducible decrease                                                            amounts glycolipids C INO1 conditional knockout cells grown                                                            absence tetracycline (Fig. 6, lane 7). suggests GPI biosynthetic <br />                                                            pathway slowed significantly affect dynamic pools glycolipids C. <br />  <br />                                                            conditional double knockout cells grown absence tetracycline 2 <br />                                                            days pulse chase labelled [3H]-inositol, significant change                                                            levels PI, lyso-PI, glycolipids C observed compared wild-type cells <br />                                                            conditional double knockout cells grown presence tetracycline (Fig. 6).                                                            [3H]-inositol labellings inferred cells viable point                                                            labelling able synthesis lipids. confirm investigated <br />                                                            cells&#226;&#8364;&#8482; ability synthesize new proteins. parallel [3H]-glucose, [3H]-mannose <br />                                                            [3H]-inositol labellings described, cells labelled [35S]-methionine.                                                            labelling quenched, proteins separated SDS-PAGE labelled proteins <br />                                                            detected autoradiography. cells lines (wild type, conditional double knockout <br />                                                            tetracycline conditional double knockout tetracycline) able                                                            incorporate significant identical amounts [35S]-methionine protein (Fig. <br />                                                            6C, lanes 1, 2 3). results clearly illustrate point labelling INO1 <br />                                                            conditional knockout cells grown non-permissive conditions viable                                                            ability synthesize protein level comparable wild-type cells conditional <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            double knockout cells grown presence tetracycline.  time                                                            labelling cells viable biochemical phenotype observed direct result                                                            deletion INO1. <br />  <br />                                                            myo-inositol containing lipids cell lines (wild-type cells, <br />                                                            conditional double knockout cells grown presence absence tetracycline)                                                            quantified GC-MS. significant difference lipid-bound <br />                                                            myo-inositol cell lines (Fig. 7A), suggesting deletion INO1                                                            perturbed overall cellular PI levels. qualitatively analysed inositol containing <br />                                                            phospholipids ES-MS-MS using parent-<span id='am-37' about='obo:IMR_0001658' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-38' property="obo:IAO_0000115" content="An atom that has either gained or lost electrons to acquire a chage; for example Na+ and Cl-." datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="ion" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0001658" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="CHEBI:24870" datatype="xsd:string"></span>ion</span> scanning m 241 negative ion mode <br />                                                            (indicative collision induced fragment PI species) did detect significant <br />                                                            difference cell lines (Fig. 7B). <br />  <br />                                            INO1 cytosolic protein bloodstream-form T. brucei <br />                                                            cellular location INO1 T. brucei bloodstream-form cells investigated using                                                            tetracycline-inducible overexpression vector, pLew82 containing TbINO1 encoding <br />                                                            C-terminal haemagglutinin (HA) tag. construct introduced bloodstream-form <br />                                                            T. brucei cells, generating cell line INO1-HATi integration confirmed PCR using <br />                                                            primers specific vector (data shown). Transcription ectopic INO1-HATi <br />                                                            induced tetracycline confirmed Northern blotting (data shown). <br />                                                            Tetracycline-induced INO1-HATi cells fixed, allowed adhere poly lysine slides <br />                                                            permeablized. INO1-HATi protein detected immunofluorescence using  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                              Page 8 <br />  <br />  <br />                                                            primary antibody HA tag, secondary  FITC conjugated) <br />                                                            primary antibody. FITC signal observed cell body,                                                            seen flagella, nucleus kinetoplast (Fig. 8A&#226;&#8364;&#8220;D), suggesting TbINO1                                                            cytoplasmic. result confirmed subcellular fractionation studies (Fig. 8I),                                                            INO1-HA protein detected soluble cytosolic fraction differential <br />                                                            centrifugation. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Discussion <br />                                                            Prior results presented study, accepted field                                                            GPI anchors contain inositol, GPI intermediates mature GPIs vivo labelled <br />                                                            [3H]-inositol bloodstream-form T. brucei, despite able efficiently label PI. <br />                                                            paradox explained myo-inositol used GPIs novo synthesized <br />                                                            glucose. results presented support hypothesis, demonstrating                                                            existence functional myo-inositol novo synthesis pathway bloodstream-form T. <br />                                                            brucei, showing novo synthesis essential survival parasite                                                            disruption pathway detrimental effects GPI anchor biosynthesis,                                                            overcome use extracellular myo-inositol. results raise interesting <br />                                                            questions distinction novo synthesized myo-inositol obtained <br />                                                            extracellular sources. <br />  <br />                                                            eukaryotic cells able novo synthesize myo-inositol glucose INO1 <br />                                                            IMPase. Recombinant expression putative TbINO1 E. coli demonstrated <br />                                                            catalytically active INO1 ability convert glucose 6-phosphate                                                            inositol-3-phosphate, dependence NAD+ stimulation NH4+ consistent <br />                                                            INO1s organisms (e.g. Donahue Henry, 1981; Johnson Sussex, <br />                                                            1995; Ju et al., 2004). Saturation kinetics shown enzyme, apparent Km <br />                                                            0.58 mM; similar human INO1 Km 0.5 mM (Ju et al., <br />                                                            2004). Interestingly, specific activity recombinant TbINO1 (756 U mg&#226;&#710;&#8217;1)                                                            approximately 10-fold higher human recombinant INO1 (70 U mg&#226;&#710;&#8217;1) (Ju et <br />                                                            al., 2004), indication importance TbINO1 T. brucei. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            previously shown GPI anchor biosynthesis essential survival                                                            bloodstream-form T. brucei (Nagamune et al., 2000; Chang et al., 2002; Smith et al., 2004); <br />                                                            surmized light  INO1 novo synthesis myo-inositol <br />                                                            essential responsible providing myo-inositol synthesis                                                            PI GPI anchors. unable replace allelic copies INO1 create null <br />                                                            mutant, suggesting TbINO1 essential gene surprisingly, addition                                                            extra myo-inositol  40 &#206;&#188;M 100 mM) media did enable creation                                                            null mutant. creation conditional double knockout cessation                                                            growth non-permissive conditions, clearly demonstrated INO1                                                            essential gene bloodstream-form T. brucei. Interestingly, substantial increase                                                            myo-inositol concentration growth media (40 &#206;&#188;M 100 mM) did enable                                                            conditional knockout cells overcome deletion INO1 grown absence                                                            tetracycline (Fig. 4D). inability overcome deletion INO1 surprising                                                            unexpected contrast &#206;&#8221;INO1 mutants date, e.g. S. cerevisiae (Henry <br />                                                            et al., 1977), L. mexicana (Ilg, 2002), M. tuberculosis (Movahedzadeh et al., 2004) <br />                                                            require 27 &#206;&#188;M, 200 &#206;&#188;M 77 mM extracellular myo-inositol respectively. clear <br />                                                            difference T. brucei INO1 conditional mutant null INO1 mutants                                                            organisms intriguing probed biochemical phenotyping INO1 <br />                                                            conditional double knockout cells grown non-permissive conditions. <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                               Page 9 <br />  <br />  <br />                                                            surprise vivo labelling wild-type cells time possible <br />                                                            label glycolipids C [3H]-inositol.  able                                                            labelled myo-inositol incorporated GPI-anchored VSG,                                                            significantly lower level [3H]-mannose [3H]-glucose.                                                             able observe labelling cells pulse chase labelled. increase                                                            labelling GPIs intermediates chase pulse chase labelling quite <br />                                                            surprising opposite observed labelling mannose Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            glucose, labelling glycolipids C actually decrease chase                                                            used GPI-VSG attachment. results suggest extracellular myo-inositol <br />                                                            entering GPI biosynthetic pathway slower rate novo <br />                                                            synthesized myo-inositol mannose. latency explained recycling,                                                            extracellular inositol PI synthesized extracellular inositol relocated cell <br />                                                            results mixing types myo-inositol/PI. results suggest                                                            novo synthesized myo-inositol primary source GPI synthesis extracellular <br />                                                            inositol secondary source. observation supported twofold:  vivo <br />                                                            labelling showed GPI biosynthesis slowed conditional double <br />                                                            knockout cells grown non-permissive conditions lack novo synthesized <br />                                                            inositol/PI, concomitant increase levels extacellular inositol feeding <br />                                                            GPI pathway. Second, significant change total cellular levels                                                            lipid-based myo-inositol wild-type cells conditional double knockout cells <br />                                                            grown non-permissive conditions.  results presented clearly demonstrate <br />                                                            deletion INO1 causing &#226;&#8364;&#732;inositol death&#226;&#8364;&#8482;, intriguingly cells <br />                                                            dying decrease novo synthesized myo-inositol, particular <br />                                                            function appears supplying GPI pathway myo-inositol/PI.                                                             results explain additional myo-inositol media did allow creation                                                            null mutant did enable conditional double knockout cells overcome                                                            deletion INO1 grown absence tetracycline. <br />  <br />                                                            clear preference use novo synthesized GPI anchors biosynthesis suggests                                                            distinction compartmentalization types myo-inositol (extracellular                                                            novo synthesized). probe idea started investigating location/s Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            enzymes novo synthesis pathway bloodstream-form T. brucei. Previously                                                            shown sole PI synthase (PIS) responsible synthesis bulk cellular PI                                                            PI destined GPI anchor synthesis located Golgi                                                            endoplasmic reticulum (ER) (Martin Smith, 2006) shown                                                            TbINO1 localized cytosol, consistent INO1s                                                            organisims. Based observations propose following:  extracellular myo- <br />                                                            inositol utilized Golgi PIS make bulk cellular PI. Second, TbINO1 catalyses                                                            conversion glucose 6-phosphate inositol-3-phosphate cytoplasm, presence                                                            phosphate group distinguishes novo synthesized myo-inositol acquired <br />                                                            extracellular sources. inositol-3-phosphate transported ER, <br />                                                            dephosphorylated IMPase used ER PIS form PI, subsequently <br />                                                            used GPI biosynthetic machinery located ER  Ferguson, 1999,                                                            references contained . evidence supporting hypothesis                                                            identification putative IMPase (GeneDB annotation) appears located                                                            ER (K.L. Martin T.K. Smith, unpublished). knowledge, report <br />                                                            organism differentiation novo synthesized extracellular myo- <br />                                                            inositol current efforts focused understanding complex mechanisms                                                            phenomena  physiological reason preference novo <br />                                                            synthesized myo-inositol remains unclear; evolutionary adaptation guarantee <br />                                                            parasites sustain high level GPI anchor production, ensuring integrity <br />                                                            cell surface protective coat VSG, enabling evasion hosts&#226;&#8364;&#8482; immune  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                         Page 10 <br />  <br />  <br />                                                            summary, results presented clearly demonstrate bloodstream-form T. brucei <br />                                                            dependent novo synthesis myo-inositol. dependence confirmed <br />                                                            genetically perturbation growth INO1 conditional double mutants grown <br />                                                            tetracycline-free media. Furthermore, results suggest compartmentalization PI <br />                                                            synthesis, reported organism date. Surprisingly,                                                            appears compensatory mechanisms extracellular myo-inositol <br />                                                            replace novo synthesized myo-inositol novo synthesis compromised, <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            strongly validating INO1 novo synthesis myo-inositol bloodstream- <br />                                                            form T. brucei potential therapeutic drug targets. Currently, investigating <br />                                                            compounds interfere novo synthesis inositol drug leads African <br />                                                            sleeping sickness related neglected diseases. <br />  <br />                                            Experimental procedures <br />                                            Cloning T. brucei INO1 gene ligation pLew vectors <br />                                                            putative INO1 gene identified T. brucei genome database (Sanger Centre) <br />                                                            using TBLASTN search S. cerevisiae INO1 genomic sequence query. ORF <br />                                                            amplified T. brucei genomic DNA using forward reverse primers 5&#226;&#8364;&#178;- <br />                                                            GAGGAGAAGCTTATGCCAGCCGTCCGTACG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            TCGTTAATTAAACTTCCCACGCCGCGAAG-3&#226;&#8364;&#178; containing HindIII PacI restriction <br />                                                            sites respectively (underlined). band expected size ~1.5 kb amplified using <br />                                                            Pfu polymerase, purified using QIAquick PCR purification kit (Qiagen) cloned                                                            pCR-Blunt II TOPO (Invitrogen). TbINO1 ORF excised using HindIII PacI <br />                                                            subsequently ligated tetracycline-inducible expression vectors pLew82                                                            pLew100 (Wirtz et al., 1999) HindIII PacI restriction sites. <br />  <br />                                            Construction T. brucei gene replacement cassettes <br />                                                            5&#226;&#8364;&#178; 3&#226;&#8364;&#178; untranslated regions (UTRs) immediately adjacent INO1 ORF                                                            amplified T. brucei genomic DNA using Pfu polymerase. primers 5&#226;&#8364;&#178;- <br />                                                            ATAAGAATGCGGCCGCAATGGCTGTTGTGGCACAAGG-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            GGATCCGTTTAAACTTACGGACCGTCAAGCTTGTACCACAACACGAATCTTCAG <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            G-3&#226;&#8364;&#178; used 5&#226;&#8364;&#178; UTR, amplifying expected 398 bp product primers 5&#226;&#8364;&#178;- <br />                                                            AAGCTTTAAACGGATCCGTAAGTTTAAACGGATCCTGAATGTAACCGTCTACTT <br />                                                            CGTTCCGC-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            ATAAGAATGCGGCCGCATGAAAAGAAAACATCGGGGAAT-3&#226;&#8364;&#178; resulted                                                            correct 356 bp product 3&#226;&#8364;&#178; UTR. amplified products used knitting <br />                                                            PCR, resulting 744 bp product 5&#226;&#8364;&#178; UTR joined 3&#226;&#8364;&#178; UTR short <br />                                                            BamHI&#226;&#8364;&#8220;HindIII linker region contained described primers (italics) NotI site <br />                                                            (underlined) end. PCR product ligated pGEM-5Zf(+) (Promega)                                                            NotI sites hygromycin phosphotransferase (HYG) puromycin <br />                                                            acetyltransferase (PAC) resistance genes ligated BamHI HindIII <br />                                                            restriction sites. Plasmid DNA prepared using QIAprep Miniprep Plasmid Kit <br />                                                            (Qiagen); digestion NotI, precipitated sodium acetate/ethanol                                                            redissolved sterile water final concentration 1 &#206;&#188;g &#206;&#188;l&#226;&#710;&#8217;1. <br />  <br />                                            Expression T. brucei INO1 E. coli <br />                                                            INO1 ORF amplified TbINO1-TOPO plasmid Taq polymerase using <br />                                                            primers 5&#226;&#8364;&#178;-CGG GATCCCGGGATGCCAGCCGTCCGTACGAAAAGT-3&#226;&#8364;&#178; 5&#226;&#8364;&#178;- <br />                                                            CCTTTCCTTCGCGGCGTGGGAAGTAGGCCGCTCGA GCGGC-3&#226;&#8364;&#178; resulting single <br />                                                            band ~1.5 kb. amplified product purified using QIAquick PCR purification kit <br />                                                            (Qiagen) ligated pBAD TOPO TA (Invitrogen) vector sequence confirmed. <br />                                                            construct transformed TOP10 competent cells (Invitrogen) expression.  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                          Page 11 <br />  <br />  <br />                                                            single colony used inoculate 200 ml Luria&#226;&#8364;&#8220;Bertani media containing 100 &#206;&#188;g ml&#226;&#710;&#8217;1 <br />                                                            ampicillin grown 37&#194;&#176;C OD (600 nm) 0.5. cells induced 0.2% <br />                                                            arabinose grown 37&#194;&#176;C 3 h, cells harvested                                                            centrifugation stored &#226;&#710;&#8217;20&#194;&#176;C. <br />  <br />                                            Purification recombinant INO1 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Frozen pellets resuspended lysis buffer (20 mM Tris pH 8.0, 300 mM NaCl)                                                            incubated ice 0.2 mg ml&#226;&#710;&#8217;1 lysozyme (Sigma) 30 min. Cells disrupted                                                            sonication (3 &#195;&#8212; 30 s) ice supernatant cleared centrifugation (50 000 g, 30 <br />                                                            min, 4&#194;&#176;C). supernatant filtered (0.45 &#206;&#188;m) prior loading Hi-Trap Chelating <br />                                                            Sepharose HP column (Amersham) charged Ni2+ equilibrated                                                            lysis buffer. column connected AKTA FPLC (Amersham) bound protein <br />                                                            eluted buffer containing 0.5 M imidazole, using linear gradient 0&#226;&#8364;&#8220;0.25 M,                                                            flow rate 1 ml min&#226;&#710;&#8217;1 collecting 1 ml fractions. Fractions containing TbINO1 <br />                                                            protein pooled dialysed 20 mM Tris pH 8.0, 0.05 mM DTT 24 h                                                            regular buffer changes. purity recombinant INO1 checked SDS-PAGE <br />                                                            (10% gel). dialysed sample adjusted 15% glycerol stored &#226;&#710;&#8217;80&#194;&#176;C. <br />  <br />                                            Enzyme assays <br />                                                            assay used INO1 activity based described Eisenberg                                                            Parthasarathy, (1987). Briefly, standard assay reaction consisted 50 mM Tris-Ac pH <br />                                                            8.0, 5 mM glucose 6-phosphate, 1 mM DTT, 1 mM NAD+, 2 mM NH4Ac 5 &#206;&#188;g                                                            purified protein total volume 150 &#206;&#188;l. assay incubated 37&#194;&#176;C 1 h,                                                            assay stopped heating 100&#194;&#176;C 5 min. determine glucose 6- <br />                                                            phosphate converted inositol-1-phosphate, 1.25 mU IMPase                                                            bovine brain (Sigma) 4 mM MgCl2 added total 50 &#206;&#188;l incubated 37&#194;&#176;C <br />                                                            1 h; assay stopped heating 100&#194;&#176;C 5 min. IMPase negligible <br />                                                            activity glucose 6-phosphate, resulted quantitative release phosphate <br />                                                            inositol-1-phosphate formed solely action TbINO1.                                                            assay substrate blank enzyme blank incubated IMPase account <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            free phosphates present sample non-specific activity                                                            IMPase. free phosphate determined malachite green method Itaya Ui <br />                                                            (1966) compared standard curve KH2PO4. Specific activity defined units <br />                                                            (U) milligram protein, 1 U enzyme catalysing formation                                                            1 nmole inositol-1-phosphate min 37&#194;&#176;C. <br />  <br />                                            Southern Northern blots <br />                                                            INO1 ORF PCR-amplified using primers described previously ligation <br />                                                            pLew vectors gel-purified QIAquick gel extraction kit (Qiagen).                                                            fragment labelled fluorescein (Gene ImagesRandom prime module, <br />                                                            Amersham) Southern blotting [&#206;&#177;-32P]-<span id='am-135' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-136' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-137' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-138' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span><span id='am-139' property="rdfs:label" content="dCTP" datatype="xsd:string"></span>dCTP</span> (RediprimeII random prime labelling <br />                                                             Amersham) Northern blotting. <br />  <br />                                                            Southern blots genomic T. brucei DNA (2 &#206;&#188;g) digested various restriction <br />                                                            enzymes, digestion products separated 0.8% agarose gel transferred                                                            Hybon-N membrane (Amersham). membrane hybridized overnight ULTRA- <br />                                                            HYB (Ambion) 42&#194;&#176;C fluorescein labelled INO1 ORF probe. Stringency washes <br />                                                            carried 42&#194;&#176;C, consisted washes low stringency 15 min (2&#195;&#8212; <br />                                                            SSC, 0.1% SDS) washes high stringency 15 min (0.2&#195;&#8212; SSC, 0.1% <br />                                                            SDS). Bound probe detected using CDP-STAR detection module (Amersham)                                                            autoradiography. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                            Page 12 <br />  <br />  <br />                                                            Northern blots total RNA purified using RNeasy Mini Kit (Qiagen) separated <br />                                                            <span id='am-124' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-125' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>formaldehyde</span> gel transferred Hybon N+ (Amersham). Hybridization conditions <br />                                                            stringency washes described Southern blotting, using 32P-labelled INO1 <br />                                                            ORF probe. Bound probe detected autoradiography. <br />  <br />                                            Cultivation genetic modification T. brucei <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Bloodstream-form T. brucei strain 427, previously modified express                                                            T7 polymerase tetracycline repressor protein (Wirtz et al., 1999), referred                                                            wild-type cells convenience. Cells grown HMI-9 media supplemented                                                            G418 (2.5 &#206;&#188;g ml&#226;&#710;&#8217;1), 37&#194;&#176;C 5% <span id='am-48' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-51' property="rdfs:label" content="CO2" datatype="xsd:string"></span><span id='am-52' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CO2</span> described (Wirtz et al., 1999; Milne <br />                                                            et al., 2001; Chang et al., 2002; Roper et al., 2002). Transformation conditions                                                            subsequent drug selection described (Wirtz et al., 1999; Milne et al., <br />                                                            2001; Chang et al., 2002; Roper et al., 2002). experiments requiring tetracycline-free <br />                                                            conditions, Tet approved fetal calf serum (Clonetech) used. tetracycline <br />                                                            added media, final concentration 1 &#206;&#188;g ml&#226;&#710;&#8217;1 used. tetracycline-free <br />                                                            experiments, cells washed times tetracycline-free HMI-9 resuspended                                                            tetracycline-free media 5 &#195;&#8212; 104 cells ml&#226;&#710;&#8217;1. Cells counted day                                                            passaged density 2 &#195;&#8212; 106 3 &#195;&#8212; 106 (normally                                                            second day). <br />  <br />                                            Subcellular localization studies <br />                                                            Mid-log T. brucei INO1-HATi bloodstream-form cells grown                                                            presence tetracycline 2 days harvested centrifugation (800 g, 10 min)                                                            used immunofluorescence differential centrifugation. immunofluorescence, <br />                                                            cells fixed 4% paraformaldehyde (4&#194;&#176;C, overnight). aliquot fixed <br />                                                            cells placed poly lysine slide allowed air dry. Cells rehydrated                                                            PBS washed PBS-<span id='am-100' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-101' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-106' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-109' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-110' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-111' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span>glycine</span> (0.1 M) permeablization Triton X-100 <br />                                                            (0.1%). cells blocked 1% BSA PBS incubation primary <br />                                                            monoclonal antibody; rat anti-HA (Roche) secondary antibody; FITC-conjugated <br />                                                            rabbit anti-rat immunoglobulins (DakoCytomation). <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            differential centrifugation, cells (1 &#195;&#8212; 108) lysed ice-cold water (100 &#206;&#188;l) <br />                                                            containing leupeptin (2 &#206;&#188;g ml&#226;&#710;&#8217;1) TLCK (0.1 &#206;&#188;M) 15 min ice, mechanical <br />                                                            disruption cells addition glass beads (425&#226;&#8364;&#8220;600 &#206;&#188;m, Sigma), vortexing <br />                                                            vigorously 3 &#195;&#8212; 30 s, 2 min intervals ice. Glass beads removed                                                            centrifugation (500 g, 5 min). Differential centrifugation described previously (Sarkar <br />                                                            et al., 2003). Briefly, unbroken cells nuclei removed centrifugation 1000 g <br />                                                            10 min. supernatant removed centrifuged 14 500 g 10 min remove <br />                                                            large granular fraction. resulting supernatant removed centrifuged 140 <br />                                                            000 g 1 h, supernatant contained cytosolic fraction pellet contained small <br />                                                            granular fraction. pellets resuspended trypanosome dilution buffer (TDB) (25 <br />                                                            mM KCl, 400 mM NaCl, 5 mM MgSO4, 100 mM Na2HPO4, NaH2PO4, 100 mM glucose) <br />                                                            equal volume cytosolic fraction. Proteins separated SDS-PAGE <br />                                                            transferred ECL-Nylon membrane Western blotting. blocking overnight                                                            PBS-5% skim milk powder, protein detected using primary monoclonal antibody; <br />                                                            rat anti-HA (Roche) secondary antibody; horseradish peroxidase-conjugated rabbit anti- <br />                                                            rat immunoglobulins (Jackson) ECL western detection reagents. <br />  <br />                                            vivo T. brucei metabolic labelling <br />                                                            metabolic labelling, 2 &#195;&#8212; 107 mid-log cells centrifuged (800 g, 10 min), <br />                                                            washed glucose-free minimal essential media, resuspended                                                            media final concentration 1 &#195;&#8212; 107 cells ml&#226;&#710;&#8217;1. Cells labelled 1 h (pulse) 2 h <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                          Page 13 <br />  <br />  <br />                                                            (pulse chase) 37&#194;&#176;C 50 &#206;&#188;Ci ml&#226;&#710;&#8217;1 D-[2-3H]-glucose (30 Ci mmol&#226;&#710;&#8217;1, ARC), D- <br />                                                            [2-3H]-mannose (14 Ci mmol&#226;&#710;&#8217;1, Amersham) D-[2-3H]-inositol (30 Ci mmol&#226;&#710;&#8217;1, <br />                                                            Amersham) shaking water bath. pulse-chase experiments, 1 h labelling                                                            equal volume normal media (HMI-9) added labelling allowed continue                                                            hour. cells collected centrifugation (800 g, 10 min) samples taken <br />                                                            lipid protein analysis. Lipids extracted using chloroform:methanol:water <br />                                                            (10:10:3 v/v) 1 h, supernatant removed pellet extracted Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            chloroform:methanol (2:1 v/v) 1 h. supernantants pooled dried                                                            samples desalted using butanol/water partioning. Lipids separated HPTLC <br />                                                            using silica 60 HPTLC plates chloroform:methanol:water (10:10:3 v/v) solvent. <br />                                                            Radiolabelled lipids detected fluorography &#226;&#710;&#8217;70&#194;&#176;C, spraying                                                            En3hance&#226;&#8222;&#162; using Kodak XAR-5 film intensifying screen,                                                            quantification required HPTLC plates imaged using Fuji FLA-2000 <br />                                                            phosphorimager. analyse proteins SDS-PAGE sample buffer added cell pellet <br />                                                            heated 95&#194;&#176;C 5 min. Proteins separated 10% SDS-PAGE gel                                                            visualized Coomassie blue staining. detect [3H]-labelled proteins destained gel <br />                                                            soaked En3hance&#226;&#8222;&#162; (NEN) 30 min, washed water twice, soaked 10% <br />                                                            glycerol dried exposed XAR-5 film &#226;&#710;&#8217;70&#194;&#176;C. Small-scale sVSG purification                                                            PNGase F (Calbiochem) performed described previously (Jones et al., 2005). <br />  <br />                                                            labelling [35S]-methionine, 1 &#195;&#8212; 107 mid-log cells collected                                                            centrifugation, washed methionine-free minimal essential media resuspended                                                            media final concentration 1 &#195;&#8212; 107 cells ml&#226;&#710;&#8217;1. cells labelled 30 min <br />                                                            20 &#206;&#188;Ci [35S]-methionine (MP Biomedicals, 1175 Ci mmol&#226;&#710;&#8217;1) 37&#194;&#176;C. quench                                                            labelling, cells diluted 20 ml cold TDB containing 1 mM <span id='am-156' about='obo:IMR_0200488' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-157' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-158' property="oboInOwl:id" content="IMR:0200488" datatype="xsd:string"></span><span id='am-159' property="oboInOwl:hasDbXref" content="KEGG:C00073" datatype="xsd:string"></span><span id='am-160' property="rdfs:label" content="L-Methionine" datatype="xsd:string"></span>L-methionine</span>                                                            centrifuged (800 g, 10 min, 4&#194;&#176;C). supernatant removed, cells                                                            resuspended TDB equal volume 2&#195;&#8212; SDS-PAGE sample buffer added prior <br />                                                            heating 100&#194;&#176;C 5 min. Proteins separated 10% SDS-PAGE gel                                                            visualized Coomassie blue staining. detect [35S]-labelled proteins destained gel <br />                                                            soaked En3hance&#226;&#8222;&#162; (NEN) 30 min, washed water twice, soaked 10% <br />                                                            glycerol dried. dried gel exposed XAR-5 film overnight &#226;&#710;&#8217;70&#194;&#176;C. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            Enzymatic digests <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> characterization radiolabelled lipid/glycolipid species <br />                                                            Digestion GPI-specific phospholipase D (GPI-PLD, Glyco) PI-specific <br />                                                            phospholipase C (PI-PLC, Glyco), deamination, base hydrolysis HF treatment                                                            described previously (G&#195;&#188;ther et al., 1994; G&#195;&#188;ther Ferguson, 1995; Smith et al., 1996). <br />                                                            deamination, reactions subjected butanol/water partitioning total [3H] cpm <br />                                                            fractions determined scintillation spectrometry using Beckman LS6000E <br />                                                            formula 989 scintillation fluid (Packard Biosciences). HPTLC detection                                                            described vivo T. brucei metabolic labelling. <br />  <br />                                            Inositol analysis <br />                                                            Mid-log cells collected centrifugation (800 g, 10 min) washed TDB stored <br />                                                            &#226;&#710;&#8217;20&#194;&#176;C. Lipids extracted samples addition 500 &#206;&#188;l chloroform <br />                                                            methanol mixture (2:1 v/v) incubated room temperature 1 h. supernatant                                                            removed pellet extracted chloroform:methanol:water mixture (1:2:0.8 v/v). <br />                                                            Pooled supernatants dried nitrogen prior desalting biphasic partitioning <br />                                                            using 2:1 butanol:water (v/v). internal standard D6-myo-inositol added samples <br />                                                            prior hydrolysis strong acid (6 M HCl, 110&#194;&#176;C), derivitization TMS analysis <br />                                                            GC-MS, according method Ferguson (1993). myo-Inositol quantified                                                            means separate analyses determined. <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                    Page 14 <br />  <br />  <br />                                            Supplementary Material <br />                                                            Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                            Acknowledgments <br />                                                            work supported Wellcome Trust Senior Research Fellowship 067441. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            References <br />                                                            Besra, GS.; Chatterjee, D. Lipids carbohydrates Mycobacterium tubercolosis.  Broom, BR., <br />                                                               editor. Tuberculosis: Pathogenesis, Protection Control. American Society Microbiology; <br />                                                               Washington, DC: 1994. p. 285-306. <br />                                                            Buxbaum LU, Milne KG, Werbovetz KA, Englund PT. Myristate exchange Trypanosoma <br />                                                               brucei variant surface glycoprotein. Proc Natl Acad Sci USA. 1996; 93:1178&#226;&#8364;&#8220;1183. [PubMed: <br />                                                               8577736] <br />                                                            Chang T, Milne KG, G&#195;&#188;ther MLS, Smith TK, Ferguson MAJ. Cloning Trypansoma brucei                                                               Leishmania major genes encoding GlcNAc-phoshatidylinositol N-acetylase                                                               glycosylphosphatidylinositol biosynthesis essential African sleeping sickness parasite. <br />                                                               J Biol Chem. 2002; 277:50176&#226;&#8364;&#8220;50182. [PubMed: 12364327] <br />                                                            Chen R, Walter EI, Parker G, Lapurga JP, Millan JL, Ikehara Y, et al. Mammalian <br />                                                               glycophosphatidylinositol anchor transfer proteins posttransfer deacylation. Proc Natl Acad <br />                                                               Sci USA. 1998; 95:9512&#226;&#8364;&#8220;9517. [PubMed: 9689111] <br />                                                            Cross GAM. Antigenic variation trypanosomes: secrets surface slowly. Bioessays. 1996; 18:283&#226;&#8364;&#8220; <br />                                                               287. [PubMed: 8967896] <br />                                                            Donahue TF, Henry SA. Myo-inositol-1-phosphate synthase. J Biol Chem. 1981; 256:7077&#226;&#8364;&#8220;7085. <br />                                                               [PubMed: 7016881] <br />                                                            Eisenberg FJ, Parthasarathy R. Measurement biosynthesis myo-inositol glucose-6- <br />                                                               phosphate. Methods Enzymol. 1987; 141:127&#226;&#8364;&#8220;143. [PubMed: 3600356] <br />                                                            Ferguson, MAJ. GPI membrane anchors: isolation analysis.  Fukuda, M.; Kobata, , editors. <br />                                                               Glycobiology: Practical Approach. IRL Press; Oxford: 1993. p. 349-383. <br />                                                            Ferguson MAJ. structure, biosynthesis functions glycosylphosphatidylinositol anchors, Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               contributions trypanosome research. J Cell Sci. 1999; 112:2799&#226;&#8364;&#8220;2809. [PubMed: 10444375] <br />                                                            Ferguson MAJ, Brimacombe JS, Brown JR, Crossman  Dix  Field RA, et al. GPI biosynthetic <br />                                                                 pathway therapeutic target African sleeping sickness. Biochim Biophys Acta. 1999; <br />                                                                 1455:327&#226;&#8364;&#8220;340. [PubMed: 10571022] <br />                                                            Flury  Benachour  Conzelmann  L031c belongs novel family membrane proteins involved <br />                                                                 transfer ethanolaminephosphate core structure glycosylphosphatidylinositol <br />                                                                 anchors yeast. J Biol Chem. 2000; 275:24458&#226;&#8364;&#8220;24465. [PubMed: 10823837] <br />                                                            Gerold P, Jung N, Azzouz N, Freiberg N, Kobe S, Schwarz RT. Biosynthesis                                                                 glycosylphosphatidylinositols Plasmodium falciparum cell-free incubation  inositol <br />                                                                 acylation needed mannosylation glycosylphosphatidylinositols. Biochem J. 1999; <br />                                                                 344:731&#226;&#8364;&#8220;738. [PubMed: 10585859] <br />                                                            Guan G, Dai P, Shechter  cDNA cloning gene expression analysis human myo-inositol 1- <br />                                                                 phosphate synthase. Arch Biochem Biophys. 2003; 417:251&#226;&#8364;&#8220;259. [PubMed: 12941308] <br />                                                            G&#195;&#188;ther MLS, Ferguson MAJ. role inositol acylation inositol deacylation GPI <br />                                                                 biosynthesis Trypanosoma brucei. EMBO J. 1995; 14:3080&#226;&#8364;&#8220;3093. [PubMed: 7621823] <br />                                                            G&#195;&#188;ther MLS, Masterson WJ, Ferguson MAJ. effects phenylmethylsulfonyl fluoride inositol- <br />                                                                 acylation fatty acid remodelling African trypanosomes. J Biol Chem. 1994; 269:18694&#226;&#8364;&#8220; <br />                                                                 18701. [PubMed: 7518442] <br />                                                            Haites  Morita YS, McConville MJ, Billman-Jacobe H. Function phosphatidylinositol                                                                 mycobacteria. J Biol Chem. 2005; 280:10981&#226;&#8364;&#8220;10987. [PubMed: 15634688] <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                  Page 15 <br />  <br />  <br />                                                            Heise N, Raper J, Buxbaum LU, Peranovich TM, Almeida ML. Identification complete <br />                                                                 precursors glycosylphosphatidylinositol protein anchors Trypanosoma cruzi. J Biol <br />                                                                 Chem. 1996; 271:16877&#226;&#8364;&#8220;16887. [PubMed: 8663209] <br />                                                            Henry SA, Atkinson KD, Kolat AL, Culbertson MR. Growth metabolism inositol-starved <br />                                                                 Saccharomyces cerevisiae. J Bacteriol. 1977; 68:2888&#226;&#8364;&#8220;2898. <br />                                                            Hirose S, Prince GM, Sevlever D, Ravi L, Rosenberry TL, Ueda E, Medof  Characterization                                                                 putative glycoinositol phospholipid anchor precursors mammalian cells. Localization Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                 phosphoethanolamine. J Biol Chem. 1992; 267:16968&#226;&#8364;&#8220;16974. [PubMed: 1380957] <br />                                                            Ilg T. Generation myo-inositol-auxotrophic Leishmania mexicana mutants targeted replacement <br />                                                                 myo-inositol-1-phosphate synthase gene. Mol Biochem Parasitol. 2002; 120:151&#226;&#8364;&#8220;156. <br />                                                                 [PubMed: 11849714] <br />                                                            Itaya K, Ui M. new micromethod colorimetric determination inorganic phosphate. Clinica <br />                                                                 Chimica Acta. 1966; 14:361&#226;&#8364;&#8220;366. <br />                                                            Johnson MD, Sussex IM. 1l-myo-inositol 1-phosphate synthase Arabidopsis thaliana. Plant <br />                                                                 Physiol. 1995; 107:613&#226;&#8364;&#8220;619. [PubMed: 12228386] <br />                                                            Jones DC, Mehlert  G&#195;&#188;ther ML, Ferguson MAJ. Deletion glucosidase II gene                                                                 Trypanosoma brucei reveals novel N-glycosylation mechanisms biosynthesis variant <br />                                                                 surface glycoprotein. J Biol Chem. 2005; 280:35929&#226;&#8364;&#8220;35942. [PubMed: 16120601] <br />                                                            Ju S, Shaltiel G, Shamir  Agam G, Greenberg ML. Human <span id='am-63' about='obo:IMR_0200349' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C04006" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-66' property="rdfs:label" content="1D-myo-Inositol 3-phosphate" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0200349" datatype="xsd:string"></span>1d-myo-inositol 3-phosphate</span> synthase                                                                 functional yeast. J Biol Chem. 2004; 279:21759&#226;&#8364;&#8220;21765. [PubMed: 15024000] <br />                                                            Kinoshita T, Inoue N. Dissecting manipulating pathway glycosylphosphatidylinositol- <br />                                                                 anchor biosynthesis. Curr Opin Chem Biol. 2000; 4:632&#226;&#8364;&#8220;638. [PubMed: 11102867] <br />                                                            Krieger S, Schwarz W, Ariyanayagam MR, Fairlamb AH, Krauth-Siegel RL, Clayton C. <br />                                                                 Trypanosomes lacking trypanothione reductase avirulent increased sensitivity                                                                 oxidative stress. Mol Microbiol. 2000; 35:542&#226;&#8364;&#8220;552. [PubMed: 10672177] <br />                                                            Lohia  Hait NC, Majumder AL. 1-myo-Inositol 1-phosphate synthase Entamoeba histolytica. <br />                                                                 Mol Biochem Parasitol. 1999; 98:67&#226;&#8364;&#8220;79. [PubMed: 10029310] <br />                                                            Macedo CS, Shams-Eldin H, Smith TK, Schwarz RT, Azzouz N. Inhibitors glycosyl- <br />                                                                 phosphatidylinositol anchor biosynthesis. Biochimie. 2003; 85:465&#226;&#8364;&#8220;472. [PubMed: 12770785] <br />                                                            McConville MJ, Menon AK. Recent developments cell biology biochemistry                                                                 glycosylphosphatidylinositol anchors. Mol Membr Biol. 2000; 17:1&#226;&#8364;&#8220;16. [PubMed: 10824734] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Majee M, Maitra S, Dastidar KD, Pattnaik S, Chatterjee  Hait NC, et al. novel salt-tolerant 1-myo- <br />                                                                 inositol-1-phosphate synthase form Porteresia coarctata (Roxb.) Tateoka, halophytic wild rice. J <br />                                                                 Biol Chem. 2004; 279:28539&#226;&#8364;&#8220;28552. [PubMed: 15016817] <br />                                                            Majumder AL, Johnson MD, Henry SA. Chapter XIX 1l-myo-Inositol-1-phosphate synthase. Biochim <br />                                                                 Biophys Acta. 1997; 1348:245&#226;&#8364;&#8220;256. [PubMed: 9370339] <br />                                                            Majumder AL, Chatterjee  Dastidar KD, Majee M. Diversification evolution 1-myo-inositol <br />                                                                 1-phosphate synthase. FEBS Lett. 2003; 553:3&#226;&#8364;&#8220;10. [PubMed: 14550537] <br />                                                            Martin KL, Smith TK. Phosphatidylinositol synthesis essential bloodstream form T. brucei. <br />                                                                 Biochem J. 2006; 396:287&#226;&#8364;&#8220;295. [PubMed: 16475982] <br />                                                            Masterson WJ, Doering TL, Hart GW, Englund PT. novel pathway glycan assembly: <br />                                                                 biosynthesis glyosyl-phosphatidylinositol anchor trypanosome variant surface <br />                                                                 glycoprotein. Cell. 1989; 56:793&#226;&#8364;&#8220;800. [PubMed: 2924349] <br />                                                            Masterson WJ, Raper J, Doering TL, Hart GW, Englund PT. Fatty acid remodeling: novel reaction <br />                                                                 sequence biosynthesis trypanosome glycosyl-phosphatidylinositol membrane anchors. <br />                                                                 Cell. 1990; 62:73&#226;&#8364;&#8220;80. [PubMed: 1694728] <br />                                                            Menon AK, Major S, Ferguson MAJ, Duszenko M, Cross GAM. Candidate glycophospholipid <br />                                                                 precursor glycosylphosphatidylinositol membrane anchor Trypanosoma brucei variant <br />                                                                 surface glycoproteins. J Biol Chem. 1988; 263:1970&#226;&#8364;&#8220;1977. [PubMed: 3339000] <br />                                                            Menon AK, Mayor S, Schwarz RT. Biosynthesis glycosyl-phosphatidylinositol lipids                                                                 Trypanosoma brucei: involvement mannosyl-phosphoryldolichol mannose donor. EMBO <br />                                                                 J. 1990a; 9:4249&#226;&#8364;&#8220;4258. [PubMed: 2148289] <br />                                                            Menon AK, Schwarz RT, Mayor S, Cross GAM. Cell-free synthesis glycosyl-phosphatidylinositol <br />                                                                 precursors glycolipid membrane anchor variant surface glycoproteins. Structural <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                                 Page 16 <br />  <br />  <br />                                                                characterization putative biosynthetic intermediates. J Biol Chem. 1990b; 265:9033&#226;&#8364;&#8220;9042. <br />                                                                [PubMed: 1693147] <br />                                                            Milne KG, G&#195;&#188;ther MLS, Ferguson MAJ. <span id='am-58' about='obo:IMR_0200065' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0200065" datatype="xsd:string"></span><span id='am-61' property="rdfs:label" content="Acyl-CoA" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:hasDbXref" content="KEGG:C00040" datatype="xsd:string"></span>Acyl-CoA</span> binding protein essential bloodstream form T. <br />                                                                brucei. Mol Biochem Parasitol. 2001; 112:301&#226;&#8364;&#8220;304. [PubMed: 11223138] <br />                                                            Morita YS, Acosta-Serrano  Buxbaum LU, Englund PT. Glycosyl phosphatidylinositol <br />                                                                myristoylation African trypanosomes. New intermediates pathway fatty acid <br />                                                                remodeling. J Biol Chem. 2000a; 275:14147&#226;&#8364;&#8220;14154. [PubMed: 10799491] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Morita, YS.; Acosta-Serrano, ; Englund, PT. biosynthesis GPI enchors.  Ernst, P.; Sinay, <br />                                                                P.; Hart, G., editors. Oligosaccharides Chemistry Biology &#226;&#8364;&#8220; Comprehensive Handbook. <br />                                                                Wiley-VCH; Weinheim, Germany: 2000b. p. 417-433. <br />                                                            Movahedzadeh F, Smith DA, Norman RA, Dinadayala P, Murray-Rust J, Russell DG, et al.                                                                Mycobacterium tuberculosis ino1 gene essential growth virulence. Mol Microbiol. 2004; <br />                                                                51:1003&#226;&#8364;&#8220;1014. [PubMed: 14763976] <br />                                                            Nagamune K, Nozaki T, Maeda Y, Ohishi K, Fukuma T, Hara T, et al. Critical roles                                                                glycosylphosphatidylinositol Trypanosoma brucei. Proc Natl Acad Sci USA. 2000; 97:10336&#226;&#8364;&#8220; <br />                                                                10341. [PubMed: 10954751] <br />                                                            Norman RA, McAlister MSB, Murray-Rust J, Movahedzadeh F, Stoker NG, McDonald NQ. Crystal <br />                                                                structure inositol 1-phosphate synthase Mycobacterium tuberculosis, key enzyme                                                                phosphatidylinositol synthesis. Structure. 2002; 10:393&#226;&#8364;&#8220;402. [PubMed: 12005437] <br />                                                            Park S, Kim J. Characterization recombinant Drosophila melanogaster myo-inositol-1-phosphate <br />                                                                synthase expressed Escherichia coli. J Microbiol. 2004; 42:20&#226;&#8364;&#8220;24. [PubMed: 15357287] <br />                                                            Puoti  Conzelmann  Characterization abnormal free glycophosphatidylinositols accumulating                                                                mutant lymphoma cells classes B, E, F, H. J Biol Chem. 1993; 268:7215&#226;&#8364;&#8220;7224. [PubMed: <br />                                                                8463257] <br />                                                            Ralton JE, McConville MJ. Delineation pathways glycosylphosphatidylinositol biosynthesis <br />                                                                Leishmania mexicana. Precursors different pathways assembled distinct pools                                                                phosphatidylinositol undergo fatty acid remodeling. J Biol Chem. 1998; 273:4245&#226;&#8364;&#8220;4257. <br />                                                                [PubMed: 9461623] <br />                                                            Ralton JE, Mullin KA, McConville MJ. Intracellular trafficking glycosylphosphatidylinositol (GPI)- <br />                                                                anchored proteins free GPIs Leishmania mexicana. Biochem J. 2002; 363:365&#226;&#8364;&#8220;375. <br />                                                                [PubMed: 11931667] <br />                                                            Roper JR, G&#195;&#188;ther MLS, Milne KG, Ferguson MAJ. Galatose metabolism essential African <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                sleeping sickness parasite Trypanosoma brucei. Proc Natl Acad Sci USA. 2002; 99:5884&#226;&#8364;&#8220;5889. <br />                                                                [PubMed: 11983889] <br />                                                            Roper JR, G&#195;&#188;ther MLS, MacRae JI, Prescott AR, Hallyburton  Acosta-Serrano  Ferguson MAJ. <br />                                                                suppression galactose metabolism procyclic form Trypanosoma brucei causes cessation <br />                                                                cell growth alters procyclin glycoprotein structure copy number. J Biol Chem. 2005; <br />                                                                280:19728&#226;&#8364;&#8220;19736. [PubMed: 15767252] <br />                                                            Sarkar M, Hamilton CJ, Fairlamb AH. Properties phosphoenolpyruvate mutase, enzyme                                                                aminoethylphophonate biosynthetic pathway Trypanosoma cruzi. J Biol Chem. 2003; <br />                                                                278:22703&#226;&#8364;&#8220;22708. [PubMed: 12672809] <br />                                                            Smith TK, Cottaz S, Brimacombe JS, Ferguson MAJ. Substrate specifities dolicol phosphate <br />                                                                mannose: glucosaminyl phospatidylinositol a1-4-mannosyltransferase                                                                glycosylphosphatidylinositol biosynthetic pathway African trypanosomes. J Biol Chem. 1996; <br />                                                                271:6476&#226;&#8364;&#8220;6482. [PubMed: 8626449] <br />                                                            Smith TK, Milne FC, Sharma DK, Crossman  Brimacombe JS, Ferguson MAJ. Early steps                                                                glycosylphosphatidylinositol biosynthesis Leishmania major. Biochem J. 1997; 326:393&#226;&#8364;&#8220;400. <br />                                                                [PubMed: 9291110] <br />                                                            Smith TK, Crossman  Brimacombe JS, Ferguson MAJ. Chemical validation GPI biosynthesis                                                                drug target African sleeping sickness. EMBO J. 2004; 23:4701&#226;&#8364;&#8220;4708. [PubMed: <br />                                                                15526036] <br />                                                            Striepen B, Dubremetz J-F, Schwarz RT. Glucosylation glycosylphosphatidylinositol membrane <br />                                                                anchors: identification <span id='am-129' about='obo:IMR_0200021' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-130' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-131' property="oboInOwl:id" content="IMR:0200021" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="Uridine" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:hasDbXref" content="KEGG:C00299" datatype="xsd:string"></span>uridine</span> diphosphate-glucose direct donor chain <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 17 <br />  <br />  <br />                                                                modification Toxoplasma gondii using <span id='am-30' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-31' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span><span id='am-32' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span>carbohydrate</span> analogues. Biochemistry. 1999; 38:1478&#226;&#8364;&#8220; <br />                                                                1487. [PubMed: 9931013] <br />                                                            Sutterlin C, Escribano MV, Gerold P, Maeda Y, Mazon MJ, Kinoshita T, et al. Saccharomyces <br />                                                                cerevisiae GPI10, functional homologue human PIG-B, required                                                                glycosylphosphatidylinositol-anchor synthesis. Biochem J. 1998; 332:153&#226;&#8364;&#8220;159. [PubMed: <br />                                                                9576863] <br />                                                            Wirtz E, Leal S, Ochatt C, Cross GAM. tightly regulated inducible expression Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                conditional gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol <br />                                                                Biochem Parasitol. 1999; 99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                     Page 18 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 1. <br />                                                             CLUSTALW alignment prediced <span id='am-78' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-79' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-83' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> sequence Trypanosoma brucei (Tb, <br />                                                            AJ866770) INO1 Leishmania mexicana (Lm, AJ344543), Saccharomyces <br />                                                            cerevisiae (Sc, A30902), Homo sapiens (Hs, AAF26444) Mycobacterium tuberculosis <br />                                                            (Mt, P71703). Identical residues shown reverse type black background,                                                            conserved residues shown reverse type grey background. NAD+-binding <br />                                                            motif GXGGXXG (residues 95&#226;&#8364;&#8220;101) identified previously (Majumder et al., 2003) shown <br />                                                            solid line. <br />                                                            B. unrooted phylogenetic tree deduced amino acid sequences INO1 proteins, <br />                                                            showing relationship protozoan parasites (T. brucei, T. cruzi, L. <br />                                                            mexicana, P. falciparum) yeast, plant, bacterial human origins. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                             Page 19 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 2. <br />                                                            Expression purification recombinant TbINO1 E. coli. <br />                                                             TbINO1 cloned expression vector pBAD TA (C-terminal hexa tag)                                                            transformed TOP10 E. coli competent cells, production recombinant TbINO1 <br />                                                            induced 0.2% arabinose. cell disruption, soluble TbINO1 purified                                                            affinity chromatography using chelating column charged Ni2+, proteins                                                            separated SDS-PAGE gel stained Coomassie brilliant blue. Lane 1, total E. <br />                                                            coli cellular protein induction; lane 2, soluble E. coli protein induction,                                                            loaded affinity chromatography column; lane 3, purified recombinant TbINO1                                                            affinity chromatography; lane 4, purified recombinant TbINO1 dialysis. <br />                                                            B. Kinetics recombinant TbINO1 glucose 6-phosphate. Enzyme activity                                                            measured described Experimental procedures, NAD+ concentration held constant <br />                                                            (1 mM) glucose 6-phosphate concentration varied. Insert shows Lineweaver-Burk plot <br />                                                            data. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                     Page 20 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 3. <br />                                                            Construction INO1 conditional double knockout cell line. <br />                                                             Schematic representation construction strategy used.   allele INO1                                                            replaced puromycin resistance gene (PAC) homologous recombination generating <br />                                                            &#206;&#8221;INO1::PAC cell line. (ii) tetracycline-inducible ectopic copy INO1 introduced <br />                                                            rDNA, generating INO1-mycTi &#206;&#8221;INO1::PAC cell line. (iii) tetracycline <br />                                                            induced expression ectopic copy, remaining allele replaced                                                            hygromycin resistance gene homologous recombination, resulting conditional double <br />                                                            knockout cell line INO1-mycTi &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG. <br />                                                            B. Confirmation genotype T. brucei INO1 conditional double knockout cell line. <br />                                                            Southern blot analysis NcoI-digested genomic DNA (~2 &#206;&#188;g) wild-type T. brucei <br />                                                            cells (lane 1), &#206;&#8221;INO1::PAC (lane 2), INO1-mycTi &#206;&#8221;INO1::PAC (lane 3) INO1- <br />                                                            mycTiINO1 &#206;&#8221;INO1::PAC/&#206;&#8221;INO1::HYG (lane 4); INO1 ORF probe shows allelic <br />                                                            TbINO1 1.9 kb ectopic copy TbINO1-mycTi ~7 kb. <br />  <br />  <br />  <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                      Page 21 <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 4. <br />                                                            Growth curves Northern blot TbINO1 conditional double knockout cell line. <br />                                                            D. Cells washed tetracycline-free HMI-9, transferred tetracycline-free <br />                                                            media counted daily. Growth curves shown wild-type cells medium  , <br />                                                            TbINO1 conditional double knockout added tetracycline (B), TbINO1 conditional <br />                                                            double knockout tetracycline (C) TbINO1 conditional double knockout                                                            tetracycline, added myo-inositol (100 mM) (D). <br />                                                            E. Northern blot total RNA procyclic cells (lane 1), wild-type cells (lane 2), <br />                                                            TbINO1 conditional double knockout tetracycline day 0 (lane 3), TbINO1 <br />                                                            conditional double knockout tetracycline day 2 (lane 4) TbINO1 conditional <br />  <br />                                                                Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                        Page 22 <br />  <br />  <br />                                                            double knockout tetracycline day 12 (lane 5). blot probed                                                            TbINO1 ORF &#206;&#178;-tubulin, detailed Experimental procedures. <br /> Europe PMC Funders Author Manuscripts <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                               Mol Microbiol. Author manuscript; available PMC 2013 October 09. <br />                                          Martin Smith                                                                                        Page 23 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            Fig. 5. <br />                                                            vivo labelling wild-type cells. <br />                                                             Wild-type cells pulse pulse chase labelled lipids extracted, <br />                                                            desalted, separated HPTLC detected fluorography. Lane 1, [3H]-inositol pulse; <br />                                                            lane 2, [3H]-inositol pulse chase; lane 3, [3H]-mannose pulse; lane 4, [3H]-mannose pulse <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                            chase; lane 5, [3H]-glucose pulse; lane 6, [3H]-glucos <br /> </body></html>