<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                               NIH Public Access <br />                                Author Manuscript <br />                                Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             Mol Biochem Parasitol. 2013 July ; 190(1): 44&#226;&#8364;&#8220;49. doi:10.1016/j.molbiopara.2013.05.002. <br />  <br />  <br />  <br />                            Identification Essential Non-essential Protein Kinases                            Fusion PCR Method Efficient Production Transgenic <br />                            Trypanosoma brucei <br />                            Christopher Merritt Kenneth Stuart1 <br />                            Seattle Biomedical Research Institute, Seattle, WA 98109, USA <br />  <br />                            Abstract <br />                                  Manipulation gene expression used elucidate gene function, explore fundamental <br />                                  biological processes identify potential drug targets Trypanosoma brucei.                                  bloodstream forms CDC2-related kinase CRK12 (Tb11.01.4130) essential                                  transcriptional inactivation conditional null mutants lethal 19 protein kinases NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                  essential null mutants viable. did using efficient methods generation null <br />                                  conditional null cell lines T. brucei approaches generate transfection constructs                                  large targeting sequences use reliable transfection selection conditions.                                  methods, described supplementary material, employ multiple <br />                                  oligonucleotides PCR reactions transfections cost effective                                  simultaneously generate 24 transfectants shifting rate limiting experimental steps                                  production cell lines analysis. <br />  <br />  <br />                            Keywords <br />                                  Trypanosoma brucei; Protein kinases; Gene knockout; Conditional expression; Fusion PCR <br />                                  Method <br />  <br />                                             Manipulation gene expression powerful tool elucidating functions genes                                             context biological assessing potential gene products                                             drug targets. functions numerous genes examined Trypanosoma brucei <br />                                             order explore fundamental biological processes search drug targets. NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             addition, diploid protozoan extracellular pathogen served experimental <br />                                             surrogate related intracellular kinetoplastid pathogens T. cruzi Leishmania <br />                                             highly homologous syntenic genomes experimental genetic <br />                                             manipulation developed T. brucei [1]. <br />  <br />                                             robust homologous recombination naturally present T. brucei                                             exploited make gene knockouts (KOs) genome modifications. gene KOs, <br />                                             cells transfected electroporation linear DNA construct contains                                             antibiotic resistance gene, e. selectable marker, flanking sequences target <br />                                             recombination results replacement endogenous gene selectable <br />                                             marker. creation null mutant, e. cell alleles eliminated,                                             definitive way identify non-essential genes. Conditional RNA interference (RNAi) [1],                                             expression target gene repressed mRNA destruction tetracycline(tet)- <br />                                             dependant manner, commonly used T. brucei assess gene function  <br />  <br />                            1 <br />                             Corresponding author. Seattle Biomedical Research Institute, 307 Westlake Avenue North, Suite 500, Seattle, WA 98109-5219. <br />                            ken.stuart@seattlebiomed.org, Tel: 206-256-7302. <br />                             Merritt Stuart                                                                                            Page 2 <br />  <br />  <br />                                                 essentiality. RNAi cell line production entails cloning single RNAi construct                                                 single transfection step appropriate cell line. ability efficiently generate <br />                                                 RNAi cell lines led systematic use assess genes chromosome [2] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 genome-wide analysis genes using deep sequencing population RNAi cell <br />                                                 lines [3]. additional advantage RNAi approach mRNAs nearly <br />                                                 identical paralogs simultaneously targeted single construct.  RNAi <br />                                                 disadvantages sufficiently reduce mRNA levels generate                                                 definitive phenotype target effects, e. degradation mRNA genes <br />                                                 intentionally targeted. Little known target RNAi effects T. <br />                                                 brucei,  genetic approaches used verify RNAi acting target <br />                                                 [4]. prepared null cell lines knocking endogenous alleles cells                                                 &#226;&#8364;&#339;conditional null&#226;&#8364;? cell line knocking endogenous alleles cells tet- <br />                                                 regulatable copy target gene&#226;&#8364;&#8482;s coding DNA sequence (CDS) determine                                                 essentiality protein kinases. Expression knockdown robust conditional <br />                                                 nulls cell lines cell lines widely used preparation <br />                                                 requires multiple transfections cloning steps.  improved efficiency                                                 production minimizing need prepare plasmid constructs improving <br />                                                 transfection selection conditions. <br />  <br />                                                 employed efficient PCR approach make gene KO DNA constructs instead                                                 sequential PCR/cloning approach (e.g. [5&#226;&#8364;&#8220;9]) time consuming NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 pitfalls reflect limitations restriction enzyme digestion, DNA fragment <br />                                                 purification, ligation efficiency. PCR approach used directly produce                                                 DNA constructs transfection single PCR reaction drug resistance markers                                                 long oligos (~100 nucleotides) contain targeting sequences gene-specific KOs (e.g. <br />                                                 [10] http://tryps.rockefeller.edu/trypsru2_genetics.html). instead employed &#226;&#8364;&#339;fusion <br />                                                 PCR&#226;&#8364;? approach generate KO constructs sizeable homologous targeting sequences <br />                                                 (Fig. 1) use long oligos PCR adds significant cost effectively limits                                                 length homologous targeting region. targeting region limit affect frequency <br />                                                 homologous recombination shown shorter targeting sequences result <br />                                                 lower recombination frequency T. brucei [11] seen organisms. <br />                                                  used long (~500 nt) targeting sequences potentially provide greater <br />                                                 recombination frequency, greater frequency transfection. larger targeting <br />                                                 regions used fusion PCR approach enhance efficiency transfection <br />                                                 construct production. <br />  <br />                                                 fusion PCR method used (Fig. 1) adapted used systems <br />                                                 (e.g. [12, 13]). Multiple PCR reactions used rapidly, inexpensively, effectively <br />                                                 generate DNA direct use transfections. separate PCR reactions used NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 amplify 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; targeting sequences T. brucei genomic DNA gene. inner <br />                                                 oligos contain sequence complementary outer sequences selectable <br />                                                 marker PCR amplified bulk plasmid DNA gel purified. 5&#226;&#8364;&#178;, 3&#226;&#8364;&#178; <br />                                                 selectable marker PCR products combined removal PCR primers using <br />                                                 commercial spin column (Qiagen) amplified PCR reaction produces <br />                                                 fusion PCR product transfection. unique oligos needed create fusion <br />                                                 product KO allele (Fig 1). 1st allele inner oligos (oligos B, D) <br />                                                 contain sequences complement 1st selectable marker, outer oligos <br />                                                 (oligos  F) provide amplification ~500 nts targeting sequence flanking gene <br />                                                 CDS, oligos (G, H) nested  e. internal oligos F) specificity <br />                                                 create fusion PCR product used transfection. additional <br />                                                 oligos (oligos C, E) needed contain sequences complementary 2nd <br />                                                 selectable marker generate PCR product used fusion PCR  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                               Page 3 <br />  <br />  <br />                                                 generate construct KO 2nd allele. original outer oligos                                                 nested PCR  e. oligos  F, G, H). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 fusion PCR transfection constructs alleles manner                                                 targeting sequences. result construct 2nd allele KO recombining <br />                                                 site 1st allele KO. use drugs combination select <br />                                                 cells making approach suitable high throughput studies.  <br />                                                 recombination 2nd allele KO construct site 1st allele KO  does) <br />                                                 occur. eliminate 2nd endogenous allele insert site 2nd <br />                                                 selectable marker retain replace 1st selectable marker.                                                 averted detailed supplementary material using oligo pairs internal  <br />                                                 B D, F PCR amplify targeting sequences 2nd selectable marker.                                                 results need additional oligos awareness possible consequences                                                 adjacent genes (e.g. deletion important regulatory coding sequences).  minimally <br />                                                  maximally  oligos needed KO alleles,                                                 number used long oligo KO approach.  total cost                                                 oligos fraction cost longer ones smaller size. Currently total <br />                                                 reagents costs fusion PCR 3&#226;&#8364;&#8220;4 times single PCR method using <br />                                                 long oligos substantially greater cost long oligos. addition,                                                 larger flanking gene targeting regions fusion PCR products provide higher <br />                                                 recombination transfection efficiencies. oligos designed similar <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 annealing temperatures PCR products generated                                                 conditions. PCR protocol scaled 96 format enables <br />                                                 simultaneous production PCR products KO 12&#226;&#8364;&#8220;24 genes  supplementary <br />                                                 materials details). PCR products produced time (e.g. 48&#226;&#8364;&#8220;96 genes) <br />                                                 subsequent steps transfection, cell maintenance confirmation desired <br />                                                 transfections effectively limit production 24 gene KOs time person. <br />  <br />                                                 Fusion PCR products prepared used KO target genes T. brucei attempt                                                 create null mutants, e. eliminate endogenous alleles, 20 predicted protein kinases. <br />                                                 creation null mutants definitively demonstrate eliminated gene                                                 essential viability. conditional null mutant produced null mutants                                                 obtained. protein kinases chosen primarily members Sterile (STE)                                                 protein kinase groups thoroughly studied genetically T. brucei. <br />                                                 Null mutants generated higher-throughput fusion PCR transfection method <br />                                                 using different selectable markers sequentially KO endogenous copies target <br />                                                 genes  supplementary materials KO schematic). constructs designed                                                 precisely delete target gene start stop codon. endogenous allele                                                 eliminated transfecting purified fusion PCR products electroporation bloodform <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 (BF) T. brucei single marker 427 cells (SM427) [14]. SM427 cell line expresses T7 <br />                                                 polymerase Tet repressor proteins &#206;&#178; tubulin locus (maintained NEO <br />                                                 selectable marker) control tet conditional expression transgenes [14].                                                 electroporation PCR fusion product using approximately 2&#195;&#8212;107 mid log <br />                                                 phase SM427 cells outlined supplementary materials. conditions affect <br />                                                 transfection efficiency generation KO cell lines. include                                                 quality DNA used, length targeting sequences, number, growth <br />                                                 phase strain cells, transfection device settings used, selectable <br />                                                 markers, concentrations plating conditions used selection                                                 transfectants. variables examined previously ([11, 15])                                                 addressed  choice transfection device used greatly affects <br />                                                 transfection frequency ([16], http://tryps.rockefeller.edu/ <br />                                                 trypsru2_protocols_index.html). importantly, drug resistance <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                              Page 4 <br />  <br />  <br />                                                 markers used concentrations drugs used selection critical reliably <br />                                                 obtaining KO, especially null, cell lines. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 average 58 independent clones (determined limiting dilution) 1st allele <br />                                                 knocked obtained ~2&#195;&#8212;107 cells using ~200&#226;&#8364;&#8220;600ng PCR fusion product. <br />                                                 20 genes analyzed, range order magnitude. correlation                                                 observed number clones obtained PCR product used <br />                                                 200ng, length target gene&#226;&#8364;&#8482;s ORF kb (data shown). <br />                                                 Transfections similar conditions using blasticidin, puromycin, phleomycin <br />                                                 selectable markers (BSD, PAC, BLE) resulted 1st allele KOs numbers                                                 clones obtained different genes different drug concentrations                                                 analyzed. Success rates 2nd allele KOs similar conditions 15&#226;&#8364;&#8220;140 fold lower. <br />                                                 Success rates 2nd allele KOs genes lower tet-regulatable <br />                                                 copy expressed (data shown). lower success rates 2nd allele KOs                                                 explained solely response elimination gene expression. Electroporation using <br />                                                 Amaxa Nucleofection device, higher transfection efficiency BTX <br />                                                 device ([16], http://tryps.rockefeller.edu/trypsru2_protocols_index.html), resulted                                                 greater number 2nd allele KOs (7&#226;&#8364;&#8220;30 fold . lower number transfectants                                                 2nd versus 1st allele KOs increased selection consequence absence <br />                                                 allele, potential effects expression genes (e.g. cluster)                                                 gene KO, competition recombination 1st allele KO locus, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 effect using multiple antibiotics. Competition recombination reduced                                                 eliminated using sequences fusion PCR product 2nd allele                                                 eliminated 1st allele KO outlined  supplementary material). <br />  <br />                                                 choice selectable marker, drug concentration, conditions selection                                                 significant variables affecting frequency allele KO clones obtained, <br />                                                 especially 2nd allele KO  supplementary material). Reducing drug concentrations <br />                                                 generally used selection T. brucei (5 &#206;&#188;g/ml hygromycin, 5 &#206;&#188;g/ml <br />                                                 blasticidin, 0.1 &#206;&#188;g/ml puromycin, 2.5 &#206;&#188;g/ml phelomycin) (http://tryps.rockefeller.edu/ <br />                                                 trypsru2_genetics.html) omitting drug selection time resulted                                                 greater frequency obtaining KO cell clones  supplementary material). suggests <br />                                                 expression levels drug resistance markers governed target genes <br />                                                 endogenous UTRs ideal selection depending selectable marker.                                                 support  providing selectable marker gene UTRs support <br />                                                 higher levels expression provided wider window drug selection level                                                 expression endogenous gene UTRs limiting, lower levels                                                 expression  supplementary material). <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 verify desired gene KOs, genomic DNA clones obtained 1st 2nd allele <br />                                                 transfection genotyped using diagnostic oligos qRT-PCR  cases,                                                 PCR gel analysis. clones obtained attempting KO 1st allele  HYG) <br />                                                 20 protein kinases showed loss endogenous coding sequence (CDS) <br />                                                 copy qRT-PCR (Fig. 2). Attempts KO 2nd allele  BSD PAC) 20 1st <br />                                                 allele KO cell lines resulted 46/47 clones (multiple clones genes)                                                 endogenous CDSs absent (Fig. 2, data shown).  unable                                                 knockout second CRK12 allele  . did analyze cell line <br />                                                 resistant selectable markers copy target gene CDS. <br />                                                 KO cell lines examined PCR using oligos outside targeting region <br />                                                 subsequent gel analysis showed presence bands consistent replacement                                                 target CDS selectable markers (Fig. 2B, data shown). data suggest <br />                                                 approach described infrequently generate cell lines resistant <br />                                                 selectable marker contain desired 2nd allele KO events  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                Page 5 <br />  <br />  <br />                                                 mis-targeting gene duplication. minimizes time needed confirm identity                                                 desired clones transfection. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Identification genes essential viability order determine                                                 function potential drug targets. Stable null cell lines obtained                                                 genes genetic compensation. Attempts knock 2nd endogenous <br />                                                 allele essential gene result viable stable cell lines multiple <br />                                                 attempts, repeatedly observed (data shown), result                                                 unintended genetic changes. Stable cell lines obtained attempts resistant <br />                                                 drugs encoded 1st 2nd KO selectable markers retain copy gene <br />                                                 CDS mis-integrating selectable marker target locus integrating                                                 targeted locus loss resident allele (e.g. result gene duplication <br />                                                 event). example, CK1.2 (Tb927.5.800) strong growth defect RNAi BFs                                                 attempts create null cell lines unsuccessful suggesting essential <br />                                                 [8]. multiple attempts create null cell lines CK1.2 resulted cell lines                                                 contained resistance markers contained copy CK1.2 CDS, consistent <br />                                                 gene essential growth BF T. brucei (data shown). <br />  <br />                                                 Conditional null cell lines generated study genes essential viability.                                                 approach, tet-regulatable copy gene CDS inserted transcriptionally <br />                                                 silent rRNA intergenic locus SM427 T. brucei cells prior KO endogenous <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 alleles. efficiency, tet-regulatable copy introduced cells 1st <br />                                                 allele knocked simultaneously parallel attempts generate null <br />                                                 mutant. 2nd endogenous allele knocked fusion PCR product                                                 2nd allele KO selection performed media contains tet ensure expression                                                 regulatable copy rRNA intergenic region. Proper gene insertion assayed PCR                                                 indicated tet-regulation assayed monitoring cell growth presence <br />                                                 (gene expressed) absence tet (gene repressed) monitoring mRNA knockdown                                                 qPCR northern blot. Expression regulatable copy monitored western <br />                                                 analysis antibody available protein epitope tag                                                 fused regulatable copy. addition, level mRNA protein knockdown <br />                                                 controlled moderate range adjusting level tet cell grown <br />                                                 media. <br />  <br />                                                 tet-regulatable copy gene CDS cloned vectors                                                 widely available  http://www.zbh.uni-heidelberg clayton/vectors.html, http:// <br />                                                 tryps.rockefeller.edu/trypsru2_plasmids.html references supplementary materials). <br />                                                 Insertion CDS appropriate tet-regulatable vector performed                                                 traditional ligation cloning restriction enzyme sites designed CDS PCR oligos NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 efficiently vitro recombination using Gateway (Invitrogen)                                                 recombinase-specific sequence tails added oligos amplify CDS. expedite <br />                                                 cloning regulatable transgenes simultaneously, modified commonly <br />                                                 used regulatable expression vector (pLEW100v5) gene CDS inserted                                                 vitro recombination  supplementary materials). <br />  <br />                                                 Using scheme outlined supplementary materials, created conditional <br />                                                 null cell lines BF procyclic form (PF) T. brucei including CDC2-related <br />                                                 kinase CRK12 (Tb11.01.4130) BF cells (Fig. 3) previously shown                                                 growth defect RNAi [17]. cell line created SM427 cells                                                 express tet repressor protein necessary regulation CDS allele                                                 rRNA locus. cell line, fusion PCR products constructed described                                                 used KO endogenous alleles Gateway version pLEW100v5 used                                                 produce tet-regulatable copy (Fig. 3A). genotype cell line verified  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                               Page 6 <br />  <br />  <br />                                                 qRT-PCR genomic DNA template (Fig. 3B). Growth cells monitored                                                 days media containing tet (regulatable CRK12  lacking tet (regulatable <br />                                                 CRK12  showed strong growth phenotype cells regulatable copy <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 longer expressed (Fig. 3C). Growth resumed days presumably loss <br />                                                 repression regulatable allele rRNA locus means                                                 resulted growth recovery.  similar loss repression seen                                                 conditional null cell lines [18&#226;&#8364;&#8220;20] shown correspond mutations                                                 result loss tet repressor coding sequences [19]. <br />  <br />                                                 method described supplementary materials enable efficient analysis                                                 gene essentiality gene function T. brucei. does substituting PCR gene <br />                                                 cloning substantial extent preparing numerous transfection constructs                                                 performing multiple transfections simultaneously. way numerous gene KO cell lines <br />                                                 conditional cell lines rapidly generated. addition, variations described                                                 increase utility conditional null cell lines. selectable <br />                                                 drug markers used selection conditions important factors obtaining KO cells <br />                                                 lines  supplementary materials). number selectable drug markers used                                                 cell line construction reduced using KO method. approach                                                 employ loss heterozygosity increasing drug pressure select recombinants                                                 inserted drug marker single allele gene KO converts wild type allele <br />                                                 mitotic recombination eliminating  approach used infrequently <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 T. brucei gene conversion occurs frequently association antigenic <br />                                                 variation. Selectable markers excised integration allow reuse                                                 subsequent transfectants. accomplished generating KO cell lines PCR <br />                                                 products contain positive selectable marker  e. HYG, PAC, BSD, BLE) fused                                                 negative selectable marker (HSVTK CDS) flanked loxP sites. Excision                                                 accomplished transient transfection Cre recombinase selection HSVTK <br />                                                 protein addition Ganciclovir media  references details supplementary <br />                                                 materials). <br />  <br />                                                 Overall, used fusion PCR-based KO technique stringently assess gene essentiality                                                 BF T. brucei producing null conditional null cell lines. Obtaining 19 null cell lines                                                 protein kinases illustrates efficiency 6 nulls protein kinases BF T. brucei <br />                                                 reported <span id='am-1' about='oboInOwl:date' typeof='owl:Thing'>date</span> [5&#226;&#8364;&#8220;9]. methods used provide alternative greater <br />                                                 stringency conditional RNAi analysis essential genes. Additional <br />                                                 methodological details provided supplementary material. <br />  <br />                               Supplementary Material <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                               Acknowledgments <br />                                                 thank Ngoc Tran technical assistance Jef Boeke oligo ordering advice. work supported                                                 National Institutes Health grant AI078962 studies initiated support National Institutes <br />                                                 Health grant AI075641. <br />  <br />  <br />                               References <br />                                                 1. Kolev NG, Tschudi C, Ullu E. RNA interference protozoan parasites: achievements                                                    challenges. Eukaryot Cell. 2011; 10:1156&#226;&#8364;&#8220;63. [PubMed: 21764910] <br />                                                 2. Subramaniam C, Veazey P, Redmond S, Hayes-Sinclair J, Chambers E, Carrington M, et al. <br />                                                    Chromosome-wide analysis gene function RNA interference african trypanosome. <br />                                                    Eukaryot Cell. 2006; 5:1539&#226;&#8364;&#8220;49. [PubMed: 16963636] <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                                  Page 7 <br />  <br />  <br />                                                 3. Alsford S, Turner DJ, Obado  Sanchez-Flores  Glover L, Berriman M, et al. High-throughput <br />                                                    phenotyping using parallel sequencing RNA interference targets African trypanosome. <br />                                                    Genome Res. 2011; 21:915&#226;&#8364;&#8220;24. [PubMed: 21363968] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 4. Ringpis GE, Lathrop RH, Aphasizhev R. iCODA: RNAi-based inducible knock                                                    Trypanosoma brucei. Methods Mol Biol. 2011; 718:23&#226;&#8364;&#8220;37. [PubMed: 21370040] <br />                                                 5. Domenicali Pfister D, Burkard G, Morand S, Renggli CK, Roditi  Vassella E. Mitogen-activated <br />                                                    protein kinase controls differentiation bloodstream forms Trypanosoma brucei. Eukaryot Cell. <br />                                                    2006; 5:1126&#226;&#8364;&#8220;35. [PubMed: 16835456] <br />                                                 6. Jensen BC, Kifer CT, Parsons M. Trypanosoma brucei: mitogen activated protein kinase <br />                                                    kinases dispensable growth virulence bloodstream form. Exp Parasitol. 2011; <br />                                                    128:250&#226;&#8364;&#8220;5. [PubMed: 21396364] <br />                                                 7. Muller IB, Domenicali-Pfister D, Roditi  Vassella E. Stage-specific requirement mitogen- <br />                                                    activated protein kinase Trypanosoma brucei. Mol Biol Cell. 2002; 13:3787&#226;&#8364;&#8220;99. [PubMed: <br />                                                    12429824] <br />                                                 8. Urbaniak MD. Casein kinase 1 isoform 2 essential bloodstream form Trypanosoma brucei. <br />                                                    Mol Biochem Parasitol. 2009; 166:183&#226;&#8364;&#8220;5. [PubMed: 19450734] <br />                                                 9. Vassella E, Kramer R, Turner CM, Wankell M, Modes C, van den Bogaard M, et al. Deletion                                                    novel protein kinase PX FYVE-related domains increases rate differentiation                                                    Trypanosoma brucei. Mol Microbiol. 2001; 41:33&#226;&#8364;&#8220;46. [PubMed: 11454198] <br />                                                 10. Gaud  Carrington M, Deshusses J, Schaller DR. Polymerase chain reaction-based gene <br />                                                      disruption Trypanosoma brucei. Mol Biochem Parasitol. 1997; 87:113&#226;&#8364;&#8220;5. [PubMed: 9233680] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 11. Barnes RL, McCulloch R. Trypanosoma brucei homologous recombination dependent                                                      substrate length homology, displays differential dependence mismatch repair                                                      substrate length decreases. Nucleic Acids Res. 2007; 35:3478&#226;&#8364;&#8220;93. [PubMed: 17478508] <br />                                                 12. Szewczyk E, Nayak T, Oakley CE, Edgerton H, Xiong Y, Taheri-Talesh N, et al. Fusion PCR                                                      gene targeting Aspergillus nidulans. Nat Protoc. 2006; 1:3111&#226;&#8364;&#8220;20. [PubMed: 17406574] <br />                                                 13. Noble SM, Johnson AD. Strains strategies large-scale gene deletion studies diploid <br />                                                      human fungal pathogen Candida albicans. Eukaryot Cell. 2005; 4:298&#226;&#8364;&#8220;309. [PubMed: 15701792] <br />                                                 14. Wirtz E, Leal S, Ochatt C, Cross GA. tightly regulated inducible expression                                                      conditional gene knock-outs dominant-negative genetics Trypanosoma brucei. Mol <br />                                                      Biochem Parasitol. 1999; 99:89&#226;&#8364;&#8220;101. [PubMed: 10215027] <br />                                                 15. Wirtz E, Hartmann C, Clayton C. Gene expression mediated bacteriophage T3 T7 RNA <br />                                                      polymerases transgenic trypanosomes. Nucleic Acids Res. 1994; 22:3887&#226;&#8364;&#8220;94. [PubMed: <br />                                                      7937108] <br />                                                 16. Burkard G, Fragoso CM, Roditi  Highly efficient stable transformation bloodstream forms                                                      Trypanosoma brucei. Mol Biochem Parasitol. 2007; 153:220&#226;&#8364;&#8220;3. [PubMed: 17408766] <br />                                                 17. Mackey ZB, Koupparis K, Nishino M, McKerrow JH. High-throughput analysis RNAi <br />                                                      library identifies novel kinase targets Trypanosoma brucei. Chem Biol Drug Des. 2011; 78:454&#226;&#8364;&#8220; <br />                                                      63. [PubMed: 21668652] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 18. Martin KL, Smith TK. glycosylphosphatidylinositol (GPI) biosynthetic pathway                                                      bloodstream-form Trypanosoma brucei dependent novo synthesis inositol. Mol <br />                                                      Microbiol. 2006; 61:89&#226;&#8364;&#8220;105. [PubMed: 16824097] <br />                                                 19. Roper JR, Guther ML, Milne KG, Ferguson MA. Galactose metabolism essential African <br />                                                      sleeping sickness parasite Trypanosoma brucei. Proc Natl Acad Sci U S  2002; 99:5884&#226;&#8364;&#8220;9. <br />                                                      [PubMed: 11983889] <br />                                                 20. Young SA, Smith TK. essential neutral sphingomyelinase involved trafficking                                                      variant surface glycoprotein bloodstream form Trypanosoma brucei. Mol Microbiol. <br />                                                      2010; 76:1461&#226;&#8364;&#8220;82. [PubMed: 20398210] <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 8 <br />  <br />  <br />                                                                                              Highlights <br />                                                     &#226;&#8364;&#162;     CRK12 essential protein kinase BF T. brucei. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                     &#226;&#8364;&#162;     19 protein kinases essential bloodstream form T. brucei. <br />                                                     &#226;&#8364;&#162;     Efficient methods producing transgenic T. brucei cell lines. <br />                                                     &#226;&#8364;&#162;     Use fusion PCR, Gateway  improved selection conditions. <br />                                                     &#226;&#8364;&#162;     Supplementary material methodological details protocols. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                          Page 9 <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 1. Schema generation transfection constructs <br />                                                 Fusion PCR method generates gene knockout (KO) constructs. Step 1: gene-specific 5&#226;&#8364;&#178; <br />                                                 3&#226;&#8364;&#178; targeting sequences alleles amplified separate PCRs using different <br />                                                 oligonucleotides (oligos). Step 2: 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; targeting fragments allele fused <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 PCR coding DNA sequence (CDS) respective selectable marker (SM) <br />                                                 amplified plasmid stock. Complementary sequences designed                                                 termini oligos illustrated nested oligos G H provide specific <br />                                                 amplification fusion PCR product. Approximately 25&#226;&#8364;&#8220;50ng 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; targeting <br />                                                 sequences drug resistance CDS PCR products combined fused PCR <br />                                                 amplification. fusion PCR products verified purified agarose gel                                                 recovered Qiagen gel extraction kit. Fusion PCR protocols oligo sequences                                                 detailed supplementary material. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 10 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 2. Genotyping putative KO cell lines <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                  qRT-PCR analysis genomic DNA isolated putative 1st allele KO 2nd allele <br />                                                 KO (putative null) cell lines. cell line, genomic DNA isolated                                                 approximately 1&#195;&#8212;107 cells DNeasy Blood <span id='am-2' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-5' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amp</span>; Tissue Kit (Qiagen). QRT-PCR                                                 performed triplicate SsoFast EvaGreen Supermix Low ROX (BioRad)                                                 7500 Real-Time PCR (Applied Biosystems). Oligos qRT-PCR designed <br />                                                 using Primer3Plus (sequences given supplementary material). Tert used internal <br />                                                 control. 1st allele KO cell lines (n=19) showed loss copy qRT-PCR. 2nd <br />                                                 allele KO cell lines (putative nulls) showed loss copies qRT-PCR. Genotypically <br />                                                 null cell lines obtained 19 genes, lines Tb11.02.4860 <br />                                                 (CAMKK) retained copy target CDS (* RT &lt; 0.0001, ** RT &lt; 0.005, &#194;&#165;                                                 amplification, # target amplification). B. Example genotyping Tb927.10.14780 <br />                                                 null cell line PCR using oligos outside targeting regions subsequent gel <br />                                                 analysis showed presence novel bands consistent replacement target <br />                                                 CDS&#226;&#8364;&#8482;s selectable marker CDSs. <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                         Page 11 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Figure 3. CRK12 conditional null cell line created fusion PCR Gateway approach <br />                                                  Fusion PCR products used KO CRK12 endogenous alleles HYG                                                 BSD. Gateway cloning approach used generate tet-regulatable copy                                                 CRK12 transfection. clone tet-regulatable copy CRK12, CRK12 CDS <br />                                                 (Tb11.01.4130) PCR amplified SM427 genomic DNA using Phusion polymerase <br />                                                 (NEB) oligos KO_078(CRK12)GWF KO_078(CRK12)GWR, contain attB <br />                                                 recombination sequences Gateway cloning. product gel purified cloned <br />                                                 pDONR221 using BP Clonase II (Invitrogen). proper clone verified                                                 restriction enzyme digestion DNA sequencing. resultant Entry Clone (pCM013) <br />                                                 recombined Gateway compatible version pLEW100v5 vector <br />                                                 (pCM038) proper clone verified restriction enzyme digestion sequencing. <br />                                                 resultant plasmid (pCM092) linearized NotI prior transfection T. brucei <br />  <br />                                                 Mol Biochem Parasitol. Author manuscript; available PMC 2014 January 10. <br />                             Merritt Stuart                                                                                        Page 12 <br />  <br />  <br />                                                 generate cell line tet-dependent CRK12 expression rRNA intergenic <br />                                                 region. B. QRT-PCR verification CRK12 conditional null cell line. Oligos amplifying                                                 region CRK12 CDS copy CDS present (tet-regulatable copy) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 sets oligos amplify 5&#226;&#8364;&#178;/CDS 3&#226;&#8364;&#178;/CDS regions specific endogenous allele                                                 CRK12 absence endogenous CRK12 CDS (* RT &lt; 0.0005). tet-regulatable <br />                                                 CRK12 different 5&#226;&#8364;&#178; 3&#226;&#8364;&#178; UTRs endogenous alleles. C. Growth curve <br />                                                 CRK12 conditional null cell line shows growth rate inhibition withdraw tet. <br />                                                 CRK12 (Tb11.01.4130) conditional null cells maintained sequential dilutions                                                 media containing tet maintain cells mid-log phase growth. CRK12 conditional null cell <br />                                                 cultures densities 7&#195;&#8212;105 &#226;&#8364;&#8220; 1&#195;&#8212;106 parasites/ml used inocula growth curves. <br />                                                 cells spun washed twice media lacking tet wash tet 5&#195;&#8212;104 <br />                                                 parasites/ml diluted 20 ml split 10 ml cultures (+tet &#226;&#710;&#8217;tet). <br />                                                 Cultures conditional null copy maintained media 1 &#206;&#188;g/ml tet maintain <br />                                                 expression. Cells Incubat <br /> </body></html>