<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>         Report <br />  <br /> Cell Cycle 13:3, 434&#226;&#8364;&#8220;439; February 1, 2014; &#194;&#169; 2014 Landes Bioscience <br />  <br />  <br />  <br />    CNBP regulates wing development Drosophila <br />     melanogaster promoting IRES-dependent <br />                 translation dMyc <br />            Laura Antonucci1,2,&#226;&#8364;&#160;, Davide D&#226;&#8364;&#8482;Amico1,&#226;&#8364;&#160;, Laura Di Magno1,2, Sonia Coni2,3, Lucia Di Marcotullio1, Beatrice Cardinali4, <br />                                    Alberto Gulino1,2,5,6,*, Laura Ciapponi7,*, Gianluca Canettieri1,* <br />         1 <br />          Department Molecular Medicine; Sapienza University; Rome, Italy; 2Istituto Pasteur - Fondazione Cenci Bolognetti; Rome, Italy; 3CNRS UMR 7277; INSERM 1091; <br />        Institut Biologie Valrose (iBV); Universit&#195;&#169; Nice-Sophia Antipolis; Nice, France; 4Cellular Biology Neurobiology Institute; IBCN; National Research Council; <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                    &#194;&#169;2014 Landes Bioscience. distribute. <br />                Monterotondo, Rome, Italy; 5Neuromed Institute; Pozzilli, Italy; 6Center Life NanoScience LaSapienza; Istituto Italiano di Tecnologia; Rome, Italy; <br />                                                     7 <br />                                                       Department Biology Biotechnologies; Sapienza University; Rome, Italy <br />  <br />                                                                     &#226;&#8364;&#160; <br />                                                                      authors contributed equally work. <br />  <br />                                                 Keywords: CNBP, oncogene, dMyc, development, Drosophila <br />  Abbreviations: CNBP, CCHC-type zinc finger nucleic acid binding protein; dm, diminutive; DM2, myotonic distrophy type 2; <br />    EDTA, Ethylenediaminetetraacetic acid; IRES, internal ribosome entry site; ITAF, IRES trans-acting factor; ODC, ornithine <br />      decarboxylase; PEST, proline, glutamate, serine, threonine domain; RGG, arginine <span id='am-8' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span> rich domain; RpL32, 60S <br />   ribosomal protein L32; RpS6, ribosomal protein S6;  terminal oligopyrimidine; VDRC, Vienna Drosophila RNAi Center <br />  <br />  <br />  <br />                 CCHC-type zinc finger nucleic acid binding protein (CNBP) small conserved protein, plays key role             development disease. Studies animal models shown absence CNBP results severe developmen- <br />             tal defects attributed ability regulate c-myc mRNA expression. Functionally, CNBP binds <br />             single-stranded nucleic acids acts molecular chaperone, regulating transcription translation. <br />                 work report Drosophila melanogaster, CNBP essential gene, absence causes early embry- <br />             onic lethality. contrast observed species, ablation CNBP does affect dMyc mRNA expression, <br />             protein levels markedly reduced. demonstrate time dCNBP regulates dMyc trans- <br />             lation IRES-dependent mechanism, knockdown dCNBP wing territory causes general <br />             reduction wing size, keeping reported role dMyc region. Consistently, reintroduction dMyc <br />             CNBP-deficient wing imaginal discs rescues wing size, supporting key role CNBP-Myc axis             context. <br />                 Collectively, data previously uncharacterized mechanism,  regulating dMyc IRES-dependent <br />             translation, CNBP controls Drosophila wing development. results relevant implications species <br />             pathophysiological conditions. <br />  <br />  <br />  <br />                                   Introduction                                               transcription translation, affecting chromatin RNA <br />                                                                                              structural remodeling. originally identified transcrip- <br />     CCHC-type zinc finger nucleic acid binding protein (CNBP,                                tional factor, bound sterol regulatory element (SRE) named ZNF9) conserved single-stranded DNA binding                                  HMGCoA gene, involved <span id='am-33' about='obo:IMR_0200486' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00187" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="Cholesterol" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0200486" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cholesterol</span> metabolism.4 protein 19 kDa.1 Mammalian CNBP contains 7 highly                                     transcriptional regulator, CNBP described served CCHC zinc finger motifs, arginine/<span id='am-20' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-21' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-23' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Gly" datatype="xsd:string"></span>glycine</span>-rich domain                              inhibitor enhancer transcription genes, (RGG), putative PEST, nuclear localization sequence, PKA                               &#206;&#178;-myosin heavy chain, macrophage colony-stimulating <br /> phosphorylation site, mRNA expressed cells                            factor.5,6 notably, CNBP shown regulate transcrip- <br /> tissues, higher levels highly proliferating cells.2                                  tion c-myc, ability bind promote forma- <br />     Functionally, CNBP binds single-stranded DNA RNA                                     tion G quadruplexes (G4) c-myc promoter.1,7 promotes rearrangements secondary structure,                                  CNBP described translational regulator, act- <br /> acting nucleic acid chaperone.3 CNBP regulate                                  ing 2 distinct mechanisms. enhancement <br />  <br />  <br />  *Correspondence  Gianluca Canettieri; Email: gianluca.canettieri@uniroma1  Laura Ciapponi; Email: laura.ciapponi@uniroma  Alberto Gulino; <br />  Email: alberto.gulino@uniroma1  Submitted: 10/16/2013; Accepted: 11/19/2013 <br />  http://dx.doi.org/10.4161/cc.27268 <br />  <br />  <br /> 434	                                                                                Cell Cycle	                                                                Volume 13 Issue 3 <br />        Report                                                                                                                                  Report <br />  <br />  <br />  translation binding G-rich sequences downstream                                              Results regions mRNAs encoding ribosomal pro- <br /> teins elongation factors, increasing rate global                structure function CNBP conserved translation.8,9 Alternatively, CNBP shown promote               Drosophila melanogaster <br /> IRES-dependent translation ornithine decarboxylase                       <span id='am-1' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>Amino acid</span> alignment Drosophila human CNBP <br /> (ODC) mRNA, acting ITAF member.10,11 abil-                   showed high degree sequence similarity 2 pro- <br /> ity regulate IRES-dependent translation observed             teins. According ClustalW algorithm, 74% conserved <br /> human yeast CNBP,12 suggesting high conservation              amino acids, 43% identical, 15.7% strongly conserved, mechanism species.                                           15.1% weakly conserved (Fig.&#194;&#160;1A). <br />     mutation gene encoding CNBP                      Primary sequence analysis Drosophila CNBP (dCNBP) <br /> patients myotonic distrophy type 2 (DM2) consists               revealed presence 6 CCHC zinc finger sequences expansion CCTG repeats intron.13 Studies              putative RGG box, mammalian orthologs contain 7 <br />  <br />  <br />  <br />  <br />                                                                                                                                                         &#194;&#169;2014 Landes Bioscience. distribute. patients reported impairment global protein syn-           CCHC domain 1 RGG box (Fig.&#194;&#160;1B; ref. 1). <br /> thesis altered IRES-dependent translation.9,11                              high degree conservation dCNBP suggests     Genetic studies different species highlighted essen-        physiological function conserved. determine <br /> tial role CNBP embryonic development. Mice lacking                dCNBP plays role development, used 2 VDRC alleles CNBP gene die E10.5, embryos                 UAS-RNAi transgenic lines, carrying GAL4-inducible dCNBP- severe forebrain truncation facial abnormalities.14              RNAi construct CNBP coding gene (GC3800), similar phenotype observed chicken15 zebrafish16             crossed driver lines expressing GAL4 CNBP depletion, indicating conserved role control-              control 3 different ubiquitous promoters: tubulin, actin, ling embryonic development. Associated CNBP depletion               69B. shown Figure&#194;&#160;2A, crossed tubGAL4, <br /> decrease cell proliferation, mainly attributed         expression CNBP RNAi led partial depletion pro- ability CNBP regulate c-myc expression.                           tein, combination 2 copies UAS-dCNBP-RNAis <br />     present work, analyzed structure                 resulted complete loss dCNBP protein. Notably, com- <br /> function CNBP Drosophila melanogaster, model                  bination single double CNBP RNAi lines function CNBP investigated                  3 GAL4-expressing transformants resulted lethality <br /> observed CNBP highly conserved fly  analogy               pupal stage (Fig.&#194;&#160;2B).  analogy species, essential development. Importantly,           species, CNBP deficiency developmentally lethal, dCNBP promotes IRES-dependent transla-                   incomplete ablation protein. <br /> tion diminutive (dm), c-myc Drosophila ortholog                   dCNBP regulates IRES-dependent translation dMyc <br /> called dMyc), targeted knockdown dCNBP                       Gene deletion studies mice, chicken, Xenopous, zebraf- <br /> wing results reduction size similar dMyc phenotype.              ish revealed absence CNBP associated  <br />  <br />  <br />  <br />  Figure&#194;&#160; 1. CNBP conserved Drosophila melanogaster.   ClustalW alignment human D. melanogaster CNBP gene products. Identical, <br />  strongly conservative weakly conservative aminoacids indicated asterisk, colon, dots, respectively, according ClustalW convention. <br />  (B)&#194;&#160;Schematic representation CNBP Homo sapiens Drosophila melanogaster. CNBP contains 7 conserved CCHC Zn-finger repeats Homo sapi- <br />  ens 6 Drosophila; arginine/glycine-rich (RGG) box conserved species. <br />  <br />  <br /> www.landesbioscience.com	                                            Cell Cycle	435 <br />  lower proliferation rate mainly attributed development district transcription factor plays downregulation c-myc expression.1 described established role. CNBP induces c-myc transcription vertebrates,7 won-           end, knocked dCNBP wing imaginal <br /> dered dMyc mRNA levels downregulated discs, territory appropriate level dMyc expression absence dCNBP. Surprisingly, larvae ubiqui- necessary achieve proportionately correct size.19 <br /> tously expressing dCNBP RNAi, levels dMyc mRNA               crossed nubGAL4 line, expressing GAL4 wing changed (Fig.&#194;&#160;2C), dMyc protein levels strongly pouch, dCNBP RNAi lines carrying 1 2 RNAi <br /> reduced (Fig.&#194;&#160;2D), suggesting post-transcriptional regulation. constructs (2&#195;&#8212;dCNBPRNAi ). agreement role played control, tested dCNBP mRNA protein levels, dMyc, observed significant reduction wing size reduced knockdown (Fig.&#194;&#160;2C D).                     single RNAi lines (Fig.&#194;&#160;4A data shown). Expression <br />    CNBP characterized IRES trans-acting factor 2 RNAis combination resulted pronounced <br /> (ITAF) ability promote IRES-dependent translation reduction wing size, associated loss patterning elements <br />  <br />  <br />  <br />  <br />                                                                                                                                                       &#194;&#169;2014 Landes Bioscience. distribute. ODC mRNA.11  c-myc translated veins (Fig.&#194;&#160;4A).  analogy dMyc deletion, <br /> internal initiation sequences,17 hypothesized dCNBP knockdown dCNBP reduces wing size. validate promote IRES-dependent translation dMyc.                     involvement dMyc context, reintroduced dMyc     analogy IRES-containing mRNAs, dCNBP-deficient wings. shown Figure&#194;&#160;4B, reintroduction <br /> dMyc 5&#226;&#8364;&#178;UTR longer average UTR length, rich dMyc rescued effect dCNBP knockdown partially GC nucleotides, appears stable secondary restoring normal size wings.  data supported <br /> structure.18                                                          hypothesis dCNBP involved control wing size <br />    tested translational function dCNBP  regulating dMyc levels. <br /> served Drosophila studying sedimentation polyribosome fractions. end, performed polysomes pelleting wild-type embryo lysates. shown Figure&#194;&#160;3A, dCNBP associated poly- pellet, ribosomal RpS6 protein, association prevented EDTA treatment separates ribosome subunits (Fig.&#194;&#160;3B). <br />    study association dCNBP poly- <br /> somes, performed sucrose fractionation lysates Drosophila S2 cells. dCNBP detected polysomes subpolysome fractions purified ribosomal protein RpS6 (Fig.&#194;&#160;3C).  data demonstrated dCNBP associates active polyribosomes. <br />    tested dCNBP regulate <br /> IRES-dependent translation dMyc. end, used reporter plasmid expressing single bicistronic <br /> transcript containing Renilla reporter coding <br /> sequence, translated cap-dependent mechanism, downstream Luciferase reporter open read- <br /> ing frame, translated pres- <br /> ence upstream IRES sequence.11 cloned 5&#226;&#8364;&#178;UTR fragment dMyc, 2 cistrones generated dMyc-Luc reporter (Fig.&#194;&#160;3D, bot- <br /> tom). Compared bicistronic vector, <br /> dMyc-Luc vector showed significant basal IRES <br /> activity (Fig.&#194;&#160;3D, , demonstrating 5&#226;&#8364;&#178;UTR region dMyc translated IRES-mediated mechanism. Importantly, activ- <br /> ity enhanced 3-fold overexpres-         Figure&#194;&#160;2. dCNBP essential development.   RNAi efficiency larvae express- <br /> sion dCNBP (Fig.&#194;&#160;3D, , demonstrating            ing single (dCNBPRNAi) double copy (2XdCNBPRNAi) dCNBP RNAi,                                                             constitutively active tubulin expression driver. (B) Phenotypic consequences dCNBP increases IRES-dependent translation dMyc          RNAi-mediated inactivation dCNBP using reported VDRC UAS-RNAi lines mRNA.                                                       combination tubGAL4, actGAL4, 69BGAL4. (C) dCNBP dMyc mRNA <br />    dCNBP regulates wing development                         levels (QPCR), normalized housekeeping RpL32 mRNA wild-type    studied newly identified abil-           tubGAL4 double copy (2XdCNBPRNAi) dCNBP RNAi-expressing larvae. **P &lt; 0.05 vs. <br /> ity dCNBP upregulate dMyc affect proper           WT. (D) Effect dCNBP depletion (2XdCNBPRNAi) dMyc expression WT tub- <br />                                                                 GAL4-driven UAS- dCNBP <br />                                                                                      RNAi <br />                                                                                           -expressing larvae. Actin loading control. <br />  <br />  <br /> 436	                                                              Cell Cycle	                                                     Volume 13 Issue 3 <br />                               Discussion                                          appears conserved Gis2p protein, CNBP <br />                                                                                  ortholog S.&#194;&#160;cerevisiae, lacks RGG domain.12    present work characterized function                     conserved translational function dCNBP supported dCNBP regulating embryonic wing development                            evidence protein fractionates actively trans- <br /> Drosophila melanogaster.                                                         lating ribosomes increases rate IRES-dependent <br />    amino acid sequence dCNBP highly conserved                      translation. <br /> shares relevant domains members spe-                       exception yeast Gis2p, ablation CNBP dif- <br /> cies, suggesting protein retains molecu-                  ferent species results embryonic lethality, severe defects <br /> lar functions orthologs. function                    forebrain formation. Structure&#226;&#8364;&#8220;function relationship stud- ability promote IRES-dependent translation, feature                     ies highlighted importance N-terminal domain <br />                                                                                       e., Zn knuckle RGG domain) pro- <br />                                                                                      moting biological function CNBP, mainly attributed <br />  <br />  <br />  <br />  <br />                                                                                                                                                          &#194;&#169;2014 Landes Bioscience. distribute. <br />                                                                                      changes cell proliferation particular territories.2 <br />                                                                                      agreement observation, yeast Gis2p,                                                                                      CNBP ortholog lacking RGG domain, does <br />                                                                                      essential cell replication. Conversely, <br />                                                                                      keeping gene ablation studies species,                                                                                      observed knockdown dCNBP,                                                                                      highly conserved N-terminal region, causes lethality                                                                                      development. <br />                                                                                          role CNBP development mainly <br />                                                                                      linked ability promote c-myc mRNA expression. <br />                                                                                       present work, observed                                                                                      mRNA levels dMyc changed CNBP-deficient <br />                                                                                      larvae, protein levels strongly downregulated. <br />                                                                                      demonstrated linked ability                                                                                      dCNBP enhance IRES-dependent translation dMyc. <br />                                                                                      observation supported studies using bicistronic <br />                                                                                      expression plasmid containing 5&#226;&#8364;&#178;UTR IRES dMyc, <br />                                                                                      robustly induced overexpression dCNBP. <br />                                                                                          Supporting identified role regulating dMyc protein <br />                                                                                      levels, ablation dCNBP wing discs results reduc- <br />                                                                                      tion wing size, partially restored reintro- <br />                                                                                      duction dMyc. rule involvement <br />                                                                                      targets regulated dCNBP, data document <br />                                                                                      relevant biological role played dCNBP-dMyc axis <br />                                                                                      Drosophila development. interesting                                                                                      study CNBP regulates IRES-dependent translation <br />                                                                                      c-myc species  adding level                                                                                      complexity current model CNBP function. <br />                                                                                          relevant evidence stemming data                                                                                      IRES-dependent regulation myc                                                                                      evolutionary conserved mechanism. regard,  Figure&#194;&#160;3. Translational control dMyc CNBP   dCNBP levels super-          reported key role IRES-dependent translation  natant (super) polysome fractions (pellet) WT embryos extracts         c-myc mouse embryogenesis.20  data <br />  ribosome pelletting (B) Polysomal recruitment dCNBP non-treated (Ctr)         support important developmental role  EDTA-treated embryos lysates. assess specificity protein  <br />  fractionation, 100 mM EDTA, pH 7.4 added cell extracts loaded <br />                                                                                      IRES-dependent control myc link regulation  100 mM EDTA (+EDTA) sucrose cushion. Distribution ribosomal               CNBP, critical conserved regulator developmental <br />  proteins gradient purity fractions controlled RpS6        processes. <br />  Vinculin staining, respectively. (C) Analysis dCNBP association poly-           summary, work demonstrated  S2 cells. Cell lysate separated 15&#226;&#8364;&#8220;50% sucrose gradients.        time essential role dCNBP Drosophila <br />  presence dCNBP protein fraction analyzed western blot- <br />  ting. Distribution ribosomal proteins gradient controlled <br />                                                                                      development. identifying ability dCNBP pro- <br />  RpS6 staining. (D) dCNBP controls IRES-dependent translation dMyc.          mote IRES-dependent translation dMyc, unveiled <br />  IRES activity HEK293T cells transfected dMyc-Luc Luc               novel mechanism regulation critical growth regu- <br />  reporters dCNBP expression vectors indicated. *P &lt; 0.01,            lator, potential relevant pathophysiological <br />  dMyc-Luc vs. Luc, **P &lt; 0.01 CNBP vs.  Results shown         implications. <br />  average &#194;&#177; SD triplicate experiments (n = 4). <br />  <br />  <br /> www.landesbioscience.com	                                              Cell Cycle	437 <br />                     Materials Methods                                   melanogaster 1:500), anti-Myc Antibody (d1&#226;&#8364;&#8220;717) (sc-28207 <br />                                                                            SantaCruz 1:500), anti-Vinculin (MH24, developed Robert <br />    Cell culture, transfection, bicistronic assay                       H Waterston, obtained DSHB). <br />    HEK293T cells cultured previously described.21 S2                  Polysome analysis <br /> cells grown complete Schneider medium 25 &#194;&#176;C.                       S2 cells lysed ice-cold TKM Lysis Buffer <br />    HEK293T cells transfected using Lipofectamine 2000                 (100mM TRIS-HCl pH 7.4, 100 mM KCl, 10 mM MgCl 2, <br /> Reagent (Invitrogen) according manufacturer&#226;&#8364;&#8482;s protocol.             0.1% TritonX-100, 1 mM DTT, 40 mg/ml pepstatin, leu- <br />    Bicistronic assays performed measuring luciferase activ-           peptin, aprotinin, 0.5 mM PMSF, RNAguard [Promega] <br /> ity Promega dual-luciferase assay  described            2.5&#194;&#160;U/&#226;&#8364;&#352;&#206;&#188; l), incubated 5 min ice centrifuged 13&#226;&#8364;&#8240;500 <br /> previously.21,22                                                           g 5 min. Supernatants loaded 15&#226;&#8364;&#8220;50% sucrose <br />    plasmid pcDNA3.1CMV-renilla-TAA-luciferase                    gradients TNaM buffer (30 mM TRIS-HCl, pH 7.4, 100 <br /> Luc) provided AJ Link (Venderbilt University).                      mM NaCl, 5 mM MgCl 2) centrifuged 110 min <br />  <br />  <br />  <br />  <br />                                                                                                                                                   &#194;&#169;2014 Landes Bioscience. distribute. <br />    pcDNA3.1CMV-renilla-TAA-IRESdMyc-luciferase          (IRES-             Beckman SW41 rotor 37&#226;&#8364;&#8240;000 rpm 4 &#194;&#176;C. Gradients <br /> containing regions dMyc, dMyc-Luc) plasmid cloned                automatically collected, monitoring optical density inserting 5&#226;&#8364;&#178;UTR-region dMyc mRNA (Flybase CG10798)                  260&#194;&#160;nm, fractions precipitated protein analysis Luc plasmid; dMyc 5&#226;&#8364;&#178;UTR amplified                    described.23 Distribution ribosomal proteins gra- <br /> Drosophila melanogaster cDNA, following primers:                  dient purity fractions controlled RpS6 western <br />    Fw:         5&#226;&#8364;&#178;     &#226;&#8364;&#8220;ATACCATGGG               CTGTTGATTT                 blot. <br /> GACTGGGTG-3&#226;&#8364;&#178;                                                                  Ribosome pelleting experiments performed described <br />    Rev:       5&#226;&#8364;&#178;    &#226;&#8364;&#8220;ATACCATGGC                 ATTGCGATTA                 previously.11 Wild-type control flies incubated 25 &#194;&#176;C, TGTTGTCTGG G-3&#226;&#8364;&#178;                                                            embryos collected apple plates (cleared apple juice 25%, <br />    Ligation performed cutting dMyc 5&#226;&#8364;&#178;UTR,                    sucrose 2,5 %, agar 2,5%) 16&#226;&#8364;&#8220;18 h. Embryos washed Luc vector PvuII NcoI enzyme <br /> (New England Biolabs). <br />    mammalian expression vector pcDNA3HA- <br /> dCNBP obtained PCR-amplification CNBP homolog Drosophila melanogaster <br /> cDNA. <br />    RNA extraction quantitative PCR <br />    Total RNA extracted wild-type RNA- <br /> interference-expressing instar larvae TRI <br /> Reagent (AM9738, Ambion), reverse transcribed, analyzed quantitative real-time PCR custom designed Syber Green oligonucleotide listed  <br />    dMyc          QPCR_Fw:        5&#226;&#8364;&#178;-CCGAAAGCGA <br /> CTGGAAAGC; <br />    dMyc          QPCR_Rev:         5&#226;&#8364;&#178;-TGAGTTATAT <br /> GCCTCGCTGA ACA; <br />    RpL32         QPCR_Fw:        5&#226;&#8364;&#178;-CAAGAAGTTC <br /> CTGGTGCACA AC <br />    RpL32         QPCR_Rev:       5&#226;&#8364;&#178;-AAACGCGGTT <br /> CTGCATGAG <br />    dCNBP          QPCR_Fw:       5&#226;&#8364;&#178;-GAGGCCGTCA <br /> ACGAACGT <br />    dCNBP          QPCR_Rev:      5&#226;&#8364;&#178;-CGCAGCCGTA <br /> GCAAGTCTTC <br />    Immunoblot antibodies <br />    immunoblot analysis, instar larvae collected, lysed 2&#195;&#8212; Laemli Reducing Buffer,        Figure&#194;&#160; 4. Expression dMyc rescues undergrowth caused dCNBP depletion <br /> incubated 10 min 98 &#194;&#176;C, centrifuged 5     RNAi-mediated knockdown. Adult wings flies expressing indicated UAS- <br />                                                         constructs control nubGAL4 driver.   nubGAL4-mediated expression <br /> min 10&#226;&#8364;&#8240;000 g. extracts analyzed SDS-     UAS-dCNBPRNAi causes reduction size; expression 2 RNAis combination <br /> PAGE previously described.21                         (2xdCNBPRNAi) causes stronger reduction wing size loss patterning elements, <br />    following antibodies used: poly-            veins. (B) Reconstitution dMyc expression 2xdCNBPRNAi wings results <br /> clonal anti-CNBP (ab48027 AbCam, 1:500), anti-          partial rescue tissue growth. Wings areas indicated media mea- <br /> S6 Ribosomal Protein (Cell Signaling #2317 D.           sured using ImageJ (National Institutes Health); error bars represent SEM (n &#226;&#8240;&#165; 15). <br />                                                               *P&#194;&#160;&lt;&#194;&#160;0.05. <br />  <br />  <br /> 438	                                                              Cell Cycle	                                                 Volume 13 Issue 3 <br />  twice washing buffer (NaCl 0,7% TritonX-100 0,04%)                                   used obtained Bloomington stock center (http://flys- <br /> dechorionated 13% sodium hypoclorite 3 min. 3                                  tocks.bio.indiana.edu/). <br /> washes washing buffer, embryos dried lysed                                    Statistical analysis <br /> ice-cold TKM lysis buffer, incubated 8 min ice, centri-                                   Statistical analysis performed using StatView 4.1 software <br /> fuged 11&#226;&#8364;&#8240;300 rpm 5 min. supernatants loaded                                   (Abacus Concepts). Results expressed mean &#194;&#177; s.d. 1 M sucrose cushion (&#226;&#710;&#8217;/+100 mM EDTA) centrifuged                                   3 separate experiments, performed triplicate. Statistical 38&#226;&#8364;&#8240;000 g 1 h rotor type 80Ti.                                                     differences analyzed Mann&#226;&#8364;&#8220;Whitney U test    Distribution ribosomal proteins fractions                                 non-parametric values P &lt; 0.05 considered significant. <br /> controlled RpS6 Vinculin western blot. <br />    Fly stocks                                                                                                  Disclosure Potential Conflicts    RNAi transgenic fly lines obtained                                           potential conflicts disclosed. <br /> Vienna Drosophila RNAi Center (VDRC) (Dietzl et al., 2007). <br />  <br />  <br />  <br />  <br />                                                                                                                                                                                              &#194;&#169;2014 Landes Bioscience. distribute. <br /> Transgenic UAS-dCNBPRNAi fly lines obtained VDRC                                                                     Acknowledgments <br /> (<span id='am-32' about='oboInOwl:id' typeof='owl:Thing'>ID</span> 16283 16284) crossed nubGAL4, tubGAL4, act-                                      work supported AIRC (Associazione Italiana <br /> GAL4, 69BGAL4 driver flies incubated 29 &#194;&#176;C                                  Ricerca Cancro), MIUR, FIRB, PRIN, Ministry Health, <br /> 25 &#194;&#176;C. named 2xdCNBPRNAi stock consisted line carry-                                 EU HEALING grant, Italian Institute Technology, Agenzia <br /> ing 2 UAS-dCNBPRNAi constructs (ID 16283 16284) inserted                                  Spaziale Italiana. different chromosomes. driver lines used                                  LDM, LA SC supported Istituto Pasteur - <br /> previously described. nubGAL4 kind gift M                                      Fondazione Cenci Bolognetti, Rome, Italy. thank Drs Link <br /> M&#195;&#172;lan; driver lines UAS-dMyc line (#9675)                                   M Sammons providing bicistronic vector. <br />                                                               9.	 Huichalaf C, Schoser B, Schneider-Gold C, Jin              17.	 Stoneley M, Paulin FE, Le Quesne JP, Chappell <br />                       References                                                                                                  SA, Willis AE. C-Myc 5&#226;&#8364;&#178; untranslated region  <br />                                                                    B, Sarkar P, Timchenko L. Reduction 1.	 Calcaterra NB, Armas P, Weiner  Borgognone                  rate protein translation patients myo-              tains internal ribosome entry segment. Oncogene <br />     M. CNBP: multifunctional nucleic acid chaper-                tonic dystrophy 2. J Neurosci 2009; 29:9042-9;                 1998; 16:423-8; PMID:9467968; http://dx.doi. <br />     involved cell death proliferation control.          PMID:19605641;              http://dx.doi.org/10.1523/         org/10.1038/sj.onc.1201763 <br />     IUBMB Life 2010; 62:707-14; PMID:20960530;                     JNEUROSCI.1983-09.2009                                    18.	 Baird SD, Turcotte M, Korneluk RG, Holcik M. <br />     http://dx.doi.org/10.1002/iub.379                         10.	 Gerbasi VR, Link AJ. myotonic dystrophy type               Searching IRES. RNA 2006; 12:1755-85; <br /> 2.	 Armas P, Ag&#195;&#188;ero TH, Borgognone M, Aybar MJ,                    2 protein ZNF9 ITAF complex                 PMID:16957278;              http://dx.doi.org/10.1261/ <br />     Calcaterra NB. Dissecting CNBP, zinc-finger                  promotes cap-independent translation. Mol Cell                 rna.157806 <br />     protein required neural crest development,              Proteomics 2007; 6:1049-58; PMID:17327219;                19.	 Wu DC, Johnston LA. Control wing size pro- <br />     structural functional domains. J Mol Biol              http://dx.doi.org/10.1074/mcp.M600384-MCP200                   portions Drosophila myc. Genetics 2010; 184:199- <br />     2008; 382:1043-56; PMID:18703071; http://dx.doi.          11.	 Sammons MA, Antons AK, Bendjennat M, Udd                       211; PMID:19897747; http://dx.doi.org/10.1534/ <br />     org/10.1016/j.jmb.2008.07.079                                  B, Krahe R, Link AJ. ZNF9 activation IRES-                  genetics.109.110379 <br /> 3.	 Armas P, Nasif S, Calcaterra NB. Cellular nucleic acid         mediated translation human ODC mRNA             20.	 Cr&#195;&#169;ancier L, Mercier P, Prats AC, Morello D. c-myc <br />     binding protein binds G-rich single-stranded nucleic           decreased myotonic dystrophy type 2. PLoS               Internal ribosome entry site activity developmen- <br />     acids function nucleic acid chaperone. J          2010; 5:e9301; PMID:20174632; http://dx.doi.                   tally controlled subjected strong translational <br />     Cell Biochem 2008; 103:1013-36; PMID:17661353;                 org/10.1371/journal.pone.0009301                               repression adult transgenic mice. Mol Cell Biol <br />     http://dx.doi.org/10.1002/jcb.21474                       12.	 Sammons MA, Samir P, Link AJ. Saccharomyces                    2001; 21:18 <br /> </body></html>