<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>The Role Protein Interactions Mediating Essentiality Synthetic Lethality <br /> David Talavera, David L. Robertson, Simon C. Lovell* <br /> Faculty Life Sciences, University Manchester, Manchester, United Kingdom <br />  <br />  <br />  <br />      Abstract <br />      Genes characterized essential knockout associated lethal phenotype, &#226;&#8364;&#732;&#226;&#8364;&#732;essential genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; play <br />      central role biological function. addition, genes essential deleted pairs, phenomenon known <br />      synthetic lethality. consider genes displaying synthetic lethality &#226;&#8364;&#732;&#226;&#8364;&#732;essential pairs&#226;&#8364;&#8482;&#226;&#8364;&#8482; genes, analyze      properties yeast essential genes synthetic lethal pairs  gene duplication initially produces identical <br />      pair sets genes, invoked explanation synthetic lethality.  duplication explains <br />      minority cases synthetic lethality. Similarly, disruption metabolic pathways leads relatively examples      synthetic lethality. contrast, vast majority synthetic lethal gene pairs code proteins related functions      share interaction partners. essential genes synthetic lethal pairs cluster protein-protein <br />      interaction network. results suggest synthetic lethality strongly dependent formation protein-protein <br />      interactions. Compensation duplicates does usually occur mainly genes involved recent duplicates, <br />      commonly functional similarity permits preservation essential protein complexes. unified <br />      view, combining genes individually essential form essential pairs, suggests essentiality      feature physical interactions proteins protein-protein interactions, inherent gene      protein products  <br />  <br />   Citation: Talavera D, Robertson DL, Lovell SC (2013) Role Protein Interactions Mediating Essentiality Synthetic Lethality. PLoS 8(4): e62866. <br />   doi:10.1371/journal.pone.0062866 <br />   Editor: John Parkinson, Hospital Sick Children, Canada <br />   Received September 7, 2012; Accepted March 27, 2013; Published April 29, 2013 <br />   Copyright: &#195;&#376; 2013 Talavera et al. open-access article distributed terms Creative Commons Attribution License, permits <br />   unrestricted use, distribution, reproduction medium, provided original author source credited. <br />   Funding: work supported Biotechnology Biological Sciences Research Council (www.bbsrc.ac.uk), grant number BB/F013337/1. funders <br />   role study design, data collection analysis, decision publish, preparation manuscript. <br />   Competing Interests: authors declared competing interests exist. <br />   * E-mail: simon.lovell@manchester.ac.uk <br />  <br />  <br />  <br /> Introduction                                                                         synthetic-lethality, genes essential mutated <br />                                                                                      deleted individually, lethal altered simultaneously. <br />    organisms complex relationship                          phenomenon understood buffering effect <br /> genotype phenotype. key tool genetic research                        [10]: gene X buffer phenotypic effect loss gene Y <br /> generation analysis null mutations, particular                      vice versa.  lost simultaneously gene rendered non-functional, deletion. Analysis                    buffering possible. Alternatively, direct, null mutants allows identification &#226;&#8364;&#732;&#226;&#8364;&#732;essential genes&#226;&#8364;&#8482;&#226;&#8364;&#8482;,              additive effect, deletion causes decrease ability inferred significant contributions function                  pathway function. number mechanisms involving given environment.                                                              protein products explain buffering come    yeast (Saccharomyces cerevisiae) 20% genes                  (Figure 1), including previously proposed [10]. Using essential grown laboratory [1]. factors                             nomenclature Kelley Ideker [11],  B, D E determine given gene classed essential: essentiality                 &#226;&#8364;&#732; pathway&#226;&#8364;&#8482;&#226;&#8364;&#8482; explanations genetic interactions, C linked protein product highly connected                   F &#226;&#8364;&#732; pathway&#226;&#8364;&#8482;&#226;&#8364;&#8482; explanations. terms &#226;&#8364;&#732; networks protein-protein interactions [2], called                module&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732; module&#226;&#8364;&#8482;&#226;&#8364;&#8482; used similar <br /> &#226;&#8364;&#732;&#226;&#8364;&#732;centrality-lethality rule&#226;&#8364;&#8482;&#226;&#8364;&#8482; disputed [3]. Alternatively,                  concepts [12]. <br /> genes essential proteins code involved                                                                                         Protein-protein interactions account relatively small <br /> specific biological modules protein complexes [4]                                                                                      proportion genetic interactions [9].  protein complexes <br /> participate subset interactions essential <br />                                                                                      account subsets monochromatic genetic <br /> [5].  clear essential nature   <br />                                                                                      interactions  positive negative, equally gene strongly dependent context environmental <br />                                                                                      mixed) grouped functional modules [12]. Interestingly, conditions [6,7]. <br />                                                                                      recent study related strength genetic interaction <br />    important aspect context gene functions                                                                                       positive negative) proportion common genetic background expressed [8]. Genetic <br />                                                                                      physical interactors [13]. Similarly, genetic interactions interactions common [9], arise simultaneous <br />                                                                                      rationalized using metabolic models [14]. mutation gene results phenotypic effects <br />                                                                                      discrepancies predicted genetic interactions greater multiplicative effects mutating gene <br /> individually. extreme case negative genetic interaction                  measured experimentally, pleiotropy explained <br />  <br />  <br /> PLOS | www.plosone.org                                                       1                                  April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                  Protein Interactions Essentiality <br />  <br />  <br />                                                                              interactions. instance, unknown synthetic lethality <br />                                                                               synthetic sickness) regulated mechan- <br />                                                                              isms, different origins, e.g. set Mendelian traits <br />                                                                              quantitative traits. <br />                                                                                 basic potential mechanisms synthetic lethality <br />                                                                              arise (Figure 1), mechanisms  C, D F clearly arise <br />                                                                              gene duplication. mechanism  proteins X Y, <br />                                                                              coded synthetic lethal pair genes X Y, functionally <br />                                                                              compensate binding partner                                                                              interface. X Y recently duplicated paralogs,                                                                              protein products expected similar binding <br />                                                                              specificities, able compensate way. Similarly,                                                                              mechanism D, proteins X Y carry step                                                                              metabolic signaling pathway. Mechanisms C F arise <br />                                                                              duplication subsystem. mechanism C, Figure 1. Potential mechanisms rise synthetic               entire complexes X Y members lethality (essential pairs). X Y indicate proteins coded <br />                                                                              functionally redundant, paralogous. Duplication synthetic-lethal gene pairs. C gray indicates protein <br /> complexes; D&#226;&#8364;&#8220;E gray circles indicate proteins         complexes happen concert remain pathway.  Proteins X Y functionally redundant members         functional; achieved genome duplica- multi-protein complex. B. Proteins X Y members         tion [33]. Similarly mechanism F arise duplication protein complex. C. Proteins X Y essential members           subsection pathway. Alternatively different protein complexes functionally compensate        mechanisms C F alternative independent  D. Proteins X Y redundant members metabolic <br /> pathway. E. Proteins X Y play synergistic role metabolic          mechanism produce function. case <br /> pathway. F. Proteins X F essential members alternate              functional redundancy arise members metabolic pathways.                                                          complexes pathways paralogous. Obviously, basic <br /> doi:10.1371/journal.pone.0062866.g001                                        mechanisms complicated; e.g.: parallel <br />                                                                              pathways, present. use models [14]. alternative hypothesis                  proteins pairs coded synthetic lethal genes <br /> relates synthetic lethality rearrangements genetic                 participate protein complexes [34], synthetic lethality interaction network gene deleted [15];            arise paralogs compensate complexes understand rearrangements occur,                (Figure 1A) [35].  studies relationship <br /> possible predict outcome novel genetic interactions.               genetic interactions physical interaction networks <br />    key aspect genetic context gene presence              concluded majority genetic interactions occur duplicate copy. Gene duplication initially produces            proteins different complexes [11,36], suggesting <br /> identical genes diverge function [16]. duplicated          minimal functional compensation. Interestingly, genome-wide <br /> genes functionally compensate [17,18],               association study reached conclusions [37]. Mapping regulating expression remaining gene needed [19].           majority genetic interactions yeast demonstrates 1) 30% yeast duplicate genes functionally                      modest overlap genetic interactions direct <br /> redundant, case arise                physical interactions 2) proteins similar functions <br /> genome duplication smaller scale duplications [16,20].               similar pattern genetic interactions [38]. Exceptions  little evidence functional redundancy             include sets interactions termed &#226;&#8364;&#732;&#226;&#8364;&#732;monochromatic&#226;&#8364;&#8482;&#226;&#8364;&#8482;  e., <br /> unrelated singleton genes.  genes                    predominantly positive predominantly negative); modules <br /> duplicated likely essential              defined way frequently associated protein <br /> duplicated [17,21,22].  true mammals                  complexes, account minority cases <br /> [21,23&#226;&#8364;&#8220;25], probably complexity development                 [12]. studies demonstrated protein complexes <br /> higher organisms [26,27].                                                    negative interactions components coded    Duplicates high degree sequence similarity             essential genes [39,40]. Surprisingly, studies included genetic likely able functionally substitute [28].       interactions essential genes carrying hypomorphic mutations, <br /> Conversely, genes duplicated genes               affect results proteins time duplicate pairs high degree sequence divergence            criterion analyzing data (complexes unlikely able compensate [22,29].           monochromatic negative genetic interactions), object maintenance highly-similar paralogs detrimental              study. <br /> [30], examples network redundancy ensure                     present study relationship essential high metabolic flux [6] provide rapid response changing            genes synthetic lethal pairs yeast, consider having <br /> conditions. example regulation gene               paired essentiality. restrict analysis synthetic lethal pairs, <br /> expression transcription factors [31].                                    differentiate negative genetic interactions,    estimates percentage negative           approach differs previous studies [12,39,40]. genetic interactions caused simultaneous deletion           demonstrate 1) presence duplicate copies duplicate genes [18,32];  estimates greatly differ.           major determinant gene essential  2) products differences size dataset negative         essential genes members synthetic lethal pairs frequently <br /> interactions, methodology finding duplicates.            interact  expand analysis Zhang <br /> study exclusively devoted synthetic lethal pairs,          [5] include essential genes essential pairs. know proportions particular set negative genetic         present new view essentiality, emphasizing importance  <br />  <br /> PLOS | www.plosone.org                                               2                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                    Protein Interactions Essentiality <br />  <br />  <br /> protein complexes. view focuses protein-protein interac-               robustness existence alternative pathways tions, proteins                                  complexes perform function paralogs <br />                                                                                [10,22]. Alternatively, members essential pair Results                                                                        present complex metabolic pathway [10], adding <br />                                                                                small synergistic effect  Figure 1B<span id='am-1' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>amp</span>;E). <br /> Gene Duplication does Explain Synthetic Lethality <br />    Presence absence duplicate genes used explain             Members Synthetic Lethal Pairs Functionally existence synthetic lethal pairs essential genes,                   Related <br /> respectively. duplicate genes main explanation                                                                                    Synthetic-lethal pairs able compensate expect 1) nearly essential genes singletons  e., <br />                                                                                physical substitution  <span id='am-8' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span>, interchangeable, paralog);  2) synthetic lethal pairs                                                                                functionally related. Valente workers <br /> paralogous. Importantly, expect gene <br />                                                                                [35] examples paralogs likely duplications rise synthetic lethal pair [18]. <br />                                                                                interchangeable obligate transient complexes.    yeast 79.1% essential genes selected using                                                                                agreement low percentage synthetic lethal pairs stringent criteria singletons (78.5% selected using                                                                                paralogs (Figure 2 Table S1), 5.4% <br /> tolerant criteria, methods section selection criteria; Figure 2 <br />                                                                                (3.6% using tolerant criterion) protein products Table S1). higher expected chance <br />                                                                                essential gene pairs functional domain, defined <br /> (58,9%; p-value ,1024). contrast, 50.1% genes                                                                                InterPro, common (Figure 3 Table S2). synthetic lethal pairs selected using stringent criterion                                                                                percentage higher random expectation, confirms singletons (49.5% selected using tolerant criteria), <br />                                                                                small minority synthetic lethal pairs generated lower expected (56.3%; p-value = 361024).                                                                                simultaneous knockout evolutionary-related genes. <br /> results agree previous research, indicating single-gene- <br />                                                                                    contrast, disruption related functions explain <br /> associated essentiality derives lack paralogs [21],                                                                                large proportion synthetic lethality, demonstrated synthetic lethality originates gene duplication [22,29,35,41]. <br />                                                                                studies using synthetic genetic arrays [42].  complete set negative interactions <br /> [17,18,32], complete explanation.                identified functions gene compared assignments <br /> 20% yeast essential genes paralogs yeast genome.                members synthetic lethal pairs. Second, half genes synthetic lethal pairs            expected chance (60.0% vs. 12.6% using stringent <br /> paralog.  half genes members                 criterion; p-value ,1024) synthetic lethality involves deletion <br /> synthetic lethal pairs paralogs, percentage synthetic              genes working related functions (Figure 3 lethal pairs members paralogous genes just 4.3%               Table S2). indicates examples synthetic lethality <br /> (2.5%, using tolerant criteria). percentage              caused single functional process disrupted, similar originally set quantitative             deletion redundant proteins, disruption alternate <br /> negative interactions [32] recent estimates [17,18].              pathways. figures higher Tong    paralogs essential genes correspond                 coworkers [32]; results difficult compare <br /> highly diverged duplicates paralogs substitution                    consider synthetic sick pairs synthetic lethal <br /> functionally important residues. cases, modifications                  pairs, restrict analysis synthetic lethality. prevent functional compensation [16,28]. case                  functional compensation mainly related synthetic lethal gene pairs, unlikely pairs        synthetic lethality, synthetic sick interactions emerging duplicates able compensate functionally,                  different causes (Table S3).  differences sample size <br /> diverged prevent identification paralogs.            ambiguity definition genetic interactions highly probable mechanisms duplication               suggest caution interpreting result. possibility events contribute determining gene                 &#226;&#8364;&#732;&#226;&#8364;&#732;study bias&#226;&#8364;&#8482;&#226;&#8364;&#8482;, genes selected experimental testing essential.                                                                     likely functionally related. bias functional relatedness <br />    synthetic lethal pairs participate functional             introduced 1) performing small-scale experiments <br /> redundancy duplication. contribute  <br />  <br />  <br />  <br /> Figure 2. Duplication essential genes members                        Figure 3. Evolutionary functional similarity essential <br /> essential pairs. Genes selected using stringent criteria. inset       pairs. Genes selected using stringent criteria. expected values <br /> shows percentage essential gene pairs paralogs.                mean standard deviation shown. <br /> doi:10.1371/journal.pone.0062866.g002                                          doi:10.1371/journal.pone.0062866.g003 <br />  <br />  <br /> PLOS | www.plosone.org                                                 3                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                     Protein Interactions Essentiality <br />  <br />  <br />  323 440 sources reporting just synthetic lethal pair, pair functionally related);  2) using epistatic <br /> miniarray profiles (E-MAP), designed testing subset interactions suspected enriched positive negative interactions [43,44]. majority data data set <br /> came experiments using functional information design; subsets displayed degree functional relatedness (Table S4). <br />  Synthetic Lethal Pairs Caused Deletion Members Protein Complexes <br />    Synthetic lethality functional redundancy occur genes code proteins buffer   protein complexes acting metabolic pathways, deletion gene buffered s presence (Figure 1); wider <br /> explanation causes epistasis, synthetic lethality ultimate consequence, given Lehner [45]. order               Figure 4. Essential pairs members bind determine membership protein complex                          interactor. interaction networks built combining participation pathway equally important understand-              criteria selection essentiality physical interactions (SS ing synthetic lethality, calculated percentage synthetic            stringent criteria used; ST stringent criteria used                                                                               selecting physical interactions; TS essentiality stringently <br /> lethal pairs members assigned metabolic                  selected; TT tolerant criteria used). expected <br /> pathway detected affinity capture experiments.                  values mean standard deviation shown. <br /> assume affinity capture experiments retrieve multi-                 doi:10.1371/journal.pone.0062866.g004 <br /> protein complexes. assign status member protein <br /> complex protein involved protein-protein interactions               indicate number synthetic lethal pairs code putatively <br /> detected affinity capture experiments. results               interchangeable proteins form protein complexes. case 8961% synthetic lethal pairs contain genes               simultaneous knockout proteins produces lethal <br /> code proteins, suggesting gene products                   phenotype complexes formed. <br /> members different protein complexes main reasons synthetic lethality. proteins                                                                               &#226;&#8364;&#732;&#226;&#8364;&#732;Essential Proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482; Group Interaction Sub- hits affinity capture experiments despite involved protein complexes, used alternative                network restrictive definition complex membership, based                   importance protein complexes synthetic lethality <br /> terms. percentage synthetic lethal pairs                indicates formation protein complexes crucial genes members protein complexes lower,                 survival cell. implies components definition figures 4060% using         complexes essential [39]. Accordingly, annotations IEA, NAS ND ones, 3860%                hypothesize &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;, regardless using annotations experimental evidence code. Converse-               product essential genes members synthetic lethal <br /> ly, disruption metabolic pathways explain                     pairs, highly interconnected. expansion minority (1260%) cases synthetic lethality.                  original essential interactions hypothesis proposed 2265% synthetic lethal pairs caused metabolic              Zhang [5] supported previous finding factor occur pathway, suggesting alternate                   protein complexes enriched single-gene essential proteins <br /> pathways account ,10% synthetic lethal pairs.                          products synthetic lethal pairs [34]. <br />    Synthetic-lethal pairs gene products different complexes              order verify expected enrichment interactions common pairs complex [11,36],                       essential proteins, determined number essential methodological statistical biases [11,36].           proteins physically interact essential proteins.  examples products                    control, calculated percentage non-essential proteins <br /> synthetic lethal pairs present protein complex                interacting single-gene essential proteins members [11,34,36], expected directly interact [9].           synthetic pairs. 7569% single-gene essential  proteins compensatory role expect              proteins 8165% members synthetic lethal pairs interact interact                     essential proteins, just 57610% non- <br /> interacting partners common [13]. case                essential proteins interact essential proteins. synthetic lethal pairs (Figure 4 Table S5). observed                  products coded essential genes tend percentages higher obtained chance (7.6%               highly connected. <br /> vs. 0.1&#226;&#8364;&#8220;0.7% analyzing SS network; p-value ,1024)                   determined number interactions independent structure synthetic lethality network,          essential proteins, comparing results networks altering network topology substituting reshuffling           annotated using purely randomized lists interactions. Annota- <br /> nodes produce significant decrease number synthetic               tion independent synthetic lethality network  e. degree <br /> lethal pairs common physical interactor. importantly,             distribution), takes account gene 6364% pairs interacting partner common,                 involved synthetic lethality interaction.  determined <br /> shared interactor product essential gene               frequency essential-gene proteins interact member synthetic lethal pair. Increase number                 just 2.6%&#226;&#8364;&#8220;3.8% physical interactions occur <br /> experimentally-identified protein-protein interactions               products essential genes, results increase number pairs sharing interactors. findings              higher expected chance (0.3&#226;&#8364;&#8220;0.4%; p-value ,1024; <br />  <br />  <br /> PLOS | www.plosone.org                                                4                                April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  <br /> Figure 5 Table S6).   single-gene                fact proportion products single-gene <br /> essential proteins tend interact [5,46], meaning         essentiality higher networks obligate physical cluster functional modules [4], tend         interactions (Table S14). Nonetheless, prefer cautious complexes [34,47]. addition, number             results, caused definition physical interactions involving members synthetic lethal pairs        obligate transient interactions  Methods). <br /> higher expected chance (Figure 5 Table S7). <br /> Finally, observe enrichment interactions              Functional Convergence Diminish Number essential genes members synthetic lethal pairs (Figure 5         Essential Genes <br /> Table S8). enrichment independent dataset                 Clearly, randomly removing essential proteins analysis <br /> randomisation used, conclude interactions             diminish enrichment essential sub-network. caused distribution just type essentiality.                separately excluded   members synthetic lethal pairs Importantly, removing physical interactions detected                paralogs (b) members synthetic lethal pairs shared <br /> affinity capture methods causes small, non-significant               common interacting partner.  effect decrease proportion physical interactions involving               exclusion paralogs expected, exclusion pairs essential genes members synthetic lethal pairs (Table S9).            interacting partner common greater effect (Figure 7 results corroborate essential interactions hypothesis         Table S15). <br /> demonstrate essential proteins highly inter-connected,            results demonstrate ability form correct, essential proteins define interaction sub-network (Figure 6).         cognate interaction leads enrichment physical interactions apparently members synthetic lethal pairs              &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;, evolutionary  <br /> likely self-interact (Table S10), probably product       lationship.  functional convergence different combination rules hetero- homo-complexes.             preserve essential protein complexes. speculate  essential interaction sub-network mainly formed           singletons members synthetic lethal pairs hetero-interactions (Table S10).                                            non-essential buffering mechanism. <br />    Finally, determined differences networks exclusively composed obligate transient protein              Discussion <br /> interactions. types protein interactions quite <br /> disparate [48];  potentially, introduce differences            suggested relatively simple organisms, network. clear differences       yeast, phenomenon gene essentiality easily related <br /> proportion interactions involved essential subnetwork             duplication [21]: singleton genes  e., clear <br /> (Table S11), clear physical interactions          paralog) likely essential, duplicated genes products essential genes relevant obligate                   compensate [28]. results interactions (Table S12), transient interactions contain            yeast straightforward relationship essentiality greater percentage interactions proteins coded          gene duplication. <br /> genes involved synthetic lethality (Table S13). differences            fifth essential yeast genes paralogs; <br />                                                                              synthetic lethal pairs considered, 4.3% <br />                                                                             cases members pair paralogous. figure                                                                             lower estimates weak phenotypic effect                                                                             duplicate redundancy [17,18], similar older <br />                                                                             study demonstrated just 2% negative genetic <br />                                                                             interactions involved duplicate genes [32], notwithstanding <br />                                                                             methodological differences.  clear                                                                              yeast, synthetic lethality explained duplication  <br />                                                                                 unsurprisingly, ability synthetic lethal gene pairs <br />                                                                             compensate functionally important.                                                                             case divergently convergently evolved.                                                                             functional compensation highly likely                                                                             membership protein complex, frequently                                                                             members synthetic lethal pair sharing common <br />                                                                             interacting partner.  common type functional <br />                                                                             compensation arises proteins able compensate                                                                             s protein-protein interactions,                                                                             mechanism (Figure 1). Conversely, genes property <br />                                                                             single-essentiality protein products make interactions <br />                                                                             compensated. key observations                                                                                clustering essential genes synthetic lethal pairs                                                                             protein interaction network suggests synthetic lethality                                                                             strongly dependent formation protein-protein interac- <br />                                                                             tions (ii) compensation frequently arises Figure 5. Essential interactions.  SS network. B. ST network. C. TS       functional similarity results preservation essential <br /> network. D. TT network. &#226;&#8364;&#732;&#226;&#8364;&#732;Randomized genes&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates mean         protein complexes. central role interactions synthetic <br /> standard deviation expected values random assignments essential genes. &#226;&#8364;&#732;&#226;&#8364;&#732;Randomize pairs&#226;&#8364;&#8482;&#226;&#8364;&#8482; indicates mean standard <br />                                                                             lethality allows alternative phrasing: deviation expected values random assignments essential           interaction essential, gene. <br /> pairs.                                                                          proposed interaction-centric view correct, help <br /> doi:10.1371/journal.pone.0062866.g005                                       understanding human disease [49]. applied  <br />  <br /> PLOS | www.plosone.org                                              5                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  <br />  <br />  <br /> Figure 6. Biggest component physical interaction network. Proteins coded essential gens coloured dark blue; proteins members synthetic lethal pairs coloured light blue;  non-essential proteins coloured red. <br /> doi:10.1371/journal.pone.0062866.g006 <br />  <br /> understanding essentiality, extending work               ally-related &#226;&#8364;&#732;&#226;&#8364;&#732;essential proteins&#226;&#8364;&#8482;&#226;&#8364;&#8482;. essential sub-network <br /> Zhang [5]. interaction focus allows unify analysis           apparent essential genes genes paired- <br /> essential genes, synthetic lethality functional compensation             essentiality (synthetic lethal pairs) considered  single framework. propose single framework             cautiously, results suggests macromolecular <br /> helps overcome contradictions described                      machinery make physical interactions involving <br /> researchers [4].                                                             protein products single-essential genes,    Previous research shown presence products               members synthetic lethal pairs relatively essential genes genes negative genetic interactions          important role transient interactions. Previous research complex [39,40].  demonstrate enrichment                    essential interactions evolutionary-conserved physical interactions subset products: e.       non-essential interactions [5,50]. true significant interaction sub-network physically links function-          genetic interaction network [51]. possible  <br />  <br /> PLOS | www.plosone.org                                               6                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                      Protein Interactions Essentiality <br />  <br />  <br />                                                                                  essential functional unit [12], specificity protein- <br />                                                                                  protein interaction key compensation. <br />                                                                                     expected conservation interactions involving <br />                                                                                  essential proteins [5,50,51], likely definition                                                                                  comprehensive essential interaction sub-network aid                                                                                  prediction protein-protein interactions related organ- <br />                                                                                  isms. Experimentally-generated genetic interactions                                                                                  mechanistic explanation phenotype associated                                                                                  range naturally-occurring mutations [37].  protein <br />                                                                                  complexes form functional units, play central <br />                                                                                  role linking functional units [12].                                                                                  great benefit biotechnology medicine studying                                                                                  consequences protein mutations associated disruption <br />                                                                                  essential interactions, protein itself.                                                                                   hypothesize interaction-centric view essentiality likely <br />                                                                                  facilitate understanding essentiality mammals. <br />  <br />                                                                                  Methods <br />                                                                                  Data <br />                                                                                     complete list physical synthetic lethal interactions <br />                                                                                  data downloaded BIOGRID repository (http:// <br />                                                                                  thebiogrid.org/; version 2.0.63 downloaded 25th March 2010) <br />                                                                                  [53]. Essential genes recovered different sources: 1083 <br />                                                                                  genes Saccharomyces Genome Deletion Project [1,54]; <br />                                                                                  1007 genes CYGD database (http://mips.helmholtz- <br />                                                                                  muenchen genre/proj/yeast/) [55]; 1136 genes SGD <br />                                                                                  database (http://www.yeastgenome.org/) [56] 1107 genes <br />                                                                                  recent study using synthetic genetic array technology [9]. <br />                                                                                  order proper balance quality size <br />                                                                                  dataset, used different criteria select type                                                                                  data. Paralogs InterPro domains assignments  <br />                                                                                  loaded Ensembl Genomes [57] BioMart portal [58]. <br />                                                                                  Functional clusters extracted DAVID bioinfor- <br />                                                                                  matics resources (http://david.abcc.ncifcrf.gov/) [59], using Figure 7. Contributors essential interactions.  Exclusion                highest stringency setting initial final group member- <br /> essential pairs genes paralogs. B. Exclusion essential pairs proteins share interactors. &#226;&#8364;&#732; exclusion&#226;&#8364;&#8482;&#226;&#8364;&#8482; original values <br />                                                                                  ship 2. exclusion essential pairs; blue; &#226;&#8364;&#732; exclusion&#226;&#8364;&#8482;&#226;&#8364;&#8482; observed <br /> values exclusion selected pairs; &#226;&#8364;&#732;&#226;&#8364;&#732;Model 10 mean                Physical Interactions <br /> standard deviation expected values based model 1 (exclusion                   case physical interactions, encountered based number genes); &#226;&#8364;&#732;&#226;&#8364;&#732;Model 20 mean standard deviation                                                                                  problems: (1) methodology used detecting interaction, <br /> expected values based model 2 (exclusion based number pairs).                                                                          (2) reproducibility results. Different methodologies <br /> doi:10.1371/journal.pone.0062866.g007                                            account different types detected interactions,                                                                                  strengths weaknesses. instance, yeast hybrid experi- <br /> set interactions represents ancestral network, core            ments identify proteins bind   <br /> set functions common wide range organisms.                biologically irrelevant interactions detected. Affinity <br />                                                                                  capture methods believed account false positives, expect essential sub-network                                                                                  allow detection interactions need related organisms. Interestingly, research bacterial <br />                                                                                  protein expression. relative merits discussed <br /> metabolic networks shows network core                                                                                  previously [60&#226;&#8364;&#8220;62].  Yu coworkers conserved, peripheral interactions originated niche <br />                                                                                  yeast hybrid technique better finding direct <br /> specialization different organisms [52]. <br />                                                                                  interactions affinity purification methodologies,    compensation does mainly occur                                                                                  prone biased essential genes genes encoding <br /> evolutionary relationship, removing paralogous synthetic lethal <br />                                                                                  specific functions [46]. <br /> pairs network analysis greater effect                                                                                     methods sources errors, (1) selected <br /> essential sub-network removing random essential pairs.                                                                                  detected using different experimental systems contrast, higher depletion essential interactions observed <br />                                                                                  reported different scientific papers (2350 nodes, 4951 edges, removing pairs interacting partner                stringent criteria)  (2) selected different pieces common. functional similarity, commonly manifested                     evidence, e., reported twice detected using similarity protein-protein interactions, permits preservation              different methods (3335 nodes, 12153 edges, tolerant criteria) essential protein complexes. Consequently, cases,                     Figure S1 Tables S16 S17 details). lethality lack essential component protein            physical interactions selected using stringent criterion likely <br /> complex. results highlight protein complex basic                true positives terms protein&#226;&#8364;&#8482;s ability bind                                                                                  biological relevance.  rule  <br />  <br /> PLOS | www.plosone.org                                                   7                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                 Protein Interactions Essentiality <br />  <br />  artifacts identified using different experimental               biological data available (ND evidence code) inferred designs.                                                                    electronic annotation (IEA evidence code). <br />  <br /> Essential Genes Synthetic Lethal Pairs                                  Obligate Transient Interactions <br />    order clearly differentiate essential genes genes              divided set physical interactions different <br /> synthetic-lethal pairs, essential genes members         subsets: obligate transient interactions. Unambiguous differ- synthetic-lethal pairs removed datasets.                   entiation interaction types challenging genetic interactions measured non-              continuum binding affinity [48].  classified essential gene essential gene containing hypomorphic               obligate interactions detected affinity capture <br /> mutation. interactions biologically relevant,            experiments involving proteins annotated participating harder interpret.  logically              macromolecular complexes  0032991) experimental <br /> correct classify gene essential non-essential             evidence. Conversely, transient interactions included simultaneously. possible consequence using data              detected method affinity capture involving results members synthetic-lethal pairs           proteins assigned participate protein heavily influenced results essential genes.               complexes. noteworthy 1) interactions contrast previous research [39,40], removed genes               assigned subsets;  2) way built analyses. problem encountered                   networks does allow crosstalk proteins obligate <br /> genetic interactions comprehensively tested, high-          interactions proteins transient interactions. <br /> throughput studies. Experiments focus specific <br /> sets genes. addition, high-throughput experiments limited         Annotated Networks Physical Interactions small number research groups. believe             Combining selection criteria, built networks numbers synthetic-lethal interactions analyzed           physical interactions annotated essentiality information  fair representation current knowledge.                             Figure S3). networks labeled SS (stringent physical <br />    Data set selection illustrated Figure S2, described         interactions; stringent essentiality), ST (stringent physical interac- <br /> Tables S18 S19. using stringent criterion,                  tions; tolerant essentiality), TS (tolerant physical interactions; <br /> selected synthetic pairs reported twice independently (694              stringent essentiality) TT (tolerant physical interactions; <br /> nodes, 1621 edges) essential genes present          tolerant essentiality). Additionally, built control <br /> datasets (387 nodes). using tolerant criterion, selected        networks account affinity capture methods synthetic-lethal pairs (2007 nodes, 8055 edges)             identify members complex having straight <br /> essential proteins reported datasets (437 nodes).         physical interaction. Starting strictest network (SS), 1) synthetic-lethal interactions reported               removed analysis physical interactions                                                                             affinity capture methods, resulting twice research group, possible existence                                                                             network containing 3811 edges;  2) removed ambiguity synthetic-lethality synthetic-sickness,                                                                             network physical interaction detected variety sources  Tables S18 S19) make confident                                                                             different non-affinity capture methods, getting network synthetic-lethal interactions selected using stringent <br />                                                                             having 1762 edges. <br /> criteria true positives.  built additional <br /> control lists: high-confident synthetic-lethal pairs, possible synthetic sick pairs. order generate set       Random Models <br /> pairs small chance containing false positives, explored              analyses compared null hypothesis list generated stringent criteria, selected 113       emergence essentiality synthetic lethality purely random <br />                                                                             events. pool genes list yeast genes, pairs reported different publications.                                                                             previously removed analysis <br /> fact 92 different sets publications reassures                                                                             (essential genes participating synthetic-lethality pairs small chance reporting bias. generating list                                                                             hypomorphic mutations).  ensured random <br /> synthetic sick pairs, selected pairs genes involved                                                                             networks essentiality ambiguity excluding genetic interactions termed &#226;&#8364;&#732;&#226;&#8364;&#732;Negative Genetic&#226;&#8364;&#8482;&#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;Synthetic <br />                                                                             randomisation fixed-state genes; e. creating random <br /> Growth Defect&#226;&#8364;&#8482;&#226;&#8364;&#8482;.  discarded pairs                                                                             lists essential genes, excluded members synthetic-lethal <br /> identified participating synthetic lethal interaction,                                                                             pairs; generating random synthetic-lethal pairs, did pairs containing gene identified essential 4             use essential genes. criterion, generated 10000 <br /> datasets previously mentioned. resulted list 53902 pairs       random lists genes assigned essential genes 10000 putative synthetic sick pairs.                                           lists pairs proteins assigned synthetic-lethal pairs.                                                                             lists contained number genes  pairs) original <br /> Protein Complexes                                                           list. lists subsequently used annotate original <br />    used different definitions protein complex.          network physical interactions  Figure S3). P-values defined protein member protein complex           calculated number random models having result equal <br /> detected participating protein interactions means affinity         higher obtained using real data. <br /> capture methods.  used similar strategy                    network itself, randomisation lists synthetic <br /> Michaut coworkers [12]: considered genes annotated               lethal genes poses additional problem: sure term macromolecular complex  0032991)                observed differences identity nodes  genes <br /> children terms, excluding annotations qualifiers               involved synthetic lethality) changes topology &#226;&#8364;&#732; &#226;&#8364;&#8482; &#226;&#8364;&#732;&#226;&#8364;&#732;colocalizes_with&#226;&#8364;&#8482;&#226;&#8364;&#8482;. generated different sets           network  Table S20). control added proteins: 1) experimental evidence         additional random models synthetic lethality annotation;  2) annotated means            degree distribution real network.  additional non-traceable author statement (NAS evidence code),             models topology original synthetic-lethality <br />  <br />  <br /> PLOS | www.plosone.org                                              8                              April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                Protein Interactions Essentiality <br />  <br />  <br /> network. model, substituted node synthetic             Table S4 Number synthetic lethal pairs lethality network gene pool genes.           functional relatedness main contributing model just used genes present synthetic          sources.  pairs selected using tolerant criterion <br /> lethality network reshuffled nodes. model          considered.  pairs reported single difference networks position nodes             study taken account.  model network contain                   (DOCX) <br /> different nodes despite keeping original topology  Table <br />                                                                              Table S5 Percentage synthetic-lethal pairs sharing S20). cases, built 10000 lists, used  <br />                                                                              interactor. <br /> annotate original network. P-values calculated  <br />                                                                              (DOCX) <br /> mentioned. <br />                                                                              Table S6 Percentage physical interactions occurring <br /> Random Exclusion                                                             essential proteins. <br />    randomly excluded synthetic lethal pairs                 (DOCX) <br /> analyses. used different protocols: 1) excluding             Table S7 Percentage physical interactions occurring <br /> number genes (regardless number pairs) real           members synthetic-lethal pairs  includes <br /> data (Model 1); 2) excluding number pairs                    pairs). <br /> (regardless number genes) real data (Model 2).           (DOCX) ran 10000 simulations analysis. P-values calculated                                                          Table S8 Percentage physical interactions occurring <br />                                                                              member synthetic-lethal pair                                                                              essential protein. <br /> Supporting Information <br />                                                                              (DOCX) <br /> Figure S1 Flowchart selection data physical <br />                                                                              Table S9 Analysis effect affinity capture <br /> interactions. discarded data just evidence                                                                              methods detection physical interactions  <br /> BioGRID database. Physical interactions detected using                                                                              volving essential genes members synthetic lethal <br /> different methods reported twice independently                                                                              pairs SS network. P-values calculated comparing selected using stringent criterion. rest physical <br />                                                                              proportions obtained control original <br /> interactions selected tolerant criterion.                                                                              network assuming binomial distribution. <br /> interactions selected using stringent criterion                                                                              (DOCX) <br /> included dataset obtained using tolerant criterion. <br /> (PPTX)                                                                       Table S10 Analysis effect self-interaction                                                                              essential subnetwork. P-values calculated comparing <br /> Figure S2     Flowchart selection data essentiality <br />                                                                              proportions obtained control original <br /> relationships (gene essentiality synthetic lethality). <br />                                                                              network assuming binomial distribution. synthetic lethality interactions present BioGRID <br />                                                                              (DOCX) <br /> database selected using tolerant criterion.  multiple evidences selected using stringent              Table S11 Analysis essentiality transient criterion. case gene essentiality, selected genes       obligate interaction networks (essential subnetwork). <br /> present 3 datasets using tolerant criterion,          P-values calculated comparing proportions assuming <br /> selected present datasets using stringent           binomial distribution. <br /> criterion. did use analyses data leading                    (DOCX) <br /> ambiguity: essential genes involved synthetic lethality interac- <br />                                                                              Table S12 Analysis essentiality transient tions (probably result hypomorphic mutation),                                                                              obligate interaction networks (interactions corresponding synthetic lethal pairs. Obviously, dataset <br />                                                                              products single-essentiality genes). P-values calcu- <br /> selected using stringent criterion subset data <br />                                                                              lated comparing proportions assuming binomial <br /> selected using tolerant criterion. <br />                                                                              distribution. <br /> (PPTX) <br />                                                                              (DOCX) <br /> Figure S3  Annotation physical interaction network <br />                                                                              Table S13 Analysis essentiality transient information &#226;&#8364;&#732;&#226;&#8364;&#732;essentiality&#226;&#8364;&#8482;&#226;&#8364;&#8482;. <br />                                                                              obligate interaction networks (interactions (PPTX) <br />                                                                              members synthetic lethal pairs). P-values calculated <br /> Table S1 Analysis paralogy essential genes                         comparing proportions assuming binomial distribution. <br /> members synthetic-lethal pairs.                                           (DOCX) <br /> (DOCX) <br />                                                                              Table S14 Composition transient obligate phys- <br /> Table S2   Analysis function similarity synthetic-                     ical interaction networks. figure corresponds lethal pairs.                                                                proportion proteins transient network. second <br /> (DOCX)                                                                       figure corresponds proportion proteins obligate <br />                                                                              network. P-values calculated comparing proportions Table S3 Control different features synthetic <br />                                                                              assuming binomial distribution. <br /> lethal synthetic sick pairs. P-values calculated <br />                                                                              (DOCX) <br /> comparing proportions obtained control synthetic lethal pairs selected using stringent criteria         Table S15 Analysis importance evolutionary <br /> assuming binomial distribution.                                            functional factors essential interactome. <br /> (DOCX)                                                                       (DOCX) <br />  <br />  <br /> PLOS | www.plosone.org                                               9                             April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                                            Protein Interactions Essentiality <br />  <br />  <br /> Table S16 Description filtering selecting                                           (DOCX) <br /> datasets physical interactions. <br />                                                                                                Table S20 Features randomized networks (DOCX) <br />                                                                                                synthetic lethal interactions. <br /> Table S17 Sets methodologies commonly used                                             (DOCX) detect physical interactions selected using strin- <br /> gent criterion.                                                                                Acknowledgments <br /> (DOCX) <br />                                                                                                authors wish acknowledge Brad T. Sherman assistance Table S18    Description filtering origin                                        DAVID analysis, Kathyrn Hentges critical reading synthetic-lethal interactions. considered high-throughput                                   manuscript. <br /> experiments reporting 50 interactions, small-scale <br /> experiments reporting interactions, medium-                             Author Contributions <br /> scale experiments reporting 6 50 interactions. <br /> (DOCX)                                                                                         Conceived designed experiments: DT DLR SCL. Performed                                                                                                experiments: DT. Analyzed data: DT SCL. Wrote paper: DT DLR <br /> Table S19 Papers reporting multiple synthetic-lethal                                           SCL. <br /> interactions selected using stringent criterion. <br />  <br /> References <br />  1. Giaever G, Chu  Ni L, Connelly C, Riles L, et al. (2002) Functional                     26. Makino T, Hokamp K, McLysaght (2009) complex relationship gene <br />     profiling Saccharomyces cerevisiae genome. Nature 418: 387&#226;&#8364;&#8220;391.                         duplication essentiality. Trends Genet 25: 152&#226;&#8364;&#8220;155. <br />  2. Jeong H, Mason SP, Barabasi AL, Oltvai ZN (2001) Lethality centrality               27. Liang H, Li WH (2009) Functional compensation duplicated genes mouse. <br />     protein networks. Nature 411: 41&#226;&#8364;&#8220;42.                                                           Trends Genet 25: 441&#226;&#8364;&#8220;442. <br />  3. Coulomb S, Bauer M, Bernard D, Marsolier-Kergoat MC (2005) Gene                            28. VanderSluis B, Bellay J, Musso G, Costanzo M, Papp B, et al. (2010) Genetic <br />     essentiality topology protein interaction networks. Proc Biol Sci 272:              interactions reveal evolutionary trajectories duplicate genes. Mol Syst Biol <br />     1721&#226;&#8364;&#8220;1725.                                                                                     6: 429. <br />  4. Zotenko E, Mestre J, O&#226;&#8364;&#8482;Leary DP, Przytycka TM (2008) hubs                    29. Conant GC, Wagner (2004) Duplicate genes robustness transient gene <br />     yeast protein interaction network tend essential: reexamining                        knock-downs Caenorhabditis elegans. Proc Biol Sci 271: 89&#226;&#8364;&#8220;96. <br />     connection network topology essentiality. PLoS Comput Biol                 30. Papp B, Pal C, Hurst LD (2003) Dosage sensitivity evolution gene <br />     4: e1000140.                                                                                   families yeast. Nature 424: 194&#226;&#8364;&#8220;197. <br />  5. X, Zhang J (2006) hubs tend essential protein networks?                 31. Stuart GR, Copeland WC, Strand MK (2009) Construction application     PLoS Genet 2: e88.                                                                             protein genetic interaction network (yeast interactome). Nucleic Acids Res <br />  6. Papp B, Pal C, Hurst LD (2004) Metabolic network analysis causes                    37: e54. <br />     evolution enzyme dispensability yeast. Nature 429: 661&#226;&#8364;&#8220;664.                          32. Tong AH, Lesage G, Bader GD, Ding H, Xu H, et al. (2004) Global mapping  7. D&#226;&#8364;&#8482;Elia MA, Pereira MP, Brown ED (2009) essential genes really essential?                   yeast genetic interaction network. Science 303: 808&#226;&#8364;&#8220;813. <br />     Trends Microbiol 17: 433&#226;&#8364;&#8220;438.                                                              33. Hakes L, Pinney JW, Lovell SC, Oliver SG, Robertson DL (2007) duplicates <br />  8. Dowell RD, Ryan O, Jansen  Cheung D, Agarwala S, et al. (2010) Genotype                      equal: difference small-scale genome duplication. <br />     phenotype: complex problem. Science 328: 469.                                             Genome Biol 8: R209. <br />  9. Costanzo M, Baryshnikova  Bellay J, Kim Y, Spear ED, et al. (2010)                   34. Hollunder J, Beyer  Wilhelm T (2005) Identification characterization     genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431.                                             protein subcomplexes yeast. Proteomics 5: 2082&#226;&#8364;&#8220;2089. <br /> 10. Kaelin Jr (2005) concept synthetic lethality context                   35. Valente AX, Roberts SB, Buck GA, Gao Y (2009) Functional organization     anticancer therapy. Nat Rev Cancer 5: 689&#226;&#8364;&#8220;698.                                                 yeast proteome yeast interactome map. Proc Natl Acad Sci U S 106: <br /> 11. Kelley R, Ideker T (2005) Systematic interpretation genetic interactions using              1490&#226;&#8364;&#8220;1495. <br />     protein networks. Nat Biotechnol 23: 561&#226;&#8364;&#8220;566.                                              36. Le Meur N, Gentleman R (2008) Modeling synthetic lethality. Genome Biol 9: <br /> 12. Michaut M, Baryshnikova  Costanzo M, Myers CL, Andrews BJ, et al. (2011)                     R135. <br />     Protein complexes central yeast genetic landscape. PLoS Comput Biol             37. Hannum G, Srivas R, Guenole  van Attikum H, Krogan NJ, et al. (2009) <br />     7: e1001092.                                                                                   Genome-wide association data reveal global map genetic interactions <br /> 13. Feiglin  Moult J, Lee B, Ofran Y, Unger R (2012) Neighbor overlap                         protein complexes. PLoS Genet 5: e1000782. <br />     enriched yeast interaction network: analysis implications. PLoS             38. Costanzo M, Baryshnikova  Bellay J, Kim Y, Spear ED, et al. (2010)     7: e39662.                                                                                     genetic landscape cell. Science 327: 425&#226;&#8364;&#8220;431. <br /> 14. Szappanos B, Kovacs K, Szamecz B, Honti F, Costanzo M, et al. (2011)                    39. Bandyopadhyay S, Kelley R, Krogan NJ, Ideker T (2008) Functional maps     integrated approach characterize genetic interaction networks yeast                      protein complexes quantitative genetic interaction data. PLoS Comput Biol <br />     metabolism. Nat Genet 43: 656&#226;&#8364;&#8220;662.                                                             4: e1000065. <br /> 15. Tischler J, Lehner B, Fraser AG (2008) Evolutionary plasticity genetic                  40. Baryshnikova  Costanzo M, Kim Y, Ding H, Koh J, et al. (2010) Quantitative <br />     interaction networks. Nat Genet 40: 390&#226;&#8364;&#8220;391.                                                   analysis fitness genetic interactions yeast genome scale. Nat <br /> 16. Li J, Yuan  Zhang (2010) cellular robustness genetic redundancy                  Methods 7: 1017&#226;&#8364;&#8220;1024. <br />     budding yeast. PLoS Genet 6: e1001187.                                                     41. Guan Y, Dunham MJ, Troyanskaya OG (2007) Functional analysis gene <br /> 17. Wagner (2005) Distributed robustness versus redundancy causes                          duplications Saccharomyces cerevisiae. Genetics 175: 933&#226;&#8364;&#8220;943. <br />     mutational robustness. Bioessays.                                                          42. Tong AH, Evangelista M, Parsons AB, Xu H, Bader GD, et al. (2001) <br /> 18. Ihmels J, Collins SR, Schuldiner M, Krogan NJ, Weissman JS (2007) Backup                       Systematic genetic analysis ordered arrays yeast deletion mutants. <br />     redundancy: genetic interactions reveal cost duplicate gene loss.               Science 294: 2364&#226;&#8364;&#8220;2368. <br />     Molecular Systems Biology 3: 86.                                                           43. Schuldiner M, Collins SR, Thompson NJ, Denic V, Bhamidipati  et al. (2005) <br /> 19. DeLuna  Springer M, Kirschner MW, Kishony R (2010) Need-based                             Exploration function organization yeast early secretory <br />     regulation protein levels response deletion duplicate genes. PLoS            pathway epistatic miniarray profile. Cell 123: 507&#226;&#8364;&#8220;519. <br />     Biol 8: e1000347.                                                                          44. Collins SR, Roguev  Krogan NJ (2010) Quantitative genetic interaction <br /> 20. Dean EJ, Davis JC, Davis RW, Petrov DA (2008) Pervasive persistent                         mapping using E-MAP approach. Methods Enzymol 470: 205&#226;&#8364;&#8220;231. <br />     redundancy duplicated genes yeast. PLoS Genet 4: e1000113.                        45. Lehner B (2011) Molecular mechanisms epistasis genes. <br /> 21. Doyle MA, Gasser RB, Woodcroft BJ, Hall RS, Ralph SA (2010) Drug target                        Trends Genet 27: 323&#226;&#8364;&#8220;331. <br />     prediction prioritization: using orthology predict essentiality parasite         46. Yu H, Braun P, Yildirim MA, Lemmens  Venkatesan K, et al. (2008) High- <br />     genomes. BMC Genomics 11: 222.                                                                 quality binary protein interaction map yeast interactome network. Science <br /> 22. Gu  Steinmetz LM, Gu X, Scharfe C, Davis RW, et al. (2003) Role                           322: 104&#226;&#8364;&#8220;110. <br />     duplicate genes genetic robustness null mutations. Nature 421: 63&#226;&#8364;&#8220;66.           47. Hart GT, Lee  Marcotte ER (2007) high-accuracy consensus map yeast <br /> 23. Liao  Zhang J (2007) Mouse duplicate genes essential singletons.                  protein complexes reveals modular nature gene essentiality. BMC Bioinfor- <br />     Trends Genet 23: 378&#226;&#8364;&#8220;381.                                                                      matics 8: 236. <br /> 24. Liang H, Li WH (2007) Gene essentiality, gene duplicability protein                    48. Ozbabacan SE, Engin HB, Gursoy  Keskin O (2011) Transient protein- <br />     connectivity human mouse. Trends Genet 23: 375&#226;&#8364;&#8220;378.                                     protein interactions. Protein Eng Des Sel 24: 635&#226;&#8364;&#8220;648. <br /> 25. Su  Gu X (2008) Predicting proportion essential genes mouse                    49. Zhong Q, Simonis N, Li QR, Charloteaux B, Heuze F, et al. (2009) Edgetic <br />     duplicates based biased mouse knockout genes. J Mol Evol 67: 705&#226;&#8364;&#8220;709.                       perturbation models human inherited disorders. Mol Syst Biol 5: 321. <br />  <br />  <br />  <br /> PLOS | www.plosone.org                                                                10                                    April 2013 | Volume 8 | Issue 4 | e62866 <br />                                                                                                                                        Protein Interactions Essentiality <br />  <br />  <br /> 50. Pereira-Leal JB, Audit B, Peregrin-Alvarez JM, Ouzounis CA (2005)                    56. Cherry JM, Ball C, Weng S, Juvik G, Schmidt R, et al. (1997) Genetic     exponential core heart yeast protein interaction network. Mol Biol            physical maps Saccharomyces cerevisiae. Nature 387: 67&#226;&#8364;&#8220;73. <br />     Evol 22: 421&#226;&#8364;&#8220;425.                                                                       57. Kersey PJ, Lawson D, Birney E, Derwent PS, Haimel M, et al. (2010) Ensembl <br /> 51. Dixon SJ, Fedyshyn Y, Koh JL, Prasad TS, Chahwan C, et al. (2008) Significant               Genomes: extending Ensembl taxonomic space. Nucleic Acids Res 38: <br />     conservation synthetic lethal genetic interaction networks distantly             D563&#226;&#8364;&#8220;569. <br />     related eukaryotes. Proc Natl Acad Sci U S 105: 16653&#226;&#8364;&#8220;16658.                          58. Smedley D, Haider S, Ballester B, Holland R, London D, et al. (2009) BioMart&#226;&#8364;&#8220; <br /> 52. Kreimer  Borenstein E, Gophna U, Ruppin E (2008) evolution                         biological <br /> </body></html>