<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>APPLIED ENVIRONMENTAL MICROBIOLOGY, Sept. 2011, p. 6133&#226;&#8364;&#8220;6140                                                              Vol. 77,  17 <br /> 0099-2240/11/$12.00 doi:10.1128/AEM.00630-11 <br /> Copyright &#194;&#169; 2011, American Society Microbiology. Rights Reserved. <br />  <br />  <br />  <br />           Determining Extremes Cellular NAD(H) Level             Using Escherichia coli NAD&#226;&#171;&#185;-Auxotrophic Mutant&#228;&#338;&#164; <br />   Yongjin Zhou,1,3 Lei Wang,1,3 Fan Yang,1 Xinping Lin,1,3 Sufang Zhang,1 Zongbao K. Zhao1,2* <br /> Dalian Institute <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> Physics, Chinese Academy Sciences, Dalian 116023, People&#226;&#8364;&#8482;s Republic China1; Dalian National <br />               Laboratory Clean Energy, Dalian 116023, People&#226;&#8364;&#8482;s Republic China2; Graduate University                            Chinese Academy Sciences, Beijing 100049, People&#226;&#8364;&#8482;s Republic China3 <br />                                                Received 21 March 2011/Accepted 28 June 2011 <br />  <br />              NAD (NAD&#226;&#180;&#8482;) reduced form (<span id='am-11' about='obo:IMR_0200521' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-12' property="rdfs:label" content="NADH" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00004" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200521" datatype="xsd:string"></span>NADH</span>) omnipresent cofactors biological systems.            difficult determine extremes cellular NAD(H) level live cells NAD&#226;&#180;&#8482; level tightly <br />           controlled biosynthesis regulation mechanism.  developed strategy determine extreme <br />           NAD(H) levels Escherichia coli cells genetically engineered NAD&#226;&#180;&#8482; auxotrophic.            expressed ntt4 gene encoding NAD(H) transporter E. coli mutant YJE001, deletion <br />           nadC gene responsible NAD&#226;&#180;&#8482; novo biosynthesis, showed NTT4 conferred mutant <br />           strain better growth presence exogenous NAD&#226;&#180;&#8482;. constructed NAD&#226;&#180;&#8482;-auxotrophic mutant <br />           YJE003 disrupting essential gene nadE, responsible step NAD&#226;&#180;&#8482; biosynthesis           cells harboring ntt4 gene. minimal NAD&#226;&#180;&#8482; level determined M9 medium proliferating YJE003 <br />           cells preloaded NAD&#226;&#180;&#8482;, maximal NAD(H) level determined exposing <br />           cells high concentrations exogenous NAD(H). Compared supplementation NADH, cells <br />           grew faster higher intracellular NAD(H) level NAD&#226;&#180;&#8482; fed. intracellular NAD(H) <br />           level increased increase exogenous NAD&#226;&#180;&#8482; concentration, reached plateau.            minimal NAD(H) level 0.039 mM maximum 8.49 mM determined, 0.044&#226;&#180;&#8250;           9.6&#226;&#180;&#8250; wild-type cells, respectively. Finally, potential application strategy biotech- <br />           nology briefly discussed. <br />  <br />  <br />    NAD (NAD&#226;&#171;&#185;; reduced state NADH) plays                              inner membrane, NAD&#226;&#171;&#185; se im- <br /> essential role cellular metabolism function                 permeable bacterial cytoplasmic membrane (Fig. 1B) <br /> cofactor 300 redox reactions (10).  NAD&#226;&#171;&#185;                      (21). NAD&#226;&#171;&#185; transporter, NTT4, discovered level NADH/NAD&#226;&#171;&#185; ratio determine metabolic                          prokaryote chlamydial endosymbiont Protochlamydia <br /> fluxes pathways transcriptional level                   amoebophila UWE25, transport intact NAD(H) genes vivo (13, 14). example, high NADH/                      counterexchange <span id='am-5' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span> (11). Recently, NAD&#226;&#171;&#185; <br /> NAD&#226;&#171;&#185; ratio triggers overflow metabolism (33), low                        carriers located chloroplasts mitochondria NADH/NAD&#226;&#171;&#185; ratio enhances glycolytic flux (38).                             discovered eukaryotes Saccharomyces cerevisiae Manipulating NADH/NAD&#226;&#171;&#185; ratio considered                          Arabidopsis thaliana (20, 28). transporters import effective strategies improving productivity                   NAD&#226;&#171;&#185;, counterexchange nucleotides (ADP, <span id='am-16' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-19' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span>AMP</span>, products (19, 23). metabolic path-                       <span id='am-97' about='obo:IMR_0200292' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-98' property="rdfs:label" content="GMP" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:id" content="IMR:0200292" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasDbXref" content="KEGG:C00144" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>GMP</span>), cells organelles lack NAD&#226;&#171;&#185; biosynthesis <br /> ways NAD&#226;&#171;&#185; biosynthesis Escherichia coli, novo                    capacity support biological process (Fig. 1B). <br /> pathway salvage pathway (3, 15). novo path-                        level intracellular NAD(H) tightly controlled way starts oxidation aspartate iminosuccinic                    variety mechanisms, including biosynthesis sal- <br /> acid, catalyzed NadB turn reacts                       vage NadR regulation (10). internal <br /> dihydroxyacetone phosphate quinolinic acid (QA),                       NAD(H) level low, NadR interacts integral mem- <br /> followed phosphoribosylation decarboxylation,                        brane protein PnuC, responsible transport <br /> sulting <span id='am-102' about='obo:IMR_0200415' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-103' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:id" content="IMR:0200415" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="Nicotinic acid" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00253" datatype="xsd:string"></span>nicotinic acid</span> mononucleotide (NAMN). sal-                      NMN cytoplasmic membrane, promoting net <br /> vage pathway converts precursors (nicotinic acid                       NMN transportation cell. NAD(H) level <br /> [NA], <span id='am-92' about='obo:IMR_0200025' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-93' property="oboInOwl:id" content="IMR:0200025" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Nicotinamide" datatype="xsd:string"></span><span id='am-95' property="oboInOwl:hasDbXref" content="KEGG:C00153" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nicotinamide</span> [NM], nicotinamide mononucle-                           increases, NadR binds operator regions target <br /> otide [NMN]) NAMN, converted                            genes (nadA, nadB, pncB) represses transcrip- <br /> NAD&#226;&#171;&#185; adenylation amidation catalyzed                            tion decrease biosynthesis.  studies <br /> NadD NadE, respectively (Fig. 1A). Like                        succeeded enhancing NAD(H) pool manip- <br /> precursors, NA, NMN taken                           ulating biosynthetic pathway regulation <br /> medium (25);  extracellular NAD&#226;&#171;&#185; degraded                         overexpression pncB control <br /> NMN <span id='am-62' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-63' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-70' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-72' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-74' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span>adenosine</span> periplasm passing                              native operator promoter sequences, intra- <br />                                                                                cellular NAD(H) increased 5-fold cells cultured <br />                                                                                presence 0.1 mM exogenous nicotinic acid (34). <br />    * Corresponding author. Mailing address: Dalian Institute Chem- <br /> ical Physics, CAS, Dalian 116023, People&#226;&#8364;&#8482;s Republic China. Phone <br />                                                                                pncB mutated operator region protect fax: 86-411-84379211. E-mail: zhaozb@dicp.ac.cn.                           regulatory effect NadR overexpressed <br />    &#228;&#338;&#164; <br />      Published ahead print 8 July 2011.                                  lac promoter, resulting 2.2-fold increase  <br />                                                                         6133 <br />  6134     ZHOU ET AL.                                                                                              APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />  <br />  <br />    FIG. 1. Overview NAD&#226;&#171;&#185; metabolism.   NAD&#226;&#171;&#185; biosynthetic salvage pathways E. coli. (B) Schematic representation linkage NAD&#226;&#171;&#185; biosynthesis transport pathway. Dashed lines NAD&#226;&#171;&#185; transport model chloroplasts mitochondria, <br /> discovered eukaryotes S. cerevisiae  thaliana; solid lines refer NAD&#226;&#171;&#185; metabolic pathway transport pathway prokaryotes E. <br /> coli Chlamydia, respectively. Abbreviations: ASP, asparate; DHAP, dihydroxyacetone phosphate; ImASP, imino asparate; QA, quinolinate; <br /> NA, nicotinate; NM, nicotinamide; NMN, nicotinamide mononucleotide; NmR, nicotinamide ribonucleoside; NaR, nicotinate ribonucleoside; <br /> NAMN, nicotinic acid mononucleotide; dNAD, deamino NAD&#226;&#171;&#185;; Ade, <span id='am-77' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-78' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-79' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-80' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-85' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-87' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-88' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-89' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-90' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-91' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span>adenosine</span>. Enzymes: NadB, L-aspartate oxidase; NadA, quinolinate <br /> synthetase; NadC, quinolinate phosphoribosyltransferase; NadD, NAMN adenyltransferase; NadE, NAD&#226;&#171;&#185; synthetase; UshA, NMN glycohydro- <br /> lase; DeoD, purine-<span id='am-57' about='obo:IMR_0001697' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-58' property="rdfs:label" content="nucleoside" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001697" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="CHEBI:25611" datatype="xsd:string"></span>nucleoside</span> phosphorylase PncA, Nm deamidase; PncB, NA phosphoribosyltransferase; NudC, NAD&#226;&#171;&#185; pyrophosphatase; NadK, <br /> NAD&#226;&#171;&#185; kinase. ScNDT1 ScNDT2, NAD&#226;&#171;&#185; carriers NDT1 NDT2 S. cerevisiae; AtNDT1 AtNDT2, NAD&#226;&#171;&#185; carriers NDT1 NDT2  thaliana. <br />  <br />  <br />  NAD(H) level (22). Cooverexpression pncB                   study, manipulated NAD(H) level feeding <br /> nadE gene led dramatic 7-fold increase                   NAD(H) NAD&#226;&#171;&#185;-auxotrophic E. coli mutant NAD(H) level (12). NAD(H) level reduced                      constructed disrupting essential gene nadE, nadD mutation, encoded NAMN adeny-                     responsible step NAD&#226;&#171;&#185; biosynthesis cells <br /> lyltransferase (25).                                                      NAD&#226;&#171;&#185; transporter NTT4 expression background. <br />    NAD(H) pool adjusted manipu-                    led successful determination extremes lating biosynthetic pathway, remains difficult inves-           intracellular NAD(H) level. Finally, discuss potential <br /> tigate effects extremes NAD(H) level cell                application results fields biotechnology  <br /> growth corresponding self-regulation mechanism.                lated cofactor engineering. <br />  VOL. 77, 2011                                                                           EXTREMES CELLULAR NAD(H) LEVEL                                   6135 <br />  <br />  <br />                                                     TABLE 1. Strains plasmids used study <br />  Strain plasmid                                                 Genotype characteristic                                                Resource reference <br />  <br /> E. coli strains <br />   DH5&#226;?&#163;                         F&#226;&#171;&#186; &#226;?&#190;80dlacZ&#226;&#338;&#172;M15 &#226;&#338;&#172;(lacZYA-argF)U169 deoR recA1 endA1 hsdR17(rk&#226;&#171;&#186; mk&#226;&#171;&#185;) phoA supE44                                     TaKaRa <br />                                   &#226;?&#173;&#226;&#171;&#186; thi-1 gyrA96 relA1 <br />   BW25113                      rrnB3 &#226;&#338;&#172;lacZ4787 hsdR514 &#226;&#338;&#172;(araBAD)567 &#226;&#338;&#172;(rhaBAD)568 rph-1                                                             8 <br />   JW0105                       BW25113/nadC::kan                                                                                                   2 <br />   YJE001                       BW25113/nadC::kan/pET15b-NTT4                                                                                       study <br />   YJE002                       BW25113/pKD46/pET15K-NTT4                                                                                           study <br />   YJE003                       BW25113/pET15K-NTT4/nadE::cat                                                                                       study <br />  <br /> Plasmids <br />   pMD18-T                      lacZ pBR322 ori bla; cloning vector                                                                                 TaKaRa <br />   pET15b                       T7 promoter, lacI pBR322 ori bla                                                                                    Novagen <br />   pET15b-NTT4                  ntt4 inserted NdeI-BamHI site, NTT4 expression, bla                                                              study <br />   pET15K-NTT4                  ntt4 inserted NdeI-BamHI site, NTT4 expression, kan                                                              study <br />   pKD3                         oriR6K&#226;?&#165; bla cat rgnB(Ter)                                                                                           8 <br />   pKD4                         oriR6K&#226;?&#165; bla kan rgnB(Ter)                                                                                           8 <br />   pKD46                        araBp-gam-bet-exo bla repA101(Ts) oriR101                                                                           8 <br />  <br />  <br />  <br />                           MATERIALS METHODS                                      Briefly, reaction mixture consisted plasmid (200 ng), megaprimer <br />                                                                                     Kan (600 ng), 5&#226;&#171;&#187; PrimeSTAR buffer (10 &#226;?&#174;l), 2.5 mM deoxynucleoside triphos- <br />    Bacterial strains plasmids. Cloning plasmid propagation  <br />                                                                                     phates (6 &#226;?&#174;l), PrimeSTAR HS DNA polymerase (1.25 U) total volume 50 <br /> formed E. coli DH5&#226;?&#163;. E. coli BW25113(pKD3, pKD4, pKD46) kindly <br />                                                                                     &#226;?&#174;l. thermocycling conditions 95&#194;&#176;C 5 min, 35 cycles 95&#194;&#176;C 1 <br /> provided B. L. Wanner (Purdue University). E. coli JW0105 obtained <br />                                                                                     min, 55&#194;&#176;C 45 s, 68&#194;&#176;C 6 min, final 68&#194;&#176;C 20 min. reaction KEIO knockout collection E. coli BW25113 background <br /> (2). strains plasmids used listed Table 1.                        mix treated DpnI 37&#194;&#176;C 4 h digest methylated parental <br />    Growth media conditions. E. coli routinely cultivated agitation     plasmid transformed E. coli DH5&#226;?&#163; competent cells. 37&#194;&#176;C 30&#194;&#176;C  gene disruption) 200 rpm LB broth (10 g             nadE 200-bp flanking regions amplified E. coli BW25113 <br /> tryptone [Oxoid], 5 g yeast extract [Oxoid], 10 g NaCl liter water), SOB        genomic DNA using primer pair nadE-F/nadE-R cloned broth (20 g tryptone [Oxoid], 5 g yeast extract [Oxoid], 0.5 g NaCl, 2.5 mM         pMD18-T. Disruption cassettes nadE constructed replacing nadE <br /> KCl, 1 mM MgCl2, 1 mM MgSO4 liter water), M9 minimal medium (6               cat amplified pKD3 primer pair cat(E)-F/cat(E)-R, resulting <br /> g Na2HPO4, 3 g KH2PO4, 0.5 g NaCl, 1 g NH4Cl, 1 mM MgSO4, 0.1 mM                    pMD18T-nadE::CAT. <br /> CaCl2 liter water) containing 2% glucose, 1,000&#226;&#171;&#187; trace metal mix (27           Deletion nadE performed described previously (8) minor <br /> g FeCl3 &#228;&#161;&#160; 6H2O, 2 g ZnCl2 &#228;&#161;&#160; 4H2O, 2 g CaCl2 &#228;&#161;&#160; 2H2O, 2 g Na2MoO4 &#228;&#161;&#160; 2H2O,             modification. nadE disruption cassette nadE::CAT PCR amplified 1.9 g CuSO4 &#228;&#161;&#160; 5H2O, 0.5 g H3BO3 liter water). strains inocu-           pMD18T-nadE::CAT prime pair nadE-F/nadE-R transformed <br /> lated overnight cultures 37&#194;&#176;C, washed 2 times M9 medium,           E. coli YJE002 electroporation. mutants selected SOB <br /> transferred 50 ml M9 medium supplemented NAD(H)                  plates supplemented kanamycin sulfate (50 &#226;?&#174;g/ml), chloramphenicol (30 <br /> optical density 600 nm (OD600) 0.02. Samples regularly taken         &#226;?&#174;g/ml) NAD&#226;&#171;&#185; (100 &#226;?&#174;M). <br /> measurement OD600 NAD(H).                                                   Cofactor measurements. Cofactor recycling assay used determine  <br />    Reagents. Pfu DNA polymerase supplied Tiangen Biotech (Beijing,           tracellular concentrations NAD(H) NADP(H) minor modifications <br /> China). primers used study (Table 2) purchased Invitrogen     (7, 22). assay, 5 &#226;&#171;&#187; 109 cells collected centrifugation (Shanghai, China), DNA sequencing performed TaKaRa (Dalian,              8,000 &#226;&#171;&#187; g 2 min cell pellets washed twice ice-cold phos- <br /> China). DNA gel purification kit plasmid extraction kit purchased        phate-buffered saline (PBS) buffer.  300 &#226;?&#174;l 0.2 M NaOH  NAD(P)H Beyotime (Haimen, China). DpnI purchased New England Bio-             extraction] HCl  NAD(P)&#226;&#171;&#185; extraction] added cell pellets, <br /> Labs (Beijing, China), PrimeSTAR HS DNA polymerase                heated 10 min 55&#194;&#176;C. extracts neutralized reagents purchased TaKaRa (Dalian, China). chemicals             adding 300 &#226;?&#174;l 0.1 M HCl  NAD(P)H extraction] NaOH  purchased Sigma (Shanghai, China).                                             NAD(P)&#226;&#171;&#185; extraction]. cellular debris removed centrifuging    DNA manipulation. ntt4 gene (11) custom synthesized TaKaRa            12,000 &#226;&#171;&#187; g 5 min. Supernatants transferred new tubes stored <br /> (Dalian, China) cloned NdeI-BamHI site pET15b         &#226;&#171;&#186;20&#194;&#176;C 24 h. <br /> primer pair NTT4-F/NTT4-R, resulting NTT4 expression vector pET15b-             cycling assay performed using reagent mixture consisting equal <br /> NTT4. pET15K-NTT4 constructed substituting bla gene kan             volumes 1.0 M Bicine buffer (pH 8.0), absolute ethanol  NAD(H) <br /> amplified Kan-F/Kan-R, using restriction-free (RF) cloning strategy        assay] 30 mM glucose-6-phosphate  NADP(H) assay], 40 mM EDTA <br /> (32). RF cloning experiment according procedure described       (pH 8.0), 4.2 mM 3-(4,5-dimethyl-2-thiazolyl)-2,5-diphenyl-2H-tetrazolium bro- <br />  <br />  <br />  <br />                                                             TABLE 2. Primers used study <br />  Primer                                                             Sequence (5&#226;&#172;&#732;&#226;&#8364;&#8220;3&#226;&#172;&#732;                                                                 Function <br />  <br /> NTT4-F                GAGCATATGAGTAAAACAAACCAG                                                                                                   ntt4 cloning <br /> NTT4-R                TACGGATCCTTAGTGATGATGATGATGATGTTTTTTTATAAAAG <br /> Kan-F                 TTAAACATGCCAGTGATGCAAAGGTAGTGCAAGAGCTATGACATATACCTGCCGTTCAC                                                                kan cloning <br /> Kan-R                 TCAGGCGGTCAGTGTATCATCACTCATACTCTGCCCGACACCATGGTCATAGCTGTTTC <br /> nadE-F                GTTAATACCGCCGCTGCACTCAG                                                                                                    nadE cloning <br /> nadE-R                TCACCATGAAACAGATACACCCCTG <br /> cat(E)-F              TGTCTTTTCTGTCTGGAGGGGTTCAATGATGGGAATTAGCCATGGTCC                                                                           cat amplification <br /> cat(E)-R              CAAATTATTACTTTTTCCAGAAATCATCGTGTAGGCTGGAGCTGCTTC <br />         restriction sites underlined. <br />  6136       ZHOU ET AL.                                                                                                       APPL. ENVIRON. MICROBIOL. <br />  <br /> mide, twice volume 16.6 mM phenazine ethosulfate, 3 volumes water, previously incubated 25&#194;&#176;C. following volumes added 1-ml <br /> cuvettes: 50 &#226;?&#174;l neutralized extract, 0.9 ml reagent mixture. NAD(H) <br /> assay, reaction started adding 50 &#226;?&#174;l yeast ADH II (500 U/ml Bicine buffer). Fifty microliters yeast glucose-6-phosphate dehydrogenase (10 <br /> U/ml) added start NADP(H) assay. absorbance 570 nm recorded 2 min 25&#194;&#176;C. volume cell 10&#226;&#171;&#186;12 ml, cell <br /> concentrations calculated based assumption OD600 1.0 <br /> corresponds 109 cells/ml (5);  cell density correlated cell <br /> volume, 1 ml/liter equivalent OD600 1. <br />    Determination minimal NAD(H) level. colonies NAD&#226;&#171;&#185;-auxo- <br /> trophic mutant YJE003 picked LB medium containing 100 &#226;?&#174;M NAD&#226;&#171;&#185; cultivated 12 h agitation 37&#194;&#176;C, 200 rpm.  cells inoculated 100 ml M9 medium supplemented 100 &#226;?&#174;M NAD&#226;&#171;&#185; OD600 0.02 cultivated early exponential phase (OD600, &#226;&#172;&#8225;0.8). cells collected washed twice M9 medium inoculated 50 ml M9 medium, initial OD600 0.1, 0.2, 0.3, respectively, cultivated stationary phase, cofactor level minimal <br /> level E. coli growth. cells collected washed ice-cold PBS <br /> buffer used intracellular NAD(H) measurements. <br />    Determination maximal NAD(H) level. NAD&#226;&#171;&#185;-auxotrophic mu- <br /> tant YJE003 picked LB medium containing 100 &#226;?&#174;M NAD&#226;&#171;&#185; cultivated 12 h agitation 37&#194;&#176;C, 200 rpm. cells collected washed twice M9 medium, inoculated 50 ml M9 medium <br /> supplemented different concentrations NAD&#226;&#171;&#185;, initial OD600 0.02 cultivated agitation 37&#194;&#176;C, 200 rpm stationary phase. cells used intracellular NAD(H) measurements washing twice ice-cold PBS buffer. <br />  <br />  <br />                                   RESULTS <br />                                                                                        FIG. 2. Verification function NTT4 &#226;&#338;&#172;nadC mutant. <br />    Verification NTT4 function &#226;&#338;&#172;nadC mutant.                                  NTT4 function verified expressing E. coli JW0105, <br /> NAD(H) se impermeable bacterial cytoplasmic                            lacks novo NAD&#226;&#171;&#185; biosynthesis pathway nadC <br />                                                                                      disruption. (B) NTT4-expressing strain YJE001 better growth membrane (1, 21), usually intermediates,                         control strain cultivated M9 medium containing 100 &#226;?&#174;M AMP, NMN, NA, directly assimilated. Recently,                          NAD&#226;&#171;&#185;. NAD&#226;&#171;&#185; transporter, NTT4, discovered chlamyd- <br /> ial endosymbiont P. amoebophila UWE25 (11). properties transporter studied cloning ntt4 gene E.                    BW25113 obtained disruption nadE gene obtain <br /> coli analyzing transportation activity 32P-labeled                      NAD&#226;&#171;&#185;-auxotrophic mutant YJE003. mutant strain <br /> nucleotides cells resting state.  remains                      grew LB plates supplemented NAD&#226;&#171;&#185; unknown transportation activity NTT4 suf-                         NADH failed grow plates NAD&#226;&#171;&#185; intermediates, <br /> ficient support cell growth exogenous NAD&#226;&#171;&#185;.                             NA NMN (Fig. 3B). <br /> structed E. coli YJE001, ntt4 gene expressed                           Comparison cofactor levels exogenous <br /> E. coli JW0105, strain lacks novo NAD&#226;&#171;&#185; biosyn-                         NAD&#226;&#180;&#8482;- NADH-fed cells. previous study showed thesis pathway nadC disruption (Fig. 2A).                        NTT4 transport NAD&#226;&#171;&#185; NADH YJE001 grew 20% faster exponential phase                        NADH uptake rate little lower resting cells (11), presence NAD&#226;&#171;&#185; medium (Fig. 2B),                              growing cells NTT4 transportation activity differences <br /> wild-type BW25113 cells insensitive exogenous NAD&#226;&#171;&#185;                           NAD&#226;&#171;&#185; NADH remained undetermined.  <br /> (data shown), suggesting NTT4 protein func-                         fore, examined effects concentration exoge- <br /> tional NAD&#226;&#171;&#185; transporter E. coli. surmised                       nous NAD&#226;&#171;&#185; NADH intracellular cofactor levels activity NTT4 able rescue cells NAD&#226;&#171;&#185;                         cell growth profile. turned cells grew <br /> biosynthesis completely blocked. test conjecture,                        faster fed NAD&#226;&#171;&#185;, instead NADH. cells fed NAD&#226;&#171;&#185;, decided disrupt essential gene nadE,                           total intracellular NAD(H) levels 4.82 mM 14 h volved step NAD&#226;&#171;&#185; biosynthesis.                                        7.55 mM 26 h, higher levels    Construction NAD&#226;&#180;&#8482;-auxotrophic mutant nadE dis-                          fed NADH (Table 3). results sug- <br /> ruption. order manipulate intracellular NAD&#226;&#171;&#185; level,                        gested NTT4 prefers NAD&#226;&#171;&#185; NADH,  attempted remove NAD&#226;&#171;&#185; regulation effect dis-                            sistent results observed using resting cells (11). rupting essential gene nadE, responsible                        NAD&#226;&#171;&#185;/NADH ratio higher NAD&#226;&#171;&#185;- step NAD&#226;&#171;&#185; biosynthesis.  impractical                        fed cells NADH-fed cells, indicating NAD&#226;&#171;&#185;-fed cells did access exogenous NAD&#226;&#171;&#185; (2).                           cells higher oxidative state. <br /> &#226;&#338;&#172;nadC mutant carrying ntt4 gene showed advantageous                              Determination minimal cofactor level E. coli cell <br /> growth profile presence exogenous NAD&#226;&#171;&#185;,                                 proliferation. studies successfully perturbed thought possible construct NAD&#226;&#171;&#185;-auxotro-                           intracellular cofactor level, especially increasing NAD&#226;&#171;&#185; <br /> phic mutant NTT4 expression background gener-                             level (34) manipulation NAD&#226;&#171;&#185;/NADH ratio (22), <br /> ated (Fig. 3A).  expressed ntt4 gene E. coli                          difficult determine minimal cofactor level  VOL. 77, 2011                                                                   EXTREMES CELLULAR NAD(H) LEVEL                           6137 <br />  <br />  <br />  <br />  <br />    FIG. 3. Construction characterization NAD&#226;&#171;&#185;-auxotro- <br /> phic mutant YJE003.   NAD&#226;&#171;&#185;-auxotrophic mutants  <br /> structed disrupting nadE gene NTT4 expression  <br /> ground. (B) Phenotype NAD&#226;&#171;&#185;-auxotrophic mutant YJE003. 109 overnight cultured cells suspended 1 ml double- <br /> distilled <span id='am-44' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-45' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-47' property="rdfs:label" content="H2O" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span>H2O</span> (OD600, 0.1), 10-&#226;?&#174;l aliquots dilutions 10&#226;&#171;&#186;1        FIG. 4. Determination minimal cofactor level cell <br /> 10&#226;&#171;&#186;6 spotted corresponding plates  left right).          growth, based NAD&#226;&#171;&#185; distribution strategy NAD&#226;&#171;&#185;- <br />                                                                              auxotrophic mutant YJE003.   Schematic NAD&#226;&#171;&#185; distribution <br />                                                                              strategy. auxotrophic mutant YJE003 preloaded certain <br />                                                                              NAD&#226;&#171;&#185; cultivating M9 medium containing 100 &#226;?&#174;M <br />                                                                              NAD&#226;&#171;&#185; early log phase (OD600, &#226;&#172;&#8225;0.8), washed twice M9  <br /> cell proliferation. NAD&#226;&#171;&#185; level regulated <br />                                                                              dium, transferred M9 medium NAD&#226;&#171;&#185;. cells NAD&#226;&#171;&#185; biosynthesis pathways, activated supply                 proliferated, cofactor distributed daughter cells NAD&#226;&#171;&#185; NAD&#226;&#171;&#185; level low. NAD&#226;&#171;&#185;-auxo-                         cellular cofactor limited cell division. (B) Growth curve trophic mutant, minimal cofactor level cell growth                   YJE003 M9 medium NAD&#226;&#171;&#185; different inoculation determined based NAD&#226;&#171;&#185; distribution strategy                  densities (0.1 0.3). (C) Intracellular cofactor concentrations                                                                              measured cells reached stationary phase. data <br /> (Fig. 4A), inspired plasmid distribution                    represent averages &#226;&#171;&#190; standard deviations model (31). Briefly, auxotrophic mutant YJE003 pre-                  independent samples. <br /> loaded certain NAD&#226;&#171;&#185; transferred minimal medium NAD&#226;&#171;&#185;. cells divided cofactor distributed daughter cells cel-              0.3, maximal OD600 1.7 (Fig. 4B), lular cofactor level reached minimum limited cell                   lower OD600 2.7 obtained YJE003 cells <br /> division. interesting matter initial              cultivated presence 100 &#226;?&#174;M NAD&#226;&#171;&#185; (Fig. 5A). <br /> OD600 0.1, 0.2, 0.3, YJE003 cells reached stationary              observations suggested cofactor level  <br /> phase cultivation 4 h, cell density                  deed limiting cell division. <br /> increased 4-fold. example, initial OD600                    cell density YJE003 increased 4-fold M9  <br />  <br />  <br />                             TABLE 3. Intracellular NAD(H) levels E. coli YJE003 cells fed 100 &#226;?&#174;M NAD&#226;&#171;&#185; NADH <br />                                                                            Intracellular levela (mM) <br />  Feed                   Duration (h) feeding                                                                                   NAD&#226;&#171;&#185;/NADH ratio <br />                                                           NAD&#226;&#171;&#185;                     NADH                    NAD(H) <br />  <br /> NAD&#226;&#171;&#185;                              14                   4.26 &#226;&#171;&#190; 0.55               0.56 &#226;&#171;&#190; 0.09             4.82 &#226;&#171;&#190; 0.62                     7.66 <br />                                   26                   6.23 &#226;&#171;&#190; 0.13               1.32 &#226;&#171;&#190; 0.10             7.55 &#226;&#171;&#190; 0.17                     4.72 <br />  <br /> NADH                              14                   2.21 &#226;&#171;&#190; 0.21               0.41 &#226;&#171;&#190; 0.12             2.62 &#226;&#171;&#190; 0.32                     5.33 <br />                                   26                   2.46 &#226;&#171;&#190; 0.06               0.68 &#226;&#171;&#190; 0.04             3.14 &#226;&#171;&#190; 0.09                     3.63 <br />         Values means &#226;&#171;&#190; standard deviations. <br />  6138      ZHOU ET AL.                                                                                                      APPL. ENVIRON. MICROBIOL. <br />  <br />  <br />                                                                              &#226;?&#174;M, respectively. total NAD(H) level reached ex- <br />                                                                              tremely high value 8.49 mM initial feeding NAD&#226;&#171;&#185; <br />                                                                              concentration 1 mM (Fig. 5B), 9.6-fold                                                                              wild-type cells. <br />  <br />  <br />                                                                                                            DISCUSSION <br />                                                                                 previous study, NTT4 chlamydial P. amoebophila <br />                                                                              UWE25 expressed E. coli characterized                                                                              NAD&#226;&#171;&#185; transporter incubating resting recombinant E. <br />                                                                              coli cell 32P-labeled nucleotides (11), means                                                                              determine effect transport NAD&#226;&#171;&#185; growing cells <br />                                                                              remained unavailable. present study, confirmed <br />                                                                              function NTT4 E. coli disrupting nadC gene, <br />                                                                              responsible novo biosynthesis NAD&#226;&#171;&#185; <br />                                                                              (Fig. 2). showed NAD&#226;&#171;&#185; transportation <br />                                                                              activity NTT4 support cell growth fully, depending                                                                              uptake extracellular NAD&#226;&#171;&#185; (Fig. 3). known                                                                              disruption essential genes nadD nadE prohibitive <br />                                                                              (2, 25), successful used E. coli strain    FIG. 5. Manipulation intracellular NAD(H) level dif-          carried ntt4 gene (Fig. 3B).  strategy ferent exogenous NAD&#226;&#171;&#185; feeding levels.   Growth curve YJE003             explored alternative identification NAD&#226;&#171;&#185; <br /> cultivated M9 medium containing different concentrations               transporter utilization radioactive nucleotides. NAD&#226;&#171;&#185;. &#226;?&#174; growth rate (h&#226;&#171;&#186;1), calculated linear slopes logarithmic plots growth curves. (B) Intracellular   used identify characterize NAD&#226;&#171;&#185; <br /> NAD(H) concentrations measured 24 h cells reached                biosynthetic pathways, recently discovered stationary phase. data represent averages &#226;&#171;&#190; standard devia-          Francisella tularensis (24). <span id='am-33' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-34' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-42' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-43' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span> essential energy cofactor <br /> tions independent samples.                                determines energy metabolism metabolic flux <br />                                                                              (17). ATP transporters identified het- <br />                                                                              erologously expressed E. coli (27, 29, 30). extremes dium NAD&#226;&#171;&#185; supply, estimated total cofactor                 cellular ATP levels determined using similar <br /> level reduced 20% parent                  strategy. impacts extreme cofactor levels cells. results cofactor measurements consistent              metabolism, transcriptome, interactions cofactor expectations. total NAD(H) NADP(H) level                    enzyme (35) investigated future, 0.12 mM, 20% parent cells                    studies new insights microbial cell biology. <br /> (Fig. 4C) 10% wild-type BW25113 cells.                     level NAD tightly controlled variety mech- NAD(H) level 0.04 mM, 4.4%                      anisms, including biosynthesis salvage. Studies wild-type cells (Fig. 4C Table 4). cellular              successfully manipulated NAD&#226;&#171;&#185; level overexpressing <br /> NADP(H) level 0.08 mM, 50% 20%                       pncB changing activity NAMN adenylyltransferase parent cells wild-type cells, respectively               constructing nadD mutant library (25, 26).  (Fig. 4C). pointed NAD(H) levels                extremes NAD(H) level, e., minimal maximal wild-type cells consistent previous report (9),              level, remain undetermined live E. coli cells.  <br /> indicating data accurate.                                      structed NAD&#226;&#171;&#185;-auxotrophic mutant NAD&#226;&#171;&#185; bio- <br />    Determination maximal cofactor level live E. coli               synthesis completely blocked, designed NAD&#226;&#171;&#185; <br /> cells. NAD&#226;&#171;&#185;-auxotrophic mutant, minimal cofac-                  distribution strategy determine minimal NAD&#226;&#171;&#185; level tor determined cell division strategy.                  cell growth (Fig. 4). mutant strain YJE003, does NAD(H) level change response concentra-                   showed minimal NAD(H) level 4.4% tion exogenous NAD&#226;&#171;&#185;? maximal NAD(H) level                      wild-type cells. demonstrated 20% total live E. coli cells uphold? order answer                 cellular NAD(H) wild-type cells freely accessible questions, cultivated YJE003 cells presence NAD&#226;&#171;&#185; different concentrations (10 &#226;?&#174;M 10 mM). lag <br /> phase cells decreased increases exogenous                 TABLE 4. Intracellular NAD(H) levels NAD&#226;&#171;&#185;-auxotrophic <br /> NAD&#226;&#171;&#185; concentration, growth rate exponential                               mutant YJE003 wild-type E. coli <br /> phase remained (5.1 5.3 h&#226;&#171;&#186;1). suggested                                                                                                                   NAD(H) level (mM higher NAD&#226;&#171;&#185; transportation rate resulting higher                  Cofactor <br /> exogenous NAD&#226;&#171;&#185; concentration beneficial early                                        Maximal                  Minimal          Wild type <br /> stage cell growth advantages exponential                   NAD&#226;&#171;&#185;              7.50 &#226;&#171;&#190; 0.10           0.019 &#226;&#171;&#190; 0.002        0.64 &#226;&#171;&#190; 0.04 <br /> growth (Fig. 5A). intracellular NAD(H) levels increased                  NADH              0.99 &#226;&#171;&#190; 0.15           0.020 &#226;&#171;&#190; 0.004        0.24 &#226;&#171;&#190; 0.01 increase initial exogenous NAD&#226;&#171;&#185; concentration reached maxima 7.50 mM NAD&#226;&#171;&#185; 1.51 mM NADH                          Total             8.49 &#226;&#171;&#190; 0.24           0.039 &#226;&#171;&#190; 0.002        0.88 &#226;&#171;&#190; 0.02 initial feeding NAD&#226;&#171;&#185; concentration 1 mM 60                                                                                                       Data means &#226;&#171;&#190; standard deviations. <br />  VOL. 77, 2011                                                             EXTREMES CELLULAR NAD(H) LEVEL                                     6139 <br />  <br />  <br /> majority (80%) bound enzymes (35). fact             thank National BioResource Project (NIG, Japan) providing <br /> minimal NAD(H) level mutant strain far lower           &#226;&#338;&#172;nadC mutant JW0106 KEIO Collection Barry L. Wan- <br />                                                                        ner (Purdue University) providing E. coli BW25113(pKD3, pKD4, freely accessible NAD(H) level wild-type cells suggests         pKD46). binding affinity NAD(H) enzymes higher observed previously.  min-                                             REFERENCES <br /> imal cellular NADP(H) level 2-fold minimal              1. Adler, L. W., B. P. Rosen. 1977. Functional mosaicism membrane <br /> NAD(H) level, NADP(H) level lower                   proteins vesicles Escherichia coli. J. Bacteriol. 129:959&#226;&#8364;&#8220;966. <br />                                                                         2. Baba, T., et al. 2006. Construction Escherichia coli K-12 frame, single- <br /> NAD(H) level wild-type cells (Fig. 4C), indicating E.              gene knockout mutants: Keio collection. Mol. Syst. Biol. 2:2006.0008. <br /> coli cells need NADP(H) biosynthesis biomass                3. Begley, T. P., C. Kinsland, R.  Mehl,  Osterman, P. Dorrestein. 2001. <br /> precursors cofactor starvation state, NADP(H)                biosynthesis nicotinamide <span id='am-23' about='obo:IMR_0200350' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00147" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="Adenine" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:id" content="IMR:0200350" datatype="xsd:string"></span>adenine</span> dinucleotides bacteria. Vitam. <br />                                                                            Horm. 61:103&#226;&#8364;&#8220;119. main electron carrier anabolism (4).                             4. Blank, L. M., B. E. Ebert, K. Buehler, B. Buhler. 2010. Redox bioca- <br />    studies, cellular cofactor concentration             talysis metabolism: molecular mechanisms metabolic network anal- increased manipulating NAD&#226;&#171;&#185; biosynthetic                  ysis. Antioxid. Redox Signal. 13:349&#226;&#8364;&#8220;394. <br />                                                                         5. Brumaghim, J. L., Y. Li, E. Henle, S. Linn. 2003. Effects hydrogen <br /> pathway (12, 16) feeding permeabilized cells (18, 36, 37).              peroxide nicotinamide <span id='am-49' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-50' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-54' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-55' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span> metabolism Escherichia coli: NAD(H) level increased 6-fold previously (12).               changes enzyme levels nicotinamide nucleotide pools studies                                                                            oxidation NAD(P)H Fe(III). J. Biol. Chem. 278:42495&#226;&#8364;&#8220;42504.  maximal NAD(H) level live cells               6. Buhler, B., J. B. Park, L. M. Blank,  Schmid. 2008. NADH availability <br /> tolerate? address fundamental question, increased               limits asymmetric biocatalytic epoxidation growing recombinant Esche- intracellular NAD(H) level exposing NAD&#226;&#171;&#185;-auxo-                  richia coli strain. Appl. Environ. Microbiol. 74:1436&#226;&#8364;&#8220;1446. <br />                                                                         7. Chemler, J. ,  L. Fowler, K. P. McHugh, M.  Koffas. 2009. <br /> trophic mutant YJE003 cells higher exogenous NAD&#226;&#171;&#185;                   Improving <span id='am-28' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-29' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="NADPH" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span>NADPH</span> availability natural product biosynthesis Esche- <br /> centrations M9 medium. observed 9.6-fold                    richia coli metabolic engineering. Metab. Eng. 12:96&#226;&#8364;&#8220;104. <br /> crease NAD(H) level compared wild-type            8. Datsenko, K. , B. L. Wanner. 2000. step inactivation chromo- <br />                                                                            somal genes Escherichia coli K-12 using PCR products. Proc. Natl. Acad. <br /> BW25113 cells presence 1 mM exogenous NAD&#226;&#171;&#185;.                      Sci. U. S.  97:6640&#226;&#8364;&#8220;6645. intracellular NAD(H) level did increase            9. Dhamdhere, G., H.  Zgurskaya. 2010. Metabolic shutdown Esche- <br /> exogenous NAD&#226;&#171;&#185; concentrations 10 mM, indicating                 richia coli cells lacking outer membrane channel TolC. Mol. Microbiol. <br />                                                                            77:743&#226;&#8364;&#8220;754. maximal NAD(H) level E. coli 8.49 mM (Fig.           10. Foster, J. W., Y. K. Park, T. Penfound, T. Fenger, M. P. Spector. 1990. <br /> 5B). wild-type cells, NAD&#226;&#171;&#185; biosynthesis repressed            Regulation NAD metabolism Salmonella typhimurium: molecular se- <br />                                                                            quence analysis bifunctional nadR regulator nadA-pnuC <br /> salvage NAD&#226;&#171;&#185; degradation activated intracel-               operon. J. Bacteriol. 172:4187&#226;&#8364;&#8220;4196. <br /> lular NAD(H) level high. contrast, NAD&#226;&#171;&#185; transportation           11. Haferkamp, , et al. 2004. candidate NAD&#226;&#171;&#185; transporter intracellular mutant strain YJE003 likely independent                  bacterial symbiont related chlamydiae. Nature 432:622&#226;&#8364;&#8220;625. <br />                                                                        12. Heuser, F., K. Schroer, S. Lutz, S. Bringer-Meyer, H. Sahm. 2007. <br /> biosynthesis regulation mechanism, leading accumulation              Enhancement NAD(P)(H) pool Escherichia coli biotransforma- <br /> intracellular NAD(H) high level.                                 tion. Eng. Life Sci. 7:343&#226;&#8364;&#8220;353. <br />                                                                        13. Holm,  K., et al. 2010. Metabolic transcriptional response cofactor <br />    Cellular cofactor concentration important cell              perturbations Escherichia coli. J. Biol. Chem. 285:17498&#226;&#8364;&#8220;17506. <br /> redox biocatalysis (4, 6), determines biocat-        14. Hou, J., G. Scalcinati, M. Oldiges, G. N. Vemuri. 2010. Metabolic impact <br /> alyst operates effectively given condition. cellular        increased NADH availability Saccharomyces cerevisiae. Appl. Environ. <br />                                                                            Microbiol. 76:851&#226;&#8364;&#8220;859. <br /> NAD(H) concentration lower Km, biocatalyst          15. Hove-Jensen, B. 1996. Phosphoribosyl diphosphate synthetase-independent working optimal condition, based                   NAD&#226;&#171;&#185; novo synthesis Escherichia coli: new phenotype phosphate <br /> sumption apparent kinetics NAD(H) follows                  regulon mutants. J. Bacteriol. 178:714&#226;&#8364;&#8220;722. <br />                                                                        16. Knepper, , M. Schleicher, M. Klauke, M. Weuster-Botz. 2008.  <br /> classic rate equation V &#226;&#171;&#189; Vmax [NAD(H)]/(Km &#226;&#171;&#185; [NAD(H)]).                   hancement NAD(P)(H) pool Saccharomyces cerevisiae. Eng. Life  determination apparent kinetics NAD(H)                Sci. 8:381&#226;&#8364;&#8220;389. <br />                                                                        17. Koebmann, B. J., H. V. Westerhoff, J. L. Snoep, D. Nilsson, P. R. Jensen. necessary optimizing designing biocatalysts.             2002. glycolytic flux Escherichia coli controlled demand difficult acquire apparent kinetics data using                ATP. J. Bacteriol. 184:3909&#226;&#8364;&#8220;3916. <br /> wild-type cells, self-regulation mechanism limits          18. Kratzer, R., et al. 2011. Enzyme identification development  <br />                                                                            cell biotransformation asymmetric reduction o-chloroacetophenone. fluctuation cellular cofactor <br /> </body></html>