<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Structure Mth11&#229;&#8230;&#190;Mth Rpp29, essential protein <br /> subunit archaeal eukaryotic RNase P <br />  determined solution structure Mth11 (Mth Rpp29),              (Fig. 1). eukaryotic homologs protein contain, essential subunit RNase P enzyme archaebac-               addition conserved domain, domain N terminus <br /> terium Methanothermobacter thermoautotrophicus (Mth). RNase                  includes nucleolar targeting signal P ubiquitous ribonucleoprotein enzyme primarily responsible             functions dispensable organism lacking nucleus. cleaving 5&#226;&#180;&#8226; leader sequence maturation tRNAs            essentiality Mth Rpp29 RNase P activity confirmed domains life. eubacteria, enzyme          vitro reconstitution assay. subunits: large RNA (&#226;&#172;&#8225;120 kDa) responsible mediating                  provide insights role Mth Rpp29 RNase P <br /> catalysis, small protein cofactor (&#226;&#172;&#8225;15 kDa) modulates             assembly, examined interaction protein substrate recognition required efficient vivo catalysis.       314-nt Mth RNA subunit using NMR <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="rdfs:label" content="chemical" datatype="xsd:string"></span>chemical</span> shift pertur- contrast, multiple proteins associated eukaryotic            bation. data allowed identify residues archaeal RNase P, proteins exhibit recognizable                 protein interact nucleic acid. data homology conserved bacterial protein subunit. reconsti-            critical step developing structural model complex <br /> tution experiments recombinantly expressed purified                 RNase P holoenzyme Archaea higher organisms. <br /> protein subunits, Mth Rpp29, homolog Rpp29 protein subunit eukaryotic RNase P, essential               Materials Methods <br /> protein component archaeal holoenzyme. Consistent                Protein Expression Purification. Escherichia coli BL21(DE3) role mediating protein&#226;&#8364;&#8220;RNA interactions, report Mth           cells cotransformed Mth11&#229;&#8230;&#190;pET-15b pARG-U <br /> Rpp29 member oligonucleotide&#229;&#8230;&#190;oligosaccharide binding             (20, 21) plasmids grown 37&#194;&#176;C minimal M9 media (22) <br /> fold family. addition structured &#226;&#65533;&#164;-barrel core, possesses         containing 1 g&#229;&#8230;&#190;liter 15N ammonium chloride 2 g&#229;&#8230;&#190;liter 13C <br /> unstructured N- C-terminal extensions bearing highly             glucose nitrogen carbon sources, respectively, sup- <br /> conserved <span id='am-23' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-24' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-25' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-28' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> residues. identify possible RNA contacts             plemented 30 &#226;&#65533;&#174;g&#229;&#8230;&#190;liter kanamycin, 50 &#226;&#65533;&#174;g&#229;&#8230;&#190;liter carbenicillin, protein&#226;&#8364;&#8220;RNA complex, examined interaction               Eagle Basal <span id='am-13' about='obo:IMR_0001656' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-14' property="rdfs:label" content="vitamin" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasDbXref" content="CHEBI:27305" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:id" content="IMR:0001656" datatype="xsd:string"></span>Vitamin</span> Mix (Life Technologies, Gaithersburg, <br /> 11-kDa protein 100-kDa Mth RNA subunit using                MD). cells induced addition 1 mM isopropyl <br /> NMR chemical shift perturbation. findings represent critical           &#226;&#65533;&#164;-D-thiogalactoside cells reached OD600 &#226;&#172;&#8225;0.6 (mid-log <br /> step structural model RNase P holoenzyme                phase), harvested 4 h centrifugation (10 min, 5,000 &#226;&#171;&#187; g, <br /> archaebacteria higher organisms.                                         4&#194;&#176;C), stored &#226;&#171;&#186;20&#194;&#176;C. Thawed cells suspended 30 <br />                                                                              ml buffer (50 mM potassium phosphate, pH 7.0&#229;&#8230;&#190;0.5 M KCl) <br />                                                                              lysed ice sonication, cell debris pelleted R    ibonuclease (RNase) P essential ubiquitous  <br />      zyme primarily responsible cleaving 5&#226;&#172;&#732; leader se- <br /> quence maturation tRNAs domains life <br />                                                                              centrifugation (30 min 20,000 &#226;&#171;&#187; g, 4&#194;&#176;C). Mth Rpp29                                                                              purified reported (23) following modifications: (1, 2). bacteria, ribonucleoprotein enzyme              6-affinity tag removed incubation 37&#194;&#176;C large RNA subunit (&#226;&#172;&#8225;120 kDa) responsible mediating                     thrombin (10 units&#229;&#8230;&#190;mg Mth Rpp29) 12 h, resulting <br /> catalysis (3) small protein cofactor (&#226;&#172;&#8225;15 kDa)                    97-residue protein consisting plasmid-encoded <br /> modulates substrate recognition (4&#226;&#8364;&#8220;6) required effi-              residues (<span id='am-30' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-31' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-37' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span>Gly</span>-Ser Met) followed 93 natively encoded <br /> cient vivo catalysis (7). higher organisms, RNase P            residues. cleavage affinity tag, protein RNA protein subunits (8&#226;&#8364;&#8220;12).                           purified C4 reversed-phase HPLC chromatography (Vydac <br />    Despite significant similarity RNA subunits                214TP1010, 1 &#226;&#171;&#187; 25 cm; 180-ml linear gradient 35&#226;&#8364;&#8220;50% <br /> bacterial archaeal RNase P (13), database mining                  acetonitrile, 0.1% trifluoroacetic acid). Pooled fractions available archaeal genomes failed identify protein                  lyophilized refolded dissolving NMR buffer (50 mM <br /> homologous bacterial RNase P protein. Instead, analysis               potassium phosphate, pH 7&#229;&#8230;&#190;50 mM KCl&#229;&#8230;&#190;0.02% sodium azide&#229;&#8230;&#190; purified RNase P Methanothermobacter thermoautotro-                  10% D2O) concentration &#226;&#172;&#8225;4 mg&#229;&#8230;&#190;ml concen- <br /> phicus (Mth) identified proteins (Mth11, Mth1618, Mth687,               trated ultra filtration (Centriprep-3, Amicon). Mth Mth688) homologous eukaryotic nuclear               Rpp30 (Mth688), Mth Rpp21 (Mth1618), Pfu Pop5 (Pf1378) <br /> RNase P (14): Rpp29, Rpp21, Pop5, Rpp30, respectively.                   proteins obtained similar manner (details proteins subsequently shown                      presented . Sample purity assayed SDS&#229;&#8230;&#190;PAGE <br /> required vitro reconstitution archaeal RNase P. Using <br /> bacterially expressed, purified proteins vitro-transcribed P          work presented Keystone Symposium NMR Molecular Biology, <br /> RNA hyperthermophile, Pyrococcus horikoshii, precur-                February 4 &#226;&#8364;&#8220;10, 2003, Taos, NM. <br /> sor tRNA (ptRNA)-processing activity recovered                   paper submitted directly (Track II) PNAS office. addition proteins, activity                 Data deposition: atomic coordinates deposited Protein Data Bank, <br /> observed addition single protein subunit (15).                   www.rcsb.org (PDB <span id='am-66' about='oboInOwl:id' typeof='owl:Thing'>ID</span> code 1OQK). <br />    report solution structure Mth11&#229;&#8230;&#190;Mth Rpp29,                 Abbreviations: Mth, Methanothermobacter thermoautotrophicus; OB-fold, oligonucleo- homolog human RNase P protein Rpp29 (16, 17),                     tide&#229;&#8230;&#190;oligosaccharide-binding fold; ptRNA, precursor tRNA; HSQC, heteronuclear single <br /> encoded Mth ORF 0011 (14, 18, 19). Mth Rpp29 small                   quantum correlation; NOE, nuclear Overhauser effect. <br /> 93-residue protein containing domain (Pfam UPF0086)                   correspondence addressed. E-mail: foster.281@osu.edu. <br /> highly conserved Archaea Eukarya                 &#194;&#169; 2003 National Academy Sciences USA <br />  <br />  <br />  <br /> 15398 &#226;&#8364;&#8220;15403 &#229;&#8230;&#169; PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  26                                                     www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100 <br />  Fig. 1. Sequence alignment Mth Rpp29 Rpp29 sequences Archaea Eukarya. Alignment generated CLUSTALW (57) colored according similarity using Risler scoring matrix (58); shaded <br /> residues indicate identity, boxed residues indicate global similarity score <br /> &#226;&#172;&#381;0.7. Secondary structural features observed structure ensemble indicated. Arrows indicate core protein protected trypsin <br /> digestion identified electrospray-MS (9,114 Da expected; 9,115 Da <br /> observed). <br />  <br />  <br />  Coomassie blue staining electrospray-MS. Protein <br /> concentration estimated Bradford assay (Coomassie Plus <br /> Protein Assay Reagent, Pierce). <br />  <br /> Limited Proteolysis. Purified, refolded Mth Rpp29 subjected limited proteolysis adding 5% trypsin (wt&#229;&#8230;&#190;wt) protein <br /> solution (&#226;&#172;&#8225;100 &#226;&#65533;&#174;M NMR buffer) incubating room <br /> temperature intervals 30 min, quenched 0.6 mM <br /> PMSF, followed electrospray mass analysis (Q-TOF-II, Mi- <br /> cromass, Manchester, U.K.). <br />  <br /> Transcription Mth P RNA. gene encoding Mth P RNA [Fig. <br /> 2a, 309 nt: 1,173,861 1,174,169 gi&#226;&#177;&#65533;15678031 (16, 24)]  <br />  <br />  <br />  <br />                                                                                                                                                                        BIOCHEMISTRY <br /> obtained PCR Mth genomic DNA template appropriate gene-specific primers (forward: <br /> CGGGATCCTAATACGACTCACTATA*ggACCGGGCAAG <br /> CCGA AGGGC; reverse: GGaattcACCGGGCATGC- <br /> CGAGAGTAAC; BamHI EcoRI restriction sites ital- <br /> icized, T7 promoter underlined, asterisk denotes transcription start site, nucleotides lowercase intro- <br /> duced primers). PCR product containing gene, placed control bacteriophage T7 RNA polymerase promoter, subcloned pUC19. DNA sequencing confirmed                         Fig. 2. vitro reconstitution Mth RNase P activity.   Sequence correct sequence length P RNA gene. Mth P RNA                       phylogenetically predicted secondary structure Mth RNase P RNA <br />                                                                                     subunit (24). (b) Reconstitution experiment shows Mth Rpp29 essential generated vitro runoff transcription T7 RNA poly- <br />                                                                                     effective Mth RNase P activity E. coli ptRNATyr. negative control merase (purified house) linearization plasmid                  substrate  positive control substrate incubated E. coli <br /> EcoRI. vitro-transcribed RNA (316 nt) refolded                            RNase P; Pfu Pop5 used place Mth Pop5. <br /> binding buffer (10 mM Hepes, pH 7.5&#229;&#8230;&#190;10 mM MgOAc&#229;&#8230;&#190;400 mM <br /> NH4OAc&#229;&#8230;&#190;5% glycerol&#229;&#8230;&#190;0.01% Nonidet P-40), verifying proper fold monitoring ability RNA cleave                          Pyroccocus furiosus (Pf1378; Fig. 1b) used place Mth <br /> ptRNATyr high salt conditions (25).                                           Pop5 presumed problems solubility folding <br />                                                                                     Mth homolog. proteins purified homogeneity Vitro Reconstitution. Reconstitution experiments                        (&#226;&#172;&#381;95%) verified lack independent ptRNA processing <br /> formed incubating Mth P RNA (0.1 &#226;&#65533;&#174;M) protein                            activity. <br /> subunits (3 &#226;&#65533;&#174;M) 10 &#226;&#65533;&#174;l buffer [50 mM Tris&#228;&#161;&#160;HCl, pH 7.6&#229;&#8230;&#190;10 <br /> mM MgCl2&#229;&#8230;&#190;800 mM NH4Cl&#229;&#8230;&#190;0.025 units&#229;&#8230;&#190;&#226;&#65533;&#174;l RNasin (Invitro-                               NMR Spectroscopy. NMR spectra recorded 25&#194;&#176;C 600- <br /> gen), 37&#194;&#176;C, 5 min]. Activity assayed adding 0.02 pmol                     800-MHz Bruker (Billerica, MA) Avance DMX DRX <br /> E. coli ptRNAtyr internally labeled [&#226;&#65533;&#163;-32P]<span id='am-5' about='obo:IMR_0001354' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="rdfs:label" content="GTP" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasDbXref" content="KEGG:C00044" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasExactSynonym" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-10' property="rdfs:label" content="guanosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasAlternativeId" content="IMR:0200086" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:id" content="IMR:0001354" datatype="xsd:string"></span>GTP</span> (2 nM)                      spectrometers equipped triple-resonance probes triple- <br /> incubating 3.5 h. reactions quenched <span id='am-67' about='obo:IMR_0200111' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-69' property="rdfs:label" content="Urea" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00086" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0200111" datatype="xsd:string"></span>urea</span>,                        axis pulsed-field gradients. NMR data processing analysis <br /> electrophoresed 8% (wt&#229;&#8230;&#190;vol) polyacrylamide&#229;&#8230;&#190;7 M urea                           performed NMRPIPE (26) NMRVIEW (27). NMR <br /> gel, analyzed autoradiography. Pop5 homolog                         samples typically contained 1 mM Mth Rpp29 50 mM <br />  <br /> Boomershine et al.                                                                                    PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  26 &#229;&#8230;&#169; 15399 <br />  Table 1. Structure refinement statistics <br /> NMR constraints                                             1,247 <br />   Distance                                                  1,059 <br />     Intraresidue                                              488 <br />     Sequential   1)                                       207 <br />     Medium range   j &#226;&#177;&#8226; 4)                                    41 <br />     Long range   j &#226;&#172;&#381; 5)                                     209 <br />     Ambiguous                                                 114 <br />     (Restraints residue: 11) <br />   Torsion (&#226;&#65533;&#190;, &#226;&#65533;&#186;)                                              162 <br />   H bond                                                       26 <br /> Violations (20 SA structures; NOE <br />     violations &#226;&#172;&#381; 0.5 &#195;&#8230;) <br />   NOE violations (&#226;&#172;&#381; 0.2 &#195;&#8230;) <br />     Number                                               1.91 &#226;&#171;&#190; 0.684 <br />     RMSD, &#195;&#8230;                                            0.0187 &#226;&#171;&#190; 0.0009 <br />   Torsion violations <br />     Number &#226;&#172;&#381; 5 deg                                       0.55 &#226;&#171;&#190; 0.80 <br />     RMSD, deg                                            0.66 &#226;&#171;&#190; 0.15 <br />   Deviation ideal geometry <br />     Bond length, &#195;&#8230;                                     0.0025 &#226;&#171;&#190; 0.00028 <br />     Bond angle, deg                                    0.4121 &#226;&#171;&#190; 0.0350 <br /> RMSD mean structure (&#195;&#8230;) <br />     (residues 12&#226;&#8364;&#8220;80) <br />   Backbone                                                   0.55 <br />   Heavy atoms                                                1.26 <br /> Ramachandran analysis, % <br />   favored region                                       87.3 <br />   Additionally allowed region                               12.7 <br />  <br />   RMSD, rms deviation. <br />  <br />  <br />  <br /> potassium phosphate (pH 7), 50 mM KCl, 90% <span id='am-18' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span>&#229;&#8230;&#190;10% <br /> D2O 100% D2O. Backbone assignments using <br /> standard triple resonance spectra [HNCACB, CBCA NH, HNCO] (28, 29) recorded 600 MHz. chain 1H 13C resonance assignments obtained 3D                   Fig. 3. NMR spectra Mth Rpp29.   15N HSQC spectrum backbone <br /> spectra: 15N-separated total correlation spectroscopy (TOCSY)             amide assignments indicated. (b) Heteronuclear {1H}-15N NOE data Mth <br /> (800 MHz, 40 ms &#226;&#65533;&#182;m, 9.2 kHz nominal B1 field), CCONH-                     Rpp29 indicates N- C-terminal residues highly flexible. trypsin- <br /> TOCSY (600 MHz, &#226;&#65533;&#182;m 13 ms, 8.3 kHz nominal B1 field), 3D                   resistant core consists residues 6 &#226;&#8364;&#8220; 85. <br /> HCCH-COSY (600 MHz), HCCH-TOCSY (600 MHz, &#226;&#65533;&#182;m 12 <br /> ms, 8.3 kHz nominal B1 field). Interproton distance restraints <br />                                                                           proton chemical shifts. final set 20 refined structures obtained assignment nuclear Overhauser effect <br />                                                                           met input restraints selected analysis; statistics <br /> (NOE) cross peaks 3D 15N-separated NOESY (800 MHz, &#226;&#65533;&#182;m <br />                                                                           (Table 1) calculated PROCHECK-NMR (33). <br /> 100 ms) 13C-separated NOESY spectra (&#226;&#65533;&#182;m 150 ms). 13C-separated NOESY spectra recorded, 90% <br />                                                                           NMR Mth Rpp29&#229;&#8230;&#190;P RNA Complex. Interactions Mth <br /> H2O&#229;&#8230;&#190;10% D2O (800 MHz) 100% D2O (600 MHz). <br />                                                                           Rpp29 Mth P RNA examined using 15N-edited <br />    Heteronuclear {1H}-15N NOE data (30) obtained recording spectra interleaved manner (NOE)                 heteronuclear single quantum correlation (HSQC) spectra (800  NOE) 1H saturation achieved train 120&#194;&#176;               MHz) 1:1 ratio protein RNA. complex pulses (14 kHz bandwidth) applied 18 msec 5 sec.                assembled adding highly concentrated RNA (100 &#226;&#65533;&#174;l; 600 &#226;&#65533;&#174;M) <br /> Heteronuclear NOE values determined ratios               protein solution (200 &#226;&#65533;&#174;l; 300 &#226;&#65533;&#174;M), mixed pipetting, peak intensities spectra.                             transferred reduced volume NMR tube (Shigemi, Allison <br />                                                                           Park, PA); final protein RNA concentrations 200 &#226;&#65533;&#174;M. <br /> Mth Rpp29 Structure Determination. structure Mth               Initial spectra recorded conditions (low salt, 25&#194;&#176;C) <br /> Rpp29 protein determined using simulated annealing             excessively broadened nonspecific interactions; im- energy minimization protocols CNS program suite            proved spectra obtained addition 1 M KCl final <br /> (31), identified range structures consistent     concentration 400 mM, increasing temperature distance angle restraints derived NMR               50&#194;&#176;C. reference, spectrum free protein recorded data. Distance constraints derived intensities           buffer temperature. Backbone amide assignments <br /> cross peaks multidimensional NOE spectra (&#226;&#65533;&#182;m 100&#226;&#8364;&#8220;150                 50&#194;&#176;C obtained recording 15N HSQC spectra 5&#194;&#176; <br /> ms). Peak intensities placed categories:           intervals. Weighted average shift perturbations calculated <br /> strong (&#226;&#172;&#65533;2.8 &#195;&#8230;), medium (2.8&#226;&#8364;&#8220;3.4 &#195;&#8230;), weak (3.4&#226;&#8364;&#8220;5 &#195;&#8230;).                  (((&#226;&#65533;&#166;&#226;&#338;&#172;H)2 &#226;&#171;&#185; (&#226;&#65533;&#166;&#226;&#338;&#172;N&#229;&#8230;&#190;5)2)&#229;&#8230;&#190;2)0.5 (34). <br /> Backbone torsion angle restraints phi psi torsion angles obtained analysis backbone chemical shifts          Results <br /> TALOS (32). Hydrogen bond restraints obtained                   Mth Rpp29 Essential Subunit Mth RNase P. test analysis NOEs temperature dependence amide                  Mth Rpp29 essential RNase P activity, Mth P RNA  <br /> 15400 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100                                                                                   Boomershine et al. <br />  Fig. 4. Mth Rpp29 solution structure.   Ensemble 20 low-energy structures Mth Rpp29 superposed backbone heavy atoms residues 12&#226;&#8364;&#8220; 82; clarity, main-chain atoms (N, C&#226;&#65533;&#163;, C&#226;&#172;&#732;) structured core residues shown. Strands blue, turns coil regions gray, C-terminal <br /> helix red. (b) Close showing potential salt bridge conserved Lys-56 Glu-37 residues; hydrophobic residues core green. (c) Surface <br /> representation illustrates potential surface salt bridges. chains Lys Arg residues blue, Asp Glu red, hydrophobic residues green. Images rendered MOLMOL (59). <br />  <br />  <br />  <br /> incubated ptRNA substrate absence presence                       ing secondary structural elements, invariant Gly-32 known protein subunits (14). Using conditions similar                      allows sharp turn strands &#226;&#65533;&#164;2 &#226;&#65533;&#164;3 opposing reported P. horikoshii RNase P reconstitution (15),                    sheets. ensemble, chain invariant Lys-56 single archaeal protein able reconstitute measurable                      positioned edge hydrophobic core, appears <br /> activity (data shown).  enzyme activity                        form salt bridge chain Glu-37 (Fig. 4b); obtained preincubating RNA proteins (Fig.                      Glu Asp Rpp29 family members (Fig. <br /> 2b). essentiality Rpp29 demonstrated lack                       1). invariant residues, Arg-81 Arg-85, activity Rpp29 omitted reaction (Fig. 2b,                         located C-terminal helix preceding disordered <br /> compare lanes 5 6).                                                             C terminus. surface structured domain (Fig. 4c) <br />                                                                                     exhibits pairings basic acidic residues, indicating <br /> NMR Spectroscopy Assignments. removal  6-                      possible salt bridges, significant hydrophobic surface <br /> tag thrombin, expressed Mth Rpp29 consisted WT                          area. <br /> protein (93 residues), plus additional N-terminal residues: <br /> GSHM. 2D 1H-15N correlated NMR spectra (Fig. 3a) indicated                          Interactions RNA Subunit. low salt, Rpp29 resonance majority resonances dispersed, typical                    lines excessively broadened presence RNA folded protein;  near random coil chemical shifts                     subunit, presumably nonspecific interactions. narrow line widths resonances indicated                         addition salt (400 mM KCl 50 mM phosphate) raising <br /> regions protein flexible disordered. Limited                         temperature 50&#194;&#176;C, NMR spectra complex trypsin proteolysis heteronuclear {1H}-15N NOE data (Fig.                       recorded. Comparison 15N HSQC spectra Mth Rpp29 <br />  <br />  <br />  <br />  <br />                                                                                                                                                                        BIOCHEMISTRY <br /> 3b) demonstrated N C termini flexible                     absence presence catalytic P RNA subunit (Fig. <br /> disordered, structured protein consisting 80                        5a) reveals small shift perturbations localized line <br /> central residues (residues 6&#226;&#8364;&#8220;85, arrows Fig. 1). Backbone                    broadening, consistent weak binding. RNA-induced shift <br /> amide resonance assignments (Fig. 3a) obtained                     perturbations apparent multiple sites protein; residues 9&#226;&#8364;&#8220;13 85, addition prolines                          largest measurable shift perturbations induced (residues 4, 55, 66, 82, 93).                                                   amides Arg-25 &#226;&#65533;&#164;1-&#226;&#65533;&#164;2 loop Phe-63 Thr-65 &#226;&#65533;&#164;6 <br />  <br /> Structure Determination. solution structure Mth Rpp29 defined residues 12&#226;&#8364;&#8220;80, mean backbone rms <br /> deviation 0.6 &#195;&#8230; (Fig. 4a). proteolysis hetero- <br /> nuclear NOE data indicate residues 6&#226;&#8364;&#8220;85 structured, <br /> insufficient restraints obtained residues 6&#226;&#8364;&#8220;11 82&#226;&#8364;&#8220;85 properly define structure. core protein adopts oligonucleotide&#229;&#8230;&#190;oligosaccharide-binding fold (OB-fold) (35, <br /> 36) consists closed barrel seven antiparallel &#226;&#65533;&#164; strands short 310 helix, &#226;&#65533;&#163;1, near C terminus. Strands &#226;&#65533;&#164;1, &#226;&#65533;&#164;2, &#226;&#65533;&#164;6, &#226;&#65533;&#164;7 form stranded &#226;&#65533;&#164;-sheet, strands &#226;&#65533;&#164;3, &#226;&#65533;&#164;4 &#226;&#65533;&#164;5 form stranded &#226;&#65533;&#164;-sheet. core protein packed highly conserved hydrophobic residues, particularly positions 12, 13, 15, 17, 19, 30, 34, 42, 44, 54, 61, 63, 71, <br /> 73, 78. core appears somewhat exposed solvent,                     Fig. 5. Identification protein&#226;&#8364;&#8220;RNA contacts 1H-15N correlated NMR particularly flanked &#226;&#65533;&#164;3 &#226;&#65533;&#164;7, possible                 113-kDa Mth Rpp29 &#226;&#8364;&#8220;RNA complex.   Overlay portion HSQC partially buried N-terminal 14                      spectra free Mth Rpp29 (black) 1:1 complex 11-kDa (93 <br />                                                                                     residues) protein 102-kDa (314 nt) Mth P RNA (800 MHz, 50&#194;&#176;C, 50 mM <br /> residues. surface protein highly populated                    phosphate, 400 mM KCl). (b) Weighted average amide proton nitrogen <br /> conserved acidic (Glu, Asp) basic residues (Arg, .                          weighted-average shift perturbations (34) mapped ribbon dia- <br />    Amino acid residues highly conserved Rpp29 family                     gram Rpp29 OB-fold using linear color ramp gray  change) <br /> (Fig. 1) occupy prominent places structure.                    red [&#226;&#338;&#172;av(NH) &#226;&#171;&#189; 0.15]. Residues green effect RNA invariant <span id='am-42' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-43' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-45' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-49' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-50' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-53' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span>glycine</span> residues (Gly-14, Gly-75) form turns flank-                  binding assessed. <br />  <br />  <br /> Boomershine et al.                                                                                    PNAS &#229;&#8230;&#169; December 23, 2003 &#229;&#8230;&#169; vol. 100 &#229;&#8230;&#169;  26 &#229;&#8230;&#169; 15401 <br />  (Fig. 5b). addition, significant shift perturbations ob-        indicate 400 mM salt, interaction Mth Rpp29 served residues flexible N- C-terminal segments          RNA subunit relatively weak exhibits spectra charac- domain (data shown).                                         teristic rapid exchange free bound states.                                                                         addition, footprinting studies examining susceptibility Mth <br /> Discussion                                                              P RNA digestion RNase T1 RNase V1 absence <br /> Despite significant differences primary sequences         presence Rpp29 indicate components RNA protein subunits bacterial RNase P, overall            interact strongly absence proteins (H.-Y.T. <br /> structures remarkably conserved (1, 2, 37&#226;&#8364;&#8220;39), likely             V.G., unpublished results).  comparison reflection evolutionary pressure provided             15N-edited HSQC spectra Mth Rpp29 free complex <br />  <br /> served function enzyme. hand,          P RNA yields specific information residues RNA subunits archaebacterial RNase P similar sec-             interact P RNA. Given P RNA larger <br /> ondary structure bacterial counterparts (14, 25), se-          Mth Rpp29  10-fold) chemical shift perturba- <br /> quence analysis suggests archaeal protein subunits         tions observed ends protein, likely structurally unrelated bacterial protein, exhibiting instead     P RNA wraps Mth Rpp29 protein fits sequence homology proteins eukaryotic enzyme                large pocket RNA. increased salt concentration <br /> (14). recently (15), inability reconstitute eu-        (400 mM) used obtain spectra Mth Rpp29 complex karyotic archaeal enzymes purified components            P RNA reduce possibility observed difficult demonstrate direct functional role       spectral perturbations dominated nonspecific interactions protein subunits originally identified virtue        cationic protein anionic RNA. fact, association biochemically purified enzyme activity (8, 14,         residues exhibiting largest shift changes 40&#226;&#8364;&#8220;44), leaving open possibility undiscovered        cationic arginines lysines (Fig. 5), include hydrophobic <br /> bacterial-like protein required activity. Furthermore,      (e.g., Val-17, Ile-27, Phe-63, Ile-73) polar (e.g., Thr-53, <br /> poor sequence homology identified subunits               Thr-65) residues.  recent report demonstrating sig- <br /> archaeal RNase P proteins known structure furnished           nificant role hydrophobic residues nucleic acid binding clues roles RNase P-mediated ptRNA processing.           telomere-end binding protein, Cdc13, member impasse overcome demonstration            OB-fold protein family (36), indicates observations functional archaeal RNase P holoenzyme reconstituted           unique  vitro-transcribed P RNA recombinantly ex-                  speculate role Rpp29 stabilize active <br /> pressed purified Rpp29, Rpp21, Rpp30, Pop5 proteins             conformation RNA subunit forming favorable hydro- P. horikoshii, hyperthermophile (15), Mth, moderate          phobic electrostatic interactions.  single <br /> thermophile (Fig. 2b).  functional roles          protein complete bacterial enzyme, single individual protein subunits remain elucidated,        archaeal RNase P protein sufficiently stabilize RNA <br /> studies demonstrate Rpp29 essential enzyme activity         structure promote activity (Fig. 2 ref. 15). Instead, reconstitution assay. study Mth Rpp29 provides         appears protein subunits exhibit cooperativity glimpse structure&#226;&#8364;&#8220;function relationships archaeal&#229;&#8230;&#190;            ability promote catalysis (H.-Y.T. V.G., unpublished <br /> eukaryotic RNase P.                                                     work).  caveat native <br />    investigation revealed Rpp29 member        RNA&#226;&#8364;&#8220;protein interactions subunits OB-fold family proteins (35, 36). OB-fold contain-          achieved protein components ing proteins difficult identify primary se-           holoenzyme present, NMR chemical shift perturbation <br /> quence, represented ribonucleoprotein com-             data presented provide insights possible protein&#226;&#8364;&#8220; <br /> plexes domains life. Examples include S1          nucleic acid contacts holoenzyme. Furthermore, based S17 bacterial ribosomal proteins (45), anticodon-               data, suggest protein interacts RNA recognition domain aspartyl lysyl tRNA synthetases,              conserved OB-fold domain highly conserved C <br /> single-stranded DNA binding proteins Thermus thermophi-            terminus  potentially N terminus) Mth Rpp29  <br /> lus Thermus aquaticus (46), translation initiation factor           comes structured binding Mth P RNA (and&#229;&#8230;&#190;or aIF-1A Methanococcus jannaschii (47), single-stranded          protein subunits). <br /> telomeric DNA binding protein Saccharomyces cerevisiae,               proposal Rpp29 functions direct interaction Cdc13 (48), N-terminal segment human translation             RNA subunit supported results earlier <br /> initiation factor &#226;&#65533;&#163; (49). surprisingly, Mth Rpp29 exhibits       studies. instance, hybrid assays yeast revealed significant structural similarity protein subunit bacterial   human yeast Rpp29 interact respective <br /> RNase P, adopts &#226;&#65533;&#163;-&#226;&#65533;&#164; sandwich fold (37&#226;&#8364;&#8220;39).                     RNase P RNA subunit (54&#226;&#8364;&#8220;56). Additionally, findings    surface structured core Rpp29 (Fig. 4c) exhibits       genetic study RNase MRP, endonuclease related unique array cationic, anionic, hydrophobic patches       RNase P, support idea Rpp29 involved mediate specific protein&#226;&#8364;&#8220;protein protein&#226;&#8364;&#8220;RNA rec-             binding cognate RNA subunit (41). eukaryotes, RNase <br /> ognition. addition structured core containing             MRP series enzymes involved rRNA processing. <br /> OB-fold, unstructured N- C-terminal extensions bear             RNase MRP RNase P, acting different sub- highly conserved arginine lysine residues (Fig. 1)     strates, related ribonucleoprotein provide additional stabilizing interactions RNA.             complexes share protein subunit similar  likely hydrophobic patches salt           RNA subunits. study S. cerevisiae RNase MRP (41), bridges surface protein, unstructured      class extragenic suppressors temperature-sensitive mu- <br /> N C termini critical roles induced-fit recognition      tation rrp2&#226;&#8364;&#8220;2 RNA subunit RNase MRP mapped RNA subunits (50, 51). expectation          alterations single codon POP4&#229;&#8230;&#190;RPP29 gene. Based precedent: highly charged unstructured N C           observation, proposed rpp2&#226;&#8364;&#8220;2 mutation termini globular ribosomal proteins, L15, L21e, L37e         MRP RNA disrupt RNA&#226;&#8364;&#8220;protein interactions (52) penetrate ribosome make important structural               Pop4 (Rpp29), reasonable premise far contacts ribosomal RNA (52, 53).                               untested. <br />    nature interaction Mth Rpp29             mechanistic structural details Mth Rpp29 <br /> RNA subunit? NMR spectra RNP complex (Fig. 5a)                   interacts catalytic P RNA protein subunits <br />  <br /> 15402 &#229;&#8230;&#169; www.pnas.org&#229;&#8230;&#190;cgi&#229;&#8230;&#190;doi&#229;&#8230;&#190;10.1073&#229;&#8230;&#190;pnas.2535887100                                                                      Boomershine et al. <br />  remain uncovered. studies help reveal                            thank C. Amero, C. Cottrell, S. Kraynik, W. Woznick, C. Yuan <br /> function Mth Rpp29 holoenzyme allow com-                              (Ohio State University) assistance technical support; J. W. <br /> parisons functions protein subunits                               Brown (North Carolina State University, Raleigh) helpful discussions <br /> bacterial archaeal&#229;&#8230;&#190;eukaryotic RNase P. Given stark                              Rpp21 expression plasmid;  Edwards S. Arrowsmith <br /> difference structure Rpp29 bacterial                             (University Toronto, Toronto) Rpp29, Rpp30, Pop5 <br /> protein subunit, unclear subunits archaeal                        expression constructs, M. L. S. Raj (Ohio State University) Mth <br /> RNase P required supplant function single                           P RNA clone; J. Reeve (Ohio State University) Mth genomic DNA, <br /> bacterial counterpart. comprehensive understanding                           M. Adams (University Georgia, Athens) Pfu genomic DNA; mechanism archaeal RNase P necessitate detailed                          Ohio State University Campus Chemical Instrument Center mass <br /> studies subunits complexes, structural charac-                         spectrometric analyses; Ohio State University Plant-Microbe <br /> terization Mth Rpp29 interaction catalytic P                        Genomics Facility Plant Biotechnology Center DNA sequencing. <br />                                                                                         W.P.B. supported National Institutes Health Chemistry&#226;&#8364;&#8220; <br /> RNA represent important steps elucidating structure&#226;&#8364;&#8220; <br />                                                                                         Biology Interface training grant, H.-Y.T. supported function correlates holoenzyme. <br />                                                                                         American Heart Association predoctoral fellowship. work Note Added Proof. solution structure Rpp29 homolog                   supported Ohio State University National Science <br /> Archaeoglobus fulgidus recently reported (60)                     Foundation Grants MCB-0092962  M.P.F.) MCB-0091081  adopt architecture Mth homolog.                                         V.G.). <br />  <br />  <br />   <br /> </body></html>