<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>Grassi Tramontano BMC Evolutionary Biology 2011, 11:301 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br />  RESEARCH ARTICLE                                                                                                                              Open Access <br />  <br /> Horizontal vertical growth S. cerevisiae <br /> metabolic network <br /> Luigi Grassi1 Anna Tramontano1,2* <br />  <br />  <br />   Abstract <br />   Background: growth development biological organism reflected metabolic network,   evolution relies essential gene duplication mechanism. current views   evolution metabolic networks. retrograde model hypothesizes pathway evolves recruiting novel <br />   enzymes direction opposite metabolic flow. patchwork model instead based assumption <br />   evolution based exploitation broad-specificity enzymes capable catalysing variety   metabolic reactions. <br />   Results: analysed studied unicellular eukaryotic organism, S. cerevisiae, studied effect   removal paralogous gene products metabolic network. results, obtained using different paralog   network definitions,  initial period gene duplication instrumental expanding <br />   metabolic space, reached equilibrium subsequent gene duplications used source   specialized enzymes source novel reactions. switch   evolutionary strategies S. cerevisiae dated 350 million years ago. <br />   Conclusions: data, obtained novel analysis methodology, strongly supports hypothesis   patchwork model better explains recent evolution S. cerevisiae metabolic network. Interestingly, <br />   effects patchwork strategy acting Euascomycete-Hemiascomycete divergence   detectable today. <br />  <br />  <br /> Background                                                                           networks assign metabolites nodes pro- <br /> Metabolism defines reactions provide energy                                 cessing enzymes edges. constituents cells organisms. includes                              metabolic networks  doubt, features related growth development                               studied biological networks, models living organism. modular units metabo-                               appropriate explaining evolutionary history lism metabolic pathways, sets <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>chemical</span> reac-                              matter debate. main models tions metabolite transformed                              presented literature evolution: retro- series steps catalysed enzymes. evo-                             grade model patchwork model  recent <br /> lutionary history enzymes necessarily reflects evo-                           review [1]). retrograde model, pathways <br /> lution organism belong                                  evolve backwards key metabolite [2]. According <br /> constitutes interesting subject study.                                         model, enzymes recruited direction <br />    metabolic networks provide global view                               opposite metabolic flow environment <br /> metabolic pathways organism repre-                                  assumed rich metabolites initial <br /> sented alternative ways. enzyme-centric net-                               key metabolites intermediates. abundance works, proteins defined nodes, connected                               initial intermediate metabolites difficult recon- <br /> metabolites process, whilst metabolite-centric                                  cile network evolution occur                                                                                      organic molecules depleted environment. <br /> * Correspondence: anna.tramontano@uniroma1                                                                                      patchwork model formulated aim 1 <br />  Physics Department, Sapienza University Rome, Piazzale Aldo Moro, 5            overcoming conceptual limitation [3,4]. alter- <br /> 00185 Roma, Italy                                                                    native model hypothesizes important role broad- list author information available end article <br />  <br />                                        &#194;&#169; 2011 Grassi Tramontano; licensee BioMed Central  Open Access article distributed terms Creative <br />                                        Commons Attribution License (http://creativecommons.org/licenses 2.0), permits unrestricted use, distribution,                                        reproduction medium, provided original work properly cited. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                              Page 2 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> specificity enzymes (capable catalysing variety       Results <br /> metabolic reactions) forging evolution meta-        Evolutionary perspective <br /> bolic pathways. enzymes, thanks broad          study propose alternative perspective substrate specificities, involved meta-      study evolutionary history metabolic net- <br /> bolic pathways synthesis similar       works. primary focused novelty metabolites. context, gene duplications         enzymes progressively added network. bring selective advantage pathways: increased     metabolic space defined set level enzyme cause increase      metabolites present metabolic network. addi- <br /> key metabolites. Finally, specialization differ-   tion new enzyme metabolic network ent pathways occur result specialization     essentially consequences:  creation paralogous enzymes.                                           new functionality  e. new metabolic reaction); ii)   Previous analyses showed paralogous enzymes            duplication functionality present tend closer metabolic network expected         network. chance [5,6]. result interpreted        innovation brought enzymes progressively confirmation retrograde model, studies      added metabolic network related evolu- <br /> [7,8] showed homologous enzymes dif-        tionary debate neutralism selectionism, ele- <br /> ferent pathways pathway,          gantly reconciled analysis Wagner [15,16]. accordance patchwork model. current             addition new enzyme processes new <br /> way reconcile different findings postulate     metabolites lead expansion metabolic <br /> complementarity evolutionary models,         space, reflected horizontal growth meta- commonly accepted non mutually exclusive            bolic network (Figure 1A). Alternatively vertical <br /> [9].                                                          growth metabolic network realised time <br />   study propose new approach study       enzyme able process metabolites present evolution metabolic networks based           metabolic space added network (Figure <br /> evaluation effects gene duplication given network. test case, selected S. cerevisiae <br /> metabolic network. <br />   S. cerevisiae studied eukaryotic mono-cellular <br /> organism information metabolic <br /> reactions accumulated reported thousands papers included publicly available data- <br /> bases. studies confirmed S. cerevisiae experienced genome duplication (WGD) 100 million years ago [10-12].  results careful study [13] traced evolutionary <br /> history protein-coding genes seventeen genomes Ascomycota fungi identified orthologs para- <br /> logs different periods duplication. <br />   results analysis reported S. cerevisiae metabolic network evolved discon- <br /> tinuous manner: growing moves increased number metabolites. Subsequently, newly added <br /> enzymes did increase number metabolites, specialized specific cellular compartments cell cycle phase  alternatively, included existing complexes. datation duplications        Figure 1 Network growth models. duplication proteins indicates shift network evolution            involved metabolic reactions results growth metabolic <br /> occurred Euascomycete-Hemiascomycete                network. Paralogous pairs process different metabolites increase <br />                                                                number nodes lead horizontal expansion divergence, estimated happened 350 mil-         network, depicted  Paralogs catalysing lion years ago [14]. discontinuity growing         reactions, example different conditions, result modality, results discussed           specialized expression proteins lead vertical <br /> strongly supports hypothesis patchwork            growth metabolic network number nodes <br /> model played main role forging evolution       remaining constant  B). colour scale goes red                                                                 recent duplications) blue  recent duplications). <br /> S. cerevisiae metabolic network. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                              Page 3 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> 1B). studied effects progressive addition       <span id='am-12' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> paralogs led removal 152 178 paralogous enzymes metabolic network             edges KEGG YBP networks, respectively. <br /> evaluating effects growth metabolic          consistently obtained result repli- <br /> space.                                                        cates: collapsed networks includes large majority <br />                                                               metabolites empirical network. Network analysis                                              seen inset Figure 2, reports percen- order evaluate contribution paralogous           tage metabolites YBP network collapsed <br /> enzymes metabolic network S. cerevisiae          according alignment paralog definition collapsed network maintaining member          respect empirical  99% 100% of paralogous family (collapsed network).        metabolites preserved  Additional file 1, Fig- <br /> compared properties metabolic space         ure S1 data obtained using different metabolic net- <br /> original network 1000 collapsed networks        work paralogy definitions). <br />  e. networks paralogous enzymes                effect removal node network removed randomly selected, Methods).             strictly dependent connectivity degree, e. analysis performed replicates starting        number links nodes. independent metabolic network databases            Figure 2 shows metabolites empirical <br /> paralogous family definitions. metabolite-centric         1000 collapsed networks equivalent connectiv- <br /> networks reconstructed linking products           ity distributions. result independent choice <br /> reagents processing enzymes  Methods).        paralogous families reference meta- initial networks Yeast Biochemical           bolic network  Additional file 1, Figure S2) indi- <br /> Pathways (YBP), manually curated corrected ver-         cates paralogous proteins similar connectivity. <br /> sion Metacyc database [17,18] available Saccharomyces Genome Database (SGD) [19,20]             Functional overlap paralogous proteins <br /> network reconstructed according reactions pre-         metabolic networks examination, enzymes <br /> sent KEGG database [21,22]. paralogous             represented edges. order define func- <br /> families defined according strategies.      tional properties enzyme listed pairs based comparison protein pairs.       metabolites (nodes) connects. reaction  pairs proteins aligned using EMBOSS           sidered  regardless reversibility, e.g. needle program [23] 30%              reaction &#226;&#8225;&#338; B listed AB, BA  sequence identity connected paralogy net-         Metabolic Network Reconstruction section Methods). <br /> work  Methods). connected components          compared paralogous pairs present network defined protein families  consequently, <br /> members family identified paralo- <br />                                                                            0 <br /> gous proteins. analysis repeated using               10 <br /> paralogous families retrieved ENSEMBL compara                                                          Empirical network <br />                                                                                                             Collapsed networks <br /> [24] study Wapinski et al. [13].                           -1 <br />                                                                       10 <br />    test, compared metabolic space empirical network collapsed networks <br />                                                                P(k) <br />  <br />  <br />  <br />  <br /> obtained removal paralogous proteins. aim                  -2 <br />                                                                       10 comparison understand gene                                       100 <br />  <br />  <br />  <br /> duplication contributes vertical horizontal <br />                                                                                 Metabolites (%) <br />  <br />  <br />  <br />  <br />                                                                            -3 <br /> growth metabolic network. Clearly, metabolic               10 <br /> space collapsed network smaller empirical paralogous enzymes                   -4 <br />                                                                                               99 <br />  <br />  <br />                                                                       10        0                       1                          2 add new metabolites network.                       10                      10                            10 case, example, paralogous enzymes                                                 k responsible catalysing            Figure 2 Effects paralog removal metabolic network. <br /> reaction different compartments different             main figure reports connectivity distributions P(k)                                                                metabolites empirical network (red squares) 1000 <br /> conditions.                                                    collapsed networks (blue dots). distributions    Collapsing paralogs according global sequence        similar. inset boxplot reporting fraction metabolites <br /> alignment led removal 311 198 edges          empirical network present 1000 collapsed networks KEGG YBP networks, respectively.              shown. Dashed lines indicate mean, upper lower box <br /> ENSEMBL definition paralogs used, corre-            margins correspond standard deviation (SD), whiskers <br />                                                                indicate SD. <br /> sponding figures 265 113 edges. Wapinski <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                             Page 4 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> metabolic network order quantify functional                                 similar <span id='am-18' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-19' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span>amino acid</span> sequences (72% identity 84% <br /> overlap calculating relative Dice coefficient [25].                              similarity). catalyze conversion orotate <br /> Given paralogous enzymes E1 E2, Dice M                                      orotidine-5&#226;&#8364;&#8482;-monophosphate. URA5 accounts coefficient defined twice number overlapping                               80% orotate phosphoribosyltransferase <br /> metabolite pairs (intersection) divided total num-                               activity yeast cells (major isozyme) URA10 <br /> ber metabolite pairs (union): Dice M = Dice = 2                                      responsible remaining 20% (minor isozyme) <br /> (X1&#226;&#710;&#169;X2)/(X1&#226;&#710;&#170;X2) X1 X2 metabolite                                      [26,27]. Interestingly, expression patterns <br /> pairs E1 E2. coefficient equal zero                               mildly anti-correlated accordance enzymes functional overlap, whilst equal                               results Zhang et al., according URA5 totally functionally overlapping enzymes.                                       transcription regulated dimethyl sulfoxide <br />   Figure 3A shows normalized distributions                                   URA10 expression regulated Dice M coefficient paralogous pairs (defined                                    conditions [28]. WGD paralogs YDL131W paralogy alignment network) enzymes                                   (LYS21) YDL182W (LYS20) constitute case YBP network million randomly                                         isozymes totally overlapping function. assorted pairs. paralogous pairs (~80%)                                 identical amino acid sequences (87% identity enzymes performing exactly reaction                                    90% similarity) catalyze condensation acetyl- remaining fraction formed equal                                  <span id='am-13' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>CoA</span> alpha-ketoglutarate form homocitrate number paralogous enzymes functional                                    lysine biosynthesis pathway. recent study [29] showed <br /> lap (~10%) paralogous enzymes partial                                     condition growth ethanol, homocitrate functional overlap (~10%). result, confirmed                                mainly synthesized LYS21, fermen- <br /> replicates  Additional file 1, Figure S3), indicates                                tative metabolism, LYS21 LYS20, play redundant enzymes identified paralogs                                    roles. contributed vertical growth S. cerevi- <br /> siae metabolic network, whilst small fraction                                Paralogous proteins catalyzing different reactions <br /> tributed horizontal growth.                                                      vast majority paralogous pairs   Additional file 2 reports paralogous pairs ana-                               isozymes totally overlapping functions lyzed study. pair calculated                               small fraction (~10%) paralogs catalyzing different <br /> Pearson correlation coefficient expression                                reactions. interesting minority constitutes levels using data SGD&#226;&#8364;&#8482;s Expression                                     horizontal growing moves S. cerevisiae metabolic <br /> nection tool (http://www.yeastgenome.org/cgi-bin/                                       network,  <br /> expression/expressionConnection.pl).                                                       Table 1 reports pairs enzymes identified   interesting studied example paralogous                                 paralogs paralogy assignments enzymes total functional overlap reported fig-                               functional overlap metabolic network. ure 3B. isozymes YMR271C (URA 10)                                               examination pairs immediately highlights <br /> YML106W (URA5) WGD paralogs [11]                                          enzymatic similarity proteins.  <br />  <br />                          10 <br />                                                       Randomly selected pairs <br />                                                       Paralogous pairs <br />  Normalized occurrence <br />  <br />  <br />  <br />  <br />                           8 <br />  <br />  <br />                           6 <br />  <br />  <br />                          4 <br />  <br />  <br />                           2 <br />  <br />  <br />                           0 <br />                               0   0.2   0.4           0.6        0.8            1 <br />                                               DiceM <br />                                                                                   B <br />  Figure 3 Functional overlap paralogous pairs.   DiceM coefficient paralogous pairs (red bars). Grey bars refer 106 randomly <br />  selected pairs. (B) Scheme pathway section including isozymes YMR271C (URA10) e YML106W (URA5) sharing 75% amino acid <br />  similarity responsible reaction, conversion orotate orotidine-5&#226;&#8364;&#8482;-phosphate. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                                       Page 5 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> Table 1 Paralogous pairs divergent functions <br /> Paralogous pairs                    Paralogy definition                          DiceM coefficient               Correlation expression levels <br /> Enzyme 1          Enzyme 2          Need          Ens            Wap            YBP            KEGG              Pearson coeff.                 P value <br /> YBR149W           YHR104W           Y             Y                N             0              N                    0.68                      2.2-16 <br /> YDL080C           YDR380W           Y             Y                Y            0.67             0                     0.08                      0.033 <br /> YDL246C           YLR070C           Y             Y                Y            N              0                      N                       N YDR147W           YLR133W           Y             Y                N             0              0.5                    0.09                       0.02 <br /> YDR380W           YGR087C           Y             Y                Y            0.54             0                     -0.07                     0.065 <br /> YDR380W           YLR044C           Y             Y                Y            0.54             0                     0.08                      0.047 <br /> YDR380W           YLR134W           Y             Y                Y            0.54             0                     0.01                       0.76 <br /> YDR399W           YJR133W           Y             Y                Y             0              N                    0.25                      7.6-11 <br /> YHR123W           YNL130C           Y             Y                Y             0               0                     0.50                      2.2-16 <br /> YJR159W           YLR070C           Y             Y                Y            N              0                     0.38                      2.2-16 pairs identified paralogs paralogy definitions (Need: sequence identity 30%; Ens: derived Ensembl database; Wap: <br /> listed Wapinski et al. [13] ) DiceM coefficient equal 1 metabolic network shown. complete set data shown Additional file 2. <br />  <br />  <br /> pair enzymes having exactly EC                               PDC1, PDC5 PDC6 exclusively mapped number, pairs partial overlap                         &#226;&#8364;&#339;valine degradation&#226;&#8364;?, &#226;&#8364;&#339;acetoin biosynthesis II&#226;&#8364;?, &#226;&#8364;&#339;buta- enzymes EC number                               nediol biosynthesis&#226;&#8364;? &#226;&#8364;&#339;glucose fermentation&#226;&#8364;? <br /> digit) pairs EC numbers differing                         pathways. digit. indicates paralogous                          studies showed ARO10 pairs, defined divergent network ana-                         PDC1, PDC5 PDC6 isozymes share substrates <br /> lysis, similar enzymatic point view,                         process specific substrates [31,32]. difference level substrates                         investigated growth metabolic process. YBR149W (ARA1) arabinose dehy-                               network occurs continuous fashion. words, <br /> drogenase catalyzes oxidation D-arabinose,                           asked different modalities meta- <br /> L-xylose, <span id='am-25' about='obo:IMR_0200303' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-26' property="oboInOwl:id" content="IMR:0200303" datatype="xsd:string"></span><span id='am-27' property="rdfs:label" content="L-Fucose" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C01019" datatype="xsd:string"></span>L-fucose</span> L-galactose presence                            bolic network growth happened different evolutionary <br /> NADP+; YHR104W (GRE3) paralog aldose                                   periods. answer question investigated reductase reduces cytotoxic compound methyl-                            duplication age paralogous enzymes. <br /> glyoxal (MG) (R)-lactaldehyde, purified                            Following reported Wapinski et. al [13], <br /> GRE3 catalyze reduction xylose xyli-                        divided paralogous pairs groups:  pre- <br /> tol. YBP proteins mapped different                           WGD Euascomycetes paralogous pairs, ancient <br /> pathways: ARA1 dehydro-D-arabinono-1,4-lac-                               duplications shared species Euascomycetes <br /> tone biosynthesis GRE3 xylose metabolism.                             class; ii) pre-WGD Hemiascomycetes paralogous pairs, <br /> YBR149W present KEGG reaction database                             including duplications WGD excluding DiceM coefficient relative KEGG                         Hemiascomycetes class; iii) WGD paralogous pairs; <br /> network computed. YLR044C (PDC1),                                  iv) post-WGD paralogous pairs. <br /> YGR087C (PDC6) YLR134W (PDC5) isozymes                                     pairs DiceM coefficient equal pyruvate decarboxylase activity PDC1 acting                            zero (obtained using YBP network definition) belong major decarboxylase [30]. paralogs                        pre-WGD Euascomycetes group (hypergeometric <br /> YDR380W (ARO10), phenylpyruvate decarboxylase                                  test P value = 0.0007). confirmation given catalyzes decarboxylation phenylpyruvate                          analysis network derived KEGG, phenylacetaldehyde [31]. KEGG, PDC1, PDC5                                 pairs Dice M coefficient equal PDC6 mapped &#226;&#8364;&#339;Glycolysis/Gluconeogen-                              zero belong pre-WGD Euascomycetes group <br /> esis&#226;&#8364;? pathway, whilst paralog functional                           (hypergeometric test P value = 0.0005). <br /> overlap ARO10 &#226;&#8364;&#339;Phenylalanine metabolism&#226;&#8364;?                                 result indicates paralogous pairs <br /> pathway. YBP enzymes partial functional                            functional overlap derive ancient <br /> overlap, DiceM coefficient equal 0.53.                        duplications, dated Wapinski et al. [13] proteins perform reaction &#226;&#8364;&#339;isoleucine                            Euascomycete-Hemiascomycete divergence, estimated degradation&#226;&#8364;?, &#226;&#8364;&#339;phenylalanine degradation&#226;&#8364;? &#226;&#8364;&#339;trypto-                           occurred 350 million years ago [14]. <br /> phan degradation&#226;&#8364;? pathways, ARO10                              Paralogous enzymes functional overlap gener- <br /> mapped &#226;&#8364;&#339;leucine degradation&#226;&#8364;? pathway                                  ally belong different pathways opposite  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                    Page 6 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> true enzymes catalysing reaction.        expected chance  time display enzyme pairs shown Table 1, calculated            strong chemical similarity (essentially catalyse simi- <br /> DiceP coefficient  Methods). pairs       lar reactions).  results used DiceP coefficient different 1 indicating     discriminate retrograde patchwork <br /> tend exclusively belong single pathway        model. scenario, neighbourhood dupli- <br /> information available Additional file 3).              cated enzymes interpreted proof step- <br />    confirmation finding derives      wise growth metabolic network consequent calculation mean shortest paths        stepwise addition enzymes. hand metabolites (reagents products) connected          enzymatic similarity paralogous enzymes pairs enzymes shown Table 1  Additional        interpreted supporting patchwork model, accord- <br /> file 3). According measure, hypothetical pair    ing broad specific enzymes tend catalyse <br /> enzymes connecting contiguous metabolites  e.           chemically similar reactions acting differ- <br /> products reaction catalysed enzyme          ent types substrates [43]. studies mapped reagents reaction catalysed       paralogous enzymes enzyme-centric metabolic net- mean shortest path equal close 1. mean            work, problem short distance <br /> shortest path network derived YBP 7.7 &#194;&#177;       totally overlapping enzymes dis- <br /> 3.3 6.8 &#194;&#177; 1.8 KEGG network. indicates        tinguished signal resulting contiguous horizontal growing moves tend add             enzymes. reason, selected approach <br /> enzymes distant points S. cerevisiae metabolic      problem using metabolite-centric reconstruction <br /> network.                                                      metabolic network, informative. <br />                                                               example, let consider totally overlapping <br /> Discussion                                                    paralogs E1 E2, processing reaction Gene duplication generally accepted impor-       metabolite metabolite B let assume tant source raw material functional innovation         B processed different <br /> [33-36]. S. cerevisiae, studies [37-40] pointed       metabolite C. enzyme-centric network key role gene duplication evolution      enzymes distance equal 2, regulatory networks, particularly loss,       approach, enzymes E1 E2 DiceM coef- <br /> gain rewiring regulatory interactions occurring        ficient equal 1 considered totally duplication. study aims clarifying       overlapping paralogs. <br /> effective contribution gene duplication evolu-        results strongly support conclusion tion metabolic networks. theme treated         vast majority paralogous pairs nowadays detectable authors attempt identifying          S. cerevisiae similar E1 E2 hypothetical <br /> best model able explain evolution metabolic         enzymes contribute horizontal expan- <br /> networks. study Tsoka Ouzounis [41] reports          sion metabolic network. enzymes family distributed               conclusion robust independent different pathways. Similarly, Teichmann et al.         definition metabolic network method <br /> [8] homologous enzymes twice likely         used paralogy assignment. different pathways path-        fact, metabolic space remains unchanged <br /> way. pioneering studies interestingly converge       removal paralogous enzymes  <br /> assigning high likelihood patchwork model,           nected metabolites, consistently previous studies limitations: analyse E.      demonstrating strong conservation biochemical <br /> coli, prokaryotic organism widespread          function S. cerevisiae duplicated genes [13,44,45]. <br /> phenomenon lateral gene transfer, particularly affect-       Furthermore, Gash et al. demonstrated S. cerevi- <br /> ing proteins involved metabolism [42], compro-         siae isozymes different transcriptional regulation <br /> mise reliability homology inference.               specific members isozyme families responding Furthermore studies analyse metabolism        environmental stress [46] study Ihmels et al. network viewpoint misleading           [47] elegantly showed isozyme multiplicity metabolic reactions cells, mapped     strategy adopted S. cerevisiae permit differential <br /> different pathways, close network. Subse-       regulation reactions shared different processes. <br /> quent studies analysed metabolism network          strategy provides possibility indepen- <br /> perspective. Alves et al. [5] D&#195;&#173;az-Mej&#195;&#173;a et al. [6]       dent control associated reactions response analysed metabolic networks 12 species E.       pathway-specific requirements. <br /> coli, respectively, reaching similar conclusions: homolo-       paralogous enzymes isozymes <br /> gous enzymes tend closer network            completely overlapping functions, small <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                    Page 7 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> fraction (~10% pairs) paralogs catalysing totally       space large gene duplications dated <br /> non-overlapping reactions. characterization      Euascomycete-Hemiascomycete divergence. <br /> divergent paralogs useful uncover evolutionary      subsequent duplications gave raise process definition metabolic network        enzymes performing reaction, specialized S. cerevisiae. Interestingly,       different cellular localizations differently regu- <br /> derived ancient duplications, dated              lated. result suggests patchwork model Wapinski et al. [13] Euascomycete-Hemiasco-        compatible retrograde model mycete divergence  350 million years ago).         growth modality metabolic network S. cerevi- <br /> result indicates discontinuity growth modalities     siae. confirmation given S. cerevisiae metabolic network, meta-        finding paralogs partial func- <br /> bolic space remaining unaltered unaltered       tional overlap far away metabolic network. <br /> 350 million years.                                            conclusion, described method   Interestingly, growth strategy accordance        useful studying contribution paralogous theoretical prediction appearance        enzymes metabolism S. cerevisiae enzymes occurs bursts phyletic gradu-       applied evolutionary study metabolism alism proposed Sch&#195;&#188;tte et al. [48].                        organisms. <br />   result reaching network equili- <br /> brium, growing moves add          Methods <br /> existing enzymatic reactions creating novel       Metabolic Networks Reconstruction <br /> functionalities. differentiation broad-specificity     analyses performed study relied enzymes hypothesized patchwork model             metabolic networks: YBP network KEGG event contributed reach            network. metabolite-centric networks, recon- <br /> equilibrium, enzymes added net-        structed connecting substrate products work specialized performing specific      processing enzymes. networks represented <br /> reaction.                                                     directed graphs nodes denote compounds   expect  retrograde      edges denote proteins. directionality edges <br /> model, equilibrium state reached,         reflects direction reaction. Reactions catalyzed example, possible biochemical reactions      protein complexes represented including explored, second scenario,         reaction member complex. Highly <br /> expect signs retrograde model.         connected hubs source noise [49], horizontal growing moves result dupli-          restricted analysis main reactants cated enzymes close metabolic network,       networks, excluding currency metabolites case: paralogs partial    analysis. functional overlap mapped different path-           YBP network derived SGD [19,20] (http:// <br /> ways mean distance metabolites pro-       downloads.yeastgenome.org/curation/literature/archive/ <br /> cess short expect according     yeastcyc14.0.tar.201009.gz). <br /> retrograde model.                                                referred SGD Pathway Tools version 14.0.   results suggest methodology used          file pathways.dat contains details reac- powerful straightforward way studying       tion present database (enzymes, substrates, pro- effect paralogous enzymes metabolism         ducts, directionality). used files organism unravel evolutionary strategy.            protcplxs.col enzrxns.dat. detailed list <br />                                                               protein complexes components, <br /> Conclusions                                                   whilst describes enzyme features (gene purpose analysis evaluate contri-        modality action dimer/monomer bution paralogous enzymes growth         catalyzed reaction). reactants classified &#226;&#8364;&#339;pri- <br /> metabolic network. order used sim-         maries&#226;&#8364;? taken account. <br /> ple strategy: comparison empirical metabolic          KEGG network derived KEGG <br /> network collapsed networks, obtained         PATHWAY Database [21,22]. reaction file (version <br /> random removal paralogous enzymes.                    04/03/2011) contains information reac- <br /> approach, applied specific case S. cerevisiae,      tions (substrate/product reversibility). gave interesting results. obtain results          RPAIRS (reactant pairs) classified KEGG &#226;&#8364;&#339;main&#226;&#8364;? <br /> regardless paralog network definitions used:       pairs taken account. enzyme-reaction metabolic network S. cerevisiae grew discontinu-       associations retrieved file sce01100.xml (ver- <br /> ously. moves able expand metabolic             sion 04/05/2011). <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                     Page 8 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br />   names compounds glycans                Comparison expression paralogous enzymes <br /> retrieved compound file glycan file          similarity transcriptional regulation paralo- <br /> derived KEGG LIGAND database.                        gous enzymes assessed Pearson&#226;&#8364;&#8482;s cor- <br />                                                               relation coefficient, calculated using data available Paralogy detection                                            SGD expression connection tool (http://www.yeast- analyses performed using paralogy          genome.org/cgi-bin/expression/expressionConnection. <br /> definitions. paralogy network defined consider-      pl). includes 29 datasets obtained different <br /> ing proteins S. cerevisiae nodes. pre-      experiments. joined data present pre- <br /> dicted ORFs sequences S. cerevisiae genome        clustered files  log2 space) global expression <br /> downloaded Ensembl release 59 (http://aug2010.           table. table removed columns (experi- <br /> archive.ensembl.org/) [24]. amino acid sequence           ments) 5% missing genes lines <br /> identity computed using needle, global alignment        (genes) 5% missing experiments. fil- <br /> algorithm included EMBOSS suite programs            tered table, containing 5294 genes 658 experiments, <br /> [23]. Pairs 30% sequence identity         used calculate Pearson&#226;&#8364;&#8482;s correlation coeffi- <br /> considered paralogs. paralogy assign-           cient paralogous pairs. <br /> ments derived ENSEMBL compara, release <br /> 59 (http://aug2010.archive.ensembl.org/) [24]        Additional material results analysis reported Wapinski et al. <br /> [13].                                                          Additional file 1: Supplementary Figures. Supplementary figures <br />                                                                reporting data obtained using alternative definitions metabolic <br />                                                                network paralogous pairs. <br /> Network analysis <br />                                                                Additional file 2: Supplementary Table 1. Detailed list Networks analysed using Cytoscape freeware            studied paralogous pairs. pair paralogy assignment (version 2.8.0) [50]. calculation connectivity      methods, DiceM coefficient relative networks using house developed Perl script.              Pearson correlation coefficient expression levels                                                                relative P values reported. <br />   properties empirical networks com- <br />                                                                Additional file 3: Supplementary Table 2. Paralogous pairs pared randomly collapsed networks.           partial functional overlap. table reports DiceP coefficient paralogous family formed enzymes          shortest path length pairs listed Table 1. <br /> present metabolic network, enzyme, ran- <br /> domly chosen, kept  connected nodes) removed. Short-          Abbreviations <br /> estPath finder plugin Cytoscape (http://csresources.      SGD: Saccharomyces Genome Database; YBP: Yeast Biochemical Pathways; <br /> sourceforge.net/ShortestPath/) used calculate      WGD: genome duplication. <br /> shortest path length metabolites.                   Acknowledgements <br />                                                               authors like thank Domenico Raimondo useful <br /> Dice coefficient                                              discussions. <br />                                                               work partially supported Award  KUK-I1-012-43 functional overlap paralogs measured          King Abdullah University Science Technology (KAUST), Fondazione Dice coefficient [25], using enzyme                Roma Italian Ministry Health, contract onc_ord 25/07, FIRB <br /> properties.                                                   ITALBIONET PROTEOMICA. <br />    E1 E2 enzymes, coefficient             Author details <br /> defined (DiceM) twice number shared metabo-         1 <br />                                                                Physics Department, Sapienza University Rome, Piazzale Aldo Moro, 5  <br /> lite pairs sum metabolite pairs processed      00185 Roma, Italy. 2Istituto Pasteur - FondazioneCenci Bolognetti, Sapienza <br />                                                               University Rome, Piazzale Aldo Moro, 5 00185 Roma, Italy. enzymes (DiceP) twice number com- <br /> mon pathways divided number pathways             Authors&#226;&#8364;&#8482; contributions enzymes involved:                               LG designed project wrote paper. LG performed                                                               analyses. authors read approved final manuscript. <br />    Dice = 2(X1 &#226;&#710;&#169; X2)(X1 &#226;&#710;&#170; X2) <br />                                                               Received: 12 2011 Accepted: 14 October 2011 <br />                                                               Published: 14 October 2011 <br />   X1 X2 number metabolite pairs pathways, respectively.                                    References <br />   reversibility reactions taken       1. Yamada T, Bork P: Evolution biomolecular networks: lessons account. order metabolites substrate           metabolic protein interactions. Nat Rev Mol Cell Biol 2009, 10:791-803. <br />                                                               2. Horowitz NH: Evolution Biochemical Syntheses. Proc Natl Acad <br /> &#194;&#174; product (KEGG) Left &#194;&#174; Right (YBP). Pathway                   Sci USA 1945, 31:153-157. <br /> definitions obtained KEGG YBP                   3. Jensen RA: Enzyme recruitment evolution new function. Annu Rev <br /> Metacyc.                                                          Microbiol 1976, 30:409-425. <br />  Grassi Tramontano BMC Evolutionary Biology 2011, 11:301                                                                                             Page 9 9 <br /> http://www.biomedcentral.com/1471-2148/11/301 <br />  <br />  <br />  <br />  <br /> 4.    Ycas M: earlier states biochemical  J Theor Biol 1974,        29. Quezada H, Aranda C, DeLuna  Hern&#195;&#161;ndez H, Calcagno ML, Mar&#195;&#173;n- <br />       44:145-160.                                                                        Hern&#195;&#161;ndez  Gonz&#195;&#161;lez  Specialization paralogue LYS21 <br /> 5.    Alves R, Chaleil RA, Sternberg MJ: Evolution enzymes metabolism:           determines lysine biosynthesis respiratory metabolism       network perspective. J Mol Biol 2002, 320:751-770.                                 Saccharomyces cerevisiae. Microbiology 2008, 154:1656-1667. <br /> 6.    D&#195;&#173;az-Mej&#195;&#173;a JJ, P&#195;&#169;rez-Rueda E, Segovia L: network perspective          30. Konig S: Subunit structure, function organisation pyruvate <br />       evolution metabolism gene duplication. Genome Biol 2007, 8:R26.              decarboxylases various organisms. Biochim Biophys Acta 1998, <br /> 7.    Copley RR, Bork P: Homology (betaalpha)(8) barrels: implications             1385:271-286. <br />       evolution metabolic pathways. J Mol Biol 2000, 303:627-641.         31. Vuralhan  Morais MA, Tai SL, Piper MD, Pronk JT: Identification 8.    Teichmann SA, Rison SC, Thornton JM, Riley M, Gough J, Chothia C:              characterization phenylpyruvate decarboxylase genes       evolution structural anatomy small molecule metabolic                   Saccharomyces cerevisiae. Appl Environ Microbiol 2003, 69:4534-4541. <br />       pathways Escherichia coli. J Mol Biol 2001, 311:693-708.                    32. Dickinson JR, Salgado LE, Hewlins MJ: catabolism amino acids 9.    Schmidt S, Sunyaev S, Bork P, Dandekar T: Metabolites: helping hand          long chain complex alcohols Saccharomyces cerevisiae. J Biol <br />       pathway evolution? Trends Biochem Sci 2003, 28:336-341.                            Chem 2003, 278:8028-8034. <br /> 10.   Dietrich FS, Voegeli S, Brachat S, Lerch  Gates K, Steiner S, Mohr C,        33. Ohno S: Evolution gene duplication London: George Alien <span id='am-5' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-6' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span>amp</span>; Unwin  <br />       P&#195;&#182;hlmann R, Luedi P, Choi S, et al: Ashbya gossypii genome tool           Berlin, Heidelberg New York: Springer-Verlag; 1970. <br />       mapping ancient Saccharomyces cerevisiae genome. Science               34. Conant GC, Wolfe KH: Turning hobby job: duplicated genes <br />       2004, 304:304-307.                                                                 new functions. Nat Rev Genet 2008, 9:938-950. <br /> 11.   Kellis M, Birren BW, Lander ES: Proof evolutionary analysis ancient     35. Hughes AL: evolution functionally novel proteins gene <br />       genome duplication yeast Saccharomyces cerevisiae. Nature 2004,             duplication. Proc Biol Sci 1994, 256:119-124. <br />       428:617-624.                                                                   36. Prince VE, Pickett FB: Splitting pairs: diverging fates duplicated <br /> 12.   Wolfe KH, Shields DC: Molecular evidence ancient duplication             genes. Nat Rev Genet 2002, 3:827-837. <br />       entire yeast genome. Nature 1997, 387:708-713.                             37. Conant GC: Rapid reorganization transcriptional regulatory <br /> 13.   Wapinski  Pfeffer  Friedman N, Regev  Natural history                    network genome duplication yeast. Proc Biol Sci 2010, <br />       evolutionary principles gene duplication fungi. Nature 2007,                 277:869-876. <br />       449:54-61.                                                                     38. Conant GC, Wolfe KH: Functional partitioning yeast expression <br /> 14.   Galagan JE, Henn MR, Ma LJ, Cuomo CA, Birren B: Genomics fungal             networks genome duplication. PLoS Biol 2006, 4:e109. <br />       kingdom: insights eukaryotic biology. Genome Res 2005,                    39. Fusco D, Grassi L, Bassetti B, Caselle M, Cosentino Lagomarsino M: Ordered <br />       15:1620-1631.                                                                      structure transcription network inherited yeast  <br /> 15.   Wagner  Robustness evolvability: paradox resolved. P R Soc B               genome duplication. BMC Syst Biol 2010, 4:77. <br />       2008, 275:91-100.                                                              40. Ward JJ, Thornton JM: Evolutionary models formation network <br /> 16.   Wagner  Neutralism selectionism: network-based reconciliation.             motifs modularity Saccharomyces transcription factor <br />       Nat Rev Genet 2008, 9:965-974.                                                     network. PLoS Comput Biol 2007, 3:1993-2002. <br /> 17.   Caspi R, Altman T, Dale JM, Dreher K, Fulcher CA, Gilham F, Kaipa P,           41. Tsoka S, Ouzounis CA: Functional versatility molecular diversity       Karthikeyan  Kothari  Krummenacker M, et al: MetaCyc database             metabolic map Escherichia coli. Genome Res 2001, 11:1503-1510. <br />       metabolic pathways enzymes BioCyc collection                 42. Jain R, Rivera MC, Lake JA: Horizontal gene transfer genomes:       pathway/genome databases. Nucleic Acids Res 2010, 38:D473-479.                     complexity hypothesis. Proc Natl Acad Sci USA 1999, 96:3801-3806. <br /> 18.   Karp PD, Ouzounis CA, Moore-Kochlacs C, Goldovsky L, Kaipa P, Ahr&#195;&#169;n D,         43. Light S, Kraulis P: Network analysis metabolic enzyme evolution       Tsoka S, Darzentas N, Kunin V, L&#195;&#179;pez-Bigas N: Expansion BioCyc              Escherichia coli. BMC Bioinformatics 2004, 5:15. <br />       collection pathway/genome databases 160 genomes. Nucleic Acids           44. Grassi L, Fusco D, Sellerio  Cor&#195;&#160; D, Bassetti B, Caselle M, Lagomarsino MC: <br />       Res 2005, 33:6083-6089.                                                            Identity divergence protein domain architectures yeast <br /> 19.   Dwight SS, Balakrishnan R, Christie KR, Costanzo MC, Dolinski K, Engel SR,         genome duplication event. Mol Biosyst 2010, 6:2305-2315. <br />       Feierbach B, Fisk DG, Hirschman J, Hong EL, et al: Saccharomyces genome        45. Guan Y, Dunham MJ, Troyanskaya OG: Functional analysis gene <br />       database: underlying principles organisation. Brief Bioinform 2004,            duplications Saccharomyces cerevisiae. Genetics 2007, 175:933-943. <br />       5:9-22.                                                                        46. Gasch AP, Spellman PT, Kao CM, Carmel-Harel O, Eisen MB, Storz G, <br /> 20.   Engel SR, Balakrishnan R, Binkley G, Christie KR, Costanzo MC, Dwight SS,          Botstein D, Brown PO: Genomic expression programs response       Fisk DG, Hirschman JE, Hitz BC, Hong EL, et al: Saccharomyces Genome               yeast cells environmental changes. Mol Biol Cell 2000, 11:4241-4257. <br />       Database provides mutant phenotype data. Nucleic Acids Res 2010, 38:           47. Ihmels J, Levy R, Barkai N: Principles transcriptional control       D433-436.                                                                          metabolic network Saccharomyces cerevisiae. Nat Biotechnol 2004, <br /> 21.   Kanehisa M, Goto S: KEGG: kyoto encyclopedia genes genomes.                 22:86-92. <br />       Nucleic Acids Res 2000, 28:27-30.                                               <br /> </body></html>