<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Dec. 1990, p. 6123-6131                                                                   Vol. 10,  12 <br /> 0270-7306/90/126123-09$02.00/0 <br /> Copyright C 1990, American Society Microbiology <br />  <br />  <br />                  Suppressor RNA Polymerase II Mutation                     Saccharomyces cerevisiae Encodes Subunit <br />                      Common RNA Polymerases  II, III <br />                      JACQUES ARCHAMBAULT, KEITH T. SCHAPPERT, JAMES D. FRIESEN* <br />                        Department Genetics, Hospital Sick Children, 555 University Avenue, <br />                               Department Medical Genetics, University Toronto, <br />                                             Toronto, Ontario M5G 1X8, Canada <br />                                           Received 29 1990/Accepted 3 September 1990 <br />  <br />               RNA polymerase H (RNAPII) complex multisubunit enzyme responsible synthesis pre-mRNA <br />            eucaryotes. enzyme large subunits associated smaller polypeptides, <br />            common RNA polymerase species. initiated genetic analysis RNAPII <br />            introducing mutations RP021, gene encoding largest subunit RNAPII Saccharomyces <br />            cerevisiae. used yeast genomic library isolate plasmids suppress temperature-sensitive <br />            mutation RP021 (rpo214), goal identifying gene products interact largest subunit <br />            RNAPII. increased expression wild-type RP026, single-copy, essential gene encoding            155-amino-acid subunit common RNAPI, RNAPII, RNAPIII, suppressed rpo2l4 temperature- <br />            sensitive mutation. Mutations constructed vitro resulted single <span id='am-23' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-24' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-25' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-27' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>amino acid</span> changes            carboxy-terminal portion RP026 gene product. temperature-sensitive mutation,            mutations did generate phenotype, lethal combination rpo214.            results support idea RP026 RP021 gene products interact. <br />  <br />   high degree conservation structure                   catalytic subunit bovine type 2A protein phosphatase <br /> function transcriptional apparatus eucaryotes                postulated modulate activity RNAPII established. example, shown                     altering level phosphorylation largest subunit <br /> transcription factors higher eucaryotes, TATA              enzyme. <br /> sequence-binding protein (TFIID) CCAAT-binding pro-                       Suppressors temperature-sensitive mutation teins functional homologs yeast Saccharomyces                 largest subunit RNAPII isolated. cerevisiae (12-14). addition, extensive similarity exists              independent, extragenic suppressors belonging single forms RNA polymerase (RNAP)                     complementation group shown alleles variety organisms, including single procaryotic            KEX2 gene, encodes protease required enzyme (1, 2, 6, 10, 19, 46). S. cerevisiae, RNAPII                                                                           processing alpha-factor killer-toxin precursors (31). <br /> composed 10 subunits. largest subunits                     study, clear KEX2 influence RNAPII, <br /> products RP021 [24, 48] RP022 [48] genes,                 precise mechanism remains obscure. <br /> called RPBJ RPB2) homologous bacterial &apos; proteins, respectively (1, 46). addition, RNAPII                    continuing series genetic studies, contains smaller polypeptides,               isolated linker-insertion mutations gene encoding <br /> subunits RNAPI RNAPIII (reviewed reference 43).                 largest subunit RNAPII (RP021; J. Archambault et <br />    developed genetics S. cerevisiae makes               al., unpublished data). allele particular, rpo214, <br /> attractive organism study multiprotein complexes               causes rapid growth arrest cells shifted nonper- RNAPII. isolation mutations                     missive temperature (Archambault et al., unpublished data). <br /> various subunits yeast RNAPII begun help                Pleiotropic phenotypes associated mutation include <br /> establish individual roles transcription (2, 3, 22, 24,          partial requirement inositol slow-growth <br /> 34, 35, 41, 50) promises fruitful approach           phenotype permissive temperature. inability subunit genes isolated (39). counterpoint                  grow medium lacking inositol documented mutant isolation search suppressors (26),            mutations genes encoding subunits RNAPII proven useful identifying new proteins involved             (RP021, RP022, RP024) (3, 50) accounted processes transcription. Suppressors HIS4 tran-               reduction level transcription INOI <br /> scriptional defect caused absence GCN4,                     gene, encodes enzyme inositol-1-phosphate syn- <br /> BASJ, BAS2 transcription activators identified loci              thase. <br /> (SIT] 4) (3).  SIT] SIT2, allelic                 approach identification genetic loci af- genes encoding largest subunits RNAPII.                    fecting transcription RNAPII gain SIT3, mutant form wild type (wt) high              understanding defect associated rpo214 <br /> copy number, suppress HIS4 defect absence TATA sequence. SIT4 encodes protein homologous                   temperature-sensitive mutation, isolated plasmids <br />                                                                           yeast genomic library suppress rpo214 <br />                                                                           temperature-sensitive defect. suppressor plasmid                                                                           shown carry wt RP026 gene, encodes subunit <br />   * <br />       Corresponding author.                                               common RNAPI, -II, -III. <br />  <br />                                                                    6123 <br />  6124      ARCHAMBAULT ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                         TABLE 1. Strains used                                 GAL] promoter (27, 47) yeast integrative plasmid <br />  Strain&quot;                           Relevant genotype                          carrying URA3 gene. Plasmid pRPO26cDNA (2,um <br />                                                                               TRPJ) isolated cDNA library described JAY212b .......... MATa rpo214                                                McKnight McConaughy (32). library  <br /> JAY435 .......... MATa rpo214(pRPO26) <br /> JAY430 .......... MATa rpo214(pRPO26dl)                                       structed cloning yeast cDNAs ADCJ pro- <br /> JAY429 .......... MA Ta rpo214(pRPO26d2)                                      moter CYCI terminator yeast episomal vector <br /> JAY431 .......... MATa rpo214(pRPO26d3)                                       (2,um TRPI ). yeast integrative plasmid, pJAY86, carrying <br /> JAY427 .......... MATa rpo214(pRPO26d4)                                       deletion allele RP026 (ARP026) constructed JAY432 .......... MATa rpo214(pRPO26d5)                                       isolation RP026 gap repair excision. Plasmid <br /> JAY433 .......... MATa rpo214(pJAY83)                                         pJAY86 (URA3 ARP026) carries internal deletion JAY436 .......... MATa(pRPO26)                                                RP026 spans Bsu36I sites nucleotides 230 JAY444 .......... MA Ta RP026:: LEU2(pRPO26)                                  1858. 5&apos; 3&apos; boundaries ARP026 allele JAY446 .......... MA Ta RP026:: LEU2(pRPO26)(pRPO26cDNA)                      nucleotides 1 2131, respectively. <br /> JAY467 .......... MA Ta rpo214(pRPO26cDNA) <br /> JAY472 .......... MATa rpo214 RP026:: pJAY97                                     Characteristics rpo2-4 allele used suppressor <br /> JAY476 .......... MATa RP026:: pJAY97                                         isolation. rpo214 allele isolated collection JAY490 .............. MATa rpo214 RP026:: pJAY97(prpo26-LI111)                mutant RP021 alleles generated vitro linker insertion <br /> JAY491 .......... MATa rpo214 RP026:: pJAY97(prpo26-LR111)                    mutagenesis (Archambault et al., unpublished data). site <br /> JAY492 ......... MATa rpo214 RP026:: pJAY97(prpo26-LI118)                     mutation determined DNA sequencing. JAY493 .......... MATa rpo214 RP026:: pJAY97(prpo26-LK118)                    rpo214 mutation result insertion JAY494 ......... MATa rpo214 RP026:: pJAY97(prpo26-LR118)                     12-bp XhoI linker (5&apos;-CTCGAGCTCGAG-3&apos;) HaeIII <br /> JAY495 .......... MATa rpo214 RP026:: pJAY97(prpo26-LI125)                    site position 3174 RP021 sequence (1). JAY496 ......... MATa rpo214 RP026:: pJAY97(prpo26-LK125)                     mutation results insertion amino acids, Leu Glu <br /> JAY497 ......... MATa rpo2-4 RP026:: pJAY97(prpo26-LI138) <br /> JAY498 .......... MATa rpo214 RP026:: pJAY97(prpo26-PA117)                    Leu Glu, amino acids 954 955 RP021 <br /> JAY499 ......... MATa rpo214 RP026:: pJAY97(prpo26-PA131)                     gene product. Yeast cells carrying rpo214 mutation JAY500 ......... MATa rpo214 RP026:: pJAY97(prpo26-EK144)                     temperature sensitive growth undergo round JAY501 ......... MATa rpo214 RP026:: pJAY97(pRPO26HA)                         cell division shift nonpermissive temperature <br /> JAY502 .......... MATa rpo214 RP026:: pJAY97(pFL39)                           (37&#194;&#176;C). slow-growth phenotype partial inositol <br /> JAY503 .........    MATTa RP026:: pJAY97(prpo26-LI111)                        requirement permissive temperature (30&#194;&#176;C) JAY504 .........    MATa RP026:: pJAY97(prpo26-LR111)                         associated rpo214 mutation. <br /> JAY505 ..........   MATa RP026:: pJAY97(prpo26-L1118)                            Construction yeast genomic library. construct JAY506 .........    MATa RP026:: pJAY97(prpo26-LK118)                         plasmid library used isolation RP026, DNA JAY507 ............ MATa RP026:: pJAY97(prpo26-LR118)                         yeast strain JAY357 partially digested restric- <br /> JAY508 .            MATx RP026:: pJAY97(prpo26-LI125) <br /> JAY509 .            MATTa RP026:: pJAY97(prpo26-LK125)                        tion endonuclease Sau3A, size fractionated sucrose <br /> JAY510 .            MATt RP026:: pJAY97(prpo26-LI138)                         gradient, cloned BamHI site plasmid pINT2 <br /> JAY511 .            MATot RP026:: pJAY97(prpo26-PA117)                        (URA3 CEN3 ARSI) (38); 40,000 independent bacterial <br /> JAY512 .            MATa RP026:: pJAY97(prpo26-PA131)                         colonies obtained. 80% clones carry <br /> JAY513 .            MATot RP026:: pJAY97(prpo26-EK144)                        yeast DNA inserts ranging size 5 15 kb. Yeast <br /> JAY514.             MA T RP026:: pJAY97(pRPO26HA)                             strain JAY357 fast-growing derivative JAY214 (MA Ta <br /> JAY515 .            MAT RP026:: pJAY97(pFL39)                                 ade2-1 cannl-100 his3-11,15 leu2-3,112 trpl-J ura3-1 PGAL- <br />   JAY strains constructed laboratory.     rpo214). pGAL-rpo2J4 allele constructed derived W303-1A (MATa canl-100 his3-11,15 leu2-3,112 trpl ura3-1       replacing endogenous RP021 gene rpo214 allele <br /> ade2-1) W303-1B (MATat canl-100 his3-11,15 leu2-3,112 trp1-1 ura3-1        control GAL1O promoter. characteris- <br /> ade2-1). strains obtained R. Rothstein.                        tics yeast strains JAY357 JAY214 presented   b Characteristics presented (Archambault et al., unpub- <br /> lished data). JAY212 derivative W303-1A RP021 gene   (Archambault et al., unpublished replaced rpo214.                                                      data). <br />                                                                                  <span id='am-15' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-18' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span>Nucleotide</span> sequence analysis. strands 2.0-kb <br />                                                                               KpnI-SpeI RP026 fragment sequenced chain <br />                MATERIALS METHODS                                          termination method (42), using combination deletions <br />    Strains media. S. cerevisiae strains used listed               commercially available primers primers synthesized Table 1. Growth media prepared described                        basis RP026 sequence. Single-stranded Sherman et al. (44). needed, 5-fluoro-<span id='am-5' about='obo:IMR_0200265' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:id" content="IMR:0200265" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00295" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-9' property="rdfs:label" content="Orotic acid" datatype="xsd:string"></span>orotic acid</span>                        double-stranded DNA templates used. <br /> (5-FOA) added concentration 1 mg/ml (7).                         DNA manipulation mutagenesis. Site-directed muta- <br /> plasmids propagated Escherichia coli JF1754 (22).                     genesis RP026 performed described Kunkel et <br /> Yeast transformation described Ito et al.                      al. (28). RP026 mutant alleles created sub- <br /> (25).                                                                         cloned plasmid pFL39. Hydroxylamine mutagenesis    Plasmids. Plasmid pJAY83 derivative pINT2                       plasmid pRPO26HA performed vitro essentially ClaI site located near CEN3 filled                     described previously (11). DNA manipulations using Klenow fragment DNA polymerase  Plasmid                        essentially described Maniatis et al. (30). <br /> pRPO26 contains 2.1-kb XbaI-ClaI RP026 fragment                            Isolation RP026 gap repair excision. Plasmid <br /> ClaI site derived pINT2 polylinker adjacent                    pJAY86 linearized unique Bsu36I site intro- BamHI site used construct library) cloned                 duced yeast cells. Repair integration plasmid unique XbaI ClaI sites pJAY83. Plasmid                        RP026 locus confirmed DNA blot analysis pRPO26HA contains RP026 fragment cloned                        transformants (data shown). indepen- <br /> pFL39 (TRPI CEN6 ARS; F. Lacroute). Plasmid                              dent transformants, plasmid integrated yeast <br /> pJAY97 contains portion RP026 gene (Sau3A                            chromosome recovered E. coli digestion fragment nucleotides 1026 1451) cloned downstream                     genomic DNA SpeI previously described (37). <br />  VOL. 10, 1990                                                   SUPPRESSOR RNA POLYMERASE II MUTATION                            6125 <br />  <br />                                                                                                                       PHENOTYPES <br />   STRAIN                                                                            Sequences                         30CC     3kC <br />                                RP026 FRAGMENT <br />                                                                                     remaining <br /> JAY   435                                                                          1-2131 (pRP026)                      +          + <br />  <br />  <br /> JAY   430                                                                          1- 1889 (pRP026d 1)                  +          + <br />  <br />  <br /> JAY   429                                                                          1- 1023 (pRP026d2) <br /> JAY   431                                                                          1-717 (pRP026d3)                     +_ <br />  <br />  <br /> JAY   427                                                                          1-273 (pRP026d4) <br /> JAY   432                                                                          1448-2131 (pRP026d5)                 +_ <br />  <br />  <br />  <br /> JAY   433                                                                           vector (pJAY83) <br />   FIG. 1. Suppression rpo214 temperature-sensitive phenotype DNA subclones. Various deletions created 2.0-kb <br /> suppressing fragment subcloned plasmid pJAY83. subclones introduced yeast strain JAY212 selecting Ura+ transformants. transformants tested growth 37&#194;&#176;C. bar indicates region remaining subclone. endpoints deletions determined DNA sequencing. +, Growth; -, absence growth. <br />  <br />  <br />   Nucleotide sequence accession number. nucleotide                 HindIII RP026 fragment probe. position sequence  reported appear EMBL, GenBank,              intron indicated Fig. 2. location ORF DDBJ nucleotide sequence data bases accession                consistent results deletion analysis presented <br /> number M33924.                                                          5&apos; 3&apos; ends cDNAs                                                                        indicated Fig. 2. <br />                            RESULTS                                        sequence ORF identical sequence                                                                        recently isolated gene encoding 17,908-Da subunit    Cloning suppressor rpo214. yeast genomic library          yeast RNAPII, cloned basis partial constructed  Materials Methods) used transform yeast strain carrying rpo214 allele                 amino acid sequence determination protein (49). (JAY212) order isolate plasmids suppress            refer gene RP026, sixth-largest subunit <br /> rpo214 temperature-sensitive mutation. 40,000 transfor-             RNAPII. <br /> mants obtained, 100 acquired temperature-resistant                  Examination predicted amino acid sequence phenotype 37&#194;&#176;C. Plasmid DNA 40 100 clones             RP026 gene product revealed distinct regions. isolated E. coli retested ability suppress       acidic region located 40 amino acids temperature-sensitive defect strain JAY212 (rpo214).            amino terminus protein, basic region located 40 plasmids continued display suppression;               middle portion protein amino acids 61 <br /> presumably remainder original 40 temperature-               97. regions predicted hydro- <br /> resistant clones revertants temperature-sensitive          philic (29). carboxy-terminal region RP026 gene <br /> phenotype strain JAY212. Restriction digest analysis             product contains heptad repetition leucine residues. plasmids HindlIl showed common yeast                     RP026 single-copy gene located chromosome XVI. <br /> DNA fragment 1.5 kb  used probe,                    copy number RP026 yeast genome hybridized plasmids high-stringency condi-               determined DNA blot hybridization total genomic <br /> tions. common 2.0-kb KpnI-SpeI fragment encompassing                 DNA (Fig. 3A). 1.5-kb HindlIl fragment used 1.5-kb Hindlll fragment shown carry sup-                probe RP026, difference observed pressing activity. Deletions 2.0-kb DNA fragment            pattern bands obtained high-stringency relatively constructed determine portions DNA                low stringency hybridization conditions. result suggests <br /> inserts essential suppression. Deletion sequences           single copy RP026 present yeast genome. residues 1023 1889 abolished suppression (Fig.                chromosomal location RP026 determined 1), indicating suppressor gene span region.         probing DNA blot yeast chromosomes    Nucleotide sequence analysis. nucleotide sequence            separated pulse-field gel electrophoresis (provided R. entire 2.0-kb KpnI-SpeI suppressing fragment deter-            Wevrick H. Willard). RP026 probe hybridized mined (Fig. 2). sequence revealed 155-amino-acid                 region chromosomes XIII XVI (Fig. 3B). open reading-frame (ORF) interrupted 76-nucleotide-               chromosomes distinguished reprobing long intervening sequence. putative intron contains            DNA blot chromosome XIII-specific probe highly conserved sequence motifs present yeast            ADH2 gene (33) probes simultaneously. introns (21): 5&apos; splice junction GTATGT, canonical             basis results, RP026 gene assigned branch point TACTAAC sequence near 3&apos; splice junc-                 chromosome XVI. <br /> tion, 3&apos; splice junction AG. position                RP026 essential gene. availability cloned <br /> intervening sequence amino terminus predicted            RP026 gene allowed determine gene ORF typical yeast introns. locations             required vegetative growth. disrupted allele intron boundaries confirmed sequencing ide-              RP026 constructed vitro replacing sequences <br /> pendent RP026 cDNAs isolated expression library               lie nucleotides 1023 1448 LEU2 <br /> constructed McKnight McConaughy (32).                     gene (Fig. 4). using step gene disruption method cDNAs isolated hybridization using 1.5-kb            (40), chromosomal allele RP026 replaced  6126   ARCHAMBAULT ET AL.                                                                                    MOL. CELL. BIOL. <br />  <br />                     TCTAGAAAAAGACATATCAGGACATATATGACACTTCCAATTTTTGGTTACTAGTGAGTC                      60 <br />                     GTCATGAATAATCATGTGCATTTGTAAACCATTTTCCCCCACACATGGTTTGCTACAAAT                      120 <br />                     AGGACATTTTAATTTAGGATGATCATTTTTTATATGCGATTGCAATTGTGACCATATCCG                      180 <br />                     AAACTCTTTGCAACAACCAGCAAAAGTACATTGGTAAGGATTTTCTACCTCAGGATCGTG                      240 <br />                     ATGTTTAGAAATGTGATTTCTTAGCCTATATGGCCTCTGAAAGCTTTTATTGCAGTGTGG                      300 <br />                     ACAGGTTAGTTTATGTAAATGAACAGATAAAATATGTGCCCTTAATTGTGGATGCTTGTA                      360 <br />                     GAATCGGAGGTTGCATCCTTCCTCTGGACAAATGAAAGATTTGGTATGCGTTACTTCGTG                      420 <br />                     TCGCTTCAGTTGCTGGCGAGTCGTCACTCCTTTTCCACAATAAGAACATTGGAATGGTTT                      480 <br />                     CGTATCAGAATGCGTATACAAGTGTCTCTCTAAGTGACTCTTTTTAACGAAGGATTTTGC                      540 <br />                     ACACTATCACACTGAAATGCTCTTAAACCCTGATGTACGCTTAATTGGTGTTCAGTCAAA                      600 <br />                     ATTGAAGGTCTTGTAAATGCCTATCACAGCATCATAGTCACAGAAATATGTCTTTGGCCT                      660 <br />                     ATTGCTCGATGAGCTTCTAGTTGATGTTAAACTGTTCAATGATTCTGAACTCTCTGATCG                      720 <br />                     TGATATGGGAATGGTCTCCTGTTTAAGTTCGGCCAGTGGCATTCCTTCATTATTTAGAAC                      780 <br />                     CTCTCCTCCCATTTGACAGCGATTTCAACAGTTACTACTGTGCTTAATGCCCTTATTTGA                      840 <br />                     TAGTTAGTTCTTCTTATAATAAATAATCATTGTATATTGATTATTCGTATAGTATGATAT                      900 <br />                     TTAATTGAAAAAAAAATTTTTTTTTTTTTAAGAATATCATTCAAAGGCATCAATCACAAC                      960 <br />                                                             3*          1,2* <br />                     CTTGAAGAAAGGCTAAAAAGACACATTTTGCAGGTAACAGTGTAAAGATTAAGGCTACAA                      1020 <br />                           M S D Y E E                     GTGATCATGTCAGACTACGAGGAGGCGTATGTGCATTCATATTGGAGAAATTGCGTGCTT 1080 <br />                                                                               F   N   D   G   N   E <br />                     TAGTTTTAATTATATTTACTAACTTCTATCATTGTGATACAGGTTTAACGACGGTAATGA 1140 <br />                       N   F   E   D   F   D   V   E   H   F   S   D   E   E   T   Y   E   E   K   P <br />                     GAACTTCGAAGATTTTGATGTAGAGCATTTTTCTGATGAGGAGACTTATGAGGAAAAACC 1200 <br />                      Q F K D G E T T D N G K T V T G G N <br />                     TCAATTCAAGGATGGTGAAACAACCGATGCCAACGGTAAGACCATCGTTACTGGTGGTAA 1260 <br />                      G P E D F Q Q H E Q R R K T L K E K                     TGGCCCAGAAGATTTTCAACAGCATGAGCAAATAAGAAGAAAGACACTTAAGGAAAAGGC 1320 <br />                      P K D Q R T T P Y M T K Y E R R                     CATCCCAAAAGACCAAAGAGCAACTACTCCATACATGACCAAATATGAAAGGGCAAGAAT 1380 <br />                      L G T R L Q S M N P V F V D L E G <br />                     TTTAGGTACAAGAGCCCTACAAATTTCCATGAATGCGCCAGTTTTTGTAGATTTAGAAGG 1440 <br />                      E T D P L R M K E L E K K P L V <br />                     TGAAACTGATCCATTGCGTATTGCCATGAAGGAGTTGGCTGAAAAGAAAATTCCTTTGGT 1500 <br />                      R R Y L P D G S F E D W S V E E L V <br />                     TATTAGAAGATATTTACCAGATGGTTCCTTTGAGGACTGGAGTGTGGAGGAACTCATTGT 1560 <br />                      D L - <br />                     GGATTTGTGATTACACCTGCATTTTCCTTTATGTGTATTTGCTTTGTTTGTATATTATTA 1620 <br />                                              1,2,3* <br />                     TAACTATAAAGAACGATTTATTTTGTAATGCAAATAACTGGACGAGAGAACCACGCTATA 1680 <br />                     ACCTATTTGGTAATAAAAATGTTTGTATGAATAAAAAAGCTGCTGACTGCTTTCTTTGAT 1740 <br />                        ATGTGTATATAGAATATTTATCGGTGATTGATCTAAGAAAAGCTTACCTCTGAAAGTTCT 1800 <br />                        CTCTTCGTCGCTTGCTGATTTGTAGTAAACAATTTGAAAAGTTTAGCAAATTCTTCCTCA 1860 <br />                        GGATTTTCAAAACCGGCTTCCTTTAAAGATCAATGACCTCATTCAAGGGAACGTTTAAAT 1920 <br />                        CGTGTCCTCTTCCAATGGCTTTTAAACTTTCTTCCAAATTCTTCACGCTCGGGGAAACTG 1980 <br />                        CGATAAGTTCTTCCCCATGATAGATAAGAATATGGGAAAGCTTACACCTTCTTCTCCAAC 2040 <br />                        TTCCGCGGTACTGGTGTCATTGTCAGGTACCATTTTGGACCATTCCTCATCTGTCAATGT 2100 <br />                        CTTCCCCAATGTTGCATATATCTTGGTAAGATC                                            2133 <br />   FIG. 2. Sequence 2.0-kb KpnI-SpeI suppressing fragment predicted amino acid sequence RP026 subunit. *, <br /> Locations 5&apos; 3&apos; ends RP026 cDNAs (1,2,3). <br />  <br /> RP026::LEU2 allele strain carrying plasmid-borne           ature-sensitive phenotype. mentioned  se- <br /> copy (URA3 marked) RP026 gene. deter-              quence RP026 ORF carried suppressor plasmid <br /> mined cells lost plasmid-borne copy           identical reported Woychik et al. (49). RP026 retained viability.  transformant          confirm RP026 gene carried suppressor <br /> integration RP026 ::LEU2 allele occurred        plasmid wt sequence entire length gene, <br /> chromosomal locus chosen basis DNA blot             sequenced independent copies RP026 analysis. Cells strain (JAY444) grown non-        isolated wt yeast strain method gap <br /> selective medium 10 generations            repair excision  Materials Methods). se- <br /> tested ability grow medium lacking leucine      quences copies determined  nucleotides <br /> including 5-FOA (1 mg/ml). Ura- cells grow            50 2133) compared sequence RP026 <br /> 5-FOA. cells recovered grow medium          gene originally isolated suppression rpo214 tem- <br /> containing 5-FOA (Fig. 4).  loss plasmid          perature-sensitive phenotype. cases, nucleotide <br /> taining URA3 RP026 lethal, indicating RP026       differences ORF upstream (976 <br /> gene essential. contrast, strain JAY444            nucleotides) downstream (567 nucleotides) untranslated <br /> transformed second plasmid carrying RP026               regions. results indicate increased gene dosage cDNA (JAY436), cells capable growth 5-FOA-contain-          wt RP026 sufficient bring suppression rpo214. <br /> ing medium recovered. result demonstrates             Increased expression RP026 cDNA suppresses RP026 gene product performs essential function         rpo2-4 temperature-sensitive phenotype. observation likely does belong family functionally        increased dosage RP026 sufficient suppress interchangeable proteins.                                         temperature-sensitive defect imposed rpo214 muta- <br />    Increased dosage RP026 suppresses rpo214 temper-        tion prompted investigate increasing expres- <br />  VOL. 10, 1990                                                                  SUPPRESSOR RNA POLYMERASE II MUTATION                        6127 <br />  <br />                                                                                                                                                                                              B) <br />                                              _        O <br />                                                     N                                  MOI(N <br />                                                                                          (N <br />                                    V <br />                                          o <br />                                          u        (//; <br />                                                                                          0 <br />                                                                                                  0         00- <br />                                   <span id='am-10' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-11' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-12' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-13' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span>COa</span>    2 <br />  <br />  <br />                                                                                                                          X <br />                                                                                                                          IV <br />  <br />  <br />                                         *4 <br />                                                           -   5.0 5                                                                                                                                 xv <br />                                                                                                                          XIS <br />                                                                                                                          1 <br />                                                           -4     27                                                      xii <br />                                                                                                                          XIV <br />                                                               -} 48                     114, <br />                                                                                           F~~                            X <br />                                                                                                                          xi <br />  <br />                                                           _ 981                                                          yin <br />                                                           - lg90                                                         III <br />                                                                                                                          VI <br />                                              _w           -   1 <br />                                                           -1 77 <br />                                                                   59 <br />  <br />  <br />                                                           -0 94 <br />                                                           -083 <br />  <br />  <br />  <br />   FIG. 3. DNA blot analysis chromosomal location RP026.   DNA blot analysis. Total genomic yeast DNA isolated wt <br /> yeast strain digested various restriction endonucleases. probe used 1.5-kb HindIll fragment encompassing RP026 <br /> ORF. DNA sizes expressed kilobase pairs. (B) Chromosomal location RP026. Yeast chromosomes separated contour-clamped homogeneous electric-field gel electrophoresis (16). probe used described  positions various yeast chromosomes indicated right. <br />  <br /> sion     RP026 cDNA lead              suppression,                 TRPJ) tested ability suppress temperature <br /> presumably result overproduction RP026                              sensitivity inositol auxotrophy rpo2l4 mutant cells. <br /> gene product. end, wt RP026 cDNA expressed                                   Strain JAY467 carrying RP026 cDNA acquired temper- promoter alcohol dehydrogenase 1 gene                                  ature resistance ability grow medium <br /> (pADC1) carried high-copy-number plasmid (2Rm                                 lacking inositol (Fig. 5), indicating increased transcrip- <br />  <br />  <br />                            <br />  <br />  <br />  <br />                                                                                                             H <br />                           B) <br />                                     qT~~~~~~~~~~~~~~~~~;. <br />                                            5/ <br />                                                                    s,-,- <br />  <br />  <br />                                                                                                                      v <br />  <br />  <br />  <br />  <br />   FIG. 4. Disruption RP026 gene effect cell viability.   Structure disrupted allele RP026. open box represent RP026 ORF. (B) Growth phenotypes cells spotted agar medium containing lacking 5-FOA 5 days incubation 30&apos;C. <br />  6128    ARCHAMBAULT ET AL.                                                                                              MOL. CELL. BIOL. <br />  <br />                                                                       Conditional expression RP026. yeast strain ex- <br />                                                                    presses RP026 strong inducible promoter, PGALI&apos; <br />       .-                                                           constructed verify increasing expression                                                                    chromosomal RP026 gene results suppression                                                                    rpo214 temperature-sensitive phenotype. GAL] pro- <br />                                                                    moter induced 1,000-fold cells grown medium <br />                                                                    containing galactose repressed presence                                                                    glucose (27, 47). strategy used depicted Fig. 6. <br />                                                                    Plasmid pJAY97, contains truncated RP026 ORF <br />                                                                    fused PGALIJ used transform JAY212 cells (rpo2l- <br />                                                                    4), creating strain JAY472. Integration directed                                                                    RP026 locus linearization plasmid restric- <br />                                                                    tion endonuclease AflII. Integration resulted duplication <br />                                                                    RP026 gene; truncated RP026 ORF placed <br />                                                                    control RP026 promoter, complete <br />                                                                    RP026 ORF controlled PGALI- expected, growth <br />                                                                    strain JAY472 (rpo214 RP026::pJAY97) galactose <br />                                                   Ii         2 dependent. strain longer temperature sensitive <br />                                               S w l        -&gt;      despite presence rpo214 mutation. result <br />                 rpo2r4 <br />   FIG. 5. Suppression          phenotyrpes increased expres- demonstrates suppression obtained overex- <br /> sion RP026 cDNA.   Structure th RP026 cDNA used   pression wt RP026 gene normal chromosomal <br /> transform yeast strain JAY212. RP026 cDNA cloned      location, presumably result overexpression downstream alcohol dehydrogenase promoter (ADCI)        RP026 gene product. <br /> upstream CYCI transcription terminiator 2,um plasmid.     addition, strain JAY476 (RPO21 RP026:: pJAY97) (B) Phenotype strain JAY212 untransforr ned transformed constructed. strain galactose dependent. RP026 cDNA. Cells spotted incubated 5 days agar medium 30 37&#194;&#176;C medium lacking inositol 30&apos;C. <br />                                                                    construction strains conditionally express RP026 <br /> Strain W303-1A (wt) used control                          provides rapid efficient assay determining                                                                    effects mutations RP026, cells trans- <br />                                                                    formed mutant RP026 alleles phenotypes tion wt RP026 ORF sufficient suppression.         assayed glucose-containing medium  . <br /> result strongly suggests supprevssion rpo214              Isolation temperature-sensitive mutation RP026. mutation occurs overexpressic:n RP026 gene          step understanding role RP026 gene <br /> product.                                                           product function RNAPI, -II, -III, tempera- <br />  <br />  <br />  <br />                                              j            CF      -- <br />  <br />                                   /1- <br />                                                           x <br />  <br />  <br />  <br />  <br />                                                ..1                         7- r <br />  <br />  <br />                                                                                 vp <br />  <br />  <br />  <br />  <br />   FIG. 6. Construction yeast strains conditionally express RP026.   Structures plasmid (pJAY97) used integration RP026 locus resulting partial gene duplication. (B) Growth phenotypes yeast strain JAY212 W303-1A transformed pJAY97. Cells spotted medium containing glucose galactose sole carbon source. <br />  VOL. 10, 1990                                                     SUPPRESSOR RNA POLYMERASE                           II   MUTATION   6129 <br />  <br />  <br />                               - <br />                                                              \\4L l[lil - <br />                                                                                                                        0         0 <br />  <br />  <br />  <br />  <br />                   RP026 <br />                   ALLELE                                                                    B) <br />  <br />  <br />                     WT              _   __   -_1                                ++                         0!                    0D <br />                                                                                                                  pmmq pin        - <br />                   L II                                                   +     +       LI I     L <br />                   LRI 1    &apos;l                                              +     +       LRII <br />                   LKi 18 <br />                           18                                               + <br />                                                                            + <br />                                                                                  + <br />  <br />  <br />                                                                                  - <br />                                                                                          L1l18 <br />                                                                                          LK 18 <br />                                                                                                             .1 <br />                   LRR 18                                                   +             LRI 18 <br />                                                                                                        <br />                    LI 125                                                  +    +1-      LI 125 <br />                    LK125                           K                       +             LK 1 25            r <br />                                                                                                       r <br />                    LI   138 <br />                                                                                                                                   +     -       L1138 <br />                    PIA1 17 <br />                                                               |..__   ._ <br />  <br />                                                                            +     +       PA1 17 <br />                                   ~~~ <br />                                   ~ <br />                                 -~~~~  - - -----                           +     +       PA131 <br />                    PAI 31 <br />                    EK 144                                                  ts    -       EK144 <br />                                                                                          WT <br />                                                                                          VECTOR <br />                                                                                                        -         hom   wol b- <br />    FIG. 7. Locations phenotypes point mutations RP026.   Summary locations various point mutations RP026 gene phenotypes combination RP021 rpo214 23&#194;&#176;C. RP026 gene product diagrammed schematically. filled box amino terminus represents acidic region, hatched box represents basic region, bars carboxy terminus <br /> represent heptad repeat leucine residues. +, Growth; -, absence growth. (B) Growth phenotypes strain JAY472 (rpo214 <br /> RP026:: pJAY97) transformed various RP026 alleles. cells spotted agar medium lacking <span id='am-42' about='obo:IMR_0200234' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-43' property="oboInOwl:hasDbXref" content="KEGG:C00106" datatype="xsd:string"></span><span id='am-44' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:id" content="IMR:0200234" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="Uracil" datatype="xsd:string"></span>uracil</span> containing glucose galactose sole carbon source. temperatures cells incubated indicated. <br />  <br />  <br />  <br /> ture-sensitive allele RP026 isolated using                     containing medium, growth strain dependent plasmid-shuffling strategy (7, 11). single-copy replicating              productive interaction mutant gene products. <br /> plasmid, pRP026HA (RP026 TRPI CEN4 ARSI),                              mutant rpo26 alleles (LK118, LR118, LK125, LH138, <br /> mutagenized hydroxylamine vitro  Materials                 EK144) inviable combination rpo214 (Fig. <br /> Methods) introduced yeast strain JAY444                       7). phenomenon, combination mutations <br /> [trpl-l RP026:: LEU2 (pRP026)]. Approximately 8,500                        lethal resulting cell death, <br /> Trp+ transformants grown 23&#194;&#176;C medium contain-                   termed synthetic lethality (18, 45); genetic <br /> ing 5-FOA select cells lost plasmid pRP026.                indication interaction gene products. 5-FOA-resistant colonies tested growth                interesting leucine mutations furthest <br /> 37&#194;&#176;C. temperature-sensitive lethal mutation RP026                   away carboxy terminus (LIlll, LR111, L1118) isolated carried G transition nucleotide                showed synthetic lethality. site rpo26 <br /> 1532, resulting <span id='am-30' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-31' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-33' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-38' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-39' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span><span id='am-41' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span>glutamic acid</span> lysine substitution               mutation moved closer carboxy terminus, growth amino acid 144 RP026 gene product. mutation                 presence rpo214 increasingly impaired (LK118, <br /> recessive, shown ability wt RP026 gene                   LR118, LK125, L1138), reduced level <br /> rescue temperature-sensitive phenotype (data                       mutation involved conservative change (LI125). <br /> shown).                                                                    Taken  results point role carboxy- <br />    Synthetic lethality RP021 RP026 mutations.                    terminal portion RP026 gene product interaction determine role,  heptad repetition                    RP021 gene product. interaction direct leucines carboxy end RP026 gene product,                     occur contact proteins, residues mutated vitro isoleucine, lysine,             subunits RNAPII. <br /> arginine.  proline residues located heptad repetition leucines changed alanine resi-                                          DISCUSSION <br /> dues. positions mutations phenotypes summarized Fig. 7. mutations                       Eucaryotic RNAPs complex enzymes discernible effect RP026 function; mutant                    large subunits associated smaller polypeptides <br /> alleles able complement RP026 LEU2 null                        (reviewed reference 43). begun genetic analysis <br /> allele PGALI-RPO26 allele (data shown). Comple-                 S. cerevisiae RNAPII goal defining role mentation detected 23 37&#194;&#176;C medium                       individual subunits transcription. end, containing lacking inositol.                                           constructed linker insertion mutations (Archambault et al., <br />    mutations constructed leucine-                 unpublished data) gene encoding largest subunit repeat region carboxy end RP026                   enzyme (RP021). allele, rpo214, displays tem- <br /> transferred strain containing GALI-RP026                  perature-sensitive phenotype partial inositol <br /> rpo2l4 alleles tested growth. glucose-                     requirement slow-growth phenotype permissive <br />  6130    ARCHAMBAULT ET AL.                                                                                          MOL . CELL . B IOL . <br />  <br /> temperature. gain better understanding molecular     RP021 RP026 gene products interact functionally; <br /> defect underlying various phenotypes, used       surprising, enzyme <br /> yeast genomic library isolate plasmids suppress      complex. interaction required rpo214 mutation. suppressor plasmids carry         structural functional integrity RNAPII. genes products interact functionally largest      subunits implicated DNA binding (9, <br /> subunit RNAPII.  report isolation             23); possibly basic region, located central <br /> characterization suppressor plasmid carries         portion RP026 protein, interact DNA. <br /> RP026, essential, single-copy gene encodes             feature RP026 gene product 17,980-Da subunit common RNAPs (49).                emerged sequencing analysis presence    primary finding work increased expression   highly acidic region amino terminus protein. RP026 suppress phenotypes associated             region potential fold alpha helix (15, <br /> rpo214. specifically, shown suppression        20). function domain unknown. presence rpo214 occur following conditions:            highly charged regions RP026 gene prod- <br /> increased copy number RP026 gene placing       uct provide explanation size discrepancy <br /> low-copy-number, replicating plasmid (CEN ARS); (ii)          predicted molecular weight protein (Mr <br /> creased expression RP026 placing RP026 cDNA             17,908) molecular weight observed polyacryl- control ADCJ promoter high-copy-           amide gels (Mr 23,000). Alternatively, difference number plasmid (2,um); (iii) increased transcription      phosphorylation RP026 gene product (4, 5, RP026 gene placing control strong     8) posttranslational modifications. availability <br /> GAL] promoter. cases, determined       cloned RP026 gene prove useful analysis RP026 ORF expressed wt sequence.             transcription RNAPI, -II, -III. <br /> observations strongly support suggestion             isolation subunit RNAPI, -II, -III creased wt RP026 gene product sufficient     suppressor mutation largest subunit RNAPII bring suppression. consistent            example use classical reverse genetics hypotheses:   rpo214 mutation increases dissocia-      characterize functional interaction subunits <br /> tion constant (Kd) RP026 RP021 gene              eucaryotic RNAP. reinforces validity products RP026 gene product           approach identification gene products partici- <br /> component RNAP complex, (ii) rpo214 allele         pate transcription. <br /> encodes unstable protein stabilized result forming larger complex. case,                               ACKNOWLEDGMENTS <br /> overproduction RP026 gene product shift           thank colleagues, especially Emmanuel Maicas Alan <br /> equilibrium complex formation result stabi-      Davidson, discussions, Jason Zacks Peter Warburton lize rpo214 gene product. observation increas-      technical assistance, Jacqueline Segall critical reading ing transcription rpo214, placing gene       manuscript constant support course work. control strong GALIO promoter, does suppress      thank Rachel Wevrick Huntington Willard providing temperature-sensitive phenotype relevant possi-       yeast chromosome blot. <br /> bilities (data shown) suggests increased       J  holds studentship Medical Research Council rpo214 gene product sufficient bring     Canada. work supported grant Medical suppression, determined          Research Council Canada. <br /> increased transcription rpo214 results greater accumu-                             LITERATURE CITED <br /> lation encoded gene product. <br />     Suppression increased expression wt gene          1. Allison, L. , M. Moyle, M. Shales, C. J. Ingles. 1985. observed  example, Clark-Adams et al.             Extensive homology largest subunits eukaryotic <br />                                                                      prokaryotic RNA polymerases. Cell 42:599-610. <br />  (17) screened high-copy-number clones          2. Allison, L. , K.-C. Wong, V. D. Fitzpatrick, M. Moyle,  suppress solo insertion mutations yeast.             C. J. Ingles. 1988. C-terminal domain largest subunit <br />  selection, isolated wt histone locus HTAI-            RNA polymerase II Saccharomyces cerevisiae, Droso- <br /> HTBI, contains genes previously identified genet-          phila melanogaster, mammals: conserved structure  ically SPTIJ SPT12.                                          essential function. Mol. Cell. Biol. 8:321-329. <br />     effort begin understanding regions      3. Arndt, K. T., C.  Styles, G. R. Fink. 1989. suppressor <br /> RP026 gene product important function(s),              H1S4 transcriptional defect encodes protein homol- <br />  temperature-sensitive allele RP026 isolated using         ogy catalytic subunit protein phosphatases. Cell <br />  random <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span> mutagenesis. temperature-sensitive              56:527-537. <br />                                                                   4. Bell, G. , P. Valenzuela, W. Rutter. 1976. Phosphorylation <br />  mutation results glutamic acid lysine substitution       yeast RNA polymerases. Nature (London) 261:429-431. <br />  codon 144. addition, mutations constructed        5. Bell, G. , P. Valenzuela, W. J. Rutter. 1977. Phosphory- <br />  carboxy end RP026 region heptad repeat            lation yeast DNA-dependent RNA polymerases vivo  leucines. Site-directed mutagenesis leucine residues       vitro. J. Biol. Chem. 252:3082-3091. <br />  indicated essential vegetative growth.     6. Biggs, J., L. L. Searles,  L. Greenleaf. 1985. Structure   mutations lethal combined             eukaryotic transcription apparatus: features gene  rpo214. synthetic lethality genetic indication       largest subunit Drosophila RNA polymerase II. Cell <br />  interaction RP021 RP026 gene products            42:611-621. <br />  involve carboxy-terminal portion           7. Boeke, J. D., F.    Lacroute, G. R. Fink. 1984. positive <br />                                                                      selection mutants lacking orotidine-5&apos;-phosphate decarbox- <br />  RP026 gene product. examples inviable combina-             ylase activity yeast: 5-fluoro-orotic acid resistance. Mol. Gen. <br />  tions mutant suppressor alleles observed           Genet. 197:345-346. <br />  genes encoding components cytoskeleton S.          8. Breant, B., J.-M. Buhler,  Sentenac, P. Fromageot. 1983. <br />  cerevisiae (36).                                                    phosphorylation yeast RNA polymerase B. <br />     genetic analysis presented established         Eur. J. Biochem. 130:247-251. <br />  VOL. 10, 1990                                                      SUPPRESSOR RNA POLYMERASE II MUTATION                           6131 <br />  <br />  9. Breant, B., J. Huet,  Sentenac, P. Fromageot. 1983.             29. Kyte, J., R. F. Doolittle. 1982. simple method     Analysis yeast RNA polymerases subunit-specific anti-             displaying hydropathic character protein. J. Mol. Biol. <br />     bodies. J. Biol. Chem. 258:11968-11973.                                   157:105-132. <br /> 10. Broyles, S. S., B. Moss. 1986. Homology RNA               30. Maniatis, T., E. F. Fritsch, J. Sambrook. 1982. Molecular <br />     polymerases poxviruses, prokaryotes, eukaryotes: nu-               cloning: laboratory manual. Cold Spring Harbor Laboratory, <br />     cleotide sequence transcriptional analysis vaccinia virus          Cold Spring Harbor, N.Y. <br />     genes encoding 147-kDa 22-kDa subunits. Proc. Natl. Acad.         31. Martin, C., R.  Young. 1989. KEX2 mutations suppress <br />     Sci. USA 83:3141-3145.                                                    RNA polymerase II mutants alter temperature range 11. Budd, M., J. L. Campbell. 1987. Temperature-sensitive                 yeast cell growth. Mol. Cell. Biol. 9:2341-2349. <br />     mutations yeast DNA polymerase. Proc. Natl. Acad. Sci.         32. McKnight, G. L., B. L. McConaughy. 1983. Selection     USA 84:2838-2842.                                                         functional cDNAs complementation yeast. Proc. Natl. <br /> 12. Buratowski, S., S. Hahn, P.  Sharp, L. Guarente. 1988.              Acad. Sci. USA 80:4412-4416. <br />     Function yeast TATA element-binding protein mam-            33. Mortimer, R. K., D. Schild, C. R. Contopoulou, J.  Kans. <br />     malian transcription  Nature (London) 334:37-42.                   1989. Genetic map Saccharomyces cerevisiae, edition 10. <br /> 13. Cavailini, B., J. Huet, J.-L. Plassat,  Sentenac, J.-M. Egly,       Yeast 5:321-403. <br />     P. Chambon. 1988. yeast activity substitute HeLa        34. Nonet, M., D. Sweetser, R.  Young. 1987. Eukaryotic <br />     cell TATA box factor. Nature (London) 334:77-80.                          RNA polymerase conditional mutant rapidly ceases mRNA <br /> 14. Chodosh, L. , J. Olesen, S. Hahn,  S. Baldwin, L. Guarente,           synthesis. Mol. Cell. Biol. 7:1602-1611. <br />     P.  Sharp. 1988. yeast human CCAAT-binding              35. Nonet, M., D. Sweetser, R.  Young. 1987. Functional <br />     protein heterologous subunits functionally inter-           redundancy structural polymorphism large subunit     changeable. Cell 53:25-35.                                                RNA polymerase II. Cell 50:909-915. <br /> 15. Chou, P. Y., G. D. Fasman. 1978. Prediction secondary          36. Novick, P., B. C. Osmond, D. Botstein. 1989. Suppressors     structure proteins amino-acid sequence. Adv.                yeast actin mutations. Genetics 121:659-674. <br />     Enzymol. 45:45-148.                                                   37. Orr-Weaver, T. L., J. W. Szostak, R. J. Rothstein. 1983. <br /> 16. Chu, G., D. Vollrath, R. W. Davis. 1986. Separation large          Genetic application yeast transformation linear gapped <br />     DNA molecules contour-clamped homogenous electric                      plasmids. Methods Enzymol. 101:202-210. <br />     fields. Science 234:1582-1585.                                        38. Percival-Smith, , J. Segall. 1986. Characterization 17. Clark-Adams, C. D., D. Norris, M.  Osley, J. S. Fassler,            mutational analysis cluster genes expressed pref- <br />     F. Winston. 1988. Changes histone gene dosage alter tran-              erentially sporulation Saccharomyces cerevisiae. Mol. <br />     scription yeast. Genes Dev. 2:150-159.                                 Cell. Biol. 6:2443-2451. <br /> 18. Dobzhansky, T. 1946. Genetics natural populations. XIII.           39. Riva, M., S. Memet, J.-Y. Micoin, J. Huet,  Treich, J. Dassa, <br />     Recombination variability populations Drosophila                R. Young, J.-M. Buhler,  Sentenac, P. Fromageot. 1986. <br />     pseudoobscura. Genetics 31:269-290.                                       Isolation structural genes yeast RNA polymerases 19. Falkenburg, D., B. Dworniczak, D. M. Faust, E. K. F.                  immunological screening. Proc. Natl. Acad. Sci. USA 83:1554- <br />     Bautz. 1987. RNA polymerase II Drosophila. Relation             1558. <br />     140,000 Mr subunit beta subunit Escherichia coli RNA        40. Rothstein, R. J. 1983. step gene disruption yeast. <br />     polymerase. J. Mol. Biol. 195:929-937.                                    Methods Enzymol. 101:202-211. <br /> 20. Garnier, J., D. J. Osguthorpe, B. Robson. 1978. Analysis       41. Ruet, ,  Sentenac, P. Fromageot, B. Winsor, F. Lac- <br />     accuracy implications simple methods predicting            route. 1980. mutation B220 subunit gene affects     secondary structure globular proteins. J. Mol. Biol.               structural functional properties yeast RNA polymerase B <br />     120:97-120.                                                               vitro. J. Biol. Chem. 255:6450-6455. <br /> 21. Green, M. R. 1986. Pre-mRNA splicing. Annu. Rev. Genet.               42. Sanger, F., S. Nicklen,  R. Coulson. 1977. DNA sequenc- <br />     20:671-708.                                                               ing chain-terminating inhibitors. Proc. Natl. Acad. Sci. <br /> 22. Himmelfarb, H. J., E. M. Simpson, J. D. Friesen. 1987.                USA 74:5463-5467. <br />     Isolation characterization temperature-sensitive RNA           43. Sentenac,  1985. Eukaryotic RNA polymerases. Crit. Rev. <br />     polymerase II mutants Saccharomyces cerevisiae. Mol. Cell.             Biochem. 18:31-91. <br />     Biol. 7:2155-2164.                                                    44. Sherman, F., G. R. Fink, J. B. Hicks. 1986. Laboratory <br /> 23. Huet, J., M. Riva,  Sentenac, P. Fromageot. 1985. Yeast             course manual methods yeast genetics. Cold Spring <br />     RNA polymerase C subunit. J. Biol. Chem. 260:15304-               Harbor Laboratory, Cold Spring Harbor, N.Y. <br />     15310.                                                                45. Sturtevant,  H. 1956. highly specific complementary lethal <br /> 24. Ingles, C. J., H. J. Himmelfarb, M. Shales,  L. Greenleaf,          Drosophila melanogaster. Genetics 41:118-123. <br />     J. D. Friesen. 1984. Identification, molecular cloning, mu-       46. Sweetser, D., M. Nonet, R.  Young. 1987. Prokaryotic     tagenesis Saccharomyces cerevisiae RNA polymerase genes.               eukaryotic RNA polymerases homologous core subunits. <br />     Proc. Natl. Acad. Sci. USA 81:2157-2161.                                  Proc. Natl. Acad. Sci. USA 84:1192-1196. <br /> 25. Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor-         47. Yocum, R. R., S. Hanley, R. West, Jr <br /> </body></html>