<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body> <br />                                               Bacillus subtilis DnaD protein: putative link DNA <br />                                               remodeling initiation DNA replication <br />                                               Abstract <br />                                                    Bacillus subtilis DnaD protein essential protein component oriC PriA <br />                                                    primosomal cascades, responsible loading main replicative ring helicase DnaC <br />                                                    DNA. present evidence DnaD global DNA architectural activity, <br />                                                    assembling large nucleoprotein complexes plasmid counteracting plasmid <br />                                                    compaction manner analogous recently seen histone-like Escherichia coli HU <br />                                                    proteins. DNA-remodeling role essential function initiation DNA replication <br />                                                    Gram +ve B. subtilis, highlighting DnaD link bacterial nucleoid <br />                                                    reorganization initiation DNA replication. <br />  <br />                                                    DNA replication; Atomic force microscopy; DnaD; Nucleoid <br />  <br />  <br />                                               1. Introduction <br />                                                                Bacterial chromosomal DNA organized high order condensed structure called                                                                nucleoid [1]. Histone-like DNA remodeling proteins form structural scaffolds                                                                nucleoid, dynamic nucleoprotein structure changes cell growth [2-4]. <br />                                                                architectural transitions promoted growth-phase dependent variations                                                                histone-like protein  DNA-binding proteins) composition bacterial <br />                                                                proteome [5,6]. Global nucleoid remodeling associated crucial biological functions <br />                                                                regulation gene expression [7], chromosome segregation [8] initiation                                                                DNA replication [9,10]. <br />  <br />                                                                Bacillus subtilis DnaD protein implicated primosomal DNA initiation <br />                                                                cascade involves DnaB DnaI loads replicative ring helicase, DnaC,                                                                oriC non-specific DNA PriA-mediated restart primosome [11-13].                                                                essential role DnaD confirmed genetically Staphylococcus aureus [14].                                                                interacts DnaA [15] PriA [16] early steps primosomal cascade                                                                set stage helicase loading. primosomal cascade sequential DnaA   <br />  <br />                                               &#194;&#169; 2004 Published Elsevier B.V. behalf Federation European Biochemical Societies. <br />                                               * <br />                                                 Corresponding author. Fax: +44-115-8468002. panos.soultanas@nottingham.ac.uk (P. Soultanas).. <br />                                          Turner et al.                                                                                          Page 2 <br />  <br />  <br />                                                         PriA)&#226;&#8364;&#8220;DnaD&#226;&#8364;&#8220;DnaB order set initial complex recruits DnaC [16]. DnaC <br />                                                         exists complex DnaI [17,18], whilst role DnaB unclear. act <br />                                                         DnaI forming pair helicase loaders [18] alternatively membrane <br />                                                         attachment protein regulate initiation DNA replication regulating recruitment                                                         DnaD membrane [19,20]. role cell membrane initiation DNA <br />                                                         replication B. subtilis documented [21], precise molecular mechanisms                                                         underpin function just started unraveled [22]. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         provide evidence addition role initiation DNA replication, DnaD <br />                                                         functions global regulator DNA architecture manner similar seen                                                         HU H-NS proteins Escherchia coli. forms variety higher order <br />                                                         nucleoprotein structures remodel supercoiled plasmid open circular form. <br />                                                         DNA binding activity DnaD resistant high NaCl concentration appears                                                         induce assembly higher order DnaD oligomers. Finally, DnaD disrupts stable <br />                                                         interaction helicase DnaI vitro, property functionally <br />                                                         relevant helicase loading. role DnaD initiation DNA replication                                                         ability alter global architecture DNA make obvious candidate link <br />                                                         nucleoid reorganization DNA replication B. subtilis. speculative model                                                         combining dual role presented. <br />  <br />                                            2. Materials methods <br />                                            2.1. Protein purifications <br />                                                         dnaD gene cloned PCR B. subtilis (strain 168 EMG50) genomic DNA                                                         NdeI-BamHI sites pET22b (Novagen). Overexpression DnaD carried                                                         B834 (DE3) E. coli 1 mM IPTG 7&#226;&#8364;&#8220;8 h exponentially growing cultures (OD600 <br />                                                         = 0.6). DnaD overexpressing E. coli grew noticeably slower induced cells. Cells <br />                                                         harvested 20 mM Tris, pH 7.5, 2 mM EDTA, 1 mM DTT, 500 mM NaCl, 20 w/ <br />                                                         v sucrose, sonicated incubated 30 min 37&#194;&#176;C benzonase (Novagen) cell <br />                                                         extract clarified centrifugation 17 500 rpm 30 min. Total protein                                                         clarified supernatant precipitated ammonium sulfate (2.9 mg 10 ml) Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         centrifuged 17 500 rpm, 30 min. protein pellet suspended GE (20 mM <br />                                                         <span id='am-7' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-8' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-11' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-12' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-17' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>glycine</span>, pH 9.8, 2 mM EDTA, 1 mM DTT) buffer conductivity reached 40 mS <br />                                                         loaded sourceQ column, pre-equilibrated GE, 10&#226;&#8364;&#8220;12 mS. column                                                         washed GE/100 mM NaCl DnaD eluted 100&#226;&#8364;&#8220;400 mM NaCl gradient                                                         GE. Relevant fractions pooled DnaD precipitated ammonium sulfate,                                                         described  protein pellet suspended BisTE (Bis-Tris, pH 6.7, 2 mM <br />                                                         EDTA, 1 mM DTT) buffer maintaining conductivity 40 mS loaded                                                         HiTrap 5 ml Heparin, pre-equilibrated BisTE, 15&#226;&#8364;&#8220;20 mS. column washed                                                         Bis-TE/100 mM NaCl DnaD eluted 100&#226;&#8364;&#8220;400 stepping 700 mM <br />                                                         NaCl. relevant fractions pooled loaded superdex S75 gel filtration <br />                                                         column pre-equilibrated TE/300 (20 mM Tris, pH 7.5, 2 mM EDTA, 1 mM DTT, 300 <br />                                                         mM NaCl) buffer. molecular weight DnaD 27.5 kDa elution profile <br />                                                         indicated oligomer. Typically, 8&#226;&#8364;&#8220;9 mg DnaD liter bacterial culture                                                         obtained. Purity assessed SDS&#226;&#8364;&#8220;PAGE concentration determined <br />                                                         spectrophotometrically. <br />  <br />                                            2.2. Sedimentation velocity ultracentrifugation experiments <br />                                                         Analytical ultracentrifugation sedimentation velocity experiments carried                                                         Beckman XL analytical ultracentrifuge using cells channel centerpieces, loaded <br />                                                         hole AnTi 60 rotor. experiments carried 40 000 rpm. Cells  <br />  <br />  <br />                                                             FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                            Page 3 <br />  <br />  <br />                                                          scanned 5 min 280 nm temperature set 20 &#194;&#176;C. DnaD concentrations <br />                                                          4.9, 2.4 1.2 &#206;&#188;M. <br />  <br />                                                          Sedimentation coefficient distributions generated using program SEDFIT [23] <br />                                                          using c(s) method analysis. Errors distribution estimated 67%                                                          95% confidence levels using 1000 runs Monte Carlo simulation. Frictional coefficients <br />                                                          theoretical distributions oligomers estimated using program SEDNTERP <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                          [24]. <br />  <br />                                                          Experiments designed observe DNA dependent behavior aggregation carried <br />                                                          5&#226;&#8364;&#178; HEX labeled 33mer oligonucleotide (5&#226;&#8364;&#178;-HEX- <br />                                                          TGCATGCCTGCAGGTCGACTCTAGAGGATCCCC-3&#226;&#8364;&#178;) data obtained                                                          wavelength 536 nm. <br />  <br />                                            2.3. Gel shift assays <br />                                                          DNA binding reactions carried using radioactively labeled 33mer oligonucleotide <br />                                                          (5&#226;&#8364;&#178;-TGCATGCCTGCAGGTCGACTCTAGAGGATCCCC-3&#226;&#8364;&#178;) 2.5 nM 50 mM Tris, <br />                                                          pH 7.4, 100 mM NaCl, 1 mM EDTA, 1 mM DTT increasing concentrations DnaD <br />                                                          (123&#226;&#8364;&#8220;738 nM). Mixtures incubated 10 min room temperature complexes <br />                                                          resolved non-denaturing PAGE. Gels dried radioactive bands                                                          visualized using phosphorimager. <br />  <br />                                            2.4. Analytical gel filtration <br />                                                          Gel filtration experiments carried described [17] using mixtures B. <br />                                                          stearothermophilus DnaB (4.52 &#206;&#188;M), DnaD (1.96 M) DnaI (11.22 &#206;&#188;M), appropriate. <br />                                                          Proteins incubated 50 mM Tris, pH 7.5, 250 mM NaCl, 1 mM EDTA, 1 mM <br />                                                          DTT 10 min ice fractionated superdex S200 10/30 column <br />                                                          (Amersham Pharmacia Biotech). Fractions peaks analyzed SDS-PAGE. <br />                                                          Mixtures DnaB/DnaD/DnaI, DnaB/DnaI, DnaB/DnaD, DnaI/DnaD                                                          individual proteins analyzed similar manner. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                            2.5. Atomic force microscopy imaging <br />                                                          DnaD samples diluted 0.5 0.005 &#206;&#188;g/ml 20 mM Tris, pH 7.5, 2 mM <br />                                                          EDTA, 300 mM NaCl. Plasmid pBR322 purchased Sigma-Aldrich. Plasmid <br />                                                          samples stock solution 10 &#206;&#188;g/ml dH2O diluted                                                          desired concentration 10% v/v PBS containing 0.5mM MgCl2. solutions                                                          filtered 0.2 &#206;&#188;M pore size filters (Sartorius) prior use. Protein DNA complexes <br />                                                          prepared mixing protein samples diluted 0.25&#226;&#8364;&#8220;0.0025 &#206;&#188;g/ml 20 mM Tris, pH <br />                                                          7.5, 2 mM EDTA, 300 mM NaCl, 0.5 0.15 &#206;&#188;g/ml pBR322 diluted                                                          10% v/v PBS containing 0.5 mM MgCl2 generate varying molar ratios protein DNA. <br />  <br />                                                          imaging air, samples 5&#226;&#8364;&#8220;10 &#206;&#188;l incubated freshly cleaved mica (Agar <br />                                                          scientific) 5&#226;&#8364;&#8220;10 s rinsed 5 ml dH2O allowed dry gentle flow <br />                                                          nitrogen <span id='am-1' about='obo:IMR_0001720' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-2' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-3' property="obo:IAO_0000115" content="The vapor state of matter; nonelastic fluids in which the molecules are in free movement and their mean positions far apart. Gases tend to expand indefinitely, to diffuse and mix readily with other gases, to have definite relations of volume, temperature, and pressure, and to condense or liquefy at low temperatures or under sufficient pressure." datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0001720" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:hasDbXref" content="MeSH:D005740" datatype="xsd:string"></span><span id='am-6' property="rdfs:label" content="gas" datatype="xsd:string"></span>gas</span>. imaging liquid, samples 30 &#206;&#188;l spotted single aliquot                                                          underside cantilever, placed inside liquid cell placed directly freshly <br />                                                          cleaved mica. <br />  <br />                                                          atomic force microscopy (AFM) imaging carried Digital Instruments <br />                                                          Nanoscope III Mutlimode AFM type E scanner (Veeco) tapping mode.                                                          cantilevers used silicon tapping probes spring constant 34.4&#226;&#8364;&#8220;74.2 N/m <br />                                                          (Olympus, OMCL-AC160TS). tapping set point adjusted minimize probe&#226;&#8364;&#8220; <br />                                                          sample interactions. Images recorded topography phase modes pixel <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                             Page 4 <br />  <br />  <br />                                                         size 512 &#195;&#8212; 512, flattened analyzed WSxM software 3.0 Beta 2.3 (Nanotec <br />                                                         Electronic S.L, Spain). <br />  <br />                                            3. Results discussion <br />                                            3.1. DnaD forms higher order oligomers concentration dependent manner <br />                                                         DnaD forms mainly dimers trimers smaller proportion higher order <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         oligomers [15,16]. variety techniques ranging yeast hybrid technology, <br />                                                         glycerol density sucrose density centrifugation, gel filtration glutaraldehyde <br />                                                         fixing used establish DnaD oligomeric state, cases purified DnaD                                                         tagged tag intein tag facilitate purification. purified untagged <br />                                                         DnaD subjected analytical gel filtration calibrated superdex S-75 gel <br />                                                         filtration column. DnaD eluted apparent tetramer (data shown).  <br />                                                         oligomerization estimations gel filtration inaccurate depend                                                         combination shape size [25]. <br />  <br />                                                         Sedimentation velocity experiments concentrations revealed sedimentation <br />                                                         coefficient distributions consistent species solution interact                                                         concentration dependent manner (Fig. 1A). lowest concentration (1.2 &#206;&#188;M), frictional <br />                                                         coefficients expected oligomeric species (Table 1) reveal monomer <br />                                                         (sedimenting 2.0 s) physically reasonable value (f/f0 = 1.488). Increasing                                                         concentration leads second peak 4.5 s (corresponding compact dimer                                                         extended trimer; Table 1), higher order species excess 6 s (pentamer                                                         hexamer, judged reasonable frictional coefficients) present. Separation                                                         species distinct sedimentation coefficient distributions reveals specific type                                                         behavior. known theoretically experimentally impossible separate <br />                                                         rapidly reversible monomer/dimer equilibrium using sedimentation analysis;                                                         weighted average distribution produced [26]. second species dimer                                                         presented studies, mean rate dimerization slow timescale                                                         sedimentation. second possibility samples undergo irreversible aggregation,                                                         separate distinct species appreciable rate aggregation.  <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                          unlikely dilution sample sedimentation yields small <br />                                                         species: irreversible aggregation yield proportion species, irrespective                                                         concentration, dilution. scenario species monomer/n-mer <br />                                                         rapidly reversible equilibrium, n &gt; 2. Theory experiment separation                                                         species physically possible accepts cooperative transition                                                         monomer trimer tetramer. present highly extended conformations <br />                                                          judged physically reasonable frictional coefficients). AFM images DnaD various <br />                                                         concentrations verified formation higher extended oligomeric species high <br />                                                         concentrations, giving weight hypothesis. Elongated strand-like circular <br />                                                         semi-circular structures visible high concentrations (Fig. 1B). smallest <br />                                                         features representing DnaD monomers approximately 25 nm longest dimension <br />                                                         larger features 50 nm (dimers), 79 nm (trimers) longer oligomers                                                         apparent. <br />  <br />                                            3.2. DnaD-mediated nucleoprotein complexes <br />                                                         proceeded investigate binding DnaD DNA HEX-tagged synthetic <br />                                                         oligonucleotide follow independently sedimentation protein (280 nm)                                                         Hex-oligonucleotide (536 nm). observed addition oligonucleotide                                                         DnaD, extremely large nucleoprotein complex formed (1&#226;&#8364;&#8220;2 million Daltons)                                                         impossible investigate analytical ultracentrifugation. Instead, DNA binding <br />                                                         activity investigated gel shifts using untagged radioactively labeled <br />  <br />  <br />                                                             FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                              Page 5 <br />  <br />  <br />                                                         oligonucleotide (Fig. 2A). DnaD bound oligonucleotide producing clear shifted band <br />                                                         DnaD concentration increased streak started appear higher gel                                                         shifted band ended highest DnaD concentration. remarkable <br />                                                         observation additional DnaD molecules directed exclusively initial <br />                                                         nucleoprotein complex free oligonucleotide, high DnaD <br />                                                         concentrations substrate shifted (data shown).                                                         stoichiometry DnaD versus oligonucleotide initial complex certain, Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         comparisons shifted bands proteins implied                                                         DnaD molecules bound oligonucleotide (data shown). simplest <br />                                                         interpretation data DnaD forms initial nucleoprotein complex, acts <br />                                                         nucleus recruit additional DnaD molecules eventually forming large <br />                                                         complex. Furthermore, interaction DnaD DNA salt resistant, stable <br />                                                         DnaD&#226;&#8364;&#8220;DNA complexes observed NaCl concentrations high 800 mM (data                                                         shown). feature observed nucleoid associated proteins                                                         H-NS protein E. coli [27]. <br />  <br />                                            3.3. AFM imaging DnaD-mediated nucleoprotein complexes <br />                                                         investigated nature nucleoprotein complexes AFM liquid air using <br />                                                         pBR322 plasmid. analyzed 149 images DnaD&#226;&#8364;&#8220;pBR322 complexes observed                                                         DnaD binds pBR322 opens supercoiled plasmid open circular <br />                                                         shape (Fig. 2B C). remarkable global structural effect pBR322 resembles                                                         similar effect observed HU binding supercoiled pUC19 reflects ability                                                         constrain DNA supercoils [28]. HU counteracts H-NS mediated global DNA compaction <br />                                                         effect E. coli. similar structural transition supercoiled relaxed state                                                         observed case replication initiator RepE54 protein encoded mini-F <br />                                                         plasmid case relaxation result DNA strand break local melting <br />                                                         double strand [29]. DnaD-induced relaxation result strand break <br />                                                         not known subject current investigations laboratory.                                                         remarkable features prominent DnaD&#226;&#8364;&#8220;pBR322 structures. images, <br />                                                         recorded air liquid, build DnaD inside circular plasmid                                                         foci pBR322 plasmid (Fig. 2C). Similar features observed Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         HU protein [28]. data suggest DnaD impose global DNA architectural <br />                                                         transitions, property biologically important primosomal assembly. <br />                                                         DnaD forming large scaffolds global  localized vicinity oriC) <br />                                                         remodeling bacterial nucleoid rendering replication proficient,                                                         time attaching bacterial chromosome membrane interaction                                                         membrane associated DnaB primosomal protein  . respect, DnaD&#226;&#8364;&#8220; <br />                                                         DnaB interaction functionally analogous RacA&#226;&#8364;&#8220;DivIVA interaction near                                                         oriC attaches chromosome membrane B. subtilis sporulation [30]. <br />  <br />                                            3.4. DnaD disrupts helicase&#226;&#8364;&#8220;DnaI complex vitro <br />                                                         precise role DnaD primosomal assembly unknown. appears act early                                                         primosomal assembly [15,16,31], interaction membrane associated <br />                                                         DnaB protein provide mechanism regulating initiation DNA replication                                                         B. subtilis [20]. previously reported stable interaction vitro                                                         putative B. subtilis helicase loader DnaI main replicative ring helicase DnaB B. <br />                                                         stearothermophilus [17]. order avoid confusion nomenclature, make <br />                                                         clear main replicative helicase B. subtilis known DnaC homologous <br />                                                         B. stearothermophilus DnaB (82% identity, 92% similarity). investigated effect                                                         DnaD DnaB&#226;&#8364;&#8220;DnaI complex vitro DnaD disrupts complex (Fig. <br />                                                         3A). Incubating DnaD, DnaB, DnaI proteins disrupted DnaB&#226;&#8364;&#8220;DnaI <br />                                                         interaction whilst, complex detected DnaB&#226;&#8364;&#8220;DnaD (Fig. 3B)  <br />  <br />                                                              FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                                  Page 6 <br />  <br />  <br />                                                         DnaI&#226;&#8364;&#8220;DnaD (data shown). biological significance observation                                                         relevant process helicase loading. interaction DnaI&#226;&#8364;&#8220;DnaC complex                                                         DnaD facilitate DnaC release loading DNA. <br />  <br />                                            4. Model <br />                                                         provided evidence DnaD forms large nucleoprotein assemblies affecting <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         global DNA architecture disrupts complex ring helicase                                                         helicase loader. propose DnaD link nucleoid reorganization                                                         initiation DNA replication B. subtilis (Fig. 3C). suggested previously                                                         initiation DNA replication regulated recruitment DnaD membrane [20]. <br />                                                         Maintaining DnaD cytoplasm prevents excessive initiation regulated <br />                                                         recruitment membrane DnaB (referring primosomal protein B. subtilis <br />                                                         replicative helicase B. stearothermophilus) activates initiation DNA <br />                                                         replication [20]. DnaD bind chromosomal DNA forming scaffold open                                                         nucleoid  locally vicinity oriC globally) render replication <br />                                                         proficient. DnaC recruited DnaI&#226;&#8364;&#8220;DnaC complex cytoplasm                                                         membrane attachment site direct interaction DnaB [16-18]. DnaD disrupt <br />                                                         DnaI&#226;&#8364;&#8220;DnaC complex facilitating loading oriC. DnaD provides                                                         possible link nucleoid reorganization initiation DNA replication.                                                         interaction membrane associated DnaB prime regulatory event                                                         couples chromosomal remodeling DNA replication. <br />  <br />   <br />  <br />                                                         Fig. 1. <br />                                                           Analysis DnaD oligomers sedimentation velocity ultracentrifugation. Data                                                         analyzed c(s) method Schuck [23] errors plotted 95% confidence <br />                                                         limit. low 1.2 &#206;&#188;M (blue lines), species corresponding monomer DnaD. <br />                                                         2.4 &#206;&#188;M (black) 4.9 &#206;&#188;M (red), species correspond dimers trimers                                                         higher order aggregates pentamers hexamers apparent. (B) AFM images <br />                                                         air DnaD increasing concentrations. arrow indicates increasing concentrations <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         (0.49, 4.9 49 nM). formation `chains&apos; individual DnaD molecules seen <br />                                                         clearly higher oligomers evident concentration increases. Scale bars represent 100 <br />                                                         nm. <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                           Page 9 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Fig. 2. <br />                                                           Gel shift analysis DnaD binding oligonucleotide increasing concentrations. <br />                                                         Binding reactions carried 2.5 nM oligonucleotide substrate 123, 246, 369, <br />                                                         492, 615 738 nM DnaD, lanes 1&#226;&#8364;&#8220;6, respectively. Lane C shows control radioactively <br />                                                         labeled oligonucleotide. (B) Control AFM images naked pBR322 plasmid air.                                                         typical field view representative individual pBR322 molecules adopting variety <br />                                                         compact supercoiled structures shown. scale bars field view individual <br />                                                         images represent 250 50 nm, respectively. (C) AFM topography images DnaD&#226;&#8364;&#8220; <br />                                                         pBR322 complexes. typical field view air representative individual <br />                                                         complexes air liquid shown. complexes DNA open circular <br />                                                         conformation periphery circular structures DnaD molecules forming <br />                                                         large scaffolds inside circles. images, DnaD appears form large foci                                                         pBR322 plasmid periphery circles. scale bars field views <br />                                                         individual images represent 1 &#206;&#188;m 100 nm, respectively. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                         Page 10 <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         Fig. 3. <br />                                                           Gel filtration analysis DnaB/DnaI/DnaD mixture showing complexes                                                         detected proteins incubated  (B) Gel filtration analysis DnaB/ <br />                                                         DnaI DnaB/DnaD mixtures showing formation DnaB&#226;&#8364;&#8220;DnaI complex                                                         stable complex DnaB DnaD. similar manner DnaI did form stable <br />                                                         complex DnaD (data shown). cases, samples peaks analyzed Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                         SDS&#226;&#8364;&#8220;PAGE numbers peaks correspond numbers lanes. Lanes M, <br />                                                         4, 5, 6 indicate molecular weight markers controls DnaB, DnaI DnaD proteins, <br />                                                         respectively. panels B, DnaB refers replicative helicase B. <br />                                                         stearothermophilus. (C) Speculative model coordination nucleoid remodeling                                                         initiation DNA replication B. subtilis. Indicated stoichiometries complexes                                                         membrane attachment site precise established. sizes                                                         various components depicted drawn size. DnaD depicted remodel <br />                                                         nucleoid, localized remodeling vicinity oriC                                                         case. DnaB refers primosomal protein B. subtilis. <br />  <br />  <br />  <br />  <br />                                                              FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />                                          Turner et al.                                                                                                                           Page 11 <br />  <br />  <br />  <br />  <br />                                                                                                           TABLE 1 <br />                                         Table possible frictional coefficients versus species type DnaD <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                          Species        2s       4s       6s        9s <br />  <br />                                          Monomer        1.488 <br />  <br />                                          Dimer          2.363    1.182 <br />  <br />                                          Trimer         3.059    1.549    1.032 <br />  <br />                                          Tetramer       3.750    1.875    1.250 <br />  <br />                                          Pentamer       4.352    2.176    1.451 <br />  <br />                                          Hexamer        4.914    2.457    1.638     1.092 <br />  <br />                                          Heptamer       5.446    2.723    1.815     1.210 <br />  <br />                                          Octamer        5.953    2.977    1.984     1.323 <br />  <br />                                          Nonamer        6.440    3.220    2.147     1.431 <br />  <br />                                          Decamer        6.908    3.454    2.3027    1.535 <br />  <br />                                         Physically unreasonable values grey. Frictional coefficients calculated using program SEDNTERP [24]. Higher values indicate                                         elongated molecule. <br /> Europe PMC Funders Author Manuscripts <br />  <br />  <br />  <br />  <br />                                                                       FEBS Lett. Author manuscript; available PMC 2011 February 04. <br />   <br /> </body></html>