<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>MOLECULAR CELLULAR BIOLOGY, Apr. 1989, p. 1659-1666                                                                      Vol. 9,  4 <br /> 0270-7306/89/041659-08$02.00/0 <br /> Copyright &#194;&#169; 1989, American Society Microbiology <br />  <br />  <br />           Molecular Analysis GPHI, Gene Encoding Glycogen <br />                  Phosphorylase Saccharomyces cerevisiae <br />                   PETER K. HWANG,* STUART TUGENDREICH, ROBERT J. FLETTERICK <br />          Department Biochemistry Biophysics, University California, San Francisco, California 94143-0448 <br />                                         Received 19 November 1988/Accepted 13 January 1989 <br />  <br />               yeast cells, activity glycogen phosphorylase regulated cyclic <span id='am-12' about='obo:IMR_0200083' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-13' property="oboInOwl:hasExactSynonym" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-14' property="rdfs:label" content="Adenosine 5&apos;-monophosphate" datatype="xsd:string"></span><span id='am-15' property="rdfs:label" content="AMP" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:id" content="IMR:0200083" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-18' property="oboInOwl:hasDbXref" content="KEGG:C00020" datatype="xsd:string"></span>AMP</span>-mediated phosphorylation <br />            enzyme. previously cloned gene glycogen phosphorylase (GPH1) Saccharomyces <br />            cerevisiae. assess role glycogen phosphorylase-catalyzed glycogenolysis yeast life cycle, <br />            yeast strains lacking functional GPHI gene containing multiple copies gene constructed. GPHI <br />            essential gene yeast cells. Haploid cells disrupted GPHI lacked phosphorylase <br />            activity attained higher levels intracellular glycogen similar wild-type cells. <br />            Diploid cells homozygous disruption able sporulate rise viable ascospores. Absence <br />            functional GPHI did impair cells synthesizing storing trehalose. Increases phosphorylase <br />            activity 10- 40-fold detected cells carrying multiple copies GPHI-containing 2,um plasmid. <br />            Northern (RNA) analysis indicated GPHI transcription induced late exponential growth phase, <br />            simultaneous onset intracellular glycogen accumulation.  low level glycogen            exponential cells primarily maintained regulating phosphorylation state constitutive <br />            phosphorylase. GPHI did appear formal glucose repression, transcriptional <br />            induction occurred advance glucose depletion medium. <br />  <br />    Energy required cell activity growth maintained              evolutionary relationships different enzymes (23, regulated synthesis breakdown storage                    33a). begun study biochemical carbohydrates. Glycogen phosphorylase, catalyzes                     physiological role glycogen phosphorylase S. cerevi- release glucose 1-phosphate glycogen, plays                  siae, using powerful tools genetic manipulations vital role process cells organisms.                available organism (45). <br /> regulation phosphorylase best understood vertebrate                  Yeast cells contain principal storage carbohydrates, <br /> tissues, enzyme activity modulated                 glycogen trehalose (15). carbohydrates appear changing levels effector ligands hormonal signals (20).             provide energy cells periods nonproliferation. <br /> Vertebrate phosphorylases convert inactive                     central question study yeast storage metabolism <br /> active forms reversible covalent modification                    organism utilizes different reserve carbohy- characterized regulatory kinase cascade (28). Enzyme                  drates. case cells undergoing sporulation activity regulated tissue-specific expression              respiratory adaptation does evidence exist distinct roles different isozymic forms. different isozyme              (27, 31). related issue glycogen trehalose <br /> forms identified mammals (muscle, liver, brain,               pathways directly coupled (17, 34, 38). Previous studies fetal forms, named tissues pre-                 shown levels glycogen trehalose yeast <br /> dominate) (19, 43, 54). isozymes display different                   cells change dramatically response variety different <br /> allosteric properties, reflecting evolution functions          physiological situations (29, 31, 48). enzymes involved <br /> suit specific environments physiological roles.                 synthesizing breaking carbohydrates case muscle phosphorylase rabbit skeletal tissue, <br /> regulatory mechanisms extensively character-                     regulated <span id='am-73' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-74' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-75' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-76' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-79' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-80' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-83' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span>cyclic AMP</span> (<span id='am-87' about='obo:IMR_0100024' typeof='obo:IMR_0100022 owl:Thing obo:IMR_0001355 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-88' property="oboInOwl:id" content="IMR:0100024" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="cAMP" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:hasAlternativeId" content="IMR:0200099" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasExactSynonym" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-92' property="rdfs:label" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-93' property="rdfs:label" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-94' property="rdfs:label" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="Adenosine 3&apos;,5&apos;-phosphate" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:hasDbXref" content="KEGG:C00575" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasExactSynonym" content="Cyclic adenylic acid" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasExactSynonym" content="3&apos;,5&apos;-Cyclic AMP" datatype="xsd:string"></span><span id='am-99' property="oboInOwl:hasExactSynonym" content="Cyclic AMP" datatype="xsd:string"></span><span id='am-100' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>cAMP</span>)-dependent kinase <br /> ized, biochemically crystallographicaily (12, 13, 25).                 yeasts (22, 36, 49, 50, 52, 53). Glycogen    Unlike bacterial plant phosphorylases,                    trehalose levels, fact, useful phenotypic indi- <br /> unregulated enzymes (8, 33, 35), yeast phosphorylase               cators studying various mutants RAS, adenylate <br /> regulated allosteric mechanisms. enzyme                     cyclase, cAMP kinase genes (16, 46, 47).  <br /> subject secondary-site allosteric control (glucose 6-phos-              complexity regulatory path- <br /> phate inhibition) converted active inac-                way controlling carbohydrates, difficult tive forms reversible phosphorylation (14). important                use abnormal <span id='am-5' about='obo:IMR_0001657' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-6' property="rdfs:label" content="carbohydrate" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="MeSH:D002241" datatype="xsd:string"></span><span id='am-8' property="oboInOwl:id" content="IMR:0001657" datatype="xsd:string"></span><span id='am-9' property="obo:IAO_0000115" content="The largest class of organic compounds, including STARCH; GLYCOGEN; CELLULOSE; POLYSACCHARIDES; and simple MONOSACCHARIDES. Carbohydrates are composed of carbon, hydrogen, and oxygen in a ratio of Cn(H2O)n." datatype="xsd:string"></span><span id='am-10' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasDbXref" content="CHEBI:23008" datatype="xsd:string"></span>carbohydrate</span> levels basis isolating <br /> distinction yeast phosphorylase phosphorylation             mutants specific defects glycogen trehalose  <br /> site lacks sequence similarity mammalian enzyme                 tabolism. <br /> counterpart (30). previously cloned phospho-                      cloning yeast GPHJ permits address rylase gene yeast Saccharomyces cerevisiae  refer                issues using combined molecular genetic gene yeast GPHI) derived primary struc-                  approach. present preliminary account ture enzyme (23). data, recently                  phenotypic consequences certain genetic manipulations determined primary structures bacterial, plant,                    GPHI. disrupted GPHJ gene yeast cells, <br /> isozymic variants mammalian phosphorylases, pro-                   generating haploid cells disrupted GPHJ diploid <br /> vided foundation comparing structure-function                    cells homozygous disruption, introduced <br />                                                                            cells increased gene dosage GPHI 2,um <br />                                                                            plasmids. examined expression level   * <br />       Corresponding author.                                                GPHJ transcripts growth rich medium. <br />                                                                     1659 <br />  1660     HWANG ET AL.                                                                                                       MOL. CELL. BIOL. <br />  <br />                            Plasmids <br />                                                B            R       X       S             RS        X          R       B <br />                           YCp5O:GPH1                 %11                                  11                  <br />                                                                                     GPH <br />  <br />                                                             R    X          S              S <br />                                                    5&apos; Probe                                                         Probe --                    -         Coding   Region <br />                                                                                                    ~~~~~~Probe <br />                            YCpX:GPH1                                x                     RS           x <br />                                                                                            iI <br />                            YEp24:GPH1 <br />  <br />                                                                     X       S        R         (S,X)       x <br />  <br />                          YCpX:(gph 1A::LEU2) <br />  <br />  <br />                                                             R       x           s     R        (SX) X  R <br />                                                     .....                                        I1 <br />                                                                                                     ..1W <br />  <br />  <br />                                                                                                                    1   kb <br />  <br />   FIG. 1. Structure wild-type GPHI, disrupted GPH1, plasmid constructions. YCp5O:GPHl contains 15-kb genomic fragment <br /> including entire GPHI gene. coding region GPHI previously identified (23). Fragments designated 5&apos; probe coding region <br /> probe subcloned M13 vectors used hybridization probes. Plasmids YCpX:GPHI, YEp24:GPHl, YCpX:(gphlA::LEU2) constructed described Materials Methods. Arrow indicates coding region GPHI; open bar indicates yeast LEU2 gene. <br /> Restriction enzyme sites: BamHI (B); EcoRI (R); Sall (S); Xhol (X). <br />  <br />  <br />                MATERIALS METHODS                                    inserting XhoI GPHJ fragment Sall site                                                                         YEp24. <br />    Strains media. S. cerevisiae HR125-5d (MA Ta/MATct,                 Preparation yeast crude extract. Yeast cells grown homozygous his3 his4 leu2-3 leu2-112 ura3-52 trpl-289),             liquid cultures harvested centrifugation 3,000 x g <br /> provided R. Jensen, used RNA DNA blotting                5 min 5&#194;&#176;C. cell pellets suspended cold gene disruption experiments. Escherichia coli MH1 <br />                                                                         water, repelleted centrifugation, frozen (araD139 lacX74 galU galK hsr rpsL) MH6 (lacX74 <br /> galU galK hsr rpsL leuB600 pyrF: :Tn5), M. Hall (18),              -70QC storage. Thawed pellets suspended equal used grow various plasmid constructions. <br />                                                                         volumes homogenization buffer (0.1 M Tris [pH 7.9], 10 <br />    YM-P medium (31) contained 6.3 g yeast nitrogen base              mM magnesium chloride, 2 mM calcium chloride, 1 mM amino acids (Difco Laboratories, Detroit, Mich.)            EDTA, 10% glycerol, 1 mM dithiothreitol). Homogenization <br /> liter, 4.5 g yeast extract (Difco) liter, 9 g peptone         carried vortexing samples equal volume <br /> (Difco) liter, 9 g <span id='am-101' about='obo:IMR_0200119' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-102' property="rdfs:label" content="Succinic acid" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:id" content="IMR:0200119" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-105' property="oboInOwl:hasDbXref" content="KEGG:C00042" datatype="xsd:string"></span>succinic acid</span> liter, 5.4 g NaOH        glass beads (0.5-mm diameter; Biospec Products, Bartles- liter, 2% glucose.                                              ville, Okla.) 30-s pulses. homogenized mixture <br />    Genetic methods. Standard methods yeast genetics             centrifuged 8,000 x g 10 min. portion used tetrad analysis scoring various markers (44).           supernatant extract diluted 1:200 measurement Yeast cells transformed method Ito et al. (24).          A280. Typical values optical density range 0.4 <br /> Gene disruption carried homologous recombina-                0.8. <br /> tion cloned DNA fragment genomic DNA,                        Phosphorylase assay. procedure used assay phos- <br /> using step gene replacement technique (42).                     phorylase activity yeast crude extract based    Plasmids. GPHI originally isolated 15-kilobase              method Carney et al. (7) determines (kb) genomic DNA insert BamHI site YCp5O.                 phosphate released formation glycogen glu- plasmid, designated YCp5O:GPH1, used                    cose 1-phosphate. Enzyme reactions carried 100 <br /> striction mapping genomic fragment (Fig. 1).                 mM sodium succinate (pH 5.8)-1% glycogen (prepared 3.7-kb XhoI restriction fragment GPHJ subcloned               oyster; Sigma <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>Chemical</span> , St. Louis, Mo.)-150 mM glu- <br /> digesting YCp5O:GPHJ XhoI religating                  cose 1-phosphate-1 mM dithiothreitol. Typically, yeast ex- <br /> digested mixture form plasmid YCpX:GPHI, liga-                 tracts diluted 1/100 reaction mixtures. tion GPHJ fragment XhoI fragment consisting             check Pi generated assay portion YCp50 (ampr ori) joined short region               sources yeast extract, additional assays yeast genomic DNA. gene disruption, 1.9-kb Sall                 carried absence added glycogen glucose <br /> restriction fragment GPHJ coding region                  1-phosphate. Portions reaction mixture mixed removed YCpX:GPHJ replaced 2.1-kb Sall-                   phosphate-reacting colorimetric reagent (7) extracted <br /> XhoI restriction fragment yeast LEU2 gene                equal volume butanol, measured YCpX:(gphlA::LEU2). YEp24:GPHI constructed                       A310- Phosphate concentration determined linear <br />  VOL. 9, 1989                                                           GLYCOGEN PHOSPHORYLASE S. CEREVISIAE                         1661 <br />  <br />                                                    B                                       C <br />                              1     2     3     4       1     2     3     4                       .1    2       3 <br />  <br />                                                                                                                         XIII, XVI <br />  <br />  <br />                                                                               -4.6 <br />                                                                               -3.9 <br />                                                                                                                         III <br />  <br />  <br />  <br />          DNA probes:          GPH1 5&apos; region GPH1 coding region                             GPH1           LEU2 <br />    FIG. 2. Disruption chromosomal localization GPH1 gene.  B) Total cellular DNA prepared haploid yeast strains <br /> segregated tetrad derived transformation HR125-5d gphlA::LEU2 DNA fragment. 5 Fg DNA sample digested EcoRI, electrophoresed 1.0% agarose gel, transferred nitrocellulose filter. Lanes DNA Leu- (1 3) Leu+ (2 4) segregants.   Filter hybridized (high stringency) 32P-labeled 1.7-kb EcoRI-XhoI DNA fragment 5&apos; flanking GPHI; (B) hybridized (low stringency) 1.9-kb Sail fragment GPHI coding region. panel  shift predominant signal 4.6-kb band (lanes 1 3) 3.9-kb band (lanes 2 4) predicted gphlA::LEU2 disruptants. Weaker hybridization bands <br /> indicate homologies yeast genome 5&apos;-flanking region GPHI. panel B, lack low-stringency hybridization Leu+ <br /> segregants (lanes 2 4) confirms deletion GPHI coding region absence multiple phosphorylase genes. (C) <br /> Chromosomes Leu- (wild-type GPHI) segregant (lanes 1 2) Leu+ segregant (lane&apos;3) separated OFAGE blotted nylon membrane filter. Lanes: 1, hybridized radiolabeled 3.7-kb XhoI fragment GPHI; 2 3, hybridized 2.1-kb <br /> SaII-XhoI fragment yeast LEU2 gene. <br />  <br /> interpolation spectrophotometrically measured phosphate              Pa.). Medium glucose levels determined using standards. unit phosphorylase activity represents            glucose oxidase kit Sigma. <br /> release 1 pLmol phosphate glucose 1-phosphate              Determination glycogen trehalose content. Qualita- <br /> min 30&#194;&#176;C. determination phosphorylase specific                tive glycogen determinations adding 5 10 ml <br /> activity, total protein concentration estimated              0.4% KI-0.2% iodine solution plates. Cells method Bradford (4), using bovine serum albumin                    accumulate glycogen stain brown, standard curve.                                                         remain yellow. Quantitative determinations glycogen    DNA RNA hybridization analysis. Total yeast DNA                  trehalose carried methods described Lillie <br /> prepared saturated cultures (11) digested                 Pringle (31). <br /> EcoRI. DNA electrophoresed 1.0% agarose gel blotted nitrocellulose membrane filter. DNA                                            RESULTS <br /> probes used hybridization gel purified 32p <br /> labeled (109 cpm/nug DNA) random primer method                   Construction null GPHI allele. determine (21). Hybridization conditions 650C, 6x SSPE (20x                  GPHI essential haploid cell growth, gphlA::LEU2 <br /> SSPE 3.6 M NaCl, 200 mM NaH2PO4, 20 mM EDTA                      allele constructed method Rothstein (42). [pH 7.4]), 5 mM TES [N-tris(hydroxymethyl)methyl-2-ami-                 cloned DNA fragment containing GPHI gene, 1.9-kb <br /> noethanesulfonic acid; Sigma], Sx Denhardt solution, 10 ,ug             Sail restriction fragment coding region  salmon sperm DNA ml, 0.1% sodium dodecyl                     leted replaced insertion LEU2 gene fragment <br /> sulfate. Washes carried nonstringently 2x SSPE              (Fig. 1). resulting plasmid, YCpX:(gphlA::LEU2), stringently 0.1 x SSPE presence 0.1% sodium             cut XhoI, gel-purified 3.8-kb gphlA::LEU2 <br /> dodecyl sulfate 65&#194;&#176;C.                                                DNA fragment used transform diploid HR125-5d <br />    chromosomal analysis GPHI, yeast chromosomes                  leucine prototrophy. Leu+ diploids sporu- separated orthogonal-field alternating-gel electro-             lated, tetrads analyzed. 13 16 tetrads (5 5, <br /> phoresis (OFAGE) 1.5% agarose gel (6). gel                 4 5, 4 6), spores viable, 2:2 <br /> blotted nylon membrane filter UV irradiated (9).             segregation Leu+:Leu-, grew similar rates. filter hybridized 32P-labeled DNA probes                     DNA prepared spore colonies derived described                                                         single tetrad, Southern hybridization analysis used <br />    Total cellular RNA isolated described Jensen et            confirm GPHJ replaced disrupted gene al. (26). RNA electrophoresed 1.2% agarose gel                 Leu+haploids  Fig. 2A B). addition, <br /> containing <span id='am-68' about='obo:IMR_0200017' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-69' property="oboInOwl:hasDbXref" content="KEGG:C00067" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="Formaldehyde" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:id" content="IMR:0200017" datatype="xsd:string"></span>formaldehyde</span>, transferred nylon membrane                chromosomes Leu- Leu+ haploids separated <br /> filter UV fixation (9), hybridized stringent             OFAGE (6), blotted filters, hybridized GPHI <br /> conditions (420C, 50% formamide, Sx SSPE, 5x Denhardt                   LEU2 probes (Fig. 2C). preliminary assignment solution, 0.1% sodium dodecyl sulfate) 32P-labeled               GPHI chromosome XIII XVI detection 1.9-kb Sall fragment GPHJ.                                           GPHI hybridization band 10 OFAGE separation <br />    Cellular medium parameters. growth curve mon-                (6). LEU2 probe hybridized band itoring, liquid growth carried 500-ml baffled                Leu+ cells, providing additional confirmation correct gene <br /> Erlenmeyer flasks (type 2543; Bellco Glass, , Vineland,             disruption. absence low-stringency hybridization N.J.), containing 100 ml medium, incubated             Leu+ disruptants 1.9-kb DNA encoding catalytic <br /> rotary shaker 200 rpm 30&#194;&#176;C. Cell density moni-               portions yeast phosphorylase (Fig. 2) indicated hap- <br /> tored primarily A6. calibration optical density           loid cells contained copy phosphorylase gene. cell number determined. Initial determinations cell            GPHI gphlA::LEU2 segregants number percentage budded cells using                equally viable, conclude GPHI essential cell-counting chamber (Hausser Scientific, Philadelphia,              vegetative growth. <br />  1662    HWANG ET AL.                                                                                              MOL. CELL. B3IOL. <br />  <br />    test GPHJ required growth sporula- <br /> tion diploid cells, formed homozygous gphlA::LEU2 <br /> diploid mating gph1A&amp;:LEU2 haploid strains opposite <br /> mating types. Diploid colonies identified basis inability mate MATa MATa tester strains. <br /> Homozygous gphlA::LEU2 diploids viable exhib- <br /> ited apparently normal morphology. replica plated acetate medium, gphlA::LEU2 diploid cells able sporulate produce viable gphlA::LEU2 haploid spores. <br />    Phenotype yeast cells containing gphlA::LEU2. glycogen content spore clones single tetrads               f: <br />                                                                             }: <br /> examined qualitatively iodine staining. growth YEPD plates (44) 5 7 days, resulted prolonged <br /> incubation cells stationary phase, spore colonies began differences staining intensity glycogen (Fig. 3) depended GPHI gene disrupted. <br /> Darker lighter iodine staining tended develop expected 2:2 ratio tetrad spores. Higher intracellular <br /> glycogen content, indicative reduced glycogen <br /> breakdown, segregated disruption GPHI leucine <br /> prototrophy. <br />    direct comparison GPHI function gphlA:: <br /> LEU2 GPHI cells obtained assays phos- <br /> phorylase activity homogenized crude extracts yeast <br /> cultures grown stationary phase (Table 1). assay <br /> procedure monitors release Pi phosphorylase cata-                              B <br /> lyzes incorporation glucose 1-phosphate glyco- <br /> gen. Leu+ segregants released significantly phos- <br /> phate did Leu- cells, indicated absence                        2                  c             _ <br /> phosphorylase activity. phosphate released gphiA                            3 <br />                                                                                              = <br />                                                                                                            _ <br />  ::LEU2 cells (PH3-6b PH3-6d) likely resulted phosphate-generating activity cell extract.  measure contribution obtained assay reac- <br /> tions omitting phosphorylase substrates glucose 1-phosphate glycogen (Table 1, columns). results  assays glycogen, noted glycogen diluted cell extract promote significant                                    * Leu <br />  degree phosphorylase-catalyzed phosphate release.                                            E Leu- <br />     Experiments carried determine pheno- <br />  typic differences Gph+ (GPHI) Gph- (gphiA::             FIG. 3. Glycogen staining haploid progeny derived  LEU2) cells. Wild-type mutant strains grown        GPHI-disrupted diploid cells.   Staining transformed HR125-5d <br />  identical conditions monitored various medium         cells. Cells sporulated, tetrads dissected micro- <br />  cellular parameters. YM-P (2% glucose) used grow-         manipulation. Segregants tetrads germinated  ing cultures, yeast growth carbohydrate accumu-         YEPD medium (44) 30&#194;&#176;C 2 days streaked  lation previously characterized medium (31).     YEPD plate shown, grown 5 days, stained solution  Overall, Gph+ Gph- strains exhibited similar growth       0.4% KI-0.2% 12. (B) Diagrammatic representation plate showing <br />  profiles  Fig. 4A). Doubling times exponential         grouping spores derived individual.asci segre- <br />  growth identical. consumption medium glucose          gation pattern leucine prototrophy. tetrad 8,  followed similar time courses. Adaptation fermentative       spores  GPHI segregant) viable. <br />  respirative growth entry stationary phase  appeared similar Gph+ Gph- strains. Cell              exhaustion medium. basal level  morphologies, examined microscopically, similar           glycogen exponential Gph- cells appeared  <br />  growth curve; percentage budded cells       higher Gph+ cells, general similarity  dropped manner strains reached station-       glycogen profiles presence absence phosphory- <br />  ary phase.                                                        lase indicated phosphorylase principal <br />     Intracellular glycogen trehalose contents        regulator glycogen content proliferative growth. <br />  determined growth Gph+ Gph- cells (Fig.         stationary phase, Gph- cells contained significantly <br />  4). wild-type yeast cells, levels carbohydrates   glycogen did Gph+ cells, understandably  generally low increase significantly cells approach        absence phosphorylase-catalyzed degradation  growth-arresting conditions (31). (Measurements glyco-         Gph- cells. suggested glycogen  gen trehalose vary widely depending medium         used respirative adaptation diauxic growth (31). <br />  strain used, method normalization,          indication studies  factors imprecision intrinsic assay procedures.)        adaptation consequent entry stationary phase  Gph- Gph+ cells exhibited wild-type pattern       affected loss phosphorylase function. <br />  carbohydrate accumulation. particular, glycogen accumu-           Comparison trehalose levels Gph+ Gph- <br />  lation strains began similar moments, near end     cells enabled determine absence phospho- <br />  exponential growth, advance glucose             rylase-catalyzed glycogenolysis affects pathway  VOL. 9, 1989                                                                    GLYCOGEN PHOSPHORYLASE S. CEREVISIAE                                  1663 <br />  <br />                        TABLE 1. Phosphorylase activity spore clones derived GPHI-disrupted transformanta <br />                                                                                               Phosphate release (nmollmin mg total protein)&apos; <br />            Strain              MAT            ~~~Leucine <br />                                             prototrophy <br />                                                                   Glycogenb <br />                                                                  accumulation         Complete reaction            added             added <br />                                                                                         Complterection          glucose 1-phosphate             glycogen <br /> PH3-6a                                          -                   -                      28.0                        6.7                      12.2 <br /> PH3-6b                                          +                   +                       6.4                        4.1                       9.3 <br /> PH3-6c                                          -                   -                      19.0                        3.0                      11.7 <br /> PH3-6d                                          +                   +                       9.3                        2.8                       9.1 <br /> PH3-6b/GPHI-2 .md                               +                   -                     573.0 <br /> PH3-6d/GPHI-2 .md                               +                   -                     373.0 <br />   Strains PH3-6a, PH3-6b, PH3-6c, PH3-6d represent single tetrad derived sporulation GPHI-disrupted HR125-5d cells. Cell cultures grown early stationary phase liquid medium preparation crude cell extract enzyme assay. <br />   b Detected <br />               qualitatively iodine staining. <br />     Equivalent milliunits phosphorylase activity milligram protein. <br />   d GPHI present multicopy plasmid YEp24. <br />  <br />  <br /> trehalose synthesis. Gph+ Gph- cells contained similar                       differences phosphorylase activity glycogen  <br /> amounts trehalose exponential growth                            tent, transformants appeared physiologically morpho- <br /> stationary phase. mid- late log growth, strains                   logically similar untransformed strains. particular, <br /> exhibited slightly decreasing levels trehalose. end                    differences growth profile accumulation exponential growth, Gph+ Gph- cells showed                              pattern glycogen detected. Furthermore, similar pattern trehalose accumulation.                           difference transformed wild-type strains observed wild-type yeast strains (31), trehalose                        cell viability 2-week incubation stationary phase. <br /> synthesis Gph+ Gph- strains lagged                                Growth cycle dependence GPHI mRNA expression. Total <br /> initiation glycogen accumulation continued cells                    RNA prepared wild-type cells harvested reached stationary phase. absence functional                          time points growth YM-P medium. Various medium <br /> GPHJ did impair capacity cells synthesize                      cellular parameters monitored harvesting <br /> store trehalose, did alter temporal relationship                   points (Fig. 5B). band RNA corresponding 3.3-kb <br /> trehalose glycogen accumulation.                                             RNA species detected radiolabeled GPHI probe    glycogen yeast cells normally degraded                     samples late exponential stationary cells. GPHJ <br /> stationary phase, expected GPHI gphlA::LEU2                          transcripts observed initial growing cells, cells exhibit differences cell viability incuba-                  indication GPHI gene repressed exponential <br /> tion stationary phase.  observed differ-                        growth. Interestingly, highest level GPHI transcript <br /> ence GPHI gphlA::LEU2 cultures incubated                           detected cells proliferating exponentially exhausted YM-P medium 4 weeks.                           medium glucose slightly declining. time point <br /> yeast strain showed detectable loss viability, possible                 corresponded onset glycogen accumulation (Fig. longer incubation required difference                    4). GPHI gene did appear strict glucose <br /> detected. wild-type yeast strains, net degradation                         repression, peak levels transcripts occurred glycogen trehalose undetectable 2 weeks;                       advance glucose exhaustion medium. Glycogen <br /> furthermore, cell viability fails decline 3 months                  accumulation previously shown occur incubation room temperature (31).                                             depletion glucose medium (31). coincidence    attempt discover condition GPHI                           initial appearance GPHI transcripts glycogen <br /> confers advantage yeast cells challenging                      accumulation suggests coordinated regulation glycogen <br /> gphlA::LEU2 GPHI cells grow 36&#194;&#176;C. Gph+                             phosphorylase glycogen synthase genes. <br /> Gph- cells plated YEPD medium grown                                noted detected GPHI transcripts 36&#194;&#176;C.   difference cell viability                         normalized mRNA constitutively expressed <br /> growth rate GPHI gphlA::LEU2                               gene, commonly Northern (RNA) analysis <br /> yeast strains.                                                                   profiles. study yeast cells <br />    Increased gene dosage GPHI integration                        generate quite different mRNA profiles logarithmic <br /> plasmid YEp24. phenotypic consequence                               stationary phases (3). loading close equal <br /> creased dosage GPHI determined transfecting                            amounts total RNA possible gel lanes, effect <br /> yeast cells multicopy plasmid containing GPHI                         achieved normalization rRNA species predomi- <br /> gene. YEp24:GPHI, constructed described Materials                          nate total RNA preparations. Methods, consisted 3.7-kb GPHI fragment inserted                           extension experiment described  GPHI 2,um plasmid YEp24 (URA3). YEp24:GPHI                               transcripts monitored yeast cells stationary- <br /> used transform PH3-6b PH3-6d strains. Ura+ trans-                         log-phase adaptation. Cells initially grown satu- <br /> formants strain contain levels                      ration YM-P incubated 6 days exhausted <br /> phosphorylase activity 10- 40-fold higher wild-type                      medium, pelleted suspended levels (Table 1). Using strains disabled genomic copy                     fresh medium. GPHI mRNA gradually diminished unde- GPHJ assured phosphorylase overproduction                             tectable levels, transcripts present cells <br /> involved transcription plasmid-derived GPHI gene                        doublings preexponential phase (data gene fragment cloned plasmid did                       shown). <br /> lack cis-acting element required GPHJ <br /> induction. Transformed cells contain                                                   DISCUSSION <br /> glycogen wild-type cells, determined iodine                              Yeast phosphorylase activated allosterically staining glycogen assay (data shown). Apart                         reversible phosphorylation specific threonine residue <br />  1664                 HWANG ET AL.                                                                                                                                     MOL. CELL. BIOL. <br />  <br />         2- <br />                                                                                                                                                                               Hrs.                                                                                                                          10   12 14.5 16     18.5   21       23   35       45   growth <br />                                                                                                                                                                                 GPH mRNA <br />                                                                                                    -1o8        3.2 kb-              w   t <br />                                                                                                                                         .w&apos; <br />                                                                                                                                               _          w        w        _ <br />  <br />  <br />                                                                                                           El <br />     C.)                                                                                                                                                                    B <br />                                                                     11 <br />                                                                                                           b <br />                                                                                                           U) <br />                                                                                                         Cu <br />                                          r)L <br />                                          Ili   .................    -s-9-98|    -o, <br />                                                                        .......... <br />  <br />  <br />                                                                                           , <br />                                                                                               L1   007             1 <br />                 10     15           20           25      30    35       40          45   50 <br />                                                        Time (hrs)                                                                                                                                                                                                                                                                                                                      <br />                                                                                                                    0 <br />                                                                                                                                                                                           -7c) <br />                                                                                                                                                                                                                                                                                -1&quot;                                                                                                  rA&gt; <br />                                                                                                                                                                                           -o <br />                                                                                                                    .%)                                                                    L-- <br />                                                                                                                                                                                           C) <br />                                                                                                                                                                                           Q <br />           42- <br />  <br />  <br />                                                                                                                                             25     30        35       40 <br />                                                                                                                                             Time (hrs) <br />           0-                                                                                                      FIG. 5. Time course GPHI mRNA accumulation yeast <br />                                                                                                                cells approach stationary phase rich medium.   <br />                                                                                                                Time course Northern blotting analysis GPHI mRNA levels. <br />                                                                                                                Total cellular RNA isolated yeast cells growing YM-P <br />                                                                                                                medium containing 2% glucose. Conditions              10     15           20            25     30    35       40          45   50 <br />                                                                                                                experiment shown Fig. 4. 20-S&#194;&#177;g RNA prepa- <br />                                                                                                                ration electrophoresed 1.2% agarose gel containing form- <br />                                                        Time (hrs)                                              aldehyde hybridized 32P-labeled GPHI Sall fragment.                                                                                                              (B) Accumulation GPHI mRNA yeast cells relation                                                                                                                glucose concentration rate cell proliferation                                                                                                                approach stationary phase. dashed line indicates relative <br />                                                                                                                cellular levels GPHI transcript determined densitometer <br />                                                                                                                scanning Northern blot. <br />  <br />           0 <br />     -52 -                                                                                                      logical evidence indicated phosphorylase production <br />                                                                                                                Saccharomyces carlsbergensis occurs specifically                                                                                                                late exponential growth phase (1). observed                                                                                                                intracellular levels phosphorylase highest yeast cells <br />      I1-                                                                                                       grown stationary phase (unpublished findings). <br />                                                                                                                   objectives understand physiological <br />                10     15        20               25     30     35      40           45   50 <br />                                                                                                                role allosteriQ regulatory mechanisms yeast phos- <br />                                    Time (hrs)                                                                  phorylase. findings refute model glycogen metabo- <br />       FIG. 4. Growth profiles  , glycogen accumulation (B),                                                                                                                lism constitutive level phosphorylase partici- <br />     trehalose accumulation (C) Gph+ Gph- yeast cells approach-                                          pates regulating glycogen content exponential <br />     ing stationary phase rich medium. YM-P medium containing 2%                                           growth phosphorylation-dephosphorylation inter- <br />     glucose inoculated  t = 0 h) exponentially growing cells                                       conversion mediated cAMP regulatory cascade. Al- <br />     YM-P precultures initial cell densities 104/ml.                                        study needed define role allosteric <br />     cultures incubated 30&#194;&#176;C rotary shaker 200 rpm.                                        regulation yeast phosphorylase, current information     Sampling begun cell density reached approximately                                             carbohydrate metabolism cAMP regulation yeasts <br />     107/ml. Symbols: 0, O, Gph- cells (PH3-6b, MATa gphlA::LEU2                                              suggests reasonable model. phosphorylase glyco- <br />     strain derived HR125-5d); *, *, Gph+ cells (PH3-6a; MATa                                            gen accumulate end exponential growth,  <br />     GPHI derivative HR125-5d).                                                                              zyme likely exists unphosphorylated b form.                                                                                                                predominance phosphorylase b phosphorylase     inhibited allosterically glucose 6-phosphate                                                   consistent observed decreasing concentrations     binding. Phosphorylation catalyzed specific phospho-                                               cAMP decreasing activity cAMP-dependent kinase     rylase kinase additionally cAMP-depen-                                                    haploid yeast cells approach stationary conditions (17, 51). <br />     dent protein kinase (39, 53). unphosphorylated,                                                   active enzyme, phosphorylase b     active enzyme form, phosphorylase b, potently inhibited                                                 enzymatic activity (14) sufficient meet     glucose 6-phosphate, phosphorylated                                                      minimal energy demand stationary cells account     zymej phosphorylase  sensitive effector.                                              gradual diminution intracellular glycogen content (31). <br />     addition allosteric controls, yeast phosphorylase subject                                            Degradation glycogen stationary incubation     transcriptional regulation. study indicates GPHJ                                              effectively modulated intracellular levels glucose <br />     transcript levels low exponential growth                                                 6-phosphate,  noted, effective <br />     crease entry stationary phase. Previous immuno-                                                inhibitor phosphorylase b. Interestingly, level  VOL. 9, 1989                                                       GLYCOGEN PHOSPHORYLASE S. CEREVISIAE                             1665 <br />  <br /> glucose 6-phosphate proliferating yeast cells rises          functions absence GPHI, glycogen degradation does <br /> medium glucose exhausted drops lower         appear play important role supplying glucose level cells stationary phase (41).                     trehalose synthesis. observed trehalose <br />    stationary yeast cells switched rich growth         accumulation continues exhaustion medium <br /> conditions, glycogen levels rapidly reduced (2), sugges-       glucose, gluconeogenesis probable primary source tive profound activation phosphorylase activity,      glucose trehalose synthesis. Supporting view afforded phosphorylation.  covalent           observation nonproliferating yeast cells incu- <br /> modification phosphorylase involving cAMP-mediated          bated presence alanine, nonfermentable carbon <br /> activation accessory kinase likely occurs        source, trehalose concentration increases amino <br /> adaptation rich medium. conditions shown         acid rapidly depleted (36), indication gluconeogen- promote increases cAMP levels cAMP-dependent             esis provides precursors trehalose synthesis. <br /> kinase activity yeasts (17, 32, 40, 51). glycogen            likely explanation apparent dispensabil- <br /> depletion, subsequent exponential growth obviates met-         ity phosphorylase growth conditions used <br /> abolic requirement maintain phosphorylase cell.          study, utilization trehalose energy source <br />    adaptive value maintaining different reserve         sufficiently compensated Gph- defect. carbohydrates yeast cells remains unclear  Glycogen       examine matter, like determine stationary trehalose display nonidentical patterns accumulation        conditions promote metabolism utilization varying nutritional limitations (31),        carbohydrate  alternate approach indicates reserves play distinct roles     dissecting metabolic roles glycogen trehalose, cell life cycle. assess role glycogen            possible isolate cells survive phosphorylase-catalyzed glycogenolysis yeast life           functional copy GPHI gene. glycogen cycle, constructed strains lacking functional            trehalose appear serve overlapping purposes, mutant <br /> phosphorylase gene containing multiple copies            search approach identify defective genes specific <br /> gene. GPHI does appear essential gene. Haploid        trehalose pathway. hope identify cells disrupted GPHI contained higher levels intracellu-     gene genes yeast glycogen synthase, enzyme lar glycogen appeared similar haploid Gph+        catalyzes glycogen formation, possible homology cells. Increasing gene dosage GPHI presenting      recently determined DNA protein sequences human YEp24-based plasmid resulted expected increase          glycogen synthase (5). Genetic control synthetic <br /> phosphorylase activity unaccompanied ad-            degradative pathways glycogen metabolism pro- <br /> verse phenotype. high-copy construction enables         vide ultimate tools defining role glycogen purify phosphorylase saturated yeast cultures       yeast organism. biochemical structural study (V. L. Rath, P. K. <br /> Hwang, R. J. Fletterick, manuscript preparation).                                  ACKNOWLEDGMENTS <br />    phosphorylase principal enzyme glycogenoly-         indebted Michael Hall Susan Lillie technical <br /> sis, apparent dispensability GPHJ suggests <span id='am-56' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-57' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-58' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-60' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-62' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-65' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="glycine" datatype="xsd:string"></span>gly</span>-        advice discussions Ira Herskowitz use cogen degradation yeasts essential func-          laboratory facilities. <br /> tion. Phosphorolysis,  mode               work supported National Science Foundation grant <br /> glycogen breakdown yeasts. Cells undergoing sporulation         DMB84-20489. <br /> exhibit period glycogen degradation final <br /> maturation ascospores (27) result activity                          LITERATURE CITED <br /> sporulation-specific glucoamylase encoded SGA gene            1. Becker, J. U. 1982. Mechanisms regulation glycogen <br /> (10, 55); glycogen phosphorylase, fact, undetectable             phosphorylase activity Saccharomyces carlsbergensis. J. <br /> yeast cells sporulation (10). Interestingly, recent            Gen. Microbiol. 128:447-454. <br /> studies SGA gene cloned disrupted               2.   Becker, J. U., H. J. Vohmann, C. Eilers-Konig. 1979. <br /> indicate gene required successful sporu-             Glycogen metabolism resting growing cells Saccharo- <br />                                                                           myces carlsbergensis. Arch. Microbiol. 123:143-149. <br /> lation (55). result provides supporting indication       3.   Boucherie, H. 1985. Protein synthesis transition glycogen degradation essential.                                stationary phases glucose limitation Saccharomyces <br />    Given elaborate allosteric properties yeast phospho-            cerevisiae. J. Bacteriol. 161:385-392. <br /> rylase growth-dependent regulation GPHI tran-             4.   Bradford, M. M. 1976. rapid sensitive method scription, difficult accept dispensability GPHI               quantitation microgram quantities protein utilizing function. coincidence GPHI induction glycogen                  principle protein-dye binding. Anal. Biochem. 72:248-254. <br /> accumulation end logarithmic growth supports             5.   Browner, M. F., K. Nakano,  G. Bang, R. J. Fletterick. <br /> primary physiological role phosphorylase utilizing                 1989. Human muscle glycogen synthase cDNA sequence: glycogen reserve energy source periods                     negatively charged protein asymmetric charge distribu- <br />                                                                           tion. Proc. Natl. Acad. Sci. USA 86:1443-1447. <br /> nutrient starvation. high degree structural func-         6.   Carle, G. F., M. V. Olson. 1985. electrophoretic <br /> tional conservation yeast mammalian phospho-                  karyotype yeast. Proc. Natl. Acad. Sci. USA 82:3756-3760. <br /> rylases argues yeast enzyme, like mam-          7.   Carney,  T., R. J. Beynon, J. Kay, N. Birket. 1978. malian counterparts, serves essential function                 semicontinuous assay glycogen phosphorylase. Anal. Bio- <br /> particular distress. possible enzyme confers            chem. 85:321-324. <br /> yeast cells adaptive growth advantages               8.   Chen, G. S.,  H. Segal. 1968. Purification properties apparent homogeneous cultures growth                      glycogen phosphorylase Escherichia coli. Arch. Biochem. <br /> conditions used laboratory.                                        Biophys. 127:175-186. <br />    reports suggested yeasts, trehalose          9.   Church, G. M., W. Gilbert. 1984. Genomic sequencing. <br />                                                                           Proc. Natl. Acad. Sci. USA 81:1991-1995. <br /> forms expense glycogen (17, 34). evident       10.   Colonna, W. J., P. T. Magee. 1978. Glycogenolytic enzymes study disruption GPHJ does impair                     sporulating yeast. J. Bacteriol. 95:1750-1757. <br /> capacity yeast cells accumulate high level trehalose.    11.   Davis, R. N., M. Thomas, J. Cameron, T. P. St. John, S. Scherer,  unless alternative pathway glycogenolysis                R.  Padgett. 1980. Rapid DNA isolation enzymatic  1666       HWANG ET AL.                                                                                                  MOL. CELL. BIOL. <br />  <br />      hybridization analysis. Methods Enzymol. 65:404 411.               34. Oura, S., E. Oura, H. Suomalainen. 1975. formation <br /> 12. Fletterick, R. J., N. B. Madsen. 1980. structures            glycogen trehalose baker&apos;s yeast. Eur. J. Appl. <br />      related functions phosphorylase  Annu. Rev. Biochem.               Microbiol. 2:29-37. <br />      40:31-61.                                                          35. Palm, D., R. Goerl, K. J. Burger. 1985. Evolution 13. Fletterick, R. J., S. R. Sprang. 1982. Glycogen phosphory-           catalytic regulatory sites phosphorylases. Nature <br />      lase structure function. Acc. Chem. Res. 15:361-369.                (London) 313:500-502. <br /> 14. Fosset, M., L. W. Muir, L. D. Nielsen, E. H. Fischer. 1971.     36. Panek,  C., P. S. Araujo, V. Moura Neto,  D. Panek. <br />      Purification properties yeast glycogen phosphorylase           1987. Regulation trehalose-P-synthase complex Sac- <br />      b. Biochemistry 10:4105-4113.                                       charomyces. Curr. Genet. 11:459466. <br /> 15. Fraenkel, D. 1982. Carbohydrate metabolism, p. 1-37. J. N.       37. Panek,  D. 1975. Trehalose synthesis starvation      Strathern, E. W. Jones, J. R. Broach (ed.), molecular           baker&apos;s yeast. Eur. J. Appl. Microbiol. 2:3946. <br />      biology yeast Saccharomyces: metabolism gene            38. Panek,  D.,  L. Sampaiao, G. C. Braz, S. J. Baker, J. R. <br />      expression. Cold Spring Harbor Laboratory, Cold Spring Har-             Mattoon. 1980. Genetic metabolic control trehalose      bor, N.Y.                                                               glycogen synthesis. New relationships carbohydrate <br /> 16. Fraenkel, D. G. 1985. RAS gene function yeast. Proc.               reserves, catabolite repression maltose utilization. Cell. <br />      Natl. Acad. Sci. USA 82:47404744.                                       Mol. Biol. 25:334-354. <br /> 17. Francois, J., P. Eraso, C. Gancedo. 1987. Changes        39. Pohlig, G., R. Wingender-Drissen, J. U. Becker. 1983. <br />      concentration cAMP, fructose 2,6-bisphosphate related            Characterization phosphorylase kinase activities yeast. <br />      metabolites enzymes Saccharomyces cerevisiae              Biochem. Biophys. Res. Commun. 114:331-338. <br />      growth glucose. Eur. J. Biochem. 164:369-373.                   40. Purwin, C., F. Leidig, H. Holzer. 1982. Cyclic AMP- <br /> 18. Hall, M., L. Hereford,  Herskowitz. 1984. Targeting E.          dependent phosphorylation fructose-1,6-bisphosphatase      coli P-galactosidase nucleus yeast. Cell 46:1057-1065.        yeast. Biochem. Biophys. Res. Commun. 107:1482-1489. <br /> 19. Henion, W. F., E. W. Sutherland. 1957. Immunological            41. Quain, D. E., J. M. Haslam. 1979. Changes glucose <br />      differences phosphorylases. J. Biol. Chem. 224:477-488.              6-phosphate storage carbohydrates catabolite dere- <br /> 20.  H. G. 1976. control glycogen metabolism              pression Saccharomyces cerevisiae. J. Gen. Microbiol. 113: <br />      liver. Annu. Rev. Biochem. 45:167-189.                                  195-198. <br /> 21. Hodgson, C. P., R.  Fisk. 1987. Hybridization probe size      42. Rothstein, R. J. 1983. step gene disruption yeast. Meth- <br />      control: optimized oligolabelling. Nucleic Acids Res. 15:6295.          ods Enzymol. 101:202-211. <br /> 22. Huang, K. P., E. Cabib. 1974. Yeast glycogen synthetase      43. Schane, H. P. 1965. Molecular weight estimation rat uterine <br />      glucose 6-phosphate-dependent form.  Purification             phosphorylase. Anal. Biochem. 11:371-394. <br />      properties. J. Biol. Chem. 249:3851-3857.                          44. Sherman, F., G. R. Fink, J. B. Hicks. 1983. Methods 23. Hwang, P. K., R. J. Fletterick. 1986. Convergent                 yeast genetics. Cold Spring Harbor Laboratory, Cold Spring <br />      divergent evolution regulatory sites eukaryotic phospho-           Harbor, N.Y. <br />      rylases. Nature (London) 324:80-84.                                45. Struhl, K. 1983. new yeast genetics. Nature (London) 305: <br />                                                                               391-397. <br /> 24. Ito, H., Y. Fukuda, K. Murata,  Kimura. 1983. Transfor-       46. Tatchell, K., L. C. Robinson, M. Breitenbach. 1985. RAS2      mation intact yeast cells treated alkali cations. J.             Saccharomyces cerevisiae required gluconeogenic growth <br />       Bacteriol. 153:163-168.                                                 proper response nutrient limitation. Proc. Natl. Acad. <br /> 25. Jenkins, J. , L. N. Johnson, D.  Stuart, E.  Stura, K. S.           Sci. USA 82:3785-3789. <br />      Wilson, G. Zanotti. 1981. Phosphorylase: control activ-     47. Toda, T.,  Uno, T. Ishikawa, S. Powers, T. Kataoka, D. Broek, <br />       ity. Phil. Trans. R. Soc. London Ser. B 293:2341.                       S. Cameron, J. Broach, K. Matsumoto, M. Wigler. 1985. 26. Jensen, R., G. F. Sprague,  Herskowitz. 1983. Regulation            yeast, RAS proteins controlling elements adenylate <br />       yeast mating-type interconversion: feedback control HO            cyclase. Cell 40:27-36. <br />       gene expression yeast mating type locus. Proc. Natl.        48. Trevelyan, W. E., J. S. Harrison. 1956. Studies yeast <br />       Acad. Sci. USA 80:3035-3039.                                            metabolism. 7. Yeast carbohydrate fractions. Separation  27. Kane, S. M., R. Roth. 1974. Carbohydrate metabolism                  nucleic acid, analysis, behavior anaerobic fermenta- <br />       ascospore development yeast. J. Bacteriol. 118:8-14.          tion. Biochem. J. 63:23-33. <br />  28. Krebs, E. G., J.  Beavo. 1979. Phosphorylation-dephos-        49. Uno, , K. Matsumoto, K. Adachi, T. Ishikawa. 1983. <br />       phorylation enzymes. Annu. Rev. Biochem. 48:923-959.                 Genetic biochemical evidence trehalase substrate <br />  29. Kuenzi, M. T.,  Fiechter. 1972. Regulation carbohy-             cAMP-dependent protein kinase yeast. J. Biol. Chem. <br />       drate composition Saccharomyces cerevisiae growth              258:10867-10872. <br />       limitation. Arch. Mikrobiol. 84:254-265.                           50. Van der Plaat, J. B., P. Van Solingen. 1974. Cyclic 3&apos;,5&apos;- <br />  30. Lerch, K., E. H. Fischer. 1975. <span id='am-19' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-20' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-22' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-23' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-24' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span>Amino acid</span> sequence           <span id='am-26' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-27' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-28' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-30' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-32' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-36' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-38' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span>adenosine</span> monophosphate stimulates trehalose degradation       functional sites yeast glyc <br /> </body></html>