<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                                                                                                           ORIGINAL RESEARCH ARTICLE <br />                                                                                                                                 published: 25 2012 <br />                                                                                                                          doi: 10.3389/fmicb.2012.00183 <br />  <br />  <br />  <br /> Genomic biochemical analysis diaminopimelate lysine biosynthesis pathway Verrucomicrobium <br /> spinosum: identi&#239;&#172;?cation partial characterization L,L-diaminopimelate aminotransferase <span id='am-157' about='obo:IMR_0200206' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-158' property="oboInOwl:id" content="IMR:0200206" datatype="xsd:string"></span><span id='am-159' property="rdfs:label" content="UDP" datatype="xsd:string"></span><span id='am-160' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-161' property="oboInOwl:hasDbXref" content="KEGG:C00015" datatype="xsd:string"></span>UDP</span>-N-acetylmuramoylalanyl-D-glutamyl-2,6-meso- <br /> diaminopimelate ligase <br /> Victoria R. Nachar 1 , Francisco C. Savka 1 , Sean E. McGroty 1 , Katherine  Donovan 2 , Rachel  North 2 , <br /> Renwick C. J. Dobson 2,3 , Larry J. Buckley 1 Andr&#195;&#169; O. Hudson 1 * <br /> 1 <br />   Thomas H. Gosnell School Life Sciences, Rochester Institute Technology, Rochester, NY, USA <br /> 2 <br />   Biomolecular Interaction Centre, School Biological Sciences, University Canterbury, Christchurch, New Zealand <br /> 3 <br />   Department Biochemistry Molecular Biology, Bio21 Molecular Science Biotechnology Institute, University Melbourne, Parkville, VIC, Australia <br />  <br />  <br />  <br /> Edited                                 Gram-negative bacterium Verrucomicrobium spinosum attracted recent <br /> Naomi L. Ward, University                                           years following sequencing annotation genome. Comparative genomic <br /> Wyoming, USA <br />                                           analysis V. spinosum using diaminopimelate/lysine metabolic genes Chlamydia tra- <br /> Reviewed  <br /> Uli Stingl, King Abdullah University   chomatis suggests V. spinosum employs L,L-diaminopimelate aminotransferase <br /> Science Technology, Saudi Arabia      (DapL) pathway diaminopimelate/lysine biosynthesis. open reading frame corre- <br /> Evelyn Toh, Indiana University            sponding putative dapL ortholog cloned recombinant enzyme Bloomington, USA <br />                                           shown possess L,L-diaminopimelate aminotransferase activity vitro. vivo analysis <br /> Yufeng Wang, University Texas San Antonio, USA                          using functional complementation con&#239;&#172;?rmed dapL ortholog able func- <br /> *Correspondence:                          tionally complement E. coli mutant confers auxotrophy diaminopimelate Andr&#195;&#169; O. Hudson, Thomas H.            lysine. addition role lysine biosynthesis, intermediate diaminopimelate Gosnell School Life Sciences,          integral role <span id='am-99' about='obo:IMR_0001699' typeof='owl:Thing obo:IMR_0001790 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0000001 obo:IMR_0001700'><span id='am-100' property="obo:IAO_0000115" content="Peptidoglycan (PGN) is a unique and essential component of the cell wall of virtually all bacteria, is not present in eukaryotes, and is an excellent target for the innate immune system." datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasDbXref" content="MeSH:D010457" datatype="xsd:string"></span><span id='am-102' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:hasExactSynonym" content="PGN" datatype="xsd:string"></span><span id='am-104' property="oboInOwl:id" content="IMR:0001699" datatype="xsd:string"></span><span id='am-105' property="rdfs:label" content="murein" datatype="xsd:string"></span><span id='am-106' property="oboInOwl:hasDbXref" content="KEGG:C00889" datatype="xsd:string"></span><span id='am-107' property="rdfs:label" content="peptidoglycan" datatype="xsd:string"></span><span id='am-108' property="oboInOwl:hasExactSynonym" content="murein" datatype="xsd:string"></span><span id='am-109' property="rdfs:label" content="PGN" datatype="xsd:string"></span>peptidoglycan</span> biosynthesis. end, UDP-N-acetylmuramoylalanyl- <br /> Rochester Institute Technology, 85 <br />                                           D-glutamyl-2,6-meso-diaminopimelate ligase ortholog identi&#239;&#172;?ed, cloned, Lomb Memorial Drive, Rochester NY <br /> 14623-5603, USA.                          shown possess meso-diaminopimelate ligase activity vivo. L,L-diaminopimelate <br /> e-mail: aohsbi@rit.edu                    aminotransferase pathway experimentally con&#239;&#172;?rmed bacteria,                                           deemed pathogenic animals. animals, particularly humans, lack <br />                                           genetic machinery synthesis diaminopimelate/lysine novo, enzymes <br />                                           involved pathway attractive targets development antibiotics. dapL <br />                                           essential gene bacteria currently known. V. spinosum excellent candi- <br />                                           <span id='am-10' about='oboInOwl:date' typeof='owl:Thing'>date</span> investigate essentiality dapL, bacterium employs DapL pathway <br />                                           lysine cell wall biosynthesis, non-pathogenic humans, facile grow,                                           genetically manipulated. <br />                                           Keywords: lysine, diaminopimelate, L,L-diaminopimelate aminotransferase, peptidoglycan, functional complemen- <br />                                           tation <br />  <br />  <br /> INTRODUCTION                                                                      diaminopimelate/lysine novo bacterium pro- <br /> Verrucomicrobium spinosum rod-shaped heterotrophic Gram-                     totrophic lysine. Lysine synthesized using general <br /> negative bacterium fresh water soil envi-                    pathways evolutionarily divergent respect ronments characterized non-motile contains                    intermediates used. pathway utilizes intermediate &#206;&#177;- <br /> appendages known prosthecae. V. spinosum important                       amino adipic acid, derived metabolism biotechnology medical sectors closely related                    2-ketoglutarate product <span id='am-22' about='obo:IMR_0200473' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-23' property="rdfs:label" content="Citric acid" datatype="xsd:string"></span><span id='am-24' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200473" datatype="xsd:string"></span><span id='am-26' property="oboInOwl:hasDbXref" content="KEGG:C00158" datatype="xsd:string"></span>citric acid</span> cycle. &#206;&#177;-amino Chlamydia (Wagner Horn, 2006). recent study                          adipic acid pathway primarily used fungi narrowly <br /> &#239;&#172;?rmed organism contains genetic components                    distributed species belonging domain archaea <br /> encode Type III Secretion able kill Drosophila                (Nishida et al., 1999; Velasco et al., 2002). pathway <br /> melanogaster Caenorhabditis elegans, model invertebrate                   employs intermediate diaminopimelate, derived hosts (Sait et al., 2011).                                                        oxaloacetate. diaminopimelate pathway bac- <br />    Genomic analysis V. spinousm organ-                           teria photosynthetic organisms including cyanobacteria, algae, <br /> ism contains genes necessary synthesis                         plants. <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                           2012 | Volume 3 | Article 183 | 1 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />     variants diaminopimelate/lysine pathway                   L,L-diaminopimelate tetrahydrodipicolinate transam- discovered far (Figure 1): acyl pathways,                      ination reaction using glutamate amino donor use succinyl acetyl intermediates; meso-                   tetrahydrodipicolinate amino <span id='am-147' about='obo:IMR_0200224' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-148' property="oboInOwl:id" content="IMR:0200224" datatype="xsd:string"></span><span id='am-149' property="oboInOwl:hasDbXref" content="KEGG:C00028" datatype="xsd:string"></span><span id='am-150' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-151' property="rdfs:label" content="Acceptor" datatype="xsd:string"></span>acceptor</span> single reac- <br /> diaminopimelate dehydrogenase pathway; recently dis-                   tion bypassing 2,3,4,5-tetrahydropyridine-2,6-dicarboxylate <br /> covered L,L-diaminopimelate aminotransferasae (DapL) pathway                   -acyl-transferase, acyl-diaminopimelate aminotransferase, acyl- <br /> (Hudson et al., 2006, 2008; McCoy et al., 2006). biosyn-                   diaminopimelate deacylase steps present succinylated thesis lysine aspartate diaminopimelate/lysine                 acetylated diaminopimelate/lysine pathways (Table 1, Figure 1). <br /> pathway divided main events. &#239;&#172;?rst event                  ultimate event diaminopimelate variant path- synthesis tetrahydrodipicolinate aspartate.             ways de&#239;&#172;?ned enzyme meso-diaminopimelate decarboxy- general feature diaminopimelate pathway variants                 lase catalyzes decarboxylation meso-diaminopimelate carried enzymes, aspartate kinase, aspartate                 synthesize lysine. decarboxylation step shared semialdehyde dehydrogenase, dihydrodipicolinate synthase,                  variants (Table 1, Figure 1). <br /> dihydrodipicolinate reductase, respectively (Table 1). sec-                    animals contain machinery synthesize <br /> ond event constitutes conversion tetrahydrodipicolinate                 diaminopimelate lysine, used bacteria pep- penultimate intermediate meso-diaminopimelate. syn-                 tidoglycan biosynthesis, enzymes involved bacterial <br /> thetic steps tetrahydrodipicolinate meso-diaminopimelate               diaminopimelate/lysine biosynthesis pathways de&#239;&#172;?ne uniqueness diaminopimelate variant pathways.                  scienti&#239;&#172;?c community. meso-Diaminopimelate serves acyl pathways, enzymes required. reactions               cross linking amino acids cell wall Gram- carried enzymes 2,3,4,5-tetrahydropyridine-2,6-                 negative bacteria, lysine role Gram- <br /> dicarboxylate N -acyl-transferase, acyl-diaminopimelate amino-                 positive bacteria (Hutton et al., 2007). Inhibition enzymes <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  diaminopimelate/lysine biosynthesis pathways late epimerase, respectively (Table 1, Figure 1). dehy-                 detrimental effect bacterial growth perspectives: <br /> drogenase pathway, tetrahydrodipicolinate converted meso-                diaminopimelate necessary cell wall synthesis, cell lysis <br /> diaminopimelate enzyme meso-diaminopimelate dehydro-                    occur result osmotic pressure lack pep- <br /> genase step circumventing 2,3,4,5-tetrahydropyridine-               tidoglycan (Cox, 1996; Baizman et al., 2000); addition, 2,6-dicarboxylate-acyl-transferase, acyl-diaminopimelate amino-                inhibition enzymes pathway prevent protein syn- <br /> transferase, acyl-diaminopimelate deacylase, diaminopime-                  thesis, lysine 20 common proteogenic amino <br /> late epimerase enzymatic reactions (Table 1, Figure 1). L,L-               acids. meso-Diaminopimelate incorporated cross <br /> diaminopimelate aminotransferase (DapL) pathway synthesizes                    linking amino acids cell wall Gram-negative bacteria <br />  <br />  <br />  <br />  <br />   FIGURE 1 | Diaminopimelate/lysine biosynthesis pathways.            pathways, (B) meso-diaminopimelate dehydrogenase pathway, (C)   variants diaminopimelate/lysine biosynthetic pathways starting   L,L-diaminopimelate aminotransferase (DapL) pathway. abbreviations   tetrahydrodipicolinate denoted   succinylated acetylated     enzymes enzyme reaction de&#239;&#172;?nitions listed Table 1. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                        2012 | Volume 3 | Article 183 | 2 <br />  Nachar et al.                                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 1 | List diaminopimelate/lysine biosynthesis genes V. spinosum DSM 4136T . <br />  <br /> Locus tag                   Gene symbol          Gene                         EC #             Reaction <span id='am-42' about='obo:IAO_0000115' typeof='owl:Thing'>definition</span> <br />  <br /> VspiD_010100025585          lysC                 Aspartate kinase                  2.7.2.4          <span id='am-58' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-59' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-62' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="ATP" datatype="xsd:string"></span><span id='am-64' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-65' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-67' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-68' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>ATP</span>+aspartate &#226;&#8224;&#8221; <span id='am-11' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-12' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span><span id='am-13' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-14' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-16' property="rdfs:label" content="ADP" datatype="xsd:string"></span>ADP</span>+4-phospho-aspartate <br /> VspiD_010100030480          asd                  Aspartate semialdehyde            1.2.1.11         aspartate 4-semialdehyde + <span id='am-152' about='obo:IMR_0200102' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-153' property="oboInOwl:id" content="IMR:0200102" datatype="xsd:string"></span><span id='am-154' property="rdfs:label" content="Orthophosphate" datatype="xsd:string"></span><span id='am-155' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-156' property="oboInOwl:hasDbXref" content="KEGG:C00009" datatype="xsd:string"></span>orthophosphate</span> + NADP+ &#226;&#8224;&#8221; 4- <br />                                                  dehydrogenase                                      phospho-L-aspartate + <span id='am-48' about='obo:IMR_0200463' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-49' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="KEGG:C00005" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:id" content="IMR:0200463" datatype="xsd:string"></span><span id='am-52' property="rdfs:label" content="NADPH" datatype="xsd:string"></span>NADPH</span> + H+ <br /> VspiD_010100011370          dapA                 Dihydrodipicolinate               4.2.1.52         Aspartate 4-semialdehyde + pyruvate &#226;&#8224;&#8221; L-2,3- <br />                                                  synthase                                           dihydrodipicolinate + 2H2 O <br /> VspiD_010100011365          dapB                 Dihydrodipicolinate               1.3.1.26         2,3,4,5-Tetrahydrodipicolinate + NADP + &#226;&#8224;&#8221; L-2,3- <br />                                                  reductase                                          dihydrodipicolinate + NADPH + H+ <br /> -                           dapD                 2,3,4,5-tetrahydropyridine-       2.3.1.117        <span id='am-43' about='obo:IMR_0200065' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-44' property="rdfs:label" content="Acyl-CoA" datatype="xsd:string"></span><span id='am-45' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-46' property="oboInOwl:hasDbXref" content="KEGG:C00040" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:id" content="IMR:0200065" datatype="xsd:string"></span>Acyl-CoA</span> + 2,3,4,5-tetrahydrodipicolinate + H2 O &#226;&#8224;&#8221; <span id='am-74' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-75' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="CoA" datatype="xsd:string"></span><span id='am-77' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>CoA</span> + N- <br />                                                  2,6-dicarboxylate                                  acyl-2-L-amino-6-oxoheptanedioate <br />                                                  Nacyl-transferase <br /> -                           dapC                 Acyl-diaminopimelate              2.6.1.17         N-acyl-L,L-2,6-diaminoheptanedioate + 2-oxoglutarate&#226;&#8224;&#8221;N- <br />                                                  aminotransferase                                   acyl-2-L-amino-6-oxoheptanedioate + <span id='am-110' about='obo:IMR_0100119' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-111' property="rdfs:label" content="L-Glutamate" datatype="xsd:string"></span><span id='am-112' property="rdfs:label" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-113' property="oboInOwl:hasExactSynonym" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-114' property="oboInOwl:hasAlternativeId" content="IMR:0200156" datatype="xsd:string"></span><span id='am-115' property="oboInOwl:hasExactSynonym" content="L-Glutamic acid" datatype="xsd:string"></span><span id='am-116' property="oboInOwl:id" content="IMR:0100119" datatype="xsd:string"></span><span id='am-117' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-118' property="oboInOwl:hasExactSynonym" content="L-Glutamate" datatype="xsd:string"></span><span id='am-119' property="rdfs:label" content="glutamic acid" datatype="xsd:string"></span><span id='am-120' property="rdfs:label" content="L-Glutaminic acid" datatype="xsd:string"></span><span id='am-121' property="oboInOwl:hasDbXref" content="KEGG:C00025" datatype="xsd:string"></span>L-glutamate</span> <br /> -                           dapE                 Acyl-diaminopimelate              3.5.1.18         N-acyl-L,L-2,6-diaminoheptanedioate + H2 O &#226;&#8224;&#8221; acyl + L,L- <br />                                                  deacylase                                          2,6-diaminoheptanedioate <br /> -                           ddh                  Diaminopimelate                   1.4.1.16         meso-2,6-Diaminoheptanedioate + NADP+ + <span id='am-69' about='obo:IMR_0200357' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-70' property="oboInOwl:hasDbXref" content="KEGG:C00001" datatype="xsd:string"></span><span id='am-71' property="oboInOwl:id" content="IMR:0200357" datatype="xsd:string"></span><span id='am-72' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-73' property="rdfs:label" content="H2O" datatype="xsd:string"></span>H2O</span> &#226;&#8224;&#8221; L-2- <br />                                                  dehydrogenase                                      amino-6-oxoheptanedioate + <span id='am-37' about='obo:IMR_0200075' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-38' property="oboInOwl:hasDbXref" content="KEGG:C00014" datatype="xsd:string"></span><span id='am-39' property="rdfs:label" content="NH3" datatype="xsd:string"></span><span id='am-40' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:id" content="IMR:0200075" datatype="xsd:string"></span>NH3</span> + NADPH <br /> VspiD_010100012510          dapL                 L,L-diaminopimelate               2.6.1.83         L,L-2,6-diaminopimelate + 2-oxoglutarate &#226;&#8224;&#8221; 2,3,4,5- <br />                                                  aminotransferase                                   tetrahydrodipicolinate + L-glutamate + H2 O <br /> VspiD_010100011375          dapF                 Diaminopimelate                   5.1.1.7          L,L-2,6-diaminopimelate&#226;&#8224;&#8221;meso-2,6-diaminopimelate <br />                                                  epimerase <br /> VspiD_010100003650          lysA                 Diaminopimelate                   4.1.1.20         meso-2,6-Diaminopimelate&#226;&#8224;&#8217;lysine + <span id='am-27' about='obo:IMR_0200324' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-28' property="oboInOwl:id" content="IMR:0200324" datatype="xsd:string"></span><span id='am-29' property="oboInOwl:hasDbXref" content="KEGG:C00011" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-31' property="rdfs:label" content="CO2" datatype="xsd:string"></span>CO2</span> <br />                                                  decarboxylase <br />  list generated using genomic information deposited IMG database (http://img.jgi.doe.gov/cgi-bin/w/main.cgi). <br />  <br />  <br />  enzyme UDP-N -acetylmuramoylalanyl-d-glutamyl-2,6-                          diaminopimelate/lysine biosynthesis. addition, identify meso-diaminopimelate ligase (MurE) (E.C. 6.3.2.15),                       characterize MurE V. spinosum, suggesting bacterium <br /> encoded murE gene.                                                          incorporates meso-diaminopimelate peptidoglycan     DapL pathway identi&#239;&#172;?ed numerous bacteria                       cross linking <span id='am-92' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-93' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-94' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-95' property="rdfs:label" content="amino acid" datatype="xsd:string"></span><span id='am-96' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span>amino acid</span>. experimentally con&#239;&#172;?rmed limited number bac- <br /> teria, deemed pathogenic (McCoy et al., 2006;                    RESULTS <br /> Hudson et al., 2008; Liu et al., 2010). dimensional crys-                IDENTIFICATION PUTATIVE dapL murE GENES V. SPINOSUM <br /> tal structure DapL Arabidopsis thaliana solved                    dapL ortholog V. spinosum identi&#239;&#172;?ed using (Watanabe et al., 2007, 2008) Chlamydia trachomatis (Watan-                   DapL protein C. trachomatis Ct 390 (NP_219900) abe et al., 2011) algae Chlamydomonas reinhardtii                     query BlastP algorithm Integrated Microbial <br /> (Dobson et al., 2011; Hudson et al., 2011a). Inhibitors Ara-               Genomes (IMG) database1 . search identi&#239;&#172;?ed putative L,L- <br /> bidopsis DapL ortholog reported using vitro studies                   diaminopimelate aminotransferase annotated locus tag <br /> (Fan et al., 2010). noted inhibitors                    VspiD_010100012510 (ZP_02927470) 38% amino acid Arabidopsis DapL  known                     identity C. trachomatis DapL. murE ortholog iden- <br /> compounds inhibit aminotransferases.                                         ti&#239;&#172;?ed using C. trachomatis MurE (NP_219774) query.     identi&#239;&#172;?cation dapL murE genes, addition                       search resulted identi&#239;&#172;?cation murE ortholog V. easy culturability genetic manipulation V. spinosum,                    spinosum annotated locus tag VspiD_010100019130, makes bacterium excellent model organism experi-                        37% identical C. trachomatis MurE. <br /> ments addressing essentiality dapL murE respect lysine peptidoglycan biosynthesis (Domman et al., 2011).                    CRUDE SOLUBLE PROTEIN EXTRACT V. SPINOSUM CONTAINS bacterial species contain DapL pathway dif&#239;&#172;?-                     DapL ACTIVITY <br /> cult work  slow growing, dif&#239;&#172;?cult                 V. spinosum utilizes DapL pathway diaminopime- <br /> culture, anaerobic, pathogenic humans.                                      late/lysine biosynthesis, suspected genomic analysis,     present &#239;&#172;?rst genomic biochemical analysis                      crude soluble protein extract possess DapL activity. diaminopimelate/lysine biosynthesis pathway V. spin-                   ortho-aminobenzaldehyde assay employed using crude sol- <br /> osum. identify genes necessary diaminopimelate/lysine                   uble protein extract V. spinosum. Using L,L-diaminopimelate <br /> biosynthesis genome V. spinosum demonstrate bacterium uses recently discovered DapL pathway                        1 http://www.jgi.doe.gov/ <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            2012 | Volume 3 | Article 183 | 3 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  amino donor 2-ketoglutarate amino accep-                     consistent genomic analysis regarding absence tor, assay measures production dihydroquinazolium                   diaminopimelate dehydrogenase ortholog genome V. product interaction tetrahydrodipicol-             spinosum. <br /> inate ortho-aminobenzaldehyde. DapL activity detected crude soluble extract showing rate                    GENE ANNOTATED LOCUS TAG VspiD_010100012510 <br /> reaction proportional V. spinosum pro-                     (VsdapL) ENCODES AUTHENTIC L,L-DIAMINOPIMELATE <br /> tein extract added assay (Figure 2). Enzymatic activity                 AMINOTRANSFERASE observed amino donor, amino accep-                         putative VsdapL gene cloned recombinant <br /> tor, omitted assay. addition, activity                  enzyme puri&#239;&#172;?ed homogeneity using af&#239;&#172;?nity chromatogra- <br /> observed protein extract heated boiling water                  phy (Figure 3). ortho-aminobenzaldehyde assay used 5 min.                                                                     determine putative VsDapL L,L-diaminopimelate <br />                                                                                aminotransferase activity. results analysis DapL ROUTE DIAMINOPIMELATE/LYSINE                              open reading frame annotated locus tag encodes BIOSYNTHESIS V. SPINOSUM                                                    authentic L,L-diaminopimelate aminotransferase enzyme. genome V. spinosum searched catalog genes                    L,L-diaminopimelate amino donor 2-ketoglutarate necessary novo anabolism lysine aspartate.                  amino acceptor, speci&#239;&#172;?c activity VsDapL 4.1 &#194;&#177; 0.25 DapL activity detected soluble extract,                    &#206; 440 min&#226;&#710;&#8217;1 mg&#226;&#710;&#8217;1 . Unlike plant algae enzyme, VsDapL <br /> wanted know DapL pathway sole route                     possesses transamination activity diamine donors lysine biosynthesis. examples litera-                  structurally similar L,L-diaminopimelate, including ture multiple pathways lysine biosynthesis bacteria;                 racemic isomer meso-diaminopimelate (Table 2), albeit rela- example, genomes Bacteroides fragilis Clostridium               tively low rates. enzyme able use oxoacids <br /> thermocellum contain DapL meso-                     amino acceptors addition 2-ketoglutarate (Table 2); diaminopimelate dehydrogenase pathways (Hudson et al., 2011b).                 relative rates low.  Corynebacterium glutamicum employs acyl pathway conjunction Ddh pathway (Schrumpf et al., 1991).                    VsdapL ABLE FUNCTIONALLY COMPLEMENT E. COLI <br /> search V. spinosum genome suggests DapL path-                  &#206;&#8221;dapD/dapE (AOH1) MUTANT <br /> way route diaminopimelate/lysine biosynthesis,                 E. coli strain AOH1 harbors deletion dapD orthologs acyl dehydrogenase genes                  gene mutation dapE renders enzyme non- identi&#239;&#172;?ed genome (Table 1; Figure 1).                    functional (Hudson et al., 2006).  mutant unable search did identify diaminopimelate dehydrogenase                    synthesize meso-diaminopimelate lysine cell wall <br /> ortholog, tested dehydrogenase activity using                  biosynthesis. result, cells lyse osmotic pressure <br /> protein extract used detect DapL activity. test               lack meso-diaminopimelate cross linking using 1.0 mg total protein different buffer                 amino acid cell wall deemed auxotrophic systems, &#239;&#172;?nding activity. recombinant diaminopime-                      diaminopimelate lysine. complementation analysis, E. <br /> late dehydrogenase Clostridium thermocellum used                      coli AOH1 transformed vector (pBAD33), positive control (Hudson et al., 2011b). result                  DapL expression vectors (pBAD33 + dapL)                                                                                plant Arabidopsis thaliana (AtdapL; Hudson et al., 2006), <br />                                                                                alga C. reinhardtii (CrdapL; Dobson et al., 2011)                                                                                bacterium V. spinosum (VsdapL). E. coli AOH1 able <br />                                                                                grow media supplemented L,L-diaminopimelate, <br />                                                                                auxotrophic mutant expressing authentic orthologous <br />  <br />  <br />  <br />  <br />                                                                                 FIGURE 3 | Recombinant expression purification VsDapL using <br />  FIGURE 2 | DapL activity crude protein extract V. spinosum.          affinity chromatography. Lane (1) Protein Marker (kDa), Lane (2) 10 &#206;&#188;g  graph shows relationship protein reaction       soluble protein uninduced cells, Lane (3) 10 &#206;&#188;g soluble proteins <br />  rate. DapL ortho-aminobenzaldehyde assay described Section       induced cells, Lane (4) 1 &#206;&#188;g puri&#239;&#172;?ed VsDapL. proteins  &#226;&#8364;&#339;Materials Methods.&#226;&#8364;? assay triplicates protein    resolved 10% (w/v) acrylamide gel stained Coomassie <br />  concentration.                                                                 blue visualization. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                       2012 | Volume 3 | Article 183 | 4 <br />  Nachar et al.                                                                    Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> DapLs able grow L,L-diaminopimelate-free media                  VsmurE ABLE FUNCTIONALLY COMPLEMENT E. COLI murE <br /> (Figure 4). analysis demonstrates recombinant               MUTANT <br /> enzymes able convert tetrahydrodipicolinate L,L-                meso-Diaminopimelate serves penultimate pre- <br /> diaminopimelate, bypassing acyl enzymatic reactions present         cursor lysine biosynthesis, incorporated E. coli acyl pathway, facilitate synthesis meso-         cross linking amino acids cell wall Gram- <br /> diaminopimelate lysine cell wall biosynthesis (Figures 1          negative bacteria. facilitated enzyme UDP-N - 5).                                                                   acetylmuramoylalanyl-D-glutamyl-2,6-meso-diaminopimelate lig- <br />                                                                           ase (MurE; Figure 5). V. spinosum classi&#239;&#172;?ed Gram- <br /> VsDapL BELONGS DAPL1 FORM L,L-DIAMINOPIMELATE                   negative bacterium, possess MurE ortholog (Schlesner <br /> AMINOTRANSFERASES                                                         et al., 2006). VsmurE ortholog identi&#239;&#172;?ed open <br /> Diaminopimelate aminotransferases approximately 400 amino             reading frame cloned expression vector test acids length members <span id='am-5' about='obo:IMR_0200504' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-6' property="oboInOwl:id" content="IMR:0200504" datatype="xsd:string"></span><span id='am-7' property="oboInOwl:hasDbXref" content="KEGG:C00250" datatype="xsd:string"></span><span id='am-8' property="rdfs:label" content="Pyridoxal" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>pyridoxal</span>-5  phosphate             functional complementation gene encodes authen- <br /> dependent family class II transaminases. recent study              tic MurE enzyme. E. coli mutant TKL-11 harbors muta- <br /> revealed diverged forms DapL enzymes;               tion murE gene results temperature sensitive <br /> DapL1 DapL2 (Hudson et al., 2008). forms                   growth phenotype mutant able grow  <br /> DapLs share approximately 30% homology amino acid                  missive temperature 30&#203;&#353;C, 42&#203;&#353;C (Lugtenberg level. DapL ortholog C. trachomatis evolutionar-              van Schijndel-van Dam, 1972). mutant transformed ily related DapL1 enzymes (Hudson et al., 2008).             vector (pBAD33) vector expressing puta- <br /> homology V. spinosum DapL 38% C. tra-               tive MurE (pBAD33 + VsmurE). Using replica-plating, results <br /> chomatis ortholog, form V. spinosum DapL               analysis demonstrate permissive temperature <br /> apparent. Using phylogenetic analysis, DapL ortholog             30&#203;&#353;C, mutant harboring vector control vec- <br /> V. spinosum clusters enzymes                 tor expressing recombinant enzyme able grow.  <br /> experimentally con&#239;&#172;?rmed DapL1 orthologs patho-                exposed non-permissive temperature 42&#203;&#353;C, genic organisms (Hudson et al., 2008, 2011b). analysis shows          mutant expressing MurE ortholog able grow (Figure 7A). distinct clades, DapL1, DapL2. aspartate aminotrans-              result corroborated assessing bacterial growth ferase Yersinia pseudotuberculosis, classi&#239;&#172;?ed          period 10 h. 30&#203;&#353;C mutant harboring vector class II transaminase used outgroup (Figure 6).           murE expression vector grew expected.  analyses performed (ML, MP,  NJ)                cultures switched non-permissive temperature 42&#203;&#353;C <br /> moderate strong support DapL2 clade (57&#226;&#8364;&#8220;100%)                  5 h 30&#203;&#353;C, mutant harboring murE expres- DapL1 clade (56&#226;&#8364;&#8220;99%). variable maximum                   sion vector continued grow. optical density vector <br /> likelihood bootstrap support alignment dependent alter-           culture declined attributed rapid lysis native alignments techniques (Clustal vs. MUSCLE) exhibiting              cell lack proper peptidoglycan synthesis (Figure 7B). <br /> higher ML bootstrap values DapL1 DapL2 clades                 assessment crude soluble protein extracts com- <br /> respectively. Muscle-aligned sequences generally exhibit higher           plementation experiment using SDS-PAGE analysis con&#239;&#172;?rmed bootstrap support DapL1 DapL2 clades                    expression recombinant MurE (59 kDa) TKL-11 cells <br /> Clustal.                                                                  harboring expression vector (pBAD33 + VsmurE) grown  <br />  <br />  <br />  <br />   FIGURE 4 | Functional complementation E. coli &#206;&#8221;dapD/dapE (AOH1) mutant. AOH1 mutant harboring plasmids pBAD33, pBAD33 + AtdapL, <br />   pBAD33 + CrdapL, pBAD33 +VsdapL replica-plated LB medium supplemented 0.2% (w/v) arabinose 50 &#206;&#188;g/mL <br />   L,L-diaminopimelate. presentation purposes colony vector recombinant plasmids represented diagram. <br />  <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                2012 | Volume 3 | Article 183 | 5 <br />  Nachar et al.                                                                         Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 5 | Schematic representation bacterial peptidoglycan           muramoylalanine-D-glutamate ligase), MurE (UDP-N-acetylmuramoyl- <br />   synthesis using m-DAP cross linking amino acids.           L-alanyl-D-glutamate 2,6-meso-diaminopimelate ligase), MurF <br />   cytosolic, membrane, periplasmic steps shown. abbreviations   (UDP-N-acetylmuramoyl-tripeptide-D-alanyl-d-alanine ligase), Ddl <br />   enzymes follows: MurA (UDP-N-acetylglucosamine enolpyruvyl        (D-alanine-D-alanine ligase B), MraY (phospho-N-acetylmuramoyl- <br />   transferase), MurB (UDP-N-acetylenolpyruvoylglucosaminereductase), MurC      pentapeptide-transferase), MurG (pyrophosphoryl-undecaprenol- <br />   (UDP-N-acetylmuramate-<span id='am-53' about='obo:IMR_0200251' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-54' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasDbXref" content="KEGG:C00041" datatype="xsd:string"></span><span id='am-56' property="oboInOwl:id" content="IMR:0200251" datatype="xsd:string"></span><span id='am-57' property="rdfs:label" content="L-Alanine" datatype="xsd:string"></span>L-alanine</span> ligase), MurD (UDP-N-acetyl-                 N-acetylglucosamine transferase), PBPs (penicillin binding proteins). <br />  <br />  <br />  <br /> 42&#203;&#353;C present extract TKL-11 harboring              (residues red bold), putative active site residues vector control (pBAD33) grown 30&#203;&#353;C (Figure 7C).                        conserved DapL enzymes. addition, loop <br />                                                                                regions form active site cleft generally  AMINO ACIDS CONSTITUTE ACTIVE SITE                            served. particular, loops C conserved, CONSERVED VsDapL                                                            loop B shows variability. previous structural studies examine closely VsDapL variant amino acid                   (Watanabe et al., 2007, 2011; Dobson et al., 2011), known <br /> level, aligned amino acid sequence DapL                loop B disordered role catalysis clear. <br /> orthologs functionally structurally characterized              noting increased substrate promiscuity displayed Arabidopsis thaliana (Watanabe et al., 2007), C. reinhardtii (Dob-             VsDapL, possible loop B involved substrate <br /> son et al., 2011), C. trachomatis (Watanabe et al., 2011;                  binding step. <br /> Figure 8). VsDapL sequence similarity                         homology model VsDapL generated using CrDapL DapL sequences, like Ct DapL sequence,                   Swiss-Model Protein Modeling Server (Figure 9). tertiary truncated N-terminal region. longer N-terminal regions               structure DapL monomer annotated hav- eukaryotic sequences attributed subcellular localiza-            ing large small domain, includes arm <br /> tion sequences, used  thaliana C. reinhardtii target               region (Figure 9A). aminotransferases approximately enzyme chloroplast, lysine biosynthesis known              100 kDa mass, representing homodimeric quaternary struc- occur (Mills Wilson, 1978). seen Figure 8                  ture. Assuming functional unit VsDapL enzyme  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                          2012 | Volume 3 | Article 183 | 6 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />  FIGURE 6 | Phylogenetic relationship DapL amino acid sequences.           Branch lengths proportional divergence; scale tree <br />  VsDapL ortholog annotated black circle. Minimum evolution      indicates percent divergence branch lengths. tree rooted <br />    topology shown maximum likelihood (ML), maximum                    single outgroup aspartate aminotransferase (AspC)  parsimony (MP), minimum evolution  , neighbor-joining (NJ)            Y. pseudotuberculosis. Figure drawing produced using FigTree (ver. 1.6.1; <br />  bootstrap values DapL1 DapL2 clades  left right).   Rambaut, 2009). <br />  <br />  <br />  dimer, required proper orientation conserved            essentiality dapL mutagenic experiments using transposon <br /> active site residues active site, VsDapL probably              homologous recombination.  organism active sites (Figure 9B). Closer examination active site           aerobic, relatively easy culture non-pathogenic humans, <br /> (Figure 9C) highlights residues conserved             unlike bacteria contain DapL pathway, Lep- <br /> multiple sequence alignment (Figure 8) key loop regions               tospira interrogans, Bacteriodes fragilis, C. trachomatis (McCoy line active site. Importantly, model predicts loop          et al., 2006; Hudson et al., 2008). <br /> B positioned entrance active site cleft, ideally placed          Recently, laboratories demonstrated orthologous interact substrates entering active site.                         dapL model plant Arabidopsis thaliana essen- <br />                                                                               tial growth development (Dobson et al., 2011).  <br /> DISCUSSION                                                                    known true bacteria exclusively discovery new biosynthetic route diaminopimelate                  utilize DapL pathway diaminopimelate/lysine synthesis. lysine plants, algae, bacteria provides fresh target         known aminotransferase capable inter- discovery novel herbicides, algaecides, antibiotics.           converting tetrahydrodipicolinate L,L-diaminopimelate exists <br /> amino acids constitute active site DapL conserved             bacteria solely contain DapL pathway.  suit- V. spinosum ortholog based protein alignment mod-               able dapL bacterial mutant auxotrophic diaminopime- <br /> eling.  inhibitors Ct DapL DapL likely             late/lysine important step forward especially inhibit VsDapL. end, V. spinosum suitable bacterial              examples literature regarding substrate promis- development vivo assays discovery             cuity aminotransferases involved amino acid metabolism. <br /> compounds able inhibit DapL facilitate antibiotic              instance, aspartate aminotransferase, tyrosine amino- <br /> development.                                                                  transferase, branched-chain amino acid aminotransferase <br />     V. spinosum genetically altered using exogenous              shown possess overlapping activities E. coli (Gu <br /> DNA, V. spinosum excellent candidate determining                et al., 1998). addition, recent study demonstrated  <br />  <br />  <br /> www.frontiersin.org                                                                                                       2012 | Volume 3 | Article 183 | 7 <br />  Nachar et al.                                                                           Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />                                                                                  supplemented diaminopimelate medium observe <br />                                                                                  growth phenotype. inhibitory compound speci&#239;&#172;?c                                                                                  DapL, expect effect                                                                                  mutant.  compound effect                                                                                  wild-type parental strain used create dapL mutant.                                                                                  inhibition DapL prevent m-DAP production                                                                                  lead cell death caused lack proper protein cell wall <br />                                                                                  biosynthesis. <br />                                                                                     present elucidation biosynthetic pathway <br />                                                                                  diaminopimelate/lysine V. spinosum using genomic, bio- <br />                                                                                  <span id='am-1' about='obo:IMR_0000947' typeof='owl:Thing obo:IMR_0000001'><span id='am-2' property="rdfs:label" content="chemical" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:id" content="IMR:0000947" datatype="xsd:string"></span>chemical</span>, structural approaches. enzymes involved                                                                                  bacterial diaminopimelate/lysine biosynthetic pathways pre- <br />                                                                                  sumed targets antibiotic development. identi&#239;&#172;?cation                                                                                  characterization enzymes provides valuable information <br />                                                                                  pertaining experiments aiming elucidate essentiality                                                                                  genes involved bacterial diaminopimelate/lysine biosyn- <br />                                                                                  thesis pathways. DapL pathway sole route                                                                                  diaminopimelate/lysine V. spinosum. addition, bacterium <br />                                                                                  genetically modi&#239;&#172;?ed, organism relatively easy cul- <br />                                                                                  ture non-pathogenic animals. criteria make V. spin- <br />                                                                                  osum excellent bacterial model elucidating essentiality <br />                                                                                  genes involved diaminopimelate/lysine metabolism. <br />  <br />                                                                                  MATERIALS METHODS <br />                                                                                  V. SPINOSUM GROWTH CONDITIONS <br />                                                                                  plasmids strains used study listed Table 3.                                                                                  V. spinosum DSM 4136T organism cultured R2A medium <br />                                                                                  26&#203;&#353;C. <br />  <br />                                                                                  MULTIPLE-PROTEIN SEQUENCE ALIGNMENT, PHYLOGENETIC TREE <br />                                                                                  CONSTRUCTION, HOMOLOGY MODEL VsDapL <br />                                                                                  DapL protein sequence alignment generated using   FIGURE 7 | Functional complementation E.coli murE mutant.               ClustalW server2 . Phylogenetic trees constructed MUS- <br />     E. coli mutant TKL-11 harboring pBAD33 pBAD33 +VsmurE.                                                                                  CLE aligned amino acid sequences using maximum likelihood <br />   bacteria grown OD 0.1 600 nm serially diluted   10&#226;&#710;&#8217;1 , 10&#226;&#710;&#8217;2 , 10&#226;&#710;&#8217;3 using 0.85% (w/v) saline. bacteria             (ML), maximum parsimony (MP) minimum evolution  , <br />   replica-plated LB medium supplemented 0.2% (w/v) arabinose.        neighbor-joining (NJ) techniques MEGA 5.05 (Tamura <br />   growth assessed 30 42&#203;&#353;C 24 h. colony        et al., 2011). ML searches performed using likelihood- <br />   vector recombinant plasmid used create diagram.            based WAG (Whelan Goldman, 2001) model amino acid <br />   (B) Growth curve E. coli TKL-11 harboring vector <br />                                                                                  character evolution, gamma distributed rates sites,   pBAD33 (circles), pBAD33 +VsmurE (triangles). cells grown   30&#203;&#353;C 5 h shifted 42&#203;&#353;C additional &#239;&#172;?ve hours (shaded area).   proportion invariant sites estimated data set <br />   (C) SDS-PAGE analysis proteins extracted TKL-11 grown LB            (WAG + G + . NJ tree searches performed using <br />   supplemented 0.2% arabinose (w/v). Lane (1) protein ladder (kDa),         parsimony-based JTT model (Jones et al., 1992) gamma <br />   Lane (2) 10 &#206;&#188;g soluble protein extract TKL-11 harboring pBAD33         distributed rates (JTT + G). Optimal models determined <br />   grown 30&#203;&#353;C, Lane (3) 10 &#206;&#188;g soluble protein extract harboring <br />   pBAD33 +VsmurE grown 42&#203;&#353;C. proteins resolved 10% <br />                                                                                  MEGA 5.05 iteratively tests partition hierar- <br />   (w/v) acrylamide gel stained Coomassie blue                  chically nested models evolution obtain best &#239;&#172;?t. Node <br />   visualization.                                                                 strength assessed using bootstrap technique 100 <br />                                                                                  pseudo-replicate data sets analysis type (ML, MP,                                                                                   NJ). accessions numbers following DapL proteins  <br /> aminotransferases involved alanine biosynthesis E. coli                Arabidopsis thaliana (AEE86265), V. spinosum (ZP_02927470), <br /> (Yoneyama et al., 2011).                                                         C. reinhardtii (XP_001693061), C. trachomatis (NP_219900), <br />    Aminotransferases common genome V. spinosum                     Parachlamydia UWE25 (YP_007684), Synechocystis sp. PCC 6803 <br /> given integral role multiple anabolic catabolic path-               (BAA10583), Thermosynechococcus elongatus BP1 NP_682892), <br /> ways. compound inhibits bacterial DapL vitro,                Bacteroides fragilis NCTC 9343 (YP_212286), Clostridium <br /> case Arabidopsis enzyme, question com-                thermocellum 27405 (YP_001039489), Methanothermobacter <br /> pound speci&#239;&#172;?c DapL vivo investigated. way answer question use vivo studies exposing putative inhibitory compound dapL mutant                    2 http://www.ebi.ac.uk/Tools/msa/clustalw2/ <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                              2012 | Volume 3 | Article 183 | 8 <br />  Nachar et al.                                                                        Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 8 | Protein sequence alignment DapL V. spinosum, C.          loops line active site, asterisk refers loop <br />   trachomatis,  thaliana, C. reinhardtii generated using ClustalW   contributing active site opposing monomer. ClustalW scores <br />   server (http://www.ebi.ac.uk/Tools/msa/clustalw2/). Putative active-site    relative V. spinosum sequence  vs. C. reinhardtii = 49 <br />   residues, based structural studies ligand bound DapL   (length = 443 residues, identity); vs.  thaliana = 45 (length = 426 residues); <br />   Cr DapL enzymes, shown red. loop regions correspond key   vs. C. trachomatis = 39 (length = 394 residues). <br />  <br />  <br />  <br /> thermoautotrophicus (NP_275195), Geobacter sulfurreducens                     (NP_927054), Methanosaeta thermophila PT (YP_843230), <br /> (NP_951224), Leptospira interrogans (YP_002757), Nostoc                       Methanospirillum hungatei JF1 (YP_504354), Dehalococcoides <br /> PCC 7120 (NP_488367), Gloeobacter violaceus PCC 7421                          CBDB1 (YP_307791), Methanococcoides burtonii DSM 6242 <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                        2012 | Volume 3 | Article 183 | 9 <br />  Nachar et al.                                                                Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br />   FIGURE 9 | VsDapL homology model generated                 N-terminal arm region compared Cr DapL DapL. (B) Assuming <br />   Swiss-Model Protein Modeling Server (Arnold et al., 2006) using     dimeric structure, required place conserved active site <br />   Cr DapL apo structure template (PDB <span id='am-146' about='oboInOwl:id' typeof='owl:Thing'>id</span>: 3QGU Dobson        resides (highlighted red bold Figure 8), active sites   et al., 2011; Hudson et al., 2011a, BLAST identity = 51% (207/404   dimer shown. (C) close view active site 1 showing   residues), QMEAN4 score = 0.673).   domain structure   conserved active site residues loops lining active site    monomer. Note  seen Figure 8, VsDapL truncated     Figure 8). <br />  <br />  <br />  <br /> (YP_565702), Methanosarcina barkeri Fusaro (YP_306095),               appropriate geometry using visualization software COOT <br /> Methanosarcina acetivorans C2A (NP_616639), Syntrophobac-             (Emsley Cowtan, 2004). <br /> ter fumaroxidans MPOB (YP_844192), Planctomyces brasilien- <br /> sis DSM 5305 (YP_004269630), Planctomyces limnophilus DSM             PCR AMPLIFICATION CLONING V. SPINOSUM dapL 3776 (YP_003632005), Verrucomicrobiae bacterium DG1235                murE OPEN READING FRAMES <br /> (ZP_05058547). accession number Yersinia pseudo-          length ORFs annotated locus tags VspiD_0101000 <br /> tuberculosis IP 32953 aspartate aminotransferase (AspC)            12510 (dapL, L,L-diaminopimelate aminotransferase) (CAH20674).                                                           VspiD_010100019130 (murE, UDP-N -acetylmuramoylalanyl-d- <br />     homology model VsDapL protein generated using        glutamyl-2,6-meso-diaminopimelate ligase) ampli&#239;&#172;?ed Swiss-Model Protein Modeling Server (Arnold et al., 2006)         PCR. ORFs ampli&#239;&#172;?ed using: 12 pmol forward using CrDapL apo structure template (PDB id: 3QGU;           reverse primers, 1 mM MgSO4 , 0.5 mM Dobson et al., 2011). model examined hand clashes      deoxynucleotide triphosphates, 0.5 ng genomic DNA 1 unit <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                2012 | Volume 3 | Article 183 | 10 <br />  Nachar et al.                                                                               Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Table 2 | Activity VsDapL various amino donors amino                     Table 3 | Plasmids strains used study. <br /> acceptors. <br />                                                                                      Plasmid/strains                                 Vendor/Reference <br /> Amino donor                  Relative         Amino acceptor        Relative <br />                              activity (%)                           activity (%)     pET30A                                          Novagen, USA <br />                                                                                      pET100D/Topo                                    Invitrogen, USA <br /> L,L-diaminopimelate          100              2-Ketoglutarate       100              pBAD33                                          Guzman et al. (1995) <br /> meso-Diaminopimelate         1.6 &#194;&#177; 0.35       Pyruvate              8.1 &#194;&#177; 0.3        pBAD33 + AtdapL                                 Hudson et al. (2006) <br /> <span id='am-17' about='obo:IMR_0200140' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-18' property="oboInOwl:id" content="IMR:0200140" datatype="xsd:string"></span><span id='am-19' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasDbXref" content="KEGG:C00047" datatype="xsd:string"></span><span id='am-21' property="rdfs:label" content="L-Lysine" datatype="xsd:string"></span>L-Lysine</span>                     3.0 &#194;&#177; 0.2        Oxaloacetate          7.4 &#194;&#177; 0.15       pBAD33 + CrdapL                                 Dobson et al. (2011) <br /> <span id='am-32' about='obo:IMR_0200342' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0001695 obo:IMR_0000001'><span id='am-33' property="oboInOwl:id" content="IMR:0200342" datatype="xsd:string"></span><span id='am-34' property="rdfs:label" content="L-Ornithine" datatype="xsd:string"></span><span id='am-35' property="oboInOwl:hasDbXref" content="KEGG:C00077" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>L-Ornithine</span>                  5.1 &#194;&#177; 0.2        Oxovalerate           7.9 &#194;&#177; 0.15       pET30A +VsdapL                                  study <br />                                                                                      pET100D +VsmurE                                 study assay measures production dihydroquinazolium 440 nm using <br />                                                                                      pBAD33 +VsdapL                                  study ortho-aminobenzaldehyde assay. assays pertaining amino <br />                                                                                      pBAD33 +VsmurE                                  study <br /> donor/acceptor combination triplicates, relative activity based <br />                                                                                      Verrucomicrobium spinosum DSM 4136              Schlesner (1987) activity enzyme L,L-diaminopimelate 2-ketoglutarate                                                                                      5-alpha competent cells                         New England Biolabs, USA <br /> substrates. <br />                                                                                      BL21 codon plus RIPL                            Agilent Technologies, USA Platinum Pfx DNA polymerase (Invitrogen Corporation, Carls-                       AOH1 (&#206;&#8221;dapD/dapE )                              Hudson et al. (2006) <br /> bad, CA, USA) using following PCR conditions: 1 cycle                         TKL-11 (murE )                                  Lugtenberg van <br /> 94&#203;&#353;C 2 min, followed 30 cycles 94&#203;&#353;C 15 s, 60&#203;&#353;C                                                                     Schijndel-van Dam (1972) <br /> 30 s 72&#203;&#353;C 2 min. forward reverse primers used amplify ORFs VsdapL F VsdapL R dapL ORF                         Table 4 | List Primers used PCR amplification, cloning VsmurE F VsmurE R murE ORF. <span id='am-79' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-80' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-81' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-82' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-83' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-84' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-85' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-86' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span>nucleotide</span>                           nucleotide sequencing. <br /> sequences primers listed Table 4. underlined <br /> sequence represents restriction enzyme sites used facilitate                  Primer Sequence  5  3  ) <br /> cloning ORF bolded italicized sequences repre- <br />                                                                                      VsdapL F         CCCCGAATTCATGGCCCTCATCAACGAAAACTTCCTCAAG <br /> sent initiation termination codons. cloning, dapL PCR <br />                                                                                      VsdapL R         CCCCGTCGACCTACTTCAGCGCGGCGATACGGCGGCAGAC <br /> fragment digested EcoRI SalI ligated                                                                                      VsmurE F         CACCATGACCATTTTGCGCGATCTTATCGAGGGT <br /> plasmid pET30a (EMD Biosciences, Gibbstown, NJ, USA) pro- <br />                                                                                      VsmurE R         GTCGACTCACTGACGGTCATCCCTCCTTTGGCGTGC <br /> duce plasmid pET30a + VsdapL. murE PCR fragment                                                                                      T7 promoter      TAATACGACTCACTATAGGG <br /> ligated plasmid pET100D/topo (Invitrogen Corporation, <br />                                                                                      T7 R             TAGTTATTGCTCAGCGGTGG <br /> Carlsbad, CA, USA) produce plasmid pE100D + VsmurE. <br />                                                                                      T7 terminator TATGCTAGTTATTGCTCAG recombinant protein derived plasmid carries hexa- <br /> histidine tag derived pET30a pET100D plasmids                        underlined sequences represent restriction enzyme sites bolded ital- <br /> amino terminus. con&#239;&#172;?rm &#239;&#172;?delity PCR reactions,                      icized sequences denoted start stop codons open reading <br /> dapL ORF sequenced pET30a using T7 promoter                             frames (ORFs). T7 terminator primer (Table 4). murE ORF sequenced pET100D using T7 promoter T7 R primer                         V. spinosum (pBAD33 + VsdapL), Arabidopsis thaliana <br /> (Table 4). dapL murE ORFs 100% identical                     (pBAD33 + AtdapL), C. reinhardtii (pBAD33 + CrdapL). <br /> sequences deposited Integrated Microbial Genomes (IMG)                        Transformants selected LB agar medium supplemented <br /> public database3 .                                                                   50 &#206;&#188;g mL&#226;&#710;&#8217;1 DAP 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol.                                                                                      individual colonies vector recombinant <br /> FUNCTIONAL COMPLEMENTATION PLASMID CONSTRUCTS                                        plasmid replica-plated LB medium plus 0.2% plasmids used functional complementation E.                           (w/v) arabinose 50 &#206;&#188;g mL&#226;&#710;&#8217;1 DAP. cultures <br /> coli &#206;&#8221;dapD/dapE double mutant E. coli murE mutant                            grown 30&#203;&#353;C 24 h. E. coli murE mutant (TKL-11) produced sub-cloning XbaI SalI fragment                         (thr-1, leuB6  , murE1, fhuA21, codA1, lacY1, tsx-95, glnV44 <br /> pET30a + VsdapL pET100D + VsmurE plasmids                                , &#206;&#187;&#226;&#710;&#8217; , pyrF101, 108, thyA6, argG66, ilvA634, thi-1, deoC1) <br /> pBAD33 produce pBAD33 + VsdapL pBAD33 + VsmurE                                transformed pBAD33 pBAD33 + VsmurE (Table 3). <br /> (Guzman et al., 1995). fusion proteins produced                         Transformants selected LB agar medium supplemented <br /> pBAD33 constructs identical proteins produced                        50 &#206;&#188;g mL&#226;&#710;&#8217;1 <span id='am-87' about='obo:IMR_0200036' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-88' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-89' property="rdfs:label" content="Thymine" datatype="xsd:string"></span><span id='am-90' property="oboInOwl:id" content="IMR:0200036" datatype="xsd:string"></span><span id='am-91' property="oboInOwl:hasDbXref" content="KEGG:C00178" datatype="xsd:string"></span>thymine</span> 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol pET30a pET100D constructs.                                                   30&#203;&#353;C. individual colonies vector recombi- <br />                                                                                      nant plasmid transformation replica-plated LB medium <br /> FUNCTIONAL COMPLEMENTATION E. COLI dapD/E murE                            plus 0.2% (w/v) arabinose 50 &#206;&#188;g mL&#226;&#710;&#8217;1 thymine. cul- <br /> MUTANTS                                                                              tures grown 30 42&#203;&#353;C 24 h assess functional E. coli mutant AOH1 ( dapD::Kan2, dapE6 ) trans-                             complementation. <br /> formed pBAD33 plasmids harboring dapL orthologs <br />                                                                                      PROTEIN EXPRESSION PURIFICATION VsDapL <br />                                                                                      E. coli BL21-CodonPlus-RIPL strain transformed 3 http://img.jgi.doe.gov/cgi-bin/w/main.cgi                                          plasmid pET30a + VsdapL grown LB broth containing <br />  <br />  <br />  <br /> www.frontiersin.org                                                                                                            2012 | Volume 3 | Article 183 | 11 <br />  Nachar et al.                                                                                 Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> 50 &#206;&#188;g mL&#226;&#710;&#8217;1 kanamycin 34 &#206;&#188;g mL&#226;&#710;&#8217;1 chloramphenicol 37&#203;&#353;C                           MESO-DIAMINOPIMELATE DEHYDROGENASE ACTIVITY OD600 0.5. Protein expression induced 1.0 L                        Ddh activity assessed using oxidative deamination. assay <br /> culture using isopropyl &#206;&#178;-d-1-thiogalactopyranoside &#239;&#172;?nal                         consisted 0.5 mM m-DAP, 0.5 mM NADP+ , 1 mg crude <br /> concentration 0.5 mM 6 h 20&#203;&#353;C. cell pellet lysed                    soluble V. spinosum extract 10 &#206;&#188;g pure recombinant meso- sonication solution 50 mM sodium phosphate (pH 8.0)                        diaminopimelate dehydrogenase C. thermocellum (Hudson 300 mM NaCl. soluble extract incubated 1 mL                          et al., 2011b) 100 mM <span id='am-122' about='obo:IMR_0100118' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001695 obo:IMR_0000001 obo:IMR_0100116'><span id='am-123' property="oboInOwl:hasExactSynonym" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-124' property="oboInOwl:hasExactSynonym" content="Glycine" datatype="xsd:string"></span><span id='am-125' property="oboInOwl:hasExactSynonym" content="Gly" datatype="xsd:string"></span><span id='am-126' property="oboInOwl:hasAlternativeId" content="IMR:0200057" datatype="xsd:string"></span><span id='am-127' property="rdfs:label" content="Glycine" datatype="xsd:string"></span><span id='am-128' property="oboInOwl:hasDbXref" content="KEGG:C00037" datatype="xsd:string"></span><span id='am-129' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-130' property="rdfs:label" content="Aminoacetic acid" datatype="xsd:string"></span><span id='am-131' property="rdfs:label" content="Gly" datatype="xsd:string"></span><span id='am-132' property="rdfs:label" content="glycine" datatype="xsd:string"></span><span id='am-133' property="oboInOwl:id" content="IMR:0100118" datatype="xsd:string"></span>glycine</span>-KOH (pH 10.5) 100 mM <br /> bed volume Talon Metal Af&#239;&#172;?nity Resin (Clontech, Mountain                           HEPES-KOH (pH 7.6) &#239;&#172;?nal volume 0.5 mL. produc- <br /> View, CA, USA) 30 min 4&#203;&#353;C. resin washed &#239;&#172;?ve                            tion NADPH measured continuously 340 nm 30&#203;&#353;C <br /> times 30 mL sonication buffer containing 10 mM imi-                           using Beckman DU640 spectrophotometer <br /> dazole (pH 8.0) 15 min  enzyme eluted 10 mL sonication buffer containing 250 mM imidazole (pH                            ACKNOWLEDGMENTS <br /> 8.0). pure protein concentrated Amicon Ultra                            research supported United States National Science <br /> 10,000 molecular weight cutoff &#239;&#172;?lter unit replacing elution                       Foundation (NSF) award AOH (MCB-1120541). Victoria R. <br /> buffer 100 mM HEPES-KOH containing 1 mM DTT, 2 mM                                Nachar supported Undergraduate Summer Research <br /> EDTA (pH 7.6). recombinant protein stored 50%                              Fellowship College Science RIT. Victoria R. Nachar glycerol. Protein concentration measured using Brad-                          Francisco C. Savka enrolled Biology Program SEM <br /> ford assay bovine serum albumin standard (Bradford,                       enrolled Bioinformatics Program RIT. Victoria R. Nachar, <br /> 1976).                                                                                Francisco C. Savka, Sean E. McGroty supported                                                                                       NSF award (MCB-1120541) Andr&#195;&#169; O. Hudson. Renwick C. J. <br /> L,L-DIAMINOPIMELATE AMINOTRANSFERASE ENZYME                                           Dobson acknowledges CR Roper Bequest Fellowship sup- <br /> ORTHO-AMINOBENZALDEHYDE ASSAY                                                         port, New Zealand Royal Society Marsden Fund (UOC1013) ortho-aminobenzaldehyde assay measured formation                           funding support material based work sup- <br /> THDPA L,L-DAP using ortho-aminobenzaldehyde,                               ported U. S. Army Research Laboratory U. S. <br /> forms dihydroquinazolium adduct maximum absorbance                           Army Research Of&#239;&#172;?ce contract/grant number W911NF-11- 440 nm. assay contained 100 mM HEPES-KOH (pH 7.6),                             1-0481. Katherine  Donovan Rachel  North acknowledge <br /> 0.5 mM amino donor, 2 mM amino acceptor 1.25 mM ortho-                            University Canterbury Summer Scholarship Scheme. Andr&#195;&#169; <br /> aminobenzaldehyde 10.0 &#206;&#188;g puri&#239;&#172;?ed recombinant VsDapL                           O. Hudson thanks Naomi L. Ward form University Wyoming <br /> enzyme 0.5 mL reaction. reactions incubated                          graciously providing V. spinosum strain/genomic DNA. 30&#203;&#353;C change absorbance measured continuously                           authors thank Michael  Savka (RIT) helpful discussions 440 nm using Beckman DU640 spectrophotometer.                                    revisions manuscript. <br />  <br /> REFERENCES                                    development. PLoS 6, e20439.          novel diaminopimelate biosynthe-          Jones, D. T., Taylor,W. R., Thornton, <br /> Arnold, K., Bordoli, L., Kopp, J.,        doi:10.1371/journal.pone.0020439          sis pathway prokaryotes iden-             J. M. (1992). rapidgeneration <br />    Schwede, T. (2006). SWISS-          Domman, D. B., Steven, B. T.,            ti&#239;&#172;?es diverged form L,L-                mutationdatamatrices pro- <br />    MODEL workspace: web-based               Ward, N. L. (2011). Random trans-         diaminopimelate aminotransferase.            teinsequences. Comput. Appl. Biosci. <br />    environment protein structure          poson mutagenesis Verrucomi-           J. Bacteriol. 190, 3256&#226;&#8364;&#8220;3263.                8, 275&#226;&#8364;&#8220;282. <br />    homology modeling. Bioinformatics          crobium spinosum DSM 4136(T).           Hudson,  O., Gir&#195;&#179;n, , Dobson,       Liu, Y., White, R. H., Whitman, <br />    22, 195&#226;&#8364;&#8220;201.                               Arch. Microbiol. 193, 307&#226;&#8364;&#8220;312.            R. C. J. (2011a). Crystallization        W. B. (2010). Methanococci use Baizman, E. R., Branstrom,  , Lon-     Emsley, P., Cowtan, K. (2004). Coot:     preliminary X-ray diffraction analy-         diaminopimelate aminotransferase <br />    gley, C. B., Allanson, N., So&#239;&#172;  M.        model-building tools molecular        sis L,L-diaminopimelate amino-            (DapL) pathway lysine biosyn- <br />    J., Gange, D., Goldman, R.             graphics. Acta Crystallogr. D Biol.       transferase (DapL) Chalmy-              thesis. J. Bacteriol. 192, 3304&#226;&#8364;&#8220;3310. <br />    C. (2000). Antibacterial activity       Crystallogr. 60, 2126&#226;&#8364;&#8220;2132.               domonas reinhardtii. Acta Crystal-        Lugtenberg, E. J. J., van Schijndel- <br />    synthetic analogues based        Fan, C., Clay, M. D., Dehyholos, M.          logr. Sect. F Struct. Biol. Cryst. Com-      van Dam,  (1972). Temperature- <br />    disaccharide structure moeno-           K., Vederas, J. C. (2010).            mun. 67, 140&#226;&#8364;&#8220;143.                            sensitive mutants Escherichia <br />    mycin, inhibitor bacterial           Exploration inhibitors           Hudson,  O., Klartag, , Gilvarg,           coli K-12 low activity    transglycosylase. Microbiology 146,        diaminopimelate aminotransferase.         C., Dobson, R. C. J., Marques, F.            diaminopimelic acid adding <br />    3129&#226;&#8364;&#8220;3140.                                 Bioorg. Med. Chem. 18, 2141&#226;&#8364;&#8220;2151.         G., Leustek, T. (2011b). Dual            enzyme. J. Bacteriol. 110, 41&#226;&#8364;&#8220;46. <br /> Bradford, M. M. (1976). rapid        Gu, W., Song, J., Bonner, C. , Xie,        diaminopimelate biosynthesis path-        McCoy,  J, Adams, N. E., Hud- <br />    sensitive method quanti&#239;&#172;?ca-        G., Jensen, R.  (1998). PhhC        ways Bacteroides fragilis             son,  O., Gilvarg, C., Leustek, T., <br />    tion microgram quantities pro-       essential aminotransferase      Clostridium thermocellum. Bochim.            Maurelli,  T. (2006). L,L- <br />    tein utilizing principle pro-       aromatic amino acid metabolism         Biophys. Acta 1814, 1162&#226;&#8364;&#8220;1168.               Diaminopimelate aminotransferase, <br />    tein dye binding. Anal. Biochem. 72,       Pseudomonas areuginosa. Micrbiol-       Hudson,  O., Singh, B. K., Leustek,          trans-kingdom enzyme shared <br />    248&#226;&#8364;&#8220;254.                                   ogy 144, 3127&#226;&#8364;&#8220;3134.                       T., Gilvarg, C. (2006). L,L-          Chlamydia plants syn- <br /> Cox, R. J. (1996). DAP pathway         Guzman, L. M., Belin, D., Carson, M. J.,     diaminopimelate aminotransferase             thesis diaminopimelate/lysine. <br />    lysine target antimi-          Beckwith, J. (1995). Tight reg-       de&#239;&#172;?nes novel variant lysine         Proc. Natl. Acad. Sci. U.S  103, <br />    crobial agents. Nat. Prod. Rep. 13,        ulation, modulation, high level       biosynthesis pathway plants. Plant        17909&#226;&#8364;&#8220;17914. <br />    29&#226;&#8364;&#8220;43.                                     expression vectors containing      Physiol. 140, 292&#226;&#8364;&#8220;301.                    Mills, W. R., Wilson, K. G. <br /> Dobson, R. C. J., Gir&#195;&#179;n, ,              arabinose pBAD promoter. J. Bacte-      Hutton, C. , Perugini, M. ,            (1978). Amino acid biosynthe- <br />    Hudson,  O. (2011). L,L-                 riol. 177, 4121&#226;&#8364;&#8220;4130.                     Gerrard, J.  (2007). Inhibition            sis isolated pea chloroplast: <br />    diaminopimelate aminotransferase        Hudson,  O., Gilvarg, C.,              lysine biosynthesis: evolving          metabolism labeled aspar- <br />          Chlamydomonas        rein-      Leustek, T. (2008). Biochemical       antibiotic strategy. Mol. Biosyst. 3,        tate sulfate. FEBS Lett. 92, <br />    hardtii: target algaecide            phylogenetic characterization        458&#226;&#8364;&#8220;465.                                     129&#226;&#8364;&#8220;132. <br />  <br />  <br />  <br />  <br /> Frontiers Microbiology | Evolutionary Genomic Microbiology                                                                 2012 | Volume 3 | Article 183 | 12 <br />  Nachar et al.                                                                                   Diaminopimelate lysine biosynthesis Verrucomicrobium spinosum <br />  <br />  <br />  <br />  <br /> Nishida, H. M., Nishiyama, N.,                  Sahm, H. (1991). functionally            James, M. N. (2011). struc-          conducted absence com- <br />     Kobayashi, T., Kosuge, T., Hoshimo,         split pathway lysine synthesis         ture L,L-diaminopimelate amino-       mercial &#239;&#172;?nancial relationships     T., Yamane, H. (1999).                Corynebacterium glutamicum. J.          transferase Chlamydia tra-          construed potential  <br />     prokaryotic gene cluster involved           Bacteriol. 173, 4510&#226;&#8364;&#8220;4516.                 chomatis: implications broad     &#239;&#172;&#8218;ict  <br />     synthesis lysine           Tamura, K., Peterson, D., Peterson,           substrate speci&#239;&#172;?city. J. Mol Biol. 19, <br />     amino adipate pathway:                N., Stecher, G., Nei, M.,              649&#226;&#8364;&#8220;660.                                 Received: 14 January 2012; accepted: 02 <br />     key evolution amino               Kumar, S. (2011). MEGA5: mole-          Watanabe, N., Clay, M. D., van Belkum,      2012; published online: 25     acid biosynthesis. Genome Res. 9,           cular evolutionary genetics analy-         M. J., Marcus, S. L., Flegel, M.         2012. <br />     1175&#226;&#8364;&#8220;1183.                                  sis using maximum likelihood,              D., Clay, M. D., Deyholos, M.            Citation: Nachar VR, Savka FC, <br /> Rambaut,  (2009). FigTree: Tree Figure        evolutionary distance, maxi-           K., Vederas, J. C., James, M.        McGroty SE, Donovan KA, North RA, <br />     Drawing Tool. Version 1.3.1, Insti-         mumparsimony. Mol. Biol. Evol. 28,         N. (2007). Crystal structure          Dobson RCJ, Buckley LJ Hudson <br />     tute Evolutionary Biology, Uni-          2731&#226;&#8364;&#8220;2739.                                 L,L-diaminopimelate aminotrans-          AO (2012) Genomic biochemi- <br />     versity Edinburgh. Available       Velasco,  M., Leguina, J. , Laz-      ferase Arabidopsis thaliana:      cal analysis diaminopimelate <br />     http://tree.bio.ed.ac.uk/                   cano,  (2002). Molecular evolution       recently discovered enzyme        lysine biosynthesis pathway Sait, M., Kamneva, O. K., Fay, D. S.,           lysine biosynthetic pathways.       biosynthesis L-Lysine plants       Verrucomicrobium spinosum: identi&#239;&#172;?- <br />     Kirienko, N. V., Polek, J., Shirasu-        J. Mol. Evol. 55, 445&#226;&#8364;&#8220;459.                 Chlamydia. J. Mol. Biol. 371,        cation partial characterization     Hiza, M. M., Ward, N. L.             Wagner, M., Horn, M. (2006).              685&#226;&#8364;&#8220;702.                                 L,L-diaminopimelate aminotransferase <br />     (2011). Genomic experimental            Planctomycetes, Verrucomi-          Whelan, S., Goldman, N. (2001).               UDP-N-acetylmuramoylalanyl- <br />     evidence suggests Verrucomi-           crobia, Chlamydiae sister              general empirical model pro-        D-glutamyl-2,6-meso- <br />     crobium spinosum interacts             phyla comprise superphylum               tein evolution derived multiple     diaminopimelate ligase.  Microbio. <br />     eukaryotes.  Microbiol. 2:211.        biotechnological medical          protein families using maximum-        3:183. doi: 10.3389/fmicb.2012.00183 <br />     doi:10.33389/fmicb.2011.00211               relevance. Curr. Opin <br /> </body></html>