<?xml version="1.0" encoding="UTF-8"?><html xmlns="http://www.w3.org/1999/xhtml" xmlns:oboInOwl="http://www.geneontology.org/formats/oboInOwl#" xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#" xmlns:obo="http://purl.obolibrary.org/obo/" xmlns:owl="http://www.w3.org/2002/07/owl#" xmlns:xsd="http://www.w3.org/2001/XMLSchema#" xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#" xmlns:dublincorens="http://purl.org/dc/elements/1.1/"  version="XHTML+RDFa 1.0" ><body>                            NIH Public Access <br />                             Author Manuscript <br />                             J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                            Published final edited form  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                             J Theor Biol. 2014 March 21; 345: 12&#226;&#8364;&#8220;21. doi:10.1016/j.jtbi.2013.12.009. <br />  <br />  <br />  <br />                            Incorporation Flexible Objectives Time-Linked Simulation <br />                            Flux Balance Analysis <br />                            Elsa W. Bircha, Madeleine Udellb, Markus W. Covertc <br />                            aChemical Engineering, Stanford University, Stanford, CA, 94305, USA <br />  <br />                            bComputational       Mathematical Engineering, Stanford University, Stanford, CA, 94305, USA <br />                            cBioengineering,       Stanford University, Stanford, CA, 94305, USA <br />  <br />                            Abstract <br />                                 present modifications Flux Balance Analysis (FBA) metabolic modeling <br />                                 framework relax implicit assumptions biomass reaction. flexible Flux Balance <br />                                 Analysis (flexFBA) objective removes fixed proportion reactants, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                 produce subset biomass reactants. time-linked Flux Balance Analysis (tFBA) simulation <br />                                 removes fixed proportion reactants byproducts,                                 transitions metabolic steady states. Used  flexFBA tFBA model time scale <br />                                 shorter regulatory growth steady state encoded biomass reaction.                                 combined short-time FBA method intended integrated modeling applications enable <br />                                 detailed dynamic depictions microbial physiology cell modeling.                                 example, modeling Escherichia coli, avoids artifacts caused low-copy-number <br />                                 enzymes single-cell models kinetic bounds. outside integrated modeling contexts,                                 detailed predictions flexFBA tFBA complement existing FBA techniques.                                 detailed metabolite production silico knockouts used identify correct essentiality <br />                                 predictions wrong reason. <br />  <br />  <br />                            Keywords <br />                                 Metabolism; Integrated Modeling <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                            1. Introduction <br />                                              Quantitative metabolic models important tools understanding engineering                                              behavior microorganisms. Flux Balance Analysis (FBA) powerful technique                                              simulate large metabolic networks kinetic parameters unavailable. FBA <br />                                              simulations capture microorganism growth, nutritional resource consumption, waste- <br />                                              product secretion rates [1, 2]. addition FBA generate knockout essentiality <br />                                              predictions treated hypotheses explore organism&apos;s metabolic capability <br />                                              [3, 4]. <br />  <br />  <br />  <br />  <br />                            Crown Copyright &#194;&#169; 2013 Authors. Published Elsevier  rights reserved <br />                            Publisher&apos;s Disclaimer: PDF file unedited manuscript accepted publication. service                            customers providing early version manuscript. manuscript undergo copyediting, typesetting, review                            resulting proof published final citable form. note production process errors                            discovered affect content, legal disclaimers apply journal pertain. <br />                             Birch et al.                                                                                            Page 2 <br />  <br />  <br />                                           Classical implementations FBA quantify microbial growth using rigid biomass reaction <br />                                           represents processes cell replication single proportion reactants <br />                                           required byproducts returned. used quantify microbial growth NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           objective used refine flux predictions evaluate perturbations [5, 6, 7]. <br />  <br />                                           biomass reaction produce balanced growth complete inactivity                                           predictions. applications assumptions underlying behavior <br />                                           valid results useful.  current silico biology <br />                                           incorporates FBA integrated models combine mathematical models different <br />                                           types interact simulation [8, 9]. applications &#226;&#8364;&#8221; notably cell <br />                                           models [10] &#226;&#8364;&#8221; rigid biomass reaction limitation. <br />  <br />                                           enable cell modeling, require nuanced alternative biomass <br />                                           reaction FBA produce metabolites non-wild-type non-steady-state <br />                                           proportions. <br />  <br />                                           work, relax implicit assumptions biomass reaction construct new <br />                                           FBA methods. assumption balanced population average growth, encoded                                           biomass reaction&apos;s fixed proportion reactants. second assumption steady <br />                                           state growth, encoded biomass reaction&apos;s fixed proportion byproducts reactants. <br />                                           Relaxing reactant byproduct assumptions results flexible FBA (flexFBA) NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           time-linked FBA (tFBA) approaches, respectively. <br />  <br />                                            balanced steady-state growth assumptions inherent biomass reaction                                           FBA make method applicable timescale longer regulatory cell process <br />                                           interactions. combining flexFBA tFBA methods relax assumptions, <br />                                           obtain short-time FBA appropriate use cell models. short-time scale                                           consistent cell models evaluate metabolic model timescales shorter <br />                                           regulatory process interactions explicitly represent. <br />  <br />                                1.1. Biomass Reaction Assumptions <br />                                           biomass reaction ubiquitous microbial FBA lends great predictive power <br />                                           constrained metabolic network. succinct mathematical form                                           composed straightforward parameter values. addition quantifying growth,                                           biomass reaction flux used optimization objective case                                           called `biomass objective&apos; [11]. literature evaluates ability various FBA <br />                                           objectives mimic observed growth, gene essentiality, flux states [6, 12, 13, 14, 15], <br />                                           comparison `biomass objective&apos; performance. contrast, discuss <br />                                           simulation regimes biomass reaction does adequately model range NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolic network function, longer relevant quantification growth. <br />  <br />                                           constraining process reactant requirements byproduct returns                                           biomass reaction combines subtly different assumptions deal (1) <br />                                           reactant reactant (2) byproduct reactant groups. <br />  <br />                                           Reactant reactant fixed proportion biomass reaction assumes population average <br />                                           balanced growth: homogeneity cells cells time. assumption                                           contained biomass reaction&apos;s negative coefficients. consequence, biomass <br />                                           reaction scales fractional fulfillment process reactants whichever                                           limited. Homogeneity cells arises biomass reaction coefficients <br />                                           bulk cell composition values. single cells short timescales homogeneity <br />                                           conflicts biological reality. Bulk phenotypes given average neglect <br />                                           variance underlying population [16, 17]. Strict temporal homogeneity metabolite <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 3 <br />  <br />  <br />                                           production ratios unreasonable transcriptional translational regulatory <br />                                           mechanisms enforce operate timescales longer typical FBA time <br />                                           step (1 sec minutes [18, 19]). Furthermore, regulatory interactions exist <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolites included biomass reaction enforce proportional <br />                                           production. Experimental observations reveal essential metabolites                                           produced non-wild-type proportions [20, 21, 22, 23]. Additionally, metabolites <br />                                           included biomass reaction essential model growth. biomass reaction <br />                                           includes process reactants non-essential cell replication, false-essential <br />                                           predictions result [24]. Previously, inflexible ratio essentiality biomass <br />                                           reaction addressed alternate biomass reaction definitions, [24, 25] reactions <br />                                           allowing similar metabolites substitute [26]; approaches                                           practical entire scale pathways metabolism. <br />  <br />                                           Byproduct reactant fixed proportion biomass reaction assumes steady state <br />                                           metabolic function. assumption contained biomass reaction&apos;s positive coefficients. <br />                                           principle example return spent energy carrier <span id='am-1' about='obo:IMR_0001360' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-2' property="rdfs:label" content="ADP" datatype="xsd:string"></span><span id='am-3' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-4' property="oboInOwl:hasAlternativeId" content="IMR:0200375" datatype="xsd:string"></span><span id='am-5' property="oboInOwl:id" content="IMR:0001360" datatype="xsd:string"></span><span id='am-6' property="oboInOwl:hasDbXref" content="KEGG:C00008" datatype="xsd:string"></span>ADP</span> set proportional                                           <span id='am-28' about='obo:IMR_0001351' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001349 obo:IMR_0000001 obo:IMR_0001698'><span id='am-29' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-30' property="oboInOwl:hasAlternativeId" content="IMR:0100127" datatype="xsd:string"></span><span id='am-31' property="oboInOwl:hasExactSynonym" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-32' property="rdfs:label" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-33' property="oboInOwl:hasExactSynonym" content="adenosine 5&apos;-triphosphate" datatype="xsd:string"></span><span id='am-34' property="oboInOwl:hasDbXref" content="KEGG:C00002" datatype="xsd:string"></span><span id='am-35' property="rdfs:label" content="adenosine triphosphate" datatype="xsd:string"></span><span id='am-36' property="oboInOwl:id" content="IMR:0001351" datatype="xsd:string"></span><span id='am-37' property="oboInOwl:hasAlternativeId" content="IMR:0200394" datatype="xsd:string"></span><span id='am-38' property="rdfs:label" content="ATP" datatype="xsd:string"></span>ATP</span> produced time step. Proportional byproducts reactants means                                           ADP return immediately matched capacity metabolism recharge ATP, <br />                                           consistent previous time step&apos;s metabolic conditions. Relating                                           reactant byproduct quantities reasonable, long-time assumption implied                                           single evaluation FBA optimization. Perturbations changes available media <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           resources result immediate transition new steady state single time step <br />                                           applied. type transition unrealistic short time step evaluations <br />                                           energy carrier supply turnover limited. <br />  <br />                                2. Methods <br />                                           Mathematically biomass reaction consists coefficients mi metabolites Mi appended <br />                                           single reaction column stoichiometric matrix S equation, <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                         (1) <br />  <br />                                           values mi &lt; 0 represent consumption process reactants, mi &gt; 0 represent <br />                                           byproduct return. majority n total mi values zero associated <br />                                           metabolites participate cell processes metabolism. Coefficient magnitudes <br />                                           anabolic products given relative quantities bulk biomass [11]; Coefficient <br />                                           magnitudes case catabolic energy carriers given requirements                                           macromolecule synthesis calculated bulk yields [11]. Units mi usually chosen <br />                                           flux biomass reaction, element flux vector v denoted vbio, <br />  <br />                                           directly interpreted microbial growth rate [1]. Metabolite accumulation                                                 set zero [1], applying steady state assumption metabolic network intermediates  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                Page 4 <br />  <br />  <br />                                             timescale evaluation &#206;&#8221;t, typically 1 s longer. Using biomass reaction flux                                             maximization objective, optimization problem  <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                     (2) <br />  <br />  <br />  <br />                                             S&#226;&#8364;&#178; = [S | m] v&#226;&#8364;&#178; vbio appended end v Equation 1. <br />  <br />                                             coefficients mi quantities required basis cell mass,                                             convenient think biomass reaction flux vbio fractional fulfillment                                             requirement time. classical FBA biomass reaction requires                                             fractional fulfillment metabolite requirements  <br />  <br />                                2.1. Designing Biomass Reaction Alternative <br />                                             sought relax biomass reaction&apos;s assumptions, wanted simultaneously <br />                                             preserve behavior wild-type long-time limits. developed flexFBA                                             produce possible process reactants inhibition distant/unrelated blocked <br />                                             pathways maintaining population average cell composition wild type networks. <br />                                             developed tFBA allow observation transient behavior integration                                             biological process models reproducing steady state growth. additional design <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             constraint need integrated modeling methods computationally efficient                                             function human supervision. <br />  <br />                                             flexFBA tFBA rely interfere  implemented <br />                                             separately  treat individually initial methods explanation <br />                                             evaluation.  flexFBA tFBA relax separate long-time assumptions <br />                                             biomass reaction, implementing achieve short-time FBA <br />                                             suitable use cell models. <br />  <br />                                2.2. Reactant Flexibility: flexFBA <br />                                             long process reactants participate reaction provided                                             network rates related nonzero constant multiplier. remove constraint                                             produce independently, append coefficients separate reactions                                             stoichiometric matrix: <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                                                                                                              (3) <br />  <br />                                             fluxes corresponding reactions fi appended v, intuitively <br />                                             represent fractional fulfillments requirement metabolites processes,                                             vary  combined matrix Equation                                             combined vector . blocks appended S square diagonal general forms                                             Equations 3 5, zero columns neglected practical use. <br />  <br />  <br />                                                  J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                            Page 5 <br />  <br />  <br />                                           objective criteria applied incentivize process reactant metabolite production &#226;&#8364;&#8221; <br />                                           large values fi &#226;&#8364;&#8221; simultaneously encourage proportional production &#226;&#8364;&#8221; similar values <br />                                           fi. wild type case, objective result process reactant exchange NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           metabolic network identical biomass reaction flux maximization case                                           Equation (2). achieve mathematically using objective, <br />  <br />                                                                                                                   (4) <br />  <br />  <br />                                           fatp fractional fulfillment energy carrier ATP. maximizes fractional <br />                                           fulfillment ATP penalizing metabolite produced proportionally                                           ATP. weight &#206;&#179; applied penalty terms constant, explain choose <br />                                           later. Figure 1a compares biomass reaction flux maximization flexible form. <br />  <br />                                           &#226;&#8222;&#8220;1 norm penalty encourages sparsity arguments; incentivizing fi match <br />                                           fatp drawing fi constrained low. Similar <br />                                           motivations &#226;&#8222;&#8220;1 norm use robust regularized regression [27, 28].                                           contrast, &#226;&#8222;&#8220;2 norm does penalize small errors, draws solution outliers. <br />                                           &#226;&#8222;&#8220;2 norm numerically stable, failing reliable results                                           metabolic  <br />  <br />                                           ATP production fatp used representative metabolic function, based biological <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           importance large process requirement mi. Including ATP explicitly objective                                           consistent previous FBA implementations include ATP rate yield                                           optimization criteria maintenance energy flux constraints [6]. Alternatives choice <br />                                           include functions fi values mean, prone trade-offs                                           metabolites, especially orders magnitude spanned values mi. <br />  <br />                                           additionally introduce constraint fatp &#226;&#8240;&#165; fi, resulting simplification                                           difference penalty terms.  constraint optional &#226;&#8222;&#8220;1 norm penalties                                           used subsequent equations instead. ATP coefficient mATP based bulk <br />                                           yield measurements accounts thermodynamic inefficiencies                                           metabolism, chosen maintain strictly fatp &#226;&#8240;&#165; fi constraint.                                           include optional biomass reaction flux weight &#206;&#178; optimization criteria,                                           problem  <br />  <br />  <br />  <br />  <br />                                                                                                                         (5) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           biomass group term gbio, included su ciently large &#206;&#178;, assures                                           metabolite produced using biomass reaction FBA. Including                                           biomass group alters exFBA simulation pathway partially restricted <br />                                           upstream branch multiple process reactant metabolites, discuss subsequently. <br />                                           problem Equation 5 forced biomass reaction solution setting <br />                                           0 &#226;&#8240;&#164; gbio = fi. <br />  <br />                                           Basic flexFBA simulation results shown Figure 1b identical metabolite <br />                                           production biomass reaction FBA wild-type network. glycogen synthesis <br />                                           capacity removed simulated gene knockout, biomass reaction predicts  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                            Page 6 <br />  <br />  <br />                                           metabolite production, flexFBA predicts wild type level production                                           metabolites glycogen. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Simulations Figure 2 exemplify contribution biomass group gbio term. <br />                                           Pathway restriction makes metabolic precursor tyrosine phenylalanine limited <br />                                           resource. E. coli biomass contains slightly greater amounts phenylalanine tyrosine <br />                                           (mtyr &lt; mphe), 57% flux branch point heading phenylalanine.                                           process requirement tyrosine smaller means increment fractional <br />                                           fulfillment costly terms limited precursor flexible terms (fatp                                           penalty Equation (4) produce ftyr &gt; fphe. biomass group incentivizes <br />                                           proportional production extent flux permitted: ftyr = fphe single <br />                                           knockdown (left columns). contribution limited restricted process <br />                                           metabolites, just biomass reaction  preference                                           flexFBA smaller requirement downstream restricted branch biomass <br />                                           group additionally constrained &#226;&#8364;&#8221; Figure 2 glycogen pathway restriction                                           double knockdown (right columns, Figure 2). <br />  <br />                                           addition &#206;&#178;gbio flexible objective ways increase fi similarity <br />                                           subsets process metabolites. Mathematically terms positively <br />                                           weight group production biomass group term. Taking example reactants <br />                                           DNA synthesis, entail adding &#206;&#178;dNTPgdNTP maximization criteria NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           constraint gdNTP &#226;&#8240;&#164; fj j = {<span id='am-23' about='obo:IMR_0200129' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-24' property="oboInOwl:hasDbXref" content="KEGG:C00131" datatype="xsd:string"></span><span id='am-25' property="oboInOwl:id" content="IMR:0200129" datatype="xsd:string"></span><span id='am-26' property="rdfs:label" content="dATP" datatype="xsd:string"></span><span id='am-27' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>dATP</span>, <span id='am-18' about='obo:IMR_0200084' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-19' property="oboInOwl:id" content="IMR:0200084" datatype="xsd:string"></span><span id='am-20' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-21' property="oboInOwl:hasDbXref" content="KEGG:C00459" datatype="xsd:string"></span><span id='am-22' property="rdfs:label" content="dTTP" datatype="xsd:string"></span>dTTP</span>, <span id='am-13' about='obo:IMR_0200286' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-14' property="rdfs:label" content="dGTP" datatype="xsd:string"></span><span id='am-15' property="oboInOwl:id" content="IMR:0200286" datatype="xsd:string"></span><span id='am-16' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-17' property="oboInOwl:hasDbXref" content="KEGG:C00286" datatype="xsd:string"></span>dGTP</span>, <span id='am-99' about='obo:IMR_0200235' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-100' property="oboInOwl:hasDbXref" content="KEGG:C00458" datatype="xsd:string"></span><span id='am-101' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-102' property="rdfs:label" content="dCTP" datatype="xsd:string"></span><span id='am-103' property="oboInOwl:id" content="IMR:0200235" datatype="xsd:string"></span>dCTP</span>} problem Equation 5. <br />                                           Similarly, deviations group penalized using form like flexible <br />                                           objective itself, strict limit imposed members fj &#226;&#8240;&#164; mink fk, j, k = {dATP, dTTP, <br />                                           dGTP, dCTP}. additional terms apply assumption varying strictness <br />                                           molecular interactions maintain proportion metabolites timescale <br />                                           FBA evaluation.  assigning groups weighting terms increases                                           parameterization potentially complicates optimization problem. consider current <br />                                           experimental evidence insufficient guide justify including groups generally. <br />  <br />                                2.3. Time-Linked Byproducts: tFBA <br />                                           Biomass reaction FBA represent steady states growth transients <br />                                           steady states. limitation process byproducts returned                                           reactants provided processes optimization time step. tFBA                                           byproducts time step t available metabolic network time step t + &#206;&#8221;t                                           modification remove long term steady state assumption biomass <br />                                           reaction. <br />  <br />                                           Additional motivation tFBA construct integrated models advantageous NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           separate process reactant requirements byproduct return. biomass reaction                                           thought integrated model lumping cell processes  biomass <br />                                           reaction expressed single linear equation, included linear                                           optimization.  possible represent processes differential equation form, <br />                                           especially stochastic processes, framework. required evaluate <br />                                           FBA optimization process models separately, simulation include methods                                           assure metabolic solution exists overall simulation conserves mass &#226;&#8364;&#8221;                                           processes metabolism consistent. solution maintaining process-metabolism <br />                                           consistency preserving byproduct reactant ratio FBA optimization <br />                                           required multiple evaluations simulated time step. applied solution <br />                                           FBA integrated modeling Birch et al. 2012 [8], computationally <br />                                           inefficient, prone time step dependent artifacts, problematic                                           stochastic models. preferable alternative tFBA: process-metabolism consistency  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                               Page 7 <br />  <br />  <br />                                           enforced straightforward metabolite quantities separating process reactant <br />                                           consumption byproduct return occur time steps. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           biomass reaction summarizes molecular steps share reactants                                           byproducts, mi summarizes reactant byproduct stoichiometric coefficient ri pi <br />                                           respectively: mi = pi + ri, ri &lt; 0 pi &gt; 0. tFBA replace mi reactant <br />                                           coefficient ri reactions appended S matrix, <br />  <br />  <br />  <br />  <br />                                                                                                                                          (6) <br />  <br />                                           separate exchange reactions xi process byproducts added, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           matrix flux vector denoted&#197;&#339; v&#204;&#8218;. bounds byproduct exchanges xl &#226;&#8240;&#164; x &#226;&#8240;&#164; 0 <br />                                           set uptake, based available byproduct metabolite returned                                           processes prior time step. case simple proportionality used                                           represent processes, steady state wild type network exchange reactions                                           flux bounds values xi = xl &#226;&#8364;&#8220; pif f fractional fulfillment                                           metabolites steady state. transient metabolic states bounds xl                                           depend ci,j, fj,(t&#226;&#8364;&#8220;&#206;&#8221;t) fractional fulfillment related reactant Mj previous <br />                                           time step. Note maintain FBA steady state assumption respect metabolic <br />                                           intermediates, addresses intermediate timescale fluctuations                                           metabolite concentrations enzyme-small molecule interactions respond quickly, <br />                                           longer regulatory responses. optimization problem statement tFBA  <br />  <br />  <br />  <br />  <br />                                                                                                                      (7) <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           case flexFBA used. implement tFBA retain fixed <br />                                           proportion reactants constraint Equation 7 0 &#226;&#8240;&#164; gbio = fi. <br />  <br />                                           present impact tFBA method simulations, compare previous <br />                                           methods generating FBA time courses. methods consist updating media <br />                                           concentrations based resources consumed discrete time steps,                                           thorough accounting called dynamic FBA (dFBA) Mahadevan et. al. 2002 [2].                                           simulations capture organism growth rate shifts time step resources                                           exhausted.  time step growth steady state respect biomass <br />                                           reaction, resulting time course sequence growth steady states. Figure 3a b <br />                                           illustrate reactant-byproduct relationship time course biomass reaction FBA <br />                                           steady states, tFBA, respectively. change conditions, biomass reaction FBA <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 8 <br />  <br />  <br />                                            byproduct return informed new conditions &#226;&#8364;&#8221; arrow                                            reactants byproducts Figure 3a appears reversed respect time &#226;&#8364;&#8221;                                            immediately achieves steady state. comparison, tFBA byproduct return function NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            reactant consumption previous time step condition. result, transitions                                            number evaluations new steady state. steady-state growth, time steps                                            identical result obtained long time limit. <br />  <br />                                            Figures 3c d process reactants byproducts example transition                                            acetate glucose media conditions biomass reaction FBA tFBA simulations. <br />                                            Transition time distinction, tFBA accounts difference                                            resources needed maintain states. E. coli network purine energy carriers ATP <br />                                            ADP important example. Using biomass reaction FBA,                                            modeling assumptions additional ADP assumed exist soon                                            phosphorylated glucose media condition, tFBA simulation includes synthesis <br />                                            additional purine molecules metabolic network. higher glucose growth <br />                                            rate requires twice number ATP plus ADP sustain [29], certainly preferable                                            account resources used synthesis. Figure 3 simulations employ                                            wild type network necessary media resources available produce balanced growth, <br />                                            flexFBA biomass reaction objective produce results shown. <br />  <br />                                            note quantities reactants byproducts exchanged metabolism <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            processes representative physiological metabolite concentrations. <br />                                            quantities accounting practicality representing discrete metabolic steps                                            cell function linear representation.  variables                                            metabolism-process interface provide opportunity methods development                                            FBA cell modeling fields. <br />  <br />                                2.4. Implementation <br />                                            genome scale metabolic model used main text figures slight expansion                                            iMC1010 [30]. Text files used reaction network modeling information                                            included code associated publication. Kinetic bounds reaction <br />                                            perturbations included source code. Simulations implemented Python,                                            linear programming completed using CVXOPT [31] GLPK MOSEK. Source code                                            simulation figure generation available simtk.org/home/flexfbatfba. <br />  <br />                                            Simulations silico knockouts entail setting associated reaction flux constraints vl = <br />                                            vu = 0. tFBA simulations, use process byproduct protons metabolism required. <br />                                            Fluxes accounted simulations cell gram dry cell <br />                                            weight basis, displayed unless noted. &#206;&#178; nonzero hold NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            value large, &#206;&#178; &#226;&#8240;&#171; 1. <br />  <br />                                            weighting penalty terms 0 &lt; &#206;&#179; &lt; 1 production biomass <br />                                            metabolites ATP, tighter lower upper bounds depending                                            metabolic network biomass metabolite count (theoretically &#206;&#179; &#226;&#8240;&#165; 1 feasible                                            practically interferes solver). main text simulations work value 0.1 <br />                                            used, chosen based total count process reactants terms avoid solution <br />                                            convergence biomass reaction limit. analysis metabolite production sensitivity <br />                                            &#206;&#179; (Supplementary Figure S1) shows value chosen produce                                            desired qualitative flexFBA solution, robust various conditions multiple <br />                                            perturbations. Figure S1 includes simulations physiologically reasonable range                                            growth conditions, range metabolites constrained low production. <br />  <br />  <br />  <br />  <br />                                                 J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 9 <br />  <br />  <br />                                3. Results <br />                                3.1. Knockouts <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             Using combined flexFBA tFBA achieve short-time FBA, simulate                                             metabolic network dynamic response silico gene knockout perturbation.                                             applied knockouts, observed broad types simulation results, examples                                             presented Figure 4. <br />  <br />                                             knockouts converge immediately steady state, Figure 4a. example <br />                                             pgsA catalyzes step synthesis E. coli membrane components cardiolipin <br />                                             phosphatidylglycerol. Immediately constraining pgsA flux zero,                                             associated process reactant production zero  left Figure 4a) continues                                             long time limits  right Figure 4a).  unrelated biomass reactants                                             byproducts continue produced  left right Figure 4a). Lack cardiolipin <br />                                             phosphatidylglycerol pgsA knockout simulation consistent experimental <br />                                             evidence strain shown lack membrane components [32]                                             survive conditions. <br />  <br />                                             smaller group knockouts display qualitatively different metabolite synthesis <br />                                             immediately long perturbation, exemplified purA Figure 4b. Flux <br />                                             reaction novo purine synthesis set zero, short term <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             purines provided processes  right Figure 4b). Continued purine <br />                                             availability tFBA purines returned previous step byproduct spent <br />                                             energy carrier ADP. Long-term gradual decrease occurs time step                                             returned purines sequestered cell macromolecules RNA `soluble pool&apos; <br />                                             maintenance &#226;&#8364;&#8221; mathematically mamp + matp &lt; &#226;&#710;&#8217;madp &#226;&#8364;&#8221; ability synthesize <br />                                             <span id='am-64' about='obo:IMR_0100126' typeof='owl:Thing obo:IMR_0000967 obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000965 obo:IMR_0001697 obo:IMR_0000001 obo:IMR_0001698'><span id='am-65' property="oboInOwl:hasExactSynonym" content="Adenosine" datatype="xsd:string"></span><span id='am-66' property="oboInOwl:hasExactSynonym" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-67' property="oboInOwl:id" content="IMR:0100126" datatype="xsd:string"></span><span id='am-68' property="rdfs:label" content="Adenine riboside" datatype="xsd:string"></span><span id='am-69' property="oboInOwl:hasExactSynonym" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-70' property="oboInOwl:hasAlternativeId" content="IMR:0200246" datatype="xsd:string"></span><span id='am-71' property="rdfs:label" content="9-B-D-ribofuranosyl-9H-purin-6-amine" datatype="xsd:string"></span><span id='am-72' property="rdfs:label" content="adenosine" datatype="xsd:string"></span><span id='am-73' property="oboInOwl:hasDbXref" content="KEGG:C00212" datatype="xsd:string"></span><span id='am-74' property="obo:IAO_0000115" content="A nucleoside that is composed of adenine and d-ribose. Adenosine or adenosine derivatives play many important biological roles in addition to being components of DNA and RNA. Adenosine itself is a neurotransmitter." datatype="xsd:string"></span><span id='am-75' property="oboInOwl:hasExactSynonym" content="Adenine riboside" datatype="xsd:string"></span><span id='am-76' property="rdfs:label" content="Adenosine" datatype="xsd:string"></span><span id='am-77' property="rdfs:label" content="Ribofuranosyladenine" datatype="xsd:string"></span><span id='am-78' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>adenosine</span> small molecule nutrients free declines  right Figure 4b). <br />                                             Based flexFBA constraint fm &lt; fatp, represents biological dependance                                             metabolic activity ATP, biomass reactants decrease proportionally  left <br />                                             right Figure 4b). Experimentally, purA knockouts purine auxotrophs [33], consistent <br />                                             flexFBA tFBA phenotype predictions. <br />  <br />                                             flexFBA tFBA techniques allow compare knockout simulations experiment <br />                                             metabolite individually, binary essential/non-essential gene <br />                                             classifications biomass reaction FBA. Previously, detailed predictions                                             possible testing media supplements simulations,                                             computational searches FBA solution space [34]. Figure 5 emphasizes                                             increase prediction techniques summarizing results like Figure <br />                                             4. Figure 5, genes knocked computationally true-essential predictions glucose <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                             minimal media Feist et al. 2007 [24]. Process reactant absence indicated dark <br />                                             shading metabolite, short-term long-term border. contrast                                             results analogous biomass reaction results (small grids Figure 5): using                                             biomass reaction useful contained completely shaded grid (right Figure 5). <br />  <br />                                             make graded assessments metabolic network predictions. knockout                                             historically called true-essential biomass reaction flux value                                             threshold gene experimentally necessary growth. true-essential <br />                                             matches considered metric quality metabolic network reconstructions. <br />                                              provides information prediction correct relevant <br />                                             biological reason. compared literature information flexFBA metabolites absences                                             Figure 5, indicate agreement color (associated references Table S1). strains <br />                                             missing biomass metabolites analogous auxotrophs,                                             experimental literature confirms media supplemented metabolites <br />  <br />  <br />                                                  J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                              Page 10 <br />  <br />  <br />                                            unavailable simulation precursors. genes experimental evidence                                            consistent silico phenotype, expected critical components                                            exchanged media, assigned  agreement&apos; classification.  NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            agreement&apos; group consists membrane components, electron transport chain <br />                                            enzymes, <span id='am-44' about='obo:IMR_0001349' typeof='owl:Thing obo:IMR_0001657 obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001698'><span id='am-45' property="rdfs:label" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-46' property="rdfs:label" content="nucleotide" datatype="xsd:string"></span><span id='am-47' property="oboInOwl:hasExactSynonym" content="nucleoside phosphate" datatype="xsd:string"></span><span id='am-48' property="oboInOwl:id" content="IMR:0001349" datatype="xsd:string"></span><span id='am-49' property="obo:IAO_0000115" content="A nucleotide consists of a nitrogen-containing base, a five-carbon sugar, and one or more phosphate groups. Nucleotides can carry chemical energy (e.g. ATP), form coenzymes (e.g. CoA), be used as specific signaling molecules (e.g. cAMP) in the cell. Nucleotides are the subunits of the nucleic acids. " datatype="xsd:string"></span><span id='am-50' property="oboInOwl:hasDbXref" content="CHEBI:25608" datatype="xsd:string"></span><span id='am-51' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>nucleotide</span> kinases. <br />  <br />                                            Shown pie inset Figure 5, majority true-essential predictions occur                                            biologically relevant consequences metabolic network. agreement means                                            detailed results largely consistent accuracy classification implied `true&apos;                                            true-essential biomass reaction predictions. 14 metabolic phenotypes                                            partially inconsistent biochemical genetic evidence. phenotypes include <br />                                            cases FBA technically correct predictions essentiality, faulty <br />                                            reasoning. example metK knockout <span id='am-94' about='obo:IMR_0200175' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-95' property="oboInOwl:id" content="IMR:0200175" datatype="xsd:string"></span><span id='am-96' property="rdfs:label" content="Spermidine" datatype="xsd:string"></span><span id='am-97' property="oboInOwl:hasDbXref" content="KEGG:C00315" datatype="xsd:string"></span><span id='am-98' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span>spermidine</span> synthesis blocked                                            silico, preventing biomass reaction flux. zero biomass reaction flux traditional <br />                                            FBA prediction metK essential gene, matches metK essentiality                                            experimentally.  metK knockout lethal spermidine absence,                                            strains isolated polyamines [35]. Instead, reason metK knockout <br />                                            lethal involves production critical cofactors required methionine biosynthesis <br />                                            [36]. cofactors used cycles, means FBA `balance&apos; fluxes <br />                                            cofactors produced. new approach allows resolve                                            discrepancy biochemical evidence silico phenotype, NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            previous FBA methods claimed correct prediction. folA knockout prediction                                            analogous case metK; serA, serB, serC knockouts demonstrate opposite <br />                                            phenomena cycles possible FBA cells disrupted.                                            number incorrect predictions involve cofactor cycling, folA metK <br />                                            knockouts able produce metabolites synthesis reality prevented.                                            cases serA, serB, serC knockouts balance interconversion acetyl- <br />                                            <span id='am-39' about='obo:IMR_0200354' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-40' property="oboInOwl:id" content="IMR:0200354" datatype="xsd:string"></span><span id='am-41' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-42' property="oboInOwl:hasDbXref" content="KEGG:C00010" datatype="xsd:string"></span><span id='am-43' property="rdfs:label" content="CoA" datatype="xsd:string"></span>CoA</span> CoA perturbed, FBA apparently sensitive metabolism                                            vivo. <br />  <br />                                3.2. Expression Bursts <br />                                            Figure 6 compares results short-time FBA biomass reaction simulations                                            case enzyme copy number fluctuations single cell. process reactant metabolites <br />                                            required cell small amounts, enzymes produce exist                                            order tens cell. Increasingly, FBA approaches refined addition                                            enzymatic parameters bounds calculation [14], example using protein counts                                            turnover number constrain internal network fluxes [10]. kinetic flux bounds                                            applied single cell integrated models, flexFBA required avoid biomass reaction <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                            artifacts low copy number enzymes. <br />  <br />                                            Coenzyme (CoA) small process requirement metabolite example Figure 6 <br />                                            simulations, total inclusion biomass acetyland <span id='am-52' about='obo:IMR_0200037' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001 obo:IMR_0001998'><span id='am-53' property="oboInOwl:hasDbXref" content="KEGG:C00091" datatype="xsd:string"></span><span id='am-54' property="oboInOwl:id" content="IMR:0200037" datatype="xsd:string"></span><span id='am-55' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-56' property="rdfs:label" content="Succinyl-CoA" datatype="xsd:string"></span>succinyl-CoA</span>                                            orders magnitude smaller process requirement ATP. essential step CoA <br />                                            synthesis performed coaBC, observed average                                            cell, produced transcription events cell cycle [16]. input                                            approximation coaBC expression translational bursts, informs kinetic bounds                                            CoA synthesis shown Figure 6a b, compared kinetic bounds smoothly <br />                                            increase according population average. biomass reaction flexFBA predictions <br />                                            CoA synthesis rate similar, restricted kinetic bounds single <br />                                            cell lower smooth population average (Figure 6a b). difference                                            CoA synthesis constrained smooth level, using biomass reaction, process <br />                                            reactant metabolite synthesis constrained (Figure 6c), unperturbed  <br />  <br />                                                 J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                      Page 11 <br />  <br />  <br />                                           CoA limits using flexFBA (Figure 6d). Biomass reaction accumulation macromolecular <br />                                           precursors lowered periods restriction. deficit seen slightly lower <br />                                           fluxes compared smooth case coaBC enzyme limiting. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                4. Discussion Conclusions <br />                                           constructed FBA objective able produce subsets process <br />                                           reactants, reproduces traditional biomass production wild-type network case. <br />                                           addition, time-linked simulations allow observe transitions FBA steady <br />                                           states. Critically, methods satisfy requirement quick unsupervised operation                                           used integrated modeling applications, using shelf                                           open-source optimization packages. Furthermore, flexFBA functions robustly                                           single added adjustable parameter penalty term weighting &#206;&#179;. additional strength                                           penalty weighting approaches question asked converges                                            metabolism make&apos; classic population survival assessment. <br />  <br />                                           detailed results flexFBA tFBA silico knockout predictions offer new <br />                                           window metabolic reconstructions. Correctness incorrectness essentiality                                           metabolic phenotype predictions properties metabolic reconstruction gene <br />                                           associations. methods make possible identify correct predictions                                           wrong reason, indicating problem metabolic reconstruction. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           Identification erroneous true-essential predictions important cause <br />                                           problems applications FBA. example, metabolic engineering design <br />                                           lead misguided computational suggestions strain development efforts. <br />  <br />                                           short-time FBA necessary avoid artifacts apply FBA short-time scale <br />                                           single-cell level enzyme count kinetic parameters constraints. Using biomass <br />                                           reaction, models predict cell constrains distant pathways, including large fluxes <br />                                           catalyzed high copy number enzymes, match transcriptional fluctuations single <br />                                           rare protein. second timescales, strict constraint unreasonable                                           evolutionary perspective mechanistic understanding. Furthermore,                                           coaBC example saw impact reaction limited enzyme produced                                           bursts, exist  simulation simultaneously, result                                           dramatic limitation. methods introduce FBA solution avoids propagating <br />                                           low copy enzyme bounds implausibly. <br />  <br />                                           biomass reaction previously limited application genome scale stoichiometric <br />                                           metabolic models population average steady state growth. FlexFBA tFBA <br />                                           bring relevant timescale intermediate range allow NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           represent cell physiological  combined short-time FBA instrumental <br />                                           cell simulations understanding heterogeneity underlies critical <br />                                           phenomena microbiology. <br />  <br />                                Supplementary Material <br />                                           Refer Web version PubMed Central supplementary material. <br />  <br />  <br />                                Acknowledgments <br />                                           thank Derek Macklin Jonathan Karr helpful discussions methods presentation. work                                           supported NIH Directors Pioneer Award (5DP1LM01150-05), Allen Distinguished Investigator Award,                                           Hellman Faculty Scholarship M.W.C; Stanford Bio-X Fellowship E.W.B; NSF Graduate Research <br />                                           Fellowship DGE-1147470 Gabilan Stanford Graduate Fellowship Funds M.U. <br />  <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                  Page 12 <br />  <br />  <br />                                References <br />                                           [1]. Varma  Palsson BO. Metabolic capabilities Escherichia coli:  synthesis biosynthetic <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 precursors cofactors. J Theor Biol. 1993; 165(4):477&#226;&#8364;&#8220;502. [PubMed: 21322280] <br />                                           [2]. Mahadevan RR, Edwards JSJ, Doyle FJF. Dynamic flux balance analysis diauxic growth                                                 Escherichia coli. Biophys J. 2002; 83(3):1331&#226;&#8364;&#8220;1340. [PubMed: 12202358] <br />                                           [3]. Covert MW, Schilling CH, Famili  Edwards JS, Goryanin II, Selkov E, Palsson BO. Metabolic <br />                                                 modeling microbial strains silico. Trends Biochem Sci. 2001; 26(3):179&#226;&#8364;&#8220;186. [PubMed: <br />                                                 11246024] <br />                                           [4]. Edwards JS, Palsson BO. Metabolic flux balance analysis silico analysis Escherichia <br />                                                 coli K-12 gene deletions. BMC Bioinformatics. 2000; 1:1. [PubMed: 11001586] <br />                                           [5]. Raman K, Chandra N. Flux balance analysis biological systems: applications challenges. <br />                                                 Briefings Bioinformatics. 2009; 10(4):435&#226;&#8364;&#8220;449. [PubMed: 19287049] <br />                                           [6]. Schuetz R, Kuepfer L, Sauer U. Systematic evaluation objective functions predicting <br />                                                 intracellular fluxes Escherichia coli. Molecular Systems Biology. 2007; 3:119&#226;&#8364;&#8220;119. [PubMed: <br />                                                 17625511] <br />                                           [7]. Segr&#195;&#168; D, Vitkup D, Church GM. Analysis optimality natural perturbed metabolic <br />                                                 networks. Proc Natl Acad Sci USA. 2002; 99(23):15112&#226;&#8364;&#8220;15117. [PubMed: 12415116] <br />                                           [8]. Birch EW, Ruggero NA, Covert MW. Determining Host Metabolic Limitations Viral <br />                                                 Replication Integrated Modeling Experimental Perturbation. PLoS Comput Biol. 2012; <br />                                                 8(10):e1002746. [PubMed: 23093930] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [9]. Gon&#195;&#167;alves E, Bucher J, Ryll  Niklas J, Mauch K, Klamt S, Rocha M, Saez-Rodriguez J. <br />                                                 Bridging layers: integration signal transduction, regulation metabolism                                                 mathematical models. Mol. BioSyst. <br />                                           [10]. Karr JR, Sanghvi JC, Macklin DN, Gutschow MV, Jacobs JM, Bolival B, Assad-Garcia N, Glass <br />                                                 JI, Covert MW. cell computational model predicts phenotype genotype. Cell. <br />                                                 2012; 150(2):389&#226;&#8364;&#8220;401. [PubMed: 22817898] <br />                                           [11]. Feist  Palsson BO. biomass objective function. Curr Opin Microbiol. 2010; 13(3):344&#226;&#8364;&#8220; <br />                                                 349. [PubMed: 20430689] <br />                                           [12]. Harcombe WR, Delaney NF, Leiby N, Klitgord N, Marx CJ. Ability Flux Balance <br />                                                 Analysis Predict Evolution Central Metabolism Scales Initial Distance                                                 Optimum. PLoS Comput Biol. 2013; 9(6):e1003091. [PubMed: 23818838] <br />                                           [13]. Burgard AP, Maranas CD. Optimization-based framework inferring testing hypothesized <br />                                                 metabolic objective functions. Biotechnol Bioeng. 2003; 82(6):670&#226;&#8364;&#8220;677. [PubMed: 12673766] <br />                                           [14]. Reed JL. Shrinking Metabolic Solution Space Using Experimental Datasets. PLoS Comput <br />                                                 Biol. 2012; 8(8):e1002662. [PubMed: 22956899] <br />                                           [15]. Zomorrodi AR, Suthers PF, Ranganathan S, Maranas CD. Mathematical optimization <br />                                                 applications metabolic networks. Metab Eng. 2012; 14(6):672&#226;&#8364;&#8220;686. [PubMed: 23026121] <br />                                           [16]. Taniguchi Y, Choi PJ, Li GW, Chen H, Babu M, Hearn J, Emili  Xie XS. Quantifying E. coli <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Proteome Transcriptome Single-Molecule Sensitivity Single Cells. Science. 2010; <br />                                                 329(5991):533&#226;&#8364;&#8220;538. [PubMed: 20671182] <br />                                           [17]. Lidstrom  Konopka MC. role physiological heterogeneity microbial population <br />                                                 behavior. Nat Chem Biol. 2010; 6(10):705&#226;&#8364;&#8220;712. [PubMed: 20852608] <br />                                           [18]. Covert MW, Palsson B&#195;&#732;. Transcriptional regulation constraints-based metabolic models                                                 Escherichia coli. J Biol Chem. 2002; 277(31):28058&#226;&#8364;&#8220;28064. [PubMed: 12006566] <br />                                           [19]. Covert MW, Xiao N, Chen TJ, Karr JR. Integrating metabolic, transcriptional regulatory                                                 signal transduction models Escherichia coli. Bioinformatics. 2008; 24(18):2044&#226;&#8364;&#8220;2050. <br />                                                 [PubMed: 18621757] <br />                                           [20]. Jackowski S, Rock  Regulation coenzyme biosynthesis. Journal Bacteriology. <br />                                                 1981; 148(3):926&#226;&#8364;&#8220;932. <br />                                           [21]. Goldstein  Goldstein DB, Brown BJ, Chou S-C. <span id='am-57' about='obo:IMR_0001695' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-58' property="oboInOwl:hasDbXref" content="CHEBI:33709" datatype="xsd:string"></span><span id='am-59' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-60' property="oboInOwl:id" content="IMR:0001695" datatype="xsd:string"></span><span id='am-61' property="oboInOwl:hasDbXref" content="MeSH:D000596" datatype="xsd:string"></span><span id='am-62' property="obo:IAO_0000115" content="Organic compounds that generally contain an amino (-NH2) and a carboxyl (-COOH) group. Twenty alpha-amino acids are the subunits which are polymerized to form proteins." datatype="xsd:string"></span><span id='am-63' property="rdfs:label" content="amino acid" datatype="xsd:string"></span>Amino acid</span> starvation Escherichia coli <br />                                                 auxotroph.  E ects protein nucleic acid synthesis cell devision. Biochim Biophys <br />                                                 Acta. 1959; 36:163&#226;&#8364;&#8220;172. [PubMed: 13850757] <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                                   Page 13 <br />  <br />  <br />                                           [22]. Goss WA, Deitz WH, Cook TM. Mechanism Action Naladixic Acid Escherichia coli. <br />                                                 Journal Bacteriology. 1964; 88:1112&#226;&#8364;&#8220;1118. <br />                                           [23]. Ohashi Y, Hirayama  Ishikawa T, Nakamura S, Shimizu K, Ueno Y, Tomita M, Soga T. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                 Depiction metabolome changes histidine-starved Escherichia coli CE-TOFMS. Mol. <br />                                                 BioSyst. 2008; 4(2):135. [PubMed: 18213407] <br />                                           [24]. Feist  Henry CS, Reed JL, Krummenacker M, Joyce AR, Karp PD, Broadbelt LJ, <br />                                                 Hatzimanikatis V, Palsson B&#195;&#732;. genome-scale metabolic reconstruction Escherichia coli <br />                                                 K-12 MG1655 accounts 1260 ORFs thermodynamic information. Molecular Systems <br />                                                 Biology. 2007; 3:121. [PubMed: 17593909] <br />                                           [25]. Nookaew  Jewett MC, Meechai  Thammarongtham C, Laoteng K, Cheevadhanarak S, <br />                                                 Nielsen J, Bhumiratana S. genome-scale metabolic model iIN800 Saccharomyces <br />                                                 cerevisiae validation: scaffold query <span id='am-7' about='obo:IMR_0001362' typeof='owl:Thing obo:IMR_0000947 obo:IMR_0000001'><span id='am-8' property="rdfs:label" content="lipid" datatype="xsd:string"></span><span id='am-9' property="oboInOwl:hasDbXref" content="CHEBI:25054" datatype="xsd:string"></span><span id='am-10' property="oboInOwl:id" content="IMR:0001362" datatype="xsd:string"></span><span id='am-11' property="oboInOwl:hasOBONamespace" content="molecule_role" datatype="xsd:string"></span><span id='am-12' property="obo:IAO_0000115" content="Organic molecule that is insoluble in water but tends to dissolve in nonpolar organic solvent." datatype="xsd:string"></span>lipid</span> metabolism. BMC Syst Biol. 2008; 2(1):71. <br />                                                 [PubMed: 18687109] <br />                                           [26]. Heavner BD, Smallbone K, Barker B, Mendes P, Walker LP. Yeast 5 &#226;&#8364;&#8220; expanded <br />                                                 reconstruction Saccharomyces cerevisiae metabolic network. BMC Syst Biol. 2012; 6(1): <br />                                                 55. [PubMed: 22663945] <br />                                           [27]. Boyd, S.; Vandenberghe, L. Convex optimization. <br />                                           [28]. Sra, S.; Nowozin, S.; Wright, SJ. Optimization Machine Learning. MIT Press; 2012. <br />                                           [29]. Bennett BD, Kimball EH, Gao M, Osterhout R, Van Dien SJ, Rabinowitz JD. Absolute <br />                                                 metabolite concentrations implied enzyme active site occupancy Escherichia coli. Nature <br />                                                 Publishing Group. 2009; 5(8):593&#226;&#8364;&#8220;599. <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           [30]. Covert MW, Knight EM, Reed JL, Herrgard MJ, Palsson BO. Integrating high-throughput                                                 computational data elucidates bacterial networks. Nature. 2004; 429(6987):92&#226;&#8364;&#8220;96. [PubMed: <br />                                                 15129285] <br />                                           [31]. Andersen, MS.; Dahl, J.; Vandenberghe, L. CVXOPT: Python package convex <br />                                                 optimization. <br />                                           [32]. Mileykovskaya E, Ryan AC, Mo X, Lin C-C, Khalaf KI, Dowhan W, Garrett TA. Phosphatidic <br />                                                 acid N-acylphosphatidylethanolamine form membrane domains Escherichia coli mutant <br />                                                 lacking cardiolipin phosphatidylglycerol. J Biol Chem. 2009; 284(5):2990&#226;&#8364;&#8220;3000. [PubMed: <br />                                                 19049984] <br />                                           [33]. Zalkin, H. Escherichia coli Salmonella. American Society Microbiology; Washington, <br />                                                 DC: 1996. Biosynthesis purine nucleotides. <br />                                           [34]. Imielinski M, Belta C, Halasz  Rubin H. Investigating metabolite essentiality genome- <br />                                                 scale analysis Escherichia coli production capabilities. Bioinformatics. 2005; 21(9):2008&#226;&#8364;&#8220; <br />                                                 2016. [PubMed: 15671116] <br />                                           [35]. Tabor CW, Tabor H. Polyamines microorganisms. Microbiol Rev. 1985; 49(1):81. [PubMed: <br />                                                 3157043] <br />                                           [36]. Wei Y, Newman EB. Studies role metK gene product Escherichia coli K-12. <br />                                                 Molecular Microbiology. 2002; 43(6):1651&#226;&#8364;&#8220;1656. [PubMed: 11952912] <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                                J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                               Page 14 <br />  <br />  <br />  <br />                                           Highlights <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           rigid Flux Balance Analysis (FBA) biomass reaction hinders cell <br />                                           modeling. <br />                                           New flexible FBA produce subsets biomass reactants. <br />                                           Time-linked FBA removes reactant byproduct long-time assumption. <br />                                           new methods avoid low-copy enzyme metabolic artifacts cell <br />                                           modeling. <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                                           J Theor Biol. Author manuscript; available PMC 2015 March 21. <br />                             Birch et al.                                                                                       Page 15 <br /> NIH-PA Author Manuscript <br /> NIH-PA Author Manuscript <br />  <br />  <br />  <br />  <br />                         <br /> </body></html>